---
title: Methods and compositions for detecting target sequences
abstract: The present invention relates to compositions and methods for the detection and characterization of nucleic acid sequences and variations in nucleic acid sequences. The present invention relates to methods for forming a nucleic acid cleavage structure on a target sequence and cleaving the nucleic acid cleavage structure in a site-specific manner. For example, in some embodiments, a 5â€² nuclease activity from any of a variety of enzymes is used to cleave the target-dependent cleavage structure, thereby indicating the presence of specific nucleic acid sequences or specific variations thereof.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=06913881&OS=06913881&RS=06913881
owner: Third Wave Technologies, Inc.
number: 06913881
owner_city: Madison
owner_country: US
publication_date: 20001115
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"p":["This application is a continuation-in-part of U.S. Ser. No. 09\/381,212 filed Feb. 8, 2000, which claims priority to PCT\/US98\/05809 filed Mar. 24, 1998, which is a PCT application that claims priority to U.S. Ser. No. 08\/823,516 filed Mar. 24, 1997, now issued as U.S. Pat. No. 5,994,069 on Nov. 30, 1999, which is continuation-in-part of U.S. Ser. No. 08\/759,038 filed Dec. 2, 1996, now issued as U.S. Pat. No. 6,090,543 on Jul. 18, 2000, which is a continuation-in-part of U.S. Ser. No. 08\/756,386 filed Nov. 29, 1996, now issued as U.S. Pat. No. 5,985,557 on Nov. 16, 1999, which is a continuation-in-part of U.S. Ser. No. 08\/682.853 filed Jul. 12, 1996, now issued as U.S. Pat. No. 6,001.567 on Dec. 14, 1999, which is a continuation-in-part of U.S. Ser. No. 08\/599,491 filed on Jan. 24, 1996, now issued as U.S. Pat. No. 5,846,717 on Dec. 8, 1998. This application is also a continuation-in-part of U.S. Ser. No. 09\/350,309 filed on Jul. 9, 1999, now issued as U.S. Pat. No. 6,348,314 in Feb. 19, 2002, which is a divisional application of U.S. Pat. No. 08\/756,386 filed Nov. 29, 1996, now issued as U.S. Pat. No. 5,985,557 on Nov. 16, 1999.","The present invention relates to compositions and methods for the detection and characterization of nucleic acid sequences and variations in nucleic acid sequences. The present invention relates to methods for forming a nucleic acid cleavage structure on a target sequence and cleaving the nucleic acid cleavage structure in a site-specific manner. For example, in some embodiments, a 5\u2032 nuclease activity from any of a variety of enzymes is used to cleave the target-dependent cleavage structure, thereby indicating the presence of specific nucleic acid sequences or specific variations thereof.","Methods for the detection and characterization of specific nucleic acid sequences and sequence variations have been used to detect the presence of viral or bacterial nucleic acid sequences indicative of an infection, to detect the presence of variants or alleles of genes associated with disease and cancers. These methods also find application in the identification of sources of nucleic acids, as for forensic analysis or for paternity determinations.","Various methods are known to the art that may be used to detect and characterize specific nucleic acid sequences and sequence variants. Nonetheless, with the completion of the nucleic acid sequencing of the human genome, as well as the genomes of numerous pathogenic organisms, the demand for fast, reliable, cost-effective and user-friendly tests for the detection of specific nucleic acid sequences continues to grow. Importantly, these tests must be able to create a detectable signal from samples that contain very few copies of the sequence of interest. The following discussion examines two levels of nucleic acid detection assays currently in use: I. Signal Amplification Technology for detection of rare sequences; and II. Direct Detection Technology for quantitative detection of sequences.","The \u201cPolymerase Chain Reaction\u201d (PCR) comprises the first generation of methods for nucleic acid amplification. However, several other methods have been developed that employ the same basis of specificity, but create signal by different amplification mechanisms. These methods include the \u201cLigase Chain Reaction\u201d (LCR), \u201cSelf-Sustained Synthetic Reaction\u201d (3SR\/NASBA), and \u201cQ\u03b2-Replicase\u201d (Q\u03b2).","Polymerase Chain Reaction (PCR)","The polymerase chain reaction (PCR), as described in U.S. Pat. Nos. 4,683,195, 4,683,202, and 4,965,188 to Mullis and Mullis et al. (the disclosures of which are hereby incorporated by reference), describe a method for increasing the concentration of a segment of target sequence in a mixture of genomic DNA without cloning or purification. This technology provides one approach to the problems of low target sequence concentration. PCR can be used to directly increase the concentration of the target to an easily detectable level. This process for amplifying the target sequence involves introducing a molar excess of two oligonucleotide primers that are complementary to their respective strands of the double-stranded target sequence to the DNA mixture containing the desired target sequence. The mixture is denatured and then allowed to hybridize. Following hybridization, the primers are extended with polymerase so as to form complementary strands. The steps of denaturation, hybridization, and polymerase extension can be repeated as often as needed, in order to obtain relatively high concentrations of a segment of the desired target sequence.","The length of the segment of the desired target sequence is determined by the relative positions of the primers with respect to each other, and, therefore, this length is a controllable parameter. Because the desired segments of the target sequence become the dominant sequences (in terms of concentration) in the mixture, they are said to be \u201cPCR-amplified.\u201d","Ligase Chain Reaction (LCR or LAR)","The ligase chain reaction (LCR; sometimes referred to as \u201cLigase Amplification Reaction\u201d (LAR) described by Barany, Proc. Natl. Acad. Sci., 88:189 (1991); Barany, PCR Methods and Applic., 1:5 (1991); and Wu and Wallace, Genomics 4:560 (1989) has developed into a well-recognized alternative method for amplifying nucleic acids. In LCR, four oligonucleotides, two adjacent oligonucleotides which uniquely hybridize to one strand of target DNA, and a complementary set of adjacent oligonucleotides, that hybridize to the opposite strand are mixed and DNA ligase is added to the mixture. Provided that there is complete complementarity at the junction, ligase will covalently link each set of hybridized molecules. Importantly, in LCR, two probes are ligated together only when they base-pair with sequences in the target sample, without gaps or mismatches. Repeated cycles of denaturation, hybridization and ligation amplify a short segment of DNA. LCR has also been used in combination with PCR to achieve enhanced detection of single-base changes. Segev, PCT Public. No. WO9001069 A1 (1990). However, because the four oligonucleotides used in this assay can pair to form two short ligatable fragments, there is the potential for the generation of target-independent background signal. The use of LCR for mutant screening is limited to the examination of specific nucleic acid positions.","Self-Sustained Synthetic Reaction (3SR\/NASBA)","The self-sustained sequence replication reaction (3SR) (Guatelli et al., Proc. Natl. Acad. Sci., 87:1874-1878 [1990], with an erratum at Proc. Natl. Acad. Sci., 87:7797 [1990]) is a transcription-based in vitro amplification system (Kwok et al., Proc. Natl. Acad. Sci., 86:1173-1177 [1989]) that can exponentially amplify RNA sequences at a uniform temperature. The amplified RNA can then be utilized for mutation detection (Fahy et al., PCR Meth. Appl., 1:25-33 [1991]). In this method, an oligonucleotide primer is used to add a phage RNA polymerase promoter to the 5\u2032 end of the sequence of interest. In a cocktail of enzymes and substrates that includes a second primer, reverse transcriptase, RNase H, RNA polymerase and ribo- and deoxyribonucleoside triphosphates, the target sequence undergoes repeated rounds of transcription, cDNA synthesis and second-strand synthesis to amplify the area of interest. The use of 3SR to detect mutations is kinetically limited to screening small segments of DNA (e.g., 200-300 base pairs).","Q-Beta (Q\u03b2) Replicase","In this method, a probe that recognizes the sequence of interest is attached to the replicatable RNA template for Q\u03b2 replicase. A previously identified major problem with false positives resulting from the replication of unhybridized probes has been addressed through use of a sequence-specific ligation step. However, available thermostable DNA ligases are not effective on this RNA substrate, so the ligation must be performed by T4 DNA ligase at low temperatures (37\u00b0 C.). This prevents the use of high temperature as a means of achieving specificity as in the LCR, the ligation event can be used to detect a mutation at the junction site, but not elsewhere.","Table 1 below, lists some of the features desirable for systems useful in sensitive nucleic acid diagnostics, and summarizes the abilities of each of the major amplification methods (See also, Landgren, Trends in Genetics 9:199 [1993]).","A successful diagnostic method must be very specific. A straight-forward method of controlling the specificity of nucleic acid hybridization is by controlling the temperature of the reaction. While the 3SR\/NASBA, and Q\u03b2 systems are all able to generate a large quantity of signal, one or more of the enzymes involved in each cannot be used at high temperature (i.e., >55\u00b0 C.). Therefore the reaction temperatures cannot be raised to prevent non-specific hybridization of the probes. If probes are shortened in order to make them melt more easily at low temperatures, the likelihood of having more than one perfect match in a complex genome increases. For these reasons, PCR and LCR currently dominate the research field in detection technologies.",{"@attributes":{"id":"p-0014","num":"0013"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}}],"thead":{"row":{"entry":[{},"TABLE 1"]}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]},{"entry":[{},"Method"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},"PCR &","3SR",{}]},{"entry":["Feature","PCR","LCR","LCR","NASBA","Q\u03b2"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}},{"entry":["Amplifies Target","+","+","+","+",{}]},{"entry":["Recognition of Independent","+","+","+","+","+"]},{"entry":"Sequences Required"},{"entry":["Performed at High Temp.","+","+"]},{"entry":["Operates at Fixed Temp.",{},{},{},"+","+"]},{"entry":["Exponential Amplification","+","+","+","+","+"]},{"entry":["Generic Signal Generation",{},{},{},{},"+"]},{"entry":"Easily Automatable"},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]}}]}}},"The basis of the amplification procedure in the PCR and LCR is the fact that the products of one cycle become usable templates in all subsequent cycles, consequently doubling the population with each cycle. The final yield of any such doubling system can be expressed as: (1+X)=y, where \u201cX\u201d is the mean efficiency (percent copied in each cycle), \u201cn\u201d is the number of cycles, and \u201cy\u201d is the overall efficiency, or yield of the reaction (Mullis, PCR Methods Applic., 1:1 [1991]). If every copy of a target DNA is utilized as a template in every cycle of a polymerase chain reaction, then the mean efficiency is 100%. If 20 cycles of PCR are performed, then the yield will be 2, or 1,048,576 copies of the starting material. If the reaction conditions reduce the mean efficiency to 85%, then the yield in those 20 cycles will be only 1.85, or 220,513 copies of the starting material. In other words, a PCR running at 85% efficiency will yield only 21% as much final product, compared to a reaction running at 100% efficiency. A reaction that is reduced to 50% mean efficiency will yield less than 1% of the possible product.","In practice, routine polymerase chain reactions rarely achieve the theoretical maximum yield, and PCRs are usually run for more than 20 cycles to compensate for the lower yield. At 50% mean efficiency, it would take 34 cycles to achieve the million-fold amplification theoretically possible in 20, and at lower efficiencies, the number of cycles required becomes prohibitive. In addition, any background products that amplify with a better mean efficiency than the intended target will become the dominant products.","Also, many variables can influence the mean efficiency of PCR, including target DNA length and secondary structure, primer length and design, primer and dNTP concentrations, and buffer composition, to name but a few. Contamination of the reaction with exogenous DNA (e.g., DNA spilled onto lab surfaces) or cross-contamination is also a major consideration. Reaction conditions must be carefully optimized for each different primer pair and target sequence, and the process can take days, even for an experienced investigator. The laboriousness of this process, including numerous technical considerations and other factors, presents a significant drawback to using PCR in the clinical setting. Indeed, PCR has yet to penetrate the clinical market in a significant way. The same concerns arise with LCR, as LCR must also be optimized to use different oligonucleotide sequences for each target sequence. In addition, both methods require expensive equipment, capable of precise temperature cycling.","Many applications of nucleic acid detection technologies, such as in studies of allelic variation, involve not only detection of a specific sequence in a complex background, but also the discrimination between sequences with few, or single, nucleotide differences. One method for the detection of allele-specific variants by PCR is based upon the fact that it is difficult for Taq polymerase to synthesize a DNA strand when there is a mismatch between the template strand and the 3\u2032 end of the primer. An allele-specific variant may be detected by the use of a primer that is perfectly matched with only one of the possible alleles; the mismatch to the other allele acts to prevent the extension of the primer, thereby preventing the amplification of that sequence. This method has a substantial limitation in that the base composition of the mismatch influences the ability to prevent extension across the mismatch, and certain mismatches do not prevent extension or have only a minimal effect (Kwok et al., Nucl. Acids Res., 18:999 [1990]).)","A similar 3\u2032-mismatch strategy is used with greater effect to prevent ligation in the LCR (Barany, PCR Meth. Applic., 1:5 [1991]). Any mismatch effectively blocks the action of the thermostable ligase, but LCR still has the drawback of target-independent background ligation products initiating the amplification. Moreover, the combination of PCR with subsequent LCR to identify the nucleotides at individual positions is also a clearly cumbersome proposition for the clinical laboratory.","When a sufficient amount of a nucleic acid to be detected is available, there are advantages to detecting that sequence directly, instead of making more copies of that target, (e.g., as in PCR and LCR). Most notably, a method that does not amplify the signal exponentially is more amenable to quantitative analysis. Even if the signal is enhanced by attaching multiple dyes to a single oligonucleotide, the correlation between the final signal intensity and amount of target is direct. Such a system has an additional advantage that the products of the reaction will not themselves promote further reaction, so contamination of lab surfaces by the products is not as much of a concern. Traditional methods of direct detection including Northern and Southern blotting and RNase protection assays usually require the use of radioactivity and are not amenable to automation. Recently devised techniques have sought to eliminate the use of radioactivity and\/or improve the sensitivity in automatable formats. Two examples are the \u201cCycling Probe Reaction\u201d (CPR), and \u201cBranched DNA\u201d (bDNA).","The cycling probe reaction (CPR) (Duck et al., BioTech., 9:142 [1990]), uses a long chimeric oligonucleotide in which a central portion is made of RNA while the two termini are made of DNA. Hybridization of the probe to a target DNA and exposure to a thermostable RNase H causes the RNA portion to be digested. This destabilizes the remaining DNA portions of the duplex, releasing the remainder of the probe from the target DNA and allowing another probe molecule to repeat the process. The signal, in the form of cleaved probe molecules, accumulates at a linear rate. While the repeating process increases the signal, the RNA portion of the oligonucleotide is vulnerable to RNases that may be carried through sample preparation.","Branched DNA (bDNA), described by Urdea et al., Gene 61:253-264 (1987), involves oligonucleotides with branched structures that allow each individual oligonucleotide to carry 35 to 40 labels (e.g., alkaline phosphatase enzymes). While this enhances the signal from a hybridization event, signal from non-specific binding is similarly increased.","While both of these methods have the advantages of direct detection discussed above, neither the CPR or bDNA methods can make use of the specificity allowed by the requirement of independent recognition by two or more probe (oligonucleotide) sequences, as is common in the signal amplification methods described in Section 1, above. The requirement that two oligonucleotides must hybridize to a target nucleic acid in order for a detectable signal to be generated confers an extra measure of stringency on any detection assay. Requiring two oligonucleotides to bind to a target nucleic acid reduces the chance that false \u201cpositive\u201d results will be produced due to the non-specific binding of a probe to the target. The further requirement that the two oligonucleotides must bind in a specific orientation relative to the target, as is required in PCR, where oligonucleotides must be oppositely but appropriately oriented such that the DNA polymerase can bridge the gap between the two oligonucleotides in both directions, further enhances specificity of the detection reaction. However, it is well known to those in the art that even though PCR utilizes two oligonucleotide probes (termed primers) \u201cnon-specific\u201d amplification (i.e., amplification of sequences not directed by the two primers used) is a common artifact. This is in part because the DNA polymerase used in PCR can accommodate very large distances, measured in nucleotides, between the oligonucleotides and thus there is a large window in which non-specific binding of an oligonucleotide can lead to exponential amplification of inappropriate product. The LCR, in contrast, cannot proceed unless the oligonucleotides used are bound to the target adjacent to each other and so the full benefit of the dual oligonucleotide hybridization is realized.","An ideal direct detection method would combine the advantages of the direct detection assays (e.g., easy quantification and minimal risk of carry-over contamination) with the specificity provided by a dual oligonucleotide hybridization assay.","The present invention relates to compositions and methods for the detection and characterization of nucleic acid sequences and variations in nucleic acid sequences. The present invention relates to methods for forming a nucleic acid cleavage structure on a target sequence and cleaving the nucleic acid cleavage structure in a site-specific manner. For example, in some embodiments, a 5\u2032 nuclease activity of any of a variety of enzymes is used to cleave the target-dependent cleavage structure, thereby indicating the presence of specific nucleic acid sequences or specific variations thereof.","The present invention provides structure-specific cleavage agents (e.g., nucleases) from a variety of sources, including mesophilic, psychrophilic, thermophilic, and hyperthermophilic organisms. The preferred structure-specific nucleases are thermostable. Thermostable structure-specific nucleases arc contemplated as particularly useful in that they operate at temperatures where nucleic acid hybridization is extremely specific, allowing for allele-specific detection (including single-base mismatches). In one embodiment, the thermostable structure-specific nucleases are thermostable 5\u2032 nucleases comprising altered polymerases derived from the native polymerases of species, including, but not limited to , and . However, the invention is not limited to the use of thermostable 5\u2032 nucleases. Thermostable structure-specific nucleases from the FEN-1, RAD2 and XPG class of nucleases are also preferred.","The present invention provides a method for detecting a target sequence (e.g., a mutation, polymorphism, etc), comprising providing a sample suspected of containing the target sequence; oligonucleotides capable of forming an invasive cleavage structure in the presence of the target sequence; and an agent for detecting the presence of an invasive cleavage structure; and exposing the sample to the oligonucleotides and the agent. In some embodiments, the method further comprises the step of detecting a complex comprising the agent and the invasive cleavage structure (directly or indirectly). In some embodiments, the agent comprises a cleavage agent. In some preferred embodiments, the exposing of the sample to the oligonucleotides and the agent comprises exposing the sample to the oligonucleotides and the agent under conditions wherein an invasive cleavage structure is formed between the target sequence and the oligonucleotides if the target sequence is present in the sample, wherein the invasive cleavage structure is cleaved by the cleavage agent to form a cleavage product. In some embodiments, the method further comprises the step of detecting the cleavage product. In some embodiments, the target sequence comprises a first region and a second region, the second region downstream of and contiguous to the first region, and wherein the oligonucleotides comprise first and second oligonucleotides, wherein at least a portion of the first oligonucleotide is completely complementary to the first portion of the target sequence and wherein the second oligonucleotide comprises a 3\u2032 portion and a 5\u2032 portion, wherein the 5\u2032 portion is completely complementary to the second portion of said target nucleic acid.","The present invention also provides a kit for detecting such target sequences, said kit comprising oligonucleotides capable of forming an invasive cleavage structure in the presence of the target sequence. In some embodiments, the kit further comprises an agent for detecting the presence of an invasive cleavage structure (e.g., a cleavage agent). In some embodiments, the oligonucleotides comprise first and second oligonucleotides, said first oligonucleotide comprising a 5\u2032 portion complementary to a first region of the target nucleic acid and said second oligonucleotide comprising a 3\u2032 portion and a 5\u2032 portion, said 5\u2032 portion complementary to a second region of the target nucleic acid downstream of and contiguous to the first portion. In some preferred embodiments, the target sequence comprises human cytomegalovirus viral DNA; sequence containing polymorphisms in the human apolipoprotein E gene (ApoE); sequence containing mutations in the human hemochromatosis (HH) gene; sequence containing mutations in human MTHFR; sequence containing prothrombin 20210GA polymorphism; sequence containing HR-2 mutation in human factor V gene; sequence containing single nucleotide polymorphisms in human TNF-\u03b1 gene, and sequence containing the Leiden mutation in human factor V gene. In some preferred embodiments, kits comprise oligonucleotides for detecting two or more target sequences. For example, information on two or more mutations may provide medically relevant information such that kits allowing detection of the plurality of mutations would be desired (e.g., Factor V and HR-2 detection). In some preferred embodiments kits are probed containing a probe oligonucleotide comprising a sequence of SEQ ID NOs: 197, 198, 199, 200, 208, 209, 211, 212, 217, 218, 223, 224, 229, 232, 236, 237, 241, 242, or 244. In still other embodiments, kits provide oligonuclcotide sets, the sets including one or more of the oligonucleotides: SEQ ID NOs:195, 197, and 198 for ApoE detection; 196, 199, and 200 for ApoE detection; 202, 208, and 209 for HH detection; 203, 211, and 212 for HH detection; 216, 217, and 218 for MTHFR detection; 222, 223, and 224 for prothrombin polymorphism detection; 228, 229, 231, and 232 for HR-2 detection; 235, 236, and 237 for TNF-\u03b1 detection; 240, 241, and 242 for Factor V detection; and 243, 244, 246, and 247 for MRSA detection.","The present invention also provides methods for detecting the presence of a target nucleic acid molecule by detecting non-target cleavage products comprising providing: a cleavage agent; a source of target nucleic acid, the target nucleic acid comprising a first region and a second region, the second region downstream of and contiguous to the first region; a first oligonucleotide, wherein at least a portion of the first oligonucleotide is completely complementary to the first portion of the target nucleic acid; and a second oligonucleotide comprising a 3\u2032 portion and a 5\u2032 portion, wherein the 5\u2032 portion is completely complementary to the second portion of the target nucleic acid; mixing the cleavage agent, the target nucleic acid, the first oligonucleotide and the second oligonucleotide to create a reaction mixture under reaction conditions such that at least the portion of the first oligonucleotide is annealed to the first region of said target nucleic acid and wherein at least the 5\u2032 portion of the second oligonucleotide is annealed to the second region of the target nucleic acid so as to create a cleavage structure, and wherein cleavage of the cleavage structure occurs to generate non-target cleavage product; and detecting the cleavage of the cleavage structure.","The detection of the cleavage of the cleavage structure can be carried out in any manner. In some embodiments, the detection of the cleavage of the cleavage structure comprises detecting the non-target cleavage product. In yet other embodiments, the detection of the cleavage of the cleavage structure comprises detection of fluorescence, mass, or fluorescence energy transfer. Other detection methods include, but are not limited to detection of radioactivity, luminescence, phosphorescence, fluorescence polarization, and charge. In some embodiments, detection is carried out by a method comprising providing the non-target cleavage product; a composition comprising two single-stranded nucleic acids annealed so as to define a single-strandcd portion of a protein binding region; and a protein; and exposing the non-target cleavage product to the single-stranded portion of the protein binding region under conditions such that the protein binds to the protein binding region. In some embodiments, the protein comprises a nucleic acid producing protein, wherein the nucleic acid producing protein binds to the protein binding region and produces nucleic acid. In some embodiments, the protein binding region is a template-dependent RNA polymerase binding region (e.g., a T7 RNA polymerase binding region). In other embodiments, the detection is carried out by a method comprising providing the non-target cleavage product; a single continuous strand of nucleic acid comprising a sequence defining a single strand of an RNA polymerase binding region; a template-dependent DNA polymerase; and a template-dependent RNA polymerase; exposing the non-target cleavage product to the RNA polymerase binding region under conditions such that the non-target cleavage product binds to a portion of the single strand of the RNA polymerase binding region to produce a bound non-target cleavage product; exposing the bound non-target cleavage product to the template-dependent DNA polymerase under conditions such that a double-stranded RNA polymerase binding region is produced; and exposing the double-stranded RNA polymerase binding region to the template-dependent RNA polymerase under conditions such that RNA transcripts are produced. In some embodiments, the method further comprises the step of detecting the RNA transcripts. In some embodiments, the template-dependent RNA polymerase is T7 RNA polymerase.","The present invention is not limited by the nature of the 3\u2032 portion of the second oligonucleotide. In some preferred embodiments, the 3\u2032 portion of the second oligonucleotide comprises a 3\u2032 terminal nucleotide not complementary to the target nucleic acid. In some embodiments, the 3\u2032 portion of the second oligonucleotide consists of a single nucleotide not complementary to the target nucleic acid.","Any of the components of the method may be attached to a solid support. For example, in some embodiments, the first oligonucleotide is attached to a solid support. In other embodiments, the second oligonucleotide is attached to a solid support.","The cleavage agent can be any agent that is capable of cleaving invasive cleavage structures. In some preferred embodiments, the cleavage agent comprises a structure-specific nuclease. In particularly preferred embodiments, the structure-specific nuclease comprises a thermostable structure-specific nuclease (e.g., a thermostable 5\u2032 nuclease). Thermostable structure-specific nucleases include, but are not limited to, those having an amino acid sequence homologous to a portion of the amino acid sequence of a thermostable DNA polymerase derived from a thermophilic organism (e.g., , and ). In other embodiments, the thermostable structure-specific nucleases is from the FEN-1, RAD2 or XPG class of nucleases, a chimerical structures containing one or more portions of any of the above cleavage agents.","The method is not limited by the nature of the target nucleic acid. In some embodiments, the target nucleic acid is single stranded or double stranded DNA or RNA. In some embodiments, double stranded nucleic acid is rendered single stranded (e.g., by heat) prior to formation of the cleavage structure. In some embodiment, the source of target nucleic acid comprises a sample containing genomic DNA. Sample include, but are not limited to, blood, saliva, cerebral spinal fluid, pleural fluid, milk, lymph, sputum and semen.","In some embodiments, the reaction conditions for the method comprise providing a source of divalent cations. In some preferred embodiments, the divalent cation is selected from the group comprising Mn and Mg ions. In some embodiments, the reaction conditions for the method comprise providing the first and the second oligonucleotides in concentration excess compared to the target nucleic acid.","In some embodiments, the method further comprises providing a third oligonuclcotide complementary to a third portion of said target nucleic acid upstream of the first portion of the target nucleic acid, wherein the third oligonucleotide is mixed with the reaction mixture.","The present invention also provides a method for detecting the presence of a target nucleic acid molecule by detecting non-target cleavage products comprising providing: a cleavage agent; a source of target nucleic acid, the target nucleic acid comprising a first region and a second region, the second region downstream of and contiguous to the first region; a plurality of first oligonucleotides, wherein at least a portion of the first oligonucleotides is completely complementary to the first portion of the target nucleic acid; a second oligonucleotide comprising a 3\u2032 portion and a 5\u2032 portion, wherein said 5\u2032 portion is completely complementary to the second portion of the target nucleic acid; mixing the cleavage agent, the target nucleic acid, the plurality of first oligonucleotides and second oligonucleotide to create a reaction mixture under reaction conditions such that at least the portion of a first oligonucleotide is annealed to the first region of the target nucleic acid and wherein at least the 5\u2032 portion of the second oligonucleotide is annealed to the second region of the target nucleic acid so as to create a cleavage structure, and wherein cleavage of the cleavage structure occurs to generate non-target cleavage product, wherein the conditions permit multiple cleavage structures to form and be cleaved from the target nucleic acid; and detecting the cleavage of said cleavage structures. In some embodiments, the conditions comprise isothermal conditions that permit the plurality of first oligonucleotides to dissociate from the target nucleic acid. While the present invention is limited by the number of cleavage structure formed on a particular target nucleic acid, in some preferred embodiments, two or more (3, 4, 5, . . . , 10, . . . , 10000, . . . ) of the plurality of first oligonucleotides form cleavage structures with a particular target nucleic acid, wherein the cleavage structures are cleaved to produce the non-target cleavage products.","The present invention also provide methods where a cleavage product from the above methods is used in a further invasive cleavage reaction. For example, the present invention provides a method comprising providing a cleavage agent; a first target nucleic acid, the first target nucleic acid comprising a first region and a second region, the second region downstream of and contiguous to the first region; a first oligonucleotide, wherein at least a portion of the first oligonucleotide is completely complementary to the first portion of the first target nucleic acid; a second oligonucleotide comprising a 3\u2032 portion and a 5\u2032 portion, wherein the 5\u2032 portion is completely complementary to the second portion of the first target nucleic acid; a second target nucleic acid, said second target nucleic acid comprising a first region and a second region, the second region downstream of and contiguous to the first region; and a third oligonucleotide, wherein at least a portion of the third oligonucleotide is completely complementary to the first portion of the second target nucleic acid; generating a first cleavage structure wherein at least said portion of the first oligonucleotide is annealed to the first region of the first target nucleic acid and wherein at least the 5\u2032 portion of the second oligonucleotide is annealed to the second region of the first target nucleic acid and wherein cleavage of the first cleavage structure occurs via the cleavage agent thereby cleaving the first oligonucleotide to generate a fourth oligonucleotide, said fourth oligonucleotide comprising a 3\u2032 portion and a 5\u2032 portion, wherein the 5\u2032 portion is completely complementary to the second portion of the second target nucleic acid; generating a second cleavage structure under conditions wherein at least said portion of the third oligonucleotide is annealed to the first region of the second target nucleic acid and wherein at least the 5\u2032 portion of the fourth oligonucleotide is annealed to the second region of the second target nucleic acid and wherein cleavage of the second cleavage structure occurs to generate a cleavage fragment; and detecting the cleavage of the second cleavage structure. In some preferred embodiments, the 3\u2032 portion of the fourth oligonucleotide comprises a 3\u2032 terminal nucleotide not complementary to the second target nucleic acid. In some embodiments, the 3\u2032 portion of the third oligonucleotide is covalently linked to the second target nucleic acid. In some embodiments, the second target nucleic acid further comprises a 5\u2032 region, wherein the 5\u2032 region of the second target nucleic acid is the third oligonucleotide. The present invention further provides kits comprising: a cleavage agent; a first oligonucleotide comprising a 5\u2032 portion complementary to a first region of a target nucleic acid; and a second oligonucleotide comprising a 3\u2032 portion and a 5\u2032 portion, said 5\u2032 portion complementary to a second region of the target nucleic acid downstream of and contiguous to the first portion. In some embodiments, the 3\u2032 portion of the second oligonucleotide comprises a 3\u2032 terminal nucleotide not complementary to the target nucleic acid. In preferred embodiments, the 3\u2032 portion of the second oligonucleotide consists of a single nucleotide not complementary to the target nucleic acid. In some embodiments, the kit further comprises a solid support. For example, in some embodiments, the first and\/or second oligonucleotide is attached to said solid support. In some embodiments, the kit further comprises a buffer solution. In some preferred embodiments, the buffer solution comprises a source of divalent cations (e.g., Mn and\/or Mg ions). In some specific embodiments, the kit further comprises a third oligonucleotide complementary to a third portion of the target nucleic acid upstream of the first portion of the first target nucleic acid. In yet other embodiments, the kit further comprises a target nucleic acid. In some embodiments, the kit further comprises a second target nucleic acid. In yet other embodiments, the kit further comprises a third oligonucleotide comprising a 5\u2032 portion complementary to a first region of the second target nucleic acid. In some specific embodiments, the 3\u2032 portion of the third oligonucleotide is covalently linked to the second target nucleic acid. In other specific embodiments, the second target nucleic acid further comprises a 5\u2032 portion, wherein the 5\u2032 portion of the second target nucleic acid is the third oligonucleotide. In still other embodiments, the kit further comprises an ARRESTOR molecule (e.g., ARRESTOR oligonucleotide).","The present invention further provides a composition comprising a cleavage structure, the cleavage structure comprising: a) a target nucleic acid, the target nucleic acid having a first region, a second region, a third region and a fourth region, wherein the first region is located adjacent to and downstream from the second region, the second region is located adjacent to and downstream from the third region and the third region is located adjacent to and downstream from the fourth region; b) a first oligonucleotide complementary to the fourth region of the target nucleic acid; c) a second oligonucleotide having a 5\u2032 portion and a 3\u2032 portion wherein the 5\u2032 portion of the second oligonucleotide contains a sequence complementary to the second region of the target nucleic acid and wherein the 3\u2032 portion of the second oligonucleotide contains a sequence complementary to the third region of the target nucleic acid; and d) a third oligonucleotide having a 5\u2032 portion and a 3\u2032 portion wherein the 5\u2032 portion of the third oligonucleotide contains a sequence complementary to the first region of the target nucleic acid and wherein the 3\u2032 portion of the third oligonucleotide contains a sequence complementary to the second region of the target nucleic acid.","The present invention is not limited by the length of the four regions of the target nucleic acid. In one embodiment, the first region of the target nucleic acid has a length of 11 to 50 nucleotides. In another embodiment, the second region of the target nucleic acid has a length of one to three nucleotides. In another embodiment, the third region of the target nucleic acid has a length of six to nine nucleotides. In yet another embodiment, the fourth region of the target nucleic acid has a length of 6 to 50 nucleotides.","The invention is not limited by the nature or composition of the of the first, second, third and fourth oligonucleotides; these oligonucleotides may comprise DNA, RNA, PNA and combinations thereof as well as comprise modified nucleotides, universal bases, adducts, etc. Further, one or more of the first, second, third and the fourth oligonucleotides may contain a dideoxynucleotide at the 3\u2032 terminus.","In one preferred embodiment, the target nucleic acid is not completely complementary to at least one of the first, the second, the third and the fourth oligonucleotides. In a particularly preferred embodiment, the target nucleic acid is not completely complementary to the second oligonucleotide.","As noted above, the present invention contemplates the use of structure-specific nucleases in detection methods. In one embodiment, the present invention provides a method of detecting the presence of a target nucleic acid molecule by detecting non-target cleavage products comprising: a) providing: i) a cleavage means, ii) a source of target nucleic acid, the target nucleic acid having a first region, a second region, a third region and a fourth region, wherein the first region is located adjacent to and downstream from the second region, the second region is located adjacent to and downstream from the third region and the third region is located adjacent to and downstream from the fourth region; iii) a first oligonucleotide complementary to the fourth region of the target nucleic acid; iv) a second oligonucleotide having a 5\u2032 portion and a 3\u2032 portion wherein the 5\u2032 portion of the second oligonucleotide contains a sequence complementary to the second region of the target nucleic acid and wherein the 3\u2032 portion of the second oligonucleotide contains a sequence complementary to the third region of the target nucleic acid; iv) a third oligonucleotide having a 5\u2032 and a 3\u2032 portion wherein the 5\u2032 portion of the third oligonucleotide contains a sequence complementary to the first region of the target nucleic acid and wherein the 3\u2032 portion of the third oligonucleotide contains a sequence complementary to the second region of the target nucleic acid; b) mixing the cleavage means, the target nucleic acid, the first oligonucleotide, the second oligonucleotide and the third oligonucleotide to create a reaction mixture under reaction conditions such that the first oligonucleotide is annealed to the fourth region of the target nucleic acid and wherein at least the 3\u2032 portion of the second oligonucleotide is annealed to the target nucleic acid and wherein at least the 5\u2032 portion of the third oligonucleotide is annealed to the target nucleic acid so as to create a cleavage structure and wherein cleavage of the cleavage structure occurs to generate non-target cleavage products, each non-target cleavage product having a 3\u2032-hydroxyl group; and c) detecting the non-target cleavage products.","The invention is not limited by the nature of the target nucleic acid. In one embodiment, the target nucleic acid comprises single-stranded DNA. In another embodiment, the target nucleic acid comprises double-stranded DNA and prior to step c), the reaction mixture is treated such that the double-stranded DNA is rendered substantially single-stranded. In another embodiment, the target nucleic acid comprises RNA and the first and second oligonucleotides comprise DNA.","The invention is not limited by the nature of the cleavage means. In one embodiment, the cleavage means is a structure-specific nuclease; particularly preferred structure-specific nucleases are thermostable structure-specific nucleases. In one preferred embodiment, the thermostable structure-specific nuclease is encoded by a DNA sequence selected from the group consisting of SEQ ID NOS:1-3, 9, 10, 12, 21, 30, 31, 101, 106, 110, 114, 129, 131, 132, 137, 140, 141, 142, 143, 144, 145, 147, 150, 151, 153, 155, 156, 157, 158, 161, 163, 178, 180, and 182.","In another preferred embodiment, the thermostable structure-specific nuclease is a nuclease from the FEN-1\/RAD2\/XPG class of nucleases. In another preferred embodiment the thermostable structure specific nuclease is a chimerical nuclease.","In an alternative preferred embodiment, the detection of the non-target cleavage products comprises electrophoretic separation of the products of the reaction followed by visualization of the separated non-target cleavage products.","In another preferred embodiment, one or more of the first, second, and third oligonucleotides contain a dideoxynucleotide at the 3\u2032 terminus. When dideoxynucleotide-containing oligonucleotides are employed, the detection of the non-target cleavage products preferably comprises: a) incubating the non-target cleavage products with a template-independent polymerase and at least one labeled nucleoside triphosphate under conditions such that at least one labeled nucleotide is added to the 3\u2032-hydroxyl group of the non-target cleavage products to generate labeled non-target cleavage products; and b) detecting the presence of the labeled non-target cleavage products. The invention is not limited by the nature of the template-independent polymerase employed; in one embodiment, the template-independent polymerase is selected from the group consisting of terminal deoxynucleotidyl transferase (TdT) and poly A polymerase. When TdT or polyA polymerase are employed in the detection step, the second oligonucleotide may contain a 5\u2032 end label, the 5\u2032 end label being a different label than the label present upon the labeled nucleoside triphosphate. The invention is not limited by the nature of the 5\u2032 end label; a wide variety of suitable 5\u2032 end labels are known to the art and include biotin, fluorescein, tetrachlorofluorescein, hexachlorofluorescein, Cy3 amidite, Cy5 amidite and digoxigenin.","In another embodiment, detecting the non-target cleavage products comprises: a) incubating the non-target cleavage products with a template-independent polymerase and at least one nucleoside triphosphate under conditions such that at least one nucleotide is added to the 3\u2032-hydroxyl group of the non-target cleavage products to generate tailed non-target cleavage products; and b) detecting the presence of the tailed non-target cleavage products. The invention is not limited by the nature of the template-independent polymerase employed; in one embodiment, the template-independent polymerase is selected from the group consisting of terminal deoxynucleotidyl transferase (TdT) and poly A polymerase. When TdT or polyA polymerase are employed in the detection step, the second oligonucleotide may contain a 5\u2032 end label. The invention is not limited by the nature of the 5\u2032 end label; a wide variety of suitable 5\u2032 end labels are known to the art and include biotin, fluorescein, tetrachlorofluorescein, hexachlorofluorescein, Cy3 amidite, Cy5 amidite and digoxigenin.","In a preferred embodiment, the reaction conditions comprise providing a source of divalent cations; particularly preferred divalent cations are Mn and Mg ions.","The present invention further provides a method of detecting the presence of a target nucleic acid molecule by detecting non-target cleavage products comprising: a) providing: i) a cleavage means, ii) a source of target nucleic acid, the target nucleic acid having a first region, a second region and a third region, wherein the first region is located adjacent to and downstream from the second region and wherein the second region is located adjacent to and downstream from the third region, iii) a first oligonucleotide having a 5\u2032 and a 3\u2032 portion wherein the 5\u2032 portion of the first oligonucleotide contains a sequence complementary to the second region of the target nucleic acid and wherein the 3\u2032 portion of the first oligonucleotide contains a sequence complementary to the third region of the target nucleic acid; iv) a second oligonucleotide having a length between eleven to fifteen nucleotides.and further having a 5\u2032 and a 3\u2032 portion wherein the 5\u2032 portion of the second oligonucleotide contains a sequence complementary to the first region of the target nucleic acid and wherein the 3\u2032 portion of the second oligonucleotide contains a sequence complementary to the second region of the target nucleic acid; b) mixing the cleavage means, the target nucleic acid, the first oligonucleotide and the second oligonucleotide to create a reaction mixture under reaction conditions such that at least the 3\u2032 portion of the first oligonuclcotide is annealed to the target nucleic acid and wherein at least the 5\u2032 portion of the second oligonucleotide is annealed to the target nucleic acid so as to create a cleavage structure and wherein cleavage of the cleavage structure occurs to generate non-target cleavage products, each non-target cleavage product having a 3\u2032-hydroxyl group; and c) detecting the non-target cleavage products. In a preferred embodiment the cleavage means is a structure-specific nuclease, preferably a thermostable structure-specific nuclease.","The invention is not limited by the length of the various regions of the target nucleic acid. In a preferred embodiment, the second region of the target nucleic acid has a length between one to five nucleotides. In another preferred embodiment, one or more of the first and the second oligonucleotides contain a dideoxynucleotide at the 3\u2032 terminus. When dideoxynucleotide-containing oligonucleotides are employed, the detection of the non-target cleavage products preferably comprises: a) incubating the non-target cleavage products with a template-independent polymerase and at least one labeled nucleoside triphosphate under conditions such that at least one labeled nucleotide is added to the 3\u2032-hydroxyl group of the non-target cleavage products to generate labeled non-target cleavage products; and b) detecting the presence of the labeled non-target cleavage products. The invention is not limited by the nature of the template-independent polymerase employed; in one embodiment, the template-independent polymerase is selected from the group consisting of terminal deoxynucleotidyl transferase (TdT) and poly A polymerase. When TdT or polyA polymerase is employed in the detection step, the second oligonucleotide may contain a 5\u2032 end label, the 5\u2032 end label being a different label than the label present upon the labeled nucleoside triphosphate. The invention is not limited by the nature of the 5\u2032 end label; a wide variety of suitable 5\u2032 end labels are known to the art and include biotin, fluorescein, tetrachlorofluorescein, hexachlorofluorescein, Cy3 amidite, Cy5 amidite and digoxigenin.","In another embodiment, detecting the non-target cleavage products comprises: a) incubating the non-target cleavage products with a template-independent polymerase and at least one nucleoside triphosphate under conditions such that at least one nucleotide is added to the 3\u2032-hydroxyl group of the non-target cleavage products to generate tailed non-target cleavage products; and b) detecting the presence of the tailed non-target cleavage products. The invention is not limited by the nature of the template-independent polymerase employed; in one embodiment, the template-independent polymerase is selected from the group consisting of terminal deoxynucleotidyl transferase (TdT) and poly A polymerase. When TdT or polyA polymerases are employed in the detection step, the second oligonucleotide may contain a 5\u2032 end label. The invention is not limited by the nature of the 5\u2032 end label; a wide variety of suitable 5\u2032 end labels are known to the art and include biotin, fluorescein, tetrachlorofluorescein, hexachlorofluorescein, Cy3 amidite, Cy5 amidite and digoxigenin.","The novel detection methods of the invention may be employed for the detection of target DNAs and RNAs including, but not limited to, target DNAs and RNAs comprising wild type and mutant alleles of genes, including genes from humans or other animals that are or may be associated with disease or cancer. In addition, the methods of the invention may be used for the detection of and\/or identification of strains of microorganisms, including bacteria, fungi, protozoa, ciliates and viruses (and in particular for the detection and identification of RNA viruses, such as HCV).","The present invention further provides improved enzymatic cleavage means. In one embodiment, the present invention provides a thermostable structure-specific nuclease having an amino acid sequence selected from the group consisting of SEQ ID NOS:102, 107, 130, 132, 179, 181, 183, 184, 185, 186, 187, and 188. In another embodiment, the nuclease is encoded by a DNA sequence selected from the group consisting of SEQ ID NOS:101, 106, 129 131, 178, 180, and 82.","The present invention also provides a recombinant DNA vector comprising DNA having a nucleotide sequence encoding a structure-specific nuclease, the nucleotide sequence selected from the group consisting of SEQ ID NOS:101, 106, 129 131, 137, 140, 141, 142, 143, 144, 145, 147, 150, 151, 153, 155, 156, 157, 158, 161, 163, 178, 180, and 182. In a preferred embodiment, the invention provides a host cell transformed with a recombinant DNA vector comprising DNA having a nucleotide sequence encoding a structure-specific nuclease, the nucleotide sequence selected from the group consisting of SEQ ID NOS:101, 106, 129, 131, 178, 180, and 182. The invention is not limited by the nature of the host cell employed. The art is well aware of expression vectors suitable for the expression of nucleotide sequences encoding structure-specific nucleases which can be expressed in a variety of prokaryotic and eukaryotic host cells. In a preferred embodiment, the host cell is an cell.","The present invention provides purified FEN-1 endonucleases. In one embodiment, the present invention provides FEN-1 endonuclease. In a preferred embodiment, the purified FEN-1 endonuclease has a molecular weight of about 38.7 kilodaltons (the molecular weight may be conveniently estimated using SDS-PAGE as described in Ex. 28).","The present invention further provides an isolated oligonucleotide encoding a FEN-1 endonuclease, the oligonucleotide having a region capable of hybridizing to an oligonucleotide sequence selected from the group consisting of SEQ ID NOS: 116-119. In a preferred embodiment, the oligonucleotide encoding the purified Pyrococcus woesei FEN-1 endonuclease is operably linked to a heterologous promoter. The present invention is not limited by the nature of the heterologous promoter employed; in a preferred embodiment, the heterologous promoter is an inducible promoter (the promoter chosen will depend upon the host cell chosen for expression as is known in the art). The invention is not limited by the nature of the inducible promoter employed. Preferred inducible promoters include the -Ppromoter, the tac promoter, the trp promoter and the trc promoter.","In another preferred embodiment, the invention provides a recombinant DNA vector comprising an isolated oligonucleotide encoding a (P) FEN-1 endonuclease, the oligonucleotide having a region capable of hybridizing to an oligonucleotide sequence selected from the group consisting of SEQ ID NOS:116-119. Host cells transformed with these recombinant vectors are also provided. In a preferred embodiment, the invention provides a host cell transformed with a recombinant DNA vector comprising DNA having a region capable of hybridizing to an oligonucleotide sequence selected from the group consisting of SEQ ID NOS: 116-119; these vectors may further comprise a heterologous promoter operably linked to the Pwo FEN-1-encoding polynucleotides. The invention is not limited by the nature of the host cell employed. The art is well aware of expression vectors suitable for the expression of Pwo FEN-1-encoding polynucleotides which can be expressed in a variety of prokaryotic and eukaryotic host cells. In a preferred embodiment, the host cell is an cell.","In yet another embodiment, the invention provides an isolated oligonucleotide comprising a gene encoding a FEN-1 endonuclease having a molecular weight of about 38.7 kilodaltons. In another embodiment, the encoding a FEN-1 endonuclease is operably linked to a heterologous promoter. The present invention is not limited by the nature of the heterologous promoter employed; in a preferred embodiment, the heterologous promoter is an inducible promoter (the promoter chosen will depend upon the host cell chosen for expression as is known in the art). The invention is not limited by the nature of the inducible promoter employed. Preferred inducible promoter include the -Ppromoter, the tac promoter, the trp promoter and the trc promoter.","The invention further provides recombinant DNA vectors comprising DNA having a nucleotide sequence encoding FEN-1 endonucleases. In one preferred embodiment, the present invention provides a FEN-1 endonuclease having a molecular weight of about 38.7 kilodaltons. Still further, a host cell transformed with a recombinant DNA vector comprising DNA having a nucleotide sequence encoding FEN-1 endonuclease. In a preferred embodiment, the host cell is transformed with a recombinant DNA vector comprising DNA having a nucleotide sequence encoding a FEN-1 endonuclease having a molecular weight of about 38.7 kilodaltons is provided. The invention is not limited by the nature of the host cell employed. The art is well aware of expression vectors suitable for the expression of Pwo FEN-1-encoding polynucleotides which can be expressed in a variety of prokaryotic and eukaryotic host cells. In a preferred embodiment, the host cell is an cell.","Thus, the present invention provides multiple purified FEN-1 endonucleases, both purified native forms of the endonucleases, as well as recombinant endonucleases. In preferred embodiments, the purified FEN-1 endonucleases are obtained from archaebacterial or eubacterial organisms. In particularly preferred embodiments, the FEN-1 endonucleases are obtained from organisms selected from the group consisting of and . In a preferred embodiment, the purified FEN-1 endonucleases have molecular weights of about 39 kilodaltons (the molecular weight may be conveniently estimated using SDS-PAGE as described in Ex. 28).","The present invention further provides isolated oligonucleotides encoding and FEN-1 endonucleases, the oligonucleotides each having a region capable of hybridizing to at least a portion of an oligonuclcotide sequence, wherein the oligonuclcotide sequence is selected from the group consisting of SEQ ID NOS:170, 171, 172, and 173. In some preferred embodiment, the oligonucleotides encoding the and FEN-1 endonucleases are operably linked to heterologous promoters. However, it is not intended that the present invention be limited by the nature of the heterologous promoter employed. It is contemplated that the promoter chosen will depend upon the host cell chosen for expression as is known in the art. In some preferred embodiments, the heterologous promoter is an inducible promoter. The invention is not limited by the nature of the inducible promoter employed. Preferred inducible promoters include the -Ppromoter, the tac promoter, the trp promoter and the trc promoter.","In another preferred embodiment, the invention provides recombinant DNA vectors comprising isolated oligonucleotides encoding or FEN-1 endonucleases, each oligonucleotides having a region capable of hybridizing to at least a portion of an oligonucleotide sequence, wherein the oligonucleotide sequence is selected from the group consisting of SEQ ID NOS:170, 171, 172, and 173. The present invention further provides host cells transformed with these recombinant vectors. In a preferred embodiment, the invention provides a host cell transformed with a recombinant DNA vector comprising DNA having a region capable of hybridizing to at least a portion of an oligonucleotide sequence, wherein the oligonucleotide sequence is selected from the group consisting of SEQ ID NOS:170, 171, 172 and 173. In some embodiments, these vectors may further comprise a heterologous promoter operably linked to the FEN-1-encoding polynucleotides. The invention is not limited by the nature of the host cell employed. The art is well aware of expression vectors suitable for the expression of FEN-1 -encoding polynucleotides which can be expressed in a variety of prokaryotic and eukaryotic host cells. In a preferred embodiment, the host cell is an cell.","The present invention further provides chimeric structure-specific nucleases. In one embodiment, the present invention provides chimeric endonucleases comprising amino acid portions derived from the endonucleases selected from the group of FEN-1, XPG and RAD homologs. In a preferred embodiment, the chimeric endonucleases comprise amino acid portions derived from the FEN-1 endonucleases selected from the group of and . In a more preferred embodiment, the chimeric FEN-1 endonucleases have molecular weights of about 39 kilodaltons (the molecular weight may be conveniently estimated using SDS-PAGE as described in Ex. 28).","The present invention further provides isolated oligonucleotides encoding chimeric endonucleases. In one embodiment, the oligonucleotides encoding the chimeric endonucleases comprise nucleic acid sequences derived from the genes selected from the group of FEN-1, XPG and RAD homologs. In a preferred embodiment the oligonucleotides encoding the chimeric endonucleases comprise nucleic acid sequences derived from the genes encoding the FEN-1 endonucleases selected from the group of and . In a particularly preferred embodiment, the genes for the chimeric endonucleases are operably linked to heterologous promoters. The present invention is not limited by the nature of the heterologous promoter employed. It is contemplated that the promoter chosen will depend upon the host cell selected for expression, as is known in the art. In preferred embodiments, the heterologous promoter is an inducible promoter. The invention is not limited by the nature of the inducible promoter employed. Preferred inducible promoter include the -Ppromoter, the tac promoter, the trp promoter and the trc promoter.","In another preferred embodiment, the invention provides recombinant DNA vectors comprising isolated oligonucleotides encoding the chimeric endonucleases described above. In one embodiment, the recombinant DNA vectors comprise isolated oligonucleotides encoding nucleic acid sequences derived from the genes selected from the group of FEN-1, XPG and RAD homologs. In a preferred embodiment, the recombinant DNA vectors comprise isolated oligonucleotides encoding the chimeric endonucleases comprising nucleic acid sequences derived from the genes encoding the FEN-1 endonucleases selected from the group of and . These vectors may further comprise a heterologous promoter operably linked to the chimeric nuclease-encoding polynucleotides.","Host cells transformed with these recombinant vectors are also provided. The invention is not limited by the nature of the host cell employed. The art is well aware of expression vectors suitable for the expression of FEN-1-encoding polynucleotides which can be expressed in a variety of prokaryotic and eukaryotic host cells. In a preferred embodiment, the host cell is an cell.","The present invention further provides mixtures comprising a first structure-specific nuclease, wherein the first nuclease consists of a purified FEN-1 endonuclease and a second structure-specific nuclease. In preferred embodiments, the second structure-specific nuclease of the mixture is selected from the group consisting FEN-1 endonuclease, FEN-1FEN-1 endonuclease, FEN-1 endonuclease, FEN-1, and chimerical FEN-1 endonucleases. In alternative embodiments, the purified FEN-1 endonuclease ofthe mixture is selected from the group consisting FEN-1 endonuclease, FEN-1 endonuclease, FEN-1 endonuclease, FEN-1 endonuclease, FEN-1, and chimerical FEN-1 endonucleases. In yet other preferred embodiments of the mixture, the second nuclease is a 5\u2032 nuclease derived from a thermostable DNA polymerase altered in amino acid sequence such that it exhibits reduced DNA synthetic activity from that of the wild-type DNA polymerase but retains substantially the same 5\u2032 nuclease activity of the wild-type DNA polymerase. In some preferred embodiments of the mixture, the second nuclease is selected from the group consisting of the Cleavase\u00ae BN enzyme, DNA polymerase, DNA polymerase, Exo III, Rad1\/Rad10 complex.","The present invention also provides methods for treating nucleic acid, comprising: a) providing a purified FEN-1 endonuclease; and a nucleic acid substrate; b) treating the nucleic acid substrate under conditions such that the substrate forms one or more cleavage structures; and c) reacting the endonuclease with the cleavage structures so that one or more cleavage products are produced. In some embodiments, the purified FEN-1 endonuclease is selected from the group consisting FEN-1 endonuclease, FEN-1 endonuclease, FEN-1 endonuclease, FEN-1 endonuclease, FEN-1, and chimerical FEN-1 endonucleases. In other embodiments, the method further comprises providing a structure-specific nuclease derived from a thermostable DNA polymerase altered in amino acid sequence such that it exhibits reduced DNA synthetic activity from that of the wild-type DNA polymerase but retains substantially the same 5\u2032 nuclease activity of the wild-type DNA polymerase.","In alternative embodiments of the methods, a portion of the amino acid sequence of the second nuclease is homologous to a portion of the amino acid sequence of a thermostable DNA polymerase derived from a eubacterial thermophile of the genus . In yet other embodiments, the thermophile is selected from the group consisting of and . In some alternative embodiments, the structure-specific nuclease is selected from the group consisting of the Cleavase\u00ae BN enzyme, DNA polymerase, DNA polymerase, Exo III, ","Rad1\/Rad10 complex. In some preferred embodiments, the structure-specific nuclease is the Cleavase\u00ae BN nuclease. In yet other embodiments, the nucleic acid of step (a) is substantially single-stranded. In further embodiments, the nucleic acid is selected from the group consisting of RNA and DNA. In yet further embodiments, the nucleic acid of step (a) is double stranded.","In other embodiments of the methods, the treating of step (b) comprises:\n\n","The present invention also provides methods for treating nucleic acid, comprising:\n\n","The present invention also provides nucleic acid treatment kits, comprising: a) a composition comprising at least one purified FEN-1 endonuclease; and b) a solution containing manganese. In some embodiments of the kits, the purified FEN-1 endonuclease is selected from the group consisting Pyrococcus woesei FEN-1 endonuclease, FEN-1 endonuclease, FEN-1 endonuclease, FEN-1 endonuclease, FEN-1, and chimerical FEN-1 endonucleases. In other embodiments, the kits further comprise at least one second structure-specific nuclease. In some preferred embodiments, the second nuclease is a 5\u2032 nuclease derived from a thermostable DNA polymerase altered in amino acid sequence such that it exhibits reduced DNA synthetic activity from that of the wild-type DNA polymerase but retains substantially the same 5\u2032 nuclease activity of the wild-type DNA polymerase. In yet other embodiments of the kits, the portion of the amino acid sequence of the second nuclease is homologous to a portion of the amino acid sequence of a thermostable DNA polymerase derived from a eubacterial thermophile of the genus . In further embodiments, the thermophile is selected from the group consisting of and . In yet other preferred embodiments, the kits further comprise reagents for detecting the cleavage products.","The present invention also provides methods for improving the methods and enzymes disclosed herein. For example, the present invention provides methods of improving enzymes for any intended purpose (e.g., use in cleavage reactions, amplification reactions, binding reactions, or any other use) comprising the step of providing an enzyme disclosed herein and modifying the enzyme (e.g., altering the amino acid sequence, adding or subtracting sequence, adding post-translational modifications, adding any other component whether biological or not, or any other modification). Likewise, the present invention provides methods for improving the methods disclosed herein comprising, conducting the method steps with one or more changes (e.g., change in a composition provided in the method, change in the order of the steps, or addition or subtraction of steps).","To facilitate an understanding of the present invention, a number of terms and phrases are defined below:","As used herein, the terms \u201ccomplementary\u201d or \u201ccomplementarity\u201d are used in reference to polynucleotides (i.e., a sequence of nucleotides such as an oligonucleotide or a target nucleic acid) related by the base-pairing rules. For example, for the sequence\u201c5\u2032-A-G-T-3\u2032,\u201d is complementary to the sequence \u201c3\u2032-T-C-A-5\u2032.\u201d Complementarity may be \u201cpartial,\u201d in which only some of the nucleic acids\u2032 bases are matched according to the base pairing rules. Or, there may be \u201ccomplete\u201d or \u201ctotal\u201d complementarity between the nucleic acids. The degree of complementarity between nucleic acid strands has significant effects on the efficiency and strength of hybridization between nucleic acid strands. This is of particular importance in amplification reactions, as well as detection methods which depend upon binding between nucleic acids. Either term may also be used in reference to individual nucleotides, especially within the context of polynucleotides. For example, a particular nucleotide within an oligonucleotide may be noted for its complementarity, or lack thereof, to a nucleotide within another nucleic acid strand, in contrast or comparison to the complementarity between the rest of the oligonucleotide and the nucleic acid strand.","The term \u201chomology\u201d and \u201chomologous\u201d refers to a degree of identity. There may be partial homology or complete homology. A partially homologous sequence is one that is less than 100% identical to another sequence.","As used herein, the term \u201chybridization\u201d is used in reference to the pairing of complementary nucleic acids. Hybridization and the strength of hybridization (i.e., the strength of the association between the nucleic acids) is influenced by such factors as the degree of complementary between the nucleic acids, stringency of the conditions involved, and the Tof the formed hybrid. \u201cHybridization\u201d methods involve the annealing of one nucleic acid to another, complementary nucleic acid, i.e., a nucleic acid having a complementary nucleotide sequence. The ability of two polymers of nucleic acid containing complementary sequences to find each other and anneal through base pairing interaction is a well-recognized phenomenon. The initial observations of the \u201chybridization\u201d process by Marmur and Lane, Proc. Natl. Acad. Sci. USA 46:453 (1960) and Doty et al., Proc. Natl. Acad. Sci. USA 46:461 (1960) have been followed by the refinement of this process into an essential tool of modem biology.","With regard to complementarity, it is important for some diagnostic applications to determine whether the hybridization represents complete or partial complementarity. For example, where it is desired to detect simply the presence or absence of pathogen DNA (such as from a virus, bacterium, fungi, mycoplasma, protozoan) it is only important that the hybridization method ensures hybridization when the relevant sequence is present; conditions can be selected where both partially complementary probes and completely complementary probes will hybridize. Other diagnostic applications, however, may require that the hybridization method distinguish between partial and complete complementarity. It may be of interest to detect genetic polymorphisms. For example, human hemoglobin is composed, in part, of four polypeptide chains. Two of these chains are identical chains of 141 amino acids (alpha chains) and two of these chains are identical chains of 146 amino acids (beta chains). The gene encoding the beta chain is known to exhibit polymorphism. The normal allele encodes a beta chain having glutamic acid at the sixth position. The mutant allele encodes a beta chain having valine at the sixth position. This difference in amino acids has a profound (most profound when the individual is homozygous for the mutant allele) physiological impact known clinically as sickle cell anemia. It is well known that the genetic basis of the amino acid change involves a single base difference between the normal allele DNA sequence and the mutant allele DNA sequence.","The complement of a nucleic acid sequence as used herein refers to an oligonucleotide which, when aligned with the nucleic acid sequence such that the 5\u2032 end of one sequence is paired with the 3\u2032 end of the other, is in \u201cantiparallel association.\u201d Certain bases not commonly found in natural nucleic acids may be included in the nucleic acids of the present invention and include, for example, inosine and 7-deazaguanine. Complementarity need not be perfect; stable duplexes may contain mismatched base pairs or unmatched bases. Those skilled in the art of nucleic acid technology can determine duplex stability empirically considering a number of variables including, for example, the length of the oligonucleotide, base composition and sequence of the oligonucleotide, ionic strength and incidence of mismatched base pairs.","As used herein, the term \u201cT\u201d is used in reference to the \u201cmelting temperature.\u201d The melting temperature is the temperature at which a population of double-stranded nucleic acid molecules becomes half dissociated into single strands. Several equations for calculating the Tof nucleic acids are well known in the art. As indicated by standard references, a simple estimate of the Tvalue may be calculated by the equation: T=81.5+0.41(% G+C), when a nucleic acid is in aqueous solution at 1 M NaCl (see e.g., Anderson and Young, Quantitative Filter Hybridization, in Nucleic Acid Hybridization (1985). Other references (e.g., Allawi, H. T. & SantaLucia, J., Jr. Thermodynamics and NMR of internal G.T mismatches in DNA. Biochemistry 36, 10581-94 (1997) include more sophisticated computations which take structural and environmental, as well as sequence characteristics into account for the calculation of T.","As used herein the term \u201cstringency\u201d is used in reference to the conditions of temperature, ionic strength, and the presence of other compounds, under which nucleic acid hybridizations are conducted. With \u201chigh stringency\u201d conditions, nucleic acid base pairing will occur only between nucleic acid fragments that have a high frequency of complementary base sequences. Thus, conditions of \u201cweak\u201d or \u201clow\u201d stringency are often required when it is desired that nucleic acids which are not completely complementary to one another be hybridized or annealed together.","\u201cHigh stringency conditions\u201d when used in reference to nucleic acid hybridization comprise conditions equivalent to binding or hybridization at 42 C in a solution consisting of 5\u00d7 SSPE (43.8 g\/l NaCl, 6.9 g\/l NaHPOHO and 1.85 g\/l EDTA, pH adjusted to 7.4 with NaOH), 0.5% SDS, 5\u00d7 Denhardt's reagent and 100 \u03bcg\/ml denatured salmon sperm DNA followed by washing in a solution comprising 0.1\u00d7 SSPE, 1.0% SDS at 42 C when a probe of about 500 nucleotides in length is employed.","\u201cMedium stringency conditions\u201d when used in reference to nucleic acid hybridization comprise conditions equivalent to binding or hybridization at 42 C in a solution consisting of 5\u00d7 SSPE (43.8 g\/l NaCI, 6.9 g\/l NaHPOHO and 1.85 g\/l EDTA, pH adjusted to 7.4 with NaOH), 0.5% SDS, 5\u00d7 Denhardt's reagent and 100 \u03bcg\/ml denatured salmon sperm DNA followed by washing in a solution comprising 1.0\u00d7 SSPE, 1.0% SDS at 42 C when a probe of about 500 nucleotides in length is employed.","\u201cLow stringency conditions\u201d comprise conditions equivalent to binding or hybridization at 42 C in a solution consisting of 5\u00d7 SSPE (43.8 g\/l NaCl, 6.9 g\/l NaHPOHO and 1.85 g\/l EDTA, pH adjusted to 7.4 with NaOH), 0.1% SDS, 5\u00d7 Denhardt's reagent [50\u00d7 Denhardt's contains per 500 ml: 5 g Ficoll (Type 400, Pharamcia), 5 g BSA (Fraction V; Sigma)] and 100 g\/ml denatured salmon sperm DNA followed by washing in a solution comprising 5\u00d7 SSPE, 0. 1% SDS at 42 C when a probe of about 500 nucleotides in length is employed.","The term \u201cgene\u201d refers to a DNA sequence that comprises control and coding sequences necessary for the production of an RNA having a non-coding function (e.g., a ribosomal or transfer RNA), a polypeptide or a precursor. The RNA or polypeptide can be encoded by a full length coding sequence or by any portion of the coding sequence so long as the desired activity or function is retained.","The term \u201cwild-type\u201d refers to a gene or a gene product that has the characteristics of that gene or gene product when isolated from a naturally occurring source. A wild-type gene is that which is most frequently observed in a population and is thus arbitrarily designated the \u201cnormal\u201d or \u201cwild-type\u201d form of the gene. In contrast, the term \u201cmodified\u201d, \u201cmutant\u201d or \u201cpolymorphic\u201d refers to a gene or gene product which displays modifications in sequence and or functional properties (i.e., altered characteristics) when compared to the wild-type gene or gene product. It is noted that naturally-occurring mutants can be isolated; these are identified by the fact that they have altered characteristics when compared to the wild-type gene or gene product.","The term \u201crecombinant DNA vector\u201d as used herein refers to DNA sequences containing a desired heterologous sequence. For example, although the term is not limited to the use of expressed sequences or sequences that encode an expression product, in some embodiments, the heterologous sequence is a coding sequence and appropriate DNA sequences necessary for either the replication of the coding sequence in a host organism, or the expression of the operably linked coding sequence in a particular host organism. DNA sequences necessary for expression in prokaryotes include a promoter, optionally an operator sequence, a ribosome binding site and possibly other sequences. Eukaryotic cells are known to utilize promoters, polyadenlyation signals and enhancers.","The term \u201cLTR\u201d as used herein refers to the long terminal repeat found at each end of a provirus (i.e., the integrated form of a retrovirus). The LTR contains numerous regulatory signals including transcriptional control elements, polyadenylation signals and sequences needed for replication and integration of the viral genome. The viral LTR is divided into three regions called U3, R and U5.","The U3 region contains the enhancer and promoter elements. The U5 region contains the polyadenylation signals. The R (repeat) region separates the U3 and U5 regions and transcribed sequences of the R region appear at both the 5\u2032 and 3\u2032 ends of the viral RNA.","The term \u201coligonucleotide\u201d as used herein is defined as a molecule comprising two or more deoxyribonucleotides or ribonucleotides, preferably at least 5 nucleotides, more preferably at least about 10-15 nucleotides and more preferably at least about 15 to 30 nucleotides. The exact size will depend on many factors, which in turn depend on the ultimate function or use of the oligonucleotide. The oligonucleotide may be generated in any manner, including chemical synthesis, DNA replication, reverse transcription, PCR, or a combination thereof.","Because mononucleotides are reacted to make oligonucleotides in a manner such that the 5\u2032 phosphate of one mononucleotide pentose ring is attached to the 3\u2032 oxygen of its neighbor in one direction via a phosphodiester linkage, an end of an oligonucleotide is referred to as the \u201c5\u2032 end\u201d if its 5\u2032 phosphate is not linked to the 3\u2032 oxygen of a mononucleotide pentose ring and as the \u201c3\u2032 end\u201d if its 3\u2032 oxygen is not linked to a 5\u2032 phosphate of a subsequent mononucleotide pentose ring. As used herein, a nucleic acid sequence, even if internal to a larger oligonucleotide, also may be said to have 5\u2032 and 3\u2032 ends. A first region along a nucleic acid strand is said to be upstream of another region if the 3\u2032 end of the first region is before the 5\u2032 end of the second region when moving along a strand of nucleic acid in a 5\u2032 to 3\u2032 direction.","When two different, non-overlapping oligonucleotides anneal to different regions of the same linear complementary nucleic acid sequence, and the 3\u2032 end of one oligonucleotide points towards the 5\u2032 end of the other, the former may be called the \u201cupstream\u201d oligonucleotide and the latter the \u201cdownstream\u201d oligonucleotide. Similarly, when two overlapping oligonucleotides are hybridized to the same linear complementary nucleic acid sequence, with the first oligonucleotide positioned such that its 5\u2032 end is upstream of the 5\u2032 end of the second oligonucleotide, and the 3\u2032 end of the first oligonucleotide is upstream of the 3\u2032 end of the second oligonucleotide, the first oligonucleotide may be called the \u201cupstream\u201d oligonucleotide and the second oligonucleotide may be called the \u201cdownstream\u201d oligonucleotide.","The term \u201cprimer\u201d refers to an oligonucleotide that is capable of acting as a point of initiation of synthesis when placed under conditions in which primer extension is initiated. An oligonucleotide \u201cprimer\u201d may occur naturally, as in a purified restriction digest or may be produced synthetically.","A primer is selected to be \u201csubstantially\u201d complementary to a strand of specific sequence of the template. A primer must be sufficiently complementary to hybridize with a template strand for primer elongation to occur. A primer sequence need not reflect the exact sequence of the template. For example, a non-complementary nucleotide fragment may be attached to the 5\u2032 end of the primer, with the remainder of the primer sequence being substantially complementary to the strand. Non-complementary bases or longer sequences can be interspersed into the primer, provided that the primer sequence has sufficient complementarity with the sequence of the template to hybridize and thereby form a template primer complex for synthesis of the extension product of the primer.","The term \u201clabel\u201d as used herein refers to any atom or molecule that can be used to provide a detectable (preferably quantifiable) signal, and that can be attached to a nucleic acid or protein. Labels may provide signals detectable by fluorescence, radioactivity, colorimetry, gravimetry, X-ray diffraction or absorption, magnetism, enzymatic activity, and the like. A label may be a charged moiety (positive or negative charge) or alternatively, may be charge neutral. Labels can include or consist of nucleic acid or protein sequence, so long as the sequence comprising the label is detectable.","The term \u201ccleavage structure\u201d as used herein, refers to a structure that is formed by the interaction of at least one probe oligonucleotide and a target nucleic acid, forming a structure comprising a duplex, the resulting structure being cleavable by a cleavage means, including but not limited to an enzyme. The cleavage structure is a substrate for specific cleavage by the cleavage means in contrast to a nucleic acid molecule that is a substrate for non-specific cleavage by agents such as phosphodiesterases which cleave nucleic acid molecules without regard to secondary structure (i.e., no formation of a duplexed structure is required).","The term \u201cfolded cleavage structure\u201d as used herein, refers to a region of a single-stranded nucleic acid substrate containing secondary structure, the region being cleavable by an enzymatic cleavage means. The cleavage structure is a substrate for specific cleavage by the cleavage means in contrast to a nucleic acid molecule which is a substrate for non-specific cleavage by agents such as phosphodiesterases which cleave nucleic acid molecules without regard to secondary structure (i.e., no folding of the substrate is required).","As used herein, the term \u201cfolded target\u201d refers to a nucleic acid strand that contains at least one region of secondary structure (i.e., at least one double stranded region and at least one single-stranded region within a single strand of the nucleic acid). A folded target may comprise regions of tertiary structure in addition to regions of secondary structure.","The term \u201ccleavage means\u201d or \u201ccleavage agent\u201d as used herein refers to any means that is capable of cleaving a cleavage structure, including but not limited to enzymes. The cleavage means may include native DNAPs having 5\u2032 nuclease activity (e.g., Taq DNA polymerase, DNA polymerase I) and, more specifically, modified DNAPs having 5\u2032 nuclease but lacking synthetic activity. . \u201cStructure-specific nucleases\u201d or \u201cstructure-specific enzymes\u201d are enzymes that recognize specific secondary structures in a nucleic molecule and cleave these structures. The cleavage means of the invention cleave a nucleic acid molecule in response to the formation of cleavage structures; it is not necessary that the cleavage means cleave the cleavage structure at any particular location within the cleavage structure.","The cleavage means is not restricted to enzymes having solely 5\u2032 nuclease activity. The cleavage means may include nuclease activity provided from a variety of sources including the Cleavase enzymes, the FEN-1 endonucleases (including RAD2 and XPG proteins), Taq DNA polymerase and DNA polymerase I.","The term \u201cthermostable\u201d when used in reference to an enzyme, such as a 5\u2032 nuclease, indicates that the enzyme is functional or active (i.e., can perform catalysis) at an elevated temperature, i.e., at about 55\u00b0 C. or higher.","The term \u201ccleavage products\u201d as used herein, refers to products generated by the reaction of a cleavage means with a cleavage structure (i.e., the treatment of a cleavage structure with a cleavage means).","The term \u201ctarget nucleic acid\u201d refers to a nucleic acid molecule containing a sequence that has at least partial complementarity with at least a probe oligonucleotide and may also have at least partial complementarity with an INVADER oligonucleotide. The target nucleic acid may comprise single- or double-stranded DNA or RNA.","The term \u201cprobe oligonucleotide\u201d refers to an oligonucleotide that interacts with a target nucleic acid to form a cleavage structure in the presence or absence of an INVADER oligonucleotide. When annealed to the target nucleic acid, the probe oligonucleotide and target form a cleavage structure and cleavage occurs within the probe oligonucleotide.","The term \u201cnon-target cleavage product\u201d refers to a product of a cleavage reaction that is not derived from the target nucleic acid. As discussed above, in the methods of the present invention, cleavage of the cleavage structure generally occurs within the probe oligonucleotide. The fragments of the probe oligonucleotide generated by this target nucleic acid-dependent cleavage are \u201cnon-target cleavage products.\u201d","The term \u201cINVADER oligonucleotide\u201d refers to an oligonucleotide that hybridizes to a target nucleic acid at a location near the region of hybridization between a probe and the target nucleic acid, wherein the INVADER oligonucleotide comprises a portion (e.g., a chemical moiety, or nucleotide\u2014whether complementary to that target or not) that overlaps with the region of hybridization between the probe and target. In some embodiments, the INVADER oligonucleotide contains sequences at its 3\u2032 end that are substantially the same as sequences located at the 5\u2032 end of a probe oligonucleotide.","The term \u201csubstantially single-stranded\u201d when used in reference to a nucleic acid substrate means that the substrate molecule exists primarily as a single strand of nucleic acid in contrast to a double-stranded substrate which exists as two strands of nucleic acid which are held together by inter-strand base pairing interactions.","The term \u201csequence variation\u201d as used herein refers to differences in nucleic acid sequence between two nucleic acids. For example, a wild-type structural gene and a mutant form of this wild-type structural gene may vary in sequence by the presence of single base substitutions and\/or deletions or insertions of one or more nucleotides. These two forms of the structural gene are said to vary in sequence from one another. A second mutant form of the structural gene may exist. This second mutant form is said to vary in sequence from both the wild-type gene and the first mutant form of the gene.","The term \u201cliberating\u201d as used herein refers to the release of a nucleic acid fragment from a larger nucleic acid fragment, such as an oligonucleotide, by the action of, for example, a 5\u2032 nuclease such that the released fragment is no longer covalently attached to the remainder of the oligonucleotide.","The term \u201cK\u201d as used herein refers to the Michaelis-Menten constant for an enzyme and is defined as the concentration of the specific substrate at which a given enzyme yields one-half its maximum velocity in an enzyme catalyzed reaction.","The term \u201cnucleotide analog\u201d as used herein refers to modified or non-naturally occurring nucleotides such as 7-deaza purines (i.e., 7-deaza-dATP and 7-deaza-dGTP). Nucleotide analogs include base analogs and comprise modified forms of deoxyribonucleotides as well as ribonucleotides.","The term \u201cpolymorphic locus\u201d is a locus present in a population that shows variation between members of the population (e.g., the most common allele has a frequency of less than 0.95). In contrast, a \u201cmonomorphic locus\u201d is a genetic locus at little or no variations seen between members of the population (generally taken to be a locus at which the most common allele exceeds a frequency of 0.95 in the gene pool of the population).","The term \u201cmicroorganism\u201d as used herein means an organism too small to be observed with the unaided eye and includes, but is not limited to bacteria, virus, protozoans, fungi, and ciliates.","The term \u201cmicrobial gene sequences\u201d refers to gene sequences derived from a microorganism.","The term \u201cbacteria\u201d refers to any bacterial species including eubacterial and archaebacterial species.","The term \u201cvirus\u201d refers to obligate, ultramicroscopic, intracellular parasites incapable of autonomous replication (i.e., replication requires the use of the host cell's machinery).","The term \u201cmulti-drug resistant\u201d or multiple-drug resistant\u201d refers to a microorganism which is resistant to more than one of the antibiotics or antimicrobial agents used in the treatment of said microorganism.","The term \u201csample\u201d in the present specification and claims is used in its broadest sense. On the one hand it is meant to include a specimen or culture (e.g., microbiological cultures). On the other hand, it is meant to include both biological and environmental samples. A sample may include a specimen of synthetic origin.","Biological samples may be animal, including human, fluid, solid (e.g., stool) or tissue, as well as liquid and solid food and feed products and ingredients such as dairy items, vegetables, meat and meat by-products, and waste. Biological samples may be obtained from all of the various families of domestic animals, as well as feral or wild animals, including, but not limited to, such animals as ungulates, bear, fish, lagamorphs, rodents, etc.","Environmental samples include environmental material such as surface matter, soil, water and industrial samples, as well as samples obtained from food and dairy processing instruments, apparatus, equipment, utensils, disposable and non-disposable items. These examples are not to be construed as limiting the sample types applicable to the present invention.","The term \u201csource of target nucleic acid\u201d refers to any sample that contains nucleic acids (RNA or DNA). Particularly preferred sources of target nucleic acids are biological samples including, but not limited to blood, saliva, cerebral spinal fluid, pleural fluid, milk, lymph, sputum and semen.","An oligonucleotide is said to be present in \u201cexcess\u201d relative to another oligonucleotide (or target nucleic acid sequence) if that oligonucleotide is present at a higher molar concentration that the other oligonucleotide (or target nucleic acid sequence). When an oligonucleotide such as a probe oligonucleotide is present in a cleavage reaction in excess relative to the concentration of the complementary target nucleic acid sequence, the reaction may be used to indicate the amount of the target nucleic acid present. Typically, when present in excess, the probe oligonucleotide will be present at least a 100-fold molar excess; typically at least 1 pmole of each probe oligonucleotide would be used when the target nucleic acid sequence was present at about 10 fmoles or less.","A sample \u201csuspected of containing\u201d a first and a second target nucleic acid may contain either, both or neither target nucleic acid molecule.","The term \u201ccharge-balanced\u201d oligonucleotide refers to an oligonucleotide (the input oligonucleotide in a reaction) that has been modified such that the modified oligonucleotide bears a charge, such that when the modified oligonucleotide is either cleaved (i.e., shortened) or elongated, a resulting product bears a charge different from the input oligonucleotide (the \u201ccharge-unbalanced\u201d oligonucleotide) thereby permitting separation of the input and reacted oligonucleotides on the basis of charge. The term \u201ccharge-balanced\u201d does not imply that the modified or balanced oligonucleotide has a net neutral charge (although this can be the case). Charge-balancing refers to the design and modification of an oligonucleotide such that a specific reaction product generated from this input oligonucleotide can be separated on the basis of charge from the input oligonucleotide.","For example, in an INVADER oligonucleotide-directed cleavage assay in which the probe oligonucleotide bears the sequence: 5\u2032 TTCTTTTCACCAGCGAGACGGG 3\u2032 (i.e., SEQ ID NO:61 without the modified bases) and cleavage of the probe occurs between the second and third residues, one possible charge-balanced version of this oligonucleotide would be: 5\u2032 Cy3-AminoT-Amino-TCTTTTCACCAGCGAGAC GGG 3\u2032. This modified oligonucleotide bears a net negative charge. After cleavage, the following oligonucleotides are generated: 5\u2032 Cy3-AminoT-Amino-T 3\u2032 and 5\u2032 CTTTTCACCAGCGAGACGGG 3\u2032 (residues 3-22 of SEQ ID NO:61). 5\u2032 Cy3-AminoT-Amino-T 3\u2032 bears a detectable moiety (the positively-charged Cy3 dye) and two amino- modified bases. The arnino-modified bases and the Cy3 dye contribute positive charges in excess of the negative charges contributed by the phosphate groups and thus the 5\u2032 Cy3-AminoT-Amino-T 3\u2032 oligonucleotide has a net positive charge. The other, longer cleavage fragment, like the input probe, bears a net negative charge. Because the 5\u2032 Cy3-AminoT-Amino-T 3\u2032 fragment is separable on the basis of charge from the input probe (the charge-balanced oligonucleotide), it is referred to as a charge-unbalanced oligonucleotide. The longer cleavage product cannot be separated on the basis of charge from the input oligonucleotide as both oligonucleotides bear a net negative charge; thus, the longer cleavage product is not a charge-unbalanced oligonucleotide.","The term \u201cnet neutral charge\u201d when used in reference to an oligonucleotide, including modified oligonucleotides, indicates that the sum of the charges present (i.e., R-NH3+ groups on thymidines, the N3 nitrogen of cytosine, presence or absence or phosphate groups, etc.) under the desired reaction or separation conditions is essentially zero. An oligonucleotide having a net neutral charge would not migrate in an electrical field.","The term \u201cnet positive charge\u201d when used in reference to an oligonucleotide, including modified oligonucleotides, indicates that the sum of the charges present (i.e., R-NH3+ groups on thymidines, the N3 nitrogen of cytosine, presence or absence or phosphate groups, etc.) under the desired reaction conditions is +1 or greater. An oligonucleotide having a net positive charge would migrate toward the negative electrode in an electrical field.","The term \u201cnet negative charge\u201d when used in reference to an oligonucleotide, including modified oligonucleotides, indicates that the sum of the charges present (i.e., R-NH3+ groups on thymidines, the N3 nitrogen of cytosine, presence or absence or phosphate groups, etc.) under the desired reaction conditions is \u22121 or lower. An oligonucleotide having a net negative charge would migrate toward the positive electrode in an electrical field.","The term \u201cpolymerization means\u201d or \u201cpolymerization agent\u201d refers to any agent capable of facilitating the addition of nucleoside triphosphates to an oligonucleotide. Preferred polymerization means comprise DNA and RNA polymerases.","The term \u201cligation means\u201d or \u201cligation agent\u201d refers to any agent capable of facilitating the ligation (i.e., the formation of a phosphodiester bond between a 3\u2032-OH and a 5\u2032 P located at the termini of two strands of nucleic acid). Preferred ligation means comprise DNA ligases and RNA ligases.","The term \u201creactant\u201d is used herein in its broadest sense. The reactant can comprise, for example, an enzymatic reactant, a chemical reactant or light (e.g., ultraviolet light, particularly short wavelength ultraviolet light is known to break oligonucleotide chains). Any agent capable of reacting with an oligonucleotide to either shorten (i.e., cleave) or elongate the oligonucleotide is encompassed within the term \u201creactant.\u201d","The term \u201cadduct\u201d is used herein in its broadest sense to indicate any compound or element that can be added to an oligonucleotide. An adduct may be charged (positively or negatively) or may be charge-neutral. An adduct may be added to the oligonucleotide via covalent or non-covalent linkages. Examples of adducts include, but are not limited to, indodicarbocyanine dye amidites, amino-substituted nuclcotides, ethidium bromide, ethidium homodimer, (1,3-propanediamino)propidium, (diethylenetriamino)propidium, thiazole orange, (N-N\u2032-tetramethyl-1,3-propanediamino)propyl thiazole orange, (N-N\u2032-tetramethyl-1,2-ethanediamino)propyl thiazole orange, thiazole orange-thiazole orange homodimer (TOTO), thiazole orange-thiazole blue heterodimer (TOTAB), thiazole orange-ethidium heterodimer 1 (TOED 1), thiazole orange-ethidium heterodimer 2 (TOED2) and fluorescein-ethidium heterodimer (FED), psoralens, biotin, streptavidin, avidin, etc.","Where a first oligonucleotide is complementary to a region of a target nucleic acid and a second oligonucleotide has complementary to the same region (or a portion of this region) a \u201cregion of overlap\u201d exists along the target nucleic acid. The degree of overlap will vary depending upon the nature of the complementarity (see, e.g., region \u201cX\u201d in  and the accompanying discussions).","As used herein, the term \u201cpurified\u201d or \u201cto purify\u201d refers to the removal of contaminants from a sample. For example, recombinant Cleavase nucleases are expressed in bacterial host cells and the nucleases are purified by the removal of host cell proteins; the percent of these recombinant nucleases is thereby increased in the sample.","The term \u201crecombinant DNA molecule\u201d as used herein refers to a DNA molecule that comprises of segments of DNA joined together by means of molecular biological techniques.","The term \u201crecombinant protein\u201d or \u201crecombinant polypeptide\u201d as used herein refers to a protein molecule that is expressed from a recombinant DNA molecule.","As used herein the term \u201cportion\u201d when in reference to a protein (as in \u201ca portion of a given protein\u201d) refers to fragments of that protein. The fragments may range in size from four amino acid residues to the entire amino acid sequence minus one amino acid (e.g., 4, 5, 6, . . . , n\u22121).","The term \u201cnucleic acid sequence\u201d as used herein refers to an oligonucleotide, nucleotide or polynucleotide, and fragments or portions thereof, and to DNA or RNA of genomic or synthetic origin which may be single or double stranded, and represent the sense or antisense strand. Similarly, \u201camino acid sequence\u201d as used herein refers to peptide or protein sequence.","The term \u201cpeptide nucleic acid\u201d (\u201cPNA\u201d) as used herein refers to a molecule comprising bases or base analogs such as would be found in natural nucleic acid, but attached to a peptide backbone rather than the sugar-phosphate backbone typical of nucleic acids. The attachment of the bases to the peptide is such as to allow the bases to base pair with complementary bases of nucleic acid in a manner similar to that of an oligonucleotide. These small molecules, also designated anti gene agents, stop transcript elongation by binding to their complementary strand of nucleic acid (Nielsen, et al. Anticancer Drug Des. 8:53 63 [1993]).","As used herein, the terms \u201cpurified\u201d or \u201csubstantially purified\u201d refer to molecules, either nucleic or amino acid sequences, that are removed from their natural environment, isolated or separated, and are at least 60% free, preferably 75% free, and most preferably 90% free from other components with which they are naturally associated. An \u201cisolated polynucleotide\u201d or \u201cisolated oligonucleotide\u201d is therefore a substantially purified polynucleotide.","An isolated oligonucleotide (or polynucleotide) encoding a Pyrococcus woesei (Pwo) FEN-1 endonuclease having a region capable of hybridizing to SEQ ID NO:116 is an oligonucleotide containing sequences encoding at least the amino-terminal portion of Pwo FEN-1 endonuclease. An isolated oligonucleotide (or polynucleotide) encoding a Pwo FEN-1 endonuclease having a region capable of hybridizing to SEQ ID NO:117 is an oligonucleotide containing sequences encoding at least the carboxy-terninal portion of Pwo FEN-1 endonuclease. An isolated oligonucleotide (or polynucleotide) encoding a Pwo FEN-1 endonuclease having a region capable of hybridizing to SEQ ID NOS:118 and 119 is an oligonucleotide containing sequences encoding at least portions of Pwo FEN-1 endonuclease protein located internal to either the amino or carboxy-termini of the Pwo FEN-1 endonuclease protein.","As used herein, the term \u201cfusion protein\u201d refers to a chimeric protein containing the protein of interest (e.g., Cleavase BN\/thrombin nuclease and portions or fragments thereof) joined to an exogenous protein fragment (the fusion partner which consists of a non Cleavase BN\/thrombin nuclease protein). The fusion partner may enhance solubility of recombinant chimeric protein (e.g., the Cleavase BN\/thrombin nuclease) as expressed in a host cell, may provide an affinity tag (e.g., a his-tag) to allow purification of the recombinant fusion protein from the host cell or culture supernatant, or both. If desired, the fusion protein may be removed from the protein of interest (e.g., Cleavase BN\/thrombin nuclease or fragments thereof) by a variety of enzymatic or chemical means known to the art.","As used herein, the terms \u201cchimeric protein\u201d and \u201cchimerical protein\u201d refer to a single protein molecule that comprises amino acid sequences portions derived from two or more parent proteins. These parent molecules may be from similar proteins from genetically distinct origins, different proteins from a single organism, or different proteins from different organisms. By way of example but not by way of limitation, a chimeric structure-specific nuclease of the present invention may contain a mixture of amino acid sequences that have been derived from FEN-1 genes from two or more of the organisms having such genes, combined to form a non-naturally occurring nuclease. The term \u201cchimerical\u201d as used herein is not intended to convey any particular proportion of contribution from the naturally occurring genes, nor limit the manner in which the portions are combined. Any chimeric structure-specific nuclease constructs having cleavage activity as determined by the testing methods described herein are improved cleavage agents within the scope of the present invention.","The term \u201ccontinuous strand of nucleic acid\u201d as used herein is means a strand of nucleic acid that has a continuous, covalently linked, backbone structure, without nicks or other disruptions. The disposition of the base portion of each nucleotide, whether base-paired, single-stranded or mismatched, is not an element in the definition of a continuous strand. The backbone of the continuous strand is not limited to the ribose-phosphate or deoxyribose-phosphate compositions that are found in naturally occurring, unmodified nucleic acids. A nucleic acid of the present invention may comprise modifications in the structure of the backbone, including but not limited to phosphorothioate residues, phosphonate residues, 2\u2032 substituted ribose residues (e.g., 2\u2032-O-methyl ribose) and alternative sugar (e.g., arabinose) containing residues.","The term \u201ccontinuous duplex\u201d as used herein refers to a region of double stranded nucleic acid in which there is no disruption in the progression of basepairs within the duplex (i.e., the base pairs along the duplex are not distorted to accommodate a gap, bulge or mismatch with the confines of the region of continuous duplex). As used herein the term refers only to the arrangement of the basepairs within the duplex, without implication of continuity in the backbone portion of the nucleic acid strand. Duplex nucleic acids with uninterrupted basepairing, but with nicks in one or both strands are within the definition of a continuous duplex.","The term \u201cduplex\u201d refers to the state of nucleic acids in which the base portions of the nucleotides on one strand are bound through hydrogen bonding the their complementary bases arrayed on a second strand. The condition of being in a duplex form reflects on the state of the bases of a nucleic acid. By virtue of base pairing, the strands of nucleic acid also generally assume the tertiary structure of a double helix, having a major and a minor groove. The assumption of the helical form is implicit in the act of becoming duplexed.","The term \u201cduplex dependent protein binding\u201d refers to the binding of proteins to nucleic acid that is dependent on the nucleic acid being in a duplex, or helical form.","The term \u201cduplex dependent protein binding sites or regions\u201d as used herein refers to discrete regions or sequences within a nucleic acid that are bound with particular affinity by specific duplex-dependent nucleic acid binding proteins. This is in contrast to the generalized duplex-dependent binding of proteins that are not site-specific, such as the histone proteins that bind chromatin with little reference to specific sequences or sites.","The term \u201cprotein binding region\u201d as used herein refers to a nucleic acid region identified by a sequence or structure as binding to a particular protein or class of proteins. It is within the scope of this definition to include those regions that contain sufficient genetic information to allow identifications of the region by comparison to known sequences, but which might not have the requisite structure for actual binding (e.g., a single strand of a duplex-depending nucleic acid binding protein site). As used herein \u201cprotein binding region\u201d excludes restriction endonuclease binding regions.","The term \u201ccomplete double stranded protein binding region\u201d as used herein refers to the minimum region of continuous duplex required to allow binding or other activity of a duplex-dependent protein. This definition is intended to encompass the observation that some duplex dependent nucleic acid binding proteins can interact with full activity with regions of duplex that may be shorter than a canonical protein binding region as observed in one or the other of the two single strands. In other words, one or more nucleotides in the region may be allowed to remain unpaired without suppressing binding. As used here in, the term \u201ccomplete double stranded binding region\u201d refers to the minimum sequence that will accommodate the binding function. Because some such regions can tolerate non-duplex sequences in multiple places, although not necessarily simultaneously, a single protein binding region might have several shorter sub-regions that, when duplexed, will be fully competent for protein binding.","The term \u201ctemplate\u201d refers to a strand of nucleic acid on which a complementary copy is built from nucleoside triphosphates through the activity of a template-dependent nucleic acid polymerase. Within a duplex the template strand is, by convention, depicted and described as the \u201cbottom\u201d strand. Similarly, the non-template strand is often depicted and described as the \u201ctop\u201d strand.","The term \u201ctemplate-dependent RNA polymerase\u201d refers to a nucleic acid polymerase that creates new RNA strands through the copying of a template strand as described above and which does not synthesize RNA in the absence of a template. This is in contrast to the activity of the template-independent nucleic acid polymerases that synthesize or extend nucleic acids without reference to a template, such as terminal deoxynucleotidyl transferase, or Poly A polymerase.","The term \u201cARRESTOR\u201d refers to an agent added to or included in an invasive cleavage reaction in order to stop one or more reaction components from participating in a subsequent action or reaction. This may be done by sequestering or inactivating some reaction component (e.g., by binding or base-pairing a nucleic acid component, or by binding to a protein component). The term \u201cARRESTOR oligonucleotide\u201d refers to an oligonucleotide included in an invasive cleavage reaction in order to stop or arrest one or more aspects of any reaction (e.g., the first reaction and\/or any subsequent reactions or actions; it is not intended that the ARRESTOR oligonucleotide be limited to any particular reaction or reaction step). This may be done by sequestering some reaction component (e.g., base-pairing to another nucleic acid, or binding to a protein component). However, it is not intended that the term be so limited as to just situations in which a reaction component is sequestered.","Introduction:","The present invention relates to methods and compositions for treating nucleic acid, and in particular, methods and compositions for detection and characterization of nucleic acid sequences and sequence changes.","In preferred embodiments, the present invention relates to means for cleaving a nucleic acid cleavage structure in a site-specific manner. While the present invention provides a variety of cleavage agents, in some embodiments, the present invention relates to a cleaving enzyme having 5\u2032 nuclease activity without interfering nucleic acid synthetic ability. In other embodiments, the present invention provides novel polymerases (e.g., thermostable polymerases) possessing altered polymerase and\/or nucleases activities.","For example, in some embodiments, the present invention provides 5\u2032 nucleases derived from thermostable DNA polymerases that exhibit altered DNA synthetic activity from that of native thermostable DNA polymerases. The 5\u2032 nuclease activity of the polymerase is retained while the synthetic activity is reduced or absent. Such 5\u2032 nucleases are capable of catalyzing the structure-specific cleavage of nucleic acids in the absence of interfering synthetic activity. The lack of synthetic activity during a cleavage reaction results in nucleic acid cleavage products of uniform size.","The novel properties of the nucleases of the invention form the basis of a method of detecting specific nucleic acid sequences. This method relies upon the amplification of the detection molecule rather than upon the amplification of the target sequence itself as do existing methods of detecting specific target sequences.","DNA polymerases (DNAPs), such as those isolated from or from thermophilic bacteria of the genus Thermus as well as other organisms, arc enzymes that synthesize new DNA strands. Several of the known DNAPs contain associated nuclease activities in addition to the synthetic activity of the enzyme.","Some DNAPs are known to remove nucleotides from the 5\u2032 and 3\u2032 ends of DNA chains (Kornberg, DNA Replication, W.H. Freeman and Co., San Francisco, pp. 127-139 [1980]). These nuclease activities are usually referred to as 5\u2032 exonuclease and 3\u2032 exonuclease activities, respectively. For example, the 5\u2032 exonuclease activity located in the N-terminal domain of several DNAPs participates in the removal of RNA primers during lagging strand synthesis during DNA replication and the removal of damaged nucleotides during repair. Some DNAPs, such as the DNA polymerase (DNAPEc 1), also have a 3\u2032 exonuclease activity responsible for proof-reading during DNA synthesis (Komberg, supra).","A DNAP isolated from , termed Taq DNA polymerase (DNAPTaq), has a 5\u2032 exonuclease activity, but lacks a functional 3\u2032 exonucleolytic domain (Tindall and Kunkell, Biochem., 27:6008 [1988]). Derivatives of DNAPEc1 and DNAPTaq, respectively called the Klenow and Stoffel fragments, lack 5\u2032 exonuclease domains as a result of enzymatic or genetic manipulations (Brutlag et al., Biochem. Biophys. Res. Commun., 37:982 [1969]; Erlich et al., Science 252:1643 [1991]; Setlow and Komberg, J. Biol. Chem., 247:232 [1972]).","The 5\u2032 exonuclease activity of DNAPTaq was reported to require concurrent synthesis (Gelfand, PCR Technology\u2014Principles and Applications for DNA Amplification, H.A. Erlich, [Ed.], Stockton Press, New York, p. 19 [1989]). Although mononucleotides predominate among the digestion products of the 5\u2032 exonucleases of DNAPTaq and DNAPEc1, short oligonucleotides (\u226612 nucleotides) can also be observed implying that these so-called 5\u2032 exonucleases can function endonucleolytically (Setlow, supra; Holland et al., Proc. Natl. Acad. Sci. USA 88:7276 [1991]).","In WO 92\/06200, Gelfand et al. show that the preferred substrate of the 5\u2032 exonuclcase activity of the thermostable DNA polymerases is displaced single-stranded DNA. Hydrolysis of the phosphodiester bond occurs between the displaced single-stranded DNA and the double-helical DNA with the preferred exonuclease cleavage site being a phosphodiester bond in the double helical region. Thus, the 5\u2032 exonuclease activity usually associated with DNAPs is a structure-dependent single-stranded endonuclease and is more properly referred to as a 5\u2032 nuclease. Exonucleases are enzymes that cleave nucleotide molecules from the ends of the nucleic acid molecule. Endonucleases, on the other hand, are enzymes that cleave the nucleic acid molecule at internal rather than terminal sites. The nuclease activity associated with some thermostable DNA polymerases cleaves endonucleolytically but this cleavage requires contact with the 5\u2032 end of the molecule being cleaved. Therefore, these nucleases are referred to as 5\u2032 nucleases.","When a 5\u2032 nuclease activity is associated with a eubacterial Type A DNA polymerase, it is found in the one third N-terminal region of the protein as an independent functional domain. The C-terminal two-thirds of the molecule constitute the polymerization domain that is responsible for the synthesis of DNA. Some Type A DNA polymerases also have a 3\u2032 exonuclease activity associated with the two-third C-terminal region of the molecule.","The 5\u2032 exonuclease activity and the polymerization activity of DNAPs can be separated by proteolytic cleavage or genetic manipulation of the polymerase molecule. The Klenow or large proteolytic cleavage fragment of DNAPEc l contains the polymerase and 3\u2032 exonuclease activity but lacks the 5\u2032 nuclease activity. The Stoffel fragment of DNAPTaq (DNAPStf) lacks the 5\u2032 nuclease activity due to a genetic manipulation that deleted the N-terminal 289 amino acids of the polymerase molecule (Erlich et al., Science 252:1643 [1991]). WO 92\/06200 describes a thermostable DNAP with an altered level of 5\u2032 to 3\u2032 exonuclease. U.S. Pat. No. 5,108,892 describes a Thermus aquaticus DNAP without a 5\u2032 to 3\u2032 exonuclease. Thermostable DNA polymerases with lessened amounts of synthetic activity are available (Third Wave Technologies, Madison, Wis.) and are described in U.S. Pat. Nos. 5,541,311, 5,614,402, 5,795,763, 5,691,142, and 5,837,450, herein incorporated by reference in their entireties. The present invention provides 5\u2032 nucleases derived from thermostable Type A DNA polymerases that retain 5\u2032 nuclease activity but have reduced or absent synthetic activity. The ability to uncouple the synthetic activity of the enzyme from the 5\u2032 nuclcase activity proves that the 5\u2032 nuclease activity does not require concurrent DNA synthesis as was previously reported (Gelfand, PCR Technology, supra).","In addition to the 5\u2032-exonuclease domains of the DNA polymerase I proteins of Eubacteria, described above, 5\u2032 nucleases have been found associated with bacteriophage, eukaryotes and archaebacteria. Overall, all of the enzymes in this family display very similar substrate specificities, despite their limited level of sequence similarity. Consequently, enzymes suitable for use in the methods of the present invention may be isolated or derived from a wide array of sources.","A mammalian enzyme with functional similarity to the 5\u2032-exonuclease domain of Pol I was isolated nearly 30 years ago (Lindahl, et al., Proc Natl Acad Sci U S A 62(2): 597-603 [1969]). Later, additional members of this group of enzymes called flap endonucleases (FEN1) from Eukarya and Archaea were shown to possess a nearly identical structure specific activity (Harrington and Lieber. Embo J 13(5), 1235-46 [1994]; Murante et al., J Biol Chem 269(2), 1191-6 [1994]; Robins, et al., J Biol Chem 269(46), 28535-8 [1994]; Hosfield, et al., J Biol Chem 273(42), 27154-61 [1998]), despite limited sequence similarity. The substrate specificities of the FENI enzymes, and the eubacterial and related bacteriophage enzymes have been examined and found to be similar for all enzymes (Lyamichev, et al., Science 260(5109), 778-83 [1993], Harrington and Lieber, supra, Murante, et al., supra, Hosfield, et al, supra, Rao, et al., J Bacteriol 180(20), 5406-12 [1998], Bhagwat, et al,. J. Biol Chem 272(45), 28523-30 [1997], Garforth and Sayers, Nucleic Acids Res 25(19), 3801-7 [1997]).","Using preformed substrates, many of the studies cited above determined that these nucleases leave a gap upon cleavage, leading the authors to speculate that DNA polymerase must then act to fill in that gap to generate a ligatable nick. A number of other 5\u2032 nucleases have been shown to leave a gap or overlap after cleavage of the same or similar flap substrates. It has since been determined that all the structure-specific 5\u2032-exonucleases leave a nick after cleavage if the substrate has an overlap between the upstream and downstream duplexes (Kaiser et al., J. Biol Chem. 274(30):21387-21394 [1999]). While duplexes having several bases of overlapping sequence can assume several different conformations through branch migration, it was determined that cleavage occurs in the conformation where the last nucleotide at the 3\u2032 end of the upstream strand is unpaired, with the cleavage rate being essentially the same whether the end of the upstream primer is A, C, G, or T. It was determined to be positional overlap between the 3\u2032 end of the upstream primer and downstream duplex, rather then sequence overlap, that is required for optimal cleavage. In addition to allowing these enzymes to leave a nick after cleavage, the single base of overlap causes the enzymes to cleave several orders of magnitude faster than when a substrate lacks overlap (Kaiser et al., supra).","Any of the 5\u2032 nucleases described above may find application in one or more embodiments of the methods described herein. FEN1 nucleases of particular utility in the methods of present invention include but are not limited to those of and ; particularly preferred FEN1 enzymes are from and ","The detailed description of the invention is presented in the following sections:",{"@attributes":{"id":"p-0327","num":"0000"},"ul":{"@attributes":{"id":"ul0003","list-style":"none"},"li":["I. Detection of Specific Nucleic Acid Sequences Using 5\u2032 Nucleases in an INVADER Directed Cleavage Assay;","II. Effect of ARRESTOR Oligonucleotides on Signal and Background in Sequential Invasive Cleavage Reactions.","III. Signal Enhancement By Incorporating The Products Of An Invasive Cleavage Reaction Into A Subsequcnt Invasive Cleavage Reaction;","IV. Fractionation Of Specific Nucleic Acids By Selective Charge Reversal;","V. Signal Enhancement By Tailing Of Reaction Products In The INVADER oligonucleotide-directed Cleavage Assay;","VI. Signal Enhancement By Completion Of An Activated Protein Binding Site;","VII. Generation of 5\u2032 Nucleases Derived From Thermostable DNA Polymerases;","VIII. Improved Enzymes For Use In INVADER oligonucleotide-directed Cleavage Reactions;","IX. The INVADER assay for direct detection and measurement of specific analytes."]}},"1. INVADER Assay Reaction Design","The present invention provides means for forming a nucleic acid cleavage structure that is dependent upon the presence of a target nucleic acid and cleaving the nucleic acid cleavage structure so as to release distinctive clcavagc products. 5\u2032 nuclease activity, for example, is used to cleave the target-dependent cleavage structure and the resulting cleavage products are indicative of the presence of specific target nucleic acid sequences in the sample. When two strands of nucleic acid, or oligonucleotides, both hybridize to a target nucleic acid strand such that they form an overlapping invasive cleavage structure, as described below, invasive cleavage can occur. Through the interaction of a cleavage agent (e.g., a 5\u2032 nuclease) and the upstream oligonucleotide, the cleavage agent can be made to cleave the downstream oligonucleotide at an internal site in such a way that a distinctive fragment is produced. Such embodiments have been termed the INVADER assay (Third Wave Technologies) and are described in U.S. patent application Ser. Nos. 5,846,717, 5,985,557, 5,994,069, 6,001,567, and 6,090,543 and PCT Publications WO 97\/27214 and WO 98\/42873, herein incorporated by reference in their entireties.","The present invention further provides assays in which the target nucleic acid is reused or recycled during multiple rounds of hybridization with oligonucleotide probes and cleavage of the probes without the need to use temperature cycling (i.e., for periodic denaturation of target nucleic acid strands) or nucleic acid synthesis (i.e., for the polymerization-based displacement of target or probe nucleic acid strands). When a cleavage reaction is run under conditions in which the probes are continuously replaced on the target strand (e.g. through probe-probe displacement or through an equilibrium between probe\/target association and disassociation, or through a combination comprising these mechanisms, [The kinetics of oligonucleotide replacement. Luis P. Reynaldo, Alexander V. Vologodskii, Bruce P. Neri and Victor I. Lyamichev. J. Mol. Biol. 97: 511-520 (2000)], multiple probes can hybridize to the same target, allowing multiple cleavages, and the generation of multiple cleavage products.","By the extent of its complementarity to a target nucleic acid strand, an oligonucleotide may be said to define a specific region of said target. In an invasive cleavage structure, the two oligonucleotides define and hybridize to regions of the target that are adjacent to one another (i.e., regions without any additional region of the target between them). Either or both oligonucleotides may comprise additional portions that are not complementary to the target strand. In addition to hybridizing adjacently, in order to form an invasive cleavage structure, the 3\u2032 end of the upstream oligonucleotide must comprise an additional moiety. When both oligonucleotides are hybridized to a target strand to form a structure and such a 3\u2032 moiety is present on the upstream oligonucleotide within the structure, the oligonucleotides may be said to overlap, and the structure may be described as an overlapping, or invasive cleavage structure.","In one embodiment, the 3\u2032 moiety of the invasive cleavage structure is a single nucleotide. In this embodiment the 3\u2032 moiety may be any nucleotide (i.e., it may be, but it need not be complementary to the target strand). In a preferred embodiment the 3\u2032 moiety is a single nucleotide that is not complementary to the target strand. In another embodiment, the 3\u2032 moiety is a nucleotide-like compound (i.e., a moiety having chemical features similar to a nucleotide, such as a nucleotide analog or an organic ring compound; See e.g., U.S. Pat. No. 5,985,557). In yet another embodiment the 3\u2032 moiety is one or more nucleotides that duplicate in sequence one or more nucleotides present at the 5\u2032 end of the hybridized region of the downstream oligonucleotide. In a further embodiment, the duplicated sequence of nucleotides of the 3\u2032 moiety is followed by a single nucleotide that is not further duplicative of the downstream oligonucleotide sequence, and that may be any other nucleotide. In yet another embodiment, the duplicated sequence of nucleotides of the 3\u2032 moiety is followed by a nucleotide-like compound, as described above.","The downstream oligonucleotide may have, but need not have, additional moieties attached to either end of the region that hybridizes to the target nucleic acid strand. In a preferred embodiment, the downstream oligonucleotide comprises a moiety at its 5\u2032 end (i.e., a 5\u2032 moiety). In a particularly preferred embodiment, said 5\u2032 moiety is a 5\u2032 flap or arm comprising a sequence of nucleotides that is not complementary to the target nucleic acid strand.","When an overlapping cleavage structure is formed, it can be recognized and cleaved by a nuclease that is specific for this structure (i.e., a nuclease that will cleave one or more of the nucleic acids in the overlapping structure based on recognition of this structure, rather than on recognition of a nucleotide sequence of any of the nucleic acids forming the structure). Such a nuclease may be termed a \u201cstructure-specific nuclease\u201d. In some embodiments, the structure-specific nuclease is a 5\u2032 nuclease. In a preferred embodiment, the structure-specific nuclease is the 5\u2032 nuclease of a DNA polymerase. In another preferred embodiment, the DNA polymerase having the 5\u2032 nuclease is synthesis-deficient. In another preferred embodiment, the 5\u2032 nuclease is a FEN-1 endonuclease. In a particularly preferred embodiment, the 5\u2032 nuclease is thermostable.","In some embodiments, said structure-specific nuclease preferentially cleaves the downstream oligonucleotide. In a preferred embodiment, the downstream oligonucleotide is cleaved one nucleotide into the 5\u2032 end of the region that is hybridized to the target within the overlapping structure. Cleavage of the overlapping structure at any location by a structure-specific nuclease produces one or more released portions or fragments of nucleic acid, termed \u201ccleavage products\u201d.","In some embodiments, cleavage of an overlapping structure is performed under conditions wherein one or more of the nucleic acids in the structure can disassociate (i.e. un-hybridize, or melt) from the structure. In one embodiment, full or partial disassociation of a first cleavage structure allows the target nucleic acid to participate in the formation of one or more additional overlapping cleavage structures. In a preferred embodiment, the first cleavage structure is partially disassociated. In a particularly preferred embodiment only the oligonucleotide that is cleaved disassociates from the first cleavage structure, such that it may be replaced by another copy of the same oligonucleotide. In some embodiments, said disassociation is induced by an increase in temperature, such that one or more oligonucleotides can no longer hybridize to the target strand. In other embodiments, said disassociation occurs because cleavage of an oligonucleotide produces only cleavage products that cannot bind to the target strand under the conditions of the reaction. In a preferred embodiment, conditions arc selected wherein an oligonucleotide may associate with (i.e., hybridize to) and disassociate from a target strand regardless of cleavage, and wherein the oligonucleotide may be cleaved when it is hybridized to the target as part of an overlapping cleavage structure. In a particularly preferred embodiment, conditions are selected such that the number of copies of the oligonucleotide that can be cleaved when part of an overlapping structure exceeds the number of copies of the target nucleic acid strand by a sufficient amount that when the first cleavage structure disassociates, the probability that the target strand will associate with an intact copy of the oligonucleotide is greater than the probability that it will associate with a cleaved copy of the oligonucleotide.","In some embodiments, cleavage is performed by a structure-specific nuclease that can recognize and cleave structures that do not have an overlap. In a preferred embodiment, cleavage is performed by a structure-specific nuclease having a lower rate of cleavage of nucleic acid structures that do not comprise an overlap, compared to the rate of cleavage of structures comprising an overlap. In a particularly preferred embodiment, cleavage is performed by a structure-specific nuclease having less than 1% of the rate of cleavage of nucleic acid structures that do not comprise an overlap, compared to the rate of cleavage of structures comprising an overlap.","In some embodiments it is desirable to detect the cleavage of the overlapping cleavage structure. Detection may be by analysis of cleavage products or by analysis of one or more of the remaining uncleaved nucleic acids. For convenience, the following discussion will refer to the analysis of cleavage products, but it will be appreciated by those skilled in the art that these methods may as easily be applied to analysis of the uncleaved nucleic acids in an invasive cleavage reaction. Any method known in the art for analysis of nucleic acids, nucleic acid fragments or oligonucleotides may be applied to the detection of cleavage products.","In one embodiment, the cleavage products may be identified by chemical content, e.g., the relative amounts of each atom, each particular type of reactive group or each nucleotide base (Chargaffet al., . 177: 405 [1949]) they contain. In this way, a cleavage product may be distinguished from a longer nucleic acid from which it was released by cleavage, or from other nucleic acids.","In another embodiment, the cleavage products may be distinguished by a particular physical attribute, including but not limited to length, mass, charge, or charge-to-mass ratio. In yet another embodiment, the cleavage product may be distinguished by a behavior that is related to a physical attribute, including but not limited to rate of rotation in solution, rate of migration during electrophoresis, coefficient of sedimentation in centrifugation, time of flight in MALDI-TOF mass spectrometry, migration rate or other behavior in chromatography, melting temperature from a complementary nucleic acid, or precipitability from solution.","Detection of the cleavage products may be through release of a label. Such labels may include, but are not limited to one or more of any of dyes, radiolabels such as P or S, binding moieties such as biotin, mass tags, such as metal ions or chemical groups, charge tags, such as polyamines or charged dyes, haptens such as digoxgenin, luminogenic, phosphorescent or fluorogenic moieties, and fluorescent dyes, either alone or in combination with moieties that can suppress or shift emission spectra, such as by fluorescence resonance energy transfer (FRET) or collisional fluorescence energy transfer.","In some embodiments, analysis of cleavage products may include physical resolution or separation, for example by electrophoresis, hybridization or by selective binding to a support, or by mass spectrometry methods such as MALDI-TOF. In other embodiments, the analysis may be performed without any physical resolution or separation, such as by detection of cleavage-induced changes in fluorescence as in FRET-based analysis, or by cleavage-induced changes in the rotation rate of a nucleic acid in solution as in fluorescence polarization analysis.","Cleavage products can be used subsequently in any reaction or read-out method that can make use of oligonucleotides. Such reactions include, but are not limited to, modification reactions, such as ligation, tailing with a template-independent nucleic acid polymerase and primer extension with a tcmplate-dependent nucleic acid polymerase. The modification of the cleavage products may be for purposes including, but not limited to, addition of one or more labels or binding moieties, alteration of mass, addition of specific sequences, or for any other purpose that would facilitate analysis of either the cleavage products or analysis of any other by-product, result or consequence of the cleavage reaction.","Analysis of the cleavage products may involve subsequent steps or reactions that do not modify the cleavage products themselves. For example, cleavage products may be used to complete a functional structure, such as a competent promoter for in vitro transcription or another protein binding site. Analysis may include the step of using the completed structure for or to perform its function. One or more cleavage products may also be used to complete an overlapping cleavage structure, thereby enabling a subsequent cleavage reaction, the products of which may be detected or used by any of the methods described herein, including the participation in further cleavage reactions.","Certain preferred embodiments of the invasive cleavage reactions are provided in the following descriptions. As exemplified by the diagram in , the methods of the present invention employ at least a pair of oligonucleotides that interact with a target nucleic acid to form a cleavage structure for a structure-specific nuclease. In some embodiments, the cleavage structure comprises i) a target nucleic acid that may be either single-stranded or double-stranded (when a double-stranded target nucleic acid is employed, it may be rendered single stranded, e.g., by heating); ii) a first oligonucleotide, termed the \u201cprobe,\u201d that defines a first region of the target nucleic acid sequence by being the complement of that region (regions X and Z of the target as shown in FIG. ); iii) a second oligonucleotide, termed the \u201cINVADER,\u201cthe 5\u2032 part of which defines a second region of the same target nucleic acid sequence (regions Y and X in FIG. ), adjacent to and downstream of the first target region (regions X and Z), and the second part of which overlaps into the region defined by the first oligonucleotide (region X depicts the region of overlap). The resulting structure is diagrammed in FIG. .","While not limiting the invention or the instant discussion to any particular mechanism of action, the diagram in  represents the effect on the site of cleavage caused by this type of arrangement of a pair of oligonucleotides. The design of such a pair of oligonucleotides is described below in detail. In , the 3\u2032 ends of the nucleic acids (i.e., the target and the oligonucleotides) are indicated by the use of the arrowheads on the ends of the lines depicting the strands of the nucleic acids (and where space permits, these ends are also labeled \u201c3\u2032\u201d). It is readily appreciated that the two oligonucleotides (the INVADER and the probe) are arranged in a parallel orientation relative to one another, while the target nucleic acid strand is arranged in an anti-parallel orientation relative to the two oligonucleotides. Further, it is clear that the INVADER oligonucleotide is located upstream of the probe oligonucleotide and that with respect to the target nucleic acid strand, region Z is upstream of region X and region X is upstream of region Y (that is, region Y is downstream of region X and region X is downstream of region Z). Regions of complementarity between the opposing strands are indicated by the short vertical lines. While not intended to indicate the precise location of the site(s) of cleavage, the area to which the site of cleavage within the probe oligonucleotide is shifted by the presence of the INVADER oligonucleotide in this embodiment is indicated by the solid vertical arrowhead. An alternative representation of the target\/INVADER\/probe cleavage structure is shown in . Neither diagram (i.e.,  or ) is intended to represent the actual mechanism of action or physical arrangement of the cleavage structure and further it is not intended that the method of the present invention be limited to any particular mechanism of action.","It can be considered that the binding of these oligonucleotides in this embodiment divides the target nucleic acid into three distinct regions: one region that has complementarity to only the probe (shown as \u201cZ\u201d); one region that has complementarity only to the INVADER oligonucleotide (shown as \u201cY\u201d); and one region that has complementarity to both oligonucleotides (shown as \u201cX\u201d). As discussed above, in some preferred embodiments of the present invention, the overlap may comprise moieties other than overlapping complementary bases. Thus, in some embodiments, the region shown as \u201cX\u201d can represent a region where there is a physical, but not sequence, overlap between the INVADER and probe oligonucleotides, i.e., in these latter embodiments, there is not a region of the target nucleic acid between regions \u201cZ\u201d and \u201cY\u201d that has complementarity to both oligonucleotides.","a) Oligonucleotide Design","Design of these oligonucleotides (i.e., the INVADER oligonucleotide and the probe) is accomplished using practices that are standard in the art. For example, sequences that have self complementarity, such that the resulting oligonucleotides would either fold upon themselves, or hybridize to each other at the expense of binding to the target nucleic acid, are generally avoided.","One consideration in choosing a length for these oligonucleotides is the complexity of the sample containing the target nucleic acid. For example, the human genome is approximately 3\u00d710basepairs in length. Any 10-nucleotide sequence will appear with a frequency of 1:4, or 1:1048,576 in a random string of nucleotides, which would be approximately 2,861 times in 3 billion basepairs. Clearly, an oligonucleotide of this length would have a poor chance of binding uniquely to a 10 nucleotide region within a target having a sequence the size of the human genome. If the target sequence were within a 3 kb plasmid, however, such an oligonucleotide might have a very reasonable chance of binding uniquely. By this same calculation it can be seen that an oligonucleotide of 16 nucleotides (i.e., a 16-mer) is the minimum length of a sequence that is mathematically likely to appear once in 3\u00d710basepairs. This level of specificity may also be provided by two or more shorter oligonucleotides if they are configured to bind in a cooperative fashion (i.e., such that they can produce the intended complex only if both or all are bound to their intended target sequences), wherein the combination of the short oligonucleotides provides the desired specificity. In one such embodiment,.the cooperativity between the shorter oligonucleotides is by a coaxial stacking effect that can occur when the oligonucleotides hybridize to adjacent sites on a target nucleic acid. In another embodiment, the shorter oligonucleotides are connected to one another, either directly, or by one or more spacer regions. The short oligonucleotides thus connected may bind to distal regions of the target and may be used to bridge across regions of secondary structure in a target. Examples of such bridging oligonucleotides are described in PCT Publication WO 98\/50403, herein incorporated by reference in its entirety.","A second consideration in choosing oligonucleotide length is the temperature range in which the oligonucleotides will be expected to function. A 16-mer of average base content (50% G-C bases) will have a calculated Tof about 41\u00b0 C., depending on, among other things, the concentration of the oligonucleotide and its target, the salt content of the reaction and the precise order of the nucleotides. As a practical matter, longer oligonucleotides are usually chosen to enhance the specificity of hybridization. Oligonucleotides 20 to 25 nucleotides in length are often used, as they are highly likely to be specific if used in reactions conducted at temperatures which are near their Ts (within about 5\u00b0 C. of the T). In addition, with calculated Ts in the range of 50 to 70\u00b0 C., such oligonucleotides (i.e., 20 to 25-mers) are appropriately used in reactions catalyzed by thermostable enzymes, which often display optimal activity near this temperature range.","The maximum length of the oligonucleotide chosen is also based on the desired specificity. One must avoid choosing sequences that are so long that they are either at a high risk of binding stably to partial complements, or that they cannot easily be dislodged when desired (e.g., failure to disassociate from the target once cleavage has occurred or failure to disassociate at a reaction temperature suitable for the enzymes and other materials in the reaction).","The first step of design and selection of the oligonucleotides for the INVADER oligonucleotide-directed cleavage is in accordance with these sample general principles. Considered as sequence-specific probes individually, each oligonucleotide may be selected according to the guidelines listed above. That is to say, each oligonucleotide will generally be long enough to be reasonably expected to hybridize only to the intended target sequence within a complex sample, usually in the 20 to 40 nucleotide range. Alternatively, because the INVADER oligonucleotide-directed cleavage assay depends upon the concerted action of these oligonucleotides, the composite length of the 2 oligonucleotides which span\/bind to the X, Y, Z regions may be selected to fall within this range, with each of the individual oligonucleotides being in approximately the 13 to 17 nucleotide range. Such a design might be employed if a non-thermostable cleavage means were employed in the reaction, requiring the reactions to be conducted at a lower temperature than that used when thermostable cleavage means are employed. In some embodiments, it may be desirable to have these oligonucleotides bind multiple times within a single target nucleic acid (e.g., to bind to multiple variants or multiple similar sequences within a target). It is not intended that the method of the present invention be limited to any particular size of the probe or INVADER oligonucleotide.","The second step of designing an oligonucleotide pair for this assay is to choose the degree to which the upstream \u201cINVADER\u201d oligonucleotide sequence will overlap into the downstream \u201cprobe\u201d oligonucleotide sequence, and consequently, the sizes into which the probe will be cleaved. A key feature of this assay is that the probe oligonucleotide can be made to \u201cturn over,\u201d that is to say probe can be made to depart to allow the binding and cleavage of other copies of the probe molecule, without the requirements of thermal denaturation or displacement by polymerization. While in one embodiment of this assay probe turnover may be facilitated by an exonucleolytic digestion by the cleavage agent, it is central to the present invention that the turnover does not require this exonucleolytic activity. For example, in some embodiments, a reaction temperature and reaction conditions are selected so as to create an equilibrium wherein the probe hybridizes and disassociates from the target. In other embodiments, temperature and reaction conditions are selected so that unbound probe can initiate binding to the target strand and physically displace bound probe. In still other embodiments, temperature and reaction conditions are selected such that either or both mechanisms of probe replacement may occur in any proportion. The method of the present invention is not limited to any particular mechanism of probe replacement. By any mechanism, when the probe is bound to the target to form a cleavage structure, cleavage can occur. The continuous cycling of the probe on and off of the target allows multiple probes to bind and be cleaved for each copy of a target nucleic acid.","i) Choosing The Amount Of Sequence Overlap","One way of accomplishing such turnover, where the INVADER oligonucleotide and probe oligonucleotide share a region of complementarity, can be envisioned by considering the diagram in FIG. . It can be seen that the Tof each oligonucleotide will be a function of the full length of that oligonucleotide: i.e., the Tof the INVADER oligonucleotide=T(Y+X), and the Tof the probe=T(X+Y) for the probe. When the probe is cleaved the X region is released, leaving the Z section. If the Tof Z is less than the reaction temperature, and the reaction temperature is less than the T(X+Z), then cleavage of the probe will lead to the departure of Z, thus allowing a new (X+Z) to hybridize. It can be seen from this example that the X region must be sufficiently long that the release of X will drop the Tof the remaining probe section below the reaction temperature: a G-C rich X section may be much shorter than an A-T rich X section and still accomplish this stability shift.","In other embodiments described herein, probe turn over is not related to a change in Tcaused by cleavage of the probe, but rather is related to the association and disassociation behavior of the probe in the selected conditions, regardless of cleavage. Thus, it is not intended that the present invention be limited to the use of probes that, upon cleavage, yield products having a Ts below the reaction temperature, as described above.","ii) Non-sequence Overlaps","It has been determined that the relationship between the 3\u2032 end of the upstream oligonucleotide and the desired site of cleavage on the probe should be carefully designed. It is known that the preferred site of cleavage for the types of structure-specific endonucleases employed herein is one basepair into a duplex (Lyamichev et al., supra). It was previously believed that the presence of an upstream oligonucleotide or primer allowed the cleavage site to be shifted away from this preferred site, into the single stranded region of the 5\u2032 arm (Lyamichev et al., supra and U.S. Pat. No. 5,422,253). In contrast to this previously proposed mechanism, and while not limiting the present invention to any particular mechanism, it is believed that the nucleotide immediately 5\u2032, or upstream of the cleavage site on the probe (including miniprobe and mid-range probes) should be able to basepair with the target for efficient cleavage to occur. In the case of the present invention, this would be the nucleotide in the probe sequence immediately upstream of the intended cleavage site. In addition, as described herein, it has been observed that in order to direct cleavage to that same site in the probe, the upstream oligonucleotide should have its 3\u2032 base (i.e., nt) immediately upstream of the intended cleavage site of the probe. In embodiments where the INVADER and probe oligonucleotides share a sequence overlap, this places the 3\u2032 terminal nucleotide of the upstream oligonucleotide and the base of the probe oligonucleotide 5\u2032 of the cleavage site in competition for pairing with the corresponding nucleotide of the target strand.","To examine the outcome of this competition (i.e. which base is paired during a successful cleavage event), substitutions were made in the probe and INVADER oligonucleotides such that either the probe or the INVADER oligonucleotide were mismatched with the target sequence at this position. The effects of both arrangements on the rates of cleavage were examined. When the INVADER oligonucleotide is unpaired at the 3\u2032 end, the rate of cleavage was not reduced. If this base was removed, however, the cleavage site was shifted upstream of the intended site. In contrast, if the probe oligonucleotide was not base-paired to the target just upstream of the site to which the INVADER oligonucleotide was directing cleavage, the rate of cleavage was dramatically reduced, suggesting that when a competition exists, the probe oligonucleotide was the molecule to be base-paired in this position. It appears that the 3\u2032 end of the upstream INVADER oligonucleotide is unpaired during cleavage, and yet is important for accurate positioning of the cleavage. To examine which part(s) of the 3\u2032 terminal nucleotide are required for the positioning of cleavage, INVADER oligonucleotides were designed that terminated on this end with nucleotides that were altered in a variety of ways. Sugars examined included 2\u2032 dcoxyribose with a 3\u2032 phosphate group, a dideoxyribose, 3\u2032 deoxyribose, 2\u2032 O-methyl ribose, arabinose and arabinose with a 3\u2032 phosphate. Abasic ribose, with and without 3\u2032 phosphate were tested. Synthetic \u201cuniversal\u201d bases such at 3-nitropyrrole and 5-3 nitroindole on ribose sugars were tested. Finally, a base-like aromatic ring structure, acridine, linked to the 3\u2032 end the previous nucleotide without a sugar group was tested. The results obtained support the conclusion that the aromatic ring of the base (at the 3\u2032 end of the INVADER oligonuceotide) is an important moiety for accomplishing the direction of cleavage to the desired site within the downstream probe. The 3\u2032 terminal moiety of the INVADER oligonucleotide need not be a base that is complementary to the target nucleic acid.","iii) Miniprobes And Mid-Range Probes;","As discussed above, the INVADER oligonucleotide-directed cleavage assay may be performed using INVADER and probe oligonucleotides that have a length of about 13-25 nucleotides (typically 20-25 nucleotides). It is also contemplated that the oligonucleotides may themselves be composed of shorter oligonucleotide sequences that align along a target strand but that are not covalently linked. This is to say that there is a nick in the sugar-phosphate backbone of the composite oligonucleotide, but that there is no disruption in the progression of base-paired nucleotides in the resulting duplex. When short strands of nucleic acid align contiguously along a longer strand the hybridization of each is stabilized by the hybridization of the neighboring fragments because the basepairs can stack along the helix as though the backbone was in fact uninterrupted. This cooperativity of binding can give each segment a stability of interaction in excess of what would be expected for the segment hybridizing to the longer nucleic acid alone. One application of this observation has been to assemble primers for DNA sequencing, typically about 18 nucleotides long, from sets of three hexamer oligonucleotides that are designed to hybridize in this way (Kotler et al. Proc. Natl. Acad. Sci. USA 90:4241 [1993]). The resulting doubly-nicked primer can be extended enzymatically in reactions performed at temperatures that might be expected to disrupt the hybridization of hexamers, but not of 18-mers.","The use of composite or split oligonucleotides is applied with success in the INVADER-directed cleavage assay. For example, the probe oligonucleotide may be split into two oligonucleotides that anneal in a contiguous and adjacent manner along a target oligonucleotide as diagrammed in FIG. . In this Figure, the downstream oligonucleotide (analogous to the probe of ) is assembled from two smaller pieces: a short segment of 6-10 nts (termed the \u201cminiprobe\u201d), that is to be cleaved in the course of the detection reaction, and an oligonucleotide that hybridizes immediately downstream of the miniprobe (termed the \u201cstacker\u201d), that serves to stabilize the hybridization of the probe. To form the cleavage structure, an upstream oligonucleotide (the INVADER oligonucleotide) is provided to direct the cleavage activity to the desired region of the miniprobe. Assembly of the probe from non-linked pieces of nucleic acid (i.e., the miniprobe and the stacker) allows regions of sequences to be changed without requiring the re-synthesis of the entire proven sequence, thus improving the cost and flexibility of the detection system. In addition, the use of unlinked composite oligonucleotides makes the system more stringent in its requirement of perfectly matched hybridization to achieve signal generation, allowing this to be used as a sensitive means of detecting mutations or changes in the target nucleic acid sequences.","As illustrated in , in one embodiment, the methods of the present invention employ at least three oligonucleotidcs that interact with a target nucleic acid to form a cleavage structure for a structure-specific nuclease. More specifically, the cleavage structure comprises i) a target nucleic acid that may be either single-stranded or double-stranded (when a double-stranded target nucleic acid is employed, it may be rendered single-stranded, e.g., by heating); ii) a first oligonucleotide, termed the \u201cstacker,\u201d that defines a first region of the target nucleic acid sequence by being the complement of that region (region W of the target as shown in FIG. ); iii) a second oligonucleotide, termed the \u201cminiprobe,\u201d that defines a second region of the target nucleic acid sequence by being the complement of that region (regions X and Z of the target as shown in FIG. ); iv) a third oligonucleotide, termed the \u201cINVADER,\u201d the 5\u2032 part of which defines a third region of the same target nucleic acid sequence (regions Y and X in FIG. ), adjacent to and downstream of the second target region (regions X and Z), and the second or 3\u2032 part of which overlaps into the region defined by the second oligonucleotide (region X depicts the region of overlap). The resulting structure is diagrammed in FIG. . As described above for embodiments that do not employ a stacker, the region shown as \u201cX\u201d can represent a region where there is a physical, but not sequence, overlap between the INVADER and probe oligonucleotides.","While not limiting the invention or the instant discussion to any particular mechanism of action, the diagram in  represents the effect on the site of cleavage caused by this type of arrangement of three oligonucleotides. The design of these three oligonucleotides is described below in detail. In , the 3\u2032 ends of the nucleic acids (i.e., the target and the oligonucleotides) are indicated by the use of the arrowheads on the ends of the lines depicting the strands of the nucleic acids (and where space permits, these ends are also labeled \u201c3\u2032\u201d). It is readily appreciated that the three oligonucleotides (the INVADER, the miniprobe and the stacker) are arranged in a parallel orientation relative to one another, while the target nucleic acid strand is arranged in an anti-parallel orientation relative to the three oligonucleotides. Further it is clear that the INVADER oligonucleotide is located upstream of the miniprobe oligonucleotide and that the miniprobe oligonucleotide is located upstream of the stacker oligonucleotide and that with respect to the target nucleic acid strand, region W is upstream of region Z, region Z is upstream of upstream of region X and region X is upstream of region Y (that is region Y is downstream of region X, region X is downstream of region Z and region Z is downstream of region W). Regions of complementarity between the opposing strands are indicated by the short vertical lines. While not intended to indicate the precise location of the site(s) of cleavage, the area to which the site of cleavage within the miniprobe oligonucleotide is shifted by the presence of the INVADER oligonucleotide is indicated by the solid vertical arrowhead.  is not intended to represent the actual mechanism of action or physical arrangement of the cleavage structure and further it is not intended that the method of the present invention be limited to any particular mechanism of action.","It can be considered that the binding of these oligonucleotides divides the target nucleic acid into four distinct regions: one region that has complementarity to only the stacker (shown as \u201cW\u201d); one region that has complementarity to only the miniprobe (shown as \u201cZ\u201d); one region that has complementarity only to the INVADER oligonucleotide (shown as \u201cY\u201d); and one region that has complementarity to both the INVADER and miniprobe oligonucleotides (shown as \u201cX\u201d). As discussed above, the INVADER oligonucleotide may also be employed such that a physical overlap rather than a sequence overlap with the probe is provided.","In addition to the benefits cited above, the use of a composite design for the oligonucleotides that form the cleavage structure allows more latitude in the design of the reaction conditions for performing the INVADER-directed cleavage assay. When a longer probe (e.g., 16-25 nt), as described above, is used for detection in reactions that are performed at temperatures below the Tof that probe, the cleavage of the probe may play a significant role in destabilizing the duplex of which it is a part, thus allowing turnover and reuse of the recognition site on the target nucleic acid. In contrast, reaction temperatures that are at or above the Tof the probe mean that the probe molecules are hybridizing and releasing from the target quite rapidly even without cleavage of the probe. When an upstream INVADER oligonucleotide and a cleavage means are provided the probe will be specifically cleaved, but the cleavage will not be necessary to the turnover of the probe. When a long probe (e.g., 16-25 nt) is used in this way the temperatures required to achieve this state is high, around 65 to 70\u00b0 C. for a 25-mer of average base composition. Requiring the use of such elevated temperatures limits the choice of cleavage agents to those that are very thermostable, and may contribute to background in the reactions, depending of the means of detection, through thermal degradation of the probe oligonucleotides. With miniprobes, this latter mechanism of probe replacement may be accomplished at a lower temperature. Thus, shorter probes are preferred for embodiments using lower reaction temperatures.","The miniprobe of the present invention may vary in size depending on the desired application. In one embodiment, the probe may be relatively short compared to a standard probe (e.g., 16-25 nt), in the range of 6 to 10 nucleotides. When such a short robe is used, reaction conditions can be chosen that prevent hybridization of the miniprobe in the absence of the stacker oligonuclcotide. In this way a short probe can be made to assume the statistical specificity and selectivity of a longer sequence. In the event of a perturbation in the cooperative binding of the miniprobe and stacker nucleic acids, as might be caused by a mismatch within the short sequence that is otherwise complementary to the target nucleic acid or at the junction between the contiguous duplexes, this cooperativity can be lost, dramatically reducing the stability of the shorter duplex (i.e., that of the miniprobe), and thus reducing the level of cleaved product in the assay of the present invention.","It is also contemplated that probes of intermediate size may be used. Such probes, in the 11 to 15 nucleotide range, may blend some of the features associated with the longer probes as originally described, these features including the ability to hybridize and be cleaved absent the help of a stacker oligonucleotide. At temperatures below the expected Tof such probes, the mechanisms of turnover may be as discussed above for probes in the 20 nt range, and be dependent on the removal of the sequence in the \u2018X\u2019 region for destabilization and cycling.","The mid-range probes may also be used at elevated temperatures, at or above their expected T, to allow melting rather than cleavage to promote probe turnover. In contrast to the longer probes described above, however, the temperatures required to allow the use of such a thermally driven turnover are much lower (about 40 to 60\u00b0 C.), thus preserving both the cleavage means and the nucleic acids in the reaction from thermal degradation. In this way, the mid-range probes may perform in some instances like the miniprobes described above. In a further similarity to the miniprobes, the accumulation of cleavage signal from a mid-range probe may be helped under some reaction conditions by the presence of a stacker.","To summarize, a standard long probe usually does not benefit from the presence of a stacker oligonucleotide downstream (the exception being cases where such an oligonucleotide may also disrupt structures in the target nucleic acid that interfere with the probe binding), and it may be used in conditions requiring several nucleotides to be removed to allow the oligonucleotide to release from the target efficiently. If temperature of the reaction is used to drive exchange of the probes, standard probes may require use of a temperature at which nucleic acids and enzymes are at higher risk of thermal degradation.","The miniprobe is very short and performs optimally in the presence of a downstream stacker oligonucleotide. The miniprobes are well suited to reactions conditions that use the temperature of the reaction to drive rapid exchange of the probes on the target regardless of whether any bases have been cleaved. In reactions with sufficient amount of the cleavage means, the probes that do bind will be rapidly cleaved before they melt off.","The mid-range or midiprobe combines features of these probes and can be used in reactions like those favored by long probes, with longer regions of overlap (\u201cX\u201d regions) to drive probe turnover at lower temperature. In a preferred embodiment, the midrange probes are used at temperatures sufficiently high that the probes are hybridizing to the target and releasing rapidly regardless of cleavage. The mid-range probe may have enhanced performance in the presence of a stacker under some circumstances.","The distinctions between the mini-, midi- (i.e., mid-range) and long probes are not contemplated to be inflexible and based only on length. The performance of any given probe may vary with its specific sequence, the choice of solution conditions, the choice of temperature and the selected cleavage means.","It is shown in Example 17 that the assemblage of oligonucleotides that comprises the cleavage structure of the present invention is sensitive to mismatches between the probe and the target. The site of the mismatch used in Ex. 17 provides one example and is not intended to be a limitation in location of a mismatch affecting cleavage. It is also contemplated that a mismatch between the INVADER oligonucleotide and the target may be used to distinguish related target sequences. In the 3-oligonucleotide system, comprising an INVADER, a probe and a stacker oligonucleotide, it is contemplated that mismatches may be located within any of the regions of duplex formed between these oligonucleotides and the target sequence. In a preferred embodiment, a mismatch to be detected is located in the probe. In a particularly preferred embodiment, the mismatch is in the probe, at the basepair immediately upstream (i.e., 5\u2032) of the site that is cleaved when the probe is not mismatched to the target.","In another preferred embodiment, a mismatch to be detected is located within the region \u2018Z\u2019 defined by the hybridization of a miniprobe. In a particularly preferred embodiment, the mismatch is in the miniprobe, at the basepair immediately upstream (i.e., 5\u2032) of the site that is cleaved when the miniprobe is not mismatched to the target.","b) Design of the Reaction Conditions","Target nucleic acids that may be analyzed using the methods of the present invention that employ a 5\u2032 nuclease or other appropriate cleavage agents include of both RNA and DNA. Such nucleic acids may be obtained using standard molecular biological techniques. For example, nucleic acids (RNA or DNA) may be isolated from a tissue sample (e.g., a biopsy specimen), tissue culture cells, samples containing bacteria and\/or viruses (including cultures of bacteria and\/or viruses), etc. The target nucleic acid may also be transcribed in vitro from a DNA template or may be chemically synthesized or amplified in by polymerase chain reaction. Furthermore, nucleic acids may be isolated from an organism, either as genomic material or as a plasmid or similar extrachromosomal DNA, or they may be a fragment of such material generated by treatment with a restriction endonuclease or other cleavage agent, or a shearing force, or it may be synthetic.","Assembly of the target, probe, and INVADER oligonucleotide nucleic acids into the cleavage reaction of the present invention uses principles commonly used in the design of oligonucleotide-based enzymatic assays, such as dideoxynucleotide sequencing and polymerase chain reaction (PCR). As is done in these assays, the oligonucleotides are provided in sufficient excess that the rate of hybridization to the target nucleic acid is very rapid. These assays are commonly performed with 50 fmoles to 2 pmoles of each oligonucleotide per microliter of reaction mixture, although they are not necessarily limited to this range In the Examples described herein, amounts of oligonucleotides ranging from 250 fmoles to 5 pmoles per microliter of reaction volume were used. These values were chosen for the purpose of ease in demonstration and are not intended to limit the performance of the present invention to these concentrations. Other (e.g., lower) oligonucleotide concentrations commonly used in other molecular biological reactions are also contemplated.","It is desirable that an INVADER oligonucleotide be immediately available to direct the cleavage of each probe oligonucleotide that hybridizes to a target nucleic acid. In some embodiments described herein, the INVADER oligonucleotide is provided in excess over the probe oligonucleotide. While this is an effective means of making the INVADER oligonucleotide immediately available in such embodiments it is not intended that the practice of the present invention be limited to conditions wherein the INVADER oligonucleotide is in excess over the probe, or to any particular ratio of INVADER-to-probe (e.g., in some preferred embodiments described herein, the probe is provided in excess over the INVADER oligonucleotide). Another means of assuring the presence of an INVADER oligonucleotide whenever a probe binds to a target nucleic acid is to design the INVADER oligonucleotide to hybridize more stably to the target, i.e., to have a higher Tthan the probe. This can be accomplished by any of the means of increasing nucleic acid duplex stability discussed herein (e.g., by increasing the amount of complementarity to the target nucleic acid).","Buffer conditions should be chosen that will be compatible with both the oligonucleotide\/target hybridization and with the activity of the cleavage agent. The optimal buffer conditions for nucleic acid modification enzymes, and particularly DNA modification enzymes, generally included enough mono- and di-valent salts to allow association of nucleic acid strands by base-pairing. If the method of the present invention is performed using an enzymatic cleavage agent other than those specifically described here, the reactions may generally be performed in any such buffer reported to be optimal for the nuclease function of the cleavage agent. In general, to test the utility of any cleavage agent in this method, test reactions are performed wherein the cleavage agent of interest is tested in the MOPS\/MnCl\/KCl buffer or Mg-containing buffers described herein and in whatever buffer has been reported to be suitable for use with that agent, in a manufacturer's data sheet, a journal article, or in personal communication.","The products of the INVADER oligonucleotide-directed cleavage reaction are fragments generated by structure-specific cleavage of the input oligonucleotides. The resulting cleaved and\/or uncleaved oligonucleotides may be analyzed and resolved by a number of methods including, but not limited to, electrophoresis (on a variety of supports including acrylamide or agarose gels, paper, etc.), chromatography, fluorescence polarization, mass spectrometry and chip hybridization. In some Examples the invention is illustrated using electrophoretic separation for the analysis of the products of the cleavage reactions. However, it is noted that the resolution of the cleavage products is not limited to electrophoresis. Electrophoresis is chosen to illustrate the method of the invention because electrophoresis is widely practiced in the art and is easily accessible to the average practitioner. In other Examples, the invention is illustrated without electrophoresis or any other resolution of the cleavage products.","The probe and INVADER oligonucleotides may contain a label to aid in their detection following the cleavage reaction. The label may be a radioisotope (e.g., a P or S-labelled nucleotide) placed at either the 5\u2032 or 3\u2032 end of the oligonucleotide or alternatively, the label may be distributed throughout the oligonucleotide (i.e., a uniformly labeled oligonucleotide). The label may be a nonisotopic detectable moiety, such as a fluorophore, that can be detected directly, or a reactive group that permits specific recognition by a secondary agent. For example, biotinylated oligonucleotides may be detected by probing with a streptavidin molecule that is coupled to an indicator (e.g., alkaline phosphatase or a fluorophore) or a hapten such as dioxigenin may be detected using a specific antibody coupled to a similar indicator. The reactive group may also be a specific configuration or sequence of nucleotides that can bind or otherwise interact with a secondary agent, such as another nucleic acid, and enzyme, or an antibody.","c) Optimization of Reaction Conditions","The INVADER oligonucleotide-directed cleavage reaction is useful to detect the presence of specific nucleic acids. In addition to the considerations listed above for the selection and design of the INVADER and probe oligonucleotides, the conditions under which the reaction is to be performed may be optimized for detection of a specific target sequence.","One objective in optimizing the INVADER oligonucleotide-directed cleavage assay is to allow specific detection of the fewest copies of a target nucleic acid. To achieve this end, it is desirable that the combined elements of the reaction interact with the maximum efficiency, so that the rate of the reaction (e.g., the number of cleavage events per minute) is maximized. Elements contributing to the overall efficiency of the reaction include the rate of hybridization, the rate of cleavage, and the efficiency of the release of the cleaved probe.","The rate of cleavage will be a function of the cleavage means chosen, and may be made optimal according to the manufacturer's instructions when using commercial preparations of enzymes or as described in the examples herein. The other elements (rate of hybridization, efficiency of release) depend upon the execution of the reaction, and optimization of these elements is discussed below.","Three elements of the cleavage reaction that significantly affect the rate of nucleic acid hybridization are the concentration of the nucleic acids, the temperature at which the cleavage reaction is performed and the concentration of salts and\/or other charge- shielding ions in the reaction solution.","The concentrations at which oligonucleotide probes are used in assays of this type are well known in the art, and are discussed above. One example of a common approach to optimizing an oligonucleotide concentration is to choose a starting amount of oligonucleotide for pilot tests; 0.01 to 2 \u03bcM is a concentration range used in many oligonucleotide-based assays. When initial cleavage reactions are performed, the following questions may be asked of the data: Is the reaction performed in the absence of the target nucleic acid substantially free of the cleavage product?; Is the site of cleavage specifically positioned in accordance with the design of the INVADER oligonucleotide?; Is the specific cleavage product easily detected in the presence of the uncleaved probe (or is the amount of uncut material overwhelming the chosen visualization method)?","A negative answer to any of these questions would suggest that the probe concentration is too high, and that a set of reactions using serial dilutions of the probe should be performed until the appropriate amount is identified. Once identified for a given target nucleic acid in a give sample type (e.g., purified genomic DNA, body fluid extract, lysed bacterial extract), it should not need to be re-optimized. The sample type is important because the complexity of the material present may influence the probe concentration optimum.","Conversely, if the chosen initial probe concentration is too low, the reaction may be slow, due to inefficient hybridization. Tests with increasing quantities of the probe will identify the point at which the concentration exceeds the optimum (e.g., at which it produces an undesirable effect, such as background cleavage not dependent on the target sequence, or interference with detection of the cleaved products). Since the hybridization will be facilitated by excess of probe, it is desirable, but not required, that the reaction be performed using probe concentrations just below this point.","The concentration of INVADER oligonucleotide can be chosen based on the design considerations discussed above. In some embodiments, the INVADER oligonucleotide is in excess of the probe oligonucleotide. In a preferred embodiment, the probe oligonucleotide is in excess of the INVADER oligonucleotide.","Temperature is also an important factor in the hybridization of oligonucleotides. The range of temperature tested will depend in large part on the design of the oligonucleotides, as discussed above. Where it is desired to have a reaction be run at a particular temperature (e.g., because of an enzyme requirement, for convenience, for compatibility with assay or detection apparatuses, etc.), the oligonucleotides that function in the reaction can be designed to optimally perform at the desired reaction temperature. Each INVADER reaction includes at least two target sequence-specific oligonucleotides for the primary reaction: an upstream INVADER oligonucleotide and a downstream probe oligonucleotide. In some preferred embodiments, the INVADER oligonucleotide is designed to bind stabily at the reaction temperature, while the probe is designed to freely associate and disassociate with the target strand, with cleavage occurring only when an uncut probe hybridizes adjacent to an overlapping INVADER oligonucleotide. In preferred embodiments, the probe includes a 5\u2032 flap that is not complementary to the target, and this flap is released from the probe when cleavage occurs. The released flap can be detected directly or indirectly. In some preferred embodiments, as discussed in detail below, the released flap participate as in INVADER oligonucleotide in a secondary reaction.","Optimum conditions for the INVADER assay are generally those that allow specific detection of the smallest amount of a target nucleic acid. Such conditions may be characterized as those that yield the highest target-dependent signal in a given timeframe, or for a given amount of target nucleic acid, or that allow the highest rate of probe cleavage (i.e., probes cleaved per minute). To select a probe sequence that will perform optimally at a pre-selected reaction temperature, the melting temperature (T) of its analyte specific region (ASR, the region that is complementary to the target nucleic acid) is calculated using the nearest-neighbor model and published parameters for DNA duplex formation (SantaLucia, J., 95, 1460-5 (1998), Allawi, H. T. & SantaLucia, J., Jr. . 36, 10581-94 (1997). However, there are several differences between the conditions under which the published parameters were measured and the conditions under which the INVADER assay is run in preferred embodiments. The salt concentrations are often different than the solution conditions in which the nearest-neighbor parameters were obtained (1M NaCl and no divalent metals). One can compensate for this factor by varying the value provided for the salt concentration within the melting temperature calculations. In addition to the salt concentration, the presence of and concentration of the enzyme influences the optimal reaction temperature, and an additional adjustment should be made to the calculated Tto determine the optimal temperature at which to perform a reaction. By observing the optimal temperature for a number of INVADER reactions (i.e., the temperature at which the rate of signal accumulation is highest) it has been possible to further alter the value for salt concentration within these calculations to allow the algorithm for Tcalculation to be modified to instead provide an optimal cleavage reaction temperature for a given probe sequence. This additional adjustment is termed a \u201csalt correction\u201d. As used herein, the term \u201csalt correction\u201d refers to a variation made in the value provided for a salt concentration, for the purpose of reflecting the effect on a Tcalculation for a nucleic acid duplex of a non-salt parameter or condition affecting said duplex. Variation of the values provided for the strand concentrations will also affect the outcome of these calculations. By using a value of 0.5 M NaCl [SantaLucia, J., 95, 1460-5 (1998)] and strand concentrations of about 1 \u03bcM of the probe and 1 fM target, the algorithm used for calculating probe-target melting temperature has been adapted for use in predicting optimal INVADER assay reaction temperature. For a set of about 30 probes, the average deviation between optimal assay temperatures calculated by this method and those experimentally determined was about 1.5\u00b0 C.","As noted above, the concentration of the cleavage agent can affect the actual optimum temperature for a cleavage reaction. Additionally, different cleavage agents, even if used at identical concentrations, can affect reaction temperature optima differently (e.g., the difference between the calculated probe Tand the observed optimal reaction temperature may be greater for one enzyme than for another). Determination of appropriate salt corrections for reactions using different enzymes or concentrations of enzymes, or for any other variation made in reaction conditions, involves a two step process of a) measuring reaction temperature optima under the new reaction conditions, and varying the salt concentration within the Talgorithm to produce a calculated temperature matching or closely approximating the observed optima. Measurement of an optimum reaction temperature generally involves performing reactions at a range of temperatures selected such that the range allows observation of an increase in performance as an optimal temperature is approached (either by increasing or decreasing temperatures), and a decrease in performance when an optimal temperature has been passed, thereby allowing identification of the optimal temperature or temperature range [see, for example, V. I. Lyamichev, et al., Biochemistry 39, No. 31: 9523-9532 (2000)].","The length of the downstream probe analyte-specific region (ASR) is defined by the temperature selected for running the reaction, e.g., 63\u00b0 C. in the experiments described in Examples 54 through 60. To select a probe sequence based on a desired reaction temperature, the probe sequence is selected in the following way (as illustrated for the design of a probe for the detection of a sequence difference at a particular location). Starting from the position of the variant nucleotide on the target DNA (position N, FIG. ); the target base that is paired to the probe nucleotide 5\u2032 of the intended cleavage site), an iterative procedure is used by which the length of the ASR is increased by one base pair until a calculated optimal reaction temperature (Tplus salt correction to compensate for enzyme and any other reaction conditions effects) matching the desired reaction temperature is reached. The non-complementary arm of the probe is preferably selected (by a similar iterative process) to allow the secondary reaction to cycle at the same reaction temperature, and the entire probe design (ASR and 5\u2032 noncomplementary arm) is screened using programs such as mfold [Zuker, M. 244, 48-52 (1989)] or Oligo 5.0 [Rychlik, W. & Rhoads, R. E. 17, 8543-51 (1989)] for the possible formation of dimer complexes or secondary structures that could interfere with the reaction. The same principles are also followed for INVADER oligonucleotide design. The following describes design of an INVADER assay embodiment wherein the 3\u2032 end of the INVADER oligonucleotide, at position N on the target DNA, is designed to have a nucleotide not complementary to either allele suspected of being contained in the sample to be tested. The mismatch does not adversely affect cleavage [Lyarnichev, V. et al. 17, 292-296 (1999)], and it can enhance probe cycling, presumably by minimizing coaxial stabilization effects between the two probes. Briefly, starting from the position N, additional residues complementary to the target DNA starting from residue N-I are then added in the upstream direction until the stability of the INVADER-target hybrid exceeds that of the probe (and therefore the planned assay reaction temperature). In preferred embodiments, the stability of the INVADER-target hybrid exceeds that of the probe by 15-20\u00b0 C.","In some embodiments, where the released cleavage fragment from a primary reaction is to be used in a secondary reaction, one should also consider the reaction conditions of the secondary reaction in designing the oligonucleotides for the primary reaction (e.g., the sequence of the released non-complementary 5\u2032 flap of the probe in the primary reaction can be designed to optimally function in a secondary reaction). For example, as described in detail below, in some embodiments, a secondary reaction is used where the released cleavage fragment from a primary reaction hybridizes to a synthetic cassette to form a secondary cleavage reaction. In some preferred embodiments, the cassette comprises a fluorescing moiety and a quenching moiety, wherein cleavage of the secondary cleavage structure separates the fluorescing moiety from the quenching moiety, resulting in a detectable signal (e.g., FRET detection). The secondary reaction can be configured a number of different ways. For example, in some embodiments, the synthetic cassette comprises two oligonucleotides: an oligonucleotide that contains the FRET moieties and a FRET\/INVADER oligonucleotide bridging oligonucleotide that allows the INVADER oligonucleotide (i.e., the released flap from the primary reaction) and the FRET oligonucleotide to hybridize thereto, such that a cleavage structure is formed. In some embodiments, the synthetic cassette is provided as a single oligonucleotide, comprising a hairpin structure (i.e., the FRET oligonucleotide is connected at its 3\u2032 end to the bridging oligonucleotide by a loop). The loop may be nucleic acid, (e.g., a string of nucleotides, such as the four T residues depicted in several Figures, including A) or a non-nucleic acid spacer or linker. The linked molecules may together be described as a FRET cassette. In the secondary reaction using a FRET cassette the released flap from the primary reaction, which acts as an INVADER oligonucleotide, should be able to associate and disassociate with the FRET cassette freely, so that one released flap can direct the cleavage of multiple FRET cassettes. It is one aspect of the assay design that all of the probe sequences may be selected to allow the primary and secondary reactions to occur at the same optimal temperature, so that the reaction steps can run simultaneously. In an alternative embodiment, the probes may be designed to operate at different optimal temperatures, so that the reactions steps are not simultaneously at their temperature optima. As noted above, the same iterative process used to select the ASR of the probe can be used in the design of the portion of the primary probe that participates in a secondary reaction.","Another determinant of hybridization efficiency is the salt concentration of the reaction. In large part, the choice of solution conditions will depend on the requirements of the cleavage agent, and for reagents obtained commercially, the manufacturer's instructions are a resource for this information. When developing an assay utilizing any particular cleavage agent, the oligonucleotide and temperature optimizations described above should be performed in the buffer conditions best suited to that cleavage agent.","A \u201cno enzyme\u201d control allows the assessment of the stability of the labeled oligonucleotides under particular reaction conditions, or in the presence of the sample to be tested e.g., in assessing the sample for contaminating nucleases). In this manner, the substrate and oligonucleotides are placed in a tube containing all reaction components, except the enzyme and treated the same as the enzyme-containing reactions. Other controls may also be included. For example, a reaction with all of the components except the target nucleic acid will serve to confirm the dependence of the cleavage on the presence of the target sequence.","d) Selection of a Cleavage Agent","As demonstrated in a number of the Examples, some 5\u2032 nucleases do not require an upstream oligonucleotide to be active in a cleavage reaction. Although cleavage may be slower without the upstream oligonuclcotide, it may still occur (Lyamichev et al., Science 260:778 [1993], Kaiser et al., J. Biol. Chem., 274:21387 [1999]). When a DNA strand is the template or target strand to which probe oligonucleotides are hybridized, the 5\u2032 nucleases derived from DNA polymerases and some flap endonucleases (FENs), such as that from Methanococcus jannaschii, can cleave quite well without an upstream oligonucleotide providing an overlap (Lyamichev et al., Science 260:778 [1993], Kaiser et al., J. Biol. Chem., 274:21387 [1999], and U.S. Pat. No. 5,843,669, herein incorporated by reference in its entirety). These nucleases may be selected for use in some embodiments of the INVADER assay, e.g., in embodiments wherein cleavage of th probe in the absence of an INVADER oligonucleotide gives a different cleavage product, which does not interfere with the intended analysis, or wherein both types of cleavage, INVADER oligonucleotide-directed and INVADER oligonucleotide-independent, are intended to occur.","In other embodiments it is preferred that cleavage of the probe be dependent on the presence of an upstream INVADER oligonucleotide, and enzyme having this requirement would be used. Other FENs, such as those from () and (), cleave an overlapped structure on a DNA target at so much greater a rate than they do a non-overlapping structure (i.e., either missing the upstream oligonucleotide or having a non-overlapping upstream oligonucleotide) that they can be viewed as having an essentially absolute requirement for the overlap (Lyamichev et al., Nat. Biotechnol., 17:292 [1999], Kaiser et al., J. Biol. Chem., 274:21387 [1999]). When an RNA target is hybridized to DNA oligonucleotide probes to form a cleavage structure, many FENs cleave the downstream DNA probe poorly, regardless of the presence of an overlap. On such an RNA-containing structure, the 5\u2032 nucleases derived from DNA polymerases have a strong requirement for the overlap, and are essentially inactive in its absence.","e) Probing For Multiple Alleles","The INVADER oligonuclcotide-directed cleavage reaction is also useful in the detection and quantification of individual variants or alleles in a mixed sample population. By way of example, such a need exists in the analysis of tumor material for mutations in genes associated with cancers. Biopsy material from a tumor can have a significant complement of normal cells, so it is desirable to detect mutations even when present in fewer than 5% of the copies of the target nucleic acid in a sample. In this case, it is also desirable to measure what fraction of the population carries the mutation. Similar analyses may also be done to examine allelic variation in other gene systems, and it is not intended that the method of the present invention by limited to the analysis of tumors.","As demonstrated below, in one embodiment, reactions can be performed under conditions that prevent the cleavage of probes bearing even a single-nucleotide difference mismatch within the region of the target nucleic acid termed \u201cZ\u201d in , but that permit cleavage of a similar probe that is completely complementary to the target in this region. In a preferred embodiment, a mismatch is positioned at the nucleotide in the probe that is 5\u2032 of the site where cleavage occurs in the absence of the mismatch.","In other embodiments, the INVADER assay may be performed under conditions that have a tight requirement for an overlap (e.g., using the Afu FEN for DNA target detection or the 5\u2032 nuclease of DNA polymerase for RNA target detection, as described above), providing an alternative means of detecting single nucleotide or other sequence variations. In one embodiment, the probe is selected such that the target base suspected of varying is positioned at the 5\u2032 end of the target-complementary region of this probe. The upstream INVADER oligonucleotide is positioned to provide a single base of overlap. If the target and the probe oligonucleotide are complementary at the base in question, the overlap forms and cleavage can occur. This embodiment is diagrammed in FIG. . However, if the target does not complement the probe at this position, that base in the probe becomes part of a non-complementary 5\u2032 arm, no overlap between the INVADER oligonucleotide and probe oligonucleotide exists, and cleavage is suppressed.","It is also contemplated that different sequences may be detected in a single reaction. Probes specific for the different sequences may be differently labeled. For example, the probes may have different dyes or other detectable moieties, different lengths, or they may have differences in net charges of the products after cleavage. When differently labeled in one of these ways, the contribution of each specific target sequence to final product can be tallied. This has application in detecting the quantities of different versions of a gene within a mixture. Different genes in a mixture to be detected and quantified may be wild type and mutant genes (e.g., as may be found in a tumor sample, such as a biopsy). In this embodiment, one might design the probes to precisely the same site, but one to match the wild-type sequence and one to match the mutant. Quantitative detection of the products of cleavage from a reaction performed for a set amount of time will reveal the ratio of the two genes in the mixture. Such analysis may also be performed on unrelated genes in a mixture. This type of analysis is not intended to be limited to two genes. Many variants within a mixture may be similarly measured.","Alternatively, different sites on a single gene may be monitored and quantified to verify the measurement of that gene. In this embodiment, the signal from each probe would be expected to be the same.","It is also contemplated that multiple probes may be used that are not differently labeled, such that the aggregate signal is measured. This may be desirable when using many probes designed to detect a single gene to boost the signal from that gene. This configuration may also be used for detecting unrelated sequences within a mix. For example, in blood banking it is desirable to know if any one of a host of infectious agents is present in a sample of blood. Because the blood is discarded regardless of which agent is present, different signals on the probes would not be required in such an application of the present invention, and may actually be undesirable for reasons of confidentiality.","Just as described for the two-oligonucleotide system, above, the specificity of the detection reaction will be influenced by the aggregate length of the target nucleic acid sequences involved in the hybridization of the complete set of the detection oligonucleotides. For example, there may be applications in which it is desirable to detect a single region within a complex genome. In such a case the set of oligonucleotides may be chosen to require accurate recognition by hybridization of a longer segment of a target nucleic acid, often in the range of 20 to 40 nucleotides. In other instances it may be desirable to have the set of oligonucleotides interact with multiple sites within a target sample. In these cases one approach would be to use a set of oligonucleotides that recognize a smaller, and thus statistically more common, segment of target nucleic acid sequence.","In one preferred embodiment, the INVADER and stacker oligonucleotides may be designed to be maximally stable, so that they will remain bound to the target sequence for extended periods during the reaction. This may be accomplished through any one of a number of measures well known to those skilled in the art, such as adding extra hybridizing sequences to the length of the oligonucleotide (up to about 50 nts in total length), or by using residues with reduced negative charge, such as phosphorothioates or peptide-nucleic acid residues, so that the complementary strands do not repel each other to degree that natural strands do. Such modifications may also serve to make these flanking oligonucleotides resistant to contaminating nucleases, thus further ensuring their continued presence on the target strand during the course of the reaction. In addition, the INVADER and stacker oligonucleotides may be covalently attached to the target (e.g., through the use of psoralen cross-linking).","As described above, and demonstrated in Example 36, the concentration of the probe that is cleaved can be used to increase the rate of signal accumulation, with higher concentrations of probe yielding higher final signal. However, the presence of large amounts of residual uncleaved probe can present problems for subsequent use of the cleaved products for detection or for further amplification. If the subsequent step is a simple detection (e.g., by gel resolution), the excess uncut material may cause background by streaking or scattering of signal, or by overwhelming a detector (e.g., over-exposing a film in the case of radioactivity, or exceeding the quantitative detection limits of a fluorescence imager). This can be overcome by partitioning the product from the uncut probe (e.g., by using the charge reversal method described in Example 22 and discussed in detail below). In more complex detection methods, the cleaved product may be intended to interact with another entity to indicate cleavage. As noted above, the cleaved product can be used in any reaction that makes use of oligonucleotides, such as hybridization, primer extension, ligation, or the direction of invasive cleavage. In each of these cases, the fate of the residual uncut probe should be considered in the design of the reaction. In a primer extension reaction, the uncut probe can hybridize to a template for extension. If cleavage is required to reveal the correct 3\u2032 end for extension, the hybridized uncut probe will not be extended. It may, however, compete with the cleaved product for the template. If the template is in excess of the combination of cleaved and uncleaved probe, then both of the latter should be able to find a copy of template for binding. If, however, the template is limiting, any competition may reduce the portion of the cleaved probe that can find successfully bind to the available template. If a vast excess of probe was used to drive the initial reaction, the remainder may also be in vast excess over the cleavage product, and thus may provide a very effective competitor, thereby reducing the amount of the final reaction (e.g., extension) product for ultimate detection.","The participation of the uncut probe material in a secondary reaction can also contribute to background in these reactions. While the presentation of a cleaved probe for a subsequent reaction may represent an ideal substrate for the enzyme to be used in the next step, some enzymes may also be able to act, albeit inefficiently, on the uncut probe as well. It was shown in Example 43 that transcription can be promoted from a nicked promoter even when one side of the nick has additional unpaired nucleotides (termed a \u201cbranched promoter\u201d in that Example). Similarly, when the subsequent reaction is to be an invasive cleavage, the uncleaved probe may bind to the elements intended to form the second cleavage structure with the cleaved probe. Two of the possible configurations are shown schematically in . The right hand structure in the second step in each Figure shows a possible configuration formed by the secondary reaction elements (e.g., secondary targets and\/or probes) and the uncleaved primary probe. In each of these cases, it was found that some of the 5\u2032 nucleases described herein can catalyze some measure of cleavage of these defective structures. Even at a low level, this aberrant cleavage can be misinterpreted as positive target-specific cleavage signal.","With these negative effects of the surfeit of uncut probe considered, there is clearly a need for some method of preventing these interactions. As noted above, it is possible to partition the cleaved product from the uncut probe after the primary reaction by traditional methods. However, these methods are often time consuming, may be expensive (e.g., disposable columns, gels, etc.), and may increase, the risk for sample mishandling or contamination. It is far preferable to configure the sequential reactions such that the original sample need not be removed to a new vessel for subsequent reaction.","The present invention provides a method for reducing interactions between the primary probe and other reactants. This method provides a means of specifically diverting the uncleaved probes from participation in the subsequent reactions. The diversion is accomplished by the inclusion in the next reaction step an agent designed to specifically interact with the uncleaved primary probe. While the primary probe in an invasive cleavage reaction is discussed for reasons of convenience, it is contemplated that the ARRESTOR molecules may be used at any reaction step within a chain of invasive cleavage steps, as needed or desired for the design of an assay. It is not intended that the ARRESTOR molecules of the present invention be limited to any particular step.","The method of diverting the residual uncut probes from a primary reaction makes use of agents that can be specifically designed or selected to bind to the uncleaved probe molecules with greater affinity than to the cleaved probes, thereby allowing the cleaved probe species to effectively compete for the elements of the subsequent reaction, even when the uncut probe is present in vast excess. These agents have been termed \u201cARRESTOR molecules,\u201d due to their function of stopping or arresting the primary probe from participation in the later reaction. In various Examples below, an oligonucleotide is provided as an ARRESTOR oligonucleotide in an invasive cleavage assay. It can be appreciated that any molecule or chemical that can discriminate between the full-length uncut probe and the cleaved probe, and that can bind or otherwise disable the uncleaved probe preferentially may be configured to act as an ARRESTOR molecules within the meaning of the present invention. For example, antibodies can be derived with such specificity, as can the \u201captamers\u201d that can be selected through multiple steps of in vitro amplification (e.g., \u201cSELEX,\u201d U.S. Pat. Nos. 5,270,163 and 5,567,588; herein incorporated by reference) and specific rounds of capture or other selection means.","In one embodiment, the ARRESTOR molecule is an oligonucleotide. In another embodiment the ARRESTOR oligonucleotides is a composite oligonucleotide, comprising two or more short oligonucleotides that are not covalently linked, but that bind cooperatively and are stabilized by co-axial stacking. In a preferred embodiment, the oligonucleotide is modified to reduce interactions with the cleavage agents of the present invention. When an oligonucleotide is used as an ARRESTOR oligonucleotide, it is intended that it not participate in the subsequent reactive step. Consideration of the schematic diagrams in , particularly the right-most Figure in step 2b of each Figure, will show that the binding of the ARRESTOR oligonucleotide to the primary probe may, either with the participation of the secondary target, or without such participation, create a bifurcated structure that is a substrate for cleavage by the 5\u2032 nucleases used in some embodiments of the methods of the present invention. Formation of such structures would lead to some level of unintended cleavage that could contribute to background, reduce specific signal or compete for the enzyme. It is preferable to provide ARRESTOR oligonucleotides that will not create such cleavage structures. One method of doing this is to add to the ARRESTOR oligonucleotides such modifications as have been found to reduce the activity of INVADER oligonucleotides, as the INVADER oligonucleotides occupy a similar position within a cleavage structure (i.e., the 3\u2032 end of the INVADER oligonucleotide positions the site of cleavage of an unpaired 5\u2032 arm). Modification of the 3\u2032 end of the INVADER oligonucleotides was examined for the effects on cleavage in Example 35; a number of the modifications tested were found to be significantly debilitating to the function of the INVADER oligonucleotide. Other modifications not described herein may be easily characterized by performing such a test using the cleavage enzyme to be used in the reaction for which the ARRESTOR oligonucleotide is intended.","In a preferred embodiment, the backbone of an ARRESTOR oligonucleotide is modified. This may be done to increase the resistance to degradation by nucleases or temperature, or to provide duplex structure that is a less favorable substrate for the enzyme to be used (e.g., A-form duplex vs. B-form duplex). In particularly preferred embodiment, the backbone modified oligonucleotide further comprises a 3\u2032 terminal modification. In a preferred embodiment, the modifications comprise 2\u2032 O-methyl substitution of the nucleic acid backbone, while in a particularly preferred embodiment, the 2\u2032 O-methyl modified oligonucleotide further comprises a 3\u2032 terminal amine group.","The purpose of the ARRESTOR oligonucleotide is to allow the minority population of cleaved probe to effectively compete with the uncleaved probe for binding whatever elements are to be used in the next step. While an ARRESTOR oligonucleotide that can discriminate between the two probe species absolutely (i.e., binding only to uncut and never to cut) may be of the greatest benefit in some embodiments, it is envisioned that in many applications, including the sequential INVADER assays described herein, the ARRESTOR oligonucleotide of the present invention may perform the intended function with only partial discrimination. When the ARRESTOR oligonucleotide has some interaction with the cleaved probe, it may prevent detection of some portion of these cleavage products, thereby reducing the absolute level of signal generated from a given amount of target material. If this same ARRESTOR oligonucleotide has the simultaneous effect of reducing the background of the reaction (i.e., from non-target specific cleavage) by a factor that is greater than the factor of reduction in the specific signal, then the significance of the signal (i.e., the ratio of signal to background), is increased, even with the lower amount of absolute signal. Any potential ARRESTOR molecule design may be tested in a simple fashion by comparing the levels of background and specific signals from reactions that lack ARRESTOR molecules to the levels of background and specific signal from similar reactions that include ARRESTOR oligonucleotides. Each of the reactions described in Examples 49-53 demonstrate the use of such comparisons, and these can easily be adapted by those skilled in the art to other ARRESTOR molecules and target embodiments. What constitutes an acceptable level of tradeoff of absolute signal for specificity will vary for different applications (e.g., target levels, read-out sensitivity, etc.), and can be determined by any individual user using the methods of the present invention.","As noted above, the oligonucleotide product released by the invasive cleavage can be used subsequently in any reaction or read-out method that uses oligonucleotides in the size range of a cleavage product. In addition to the reactions involving primer extension and transcription, described herein, another enzymatic reaction that makes use of oligonucleotides is the invasive cleavage reaction. The present invention provide means of using the oligonucleotide released in a primary invasive cleavage reaction as a component to complete a cleavage structure to enable a secondary invasive cleavage reaction. One possible configuration of a primary cleavage reaction supplying a component for a secondary cleavage structure is diagrammed in FIG. . Is not intended that the sequential use of the invasive cleavage product be limited to a single additional step. It is contemplated that many distinct invasive cleavage reactions may be performed in sequence.","The polymerase chain reaction uses a DNA replication method to create copies of a targeted segment of nucleic acid at a logarithmic rate of accumulation. This is made possible by the fact that when the strands of DNA are separated, each individual strand contains sufficient information to allow assembly of a new complementary strand. When the new strands are synthesized the number of identical molecules has doubled. Within 20 iterations of this process, the original may be copied I million-fold, making very rare sequences easily detectable. The mathematical power of a doubling reaction has been incorporated into a number of amplification assays, several of which are cited in Table 1.","By performing multiple, sequential invasive cleavage reactions the method of the present invention captures an exponential mathematical advantage without producing additional copies of the target analyte. In a simple invasive cleavage reaction the yield, Y, is simply the turnover rate, K, multiplied by the time of the reaction, t (i.e., Y=(K)(t)). If Y is used to represent the yield of a simple reaction, then the yield of a compound (i.e., a multiple, sequential reaction), assuming that each of the individual invasive cleavage steps has the same turnover rate, can be simply represented as Y, where n is the number of invasive cleavage reactions that have been performed in the series. If the yields of each step differ the ultimate yield can be represented as the product of the multiplication of the yields of each individual reaction in the series. For example, if a primary invasive cleavage reaction can produce one thousand products in 30 minutes, and each of those products can in turn participate in 1000 additional reactions, there will be 1000copies (1000\u00d71000) of the ultimate product in a second reaction. If a third reaction is added to the series, then the theoretical yield will be 1000(1000\u00d71000\u00d71000). In the methods of the present invention the exponent comes from the number of invasive cleavage reactions in the cascade. This can be contrasted to the amplification methods described above (e.g., PCR) in which Y is limited to 2 by the number of strands in duplex DNA, and the exponent n is the number of cycles performed, so that many iterations are necessary to accumulate large amounts of product.","To distinguish the exponential amplifications described above from those of the present invention, the former can be considered reciprocating reactions because the products the reaction feed back into the same reaction (e.g., event one leads to some number of events 2, and each event 2 leads back to some number of events 1). In contrast, the events of the present invention are sequential (e.g., event I leads to some number of events 2; each event 2 leads to some number of events 3, etc., and no event can contribute to an event earlier in the chain).","The sensitivity of the reciprocating methods is also one of the greatest weaknesses when these assays are used to determine if a target nucleic acid sequence is present or absent in a sample. Because the product of these reactions is detectable copies of the starting material, contamination of a new reaction with the products of an earlier reaction can lead to false positive results, (i.e., the apparent detection of the target nucleic acid in samples that do not actually contain any of that target analyte). Furthermore, because the concentration of the product in each positive reaction is so high, amounts of DNA sufficient to create a strong false positive signal can be communicated to new reactions very easily either by contact with contaminated instruments or by aerosol. In contrast to the reciprocating methods, the most concentrated product of the sequential reaction (i.e., the product released in the ultimate invasive cleavage event) is not capable of initiating a like reaction or cascade if carried over to a fresh test sample. This is a marked advantage over the exponential amplification methods described above because the reactions of the present invention may be performed without the costly containment arrangements (e.g., either by specialized instruments or by separate laboratory space) required by any reciprocating reaction. While the products of a penultimate event may be inadvertently transferred to produce a background of the ultimate product in the absence of the a target analyte, the contamination would need to be of much greater volume to give an equivalent risk of a false positive result.","When the term sequential is used it is not intended to limit the invention to configurations in which that one invasive cleavage reaction or assay must be completed before the initiation of a subsequent reaction for invasive cleavage of a different probe. Rather, the term refers to the order of events as would occur if only single copies of each of the oligonucleotide species were used in an assay. The primary invasive cleavage reaction refers to that which occurs first, in response to the formation of the cleavage structure on the target nucleic acid. Subsequent reactions may be referred to as secondary, tertiary and so forth, and may involve artificial \u201ctarget\u201d strands that serve only to support assembly of a cleavage structure, and which are unrelated to the nucleic acid analyte of interest. While the complete assay may, if desired, be configured with each step of invasive cleavage separated either in space (e.g., in different reaction vessels) or in time (e.g., using a shift in reaction conditions, such as temperature, enzyme identity or solution condition, to enable the later cleavage events), it is also contemplated that all of the reaction components may be mixed so that secondary reactions may be initiated as soon as product from a primary cleavage becomes available. In such a format, primary, secondary and subsequent cleavage events involving different copies of the cleavage structures may take place simultaneously.","Several levels of this sort of linear amplification can be envisioned, in which each successive round of cleavage produces an oligonucleotide that can participate in the cleavage of a different probe in subsequent rounds. The primary reaction would be specific for the analyte of interest with secondary (and tertiary, etc.) reactions being used to generate signal while still being dependent on the primary reaction for initiation.","The released product may perform in several capacities in the subsequent reactions. One of the possible variations is shown in , in which the product of one invasive cleavage reaction becomes the INVADER oligonucleotide to direct the specific cleavage of another probe in a second reaction. In , the first invasive cleavage structure is formed by the annealing of the INVADER oligonucleotide (\u201cInvader\u201d) and the probe oligonucleotide (\u201cProbe 1\u201d) to the first target nucleic acid (\u201cTarget 1\u201d). The target nucleic acid is divided into three regions based upon which portions of the INVADER and probe oligonucleotides are capable of hybridizing to the target (as discussed above and as shown in FIG. ). Region I (region Y in ) of the target has complementarity to only the INVADER oligonucleotide; region 3 (region Z in ) of the target has complementarity to only the probe; and region 2 (region X in ) of the target has complementarity to both the INVADER and probe oligonucleotides. It is noted that the sequential invasive cleavage reaction diagrammed in  employs an INVADER and a probe oligonucleotide; the sequential cleavage reaction is not limited to the use of such a first cleavage structure. The first cleavage structure in the sequential reaction may also employ an INVADER oligonucleotide, a mini probe and a stacker oligonucleotide as discussed above. Further, as discussed above, the overlap in any or all of the cleavage structures in the sequential reactions may comprise moieties other than overlapping complementary bases, such that the region shown as \u201cX\u201d represents a region where there is a physical rather than sequence overlap between the INVADER and probe oligonucleotides","In , cleavage of Probe 1 releases the \u201cCut Probe 1\u201d (indicated by the hatched line in both the cleaved and uncleaved Probe 1 in FIG. ). The released Probe 1 is then used as the INVADER oligonucleotide in second cleavage. The second cleavage structure is formed by the annealing of the Cut Probe 1, a second probe oligonucleotide (\u201cProbe 2\u201d) and a second target nucleic acid (\u201cTarget 2\u201d) In some embodiments, Probe 2 and the second target nucleic acid are covalently connected, preferably at their 3\u2032 and 5\u2032 ends, respectively, thus forming a hairpin stem and loop, termed herein a \u201ccassette\u201d. The loop may be nucleic acid, (e.g., a string of nucleotides, such as the four T residues depicted in several Figures, including A) or a non-nucleic acid spacer or linker. Inclusion of an excess of the cassette molecule allows each Cut Probe 1 to serve as an INVADER to direct the cleavage of multiple copies of the cassette.","Probe 2 may be labeled (e.g., as indicated by the star in ) and detection of cleavage of the second cleavage structure may be accomplished by detecting the labeled cut Probe 2; the label may a radioisotope (e.g., P, S), a fluorophore (e.g., fluorescein), a reactive group capable of detection by a secondary agent (e.g., biotin\/streptavidin), a positively charged adduct which permits detection by selective charge reversal (as discussed in Section IV above), etc. Alternatively, the cut Probe 2 may used in a tailing reaction, or to complete or activate a protein binding site, or may be detected or used by any of the means for detecting or using an oligonucleotide described herein.","Another possible configuration for performing a sequential invasive cleavage reaction is diagrammed in FIG. . In this embodiment, probe oligonucleotides that are cleaved in the primary reaction can be designed to fold back on themselves (i.e., they contain a region of self-complementarity) to create a molecule that can serve as both the INVADER and target oligonucleotide (termed here an \u201cIT\u201d complex). The IT complex then enables cleavage of a different probe present in the secondary reaction. Inclusion of an excess of the secondary probe molecule (\u201cProbe 2\u201d), allows each IT molecule to serve as the platform for the generation of multiple copies of cleaved secondary probe. In , the regions of self-complementarity contained within the 5\u2032 portion of the INVADER oligonucleotide is indicated by the hatched ovals; the arrow between these two ovals indicates that these two regions can self-pair (as shown in the \u201cCut Probe 1\u201d). The target nucleic acid is divided into three regions based upon which portions of the INVADER and probe oligonucleotides are capable of hybridizing to the target (as discussed above and it is noted that the target may be divided into four regions if a stacker oligonucleotide is employed). The second cleavage structure is formed by the annealing of the second probe (\u201cProbe 2\u201d) to the fragment of Probe 1 (\u201cCut Probe 1\u201d) that was released by cleavage of the first cleavage structure. The Cut Probe 1 forms a hairpin or stem\/loop structure near its 3\u2032 terminus by virtue of the annealing of the regions of self-complementarity contained within Cut Probe 1 (this self-annealed Cut Probe 1 forms the IT complex). The IT complex (Cut Probe 1) is divided into three regions. Region 1 of the IT complex has complementarity to the 3\u2032 portion of Probe 2; region 2 has complementarity to both the 3\u2032 end of Cut Probe I and to the 5\u2032 portion of Probe 2 (analogous to the region of overlap \u201cX\u201d shown in FIG. ); and region 3 contains the region of self-complementarity (i.e., region 3 is complementary to the 3\u2032 portion of the Cut Probe 1). Note that with regard to the IT complex (i.e., Cut Probe 1), region 1 is located upstream of region 2 and region 2 is located upstream of region 3. As for other embodiments of invasive cleavage, the region shown as \u201c2\u201d can represent a region where there is a physical, but not sequence, overlap between the INVADER portion of the Cut Probe 1 and the Probe 2 oligonucleotide.","The cleavage products of the secondary invasive cleavage reaction (i.e., Cut Probe 2) can either be detected, or can in turn be designed to constitute yet another integrated INVADER-target complex to be used with a third probe molecule, again unrelated to the preceding targets.","The present invention is not limited to the configurations diagrammed in . It is envisioned that the oligonucleotide product of a primary cleavage reaction may fill the role of any of the oligonucleotides described herein (e.g., it may serve as a target strand without an attached INVADER oligonucleotide-like sequence, or it may serve as a stacker oligonucleotide, as described above), to enhance the turnover rate seen in the secondary reaction by stabilizing the probe hybridization through coaxial stacking.","Secondary cleavage reactions in some preferred embodiments of the present invention include the use of FRET cassettes such as those described in Examples 54 through 62. Such molecules provide both a secondary target and a FRET labeled cleavable sequence, allowing homogeneous detection (i.e., without product separation or other manipulation after the reaction) of the sequential invasive cleavage reaction. Other preferred embodiments use a secondary reaction system in which the FRET probe and synthetic target are provided as separate oligonucleotides.","In a preferred embodiment, each subsequent reaction is facilitated by (i.e., is dependent upon) the product of the previous cleavage, so that the presence of the ultimate product may serve as an indicator of the presence of the target analyte. However, cleavage in the second reaction need not be dependent upon the presence of the product of the primary cleavage reaction; the product of the primary cleavage reaction may merely measurably enhance the rate of the second cleavage reaction.","In summary, the INVADER assay cascade (i.e., sequential invasive cleavage reactions) of the present invention is a combination of two or more linear assays that allows the accumulation of the ultimate product at an exponential rate, but without significant risk of carryover contamination. It is an important to note that background that does not arise from sequential cleavage, such as thermal breakage of the secondary probe, generally increases linearly with time. In contrast, signal generation from a 2-step sequential reaction follows quadratic kinetics. Thus, collection of data as a time course, either by taking time points or through the use of an instrument that allows real-time detection during the INVADER assay reaction incubations, provides the attractive capability of discriminating between the true signal and any background solely on the basis of quadratic versus linear increases in signal over time. For example, when viewed graphically, the real signal will appear as a quadratic curve, while any accumulating background will be linear, and thus easy to distinguish, even if the absolute level of the background signal (e.g., fluorescence in a FRET detection format) is substantial.","The sequential invasive cleavage amplification of the present invention can be used as an intermediate boost to any of the detection methods (e.g., gel based analysis by either standard or by charge reversal), polymerase tailing, and incorporation into a protein binding region, described herein. When used is such combinations the increased production of a specific cleavage product in the invasive cleavage assay reduces the burdens of sensitivity and specificity on the read-out systems, thus facilitating their use.","In addition to enabling a variety of detection platforms, the cascade strategy is suitable for multiplex analysis of individual analytes (i.e., individual target nucleic acids) in a single reaction. The multiplex format can be categorized into two types. In one case, it is desirable to know the identity (and amount) of each of the analytes that can be present in a clinical sample, or the identity of each of the analytes as well as an internal control. To identify the presence of multiple individual analytes in a single sample, several distinct secondary amplification systems may be included. Each probe cleaved in response to the presence of a particular target sequence (or internal control) can be designed to trigger a different cascade coupled to different detectable moieties, such as different sequences to be extended by DNA polymerase or different dyes in an FRET format. The contribution of each specific target sequence to final product can thereby be tallied, allowing quantitative detection of different genes or alleles in a sample containing a mixture of genes or alleles.","In the second configuration, it is desirable to determine if any of several analytes are present in a sample, but the exact identity of each does not need to be known. For example, in blood banking it is desirable to know if any one of a host of infectious agents is present in a sample of blood. Because the blood is discarded regardless of which agent is present, different signals on the probes would not be required in such an application of the present invention, and may actually be undesirable for reasons of confidentiality. In this case, the 5\u2032 arms (i.e., the 5\u2032 portion which will be released upon cleavage) of the different analyte-specific probes would be identical and would therefore trigger the same secondary signal cascade. A similar configuration would permit multiple probes complementary to a single gene to be used to boost the signal from that gene or to ensure inclusivity when there are numerous alleles of a gene to be detected.","In the primary INVADER reaction, there are two potential sources of background. The first is from INVADER-independent cleavage of probe annealed to the target, to itself, or to one of the other oligonucleotides present in the reaction. It can be seen by consideration of  that the probes of the primary cleavage reactions depicted are designed to have regions of complementarity to the other oligonucleotides involved in the subsequent reactions, and, as depicted in , to other regions of the same molecule. The use of an enzyme that cannot efficiently cleave a structure that lacks a primer (e.g., that cannot cleave the structures diagrammed in  or D) is preferred for this reason. As shown in , the enzyme Pfu FEN-1 gives no detectable cleavage in the absence of the upstream oligonucleotide or even in the presence of an upstream oligonucleotide that fails to invade the probe-target complex. This indicates that the Pfu FEN-1 endonuclease is a suitable enzyme for use in the methods of the present invention.","Other structure-specific nucleases may be suitable as a well. As discussed in the first example, some 5\u2032 nucleases can be used in conditions that significantly reduce this primer-independent cleavage. For example, it has been shown that when the 5\u2032 nuclease of DNAPTaq is used to cleave hairpins the primer-independent cleavage is markedly reduced by the inclusion of a monovalent salt in the reaction (Lyamichev, et al., [1993], supra).","Test For INVADER Oligonucleotide-Independent Cleavage","A simple test can be performed for any enzyme in combination with any reaction buffer to gauge the amount of INVADER oligonucleotide-independent cleavage to be expected from that combination. A small hairpin-like test molecule that can be used with or without a primer hybridized to a 3\u2032 arm, the S-60 molecule, is depicted in FIG. . The S-60 and the oligonucleotide P15 are a convenient set of molecules for testing the suitability of an enzyme for application in the present invention and conditions for using these molecules are described in Example 11. Other similar hairpins may be used. A cleavage structure may be assembled from separate oligonucleotides as diagrammed in . Reactions using these structures to examine the activity of the Pfu FEN-1 enzyme in the presence or absence of an upstream overlapping oligonucleotide are described in Example 45 and the results are displayed in FIG. . To test any particular combination of enzyme and cleavage conditions, similar reactions can be assembled. Outside of the variables of reaction conditions to be tested for any particular enzyme (e.g., salt sensitivities, divalent cation requirements) the test reactions should accommodate any known limitations of the test enzyme. For example, the test reactions should be performed at a temperature that is within the operating temperature range of the candidate enzyme, if known.","It is not necessary that multiple lengths of overlap be demonstrated for each candidate enzyme, but the activity of the enzyme in the absence of an upstream oligonucleotide (sequence or physical overlap) (as shown in ) and in the presence of an oligonucleotide that does not overlap () should be assessed. It is preferable that structures lacking an upstream oligonucleotide be cleaved at less than one half of the rate seen in the presence of an upstream overlapping oligonucleotide. It is more preferable that these structures be cleaved at less than about on tenth the rate of the invasive cleavage structure. It is most preferred that cleavage of these structures occur at less than one percent the rate of the invasive cleavage structure.","If the cleaved product is to serve as an upstream oligonucleotide in a subsequent cleavage reaction, as diagrammed in , the most rapid reaction will be achieved if the other components of the second cleavage structure (i.e., Target 2 and Probe 2 in ) are provided in excess compared to the amount of first cleavage product, so that cleavage may proceed immediately after the upstream oligonucleotide (i.e, Cut Probe 1 in ) is made available. To provide an abundance of the second target strand or cassette (Target 2 in ) one may use an isolated natural nucleic acid, such as bacteriophage M13 DNA, or one may use a synthetic oligonucleotide. If a synthetic oligonucleotide is chosen as the second target sequence, the sequence employed should be examined for regions of unintended self-complementarity (similar considerations apply to short isolated natural nucleic acids such as restriction enzyme fragments or PCR products; natural nucleic acid targets whose 3\u2032 end is located \u2267100 nuclcotides downstream of the probe binding site on the target strand are generally long enough to obviate the design considerations discussed below). Specifically, it should be determined that the 3\u2032 end of the synthetic oligonucleotide may not hybridize to the target strand (i.e., intra-strand hybridization) upstream of the probe, triggering unintended cleavage. Simple examination of the sequence of the synthetic oligonucleotide should reveal if the 3\u2032 end has sufficient complementarity to the region of the target upstream of the probe binding site to pose a problem (i.e, it would reveal whether the synthetic oligonucleotide can form a hairpin at its 3\u2032 end which could act as an invading oligonucleotide to cause cleavage of the 2probe in the absence of the hybridization of the intended INVADER oligonucleotide (i.e., the cleavage product from the first invasive cleavage reaction)). If 3 or more of the last 4 to 7 nucleotides (the 3\u2032 terminal region) of the synthetic target can basepair upstream of the probe such that there is an overlap with the probe-target duplex, or such that the duplexes formed by the synthetic target strand with its own 3\u2032 terminal region and with the probe abut without a gap and the 3\u2032 terminal region has an additional 1 or 2 nucleotides unpaired at the extreme 3\u2032 end of the synthetic target, then the sequence of the synthetic target oligonucleotide should be modified. The sequence may be changed to disrupt the interaction of the 3\u2032 terminal region or to increase the distance between the probe binding site and the regions to which the 3\u2032 terminus is binding. Alternatively, the 3\u2032 end may be modified to reduce its ability to direct cleavage (e.g., by adding a 3\u2032 phosphate during synthesis) (see Ex. 35, Table 3) or by adding several additional nucleotides that will not basepair in a self-complementary manner (i.e., they will not participate in the formation of a hairpin structure).","When the product of a first invasive cleavage reaction is designed to form a target that can fold on itself to direct cleavage of a second probe, the IT complex as diagrammed in , the design of the sequence used to form the stem\/loop of the IT complex should be considered. To be factored into the design of such a probe are 1) the length of the region of self-complementarity, 2) the type of overlap (i.e., what 3\u2032 moiety) and, if an overlap in sequence is selected, the length of the region of overlap (region \u201cX\u201d in ) and 3) the stability of the hairpin or stem\/loop structure as predicted by both Watson-Crick base pairing and by the presence or absence of a particularly stable loop sequence (e.g., a tetraloop [Tinoco et al., supra], or a triloop [Hirao et al., supra]). It is desirable that this sequence have nuclcotides that can base pair (intrastrand), so that the second round of invasive cleavage may occur, but that the structure not be so strong that its presence will prevent the cleavage of the probe in the primary reaction (i.e., Probe 1 in FIG. ). As shown herein, the presence of a secondary structure in the 5\u2032 arm of a cleavage structure cleaved by a structure-specific nuclease may inhibit cleavage by some structure-specific nucleases (Ex. 1).","The length of the region of self-complementarity within Probe 1 determines the length of the region of the duplex upstream of Probe 2 in the second cleavage structure (see FIG. ). Different enzymes have different length requirements for this duplex to effect invasive cleavage efficiently. For example, the Pfu FEN-1 and Mja FEN-1 enzymes have been tested for the effect of this duplex length using the set of target\/INVADER oligonucleotide molecules depicted in  (i.e., SEQ ID NOS:118, 119, 147-151). The invasive cleavage reactions were performed as described in Example 38, using 1 pM IT3 (SEQ ID NO:118), 2 \u03bcM probe PR1 (SEQ ID NO:119) for 5 min, and the rates of cleavage are shown in Table 2.",{"@attributes":{"id":"p-0446","num":"0455"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"center"}}],"thead":{"row":[{"entry":"TABLE 2"},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":[{},"Pfu FEN-1 Turnover,","Mja FEN-1 Turnover, per"]},{"entry":["Length of Duplex","per min.","min."]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["0","0","0"]},{"entry":["3","1","29"]},{"entry":["4","10","57"]},{"entry":["6","44","51"]},{"entry":["8","45","46"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"The data shown in Table 2 demonstrate that the Pfu FEN-1 enzyme can be used with stems of 3 or 4 bases, but that the rate of cleavage is maximized when the stem is greater than 4 basepairs in length. Table 2 shows that the Mja FEN-1 enzyme can cleave efficiently using shorter stems; however, as this enzyme can also cleave a probe in the absence of an upstream oligonucleotide, Mja FEN-1 is not preferred for use in the sequential invasive cleavage methods of the present invention.","A similar test can be performed using any candidate enzyme to determine how much self-complementarity may be designed into the Probe 1. The use of a shorter stem means that the overall probe may be shorter. This is beneficial because shorter probes are less costly to synthesize, and because shorter probes will have fewer sequences that might form unintended intrastrand structures. In assessing the activity of a candidate enzyme on the structures such as those shown in  it is not required that the stem length chosen allow the maximum rate of cleavage to occur. For example, in considering the case of Pfu FEN-1, the advantages of using a 4 basepair stem (e.g., cost or sequence limitations), with a cleavage rate of 10 cleavages per minute, may outweigh the rate advantage of using a longer 6 basepair stem (44 cleavages\/min.), in the context of a particular experiment. It is within the scope of the present invention that some elements chosen for use in the assay be sub-optimal for performance of that particular element, if the use of a sub-optimal design benefits the objectives of that particular experiment as a whole.","In designing oligonucleotides to be employed as a probe that, once cleaved, forms a stem-loop structure as diagrammed in  (i.e., Probe 1 in FIG. ), it has been found that the stability of the loop is not a factor in the efficiency of cleavage of either Probe 1 or Probe 2. Loops tested have included stable triloops, loops of 3 and 4 nucleotides that were not predicted to be particularly stable (i.e., the stability is determined by the duplex sequence and not by additional stabilizing interactions within the loop), and large loops of up to about 25 nucleotides.","Some nucleic acid-based detection assays involve the elongation and\/or shortening of oligonucleotide probes. For example, as described herein, the primer-directed, primer-independent, and INVADER-directed cleavage assays, as well as the \u201cnibbling\u201d assay all involve the cleavage (i.e., shortening) of oligonucleotides as a means for detecting the presence of a target nucleic sequence. Examples of other detection assays that involve the shortening of an oligonucleotide probe include the \u201cTaqMan\u201d or nick-translation PCR assay described in U.S. Pat. No. 5,210,015 to Gelfand et al. (the disclosure of which is herein incorporated by reference), the assays described in U.S. Pat. Nos. 4,775,619 and 5,118,605 to Urdea (the disclosures of which are herein incorporated by reference), the catalytic hybridization amplification assay described in U.S. Pat. No. 5,403,711 to Walder and Walder (the disclosure of which is herein incorporated by reference), and the cycling probe assay described in U.S. Pat. Nos. 4,876,187 and 5,011,769 to Duck et al. (the disclosures of which are herein incorporated by reference). Examples of detection assays that involve the elongation of an oligonucleotide probe (or primer) include the polymerase chain reaction (PCR) described in U.S. Pat. Nos. 4,683,195 and 4,683,202 to Mullis and Mullis et al. (the disclosures of which are herein incorporated by reference) and the ligase chain reaction (LCR) described in U.S. Pat. Nos. 5,427,930 and 5,494,810 to Birkenmeyer et al. and Barany et al. (the disclosures of which are herein incorporated by reference). The above examples are intended to be illustrative of nucleic acid-based detection assays that involve the elongation and\/or shortening of oligonucleotide probes and do not provide an exhaustive list.","Typically, nucleic acid-based detection assays that involve the elongation and\/or shortening of oligonucleotide probes require post-reaction analysis to detect the products of the reaction. It is common that the specific reaction product(s) must be separated from the other reaction components, including the input or unreacted oligonucleotide probe. One detection technique involves the electrophoretic separation of the reacted and unreacted oligonucleotide probe. When the assay involves the cleavage or shortening of the probe, the unreacted product will be longer than the reacted or cleaved product. When the assay involves the elongation of the probe (or primer), the reaction products will be greater in length than the input. Gel-based electrophoresis of a sample containing nucleic acid molecules of different lengths separates these fragments primarily on the basis of size. This is due to the fact that in solutions having a neutral or alkaline pH, nucleic acids having widely different sizes (i.e., molecular weights) possess very similar charge-to-mass ratios and do not separate (Andrews, Electrophoresis, 2nd Edition, Oxford University Press (1986), pp. 153-154]. The gel matrix acts as a molecular sieve and allows nucleic acids to be separated on the basis of size and shape (e.g., linear, relaxed circular or covalently closed supercoiled circles).","Unmodified nucleic acids have a net negative charge due to the presence of negatively charged phosphate groups contained within the sugar-phosphate backbone of the nucleic acid. Typically, the sample is applied to gel near the negative pole and the nucleic acid fragments migrate into the gel toward the positive pole with the smallest fragments moving fastest through the gel.","The present invention provides a novel means for fractionating nucleic acid fragments on the basis of charge. This novel separation technique is related to the observation that positively charged adducts can affect the electrophoretic behavior of small oligonucleotides because the charge of the adduct is significant relative to charge of the whole complex. In addition to the use of positively charged adducts (e.g., Cy3 and Cy5 fluorescent dyes, the positively charged heterodimeric DNA-binding dyes shown in , etc.), the oligonucleotide may contain amino acids (particularly useful amino acids are the charged amino acids: lysine, arginine, asparate, glutamate), modified bases, such as amino-modified bases, and\/or a phosphonate backbone (at all or a subset of the positions). In other embodiments, as discussed further below, a neutral dye or detection moiety (e.g., biotin, streptavidin, etc.) may be employed in place of a positively charged adduct, in conjunction with the use of amino-modified bases and\/or a complete or partial phosphonate backbone.","This observed effect is of particular utility in assays based on the cleavage of DNA molecules. Using the assays described herein as an example, when an oligonucleotide is shortened through the action of a CLEAVASE enzyme or other cleavage agent, the positive charge can be made to not only significantly reduce the net negative charge, but to actually override it, effectively \u201cflipping\u201d the net charge of the labeled entity. This reversal of charge allows the products of target-specific cleavage to be partitioned from uncleaved probe by extremely simple means. For example, the products of cleavage can be made to migrate towards a negative electrode placed at any point in a reaction vessel, for focused detection without gel-based electrophoresis; Example 24 provides examples of devices suitable for focused detection without gel-based electrophoresis. When a slab gel is used, sample wells can be positioned in the center of the gel, so that the cleaved and uncleaved probes can be observed to migrate in opposite directions. Alternatively, a traditional vertical gel can be used, but with the electrodes reversed relative to usual DNA gels (i.e., the positive electrode at the top and the negative electrode at the bottom) so that the cleaved molecules enter the gel, while the uncleaved disperse into the upper reservoir of electrophoresis buffer.","An important benefit of this type of readout is the absolute nature of the partition of products from substrates (i.e., the separation is virtually 100%). This means that an abundance of uncleaved probe can be supplied to drive the hybridization step of the probe-based assay, yet the unconsumed (i.e., unreacted) probe can, in essence, be subtracted from the result to reduce background by virtue of the fact that the unreacted probe will not migrate to the same pole as the specific reaction product.","Through the use of multiple positively charged adducts, synthetic molecules can be constricted with sufficient modification that the normally negatively charged strand is made nearly neutral. When so constructed, the presence or absence of a single phosphate group can mean the difference between a net negative or a net positive charge. This observation has particular utility when one objective is to discriminate between enzymatically generated fragments of DNA, which lack a 3\u2032 phosphate, and the products of thermal degradation, which generally retain a 3\u2032 phosphate (and thus two additional negative charges). Examples 22 and 23 demonstrate the ability to separate positively charged reaction products from a net negatively charged substrate oligonucleotide. As discussed in these examples, oligonucleotides may be transformed from net negative to net positively charged compounds. In Example 23, the positively charged dye, Cy3 was incorporated at the 5\u2032 end of a 22-mer (SEQ ID NO:50) which also contained two amino-substituted residues at the 5\u2032 end of the oligonucleotide; this oligonucleotide probe carries a net negative charge. After cleavage, which occurred 2 nucleotides into the probe, the following labeled oligonucleotide was released: 5\u2032-Cy3-AminoT-AminoT-3\u2032 (in addition to unlabeled fragment comprising the remaining 20 nucleotides of SEQ ID NO:50). This short fragment bears a net positive charge while the remainder of the cleaved oligonucleotide and the unreacted or input oligonucleotide bear net negative charges.","The present invention contemplates embodiments wherein the specific reaction product produced by any cleavage of any oligonucleotide can be designed to carry a net positive charge while the unreacted probe is charge neutral or carries a net negative charge. The present invention also contemplates embodiments where the released product may be designed to carry a net negative charge while the input nucleic acid carries a net positive charge. Depending on the length of the released product to be detected, positively charged dyes may be incorporated at the one end of the probe and modified bases may be placed along the oligonucleotide such that upon cleavage, the released fragment containing the positively charged dye carries a net positive charge. Amino-modified bases may be used to balance the charge of the released fragment in cases where the presence of the positively charged adduct (e.g., dye) alone is not sufficient to impart a net positive charge on the released fragment. In addition, the phosphate backbone may be replaced with a phosphonate backbone at a level sufficient to impart a net positive charge (this is particularly useful when the sequence of the oligonucleotide is not amenable to the use of amino-substituted bases);  show the structure of short oligonucleotides containing a phosphonate group on the second T residue). An oligonucleotide containing a fully phosphonate-substituted backbone would be charge neutral (absent the presence of modified charged residues bearing a charge or the presence of a charged adduct) due to the absence of the negatively charged phosphate groups. Phosphonate-containing nucleotides (e.g., methylphosphonate-containing nucleotides are readily available and can be incorporated at any position of an oligonucleotide during synthesis using techniques which are well known in the art.","In essence, the invention contemplates the use of charge-based separation to permit the separation of specific reaction products from the input oligonucleotides in nucleic acid-based detection assays. The foundation of this novel separation technique is the design and use of oligonucleotide probes (typically termed \u201cprimers\u201d in the case of PCR) which are \u201ccharge balanced\u201d so that upon either cleavage or elongation of the probe it becomes \u201ccharge unbalanced,\u201d and the specific reaction products may be separated from the input reactants on the basis of the net charge.","In the context of assays that involve the elongation of an oligonucleotide probe (i.e., a primer), such as is the case in PCR, the input primers are designed to carry a net positive charge. Elongation of the short oligonucleotide primer during polymerization will generate PCR products that now carry a net negative charge. The specific reaction products may then easily be separated and concentrated away from the input primers using the charge-based separation technique described herein (the electrodes will be reversed relative to the description in Example 23 as the product to be separated and concentrated after a PCR will carry a negative charge).","It has been determined that when oligonucleotide probes are used in cleavage detection assays at elevated temperature, some fraction of the truncated probes will have been shortened by nonspecific thermal degradation, and that such breakage products can make the analysis of the target-specific cleavage data more difficult. The thermal degradation that creates a background ladder of bands when the probes of the present invention are treated at high temperature for more than a few minutes occurs as a two step process. In the first step the N-glycosyl bond breaks, leaving an abasic site in the DNA strand. At the abasic site the DNA chain is weakened and undergoes spontaneous cleavage through a beta-elimination process. It has been determined that purine bases are about 20 times more prone to breakage than pyrimidine bases (Lindahl, Nature 362:709 [1993]). This suggests that one way of reducing background in methods using oligonucleotides at elevated temperatures is to select target sequences that allow the use of pyrimidine-rich probes. It is preferable, where possible, to use oligonucleotides that are entirely composed of pyrimidine residues. If only one or a few purines are used, the background breakage will appear primarily at the corresponding sites, and these bands (due to thermal breakdown) may be mistaken for the intended cleavage products if care is not taken in the data analysis (i.e., proper controls must be run).","Background cleavage due to thermal breakdown of probe oligonucleotides can, when not resolved from specific cleavage products, reduce the accuracy of quantitation of target nucleic acids based on the amount of accumulated product in a set timeframe. One means of distinguishing the specific from the nonspecific products is disclosed above, and is based on partitioning the products of these reactions by differences in the net charges carried by the different molecular species in the reaction. As was noted in that discussion, the thermal breakage products usually retain 3\u2032 phosphates after breakage, while the enzyme-cleaved products do not. The two negative charges on the phosphate facilitate charge-based partition of the products.","The absence of a 3\u2032 phosphate on the desired subset of the probe fragments may be used to advantage in enzymatic assays as well. Nucleic acid polymerases, both non-templated (e.g., terminal deoxynucleotidyl transferase, polyA polymerase) and template-dependent (e.g., Pol I-type DNA polymerases), require an available 3\u2032 hydroxyl by which to attach further nucleotides. This enzymatic selection of 3\u2032 end structure may be used as an effective means of partitioning specific from non-specific products.","In addition to the benefits of the partitioning described above, the addition of nucleotides to the end of the specific product of an INVADER oligonucleotide-specific cleavage offers an opportunity to either add label to the products, to add capturable tails to facilitate solid-support based readout systems, or to do both of these things at the same time. Some possible embodiments of this concept are illustrated in FIG. .","In , an INVADER cleavage structure comprising an INVADER oligonucleotide containing a blocked or non-extendible 3\u2032 end (e.g., a 3\u2032 dideoxynudleotide) and a probe oligonucleotide containing a blocked or non-extendable 3\u2032 end (the open circle at the 3\u2032 end of the oligonucleotides represents a non-extendible nucleotide) and a target nucleic acid is shown; the probe oligonucleotide may contain a 5\u2032 end label such as a biotin or a fluorescein (indicated by the stars) label (cleavage structures which employ a 5\u2032 biotin-labeled probe or a 5\u2032 fluorescein-labeled probe are shown below the large diagram of the cleavage structure to the left and the right, respectively). Following cleavage of the probe (the site of cleavage is indicated by the large arrowhead), the cleaved biotin-labeled probe is extended using a template- independent polymerase (e.g., TdT) and fluoresceinated nucleotide triphosphates. The fluorescein tailed cleaved probe molecule is then captured by binding via its 5\u2032 biotin label to streptavidin and the fluorescence is then measured. Alternatively, following, cleavage of a 5\u2032-fluoresceinated probe, the cleaved probe is extended using a template-independent polymerase (e.g., TdT) and dATP. The polyadenylated (A-tailed) cleaved probe molecule is then captured by binding via the polyA tail to oligo dT attached to a solid support.","The examples described in  are based on the use of TdT to tail the specific products of INVADER-directed cleavage. The description of the use of this particular enzyme is presented by way of example and is not intended as a limitation (indeed, when probe oligonucleotides comprising RNA are employed, cleaved RNA probes may be extended using polyA polymerase). It is contemplated that an assay of this type can be configured to use a template-dependent polymerase, as described above. While this would require the presence of a suitable copy template distinct from the target nucleic acid, on which the truncated oligonucleotide could prime synthesis, it can be envisaged that a probe that before cleavage would be unextendible, due to either mismatch or modification of the 3\u2032 end, could be activated as a primer when cleaved by an INVADER oligonucleotide-directed cleavage. A template directed tailing reaction also has the advantage of allowing greater selection and control of the nucleotides incorporated.","The use of nontemplated tailing does not require the presence of any additional nucleic acids in the detection reaction, avoiding one step of assay development and troubleshooting. In addition, the use of non-templated synthesis eliminated the step of hybridization, potentially speeding up the assay. Furthermore, the TdT enzyme is fast, able to add at least >700 nucleotides to substrate oligonucleotides in a 15 minute reaction.","As mentioned above, the tails added can be used in a number of ways. It can be used as a straight-forward way of adding labeled moieties to the cleavage product to increase signal from each cleavage event. Such a reaction is depicted in the left side of FIG. . The labeled moieties may be anything that can, when attached to a nucleotide, be added by the tailing enzyme, such as dye molecules, haptens such as digoxigenin, or other binding groups such as biotin.","In a preferred embodiment the assay includes a means of specifically capturing or partitioning the tailed INVADER oligonucleotide-directed cleavage products in the mixture. It can be seen that target nucleic acids in the mixture may be tailed during the reaction. If a label is added, it is desirable to partition the tailed INVADER oligonucleotide-directed cleavage products from these other labeled molecules to avoid background in the results. This is easily done if only the cleavage product is capable of being captured. For example, consider a cleavage assay of the present invention in which the probe used has a biotin on the 5\u2032 end and is blocked from extension on the 3\u2032 end, and in which a dye is added during tailing. Consider further that the products are to be captured onto a support via the biotin moiety, and the captured dye measured to assess the presence of the target nucleic acid. When the label is added by tailing, only the specifically cleaved probes will be labeled. The residual uncut probes can still bind in the final capture step, but they will not contribute to the signal. In the same reaction, nicks and cuts in the target nucleic acid may be tailed by the enzyme, and thus become dye labeled. In the final capture these labeled targets will not bind to the support and thus, although labeled, they will not contribute to the signal. If the final specific product is considered to consist of two portions, the probe-derived portion and the tail portion, it can be seen from this discussion that it is particularly preferred that, when the probe-derived portion is used for specific capture, whether by hybridization, biotin\/streptavidin, or other method, that the label be associated with the tail portion. Conversely, if a label is attached to the probe-derived portion, then the tail portion may be made suitable for capture, as depicted on the right side of FIG. . Tails may be captured in a number of ways, including hybridization, biotin incorporation with streptavidin capture, or by virtue if the fact that the longer molecules bind more predictably and efficiently to a number of nucleic acid minding matrices, such as nitrocellulose, nylon, or glass, in membrane, paper, resin, or other form. While not required for this assay, this separation of functions allows effective exclusion from signal of both unreacted probe and tailed target nucleic acid.","In addition to the supports described above, the tailed products may be captured onto any support that contains a suitable capture moiety. For example, biotinylated products are generally captured with avidin-treated surfaces. These avidin surfaces may be in microtitre plate wells, on beads, on dipsticks, to name just a few of the possibilities. Such surfaces can also be modified to contain specific oligonucleotides, allowing capture of product by hybridization. Capture surfaces as described herein are generally known to those skilled in the art and include nitrocellulose dipsticks (e.g., GENECOMB, BioRad, Hercules, Calif.).","In addition to the DNA polymerase tailing reaction described above, the present invention also contemplates the use of the products of the invasive cleavage reaction to form activated protein binding sites, such as RNA polymerase promoter duplexes, thereby allowing the interaction of the completed site to be used as an indicator of the presence of the nucleic acid that is the target of the invasive cleavage reaction. By way of example, when an RNA polymerase promoter duplex is activated by being made complete (i.e., double-stranded over that portion of the promoter region required for polymerase binding) through the hybridization of the oligonucleotide product of the invasive cleavage reaction, the synthesis of RNA can be used as such an indicator.","It is not intended that the transcription reaction of the present invention be limited to the use of any particular RNA polymerase or RNA polymerase promoter region. Promoter sequences are well characterized for several bacteriophage, including bacteriophage SP6, T7 and T3. In addition, promoter sequences have been well characterized for a number of both eukaryotic and prokaryotic RNA polymerases. In a preferred embodiment, the promoter used enables transcription from one of the bacteriophage RNA polymerases. In a particularly preferred embodiment, the promoter used enables transcription by T7 RNA polymerase. Means of performing transcription in vitro are well known in the art and commercial kits are available for performing transcription with eukaryotic, prokaryotic or bacteriophage RNA polymerases (e.g., from Promega Corp., Madison, Wis.).","The protein binding regions of the present invention are not limited to the bacteriophage RNA polymerase promoters described above. Other promoter sequences that are contemplated are those of prokaryotes and eukaryotes. For example, many strains of bacteria and fungi are used for the expression of heterologous proteins. The minimal promoters required for transcription by the RNA polymerases of organisms such as yeast and other fungi, eubacteria, nematodes, and cultured mammalian cells are well described in the literature and in the catalogs of commercial suppliers of DNA vectors for the expression of foreign proteins in these organisms.","The binding sites for other types of nucleic acid (e.g., DNA) binding proteins are contemplated for use in the present invention. For example, proteins involved in the regulation of genes exert their effects by binding to the DNA in the vicinity of the promoter from which the RNA from that gene is transcribed. The lac operator of is one example of a particularly well characterized and commonly used gene regulation system in which the lac repressor protein binds to specific sequences that overlap, and thus block, the promoter for the genes under the repressor's control (Jacob and Monod, Cold Spring Harbor Symposium on Quantitative Biol. XXVI:193-211 [1961]). Many similar systems have been described in bacteria, including the trp and AraC regulatory systems. Given the large amount of information available about bacterial promoters, the steps described below for the design of suitable partial promoters for the bacteriophage RNA polymerases can be readily adapted to the design of detection systems based on these other promoters.","As noted above, many of the bacterial promoters are under the control of a repressor or other regulatory protein. It is considered to be within the scope of the present invention to include the creation of composite binding sites for these regulatory proteins through the provision of a nucleic acid fragment (e.g., a non-target cleavage product generated in an invasive cleavage reaction). The binding of the regulatory protein to the completed protein binding region (e.g., the composite binding region) can be assessed by any one of a number of means, including slowed electrophoretic migration of either the protein or the DNA fragment, or by a con formational change in the protein or DNA upon binding. In addition, transcription from a downstream promoter can be monitored for up- or down-regulation as a result of the binding of the regulatory protein to the completed protein binding region.","In addition to the bacterial systems described above, many genes in eukaryotic systems have also been found to be under the control of specific proteins that bind to specific regions of duplex DNA. Examples include, but are not limited to, the OCT-1, OCT-2 and AP-4 proteins in mammals and the GAL4 and GCN4 proteins in yeast. Such regulatory proteins usually have a structural motif associated with duplex nucleic acid binding, such as a helix-turn-helix, a zinc finger or a leucine zipper [for review, see, , Wolfe (Ed.), Wadsworth Publishing Co., Belmont, Calif., pp. 694-715 [1993]).","For simplicity the test reaction described here will refer to T7 RNA polymerase, and its promoter. This is not intended to limit the invention to the use of this RNA polymerase, and those skilled in the art of molecular biology would be able to readily adapt this described test to the examination of any of the DNA binding proteins, RNA polymerascs and their binding or promoter sites discussed above.","It is known in the art that active T7 promoters can be formed by the hybridization of two oligonucleotides, each comprising either the top or bottom strand of the promoter sequence, such that a complete un-nicked duplex promoter is formed (Milligan et al., Nucl. Acids Res., 15:21, 8783-8798 (1987)]. The present invention shows that one way of making the initiation of transcription dependent on the products of an invasive cleavage reaction is to design the probe for the cleavage reaction such that a portion of an RNA polymerase promoter is released as product. The remaining DNA piece or pieces required to assemble a promoter duplex may either be provided as elements in the reaction mixture, or they may be produced by other invasive cleavage events. If the oligonucleotide pieces are designed to comprise appropriate regions of complementarity they may base pair to form a complete promoter duplex composed of three or more nucleic acid fragments, as depicted in  promoter assembled in this way will have nicks in the backbone of one or both strands. In one embodiment, these nicks may be covalently closed through the use of a DNA ligase enzyme. In a preferred embodiment, the nicks are positioned such that transcription can proceed without ligation. In selecting the site of a nick created by the assembly of the partial promoter fragment, at least one nick should be within the recognized promoter region for the RNA polymerase to be used. When a bacteriophage promoter is used, a nick should be between nucleotides \u221217 and \u22121, measured from the site of transcription initiation at +1. In a preferred embodiment, a nick will be between nucleotides \u221213 and \u22128. In a particularly preferred embodiment, a nick will be between nucleotides \u221212 and \u221210 on the non-template strand of the bacteriophage promoter.","When nicks are to be left unrepaired (i.e., not covalently closed with a DNA ligase) it is important to assess the effect of the nick location on the level of transcription from the assembled promoter. A simple test is to combine the oligonucleotides that comprise the separate portions of the promoter with an oligonucleotide that comprises one entire strand of the promoter to be assembled, thereby forming a duplex promoter with a nick in one strand. If the nick is in the top, or non-template strand of the promoter, then the oligonucleotide that comprises the complete promoter is made to include additional non-promoter sequence on its 5\u2032 end to serve as a template to be copied in the transcription. This arrangement is depicted in FIG. B. Alternatively, if the nick is to be in the bottom, or template strand of the promoter, then the partial promoter oligonucleotide that covers the +1 position, the transcription start site, will include the additional template sequence. This arrangement is depicted in  (this Figure shows several different embodiments in which a cut probe or non-target cleavage product is used to form a composite promoter which contains one or more nicks on the template strand). In either case, the separate oligonucleotides are combined to form the complete promoter, and the assembly is used in a transcription reaction to create RNA.","To measure the effect of the nick, a substantially identical promoter fragment is created by hybridization of two oligonucleotides that each comprise one strand of the full-length promoter to create an un-nicked version of the same promoter. These two molecular assemblies are tested in parallel transcription reactions and the amount of the expected RNA that is produced in each reaction is measured for both size and yield. A preferred method of assessing the size of the RNA is by electrophoresis with subsequent visualization. If a labeled nucleotide (e.g., P-GTP, or fluorescein-UTP) is used in the transcription, the RNA can be detected and quantitated by autoradiography, fluorescence imaging or by transfer to support membrane with subsequent detection (e.g., by antibody or hybridization probing). Alternatively, if unlabeled RNA is produced the amounts may be determined by other methods known in the art, such as by spectrophotometry or by electrophoresis with subsequent staining and comparison to known standards.","If the size of the RNA is as predicted by the template sequence, or if it matches that produced from the control promoter, it can be presumed to have initiated transcription at the same site in the complex, and to have produced essentially the same RNA product. If the product is much shorter then transcription is either initiating at an internal site or is terminating prematurely (Schenbom and Mierendorf, Nucl. Acids Res., 13:17, 6223 [1985]; and Milligan et al., supra.). While this does not indicate that the assembly tested is completely unsuitable for the assay, the partial transcripts will reduce the gross amount of RNA created, perhaps compromising the signal from the assay, and such products would require further characterization (e.g., finger printing or sequencing) to identify the nucleotide content of the product. It is preferred that the size of the RNA produced matches that of the RNA produced in the control reaction.","The yield of the reaction is also examined. It is not necessary that the level of transcription matches that of the control reaction. In some instances (see Ex. 41, below) the nicked promoter may have an enhanced rate of transcription, while in other arrangements transcription may be reduced (relative to the rate from the un-nicked promoter assembly). It is only required that the amount of product be within the detection limits of the method to be used with the test promoter.","It is reported that transcription from a bacteriophage promoter can produce 200 to 1000 copies of each transcription template (template plus active promoter) in a reaction. These levels of transcription are not required by the present invention. Reactions in which one RNA is produced for each template are also contemplated.","The test described above will allow a promoter with a nick in any position to be assessed for utility in this assay. It is an objective of this invention to provide one or more of the oligonucleotides that comprise a partial promoter region through invasive cleavage event(s). In this embodiment, the partial promoter sequences are attached to the probe oligonucleotide in the invasive cleavage assay, and are released by cleavage at specific site, as directed by the INVADER oligonucleotide. It is also intended that transcription be very poor or nonexistent in the absence of the correctly cleaved probe. To assess the success of any oligonucleotide design at meeting these objectives, several transcription reaction tests can be performed.","For a promoter assembly that will have a nick on the non-template strand, several partial assemblies that should be tested are shown in . By way of example, but not by way of limitation, this Figure depicts the tests for a nicked promoter in which the upstream, or 5\u2032 portion of the non-template strand is to be provided by the invasive cleavage assay. This fragment is seen in  labeled as \u201ccut probe\u201d. Transcription reactions incubated in the presence of the duplex shown in  will test the ability of the upstream partial promoter to allow initiation of transcription when hybridized to a bottom strand, termed a \u201ccopy template.\u201d Similarly, a reaction performed in the presence of the duplex depicted in  will test the ability of the partial promoter fragment nearest the initiation site (the +1 site, as indicated in ) to support transcription of the copy template. It is an important feature of the present invention that neither of these partial promoter duplexes be able to support transcription at the same level as would by seen in transcription from an intact promoter as depicted in FIG. B. It is preferred that neither of these partial promoters be sufficient to initiate detectable transcription in the time course of an average transcription reaction (i.e., within about an hour of incubation).",{"@attributes":{"id":"p-0485","num":"0494"},"figref":["FIGS. 86C and 86D","FIG. 86C","FIG. 86D"],"b":"85"},"In one embodiment of the transcription system of the present invention, the initiation of transcription from the copy template in the absence of a complete promoter, or in the presence of a branched promoter, is prevented by the judicious placement of the nick or nicks in the composite promoter. For example, as shown in the examples below, placement of a nick between the \u221212 and \u221211 nucleotides of the non-template strand of the bacteriophage T7 promoter allows transcription to take place only when the probe has been successfully cut, as in an invasive cleavage reaction. However, in some instances where the invasive cleavage reaction is to provide the upstream portion of the non-template strand of the promoter (e.g., as depicted in ) it may be necessary or desirable to place the nick on that strand in a particular position for reasons other than providing an optimal composite promoter (i.e., one that is inactive in the absence of any one of the promoter pieces). It may be necessary or desirable to place the nick in such a way that the creation of a branched complete promoter () has an undesirable level of transcription, reducing dependence of RNA production on the success of the invasive cleavage step. It is shown in the examples below that transcription from such a branched promoter can be suppressed by a modification of the downstream non-template promoter piece, shown as the \u201cPartial Promoter Oligonucleotide\u201d in , ,  and D. As depicted in , the partial promoter oligonucleotide can be provided with a 5\u2032 \u201ctail\u201d of nucleotides that are not complementary to the template strand of the promoter, but that are complementary to the 3\u2032 portion of the probe oligonucleotide that would be removed in the invasive cleavage reaction. When uncut probe hybridizes to the copy template with the bound 5\u2032 tailed partial promoter oligonucleotide, the 5\u2032 tail can basepair to the 3\u2032 region of the probe, forming a three-way junction as depicted in FIG. A. This can effectively shut off transcription, as shown below. When a cut probe hybridizes, as shown in , a promoter with a small branch is formed, and it is shown herein that such a branched promoter can initiate transcription. Furthermore, if care is taken in selecting the sequence of the 5\u2032 tail (i.e., if the first unpaired base is the same nucleotide at the 3\u2032 nucleotide of the cut probe, so that they compete for hybridization to the same template strand base), the resulting branched structure may also be cleaved by one of the structure specific nucleases of the present invention, creating the un-branched promoter depicted in , in some instances enhancing transcription over that seen with the  promoter.","The promoter duplex that is intended to be created, in this embodiment, by the successful execution of the INVADER directed cleavage assay will include both the \u201ccut probe\u201d and the partial promoter oligonucleotide depicted in , aligned on a single copy template nucleic acid. The testing of the efficiency of transcription of such a nicked promoter segment in comparison to the intact promoter is described above. All of the oligonucleotides described for these test molecules may be created using standard synthesis chemistries.","The set of test molecules depicted in  is designed to assess the transcription capabilities of the variety of structures that may be present in reactions in which the 5\u2032 portion of the non-template strand of the promoter is to be supplied by the INVADER directed cleavage. It is also envisioned that a different portion of partial promoter may be supplied by the invasive cleavage reaction (e.g., the downstream segment of the non-template strand of the promoter), as is shown in FIG. . Portions of the template strand of the promoter may also be provided by the cut probe, as shown in . An analogous set of test molecules, including \u201ccut\u201d and uncut versions of the probe to be used in the invasive cleavage assay may be created to test any alternative design, whether the nick is to be located on the template or non template strand of the promoter.","The transcription-based visualization methods of the present invention may also be used in a multiplex fashion. Reactions can be constructed such that the presence of one particular target leads to transcription from one type of promoter, while the presence of a different target sequence (e.g., a mutant or variant) or another target suspected of being present, may lead to transcription from a different (i.e., a second) type of promoter. In such an embodiment, the identity of the promoter from which transcription was initiated could be deduced from the type or size of the RNA produced.","By way of example, but not by way of limitation, the bacteriophage promoters can be compared with such an application in view. The promoters for the phage T7, T3 and SP6 are quite similar, each being about 15 to 20 basepairs long, and sharing about 45% identity between \u221217 and \u22121 nucleotides, relative to the start of transcription. Despite these similarities, the RNA polymerases from these phage are highly specific for their cognate promoters, such that the other promoters may be present in a reaction, but will not be transcribed (Chamberlin and Ryan, Enzymes XV:87-108 [1982]). Because these promoters are similar in size and in the way in which they are recognized by their polymerases (Li et al., Biochem. 35:3722 [1996]) similar nicked versions of the promoters may be designed for use in the methods of the present invention by analogy to the examples described herein which employ the T7 promoter. Because of the high degree of specificity of the RNA polymerases, these nicked promoters may be used together to detect multiple targets in a single reaction. There are many instances in which it would be highly desirable to detect multiple nucleic acid targets in a single sample, including cases in which multiple infectious agents may be present, or in which variants of a single type of target may need to be identified. Alternatively, it is often desirable to use a combination of probes to detect both a target sequence and an internal control sequence, to gauge the effects of sample contaminants on the output of the assay. The use of multiple promoters allows the reaction to be assessed for both the efficiency of the invasive cleavage and the robustness of the transcription.","As stated above, the phage promoters were described in detail as an example of suitable protein binding regions (e.g., which can be used to generate a composite promoter) for use in the methods of the present invention. The invention is not limited to the use of phage RNA polymerase promoter regions, in particular, and RNA polymerase promoter regions, in general. Suitably specific, well characterized promoters are also found in both prokaryotic and eukaryotic systems.","The RNA that is produced in a manner that is dependent of the successful detection of the target nucleic acid in the invasive cleavage reaction may be detected in any of several ways. If a labeled nucleotide is incorporated into the RNA during transcription, the RNA may be detected directly after fractionation (e.g., by electrophoresis or chromatography). The labeled RNA may also be captured onto a solid support, such as a microtitre plate, a bead or a dipstick (e.g., by hybridization, antibody capture, or through an affinity interaction such as that between biotin and avidin). Capture may facilitate the measuring of incorporated label, or it may be an intermediate step before probe hybridization or similar detection means. If the maximum amount of label is desired to be incorporated into each transcript, it is preferred that the copy template be very long, around 3 to 10 kilobases, so that each RNA molecule will carry many labels. Alternatively, it may be desired that a single site or a limited number of sites within the transcript be specifically labeled. In this case, it may be desirable to have a short copy template with only one or a few residues that would allow incorporation of the labeled nucleotide.","The copy template may also be selected to produce RNAs that perform specified functions. In a simple case, if an duplex-dependent intercalating fluorophore is to be used to detect the RNA product, it may be desirable to transcribe an RNA that is known to form duplexed secondary structures, such as a ribosomal RNA or a tRNA. In another embodiment, the RNA may be designed to interact specifically, or with particular affinity, with a different substance. It has been shown that a process of alternating steps of selection (e.g., by binding to a target substance) and in vitro amplification (e.g., by PCR) can be used to identify nucleic acid ligands with novel and useful properties (Tuerk and Gold, Science 249:505 [1990]). This system has been used to identify RNAs, termed ligands or aptamers, that bind tightly and specifically to proteins and to other types of molecules, such as antibiotics (Wang et al., Biochem. 35:12338 [1996]) and hormones. RNAs can even be selected to bind to other RNAs through non-Watson-Crick interactions (Schmidt et al., Ann. N.Y. Acad. Sci. 782:526 [1996]). A ligand RNA may be used to either inactivate or enhance the activity of a molecule to which it binds. Any RNA segment identified through such a process may also be produced by the methods of the present invention, so that the observation of the activity of the RNA ligand may be used as a specific sign of the presence of the target material in the invasive cleavage reaction. The ligand binding to its specific partner may also be used as another way of capturing a readout signal to a solid support.","The product RNA might also be designed to have a catalytic function (e.g., to act as a ribozyme), allowing cleavage another molecule to be indicative of the success of the primary invasive cleavage reaction (Uhlenbeck, Nature 328:596 [1987]). In yet another embodiment, the RNA may be made to encode a peptide sequence. When coupled to an in vitro translation system (e.g., the S-30 system derived from [Lesley, Methods Mol. Biol., 37:265 (1985)], or a rabbit reticulocyte lysate system [Dasso and Jackson, Nucleic Acids Res. 17:3129 (1989)], available from Promega), the production of the appropriate protein may be detected. In a preferred embodiment, the proteins produced include those that allow either colorimetric or luminescent detection, such as beta-galactosidase (lac-Z) or luciferase, respectively.","The above discussion focused on the use of the present transcription visualization methods in the context of the INVADER-directed cleavage assay (i.e., the non-target cleavage products produced in the INVADER assay were used to complete and activate a protein binding region, such as a promoter region). However, the transcription visualization methods are not limited to this context. Any assay that produces an oligonucleotide product having relatively discrete ends can be used in conjunction with the present transcription visualization methods. For example, the homogenous assay described in U.S. Pat. No. 5,210,015, particularly when conducted under conditions where polymerization cannot occur, produces short oligonucleotide fragments as the result of cleavage of a probe. If this assay is conducted under conditions where polymerization occurs, the site of cleavage of the probe may be focused through the use of nucleotide analogs that have uncleavable linkages at particular positions within the probe. These short oligonucleotides can be employed in a manner analogous to the cut probe or non-target cleavage products produced in the invasive cleavage reactions of the present invention. Additional assays that generate suitable oligonucleotide products are known to the art. For example, the non-target cleavage products produced in assays such as the \u201cCycling Probe Reaction\u201d (Duck et al., BioTech., 9:142 [1990] and U.S. Pat. Nos. 4,876,187 and 5,011,769, herein incorporated by reference), in which shorter oligonucleotides are released from longer oligonucleotides after hybridization to a target sequence would be suitable, as would short restriction fragments released in assays where a probe is designed to be cleaved when successfully hybridized to an appropriate restriction recognition sequence (U.S. Pat. No. 4,683,194, herein incorporated by reference).","Assays that generate short oligonucleotides having \u201cragged\u201d (i.e., not discrete) 3\u2032 ends can also be employed with success in the transcription reactions of the present invention when the oligonucleotide provided by this non-transcription reaction are used to provide a portion of the promoter region located downstream of the other oligonucleotide(s) that are required to complete the promoter region (that is a 3\u2032 tail or unpaired extension can be tolerated when the oligonucleotide is being used as the \u201cCut Probe\u201d is in FIGS.  and A).","The 5\u2032 nucleases of the invention form the basis of a novel detection assay for the identification of specific nucleic acid sequences.  provides a schematic of one embodiment of the detection method of the present invention. The target sequence is recognized by two distinct oligonucleotides in the triggering or trigger reaction. In a preferred embodiment, one of thesc oligonucleotides is provided on a solid support. The other can be provided free in solution. In  the free oligonucleotide is indicated as a \u201cprimer\u201d and the other oligonucleotide is shown attached to a bead designated as type 1. The target nucleic acid aligns the two oligonucleotides for specific cleavage of the 5\u2032 arm (of the oligonucleotide on bead 1) by the 5\u2032 nucleases of the present invention (not shown in FIG. A). The site of cleavage (indicated by a large solid arrowhead) is controlled by the position of the 3\u2032 end of the \u201cprimer\u201d relative to the downstream fork of the oligonucleotide on bead 1.","Successful cleavage releases a single copy of what is referred to as the alpha signal oligonucleotide. This oligonucleotide may contain a detectable moiety (e.g., fluorescein). On the other hand, it may be unlabeled.","In one embodiment of the detection method, two more oligonucleotides are provided on solid supports. The oligonucleotide shown in FIG. IA on bead 2 has a region that is complementary to the alpha signal oligonucleotide (indicated as alpha prime) allowing for hybridization. This structure can be cleaved by the 5\u2032 nucleases of the present invention to release the beta signal oligonucleotide. The beta signal oligonucleotide can then hybridize to type 3 beads having an oligonucleotide with a complementary region (indicated as beta prime). Again, this structure can be cleaved by the 5\u2032 nucleases of the present invention to release a new alpha oligonucleotide.","Up to this point, the amplification has been linear. To increase the power of the method, it is desired that the alpha signal oligonucleotide hybridized to bead type 2 be liberated after release of the beta oligonucleotide so that it may go on to hybridize with other oligonucleotides on type 2 beads. Similarly, after release of an alpha oligonucleotide from type 3 beads, it is desired that the beta oligonucleotide be liberated.","With the liberation of signal oligonucleotides by such techniques, each cleavage results in a doubling of the number of signal oligonucleotides. In this manner, detectable signal can quickly be achieved.",{"@attributes":{"id":"p-0502","num":"0511"},"figref":"FIG. 1B","b":"1"},"The second part of the detection method allows the annealing of the fragment of the second oligonucleotide liberated by the cleavage of the first cleavage structure formed in the triggering reaction (called the third or trigger oligonucleotide) to a first hairpin structure. This first hairpin structure has a single-stranded 5\u2032 arm and a single-stranded 3\u2032 arm. The third oligonucleotide triggers the cleavage of this first hairpin structure by annealing to the 3\u2032 arm of the hairpin thereby forming a substrate for cleavage by the 5\u2032 nuclease of the present invention. The cleavage of this first hairpin structure generates two reaction products: 1) the cleaved 5\u2032 arm of the hairpin called the fourth oligonucleotide, and 2) the cleaved hairpin structure that now lacks the 5\u2032 arm and is smaller in size than the uncleaved hairpin. This cleaved first hairpin may be used as a detection molecule to indicate that cleavage directed by the trigger or third oligonucleotide occurred. Thus, this indicates that the first two oligonucleotides found and annealed to the target sequence thereby indicating the presence of the target sequence in the sample.","The detection products may be amplified by having the fourth oligonucleotide anneal to a second hairpin structure. This hairpin structure has a 5\u2032 single-stranded arm and a 3\u2032 single-stranded arm. The fourth oligonucleotide generated by cleavage of the first hairpin structure anneals to the 3\u2032 arm of the second hairpin structure thereby creating a third cleavage structure recognized by the 5\u2032 nuclease. The cleavage of this second hairpin structure also generates two reaction products: 1) the cleaved 5\u2032 arm of the hairpin called the fifth oligonucleotide, and 2) the cleaved second hairpin structure which now lacks the 5\u2032 arm and is smaller in size than the uncleaved hairpin. In one embodiment, the fifth oligonucleotide is similar or identical in sequence to the third nucleotide. The cleaved second hairpin may be viewed as a detection molecule that amplifies the signal generated by the cleavage of the first hairpin structure. Simultaneously with the annealing of the forth oligonucleotide, the third oligonucleotide is dissociated from the cleaved first hairpin molecule so that it is free to anneal to a new copy of the first hairpin structure. The disassociation of the oligonucleotides from the hairpin structures may be accomplished by heating or other means suitable to disrupt base-pairing interactions. As described above, conditions may be selected that allow the association and disassociation of hybridized oligonucleotides without temperature cycling.","If fifth oligonucleotide is similar or identical in sequence to the third oligonucleotide, further amplification of the detection signal is achieved by annealing the fifth oligonucleotide to another molecule of the first hairpin structure. Cleavage is then performed and the oligonucleotide that is liberated then is annealed to another molecule of the second hairpin structure. Successive rounds of annealing and cleavage of the first and second hairpin structures, provided in excess, are performed to generate a sufficient amount of cleaved hairpin products to be detected.","As discussed above for other embodiments of detection using structure-specific nuclease cleavage, any method known in the art for analysis of nucleic acids, nucleic acid fragments or oligonucleotides may be applied to the detection of these cleavage products.","The hairpin structures may be attached to a solid support, such as an agarose, styrene or magnetic bead, via the 3\u2032 end of the hairpin. A spacer molecule may be placed between the 3\u2032 end of the hairpin and the bead, if so desired. The advantage of attaching the hairpin structures to a solid support is that this prevents the hybridization of the two hairpin structures to one another over regions which are complementary. If the hairpin structures anneal to one another, this would reduce the amount of hairpins available for hybridization to the primers released during the cleavage reactions. If the hairpin structures are attached to a solid support, then additional methods of detection of the products of the cleavage reaction may be employed. These methods include, but are not limited to, the measurement of the released single-stranded 5\u2032 arm when the 5\u2032 arm contains a label at the 5\u2032 terminus. This label may be radioactive, fluorescent, biotinylated, etc. If the hairpin structure is not cleaved, the 5\u2032 label will remain attached to the solid support. If cleavage occurs, the 5\u2032 label will be released from the solid support.","The 3\u2032 end of the hairpin molecule may be blocked through the use of dideoxynucleotides. A 3\u2032 terminus containing a dideoxynucleotide is unavailable to participate in reactions with certain DNA modifying enzymes, such as terminal transferase. Cleavage of the hairpin having a 3\u2032 terminal dideoxynucleotide generates a new, unblocked 3\u2032 terminus at the site of cleavage. This new 3\u2032 end has a free hydroxyl group that can interact with terminal transferase thus providing another means of detecting the cleavage products.","The hairpin structures are designed so that their self-complementary regions are very short (generally in the range of 3-8 base pairs). Thus, the hairpin structures are not stable at the high temperatures at which this reaction is performed (generally in the range of 50-75\u00b0 C.) unless the hairpin is stabilized by the presence of the annealed oligonucleotide on the 3\u2032 arm of the hairpin. This instability prevents the polymerase from cleaving the hairpin structure in the absence of an associated primer thereby preventing false positive results due to non-oligonucleotide directed cleavage.","A cleavage structure is defined herein as a structure that is formed by the interaction of a probe oligonucleotide and a target nucleic acid to form a duplex, the resulting structure being cleavable by a cleavage agent, including but not limited to an enzyme. The cleavage structure is further defined as a substrate for specific cleavage by the cleavage means in contrast to a nucleic acid molecule that is a substrate for nonspecific cleavage by agents such as phosphodiesterases. Examples of some possible cleavage structures are shown in FIG. . In considering improvements to enzymatic cleavage agents, one may consider the action of said enzymes on any of these structures, and on any other structures that fall within the definition of a cleavage structure. The cleavage sites indicated on the structures in  are presented by way of example. Specific cleavage at any site within such a structure is contemplated.","Improvements in an enzyme may be an increased or decreased rate of cleavage of one or more types of structures. Improvements may also result in more or fewer sites of cleavage on one or more of said cleavage structures. In developing a library of new structure-specific nucleases for use in nucleic acid cleavage assays, improvements may have many different embodiments, each related to the specific substrate structure used in a particular assay.","As an example, one embodiment of the INVADER oligonucleotide-directed cleavage assay of the present invention may be considered. In the INVADER oligonucleotide-directed cleavage assay, the accumulation of cleaved material is influenced by several features of the enzyme behavior. Not surprisingly, the turnover rate, or the number of structures that can be cleaved by a single enzyme molecule in a set amount of time, is very important in determining the amount of material processed during the course of an assay reaction. If an enzyme takes a long time to recognize a substrate (e.g., if it is presented with a less-than-optimal structure), or if it takes a long time to execute cleavage, the rate of product accumulation is lower than if these steps proceeded quickly. If these steps are quick, yet the enzyme \u201cholds on\u201d to the cleaved structure, and does not immediately proceed to another uncut structure, the rate will be negatively affected.","Enzyme turnover is not the only way in which enzyme behavior can negatively affect the rate of accumulation of product. When the means used to visualize or measure product is specific for a precisely defined product, products that deviate from that definition may escape detection, and thus the rate of product accumulation may appear to be lower than it is. For example, if one had a sensitive detector for trinucleotides that could not see di- or tetranucleotides, or any sized oligonucleotide other that 3 residues, in the INVADER-directed cleavage assay of the present invention any errant cleavage would reduce the detectable signal proportionally. It can be seen from the cleavage data presented here that, while there is usually one site within a probe that is favored for cleavage, there are often products that arise from cleavage one or more nucleotides away from the primary cleavage site. These are products that are target-dependent, and are thus not non-specific background. Nevertheless, if a subsequent visualization system can detect only the primary product, these represent a loss of signal. One example of such a selective visualization system is the charge reversal readout presented herein, in which the balance of positive and negative charges determines the behavior of the products. In such a system the presence of an extra nucleotide or the absence of an expected nucleotide can excluded a legitimate cleavage product from ultimate detection by leaving that product with the wrong balance of charge. It can be easily seen that any assay that can sensitively distinguish the nucleotide content of an oligonucleotide, such as standard stringent hybridization, suffers in sensitivity when some fraction of the legitimate product is not eligible for successful detection by that assay.","These discussions suggest two highly desirable traits in any enzyme to be used in the method of the present invention. First, the more rapidly the enzyme executes an entire cleavage reaction, including recognition, cleavage and release, the more signal it may potentially created in the INVADER oligonucleotide-directed cleavage assay. Second, the more successful an enzyme is at focusing on a single cleavage site within a structure, the more of the cleavage product can be successfully detected in a selective read-out.","The rationale cited above for making improvements in enzymes to be used in the INVADER oligonucleotide-directed cleavage assay are meant to serve as an example of one direction in which improvements might be sought, but not as a limit on either the nature or the applications of improved enzyme activities. As another direction of activity change that would be appropriately considered improvement, the DNAP-associated 5\u2032 nucleases may be used as an example. In creating some of the polymerase-deficient 5\u2032 nucleases described herein it was found that the those that were created by deletion of substantial portions of the polymerase domain, as depicted in , assumed activities that were weak or absent in the parent proteins. These activities included the ability to cleave the non-forked structure shown in , a greatly enhanced ability to exonucleolytically remove nucleotides from the 5\u2032 ends of duplexed strands, and a nascent ability to cleave circular molecules without benefit of a free 5\u2032 end.","In addition to the 5\u2032 nucleases derived from DNA polymerases, the present invention also contemplates the use of structure-specific nuclcases that are not derived from DNA polymerases. For example, a class of eukaryotic and archaebacterial endonucleases have been identified which have a similar substrate specificity to 5\u2032 nucleases of Pol 1-type DNA polymerases. These are the FENI (Flap EndoNuclease), RAD2, and XPG (Xeroderma Pigmentosa-complementation group G) proteins. These proteins are involved in DNA repair, and have been shown to favor the cleavage of structures that resemble a 5\u2032 arm that has been displaced by an extending primer during polymerization, similar to the model depicted in FIG. B. Similar DNA repair enzymes have been isolated from single cell and higher eukaryotes and from archaea, and there are related DNA repair proteins in eubacteria. Similar 5\u2032 nucleases have also been associated with bacteriophage such as T5 and T7.","Recently, the 3-dimensional structures of DNAPTaq and T5 phage 5\u2032-exonuclease () were determined by X-ray diffraction (Kim et al., Nature 376:612 [1995]; and Ceska et al., Nature 382:90 [1995]). The two enzymes have very similar 3-dimensional structures despite limited amino acid sequence similarity. The most striking feature of the T5 5\u2032-exonuclease structure is the existence of a triangular hole formed by the active site of the protein and two alpha helices (FIG. ). This same region of DNAPTaq is disordered in the crystal structure, indicating that this region is flexible, and thus is not shown in the published 3-dimensional structure. However, the 5\u2032 nuclease domain of DNAPTaq is likely to have the same structure, based its overall 3-dimensional similarity to T5 5\u2032-exonuclease, and that the amino acids in the disordered region of the DNAPTaq protein are those associated with alpha helix formation. The existence of such a hole or groove in the 5\u2032 nuclease domain of DNAPTaq was predicted based on its substrate specificity (Lyamichev et al., supra).","It has been suggested that the 5\u2032 arm of a cleavage structure must thread through the helical arch described above to position said structure correctly for cleavage (Ceska et al., supra). One of the modifications of 5\u2032 nucleases described herein opened up the helical arch portion of the protein to allow improved cleavage of structures that cut poorly or not at all (e.g., structures on circular DNA targets that would preclude such threading of a 5\u2032 arm). The gene construct that was chosen as a model to test this approach was the one called CLEAVASE BN, which was derived from DNAPTaq but does not contain the polymerase domain (Ex. 2). It comprises the entire 5\u2032 nuclease domain of DNAP Taq, and thus should be very close in structure to the T5 5\u2032 exonuclease. This 5\u2032 nuclease was chosen to demonstrate the principle of such a physical modification on proteins of this type. The arch-opening modification of the present invention is not intended to be limited to the 5\u2032 nuclease domains of DNA polymerases, and is contemplated for use on any structure-specific nuclease that includes such an aperture as a limitation on cleavage activity. The present invention contemplates the insertion of a thrombin cleavage site into the helical arch of DNAPs derived from the genus as well as 5\u2032 nucleases derived from DNAPs derived from the genus . The specific example shown herein using the CLEAVASE BN\/thrombin nuclease merely illustrates the concept of opening the helical arch located within a nuclease domain. As the amino acid sequence of DNAPs derived from the genus are highly conserved, the teachings of the present invention enable the insertion of a thrombin site into the helical arch present in these DNAPs and 5\u2032 nucleases derived from these DNAPs.","The opening of the helical arch was accomplished by insertion of a protease site in the arch. This allowed post-translational digestion of the expressed protein with the appropriate protease to open the arch at its apex. Proteases of this type recognize short stretches of specific amino acid sequence. Such proteases include thrombin and factor Xa. Cleavage of a protein with such a protease depends on both the presence of that site in the amino acid sequence of the protein and the accessibility of that site on the folded intact protein. Even with a crystal structure it can be difficult to predict the susceptibility of any particular region of a protein to protease cleavage. Absent a crystal structure it must be determined empirically.","In selecting a protease for a site-specific cleavage of a protein that has been modified to contain a protease cleavage site, a first step is to test the unmodified protein for cleavage at alternative sites. For example, DNAPTaq and CLEAVASE BN nuclease were both incubated under protease cleavage conditions with factor Xa and thrombin proteases. Both nuclease proteins were cut with factor Xa within the 5\u2032 nuclease domain, but neither nuclease was digested with large amounts of thrombin. Thus, thrombin was chosen for initial tests on opening the arch of the CLEAVASE BN enzyme.","In the protease\/CLEAVASE modifications described herein the factor Xa protease cleaved strongly in an unacceptable position in the unmodified nuclease protein, in a region likely to compromise the activity of the end product. Other unmodified nucleases contemplated herein may not be sensitive to the factor Xa, but may be sensitive to thrombin or other such proteases. Alternatively, they may be sensitive to these or other such proteases at sites that are immaterial to the function of the nuclease sought to be modified. In approaching any protein for modification by addition of a protease cleavage site, the unmodified protein should be tested with the proteases under consideration to determine which proteases give acceptable levels of cleavage in other regions.","Working with the cloned segment of DNAPTaq from which the CLEAVASE BN protein is expressed, nucleotides encoding a thrombin cleavage site were introduced in-frame near the sequence encoding amino acid 90 of the nuclease gene. This position was determined to be at or near the apex of the helical arch by reference to both the 3-dimensional structure of DNAPTaq, and the structure of T5 5\u2032 exonuclease. The encoded amino acid sequence, LVPRGS, was inserted into the apex of the helical arch by site-directed mutagenesis of the nuclease gene. The proline (P) in the thrombin cleavage site was positioned to replace a proline normally in this position in CLEAVASE BN because proline is an alpha helix-breaking amino acid, and may be important for the 3-dimensional structure of this arch. This construct was expressed, purified and then digested with thrombin. The digested enzyme was tested for its ability to cleave a target nucleic acid, bacteriophage M13 gcnomic DNA, that does not provide free 5\u2032 ends to facilitate cleavage by the threading model.","While the helical arch in this nuclease was opened by protease cleavage, it is contemplated that a number of other techniques could be used to achieve the same end. For example, the nucleotide sequence could be rearranged such that, upon expression, the resulting protein would be configured so that the top of the helical arch (amino acid 90) would be at the amino terminus of the protein, the natural carboxyl and amino termini of the protein sequence would be joined, and the new carboxyl terminus would lie at natural amino acid 89. This approach has the benefit that no foreign sequences arc introduced and the enzyme is a single amino acid chain, and thus may be more stable that the cleaved 5\u2032 nuclease. In the crystal structure of DNAPTaq, the amino and carboxyl termini of the 5\u2032-exonuclease domain lie in close proximity to each other, which suggests that the ends may be directly joined without the use of a flexible linker peptide sequence as is sometimes necessary. Such a rearrangement of the gene, with subsequent cloning and expression could be accomplished by standard PCR recombination and cloning techniques known to those skilled in the art.","The present invention also contemplates the use of nucleases isolated from organisms that grow under a variety of conditions. The genes for the FEN-1\/XPG class of enzymes are found in organisms ranging from bacteriophage to humans to the extreme thermophiles of Kingdom Archaea. For assays in which high temperature is to be used, it is contemplated that enzymes isolated from extreme thermophiles may exhibit the thermostability required of such an assay. For assays in which it might be desirable to have peak enzyme activity at moderate temperature or in which it might be desirable to destroy the enzyme with elevated temperature, those enzymes from organisms that favor moderate temperatures for growth may be of particular value.","An alignment of a collection of FEN-1 proteins sequenced by others is shown in  (SEQ ID NOS:135-145). It can be seen from this alignment that there are some regions of conservation in this class of proteins, suggesting that they are related in function, and possibly in structure. Regions of similarity at the amino acid sequence level can be used to design primers for in vitro amplification (PCR) by a process of back translating the amino acid sequence to the possible nucleic acid sequences, then choosing primers with the fewest possible variations within the sequences. These can be used in low stringency PCR to search for related DNA sequences. This approach permits the amplification of DNA encoding a FEN-1 nuclease without advance knowledge of the actual DNA sequence.","It can also be seen from this alignment that there are regions in the sequences that are not completely conserved. The degree of difference observed suggests that the proteins may have subtle or distinct differences is substrate specificity. In other words, they may have different levels of cleavage activity on the cleavage structures of the present invention. When a particular structure is cleaved at a higher rate than the others, this is referred to a preferred substrate, while a structure that is cleaved slowly is considered a less preferred substrate. The designation of preferred or less preferred substrates in this context is not intended to be a limitation of the present invention. It is contemplated that some embodiments the present invention will make use of the interactions of an enzyme with a less preferred substrate. Candidate enzymes are tested for suitability in the cleavage assays of the present invention using the assays described below.","1. Structure Specific Nuclease Assay","Testing candidate nucleases for structure-specific activities in these assays is done in much the same way as described for testing modified DNA polymerases in Example 2, but with the use of a different library of model structures. In addition to assessing the enzyme performance in primer-independent and primer-directed cleavage, a set of synthetic hairpins are used to examine the length of duplex downstream of the cleavage site preferred by the enzyme.","The FEN-1 and XPG 5\u2032 nucleases used in the present invention should be tested for activity in the assays in which they are intended to be used, including but not limited to the INVADER-directed cleavage detection assay of the present invention and the CFLP method of characterizing nucleic acids (the CFLP method is described in U.S. Pat. Nos. 5,843,654, 5,843,669, 5,719,028, and 5,888,780 and PCT Publication WO 96\/15267; the disclosures of which are incorporated herein by reference). The INVADER assay uses a mode of cleavage that has been termed \u201cprimer directed\u201d of \u201cprimer dependent\u201d to reflect the influence of the an oligonucleotide hybridized to the target nucleic acid upstream of the cleavage site. In contrast, the CFLP reaction is based on the cleavage of folded structure, or hairpins, within the target nucleic acid, in the absence of any hybridized oligonucleotide. The tests described herein are not intended to be limited to the analysis of nucleases with any particular site of cleavage or mode of recognition of substrate structures. It is contemplated that enzymes may be described as 3\u2032 nucleases, utilizing the 3\u2032 end as a reference point to recognize structures, or may have a yet a different mode of recognition. Further, the use of the term 5\u2032 nucleases is not intended to limit consideration to enzymes that cleave the cleavage structures at any particular site. It refers to a general class of enzymes that require some reference or access to a 5\u2032 end to effect cleavage of a structure.","A set of model cleavage structures have been created to allow the cleavage ability of unknown enzymes on such structures to be assessed. Each of the model structures is constructed of one or more synthetic oligonucleotides made by standard DNA synthesis chemistry. Examples of such synthetic model substrate structures are shown in . These are intended only to represent the general folded configuration desirable is such test structures. While a sequence that would assume such a structure is indicated in the Figures, there are numerous other sequence arrangements of nucleotides that would be expected to fold in such ways. The essential features to be designed into a set of oligonucleotides to perform the tests described herein are the presence or absence of a sufficiently long 3\u2032 arm to allow hybridization of an additional nucleic acid to test cleavage in a \u201cprimer-directed\u201d mode, and the length of the duplex region. In the set depicted in , the duplex lengths of the S-33 and the 11-8-0 structures are 12 and 8 basepairs, respectively. This difference in length in the test molecules facilitates detection of discrimination by the candidate nuclease between longer and shorter duplexes. Additions to this series expanding the range of duplex molecules presented to the enzymes, both shorter and longer, may be used. The use of a stabilizing DNA tetraloop (Antao et al., Nucl. Acids Res., 19:5901 [1991]) or triloop (Hiraro et al., Nuc. Acids Res., 22:576 [1994]) at the closed end of the duplex helps ensure formation of the expected structure by the oligonucleotide.","The model substrate for testing primer directed cleavage, the \u201cS-60 hairpin\u201d (SEQ ID NO:40) is described in Example 11. In the absence of a primer this hairpin is usually cleaved to release 5\u2032 arm fragments of 18 and 19 nucleotides length. An oligonucleotide, termed P-14 (5\u2032-CGAGAGACCACGCT-3\u2032; SEQ ID NO:108), that extends to the base of the duplex when hybridized to the 3\u2032 arm of the S-60 hairpin gives cleavage products of the same size, but at a higher rate of cleavage.","To test invasive cleavage a different primer is used, termed P-15 (5\u2032-CGAGAGACCACGCTG-3\u2032; SEQ ID NO:30). In a successful invasive cleavage the presence of this primer shifts the site of cleavage of S-60 into the duplex region, usually releasing products of 21 and 22 nucleotides length.","The S-60 hairpin may also be used to test the effects of modifications of the cleavage structure on either primer-directed or invasive cleavage. Such modifications include, but are not limited to, use of mismatches or base analogs in the hairpin duplex at one, a few or all positions, similar disruptions or modifications in the duplex between the primer and the 3\u2032 arm of the S-60, chemical or other modifications to one or both ends of the primer sequence, or attachment of moieties to, or other modifications of the 5\u2032 arm of the structure. In all of the analyses using the S-60 or a similar hairpin described herein, activity with and without a primer may be compared using the same hairpin structure.","The assembly of these test reactions, including appropriate amounts of hairpin, primer and candidate nuclease arc described in Example 2. As cited therein, the presence of cleavage products is indicated by the presence of molecules which migrate at a lower molecular weight than does the uncleaved test structure. When the reversal of charge of a label is used the products will carry a different net charge than the uncleaved material. Any of these cleavage products indicate that the candidate nuclease has the desired structure-specific nuclease activity. By \u201cdesired structure-specific nuclease activity\u201d it is meant only that the candidate nuclease cleaves one or more test molecules. It is not necessary that the candidate nuclease cleave at any particular rate or site of cleavage to be considered successful cleavage.","The following description provides illustrative examples of target sequence detection through the use of the compositions and methods of the present invention. These example include the detection of human cytomegalovirus viral DNA, single nucleotide polymorphisms in the human apolipoprotein E gene, mutations in the human hemochromatosis gene, mutations in the human MTHFR, prothrombin 20210GA polymorphism, the HR-2 mutation in the human Factor V gene, single nucleotide polymorphisms in the human TNF-\u03b1 Gene, and Leiden mutation in the human Factor V gene. Included in these descriptions are novel nucleic acid compositions for use in the detection of such sequence. Examples 54-61 below provide details on the design and execution of these illustrative embodiments.","A. Detection of Human Cytomegalovirus Viral DNA By Invasive Cleavage","Human cytomegalovirus (HCMV) causes, or is associated with, a wide variety of diseases in humans (Table 3). More than 90% of bone marrow or kidney transplant recipients (immunocompromised hosts) develop HCMV infections, most of which are due to reactivation of latent virus by immunosuppressive drugs, as well as transmission of virus by latently infected donor tissue or blood (Ackerman el al., Transplant. Proc., 20(S1):468 [1988]; and Peterson et al., Medicine 59:283 [1980]).",{"@attributes":{"id":"p-0537","num":"0546"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 3"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Diseases Caused By Human Cytomegalovirus"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"168pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"cytomegalic inclusion"]},{"entry":[{},"heterophil-negative disease in neonates"]},{"entry":[{},"mononucleosis"]},{"entry":[{},"interstitial pneumonia"]},{"entry":[{},"pneumonitis"]},{"entry":[{},"retinitis"]},{"entry":[{},"hepatitis"]},{"entry":[{},"pancreatitis"]},{"entry":[{},"meningoencephalitis"]},{"entry":[{},"gastrointestinal disease"]},{"entry":[{},"disseminated infection"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}}},"There are instances in which rapid, sensitive, and specific diagnosis of HCMV disease is imperative. In recent years, the number of patients undergoing organ and tissue transplantations has increased markedly. HCMV is the most frequent cause of death in immunocompromised transplant recipients, thereby confirming the need for rapid and reliable laboratory diagnosis. Lymphocytes, monocytes, and possibly arterial endothelial or smooth muscle cells, are sites of HCMV latency. Therefore, prevention of HCMV infections in immunocompromised individuals (e.g., transplant recipients) includes use of HCMV-negative blood products and organs. Additionally, HCMV can be spread transplacentally, and to newborns by contact with infected cervical secretions during birth. Thus, a rapid, sensitive, and specific assay for detecting HCMV in body fluids or secretions may be desirable as a means to monitor infection, and consequently, determine the necessity of cesarean section.","Diagnosis of HCMV infection may be performed by conventional cell culture using human fibroblasts; shell vial centrifugation culture utilizing monoclonal antibodies and immunofluorescent staining techniques; serological methods; the HCMV antigenemia assay which employs a monoclonal antibody to detect HCMV antigen in peripheral blood leukocytes (PBLs); or by nucleic acid hybridization assays. These various methods have their advantages and limitations. Conventional cell culture is sensitive but slow, as cytopathic effect (CPE) may take 30 or more days to develop. Shell vial centrifugation is more rapid but still requires 24-48 hours for initial results. Both culture methods are affected by antiviral therapy. In immunocompromised patients, the ability to mount IgG and\/or IgM antibody responses to HCMV infection are impaired, and serological methods are thus not reliable in this setting. Alternatively, IgM antibodies may be persistent for months after infection is resolved, and thus their presence may not be indicative of active infection. The HCMV antigenemia assay is labor intensive and is not applicable to specimens other than PBLs.","Recent advances in molecular biology have spurred the use of DNA probes in attempts to provide a more rapid, sensitive and specific assay for detecting HCMV in clinical specimens. For example, radiolabeled DNA probes have been used to hybridize to tissue cultures infected with or by HCMV, or in clinical samples suspected of containing HCMV (\u201chybridization assays\u201d). However, probing of tissue cultures requires at least 18-24 hours for growth to amplify the antigen (HCMV) to be detected, if present, and additional time for development of autoradiographic detection systems. Using hybridization assays for assaying clinical specimens for HCMV may lack sensitivity, depending upon the titer of virus and the clinical sample assayed. Detection of HCMV in clinical samples has been reported using the polymerase chain reaction (PCR) to enzymatically amplify HCMV DNA. Methods using PCR compare favorably with virus isolation, in situ hybridization assays, and Southern blotting; See, e.g., Bamborschke et al,. J. Neurol., 239:205 [1992]; Drouet et al., J. Virol. Meth., 45:259 [1993]; Einsele et al., Blood 77:1104-1110 [1991]; Einsele et al., Lancet 338:1170 [1991]; Lee et al., Aust. NZ J. Med., 22:249 [1992]; Miller et al., J. Clin. Microbiol., 32:5 [1994]; Rowley et al., Transplant. 51:1028 [1991]; Spector et al. J. Clin. Microbiol., 30:2359 [1992]; and Stanier et al., Mol. Cell. Probes 8:51 [1992]). Others, comparing the IICMV antigenemia assay with PCR methods, have found PCR methods as efficient or slightly more efficient in the detection of HCMV (van Dorp et al. (1992) Transplant. 54:661; Gerna et al. (1991) J. Infect. Dis. 164:488; Vleiger et al. (1992) Bone Marrow Transplant. 9:247; Zipeto et al. (1992) J. Clin. Microbiol. 30:527]. In addition, PCR methods have exhibited great sensitivity when specimens other than PBLs are assayed (Natori et al., Kansenshogaku Zasshi 67:1011 [1993]; Peterson et al., Medicine 59:283 [1980]; Prosch et al., J. Med. Virol., 38:246 [1992]; Ratnamohan et al., J. Med. Virol. 38:252 [1992]). However, because of the dangers of false positive reactions, these PCR-based procedures require rigid controls to prevent contamination and carry over (Ehrlich et al., in -, Ehrlich and Greenberg (eds), Blackwell Scientific Publications, [1994], pp.3-18). Therefore, there exists a need for a rapid, sensitive, and specific assay for HCMV that has a reduced risk of false positive result due to contamination by reaction product carried over from other samples.","As shown herein, the INVADER-directed cleavage assay is rapid, sensitive and specific. Because the accumulated products do not contribute to the further accumulation of signal, reaction products carried over from one standard (i.e., non-sequential) INVADER-directed cleavage assay to another cannot promote false positive results. The use of multiple sequential INVADER-directed cleavage assays will further boost the sensitivity of HCMV detection without sacrifice of these advantages.","B. Detection of Single Nucleotide Polymorphisms in the Human Apolipoprotein E Gene","Apolipoprotein E (ApoE) performs various functions as a protein constituent of plasma lipoproteins, including its role in cholesterol metabolism. It was first identified as a constituent of liver-synthesized very low density lipoproteins which function in the transport of triglycerides from the liver to peripheral tissues. There are thrce major isoforms of ApoE, referred to as ApoE2, ApoE3 and ApoE4 which are products of three alleles at a single gene locus. Three homozygous phenotypes (Apo-E2\/2, E3\/3, and E4\/4) and three heterozygous phenotypes (ApoE3\/2, E4\/3 and E4\/2) arise from the expression of any two of the three alleles. The most common phenotype is ApoE3\/3 and the most common allele is E3. See Mahley, R. W., Science 240:622-630 (1988).","The amino acid sequences of the three types differ only slightly. ApoE4 differs from ApoE3 in that in ApoE4 arginine is substituted for the normally occurring cysteine at amino acid residue 112. The most common form of ApoE2 differs from ApoE3 at residue 158, where cysteine is substituted for the normally occurring arginine. See Mahley, Science, supra.","The frequency of the apoE4 allele has been shown to be markedly increased in sporadic Alzheimer's Disease (AD) (Poirier, J. et al., 1993, Apolipoprotein E phenotype and Alzheimer's Disease, Lancet, 342:697-699; Noguchi, S. et al., 1993, Lancet (letter), 342:737) and late onset familial Alzheimer's disease (AD) (Corder, E. H. et al., 1993, Science, 261:921-923; Payami, H. et al., 1993, Lancet (letter), 342:738). This gene dosage effect was observed in both sporadic and familial cases (i.e., as age of onset increases, E4 allele copy number decreases). Women, who are generally at a greater risk of developing Alzheimer's disease, show increased E4 allele frequency when compared to age matched men.","C. Detection of Mutations in the Human Hemochromatosis Gene","Hereditary hemochromatosis (HH) is an inherited disorder of iron metabolism wherein the body accumulates excess iron. In symptomatic individuals, this excess iron leads to deleterious effects by being deposited in a variety of organs leading to their failure, and resulting in cirrhosis, diabetes, sterility, and other serious illnesses.","HH is inherited as a recessive trait; heterozygotes are asymptomatic and only homozygotes are affected by the disease. It is estimated that approximately 10% of individuals of Western European descent carry an HH gene mutation and that there are about one million homozygotes in the United States. Although ultimately HH produces debilitating symptoms, the majority of homozygotes have not been diagnosed. Indeed, it has been estimated that no more than 10,000 people in the United States have been diagnosed with this condition. The symptoms are often confused with those of other conditions, and the severe effects of the disease often do not appear immediately. It would be desirable to provide a method to identify persons who are ultimately destined to become symptomatic in order to intervene in time to prevent excessive tissue damage. One reason for the lack of early diagnosis is the inadequacy of presently available diagnostic methods to ascertain which individuals are at risk.","Although blood iron parameters can be used as a screening tool, a confirmed diagnosis often employs HLA typing, which is tedious, nonspecific, and expensive and\/or liver biopsy which is undesirably invasive and costly. Accordingly, others have attempted to develop inexpensive and noninvasive diagnostics both for detection of homozygotes having existing disease, in that presymptomatic detection would guide intervention to prevent organ damage, and for identification of carriers. The need for such diagnostics is documented for example, in Finch, C. A. West J Med (1990) 153:323-325; McCusick, V. et al. Mendelian Inheritance in Man 11th ed., Johns Hopkins University Press (Baltimore, 1994) pp. 1882-1887; Report of the Joint World Health Organization\/HH Foundation\/French HH Association Meeting, 1993.","D. Detection of Mutations in the Human MTHFR","Folic acid derivatives are coenzymes for several critical single-carbon transfer reactions, including reactions in the biosynthesis of purines, thymidylate and methionine. Methylenetetrahydrofolate reductase (MTHFR; EC 1.5.1.20) catalyzes the NADPH-linked reduction of 5,10-methylenetetrahydrofolate to 5-methyltetrahydrofolate, a co-substrate for methylation of homocysteine to methionine. The porcine liver enzyme, a flavoprotein, has been purified to homogeneity; it is a homodimer of 77-kDa subunits. Partial proteolysis of the porcine peptide has revealed two spatially distinct domains: an N-terminal domain of 40 kDa and a C-terminal domain of 37 kDa. The latter domain contains the binding site for the allosteric regulator S-adenosylmethionine.","Hereditary deficiency of MTHFR, an autosomal recessive disorder, is the most common inborn error of folic acid metabolism. A block in the production of methyltetrahydrofolate leads to elevated homocysteine with low to normal levels of methionine. Patients with severe deficiencies of MTHFR (0-20% activity in fibroblasts) can have variable phenotypes. Developmental delay, mental retardation, motor and gait abnormalities, peripheral neuropathy, seizures and psychiatric disturbances have been reported in this group, although at least one patient with severe MTHFR deficiency was asymptomatic. Pathologic changes in the severe form include the vascular changes that have been found in other conditions with elevated homocysteine, as well as reduced neurotransmitter and methionine levels in the CNS. A milder deficiency of MTHFR (35-50% activity) has been described in patients with coronary artery disease. Genetic heterogeneity is likely, considering the diverse clinical features, the variable levels of enzyme activity, and the differential heat inactivation profiles of the reductase in patients' cells. Methods to detect the MTHFR mutation include: AS-PCR (Hessner, et al. 106, 237-9 (1999)) and PCR-RFLP (Nature Genetics, Frosst et al. 1995:10; 111-113).","E. Detection of Prothrombin 20210GA Polymorphism and the Factor V Leiden Polymorphism","The coagulation cascade is a complex series of zymogen activations, inactivations and feed back loops involving numerous enzymes and their cofactors. The entire cascade, from tissue injury or venous trauma to clotting has been well described (refs). The cascade culminates in the conversion of prothrombin (Factor II) to thrombin. This is catalyzed by the activated form of factor X, factor Xa and its cofactor, activated factor V, factor Va. Thrombin then converts fibrinogen to fibrin and promotes fibrin cross-linking and clot formation by activating factor XIII. In addition to the above stated functions, thrombin a serine protease, can also activate factor V in a positive feed-back loop. Factor Va is a pro-coagulant cofactor in the clotting cascade, and when clot formation is sufficient, is inactivated by activated protein C (APC).","Venous thrombosis is the obstruction of the circulation by clots that have been formed in the veins or have been released from a thrombus formed elsewhere. The most frequent sites of clot formation are the deep veins of the legs, but it also may occur in veins in the brain, retina, liver and mesentery. Factors other than heritable defects that can play a role in the development of thrombosis include recent surgery, malignant disorders, pregnancy and labor and long term immobilization.","Studies of hereditary thrombophilia, defined as an increased tendency towards venous thrombotic disease in relatively young adults, have provided insights into the genetic factors that regulate thrombosis. In 1993, Dahlback et al. (Proc Natl Acad Sci USA 1993;90:1004-1008) described an insensitivity to APC, a critical anti-coagulant in the clotting cascade, in three unrelated families with hereditary thrombophilia. The anticoagulant property of APC resides in its capacity to inactivate the activated cofactors Va and VIIIa by limited proteolysis (ref 3). This inactivation of cofactors Va and VIIIa results in reduction of the rate of formation of thrombin, the end product of the cascade. This observation was confirmed by other investigators (ref) and the term \u201cAPC resistance\u201d was coined to describe this particular phenotype in thrombophilic patients. In a subsequent study of 20 families with thrombophilia and APC resistance, an autosomal dominant pattern of inheritance was observed (17). Bertina et al (Nature, 1994, May 5; 369 (6475):64-7) then demonstrated that the phenotype of APC resistance is associated with heterozygosity or homozygosity for a single point mutation at nucleotide 1691 in exon 10 of the factor V gene. This single base change, a guanine to adenine substitution, yields a mutant factor V molecule wherein the arginine at position 506 is replaced with glutamine. This form of the factor V molecule, characterized at Leiden University, (Bertenia et al) is known as the FV Q506 or FV Leiden mutation, and is inactivated less efficiently by APC than the wild type protein. It has been postulated that the prolonged circulation of activated factor V promotes a hypercoagulable state and increases the risk of thrombosis. Subsequent analysis of various patient groups exhibiting symptoms of venous thrombosis indicate that the factor V Leiden mutation is the single most common heritable factor contributing to an increased risk of venous thrombosis.","In 1996, studies by Poort et al. (Blood. 1996:88; 3698-703) revealed the second most common heritable factor contributing to increase thrombotic risk. In studying the sequence of the prothrombin gene in selected patients with a documented familial history of venous thrombophilia, the Poort group identified a single point mutation in the 3\u2032 untranslated region. This G to A transition at position 20210 is strongly correlated with elevated plasma prothrombin levels, and was also shown to be associated with an almost threefold increased risk of venous thrombosis (abstract, Howard).","The first reported case of a thrombophilia pateint genetically homozygous for the G to A polymorphism in the 3\u2032 untranslated region was by Howard, et al (1997 Jul; 8(5):316-9). The patient, a healthy young Mexican male presented with a myocardial infarction, venous thrombosis and embolism. The patient was found to be homozygous for the prothrombin mutation and heterozygous for the Factor V Leiden mutation, supporting the doublehit theory for thrombophilia in young patients.","Studies by Hessner et al. show that the prothrombin 20210GA genotype was nearly 5 times as prevalent in the symptomoatic FVL carriers than in a random Caucasian control group (, 1999, 106), and that allele frequencies for the prothrombin and Factor V mutants vary arnong different ethnic backgrounds (1999; 81:733-8). The above discussion confirms that early detection of the factor V Leiden mutation and the factor II prothrombin mutation are paramount in hereditary thrombotic risk assessment. The nature of these two mutations, that is, a single base change in the nucleic acid sequence, make them amenable to a variety of nucleic acid detection methods known to the art, though the demand for faster, more reliable, cost-effective and user-friendly tests for the detection of specific nucleic acid sequences continues to grow. The most common methods to test for these mutations include PCR\/RFLP, AS-PCR and functional, coagulation assay.","F. Detection of the HR-2 Mutation in the Human Factor V Gene","The R-2 polymorphism is located in exon 13 of the factor V gene, and is the result of an A to G transition at base 4070, replacing the wild-type amino acid histine with the mutant argenine in the mature protein. The R-2 polymorphism is one of a set of mutations termed collectively HR-2. The HR-2 haplotype is defined by 6 nucleotide base substitutions in exons 13 and 16 of the factor V gene. The haplotype is associated with an increased functional resistance to activated protein C both in normal subjects and in thrombophilic patients. When present as a compound heterozygote in conjunction with the factor V Leiden mutation, clinical symptoms are comparable to those seen in patients homozygous for the factor V Leiden mutation, and include increased risk of deep vein thrombosis.","G. Detection of Single Nucleotide Polymorphisms in the Human TNF-\u03b1Gene","The human cytokine tumor necrosis factor alpha (TNF-alpha) has been shown to be a major factor in graft rejection; the more TNF-alpha present in the system, the greater the rejection response to transplanted tissue. Mutations in TNF-alpha have also been correlated with cerebral malaria (Nature 1994;371:508-510), fulminas purpura (J Infect Dis. 1996;174:878-880), and (J Exp Med. 1995;182:1259-1264). The mutation detected in this example is located in the promoter region of the TNF-alpha gene at position minus 308. The wild-type guanine (G) is replaced with a mutant adenine (A). This result of this promoter mutation is the enhancement of transcription of TNF-alpha by 6-7 fold. Methods to detect mutations in TNF-alpha include sequencing, denaturing gradient gel electorphoresis, PCR methods, and methods involving both PCR and post-PCR hybridization with specific oligos.","H. Detection of Methicillin Resistant ",{"@attributes":{"id":"p-0565","num":"0574"},"i":["Staphylococcus aureus ","S. aureus ","Staphalococcus aureus","S. aureus ","Staph. aureus "]},"The following examples serve to illustrate certain preferred embodiments and aspects of the present invention and are not to be construed as limiting the scope thereof.","In the disclosure which follows, the following abbreviations apply: Afu (); Mth (); Mja (); Pfu (); Pwo (); Taq (); Taq DNAP, DNAPTaq, and Taq Pol I (DNA polymerase I); DNAPStf (the Stoffel fragment of DNAPTaq); DNAPEc I (DNA polymerase I); Tth (); Ex. (Example); FIG. (Figure); \u00b0 C. (degrees Centigrade); g (gravitational field); hr (hour); min (minute); olio (oligonucleotide); rxn (reaction); vol (volume); w\/v (weight to volume); v\/v (volume to volume); BSA (bovine serum albumin); CTAB (cetyltrimcthylammonium bromide); HPLC (high pressure liquid chromatography); DNA (deoxyribonucleic acid); p (plasmid); \u03bcl (microliters); ml (milliliters); \u03bcg (micrograms); mg (milligrams); M (molar); mM (milliMolar); \u03bcM (microMolar); pmoles (picomoles); amoles (attomoles); zmoles (zeptomoles); nm (nanometers); kdal (kilodaltons); OD (optical density); EDTA (ethylene diamine tetra-acetic acid); FITC (fluorescein isothiocyanate); SDS (sodium dodecyl sulfate); NaPO(sodium phosphate); NP-40 (Nonidet P-40); Tris (tris(hydroxymethyl)-aminomethane); PMSF (phenylmethylsulfonylfluoride); TBE (Tris-Borate-EDTA, i.e., Tris buffer titrated with boric acid rather than HCl and containing EDTA); PBS (phosphate buffered saline); PPBS (phosphate buffered saline containing 1 mM PMSF); PAGE (polyacrylamide gel electrophoresis); Tween (polyoxyethylene-sorbitan); ATCC (American Type Culture Collection, Rockville, Md.); Coriell (Coriell Cell Repositories, Camden, N.J.); DSMZ (Deutsche Sammlung von Mikroorganismen und Zellculturen, Braunschweig, Germany); Ambion (Ambion, Inc., Austin, Tex.); Boehringer (Boehringer Mannheim Biochemical, Indianapolis, Ind.); MJ Research (MJ Research, Watertown, Mass.; Sigma (Sigma Chemical Company, St. Louis, Mo.); Dynal (Dynal A.S., Oslo, Norway); Gull (Gull Laboratories, Salt Lake City, Utah); Epicentre (Epicentre Technologies, Madison, Wis.); Lampire (Biological Labs., Inc., Coopersberg, Pa.); MJ Research (MJ Research, Watertown, Mass.); National Biosciences (National Biosciences, Plymouth, Minn.); NEB (New England Biolabs, Beverly, Mass.); Novagen (Novagcn, Inc., Madison, Wis.); Perkin Elmer (Perkin-Elmer\/ABI, Norwalk, Conn.); Promega (Promega, Corp., Madison, Wis.); Stratagene (Stratagene Cloning Systems, La Jolla, Calif.); Clonetech (Clonetech, Palo Alto, Calif.) Pharmacia (Pharmacia, Piscataway, N.J.); Milton Roy (Milton Roy, Rochester, N.Y.); Amersham (Arnersham International, Chicago, Ill.); and USB (U.S. Biochemical, Cleveland, Ohio). Glen Research (Glen Research, Sterling, Va.); Coriell (Coriell Cell Repositories, Camden, N.J.); Gentra (Gentra, Minneapolis, Minn.); Third Wave Technologies (Third Wave Technologies, Madison, Wis.); PerSeptive Biosystems (PerSeptive Biosystems, Framington, Mass.); Microsoft (Microsoft, Redmond, Wash.), Qiagen (Qiagen, Valencia, Calif.); Molecular Probes (Molecular Probes, Eugene, Oreg.); VWR (VWR Scientific,); Advanced Biotechnologies (Advanced Biotechnologies, INC., Columbia, Md.); and Perkin elmer (Perkin Elmer,).","A. 5\u2032 Nuclease Activity of DNAPTaq","During the polymerase chain reaction (PCR) (Saiki et al., Science 239:487 [1988]; Mullis and Faloona, Meth. Enzymol., 155:335 [1987]), DNAPTaq is able to amplify many, but not all, DNA sequences. One sequence that cannot be amplified using DNAPTaq is shown in  (Hairpin structure is SEQ ID NO:15,  also shows a primer: SEQ ID NO:17) This DNA sequence has the distinguishing characteristic of being able to fold on itself to form a hairpin with two single-stranded arms, which correspond to the primers used in PCR.","To test whether this failure to amplify is due to the 5\u2032 nuclease activity of the enzyme, the abilities of DNAPTaq and DNAPStf to amplify this DNA sequence during 30 cycles of PCR were compared. Synthetic oligonucleotides were obtained from The Biotechnology Center at the University of Wisconsin-Madison. The DNAPTaq and DNAPStf were from Perkin Elmer (i.e., AMPLITAQ DNA polymerase and the Stoffel fragment of AMPLITAQ DNA polymerase). The substrate DNA comprised the hairpin structure shown in  cloned in a double-stranded form into pUC19. The primers used in the amplification are listed as SEQ ID NOS:16-17. Primer SEQ ID NO:17 is shown annealed to the 3\u2032 arm of the hairpin structure in FIG. . Primer SEQ ID NO:16 is shown as the first 20 nucleotides in bold on the 5\u2032 arm of the hairpin in FIG. .","Polymerase chain reactions comprised 1 ng of supercoiled plasmid target DNA, 5 pmoles of each primer, 40 \u03bcM each dNTP, and 2.5 units of DNAPTaq or DNAPStf, in a 50 \u03bcl solution of 10 mM Tris\u00b7Cl pH 8.3. The DNAPTaq reactions included 50 mM KCl and 1.5 mM MgCl. The temperature profile was 95\u00b0 C. for 30 sec., 55\u00b0 C. for 1 min. and 72\u00b0 C. for 1 min., through 30 cycles. Ten percent of each reaction was analyzed by gel electrophoresis through 6% polyacrylamide (cross-linked 29:1) in a buffer of 45 mM Tris\u00b7Borate, pH 8.3, 1.4 mM EDTA.","The results are shown in FIG. . The expected product was made by DNAPStf (indicated simply as \u201cS\u201d) but not by DNAPTaq (indicated as \u201cT\u201d). It was concluded that the 5\u2032 nuclease activity of DNAPTaq is responsible for the lack of amplification of this DNA sequence.","To test whether the 5\u2032 unpaired nuclcotides in the substrate region of this structured DNA are removed by DNAPTaq, the fate of the end-labeled 5\u2032 arm during four cycles of PCR was compared using the same two polymerases (FIG. ). The hairpin templates, such as the one described in , were made using DNAPStf and a 32P-5\u2032-end-labeled primer. The 5\u2032-end of the DNA was released as a few large fragments by DNAPTaq but not by DNAPStf. The sizes of these fragments (based on their mobilities) show that they contain most or all of the unpaired 5\u2032 arm of the DNA. Thus, cleavage occurs at or near the base of the bifurcated duplex. These released fragments terminate with 3\u2032 OH groups, as evidenced by direct sequence analysis, and the abilities of the fragments to be extended by terminal deoxynucleotidyl transferase.",{"@attributes":{"id":"p-0574","num":"0583"},"figref":["FIGS. 8-10","FIG. 5"],"sub":"2"},"Reactions were initiated at the final reaction temperature by the addition of either the MgClor enzyme. Reactions were stopped at their incubation temperatures by the addition of 8 \u03bcl of 95% formamide with 20 mM EDTA and 0.05% marker dyes. The Tcalculations listed were made using the Oligo\u2122 primer analysis software from National Biosciences, Inc. These were determined using 0.25 \u03bcM as the DNA concentration, at either 15 or 65 mM total salt (the 1.5 mM MgClin all reactions was given the value of 15 mM salt for these calculations).",{"@attributes":{"id":"p-0576","num":"0585"},"figref":["FIG. 8","FIG. 8A","FIG. 8B"]},"Surprisingly, cleavage by DNAPTaq requires neither a primer nor dNTPs (See FIG. A). Thus, the 5\u2032 nuclease activity can be uncoupled from polymerization. Nuclease activity requires magnesium ions, though manganese ions can be substituted, albeit with potential changes in specificity and activity. Neither zinc nor calcium ions support the cleavage reaction. The reaction occurs over a broad temperature range, from 25\u00b0 C. to 85\u00b0 C., with the rate of cleavage increasing at higher temperatures.","Still referring to , the primer is not elongated in the absence of added dNTPs. However, the primer influences both the site and the rate of cleavage of the hairpin. The change in the site of cleavage () apparently results from disruption of a short duplex formed between the arms of the DNA substrate. In the absence of primer, the sequences indicated by underlining in  could pair, forming an extended duplex. Cleavage at the end of the extended duplex would release the 11 nucleotide fragment seen on the  lanes with no added primer. Addition of excess primer (, lanes 3 and 4) or incubation at an elevated temperature () disrupts the hort extension of the duplex and results in a longer 5\u2032 arm and, hence, longer cleavage roducts.","The location of the 3\u2032 end of the primer can influence the precise site of cleavage. Electrophoretic analysis revealed that in the absence of primer (FIG. B), cleavage occurs at the end of the substrate duplex (either the extended or shortened form, depending on the temperature) between the first and second base pairs. When the primer extends up to the base of the duplex, cleavage also occurs one nucleotide into the duplex. However, when a gap of four or six nucleotides exists between the 3\u2032 end of the primer and the substrate duplex, the cleavage site is shifted four to six nucleotides in the 5\u2032 direction.",{"@attributes":{"id":"p-0580","num":"0589"},"figref":["FIG. 9","FIG. 9A","FIG. 9B","FIG. 9A","FIGS. 9A and 9B"],"b":"9"},"The relative influence of primer on cleavage rates becomes much greater when both reactions are run in 50 mM KCl. In the presence of primer, the rate of cleavage increases with KCl concentration, up to about 50 mM. However, inhibition of this reaction in the presence of primer is apparent at 100 mM and is complete at 150 mM KCl. In contrast, in the absence of primer the rate is enhanced by concentration of KCl up to 20 mM, but it is reduced at concentrations above 30 mM. At 50 mM KCl, the reaction is almost completely inhibited. The inhibition of cleavage by KCl in the absence of primer is affected by temperature, being more pronounced at lower temperatures.","Recognition of the 5\u2032 end of the arm to be cut appears to be an important feature of substrate recognition. Substrates that lack a free 5\u2032 end, such as circular M13 DNA, cannot be cleaved under any conditions tested. Even with substrates having defined 5\u2032 arms, the rate of cleavage by DNAPTaq is influenced by the length of the arm. In the presence of primer and 50 mM KCl, cleavage of a 5\u2032 extension that is 27 nuclcotides long is essentially complete within 2 minutes at 55\u00b0 C. In contrast, cleavages of molecules with 5\u2032 arms of 84 and 188 nucleotides are only about 90% and 40% complete after 20 minutes. Incubation at higher temperatures reduces the inhibitory effects of long extensions indicating that secondary structure in the 5\u2032 arm or a heat-labile structure in the enzyme may inhibit the reaction. A mixing experiment, run under conditions of substrate excess, shows that the molecules with long arms do not preferentially tie up the available enzyme in non-productive complexes. These results may indicate that the 5\u2032 nuclease domain gains access to the cleavage site at the end of the bifurcated duplex by moving down the 5\u2032 arm from one end to the other. Longer 5\u2032 arms would be expected to have more adventitious secondary structures (particularly when KCl concentrations are high), which would be likely to impede this movement.","Cleavage does not appear to be inhibited by long 3\u2032 arms of either the substrate strand target molecule or pilot nucleic acid, at least up to 2 kilobases. At the other extreme, 3\u2032 arms of the pilot nucleic acid as short as one nucleotide can support cleavage in a primer-independent reaction, albeit inefficiently. Fully paired oligonucleotidcs do not elicit cleavage of DNA templates during primer extension.","The ability of DNAPTaq to cleave molecules even when the complementary strand contains only one unpaired 3\u2032 nucleotide may be useful in optimizing allele-specific PCR. PCR primers that have unpaired 3\u2032 ends could act as pilot oligonucleotides to direct selective cleavage of unwanted templates during preincubation of potential template-primer complexes with DNAPTaq in the absence of nucleoside triphosphates.","B. 5\u2032 Nuclease Activities of Other DNAPs","To determine whether other 5\u2032 nucleases in other DNAPs would be suitable for the present invention, an array of enzymes, several of which were reported in the literature to be free of apparent 5\u2032 nuclease activity, were examined. The ability of these other enzymes to cleave nucleic acids in a structure-specific manner was tested using the hairpin substrate shown in  under conditions reported to be optimal for synthesis by each enzyme.","DNAPEcl, and DNAP Klenow were obtained from Promega; the DNAP of (\u201cPfu\u201d, Bargseid et al., Strategies 4:34 [1991]) was from Stratagene; the DNAP of (\u201cTli\u201d, Vent\u2122(exo\u2212), Perler et al., Proc. Natl. Acad. Sci. USA 89:5577 [1992] was from New England Biolabs; the DNAP of (\u201cTfl\u201d, Kaledin et al., Biokhimiya 46:1576 [1981] was from Epicentre Technologies; and the DNAP of (\u201cTth\u201d, Carballeira et al., Biotechn., 9:276 [1990]; Myers et al., Biochem., 30:7661 (1991)] was from U.S. Biochemicals.","0.5 units of each DNA polymerase was assayed in a 20 \u03bcl reaction, using either the buffers supplied by the manufacturers for the primer-dependent reactions, or 10 mM Tris\u00b7Cl, pH 8.5, 1.5 MM MgCl, and 20 mM KCl. Reaction mixtures were at held 72\u00b0 C. before the addition of enzyme.",{"@attributes":{"id":"p-0589","num":"0598"},"figref":["FIG. 10","FIG. 10A","FIG. 10B","FIG. 8A"]},"Referring to , DNAPs from the eubacteria and cleave the substrate at the same place as DNAPTaq, both in the presence and absence of primer. In contrast, DNAPs from the archaebacteria and are unable to cleave the substrates endonucleolytically. The DNAPs from and share little sequence homology with eubacterial enzymes (Ito et al., . 19:4045 (1991); Mathur et al., . 19:6952 (1991); see also Perler et al.). Referring to , DNAPEcl also cleaves the substrate, but the resulting cleavage products are difficult to 30 detect unless the 3\u2032 exonuclease is inhibited. The amino acid sequences of the 5\u2032 nuclease domains of DNAPEcl and DNAPTaq are about 38% homologous (Gelfand, supra).","The 5\u2032 nuclease domain of DNAPTaq also shares about 19% homology with the 5\u2032 exonuclease encoded by gene 6 of bacteriophage T7 (Dunn et al., J. Mol. Biol.,166:477 [1983]). This nuclease, which is not covalently attached to a DNAP polymerization domain, is also able to cleave DNA endonucleolytically, at a site similar or identical to the site that is cut by the 5\u2032 nucleases described above, in the absence of added primers.","C. Transcleavage","The ability of a 5\u2032 nuclease to be directed to cleave efficiently at any specific sequence was demonstrated in the following experiment. A partially complementary oligonucleotide termed a \u201cpilot oligonucleotide\u201d was hybridized to sequences at the desired point of cleavage. The non-complementary part of the pilot oligonucleotide provided a structure analogous to the 3\u2032 arm of the template (see FIG. ), whereas the 5\u2032 region of the substrate strand became the 5\u2032 arm. A primer was provided by designing the 3\u2032 region of the pilot so that it would fold on itself creating a short hairpin with a 15 stabilizing tetra-loop (Antao et al., . 19:5901 [1991). Two pilot oligonucleotides are shown in FIG. A. Oligonucleotides 19-12 (SEQ ID NO:18), 30-12 (SEQ ID NO:19) and 30-0 (SEQ ID NO:20) are 31, 42 or 30 nucleotides long, respectively. However, oligonucleotides 19-12 (SEQ ID NO:18) and 34-19 (SEQ ID NO:19) have only 19 and 30 nucleotides, respectively, that are complementary to different sequences in the substrate strand. The pilot oligonucleotides are calculated to melt off their complements at about 50\u00b0 C. (19-12) and about 75\u00b0 C. (30-12). Both pilots have 12 nucleotides at their 3\u2032 ends, which act as 3\u2032 arms with base-paired primers attached.","To demonstrate that cleavage could be directed by a pilot oligonucleotide, a single-stranded target DNA with DNAPTaq was incubated in the presence of two potential pilot oligonucleotides. The transcleavage reactions, where the target and pilot nucleic acids are not covalently linked, includes 0.01 pmoles of single end-labeled substrate DNA, 1 unit of DNAPTaq and 5 pmoles of pilot oligonucleotide in a volume of 20 \u03bcl of the same buffers. These components were combined during a one minute incubation at 95\u00b0 C., to denature the PCR-generated double-stranded substrate DNA, and the temperatures of the reactions were then reduced to their final incubation temperatures. Oligonucleotides 30-12 and 19-12 can hybridize to regions of the substrate DNAs that are 85 and 27 nucleotides from the 5\u2032 end of the targeted strand.",{"@attributes":{"id":"p-0595","num":"0604"},"figref":"FIG. 19","sub":"2"},"Cleavage of the substrate DNA occurred in the presence of the pilot oligonucleotide 19-12 at 50\u00b0 C. (, lanes 1 and 7) but not at 75\u00b0 C. (lanes 4 and 10). In the presence of oligonucleotide 30-12 cleavage was observed at both temperatures. Cleavage did not occur in the absence of added oligonucleotides (lanes 3, 6 and 12) or at about 80\u00b0 C. even though at 50\u00b0 C. adventitious structures in the substrate allowed primer-independent cleavage in the absence of KCl (, lane 9). A non-specific oligonucleotide with no complementarity to the substrate DNA did not direct cleavage at 50\u00b0 C., either in the absence or presence of 50 mM KCl (lanes 13 and 14). Thus, the specificity of the cleavage reactions can be controlled by the extent of complementarity to the substrate and by the conditions of incubation.","D. Cleavage of RNA","A shortened RNA version of the sequence used in the transcleavage experiments discussed above was tested for its ability to serve as a substrate in the reaction. The RNA is cleaved at the expected place, in a reaction that is dependent upon the presence of the pilot oligonucleotide. The RNA substrate, made by T7 RNA polymerase in the presence of (\u03b1-P)UTP, corresponds to a truncated version of the DNA substrate used in FIG. B. Reaction conditions were similar to those in used for the DNA substrates described above, with 50 mM KCl; incubation was for 40 minutes at 55\u00b0 C. The pilot oligonucleotide used is termed 30-0 (SEQ ID NO:20) and is shown in FIG. A.","The results of the cleavage reaction is shown in FIG. B. The reaction was run either in the presence or absence of DNAPTaq or pilot oligonucleotide as indicated in FIG. B.","Strikingly, in the case of RNA cleavage, a 3\u2032 arm is not required for the pilot oligonucleotide. It is very unlikely that this cleavage is due to previously described RNaseH, which would be expected to cut the RNA in several places along the 30 base-pair long RNA-DNA duplex. The 5\u2032 nuclease of DNAPTaq is a structure-specific RNaseH that cleaves the RNA at a single site near the 5\u2032 end of the heteroduplexed region.","It is surprising that an oligonucleotide lacking a 3\u2032 arm is able to act as a pilot in directing efficient cleavage of an RNA target because such oligonucleotides are unable to direct efficient cleavage of DNA targets using native DNAPs. However, some 5\u2032 nucleases of the present invention (for example, clones E, F and G of ) can cleave DNA in the absence of a 3\u2032 arm. In other words, a non-extendable cleavage structure is not required for specific cleavage with some 5\u2032 nucleases of the present invention derived from thermostable DNA polymerases.","Tests were then conducted to determine whether cleavage of an RNA template by DNAPTaq in the presence of a fully complementary primer could help explain why DNAPTaq is unable to extend a DNA oligonucleotide on an RNA template, in a reaction resembling that of reverse transcriptase. Another thermophilic DNAP, DNAPTth, is able to use RNA as a template, but only in the presence of Mn++, so it was predicted that this enzyme would not cleave RNA in the presence of this cation. Accordingly, an RNA molecule was incubated with an appropriate pilot oligonucleotide in the presence of DNAPTaq or DNAPTth, in buffer containing either Mg++ or Mn++. As expected, both enzymes cleaved the RNA in the presence of Mg++. However, DNAPTaq, but not DNAPTth, degraded the RNA in the presence of Mn++. It was concluded that the 5\u2032 nuclease activities of many DNAPs may contribute to their inability to use RNA as templates.","Thermostable DNA polymerases were generated which have reduced synthetic activity, an activity that is an undesirable side-reaction during DNA cleavage in the detection assay of the invention, yet have maintained thermostable nuclease activity. The result is a thermostable polymerase which cleaves nucleic acids DNA with extreme specificity.","Type A DNA polymerases from eubacteria of the genus share extensive protein sequence identity (90% in the polymerization domain, using the Lipman-Pearson method in the DNA analysis software from DNAStar, Wis.) and behave similarly in both polymerization and nuclease assays. Therefore, the gcnes for the DNA polymerase of (DNAPTaq) and (DNAPTfl) are used as representatives of this class. Polymerase genes from other eubacterial organisms, such as sp., and are equally suitable. The DNA polymerases from these thernophilic organisms are capable of surviving and performing at elevated temperatures, and can thus be used in reactions in which temperature is used as a selection against non-specific hybridization of nucleic acid strands.","The restriction sites used for deletion mutagenesis, described below, were chosen for convenience. Different sites situated with similar convenience are available in the gene and can be used to make similar constructs with other Type A polymerase genes from related organisms.","A. Creation of 5\u2032 Nuclease Constructs","1. Modified DNAPTaq Genes","The first step was to place a modified gene for the Taq DNA polymerase on a plasmid under control of an inducible promoter. The modified Taq polymerase gene was isolated as follows: The Taq DNA polymerase gene was amplified by polymerase chain reaction from genomic DNA from , strain YT-1 (Lawyer et al., supra), using as primers the oligonucleotides described in SEQ ID NOS:13-14. The resulting fragment of DNA has a recognition sequence for the restriction endonuclease EcoRI at the 5\u2032 end of the coding sequence and a BglII sequence at the 3\u2032 end. Cleavage with BglII leaves a 5\u2032 overhang or \u201csticky end\u201d that is compatible with the end generated by BamHI. The PCR-amplified DNA was digested with EcoRI and BamHI. The 2512 bp fragment containing the coding region for the polymerase gene was gel purified and then ligated into a plasmid which contains an inducible promoter.","In one embodiment of the invention, the pTTQ18 vector, which contains the hybrid trp-lac (tac) promoter, was used (Stark, 5:255 [1987]) and shown in FIG. . The tac promoter is under the control of the lac repressor. Repression allows the synthesis of the gene product to be suppressed until the desired level of bacterial growth has been achieved, at which point repression is removed by addition of a specific inducer, isopropyl-\u03b2-D-thiogalactopyranoside (IPTG). Such a system allows the expression of foreign proteins that may slow or prevent growth of transformants.","Bacterial promoters, such as tac, may not be adequately suppressed when they are present on a multiple copy plasmid. If a highly toxic protein is placed under control of such a promoter, the small amount of expression leaking through can be harmful to the bacteria. In another embodiment of the invention, another option for repressing synthesis of a cloned gene product was used. The non-bacterial promoter, from bacteriophage T7, found in the plasmid vector series pET-3 was used to express the cloned mutant Taq polymerase genes (; Studier and Moffatt, J. Mol. Biol., 189:113 [1986]). This promoter initiates transcription only by T7 RNA polymerase. In a suitable strain, such as BL21 (DE3)pLYS, the gene for this RNA polymerase is carried on the bacterial genome under control of the lac operator. This arrangement has the advantage that expression of the multiple copy gene (on the plasmid) is completely dependent on the expression of T7 RNA polymerase, which is easily suppressed because it is present in a single copy.","For ligation into the pTTQ18 vector (FIG. ), the PCR product DNA containing the Taq polymerase coding region (mutTaq, clone 4B, SEQ ID NO:21) was digested with EcoRI and BglII and this fragment was ligated under standard \u201csticky end\u201d conditions (Sambrook et al. , Cold Spring Harbor Laboratory Press, Cold Spring Harbor, pp. 1.63-1.69 [1989]) into the EcoRI and BamHI sites of the plasmid vector pTTQ18. Expression of this construct yields a translational fusion product in which the first two residues of the native protein (Met-Arg) are replaced by three from the vector (Met-Asn-Ser), but the remainder of the natural protein would not change. The construct was transformed into the JM109 strain of and the transformants were plated under incompletely repressing conditions that do not permit growth of bacteria expressing the native protein. These plating conditions allow the isolation of genes containing pre-existing mutations, such as those that result from the infidelity of Taq polymerase during the amplification process.","Using this amplification\/selection protocol, a clone (depicted in ) containing a mutated Taq polymerase gene (mutTaq, clone 3B) was isolated. The mutant was first detected by its phenotype, in which temperature-stable 5\u2032 nuclease activity in a crude cell extract was normal, but polymerization activity was almost absent (approximately less than 1% of wild type Taq polymerase activity).","DNA sequence analysis of the recombinant gene showed that it had changes in the polymerase domain resulting in two amino acid substitutions: an A to G change at nucleotide position 1394 causes a Glu to Gly change at amino acid position 465 (numbered according to the natural nucleic and amino acid sequences, SEQ ID NOS:1 and 4) and another A to G change at nucleotide position 2260 causes a Gln to Arg change at amino acid position 754. Because the Gln to Gly mutation is at a nonconserved position and because the Glu to Arg mutation alters an amino acid that is conserved in virtually all of the known Type A polymerases, this latter mutation is most likely the one responsible for curtailing the synthesis activity of this protein. The nucleotide sequence for the  construct is given in SEQ ID NO:21. The enzyme encoded by this sequence is referred to as Cleavase\u00ae A\/G.","Subsequent derivatives of DNAPTaq constructs were made from the mutTaq gene, thus, they all bear these amino acid substitutions in addition to their other lterations, unless these particular regions were deleted. These mutated sites are indicated by black boxes at these locations in the diagrams in FIG. . In , the designation \u201c3\u2032 Exo\u201d is used to indicate the location of the 3\u2032 exonuclease activity associated with Type A polymerases which is not present in DNAPTaq. All constructs except the genes shown in , F and G were made in the pTTQ18 vector.","The cloning vector used for the genes in  was from the commercially available pET-3 series, described above. Though this vector series has only a BamHI site for cloning downstream of the T7 promoter, the series contains variants that allow cloning into any of the three reading frames. For cloning of the PCR product described above, the variant called pET-3c was used (FIG. ). The vector was digested with BamHI, dephosphorylated with calf intestinal phosphatase, and the sticky ends were filled in using the Kienow fragment of DNAPEcl and dNTPs. The gene for the mutant Taq DNAP shown in  (mutTaq, clone 3B) was released from pTTQ18 by digestion with EcoRI and SalI, and the \u201csticky ends\u201d were filled in as was done with the vector. The fragment was ligated to the vector under standard blunt-end conditions (Sambrook et al., Molecular Cloning, supra), the construct was transformed into the BL21 (DE3)pLYS strain of , and isolates were screened to identify those that were ligated with the gene in the proper orientation relative to the promoter. This construction yields another translational fusion product, in which the first two amino acids of DNAP Taq (Met-Arg) are replaced by 13 from the vector plus two from the PCR primer (Met-Ala-Ser-Met-Thr-Gly-Gly-Gln-Gln-Met-Gly-Arg-Ile-Asn-Scr) (SEQ ID NO:24).","In these experiments, the goal was to generate enzymes that lacked the ability to synthesize DNA, but retained the ability to cleave nucleic acids with a 5\u2032 nuclease activity. The act of primed, templated synthesis of DNA is actually a coordinated series of events, so it is possible to disable DNA synthesis by disrupting one event while not affecting the others. These steps include, but are not limited to, primer recognition and binding, dNTP binding and catalysis of the inter-nucleotide phosphodiester bond. Some of the amino acids in the polymerization domain of DNAPEcI have been linked to these functions, but the precise mechanisms are as yet poorly defined.","One way of destroying the polymerizing ability of a DNA polymerase is to delete all or part of the gene segment that encodes that domain for the protein, or to otherwise render the gene incapable of making a complete polymerization domain. Individual mutant enzymes may differ from each other in stability and solubility both inside and outside cells. For instance, in contrast to the 5\u2032 nuclease domain of DNAPEcl, which can be released in an active form from the polymerization domain by gentle proteolysis (Setlow and Kornberg, J. Biol. Chem., 247:232 [1972]), the nuclease domain, when treated similarly, becomes less soluble and the cleavage activity is often lost.","Using the mutant gene shown in  as starting material, several deletion constructs were created. All cloning technologies were standard (Sambrook et al., supra) and are summarized briefly, as follows:","FIG. C: The mutTaq construct was digested with PstI, which cuts once within the polymerase coding region, as indicated, and cuts immediately downstream of the gene in the multiple cloning site of the vector. After release of the fragment between these two sites, the vector was re-ligated, creating an 894-nucteotide deletion, and bringing into frame a stop codon 40 nucleotides downstream of the junction. The nucleotide sequence of this 5\u2032 nuclease (clone 4C) is given in SEQ ID NO:9.","FIG. D: The mutTaq construct was digested with NheI, which cuts once in the gene at position 2047. The resulting four-nucleotide 5\u2032 overhanging ends were filled in, as described above, and the blunt ends were re-ligated. The resulting four-nucleotide insertion changes the reading frame and causes termination of translation ten amino acids downstream of the mutation. The nucleotide sequence of this 5\u2032 nuclease (clone 3D) is given in SEQ ID NO:10.","FIG. E: The entire mutTaq gene was cut from pTTQ18 using EcoRI and SalI and cloned into pET-3c, as described above. This clone was digested with BstXI and Xcml, at unique sites that are situated as shown in FIG. E. The DNA was treated with the Klenow fragment of DNAPEcI and dNTPs, which resulted in the 3\u2032 overhangs of both sites being trimmed to blunt ends. These blunt ends were ligated together, resulting in an out-of-frame deletion of 1540 nucleotides. An in-frame termination codon occurs 18 triplets past the junction site. The nucleotide sequence of this 5\u2032 nuclease (clone 3E) is given in SEQ ID NO:11, with the appropriate leader sequence given in SEQ ID NO:25. It is also referred to as Cleavase\u00ae BX.","FIG. F: The entire mutTaq gene was cut from pTTQ18 using EcoRI and SalI and cloned into pET-3c, as described above. This clone was digested with BstXI and BamHI, at unique sites that are situated as shown in the diagram. The DNA was treated with the Klenow fragment of DNAPEcl and dNTPs, which resulted in the 3\u2032 overhang of the BstXI site being trimmed to a blunt end, while the 5\u2032 overhang of the BamHI site was filled in to make a blunt end. These ends were ligated together, resulting in an in-frame deletion of 903 nucleotides. The nucleotide sequence of the 5\u2032 nuclease (clone 3F) is given in SEQ ID NO:12. It is also referred to as Cleavase\u00ae BB.","FIG. G: This polymerase is a variant of that shown in FIG. E. It was cloned in the plasmid vector pET-21 (Novagen). The non-bacterial promoter from bacteriophage T7, found in this vector, initiates transcription only by 17 RNA polymerase. See Studier and Moffatt, supra. In a suitable strain, such as (DES)pLYS, the gene for this RNA polymerase is carried on the bacterial genome under control of the lac operator. This arrangement has the advantage that expression of the multiple copy gene (on the plasmid) is completely dependent on the expression of T7 RNA polymerase, which is easily suppressed because it is present in a single copy. Because the expression of these mutant genes is under this tightly controlled promoter, potential problems of toxicity of the expressed proteins to the host cells are less of a concern.","The pET-21 vector also features a \u201cHis*Tag\u201d, a stretch of six consecutive histidine residues that are added on the carboxy terminus of the expressed proteins. The resulting proteins can then be purified in a single step by metal chelation chromatography, using a commercially available (Novagen) column resin with immobilized Ni ions. The 2.5 ml columns are reusable, and can bind up to 20 mg of the target protein under native or denaturing (guanidine*HCl or urea) conditions.",{"@attributes":{"id":"p-0625","num":"0634"},"i":"E. coli "},"Some mutant proteins are sequestered by the cells into inclusion bodies. These are granules that form in the cytoplasm when bacteria are made to express high levels of a foreign protein, and they can be purified from a crude lysate, and analyzed by SDS-PAGE to determine their protein content. If the cloned protein is found in the inclusion bodies, it must be released to assay the cleavage and polymerase activities. Different methods of solubilization may be appropriate for different proteins, and a variety of methods are known (See e.g., Builder & Ogez, U.S. Pat. No. 4,511,502 (1985); Olson, U.S. Pat. No. 4,518,526 (1985); Olson & Pai, U.S. Pat. No. 4,511,503 (1985); and Jones et al., U.S. Pat. No. 4,512,922 (1985), all of which are hereby incorporated by reference).","The solubilized protein is then purified on the Ni column as described above, following the manufacturers instructions (Novagen). The washed proteins are eluted from the column by a combination of imidazole competitor (1 M) and high salt (0.5 M NaCl), and dialyzed to exchange the buffer and to allow denature proteins to refold. Typical recoveries result in approximately 20 \u03bcg of specific protein per ml of starting culture. The DNAP mutant is referred to as the CLEAVASE BN nuclease and the sequence is given in SEQ ID NO:26 (the amino acid sequence of the CLEAVASE BN nuclease is obtained by translating the DNA sequence of SEQ ID NO:26).","2. Modified DNAPTfl Gene","The DNA polymerase gene of was isolated from the \u201c\u201d AT-62 strain obtained from the American Type Tissue Collection (ATCC 33923). This strain has a different restriction map then does the strain used to generate the sequence published by Akhmetzjanov and Vakhitov, supra. The published sequence is listed as SEQ ID NO:2. No sequence data has been published for the DNA polym erase gene from the AT-62 strain of ","Genomic DNA from was amplified using the same primers used to amplify the DNA polymerase gene (SEQ ID NOS:13-14). The approximately 2500 base pair PCR fragment was digested with EcoRI and BamHI. The over-hanging ends were made blunt with the Kienow fragment of DNAPEc1 and dNTPs. The resulting approximately 1800 base pair fragment containing the coding region for the N-terminus was ligated into pET-3c, as described above. This construct, clone 4B, is depicted in FIG. B. The wild type DNA polymerase gene is depicted in FIG. A. The 4B clone has the same leader amino acids as do the DNAPTaq clones 4E and F which were cloned into pET-3c; it is not known precisely where translation termination occurs, but the vector has a strong transcription termination signal immediately downstream of the cloning site.","B. Growth and Induction of Transformed Cells","Bacterial cells were transformed with the constructs described above using standard transformation techniques and used to inoculate 2 mls of a standard growth medium (e.g., Luria-Bertani broth). The resulting cultures were incubated as appropriate for the particular strain used, and induced if required for a particular expression system. For all of the constructs depicted in , the cultures were grown to an optical density (at 600 nm wavelength) of 0.5 OD.","To induce expression of the cloned genes, the cultures were brought to a final concentration of 0.4 mM IPTG and the incubations were continued for 12 to 17 hours. Then, 50 \u03bcl aliquots of each culture were removed both before and after induction and were combined with 20 \u03bcl of a standard gel loading buffer for sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE). Subsequent staining with Coomassie Blue (Sambrook et al., supra) allows visualization of the foreign proteins if they account for about 3-5% of the cellular protein and do not co-migrate with any of the major protein bands. Proteins that do co-migrate with a major host protein must be expressed as more than 10% of the total protein to be seen at this stage of analysis.","C. Heat Lysis and Fractionation","Expressed thermostable proteins (i.e., the 5\u2032 nucleases), were isolated by heating crude bacterial cell extracts to cause denaturation and precipitation of the less stable proteins. The precipitated proteins were then, along with other cell debris, removed by centrifugation. Then, 1.7 mls of the culture were pelleted by microcentrifugation at 12,000 to 14,000 rpm for 30 to 60 seconds. After removal of the supernatant, the cells were resuspended in 400 \u03bcl of buffer A (50 mM Tris-HCl, pH 7.9, 50 mM dextrose, 1 mM EDTA), re-centrifuged, then resuspended in 80 \u03bcl of buffer A with 4 mg\/ml lysozyme. The cells were incubated at room temperature for 15 minutes, then combined with 80 \u03bcl of buffer B (10 mM Tris-HCl, pH 7.9, 50 mM KCl, 1 mM EDTA, 1 mM PMSF, 0.5% Tween-20, 0.5% Nonidet-P40).","This mixture was incubated at 75\u00b0 C. for 1 hour to denature and precipitate the host proteins. This cell extract was centrifuged at 14,000 rpm for 15 minutes at 4\u00b0 C., and the supernatant was transferred to a fresh tube. An aliquot of 0.5 to 1 \u03bcl of this supernatant was used directly in each test reaction, and the protein content of the extract was determined by subjecting 7 \u03bcl to electrophoretic analysis, as above. The native recombinant Taq DNA polymerase (Engelke, Anal. Biochem., 191:396 [1990]), and the double point mutation protein shown in  are both soluble and active at this point.","The foreign protein may not be detected after the heat treatments due to sequestration of the foreign protein by the cells into inclusion bodies. These are granules that form in the cytoplasm when bacteria are made to express high levels of a foreign protein, and they can be purified from a crude lysate, and analyzed SDS PAGE to determine their protein content. Many methods have been described in the literature, and one approach is described below.","D. Isolation and Solubilization of Inclusion Bodies","A small culture was grown and induced as described above. A 1.7 ml aliquot was pelleted by brief centrifugation, and the bacterial cells were resuspended in 100 \u03bcl of Lysis buffer (50 mM Tris-HCl, pH 8.0, 1 mM EDTA, 100 nuM NaCl). Then, 2.5 \u03bcl of 20 mM PMSF were added for a final concentration of 0.5 mM, and lysozyme was added to a concentration of 1.0 mg\/ml. The cells were incubated at room temperature for 20 minutes, deoxycholic acid was added to 1 mg\/ml (1 \u03bcl of 100 mg\/ml solution), and the mixture was further incubated at 37\u00b0 C. for about 15 minutes or until viscous. DNAse I was added to 10 \u03bcg\/ml and the mixture was incubated at room temperature for about 30 minutes or until it was no longer viscous.","From this mixture the inclusion bodies were collected by centrifugation at 14,000 rpm for 15 minutes at 4\u00b0 C., and the supernatant was discarded. The pellet was resuspended in 100 \u03bcl of lysis buffer with 10 mM EDTA (pH 8.0) and 0.5% Triton X-100. After 5 minutes at room temperature, the inclusion bodies were pelleted as before, and the supernatant was savcd for later analysis. The inclusion bodies were resuspended in 50 \u03bcl of distilled water, and 5 \u03bcl was combined with SDS gel loading buffer (which dissolves the inclusion bodies) and analyzed electrophoretically, along with an aliquot of the supernatant.","If the cloned protein is found in the inclusion bodies, it may be released to assay the cleavage and polymerase activities and the method of solubilization must be compatible with the particular activity. Different methods of solubilization may be appropriate for different proteins, and a variety of methods are discussed in (Sambrook et al., supra). The following is an adaptation used for several of the isolates used in the development of the present invention.","Twenty \u03bcl of the inclusion body-water suspension were pelleted by centrifugation at 14,000 rpm for 4 minutes at room temperature, and the supernatant was discarded. To further wash the inclusion bodies, the pellet was resuspended in 20 \u03bcl of lysis buffer with 2M urea, and incubated at room temperature for one hour. The washed inclusion bodies were then resuspended in 2 \u03bcl of lysis buffer with 8 M urea; the solution clarified visibly as the inclusion bodies dissolved. Undissolved debris was removed by centrifugation at 14,000 rpm for 4 minutes at room temperature, and the extract supematant was transferred to a fresh tube.","To reduce the urea concentration, the extract was diluted into KHPO. A fresh tube was prepared containing 180 \u03bcl of 50 mM KHPO, pH 9.5, 1 mM EDTA and 50 mM NaCl. A 2 \u03bcl aliquot of the extract was added and vortexed briefly to mix. This step was repeated until all of the extract had been added for a total of 10 additions. The mixture was allowed to sit at room temperature for 15 minutes, during which time some precipitate often forms. Precipitates were removed by centrifugation at 14,000 rpm, for 15 minutes at room temperature, and the supernatant was transferred to a fresh tube. To the 200 \u03bcl of protein in the KHPOsolution, 140-200 \u03bcl of saturated (NH)SOwere added, so that the resulting mixture was about 41% to 50% saturated (NH)SO. The mixture was chilled on ice for 30 minutes to allow the protein to precipitate, and the protein was then collected by centrifugation at 14,000 rpm, for 4 minutes at room 1 5 temperature. The supernatant was discarded, and the pellet was dissolved in 20 \u03bcl Buffer C (20 mM HEPES, pH 7.9, 1 mM EDTA, 0.5% PMSF, 25 mM KCl and 0.5% each of Tween-20 and Nonidet P 40). The protein solution was centrifuged again for 4 minutes to pellet insoluble materials, and the supernatant was removed to a fresh tube. The protein contents of extracts prepared in this manner were visualized by resolving 1-4 \u03bcl by SDS-PAGE; 0.5 to 1 \u03bcl of extract was tested in the cleavage and polymerization assays as described.","E. Protein Allalysis for Presence of Nuclease and Synthetic Activity","The 5\u2032 nucleases described above and shown in  were analyzed by the following methods.","1. Structure Specific Nuclease Assay","A candidate modified polymerase is tested for 5\u2032 nuclease activity by examining its ability to catalyze structure-specific cleavages. By the term \u201ccleavage structure\u201d as used herein, is meant a nucleic acid structure which is a substrate for cleavage by the 5\u2032 nuclease activity of a DNAP.","The polymerase is exposed to test complexes that have the structures shown in FIG. . Testing for 5\u2032 nuclease activity involves three reactions: 1) a primer-directcd cleavage () is performed because it is relatively insensitive to variations in the salt concentration of the reaction and can, therefore, be performed in whatever solute conditions the modified enzyme requires for activity; this is generally the same conditions preferred by unmodified polymerases; 2) a similar primer-directed cleavage is performed in a buffer which permits primer-independent cleavage (i.e., a low salt buffer), to demonstrate that the enzyme is viable under these conditions; and 3) a primer-independent cleavage () is performed in the same low salt buffer.","The bifurcated duplex is formed between a substrate strand and a template strand as shown in FIG. . By the term \u201csubstrate strand\u201d as used herein, is meant that strand of nucleic acid in which the cleavage mediated by the 5\u2032 nuclease activity occurs. The substrate strand is always depicted as the top strand in the bifurcated complex which serves as a substrate for 5\u2032 nuclease cleavage (FIG. ). By the term \u201ctemplate strand\u201d as used herein, is meant the strand of nucleic acid which is at least partially complementary to the substrate strand and which anneals to the substrate strand to form the cleavage structure. The template strand is always depicted as the bottom strand of the bifurcated cleavage structure (FIG. ). If a primer (a short oligonucleotide of 19 to 30 nucleotides in length) is added to the complex, as when primer-dependent cleavage is to be tested, it is designed to anneal to the 3\u2032 arm of the template strand (FIG. B). Such a primer would be extended along the template strand if the polymerase used in the reaction has synthetic activity.","The cleavage structure may be made as a single hairpin molecule, with the 3\u2032 end of the target and the 5\u2032 end of the pilot joined as a loop as shown in  primer oligonucleotide complementary to the 3\u2032 arm is also required for these tests so that the enzyme's sensitivity to the presence of a primer may be tested.","Nucleic acids to be used to form test cleavage structures can be chemically synthesized, or can be generated by standard recombinant DNA techniques. By the latter method, the hairpin portion of the molecule can be created by inserting into a cloning vector duplicate copies of a short DNA segment, adjacent to each other but in opposing orientation. The double-stranded fragment encompassing this inverted repeat, and including enough flanking sequence to give short (about 20 nucleotides) unpaired 5\u2032 and 3\u2032 arms, can then be released from the vector by restriction enzyme digestion, or by PCR performed with an enzyme lacking a 5\u2032 exonuclease (e.g., the Stoffel fragment of AMPLIFAQ DNA polymerase, Vent\u2122 DNA polymerase).","The test DNA can be labeled on either end, or internally, with either a radioisotope, or with a non-isotopic tag. Whether the hairpin DNA is a synthetic single strand or a cloned double strand, the DNA is heated prior to use to melt all duplexes. When cooled on ice, the structure depicted in  is formed, and is stable for sufficient time to perform these assays.","To test for primer-directed cleavage (Reaction 1), a detectable quantity of the test molecule (typically 1-100 fmol of P-labeled hairpin molecule) and a 10 to 100-fold molar excess of primer are placed in a buffer known to be compatible with the test enzyme. For Reaction 2, where primer-directed cleavage is performed under condition which allow primer-independent cleavage, the same quantities of molecules are placed in a solution that is the same as the buffer used in Reaction 1 regarding pH, enzyme stabilizers (e.g., bovine serum albumin, nonionic detergents, gelatin) and reducing agents (e.g., dithiothreitol, 2-mercaptoethanol) but that replaces any monovalent cation salt with 20 mM KCl; 20 mM KCl is the demonstrated optimum for primer-independent cleavage. Buffers for enzymes, such as DNAPEc1, that usually operate in the absence of salt are not supplemented to achieve this concentration. To test for primer-independent cleavage (Reaction 3) the same quantity of the test molecule, but no primer, are combined under the same buffer conditions used for Reaction 2.","All three test reactions are then exposed to enough of the enzyme that the molar ratio of enzyme to test complex is approximately 1:1. The reactions are incubated at a range of temperatures up to, but not exceeding, the temperature allowed by either the enzyme stability or the complex stability, whichever is lower, up to 80\u00b0 C. for enzymes from thermophiles, for a time sufficient to allow cleavage (10 to 60 minutes). The products of Reactions 1, 2 and 3 are resolved by denaturing polyacrylamide gel electrophoresis, and visualized by autoradiography or by a comparable method appropriate to the labeling system used. Additional labeling systems include chemiluminescence detection, silver or other stains, blotting and probing and the like. The presence of cleavage products is indicated by the presence of molecules which migrate at a lower molecular weight than does the uncleaved test structure. These cleavage products indicate that the candidate polymerase has structure-specific 5\u2032 nuclease activity.","To determine whether a modified DNA polymerase has substantially the same 5\u2032 nuclease activity as that of the native DNA polymerase, the results of the above-described tests are compared with the results obtained from these tests performed with the native DNA polymerase. By \u201csubstantially the same 5\u2032 nuclease activity\u201d it is meant that the modified polymerase and the native polymerase will both cleave test molecules in the same manner. It is not necessary that the modified polymerase cleave at the same rate as the native DNA polymerase.","Some enzymes or enzyme preparations may have other associated or contaminating activities that may be functional under the cleavage conditions described above and that may interfere with 5\u2032 nuclease detection. Reaction conditions can be modified in consideration of these other activities, to avoid destruction of the substrate, or other masking of the 5\u2032 nuclease cleavage and its products. For example, the DNA polymerase I of (Pol I), in addition to its polymerase and 5\u2032 nuclease activities, has a 3\u2032 exonuclease that can degrade DNA in a 3\u2032 to 5\u2032 direction. Consequently, when the molecule in  is exposed to this polymerase under the conditions described above, the 3\u2032 exonuclease quickly removes the unpaired 3\u2032 arm, destroying the bifurcated structure required of a substrate for the 5\u2032 exonuclease cleavage and no cleavage is detected. The true ability of Pol I to cleave the structure can be revealed if the 3\u2032 exonuclease is inhibited by a change of conditions (e.g., pH), mutation, or by addition of a competitor for the activity. Addition of 500 pmoles of a single-stranded competitor oligonucleotide, unrelated to the  structure, to the cleavage reaction with Pol I effectively inhibits the digestion of the 3\u2032 arm of the  structure without interfering with the 5\u2032 exonuclease release of the 5\u2032 arm. The concentration of the competitor is not critical, but should be high enough to occupy the 3\u2032 exonuclease for the duration of the reaction.","Similar destruction of the test molecule may be caused by contaminants in the candidate polymerase preparation. Several sets of the structure specific nuclease reactions may be performed to determine the purity of the candidate nuclease and to find the window between under and over exposure of the test molecule to the polymerase preparation being investigated.","The above described modified polymerases were tested for 5\u2032 nuclease activity as follows: Reaction 1 was performed in a buffer of 10 mM Tris-Cl, pH 8.5 at 20\u00b0 C., 1.5 mM MgCland 50 mM KCl and in Reaction 2 the KCl concentration was reduced to 20 mM. In Reactions 1 and 2, 10 fmoles of the test substrate molecule shown in  were combined with 1 pmole of the indicated primer and 0.5 to 1.0 \u03bcl of extract containing the modified polymerase (prepared as described above). This mixture was then incubated for 10 minutes at 55\u00b0 C. For all of the mutant polymerases tested these conditions were sufficient to give complete cleavage. When the molecule shown in  was labeled at the 5\u2032 end, the released 5\u2032 fragment, 25 nucleotides long, was conveniently resolved on a 20% polyacrylamide gel (19:1 cross-linkcd) with 7 M urea in a buffer containing 45 mM Tris-borate pH 8.3, 1.4 mM EDTA. Clones 3C-F and 4B exhibited structure-specific cleavage comparable to that of the unmodified DNA polymerase. Additionally, clones 3E, 3F and 3G have the added ability to cleave DNA in the absence of a 3\u2032 arm as discussed above. Representative cleavage reactions are shown in FIG. .","For the reactions shown in , the mutant polymerase clones 3E (Taq mutant) and 4B (Tfl mutant) were examined for their ability to cleave the hairpin substrate molecule shown in FIG. E. The substrate molecule was labeled at the 5\u2032 terminus with P. Ten fmoles of heat-denatured, end-labeled substrate DNA and 0.5 units of DNAPTaq (lane 1) or 0.5 \u03bcl of 3E or 4B extract (, lanes 2-7, extract was prepared as described above) were mixed together in a buffer containing 10 mM Tris-Cl, pH 8.5, 50 mM KCl and 1.5 mM MgCl. The final reaction volume was 10 \u03bcl. Reactions shown in lanes 4 and 7 contain in addition 50 \u03bcM of each dNTP. Reactions shown in lanes 3, 4, 6 and 7 contain 0.2 \u03bcM of the primer oligonucleotide (complementary to the 3\u2032 arm of the substrate and shown in FIG. E). Reactions were incubated at 55\u00b0 C. for 4 minutes. Reactions were stopped by the addition of 8 \u03bcl of 95% formamide containing 20 mM EDTA and 0.05% marker dyes per 10 \u03bcl reaction volume. Samples were then applied to 12% denaturing acrylamide gels. Following electrophoresis, the gels were autoradiographed.  shows that clones 3E and 4B exhibit cleavage activity similar to that of the native DNAPTaq. Note that some cleavage occurs in these reactions in the absence of the primer. When long hairpin structure, such as the one used here.(FIG. E), are used in cleavage reactions performed in buffers containing 50 mM KCl a low level of primer-independent cleavage is seen. Higher concentrations of KCl suppress, but do not eliminate, this primer-independent cleavage under these conditions.","2. Assay For Synthetic Activity","The ability of the modified enzyme or proteolytic fragments is assayed by adding the modified enzyme to an assay system in which a primer is annealed to a template and DNA synthesis is catalyzed by the added enzyme. Many standard laboratory techniques employ such an assay. For example, nick translation and enzymatic sequencing involve extension of a primer along a DNA template by a polymerase molecule.","In a preferred assay for determining the synthetic activity of a modified enzyme an oligonucleotide primer is annealed to a single-stranded DNA template (e.g., bacteriophage M13 DNA), and the primer\/template duplex is incubated in the presence of the modified polymerase in question, deoxynucleoside triphosphates (dNTPs) and the buffer and salts known to be appropriate for the unmodified or native enzyme. Detection of either primer extension (by denaturing gel electrophoresis) or dNTP incorporation (by acid precipitation or chromatography) is indicative of an active polymerase. A label, either isotopic or non-isotopic, is preferably included on either the primer or as a dNTP to facilitate detection of polymerization products. Synthetic activity is quantified as the amount of free nucleotide incorporated into the growing DNA chain and is expressed as amount incorporated pcr unit of time under specific reaction conditions.","Representative results of an assay for synthetic activity is shown in FIG. . The synthetic activity of the mutant DNAPTaq clones 3B-F was tested as follows: A master mixture of the following buffer was made: 1.2\u00d7 PCR buffer (1\u00d7 PCR buffer contains 50 mM KCl, 1.5 mM MgCl, 10 mM Tris-Cl, pH 8.5 and 0.05% each Tween 20 and Nonidet P40), 50 \u03bcM each of dGTP, dATP and dTTP, 5 \u03bcM dCTP and P-dCTP at 600 Ci\/mmol. Before adjusting this mixture to its final volume, it was divided into two equal aliquots. One received distilled water up to a volume of 50 \u03bcl to give the concentrations above. The other received 5 \u03bcg of single-stranded M13mp18 DNA (approximately 2.5 pmol or 0.05 \u03bcM final concentration) and 250 pmol of M13 sequencing primer (5 \u03bcM final concentration) and distilled water to a final volume of 50 \u03bcl. Each cocktail was warmed to 75\u00b0 C. for 5 minutes and then cooled to room temperature. This allowed the primers to anneal to the DNA in the DNA-containing mixtures.","For each assay, 4 \u03bcl of the cocktail with the DNA was combined with 1 \u03bcl of the mutant polymerase, prepared as described, or 1 unit of DNAPTaq (Perkin Elmer) in 1 \u03bcl of dHO. A \u201cno DNA\u201d control was done in the presence of the DNAPTaq (, lane 1), and a \u201cno enzyme\u201d control was done using water in place of the enzyme (lane 2). Each reaction was mixed, then incubated at room temperature (approx. 22\u00b0 C.) for 5 minutes, then at 55\u00b0 C. for 2 minutes, then at 72\u00b0 C. for 2 minutes. This step incubation was done to detect polymerization in any mutants that might have optimal temperatures lower than 72\u00b0 C. After the final incubation, the tubes were spun briefly to collect any condensation and were placed on ice. One \u03bcl of each reaction was spotted at an origin 1.5 cm from the bottom edge of a polyethyleneimine (PEI) cellulose thin layer chromatography plate and allowed to dry. The chromatography plate was run in 0.75 M NaHPO, pH 3.5, until the buffer front had run approximately 9 cm from the origin. The plate was dried, wrapped in plastic wrap, marked with luminescent ink, and exposed to X-ray film. Incorporation was detected as counts that stuck where originally spotted, while the unincorporated nucleotides were carried by the salt solution from the origin.","Comparison of the locations of the counts with the two control lanes confirmed the lack of polymerization activity in the mutant preparations. Among the modified DNAPTaq clones, only clone 3B retains any residual synthetic activity as shown in FIG. .","The ability of the 5\u2032 nucleases to cleave hairpin structures to generate a cleaved hairpin structure suitable as a detection molecule was examined. The structure and sequence of the hairpin test molecule is shown in  (SEQ ID NO:15). The oligonucleotide (labeled \u201cprimer\u201d in , SEQ ID NO:22) is shown annealed to its complementary sequence on the 3\u2032 arm of the hairpin test molecule. The hairpin test molecule was single-end labeled with P using a labeled T7 promoter primer in a polymerase chain reaction. The label is present on the 5\u2032 arm of the hairpin test molecule and is represented by the star in FIG. A.","The cleavage reaction was performcd by adding 10 fmoles of heat-denatured, end-labeled hairpin test molecule, 0.2 \u03bcM of the primer oligonucleotide (complementary to the 3\u2032 arm of the hairpin), 50 \u03bcM of each dNTP and 0.5 units of DNAPTaq (Perkin Elmer) or 0.5 \u03bcl of extract containing a 5\u2032 nuclease (prepared as described above) in a total volume of 10 \u03bcl in a buffer containing 10 mM Tris-Cl, pH 8.5, 50 mM KCl and 1.5 mM MgCl. Reactions shown in lanes 3, 5 and 7 were run in the absence of dNTPs.","Reactions were incubated at 55\u00b0 C. for 4 minutes. Reactions were stopped at 55\u00b0 C. by the addition of 8 \u03bcl of 95% formamide with 20 mM EDTA and 0.05% marker dyes per 10 \u03bcl reaction volume. Samples were not heated before loading onto denaturing polyacrylamide gels (10% polyacrylamide, 19:1 crosslinking, 7 M urea, 89 mM Tris-borate, pH 8.3, 2.8 mM EDTA). The samples were not heated to allow for the resolution of single-stranded and re-duplexed uncleaved hairpin molecules.",{"@attributes":{"id":"p-0669","num":"0678"},"figref":["FIG. 18B","FIG. 18B","FIG. 18B"]},"From the above, it should be clear that native (i.e., \u201cwild type\u201d) thermostable DNA polymerases are capable of cleaving hairpin structures in a specific manner and that this discovery can be applied with success to a detection assay. In this example, the mutant DNAPs of the present invention are tested against three different cleavage structures shown in FIG. A. Structure 1 in  is simply single stranded 206-mer (the preparation and sequence information for which was discussed in Example 1C). Structures 2 and 3 are duplexes; structure 2 is the same hairpin structure as shown in  (bottom), while structure 3 has the hairpin portion of structure 2 removed.","The cleavage reactions comprised 0.01 pmoles of the resulting substrate DNA, and 1 pmole of pilot oligonucleotide in a total volume of 10 \u03bcl of 10 mM Tris-Cl, pH 8.3, 100 mM KCl, 1 mM MgCl. Reactions were incubated for 30 minutes at 55\u00b0 C., and stopped by the addition of 8 \u03bcl of 95% fornamide with 20 mM EDTA and 0.05% marker dyes. Samples were heated to 75\u00b0 C. for 2 minutes immediately before electrophoresis through a 10% polyacrylamide gel (19:1 cross link), with 7M urea, in a buffer of 45 mM Tris-Borate, pH 8.3, 1.4 mM EDTA.","The results were visualized by autoradiography and are shown in  with the enzymes indicated as follows: I is native Taq DNAP; II is native Tfl DNAP; III is CLEAVASE BX shown in ; IV is CLEAVASE BB shown in ; V is the mutant shown in ; and VI is CLEAVASE BN shown in FIG. G.","Structure 2 was used to \u201cnormalize\u201d the comparison. For example, it was found that it took 50 ng of Taq DNAP and 300 ng of CLEAVASE BN to give similar amounts of cleavage of Structure 2 in thirty (30) minutes. Under these conditions native Taq DNAP is unable to cleave Structure 3 to any significant degree. Native Tfl DNAP cleaves Structure 3 in a manner that creates multiple products.","By contrast, all of the mutants tested cleave the linear duplex of Structure 3. This finding indicates that this characteristic of the mutant DNA polymerases is consistent of thermostable polymerases across thermophilic species.","It has been found that thermostable DNAPs, including those of the present invention, have a true 5\u2032 exonuclease capable of nibbling the 5\u2032 end of a linear duplex nucleic acid structures. In this Example, the 206 base pair DNA duplex substrate is again employed (See, Example 1C). In this case, it was produced by the use of one P-labeled primer and one unlabeled primer in a polymerase chain reaction. The cleavage reactions comprised 0.01 pmoles of heat-denatured, end-labeled substrate DNA (with the unlabeled strand also present), 5 pmoles of pilot oligonucleotide (see pilot oligos in ) and 0.5 units of DNAPTaq or 0.5\u03bc of CLEAVASE BB in the extract (see above), in a total volume of 10 \u03bcl of 10 mM TrisCl, pH 8.5, 50 mM KCl, 1.5 mM MgCl.","Reactions were initiated at 65\u00b0 C. by the addition of pre-warmcd enzyme, then shifted to the final incubation temperature for 30 minutes. The results are shown in FIG. A. Samples in lanes 1-4 are the results with native Taq DNAP, while lanes 5-8 shown the results with CLEAVASE BB. The reactions for lanes 1, 2, 5, and 6 were performed at 65\u00b0 C. and reactions for lanes 3, 4, 7, and 8 were performed at 50\u00b0 C. and all were stopped at temperature by the addition of 8 \u03bcl of 95% formamide with 20 mM EDTA and 0.05% marker dyes. Samples were heated to 75\u00b0 C. for 2 minutes immediately before electrophoresis through a 10% acrylamide gel (19:1 cross-linked), with 7 M urea, in a buffer of 45 mM TrisBorate, pH 8.3, 1.4 mM EDTA. The expected product in reactions 1, 2, 5, and 6 is 85 nucleotides long; in reactions 3 and 7, the expected product is 27 nucleotides long. Reactions 4 and 8 were performed without pilot, and should remain at 206 nucleotides. The faint band seen at 24 nucleotides is residual end-labeled primer from the PCR.","The surprising result is that CLEAVASE BB under these conditions causes all of the label to appear in a very small species, suggesting the possibility that the enzyme completely hydrolyzed the substrate. To determine the composition of the fastest-migrating band seen in lanes 5-8 (reactions performed with the deletion mutant), samples of the 206 base pair duplex were treated with either T7 gene 6 exonuclease (USB) or with calf intestine alkaline phosphatase (Promega), according to manufacturers' instructions, to produce either labeled mononucleotide (lane a of ) or free P-labeled inorganic phosphate (lane b of FIG. B), respectively. These products, along with the products seen in lane 7 of panel A were resolved by brief electrophoresis through a 20% acrylamide gel (19:1 cross-link), with 7 M urea, in a buffer of 45 mM TrisBorate, pH 8.3, 1.4 mM EDTA. CLEAVASE BB is thus capable of converting the substrate to mononucleotides.","The nibbling by CLEAVASE BB is duplex dependent. In this Example, internally labeled, single strands of the 206-mer were produced by 15 cycles of primer extension incorporating \u03b1-P labeled dCTP combined with all four unlabeled dNTPs, using an unlabeled 206-bp fragment as a template. Single and double stranded products were resolved by electrophoresis through a non-denaturing 6% polyacrylamide gel (29:1 cross-link) in a buffer of 45 mM TrisBorate, pH 8.3, 1.4 mM EDTA, visualized by autoradiography, excised from the gel, eluted by passive diffusion, and concentrated by ethanol precipitation.","The cleavage reactions comprised 0.04 pmoles of substrate DNA, and 2 \u03bcl of CLEAVASE BB (in an extract as described above) in a total volume of 40 \u03bcl of 10 mM TrisCl, pH 8.5, 50 mM KCl, 1.5 mM MgCl. Reactions were initiated by the addition of pre-warmed enzyme; 10 \u03bcl aliquots were removed at 5, 10, 20, and 30 minutes, and transferred to prepared lubes containing 8 \u03bcl of 95% formamide with 30 mM EDTA and 0.05% marker dyes. Samples were heated to 75\u00b0 C. for 2 minutes immediately before electrophoresis through a 10% acrylamide gel (19:1 cross-linked), with 7 M urea, in a buffer of 45 mM TrisBorate, pH 8.3, 1.4 mM EDTA. Results were visualized by autoradiography as shown in FIG. . Clearly, the cleavage by CLEAVASE BB depends on a duplex structure; no cleavage of the single strand structure is detected whereas cleavage of the 206-mer duplex is complete.","The nibbling activity of the DNAPs of the present invention can be employed with success in a detection assay. One embodiment of such an assay is shown in FIG. . In this assay, a labeled oligo is employed that is specific for a target sequence. The oligo is in excess of the target so that hybridization is rapid. In this embodiment, the oligo contains two fluorescein labels whose proximity on the oligo causes their emission to be quenched. When the DNAP is permitted to nibble the oligo the labels separate and are detectable. The shortened duplex is destabilized and disassociates. Importantly, the target is now free to react with an intact labeled oligo. The reaction can continue until the desired level of detection is achieved. An analogous, although different, type of cycling assay has been described employing lambda exonuclease. See C. G. Copley and C. Boot, 13:888 (1992).","The success of such an assay depends on specificity. In other words, the oligo must hybridize to the specific target. It is also preferred that the assay be sensitive; the oligo ideally should be able to detect small amounts of target.  shows a 5\u2032-end P-labeled primer bound to a plasmid target sequence. In this case, the plasmid was pUC19 (commercially available) which was heat denatured by boiling two (2) minutes and then quick chilling. The primer is a 21 -mer (SEQ ID NO:28). The enzyme employed was CLEAVASE BX (a dilution equivalent to 5\u00d710\u03bcl extract) in 100 mM KCl, 10 mM Tris-Cl, pH 8.3, 2 mM MnCl. The reaction was performed at 55\u00b0 C. for sixteen (16) hours with or without genomic background DNA (from chicken blood). The reaction was stopped by the addition of 8 \u03bcl of 95% formamide with 20 mM EDTA and marker dyes.","The products of the reaction were resolved by PAGE (10% polyacrylamide, 19:1 cross link, 1\u00d7 TBE) as seen in FIG. B. Lane \u201cM\u201d contains the labeled 21 -mer. Lanes 1-3 contain no specific target, although Lanes 2 and 3 contain 100 ng and 200 ng of genomic DNA, respectively. Lanes 4, 5 and 6 all contain specific target with either 0 ng, 100 ng, or 200 ng of genomic DNA, respectively. It is clear that conversion to mononucleotides occurs in Lanes 4, 5 and 6 regardless of the presence or amount of background DNA. Thus, the nibbling can be target directed and specific.","As noted above, expressed thermostable proteins (i.e., the 5\u2032 nucleases), were isolated by crude bacterial cell extracts. The precipitated proteins were then, along with other cell debris, removed by centrifugation. In this Example, cells expressing the BN clone were cultured and collected (500 grams). For each gram (wet weight) of , 3 ml of lysis buffer (50 mM Tris-HCl, pH 8.0, 1 mM EDTA, 100 \u03bcM NaCl) was added. The cells were lysed with 200 \u03bcg\/ml lysozyme at room temperature for 20 minutes. Thereafter deoxycholic acid was added to make a 0.2% final concentration and the mixture was incubated 15 minutes at room temperature.","The lysate was sonicated for approximately 6-8 minutes at 0\u00b0 C. The precipitate was removed by centrifugation (39,000 g for 20 minutes). Polyethyleneimine was added (0.5%) to the supernatant and the mixture was incubated on ice for 15 minutes. The mixture was centrifuged (5,000 g for 15 minutes) and the supernatant was retained. This was heated for 30 minutes at 60\u00b0 C. and then centrifuged again (5,000 g for 15 minutes) and the supematant was again retained.","The supernatant was precipitated with 35% ammonium sulfate at 4\u00b0 C. for 15 minutes. The mixture was then centrifuged (5,000 g for 15 minutes) and the supernatant was removed. The precipitate was then dissolved in 0.25M KCl, 20 Tris pH 7.6, 0.2% Tween and 0.1 EDTA) and then dialyzed against Binding Buffer (8\u00d7 Binding Buffer comprises: 40 mM imidazole, 4M NaCl, 160 mM Tris-HCl, pH 7.9).","The solubilized protein is then purified on the Ni column (Novagen). The Binding Buffer is allows to drain to the top of the column bed and load the column with the prepared extract. A flow rate of about 10 column volumes per hour is optimal for efficient purification. If the flow rate is too fast, more impurities will contaminate the eluted fraction.","The column is washed with 25 ml (10 volumes) of 1\u00d7 Binding Buffer and then washed with 15 ml (6 volumes) of 1\u00d7 Wash Buffer (8\u00d7 Wash Buffer comprises: 480 mM imidazole, 4 M NaCl, 160 mM Tris-HCl, pH 7.9). The bound protein was eluted with 15 ml (6 volumes) of 1\u00d7 Elute Buffer (4\u00d7 Elute Buffer comprises: 4 mM imidazole, 2 M NaCl, 80 mM Tris-HCl, pH 7.9). Protein is then reprecipitated with 35% ammonium sulfate as above. The precipitate was then dissolved and dialyzed against: 20 mM Tris, 100 mM KCl, 1 mM EDTA). The solution was brought up to 0.1% each of Tween 20 and NP-40 and stored at 4\u00b0 C.","In comparing the 5\u2032 nucleases generated by the modification and\/or deletion of the C-terminal polymerization domain of DNA polymerase (DNAPTaq), as diagrammed in , significant differences in the strength of the interactions of these proteins with the 3\u2032 end of primers located upstream of the cleavage site (as depicted in ) were noted. In describing the cleavage of these structures by Pol I-type DNA polymerases (See, Example 1, and Lyamichev et al., Science 260:778 (1993]), it was observed that in the absence of a primer, the location of the junction between the double-stranded region and the single-stranded 5\u2032 and 3\u2032 arms determined the site of cleavage, but in the presence of a primer, the location of the 3\u2032 end of the primer became the determining factor for the site of cleavage. It was postulated that this affinity for the 3\u2032 end was in accord with the synthesizing function of the DNA polymerase.","Structure 2, shown in , was used to test the effects of a 3\u2032 end proximal to the cleavage site in cleavage reactions comprising several different solutions (e.g., solutions containing different salts [KCl or NaCl], different divalent cations [Mn or Mg], etc.) as well as the use of different temperatures for the cleavage reaction. When the reaction conditions were such that the binding of the enzyme (e.g., a DNAP comprising a 5\u2032 nuclease, a modified DNAP or a 5\u2032 nuclease) to the 3\u2032 end (of the pilot oligonucleotide) near the cleavage site was strong, the structure shown is cleaved at the site indicated in FIG. A. This cleavage releases the unpaired 5\u2032 arm and leaves a nick between the remaining portion of the target nucleic acid and the folded 3\u2032 end of the pilot oligonucleotide. In contrast, when the reaction conditions are such that the binding of the DNAP (comprising a 5\u2032 nuclease) to the 3\u2032 end was weak, the initial cleavage was as described above, but after the release of the 5\u2032 arm, the remaining duplex is digested by the exonuclease function of the DNAP.","One way of weakening the binding of the DNAP to the 3\u2032 end is to remove all or part of the domain to which at least some of this function has been attributed. Some of 5\u2032 nucleases created by deletion of the polymerization domain of DNAPTaq have enhanced true exonuclease function, as demonstrated in Example 5.","The affinity of these types of enzymes (i.e., 5\u2032 nucleases associated with or derived from DNAPs) for recessed 3\u2032 ends may also be affected by the identity of the divalent cation present in the cleavage reaction. It was demonstrated by Longley et al. (Nucl. Acids Res., 18:7317 [1990]) that the use of MnClin a reaction with DNAPTaq enabled the polymerase to remove nucleotides from the 5\u2032 end of a primer annealed to a template, albeit inefficiently. Similarly, by examination of the cleavage products generated using Structure 2 from , as described above, in a reaction containing either DNAPTaq or the CLEAVASE BB nuclease, it was observed that the substitution of MnClfor MgClin the cleavage reaction resulted in the exonucleolytic \u201cnibbling\u201d of the duplex downstream of the initial cleavage site. While not limiting the invention to any particular mechanism, it is thought that the substitution of MnClfor MgClin the cleavage reaction lessens the affinity of these enzymes for recessed 3\u2032 ends.","In all cases, the use of MnClenhances the 5\u2032 nuclease function, and in the case of the CLEAVASE BB nuclease, a 50- to 100-fold stimulation of the 5\u2032 nuclease function is seen. Thus, while the exonuclease activity of these enzymes was demonstrated above in the presence of MgCl, the assays described below show a comparable amount of exonuclease activity using 50 to 100-fold less enzyme when MnClis used in place of MgCl. When these reduced amounts of enzyme are used in a reaction mixture containing MgCl, the nibbling or exonuclease activity is much less apparent than that seen in Examples 5-7.","Similar effects are observed in the performance of the nucleic acid detection assay described in Examples 10-39 below when reactions performed in the presence of either MgClor MnClare compared. In the presence of either divalent cation, the presence of the INVADER oligonucleotide (described below) forces the site of cleavage into the probe duplex, but in the presence of MnClthe probe duplex can be further nibbled producing a ladder of products that are visible when a 3\u2032 end label is present on the probe oligonucleotide. When the INVADER oligonucleotide is omitted from a reaction containing Mn, the probe is nibbled from the 5\u2032 end. Mg-based reactions display minimal nibbling of the probe oligonucleotide. In any of these cases, the digestion of the probe is dependent upon the presence of the target nucleic acid. In the examples below, the ladder produced by the enhanced nibbling activity observed in the presence of Mn+ is used as a positive indicator that the probe oligonucleotide has hybridized to the target sequence.","As described in the Examples above, 5\u2032 nucleases cleave near the junction between single-stranded and base-paired regions in a bifurcated duplex, usually about one base pair into the base-paired region. In this Example, it is shown that thermostable 5\u2032 nucleases, including those of the present invention (e.g., CLEAVASE BN nuclease, CLEAVASE A\/G nuclease), have the ability to cleave a greater distance into the base paired region when provided with an upstream oligonucleotide bearing a 3\u2032 region that is homologous to a 5\u2032 region of the subject duplex, as shown in FIG. .",{"@attributes":{"id":"p-0695","num":"0704"},"figref":["FIG. 26","FIG. 26","FIG. 26"]},"The S-60 hairpin molecule was labeled on its 5\u2032 end with biotin for subsequent detection. The S-60 hairpin was incubated in the presence of a thermostable 5\u2032 nuclease in the presence or the absence of the P-15 oligonucleotide. The presence of the full duplex that can be formed by the S-60 hairpin is demonstrated by cleavage with the CLEAVASE BN 5\u2032 nuclease, in a primer-independent fashion (i.e., in the absence of the P-15 oligonucleotide). The release of 18 and 19-nucleotide fragments from the 5\u2032 end of the S-60 hairpin molecule showed that the cleavage occurred near the junction between the single and double stranded regions when nothing is hybridized to the 3\u2032 arm of the S-60 hairpin (, lane 2).","The reactions shown in  were conducted as follows. Twenty finole of the 5\u2032 biotin-labeled hairpin DNA (SEQ ID NO:29) was combined with 0.1 ng of CLEAVASE BN enzyme and 1 \u03bcl of 100 mM MOPS (pH 7.5) containing 0.5% each of Tween-20 and NP-40 in a total volume of 9 \u03bcl. In the reaction shown in lane 1, the enzyme was omitted and the volume was made up by addition of distilled water (this served as the uncut or no enzyme control). The reaction shown in lane 3 of  also included 0.5 pmole of the P15 oligonucleotide (SEQ ID NO:30), which can hybridize to the unpaired 3\u2032 arm of the S-60 hairpin (SEQ ID NO:29), as diagrammed in FIG. .","The reactions were overlaid with a drop of mineral oil, heated to 95\u00b0 C. for 15 seconds, then cooled to 37\u00b0 C., and the reaction was started by the addition of 1 \u03bcl of 10 mM MnClto each tube. After 5 minutes, the reactions were stopped by the addition of 6 \u03bcl of 95% formamide containing 20 mM EDTA and 0.05% marker dyes. Samples were heated to 75\u00b0 C. for 2 minutes immediately before electrophoresis through a 15% acrylamide gel (19:1 cross-linked), with 7 M urea, in a buffer of 45 mM Tris-Borate, pH 8.3, 1.4 mM EDTA.","After electrophoresis, the gel plates were separated allowing the gel to remain flat on one plate. A 0.2 mm-pore positively-charged nylon membrane (NYTRAN, Schleicher and Schuell, Keene, N.H.), pre-wetted in HO, was laid on top of the exposed gel. All air bubbles were removed. Two pieces of 3MM filter paper (Whatman) were then placed on top of the membrane, the other glass plate was replaced, and the sandwich was clamped with binder clips. Transfer was allowed to proceed overnight. After transfer, the membrane was carefully peeled from the gel and allowed to air dry. After complete drying, the membrane was washed in 1.2\u00d7 Sequenase Images Blocking Buffer (United States Biochemical) using 0.3 ml of buffer\/cmof membrane. The wash was performed for 30 minutes at room temperature. A streptavidin-alkaline phosphatase conjugate (SAAP, United States Biochemical) was added to a 1:4000 dilution directly to the blocking solution, and agitated for 15 minutes. The membrane was rinsed bricfly with HO and then washed three times for 5 minutes per wash using 0.5 ml\/cmof 1\u00d7 SAAP buffer (100 mM Tris-HCl, pH 10, 50 mM NaCl) with 0.1% sodium dodecyl sulfate (SDS). The membrane was rinsed briefly with HO between each wash. The membrane was then washed once in 1\u00d7 SAAP buffer containing 1 mM MgClwithout SDS, drained thoroughly and placed in a plastic heat-sealable bag. Using a sterile pipet, 5 mls of CDP-Star\u2122 (Tropix, Bedford, Mass.) chemiluminescent substrate for alkaline phosphatase were added to the bag and distributed over the entire membrane for 2-3 minutes. The CDP-StarTM-treated membrane was exposed to XRP X-ray film (Kodak) for an initial exposure of 10 minutes.","The resulting autoradiograph is shown in FIG. . In , the lane labeled \u201cM\u201d contains the biotinylated P-15 oligonucleotide, which served as a marker. The sizes (in nucleotides) of the uncleaved S-60 hairpin (60 nuc; lane 1), the marker (15 nuc; lane \u201cM\u201d) and the cleavage products generated by cleavage of the S-60 hairpin in the presence (lane 3) or absence (lane 2) of the P-15 oligonucleotide are indicated.","Because the complementary regions of the S-60 hairpin are located on the same molecule, essentially no lag time should be needed to allow hybridization (i.e., to form the duplex region of the hairpin). This hairpin structure would be expected to form long before the enzyme could locate and cleave the molecule. As expected, cleavage in the absence of the primer oligonucleotide was at or near the junction between the duplex and single-stranded regions, releasing the unpaired 5\u2032 arm (, lane 2). The resulting cleavage products were 18 and 19 nucleotides in length.","It was expected that stability of the S-60 hairpin with the tri-loop would prevent the P-15 oligonucleotide from promoting cleavage in the \u201cprimer-directed\u201d manner described in Example 1 above, because the 3\u2032 end of the \u201cprimer\u201d would remain unpaired. Surprisingly, it was found that the enzyme seemed to mediate an \u201cinvasion\u201d by the P-15 primer into the duplex region of the S-60 hairpin, as evidenced by the shifting of the cleavage site 3 to 4 basepairs further into the duplex region, releasing the larger products (22 and 21 nuc.) observed in lane 3 of FIG. .","The precise sites of cleavage of the S-60 hairpin are diagrammed on the structure in , with the solid black arrowheads indicating the sites of cleavage in the absence of the P-15 oligonucleotide and the hollow arrow heads indicating the sites of cleavage in the presence of P-15.","These data show that the presence on the 3\u2032 arm of an oligonucleotide having some sequence homology with the first several bases of the similarly oriented strand of the downstream duplex can be a dominant factor in determining the site of cleavage by 5\u2032 nucleases. Because the oligonucleotide that shares some sequence homology with the first several bases of the similarly oriented strand of the downstream duplex appears to invade the duplex region of the hairpin, it is referred to as an \u201cINVADER\u201d oligonucleotide. As shown in the Examples below, an INVADER oligonucleotide appears to invade (or displace) a region of duplexed nucleic acid regardless of whether the duplex region is present on the same molecule (i.e., a hairpin) or whether the duplex is formed between two separate nucleic acid strands.","In Example 10, it was demonstrated that an INVADER oligonucleotide could shift the site at which a 5\u2032 nuclease cleaves a duplex region present on a hairpin molecule. In this Example, the ability of an INVADER oligonucleotide to shift the site of cleavage within a duplex region formed between two separate strands of nucleic acid molecules was examined.","A single-stranded target DNA comprising the single-stranded circular M13mp19 molecule and a labeled (fluorescein) probe oligonucleotide were mixed in the presence of the reaction buffer containing salt (KCl) and divalent cations (Mg or Mn) to promote duplex formation. The probe oligonucleotide refers to a labeled oligonucleotide that is complementary to a region along the target molecule (e.g., M13mp19). A second oligonucleotide (unlabeled) was added to the reaction after the probe and target had been allowed to anneal. The second oligonucleotide binds to a region of the target that is located downstream of the region to which the probe oligonucleotide binds. This second oligonucleotide contains sequences that are complementary to a second region of the target molecule. If the second oligonucleotide contains a region that is complementary to a portion of the sequences along the target to which the probe oligonucleotide also binds, this second oligonucleotide is referred to as an INVADER oligonucleotide (see ).",{"@attributes":{"id":"p-0707","num":"0716"},"figref":["FIG. 32","FIG. 28","FIG. 28","FIG. 28","FIG. 28","FIG. 28","FIG. 28"],"i":["a","a","b","c"]},"The effect of the presence of an INVADER oligonucleotide upon the pattern of cleavage in a probe\/target duplex formed prior to the addition of the INVADER was examined. The INVADER oligonucleotide and the enzyme were added after the probe was allowed to anneal to the target and the position and extent of cleavage of the probe were examined to determine a) if the INVADER was able to shift the cleavage site to a specific internal region of the probe, and b), if the reaction could accumulate specific cleavage products over time, even in the absence of thermal cycling, polymerization, or exonuclease removal of the probe sequence.","The reactions were carried out as follows. Twenty \u03bcl each of two enzyme mixtures were prepared, containing 2 \u03bcl of CLEAVASE A\/G nuclease extract (prepared as described in Example 2), with or without 50 pmole of the INVADER oligonucleotide (SEQ ID NO:35), as indicated, per 4 \u03bcl of the mixture. For each of the eight reactions shown in , 150 fmole of M13mp19 single-stranded DNA (available from Life Technologies, Inc.) was combined with 5 pmoles of fluorescein labeled probe (SEQ ID NO:32), to create the structure shown in , but without the INVADER oligonucleotide present (the probe\/target mixture). One half (4 tubes) of the probe\/target mixtures were combined with 1 \u03bcl of 100 mM MOPS, pH 7.5 with 0.5% each of Twecn-20 and NP-40, 0.5 \u03bcl of 1 M KCl and 0.25 \u03bcl of 80 mM MnCl, and distilled water to a volume of 6 \u03bcl. The second set of probe\/target mixtures were combined with 1 \u03bcl of 100 mM MOPS, pH 7.5 with 0.5% each of Tween-20 and NP-40, 0.5 \u03bcl of 1 M KCl and 0.25 \u03bcl of 80 mM MgCl. The second set of mixtures therefore contained MgClin place of the MnClpresent in the first set of mixtures.","The mixtures (containing the probe\/target with buffer, KCl and divalent cation) were covered with a drop of CHILLOUT evaporation barrier and were brought to 60\u00b0 C. for 5 minutes to allow annealing. Four \u03bcl of the above enzyme mixtures without the INVADER oligonucleotide was added to reactions whose products are shown in lanes 1, 3, 5 and 7 of FIG. . Reactions whose products are shown lanes 2, 4, 6, and 8 of  received the same amount of enzyme mixed with the INVADER oligonucleotide (SEQ ID NO:35). Reactions 1, 2, 5 and 6 were incubated for 5 minutes at 60\u00b0 C. and reactions 3, 4, 7 and 8 were incubated for 15 minutes at 60\u00b0 C.","All reactions were stopped by the addition of 8 \u03bcl of 95% formamide with 20 mM EDTA and 0.05% marker dyes. Samples were heated to 90\u00b0 C. for 1 minute immediately before electrophoresis through a 20% acrylamide gel (19:1 cross-linked), containing 7 M urea, in a buffer of 45 mM Tris-Borate, pH 8.3, 1.4 mM EDTA. Following electrophoresis, the reaction products and were visualized by the use of an Hitachi FMBIO fluorescence imager, the output of which is seen in FIG. . The very low molecular weight fluorescent material seen in all lanes at or near the salt front in FIG.  and other fluoro-imager Figures is observed when fluorescently-labeled oligonucleotides arc electrophoresed and imaged on a fluoro-imager. This material is not a product of the cleavage reaction.","The use of MnClin these reactions (lanes 1-4) stimulates the true exonuclease or \u201cnibbling\u201d activity of the CLEAVASE enzyme, as described in Example 6, as is clearly seen in lanes 1 and 3 of FIG. . This nibbling of the probe oligonucleotide (SEQ ID NO:32) in the absence of INVADER oligonucleotide (SEQ ID NO:35) confirms that the probe oligonucleotide is forming a duplex with the target sequence. The ladder-like products produced by this nibbling reaction may be difficult to differentiate from degradation of the probe by nucleases that might be present in a clinical specimen. In contrast, introduction of the INVADER oligonucleotide (SEQ ID NO:35) caused a distinctive shift in the cleavage of the probe, pushing the site of cleavage 6 to 7 bases into the probe, confirming the annealing of both oligonucleotides. In presence of MnCl, the exonuclease \u201cnibbling\u201d may occur after the INVADER-directed cleavage event, until the residual duplex is destabilized and falls apart.","In a magnesium based cleavage reaction (lanes 5-8), the nibbling or true exonuclease function of the CLEAVASE A\/G is enzyme suppressed (but the endonucleolytic function of the enzyme is essentially unaltered), so the probe oligonucleotide is not degraded in the absence of the INVADER (, lanes 5 and 7). When the INVADER is added, it is clear that the INVADER oligonucleotide can promote a shift in the site of the endonucleolytic cleavage of the annealed probe. Comparison of the products of the 5 and 15 minute reactions with INVADER (lanes 6 and 8 in ) shows that additional probe hybridizes to the target and is cleaved. The calculated melting temperature (T) of the portion of probe that is not invaded (i.e., nucleotides 9-26 of SEQ ID NO:32) is 56\u00b0 C., so the observed turnover (as evidenced by the accumulation of cleavage products with increasing reaction time) suggests that the full length of the probe molecule, with a calculated Tof 76\u00b0 C., is must be involved in the subsequent probe annealing events in this 60\u00b0 C. reaction.","In Example 11, the ability of an INVADER oligonucleotide to cause a shift in the site of cleavage of a probe annealed to a target molecule was demonstrated. In this Example, experiments were conducted to examine whether the presence of an oligonucleotide upstream from the probe was sufficient to cause a shift in the cleavage site(s) along the probe or whether the presence of nucleotides on the 3\u2032 end of the INVADER oligonucleotide that have the same sequence as the first several nucleotides at the 5\u2032 end of the probe oligonucleotide were required to promote the shift in cleavage.","To examine this point, the products of cleavage obtained from three different arrangements of target-specific oligonucleotides are compared. A diagram of these oligonucleotides and the way in which they hybridize to a test nucleic acid, M13mp19, is shown in FIG. . In , the 3\u2032 end of the upstream oligonucleotide (SEQ ID NO:33) is located upstream of the 5\u2032 end of the downstream \u201cprobe\u201d oligonucleotide (SEQ ID NO:32) such that a region of the M13 target that is not paired to either oligonucleotide is present. In , the sequence of the upstream oligonucleotide (SEQ ID NO:34) is immediately upstream of the probe (SEQ ID NO:32), having neither a gap nor an overlap between the sequences. diagrams the arrangement of the substrates used in the assay of the present invention, showing that the upstream \u201cINVADER\u201d oligonucleotide (SEQ ID NO:35) has the same sequence on a portion of its 3\u2032 region as that present in the 5\u2032 region of the downstream probe (SEQ ID NO:32). That is to say, these regions will compete to hybridize to the same segment of the M13 target nucleic acid.","In these experiments, four enzyme mixtures were prepared as follows (planning 5 \u03bcl per digest): Mixture 1 contained 2.25 \u03bcl of CLEAVASE A\/G nuclease extract (prepared as described in Example 2) per 5 \u03bcl of mixture, in 20 mM MOPS, pH 7.5 with 0.1% each of Tween 20 and NP-40, 4 mM MnCland 100 mM KCl. Mixture 2 contained 11.25 units of Taq DNA polymerase (Promega) per 5 \u03bcl of mixture in 20 mM MOPS, pH 7.5 with 0.1% each of Tween 20 and NP-40, 4 mM MnCland 100 mM KCl. Mixture 3 contained 2.25 \u03bcl of CLEAVASE A\/G nuclease extract per 5 \u03bcl of mixture in 20 mM Tris-HCl, pH 8.5, 4 mM MgCland 100 mM KCl. Mixture 4 contained 11.25 units of Taq DNA polymerase per 5 \u03bcl of mixture in 20 mM Tris-HCl, pH 8.5, 4 mM MgCland 100 mM KCl.","For each reaction, 50 fmole of M13mp19 single-stranded DNA (the target nucleic acid) was combined with 5 pmole of the probe oligonucleotide (SEQ ID NO:32 which contained a fluorescein label at the 3\u2032 end) and 50 pmole of one of the three upstream oligonucleotides diagrammed in  (i.e., one of SEQ ID NOS:33-35), in a total volume of 5 \u03bcl of distilled water. The reactions were overlaid with a drop of ChillOut\u2122 evaporation barrier and warmed to 62\u00b0 C. The cleavage reactions were started by the addition of 5 \u03bcl of an enzyme mixture to each tube, and the reactions were incubated at 62\u00b0 C. for 30 min. The reactions shown in lanes 1-3 of  received Mixture 1; reactions 4-6 received Mixture 2; reactions 7-9 received Mixture 3 and reactions 10-12 received Mixture 4.","After 30 minutes at 62\u00b0 C., the reactions were stopped by the addition of 8 \u03bcl of 95% formamide with 20 mM EDTA and 0.05% marker dyes. Samples were heated to 75\u00b0 C. for 2 minutes immediately before electrophoresis through a 20% acrylamide gel (19:1 cross-linked), with 7 M urea, in a buffer of 45 mM Tris-Borate, pH 8.3, 1.4 mM EDTA.","Following electrophoresis, the products of the reactions were visualized by the use of an Hitachi FMBIO fluorescence imager, the output of which is seen in FIG. . The reaction products shown in lanes 1, 4, 7 and 10 of  were from reactions that contained SEQ ID NO:33 as the upstream oligonucleotide (see ). The reaction products shown in lanes 2, 5, 8 and 11 of  were from reactions that contained SEQ ID NO:34 as the upstream oligonucleotide (see ). The reaction products shown in lanes 3, 6, 9 and 12 of  were from reactions that contained SEQ ID NO:35, the INVADER oligonucleotide, as the upstream oligonucleotide (see ).","Examination of the Mn based reactions using either CLEAVASE A\/G nuclease or DNAPTaq as the cleavage agent (lanes 1 through 3 and 4 through 6, respectively) shows that both enzymes have active exonuclease function in these buffer conditions. The use of a 3\u2032 label on the probe oligonucleotide allows the products of the nibbling activity to remain labeled, and therefore visible in this assay. The ladders seen in lanes 1, 2, 4 and 5 confirm that the probe hybridize to the target DNA as intended. These lanes also show that the location of the non-invasive oligonucleotides have little effect on the products generated. The uniform ladder created by these digests would be difficult to distinguish from a ladder causes by a contaminating nuclease, as one might find in a clinical specimen. In contrast, the products displayed in lanes 3 and 6, where an INVADER oligonucleotide was provided to direct the cleavage, show a very distinctive shift, so that the primary cleavage product is smaller than those seen in the non-invasive cleavage. This product is then subject to further nibbling in these conditions, as indicated by the shorter products in these lanes. These INVADER-directed cleavage products would be easily distinguished from a background of non-specific degradation of the probe oligonucleotide.","When Mg is used as the divalent cation the results are even more distinctive. In lanes 7, 8, 10 and 11 of , where the upstream oligonucleotides were not invasive, minimal nibbling is observed. The products in the DNAPTaq reactions show some accumulation of probe that has been shortened on the 5\u2032 end by one or two nucleotides consistent with previous examination of the action of this enzyme on nicked substrates (Longley et al., supra). When the upstream oligonucleotide is invasive, however, the appearance of the distinctively shifted probe band is seen. These data clearly indicated that it is the invasive 3\u2032 portion of the upstream oligonucleotide that is responsible for fixing the site of cleavage of the downstream probe.","Thus, the above results demonstrate that it is the presence of the free or initially non-annealed nucleotides at the 3\u2032 end of the INVADER oligonucleotide that mediate the shift in the cleavage site, not just the presence of an oligonucleotide annealed upstream of the probe. Nucleic acid detection assays that employ the use of an INVADER oligonucleotide are termed \u201cINVADER-directed cleavage\u201d assays.","For a nucleic acid detection method to be broadly useful, it must be able to detect a specific target in a sample that may contain large amounts of other DNA, (e.g., bacterial or human chromosomal DNA). The ability of the INVADER directed cleavage assay to recognize and cleave either single- or double-stranded target molecules in the presence of large amounts of non-target DNA was examined. In these experiments a model target nucleic acid, M13, in either single or double stranded form (single-stranded M13mp18 is available from Life Technologies, Inc and double-stranded M13mp19 is available from NEB), was combined with human genomic DNA (Novagen) and then utilized in INVADER-directed cleavage reactions. Before the start of the cleavage reaction, the DNAs were heated to 95\u00b0 C. for 15 minutes to completely denature the samples, as is standard practice in assays, such as polymerase chain reaction or enzymatic DNA sequencing, which involve solution hybridization of oligonucleotides to double-stranded target molecules.","For each of the reactions shown in lanes 2-5 of , the target DNA (25 fmole of the ss DNA or 1 pmole of the ds DNA) was combined with 50 pmole of the INVADER oligonucleotide (SEQ ID NO:35); for the reaction shown in lane 1 the target DNA was omitted. Reactions 1, 3 and 5 also contained 470 ng of human genomic DNA. These mixtures were brought to a volume of 10 \u03bcl with distilled water, overlaid with a drop of ChillOut\u2122 evaporation barrier, and brought to 95\u00b0 C. for 15 minutes. After this incubation period, and still at 95\u00b0 C., each tube received 10 \u03bcl of a mixture comprising 2.25 \u03bcl of CLEAVASE A\/G nuclease extract (prepared as described in Example 2) and 5 pmole of the probe oligonucleotide (SEQ ID NO:32), in 20 mM MOPS, pH 7.5 with 0.1 % each of Tween 20 and NP-40, 4 nM MnCland 100 mM KCl. The reactions were brought to 62\u00b0 C. for 15 minutes and stopped by the addition of 12 \u03bcl of 95% formamide with 20 mM EDTA and 0.05% marker dyes. Samples were heated to 75\u00b0 C. for 2 minutes immediately before electrophoresis through a 20% acrylamide gel (19:1 cross-linked), with 7 M urea, in a buffer of 45 mM Tris-Borate, pH 8.3, 1.4 mM EDTA. The products of the reactions were visualized by the use of an Hitachi FMBIO fluorescence imager. The results are displayed in FIG. .","In , lane 1 contains the products of the reaction containing the probe (SEQ ID NO:32), the INVADER oligonucleotide (SEQ ID NO:35) and human genomic DNA. Examination of lane 1 shows that the probe and INVADER oligonucleotides are specific for the target sequence, and that the presence of genomic DNA does not cause any significant background cleavage.","In , lanes 2 and 3 contain reaction products from reactions containing the single-stranded target DNA (M13mp18), the probe (SEQ ID NO:32) and the INVADER oligonucleotide (SEQ ID NO:35) in the absence or presence of human genomic DNA, respectively. Examination of lanes 2 and 3 demonstrate that the INVADER detection assay may be used to detect the presence of a specific sequence on a single-stranded target molecule in the presence or absence of a large excess of competitor DNA (human genomic DNA).","In , lanes 4 and 5 contain reaction products from reactions containing the double-stranded target DNA (M13mp19), the probe (SEQ ID NO:32) and the INVADER oligonucleotide (SEQ ID NO:35) in the absence or presence of human genomic DNA, respectively. Examination of lanes 4 and 5 show that double stranded target molecules are eminently suitable for INVADER-directed detection reactions. The success of this reaction using a short duplexed molecule, M13mp19, as the target in a background of a large excess of genomic DNA is especially noteworthy as it would be anticipated that the shorter and less complex M13 DNA strands would be expected to find their complementary strand more easily than would the strands of the more complex human genomic DNA. If the M13 DNA reannealed before the probe and\/or INVADER oligonucleotides could bind to the target sequences along the M13 DNA, the cleavage reaction would be prevented. In addition, because the denatured genomic DNA would potentially contain regions complementary to the probe and\/or INVADER oligonucleotides it was possible that the presence of the genomic DNA would inhibit the reaction by binding these oligonucleotides thereby preventing their hybridization to the M13 target. The above results demonstrate that these theoretical concerns are not a problem under the reaction conditions employed above.","In addition to demonstrating that the INVADER detection assay may be used to detect sequences present in a double-stranded target, these data also show that the presence of a large amount of non-target DNA (470 ng\/20 \u03bcl reaction) does not lessen the specificity of the cleavage. While this amount of DNA does show some impact on the rate of product accumulation, probably by binding a portion of the enzyme, the nature of the target sequence, whether single- or double-stranded nucleic acid, does not limit the application of this assay.","To investigate whether the INVADER-directed cleavage assay could be used to indicate the amount of target nucleic acid in a sample, the following experiment was performed. Cleavage reactions were assembled that contained an INVADER oligonucleotide (SEQ ID NO:35), a labeled probe (SEQ ID NO:32) and a target nucleic acid, M13mp19. A series of reactions, which contained smaller and smaller amounts of the M13 target DNA, was employed in order to examine whether the cleavage products would accumulate in a manner that reflected the amount of target DNA present in the reaction.","The reactions were conducted as follows. A master mix containing enzyme and buffer was assembled. Each 5 \u03bcl of the master mixture contained 25 ng of CLEAVASE BN nuclease in 20 mM MOPS (pH 7.5) with 0.1% each of Tween 20 and NP-40, 4 mM MnCland 100 mM KCl. For each of the cleavage reactions shown in lanes 4-13 of , a DNA mixture was generated that contained 5 pmoles of the fluorescein-labeled probe oligonucleotide (SEQ ID NO:32), 50 pmoles of the INVADER oligonucleotide (SEQ ID NO:35) and 100, 50, 10, 5, 1, 0.5, 0.1, 0.05, 0.01 or 0.005 fmoles of single-stranded M13mp19, respectively, for every 5 \u03bcl of the DNA mixture. The DNA solutions were covered with a drop of CHILLOUT evaporation barrier and brought to 61\u00b0 C. The cleavage reactions were started by the addition of 5 \u03bcl of the enzyme mixture to each of tubes (final reaction volume was 10\u03bcl). After 30 minutes at 61\u00b0 C., the reactions were terminated by the addition of 8 \u03bcl of 95% formamide with 20 mM EDTA and 0.05% marker dyes. Samples were heated to 90\u00b0 C. for 1 minute immediately before electrophoresis through a 20% denaturing acrylamide gel (19:1 cross-linked) with 7 M urea, in a buffer containing 45 mM Tris-Borate (pH 8.3), 1.4 mM EDTA. To provide reference (i.e., standards), 1.0, 0.1 and 0.01 pmole aliquots of fluorescein-labeled probe oligonucleotide (SEQ ID NO:32) were diluted with the above formamide solution to a final volume of 18 \u03bcl. These reference markers were loaded into lanes 1-3, respectively of the gel. The products of the cleavage reactions (as well as the reference standards) were visualized following electrophoresis by the use of a Hitachi FMBIO fluorescence imager. The results are displayed in FIG. .","In , boxes appear around fluorescein-containing nucleic acid (i.e., the cleaved and uncleaved probe molecules) and the amount of fluorescein contained within each box is indicated under the box. The background fluorescence of the gel (see box labeled \u201cbackground\u201d) was subtracted by the fluoro-imager to generate each value displayed under a box containing cleaved or uncleaved probe products (the boxes are numbered 1-14 at top left with a V followed by a number below the box). The lane marked \u201cM\u201d contains fluoresceinated oligonucleotides, which served as markers.","The results shown in , demonstrate that the accumulation of cleaved probe molecules in a fixed-length incubation period reflects the amount of target DNA present in the reaction. The results also demonstrate that the cleaved probe products accumulate in excess of the copy number of the target. This is clearly demonstrated by comparing the results shown in lane 3, in which 10 fmole (0.01 pmole) of uncut probe are displayed with the results shown in 5, where the products that accumulated in response to the presence of 10 fmole of target DNA are displayed. These results show that the reaction can cleave hundreds of probe oligonucleotide molecules for each target molecule present, dramatically amplifying the target-specific signal generated in the INVADER-directed cleavage reaction.","For a nucleic acid detection method to be useful in a medical (i.e., a diagnostic) setting, it must not be inhibited by materials and contaminants likely to be found in a typical clinical specimen. To test the susceptibility of the INVADER-directed cleavage assay to various materials, including but not limited to nucleic acids, glycoproteins and carbohydrates, likely to be found in a clinical sample, a sample of human saliva was prepared in a manner consistent with practices in the clinical laboratory and the resulting saliva extract was added to the INVADER-directed cleavage assay. The effect of the saliva extract upon the inhibition of cleavage and upon the specificity of the cleavage reaction was examined.","One and one-half milliliters of human saliva were collected and extracted once with an equal volume of a mixture containing phenol:chloroform:isoamyl alcohol (25:24:1). The resulting mixture was centrifuged in a microcentrifuge to separate the aqueous and organic phases. The upper, aqueous phase was transferred to a fresh tube. One-tenth volumes of 3 M NaOAc were added and the contents of the tube were mixed. Two volumes of 100% ethyl alcohol were added to the mixture and the sample was mixed and incubated at room temperature for 15 minutes to allow a precipitate to form. The sample was centrifuged in a microcentrifuge at 13,000 rpm for 5 minutes and the supernatant was removed and discarded. A milky pellet was easily visible. The pellet was rinsed once with 70% ethanol, dried under vacuum and dissolved in 200 \u03bcl of 10 mM Tris-HCl, pH 8.0, 0.1 mM EDTA (this constitutes the saliva extract). Each \u03bcl of the saliva extract was equivalent to 7.5 \u03bcl of saliva. Analysis of the saliva extract by scanning ultraviolet spectrophotometry showed a peak absorbance at about 260 nm and indicated the presence of approximately 45 ng of total nucleic acid per \u03bcl of extract.","The effect of the presence of saliva extract upon the following enzymes was examined: CLEAVASE BN nuclease, CLEAVASE A\/G nuclease and three different lots of DNAPTaq: AmpliTaq\u00ae (Perkin Elmer; a recombinant form of DNAPTaq), AmpliTaq\u00ae LD (Perkin-Elmer; a recombinant DNAPTaq preparation containing very low levels of DNA) and Taq DNA polymerase (Fischer). For each enzyme tested, an enzyme\/probe mixture was made comprising the chosen amount of enzyme with 5 pmole of the probe oligonucleotide (SEQ ID NO:32) in 10 \u03bcl of 20 mM MOPS (pH 7.5) containing 0.1% each of Tween 20 and NP-40, 4 mM MnCl, 100 mM KCl and 100 \u03bcg\/ml BSA. The following amounts of enzyme were used: 25 ng of CLEAVASE BN prepared as described in Example 8; 2 \u03bcl of CLEAVASE A\/G nuclease extract prepared as described in Example 2; 2.25 \u03bcl (11.25 polymerase units) the following DNA polymerases: AmpliTaq\u00ae DNA polymerase (Perkin Elmer); AmpliTaq\u00ae DNA polymerase LD (low DNA; from Perkin Elmer); Taq DNA polymerase (Fisher Scientific).","For each of the reactions shown in , except for that shown in lane 1, the target DNA (50 fmoles of single-stranded M13mp19 DNA) was combined with 50 pmole of the INVADER oligonucleotide (SEQ ID NO:35) and 5 pmole of the probe oligonucleotide (SEQ ID NO:32); target DNA was omitted in reaction 1 (lane 1). Reactions 1, 3, 5, 7, 9 and 11 included 1.5 \u03bcl of saliva extract. These mixtures were brought to a volume of 5 \u03bcl with distilled water, overlaid with a drop of CHILLOUT evaporation barrier and brought to 95\u00b0 C. for 10 minutes. The cleavage reactions were then started by the addition of 5 \u03bcl of the desired enzyme\/probe mixture; reactions 1, 4 and 5 received CLEAVASE A\/G nuclease. Reactions 2 and 3 received CLEAVASE BN; reactions 6 and 7 received AmpliTaq\u00ae; reactions 8 and 9 received AmpliTaq\u00ae LD; and reactions 10 and 11 received Taq DNA Polymerase from Fisher Scientific.","The reactions were incubated at 63\u00b0 C. for 30 minutes and were stopped by the addition of 6 \u03bcl of 95% formamide with 20 mM EDTA and 0.05% marker dyes. Samples were heated to 75\u00b0 C. for 2 minutes immediately before electrophoresis through a 20% acrylamide gel (19:1 cross-linked), with 7 M urea, in a buffer of 45 mM Tris-Boratc, pH 8.3, 1.4 mM EDTA. The products of the reactions were visualized by the use of an Hitachi FMBIO fluorescence imager, and the results are displayed in FIG. .","A pairwise comparison of the lanes shown in  without and with the saliva extract, treated with each of the enzymes, shows that the saliva extract has different effects on each of the enzymes. While the CLEAVASE BN nuclease and the AmpliTaq\u00ae are significantly inhibited from cleaving in these conditions, the CLEAVASE A\/G nuclease and AmpliTaq\u00ae LD display little difference in the yield of cleaved probe. The preparation of Taq DNA polymerase from Fisher Scientific shows an intermediate response, with a partial reduction in the yield of cleaved product. From the standpoint of polymerization, the three DNAPTaq variants should be equivalent; these should be the same protein with the same amount of synthetic activity. It is possible that the differences observed could be due to variations in the amount of nuclease activity present in each preparation caused by different handling during purification, or by different purification protocols. In any case, quality control assays designed to assess polymerization activity in commercial DNAP preparations would be unlikely to reveal variation in the amount of nuclease activity present. If preparations of DNAPTaq were screened for full 5\u2032 nuclease activity (i.e., if the 5\u2032 nuclease activity was specifically quantitated), it is likely that the preparations would display sensitivities (to saliva extract) more in line with that observed using CLEAVASE A\/G nuclease, from which DNAPTaq differs by a very few amino acids.","It is worthy of note that even in the slowed reactions of CLEAVASE BN and the DNAPTaq variants there is no noticeable increase in non-specific cleavage of the probe oligonucleotide due to inappropriate hybridization or saliva-borne nucleases.","A number of eubacterial Type A DNA polymerases (i.e., Pol I type DNA polymerases) have been shown to function as structure specific endonucleases (See, Example 1, and Lyamichev et al., supra). In this Example, it was demonstrated that the enzymes of this class can also be made to catalyze the INVADER-directed cleavage of the present invention, albeit not as efficiently as the CLEAVASE enzymes.","CLEAVASE BN nuclease and CLEAVASE A\/G nuclease were tested along side three different thermostable DNA polymerases: DNA polymerase (Promega), and DNA polymerases (Epicentre). The enzyme mixtures used in the reactions shown in lanes 1-11 of  contained the following, each in a volume of 5 \u03bcl: Lane 1: 20 mM MOPS (pH 7.5) with 0.1% each of Tween 20 and NP-40, 4 mM MnCl, 100 mM KCl; Lane 2: 25 ng of CLEAVASE BN nuclease in the same solution described for lane 1; Lane 3: 2.25 \u03bcl of CLEAVASE A\/G nuclease extract (prepared as described in Example 2), in the same solution described for lane 1; Lane 4: 2.25 \u03bcl of CLEAVASE A\/G nuclease extract in 20 mM Tris-Cl, (pH 8.5), 4 mM MgCland 100 mM KCl; Lane 5: 11.25 polymerase units of Taq DNA polymerase in the same buffer described for lane 4; Lane 6: 11.25 polymerase units of Tth DNA polymerase in the same buffer described for lane 1; Lane 7: 11.25 polymerase units of Tth DNA polymerase in a 2\u00d7 concentration of the buffer supplied by the manufacturer, supplemented with 4 mM MnCl; Lane 8: 11.25 polymerase units of Tth DNA polymerase in a 2\u00d7 concentration of the buffer supplied by the manufacturer, supplemented with 4 mM MgCl; Lane 9: 2.25 polymerase units of Tfl DNA polymerase in the same buffer described for lane 1; Lane 10: 2.25 polymerase units of Tfl polymerase in a 2\u00d7 concentration of the buffer supplied by the manufacturer, supplemented with 4 mM MnCl; Lane 11: 2.25 polymerase units of Tfl DNA polymerase in a 2\u00d7 concentration of the buffer supplied by the manufacturer, supplemented with 4 mM MgCl.","Sufficient target DNA, probe and INVADER for all 11 reactions was combined into a master mix. This mix contained 550 fmoles of single-stranded M13mp19 target DNA, 550 pmoles of the INVADER oligonucleotide (SEQ ID NO:35) and 55 pmoles of the probe oligonucleotide (SEQ ID NO:32), each as depicted in , in 55 \u03bcl of distilled water. Five \u03bcl of the DNA mixture was dispensed into each of 11 labeled tubes and overlaid with a drop of CHILLOUT evaporation barrier. The reactions were brought to 63\u00b0 C. and cleavage was started by the addition of 5 \u03bcl of the appropriate enzyme mixture. The reaction mixtures were then incubated at 63\u00b0 C. temperature for 15 minutes. The reactions were stopped by the addition of 8 \u03bcl of 95% formamide with 20 mM EDTA and 0.05% marker dyes. Samples were heated to 90\u00b0 C. for 1 minute immediately before electrophoresis through a 20% acrylamide gel (19:1 cross-linked), with 7 M urea, in a buffer of 45 mM Tris-Borate (pH 8.3), 1.4 mM EDTA. Following electrophoresis, the products of the reactions were visualized by the use of an Hitachi FMBIO fluorescence imager, and the results are displayed in FIG. . Examination of the results shown in  demonstrates that all of the 5\u2032 nucleases tested have the ability to catalyze INVADER-directed cleavage in at least one of the buffer systems tested. Although not optimized here, these cleavage agents are suitable for use in the methods of the present invention.","The ability of the INVADER-directed cleavage assay to detect single base mismatch mutations was examined. Two target nucleic acid sequences containing CLEAVASE enzyme-resistant phosphorothioate backbones were chemically synthesized and purified by polyacrylamide gel electrophoresis. Targets comprising phosphorothioate backbones were used to prevent exonucleolytic nibbling of the target when duplexed with an oligonucleotide. A target oligonucleotide, which provides a target sequence that is completely complementary to the INVADER oligonucleotide (SEQ ID NO:35) and the probe oligonucleotide (SEQ ID NO:32), contained the following sequence: 5\u2032-CCTTTCGCTTTCTTCCCTTCCTTTCTCGCCACGTTCGCCGGC-3\u2032 (SEQ ID NO:36). A second target sequence containing a single base change relative to SEQ ID NO:36 was synthesized: 5\u2032-CCTTTCGTCTCTTCCCTTCCTTTCTCGCC ACGTTCGCCGGC-3 (SEQ ID NO:37; the single base change relative to SEQ ID NO:36 is shown using bold and underlined type). The consequent mismatch occurs within the \u201cZ\u201d region of the target as represented in FIG. .","To discriminate between two target sequences that differ by the presence of a single mismatch), INVADER-directed cleavage reactions were conducted using two different reaction temperatures (55\u00b0 C. and 60\u00b0 C.). Mixtures containing 200 fmoles of either SEQ ID NO:36 or SEQ ID NO:37, 3 pmoles of fluorescein-labeled probe oligonucleotide (SEQ ID NO:32), 7.7 pmoles of INVADER oligonucleotide (SEQ ID NO:35) and 2 \u03bcl of CLEAVASE AIG nuclease extract (prepared as described in Example 2) in 9 \u03bcl of 10 mM MOPS (pH 7.4) with 50 mM KCl were assembled, covered with a drop of CHILLOUT evaporation barrier and brought to the appropriate reaction temperature. The cleavage reactions were initiated by the addition of 1 \u03bcl of 20 mM MgCl. After 30 minutes at either 55\u00b0 C. or 60\u00b0 C., 10 \u03bcl of 95% formamide with 20 mM EDTA and 0.05% marker dyes was added to stop the reactions. The reaction mixtures where then heated to 90\u00b0 C. for one minute prior to loading 4 \u03bcl onto 20% denaturing polyacrylamide gels. The resolved reaction products were visualized using a Hitachi FMBIO fluorescence imager. The resulting image is shown in FIG. .","In , lanes 1 and 2 show the products from reactions conducted at 55\u00b0 C.; lanes 3 and 4 show the products from reactions conducted at 60\u00b0 C. Lanes 1 and 3 contained products from reactions containing SEQ ID NO:36 (perfect match to probe) as the target. Lanes 2 and 4 contained products from reactions containing SEQ ID NO:37 (single base mis-match with probe) as the target. The target that does not have a perfect hybridization match (i.e., complete complementarity) with the probe will not bind as strongly (i.e., the Tof that duplex will be lower than the Tof the same region if perfectly matched). The results presented here show that reaction conditions can be varied to either accommodate the mis-match (e.g., by lowering the temperature of the reaction) or to exclude the binding of the mismatched sequence (e.g., by raising the reaction temperature).","The results shown in  demonstrate that the specific cleavage event that occurs in INVADER-directed cleavage reactions can be eliminated by the presence of a single base mis-match between the probe oligonucleotide and the target sequence. Thus, reaction conditions can be chosen so as to exclude the hybridization of mismatched INVADER-directed cleavage probes thereby diminishing or even eliminating the cleavage of the probe. In an extension of this assay system, multiple cleavage probes, each possessing a separate reporter molecule (i.e., a unique label), could also be used in a single cleavage reaction, to simultaneously probe for two or more variants in the same target region. The products of such a reaction would allow not only the detection of mutations that exist within a target molecule, but would also allow a determination of the relative concentrations of each sequence (i.e., mutant and wild type or multiple different mutants) present within samples containing a mixture of target sequences. When provided in equal amounts, but in a vast excess (e.g., at least a 100-fold molar excess; typically at least 1 pmole of each probe oligonucleotide would be used when the target sequence was present at about 10 fmoles or less) over the target and used in optimized conditions. As discussed above, any differences in the relative amounts of the target variants will not affect the kinetics of hybridization, so the amounts of cleavage of each probe will reflect the relative amounts of each variant present in the reaction.","The results shown in the Example clearly demonstrate that the INVADER-directed cleavage reaction can be used to detect single base difference between target nucleic acids.","The results shown above demonstrated that the INVADER-directed cleavage reaction can be used for the detection of target nucleic acid sequences and that this assay can be used to detect single base difference between target nucleic acids. These results demonstrated that 5\u2032 nucleases (e.g., CLEAVASEBN, CLEAVASE A\/G, DNAPTaq, DNAPTth, DNAPTfl) could be used in conjunction with a pair of overlapping oligonucleotides as an efficient way to recognize nucleic acid targets. In the experiments below it is demonstrated that invasive cleavage reaction is relatively insensitive to large changes in conditions thereby making the method suitable for practice in clinical laboratories.","The effects of varying the conditions of the cleavage reaction were examined for their effect(s) on the specificity of the invasive cleavage and the on the amount of signal accumulated in the course of the reaction. To compare variations in the cleavage reaction a \u201cstandard\u201d INVADER cleavage reaction was first defined. In each instance, unless specifically stated to be otherwise, the indicated parameter of the reaction was varied, while the invariant aspects of a particular test were those of this standard reaction. The results of these tests arc either shown in , or the results described below.","a) The Standard INVADER-Directed Cleavage Reaction","The standard reaction was defined as comprising 1 fmole of M13mp18 single-stranded target DNA (NEB), 5 pmoles of the labeled probe oligonucleotide (SEQ ID NO:38), 10 pmole of the upstream INVADER oligonucleotide (SEQ ID NO:39) and 2 units of CLEAVASE A\/G in 10 \u03bcl of 10 mM MOPS, pH 7.5 with 100 mM KCl, 4 mM MnCl, and 0.05% each Tween-20 and Nonidet-P40. For each reaction, the buffers, salts and enzyme were combined in a volumc of 5 \u03bcl; the DNAs (target and two oligonucleotides) were combined in 5 \u03bcl of dHO and overlaid with a drop of CHILLOUT evaporation barrier. When multiple reactions were performed with the same reaction constituents, these formulations were expanded proportionally.","Unless otherwise stated, the sample tubes with the DNA mixtures were warmed to 61\u00b0 C., and the reactions were started by the addition of 5 \u03bcl of the enzyme mixture. After 20 minutes at this temperature, the reactions were stopped by the addition of 8 \u03bcl of 95% formamide with 20 mM EDTA and 0.05% marker dyes. Samples were heated to 75\u00b0 C. for 2 minutes immediately before electrophoresis through a 20% acrylamide gel (19:1 cross-linked), with 7 M urea, in a buffer of 45 mM Tri s-Borate, pH 8.3, 1.4 mM EDTA. The products of the reactions were visualized by the use of an Hitachi FMBIO fluorescence imager. In each case, the uncut probe material was visible as an intense black band or blob, usually in the top half of the panel, while the desired products of INVADER specific cleavage were visible as one or two narrower black bands, usually in the bottom half of the panel. Under some reaction conditions, particularly those with elevated salt concentrations, a secondary cleavage product is also visible (thus generating a doublet). Ladders of lighter grey bands generally indicate either exonuclease nibbling of the probe oligonucleotide or heat-induced, non-specific breakage of the probe.",{"@attributes":{"id":"p-0753","num":"0762"},"figref":["FIG. 37","FIG. 37"],"sub":"3"},"b) KCl Titration",{"@attributes":{"id":"p-0755","num":"0764"},"figref":"FIG. 38","sub":"2"},"As shown in , the INVADER-directed cleavage reaction requires the presence of salt (e.g., KCl) for effective cleavage to occur. In other reactions, it has been found that KCl can inhibit the activity of certain CLEAVASE enzymes when present at concentrations above about 25 mM. For example, in cleavage reactions using the S-60 oligonucleotide shown in , in the absence of primer, the CLEAVASE BN enzyme loses approximately 50% of its activity in 50 mM KCl. Therefore, the use of alternative salts in the INVADER-directed cleavage reaction was examined. In these experiments, the potassium ion was replaced with either Na or Li or the chloride ion was replaced with glutamic acid. The replacement of KCl with alternative salts is described below in Sections c-e.","c) NaCl Titration","NaCl was used in place of KCl at 75, 100, 150 or 200 mM, in combination with the use 2 mM MnCl, in an otherwise standard reaction. These results showed that NaCl can be used as a replacement for KCl in the INVADER-directed cleavage reaction, with like concentration giving like results, (i.e., the presence of NaCl, like KCl, enhances product accumulation).","d) LiCl Titration","LiCl was used in place of KCl in otherwise standard reactions. Concentrations tested were 25, 50, 75, 100, 150 and 200mM LiCl. The results demonstrated that LiCl can be used as a suitable replacement for KCl in the INVADER-directed cleavage reaction (i.e., the presence of LiCl, like KCl, enhances product accumulation), in concentrations of about 100 mM or higher.","e) KGlu Titration","The results of using a glutamate salt of potassium (KGlu) in place of the more commonly used chloride salt (KCl) in reactions performed over a range of temperatures were examined. KGlu has been shown to be a highly effective salt source for some enzymatic reactions, showing a broader range of concentrations that permit maximum enzymatic activity (Leirmo et al., Biochem., 26:2095 [1987]). The ability of KGlu to facilitate the annealing of the probe and INVADER oligonucleotides to the target nucleic acid was compared to that of LiCl. In these experiments, the reactions were run for 15 minutes, rather than the standard 20 minutes, in standard reactions that replaced KCl 200 mM, 300 mM or 400 mM KGlu. The reactions were run at 65\u00b0 C., 67\u00b0 C., 69\u00b0 C. or 71\u00b0 C. The results showed demonstrated that KGlu was very effective as a salt in the invasive cleavage reactions, with full activity apparent even at 400 mM KGlu, though at the lowest temperature cleavage was reduced by about 30% at 300 mM KGlu, and by about 90% to 400 mM KGlu.","f) MnClAnd MgClTitration and Ability to Replace MnClwith MgCl","In some instances it may be desirable to perform the invasive cleavage reaction in the presence of Mg, either in addition to, or in place of Mn as the necessary divalent cation required for activity of the enzyme employed. For example, some common methods of preparing DNA from bacterial cultures or tissues use MgClin solutions that are used to facilitate the collection of DNA by precipitation. In addition, elevated concentrations (i.e., greater than 5 mM) of divalent cation can be used to facilitate hybridization of nucleic acids, in the same way that the monovalent salts were used above, thereby enhancing the invasive cleavage reaction. In this experiment, the tolerance of the invasive cleavage reaction was examined for 1) the substitution of MgClfor MnCland for the ability to produce specific product in the presence of increasing concentrations of MgCland MnCl.",{"@attributes":{"id":"p-0765","num":"0774"},"figref":"FIG. 39","sub":["2 ","2 ","2 ","2 ","2","2 ","2","2","2","2 ","2","2 ","2 "]},"In addition to examining the effects of the salt environment on the rate of product accumulation in the invasive cleavage reaction, the use of reaction constituents shown to be effective in enhancing nucleic acid hybridization in either standard hybridization assays (e.g., blot hybridization) or in ligation reactions was examined. These components may act as volume excluders, increasing the effective concentration of the nucleic acids of interest and thereby enhancing hybridization, or they may act as charge-shielding agents to minimize repulsion between the highly charged backbones of the nucleic acids strands. The results of these experiments are described in Sections g and h below.","g) Effect of CTAB Addition","The polycationic detergent cetyltrietheylammonium bromide (CTAB) has been shown to dramatically enhance hybridization of nucleic acids (Pontius and Berg, Proc. Natl. Acad. Sci. USA 88:8237 [1991]). The effect of adding the detergent CTAB in concentrations from 100 mM to 1 mM to invasive cleavage reactions in which 150 mM LiCl was used in place of the KCl in otherwise standard reactions was also investigated. These results showed that 200 mM CTAB may have a very moderate enhancing effect under these reaction conditions, and the presence of CTAB in excess of about 500 \u03bcM was inhibitory to the accumulation of specific cleavage product.","h) Effect of PEG Addition","The effect of adding polyethylene glycol (PEG) at 4.8 or 12% (w\/v) concentrations to otherwise standard reactions was also examined. The effects of increasing the reaction temperature of the PEG-containing reactions was examined by performing duplicate sets of PEG titration reactions at 61\u00b0 C. and 65\u00b0 C. The results showed that at all percentages tested, and at both temperatures tested, the inclusion of PEG substantially eliminated the production of specific cleavage product.","In addition to, the presence of 1\u00d7 Denhardts in the reaction mixture was found to have no adverse effect upon the cleavage reaction (SOX Denhardts contains per 500 ml: 5 g Ficoll, 5 g polyvinylpyrrolidone, 5 g BSA). Further, the presence of each component of Denhardt's was examined individually (i.e., Ficoll alone, polyvinylpyrrolidone alone, BSA alone) for the effect upon the INVADER-directed cleavage reaction; no adverse effect was observed.","i) Effect of the Addition of Stabilizing Agents","Another approach to enhancing the output of the invasive cleavage reaction is to enhance the activity of the enzyme employed, either by increasing its stability in the reaction environment or by increasing its turnover rate. Without regard to the precise mechanism by which various agents operate in the invasive cleavage reaction, a number of agents commonly used to stabilize enzymes during prolonged storage were tested for the ability to enhance the accumulation of specific cleavage product in the invasive cleavage reaction.","The effects of adding glycerol at 15% and of adding the detergents Tween-20 and Nonidet-P40 at 1.5%, alone or in combination, in otherwise standard reactions were also examined. The results demonstrated that under these conditions these adducts had little or no effect on the accumulation of specific cleavage product.","The effects of adding gelatin to reactions in which the salt identity and concentration were varied from the standard reaction were also investigated. The results demonstrated that in the absence of salt the gelatin had a moderately enhancing effect on the accumulation of specific cleavage product, but when either salt (KCl or LiCl) was added to reactions performed under these conditions, increasing amounts of gelatin reduced the product accumulation.","j) Effect of Adding Large Amounts of Non-Target Nucleic Acid","In detecting specific nucleic acid sequences within samples, it is important to determine if the presence of additional genetic material (i.e., non-target nucleic acids) will have a negative effect on the specificity of the assay. In this experiment, the effect of including large amounts of non-target nucleic acid, either DNA or RNA, on the specificity of the invasive cleavage reaction was examined. The data was examined for either an alteration in the expected site of cleavage, or for an increase in the nonspecific degradation of the probe oligonucleotide.",{"@attributes":{"id":"p-0778","num":"0787"},"figref":["FIG. 40","FIG. 40","FIG. 40"]},"In addition to the data presented above, invasive cleavage reactions were run with succinate buffer at pH 5.9 in place of the MOPS buffer used in the \u201cstandard\u201d reaction; no adverse effects were observed.","The data shown in  and described above demonstrate that the invasive cleavage reaction can be performed using a wide variety of reaction conditions and is therefore suitable for practice in clinical laboratories.","In addition to the clinical need to detect specific DNA sequences for infectious and genetic diseases, there is a need for technologies that can quantitatively detect target nucleic acids that are composed of RNA. For example, a number of viral agents, such as hepatitis C virus (HCV) and human immunodeficiency virus (HIV) have RNA genomic material, the quantitative detection of which can be used as a measure of viral load in a patient sample. Such information can be of critical diagnostic or prognostic value.","Hepatitis C virus (HCV) infection is the predominant cause of post-transfusion non-A, non-B (NANB) hepatitis around the world. In addition, HCV is the major etiologic agent of hepatocellular carcinoma (HCC) and chronic liver disease world wide. The genome of HCV is a small (9.4 kb) RNA molecule. In studies of transmission of HCV by blood transfusion it has been found the presence of HCV antibody, as measured in standard immunological tests, does not always correlate with the infectivity of the sample, while the presence of HCV RNA in a blood sample strongly correlates with infectivity. Conversely, serological tests may remain negative in immunosuppressed infected individuals, while HCV RNA may be easily detected (Cuthbert, Clin. Microbiol. Rev., 7:505 [1994]).","The need for and the value of developing a probe-based assay for the detection the HCV RNA is clear. The polymerase chain reaction has been used to detect HCV in clinical samples, but the problems associated with carry-over contamination of samples has been a concern. Direct detection of the viral RNA without the need to perform either reverse transcription or amplification would allow the elimination of several of the points at which existing assays may fail.","The genome of the positive-stranded RNA hepatitis C virus comprises several regions including 5\u2032 and 3\u2032 noncoding regions (i.e., 5\u2032 and 3\u2032 untranslated regions) and a polyprotein coding region that encodes the core protein (C), two envelope glycoproteins (E1 and E2\/NS1) and six nonstructural glycoproteins (NS2-NS5b). Molecular biological analysis of the HCV genome has showed that some regions of the genome are very highly conserved between isolates, while other regions are fairly rapidly changeable. The 5\u2032 noncoding region (NCR) is the most highly conserved region in the HCV. These analyses have allowed these viruses to be divided into six basic genotype groups, and then further classified into over a dozen sub-types (the nomenclature and division of HCV genotypes is evolving; see Altamirano et al., J. Infect. Dis., 171:1034 (1995) for a recent classification scheme).","In order to develop a rapid and accurate method of detecting HCV present in infected individuals, the ability of the INVADER-directed cleavage reaction to detect HCV RNA was examined. Plasmids containing DNA derived from the conserved 5\u2032-untranslated region of six different HCV RNA isolates were used to generate templates for in vitro transcription. The HCV sequences contained within these six plasmids represent genotypes 1 (four sub-types represented; 1a, 1b, 1c, and \u03941c), 2, and 3. The nomenclature of the HCV genotypes used herein is that of Simmonds et al. (as described in Altamirano et al., supra). The \u03941c subtype was used in the model detection reaction described below.","a) Generation of Plasmids Containing HCV Sequences","Six DNA fragments derived from HCV were generated by RT-PCR using RNA extracted from serum samples of blood donors; these PCR fragments were a gift of Dr. M. Altamirano (University of British Columbia, Vancouver). These PCR fragments represent HCV sequences derived from HCV genotypes 1a, 1b, 1c, \u03941c, 2c and 3a.","The RNA extraction, reverse transcription and PCR were performed using standard techniques (Altamirano et al., supra). Briefly, RNA was extracted from 100 \u03bcl of serum using guanidine isothiocyanate, sodium lauryl sarkosate and phenol-chloroform (Inchauspe et al., Hepatol., 14:595 [1991]). Reverse transcription was performed according to the manufacturer's instructions using a GeneAmp rTh reverse transcriptase RNA PCR kit (Perkin-Elmer) in the presence of an external antisense primer, HCV342. The sequence of the HCV342 primer is 5\u2032-GGTTTTTCTTTGAGGTTTAG-3\u2032 (SEQ ID NO:40). Following termination of the RT reaction, the sense primer HCV7 (5\u2032-GCGACACTCCACCATAGAT-3\u2032 [SEQ ID NO:41]) and magnesium were added and a first PCR was performed. Aliquots of the first PCR products were used in a second (nested) PCR in the presence of primers HCV46 (5\u2032-CTGTCTTCACGCAGAAAGC-3\u2032 [SEQ ID NO:42]) and HCV308 [5\u2032-GCACGGT CTACGAGACCTC-3\u2032 [SEQ ID NO:43]). The PCRs produced a 281 bp product that corresponds to a conserved 5\u2032 noncoding region (NCR) region of HCV between positions \u2212284 and \u22124 of the HCV genome (Altramirano et al., supra).","The six 281 bp PCR fragments were used directly for cloning or they were subjected to an additional amplification step using a 50 \u03bcl PCR comprising approximately 100 fmoles of DNA, the HCV46 and HCV308 primers at 0.1 \u03bcM, 100 \u03bcM of all four dNTPs and 2.5 units of Taq DNA polymerase in a buffer containing 10 mM Tris-HCl, pH 8.3, 50 mM KCl, 1.5 mM MgCland 0.1% Tween 20. The PCRs were cycled 25 times at 96\u00b0 C. for 45 sec., 55\u00b0 C. for 45 sec. and 72\u00b0 C. for 1 min. Two microliters of either the original DNA samples or the reamplificd PCR products were used for cloning in the linear pT7Blue T-vector (Novagen) according to manufacturer's protocol. After the PCR products were ligated to the pT7Blue T-vector, the ligation reaction mixture was used to transform competent JM109 cells (Promega). Clones containing the pT7Blue T-vector with an insert were selected by the presence of colonies having a white color on LB plates containing 40 \u03bcg\/ml X-Gal, 40 \u03bcg\/ml IPTG and 50 \u03bcg\/ml ampicillin. Four colonies for each PCR sample were picked and grown overnight in 2 ml LB media containing 50 \u03bcg\/ml carbenicillin. Plasmid DNA was isolated using the following alkaline miniprep protocol. Cells from 1.5 ml of the overnight culture were collected by centrifugation for 2 min. in a microcentrifuge (14K rpm), the supernatant was discarded and the cell pellet was resuspended in 50 \u03bcl TE buffer with 10 \u03bcg\/ml RNAse A (Pharmacia). One hundred microliters of a solution containing 0.2 N NaOH, 1% SDS was added and the cells were lysed for 2 min. The lysate was gently mixed with 100 \u03bcl of 1.32 M potassium acetate, pH 4.8, and the mixture was centrifuged for 4 min. in a microcentrifuge (14K rpm); the pellet comprising cell debris was discarded. Plasmid DNA was precipitated from the supernatant with 200 \u03bcl ethanol and pelleted by centrifugation a microcentrifuge (14K rpm). The DNA pellet was air dried for 15 min. and was then redissolved in 50 \u03bcl TE buffer (10 mM Tris-HCl, pH 7.8, 1 mM EDTA).","b) Reamplification of HCV Clones to Add the Phage T7 Promoter for Subsequent In Vitro Transcription","To ensure that the RNA product of transcription had a discrete 3\u2032 end it was necessary to create linear transcription templates that stopped at the end of the HCV sequence. These fragments were conveniently produced using the PCR to reamplify the segment of the plasmid containing the phage promoter sequence and the HCV insert. For these studies, the clone of HCV type \u03941c was reamplified using a primer that hybridizes to the T7 promoter sequence: 5\u2032-TAATACGACTCACTATAGGG-3\u2032 (SEQ ID NO:44; \u201cthe T7 promoter primer\u201d) (Novagen) in combination with the 3\u2032 terminal HCV-specific primer HCV308 (SEQ ID NO:43). For these reactions, 1 \u03bcl of plasmid DNA (approximately 10 to 100 ng) was reamplified in a 200 \u03bcl PCR using the T7 and HCV308 primers as described above with the exception that 30 cycles of amplification were employed. The resulting amplicon was 354 bp in length. After amplification the PCR mixture was transferred to a fresh 1.5 ml microcentrifuge tube, the mixture was brought to a final concentration of 2 M NHOAc, and the products were precipitated by the addition of one volume of 100% isopropanol. Following a 10 min. incubation at room temperature, the precipitates were collected by centrifugation, washed once with 80% ethanol and dried under vacuum. The collected material was dissolved in 100 \u03bcl nuclease-free distilled water (Promega).","Segments of RNA were produced from this amplicon by in vitro transcription using the RiboMAX\u2122 Large Scale RNA Production System (Promega) in accordance with the manufacturer's instructions, using 5.3 \u03bcg of the amplicon described above in a 100 \u03bcl reaction. The transcription reaction was incubated for 3.75 hours, after which the DNA template was destroyed by the addition of 5-6 \u03bcl of RQ1 RNAse-free DNAse (1 unit\/\u03bcl) according to the RiboMAX\u2122 kit instructions. The reaction was extracted twice with phenol\/chloroform\/isoamyl alcohol (50:48:2) and the aqueous phase was transferred to a fresh microcentrifuge tube. The RNA was then collected by the addition of 10 \u03bcl of 3M NHOAc, pH 5.2 and 110 \u03bcl of 100% isopropanol. Following a 5 min. incubation at 4\u00b0 C., the precipitate was collected by centrifugation, washed once with 80% ethanol and dried under vacuum. The sequence of the resulting RNA transcript (HCV 1.1 transcript) is listed in SEQ ID NO:45.","c) Detection of the HCV1.1 Transcript in the INVADER-Directed Cleavage Assay","Detection of the HCV1.1 transcript was tested in the INVADER-directed cleavage assay using an HCV-specific probe oligonucleotide (5\u2032-CCGGTCGTCCTGGCAAT XCC-3\u2032 [SEQ ID NO:46]); X indicates the presence of a fluorescein dye on an abasic linker) and an HCV-specific INVADER oligonucleotide (5\u2032-GTTTATCCAAGAAAGGAC CCGGTC-3\u2032 [SEQ ID NO:47]) that causes a 6-nucleotide invasive cleavage of the probe.","Each 10 \u03bcl of reaction mixture comprised 5 pmole of the probe oligonucleotide (SEQ ID NO:46) and 10 pmole of the INVADER oligonucleotide (SEQ ID NO:47) in a buffer of 10 mM MOPS, pH 7.5 with 50 mM KCl, 4 mM MnCl, 0.05% each Tween-20 and Nonidet-P40 and 7.8 units RNasin\u00ae ribonuclease inhibitor (Promega). The cleavage agents employed were CLEAVASE A\/G (used at 5.3 ng\/10 \u03bcl reaction) or DNAPTth (used at 5 polymerase units\/10 \u03bcl reaction). The amount of RNA target was varied as indicated below. When RNAse treatment is indicated, the target RNAs were pre-treated with 10 \u03bcg of RNasc A (Sigma) at 37\u00b0 C. for 30 min. to demonstrate that the detection was specific for the RNA in the reaction and not due to the presence of any residual DNA template from the transcription reaction. RNase-treated aliquots of the HCV RNA were used directly without intervening purification.","For each reaction, the target RNAs were suspended in the reaction solutions as described above, but lacking the cleavage agent and the MnClfor a final volume of 10 \u03bcl, with the INVADER and probe at the concentrations listed above. The reactions were warmed to 46\u00b0 C. and the reactions were started by the addition of a mixture of the appropriate enzyme with MnCl. Atler incubation for 30 min. at 46\u00b0 C., the reactions were stopped by the addition of 8 \u03bcl of 95% formamide, 10 mM EDTA and 0.02% methyl violet (methyl violet loading buffer). Samples were then resolved by electrophoresis through a 15% denaturing polyacrylamide gel (19:1 cross-linked), containing 7 M urea, in a buffer of 45 mM Tris-Borate, pH 8.3, 1.4 mM EDTA. Following electrophoresis, the labeled reaction products were visualized using the FMBIO-100 Image Analyzer (Hitachi), with the resulting imager scan shown in FIG. .","In , the samples analyzed in lanes 1-4 contained 1 pmole of the RNA target, the reactions shown in lanes 5-8 contained 100 fmoles of the RNA target and the reactions shown in lanes 9-12 contained 10 fmoles of the RNA target. All odd-numbered lanes depict reactions performed using CLEAVASE A\/G enzyme and all even-numbered lanes depict reactions performed using DNAPTth. The reactions analyzed in lanes 1, 2, 5, 6, 9 and 10 contained RNA that had been pre-digested with RNase A. These data demonstrate that the invasive cleavage reaction efficiently detects RNA targets and further, the absence of any specific cleavage signal in the RNase-treated samples confirms that the specific cleavage product seen in the other lanes is dependent upon the presence of input RNA.","In this Example, the fate of the RNA target in the INVADER-directed cleavage reaction was examined. As shown above in Example 1D, when RNAs are hybridized to DNA oligonucleotides, the 5\u2032 nucleases associated with DNA polymerases can be used to cleave the RNAs; such cleavage can be suppressed when the 5\u2032 arm is long or when it is highly structured (Lyamichev et al., Science 260:778 [1993], and U.S. Pat. No. 5,422,253, the disclosure of which is herein incorporated by reference). In this experiment, the extent to which the RNA target would be cleaved by the cleavage agents when hybridized to the detection oligonucleotides (i.e., the probe and INVADER oligonucleotides) was examined using reactions similar to those described in Example 20, performed using fluorescein-labeled RNA as a target.","Transcription reactions were performed as described in Example 19 with the exception that 2% of the UTP in the reaction was replaced with fluorescein-12-UTP (Boehringer Mannheim) and 5.3 \u03bcg of the amplicon was used in a 100 \u03bcl reaction. The transcription reaction was incubated for 2.5 hours, after which the DNA template was destroyed by the addition of 5-6 \u03bcl of RQ1 RNAse-free DNAse (1 unit\/\u03bcl) according to the RiboMAX\u2122 kit instructions. The organic extraction was omitted and the RNA was collected by the addition of 10 \u03bcl of 3M NaOAc, pH 5.2 and 110 \u03bcl of 100% isopropanol. Following a 5 min. incubation at 4\u00b0 C., the precipitate was collected by centrifugation, washed once with 80% ethanol and dried under vacuum. The resulting RNA was dissolved in 100 \u03bcl of nuclcase-free water. Half (i.e., 50%) of the sample was purified by electrophoresis through a 8% denaturing polyacrylamide gel (19:1 cross-linked), containing 7 M urea, in a buffer of 45 mM Tris-Borate, pH 8.3, 1.4 mM EDTA. The gel slice containing the full-length material was excised and the RNA was eluted by soaking the slice overnight at 4\u00b0 C. in 200 \u03bcl of 10 mM Tris-Cl, pH 8.0, 0.1 mM EDTA and 0.3 M NaOAc. The RNA was then precipitated by the addition of 2.5 volumes of 100% ethanol. After incubation at \u221220\u00b0 C. for 30 min., the precipitates were recovered by centrifugation, washed once with 80% ethanol and dried under vacuum. The RNA was dissolved in 25 \u03bcl of nuclease-free water and then quantitated by UV absorbance at 260 nm.","Samples of the purified RNA target were incubated for 5 or 30 min. in reactions that duplicated the CLEAVASE A\/G and DNAPTth INVADER reactions described in Example 20 with the exception that the reactions lacked probe and INVADER oligonucleotides. Subsequent analysis of the products showed that the RNA was very stable, with a very slight background of non-specific degradation, appearing as a gray background in the gel lane. The background was not dependent on the presence of enzyme in the reaction.","INVADER detection reactions using the purified RNA target were performed using the probe\/INVADER pair described in Example 19 (SEQ ID NOS:46 and 47). Each reaction included 500 fmole of the target RNA, 5 pmoles of the fluorescein-labeled probe and 10 pmoles of the INVADER oligonucleotide in a buffer of 10 mM MOPS, pH 7.5 with 150 mM LiCl, 4 mM MnCl, 0.05% each Tween-20 and Nonidet-P40 and 39 units RNAsin\u00ae (Promega). These components were combined and warmed to 50\u00b0 C. and the reactions were started by the addition of either 53 ng of CLEAVASE A\/G or 5 polymerase units of DNAPTth. The final reaction volume was 10 \u03bcl. After 5 min at 50\u00b0 C., 5 \u03bcl aliquots of each reaction were removed to tubes containing 4 \u03bcl of 95% formamide, 10 mM EDTA and 0.02% methyl violet. The remaining aliquot received a drop of CHILLOUT evaporation barrier and was incubated for an additional 25 min. These reactions were then stopped by the addition of 4 \u03bcl of the above formamide solution. The products of these reactions were resolved by electrophoresis through separate 20% denaturing polyacrylamide gels (19:1 cross-linked), containing 7 M urea, in a buffer of 45 mM Tris-Borate, pH 8.3, 1.4 mM EDTA. Following electrophoresis, the labeled reaction products were visualized using the FMBIO-100 Image Analyzer (Hitachi), with the resulting imager scans shown in  (5 min reactions) and B (30 min. reactions).","In  the target RNA is seen very near the top of each lane, while the labeled probe and its cleavage products are seen just below the middle of each panel. The FMBIO-100 Image Analyzer was used to quantitate the fluorescence signal in the probe bands. In each panel, lane 1 contains products from reactions performed in the absence of a cleavage agent, lane 2 contains products from reactions performed using CLEAVASE A\/G and lane 3 contains products from reactions performed using DNAPTth.","Quantitation of the fluorescence signal in the probe bands revealed that after a 5 min. incubation, 12% or 300 fmole of the probe was cleaved by the CLEAVASE A\/G and 29% or 700 fmole was cleaved by the DNAPTth. After a 30 min. incubation, 20 CLEAVASE A\/G had cleaved 32% of the probe molecules and DNAPTth had cleaved 70% of the probe molecules. (The images shown in  were printed with the intensity adjusted to show the small amount of background from the RNA degradation, so the bands containing strong signals are saturated and therefore these images do not accurately reflect the differences in measured fluorescence)","The data shown in  clearly shows that, under invasive cleavage conditions, RENA molecules are sufficiently stable to be detected as a target and that each RNA molecule can support many rounds of probe cleavage.","One of the primary benefits of the INVADER-directed cleavage assay as a means for detection of the presence of specific target nucleic acids is the correlation between the amount of cleavage product generated in a set amount of time and the quantity of the nucleic acid of interest present in the reaction. The benefits of quantitative detection of RNA sequences was discussed in Example 19. In this Example, the quantitative nature of the detection assay was demonstrated through the use of various amounts of target starting material. In addition to demonstrating the correlation between the amounts of input target and output cleavage product, these data graphically show the degree to which the RNA target can be recycled in this assay","The RNA target used in these reactions was the fluorescein-labeled material described in Example 20 (i.e., SEQ ID NO:45). Because the efficiency of incorporation of the fluorescein-12-UTP by the T7 RNA polymerase was not known, the concentration of the RNA was determined by measurement of absorbance at 260 nm, not by fluorescence intensity. Each reaction comprised 5 pmoles of the fluorescein-labeled probe (SEQ ID NO:46) and 10 pmoles of the INVADER oligonucleotide (SEQ ID NO:47) in a buffer of 10 mM MOPS, pH 7.5 with 150 mM LiCl, 4 mM MnCl, 0.05% each Tween-20 and Nonidet-P40 and 39 units of RNAsin\u00ae (Promega). The amount of target RNA was varied from 1 to 100 fmoles, as indicated below. These components were combined, overlaid with CHILLOUT evaporation barrier and warmed to 50\u00b0 C., the reactions were started by the addition of either 53 ng of CLEAVASE A\/G or 5 polymerase units of DNAPTth, to a final reaction volume of 10 \u03bcl. After 30 minutes at 50\u00b0 C., reactions were stopped by the addition of 8 \u03bcl of 95% formamide, 10 mM EDTA and 0.02% methyl violet. The unreacted markers in lanes 1 and 2 were diluted in the same total volume (18 \u03bcl). The samples were heated to 90\u00b0 C. for 1 minute and 2.5 \u03bcl of each of these reactions were resolved by electrophoresis through a 20% denaturing polyacrylamide gel (19:1 cross link) with 7M urea in a buffer of 45 mM Tris-Borate, pH 8.3, 1.4 mM EDTA, and the labeled reaction products were visualized using the FMBIO-100 Image Analyzer (Hitachi), with the resulting imager scans shown in FIG. .","In , lanes 1 and 2 show 5 pmoles of uncut probe and 500 fmoles of untreated RNA, respectively. The probe is the very dark signal near the middle of the panel, while the RNA is the thin line near the top of the panel. These RNAs were transcribed with a 2% substitution of fluorescein-12-UTP for natural UTP in the transcription reaction. The resulting transcript contains 74 U residues, which would give an average of 1.5 fluorescein labels per molecule. With one tenth the molar amount of RNA loaded in lane 2, the signal in lane 2 should be approximately one seventh (0.15\u00d7) the fluorescence intensity of the probe in lane 1. Measurements indicated that the intensity was closer to one fortieth, indicating an efficiency of label incorporation of approximately 17%. Because the RNA concentration was verified by A260 measurement this does not alter the experimental observations below, but it should be noted that the signal from the RNA and the probes does not accurately reflect the relative amounts in the reactions.","The reactions analyzed in lanes 3 through 7 contained 1, 5, 10, 50 and 100 fmoles of target, respectively, with cleavage of the probe accomplished by CLEAVASE A\/G. The reactions analyzed in lanes 8 through 12 repeated the same array of target amounts, with cleavage of the probe accomplished by DNAPTth. The boxes seen surrounding the product bands show the area of the scan in which the fluorescence was measured for each reaction. The number of fluorescence units detected within each box is indicated below each box; background florescence was also measured.","It can be seen by comparing the detected fluorescence in each lane that the amount of product formed in these 30 minute reactions can be correlated to the amount of target material. The accumulation of product under these conditions is slightly enhanced when DNAPTth is used as the cleavage agent, but the correlation with the amount of target present remains. This demonstrates that the INVADER assay can be used as a means of measuring the amount of target RNA within a sample.","Comparison of the fluorescence intensity of the input RNA with that of the cleaved product shows that the INVADER-directed cleavage assay creates signal in excess of the amount of target, so that the signal visible as cleaved probe is far more intense than that representing the target RNA. This further confirms the results described in Example 20, in which it was demonstrated that each RNA molecule could be used many times.","The detection of specific targets is achieved in the INVADER-directed cleavage assay by the cleavage of the probe oligonucleotide. In addition to the methods described in the preceding Examples, the cleaved probe may be separated from the uncleaved probe using the charge reversal technique described below. This novel separation technique is related to the observation that positively charged adducts can affect the electrophoretic behavior of small oligonucleotides because the charge of the adduct is significant relative to charge of the whole complex. Observations of aberrant mobility due to charged adducts have been reported in the literature, but in all cases found, the applications pursued by other scientists have involved making oligonucleotides larger by enzymatic extension. As the negatively charged nucleotides are added on, the positive influence of the adduct is reduced to insignificance. As a result, the effects of positively charged adducts have been dismissed and have received infinitesimal notice in the existing literature.","This observed effect is of particular utility in assays based on the cleavage of DNA molecules. When an oligonucleotide is shortened through the action of a CLEAVASE enzyme or other cleavage agent, the positive charge can be made to not only significantly reduce the net negative charge, but to actually override it, effectively \u201cflipping\u201d the net charge of the labeled entity. This reversal of charge allows the products of target-specific cleavage to be partitioned from uncleaved probe by extremely simple means. For example, the products of cleavage can be made to migrate towards a negative electrode placed at any point in a reaction vessel, for focused detection without gel-based electrophoresis. When a slab gel is used, sample wells can be positioned in the center of the gel, so that the cleaved and uncleaved probes can be observed to migrate in opposite directions. Alternatively, a traditional vertical gel can be used, but with the electrodes reversed relative to usual DNA gels (i.e., the positive electrode at the top and the negative electrode at the bottom) so that the cleaved molecules enter the gel, while the uncleaved disperse into the upper reservoir of electrophoresis buffer.","An additional benefit of this type of readout is that the absolute nature of the partition of products from substrates means that an abundance of uncleaved probe can be supplied to drive the hybridization step of the probe-based assay, yet the unconsumed probe can be subtracted from the result to reduce background.","Through the use of multiple positively charged adducts, synthetic molecules can be constructed with sufficient modification that the normally negatively charged strand is made nearly neutral. When so constructed, the presence or absence of a single phosphate group can mean the difference between a net negative or a net positive charge. This observation has particular utility when one objective is to discriminate between enzymatically generated fragments of DNA, which lack a 3\u2032 phosphate, and the products of thermal degradation, which retain a 3\u2032 phosphate (and thus two additional negative charges).","a) Characterization of the Products of Thermal Breakage of DNA Oligonucleotides","Thermal degradation of DNA probes results in high background that can obscure signals generated by specific enzymatic cleavage, decreasing the signal-to-noise ratio. To better understand the nature of DNA thermal degradation products, the 5\u2032 tetrachloro-fluorescein (TET)-labeled oligonucleotides 78 (SEQ ID NO:48) and 79 (SEQ ID NO:49) (100 pmole each) were incubated in 50 \u03bcl 10 mM NaCO(pH 10.6), 50 mM NaCl at 90\u00b0 C. for 4 hours. To prevent evaporation of the samples, the reaction mixture was overlaid with 50 \u03bcl of CHILLOUT liquid wax. The reactions were then divided in two equal aliquots (A and B). Aliquot A was mixed with 25 \u03bcl of methyl violet loading buffer and Aliquot B was dephosphorylated by addition of 2.5 \u03bcl of 100 mM MgCland 1 \u03bcl of 1 unit\/\u03bcl Calf Intestinal Alkaline Phosphatase (CIAP) (Promega), with incubation at 37\u00b0 C. for 30 min. after which 25 \u03bcl of methyl violet loading buffer was added. One microliter of each sample was resolved by electrophoresis through a 12% polyacrylamide denaturing gel and imaged as described in Example 21; a 585 nm filter was used with the FMBIO Image Analyzer. The resulting imager scan is shown in FIG. .","In , lanes 1-3 contain the TET-labeled oligonucleotide 78 and lanes 4-6 contain the TET-labeled oligonucleotides 79. Lanes 1 and 4 contain products of reactions that were not heat treated. Lanes 2 and 5 contain products from reactions that were heat treated and lanes 3 and 6 contain products from reactions that were heat treated and subjected to phosphatase treatment.","As shown in , heat treatment causes significant breakdown of the 5\u2032-TET-labeled DNA, generating a ladder of degradation products (, lanes 2, 3, 5 and 6). Band intensities correlate with purine and pyrimidine base positioning in the oligonuclcotide sequences, indicating that backbone hydrolysis may occur through formation of abasic intermediate products that have faster rates for purines than for pyrimidines (Lindahl and Karlstrom, Biochem., 12:5151 [1973]).","Dephosphorylation decreases the mobility of all products generated by the thermal degradation process, with the most pronounced effect observed for the shorter products (, lanes 3 and 6). This demonstrates that thermally degraded products possess a 3\u2032 end terminal phosphoryl group that can be removed by dephosphorylation with CIAP. Removal of the phosphoryl group decreases the overall negative charge by 2. Therefore, shorter products that have a small number of negative charges are influenced to a greater degree upon the removal of two charges. This leads to a larger mobility shift in the shorter products than that observed for the larger species.","The fact that the majority of thermally degraded DNA products contain 3\u2032 end phosphate groups and CLEAVASE enzyme-generated products do not allowed the development of simple isolation methods for products generated in the INVADER-directed cleavage assay. The extra two charges found in thermal breakdown products do not exist in the specific cleavage products. Therefore, if one designs assays that produce specific products that contain a net positive charge of one or two, then similar thermal breakdown products will either be negative or neutral. The difference can be used to isolate specific products by reverse charge methods as shown below.","b) Dephosphorylation of Short Amino-Modified Oligonucleotides can Reverse the Net Charge of the Labeled Product","To demonstrate how oligonucleotides can be transformed from net negative to net positively charged compounds, the four short amino-modified oligonucleotides labeled 70, 74, 75 and 76 and shown in  were synthesized ( shows both oligonucleotides 70 and 74). All four modified oligonucleotides possess Cy-3 dyes positioned at the 5\u2032-end, which individually are positively charged under reaction and isolation conditions described in this Example. Compounds 70 and 74 contain two amino modified thymidines that, under reaction conditions, display positively charged R-NH groups attached at the C5 position through a Cor Clinker, respectively. Because compounds 70 and 74 are 3\u2032-end phosphorylated, they consist of four negative charges and three positive charges. Compound 75 differs from 74 in that the internal Camino modified thymidine phosphate in 74 is replaced by a thymidine methyl phosphonate. The phosphonate backbone is uncharged and so there are a total of three negative charges on compound 75. This gives compound 75 a net negative one charge. Compound 76 differs from 70 in that the internal amino modified thymidine is replaced by an internal cytosine phosphonate. The pKof the N3 nitrogen of cytosine can be from 4 to 7. Thus, the net charges of this compound, can be from \u22121 to 0 depending on the pH of the solution. For the simplicity of analysis, each group is assigned a whole number of charges, although it is realized that, depending on the pKof each chemical group and ambient pH, a real charge may differ from the whole number assigned. It is assumed that this difference is not significant over the range of pHs used in the enzymatic reactions studied here.","Dephosphorylation of these compounds, or the removal of the 3\u2032 end terminal phosphoryl group, results in elimination of two negative charges and generates products that have a net positive charge of one. In this experiment, the method of isoelectric focusing (IEF) was used to demonstrate a change from one negative to one positive net charge for the described substrates during dephosphorylation.","Substrates 70, 74, 75 and 76 were synthesized by standard phosphoramidite chemistries and deprotected for 24 hours at 22\u00b0 C. in 14 M aqueous ammonium hydroxide solution, after which the solvent was removed in vacuo. The dried powders were resuspended in 200 \u03bcl of HO and filtered through 0.2 \u03bcm filters. The concentration of the stock solutions was estimated by UV-absorbance at 261 nm of samples diluted 200-fold in HO using a spectrophotometer (Spectronic Genesys 2, Milton Roy, Rochester, N.Y.).","Dephosphorylation of compounds 70 and 74, 75 and 76 was accomplished by treating 10 \u03bcl of the crude stock solutions (ranging in concentration from approximately 0.5 to 2 mM) with 2 units of CIAP in 100 \u03bcl of ClAP buffer (Promega) at 37\u00b0 C. for 1 hour. The reactions were then heated to 75\u00b0 C. for 15 min. in order to inactivate the CIAP. For clarity, dephosphorylated compounds are designated \u2018dp\u2019. For example, after dephosphorylation, substrate 70 becomes 70 dp.","To prepare samples for IEF experiments, the concentration of the stock solutions of substrate and dephosphorylated product were adjusted to a uniform absorbance of 8.5\u00d710at 532 nm by dilution with water. Two microliters of each sample were analyzed by IEF using a PhastSystem electrophoresis unit (Pharmacia) and PhastGel IEF 3-9 media (Pharmacia) according to the manufacturer's protocol. Separation was performed at 15\u00b0 C. with the following program: pre-run; 2,000 V, 2.5 mA, 3.5 W, 75 Vh; load; 200 V, 2.5 mA, 3.5 W, 15 Vh; run; 2,000 V; 2.5 mA; 3.5 W, 130 Vh. After separation, samples were visualized by using the FMBIO Image Analyzer (Hitachi) fitted with a 585 nm filter. The resulting imager scan is shown in FIG. .",{"@attributes":{"id":"p-0827","num":"0836"},"figref":"FIG. 48"},"The results shown in  demonstrate that substrates 70, 74, 75 and 76 migrated toward the positive electrode, while the dephosphorylated products 70 dp, 74 dp, 75 dp and 76 dp migrated toward negative electrode. The observed differences in mobility direction was in accord with predicted net charge of the substrates (minus one) and the products (plus one). Small perturbations in the mobilities of the phosphorylated compounds indicate that the overall pIs vary. This was also true for the dephosphorylated compounds. The presence of the cytosine in 76dp, for instance, moved this compound further toward the negative electrode, which was indicative of a higher overall pI relative to the other dephosphorylated compounds. It is important to note that additional positive charges can be obtained by using a combination of natural amino modified bases (70 dp and 74 dp) along with uncharged methylphosphonate bridges (products 75 dp and 76 dp).","The results shown above demonstrate that the removal of a single phosphate group can flip the net charge of an oligonucleotide to cause reversal in an electric field, allowing easy separation of products, and that the precise base composition of the oligonucleotides affect absolute mobility but not the charge-flipping effect.","In this Example the ability to isolate products generated in the INVADER-directed cleavage assay from all other nucleic acids present in the reaction cocktail was demonstrated using charge reversal. This experiment utilized the following Cy3-labeled oligonucleotide: 5\u2032-Cy3-AminoT-AminoT-CTTTTCACCAGCGAGACGGG-3\u2032 (SEQ ID NO:50; termed \u201coligo 61\u201d). Oligo 61 was designed to release upon cleavage a net positively charged labeled product. To test whether or not a net positively charged 5\u2032-end labeled product would be recognized by the CLEAVASE enzymes in the INVADER-directed cleavage assay format, probe oligo 61 (SEQ ID NO:50) and invading oligonucleotide 67 (SEQ ID NO:5 1) were chemically synthesized on a DNA synthesizer (ABI 391) using standard phosphoramidite chemistries and reagents obtained from Glen Research (Sterling, Va.).","Each assay reaction comprised 100 fmoles of M13mp18 single stranded DNA, 10 pmoles each of the probe (SEQ ID NO:50) and INVADER (SEQ ID NO:51) oligonucleotides, and 20 units of CLEAVASE A\/G in a 10 \u03bcl solution of 10 mM MOPS, pH 7.4 with 100 mM KCl. Samples were overlaid with mineral oil to prevent evaporation. The samples were brought to either 50\u00b0 C., 55\u00b0 C., 60\u00b0 C., or 65\u00b0 C. and cleavage was initiated by the addition of 1 \u03bcl of 40 mM MnCl. Reactions were allowed to proceed for 25 minutes and then were terminated by the addition of 10 \u03bcl of 95% formamide containing 20 mM EDTA and 0.02% methyl violet. The negative control experiment lacked the target M13mp18 and was run at 60\u00b0 C. Five microliters of each reaction were loaded into separate wells of a 20% denaturing polyacrylamide gel (cross-linked 29:1) with 8 M urea in a buffer containing 45 mM Tris-Borate (pH 8.3) and 1.4 mM EDTA. An electric field of 20 watts was applied for 30 minutes, with the electrodes oriented as indicated in  (i.e., in reverse orientation). The products of these reactions were visualized using the FMBIO fluorescence imager and the resulting imager scan is shown in FIG. B.",{"@attributes":{"id":"p-0832","num":"0841"},"figref":["FIG. 49A","FIG. 49A"]},"In , lanes 1-6 contain reactions performed at 50\u00b0 C., 55\u00b0 C., 60\u00b0 C., and 65\u00b0 C., respectively; lane 5 contained the control reaction (lacking target). In , the products of cleavage are seen as dark bands in the upper half of the panel; the faint lower band seen appears in proportion to the amount of primary product produced and, while not limiting the invention to a particular mechanism, may represent cleavage one nucleotide into the duplex. The uncleaved probe does not enter the gel and is thus not visible. The control lane showed no detectable signal over background (lane 5). As expected in an invasive cleavage reaction, the rate of accumulation of specific cleavage product was temperature-dependent. Using these particular oligonucleotides and target, the fastest rate of accumulation of product was observed at 55\u00b0 C. (lane 2) and very little product observed at 65\u00b0 C. (lane 4).","When incubated for extended periods at high temperature, DNA probes can break non-specifically (i.e., suffer thermal degradation) and the resulting fragments contribute an interfering background to the analysis. The products of such thermal breakdown are distributed from single-nucleotides up to the full length probe. In this experiment, the ability of charge based separation of cleavage products (i.e., charge reversal) would allow the sensitive separation of the specific products of target-dependent cleavage from probe fragments generated by thermal degradation was examined.","To test the sensitivity limit of this detection method, the target M13mp18 DNA as serially diluted ten fold over than range of 1 fmole to 1 amole. The INVADER and robe oligonucleotides were those described above (i.e., SEQ ID NOS:50 and 51). The invasive cleavage reactions were run as described above with the following modifications: the reactions were performed at 55\u00b0 C., 250 mM or 100 mM KGlu was used in place of the 100 mM KCl and only 1 pmole of the INVADER oligonucleotide was added. The reactions were initiated as described above and allowed to progress for 12.5 hours. A negative control reaction that lacked added M13m18 target DNA was also run. The reactions were terminated by the addition of 10 \u03bcl of 95% formamide containing 20 mM EDTA and 0.02% methyl violet, and 5 \u03bcl of these mixtures were electrophoresed and visualized as described above. The resulting imager scan is shown in FIG. .","In , lane 1 contains the negative control; lanes 2-5 contain reactions performed using 100 mM KGlu; lanes 6-9 contain reactions performed using 250 mM KGlu. The reactions resolved in lanes 2 and 6 contained 1 fmole of target DNA; those in lanes 3 and 7 contained 100 amole of target; those in lanes 4 and 8 contained 10 amole of target and those in lanes 5 and 9 contained 1 amole of target. The results shown in  demonstrate that the detection limit using charge reversal to detect the production of specific cleavage products in an invasive cleavage reaction is at or below 1 attomole or approximately 6.02\u00d710target molecules. No detectable signal was observed in the control lane, which indicates that non-specific hydrolysis or other breakdown products do not migrate in the same direction as enzyme-specific cleavage products. The excitation and emission maxima for Cy3 are 554 and 568, respectively, while the FMBIO Imager Analyzer excites at 532 and detects at 585. Therefore, the limit of detection of specific cleavage products can be improved by the use of more closely matched excitation source and detection filters.","This Example is directed at methods and devices for isolating and concentrating specific reaction products produced by enzymatic reactions conducted in solution whereby the reactions generate charged products from either a charge neutral substrate or a substrate bearing the opposite charge borne by the specific reaction product. The methods and devices of this Example allow isolation of, for example, the products generated by the INVADER-directed cleavage assay of the present invention.","The methods and devices of this Example are based on the principle that when an electric field is applied to a solution of charged molecules, the migration of the molecules toward the electrode of the opposite charge occurs very rapidly. If a matrix or other inhibitory material is introduced between the charged molecules and the electrode of opposite charge such that this rapid migration is dramatically slowed, the first molecules to reach the matrix will be nearly stopped, thus allowing the lagging molecules to catch up. In this way a dispersed population of charged molecules in solution can be effectively concentrated into a smaller volume. By tagging the molecules with a detectable moiety (e.g., a fluorescent dye), detection is facilitated by both the concentration and the localization of the analytes. This Example illustrates two embodiments of devices contemplated by the present invention; of course, variations of these devices will be apparent to those skilled in the art and are within the spirit and scope of the present invention.",{"@attributes":{"id":"p-0839","num":"0848"},"figref":["FIG. 51","FIG. 51"],"b":["10","11","13","13","11","13","13","11","11"]},"As illustrated in , capillary A is connected to the positive electrode of a power supply () (e.g., a controllable power supply available through the laboratory suppliers listed above or through electronics supply houses like Radio Shack) and capillary B is connected to the negative electrode of the power supply (). Capillary B is filled with a trapping material () capable of trapping the positively-charged reaction products by allowing minimal migration of products that have entered the trapping material (). Suitable trapping materials include, but are not limited to, high percentage (e.g., about 20%) acrylamide polymerized in a high salt buffer (0.5 M or higher sodium acetate or similar salt); such a high percentage polyacrylamide matrix dramatically slows the migration of the positively-charged reaction products. Alternatively, the trapping material may comprise a solid, negatively-charged matrix, such as negatively-charged latex beads, that can bind the incoming positively-charged products. It should be noted that any amount of trapping material () capable of inhibiting any concentrating the positively-charged reaction products may be used. Thus, while the capillary B in  only contains trapping material in the lower, submerged portion of the tube, the trapping material () can be present in the entire capillary (B); similarly, less trapping material () could be present than that shown in  because the positively-charged reaction products generally accumulate within a very small portion of the bottom of the capillary (B). The amount of trapping material need only be sufficient to make contact with the reaction solution () and have the capacity to collect the reaction products. When capillary B is not completely filled with the trapping material, the remaining space is filled with any conductive material (); suitable conductive materials are discussed below.","By comparison, the capillary (A) connected to the positive electrode of the power supply  may be filled with any conductive material (; indicated by the hatched lines in FIG. ). This may be the sample reaction buffer (e.g., 10 mM MOPS, pH 7.5 with 150 mM LiCl, 4 mM MnCl), a standard electrophoresis buffer (e.g., 45 mM Tris-Borate, pH 8.3, 1.4 mM EDTA), or the reaction solution () itself. The conductive material () is frequently a liquid, but a semi-solid material (e.g., a gel) or other suitable material might be easier to use and is within the scope of the present invention. Moreover, that trapping material used in the other capillary (i.e., capillary B) may also be used as the conductive material. Conversely, it should be noted that the same conductive material used in the capillary (A) attached to the positive electrode may also be used in capillary B to fill the space above the region containing the trapping material () (see FIG. ).","The top end of each of the capillaries (A and B) is connected to the appropriate electrode of the power supply () by electrode wire () or other suitable material. Fine platinum wire (e.g., 0.1 to 0.4 mm, Aesar Johnson Matthey, Ward Hill, Mass.) is commonly used as conductive wire because it does not corrode under electrophoresis conditions. The electrode wire () can be attached to the capillaries (A and B) by a nonconductive adhesive (not shown), such as the silicone adhesives that are commonly sold in hardware stores for sealing plumbing fixtures. If the capillaries are constructed of a flexible material, the electrode wire () can be secured with a small hose clamp or constricting wire (not shown) to compress the opening of the capillaries around the electrode wire. If the conducting material () is a gel, an electrode wire () can be embedded directly in the gel within the capillary.","The cleavage reaction is assembled in the reaction tube () and allowed to proceed therein as described in proceeding Examples (e.g., Examples 22-23). Though not limited to any particular volume of reaction solution (), a preferred volume is less than 10 ml and more preferably less than 0.1 ml. The volume need only be sufficient to permit contact with both capillaries. After the cleavage reaction is completed, an electric field is applied to the capillaries by turning on the power source (). As a result, the positively-charged products generated in the course of the INVADER-directed cleavage reaction that employs an oligonucleotide, which when cleaved, generates a positively charged fragment (described in Ex. ) but when uncleaved bears a net negative charge, migrate to the negative capillary, where their migration is slowed or stopped by the trapping material (), and the negatively-charged uncut and thermally degraded probe molecules migrate toward the positive electrode. Through the use of this or a similar device, the positively-charged products of the invasive cleavage reaction are separated from the other material (i.e., uncut and thermally degraded probe) and concentrated from a large volume. Concentration of the product in a small amount of trapping material () allows for simplicity of detection, with a much higher signal-to-noise ratio than possible with detection in the original reaction volume. Because the concentrated product is labeled with a detectable moiety like a fluorescent dye, a commercially-available fluorescent plate reader (not shown) can be used to ascertain the amount of product.","Suitable plate readers include both top and bottom laser readers. Capillary B can be positioned with the reaction tube () at any desired position so as to accommodate use with either a top or a bottom plate reading device.","In the alternative embodiment of the present invention depicted in , the procedure described above is accomplished by utilizing only a single capillary (B). The capillary (B) contains the trapping material () described above and is connected to an electrode wire (), which in turn is attached to the negative electrode of a power supply (). The reaction tube () has an electrode () embedded into its surface such that one surface of the electrode is exposed to the interior of the reaction tube () and another surface is exposed to the exterior of the reaction tube. The surface of the electrode () on the exterior of the reaction tube is in contact with a conductive surface () connected to the positive electrode of the power supply () through an electrode wire (). Variations of the arrangement depicted in  are also contemplated by the present invention. For example, the electrode () may be in contact with the reaction solution () through the use of a small hole in the reaction tube (); furthermore, the electrode wire () can be directly attached to the electrode wire (), thereby eliminating the conductive surface ().","As indicated in , the electrode () is embedded in the bottom of a reaction tube () such that one or more reaction tubes may be set on the conductive surface (). This conductive surface could serve as a negative electrode for multiple reaction tubes; such a surface with appropriate contacts could be applied through the use of metal foils (e.g., copper or platinum, Aesar Johnson Matthey, Ward Hill, Mass.) in much the same way contacts are applied to circuit boards. Because such a surface contact would not be exposed to the reaction sample directly, less expensive metals, such as the copper could be used to make the electrical connections.","The above devices and methods are not limited to separation and concentration of positively charged oligonucleotides. As will be apparent to those skilled in the art, negatively charged reaction products may be separated from neutral or positively charged reactants using the above device and methods with the exception that capillary B is attached to the positive electrode of the power supply () and capillary A or alternatively, electrode , is attached to the negative electrode of the power supply ().","As discussed above in Example 1, the presence of a primer upstream of a bifurcated duplex can influence the site of cleavage, and the existence of a gap between the 3\u2032 end of the primer and the base of the duplex can cause a shift of the cleavage site up the unpaired 5\u2032 arm of the structure (see also Lyamichev et al., supra and U.S. Pat. No. 5,422,253). The resulting non-invasive shift of the cleavage site in response to a primer is demonstrated in ,  and , in which the primer used left a 4-nucleotide gap (relative to the base of the duplex). In , all of the \u201cprimer-directed\u201d cleavage reactions yielded a 21 nucleotide product, while the primer-independent cleavage reactions yielded a 25 nucleotide product. The site of cleavage obtained when the primer was extended to the base of the duplex, leaving no gap was examined. The results are shown in  ( is a reproduction of  in Lyamichev et al. These data were derived from the cleavage of the structure shown in , as described in Example 1. Unless otherwise specified, the cleavage reactions comprised 0.01 pmoles of heat-denatured, end-labeled hairpin DNA (with the unlabeled complementary strand also present), 1 pmole primer (complementary to the 3\u2032 arm shown in FIG.  and having the sequence: 5\u2032-GAATTCGATTTAGGTGACAC TATAGAATACA [SEQ D NO:53]) and 0.5 units of DNAPTaq (estimated to be 0.026 pmoles) in a total volume of 10 \u03bcl of 10 mM Tris-Cl, pH 8.5, and 1.5 mM MgCland 50 mM KCl. The primer was omitted from the reaction shown in the First lane of FIG.  and included in lane 2. These reactions were incubated at 55\u00b0 C. for 10 minutes. Reactions were initiated at the final reaction temperature by the addition of either the MgClor enzyme. Reactions were stopped at their incubation temperatures by the addition of 8 \u03bcl of 95% formamide with 20 mM EDTA and 0.05% marker dyes.",{"@attributes":{"id":"p-0849","num":"0858"},"figref":"FIG. 53"},"These data show that the presence of a primer that is adjacent to a downstream duplex (lane 2) produces cleavage at the same site as seen in reactions performed in the absence of the primer (lane 1). (See , B and A for additional comparisons). When the 3\u2032 terminal nucleotides of the upstream oligonucleotide can base pair to the template strand but are not homologous to the displaced strand in the region immediately upstream of the cleavage site (i.e., when the upstream oligonucleotide is opening up a \u201cbubble\u201d in the duplex), the site to which cleavage is apparently shifted is not wholly dependent on the presence of an upstream oligonucleotide.","As discussed above in the Background, and in Table 1, the requirement that two independent sequences be recognized in an assay provides a highly desirable level of specificity. In the invasive cleavage reactions of the present invention, the INVADER and probe oligonucleotides must hybridize to the target nucleic acid with the correct orientation and spacing to enable the production of the correct cleavage product. When the distinctive pattern of cleavage is not dependent on the successful alignment of both oligonucleotides in the detection system these advantages of independent recognition are lost.","While not limiting the present invention to any particular mechanism, invasive cleavage occurs when the site of cleavage is shifted to a site within the duplex formed between the probe and the target nucleic acid in a manner that is dependent on the presence of an upstream oligonucleotide that shares a region of overlap with the downstream probe oligonucleotide. In some instances, the 5\u2032 region of the downstream oligonucleotide may not be completely complementary to the target nucleic acid. In these instances, cleavage of the probe may occur at an internal site within the probe even in the absence of an upstream oligonucleotide (in contrast to the base-by-base nibbling seen when a fully paired probe is used without an INVADER). Invasive cleavage is characterized by an apparent shifting of cleavage to a site within a downstream duplex that is dependent on the presence of the INVADER oligonucleotide.","A comparison between invasive cleavage and primer-directed cleavage may be illustrated by comparing the expected cleavage sites of a set of probe oligonucleotides having decreasing degrees of complementarity to the target strand in the 5\u2032 region of the probe (i.e., the region that overlaps with the INVADER). A simple test, similar to that performed on the hairpin substrate above (Ex. ), can be performed to compare invasive cleavage with the non- invasive primer-directed cleavage described above. Such a set of test oligonucleotides is diagrammed in FIG. . The structures shown in  are grouped in pairs, labeled \u201ca\u201d, \u201cb\u201d, \u201cc\u201d, and \u201cd\u201d . Each pair has the same probe sequence annealed to the target strand (SEQ ID NO:54), but the top structure of each pair is drawn without an upstream oligonucleotide, while the bottom structure includes this oligonucleotide (SEQ ID NO:55). The sequences of the probes shown in -are listed in SEQ ID NOS:32, 56, 57 and 58, respectively. Probable sites of cleavage are indicated by the black arrowheads. (It is noted that the precise site of cleavage on each of these structures may vary depending on the choice of cleavage agent and other experimental variables. These particular sites are provided for illustrative purposes only.)","To conduct this test, the site of cleavage of each probe is determined both in the presence and the absence of the upstream oligonucleotide, in reaction conditions such as those described in Example 18. The products of each pair of reactions are then be compared to determine whether the fragment released from the 5\u2032 end of the probe increases in size when the upstream oligonucleotide is included in the reaction.","The arrangement shown in , in which the probe molecule is completely complementary to the target strand, is similar to that shown in FIG. . Treatment of the top structure with the 5\u2032 nuclease of a DNA polymerase would cause exonucleolytic nibbling of the probe (i.e., in the absence of the upstream oligonucleotide). In contrast, inclusion of an INVADER oligonucleotide would cause a distinctive cleavage shift similar, to those observed in FIG. .","The arrangements shown in and have some amount of unpaired sequence at the 5\u2032 terminus of the probe (3 and 5 bases, respectively). These small 5\u2032 arms are suitable cleavage substrate for the 5\u2032 nucleases and would be cleaved within 2 nucleotide's of the junction between the single stranded region and the duplex. In these arrangements, the 3\u2032 end of the upstream oligonucleotide shares identity with a portion of the 5\u2032 region of the probe that is complementary to the target sequence (that is the 3\u2032 end of the INVADER has to compete for binding to the target with a portion of the 5\u2032 end of the probe). Therefore, when the upstream oligonucleotide is included it is thought to mediate a shift in the site of cleavage into the downstream duplex (although the present invention is not limited to any particular mechanism of action), and this would, therefore, constitute invasive cleavage. If the extreme 5\u2032 nucleotides of the unpaired region of the probe were able to hybridize to the target strand, the cleavage site in the absence of the INVADER might change but the addition of the INVADER oligonucleotide would still shift the cleavage site to the proper position.","Finally, in the arrangement shown in , the probe and upstream oligonucleotides share no significant regions of homology, and the presence of the upstream oligonucleotide would not compete for binding to the target with the probe. Cleavage of the structures shown in would occur at the same site with or without the upstream oligonucleotide, and is thus would not constitute invasive cleavage.","By examining any upstream oligonucleotide\/probe pair in this way, it can easily be determined whether the resulting cleavage is invasive or merely primer-directed. Such analysis is particularly useful when the probe is not fully complementary to the target nucleic acid, so that the expected result may not be obvious by simple inspection of the sequences.","In order to develop nucleases having useful activities for the cleavage of nucleic acids the following modified nucleases were produced.","a) CLEAVASE BN\/thrombin Nuclease","i) Cloning and Expression of CLEAVASE BN\/thrombin Nuclease","Site directed mutagenesis was used to introduce a protein sequence recognized by the protease thrombin into the region of the CLEAVASE BN nuclease that is thought to form the helical arch of the protein through which the single-stranded DNA that is cleaved must presumably pass. Mutagenesis was carried out using the Transformer\u2122 mutagenesis kit (Clonetech, Palo Alto, Calif.) according to manufacturer's protocol using the mutagenic oligonucleotide 5\u2032-GGGAAAGTCCTCGCAGCCGCGCG GGACGAGCGTGGGGGCCCG (SEQ ID NO:59). After mutagenesis, the DNA was sequenced to verify the insertion of the thrombin cleavage site. The DNA sequence encoding the CLEAVASE BN\/thrombin nuclease is provided in SEQ ID NO:60; the amino acid sequence of CLEAVASE BN\/thrombin nuclease is provided in SEQ ID NO:61.","A large scale preparation of the thrombin mutant (i.e., CLEAVASE BN\/thrombin) was done using cells overexpressing the CLEAVASE BN\/thrombin nuclease as described in Example 28.","ii) Thrombin Cleavage of CLEAVASE BN\/thrombin","Six point four (6.4) mg of the purified CLEAVASE BN\/thrombin nuclease was digested with 0.4 U of thrombin (Novagen) for 4 hours at 23\u00b0 C. or 37\u00b0 C. Complete digestion was verified by electrophoresis on a 15% SDS polyacrylamide gel followed by staining with Coomassie Brilliant Blue R. Wild-type CLEAVASE BN nuclease was also digested with thrombin as a control. The resulting gel is shown in FIG. .","In , lane 1 contains molecular weight markers (Low-Range Protein Molecular Weight Markers; Promega), lane 2 contains undigested CLEAVASE BN\/thrombin nuclease, lanes 3 and 4 contain CLEAVASE BN\/thrombin nuclease digested with thrombin at 23\u00b0 C. for 2 and 4 hours, respectively, and lanes 5 and 6 contain CLEAVASE BN\/thrombin nuclease digested with thrombin at 37\u00b0 C. for 2 and 4 hours, respectively. These results show that the CLEAVASE BN\/thrombin nuclease has an apparent molecular weight of 36.5 kilodaltons and demonstrate that CLEAVASE BN\/thrombin nuclease is efficiently cleaved by thrombin. In addition, the thrombin cleavage products have approximate molecular weights of 27 kilodaltons and 9 kilodaltons, the size expected based upon the position of the inserted thrombin site in the CLEAVASE BN\/thrombin nuclease.","To determine the level of hairpin cleavage activity in digested and undigested CLEAVASE BN\/thrombin nuclease, dilutions were made and used to cleave a test hairpin containing a 5\u2032 fluoroscein label. Varying amounts of digested and undigested CLEAVASE BN\/thrombin nuclease were incubated with 5 \u03bcM oligonucleotide S-60 hairpin (SEQ ID NO:29; see ) in 10 mM MOPS (pH 7.5), 0.05% Tween-20, 0.05% NP-40, and 1 mM MnClfor 5 minutes at 60\u00b0 C. The digested mixture was electrophoresed on a 20% acrylamide gel and visualized on a Hitachi FMBIO 100 fluoroimager. The resulting image is shown in FIG. .","In , lane 1 contains the no enzyme control, lane 2 contains reaction products produced using 0.01 ng of CLEAVASE BN nuclease, lanes 3, 4, and 5 contain reaction products produced using 0.01 ng, 0.04 ng, and 4 ng of undigested CLEAVASE BN\/thrombin nuclease, respectively, and lanes 6, 7, and 8 contain reaction products produced using 0.01 ng, 0.04 ng, and 4 ng of thrombin-digested CLEAVASE BN\/thrombin nuclease, respectively. The results shown in  demonstrated that the insertion of the thrombin cleavage site reduced cleavage activity about 200-fold (relative to the activity of CLEAVASE BN nuclease), but that digestion with thrombin did not reduce the activity significantly.","M13 single-stranded DNA was used as a substrate for cleavage by CLEAVASE BN nuclease and digested and undigested CLEAVASE BN\/thrombin nuclease. Seventy nanograms of single-stranded M13 DNA (NEB) was incubated in 10 mM MOPS, pH 7.5, 0.05% Tween-20, 0.05% NP-40, 1 mM MgClor 1 mM MnCl, with 8 ng of CLEAVASE BN nuclease, undigested CLEAVASE BN\/thrombin nuclease, or digested CLEAVASE BN\/thrombin nuclease for 10 minutes at 50\u00b0 C. Reaction mixtures were electrophoresed on a 0.8% agarose gel and then stained with a solution containing 0.5 \u03bcg\/ml ethidium bromide (EtBr) to visualize DNA bands. A negative image of the EtBr-stained gel is shown in FIG. .","In , lane 1 contains the no enzyme control, lane 2 contains reaction products produced using CLEAVASE BN nuclease and 1 mM MgCl, lane 3 contains reaction products produced using CLEAVASE BN nuclease and 1 mM MnCl, lane 4 contains reaction products produced using undigested CLEAVASE BN\/thrombin nuclease and 1 mM MgCl, lane 5 contains reaction products produced using undigested CLEAVASE BN\/thrombin nuclease and 1 mM MnCl, lane 6 contains reaction products produced using thrombin-digested CLEAVASE BN\/thrombin nuclease and 1 mM MgCl, and lane 7 contains reaction products produced using thrombin-digested CLEAVASE BN\/thrombin nuclease and 1 mM MnCl. The results shown in  demonstrated that the CLEAVASE BN\/thrombin nuclease had an enhanced ability to cleave circular DNA (and thus a reduced requirement for the presence of a free 5\u2032 end) as compared to the CLEAVASE BN nuclease.","It can be seen from these data that the helical arch of these proteins can be opened without destroying the enzyme or its ability to specifically recognize cleavage structures. The CLEAVASE BN\/thrombin mutant has an increased ability to cleave without reference to a 5\u2032 end, as discussed above. The ability to cleave such structures will allow the cleavage of long molecules, such as genomic DNA that, while often not circular, may present many desirable cleavage sites that are at a far removed from any available 5\u2032 end. Cleavage structures may be made at such sites either by folding of the strands (i.e., CFLP\u00ae cleavage) or by the introduction of structure-forming oligonucleotides (U.S. Pat. No. 5,422,253). 5\u2032 ends of nucleic acids can also be made unavailable because of binding of a substance too large to thread through the helical arch. Such binding moieties may include proteins such as streptavidin or antibodies, or solid supports such as beads or the walls of a reaction vessel. A cleavage enzyme with an opening in the loop of the helical arch will be able to cleave DNAs that are configured in this way, extending the number of ways in which reactions using such enzymes can be formatted.","b) CLEAVASE DN Nuclease","i) Construction and Expression of CLEAVASE DN Nuclease","A polymerization deficient mutant of Taq DNA polymerase, termed CLEAVASE DN nuclease, was constructed. CLEAVASE DN nuclease contains an asparagine residue in place of the wild-type aspartic acid residue at position 785 (D785N).","DNA encoding the CLEAVASE DN nuclease was constructed from the gene encoding for CLEAVASE A\/G (mutTaq, Ex. 2) in two rounds of site-directed mutagenesis. First, the G at position 1397 and the G at position 2264 of the CLEAVASE A\/G gene (SEQ ID NO:21) were changed to A at each position to recreate a wild-type DNAPTaq gene. As a second round of mutagenesis, the wild type DNAPTaq gene was converted to the CLEAVASE DN gene by changing the G at position 2356 to A. These manipulations were performed as follows.","DNA encoding the CLEAVASE A\/G nuclease was recloned from pTTQ18 plasmid (Ex. 2) into the pTrc99A plasmid (Pharmacia) in a two step procedure. First, the pTrc99A vector was modified by removing the G at position 270 of the pTrc99A map, creating the pTrc99G cloning vector. To this end, pTrc99A plasmid DNA was cut with NcoI and the recessive 3\u2032 ends were filled-in using the Klenow fragment of polymerase 1 in the presence of all four dNTPs at 37\u00b0 C. for 15 min. After inactivation of the Klenow fragment by incubation at 65\u00b0 C. for 10 min, the plasmid DNA was cut with EcoRI, the ends were again filled-in using the Klenow fragment in the presence of all four dNTPs at 37\u00b0 C. for 15 min. The Klenow fragment was then inactivated by incubation at 65\u00b0 C. for 10 min. The plasmid DNA was ethanol precipitated, recircularized by ligation, and used to transform JM109 cells (Promega). Plasmid DNA was isolated from single colonies and deletion of the G at position 270 of the pTrc99A map was confirmed by DNA sequencing.","As a second step, DNA encoding the CLEAVASE A\/G nuclease was removed from the pTTQ18 plasmid using EcoRI and SalI and the DNA fragment carrying the CLEAVASE A\/G nuclease gene was separated on a 1% agarose gel and isolated with Geneclean II Kit (Bio 101, Vista, Calif.). The purified fragment was ligated into the pTrc99G vector, which had been cut with EcoRI and SalI. The ligation mixture was used to transform competent JM109 cells (Promega). Plasmid DNA was isolated from single colonies and insertion of the CLEAVASE A\/G nuclease gene was confirmed by restriction analysis using EcoRI and SalI.","Plasmid DNA pTrcAG carrying the CLEAVASE A\/G nuclease gene cloned into the pTrc99A vector was purified from 200 ml of JM109 overnight culture using QIAGEN Plasmid Maxi kit (QIAGEN, Chatsworth, Calif.) according to manufacturer's protocol. pTrcAG plasmid DNA was mutagenized using two mutagenic primers, E465 (SEQ ID NO:62) (Integrated DNA Technologies, Iowa) and R754Q (SEQ ID NO:63) (Integrated DNA Technologies), and the selection primer Trans Oligo AlwNI\/SpeI (Clontech, Palo Alto, Calif., catalog #6488-1) according to Transformer\u2122 Site-Directed Mutagenesis Kit protocol (Clontech) to produce a restored wild-type DNAPTaq gene (pTrcWT).","pTrcWT plasmid DNA carrying the wild-type DNAPTaq gene cloned into the pTrc99A vector was purified from 200 ml of JM109 overnight culture using QIAGEN Plasmid Maxi kit (QIAGEN, Chatsworth, Calif.) according to manufacturer's protocol. pTrcWT was then mutagenized using the mutagenic primer D785N (SEQ ID NO:64) (Integrated DNA Technologies) and the selection primer Switch Oligo SpeI\/AlwNI (Clontech, catalog #6373-1) according to Transformer\u2122 Site-Directed Mutagenesis Kit protocol (Clontech) to create a plasmid containing DNA encoding the CLEAVASE DN nuclease. The DNA sequence encoding the CLEAVASE DN nuclease is provided in SEQ ID NO:65; the amino acid sequence of CLEAVASE DN nuclease is provided in SEQ ID NO:66.","A large scale preparation of the CLEAVASE DN nuclease was done using cells overexpressing the CLEAVASE DN nuclease as described in Example 28.","c) CLEAVASE DA Nuclease and CLEAVASE DV Nuclease","Two polymerization deficient mutants of Taq DNA polymerase, termed CLEAVASE DA nuclease and CLEAVASE DV nuclease, were constructed. The CLEAVASE DA nuclease contains a alanine residue in place of the wild-type aspartic acid residue at position 610 (D785A). The CLEAVASE DV nuclease contains a valine residue in place of the wild-type aspartic acid residue at position 610 (D610V).","i) Construction and Expression of the CLEAVASE DA and CLEAVASE DV Nucleases","To construct vectors encoding the CLEAVASE DA and DV nucleases, the CLEAVASE A\/G nuclease gene contained within pTrcAG was mutagenized with two mutagenic primers, R754Q (SEQ ID NO:63) and D610AV (SEQ ID NO:67) and the selection primer Trans Oligo AlwNI\/SpeI (Clontech, catalog #6488-1) according to the Transformer\u2122 Site-Directed Mutagenesis Kit protocol (Clontech,) to create a plasmid containing DNA encoding the CLEAVASE DA nuclease or CLEAVASE DV nuclease. The D610AV oligonucleotide was synthesized to have a purine, A or G, at position 10 from the 5\u2032 end of the oligonucleotide. Following mutagenesis, plasmid DNA was isolated from single colonies and the type of mutation present, DA or DV, was determined by DNA sequencing. The DNA sequence encoding the CLEAVASE DA nuclease is provided in SEQ ID NO:68; the amino acid sequence of CLEAVASE DA nuclease is provided in SEQ ID NO:69. The DNA sequence encoding the CLEAVASE DV nuclease is provided in SEQ ID NO:70; the amino acid sequence of CLEAVASE DV nuclease is provided in SEQ ID NO:71.","Large scale preparations of the CLEAVASE DA and CLEAVASE DV nucleases was done using cells overexpressing the CLEAVASE DA nuclease or the CLEAVASE DV nuclease as described in Example 28.","Sequences encoding thermostable FEN-1 proteins derived from three Archaebacterial species were cloned and overexpressed in . This Example involved a) cloning and expression of a FEN-1 endonuclease from ; b) cloning and expression of a FEN-1 endonuclease from ; c) cloning and expression of a FEN-1 endonuclease from ; d) cloning and expression of a FEN-1 endonuclease from ; e) large scale preparation of recombinant thermostable FEN-1 proteins; and f) activity assays using FEN-1 endonucleases.","a) Cloning and Expression of a FEN-1 Endonuclease from ","DNA encoding the FEN-1 endonuclease from () was isolated from cells and inserted into a plasmid under the transcriptional control of an inducible promoter as follows. Genomic DNA was prepared from 1 vial of live bacteria (DSMZ, Deutsche Sammlung von Mikroorganismen und Zellkulturen, Braunschweig, Germany #2661) with the DNA XTRAX kit (Gull Laboratories, Salt Lake City, Utah) according to the manufacturer's protocol. The final DNA pellet was resuspended in 100 \u03bcl of TE (10 mM Tris HCl, pH 8.0, 1 mM EDTA). One microliter of the DNA solution was employed in a PCR using the Advantage\u2122 cDNA PCR kit (Clonetech); the PCR was conducted according to manufacturer's recommendations. The 5\u2032-end primer (SEQ ID NO:72) is complementary to the 5\u2032 end of the Mja FEN-1 open reading frame with a one base substitution to create an NcoI restriction site (a fragment of the genome that contains the gene encoding (Mja) FEN-1 is available from GenBank as accession # U67585). The 3\u2032-end primer (SEQ ID NO:73) is complementary to a sequence about 15 base pairs downstream from the 3\u2032 end of the Mja FEN-1 open reading frame with 2 base substitutions to create a SalI restriction enzyme site. The sequences of the 5\u2032-end and 3\u2032-end primers are: 5\u2032-GGGATACCA TGGGAGTGCAGTTTGG-3\u2032 (SEQ ID NO:72) and 5\u2032-GGTAAATTTTTCTCGTCGA CATCCCAC-3\u2032 (SEQ ID NO:73), respectively. The PCR reaction resulted in the amplification (i.e., production) of a single major band about 1 kilobase in length. The open reading frame (ORF) encoding the Mja FEN-1 endonuclease is provided in SEQ ID NO:74; the amino acid sequence encoded by this ORF is provided in SEQ ID NO:75.","Following the PCR amplification, the entire reaction was clectrophoresed on a 1.0% agarose gel and the major band was excised from the gel and purified using the Geneclean II kit (Bio101, Vista, Calif.) according to manufacturer's instructions. Approximately 1 \u03bcg of the gel-purified Mja FEN-1 PCR product was digested with NcoI and SalI. After digestion, the DNA was purified using the Geneclean II kit according to manufacturer's instructions. One microgram of the pTrc99a vector (Pharmacia) was digested with NcoI and SalI in preparation for ligation with the digested PCR product. One hundred nanograms of digested pTrc99a vertor and 250 ng of digested Mja FEN-1 PCR product were combined and ligated to create pTrc99-MJFEN1. pTrc99-MJFEN1 as used to transform competent JM109 cells (Promega) using standard echniques.","b) Cloning and Expression Of A FEN-1 Endonuclease from ","DNA encoding the () FEN-1 endonuclease was obtained by PCR amplification using a plasmid containing DNA encoding the (Pfu) FEN-1 endonuclease (obtained from Dr. Frank Robb, Center of Marine Biotechnology, Baltimore, Md.). DNA sequences encoding a portion of the Pfu FEN-1 endonuclease can be obtained from GenBank as accession Nos. AA113505 and W36094. The amplified Pfu FEN-1 gene was inserted into the pTrc99a expression vector (Pharmacia) to place the Pfu FEN-1 gene under the transcriptional control of the inducible trc promoter. The PCR amplification was conducted as follows. One hundred microliter reactions contained 50 mM Tris HCl, pH 9.0, 20 mM (NH)SO, 2 mM MgCl, 50 \u03bcM dNTPs, 50 pmole each primer, 1 U Tfl polymerase (Epicentre Technologies, Madison, Wis.) and 1 ng of FEN-1 gene-containing plasmid DNA. The 5\u2032-end primer (SEQ ID NO:76) is complementary to the 5\u2032 end of the Pfu FEN-1 open reading frame but with two substitutions to create an NcoI site and the 3\u2032-end primer (SEQ ID NO:77) is complementary to a region located about 30 base pairs downstream of the FEN-1 open reading frame with two substitutions to create a PstI site. The sequences of the 5\u2032-end and 3\u2032-end primers are: 5\u2032-GAGGTGATACCATG GGTGTCC-3\u2032 (SEQ ID NO:76) and 5\u2032-GAAACTCTGCAGCGCGTCAG-3\u2032 (SEQ ID NO:77), respectively. The PCR reaction resulted in the amplification of a single major band about 1 kilobase in length. The open reading frame (ORF) encoding the Pfu FEN-1 endonuclease is provided in SEQ ID NO:78; the amino acid sequence encoded by this ORF is provided in SEQ ID NO:79.","Following the PCR amplification, the entire reaction was electrophoresed on a 1.0% agarose gel and the major band was excised from the gel and purified using the Geneclean II kit (Bio101) according to manufacturer's instructions. Approximately 1 \u03bcg of gel purified Pfu FEN-1 PCR product was digested with NcoI and PstI. After digestion, the DNA was purified using the Geneclean II kit according to manufacturer's instructions. One microgram of the pTrc99a vector was digested with NcoI and PstI prior to ligation with the digested PCR product. One hundred nanograms of digested pTrc99a and 250 ng of digested Pfu FEN-1 PCR product were combined and ligated to create pTrc99-PFFEN1. pTrc99-PFFEN1 was used to transform competent JM109 cells (Promega) using standard techniques.","c) Cloning and Expression of a FEN-1 Endonuclease from ","For the cloning of DNA encoding the (Pwo) FEN-1 endonuclcase, DNA was prepared from lyophilized bacteria (DSMZ # 3773) as described (Zwickl et at., J. Bact., 172:4329 [1990]) with several changes. Briefly, one vial of bacteria was rehydrated and resuspended in 0.5 ml of LB (Luria broth). The cells were centrifuged at 14,000\u00d7g for 1 min and the cell pellet was resuspended in 0.45 ml of TE. Fifty microliters of 10% SDS was added and the mixture was incubated at RT for 5 min. The cell lysate was then extracted three time with 1:1 phenol:chlorofonn and three times with chloroform. Five hundred microliters of isopropanol was added to the extracted lysate and the DNA was pelleted by centrifugation at 14,000\u00d7g for 10 min. The DNA pellet was washed in 0.5 ml of 70% ethanol and the DNA was pelleted again by centrifugation at 14,000\u00d7g for 5 min. The DNA pellet was dried and resuspended in 100 \u03bcl of TE and used for PCR reactions without further purification.","To generate a FEN-1 gene fragment for cloning into an expression vector, low stringency PCR was attempted with primers complementary to the ends of the FEN-1 gene open reading frame. The sequences of the 5\u2032-end and 3\u2032-end primers are 5\u2032-GATACCATGGGTGTCCCAATTGGTG-3\u2032 (SEQ ID NO:80) and 5\u2032-TCGACGTCGACTTATCTCTTGAACCAACTTTCAAGGG-3\u2032 (SEQ ID NO:81), respectively. The high level of sequence similarity of protein homologs (i.e., proteins other than FEN-1 proteins) from and suggested that there was a high probability that the FEN-1 gene could be amplified using primers containing sequences complementary to the FEN-1 gene. However, this approach was unsuccessful under several different PCR conditions.","The DNA sequence of FEN-1 genes from and were aligned and blocks of sequence identity between the two genes were identified. These blocks were used to design internal primers (i.e., complementary to sequences located internal to the 5\u2032 and 3\u2032 ends of the ORF) for the FEN-1 gene that are complementary to the FEN-1 gene in those conserved regions. The sequences of the 5\u2032- and 3\u2032-internal primers are 5\u2032-AGCGAGGGAGAGGCCCAAGC-3\u2032 (SEQ ID NO:82) and 5\u2032-GCCTATGCCCTTTATTCCTCC-3\u2032 (SEQ ID NO:83), respectively. A PCR employing these internal primers was conducted using the Advantage\u2122 PCR kit and resulted in production of a major band of \u02dc300 bp.","Since the PCR with the internal primers was successful, reactions were attempted that contained mixtures of the internal (SEQ ID NOS:82 and 83) and external (SEQ ID NOS:80 and 81) primers. A reaction containing the 5\u2032-end external primer (SEQ ID NO:80) and 3\u2032-end internal primer (SEQ ID NO:83) resulted in the production of a 600 bp band and a reaction containing the 5\u2032-end internal primer (SEQ ID NO:82) and 3\u2032-end external primer (SEQ ID NO:81) resulted in the production of a 750 bp band. These overlapping DNA fragments were gel-purified and combined with the external primers (SEQ ID NOS:80 and 81) in a PCR reaction. This reaction generated a 1 kb DNA fragment containing the entire Pwo FEN-1 gene open reading frame. The resulting PCR product was gel-purified, digested, and ligated exactly as described above for the Mja FEN-1 gene PCR product. The resulting plasmid was termed pTrc99-PWFEN1. pTrc99-PWFEN1 was used to transform competent JM109 cells (Promega) using standard techniques.","d) Cloning and Expression of a FEN-1 Endonuclease from ","The preliminary (Afu) chromosome sequence of 2.2 million bases was downloaded from the TIGR (The Institute for Genomic Research) world wide web site, and imported into a software program (MacDNAsis), used to analyze and manipulate DNA and protein sequences. The unannotated sequence was translated into all 6 of the possible reading frames, each comprising approximately 726,000 amino acids. Each frame was searched individually for the presence of the amino acid sequence \u201cVFDG\u201d (valine, phenylalanine, aspartic acid, glycine), a sequence that is conserved in the FEN-1 family. The amino acid sequence was found in an open reading frame that contained other amino acid sequences conserved in the FEN-1 genes and that was approximately the same size as the other FEN-1 genes. The ORF DNA sequence is shown in SEQ ID NO:164, while the ORF protein sequence is shown in SEQ ID NO:165. Based on the position of this amino acid sequence within the reading frame, the DNA sequence encoding a putative FEN-1 gene was identified.","The sequence information was used to design oligonucleotide primers that were used for PCR amplification of the FEN-1-like sequence from genomic DNA. Genomic DNA was prepared from as described in Ex. 29a for , except that one vial (approximately 5 ml of culture) of live bacteria from DSMZ (DSMZ #4304) was used. One microliter of the genomic DNA was used for PCR reaction as described in Ex. 29a. The 5\u2032 end primer is complementary to the 5\u2032 end of the Afu FEN-1 gene except it has a 1 base pair substitution to create an Nco I site. The 3\u2032 end primer is complentary to the 3\u2032 end of the Afu FEN-1 gene downstream from the FEN-1 ORF except it contains a 2 base substitution to create a SalI site. The sequences of the 5\u2032 and 3\u2032 end primers are 5\u2032-CCGTCAACATTTACCATGGGTGCGGA-3\u2032 (SEQ ID NO:166) and 5\u2032-CCGCCACCTCGTAGTCGACATCCTTTTCGTG (SEQ ID NO:167), respectively.","Cloning of the resulting fragment was as described for the PfuFEN1 gene, above, to create the plasmid pTrc99-AFFEN1. The pTrcAfuHis plasmid was constructed by modifying pTrc99-AFFEN1, by adding a histidine tail to facilitate purification. To add this histidine tail, standard PCR primer-directed mutagenesis methods were used to insert the coding sequence for six histidine residues between the last amino acid codon of the pTrc99-AFFEN1 coding region and the stop codon. The resulting plasmid was termed pTrcAfuHis. The protein was then expressed as described in Example 28(e), and purified by binding to a Ni++ affinity column, as described in Example 8.","e) Large Scale Preparation of Recombinant Thermostable FEN-1 Proteins","The Mja, Pwo and Pfu FEN-1 proteins were purified by the following technique, which is derived from a Taq DNA polymerase preparation protocol (Engelke et al., Anal. Biochem., 191:396 [1990]) as follows. cells (strain JM109) containing either pTrc99-PFFEN1, pTrc99-PWFEN1, or pTrc99-MJFEN1 were inoculated into 3 ml of LB (Luria Broth) containing 100 \u03bcg\/ml ampicillin and grown for 16 hrs at 37\u00b0 C. The entire overnight culture was inoculated into 200 ml or 350 ml of LB containing 100 \u03bcg\/ml ampicillin and grown at 37\u00b0 C. with vigorous shaking to an A600 of 0.8. IPTG (1 M stock solution) was added to a final concentration of 1 mM and growth was continued for 16 hrs at 37\u00b0 C.","The induced cells were pelleted and the cell pellet was weighed. An equal volume of 2\u00d7 DG buffer (100 mM Tris-HCl, pH 7.6, 0.1 mM EDTA) was added and the pellet was resuspended by agitation. Fifty mg\/ml lysozyme (Sigma, St. Louis, Mo.) was added to 1 mg\/ml final concentration and the cells were incubated at room temperature for 15 min. Deoxycholic acid (10% solution) was added dropwise to a final concentration of 0.2% while vortexing. One volume of HO and 1 volume of 2\u00d7 DG buffer was added and the resulting mixture was sonicated for 2 minutes on ice to reduce the viscosity of the mixture. After sonication, 3 M (NH)SOwas added to a final concentration of 0.2 M and the lysate was centrifuged at 14000 \u00d7g for 20 min at 4\u00b0 C. The supernatant was removed and incubated at 70\u00b0 C. for 60 min at which time 10% polyethylimine (PEI) was added to 0.25%. After incubation on ice for 30 min., the mixture was centrifuged at 14,000\u00d7g for 20 min at 4\u00b0 C. At this point, the supernatant was removed and the FEN-1 proteins was precipitated by the addition of (NH)SOas follows.","For the Pwo and the Pfu FEN-1 preparations, the FEN-1 protein was precipitated by the addition of 2 volumes of 3 M (NH)SO. The mixture was incubated overnight at room temperature for 16 hrs and the protein was centrifuged at 14,000\u00d7g for 20 min at 4\u00b0 C. The protein pellet was resuspended in 0.5 ml of Q buffer (50 nM Tris-HCl, pH 8.0, 0.1 mM EDTA, 0.1% Tween 20). For the Mja FEN-1 preparation, solid (NH)SOwas added to a final concentration of 3 M (\u02dc75% saturated), the mixture was incubated on ice for 30 min, and the protein was spun down and resuspended as described above.","The resuspended protein preparations were quantitated by determination of the Aand aliquots containing 2-4 \u03bcg of total protein were electrophoresed on a 10% SDS polyacrylamide gel (29:1 acrylamide: bis-acrylamide) in standard Laemmli buffer [Laemmli, Nature 277:680 [1970]) and stained with Coomassie Brilliant Blue R; the esults are shown in FIG. .","In , lane 1 contains molecular weight markers (Mid-Range Protein Molecular Weight Markers; Promega); the size of the marker proteins is indicated to the left of the gel. Lane 2 contains purified CLEAVASE BN nuclease; lanes 3-5 contain extracts prepared from expressing the Pfu, Pwo and Mja FEN-1 nucleases, respectively. The calculated (i.e., using a translation of the DNA sequence encoding the nuclease) molecular weight of the Pfu FEN-1 nuclease is 38,714 daltons and the calculated molecular weight for the Mja FEN-1 nuclease is 37,503 Daltons. The Pwo and Pfu FEN-1 proteins co-migrated on the SDS-PAGE gel and therefore, the molecular weight of the Pwo FEN-1 nuclease was estimated to be 38.7 kDa.","f) Activity Assays Using FEN-1 Endonucleases","i) Mixed Hairpin Assay","The CLEAVASE BN nuclease has an approximately 60-fold greater affinity for a 12 base pair stem-loop structure than an 8 base pair stem-loop DNA structure. As a test for activity differences between the CLEAVASE BN nuclease and the FEN-1 nucleases, a mixture of oligonucleotides having either a 8 or a 12 bp stem-loop (see , which depicts the S-33 and 11-8-0 oligonucleotides) was incubated with an extract prepared from cells overexpressing the Mja FEN-1 nuclease (prepared as described above). Reactions contained 0.05 \u03bcM of oligonucleotides S-33 (SEQ ID NO:84) and 11-8-0 (SEQ ID NO:85) (both oligonucleotides contained 5\u2032-fluorescein labels), 10 mM MOPS, pH 7.5, 0.05% Tween-20, 0.05% NP-40, 1 mM MnCl. Reactions were heated to 90\u00b0 C. for 10 seconds, cooled to 55\u00b0 C., then 1 \u03bcl of crude extract (Mja FEN-1) or purified enzyme (CLEAVASE BN nuclease) was added and the mixtures were incubated at 55\u00b0 C. for 10 minutes; a no enzyme control was also run. The reactions were stopped by the addition of fornamide\/EDTA, the samples were electrophoresed on a denaturing 20% acrylamide gel and visualized on a Hitachi FMBIO 100 fluoroimager. The resulting image is shown in FIG. .","In , lane 1 contains the reaction products generated by the CLEAVASE BN nuclcase, lane 2 contains the reaction products from the no enzyme control reaction and lane 3 contains the reaction products generated by the Mja FEN-1 nuclease. The data shown in  demonstrates that the CLEAVASE BN nuclease strongly prefers the S33 structure (12 bp stem-loop) while the Mja FEN-1 nuclease cleaves structures having either an 8 or a 12 bp stem-loop with approximately the same efficiency. This shows that the Mja FEN-1 nuclease has a different substrate specificity than the CLEAVASE BN nuclease, a useful feature for INVADER assays or CFLP\u00ae analysis as discussed in the Description of the Invention.","The majority of thermal degradation products of DNA probes will have a phosphate at the 3\u2032-end. To investigate if the template-independent DNA polymerase, terminal deoxynucleotide transferase (TdT) can tail or polymerize the aforementioned 3\u2032-end phosphates (i.e., add nucleotide triphosphates to the 3\u2032 end) the following experiment was performed.","To create a sample containing a large percentage of thermal degradation products, the 5\u2032 fluorescein-labeled oligonucleotide 34-078-01 (SEQ ID NO:86) (200 pmole) was incubated in 100 \u03bcl 10 mM NaCO(pH 10.6), 50 mM NaCl at 95\u00b0 C. for 13 hours. To prevent evaporation, the reaction mixture was overlaid with 60 \u03bcl ChillOut\u2122 14 liquid wax. The reaction mixture was then divided into two equal aliquots (A and B). Aliquot A was mixed with one-tenth volume 3M NaOAc followed by three volumes ethanol and stored at \u221220\u00b0 C. Aliquot B was dephosphorylated by the addition of 0.5 \u03bcl of 1 M MgCland 1 \u03bcl of 1 unit\/\u03bcl Calf Intestine Alkaline Phosphatase (CIAP) (Promega), with incubation at 37\u00b0 C. for 30 minutes. An equal volume of phenol:chloroform: isomayl alcohol (24:24:1) was added to the sample followed by vortexing for one minute and then centrifugation 5 minutes at maximum speed in a microcentrifuge to separate the phases. The aqueous phase was removed to a new tube to which one-tenth volume 3M NaOAc, and three volumes ethanol was added followed by storage at \u221220\u00b0 C. for 30 minutes. Both aliquots (A and B) were then centrifuged for 10 minutes at maximum speed in a microcentrifuge to pellet the DNA. The pellets were then washed two times each with 80% ethanol and then desiccated to dryness. The dried pellets were then dissolved in 70 \u03bcl ddHO each.","The TdT reactions were conducted as follows. Six mixes were assembled, all mixes contained 10 mM TrisOAc (pH 7.5), 10 mM MgOAc, 50 mM KCl, and 2 mM dATP. Mixes 1 and 2 contained one pmole of untreated 34-078-01 (SEQ ID NO:86), mixes 3 and 4 contained 2 \u03bcl of aliquot A (above), mixes 5 and 6 contained 2 \u03bcl of aliquot B (above). To each 9 \u03bcl of mixes 1,3 and 5, 1 \u03bcl ddHO was added, to each 9 \u03bcl of mixes 2, 4, and 6, 1 \u03bcl of 20 units\/\u03bcl TdT (Promega) was added. The mixes were incubated at 37\u00b0 C. for 1 hour and then the reaction was terminated by the addition of 5 \u03bcl 95% formamide with 10 mM EDTA and 0.05% marker dyes. Five microliters of each mixture was resolved by electrophoresis through a 20% denaturing acrylamide gel (19:1 cross-linked) with 7 M urea, in a buffer containing 45 mM Tris-Borate (pH 8.3), 1.4 mM EDTA, and imaged using with the FMBIO Image Analyzer with a 505 nm filter. The resulting imager scan is shown in FIG. .","In , lanes 1, 3 and 5 contain untreated 34-078-01 (SEQ ID NO:86), heat-degraded 34-078-01, and heat-degraded, dephosphorylated, 34-078-01, respectively incubated in the absence of TdT. Lanes 2, 4 and 6 contain, untreated 34-078-01, heat-degraded 34-078-01, and heat-degraded, dephosphorylated, 34-078-01, respectively incubated in the presence of TdT.","As shown in , lane 4, TdT was unable to extend thermal degradation products that contain a 3\u2032-end phosphate group, and selectively extends molecules that have a 3\u2032-end hydroxyl group.","When TdT is used to extend the specific products of cleavage, one means of detecting the tailed products is to selectively capture the extension products on a solid support before visualization. This Example demonstrates that the cleavage products can be selectively tailed by the use of TdT and deoxynucleotide triphosphates, and that the tailed products can be visualized by capture using a complementary oligonucleotide bound to a nitrocellulose support.","To extend the cleavage product produced in an INVADER-directed cleavage reaction, the following experiment was performed. Three reaction mixtures were assembled, each in a buffer of 10 mM MES (pH 6.5), 0.5%Tween-20, 0.5% NP-40. The first mixture contained 5 fmols of target DNA-M13mp18, 10 pmols of probe oligo 32-161-2 (SEQ ID NO:87; this probe oligonucleotide contains 3\u2032 ddC and a Cy3 amidite group near the 3\u2032 end), and 5 pmols of INVADER oligonucleotide 32 161-1 (SEQ ID NO:88; this oligo contains a 3\u2032 ddC). The second mixture contained the probe and INVADER oligonucleotides without target DNA. The third mixture was the same as the first mixture, and contained the same probe sequence, but with a 5\u2032 fluorescein label (oligo 32-161-4 [SEQ ID NO:89; this oligo contains a 3\u2032 ddC, 5\u2032 fluorescein label, and a Cy3 dye group near the 3\u2032 end]), so that the INVADER-directed cleavage products could be detected before and after cleavage by fluorescence imaging. The probe only control sample contained 10 pmols of oligo 32-161-2 (SEQ ID NO:87). Each 3 \u03bcl of enzyme mix contained 5 ng of CLEAVASE DN nuclease in 7.5 MM MgCl. The TdT mixture (per each 4 \u03bcl) contained: 10 U of TdT (Promega), 1 MM CoCl, 50 mM KCl, and 100 \u03bcM of dTTP. The INVADER cleavage reaction mixtures described above were assembled in thin wall tubes, and the reactions were initiated by the addition of 3 \u03bcl of CLEAVASE DN enzyme mix. The reactions were incubated at 65\u00b0 C. for 20 min. After cooling to 37\u00b0 C., 4 \u03bcl of the TdT mix was added and the samples were incubated for 4 min at 37\u00b0 C., Biotin-16-dUTP was then added to 100 \u03bcM and the samples were incubated for 50 min at 37\u00b0 C. The reactions were terminated by the addition of 1 \u03bcl of 0.5 M EDTA.","To test the efficiency of tailing the products were run on an acrylamide gel. Four microliters of each reaction mixture was mixed with 2.6 \u03bcl of 95% formrnamide, 10 mM EDTA and 0.05% methyl violet and heated to 90\u00b0 C. for 1 min, and 3 \u03bcl were loaded on a 20% denaturing acrylamide gel (19:1 cross-linked ) with 7 M urea, in buffer containing 45 mM Tris-Borate (pH 8.3), 1.4 mM EDTA. A marker \u03a6X174-HinfI [fluorescein labeled]) also was loaded. After electrophoresis, the gel was analyzed using a FMBIO-100 Image Analyzer (Hitachi) equipped with a 505 nm filter. The resulting scan is shown in FIG. .","In , lane 1 contained the probe 32-161-2 only, without any treatment. Lanes 2 and 3 contained the products of reactions run without target DNA, without or with subsequent TdT tailing, respectively. Lanes 4 and 5 contained the products of reactions run with target DNA, probe oligo 32-161-2 (SEQ ID NO:87) and INVADER oligo 32-161-1 (SEQ ID NO:88), without or with subsequent TdT tailing, respectively. Lanes 6 and 7 show the products of reactions containing target DNA, probe oligo 32-161-4 (SEQ ID NO:89) and INVADER oligo 32-161-1 (SEQ ID NO:88), without or with subsequent TdT tailing, respectively. Lane M contains the marker \u03a6X174-HinfI.","The reaction products in lanes 4 and 5 are the same as those seen in lanes 6 and 7, except that the absence of a 5\u2032 fluorescein on the probe prevents detection of the relased 5\u2032 product (indicated as \u201cA\u201d near the bottom of the gel) or the TdT extended 5\u2032 product (indicated as \u201cB\u201d, near the top of the gel). The Cy3-labeled 3\u2032 portion of the cleaved probe is visible in all of these reactions (indicated as \u201cC\u201d, just below the center of the gel).","To demonstrate detection of target-dependent INVADER-directed cleavage products on a solid support, the reactions from lanes 3 and 5 were tested on the Universal GENECOMB (Bio-Rad), which is a standard nitrocellulose matrix on a rigid nylon backing styled in a comb format, as depicted in FIG. . Following the manufacturer's protocol, with one modification: 10 \u03bcl of the INVADER-directed cleavage reactions were used instead the recommended 10% of a PCR. To capture the cleavage products, 2.5 pmols of the capture oligo 59-28-1 (SEQ ID NO:90) were spotted on each tooth. The capture and visualization steps were conducted according to the manufacturer's directions. The results are shown in FIG. .","In , teeth numbered  and  show the capture results of reactions performed without and with target DNA present. Tooth  shows the kit positive control.","The darkness of the spot seen on tooth , when compared to tooth , clearly indicates that products of INVADER-directed cleavage assays may be specifically detected on solid supports. While the Universal GENECOMB was used to demonstrate solid support capture in this instance, other support capture methods known to those skilled in the art would be equally suitable. For example, beads or the surfaces of reaction vessels may easily be coated with capture oligonucleotides so that they can then be used in this step. Alternatively, similar solid supports may easily be coated with streptavidin or antibodies for the capture of biotin- or hapten-tagged products of the cleavage\/tailing reaction. In any of these embodiments, the products may be appropriately visualized by detecting the resulting fluorescence, chemiluminescence, colorimetric changes, radioactive emissions, optical density change or any other distinguishable feature of the product.","To investigate the effect of the length of invasion as well as the effect of the type of dye on ability of Pfu FEN-1 and the CLEAVASE A\/G nuclease to cleave 5\u2032 arms, the following experiment was performed. Three probes of similar sequences labeled with either fluorescein, TET, or Cy3, were assembled in reactions with three INVADER oligonucleotides that created overlapping target hybridization regions of eight, five, and three bases along the target nucleic acid, M13mp18.","The reactions were conducted as follows. All conditions were performed in duplicate. Enzyme mixes for Pfu FEN-1 and the CLEAVASE A\/G nuclease were assembled. Each 2 \u03bcl of the Pfu FEN-1 mix contained 100 ng of Pfu FEN-1 (prepared as described in Ex. 28) and 7.5 mM MgCl. Each 2 \u03bcl of the CLEAVASE A\/G mix contained 5.3 ng of the CLEAVASE A\/G nuclease and 4.0 mM MnCl. Six master mixes containing buffer, M13mp18, and INVADER oligonucleotides were assembled. Each 7 \u03bcl of mixes 1-3contained 1 fmol M13mp18, 10 pmoles INVADER oligonucleotide (34-078-4 [SEQ ID NO:39], 24-181-2 [SEQ ID NO:91], or 24-181-1 [SEQ ID NO: 92], in 10 mM MOPS (pH 7.5), 150 mM LiCl. Each 7 \u03bcl of mixes 4-6 contained 1 fmol of M13mp18, 10 pmoles of INVADER oligonucleotide [34-078-4 (SEQ ID NO:39), 24-181-2 (SEQ ID NO:91), or 24-181-1 (SEQ ID NO:92)] in 10 mM Tris (pH 8.0). Mixtures 1-6 were then divided into three mixtures each, to which was added either the fluorescein-labeled probe (oligo 34-078-01; SEQ ID NO:86), the Cy3-labeled probe (oligo 43-20; SEQ ID NO:93) or the TET-labeled probe (oligo 90; SEQ ID NO:32 containing a 5\u2032 TET label). Each 7 \u03bcl of all mixtures contained 10 pmoles of corresponding probe. The DNA solutions described above were covered with 10 \u03bcl of CHILLOUT evaporation barrier and brought to 68\u00b0 C.","The reactions made from mixes 1-3 were started with 2 \u03bcl of the CLEAVASE A\/G nuclease mix, and the reactions made from mixes 4-6 were started with 2 \u03bcl of the Pfu FEN-1 mix. After 30 minutes at 68\u00b0 C., the reactions were terminated by the addition of 8 \u03bcl of 95% formamide with 10 mM EDTA and 0.05% marker dyes. Samples were heated to 90\u00b0 C. for 1 minute immediately before electrophoresis through a 20% denaturing acrylamide gel (19:1 cross-linked) with 7 M urea, in a buffer containing 45 mM Tris-Borate (pH 8.3), 1.4 mM EDTA. The products of the cleavage reactions were visualized following electrophoresis by the use of a Hitachi FMBIO fluorescence imager. Results from the fluorescein-labeled probe are shown in , results from the Cy3-labeled probe in , and results from the TET-labeled probe in FIG. . In each of these Figures, the products of cleavage by CLEAVASE A\/G are shown in lanes 1-6 and the products of cleavage by PfuFEN-1 are shown in lanes 7-12. In each in case the uncut material appears as a very dark band near the top of the gel, indicated by a \u201cU\u201d on the left. The products of cleavage directed by INVADER oligonucleotides with 8, 5 or 3 bases of overlap (i.e., the \u201cX\u201d region was 8, 5, or 3 nt long) are shown in the first, second and third pair of lanes in each set, respectively and the released labeled 5\u2032 ends from these reactions are indicated by the numbers 8, 5, and 3 on the left. Note that in the cleavage reactions shown in  the presence of the positively charged Cy3 dye causes the shorter products to migrate more slowly than the larger products. These products do not contain any additional positive charges (e.g., amino modifications as used in Example 23), and thus still carry a net negative charge, and migrate towards the positive electrode in a standard electrophoresis run.","It can be seen from these data that the CLEAVASE A\/G and Pfu FEN-1 structure-specific nucleases respond differently to both dye identity and to the size of the piece to be cleaved from the probe. The Pfu FEN-1 nuclease showed much less variability in response to dye identity than did the CLEAVASE A\/G nuclease, showing that any dye wold be suitable for use with this enzyme. In contrast, the amount of cleavage catalyzed by the CLEAVASE A\/G nuclease varied substantially with dye identity. Use of the fluorescein dye gave results very close to those seen with the Pfu FEN-1 nuclease, while the use of either Cy3 or TET gave dramatically reduced signal when compared to the Pfu FEN-1 reactions. The one exception to this was in the cleavage of the 3 nt product carrying a TET dye (lanes 5 and 6, FIG. ), in which the CLEAVASE A\/G nuclease gave cleavage at the same rate as the Pfu FEN-1 nuclease. These data indicate that, while CLEAVASE A\/G may be used to cleave probes labeled with these other dyes, the Pfu FEN-1 nuclease is a preferred nuclease for cleavage of Cy3- and TET-labeled probes.","To investigate whether the positive charges on 5\u2032 end of probe oligonucleotides containing a positively charged adduct(s) (i.e., charge reversal technology or CRT probes as described in Ex. 23 and 24 have an effect on the ability of the CLEAVASE A\/G or Pfu FEN-1 nucleases to cleave the 5\u2032 arm of the probe, the following experiment was performed.","Two probe oligonucleotides having the following sequences were utilized in INVADER reactions: Probe 34-180-1: (N-Cy3)TTCCAGAGCCTAATTTGCC AGT(N-fluorescein)A, where N represents a spacer containing either the Cy3 or fluorescein group (SEQ ID NO:94) and Probe 34-180-2: 5\u2032-(N-TET)TTCCAGAGCC TAATTTGCCAGT-(N-fluorescein)A, where N represents a spacer containing either the TET or fluorescein group (SEQ ID NO:95). Probe 34-180-1 has amino-modifiers on the two 5\u2032 end T residues and a Cy3 label on the 5\u2032 end, creating extra positive charges on the 5\u2032 end. Probe 34-180-2 has a TET label on the 5\u2032 end, with no extra positive charges. The fluorescein label on the 3\u2032 end of probe 34-180-1 enables the visualization of the 3\u2032 cleaved products and uncleaved probes together on an acrylamide gel run in the standard direction (i.e., with the DNA migrating toward the positive electrode). The 5\u2032 cleaved product of probe 34-180-1 has a net positive charge and will not migrate in the same direction as the uncleaved probe, and is thus visualized by resolution on a gel run in the opposite direction (i.e.; with this DNA migrating toward the negative electrode).","The cleavage reactions were conducted as follows. All conditions were performed in duplicate. Enzyme mixes for the Pfu FEN-1 and CLEAVASE A\/G nucleases were assembled. Each 2 \u03bcl of the Pfu FEN-1 mix contained 100 ng of Pfu FEN-1 (prepared as described in Ex. 28) and 7.5 mM MgCl. Each 2 \u03bcl of the CLEAVASE A\/G nuclease mix contained 26.5 ng of CLEAVASE A\/G nuclease and 4.0 mM MnCl. Four master mixes containing buffer, M13mp18, and INVADER oligonucleotides were assembled. Each 7 \u03bcl of mix 1 contained 5 fmol M13mp18, 10 pmoles INVADER oligonucleotide 123 (SEQ ID NO:96) in 10 mM HEPES (pH 7.2). Each 7 \u03bcl of mix 2 contained 1 fmol M13mp18, 10 pmoles INVADER oligonucleotide 123 in 10 mM HEPES (pH 7.2). Each 7 \u03bcl of mix 3 contained 5 fmol M13mp18, 10 pmoles INVADER oligonucleotide 123 in 10 mM HEPES (pH 7.2), 250 mM KGlu. Each 7 \u03bcl of mix 4 contained 1 fmol M13mp18, 10 pmoles INVADER oligonucleotide 123 in 10 mM HEPES (pH 7.2), 250 mM KGlu. For every 7 \u03bcl of each mix, 10 pmoles of either probe 34-180-1 (SEQ ID NO:94) or probe 34-180-2 (SEQ ID NO:95) was added. The DNA solutions described above were covered with 10 \u03bcl of CHILLOUT evaporation barrier and brought to 65\u00b0 C. The reactions made from mixes 1-2 were started by the addition of 2 \u03bcl of the Pfu FEN-1 mix, and the reactions made from mixes 3-4 were started by the addition of 2 \u03bcl of the CLEAVASE A\/G nuclease mix. After 30 minutes at 65\u00b0 C., the reactions were terminated by the addition of 8 \u03bcl of 95% formamide containing 10 mM EDTA. Samples were heated to 90\u00b0 C. for 1 minute immediately before electrophoresis through a 20% denaturing acrylamide gel (19:1 cross-linked) with 7 M urea, in a buffer containing 45 mM Tris-Borate (pH 8.3), 1.4 mM EDTA and a 20% native acrylamide gel (29:1 cross-linked) in a buffer containing 45 mM Tris-Borate (pH 8.3), 1.4 mM EDTA.","The products of the cleavage reactions were visualized following electrophoresis by the use of a Hitachi FMBIO fluorescence imager. The resulting images are shown in FIG. .  shows the denaturing gel, which was run in the standard electrophoresis direction, and  shows the native gel, which was run in the reverse direction. The reaction products produced by Pfu FEN-1 and CLEAVASE A\/G nucleases arc shown in lanes 1-8 and 9-16, respectively. The products from the 5 fmol M13mp18 and 1 fmol M13mp18 reactions are shown in lanes 1-4, 9-12 (5 fmol) and 5-8, 13-16 (1 fmol). Probe 34-180-1 is in lanes 1-2, 5-6, 9-10, 13-14 and probe 34-180-2 is in lanes 3-4, 7-8, 11-12, 15-16.","The fluorescein-labeled 3\u2032 end fragments from all cleavage reactions are shown in , indicated by a \u201c3\u2032\u201d mark at the left. The 3 nt 5\u2032 TET-labeled products are not visible in this Figure, while the 5\u00b0 Cy3-labeled products are shown in FIG. B.","The 3\u2032 end bands in  can be used to compare the rates of cleavage by the different enzymes in the presence of the different 5\u2032 end labels. It can be seen from this band that regardless of the amount of target nucleic acid present, both the Pfu FEN-1 and the CLEAVASE A\/G nucleases show more product from the 5\u2032 TET-labeled probe. With the Pfu FEN-1 nuclease this preference is modest, with only an approximately 25 to 40% increase in signal. In the case of the CLEAVASE A\/G nuclease, however, there is a strong preference for the 5\u2032 TET label. Therefore, although when the charge reversal method is used to resolve the products, a substantial amount of product is observed from the CLEAVASE A\/G nuclease-catalyzed reactions, the Pfu FEN-1 nuclease is a preferred enzyme for cleavage of Cy3-labeled probes.","The term \u201cdegenerate base\u201d refers to a base on a nucleotide that does not hydrogen bond in a standard \u201cWatson-Crick\u201d fashion to a specific base complement (i.e., A to T and G to C). For example, the inosine base can be made to pair via one or two hydrogen bonds to all of the natural bases (the \u201cwobble\u201d effect) and thus is called degenerate. Alternatively, a degenerate base may not pair at all; this type of base has been referred to as a \u201cuniversal\u201d base because it can be placed opposite any nucleotide in a duplex and, while it cannot contribute stability by base-pairing, it does not actively destabilize by crowding the opposite base. Duplexes using these universal bases are stabilized by stacking interactions only. Two examples of universal bases, 3-nitropyrrole and 5-nitroindole, are shown in FIG. . In hybridization, placement of a 3-nitropyrrole three bases from a mismatch position enhances the differential recognition of one base mismatches. The enhanced discrimination seems to come from the destabilizing effect of the unnatural base (i.e., an altered Tin close proximity to the mismatch). To test this same principle as a way of sensitively detecting mismatches using the INVADER-directed cleavage assay, INVADER oligonucleotides were designed using the universal bases shown in , in the presence or absence of a natural mismatch. In these experiments, the use of single nitropyrrole bases or pairs of nitroindole bases that flank the site of the mismatch were examined.","The target, probe and INVADER oligonucleotides used in these assays are shown in  43 nucleotide oligonucleotide (oligo 109; SEQ ID NO:97) was used as the target. The probe oligonucleotide (oligo 61; SEQ ID NO:50) releases a net positively charged labeled product upon cleavage. In , the INVADER oligonucleotide is shown schematically above the target oligonucleotide as an arrow; the large arrowhead indicates the location of the mismatch between the INVADER oligos and the target. Under the target oligonucleotide, the completely complementary, all natural (i.e., no universal bases) INVADER oligo (oligo 67; SEQ ID NO:51) and a composite of INVADER oligos containing universal bases (\u201cX\u201d) on either side of the mismatch (\u201cM\u201d) are shown. The following INVADER oligos were employed: oligo 114 (SEQ ID NO:98), which contains a single nt mismatch; oligo 115 (SEQ ID NO:99), which contains two 5-nitroindole bases and no mismatch; oligo 116 (SEQ ID NO:100), which contains two 5-nitroindole bases and a single nt mismatch; oligo 112 (SEQ ID NO:101), which contains one 3-nitropyrrole base and no mismatch; oligo 113 (SEQ ID NO:102), which contains one 5-nitropyrrole base and a single nt mismatch; and oligo 67 (SEQ ID NO:51), which is completely complementary to the target.","The INVADER-directed cleavage reactions were carried out in 10 \u03bcl of 10 mM MOPS (pH 7.2), 100 mM KCl, containing 1 \u03bcM of the appropriate invading oligonucleotide (oligos 67, 112-116), 10 nM synthetic target 109, 1 \u03bcM Cy-3 labeled to probe 61 and 2 units of CLEAVASE DV (prepared as described in Ex. 27). The reactions were overlayed with Chill-Out\u00ae liquid wax, brought to the appropriate reaction temperature, 52\u00b0 C., 55\u00b0 C., or 58\u00b0 C. and initiated with the addition of 1 \u03bcl of 40 mM MnCl. Reactions were allowed to proceed for 1 hour and were stopped by the addition of 10 \u03bcl formamide. One fourth of the total volume of each reaction was loaded onto 20% non-denaturing polyacrylamide gels, which were electrophoresed in the reverse direction. The products were visualized using an Hitachi FMBIO-100 fluorescent scanner using a 585 nm filter. The resulting images are shown in . In each panel, lanes 1-6 contain reactions products from reactions using INVADER oligo 67, 114, 115, 116, 112 and 113, respectively. Reactions run at 52\u00b0 C., 55\u00b0 C. and 58\u00b0 C. are shown in Panels A, B and C, respectively.","These data show that two flanking 5-nitroindoles display a significantly greater differentiation then does the one 3-nitropyrrole system, or the all natural base hybridization, and this increased sensitivity is not temperature dependent. This demonstrates that the use of universal bases is a useful means of sensitively detecting single base mismatches between the target nucleic acid and the complex of detection oligonucleotides of the present invention.","It is demonstrated herein that very short probes can be used for sensitive detection of target nucleic acid sequences (Ex. 37). In this Example, it is demonstrated that the short probes work very poorly when mismatched to the target, and thus can be used to distinguish a given nucleic acid sequence from a close relative with only a single base difference. To test this system synthetic human ras oncogene target sequences were created that varied from each other at one position. Oligonucleotide 166 (SEQ ID NO:103) provided the wild-type ras target sequence. Oligonucleotide 165 (SEQ ID NO:104) provided the mutant ras target sequence. The sequence of these oligonucleotides are shown in , and the site of the sequence variation in the site corresponding to codon 13 of the ras gene is indicated. The INVADER oligonucleotide (oligo 162) has the sequence: 5\u2032-GCTCAAGGCACTCTTGCC TACGA-3\u2032 (SEQ ID NO:105), where the \u201cS\u201d indicates thiol linkages (i.e., these are 2\u2032-dcoxynucleotide-5\u2032-O-(1-thiomonophates)). The miniprobe (oligo 161) has the sequence: 5\u2032-(N-Cy3) TTCACCAG-3\u2032 (SEQ ID NO:106) and is designed to detect the mutant ras target sequence (i.e., it is completely complementary to oligo 165). The stacker oligonucleotide (oligo 164) has the sequence: 5\u2032-CTCCAACTACCACAAGTTTATATTCAG-3\u2032 (SEQ ID NO:107). A schematic showing the assembly of these oligonucleotides into a cleavage structure is depicted in FIG. .","Each cleavage reaction contained 100 nM of both the invading (oligo 162) and stacking (oligo 164) oligonucleotides, 10 \u03bcM Cy3-labeled probe (oligo 161) and 100 pM of either oligo 165 or oligo 166 (target DNA) in 10 \u03bcl of 10 mM HEPES (pH 7.2), 250 mM KGlu, 4 mM MnCl. The DNA mixtures were overlaid with mineral oil, heated to 90\u00b0 C. for 15 sec then brought to a reaction temperature of 47\u00b0, 50\u00b0, 53\u00b0 or 56\u00b0 C. Reactions were initiated by the addition of 1 \u03bcl of 100 ng\/\u03bcl Pfu FEN-1. Reactions were allowed to proceed for 3 hours and stopped by the addition of 10 \u03bcl formamide. One fourth of the total volume od each reaction was loaded onto a 20% non-denaturing polyacrylamide gel, which was electrophoresed in the reverse direction. The gel was scanned using an Hitachi FMBIO-100 fluorescent scanner fitted with a 585 nm filter, and the resulting image is shown in FIG. .","In , for each reaction temperature tested, the products from reactions containing either the mutant ras target sequence (oligo 165) or the wild-type (oligo 166) are shown.","These data demonstrate that the miniprobe can be used to sensitively discriminate between sequences that differ by a single nucleotide. The miniprobe was cleaved to produce a strong signal in the presence of the mutant target sequence, but little or no miniprobe was cleaved in the presence of the wild-type target sequence. Furthermore, the discrimination between closely related targets is effective over a temperature range of at least 10\u00b0 C., which is a much broader range of temperature than can usually be tolerated when the selection is based on hybridization alone (e.g., hybridization with ASOs). This suggests that the enzyme may be a factor in the discrimination, with the perfectly matched miniprobe being the preferred substrate when compared to the mismatched miniprobe. Thus, this system provides sensitive and specific detection of target nucleic acid sequences.","As described in the Examples above, structure-specific nucleases cleave near the junction between single-stranded and base-paired regions in a bifurcated duplex, usually about one base pair into the base-paired region. It was shown in Example 10 that thermostable 5\u2032 nucleases, including those of the present invention (e.g., CLEAVASE BN nuclease, CLEAVASE A\/G nuclease), have the ability to cleave a greater distance into the base paired region when provided with an upstream oligonucleotide bearing a 3\u2032 region that is homologous to a 5\u2032 region of the subject duplex, as shown in FIG. . It has also been determined that the 3\u2032 terminal nucleotide of the INVADER oligonucleotide may be unpaired to the target nucleic acid, and still shift cleavage the same distance into the down stream duplex as when paired. It is shown in this Example that it is the base component of the nucleotide, not the sugar or phosphate, that is necessary to shift cleavage.",{"@attributes":{"id":"p-0943","num":"0952"},"figref":["FIGS. 78A and B","FIG. 78B","FIG. 78"]},"The S-60 hairpin molecule was labeled on its 5\u2032 end with fluorescein for subsequent detection. The S-60 hairpin was incubated in the presence of a thermostable 5\u2032 nuclease in the presence or the absence of the P-15 oligonucleotide. The presence of the full duplex that can be formed by the S-60 hairpin is demonstrated by cleavage with the CLEAVASE BN 5\u2032 nuclease, in a primer-independent fashion (i.e., in the absence of the P-15 oligonucleotide). The release of 18 and 19-nucleotide fragments from the 5\u2032 end of the S-60 hairpin molecule showed that the cleavage occurred near the junction between the single and double stranded regions when nothing is hybridized to the 3\u2032 arm of the S-60 hairpin (, lane 2).","The reactions shown in  were conducted in 10 \u03bcl IX CFLP buffer with 1 mM MnCland 50 mM K-Glutamate, in the presence of 0.02 \u03bcM S-60, 0.5 \u03bcM INVADER oligonucleotide and 0.01 ng per \u03bcl CLEAVASE BN nuclease. Reactions were incubated at 40\u00b0 C. for 5 minutes and stopped by the addition of 8 \u03bcl of stop buffer (95% formamide, 20 mM EDTA, 0.02% methyl violet). Samples were heated to 75\u00b0 C. for 2 min immediately before electrophoresis through a 15% acrylamide gel (19:1 cross-linked), with 7 M urea, in a buffer of 45 mM Tris-Borate, pH 8.3, 1.4 mM EDTA. Gels were then analyzed with a FMBIO-100 Image Analyzer (Hitachi) equipped with 505 nm filter. The resulting image is shown in FIG. C.","In  lane 1 contains products from the no enzyme control; lane 2 contains products from a reaction run in the absence of an INVADER oligo; lanes 3-6 contain products from reactions run the presence of the P-14d, P-14dp, P-15 and P-15p INVADER oligos, respectively.","From the data shown in , it can be seen that the use of the P-15 INVADER oligonucleotide produces a shift in the cleavage site, while the P14 INVADER oligonucleotide with either a ribose (P14d) or a phosphorylated ribose (P14dp) did not This indicates that the 15th residue of the INVADER oligonucleotide must have the base group attached to promote the shift in cleavage. Interestingly, the addition of phosphate to the 3\u2032 end of the P15 oligonucleotide apparently reversed the shifting of cleavage site. The cleavage in this lane may in fact be cleavage in the absence of an INVADER oligonucleotide as is seen in lane 2. In experiments with 5\u2032 dye-labeled INVADER oligonucleotides with 3\u2032 phosphate groups these oligonucleotides have been severely retarded in gel migration, suggesting that either the enzyme or another constituent of the reaction (e.g., BSA) is able to bind the 3\u2032 phosphate irrespective of the rest of the cleavage structure. If the INVADER oligonucleotides are indeed being sequestered away from the cleavage structure, the resulting cleavage of the S-60 hairpin would occur in a \u201cprimer-independent\u201d fashion, and would thus not be shifted.","In addition to the study cited above, the effects of other substituents on the 3\u2032 ends of the INVADER oligonucleotides were investigated in the presence of several different enzymes, and in the presence of either Mn++ or Mg++. The effects of these 3\u2032 end modifications on the generation of cleaved product are summarized in the following table. All of modifications were made during standard oligonucleotide synthesis by the use of controlled pore glass (CPG) synthesis columns with the listed chemical moiety provided on the support as the synthesis starting residue. All of these CPG materials were obtained from Glen Research Corp. (Sterling, Va.).",{"@attributes":{"id":"p-0949","num":"0958"},"figref":"FIG. 79"},{"@attributes":{"id":"p-0950","num":"0959"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 4"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Modification Studies At 3\u2032 End Of INVADER Oligo"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Extension By","Effect on INVADER Rxn. (As"]},{"entry":[{},"Terminal","INVADER) Enzyme:Condition -"]},{"entry":["3\u2032-End Modification","Transferase","Effect"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["3\u2032 phosphate","no","A:5 - inhibits reaction,"]},{"entry":["Glen part # 20-2900-42",{},"no detectable activity"]},{"entry":["3\u2032 acridine","yes, poorly","A:5 - decrease in activity, <10%"]},{"entry":["Glen part # 20-2973-42",{},"B:5 - decrease in activity, <10%"]},{"entry":[{},{},"B:4 - decrease in activity, <10%"]},{"entry":[{},{},"C:1 -decrease in activity, <10%"]},{"entry":[{},{},"C:2 - decrease in activity, \u02dc20%"]},{"entry":[{},{},"C:4 - decrease in activity, \u02dc50%"]},{"entry":[{},{},"C:3 - decrease in activity, <5%"]},{"entry":["3\u2032 carboxylate","no","A:1 - decrease in activity, \u02dc50%"]},{"entry":["Glen part # 20-4090-42",{},"activity shift in cleavage site"]},{"entry":[{},{},"C:3 - reduces rate, <10% activity"]},{"entry":["3\u2032 nitropyrole","yes","A:5 - increase in activity, 2X"]},{"entry":"Glen part # 20-2143-42"},{"entry":["3\u2032 nitroindole","yes","A:5 - decrease in activity, \u02dc33%"]},{"entry":["Glen part # 20-2144-42",{},"activity"]},{"entry":["3\u2032 arabinose","yes","A:5 - decrease in activity, \u02dc50%"]},{"entry":["Glen part # 10-4010-90",{},"activity"]},{"entry":["3\u2032 dideoxyUTP-","no","A:5 - decrease in activity, \u02dc40%"]},{"entry":["flourescein",{},"activity"]},{"entry":["3\u2032-3\u2032 linkage","no","A:1 - equivalent cleavage"]},{"entry":["Glen part # 20-0002-01",{},"activity shift in cleavage site"]},{"entry":[{},{},"C:3 - decrease in activity, \u02dc25%"]},{"entry":[{},{},"activity"]},{"entry":["3\u2032 glyceryl","yes,","C:3 - decrease in activity, \u02dc30%"]},{"entry":["Glen part # 20-2902-42","very","activity loss of specificity of"]},{"entry":[{},"poorly","cleavage (2 sites)"]},{"entry":["3\u2032 amino modifier C7","yes","C:3 - decrease in activity, \u02dc30%"]},{"entry":["Glen part # 20-2957-42",{},"activity loss of specificity,"]},{"entry":[{},{},"multiple sites"]},{"entry":["3\u2032 phosphate","no","A:5 - inhibits reaction,"]},{"entry":["Glen part # 20-2900-42",{},"no detectable activity"]},{"entry":["3\u2032 deoxy, 2\u2032OH","yes,","A:5 - decrease in activity, <20%"]},{"entry":["Glen part # 20-2104-42","very","activity"]},{"entry":[{},"poorly","B:5 - decrease in activity, <20%"]},{"entry":[{},{},"activity"]},{"entry":[{},{},"B:3 - decrease in activity, <20%"]},{"entry":[{},{},"activity"]},{"entry":[{},{},"C:1 - equivalent activity"]},{"entry":[{},{},"C:2 - equivalent activity"]},{"entry":[{},{},"C:4 - ? increase in activity"]},{"entry":[{},{},"C:3 - decrease in activity, \u02dc40%"]},{"entry":[{},{},"activity"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}},"br":{},"ul":{"@attributes":{"id":"ul0004","list-style":"none"},"li":{"@attributes":{"id":"ul0004-0001","num":"0000"},"ul":{"@attributes":{"id":"ul0005","list-style":"none"},"li":["A) CLEAVASE DV nuclease","B) CLEAVASE BN nuclease","C) Pfu FEN-1\n\nCondition:\n","1) 4 mM MnCl, 150 mM LiCl","2) 4 mM MnClmM KCl","3) 7.5 mM MgCl, no monovalent","4) 4 mM MgCl, 50 mM KCl","5) 10 mM MgOAc, 50 mM KCl"]}}}},"It can be seen from these data that many different modifications can be used on the 3\u2032 end of the INVADER oligonucleotide without detriment. In various embodiments of the present invention, such 3\u2032 end modifications may be used to block, facilitate, or otherwise alter the hybridization characteristics of the INVADER oligonucleotide, (e.g., to increase discrimination against mismatches, or to increase tolerance of mismatches, or to tighten the association between the INVADER oligonucleotide and the target nucleic acid). Some substituents may be used to alter the behavior of the enzyme in recognizing and cleaving within the assembled complex.","Altered 3\u2032 ends may also be used to prevent extension of the INVADER oligonucleotide by either template-dependent or template-independent nucleic acid polymerase. The use of otherwise unmodified dideoxynucleotides (i.e., without attached dyes or other moieties) are a particularly preferred means of blocking extension of INVADER oligonucleotides, because they do not decrease cleavage activity, and they are absolutely unextendable.","The stacker oligonucleotides employed to form cleavage structures may serve two purposes in the detection of a nucleic acid target using a miniprobe. The stacker oligonucleotide may help stabilize the interaction of the miniprobe with the target nucleic acid, leading to greater accumulation of cleaved probe. In addition, the presence of this oligo in the complex elongates the duplex downstream of the cleavage site, which may enhance the cleavage activity of some of the enzymes of the present invention. An example of different preferences for the length of this duplex by different structure-specific nucleases is seen in the comparison of the CLEAVASE BN nuclease and the Mja FEN-1 nuclease cleavage of 8 bp and 12 bp duplex regions in FIG. . Increased affinity of the enzyme for the cleavage structure also results in increased accumulation of cleaved probe during reactions done for a set amount of time.","The amount of miniprobe binding to the target is also affected by the concentration of the miniprobe in the reaction mixture. Even when a miniprobe is only marginally likely to hybridize (e.g., when the reaction is performed at temperatures in excess of the expected melting temperature of the probe\/target duplex), the amount of probe on the target at any given time can be increased by using high concentrations of the miniprobe.","The need for a stacker oligonucleotide to enhance cleavage of the miniprobe was examined at both low and high probe concentrations. The reactions were carried out in 10 \u03bcl of 10 mM HEPES (pH 7.2), 250 mM KGlu, 4 mM MnCl, containing 100 nM of both the invading (oligo 135; SEQ ID NO:112) and stacking oligonucleotides (oligo 147; SEQ ID NO:113) and 100 pM ssM 13 DNA. The reactions were overlayed with mineral oil, heated to 90\u00b0 C. for 15 sec then brought to the reaction temperature. Reactions were performed at 35\u00b0, 40\u00b0, 45\u00b0, 50\u00b0, 55\u00b0, 60\u00b0, and 65\u00b0 C. The cleavage reactions were initiated by the addition of 1 \u03bcl of 100 ng\/\u03bcl Pfu FEN-I and 1 \u03bcl of varying concentrations of Cy-3 labeled 142 miniprobe oligonucleotide (SEQ ID NO:114). Reactions were allowed to proceed for 1 hour and stopped by the addition of 10 \u03bcl formaldehyde. One fourth of the total volume of each reaction was loaded onto 20% non-denaturing polyacrylamide gels, which were electrophoresed in the reverse direction. Gels were visualized using an Hitachi FMBIO-100 fluorescent scanner using a 585 nm filter. The fluorescence in each product band was measured and the graph shown in  was created using a Microsoft Excel spreadsheet.","The data summarized in  showed that the concentration of the miniprobe had a significant effect on the final measure of product, showing dramatic increases as the concentration was raised. Increases in the concentration of the miniprobe also shifted the optimum reaction temperature upward. It is known in the art that the concentration of the complementary strands in a hybridization will affect the apparent Tof the duplex formed between them. More significantly to the methods and compositions of the present invention is the fact that the presence of the stacker oligonucleotide has a profound influence on the cleavage rate of the miniprobe at all probe concentrations. At each of the probe concentrations the presence of the stacker as much as doubled the signal from the cleavage product. This demonstrated the utility of using the stacker oligonucleotide in combination with the miniprobes described herein.","In any nucleic acid detection assay it is of additional benefit if the assay can be made to sensitively detect minor differences between related nucleic acids. In the following experiment, model cleavage substrates were used that were identical except for the presence or absence of a mismatch near the 3\u2032 end of the INVADER oligonucleotide when hybridized to the model target nucleic acid. The effect of a mismatch in this region on the accumulation of cleaved probe was then assessed.","To demonstrate the effect of the presence of a mismatch in the INVADER oligonucleotide on the ability of the CLEAVASE A\/G nuclease to cleave the probe oligonucleotide in an INVADER assay the following experiment was conducted. Cleavage of the test oligonucleotide IT-2 (SEQ ID NO:115) in the presence of INVADER oligonucleotides IT-1 (SEQ ID NO:116) and IT-IA4 (SEQ ID NO:117). Oligonucleotide IT-1 is fully complementary to the 3\u2032 arm of IT-2, whereas oligonucleotide IT-IA4 has a T\u2192A substitution at position 4 from the 3\u2032 end that results in an A\/A mismatch in the INVADER-target duplex. Both the matched and mismatched INVADER oligonucleotides would be expected to hybridize at the temperature at which the following reaction was performed.  provides a schematic showing IT-1 annealed to the folded IT-2 structure and showing IT-IA4 annealed to the folded IT-2 structure.","The reactions were conducted as follows. Test oligonucleotide IT-2 (0.1 \u03bcM), labeled at the 5\u2032 end with fluorescein (Integrated DNA Technologies), was incubated with 0.26 ng\/\u03bcl CLEAVASE AG in 10 \u03bcl of CFLP\u00ae buffer with 4 mM MgCl, in the presence of 1 \u03bcM IT-1 or IT-IA4 at 40\u00b0 C. for 10 min; a no enzyme control was also run. Samples were overlaid with 15 \u03bcl Chill-Out\u00ae liquid wax to prevent evaporation. Reactions were stopped by addition of 4 \u03bcl stop buffer (95% formamide, 20 mM EDTA, 0.02% methyl violet). The cleavage products were separated on a 20% denaturing polyacrylamide gel and analyzed with the FMBIO-100 Image Analyzer (Hitachi) equipped with 505 nm filter. The resulting image is shown in FIG. .","In , lane 1 contains reaction products from the no enzyme control and shows the migration of the uncut IT-2 oligo; lanes 2-4 contain products from reactions containing no INVADER oligo, the IT-1 INVADER oligo and the IT-1A4 INVADER oligo, respectively.","These data show that cleavage is markedly reduced by the presence of the mismatch, even under conditions in which the mismatch would not be expected to disrupt hybridization. This demonstrates that the INVADER oligonucleotide binding region is one of the regions within the complex in which can be used for mismatch detection, as revealed by a drop in the cleavage rate.","To compare the activity of the Pfu FEN-1 and the Mja FEN-1 nucleases in INVADER reaction the following experiment was performed. A test oligonucleotide IT3 (SEQ ID NO:118) that forms an INVADER-Target hairpin structure and probe oligonucleotide PR1 (SEQ ID NO:119) labeled at the 5\u2032 end with fluorescein (Integrated DNA Technologies) were employed in INVADER assays using either the Pfu FEN-1 or the Mja FEN-1 nucleases.","The assays were conducted as follows. Pfu FEN-1 (13 ng\/\u03bcl) and Mja FEN-1 (10 ng\/\u03bcl) (prepared as described in Ex. 28) were incubated with the IT3 (0.1 nM) and PRI (2 and 5 \u03bcM) oligonucleotides in 10 \u03bcL CFLP\u00ae buffer, 4 mM MgCl, 20 mg\/ml tRNA at 55\u00b0 C. for 41 min. Samples were overlaid with 15 \u03bcl Chill-Out\u00ae evaporation barrier to prevent evaporation. Reactions were stopped by addition of 70 \u03bcl stop buffer (95% formamide, 20 mM EDTA, 0.02% methyl violet). Reaction products (1 \u03bcl) were separated on a 20% denaturing polyacrylamide gel, visualized using a fluroimager and the bands corresponding to the probe and the product were quantitiated. The resulting image is shown in FIG. . In , the turnover rate per target per minute is shown below the image for each nuclease at each concentration of probe and target tested.","It was demonstrated in Example 32 that the use of the Pfu FEN-1 structure-specific nuclease in the INVADER-directed cleavage reaction resulted in a faster rate of product accumulation than did the use of the CLEAVASE A\/G. The data presented here demonstrates that the use of Mja FEN-1 nuclease with the fluorescein labeled probe further increases the amount of product generated by an average of about 50%, demonstrating that, in addition to the Pfu FEN-1 nuclease, the Mja FEN-1 nuclease is a preferred structure-specific nuclease for the detection of nucleic acid targets by the method of the present invention.","In addition to the detection of the M13 DNA target material described above, a miniprobe\/stacker system was designed to detect the HCV-derived RNA sequences described in Example 19. A probe of intermediate length, either a long mid-range or a short standard probe, was also tested. The miniprobe used (oligo 42-168-1) has the sequence: 5\u2032-TET-CCGGTCGTCCTGG-3\u2032 (SEQ ID NO:120), the stacker oligonucleotide used (oligo 32-085) with this miniprobe has the sequence: 5\u2032-CAATTCCGGTGTACTACCGGTTCC-3\u2032 (SEQ ID NO:121). The slightly longer probe, used without a stacker (oligo 42-088), has the sequence: 5\u2032-TET-CCGGTCGTCCTGGCAA-3\u2032 (SEQ ID NO:122). The INVADER oligonucleotide used with both probes has the sequence: 5\u2032-GTTTATCCAAGAAAGGACCCGGTC-3\u2032 (SEQ ID NO:47). The reactions included 50 fmole of target RNA, 10 pmole of the INVADER oligonucleotide and 5 pmole of the miniprobe oligonucleotide in 10 \u03bcl of buffer containing 10 mM MES, pH 6.5 with 150 mM LiCl, 4 mM MnCl, 0.05% each Tween-20 and NP-40, and 39 units of RNAsin (Promega). When used, 10 pmoles of the stacker oligonucleotide was added. These components were combined, overlaid with CHILLOUT evaporation barrier, and warmed to 50\u00b0 C.; the reactions were started by the addition of 5 polymerase units of DNAPTth, to a final reaction volume of 10 \u03bcl. After 30 minutes at 50\u00b0 C., reactions were stopped by the addition of 8 \u03bcl of 95% formamide, 10 mM EDTA and 0.02% methyl violet. The samples were heated to 90\u00b0 C. for 1 minute and 2.5 \u03bcl of each of these reactions were resolved by electrophoresis through a 20% denaturing polyacrylarnide (19:1 cross link) with 7M urea in a buffer of 45 mM Tris-Borate, pH 8.3, 1.4 mM EDTA, and the labeled reaction products were visualized using the FMBIO-100 Image Analyzer (Hitachi). The resulting image is shown in FIG. .","In , lanes 1 and 2 show the products of reactions containing the HCV INVADER oligonucleotide and the longer probe (oligo 4-088), without and with the target RNA present, respectively. Lanes 3, 4, and 5 show the products of reactions containing the INVADER oligonucleotide and the shorter probe (oligo 42-168-1). Lane 3 is a control reaction without target RNA present, while lanes 4 and 5 have the target, but are without or with the stacker oligonucleotide, respectively.","Under these conditions the slightly longer (16 nt) probe oligonucleotide was cleaved quite easily without the help of a stacker oligonucleotide. In contrast, the shorter probe (13 nt) required the presence of the stacker oligonucleotide to produce detectable levels of cleavage. These data show that the miniprobe system of target detection by INVADER-directed cleavage is equally applicable to the detection of RNA and DNA targets, in addition, the comparison of the cleavage performance of longer and shorter probes in the absence of a stacker oligonucleotide give one example of the distinction between the performance of the miniprobe\/stacker system and the performance of the mid-range and long probes in the detection of nucleic acid targets.","In designing the method of transcription-based visualization of the products of INVADER-directed cleavage, it was first necessary to assess the effect of a 3\u2032 tail on the efficiency of transcription from a full length promoter. The duplexes tested in this Example are shown at the bottom of , and are shown schematically in .","Transcription reactions were performed using the MEGAshortscript\u2122 system from Ambion, Inc. (Austin, Tex.), in accordance with the manufacturer's instructions with the exception that a fluorescein labeled ribonucleotide was added. Each DNA sample was assembled in 4 \u03bcl of RNAse-free dHO. Reactions 1-3 each contained 10 pmole of the copy template oligo 150 (SEQ ID NO:123); reaction 2 contained 10 pmole of the promoter oligo 151 (SEQ ID NO:124); sample 3 contained 10 pmole of the 3\u2032 tailed promoter oligo 073-065 (SEQ ID NO:125); sample 4 had no added DNA. To each sample, 6 \u03bcl of a solution containing 1 \u03bcl of 10\u00d7 Transcription Buffer, 7.5 mM each rNTP, 0.125 mM fluorescein-12-UTP (Boehringer) and 1 \u03bcl T7 MEGAshortscript\u2122 Enzyme Mix was added. The samples were then incubated at 37\u00b0 C. for 1 hour. One microliter of RNase-free DNase 1 (2 U\/\u03bcl) was added to each sample and the samples were incubated an additional 15 minutes at 37\u00b0 C. The reactions were then stopped by the addition of 10 \u03bcl of a solution of 95% formamide, 5 mM NaEDTA, with loading dyes. All samples were heated to 95\u00b0 C. for 2 minutes and 4 \u03bcl of each sample were resolved by electrophoresis through a 20% denaturing acrylamide gel (19:1 cross-linked) with 7M urea, in a buffer containing 45mM Tris-Borate (pH 8.3), 1.4mM EDTA. The gel was analyzed with a FMBIO II fluorescence image analyzer, and the resulting image is shown in FIG. . The RNA produced by successful transcription appears near the middle of the panel, as indicated (\u201cRNA\u201d).","Examination of the products of transcription shown in lanes 2 and 3 show that the presence of the 3\u2032 tail on the full-length promoter has an adverse affect on the efficiency of transcription, but does not shut it off completely. Because the objective of the transcription-based visualization assays of the present invention is to discriminate between uncleaved probe and the shorter products of the invasive cleavage assay (cut probe), these data indicate that production of a full-length promoter in the cleavage reaction would be difficult to resolve from the background created by transcription from promoters containing the uncleaved probe if no other oligonucleotides were included in the assay. Means of suppressing transcription from such a branched promoter are discussed in the Description of the Invention and discussed below in Ex. 43.","In the Description of the Invention, the procedure for testing prospective promoter pieces for suitability in an invasive cleavage-linked assay is described. One aspect of the test is to examine the effect a chosen nick site has on the efficiency of transcription from the final composite promoter. In addition, the individual pieces of nicked promoter are tested for transcription activity in the presence of the full-length un-nicked strand. In this experiment, a comparison on these points is made between a composite promoter having a nick in the non-template strand between nucleotides \u221211 and \u221210 relative to the initiation site (+1), and a promoter having a nick on the same strand, but positioned between nucleotides \u22128 and \u22127. The Figure numbers for the schematic representations of the contents of each reaction are indicated below each lane (e.g., 85A=FIG. A). The site where the nick would be in a fully assembled composite promoter using the reaction oligonucleotides is also indicated below each lane (\u201c\u221211\/\u221210\u201d and \u201c\u22128\/\u22127\u201d).","Transcription reactions were performed using the MEGAshortscript\u2122 system, in accordance with the manufacturers instructions, but with the exception that a fluorescein labeled ribonucleotide was added. Each DNA sample was assembled in 4 \u03bcl of RNAse-free dHO. Reaction 1 had no added DNA. Reactions 2-9 each contained 10 pmole of the copy template oligo 150 (SEQ ID NO:123). Reactions 3 and 4 contained 10 pmole of the \u221211 \u201ccut\u201d probe (oligo 073-061-01; SEQ ID NO:127) or 20 pmole of the \u221210 partial promoter oligo 073-061-02 (SEQ ID NO:130), respectively, and reaction 5 contained both. Reactions 6 and 7 contained either the 10 pmole of the \u22128 \u201ccut\u201d probe (oligo 073-062-01; SEQ ID NO:126) or 20 pmoles of the \u22127 partial promoter oligo 073-062-02 (SEQ ID NO:129), respectively, and reaction 8 contained them both. Reaction 9 contained 10 pmole of the intact promoter oligo 151 (SEQ ID NO:124).","The transcription reactions were initiated, incubated, terminated and the reaction products were resolved and imaged as described in Ex. 40. The resulting image is shown in FIG. . The reaction numbers correspond to the lane numbers above the image. The RNA created by successful transcription appears in the upper third of the image. Comparison to the positive control reaction (rxn. 9) shows that the full-length RNA produced by each of the composite promoters is the same size as that produced in the control reaction, indicated that transcription initiated at the same site in each reaction.","In , lanes 3, 4, and 5 compare transcription from the two species of partially assembled promoters (see schematics in ) and the fully assembled composite promoter () having a nick between nucleotides \u221211 and \u221210 relative to the start of transcription. It can be seen from these data that neither partial promoter (lanes 3 and 4) is able to support transcription of the copy template, but that the composite promoter (lane 5) with this nick site is strongly transcribed. Surprisingly, comparison to the control reaction (lane 9) shows that the presence of a nick at this site (\u221211\/\u221210) actually enhances transcription. While not limiting the present invention to any particular mechanism, it is believed that the enhancement of transcription is a result of both suppressing the formation of the shorter abortive transcripts and by allowing greater accumulation of the full length product. This result is highly reproducible.","In , lanes 6, 7, and 8 compare transcription a similar set of partial and complete promoters in which the nick is shifted 3 residues closer to the transcription start site. Examination of lane 6 shows that the presence of 3 extra bases on the \u22128\u201ccut\u201d probe (compared to the \u221211 \u201ccut\u201d probe in lane 3) allow this partial promoter to initiate transcription. This indicates that the \u22128\/\u22127 site would be a poor choice for use in this embodiment of the present invention.","This experiment demonstrates the process for determining the suitable placement of a nick within a promoter assembly to achieve the desired result. Similar tests can easily be designed for testing other nicks within the bacteriophage T7 promoter tested in this Example, or for testing suitable nick placement in any desired phage, prokaryotic or eukaryotic promoter.","The Examples described above indicate that a small oligonucleotide can be used to complete assembly of a composite T7 promoter, thereby enabling transcription from that promoter. Earlier Examples demonstrate that the invasive cleavage reaction can be used release specific small oligonucleotide products from longer probe oligonucleotides. In this Example, it is demonstrated that these two observations can be combined, and that the products of the invasive cleavage reaction can be used to complete a promoter and enable subsequent transcription. The schematic representations of the composite promoters tested in this Example are shown in FIG. .","Two invasive cleavage reactions were set up, one without (rxn. 1) and one with (rxn. 2) input target DNA. The reactions (1 and 2) comprised 10 mM MOPS (pH 7.5), 0.05% Tween-20, 0.05% NP-40 and 20 pmoles probe oligo 073-067-01 (SEQ ID NO:132) and 10 pmoles INVADER oligo 073-073-02 (SEQ ID NO:134) in a volume of 14 \u03bcl. Reaction 2 also included 100 fmoles M13mp18 ssDNA. The samples were placed at 60\u00b0 C. and 6 \u03bcl of a solution containing 20 ng of Mja FEN-1 and 40 mM MgCl were added to each sample to start the reactions. The samples were incubated at 60\u00b0 C. for 30 minutes and stopped by the addition of 3 \u03bcl of 2.5M NaOAc, 83mM NaEDTA (pH 8.0). Each sample was transferred to a 1.5 ml microcentrifuge tube and then the DNAs were precipitated by the addition of 60 \u03bcl of chilled 100% ethanol, and were stored at \u221220\u00b0 C. for 20 minutes. The pellets were collected by microcentrifugation, washed once with 80% ethanol to remove excess salt, then dried under vacuum. The product of this invasive cleavage reaction is a 12 nt oligonucleotide having the sequence: 5\u2032-CGAAATTAATAC-3\u2032 (SEQ ID NO:128), termed the \u221212 cut probe (same sequence as oligo 073-073-03).","For transcription, the dried samples were each dissolved in 4 \u03bcl of a solution containing 1 pmole copy template oligo 150 and 2 pmoles \u221211 partial promoter oligo 073-073-012 (SEQ ID NO:13 1). Control samples 3 and 4 each contained 1 pmole of the copy template oligo 150; sample 3 also contained 1 pmole probe oligo 073-067-01 (SEQ ID NO:132) and 2 pmoles \u221211 partial promoter oligo 073-073-012 (see structure 88A); sample 4 contained 1 pmole \u221212 \u201ccut\u201d probe oligo 073-073-03 (SEQ ID NO:128) and 2 pmoles \u221211 partial promoter oligo 073-073-012 (see structure 88B). These are the structures that would be expected to exist in the transcription reactions from the two invasive cleavage reactions described above.","The transcription reactions were initiated, incubated, terminated and the products were resolved and imaged as described in Ex. 40. The resulting image is shown in the right half of  (lanes 6-9). Samples 3 and 4 appear in lanes 6 and 7, respectively, and the reactions 1 and 2 from the invasive cleavage reaction products (indicated by the use of the lower case \u201ci\u201d), appear in lanes 8 and 9, respectively. The number of the Figure showing the schematic representation of the expected promoter structure in each reaction is indicated above each lane, and the placement of the nick is also indicated. The uppercase letters indicate which structure in the particular Figure to examine for each reaction. The lowercase \u201ci\u201d above lanes 8 and 9 indicate that these transcriptions were derived from actual invasive cleavage reactions. These products are compared to the RNA produced in the control reaction in lane 5, the procedure for which is described in Ex. 44. The RNA created by successful transcription appears in the upper third of the panel (indicated by \u201cRNA\u201d).","The reaction shown in lane 6 shows no transcription. This demonstrates that a nick between nucleotides \u221212 and \u221211 in the on-template strand of the T7 promoter eliminates transcription if the promoter is assembled from uncut probe such as the 3\u2032 end of the probe forms a branch within the promoter sequence. This is in contrast tot he results seen with the \u221211\/\u221210 nick examined below. Further, the transcript apparent in lane 7 shows that an unbranched promoter with a nick at the same site (\u221212\/\u221211) produces the correct RNA, with few abortive initiation products (see lanes 2 and 5 of , described in Ex. 44). The reactions in lanes 8 and 9 demonstrate that the same effect is observed when the invasive cleavage reaction is the sole source of the upstream piece (\u221212 cut probe) of the T7 promoter. It is worthy of note that the promoter that is transcribed in lane 8 is made complete by the presence of 1 pmole of a synthetic \u201ccut\u201d probe oligo, without any uncut probe in the mixture, while the promoter that is transcribed in lane 9 is completed by the product of an invasive cleavage reaction that had only 100 fmole of target DNA in it. This reaction also included the residual uncut probe (up to approx. 10 pmoles), which may compete for binding at the same site. Nonetheless, the transcriptions from the invasive cleavage reaction products are only slightly reduced in efficiency, and are just as free of background as is the \u201cno target\u201d sample (lane 8). This Example clearly demonstrates that the cleavage products from the invasive cleavage reaction can be used in combination with a partial promoter oligo to promote the production of RNA, without background transcription generated by the presence of the uncut probe. This RNA product is clearly dependent on the presence of the target material in the invasive cleavage reaction.","The previous Example demonstrated that placement of a nick in the non-template strand of a bacteriophage T7 promoter between the \u221212 and \u221211 nucleotides, relative to the transcription start site, prevents transcription of the branched promoter while allowing transcription when the composite promoter is assembled using the cut probe. When the nick is placed in other locations in the T7 promoter, transcription may be initiated from either promoter, although it is usually less efficient from the branched promoter. This Example demonstrates that the addition of a 5\u2032 tail that can base pair to the uncut probe () to the downstream partial promoter piece effectively blocks transcription from that promoter, but does not prevent transcription when a cut probe completes the promoter (FIG. B).","Two invasive cleavage reactions were set up, one without (rxn. 7) and one with (rxn. 8) input target DNA. The reactions (7 and 8) comprised 10 mM MOPS (pH 7.5), 0.05% Tween-20, 0.05% NP-40 and 20 pmoles probe oligo 073-067-01 (SEQ ID NO:132)and 10 pmoles INVADER oligo 073-067-02 (SEQ ID NO:133) in a volume of 14 \u03bcl. Reaction 8 also included 100 fmoles M13mp18 ssDNA. The samples were placed at 60\u00b0 C. and 6 \u03bcl of a solution containing 20 ng of Mja FEN-1 and 40mM MgCl were added to each sample to start the reactions. The samples were incubated at 60\u00b0 C. for 30 minutes and then stopped by the addition of 3 \u03bcl of 2.5M NaOAc, 83 mM NaEDTA (pH.8.0). Each sample was transferred to a 1.5 ml microcentrifuge tube and the DNAs were precipitated, washed and dried as described in Ex. 42. The product of this invasive cleavage reaction is 13 nt oligonucleotide sequence, 5\u2032-CGAAATTAATACG-3\u2032 (SEQ ID NO:127), termed the \u221211 cut probe (same sequence as oligo 073-061-01 which is referred to as the \u221211 \u201ccut\u201d probe to indicate it was not generated in an invasive cleavage reaction).","In the transcription reactions, all of the DNAs were dissolved in 4 \u03bcl of RNase-free dHO. Sample 1 had no added DNA, samples 2-8 contained 1 pmole of the copy template oligo 150 (SEQ ID NO:123). In addition, sample 3 contained 1 pmole of \u221211 \u201ccut\u201d probe oligo 073-061-01 (SEQ ID NO:127) and 2 pmoles of \u221210 partial promoter oligo 073-061-02 (SEQ ID NO:130), sample 4 contained 1 pmole of probe oligo 073-067-01 and 2 pmoles of \u221210 partial promoter oligo 073-061-02. Control sample  contained 1 pmole of probe oligo 073-067-01 and 2 pmoles of partial promoter w\/5\u2032 tail oligo 073-074 (5\u2032-TACTGACTCACTATAGGGTCTTCTATGGAG GTC-3\u2032 (SEQ ID NO:146) (see structure in ) and sample 6 contained 1 pmole of \u221211 \u201ccut\u201d probe oligo 073-061-01 and 2 pmoles of partial promoter w\/5\u2032 tail oligo 073-074 (see structure in FIG. B). These are the structures (i.e., 90A and 90B) that would be expected to exist in the transcription reactions from the two invasive cleavage reactions described above.","The dried samples 7 and 8 from the invasive cleavage (above) were each dissolved in 4 \u03bcl of dHO containing 1 pmole copy template oligo 150 and 2 pmoles partial promoter w\/5\u2032 tail oligo 073-074. The transcription reactions were initiated, incubated, terminated and the reaction products were resolved and imaged as described in Ex. 40. The resulting image is shown in FIG. .","In  the lane numbers correspond to the sample numbers; the number of the Figure showing the schematic representation of the expected promoter structure in each reaction is indicated above each lane (\u201c88\u201d and \u201c90\u201d), and the placement of the nick is also indicated (\u201c\u221211\/\u221210\u201d). The upper-case letters indicate which structure in the particular Figure to examine for each reaction. The lower case \u201ci\u201d above lanes 7 and 8 indicates that these transcriptions were derived from actual invasive cleavage reactions. The RNA created by successful transcription appears in the upper third of the panel, as indicated (\u201cRNA\u201d).","The control reactions in lanes 1 and 2, having either no DNA or having the only the copy template, produced no RNA as expected. The product in lane 4 demonstrates that the branched T7 promoter with a nick in the non-template strand between nucleotides \u221211 and \u221210 can support transcription, albeit not as efficiently as the un-branched promoter with the nick at the same site (lane 3). Examination of lane 5 shows that the use of a partial promoter oligonucleotide with a short 5\u2032 tail that can basepair to the uncut probe as depicted in , effectively suppresses this transcription but allows transcription when the probe does not have a 3\u2032 tail (lane 6; schematic FIG. B). The reactions in lanes 7 and 8 demonstrate that the same effect as observed when the invasive cleavage reaction is the sole source of the upstream piece (\u221211 cut probe, SEQ ID NO:127) of the T7 promoter. It is worthy of note that the promoter that is transcribed in sample 6 is made complete by the presence of 1 pmole of a synthetic \u201ccut probe\u201d, without any uncut probe in the mixture, while the promoter that is transcribed in sample 8 is completed by the product of an invasive cleavage reaction that had only 100 fmole of target DNA in it. This reaction also included the residual uncut probe (up to approximately 19 pmoles), which may compete for binding at the same site. Nonetheless, the transcriptions from the invasive cleavage reaction products are just as strong and just as free of background in the \u201cno target\u201d samples.","This Example clearly demonstrates that the cleavage products from the invasive cleavage reaction can be used in combination with a partial promoter oligonucleotide having a 5\u2032 tail to promote the production of RNA, without background transcription generated by the uncut probe. This RNA product is clearly dependent on the presence of the target material in the invasive cleavage reaction.","As demonstrated in the Examples above, transcription cannot occur from the T7 promoter unless a complete promoter region is present. In the above Examples, a complete promoter containing a nick in one strand was created by annealing a cut probe generated from an invasive cleavage reaction to a copy template that was annealed to a partial promoter oligo. An alternative means of creating a complete promoter in a manner dependent upon detection of a target sequence in an invasive cleavage reaction is to anneal the cut probe to a copy template devoid of a partial promoter oligo. The 3\u2032-OH present at the end of the annealed cut probe is then extended by a DNA polymerase to create a complete and un-nicked promoter that is transcription-competent.","In this Example, the promoter was made complete through the use of primer extension, rather that by the co-hybridization of another oligonucleotide. The reaction steps are diagrammed schematically in FIG. . Two invasive cleavage reactions were set up, one without (rxn. 1) and one with (rxn. 2) input target DNA. The reactions (1 and 2) comprised 10 mM MOPS (pH 7.5), 0.05% Tween-20, 0.05% NP-40 and 20 pmoles probe oligo 073-067-01 (SEQ ID NO:132) and 10 pmoles INVADER oligo 073-073-02 (SEQ ID NO:134) in a volume of 14 \u03bcl. Reaction 2 also included 100 fmoles M13mp18 ssDNA. The samples were placed at 60\u00b0 C. and 6 \u03bcl of a solution containing 20 ng of Mja FEN-1 and 40 mM MgCl were added to each sample to start the reactions. The samples were incubated at 60\u00b0 C. for 30 minutes and stopped by the addition of 3 \u03bcl of 2.5M NaOAc, 83 mM NaEDTA (pH 8.0). Each sample was transferred to a 1.5 ml microcentrifuge tube and then the DNAs were precipitated, washed and dried as described in Ex. 42. The product of this invasive cleavage reaction is the 12 nt oligonucleotide sequence: 5\u2032-CGAAATTAATAC-3\u2032 (SEQ ID NO:128), termed the \u221212 cut probe (same sequence as oligo 073-073-03 which is referred to as the \u221212 \u201ccut\u201d probe to indicate it was not generated in an invasive cleavage reaction).","To allow extension of these products using a template-dependent DNA polymerase, a 20 \u03bcl solution containing 20mM Tris-HCl (pH 8.5), 1.5 mM MgCl, 50 mM KCl, 0.05% Tween-20, 0.05% NP-40, 25 \u03bcM each dNTP, 0.25 units Taq DNA polymerase (Boehringer) and 2 \u03bcM copy template oligo 150 (SEQ ID NO:123) was added to each of the dried cleavage samples. The samples were incubated at 30\u00b0 C. for 1 hr. The primer extension reactions were stopped by the addition of 3 \u03bcl of 2.5M NaOAc with 83 mM NaEDTA (pH 8.0)\/sample. Each sample was transferred to a 1.5 ml microcentrifuge tube and the DNAs were precipitated, washed and dried as described in Ex. 42.","Samples 1 and 2 were then dissolved in 4 \u03bcl RNase-free dHO. Samples 3, 4 and 5 are control reactions: sample 3 was 4 \u03bcl of RNase-free dHO without added DNA, sample 4 contained 1 pmole of the copy template oligo 150 (SEQ ID NO:123) in 4 \u03bcl of RNase-free dHO, and sample 5 contained 1 pmole of the same copy template and 1 pmole of the complete promoter oligo 151 (SEQ ID NO:124) in RNase-free dHO.","Transcription reactions were performed using the MEGAshortscript\u2122 system, in accordance with the manufacturer's instructions, but with the addition of a fluorescein labeled ribonucleotide. To each sample, 6 \u03bcl of a solution containing 1 \u03bcl of 10\u00d7 Transcription Buffer, 7.5 mM each rNTP, 0.125 mM fluorescein-12-UTP (Boehringer) and 1 \u03bcl T7 MEGAshortscript\u2122 Enzyme Mix was added. The samples were incubated at 37\u00b0 C. for 1 hour. One \u03bcl of RNase-free DNase 1 (2 U\/\u03bcl) was added to each sample and they were incubated an additional 15 minutes at 37\u00b0 C. The reactions were stopped by the addition of 10 \u03bcl of a solution of 95% formamide, 5 mM NaEDTA, with loading dyes. All samples were heated to 95\u00b0 C. for 2 minutes and four \u03bcl of each sample were resolved by electrophoresis through a 20% denaturing acrylamide gel (19:1 cross-linked) with 7 M urea, in a buffer containing 45 mM Tris-Borate (pH 8.3), 1.4 mM EDTA. The results were imaged using the Molecular Dynamics Fluoroimager 595, with excitation at 488 nm and, emission detected at 530 nm.","The resulting image is shown in lanes 1 through 5 of ; the lane numbers correspond to the sample numbers. The Figure numbers corresponding to the schematic representations of the promoters transcribed in each reaction as indicated above the lanes. The RNA product from successful transcription appears in the upper third of the panel, as indicated (\u201cRNA\u201d). Unincorporated labeled nucleotide appears as a dense signal near the bottom (\u201cNTPs\u201d). Short transcription products caused by aborted initiation events (Milligan and Uhlenbeck, Methods Enzymol., 180:51 [1989]) appear as bands just above the free nucleotide in the lanes showing active transcription (i.e., lanes 2 and 5).","It can clearly be seen from the data in lanes 1 and 2 that the transcription is dependent on the presence of the target material in the invasive cleavage reaction. It is shown elsewhere (see lane 3, ) that the product of the cleavage reaction is not in itself sufficient to allow transcription from the copy template. Thus, the action of the DNA polymerase in extending the hybridized cut probe across the promoter is a necessary step in enabling the transcription in this embodiment. These data clearly demonstrate that both template-dependent extension by DNA polymerase, and extension followed by transcription are suitable methods of visualizing the products of the invasive cleavage assay. As discussed in the Description of the Invention, the products of thermal breakdown that possess 3\u2032 terminal phosphates would not be extended, and would thus be precluded from contributing to background transcription.","When choosing a structure-specific nuclease for use in a sequential invasive cleavage reaction it is preferable that the enzyme have little ability to cleave a probe 1) in the absence of an upstream oligonucleotide, and 2) in the absence of overlap between the upstream oligonucleotide and the downstream labeled probe oligonucleotide. -depicts the several structures that can be used to examine the activity of an enzyme that is confronted with each of these types of structures. The structure a () shows the alignment of a probe oligonucleotide with a target site on bacteriophage M13 DNA (M13 sequences shown in  are provided in SEQ ID NO:163) in the absence of an upstream oligonucleotide. Structure b () is provided with an upstream oligonucleotide that does not contain a region of overlap with the labeled probe (the label is indicated by the star). In structures c, d and e (-) the upstream oligonucleotides have overlaps of 1, 3 or 5 nucleotides, respectively, with the downstream probe oligonucleotide and each of these structures represents a suitable invasive cleavage structure. The enzyme Pfu FEN-1 was tested for activity on each of these structures and all reactions were performed in duplicate.","Each reaction comprised 1 \u03bcM 5\u2032 TET labeled probe oligonucleotide 89-15-1 (SEQ ID NO:152), 50 nM upstream oligonucleotide (either oligo 81-69-2 [SEQ ID NO:153], oligo 81-69-3 [SEQ ID NO:154], oligo 81-69-4 [SEQ ID NO:155], oligo 81-69-5 [SEQ ID NO:156], or no upstream oligonucleotide), 1 fmol M13 target DNA, 10 mg\/ml tRNA and 10 ng of Pfu FEN-1 in 10 \u03bcl of 10 mM MOPS (pH 7.5), 7.5 mM MgClwith 0.05% each of Tween 20 and Nonidet P-40.","All of the components except the enzyme and the MgClwere assembled in a final volume of 8 \u03bcl and were overlaid with 10 \u03bcl of Chill-Out\u2122 liquid wax. The samples were heated to the reaction temperature of 69\u00b0 C. The reactions were started by the addition of the Pfu FEN-1 and MgCl, in a 2 \u03bcl volume. After incubation at 69\u00b0 C. for 30 minutes, the reactions were stopped with 10 \u03bcl of 95% formamide, 10 mM EDTA, 0.02% methyl violet. Samples were heated to 90\u00b0 C. for 1 min immediately before electrophoresis through a 20% denaturing acrylamide gel (19:1 cross-linked), with 7 M urea, in a buffer of 45 mM Tris-Borate, pH 8.3, 1.4 mM EDTA. Gels were then analyzed with a FMBIO-100 Hitachi FMBIO fluorescence imager. The resulting image is displayed in FIG. .","In , lanes labeled \u201ca\u201d contain the products generated from reactions conducted without an upstream oligonucleotide (structure a), lanes labeled \u201cb\u201d contain an upstream oligonucleotide that does not invade the probe\/target duplex (structure b). Lanes labeled \u201cc\u201d, \u201cd\u201d and \u201ce\u201d contain the products generated from reactions conducted using an upstream oligonucleotide that invades the probe\/target duplex by 1, 3 or 5 bases, respectively. The size (in nucleotides) of the uncleaved probe and the cleavage products is indicated to the left of the image in FIG. .","As shown in , cleavage of the probe was not detectable when structures a and b were utilized. In contrast, cleavage products were generated when invasive cleavage structures were utilized (structures c-e). These data show that the Pfu FEN-1 enzyme requires an overlapping upstream oligonucleotide for specific cleavage of the probe.","Any enzyme may be examined for its suitability for use in a sequential invasive cleavage reaction by examining the ability of the test enzyme to cleave structures a-e (it is understood by those in the art that the specific oligonucleotide sequences shown in -need not be employed in the test reactions; these structures are merely illustrative of suitable test structures). Desirable enzymes display little or no cleavage of structures a and b and display specific cleavage of structures c-e (i.e., they generate cleavage products of the size expected from the degree of overlap between the two oligonucleotides employed to form the invasive cleavage structure).","As discussed in the Description of The Invention above, the detection sensitivity of the invasive cleavage reaction can be increased by the performing a second round of invasive cleavage using the products of the first reaction to complete the cleavage structure in the second reaction (shown schematically in FIG. ). In this Example, the use of a probe that, when cleaved in a first invasive cleavage reaction, forms an integrated INVADER oligo and target molecule for use in a second invasive cleavage reaction, is illustrated (shown schematically in FIG. ).","A first probe was designed to contain some internal complementarity so that when cleaved in a first invasive cleavage reaction the product (\u201cCut Probe 1\u201d) could form a target strand comprising an integral INVADER oligonucleotide, as depicted in  second probe was provided in the reaction that would be cleaved at the intended site when hybridized to the newly formed target\/INVADER (FIG. ). To demonstrate the gain in signal due to the performance of sequential invasive cleavages, a standard invasive cleavage assay, as described above, was performed in parallel.","All reactions were performed in duplicate. Each standard (i.e., non-sequential) invasive cleavage reaction comprised 1 \u03bcM 5\u2032 fluorescein-labeled probe oligo 073-182 (5\u2032 F1-AGAAAGGAAGGGAAGAAAGCGAA-3\u2032; SEQ ID NO:157), 10 nM upstream oligo 81-69-4 (5\u2032-CTTGACGGGGAAAGCCGGCGAACGTGGCGA-3\u2032; SEQ ID NO:155), 10 to 100 attomoles of M13 target DNA, 10 mg\/ml tRNA and 10 ng of Pfu FEN-1 in 10 \u03bcl of 10 mM MOPS (pH 7.5), 8 mM MgClwith 0.05% each of Tween 20 and Nonidet P-40. All of the components except the enzyme and the MgClwere assembled in a volume of 7 \u03bcl and were overlaid with 10 \u03bcl of Chill-Out\u2122 liquid wax. The samples were heated to the reaction temperature of 62\u00b0 C. The reactions were started by the addition of the Pfu FEN-1 and MgCl, in a 2 \u03bcl volume. After incubation at 62\u00b0 C. for 30 minutes, the reactions were stopped with 10 \u03bcl of 95% formamide, 10 mM EDTA, 0.02% methyl violet.","Each sequential invasive cleavage reaction comprised 1 \u03bcM 5\u2032 fluorescein-labeled oligonucleotide 073-191 (the first probe or \u201cProbe 1\u201d, 5\u00b0 F1-TGGAGGTCAAAACATCG ATAAGTCGAAGAAAGGAAGGGAAGAAAT-3\u2032; SEQ ID NO:158), 10 nM upstream oligonucleotide 81-69-4 (5\u2032-CTTGACGGGGAAA GCCGGCGAACGTGGCGA-3\u2032; SEQ ID NO:155), 1 \u03bcM of 5\u2032 fluorescein labeled oligonucleotide 106-32 (the second probe or \u201cProbe 2\u201d, 5\u2032 F1-TGTTTTGACCT CCA-3\u2032; SEQ ID NO:159), 1 to 100 amol of M13 target DNA, 10 mg\/ml tRNA and 10 ng ofPfu FEN-1 in 10 \u03bcl of 10 mM MOPS (pH 7.5), 8 mM MgClwith 0.05% each of Tween 20 and Nonidet P-40. All of the components except the enzyme and the MgClwere assembled in a volume of 8 \u03bcl and were overlaid with 10 \u03bcl of Chill-Out\u2122 liquid wax. The samples were heated to the reaction temperature of 62\u00b0 C. (this temperature is the optimum temperature for annealing of Probe 1 to the first target). The reactions were started by the addition of Pfu FEN-1 and MgCl, in a 2 \u03bcl volume. After incubation at 62\u00b0 C. for 15 minutes, the temperature was lowered to 58\u00b0 C. (this temperature is the optimum temperature for annealing of Probe 2 to the second target) and the samples were incubated for another 15 min. Reactions were stopped by the addition of 10 \u03bcl of 95% formamide, 20 mM EDTA, 0.02% methyl violet.","Samples from both the standard and the sequential invasive cleavage reactions were heated to 90\u00b0 C. for 1 min immediately before electrophoresis through a 20% denaturing acrylamide gel (19:1 cross-linked), with 7 M urea, in a buffer of 45 mM Tris-Borate, pH 8.3, 1.4 mM EDTA. The gel was then analyzed with a Molecular Dynamics Fluorlmager 595. The resulting image is displayed in . A graph showing measure of fluorescence intensity for each of the product bands is shown in ","In , lanes 1-5 contain the products generated in standard invasive cleavage reactions that contained either no target (lane 1), 10 amol of target (lanes 2 and 3) or with 100 amol of target (lanes 4 and 5). The uncleaved probe is seen as a dark band in each lane about half way down the panel and the cleavage products appear as a smaller black band near the bottom of the panel, the position of the cleavage product is indicated by an arrow head to the left of . The gray ladder of bands seen in lanes 1-5 is due to the thermal degradation of the probe as discussed above and is not related to the presence or absence of the target DNA. The remaining lanes display products generated in sequential invasive cleavage reactions that contained 1 amol of target (lanes 6 and 7), 10 amol of target (lanes 8 and 9) and 100 amol of target (lanes 10 and 11). The uncleaved first probe (Probe 1; labeled \u201c1 uncut\u201d) is seen near the top of the panel, while the cleaved first probe is indicated as \u201c1: cut\u201d . Similarly, the uncleaved and cleaved second probe are indicated as \u201c2: uncut\u201d and 2: cut,\u201d respectively.","The graph shown in compares the amount of product generated from the standard reaction (\u201cSeries 1\u201d) to the amount of product generated from the second step of the sequential reaction (\u201cSeries 2\u201d). The level of background fluorescence measured from a reaction that lacked target DNA was subtracted from each measurement. It can be seen from the table located below the graph that the signal from the standard invasive cleavage assay that contained 100 attomoles of target DNA was nearly identical to the signal from the sequential invasive cleavage assay in which 1 attomole of target was present, indicating that the inclusion of a second cleavage structure increases the sensitivity of the assay 100 to 200-fold. This boost in signal allows easy detection of target nucleic acids at the sub attomole level using the sequential invasive cleavage assay, while the standard assay, when performed using this enzyme for only 30 minutes, does not generate detectable product in the presence of 10 attomoles of target.","When the amount of target was decreased by 10 or 100 fold in the sequential invasive cleavage assay, the intensity of the signal was decreased by the same proportion. This indicates that the quantitative capability of the invasive cleavage assay is retained even when reactions are performed in series, thus providing a nucleic acid detection method that is both sensitive and quantitative.","While in this Example, the two probes used had different optimal hybridization temperatures (i.e., the temperature empiracally determined to give the greatest turnover rate in the given reaction conditions), the probes may also be selected (i.e., designed) to have the same optimal hybridization temperature so that a temperature shift during incubation is not necessary.","As discussed in the Description of the Invention, the serial nature of the multiple invasive cleavage events that occur in the sequential invasive cleavage reaction, in contrast to the reciprocating nature of the polymerase chain reaction and similar doubling assays, means that the sequential invasive cleavage reaction is not subject to contamination by the products of like reactions because the products of the first cleavage reaction do not participate in the generation of new signal in the second cleavage reaction. If a large amount of a completed reaction were to be added to a newly assembled reaction, the background that would be produced would come from the amount of target that was also carried in, combined with the amount of already-cleaved probe that was carried in. In this Example, it is demonstrated that a very large portion of a primary reaction must be introduced into the secondary reaction to create significant signal.","A first or primary sequential invasive cleavage reaction was performed as described above using 100 amol of target DNA. A second set of 5 reactions were assembled as described in Ex. 46 with the exception that portions of the first reaction were introduced and no additional target DNA was included. These secondary reactions were initiated and incubated as described above, and included 0, 0.01, 0.1, 1, or 10% of the first reaction material. A control reaction including 100 amol of target was included in the second set also. The reactions were stopped, resolved by electrophoresis and visualized as described above, and the resulting image is displayed in FIG. . The primary probe, uncut second probe and the cut 2nd probe are indicated on the left as \u201c1: cut\u201d, 2: uncut \u201cand 2: cut\u201d, respectively.","In , lane 1 shows the results of the first reaction with the accumulated product at the bottom of the panel, and lane 2 show a 1:10 dilution of the same reaction, to demonstrate the level of signal that could be expected from that level of contamination, without further amplification. Lanes 3 through 7 show the results of the secondary cleavage reactions that contained 0, 10, 1, 0.1 or 0.01% of the first reaction material added as contaminant, respectively and lane 8 shows a control reaction that had 100 amol of target DNA added to verify the activity of the system in the secondary reaction. The signal level in lane 4 is as would be expected when 10% of the pre-cleaved material is transferred (as in lane 2) and 10% of the transferred target material from the lane 1 reaction is allowed to further amplify. At all levels of further dilution the signal is not readily distinguished from background. These data demonstrate that while a large-scale transfer from one reaction to another may be detectable, cross contamination by the minute quantities that would be expected from aerosol or from equipment contamination would not be easily mistaken for a false positive result. These data also demonstrate that when the products of one reaction are deliberately carried over into a fresh sample, these products do not participate in the new reaction, and thus do not affect the level of target-dependent signal that may be generated in that reaction.","The previous Example demonstrates the ability of the invasive cleavage reaction to detect minute quantities of viral DNA in the presence of human genomic DNA. In this Example, the probe and INVADER oligonucleotides were designed to target the 3104-3061 region of the major immediate early gene of human cytomegalovirus (HCMV) as shown in FIG. . In , the INVADER oligo (89-44; SEQ ID NO:160) and the fluorescein (Fl)-labeled probe oligo (89-76; SEQ ID NO:161) are shown annealed along a region of the HCMV genome corresponding to nucleotides 3057-3110 of the viral DNA (SEQ ID NO:162). The probe used in this Example is a polypyrimidine probe and as shown herein the use of a poly-pyrimidine probe reduces background signal generated by the thermal breakage of probe oligos.","The genomic viral DNA was purchased from Advanced Biotechnologies, Incorporated (Columbia, Md.). The DNA was estimated (but not certified) by personnel at Advanced Biotechnologies to be at a concentration of 170 amol (1\u00d710copies) per microliter. The reactions were performed in quadruplicate. Each reaction comprised 1 \u03bcM 5\u2032 fluorescein labeled probe oligonucleotide 89-76 (SEQ ID NO:161), 100 nM INVADER oligonucleotide 89-44 (SEQ ID NO:160), 1 ng\/ml human genomic DNA, and one of five concentrations of target HCMV DNA in the amounts indicated above each lane in , and 10 ng of Pfu FEN-1 in 10 \u03bcl of 10 mM MOPS (pH 7.5), 6 mM MgClwith 0.05% each of Tween 20 and Nonidet P-40. All of the components except the labeled probe, enzyme and MgClwere assembled in a final volume of 7 \u03bcl and were overlaid with 10 \u03bcl of Chill-Out\u2122 liquid wax. The samples were heated to 95\u00b0 C. for 5 min, then reduced to 62\u00b0 C. The reactions were started by the addition of probe, Pfu FEN-1 and MgCl, in a 3 \u03bcl volume. After incubation at 62\u00b0 C. for 60 minutes, the reactions were stopped with 10 \u03bcl of 95% formamide, 10 mM EDTA, 0.02% methyl violet. Samples were heated to 90\u00b0 C. for 1 min immediately before electrophoresis through a 20% acrylamide gel (19:1 cross-linked), with 7 M urea, in a buffer of 45 mM Tris-Borate, pH 8.3, 1.4 mM EDTA. Gels were then analyzed with a Molecular Dynamics FluorImager 595.","The resulting image is displayed in FIG. . The replicate reactions were run in groups of four lanes with the target HCMV DNA content of the reactions indicated above each set of lanes (0-170 amol). The uncleaved probe is seen in the upper third of the panel (\u201cUncut 89-76\u201d) while the cleavage products are seen in the lower two-thirds of the panel (\u201cCut 89-76\u201d). It can be seen that the intensity of the accumulated cleavage product is proportional to the amount of the target DNA in the reaction. Furthermore, it can be clearly seen in reactions that did not contain target DNA (\u201cno target\u201d) that the probe is not cleaved, even in a background of human genomic DNA. While 10 ng of human genomic DNA was included in each of the reactions shown in , inclusion of genomic DNA up to 200 ng has slight impact on the amount of product accumulated. The data did not suggest that 200 ng per 10 \u03bcl of reaction mixture represented the maximum amount of genomic DNA that could be tolerated without a significant reduction in signal accumulation. For reference, this amount of DNA exceeds what might be found in 0.2 ml of urine (a commonly tested amount for HCMV in neonates) and is equivalent to the amount that would be found in about 5 \u03bcl of whole blood.","These results demonstrate that the standard (i.e., non-sequential) invasive cleavage reaction is a sensitive, specific and reproducible means of detecting viral DNA. It can also be seen from these data that the use of a poly-pyrimidine probe reduces the background from thermal breakage of the probe, as discussed in Example 22. Detection of 1.7 amol of target is roughly equivalent to detection of 10copies of the virus. This is equivalent to the number of viral genomes that might be found in 0.2 mls of urine from a congenitally infected neonate (10to 10genome equivalents per 0.2 mls; Stagno et al., J. Infect. Dis., 132:568 [1975]). Use of the sequential invasive cleavage assay would permit detection of even fewer viral DNA molecules, facilitating detection in blood (10to 10viral particles per ml; Pector et al., J. Clin. Microbiol., 30:2359 [1992]), which carries a much larger amount of heterologous DNA.","From the above it is clear that the invention provides reagents and methods to permit the detection and characterization of nucleic acid sequences and variations in nucleic acid sequences. The INVADER-directed cleavage reaction and the sequential INVADER-directed cleavage reaction of the present invention provide ideal direct detection methods that combine the advantages of the direct detection assays (e.g., easy quantification and minimal risk of carry-over contamination) with the specificity provided by a dual or tri oligonucleotide hybridization assay.","As indicated in the Description of the Invention, the use of sequential invasive cleavage reactions can present the problem of residual uncut first, or primary, probe interacting with the secondary target, and either competing with the cut probe for binding, or creating background through low level cleavage of the resulting structure. This is shown diagramatically in . In , the reaction depicted makes use of the cleavage product from the first cleavage structure to form an INVADER oligonucleotide for a second cleavage reaction. The structure formed between the secondary target, the secondary probe and the uncut primary probe is depicted in , as the right hand structure shown in step . This structure is recognized and cleaved by the 5\u2032 nucleases, albeit very inefficiently (i.e., at less than about 1% in most reaction conditions). Nonetheless, the resulting product is indistinguishable from the specific product, and thus may lead to a false positive result. The same effect can occur when the cleaved primary probe creates and integrated INVADER\/target (IT) molecule, as described in Example 46; the formation of the undesirable complex is depicted schematically in , as the right hand structure shown in step ","The improvements provided by the inclusion of ARRESTOR oligonucleotides of various compositions in each of these types of sequential INVADER assays are demonstrated in the following Examples. These ARRESTOR oligonucleotides are configured to bind the residual uncut probe from the first cleavage reaction in the series, thereby increasing the efficacy of and reducing the non-specific background in the subsequent reaction(s).","In this Example, the effect of including an ARRESTOR oligonucleotide on the generation of signal using the IT probe system depicted in  is demonstrated. The ARRESTOR oligonucleotide hybridizes to the primary probe, mainly in the portion that recognizes the target nucleic acid during the first cleavage reaction. In addition to examining the effects of adding an ARRESTOR oligonucleotide, the effects of using ARRESTOR oligonucleotides that extended in complementarity different distances into the region of the primary probe that composes the secondary IT structure were also investigated. These effects were compared in reactions that included the target DNA over a range of concentrations, or that lacked target DNA, in order to demonstrate the level of nonspecific (i.e., not related to target nucleic acid) background in each set of reaction conditions.","The target DNA for these reactions was a fragment that comprised the full length of the hepatitis B genome from strain of serotype adw. This material was created using the polymerase chain reaction from plasmid pAM6 (ATCC #45020D). The PCRs were conducted using a vector-based forward primer, oligo # 156-022-001 (5\u2032-ggcgaccacacccgtcctgt-3\u2032; SEQ ID NO:168) and a reverse primer, oligo #156-022-02 (5\u2032-ccacgatgcgtccggcgtag-3\u2032; SEQ ID NO:1 69) to amplify the full length of the HBV insert, an amplicon of about 3.2 kb. The cycling conditions included a denaturation of the plasmid at 95\u00b0 C. for 5 minutes, followed by 30 cycles of 95\u00b0 C., 30 seconds; 60\u00b0 C., 40 seconds; and 72\u00b0 C., 4 minutes. This was followed by a final extension at 72\u00b0 C. for 10 minutes. The resulting amplicon, termed pAM6 #2, was adjusted to 2 M NHOAc, and collected by precipitation wiht isopropanol. After drying in vacuo, DNA was dissolved in 10 mM Tris pH 0.0, 0.1 mM EDTA. The concentration was determined by ODmeasurement, and by INVADER assay with comparison to a standard of known concentration.","The INVADER reactions were conducted as follows. Five master mixes, termed \u201cA,\u201d \u201cB,\u201d \u201cC,\u201d \u201cD,\u201d and \u201cE,\u201d were assembled; all mixes contained 12.5 mM MOPS, pH 7.5, 500 fmoles primary INVADER oligo #218-55-05 (SEQ ID NO:171), 10 ng human genomic DNA (Novagen) and 30 ng AfuFENl enzyme, for every 8 \u03bcl of mix. Mix A contained no added HBV genomic amplicon DNA; mix B contained 600 molecules of HBV genomic amplicon DNA pAM6 #2; mix C contained 6,000 molecules pAM6 #2; mix D contained 60,000 molecules pAM6 #2; and mix E contained 600,000 molecules pAM6 #2. The mixes were aliquotted to the reaction tubes, 8 \u03bcl\/tube: mix A to tubes 1, 2, 11, 12, 21 and 22; mix B to tubes 3, 4, 13, 14, 23 and 24; mix C to tubes 5, 6, 15, 16, 25 and 26; mix D to tubes 7, 8, 17, 18, 27 and 28; and mix E to tubes 9, 10, 19, 20, 29 and 30. The samples were incubated at 95\u00b0 C. for 4 minutes to denature the HBV genomic amplicon DNA. The reactions were then cooled to 67\u00b0 C., and 2 \u03bcl of a mix containing 37.5 mM MgCland 2.5 pmoles 218-95-06 (SEQ ID NO:183) for every 2 \u03bcl, was added to each sample. The samples were incubated at 67\u00b0 C. for 60 minutes. Three secondary reaction master mixes were prepared, all mixes contained 10 pmoles of secondary probe oligonucleotide #228-48-04 (SEQ ID NO:173) for every 2 \u03bcl of mix. Mix 2A contained no additional oligonucleotide, mix 2B contained 5 pmoles \u201cARRESTOR\u201d oligo # 218-95-03 (SEQ ID NO:184) and mix 2C contained 5 pmoles of \u201cARRESTOR\u201d oligo # 218-95-01 (SEQ ID NO:174). After the 60 minute incubation at 67\u00b0 C. (the primary reaction described above), 2 \u03bcl of the secondary reaction mix was added to each sample: Mix 2A was added to samples #1-10; Mix 2B was added to samples #11-20; and Mix 2C was added to samples #21-30. The temperature was adjusted to 52\u00b0 C. and the samples were incubated for 30 minutes at 52\u00b0 C. The reactions were then stopped by the addition of 10 \u03bcl of a solution of 95% formamide, 5 mM EDTA and 0.02% crystal violet. All samples were heated to 95\u00b0 C. for 2 minutes, and 4 \u03bcl of each sample were resolved by electrophoresis through 20% denaturing acrylamide gel (19:1 cross-linked) with 7 M urea, in a buffer containing 45 mM Tris-Borate (pH8.3) and 1.4 mM EDTA. The results were imaged using the Molecular Dynamics Fluoroimager 595, excitation 488, emission 530. The resulting images arc shown in FIG. .","In , Panel A shows the results of the target titration when no ARRESTOR oligonucleotide was included in the secondary reaction; Panel B shows the results of the same target titration using an ARRESTOR oligonucleotide that extended 2 nt into the non-target complementary region of the primary probe; and Panel C shows the results of the same target titration using an ARRESTOR oligonucleotide that extended 4 nt into the non-target complementary region of the primary probe. The product of the secondary cleavage reaction is seen as a band near the bottom of each panel. The first two lanes of each panel (i.e., 1 and 2, 11 and 12, 21 and 22) lacked target DNA, and the signal the co-migrates with the product band represents the nonspecific background under each set of conditions.","It can be seen by visual inspection of these panels that the background signal is both reduced, and made more predictable, by the inclusion of either species of ARRESTOR oligonucleotides. In addition to reducing the background in the no-target control lanes, the background reduction in the reactions that had the more dilute amounts of target included is reduced, leading to a signal that is a more accurate reflection of the target contained within the reaction, thus improving the quantitative range of the multiple, sequential invasive cleavage reaction.","To quantify the impact of including the ARRESTOR oligonucleotide in the secondary cleavage reaction under these conditions, the average product band signal from the reactions having the largest amount of target (i.e., averages of the signals from lanes 9 and 10, lanes 19 and 20, and lanes 29 and 30), were compared to the averaged signal from the no-target contol lanes for each panel, determine the \u201cfold over background,\u201d the factor of signal amplification over background, under each set of conditions. For the reactions without the ARRESTOR oligonucleotide, Panel A, the fold over background was 5.3; for Panel B, the fold over background was 12.7; and for Panel C, the fold over background was 13.4, indicating that in this system inclusion of any ARRESTOR oligonucleotide at least doubled the specificity of the signal over the ARRESTOR oligonucleotide -less reactions, and that the ARRESTOR oligonucleotide that extended slightly farther into the non-target complementary region may be slightly more effective, at least in this embodiment of the system. This clearly shows the benefits of using an ARRESTOR oligonucleotide to enhance the specificity of these reactions, an advantage that is of particular benefit at low levels of target nucleic acid.","It was demonstrated in Example 36, that increasing the concentration of the probe in the invasive cleavage reaction could dramatically increase the amount of signal generated for a given amount of target DNA. While not intending to limit the explanation to any specific mechanism, this is believed to be caused by the fact that increased concentration of probe increases the rate at which the cleaved probe is supplanted by an uncleaved copy, thereby increasing the apparent turnover rate of the cleavage reaction. Unfortunately, this effect could not heretofore be applied in the primary cleavage reaction of a multiple sequential INVADER assay because the residual uncleaved primary probe can hybridize to the secondary target, in competition with the cleaved molecules, thereby reducing the efficacy of the secondary reaction. Elevated concentrations of primary probe exacerbate this problem. Further, the resulting complexes, as described above, can be cleaved at a low level, contributing to background. Therefore, increasing the primary probe can have the double negative effect of both slowing the secondary reaction and increasing the level of this form of non target-specific background. The use of an ARRESTOR oligonucleotide to sequester or neutralize the residual primary probe allows this concentration-enhancing effect to be applied to these sequential reactions.","To demonstrate this effect, two sets of reactions were conducted. In the first set of reactions, the reactions were conducted using a range of primary probe concentrations, but no ARRESTOR oligonucleotide was supplied in the secondary reaction. In the second set of reactions, the same probe concentrations were used, but an ARRESTOR oligonucleotide was added for the secondary reactions.","All reactions were performed in duplicate. Primary INVADER reactions were done in a final volume of 10 \u03bcl and contained: 10 mM MOPS, pH 7.5, 7.5 mM MgCl, 500 fm of primary INVADER (218-55-05; SEQ ID NO:171); 30 ng of AfuFEN1 enzyme and 10 ng of human genomic DNA. 100 zeptomoles of HBV pAM6 #2 amplicon was included in all even numbered reactions (by reference to FIGS. A and B). Reactions included 10 pmoles, 20 pmoles, 50 pmoles, 100 pmoles or 150 pmoles of primary probe (218-55-02; SEQ ID NO:170). MOPS, target and INVADER oligonucleotides were combined to a final volume of 7 \u03bcl. Samples were heat denatured at 95\u00b0 C. for 5 minutes, then cooled to 67\u00b0 C. During the 5 minute denaturation, MgCl, probe and enzyme were combined. The primary INVADER reactions were initiated by the addition of 3 \u03bcl of MgCl, probe and enzyme mix, to the final concentrations indicated above. Reactions were incubated for 30 minutes at 67\u00b0 C. The reactions were then cooled to 52\u00b0 C., and each primary INVADER reaction received the following secondary reaction components in a total volume of 4 \u03bcl: 2.5 pmoles secondary target (oligo number 218-95-04; SEQ ID NO:172); 10 pmoles secondary probe (oligo number 228-48-04; SEQ ID NO:173). The reactions that included the ARRESTOR oligonucleotide had either 40 pmoles, 80 pmoles, 200 pmoles, 400 pmoles or 600 pmoles of ARRESTOR oligonucleotide (oligo number 218-95-01; SEQ ID NO:174), added at a 4-fold molar excess over the primary probe amount for each reaction, with this mix. Reactions were then incubated at 52\u00b0 C. for 30 minutes. The reactions were stopped by the addition of 10 \u03bcl of a solution of 95% formamide, 10 mM EDTA and 0.02% crystal violet. All samples were heated to 95\u00b0 C. for 1 minute, and 4 \u03bcl of each sample were resolved by electrophoresis through 20% denaturing acrylamidc gel (19:1 cross-linked) with 7 M urea, in a buffer containing 45 mM Tris-Borate (pH8.3) and 1.4 mM EDTA. The results were imaged using the Molecular Dynamics Fluoroimager 595, excitation 488, emission 530. The resulting images for the reactions either without or with an ARRESTOR oligonucleotide are shown in , respectively. The products of cleavage of the secondary probe are seen as a band near the bottom of each panel.","In , lane sets 1 and 2 show results with 10 pmoles of primary probe; 3 and 4 had 20 pmoles; 5 and 6 had 50 pmoles; 7 and 8 had 100 pmoles; and 9 and 10 had 150 pmoles. It can be seen by visual examination, that the increases in the amount of primary probe have the combined effect of slightly increasing the background in the no-target lanes (odd numbers) while reducing the specific signal in the presence of target (even numbered lanes), and therefore the reducing the specificity of the reaction if viewed as the measure of \u201cfold over background,\u201d demonstrating that the approach of increasing signal by increasing probe cannot be applied in these sequential reactions.","In , lane sets 1 and 2 show results with 10 pmoles of primary probe; while 3 and 4 had 20 pmoles; 5 and 6 had 50 pmoles; 7 and 8 had 100 pmoles; and 9 a 10 had 150 pmoles. In addition, each reaction included 4-fold molar excess of the ARRESTOR oligonucleotide added before the secondary cleavage reaction. It can be seen by visual examination that the background in the no-target lanes (odd numbers) is lower in all cases, while the specific signal in the presence of target (even numbered lanes) increases with increased amounts of primary probe, leading to a greater \u201cfold over background\u201d sensitivity at this target level.","To quantitatively compare these effects, the fluorescence signal from the products of both non-specific and specific cleavage were measured. The results are depicted graphically in , graphed as a measure of the percentage of the secondary probe cleaved during the reaction, compared to the amount of primary probe used. Examination of the plots from the no-target reactions confirms that the background in the absence of the ARRESTOR oligonucleotide is, in general, roughly two-fold higher, and that both increase slightly with the increasing probe amounts. The specific signals however, diverge between the two sets of reaction more dramatically. While the signal in the no-ARRESTOR oligonucleotide reactions decreases steadily as primary probe was increased, the signal in the ARRESTOR oligonucleotide reactions continued to increase. At the highest primary probe concentrations tested, the no-ARRESTOR oligonucleotide reactions had specific signal that was only 1.7 fold over background, while the ARRESTOR oligonucleotide reactions detected the 100 zmoles (60,000 copies) of target with a signal 6.5 fold over background, thus demonstrating the improvement in the sequential invasive cleavage reaction when an ARRESTOR oligonucleotide is included.","The reactions described in the previous two Examples used ARRESTOR oligonucleotides that were constructed using 2\u2032 O-methyl ribose backbone, and that included a positively charged amine group on the 3\u2032 terminal nucleotide. The modifications were made specifically to reduce enzyme interaction with the primary probe\/ARRESTOR oligonucleotide complex. During the development of the present invention, it was determined that the 2\u2032 O-methy modified oligonucleotides are somewhat resistant to cleavage by the 5\u2032 nucleases, just as they are slowly degraded by nucleases when used in antisense applications (See e.g., Kawasaki et al, J. Med. Chem., 36:831 [1993]).","Further, as demonstrated in Example 35, the presence of an amino group on the 3\u2032 end of an oligonucleotide reduces its ability to direct invasive cleavage. To reduce the possibility that the ARRESTOR oligonucleotide would form a cleavage structure in this way, an amino group was included in the design of the experiments described in this and other Examples.","Initial designs of the ARRESTOR oligonucleotides (sometimes referred to as \u201cblockers\u201d) did not include these modifications, and these molecules were found to provide no benefit in reducing background cleavage in the sequential invasive cleavage assay and, in fact, sometimes contributed to background by inducing cleavage at an unanticipated site, presumably by providing some element to an alternative cleavage structure. The effects of natural and modified ARRESTOR oligonucleotide on the background noise in these reactions are examined in this Example.","The efficacy of an \u201call-natural ARRESTOR oligonuclcotide (i.e., an ARRESTOR oligonucleotide that did not contain any base analogs or modifications) was examined by comparison to an identical reactions that lacked ARRESTOR oligonucleotide. All reactions were performed in duplicate, and were conducted as follows. Two master mixes were assembled, each containing 12.5 mM MOPS, pH 7.5, 500 fmoles primary INVADER oligonucleotide #218-55-05 (SEQ ID NO:171), 10 ng human genomic DNA (Novagen) and 30 ng AfuFEN1 enzyme for every 8 \u03bcl of mix. Mix A contained no added HBV genomic amplicon DNA, mix B contained 600,000 molecules of HBV genomic amplicon DNA, pAM6 #2. The mixes were distributed to the reaction tubes, in aliquots of 8 \u03bcl\/tube as follows: mix A to tubes 1, 2, 5 and 6; and mix B to tubes 3, 4, 7 and 8. The samples were incubated at 95\u00b0 C. for 4 minutes to denature the HBV genomic amplicon DNA. The reactions were then cooled to 67\u00b0 C. and 2 ul of a mix containing 37.5 mM MgCland 10 pmoles 218-55-02B (SEQ ID NO:185) for every 2 \u03bcl, was added to each sample. The samples were then incubated at 67\u00b0 C. for 30 minutes. Two secondary reaction master mixes were prepared, each containing 10 pmoles of secondary probe oligo #228-48-04N (SEQ ID NO:178) and 2.5 pmoles of secondary target oligonucleotide #218-95-04 (SEQ ID NO:172) for every 3 \u03bcl of mix. Mix 2A contained no additional oligonucleotide, while mix 2B contained 50 pmoles of the natural \u201cARRESTOR\u201d oligonucleotide #241-62-02 (SEQ ID NO:186). After the initial 30 minute incubation at 67\u00b0 C., the temperature was adjusted to 52\u00b0 C., and 3 \u03bcl of a secondary reaction mix was added to each sample, as follows: Mix 2A was added to samples #1-4; and Mix 2B was added to samples #5-8. The samples were then incubated for 30 minutes at 52\u00b0 C. The reactions were then stopped by the addition of 10 \u03bcl of a solution of 95% formamide, 10 mM EDTA and 0.02% crystal violet.","All of the samples were heated to 95\u00b0 C. for 2 minutes, and 4 il of each sample were resolved by electrophoresis through a 20% denaturing acrylamide gel (19:1 cross-linked) with 7 M urea, in a buffer containing 45 mM Tris-Borate (pH8.3) and 1.4 mM EDTA. The results were imaged using the Molecular Dynamics Fluoroimager 595, excitation 488, emission 530. The resulting image is shown in FIG. A.","To compare the effects of the various modifications made to the ARRESTOR oligonucleotides, reactions were performed using ARRESTOR oligonucleotides having all natural bases, but including a 3\u2032 terminal amine; ARRESTOR oligonucleotide having the 3\u2032 portion composed of 2\u2032 O-methyl nucleotides, plus the 3\u2032 terminal amine; and ARRESTOR oligonucleotide composed entirely of 2\u2032 O-methyl nucleotides, plus the 3\u2032 terminal amine. These were compared to reactions performed without an ARRESTOR oligonucleotide. The reactions were conducted as follows. Two master mixes were assembled, all mixes contained 14.3 mM MOPS, pH 7.5, 500 fmoles primary INVADER oligo #218-55-05 (SEQ ID NO:171) and 10 ng human genomic DNA (Novagen) for every 7 \u03bcl of mix. Mix A contained no added HBV genomic amplicon DNA, mix B contained 600,000 molecules of HBV genomic amplicon DNA, pAM6 #2. The mixes were distributed to the reaction tubes, at 7 \u03bcl\/tube: mix A to tubes 1, 2, 5, 6, 9, 10, 13 and 14; and mix B to tubes 3, 4, 7, 8, 11, 12, 15 and 16. The samples were warmed to 95\u00b0 C. for 4 minutes to denature the HIBV DNA. The reactions were then cooled to 67\u00b0 C. and 3 \u03bcl of a mix containing 25 MM MgCl, 25 pmoles 218-55-02B (SEQ ID NO:185) and 30 ng AfuFENI enzyme per 3 \u03bcl, were added to each sample. The samples were then incubated at 67\u00b0 C. for 30 minutes. Four secondary reaction master mixes were prepared; all mixes contained 10 pmoles of secondary probe oligonucleotide #228-48-04B (SEQ ID NO:190) and 2.5 pmoles of secondary target oligonucleotide #218-95-04 (SEQ ID NO:172) for every 3 \u03bcl of mix. Mix 2A contained no additional oligonucleotide, while mix 213 contained 100 pmoles of the natural+amine ARRESTOR oligonucleotide # 241 -62-01 (SEQ ID NO:187), mix 2C contained 100 pmoles of partially O-methyl+amine oligonucleotide # 241-62-03 (SEQ ID NO:188) and mix 2D contained 100 pmoles of all O-methyl+amine oligonucleotide # 241-64-01 (SEQ ID NO:189). After the initial 30 minute incubation at 67\u00b0 C., the temperature was adjusted to 52\u00b0 C. and 3 \u03bcl of a secondary reaction mix was added to each sample, as follows: mix 2A was added to samples #1-4; mix 2B was added to samples #5-8; mix 2C was added to samples #9-12; and mix 2D was added to samples #13-16. The samples were incubated for 30 minutes at 52\u00b0 C., then stopped by the addition of 10 \u03bcl of a solution of 95% formamide, 10 mM NaEDTA, and 0.2% crystal violet.","All samples were heated to 95\u00b0 C. for 2 minutes, and 4 \u03bcl of each sample were resolved by electrophoresis through a 20% denaturing acrylamide gel (19:1 cross-linked) with 7 M urea, in a buffer containing 45 mM Tris-Borate (pH8.3) and 1.4 mM EDTA. The results were imaged using the Molecular Dynamics Fluoroimager 595, excitation 488, emission 530. The resulting image is shown in FIG. B.","In , the left hand panel shows the reactions that lacked an ARRESTOR oligonucleotide, while the right hand panel shows the data from reactions that included the all natural ARRESTOR oligonucleotide. The first two lanes of each panel are from no-target controls, the second set of lanes contained target. The products of cleavage are visible in the bottom one\/fourth of each panel. The position at which the specific reaction products should run is indicated by arrows on left and right.","It can be seen by examination of these data, that the reactions run in the absence of ARRESTOR oligonucleotide show reproducible quality between the replicates, and show significant cleavage only when target is present. In contrast, the addition of another unmodified oligonucleotide into the reactions causes great variation between the replicate lanes (e.g., lanes 5 and 6 were provided with the same reactants, but produced markedly different results). The introduction of the all natural ARRESTOR oligonucleotide produced, rather than reduced, background in these no-target lanes, and increased cleavage at other sites (i.e., the bands other that those indicated by the arrows flanking the panels). For these reasons the modifications that are described above, the effects of which are shown on , were incorporated.","The first 4 lanes of  show the products of duplicate reactions without an ARRESTOR oligonucleotide, plus or minus the HBV target (lanes 1, 2, and lanes 3, 4, respectively); The next 4 lanes, 5, 6 and 7, 8 used a natural ARRESTOR oligonucleotide having a 3\u2032 terminal amine; lanes 9, 10 and 11, 12 used the ARRESTOR oligonucleotide with a 3\u2032 portion composed of 2\u2032 O-methyl nucleotides, and having a 3\u2032 terminal amine; lanes 13, 14 and 15, 16 used the ARRESTOR oligonucleotide composed entirely of 2\u2032 O-methyl nucleotides and having a 3\u2032 terminal amine. The products of cleavage of the secondary probe are visible in the lower one third of each panel.","Visual inspection of these data shows that the addition of the 3\u2032 tcrminal amino to the natural ARRESTOR oligonucleotide suppresses the aberrant cleavage seen in , but this ARRESTOR oligonucleotide does not improve the performance of the reaction, as compared to the no-ARRESTOR oligonucleotide controls. In contrast, the use of the 2\u2032 O-methyl nucleotides in the body of the ARRESTOR oligonucleotide does reduce background, whether partially or completely substituted. To quantify the relative effects of these modifications, the fluorescence from each of the co-migrating product bands was measured, the signals from the duplicate lanes were averaged and the \u201cfold over background\u201d was calculated for each reaction containing target nucleic acid.","When ARRESTOR oligonucleotide was omitted, the target specific signal (lanes 3, 4) was 27-fold over the no target background; the natural ARRESTOR oligonucleotide+amine gave a signal of 17-fold over background; the partial 2\u2032 O-methyl+amine gave a signal of 47-fold over background; and the completely 2\u2032 O-methyl+amine gave a signal of 33 fold over background.","These Figures show that both modifications can have a beneficial effect on the specificity of the multiple, sequential invasive cleavage assay. They also show that the use of the 2\u2032 O-methyl substituted backbone, either partial or entire, markedly improves the specificity of these reactions. It is intended that in various embodiments of the present invention, that any number of modifications that make either the ARRESTOR oligonucleotide or the complex it forms with the primary target resistant to nucleases will provide similar enhancement.","Effect of ARRESTOR Oligonucleotide Length on Signal Enhancement in Multiple Sequential Invasive Cleavage Assays","As noted in the Description of the Invention, the optimal length for an ARRESTOR oligonucleotide depends upon the design of the other nucleic acid elements of the INVADER reaction, particularly on the design of the primary probe. In this Example, the effects of varying the length of the ARRESTOR oligonucleotide were explored in systems using two different secondary probes. A schematic diagram showing these ARRESTOR oligonucleotides aligned as they would hybridize to the primary probe oligonucleotide is provided in FIG. C. In this Figure, the region of the primary probe that recognizes the target nucleic acid is shown underlined; the non-underlined portion, plus the first underlined base is the portion that is released by the first cleavage, and goes on to participate in the second or subsequent cleavage structure.","All reactions were performed in duplicate. The INVADER reactions were done in a final volume of 10 \u03bcl final volume containing 10 mM MOPS, pH 7.5, mM MgCl, 500 fmoles of primary INVADER 241-95-01, (SEQ ID NO:176), 25 pmoles of primary probe 241-95-02 (SEQ ID NO:175), 30 ng of AfuFEN1 enzyme, and 10 ng of human genomic DNA, and if included, 1 amoles of HBV amplicon pAM 6 #2. MOPS, target DNA, and INVADER oligonucleotides were combined to a final volume of 7 \u03bcl. Samples were heat denatured at 95\u00b0 C. for 5 minutes, then cooled to 67\u00b0 C. During the 5 minute denaturation, MgCl, probe and enzyme were combined. The primary INVADER reactions were initiated by the addition of 3 \u03bcl of MgCl, probe and enzyme mix, to the final concentrations indicated above. Reactions were incubated for 30 minutes at 67\u00b0 C. The reaction were then cooled to 52\u00b0 C., and each primary INVADER reaction received the following secondary reaction components in a total volume of 3 \u03bcl: 2.5 pmoles secondary target 241-95-07 (SEQ ID NO:177), 10 pmoles of either secondary probe 228-48-04 (SEQ ID NO:173), or 228-48-04N (SEQ ID NO:178) and 100 pmoles of an ARRESTOR oligonucleotide, either 241-95-03 (SEQ ID NO:179), 241-95-04 (SEQ ID NO:180), 241-95-05 (SEQ ID NO:181) or 241-95-06 (SEQ ID NO:182). The ARRESTOR oligonucleotide were omitted from some reactions as controls for ARRESTOR oligonucleotide effects.","The reactions were incubated at 52\u00b0 C. for 34 minutes, and were then stopped by the addition of 10 \u03bcl of 95% formamide, 10 mM EDTA, and 0.02% crystal violet. All samples were heated to 95\u00b0 C. for 1 minute, and 4 \u03bcl of each sample were resolved by electrophoresis through 20% denaturing acrylamide gel (19:1 cross-linked) with 7 M urea, in a buffer containing 45 mM Tris-Borate (pH8.3) and 1.4 mM EDTA. The results were imaged using the Molecular Dynamics Fluoroimager 595, excitation 488, emission 530. The resulting images for the reactions with the shorter and longer secondary probes are shown in , respectively.","In each Figure, the products of cleavage are visible as bands in the bottom half of each lane. The first 4 lanes of each Figure show the products of duplicate reactions without an ARRESTOR oligonucleotide, plus or minus the HBV target (lanes sets 1 and 2 respectively); in the next 4 lanes, sets 3 and 4 used the shortest ARRESTOR 241-95-03 (SEQ ID NO:179); lanes 5 and 6 used 241-95-04 (SEQ ID NO:180); lanes 7 and 8 used 241-95-05 (SEQ ID NO:181); and lanes 9 and 10 used 241-95-06 (SEQ ID NO:182).","The principal background of concern is the band that appears in the \u201cno target\u201d control lanes (odd numbers; this band co-migrates with the targct-specific signal near the bottom of each gel panel). Visual inspection shows that the shortest ARRESTOR oligonucleotide was the least effective at suppressing this background, and that the efficacy was increased when the ARRESTOR oligonucleotide extended further into the portion that participates in the subsequent cleavage reaction. Even with this difference in effect, it can be seen from these data that there is much latitude in the design of the ARRESTOR oligonucleotide. The choice of lengths will be influenced by the temperature at which the reaction making use of the ARRESTOR oligonucleotide is performed, the lengths of the duplexes formed between the primary probe and the target, the primary probe and the secondary target, and the relative concentrations of the different nucleic acid species in the reactions.","In examining the effects of including ARRESTOR oligonucleotides in these cleavage reactions, it was of interest to determine if the concentration of the ARRESTOR oligonucleotide in excess of the primary probe concentration would have an effect on yields of either non-specific or specific signal, and if the length of the ARRESTOR oligonucleotide would be a factor. These two variable were investigated in the following Example.","All reactions were performed in duplicate. The primary INVADER reactions were done in a final volume of 10 \u03bcl and contained 10 mM MOPS, pH 7.5; 7.5 mM MgCl, 500 fmoles of primary INVADER 241-95-01 (SEQ ID NO:176), 25 pmoles of primary probe 241-95-02 (SEQ ID NO:175), 30 ng of AfuFENI enzyme, and 10 ng of human genomic DNA. Where included, the target DNA was 1 amole of HBV amplicon pAM 6 #2, as described above. MOPS, target and INVADER were combined to a final volume of 7 \u03bcl. The samples were heat denatured at 95\u00b0 C. for 5 minutes, then cooled to 67\u00b0 C. During the 5 minute denaturation, MgCl, probe and enzyme were combined. The primary INVADER reactions were initiated by the addition of 3 il of MgCl, probe and enzyme mix. The reactions were incubated for 30 minutes at 67\u00b0 C. The reactions were then cooled to 52\u00b0 C. and each primary INVADER reaction received the following secondary reaction components: 2.5 pmoles secondary target 241-95-07 (SEQ ID NO:177), 10 pmoles secondary probe 228-48-04 (SEQ ID NO:173); and, if included, 50, 100 or 200 pmoles of either ARRESTOR oligonuclcotide 241-95-03 (SEQ ID NO:179) or 241-95-05 (SEQ ID NO:181), in a total volume of 3 \u03bcl. Reactions were then incubated at 52\u00b0 C. for 35 minutes. Reactions were stopped by the addition of 10 \u03bcl of 95% formamide, 10 mM EDTA, and 0.02% crystal violet. All of the samples were heated to 95\u00b0 C. for 1 minute, and 4 \u03bcl of each sample were resolved by electrophoresis through 20% denaturing acrylamide gel (19:1 cross-linked) with 7 M urea, in a buffer containing 45 mM Tris-Borate (pH8.3), and 1.4 mM EDTA. The results were imaged using the Molecular Dynamics Fluoroimager 595, excitation 488, emission 530. The resulting images are shown as a composite image in FIG. .","Each of the duplicate reactions were loaded on the gel in adjacent lanes and are labeled with a single lane number. All odd numbered lanes were no-target controls. Lanes 1 and 2 had no ARRESTOR oligonucleotide added; lanes 3-8 show results from reactions containing the shorter ARRESTOR oligonucleotide, 241-95-03 (SEQ ID NO:179); lanes 9-14 show results from reactions containing the longer ARRESTOR oligonucleotide, 241-95-05 (SEQ ID NO:181). The products of cleavage from the secondary reaction are visible in the bottom one third of each panel. Visual inspection of these data (i.e., comparison of the specific products to the background bands) shows that both ARRESTOR oligonucleotides have some beneficial effect at all concentration.","To quantify the relative effects of ARRESTOR oligonucleotide length and concentration, the fluorescence from each of the co-migrating product bands was measured, the signals from the duplicate lanes were averaged and the \u201cfold over background\u201d (signal+target\/signal\u2212target) was calculated for each reaction containing target nucleic acid. The reaction lacking an ARRESTOR oligonucleotide yielded a signal approximately 27-fold over background. Inclusion of the shorter ARRESTOR oligonucleotide at 50, 100 or 200 pmoles produced products at 42, 51 and 60-fold over background, respectively. This shows that while the short arrestr at the lowest concentration seems to be less effective than the longer ARRESTOR oligonucleotides (See, previous Example) this can be compensated for by increasing the concentration of ARRESTOR oligonucleotide, and thereby the ARRESTOR oligonucleotide:primary probe ratio.","In contrast, inclusion of the longer ARRESTOR oligonucleotide at 50, 100 or 200 pmoles produced products at 60, 32 and 24 fold over background, respectively. At the lowest concentration, the efficacy of this longer ARRESTOR oligonucleotide relative to the shorter ARRESTOR oligonucleotide is consistent with the previous Example. Increasing the concentration, however, decreased the yield of specific product, suggesting a competition effect with some element of the secondary cleavage reaction.","These data show that the ARRESTOR oligonucleotides can be used to advantage in a number of specific reaction designs. The choice of concentration will be influenced by the temperature at which the reaction making use of the ARRESTOR oligonucleotide is performed, the lengths of the duplexes formed between the primary probe and the target, the primary probe and the secondary target, and between the primary probe and the ARRESTOR oligonucleotide.","Selection of oligonucleotides for target nucleic acids other than the HBV shown here, (e.g., oligonucleotide composition and length), and the optimization of cleavage reaction conditions in accord with the models provided here follow routine methods and common practice well known to those skilled in the methods of molecular biology.","Example 45 demonstrated that some enzymes require an overlap between an upstream INVADER oligonucleotide and a downstream probe oligonucleotide to create a cleavage structure (FIG. ). It has also been determined that the 3\u2032 terminal nucleotide of the INVADER oligonucleotide need not be complementary to the target strand, even if it is the only overlapping base in the INVADER oligonucleotide (e.g., as with the HCMV probes shown in FIG. ). The requirement for an overlap can serve as a convenient basis for detecting single base polymorphisms (SNPs) or mutations in a nucleic acid sample.","For detection of single base variations, at least two oligonucleotides (e.g., a probe and an INVADER oligonucleotide) hybridize in tandem to the target nucleic acid to form the overlapping structure recognized by the CLEAVASE enzyme to be used in the reaction. An unpaired \u201cflap\u201d is included on the 5\u2032 end of the probe. The enzyme recognizes the overlap and cleaves off the unpaired flap, releasing it as a target-specific product. Enzymes that have a strong preference for an overlapping structure, i.e., that cleave the overlapping structure at a much greater rate than they cleave a non-overlapping structure include the FEN-1 enzymes from and and such enzymes are particularly preferred in the detection of mutations and SNPs. In the secondary reaction, the released flap serves as an INVADER oligonucleotide to create another overlapping cleavage structure (e.g., as shown in FIG. ). In the following examples, the released flap creates this overlapping structure in conjunction with a FRET cassette, a single oligonucleotide having a region of self-complementarity to form a hairpin () When the FRET cassette is cleaved to release its 5\u2032 nucleotide, the fluorescent dye (F) and the quencher (Q) on the cassette are separated and a detectable fluorescence signal is produced. If the probe and the target sequence do not match perfectly at the cleavage site (e.g., as in FIG. B), the overlapped structure does not form, cleavage is suppressed, and no fluorescence will be produced.","The reactions may be performed under conditions in which the probes and FRET cassettes turn over continuously without temperature cycling or cleavage. When an uncut probe hybridizes to the target next to an overlapping INVADER oligonucleotide, the probe can be cleaved to produce a target-specific product, which in turn enables the cleavage of many FRET cassettes.","The following eight examples demonstrate the design and application of the sequential INVADER assay with FRET cassette detection to analysis of SNPs and mutations In a variety of nucleic acid samples.","This Example describes an assay for the detection of SNPs in the human Apolipoprotein E gene. Probe and INVADER oligonucleotides were designed to target single nucleotide polymorphisms (SNPs) at two positions in the human apolipoprotein E (apoE) gene. Three different alleles exist for the apoE gene, epsilon 2, epsilon 3, and epsilon 4, which code for 3 different isoforms of the apoE protein, termed E2, E3 and E4. The different isoforms vary in at amino acid positions 112 or 158 (see Table A), and each variation is caused by a single base change in the corresponding codon.",{"@attributes":{"id":"p-1064","num":"1081"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"center"}}],"thead":{"row":{"entry":[{},"TABLE A"]}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"112","158"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["ISOFORM","codon","amino acid","codon","amino acid"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":["E2","TGC","cys","TGC","cys"]},{"entry":["E3","TGC","cys","CGC","arg"]},{"entry":["E4","CGC","arg","CGC","arg"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},"INVADER and probe oligonucleotides were designed using the algorithm described above (detailed description of the invention), with the probe set selected to operate at 63\u00b0 C. As shown in  A-D, one INVADER oligonucleotide and two unlabelled, probe oligonucleotides, one for each variant at a given locus, were designed for each codon. For codon 112, the probes were designed to detect either the C nucleotide (to code for arginine; SEQ ID NO:197) or a T nucleotide (to code for cysteine; SEQ ID NO:198). For codon 158, the probes were designed to detect either a C nucleotide (to code for arginine; SEQ ID NO:199) or a T nucleotide (to code for cysteine; SEQ ID NO:200). A FRET cassette to be used with all of the probe sets was also synthesized (SEQ ID NO:201, FIG. ). In this Example and in the following Examples, all oligonucleotides were synthesized using standard phosphoramidite chemistries. Primary probe oligonucleotides were unlabeled. The FRET cassettes were labeled by the incorporation of Cy3 phosphoramidite and fluorescein phosphoramidite (Glen Research). While designed for 5\u2032 terminal use, the Cy3 phosphoramidite has an additional monomethoxy trityl (MMT) group on the dye that can be removed to allow further synthetic chain extension, resulting in an internal label with the dye bridging a gap in the sugar-phosphate backbone of the oligonucleotide (as diagrammed in the panels of FIG. ). While a nucleotide may be omitted at this position to accommodate the dye, we have determined it is not necessary, and no nucleotides were omitted from the FRET cassettes used in these examples. Amine or phosphate modifications, where indicated, were used on the 3\u2032 ends of the primary probes and the FRET cassettes to prevent their use as invasive oligonucleotides. 2\u2032-O-methyl bases in the secondary target oligonucleotides are indicated by underlining and were also used to minimize enzyme recognition of 3\u2032 ends. In addition, reactions having synthetic target DNAs were used as positive controls to verify the activity of the reaction components. The control targets are illustrated in  A-D. The 112 arg, 112 cys, 158 arg, and 158 cys control oligonucleotides are SEQ ID NOS:191-194, respectively.","Genomic DNA sample AG09714 was purchased from Coriell. This sample was quantitated via Pico Green and diluted with Tris with 0.1 mM EDTA to a concentration of approximately 100 ng\/\u03bcl. This sample (9714 in ) was used to test for the 112 mutation. Samples 39634, 32435 and 31071 were purchased as whole blood from Lampire Biological Labs., Inc. Samples 511, 537, 538 and 539 were whole blood samples donated by the Blood Center of Southeast Wisconsin (Milwaukee, Wis.). Genomic DNA was prepared via the PUREGENE Blood Kit (Gentra) according to the manufacturer's instructions. Samples 39634 and 32435 were used to test for the 112 mutation, and sample 31071, 511, 537, 538 and 539 were used to test for the 158 mutation. One \u03bcl of genomic DNA was used per reaction, with 9\u03bcl of water, for a final volume of 10 \u03bcl. Although determination of the full genotype for any one sample generally requires analysis of both loci, the samples listed above were selected to show representative signals from, and thus the functioning of, each probe set. Complete genotyping requires each sample to be tested with both probe sets.","The experiment comprised testing each genomic DNA for the indicated alleles, along with reactions having no target DNA to allow measurement of any background signal not attributable to the presence of a target sequence. Reactions testing the 112 locus were done in quadruplicate, reactions testing the 158 locus were done in triplicate.","Reaction components were prepared as batch mixes for dispensing to the individual test reactions. Batches of INVADER mix for each allele were prepared, comprising for each planned reaction: 4 \u03bcl of 16% PEG 8000\/50 mM MOPS pH 7.5 and 1 \u03bcl of 1 \u03bcM INVADER oligonucleotide (either the 112 or the 158). Batches of CLEAVASE enzyme\/Mg\/probe mix for each allele were prepared, comprising for each planned reaction: 2 \u03bcl of 75 mM MgCl, 1 \u03bcl of 10 \u03bcM FRET cassette, 1 \u03bcl 10 \u03bcM probe oligonucleotide (any one of the 112 or the 158 T or C probe oligonucleotides), and 1 \u03bcl of 200 ng\/\u03bcl Afu FEN-1 enzyme.","For each reaction, 5 \u03bcl of INVADER reaction mix were aliquoted into each well of a 96-well Low Profile Polypropylene Microplate (MJ Research,). Ten \u03bcl of each control DNA or genomic sample (approximately 100 ng-190 ng) were added and mixed by pipetting up and down. The no-target controls received 1 \u03bcg of yeast tRNA instead of target DNA. The reactions comprising the synthetic targets as positive controls included either 150 or 100 zeptomoles (zmoles) of the 112 or 158 synthetic targets, respectively, and 1 \u03bcg of yeast tRNA. Each reaction was overlaid with 20 \u03bcl of clear CHILLOUT liquid wax, and incubated at 95\u00b0 C. for 5 minutes. The reaction temperature was then lowered to 63\u00b0 C., 5 \u03bcl of the appropriate CLEAVASE enzyme\/ Mg\/Probe reaction mix was added to each reaction and mixed by pipetting up and down 3-5 times, and the reactions were further incubated for 4 hours at 63\u00b0 C., and were read directly on a CYTOFLUOR Series 4000 Fluorescence Multi-well Plate Reader, (PerSeptive Biosystems), using the following settings: Excitation (wavelength\/bandwidth): 485\/20 nm; Emission (wavelength\/bandwidth): 530\/25 nm; Gain: 37. The net averaged fluorescence signal was calculated by subtracting the averaged no-target signal (background) from the corresponding averaged target DNA reaction signal and the data were plotted using Excel spreadsheet software (Microsoft).","Results for the ApoE 112 and ApoE 158 loci are shown graphically in , respectively, with target DNAs indicated on the horizontal axis and the fluorescence units indicated on the vertical axis. The white bars represent the net averaged signal from the \u201cC\u201d probe, while the dark bars represent the net averaged signal from the \u201cT\u201d probe. The reactions having synthetic targets are indicated as Syn C and Syn T for the C and T controls, respectively. At both loci, the samples that are homozygous for either the C or T allele are easily identified by having a strong signal from only one probe or the other, while the heterozygous samples, are easily identified by having strong signals from both the C and T probes.","The human hemochromatosis (HFE gene) gene is located in the MHC region of chromosome 6, and was initially named HLA-H. It was later renamed the HFE gene, in accordance with the WHO Nomenclature Committee for Factors of the HLA System. Two different, single-base variations in the HFE gene are responsible for the vast majority of the cases of hereditary hemochromatosis, or iron overload disease. The most common variant, termed C282Y, is caused by a change from the wild type (WT) adenine (A) to the mutant (MT) guanine (G) at codon 282 that causes an amino acid change from a cysteine to a tyrosine residue. The second variation commonly detected in individuals suffering from iron overload disorder is termed H63D, and is caused by a WT cytosine (C) to a MT guanine (G) change at codon 63 that causes an amino acid change from a histidine to an aspartic acid residue in the expressed protein.","INVADER and probe oligonucleotide sets were designed as described above to detect both the WT and MT alleles for both the C282Y and H63D sites and are shown in . Detection of the C282 polymorphism used one INVADER oligonucleotide (SEQ ID NO:202), one probe specific for each variant of the allele (WT and MT, SEQ ID NOS:208 and.209, respectively) and a first FRET cassette (SEQ ID NO:210). Oligonucleotides for the H63 locus included an INVADER oligonucleotide (SEQ ID NO:203), one probe specific for each variant of the allele (WT and MT, SEQ ID NOS:211 and 212, respectively), and a second FRET cassette (SEQ ID NO:213). In addition, reactions having synthetic target DNAs were used as positive controls to verify the activity of the reaction components. The control targets are illustrated in  A-D. The C282 WT, C282 MT, H63 WT and H63 MT control oligonucleotides are SEQ ID NOS: 204-207, respectively. Human genomic DNA samples 14640, 14641, 14646, 14690, and 14691 were purchased from Coriell Cell Repositories (Catalog #s NA 14640, NA14641, NA1446, NA14690, and NA14691).","Reactions were performed and analyzed as described in Example 54. The reactions comprising the synthetic targets as positive controls included 100 zmoles of the synthetic target and 1 \u03bcg of yeast tRNA.","Results are shown graphically in , with target DNAs indicated on the horizontal axis and the fluorescence units indicated on the vertical axis. The white bars represent the net signal from the wild-type probe, while the dark bars represent the net signal from the mutant probe. The left half of the graph indicates samples tested for the C282Y polymorphism, while the right half of the graph indicates samples tested for the H63D polymorphism. No target controls are indicated as \u201cNT\u201d and the synthetic targets are indicated as SynWT and SynMT for wild-type and mutant controls, respectively. At both loci, the samples that are homozygous for either WT or MT are easily identified by having a strong signal from only one probe or the other, while the heterozygous samples, are easily identified by having strong signals from both the WT and MT probes.","Human 5, 10-methylene-tetrahydrofolate reductase (MTHFR) is a major enzyme in the folate-dependent regulation of methionine and homocysteine concentrations. The wild-type protein plays a critical role in the conversion of homocysteine to methionine. A particular variation in the MTHFR protein, termed C677T and caused by a C to T transition in the MTHFR gene, has been correlated with myriad diseases and defects, including cardiovascular and neurological disorders. This Example describes an assay for the detection of the WT and MT alleles for the MTHFR 677 SNP.","INVADER and probe oligonucleotide sets were designed as described above to detect both the WT and MT allele for the MTHFR 677 site (INVADER, WT probe, MT probe, and FRET cassette are SEQ ID NOS:216, 217, 218 and 225, respectively). Positive control targets were synthesized for the WT and MT alleles at the 677 site (SEQ ID NOS:214 and 215, respectively); oligonucleotides are shown in .","Human genomic DNA samples 01532 and 01560 were purchased from Coriell. These samples had been characterized by \u201cPCR\u201d at Coriell for the MTHFR genotype. They were also characterized in house by PCR\/RFLP analysis for genotype confirmation. Human genomic sample 32435 was purchased as whole blood from Lampire Biological Labs., Inc. (Coopersberg, Pa.), and genomic DNA was prepared via the Gentra PUREGENE Blood Kit (Minneapolis, Minn.) according to the manufacturer's instructions. Samples were quantitated via PicoGreen (Molecular Probes, Eugene OR) and diluted with TE to a concentration of approximately 10 ng\/\u03bcl. 100 ng (10 \u03bcl) of each sample was used in each reaction.\n\nTriplicate reactions were performed and analyzed as described in Example 54, except the INVADER mixes contained 1 \u03bcl of 0.5 \u03bcM INVADER oligonucleotide for each reaction. The reactions comprising the synthetic targets as positive controls included 50 zmoles of the synthetic target and 1 \u03bcg of yeast TRNA. Reactions simulating a heterozygous sample included 50 zmoles of each control target.\n","Results are shown graphically in , with target DNAs indicated on the horizontal axis and the fluorescence units indicated on the vertical axis. The white bars represent the net averaged signal from the wild-type probe, while the dark bars represent the net averaged signal from the mutant probe. The synthetic targets are indicated as SynWT and SynMT for wild-type and mutant controls, respectively and SynHET for a mixture of the two to simulate a heterozygous sample. At both loci, the samples that are homozygous for either WT or MT are easily identified by having a strong signal from only one probe or the other, while the heterozygous samples are easily identified by having strong signals from both the WT and MT probes.","The prothrombin A20210G mutation has been determined to be a risk factor for thromboembolism. The mutation occurs in the 3\u2032 untranslated region of the prothrombin gene, replacing the WT adenine (A) at position 20210 with the MT guanine (G). This Example describes an assay for the detection of the WT and MT alleles of the prothrombin A20210G mutation","INVADER and probe oligonucleotide sets were designed as described above to detect both the WT and MT alleles at the prothrombin 20210 site (INVADER, WT probe, MT probe and FRET cassette are SEQ ID NOS: 222-225, respectively). Positive control targets were synthesized for the WT and MT alleles at the 20210 site (SEQ ID NOS:220 and 221, respectively); oligonucleotides are shown in .","Three patient samples of human genomic DNA were donated by the Blood Center of Southeast Wisconsin, identified as 2196, 2263 and 2265. These samples were purified at the Blood Center via QIAGEN BioRobot 9600 (QIAGEN # 900200), quantitated by Pico Green (Molecular Probes) and were found to be at a concentration of 30-50 ng\/ul.","Duplicate reactions were performed and analyzed as described in Example 54, except the INVADER mixes contained 1 \u03bcl of 0.5 \u03bcM INVADER oligonucleotide for each reaction, and the mix was diluted with 1 volume of dHO (i.e., 5 \u03bcl per reaction) so that the INVADER mix was dispensed 10 \u03bcl aliquots, while the DNA was dispensed in 5 \u03bcl aliquots, adding 150-250 ng of genomic DNA per reaction. Each no target control reaction received 1 \u03bcg of yeast tRNA, and the reactions comprising the synthetic targets as positive controls included 300 zmoles of WT control or 50 zmoles of MT control and 1 \u03bcg of yeast tRNA.","Results are shown graphically in , with target DNAs indicated on the horizontal axis and the fluorescence units indicated on the vertical axis. The white bars represent the net averaged signal from the wild-type probe, while the dark bars represent the net averaged signal from the mutant probe. Synthetic targets are indicated as SynWT and SynMT for wild-type and mutant controls, respectively. Samples that are homozygous for the WT are easily identified by having a strong signal from only the WT probe, while the heterozygous sample is easily identified by having strong signals from both the WT and MT probes.","This Example describes an assay for the deteciton of the R-2 polymorphism in the human factor V gene. The R-2 polymorphism in the human factor V gene is located in exon 13 of the factor V gene, and is the result of an A to G transition at base 4070, replacing the wild-type amino acid histidine with the mutant arginine in the mature protein. The R-2 polymorphism is one of a set of mutations termed collectively HR-2. The HR-2 haplotype is defined by 6 nucleotide base substitutions in exons 13 and 16 of the factor V gene, and is associated with an increased functional resistance to activated protein C both in normal subjects and in thrombophilic patients. When present as a compound heterozygote in conjunction with the Leiden mutation (see Example 60, below), clinical symptoms are comparable to those seen in patients homozygous for the Leiden mutation.","Within about a 600-base pair region surrounding the R2 allele, four sub-regions of DNA, each encompassing the sequence of the WT INVADER probe set, (approximately 22 bases in length), contain sequence similar to that immediately surrounding the R2 allele. These repeated sequences can be detected by an INVADER and probe oligonucleotide set designed for the wild-type R-2 sequence. Because of the repeated sequence, reactions with this INVADER\/probe set yield very high signal, even with genomic samples containing R-2 mutants, thus greatly increasing the risk of misinterpreting the data. In the example of an R-2 heterozygote, the signal generated by the R-2 mutant would be extremely low compared to the wild-type signal. It is thus possible that one might err and interpret the data as wild-type, not as heterozygous. The same would hold true even for a homozygous mutant R-2 sample. Therefore, instead of having an INVADER\/probe set to detect the WT R-2 allele, an INVADER\/probe set was developed to detect sequences in the single copy \u03b1-actin gene, thus providing an internal reaction control, as well as an internal signal intensity control. Since the \u03b1-actin gene is single copy, the signal levels generated in the detection of this sequence will be comparable to that generated in the detection of the R-2 mutant allele, and the probability of incorrect data interpretation due to WT signal overwhelming that generated by the MT is no longer an issue.","In the previous examples, each sample was assayed in two different reactions, one reaction tested for the presence of wild-type sequence and one reaction tested for the presence of mutant sequence. In this example, each sample is tested with both the \u03b1-actin internal control and the MT R-2 INVADER\/probe sets. INVADER and probe oligonucleotide sets were designed as described above to detect both the MT R-2 and \u03b1-actin control sequence (MT R-2 INVADER and probe, and the FRET cassette are SEQ ID NOS:228, 229 and 230, respectively; \u03b1-actin INVADER and probe are SEQ ID NOS: 231 and 232, respectively). Positive control targets were synthesized for the Mutant R-2 allele and the \u03b1-actin gene (SEQ ID NOS:226 and 227, respectively); oligonucleotides are shown in .","Human genomic DNA sample 39021 was obtained from Sigma (Catalog # D6537) and uncharacterized human genomic sample 15506 was obtained from Coriell(Catalog # NA15506, Camden, N.J. 08103) Samples were diluted to 10 ng\/\u03bcl with Tris-EDTA, pH 8.0. Ten \u03bcl (100 ng) was used per reaction. No target controls received 1 \u03bcg of yeast tRNA instead of human genomic DNA. Reactions were performed and analyzed as described in Example 54, except the INVADER mixes contained 0.5 \u03bcl each of 2 \u03bcM R-2 INVADER oligonucleotide and 2.0 \u03bcM \u03b1-actin INVADER oligonucleotide. The probe master mixes contained 1 \u03bcl of either the 10 \u03bcM R-2 probe or 1 \u03bcl of 10 \u03bcM \u03b1-actin probe, 2 \u03bcl of 75 mM MgCl, 1 \u03bcl of 10 \u03bcM FRET cassette and 1 \u03bcl 200 ng\/\u03bcl Afu FEN-1 enzyme per reaction. The reactions comprising the synthetic targets as positive controls included 100 zmoles of the synthetic target and 1 \u03bcg of yeast tRNA. The SynHET and SynMT reactions contained mixtures of synthetic targets at 2:1 and 1:1 of the \u03b1-actin:R-2 mutant targets, respectively.","Reactions were read directly on a CYTOFLUOR Multi-well Plate Reader Series 4000 (PerSeptive Biosystems) using the following parameters: Excitation wavelength\/bandwidth 485nm\/20 nm, Emission wavelength\/bandwidth 530 nm\/25 nm, gain 36.","Results are shown graphically in , with target DNAs indicated on the horizontal axis and the fluorescence units indicated on the vertical axis. The white bars represent the net signal from the internal control probe, while the dark bars represent the net signal from the R-2 mutant probe. Synthetic targets are indicated as SynIC and SynMT for internal control and mutant R-2 controls, respectively and SynHET for a mixture of the two to simulate a sample that is heterozygous at the R-2 allele. The sample that does not have the MT R-2 allele is easily identified by having a strong signal from only the IC probe, while that which is heterozygous at the R-2 allele is identified by having signals from both the IC and the MT R-2 probes, but at a ratio near 2:1. A sample homozygous for the mutation at the R-2 allele (not shown) would show nearly equal signal from each probe, as shown with the SynMT control.","The human cytokine tumor necrosis factor \u03b1 (TNF-\u03b1) gene has been shown to be a major factor in graft rejection; the more TNF-\u03b1 present in the system, the greater the rejection response to transplanted tissue. The mutation detected in this example is located in the promoter region of the TNF-\u03b1 gene at position \u2212308 (minus 308). The WT guanine (G) is replaced with a MT adenine (A). This result of this promoter mutation is the enhancement of transcription of TNF-\u03b1 by 6 to 7 fold. This Example describes an assay for the detection of the \u2212308 mutation in the promoter of the human TNF-\u03b1 gene.","INVADER and probe oligonucleotide sets were designed as described above to detect both the WT and MT alleles at the \u2212308 site (INVADER, WT probe, and MT probe are SEQ ID NOS:235, 236 and 237, respectively; FRET cassette is SEQ ID NO:225).","Positive control targets were synthesized for the WT and MT alleles at the \u2212308 site (SEQ ID NOS:233 and 234, respectively); oligonucleotides are shown in .","Purified human genomic DNA samples (M2, M3 and M4) were donated by the Mayo Clinic (Rochester Minn.).","Triplicate reactions were performed as described in Example 54, except they were stopped after the four hour incubation at 63\u00b0 C. by the addition 100 \u03bcl of 100 mM EDTA. Reactions comprising the synthetic targets as positive controls included 100 zmoles of the synthetic target and 1 \u03bcg of yeast tRNA. No target controls received 1 \u03bcg of yeast tRNA instead of human genomic DNA. 100 \u03bcl of each stopped reaction was transferred to a Nunc 96 well Maxisorb plate (VWR Scientific) and read on a CYTOFLUOR Multi-well Plate Reader Series 4000 (PerSeptive Biosystems) using the following parameters: Excitation wavelength\/bandwidth 485 nm\/20 nm, Emission wavelength\/bandwidth 530 nm\/25 nm; gain 65.","Results are shown graphically in , with target DNAs indicated on the horizontal axis and the fluorescence units indicated on the vertical axis. The white bars represent the net averaged signal from the wild-type probe, while the dark bars represent the net averaged signal from the mutant probe. The synthetic targets are indicated as SynWT and SynMT for wild-type and mutant controls, respectively and SynHET for a mixture of the two to simulate a heterozygous sample. The samples that are homozygous for either WT or MT are easily identified by having a strong signal from only one probe or the other, while the heterozygous sample is easily identified by having signals from both the WT and MT probes.","The \u201cLeiden \u201d mutation in blood coagulation factor V results from a cytosine \u201cC\u201d to a thymidine \u201cT\u201d base change in exon 1 of the factor V gene. The mutant protein has glutamine at amino acid position 506, instead of the wild-type arginine. This substitution prevents activated protein C from cleaving and inactivating factor V. The active form of the protein therefore remains abundant in the blood stream and continues to promote coagulation. This Example describes an assay for the detection of the Leiden mutation of the human factor V gene.","INVADER and probe oligonucleotide sets were designed as described above to detect both the WT and MT alleles at the 506 site (INVADER, WT probe, and MT probe are SEQ ID NOS:240, 241 and 242, respectively; the FRET cassette is SEQ ID NO:225). Positive control targets were synthesized for the WT and MT alleles at the 506 site (SEQ ID NOS:238 and 239, respectively); oligonucleotides are shown in .","Whole blood samples were obtained from the Midwest Hemostasis Center (Muncie, Ind.). Samples were characterized for the Factor V Leiden genotype by the Midwest Hemostasis Center via methods involving PCR. Buffy coats were isolated as previously described, and genomic DNA was purified using the QIAamp 96 DNA Blood Kit (Qiagen, Valencia) according to the manufacturer's instructions, except that the samples were eluted in 200 \u03bcl of elution buffer. The purified DNA samples were quantitated by Pico Green (Molecular Probes) and were then diluted with TE to a concentration of 15-60 ng per \u03bcl. Single reactions were performed as described in Example 54. The no-target control reaction received 1 \u03bcg of yeast tRNA instead of DNA, and the reactions comprising the synthetic targets as positive controls included 200 zmoles of the synthetic target and 1 \u03bcg of yeast tRNA.","After the 4 hour incubation at 63\u00b0 C., reactions were read directly on a CYTOFLUOR Multi-well Plate Reader Series 4000 (PerSeptive Biosystems) using the following parameters: Excitation wavelength\/bandwidth 485 nm\/20 nm, Emission wavelength\/bandwidth 530 nm\/25 nm, gain 65.","Results are shown graphically in , with target DNAs indicated on the horizontal axis and the fluorescence units indicated on the vertical axis. The white bars represent the net averaged signal from the wild-type probe, while the dark bars represent the net averaged signal from the mutant probe. The synthetic targets are indicated as SynWT and SynMT for wild-type and mutant controls, respectively and SynHET for a mixture of the two to simulate a heterozygous sample. The samples that are homozygous for either WT or MT are easily identified by having a strong signal from only one probe or the other, while the heterozygous samples are easily identified by having signals from both the WT and MT probes.",{"@attributes":{"id":"p-1101","num":"1118"},"i":["Staphylococcus aureus ","S. aureus ","Staphalococcus aureus","S. aureus "]},"INVADER and probe oligonucleotide sets were designed as described above to detect both the mecA gene and the nuc gene (meA INVADER and probe, are SEQ ID NOS:243 and 244, respectively; nuc INVADER and probe are SEQ ID NOS:246 and 247, respectively; the FRET cassette for both INVADER\/probe sets is SEQ ID NO:245); oligonucleotides are shown in . The mecA and nuc target sequences shown are SEQ ID NOS:252 and 253, respectively. Samples of methicillin-resistant were purchased from American Type Culture Collection (ATCC, Catalog # 33591). Samples of methicillin sensitive (MSSA) were obtained from Gene Trak, Inc. (GT#2431), and samples of were obtained from ATCC (ATCC#29970). The bacterial samples were streaked onto standard blood agar plates and grown at 37\u00b0 C. for 14-18 hours. DNA samples from MRSA, MSSA, and were prepared as follows. A single colony was suspended in 50 \u03bcl of 10 mM TRIS pH 7.5 in a 1.5 ml microfuge tube. The sample was incubated at 65\u00b0 C. for 5 minutes, and then micro-waved on the highest setting for 4 minutes. Ten \u03bcl of this preparation was used in each reaction.","Positive controls were created by polymerase chain reaction. A 533 base-pair DNA fragment of the mecA sequence and a 467 base pair DNA fragment of the nuc gene sequence were amplified and isolated as follows. PCR primer sequences used for mecA gene amplification were 5\u2032-AAA ATC GAT GGT AAA GGT TGG C-3\u2032 (SEQ ID NO:248)and 5\u2032-AGT TCT GCA GTA CCG GAT TTG C-3\u2032 (SEQ ID NO:249). PCR primer sequences used for nuc gene sequence amplification were 5\u2032-TCGCTACTAGTTGCTTAGTG-3\u2032 (SEQ ID NO:250) and 5\u2032-GTAAACATAAGCAACTTTAG-3\u2032 (SEQ ID NO:251). MRSA and MSSA target DNA was isolated as described above. PCR reactions were done using the AMPLITAQ DNA Polymerase Kit with GENEAMP (PE Corporation Catalog # N808-0152). Separate reactions were done for the mecA and nuc sequences, and were performed in a 100 \u03bcl final volume containing the following components: 10 \u03bcl of 10\u00d7 PCR buffer, 2.5 \u03bcl of 10 \u03bcM upstream primer and downstream primer, 2 \u03bcl of 10 mM dNTP mix, 1.0 \u03bcl AMPLITAQ DNA polymerase, 2 \u03bcl (10-50ng) of bacterial DNA (the MRSA or the MSSA), and 80 \u03bcl of water for a final volume of 100 \u03bcl. Reactions were covered with approximately 50 \u03bcl of CHILLOUT liquid wax (MJ Research) and cycled as follows: the mecA reactions were denatured at 97\u00b0 C. for 3 minutes. Reactions were then cycled at 97\u00b0 C. for 1 minute, 52\u00b0 C. for 30 seconds, 72\u00b0 C. for 1 minute. This was repeated 5 times. After the final 72\u00b0 C. 1 minute incubation, reactions were again heated to 94\u00b0 C. for 30 seconds, 52\u00b0 C. for 30 seconds and 72\u00b0 C. for 1 minute, for 30 cycles. mecA reactions were then incubated at 72\u00b0 C. for 7 minutes, then held at 4\u00b0 C. until purification.","The nuc amplification reactions were denatured at 97\u00b0 C. for 3 minutes. Reactions were then cycled at 97\u00b0 C. for 1 minute, 48\u00b0 C. for 30 seconds, 72\u00b0 C. for 1 minute. This was repeated for 5 cycles. After the final 72\u00b0 C., 1 minute incubation, reactions were heated to 94\u00b0 C. for 30 seconds, 48\u00b0 C. for 30 seconds and 72\u00b0 C. for 1 minute. This was repeated for 30 cycles. Reactions were then incubated at 72\u00b0 C. for 7 minutes, and finally cooled to 4\u00b0 C. and held until purification.","After amplification, reactions were run on a 1% agarose gel in 1% TBE buffer with 50-2000 base-pair markers (Novagen,Cat# 69278-3); bands were visualized via ethidium bromide staining followed by ultraviolet illumination. The appropriately sized bands were excised from the gel and purified by the QIAquick Gel Extraction Kit (QiagenCat# 28704) according to the manufacturer's protocol. Each column was eluted twice with 50 \u03bcl of elution buffer. The concentration of the purified PCR synthetic target DNA was determined by OD, and diluted to a stock concentration of 50 fmoles per microliter. Positive controls were used at a concentration of 10 amole per reaction with a 10 \u03bcl addition. Control reactions with no target were also performed, using human genomic DNA at 10 ng\/\u03bcl in place of a bacterial sample. All samples were added in a volume of 10 \u03bcl.","INVADER reactions were performed in MJ 96 well Low Profile Polypropylene Microplates (MJ Research MLL-9601) in duplicate in a final volume of 15 \u03bcl. An INVADER reaction master mix was prepared and contained (per reaction) 1.5 \u03bcl water, 2.5 \u03bcl 100 mM MOPS, 5 \u03bcl of 20% PEG (8000 MW), 0.5 \u03bcl of 1 \u03bcM nuc INVADER oligonucleotide and 0.5 \u03bcl of mecA INVADER oligonucleotide. Five \u03bcl of the INVADER master mix were added to each sample and control well. 10 \u03bcl of the target bacterial DNA samples, prepared as described above, or 10 \u03bcl (10 amoles) of positive control target, or 10 \u03bcl (100 ng of human genomic DNA) of the no target control sample were added to each well. Samples were overlaid with 15 \u03bcl clear Chill-out wax (MJ Research, ) and incubated at 95\u00b0 C. for 5 minutes in an MJ Research Thermocycler with Hot Bonnet. Two different probe master mixes were prepared, one containing the mecA probe oligonucleotide, one containing the nuc probe oligonucleotide. The probe master mixes contained (per reaction) 2 \u03bcl of 93.75 mM MgCl, 1 \u03bcl of 10 \u03bcM FRET oligonucleotide, 1 \u03bcl of either 10 \u03bcM mecA probe oligonucleotide or 10 \u03bcM nuc probe oligonucleotide, 1 \u03bcl of 100 ng\/\u03bcl Afu FEN-1 enzyme (Third Wave Technologies, Cat # 96004D). The temperature was cooled to 64\u00b0 C. and 5 \u03bcl of the appropriate probe master mix (such that each control or sample reaction is tested with both the mecA and the nuc probe) was added below the Chill-out wax layer to each well and mixed by pipetting up and down 5 times. The plate was covered with MICROSEAL \u2018A\u2019 Film (MJ Research,) and incubated at 64\u00b0 C. for 30 minutes. After the 30 minute incubation, reactions were cooled to room temperature, placed on a Perkin Elmer MICROAMP base (cat# N801-0531) and read on a CYTOFLUOR Multi-well Plate Reader Series 4000 (PerSeptive Biosystems) using the following parameters: Excitation wavelength\/bandwidth 485 nm\/20 nm, Emission wavelength\/bandwidth 530 nm\/25 nm, gain 40, 30 reads per well, temperature 25\u00b0 C.","Results are shown graphically in , with target DNAs indicated on the horizontal axis and the fluorescence units indicated on the vertical axis. The white bars represent the net averaged signal from the mecA probe, while the dark bars represent the net averaged signal from the nuc probe.","All publications and patents mentioned in the above specification are herein incorporated by reference. Various modifications and variations of the described methods and systems of the invention will be apparent to those skilled in the art without departing from the scope and spirit of the invention. Although the invention has been described in connection with specific preferred embodiments, it should be understood that the invention as claimed should not be unduly limited to such specific embodiments. Indeed, various modifications of the described modes for carrying out the invention which are obvious to those skilled in the relevant fields are intended to be within the scope of the following claims."],"BRFSUM":[{},{}],"heading":["FIELD OF THE INVENTION","BACKGROUND OF THE INVENTION","I. Signal Amplification Technology Methods For Amplification","II. Direct Detection Technology","SUMMARY OF THE INVENTION","DEFINITIONS","DESCRIPTION OF THE INVENTION","I. Detection of Specific Nucleic Acid Sequences Using 5\u2032 Nucleases in an INVADER Directed Cleavage Assay","II. Effect of ARRESTOR Molecules on Signal and Background in Sequential Invasive Cleavage Reactions.","III. Signal Enhancement by Incorporating the Products of an Invasive Cleavage Reaction into a Subsequent Invasive Cleavage Reaction","IV. Fractionation of Specific Nucleic Acids by Selective Charge Reversal","V. Signal Enhancement by Tailing of Reaction Products in the INVADER oligonucleotide-directed Cleavage Assay","VI. Signal Enhancement by Completion of an Activated Protein Binding Site","VII. Generation of 5\u2032 Nucleases Derived From Thermostable DNA Polymerases","VIII. Improved Enzymes for Use in INVADER Oligonucleotide-directed Cleavage Reactions","IX. The INVADER Assay for Direct Detection and Measurement of Specific Analytes.","EXAMPLES","Example 1","Characteristics of Native Thermostable DNA Polymerases","Example 2","Generation of 5\u2032 Nucleases from Thermostable DNA Polymerases","Example 3","5\u2032 Nucleases Derived from Tliermostable DNA Polymerases can Cleave Short Hairpin Structures with Specificity","Example 4","Cleavage of Linear Nucleic Acid Substrates","Example 5","5\u2032 Exonucleolytic Cleavage (\u201cNibbling\u201d) by Thermostable DNAPs","Example 6","Nibbling is Duplex Dependent","Example 7","Nibbling can be Target Directed","Example 8","Cleavase Purification","Example 9","The Use of Various Divalent Cations in the Cleavage Reaction Influences the Nature of the Resulting Cleavage Products","Example 10","Invasive 5\u2032 Endonucleolytic Cleavage by Thermostable 5\u2032 Nucleases in the Absence of Polymerization","Example 11","The INVADER Oligonucleotide Shifts the Site of Cleavage in a Pre-Formed Probe\/Target Duplex","Example 12","The Overlap of the 3\u2032 INVADER Oligonucleotide Sequence with the 5\u2032 Region of the Probe Causes a Shift in the Site of Cleavage","Example 13","INVADER-Directed Cleavage Recognizes Single and Double Stranded Target Molecules in a Background of Non-Target DNA Molecules","Example 14","Signal Accumulation in the INVADER-Directed Cleavage Assay as a Function of Target Concentration","Example 15","Effect of Saliva Extract on the INVADER-Directed Cleavage Assay","Example 16","Comparison of Additional 5\u2032 Nucleases in the INVADER-Directed Cleavage Assay","Example 17","The INVADER-Directed Cleavage Assay can Detect Single Base Differences in Target Nucleic Acid Sequences","Example 18","The INVADER-Directed Cleavage Reaction is Insensitive to Large Changes in Reaction Conditions","Example 19","Detection of RNA Targets by INVADER-Directed Cleavage","Example 20","The Fate of the Target RNA in the INVADER-Directed Cleavage Reaction","Example 21","Titration of Target RNA in the INVADER-Directed Cleavage Assay","Example 22","Detection of DNA by Charge Reversal","Example 23","Detection of Specific Cleavage Products in the INVADER-Directed Cleavage Reaction by Charge Reversal","Example 24","Devices and Methods for the Separation and Detection of Charged Reaction Products","Example 25","Primer-Directed and Primer Independent Cleavage Occur at the Same Site When the Primer Extends to the 3\u2032 Side of a Mismatched \u201cBubble\u201d in the Downstream Duplex","Example 26","Invasive Cleavage and Primer-directed Cleavage when there is Only Partial Homology in the \u201cX\u201d Overlap Region","Example 27","Modified CLEAVASE Enzymes","Example 28","Cloning and Expression of Thermostable FEN-1 Endonucleases","Example 29","Terminal Deoxynucleotidyl Transferase Selectively Extends the Products of INVADER-Directed Cleavage","Example 30","Specific TdT Tailing of the Products of INVADER-Directed Cleavage with Subsequent Capture and Detection on Nitrocellulose Supports","Example 31","Comparison of the Effects of Invasion Length and 5\u2032 Label of the Probe on INVADER-Directed Cleavage by the CLEAVASE A\/G and Pfu FEN-1 Nucleases","Example 32","Examination of the Effects of a 5\u2032 Positive Charge on the Rate of Invasive Cleavage Using the CLEAVASE A\/G or Pfu FEN-1 Nucleases","Example 33","The Use of Universal Bases in the Detection of Mismatches by INVADER Directed Cleavage","Example 34","Detection of Point Mutations in the Human Ras Oncogene Using a Miniprobe","Example 35","Effects of 3\u2032 End Identity on Site of Cleavage of a Model Oligonucleotide Structure","Example 36","Effect of Probe Concentration, Temperature and a Stacker Oligonucleotide on the Cleavage of Miniprobes by INVADER-Directed Cleavage","Example 37","The Presence of a Mismatch in the INVADER Oligonucleotide Decreases the Cleavage Activity of the CLEAVASE A\/G Nuclease","Example 38","Comparison of the Activity of the Pfu FEN-1 and Mja EN-1 Nucleases in the INVADER Reaction","Example 39","Detection of RNA Target Nucleic Acids Using Miniprobe and Stacker Oligonucleotides","Example 40","Effect of an Unpaired 3\u2032 Tail on Transcription from a Complete (Un-Nicked) Promoter","Example 41","Examination of the Influence of the Position of the Nick on the Efficiency of Transcription from Partial and Complete Composite Bacteriophage T7 Promoters","Example 42","Detection of the Products of INVADER-Directed Cleavage through Transcription from a Composite Promoter","Example 43","Shutting Down Transcription from a \u201cLeaky\u201d Branched T7 Composite Promoter through the Use of a Downstream Partial Promoter Oligonucleotide having a 5\u2032 Tail","Example 44","Creation of a Complete Bacteriophage T7 Promoter by DNA Polymerase Mediated Extension of a Cut Probe Comprising a Partial T7 Promoter","Example 45","Test for the Dependence of an Enzyme on the Presence of an Upstream Oligonucleotide","Example 46","Use of the Products of a First Invasive Cleavage Reaction to Enable a Second Invasive Cleavage Reaction with a Net Gain in Sensitivity","Example 47","The Products of a Completed Sequential Invasive Cleavage Reaction Cannot Cross Contaminate Subsequent Similar Reactions","Example 48","Detection of Human Cytomegalovirus Viral DNA by Invasive Cleavage","EXAMPLE 49","\u201cARRESTOR\u201d Oligonucleotides Improve Sensitivity of Multiple Sequential Invasive Cleavage Assays","Example 50","\u201cARRESTOR\u201d Oligonucleotides Allow use of Higher Concentrations of Primary Probe without Increasing Background Signal","Example 51","Modified Backbones Improve Performance of ARRESTOR Oligonucleotides All Natural \u201cARRESTOR\u201d Oligo with No 3\u2032-Amine","Example 52","Example 53","Effect of ARRESTOR Oligonucleotide Concentration on Signal Enhancement in Multiple Sequential Invasive Cleavage Assays","Example 54","Detection of Single Nucleotide Polymorphisms in the Human Apolipoprotein E Gene","Example 55","Detection of Mutations in the Human Hemochromatosis (HFE) Gene","Example 56","Detection of Mutations in the Human MTHFR","Example 57","Detection of Mutations in the Human Prothrombin (Factor II) Gene","Example 58","Detection of the HR-2 Mutation in the Human Factor V Gene","Example 59","Detection of Single Nucleotide Polymorphisms in the Human TNF-\u03b1 Gene","Example 60","Detection of the Factor V Leiden Mutation","Example 61","Detection of Methicillin-Resistant "],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0157","num":"0158"},"figref":"FIGS. 1A-1H","i":["Thermus aquaticus ","Thermus flavus ","Thermus thermophilus "]},{"@attributes":{"id":"p-0158","num":"0159"},"figref":"FIGS. 2A-2C","i":["Thermus aquaticus ","Thermus flavus ","Thermus thermophilus "]},{"@attributes":{"id":"p-0159","num":"0160"},"figref":"FIGS. 3A-G"},{"@attributes":{"id":"p-0160","num":"0161"},"figref":"FIG. 4A","i":"Thermus flavus "},{"@attributes":{"id":"p-0161","num":"0162"},"figref":"FIG. 4B","i":"Thermus flavus "},{"@attributes":{"id":"p-0162","num":"0163"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0163","num":"0164"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0164","num":"0165"},"figref":"FIG. 7"},{"@attributes":{"id":"p-0165","num":"0166"},"figref":"FIGS. 8A-B"},{"@attributes":{"id":"p-0166","num":"0167"},"figref":"FIGS. 9A-B"},{"@attributes":{"id":"p-0167","num":"0168"},"figref":"FIGS. 10A-B"},{"@attributes":{"id":"p-0168","num":"0169"},"figref":"FIG. 11A"},{"@attributes":{"id":"p-0169","num":"0170"},"figref":"FIG. 11B","b":"12"},{"@attributes":{"id":"p-0170","num":"0171"},"figref":"FIG. 12A"},{"@attributes":{"id":"p-0171","num":"0172"},"figref":"FIG. 12B","b":"13"},{"@attributes":{"id":"p-0172","num":"0173"},"figref":"FIG. 13"},{"@attributes":{"id":"p-0173","num":"0174"},"figref":"FIG. 14"},{"@attributes":{"id":"p-0174","num":"0175"},"figref":"FIGS. 15A-E","b":"15"},{"@attributes":{"id":"p-0175","num":"0176"},"figref":"FIG. 16"},{"@attributes":{"id":"p-0176","num":"0177"},"figref":"FIG. 17"},{"@attributes":{"id":"p-0177","num":"0178"},"figref":"FIG. 18A"},{"@attributes":{"id":"p-0178","num":"0179"},"figref":"FIG. 18B","b":"19"},{"@attributes":{"id":"p-0179","num":"0180"},"figref":"FIG. 19"},{"@attributes":{"id":"p-0180","num":"0181"},"figref":["FIGS. 20A and B","FIG. 21"],"i":["Thermus aquaticus ","Thermus flavus. "]},{"@attributes":{"id":"p-0181","num":"0182"},"figref":"FIG. 21A"},{"@attributes":{"id":"p-0182","num":"0183"},"figref":["FIG. 21B","FIG. 21A"]},{"@attributes":{"id":"p-0183","num":"0184"},"figref":"FIGS. 22A and 22B"},{"@attributes":{"id":"p-0184","num":"0185"},"figref":"FIG. 23"},{"@attributes":{"id":"p-0185","num":"0186"},"figref":"FIGS. 24A and B"},{"@attributes":{"id":"p-0186","num":"0187"},"figref":"FIG. 25"},{"@attributes":{"id":"p-0187","num":"0188"},"figref":"FIG. 26"},{"@attributes":{"id":"p-0188","num":"0189"},"figref":"FIG. 27"},{"@attributes":{"id":"p-0189","num":"0190"},"figref":"FIG. 28"},{"@attributes":{"id":"p-0190","num":"0191"},"figref":"FIG. 29"},{"@attributes":{"id":"p-0191","num":"0192"},"figref":"FIG. 30","b":"28"},{"@attributes":{"id":"p-0192","num":"0193"},"figref":"FIG. 31"},{"@attributes":{"id":"p-0193","num":"0194"},"figref":"FIG. 32"},{"@attributes":{"id":"p-0194","num":"0195"},"figref":"FIG. 33"},{"@attributes":{"id":"p-0195","num":"0196"},"figref":"FIG. 34"},{"@attributes":{"id":"p-0196","num":"0197"},"figref":"FIG. 35"},{"@attributes":{"id":"p-0197","num":"0198"},"figref":"FIG. 36A"},{"@attributes":{"id":"p-0198","num":"0199"},"figref":"FIG. 36B"},{"@attributes":{"id":"p-0199","num":"0200"},"figref":"FIG. 37"},{"@attributes":{"id":"p-0200","num":"0201"},"figref":"FIG. 38"},{"@attributes":{"id":"p-0201","num":"0202"},"figref":"FIG. 39","sub":["2 ","2"]},{"@attributes":{"id":"p-0202","num":"0203"},"figref":"FIG. 40"},{"@attributes":{"id":"p-0203","num":"0204"},"figref":"FIG. 41"},{"@attributes":{"id":"p-0204","num":"0205"},"figref":"FIGS. 42A and 42B"},{"@attributes":{"id":"p-0205","num":"0206"},"figref":"FIG. 43"},{"@attributes":{"id":"p-0206","num":"0207"},"figref":"FIG. 44"},{"@attributes":{"id":"p-0207","num":"0208"},"figref":"FIG. 45"},{"@attributes":{"id":"p-0208","num":"0209"},"figref":"FIG. 46"},{"@attributes":{"id":"p-0209","num":"0210"},"figref":"FIG. 47"},{"@attributes":{"id":"p-0210","num":"0211"},"figref":"FIG. 48"},{"@attributes":{"id":"p-0211","num":"0212"},"figref":"FIG. 49A"},{"@attributes":{"id":"p-0212","num":"0213"},"figref":"FIG. 49B"},{"@attributes":{"id":"p-0213","num":"0214"},"figref":"FIG. 50"},{"@attributes":{"id":"p-0214","num":"0215"},"figref":"FIG. 51"},{"@attributes":{"id":"p-0215","num":"0216"},"figref":"FIG. 52"},{"@attributes":{"id":"p-0216","num":"0217"},"figref":"FIG. 53"},{"@attributes":{"id":"p-0217","num":"0218"},"figref":"FIGS. 54A-D","b":"54"},{"@attributes":{"id":"p-0218","num":"0219"},"figref":"FIG. 55"},{"@attributes":{"id":"p-0219","num":"0220"},"figref":"FIG. 56"},{"@attributes":{"id":"p-0220","num":"0221"},"figref":"FIG. 57"},{"@attributes":{"id":"p-0221","num":"0222"},"figref":"FIG. 58"},{"@attributes":{"id":"p-0222","num":"0223"},"figref":"FIGS. 59A-59E","i":["Methanococcus jannaschii ","Pyrococcus furiosus ","Saccharomyces cerevisiae ","Saccharomyces cerevisiae ","Shizosaccharomyces pombe ","Xenopus laevis ","C. elegans "]},{"@attributes":{"id":"p-0223","num":"0224"},"figref":"FIG. 60"},{"@attributes":{"id":"p-0224","num":"0225"},"figref":"FIG. 61"},{"@attributes":{"id":"p-0225","num":"0226"},"figref":"FIG. 62"},{"@attributes":{"id":"p-0226","num":"0227"},"figref":"FIG. 63"},{"@attributes":{"id":"p-0227","num":"0228"},"figref":"FIG. 64"},{"@attributes":{"id":"p-0228","num":"0229"},"figref":"FIG. 65"},{"@attributes":{"id":"p-0229","num":"0230"},"figref":"FIG. 66"},{"@attributes":{"id":"p-0230","num":"0231"},"figref":"FIG. 67"},{"@attributes":{"id":"p-0231","num":"0232"},"figref":"FIG. 68"},{"@attributes":{"id":"p-0232","num":"0233"},"figref":"FIG. 69"},{"@attributes":{"id":"p-0233","num":"0234"},"figref":"FIG. 70"},{"@attributes":{"id":"p-0234","num":"0235"},"figref":"FIG. 71"},{"@attributes":{"id":"p-0235","num":"0236"},"figref":["FIGS. 72A and 72B","FIG. 83A","FIG. 84B"]},{"@attributes":{"id":"p-0236","num":"0237"},"figref":"FIG. 73"},{"@attributes":{"id":"p-0237","num":"0238"},"figref":"FIG. 74"},{"@attributes":{"id":"p-0238","num":"0239"},"figref":"FIGS. 75A-C"},{"@attributes":{"id":"p-0239","num":"0240"},"figref":"FIG. 76"},{"@attributes":{"id":"p-0240","num":"0241"},"figref":"FIG. 77"},{"@attributes":{"id":"p-0241","num":"0242"},"figref":"FIGS. 78A-C"},{"@attributes":{"id":"p-0242","num":"0243"},"figref":"FIG. 79"},{"@attributes":{"id":"p-0243","num":"0244"},"figref":"FIG. 80"},{"@attributes":{"id":"p-0244","num":"0245"},"figref":"FIG. 81"},{"@attributes":{"id":"p-0245","num":"0246"},"figref":["FIG. 82","FIG. 92"]},{"@attributes":{"id":"p-0246","num":"0247"},"figref":"FIG. 83"},{"@attributes":{"id":"p-0247","num":"0248"},"figref":"FIG. 84"},{"@attributes":{"id":"p-0248","num":"0249"},"figref":"FIGS. 85A-C"},{"@attributes":{"id":"p-0249","num":"0250"},"figref":"FIGS. 86A-D"},{"@attributes":{"id":"p-0250","num":"0251"},"figref":"FIG. 87"},{"@attributes":{"id":"p-0251","num":"0252"},"figref":"FIGS. 88A and 88B"},{"@attributes":{"id":"p-0252","num":"0253"},"figref":"FIG. 89"},{"@attributes":{"id":"p-0253","num":"0254"},"figref":"FIGS. 90A-C"},{"@attributes":{"id":"p-0254","num":"0255"},"figref":"FIG. 91"},{"@attributes":{"id":"p-0255","num":"0256"},"figref":"FIG. 92"},{"@attributes":{"id":"p-0256","num":"0257"},"figref":"FIG. 93"},{"@attributes":{"id":"p-0257","num":"0258"},"figref":"FIG. 94"},{"@attributes":{"id":"p-0258","num":"0259"},"figref":"FIGS. 95A-D"},{"@attributes":{"id":"p-0259","num":"0260"},"figref":"FIG. 96"},{"@attributes":{"id":"p-0260","num":"0261"},"figref":"FIG. 97"},{"@attributes":{"id":"p-0261","num":"0262"},"figref":"FIG. 98"},{"@attributes":{"id":"p-0262","num":"0263"},"figref":["FIGS. 99A-99E","FIGS. 99A-99E"]},{"@attributes":{"id":"p-0263","num":"0264"},"figref":"FIG. 100"},{"@attributes":{"id":"p-0264","num":"0265"},"figref":"FIG. 101","i":"a "},{"@attributes":{"id":"p-0265","num":"0266"},"figref":"FIG. 101","i":"b "},{"@attributes":{"id":"p-0266","num":"0267"},"figref":"FIG. 102"},{"@attributes":{"id":"p-0267","num":"0268"},"figref":["FIG. 103","FIG. 103"]},{"@attributes":{"id":"p-0268","num":"0269"},"figref":"FIG. 104"},{"@attributes":{"id":"p-0269","num":"0270"},"figref":"FIG. 105"},{"@attributes":{"id":"p-0270","num":"0271"},"figref":"FIG. 106"},{"@attributes":{"id":"p-0271","num":"0272"},"figref":"FIGS. 107A-107C"},{"@attributes":{"id":"p-0272","num":"0273"},"figref":"FIG. 108A"},{"@attributes":{"id":"p-0273","num":"0274"},"figref":"FIG. 108B"},{"@attributes":{"id":"p-0274","num":"0275"},"figref":"FIG. 108C"},{"@attributes":{"id":"p-0275","num":"0276"},"figref":"FIG. 109A"},{"@attributes":{"id":"p-0276","num":"0277"},"figref":"FIG. 109B"},{"@attributes":{"id":"p-0277","num":"0278"},"figref":"FIG. 110A"},{"@attributes":{"id":"p-0278","num":"0279"},"figref":["FIG. 110B","FIG. 110A"]},{"@attributes":{"id":"p-0279","num":"0280"},"figref":"FIG. 110C"},{"@attributes":{"id":"p-0280","num":"0281"},"figref":"FIG. 111"},{"@attributes":{"id":"p-0281","num":"0282"},"figref":"FIG. 112A"},{"@attributes":{"id":"p-0282","num":"0283"},"figref":"FIG. 113A"},{"@attributes":{"id":"p-0283","num":"0284"},"figref":"FIG. 113B"},{"@attributes":{"id":"p-0284","num":"0285"},"figref":"FIG. 113C"},{"@attributes":{"id":"p-0285","num":"0286"},"figref":"FIG. 113D"},{"@attributes":{"id":"p-0286","num":"0287"},"figref":"FIG. 114A"},{"@attributes":{"id":"p-0287","num":"0288"},"figref":"FIG. 114B"},{"@attributes":{"id":"p-0288","num":"0289"},"figref":"FIG. 115A"},{"@attributes":{"id":"p-0289","num":"0290"},"figref":"FIG. 115B"},{"@attributes":{"id":"p-0290","num":"0291"},"figref":"FIG. 115C"},{"@attributes":{"id":"p-0291","num":"0292"},"figref":"FIG. 115D"},{"@attributes":{"id":"p-0292","num":"0293"},"figref":"FIG. 116"},{"@attributes":{"id":"p-0293","num":"0294"},"figref":"FIG. 117A"},{"@attributes":{"id":"p-0294","num":"0295"},"figref":"FIG. 117B"},{"@attributes":{"id":"p-0295","num":"0296"},"figref":"FIG. 118"},{"@attributes":{"id":"p-0296","num":"0297"},"figref":"FIG. 119A"},{"@attributes":{"id":"p-0297","num":"0298"},"figref":"FIG. 119B"},{"@attributes":{"id":"p-0298","num":"0299"},"figref":"FIG. 120"},{"@attributes":{"id":"p-0299","num":"0300"},"figref":"FIG. 121A"},{"@attributes":{"id":"p-0300","num":"0301"},"figref":"FIG. 121B"},{"@attributes":{"id":"p-0301","num":"0302"},"figref":"FIG. 122"},{"@attributes":{"id":"p-0302","num":"0303"},"figref":"FIG. 123A"},{"@attributes":{"id":"p-0303","num":"0304"},"figref":"FIG. 123B"},{"@attributes":{"id":"p-0304","num":"0305"},"figref":"FIG. 124"},{"@attributes":{"id":"p-0305","num":"0306"},"figref":"FIG. 125A"},{"@attributes":{"id":"p-0306","num":"0307"},"figref":"FIG. 125B"},{"@attributes":{"id":"p-0307","num":"0308"},"figref":"FIG. 126"},{"@attributes":{"id":"p-0308","num":"0309"},"figref":"FIG. 127A","i":"S. aureus. "},{"@attributes":{"id":"p-0309","num":"0310"},"figref":"FIG. 127B","i":["S. aureus ","S. haemolyticus. "]},{"@attributes":{"id":"p-0310","num":"0311"},"figref":"FIG. 128","i":["S. aureus","S. aureus ","S. aureus ","S. haemolyticus"]}]},"DETDESC":[{},{}]}
