---
title: Chitobiase as a reporter enzyme
abstract: The present invention relates to reporter gene constructs encoding a cytoplasmic form of chitobiase (N,N′-diacetylchitobiase) and methods of using these reporter gene constructs. The use of a cytoplasmic form of chitobiase as a reporter enzyme is generally applicable to the study of gene expression in organisms which do not contain N-acetyl-β-D-glucosaminidases.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=06838239&OS=06838239&RS=06838239
owner: San Diego State University Foundation
number: 06838239
owner_city: San Diego
owner_country: US
publication_date: 20000802
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["RELATED APPLICATIONS","GOVERNMENT INTEREST IN THE INVENTION","BACKGROUND OF THE INVENTION","Field of the Invention","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENT","EXAMPLES","Example 1","Construction of Vectors","Example 2","Site-specific Recombination","Example 3","Bacteriophage P1 Transduction","Example 4","p-Nitrophenol Chitobiase Activity Assay","Example 5","Determination of Chitobiase Activity in Cells Containing Vectors pJM3 and pJM4","Example 6","Use of the Chitobiase Reporter Enzyme to Study dnaA Gene regulation","Example 7","Identification of Promoters in Test Sequences","Example 8","Detecting Successful Transformation or Transfection Using Chitobiase","Conclusion"],"p":["This application claims priority to U.S. Provisional Patent Application Ser. No. 60\/159,221, filed Oct. 13, 1999, the disclosure of which is incorporated herein by reference in its entirety.","Work performed in developing the present invention was supported, in part, by National Science Foundation grant MCB-9507209, NIH MBRS grant GM45765, and NIH NIGMS MARC F31 GM14967-0451. Accordingly, the United States Government may have certain rights in the invention.","Reporter enzymes are enzymes whose activities are easily assayed when present inside cells. In order to study the regulation of a gene whose expression is regulated by various environmental and\/or cellular factors or influences, a gene encoding a reporter enzyme may be fused to the coding region or to the regulatory region of the regulated gene.","One of the most popular cytoplasmic reporter enzymes for use in bacteria is \u03b2-galactosidase. It is widely used in the art; however, because bacteria such as contain an endogenous \u03b2-galactosidase encoding gene and \u03b2-galactosidase may be present in the cytoplasm of such bacteria, deletions of the LacZ gene, the source of the enzyme, must be introduced into the host cell line prior to its use. One goal of the present invention is to provide an alternative intracellular enzyme for use as a reporter.","This invention relates to genetic constructs and methods of using a cytoplasmic form of the chitobiase enzyme as a reporter. The invention also comprises expression vectors which express the cytoplasmic form of chitobiase. As used herein, all instances of the terminology \u201cchitobiase\u201d refer to a form of chitobiase which is present in the cytoplasm of the cell. Cytoplasmic forms of chitobiase may be generated via genetic engineering or microbial selection techniques.","One embodiment of the present invention is a method for characterizing a promoter comprising providing a construct comprising the promoter operably linked to a nucleic acid encoding a cytoplasmic form of chitobiase, introducing the construct into host cells, and identifying sequences in the promoter which regulate transcription levels. In one aspect of this embodiment, the cytoplasmic form of chitobiase lacks a signal sequence. In another aspect of this embodiment, the nucleic acid encoding a cytoplasmic form of chitobiase encodes a fusion protein which comprises a cytoplasmic form of chitobiase fused to a heterologous polypeptide. In still another aspect of this embodiment, the nucleic acid encoding a cytoplasmic form encodes a cytoplasmic form of chitobiase obtained from an organism selected from the group consisting of Alteromonas sp. 0-7, Pseudoalteromonas sp. S9, and . In a further aspect of this embodiment, the method of identifying sequences which are involved in directing transcription comprises mutagenizing the promoter. In another aspect of this embodiment, the method of identifying sequences which are involved in transcription comprises constructing deletions in the promoter.","Another embodiment of the present invention is a method for identifying a regulatory element capable of directing or regulating transcription within a test nucleic acid sequence comprising providing a construct comprising the test nucleic acid sequence operably linked to a nucleic acid encoding a cytoplasmic form of chitobiase, introducing the construct into host cells, and determining the level of chitobiase activity. In one aspect of this embodiment, the cytoplasmic form of chitobiase lacks a signal sequence. In another aspect of this embodiment, the nucleic acid encoding a cytoplasmic form of chitobiase encodes a fusion protein, the fusion protein comprising a cytoplasmic form of chitobiase fused to a heterologous polypeptide. In still another aspect of this embodiment, the nucleic acid encoding a cytoplasmic form encodes a cytoplasmic form of chitobiase obtained from an organism selected from the group consisting of Alteromonas sp. 0-7, Pseudoalteromonas sp. S9. In one aspect of this embodiment, the reporter gene construct is introduced transiently. In another aspect of this embodiment, the reporter gene construct is introduced stably. The host cells may be selected from the group consisting of prokaryotic cells and eukaryotic cells. In another aspect of this embodiment, the method further comprises permeabilizing or lysing the host cells. The permeabilizing or lysing step may comprise treating the host cells with toluene. The step of determining the level of chitobiase activity may be selected from the group consisting of measuring the amount of a chemiluminescent product produced from a substrate, measuring the amount of a fluorescent product produced from a substrate, measuring the amount of light absorbed by a product produced from a substrate and measuring a decrease in the amount of a detectable substrate. In another embodiment, the step of determining the level of chitiobiase activity may comprise determining the level of p-nitrophenol released from a substrate. In another aspect of this embodiment, the test nucleic acid sequence comprises a portion of genomic DNA. In a further aspect of this embodiment, the step of determining the level of chitobiase activity comprises determining the level of chitobiase activity after exposing the host cells to a desired set of environmental conditions. In still another aspect of this embodiment, the step of determining the level of chitobiase activity comprises determining the level of chitobiase activity after contacting the host cells with a compound to be tested for its influence on the level of transription from siad regulartory element.","Another embodiment of the present invention is a method of detecting successful transformation, comprising the steps of introducing a nucleic acid encoding a cytoplasmic form of chitobiase into host cells, and detecting chitobiase expression in the host cells.","Another embodiment of the present invnetion is a fusion protein-reporter gene construct comprising a promoter operably linked to a nucleic acid encoding a cytoplasmic form of chitobiase fused in frame with a nucleic acid encoding a heterologous polypeptide, wherein the heterologous polypeptide is not p-galactosidase or a portion thereof, and wherein the heterologous polypeptide does not contain a signal peptide. In one aspect of this embodiment, the nucleic acid encodes a cytoplasmic form of chitobiase obtained from an organism selected from the group consisting of Alteromonas sp. 0-79, and . In another aspect of this embodiment, the nucleic acid further comprises a \u03bb site-specific recombination sequence.","Another embodiment of the present invention is a reporter gene construct comprising plasmid pJMF3.","Another embodiment of the present invention is a reporter gene construct comprising plasmid pJMF4.","Another embodiment of the present invention is a reporter gene construct comprising plasmid pDYK9.","Another embodiment of the present invention is a reporter gene construct comprising plasmid pDYK11.","Another embodiment of the present invention is a host cell comprising a fusion protein-reporter gene construct comprising a promoter operably linked to a nucleic acid encoding a cytoplasmic form of chitobiase fused in frame with a nucleic acid encoding a heterologous polypeptide, wherein the heterologous polypeptide is not \u03b2-galactosidase or a portion thereof, and wherein the heterologous polypeptide does not contain a signal peptide. In one aspect of this embodiment, the nucleic acid is integrated into a chromosome of the cell. In another aspect of this embodiment, the nucleic acid is transiently expressed in the host cell.","Another embodiment of the present invention is a nucleic acid encoding a cytoplasmic form of chitobiase in which the signal sequence of native chitobiase has been inactivated or deleted. In one aspect of this embodiment, the signal sequence has been mutated to inactivate it.","An isolated or purified polypeptide comprising a cytoplasmic form of chitobiase fused in frame with a heterologous polypeptide, wherein the heterologous polypeptide is not \u03b2-galactosidase or a portion thereof and wherein the heterologous polypeptide does not contain a signal peptide.","Another embodiment of the present invention is an isolated or purified polypeptide comprising a cytoplasmic form of chitobiase in which the signal peptide of native chitobiase has been inactivated or deleted. In one aspect of this embodiment, the signal sequence has been mutated to inactivate it.","Another embodiment of the present invention is a method for monitoring the activity of a promoter comprising providing a construct comprising the promoter operably linked to a nucleic acid encoding a cytoplasmic form of chitobiase, introducing the construct into host cells, and determining the level of chitobiase activity. In one aspect of this embodiment, the cytoplasmic form of chitobiase lacks a signal sequence. In another aspect of this embodiment, the nucleic acid encoding a cytoplasmic form of chitobiase encodes a fusion protein, the fusion protein comprising a cytoplasmic form of chitobiase fused to a heterologous polypeptide. In one aspect of this embodiment, the nucleic acid encoding a cytoplasmic form encodes a cytoplasmic form of chitobiase obtained from an organism selected from the group consisting of Alteromonas sp. 0-7, Pseudoalteromonas sp. S9, and . In another aspect of this embodiment, the reporter gene construct is introduced transiently. In still another aspect of this embodiment, the reporter gene construct is introduced stably. In a further aspect of this embodiment, the host cells are selected from the group consisting of prokaryotic cells and eukaryotic cells. In yet another aspect of this embodiment, the method further comprises permeabilizing or lysing the host cells. For example, the permeabilizing or lysing step may comprise treating the host cells with toluene. In yet another aspect of this embodiment, the step of determining the level of chitobiase activity may be selected from the group consisting of measuring the amount of a chemiluminescent product produced from a substrate, measuring the amount of a fluorescent product produced from a substrate, measuring the amount of light absorbed by a product produced from a substrate and measuring a decrease in the amount of a detectable substrate. In still another aspect of this embodiment, the step of determining the level of chitiobiase activity comprises determining the level of p-nitrophenol released from a substrate. In still another aspect of this embodiment, the step of determining the level of chitobiase activity comprises determining the level of chitobiase activity after exposing the host cells to a desired set of environmental conditions. In a further aspect of this embodiment, the step of determining the level of chitobiase activity comprises determining the level of chitobiase activity after contacting the host cells with a compound to be tested for its influence on the level of transription from siad regulartory element. For example, the compound may comprise a compound to be tested for activity as a drug.","Reporter genes and reporter gene constructs play a number of important roles in a variety of molecular biology techniques. For example, reporter genes may be used to determine whether a sequence contains a promoter or other cis-acting element which directs transcription, such as an enhancer. In addition, reporter genes may be used to identify regulatory sites in promoters or other cis-acting elements and to determine the effects of mutating these regulatory sites on the level of gene expression directed by the promoters or other cis-acting elements. Reporter genes may also be used to detect successful transformation. In addition, reporter genes may be used to monitor gene expression under various conditions and to identify drugs.","The structure of a reporter gene construct containing the cytoplasmic form of chitobiase will vary according to its purposes. When the reporter gene construct is a vector, one must decide between a vector that incorporates the reporter gene into the host's genome or one that replicates extrachromosomally, such as a plasmid. When the reporter construct is not designed to integrate into the host genome, the vector can contain an origin of replication with activity in the host cell of interest. This feature provides the reporter gene vector the ability to replicate within the host cell in which it has been introduced. In addition to the origin of replication, reporter constructs often contain a promoter, a multiple cloning site, a selectable marker, and of course a reporter gene. Reporter constructs for use in eukaryotic cells may also contain a polyA site adjacent to the reporter gene.","Given the utility of reporter gene constructs, it is not surprising that a number of cytoplasmic reporter gene constructs and different reporter genes are available for use by those of skill in the art. For example, the cytoplasmic reporter enzymes chloramphenicol acetyltransferase (CAT), firefly luciferase, \u03b2-glucuronidase (GUS), green fluorescent protein (GFP), and \u03b2-galactosidase have been used extensively. However, such reporters all have individual shortcomings that may limit or preclude their usage under some conditions. For example, high levels of GFP are toxic to the cell. In addition, reporter enzymes are not expressed equally in all cell types nor are they equally stable when expressed in all cell types. Furthermore, there is a recognized need for multiple reporter enzymes that can be assayed independently of one another in order to simultaneously study the regulation of multiple genes within a single cell type. Therefore, there exists a continuing need to identify reporter enzymes with useful properties.","The cytoplasmic enzyme \u03b2-galactosidase is widely used as a reporter gene in various microbiological and molecular biological studies. This enzyme is used in both in vitro and in vivo assays. The wide acceptance of this reporter system results, in part, because it is non-isotopic and extremely flexible. It is used in a number of assay formats and has an extremely broad liner range. Nevertheless, because \u03b2-galactosidase is present in the cytoplasm of various host cells such as , deletion of the lacZ gene, the source of the enzyme, is often required prior to its use in a host cell system. One goal of the present invention was to provide an alternative intracellular enzyme for use as a reporter.","An extensive discussion of various molecular biology techniques is available in Ausubel, et al., (eds) \u201cShort Protocols in Molecular Biology,\u201d Wiley and Sons, Inc., New York (1997), the disclosures of which are incorporated herein by reference in their entireties. Examples of such techniques include isolating and preparing DNA for manipulation, gel electrophoresis, polymerase chain reaction (PCR), determining nucleic acid sequences, screening nucleic acid libraries, mutagenesis of DNA, and introducing DNA into host cells.","The structure of a reporter gene construct will vary according to its purposes. The reporter gene constructs are constructed according to standard techniques of molecular biology well known in the art.","When the reporter gene construct is a vector, one must decide between a vector that incorporates the reporter gene into the host's genome or one that replicates extrachromosomally, such as a plasmid. When integration of the reporter gene is a desired result, a reporter gene construct will contain sequences that will facilitate incorporation.","One example of integration sequences that can be included in a reporter gene construct is the \u03bb attP site. This site permits a single copy of the reporter gene to be incorporated into a host bacterial genome. Integration-promoting sequences with utility in mammalian cells include the long terminal repeats found in retroviral genomes. These sequences promote viral genome integration in a host genome and have been used extensively by those of skill in the art to promote the integration of exogenous sequences in mammalian host cells.","When the reporter construct is not designed to integrate into the host genome, it is common that the vector contain an origin of replication with activity in the host cell of interest. This feature provides the reporter gene vector the ability to replicate within the host cell in which it has been introduced.","In addition to the origin of replication, reporter constructs will often contain additional features that promote the expression of the nucleic acid sequence or sequences contained in the construct. These additional sequences can include a polyA site, a multiple cloning site, a drug resistance marker, and of course a reporter gene.","The present invention relates to the use of chitobiase as a reporter gene. The chitobiase may be used as a reporter in bacteria, plants, mammalian cells and other host cell lines. One possible alternative to using \u03b2-galactosidase as a reporter gene was to develop cytoplasmic N,N\u2032-diacetylchitobiase (N-acetyl-\u03b2-D-glucosaminidase, EC 3.2.1.30) for use as a reporter enzyme. One advantage of the enzyme N-acetyl-\u03b2-D-glucosaminidase or \u201cchitobiase\u201d over \u03b2-galactosidase is that genes encoding chitobiase are missing from many bacteria, including , some fungi, and some eukaryotic cells. Thus, it is not necessary to engineer many host cells to lack reporter activity as is the case with \u03b2-galactosidase. The present invention also relates to various protein expression vectors that can be used to express the reporter gene. In addition, the present invention may be used in conjunction with other reporter enzymes in systems in which the regulation or activities of multiple genes is to be studied simultaneously.","Chitobiase is one of two enzymes that hydrolyze chitin, an abundant insoluble polysaccharide, to its monomeric unit, N-acetylglucosamine (GlcNac). Chitobiase is known to be present in a number of organisms. For example, the chitobiase enzyme is known to be present in various genera including Arabidopsis, Bacillus, Bombyx, Bos, Caenorhabditis, Candida, Dictyostelium, Entamoeba, Felis, Homo, Korat, Lactobacillus, Leishmania, Mus, Pisum, Porphyromonas, Pseudoalteromonas, Rattus, Serratia, Streptomyces, Sus, Trichoderma, and Vibrio. Specific examples of organisms known to contain chitobiase include Alteromonas sp. 0-7, Pseudoalteromonas sp. S9, and ","One source of the enzyme is the marine bacterium, cells harboring a plasmid carrying the chb gene from were reported to produce the enzyme, which was found to be associated with the outer membrane of the bacterial cells. These are described in Jannatipour, M. et al., \u201cTranslocation of N,N\u2032-diacetylchitobiase to the outer membrane of .,\u201d J. Bacteriol. 169:3785-3791 (1987) and Soto-Gil & Zyskind, N,N\u2032-Diacetylchitobiase of primary structure, processing, and evolutionary relationships. J. Biol Chem. 264:14778-14782 (1989), both of which are hereby incorporated by reference.","The present invention contemplates expressed cytoplasmic forms of chitobiase in various forms. In one embodiment, the signal sequence is deleted from the amino terminal portion of the protein. Presumably the removal of this sequence results in the expression of a cytoplasmic form of the enzyme that is not secreted from the host cell or incorporated into the membrane of the host cell producing the enzyme.","The present invention also contemplates the generation of fusion proteins comprising a fusion polypeptide joined in frame to chitobiase. Preferably, the fusion polypeptide comprises a polypeptide other than chitobiase, such as a heterologous protein. The heterologous polypeptide may comprise a polypeptide having a biological activity (such as an enzymatic or other activity besides activity as an immunogen) or the heterologous polypeptide may not have a biological activity. The heterologous polypeptide does not include a signal sequence which directs its secretion. Preferably the heterologous polypeptide is not P-galactosidase or a portion thereof. Thus, the fusion reporter gene construct contains a sequence encoding the fusion polypeptide genetically fused in frame with a sequence encoding chitobiase. In one embodiment, this fusion may remove the amino-terminal signal peptide sequence of chitobiase and replace it with a heterologous protein.","In another embodiment, the fusion protein construct comprises a chitobiase gene sequence that has been truncated to remove at least the signal peptide sequence of the gene. Alternatively, mutations may be introduced into the signal peptide sequence so that it is no longer functional. Such mutations may be introduced using a variety of techniques familiar to those skilled in the art, including site directed mutagenesis, cassette mutagenesis, and chemical mutagenesis.","Once the reporter gene construct is made it is introduced into a host cell line for testing. Host cells of prokaryotic and eukaryotic origin can be used with the reporter gene constructs of the present invention. A variety of methods are available to introduce reporter gene constructs into host prokaryotic cells. For example, bacteria can be transformed using calcium chloride, electroporation, or viral vectors such as the filamentous phages. These and other prokaryotic transformation protocols are well known in the art.","Alternatively, the sequence encoding chitobiase may be introduced in eukaryotic cells, including yeast, mammalian, plant, and insect cells. For example, the sequence encoding chitobiase may be inserted into a yeast artificial chromosome, a yeast plasmid, a bovine papilloma virus vector or other extrachromosomal vector, a retroviral vector, a Ti-plasmid, or a baculovirus vector. A variety of such vectors are known to those skilled in the art. The vectors may be introduced into any of the yeast, mammalian, plant, and insect cells familiar to those skilled in the art.","The introduction of the reporter gene construct into mammalian cells can likewise utilize a number of transfection protocols well known to those of skill in the art. As discussed above, transfections can be transient or stable. Examples of suitable transfer protocols include calcium phosphate transfection, DEAE-Dextran, electroporation, liposome-mediated transfection, and viral transfection. These and other eukaryotic transformation protocols are well known in the art.","Following introduction of the reporter gene construct into the host cell of interest, the enzymatic activity of the reporter gene is measured. Preferably, the chitobiase assays are performed after permeabilizing or lysing the host cells. There are a variety of cell permeabilization and cell lysis procedures available to those of ordinary skill in the art, including methods such as sonication or lysozyme treatment. One embodiment of the present invention uses toluene treatment to permeabilize cells. The details of this method are discussed in D. Y. Kalabat et al., BioTechniques 25:1030-1035 (1998) and Miller, J. H. A Short Course in Bacterial Genetics, CSH Laboratory Press, Cold Spring Harbor, N.Y. 1992, the disclosures of which are incorporated herein by reference in their entireties.","Cellular chitobiase activity can be measured quantitatively by following the hydrolysis of chitobiase substrates. Examples of substrates with utility in chitobiase activity assays include N,N\u2032-diacetylchitobiose (chitobiose), p-nitrophenyl-N-acetyl-\u03b2-D-glucosaminide (PNAG)(Sigma Chemical, St. Louis, Mo.), and 5-bromo-4chloro-3-indolyl-N-acetyl-\u03b2-D-glucosaminide (X-Gluc)(Sigma Chemical, St. Louis, Mo.). Other substrates are also contemplated for use in the assays of the present invention.","Products produced by the hydrolysis of the chitobiase substrates are monitored using various means familiar to those skilled in the art. For example, various optical means are known to those skilled in the art. One such optical means may comprise detection of chemiluminescent or fluorescent products released from a substrate. Alternatively, the level of chitobiase activity may be determined by measuring the amount of light absorbed by a product produced from a substrate or measuring a decrease in the amount of a detectable substrate. In one embodiment, P-nitrophenol is released from the substrate and measured at 400 nm. Other monitoring methods well known in the art can be used to quantitate signals produced in the chitobiase assay. These may inlcude use of radioactive substrates or substrates having radiofrequency tags. In another embodiment, blue\/white colony indicator plates are used to monitor enzyme activity.","Another embodiment of the present invention is a kit. One aspect of this embodiment includes a reporter gene construct comprising a vector containing a chitobiase reporter gene. The reporter gene construct also contains a multiple cloning site containing a variety of restriction endonuclease cutting sites that facilitate the introduction of exogenous DNA into the construct.","The kit embodiment of the present invention also includes those components necessary to assay for chitobiase activity produced by the reporter gene construct. For example, in one embodiment, the kit will include a supply of a suitable chitobiase substrate whose metabolism into product by the reporter enzyme can be assayed.","The following Examples are disclosed to assist in the understanding of the present invention. The Examples below should not be construed to limit the scope of the invention and such variations of the invention now known or later developed, which would be within the purview of one of ordinary skill in the art and are considered to fall within the scope of the invention hereinafter claimed.","Vectors were constructed using polymerase chain reaction (PCR) products that were cloned first at the SmaI or EcoRV restriction sites of plasmid pBluescript II (pKSII+)(Stratagene; San Diego, Calif.). The nucleotide sequence of all PCR products was determined using standard techniques well known in the art. Unmethylated plasmid DNA was isolated from an dam strain when cutting with the BclI enzyme was required.","The pDYK9 (SEQ ID NO: 11) plasmid was constructed by ligating a SphI-KpnI PCR product containing the dnaA promoter region into plasmid pRSG196 that contains the chitobiase gene (see Jannatipour, M., et al.). Briefly, the chitobiase gene was prepared by cloning a Sau3A partial digest of DNA into the single BamHI site with the tee gene of pMK2004 [See Soto-Gill & Zyskind, \u201cCloning of chitinase and chitobiase genes in ,\u201d pp. 209-223, In J. P. Zikakis (ed.), Chitin, chitosan, and related enzymes,\u201d Academic Press, Inc., New York (1984)]. The clones containing the gene of interest were detected by the presence of the yellow p-nitrophenylate product after individual colonies of the clone bank were sprayed with 10 mM PNAG dissolved in 100 mM sodium phosphate, pH 7.0. Restriction maps of the positive clones were made according to standard techniques well known in the art.","The chb gene was subcloned in a 3.5-kb EcoRI fragment from pRSG14 [See Jannatipour, M. et al., \u201cTranslocation of N,N\u2032-diacetylchitobiase to the outer membrane of .,\u201d J. Bacteriol. 169:3785-3791 (1987)] into the EcoRi site of pUC19 [Yanisch-Perron, c.J. et al., Gene 33:103-119 (1985)]. Plasmid pRSG196 was constructed by deleting a 0.5-kb SstI fragment from one of these clones [See Jannatipour, M. et al., \u201cTranslocation of N,N\u2032-diacetylchitobiase to the outer membrane of .,\u201d J. Bacteriol. 169:3785-3791 (1987)].","The pRSG196 plasmid was also cut with SphI and KpnI to accommodate the insertion of the dnaA fragment. Plasmid pAC17 [described in Chiaramello & Zyskind, \u201cCoupling of DNA replication to growth rate in : a possible role for guanosine tetraphosphate,\u201d J. Bacteriol, 172:2013-2019 (1992) hereby incorporated by reference] served as template with primers 5\u2032-GCA CAT GCA TGC TGG TCA TTA AAT TTT CC-3\u2032 (SEQ ID NO 1) and 5\u2032-CGG GGT ACC AAC TCA TCC TGC AAT CG-3\u2032 (SEQ ID NO 2) producing a PCR product 374 bp long that contains 353 bp from the dnaA promoter region [bases 583 to 935, numbering according to Hansen, F. G., et al., \u201cThe nucleotide sequence of the dnaA gene promoter and of the adjacent rpmH gene, coding for the ribosomal protein L34, of ,\u201d EMBO J. 1:1043-1048 (1982)]. The forward primer (SphI primer II) contained an SphI site and the reverse primer (KpnI primer) contained a KpnI site for cloning. This created an in-frame fusion between the amino-terminal 17 amino acids of DnaA and the carboxy-terminal end of chitobiase deleted for the amino-terminal 22 amino acids including the signal peptide.","The next step in constructing the pDYK9 vector was taking the 3270 bp DraI-HindIII (partial digest) fragment containing the dnaA-chb fusion and Ligating it to the pACYC184 2555 bp HincII-HindIII fragment carrying chloramphenicol-resistance (Cm) and the P15A origin. This fragment was described in Chang & Cohen, \u201cConstruction and characterization of amplifiable multicopy DNA cloning vehicles derived from the p15A cryptic miniplasmid,\u201d J. Bacteriol. 134:1141-1156 (1987).","A NotI site was introduced at the AccI site after digestion with AccI, treatment with Mung bean nuclease, and ligation to phosphorylated NotI linkers (New England Biolabs, Inc.; Beverly, Mass.). A NotI site was introduced at the AseI site after digestion with AseI, treatment with Mung bean nuclease, and ligation to phosphorylated NotI linkers (New England Biolabs, Inc.; Beverly, Mass.). An XbaI-SphI PCR product containing the rrnBt1t2 terminator was ligated into this plasmid cut with the same enzymes creating pDYK7chromosomal DNA served as template with primers 5\u2032-CTA GTC TAG ATG CCG AAC TCA GAA GTG A-3\u2032 (SEQ ID NO 3) and 5\u2032-GCA CAT GCA TGC GGG GGA TGG CTT GTA GAT-3\u2032 (SEQ ID NO 4) to produce a PCR product 357 bp long that contains bases 6534 to 6869 from the rrnB operon [numbering according to Brosius, J., et al., \u201cGene organization and primary structure of a ribosomal RNA operon from .\u201d J. Mol. Biol. 148:107-127 (1981)] and includes the complex transcription termination region of this operon which is described in Orosz, A., et al., \u201cAnalysis of the complex transcription termination region of the rrnB gene,\u201d Eur. J. Biochem. 201:653-659 (1991). The forward primer contained an XbaI site and the reverse primer contained an SphI site.","A BclI-SmaI PCR product containing the \u03bb attP site was ligated into pDYK7 cut with TthlllI, treated with Mung bean nuclease, then digested with BclI. Plasmid pHN894 [described in Goodman, S. D., et al., \u201cDeformation of DNA during site-specific recombination of bacteriophage lambda: Replacement of IHF protein by HU protein or sequence-directed bends,\u201d Proc. NatI. Acad. USA 89:11910-11914 (1992)] served as template with the forward primer 5\u2032-CAT GAT CAT GCG ACA GGT TTG ATG A-3\u2032 (SEQ ID NO 5) and the reverse primer 5\u2032-GGG GGC GCC TAC CTT TCA CGA G-3\u2032 (SEQ ID NO 6) producing a PCR product 466 bp long that contains the \u03bb attP site. The 466 bp PCR product containing the \u03bb attP site was first cloned into the SmaI site of the pKSII+ plasmid to produce pDYK8. The forward primer contains a BclI site and the reverse primer contains G's at the 5\u2032 end in order to recreate a SmaI site when cloning the PCR fragment into a SmaI site. This PCR product includes bases \u2212211 to +241 from the center of the attP core and the sequence required for optimum \u03bb attP site integration as described in Nagaraja & Weisberg, \u201cSpecificity determinants in the attachment sites of bacteriophages HK022 and \u03bb,\u201d J. Bacteriol. 172:6540-6550 (1990). The orientation of attP is such that when the fusion is integrated at attB, the transcription direction of dnaA p1 and p2 promoters is the same as replication fork movement mimicking the orientation at the wild type dnaA promoters.","Vector pDYK11 (SEQ ID NO: 12) was constructed by ligating an SphI-KpnI PCR product containing the rpmH-dnaA promoter region into pDYK9 also digested with SphI and KpnI. Plasmid pAC17, described in Chiaramello & Zyskind, (1992), served as template with the primers 5\u2032-CAT GCA TGC ATG AAA CGA TGG ACA CC-3\u2032 (SEQ ID NO 7) and 5\u2032-CGG GGT ACC AAC TCA TCC TGC AAT CG-3\u2032 (SEQ ID NO 8) to produce a PCR product 616 bp long that contains 598 bp from the rpmH-dnaA regulatory region [bases 338 to 935, numbering according to Hansen, F. G., et al., (1982)]. The forward primer (SphI primer I) contained an SphI site and the reverse primer (KpnI primer) contained a KpnI site for cloning. SEQ ID NO: 18 provides the complete coding sequence of the DNA encoding the dnaA\/chitobiase fusion protein. The complete sequence of the dnaA\/chitobiase fusion protein is provided in SEQ ID NO: 19.","The vector pJMF3 (SEQ ID NO: 13) was constructed by first ligating an AseI linker (5\u2032-CATTAATGCATG-3\u2032 (SEQ ID NO 9) self-hybridized) into the SphI site of pDYK11. The pUC19, [described in Yanisch-Perron C., et al., \u201cImproved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors,\u201d Gene 33:103-19 (1985)] AseI-KpnI fragment containing the lacPO-polylinker region was ligated to this plasmid after digestion with AseI and KpnI. The resulting in-frame fusion between the amino-terminal 21 amino acids of the pUC19 lacZ(\u03b1) peptide [Yanisch-Perron C., et al., 1985] and the carboxy-terminal end of chitobiase deleted for the amino-terminal 22 amino acids is identical to the protein fusion in pRSG196 [Jannatipour, M., et al., (1995)].","Vector pJMF4 (SEQ ID NO: 14 ) was constructed by ligating the BamHI-EcoRV fragment of pDYK8 containing the \u03bb attP site into pDYK7 which had been digested with TthlllI , treated with Mung bean nuclease, then digested with BclI, to create pTKP9. This reversed the orientation of the attP site relative to the attP site in pDYK9. The pDYK11 BspMII-KpnI fragment containing the rpmH-dnaA promoter region was then ligated into pTKP9 digested with the same enzymes to create pTKP11. This reversed the orientation of the attP site relative to the attP site in pDYK11. An AseI linker (5\u2032-CATTAATGCATG-3\u2032 (SEQ ID NO 10) self-hybridized) was ligated into the SphI site of pTKP11 to create pJMF2. The pUC19 [as described in Yanisch-Perron C., et al., (1985)] AseI-KpnI fragment containing the lacPO-polylinker region was ligated to pJMF2 cut with AseI and KpnI to create pJMF4.","To move the chitobiase fusions in pDYK9 and pDYK11 to the attB site in the chromosome, Not fragments from these plasmids were self-ligated and transformed or electroporated into strain WM2269 (DH5\u03b1 containing pLDR8) [See Zyskind, J. W. and S. I. Bernstein (1992) \u201cRecombinant DNA Laboratory Manual,\u201d Academic Press, San Diego, Calif.]. Plasmid pLDR8, described in Diederich, L. L. J., et al., \u201cNew cloning vectors for integration into the A attachment site attB of chromosome,\u201d Plasmid 28:14-24 (1992), expresses integrase from the \u03bb Ppromoter and contains the \u03bb cIrepressor gene, a kanamycin resistance gene, and a temperature-sensitive origin of replication. The transformed or electroporated cells were incubated at 42\u00b0 C. with shaking for 30 min then moved to 37\u00b0 C. for 1 h followed by selection on Luria broth agar plates containing chloramphenicol (25 \u03bcg\/ml) at 42\u00b0 C. Transformants were screened for loss of kanamycin resistance and, therefore, loss of pLDR8.","Transduction with P1 bacteriophage by the method of Zyskind & Bernstein, Recombinant DNA Laboratory Manual. Academic Press, San Diego, Calif. (1992) was used to construct strains and to confirm the chromosomal location of the dnaA-chb fusions. Cotransduction of Cm(carried by the fusion) and galK (linked to attB) indicated that Cmand galK are linked on the chromosomes of strains DYK9W, DYK9F, DYK11W, and DYK11F.","Chitobiase activity is located in the cytoplasm when its signal peptide is replaced by fusion with another peptide. Accordingly, chitobiase assays are performed on permeabilized or lysed cells. A variety of permabilization or cell lysis protocols are available to liberate the enzyme from within a cell population being tested. One such protocol involves toluene-treated cells washed once with M9 salts according to the method of Miller, J. H. A Short Course in Bacterial Genetics, CSH Laboratory Press, Cold Spring Harbor, N.Y. 1992, which is hereby incorporated by reference. The toluenized cells are placed in a chitobiase buffer (10 mM Tris-HCl, pH 8.0, and 0.5 M NaCl) and 666 \u03bcM PNAG, the chitobiase enzyme substrate. NaCl is included because chitobiase has approximately 80% of full activity in the absence of salt, with maximal activity occurring between 0.25 and 0.6 M NaCl. Toluenized cells (0.772 ml) are preincubated at 28\u00b0 C. and the reaction started with the addition of 0.228 ml PNAG (1 mg\/ml). After incubation at 28\u00b0 C., the reaction is stopped by the addition of 1 ml of 1 M Tris base. The release of p-nitrophenol is measured at 400 nm and turbidity at 550 nm. p-Nitrophenol release is measured immediately at 400 nm with a molar absorptivity of 1.8\u00d710liters molcm. Units are calculated after subtracting the light scattering factor (1.5\u00d7OD) from ODof the sample. The normalizing factor of 1.5 was determined previously by measuring the light scattering ratio of bacteria at ODand OD. One unit of chitobiase activity is the amount of enzyme that catalyzes the formation of 1 pmol of p-nitrophenol per min at 28\u00b0 C. For comparison to Miller units of \u03b2-galactosidase [described in Miller, J. H., A Short Course in Bacterial Genetics. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. (1992)], the units are normalized to 1 ml of culture at OD=1.","() contain the lacPO promoter with the first 21 amino acids of lacZ\u03b1 (from pUC19) fused in-frame to the chb gene. These plasmids also contain the \u03bb phage attP recombination site in different orientations, the gene encoding chloramphenicol acetyltransferase (cat), and a ribosomal terminator, rrnBt1t2, inserted upstream of the lac-chb fusion to prevent read-through from other promoters. The chitobiase activity associated with these plasmids (Table 1) is high in the absence of IPTG because of titration of lac repressor expressed from a single copy chromosomal gene. Induction by IPTG is approximately 10-fold (Table 1).",{"@attributes":{"id":"P-d0e2729","num":"00002"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Chitobiase Activity of lacZ-chb Fusion"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"105pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Chitobiase(U)",{}]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Plasmid","\u2212IPTG","+IPTG (1 mM)"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["pJMf3 in DH5\u03b1","668 \u00b1 45","9320 \u00b1 347"]},{"entry":["pJMF4 in DH5\u03b1","788 \u00b1 44","7188 \u00b1 477"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"left"},"sup":"a","sub":["450 ","450 "]}},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"left"},"sup":"b"}},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"left"},"sup":"c","sub":"450 "}}]}}]}}},"The lac promoter can be replaced with another promoter and a fusion protein created with chitobiase by cutting with SphI or AseI and either SalI, KpnI, or SstI (FIG. ). Fusions created with these vectors can be moved to the chromosome by site specific recombination at the \u03bb attB site to permit single copy analysis of the activity of the promoter. The protocol, as described more fully in Diederich, et al., \u201cNew cloning vectors for integration into the \u03bb attachment site attB of chromosome,\u201d Plasmid 28:14-24 (1992), involves two components, (i) a circular DNA containing the \u03bb attachment site, attP, the promoter-chb gene fusion, and the cat gene, and (ii) a helper plasmid, pLDR8, which contains the int gene under the control of the temperature-sensitive repressor, cI857 and a temperature-sensitive origin of replication. The plasmid is digested with NotI to remove the P15A origin, and the fragment containing the chitobiase fusion is self-ligated prior to transformation into cells containing pLDR8. Integration occurs by site-specific recombination between attP and attB (17.4 min on the chromosome, FIG. ).","Two plasmids, pDYK9 and pDYK11, discussed in the Examples above, were constructed to assess the regulation of the dnaA gene using chitobiase as a reporter enzyme. These plasmids differ by the absence of the rpmH regulatory region in pDYK9 (FIG. ). These fusions were moved from the plasmid to the chromosomal attB site for single copy analysis as described above. After transformation of strain WM2269 with the ligated DNA, integration occurred by site specific recombination between the attP and the attB sites. The orientation of attP in pDYK9 and pDYK11 is such that when the fusion is integrated at attB, the transcription direction of dnaAp1 and dnaAp2 promoters is the same as movement of the replication fork. This orientation is the same as at the dnaA wild type location. The genetic location of the fusions was confirmed in the Cmtransformants by demonstrating cotransduction of Cmand galK.","The fusions created in strain WM2269 were moved by P1 transduction to MG1655, creating strain DYK9W with pDYK9 and strain DYK11W with pDYK11. Deletion of the rpmH promoters had very little effect (1.4-fold) on chitobiase activity (Table 2, compare lines 1 and 3)",{"@attributes":{"id":"P-d0e3016","num":"00002"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Chitobiase and \u03b2-Gal Activities of dnaA-chb and dnaA-lacZ"},{"entry":"Fusions in WT and fis Mutant Backgrounds"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"70pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},"\u03b2Gal"]},{"entry":[{},"Strains","Chitobiase(U)","(Miller U)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"DYK11W fis+","30.0 \u00b1 2.1",{}]},{"entry":[{},"DYK11F fis::985","80.0 \u00b1 1.2"]},{"entry":[{},"DYK9W fis+","44.0 \u00b1 2.1"]},{"entry":[{},"DYK9F fis::985","96.0 \u00b1 3.5"]},{"entry":[{},"RB220 fis+",{},"\u200259.8 \u00b1 7.9"]},{"entry":[{},"TP220 fis::767",{},"115.2 \u00b1 7.2"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"left"},"sup":"a"}]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"left"},"sup":"b","sub":"450 "}]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"left"},"sup":"c"}]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"left"},"sup":"d"}]}]}}]}}},"Fis protein binds to a site in the dnaAp2 promoter that covers the \u221235 sequence (FIG. ), and appears to be a repressor of DnaA expression. A fusion protein with \u03b2-galactosidase activity that is expressed from the rpmH-dnaA regulatory region has increased \u03b2-galactosidase activity (1.9-fold) in a fis mutant when compared to Fis wild type cells [as described in Froelich, J. M., et al., \u201cFis binding in the dnaA operon promoter region,\u201d J. Bacteriol. 178:6006-6012 (1996), data shown in Table 3]. Similarly, the absence of Fis leads to a greater than 2-fold increase in chitobiase activity of the DnaA-chitobiase fusion protein for the DYK9F and DYK11F strains, comparable in extent to that observed with the dnaA-lacZ fusion strain, TP220 (Table 3).","In the reporter gene constructs discussed in the Example, all upstrearn transcriptional activity was prevented from entering the chb reporter gene. The plasmid vectors, pJMF3 and pJMF4, described in the Examples above, contained the rrnbt1t2 terminator upstream of the promoter fusion, which prevented readthrough from chromosomal promoters near the insertion site. Only chitobiase activity originating from the promoters of interest was expressed.","The attP site in these vectors allowed integration at the chromosomal attB in a specific orientation, depending on the vector used. With these vectors, any chitobiase fusion involving an essential gene can be moved to the chromosome, thus permitting single copy analysis with a chromosomal orientation similar to the wild-type gene.","A nucleic acid prospectively containing a promoter is inserted upstream of a nucleic acid encoding a cytoplasmic form of chitobiase as described above. For example, the nucleic acid prospectively containing a promoter may be inserted into a restriction site in a sequence containing a plurality of restriction sites, such as a polylinker, which is located upstream of the nucleic acid encoding chitobiase. The test sequence may comprise any nucleic acid to be tested for promoter activity. In one embodiment, the test sequence may comprise a genomic DNA sequence. For example, the genomic DNA sequence may be a randomly generated DNA fragment, such as a fragment generated using shotgun cloning techniques, a restriction fragment, or any other sequence.","The vectors containing the test sequence upstream of the nucleic acid encoding chitobiase are introduced into an appropriate host cell. The level of chitobiase activity is assayed and compared to the level obtained from a control vector which lacks an insert in the cloning site. The presence of an elevated expression level in cell containing the vector containing the insert with respect to the level in cells containing the control vector without the insert indicates the presence of a promoter in the insert.","In some embodiments, the activity of the promoter in the test sequence may be assayed after exposure of the host cells to conditions which may influence the level of transcription from the promoter. For example, the environment of the host cells may be altered to determine whether the transcription level is influenced by environmental factors, including factors such as temperature, pH, nutrients, or availability of oxygen. In such analyses, chitobiase levels are assayed under a variety of environmental conditions to determine the effects of the environmental conditions on transcription levels from the promoter. In addition, the activity of the promoters may be examined in the presence or absence of compounds to be tested for regulatory activity. For example, the activity of the promoters may be tested by determining the levels of chitobiase produced in the presence or absence of compounds to be tested for activity as drugs.","Promoter sequences within the test sequences may be further defined by constructing nested deletions in the test sequences using conventional techniques such as Exonuclease III digestion. The resulting deletion fragments can be inserted into the promoter reporter vector to determine whether the deletion has reduced or obliterated promoter activity as determined by measuring chitobiase activity in cells containing the deletion vectors. In this way, the boundaries of the promoters may be defined. If desired, potential individual regulatory sites within the promoter may be identified using techniques such as site directed mutagenesis or linker scanning to obliterate potential transcription factor binding sites within the promoter individually or in combination. The effects of these mutations on transcription levels may be determined by inserting the mutations into the cloning sites in the promoter reporter vectors and measuring the levels of chitobiase produced from the mutated promoters.","The activity of known promoters may also be monitored by operably linking them to a nucleic acid encoding a cytoplasmic form of chitobiase. The activity of the promoters may be analyzed under various environmental conditions as described above. In addition, the activity of the promoters may be analyzed in the presence or absence of compounds to be tested for the ability to affect transcription from the promoters. For example, the compounds may be tested for activity as drugs.","In some embodiments, the chitobiase reporter constructs may be used in systems for identifying compounds that modulate cell surface protein-mediated activity or compounds which modulate the activities of intracellular signaling systems. Techniques for using reporter genes to identify compounds which modulate cell surface protein-mediated activity have been described in U.S. Pat. No. 5,401,629 and U.S. Pat. No. 5,436,128, the disclosures of which are incorporated herein by reference in their entireties. Briefly, in such methods, a construct comprising a promoter operably linked to a nucleic acid encoding a reporter enzyme is introduced into cells which express the cell surface protein and cells which do not express the cell surface protein. Each of the cells are contacted with test compounds and the effects of these compounds on transcription levels is measured by determining the level of activity of the reporter enzyme. The level of expression of the reporter gene in cells expressing the cell surface protein is compared to the level in cells which do not express the cell surface protein to identify compounds that modulate cell surface protein activity.","Similarly, the chitobiase reporter constructs may be used to identify compounds which influence the activity of intracellular signaling pathways, such as cAMP-based or phosphorylation-based pathways. In such methods, a promoter which is activated via such pathways is operably linked to a nucleic acid encoding a cytoplasmic form of chitobiase. The cells are contacted with test compounds. Those compounds which activate the pathway to which the promoter responds will produce an enhanced level of chitobiase activity in the cells as compared to the level of chitobiase activity in control cells which have not been contacted with the test compound.","A vector comprising a sequence encoding a cytoplasmic form of chitobiase operably linked to a sequence capable of directing transcription of the chitobiase gene is introduced into a host cell. The host cells are contacted with a chitobiase substrate and those host cells which contain chitobiase activity are identified as cells which were successfully transformed or transfected. In some embodiments, a portion or replica of a colony may be lysed or permeabilized prior and the lysate or permeabilized cells may be contacted with the chitobiase substrate.","New gene reporter systems that use chitobiase have been described. Chitobiase has advantages over other reporter gene systems in that chitobiase is not found in many cell lines traditionally used in reporter gene systems.","Finally, the forgoing examples are not intended to limit the scope of the present invention, which is set forth in the following claims. In particular, various equivalents and substitutions will be recognized by those of ordinary skill in the art in view of the foregoing disclosure, and these are contemplated to be within the scope of the present invention. All references cited herein are incorporated herein by reference in their entireties."],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"P-00020","num":"00020"},"figref":"FIG. 1","b":"2"},{"@attributes":{"id":"P-00021","num":"00021"},"figref":["FIG. 2","FIG. 2"]},{"@attributes":{"id":"P-00022","num":"00022"},"figref":"FIG. 3"},{"@attributes":{"id":"P-00023","num":"00023"},"figref":"FIG. 4"},{"@attributes":{"id":"P-00024","num":"00024"},"figref":"FIG. 5","i":"Escherichia coli "}]},"DETDESC":[{},{}]}
