---
title: Automated pathway recognition system
abstract: There is a pressing need for computer-implemented tools that can summarize and present the enormous amounts of public literature to facilitate analysis of gene expression data. The present invention provides techniques and systems for efficiently integrating public literature regarding gene function with data from gene expression profiling experiments. Information from literature databases relating to a particular set of DNA sequences of known expression pattern is retrieved, processed, cross-referenced and viewed to provide further information about a particular DNA sequence to facilitate its identification as a candidate gene.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=06876930&OS=06876930&RS=06876930
owner: AGY Therapeutics, Inc.
number: 06876930
owner_city: South San Francisco
owner_country: US
publication_date: 20020304
---

{"@attributes":{"id":"description"},"BRFSUM":[{},{}],"heading":["BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENTS"],"p":["The present invention relates to bioinformatics and its use in methods of characterizing and identifying candidate genes. More particularly, the invention relates to the use of information extraction in the analysis of data from high-throughout gene expression profiling experiments.","High-throughput gene expression profiling techniques, such as those employing DNA microarrays, have become a standard and widely used technique for the identification of drug targets, gene therapy targets and therapeutic protein targets in human medicine (see U.S. Pat. Nos. 5,807,522 and 5,593,839). The early and efficient identification of target genes, otherwise known as \u201ccandidate\u201d genes, using these techniques could significantly reduce the overall costs and time taken to develop and market actual products.","The rise in popularity of gene expression profiling, coupled with the increase in complexity of the experiments, has led to a tremendous increase in the amount of information that has to be organized and processed. One experiment alone, for example a time course of a disease process or a comparison between a treated sample with a non-treated control, may provide data on several thousand different genes. Analyzing and storing this data in a meaningful way has become rate limiting for biologists.","Several computational tools have been applied to this problem. For example, computational methods have assigned names to DNA sequences by comparing their sequence with sequence of named genes in public databases using such algorithms as BLAST (see generally U.S. Pat. No. 6,023,659). DNA sequences have also been assembled and grouped into functional hierarchies by specific algorithms to help investigators interpret gene expression data (U.S. Pat. No. 6,023,659).","More recently, gene expression profiles have been examined using methods that can cross-compare the expression profiles of many thousands of genes across many different experiments (for example Eisen et al P.N.A.S. 95, 14863-8). These methods employ pattern recognition algorithms to cluster genes with a similar expression patterns facilitating the facile identification of groups of genes that are co-regulated. Both supervised and unsupervised pattern recognition algorithms can be used to for clustering. Supervised pattern recognition algorithms require a priori knowledge that forms a training set, whereas unsupervised pattern recognition algorithms do not need a priori knowledge and are typically used to discover latent patterns. Many unsupervised clustering methods have been applied to gene expression profile data: these include hierarchical, K-means, self-organizing maps (Tamayo et al. PNAS 96:2907-12), or support vector machines (M. Brown et al. PNAS 97:262-7).","Once gene expression data has been gathered and analyzed, mostly by computer, researchers typically spend a significant amount of time gathering information from public databases, in particular public literature databases, in order to annotate their genes of interest, increase their confidence in a particular result, and permit the discovery of candidate genes. These methods are typically manually performed, in part due to a lack of tools to organize and process the enormous amount of public literature that is available for many of these genes. Because of this manual step, the available methods do not allow the efficient and facile identification of candidate genes.","Thus, there is a pressing need for tools that can process, summarize and cross-reference the enormous amounts of public literature, and allow this data to be used in combination with gene expression profiles to aid in discovering candidate genes. Since public literature is making a transition from printed media to digital media in the form of literature databases, an opportunity has emerged for computers to assist in this effort. There have been several attempts to use information extraction (IE) and natural language processing (NLP) methods within the context of biology. For example, protein-protein interactions can be examined using IE approaches (Science. 1997,275(5298):327-334; Proux (1988) Genome Inf. Workshop 9, 72-80; Hishiki et al, (1998) Genome Inf. Workshop 9, 81-90). However, IE and NLP have not yet been used in the context of examining gene expression profiling data to identify candidate genes.","Thus, within the art, there is a need for methods and techniques that can efficiently annotate genes with known information, in particular information from public literature databases regarding relationships between gene functions, and organize this information with gene expression profiles, facilitating the identification of candidate genes.","The present invention provides techniques and systems for facilitating identification of candidate genes from a plurality of DNA sequences. More particularly the invention uses computer-implemented methods and systems to efficiently extract and process information on gene pathways and gene relationships and combine this information with results of other analyses in order to facilitate the rapid analysis of gene expression data.","According to one embodiment, the present invention provides a method that integrates the enormous amount of public literature regarding gene function with data from gene expression profiling experiments. Information from literature databases relating to a particular set of DNA sequences is retrieved, and using information extraction methods the literature is processed. The data is then cross-referenced and viewed to provide further information about a particular DNA sequence to facilitate its identification as a candidate gene. This cross-referenced information may be stored in a database. Additional annotation can be added by a user to this database, and these comments may further facilitate the identification of candidate genes by altering the manner in which information is cross-referenced in future iterations of this process.","In another embodiment, the present invention provides a method for analyzing a group of genes identified through analysis of gene expression profiling experiments, wherein the groups of genes have been grouped together by a commonality in their gene expression patterns. Clustering algorithms may be employed to automatically group genes by their expression pattern and a cluster of genes may represent a group of genes. These clustering algorithms may be supervised or unsupervised. A further embodiment of the invention provides a method for using both supervised and unsupervised clustering algorithms to automatically group genes by their expression pattern. The gene expression data analyzed may be from microarray experiments.","In yet another embodiment, information may be extracted from one of many literature databases, for example MEDLINE, the U.S.P.T.O. and W.I.P.O. patent database, KEGG, OMIM or MIPS, etc, and can be ranked by quality, based on a citation index, where information from highly cited journals is of higher quality than information from lower cited journals. Information can be extracted from literature database using natural language processing algorithms. The extracted information can pertain to a pathway in which a gene functions, or information about interactions between different genes on a list.","In further embodiments, the invention provides a data processing system for identifying candidate genes where the system has instructions for accessing, extracting, cross-referencing and viewing information from a literature database pertaining to a gene with a known expression pattern. The system of the invention contains a processor, memory, and instructions for accessing, extracting, cross-referencing and viewing the information. The cross-referenced information may be stored in a database and readily accessed by a user.","In yet another embodiment, the data processing system contains instructions to access a biomedical journal literature database, and further instructions for ranking the biomedical publications using a citation index and associating the ranking score with the information extracted from the journal. The ranking of this information allows information extracted from a literature database to be given a significance score.","The invention also provides an integrated data processing system containing instructions for clustering genes based on expression pattern behavior and has instructions for accessing, extracting, cross-referencing and viewing information from a literature database pertaining to a gene or a group of genes that have similar behavioral pattern.","The present invention provides an improved method for examining gene expression profile data for the discovery of candidate genes. The invention employs information extraction techniques such as natural language processing, and can be used to rapidly determine the relationships between different genes once gene expression profiling experiments have been performed on those genes. In a preferred embodiment, the genes are clustered by their expression patterns using data provided by microarray experiments. The automated information extraction provides relevant information about the relationships between individual genes in a group of genes that has undergone cluster analysis. In the following description, numerous specific details are set forth in order to provide a thorough understanding of the present invention. It will be apparent, however, that the present invention may be practiced without limitation to some of the specific details presented herein.","The following examples are put forth so as to provide those of ordinary skill in the art with a complete disclosure and description of how to make and use the present invention, and are not intended to limit the scope of what the inventors regard as their invention nor are they intended to represent that the experiments below are all or the only experiments performed. Efforts have been made to ensure accuracy with respect to numbers used (e.g. amounts, temperature, etc.) but some experimental errors and deviations should be accounted for. Unless indicated otherwise, parts are parts by weight, molecular weight is weight average molecular weight, temperature is in degrees Centigrade, and pressure is at or near atmospheric.","This invention is not limited to particular methods described, as such may, of course, vary. It is also to be understood that the terminology used herein is for the purpose of describing particular embodiments only, and is not intended to be limiting, since the scope of the present invention will be limited only by the appended claims.","Where a range of values is provided, it is understood that each intervening value, to the tenth of the unit of the lower limit unless the context clearly dictates otherwise, between the upper and lower limit of that range and any other stated or intervening value in that stated range is encompassed within the invention. The upper and lower limits of these smaller ranges may independently be encompassed within the invention, subject to any specifically excluded limit in the stated range. Where the stated range includes one or both of the limits, ranges excluding either both of those included limits are also included in the invention.","Unless defined otherwise, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Although any methods and materials similar or equivalent to those described herein can also be used in the practice or testing of the present invention, the preferred methods and materials are now described.","It must be noted that as used herein and in the appended claims, the singular forms \u201ca\u201d, \u201cand\u201d, and \u201cthe\u201d include plural referents unless the context clearly dictates otherwise.","The publications discussed herein are provided solely for their disclosure prior to the filing date of the present application. Nothing herein is to be construed as an admission that the present invention is not entitled to antedate such publication by virtue of prior invention. Further, the dates of publication provided may be different from the actual publication dates, which may need to be independently confirmed. All publications mentioned herein are incorporated herein by reference to disclose and describe the methods and\/or materials in connection with which the publications are cited.","The following terms are used throughout the specification. The description are provided to assist in understanding the specification, but do not necessarily limit the scope of the invention.","Database\u2014A database can be any computer medium wherein data can be stored. A database can be relational or object-oriented, and can be a spreadsheet, a flat text file, a table or a list, for example.","Gene\u2014A gene, in this context, includes any nucleic acid coding sequence or a gene product such as an mRNA or protein encoded by a gene.","DNA sequence database\u2014This is a database containing information pertaining to DNA sequences. The information may be the sequence itself, a unique identifying name of the sequence, a name of the gene represented by the sequence, a function of the gene in terms of its enzymatic or other activity, its expression pattern, a pathway or physiology it is associated with, or any other information added by a user, or by any computer-assisted methods, which relates to the gene represented by the sequence.","Public literature database\u2014This is a database, normally a publicly available database or a database that is commercially available, containing information about genes. The literature databases disclosed herein are the public MEDLINE database of the National Library of Medicine, as made available through PubMed, a database of full-text issued patents and published US and worldwide patent applications, such as the databases searchable through the world wide web at the U.S. Patent and Trademark Office (U.S.P.T.O.), or the World Intellectual Property Organization (W.I.P.O.), the Kyoto Encyclopedia of Genes and Genomes (KEGG) (Ogata et al, Nucleic Acids Res. (1999) 27:29-34), the Munich Information Center for Protein Sequences (MIPS) (Mewes et al, Nucleic Acids Res. (1999) 27:44-8), or the Online Mendelian Inheritance in Man (OMIM) database, although any database containing abstracts from periodicals, full-text articles, summaries of gene interactions, patent information or any other textual information about genes may suffice. In this context a text database is synonymous with a literature database.","Internal database\u2014This is an internal database containing, for example information about gene expression patterns, sequences, annotation and literature etc. The internal database will normally contain information that has been generated through downloading or transferring information from any public literature database to a database within an internal network. Internal databases are typically maintained in confidentiality from the public through a firewall.","Gene expression database\u2014A database containing information about the expression patterns of genes. Gene expression may be described e.g. as ratios, normalized ratios, absolute values or normalized values, etc. A gene expression pattern may be the result of performing gene expression profiling experiments. Gene expression profiling experiments can provide data on the expression patterns of up to several thousand genes in several different experimental and control samples. The gene expression database stores these patterns. The data are stored in a convenient format, such as a tab-delimited text file, for further analysis. The gene expression database also contains cluster information.","Cluster\u2014A cluster is a group of genes e.g. that have some commonality in their expression pattern. For example, a cluster of genes may be a group of genes that are all up-regulated in a certain cell type that has been treated with a certain agent. A cluster can also be a group of genes that are up-regulated or down regulated by certain agents but not others. Clusters can be the result of cluster analysis of a gene expression data, derived from the gene expression database with supervised and\/or unsupervised pattern recognition algorithms, such as hierarchical, K-means, self-organizing maps or support vector machines.","Supervised and unsupervised pattern recognition algorithms\u2014Pattern recognition algorithms are either supervised or unsupervised. Supervised pattern recognition methods require a priori knowledge which forms a training set, and are applied towards categorizing data according to pre-defined groups. A simple Bayesian approach can be taken, or more sophisticated maximum entropy or nearest neighbor analysis may by preferred. Unsupervised methods encompass clustering and dimension reduction strategies, and clustering can be accomplished through self-organizing maps, K-means, or support vector machines clustering. Unsupervised methods may be used to discover patterns (for example, of gene expression) that only become apparent under certain physiological, developmental, transgenic, gene knock-out or drug-induced conditions whereas supervised methods may be used to discover genes with a particular expression profile. A combination of supervised and unsupervised pattern recognition algorithms may be used in the methods of this invention.","Information extraction (IE)\u2014IE is a natural language processing (NLP) method for transferring knowledge from unstructured natural language data, for example a literature database such as MEDLINE, into a structured and summarized form that can be placed in another database for easy access and use. The major goal of information extraction within the scope of this invention is to discover the relationships between genes by searching literature for gene names, examining text phrases or sentences containing the gene names, and determining the relationship between the genes using information extraction. Information extraction will often employ NLP techniques, such as part-of-speech tagging, stemming, tokenization, stop word removal, multi-word (noun) phrase identification, template filling, Hidden Markov Models and anaphora detection.","Cross-referencing\u2014The determination of the biological relationship between the functions of two or more genes. The biological relationship between genes may be simple, or complex. For example, genes whose products encode enzymes in the same biochemical pathway, or genes whose products physically interact, for example in a signal transduction pathway, have a simple relationship. When two different genes that are not related by any simple relationship or interaction show similar gene expression behaviors, they have a complex relationship. A relationship between two genes may also be inductive, as in one gene product may induce a second gene product. More complex relationships may exist between genes if the two genes both induce the same gene product. Relationships between genes can be said to be activating, binding, inhibiting, adhering inducing, transporting, exporting, creating, phosphorylating, de-phosphorylating, upstream of or downstream of, for example. A biological pathway can be constructed from cross-referenced information about relationships between genes.","Candidate gene\u2014A candidate gene may trigger, prevent, ameliorate or affects a variety of diseases or physiological states. In a drug development process, potential drugs may be tested against products of candidate genes, and candidate genes may also be used to diagnose diseases or physiological states.","1) The Database Environment",{"@attributes":{"id":"P-00051","num":"00051"},"figref":"FIG. 1","b":["10","18","12","14","16","1","16","2","15","17","11"]},"The network  may be any one of a number of conventional network systems, including a local area network (LAN) or a wide area network (WAN), as is known in the art. The network includes functionality for packaging client calls into a well known format, for example a URL, together with any parameter information into a format suitable for transmission across a cable or wire for delivery to server .","Server  includes hardware necessary for running software to access database data for processing user requests and provide an interface for serving information to client machines - and -. In a preferred embodiment, the software running on the server machine supports the World Wide Web (web) protocol for providing page data between a web server  and a client - or -. The web application  provides a user interface between server and a client. Server  receives information through the network to carry out instruction provided by a user, and requests are directed to the appropriate hardware and analysis applications . As may be necessary, the server may be distributed over two or more machines. A first server may be a web server running web applications  and the second server may be an analysis server running analysis applications . Optionally an analysis computational node can added to the network to provide for increased computational capacity. As shown, server  also includes a database management system, for example the Oracle Suite, or the IBM DB2 Discovery Link systems for managing, retrieving, organizing and accessing data.","Analysis application  includes executable code necessary for generation of database queries, for example, embedded SQL statements. In addition, analysis application  includes files which contain references, pointers and addresses to the various software entities that are contained within the server as well as the various data which must be accessed to service user requests.","Each of clients includes a web browser for providing a user interface to server. Through the web browser, clients construct requests for retrieving and processing data from public  or internal data, defined by , , , and . Thus, the user will typically type into a user interface, or point and click to user interface elements such as buttons, pull down menus; and the like, as typically employed in graphical user interfaces. The requests so formulated with the client's web browser are transmitted to web application  which transmits requests to the analysis application which formats and executes the instructions","In the embodiment shown, analysis application constructs a query in database language, for example Sybase or Oracle SQL, and data in a database is accessed and relevant information extracted from the database typically through a database management system .","Web application  provides an appropriate graphical user interface, such as Hypertext Mark-up Language (HTML) browser to a client machines - and -, are is employed by the user to formulate his or her requests. The web application  sends instructions for analysis, as determined by the request, to the analysis application . The analysis application  converts the request to an SQL query, which is used by database management system  to access relevant data in databases and provide that data to server  in an appropriate format. Server  then processes the data, stores said processed data in expert database  and generates a new HTML document relaying the processed data to the client as a view in user interface - and -. While a server  and web browser is employed in , other communication systems will be suitable that do not rely on web applications .","When a network employs a server with web applications and clients, it should support a TCP IP protocol. Local networks such as this can be referred to as \u201cintranets\u201d. An advantage of such intranets is that they allow easy communication with public domain databases residing on the World Wide Web, such as GenBank and MEDLINE. Thus, in an embodiment, clients can directly access data in public databases residing on the internet using an HTML interface provided by web browsers and web server. A firewall preserves the privacy of the gene expression data , sequence data , expert database  and other internal data .","In a preferred embodiment, DNA sequence data  is stored in a flat file database including partial or full-length nucleotide sequences of genes. Associated with this nucleotide sequence is other information about the nucleotide sequence, such as the gene name to which it corresponds, and the name of the gene product, e.g. enzyme that the gene encodes. The gene expression data is preferably stored in a database in a relational format, for example Oracle\u2122 or Sybase database architectures can be used. The database(s) containing DNA sequence data , gene expression data , internal data and expert data  can be XML, relational or object-oriented, depending upon the requirements of the system. In a preferred embodiment, public data  is utilized. An internal database  of public data can be created by transferring information from a public database  to a private network.","Suitable dual or multi-processor server computer machines may be selected, for example, from any of the following: Sun Enterprise 450 or Sun Enterprise 3500, (Sun Microsystems, Inc. of Mountain View, Calif.), Compaq AlphaServer ES45 (Compaq Computer Corporation of Houston, Tex.), and IBM iSeries Model 820 (International Business Machines of White Plains, N.Y.). These server workstations may have an operating system e.g. Sun Solaris, Compaq Tru64, Linux, Unix-variants, etc. The client computer machines may be, for example, a Macintosh\u2122 (Apple Computer Inc of Cupertino, Calif.), a PC, Unix workstation, etc. Workstations may be TCP\/IP capable with a Netscape or Microsoft Internet Explorer Web Browser. The network may include a 10\/100-base-T or greater connection, be TCP\/IP capable, and provide access to the internet.",{"@attributes":{"id":"P-00061","num":"00061"},"figref":"FIG. 2","b":["20","20","24","22","32","34","40","30","28","26","20","26","12","12"]},"User interface input devices  may include a keyboard, pointing devices such as a mouse, trackball, touchpad, or graphics tablet, a scanner, a touchscreen incorporated into the display, audio input devices such as voice recognition systems, microphones, and other types of input devices. In general, use of the term \u201cinput device\u201d is intended to include all possible types of devices and ways to input information into computer system  or onto computer network .","User interface output devices  may include a display subsystem, a printer, a fax machine, or non-visual displays such as audio output devices. The display subsystem may be a cathode ray tube (CRT), a flat-panel device such as a liquid crystal display (LCD), or a projection device. The display subsystem may also provide non-visual display such as via audio output devices. In general, use of the term \u201coutput device\u201d is intended to include all possible types of devices and ways to output information from computer system  to a human or to another machine or computer system.","Storage subsystem  stores the basic programming and data constructs that provide the functionality of the various systems embodying the present invention. For example, the various modules implementing the functionality of the present invention may be stored in storage subsystem . These software modules are generally executed by processor . In a distributed environment, the software modules may be stored on a plurality of computer systems and executed by processors of the plurality of computer systems. Storage subsystem  also provides a repository for storing the various databases storing information according to the present invention. Storage subsystem  typically comprises memory subsystem  and file storage subsystem .","Memory subsystem  typically includes a number of memories including a main random access memory (RAM)  for storage of instructions and data during program execution and a read only memory (ROM)  in which fixed instructions are stored. File storage subsystem  provides persistent (non-volatile) storage for program and data files, and may include a hard disk drive, a floppy disk drive along with associated removable media, a Compact Digital Read Only Memory (CD-ROM) drive, an optical drive, removable media cartridges, and other like storage media. One or more of the drives may be located at remote locations on other connected computers at another site on communication network . Information stored according to the teachings of the present invention may also be stored by file storage subsystem .","Bus subsystem  provides a mechanism for letting the various components and subsystems of computer system  communicate with each other. The various subsystems and components of computer system  need not be at the same physical location but may be distributed at various locations within distributed network . Although bus subsystem  is shown schematically as a single bus, alternate embodiments of the bus subsystem may utilize multiple busses.","Computer system  itself can be of varying types including a personal computer, a portable computer, a workstation, a computer terminal, a network computer, a television, a mainframe, or any other data processing system. Due to the ever-changing nature of computers and networks, the description of computer system  depicted in  is intended only as one example, only for purposes of illustrating an embodiment of the present invention. Many other configurations of a computer system having more or less components than the computer system depicted in  also find use. Client computer systems  and server computer systems  may have the same configuration as shown in , with the server systems generally having more storage capacity and computing power than the client systems.","2) Workflow","FIG. . depicts a simplified flowchart showing processing performed by an embodiment of the present invention to facilitate identification of candidate genes from a plurality of input DNA sequences. As shown in , processing is initiated when the server system  accesses results of a homology search from the plurality of input DNA sequences . These steps are typically performed by client, or server, or by a plurality of servers.","The present invention then extracts relevant information from the homology analysis output as described above for each input DNA sequence, organizes the information, and stores it in a format which facilitates further processing and analysis of the information . Typically at least the name of the gene represented by a nucleotide sequence is stored in a database, however more functional annotative information may be stored for the genes according to an embodiment of the present invention, for example, the sequence name, best BLAST hits, and its biological role or multiple roles deduced from the best BLAST or Smith-Waterman hits.","Server  then obtains descriptive annotative information on the biochemical function(s) and the physiological role(s) for the known genes from the plurality of nucleotide sequences  and stores the information in an expert database . Several different techniques may be used by the present invention to obtain the functional information, including information extraction.","The present invention then obtains  and stores  expression profile data for the genes and their homologs. Profiles can be created for genes expressed in cells or tissues under influences of an agent, as a cell or tissue develops, or during changes to the physiological state of the cell or tissue, or in response to the development of disease in humans or an animal model. The corresponding nucleotide sequences may then be clustered by their expression patterns by one of any number of means  and the clustering analysis is stored in a database .","In one embodiment of the present invention, pathway information, represented as cross-referenced information about the biological relationships of genes represented by the nucleotide sequences and their homologs, is obtained  from literature databases such as MEDLINE by natural language processing and information extraction procedures, cross-referenced, and may be stored  in the expert database for convenient viewing by a user. Other public literature databases include, without limitation, the Kyoto Encyclopedia of Genes and Genomes (KEGG) or the Munich Information Center for Protein Sequences (MIPS), the U.S.P.T.O. and W.I.P.O. patent database, the Online Mendelian Inheritance in Man (OMIM) database etc. Additional annotative information concerning, for example, genes and diseases or gene expression information may be obtained and processed from the above databases.","The information stored in the databases according to the present invention facilitates the identification of candidate genes . In particular, the combination of gene expression profile data and pathway information generated through computer implemented natural language processing and information extraction facilitates the identification of candidate genes. A more detailed description of the methods is given below.","3) Homology Search Results","The nucleotide sequences which are input as queries to the homology search are generally obtained from complementary cDNA sequences which have been synthesized using isolated messenger RNA (mRNA; the transcription products of expressed genes). The cDNA sequences are used as input sequences to the homology search analysis since cDNAs represent expressed genomic regions and are thus believed to identify parts of the genome with the most biological and medical significance.","As part of the homology search, nucleotide and protein sequence databases are searched to find sequences that are related to the input or query nucleotide sequences. For example, given a set of differentially expressed query sequences, corresponding to the mRNA of their cognate genes, a homology search identifies known, similar and unknown genes. A homology search is generally performed by using computer-implemented search algorithms to compare the query sequence with sequence information stored in a plurality of databases accessible via a communication network, for example, the Internet. Examples of such algorithms include the Basic Local Alignment Search Tool (BLAST) algorithm, the PSI-blast algorithm, the Smith-Waterman algorithm, the Hidden Markov Model (HMM) algorithm, and other like algorithms. For example, a \u201cblastn\u201d program utilizing the BLAST algorithm may be used to search the Genbank database for homologs of the query cDNA sequences. According to an embodiment of the homology search, the query cDNA sequences may be grouped as \u201cknown,\u201d \u201cunknown,\u201d or \u201csimilar\u201d sequences. \u201cKnown\u201d cDNA sequences include sequences with substantial sequence identity to existing sequence entries in a sequence database, such as the GenBank database. \u201cUnknown\u201d cDNA sequences include sequences similar to existing sequence entries in a sequence database but lacking functional annotation, or those sequences with no matching sequences in existing sequence databases. \u201cSimilar\u201d cDNA sequences include sequences for which no matches are found in the sequence database, but which exhibit similarity, as defined below, to existing entries in sequence databases.","Two or more sequences, either polynucleotide or polypeptide, may exhibit \u201csubstantial sequence identity\u201d if the sequences have at least 70%, preferably 80%, most preferably 90%, 95%, 98% or 99% nucleotide or amino acid residue identity, when compared and aligned for maximum correspondence, as measured using a particular sequence comparison algorithm or by using visual inspection.","Several different sequence comparison algorithms may be used. According to a first technique, using global or local alignments, two sequences (amino acid or nucleotide) can be compared over their full-length (e.g. the length of the shorter of the two, if they are of substantially different lengths) or over sub-sequences of 200, at least about 200, at least about 500 or at least about 1000 contiguous nucleotides or at least about 40, at least about 50, or at least about 100 contiguous amino acid residues. According to an embodiment of the present invention, a query sequence may qualify as a \u201cknown\u201d gene if the query DNA sequence meets the following stringent criteria: (1) a sequence length greater than 200 nucleotides with greater than or equal to 80% identity over 70% of the query sequence length with an E-value (a probability value of a match occurring if the sequence were randomized) of less than 1; and (2) for the predicted amino acid homology, greater than or equal to 80% identity for a segment length greater than 50 amino acids and an E-value of less than 1. Sequences that meet either, but not both, the nucleotide or polypeptide sequence criteria may be grouped as \u201csimilar\u201d genes after examination of the respective nucleic acid or amino acid alignments.","For sequence comparison, typically one sequence acts as a reference sequence, to which test sequences are compared. When using a sequence comparison algorithm, test and reference sequences are input to a computer, subsequence coordinates are designated, if necessary, and sequence algorithm program parameters are designated. The sequence comparison algorithm then calculates the percent sequence identity for the test sequence(s) relative to the reference sequence, based on the designated program parameters.","As stated above, a plurality of homology search algorithms may be used to determine optimal alignment of sequences. These include the local homology algorithm of Smith & Waterman, Adv. Appl. Math. 2:482 (1981), the homology alignment algorithm of Needleman & Wunsch, J. Mol. Biol., 48:443 (1970), the similarity method of Pearson & Lipman, Proc. Natl. Acad. Sci. USA 85:2444 (1988), the PSI-Blast homology algorithm of Altschul et al., Nucleic Acids Res. 25:3389-402 (1997), the computerized implementations of algorithms GAP, BESTFIT, FASTA, and TFASTA included in the Wisconsin Genetics Software Package, Genetics Computer Group, 575 Science Dr., Madison, Wis.), by Hidden Markov Models (HMM, Durbin, Eddy, Krogh & Mitchison, Cambridge University Press, 1998), or EMotif\/EMatrix to identify sequence motifs (Nevill-Manning, Wu, & Brutlag, Proc Natl. Acad. Sci U S A. 1998 May 26;95(11):5865-71), or by visual inspection (see generally Ausubel et al., supra). Each of the above identified algorithms and the references are herein incorporated by reference in its entirety for all purposes. These algorithms are well known to one of ordinary skill in the art of molecular biology and bioinformatics. When using any of the aforementioned algorithms, the default parameters for \u201cWindow\u201d, gap penalty, etc., are usually used. Practitioners of the art molecular biology with average skill will recognize these parameters as: (a) the \u201cwindow\u201d is typically a 9, 10 or 11 nucleotide word length of sequence over which the homology is determined; and (b) gap penalty is a scoring value to prevent large gaps from occurring in reported alignments.","The BLAST algorithm is welt suited for determining percent sequence identity and sequence similarity. The BLAST algorithm is described in Altschul et al., J Mol. 215:403-410, (1990), the entire contents of which are herein incorporated by reference for all purposes. Several software programs incorporating the BLAST algorithm are publicly available through the National Center for Biotechnology Information (NCBI). These programs include the blastp, blastn, blastx, tblastn, tblastx, and PSI-blast software programs. Due to codon wobble or species differences, more informative homologies can be found by comparing the predicted protein sequence of a DNA query sequence to a protein sequence database. For this task, the Smith-Waterman or PSI-BLAST algorithms may be used. Similarly, for weak homologs, functional domains of proteins may be discerned by Smith-Waterman, HMM or Emotif algorithms. Software for performing HMM and Smith-Waterman analysis can be obtained from a variety of public sources and\/or from vendors that sell accelerated computer hardware to rapidly process large batches of sequences (e.g. Paracel, Pasadena, Calif. TimeLogic, Crystal Bay, Nev.). Software for EMotif\/Ematrix can be obtained from sources such as the Brutlag Bioinformatics Group, Stanford University, Stanford, Calif.","The BLAST heuristic search algorithm is optimized for speed and searches sequence databases accessible to server  for optimal local alignments to the input query nucleotide sequences. Databases which may be searched using the BLAST programs include the SWISS-PROT protein sequence database, GenBank database, the Genome Sequence database (GSDB), the European Molecular Biology Laboratory (EMBL) Nucleotide Sequence database, the DNA Database of Japan (DDBJ), and other like databases.","The BLAST algorithm identifies high scoring sequence pairs (HSPs) by identifying short words of length \u201cW\u201d in the query cDNA sequence, which either match or satisfy some positive-value threshold score \u201cT\u201d when aligned with a word of the same length in a database sequence. \u201cT\u201d is referred to as the neighborhood word score threshold (Altschul et al, supra). An \u201cX\u201d parameter is a positive integer representing the maximum permissible decay of the cumulative segment score during word hit extension. These initial neighborhood word hits act as seeds for initiating searches to find longer HSPs containing them. The word hits are then extended in both directions along each sequence for as far as the cumulative alignment score can be increased. Extension of the word hits in each direction are halted when the cumulative alignment score goes to zero or below, due to the accumulation of one or more negative-scoring residue alignments, or when the end of either sequence is reached. The BLAST algorithm parameters \u201cW\u201d, \u201cT\u201d, and \u201cX\u201d determine the sensitivity and speed of the alignment. Accordingly, the stringency of a BLAST search can be adjusted by appropriately setting the search parameters. However, if the search parameters are too loose, an excessive amount of biologically questionable \u201chits\u201d may be returned. The BLAST program uses as defaults a wordlength (W) of 11, the BLOSUM62 scoring matrix (see Henikoff & Henikoff, Proc. Natl. Acad. Sci. USA 89:10915 (1989)) alignments (B) of 50, expectation (E) of 10, M=5, N\u22124, and a comparison of both strands. Typically, the default parameters can yield from zero to scores of likely homologs for the input query DNA sequences.","In addition to calculating percent sequence identity, the BLAST algorithm also performs a statistical analysis of the similarity between two sequences (see, e.g. Karlin & Altschul, Proc. Natl. Acad. Sci. USA 90:5873-5787 (1993)). One measure of similarity provided by the BLAST algorithm is the smallest sum probability (P(N) or E-value as an expected value), which provides an indication of the probability by which a match between two nucleotide or amino acid sequences would occur by chance. For example, a nucleic acid is considered similar to a reference sequence if the smallest sum probability in a comparison of the test nucleic acid to the reference nucleic acid is less than about 0.01, more preferably less than about 0.001, and most preferably less than about 0.0001.","A further indication that two nucleic acid sequences or polypeptides are substantially identical is that the polypeptide encoded by the first nucleic acid is immunologically cross reactive with the polypeptide encoded by the second nucleic acid. Thus, a polypeptide is typically substantially identical to a second polypeptide, for example, where the two peptides differ only by conservative substitutions.","As is well known to one of ordinary skill in the art, results from a homology search or analysis include: a plurality of query sequences; a list of homologous (target) sequences; an E-Value that describes the probability that the original (query) sequence match with the target sequence could occur randomly; the annotation of the target sequence, if provided; an alignment of the query sequence to each target sequence; the percent identity of the query sequence to the target sequence; the hit length, or length of the sequence over which the percent identity is determined.","The complete homology analysis of a plurality of sequences according to an embodiment of the present invention is composed of a process described in FIG. . The output(s) from the process shown in  may be used as the input to step  in FIG. . The rationale for this sequential strategy of homology analysis is to automate the method of sequence classification. According to the embodiment shown in , input sequences  are subjected to BLAST analysis  against an internal database of sequences . Near identical homologs (E-value<1) are sieved and recorded as being strong homologs of previously classified entries  of the internal database. Those sequences failing this test, are subjected to blastn analysis  against the GenBank nucleotide (NT) and patent databases . Those sequences showing strong similarity (E-value <1with sequence length>200 nucleotides, 80% identity over 70% of the query sequence length) are classified as \u201cknown\u201d genes . Those sequences failing this test are subjected to Smith-Waterman analysis  against the protein databases of Swiss-Prot and the translated patent database . Those sequences with E-values<1with 80% identity over a segment length>50 amino acids are classified as \u201cknown\u201d genes  while sequences with an E-value >1are subjected in parallel to (a) HMM  and EMotif  analysis against the Swiss-Prot and GenBank non-redundant (NR) protein databases  and (b) BLASTN analysis  against the GenBank EST and genomic databases . Those sequences with an E-value<1after HMM or EMotif are scored as \u201cSimilar\u201d genes  while sequences with an E-value<1after the final BLASTN analysis  are classified as \u201cunknown\u201d . Any sequences failing this last test are classified as \u201cNovel\u201d .","Methods of the present invention extract relevant information from the homology analysis output as described above for each input DNA sequence, organize the information, and store it in a format which facilitates further processing and analysis of the information (step ). According to an embodiment of the present invention, the information extracted from the BLAST, Smith-Waterman and HMM search output is stored in a database. The information extracted and stored by the present invention during step  is shown by the database schema depicted in FIG. .  depict other database structures for storing information according to an embodiment of the present invention.",{"@attributes":{"id":"P-00090","num":"00090"},"figref":"FIG. 5","b":"120","sup":["e-50 ","e-50 "]},"For each homolog, the present invention stores, in database tables \u201cDNAsequence\u201d  and \u201cHomologyResults\u201d , the name of the sequence (attribute \u201cseqFile\u201d -and -), the sequence (\u201cSequence\u201d -), the quality scores or Phred values (Ewing, Hiller, Wendl & Green, Genome Research, 8:175-185, 1998), (\u201cQualityScores\u201d -), the accession number of any homolog, i.e. the GenBank identifier number (\u201cGID\u201d -), the best GID derived from BLAST analysis (\u201cBestBlastnGID\u201d -), the best GID derived from BLAST against the patent DNA database analysis (\u201cBestPatent-GID\u201d -), the best GID derived from Smith-Waterman analysis derived from the Swiss-Prot database (\u201cBestSW-GID\u201d -), the best GID derived from Smith-Waterman analysis of the patent (database \u201cBestPatent-SW-GID\u201d -), the best GID derived from the best human homolog in BLAST analysis (\u201cBestHumanBlastn-GID\u201d -), and the best GID derived from the best human homolog derived from Smith-Waterman analysis (\u201cBestHuman-SW-GID\u201d -). For any homolog, the algorithm (e.g. BLAST or HMM) used for the homology search is recorded (\u201cAlgorithm\u201d -), the frame of the predicted protein for protein comparisons (\u201cFrame\u201d -), the database searched (\u201cDatabase\u201d -), the GenBank annotation for any homolog (\u201cHitDescription\u201d -), the species of the annotation (\u201cSpecies\u201d -), the E-value (\u201cE-value\u201d -), the length of the alignment region (\u201cAlignLength\u201d -), the percent identity of the aligned sequences (\u201cPercentIdentity\u201d -), the length of the query in the alignment (\u201cQueryLength\u201d -), the length of the target in the alignment (\u201cTargetLength\u201d -), a number representing the fraction of the total query length represented in the hit region (\u201cALength\/QLength\u201d -), the start position of the query sequence in the alignment (\u201cQueryStart\u201d -), the position of the end of the query (\u201cQueryEnd\u201d -), the start position of the target sequence (\u201cTargetStart\u201d -), the end position of the target sequence (\u201cTargetEnd\u201d -), the query sequence in the alignment (\u201cQSequence\u201d -), the consensus of the alignment (\u201cConsensus\u201d ), and the target sequence in the alignment (\u201cTSequence\u201d -).","4) Annotation of DNA Sequences with Functional Information","Referring back to , server  then obtains (step ) descriptive annotative information on the biochemical function(s) and the physiological role(s) for the known genes from the plurality of cDNA sequences and stores the information in the database (step ).  depicts a simplified flowchart  showing processing performed by an embodiment of the present invention for obtaining descriptive annotative information for the known genes. As shown in , several different techniques may be used by the present invention to obtain the functional information. According to a first technique, the present invention accesses information sources containing functional information related to the known genes (step ). The information sources may include articles, published material, and other like material accessible to server . According to a specific embodiment, the present invention may use the accession numbers or the GenBank identifiers (GIDs) associated with the DNA sequences and their homologs to find the published material. Text processing tools may then be used by the present invention to automatically extract functional information from the information sources accessed in step  (step ). The extracted information may then be summarized (step ) and stored in the database (step ).","According to another technique, the present invention may obtain the functional information from databases storing functional information and which are accessible to server  (step ). Examples of such databases include databases provided by Proteome of Boston, Mass., DoubleTwist of Oakland, Calif., of Rockville, Md., the Genbank database of deposited DNA and protein sequence data, the SWISS-PROT protein database, the PubMed or Medline (NCBI) databases of abstracts derived from thousands of peer-reviewed biomedical journals, and other like databases. The Proteome databases are concise descriptions of known genes, their protein products and their functions and roles and known interactors as described in the current literature. The information extracted from the published material and genomic databases may then be summarized (step ) and stored in the database (step ).","The GenBank record of a cDNA or gene sequence commonly contains references to peer-reviewed publication information, stored in the Medline database about the gene. The Medline database can be accessed via the Internet via the PubMed interface. Alternatively, the GenBank record contains informative keywords related to the gene which may be used to perform broad topic searches on the Medline database. For example, protein products of genes participate in many processes essential to metabolism, development and reproduction. In some cases, a protein encoded by a gene may have more than one function and\/or more than one role. For example, the yeast inositol 1-4-5 triphosphate kinase enzyme adds a phosphate moiety to phosphoinositol\u2014an important component involved in signaling. However, this protein also can act as a regulatory scaffolding protein for transcription factors in the nucleus (Audrey R. et al. Science 287:2026-2029, 2000). Thus, this single protein can function as both an enzyme and a structural protein. Similarly, this gene product has two roles: it can participate in signaling processes and mRNA transcription. These instances are also examples of general pathways but further annotative information from the published literature may refine these topics to even more specific pathways. For example, the enzymatic activity might be most important for a growth hormone pathway and the structural role might be more important to a specific subset of transcription factors engaged in controlling cell division. In this invention, these relational links between genes and cellular or organismal processes constitute a web of interacting pathways that are extracted accurately and comprehensively.","There is a demand for biological information extraction from published material, such as abstracts, etc., in a comprehensive and consistent manner. Traditionally, extraction of information has been done manually with varying degrees of consistency and accuracy. With recent advances in information extraction technologies, various software programs have been developed to automate information extraction and to summarize the extracted information. Examples of such programs include programs provided by InXight Corp. of Santa Clara, Calif. Another example of a software package for information or knowledge extraction is the Crystal-Badger-Marmot suite from the Center for Intelligent Information Retrieval, Univ. of Massachusetts, Amherst, Mass. Such software programs have been applied to extract information from abstracts of published papers as well as from full-text papers. According to an embodiment of the present invention, these techniques are applied to generate tables of genes, tables of pathways composed of genes, and tables of relationships between and amongst genes and pathways. As described below, the relationship between or amongst genes is validated in a quantitative fashion.","According to an embodiment of the present invention, information extraction programs, such as those discussed above and others, are used to extract (step  in ) descriptive annotation information from information accessible to server  and to summarize (step  in ) the information. According to an aspect of the present invention, the annotative information is stored in a database.","According to the present invention, information may be extracted and stored for both the majority views and potentially multiple minority views. This is due to dramatic shifts in the understanding of biological systems over time. These shifts are also referred to as \u201cparadigm shifts\u201d (Kuhn, T., The Structure of Scientific Revolutions, Univ. Chicago Press 1962). According to these paradigm shifts, a minority view becomes accepted as being the correct interpretation after critical new data is acquired. The change in accepted \u201ctruth\u201d of a paradigm can be dramatic or subtle in various domains of knowledge, and in the realm of biology both extremes can occur\u2014hence the need for comprehensive collections of entity-relationships amongst genes, functions, roles and pathways. The need for dynamically storing both the majority and minority views becomes important when one realizes that much remains to be known about the laws of biology. This is substantially different from other techniques which only store information related to the majority view (e.g. T. Rindflesch, L. Tanabe, J. Weinstein & L. Hunter PSB 2000:517-528).","For example, for a given biological topic, perhaps about 51, about 75, or about 90 out of 100 published abstracts may describe a phenomenon as being caused by the interactions of genes A and B whereas a smaller subset of abstracts, perhaps about 10, about 25 or about 49 describe a more complex interaction between genes A and C prior to gene B. The former A-B model would be considered the consensus, \u201cmajority view\u201d model (a \u201ctruth\u201d) and the latter A-C-B model would be considered a \u201cminority view\u201d and likely regarded as being \u201cfalse.\u201d According to traditional bioinformatics techniques, only information related to strict \u201ctruths\u201d was maintained and information related to the minority view(s) was discarded to reduce the amount of data being stored.","According to an embodiment of the present invention, minority views (e.g. unusual or unexpected relationships between genes or metabolic pathways) are also stored in the database but assigned a lower reference score (see \u201cRefScore \u201cattribute -in table \u201cReference\u201d  in , \u201cFunctionScores\u201d attribute -in table \u201cFunction\u201d , \u201cRoleScores\u201d attribute -in table \u201cRole\u201d , and attributes -through -of table \u201cRefScore\u201d ) associated with the descriptive annotation of the known genes from the plurality of cDNA. The reference score (or their summary scores, \u201cFunctionScores\u201d -and \u201cRoleScores\u201d -) quantifies the \u201cacceptance\/majority opinion\u201d for an alleged role or function of a gene. Of particular importance to \u201cminority\u201d views is the extraction and recording of special circumstances or boundary conditions under which the phenomena or relationship amongst genes might exist.","The metric for evaluating a specific published reference article also assigns a score derived from the Citation Index database (Institute for Science Information, Philadelphia) which quantitatively ranks the impact of a given paper by the number of times that paper is subsequently referenced. For the most significant papers, a published article can be referenced thousands of times. The Citation Index also ranks journals with high impact but only from the same criteria of frequently-cited papers from the journals regardless of whether the published paper is ultimately revised or shown to be inaccurate or limited to a set of conditions. Hence, one embodiment of this invention provides a mechanism to take into account the quality of the information source. This is both general and a specific measure. In general, articles in journals respected by a consensus of biomedical and genomics practitioners are believed to be reliable. For example, a publication in journals with a recognized, rigorous peer-review process (e.g. Science, Nature, the Journal of Biological Chemistry, or the Journal of Clinical Investigations) would receive 100 points or>90 points whereas publication in \u201clesser\u201d journals (e.g. Journal of Antisense Research or Experimental Cell Research) would only receive 10 or 40 points.","Table 1 is an exemplary look-up table for general rankings of such biomedical journals. However, scores from table 1 may be adjusted because the information source's peer-review process can be dependent upon the reviewers for a given domain or the degree of democratic consensus of a journal's editorial board. A domain specific weighting factor is derived for the major journals and can be applied systematically while in other cases, a human annotator must make the judgment. The adjustment can range between 10 and 50% of the original score and an article in a \u201clower-quality\u201d journal can be upgraded or an article in a \u201chigher-quality\u201d journal can be downgraded.",{"@attributes":{"id":"P-d0e3274","num":"00002"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"left"}}],"thead":{"row":[{"entry":[{},"TABLE 1"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"Score","Journal"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"100","Nature"]},{"entry":[{},"100","Science"]},{"entry":[{},"97","Cell"]},{"entry":[{},"96","Neuron"]},{"entry":[{},"96","Nature Structural Biology"]},{"entry":[{},"95","Nature Cell Biology"]},{"entry":[{},"95","Nature Genetics"]},{"entry":[{},"95","Nature Medicine"]},{"entry":[{},"95","Nature Neuroscience"]},{"entry":[{},"94","Journal of Structural"]},{"entry":[{},{},"Biology"]},{"entry":[{},"94","New England Journal of"]},{"entry":[{},{},"Medicine"]},{"entry":[{},"92","Journal of Biological"]},{"entry":[{},{},"Chemistry"]},{"entry":[{},"92","Journal of Molecular"]},{"entry":[{},{},"Biology"]},{"entry":[{},"92","Neuroscience"]},{"entry":[{},"91","Journal of Bacteriology"]},{"entry":[{},"91","Journal of Cell Biology"]},{"entry":[{},"91","Journal of Immunology"]},{"entry":[{},"91","Journal of Neurochemistry"]},{"entry":[{},"90","Biochemistry"]},{"entry":[{},"90","Bioinformatics"]},{"entry":[{},"90","Immunology"]},{"entry":[{},"90","Journal of Clinical"]},{"entry":[{},{},"Investigation"]},{"entry":[{},"90","Journal of Internal"]},{"entry":[{},{},"Medicine"]},{"entry":[{},"90","Journal of Medicinal"]},{"entry":[{},{},"Chemistry"]},{"entry":[{},"90","Journal of Membrane"]},{"entry":[{},{},"Biology"]},{"entry":[{},"90","Journal of Neuroscience"]},{"entry":[{},"90","Journal of Virology"]},{"entry":[{},"90","Molecular Microbiology"]},{"entry":[{},"89","Cell Biology"]},{"entry":[{},"89","Genetics"]},{"entry":[{},"89","Journal of Experimental"]},{"entry":[{},{},"Medicine"]},{"entry":[{},"89","Journal of Neurobiology"]},{"entry":[{},"89","Neurology"]},{"entry":[{},"88","Infection and Immunity"]},{"entry":[{},"87","Journal of"]},{"entry":[{},{},"Neuroendocrinology"]},{"entry":[{},"87","Journal of"]},{"entry":[{},{},"Neuroimmunology"]},{"entry":[{},"87","Journal of Neurology"]},{"entry":[{},"87","Neuroendocrinology"]},{"entry":[{},"86","Neuroscience Letters"]},{"entry":[{},"85","Development"]},{"entry":[{},"85","Proceedings National"]},{"entry":[{},{},"Academy of Sciences"]},{"entry":[{},"85","Immunogenetics"]},{"entry":[{},"85","Journal of Cell Science"]},{"entry":[{},"85","Journal of Infectious"]},{"entry":[{},{},"Diseases"]},{"entry":[{},"85","Journal of"]},{"entry":[{},{},"Neurophysiology"]},{"entry":[{},"85","Molecular Pharmacology"]},{"entry":[{},"85","Molecular and Cellular"]},{"entry":[{},{},"Biology"]},{"entry":[{},"84","Brain Research"]},{"entry":[{},"84","Developmental Biology"]},{"entry":[{},"84","Developmental"]},{"entry":[{},{},"Neuroscience"]},{"entry":[{},"84","Molecular Biology of the"]},{"entry":[{},{},"Cell"]},{"entry":[{},"83","Cancer"]},{"entry":[{},"82","Analytical Biochemistry"]},{"entry":[{},"82","Journal of Endocrinology"]},{"entry":[{},"82","Journal of Experimental"]},{"entry":[{},{},"Biology"]},{"entry":[{},"82","Journal of Histochemistry"]},{"entry":[{},{},"and Cytochemistry"]},{"entry":[{},"82","Journal of Molecular"]},{"entry":[{},{},"Endocrinology"]},{"entry":[{},"82","Molecular Immunology"]},{"entry":[{},"82","Natural Immunity"]},{"entry":[{},"82","Nucleic Acids Research"]},{"entry":[{},"80","Brain"]},{"entry":[{},"80","Journal of Physiology"]},{"entry":[{},"80","Molecular and Cellular"]},{"entry":[{},{},"Endocrinology"]},{"entry":[{},"80","Molecular and Cellular"]},{"entry":[{},{},"Neurosciences"]},{"entry":[{},"78","Genome Research"]},{"entry":[{},"78","Journal of Clinical"]},{"entry":[{},{},"Epidemiology"]},{"entry":[{},"78","Pharmacology"]},{"entry":[{},"75","Genomics"]},{"entry":[{},"75","Journal of Cellular"]},{"entry":[{},{},"Biochemistry"]},{"entry":[{},"75","Journal of Investigative"]},{"entry":[{},{},"Dermatology"]},{"entry":[{},"75","Journal of Medical Genetics"]},{"entry":[{},"75","Journal of Medical Virology"]},{"entry":[{},"75","Journal of Psychiatric"]},{"entry":[{},{},"Research"]},{"entry":[{},"75","Nitric Oxide"]},{"entry":[{},"75","Toxicology"]},{"entry":[{},"74","Brain and Development"]},{"entry":[{},"73","Cancer Immunology,"]},{"entry":[{},{},"Immunotherapy"]},{"entry":[{},"70","Brain Injury"]},{"entry":[{},"70","Cytogenetics and Cell"]},{"entry":[{},{},"Genetics"]},{"entry":[{},"70","Glia"]},{"entry":[{},"70","Journal of Biotechnology"]},{"entry":[{},"70","Obesity Research"]},{"entry":[{},"69","Immunopharmacology"]},{"entry":[{},"68","Cellular Signalling"]},{"entry":[{},"68","Immunology and Cell"]},{"entry":[{},{},"Biology"]},{"entry":[{},"68","Stem Cells"]},{"entry":[{},"68","Stroke"]},{"entry":[{},"68","Synapse"]},{"entry":[{},"67","Immunology Letters"]},{"entry":[{},"65","Cell Motility and the"]},{"entry":[{},{},"Cytoskeleton"]},{"entry":[{},"65","Gene"]},{"entry":[{},"65","Hormone Research"]},{"entry":[{},"65","Journal of Cognitive"]},{"entry":[{},{},"Neuroscience"]},{"entry":[{},"65","Journal of the Neurological"]},{"entry":[{},{},"Sciences"]},{"entry":[{},"65","Pharmacological Research"]},{"entry":[{},"63","American Journal of"]},{"entry":[{},{},"Physiology"]},{"entry":[{},"62","American Journal of"]},{"entry":[{},{},"Medicine"]},{"entry":[{},"62","Journal of Comparative"]},{"entry":[{},{},"Neurology"]},{"entry":[{},"60","American Journal of Human"]},{"entry":[{},{},"Genetics"]},{"entry":[{},"60","Cell Proliferation"]},{"entry":[{},"55","Journal of Affective"]},{"entry":[{},{},"Disorders"]},{"entry":[{},"50","DNA and Cell Biology"]},{"entry":[{},"45","Anesthesiology"]},{"entry":[{},"41","Biochemical Journal"]},{"entry":[{},"40","Anesthesia and Analgesia"]},{"entry":[{},"40","Biochemical Pharmacology"]},{"entry":[{},"40","Cell Biochemistry and"]},{"entry":[{},{},"Function"]},{"entry":[{},"40","Cell and Tissue Research"]},{"entry":[{},"40","Cerebral Cortex"]},{"entry":[{},"40","Experimental Cell Research"]},{"entry":[{},"40","Histochemistry and Cell"]},{"entry":[{},{},"Biology"]},{"entry":[{},"35","Journal of Dermatological"]},{"entry":[{},{},"Science"]},{"entry":[{},"20","Chemistry and Biology"]},{"entry":[{},"20","Genes, Chromosomes and"]},{"entry":[{},{},"Cancer"]},{"entry":[{},"20","International Journal of"]},{"entry":[{},{},"Biochemistry and Cell"]},{"entry":[{},{},"Biology"]},{"entry":[{},"20","International Journal of"]},{"entry":[{},{},"Biochemistry and Cell"]},{"entry":[{},{},"Biology"]},{"entry":[{},"10","Archives of Biochemistry"]},{"entry":[{},{},"and Biophysics"]},{"entry":[{},"10","International Journal of"]},{"entry":[{},{},"Antimicrobial Agents"]},{"entry":[{},"10","International Journal of"]},{"entry":[{},{},"Cancer"]},{"entry":[{},"10","International Journal of"]},{"entry":[{},{},"Experimental Pathology"]},{"entry":[{},"10","Journal of Antisense"]},{"entry":[{},{},"Research"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"Table 1, an exemplary look-up table for general rankings of biomedical journals.","while subject to a degree of subjectivity, these standards for ranking journals for their domain preferences are the same as those used by faculty-tenure review committee in major medical schools in the United States of America in order to evaluate the publication record of a tenure-candidate. Similarly, human experts in various domains recognize that certain information sources can have a predisposition to disregard or highly regard certain authors or types of submitted work. Since the editorial board and peer-reviewers of journals change with time, the tables for grading journals are not static but must be revised over time as reviewers or editors specific to domain specialties change. In combination with the Citation Index of impact journals, these criteria enable the scoring of a reference's support of gene's annotation.","Another variable used in the evaluation of the experimental support for an alleged role or function for a gene is a \u201cfollow-on\u201d parameter. Reliable experimentalists often will publish a series of papers in reputable journals. They may publish on the same gene or encoded protein (\u201cGeneRef\u201d -attribute of table \u201cFollowOnWork\u201d  in , or \u201cProteinRef\u201d -), a close homolog (\u201cFamilyMemberRef\u201d -), another gene in the same pathway (\u201cPathwayRef\u201d -) or the same gene or pathway in another organism (\u201caltOrganismRef\u201d -). When a large body of work from an individual author or group of authors accumulates, then the probability of \u201ctruth\u201d is high. In contrast, a single publication by an author that alleges unusual relationships amongst genes that fails to engender follow-on work (as roughly measured by the Citation Index) by the original author or others has a lower probability of \u201ctruth\u201d which is reflected by a lower reference score (\u201cRefScore\u201d -). An intermediate reference score occurs where a single publication triggers much work by other investigators, e.g. a high Citation Index but low \u201cfollow-on\u201d value. Thus, this strategy compensates for the overall weakness of the Citation Index\u2014by merely enumerating the occurrences of a referenced paper, the Citation Index may not be accurately represent the relatedness of subsequent work.",{"@attributes":{"id":"P-00106","num":"00106"},"figref":["FIG. 7","FIG. 7"],"b":["160","170","180","190","200","210","220","230","14","160","160","160","130","130","170","160","180","160","190","160","170","180","190","200","200","200","170","170","170","170","170","170","180","180","180","180","180","180","180","180","0","180","180","180","200","170","180","180","180","190","190","190","190","190","190","190","190","190","200"],"i":["a","b","f","c","d","e","a","a","b","c","d","e","f","a","b","c","d","e","f","g","h","i","j","k","a","g","l","b ","c","b","c","d","e","f","g","h","j","k","a"]},"Referring further to , tables are shown to record the information about any pathway or reference . For any pathway (\u201cPathway\u201d -in table \u201cPathway\u201d ), a role may be assigned (\u201cRole\u201d -), genes of the pathway listed (\u201cGeneList\u201d -) and the location of the pathway identified (\u201cLocations\u201d -). For any reference, a unique identifier (\u201cRefID\u201d -) is recorded, the authors listed (\u201cAuthor\u201d -), the article title (\u201cTitle\u201d -), the journal in which the article was published (\u201cJournal\u201d -), the volume of the journal (\u201cVolume\u201d -), the page numbers of the article (\u201cPage\u201d -), the year of the article's publication (\u201cYear\u201d -), and the reference score link (\u201cRefScore\u201d -). The reference score link -corresponds to the \u201cRefScore\u201d object\/table  which also contains the reference identifier (\u201cRefID\u201d -), the citation index value (if any) (\u201cCitationIndex\u201d -), the topic field (e.g. immunology or neurobiology) (\u201cDomain\u201d -), a domain weight-adjusted value for the journal quality, as described above, (\u201cJournalRigor\u201d -), and the link to follow-on work table  (\u201cFollowOnWork\u201d -). The follow-on table  consists of a reference to any subsequent work in which the same gene (\u201cGeneRef\u201d -) or protein (\u201cProteinRef\u201d -), or homologous gene (\u201cFamilyMemberRef\u201d -), or the same pathway (\u201cPathwayRef\u201d -) or alternate organism (\u201caltOrganismRef\u201d -) was studied by the original investigators.","5) Expression Profiling and Cluster Analysis","Referring back to , the present invention then obtains (step ) and stores (step ) expression profile data for the genes and their homologs. The expression profile data for a gene describes when and where the gene is expressed (i.e. transcribed to RNA). Profiles can be created for genes in cells or tissues under the influence of a drug; as a cell or tissue develops; during changes to the physiological state of the cell or tissue; in response to the development of disease in humans or an animal model; changes in response to a transgene or gene knock-out; and the like. For example, the expression profile data may indicate whether a gene is up-regulated\/down-regulated during a stroke.",{"@attributes":{"id":"P-00110","num":"00110"},"figref":["FIG. 8","FIG. 8"],"b":["240","250","260","270","240","240","240","1","1","240","2","2","240","240","240","240","240","250","250","250","250","250","250","250","250","250","250","250","250","250"],"i":["a","b","c","d","e","f","g","h","a","b","c","d","e","g","h","i","j","k","l","m"]},"The probe data table  contains attributes for the probe identifier (\u201cProbeID\u201d -), the date of probe generation (\u201cDate\u201d -), the type (first strand cDNA or double-stranded cDNA) of probe (\u201cType\u201d -), the biological model (\u201cModel\u201d -), the identifier for the preparation of RNA (\u201cRNAprep\u201d -), the labeling (radioactive or fluorescent) method (\u201cLabelType\u201d -), the time point at which the RNA was collected (\u201cTimePt\u201d -), the biological state of the probe's mRNA origin (\u201cState\u201d -), and a comment (\u201cComment\u201d -).","The raw data table  contains attributes for the experimental identifier (\u201cExptID\u201d -), the sequence name (\u201cseqFile\u201d -), the probe name (\u201cProbe\u201d -), the raw intensity value (\u201cRawValue\u201d -), the local background or normalization factor (\u201cLocalBgnd\/factor\u201d -), and the arithmetically corrected intensity value (\u201cCorrectedValue\u201d -).","Referring back to , the present invention then performs clustering analysis on the behavior of DNA sequences in expression profile studies (step ). According to clustering analysis, data complexity is reduced by partitioning the genes into groups or \u201cclusters\u201d that have similar attributes. These attributes can be the behavior of genes monitored over multiple time points in response to an injury, onset of disease or altered physiological state (e.g. intensity or ratio of intensities resulting from hybridization of a gene set with probes derived from normal and diseased tissue). Also, these attributes can simply be the response of genes from cells, tissues or animals treated with multiple concentrations (e.g. 5, 6 or 7 concentrations) of many drugs (e.g. 10, 100, 1000 or 10,000) with differing mechanisms of action at a single time point. These attributes can also be the response of cells or animals subjected to many altered physiological states (e.g. elevated or diminished nutrients, ions or temperature, transient ischemia, shock, anxiety, enriched environment, discomfort or depression) or genetic states (e.g. transgenic or gene knock-out animals) monitored at a single time point relative to untreated cells or tissues. The result of clustering gene expression data is clusters of genes with similar expression profiles.","An embodiment of the present invention implements a method of gene clustering that is tuned to the simplified, yet specific nature of the array data itself. In order to reduce data complexity, many clustering methods have been applied to gene expression profile data: these include hierarchical, K-means, self-organizing maps (Tamayo et al. PNAS 96:2907-12), or support vector machines (M. Brown et al. PNAS 97:262-7). An embodiment of the present invention uses a K-means distance with Euclidean distance or other distance metrics (provided by Partek of St. Louis Mo.) because of its ability to efficiently cluster data in an automated unsupervised manner. One of the common criticisms of K-means clustering is that the number of clusters must be determined a priori. However, the present invention uses the Davies-Bouldin algorithm (IEEE Transactions on Pattern Analysis and Machine Intelligence, Vol. PAM1-1, April 1979) which determines the optimal number of clusters based upon the dispersion and flatness of clusters.","According to an embodiment of the present invention, the present invention clusters the genes based on time-course data, as described by the expression profile data. According to a specific embodiment of the present invention, packages provided by Partek Inc. and\/or SAS Institute, Incorporated of Cary, North Carolina may be used to perform the clustering analysis. For time-course data, the clustering analysis may also include causality analysis to predict ordered relationships between clusters on a time basis. Causality analysis is performed using an auto-regressive and auto-correlative method using software packages such as the Statistical Analysis Software from SAS Institute, Incorporated. The results from the clustering analysis are stored in a database (step ). The cluster analysis results are inserted into the array data table  of FIG. : for each gene (\u201cseqFile\u201d -), the clustering method (\u201cClusterMethod\u201d -), a cluster number (\u201cCluster\u201d -), the total number of clusters (\u201cTotalClusters\u201d -), and the cluster order (\u201cClusterOrder\u201d -).","The type of clustering method(s) used to analyze array data may be selected based on factors such as a priori knowledge about the behavior of the immobilized genes; the composition of the gene set itself; the choice of array technologies, etc. Array technologies take different forms, for example cDNA and oligonucleotide arrays, etc. Since the oligonucleotide arrays can have a higher density than cDNA arrays, an effort has been made to increase the number of sequences per unit surface area in order to gain thoroughness. Thus, many array chip designs seek to deposit large numbers of nucleotide sequences per chip, e.g. species-specific chips (mouse, rat or human chips from Affymetrix, Santa Clara, Calif.); genes representative of a field (apoptosis, cancer or neurobiology chips) and the like. However, analysis of such chips can be complicated by the fact that many sequences or polynucleotides on the chip are irrelevant to the biological system being studied.","In one embodiment of the present invention, the analysis of gene clusters is simplified by the use of immobilized groups of polynucleotides that are disease- or physiology-specific, or clustered in a biologically relevant group. Such collections of genes can be generated, for example by a method that enables the identification of genes expressed at a measurable level higher in one state than another. For example, in tumors or animals subjected to ischemia, those skilled in the art of molecular cloning can identify and isolate cDNA clones and derive the sequences thereof for genes whose expression is elevated 2, 3 or 10 fold higher in the altered physiological state; e.g. differential display and subtractive cloning are two such methods. Examples of genes and groups of genes are identified in co-pending U.S. patent application Ser. No. 09\/627,362, filed on Jul. 28, 2000. The number of disease-related or physiologically-related genes may range from about 1000, about 6000, about 10,000, or about 20,000 per chip.","When analyzed by principal components analysis, typically up to 90% of the variability in the gene expression profile data generated by arrays of 6000-10,000 disease- or paradigm-specific gene targets can be explained by the first 3 principal components or eigenvectors. With a large number of genes unrelated to the biological paradigm of the probe (e.g. 40,000-60,000 genes present on some general arrays), the data variability is likely explained by many more principal components, which makes it difficult to analyze more than any 3 of all principal components in 3-dimensional space. For analysis of such general array data, other clustering methods might be more appropriate, such as hierarchical clustering, although optimal hierarchical clustering is highly iterative and false clusters are often generated.","In order to infer the time-order of gene clusters derived from the above, it is possible to calculate likely causality by a moving auto-regressive analysis. A time-order is a linear ranking of clusters by a deduced set of relationships ordering the first possible cluster relative to other clusters in an iterative process. A biological example of this problem is the goal of understanding which genes respond earliest to an injury or infection, followed by the elucidation of time of activation of subsequent, related or unrelated genes. A ordered set of clusters from expression profile data is achieved initially by selecting a representative subset of genes near the centroid of each cluster (e.g. 2, 5 or 10 representing about 1-10% of the total number of genes) and performing a moving auto-regressive test against the remaining genes of the monitored population of genes (e.g. 2, 5 or 10 genes compared to all 6000 or 10,000 genes) from all clusters (Statistical Analysis Software of SAS Institute, Incorporated, Cary, N.C.). The ranked order of clusters is stored in \u201cClusterOrder\u201d (-) in step .","The accuracy of ordering clusters is dependent on the completeness of the calculation, but calculation of cluster order is computationally intensive. For example, according to a specific embodiment, the above calculation, for example requires about 24 hours on a standard single CPU Unix workstation with 1 gigabyte of RAM; e.g. a Sun Ultra10 workstation with 300 MHz CPU. This time-series analysis is only applicable to datasets with regularly spaced time-points (e.g. 10, 20 or 40 instances spaced 30 min, 1 hr or 3 hrs apart). The time-resolution of the causality analysis is dependent upon the density of intervals over the entire course experimental course. For the highest resolution of time-ordered relationships amongst clusters, 20, 50, or 100 time-points are preferable. For the highest accuracy amongst clusters, a comprehensive auto-regression is calculated provided sufficient computer power (e.g. 6000 genes compared to 6000 genes or 10,000 genes compared to 10,000 genes requires supercomputer ability or the efforts of a cluster of workstations such as a Beowulf cluster).","6) Obtaining Pathway Information","The computer-implemented methods of this invention employ information extraction techniques to generate summarized information about the biological relationships between genes in a list of genes. A goal of these methods is to automatically and systematically generate and present summarized information about the biological relationships between the genes of a list. Without these methods, users typically spend considerable time manually searching databases to find information about a particular gene and its relationships to other genes before the particular gene product can be developed as, e.g. a drug target, a therapeutic protein etc.","Referring back to , after the clustering analysis, the present invention obtains pathway information regarding the biological relationships between genes in a list, the list representing a cluster or any other group of genes of interest. Pathway information can be accessed from public databases such as MEDLINE, OMIM and the U.S.P.T.O. patent databases through information retrieval and extraction procedures. Pathway information can also be gathered from pathway databases such as the Kyoto Encyclopedia of Genes and Genomes (KEGG) or the Munich Information Center for Protein Sequences (MIPS), as described earlier. One embodiment of the invention is that data from a public databases are downloaded to a private network, thus providing a set of internal data.","The determination of biological relationships is normally carried out in three steps. Firstly literature relevant to genes in a list of genes is identified, downloaded and stored. Second, information regarding relationships between two or more gene products is extracted from the literature using information extraction techniques, and third, this information is cross referenced and a graph is built to display the cross-referenced information. Typically the data produced from these steps, such as a list of interactions, and a directed graph, is stored in the expert database  and is available for viewing along with gene expression data, allowing the user to quickly determine whether gene expression results are meaningful, based on other evidence in the public database , extracted and summarized by these procedures.","In one embodiment, the automated extraction, processing, cross-referencing and presentation of information regarding the biological relationships of genes and gene products, or other products requires retrieval of a collection of journal articles by word search, performing natural language processing (NLP) and information extraction, saving a list of interactions, cross-referencing the interactions and visualizing the cross-referenced interactions. Within NLP, several processes may take place. Tokenization pre-processes text into words and phrases, separated by white space and punctuation. Stemming is used to find and standardize variations of the same word, for example converting \u201cregulation of\u201d to the stemmed form \u201cregulate\u201d, using the SPECIALIST Lexicon medical vocabularies of the National Library of Medicine's Unified Medical Language System (UMLS; Humphreys (1998) J. Am. Med. Inform. Assoc. 5, 1-11). Finally tagging algorithms annotate the tokenized and stemmed text with parts of speech (i.e. syntactic) or class (semantic) information to define nouns, verbs, adjectives and any other parts of speech, and the classification of the words. The combination of syntactic tagging and semantic tagging can further determine which nouns are the subject of which verbs. Tagged text is then presented to information extraction algorithms.","Information extraction systems may recognize noun phrases that contain verbs of interest (e.g. Sekimizu (1988) Genome Inform. Ser. Workship Genome Inform. 9, 62-71). For example, a system may identify the verbs activate, bind, interact, regulate, encode, signal and function, any other word used to describe interactions commonly found in MEDLINE abstracts, such as those described by Stephens et al (Pac Symp Biocomput. (2001) 483-95) or any other word that can describe interactions found within the SPECIALIST Lexicon medical vocabularies of the National Library of Medicine's UMLS (Humphreys (1998) J. Am. Med. Inform. Assoc. 5, 1-11). This system finds noun phrases that are contained within phrases or sentences with one or more of the specified verbs. A parser using supplied rules, comprising 1) the specification of semantic and syntactic components, 2) specification of the output form if the supplied rule is successful and 3) a system for assigning a quality score or grade based on how well the input fits the rules. The output from such an information extraction system may be X interacts with Y, or Y regulates Z. With the addition of nested actions, modifiers of objects and actions, relations between actions, relative clauses and conjunctions to template recognition and HMM models, more complicated relationships can be determined, for example, X interacts with Y in cancerous cells.","Once the relationships between genes have been determined, they can be displayed in the form of a directed graph. Several different graphing programs and algorithms can be used for generating directed graphs, as is known by one of skill in the art, for example those described and referenced by Becker (Bioinformatics 17: 461-467 and references therein). One suitable graphing tool is the Java applet for visualizing protein-protein interactions developed by Mrowka (Bioinformatics (2001) 17: 669-670). The applet is based on the Graph.Java applet distributed in the Java development distribution of SUN Microsystems. A directed graph is displayed on FIG. . The thickness, color, texture and\/or transparency of the graphical links (edges) between the genes (nodes) may represent a confidence, based on the rankings of journals providing the information extracted, that the interaction is a real biological phenomenon.","A diagrammatic representation of an input graphical interface into which a user enters a list of genes in one embodiment of the invention is shown in FIG. . User inputs a list of genes , preferably through an input graphical user interface of a client machine - and -. Web browser  displays fields for hand entry of a list of genes , a field for comments  and means for identifying a previously made list of genes , for example an output list of genes from a clustering algorithm wherein the genes in a list of genes form a cluster, either selected from the gene expression database or selected by a user. Optionally, the system may automatically extract information from lists of clustered genes stored in the database, without input from the human user. User interface  also includes a button, or other means, for initiating the execution of the described pathway model construction methods. The input information about a gene could be a seqFile number -, a BestHits name -, or any other identifier or key that can link the input gene information with a particular sequence or particular gene name through the DNA sequence database  or the annotation summary of the expert database . In one embodiment, the user inputs a list of gene names , in another embodiment the user enters a list of seqFile identifiers, and in another embodiment, the user enters a list of other identifiers that can be corresponded to seqFile identifiers using a conversion table. If user has put in a list of identifiers that correspond directly or indirectly to a seqFile number (-), a query will be used to extract names for genes represented by the input list. The name of the gene can be extracted as the BestHits field () of the DNA sequence database () or the BestHits field (-) of the annotation summary ().",{"@attributes":{"id":"P-00129","num":"00129"},"figref":"FIG. 10","b":["300","305","310","315","14","320","325","14"]},"After all relevant literature has been down-loaded and stored as text files, gene names and their synonyms are tagged (). Tagging is a means by which words, in particular nouns and verbs, are labeled within larger bodies of text. In one embodiment, genes names found within the HUGO curated list of gene names, the SWISS-PROT list of protein names the Galton Library of the Medical Research Council, UK, or any other gene, chemical, process, or agent listed in the UMLS can be used for tagging. In a particular example (), the tagging mechanism, using Extensible Markup Language (XML), gene names are labeled by placing \u201c<GENE>\u201d before the gene name and <\/GENE> after the gene name. In a body of text, Gene A would be labeled as follows: <GENE>GeneA<\/GENE>. Several different methods of tagging can be used in information extraction, as is known to one of skill in the art. An example of an exact match tagging algorithm is the Marmot program, developed by the University of Massachusetts.","Once gene names are tagged within the downloaded text files, information regarding biological relationships is extracted from the text files (). Information extraction may require a list of words that describe interactions between a supplied list of nouns. Verbs used to describe the interactions between different genes can be \u201cactivate\u201d, \u201cassociate\u201d, \u201cinteract\u201d, \u201cbind\u201d, \u201cinhibit\u201d, \u201cregulate\u201d, \u201cup-regulate\u201d, \u201cdown-regulate\u201d or \u201ccomplex\u201d, however any verbs describing relationships as listed in the UMLS, any verb commonly used in the MEDLINE database, or any other useful verb can be used.","Information extraction relies on such techniques as template filling and Hidden Markov Models (), although several other techniques for information extraction are known to one of skill in the art. An exemplary list of templates used in the preferred embodiment are as follows: \u201cGene A can activate Gene B . . . \u201d, \u201cGene A is associated with Gene B . . . \u201d, \u201cThe interaction of Gene B and Gene A, . . . \u201d,\u201cGene A binds to the N-terminal domain of Gene B, . . . \u201d and \u201cGene B induces inhibition of Gene A in . . . \u201d.","The output of information extraction in this example is a summary of an interaction between two genes, for example \u201cGene A inhibits Gene B\u201d  wherein two genes interact by an interaction type, in this case the type is inhibits. Further phrasal recognition and templates examples can be used to develop more complex relationships, for example \u201cGene A inhibits Gene B in particular conditions\u201d, for example in cancer, or when Gene A gene product has been phosphorylated. In one embodiment of the present invention, information extraction is performed by one or a combination of the techniques of template filling, Hidden Markov Models and phrasal recognition. Information extraction may also employ text comparison algorithms such as BLAST, Teiresias or Swith-Waterman to identify phrases or words that are similar to phrases or words of interest.","Template-filling is a NLP technique which utilizes domain-specific rules for information extraction. A template defines a set of rules that syntactically and semantically generalizes slots or positions and relations of words or word phrases. For example, a simple template may define the rule to extract \u201cGene A\u201d, \u201cinhibit\u201d, and \u201cGene B\u201d from the phrase \u201cGene A can inhibit Gene B.\u201d The template may be defined as \u201c[noun (GeneName)] [verb phrase (Interaction Verb)] [noun (GeneName)].\u201d The slots of the rule are contained within the brackets. The syntax of the rule is defined by (1) the grammatical order and (2) the part-of-speech of the slots. The semantics rules are defined within parentheses of each slot brackets, ie \u201cGeneName\u201d and \u201cInteractionVerb.\u201d The complexity of templates can vary depending on the complexity of the syntactical and semantic structure of phrases from which information is being extracted. Ultimately, a group of templates can be used to match and extract information from domain literature.","Markov models and Hidden Markov models (Bernard Merialdo (1994).","Computational Linguistics 20(2):155-172), or HMMs, are finite-state, statistical models of random sequences. In NLP, HMMs can be used to model a sequence of words or phrases by defining a set of states, ie words, and probabilities of movement between states, ie syntactic or semantic rules between the words. HMMs can be trained to probabilistically match a specific set of sentences. A trained HMM can be used to extract the best states matched in a sentence, ie underlying words and phrases, and probability of match.","Machine learning algorithms, such as the C4.5 decision tree algorithm, are another type of extraction NLP technique. The University of Massachusetts. CIIR \u201cBadger\u201d tool utilizes trained decision trees to extract information from text. Decision trees utilize a branching structure of states to define a set of cascading rules by which information can be extracted. Decision trees are trained with a syntactically and semantically tagged set of sentences to generalize their structure to optimally extract information. Neural networks are another example of machine algorithm used for information extraction in NLP (B. Widrow, D. Rumelhart, and M. A. Lehr. Neural Networks: Applications in Industry, Business, and Science. Comm of the ACM, volume 37, number 3, pages 93-105, March 1994).","Using information extraction techniques such as those described above, a list of biological interactions is created and stored  in a database. This list comprises many pairs of interactors, for example Gene A and Gene B, and an interaction type associated with the pair, for example, \u201cinhibits\u201d . Other information, such as the source of the information, for example a journal citation and\/or the author list, can also be stored with the interactions. As a final step in the information extraction process the stored relationships for all tagged genes is queried with the original input gene list  and a table is created describing only those interactions that involve a gene name or its synonym on the original input gene list . This process may be iterative, in that the interactors of genes on an original input gene list may be used to populate another input gene list, and the process can be initiated again. In this embodiment, relationships between distantly-interacting genes can be identified.","The table describing interactions involving gene names on the original input gene list can be presented to the user in many ways, including as a simple table, a two-dimensional interaction matrix, or many different types of graphs. In , a pathway model is constructed  and a directed graph is generated  and visualized . A graphical display of the output user interface is shown in FIG. . Web browser  contains three windows, a window  showing a directed graph  with a scroll bar , a window  showing literature corresponding to an interaction, and a window  showing gene expression information derived from gene expression profiling experiments. Thus, gene expression data for a group of genes or DNA sequences can be viewed simultaneously with a graph of interactions between a group of genes or DNA sequences.","The preferred method for presenting this interaction information to the user is by cross-referencing the interaction information and generating a directed graph , wherein a gene can be represented by a node , and the relationship is represented by the edges of the graph . The nodes are typically labeled with the gene name; and an edge value, corresponding to the strength of the interaction, can be used to label the edge. The edge may optionally be labeled with the relationship, and hyperlinks leading from the edges can be used to download, via a query of the expert database, information such as the abstracts and publications that were used to establishing the relationship.","Graphing algorithms may typically take into consideration interaction scores based on citation indexes scores or author scores, as described above, and how many times a particular reaction has been referenced for edge values. Gene expression information can also be taken into consideration in the calculation of edge values, for example if two genes are co-regulated according to data in the gene expression database, the edge value for a relationship between these two genes may be higher. Graphs are stored in the expert database for viewing by a user. In a preferred embodiment, the graphs are dynamically generated per user request, in order to incorporate updated data.","In one embodiment of this invention, the process of gathering and presenting information regarding the biological relationships between genes produces several tables that can be part of the expert database. A simple schema diagramming these tables, the relationship between these tables, and how the tables are linked to the expert database of , is shown in FIG. . One of skill in the art would recognize that there are several schemas for such a set of tables.","The process is initiated when the user inputs a list , usually of seqFile Ids  (from FIG. ), into the input interface of a client machine. Querying the BestHits table  using a seqFile ID  returns the BestHitID  or BestHitIDs corresponding to the given seqFile ID . For a given seqFile ID, zero, one or many BestHitIDs can be obtained via this query. The Accession number , Functional Annotation , Pscore , Rank , and Algorithm  for each BestHitID  are also obtained via query of the BestHit table . The BestHitID  with the highest Rank for a seqFile ID  is entered into a GeneGeneList table  that contains at least two fields: the BestHitID  and the GeneListID , a unique ID that has been assigned to each list of genes by the database management system. This process, of obtaining the BestHitID  with maximal Rank for each seqFile ID  and filling the GeneGeneList table , is repeated for each seqFile ID in the input list . In the GeneList table , other information about the list of genes, keyed by GeneListID , such as the date  (AddDate), who entered the list  (EnteredBy), the method by which the list was generated  (GeneratedBy) and additional comments  (Comment) can be entered. After the GeneList table  has been populated, the information retrieval and information extraction processes are carried out for all genes associated with a particular input list, and each process produces a pathway model. Each pathway model is assigned a unique ID by the computer, PathwayModelID , and information such as the input list to which the model corresponds  (GeneListID), the date performed  (AddDate), how the pathway was generated  (GeneratedBy) and general comments on the pathway  (Comment) are stored in the PathwayModel table . The PathwayModel2interaction table  defines the interactions of a pathway model using the PathwayModelID . For a given PathwayModelID , zero, one or more InteractionIDs  may exist in the PathwayModel2interaction table . The unique InteractionID  links to the Interaction table which identifies a literature-referenced  (RefID), gene-to-gene (Gene1ID , Gene2ID ) interaction, added on a date  (AddDate), the interaction type  (Type), specific experimental evidence  (Evidence), confidence in the experimental evidence  (Score) and a general comment  (Comment) are stored in the Interaction table. In a further embodiment of the invention, a plurality of BestHitIDs , are used to generate various pathway models. A further embodiment would utilize not only gene-to-gene interaction data, but also utilize gene-to-disease, gene-to-pathway, gene mutation, and gene knockout data to define pathway models.","7) Using the Expert Database","The information stored in the database according to the present invention facilitates the identification of candidate genes (step  in FIG. ). Identification of candidate genes results from the viewing of gene expression information in combination with the function(s), role(s) and\/or pathway(s) information about groups of genes. The reference score-based assignments for either majority or minority view annotations of function(s), role(s) and\/or pathway(s) enables the identification of new or serendipitous relationships. Such biological novelty, i.e. the unexpected up- or down-regulation of a gene in the context of an existing or new pathway, can be one of the hallmarks of candidate genes. For example, in a signaling pathway, study of a disease model may reveal that one, two or three known phosphodiesterases are up-regulated in the context of a pathway not normally characterized by those enzymes. Or, a new family member of this enzyme class might be discovered to be up-regulated along with the expected enzyme. Both are examples of candidate genes revealed by the combination of annotated DNA sequences and expression profiling data\u2014particularly if the published literature contained an obscure reference to such a relationship under abnormal circumstances dissimilar to the conditions of the experimental paradigm. The latter result would be significant due to the redundancy of biological systems. Conversely, if 7, 8 or 9 of 10 genes of a well known pathway are found to be up-regulated in a disease or injury model (as determined by a comparison of all pathways of each gene expression profile cluster), then the 1, 2 or 3 genes that failed to be induced (as determined by a query comparison to the pathway database) might also be considered candidate genes. In this example, the user might conclude that a new inhibitor is blocking the 1, 2, or 3 missing genes and hence blocking the inhibitor might diminish the pathology or improve recovery. The user might then search for known or postulated inhibitors of any member of the pathway.","The information stored in the database may be accessed or queried by users interested in identifying candidate genes. According to a specific embodiment, the present invention provides an interface allowing users to specify a query including criteria characterizing candidate genes. In response to the user query, the present invention searches the database to identify genes which satisfy the user-specified search criteria. A typical search might examine the group of classified genes (e.g. by function, role or pathway) appearing in an early or middle expression cluster (based on \u201cCluster\u201d -and \u201cClusterOrder\u201d -I). By comparing the similar attributes (e.g. a query of the type \u201cwhat apoptotic regulator genes are present in early clusters along chemokine genes?\u201d) within upstream or downstream clusters, the user may be able to deduce, for example, that the apoptotic pathway in a particular infection model of immune cells was altered by either (a) the appearance of a new apoptotic regulator gene or chemokine at an unexpected time or cluster, or (b) the absence of altered expression for a gene known to be induced in the pathway. Alternatively, the user might query what low-likelihood roles or pathways might explain the presence of a given class of receptors. In response to the user query, the present invention uses the user-specified query criteria to search the information stored in the database and outputs genes which satisfy the user-specified search criteria by either their presence or omission from either known or low-likelihood roles (or pathways) or lists of genes with known function(s) or role(s). In this manner, the information stored for the plurality of DNA sequences and their behavior in expression profile data facilitates identification of candidate genes.","A particular use for the expert database described above is to allow users to quickly access information regarding biological relationships of genes that have been clustered together by their pattern of expression. This information may allow a user to be more confident that a particular gene is involved in a particular process, which will facilitate the selection of that gene as a target for future therapies or treatments. Because of the size of the public literature databases, in particular the MEDLINE database, it may take a researcher several hours, or in many cases, days to laboriously research the possible interactions between genes of a group of genes to identify which of them can be promoted as a candidate. The methods of this invention are computer-implemented and requires very little of the investigator's time and energy e.g. to input a list of genes or sequence identifiers, and allow the researcher to view this data in a summarized, organized, and cross-referenced format. While the computer is analyzing data according to the user's request, the user can focus on other matters. Users can alter the confidence scoring of the interactions or pathway models, which influences the analysis of interactions and pathway models in future analyses.","A feature of the present invention is that it provides support for both intra-and inter- time-resolved gene cluster components; i.e. between or amongst genes in subsequent or previous groups of genes. Thus, a human expert can choose from a palette of options to refine a first iteration of gene network or pathway building. The parameters in turn can be used to recalculate the likelihood of other annotations and pathways to explain the behavior of a single gene, group of genes, or cluster of genes. Collectively, these methods can reduce the number of differentially regulated genes to a smaller group; from which candidate genes can be chosen by the human expert.","Although specific embodiments of the invention have been described, various modifications, alterations, alternative constructions, and equivalents are also encompassed within the scope of this application. The described invention is not restricted to operation within certain specific data processing environments, but is free to operate within a plurality of data processing environments. For example, although the present invention has been described in a distributed computer network environment, the present invention may also be incorporated in a single stand-alone computer system. In such an environment, the same stand-alone computer has access to the various biological databases according to the present invention and may act both as a client and a server. Additionally, although the present invention has been described using a particular series of transactions and steps, it should be apparent to those skilled in the art that the scope of the present invention is not limited to the described series of transactions and steps.","Further, while the present invention has been described using a particular combination of hardware and software, it should be recognized that other combinations of hardware and software are also within the scope of the present invention. The present invention may be implemented only in hardware or only in software or using combinations thereof.","The specification and drawings are, accordingly, to be regarded in an illustrative rather than a restrictive sense. It will, however, be evident that additions, subtractions, deletions, and other modifications and changes may be made thereunto without departing from the broader spirit and scope of the invention as set forth in the claims."],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":["Preferred embodiments according to the invention will be described with reference to the Figures, in which like reference numerals denote like elements and in which:",{"@attributes":{"id":"P-00018","num":"00018"},"figref":"FIG. 1"},{"@attributes":{"id":"P-00019","num":"00019"},"figref":"FIG. 2"},{"@attributes":{"id":"P-00020","num":"00020"},"figref":"FIG. 3"},{"@attributes":{"id":"P-00021","num":"00021"},"figref":"FIG. 4"},{"@attributes":{"id":"P-00022","num":"00022"},"figref":"FIG. 5"},{"@attributes":{"id":"P-00023","num":"00023"},"figref":"FIG. 6"},{"@attributes":{"id":"P-00024","num":"00024"},"figref":"FIG. 7"},{"@attributes":{"id":"P-00025","num":"00025"},"figref":"FIG. 8"},{"@attributes":{"id":"P-00026","num":"00026"},"figref":"FIG. 9"},{"@attributes":{"id":"P-00027","num":"00027"},"figref":"FIG. 10"},{"@attributes":{"id":"P-00028","num":"00028"},"figref":"FIG. 11"},{"@attributes":{"id":"P-00029","num":"00029"},"figref":"FIG. 12"},{"@attributes":{"id":"P-00030","num":"00030"},"figref":"FIG. 13"}]},"DETDESC":[{},{}]}
