---
title: Methods of selection, reporting and analysis of genetic markers using broad-based genetic profiling applications
abstract: Disclosed are a method for determining whether an individual has an enhanced, diminished, or average probability of exhibiting one or more phenotypic attributes and related methods of selecting a set of genetic markers; for providing relevant genetic information to an individual; of evaluating the probability that progeny of two individuals of the opposite sex will exhibit one or more phenotypic attributes; and for determining the genomic ethnicity of an individual.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08417459&OS=08417459&RS=08417459
owner: Omicia, Inc.
number: 08417459
owner_city: Emeryville
owner_country: US
publication_date: 20040409
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS REFERENCE TO RELATED APPLICATIONS","BACKGROUND OF INVENTION","SUMMARY OF INVENTION","DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENTS","EXAMPLES","Example 1","Use of a Genetic Marker as an Index to Literature","Example 2","Association Representation","Example 3","Characterizing Patterns of Associations Found in an Individual's Genetic Profile","Example 4","Constructing Multivariate Matrix","Example 5","Ontologies for Marker Selection","Example 6","Sample of Organizational Matrix for Communicating the Genetic Information to the Individual"],"p":["This application is a National Stage filing under 35 U.S.C. \u00a7371 of International Application No. PCT\/US04\/10905, filed Apr. 9, 2004, and claims priority under 35 U.S.C. \u00a7119(e) of U.S. Provisional application Ser. No. 60\/461,740, filed Apr. 9, 2003; and U.S. Provisional application Ser. No. 60\/492,707, filed Aug. 4, 2003, each of which is hereby incorporated by reference in its entirety.","Personalized human health care products and services that enable individuals to more actively manage their health based on their genetic profiles have been increasingly heralded following the publication of a draft human genome sequence in June 2000 (Venter, J C, Funct Integr Genomics. 2000 November; 1(3):154-5) and a substantially complete sequence of the human genome in February 2001. (Venter, J C et al., Science 291(5507):1304-51 [2001]; Lander E S et al., Nature 409(6822):860-921 [2001]). To date, however, the commercial availability of personalized genetic profile products and services has been extremely limited and costly.","The \u201cgenome\u201d of an individual member of a species comprises that individual's complete set of genes. Particular locations within the genome of a species are referred to as \u201cloci\u201d or \u201csites\u201d. \u201cAlleles\u201d are varying forms of the genomic DNA located at a given site. In the case of a site where there are two distinct alleles in a species, referred to as \u201cA\u201d and \u201cB\u201d, each individual member of the species can have one of four possible combinations: AA; AB; BA; and BB. The first allele of each pair is inherited from one parent, and the second from the other.","The \u201cgenotype\u201d of an individual at a specific site in the individual's genome refers to the specific combination of alleles that the individual has inherited. A \u201cgenetic profile\u201d for an individual includes information about the individual's genotype at a collection of sites in the individual's genome. As such, a genetic profile is comprised of a set of data points, where each data point is the genotype of the individual at a particular site.","Genotype combinations with identical alleles (e.g., AA and BB) at a given site are referred to as \u201chomozygous\u201d; genotype combinations with different alleles (e.g., AB and BA) at that site are referred to as \u201cheterozygous.\u201d It has to be noted that in determining the allele in a genome using standard techniques AB and BA cannot be differentiated, meaning it is impossible to determine from which parent a certain allele was inherited, given solely the genomic information of the individual tested. Moreover, variant AB parents can pass either variant A or variant B to their children. While such parents may not have a predisposition to develop a disease, their children may. For example, two variant AB parents can have children who are variant AA, variant AB, or variant BB. For example, one of the two homozygotic combinations in this set of three variant combinations may be associated with a disease. Having advance knowledge of this possibility allows potential parents to make the best possible decisions about their children's health.","Diseases are often associated with the collection of atoms, molecules, macromolecules, cells, tissues, organs, structures, fluids, metabolic, respiratory, pulmonary, neurological, reproductive or other physiological function, reflexes, behaviors and other physical characteristics observable in the individual through various means. The \u201cphenotype\u201d of an individual refers to one or more of these observable physical characteristics. An individual's phenotype is driven in large part by constituent proteins in the individual's proteome, the collection of all proteins produced by the cells comprising the individual and coded for in the individual's genome.","In many cases, a given phenotype can be associated with a specific genotype. For example, an individual with a certain pair of alleles for the gene that encodes for a particular lipoprotein associated with lipid transport may exhibit a phenotype characterized by a susceptibility to a hyperlipidemous disorder that leads to heart disease.","While efforts have been undertaken to create new personalized active health management products and services based on genetic profiles, several shortcomings characterize the existing art. Among these shortcomings are the following:\n\n","The objective of personalized genetic profile health management products and services is to provide individuals with information about their predisposition to diseases. Armed with this information, individuals can, in many instances, make decisions about their dietary practices, pharmaceutical use, exercise, and other lifestyle habits that are designed to better manage their predisposition to diseases.","From individual to individual within any species, genes are characterized by a very high degree of conservation in the sequence of nucleotide base pairs comprising them. At certain locations in many sites, however, the specific nucleotides that comprise a gene can undergo alteration, or mutation. Mutations can be inherited from a parent or acquired during a person's life. A hereditary mutation will be present in all of a person's cells and will be passed on to future generations, because the person's reproductive cells (sperm or egg) will contain the mutation. An acquired mutation can arise in the DNA of individual cells as a result of many possible factors. For example, mutations in the DNA of skin cells can be caused by exposure to the sun's UV radiation Genetic mutations in other cells can arise from errors that occur just prior to cell division, during which a cell makes a copy of its DNA before dividing into two. Genetic profile products and services tend to focus on hereditary mutations.","The situation in which two or more sequence variants of an allele exist at a site across different members of a population is called a \u201cpolymorphism,\u201d typically defined as having an occurrence frequency of larger than 1% within that population. Several different types of polymorphisms are known in the art. By far the most common form of polymorphisms are those involving single nucleotide variations between individuals of the same species; such polymorphisms are called \u201csingle nucleotide polymorphisms\u201d, or \u201cSNPs\u201d. To date, at least 1.42 million SNPs have been identified in the human genome. (Sachidanandam R et al., Nature 409(6822):928-33 [2001]). While it is believed that the great preponderance of these SNPs are harmless, there is a substantial number that have been associated with various diseases.","SNPs that occur in the protein coding regions of genes that give rise to the expression of variant or defective proteins are potentially the cause of a genetic-based disease. Even SNPs that occur in non-coding regions can result in altered mRNA and\/or protein expression. Examples are SNPs that defective splicing at exon\/intron junctions. Exons are the regions in genes that contain three-nucleotide codons that are ultimately translated into the amino acids that form proteins. Introns are regions in genes that can be transcribed into pre-messenger RNA but do not code for amino acids. In the process by which genomic DNA is transcribed into messenger RNA, introns are often spliced out of pre-messenger RNA transcripts to yield messenger RNA.","For example, in the \u201chealthy\u201d form of the protein hemoglobin, the amino acid at the sixth position in the protein's beta chain is glutamic acid. This amino acid is encoded in the hemoglobin gene by the DNA codon guanine-adenine-guanine (GAG). In some individuals, however, the adenine nucleotide in this codon is replaced with the thymine nucleotide, resulting in a GTG codon which codes for the amino acid valine. This substitution of valine for glutamic acid alters the normal shape of the hemoglobin protein. Red blood cells that contain these abnormally shaped hemoglobin proteins exhibit a sickle shape and are unable to perform the oxygen-transport function normally associated with red blood cells. Individuals who are GTG homozygous (i.e., have inherited a GTG variant from each parent) suffer from sickle cell anemia.","In addition to sickle cell anemia, SNPs have been associated with diseases such as cystic fibrosis, Huntington's chorea, beta-thalassemia, muscular dystrophy, fibro muscular displasia, pheny ketonuria, Type II diabetes, a hyperlipidemous disorder associated with Apolipoprotein E2, at least one form of hypertension, and some forms of migraine headaches. These disease-associated SNPs are inherited through classic Mendelian mechanisms. This type of SNP, however, is not believed to be the predominant form of SNPs associated with the most common diseases. This view is supported by the theory that common mutations in the protein coding regions would entirely dysfunction protein structures and therefore completely shutdown a specific pathway or parts of such pathways, a result which is not supported by observation. Nevertheless, it is believed that functional variants associated with phenotypes further associated with diseases should be clustered around non-coding sites that play an important role in the functioning of the genome.","An example of such functional, non-coding sites are the \u201csplice sites\u201d at which pre-messenger RNA transcripts are spliced into messenger RNA (mRNA). The need for splicing arises from the fact that within the pre-messenger RNA transcripts are RNA base pairs that correspond to introns in the genomic DNA from which the pre-messenger RNA transcript derives. The complex of proteins and RNA at which splicing occurs is called the \u201cspliceosome\u201d. (See, e.g., Fairbrother et al. 2002).","A few different methods are commonly used to analyze DNA for polymorphisms and genotype. The most definitive method is to sequence the DNA to determine the actual base sequence (see, A. M. Maxam and W. Gilbert, Proc. Natl. Acad. Sci. USA 74:560 (1977); Sanger et al., Proc. Natl. Acad. Sci. USA 74:5463 (1977)). Patent application 20020082869, \u201cMethod and system for providing and updating customized health care information based on an individual's genome\u201d, Anderson, Glen J., describes a system for delivering personalized genetic profiling information based on sequencing. Although such a method is the most definitive it is also the most expensive and time-consuming method. Accordingly, the sequencing of the human genome has only been performed for research purposes such as the Human Genome Project on samples from a very small number of individual humans, and at a cost of millions of dollars per individual. While the cost of sequencing the genome of an individual human has been following a steeply declining price\/performance curve, where performance is measured in terms of accuracy and time, the substantial cost that still stands today prohibits its use on a broad commercial scale. Until the cost of sequencing technologies declines substantially further, the delivery of genetic profiles to a significantly large number of individuals cannot be cost effectively based on genome sequencing. Moreover, as described below, simply being able to sequence an individual's genome is not sufficient to generate and provide a comprehensive genetic profile product or service to the individual.","Another method of analyzing DNA for polymorphisms and genotype is restriction mapping analysis. With this method genomic DNA is digested with a restriction enzyme and the resulting fragments are analyzed on an electrophoresis gel or with a Southern blot to determine the presence or absence of a polymorphism that changes the recognition site for the restriction enzyme. This method can also be used to determine the presence or absence of gross insertions or deletions in genomic DNA by observing the lengths of the resulting DNA fragments. In this respect, restriction mapping analysis has limited use in the type of genome-wide search for polymorphisms and genotyping analysis required for providing genetic profile products and services of the type contemplated by the present invention.","Another method of determining the genotype of an individual at a given site is to detect the presence of one or more nucleotide sequences at that site known to be associated with a predisposition, disease or other phenotypic abnormality. These sites, also called \u201cgenetic markers,\u201d can be detected using various tagged oligonucleotide hybridization technologies that are significantly less costly than genomic sequencing and allele-specific hybridization. Means now exist for constructing and performing large-scale, multiplexed genetic marker hybridization tests on biological samples from individuals, such as samples of blood, saliva and urine. These means, such as very dense chip and bead arrays, can enable a sample from an individual to be tested simultaneously for the presence of thousands of genetic markers. (Oliphant A et al., Biotechniques Suppl:56-8, 60-1 [2002]; and Fodor S P, Science 251(4995):767-73 [1991]).","Splice junctions in pre-messenger RNA, 5-prime (exon to intron transition) and 3-prime (intron to exon transitions), are the sequence regions that are used as recognition sites for the spliceosome and contain certain sequence motifs which typically are conserved between related species. Nucleotide changes in these binding sites can have a substantial effect on the spliced mRNA product, depending on the effect of the mutation on the overall binding affinity of the spliceosome components with the mRNA sequence. Known mis-splicing behavior arises from exon skipping, alternative splicing, protein coding truncation through the introduction of a frame shift, and the disruption of the entire mRNA production process. These changes have significant effects in the mRNA and protein processing step and can totally change their production. In addition, smaller changes can partially regulate and influence quantitatively the splicing behavior of certain genes. Additional sites known to be involved and sometimes even known to regulate splicing, are the branch-point, enhancer and silencer sequences (Fairbrother et al. 2002). Splice sites constitute locations in the genome for evolutionary pressure to function through nucleotide mutations.","Similarly, promoter regions in genes constitute locations in the genome where the presence of a SNP can be used for determining an individual's genotype. As gene-expression regulatory mechanisms, promoter regions include the transcription start site and various transcription factor-binding sites, including all the regions that are involved in gene regulation.","The determination of the presence of polymorphisms or, less frequently, mutations, in DNA has become a very important tool for a variety of purposes. Detecting mutations that are known to cause or to predispose persons to disease is one of the more important uses of determining the possible presence of a mutation. One example is the analysis of the gene named BRCA1 that may result in breast cancer if it is mutated (see, Mild et al., Science, 266:66-71, 1994). Several known mutations in the BRCA1 gene have been causally linked with breast cancer. It is now possible to screen women for these known mutations to determine whether they are predisposed to develop breast cancer. Some other uses for determining polymorphisms or mutations are for genotyping and for mutational analysis for positional cloning experiments.","In some cases, as illustrated in the case of the hemoglobin SNP and sickle cell anemia, the association of a SNP with a disease is direct and well-established and can be simply diagnosed. In many other cases, however, the association of a SNP with a phenotype that gives rise to disease or other adverse medical condition is not well-established and different diseases, disorders have different associations with different genotypes and SNPs. In these cases, the association between genotype and phenotype can vary from individual to individual in a complex manner that depends on the individual's genome, age, family history, life style habits, and other personal health and demographic factors. Consequently, direct testing of the individual's DNA is not accurate 100% of the time in predicting the onset of a genetically-based disease or other adverse medical condition. In these more complex cases, there is a probabilistic relationship between genotype (as characterized by different variants and SNPs) and phenotype (as characterized by the association of phenotypes with different diseases). In these cases, the presence of a SNP at a given genetic site is not sufficient by itself for the development of a pathological condition In addition, not all persons possessing a given SNP in a given variant will develop a disease associated with that SNP. The onset of a genetically-based disease may also depend on exposure to certain conditions in a person's environment. Moreover, the same disease, disorder, or other adverse medical condition associated with a given SNP in a given variant may result from a different SNP at another site. Consequently, comprehensive analysis of the relationship between an individual's genotype and phenotype requires a scoring matrix of variables along various dimensions and a method of using this matrix to determine the probability that a given genotype in a given individual win result in a given phenotype.","To further illustrate the complexity of associating genotypes with phenotypes, it is currently believed that the human genome comprises approximately 30,000 genes while the human proteome comprises potentially millions of proteins. The process by which the information contained in the DNA comprising 30,000 distinct genes is transcribed into messenger RNA, which is in turn translated into the sequence of amino acids comprising potentially millions of distinct proteins, therefore adds significant complexity to associations of genotypes with phenotypes. In addition, in the search for unknown disease-causing variants, whole-genome association scans using hundreds of thousands of genetic markers simultaneously are likely to face serious theoretical-statistical challenges, as well as practical difficulties associated with the management of data sets of enormous size and complexity. One obvious problem is the fact that, the more genetic markers are used, the higher the expected number of apparent, spurious associations that are the result of statistical chance as opposed to true association stemming from shared genealogy between genetic marker and causative allele.","Beyond this complexity of associating genotypes with phenotypes, there has been rapid growth of data on the existence of SNPs, their locations in the human genome, and associations of SNPs with phenotypes that are further associated with various diseases. This data arises from research on genomics, proteomics, preclinical and clinical studies of pharmaceuticals and related research gathered from laboratories, hospitals and medical clinics around the world.","Genomics has the potential to change the way medicine is practiced and impact the health of individuals. (E.g., Guttmacher A E, Collins F S, , N Engl J Med 347: 1512-20. and Collins [2002]; Varmus, -, N Engl J Med 347: 1526-7 [2002]). Through sequencing and genotyping, extensive personal genetic information is expected to continue to be generated in large quantities in coming years. (Trager R S, 1000 , Science 298: 947 [2002]). The rapid growth in volume of available genomic and proteomic data has been characterized by substantial disorder and information obsolescence. For example, currently, the Gene Ontology (GO) consortium (Ashburner, M et al., . The Gene Ontology Consortium. Nat Genet 25: 25-9 [2000]) and the National Library of Medicine's MESH (Schulman J-L, 2001 , NLM Tech Bull. 317 [2001]) are two of the best-known ontologies in the bioinformatics domain Neither of these ontologies, however, currently contains the necessary information to support research about the relationships between genes and disease in the context of the human genome. While GO is well suited to classify a gene product in terms of its function, process and location, it has no terms to describe human diseases and the relations between them, whereas MESH, while rich in descriptions and classifications of human disease, contains no information about sequences, little information about genes, and no information about disease causing mutations and SNPs. This is an unfortunate situation, especially in light of the recent completion of the human genome sequence, and its annotation.","A key aspect of research in genetics is the association of sequence variation with disease genes and phenotypes. Sequence variation data are currently available, for example, from OMIM, HGMD (Hamosh A, et al., Online Mendelian Inheritance in Man (OMIM), a Knowledge base of human genes and genetic disorders, Nucleic Acids Res 30: 52-5 [2002]; Krawczak, M et al., , Hum Mutat 15: 45-51 [2000]; McKusick V A, (), McKusick-Nathans Institute for Genetic Medicine, Johns Hopkins [2000]) and others, both of which provide phenotypic information and describe amino acid variation. Unfortunately, in most cases these variation references do not provide sufficient information to support their direct mapping onto current genomic sequences and the associated annotated genes. Single nucleotide polymorphism (SNP) data are held in dbSNP and other publicly accessible databases. (E.g., Sachidanandam, R et al., 1.42 , Nature 409: 928-33 [2001]; Sherry, S T et al., , Nucleic Acids Res 29: 308-11.2001). While these databases contain millions of entries each including the position of the SNIP on the genome, they do not provide significant phenotypic information about the SNPs at the levels which need to be reached, namely from the genome to the phenotype and the clinic.","Moreover, as the volume of genomic and proteomic data grows there are requirements to synthesize vast amounts of information to enable clearer understanding of an individual's genetic profile. Patent application 20020052761 (\u201cMethod and system for genetic screening data collection, analysis, report generation and access\u201d, Fey, Christopher T.; et al.) describes a system for generating highly complex personal health reports to individuals concerning their genetic test results, based on an aggregate set of genetic markers and phenotypes.","Over the years most genomic and clinical advances have been published in scientific journals. Molecular biology advances and finding have been published predominantly in molecular biology journals (e.g., Cell, Nat Genetics, Am J. Hum. Genetics, etc.), and clinical phenotype related findings have been published predominantly in medical journals (e.g., N Eng J Medicine, Lancet, etc.). Because of these different journals are directed to different communities, large communication gaps have been created. Thus, there now exists in the public domain two distinct information resources, and neither is as valuable as it potentially could be because current research efforts require their integration. One part includes all large public genomic databases and the other is the vast amount of clinical research data, mostly held in publication, but increasingly accessible electronically. There is a clear tendency in the community for a computer-based classification of disease through ontologies and relating medical diagnostic classification schemes such as the ICD-9 with gene diseases (e.g., the NuGene project; Chisholm, R et al., 2003]).","There are currently various standardization efforts occurring within molecular biology, most notably the gene ontology (GO) consortium efforts. (Ashburner, M et al., . Nat Genet 25: 25-9 [2000]). Additional ontologies such as the sequence ontology, the mutation ontology and others are in work in process (for an overview of ontology development please see the \u201cglobal open biological ontologies\u201d (GOBO) web site (www.geneontology.org\/doc\/gobo.html).","Databases containing information on polymorphisms are also expected to have an important impact in the field of pharmacogenomics. Pharmacogenomics is an area of research focused on how variations in a patient's DNA can cause pharmaceuticals to respond differently. The importance of understanding these variations is underscored by the number of hospitalizations and deaths that occur each year that are caused by adverse drug reactions. One method of characterizing the genetic basis of drug response is by cataloging variations in drug response as a function of SNPs. The more SNPs cataloged, the more robust and effective the database. However, collecting and sorting the SNPs becomes a huge undertaking. In U.S. Patent application 20020049772, Reinhoff, et al, provides a broad overview of polymorphisms, pharmacogenomics, and classifying populations based upon sets of polymorphisms.","In addition to the scientific, technological and medical complexities that characterize the development and commercialization of genetic profile products and services, there are growing legal and regulatory complexities. For example, patient privacy has been a growing concern in multiple jurisdictions. In Europe, the European Union Directive 95\/46\/EC is designed to protect individuals with regard to the processing and movement of their personal data. In the United States, under the Health Insurance Portability and Accountability Act of 1996, commonly referred to as \u201cHIPAA\u201d, regulations have been adopted that set forth \u201cStandards for Privacy of Individually Identifiable Health Information\u201d. The purpose of these regulations is to help guarantee privacy and confidentiality of patient medical records. These Standard are quite extensive and apply to health care providers, insurers, payors and employers.","The confluence of all of the factors discussed above leads to the conclusion that what has been lacking from the art, but necessary for viable broad-based commercial provision of personalized health management products and services based on genetic profiling, is a method that satisfies the following requisites:\n\n","Various means exist for practicing each of these separately. Each such means, however, suffers from various deficiencies, and a method of collectively optimizing their combined practice is required in order to provide health care management products and services on a broad commercial scale at prices that are economically attractive to both provider and customer. The present invention provides these and other benefits.","The present invention relates to a method for determining an individual's probability, whether enhanced, diminished, or average probability, of exhibiting one or more phenotypic attributes through evaluating genomic markers from that individual for zygosity for the members of a preselected set of genetic markers. In accordance with the method, the markers are compared to a multivariate scoring matrix to obtain a marker score, from which it is determined whether an enhanced, diminished, or average probability of exhibiting one or more phenotypic attributes is indicated. The multivariate scoring matrix correlates patterns of marker zygosity with probabilities of exhibiting phenotypic attributes.","The present invention is further directed to a method of selecting a set of genetic markers. The method involves filtering markers for inclusion in the set, based on a determination of measures of phenotypic value and\/or prioritization, such as but not limited to, penetrance of the marker in a population or subpopulation of interest; the degree of linkage of the marker to a particular phenotype; the relative contribution of the marker to communicating the phenotype; and the degree of statistical or scientific confidence to be placed in any data associated with any of the measures of phenotypic value or priority used.","The present invention further relates to a method for providing relevant genetic information to an individual, or about an individual to another interested party (e.g., a physician, veterinarian, researcher, breeder, or owner). The method involves identifying genotypic characteristics of the individual that correlate with a relative probability of exhibiting one or more phenotypic characteristics. The method also involves determining for each of the one or more phenotypic characteristics whether the individual has an enhanced, diminished, or average probability, of exhibiting the characteristic by evaluating genomic markers for zygosity (for example, but not limited to heterozygosity or homozygosity) at each member of a preselected set of markers and comparing the zygosity of the markers to a multivariate matrix that correlates patterns of marker zygosity with probabilities of exhibiting phenotypic attributes, determining whether the marker score resulting from this comparison indicates an enhanced, diminished, or average probability of exhibiting the one or more phenotypic attributes. Then one or more selection criteria is applied for each of the one or more phenotypic characteristics. Each selection criterion applied imposes a total, a partial, or no limitation on the information communicated to the individual. Subsequently, information is identified that is relevant to the individual's probabilities of exhibiting the one or more phenotypic characteristics and is consistent with the limitations imposed by the selection criteria, and the information is communicated to the individual.","The present invention is further directed to a method of evaluating the genetic profile combination (i.e., \u201ccompatibility\u201d) of two individuals of the opposite sex (i.e., male and female) of the same species, whether human or a non-human species, in other words, a method of evaluating the probability that progeny of two individuals of the opposite sex will exhibit one or more phenotypic attributes. The method involves evaluating genomic markers from each of the two individuals for zygosity (for example, but not limited to marker heterozygosity or homozygosity) at each member of a preselected set of markers, determining a probability distribution for the zygosity for each member of the preselected set of markers in the genomes of the progeny of the two individuals, and comparing the probability distributions to a multivariate matrix to obtain a probability distribution score, wherein the matrix correlates patterns of marker zygosity with probabilities of exhibiting phenotypic attributes, and determining whether the probability distribution score indicates that the progeny of the two individuals have an enhanced, diminished, or average probability of exhibiting one or more phenotypic attributes.","The present invention further provides a method for determining the genomic ethnicity of an individual comprising evaluating genomic markers from an individual at each member of a preselected set of genetic markers, comparing the genotype for each of the markers to a multivariate matrix, wherein the matrix correlates patterns of genotypes with probabilities of examining phenotypic attributes, and determining the genomic ethnicity of individual as a pattern of the probabilities of exhibiting the phenotypic attributes.","The inventive methods can be used to provide individualized health management products and services based on an individual's genetic profile as well as for pharmacogenomic studies in human or other populations or subpopulations of varying sizes and compositions (e.g., members of defined ethnicity, gender, age, or disease or other physiologically abnormal condition, etc.). They can also be applied to the analysis and reporting of genetic profiles of various human populations and subpopulations, as well as populations and subpopulations of other non-human species.","Thus, the present invention provides the benefits of economical and effective genetic profiling that involves (i) genotyping each individual who seeks such products and services for a broad set of genetic variants, (ii) scoring these variants for their association with the individual's phenotypic susceptibility to the later onset of various diseases, (iii) providing information on such genotype and phenotype information in a manner that can be understood by the individual and the individual's physicians and medical\/health care counselors so that appropriate health management steps can be taken prior to onset, (iv) tracking on-going advances in research tailored specifically for individuals based on the their genotypes and other personal information, and (v) providing information on these on-going advances to the individuals and their physicians and medical\/health care counselors. The practice of the present invention is applicable to the provision of genetic profile-based health management products and services characterized by logistical efficiencies and economies of scale as well accuracy of analysis, interpretation and reports, and the effectiveness of such tracking of related scientific research and medical advances, and which can be offered in accordance with applicable health care, privacy and related laws and regulations. Moreover, the practice of the present invention is a valuable tool for pharmacogenomics.","These and other advantages and features of the present invention will be described more fully in a detailed description of the preferred embodiments which follows.","The present invention relates to a method for determining whether an individual has an enhanced, diminished, or average probability of exhibiting one or more phenotypic attributes. In accordance with the method, genomic markers from an individual are evaluated for their zygosity at each member of a preselected set of markers.","For purposes of the present invention, an \u201cindividual\u201d can be of any species of interest, including a human or a non-human species.","For purposes of the present invention, an enhanced, diminished, or average probability of exhibiting one or more phenotypic attributes, or a \u201crelative probability,\u201d is with respect to the general population in a particular geographical area or areas, or with respect to a defined subpopulation thereof, for example, but not limited to, a particular gender, age grouping, or ethnicity, or some other identifying feature.","\u201cZygosity\u201d of a marker is its genotype with respect to any combination of heterozygosity or homozygosity for one or more possible alleles at a locus, including, but not limited to, a condition of heterozygosity or homozygosity for a particular allele or alleles between chromosomes, or allele heterozygosity among several possible alleles at a locus on one chromosome.","In accordance with the present invention, the preselection of the set of markers is based on genotype\/phenotype associations with disease conditions or predispositions for disease conditions. The association of genotypes with phenotypes and further associating such phenotypes with diseases and predisipositions includes utilization of existing data in the relevant literature and can also include data acquired from additional studies incorporating the practice of the present invention  shows an exemplary set of 2236 genes that can be included (in its entirety or in any combination of subsets) in a preselected set of marker in accordance with the invention. The skilled practitioner is aware of other markers that can be included in the preselected set of markers.","In one embodiment, preselecting the set of markers can be achieved using haplotype mapping to capture the majority of historic recombination events that have occurred since the most recent common ancestor of the sample group or population analyzed. An example of such a haplotype map is by the National Institute for Genome Research, which comprises a set of a few hundred thousand polymorphic markers covering the entire human genome with sufficient density along each chromosome to measure the diversity of common haplotypes prevailing in each local region. Nevertheless, haplotypes also can be defined as a block or region of DNA that has been inherited as a unit. SNPs within that block of DNA will also have been inherited together. These SNPs can then be used to identify the presence of a particular block. This block can have any size from a few base pairs to large hundreds of kilobases of nucleotides. (Stephens J C, Mol Diagn. 4(4):309-17 [1999]). In one embodiment of the present invention these markers are used to find common alleles or variants associated with complex diseases and are used to interpret local variation patterns and provide information on the evolutionary origin of known polymorphisms that are the cause of functional differences between alleles or variants. The markers are prioritized in order to construct or \u201cpreselect\u201d a reduced marker set to avoid the data management and statistical problems that attend whole genome scans using hundreds of thousands of genetic markers simultaneously. In a preferred embodiment of the present invention, certain genotypes are associated with certain phenotypes.","In another embodiment, the predicted and\/or measured effect of different genetic markers on the DNA\/RNA sequence changes that occur in the splicing process at exon\/intron junctions is used to classify and preselect the set of genetic markers. Such classification can be accomplished by scoring each genetic marker candidate through the application of one or more predictive models applied to the data in a multivariate scoring matrix for that candidate. In this embodiment, all splice sites relevant to a variant are first identified by mRNA identification and alignments to the genomic DNA either through existing annotations or new annotations, thereby enabling both information and chemical retrieval of 5\u2032 and 3\u2032 splice site sequence regions. Second, variants with SNPs within these splice site regions and the associated sites are identified by the integrating and mapping of public and private mutations within these splicing associated sites. Third, the wild-type or original sequences and the mutated sequences are scored with one or more predictive models selected from the universe of: statistical models, such as weight matrix, Bayesian, hidden Markov models, semi- or general hidden Markov models; artificial intelligence models; and discriminative and predictive models related to binding affinity. This process results in splicing strength scores for the original variant and the variant with the SNP. The difference of these scores of any other metric applied to these scores can then be used to preselect markers for genotyping analysis and to score, rank, and prioritize factors in the construction of a multivariate scoring matrix.","In one embodiment of the present invention, the preselected set of genetic markers comprises a plurality of exon\/intron junction sequences. In another embodiment of the present invention at least about 20% of the genetic markers in the preselected set of genetic markers are exon\/intron junction sequences. In another embodiment of the present invention at least about 40% of the genetic markers in the preselected set of genetic markers are exon\/intron junction sequences. In another embodiment of the present invention at least about 60% of the genetic markers in the preselected set of genetic markers are exon\/intron junction sequences. In another embodiment of the present invention at least about 80% of the genetic markers in the preselected set of genetic markers are exon\/intron junction sequences.","In addition to applying predictive models to such splice sites for purposes of preselecting genetic markers and for constructing genotyping assays, such predictive models can also be applied for such purposes to large-scale promoter regions of genomic DNA that can contain one or more SNPs, functional binding sites within the 5\u2032 or 3\u2032untranslated regions (\u201cUTRs\u201d), RNAi-genes, and mRNAs. In a preferred embodiment of the present invention, a method is provided of subjecting genetic markers within promoter regions to the same analytic selection process as specified above for splice sites. In one preferred embodiment of the present invention, the preselected set of genetic markers comprises a plurality of promoter sequences. In another preferred embodiment of the present invention, at least about 20% of the markers in the preselected set are promoter sequences. In another preferred embodiment of the present invention, at least about 40% of the markers in the preselected set are promoter sequences. In another preferred embodiment of the present invention, at least about 60% of the markers in the preselected set are promoter sequences. In another preferred embodiment of the present invention, at least about 80% of the markers in the preselected set are promoter sequences.","In another embodiment of the present invention, genetic markers of interest are identified and selected for inclusion in a preselected set, based upon a matrix with dimensions comprising genotypic and phenotypic criteria. These criteria can include, among others: base pair sequence homology to another known genetic marker sequence of interest; the presence of two or more regions of DNA on the same chromosome or genetic marker (synteny); relevance to the description of the molecular function, biological process and cellular component of the protein coded by the gene under investigation (ontology) and ontological classification; conservation of mutated sequence sites at conserved or less conserved sequence homology sites in the genome; quality of research on the genotype, genetic marker and phenotype under investigation; biological significance of the genetic marker (for example, whether the marker specifies a protein coding change); and regulatory value and classifications of the amino acid(s) specified by the genetic marker.","\u201cLexica\u201d, \u201cthesauri\u201d, and \u201contologies\u201d are used to semantically classify and define genomics data. A \u201clexicon\u201d is a list of terms belonging to the same semantic class: BMP4 and DPP, for example, both belong to the semantic class of \u201cBMP\u201d (i.e., bone morphogenetic factor). A \u201cthesaurus\u201d provides a listing of the synonyms for a term, or semantic class, and hierarchical \u201contologies\u201d are used to \u201cdefine\u201d the terms contained in a lexicon and a thesaurus. Long a cornerstone of computer science, ontologies have recently become a major focus of research in bioinformatics. Like controlled vocabularies, ontologies also enable data sharing but, because they contain hierarchical relationships between their terms, ontologies enable logical inference and deduction (see ) on the data they contain, making them powerful tools for hypothesis generation. The definition of a term is produced by tracing the path from a term to the root of the ontology (, panel c, path starting from \u201cBMP\u201d). The simple ontology shown in panel c of , for example, defines \u201cBMP\u201d as \u201ca TGF-\u03b2 growth factor\u201d. Definitions apply to all members of a semantic class and their synonyms, and can be used as a basis for logical inference: e.g., \u201cDPP is a DVR, a DVR is a BMP, a BMP is a TGF-\u03b2, and a TGF-\u03b2 is a growth factor\u201d; therefore it can be inferred that \u201cDPP is a growth factor\u201d even if no document explicitly states this fact. Note that the ontology shown in panel c of  is a particular type known as an \u2018isa-hierarchy\u2019; other types of ontologies exist, not all of which are suitable for definition.","In one embodiment the selection of markers is based on microsatellite markers. Microsatellite include simple sequence repeats, short tandem repeats and simple sequence length polymorphisms that are characterized as relatively short tandem repeat nucleotide sequences, e.g. (TG)n or (AAT)n, generally less than 5 base pairs in length. Microsatellite markers have adequate physical densities for family-based (linkage) studies where the number of recombination events between closely related family members is small, hence markers even as distant as mega bases apart are likely to be co-inherited. In addition, microsatellite markers are generally useful in genetic studies because of their hypervariability, co-dominance and reproducibility of microsatellite markers.","The present invention is also directed to a method of selecting a set of genetic markers which can be employed, for example, in preselecting the set of genetic markers in accordance with the method for determining whether an individual has an enhanced, diminished, or average probability of exhibiting one or more phenotypic attributes. Selection of a set of genetic markers according to the present invention results in large set of genetic markers that map to splice site regulation loci and promoter loci for gene clusters and gene families associated with specific phenotypes as well as complete genome-wide splice site and promoter site genetic marker profiles.","Selecting the set of genetic markers is optimized by a meaningful marker prioritization which requires the incorporation of additional facets of information to characterize the local genome context. One useful approach is to employ existing lists of candidate genes thought to be involved in a given disease. By computational mining of the disease literature, identifying sets of genes already implicated in related clinical phenotypes (or homologous genes involved in related phenotypes in model organisms), and by locating disease loci and disease-causing mutations within the common frame of reference of the genome sequence one can hope to extend these lists. Although extended, such lists still drastically limit the number of loci one needs to scan for disease association, while they are, hopefully, still inclusive enough to reduce the risk of omission of true causative loci. Within the targeted regions, typically a gene locus, one has the choice of either including all known polymorphic markers, or trying to make further reductions.","Some other useful considerations include marker location relative to the functional units of the gene (coding, UTR, splice site, regulatory, intron, etc.), marker zygosity (related to population frequency), or ease of assay development (local repeat structure, sequence composition for oligo design). Typically, focusing on extended candidate lists reduces the DNA search space from the entire genome to about one thousand to a few thousand loci, or roughly 5% of the genome. Additional pruning can introduce an additional 5-fold reduction. The result is a scenario where, at the cost of genotyping 1% of all the markers available for a genome-wide scan, one types all candidate loci, with the reasonable expectation that at least half of the causative loci are investigated. In addition to the reduction in genotyping cost, an advantageous reduction of false positive associations results, which false positive associations have been known to plague even very large clinical association studies and have been cited as an important cause of irreproducible reports of disease association.","In a preferred embodiment of the present invention, the preselected set of markers comprises genetic markers that map to at least about 1,000 discrete loci. In another preferred embodiment of the present invention, the preselected set of markers comprises genetic markers that map to at least about 2,000 discrete loci. In another preferred embodiment of the present invention, the preselected set of markers comprises genetic markers that map to at least about 3,000 to about 5,000 discrete loci. In another preferred embodiment of the present invention, the preselected set of markers comprises genetic markers that map to at least about 5,000 to about 10,000 discrete loci. In another preferred embodiment of the present invention, the preselected set of markers comprises genetic markers that map to at least about 10,000 to about 15,000 discrete loci. In another preferred embodiment of the present invention, the preselected set of markers comprises genetic markers that map to least about 15,000 to about 30,000 discrete loci.","In accordance with the invention and the method for determining whether an individual has an enhanced, diminished, or average probability of exhibiting one or more phenotypic attributes genomic markers corresponding to at least the preselected set of genetic markers from an individual are evaluated for zygosity. For purposes of the present invention, zygosity for a marker is detected in a biological sample collected from the individual that contains the individual's genomic DNA (such as, but not limited to, a blood, saliva, or tissue biopsy sample, which biological sample can be freshly collected or suitably stored to preserve the DNA) by employing suitable biochemical genotyping analytical assay means. Analytical hybridization or polynucleotide sequencing means are typically employed, optionally after amplification of DNA in the biological sample, for example, by using PCR-based amplification means. High throughput analyses can optionally be achieved by multiplexing techniques known in the art. The genotyping analytical assay means can optionally be performed with commonly available robotic apparati and\/or very dense array detection apparati.","The determined zygosity of the markers for the individual is compared to the multivariate scoring matrix to obtain a marker score, wherein the multivariate scoring matrix correlates patterns of marker zygosity with probabilities of exhibiting phenotypic attributes. In accordance with the invention, the multivariate scoring matrix correlates patterns of marker zygosity with probabilities of exhibiting phenotypic attributes, based on scoring matrix vectors that can include descriptors of: family history, general medical physiological measures or values (such as, but not limited to, cholesterol levels, triglyceride levels, blood pressure, heart rate, HGH or other hormone levels, red blood cells, bone density, CD scan results, etc.), mRNA expression profiles, methylation profiles, protein expression profiles, enzyme activity, antibody load, and the like. The comparison with the multivariate scoring matrix can be done manually or, preferably, by employing a suitable computer software instantiation in which the multivariate scoring matrix is algorithmically constructed and manipulated via a programming language, for example, but not limited to, Java, Perl, or c++.","Based on this comparison it is determined whether the marker score indicates an enhanced, diminished, or average probability of exhibiting one or more phenotypic attributes, relative to a reference population, e.g., the general population of a chosen geographical area, or another chosen subpopulation thereof in terms of ethnicity, gender, age, or other identifying feature of interest.","The present invention further provides a means for providing relevant genetic information to an individual whose genetic profile has been determined in accordance with the present invention.","The method for providing relevant genetic information to an individual, and for generating information reports to be communicated, includes identifying the individual's genotypic characteristics by correlating those genotypic characteristics with a relative probability of exhibiting one or more phenotypic characteristics and determining for each of the one or more phenotypic characteristics whether the individual has an enhanced, diminished, or average probability of exhibiting the phenotypic characteristic, in accordance with the present invention as described hereinabove.","The method for providing relevant genetic information to an individual then includes applying one or more selection criteria for each such phenotypic characteristic, wherein each selection criterion imposes total, partial, or no limitation on the information communicated to the individual, identifying information that is relevant to the individual's probabilities of exhibiting such phenotypic characteristics and consistent with the limitations imposed by the selection criteria, and communicating the information to the individual in a report as described above.","In one embodiment of the present invention, the same or different selection criteria are applied one or more additional times to the determined probabilities of exhibiting each of the phenotypic characteristics, information is identified that is relevant to the individual's probabilities of exhibiting the one or more phenotypic characteristics and consistent with the limitations imposed by the selection criteria, and the information is communicated to the individual.","In one embodiment of the present invention, at least one of the selection criteria is specified in advance by the individual.","In another embodiment of the present invention, at least one of the selection criteria is a function of the availability of treatments effective to modify the phenotypic characteristic.","In other embodiments, at least one of the selection criteria is a function of the scope and quality of known research relating to the phenotypic characteristic; or at least one of the selection criteria is a function of the probability determination(s) for one or more other phenotypic characteristics.","The information is then communicated to the individual, whether directly, or indirectly via an appropriate intermediary, in a report that generally includes an explanation of relevant terminology; a summary of the genetic profile of such individual subject; an explanation of each genotype assay performed on biological samples from the individual, typically as categorized by types of disease (such as, but not limited to, cancer, cardiovascular disease, and the like), and in relation, where applicable, to specific body organs, tissues and metabolic, reproductive or other bodily functions and systems; a summary and detailed results for each such genotype assay performed; a health risk appraisal for such individual subject; general information about genetics and genomics; and references for further information. Such report can include summary sections of genes and gene families important to individuals based on their specific phenotypic impact and the demographic, ethnic, gender, age, and related characterizations of each individual. For example, the categories for such characterization can include, but are not limited to, aging, women's health, and drug interactions. These summaries can include (i) an assessment of the overall genetic health of the individual in each health category presented and the overall genetic health of the individual and (ii) cross references to the more detailed information within the report.","The information report can be communicated orally, but is preferably communicated in a documentary format, whether written or electronic, and in any suitable order of report presentation. Prior to communicating the information to the individual, the information is preferably, but not necessarily, formatted to present the relevant phenotypic attributes according to an organizational matrix, wherein the organizational matrix determines the grouping and presentation of information to the individual. More preferably the organizational matrix groups the various phenotypic characteristics for which the individual has an enhanced probability together. In some embodiments the organizational matrix groups phenotypic characteristics related to similar physiological systems together. In other embodiments, the organizational matrix ranks the phenotypic characteristics as a function of the potential impact on the individual's lifestyle or quality of life, or the organizational matrix ranks the phenotypic characteristics as a function of the \u201cgenomic ethnicity\u201d of the individual, as described herein.","In a preferred embodiment of the present invention, the information report relates to a broad selection of diseases. Such diseases may include, among others, cancer and those relating to the following organs, tissues, and metabolic, reproductive and other bodily functions and systems involved in human health, including, but not limited to, cardiovascular, respiratory, kidney and urinary tract; immune system, gastrointestinal, neurological, psychoneurological, and hematological functions and systems.","In a preferred embodiment of the present invention, the report for each such disease, disorder and other adverse medical condition comprises information about the relevant genes, the sites assayed in such gene, the clinical association of variants at such sites with relevant disease, the genotype of the individual subject at each such site, information about the association of such genotype with the phenotype associated with such disease, and information about drug interactions for the individual subject based on such genotype.","Prior to communicating the information to the individual, the identity of the individual need not be associated with the data corresponding to the genotypic characteristics, the relative probabilities of exhibiting the phenotypic characteristics, or the identified relevant information A suitable coding or \u201cblind\u201d system can be employed to shield the identity of the individual, if appropriate.","The present invention provides that the format and substance of the information report can be modified from time to time on an ongoing basis, particularly in response to questions from individuals, physicians and medical counselors about genetic markers, genotypes, genotype assays, the association of genotype with phenotype and diseases, and other aspects of such reports. In a preferred embodiment of the resent invention, such questions are stored in a database and the overall effectiveness of communication of the text and graphics presented in such reports are from time to time assessed. In a preferred embodiment of the present information, the report generator is constructed so that changes to the information report format and substance can be effected efficiently and economically.","The present invention also relates to a method of evaluating the genetic profile combination (i.e., \u201ccompatibility\u201d) of two individuals of the opposite sex of the same species, in other words, a method of evaluating the probability that progeny of two individuals of the opposite sex will exhibit one or more phenotypic attributes. The method is applicable to providing genetic counseling to pairs of individuals of the opposite sex relating to preselected risks engendered by their respective genotypes, or genetic profiles, to their progeny, whether prospectively or after the birth of the progeny. Alternatively, the method is applicable to providing genetic counseling to a recipient parent (or recipient parents) of progeny resulting from, or intended to result from, the use of donor gametes, e.g., donor sperm and\/or ova.","The method involves evaluating genomic markers from each of the two individuals whose gametes are to contribute, or contributed, to the genetic inheritance of the progeny by the formation of a zygote, i.e., the genetic or so-called \u201cbiological\u201d parents. Zygosity at each member of a preselected set of markers is evaluated, and a determination is made of a probability distribution for the zygosity for each member of the preselected set of markers in the genomes of the progeny of the two individuals. The probability distributions are compared to a multivariate matrix to obtain a probability distribution score. The multivariate matrix correlates patterns of marker zygosity with probabilities of exhibiting phenotypic attributes, as described hereinabove. Then it is determined whether the probability distribution score indicates that the progeny of the two individuals would have an enhanced, diminished, or average probability of exhibiting one or more phenotypic attributes.","The present invention further provides a method for determining the genomic ethnicity of an individual for the purpose of determining the likely applicability of clinical research results based upon the genetic profile of the individual. Clinical research results are often only found in particular populations or subpopulations, and these results are frequently correlated with the ethnicity of the population or subpopulation. \u201cGenomic ethnicity\u201d means a genetic profile of an individual having a distribution of genetic markers in a preselected set of markers, based on genotype\/phenotype associations with disease conditions, that is preponderantly consistent with the distribution of those markers that is determined to be or is known in the art to be characteristic of a particular ethnic population or subpopulation. The determination of genomic ethnicity in accordance with the method can often provide more useful information than an individual's mere self-reporting of ethnicity.","The method involves evaluating genomic markers from an individual at each member of a preselected set of markers, comparing the genotype for each of the markers to a multivariate matrix, wherein the multivariate matrix correlates patterns of genotypes with probabilities of exhibiting particular phenotypic attributes, and determining the genomic ethnicity of the individual as a pattern of the probabilities of exhibiting the phenotypic attributes.","In should be borne in mind that the genotype-phenotype relation requires two axes, the phenotype (e.g., a particular disease) and the mutations. Most known disease-causing mutations are found in single-gene disorders because these have been the easiest to find; Most of the OMIM disease entries are in this class; hence most OMIM mutations cause single-gene disorders. These disorders are usually very rare in the general population (although can be much more prevalent in a given family or population group). For diseases in this group, allele state often predicts disease with an almost certainty. But there are more complex scenarios: (i) with the current state of knowledge, genotype-phenotype association can only be cast in statistical terms (i.e. having the \u201cbad\u201d allele does not necessarily mean the individual will ever develop the disease, but means a heightened chance or predisposition, relative to the general population or a defined subpopulation, of developing it in the future), or (ii) the association is only seen in some studies but not in others, or (iii) the association is only seen in a single family, in a population isolate, or one of the world populations, with no knowledge of relevance for other populations. Also typical in these cases is that the associated marker is almost certainly not itself causative, merely an associated marker for the disease. In accordance with the methods of the present invention, each of these scenarios are taken into account as appropriate in constructing the multivariate matrix, or multivariate scoring matrix, and as the available information improves concerning genotype\/haplotype relationships with a given phenotype, the multivariate matrix, or multivariate scoring matrix, can be updated, expanded, thereby enhancing its predictive value when employed within the inventive methods. However, the practice of the inventive methods is not limited by the precise contents of the multivariate matrix at any given time, which will vary as new information is incorporated.","The information about the genetic profiles of individuals for whom genetic profiles have been generated of the type produced in accordance with the present invention can facilitate delivery of appropriate therapies to the individuals, can enable the further identification of novel genotype-phenotype relationships to support the discovery of new therapies, and can provide novel indicators of health. Thus the present invention can be applied to pharmacogenomic analysis and the development of more effectively directed drug therapies.","In another application, the information generated in accordance with the present invention can be maintained in an updatable database. The database can be constructed using widely available database management tools, such as Oracle or open source database tools such as postgres. The information in the database can be used to select individuals for whom investigational therapies or newly commercially available therapies could probably be most effective, based on their genetic profiles. For example, such selection can be made positively on an indication that individuals with a disease or predisposition to disease can benefit from particular therapies, or negatively on an indication that such individuals are at risk with a particular therapy. Such classification schemes can be used to identify subsets of populations with complex disease for purposes of course of therapy decisions.","A genetic marker, and a variant found at that marker, can be characterized by a number of factors. These can include a unique sequence surrounding it, a fixed offset from a known reference sequence point, or a specific amino acid change in a particular protein. The clinical literature refers to markers, mutations, or polymorphisms in a variety of different ways. In accordance with the present invention, individual markers and variants can be characterized using these and other attributes in order to use them to index clinical literature. This index can be constructed of a single marker, or of a set of markers. By indexing literature in this way, a consolidated report can be constructed based upon these markers. If for example, there exists a genotyped, or sequenced database of markers that have been found to be present in an individual, this marker database can be used to create an extract of clinical literature that is particularly relevant to the individual. This use of an individual's markers or variants as a literature index, is a useful component in providing personalized communication of information from literature and other databases, based upon an individual's genetic profile.","As genetic disease associations become more complex, interpreting the implications of these associations becomes more complicated. The overall quality of the association, in a number of dimensions, becomes increasingly important. In addition, in order to understand the importance of a particular association to an individual, an association, in all of its texture, needs to be viewed in the overall context of the individual's genetic profile. Characterizing and representing this quality, both independent of the individual and in the context of the individual's genetic profile become fundamental to the appropriate usage of the information.","The factors that shape the implications of a genetic association are diverse. They range from the causal linkage seen with the association (e.g., the percentage of disease cases that are associated with a particular association and the percentage of individuals with a given marker who develop the disease), to factors related to the breadth of the research (e.g., number of studies, number of individuals studied, and number of ethnicities studied). By characterizing and representing the associations in this manner, individuals, their counselors, and their health care providers can better understand, appropriately value, and make use of them.","One method of representing the quality of an association, for the purposes of analyzing, evaluating, determining, identifying, comparing, or formatting the information and\/or of communicating or presenting the information to the individual, is to place associations on a two dimensional grid. The axes of this grid would be the percentage of disease cases that are associated with a particular association and the percentage of individuals with a given marker who develop the disease. Each association would be represented, e.g., by a colored circle, on this grid; the color can represent the number of studies of this association, with for example, red representing a low number, yellow representing an intermediate number and green representing a large number of studies. The darkness of the circle can represent the number of ethnicities studied, and the size of the circle can represent the number of individuals. In this way, at a glance, a viewer can understand how a particular association compares to others, and the overall quality of the association itself. For example, a light, red, circle close to the origin is a very weak association. A dark, green circle in the top right of the graph is very significant, well-studied in large populations.","The same graph can be constructed of associations for those markers that have been found in an individual's genetic profile. With this, an individual or his or her counselor or health care provider is immediately focused on those associations that are most significant.","Of course, this multi-dimensional representation can take many graphical forms. An alternative method might use a radar graph. A radar graph is a two dimensional polar graph that enables one to simultaneously display many variables. It does this by plotting each variable along a different radial axis emanating from the origin of the polar plot. If one has ten variables, then there will be ten radial axes thirty six degrees apart Small values are near the center of the polar plot and large values near the outer circumference. An experiment might result in multiple measured variables and the investigator wants to compare the results of this experiment repeated under different conditions. If one connects the variable values on each radial axis with a straight line then a distorted star-like pattern (or \u201cpolygon\u201d) is created. One can then visually compare the patterns created by the lines representing the different conditions for the experiment. In addition, several polygons can be plotted on the same set of radial arms through the use of one color or line pattern per polygon.","The preceding are illustrative, and not exhaustive examples, of how graphical representation of a number of factors that characterize a genetic association can support an enhanced understanding of the association, both individually and in the context of other associations, as well as independently and in the context of an individual's genetic profile.","Genetic profiles, defined as a set of markers found in an individual's genome ranging from a small set of markers to the entire genetic sequence, are used to generate or predict attributes of individuals that can be used for a variety of applications. These applications can range from identity verification to determining disease predisposition and drug interactions to feature prediction (e.g., height, hair color or eye color). In addition, the attributes can range from being absolutely correlated to the genetic profile, to being shaped by the genetic profile, to being unpredictable from the genetic profile. Understanding and characterizing the degree to which a genetic profile specifies an attribute has great utility. For example, the generation of a profile from DNA of a suspected criminal is much more useful when the degree to which the feature can be predicted is well-understood. If red hair is almost assured, but height much less specified by the profile, searches can be better refined. Capturing and representing this information becomes much more important as the number of attributes to be analyzed grows. The methods that can be used to extract and capture these patterns include statistical models, Bayesian networks, hidden Markov models, and other machine learning techniques.","A goal of the present invention is to bridge the time gap that currently exists between the latest research advances and the final benefit to the consumers in the health care system. Through genetic testing and the information delivery this gap can be bridged and shortened by a huge factor. For example, individuals can be tested for a newly identified functional polymorphism (e.g., the RANTES gene; Hizawa, N. et al., -, Am J Respir Crit Care Med 166: 686-90 [2002]; see Table 1 below), information about which can be added to the multivariate matrix, or multivariate scoring matrix in accordance with the present invention. Suitable genotyping assays can be developed for the newly identified polymorphisms described in the original article in a matter of weeks. After the genetic testing with the genotyping assay, the result can be communicated to the tested individual about her or his genetic profile and the relevant clinical phenotype information like specific risk factors\u2014in the RANTES gene example its relationship with asthma\u2014that are correlated with a specific genetic profile. The complexity of the clinical information for the RANTES gene is not limited to this article but there exist many more articles describing the relationship between genetic markers in the RANTES gene and a clinical phenotype. One useful database to capture more comprehensively genetic literature information concerning a disease can be found in the \u201cOnline Mendelian Inheritance of Man\u201d (OMIM) database (Hamosh, A et al., (OMIM), , Nucleic Acids Res 30: 52-5 [2002]; McKusick V A, (). McKusick-Nathans Institute for Genetic Medicine, Johns Hopkins [2000]). The OMIM database contains textual information and references on inherited diseases and genetic disorders. It also contains copious links to MEDLINE and sequence records in the Entrez system, and links to additional related resources at NCBI and elsewhere. This database is accessible on the internet at the ncbi website with the extention nlm.nih.gov.\/entrez\/query.fcgi?db=OMIM, as well as in a hard copy book. The electronic version is also distributed in XML format under an NIH license from the National Library of Medicine. While the OMIM database is very useful for clinical and academic research it is still far too technical for an individual and her\/his healthcare provider (see as an example the OMIM entry for the RANTES gene in Table 2 below).","A prototype system was developed that allows scientists to position alleles described in the Online Mendelian Inheritance in Man (OMIM) database on the build 33 assembly of the Human Genome from the National Center for Biomedical Information (NCBI). Once placed on the assembly, the system automatically classifies the alleles within the context of a gene and its genomic structure; i.e., coding (synonymous\/non-synonymous), UTR, etc. The OMIM database is currently the most comprehensive and relevant database for human genetic disease, and we have therefore focused our efforts on mapping the OMIM alleles to the genome. Toward this end we have developed a \u2018deterministic\u2019 algorithm for OMIM allele mapping. Most of the alleles described within OMIM are clinically relevant, and have links to the appropriate scientific publications; in an attempt to better manage the clinical data associated with these alleles, we have also developed a concise and high-level disease ontology.","All five subsystems of the prototype system have been tested with a set of known, well-characterized disease genes. Specific care and focus have been given to the user-friendliness and user interaction of the annotation station, an extendable and flexible design for the database infrastructure and a practical approach for the OMIM allele mutation mapping algorithm, to capture as many OMIM mutations within the genome as possible, by avoiding false positives. During the genetic disease ontology development, we largely made use of existing medical textbook classifications, due to the widespread familiarity of these classifications within the medical community. The following sections detail the software and the science developed.","(a) Prototype Visualization Annotation Station","We built a prototype SNP and mutation oriented genome annotation station, in which we leveraged existing open source genome browser software as developed in the wormbase project (Stein, L. et al., . Nucleic Acids Res 29: 82-6 [2001]). For additional data manipulation work we have used PERL code from the open source BioPerl project (Stajich, J E et al, , Genome Res 12: 1611-8 [2002]). The annotation station is centered on genes, and with a gene symbol the genomic region for the specified gene can be viewed.","Our \u201cGeneViewer\u201d can display different types of genomic sequence variations within a graphical user interface. The display now shows graphically a genomic gene region with 5 kb sequence upstream and downstream of the start and stop codon of a gene. It also displays neighboring genes that are located within this +\/\u22125 kb window. The gene structure is displayed using the \u201ctraditional\u201d exon-intron viewer, in which exons are linked with arcs that span the introns. Currently the display color-codes and groups missense, nonsense, coding and non-coding mutations onto the gene structure. The variation symbols list the nucleotide and\/or protein amino acid change in a small text glyph. A variation map screen shot of this GeneViewer can be seen in , which shows the MTRR gene.","Following the GeneViewer is a short \u201cGene Summary\u201d, which tabulates all the general information about a gene, such as NCBI's LocusLink identifier(s), genomic Genbank records, RefSeq links, full textual gene names as well as internal disease classifications from our Ontology (see below) that are assigned by the annotators.","Following the Gene Summary is the \u201cVariation Summary\u201d, which seeks to list in table format all known genetic markers within this gene from different mutation databases. This table currently lists OMIM alleles and dbSNP entries including there sequence position within the genomic sequence as well as the protein sequence; Furthermore, codon position, codon changes, and a short functional characterization (missense, non-sense, exonic, intronic, 5\u2032UTR, 3\u2032UTR, upstream or downstream) are included. A snapshot for the MTRR gene can be seen in .","In the next two sections, the \u201cmRNA Summary\u201d and the \u201cCDS Summary\u201d, raw mRNA and coding sequences (CDS) including the Genbank annotations. On demand, the genomic sequence can also be displayed.","The \u201cProtein Summary\u201d follows, which besides the translated coding region lists under the wild type sequence the protein mutations caused by the sequence variation lists in the Variation Summary table. This is very useful, because for each occurring nucleotide change the resulting protein mutation can be seen.","At the end of each gene page, the corresponding OMIM entry is listed from a local OMIM copy from our database to avoid the reloading of every OMIM entry from the NCBI site through the internet (\u201cOMIM Summary\u201d). This allows the annotators to review the verbal gene annotation within the same display.","The data required for effective and accurate annotation are diverse. To address this diversity, the visual annotation station is split into these different views. The prototype also allows to add other information sources and even to generate new summary views if necessary. Improvements can be added, including various hyperlinks with the interface, that can link to external (NCBI: dbSNP, RefSeq, Genbank) as well as internal sources (OMIM, filtered version of PUBMED; see text mining algorithms below). For example, this can be addressed by downloading the OMIM database from the NIH and incorporating the OMIM locus entries within the genome annotation station. This allows to cross-link mutations within a gene document and get fast access to the allele variation description in OMIM on the same page.","(b) Prototype Annotation Pipeline Software","We have built a selection of PERL scripts to download the latest human genome assembly as well as annotations from Genbank and RefSeq. The current prototype system has loaded the build 33 assembly, but is easily updated. Furthermore, dbSNP and the OMIM databases have been licensed and downloaded from the NCBI ftp site.","To capture and identify all known human genes predisposing to disease, we have begun with a widely accepted, recently published paper tabulating and summarizing all known human disease genes (Jimenez-Sanchez, G et al., , Nature 409: 853-5 [2001]). This set consisted of the non-somatic genetic associations contained in the OMIM database. This set of 923 human disease genes was expanded with a collection of genes derived from an in-house analysis of hundreds of current clinical research papers, selecting those associations that were most significant and \u201cactionable\u201d by the clients of our service. Any hint of an association between a novel disease gene and a clinical phenotype within an abstract triggered the collection of the paper from the library or online databases. The original papers were then reviewed and if significant, the gene was manually added to our collection of disease genes. Additional papers that justify the selection of a certain gene into our disease gene set were noted as additional evidence. A broad disease classification scheme (see \u201chigh-level disease ontology section\u201d) was developed during this process as well. The total number of genes in our system is currently 1,732. Given that the current estimate of the total number of human genes is between 25,000 and 30,000, this would mean that between 5.5% and 7.0% of human genes can be classified to be disease related. Of course, this is only a snapshot of the current clinical research and we expect the percentage to increase over time. 98% of these genes already had OMIM entries.","Of the 1,732 hand-selected genes, only 1,614 could be uniquely located in the latest genome assembly (see Table 3). The remaining 118 genes were either annotated at multiple loci within the human genome or were missing from both RefSeq and Genbank. The location of these 118 genes within the human genome sequence is next to be manually determined, as many belong to small paralogous families that have significant clinical implications.","During the loading process of the flat file OMIM entries, we have parsed out the OMIM allele identifiers. It has been very difficult to develop this PERL parser due to the incredible mixture of formats for allele annotations. The allele identifiers have then been used to locate the protein mutations within the human genome assembly and the Genbank annotations. As can be seen in Table 3, 57.29% of the OMIM alleles could be localized directly within the genome assembly through the Genbank annotations using only the protein variation allele identifiers. This procedure of comparing OMIM alleles with the Genbank annotations is a quality check for the Genbank annotations. The main reason why not all of the OMIM alleles could be easily mapped to the genome has primarily to do with the different protein sequences, in which the first mutations have been described Case studies for some of these genes have shown that the underlying protein sequence has usually been extended on the 5\u2032end of the coding sequence over time. In previous publications only the 3\u2032 part of a gene sequence has been known\u2014typically because of the early 3\u2032 EST sequencing efforts\u2014and old, incorrect amino acids variation annotations have been propagated through the literature until today. In an attempt to circumvent this problem we have developed a \u2018deterministic\u2019 algorithm for OMIM allele mutation mapping; see below.","The integration of dbSNP polymorphisms has been straight forward, because they are linked to a specific gene sequence within Genbank using the dbSNP identifiers (\u201crs\u201d and \u201css\u201d numbers). Surprisingly only a very small number, 1.87%, of OMIM alleles have a dbSNP entry (see Table 3). We speculate that there are two reasons for this. First, most of the alleles in OMIM are Mendelian-mutations and have a very low minor-allele frequency. Secondly, SNPs have only recently been discovered during the human genome project and the SNP consortium efforts and clinical studies showing clinical correlations are still under way and they therefore have no allele entry within OMIM.","A preliminary study encouraged by the surprising low overlap between OMIM alleles and dbSNP entries performed using HGMD showed that OMIM alleles are represented in HGMD. We could find approximately 40% of the OMIM alleles from our 1,732 disease genes annotated in HGMD. This is a most promising finding and allows us to conclude that a lot of information can be gained by using the annotations from HGMD to link OMIM with the human genome assembly.","(c) Prototype Database Infrastructure","A prototype database infrastructure was built. Existing public domain database schemas were adapted to fulfill specific variation related needs. Furthermore, software was developed to load and update the specific genome and clinical databases. All annotation pipeline software as well as the visualization annotation station are driven by this database system. The detail of the prototype database system is as follows.","The database uses a loosely-coupled modular architecture (). Within each module, tables are tightly-coupled by means of relational foreign keys. Between modules, tables are implicitly linked by means of dbxrefs (database accessions) and gene symbols.","This modular architecture facilitates better software engineering and database management. Modules can be swapped in and out easily. Loading large data bulks or data management can be carried out on any module independent of the others. For example, if NCBI releases a new human genome assembly build, or a new dbSNP build, we can bulkload this without perturbing our marker data.","We used the Postgresq1 version 7.3 relational database management system (RDBMS). Postgresq1 has the advantage of being open source, as well as extremely robust, with a large community of users and developers who are eager to provide support for free. It is much simpler to manage and administer than commercial RDBMSs such as Oracle, and has many advantages over the other open source RDBMS like MySQL. We believe MySQL is not suited to a production environment because of its lack of full SQL92 support, such as views, sub-selects and foreign key integrity.","We have taken care not to incorporate any Postgresq1 specific functionality into the design, which means that a port to a commercial RDBMS such as Oracle should in the future be a relatively uncomplicated task.","(d) BioSQL Module","BioSQL (obda.open-bio.org) is a third party open source database schema and a PERL API (Application Programmer Interface). It is a generic database of sequences and sequence features. It is ideal for representing sequence data such as that from GenBank, the EMBL sequence database, SwissProt and RefSeq. We used it to store the NCBI Human Genome and NCBI human RefSeq. Data, loaded from NCBI flat file format using a PERL script that comes with the BioSQL distribution.","The BioSQL data model is based around the concept of \u2018bioentries\u2019 and \u2018seq_features\u2019. This is equivalent to a Genbank\/RefSeq record and feature table entry, respectively. One of these records typically contains features of type gene, mRNA, CDS and variation. The intersection between these locations tells us where the markers such as SNPs are with respect to exons, introns, untranslated regions (UTRs) and up\/downstream regions. This data model is also suitable for housing the locations of the HGMD and OMIM variations mapped onto genomic coordinates. The most relevant part of the BioSQL schema can be seen in .","The snp table () stores single base pair and other variation features. Any none marker can have multiple effects, at the genomic, transcriptional and protein levels\u2014this is what the snp_function table is for. It allows us to easily query for markers that affect a particular genomic area (exon, intron, UTR, protein, intergenic) or for protein-affecting markers what is the amino acid modification. Each of these effects is with respect to a particular gene; we have a weak (a non-foreign key that is not enforced) link to the gene table (in the DiseaseGene module, see below) via the official gene symbol.","(e) DiseaseGene Module","The DiseaseGene module () is primarily for storing literature based curation of genes implicated in diseases, and associated information such as likely markers. Note that the gene_to_snp table is primarily for storing information on the literature curator's notes on possible markers, and is only weakly linked (implied via locational correspondence) with the snp table in the so-named OMICIASNP module. The gene table uses the official gene symbol; alternate symbols are stored in the gene synonym table. Each gene can have multiple disease categories, associated diseases or OMIM IDs.","Phenotype Module","The characteristics of the Web based graphical user interface (GUI) that support the manual mutation curation and map ping of phenotype information to the current genome assembly, are of a client\/server design, with a graphical representation of a gene sequence with the associated mutation data through a Web browser, delivered through an integrated database. Using a web browser, the user interface can be accessed internally and secured externally for annotation and reviewing. From a user's perspective, the diverse analytic tools and algorithms, described below, appear as a single unified application. Users are able to edit the data and add annotations to the gene sequence throughout the process.","(g) OMIM-R","The OMIM-R module () is a normalized relational model for housing data imported from OMIM. This module is focused around the OMIM disease table, which has an OMIM identifier as a primary key; this also serves as a weak key for integration with other modules in the OMICIADB schema. The cryptically named columns are named after multi-valued OMIM fields of the same name. Each disease can have multiple phenotypes attached. Each disease can also have multiple known mutations, and each of these can have multiple amino acid modifications.","(h) DbSNP-xml Module","This is a fairly simple module, as we opted to store each dbSNP entry as a single denormalised XML \u2018blob\u2019, accessed by a dbSNP identifier. This module is a single table, with a column for the dbSNP identifier, and the XML stored as a text field.","(i) Dataflow and Software Architecture for Database Loading","OMIM records are parsed into XML and then loaded into the OMIM-R module; any mutational information from the OMIM allele entries is mapped from protein coordinate space onto genomic coordinates from Genbank, and the resulting features are stored in the BioSQL module. The NCBI human genome build and RefSeq database are loaded into the BioSQL module. dbSNP is loaded into the dbSNP-xml module. Any disease gene information from the literature curation process is parsed into XML and then stored in the DiseaseGene module.","After any of these steps, which can be carried out independently, all genes in the DiseaseGene module are iterated through. For every gene we look for the corresponding gene annotation in BioSQL and find nearby marker features. We instantiate marker entries for all of these markers within a gene in the so-named OMICIASNP module. We also create a so-named OMICIAGeneSummary XML file, which integrates information from all modules into one XML document. The Annotation Station web interface code accesses this cached data for building the web displays. This is achieved through two Application Programming Interfaces (APIs)\u2014the open source BioSQL PERL API and a PERL API. A proprietary PERL API developed by Omicia (so-named \u201cOMICIA API\u201d) provides a unified access layer for all so-named OMICIADB modules and a simplified layer on top of the BioSQL API. However, the skilled artisan is able to construct his or her own API.","(j) Deterministic Algorithm for OMIM Allele Mutation Mapping","Many OMIM alleles could not be mapped directly, usually because some of the literature references were based on outdated versions of the reference sequence, or actually predated the availability of any reference sequence. When less than 50% of OMIM alleles for a gene could not be mapped, a simple codon position shift\/deterministic algorithm was applied to get a higher percentage of mapped OMIM alleles. The algorithm is as follows: For a given gene, all OMIM allele codon positions are ordered and stored in a hash. Every neighboring OMIM protein allele is compared to the original protein sequence from Genbank. If any of these pairs match with the Genbank annotation, all other OMIM protein alleles are \u201cshifted\u201d according to the offset into the Genbank annotation as well. By using this deterministic algorithm, an additional 12.11% of OMIM alleles were mapped (see Table 4).","In order to evaluate software system, 97 genes were selected for annotation. The annotation consisted of reviewing the clinical associations for each marker within a gene, assessing it for significance, and then updating the database with a clinical reference and rationale for selection. Biologists supported by computer scientists performed this annotation process. The process was assessed and modified throughout the effort as bottlenecks were identified. More expeditious approaches were developed on the basis of overcoming these obstacles. Table 4 shows a summary of the selected 97 genes. OMIM had annotated a total of 529 alleles for these sequences. This corresponds in average to 5.45 allele annotations per gene. This is significantly higher than for our total set of disease genes, which had in average 3.25 alleles per gene (see Table 3). We believe this bias is due to the selection process, by which we mostly used well-known and therefore well-studied genes, for which many clinical studies have been performed over the years. Typical genes in our set include APOE, CFTR, APC, MTTR and all genes starting with the letter \u201cR\u201d. Even with this result, about 30.60% of the OMIM alleles remain unmapped. More sophisticated algorithms can be developed to map a higher percentage of the remaining OMIM alleles.","We found that on average, the annotation of each gene took approximately 2 hours, but this time can be reduced through the use of text mining tools, and the development and refinement of a ranking scheme for the markers, as the integration of the clinical and pharmaceutical information, which is of great value to the individual client, is the most burdensome.","On the average 4.24 markers per gene were selected. The criteria for marker selection were based on the association significance of a marker with a disease. Markers were selected that were believed to be informative and useful for a typical individual. Based on this initial study, it is estimated that a set of approximately 7,344 (1,732 total disease genes\u00d74.24) informative disease markers is supported by the current clinical literature.","(k) A \u201cHigh-Level\u201d Genetic Disease Ontology","The genetic associations were placed into high-level categories, based upon a leading medical text, Harrison's Principles of Internal Medicine. The categories used were: Cancer; Cardiovascular; Endocrinology and metabolism; Gastrointestinal system; Hematology\/blood disorders; Kidney and urinary tract; Immune system; Neurological and psychiatric disorders; Respiratory; and Others. This effort made clear that a uniform terminology does not exist within the medical literature. Therefore, the associations are preferably categorized using the MESH disease ontology (see Example 5 below).",{"@attributes":{"id":"p-0135","num":"0144"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 1"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"A typical disease gene research abstract for the RANTES"},{"entry":"gene (published in American Journal of Respiratory"},{"entry":"and Critical Care Medicine citation)."},{"entry":"Original Article"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"A Functional Polymorphism in the RANTES Gene Promoter Is"},{"entry":"Associated with the Development of Late-Onset Asthma"},{"entry":"Nobuyuki Hizawa, Etsuro Yamaguchi, Satoshi Konno, Yoko Tanino,"},{"entry":"Eisei Jinushi and Masaharu Nishimura"},{"entry":"First Department of Medicine, Hokkaido University School of"},{"entry":"Medicine, Sapporo, Japan"},{"entry":"Correspondence: Correspondence and requests for reprints should"},{"entry":"be addressed to Nobuyuki Hizawa, First Department of Medicine,"},{"entry":"Hokkaido University School of Medicine, Kita-Ku, N-15 N-7,"},{"entry":"Sapporo, Japan 060-8638. E-mail: nhizawa@med.hokudai.ac.jp"},{"entry":"The CC chemokine regulated upon activation, normal T-cell"},{"entry":"expressed and secreted (RANTES) attracts eosinophils, basophils,"},{"entry":"and T cells during inflammation and immune response, indicating"},{"entry":"a possible role for this chemokine in asthma. Both the -403A"},{"entry":"and -28G alleles of the RANTES promoter region exhibit"},{"entry":"significantly enhanced promoter activity in reporter constructs"},{"entry":"in vitro. We therefore investigated the genetic influence of"},{"entry":"these alleles on the development of asthma using case-control"},{"entry":"analysis in a Japanese population (298 patients with asthma"},{"entry":"and 311 control subjects). Given the evidence for heterogeneity"},{"entry":"of asthma according to age at onset, we divided patients with"},{"entry":"asthma into three subgroups: 117 late-onset patients with asthma"},{"entry":"(onset at more than 40 years of age), 83 middle-onset patients"},{"entry":"with asthma (onset at 20 to 40 years of age), and 98 early-onset"},{"entry":"patients with asthma (onset at less than 20 years of age)."},{"entry":"The -28G allele was significantly associated with late-onset asthma"},{"entry":"(odds ratio = 2.033; 95% confidence interval, 1.379\u20132.998;"},{"entry":"corrected p < 0.0025) but was not associated with the other two"},{"entry":"asthma subgroups. The -403A allele was not associated with any of"},{"entry":"the asthma subgroups. Further evidence of the importance of"},{"entry":"the -28G allele was a significant increase in the production of"},{"entry":"RANTES in vitro in individuals who carried this allele. Our findings"},{"entry":"suggest that, among Japanese, the -28G allele of the RANTES"},{"entry":"promoter region confers susceptibility to late-onset asthma."},{"entry":"Key Words: late-onset asthma \u00b7 RANTES \u00b7 single nucleotide"},{"entry":"polymorphism (SNP)"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0136","num":"0145"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"287pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 2"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"OMIM entry 18701 for the RANTES gene."},{"entry":"*187011"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"287pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"CHEMOKINE, CC MOTIF, LIGAND 5; CCL5"},{"entry":"Alternative titles; symbols"},{"entry":"SMALL INDUCIBLE CYTOKINE A5, FORMERLY; SCYA5, FORMERLY"},{"entry":"REGULATED UPON ACTIVATION, NORMALLY T-EXPRESSED, AND PRESUMABLY"},{"entry":"SECRETED; RANTES"},{"entry":"T CELL-SPECIFIC RANTES"},{"entry":"T CELL-SPECIFIC PROTEIN p228; TCP228"},{"entry":"Gene map locus 17q11.2\u2013q12"},{"entry":"CLONING"},{"entry":"Using a human cDNA library that was enriched by subtractive hybridization for sequences"},{"entry":"expressed by T lymphocytes but not B lymphocytes, Schall et al. (1988) isolated a gene"},{"entry":"(D17S136E), which they designated RANTES, that encodes a novel T cell-specific molecule."},{"entry":"(RANTES is an acronym for \u2018Regulated upon Activation, Normally T-Expressed, and presumably"},{"entry":"Secreted.\u2019) The gene product was predicted to be a 10-kD protein which, after cleavage of the"},{"entry":"signal peptide, could be expected to be approximately 8 kD. Of the 68 residues, 4 are cysteines,"},{"entry":"and there are no sites for N-linked glycosylation. Significant homology (30 to 70%) was found"},{"entry":"between the RANTES sequence and several other T-cell genes, suggesting that they constitute"},{"entry":"a family of small, secreted T-cell molecules."},{"entry":"Schall et al. (1988) found that RANTES, also designated p228 (TCP228) was expressed in 10"},{"entry":"functional T-cell lines, but not in 8 hematopoietic tumor lines or in 6 T-cell tumor lines. Its"},{"entry":"expression was increased more than 10-fold in peripheral blood lymphocytes 3 to 5 days"},{"entry":"following mitogenic or antigenic stimulation."},{"entry":"GENE FUNCTION"},{"entry":"CD8-positive T lymphocytes are involved in the control of human immunodeficiency virus"},{"entry":"(HIV) infection in vivo. Cocchi et al. (1995) demonstrated that the chemokines RANTES, MIP-1-"},{"entry":"alpha (182283), and MIP-1-beta (182284) are the major HIV-suppressive factors produced by"},{"entry":"CD8-positive T cells. HIV-suppressive factor activity produced by either immortalized or"},{"entry":"primary CD8-positive T cells was completely blocked by a combination of neutralizing"},{"entry":"antibodies against these 3 cytokines. On the other hand, recombinant forms of the 3 human"},{"entry":"cytokines induced a dose-dependent inhibition of different strains of HIV-1, HIV-2, and simian"},{"entry":"immunodeficiency virus (SIV). Cocchi et al. (1995) speculated that chemokine-mediated control"},{"entry":"of HIV may occur either directly, through their inherent anti-lentiretroviral activity, or indirectly,"},{"entry":"through their ability to chemoattract T cells and monocytes to the proximity of the infection foci."},{"entry":"However, this latter mechanism may also have the opposite effect of providing new, uninfected"},{"entry":"targets for HIV infection. The authors noted that the findings may be relevant for the prevention"},{"entry":"and therapy of AIDS."},{"entry":"Arenzana-Seisdedos et al. (1996) investigated a derivative of RANTES as a possible therapeutic"},{"entry":"agent for inhibition of HIV infection. The derivative, called RANTES(9\u201368), lacks the first 8-N-"},{"entry":"terminal amino acids and has no chemotactic or leukocyte-activating properties. RANTES(9\u201368)"},{"entry":"was a potent receptor antagonist and inhibited infection of macrophage-tropic HIV. The anti-HIV"},{"entry":"activity was somewhat lower than that of RANTES itself, which correlated with its lower affinity"},{"entry":"for CC chemokine receptors. Arenzana-Seisdedos et al. (1996) found that the anti-HIV activity of"},{"entry":"RANTES and RANTES(9\u201368) showed some variability depending on the donor cells. The authors"},{"entry":"concluded that structural modification of a chemokine can yield variants lacking activation"},{"entry":"properties but retaining both high-affinity for chemokine receptors and the ability to block HIV"},{"entry":"infection."},{"entry":"Pritts et al. (2002) investigated the effect of PPAR-gamma ligands upon transcription and"},{"entry":"translation of RANTES in human endometrial stromal cells. Three putative PPAR response"},{"entry":"elements (PPREs) were found in the human RANTES promoter. In cells transfected both with"},{"entry":"RANTES promoter vectors containing 958 bp and 3 PPREs, the addition of 2 PPAR-gamma"},{"entry":"ligands inhibited promoter activity by 60% (P less than 0.01) and 48% (P less than 0.02),"},{"entry":"respectively. Truncation of the gene promoter to delete all putative PPREs abrogated the ligand-"},{"entry":"induced inhibition. Stromal cells showed a 40% decrease in RANTES protein secretion when"},{"entry":"treated with a PPAR-gamma ligand (P less than 0.01). The authors concluded that use of PPAR-"},{"entry":"gamma ligands to reduce chemokine production and inflammation may be a productive strategy"},{"entry":"for future therapy of endometrial disorders, such as endometriosis."},{"entry":"MAPPING"},{"entry":"By analysis of somatic cell hybrids and by in situ hybridization using the cDNA probe, Donion et"},{"entry":"al. (1990) assigned the RANTES locus to 17q11.2\u2013q12. A secondary hybridization peak was"},{"entry":"noted in the region 5q31\u2013q34, which may represent the location of other members of the gene"},{"entry":"family. The region on chromosome 5 overlaps with the location of an extended linked cluster of"},{"entry":"growth factor and receptor genes, some of which may be coregulated with members of the"},{"entry":"RANTES gene family."},{"entry":"MOLECULAR GENETICS"},{"entry":"RANTES is one of the natural ligands for the chemokine receptor CCR5 (CMKBR5; 601373) and"},{"entry":"potently suppresses in vitro replication of the R5 strains of HIV-1, which use CCR5 as a"},{"entry":"coreceptor. Previous studies showing that peripheral blood mononuclear cells or CD4+"},{"entry":"lymphocytes obtained from different individuals have wide variations in their ability to secrete"},{"entry":"RANTES prompted Liu et al. (1999) to analyze the upstream noncoding region of the RANTES"},{"entry":"gene, which contains cis-acting elements involved in RANTES promoter activity, in 272 HIV-1-"},{"entry":"infected and 193 non-HIV-1-infected individuals in Japan. They found 2 polymorphic positions,"},{"entry":"1 of which was associated with reduced CD4+ lymphocyte depletion rates during untreated"},{"entry":"periods in HIV-1-infected individuals. This -28G mutation of the RANTES gene (187011.0001)"},{"entry":"occurred at an allele frequency of approximately 17% in the non-HIV-1-infected Japanese"},{"entry":"population and exerted no influence on the incidence of HIV-1 infection. Functional analyses of"},{"entry":"RANTES promoter activity indicated that the -28G mutation increases transcription of the"},{"entry":"RANTES gene. Taken together, these data suggested that the -28G mutation increases RANTES"},{"entry":"expression in HIV-1-infected individuals and thus delays the progression of the HIV-1 disease."},{"entry":"ALLELIC VARIANTS"},{"entry":"(selected examples)"},{"entry":".0001 HUMAN IMMUNODEFICIENCY VIRUS TYPE 1, DELAYED DISEASE"},{"entry":"PROGRESSION WITH INFECTION BY [SCYA5, -28C-G]"},{"entry":"In a large Japanese cohort of HIV-1-infected and non-HIV-1-infected individuals, Liu et al."},{"entry":"(1999) identified a C-to-G transversion at position -28 in the promoter of the SCYA5 gene, also"},{"entry":"referred to as the RANTES gene. The -28G allele had a frequency of approximately 17% in the"},{"entry":"Japanese population and appeared to have no influence on the incidence of HIV-1 infection."},{"entry":"However, functional analyses indicated that the -28G mutation increased transcription of the"},{"entry":"RANTES gene. Liu et al. (1999) suggested that the -28G mutation increases RANTES expression"},{"entry":"in HIV-1-infected individuals and thus delays the progression of the HIV-1 disease. They"},{"entry":"showed that the -28G mutation is associated with reduced rates of depletion of CD4+"},{"entry":"lymphocytes in HIV-1-infected individuals, thus confirming that this polymorphism delays HIV-"},{"entry":"1 disease progression."},{"entry":".0002 HUMAN IMMUNODEFICIENCY VIRUS TYPE 1, RAPID DISEASE PROGRESSION"},{"entry":"WITH INFECTION BY [SCYA5, 168923, T\/C]. Among 7 SNPs within the RANTES gene"},{"entry":"investigated by An et al. (2002), one was the intronic RANTES regulatory element, In1.1T\/C"},{"entry":"(168923T\/C). They found that In1.1C-bearing genotypes accounted for 37% of the attributable"},{"entry":"risk for rapid progression to AIDS among African Americans. Because 36% of African"},{"entry":"Americans carry the In1.1C allele, it is likely that In1.1C may have a significant impact on the"},{"entry":"AIDS epidemic in sub-Saharan Africa."},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"REFERENCES"},{"entry":"1. An, P.; Nelson, G. W.; Wang, L.; Donfield, S.; Goedert, J. J.; Phair, J.; Vlahov, D.; Buchbinder, S.; Farrar, W. L.; Modi, W.; O'Brien, S. J.; Winkler, C. A.: Modulating influence on HIV\/AIDS by interacting RANTES gene variants. Proc. Nat. Acad. Sci. 99: 1002\u20131007, 2002. PubMed ID: 11792860"},{"entry":"2. Arenzana-Seisdedos, F.; Virelizier, J.-L.; Rousset, D.; Clark-Lewis, I.; Loetscher, P.; Moser, B.; Baggiolini, M.: HIV blocked by chemokine antagonist. (Letter) Nature 383: 400 only, 1996. PubMed ID: 8837769"},{"entry":"3. Bakhiet, M.; Tjernlund, A.; Mousa, A.; Gad, A.; Stromblad, S.; Kuziel, W. A.; Seiger, A.; Andersson, J.: RANTES promotes growth and survival of human first-trimester forebrain astrocytes. Nature Cell Biol. 3: 150\u2013157, 2001."}]}}]}}},{"@attributes":{"id":"p-0137","num":"0146"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Statistical summary on the initial OMIM mapping efforts."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"175pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Disease genes","Percentage"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"140pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Total \u201cdisease genes\u201d","1,732",{}]},{"entry":["Total \u201cdisease genes\u201d mapped to build 30","1,614","93.19%"]},{"entry":["Non-uniquely mapped \u201cdisease genes\u201d","118","6.81%"]},{"entry":["Total OMIM alleles:","5,624"]},{"entry":["OMIM alleles per gene:","3.25"]},{"entry":["OMIM alleles mapped to build30","3,222","57.29%"]},{"entry":["OMIM alleles mapped to build30 using","681","12.11%"]},{"entry":["deterministic algorithm",{}]},{"entry":["Total OMIM alleles mapped:","3,903","69.40%"]},{"entry":["OMIM alleles that are in dbSNP (incl.","105","1.87%"]},{"entry":"deterministic shift algorithm)"},{"entry":["OMIM alleles that are in HGMD (incl.","2,276","40.47%"]},{"entry":["deterministic shift algorithm)",{}]},{"entry":["Total OMIM alleles either in dbSNP or HGMD","2,381","42.34%"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0138","num":"0147"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 4"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Annotated test set of 97 disease genes."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Annotated Genes"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Total # of genes:","97","5.60%"]},{"entry":["Total OMIM alleles:","529"]},{"entry":["OMIM alleles per gene:","5.45"]},{"entry":["Total OMIM alleles either in dbSNP or HGMD","432","81.66%"]},{"entry":["Total markers picked for annotated genes","411"]},{"entry":["Markers picked per gene","4.24"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"In one embodiment of selecting one or more of the markers to be included in a preselected set of markers, in accordance with the present invention, an example of applicable techniques that can be applied follows. Briefly, \u2018porting\u2019 the MESH ontology to GO format facilitates the use of various software applications that have been developed for manual curation, editing and maintenance of the Gene Ontology (GO), allowing the rapid addition of human gene and disease information to MESH, and where necessary, allowing the alteration of the MESH ontology itself so as to render it a more suitable container for this information. A further integration of the GO terminology and refinement of the disease ontology for genetic disorders can facilitate the preselection of the set of markers.","We outline a methodology for doing so below, and also propose a text-mining approach (Yandell M D, Majoros W H, , Nat Rev Genet 3: 601-10 [2002]) that makes use of the HGMD database to jump-start the entire effort by rapidly assigning the 1,037 human disease causing genes contained in HGMD to MESH via an automatic procedure. Preliminary studies indicate that assigning the human disease genes to MESH can bring many, as yet unappreciated relationships, between sequence homology and disease to light.  illustrates that in several cases genes involved in hand deformities are also involved in foot deformities, and that in some cases these genes are homologous to one another, e.g., PAX3, HOXA13, and MSX1 (all are homeodomain proteins); whereas FOXC2 and FOXE1 are paralogous genes of the Fork head class. In other cases, genes involved in related diseases are involved in the interacting signaling pathways and developmental processes, e.g., FGFR2 and GDF5 (for reviews see, Blundell T L et al., -, Biol Chem 381: 955-9 [2000]; Buxton P et al., -5 (-5) , J Bone Joint Surg Am 83-A: S23-30 [2001]; Thiery J P, , Bull Acad Natl Med 185: 1279-92 [2001]). By systemically assigning all known human disease genes to MESH, previously unnoticed relationships between disease-causing genes. In many cases, genes homologous to a known disease gene\u2014but not yet implicated in that disease\u2014may prove to be involved in causing those same diseases. Tying genes to MESH provides an important resource allowing hypothesis generation for human disease research. Statistically significant associations between SNPs, haplotypes, and complex diseases can be significantly simplified by first assigning genes to MeSH.","In accordance with the invention, the assignment of genes to MESH provides a logical starting point by which to choose sets of genetic markers already implicated in a similar clinical phenotype, thereby revealing significant correlations between these gene marker sets and (related) disease phenotypes using SNP and haplotype data\u2014and thereby, circumventing much of the statistical noise associated with large scale studies that attempt to identify groups of genes (without prior knowledge) using polymorphism data for all or a randomly chosen subset of genes. Placing genes in MESH also provides a means by which to cross-validate more traditional approaches to SNP and haplotype analysis.","The core database system consists of four major database components:\n\n","The disease marker database has very many unstructured data points from genome sequence strings to disease ontologies such as MESH (Schulman J-L, 2001 , NLM Tech Bull. 317 [2001]). While loading these databases from public data repositories, various data consistency checks need to be implemented due to the unreliable and error-prone data stored in these databases. Further attention has to be paid to the fact that there is much redundancy in gene databases, which is mostly due to the unstructured gene nomenclature, only recently subject to efforts at unification.","For individualized genome-profiles there is physically separate storage of genetic data and the individual's geographical data due to extremely important privacy concerns. The genetic profile report database is a virtual joint over many entries from all the databases above. Nevertheless, any updates made by hand-editors such as genetic counselors should propagate back to the origin of the information storage. This versioning system is important to reduce redundant hand-editing.","The relationship between genes including their DNA sequences, mRNA sequences and protein sequences, and clinical disease phenotypes is annotated by automatic text mining algorithms to search and pre-filter literature database such as MEDLINE, employing a computer interface for disease marker phenotype annotation using disease ontology and clinical study abstracts (see, e.g., Example 4 above).","Existing disease ontologies such as MESH (Schulman, J-L, 2001 , NLM Tech Bull. 317 [2001]) are very technical and therefore not generally suited for the communication to individuals. The following example (Table 5) illustrates one embodiment of an organizational matrix for communicating genetic information to the individual in accordance with the invention. Specific focus is given to the most significant markers, most informative markers, markers specifically relevant for the gender, markers related to aging and markers that have a well-described pharmacogenomics application. Here, the individual is a hypothetical \u201cMs. Smith,\u201d to whom the terminology and structure of the report is explained in overview, and then detailed information is provided. Examples of other sections that are not shown in table 5 but which can be included are a glossary, a list of references for further information, a health risk appraisal, and an index. A \u201cRare Genetic Conditions or Carrier Report\u201d at the end of the sample report lists, in alphabetical order, a panoply of genetic conditions for which testing was done, but is presented here in truncated form for the sake of brevity. \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n"],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0043","num":"0052"},"figref":"FIG. 1","i":"Gene ontology: tool for the unification of biology. The Gene Ontology Consortium"},{"@attributes":{"id":"p-0044","num":"0053"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0045","num":"0054"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0046","num":"0055"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0047","num":"0056"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0048","num":"0057"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0049","num":"0058"},"figref":"FIG. 7"},{"@attributes":{"id":"p-0050","num":"0059"},"figref":["FIG. 8","FIG. 8"],"i":"WormBase: network access to the genome and biology of Caenorhabditis elegans"},{"@attributes":{"id":"p-0051","num":"0060"},"figref":"FIG. 9"}]},"DETDESC":[{},{}]}
