---
title: Desaturases and methods of using them for synthesis of polyunsaturated fatty acids
abstract: The amino acid and nucleic acid sequences of a Δ-desaturase enzyme and a Δ-desaturase enzyme are disclosed. The nucleic acid sequences can be used to design recombinant DNA constructs and vectors. These vectors can then be used to transform various organisms, including for example, plants and yeast. The transformed organisms will then produce polyunsaturated fatty acids. The amino acid sequences are useful for generating enzyme-specific antibodies that are useful for identifying the desaturases.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=07402735&OS=07402735&RS=07402735
owner: Washington State University Research Foundation
number: 07402735
owner_city: Pullman
owner_country: US
publication_date: 20041026
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS REFERENCE TO RELATED APPLICATIONS","STATEMENT OF GOVERNMENTAL INTEREST","FIELD OF THE INVENTION","BACKGROUND","SUMMARY OF THE DISCLOSURE","SEQUENCE LISTING","DESCRIPTION OF THE INVENTION","DESCRIPTION AND GENERAL METHODS OF THE DISCLOSURE","EXPERIMENTAL EXAMPLES","Example 1","Organism Strains and Culture","Example 2","Database Searches for Gene Homologs","Example 3","RNA Isolation, Reverse Transcription PCR, and RACE (Rapid Amplification of cDNA Ends)","Example 4","PCR Amplification of the Genes Encoding \u0394and \u0394Desaturases","Example 5","Expression of \u0394and \u0394-desaturases","Example 6","Analysis of Fatty Acids Using Gas Chromatography and GC-Mass Spectrometry","Example 7","Identification and Amplification of the -desaturase Gene","Example 8","Similarity Between -desaturase and Other Proteins","Example 9","Activity of the -desaturase Protein","Example 10","Identification and Cloning of Two Fatty Acid Desaturase Genes","Example 11","Sequence Comparisons for ","Example 12","Fatty Acid Desaturase Activity and Substrate Specificity in Yeast for ","Example 13","Discussion of the -desaturase","Example 14","Discussion of the -desaturase","Example 15","A Plant Cell Transformed with the \u0394and \u0394-desaturase Genes of the Invention","Example 16","A Yeast Cell Transformed with the \u0394- and \u0394-desaturase Genes of the Invention"],"p":["This is a divisional of U.S. patent application Ser. No. 09\/857,583, filed Aug. 17, 2001, now U.S. Pat. No. 6,825,017 which is the National Stage of International Application No. PCT\/US99\/28655, filed Dec. 6, 1999, and claims the benefit of U.S. Provisional Application No. 60\/111,301, filed Dec. 7, 1998, all three of which are incorporated herein by reference.","This work was supported by funds from the U.S. Department of Agriculture under NRICGP contract numbers 95-37301-2287 and 97-35301-4426. The government may have certain rights in this invention.","The invention relates to desaturase enzymes that can be used to produce polyunsaturated fatty acids with important dietary applications.","Fatty acids are fundamental components of living systems. They make up the major component of cytoplasmic membranes, common to plants, animals and protists alike.","Fatty acids of 20 carbons, with more than one unsaturated carbon-carbon bond along the hydrocarbon chain, are known to be of particular importance. Arachidonate (20:4) (Heinz, , pp. 33-89, 1993; Yamazaki et al. 1123:18-26, 1992; Ulsamer et al., 43:105-114, 1969; and Albert et al. 14:498-500, 1979) and eicosapentaenoate (20:5) (Heinz, , pp. 33-89, 1993; Yamazaki et al., 1123:18-26, 1992; Ulsamer et al., 43:105-114, 1969; Albert et al. 14:498-500, 1979; and Cook et al., 32:1265-1273, 1991), commonly referred to as EPA, are significant components of mammalian cell membranes and are also precursors of signal molecules including prostaglandins. Certain specialized mammalian tissues such as brain (Naughton, 13:21-32, 1981), testes (Wilder and Coniglio, 177:399-405, 1984), and retina (Aveldano de Caldironi et al., 20:49-57, 1981) are especially rich in unsaturated fatty acids.","Arachidonate and eicosapentaenoate serve both as precursors for synthesis of 22-carbon polyunsaturated fatty acids and, with dihomo-\u03b3-linoleate (20:3) (Yamazaki et al., 1123:18-26, 1992; Ulsamer et al., 43:105-114, 1969; and Albert et al., 14:498-500, 1979), as precursors to the synthesis of eicosanoid metabolic regulators (Hwang, 545-557, 1992). Key enzymes in the synthesis of 20-carbon fatty acids are desaturases, which introduce cis double bonds by removing two hydrogen atoms at specific locations along the aliphatic hydrocarbon chains. Desaturase enzymes are specific to the position, number, and stereochemistry of the double bonds already present in the target fatty acid (Heinz, 33-89, 1993).","To synthesize 20-carbon polyunsaturated fatty acids, mammals must acquire the essential fatty acids 18:2 (Brenner, , pp. 45-79, 1989) and 18:3 (Nelson, , pp. 437-471, 1992; Brenner, , pp. 45-79, 1989; and Hulanicka et al. 239:2778-2787, 1964) from their diet (Nelson, 437-471, 1992). These dietary polyunsaturated fatty acids are metabolized in the endoplasmic reticulum by an alternating series of position-specific desaturases and malonyl-CoA-dependent chain-elongation steps (), which results in the characteristic methylene-interrupted double bond pattern. In the liver, which is the primary organ of human lipid metabolism, the first step in biosynthesis of 20-carbon fatty acids is desaturation of the essential fatty acids at the \u0394position. The desaturation products are elongated to 20:3 and 20:4 (Cook et al., 32:1265-1273, 1991). In turn, these 20-carbon products are desaturated by a \u0394-desaturase to produce arachidonate and eicosapentaenoate. The \u0394-desaturation step is rate-limiting in this metabolic pathway (Bernet and Sprecher, 398:354-363, 1975; and Yamazaki et al., 1123:18-26, 1992) and, not surprisingly, is subject to regulation by dietary and hormonal changes (Brenner, , pp. 45-79, 1989).","In contrast to the liver, an alternate pathway for biosynthesis of 20-carbon polyunsaturated fatty acids has been demonstrated in a few organisms and tissues (). Instead of desaturation, the first step in the alternate pathway is elongation of the essential 18-carbon fatty acids to 20-carbon chain lengths, producing 20:2 (Ulsamer et al., 43:105-114, 1969; and Albert et al. 14:498-500, 1979) and 20:3. Subsequent desaturation occurs via a \u0394-desaturase (). The products of this elongation-desaturation, 20:3 and 20:4, are the same as the more usual desaturation-elongation pathway. The \u0394pathway is present in the soil amoebae sp. (Ulsamer et al, 43:105-114, 1969), and in euglenoid species, where it is the dominant pathway for formation of 20-carbon polyunsaturated fatty acids (Hulanicka et al., 239:2778-2787, 1964).","This \u0394-desaturation pathway occurs in mammals, both in rat testis (Albert and Coniglio, 489:390-396, 1977) and in human testis (Albert et al., 14:498-500, 1979). While \u0394activity has been observed in breast cancer cell lines (Grammatikos et al., 70:219-227, 1994; and Bardon et al., 99:51-58, 1996) and in glioma (Cook et al., 32:1265-1273, 1991), no \u0394activity is detectable in a corresponding non-cancerous breast cell line (Grammatikos et al., 70:219-227, 1994) or in the brain (Dhopeshwarkar and Subramanian, 36:1175-1179, 1976). The significance of \u0394-desaturation to normal or cancer cell metabolism is unclear, since analysis of desaturase activities in mammalian systems is frequently complicated by the presence of competing \u0394reactions and chain-shortening retroconversion of fatty acid substrates in tissue (Sprecher and Lee, 388:113-125, 1975; Geiger et al., 1170:137-142, 1993).","Polyunsaturated 20-carbon fatty acids are, for the reasons outlined above, important in the human diet, and there has been considerable recent interest in incorporating such fatty acids into infant food, baby formula, dietary supplements, and nutriceutical formulations.","It would therefore be desirable to produce new transgenic plants and animals with enhanced ability to produce polyunsaturated 20-carbon fatty acids.","The invention provides novel \u0394\u2014() and \u0394\u2014() desaturase enzymes that may be cloned and expressed in the cells of various organisms, including plants, to produce 20-carbon polyunsaturated fatty acids. Expression of such fatty acids enhances the nutritional qualities of such organisms. For instance, oil-seed plants may be engineered to incorporate the \u0394- and \u0394-desaturases of the invention. Such oil-seed plants would produce seed-oil rich in polyunsaturated 20:3, 20:4, 20:5, 22:4, and 22:5 fatty acids. Such fatty acids could be incorporated usefully into infant formula, foods of all kinds, dietary supplements, nutriceutical, and pharmaceutical formulations.","The invention also provides proteins differing from the proteins of  and  by one or more conservative amino acid substitutions. Also provided are proteins that exhibit \u201csubstantial similarity\u201d (defined in the \u201cDefinitions\u201d section) with the proteins of  and .","The invention provides isolated novel nucleic acids that encode the above-mentioned proteins, recombinant nucleic acids that include such nucleic acids and cells, and plants and organisms containing such recombinant nucleic acids.","The novel \u0394- and \u0394-desaturase enzymes can be used individually, or in conjunction with one another, for instance in a metabolic pathway, to produce polyunsaturated fatty acids, such as 20:3, 20:4, 20:5, 22:4, and 22:5 fatty acids.","The scope of the invention also includes portions of nucleic acids encoding the novel \u0394- and \u0394-desaturase enzymes, portions of nucleic acids that encode polypeptides substantially similar to these novel enzymes, and portions of nucleic acids that encode polypeptides that differ from the proteins of  and  by one or more conservative amino acid substitutions. Such portions of nucleic acids may be used, for instance, as primers and probes for research and diagnostic purposes. Research applications for such probes and primers include the identification and cloning of related \u0394- and \u0394-desaturases in other organisms including humans.","The invention also includes methods that utilize the \u0394- and\/or the \u0394-desaturase enzymes of the invention. An example of this embodiment is a yeast or plant cell that carries genes for one or both desaturases of the invention and that, by virtue of these desaturases, is able to produce arachidonic acid and\/or EPA.","The nucleic and amino acid sequences listed in the accompanying sequence listing are shown using standard letter abbreviations for nucleotide bases, and three-letter code for amino acids. Only one strand of each nucleic acid sequence is shown, but the complementary strand is understood to be included by any reference to the displayed strand.","SEQ ID NO: 1 is the nucleotide sequence corresponding to the open reading frame of the fatty acid \u0394-desaturase from ","SEQ ID NO: 2 is the primary amino acid sequence of the fatty acid \u0394-desaturase from ","SEQ ID NO: 3 is the nucleotide sequence corresponding to the open reading frame of the fatty acid \u0394-desaturase from the protist ","SEQ ID NO: 4 is the primary amino acid sequence of fatty acid \u0394-desaturase from the protist ","SEQ ID NOs: 5-8 are primers used to amplify and clone the \u0394-desaturase-encoding nucleic acid sequence.","SEQ ID NO: 9 is a polyadenylation signal.","SEQ ID NO: 10 is a primer used to amplify and clone the \u0394-desaturase-encoding nucleic acid sequence.","SEQ ID NO: 11 is a short RNA leader sequence.","SEQ ID NO: 12 is the amino acid sequence of a histidine box motif.","SEQ ID NO: 13 is the amino acid sequence of a histidine box motif","SEQ ID NO: 14 is the primary amino acid sequence of the fatty acid \u0394-desaturase from ","SEQ ID NO: 15 is the primary amino acid sequence of the fatty acid \u0394-desaturase from ","SEQ ID NO: 16 is the primary amino acid sequence of the fatty acid \u0394-desaturase from ","SEQ ID NO: 17 is the amino acid sequence of a histidine box motif.","The following definitions and methods are provided to better define the present invention and to guide those of ordinary skill in the art in the practice of the present invention. Unless otherwise noted, terms are to be understood according to conventional usage by those of ordinary skill in the relevant art. Definitions of common terms in molecular biology may also be found in Rieger et al., 5th edition, Springer-Verlag: New York, 1991; and Lewin, , Oxford University Press: New York, 1997. The nomenclature for DNA bases as set forth at 37 C.F.R. \u00a7 1.822 is used. The standard one- and three-letter nomenclature for amino acid residues is used.","Definitions","Portion: A portion of a nucleic acid molecule is a stretch of contiguous nucleic acids corresponding to the sequence of that molecule that may be about 15, 20, 30, 40, 50, or 60 nucleic acids in length. Such nucleotide portions may be used as probes or primers. A portion of a protein is a stretch of contiguous amino acids corresponding to the amino acid sequence of that protein that may be about 5, 10, 20, 30, 40, or 50 residues in length. As used herein, such a portion may correspond to any segment of a nucleic acid molecule, for instance such a portion may correspond to a segment consisting of nucleotides 1-500, 501-1000, or 1001-1451 of the sequence shown in , or nucleotides 1-400, 401-800, 801-1251 of the sequence shown in .","Desaturase: A desaturase is an enzyme that promotes the formation of carbon-carbon double bonds in a hydrocarbon molecule.","Desaturase activity may be demonstrated by assays in which a preparation containing an enzyme is incubated with a suitable form of substrate fatty acid and analyzed for conversion of this substrate to the predicted fatty acid product. Alternatively, a DNA sequence proposed to encode a desaturase protein may be incorporated into a suitable vector construct and thereby expressed in cells of a type that do not normally have an ability to desaturate a particular fatty acid substrate. Activity of the desaturase enzyme encoded by the DNA sequence can then be demonstrated by supplying a suitable form of substrate fatty acid to cells transformed with a vector containing the desaturase-encoding DNA sequence and to suitable control cells (for example, transformed with the empty vector alone). In such an experiment, detection of the predicted fatty acid product in cells containing the desaturase-encoding DNA sequence and not in control cells establishes the desaturase activity. Examples of this type of assay have been described in, for example, Lee et al., 280:915-918, 1998; Napier et al., 330:611-614, 1998; and Michaelson et al., 273:19055-19059, 1998, which are incorporated herein by reference.","The \u0394-desaturase activity may be assayed by these techniques using, for example, 20:3\u0394as substrate and detecting 20:4\u0394as the product (Michaelson et al., 273:19005-19059, 1998). Other potential substrates for use in \u0394activity assays include (but are not limited to) 10:2\u0394(yielding 20:5\u0394as the product) and 20:3\u0394(yielding 20:4\u0394as the product).","The \u0394-desaturase may be assayed by similar techniques using, for example, 20:3\u0394as the substrate and detecting 20:4\u0394as the product.","ORF: Open reading frame. An ORF is a contiguous series of nucleotide triplets coding for amino acids. These sequences are usually translatable into a peptide.","Homologs: Two nucleotide or amino acid sequences that share a common ancestral sequence and diverged when a species carrying that ancestral sequence split into two species. Homologs frequently show a substantial degree of sequence identity.","Transformed: A transformed cell is a cell into which has been introduced a nucleic acid molecule by molecular biology techniques. The term encompasses all techniques by which a nucleic acid molecule might be introduced into such a cell, including transfection with viral vectors, transformation with plasmid vectors, and introduction of naked DNA by electroporation, lipofection, and particle gun acceleration.","Purified: The term purified does not require absolute purity; rather, it is intended as a relative term. Thus, for example, a purified protein preparation is one in which the subject protein or other substance is more pure than in its natural environment within a cell. Generally, a protein preparation is purified such that the protein represents at least 50% of the total protein content of the preparation.","Operably linked: A first nucleic acid sequence is operably linked with a second nucleic acid sequence when the first nucleic acid sequence is placed in a functional relationship with the second nucleic acid sequence. For instance, a promoter is operably linked to a coding sequence if the promoter affects the transcription or expression of the coding sequence. Generally, operably linked DNA sequences are contiguous and, where necessary to join two protein coding regions, in the same reading frame. If introns are present, the operably linked DNA sequences may not be contiguous.","Cell: A plant, animal, protist, bacterial, or fungal cell.","Sequence similarity: The similarity between two nucleic acids or two amino acid sequences is expressed in terms of percentage sequence identity. The higher the percentage sequence identity between two sequences, the more similar the two sequences are.","In the case of protein alignments, similarity is measured not only in terms of percentage identity, but also takes into account conservative amino acid substitutions.","Such conservative substitutions generally preserve the hydrophobicity and acidity of the substituted residue, thus preserving the structure (and therefore function) of the folded protein. The computer programs used to calculate protein similarity employ standardized algorithms that, when used with standardized settings, allow the meaningful comparison of similarities between different pairs of proteins.","Sequences are aligned, with allowances for gaps in alignment, and regions of identity are quantified using a computerized algorithm. Default parameters of the computer program are commonly used to set gap allowances and other variables.","Methods of alignment of sequences for comparison are well-known in the art. Various programs and alignment algorithms are described by Pearson et al., 24: 307-331, 1994, and in Altschul et al., 6:119-129, 1994. Altschul et al. presents a detailed consideration of sequence alignment methods and homology calculations.","The NCBI Basic Local Alignment Search Tool (BLAST\u2122) (Altschul et al., 215:403-410, 1990 is available from several sources, including the National Center for Biotechnology Information (NBCI, Bethesda, Md.) and on the Internet, for use in connection with the sequence analysis programs blastp, blastn, blastx, tblastn and tblastx. BLAST\u2122 can be accessed at htp:\/\/www.ncbi.nlm.nih.gov\/BLAST\/. A description of how to determine sequence identity using this program is available at the web site. As used herein, sequence identity is commonly determined with the BLAST\u2122 software set to default parameters. For instance, blastn (version 2.0) software may be used to determine sequence identity between two nucleic acid sequences using default parameters (expect=10, matrix=BLOSUM62, filter=DUST (Tatusov and Lipmann, in preparation as of Dec. 1, 1999; and Hancock and Armstrong, 10:67-70, 1994), gap existence cost=11, per residue gap cost=1, and lambda ratio=0.85). For comparison of two polypeptides, blastp (version 2.0) software may be used with default parameters (expect 10, filter=SEG (Wootton and Federhen, 17:149-163, 1993), matrix=BLOSUM62, gap existence cost=11, per residue gap cost=1, lambda=0.85).","When aligning short peptides (fewer than around 30 amino acids), the alignment should be performed using the Blast 2 sequences function, employing the PAM30 matrix set to default parameters (open gap 9, extension gap 1 penalties).","An alternative alignment tool is the ALIGN Global Optimal Alignment tool (version 3.0) available from Biology Workbench at http:\/\/biology.ncsa.uiuc.edu. This tool may be used with settings set to default parameters to align two known sequences. References for this tool include Meyers and Miller, 4:11-17, 1989.","Conservative amino acid substitutions are those substitutions that, when made, least interfere with the properties of the original protein, i.e., the structure and especially the function of the protein is conserved and not significantly changed by such substitutions. The table below shows amino acids that may be substituted for an original amino acid in a protein and that are regarded as conservative substitutions.",{"@attributes":{"id":"p-0072","num":"0071"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"left"}}],"thead":{"row":[{"entry":"TABLE 1"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["Original Residue","Conservative Substitutions"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["ala","ser"]},{"entry":["arg","lys"]},{"entry":["asn","gln; his"]},{"entry":["asp","glu"]},{"entry":["cys","ser"]},{"entry":["gln","asn"]},{"entry":["glu","asp"]},{"entry":["gly","pro"]},{"entry":["his","asn; gln"]},{"entry":["ile","leu; val"]},{"entry":["leu","ile; val"]},{"entry":["lys","arg; gln; glu"]},{"entry":["met","leu; ile"]},{"entry":["phe","met; leu; tyr"]},{"entry":["ser","thr"]},{"entry":["thr","ser"]},{"entry":["trp","tyr"]},{"entry":["tyr","trp; phe"]},{"entry":["val","ile; leu"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}}}},"Conservative substitutions generally maintain (a) the structure of the polypeptide backbone in the area of the substitution, for example, as a sheet or helical conformation, (b) the charge or hydrophobicity of the molecule at the target site, or (c) the bulk of the side chain.","The substitutions which in general are expected to produce the greatest changes in protein properties will be non-conservative, for instance changes in which (a) a hydrophilic residue, e.g., seryl or threonyl, is substituted for (or by) a hydrophobic residue, e.g., leucyl, isoleucyl, phenylalanyl, valyl or alanyl; (b) a cysteine or proline is substituted for (or by) any other residue; (c) a residue having an electropositive side chain, e.g., lysyl, arginyl, or histadyl, is substituted for (or by) an electronegative residue, e.g., glutamyl or aspartyl; or (d) a residue having a bulky side chain, e.g., phenylalanine, is substituted for (or by) one not having a side chain, e.g., glycine.","Probe: An isolated nucleic acid attached to a detectable label or reporter molecule. Typical labels include radioactive isotopes, ligands, chemiluminescent agents, and enzymes.","Primers: Short nucleic acids, preferably DNA oligonucleotides 10 nucleotides or more in length, that are annealable to a complementary target DNA strand by nucleic acid hybridization to form a hybrid between the primer and the target DNA strand, then extendable along the target DNA strand by a DNA polymerase enzyme. Primer pairs can be used for amplification of a nucleic acid sequence, e.g., by the polymerase chain reaction (PCR) or other nucleic-acid amplification methods known in the art.","Probes and primers as used in the present invention typically comprise at least 15 contiguous nucleotides. In order to enhance specificity, longer probes and primers may also be employed, such as probes and primers that comprise at least 20, 30, 40, 50, 60, 70, 80, 90, 100, or 150 consecutive nucleotides of the disclosed nucleic acid sequences.","Alternatively, such probes and primers may comprise at least 15, 20, 30, 40, 50, 60, 70, 80, 90, 100, or 150 consecutive nucleotides that share a defined level of sequence identity with one of the disclosed sequences, for instance, at least a 50%, 60%, 70%, 80%, 90%, or 95% sequence identity.","Alternatively, such probes and primers may be nucleotide molecules that hybridize under specific conditions and remain hybridized under specific wash conditions such as those provided below. These conditions can be used to identifying variants of the desaturases. Nucleic acid molecules that are derived from the desaturase cDNA and gene sequences include molecules that hybridize under various conditions to the disclosed desaturase nucleic acid molecules, or fragments thereof. Generally, hybridization conditions are classified into categories, for example very high stringency, high stringency, and low stringency. The conditions for probes that are about 600 base pairs or more in length are provided below in three corresponding categories.","Very High Stringency (detects sequences that share 90% sequence identity)",{"@attributes":{"id":"p-0080","num":"0079"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"56pt","align":"left"}}],"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Hybridization","in","SSC","at","65\u00b0 C.","16 hours"]},{"entry":["Wash twice","in","SSC","at","room temp.","15 minutes each"]},{"entry":["Wash twice","in","SSC","at","65\u00b0 C.","20 minutes each"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]}}}},"br":{}},{"@attributes":{"id":"p-0081","num":"0080"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"63pt","align":"left"}}],"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Hybridization","in","SSC","at","65\u00b0 C.-70\u00b0 C.","16-20 hours"]},{"entry":["Wash twice","in","SSC","at","room temp.","\u20025-20 minutes each"]},{"entry":["Wash twice","in","SSC","at","55\u00b0 C.-70\u00b0 C.","30 minutes each"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]}}}},"br":{}},{"@attributes":{"id":"p-0082","num":"0081"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"42pt","align":"left"}}],"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Hybridization","in","SSC","at","room temp.-55\u00b0 C.","16-20 hours"]},{"entry":["Wash at least","in","SSC","at","room temp.-55\u00b0 C.","20-30"]},{"entry":["twice",{},{},{},{},"minutes each"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]}}}}},"Methods for preparing and using probes and primers are described in the references, for example Sambrook et al. 2ed., Cold Spring Harbor Laboratory Press, NY, 1989; Ausubel et al. , Greene Publishing Associates and Wiley-Intersciences, 1987; and Innis et al., , Academic Press, Inc., San Diego, Calif. 1990. PCR primer pairs can be derived from a known sequence, for example, by using computer programs intended for that purpose such as Primer (Version 0.5, 1991, Whitehead Institute for Biomedical Research, Cambridge, Mass.).","Recombinant nucleic acid: A sequence that is not naturally occurring or has a sequence that is made by an artificial combination of two otherwise separated segments of sequence. This artificial combination is often accomplished by chemical synthesis or, more commonly, by the artificial manipulation of isolated segments of nucleic acids, e.g., by genetic engineering techniques such as those described in Sambrook et al. 2ed., Cold Spring Harbor Laboratory Press, NY, 1989. The term recombinant includes nucleic acids that have been altered solely by addition, substitution, or deletion of a portion of the nucleic acid.","Native: The term \u201cnative\u201d refers to a naturally-occurring (\u201cwild-type\u201d) nucleic acid or polypeptide. The native nucleic acid or protein may have been physically derived from a particular organism in which it is naturally occurring or may be a synthetically constructed nucleic acid or protein that is identical to the naturally-occurring nucleic acid or protein.","Isolated: An \u201cisolated\u201d nucleic acid is one that has been substantially separated or purified away from other nucleic acid sequences in the cell of the organism in which the nucleic acid naturally occurs, i.e., other chromosomal and extrachromosomal DNA and RNA, by conventional nucleic acid-purification methods. The term also embraces recombinant nucleic acids and chemically synthesized nucleic acids.","Plant: The term \u201cplant\u201d encompasses any higher plant and progeny thereof, including monocots (e.g., corn, rice, wheat, barley, rapeseed, soy, sunflower, etc.), dicots (e.g., potato, tomato, etc.), and includes parts of plants, including seeds, fruit, tubers, etc.","The invention will be better understood by reference to the Examples herein. The scope of the invention is not to be considered limited thereto.","The present invention utilizes standard laboratory practices for the cloning, manipulation, and sequencing of nucleic acids, the purification and analysis of proteins, and other molecular biological and biochemical techniques, unless otherwise stipulated. Such techniques are explained in detail in standard laboratory manuals such as Sambrook et al., 2ed., Cold Spring Harbor Laboratory Press, NY, 1989; and Ausubel et al., , Green and Wiley-Interscience, NY, 1987.","The inventors have identified, cloned, and expressed a novel fatty acid \u0394-desaturase from the protist and a novel fatty acid \u0394-desaturase from that may be used together to produce polyunsaturated fatty acids.","The invention provides novel purified \u0394and \u0394proteins ( and , respectively). The invention also provides proteins differing from the proteins of  and  by one or more conservative amino acid substitutions, as well as proteins that show \u201csubstantial similarity\u201d with the proteins of  and . Substantial similarity is defined in the \u201cDefinitions\u201d section. Proteins of the invention include proteins that show at least 50% amino acid similarity with the proteins shown in  and . The term \u201c50% amino acid similarity\u201d is objectively and consistently defined by use of blastp sequence analysis software set at default parameters. Proteins of the invention also include proteins showing at least 60%, at least 70%, at least 80%, at least 90%, and at least 95% similarity (to the sequences of  or ) using blastp with default parameters.","The invention provides isolated novel nucleic acids that encode the above-mentioned proteins, recombinant nucleic acids that include such nucleic acids and cells containing such recombinant nucleic acids. Nucleic acids of the invention thus include nucleic acids that encode: (1) amino acid sequences as shown in  and ; (2) amino acid sequences that differ from the sequences shown in  and  by one or more conservative amino acid substitutions; and (3) amino acid sequences that show at least 50% similarity (as measured by blastp at default parameters) with the sequence of  and .","Nucleic acids of the invention also include nucleic acids that show at least the term \u201c50% similarity\u201d with the nucleic acids shown in  and . The term \u201c50% similarity\u201d is objectively defined by the use of blastn software set at default perimeters. Nucleic acids of the invention also include nucleic acids showing at least 60%, at least 70%, at least 80%, at least 90%, and at least 95% similarity (to the sequences of  and ) using blastn with default perimeters.","The novel \u0394- and \u0394-desaturase enzymes can be used individually, or in conjunction with one another, for instance in a metabolic pathway, to produce polyunsaturated fatty acids, such as 20:3 and 20:4 fatty acids.  shows an example of such a metabolic pathway. Such a pathway may be engineered into any cell by use of appropriate expression systems. A simple way to provide such elements is by the use of commercially available expression systems, discussed in detail below.","The scope of the invention covers not only entire nucleic acids encoding the novel \u0394- and \u0394desaturase enzymes (and substantially similar derivatives of such enzymes) but also covers \u201cportions\u201d of such nucleic acids (as defined in the \u201cDefinitions\u201d section, herein). Such claimed portions are identified by their possession of a particular degree of similarity with similar sized portions of the nucleotides of  and  and may have a length of about 15, 20, 30, 40, or 50 contiguous nucleotides. Similarity is objectively measured by sequence comparison software, such as the \u201cblastn\u201d and \u201cblastp\u201d software available from the National Center for Biotechnology Information (NBCI, Bethesda, Md.) and on the Internet at htp:\/\/www.ncbi.nlm.nih.gov\/BLAST\/. Similarity between portions of nucleic acids claimed and similar sized portions of the nucleic acid sequences of  and  may be at least 50%, 60%, 70%, 80%, 90%, 95%, or even 98%. Such portions of nucleic acids may be used, for instance, as primers and probes for research and diagnostic purposes. Portions of nucleic acids may be selected from any area of the sequences shown in  or , for instance the first, second, third, etc., group of 100 nucleic acids as numbered in the figures.","Recombinant nucleic acids, as mentioned above, may, for instance, contain all or portion of a disclosed nucleic acid operably linked to another nucleic acid element such as a promoter, for instance, as part of a clone designed to express a protein. Cloning and expression systems are commercially available for such purposes.","Various yeast strains and yeast-derived vectors are commonly used for expressing and purifying proteins, for example, expression systems are available from Invitrogen (Carlsbad, Calif.). Such systems include suitable strains, vectors, reagents, sequencing primers, and media. A similar system for expression of proteins in is also available from Invitrogen, which includes vectors, reagents and media. For example, a nucleotide sequence (e.g., a gene coding for the \u0394- or \u0394-desaturase enzyme of the invention) may be cloned into the yeast expression vector pYES2 and expressed under the control of an inducible promoter, such as a galactose-inducible promoter (GAL1).","Non-yeast eukaryotic vectors may also be used for expression of the desaturases of the invention. Examples of such systems are the well known Baculovirus system, the Ecdysone-inducible mammalian expression system that uses regulatory elements from to allow control of gene expression, and the Sindbis viral expression system that allows high level expression in a variety of mammalian cell lines. These expression systems are also available from Invitrogen.","Standard prokaryotic cloning vectors may also be used, for example pBR322, pUC18 or pUC19 as described in Sambrook et al, 1989. Nucleic acids encoding the desaturases of the invention may be cloned into such vectors that may then be transformed into bacteria such as () which may then be cultured so as to express the protein of interest. Other prokaryotic expression systems include, for instance, the arabinose-induced pBAD expression system that allows tightly controlled regulation of expression, the IPTG-induced pRSET system that facilitates rapid purification of recombinant proteins and the IPTG-induced pSE402 system that has been constructed for optimal translation of eukaryotic genes. These three systems are available commercially from Invitrogen and, when used according to the manufacturer's instructions, allow routine expression and purification of proteins.","Alternatively, and of particular importance to this invention, a plant expression system could be used. Plant expression systems are commercially available. A gene of interest of the invention may be cloned into a vector and the construct used to transform a plant cell. Any well known vector suitable for stable transformation of plant cells and\/or for the establishment of transgenic plants may be used, including those described in, e.g., Pouwels et al., 1985, supp. 1987; Weissbach and Weissbach, , Academic Press, 1989; and Gelvin et al., , Kluwer Academic Publishers, 1990. Such plant expression vectors can include expression control sequences (e.g., inducible or constitutive, environmentally or developmentally regulated, or cell- or tissue-specific expression-control sequences).","Examples of constitutive plant promoters useful for expressing desaturase enzymes in plants include, but are not limited to, the cauliflower mosaic virus (CaMV) 35S promoter (see, e.g., Odel et al., 313:810, 1985; Dekeyser et al., 2:591, 1990; and Terada and Shimamoto, 220:389, 1990); the nopaline synthase promoter (An et al., 88:547, 1988) and the octopine synthase promoter (Fromm et al., 1:977, 1989).","A variety of plant-gene promoters that are regulated in response to environmental, hormonal, chemical, and\/or developmental signals, also can be used for protein expression in plant cells, including promoters regulated by (1) heat (Callis et al., 88:965, 1988), (2) light (e.g., pea rbcS-3A promoter, Kuhlemeier et al., 1:471, 1989; maize rbcS promoter, Schaffner and Sheen, 3:997, 1991; or chlorophyll a\/b-binding protein promoter, Simpson et al., 4:2723, 1985), (3) hormones, such as abscisic acid (Marcotte et al., 1:969, 1989), (4) wounding (e.g., wunI, Siebertz et al., 1:961, 1989); or (5) chemicals such as methyl jasmonate, salicylic acid, or a safener. It may also be advantageous to employ (6) organ-specific promoters (e.g., Roshal et al., 6:1155, 1987; Schernthaner et al., 7:1249, 1988; Bustos et al., 1:839, 1989; Zheng et al., 4:357-366, 1993). Tissue-specific expression may be facilitated by use of certain types of promoters, for example, the napin promoter is a seed-storage protein promoter from and specific to developing seeds. The \u03b2-conglycinin promoters drive the expression of recombinant nucleic acids thus allowing, the \u0394or \u0394proteins of the invention to be expressed only in specific tissues, for example, seed tissues.","Plant expression vectors can include regulatory sequences from the 3\u2032-untranslated region of plant genes (Thornburg et al., 84:744, 1987; An et al., 1:115, 1989), e.g., a 3\u2032 terminator region to increase mRNA stability of the mRNA, such as the PI-II terminator region of potato or the octopine or nopaline synthase 3\u2032 terminator regions.","Useful dominant selectable marker genes for expression in plant cells include, but are not limited to: genes encoding antibiotic-resistance genes (e.g., resistance to hygromycin, kanamycin, bleomycin, G418, streptomycin, or spectinomycin); and herbicide-resistance genes (e.g., phosphinothricin acetyltransferase). Useful screenable markers include, but are not limited to, \u03b2-glucuronidase and green fluorescent protein.","The invention also provides cells or plants or organisms transformed with recombinant nucleic acid constructs that include all or a portion of the newly discovered polynucleotides that encode the novel \u0394and\/or \u0394desaturase enzymes. An example of such a transformed plant or organism would be a potato, tomato, rapeseed, sunflower, soy, wheat, or corn plant. Multi-celled fungi, such as edible mushrooms, may also be transformed. Transformed oil-seed plants are of particular interest as 20-carbon polyunsaturated fatty acids would accumulate within the seed-oil.","Nucleic acid constructs that express a nucleic acid according to the invention can be introduced into a variety of host cells or organisms in order to alter fatty acid biosynthesis. Higher plant cells, eukaryotic, and prokaryotic host cells all may be so transformed using an appropriate expression system as described above.","After a cDNA (or gene) encoding a desaturase has been isolated, standard techniques may be used to express the cDNA in transgenic plants in order to modify the particular plant characteristic. The basic approach is to clone the cDNA into a transformation vector, such that the cDNA is operably linked to control sequences (e.g., a promoter) directing expression of the cDNA in plant cells. The transformation vector is then introduced into plant cells by any of various techniques, for example by -mediated transformation of plants or plant tissues, or by electroporation of protoplasts, and progeny plants containing the introduced cDNA are selected. All or part of the transformation vector stably integrates into the genome of the plant cell. That part of the transformation vector that integrates into the plant cell and that contains the introduced cDNA and associated sequences for controlling expression (the introduced \u201ctransgene\u201d) may be referred to as the \u201crecombinant expression cassette.\u201d","Selection of progeny plants containing the introduced transgene may be made based upon the detection of an altered phenotype. Such a phenotype may result directly from the cDNA cloned into the transformation vector or may be manifested as enhanced resistance to a chemical agent (such as an antibiotic) as a result of the inclusion of a dominant selectable marker gene incorporated into the transformation vector.","Successful examples of the modification of plant characteristics by transformation with cloned cDNA sequences are replete in the technical and scientific literature. Selected examples, which serve to illustrate the knowledge in this field of technology include:\n\n","These examples include descriptions of transformation vector selection, transformation techniques, and the construction of constructs designed to over-express the introduced cDNA. In light of the foregoing and the provision herein of the desaturase amino acid sequences and nucleic acid sequences, it is thus apparent that one of skill in the art will be able to introduce the cDNAs, or homologous or derivative forms of these molecules, into plants in order to produce plants having enhanced desaturase activity. Furthermore, the expression of one or more desaturases in plants may give rise to plants having increased production of poly-unsaturated fatty acids.","The invention also pertains to antibodies to the desaturase enzymes, and fragments thereof, these antibodies may be useful for purifying and detecting the desaturases. The provision of the desaturase sequences allows for the production of specific antibody-based binding agents to these enzymes.","Monoclonal or polyclonal antibodies may be produced to the desaturases, portions of the desaturases, or variants thereof. Optimally, antibodies raised against epitopes on these antigens will specifically detect the enzyme. That is, antibodies raised against the desaturases would recognize and bind the desaturases, and would not substantially recognize or bind to other proteins. The determination that an antibody specifically binds to an antigen is made by any one of a number of standard immunoassay methods; for instance, Western blotting, Sambrook et al. (ed.), 2nd ed., Vols. 1-3, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1989.","To determine that a given antibody preparation (such as a preparation produced in a mouse against the \u0394-desaturase) specifically detects the desaturase by Western blotting, total cellular protein is extracted from cells and electrophoresed on a SDS-polyacrylamide gel. The proteins are then transferred to a membrane (for example, nitrocellulose) by Western blotting, and the antibody preparation is incubated with the membrane. After washing the membrane to remove non-specifically bound antibodies, the presence of specifically bound antibodies is detected by the use of an anti-mouse antibody conjugated to an enzyme such as alkaline phosphatase; application of 5-bromo-4-chloro-3-indolyl phosphate\/nitro blue tetrazolium results in the production of a densely blue-colored compound by immuno-localized alkaline phosphatase.","Antibodies that specifically detect a desaturase will, by this technique, be shown to bind substantially only the desaturase band (having a position on the gel determined by the molecular weight of the desaturase). Non-specific binding of the antibody to other proteins may occur and may be detectable as a weaker signal on the Western blot (which can be quantified by automated radiography). The non-specific nature of this binding will be recognized by one skilled in the art by the weak signal obtained on the Western blot relative to the strong primary signal arising from the specific anti-desaturase binding.","Antibodies that specifically bind to desaturases belong to a class of molecules that are referred to herein as \u201cspecific binding agents.\u201d Specific binding agents that are capable of specifically binding to the desaturase of the present invention may include polyclonal antibodies, monoclonal antibodies, and fragments of monoclonal antibodies such as Fab, F(ab\u2032)2, and Fv fragments, as well as any other agent capable of specifically binding to one or more epitopes on the proteins.","Substantially pure desaturase suitable for use as an immunogen can be isolated from transfected cells, transformed cells, or from wild-type cells. Concentration of protein in the final preparation is adjusted, for example, by concentration on an Amicon filter device, to the level of a few micrograms per milliliter. Alternatively, peptide fragments of a desaturase may be utilized as immunogens. Such fragments may be chemically synthesized using standard methods, or may be obtained by cleavage of the whole desaturase enzyme followed by purification of the desired peptide fragments. Peptides as short as three or four amino acids in length are immunogenic when presented to an immune system in the context of a Major Histocompatibility complex (MHC) molecule, such as MHC class I or MHC class II. Accordingly, peptides comprising at least 3 and preferably at least 4, 5, 6, or more consecutive amino acids of the disclosed desaturase amino acid sequences may be employed as immunogens for producing antibodies.","Because naturally occurring epitopes on proteins frequently comprise amino acid residues that are not adjacently arranged in the peptide when the peptide sequence is viewed as a linear molecule, it may be advantageous to utilize longer peptide fragments from the desaturase amino acid sequences for producing antibodies. Thus, for example, peptides that comprise at least 10, 15, 20, 25, or 30 consecutive amino acid residues of the amino acid sequence may be employed. Monoclonal or polyclonal antibodies to the intact desaturase, or peptide fragments thereof may be prepared as described below.","Monoclonal antibody to any of various epitopes of the desaturase enzymes that are identified and isolated as described herein can be prepared from murine hybridomas according to the classic method of Kohler & Milstein, 256:495, 1975, or a derivative method thereof. Briefly, a mouse is repetitively inoculated with a few micrograms of the selected protein over a period of a few weeks. The mouse is then sacrificed, and the antibody-producing cells of the spleen isolated. The spleen cells are fused by means of polyethylene glycol with mouse myeloma cells, and the excess unfused cells destroyed by growth of the system on selective media comprising aminopterin (HAT media). The successfully fused cells are diluted and aliquots of the dilution placed in wells of a microtiter plate where growth of the culture is continued. Antibody-producing clones are identified by detection of antibody in the supernatant fluid of the wells by immunoassay procedures, such as ELISA, as originally described by Engvall, 70:419, 1980, or a derivative method thereof. Selected positive clones can be expanded and their monoclonal antibody product harvested for use. Detailed procedures for monoclonal antibody production are described in Harlow & Lane, , Cold Spring Harbor Laboratory, New York, 1988.","Polyclonal antiserum containing antibodies to heterogenous epitopes of a single protein can be prepared by immunizing suitable animals with the expressed protein, which can be unmodified or modified, to enhance immunogenicity. Effective polyclonal antibody production is affected by many factors related both to the antigen and the host species. For example, small molecules tend to be less immunogenic than other molecules and may require the use of carriers and an adjuvant. Also, host animals vary in response to site of inoculations and dose, with both inadequate or excessive doses of antigen resulting in low-titer antisera. Small doses (ng level) of antigen administered at multiple intradermal sites appear to be most reliable. An effective immunization protocol for rabbits can be found in Vaitukaitis et al., 33:988-991, 1971.","Booster injections can be given at regular intervals, and antiserum harvested when the antibody titer thereof, as determined semi-quantitatively, for example, by double immunodiffusion in agar against known concentrations of the antigen, begins to fall. See, for example, Ouchterlony et al., , Wier, D. (ed.), Chapter 19, Blackwell, 1973. A plateau concentration of antibody is usually in the range of 0.1 to 0.2 mg\/mL of serum (about 12 \u03bcM). Affinity of the antisera for the antigen is determined by preparing competitive binding curves using conventional methods.","Antibodies may be raised against the desaturases of the present invention by subcutaneous injection of a DNA vector that expresses the enzymes in laboratory animals, such as mice. Delivery of the recombinant vector into the animals may be achieved using a hand-held form of the Biolistic system (Sanford et al., 5:27-37, 1987, as described by Tang et al., () 356:153-154, 1992). Expression vectors suitable for this purpose may include those that express the cDNA of the enzyme under the transcriptional control of either the human \u03b2-actin promoter or the cytomegalovirus (CMV) promoter. Methods of administering naked DNA to animals in a manner resulting in expression of the DNA in the body of the animal are well known and are described, for example, in U.S. Pat. No. 5,620,896 (\u201cDNA Vaccines Against Rotavirus Infections\u201d); U.S. Pat. No. 5,643,578 (\u201cImmunization by Inoculation of DNA Transcription Unit\u201d); and U.S. Pat. No. 5,593,972 (\u201cGenetic Immunization\u201d), and references cited therein.","Antibody fragments may be used in place of whole antibodies and may be readily expressed in prokaryotic host cells. Methods of making and using immunologically effective portions of monoclonal antibodies, also referred to as \u201cantibody fragments,\u201d are well known and include those described in Better & Horowitz, 178:476-496, 1989; Glockshuber et al. 29:1362-1367, 1990; and U.S. Pat. No. 5,648,237 (\u201cExpression of Functional Antibody Fragments\u201d); U.S. Pat. No. 4,946,778 (\u201cSingle Polypeptide Chain Binding Molecules\u201d); and U.S. Pat. No. 5,455,030 (\u201cImmunotherapy Using Single Chain Polypeptide Binding Molecules\u201d), and references cited therein.","The strain Z was obtained from Columbia Scientific. The organism was cultured on Cramer and Meyers medium (Cramer, and Meyers, 17:384-402, 1952) with the addition of sucrose as a carbon source. Cultures were maintained at 25\u00b0 C. in absolute darkness.",{"@attributes":{"id":"p-0124","num":"0133"},"i":["C. elegans ","Caenorhabditis ","The Nematode Caenorhabditis elegans "]},"The Sanger Center (http:\/\/www.sanger.ac.uk\/projects\/c_elegans\/blast_server.shtml) genomic database was searched using BLAST\u2122 with sequences of plant desaturase enzymes, including the -desaturase (GenBank accession number U79010). Two polypeptides with the highest scores were a peptide on cosmid W08D2 (high score 163), and one on T13F2 (high score 121).","For the gene, total RNA was isolated from heterotrophic cultures of using a phenol-SDS protocol (Ausubel, 1988). Messenger RNA was purified from total RNA using the PolyA-tract system (Promega Scientific, Madison, Wis.). Reverse transcription reactions were carried out using Superscript II (Life Technologies, Rockville, Md.). First-strand synthesis in the initial reactions was primed using anchored polyT primers (Clontech, Palo Alto, Calif.). Second-strand synthesis was conducted as described (Life Technologies), and polymerase chain reaction amplification of the core region of the gene was accomplished using the primers (GGCTGGCTGACNCAYGARTTYTGYCAY; SEQ. ID NO. 5) and (CATCGTTGGAAANARRTGRTGYTCDATYTG; SEQ. ID NO. 6), designed to be completely degenerate to sequences overlapping the first and third His-box regions of the \u0394-desaturase gene.","The amplification protocol was developed using published guidelines for use of degenerate primers (Compton, 1990). The amplification consisted of 5 preliminary cycles at very low annealing temperature (30 seconds at 94\u00b0 C., 1-minute ramp to 37\u00b0 C., 45 seconds at 37\u00b0 C., 3-minute ramp to 72\u00b0 C.) followed by 30 cycles with higher temperature (30 seconds at 94\u00b0 C., 1-minute ramp to 50\u00b0 C., 45 seconds at 50\u00b0 C., 3-minute ramp to 72\u00b0 C. Preliminary amplifications to optimize thermal cycling parameters used Pfu DNA polymerase (Stratagene, La Jolla, Calif.). Amplification was successful at 3 mM magnesium and each primer at 4 \u03bcM. Subsequently Taq polymerase was used for amplification under identical conditions.","Polymerase chain reaction products from 350 to 750 bp were isolated from agarose gels with commercial reagents (Qiagen, Valencia, Calif.) and sequenced directly using the degenerate primers and dye-termination sequencing technology (Applied Biosystems, Foster City, Calif.). A group of identical amplification products contained an open reading frame that was homologous to known desaturases when analyzed by BLAST\u2122 search (Altschul et al., 25:3389-3402, 1997). The 5\u2032 and 3\u2032 sequences of the complete mRNA were obtained with the Marathon RACE system (Clontech), using pairs of nested primers designed to amplify from within the core sequence. To clone the complete 5\u2032 end of the gene, it was necessary to repeat the reverse transcription with a primer specific to the sequence of the open reading frame and repeat the 5\u2032 RACE amplification.","For the gene, RNA isolation and reverse transcription-PCR were performed as follows. RT-PCR was used to amplify the coding sequences of the two putative desaturase genes. Total RNA from mixed stage was used for the RT-PCR template. The nematodes were grown on agar plates as described and RNA was isolated using the phenol\/SDS method (Sluder et al., 184:303-319, 1997). RT-PCR was performed using the Superscript\u2122 One-Step RT-PCR system (Gibco-BRL\/Life Technologies). Approximately 1 \u03bcg of total RNA was added to a reaction mixture consisting 0.2 mM of each dNTP, 1.2 mM MgSO, Superscript II\u2122 RT\/Taq polymerase mix, and 200 \u03bcM of appropriate downstream and upstream primers. The reactions were incubated at 50\u00b0 C. for 30 min., then subject to 35 cycles of PCR amplification. For the T13F2.1 gene (fat-4) a 5\u2032 primer corresponding to bases 34339-34361 of cosmid T13F2 was used. SmaI, HindIII, and XhoI restriction sites were added to these sequences to facilitate cloning. The resulting primer (CCCGGGAAGCTTCTCGAGGAATTTCAATCCTCCTTGGGTC; SEQ. ID NO: 7) anneals to the cosmid T13F2 19-42 base pairs upstream of the putative start codon ATG of the fat-4 gene. To amplify the 3\u2032 end of the fat-4 gene, a primer was used corresponding to the inverse complement of bases 37075-37095 of cosmid T13F2, with the addition of SmaI and BamH1 sites to facilitate cloning the polynucleotide of interest: CCCGGGTGGATCCGGAACATATCACACGAAACAG; SEQ. ID NO. 8. This primer begins 93 base pairs after the putative stop codon TAG and ends 20 base pairs upstream of the predicted polyadenylation signal (AAUAAA; SEQ ID NO: 9) of the fat-4 gene.","For the determination of trans-splicing of specific leader sequences, downstream primers corresponding to the complement of bases 35009-35028 of the T13F2 (TCTGGGATCTCTGGTTCTTG; SEQ. ID NO: 10) were used for the T13F2.1 gene. The upstream PCR primers were either SL1-20 or SL2-20 (Spieth et al., 73:521-532, 1993). The homologue of the ribosomal-protein L37 was used as an SL1-specific control, and K06H7.3 was used as an SL2-specific control (Zorio et al., 372:270-272, 1994. SL1-20, SL2-20, and control primers were kindly provided by Diego A. R. Zorio. RT-PCR products visualized by gel electrophoresis were confirmed by blotting the gel and probing with gene-specific oligonucleotides corresponding to the appropriate gene as previously described (Spieth et al., 73:21-532, 1993).","DNA and protein sequences were analyzed using the Wisconsin-GCG package of programs (Devereux et al., 12:387-95, 1984).","To clone the (\u0394) open reading frame as a single DNA fragment, a set of primers was used to prime a reverse transcription specifically for the open reading frame. The primer for the 5\u2032 end of the gene began 3 nucleotides before the start codon and included the first 26 nucleotides of the open reading frame. The 3\u2032 primer was complementary to the sequence between 22 and 52 nucleotides downstream from the predicted termination codon. This PCR amplification was conducted with Pfu polymerase to minimize the chance of an amplification error. The PCR reactions produced a single band of the predicted size when analyzed by agarose gel electrophoresis. This band was cloned into the vector pCR-Script Cam\u2122 (Stratagene), and a single clone designated pJW541 was chosen for analysis.","The plasmid containing SEQ ID NO: 3, described herein as pJW541, was deposited in the American Type Culture Collection (10801 University Boulevard, Manassas, Va. 20110) on May 20, 2003 (ATCC Accession No. PTA-5206).","To express the -desaturase, the fat-4 cDNA amplification product (see Example 3) was digested with HindIII and BamH1 and ligated to the yeast expression vector pYES2 (Invitrogen) cut with HindIII and BamH1. The resulting plasmid was named pYFAT4.","For , the cloned \u0394gene was transferred to the yeast expression vector pYES2 (Invitrogen, Carlsbad, Calif.) by standard cloning techniques (Ausubel, 1988) using enzymes obtained from New England Biolabs, Beverly, Mass. The resulting yeast expression construct containing the open reading frame under the control of a galactose-inducible promoter was designated pYES2-541.",{"@attributes":{"id":"p-0136","num":"0145"},"i":["Saccharomyces cerevisiae ","Current Protocols In Molecular Biology, ","J. Biol. Chem. "]},"For the gene, the constructs were transformed into strain INVSc1 using the S.c. EasyComp transformation kit (Invitrogen). For experiments with the FAT-4 peptide, transformed yeast were grown overnight in uracil-deficient media containing 2% galactose, 0.2 mM fatty acid, and 1% NP-40. Under these conditions the percentage of these supplemented fatty acids which were incorporated into yeast lipids ranged from 14-28% of the total yeast fatty acids. For experiments in which 20:1\u0394was used as a substrate, the 1% NP-40 was replaced by 5% DMSO to achieve better incorporation of this fatty acid.","Extraction of lipids and preparation of fatty acid methyl esters was carried out by standard methods (Miquel and Browse, 267:1502-1509, 1992). Gas chromatography of the methyl esters was conducted by established methods (Spychalla et al., 94:1142-1147, 1997). Fatty acid 4,4-Dimethyloxazoline (DMOX) derivatives of yeast lipid extracts were prepared by standard methods (Fay and Richli, 541:89-98, 1991). GC-mass spectrometry was conducted on a Hewlett-Packard 6890 series GC-MS fitted with a 30 m\u00d70.25 \u03bcm HP5MS column, operating at an ionization voltage of 70 eV with a scan range of 50-550 Da. Fatty acids and their derivatives were identified where possible by comparison with authentic standards (NuCheck Prep).","Messenger RNA isolated from heterotrophic cultures was used as template for reverse transcription followed by PCR amplification using degenerate primers that spanned the first and third conserved histidine-rich regions of microsomal desaturase proteins. The -desaturase gene, FAT-3, was used as the principal basis for primer design. To compensate for the high degeneracy necessary in the primer pair, amplification reactions began with five cycles of low-temperature annealing and a long temperature ramp between the annealing and polymerization steps. Preliminary amplifications to optimize thermal cycling parameters used the proofreading Pfu DNA polymerase. After successful Pfu amplification reactions using high primer and magnesium concentrations, Taq polymerase was used to generate a number of bands detectable on agarose gels.","Several of these bands, of approximately 650 bp, had identical sequence. This DNA sequence contained an open reading frame in which the predicted amino acid sequence was homologous to other membrane desaturases, and included a characteristic central His-box. Primers designed to be specific to the amplified sequence were used to amplify the termini of the cDNA using 3\u2032 and 5\u2032 RACE techniques. The full-length cDNA for this gene was 1745 bp in length. It included an open reading frame of 1272 bp and a 472-bp 3\u2032 untranslated region. Most messenger RNAs are processed through the addition of a short 5\u2032 RNA leader sequence, the trans-spliced leader (Tessier et al., 10:2621-2625, 1991). This RNA processing step left a conserved sequence (TTTTTTTCG; SEQ. ID NO. 11) at the beginning of each message (Cui et al., . (Tokyo) 115:98-107, 1994). The presence of this leader in the cDNA sequence confirmed that the message was full-length at the 5\u2032 end. RT-PCR with primers flanking the open reading frame on the 5\u2032 and 3\u2032 ends resulted in a single band that was cloned into the vector pCR-Script Cam\u2122 (Stratagene), and designated pJW541. The gene corresponding to this ORF was designated EFD1 (fatty acid desaturase 1).","The translated open reading frame indicated a protein of 422 amino acids with a predicted molecular mass of 48.8 kDa. (). A BLAST\u2122 search of sequence databases revealed that the predicted protein sequence exhibited regions of homology with the known group of membrane fatty acid desaturases, especially in the highly conserved histidine-rich regions (Shanklin et al., 33:12787-12794, 1994).","Each of the His-box motifs is present in the EFD1 protein. The first (HXXXH; SEQ ID NO: 17) starts at amino acid 146 and the second (HXXHH; SEQ ID NO: 12) at amino acid 183 (FIG. 3). EFD1 contains a variant third His-box, QXXHH (SEQ ID NO: 13), starting at amino acid 361, similar to the cloned \u0394- and \u0394-desaturases. EFD1 exhibits conservation of protein sequence in the regions surrounding the highly conserved regions, especially with FAT-3 and FAT-4, the \u0394- and \u0394-desaturases of (FIG. 3). Outside the highly conserved regions, the amino acid sequence shows considerably less similarity to other desaturases. Overall, the amino acid identity with FAT-3 and FAT-4 is 33%, compared to 28% identity with the borage \u0394-desaturase.","EFD1 also contains a cytochrome b-like motif at its N-terminus. The protein encodes seven of the eight most highly conserved amino acids characteristic of cytochrome b(), which are responsible for heme binding. Similar motifs are found at the N-terminal regions of FAT-3 and FAT-4 (), and the borage \u0394protein, as well as the carboxyl terminus of a yeast \u0394protein.","The structure of the protein also exhibits similarities with known desaturases. Membrane desaturases are type II multiple membrane spanning proteins, and hydropathy analysis of the cloned gene indicates that the predicted protein has at least three significant hydrophobic regions long enough to span the membrane bilayer twice. As is true for most desaturase enzymes, there are 31 amino acid residues between the first two His-boxes. The distance between the between the second and third His-box is 173 residues, within the range previously observed (Shanklin and Cahoon, 48:611-641, 1998).","To confirm the activity of the enzyme, the EFD1 cDNA was transferred from pJW541 to yeast expression vector pYES2 under the control of a galactose-inducible promoter. The resulting construct, pYES2-541, was introduced into . Yeast membranes do not contain 20-carbon fatty acids but incorporate them from the culture medium. Accordingly, yeast cultures were supplemented with various fatty acid soaps, using a yeast strain containing the empty vector as control, and analyzed the fatty acids of the cultures by methyl-ester derivatization and gas chromatography.","The patterns of desaturation activity in these experiments indicated that pYES2-541 expresses a \u0394-desaturase enzyme that does not have \u0394or \u0394activity. The ability of the experimental yeast strain to produce \u0394desaturation was shown when the culture medium was supplemented with 20:2 (). A desaturation peak whose retention time is identical to authentic 20:3 was produced. The vector-only control culture did not desaturate 20:2 (). The yeast strain expressing the gene also desaturated 20:3 and 20:1 (), again without desaturation activity in the control cultures. The cloned protein was most active with 20:3 and 20:2 as substrates, desaturating 70% and 73% of the total incorporated 20-carbon fatty acid. EFD1 was least active with 20:1, converting 32% of that substrate to a desaturation product ().","When the culture medium was supplemented with a substrate for \u0394-desaturation, 20:4, the fatty acid was incorporated into the yeast, but no 20:5 desaturation product was produced. Similarly, when the medium was supplemented with 18:2 and 18:3, no desaturation occurred, demonstrating that the cloned gene did not express a \u0394-desaturase.","To confirm that desaturation had occurred at the \u0394position, 4,4-dimethyloxazoline (DMOX) derivatives of yeast fatty acids were analyzed by GC-MS. DMOX derivatives have mass spectra that are more easily interpreted than spectra of methyl esters, and permit unambiguous determination of double-bond locations in polyunsaturated fatty acids (Christie, 33:343-353, 1998). For the experiment that desaturated the fatty acid 20:2, the retention time of the product on the GC-MS instrument was 16.8 min., identical to DMOX-derivatized authentic 20:3. The mass spectrum of this desaturation product and its molecular ion (m\/z 359) indicated that it was the 20:3 compound. Two spectral frequency peaks at m\/z 182 and 194, separated by only 12 a.m.u., showed that the introduced double bond was at the \u0394position (). (The substrate 20:2, which is saturated at the 8-position, has peaks at 182 and 196, separated by 14 a.m.u.) The spectrum of the product, with its desaturation peaks, was identical to that of authentic 20:3 (Luthria and Sprecher, 28:561-564, 1993). The other substrates, 20:1 and 20:3, were also desaturated at the \u0394-position by EFD1. The peaks at m\/z 182 and m\/z 194 appeared in the spectrum of each, and the molecular ion was reduced from that of the substrate by two in each case ().","Two high-scoring open reading frames were identified during a search of the genomic DNA database with the borage \u0394-desaturase protein sequence. Both proteins predicted from these open reading frames, W08D2.4 and T13F2.1, contained an N-terminal sequence resembling cytochrome b, including the characteristic (HPGG) heme binding domain, and an H\u2192Q substitution in the third histidine box. The W08D2.4 gene was denoted fat-3 and the T13F2.1 gene was denoted fat-4, since they both appear to encode fatty acid desaturases. Interestingly, the fat-3 and fat-4 genes are located next to each other on overlapping cosmids in the same 5\u2032 to 3\u2032 orientation, with only 858 nucleotide base pairs separating the putative polyadenylation signal of the fat-4 gene and the ATG start codon of the fat-3 gene (). This gene organization is reminiscent of operons, in which two or more genes are transcribed under the control of a single promoter and regulatory region.","In the polycistronic pre-mRNA is converted to monocistronic mRNA by cleavage and polyadenylation at the 3\u2032 end of the upstream gene and transplicing to the SL2 sequence at the 5\u2032 end of the downstream gene, with the two mRNAs being subsequently independently translated. However, out of more than 30 such operons that have been analyzed, the distances between the 3\u2032 end of the upstream gene and the 5\u2032 end of the downstream gene are generally about 100 base pairs, with a few separated by 300-400 base pairs (Blumenthal et al, II, pp. 117-145, Cold Spring Harbor Laboratory Press, Cold Spring, N.Y., 1997).","The fat-3 and fat-4 genes were tested to determine whether they were trans-spliced to either SL1 or SL2 in order to determine if they might be co-transcribed in a single operon. It was found that the fat-4 gene was transpliced to SL1, but that the fat-3 gene was not transpliced to either spliced leader sequence. Therefore, it was concluded that each gene contains its own 5\u2032 promoter and regulatory region.","Both of these genes were cloned using RT-PCR. The fat-4 gene sequence matched the T13F2 genomic sequence exactly. However the gene product encoded by the cDNA was seven amino acids shorter than predicted by Genefinder for T13F2.1 (GenBank accession number Z81122) because the DNA sequence encoding amino acid residues 198-204 was not present in the fat-4 cDNA. The resulting peptide length was 447 amino acids instead of the previously predicted 454 amino acids. The gene product encoded by the fat-3 cDNA also matched the genomic sequence of W08D2.4 (GenBank accession number Z70271) perfectly. However, the gene product was also shorter than the predicted protein sequence. Codons for amino acid residues 38-67 of W08D2.4 were not present in the cDNA. In both cases the gene-prediction software used in the genomic sequencing project appeared to have misidentified some intron DNA as coding sequences.","The FAT-3 and FAT-4 proteins, the -desaturase, and the -desaturase appear to be proteins of similar structure in that they all contain an N-terminal cytochrome bdomain, three histidine boxes, and distinct hydrophobic membrane-spanning domains predicted by the TMHMM program from the Center for Biological Sequence Analysis, Technical University of Denmark (http:\/\/www.cbs.dtu.dk\/services\/TMHMM-1.0\/). The predicted structure is consistent with the proposed desaturase structural model (Stukey et al., 265:20144-20149, 1990). Despite these similarities, the overall sequence identity among the four proteins is quite low. For example, the FAT-3 \u0394-desaturase and the borage \u0394-desaturase share only 28% identity on the amino acid level. The fat-4 gene product shares 25% amino acid identity with the borage \u0394-desaturase and 19% amino acid identity with the -desaturase. Indeed the only portion of the FAT-4 protein that shows extended homology to the -desaturase is a sequence of 36 residues incorporating the third His box which has 44% identity and 56% similarity. The most closely related pair of sequences are fat-3 and fat-4, which are 46% identical on the amino acid level and 54% identical over the entire cDNA sequence.",{"@attributes":{"id":"p-0154","num":"0163"},"figref":"FIG. 9","sup":["6","5","6","5"],"i":["C. elegans ","C. elegans ","Mortierella alpina \u0394"]},"To determine the enzymatic activity of the FAT-4 desaturase-like protein, the protein was expressed in supplemented with polyunsaturated fatty acid substrates that are not normally present in this yeast. The FAT-4 protein was expressed in the yeast expression vector pYES2 from the GAL1 promoter by growing the cells in the presence of galactose and various fatty acids. After 16 hours of growth, the cells were analyzed for total fatty acid composition by gas chromatography (GC). Comparison of cells supplemented with di-homo-\u03b3-linolenic acid (20:3\u0394) carrying pYES2 containing the fat-4 coding sequence and cells carrying the vector alone revealed the presence of a major new peak eluting at 14.49 minutes in the cells expressing FAT-4 (). The novel peak had a retention time identical to that of the authentic arachidonic acid methyl ester (20:4\u0394), and was determined to be arachidonic acid (20:4\u0394) because its mass spectrum was identical to that of authentic arachidonic acid methyl ester, including a mass ion peak at m\/z 318.","The identity of this compound was further verified by converting the yeast fatty acid methyl esters into oxazoline derivatives in order to produce structure specific mass spectra which simplify the determination of double-bond positions in a hydrocarbon chain. The mass spectrum of the DMOX derivative of the novel 20:4 component was consistent with the published spectrum for arachadonic acid and contained a prominent peak at m\/z 153, which is diagnostic of a double bond at the \u0394position. Therefore, it was concluded that the fat-4 gene encodes a \u0394-desaturase capable of synthesizing arachidonic acid from the substrate di-homo-\u03b3-linolenic acid. In contrast, the FAT-4 protein showed no activity when linoleic acid (18:2\u0394) or \u03b3-linolenic acid (18:3\u0394) were provided as substrates, indicating an absence of \u0394-desaturase activity.","Further analysis of the GC trace of the total fatty acids of the yeast cells expressing fat-4 revealed the presence of a second novel peak eluting at 12.91 minutes which was not present in the empty vector control cells. Analysis of the mass spectrum of this novel peak revealed a molecular ion species of 294, identical to that of a methyl ester of an 18-carbon fatty acid with two double bonds (18:2), but its retention time and mass spectrum were not identical to the common isomer 18:2\u0394.","In microsomal extracts of mammalian liver, \u0394-desaturase activity has been reported to act on a number of 18 and 20-carbon precursors to produce uncommon fatty acids such as 18:2\u0394, 20:3\u0394and 20:4\u0394(28, 29). Two species of slime molds have also been reported to produce small amounts of 18:2\u0394, 18:2\u0394, 20:3\u0394and 20:4\u0394(Rezanka, 33:1441-1444, 1993). These fatty acids are unusual in that their double bonds do not follow the conventional methylene-interrupted pattern (one double bond every three carbons).","Therefore, it was suspected that the novel peak exhibited on the GC spectrum is a result of the -desaturase acting on 18:1\u0394(or 18:1\u0394, which in constitutes 15-20% of the total 18:1) compound, to produce the uncommon isomer 18:2\u0394or 18:2\u0394. These yeast fatty acid methyl esters were converted into oxazoline derivatives. It was found that the mass spectrum of the DMOX derivative of the novel 18:2 component contained the \u0394-specific peak at m\/z 153. However the larger ion peaks characteristic of double bonds at the \u0394or \u0394position were not detected due to the small amount of this molecule present in the total yeast extracts.","To test if the -desaturase was capable of desaturating other substrates to produce other uncommon, non-methylene-interrupted fatty acids, the yeast expressing the FAT-4-desaturase was supplemented with unconventional \u0394-substrates such as 20:1\u0394, 20:2\u0394, and 20:3\u0394. No novel peaks were detected when the substrate 20:1\u0394was fed to yeast. However, when 20:2\u0394and 20:3\u0394were provided as substrates, novel peaks were detected eluting at 14.62 minutes and 14.69 minutes, respectively (, respectively). The mass-spectrum analysis of DMOX derivatives of these molecules yielded results consistent with published values for 20:3\u0394and 20:4\u0394, including a prominent ion peak of m\/z 153 (which is diagnostic of double bonds at the \u0394position). It was found, however, that these fatty acids were not produced to the same extent as arachidonic acid (20:4\u0394) (). In these experiments, 55% of exogenously fed di-homo-\u03b3-linolenic acid (20:3\u0394) was converted to arachidonic acid, while only 5%, 27%, and 26% of the 18:1, 20:3\u0394, and 20:2\u0394substrates were converted ().","The fat-3 gene was expressed in the yeast expression vector pMK195 containing the constitutive ADH promoter. The FAT-3 protein was able to desaturate linoleic acid (18:2\u0394) into \u03b3-linolenic acid (18:3\u0394), in agreement with published results (Napier et al., 330:611-614, 1998). It was also found that FAT-3 was capable of desaturating \u03b1-linolenic acid (18:3\u0394) to 18:4\u0394, a common reaction in animals. The FAT-3 protein showed no activity on 20:1\u0394, 20:2\u0394, 20:3\u0394, or 20:3\u0394. Therefore, the substrate specificities of the and \u0394-desaturases were determined to be specific and non-overlapping.","Desaturation at the \u0394-position has not been reported for any previously cloned gene (Tocher et al., 37:73-117, 1998).","The predicted EFD1 protein has 33% amino acid identity with both FAT-3 and FAT-4, (), while its identity with the cloned borage \u0394-desaturase is 28%. The highest sequence conservation is found in the His-box motifs which are critical for desaturase activity, most likely because they serve as the diiron-oxo component of the active site (Shanklin et al., 33:12787-12794, 1994). Sequence conservation is also evident in the N-terminal cytochrome b-like domain, where most of the essential residues of cytochrome b(Lederer, 76:674-692, 1994) that are retained in FAT-3 and FAT-4 are also present in the EFD1 protein ().","Expression of the EFD1 gene in yeast was used to characterize its activity. Three different 20-carbon substrates with double bonds at the \u0394-position were desaturated (), and analysis of the products indicated that, for each one, desaturation had taken place at the \u0394position (). The cloned desaturase showed a clear preference for the substrates of metabolic significance with greater than two-fold preference for 20:2 and 20:3 over 20:1 (Ulsamer et al., 43:105-114, 1969). Even though EFD1 is quite similar to other microsomal desaturases, its activity was specific, as evidenced by its inactivity on substrates for \u0394and \u0394desaturation ().","The 20-carbon substrates for \u0394desaturation are available in abundance in heterotrophically grown (). These same substrates also are available in mammals, since 20:2 and 20:3 are produced by elongation from 18:2 and 18:3, in competition with the typical \u0394desaturation (). Labeling experiments with rat liver homogenates indicate that elongation of the 18-carbon fatty acids is five-fold more rapid than the competing desaturation (Pawlosky et al., 33:1711-1717, 1992).","Implicit in the current understanding of the \u0394pathway of 20-carbon polyunsaturated fatty acid biosynthesis is a reliance on alternating desaturation and elongation to control flux through the pathway. While elongation often appears to be non-specific, most desaturations are specific both as to chain length of the substrate and to existing desaturation pattern of the fatty acid (Heinz, , pp. 33-89, 1993). However, data from experiments in mammalian tissue (Bernert and Sprecher, 398:354-363, 1975; Albert et al., 14:498-500, 1979) and with yeast expressing the -desaturase gene (), indicate that \u0394enzymes desaturate fatty acids having a double bond at the \u0394-position but not at the \u0394-position, producing the non-methylene-interrupted 20:3 and 20:4 compounds at significant rates. For the \u0394pathway, \u0394desaturation occurs in competition with \u0394activity on the substrates 20:2 and 20:3. In spite of this promiscuity of \u0394enzymes, lipid profiles of mammalian tissue do not contain fatty acids with the \u0394(Heinz, , pp. 33-89, 1993; and Ulsamer et al., 43:105-114, 1969) desaturation pattern, nor are they seen in ","One explanation may be that \u0394-desaturation of the common substrates occurs very rapidly, while \u0394-desaturation proceeds more slowly, so that little \u0394product is formed. In support of this explanation, the appears to be a very active desaturase when expressed in yeast (), compared to similarly expressed \u0394-desaturase enzymes. The desaturase must be sufficiently active to account for all the long chain polyunsaturates of rapidly growing cultures (). In contrast, the observed rates of \u0394-desaturation in mammalian tissue are relatively slow. The highest apparent rate occurs in cancerous tissues without \u0394activity, where \u0394desaturation permits production of arachidonic acid at only 17% of the level of comparable normal cells with \u0394activity (Grammatikos et al., 70:219-227, 1994).","Alternatively, the lack of \u0394unsaturated fatty acids in membranes could be explained if the \u0394desaturase accepts these fatty acids for desaturation. The convention that \u0394-desaturation precedes \u0394activity () is based on observations of desaturation reactions proceeding sequentially along the fatty acid hydrocarbon chain. The reverse order of desaturation, with \u0394-saturation preceding \u0394-desaturation, has been claimed (Takagi, 38:2055-2057, 1965) and rejected (Schlenk et al., 5:575-577, 1970) in mammalian liver, and proposed as a likely pathway based on experiments with deuterated substrates in glioma cells (Cook et al., 32:1265-1273, 1991).","In the products of \u0394-desaturation, 20:3\u0394and 20:4\u0394, may be incorporated directly into membranes, or subjected to desaturation at the \u0394-position to produce arachidonic and eicosapentaenoic acids (Hulanicka et al., 239:2778-2787, 1964). Further elongation and desaturation leads to several polyunsaturated 22-carbon fatty acids (). In mammals, whether the products are derived from \u0394or \u0394activity, similar processes produce mostly arachidonic, eicosapentaenoic, and docosahexaenoic acid (22:6) (Hwang, , pp. 545-557, 1992; Bernert and Sprecher, 398:354-363, 1975; Lees and Korn, 5:1475-1481, 1966; Albert and Coniglio, 489:390-396, 1977; Bardon et al., 99:51-58, 1996; and Sprecher and Lee, 388:113-125, 1975), although some 20:3 is metabolized directly to series 1 eicosanoid metabolic regulators (Hwang, , pp. 545-557, 1992).","It is interesting to note that the alternate pathway of \u0394-desaturation begins with an elongation step. This elongation is the standard pathway in , which produces substantial amounts of 20:2 (7.4%) and 20:3 (1.4%) (). In mammalian tissue with little or no \u0394activity (Grammatikos et al., 70:219-227, 1994), this would be the first step by which the essential fatty acids 18:2 and 18:3 are metabolized to their 20-carbon derivatives. Recently there has been a new emphasis on fatty acid chain elongation acting as a regulatory step in fatty acid biosynthesis (Garcia et al., 25:211-215, 1990; Sprecher et al., 52:99-101, 1995). Evidence that breast cancer cells may selectively elongate 18:3 in preference to 18:2, and that \u0394-desaturation follows this elongation (Bardon et al., 99:51-58, 1996) implies that \u0394-desaturation may play an important role in some cancer cells.","The identification and cloning of a \u0394-desaturase gene permit examinations of the alternate pathway for biosynthesis of 20-carbon polyunsaturated fatty acids and will give insight into the possible mechanisms of \u0394-desaturation. In mammals, operation of this alternative pathway may be confined to specialized tissues, where the demand for polyunsaturated fatty exceeds the supply provided through rate-limiting \u0394desaturation. The pathway may be of greater significance where \u0394desaturation is reduced or absent. Since fatty acid desaturase metabolism is perturbed in many cell lines, both transformed (Grammatikos et al., 745:92-105, 1994) and untransformed (Rosenthal, 26:87-124, 1987), it may be that \u0394activity is only revealed in the absence of \u0394activity. Alternatively, \u0394activity may arise or increase with cell neoplasia. The isolation and examination of this \u0394gene, and analysis of its substrate specificity, should facilitate the determination of the role of \u0394activity in both normal and cancerous mammalian tissue.","In this example, we describe a region of the genome located at position 4.88 of chromosome IV is described that contains the \u0394- and \u0394- desaturase genes. The amino acid sequences encoded by the two genes are 46% identical to each other, and each contains an N-terminal heme binding domain typical of the electron carrier cytochrome band three histidine boxes. Both genes contain the consensus sequence of the third His box (QXXHH; SEQ. ID NO. 13) that has so far been shown to be unique to the microsomal desaturases involved in double-bond insertion at carbons below position 9.","Despite these similarities, these two microsomal desaturases show absolutely non-overlapping substrate specificities. When overexpressed in the yeast , the -desaturase (FAT-3) specifically acts on two 18-carbon substrates, linoleic and \u03b3-linolenic acid, and always desaturates in a methylene-interrupted pattern (one double bond every three carbons). The mammalian \u0394-desaturase system has likewise been demonstrated to insert double bonds strictly in a methylene-interrupted pattern and to have no activity on 20-carbon substrates (Schmitz et al., 12:307-313, 1997). The -desaturase (FAT-4), in contrast, acts on a number of 20-carbon substrates, as well as on an endogenous 18:1 fatty acid of yeast, and is capable of inserting double bonds in a non-methylene interrupted pattern.","Non-methylene-interrupted fatty acids such as 20:2\u0394, 20:3\u0394, 18:2\u0394have been detected in mammalian cells by feeding C-labeled substrates to rats raised on a fat-deficient diet (Ulman et al., 248:186-197, 1971). However, these fatty acids are considered to be \u201cdead end\u201d metabolites, as they have not been demonstrated to serve as precursors to signaling molecules such as prostaglandins, nor are they detectable in tissue lipids of rats who are not preconditioned on a fat-deficient diet. (We also did not detect these fatty acids in lipid extracts.)","In yeast expressing the -desaturase gene, the amount of substrate converted was greatest for the metabolically significant substrate 20:3\u0394(). The amount of 20:2\u0394and 20:3\u0394that was desaturated less than half the amount of conventional substrate that was desaturated. This was consistent with the rates of desaturation in microsomal extracts of mammalian liver, where the rate of conversion of labeled 20:2\u0394to 20:3\u0394is 41% of the rate of conversion of labeled 20:3\u0394to 20:4\u0394(Bernet et al., 398:354-313, 1975).","The fat-3 and fat-4 genes are present in a gene cluster in the same 5\u2032 to 3\u2032 orientation. Yet, unlike other gene clusters of this sort in , the downstream fat-3 gene is not transpliced to SL2, and therefore is unlikely to be co-transcribed with the upstream fat-4 gene. The two genes could be located next to each other as a result of an ancient gene-duplication event. The DNA sequences share 54% identity over the entire cDNA coding sequence; however the genes do not share any common intron\/exon boundaries ().","This is the first disclosed sequence of a \u0394-desaturase gene from an animal. The sequence of the -desaturase is quite distant from the bacterial and fungal \u0394-desaturases that have been reported, and this animal sequence should facilitate the search for desaturase-encoding sequences from humans and other mammals. Both the \u0394- and \u0394-desaturases are important regulatory enzymes in humans. They participate in critical steps in the pathway to produce precursors for synthesis of hormone-like eicosanoid molecules from the essential dietary fatty acids, linoleic acid and \u03b1-linolenic acid. The activities of these desaturases have been shown to be under hormonal and nutritional control, but the mechanism of this control is still unknown.","Certain diseases, such as diabetes, result in low \u0394-desaturase activity, while HTC cells, isolated from an ascites tumor derived from a solid hepatoma, show increased \u0394-desaturase activity. The availability of mutational and reverse genetic tools and the expanding knowledge of cellular and developmental biology in make this an attractive system to study the roles of polyunsaturated fatty acids and their metabolic products in development, reproduction, and other cellular processes of animals.","Using the methods described herein, \u0394- and \u0394-desaturases of the invention may be cloned and expressed in plants to produce plants with enhanced amounts of 20-carbon polyunsaturated fatty acids. Such plants provide an inexpensive and convenient source of these important fatty acids in a readily harvestable and edible form.","For instance, the \u0394- and \u0394-desaturases of the invention can be cloned into a common food crop, such as corn, wheat, potato, tomato, yams, apples, pears, or into oil-seed plants such as sunflower, rapeseed, soy, or peanut plants. The resulting plant would express the appropriate enzyme that would catalyze the formation of 20-carbon polyunsaturated fatty acids. In the case of an oil-seed plant, the seed oil would be a rich source of 20-carbon polyunsaturated fatty acids.","The \u0394- and \u0394-desaturase genes may be cloned and expressed either individually, or together in a host plant cell. The corresponding desaturases can be expressed using a variety of different control sequences, such as promoters, enhancers, and 3\u2032-termination sequences. These control sequences can be used to control the expression of each desaturase individually. For example, the \u0394-desaturase can be cloned such that it is under the control of a strong promoter, and the \u0394-desaturase can be cloned such that it is under the control of a weak promoter, thus yielding a transgenic plant that expresses more \u0394-desaturase than \u0394-desaturase. Furthermore, expression can be controlled by operably linking one or more of the desaturase genes of interest to a promoter that is activated by exposure of the plant cell to an appropriate regulatory agent such as an inducer, repressor, de-repressor or inhibitor agent. Such regulation is discussed above. Alternatively, expression of non-contiguous genes may be coordinated by linking the expression of a first gene with the expression of an inducer or de-repressor molecule that induces or de-represses the expression of a second gene.","The genes of the invention can be integrated into the genome of a plant (for example, by -mediated T-DNA transfer) or animal (for example, by use of broad host-range retroviruses, e.g., an adenovirus vector) so that the \u0394and \u0394-desaturases of the invention are expressed as part of the genome. For transgenic plants, the T-DNA vector may be used that would result in integration of transgenes (\u0394and\/or \u0394) into the host cell genome.","For example, the expression of the \u0394- and \u0394-desaturases in a plant, such as , can be achieved by constructing a plant transformation vector to introduce the cDNA of each desaturase into the plant. The vector can contain a tissue specific promoter so that the desaturase protein will be expressed during seed development. Examples of seed specific promoters include that for phaseolin (van der Geest and Hall, 32:579-88, 1996) or the promoter for napin (Stalberg et al., 23: 671-83, 1993). Other seed-specific promoters that can be used are those located on the genomic BAC clone T24A18 (LOCUS ATT24A18. (1999) 45980 bp DNA chromosome 4, ACCESSION # AL035680, NID g4490701) of the genome. These promoters regulate seed storage protein expression in . Other promoters which express genes specifically in seeds like those described in (Parcy et al., 6:1567-1582, 1994) can also be used. The constructs containing the desaturase coding sequence and promoter sequence can then transferred to standard plant transformation T-DNA vectors similar to pART27 (Gleave, 20:1203-1207, 1992), pGPTV (Becker et al., 20:1195-7, 1992), or pJIT119 (Guerineau et al., 15:127-136, 1992). If the plant is to be transformed with two constructs, i.e. one encoding the \u0394-desaturase and the other encoding the \u0394-desaturase, then it is preferable to choose two different selectable markers so that only double transformants will regenerate. For example, the vector carrying the \u0394-desaturase can be constructed such that it contains the kanmycin (nptII) gene, and the vector carrying the \u0394-desaturase can be constructed such that it contains the phosphinothricin (bar) gene. Transformants are then selected on media containing kanmycin and phosphinothricin. Transformation of is readily achieved using the Agrobacterium-mediated vacuum infiltration process (Katavic et al., 245:363-70, 1994) or the floral dip modification of it (Clough and Bent, 16:735-43, 1998), although several other methods are also commonly used. Transgenic progeny will be identified by selection using the appropriate antibiotic or herbicide, either kanmycin or phosphinothricin, or both. Since the \u0394and \u0394constructs use different selectable markers the double transformants are readily isolated. Plants which survive the transgenic selection are grown to maturity and their seed harvested. The seeds of transformed plants are analyzed by isolation of fatty acid methyl esters followed by gas chromatography to determine the fatty acid composition.","Plants expressing only the \u0394-desaturase will desaturate the 20:1\u0394fatty acid that occurs naturally in the seed to 20:2\u0394. Seed harvested from plants doubly transformed with both desaturases will, in addition, convert the 20:2\u0394product of the \u0394-desaturase plants to 20:3\u0394as a result of the expression of the \u0394-desaturase. These changes will be easily detected by the fatty acid methyl ester analysis.","The cDNA portion of pJW541 (Wallis and Browse, 365:307-316, 1999) containing the -desaturase was excised from that plasmid with the restriction enzymes EcoRI and SpeI. The purified DNA fragment representing the insert was ligated into the yeast expression vector pYX232 (R&D Systems, Inc.) that had been prepared by digestion with EcoRI and NheI, to give compatible sticky ends. (Plasmid pYX232 carries the marker conferring yeast prototrophy for tryptophan (TRP1 mutation), and uses the triose phosphate isomerase (TPI) promoter for constitutive expression of the inserted DNA.) The resulting plasmid, pYX232-541, was introduced into the strain already harboring the \u0394-desaturase (pYFAT4; Watts and Browse, 362:175-182, 1999) plasmid that confers yeast prototrophy for uracil using a lithium acetate transformation procedure (Invitrogen). Transformants were selected simultaneously for uracil and tryptophan prototrophy. Selected colonies arising after the transformation were inoculated into yeast minimal medium that also lacked both uracil and tryptophan.","For analysis of activity, separate cultures were supplemented with one of three fatty acid substrates provided as sodium salts as described (Wallis and Browse, 365:307-316, 1999). After overnight culture at 28\u00b0 C., the cultures were harvested by centrifugation and washed. Fatty acid methyl esters were prepared using the standard methods described in Miquel and Browse 267:1502-1509, 1992.","Analysis by gas chromatography indicated that each substrate had been desaturated twice. The incorporation of the three substrates varied, with more unsaturated substrates becoming a greater part of the fatty acid composition of the cells, as seen in other experiments (Wallis and Browse, 365:307-316, 1999, and Watts and Browse, 362:175-182, 1999. For the tri-unsaturated substrate 20:3\u0394, the 20-carbon fatty acid represented 37% of the total cellular fatty acid, for 20:2\u0394the 20-carbon fatty acid level 21%, and for 20:1\u0394the 20-carbon fatty acid level reached only 13%. However, the activities of the desaturases were substantially identical against all three substrates. Between 70 and 72% of the substrate was not converted, and 17 or 18% underwent a single desaturation by only one of the enzymes. However, for each substrate, between 11 and 13% of the substrate was desaturated by both enzymes acting in concert to produce a fatty acid with two double bonds more than in molecules of the supplied substrate.",{"@attributes":{"id":"p-0188","num":"0197"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"70pt","align":"center"}}],"thead":{"row":[{"entry":"TABLE 2"},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":[{},"Fatty","Unconverted","One added",{}]},{"entry":["Fatty acid","acid","substrate","desatura-","Doubly desaturated"]},{"entry":["supplement","uptake*","#","tion #","product #"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["20:3","37","71","17","11","20:5"]},{"entry":["(11, 14, 17)",{},{},{},{},"(5, 8, 11, 14, 17)"]},{"entry":["20:2","21","70","18","13","20:4"]},{"entry":["(11, 14)",{},{},{},{},"(5, 8, 11, 14)"]},{"entry":["20:1","13","72","18","11","20:3"]},{"entry":["(11)",{},{},{},{},"(5, 8, 11)"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}},{"entry":"*as mass percent of whole cell fatty acids"},{"entry":"# as mass percent of incorporated 20-carbon fatty acids"}]}}]}}},"The foregoing embodiments and examples are provided only as examples and are in no way meant to limit the scope of the claimed invention.","It should be apparent to one skilled in the art that the invention described herein can be modified in arrangement and detail without departing from the scope or spirit of the invention. We claim all such modifications.","The references and publications referred to herein are hereby incorporated by reference."],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0019","num":"0018"},"figref":"FIG. 1A","sup":"6 "},{"@attributes":{"id":"p-0020","num":"0019"},"figref":"FIG. 1B","sup":["8 ","5 "]},{"@attributes":{"id":"p-0021","num":"0020"},"figref":"FIG. 1C","sup":["5","6","8"]},{"@attributes":{"id":"p-0022","num":"0021"},"figref":"FIG. 2","i":"E. gracilis "},{"@attributes":{"id":"p-0023","num":"0022"},"figref":"FIG. 3","i":["Euglena \u0394","C. elegans","C. elegans \u0394","Biochem. J. ","Biochimie ","Biochem. J. ","J. Biol. Chem. ","Annu. Rev. Plant Physiol. Plant Mol. Biol. "],"sup":["8","6 ","5 "],"sub":"5"},{"@attributes":{"id":"p-0024","num":"0023"},"figref":"FIG. 4","i":"Euglena "},"\u0394-desaturase (EFD1) gene, were supplemented with the indicated 20-carbon fatty acids. The control strain does not desaturate the exogenous fatty acids. For the experimental strain, an arrow indicates the desaturation peak.",{"@attributes":{"id":"p-0025","num":"0024"},"figref":"FIG. 5","sup":"8 "},{"@attributes":{"id":"p-0026","num":"0025"},"figref":"FIG. 6A","sup":"5","i":"Caenorhabditis elegans "},{"@attributes":{"id":"p-0027","num":"0026"},"figref":"FIG. 6B","sup":"5","i":"Caenorhabditis elegans "},{"@attributes":{"id":"p-0028","num":"0027"},"figref":"FIG. 7A","sup":"8","i":"Euglena gracilis "},{"@attributes":{"id":"p-0029","num":"0028"},"figref":"FIG. 7B","sup":"8","i":"Euglena gracilis "},{"@attributes":{"id":"p-0030","num":"0029"},"figref":"FIG. 8","i":"C. elegans ","sup":["5","6"],"sub":["5 ","5"]},{"@attributes":{"id":"p-0031","num":"0030"},"figref":"FIG. 9","sup":["6","5","6","6 ","5 "],"i":["C. elegans ","C. elegans ","Mortierella alpina \u0394","Borago officinalis \u0394","C. elegans ","C. elegans \u0394","Mortierella alpina \u0394"]},{"@attributes":{"id":"p-0032","num":"0031"},"figref":"FIGS. 10A-10C","i":"S. cerevisiae "},{"@attributes":{"id":"p-0033","num":"0032"},"figref":"FIGS. 11A and 11B","sup":["8 ","5","11,14 ","5,11,14","5","11,14,17 ","5,11,14,17"]},{"@attributes":{"id":"p-0034","num":"0033"},"figref":"FIG. 12","i":"C. elegans \u0394","sup":["5","6"]},{"@attributes":{"id":"p-0035","num":"0034"},"figref":"FIG. 13","i":["E. gracilis \u0394","S. cervisiae "],"sup":"8"}]},"DETDESC":[{},{}]}
