---
title: Use of spiegelmers to inhibit an intracellular target molecule
abstract: The present invention relates to the use of a L-nucleic acid as intracellularly active agent.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08497250&OS=08497250&RS=08497250
owner: NOXXON Pharma AG
number: 08497250
owner_city: Berlin
owner_country: US
publication_date: 20060504
---

{"@attributes":{"id":"description"},"BRFSUM":[{},{}],"p":["One aspect of the present invention relates to a new use of spiegelmers. Another aspect of the present invention relates to spiegelmers that bind HMG proteins.","With advances in molecular medicine it has become possible to identify target molecules involved in a disease or a disease state and to act on these specifically so as thereby to treat or prevent the disease or the disease state or at least to alleviate the symptoms associated therewith. The target molecules can in principle be divided into two groups. A first group includes target molecules that are present extracellularly and can thus in principle be brought into contact with an active substance by administering the latter in a body fluid or a body cavity that contains the target molecule. The first group of target molecules is herein also referred to as extracellular target molecules. The second group of target molecules includes target molecules that are present in cells, these cells being involved in the disease to be treated or in the predisposition to the disease. It is not necessary in this connection for the target molecule to be directly responsible for the disease state or directly connected with the predisposition to the disease. Instead, it is sufficient if the respective target molecule is involved in an action cascade, the course of which is influenced by the active substance, with the result that the active substance is suitable for the treatment or prevention of the disease. The second group of target molecules is herein also referred to as intracellular target molecules.","The nature of the target molecule, i.e. extracellular or intracellular target molecule, determines in principle the binding class, with which an attempt can be made to effect the interaction, necessary for the therapeutic or preventive action, between the active substance, typically the pharmaceutical active substance, and the target molecule. In virtually all cases so-called small molecules can be used, i.e. chemical compounds with a molecular weight of typically 1000 daltons or less. These molecules can interact in the desired manner directly with extracellular target molecules, as well as with intracellular target molecules.","Against this background new classes of active substances have been developed by the biotechnology industry, such as for example antibodies, in particular monoclonal antibodies, antisense molecules, siRNA molecules, aptamers and spiegelmers. Although some of these classes of molecules are still in the preliminary stage of clinical investigations, there exist at least in the case of antibodies and antisense molecules products that are already in clinical use. However, with these new classes of substances there are also significant problems as regards addressing intracellular target molecules. Thus, for example, the intracellular use of antibodies is currently still not always possible, at least not to an extent or in a way and manner that allows a routine use in patients of antibodies directed against intracellular target molecules for the purposes of treatment and\/or prophylaxis. Also, the other new classes of active substances, in particular antisense molecules and siRNA molecules, must on account of their action mechanism be introduced into the respective cell containing the target molecule or the gene coding for the target molecule. The targeted release of the active substance, also termed delivery, is also for these classes of substances the currently limiting factor for a clinical application.","The same is also true of aptamers and spiegelmers, i.e. functional nucleic acids with a defined three-dimensional structure that allows the specific interaction with the respective target molecules. The use of aptamers in order to address intracellular target molecules utilises methods of gene technology, more specifically gene therapy. The aptamers, also termed intramers, directed against an intracellular target molecule are incorporated into the respective target cell by means of gene technology methods. Such an approach is however also subject to considerable limitations, not least on account of the lack of acceptance of treatment approaches based on gene therapy. In particular the route adopted in the case of intramers, involving intracellular expression of a nucleic acid coding intracellularly for the respective aptamer, is in principle closed to spiegelmers, since no biological system exists which would be capable of synthesising spiegelmers, i.e. aptamers consisting of L-nucleotides.","The object of the present invention is accordingly to provide a class of substances that is able to interact specifically with intracellular target molecules, i.e. target molecules that are present in a cell.","According to the invention this object is achieved by the subject-matter of the accompanying independent claims. Preferred embodiments are disclosed in the sub-claims.","According to the present invention the basic object is achieved by the subject-matter of the independent claims. Preferred embodiments are disclosed in the sub-claims.","According to a first aspect of the invention the object is achieved by the use of a L-nucleic acid as intracellular active agent.","In a first embodiment of the first aspect the L-nucleic acid is a spiegelmer.","In a second embodiment of the first aspect, which is also an embodiment of the first embodiment, the L-nucleic acid interacts with an intracellular receptor.","In a third embodiment of the first aspect, which is also an embodiment of the second embodiment, the intracellular receptor is selected from the group comprising molecular receptors, enzymes, chaperone molecules, signal peptides, intracellular structures and metabolic intermediates.","In a fourth embodiment of the first aspect, which is also an embodiment of the second embodiment, the intracellular receptor is selected from the group comprising polypeptides, carbohydrates, nucleic acids, lipids and combinations thereof.","In a fifth embodiment of the first aspect, which is also an embodiment of the second, third and fourth embodiment, the L-nucleic acid interacts with an intracellular receptor within a cell.","In a sixth embodiment of the first aspect, which is also an embodiment of the second, third, fourth and fifth embodiment, the intracellular receptor is selected from the group comprising transcription factors and DNA-binding proteins binding an AT hook.","In a seventh embodiment of the first aspect, which is also an embodiment of the sixth embodiment, the intracellular receptor is selected from the group comprising HMG proteins, preferably from the group comprising HMGA1, HMGA1a, HMGA1b, and HMGA2.","According to a second aspect of the present invention this object is achieved by a method for binding an intracellular receptor, comprising:\n\n","In a first embodiment of the second aspect the incubation takes place under conditions so that the L-nucleic acid binds to the intracellular receptor in the cell.","In a second embodiment of the second aspect, which is also an embodiment of the first embodiment, the L-nucleic acid is a spiegelmer.","In a third embodiment of the second aspect, which is also an embodiment of the first and second embodiment, after the incubation of the cell with the L-nucleic acid it is determined whether a binding, in particular an intracellular binding, of the L-nucleic acid to the intracellular receptor has taken place.","In a fourth embodiment of the second aspect, which is also an embodiment of the first, second and third embodiment, the intracellular receptor is selected from the group comprising molecular receptors, metabolic intermediates and enzymes.","In a fifth embodiment of the second aspect, which is also an embodiment of the first, second, third and fourth embodiment, the intracellular receptor is selected from the group comprising polypeptides, carbohydrates, nucleic acids, lipids and combinations thereof.","In a sixth embodiment of the second aspect, which is also an embodiment of the first, second, third, fourth and fifth embodiment, the intracellular receptor is selected from the group comprising transcription factors and DNA-binding proteins binding an AT hook.","In a seventh embodiment of the second aspect, which is also an embodiment of the sixth embodiment, the intracellular receptor is selected from the group comprising HMG proteins, and is preferably selected from the group comprising HMGA1, HMGA1a, HMGA1b and HMGA2.","According to a third aspect of the invention this object is achieved by use of a L-nucleic acid to manufacture a medicament for the treatment and\/or prevention of a disease, the target molecule of the medicament being an intracellular target molecule.","In a first embodiment of the third aspect the intracellular receptor is selected from the group comprising molecular receptors, enzymes, chaperone molecules, signal peptides, intracellular structures and metabolic intermediates.","In a second embodiment of the third aspect, which is also an embodiment of the first embodiment, the intracellular receptor is selected from the group comprising polypeptides, carbohydrates, nucleic acids, lipids and combinations thereof.","In a third embodiment of the third aspect, which is also an embodiment of the first and second embodiment, the target molecule is selected from the group comprising transcription factors and DNA-binding proteins binding an AT hook.","In a fourth embodiment of the third aspect, which is also an embodiment of the third embodiment, the target molecule is selected from the group comprising HMG proteins, and is preferably selected from the group comprising HMGA1, HMGA1a, HMGA1b and HMGA2.","In a fifth embodiment of the third aspect, which is also an embodiment of the third and fourth embodiment, the disease is selected from the group comprising tumour diseases, virus infections and arteriosclerosis.","In a sixth embodiment of the third aspect, which is also an embodiment of the fifth embodiment, the tumour disease is selected from the group comprising mesenchymal tumours, epithelial tumours, benign tumours, malignant tumours and metastasising tumours.","In a seventh embodiment of the third aspect, which is also an embodiment of the third, fourth, fifth and sixth embodiment, the target molecule is HMGA and the diseases are selected from the group comprising carcinomas of the prostate, pancreas, thyroid, cervix, stomach, breast, colon\/rectum, ovaries; pneuroblastomas; lymphomas, uterine leiomyomas; lipomas; endometrial polyps; chondroid hamartomas of the lungs; pleomorphic adenomas of the salivary glands; haemangiopericytomas; chondromatous tumours; aggressive angiomyxomas; diffuse astrocytomas; osteoclastomas; skin cancer; Burkitt's lymphoma; Lewis lung cancer; leukaemia; non-small-cell lung cancer; as well as in each case metastases and\/or metastasising forms thereof.","In an eighth embodiment of the third aspect, which is also an embodiment of the fifth embodiment, the arteriosclerosis is triggered or caused by formation of arteriosclerotic plaques mediated by HMGA1, HMGA1a, HMGA1b and\/or HMGA2.","In a ninth embodiment of the third aspect, which is also an embodiment of the first, second, third, fourth, fifth, sixth, seventh and eighth embodiment, the intracellular target molecule is present intracellularly.","According to a fourth aspect of the invention the object is achieved by the use of a L-nucleic acid for the manufacture of a diagnostic agent for diagnostic purposes, the target molecule of the diagnostic agent being an intracellular target molecule.","In a first embodiment of the fourth aspect the intracellular receptor is selected from the group comprising molecular receptors, enzymes, chaperone molecules, signal peptides, intracellular structures and metabolic intermediates.","In a second embodiment of the fourth aspect, which is also an embodiment of the first embodiment, the intracellular receptor is selected from the group comprising polypeptides, carbohydrates, nucleic acids, lipids and combinations thereof.","In a third embodiment of the fourth aspect, which is also an embodiment of the first and second embodiment, the target molecule is selected from the group comprising transcription factors and DNA-binding proteins binding an AT hook.","In a fourth embodiment of the fourth aspect, which is also an embodiment of the third embodiment, the target molecule is selected from the group comprising HMG proteins, and is preferably selected from the group comprising HMGA, HMGA1a, HMGA1b and HMGA2.","In a fifth embodiment of the fourth aspect, which is also an embodiment of the third and fourth embodiment, the disease is selected from the group comprising tumour diseases, virus infections and arteriosclerosis.","In a sixth embodiment of the fourth aspect, which is also an embodiment of the fifth embodiment, the tumour disease is selected from the group comprising mesenchymal tumours, epithelial tumours, benign tumours, malignant tumours and metastasising tumours.","In a seventh embodiment of the fourth aspect, which is also an embodiment of the third, fourth, fifth and sixth embodiment, the target molecule is HMGA and the disease is selected from the group comprising carcinomas of the prostate, pancreas, thyroid, cervix, stomach, breast, colon\/rectum, ovaries; neuroblastomas; lymphomas, uterine leiomyomas; lipomas; endometrial polyps; chondroid hamartomas of the lungs; pleomorphic adenomas of the salivary glands; haemangiopericytomas; chondromatous tumours; aggressive angiomyxomas; diffuse astrocytomas; osteoclastomas; skin cancer; Burkitt's lymphoma; Lewis lung cancer; leukaemia; non-small-cell lung cancer; as well as in each case metastases and\/or metastasising forms thereof.","In an eighth embodiment of the fourth aspect, which is also an embodiment of the fifth embodiment, the arteriosclerosis is triggered by formation of arteriosclerotic plaques mediated by HMGA1, HMGA1a, HMGA1b and\/or HMGA2.","In a ninth embodiment of the fourth aspect, which is also an embodiment of the first, second, third, fourth, fifth, sixth and seventh embodiment, the intracellular target molecule is present intracellularly.","According to a fifth aspect of the invention the object is achieved by a composition comprising a L-nucleic acid binding to an intracellular target molecule, and a delivery vehicle.","In a first embodiment of the fifth aspect the delivery vehicle is a delivery vehicle suitable for the intracellular delivery of the L-nucleic acid.","In a second embodiment of the fifth aspect, which is also an embodiment of the first embodiment, the delivery vehicle is selected from the group comprising vehicles, conjugates and physical means.","In a third embodiment of the fifth aspect, which is also an embodiment of the second embodiment, the delivery vehicle is a vehicle selected from the group comprising liposomes, nanoparticles, microparticles, cyclodextrins or dendrimers, or a vesicle consisting of polypeptides, polyethyleneimine and\/or amphipathic molecules.","In a fourth embodiment of the fifth aspect, which is also an embodiment of the second embodiment, the delivery vehicle is a conjugate, wherein the conjugate is a conjugate for the receptor-mediated endocytosis, a conjugate with a fusogenic peptide, a conjugate with a signal peptide, a conjugate with a nucleic acid, preferably a conjugate with a spiegelmer, or a lipophilic conjugate.","In a fifth embodiment of the fifth aspect, which is also an embodiment of the second embodiment, the delivery vehicle is a physical means, the physical means preferably being selected from the group comprising electroporation, iontophoresis, pressure, ultrasound and shock waves.","In a sixth embodiment of the fifth aspect, which is also an embodiment of the third embodiment, the delivery vehicle comprises polyethyleneimine.","In a seventh embodiment of the fifth aspect, which is also an embodiment of the sixth embodiment, the polyethyleneimine is a branched polyethyleneimine with a molecular weight of about 25 kDa.","In an eighth embodiment of the fifth aspect, which is also an embodiment of the sixth and seventh embodiment, the polyethyleneimine forms a micelle or a micelle-like structure.","In a ninth embodiment of the fifth aspect, which is also an embodiment of the first, second, third, fourth, fifth, sixth, seventh and eighth embodiment, the L-nucleic acid is a spiegelmer.","In a tenth embodiment of the fifth aspect, which is also an embodiment of the ninth embodiment, the spiegelmer carries a modification, the said modification being selected from the group comprising PEG residues.","In an eleventh embodiment of the fifth aspect, which is also an embodiment of the tenth embodiment, the PEG residue has a molecular weight of about 1,000 to 10,000 Da, preferably a molecular weight of about 1,500 to 2,500 Da and most preferably a molecular weight of about 2,000 Da.","In a twelfth embodiment of the fifth aspect, which is also an embodiment of the tenth and eleventh embodiment, the modification is bound to the 5\u2032 terminus or to the 3\u2032 terminus of the L-nucleic acid.","In a thirteenth embodiment of the fifth aspect, which is also an embodiment of the ninth, tenth, eleventh and twelfth embodiment, in the composition the ratio of the total number of nitrogen groups of the polyethyleneimine to the total number of phosphate groups of the nucleic acid contained in the composition is about 1 to 20, preferably about 1.5 to 10, more preferably about 2 to 5 and most preferably about 2 to 3.","In a fourteenth embodiment of the fifth aspect, which is also an embodiment of the first, second, third, fourth, fifth, sixth, seventh, eighth, ninth, tenth, eleventh, twelfth and thirteenth embodiment, the composition provides the L-nucleic acid intracellularly.","According to a sixth aspect of the invention the object is achieved by the pharmaceutical composition comprising a composition according to the fifth aspect, and a pharmaceutically acceptable carrier.","In an embodiment of the use according to the first aspect the L-nucleic acid is a composition according to the fifth aspect.","In an embodiment of the method according to the second aspect the L-nucleic acid is a composition according to the fifth aspect.","In an embodiment of the use according to the third aspect the L-nucleic acid is a composition according to the fifth aspect.","In an embodiment of the use according to the fourth aspect the L-nucleic acid is a composition according to the fifth aspect.","According to a seventh aspect of the invention the object is achieved by an HMGA-binding nucleic acid, characterised in that the nucleic acid comprises a section Box A1 and a section Box A2, wherein the section Box A1 and the section Box A2 are joined to one another by an intermediate section and wherein Box A1 and Box A2 are selected individually and independently of one another from the group comprising GGGCG, GGGUG and GGGAG.","In a first embodiment of the seventh aspect the intermediate section consists either of an intermediate section Z1 comprising six or seven nucleotides, or of an intermediate section Z2 comprising 12 to 25 nucleotides.","In a second embodiment of the seventh aspect, which is also an embodiment of the first embodiment, the nucleic acid at the 5\u2032 end of the section Box A1 has a first section and at the 3\u2032 end of the section Box A2 has a second section, wherein preferably both sections independently of one another comprise four to eight nucleotides.","In a third embodiment of the seventh aspect, which is also an embodiment of the second embodiment, the two sections are at least partly or completely hybridised with one another, the hybridisation extending over four to eight nucleotide pairs.","In a fourth embodiment of the seventh aspect, which is also an embodiment of the second and third embodiments, the nucleic acid has at the 5\u2032 end of the section Box A1 a section Helix A1 and at the 3\u2032 end of the section Box A2 a section Helix A2, wherein preferably the section Helix A1 comprises four to eight nucleotides and preferably the section Helix A2 comprises four to eight nucleotides, and wherein preferably the section Helix A1 forms the first section at the 5\u2032 end of the section Box A1 or a part thereof, and wherein preferably the section Helix A2 forms the second section at the 3\u2032 end of the section Box A2 or a part thereof, the length of the section Helix A1 being independent of the length of the section Helix A2.","In a fifth embodiment of the seventh aspect, which is also an embodiment of the fourth embodiment, the sections Helix A1 and Helix A2 are at least partly or completely hybridised with one another, the hybridisation extending over four to eight nucleotide pairs.","In a sixth embodiment of the seventh aspect, which is also an embodiment of the fourth and fifth embodiment, between the 3\u2032 end of the section Helix A1 and the 5\u2032 end of the section Box A1 a section Helix B1 is arranged, and between the 3\u2032 end of the section Box A2 and the 5\u2032 end of the section Helix A2 a section Helix B2 is arranged, wherein preferably the length of the section Helix B1 and Helix B2 comprises in each case individually and independently a length of four to eight nucleotides.","In a seventh embodiment of the seventh aspect, which is also an embodiment of the sixth embodiment, the sections Helix B1 and Helix B2 are at least partly or completely hybridised with one another, the hybridisation extending over four to eight nucleotide pairs.","In an eighth embodiment of the seventh aspect, which is also an embodiment of the sixth and seventh embodiment, zero to five nucleotides are arranged between the 3\u2032 end of the section Helix A1 and the 5\u2032 end of the section Helix B1.","In a ninth embodiment of the seventh aspect, which is also an embodiment of the eighth embodiment, two nucleotides are arranged between the 3\u2032 end of the section Helix A1 and the 5\u2032 end of the section Helix B1.","In a tenth embodiment of the seventh aspect, which is also an embodiment of the sixth, seventh, eighth and ninth embodiment, zero to six nucleotides are arranged between the 3\u2032 end of the section Helix B2 and the 5\u2032 end of the section Helix A2.","In an eleventh embodiment of the seventh aspect, which is also an embodiment of the tenth embodiment, preferably insofar as this is an embodiment of the ninth embodiment, a nucleotide is arranged between the 3\u2032 end of the section Helix B2 and the 5\u2032 end of the section Helix A2.","In a twelfth embodiment of the seventh aspect, which is also an embodiment of the sixth, seventh, eighth, ninth, tenth and eleventh embodiment, the sum of the nucleotides of section Helix A1 and of section Helix B1 is ten to twelve nucleotides, and the sum of the nucleotides of section Helix A2 and of section Helix B2 is ten to twelve nucleotides.","In a thirteenth embodiment of the seventh aspect, which is also an embodiment of the twelfth embodiment, the sum of the hybridised nucleotides from the hybridisation of section Helix A1 with section Helix A2 and of section Helix B1 with section Helix B2 is ten to twelve nucleotide pairs.","In a fourteenth embodiment of the seventh aspect, which is also an embodiment of the sixth, seventh, eighth, ninth, tenth, eleventh, twelfth and thirteenth embodiment, preferably of the sixth or seventh embodiment, the nucleic acid does not comprise a section Helix A1 and Helix A2, whereby the section Helix B1 is arranged at the 5\u2032 end of the nucleic acid and the Helix B2 is arranged at the 3\u2032 end, wherein preferably the length of the section Helix B1 and Helix B2 comprises in each case individually and independently a length of four to eight nucleotides.","In a fifteenth embodiment of the seventh aspect, which is also an embodiment of the fourteenth embodiment, the sections Helix B1 and Helix B2 are at least partly or completely hybridised with one another, the hybridisation extending over four to eight nucleotide pairs.","In a sixteenth embodiment of the seventh aspect, which is an embodiment of the fourth and fifth embodiment, one to five nucleotides are arranged between the 3\u2032 end of the section Helix A1 and the 5\u2032 end of the section Box A1, and one to three nucleotides are arranged between the 3\u2032 end of the section Box A2 and the 5\u2032 end of the section Helix A2.","In a seventeenth embodiment of the seventh aspect, which is also an embodiment of the sixth, seventh, eighth, ninth, tenth, eleventh, twelfth, thirteenth, fourteenth and fifteenth embodiment, two nucleotides are arranged between the 3\u2032 end of the section Helix B1 and the 5\u2032 end of the section Box A1, and one to seven nucleotides are arranged between the 3\u2032 end of the section Box A2 and the 5\u2032 end of the section Helix B2.","In an eighteenth embodiment of the seventh aspect, which is also an embodiment of the first, second, third, fourth, fifth, sixth, seventh, eighth and tenth embodiment, insofar as this is an embodiment of the sixth, seventh and eighth embodiment, of the twelfth and thirteenth embodiment, insofar as these are embodiments of the sixth, seventh, eighth and tenth embodiments, of the fourteenth and fifteenth embodiment, insofar as these are embodiments of the sixth, seventh, eighth, tenth, twelfth and thirteenth embodiment, or of the seventeenth embodiment, insofar as these are embodiments of the sixth, eighth, tenth, twelfth, thirteenth and fifteenth embodiment, in each case in the herein restricted scope, the intermediate section Z1 comprises six or seven nucleotides.","In a nineteenth embodiment of the seventh aspect, which is also an embodiment of the eighteenth embodiment, the intermediate section Z1 comprises the sequence NNGNNNN, wherein\n\n","In a twentieth embodiment of the seventh aspect, which is also an embodiment of the nineteenth embodiment, the nucleic acid comprises a section Box A1 and a section Box A2, wherein the 3\u2032 end of the section Box A1 is joined directly to the 5\u2032 end of the intermediate section Z1, and the 3\u2032 end of the intermediate section Z1 is joined directly to the 5\u2032 end of the section Box A2.","In a twenty-first embodiment of the seventh aspect, which is also an embodiment of the eighteenth, nineteenth and twentieth embodiment, in particular of the twentieth embodiment, the nucleic acid comprises a section Helix B1 and a section Helix B2.","In a twenty-second embodiment of the seventh aspect, which is also an embodiment of the twenty-first embodiment, the sections Helix B1 and Helix B2 comprise in each case individually and independently of one another four to eight nucleotides, which are preferably completely or partly hybridised with one another.","In a twenty-third embodiment of the seventh aspect, which is also an embodiment of the twenty-first and twenty-second embodiment, two nucleotides N, Nare arranged between the 3\u2032 end of the section Helix B1 and the 5\u2032 end of the section Box A1 in the 5\u2032 3\u2032 direction, wherein Nis G, A or U, and Nis G or U.","In a twenty-fourth embodiment of the seventh aspect, which is also an embodiment of the twenty-first, twenty-second and twenty-third embodiment, there is no nucleotide between the 3\u2032 end of the section Box A2 and the 5\u2032 end of the section Helix B2, or the nucleotide sequence GNis arranged in the 5\u2032 3\u2032 direction, wherein NY comprises zero to six nucleotides, preferably 0 or 6 nucleotides.","In a twenty-fifth embodiment of the seventh aspect, which is also an embodiment of the eighteenth, nineteenth, twentieth, twenty-first, twenty-second, twenty-third and twenty-fourth embodiment, the nucleic acid comprises a section Helix A1 and Helix A2.","In a twenty-sixth embodiment of the seventh aspect, which is also an embodiment of the twenty-fifth embodiment, the sections Helix A1 and Helix A2 comprise in each case individually and independently of one another four to eight nucleotides, wherein preferably the sections Helix A1 and Helix A2 are completely or partly hybridised with one another.","In a twenty-seventh embodiment of the seventh aspect, which is also an embodiment of the twenty-fifth and twenty-sixth embodiment, a nucleotides sequence Nis arranged between the 3\u2032 end of the section Helix A1 and the 5\u2032 end of the section Helix B1, wherein Ncomprises zero to five nucleotides.","In a twenty-eighth embodiment of the seventh aspect, which is also an embodiment of the twenty-fifth, twenty-sixth and twenty-seventh embodiment, a nucleotide sequence Nis arranged between the 3\u2032 end of the section Helix B2 and the 5\u2032 end of the section Helix A2, wherein Ncomprises zero to six nucleotides.","In a twenty-ninth embodiment of the seventh aspect, which is also an embodiment of the twenty-first, twenty-second, twenty-third, twenty-fourth, twenty-fifth, twenty-sixth, twenty-seventh and twenty-eighth embodiment, the sum of the hybridised nucleotides from the hybridisation of section Helix A1 with section Helix A2 and of section Helix B1 with section Helix B2 is ten to twelve nucleotide pairs.","In a thirtieth embodiment of the seventh aspect, which is also an embodiment of the twenty-fourth, twenty-fifth, twenty-sixth, twenty-seventh, twenty-eighth and twenty-ninth embodiment, the nucleotide sequence GNis arranged between the 3\u2032 end of the section Box A2 and the 5\u2032 end of the section Helix B2 in the 5\u2032-3\u2032 direction, wherein Ncomprises zero to six nucleotides, preferably 0 or 6 nucleotides.","In a thirty-first embodiment of the seventh aspect, which is also an embodiment of the thirtieth embodiment, the HMGA-binding nucleic acid comprises the following structure",{"@attributes":{"id":"p-0098","num":"0106"},"chemistry":{"@attributes":{"id":"CHEM-US-00001","num":"00001"},"img":{"@attributes":{"id":"EMI-C00001","he":"8.30mm","wi":"72.64mm","file":"US08497250-20130730-C00001.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{},"ul":{"@attributes":{"id":"ul0004","list-style":"none"},"li":["N=U, C, A or G;","N=G or U;","N=U or C;","N=U or A;","N=G or A;","N=G, A or U;","N=G or U;","N=U or is no nucleotide;","N=zero to five nucleotides;","N=zero or six nucleotides; and","N=zero to six nucleotides;"]}},"the section Box A1 and section Box A2 are selected in each case individually and independently of one another from the group of nucleotide sequences comprising GGGCG, GGGUG and GGGAG;","the section Helix A1 and the section Helix A2 comprise in each case individually and independently of one another four to eight nucleotides, wherein preferably the sections Helix A1 and Helix A2 are completely or partly hybridised with one another, and","the sections Helix B1 and Helix B2 comprise in each case individually and independently of one another four to eight nucleotides, wherein preferably the sections Helix B1 and Helix B2 are completely or partly hybridised with one another and the hybridising region comprises four to eight nucleotides, and wherein the sum of the hybridised nucleotides from the hybridisation of section Helix A1 with section Helix A2 and of section Helix B1 with section Helix B2 is 10 to 12 nucleotide pairs.","In a thirty-second embodiment of the seventh aspect, which is also an embodiment of the thirtieth and thirty-first embodiment, the HMGA-binding nucleic acid comprises a sequence selected from the group comprising SEQ. ID. No. 1, SEQ. ID. No. 2, SEQ. ID. No. 3, SEQ. ID. No. 5, SEQ. ID. No. 6, SEQ. ID. No. 7 and SEQ. ID. No. 13.","In a thirty-third embodiment of the seventh aspect, which is also an embodiment of the twenty-fourth, twenty-fifth, twenty-sixth, twenty-seventh, twenty-eighth and twenty-ninth embodiment, the 3\u2032 end of the section Box A2 is joined directly to the 5\u2032 end of the section Helix B2.","In a thirty-fourth embodiment of the seventh aspect, which is also an embodiment of the thirty-third embodiment, the HMGA-binding nucleic acid has the following structure",{"@attributes":{"id":"p-0105","num":"0124"},"chemistry":{"@attributes":{"id":"CHEM-US-00002","num":"00002"},"img":{"@attributes":{"id":"EMI-C00002","he":"8.30mm","wi":"71.80mm","file":"US08497250-20130730-C00002.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{},"ul":{"@attributes":{"id":"ul0005","list-style":"none"},"li":["N=U, C, A or G;","N=G or U;","N=U or C;","N=U or A;","N=G or A;","N=G, A or U;","N=G or U;","N=U or is no nucleotide;","N=zero to five nucleotides; and","N=zero to six nucleotides;"]}},"the section Box A1 and Section Box A2 are selected in each case individually and independently of one another from the group of nucleotide sequences comprising GGGCG, GGGUG and GGGAG;","the section Helix A1 and the section Helix A2 comprise in each case individually and independently of one another four to eight nucleotides, wherein preferably the sections Helix A1 and Helix A2 are completely or partly hybridised with one another, and","the sections Helix B1 and Helix B2 comprise in each case individually and independently of one another four to eight nucleotides, wherein preferably the sections Helix B1 and Helix B2 are completely or partly hybridised with one another and the hybridising region comprises four to eight nucleotides, and wherein the sum of the hybridised nucleotides from the hybridisation of section Helix A1 with section Helix A2 and of section Helix B1 with section Helix B2 is 10 to 12 nucleotide pairs.","In a thirty-fifth embodiment of the seventh aspect, which is also an embodiment of the thirty-third and thirty-fourth embodiment, the HMGA-binding nucleic acid comprises a sequence including SEQ. ID. No. 3.","In a thirty-sixth embodiment of the seventh aspect, which is also an embodiment of the thirty-first embodiment, the HMGA-binding nucleic acid comprises the following structure",{"@attributes":{"id":"p-0111","num":"0140"},"chemistry":{"@attributes":{"id":"CHEM-US-00003","num":"00003"},"img":{"@attributes":{"id":"EMI-C00003","he":"3.64mm","wi":"68.24mm","file":"US08497250-20130730-C00003.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"In a thirty-seventh embodiment of the seventh aspect, which is also an embodiment of the thirty-fourth embodiment, the HMGA-binding nucleic acid comprises the following structure",{"@attributes":{"id":"p-0113","num":"0142"},"chemistry":{"@attributes":{"id":"CHEM-US-00004","num":"00004"},"img":{"@attributes":{"id":"EMI-C00004","he":"3.64mm","wi":"61.81mm","file":"US08497250-20130730-C00004.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"In a thirty-eighth embodiment of the seventh aspect, which is also an embodiment of the thirty-sixth embodiment, the HMGA-binding nucleic acid comprises a sequence which is selected from the group including SEQ. ID. No. 15 and SEQ. ID. No. 16.","In a thirty-ninth embodiment of the seventh aspect, which is also an embodiment of the first, second, third, fourth, fifth, sixth, seventh, eighth, ninth, tenth, eleventh, twelfth, thirteenth and sixteenth or seventeenth embodiment of the seventh aspect, the HMGA-binding nucleic acid comprises an intermediate section Zwhich comprises 12 to 25 nucleotides.","In a fortieth embodiment of the seventh aspect, which is also an embodiment of the thirty-ninth embodiment, the HMGA-binding nucleic acid comprises an intermediate section Z2, a section Helix C1 and a section Helix C2.","In a forty-first embodiment of the seventh aspect, which is also an embodiment of the fortieth embodiment, a central section Nis arranged between the section Helix C1 and the section Helix C2 of the HMGA-binding nucleic acid.","In a forty-second embodiment of the seventh aspect, which is also an embodiment of the fortieth or forty-first embodiment, the length of the section Helix C1 and Helix C2 of the HMGA-binding nucleic acid are identical.","In a forty-third embodiment of the seventh aspect, which is also an embodiment of the fortieth, forty-first and forty-second embodiment, the length of the section Helix C1 and Helix C2 of the HMGA-binding nucleic acid is individually and independently three to six nucleotides.","In a forty-fourth embodiment of the seventh aspect, which is also an embodiment of the fortieth, forty-first, forty-second and forty-third embodiment, the sections Helix C1 and Helix C2 of the HMGA-binding nucleic acid are completely or party hybridised with one another.","In a forty-fifth embodiment of the seventh aspect, which is also an embodiment of the thirty-ninth, fortieth, forty-first, forty-second, forty-third and forty-fourth embodiment, the central section Nof the HMGA-binding nucleic acid comprises three to five nucleotides.","In a forty-sixth embodiment of the seventh aspect, which is also an embodiment of the thirty-ninth, fortieth, forty-first, forty-second, forty-third, forty-fourth and forty-fifth embodiment, the HMGA-binding nucleic acid comprises a section Box A1 and a section Box A2, wherein a nucleotide sequence Nis arranged between the 3\u2032 end of the section Box A1 and the 5\u2032 end of the section Helix C1 and comprises three nucleotides.","In a forty-seventh embodiment of the seventh aspect, which is also an embodiment of the thirty-ninth, fortieth, forty-first, forty-second, forty-third, forty-fourth, forty-fifth and forty-sixth embodiment, the HMGA-binding nucleic acid comprises a section Box A1 and a section Box A2, wherein a nucleotide sequence Nis arranged between the 3\u2032 end of the section Helix C2 and the 5\u2032 end of the section Box A2 and comprises two to five nucleotides.","In a forty-eighth embodiment of the seventh aspect, which is also an embodiment of the thirty-ninth, fortieth, forty-first, forty-second, forty-third, forty-fourth, forty-fifth, forty-sixth and forty-seventh embodiment, the HMGA-binding nucleic acid comprises a section Helix A1 and a section Helix A2.","In a forty-ninth embodiment of the seventh aspect, which is also an embodiment of the forty-eighth embodiment, the sections Helix A1 and Helix A2 of the HMGA-binding nucleic acid comprise in each case individually and independently of one another five to six nucleotides, wherein preferably the section Helix A1 and the section Helix A2 are completely or partly hybridised with one another.","In a fiftieth embodiment of the seventh aspect, which is also an embodiment of the forty-eighth and forty-ninth embodiment, a nucleotide sequence Nis arranged between the 3\u2032 end of the section Helix A1 and the 5\u2032 end of the section Box A1 of the HMGA-binding nucleic acid, wherein Ncomprises one to five nucleotides.","In a fifty-first embodiment of the seventh aspect, which is also an embodiment of the forty-eighth, forty-ninth and fiftieth embodiment, a nucleotide sequence GNis arranged between the 3\u2032 end of the section Box A2 and the 5\u2032 end of the section Helix A2 of the HMGA-binding nucleic acid in the 5\u2032-3\u2032 direction, wherein, Ncomprises one to two nucleotides, preferably A or UU.","In a fifty-second embodiment of the seventh aspect, which is also an embodiment of the forty-eighth, forty-ninth, fiftieth and fifty-first embodiment, the section Helix C1 and the section Helix C2 of the HMGA-binding nucleic acid have in each case individually and independently of one another a length of five or six nucleotides, wherein preferably the sections Helix C1 and Helix C2 are completely or partly hybridised with one another.","In a fifty-third embodiment of the seventh aspect, which is also an embodiment of the fifty-second embodiment, the HMGA-binding nucleic acid has the following structure:",{"@attributes":{"id":"p-0130","num":"0159"},"chemistry":{"@attributes":{"id":"CHEM-US-00005","num":"00005"},"img":{"@attributes":{"id":"EMI-C00005","he":"12.87mm","wi":"76.12mm","file":"US08497250-20130730-C00005.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":[{},{},{},{},{},{}],"sub":["a","b","c","d","e"]},"the section Box A1 and the section Box A2 are selected in each case individually and independently of one another from the group comprising GGGCG, GGGUG and GGGAG,","the sections Helix A1 and Helix A2 comprise in each case individually and independently of one another five or six nucleotides, and","the sections Helix C1 and Helix C2 comprise in each case five or six nucleotides, which are preferably completely or partly hybridised with one another.","In a fifty-fourth embodiment of the seventh aspect, which is also an embodiment of the fifty-third embodiment, the HMGA-binding nucleic acid comprises a sequence which is selected from the group comprising SEQ. ID. No. 8, SEQ. ID. No. 9, SEQ. ID. No. 10, SEQ. ID. No. 11, SEQ. ID. No. 14, SEQ. ID. No. 22 and SEQ. ID. No. 24.","In a fifty-fifth embodiment of the seventh aspect, which is also an embodiment of the thirty-ninth, fortieth, forty-first, forty-second, forty-third and forty-fourth embodiment, the nucleic acid comprises a section Box 1 and a section Helix C1 of the HMGA-binding nucleic acid, wherein a nucleotide A is arranged between the 3\u2032 end of the section Box A1 and the 5\u2032 end of the section Helix C1.","In a fifty-sixth embodiment of the seventh aspect, which is also an embodiment of the fifty-fifth embodiment, the HMGA-binding nucleic acid comprises a section Helix C2 and a section Box A2, wherein a nucleotide G is arranged between the 3\u2032 end of the section Helix C2 and the 5\u2032 end of the section Box A2.","In a fifty-seventh embodiment of the seventh aspect, which is also an embodiment of the fifty-fifth or fifty-sixth embodiment, the central section Nof the HMGA-binding nucleic acid comprises four nucleotides, wherein Nis preferably GAUG.","In a fifty-eighth embodiment of the seventh aspect, which is also an embodiment of the fifty-fifth, fifty-sixth and fifty-seventh embodiment, the HMGA-binding nucleic acid comprises a section Helix B1 and a section Helix B2.","In a fifty-ninth embodiment of the seventh aspect, which is also an embodiment of the fifty-eighth embodiment, the sections Helix B1 and Helix B2 of the HMGA-binding nucleic acid comprise individually and independently of one another in each case five nucleotides, wherein preferably the section Helix B1 is hybridised with the section Helix B2.","In a sixtieth embodiment of the seventh aspect, which is also an embodiment of the fifty-eighth or fifty-ninth embodiment, a nucleotide sequence comprising two nucleotides Nis arranged between the 3\u2032 end of the section Helix B1 and the 5\u2032 end of the section Box A1 of the HMGA-binding nucleic acid, wherein Nis preferably AG.","In a sixty-first embodiment of the seventh aspect, which is also an embodiment of the fifty-eighth, fifty-ninth and sixtieth embodiment, a nucleotide G is arranged between the 3\u2032 end of the section Box A2 and the 5\u2032 end of Helix B2 of the HGMA-binding nucleic acid.","In a sixty-second embodiment of the seventh aspect, which is also an embodiment of the fifty-fifth, fifty-sixth, fifty-seventh, fifty-eighth, fifty-ninth, sixtieth and sixty-first embodiment, the HMGA-binding nucleic acid comprises a section Helix A1 and a section Helix A2.","In a sixty-third embodiment of the seventh aspect, which is also an embodiment of the sixty-second embodiment, the sections Helix A1 and Helix A2 of the HMGA-binding nucleic acid comprise individually and independently of one another in each case six nucleotides and preferably the section Helix A1 and the section Helix A2 are hybridised with one another.","In a sixty-fourth embodiment of the seventh aspect, which is also an embodiment of the sixty-second and sixty-third embodiment, a nucleotide sequence comprising two nucleotides Nis arranged between the 3\u2032 end of the section Helix A1 and the 5\u2032 end of the section Helix B1, wherein Nis preferably CA.","In a sixty-fifth embodiment of the seventh aspect, which is also an embodiment of the sixty-second, sixty-third and sixty-fourth embodiment, a nucleotide A is arranged between the 3\u2032 end of the section Helix B2 and the 5\u2032 end of the section Helix A2.","In a sixty-sixth embodiment of the seventh aspect, which is also an embodiment of the fifty-fifth to sixty-fifth embodiment, the sections Helix C1 and Helix C2 comprise in each case three nucleotides, wherein preferably the section Helix C1 and Helix C2 are hybridised with one another.","In a sixty-seventh embodiment of the seventh aspect, which is also an embodiment of the sixty-sixth embodiment, the HMGA-binding nucleic acid has the following structure:",{"@attributes":{"id":"p-0148","num":"0177"},"chemistry":{"@attributes":{"id":"CHEM-US-00006","num":"00006"},"img":{"@attributes":{"id":"EMI-C00006","he":"6.86mm","wi":"59.94mm","file":"US08497250-20130730-C00006.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":[{},{},{},{}],"sub":["i","j","c"]},"the sections Box A1 and Box A2 are in each case selected individually and independently of one another from the group comprising the sequences GGGCG, GGGUG and GGGAG;","the sections Helix A1 and Helix A2 comprise in each case individually and independently six nucleotides, which are preferably hybridised with one another;","the sections Helix B1 and Helix B2 comprise in each case individually and independently five nucleotides, wherein preferably the section Helix B1 and the section Helix B2 are hybridised with one another, and","the section Helix C1 and Helix C2 comprise in each case individually and independently three nucleotides, wherein preferably the sections Helix C1 and Helix C2 are hybridised with one another.","In a sixty-eighth embodiment of the seventh aspect, which is also an embodiment of the sixty-seventh embodiment, the HMGA-binding nucleic acid comprises a sequence which is selected from the group including SEQ. ID. No. 12.","In a sixty-ninth embodiment of the seventh aspect, which is also an embodiment of the second to sixty-seventh embodiment, the nucleic acid is one that binds to transcription factors, in particular transcription factors comprising an AT hook.","According to the invention the object is achieved in an eighth aspect by a nucleic acid, which binds to a transcription factor comprising an AT hook, wherein the nucleic acid has a structure according to the seventh aspect.","In an embodiment of the composition according to the sixth aspect the L-nucleic acid is a nucleic acid according to the seventh aspect.","In an embodiment of the use according to the first aspect the L-nucleic acid is a nucleic acid according to the seventh aspect.","In an embodiment of the method according to the second aspect the L-nucleic acid is a nucleic acid according to the seventh aspect.","In an embodiment of the use according to the third aspect the L-nucleic acid is a nucleic acid according to the seventh aspect.","In an embodiment of the method according to the fourth aspect the L-nucleic acid is a nucleic acid according to the seventh aspect.","According to the invention the object is achieved in a ninth aspect by a method for screening an HMGA antagonist or HMGA agonist, comprising the following steps:\n\n","According to the invention the object is achieved in a tenth aspect by a method for screening an HMGA agonist and\/or an HMGA antagonist, comprising the following steps:\n\n","In an embodiment of the tenth aspect it is envisaged that the determination is carried out by testing whether the nucleic acid is replaced by the candidate HMGA agonist or by the candidate HMGA antagonist.","According to the invention the object is achieved in an eleventh aspect by a kit for the detection of HMGA, comprising a nucleic acid according to the seventh aspect.","According to the invention the object is achieved in a twelfth aspect by an HMGA antagonist which is obtainable by a method according to the tenth aspect.","According to the invention the object is achieved in a thirteenth aspect by an HMGA agonist which is obtainable by a method according to the tenth aspect.","According to the invention the object is achieved in a fourteenth aspect by a complex comprising an HMGA protein and a nucleic acid according to the seventh aspect.","The present invention is based on the surprising result that, contrary to the received opinion in the prior art, it is possible to use L-nucleic acids and in particular spiegelmers in order to address intracellular target molecules. The intracellular target molecules are preferably target molecules which are present in a cell. The properties inherent in the functional L-nucleic acids due to their structure of L-nucleotides, such as high specificity of the interaction with their target molecules with at the same time a high stability and absence of toxic or immunologically active decomposition products when the L-nucleic acids are used in biological systems and in particular in the animal and human body, does not however allow the cellular mechanisms to be utilised in order, as in the case of intramers, for L-nucleic acids to be coded by a plasmid or generally a vector and thus provide the actually functional nucleic acid by the intracellularly occurring process of transcription.","This inescapable dilemma is solved by the present invention: functional L-nucleic acids and in particular spiegelmers can be transported through a cytoplasmic membrane while retaining their specificity as regards their binding to their target molecule, and their activity. This permeability of the functional L-nucleic acids is inherent in spiegelmers and can be enhanced still further by the use of delivery vehicles or delivery techniques. Without hereinafter wishing to be specific in this matter, the present inventors start from the assumption that functional L-nucleic acids can per se overcome the cytoplasmic membrane, and with the participation of endosomal transport mechanisms in overcoming the cytoplasmic membrane, are able to free this from the vesicle structures that are thereby formed, with the adoption of a two-dimensional or three-dimensional structure, which allows the specific interaction of the functional nucleic acid with its target molecule. With the technical teaching disclosed herein, the principle developed for aptamers of utilising intracellular transcription mechanisms in order to create aptamers in the cell is intentionally avoided, and for the first time means are provided for using functional L-nucleic acids and in particular spiegelmers in cells.","As employed herein in a preferred embodiment, the term functional nucleic acids denotes those nucleic acids that are different from structural, in particular naturally occurring structural nucleic acids such as rRNAs or that are different from coding nucleic acids such as mRNAs. In particular functional nucleic acids are nucleic acids which, on account of their two-dimensional and\/or three-dimensional structure, are able to bind to target molecules. In a particularly preferred embodiment the binding to the target molecule takes place not by hybridisation or base pairing on the basis of Watson-Crick base pairings or a Hoogsteen base pairing. Particularly preferred functional nucleic acids are aptamers and spiegelmers.","A L-nucleic acid is in a preferred embodiment a nucleic acid that is completely, substantially or partly synthesised from L-nucleotides. It is particularly preferred if the L-nucleic acid consists completely of L-nucleotides. The term \u201csubstantially\u201d denotes in this connection an embodiment in which that part of the L-nucleic acid which is responsible for the interaction with the target molecule, or that part which mediates the binding to the target molecule, consist of L-nucleotides or is synthesised from these.","As used herein, a functional L-nucleic acid is a functional nucleic acid which is completely, substantially or partly synthesised from L-nucleotides.","The synthesis of L-nucleic acids is known to the person skilled in the art in this field and is described for example in Nolte et al., Nat. Biotech, 14, 1116-1119, 1996; and Klussmann et al., Nat. Biotechnol, 14, 1112-1115, 1996.","The basic process for the production of aptamers is described for example in Tuerck et al. Science, 248, 505-510, 1990; or Ellington et al. Nature, 346, 818-822, 1990, while the basic process for the production of spiegelmers is described for example in Nolte et al., Nat. Biotech, 14, 1116-1119, 1996; or Klussmann et al., Nat. Biotechnol, 14, 1112-1115, 1996. Spiegelmers are thus aptamers which consist of L-nucleotides instead of D-nucleotides. In connection with the production of aptamers and spiegelmers, the term target molecule denotes that molecule which is used in the selection process to produce aptamers and spiegelmers, or denotes that molecule which is ultimately bound by the aptamer or the spiegelmer.","In a preferred embodiment an intracellularly active agent is a chemical compound which when present in a cell is able to bind to a molecule. In this connection it is particularly preferred if the cell is a cell that exists isolated in a tissue or an organ, but preferably not in a human or animal body. If the intracellularly active agent is a spiegelmer, then preferably it is an intracellularly active agent if it is able to bind to an intracellular target molecule. Alternatively the spiegelmer is an intracellularly active agent if it is able to bind to its target molecule under conditions such as exist in a cell. Tests in order to determine these properties are known to the person skilled in the art in this field, and include for example equilibrium binding assays under buffer conditions such as exist intracellularly (ionic strength and solute composition, pH, temperature), as disclosed in Example 1.","In a preferred embodiment the target molecule of the L-nucleic acid, in particular of the functional L-nucleic acid, is an intracellular receptor. An intracellular receptor, as used herein, is preferably a chemical compound or a chemical structure or respectively a part thereof, with which the functional L-nucleic acid interacts, and is preferably a compound or structure to which the functional L-nucleic acid binds, wherein the intracellular receptor, i.e. the chemical compound or the chemical structure or respectively a part thereof, is present intracellularly, i.e. is present in a cell, as is preferably described in the preceding paragraph. In this connection it is possible within the scope of the present invention for the intracellular receptor to be the target molecule in the creation of the functional nucleic acid, in particular the functional L-nucleic acid.","In one embodiment the term \u201creceptor\u201d denotes any interaction partner, preferably a specifically binding interaction partner of the functional nucleic acid, i.e. denotes an interaction partner interacting with the functional nucleic acid, which has a specific spatial structure, charge distribution, hydrophobicity distribution, etc. In a particularly preferred embodiment the interaction partner corresponds to the target molecule of the functional nucleic acid, as was used in the creation of the functional nucleic acid. In this connection it is within the scope of the present invention that a receptor can also be different from the target molecule used in the creation of the functional nucleic acid, though the specific interaction is due to a cross reactivity of the functional nucleic acid between the interaction partner and the target molecule used in the creation of the functional nucleic acid.","In a preferred embodiment the term \u201cintracellular receptor\u201d denotes a receptor that is present in a cell, or a receptor that can be present in a cell, that occurs under natural circumstances in a cell, or that under such circumstances exist in a cell. In this connection it is particularly preferred if the cell is a cell that occurs isolated in a tissue or an organ, but preferably not in a human or animal body. As used herein, the term \u201cintracellular receptor\u201d however also denotes a receptor that is present under conditions such as exist in a cell.","In a preferred embodiment the term \u201ccell\u201d denotes a cell which is selected from the group comprising prokaryotic and eukaryotic cells. Preferably the eukaryotic cell is selected from the group comprising fungal cells, plant cells, animal cells and human cells. In an alternative embodiment the term cell generally denotes herein a compartment bounded by a phospholipid double membrane, which in a preferred embodiment corresponds to a cytoplasmic membrane, and which is separated by the membrane from the surroundings. The separation from the surroundings is in this connection not a complete separation, but allows an energy transfer and a mass transfer (substance exchange) between the cell and the surroundings. The mass transfer is preferably restricted. In the case where the cell is separated from the surroundings by a cytoplasmic membrane or by a membrane similar to a cytoplasmic membrane, the restriction of the mass transfer is defined by the transport properties of the membrane. In one embodiment the term cell herein thus also includes vesicles and\/or compartments of a prokaryotic or eukaryotic cell as defined herein, which in turn are present or may be present both within a prokaryotic or eukaryotic cell, as well as outside such a prokaryotic or eukaryotic cell, for example as vesicles or parts of a prokaryotic or eukaryotic cell surrounded by a cytoplasmic membrane, which in one embodiment can be present in a body fluid. In a preferred embodiment, in a cell according to the second alternative embodiment it is envisaged that the conditions within such a cell correspond substantially to those occurring in a prokaryotic or eukaryotic cell, in particular as regards the factors which influence the binding of the functional nucleic acid to its target molecule.","In a preferred embodiment the body fluid is selected from the group comprising blood, urine, liquor (anatomical fluid), lymph fluid, serum, plasma, vaginal secretions, saliva and sperm.","In one embodiment the receptor is defined by its function in a cell. Accordingly the receptor can be selected from the group comprising molecular receptors, enzymes, metabolic intermediates, signal peptides, chaperone molecules and intracellular structures such as for example ribosomes, mitochondria, elements of the cytoskeleton such as for example tubulin and actin filaments, endosomal particles, lysosomes, other intracellular structures such as vesicles, in particular intracellular vesicles. As used herein the term molecular receptor denotes in a preferred embodiment a molecule which accepts information and transmits this within a cell, a tissue, an organ or an organism. The information is typically mediated by a molecule which interacts with the molecular receptor. As a result of the interaction the molecular receptor is able to generate a signal. Such a signal can be based on the change in the confirmation and\/or of the activity of the receptor or can be manifested therein. The signal itself is able to transmit in another form the information received or processed by it. As a result of the change in the confirmation or activity of the molecular receptor, the signal can preferably be a chemical, biochemical or an electrical signal. Preferably the molecular receptor is part of a reaction cascade, and more preferably part of a signal cascade. The information transmitted by a molecular receptor can be quantitative and\/or qualitative information, for example concerning the presence of a compound and\/or its concentration.","In a preferred embodiment the term \u201cmetabolic intermediates\u201d denotes all those compounds which, due to metabolic activities in a cell, occur as constituents of catabolism as well as of anabolism.","In a further embodiment the receptor is defined by its chemical nature. Preferably the receptor is selected from the group comprising polypeptides, carbohydrates, nucleic acids, lipids and combinations thereof. As used herein, the term polypeptide preferably denotes any polymer consisting of two or more amino acids. Preferably the amino acids are L-amino acids, though D-amino acids may also be used within the scope of the embodiment. As used herein the term \u201cnucleic acids\u201d preferably denotes a polymer of two or more nucleotides or nucleotide analogues which are known to the person skilled in the art in this field, wherein the nucleotides may be either D-nucleotides or L-nucleotides or mixtures thereof. Preferred combinations include glycosylated polypeptides and glycosylated lipids.","A particular group of intracellular receptors are transcription factors and DNA-binding proteins which bind to an AT hook. Examples of transcription factors are given in the following table 1:",{"@attributes":{"id":"p-0185","num":"0222"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 1"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Transcription factors"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"154pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"gamma)OBP"]},{"entry":[{},"(STAT5A)4"]},{"entry":[{},"120-kDa CRE-"]},{"entry":[{},"binding protein"]},{"entry":[{},"14-3-3 epsilon"]},{"entry":[{},"14-3-3 zeta"]},{"entry":[{},"50-55K protein"]},{"entry":[{},"53BP1"]},{"entry":[{},"70-75K protein"]},{"entry":[{},"80-90K protein"]},{"entry":[{},"AAF"]},{"entry":[{},"ABF-1"]},{"entry":[{},"ADA2"]},{"entry":[{},"ADA3"]},{"entry":[{},"ADA-NF1"]},{"entry":[{},"AFP1"]},{"entry":[{},"AhR"]},{"entry":[{},"AhR:Arnt"]},{"entry":[{},"AIIN3"]},{"entry":[{},"Aiolos"]},{"entry":[{},"AIRE"]},{"entry":[{},"AKNA"]},{"entry":[{},"ALF"]},{"entry":[{},"ALL-1"]},{"entry":[{},"alpha-CBF"]},{"entry":[{},"alpha-CP2b"]},{"entry":[{},"alphaH0"]},{"entry":[{},"alphaH2-alphaH3"]},{"entry":[{},"ALX3"]},{"entry":[{},"Alx-4"]},{"entry":[{},"aMEF-2"]},{"entry":[{},"AML1"]},{"entry":[{},"AML1a"]},{"entry":[{},"AML1b"]},{"entry":[{},"AML1c"]},{"entry":[{},"AML1DeltaN"]},{"entry":[{},"AML2"]},{"entry":[{},"AML3"]},{"entry":[{},"AML3a"]},{"entry":[{},"AML3b"]},{"entry":[{},"AMY-1L"]},{"entry":[{},"A-Myb"]},{"entry":[{},"ANF"]},{"entry":[{},"AP-1"]},{"entry":[{},"AP-2alphaA"]},{"entry":[{},"AP-2alphaB"]},{"entry":[{},"AP-2beta"]},{"entry":[{},"AP-2gamma"]},{"entry":[{},"AP-2rep"]},{"entry":[{},"AP-3 (1)"]},{"entry":[{},"AP-3 (2)"]},{"entry":[{},"AP-4"]},{"entry":[{},"AP-5"]},{"entry":[{},"APC"]},{"entry":[{},"AR"]},{"entry":[{},"Arnt"]},{"entry":[{},"Arnt (774 AA form)"]},{"entry":[{},"ARNT2"]},{"entry":[{},"ASC-2"]},{"entry":[{},"ASPP1"]},{"entry":[{},"ASPP2"]},{"entry":[{},"ATBF1-A"]},{"entry":[{},"ATBF1-B"]},{"entry":[{},"ATF"]},{"entry":[{},"ATF-1"]},{"entry":[{},"ATF2"]},{"entry":[{},"ATF-2"]},{"entry":[{},"ATF-2:c-Jun"]},{"entry":[{},"ATF3"]},{"entry":[{},"ATF3 deltaZIP"]},{"entry":[{},"ATF4"]},{"entry":[{},"ATF5"]},{"entry":[{},"ATF6"]},{"entry":[{},"ATF-a"]},{"entry":[{},"ATF-adelta"]},{"entry":[{},"ATOH1"]},{"entry":[{},"ATPF1"]},{"entry":[{},"Bach1"]},{"entry":[{},"Bach1t"]},{"entry":[{},"Bach2"]},{"entry":[{},"BAF155"]},{"entry":[{},"BAF47"]},{"entry":[{},"BAF53a"]},{"entry":[{},"BAF60A"]},{"entry":[{},"Barhl1"]},{"entry":[{},"Barhl2"]},{"entry":[{},"Barx1"]},{"entry":[{},"Barx2"]},{"entry":[{},"Bcl-3"]},{"entry":[{},"BCL-6"]},{"entry":[{},"beta-catenin"]},{"entry":[{},"Bin1"]},{"entry":[{},"BMAL2"]},{"entry":[{},"B-Myb"]},{"entry":[{},"BNC"]},{"entry":[{},"BP1"]},{"entry":[{},"BP2"]},{"entry":[{},"BR140"]},{"entry":[{},"Brachyury"]},{"entry":[{},"BRCA1"]},{"entry":[{},"BRCA2"]},{"entry":[{},"BRG1"]},{"entry":[{},"BRIP1"]},{"entry":[{},"Brm"]},{"entry":[{},"BTEB1"]},{"entry":[{},"BTEB2"]},{"entry":[{},"BTEB3"]},{"entry":[{},"BTEB4"]},{"entry":[{},"B-TFIID"]},{"entry":[{},"C\/EBPalpha"]},{"entry":[{},"C\/EBPbeta"]},{"entry":[{},"C\/EBPdelta"]},{"entry":[{},"C\/EBPepsilon"]},{"entry":[{},"C\/EBPgamma"]},{"entry":[{},"CA150"]},{"entry":[{},"c-abl"]},{"entry":[{},"CACCC-binding factor"]},{"entry":[{},"CAR"]},{"entry":[{},"CAR:RXR-alpha"]},{"entry":[{},"Cart-1"]},{"entry":[{},"CBAF"]},{"entry":[{},"CBF (4)"]},{"entry":[{},"CBF (5)"]},{"entry":[{},"CBP"]},{"entry":[{},"CCAAT-"]},{"entry":[{},"binding factor"]},{"entry":[{},"CCF"]},{"entry":[{},"CCG1"]},{"entry":[{},"CCK-1a"]},{"entry":[{},"CCK-1b"]},{"entry":[{},"CD28RC"]},{"entry":[{},"Cdc5"]},{"entry":[{},"cdk2"]},{"entry":[{},"cdk9"]},{"entry":[{},"Cdx-1"]},{"entry":[{},"CDX2"]},{"entry":[{},"Cdx-4"]},{"entry":[{},"c-Ets-1"]},{"entry":[{},"c-Ets-2"]},{"entry":[{},"CFF"]},{"entry":[{},"c-Fos"]},{"entry":[{},"ChCh"]},{"entry":[{},"CHOP-10"]},{"entry":[{},"Chx10"]},{"entry":[{},"CIITA"]},{"entry":[{},"c-Jun"]},{"entry":[{},"c-Jun:JunD"]},{"entry":[{},"CLIM1"]},{"entry":[{},"CLIM2"]},{"entry":[{},"CLOCK"]},{"entry":[{},"c-Myb"]},{"entry":[{},"c-Myc"]},{"entry":[{},"C-Myc 1"]},{"entry":[{},"c-Myc:Max"]},{"entry":[{},"CNBP"]},{"entry":[{},"CoS"]},{"entry":[{},"COUP-TF1"]},{"entry":[{},"COUP-TF2"]},{"entry":[{},"CP1A"]},{"entry":[{},"CP1C"]},{"entry":[{},"CP2"]},{"entry":[{},"CPBP"]},{"entry":[{},"CPE binding protein"]},{"entry":[{},"CREB"]},{"entry":[{},"CRE-BPa"]},{"entry":[{},"c-Rel"]},{"entry":[{},"c-Rel:RelA"]},{"entry":[{},"CREMalpha"]},{"entry":[{},"CREST"]},{"entry":[{},"CRF"]},{"entry":[{},"Crx"]},{"entry":[{},"CSA"]},{"entry":[{},"CSB"]},{"entry":[{},"CSBP-1"]},{"entry":[{},"CSEN"]},{"entry":[{},"c-Ski"]},{"entry":[{},"CtBP1"]},{"entry":[{},"CtBP2"]},{"entry":[{},"CTCF"]},{"entry":[{},"CTF"]},{"entry":[{},"CTF-1"]},{"entry":[{},"CTF-2"]},{"entry":[{},"CTF-3"]},{"entry":[{},"CTF-5"]},{"entry":[{},"CTF-7"]},{"entry":[{},"CUP"]},{"entry":[{},"CUTL1"]},{"entry":[{},"CUTL2"]},{"entry":[{},"Cx"]},{"entry":[{},"cyclin A"]},{"entry":[{},"cyclin T1"]},{"entry":[{},"cyclin T2"]},{"entry":[{},"cyclin T2a"]},{"entry":[{},"cyclin T2b"]},{"entry":[{},"DAP"]},{"entry":[{},"DAX1"]},{"entry":[{},"DB1"]},{"entry":[{},"DBF4"]},{"entry":[{},"DBP"]},{"entry":[{},"DbpA"]},{"entry":[{},"DbpAv"]},{"entry":[{},"DbpB"]},{"entry":[{},"DCoHm"]},{"entry":[{},"DDB"]},{"entry":[{},"DDB-1"]},{"entry":[{},"DDB-2"]},{"entry":[{},"DEC1"]},{"entry":[{},"DEC2"]},{"entry":[{},"DEF"]},{"entry":[{},"deltaCREB"]},{"entry":[{},"deltaFosB"]},{"entry":[{},"deltaMax"]},{"entry":[{},"DeltaN p63beta"]},{"entry":[{},"DeltaN p73alpha"]},{"entry":[{},"DeltaN p73beta"]},{"entry":[{},"DeltaN p73gamma"]},{"entry":[{},"DeltaNp63alpha"]},{"entry":[{},"DeltaNp63gamma"]},{"entry":[{},"Dermo-1"]},{"entry":[{},"DF-1"]},{"entry":[{},"DF-2"]},{"entry":[{},"DF-3"]},{"entry":[{},"Dlx-1"]},{"entry":[{},"Dlx-2"]},{"entry":[{},"Dlx-3"]},{"entry":[{},"Dlx-4 (long isoform)"]},{"entry":[{},"Dlx-4 (short isoform)"]},{"entry":[{},"Dlx-5"]},{"entry":[{},"Dlx-6"]},{"entry":[{},"DP-1"]},{"entry":[{},"DP-2"]},{"entry":[{},"DPBF"]},{"entry":[{},"DRIL1"]},{"entry":[{},"DSIF"]},{"entry":[{},"DSIF-p14"]},{"entry":[{},"DSIF-p160"]},{"entry":[{},"DTF"]},{"entry":[{},"DUX1"]},{"entry":[{},"DUX2"]},{"entry":[{},"DUX3"]},{"entry":[{},"DUX4"]},{"entry":[{},"E"]},{"entry":[{},"E12"]},{"entry":[{},"E2F"]},{"entry":[{},"E2F + E4"]},{"entry":[{},"E2F + p107"]},{"entry":[{},"E2F-1"]},{"entry":[{},"E2F-1:DP-1"]},{"entry":[{},"E2F-1:DP-2"]},{"entry":[{},"E2F-2"]},{"entry":[{},"E2F-3a"]},{"entry":[{},"E2F-4"]},{"entry":[{},"E2F-4:DP-1"]},{"entry":[{},"E2F-4:DP-2"]},{"entry":[{},"E2F-5"]},{"entry":[{},"E2F-6"]},{"entry":[{},"E2F-7"]},{"entry":[{},"E47"]},{"entry":[{},"E4BP4"]},{"entry":[{},"E4F"]},{"entry":[{},"E4F1"]},{"entry":[{},"E4TF2"]},{"entry":[{},"E7; HPV-16, Papilloma"]},{"entry":[{},"Virus type 16"]},{"entry":[{},"EAR2"]},{"entry":[{},"EBF"]},{"entry":[{},"EBP-80"]},{"entry":[{},"EC2"]},{"entry":[{},"EF1"]},{"entry":[{},"Egr-1"]},{"entry":[{},"Egr-2"]},{"entry":[{},"Egr-3"]},{"entry":[{},"Egr-4"]},{"entry":[{},"EllaE-A"]},{"entry":[{},"EllaE-B"]},{"entry":[{},"EllaE-Calpha"]},{"entry":[{},"EllaE-Cbeta"]},{"entry":[{},"EivF"]},{"entry":[{},"EKLF"]},{"entry":[{},"ELF-1"]},{"entry":[{},"ELFR"]},{"entry":[{},"elios"]},{"entry":[{},"Elk-1"]},{"entry":[{},"Emx-1"]},{"entry":[{},"Emx-2"]},{"entry":[{},"En-1"]},{"entry":[{},"En-2"]},{"entry":[{},"ENH-binding protein"]},{"entry":[{},"ENKTF-1"]},{"entry":[{},"EP400"]},{"entry":[{},"EPAS1"]},{"entry":[{},"Epicardin"]},{"entry":[{},"epsilonF1"]},{"entry":[{},"ER-alpha"]},{"entry":[{},"ER-alpha:ER-beta"]},{"entry":[{},"ER-beta"]},{"entry":[{},"ER-beta1"]},{"entry":[{},"ER-beta2"]},{"entry":[{},"ER-beta3"]},{"entry":[{},"ER-beta4"]},{"entry":[{},"ER-beta5"]},{"entry":[{},"ERF"]},{"entry":[{},"Erg-1"]},{"entry":[{},"Erg-2"]},{"entry":[{},"ERM"]},{"entry":[{},"ERR1"]},{"entry":[{},"ERR2"]},{"entry":[{},"ERR3"]},{"entry":[{},"ERR3-1"]},{"entry":[{},"ERR3-2"]},{"entry":[{},"ERR3-3"]},{"entry":[{},"ERRalpha1"]},{"entry":[{},"ESE-1"]},{"entry":[{},"ESE-1a"]},{"entry":[{},"ESE-1b"]},{"entry":[{},"ESE-2"]},{"entry":[{},"ESE-2a"]},{"entry":[{},"ESE-2b"]},{"entry":[{},"ESE-3"]},{"entry":[{},"ESE-3a"]},{"entry":[{},"ESE-3b"]},{"entry":[{},"ESXR1"]},{"entry":[{},"ETF"]},{"entry":[{},"Ets-1 deltaVII"]},{"entry":[{},"Evi-1"]},{"entry":[{},"EVX1"]},{"entry":[{},"EZF-2"]},{"entry":[{},"EZH1"]},{"entry":[{},"EZH2"]},{"entry":[{},"F2F"]},{"entry":[{},"FAC1"]},{"entry":[{},"factor 2"]},{"entry":[{},"FBP"]},{"entry":[{},"f-EBP"]},{"entry":[{},"FEV"]},{"entry":[{},"Fgf3"]},{"entry":[{},"FKBP59"]},{"entry":[{},"FKHL18"]},{"entry":[{},"FKHRL1P2"]},{"entry":[{},"FKLF"]},{"entry":[{},"Fli-1"]},{"entry":[{},"FosB"]},{"entry":[{},"FOXB1"]},{"entry":[{},"FOXC1"]},{"entry":[{},"FOXC2"]},{"entry":[{},"FOXD1"]},{"entry":[{},"FOXD2"]},{"entry":[{},"FOXD3"]},{"entry":[{},"FOXD4"]},{"entry":[{},"FOXE1"]},{"entry":[{},"FOXE3"]},{"entry":[{},"FOXF1"]},{"entry":[{},"FOXF2"]},{"entry":[{},"FOXG1a"]},{"entry":[{},"FOXG1b"]},{"entry":[{},"FOXG1c"]},{"entry":[{},"FOXH1"]},{"entry":[{},"FOXI1"]},{"entry":[{},"FOXJ1a"]},{"entry":[{},"FOXJ1b"]},{"entry":[{},"FOXJ2 (long isoform)"]},{"entry":[{},"FOXJ2 (short isoform)"]},{"entry":[{},"FOXJ3"]},{"entry":[{},"FOXK1"]},{"entry":[{},"FOXK2a"]},{"entry":[{},"FOXK2b"]},{"entry":[{},"FOXK2c"]},{"entry":[{},"FOXL1"]},{"entry":[{},"FOXL2"]},{"entry":[{},"FOXM1a"]},{"entry":[{},"FOXM1b"]},{"entry":[{},"FOXM1c"]},{"entry":[{},"FOXN1"]},{"entry":[{},"FOXN2"]},{"entry":[{},"FOXN3"]},{"entry":[{},"FOXO1a"]},{"entry":[{},"FOXO1b"]},{"entry":[{},"FOXO2"]},{"entry":[{},"FOXO3a"]},{"entry":[{},"FOXO3b"]},{"entry":[{},"FOXO4"]},{"entry":[{},"FOXP1"]},{"entry":[{},"FOXP2"]},{"entry":[{},"FOXP3"]},{"entry":[{},"FOXP4"]},{"entry":[{},"Fra-1"]},{"entry":[{},"Fra-2"]},{"entry":[{},"FTF"]},{"entry":[{},"FTS"]},{"entry":[{},"FXR"]},{"entry":[{},"FXR:RXR-alpha"]},{"entry":[{},"FXR-alpha"]},{"entry":[{},"FXR-beta1"]},{"entry":[{},"FXR-beta2"]},{"entry":[{},"G factor"]},{"entry":[{},"G6 factor"]},{"entry":[{},"GAAP-1"]},{"entry":[{},"GABP"]},{"entry":[{},"GABP-alpha"]},{"entry":[{},"GABPB"]},{"entry":[{},"GABP-beta1"]},{"entry":[{},"GABP-beta2"]},{"entry":[{},"GAF"]},{"entry":[{},"gammaCAAT"]},{"entry":[{},"gammaCAC1"]},{"entry":[{},"gammaCAC2"]},{"entry":[{},"GATA-1"]},{"entry":[{},"GATA-2"]},{"entry":[{},"GATA-3"]},{"entry":[{},"GATA-4"]},{"entry":[{},"GATA-5"]},{"entry":[{},"GATA-6"]},{"entry":[{},"Gbx1"]},{"entry":[{},"Gbx2"]},{"entry":[{},"GCF"]},{"entry":[{},"GCMa"]},{"entry":[{},"GCN5"]},{"entry":[{},"GCNF-1"]},{"entry":[{},"GCNF-2"]},{"entry":[{},"GF1"]},{"entry":[{},"GKLF"]},{"entry":[{},"GLI1"]},{"entry":[{},"GLI2"]},{"entry":[{},"GLI3"]},{"entry":[{},"GLIS2"]},{"entry":[{},"GMEB-1"]},{"entry":[{},"GR"]},{"entry":[{},"GR-alpha"]},{"entry":[{},"GR-beta"]},{"entry":[{},"GRF-1"]},{"entry":[{},"Gsc"]},{"entry":[{},"Gscl"]},{"entry":[{},"GT-IC"]},{"entry":[{},"GT-IIA"]},{"entry":[{},"GT-IIBalpha"]},{"entry":[{},"GT-IIBbeta"]},{"entry":[{},"H1TF1"]},{"entry":[{},"H1TF2"]},{"entry":[{},"H1TF2A"]},{"entry":[{},"H4TF-1"]},{"entry":[{},"H4TF-2"]},{"entry":[{},"HAF"]},{"entry":[{},"HAND1"]},{"entry":[{},"HAND2"]},{"entry":[{},"HB9"]},{"entry":[{},"HDAC1"]},{"entry":[{},"HDAC2"]},{"entry":[{},"HDAC3"]},{"entry":[{},"HDAC4"]},{"entry":[{},"HDAC5"]},{"entry":[{},"hDaxx"]},{"entry":[{},"HDBP1"]},{"entry":[{},"HDBP2"]},{"entry":[{},"Heat-inducing factor"]},{"entry":[{},"HEB"]},{"entry":[{},"HEB1-p67"]},{"entry":[{},"HEB1-p94"]},{"entry":[{},"HEF-1B"]},{"entry":[{},"HEF-1T"]},{"entry":[{},"HEF-4C"]},{"entry":[{},"HEN1"]},{"entry":[{},"HEN2"]},{"entry":[{},"HES-1"]},{"entry":[{},"HES-2"]},{"entry":[{},"Hesx1"]},{"entry":[{},"Hex"]},{"entry":[{},"Hey1"]},{"entry":[{},"Hey2"]},{"entry":[{},"HeyL"]},{"entry":[{},"HFH-1"]},{"entry":[{},"HIC-1"]},{"entry":[{},"Hic-5"]},{"entry":[{},"HIF-1"]},{"entry":[{},"HIF-1alpha"]},{"entry":[{},"HiNF-A"]},{"entry":[{},"HiNF-B"]},{"entry":[{},"HiNF-C"]},{"entry":[{},"HiNF-D"]},{"entry":[{},"HiNF-D3"]},{"entry":[{},"HiNF-E"]},{"entry":[{},"HiNF-P"]},{"entry":[{},"HIP1"]},{"entry":[{},"HIV-EP2"]},{"entry":[{},"Hlf"]},{"entry":[{},"HLTF"]},{"entry":[{},"HLTF (Met123)"]},{"entry":[{},"HLX"]},{"entry":[{},"HMBP"]},{"entry":[{},"HMG I"]},{"entry":[{},"HMG I(Y)"]},{"entry":[{},"HMG Y"]},{"entry":[{},"HMGB1"]},{"entry":[{},"HMGB2"]},{"entry":[{},"HMGI-C"]},{"entry":[{},"HMX1"]},{"entry":[{},"HNF-1alpha-A"]},{"entry":[{},"HNF-1alpha-B"]},{"entry":[{},"HNF-1alpha-C"]},{"entry":[{},"HNF-1beta-A"]},{"entry":[{},"HNF-1beta-B"]},{"entry":[{},"HNF-1beta-C"]},{"entry":[{},"HNF-3"]},{"entry":[{},"HNF-3alpha"]},{"entry":[{},"HNF-3beta"]},{"entry":[{},"HNF-3gamma"]},{"entry":[{},"HNF-4"]},{"entry":[{},"HNF-4alpha"]},{"entry":[{},"HNF-4alpha1"]},{"entry":[{},"HNF-4alpha2"]},{"entry":[{},"HNF-4alpha3"]},{"entry":[{},"HNF-4alpha4"]},{"entry":[{},"HNF-4alpha7"]},{"entry":[{},"HNF-4gamma"]},{"entry":[{},"HNF-6alpha"]},{"entry":[{},"hnRNP K"]},{"entry":[{},"HOX11"]},{"entry":[{},"HOXA1"]},{"entry":[{},"HOXA10"]},{"entry":[{},"HOXA10 PL2"]},{"entry":[{},"HOXA11"]},{"entry":[{},"HOXA13"]},{"entry":[{},"HOXA2"]},{"entry":[{},"HOXA3"]},{"entry":[{},"HOXA4"]},{"entry":[{},"HOXA5"]},{"entry":[{},"HOXA6"]},{"entry":[{},"HOXA7"]},{"entry":[{},"HOXA9A"]},{"entry":[{},"HOXA9B"]},{"entry":[{},"HOXB1"]},{"entry":[{},"HOXB13"]},{"entry":[{},"HOXB2"]},{"entry":[{},"HOXB3"]},{"entry":[{},"HOXB4"]},{"entry":[{},"HOXB5"]},{"entry":[{},"HOXB6"]},{"entry":[{},"HOXB7"]},{"entry":[{},"HOXB8"]},{"entry":[{},"HOXB9"]},{"entry":[{},"HOXC10"]},{"entry":[{},"HOXC11"]},{"entry":[{},"HOXC12"]},{"entry":[{},"HOXC13"]},{"entry":[{},"HOXC4"]},{"entry":[{},"HOXC5"]},{"entry":[{},"HOXC6"]},{"entry":[{},"HOXC8"]},{"entry":[{},"HOXC9"]},{"entry":[{},"HOXD10"]},{"entry":[{},"HOXD11"]},{"entry":[{},"HOXD12"]},{"entry":[{},"HOXD13"]},{"entry":[{},"HOXD3"]},{"entry":[{},"HOXD4"]},{"entry":[{},"HOXD8"]},{"entry":[{},"HOXD9"]},{"entry":[{},"Hp55"]},{"entry":[{},"Hp65"]},{"entry":[{},"HPX42B"]},{"entry":[{},"HrpF"]},{"entry":[{},"HSBP1"]},{"entry":[{},"HSF"]},{"entry":[{},"HSF1 (long)"]},{"entry":[{},"HSF1 (short)"]},{"entry":[{},"HSF2"]},{"entry":[{},"HSF4a"]},{"entry":[{},"HSF4b"]},{"entry":[{},"HSF4c"]},{"entry":[{},"hsp56"]},{"entry":[{},"Hsp90"]},{"entry":[{},"IA-1"]},{"entry":[{},"iASPP"]},{"entry":[{},"iASPP-RAI"]},{"entry":[{},"IB1"]},{"entry":[{},"IBP-1"]},{"entry":[{},"ICER-II"]},{"entry":[{},"ICER-IIgamma"]},{"entry":[{},"Id1"]},{"entry":[{},"Id1H"]},{"entry":[{},"Id2"]},{"entry":[{},"Id3"]},{"entry":[{},"Id3\/Heir-1"]},{"entry":[{},"IF1"]},{"entry":[{},"IFI-16"]},{"entry":[{},"IgPE-1"]},{"entry":[{},"IgPE-2"]},{"entry":[{},"IgPE-3"]},{"entry":[{},"Ik-1"]},{"entry":[{},"IkappaB"]},{"entry":[{},"IkappaB-alpha"]},{"entry":[{},"IkappaB-beta"]},{"entry":[{},"IkappaBR"]},{"entry":[{},"II-1 RF"]},{"entry":[{},"IL-10E1"]},{"entry":[{},"IL-6 RE-BP"]},{"entry":[{},"II-6 RF"]},{"entry":[{},"ING1"]},{"entry":[{},"ING1b"]},{"entry":[{},"INSAF"]},{"entry":[{},"IPCS-BF"]},{"entry":[{},"IPF1"]},{"entry":[{},"IPF1:Pbx"]},{"entry":[{},"IRF-1"]},{"entry":[{},"IRF-1:C\/EBPbeta"]},{"entry":[{},"IRF-2"]},{"entry":[{},"IRF-3"]},{"entry":[{},"IRF-4"]},{"entry":[{},"IRF-5"]},{"entry":[{},"IRF-6"]},{"entry":[{},"IRF-7A"]},{"entry":[{},"IRF-7B"]},{"entry":[{},"IRF-7H"]},{"entry":[{},"IRF-8"]},{"entry":[{},"IRF-9"]},{"entry":[{},"irlB"]},{"entry":[{},"IRX-1"]},{"entry":[{},"IRX2a"]},{"entry":[{},"Irx-3"]},{"entry":[{},"lrx-4"]},{"entry":[{},"ISGF-1"]},{"entry":[{},"ISGF-3"]},{"entry":[{},"ISGF-3alpha"]},{"entry":[{},"IsI-1alpha"]},{"entry":[{},"ITF"]},{"entry":[{},"ITF-1"]},{"entry":[{},"ITF-2"]},{"entry":[{},"JRF"]},{"entry":[{},"JunB"]},{"entry":[{},"JunB:Fra-1"]},{"entry":[{},"JunB:Fra-2"]},{"entry":[{},"JunD"]},{"entry":[{},"JunD:Fra-2"]},{"entry":[{},"kappaY FaKtor"]},{"entry":[{},"KBP-1"]},{"entry":[{},"KER1"]},{"entry":[{},"KER-1"]},{"entry":[{},"KLF15"]},{"entry":[{},"KLF7"]},{"entry":[{},"Kox1"]},{"entry":[{},"KR3"]},{"entry":[{},"KRF-1"]},{"entry":[{},"KRN"]},{"entry":[{},"KSR-1"]},{"entry":[{},"Ku autoantigen"]},{"entry":[{},"Ku70"]},{"entry":[{},"Ku80"]},{"entry":[{},"KUP"]},{"entry":[{},"LAF-4"]},{"entry":[{},"LANA; KSHV, Kaposi's"]},{"entry":[{},"sarcoma-associated"]},{"entry":[{},"herpes virus"]},{"entry":[{},"(herpes virus 8)"]},{"entry":[{},"LBP-1"]},{"entry":[{},"LBP-1a"]},{"entry":[{},"LBP-1d"]},{"entry":[{},"LBP-32"]},{"entry":[{},"LBP-9"]},{"entry":[{},"LBX1"]},{"entry":[{},"LCR-F1"]},{"entry":[{},"LEF-1"]},{"entry":[{},"LEF-1B"]},{"entry":[{},"LF-A1"]},{"entry":[{},"LHX1"]},{"entry":[{},"LHX2"]},{"entry":[{},"LHX3a"]},{"entry":[{},"LHX3b"]},{"entry":[{},"LHX5"]},{"entry":[{},"LHX6.1a"]},{"entry":[{},"LHX6.1b"]},{"entry":[{},"LIT-1"]},{"entry":[{},"LITAF"]},{"entry":[{},"LKLF"]},{"entry":[{},"Lmo1"]},{"entry":[{},"Lmo2"]},{"entry":[{},"LMO3"]},{"entry":[{},"LMX1A"]},{"entry":[{},"LMX1B"]},{"entry":[{},"L-Myc-1(long form)"]},{"entry":[{},"L-Myc-1(short form)"]},{"entry":[{},"L-Myc-2"]},{"entry":[{},"LUN-1"]},{"entry":[{},"LUN-2"]},{"entry":[{},"LXR-alpha"]},{"entry":[{},"LXR-alpha:RXR-alpha"]},{"entry":[{},"LXR-beta"]},{"entry":[{},"LXR-beta:RXR-alpha"]},{"entry":[{},"Lyl-1"]},{"entry":[{},"M factor"]},{"entry":[{},"Mad1"]},{"entry":[{},"Maf"]},{"entry":[{},"MafB"]},{"entry":[{},"MafF"]},{"entry":[{},"MafG"]},{"entry":[{},"MafG:MafG"]},{"entry":[{},"MafK"]},{"entry":[{},"MAML1"]},{"entry":[{},"MASH-1"]},{"entry":[{},"Max"]},{"entry":[{},"Max1"]},{"entry":[{},"Max2"]},{"entry":[{},"MAZ"]},{"entry":[{},"MAZi"]},{"entry":[{},"MAZR"]},{"entry":[{},"MBF1"]},{"entry":[{},"MBF2"]},{"entry":[{},"MBF3"]},{"entry":[{},"MBP-1 (1)"]},{"entry":[{},"MBP-1 (2)"]},{"entry":[{},"MBP-2"]},{"entry":[{},"MDBP"]},{"entry":[{},"MECP-2"]},{"entry":[{},"MEF-2A"]},{"entry":[{},"MEF-2B1"]},{"entry":[{},"MEF-2C"]},{"entry":[{},"MEF-2C\/delta32"]},{"entry":[{},"MEF-2C\/delta8"]},{"entry":[{},"MEF-2C\/delta8, 32"]},{"entry":[{},"MEF-2D00"]},{"entry":[{},"MEF-2D0B"]},{"entry":[{},"MEF-2DA0"]},{"entry":[{},"MEF-2DA'0"]},{"entry":[{},"MEF-2DAB"]},{"entry":[{},"MEF-2DA'B"]},{"entry":[{},"Meis-1"]},{"entry":[{},"Meis-2a"]},{"entry":[{},"Meis-2b"]},{"entry":[{},"Meis-2c"]},{"entry":[{},"Meis-2d"]},{"entry":[{},"Meis-2e"]},{"entry":[{},"Meis-3"]},{"entry":[{},"Mel-18"]},{"entry":[{},"Meox1"]},{"entry":[{},"Meox1a"]},{"entry":[{},"Meox2"]},{"entry":[{},"MHox (K-2)"]},{"entry":[{},"MIF-1"]},{"entry":[{},"MITF"]},{"entry":[{},"MIXL1"]},{"entry":[{},"Miz-1"]},{"entry":[{},"MLX"]},{"entry":[{},"MM-1"]},{"entry":[{},"MondoA"]},{"entry":[{},"MOP3"]},{"entry":[{},"MR"]},{"entry":[{},"MRF-2"]},{"entry":[{},"Msx-1"]},{"entry":[{},"Msx-2"]},{"entry":[{},"MTA1-L1"]},{"entry":[{},"MTB-Zf"]},{"entry":[{},"MTF-1"]},{"entry":[{},"mtTFA"]},{"entry":[{},"Mxi1"]},{"entry":[{},"Myf-3"]},{"entry":[{},"Myf-4"]},{"entry":[{},"Myf-5"]},{"entry":[{},"Myf-6"]},{"entry":[{},"Myocardin, Splice Form 1"]},{"entry":[{},"MyoD"]},{"entry":[{},"MyoD:E12"]},{"entry":[{},"MyT1"]},{"entry":[{},"MZF-1"]},{"entry":[{},"NC1"]},{"entry":[{},"NC2"]},{"entry":[{},"NCOR1"]},{"entry":[{},"NCOR2"]},{"entry":[{},"NCX"]},{"entry":[{},"NELF"]},{"entry":[{},"NERF"]},{"entry":[{},"NERF-1a"]},{"entry":[{},"NERF-1b"]},{"entry":[{},"NERF-2"]},{"entry":[{},"Net"]},{"entry":[{},"NeuroD1"]},{"entry":[{},"NEUROD-2"]},{"entry":[{},"NEUROD-3"]},{"entry":[{},"NF III-a"]},{"entry":[{},"NF III-c"]},{"entry":[{},"NF III-e"]},{"entry":[{},"NF-1"]},{"entry":[{},"NF-4FA"]},{"entry":[{},"NF-4FB"]},{"entry":[{},"NF-4FC"]},{"entry":[{},"NF-AB"]},{"entry":[{},"NF-AT1"]},{"entry":[{},"NF-AT1"]},{"entry":[{},"NF-AT2"]},{"entry":[{},"NF-AT2-alpha"]},{"entry":[{},"NF-AT2-beta"]},{"entry":[{},"NF-AT3"]},{"entry":[{},"NF-AT4"]},{"entry":[{},"NF-AT5"]},{"entry":[{},"NfbetaA"]},{"entry":[{},"NF-CLE0a"]},{"entry":[{},"NF-CLE0b"]},{"entry":[{},"NFdeltaE3A"]},{"entry":[{},"NFdeltaE3B"]},{"entry":[{},"NFdeltaE3C"]},{"entry":[{},"NFdeltaE4A"]},{"entry":[{},"NFdeltaE4B"]},{"entry":[{},"NFdeltaE4C"]},{"entry":[{},"Nfe"]},{"entry":[{},"NF-E"]},{"entry":[{},"NF-E2"]},{"entry":[{},"NF-E2 p45"]},{"entry":[{},"NF-E3"]},{"entry":[{},"NFE-6"]},{"entry":[{},"NF-Gma"]},{"entry":[{},"NF-GMb"]},{"entry":[{},"NFI\/CTF"]},{"entry":[{},"NFIA"]},{"entry":[{},"NFIB"]},{"entry":[{},"NF-IL-2\u00b0"]},{"entry":[{},"NF-IL-2B"]},{"entry":[{},"NFIX"]},{"entry":[{},"NF-jun"]},{"entry":[{},"NF-kappaB"]},{"entry":[{},"NF-kappaB(-similar)"]},{"entry":[{},"NF-kappaB1"]},{"entry":[{},"NF-kappaB1 precursor"]},{"entry":[{},"NF-kappaB2"]},{"entry":[{},"NF-kappaB2 (p49)"]},{"entry":[{},"NF-kappaB2 precursor"]},{"entry":[{},"NF-kappaE1"]},{"entry":[{},"NF-kappaE2"]},{"entry":[{},"NF-kappaE3"]},{"entry":[{},"NF-MHCIIA"]},{"entry":[{},"NF-MHCIIB"]},{"entry":[{},"NF-muE1"]},{"entry":[{},"NF-muE2"]},{"entry":[{},"NF-muE3"]},{"entry":[{},"NF-S"]},{"entry":[{},"NF-X"]},{"entry":[{},"NF-X1"]},{"entry":[{},"NF-X2"]},{"entry":[{},"NF-X3"]},{"entry":[{},"NF-Xc"]},{"entry":[{},"NF-Y"]},{"entry":[{},"NF-YA"]},{"entry":[{},"NF-Zc"]},{"entry":[{},"NF-Zz"]},{"entry":[{},"NGN3"]},{"entry":[{},"NHP-1"]},{"entry":[{},"NHP-2"]},{"entry":[{},"NHP3"]},{"entry":[{},"NHP4"]},{"entry":[{},"Nkx2-1"]},{"entry":[{},"Nkx2-2"]},{"entry":[{},"Nkx2-3"]},{"entry":[{},"Nkx2-5"]},{"entry":[{},"Nkx2-8"]},{"entry":[{},"Nkx3-1"]},{"entry":[{},"Nkx3-1 v1"]},{"entry":[{},"Nkx3-1 v2"]},{"entry":[{},"Nkx3-1 v3"]},{"entry":[{},"Nkx3-1 v4"]},{"entry":[{},"Nkx3-2"]},{"entry":[{},"Nkx6-1"]},{"entry":[{},"Nkx6-2"]},{"entry":[{},"Nmi"]},{"entry":[{},"N-Myc"]},{"entry":[{},"N-Oct-2alpha"]},{"entry":[{},"N-Oct-2beta"]},{"entry":[{},"N-Oct-4"]},{"entry":[{},"NOR1"]},{"entry":[{},"NOR1\/MINOR"]},{"entry":[{},"NPA3"]},{"entry":[{},"NPAS1"]},{"entry":[{},"NPAS2"]},{"entry":[{},"NP-TCII"]},{"entry":[{},"NRF"]},{"entry":[{},"Nrf1"]},{"entry":[{},"NRF-1"]},{"entry":[{},"Nrf1:MafG"]},{"entry":[{},"Nrf1:MafK"]},{"entry":[{},"Nrf2"]},{"entry":[{},"Nrf2:MafG"]},{"entry":[{},"Nrf2:MafK"]},{"entry":[{},"NRF-2beta1"]},{"entry":[{},"NRF-2gamma1"]},{"entry":[{},"Nrf3"]},{"entry":[{},"Nrf3:MafK"]},{"entry":[{},"NRL"]},{"entry":[{},"NRSF"]},{"entry":[{},"NRSF Form 1"]},{"entry":[{},"NRSF Form 2"]},{"entry":[{},"NTF"]},{"entry":[{},"Nur77"]},{"entry":[{},"NURR1"]},{"entry":[{},"OAZ"]},{"entry":[{},"OC-2"]},{"entry":[{},"OCA-B"]},{"entry":[{},"Octa factor"]},{"entry":[{},"Octamer"]},{"entry":[{},"binding factor"]},{"entry":[{},"Oct-B1"]},{"entry":[{},"oct-B2"]},{"entry":[{},"oct-B3"]},{"entry":[{},"OLIG2"]},{"entry":[{},"Oligo1"]},{"entry":[{},"Otx1"]},{"entry":[{},"Otx2"]},{"entry":[{},"Otx3"]},{"entry":[{},"OZF"]},{"entry":[{},"p107"]},{"entry":[{},"p130"]},{"entry":[{},"p160MBP"]},{"entry":[{},"p28 Modulator"]},{"entry":[{},"p300"]},{"entry":[{},"p38erg"]},{"entry":[{},"p40x; HTLV-I, T-cell"]},{"entry":[{},"Lymphotropic virus"]},{"entry":[{},"type I"]},{"entry":[{},"p45"]},{"entry":[{},"p49erg"]},{"entry":[{},"p50:c-Rel"]},{"entry":[{},"p53"]},{"entry":[{},"p55"]},{"entry":[{},"p55erg"]},{"entry":[{},"p63"]},{"entry":[{},"p63alpha"]},{"entry":[{},"p63beta"]},{"entry":[{},"p63delta"]},{"entry":[{},"p63gamma"]},{"entry":[{},"p65delta"]},{"entry":[{},"p73"]},{"entry":[{},"p73alpha"]},{"entry":[{},"p73beta"]},{"entry":[{},"p73delta"]},{"entry":[{},"p73epsilon"]},{"entry":[{},"p73eta"]},{"entry":[{},"p73gamma"]},{"entry":[{},"p73kappa"]},{"entry":[{},"p73zeta"]},{"entry":[{},"Pax-1"]},{"entry":[{},"Pax-2"]},{"entry":[{},"Pax-3"]},{"entry":[{},"Pax-3A"]},{"entry":[{},"Pax-3B"]},{"entry":[{},"Pax-4a"]},{"entry":[{},"Pax-5"]},{"entry":[{},"Pax-6"]},{"entry":[{},"Pax-6\/Pd-5a"]},{"entry":[{},"Pax-7"]},{"entry":[{},"Pax-8"]},{"entry":[{},"Pax-8a"]},{"entry":[{},"Pax-8b"]},{"entry":[{},"Pax-8c"]},{"entry":[{},"Pax-8d"]},{"entry":[{},"Pax-8e"]},{"entry":[{},"Pax-8f"]},{"entry":[{},"Pax-9"]},{"entry":[{},"Pbx"]},{"entry":[{},"Pbx1"]},{"entry":[{},"Pbx1:HoxB1"]},{"entry":[{},"Pbx1:HoxB2"]},{"entry":[{},"Pbx1:HoxB3"]},{"entry":[{},"Pbx1:HoxB4"]},{"entry":[{},"Pbx1:HoxB5"]},{"entry":[{},"Pbx1:HoxB6"]},{"entry":[{},"Pbx1:HoxB8"]},{"entry":[{},"Pbx1:PKNOX1"]},{"entry":[{},"Pbx1:Tcl3"]},{"entry":[{},"Pbx1a"]},{"entry":[{},"Pbx1A:HoxA5"]},{"entry":[{},"Pbx1a:Hoxb7"]},{"entry":[{},"Pbx1a:Hoxb8"]},{"entry":[{},"Pbx1a:Hoxc6"]},{"entry":[{},"Pbx1A:HoxC8"]},{"entry":[{},"Pbx1A:HoxD4"]},{"entry":[{},"Pbx1a:IPF1"]},{"entry":[{},"Pbx1b"]},{"entry":[{},"Pbx1B:HoxA5"]},{"entry":[{},"Pbx1B:HoxB7"]},{"entry":[{},"Pbx1B:HoxB8"]},{"entry":[{},"Pbx1B:HoxC8"]},{"entry":[{},"Pbx1B:HoxD4"]},{"entry":[{},"Pbx1b:PKNOX1"]},{"entry":[{},"Pbx2"]},{"entry":[{},"Pbx2:HoxB8"]},{"entry":[{},"Pbx2:Hoxc6"]},{"entry":[{},"Pbx2:PKNOX1"]},{"entry":[{},"Pbx3a"]},{"entry":[{},"Pbx3a:Hoxc6"]},{"entry":[{},"Pbx3b"]},{"entry":[{},"PC2"]},{"entry":[{},"PC4"]},{"entry":[{},"PC5"]},{"entry":[{},"PCAF"]},{"entry":[{},"PDEF"]},{"entry":[{},"PEA3"]},{"entry":[{},"PEBP2alpha"]},{"entry":[{},"PEBP2beta"]},{"entry":[{},"PGC-1"]},{"entry":[{},"PITX1"]},{"entry":[{},"PITX2"]},{"entry":[{},"PITX2A"]},{"entry":[{},"PITX2A:Nkx2.5"]},{"entry":[{},"PITX2B"]},{"entry":[{},"PITX2B:Nkx2.5"]},{"entry":[{},"PITX2C"]},{"entry":[{},"PITX2C:Nkx2.5"]},{"entry":[{},"PITX3"]},{"entry":[{},"PKNOX1"]},{"entry":[{},"PKNOX2"]},{"entry":[{},"PLAGL1"]},{"entry":[{},"PLAGL2"]},{"entry":[{},"PLZF"]},{"entry":[{},"PML"]},{"entry":[{},"PML-3"]},{"entry":[{},"Pmx2a"]},{"entry":[{},"Pmx2b"]},{"entry":[{},"PNR"]},{"entry":[{},"PO-B"]},{"entry":[{},"Pontin52"]},{"entry":[{},"POU1F1"]},{"entry":[{},"POU2F1"]},{"entry":[{},"POU2F2"]},{"entry":[{},"POU2F2 (Oct-2.1)"]},{"entry":[{},"POU2F2B"]},{"entry":[{},"POU2F2C"]},{"entry":[{},"POU2F3"]},{"entry":[{},"POU2F3, Isoform a"]},{"entry":[{},"POU2F3, Isoform d1"]},{"entry":[{},"POU2F3, Isoform d2"]},{"entry":[{},"POU3F1"]},{"entry":[{},"POU3F2"]},{"entry":[{},"POU3F2 (N-Oct-5a)"]},{"entry":[{},"POU3F2 (N-Oct-5b)"]},{"entry":[{},"POU3F3"]},{"entry":[{},"POU3F4"]},{"entry":[{},"POU4F1(l)"]},{"entry":[{},"POU4F1(s)"]},{"entry":[{},"POU4F2"]},{"entry":[{},"POU4F3"]},{"entry":[{},"POU5F1"]},{"entry":[{},"POU5F1A"]},{"entry":[{},"POU5F1B"]},{"entry":[{},"POU5F1C"]},{"entry":[{},"POU6F1"]},{"entry":[{},"PPAR-alpha"]},{"entry":[{},"PPAR-alpha:RXR-"]},{"entry":[{},"alpha"]},{"entry":[{},"PPAR-beta"]},{"entry":[{},"PPAR-gamma1"]},{"entry":[{},"PPAR-gamma2"]},{"entry":[{},"PPAR-gamma3"]},{"entry":[{},"PPAR-gamma4"]},{"entry":[{},"PPUR"]},{"entry":[{},"PR"]},{"entry":[{},"PR A"]},{"entry":[{},"PR B"]},{"entry":[{},"pRb"]},{"entry":[{},"PRDI-BF1"]},{"entry":[{},"PRDI-BFc"]},{"entry":[{},"Preb"]},{"entry":[{},"Prop-1"]},{"entry":[{},"PROX1"]},{"entry":[{},"PSE1"]},{"entry":[{},"P-TEFb"]},{"entry":[{},"PTF"]},{"entry":[{},"PTFalpha"]},{"entry":[{},"PTFbeta"]},{"entry":[{},"PTFdelta"]},{"entry":[{},"PTFgamma"]},{"entry":[{},"Pu box binding factor"]},{"entry":[{},"Pu box binding factor"]},{"entry":[{},"(BJA-B)"]},{"entry":[{},"PU.1"]},{"entry":[{},"PuF"]},{"entry":[{},"Pur factor"]},{"entry":[{},"pX; HBV, Hepatitis B"]},{"entry":[{},"Virus"]},{"entry":[{},"PXR-1"]},{"entry":[{},"PXR-1:RXR-alpha"]},{"entry":[{},"PXR-1:RXR-beta"]},{"entry":[{},"PXR-2"]},{"entry":[{},"R1"]},{"entry":[{},"R2"]},{"entry":[{},"RAR-alpha"]},{"entry":[{},"RAR-alpha:RXR-alpha"]},{"entry":[{},"RAR-alpha:RXR-beta"]},{"entry":[{},"RAR-alpha:RXR-"]},{"entry":[{},"gamma"]},{"entry":[{},"RAR-alpha1"]},{"entry":[{},"RAR-alpha2"]},{"entry":[{},"RAR-beta"]},{"entry":[{},"RAR-beta:RXR-alpha"]},{"entry":[{},"RAR-beta2"]},{"entry":[{},"RAR-gamma"]},{"entry":[{},"RAR-gamma:RXR-"]},{"entry":[{},"alpha"]},{"entry":[{},"RAR-gamma1"]},{"entry":[{},"Rb:E2F-1:DP-1"]},{"entry":[{},"RBP60"]},{"entry":[{},"RBP-Jkappa"]},{"entry":[{},"Ref-1"]},{"entry":[{},"RelA"]},{"entry":[{},"RelB"]},{"entry":[{},"REVERB-alpha"]},{"entry":[{},"REVERB-beta"]},{"entry":[{},"RFX1"]},{"entry":[{},"RFX1:RFX2"]},{"entry":[{},"RFX1:RFX3"]},{"entry":[{},"RFX2"]},{"entry":[{},"RFX3"]},{"entry":[{},"RFX4"]},{"entry":[{},"RFX5"]},{"entry":[{},"RFX5:RFXAP:RFXANK"]},{"entry":[{},"RFXANK"]},{"entry":[{},"RFXAP"]},{"entry":[{},"RFX-B-delta5"]},{"entry":[{},"RF-Y"]},{"entry":[{},"RORalpha1"]},{"entry":[{},"RORalpha2"]},{"entry":[{},"RORalpha3"]},{"entry":[{},"RORbeta"]},{"entry":[{},"RORgamma"]},{"entry":[{},"Rox"]},{"entry":[{},"RP58"]},{"entry":[{},"RPF1"]},{"entry":[{},"RPGalpha"]},{"entry":[{},"RREB-1"]},{"entry":[{},"RSRFC4"]},{"entry":[{},"RSRFC9"]},{"entry":[{},"RVF"]},{"entry":[{},"RX"]},{"entry":[{},"RXR-alpha"]},{"entry":[{},"RXR-beta"]},{"entry":[{},"RXR-gamma"]},{"entry":[{},"SAP-1a"]},{"entry":[{},"SAP-1b"]},{"entry":[{},"SF-1"]},{"entry":[{},"SHOX2a"]},{"entry":[{},"SHOX2b"]},{"entry":[{},"SHOXa"]},{"entry":[{},"SHOXb"]},{"entry":[{},"SHP"]},{"entry":[{},"SIII-p110"]},{"entry":[{},"SIII-p15"]},{"entry":[{},"SIII-p18"]},{"entry":[{},"SIM1"]},{"entry":[{},"SIM2"]},{"entry":[{},"SIP1"]},{"entry":[{},"Six-1"]},{"entry":[{},"Six-2"]},{"entry":[{},"Six-3"]},{"entry":[{},"Six-4"]},{"entry":[{},"Six-5"]},{"entry":[{},"Six-6"]},{"entry":[{},"SKIP"]},{"entry":[{},"SLUG"]},{"entry":[{},"Smad1"]},{"entry":[{},"Smad2"]},{"entry":[{},"Smad2 (437"]},{"entry":[{},"amino acids)"]},{"entry":[{},"Smad3"]},{"entry":[{},"Smad3:Smad4"]},{"entry":[{},"Smad4"]},{"entry":[{},"Smad4delta3"]},{"entry":[{},"Smad4delta4"]},{"entry":[{},"Smad4delta4-6"]},{"entry":[{},"Smad4delta4-7"]},{"entry":[{},"Smad4delta5-6"]},{"entry":[{},"Smad4delta6"]},{"entry":[{},"Smad5"]},{"entry":[{},"Smad6"]},{"entry":[{},"Smad7"]},{"entry":[{},"Smad8"]},{"entry":[{},"SMIF"]},{"entry":[{},"Sna"]},{"entry":[{},"SnoN"]},{"entry":[{},"Sox1"]},{"entry":[{},"Sox10"]},{"entry":[{},"Sox11"]},{"entry":[{},"Sox12"]},{"entry":[{},"Sox13"]},{"entry":[{},"Sox14"]},{"entry":[{},"Sox17"]},{"entry":[{},"Sox18"]},{"entry":[{},"Sox2"]},{"entry":[{},"Sox20"]},{"entry":[{},"Sox21"]},{"entry":[{},"Sox3"]},{"entry":[{},"Sox4"]},{"entry":[{},"Sox5"]},{"entry":[{},"Sox7"]},{"entry":[{},"Sox8"]},{"entry":[{},"Sox9"]},{"entry":[{},"Sp1"]},{"entry":[{},"Sp2"]},{"entry":[{},"Sp3"]},{"entry":[{},"Sp4"]},{"entry":[{},"Spi-B"]},{"entry":[{},"SPT16"]},{"entry":[{},"SRC-1"]},{"entry":[{},"SRC-3"]},{"entry":[{},"SRCAP"]},{"entry":[{},"SREBP-1a"]},{"entry":[{},"SREBP-1b"]},{"entry":[{},"SREBP-1c"]},{"entry":[{},"SREBP-2"]},{"entry":[{},"SRE-ZBP"]},{"entry":[{},"SRF"]},{"entry":[{},"SRF:SRF"]},{"entry":[{},"SRY"]},{"entry":[{},"SSRP1"]},{"entry":[{},"Staf-50"]},{"entry":[{},"STAT1"]},{"entry":[{},"STAT1:STAT1"]},{"entry":[{},"STAT1:STAT3"]},{"entry":[{},"STAT1alpha"]},{"entry":[{},"STAT1beta"]},{"entry":[{},"STAT2"]},{"entry":[{},"STAT3"]},{"entry":[{},"STAT3:STAT3"]},{"entry":[{},"STAT4"]},{"entry":[{},"STAT5A"]},{"entry":[{},"STAT5B"]},{"entry":[{},"STAT5B:STAT5B"]},{"entry":[{},"STAT6"]},{"entry":[{},"SXR"]},{"entry":[{},"SXR:RXR-alpha"]},{"entry":[{},"SYT"]},{"entry":[{},"T3R-alpha:"]},{"entry":[{},"T3R-alpha:RXR-alpha"]},{"entry":[{},"T3R-alpha1"]},{"entry":[{},"T3R-alpha2"]},{"entry":[{},"T3R-beta1"]},{"entry":[{},"T3R-beta2"]},{"entry":[{},"TAF(I)110"]},{"entry":[{},"TAF(I)48"]},{"entry":[{},"TAF(I)63"]},{"entry":[{},"TAF(II)100"]},{"entry":[{},"TAF(II)125"]},{"entry":[{},"TAF(II)135"]},{"entry":[{},"TAF(II)170"]},{"entry":[{},"TAF(II)18"]},{"entry":[{},"TAF(II)20"]},{"entry":[{},"TAF(II)250"]},{"entry":[{},"TAF(II)250Delta"]},{"entry":[{},"TAF(II)28"]},{"entry":[{},"TAF(II)30"]},{"entry":[{},"TAF(II)31"]},{"entry":[{},"TAF(II)55"]},{"entry":[{},"TAF(II)70-alpha"]},{"entry":[{},"TAF(II)70-beta"]},{"entry":[{},"TAF(Il)70-gamma"]},{"entry":[{},"TAF-I"]},{"entry":[{},"TAF-II"]},{"entry":[{},"TAF-L"]},{"entry":[{},"Tal-1"]},{"entry":[{},"Tal-1beta"]},{"entry":[{},"Tal-2"]},{"entry":[{},"TAR factor"]},{"entry":[{},"tat; HIV-1,"]},{"entry":[{},"Immunodeficiency virus"]},{"entry":[{},"type 1"]},{"entry":[{},"Tax; HTLV-I, T-cell"]},{"entry":[{},"Lymphotropic"]},{"entry":[{},"virus type I"]},{"entry":[{},"T-bet"]},{"entry":[{},"TBP"]},{"entry":[{},"Tbr-1"]},{"entry":[{},"TBR2"]},{"entry":[{},"TBX18"]},{"entry":[{},"TBX19"]},{"entry":[{},"TBX1A"]},{"entry":[{},"TBX1B"]},{"entry":[{},"TBX2"]},{"entry":[{},"TBX20"]},{"entry":[{},"Tbx22"]},{"entry":[{},"TBX3 (722"]},{"entry":[{},"amino acids)"]},{"entry":[{},"TBX3 (742"]},{"entry":[{},"amino acids)"]},{"entry":[{},"TBX4"]},{"entry":[{},"TBX5 (long isoform)"]},{"entry":[{},"TBX5 (short isoform)"]},{"entry":[{},"Tbx5:Nkx2.5"]},{"entry":[{},"TBX6"]},{"entry":[{},"TCF"]},{"entry":[{},"TCF-1"]},{"entry":[{},"TCF17"]},{"entry":[{},"TCF-1A"]},{"entry":[{},"TCF-1B"]},{"entry":[{},"TCF-1C"]},{"entry":[{},"TCF-1D"]},{"entry":[{},"TCF-1E"]},{"entry":[{},"TCF-1F"]},{"entry":[{},"TCF-1G"]},{"entry":[{},"TCF-2alpha"]},{"entry":[{},"TCF-3"]},{"entry":[{},"TCF-4"]},{"entry":[{},"TCF-4(K)"]},{"entry":[{},"TCF-4B"]},{"entry":[{},"TCF-4E"]},{"entry":[{},"TEF"]},{"entry":[{},"TEF-1"]},{"entry":[{},"TEF-2"]},{"entry":[{},"TEF-3"]},{"entry":[{},"TEF-5"]},{"entry":[{},"TEL1"]},{"entry":[{},"Tel-2a"]},{"entry":[{},"Tel-2b"]},{"entry":[{},"Tel-2c"]},{"entry":[{},"Tel-2d"]},{"entry":[{},"Tel-2e"]},{"entry":[{},"Tel-2f"]},{"entry":[{},"TFE3"]},{"entry":[{},"TFEB"]},{"entry":[{},"TFEB-A"]},{"entry":[{},"TFEC"]},{"entry":[{},"TFIIA"]},{"entry":[{},"TFIIA-alpha\/beta"]},{"entry":[{},"precursor (main form)"]},{"entry":[{},"TFIIA-alpha\/beta"]},{"entry":[{},"precursor (subsidiary form)"]},{"entry":[{},"TFIIA-gamma"]},{"entry":[{},"TFIIB"]},{"entry":[{},"TFIID"]},{"entry":[{},"TFIIE"]},{"entry":[{},"TFIIE-alpha"]},{"entry":[{},"TFIIE-beta"]},{"entry":[{},"TFIIF"]},{"entry":[{},"TFIIF-alpha"]},{"entry":[{},"TFIIF-beta"]},{"entry":[{},"TFIIH"]},{"entry":[{},"TFIIH*"]},{"entry":[{},"TFIIH-CAK"]},{"entry":[{},"TFIIH-cyclin H"]},{"entry":[{},"TFIIH-MAT1"]},{"entry":[{},"TFIIH-MO15"]},{"entry":[{},"TFIIH-p34"]},{"entry":[{},"TFIIH-p44"]},{"entry":[{},"TFIIH-p62"]},{"entry":[{},"TFIIH-p80"]},{"entry":[{},"TFIIH-p80:CAK"]},{"entry":[{},"TFIIH-p90"]},{"entry":[{},"TFII-I"]},{"entry":[{},"TFIIIA"]},{"entry":[{},"Tf-LF1"]},{"entry":[{},"Tf-LF2"]},{"entry":[{},"TFP-95"]},{"entry":[{},"TGIF"]},{"entry":[{},"TGIF2"]},{"entry":[{},"TGT3"]},{"entry":[{},"TIEG-1"]},{"entry":[{},"TIF1a"]},{"entry":[{},"TIF1g"]},{"entry":[{},"TIF2"]},{"entry":[{},"TLE1"]},{"entry":[{},"TLX"]},{"entry":[{},"TLX3"]},{"entry":[{},"TMF"]},{"entry":[{},"TR2-11"]},{"entry":[{},"TR2-5"]},{"entry":[{},"TR2-9"]},{"entry":[{},"TR4"]},{"entry":[{},"TRAP"]},{"entry":[{},"TREB-1"]},{"entry":[{},"TREB-2"]},{"entry":[{},"TREB-3"]},{"entry":[{},"TREF1"]},{"entry":[{},"TREF2"]},{"entry":[{},"TRF (2)"]},{"entry":[{},"TRRAP"]},{"entry":[{},"TWIST"]},{"entry":[{},"TxRE BP"]},{"entry":[{},"TxREF"]},{"entry":[{},"UBF"]},{"entry":[{},"UBP-1"]},{"entry":[{},"UEF-1"]},{"entry":[{},"UEF-2"]},{"entry":[{},"UEF-3"]},{"entry":[{},"UEF-4"]},{"entry":[{},"USF1"]},{"entry":[{},"USF1:USF2"]},{"entry":[{},"USF2"]},{"entry":[{},"USF2b"]},{"entry":[{},"Vav"]},{"entry":[{},"Vax-2"]},{"entry":[{},"VDR"]},{"entry":[{},"VITF; Vaccinia virus\/,"]},{"entry":[{},{"i":"Homo sapiens"}]},{"entry":[{},"Vpr; HIV-1,"]},{"entry":[{},"Immunodeficiency virus"]},{"entry":[{},"type 1"]},{"entry":[{},"WBSCR14"]},{"entry":[{},"WSTF"]},{"entry":[{},"WT1"]},{"entry":[{},"WT1 I"]},{"entry":[{},"WT1 I-KTS"]},{"entry":[{},"WT1 I-del2"]},{"entry":[{},"WTI-KTS"]},{"entry":[{},"WT1-del2"]},{"entry":[{},"XBP-1"]},{"entry":[{},"XW, V"]},{"entry":[{},"YAF2"]},{"entry":[{},"YB-1"]},{"entry":[{},"YEBP"]},{"entry":[{},"YL-1"]},{"entry":[{},"YY1"]},{"entry":[{},"ZAC"]},{"entry":[{},"ZBP89"]},{"entry":[{},"ZBP99"]},{"entry":[{},"ZEB (1124 AA)"]},{"entry":[{},"ZEB (1154 AA)"]},{"entry":[{},"ZER6 p52"]},{"entry":[{},"ZER6 p71"]},{"entry":[{},"ZF1"]},{"entry":[{},"ZF2"]},{"entry":[{},"ZFP-37"]},{"entry":[{},"ZFX"]},{"entry":[{},"ZFY"]},{"entry":[{},"ZHX1"]},{"entry":[{},"ZIC2"]},{"entry":[{},"ZID"]},{"entry":[{},"ZNF11a"]},{"entry":[{},"ZNF124"]},{"entry":[{},"ZNF133"]},{"entry":[{},"ZNF143"]},{"entry":[{},"ZNF174"]},{"entry":[{},"ZNF-20"]},{"entry":[{},"ZNF-24"]},{"entry":[{},"ZNF33a"]},{"entry":[{},"ZNF35"]},{"entry":[{},"ZNF43"]},{"entry":[{},"ZNF44"]},{"entry":[{},"ZNF45"]},{"entry":[{},"ZNF7"]},{"entry":[{},"ZNF76"]},{"entry":[{},"ZNF83"]},{"entry":[{},"ZNF85"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}}},"A further group of intracellular receptors are the intracellular target molecules listed in the following Table 2.",{"@attributes":{"id":"p-0187","num":"0224"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 2"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Intracellular target molecules"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"154pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"\u201clong-chain\u201d fatty"]},{"entry":[{},"acid CoA ligase"]},{"entry":[{},"\u201cmajor basic\u201d protein"]},{"entry":[{},"\u201cmixed function\u201d"]},{"entry":[{},"oxygenase"]},{"entry":[{},"11\u03b2-hydroxylase (EC"]},{"entry":[{},"1.14.15.4)"]},{"entry":[{},"18-hydroxylase"]},{"entry":[{},"1-acylglycerol-3-"]},{"entry":[{},"phosphate acyl transferase"]},{"entry":[{},"2,3-"]},{"entry":[{},"oxidosqualene lanosterol"]},{"entry":[{},"cyclase"]},{"entry":[{},"21-steroid hydroxylase"]},{"entry":[{},"(EC 1.14.99.10)"]},{"entry":[{},"24,28-sterol reductase"]},{"entry":[{},"3-hydroxybutyrate"]},{"entry":[{},"dehydrogenase"]},{"entry":[{},"3-ketothiolase"]},{"entry":[{},"3-\u03b2-hydroxysteroid"]},{"entry":[{},"dehydrogenase"]},{"entry":[{},"(EC5.3.3.1)"]},{"entry":[{},"5\u2032-nucleotidase"]},{"entry":[{},"8-oxoguanosine"]},{"entry":[{},"deglycosylase"]},{"entry":[{},"abl oncogene protein"]},{"entry":[{},"Acetolactate synthase"]},{"entry":[{},"Acetylcholine esterase"]},{"entry":[{},"Acetyl-CoA carboxylase"]},{"entry":[{},"Acetyl-CoA malate-citrate"]},{"entry":[{},"synthase"]},{"entry":[{},"Acetylglucosaminyl"]},{"entry":[{},"transferase"]},{"entry":[{},"Acetylspermine"]},{"entry":[{},"deacetylase"]},{"entry":[{},"Acetyl transcylase"]},{"entry":[{},"Aconitase"]},{"entry":[{},"Actin"]},{"entry":[{},"Adenosine deaminase"]},{"entry":[{},"Adenosyl homocysteine"]},{"entry":[{},"hydrolase"]},{"entry":[{},"Adenosyl methionine"]},{"entry":[{},"decarboxylase"]},{"entry":[{},"Adenylate cyclase"]},{"entry":[{},"Adenylate deaminase"]},{"entry":[{},"Adenylate kinase"]},{"entry":[{},"Adenylo-succinate lyase"]},{"entry":[{},"Adenylo-succinate"]},{"entry":[{},"synthase"]},{"entry":[{},"Alanine aminotransferase"]},{"entry":[{},"Aldolase"]},{"entry":[{},"Aldose reductase"]},{"entry":[{},"Alkaline phosphatase"]},{"entry":[{},"Alcohol dehydrogenase"]},{"entry":[{},"Amidophosphoribosyl"]},{"entry":[{},"amine transferase"]},{"entry":[{},"AMP"]},{"entry":[{},"phosphodiestererase"]},{"entry":[{},"Amyloid \u03b2\/A4 protein"]},{"entry":[{},"Amyloid precursor protein"]},{"entry":[{},"Ankarin"]},{"entry":[{},"Arginase"]},{"entry":[{},"Argininosuccinate"]},{"entry":[{},"synthetase"]},{"entry":[{},"Argininosuccinate lyase"]},{"entry":[{},"Aromatase"]},{"entry":[{},"Arylsulfatase"]},{"entry":[{},"Aspartate"]},{"entry":[{},"aminotransferase"]},{"entry":[{},"Aspartate"]},{"entry":[{},"transcarbamoylase"]},{"entry":[{},"ATPase"]},{"entry":[{},"ATP diphosphohydrolase"]},{"entry":[{},"bcl-2 oncogene protein"]},{"entry":[{},"Connective tissue-"]},{"entry":[{},"activating peptide"]},{"entry":[{},"C5a-inactivating"]},{"entry":[{},"factor"]},{"entry":[{},"Calcitonin"]},{"entry":[{},"Calmodulin"]},{"entry":[{},"Calpain I"]},{"entry":[{},"Calreticulin"]},{"entry":[{},"Carbamoyl phosphate"]},{"entry":[{},"synthetase"]},{"entry":[{},"Carbonate anhydrase"]},{"entry":[{},"Casein kinase 1"]},{"entry":[{},"Casein kinase 2"]},{"entry":[{},"Catalase"]},{"entry":[{},"Catechol"]},{"entry":[{},"methyltransferase"]},{"entry":[{},"Cathepsin"]},{"entry":[{},"Cathepsin B and L"]},{"entry":[{},"cdc 10"]},{"entry":[{},"cdc 13 p60"]},{"entry":[{},"cdc 2 p34"]},{"entry":[{},"cdc 25 p80"]},{"entry":[{},"Chaparonin"]},{"entry":[{},"Cholesterol esterase"]},{"entry":[{},"Cholesterol"]},{"entry":[{},"mono-oxygenase"]},{"entry":[{},"Citrate synthetase"]},{"entry":[{},"Clathrin"]},{"entry":[{},"Collagenase"]},{"entry":[{},"Cortisone dehydrogenase"]},{"entry":[{},"crk oncogene protein"]},{"entry":[{},"Cyclin A and B"]},{"entry":[{},"Cyclophilin"]},{"entry":[{},"Cytidine deaminase"]},{"entry":[{},"Cytidylate deaminase"]},{"entry":[{},"Cytochrome C peroxidase"]},{"entry":[{},"Cytochrome P450"]},{"entry":[{},"Cytosine"]},{"entry":[{},"methyltransferase"]},{"entry":[{},"dbl oncogene protein"]},{"entry":[{},"Defensin"]},{"entry":[{},"Diacyl glycerol"]},{"entry":[{},"acyltransferase"]},{"entry":[{},"Dihydrofolate reductase"]},{"entry":[{},"Dihydroorotatase"]},{"entry":[{},"Dihydroorotate"]},{"entry":[{},"dehydrogenase"]},{"entry":[{},"Dihydrouracil"]},{"entry":[{},"dehydrogenase"]},{"entry":[{},"Dioxygenase"]},{"entry":[{},"Dopamine monooxygenase"]},{"entry":[{},"Dynenin"]},{"entry":[{},"Elastase"]},{"entry":[{},"Elastin"]},{"entry":[{},"Elongation factor Tu"]},{"entry":[{},"Endo-rhamosidase"]},{"entry":[{},"Enolase"]},{"entry":[{},"Enoyl-ACP-hydratase"]},{"entry":[{},"Enoyl-ACP-reductase"]},{"entry":[{},"ets oncogene protein"]},{"entry":[{},"Ferritin"]},{"entry":[{},"Ferrodoxin"]},{"entry":[{},"Fatty acid synthetase"]},{"entry":[{},"fgr oncogene protein"]},{"entry":[{},"fps oncogene protein"]},{"entry":[{},"Fructose bisphosphate"]},{"entry":[{},"aldolase"]},{"entry":[{},"Fumarase"]},{"entry":[{},"GABA aminotransferase"]},{"entry":[{},"Galactosidase"]},{"entry":[{},"Gelatinase"]},{"entry":[{},"Gelsolin"]},{"entry":[{},"Glucophosphate isomerase"]},{"entry":[{},"Glucosylceramide"]},{"entry":[{},"galactosyl transferase"]},{"entry":[{},"Glutaminase"]},{"entry":[{},"Glutamine phosphoribosyl"]},{"entry":[{},"pyrophosphate"]},{"entry":[{},"amidotransferase"]},{"entry":[{},"Glycerol phosphate"]},{"entry":[{},"acyltransferase"]},{"entry":[{},"Glycerol phosphate"]},{"entry":[{},"dehydrogenase"]},{"entry":[{},"Glycinamide"]},{"entry":[{},"ribonucleotide"]},{"entry":[{},"transformylase"]},{"entry":[{},"GTP-binding protein"]},{"entry":[{},"Haemoglobin A"]},{"entry":[{},"Haemoglobin A1"]},{"entry":[{},"Haemoglobin Barcelona"]},{"entry":[{},"Haemoglobin Barts"]},{"entry":[{},"Haemoglobin Beth Israel"]},{"entry":[{},"Haemoglobin Bunbury"]},{"entry":[{},"Haemoglobin Cochin-Port"]},{"entry":[{},"Royal"]},{"entry":[{},"Haemoglobin Cowtown"]},{"entry":[{},"Haemoglobin Cranston"]},{"entry":[{},"Haemoglobin Creteil"]},{"entry":[{},"Haemoglobin D"]},{"entry":[{},"Haemoglobin D"]},{"entry":[{},"Los Angeles"]},{"entry":[{},"Haemoglobin D Punjab"]},{"entry":[{},"Haemoglobin F"]},{"entry":[{},"Haemoglobin Gower"]},{"entry":[{},"Haemoglobin"]},{"entry":[{},"Hammersmith"]},{"entry":[{},"Haemoglobin Hiroshima"]},{"entry":[{},"Haemoglobin Indianapolis"]},{"entry":[{},"Haemoglobin Kansas"]},{"entry":[{},"Haemoglobin Kariya"]},{"entry":[{},"Haemoglobin Kempsey"]},{"entry":[{},"Haemoglobin Kenya"]},{"entry":[{},"Haemoglobin Lepore"]},{"entry":[{},"Haemoglobin M"]},{"entry":[{},"Haemoglobin M"]},{"entry":[{},"Hyde Park"]},{"entry":[{},"Haemoglobin M Iwate"]},{"entry":[{},"Haemoglobin M Saskatoon"]},{"entry":[{},"Haemoglobin Nancy"]},{"entry":[{},"Haemoglobin Philly"]},{"entry":[{},"Haemoglobin Quong Sze"]},{"entry":[{},"Haemoglobin Raleigh"]},{"entry":[{},"Haemoglobin Ranier"]},{"entry":[{},"Haemoglobin S"]},{"entry":[{},"Haemoglobin Sealy"]},{"entry":[{},"Haemoglobin Seattle"]},{"entry":[{},"Haemoglobin St. Louis"]},{"entry":[{},"Haemoglobin St. Mande"]},{"entry":[{},"Haemoglobin Titusville"]},{"entry":[{},"Haemoglobin Torino"]},{"entry":[{},"Haemoglobin Wayne"]},{"entry":[{},"Haemoglobin York"]},{"entry":[{},"Haemoglobin Zurich"]},{"entry":[{},"Ha-ras oncogene protein"]},{"entry":[{},"Hexokinase"]},{"entry":[{},"Histaminase"]},{"entry":[{},"Histidine decarboxylase"]},{"entry":[{},"HSP 27"]},{"entry":[{},"Hydropyrimidine"]},{"entry":[{},"hydrolase"]},{"entry":[{},"Hydroxyacyl-CoA-"]},{"entry":[{},"dehydrogenase"]},{"entry":[{},"Hydroxymethylglutaryl"]},{"entry":[{},"CoA-splitting enzyme"]},{"entry":[{},"Hydroxymethylglutaryl-"]},{"entry":[{},"CoA-reductase"]},{"entry":[{},"Hydroxymethylglutaryl-"]},{"entry":[{},"CoA-synthetase"]},{"entry":[{},"Hydroxysteroid"]},{"entry":[{},"dehydrogenase"]},{"entry":[{},"Hypoxanthine-guanine-"]},{"entry":[{},"phosphoribosyl transferase"]},{"entry":[{},"IMP-dehydrogenase"]},{"entry":[{},"Indole lyase"]},{"entry":[{},"Inositol phosphate"]},{"entry":[{},"phosphatase"]},{"entry":[{},"int-1 oncogene protein"]},{"entry":[{},"Isocitrate lyase"]},{"entry":[{},"Kinin-forming enzyme"]},{"entry":[{},"Ki-ras oncogene protein"]},{"entry":[{},"Lactate dehydrogenase"]},{"entry":[{},"Lactoferrin"]},{"entry":[{},"Laminin"]},{"entry":[{},"Leukocyte elastase"]},{"entry":[{},"Lipocortin"]},{"entry":[{},"Lipoxygenase"]},{"entry":[{},"L-myc oncogene protein"]},{"entry":[{},"Lysozyme"]},{"entry":[{},"Malate dehydrogenase"]},{"entry":[{},"Malate synthase"]},{"entry":[{},"Malonyl transacylase"]},{"entry":[{},"Mannosidase"]},{"entry":[{},"met oncogene protein"]},{"entry":[{},"Methaemoglobin"]},{"entry":[{},"Methionine"]},{"entry":[{},"adenosyl transferase"]},{"entry":[{},"mos oncogene protein"]},{"entry":[{},"Myeloperoxidase"]},{"entry":[{},"Myofilament"]},{"entry":[{},"myristoyl transferase"]},{"entry":[{},"Na\/K ATPase"]},{"entry":[{},"N-acetylglucuronidase"]},{"entry":[{},"NAD-dependent sterol-4-"]},{"entry":[{},"carboxylase"]},{"entry":[{},"NADase"]},{"entry":[{},"NADPH-dependent 3-"]},{"entry":[{},"oxosteroid reductase"]},{"entry":[{},"Nexin"]},{"entry":[{},"N-ras oncogene protein"]},{"entry":[{},"Nucleolus protein B23"]},{"entry":[{},"Nucleoside diphosphate"]},{"entry":[{},"kinase"]},{"entry":[{},"Ornithine"]},{"entry":[{},"aminotransferase"]},{"entry":[{},"Ornithine"]},{"entry":[{},"carbamoyltransferase"]},{"entry":[{},"Ornithine decarboxylase"]},{"entry":[{},"Orotate decarboxylase"]},{"entry":[{},"Orotate"]},{"entry":[{},"phosphoribosyl transferase"]},{"entry":[{},"p53"]},{"entry":[{},"Peptidyl amidoglycolate"]},{"entry":[{},"lyase"]},{"entry":[{},"Peptidyl prolyl isomerase"]},{"entry":[{},"PF4"]},{"entry":[{},"Phenylalanine"]},{"entry":[{},"hydroxylase"]},{"entry":[{},"Phosphatidate phosphatase"]},{"entry":[{},"Phosphoenol pyruvate"]},{"entry":[{},"carboxykinase"]},{"entry":[{},"Phosphofructokinase"]},{"entry":[{},"Phosphoglucokinase"]},{"entry":[{},"Phosphoglucomutase"]},{"entry":[{},"Phosphoglycerate kinase"]},{"entry":[{},"Phosphoglyceromutase"]},{"entry":[{},"Phospholipase A2"]},{"entry":[{},"Phospholipase C"]},{"entry":[{},"Phospholipase CG1"]},{"entry":[{},"Phospholipase D"]},{"entry":[{},"Phospholipase S"]},{"entry":[{},"Phosphoribomutase"]},{"entry":[{},"Phosphoribosyl phosphate"]},{"entry":[{},"transferase"]},{"entry":[{},"pim oncogene protein"]},{"entry":[{},"Plasminogen activator-"]},{"entry":[{},"inhibitor"]},{"entry":[{},"Porin"]},{"entry":[{},"pRB (retinoblastoma gene"]},{"entry":[{},"product)"]},{"entry":[{},"pRb retinablastoma gene"]},{"entry":[{},"product"]},{"entry":[{},"Properdin"]},{"entry":[{},"Prostaglandin synthase"]},{"entry":[{},"Protein kinase C"]},{"entry":[{},"Purine nucleoside"]},{"entry":[{},"phosphorylase"]},{"entry":[{},"Pyruvate dehydrogenase"]},{"entry":[{},"Pyruvate kinase"]},{"entry":[{},"raf oncogene protein"]},{"entry":[{},"rel oncogene protein"]},{"entry":[{},"Ribonucleotide reductase"]},{"entry":[{},"Ribose phosphate-"]},{"entry":[{},"pyrophosphate kinase"]},{"entry":[{},"Ricin tropoelastin"]},{"entry":[{},"acid phosphatase"]},{"entry":[{},"acid protease"]},{"entry":[{},"Heavy meromyosin"]},{"entry":[{},"serine\/threonine kinase"]},{"entry":[{},"Spectrin"]},{"entry":[{},"Spermine synthase"]},{"entry":[{},"Squalene epoxidase"]},{"entry":[{},"Squalene monooxygenase"]},{"entry":[{},"src oncogene protein"]},{"entry":[{},"Sterol methyltransferase"]},{"entry":[{},"suc 1 p13"]},{"entry":[{},"Succinyl-CoA-synthetase"]},{"entry":[{},"Superoxide dismutase"]},{"entry":[{},"Tartrate dehydrogenase"]},{"entry":[{},"Thioesterase"]},{"entry":[{},"Thioredoxin"]},{"entry":[{},"Thrombospondin"]},{"entry":[{},"Thromboxane-A2-"]},{"entry":[{},"synthetase"]},{"entry":[{},"Thymidylate synthetase"]},{"entry":[{},"Transacylase"]},{"entry":[{},"Triose phosphate isomerase"]},{"entry":[{},"Triose phosphate"]},{"entry":[{},"dehyrogenase"]},{"entry":[{},"tRNA synthetase"]},{"entry":[{},"Tropomyosin"]},{"entry":[{},"Tryptophan synthase"]},{"entry":[{},"Tubulin"]},{"entry":[{},"Tyrosine kinase"]},{"entry":[{},"Ubioquinone reductase"]},{"entry":[{},"UPA"]},{"entry":[{},"Uridine monophosphate"]},{"entry":[{},"kinase"]},{"entry":[{},"Vitamin K reductase"]},{"entry":[{},"wee-1 gene product"]},{"entry":[{},"Xanthine dehydrogenase"]},{"entry":[{},"Xanthine oxidase"]},{"entry":[{},"Xylosyl transferase"]},{"entry":[{},"yes oncogene protein"]},{"entry":[{},"\u03b1-actin"]},{"entry":[{},"\u03b1-mannosidase"]},{"entry":[{},"\u03b1-melogenin"]},{"entry":[{},"\u03b1-tubulin"]},{"entry":[{},"\u03b2-actin"]},{"entry":[{},"\u03b2-glucuronidase"]},{"entry":[{},"\u03b2-glycerophosphatase"]},{"entry":[{},"\u03b2-ketoacyl-ACP-"]},{"entry":[{},"reductase"]},{"entry":[{},"\u03b2-ketoacyl-ACP-"]},{"entry":[{},"synthetase"]},{"entry":[{},"\u03b2-spectrin"]},{"entry":[{},"\u03b2-tropomyosin"]},{"entry":[{},"\u03b2-tubulin"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}}},"A further particularly preferred group of intracellular receptors are the HMG proteins, such as are described for example in the International Patent Application PCT\/EP96\/00716, and in particular the HMGA proteins. As used herein the term HMGA proteins preferably denotes overall the following proteins: HMGA1, HMGA1a, HMGA1b and HMGA2.","The HMGA proteins have a modular structure and each comprise three DNA-binding domains, which are termed \u201cAT hooks\u201d and are shown as DBD1 to DBD3 in , as well as a very acidic C-terminal region. It is obvious to the person skilled in the art that antagonists which bind to one of the \u201cAT hooks\u201d recognise not only the HMGA1 proteins and thus the two splice variants HMGA1A and HMGA1B (see ), but also exhibit cross reactivity with similar DNA-binding molecules such as HMGA2. Apart from HMGA2, many further proteins also have sequences similar to the \u201cAT hooks\u201d and form in each case further receptors. Such proteins are listed inter alia in Table 3:",{"@attributes":{"id":"p-0190","num":"0227"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 3"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Column 1: Protein data bank-Access codes; Column 2: Protein designation"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"217pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Q9UKB0","Human HMG-Protein-R"]},{"entry":["Q9UKY1","ZHX1_Human Zinc finger- and Homoeobox-Protein 1"]},{"entry":["P55198","AF17_HUMAN AF-17 Protein [MLLT6]"]},{"entry":["Q59F28","Human Trithorax Homologon (Fragment)"]},{"entry":["Q6PJQ2","Human ZNF406 Protein (Fragment)"]},{"entry":["Q75PJ9","Human ZFAT-1 Protein"]},{"entry":["Q75PJ7","Human ZFAT-3 Protein"]},{"entry":["Q75PJ6","Human TR-ZFAT Protein"]},{"entry":["Q9ULG1","Human KIAA1259 Protein"]},{"entry":["Q9NUK2","Human Hypothetical Protein FLJ11314"]},{"entry":["Q9NTG6","Human Hypothetical Protein DKFZp434B0616"]},{"entry":["Q8IX01","SFR14_HUMAN Presumed Splice Factor"]},{"entry":["Q9H5J8","Human Hypothetical Protein FLJ23363"]},{"entry":["Q6I9Y6","Human MGC5306 Protein"]},{"entry":["Q8IX01-2","Splice Isoform 2 of Q8IX01"]},{"entry":["Q8IX01-3","Splice Isoform 3 of Q8IX01"]},{"entry":["Q8IX01-4","Splice Isoform 4 of Q8IX01"]},{"entry":["Q15291","RBBP5_HUMAN Retinoblastoma-binding Protein 5 (RBBP-5)"]},{"entry":["P51608","MECP2_HUMAN Methyl-CpG-binding Protein 2"]},{"entry":["Q6IPE2","Human FLJ12800 Protein"]},{"entry":["Q6QHH9","Human Methyl-CpG-binding Protein 2, Isoform B"]},{"entry":["Q9H8H4","Human Hypothetical Protein FLJ13629"]},{"entry":["Q7Z384","Human Hypothetical Protein DKFZp686A24160"]},{"entry":["O42043","ENK7_HUMAN HERV-K_1q23.3 Provirus"]},{"entry":["P61569","ENK16_HUMAN HERV-K_10p14 Provirus"]},{"entry":["Q86VM3","Human MYB binding Protein 1a [MYBBP1A]"]},{"entry":["Q9UNW3","Human Coat Protein RIC-2"]},{"entry":["Q9BWE0","Human REPIN1 Protein (Hypothetical Protein ZNF464)"]},{"entry":["Q9ULL5","Human KIAA1205 Protein"]},{"entry":["Q9NZH2","Human Dhfr Oribeta-binding Protein RIP60"]},{"entry":["Q9NZI3","Human Linens epithelium-containing growth factor p52"]},{"entry":["Q9NY27","Human Regulatory Sub-Unit 2 of Proteinphosphatase-4"]},{"entry":["Q86U91","Human HMGA2\/RAD51L1 Fusion protein"]},{"entry":["O95368","Human Transcriptional Coactivator p52"]},{"entry":["Q9P015","Human HSPC145 (Mitochondrial Ribosome protein L15)"]},{"entry":["Q5U071","Human HMG Protein \u2018box 2\u2019"]},{"entry":["Q9H0Y1","Human Hypothetical Protein DKFZp564I206"]},{"entry":["Q6ZP45","Human Hypothetical Protein FLJ26517"]},{"entry":["P17096-2","Splice Isoform HMG-Y of P17096 [HMGA1]"]},{"entry":["Q9Y6X0","SETBP_HUMAN SET-binding Protein (SEB) [SETBP1]"]},{"entry":["Q8TEK3","DOT1L_HUMAN Histone-Lysine N-Methyltransferase"]},{"entry":["Q8TEK3-2","Splice Isoform 1 from Q8TEK3 [DOT1L]"]},{"entry":["Q03164","HRX_HUMAN Zinc finger-Protein HRX (ALL-1)"]},{"entry":["Q86YP1","Human Transcription factor MLL UPN96240"]},{"entry":["Q86YN9","Human Transcription factor MLL UPN95022"]},{"entry":["Q03164-2","Splice Isoform 4P-18B from Q03164 [MLL]"]},{"entry":["P04920","B3A2_HUMAN Anion Exchanger Protein 2"]},{"entry":["Q59GF1","Human Anion Exchanger-2 type a-variant"]},{"entry":["Q8TAG3","Human SLC4A2 Protein"]},{"entry":["Q6P391","Human PSIP1 Protein"]},{"entry":["O75475","Human Linens epithelium-containing growth factor p75"]},{"entry":["Q9UEY6","Human Anion exchanger-2 type a [SLC4A2]"]},{"entry":["Q9UEY5","Human Anion exchanger-2 type b2 [SLC4A2]"]},{"entry":["Q9UEY4","Human Anion exchanger-2 type b1 [SLC4A2]"]},{"entry":["Q9UER6","Human Transcriptional coactivator p75"]},{"entry":["O00256","Human DFS70"]},{"entry":["P04920-2","splice Isoform B1 of P04920 [SLC4A2]"]},{"entry":["Q9BTB1","Human Hypothetical Protein MGC10561"]},{"entry":["Q9UKB0","Human HMG Protein-R"]},{"entry":["O43167","ZBT24_HUMAN Zinc finger- and BTB-domain-containing protein"]},{"entry":["Q8N455","Human ZBTB24 Protein [ZBTB24]"]},{"entry":["Q5TED5","Human Zinc finger-Protein 450 [ZNF450]"]},{"entry":["Q96CK0","Human Zinc finger-Protein 653"]},{"entry":["Q96AS7","Human Zinc finger-Protein 653"]},{"entry":["P51888","PRELP_HUMAN Prolargine Precursor"]},{"entry":["Q5JPC9","Human Hypothetical Protein DKFZp667H216"]},{"entry":["Q6FHG6","Human PRELP-Potein"]},{"entry":["Q6ZR44","Human Hypothetical Protein FLJ46672"]},{"entry":["Q8NEZ4","MLL3_HUMAN Myeloid\/lymphoid-Leukaemia protein 3 Homologon"]},{"entry":["Q96AC6","KIFC2_HUMAN Kinesine-like Protein KIFC2"]},{"entry":["Q9C0H5","K1688_HUMAN Protein KIAA1688"]},{"entry":["P52926","HMGIC_HUMAN HMG Protein I-C"]},{"entry":["Q9UKV3","ACINU_HUMAN Inductor of apoptotic Chromatin condensation"]},{"entry":["Q59F82","Human C21orf2-Protein variant"]},{"entry":["Q5VYT7","Human OTTHUMP00000021181"]},{"entry":["Q96M56","Human Hypothetical Protein FLJ32810"]},{"entry":["Q69YJ6","Human Hypothetical Protein DKFZp667N107"]},{"entry":["Q8NEY3","SPAT4_HUMAN Spermatogene-associated Protein 4"]},{"entry":["Q12809","KCNH2_HUMAN Potassium Potential-controlled Ion channel Sub-family"]},{"entry":["Q8IYY4","Human protein similar to the DAZ-interacting protein 1 [DZIP1L]"]},{"entry":["Q6ZN04","Human Hypothetical Protein FLJ16544"]},{"entry":["Q5SXN7","Human Serologically defined colon cancer antigen 3"]},{"entry":["Q8IVG2","Human KIAA2009 Protein (Fragment) [RKHD3]"]},{"entry":["Q75VX8","Human KIAA2038 Protein (Fragment) [KIAA2038]"]},{"entry":["Q12809-2","Splice Isoform 2 von Q12809 [KCNH2]"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"Against this background the present invention also relates to L-nucleic acids and in particular spiegelmers, which are directed against any of the target molecules mentioned in Tables 1 to 3.","Since the L-nucleic acid is used as an intracellularly active agent, in particular within a cell, in order to bind there to an intracellular receptor, intracellularly different forms of the interactions between the intracellular receptor and its interaction partners can be influenced. Depending on the type of interaction partners of the intracellular receptor, the intracellular use of L-nucleic acids thus enables interactions of proteins, nucleic acids, lipids, carbohydrates, or combinations of proteins, nucleic acids, lipids, carbohydrates with one another and between one another to be influenced.","In connection with the use according to the invention of a L-nucleic acid, in particular a spiegelmer, as intracellular agent and the method for binding an intracellular receptor, it should be noted that this preferably relates to an in vitro application and to an in vitro method.","In connection with the use according to the invention of a L-nucleic acid, in particular a spiegelmer, for the production of a medicament for the treatment and\/or prevention of a disease and\/or for the production of a medicament for diagnostic purposes, the target molecule is an intracellular target molecule. In this connection the intracellular target molecule is one that is causily or non-causily involved in the disease or illness to be prevented, treated or diagnosed, but in any case its binding to a L-nucleic acid that binds specifically thereto means that, in the case of a medicament, the disease is alleviated, prevented or cured, and\/or in the case of a diagnostic agent the disease or a predisposition thereto can be established or diagnosed. As used herein the concept of diagnosis include an initial diagnosis as well as subsequent diagnoses, in particular diagnoses or investigations in order for example to follow or to determine the progression of the disease or the stages of the disease. It is within the scope of the invention that the target molecule is an intracellular receptor as described herein, in particular a transcription factor, an intracellular target molecule or an HMG protein. Within the scope of the present invention it is most particularly preferred if the target molecule is present intracellularly, i.e. within a cell, and the interaction having an influence on the disease and\/or diagnosis takes place intracellularly between the L-nucleic acid and in particular the spiegelmer, and the target molecule, i.e. the receptor. It is also within the scope of the present invention if the target molecule is present outside a cell and the interaction between the L-nucleic acid and in particular the spiegelmer, and the target molecule, i.e. the receptor, takes place extracellularly.","The indications for use of the medicament produced using an L-nucleic acid, in which the nucleic acid is directed against an intracellular target molecule, follow for the person skilled in the art from the involvement of the intracellular target molecule in the respective pathogenicity mechanism on which the indication is based. Thus, it is known for example for HMGA proteins that these are associated with carcinomas (inter alia of the breast, lungs, skin, thyroid) as well as leukaemias and lymphomas and other malignant tumours, such as inter alia sarcomas (rhabdomyosarcoma, osteosarcoma). Also, HMGA proteins are expressed in many types of mesenchymal tumours, including inter alia hamartomas (breast and lungs), fatty tissue tumours (lipomas), pleomorphic adenomas of the salivary glands, uterine leiomyomas, angiomyxomas, fibroadenomas of the breast, polyps of the endometrium and atherosclerotic plaques. HMGA is an interesting therapeutic target. Blockade of HMGA could be a suitable starting point for controlling cancer and preventing its metastatic spread. As described in detail herein, L-nucleic acids directed against HMGA proteins are also suitable for the diagnosis and\/or treatment of virus diseases and arteriosclerosis on account of the involvement of HMGA proteins in the regulation of the transcription of a large number of viral genes or the marked expression of HMGA and in particular HMGA1 in the tissues affected by arteriosclerosis, which is associated with neointimal, vascular smooth muscle cells, macrophages and new blood vessels.","Although\u2014as has been surprisingly found by the present inventors\u2014nucleic acids, preferably L-nucleic acids and particularly spiegelmers, are able as such to penetrate a phospholipid double membrane such as a cytoplasmic membrane and then to be intracellularly functional in the sense of the specific interaction with the intracellular receptor, the effectiveness of the infiltration of the L-nucleic acid can be influenced and in particular enhanced by the use of various techniques. These techniques include the use of chemical compounds or molecules as well as the use of physical measures. Irrespective of the type, these techniques are herein generally referred to as delivery vehicles. It is within the scope of the present invention that the inventors have likewise established that aptamers too exhibit this property, and like the spiegelmers can similarly be used involved together with the composition according to the invention for basically the same purposes, applications and uses.","In the use of chemical compounds and molecules, a further distinction is whether the nucleic acid needs to be modified or not for the delivery. A modification for the purposes of using a delivery vehicle is generally not necessary if the delivery vehicle is or comprises a vesicle, such as for example in the case of liposomes, polypeptide vehicles, cyclodextrins, dendrimers, nanoparticles and microparticles, and also polyethyleneimine. A modification for the purposes of using a delivery vehicle is on the other hand normally necessary if the delivery vehicle uses receptor-mediated endocytosis, fusogenic peptides, signal peptides or lipophilic conjugates. The group of physical techniques includes in particular electroporation and iontophoresis. It will be recognised that further techniques for transporting a compound through a phospholipid double membrane such as a cytoplasmic membrane are known to the person skilled in the art in this field, which in principle are also suitable for the transfer of a functional nucleic acid, such as for example an aptamer and\/or a spiegelmer.","The individual delivery vehicles which can be used within the scope of the various aspects of the present invention will be described in more detail hereinafter.","Liposomes consist of artificial cationic lipids such as N-[1-(2,3-dioleoyloxy)propyl]-N,N,N-trimethylammonium chloride (DOTMA) and N-[1-(2,3-dioleoyloxy)propyl]-N,N,N-trimethylammonium sulfate (DOTAP), in which the cationic groups interact with the negatively charged nucleic acids and neutralise their anionic charge. The transport takes place via endocytosis (PNAS, 93:11493-11498, 1996). However, cationic liposomes are cytotoxic, especially in higher concentrations, which restricts their use in vitro and in vivo (Biochem Biophys Res Commun, 197:818, 1993; Biochem Biophys Res Commun, 1372:55-68, 1998). On the other hand the amphiphilic pyridinium-based lipid SAINT-2 is a non-toxic formulation (Nucleic Acids Res, 29:2079-2087, 2001). Also, pH-sensitive liposomes are a possible alternative, which consist of amphiphatic molecules such as cholesteryl hemisuccinate (CHEMS) and dioleyl phosphatidyl ethanolamine (DOPE) (J Pharmacol Exp Ther, 297:1129-1136, 2001). Widely differing formulations of liposomes can be found in the review articles by Dass and Torchili (Drug Delivery, 9:169-180, 2002; Nat Rev Drug Disc, 4:145-160, 2005).","With receptor-mediated endocytosis (RME) transport mechanisms which are already present in the cell membrane are utilised. For this purpose the nucleic acid is coupled for example via a poly-L-lysine (PPL) linker covalently to a transporter protein (\u201ccarrier\u201d protein) The choice of transporter protein depends in this connection on the ability to bind to specific receptors of the cell membrane and to accumulate in the cell by endocytosis. A cell-specific transport can thus be realised. For example, an antisense phosphorothioate directed against c-myc could be introduced into M-14 human melanoma cells (Anticancer Res, 17:29-35, 1997). However, an effective transport by means of RME depends in this case not only on the affinity of the receptor for the ligand, but also on the limitation of the selected receptor as regards the cells\u2014especially in vivo. Furthermore the selected ligand must be inactive or have an enhancing effect as regards the therapeutic result, in order to avoid a possible toxicity of the transport vehicle. Thus, the selection and the ubiquitous propagation of the selected receptor in vivo is decisive for a successful RME-based transport. Moreover, a sequestration of nucleic acids in endosomal compartments has been observed in RME-based transport, which would appear to make this method not very promising for an intracellular transport or an intracellular release or delivery. Most important of all, the coupling between the receptor and nucleic acid must be chosen so that the function of one or other is not reduced (J Pharmaceutical Science, 92 (8):1559-1573, 2003).","Fusogenic peptides have been used to enable peptide-oligonucleotide conjugates to fuse with the cell membrane and thus effect the transport in the cell (Bioconjug Chem, 9: 466-475, 1998; Bioconjug Chem, 6:43-53, 1995; Nucleic Acids Research, 25:2730-2736, 1997).","The selected import of nuclear proteins from the cytosol into the nucleus is mediated by short peptide sequences, which are termed nuclear localisation signals (NLS). Thus, various NLS peptide derivatives can be used in order to transport nucleic acids into the nucleus (Bioconjug Chem, 10:1005-1012, 1999; Bioconjug Chem, 10:598-606, 1999; Bioconjug Chem, 6:101-108, 1995). In addition there are also so-called signal import peptides (IP), which can promote the cellular uptake of nucleic acids and could be derived for example from Kaposi's fibroblast growth factor (K-FGF) (Adv Drug Deliv Rev, 44:35-49, 2000).","Vesicles similar to viral capsids can be formed by blocks of polypeptides, which can serve as possible transport vehicles for an intracellular transport (Nat Materials, 3 (4):244-8, 2004).","The hydrophilic character of oligonucleotides and the anionic phosphodiester backbone reduce the cellular permeation. Lipophilic conjugates are therefore one possible way of increasing the ability of oligonucleotides to bind to lipoproteins and thereby improve intracellular delivery. The conjugate that has been most thoroughly investigated is cholesterol (Antisense and Nucleic Acid Drug Development, 12:103-128, 2002).","Cyclodextrins are cyclic oligosaccharides, which have a central hydrophobic cavity and multiple hydroxyl groups on the outer surface. Cyclodextrins have therefore already been used for the transport of anti-sense oligonucleotides in human T cell lines (Antisense Res Dev, 5:185-192, 1995) and have also been used in vivo for intracellular transport and for intracellular release or delivery of immunogenic CpG sequences (Biochem Pharmacol, 52:1537-1544, 1996). A wide variety of formulations of cyclodextrins are given in the review article by Davis and Brewster (Nature Reviews Drug Discovery 3:1023-1035, 2004).","Dendrimers are highly branched macromolecules, which are composed of repetitive units of, typically, polyamides. The molecules carry functional groups such as primary amino groups on their surface, which interact with other molecules by electrostatic interaction. A complex structure formation thus takes place rapidly and in a highly reproducible manner, which leads to complexes of low cytoxicity (Nucleic Acids Research, 28:4225-4231, 2000; Clin Cancer Res, 7:3606-3612, 2001).","Cyanacrylate nanoparticles have been tested since the beginning of the 1990s for the release or delivery of oligonucleotides. The interaction of oligonucleotides with the nanoparticles takes place through ion pairs of the anionic charge of the oligonucleotides with various hydrophobic cations, principally with charged nanoparticles. Polyisohexyl cyanoacrylate (PIHCA), polyisobutyl cyanoacrylate (PIBCA) or polyhexyl cyanoacrylate (PHCA) are commonly used for the formation of nanoparticles, although a large number of lipophilic cation-oligonucleotide pairs have also been tested (Pharm Res., 1:1370-1378, 1994; PNAS, 91:10460-10464, 1994; Pharm Res, 9:441-449, 1992). Also, nanoparticles have already been employed for in vivo use (Biochem Biophys Res Commun, 279:401-406, 2000; Pharm Res, 13:38-43, 1996).","Microparticles or so-called microspheres are typically formed from biodegradable polymers such as poly(d,l-lactide-co-glycolides [P(LA-GA)] and are used for the delayed release of oligonucleotides (J Pharm Sci, 91:790-799; 2000; J Controlled Release, 69:197-207, 2000; J Drug Target, 5:291-302, 1998).","Electroporation is a transport technology, which uses a strong electric field in order to destabilise the lipid double membrane, and thereby permeabilise the cell membrane and thus effect a transport of the substance to be administered, which can also be present in ionised form, into the cell (iontophoresis). Electroporation has already been successfully used in order to effect transdermal transport of oligonucleotides ex vivo as well as in vivo (Int J Pharm, 184:147-156, 1999; J Drug Target, 5:275-289, 1998; Pharm Res, 15:1596-1602, 1998; Int J Cancer, 85:260-266, 2000; Biochem Biophys Res Commun, 212:286-292, 1995; Blood, 88:731-741, 1996).","The uptake of \u201cnaked\u201d oligonucleotides into cells can be improved in vitro and ex vivo by the use of high pressure. The need for closed systems in order to use this technology means that it can only be used for ex vivo applications (PNAS, 96:6411-6416, 1999; Hum Gene Ther, 10:2355-2664, 1999).","Also, the use of shockwaves, acoustic high pressure pulses, effects the transport of oligonucleotides into cells (J Mol Med, 79:306-313, 2001; Cancer Res, 58:219-221, 1998). Ultrasound is an acoustic technology comparable to shockwaves, but employs higher frequencies (MHz instead of Hz) and shorter application times (from seconds to minutes), and has already been used in a supporting role in gene therapy techniques (Hum Gene Ther, 7:1339-1346, 1996; Invest Radiol, 32:723-727, 1997; Ultrasound Med Bio, 25:1451-1457, 1999).","In a further aspect of the present invention a new delivery vehicle is provided, which is suitable in particular for the transport of functional nucleic acids such as aptamers, preferably functional L-nucleic acids, and most particularly preferably spiegelmers. The delivery vehicle is in this case a micelle-like or liposome-like structure based on polyethyleneimine. Without wishing to be too specific in the following description, the present inventors start from the assumption that the nucleic acid is present embedded or contained in the micelle-like or liposome-like structure. Polyethyleneimine can in principle be present and also used as linear or branched polyethyleneimine, polyethyleneimine in the branched form being particularly preferred. Moreover, polyethyleneimine can exist and can also be used as high molecular weight or low molecular weight polyethyleneimine. Preferably high molecular weight polyethyleneimine has a molecular weight of about 800 kDa and low molecular weight polyethyleneimine has a molecular weight of about 3 kDa. Within the scope of the present invention a polyethyleneimine with a mean molecular weight of about 25 kDa is preferred, a branched polyethyleneimine with a molecular weight of about 25 kDa being particularly preferred.","Although it is not essential for an effective implementation, it is nevertheless preferred if in the delivery vehicle according to the invention the nucleic acid itself to be delivered also carries a modification. In this connection it is preferred if the modification is selected from the group comprising PEG residues. It is furthermore preferred if the PEG residue has a molecular weight of about 1000 to 10000 Da, preferably about 1200 to 5000 Da, more preferably about 1500 to 2500 Da and most particularly preferably about 2000 Da.","When mixing the nucleic acid with the delivery vehicle to produce a composition according to the invention, the ratio of the total number of nitrogen groups of the polyethyleneimine to the total number of phosphate groups of the nucleic acid to be delivered via or packaged with the delivery vehicle is adjusted to about 1 to 20, preferably about 1.5 to 10, more preferably 2 to 5, and most particularly preferably about 2 to 3.","The delivery vehicle according to the invention thus enables the mechanism of intracellular transport of nucleic acids via condensation or packing with charged particles or reagents and associated change in the charge of the overall complex, to be used also for functional nucleic acids such as aptamers, and in particular L-nucleic acids such as spiegelmers. This complex is readily taken up through endocytosis and thereby passes into the cytosol of the cell. A disadvantage of this method is the stability of the DNA\/RNA and the release of the nucleic acid from the endosomal compartment. In the cytosol of the cell a lysosome is rapidly formed from the tightly constricted endosome due to the introduction of proteases or nucleases and by protonation of the compartment. There nucleases break down the nucleic acids. This does not apply however to spiegelmers, since due to their unnatural configuration these are nuclease-stable. Also, nucleic acids are not stable in the acidic environment of the lysosome. However, this is more true of nucleic acids synthesised from DNA, and less true of nucleic acid from RNA. The whole complex is rapidly transported out of the cell again by exocytosis and breakdown in the Golgi apparatus, and accordingly only a few nucleic acids pass into the cell. One of the challenges which a suitable transfection system has to overcome is thus the stabilisation as well as the release of the nucleic acid from the endosomes into the cytosol. As regards stability, RNA spiegelmers have ideal properties for a transfection of eukaryotic cells, since being enantiomers they are not split by enzymes.","The use according to the invention of L-nucleic acids and in particular in connection with the composition according to the invention is important specifically for this class of active substances, since their action mechanism is based on a stoichiometric approach and not on a catalytic approach, in which the intracellular release of just a few molecules is already sufficient to achieve the desired effect. To this extent the present invention satisfies a need that was not hitherto met by the techniques of the prior art.","The transfection system according to the invention that is provided and elaborated by the delivery vehicles according to the invention is based on the formation of micelles from nucleic acids and branched polyethyleneimine (PEI). The phosphodiester backbone of the nucleic acids interacts with the free nitrogen positions of the PEI and forms small micelles through cross-linking, which have a positive charge on account of the PEI. These micelles are readily taken up as endosomes from a cell by constriction of the plasma membrane. The PEI now buffers inflowing protons, as a result of which many chloride ions in the interior of the endosome lead to a swelling of the compartment on account of the osmotic pressure. This effect of PEI is described in the literature as the proton sponge effect, and ultimately leads to the rupture of the endosome and the release of the spiegelmers into the cytosol. (Pharm Res, 22 (3): 373-80, 2005; Eur J Cell Biol 83 (3): 97-111, 2004; Gene Ther 9 (24):1700-7, 2002).","It is within the scope of the present invention to apply the composition according to the invention as an aerosol.","In addition spiegelmers can be derivatised with signal peptides for intracellular as well as intranuclear delivery, and also for organ-specific delivery. A coupling of signal peptides directly to the polyethyleneimine can be used for a targeted localisation in organs or within the cell.","In another further aspect the present invention relates to L-nucleic acids, in particular spiegelmers and more preferably RNA spiegelmers, which are directed against HMGA proteins. The spiegelmers disclosed herein directed against HMGA proteins are in particular examples of the knowledge, likewise forming the basis of the present invention, that L-nucleic acids and in particular spiegelmers are able to overcome a phospholipid double membrane or a cytoplasmic membrane of a cell and bind intracellularly with the intracellular receptor, for the specific binding to which they have been selected. As regards the configuration of the HMGA proteins and the L-nucleic acids directed against the latter, the comments made herein regarding the intracellular use of L-nucleic acids also apply in connection with the present aspect of the invention (and vice-versa), and is referred to again at this point in order to avoid unnecessary repetitions.","The HMG (high mobility group) family of DNA-binding phosphoproteins are present as non-histone components of chromatin throughout mammalian cells (Grosschedl et al. 1994). The basic HMG proteins are sub-divided into three different families\u2014HMGB (formerly HMG-1\/-2), HMGN (formerly HMG-14\/-17), and the HMGA family (formerly HMG-I\/Y\/C). Each HMG family has its characteristic functional sequence motif: the \u201cHMG box\u201d (HMGB family), the \u201cnucleosomal binding domain\u201d (HMGN family), and the \u201cAT hook\u201d (HMGA family).","According to the current state of knowledge the HMGA family comprises two genes, HMGA1 and HMGA2. Three different proteins can be expressed by alternative splicing by HMGA1, (HMGA1a [formerly: HMG-I], HMGA1b [formerly: HMG-Y], HMGA1c [formerly: HMG-I\/R]), whereas only one protein (HMGA2 [formerly: HMGI-C]), can be expressed by HMGA2. HMGA1a, HMGA1b and HMGA2 are polypeptides of approximately 100 amino acid length and have a modular sequence organisation: they possess three strongly basic regions (\u201cAT hook\u201d), which bind the narrow small channels of double-stranded AT-rich DNA (Reeves & Nissen 1990). The C-terminus on the other hand contains many acidic amino acids. The proteins do not have a stable secondary structure when free in solution, and only adopt a defined conformation when they are present in the complex with DNA or other proteins (Huth et al 1997). HMGA proteins belong to the most strongly modified proteins in the mammalian cell nucleus and are phosphorylated, acetylated and methylated (Reeves & Beckerbauer 2001).","The HMGA proteins per se do not have any transcriptional activity, but being so-called architectonic transcription factors they organise through their protein-protein and protein-DNA interactions the formation of the nucleoprotein-DNA transcription complex (Wolffe 1994). They thus exert a regulatory activating or inhibitory influence on the expression of a large number of genes. The most prominent example of a positive regulation is the involvement of HMGA1 in the regulation of IFN-\u03b2 (Thanos & Maniatis, 1992). Thus, for example in the case of the IFN-\u03b2 promoter HMGA1b stimulates the binding of NF-\u03baB and ATF-2 to the DNA double helix and at the same time alters the DNA structure in such a way that NF-\u03baB and ATF-2 can interact with one another and presumably also with the rest of the transcription machinery (Thanos & Maniatis 1992, Du et al 1993). A further transcription-activating effect in connection with arteriosclerotic pathogenesis is the CD44 gene regulation induced by HMGA1 (Foster et al 1998). CD44 is a cell surface glycoprotein and is involved in the migration and proliferation of smooth muscle cells after endothelial damage (Jain et al 1996, Cuff et al 2001). The transcriptional regulation of CD44 is induced by the binding of c-Fos and c-Jun to the AP-1 binding site in the CD44 promoter and is strengthened by the binding of HMGA1. Investigations in rats has shown that due to CD44 over-expression, there is an intensified recruitment of smooth muscle cells, which has a direct influence on the formation of arteriosclerotic lesions (Pellacani et al 1999; Foster et al. 1998; 2000).","Investigations on the expression of the HMGA1 gene localised in the chromosomal band 6p21.3 and of the HMGA2 gene localised in the region 12q14-15 showed that these are mainly active in processes of cell differentiation.","Accordingly, a strong expression of these genes can be found during embryo development and in undifferentiated cells (Chiappetta et al 1996) as well as in growth factor-stimulating cells (Friedman et al 1993; Johnson et al 1990; Ogram et al 1995; Holth et al 1997). In adult, differentiated tissue, HMGA1 is strongly expressed only in the retina, while HMGA2 is not found at all in the other tissues and HMGA1 is found only in very low concentrations (Bussemakers et al 1991; Chiappetta et al 1996; Rogalla et al 1996; Zhou et al 1995; Chau et al 2000). A reactivated expression of HMGA proteins in differentiated normal tissue is at the same time associated with the growth and differentiation of adipocytes (Zhou et al 1995; Anand & Chada 2000; Melillo et al 2001), the proliferation of smooth muscle cells in the blood vessels after vascular damage (Chin et al 1999), in the immune response in inflammatory reactions (Pellacani et al 1999), as well as in apoptotic processes (Diana et al 2001; Sgarra et al 2003). The amount of HMGA1 varies in this connection depending on the proliferation rate of the cells (Johnson et al 1990).","During the course of embryo development the HMGA1 expression is concentrated on specific organs of ectodermal, mesodermal or endodermal origin, whereas HMGA2 is restricted to mesenchymal tissue. Up to now no information exists concerning the phenotype of HMGA1 knockout mice, possibly because the lack of this factor has damaged embryo development too severely. HMGA2 knockout mice on the other hand exhibit dwarfism and have particularly little fatty tissue (Zhou et al 1995) and furthermore are resistant to diet-induced obesity (Anand & Chada 2000).","Finally, HMGA2 and HMGA1b expression is not detectable in the fatty tissue of normal mice, but is dramatically increased in the fat of fatty or diabetic mice (Chada et al. 2004), which points to a connection between adiposity\/obesity and HMGA expression.","Over-expression of HMGA1 influences in particular (Reeves et al 2001):\n\n","Abnormal regulation of HMGA1 could therefore lead to general alterations of gene expression and thereby contribute significantly to the formation of transformed and\/or metastatic phenotypes.","HMGA protein appear to play different roles in mesenchymal and epithelial tumours: in malignant epithelial tumours HMGA expression is associated rather with later stages of carcinogenesis, whereas benign tumours\u2014more often rarely converting mesenchymal tumours\u2014already express HMGA in early hyperplasia. This points to the fact that HMGA proteins in tissues of different embryonic origin fulfil different functions, from which also directly follows the corresponding uses of the L-nucleic acids according to the invention in the diagnosis and\/or treatment of corresponding diseases, as is also illustrated in more detail hereinafter.","The expression of HMGA1 in various human and animal neoplasms was investigated in animal models. The role of HMGA1 was demonstrated in animal models of tumourigenesis (Leman et al 2003; Ram et al 1993) as well as neoplastic progression (Bussemakers et al 1991; Nestl et al 2001; Ram et al 1993).","Raised expression of the HMGA1 gene has been demonstrated in the following carcinomas\n\n","The precise reason for the increased expression and the role of the HMGA1 gene in the pathogenesis of the tumour and the process of metastasis has still not been fully clarified. Various studies indicate however that the strength of the HMGA1 expression by the respective tumour as a prognostic marker correlates with its metastasing potential and thus represents a characteristic feature of a malignant transformed cell (Giancotti et al 1987).","Further HMGA1-associated\u2014in this case benign, mesenchymal tumours\u2014are characterised by chromosomal changes in the chromosomal HMGA1 region 6p21.3. Such aberrations have up to now been described inter alia in\n\n","Aberrations in the genetic mechanisms which control growth and proliferation are the primary cause of carcinogenesis. The expression of HMGA proteins is strongly associated with tumour development, as has been shown in a number of articles and papers (Giancotti et al. 1987, 1989, 1993). Thus, a significant HMGA2 expression was found in chemically or virally caused tumours as well as in spontaneously occurring tumours, whereas this protein could not be detected in non-transformed cells or healthy tissue (Giancotti et al. 1989). In accord with this, in the case of cells infected with oncogenic retroviruses in which the synthesis of HMGA2 expression had been specifically blocked, various phenotype markers for transformation were absent (Berlingieri et al. 1995).","The key role of HMGA proteins in normal as well as pathological growth has been elucidated in mouse models: HMGA2 knockout mice exhibit stunted growth, i.e. the animals are ca. 60% smaller than wild type mice. These dwarf mice however have a high resistance to chemically induced skin tumours.","In the last few years structural aberrations of the chromosome region 12q14-15 involving the HMGA2 gene have been found with the aid of cytogenetic investigations for a whole number of benign tumours of mesenchymal origin, these being the largest group of harmless neoplasias in man. Despite a large number of aberrations (Schoenmakers et al 1995; Kottickal et al 1998; Klotzb\u00fcchel et al 1999) the altered forms nevertheless always exhibit a common feature: they retain all three DNA-binding domains, but at the same time lose the acidic C-terminal domain as well as, at the RNA level, the information of the 3\u2032 UTR.","Such changes have already been found for many (mostly benign) mesenchymal HMGA-associated tumours:\n\n","The main cause of mortality and morbidity in cancer patients is the metastatic spread of the primary neoplasm in the body. Metastasis is not a simple process, since a successful colonisation of distant organs by disseminated neoplastic cells has to pass through many stages. Neoplastic cells have to be released from the primary neoplasm, enter the bloodstream, extravasate to distant sites, and finally proliferate again in the parenchyma of the corresponding organ. Many genes which express proteins such as proteases, adhesion molecules, motility factors and angiogenic factors are involved in the various stages of this highly complex, metastatic cascade.","Which of these genes is ultimately decisive as regards metastasis is not known. The HMGA1 gene, being one of the most important factors controlling this process, is however a likely candidate. The gene products of HMGA1 influence the transcription of many genes that are important for successful metastasis. For example, it has already been shown that other metastasis-associated genes are themselves expressed at a reduced level in suppression of HMGA1 expression (Battista 1998; Vallone 1997).","HMGA1 is therefore an important therapeutic target molecule. The blockade of HMGA1 is thus in principle suitable for controlling the cancer and preventing its metastatic spread (Evans 2004; Sgarra 2004). Thus for example, by using antisense RNAs directed against HMGA transcripts, cell proliferation in cancer cells has been reduced in vitro or the cells have even undergone apoptosis (Masciullo 2003; Scala 2000; Chau 2003). It has been shown in animal models that the growth of various pancreatic cancer xenografts is dramatically reduced by gene therapy (adoenoviral expression of antisense RNAs directed against HMGA transcripts) (Trapasso et al 2004).","HMGA1 could furthermore be used as a prognostic diagnostic marker in order to determine which patients would benefit from an aggressive cancer treatment. There is a close correlation between the degree of the malignant transformation and the amount of expressed HMGA1. This can in turn be correlated with a poor prognosis in many types of human cancer, such as prostate cancer (Tamimi 1996; Bussemakers 1991) and colorectal carcinoma (Abe 1999) and neuroblastoma (Giannini 2000).","HMGA proteins are used by many viruses as well as by control factors provided by the host cell for the expression of viral genes or as co-factors, inter alia by\n\n","In particular HMGA proteins are involved in the regulation of the transcription of a large number of viral genes in a host cell. Examples of this are the regulation of the expression of the early and late expressed genes of the human papovavirus JC (Leger et al. 1995), regulation of the EBNA1 (Epstein-Barr virus nuclear antigen 1) gene of the Epstein-Bar virus (EBV), which is jointly responsible for controlling viral latency (Schaefer et al. 1997), regulation of the IE-3 (immediate-early) gene of the Herpes simplex Virus-1 (HSV-1), which codes the prematurely expressed protein ICP4 (Panagiotidis et al. 1999), regulation of the promoter 2, active during the latency phase, of HSV-1 (French et al. 1996) and regulation of the LTR (long terminal repeats) promoter of the humane HIV-1 virus (Henderson et al 2000).","The requisition of HMGA by the host cell in the context of viral diseases is not only restricted to viral gene regulation. HMGA1 also appears to play a decisive role as architectonic co-factor in the integration of the viral DNA of the HIV-1 virus, of the Moloney murine leukaemia virus (MoMuLv) and sarcoma bird flu virus (ASV) into the human genome, and therefore appears to be an interesting therapeutic approach in antiviral treatment (Van Maele et al. 2006, Li et al 1998, Hindmarsh et al. 1999).","Inhibitors of HMGA proteins are therefore also suitable for the treatment and diagnosis of virus infections (Reeves & Beckerbauer 2002).","As a result of the previously demonstrated involvement of HMGA proteins in various diseases and their suitability as diagnostic markers, L-nucleic acids and in particular spiegelmers directed against these proteins can be used for the prevention, treatment and diagnosis of the above diseases. Particularly preferred spiegelmers are in this connection the spiegelmers described herein. In this connection it is recognised by those skilled in the art that although the individual spiegelmers have been developed for a specific HMGA protein, as a result of the domain approach illustrated in Example 2 these also allow a cross-reactivity with other HMGA proteins, which can be seen from the alignment illustrated in .","Furthermore, it is recognised by those skilled in the art in this field that the nucleic acids according to the invention contain a number of structural motifs, which define a class of spiegelmers that bind as intracellular receptors to HMGA proteins. The various structural motifs are illustrated in more detail in Example 1.","The nucleic acids according to the invention comprise in a preferred embodiment also those nucleic acids which are substantially homologous to the sequences specifically disclosed herein. The term \u201csubstantially homologous\u201d should preferably be understood in this connection to mean that the homology is at least 75%, preferably 85%, more preferably 90% and most preferably more than 95, 96, 97, 98 or 99%.","The term nucleic acids according to the invention or nucleic acids according to the present invention should furthermore be understood to include also those nucleic acids which comprises nucleic acid sequences such as are described herein, or parts thereof, preferably to the extent that the nucleic acids or the said parts thereof are involved in the binding to HMGA proteins. Such a nucleic acid can be derived from those disclosed herein, for example by shortening or truncation. A shortening can involve either one or both ends of the nucleic acids, as are disclosed herein. A shortening can also involve the inner sequence of nucleotides, i.e. can involve nucleotide(s) between the 5\u2032 and the 3\u2032 terminal nucleotides. Furthermore the term shortening should also be understood as referring to the deletion of as few as one individual nucleotide from the sequence of the nucleic acids disclosed herein. Shortening can also involve more than one region of the nucleic acid(s) according to the invention, in which connection each of these regions may be as small as one nucleotide long.","The nucleic acids according to the present invention may furthermore be either D-nucleic acids or L-nucleic acids. Preferably the nucleic acids according to the invention are L-nucleic acids. In addition it is possible that one or more parts of the nucleic acid is\/are present as D-nucleic acids, or that at least one or more parts of the nucleic acids is\/are L-nucleic acids. The term \u201cpart\u201d of the nucleic acids is understood to denote as little as one nucleotide. Such nucleic acids are generally referred to herein as D-nucleic acids or L-nucleic acids.","Accordingly, in a preferred embodiment the nucleic acids according to the present invention consist of L-nucleotides and include at least one D-nucleotide. Such a D-nucleotide is preferably fixed to a part that is different from the region or regions that define the nucleic acids according to the present invention, and is preferably fixed to those parts thereof which are involved in an interaction with other parts of the nucleic acids. Preferably such a D-nucleotide is fixed to the end of each region or to each nucleic acid according to the present invention. In a preferred embodiment such D-nucleotides can act as a spacer or a linker, which preferably binds modifications such as PEG and HES to the nucleic acids according to the present invention.","Within the scope of the present invention, in one embodiment the nucleic acids according to the invention also include those acids which are part of a longer nucleic acid, wherein these longer nucleic acids can include several parts, at least one part being a nucleic acid according to the present invention or a part thereof. The other part or the other parts of these longer nucleic acids can either be a D-nucleic acid or a L-nucleic acid. Any combination can be used in conjunction with the present invention and for the purposes and uses such as have been described herein for the nucleic acids according to the invention. This other part or these other parts of the longer nucleic acid can have a function that is different from the binding function, and in particular from the binding to HMGA protein. A possible function is to allow an interaction with other molecules, e.g. for the purposes of immobilisation, cross-linking, detection, amplification, modification or increasing the molecular weight.","In particular in this connection L-nucleic acids as used herein are nucleic acids which consist of L-nucleotides, and preferably consist completely of L-nucleotides.","Accordingly, in particular D-nucleic acids as used herein are nucleic acids which consist of D-nucleotides, and preferably consist completely of D-nucleotides.","Irrespective of whether the nucleic acid according to the invention consists of D-nucleotides, L-nucleotides or a combination of the two, the combination being for example a random combination or a defined sequence of regions which consist of at least one L-nucleotide and at least one D-nucleic acid, the nucleic acid can consist of one or more deoxyribonucleotides, ribonucleotides and combinations thereof.","In a further aspect the present invention relates to a pharmaceutical composition which consists of at least one of the nucleic acids according to the invention in combination with one or more other nucleic acids, in which the other nucleic acid(s) preferably binds to target molecules other than HMGA protein or exerts a function different to that of the nucleic acids according to the invention.","The construction of the nucleic acids according to the invention as L-nucleic acids is advantageous for several reasons. L-nucleic acids are enantiomers of naturally occurring nucleic acids. D-nucleic acids are however not very stable in aqueous solutions and in particular in biological systems and in biological samples, on account of the extensive presence of nucleases. Naturally occurring nucleases, in particular nucleases from animal cells, are not able to break down L-nucleic acids. As a result of this the biological half-life of the L-nucleic acid in such a system, including the human and animal body, is significantly increased. On account of the lack of degradability of L-nucleic acids no nuclease breakdown products are produced and thus no resultant side effects are observed. This aspect in fact demarcates L-nucleic acids from all other compounds that are used in the treatment of diseases and\/or disorders and include the presence of HMGA or its causal involvement. L-nucleic acids that bind specifically to a target molecule through a mechanism different from the Watson-Crick base pairing, or aptamers which consist partly or completely of L-nucleic acids, in particular those parts of the aptamer that are involved in the binding of the aptamer to the target molecule, are termed spiegelmers.","It is also within the scope of the present invention for the nucleic acids according to the invention to be in the form of single-strand or double-strand nucleic acids, regardless of whether they are present as D-nucleic acids, L-nucleic acids or D-L-nucleic acids, and whether they are DNA or RNA. Typically the nucleic acids according to the invention are single-strand nucleic acids, which on account of the primary sequence contain defined secondary structures and can therefore also form tertiary structures. The nucleic acids according to the invention may however also be double-stranded, in the sense that two strands which are complementary or partly complementary to one another are hybridised with one another. This imparts stability to the nucleic acids, which becomes important particularly if the nucleic acid exists in the naturally occurring D-form instead of the L-form.","The nucleic acids according to the invention can be modified. Such modifications can involve individual nucleotides of the nucleic acid and are well-known in the prior art. Examples of such a modification are described inter alia in Venkatesan N. et al. (2003) Curr Med. Chem. October; 10 (19):1973-91; Kusser, W. (2000) J Biotechnol, 74: 27-38; Aurup, H. et al. (1994) 22, 20-4; Cummins, L. L. et al, (1995) 23, 2019-24; Eaton, B. E. et al. (1995) 2, 633-8; Green, L. S. et al., (1995) 2, 683-95; Kawasaki, A. M. et al., (1993) 36, 831-41; Lesnik, E. A. et al., (1993) 32, 7832-8; Miller, L. E. et al., (1993) 469, 213-43. Such a modification may for example be an H atom, a F atom or a O\u2014CHgroup or NHgroup at the 2\u2032 position of an individual nucleotide that is contained in the nucleic acid. Furthermore the nucleic acid according to the present invention can include at least one LNA nucleotide. In one embodiment the nucleic acid according to the present invention consists of LNA nucleotides, and preferably completely of LNA nucleotides.","In one embodiment the nucleic acids according to the present invention can be a multi-part nucleic acid. A multi-part nucleic acid as used herein is a nucleic acid that consists of at least two nucleic acid strands. These at least two nucleic acid strands form a functional unit, the functional unit being a ligand for a target molecule. The at least two nucleic acid strands can be derived from one of the nucleic acids according to the invention either by cleavage of the nucleic acid in order to produce two strands, or by synthesis from a nucleic acid corresponding to a first part of the total nucleic acid, i.e. nucleic acid according to the invention, and a further nucleic acid corresponding to the second part of the total nucleic acid. It is recognised that cleavage as well as synthesis can be used in order to produce a multi-part nucleic acid where more than the two strands described above by way of example can be present. In other words, the at least two nucleic acid strands are preferably different from two strands that are complementary to one another and hybridise with one another, although a complementarity can exist to a certain extent between the various nucleic acid parts.","The present inventors have established that the nucleic acids according to the present invention have a very advantageous Kvalue range or dissociation value range, and therefore a very advantageous binding constant. One way of determining the binding constant is to use an equilibrium binding assay, as is described in Example 1.","The Kvalue of the nucleic acids according to the invention is preferably less than 1 \u03bcM. A Kvalue of about 1 \u03bcM should be characteristic of a non-specific binding of a nucleic acid to a target. As will be recognised by those skilled in the art, the Kvalue of a group of compounds such as for example the nucleic acids according to the present invention varies within a certain range. The Kof about 1 \u03bcM mentioned above is a preferred upper limiting value for the Kvalue. The preferred lower limiting value for the Kof nucleic acids binding the target molecule can be about picomolar or less. It is within the scope of the present invention for the Kvalues of the individual nucleic acids which bind to HMGA, preferably to lie within this range. Preferred ranges can be selected by choosing a first number within this range and a second number within this range. Preferred upper values are 0.25 \u03bcM, 0.1 \u03bcM, and preferred lower values are 100 nM, 10 nM, 1 nM and 0.05 nM.","The nucleic acids according to the invention preferably bind to HMGA1b at 37\u00b0 C. in solution with a dissociation constant K<20 nM, as illustrated in Example 2.","The nucleic acids according to the present invention can be of arbitrary length, provided that they are still able to bind to the target molecule. It is recognised in the prior art that specific lengths of the nucleic acids according to the present invention are preferred. Typically the length is between 15 and 120 nucleotides. It is also recognised by those skilled in the art that any whole number between 15 and 120 is a preferred possible length for the nucleic acids according to the present invention. Preferred ranges for the length of the nucleic acids according to the present invention are lengths of about 20 to 100 nucleotides, about 20 to 80 nucleotides, about 20 to 60 nucleotides, about 20 to 50 nucleotides and about 30 to 50 nucleotides.","In one embodiment the nucleic acids according to the invention are present in modified form. A particularly preferred form of modification is PEGylation. In this, the modification of the nucleic acids according to the invention involves coupling with polyethylene glycol (PEG) or other groups.","On account of the high stability of the nucleic acids according to the invention, in particular in the embodiment in which these exist as L-nucleic acids, it is possible to administer the nucleic acids according to the invention directly to a patient requiring such a treatment. Preferably the nucleic acids according to the invention are prepared as a physiological solution for topical or systemic application.","Apart from the direct use of the nucleic acids according to the invention for the treatment, prevention and diagnosis of the diseases described herein, these can be present or used individually or in combination with others in a pharmaceutical composition. The pharmaceutical composition according to the present invention accordingly comprises at least one of the nucleic acids according to the present invention and preferably a pharmaceutically acceptable binder. Such a binder may be any known binder or one known in the field. In particular such a binder is any binder, as is described in connection with the production of the medicament, as disclosed herein. In a further embodiment the pharmaceutical composition includes a further pharmaceutically active agent. It is within the scope of the present invention for the medicament described herein to constitute the pharmaceutical composition as is described herein.","Preferably the pharmaceutical composition is intended for intravenous administration. It is however also within the scope of the present invention for such pharmaceutical compositions to be administered intramuscularly, intraperitoneally or subcutaneously. Other administration routes are orally or intranasally, in which connection that form of administration is preferred that is least invasive, but at the same time retains the effectiveness of the pharmaceutical composition and the pharmaceutically active agent.","The nucleic acids according to the invention are preferably contained as such, or in connection with the pharmaceutical composition according to the invention, dissolved in a pharmaceutically acceptable solvent. Such solvents are in particular those that are selected from the group comprising water, physiological saline, PBS or a glucose solution, in particular a 5% glucose solution. Such a carrier can be for example water, buffer, PBS, glucose solution, preferably a 5% glucose solution (iso-osmotic), starch, sugars, gelatin or any other acceptable carrier substance. Such carriers are generally known to those skilled in the art in this field.","It is within the scope of the present invention for the pharmaceutical composition to contain at least one of the nucleic acids according to the invention in its various embodiments, including, but not restricted thereto, the nucleic acid as conjugate, as described herein.","In a further embodiment the medicament comprises a further pharmaceutically active agent. Such further pharmaceutical active agents are for example protease inhibitors, proliferation inhibitors and angiogenesis inhibitors and\/or agents that have a cytostatic effect. Alternatively or in addition, such a further pharmaceutically active agent is a further nucleic acid according to the present invention. Alternatively, the medicament comprises at least one or more nucleic acids that bind to a target molecule that is different from HMGA, or has a function that is different from one of the nucleic acids according to the present invention.","The pharmaceutical composition according to the present invention can be used for the treatment, diagnosis and\/or prevention of each of the diseases or disorders described herein.","In a further aspect the present invention relates to a method for the treatment of a living organism requiring such a treatment, wherein the method includes the administration of a pharmaceutically active amount of at least one of the nucleic acids according to the present invention. In one embodiment the living organism suffers from a disease, or there is a risk that it will suffer from such a disease, the disease being one of those mentioned herein, in particular a disease that is described in connection with the use of one of the nucleic acids according to the present invention for the production of a medicament.","Although the use of the nucleic acids according to the invention already follows from the involvement illustrated above of HMGA proteins in the various diseases and states, this aspect will be discussed further hereinafter for illustrative purposes.","HMGA proteins and their genes have in particular become increasingly involved in the diagnosis and prognosis of neoplastic diseases and have been proposed as potential biomarkers. In healthy tissue the expression level of HMGA1a\/b proteins is very low, if detectable at all. Raised HMGA1a\/b protein expression is characteristic of the phenotype of a large number of tumours and metastases of very many types of cancer (Sarhadi et al. 2006, Balcercak et al. 2005, Briese et al. 2006, Chang et al. 2005, Peters et al. 2005, Sato et al. 2005, Chiappetta et al. 2004, Li et al. 2004, Chuma et al. 2004, Donato et al. 2004, Czyz et al. 2004, Kettunen et al. 2004, Lee et al. 2004, Chen et al. 2004, Abe et al. 2003, Blacerczak et al. 2003, Flohr et al. 2003, Masciullo et al. 2003, Nam et al. 2003, Pierantoni et al. 2003). High HMGA protein expression correlates significantly with a poor prognosis and the formation of metastases. The detection of the HMGA1a\/b expression level in biopsies and its histological characterisation is a diagnostic approach to the early detection, prognosis and identification of neoplastic diseases, in particular the diseases and conditions discussed hereinbefore.","Furthermore an association between HMGA1 proteins and arteriosclerotic plaques is described in the literature (Schlueter et al. 2005). HMGA1 regulates CD44, one of the principal target genes for the formation of plaques. In this connection it was found, compared to the surrounding tissue, that the affected regions such as neo-intimal, vascular smooth muscle cells, macrophages and new blood vessels have a high expression of HMGA1. HMGA1 appears therefore to be one of the mediators in the formation of plaque and is thus a target molecule for diagnostic purposes.","The L-nucleic acids described here and in particular the spiegelmers, which bind HMGA1a\/b, can within the scope of the methods known to the person skilled in the art be used in a similar way to antibodies. Up to now only very few specific (differentiating) and affine antibodies against HMGA1 have been identified and are commercially obtainable. This appears to be due to the non-existent secondary structure of HMGA1, which is not a suitable target for the MHC complex in the generation of antibodies.","Against this background it was however surprisingly found that the biotinylated HMGA1a\/b-binding spiegelmer 5\u2032-bio-NOX-A50 recognises in the western blot procedure HMGA1a\/b as individual bands in cancer cell lines. Furthermore, as described in Example 2, recombinantly expressed HMGA1b protein could be detected. The detection of the biotinylated spiegelmer is carried out for example by an anti-biotin antibody conjugated by means of horseradish peroxidase (HRP).","The in vivo diagnosis of HMGA1a\/b is a further approach, in which the nucleic acids according to the invention can be used. Tumours and metastases are often embedded in necrotic tumour cells, which release HMGA1a\/b to the surrounding tissue. The detection of the extracellular HMGA1a\/b is one approach to the diagnosis of tumours and metastases embedded in healthy tissue.","As preferably used herein, a diagnostic tool or diagnostic agent or diagnostic means is able to detect either directly or indirectly an HMGA protein, preferably HMGA1a\/b, as described herein, and preferably HMGA1a\/b as described herein, in connection with the various disorders and diseases. The diagnostic tool is suitable for detecting and\/or searching for any of the diseases and conditions described herein. Such a detection is possible by the binding of the nucleic acids according to the present invention to HMGA1a\/b. Such a binding can be detected either directly or indirectly. The corresponding methods and means are known to those skilled in the art in this field. The nucleic acids according to the present invention can inter alia be labelled, which permits the detection of the nucleic acids according to the present invention, preferably the nucleic acid that is bound or can bind to HMGA protein and preferably HMGA1a\/b. Such a labelling is preferably selected from the group comprising radioactive, enzymatic and fluorescence labelling. In principle all known tests that have been developed for antibodies can be adapted to the nucleic acids according to the present invention, the target molecule-binding antibody being replaced by a target molecule-binding nucleic acid. In antibody tests which employ unlabelled target molecule-binding antibodies, the detection is preferably performed with a secondary antibody, which has been modified with radioactive, enzymatic or fluorescence labels and binds to the target molecule-binding antibody at its Fc fragment. In the case of a nucleic acid, preferably a nucleic acid according to the present invention, the nucleic acid is modified with such a label, the said label preferably being selected from the group consisting of biotin, CY-3 and CY-5, and such a label is detected by an antibody directed against such a label, for example an anti-biotin antibody, an anti-CY-3 antibody or an anti-CY-5 antibody, or in the case where the label is biotin, the label is detected by streptavidin or avidin, which naturally binds to biotin. Such an antibody, i.e. streptavidin or avidin, is in turn preferably modified with a corresponding label, for example a radioactive, enzymatic or fluorescence label, similarly to a secondary antibody.","In a further embodiment the nucleic acids according to the present invention are detected or analysed by a second detection agent, this detection agent being a molecular beacon. The technique of molecular beacons is known to those skilled in the art in this field. In brief, these molecular beacons are nucleic acid probes which are a reverse complement of the nucleic acid probe to be detected, and accordingly hybridise with a part of the nucleic acid probe to be detected. After the binding of the nucleic acid probe the fluorophore groups of the molecular beacon are separated from one another, which leads to a change in the fluorescence signal, preferably a change in intensity. This change correlates with the amount of nucleic acid probe that is present.","It is within the scope of the present invention that the nucleic acids according to the invention can appropriately be used as L-nucleic acids within the scope of the various aspects disclosed herein.","The nucleic acids according to the invention can furthermore be used as starting material for the design of pharmaceutical active substances (drug design). In principle there are two possible approaches to this problem. One approach consists in screening libraries of compounds, wherein such libraries of compounds are preferably libraries of low molecular weight compounds (low or small molecules). Such libraries are known to those skilled in the art in this field. In one embodiment the screening is a high throughput screening. Preferably high throughput screening is fast, efficient, and is carried out as a trial-and-error evaluation of active substances in a target molecule-based assay.","Alternatively, according to the present invention the nucleic acids can be used for the rational design of active substances. Preferably the rational design of active substances is the design of a pharmaceutical active substance candidate. Starting from the three-dimensional structure of the target molecule, which is normally determined by methods such as X-ray structure analysis or nuclear magnetic resonance spectroscopy (NMR), computer programs are used to search through data banks containing structures of a large number of different chemical compounds. The selection is carried out by computer. The selected compounds are in addition tested in the laboratory.","The rational design of active substances can take as its starting point any of the nucleic acids according to the present invention, and comprises a structure, in particular a three-dimensional structure, which is similar to the structure of the nucleic acid(s) according to the invention or is identical to that part of the structure of the nucleic acid(s) according to the invention that mediates the binding to HMG proteins. In any case, such a structure also exhibits the same or at least a similar binding behaviour to the nucleic acid(s) according to the invention. In either a further step or as an alternative step, in the rational design of active substances the preferably three-dimensional structure of those parts of the nucleic acids binding to HMG proteins is imitated by chemical groups, which are preferably different to nucleotides and nucleic acids. By means of this imitation, also termed mimicry, a compound can be constructed which is different from the nucleic acid or the nucleic acids which was\/were used as starting materials for the rational design of the active substance. Such a compound or active substance is preferably a small molecule or a peptide.","In the case of screening libraries of compounds using competitive tests which are known to those skilled in the art in the field, suitable HMG analogues, HMG agonists and HMG antagonists can be found. Such competitive assays can be designed as follows. The nucleic acid according to the invention, preferably a spiegelmer, i.e. a L-nucleic acid binding the target molecule, is coupled to a preferably solid phase. In order to identify HMG analogues, a labelled HMG protein is added to the test system. Alternatively, the HMG protein could also be coupled to a solid phase and the nucleic acid according to the invention could be labelled. A potential analogue or a potential agonist or antagonist would compete with the HMG molecules which bind to the spiegelmer, which would result in a decrease in the signal received from the corresponding label. The screening for agonists or antagonists can include the use of a cell culture test system which is known to those skilled in the art in the field.","In a further aspect the nucleic acids according to the invention can, on account of their characteristic binding behaviour to HMG protein, be used for target (target molecule) validation. The nucleic acids according to the invention can be used in an ex vivo organ model in order to study the function of HMG protein. In principle there exist ex vivo models in which HMG agonists\/antagonists can be tested.","A kit according to the present invention can comprise at least one or more of the nucleic acids according to the invention. In addition the kit can include at least one or more positive or negative controls. HMG protein against which the nucleic acid according to the invention has been screened, or to which this binds, preferably in liquid form, can be used as positive control. As negative control there can be used inter alia a peptide that behaves as regards its biophysical properties similarly to HMG protein, but which is not recognised by the nucleic acids according to the invention, or a peptide can be used having the same amino acid composition but a different sequence to HMG protein.","Furthermore the kit can include one or more buffers. The various constituents can be present in the kit in dry or lyophilised form, or dissolved in a liquid. The kit can include one or more containers, which in turn can contain one or more of the constituents of the kit. Preferably the vessels contain reaction batches, such as are necessary for a single execution of an experiment using one or more constituents of the kit.","It will be acknowledged that, unless stated to the contrary, the sequences listed herein are given in the 5\u2032-3\u2032 direction. It will furthermore be seen that the term \u201cthe two sections hybridise with one another\u201d is understood herein to mean that the sections can hybridise in vitro on the basis of general base pairing rules, or that the sections hybridise or can hybridise under the conditions of use, but are not necessarily hybridised with one another or are present in hybridised form under the conditions of use.","The various SEQ. ID., the chemical structure of the nucleic acids as disclosed herein and the target molecule HMGA1a\/1b as used herein, the actual sequences and the internal references are summarised in the following table.",{"@attributes":{"id":"p-0293","num":"0372"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"259pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"0pt","align":"left"}}],"thead":{"row":[{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":[{},"Internal",{},{},{}]},{"entry":["Seq. ID","Reference","RNA\/Peptide","Sequence"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"252pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"7pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1","132-C3, NOX-h","L-RNA (spiegelmer)","GCUGCUGCAAAUUGACGGGGGCGUGGUUGGGGCGGGUCGAUUGCAGC",{}]},{"entry":{}},{"entry":["2","132-B3, NOX-f","L-RNA (spiegelmer)","GCUGAAUGAGGAUCGCAGGGGCGUGGCUGGGGUGGGCGACCGUUCAGC"]},{"entry":[{},"(48 nt)"]},{"entry":{}},{"entry":["3","132-C4","L-RNA (spiegelmer)","GCUGCGCAAGGAGAGGGGCGCGGUUGGGGAGGCUCUAAGCGCUGCAGC"]},{"entry":{}},{"entry":["4","132-E2","L-RNA (spiegelmer)","GCUGGCGCUAUAGGACAGGGGUGCGGUUGGGGCGGUCCGCUGUCAGC"]},{"entry":{}},{"entry":["5","132-A2","L-RNA (spiegelmer)","GCUGGAUAGAACGCAGGGGUGCGGUUUGGGGUGGGCGUGAUAUGCAGC"]},{"entry":{}},{"entry":["6","132-H1, NOX-I","L-RNA (spiegelmer)","GCUGCCGUAAAGAGGGGUGAGGUUGGGGAGGCUUUACGGUUUCAGC"]},{"entry":{}},{"entry":["7","132-F1","L-RNA (spiegelmer)","GCUGCAUGCCGCGAUCAGGGGAGCGGUUGGGGCGGGAUCCGGCUCAGC"]},{"entry":{}},{"entry":["8","132-G2, NOX-g","L-RNA (spiegelmer)","GCUGCGAGGGAGGUAGCGGCUCUGCGCCGUGACGUGGGUGGAUGCAGC"]},{"entry":{}},{"entry":["9","122-A1, NOX-A","L-RNA (spiegelmer)","GGCUGAUACGUGGGUGGAUAUGGGGCAGUUCCAUGUGGGUGGUUUCAGCC"]},{"entry":{}},{"entry":["10","122-C1, NOX-B","L-RNA (spiegelmer)","GGCUGAUACGUGGGUGAAUAUGGGGCAGUUCCAUGUGGGUGGUUUCAGCC"]},{"entry":{}},{"entry":["11","122-B2","L-RNA (spiegelmer)","GGCUGAUACGUGGGAGGAAAGGUGUAACUACCUGUGGGAGGUUUCAGCC"]},{"entry":{}},{"entry":["12","122-E2, NOX-C","L-RNA (spiegelmer)","GGCUGGCACUCGCAGGGGUGAAGUGAUGAUUGGGGUGGGCGAGACCAGCC"]},{"entry":{}},{"entry":["13","122-G2, NOX-E","L-RNA (spiegelmer)","GGCUGCCGAGUGGUUGGGUGGUGUAAGGGAGGUGGAAUCCGCGGGCAGCC"]},{"entry":{}},{"entry":["14","122-B4, NOX-D","L-RNA (spiegelmer)","GGCUGUUCGUGGGAGGAAGGCUCUUGGAUAGAGUCGUGGGUGGUUCAGCC"]},{"entry":{}},{"entry":["15","132-B3 32 nt, NOX-","L-RNA (spiegelmer)","GGAUCGCAGGGGCGUGGCUGGGGUGGGCGACC"]},{"entry":[{},"f32 nt"]},{"entry":{}},{"entry":["16","132-B3 33 nt, NOX-","L-RNA (spiegelmer)","GGAUCGCAGGGGCGUGGCUGGGGUGGGCGAUCC"]},{"entry":[{},"f33 nt"]},{"entry":{}},{"entry":["17","HMGA1a\/b target","D-peptide","Biotin-EPSEVPTPKRPRGRPKGSKNK"]},{"entry":[{},"molecule domain,"]},{"entry":{}},{"entry":[{},"Biotinyl-D-"]},{"entry":[{},"HMGA1a\/b-21 mer"]},{"entry":{}},{"entry":["18","HMGA1a (human)","L-peptide","(M)SESSSKSSQPLASKQEKDGTEKRGRGRPRKQPPVSPGTALVGSQKEPSEVPTPKRPR"]},{"entry":[{},{},{},"GRPKGSKNKGAAKTRKTTTTPGRKPRGRPKKLEKEEEEGISQESSEEEQ"]},{"entry":{}},{"entry":["19","HMGA1b (human)","L-peptide","(M)SESSSKSSQPLASKQEKDGTEKRGRGRPRKQPPKEPSEVPTPKRPRGRPKGSKNKGA"]},{"entry":[{},{},{},"AKTRKTTTTPGRKPRGRPKK-LEKEEEEGISQESSEEEQ"]},{"entry":{}},{"entry":["20","HMGA2 human","L-peptide","(M)SARGEGAGQPSTSAQGQPAAPAPQKRGRGRPRKQQQEPTGEPSPKRPRGRPKGSKNK"]},{"entry":[{},{},{},"SPSKAAQKKAEATGEKRPRGRPRKWPQQVVQKKPAQEETEETSSQESAEED"]},{"entry":{}},{"entry":["21","bio-dsDNA (AT","D-DNA","5\u2032biotin-TCGAAAAAAGCAAAAAAAAAAAAAAAAAACTGGC)"]},{"entry":[{},"hook)"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"259pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":["and",{},{},{},{}]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"259pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["50",{},{},"5\u2032GCCAGTTTTTTTTTTTTTTTTTTGCTTTTTT",{}]},{"entry":{}},{"entry":["22","NOX-A-3\u2032PEG,","L-RNA","GGCUGAUACGUGGGUGGAUAUGGGGCAGUUCCAUGUGGGUGGUUUCAGCC-2kDA-PEG"]},{"entry":[{},"NOX-A-3\u2032PEG2000,"]},{"entry":[{},"NOX-A-2 kDa PEG,"]},{"entry":[{},"NOX-A PEG"]},{"entry":{}},{"entry":["23","INVERSE-3\u2032-PEG","L-RNA","CCGACUUUGGUGGGUGUACCUUGACGGGGUAUAGGUGGGUGCAUAGUCGG-2kDA-PEG"]},{"entry":[{},"INV 3\u2032-PEG"]},{"entry":{}},{"entry":["24","5\u2032-biotin-NOX-A","L-RNA","Biotin-GGCUGAUACGUGGGUGGAUAUGGGGCAGUUCCAUGUGGGUGGUUUCAGGC"]},{"entry":{}},{"entry":["25","5\u2032-biotin-NOX-A","L-RNA","Biotin-CCGACUUUGGUGGGUGUACCUUGACGGGGUAUAGGUGGGUGCAUAGUCGG"]},{"entry":[{},"inverse"]},{"entry":{}},{"entry":["26","INVERSE","L-RNA","CCGACUUUGGUGGGUGUACCUUGACGGGGUAUAGGUGGGUGCAUAGUCGG"]},{"entry":{}},{"entry":["27","POC-3\u2032-PEG","L-RNA","UAAGGAAACUCGGUCUGAUGCGGUAGCGCUGUGCAGAGCU-2kDA-PEG"]},{"entry":{}},{"entry":["28","Capture probe","L-RNA","CCCATATCCACCCACGTATCAGCCTTTTTTTT-NH"]},{"entry":[{},"NOX-A"]},{"entry":{}},{"entry":["29","Detector probe","L-RNA","Biotin-TTTTTTTTGGCTGAAACCACCCACATGG"]},{"entry":[{},"NOX-A"]},{"entry":{}},{"entry":["30","Capture probe POC","L-RNA","NH(C7)-TTTTTTTTTAGCTCTGCACAGCGCT"]},{"entry":{}},{"entry":["31","Detector probe","L-RNA","CCGCATCAGACCGAGTTTCCTTATTTTTTTT-Biotin"]},{"entry":[{},"POC"]},{"entry":{}},{"entry":["32","HMG_fwd1 Primer","D-DNA","TCGACACCATGGGTGAGTC"]},{"entry":{}},{"entry":["33","HMG_rev1 Primer","D-DNA","GTCTAGAAAGCTTCCCAACTG"]},{"entry":{}},{"entry":["34","132-C3, NOX-h","D-RNA (aptamer)","GCUGCUGCAAAUUGACGGGGGCGUGGUUGGGGCGGGUCGAUUGCAGC"]},{"entry":{}},{"entry":["35","132-B3, NOX-f","D-RNA (aptamer)","GCUGAAUGAGGAUCGCAGGGGCGUGGCUGGGGUGGGCGACCGUUCAGC"]},{"entry":[{},"(48 nt)"]},{"entry":{}},{"entry":["36","132-C4","D-RNA (aptamer)","GCUGCGCAAGGAGAGGGGCGCGGUUGGGGAGGCUCUAAGCGCUGCAGC"]},{"entry":{}},{"entry":["37","132-E2","D-RNA (aptamer)","GCUGGCGCUAUAGGACAGGGGUGCGGUUGGGGCGGUCCGCUGUCAGC"]},{"entry":{}},{"entry":["38","132-A2","D-RNA (aptamer)","GCUGGAUAGAACGCAGGGGUGCGGUUUGGGGUGGGCGUGAUAUGCAGC"]},{"entry":{}},{"entry":["39","132-H1, NOX-i","D-RNA (aptamer)","GCUGCCGUAAAGAGGGGUGAGGUUGGGGAGGCUUUACGGUUUCAGC"]},{"entry":{}},{"entry":["40","132-F1","D-RNA (aptamer)","GCUGCAUGCCGCGAUCAGGGGAGCGGUUGGGGCGGGAUCCGGCUCAGC"]},{"entry":{}},{"entry":["41","132-G2, NOX-g","D-RNA (aptamer)","GCUGCGAGGGAGGUAGCGGCUCUGCGCCGUGACGUGGGUGGAUGCAGC"]},{"entry":{}},{"entry":["42","122-A1, NOX-A","D-RNA (aptamer)","GGCUGAUACGUGGGUGGAUAUGGGGCAGUUCCAUGUGGGUGGUUUCAGCC"]},{"entry":{}},{"entry":["43","122-C1, NOX-B","D-RNA (aptamer)","GGCUGAUACGUGGGUGAAUAUGGGGCAGUUCCAUGUGGGUGGUUUCAGCC"]},{"entry":{}},{"entry":["44","122-B2","D-RNA (aptamer)","GGCUGAUACGUGGGAGGAAAGGUGUAACUACCUGUGGGAGGUUUCAGCC"]},{"entry":{}},{"entry":["45","122-E2, NOX-C","D-RNA (aptamer)","GGCUGGCACUCGCAGGGGUGAAGUGAUGAUUGGGGUGGGCGAGACCAGCC"]},{"entry":{}},{"entry":["46","122-G2, NOX-E","D-RNA (aptamer)","GGCUGCCGAGUGGUUGGGUGGUGUAAGGGAGGUGGAAUCCGCGGGCAGCC"]},{"entry":{}},{"entry":["47","122-B4, NOX-D","D-RNA (aptamer)","GGCUGUUCGUGGGAGGAAGGCUCUUGGAUAGAGUCGUGGGUGGUUCAGCC"]},{"entry":{}},{"entry":["48","132-B3 32 nt,","D-RNA (aptamer)","GGAUCGCAGGGGCGUGGCUGGGGUGGGCGACC"]},{"entry":[{},"NOX-f32 nt"]},{"entry":{}},{"entry":["49","132-B3 33 nt,","D-RNA (aptamer)","GGAUCGCAGGGGCGUGGCUGGGGUGGGCGAUCC"]},{"entry":[{},"NOX-f33 nt"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},"It is within the scope of the present invention that, if no sequences are explicitly given for the individual sections of the nucleic acids according to the invention, these can be freely chosen according to the technical teaching disclosed herein, i.e. can be chosen so that they exhibit the necessary binding behaviour to the respective target molecule and\/or are able to form the structures, in particular secondary structures, described herein.","Furthermore, it is within the scope of preferred embodiments of the present invention that in the case where, in sequences that are identified as RNA sequences, T is given instead of U, then T shall denote U.","1.1 HMGA1a\/b-Binding Sequences","The HMGA1a\/b-binding RNA spiegelmers were generated by in vitro selection against -21AS-HMGA1a\/b and subsequent shortening steps. The generated aptamers, which bind the 21AS-HMGA1a\/b domain, are shown in .","1.1.1 Ranking and Aptamer Level","The different clones (see ) were prepared as aptamers (-RNA) by means of standard phosphoramidite synthesis and were radioactively labelled at the 5\u2032 end by kinasing (see below). The clones were then analysed as regards their affinity and activity by means of equilibrium binding assay at two concentrations of -bio-21aa HMGA1a\/b.","Radioactive Labelling by Kinasing:",{"@attributes":{"id":"p-0322","num":"0401"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}}],"thead":{"row":[{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"Substance","[final]"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"RNA","\u20025 \u03bcM"]},{"entry":[{},"T4 forward reaction buffer (Invitrogen)","1x"]},{"entry":[{},"T4 polynucleotide kinase (Invitrogen)","10 U\/10 \u03bcl"]},{"entry":[{},"[\u03b3-P]-ATP","\u20021 \u03bcl\/10 \u03bcl"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}}}},"The reaction ran for 1 hour at 37\u00b0 C. and was then stopped by heating (10 minutes at 65\u00b0 C.). The separation of radioactive nucleotides from labelled oligonucleotides was carried out by an analytical polyacrylamide gel electrophoresis (PAGE) (see hereinafter). A \u201ccrush-and-soak\u201d gel elution was then carried out with ammonium acetate and precipitation with ethanol (see hereinafter). The amount of purified RNA was estimated from the radioactivity of the pellets (after the precipitation) compared to the radioactivity of the cut-out strip.","Polyacrylamide Gel Electrophoresis (PAGE)","For the preparative purification of oligonucleotides, \u00bd to 2 volumes of concentrated sample buffer for denaturing PAGE were added to the reaction batches. In addition large-scale standards were prepared as necessary (each 250 pmole) and taken up in sample buffer. The batches were denatured for 5 minutes at 95\u00b0 C. and cooled on ice. A preparative, denaturing 7% or 10% PAA gel (200\u00d7200\u00d71.5 mm) was preheated (ca. 1 hour) by applying a maximum voltage of 600 V at 40-50 W. After rinsing the cups with 1\u00d7TBE the samples were plotted. After completion of the separation (50 minutes at 50 W) the gel was placed on a fluorescing thin-layer chromatography plate protected by transparent film (dye 60F). The bands were visualised as shadows (\u201cUV shadowing\u201d) by means of UV light (254 nm) and were cut out with a scalpel. A \u201ccrush-and-soak\u201d gel elution with ammonium acetate was then performed.","\u201cCrush-and-Soak\u201d Gel Elution","To elute oligonucleotides from PAA gels, after comminuting the cut-out PAA gel strips 500 \u03bcl of ammonium acetate (2 M) was added using a pipette tip or a spatula. The \u201ccrush-and-soak\u201d elution was carried out 2\u00d71.5 hours at 68\u00b0 C. in a thermoshaker (1000 rpm). The supernatants were freed from gel residues by \u201cUltrafree-MC\u201d small columns (Millipore\/Amicon, Schwalbach, Germany) in a table centrifuge (16,100\u00d7g). The RNA eluted in this way was then desalted by precipitation with ethanol.","Ethanol Precipitation","For the ethanol precipitation 1-2 \u03bcl of glycogen were used as precipitation auxiliary. After adding 2.5 volumes of absolute ethanol and vortexing, the oligonucleotides were precipitated for 30 minutes at \u221280\u00b0 C. and centrifuged off for 30 minutes at 16,100 g, 4\u00b0 C. The pellet was washed once with 70% ethanol and centrifuged for 5 minutes at 16,100 g, 4\u00b0 C.","Recording of Binding Isotherms in the Equilibrium Binding Assay","2 pmole of each of the 5\u2032 radioactively labelled aptamers were complexed in biotinyl--HMGA1a\/b-21mer (EPSEVPTPKRPRGRPKGSKNK [Seq. ID. 17]; see ), produced by Bachem (Weil am Rhein, Germany). Solutions in the concentration range 1-3000 nM (or for the two-point measurement with 300 nM and 30 nM or 100 nM and 10 nM peptide) were incubated for 1 hour at 37\u00b0 C. in selection buffer (10 mM Tris HCl pH7.4, 5 mM KCl, 0.8 mM MgCl, 0.1% Tween). A solution without biotinylated -HMGA1a\/b-21mer served as background control. The peptide and complexes were then immobilised within 30 minutes at 37\u00b0 C. with 10 \u03bcl of streptavidin UltraLink gel. The radioactivity of the suspension was measured. The supernatant was removed. The matrix was then washed once with 100 \u03bcl of selection buffer and then precipitated with selection buffer. By measuring the radioactivity the aptamer fraction present together with biotinyl--HMGA1a\/b-21mer in the complex was determined for each peptide concentration. The dissociation constants of the active species and the proportion of active molecules were determined by graphical plotting and fit (GraFit, Version 4.0.10, Erithacus Software).","Results","For all clones synthesised as aptamers (-RNA) a dissociation constant for the binding to the 21 amino acid-long D fragment of HMGA1a\/b (Biotinyl-D-HMGA1a\/b-21mer) of 8-22 nM was determined in the equilibrium binding assay ().","1.1.2 Shortening in Example 132-B3","All selection candidates exhibited a repetitively occurring sequence motif GGGCG or GGGUG or GGGAG, which is stabilised at the 5\u2032 end and at the 3\u2032 end by a helix\/stem motif ().","An analysis of the probable structure and precipitation of the RNA aptamers according to Zuker (Nucleic Acids Res. 2003 Jul. 1; 31 (13):3406-15) showed that the predetermined stem\/helix structure had lengthened in some cases (132-C3, 132-B3, 132-C4, 132-E2, 132-A2, 132-H1, 132-F1, 122-G2, 122-E2, see ). This stem-Helix structure formed the basis for the further shortening of these candidates. This further shortening of the candidates was carried out by identifying and stabilising the minimal binding motif by precipitation analysis followed by deletion analysis of the synthetic -RNAs with respect to the binding to the HMGA1a\/b fragment. These binding properties were determined by equilibrium binding assay.  shows by way of example in the candidate NOX-f (132-B3) the shortening of the aptamer on the basis of the stabilising stem structure, which can be found in lengthened form in the candidates 132-C3, 132-B3, 132-C4, 132-E2, 132-A2, 132-H1, 132-F1, 122-G2, 122-E2 (see ). A shortening to a 32 nucleotide-long aptamer variant of NOX-f with a 6 nucleotide-long stem (NOX-f 32 nt) did not lead to any loss of the binding properties to the 21aa HMGA1a\/b fragment (). The artificial insertion of an adenosine at the third position of the 5\u2032-position stem led to a theoretical formation of a 7 nucleotide-long stem without a looped-out region and served to complete the stem in the 3\u2032 region (NOX-f 33 nt, ). The measurement of the binding properties (affinity and activity) by means of equilibrium binding assay on the 21 amino acid-long domain of HMGA1a\/b was not influenced by these changes.","The sequences 132-G2, 122-A1, 122-C1, 122-B2 and 122-B4 have on the other hand at the 5\u2032 end and 3\u2032 end of the repetitively occurring sequence motif (GGGCG or GGGUG or GGGAG) a significantly shorter stem structure. A shortening of the stem structure led to a binding loss. A possible shortening of the central region, which is longer for these sequences, between the repetitive sequence motif (GGGCG or GGGUG or GGGAG) was not carried out.","The Seq. IDs of the aptamer sequences of the HMGA-binding nucleic acids disclosed herein are as follows:",{"@attributes":{"id":"p-0333","num":"0412"},"tables":{"@attributes":{"id":"TABLE-US-00006","num":"00006"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"thead":{"row":[{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Seq. ID","Internal Reference","RNA\/Peptide"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["34","132-C3, NOX-h","D-RNA (aptamer)"]},{"entry":["35","132-B3, NOX-f (48nt)","D-RNA (aptamer)"]},{"entry":["36","132-C4","D-RNA (aptamer)"]},{"entry":["37","132-E2","D-RNA (aptamer)"]},{"entry":["38","132-A2","D-RNA (aptamer)"]},{"entry":["39","132-H1, NOX-i","D-RNA (aptamer)"]},{"entry":["40","132-F1","D-RNA (aptamer)"]},{"entry":["41","132-G2, NOX-g","D-RNA (aptamer)"]},{"entry":["42","122-A1, NOX-A","D-RNA (aptamer)"]},{"entry":["43","122-C1, NOX-B","D-RNA (aptamer)"]},{"entry":["44","122-B2","D-RNA (aptamer)"]},{"entry":["45","122-E2, NOX-C","D-RNA (aptamer)"]},{"entry":["46","122-G2, NOX-E","D-RNA (aptamer)"]},{"entry":["47","122-B4, NOX-D","D-RNA (aptamer)"]},{"entry":["48","132-B3 32nt, NOX-f 32nt","D-RNA (aptamer)"]},{"entry":["49","132-B3 33nt, NOX-f 33nt","D-RNA (aptamer)"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}}}},"As has already been discussed herein and is known to those skilled in the art in this field the enantiomer, consisting of L-nucleotides, of an aptamer, i.e. of a D-nucleic acid which was generated against a D-peptide, binds to the mirror-image enantiomer of the D-peptide, i.e. the naturally occurring L-peptide. This L-nucleic acid is herein also referred to as spiegelmer and otherwise exhibits in principle the same binding properties as the aptamer.","1.2 Characteristic Properties of HMGA\/b-Binding Spiegelmers","1.2.1 Repetitive Sequence Elements: Box A1 and Box A2","A repetitive sequence element of the sequence GGGCG or GGGUG or GGGAG is characteristic of all spiegelmers that bind to HMGA1a\/b. This sequence element appears twice in HMGA1a\/b-binding spiegelmers (). The sequence element lying closer to the 5\u2032 end of the spiegelmers is herein referred to as Box A1. The sequence element lying closer to the 3\u2032 end of the spiegelmers is on the other hand referred to herein as Box A2. Box A1 and Box A2 and their mutual arrangement probably represent the decisive feature of HMGA1a\/b-binding spiegelmers.","1.2.2 Sequence Section Between Box A1 and Box A2","Between Box A1 and Box A2 there is either a sequence section with a length of six to seven nucleic acids or 12 to 22 nucleotides (). Since these sequence sections differ not only in their length, they are discussed separately.","Case 1: Sequence Section Comprises Six to Seven Nucleotides","If the sequence section lying between Box A1 and Box A2 has a length of six nucleotides, then the sequence section exhibits the sequence UGGUUG, UGGCUG, CGGUUG, AGGUUG or GUGUAA. An insertion of one nucleotide (uracil) into the sequence CGGUUG, which leads to the sequence CGGUUUG, has neither a negative nor a positive influence on the binding properties of the spiegelmers.","Case 2: Sequence Section Comprises 12 to 22 Nucleotides","If the sequence section lying between Box A1 and Box A2 has a length of 12 to 22 nucleotides, then this sequence section comprises two sequence regions of equal length, which can possibly hybridise with one another (Helix C). The hybridisation is in this case effected by in each case three to six nucleotides. Three to five unpaired nucleotides are located between the nucleotides forming the Helix C. One to three nucleotides are present unpaired between the 3\u2032 end of Box A1 and the 5\u2032 end of Helix C. One to five nucleotides can be present unpaired between the 3\u2032 end of Helix C and the 5\u2032 end of Box A2.","1.2.3 Helical Structure at the 5\u2032 End and 3\u2032 End of the Spiegelmers","All HMGA1a\/b-binding spiegelmers are characterised at their 5\u2032 and 3\u2032 ends by sequence sections which can hybridise with one another (Helix A1 and Helix A2, (). The number of nucleotides hybridising with one another in each case can vary from four to eight. In this connection, this presumably double-strand region can extend to the 5\u2032 end of Box A1 and the 3\u2032 end of Box A2. Should this not be the case, then Box A1 and Box A2 can be flanked by nucleotides that additionally hybridise with one another (Helix B1 and Helix B2). This can involve regions of in each case four to eight nucleotides ().","Within the scope of the invention forming the basis of the present application, various classes of nucleic acids and in particular L-nucleic acids which bind to the target molecule have been identified. The following illustration and description of these classes, which are herein also termed cases, is to this extent an integral part of the present invention. For each class their principal structure and exemplary L-nucleic acids for this class are specified hereinafter using the respective abbreviations of the L-nucleic acids.",{"@attributes":{"id":"p-0341","num":"0420"},"chemistry":{"@attributes":{"id":"CHEM-US-00007","num":"00007"},"img":{"@attributes":{"id":"EMI-C00007","he":"27.69mm","wi":"69.34mm","file":"US08497250-20130730-C00007.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"ul":{"@attributes":{"id":"ul0020","list-style":"none"},"li":["N=U, C, A, G;","N=G, U;","N=U, C;","N=U, A;","N=G, A;","N=G, A, U;","N=G, U;","N=U or no nucleotide;","N=zero to five nucleotides;","N=zero or six nucleotides;","N=zero to six nucleotides;"]}},{"@attributes":{"id":"p-0342","num":"0432"},"chemistry":{"@attributes":{"id":"CHEM-US-00008","num":"00008"},"img":{"@attributes":{"id":"EMI-C00008","he":"3.73mm","wi":"55.63mm","file":"US08497250-20130730-C00008.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"Helix A1 and Helix A2=in each case four to eight nucleotides, which completely or partly hybridise with one another, in which the sum of the in each case mutually hybridising nucleotides of Helix A1 and Helix A2 and Helix B1 and Helix B2 is 10 to 12 nucleotides;","Helix B1 and Helix B2=in each case four to eight nucleotides, which hybridise with one another, in which the sum of the in each case mutually hybridising nucleotides of Helix A1 and Helix A2 and Helix B1 and Helix B2 is 10 to 12 nucleotides.","The molecules are active also after the shortening at the 5\u2032 end and at the 3\u2032 end. After the removal of the Helix A1 and A2 as well as the regions NN7 and GNthe shortened molecules retain their binding properties. This was demonstrated for the shortened variants 132-B3 32 nt (NOX-f 32 nt) and 132-B3 33 nt (NOX-f 33 nt) (see ).","Case 2A: 132-G2, 122-A1, 122-C1, 122-B2, 122-B4",{"@attributes":{"id":"p-0346","num":"0436"},"chemistry":{"@attributes":{"id":"CHEM-US-00009","num":"00009"},"img":{"@attributes":{"id":"EMI-C00009","he":"7.03mm","wi":"60.20mm","file":"US08497250-20130730-C00009.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"ul":{"@attributes":{"id":"ul0021","list-style":"none"},"li":["N=one to five nucleotides","N=three nucleotides","N=three to five nucleotides","N=two to five nucleotides","N=one to two nucleotides, preferably A or UU"]}},{"@attributes":{"id":"p-0347","num":"0442"},"chemistry":{"@attributes":{"id":"CHEM-US-00010","num":"00010"},"img":{"@attributes":{"id":"EMI-C00010","he":"3.73mm","wi":"54.95mm","file":"US08497250-20130730-C00010.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"Helix A1 and Helix A2=in each case five to six nucleotides, which completely or partly hybridise with one another,",{"@attributes":{"id":"p-0349","num":"0444"},"chemistry":{"@attributes":{"id":"CHEM-US-00011","num":"00011"},"img":{"@attributes":{"id":"EMI-C00011","he":"5.59mm","wi":"68.75mm","file":"US08497250-20130730-C00011.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{}},{"@attributes":{"id":"p-0350","num":"0445"},"chemistry":{"@attributes":{"id":"CHEM-US-00012","num":"00012"},"img":{"@attributes":{"id":"EMI-C00012","he":"6.86mm","wi":"59.94mm","file":"US08497250-20130730-C00012.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"ul":{"@attributes":{"id":"ul0022","list-style":"none"},"li":["N=two nucleotides, preferably CA","N=two nucleotides, preferably AG","N=four nucleotides, preferably GAUG"]}},{"@attributes":{"id":"p-0351","num":"0449"},"chemistry":{"@attributes":{"id":"CHEM-US-00013","num":"00013"},"img":{"@attributes":{"id":"EMI-C00013","he":"3.73mm","wi":"54.95mm","file":"US08497250-20130730-C00013.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"Helix A1 and Helix A2=in each case six nucleotides, which hybridise with one another,","Helix B1 and Helix B2=in each case five nucleotides, which hybridise with one another,",{"@attributes":{"id":"p-0354","num":"0452"},"chemistry":{"@attributes":{"id":"CHEM-US-00014","num":"00014"},"img":{"@attributes":{"id":"EMI-C00014","he":"5.59mm","wi":"56.98mm","file":"US08497250-20130730-C00014.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"2.1 Determination of the Interaction of HMGA1a\/b Spiegelmers and Recombinant HMGA1b in the Competition Assay","Execution\/Method","Cloning of His6-Labelled HMGA1b","The BD-Freedom\u2122 ORF clone GH00552L1.0 (high mobility group AT hook1) with the sequence coding for HMGA1b was purchased from BioCat Heidelberg. The sequence had already been changed therein so that the stop codon is converted into a codon coding for leucine, in order to permit C-terminal fusions. The sequence of the clone corresponds generally to the sequence stored in the RefSeq data bank under No. NM002131. The sequence coding for HMGA1b was amplified by means of a standard PCR with the primers HMG_fwd1 (TCGACACCATGGGTGAGTC, Seq. ID 34) and HMG_rev1 (GTCTAGAAAGCTTCCCAACTG, Seq. ID 35). In this connection the base after ATG was changed from A to G, in order thereby to introduce a NcoI interface. The PCR product was cleaved according to the manufacturer's instructions with the restriction enzymes NcoI and HindIII (both from NEB, Frankfurt am Main, Germany) and purified via an agarose gel. The vector pHO2d (Fasshauer et al. (1997) J. Biol. Chem. 272:28036-28041)) was similarly cleaved with NcoI and HindIII and purified via an agarose gel. The Vector pHO2d permits the expression of a protein fused to the C-terminal end with a sequence of six histidine residues (His6-tag), under the control of a T7-promoter (Fasshauer et al., 1997, JBC 272:28036).","The purified and cleaved PCR product was ligated into the prepared vector overnight at 15\u00b0 C. with the aid of a T4 ligase, corresponding to the manufacturer's instructions (MBI Fermentas, St. Leon-Roth, Germany). Bacteria of strain DH5\u25a1 were transformed with the ligation product. The correctness of the plasmids from obtained colonies was checked by sequencing. The fusion protein HMGA1b-His6 coded by pHO2d\/HMGA1b has, compared to the natural HMGA1b protein, a glycine (G) instead of serine (S) at position 2, and after the C-terminal glutamine (Q) a leucine (L) (see above), followed by five further amino acids (G S L N S) (coded by the vector), to which the six histidines (H) are joined.","Expression and Purification of HMGA1b-His6","For the expression of the fusion protein bacteria of strain BL21 were transformed with the plasmid pHO2d\/HMGA1b. The expression of the fusion protein was induced with isopropylthio-\u03b2-D-galactoside (IPTG). After 4 hours the bacteria were centrifuged off for 15 minutes at 10,000\u00d7g and the pellet was stored at \u221220\u00b0 C. until further use.","For the extraction of the fusion protein 25 ml of extraction buffer (1% n-octyl-\u03b2,D-thioglucopyranoside (OTG) in 50 mM NaPObuffer, pH 8.0, 250 mM NaCl, 10 mM imidazole and MiniProtease inhibitor tablets (Roche, Mannheim, Germany) (5 hrs\/50 ml)) were added to a frozen bacteria pellet from 500 ml of culture, followed by 5 \u03bcl of benzonase (grade1; MERCK, Darmstadt, Germany), homogenised by pipetting and pipetting off, and incubated for 5 min at RT. This was followed by centrifugation for 15 mins at 10,000\u00d7g (RT). The supernatant was filtered through a fluted filter and then added to a HIS-SELECT column (HIS-SELECT Cartridge, Sigma, Deisenhofen, Germany) equilibrated with wash buffer (50 mM NaPObuffer, pH 8.0, 250 mM NaCl, 10 mM imidazole, all from MERCK, Darmstadt, Germany). After washing the column with 10-15 ml of wash buffer the fusion protein was eluted with elution buffer (250 mM imidazole in wash buffer) in 0.5-1 ml size fractions. Protein-containing fractions were checked for purity by means of gel electrophoresis (16% polyacrylamide gel according to Sch\u00e4ger & Jagow, 1987, Anal. Biochem. 166:368-379). Fractions with fusion protein were purified, if necessary dialysed using a suitable buffer, and after protein determination were tested again for purity. The purified fusion protein was stored in aliquots at \u221220\u00b0 C.","Determination of the Interaction of HMGA1a\/b Spiegelmers and HMGA1b-His6","A test based on the 96-well format was used for a more detailed analysis of the affinity of the HMGA1a\/b-spiegelmers for HMGA1b. In this test the binding of the HMGA1a\/b spiegelmers to HMGA1b-His6 prevents its interaction with a DNA oligonucleotide that has a binding site for HMGA1a\/b. This DNA oligonucleotide (dsDNA AT hook) (Fashena et al., 1992) is labelled on one strand with a biotin molecule, via which it can be bound to plates coated with streptavidin. The detection of HMGA1b-His6 bound to DNA is carried out with horseradish peroxidase modified with nickel (Nickel-HRP), which transforms a fluorogenic substrate. In this assay the spiegelmer displaces the recombinant HMGA1b from its natural binding partner. On account of the 1:1:1 stoichiometry of spiegelmer\/rHMGA1b\/dsDNA AT Hook, a direct prediction can be made regarding the affinity of the spiegelmers for HMGA1b. The principle of the assay is illustrated in .","To carry out this test spiegelmers in various concentrations and HMGA1b-His6 (0.36 \u03bcg\/ml; ca. 30 nM) in a total volume of 100 \u03bcl are incubated for 10 minutes in a tapered floor plate at room temperature while shaking. The incubation solution also contains: 25 mM Tris\/HCl, pH 7.0 (Ambion, Austin, Tex., USA), 140 mM KCl (Ambion, Austin, Tex., USA), 12 mM NaCl (Ambion, Austin, Tex., USA), 0.8 mM MgCl2 (Ambion, Austin, Tex., USA), 0.25 mg\/ml BSA (Roche, Mannheim, Germany), 1 mM DTT (Invitrogen, Karlsruhe, Germany), 18-20 \u03bcg\/ml poly(dGdC) (Sigma, Deisenhofen, Germany)), 0.05% Tween 20 (Roche, Mannheim, Germany). 2 \u03bcl of biotinylated DNA oligonucleotides dsDNA AT hook (equimolar mixture of 5\u2032biotin-TCGAAAAAAGCAAAAAAAAAAAAAAAAAACTGGC (34 nt) and 5\u2032GCCAGTTTTTTTTTTTTTTTTTTGCTTTTTT (31 nt); 75 \u03bcM in 150 mM NaCl (Ambion, Austin, Tex., USA)) are then added and incubated for a further 10 mins at RT while shaking. The batches are then transferred to a black 96-well plate coated with streptavidin (ReactiBind from Pierce, Bonn, Germany)) and incubated for 30 mins at RT while gently shaking. Following this the wells of the plate are washed three times, each time with 200 \u03bcl of TBSTCM (20 mM Tris\/HCl, pH 7.6 (Ambion, Austin, Tex., USA); 137 mM NaCl (Ambion, Austin, Tex., USA), 1 mM MgCl2 (Ambion, Austin, Tex., USA), 1 mM CaCl2 (Sigma, Deisenhofen, Germany), 0.05% Tween 20 (Roche, Mannheim, Germany)). 50 \u03bcl of a dilute nickel-HRP solution are added to each well (ExpressDetector nickel-HRP, (Medac, Hamburg, Germany) 1:1000 in 10 mg\/ml BSA (Roche, Mannheim, Germany) in TBSTCM) and incubated for 1 hour at RT while gently shaking. The wells are then washed again three times with 200 \u03bcl TBSTCM each time.","100 \u03bcl of the fluorogenic HRP substrate (QuantaBlue, Pierce, Bonn, Germany) are then added to each well and the fluorescence is measured after 15 mins (ex: 340\/em: 405 nm).","Result","It was shown that the spiegelmers NOX-A (50 nt), NOX-f (33 nt) and NOX-f (48 nt) compete in a concentration-dependent manner with the binding of HMGA1b-His6 to the biotinylated DNA-Oligonucleotide (). A IC50 of ca. 15 nM is found for spiegelmer NOX-A.","In contrast to the active spiegelmer, a control spiegelmer with a inverse sequence to NOX-A showed in a concentration of up 0.5 \u03bcM no effect on the binding of HMGA1b-His6 to the DNA oligonucleotide, and non-specific interactions with HMGA1b-His6 occur only at concentrations above 1 \u03bcM ().","2.2 Use of Spiegelmers to Detect HMGA1b by Western Blot","Execution\/Methods","The recombinantly expressed HMGA1b was separated by gel electrophoresis on a 16% PAA-tricin gel and transferred by means of electroblotting to nitrocellulose membranes. The membrane was then blocked with 5% skimmed milk and 100 nM non-specific spiegelmer in 1\u00d7TBST (20 mM Tris\/HCl pH 7.6, 137 mM NaCl, 0.1% Tween) for 1 hour and washed three times for 10 minutes with 1\u00d7TBST. The detection of the recombinant HMGA1b was carried out with spiegelmer NOX-A biotinylated at the 5\u2032 end (5\u2032bioNOX-A). 5\u2032bioNOX-A was diluted in 1\u00d7TBST with 1 mM each of calcium and magnesium (TBST+Ca\/Mg) and 100 nM non-specific spiegelmer and incubated for 1.5 hours. The blot was then washed three times for 10 minutes with 1\u00d7TBST+Ca\/Mg and the bound biotinylated spiegelmer was incubated with an anti-biotin antibody in TBST+Ca\/Mg for 45 minutes. The blot was then washed five times for 10 minutes with 1\u00d7TBST+Ca\/Mg and the secondary antibody coupled with horseradish peroxidase (HRP) was detected by means of LumiGLO detection reagent (Cell Signaling Technology).","Result","The binding of a 5\u2032-terminal biotinylated spiegelmer to the recombinantly expressed HMGA1b was demonstrated by means of the aforedescribed process. Similarly to a detection based on antibodies, 5 \u03bcg of HMGA1b were detected with 3 nM bio-NOX-A after transfer to a blot membrane. The inverse spiegelmer of NOX-A could not recognise HMGA1b, which confirms the specific binding of NOX-A ().","3.1 Principle of the Polyethyleneimine-Mediated Transfection of Spiegelmers","The target molecule HMGA1a\/b is expressed in the cytosol and finds as transcription factor its natural binding partner, the double-strand DNA in the cell nucleus. The HMGA1a\/b-mediated cellular responses should be antagonised by binding of the spiegelmer to HMGA1a\/b in the cytosol, and competition of the HMGA1a\/b bound by the AT hooks to the DNA in cell nucleus. On account of the negative charge of the plasma membrane DNA and RNA molecule are not readily taken up by passive transport from a cell. One of the approaches to the intracellular transport by nucleic acids is the condensation or packing with charged particles or reagents, resulting in a charge of the overall complex. This complex is easily taken up through endocytosis and thus passes into the cytosol of the cell. Disadvantages of this method are the stability of the DNA\/RNA and the release of the nucleic acid from the endosomal compartment. In the cytosol of the cell a lysosome is quickly formed from the constricted endosome by the introduction of proteases or nucleases and by protonation of the compartment. Nucleases digest the nucleic acids there and in addition the nucleic acid is not stable in the acidic medium. The whole complex is rapidly transported again out of the cell by exocytosis and decomposition in the Golgi apparatus, and therefore only a few nucleic acids pass into the cell. One of the preconditions for a suitable transfection system is thus the stabilisation as well as the release of the nucleic acid from the endosome into the cytosol. As regards stability RNA spiegelmers have ideal properties for a transfection of eukaryotic cells, since being unnatural enantiomers they are not cleaved by enzymes.","The selected transfection system is based on the formation of micelles of nucleic acids with branched polyethyleneimine (PEI). The phosphate backbone of the nucleic acids interacts with the free nitrogen positions of the PEI and forms small micelles by cross-branching, which have a positive charge on account of the PEI. In this connection PEI with a molecular weight of 3 to 800 kDa is used. The smaller the PEI, the smaller are the formed micelles. The use of 25 kDa cross-branched PEI (Sigma-Aldrich Cat. No. 40; 872-7) leads on addition of nucleic acids to the formation of polyplexes of size 100 nm up to 500 nm, though typically to polyplexes of size 100 to 200 nm. As a rule a nitrogen\/phosphate ratio of 2:1 to 5:1 is used, in some cases even up to 20:1. The packing of the nucleic acid in micelles results in a change of the zeta potential of the complex to \u02dc(+)21 mV with a N\/P ratio of 3. It is known that with increasing, positive zeta potential of complexes the toxicity to culture cells rises. These micelles are however easily taken up as endosomes by a cell by constriction of the plasma membrane. The PEI now buffers inflowing protons, as a result of which many chloride ions in the interior of the endosome lead to a swelling of the compartment on account of the osmotic pressure. This effect of PEI is described in the literature as the proton sponge effect (Sonawane et al., JBC, 2003, Vol. 278; No. 45 (7) pp. 44826-44831) and ultimately leads to the rupture of the endosome and to the release of the spiegelmers into the cytosol.","The nucleic acid-PEI complex has a tendency on account of a strongly positive charge to interaction and aggregation with serum proteins, and also to exhibit the aforedescribed cell toxicity. Thus, it has been described in the literature that high doses of nucleic acid-PEI micelles after subcutaneous and intravenous injection in rats can rapidly lead to an accumulation in the lungs and thus to embolisms\/infarcts. The solution to this problem is to derivatise the nucleic acid with 2 kDa polyethylene glycol (PEG). These residues surround the micelles like a shield and prevent the binding to serum proteins (Ogris et al., Gene Therapy, 1999, 6 (595-605). Furthermore, the zeta potential is reduced to +\/\u22120 mV, which leads to a lower cell toxicity while retaining the buffer capacity of the PEI as regards the proton sponge effect.","3.2 Spiegelmer Activity with PEG2000","The lead candidates NOX-A and NOX-f were produced synthetically as aptamer and spiegelmer with a 3\u2032-terminal amino group, and were then PEGylated via the amino radical. It was shown by means of equilibrium binding assays that PEGylation has no influence on the binding properties of the aptamers to the HMGA1a\/b fragment. Furthermore, it was shown by means of competition assays with recombinant full-length HMGA1a\/b that also the binding of spiegelmers to the full-length HMGA1a\/b is independent of the 3\u2032-terminal PEGylation ().","3.3 Spiegelmer Packing","The packing of sterile, PEGylated spiegelmer was carried out in PBS by adding 25 kDa of cross-branched polyethyleneimine (PEI) (ALDRICH, Cat.: 40, 872-7) in a ratio of the absolute nitrogen fraction of the PEI to the absolute phosphate of the ribonucleic acid backbone of 2.5:1 (N\/P 2.5). The sterile, autoclaved PEI solution had a concentration of 200 mM free nitrogen groups and was adjusted to a pH of 7.4 with 1 M hydrochloric acid. The sterile filtered spiegelmer was taken in a concentration of up to 700 \u03bcM in 1\u00d7PBS with Ca\/Mg and after addition of sterile filtered PEI was incubated for 30-60 minutes at room temperature. Ideally the complex formation takes place with the smallest possible adjusted concentration of added spiegelmer, since high concentrations of spiegelmer lead to randomly large aggregates. The formation of spiegelmer micelles was measured by means of a dye exclusion assay. For this, it was determined how much spiegelmer can be detected by the dye before and after packing in micelles. RiboGreen (M. Probes) was used as dye, and the fluorescence was measured with an ELISA reader. 1 \u03bcM spiegelmer was added in each case to 100 \u03bcl 1\u00d7PBS and increasing amounts of PEI were added. 100 \u03bcl of 0.2 \u03bcg\/\u03bcl RiboGreen were placed in a 96-well microtitre plate suitable for fluorescence, and after incubating the micelle batch for 30 minutes at room temperature 10 \u03bcl were pipetted into the microtitre plate. Starting from a N\/P ratio of 2, more than 90% of the spiegelmers were present as micelles (). In this connection PEI alone had no influence on the fluorescence of the dye.","3.4 Stability of Spiegelmer Micelles","1 \u03bcM of spiegelmer micelles were stored under conditions specified in . The stability of spiegelmer micelles was measured by the dye exclusion assay described in Section 3.3 A stability study of the micelles showed that the storage of micelles in different media as well as at different temperatures has no influence on the spiegelmer micelles. The freeze drying of ribozyme\/PEI complexes without any loss of the properties of the ribozyme is also described in the literature (Brus-C et al., J. Control Release, 2004, Feb., 20, 95 (1), 199-31).","3.5 Uptake of Spiegelmer Micelles","The intracellular uptake of spiegelmer micelles was established in a cell culture system of HS578T cells. 1\u00d710HS578T cells were allowed to grow on sterile 20 mm size cover classes to a confluence of 30-40%. 5\u2032-labelled spiegelmer NOX-A-3\u2032-PEG was packed with a N\/P ratio 2.5:1 in micelles, added in a concentration of 1 \u03bcM to the cells, and incubated for 16 hours at 37\u00b0 C. As control for the passive uptake of spiegelmers, 1 \u03bcM of pure fluorescence-labelled spiegelmer was in each case incubated with the cells. The cells were then washed three times with 1 ml of PBS and fixed for 30 minutes with 3% paraformaldehyde. The preparations were again washed three times with 1 ml of PBS, incubated for a further 10-20 seconds with a DAPI solution (1 \u03bcl stock to 10 ml 1\u00d7PBS) to stain the chomatin in the cell nucleus, washed once more, and covered with a mounting solution. The preparations prepared in this way were analysed in a fluorescence microscope (emission 488 nm\/extinction 514-522 nm).","It was shown that spiegelmer micelles have a higher transfection rate compared to \u201cnaked\u201d, unpacked spiegelmers ()","The transfection efficiency was in this connection >95% of all cells and had no influence on the morphology of the cells. The 5\u2032-FITC-coupled spiegelmer was mainly to be found in the cytosol and associated with the plasma membrane. The point-like distribution indicates an inclusion in compartments and the diffuse pattern points to released spiegelmer. Only a slight spiegelmer signal could be detected in the cell nucleus.","3.6 Release of Spiegelmer","The point-like distribution of the spiegelmer in the cytosol and perinuclear space of the H578T cells points to an accumulation in compartments of the cells, for example endosomes. To check the release of the spiegelmers from these compartments the distribution pattern of individual, greatly enlarged cells was analysed (). In addition to the point-like localisation of the spiegelmers, a diffuse distribution pattern in the cytosol and on the plasma membrane was detected, which points to the endosomal release of the spiegelmers. This pattern was not found in the case of \u201cnaked\u201d spiegelmers.","4.1 Proliferation Assay without PEI","Effect on the Proliferation of MCF-7 Cells","The potential role of HMGA1a\/b in cell division was investigated by means of proliferation assays. First of all spiegelmer was added in a high dose as \u201cnaked\u201d nucleic acid to the cell culture medium and the growth of the cells was followed over time The breast cancer cell line MCF-7 was used as model, since in these cells a smaller (antagonising) expression of HMGA1a\/b was found, and the role of HMGA1a\/b in the proliferation of these cells had already been described in the literature. Reeves et al. (Reeves-R et al., Molecular and Cellular Biology, January 2001, p575-594) showed that the over-expression of HMGA1a\/b in MCF-7 cells leads to an increased proliferation, and the inhibition of HMGA1a\/b by means of expressed antisense constructs inhibits the proliferation of MCF-7 cells.","Execution\/Method","0.5\u00d710MCF-7 cells (ATCC) were seeded out in 96-well plates (Costar) with a flat, transparent base and cultured for 16-24 hours in 100 \u03bcl RPMI 1640 medium with 10% foetal calf serum (FCS). The cells were then washed with PBS and cultured for a further 48 hours with standard cell culture medium with the direct addition of sterile filtered spiegelmer. This was followed by the addition of 10 \u03bcl of a resazurin solution (0.44 mM in PBS) to the respective batches and further incubation for 2 hours at 37\u00b0 C. The transformation of resazurin by the cell metabolism correlates directly with the number of cells. The change in colour was measured in a Fluostar Optima multidetection plate reading device (BMG) (emission 544 nm, extinction 590 nm). Each value was determined three times per experiment and referred to the values of untreated control cells.","Result","NOX-A inhibited after two days in a dose-dependent manner the proliferation of MCF-7 cells (n=12) (). The maximum inhibition of the proliferation to ca. 30% of the value of untreated cells was found at 40 \u03bcM. At concentrations up to 40 \u03bcM no non-specific effect of the inverse control spiegelmer was found.","4.2 Proliferation Assay with PEI","Effect of Spiegelmer Micelles on the Proliferation of H-1299 Cells","Execution\/Method","1\u00d710NCI-H-1299 cells (lung carcinoma cells; ATCC) were seeded out in 24-well plates (Costar) with a flat, transparent base and cultured for 16-24 hours in 1 ml RPMI 1640 medium with a 10% FCS. The cells were then washed twice with PBS and cultured for a further three days with cell culture medium containing 1% FCS and spiegelmer micelles. The packing of sterile, PEGylated spiegelmer was carried out beforehand in PBS by adding 25 kDa cross-branched polyethyleneimine (PEI) (Sigma) in a ratio of the absolute nitrogen fraction of the PEI to the absolute phosphate of the ribonucleic acid backbone of 2.5:1 (N\/P 2.5). The sterile spiegelmer was used in a concentration of 30 \u03bcM and after the addition of the PEI was incubated for 30-60 minutes at room temperature. The spiegelmer micelles were then diluted to 1 \u03bcM with cell culture medium containing 1% FCS, added directly to the washed cells, and incubated for three days at 37\u00b0 C.","This was followed by addition of 100 \u03bcl resazurin solution to the respective batches and further incubation for 2 hours at 37\u00b0 C. The transformation of resazurin by the cell metabolism correlates directly with the number of cells. 100 \u03bcl were removed from the batches, transferred to a 96-well plate, and the colour change was measured in a Fluostar Optima multidetection plate reading device (BMG) (emission 544 nm, extinction 590 nm). Each value was determined twice per experiment and referred to the values of untreated control cells.","Result","The use of PEI (N\/P 2.5) with 1 \u03bcM spiegelmer did not initially have any effect on cell proliferation. By reducing the amount of FKS in the cell culture medium to below 1% it was shown that the transfection with spiegelmer micelles has an influence on the proliferation of H-1299 cells, which was not previously visible with 10% FKS (). Possibly FKS stimulates the proliferation to such an extent that the slight effect could not be observed. The reduction of the FKS concentration in MCF-7 cells lead to the death of the cells over a period of 3 days.","4.3 Inhibition Tumour Marker cdc25a (with PEI)","Effect on the HMGA1a\/b-mediated regulation of cell cycle factors, in the example of the potential oncogene cdc25a.","Reeves et al. (Molecular and Cellular Biology, January 2001, p575-594) showed by means of cDNA arrays through over-expression of HMGA1a\/b in MCF-7 cells that HMGA1a\/b induces the expression of a large number of genes. At the same time cell cycle factors and growth factors such as for example cdc25a, identified as a potential oncogene (cell division cycle 25a phosphotase), which plays a decisive role in the control of the transition from the G1 phase to the S phase of the cell cycle, are over-expressed by a factor of up to 100. The activation of such control points leads after inhibition of the cell cycle progression either to the transcription of genes which are involved in DNA repair or, if the DNA damage is irreparable, to the induction of apoptosis. As cell culture test system H-1299 cells were chosen for this purpose, since they have already exhibited an increased expression of HMGA1a\/b.","Execution\/Method","1\u00d710H-1299 cells were seeded out in 24-well plates (Costar) with a flat, transparent floor and cultured for 16-24 hours in RPMI 1640 medium containing 10% FCS (volume 1 ml). The cells were then washed twice with PBS and cultured for a further three days in cell culture medium with spiegelmer micelles containing 10% FCS. The packing of sterile, PEGylated spiegelmer was carried out beforehand in PBS by adding 25 kDa cross-branched polyethyleneimine (PEI) (Sigma) in a ratio of the absolute nitrogen fraction of the PEI to the absolute phosphate of the ribonucleic acid backbone of 2.5:1 (N\/P 2.5). The sterile spiegelmer was used in a concentration of 30 \u03bcM and, after adding PEI, was incubated for 30-60 minutes at room temperature. The spiegelmer micelles with cell culture medium containing 1% FCS were then diluted to the respective concentration, added directly to the washed cells, and incubated for three days at 37\u00b0 C. The cells were washed twice with PBS and harvested by means of a cell scraper. The mRNA of the cells was then isolated from the cells by means of Roti-Quick-Kits (Roth, Cat. No. 979.1) and 0.2-1 \u03bcg of total RNA was used as template for the PCR of cdc25a and GAPDH.","The primers for the amplification of GAPDH were as follows: forward primer: 5\u2032-ACATGTTCCAATATGATTCC-3\u2032 and reverse primer: 5-TGGACTCCACGACGTACTCAG-3\u2032 at an annealing temperature of 51\u00b0 C., and for the amplification of cdc25a: forward primer: 5\u2032-GAGGAGTCTCACCTGGAAGTACA-3\u2032 and reverse primer 5\u2032-GCCATTCAAAACCAGATGCCATAA-3\u2032 at an annealing temperature of 59\u00b0 C. The PCR conditions were as follows: 0.2-0.75 \u03bcM primer, 1.5 mM MgCl2 and 0.2 mM dNTPs. Every two PCR cycles an aliquot of 5 \u03bcl was quantified by PicoGreen and evaluated by correlation with GAPDH as load control: for this, in the first step for each investigated sample the so-called \u201ccrossing point\u201d value (CP) of the reference gene is subtracted from the CP value of the gene being investigated (dCP=CP target gene minus CP reference gene). CP is defined as the number of PCR cycles that are required in order to reach a constantly defined fluorescence value. The same amount of newly synthesised DNA is found at the CP in all reaction vessels. After this standardisation the dCP value of a control (in this case GAPDH) is subtracted from the dCP value of the experimently treated samples; one arrives at the so-called \u201cdelta-delta CT\u201d calculation model. The relative expression difference of a sample between the treatment and the control (ratio), normalised to the reference gene and referred to a standard sample, is found from the arithmetic formula 2.\n\ndCP=CP(cdc25)\u2212CP(GAPDH)\n\nddCP=dCP(treatment spiegelmer NOX-A)\u2212dCP(control: PBS or NOX-A inverse)\n\nRatio=2\n\nResult\n","cdc25a and HMGA1a\/b were detected in MCF-7 and H-1299 cells by means of RT-PCR. MCF-7 cells showed with a low expression of HMGA1a\/b also a low expression of cdc25a, whereas HMGA1a\/b and cdc25a were strongly expressed in H-1299 cells. The transfection of H-1299 cells for two days with HMGA1a\/b-binding spiegelmers led to a significant, dose-dependent reduction of the expression of cdc25a mRNA ( and ).","Up to a concentration of 4 \u03bcM a control spiegelmer exhibited no non-specific effect, neither on the GAPDH nor on the cdc25a mRNA expression. From this it can be concluded that the HMGA1a\/b-induced over-expression of the potential oncogene cdc25a can be inhibited by means of spiegelmers.","Effect of Spiegelmers on Tumour Growth In Vivo","In order to test the hypothesis that HMGA1a\/b-binding spiegelmers inhibit the growth of tumours in vivo, a xenograft model was developed for the strongly HMGA1a\/b-expressing pancreatic carcinoma cells PSN-1. On the basis of this model a therapeutic experiment was carried out with 2 mg\/kg NOX-A spiegelmer micelles at a N\/P of 2.5 (see Example 3, paragraph 3.3).","Execution\/Method","Male naked mice (NMRI: nu\/nu) (group size n=8) were subcutaneously injected in the side with in each case 10PSN-1 cells (ECACC) and the tumour growth was observed over 22 days. The animals had a mean weight of 25-27 g and were 6-8 weeks old. The active spiegelmer NOX-A-3\u2032PEG and the inverse control spiegelmer in INV-3\u2032PEG were packed in micelles as described above by adding PEI in a N\/P ratio of 2.5. 100 \u03bcl of the spiegelmer micelle suspension (corresponding to 3.46 nmole\/animal or 2 mg\/kg) were in each case subcutaneously injected daily into the vicinity of the tumour. The tumour volume and bodyweight were measured three times a week. The animals were sacrificed on day 22 and the distribution of NOX-A in the plasma, liver, kidneys and tumour was quantified.","For this purpose the tissues were homogenised in hybridisation buffer (0.5\u00d7SSC pH 7.0; 0.5% (w\/v) SDSarcosinate) and centrifuged for 10 mins at 4000\u00d7g. The supernatants obtained were stored at \u221220\u00b0 C. until further use.","The amount of spiegelmer in the plasma samples and in the tissue homogenates was investigated by means of a hybridisation assay (Drolet et al. (2000) Pharm. Res. 17:1503). The hybridisation assay is based on the following principle: the spiegelmer to be detected (L-RNA molecule) is hybridised on an immobilised L-DNA oligonucleotide probe (=capture probe NOX-A; in this case: 5\u2032-CCCATATCCACCCACGTATCAGCCTTTTTTTT-NH2-3\u2032; complementary to the 5\u2032 end of HMGA1a\/b-NOX-A) and detected with a biotinylated detection L-DNA probe (=detector probe NOX-A; in this case: 5\u2032-biotin-TTTTTTTTGGCTGAAACCACCCACATGG-3\u2032; complementary to the 3\u2032 end of HMGA1a\/b-NOX-A). For this purpose a streptavidin alkaline phosphatase conjugate is in a further step bound to the complex. After adding a chemiluminescence substrate light is generated and measured in a luminometer.","Immobilisation of the oligonucleotide probe: 100 \u03bcl of the capture probe (0.75 pmole\/ml in coupling buffer: 500 mM NaHPO, pH 8.5, 0.5 mM EDTA) were transferred to each well (depression in a plate) in DNA-BIND plates (Corning Costar) and incubated overnight at 4\u00b0 C. The probe was then washed three times with 200 \u03bcl of coupling buffer each time and incubated for 1 hour at 37\u00b0 C. with in each case 200 \u03bcl of blocking buffer (0.5% (w\/v) BSA in coupling buffer). After washing again with 200 \u03bcl of coupling buffer and 3\u00d7200 \u03bcl hybridisation buffer the plates can be used for the detection.","Hybridisation and detection: 10 \u03bcl EDTA plasma or tissue homogenate were mixed with 90 \u03bcl of detection buffer (2 pmole\/\u03bcl of detector probe in hybridisation buffer) and centrifuged. Further purifications were carried out as necessary. The batches were then denatured for 10 mins at 95\u00b0 C., transferred to the suitably prepared DNA-BIND wells (see above) and incubated for 45 mins at ca. 40\u00b0 C. The following wash steps were then carried out: 2\u00d7200 \u03bcl hybridisation buffer and 3\u00d7200 \u03bcl 1\u00d7TBS\/Tween 20 (20 mM Tris-Cl pH 7.6, 137 mM NaCl, 0.1% (v\/v) Tween 20). 1 \u03bcl streptavidin alkaline phosphatase conjugate (Promega) was diluted with 5 ml of TBS\/Tween 20. 100 \u03bcl of the diluted conjugate were added to each well and incubated for 1 hour at room temperature. The following wash steps were then carried out: 2\u00d7200 \u03bcl TBS\/Tween 20 and 2\u00d7200 \u03bcl of assay buffer (20 mM Tris-Cl pH 9.8, 1 mM MgCl). 100 \u03bcl of CSPD \u201cReady-To-Use Substrate\u201d (Applied Biosystems) were then added, incubated for 30 mins at room temperature, and the chemiluminescence was measured in a Fluostar Optima multidetecton plate reading device (BMG).","Result","In a preliminary experiment it was shown that H-1299 cells after transplanting as a tumour grew significantly more slowly than PSN-1, and on comparing the individual animals exhibited an inhomogeneous tumour growth and therefore appeared unsuitable as xenograft model for a treatment study. PSN-1 cells exhibited an aggressive tumour growth within 22 days. It was shown that NOX-A nicelles at a dose of 2 mg\/kg reduced the growth of PSN-1 tumours significantly compared to the PBS control (). The weight of the animals was unaffected by the treatment with spiegelmer micelles. The control spiegelmer did not exhibit any non-specific inhibition of the tumour growth and likewise had no effect on the weight of the animals. The differences in tumour sizes were, from day 10 of the treatment with NOX-A3\u2032PEG micelles, significant or highly significant compared to untreated animals (PBS control (student's t-test). The end point analysis after 22 days showed a highly significant, specific reduction in tumour growth (p=0.0098 compared to PBS and p=0.022 compared to inverse control spiegelmer) (). Mice treated with PBS showed an average tumour growth of 2.5 cm, animals treated with controlled spiegelmer had an average tumour volume of 2.6 cmand animals treated with NOX-A had an average tumour volume of 1.2 cmafter 22 days (box-and-whisker analysis). This corresponds to a reduction of the tumour growth of more than 50%.","The analysis of the tissue distribution of NOX-A showed a high concentration in the tumour ().","In order to check the efficient incorporation of spiegelmer micelles, a non-functional spiegelmer (Proof Of Concept=POC) was PEGylated at the 3\u2032 end with PEG 2 kDa and packed with a nitrogen\/phosphate ratio (N\/P) of 2.5 in micelles (see Example 3, paragraph 3.3). In a similar way to the protocol described in Example 5, this approach was adopted for spiegelmer packed in micelles as well as for free, unpacked spiegelmer.","Execution\/Method","Male naked mice (NMRI: nu\/nu) (group size n=8) were in each case injected subcutaneously in the side with 10PSN-1 cells (ECACC) and the tumour growth was observed over 25 days. The animals had a mean weight of 25-27 g and were 6-8 weeks old. The non-functional spiegelmer POC-3\u2032PEG was packed in micelles by adding PEI in a N\/P ratio of 2.5, as described above. Spiegelmer POC-3\u2032PEG not packed in micelles served as control for the incorporation not mediated by PEI. 100 \u03bcl of the spiegelmer-micelle suspension or spiegelmer solution (corresponding to 1500 nmole\/kg and 2000 nmole\/kg) were injected daily subcutaneously into the vicinity of the tumour. The tumour volume and body weight were measured three times a week. 24 and 96 hours after the last injection two animals from each group were sacrificed and the distribution of POC-3\u2032PEG (packed\/unpacked) in the plasma, brain, heart, lungs, liver, kidneys, gallbladder, pancreas and tumour was quantified.","For this purpose the tissue was homogenised in hybridisation buffer (0.5\u00d7SSC pH 7.0; 0.5% (w\/v) SDSarcosinate) and centrifuged for 10 mins at 4000\u00d7g. The resultant supernatants were stored at \u221220\u00b0 C. until further use.","The amount of spiegelmer in the plasma samples and in the tissues homogenates was investigated by means of a hybridisation assay (Drolet et al. (2000) Pharm. Res. 17:1503). The assay is based on the following principle: the spiegelmer (L-RNA molecule) to be detected is hybridised on an immobilised L-DNA oligonucleotide probe (=capture probe POC; here: 5\u2032\u2014NH2(C7)-TTTTTTTTTAGCTCTGCACAGCGCT-3\u2032; complementary to the 3\u2032 end of POC) and is detected with a biotinylated detection L-DNA probe (=detector probe POC; here: 5\u2032-CCGCATCAGACCGAGTTTCCTTATTTTTTTT-Biotin-3\u2032; complementary to the 5\u2032 end of POC). For this, a streptavidin alkaline phosphatase conjugate was bound in a further step to the complex. After addition of a chemiluminescence substrate, light is generated and measured in a luminometer.","Immobilisation of the oligonucleotide probe: 100 \u03bcl of the POC capture probe (0.75 pmole\/ml in coupling buffer: 500 mM NaHPOpH 8.5, 0.5 mM EDTA) were transferred to each well (depression in a plate) in DNA-BIND plates (Corning Costar) and incubated overnight at 4\u00b0 C. The probe was then washed three times with 200 \u03bcl of coupling buffer and incubated for 1 hour at 37\u00b0 C. with 200 \u03bcl of blocking buffer (0.5% (w\/v) BSA in coupling buffer). After washing again with 200 \u03bcl of coupling buffer and 3\u00d7200 \u03bcl of hybridisation buffer, the plates can be used for the detection.","Hybridisation and detection: 10 \u03bcl of EDTA plasma or tissue homogenate were mixed with 90 \u03bcl of detection buffer (2 pmole\/\u03bcl POC detector probe in hybridisation buffer) and centrifuged. Further purifications were carried out as necessary. The batches were then denatured for 10 mins at 95\u00b0 C., transferred to the suitably prepared DNA-BIND wells (see above), and incubated for 45 mins at ca. 40\u00b0 C. The following wash steps were then carried out: 2\u00d7200 \u03bcl of hybridisation buffer and 3\u00d7200 \u03bcl 1\u00d7TBS\/Tween 20 (20 mM Tris-Cl pH 7.6, 137 mM NaCl, 0.1% (v\/v) Tween 20). 1 \u03bcl of streptavidin alkaline phosphatase conjugate (Promega) was diluted with 5 ml of 1\u00d7TBS\/Tween 20. 100 \u03bcl of the dilute conjugate were added to each well and incubated for one hour at room temperature. The following wash steps were then carried out: 2\u00d7200 \u03bcl of 1\u00d7TBS\/Tween 20 and 2\u00d7200 \u03bcl of 1\u00d7 assay buffer (20 mM Tris-Cl pH 9.8, 1 mM MgCl). 100 \u03bcl of CSPD \u201cReady-To-Use Substrate\u201d (Applied Biosystems) were then added, incubated for 30 mins at room temperature, and the chemiluminescence was measured in a Fluostar Optima multidetection plate reading device (BMG).","Result","The analysis of the weight distribution of the non-functional spiegelmer POC-3\u2032PEG, which was packed in micelles, showed after 24 hours a significantly higher concentration in the tumour tissues (24.925+\/\u221213.301 pmole\/mg) compared to the unpacked spiegelmer (0.840+\/\u22120.255 pmole\/mg) (). Whereas the concentration of the packed spiegelmer had halved (11.325+\/\u22127.050 pmole\/mg) after a further three days (96 hours), only a very small amount of the unpacked spiegelmer could be detected (0.120+\/\u22120.057 pmole\/mg).","The plasma level of unpacked spiegelmer (2.950+\/\u22120.438 pmole\/ml) after 24 hours was comparable to that of the PEI-packed spiegelmer (1.930+\/\u22122.729 pmole\/ml). After 96 hours clear differences were found, in which about four times the amount of packed spiegelmer compared to the unpacked spiegelmer was detected.","A slight accumulation in the kidneys was observed after 24 hours for both formulations, whereas a slight accumulation in the liver and gallbladder was found only for unpacked spiegelmer. After 96 hours, for both formulations only minor amounts of spiegelmer were detected in the liver and kidneys. On the other hand, slightly raised values were found in the gallbladder and pancreas (but with a high standard deviation) for packed spiegelmer compared to unpacked spiegelmer.","To summarise, compared to the weight distribution (24 and 96 hours after the last injection) of spiegelmers in the presence and absence of PEI, it was found that spiegelmer micelles have a significantly prolonged residence time in the plasma and tumour compared to unpacked material () and thus represent a promising approach to the use of spiegelmers directed against intracellular target molecules.","The following citations are incorporated herein by way of reference.\n\n","The features of the invention disclosed in the preceding description, claims and drawings can be essential both individually as well as in any combination for the implementation of the invention in its various embodiments."],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"p":["The present invention is described in more detail hereinafter with the aid of the following Figures and Examples, which disclose further features, embodiments and advantages. In this connection:",{"@attributes":{"id":"p-0297","num":"0376"},"figref":"FIG. 1A","smallcaps":"D"},{"@attributes":{"id":"p-0298","num":"0377"},"figref":"FIG. 1B","smallcaps":"D"},{"@attributes":{"id":"p-0299","num":"0378"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0300","num":"0379"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0301","num":"0380"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0302","num":"0381"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0303","num":"0382"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0304","num":"0383"},"figref":"FIG. 7"},{"@attributes":{"id":"p-0305","num":"0384"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0306","num":"0385"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0307","num":"0386"},"figref":"FIG. 10"},{"@attributes":{"id":"p-0308","num":"0387"},"figref":"FIG. 11"},{"@attributes":{"id":"p-0309","num":"0388"},"figref":"FIG. 12"},{"@attributes":{"id":"p-0310","num":"0389"},"figref":"FIG. 13"},{"@attributes":{"id":"p-0311","num":"0390"},"figref":"FIG. 14"},{"@attributes":{"id":"p-0312","num":"0391"},"figref":"FIG. 15"},{"@attributes":{"id":"p-0313","num":"0392"},"figref":"FIG. 16"},{"@attributes":{"id":"p-0314","num":"0393"},"figref":"FIG. 17"},{"@attributes":{"id":"p-0315","num":"0394"},"figref":"FIG. 18"},{"@attributes":{"id":"p-0316","num":"0395"},"figref":"FIG. 19"},{"@attributes":{"id":"p-0317","num":"0396"},"figref":"FIG. 20"},{"@attributes":{"id":"p-0318","num":"0397"},"figref":"FIG. 21"},{"@attributes":{"id":"p-0319","num":"0398"},"figref":"FIG. 22"}]},"DETDESC":[{},{}],"heading":["EXAMPLE 1","HMGA1a\/b-Binding Spiegelmers","EXAMPLE 2","Domain Approach","EXAMPLE 3","PEI-Spiegelmer Formulation","EXAMPLE 4","Bioactivity In Vivo","EXAMPLE 5","Effectiveness Study: Xenograft Model","EXAMPLE 6","Comparison of the In Vivo Tissue Distribution of Packed and Unpacked Spiegelmer"]}
