---
title: Nucleotide and amino acid sequences, and assays and methods of use thereof for diagnosis of cardiac disease
abstract: Novel markers for cardiac disease that are both sensitive and accurate. These markers are differentially and/or specifically expressed in cardiac tissue, as opposed to other types of tissues, optionally and preferably including muscle tissue. The measurement of these markers, alone or in combination, in patient samples provides information that the diagnostician can correlate with a probable diagnosis of cardiac disease, including pathology and/or damage, including acute and/or chronic damage. The markers of the present invention, alone or in combination, show a high degree of differential detection between cardiac disease states and non-cardiac disease states.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=07714100&OS=07714100&RS=07714100
owner: Compugen LTD
number: 07714100
owner_city: 
owner_country: IL
publication_date: 20071029
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"p":["This application is a CIP application of application Ser. No. 11\/043,824 filed Jan. 27, 2005 now U.S. Pat. No. 7,345,142.","This application is related to Novel Nucleotide and Amino Acid Sequences, and Assays and Methods of use thereof for Diagnosis of Cardiac Disease, and claims priority to the below U.S. provisional applications which are incorporated by reference herein:\n\n","The present invention is related to novel nucleotide and protein sequences that are diagnostic markers for cardiac disease and\/or pathological conditions, including cardiac damage, and assays and methods of use thereof.","Cardiovascular diseases are an important cause of mortality and morbidity. Amongst all age groups considered, IHD is the most common cause of death not only in men but also in women. Coronary atherosclerosis is a chronic progressing process, associated with angina type symptoms and frequently result in Acute Myocardial Infarction (AMI). The diagnosis is achieved with a combination of patient physical examination, ECG since 1950's molecular markers play the most important role in the differential diagnosis of AMI from other conditions with similar symptoms. Early diagnosis is mandatory of the establishment of early treatment (including blood diluting agents, thrombolysis, catheterization and surgery).","Early molecular markers for AMI were SGOT and LDH were proved to be of very low specificity and are hardly being used at present. These markers were replaced by CPK, and later on by the heart specific CPK-MB variant. Its specificity is better than for SGOT and LDH, it is still limited both in specificity and sensitivity which reach only 67% when used together with electrocardiogram. In addition, cardiac surgery, myocarditis, and electrical cardioversion often result in elevated serum levels of the CPK-MB isoenzyme. Small infarct with minor myocardial cell necrosis often do not increase serum CPK-MB to a detected level. Myoglobin is another heart damage low molecular (17 kD) protein but is even less specific to heart muscle compared with CPK-MB. Its advantage over CPK-MB is a rapid rise from the onset of symptoms\u2014usually between 3-6 hours. It is considered one of the earliest indicators (together with H-FABP) but it lacks specificity due to significant expression in skeletal muscle\u2014its concentration is approximately two-fold lower in cardiac than skeletal muscle and the leads to seriously diminished specificity.","Cardiac troponins are currently the routine serum cardiac markers used for the diagnosis of AMI. Troponin-I and Troponin-T have amino acid sequences different from those of the skeletal muscle called cTnT and cTnI (cardiac Troponin-T and I respectively). Cardiac troponins are not found in the serum of healthy individuals and rise to up to 20 times above a predefined cut-off level, therefore are very useful and sensitive in the detection of cardiac damage. They are capable of detecting very small cardiac damage\u2014micro-infarction, it is associated with a very adverse longer term prognosis. Cardiac troponin's sensitivity is considerably higher than CPK-MB but they suffer from a few disadvantages: 1. They are not early markers\u2014cTnI and cTnT reach peak serum value in about 12 and 48 hours respectively after symptoms onset. 2. Levels of cTnI and cTnT remain elevated for up to 10 days and 14 days respectively after AMI, therefore cannot be used for the detection of re-infarction. 3. Other heart diseases such as Congestive Heart Failure and Myocarditis can increase troponins concentrations in the serum. The lack of specificity for AMI is an advantage when there are other supporting clinical evidence directing the doctor towards another diagnosis. Troponins might have a diagnostic value in assessing myocardial damage after coronary artery perfusion, monitoring progression and prognosis of unstable angina, in the detection and prognosis of cardiac contusion after blunt trauma, detecting myocarditis.","The heart specific variant H-FABP (Heart Fatty Acid binding protein) is a low molecular protein (15 Kd) soluble non-enzyme protein. H-FABP concentration in the heart muscle is greater than that in skeletal muscle, and its normal baseline concentration is several fold lower than myoglobin. In addition, it reaches peak value in the urine and blood early, within 2-3 hours from AMI. Within a period of 30-210 minutes after symptoms started, H-FABP has higher sensitivity\u2014up to 80%\u2014when compared with other cardiac markers (CPK-MB and the troponins sensitivity were reported to be 64% in the first 6 hours after AMI). Yet, H-FABP still misses every 5patient in this time scale. H-FABP has other limitations as well, including 1. rising in the plasma after exercise 2. released from muscle in skeletal damage during the course of AMI (like from intramuscular injections) 3. reduced clearance in renal failure situations.","The search for novel cardiac damage markers is ongoing. Other proteins are under trials for that purpose including glycogen phosphorylase BB, HIF and VEGF 21.","Markers for the cardiac disease and\/or cardiac pathology, including but not limited to cardiac damage in the prior art are not sufficiently sensitive and\/or accurate, alone or in combination.","The present invention overcomes these deficiencies of the background art by providing novel markers for cardiac disease and\/or cardiac pathology, including but not limited to cardiac damage that are both sensitive and accurate. Optionally and preferably, these markers are detected in a biological sample.","According to preferred embodiments of the present invention, cardiac disease and\/or pathology and\/or condition and\/or disorder may comprise one or more of Myocardial infarct, acute coronary syndrome, angina pectoris (stable and unstable), cardiomyopathy, myocarditis, congestive heart failure or any type of heart failure, the detection of reinfarction, the detection of success of thrombolytic therapy after Myocardial infarct, Myocardial infarct after surgery, assessing the size of infarct in Myocardial infarct, the differential diagnosis of heart related conditions from lung related conditions (as pulmonary embolism), the differential diagnosis of Dyspnea, and cardiac valves related conditions.","According to preferred embodiments of the present invention, examples of suitable biological samples include but are not limited to blood, serum, plasma, blood cells, urine, sputum, saliva, stool, spinal fluid, lymph fluid, the external secretions of the skin, respiratory, intestinal, and genitourinary tracts, tears, milk, neuronal tissue, and any human organ or tissue. In a preferred embodiment, the biological sample comprises cardiac tissue and\/or a serum sample and\/or a urine sample and\/or any other tissue or liquid sample. The sample can optionally be diluted with a suitable eluant before contacting the sample to the antibody.","Information given in the text with regard to cellular localization was determined according to four different software programs: (i) tmhmm (from Center for Biological Sequence Analysis, Technical University of Denmark DTU,dot cbs dot dtu dot dk\/services\/TMHMM\/TMHMM2 dot 0b dot guide dot php) or (ii) tmpred (from EMBnet, maintained by the ISREC Bionformatics group and the LICR Information Technology Office, Ludwig Institute for Cancer Research, Swiss Institute of Bioinformatics, dot ch dot embnet dot org\/software\/TMPRED_form dot html) for transmembrane region prediction; (iii) signalp_hmm or (iv) signalp_nn (both from Center for Biological Sequence Analysis, Technical University of Denmark DTU, dot cbs dot dtu dot dklservices\/SignalP\/background\/prediction dot php) for signal peptide prediction. The terms \u201csignalp_hmm\u201d and \u201csignalp_nn\u201d refer to two modes of operation for the program SignalP: hmm refers to Hidden Markov Model, while nn refers to neural networks. Localization was also determined through manual inspection of known protein localization and\/or gene structure, and the use of heuristics by the individual inventor. In some cases for the manual inspection of cellular localization prediction inventors used the ProLoc computational platform [Einat Hazkani-Covo, Erez Levanon, Galit Rotman, Dan Graur and Amit Novik; (2004) \u201cEvolution of multicellularity in metazoa: comparative analysis of the subcellular localization of proteins in and \u201d Cell Biology International 2004; 28(3):171-8.], which predicts protein localization based on various parameters including, protein domains (e.g., prediction of trans-membranous regions and localization thereof within the protein), pI, protein length, amino acid composition, homology to pre-annotated proteins, recognition of sequence patterns which direct the protein to a certain organelle (such as, nuclear localization signal, NLS, mitochondria localization signal), signal peptide and anchor modeling and using unique domains from Pfam that are specific to a single compartment.","Information is given in the text with regard to SNPs (single nucleotide polymorphisms). A description of the abbreviations is as follows. \u201cT->C\u201d, for example, means that the SNP results in a change at the position given in the table from T to C. Similarly, \u201cM->Q\u201d, for example, means that the SNP has caused a change in the corresponding amino acid sequence, from methionine (M) to glutamine (Q). If, in place of a letter at the right hand side for the nucleotide sequence SNP, there is a space, it indicates that a frameshift has occurred. A frameshift may also be indicated with a hyphen (-). A stop codon is indicated with an asterisk at the right hand side (*). As part of the description of an SNP, a comment may be found in parentheses after the above description of the SNP itself. This comment may include an FTId, which is an identifier to a SwissProt entry that was created with the indicated SNP. An FTId is a unique and stable feature identifier, which allows construction of links directly from position-specific annotation in the feature table to specialized protein-related databases. The FTId is always the last component of a feature in the description field, as follows: FTId=XXX_number, in which XXX is the 3-letter code for the specific feature key, separated by an underscore from a 6-digit number. In the table of the amino acid mutations of the wild type proteins of the selected splice variants of the invention, the header of the first column is \u201cSNP position(s) on amino acid sequence\u201d, representing a position of a known mutation on amino acid sequence.","SNPs may optionally be used as diagnostic markers according to the present invention, alone or in combination with one or more other SNPs and\/or any other diagnostic marker. Preferred embodiments of the present invention comprise such SNPs, including but not limited to novel SNPs on the known (WT or wild type) protein sequences given below, as well as novel nucleic acid and\/or amino acid sequences formed through such SNPs, and\/or any SNP on a variant amino acid and\/or nucleic acid sequence described herein.","Information given in the text with regard to the Homology to the known proteins was determined by Smith-Waterman version 5.1.2 using special (non default) parameters as follows:\n\n","Information is given with regard to overexpression of a cluster in cancer based on microarrays. As a microarray reference, in the specific segment paragraphs, the unabbreviated tissue name was used as the reference to the type of chip for which expression was measured. There are two types of microarray results: those from microarrays prepared according to a design by the present inventors, for which the microarray fabrication procedure is described in detail in Materials and Experimental Procedures section herein; and those results from microarrays using Affymetrix technology. As a microarray reference, in the specific segment paragraphs, the unabbreviated tissue name was used as the reference to the type of chip for which expression was measured. For microarrays prepared according to a design by the present inventors, the probe name begins with the name of the cluster (gene), followed by an identifying number. Oligonucleotide microarray results taken from Affymetrix data were from chips available from Affymetrix Inc, Santa Clara, Calif., USA (see for example data regarding the Human Genome U133 (HG-U133) Set at dot affymetrix dot com\/products\/arrays\/specific\/hgu133 dot affx; GeneChip Human Genome U133A 2.0 Array at dot affymetrix dot com\/products\/arrays\/specific\/hgu133av2 dot affx; and Human Genome U133 Plus 2.0 Array at dot affymetrix dot com\/products\/arrays\/specific\/hgu133plus dot affx). The probe names follow the Affymetrix naming convention. The data is available from NCBI Gene Expression Omnibus (see dot ncbi dot nlm dot nih dot gov\/projects\/geo\/ and Edgar et al, Nucleic Acids Research, 2002, Vol. 30, No. 1 207-210). The dataset (including results) is available from dot ncbi dot nlm dot nih dot gov\/geo\/query\/acc dot cgi?acc=GSE1133 for the Series GSE1133 database (published on March 2004); a reference to these results is as follows: Su et al (Proc Natl Acad Sci USA. 2004 il 20; 101(16):6062-7. Epub 2004 Apr. 9). Oligonucleotide probes for use with arrays designed by the present inventors:\n\n","In the second histogram(s) of the heart paragraph, the oligo-probe names are abbreviated\/enumerated as follows:",{"@attributes":{"id":"p-0020","num":"0064"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}}],"thead":{"row":{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"\u201cadipocyte\u201d,","\u201cA1\u201d;"]},{"entry":[{},"\u201cadrenalcortex\u201d,","\u201cA2\u201d;"]},{"entry":[{},"\u201cadrenalgland\u201d,","\u201cA3\u201d;"]},{"entry":[{},"\u201camygdala\u201d,","\u201cA4\u201d;"]},{"entry":[{},"\u201cappendix\u201d,","\u201cA5\u201d;"]},{"entry":[{},"\u201catrioventricularnode\u201d,","\u201cA6\u201d;"]},{"entry":[{},"\u201cbm_cd105_endothelial\u201d,","\u201cE1\u201d;"]},{"entry":[{},"\u201cbm_cd33_myeloid\u201d,","\u201cM1\u201d;"]},{"entry":[{},"\u201cbm_cd34_\u201d,","\u201cB1\u201d;"]},{"entry":[{},"\u201cbm_cd71_earlyerythroid\u201d,","\u201cE1\u201d;"]},{"entry":[{},"\u201cbonemarrow\u201d,","\u201cB2\u201d;"]},{"entry":[{},"\u201cbronchialepithelialcells\u201d,","\u201cB3\u201d;"]},{"entry":[{},"\u201ccardiacmyocytes\u201d,","\u201cC1\u201d;"]},{"entry":[{},"\u201ccaudatenucleus\u201d,","\u201cC2\u201d;"]},{"entry":[{},"\u201ccerebellum\u201d,","\u201cC3\u201d;"]},{"entry":[{},"\u201ccerebellumpeduncles\u201d,","\u201cC4\u201d;"]},{"entry":[{},"\u201cciliaryganglion\u201d,","\u201cC5\u201d;"]},{"entry":[{},"\u201ccingulatecortex\u201d,","\u201cC6\u201d;"]},{"entry":[{},"\u201cglobuspallidus\u201d,","\u201cG1\u201d;"]},{"entry":[{},"\u201cheart\u201d,","\u201cH1\u201d;"]},{"entry":[{},"\u201chypothalamus\u201d,","\u201cH2\u201d;"]},{"entry":[{},"\u201ckidney\u201d,","\u201cK1\u201d;"]},{"entry":[{},"\u201cliver\u201d,","\u201cL1\u201d;"]},{"entry":[{},"\u201clung\u201d,","\u201cL2\u201d;"]},{"entry":[{},"\u201clymphnode\u201d,","\u201cL3\u201d;"]},{"entry":[{},"\u201cmedullaoblongata\u201d,","\u201cM1\u201d;"]},{"entry":[{},"\u201coccipitallobe\u201d,","\u201cO1\u201d;"]},{"entry":[{},"\u201colfactorybulb\u201d,","\u201cO2\u201d;"]},{"entry":[{},"\u201covary\u201d,","\u201cO3\u201d;"]},{"entry":[{},"\u201cpancreas\u201d,","\u201cP1\u201d;"]},{"entry":[{},"\u201cpancreaticislets\u201d,","\u201cP2\u201d;"]},{"entry":[{},"\u201cparietallobe\u201d,","\u201cP3\u201d;"]},{"entry":[{},"\u201cpb_bdca4_dentritic_cells\u201d,","\u201cP4\u201d;"]},{"entry":[{},"\u201cpb_cd14_monocytes\u201d,","\u201cP5\u201d;"]},{"entry":[{},"\u201cpb_cd19_bcells\u201d,","\u201cP6\u201d;"]},{"entry":[{},"\u201cpb_cd4_tcells\u201d,","\u201cP7\u201d;"]},{"entry":[{},"\u201cpb_cd56_nkcells\u201d,","\u201cP8\u201d;"]},{"entry":[{},"\u201cpb_cd8_tcells\u201d,","\u201cP9\u201d;"]},{"entry":[{},"\u201cpituitary\u201d,","\u201cPa\u201d;"]},{"entry":[{},"\u201cplacenta\u201d,","\u201cPb\u201d;"]},{"entry":[{},"\u201cpons\u201d,","\u201cPc\u201d;"]},{"entry":[{},"\u201cprefrontalcortex\u201d,","\u201cPd\u201d;"]},{"entry":[{},"\u201cprostate\u201d,","\u201cPe\u201d;"]},{"entry":[{},"\u201csalivarygland\u201d,","\u201cS1\u201d;"]},{"entry":[{},"\u201cskeletalmuscle\u201d,","\u201cS2\u201d;"]},{"entry":[{},"\u201cskin\u201d,","\u201cS3\u201d;"]},{"entry":[{},"\u201csmoothmuscle\u201d,","\u201cS4\u201d;"]},{"entry":[{},"\u201cspinalcord\u201d,","\u201cS5\u201d;"]},{"entry":[{},"\u201csubthalamicnucleus\u201d,","\u201cS6\u201d;"]},{"entry":[{},"\u201csuperiorcervicalganglion\u201d,","\u201cS7\u201d;"]},{"entry":[{},"\u201ctemporallobe\u201d,","\u201cT1\u201d;"]},{"entry":[{},"\u201ctestis\u201d,","\u201cT2\u201d;"]},{"entry":[{},"\u201ctestisgermcell\u201d,","\u201cT3\u201d;"]},{"entry":[{},"\u201ctestisinterstitial\u201d,","\u201cT4\u201d;"]},{"entry":[{},"\u201ctestisleydigcell\u201d,","\u201cT5\u201d;"]},{"entry":[{},"\u201ctestisseminiferoustubule\u201d,","\u201cS6\u201d;"]},{"entry":[{},"\u201cthalamus\u201d,","\u201cT7\u201d;"]},{"entry":[{},"\u201cthymus\u201d,","\u201cT8\u201d;"]},{"entry":[{},"\u201cthyroid\u201d,","\u201cT9\u201d;"]},{"entry":[{},"\u201ctonsil\u201d,","\u201cTa\u201d;"]},{"entry":[{},"\u201ctrachea\u201d,","\u201cTb\u201d;"]},{"entry":[{},"\u201ctrigeminalganglion\u201d,","\u201cTc\u201d;"]},{"entry":[{},"\u201cuterus\u201d,","\u201cU1\u201d;"]},{"entry":[{},"\u201cuteruscorpus\u201d,","\u201cU2\u201d;"]},{"entry":[{},"\u201cwholeblood\u201d,","\u201cW1\u201d;"]},{"entry":[{},"\u201cwholebrain\u201d,","\u201cW2\u201d;"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}}}},"It should be noted that the terms \u201csegment\u201d, \u201cseg\u201d and \u201cnode\u201d are used interchangeably in reference to nucleic acid sequences of the present invention; they refer to portions of nucleic acid sequences that were shown to have one or more properties as described below. They are also the building blocks that were used to construct complete nucleic acid sequences as described in greater detail below. Optionally and preferably, they are examples of oligonucleotides which are embodiments of the present invention, for example as amplicons, hybridization units and\/or from which primers and\/or complementary oligonucleotides may optionally be derived, and\/or for any other use.","As used herein the phrase \u201ccardiac disease\u201d includes any type of cardiac pathology and\/or disorder and\/or damage, including both chronic and acute damage, as well as progression from acute to chronic damage of the heart, and also propagation of one acute event to another acute event. An example of the latter may occur when an infarct is followed by another infarct in a relatively short period of time, such as within 24 hours for example. An infarct may also lead to acute heart failure immediately after the infarct, as another example. These non-limiting examples are intended to demonstrate that cardiac disease may also comprise a plurality of acute events.","The term \u201cmarker\u201d in the context of the present invention refers to a nucleic acid fragment, a peptide, or a polypeptide, which is differentially present in a sample taken from patients having a cardiac disease, such as acute cardiac damage for example, as compared to a comparable sample taken from subjects who do not have cardiac disease.","As used herein the phrase \u201cdifferentially present\u201d refers to differences in the quantity of a marker present in a sample taken from patients having cardiac disease as compared to a comparable sample taken from patients who do not have cardiac disease. For example, a nucleic acid fragment may optionally be differentially present between the two samples if the amount of the nucleic acid fragment in one sample is significantly different from the amount of the nucleic acid fragment in the other sample, for example as measured by hybridization and\/or NAT-based assays. A polypeptide is differentially present between the two samples if the amount of the polypeptide in one sample is significantly different from the amount of the polypeptide in the other sample. It should be noted that if the marker is detectable in one sample and not detectable in the other, then such a marker can be considered to be differentially present. For example, in the case of acute cardiac damage, it is possible that a marker (such as a protein or fragment thereof) could optionally be present in a blood sample from the patient, indicating the presence of damage; lack of presence of such a marker (and\/or presence at a low level) would therefore optionally and preferably indicate a lack of such damage. Alternatively, chronically damaged heart might cause a low level of the marker to be present in the blood sample, while acute damage would cause a high level to be present. One of ordinary skill in the art could easily determine such relative levels of the markers; further guidance is provided in the description of each individual marker below.","As used herein the phrase \u201cdiagnostic\u201d means identifying the presence or nature of a pathologic condition. bDiagnostic methods differ in their sensitivity and specificity. The \u201csensitivity\u201d of a diagnostic assay is the percentage of diseased individuals who test positive (percent of \u201ctrue positives\u201d). Diseased individuals not detected by the assay are \u201cfalse negatives.\u201d Subjects who are not diseased and who test negative in the assay are termed \u201ctrue negatives.\u201d The \u201cspecificity\u201d of a diagnostic assay is 1 minus the false positive rate, where the \u201cfalse positive\u201d rate is defined as the proportion of those without the disease who test positive. While a particular diagnostic method may not provide a definitive diagnosis of a condition, it suffices if the method provides a positive indication that aids in diagnosis.","As used herein the phrase \u201cdiagnosing\u201d refers to classifying a disease or a symptom, determining a severity of the disease, monitoring disease progression, forecasting an outcome of a disease and\/or prospects of recovery. The term \u201cdetecting\u201d may also optionally encompass any of the above.","Diagnosis of a disease according to the present invention can be effected by determining a level of a polynucleotide or a polypeptide of the present invention in a biological sample obtained from the subject, wherein the level determined can be correlated with predisposition to, or presence or absence of the disease. It should be noted that a \u201cbiological sample obtained from the subject\u201d may also optionally comprise a sample that has not been physically removed from the subject, as described in greater detail below.","As used herein, the term \u201clevel\u201d refers to expression levels of RNA and\/or protein or to DNA copy number of a marker of the present invention.","Typically the level of the marker in a biological sample obtained from the subject is different (i.e., increased or decreased) from the level of the same variant in a similar sample obtained from a healthy individual (examples of biological samples are described herein).","Numerous well known tissue or fluid collection methods can be utilized to collect the biological sample from the subject in order to determine the level of DNA, RNA and\/or polypeptide of the variant of interest in the subject.","Examples include, but are not limited to, fine needle biopsy, needle biopsy, core needle biopsy and surgical biopsy (e.g., brain biopsy), and lavage. Regardless of the procedure employed, once a biopsy\/sample is obtained the level of the variant can be determined and a diagnosis can thus be made.","Determining the level of the same variant in normal tissues of the same origin is preferably effected along-side to detect an elevated expression and\/or amplification and\/or a decreased expression, of the variant as opposed to the normal tissues.","A \u201ctest amount\u201d of a marker refers to an amount of a marker present in a sample being tested. A test amount can be either in absolute amount (e.g., microgram\/ml) or a relative amount (e.g., relative intensity of signals).","A \u201ctest amount\u201d of a marker refers to an amount of a marker in a subject's sample that is consistent with a diagnosis of cardiac disease. A test amount can be either in absolute amount (e.g., microgram\/ml) or a relative amount (e.g., relative intensity of signals).","A \u201ccontrol amount\u201d of a marker can be any amount or a range of amounts to be compared against a test amount of a marker. For example, a control amount of a marker can be the amount of a marker in a patient with cardiac disease or a person without cardiac disease. A control amount can be either in absolute amount (e.g., microgram\/ml) or a relative amount (e.g., relative intensity of signals).","\u201cDetect\u201d refers to identifying the presence, absence or amount of the object to be detected.","A \u201clabel\u201d includes any moiety or item detectable by spectroscopic, photo chemical, biochemical, immunochemical, or chemical means. For example, useful labels include P, S, fluorescent dyes, electron-dense reagents, enzymes (e.g., as commonly used in an ELISA), biotin-streptavadin, dioxigenin, haptens and proteins for which antisera or monoclonal antibodies are available, or nucleic acid molecules with a sequence complementary to a target. The label often generates a measurable signal, such as a radioactive, chromogenic, or fluorescent signal, that can be used to quantify the amount of bound label in a sample. The label can be incorporated in or attached to a primer or probe either covalently, or through ionic, van der Waals or hydrogen bonds, e.g., incorporation of radioactive nucleotides, or biotinylated nucleotides that are recognized by streptavadin. The label may be directly or indirectly detectable. Indirect detection can involve the binding of a second label to the first label, directly or indirectly. For example, the label can be the ligand of a binding partner, such as biotin, which is a binding partner for streptavadin, or a nucleotide sequence, which is the binding partner for a complementary sequence, to which it can specifically hybridize. The binding partner may itself be directly detectable, for example, an antibody may be itself labeled with a fluorescent molecule. The binding partner also may be indirectly detectable, for example, a nucleic acid having a complementary nucleotide sequence can be a part of a branched DNA molecule that is in turn detectable through hybridization with other labeled nucleic acid molecules (see, e.g., P. D. Fahrlander and A. Klausner, Bio\/Technology 6:1165 (1988)). Quantitation of the signal is achieved by, e.g., scintillation counting, densitometry, or flow cytometry.","Exemplary detectable labels, optionally and preferably for use with immunoassays, include but are not limited to magnetic beads, fluorescent dyes, radiolabels, enzymes (e.g., horse radish peroxide, alkaline phosphatase and others commonly used in an ELISA), and calorimetric labels such as colloidal gold or colored glass or plastic beads. Alternatively, the marker in the sample can be detected using an indirect assay, wherein, for example, a second, labeled antibody is used to detect bound marker-specific antibody, and\/or in a competition or inhibition assay wherein, for example, a monoclonal antibody which binds to a distinct epitope of the marker are incubated simultaneously with the mixture.","\u201cImmunoassay\u201d is an assay that uses an antibody to specifically bind an antigen. The immunoassay is characterized by the use of specific binding properties of a particular antibody to isolate, target, and\/or quantify the antigen.","The phrase \u201cspecifically (or selectively) binds\u201d to an antibody or \u201cspecifically (or selectively) immunoreactive with,\u201d when referring to a protein or peptide (or other epitope), refers to a binding reaction that is determinative of the presence of the protein in a heterogeneous population of proteins and other biologics. Thus, under designated immunoassay conditions, the specified antibodies bind to a particular protein at least two times greater than the background (non-specific signal) and do not substantially bind in a significant amount to other proteins present in the sample. Specific binding to an antibody under such conditions may require an antibody that is selected for its specificity for a particular protein. For example, polyclonal antibodies raised to seminal basic protein from specific species such as rat, mouse, or human can be selected to obtain only those polyclonal antibodies that are specifically immunoreactive with seminal basic protein and not with other proteins, except for polymorphic variants and alleles of seminal basic protein. This selection may be achieved by subtracting out antibodies that cross-react with seminal basic protein molecules from other species. A variety of immunoassay formats may be used to select antibodies specifically immunoreactive with a particular protein. For example, solid-phase ELISA immunoassays are routinely used to select antibodies specifically immunoreactive with a protein (see, e.g., Harlow & Lane, Antibodies, A Laboratory Manual (1988), for a description of immunoassay formats and conditions that can be used to determine specific immunoreactivity). Typically a specific or selective reaction will be at least twice background signal or noise and more typically more than 10 to 100 times background.","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a transcript selected from the group consisting of SEQ ID NOs: 1, 2, 3 and 4.","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a segment selected from the group consisting of SEQ ID NOs: 65, 66, 67, 68, 69, 70, 71 and 72.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide comprising a protein variant selected from the group consisting of SEQ ID NOs: 281, 282, 283 and 284.","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a transcript selected from the group consisting of SEQ ID NOs: 5, 6, 7, 8, 9 and 10","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a segment selected from the group consisting of SEQ ID NOs: 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93 and 94.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide comprising a protein variant selected from the group consisting of SEQ ID NOs: 285, 286, 287, 288, 289, 290 and 291","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a transcript SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 12, 13, 14, 15, 16 and 17","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a segment SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide comprising a protein variant SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 292, 293, 294, 295 and 296","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a transcript SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 18 and 19.","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a segment SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 113, 114, 115, 116, 117, 118, 119, 120, 121 and 122.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide comprising a protein variant SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 297 and 298.","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a transcript SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 20 and 21.","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a segment SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 123, 124, 125, 126, 127, 128 and 129.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide comprising a protein variant SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 299 and 300.","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a transcript SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 26, 27, 28, 29 and 30.","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a segment SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160, 161, 162 and 163.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide comprising a protein variant SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 305; 306; 307 and 308","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a transcript SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 31, 32, 33, 34, 35, 36 and 37.","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a segment SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 164, 165, 166, 167, 168, 169, 170, 171, 172, 173, 174, 175, 176, 177, 178, 179, 180, 181, 182, 183, 184, 185 and 186","According to preferred embodiments of the present invention, there is provided an isolated polypeptide comprising a protein variant SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 309, 310, 311 and 312.","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a transcript SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 38, 39, 40 and 41.","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a segment SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 187, 188, 189, 190, 191, 192, 193, 194, 195 and 196.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide comprising a protein variant SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 313, 314, 315 and 316.","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a transcript SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 42, 43, 44, 45, 46, 47, 48, 49 and 50.","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a segment SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 197, 198, 199, 200, 201, 202, 203, 204, 205, 206, 207 and 208.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide comprising a protein variant SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 317, 318, 319, 320, 321, 322, 323, 324 and 325.","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a transcript SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs:51, 52, 53, 54, 55, 56, 57, 58, 59 and 60.","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a segment SELECTED FROM THE GROUP CONSISTING OF SEQ ID NOs: 209 to 273.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide comprising a protein variant selected from the group consisting of SEQ ID NOs: 326 to 334.","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a transcript selected from the group consisting of SEQ ID NOs: 22-25, 353 or 386.","According to preferred embodiments of the present invention, there is provided an isolated polynucleotide comprising a segment selected from the group consisting of SEQ ID NOs: 130-149.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide comprising a protein variant selected from the group consisting of SEQ ID NOs: 301-304, 325, 354-356 or 387.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-1855 of SEQ ID NO. 338, which also corresponds to amino acids 1-1855 of SEQ ID NO .326, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 1856-1904 of SEQ ID NO. 326, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 326, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VRRTPDTGSRCGSFFSGPTAPPSQGSSHLLLEMLLVDLTFFSRSAVSLT (SEQ ID NO:394) in SEQ ID NO. 326.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-1326 of SEQ ID NO. 339, which also corresponds to amino acids 1-1326 of SEQ ID NO. 327, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 1327-1336 of SEQ ID NO. 327, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 327, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VRPSGEGGQA (SEQ ID NO:431) in SEQ ID NO. 327.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to corresponding to amino acids 1-1508 of SEQ ID NO. 339, which also corresponds to amino acids 1-1508 of SEQ ID NO. 328, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 1509-1534 of SEQ ID NO. 328, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 328, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence GVLGVQEARDELVGGRAMQGQGEHRL (SEQ ID NO:432) in SEQ ID NO. 328.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to corresponding to amino acids 1-1763 of SEQ ID NO. 338, which also corresponds to amino acids 1-1763 of SEQ ID NO. 329, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 1764-1788 of SEQ ID NO. 329, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 329, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VSDRPPSASPKDRNKALGPGQATVL (SEQ ID NO:432) in SEQ ID NO. 329.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 1-22 of SEQ ID NO. 330, and a second amino acid sequence being at least 90% homologous to amino acids 528-1939 of SEQ ID NO. 340, which also corresponds to amino acids 23-1434 of SEQ ID NO. 330, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a head of SEQ ID NO. 330, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence MGLWKPGSVLSDSLFASSPCPQ (SEQ ID NO:395) of SEQ ID NO. 330.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-527 of SEQ ID NO. 339, which also corresponds to amino acids 1-527 of SEQ ID NO. 331, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 528-555 of SEQ ID NO. 331, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 331, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VPPWPHHLCPLLCHPDKVVAESLLHPRN (SEQ ID NO:435) in SEQ ID NO. 331.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to corresponding to amino acids 1-470 of SEQ ID NO .338, which also corresponds to amino acids 1-470 of SEQ ID NO .332, a second amino acid sequence being at least 90% homologous to amino acids 528-1855 of SEQ ID NO. 338, which also corresponds to amino acids 471-1798 of SEQ ID NO. 332, and a third amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 1799-1847 of SEQ ID NO .332, wherein said first amino acid sequence, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide encoding for an edge portion of SEQ ID NO. 332, comprising a polypeptide having a length \u201cn\u201d, wherein n is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least two amino acids comprise DP, having a structure as follows: a sequence starting from any of amino acid numbers 470\u2212x to 470; and ending at any of amino acid numbers 471+((n\u22122)\u2212x), in which x varies from 0 to n\u22122.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 332, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VRRTPDTGSRCGSFFSGPTAPPSQGSSHLLLEMLLVDLTFFSRSAVSLT (SEQ ID NO:394) in SEQ ID NO. 332.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 165-1939 of SEQ ID NO. 340, which also corresponds to amino acids 1-1775 of SEQ ID NO. 333.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to corresponding to amino acids 1165-1939 of SEQ ID NO. 340, which also corresponds to amino acids 1-775 of SEQ ID NO. 334.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-158 of SEQ ID NO. 341, which also corresponds to amino acids 1-158 of SEQ ID NO. 317.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-156 of SEQ ID NO. 341, which also corresponds to amino acids 1-156 of SEQ ID NO. 318, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 157-166 of SEQ ID NO. 318, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 318, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VSVGQECGSG (SEQ ID NO:423) in SEQ ID NO. 318.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-156 of SEQ ID NO. 341, which also corresponds to amino acids 1-156 of SEQ ID NO. 319, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 157-210 of SEQ ID NO. 319, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 319, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence DGISSLCYSSLSKSLLSQPLRETSSAINDISLLQALMPLLGWTSHWTCITVGLY (SEQ ID NO:424) in SEQ ID NO. 319.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-60 of Q96NR4 (SEQ ID NO:342), which also corresponds to amino acids 1-60 of SEQ ID NO. 320, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 61-114 of SEQ ID NO. 320, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 320, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence DGISSLCYSSLSKSLLSQPLRETSSAINDISLLQALMPLLGWTSHWTCITVGLY (SEQ ID NO:424) in SEQ ID NO. 320.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 97-156 of SEQ ID NO. 341, which also corresponds to amino acids 1-60 of SEQ ID NO. 320, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 61-114 of SEQ ID NO. 320, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-14 of SEQ ID NO. 342, which also corresponds to amino acids 1-14 of SEQ ID NO. 321, a second amino acid sequence bridging amino acid sequence comprising of S, and a third amino acid sequence being at least 90% homologous to corresponding to amino acids 62-133 of SEQ ID NO. 342, which also corresponds to amino acids 16-87 of SEQ ID NO. 321, wherein said first amino acid sequence, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for an edge portion of SEQ ID NO. 321, comprising a polypeptide having a length \u201cn\u201d, wherein n is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least three amino acids comprise VSI having a structure as follows (numbering according to SEQ ID NO. 321): a sequence starting from any of amino acid numbers 14\u2212x to 14; and ending at any of amino acid numbers 16+((n\u22122)\u2212x), in which x varies from 0 to n\u22122.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 1-15 of SEQ ID NO. 321, and a second amino acid sequence being at least 90% homologous to corresponding to amino acids 39-110 of SEQ ID NO. 343, which also corresponds to amino acids 16-87 of SEQ ID NO. 321, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a head of SEQ ID NO. 321, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence MRGEHNSTSYDSAVS (SEQ ID NO:426) of SEQ ID NO. 321.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to corresponding to amino acids 97-110 of SEQ ID NO. 341, which also corresponds to amino acids 1-14 of SEQ ID NO. 321, a second amino acid sequence bridging amino acid sequence comprising of S, and a third amino acid sequence being at least 90% homologous to corresponding to amino acids 158-229 of SEQ ID NO. 341, which also corresponds to amino acids 16-87 of SEQ ID NO. 321, wherein said first amino acid sequence, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for an edge portion of SEQ ID NO. 321, comprising a polypeptide having a length \u201cn\u201d, wherein n is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least three amino acids comprise VSI having a structure as follows (numbering according to SEQ ID NO. 321): a sequence starting from any of amino acid numbers 14\u2212x to 14; and ending at any of amino acid numbers 16+((n\u22122)\u2212x), in which x varies from 0 to n\u22122.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 320, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence DGISSLCYSSLSKSLLSQPLRETSSAINDISLLQALMPLLGWTSHWTCITVGLY (SEQ ID NO:424) in SEQ ID NO. 320.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to corresponding to amino acids 1-14 of SEQ ID NO. 342, which also corresponds to amino acids 1-14 of SEQ ID NO. 321, a second amino acid sequence bridging amino acid sequence comprising of S, and a third amino acid sequence being at least 90% homologous to corresponding to amino acids 62-133 of SEQ ID NO. 342, which also corresponds to amino acids 16-87 of SEQ ID NO. 321, wherein said first amino acid sequence, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for an edge portion of SEQ ID NO. 321, comprising a polypeptide having a length \u201cn\u201d, wherein n is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least three amino acids comprise VSI having a structure as follows (numbering according to SEQ ID NO. 321: a sequence starting from any of amino acid numbers 14\u2212x to 14; and ending at any of amino acid numbers 16+((n\u22122)\u2212x), in which x varies from 0 to n\u22122.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 1-15 of SEQ ID NO. 321, and a second amino acid sequence being at least 90% homologous to corresponding to amino acids 39-110 of SEQ ID NO. 343, which also corresponds to amino acids 16-87 of SEQ ID NO. 321, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a head of SEQ ID NO. 321, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence MRGEHNSTSYDSAVS (SEQ ID NO:426) of SEQ ID NO. 321.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to corresponding to amino acids 97-110 of SEQ ID NO. 341, which also corresponds to amino acids 1-14 of SEQ ID NO. 321, a second amino acid sequence bridging amino acid sequence comprising of S, and a third amino acid sequence being at least 90% homologous to corresponding to amino acids 158-229 of SEQ ID NO. 341, which also corresponds to amino acids 16-87 of SEQ ID NO. 321, wherein said first amino acid sequence, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for an edge portion of SEQ ID NO. 321, comprising a polypeptide having a length \u201cn\u201d, wherein n is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least three amino acids comprise VSI having a structure as follows (numbering according to SEQ ID NO. 321): a sequence starting from any of amino acid numbers 14\u2212x to 14; and ending at any of amino acid numbers 16+((n\u22122)\u2212x), in which x varies from 0 to n\u22122.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to corresponding to amino acids 1-62 of SEQ ID NO. 342, which also corresponds to amino acids 1-62 of SEQ ID NO. 322.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 1-23 of SEQ ID NO. 322., and a second amino acid sequence being at least 90% homologous to corresponding to amino acids 1-39 of SEQ ID NO. 343., which also corresponds to amino acids 24-62 of SEQ ID NO. 322., wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a head of SEQ ID NO. 322., comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence MRGEHNSTSYDSAVIYRGFWAVL (SEQ ID NO:427) of SEQ ID NO. 322.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to corresponding to amino acids 97-158 of SEQ ID NO. 341., which also corresponds to amino acids 1-62 of SEQ ID NO. 322.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to corresponding to amino acids 1-60 of SEQ ID NO. 342, which also corresponds to amino acids 1-60 of SEQ ID NO. 324, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 61-70 of SEQ ID NO. 324, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 324, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VSVGQECGSG (SEQ ID NO:423) in SEQ ID NO. 324.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 1-23 of SEQ ID NO. 324, a second amino acid sequence being at least 90% homologous to corresponding to amino acids 1-37 of SEQ ID NO. 343, which also corresponds to amino acids 24-60 of SEQ ID NO. 324, and a third amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence corresponding to amino acids 61-70 of SEQ ID NO. 324, wherein said first amino acid sequence, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a head of SEQ ID NO. 324, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence MRGEHNSTSYDSAVIYRGFWAVL (SEQ ID NO:427) of SEQ ID NO. 324.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 324, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VSVGQECGSG (SEQ ID NO:423) in SEQ ID NO. 324.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to corresponding to amino acids 97-156 of SEQ ID NO. 341, which also corresponds to amino acids 1-60 of SEQ ID NO. 324, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence corresponding to amino acids 61-70 of SEQ ID NO. 324, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 324, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VSVGQECGSG (SEQ ID NO:423) in SEQ ID NO. 324.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to corresponding to amino acids 1-115 of SEQ ID NO. 344, which also corresponds to amino acids 1-115 of SEQ ID NO. 313, and a second amino acid sequence being at least 90% homologous to corresponding to amino acids 152-319 of SEQ ID NO. 344, which also corresponds to amino acids 116-283 of SEQ ID NO. 313, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide encoding for an edge portion of SEQ ID NO. 313, comprising a polypeptide having a length \u201cn\u201d, wherein n is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least two amino acids comprise IY, having a structure as follows: a sequence starting from any of amino acid numbers 115\u2212x to 115; and ending at any of amino acid numbers 116+((n\u22122)\u2212x), in which x varies from 0 to n\u22122.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, of cluster Z36249 comprising a first amino acid sequence being at least 90% homologous to corresponding to amino acids 1-70 of SEQ ID NO. 345, which also corresponds to amino acids 1-70 of SEQ ID NO. 313, a bridging amino acid K corresponding to amino acid 71 of SEQ ID NO. 313, a second amino acid sequence being at least 90% homologous to corresponding to amino acids 72-115 of SEQ ID NO. 345, which also corresponds to amino acids 72-115 of SEQ ID NO. 313, and a third amino acid sequence being at least 90% homologous to corresponding to amino acids 152-319 of SEQ ID NO. 345, which also corresponds to amino acids 116-283 of SEQ ID NO. 313, wherein said first amino acid sequence, bridging amino acid, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to corresponding to amino acids 1-184 of SEQ ID NO. 344, which also corresponds to amino acids 1-184 of SEQ ID NO. 314, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 185-197 of SEQ ID NO. 314, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 314, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VNIFLCLGMSQKK (SEQ ID NO:421) in SEQ ID NO. 314.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to corresponding to amino acids 1-70 of SEQ ID NO. 345, which also corresponds to amino acids 1-70 of SEQ ID NO. 314, a bridging amino acid K corresponding to amino acid 71 of SEQ ID NO. 314, a second amino acid sequence being at least 90% homologous to corresponding to amino acids 72-184 of SEQ ID NO. 345, which also corresponds to amino acids 72-184 of SEQ ID NO. 314, and a third amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence corresponding to amino acids 185-197 of SEQ ID NO. 314, wherein said first amino acid sequence, bridging amino acid, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 314, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VNIFLCLGMSQKK (SEQ ID NO:421) in SEQ ID NO. 314.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide encoding for an edge portion of SEQ ID NO. 313, comprising a polypeptide having a length \u201cn\u201d, wherein n is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least two amino acids comprise IY, having a structure as follows: a sequence starting from any of amino acid numbers 115\u2212x to 115; and ending at any of amino acid numbers 116+((n\u22122)\u2212x), in which x varies from 0 to n\u22122.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to corresponding to amino acids 1-151 of SEQ ID NO. 344, which also corresponds to amino acids 1-151 of SEQ ID NO. 315, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 152-177 of SEQ ID NO. 315, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 315, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VRLMQSTAKSSSLILCFLCFTPVLLI (SEQ ID NO:422) in SEQ ID NO. 315.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-70 of SEQ ID NO. 345, which also corresponds to amino acids 1-70 of SEQ ID NO. 315, a bridging amino acid K corresponding to amino acid 71 of SEQ ID NO. 315, a second amino acid sequence being at least 90% homologous to amino acids 72-151 of SEQ ID NO. 345, which also corresponds to amino acids 72-151 of SEQ ID NO. 315, and a third amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 152-177 of SEQ ID NO. 315, wherein said first amino acid sequence, bridging amino acid, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 315, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VRLMQSTAKSSSLILCFLCFTPVLLI (SEQ ID NO:422) in SEQ ID NO. 315.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-151 of SEQ ID NO. 344, which also corresponds to amino acids 1-151 of SEQ ID NO. 316, and a second amino acid sequence being at least 90% homologous to amino acids 185-319 of SEQ ID NO. 344, which also corresponds to amino acids 152-286 of SEQ ID NO. 316, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide encoding for an edge portion of SEQ ID NO. 316, comprising a polypeptide having a length \u201cn\u201d, wherein n is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least two amino acids comprise EL, having a structure as follows: a sequence starting from any of amino acid numbers 151\u2212x to 151; and ending at any of amino acid numbers 152+((n\u22122)\u2212x), in which x varies from 0 to n\u22122.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-70 of SEQ ID NO. 345, which also corresponds to amino acids 1-70 of SEQ ID NO. 316, a bridging amino acid K corresponding to amino acid 71 of SEQ ID NO. 316, a second amino acid sequence being at least 90% homologous to amino acids 72-151 of SEQ ID NO. 345, which also corresponds to amino acids 72-151 of SEQ ID NO. 316, and a third amino acid sequence being at least 90% homologous to amino acids 185-319 of SEQ ID NO. 345, which also corresponds to amino acids 152-286 of SEQ ID NO. 316, wherein said first amino acid sequence, bridging amino acid, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide encoding for an edge portion of SEQ ID NO. 316, of cluster Z36249 comprising a polypeptide having a length \u201cn\u201d, wherein n is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least two amino acids comprise EL, having a structure as follows: a sequence starting from any of amino acid numbers 151\u2212x to 151; and ending at any of amino acid numbers 152+((n\u22122)\u2212x), in which x varies from 0 to n\u22122.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-42 of SEQ ID NO. 346, which also corresponds to amino acids 1-42 of SEQ ID NO. 309, a bridging amino acid N corresponding to amino acid 43 of SEQ ID NO. 309, a second amino acid sequence being at least 90% homologous to amino acids 44-657 of SEQ ID NO. 346, which also corresponds to amino acids 44-657 of SEQ ID NO. 309, and a third amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 658-708 of SEQ ID NO. 309, wherein said first amino acid sequence, bridging amino acid, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 309, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VRPHLTLKAPLGLRMHRDPLRTPSPKSWPLTQPLTPDATLTPQAILTPTLT (SEQ ID NO:418) in SEQ ID NO. 309.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-42 of SEQ ID NO. 346, which also corresponds to amino acids 1-42 of SEQ ID NO. 310, a bridging amino acid N corresponding to amino acid 43 of SEQ ID NO. 310, a second amino acid sequence being at least 90% homologous to amino acids 44-676 of SEQ ID NO. 346, which also corresponds to amino acids 44-676 of SEQ ID NO. 310, and a third amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 677-685 of SEQ ID NO. 310, wherein said first amino acid sequence, bridging amino acid, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 310, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence EHGRGPGKT (SEQ ID NO:419) in SEQ ID NO. 310.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-42 of SEQ ID NO. 346, which also corresponds to amino acids 1-42 of SEQ ID NO. 311, a bridging amino acid N corresponding to amino acid 43 of SEQ ID NO. 311, a second amino acid sequence being at least 90% homologous to amino acids 44-657 of SEQ ID NO. 346, which also corresponds to amino acids 44-657 of SEQ ID NO. 311, and a third amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 658-696 of SEQ ID NO. 311, wherein said first amino acid sequence, bridging amino acid, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 311, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence GPGRHAGNAGTLTQSLDCDAGVPPPAFQPLSTSYIYFSE (SEQ ID NO:420) in SEQ ID NO. 311.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-42 of SEQ ID NO. 346, which also corresponds to amino acids 1-42 of SEQ ID NO. 312, a bridging amino acid N corresponding to amino acid 43 of SEQ ID NO. 312, a second amino acid sequence being at least 90% homologous to amino acids 44-610 of SEQ ID NO. 346, which also corresponds to amino acids 44-610 of SEQ ID NO. 312, and a third amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence AMH corresponding to amino acids 611-613 of SEQ ID NO. 312, wherein said first amino acid sequence, bridging amino acid, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-381 of SEQ ID NO. 347, which also corresponds to amino acids 1-381 of SEQ ID NO. 305, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 382-387 of SEQ ID NO. 305, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 305, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence TSLSLS (SEQ ID NO:415) in SEQ ID NO. 305.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-338 of SEQ ID NO. 347, which also corresponds to amino acids 1-338 of SEQ ID NO. 306, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 339-346 of SEQ ID NO. 306, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 306, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VLLCAQWP (SEQ ID NO:416) in SEQ ID NO. 306.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-223 of SEQ ID NO. 347, which also corresponds to amino acids 1-223 of SEQ ID NO. 307, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence A corresponding to amino acids 224-224 of SEQ ID NO. 307, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-294 of SEQ ID NO. 347, which also corresponds to amino acids 1-294 of SEQ ID NO. 308, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 295-304 of SEQ ID NO. 308, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 308, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence RCYLRFLDIY (SEQ ID NO:417) in SEQ ID NO. 308.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a amino acids 1-116 of FABH_HUMAN (SEQ ID NO:348), which also corresponds to amino acids 1-116 of SEQ ID NO. 281, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 117-215 of SEQ ID NO. 281, wherein said firstand second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 281, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VRWATLELYLIGYYYCSFSQACSKKPSPPLRAVEAGTREWLWVRVVSGGNFLCSGFGLTQAGTQILPYRL HDCGQITFSKCNCKTGINNTNLVGLLGSL (SEQ ID NO:396) in SEQ ID NO. 281 .","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-116 of AAP35373 (SEQ ID NO:348), which also corresponds to amino acids 1-116 of SEQ ID NO. 281, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 117-215 of SEQ ID NO. 281, wherein said first and second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 281, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VRWATLELYLIGYYYCSFSQACSKKPSPPLRAVEAGTREWLWVRVVSGGNFLCSGFGLTQAGTQILPYRL HDCGQITFSKCNCKTGINNTNLVGLLGSL (SEQ ID NO:396) in SEQ ID NO. 281 .","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 1-116 of FABH_HUMAN (SEQ ID NO:348), which also corresponds to amino acids 1-116 of SEQ ID NO. 282, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 117-178 of SEQ ID NO. 282, wherein said first and second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 282, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence DVLTAWPSIYRRQVKVLREDEITILPWHLQWSREKATKLLRPTLPSYNNHGWEELRVGKSIV (SEQ ID NO:397) in SEQ ID NO. 282 .","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-116 of AAP35373 (SEQ ID NO:348), which also corresponds to amino acids 1-116 of SEQ ID NO. 282, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 117-178 of SEQ ID NO. 282, wherein said first and second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 282, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence DVLTAWPSIYRRQVKVLREDEITILPWHLQWSREKATKLLRPTLPSYNNHGWEELRVGKSIV (SEQ ID NO:397) in SEQ ID NO. 282 .","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence amino acids 1-116 of FABH_HUMAN (SEQ ID NO:348), which also corresponds to amino acids 1-116 of SEQ ID NO. 283, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 117-126 of SEQ ID NO. 283, wherein said first and second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 283, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence MEKLQLRNVK (SEQ ID NO:398) in SEQ ID NO. 283.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-116 of AAP35373 (SEQ ID NO:348), which also corresponds to amino acids SEQ ID NO. 283, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 117-126 of SEQ ID NO. 283, wherein said first and second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 283, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence MEKLQLRNVK (SEQ ID NO:398) in SEQ ID NO. 283.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-24 of FABH_HUMAN (SEQ ID NO:348), which also corresponds to amino acids 1-24 of SEQ ID NO. 284, second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 25-35 of SEQ ID NO. 284, and a third amino acid sequence being at least 90% homologous to amino acids 25-133 of FABH_HUMAN (SEQ ID NO:348), which also corresponds to amino acids 36-144 of SEQ ID NO. 284, wherein said first, second, third and fourth amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for an edge portion of SEQ ID NO. 284, comprising an amino acid sequence being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence encoding for AHILITFPLPS (SEQ ID NO:399), corresponding to SEQ ID NO. 284.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-24 of AAP35373 (SEQ ID NO:348), which also corresponds to amino acids 1-24 of SEQ ID NO. 284, second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 25-35 of SEQ ID NO. 284, and a third amino acid sequence being at least 90% homologous to amino acids 25-133 of AAP35373 (SEQ ID NO:348), which also corresponds to amino acids 36-144 of SEQ ID NO. 284, wherein said first, second and third amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for an edge portion of SEQ ID NO. 284, comprising an amino acid sequence being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence encoding for AHILITFPLPS (SEQ ID NO:399), corresponding to SEQ ID NO. 284.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-203 of SEQ ID NO. 349, which also corresponds to amino acids 1-203 of SEQ ID NO. 285, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 204-240 of SEQ ID NO. 285, wherein said first and second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 285, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence LWLTPVIPTLWEADGGGLHEPWSWRPAWATWLQRNYL (SEQ ID NO:400) in SEQ ID NO. 285.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-78 of SEQ ID NO. 349, which also corresponds to amino acids 1-78 of SEQ ID NO. 286, second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding,to amino acids 79-125 of SEQ ID NO. 286, and a third amino acid sequence being at least 90% homologous to amino acids 79-399 of SEQ ID NO. 349, which also corresponds to amino acids 126-446 of SEQ ID NO. 286, wherein said first, second and third amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for an edge portion of SEQ ID NO. 286, comprising an amino acid sequence being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence encoding for HWQISQWWLHFQTPREEGKMKLLELSESADGAAWKRWGGNSNTHRIQ (SEQ ID NO:401), corresponding to SEQ ID NO. 286.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-140 of SEQ ID NO. 349, which also corresponds to amino acids 1-140 of SEQ ID NO. 287, and a second amino acid sequence being at least 90% homologous to amino acids 203-399 of SEQ ID NO. 349, which also corresponds to amino acids 141-337 of SEQ ID NO. 287, wherein said first and second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide encoding for an edge portion of SEQ ID NO. 287, comprising a polypeptide having a length \u201cn\u201d, wherein \u201cn\u201d is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least two amino acids comprise DV, having a structure as follows: a sequence starting from any of amino acid numbers 140\u2212x to 140; and ending at any of amino acid numbers 141+((n\u22122)\u2212x), in which x varies from 0 to n\u22122.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a","first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 1-10 of SEQ ID NO. 288, second amino acid sequence being at least 90% homologous to amino acids 18-106 of SEQ ID NO. 349, which also corresponds to amino acids 11-99 of SEQ ID NO. 288, a third (bridging) amino acid sequence comprising D, and a fourth amino acid sequence being at least 90% homologous to amino acids 179-399 of SEQ ID NO. 349, which also corresponds to amino acids 101-321 of SEQ ID NO. 288, wherein said first, second, third and fourth amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a head of SEQ ID NO. 288, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence NETEAEQSYV (SEQ ID NO:402) of SEQ ID NO. 288.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for an edge portion of SEQ ID NO. 288, comprising a polypeptide having a length \u201cn\u201d, wherein \u201cn\u201d is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least two amino acids comprise LDY having a structure as follows (numbering according to SEQ ID NO. 288): a sequence starting from any of amino acid numbers 99\u2212x to 99; and ending at any of amino acid numbers 101+((n\u22122)\u2212x), in which x varies from 0 to n\u22122.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 1-15 of SEQ ID NO. 289, and a second amino acid sequence being at least 90% homologous to corresponding to amino acids 203-399 of SEQ ID NO. 349, which also corresponds to amino acids 16-212 of SEQ ID NO. 289, wherein said first and second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a head of SEQ ID NO. 289, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence MSSWLSAGSPSSLSV (SEQ ID NO:403) of SEQ ID NO. 289.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 1-13 of SEQ ID NO. 290, and a second amino acid sequence being at least 90% homologous to amino acids 280-399 of SEQ ID NO. 349, which also corresponds to amino acids 14-133 of SEQ ID NO. 290, wherein said first and second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a head of SEQ ID NO. 290, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence MCRGYSTLLNPVS (SEQ ID NO:404) of SEQ ID NO. 290.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-246 of SEQ ID NO. 349, which also corresponds to amino acids 1-246 of SEQ ID NO. 291, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 247-252 of SEQ ID NO. 291, wherein said first and second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 291, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence SRNWTQ (SEQ ID NO:405) in SEQ ID NO. 291.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 1-10 of SEQ ID NO. 292, second amino acid sequence being at least 90% homologous to amino acids 26-276 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 11-261 of SEQ ID NO. 292, followed by A, and a third amino acid sequence being at least 90% homologous to amino acids 278-466 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 263-451 of SEQ ID NO. 292, wherein said first, second, A, and third amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a head of SEQ ID NO. 292, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence MEISLVKCSE (SEQ ID NO:406) of SEQ ID NO. 292","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-276 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 1-276 of SEQ ID NO. 293, followed by A, a second amino acid sequence being at least 90% homologous to amino acids 278-372 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 278-372 of SEQ ID NO. 293, and a third amino acid sequence being at least 90% homologous to amino acids 401-466 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 373-438 of SEQ ID NO. 293, wherein said first, A, second, and third amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide encoding for an edge portion of SEQ ID NO. 293, comprising a polypeptide having a length \u201cn\u201d, wherein n is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least two amino acids comprise EE, having a structure as follows: a sequence starting from any of amino acid numbers 372\u2212x to 372; and ending at any of amino acid numbers 373+((n\u22122)\u2212x), in which x varies from 0 to n\u22122.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-276 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 1-276 of SEQ ID NO. 294, followed by A, a second amino acid sequence being at least 90% homologous to amino acids 278-401 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 278-401 of SEQ ID NO. 294, and a third amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 402-407 of SEQ ID NO. 294, wherein said first, A, second and third amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 294, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence PNRQDS (SEQ ID NO:407) in SEQ ID NO. 294.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-276 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 1-276 of SEQ ID NO. 295, followed by A, a second amino acid sequence being at least 90% homologous to amino acids 278-374 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 278-374 of SEQ ID NO. 295, and a third amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 375-390 of SEQ ID NO. 295, wherein said first, A, second and third amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 295, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence MSHELFSRFSLRLFGR (SEQ ID NO:408) in SEQ ID NO. 295.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-261 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 1-261 of SEQ ID NO. 296, a second amino acid sequence comprising A, and a third amino acid sequence being at least 90% homologous to amino acids 263-451 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 263-451 of SEQ ID NO. 296, wherein said first, second and third amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-132 of Q9NPI5 (SEQ ID NO:372), which also corresponds to amino acids 1-132 of SEQ ID NO. 297, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 133-145 of SEQ ID NO. 297, wherein said first and second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 297, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence LPGRHEVPRGALP (SEQ ID NO:409) in SEQ ID NO. 297.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-109 of Q9NZK3 (SEQ ID NO:373), which also corresponds to amino acids 1-109 of SEQ ID NO. 297, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 110-145 of SEQ ID NO. 297, wherein said first and second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 297, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence LVDLYSRRYFLTVPYEECKWRRSLPGRHEVPRGALP in SEQ ID NO. 297.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-107 of Q9NPI5 (SEQ ID NO:372), which also corresponds to amino acids 1-107 of SEQ ID NO. 298, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 108-121 of SEQ ID NO. 298, wherein said first and second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 298, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence NLPGRHEVPRGALP (SEQ ID NO:410) in SEQ ID NO. 298.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-107 of Q9NZK3 (SEQ ID NO:373), which also corresponds to amino acids 1-107 of SEQ ID NO. 298, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 108-121 of SEQ ID NO. 298, wherein said first and second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 298, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence NLPGRHEVPRGALP (SEQ ID NO:410) in SEQ ID NO. 298.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 51-151 of SEQ ID NO. 350, which also corresponds to amino acids 1-101 of SEQ ID NO. 299.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence MSSFSTTT (SEQ ID NO:411) corresponding to amino acids 1-8 of SEQ ID NO. 300, and a second amino acid sequence being at least 90% homologous to amino acids 42-151 of SEQ ID NO. 350, which also corresponds to amino acids 9-118 of SEQ ID NO. 300, wherein said first and second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a head of SEQ ID NO. 300, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence MSSFSTTT (SEQ ID NO:411) of SEQ ID NO. 300.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-124 of TRIC_HUMAN, which also corresponds to amino acids 1-124 of SEQ ID NO. 301, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 125-137 of SEQ ID NO. 301, wherein said first and second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 301, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VGRMGSSGTFGVG (SEQ ID NO:412) in SEQ ID NO. 301.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-8 of TRIC_HUMAN, which also corresponds to amino acids 1-8 of SEQ ID NO. 302, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 36-209 of TRIC_HUMAN, which also corresponding to amino acids 9-182 of SEQ ID NO. 302, wherein said first and second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide encoding for an edge portion of SEQ ID NO. 302, comprising a polypeptide having a length \u201cn\u201d, wherein \u201cn\u201d is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least two amino acids comprise AK, having a structure as follows: a sequence starting from any of amino acid numbers 8\u2212x to 8; and ending at any of amino acid numbers 9+((n\u22122)\u2212x), in which x varies from 0 to n\u22122.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-36 of TRIC_HUMAN, which also corresponds to amino acids 1-36 of SEQ ID NO. 303, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 37-86 of SEQ ID NO. 303, wherein said first and second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 303, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VGRGFLGAEYRRRRDPRPWEWGEEPGLRRGRGLRGGASGAEFCRGSCSDW (SEQ ID NO:413) in SEQ ID NO. 303.","According to preferred embodiments of the present invention, there is provided an isolated chimeric polypeptide, comprising a first amino acid sequence being at least 90% homologous to amino acids 1-8 of TRIC_HUMAN, which also corresponds to amino acids 1-8 of SEQ ID NO. 304, and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide sequence corresponding to amino acids 9-13 of SEQ ID NO. 304, wherein said first and second amino acid sequences are contiguous and in a sequential order.","According to preferred embodiments of the present invention, there is provided an isolated polypeptide encoding for a tail of SEQ ID NO. 304, comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VRAAG (SEQ ID NO:414) in SEQ ID NO. 304.","According to preferred embodiments of the present invention, there is provided an antibody capable of specifically binding to an epitope of an amino acid sequence in any one of cluster S67314, N56180, T10377, Z24874, HUMCDDANF, HUMTROPIA, HUMSMCK, H88495, Z36249, FLJ26352, HSACMHCP. Preferably, the amino acid sequence corresponds to any insertion, including a bridge, edge portion, tail, or head as described herein.","Preferably, the antibody is capable of differentiating between a splice variant having the epitope and a corresponding known protein.","According to preferred embodiments of the present invention, there is provided a kit for detecting heart disorders, comprising a kit detecting overexpression of a splice variant. Optionally, the kit comprises a NAT-based technology. Preferably, the kit further comprises at least one primer pair capable of selectively hybridizing to a nucleic acid sequence in any one of cluster S67314, N56180, T10377, Z24874, HUMCDDANF, HUMTROPIA, HUMSMCK, H88495, Z36249, FLJ26352, HSACMHCP.","Optionally, the kit further comprises at least one oligonucleotide capable of selectively hybridizing to a nucleic acid sequence in any one of cluster S67314, N56180, T10377, Z24874, HUMCDDANF, HUMTROPIA, HUMSMCK, H88495, Z36249, FLJ26352, HSACMHCP.","Optionally, kit comprises an antibody as described herein. Preferably, the kit further comprises at least one reagent for performing an ELISA or a Western blot.","According to preferred embodiments of the present invention, there is provided a method for detecting heart disorders, comprising detecting overexpression of a splice variant of any of cluster S67314, N56180, T10377, Z24874, HUMCDDANF, HUMTROPIA, HUMSMCK, H88495, Z36249, FLJ26352, HSACMHCP. Optionally, detecting overexpression is performed with a NAT-based technology.","Also optionally, detecting overexpression is performed with an immunoassay. Preferably, the immunoassay comprises an antibody as described herein.","According to preferred embodiments of the present invention, there is provided a biomarker capable of detecting heart disorders, comprising any of the above nucleic acid sequences or a fragment thereof, or amino acid sequences or a fragment thereof.","According to preferred embodiments of the present invention, there is provided a method for screening for heart disorders, comprising detecting cardiac disease cells or tissue with a biomarker or an antibody.","According to preferred embodiments of the present invention, there is provided a method for diagnosing heart disorders, comprising detecting heart cells or tissue with a biomarker or an antibody.","According to preferred embodiments of the present invention, there is provided a method for monitoring disease progression, or treatment efficacy, or relapse of heart disorders, or any combination thereof, comprising detecting heart cells or tissue with a biomarker or an antibody or a method or assay as described herein.","According to preferred embodiments of the present invention, there is provided a method of selecting a therapy for heart disorders, comprising detecting heart disorder cells with a biomarker or an antibody or a method or assay as described herein and selecting a therapy according to the detection.","A heart disorder and\/or cardiac disease and\/or cardiac pathology optionally comprises at least one of: Myocardial infarct, ungina pectoris (stable and unstable), cardiomyopathy, myocarditis, congestive heart failure, the detection of reinfarction, the detection of success of thrombolytic therapy after Myocardial infarct, Myocardial infarct after surgery, assessing the size of infarct in Myocardial infarct.","According to preferred embodiments of the present invention, preferably any of the above nucleic acid and\/or amino acid sequences further comprises any sequence having at least about 70%, preferably at least about 80%, more preferably at least about 90%, most preferably at least about 95% homology thereto.","All nucleic acid sequences and\/or amino acid sequences shown herein as embodiments of the present invention relate to their isolated form, as isolated polynucleotides (including for all transcripts), oligonucleotides (including for all segments, amplicons and primers), peptides (including for all tails, bridges, insertions or heads, optionally including other antibody epitopes as described herein) and\/or polypeptides (including for all proteins). It should be noted that oligonucleotide and polynucleotide, or peptide and polypeptide, may optionally be used interchangeably.","According to preferred embodiments of the present invention, there is provided:","An isolated polynucleotide comprising the polynucleotide sequence set forth in a member selected from the group consisting of SEQ ID NOs:1-60, 63, 65-273, 275, 278, 335, 361, 365, 368, 371, 376, 379, 382 and 385, 392-393 or the polynucleotide sequence at least about 95% identical thereto.","An isolated primer pair, comprising the pair of nucleic acid sequences selected from the group consisting of: SEQ NOs 61 and 62; 64 and 274; 276 and 277; 279 and 280; 336 and 337; 363 and 364; 366 and 367; 369 and 370; 374 and 375; 377 and 378; 380 and 381; and 383 and 384.","A kit for detecting a heart disorder and\/or cardiac disease and\/or cardiac pathology, comprising at least one of the foregoing primer pairs.","A method for detecting a heart disorder and\/or cardiac disease and\/or cardiac pathology, comprising detecting overexpression of the polynucleotide of sequence set forth in a member selected from the group consisting of SEQ ID NOs:1-60, 63, 65-273, 275, 278, 335, 361, 365, 368, 371, 376, 379, 382 and 385, 392-393 or the polynucleotide sequence at least about 95% identical thereto, in a sample from a patient.","The foregoing method, wherein said detecting overexpression comprises performing nucleic acid amplification.","A method for monitoring disease progression, treatment efficacy or relapse of a heart disorder and\/or cardiac disease and\/or cardiac pathology, comprising detecting the polynucleotide of sequence set forth in a member selected from the group consisting of SEQ ID NOs:1-60, 63, 65-273, 275, 278, 335, 361, 365, 368, 371, 376, 379, 382 and 385, 392-393 or the polynucleotide sequence at least about 95% identical thereto.","A method of selecting a therapy for a heart disorder and\/or cardiac disease and\/or cardiac pathology, comprising detecting the polynucleotide of sequence set forth in a member selected from the group consisting of SEQ ID NOs:1-60, 63, 65-273, 275, 278, 335, 361, 365, 368, 371, 376, 379, 382 and 385, 392-393 or the polynucleotide sequence at least about 95% identical thereto, and selecting a therapy according to said detection.","Any of the foregoing methods, wherein a heart disorder and\/or cardiac disease and\/or cardiac pathology comprises at least one of: Myocardial infarct, ungina pectoris (stable and unstable), cardiomyopathy, myocarditis, congestive heart failure, the detection of reinfarction, the detection of success of thrombolytic therapy after Myocardial infarct, Myocardial infarct after surgery, or assessing the size of infarct in Myocardial infarct.","An isolated polypeptide comprising the polypeptide sequence set forth in a member selected from the group consisting of 281-334, 394-436 and 454-458, or the polypeptide sequence at least about 95% homologous thereto, or from the group consisting of:","the polypeptide comprising a first amino acid sequence of amino acids 1-124 of SEQ ID NO. 301, and a second amino acid sequence being at least 95% about homologous to amino acids 125-137 of SEQ ID NO. 301, wherein said first and second amino acid sequences are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-8 of SEQ ID NO. 302, and a second amino acid sequence being at least 95% about homologous to amino acids 9-182 of SEQ ID NO. 302, wherein said first and second amino acid sequences are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-36 of SEQ ID NO. 303, and a second amino acid sequence being at least 95% about homologous to amino acids 37-86 of SEQ ID NO. 303, wherein said first and second amino acid sequences are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-8 of SEQ ID NO. 304, and a second amino acid sequence being at least about 95% homologous to amino acids 9-13 of SEQ ID NO. 304, wherein said first and second amino acid sequences are contiguous and in a sequential order;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise AK, having a structure as follows: starting from any of amino acid numbers 8\u2212x to 8 of SEQ ID NO: 302; and ending at any of amino acid numbers 9+((n\u22122)\u2212x) of SEQ ID NO: 302, in which x varies from 0 to n\u22122;","the isolated peptide comprising the amino acid sequence set forth in a 50 amino acid long segment of SEQ ID NO: 302 anywhere from amino acid position 1 to amino acid position 58, and including amino acids at positions 8 and 9;","the isolated peptide comprising the amino acid sequence set forth in a 40 amino acid long segment of SEQ ID NO: 302 anywhere from amino acid position 1 to amino acid position 48, and including amino acids at positions 8 and 9;","the isolated peptide comprising the amino acid sequence set forth in a 30 amino acid long segment of SEQ ID NO: 302 anywhere from amino acid position 1 to amino acid position 38, and including amino acids at positions 8 and 9;","the isolated peptide comprising the amino acid sequence set forth in a 20 amino acid long segment of SEQ ID NO: 302 anywhere from amino acid position 1 to amino acid position 28, and including amino acids at positions 8 and 9;","the polypeptide comprising a first amino acid sequence of amino acids 1-1855 of SEQ ID NO. 326, and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 1856-1904 of SEQ ID NO. 326, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-1326 of SEQ ID NO. 327 and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 1327-1336 of SEQ ID NO. 327, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-1508 of SEQ ID NO. 328 and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 1509-1534 of SEQ ID NO. 328, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-1763 of SEQ ID NO. 329 and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 1764-1788 of SEQ ID NO. 329, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 1-22 of SEQ ID NO. 330 and a second amino acid sequence of amino acids 23-1434 of SEQ ID NO. 330, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-527 of SEQ ID NO. 339 and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 528-555 of SEQ ID NO. 331, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-1798 of SEQ ID NO. 332, and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 1799-1847 of SEQ ID NO. 332, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-156 of SEQ ID NO. 318, and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 157-166 of SEQ ID NO. 318, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-156 of SEQ ID NO. 319, and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 157-210 of SEQ ID NO. 319, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-60 of SEQ ID NO. 320, and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 61-114 of SEQ ID NO. 320, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-14 of SEQ ID NO. 321, and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 15-87 of SEQ ID NO. 321, wherein said first amino acid sequence and second amino acid sequence are contiguous and in sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-60 of SEQ ID NO. 324, and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 61-70 of SEQ ID NO. 324, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-184 of SEQ ID NO. 314, and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 185-197 of SEQ ID NO. 314, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-151 of SEQ ID NO. 315, and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 152-177 of SEQ ID NO. 315, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-657 of SEQ ID NO. 309, and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 658-708 of SEQ ID NO. 309, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-676 of SEQ ID NO. 310, and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 677-685 of SEQ ID NO. 310, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-657 of SEQ ID NO. 311, and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 658-696 of SEQ ID NO. 311, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-294 of SEQ ID NO. 308, and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 295-304 of SEQ ID NO. 308, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-116 of SEQ ID NO. 281, and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 117-215 of SEQ ID NO. 281, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-116 of SEQ ID NO. 282, and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 117-178 of SEQ ID NO. 282, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-203 of SEQ ID NO. 285, and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 204-240 of SEQ ID NO. 285, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-78 of SEQ ID NO. 286, a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 79-125 of SEQ ID NO. 286, and a third amino acid sequence of amino acids 126-446 of SEQ ID NO. 286, wherein said first, second and third amino acid sequences are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 1-15 of SEQ ID NO. 289, and a second amino acid sequence of amino acids 16-212 of SEQ ID NO. 289, wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-374 of SEQ ID NO. 295, and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 375-390 of SEQ ID NO. 295, wherein said first and second amino acid sequences are contiguous and in a sequential order;","the polypeptide comprising a first amino acid sequence of amino acids 1-36 of SEQ ID NO. 303, and a second amino acid sequence being at least about 95% homologous to a polypeptide sequence corresponding to amino acids 37-86 of SEQ ID NO. 303, wherein said first and second amino acid sequences are contiguous and in a sequential order;","the polypeptide of SEQ ID NO: 414;","the polypeptide of SEQ ID NO: 413;","the polypeptide of SEQ ID NO: 412;","the polypeptide of SEQ ID NO: 411;","the polypeptide of SEQ ID NO: 410;","the polypeptide of SEQ ID NO: 409;","the polypeptide of SEQ ID NO: 408;","the polypeptide of SEQ ID NO: 407;","the polypeptide of SEQ ID NO: 406;","the polypeptide of SEQ ID NO: 405;","the polypeptide of SEQ ID NO: 404;","the polypeptide of SEQ ID NO: 403;","the polypeptide of SEQ ID NO: 402;","the polypeptide of SEQ ID NO: 401;","the polypeptide of SEQ ID NO: 400;","the polypeptide of SEQ ID NO: 399;","the polypeptide of SEQ ID NO: 398;","the polypeptide of SEQ ID NO: 397;","the polypeptide of SEQ ID NO: 396;","the polypeptide of SEQ ID NO: 417;","the polypeptide of SEQ ID NO: 416;","the polypeptide of SEQ ID NO: 415;","the polypeptide of SEQ ID NO: 420;","the polypeptide of SEQ ID NO: 419;","the polypeptide of SEQ ID NO: 418;","the polypeptide of SEQ ID NO: 422;","the polypeptide of SEQ ID NO: 421;","the polypeptide of SEQ ID NO: 427;","the polypeptide of SEQ ID NO: 426;","the polypeptide of SEQ ID NO: 424;","the polypeptide of SEQ ID NO: 423;","the polypeptide of SEQ ID NO: 435;","the polypeptide of SEQ ID NO: 395;","the polypeptide of SEQ ID NO: 432;","the polypeptide of SEQ ID NO: 394;","the polypeptide of SEQ ID NO: 431;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise DP, having a structure as follows: a sequence starting from any of amino acid numbers 470\u2212x to 470 of SEQ ID NO:332; and ending at any of amino acid numbers 471+((n\u22122)\u2212x) of SEQ ID NO:332, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least three amino acids comprise VSI, having a structure as follows: a sequence starting from any of amino acid numbers 14\u2212x to 14 of SEQ ID NO. 321; and ending at any of amino acid numbers 16+((n\u22122)\u2212x) of SEQ ID NO. 321, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise IY, having a structure as follows: a sequence starting from any of amino acid numbers 115\u2212x to 115 of SEQ ID NO:313; and ending at any of amino acid numbers 116+((n\u22122)\u2212x) of SEQ ID NO:313, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise EL, having a structure as follows: a sequence starting from any of amino acid numbers 151\u2212x to 151 of SEQ ID NO: 316; and ending at any of amino acid numbers 152+((n\u22122)\u2212x) of SEQ ID NO: 316, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise DV, having a structure as follows: a sequence starting from any of amino acid numbers 140\u2212x to 140 of SEQ ID NO: 287; and ending at any of amino acid numbers 141+((n\u22122)\u2212x) of SEQ ID NO: 287, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least three amino acids comprise LDY having a structure as follows: a sequence starting from any of amino acid numbers 99\u2212x to 99 of SEQ ID NO. 288; and ending at any of amino acid numbers 101+((n\u22122)\u2212x) of SEQ ID NO. 288, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise EE, having a structure as follows: a sequence starting from any of amino acid numbers 372\u2212x to 372 of SEQ ID NO 293; and ending at any of amino acid numbers 373+((n\u22122)\u2212x) of SEQ ID NO 293, in which x varies from 0 to n\u22122; and","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least at least two amino acids comprise AK, having a structure as follows: a sequence starting from any of amino acid numbers 8\u2212x to 8 of SEQ ID NO: 302; and ending at any of amino acid numbers 9+((n\u22122)\u2212x) of SEQ ID NO: 302, in which x varies from 0 to n\u22122.","An antibody to specifically bind to any of the foregoing amino acid sequences.","The foregoing antibody, to specifically bind to the amino acid sequence selected from the group consisting of SEQ ID NOs: 394-436, 454-458;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise AK, having a structure as follows: starting from any of amino acid numbers 8\u2212x to 8 of SEQ ID NO: 302; and ending at any of amino acid numbers 9+((n\u22122)\u2212x) of SEQ ID NO: 302, in which x varies from 0 to n\u22122;","the isolated peptide comprising the amino acid sequence set forth in a 50 amino acid long segment of SEQ ID NO: 302 anywhere from amino acid position 1 to amino acid position 58, and including amino acids at positions 8 and 9;","the isolated peptide comprising the amino acid sequence set forth in a 40 amino acid long segment of SEQ ID NO: 302 anywhere from amino acid position 1 to amino acid position 48, and including amino acids at positions 8 and 9;","the isolated peptide comprising the amino acid sequence set forth in a 30 amino acid long segment of SEQ ID NO: 302 anywhere from amino acid position 1 to amino acid position 38, and including amino acids at positions 8 and 9;","the isolated peptide comprising the amino acid sequence set forth in a 20 amino acid long segment of SEQ ID NO: 302 anywhere from amino acid position 1 to amino acid position 28, and including amino acids at positions 8 and 9;","the polypeptide of SEQ ID NO: 414;","the polypeptide of SEQ ID NO: 413;","the polypeptide of SEQ ID NO: 412;","the polypeptide of SEQ ID NO: 411;","the polypeptide of SEQ ID NO: 410;","the polypeptide of SEQ ID NO: 409;","the polypeptide of SEQ ID NO: 408;","the polypeptide of SEQ ID NO: 407;","the polypeptide of SEQ ID NO: 406;","the polypeptide of SEQ ID NO: 405;","the polypeptide of SEQ ID NO: 404;","the polypeptide of SEQ ID NO: 403;","the polypeptide of SEQ ID NO: 402;","the polypeptide of SEQ ID NO: 401;","the polypeptide of SEQ ID NO: 400;","the polypeptide of SEQ ID NO: 399;","the polypeptide of SEQ ID NO: 398;","the polypeptide of SEQ ID NO: 397;","the polypeptide of SEQ ID NO: 396;","the polypeptide of SEQ ID NO: 417;","the polypeptide of SEQ ID NO: 416;","the polypeptide of SEQ ID NO: 415;","the polypeptide of SEQ ID NO: 420;","the polypeptide of SEQ ID NO: 419;","the polypeptide of SEQ ID NO: 418;","the polypeptide of SEQ ID NO: 422;","the polypeptide of SEQ ID NO: 421;","the polypeptide of SEQ ID NO: 427;","the polypeptide of SEQ ID NO: 426;","the polypeptide of SEQ ID NO: 424;","the polypeptide of SEQ ID NO: 423;","the polypeptide of SEQ ID NO: 435;","the polypeptide of SEQ ID NO: 395;","the polypeptide of SEQ ID NO: 432;","the polypeptide of SEQ ID NO: 394;","the polypeptide of SEQ ID NO: 431;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise DP, having a structure as follows: a sequence starting from any of amino acid numbers 470\u2212x to 470 of SEQ ID NO:332; and ending at any of amino acid numbers 471+((n\u22122)\u2212x) of SEQ ID NO:332, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least three amino acids comprise VSI, having a structure as follows: a sequence starting from any of amino acid numbers 14\u2212x to 14 of SEQ ID NO. 321; and ending at any of amino acid numbers 16+((n\u22122)\u2212x) of SEQ ID NO. 321, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise IY, having a structure as follows: a sequence starting from any of amino acid numbers 115\u2212x to 115 of SEQ ID NO:313; and ending at any of amino acid numbers 116+((n\u22122)\u2212x) of SEQ ID NO:313, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise EL, having a structure as follows: a sequence starting from any of amino acid numbers 151\u2212x to 151 of SEQ ID NO: 316; and ending at any of amino acid numbers 152+((n\u22122)\u2212x) of SEQ ID NO: 316, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise DV, having a structure as follows: a sequence starting from any of amino acid numbers 140\u2212x to 140 of SEQ ID NO: 287; and ending at any of amino acid numbers 141+((n\u22122)\u2212x) of SEQ ID NO: 287, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least three amino acids comprise LDY having a structure as follows: a sequence starting from any of amino acid numbers 99\u2212x to 99 of SEQ ID NO. 288; and ending at any of amino acid numbers 101+((n\u22122)\u2212x) of SEQ ID NO. 288, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise EE, having a structure as follows: a sequence starting from any of amino acid numbers 372\u2212x to 372 of SEQ ID NO 293; and ending at any of amino acid numbers 373+((n\u22122)\u2212x) of SEQ ID NO 293, in which x varies from 0 to n\u22122; and","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least at least two amino acids comprise AK, having a structure as follows: a sequence starting from any of amino acid numbers 8\u2212x to 8 of SEQ ID NO: 302; and ending at any of amino acid numbers 9+((n\u22122)\u2212x) of SEQ ID NO: 302, in which x varies from 0 to n\u22122.","The foregoing antibody, for specifically binding only to an epitope comprising the amino acid sequence selected from the group consisting of SEQ ID NOs: 394-436, 454-458;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise AK, having a structure as follows: starting from any of amino acid numbers 8\u2212x to 8 of SEQ ID NO: 302; and ending at any of amino acid numbers 9+((n\u22122)\u2212x) of SEQ ID NO: 302, in which x varies from 0 to n\u22122;","the isolated peptide comprising the amino acid sequence set forth in a 50 amino acid long segment of SEQ ID NO: 302 anywhere from amino acid position 1 to amino acid position 58, and including amino acids at positions 8 and 9;","the isolated peptide comprising the amino acid sequence set forth in a 40 amino acid long segment of SEQ ID NO: 302 anywhere from amino acid position 1 to amino acid position 48, and including amino acids at positions 8 and 9;","the isolated peptide comprising the amino acid sequence set forth in a 30 amino acid long segment of SEQ ID NO: 302 anywhere from amino acid position 1 to amino acid position 38, and including amino acids at positions 8 and 9;","the isolated peptide comprising the amino acid sequence set forth in a 20 amino acid long segment of SEQ ID NO: 302 anywhere from amino acid position 1 to amino acid position 28, and including amino acids at positions 8 and 9;","the polypeptide of SEQ ID NO: 414;","the polypeptide of SEQ ID NO: 413;","the polypeptide of SEQ ID NO: 412;","the polypeptide of SEQ ID NO: 411;","the polypeptide of SEQ ID NO: 410;","the polypeptide of SEQ ID NO: 409;","the polypeptide of SEQ ID NO: 408;","the polypeptide of SEQ ID NO: 407;","the polypeptide of SEQ ID NO: 406;","the polypeptide of SEQ ID NO: 405;","the polypeptide of SEQ ID NO: 404;","the polypeptide of SEQ ID NO: 403;","the polypeptide of SEQ ID NO: 402;","the polypeptide of SEQ ID NO: 401;","the polypeptide of SEQ ID NO: 400;","the polypeptide of SEQ ID NO: 399;","the polypeptide of SEQ ID NO: 398;","the polypeptide of SEQ ID NO: 397;","the polypeptide of SEQ ID NO: 396;","the polypeptide of SEQ ID NO: 417;","the polypeptide of SEQ ID NO: 416;","the polypeptide of SEQ ID NO: 415;","the polypeptide of SEQ ID NO: 420;","the polypeptide of SEQ ID NO: 419;","the polypeptide of SEQ ID NO: 418;","the polypeptide of SEQ ID NO: 422;","the polypeptide of SEQ ID NO: 421;","the polypeptide of SEQ ID NO: 427;","the polypeptide of SEQ ID NO: 426;","the polypeptide of SEQ ID NO: 424;","the polypeptide of SEQ ID NO: 423;","the polypeptide of SEQ ID NO: 435;","the polypeptide of SEQ ID NO: 395;","the polypeptide of SEQ ID NO: 432;","the polypeptide of SEQ ID NO: 394;","the polypeptide of SEQ ID NO: 431;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise DP, having a structure as follows: a sequence starting from any of amino acid numbers 470\u2212x to 470 of SEQ ID NO:332; and ending at any of amino acid numbers 471+((n\u22122)\u2212x) of SEQ ID NO:332, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least three amino acids comprise VSI, having a structure as follows: a sequence starting from any of amino acid numbers 14\u2212x to 14 of SEQ ID NO. 321; and ending at any of amino acid numbers 16+((n\u22122)\u2212x) of SEQ ID NO. 321, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise IY, having a structure as follows: a sequence starting from any of amino acid numbers 115\u2212x to 115 of SEQ ID NO:313; and ending at any of amino acid numbers 116+((n\u22122)\u2212x) of SEQ ID NO:313, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise EL, having a structure as follows: a sequence starting from any of amino acid numbers 151\u2212x to 151 of SEQ ID NO: 316; and ending at any of amino acid numbers 152+((n\u22122)\u2212x) of SEQ ID NO: 316, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise DV, having a structure as follows: a sequence starting from any of amino acid numbers 140\u2212x to 140 of SEQ ID NO: 287; and ending at any of amino acid numbers 141+((n\u22122)\u2212x) of SEQ ID NO: 287, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least three amino acids comprise LDY having a structure as follows: a sequence starting from any of amino acid numbers 99\u2212x to 99 of SEQ ID NO. 288; and ending at any of amino acid numbers 101+((n\u22122)\u2212x) of SEQ ID NO. 288, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise EE, having a structure as follows: a sequence starting from any of amino acid numbers 372\u2212x to 372 of SEQ ID NO 293; and ending at any of amino acid numbers 373+((n\u22122)\u2212x) of SEQ ID NO 293, in which x varies from 0 to n\u22122; and","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least at least two amino acids comprise AK, having a structure as follows: a sequence starting from any of amino acid numbers 8\u2212x to 8 of SEQ ID NO: 302; and ending at any of amino acid numbers 9+((n\u22122)\u2212x) of SEQ ID NO: 302, in which x varies from 0 to n\u22122.","16. An antibody for specifically binding only an epitope comprising the amino acid sequence selected from the group consisting of SEQ ID NOs:394-436, 454-458;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise AK, having a structure as follows: starting from any of amino acid numbers 8\u2212x to 8 of SEQ ID NO: 302; and ending at any of amino acid numbers 9+((n\u22122)\u2212x) of SEQ ID NO: 302, in which x varies from 0 to n\u22122;","the isolated peptide comprising the amino acid sequence set forth in a 50 amino acid long segment of SEQ ID NO: 302 anywhere from amino acid position 1 to amino acid position 58, and including amino acids at positions 8 and 9;","the isolated peptide comprising the amino acid sequence set forth in a 40 amino acid long segment of SEQ ID NO: 302 anywhere from amino acid position 1 to amino acid position 48, and including amino acids at positions 8 and 9;","the isolated peptide comprising the amino acid sequence set forth in a 30 amino acid long segment of SEQ ID NO: 302 anywhere from amino acid position 1 to amino acid position 38, and including amino acids at positions 8 and 9;","the isolated peptide comprising the amino acid sequence set forth in a 20 amino acid long segment of SEQ ID NO: 302 anywhere from amino acid position 1 to amino acid position 28, and including amino acids at positions 8 and 9;","the polypeptide of SEQ ID NO: 414;","the polypeptide of SEQ ID NO: 413;","the polypeptide of SEQ ID NO: 412;","the polypeptide of SEQ ID NO: 411;","the polypeptide of SEQ ID NO: 410;","the polypeptide of SEQ ID NO: 409;","the polypeptide of SEQ ID NO: 408;","the polypeptide of SEQ ID NO: 407;","the polypeptide of SEQ ID NO: 406;","the polypeptide of SEQ ID NO: 405;","the polypeptide of SEQ ID NO: 404;","the polypeptide of SEQ ID NO: 403;","the polypeptide of SEQ ID NO: 402;","the polypeptide of SEQ ID NO: 401;","the polypeptide of SEQ ID NO: 400;","the polypeptide of SEQ ID NO: 399;","the polypeptide of SEQ ID NO: 398;","the polypeptide of SEQ ID NO: 397;","the polypeptide of SEQ ID NO: 396;","the polypeptide of SEQ ID NO: 417;","the polypeptide of SEQ ID NO: 416;","the polypeptide of SEQ ID NO: 415;","the polypeptide of SEQ ID NO: 420;","the polypeptide of SEQ ID NO: 419;","the polypeptide of SEQ ID NO: 418;","the polypeptide of SEQ ID NO: 422;","the polypeptide of SEQ ID NO: 421;","the polypeptide of SEQ ID NO: 427;","the polypeptide of SEQ ID NO: 426;","the polypeptide of SEQ ID NO: 424;","the polypeptide of SEQ ID NO: 423;","the polypeptide of SEQ ID NO: 435;","the polypeptide of SEQ ID NO: 395;","the polypeptide of SEQ ID NO: 432;","the polypeptide of SEQ ID NO: 394;","the polypeptide of SEQ ID NO: 431;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise DP, having a structure as follows: a sequence starting from any of amino acid numbers 470\u2212x to 470 of SEQ ID NO:332; and ending at any of amino acid numbers 471+((n\u22122)\u2212x) of SEQ ID NO:332, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least three amino acids comprise VSI, having a structure as follows: a sequence starting from any of amino acid numbers 14\u2212x to 14 of SEQ ID NO. 321; and ending at any of amino acid numbers 16+((n\u22122)\u2212x) of SEQ ID NO. 321, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise IY, having a structure as follows: a sequence starting from any of amino acid numbers 115\u2212x to 115 of SEQ ID NO:313; and ending at any of amino acid numbers 116+((n\u22122)\u2212x) of SEQ ID NO:313, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise EL, having a structure as follows: a sequence starting from any of amino acid numbers 151\u2212x to 151 of SEQ ID NO: 316; and ending at any of amino acid numbers 152+((n\u22122)\u2212x) of SEQ ID NO: 316, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise DV, having a structure as follows: a sequence starting from any of amino acid numbers 140\u2212x to 140 of SEQ ID NO: 287; and ending at any of amino acid numbers 141+((n\u22122)\u2212x) of SEQ ID NO: 287, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least three amino acids comprise LDY having a structure as follows: a sequence starting from any of amino acid numbers 99\u2212x to 99 of SEQ ID NO. 288; and ending at any of amino acid numbers 101+((n\u22122)\u2212x) of SEQ ID NO. 288, in which x varies from 0 to n\u22122;","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least two amino acids comprise EE, having a structure as follows: a sequence starting from any of amino acid numbers 372\u2212x to 372 of SEQ ID NO 293; and ending at any of amino acid numbers 373+((n\u22122)\u2212x) of SEQ ID NO 293, in which x varies from 0 to n\u22122; and","the isolated peptide having a length \u201cn\u201d, wherein n is selected from the group consisting of 10 amino acids in length, 20 amino acids in length, 30 amino acids in length, 40 amino acids in length and 50 amino acids in length, wherein at least at least two amino acids comprise AK, having a structure as follows: a sequence starting from any of amino acid numbers 8\u2212x to 8 of SEQ ID NO: 302; and ending at any of amino acid numbers 9+((n\u22122)\u2212x) of SEQ ID NO: 302, in which x varies from 0 to n\u22122.","An antibody to specifically bind to any of the foregoing amino acid sequences but not to specifically bind to the amino acid sequence selected from the group consisting of 338-351, 362, 372-373, 388-391, 437, 441, 445 and 449.","A kit, comprising the foregoing antibody marked with a label.","The foregoing kit, wherein said kit further comprises at least one ELISA reagent or at least one Western blot reagent.","A method for detecting a heart disorder and\/or cardiac disease and\/or cardiac pathology, comprising detecting any one of the foregoing polypeptides.","The foregoing method, wherein said detecting comprises detecting specific binding of the foregoing antibody in a sample from a patient.","A biomarker to detect a heart disorder and\/or cardiac disease and\/or cardiac pathology, comprising anyone of the foregoing polypeptides, marked with a label.","A method to screen for a heart disorder and\/or cardiac disease and\/or cardiac pathology, comprising detecting any one of the foregoing polypeptide.","A method for monitoring disease progression, treatment efficacy or relapse of a heart disorder and\/or cardiac disease and\/or cardiac pathology, comprising detecting any one of the foregoing polypeptides.","A method of selecting a therapy for a heart disorder and\/or cardiac disease and\/or cardiac pathology, comprising detecting any one of the foregoing polypeptides and selecting a therapy according to said detection.","Unless defined otherwise, all technical and scientific terms used herein have the meaning commonly understood by a person skilled in the art to which this invention belongs. The following references provide one of skill with a general definition of many of the terms used in this invention: Singleton et al., Dictionary of Microbiology and Molecular Biology (2nd ed. 1994); The Cambridge Dictionary of Science and Technology (Walker ed., 1988); The Glossary of Genetics, 5th Ed., R. Rieger et al. (eds.), Springer Verlag (1991); and Hale & Marham, The Harper Collins Dictionary of Biology (1991). All of these are hereby incorporated by reference as if fully set forth herein. As used herein, the following terms have the meanings ascribed to them unless specified otherwise.","The present invention is of novel markers for cardiac disease that are both sensitive and accurate. Biomolecular sequences (amino acid and\/or nucleic acid sequences) uncovered using the methodology of the present invention and described herein can be efficiently utilized as tissue or pathological markers and\/or as drugs or drug targets for treating or preventing a disease.","These markers are specifically released to the bloodstream under conditions of cardiac disease and\/or cardiac pathology, including but not limited to cardiac damage, and\/or are otherwise expressed at a much higher level and\/or specifically expressed in heart. The method of the present invention identifies clusters (genes) which are characterized in that the transcripts are differentially expressed in heart muscle tissue compared with other normal tissues, preferably in comparison to skeletal muscle tissue. In acute conditions under which heart muscle tissue experiences hypoxia (with or without necrosis), intracellular proteins that are not normally secreted can leak through the cell membrane to the extracellular space. Therefore, heart muscle tissue differentially expressed proteins, as through analysis of EST expression, are potential acute heart damage markers.","Leakage of intracellular content can also occur in chronic damage to the heart muscle, therefore proteins selected according to this method are potential markers for chronic heart conditions. When a protein that is differentially expressed in heart muscle is secreted, it is even more likely to be useful as a chronic heart damage marker, since secretion implies that the protein has a physiological role exterior to the cell, and therefore may be used by the heart muscle to respond to the chronic damage. This rationale is empirically supported by the non-limiting examples of the proteins BNP (brain natriuretic peptide) and ANF (atrial natriuretic factor), which are differentially expressed heart muscle proteins that are secreted and which were shown to be markers for congestive heart failure. In addition, BNP and ANF are not only differentially expressed in heart tissue, they are also overexpressed dramatically (hundreds of times greater expression) when heart failure occurs. Other heart specific secreted proteins might present similar overexpression in chronic damage.","Optionally and preferably, the markers described herein are overexpressed in heart as opposed to muscle, as described in greater detail below. The measurement of these markers, alone or in combination, in patient samples provides information that the diagnostician can correlate with a probable diagnosis of cardiac disease and\/or cardiac pathology, including but not limited to cardiac damage.","The present invention therefore also relates to diagnostic assays for cardiac disease and\/or cardiac pathology, including but not limited to cardiac damage, and methods of use of such markers for detection of cardiac disease and\/or cardiac pathology, including but not limited to cardiac damage (alone or in combination), optionally and preferably in a sample taken from a subject (patient), which is more preferably some type of blood sample.","The present invention therefore also relates to diagnostic assays for cardiac disease and\/or cardiac pathology, including but not limited to cardiac damage, and methods of use of such markers for detection of cardiac disease and\/or cardiac pathology, including but not limited to cardiac damage (alone or in combination), optionally and preferably in a sample taken from a subject (patient), which is more preferably some type of blood sample.","In another embodiment, the present invention relates to bridges, tails, heads and\/or insertions, and\/or analogs, homologs and derivatives of such peptides. Such bridges, tails, heads and\/or insertions are described in greater detail below with regard to the Examples.","As used herein a \u201ctail\u201d refers to a peptide sequence at the end of an amino acid sequence that is unique to a splice variant according to the present invention. Therefore, a splice variant having such a tail may optionally be considered as a chimera, in that at least a first portion of the splice variant is typically highly homologous (often 100% identical) to a portion of the corresponding known protein, while at least a second portion of the variant comprises the tail.","As used herein a \u201chead\u201d refers to a peptide sequence at the beginning of an amino acid sequence that is unique to a splice variant according to the present invention. Therefore, a splice variant having such a head may optionally be considered as a chimera, in that at least a first portion of the splice variant comprises the head, while at least a second portion is typically highly homologous (often 100% identical) to a portion of the corresponding known protein.","As used herein \u201can edge portion\u201d refers to a connection between two portions of a splice variant according to the present invention that were not joined in the wild type or known protein. An edge may optionally arise due to a join between the above \u201cknown protein\u201d portion of a variant and the tail, for example, and\/or may occur if an internal portion of the wild type sequence is no longer present, such that two portions of the sequence are now contiguous in the splice variant that were not contiguous in the known protein. A \u201cbridge\u201d may optionally be an edge portion as described above, but may also include a join between a head and a \u201cknown protein\u201d portion of a variant, or a join between a tail and a \u201cknown protein\u201d portion of a variant, or a join between an insertion and a \u201cknown protein\u201d portion of a variant.","Optionally and preferably, a bridge between a tail or a head or a unique insertion, and a \u201cknown protein\u201d portion of a variant, comprises at least about 10 amino acids, more preferably at least about 20 amino acids, most preferably at least about 30 amino acids, and even more preferably at least about 40 amino acids, in which at least one amino acid is from the tail\/head\/insertion and at least one amino acid is from the \u201cknown protein\u201d portion of a variant. Also optionally, the bridge may comprise any number of amino acids from about 10 to about 40 amino acids (for example, 10, 11, 12, 13 . . . 37, 38, 39, 40 amino acids in length, or any number in between).","It should be noted that a bridge cannot be extended beyond the length of the sequence in either direction, and it should be assumed that every bridge description is to be read in such manner that the bridge length does not extend beyond the sequence itself.","Furthermore, bridges are described with regard to a sliding window in certain contexts below. For example, certain descriptions of the bridges feature the following format: a bridge between two edges (in which a portion of the known protein is not present in the variant) may optionally be described as follows: a bridge portion of CONTIG-NAME_P1 (representing the name of the protein), comprising a polypeptide having a length \u201cn\u201d, wherein n is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least two amino acids comprise XX (2 amino acids in the center of the bridge, one from each end of the edge), having a structure as follows (numbering according to the sequence of CONTIG-NAME_P1): a sequence starting from any of amino acid numbers 49\u2212x to 49 (for example); and ending at any of amino acid numbers 50+((n\u22122)\u2212x) (for example), in which x varies from 0 to n\u22122. In this example, it should also be read as including bridges in which n is any number of amino acids between 10-50 amino acids in length. Furthermore, the bridge polypeptide cannot extend beyond the sequence, so it should be read such that 49\u2212x (for example) is not less than 1, nor 50+((n\u22122)\u2212x) (for example) greater than the total sequence length.","In another embodiment, this invention provides antibodies specifically recognizing the splice variants and polypeptide fragments thereof of this invention. Preferably such antibodies differentially recognize splice variants of the present invention but do not recognize a corresponding known protein (such known proteins are discussed with regard to their splice variants in the Examples below).","In another embodiment, this invention provides an isolated nucleic acid molecule encoding for a splice variant according to the present invention, having a nucleotide sequence as set forth in any one of the sequences listed herein, or a sequence complementary thereto. In another embodiment, this invention provides an isolated nucleic acid molecule, having a nucleotide sequence as set forth in any one of the sequences listed herein, or a sequence complementary thereto. In another embodiment, this invention provides an oligonucleotide of at least about 12 nucleotides, specifically hybridizable with the nucleic acid molecules of this invention. In another embodiment, this invention provides vectors, cells, liposomes and compositions comprising the isolated nucleic acids of this invention.","In another embodiment, this invention provides a method for detecting a splice variant according to the present invention in a biological sample, comprising: contacting a biological sample with an antibody specifically recognizing a splice variant according to the present invention under conditions whereby the antibody specifically interacts with the splice variant in the biological sample but do not recognize known corresponding proteins (wherein the known protein is discussed with regard to its splice variant(s) in the Examples below), and detecting said interaction; wherein the presence of an interaction correlates with the presence of a splice variant in the biological sample.","In another embodiment, this invention provides a method for detecting a splice variant nucleic acid sequences in a biological sample, comprising: hybridizing the isolated nucleic acid molecules or oligonucleotide fragments of at least about a minimum length to a nucleic acid material of a biological sample and detecting a hybridization complex; wherein the presence of a hybridization complex correlates with the presence of a splice variant nucleic acid sequence in the biological sample.","According to the present invention, the splice variants described herein are non-limiting examples of markers for diagnosing cardiac disease and\/or cardiac pathology, including but not limited to cardiac damage. Each splice variant marker of the present invention can be used alone or in combination, for various uses, including but not limited to, prognosis, prediction, screening, early diagnosis, determination of progression, therapy selection and treatment monitoring of cardiac disease and\/or cardiac pathology, including but not limited to cardiac damage.","According to optional but preferred embodiments of the present invention, any marker according to the present invention may optionally be used alone or combination. Such a combination may optionally comprise a plurality of markers described herein, optionally including any subcombination of markers, and\/or a combination featuring at least one other marker, for example a known marker. Furthermore, such a combination may optionally and preferably be used as described above with regard to determining a ratio between a quantitative or semi-quantitative measurement of any marker described herein to any other marker described herein, and\/or any other known marker, and\/or any other marker. With regard to such a ratio between any marker described herein (or a combination thereof) and a known marker, more preferably the known marker comprises the \u201cknown protein\u201d as described in greater detail below with regard to each cluster or gene.","According to other preferred embodiments of the present invention, a splice variant protein or a fragment thereof, or a splice variant nucleic acid sequence or a fragment thereof, may be featured as a biomarker for detecting cardiac disease and\/or cardiac pathology, including but not limited to cardiac damage, such that a biomarker may optionally comprise any of the above. According to still other preferred embodiments, the present invention optionally and preferably encompasses any amino acid sequence or fragment thereof encoded by a nucleic acid sequence corresponding to a splice variant protein as described herein. Any oligopeptide or peptide relating to such an amino acid sequence or fragment thereof may optionally also (additionally or alternatively) be used as a biomarker, including but not limited to the unique amino acid sequences of these proteins that are depicted as tails, heads, insertions, edges or bridges. The present invention also optionally encompasses antibodies capable of recognizing, and\/or being elicited by, such oligopeptides or peptides.","The present invention also optionally and preferably encompasses any nucleic acid sequence or fragment thereof, or amino acid sequence or fragment thereof, corresponding to a splice variant of the present invention as described above, optionally for any application.","Non-limiting examples of methods or assays are described below.","The present invention also relates to kits based upon such diagnostic methods or assays.","Nucleic Acid Sequences and Oligonucleotides","Various embodiments of the present invention encompass nucleic acid sequences described hereinabove; fragments thereof, sequences hybridizable therewith, sequences homologous thereto, sequences encoding similar polypeptides with different codon usage, altered sequences characterized by mutations, such as deletion, insertion or substitution of one or more nucleotides, either naturally occurring or artificially induced, either randomly or in a targeted fashion.","The present invention encompasses nucleic acid sequences described herein; fragments thereof, sequences hybridizable therewith, sequences homologous thereto [e.g., at least 50%, at least 55%, at least 60%, at least 65%, at least 70%, at least 75%, at least 80%, at least 85%, at least 95% or more say 100% identical to the nucleic acid sequences set forth below], sequences encoding similar polypeptides with different codon usage, altered sequences characterized by mutations, such as deletion, insertion or substitution of one or more nucleotides, either naturally occurring or man induced, either randomly or in a targeted fashion. The present invention also encompasses homologous nucleic acid sequences (i.e., which form a part of a polynucleotide sequence of the present invention) which include sequence regions unique to the polynucleotides of the present invention.","In cases where the polynucleotide sequences of the present invention encode previously unidentified polypeptides, the present invention also encompasses novel polypeptides or portions thereof, which are encoded by the isolated polynucleotide and respective nucleic acid fragments thereof described hereinabove.","A \u201cnucleic acid fragment\u201d or an \u201coligonucleotide\u201d or a \u201cpolynucleotide\u201d are used herein interchangeably to refer to a polymer of nucleic acids. A polynucleotide sequence of the present invention refers to a single or double stranded nucleic acid sequences which is isolated and provided in the form of an RNA sequence, a complementary polynucleotide sequence (cDNA), a genomic polynucleotide sequence and\/or a composite polynucleotide sequences (e.g., a combination of the above).","As used herein the phrase \u201ccomplementary polynucleotide sequence\u201d refers to a sequence, which results from reverse transcription of messenger RNA using a reverse transcriptase or any other RNA dependent DNA polymerase. Such a sequence can be subsequently amplified in vivo or in vitro using a DNA dependent DNA polymerase.","As used herein the phrase \u201cgenomic polynucleotide sequence\u201d refers to a sequence derived (isolated) from a chromosome and thus it represents a contiguous portion of a chromosome.","As used herein the phrase \u201ccomposite polynucleotide sequence\u201d refers to a sequence, which is composed of genomic and cDNA sequences. A composite sequence can include some exonal sequences required to encode the polypeptide of the present invention, as well as some intronic sequences interposing therebetween. The intronic sequences can be of any source, including of other genes, and typically will include conserved splicing signal sequences. Such intronic sequences may further include cis acting expression regulatory elements.","Preferred embodiments of the present invention encompass oligonucleotide probes.","An example of an oligonucleotide probe which can be utilized by the present invention is a single stranded polynucleotide which includes a sequence complementary to the unique sequence region of any variant according to the present invention, including but not limited to a nucleotide sequence coding for an amino sequence of a bridge, tail, head and\/or insertion according to the present invention, and\/or the equivalent portions of any nucleotide sequence given herein (including but not limited to a nucleotide sequence of a node, segment or amplicon described herein).","Alternatively, an oligonucleotide probe of the present invention can be designed to hybridize with a nucleic acid sequence encompassed by any of the above nucleic acid sequences, particularly the portions specified above, including but not limited to a nucleotide sequence coding for an amino sequence of a bridge, tail, head and\/or insertion according to the present invention, and\/or the equivalent portions of any nucleotide sequence given herein (including but not limited to a nucleotide sequence of a node, segment or amplicon described herein).","Oligonucleotides designed according to the teachings of the present invention can be generated according to any oligonucleotide synthesis method known in the art such as enzymatic synthesis or solid phase synthesis. Equipment and reagents for executing solid-phase synthesis are commercially available from, for example, Applied Biosystems. Any other means for such synthesis may also be employed; the actual synthesis of the oligonucleotides is well within the capabilities of one skilled in the art and can be accomplished via established methodologies as detailed in, for example, \u201cMolecular Cloning: A laboratory Manual\u201d Sambrook et al., (1989); \u201cCurrent Protocols in Molecular Biology\u201d Volumes I-III Ausubel, R. M., ed. (1994); Ausubel et al., \u201cCurrent Protocols in Molecular Biology\u201d, John Wiley and Sons, Baltimore, Md. (1989); Perbal, \u201cA Practical Guide to Molecular Cloning\u201d, John Wiley & Sons, New York (1988) and \u201cOligonucleotide Synthesis\u201d Gait, M. J., ed. (1984) utilizing solid phase chemistry, e.g. cyanoethyl phosphoramidite followed by deprotection, desalting and purification by for example, an automated trityl-on method or HPLC.","Oligonucleotides used according to this aspect of the present invention are those having a length selected from a range of about 10 to about 200 bases preferably about 15 to about 150 bases, more preferably about 20 to about 100 bases, most preferably about 20 to about 50 bases. Preferably, the oligonucleotide of the present invention features at least 17, at least 18, at least 19, at least 20, at least 22, at least 25, at least 30 or at least 40, bases specifically hybridizable with the biomarkers of the present invention.","The oligonucleotides of the present invention may comprise heterocylic nucleosides consisting of purines and the pyrimidines bases, bonded in a 3\u2032 to 5\u2032 phosphodiester linkage.","Preferably used oligonucleotides are those modified at one or more of the backbone, internucleoside linkages or bases, as is broadly described hereinunder.","Specific examples of preferred oligonucleotides useful according to this aspect of the present invention include oligonucleotides containing modified backbones or non-natural internucleoside linkages. Oligonucleotides having modified backbones include those that retain a phosphorus atom in the backbone, as disclosed in U.S. Pat. Nos. 4,469,863; 4,476,301; 5,023,243; 5,177,196; 5,188,897; 5,264,423; 5,276,019; 5,278,302; 5,286,717; 5,321,131; 5,399,676; 5,405,939; 5,453,496; 5,455,233; 5,466,677; 5,476,925; 5,519,126; 5,536,821; 5,541,306; 5,550,111; 5,563,253; 5,571,799; 5,587,361; and 5,625,050.","Preferred modified oligonucleotide backbones include, for example, phosphorothioates, chiral phosphorothioates, phosphorodithioates, phosphotriesters, aminoalkyl phosphotriesters, methyl and other alkyl phosphonates including 3\u2032-alkylene phosphonates and chiral phosphonates, phosphinates, phosphoramidates including 3\u2032-amino phosphoramidate and aminoalkylphosphoramidates, thionophosphoramidates, thionoalkylphosphonates, thionoalkylphosphotriesters, and boranophosphates having normal 3\u2032-5\u2032 linkages, 2\u2032-5\u2032 linked analogs of these, and those having inverted polarity wherein the adjacent pairs of nucleoside units are linked 3\u2032-5\u2032 to 5\u2032-3\u2032 or 2\u2032-5\u2032 to 5\u2032-2\u2032. Various salts, mixed salts and free acid forms can also be used.","Alternatively, modified oligonucleotide backbones that do not include a phosphorus atom therein have backbones that are formed by short chain alkyl or cycloalkyl internucleoside linkages, mixed heteroatom and alkyl or cycloalkyl internucleoside linkages, or one or more short chain heteroatomic or heterocyclic internucleoside linkages. These include those having morpholino linkages (formed in part from the sugar portion of a nucleoside); siloxane backbones; sulfide, sulfoxide and sulfone backbones; formacetyl and thioformacetyl backbones; methylene formacetyl and thioformacetyl backbones; alkene containing backbones; sulfamate backbones; methyleneimino and methylenehydrazino backbones; sulfonate and sulfonamide backbones; amide backbones; and others having mixed N, O, S and CHcomponent parts, as disclosed in U.S. Pat. Nos. 5,034,506; 5,166,315; 5,185,444; 5,214,134; 5,216,141; 5,235,033; 5,264,562; 5,264,564; 5,405,938; 5,434,257; 5,466,677; 5,470,967; 5,489,677; 5,541,307; 5,561,225; 5,596,086; 5,602,240; 5,610,289; 5,602,240; 5,608,046; 5,610,289; 5,618,704; 5,623,070; 5,663,312; 5,633,360; 5,677,437; and 5,677,439.","Other oligonucleotides which can be used according to the present invention, are those modified in both sugar and the internucleoside linkage, i.e., the backbone, of the nucleotide units are replaced with novel groups. The base units are maintained for complementation with the appropriate polynucleotide target. An example for such an oligonucleotide mimetic, includes peptide nucleic acid (PNA). United States patents that teach the preparation of PNA compounds include, but are not limited to, U.S. Pat. Nos. 5,539,082; 5,714,331; and 5,719,262, each of which is herein incorporated by reference. Other backbone modifications, which can be used in the present invention are disclosed in U.S. Pat. No. 6,303,374.","Oligonucleotides of the present invention may also include base modifications or substitutions. As used herein, \u201cunmodified\u201d or \u201cnatural\u201d bases include the purine bases adenine (A) and guanine (G), and the pyrimidine bases thymine (T), cytosine (C) and uracil (U). Modified bases include but are not limited to other synthetic and natural bases such as 5-methylcytosine (5-me-C), 5-hydroxymethyl cytosine, xanthine, hypoxanthine, 2-aminoadenine, 6-methyl and other alkyl derivatives of adenine and guanine, 2-propyl and other alkyl derivatives of adenine and guanine, 2-thiouracil, 2-thiothymine and 2-thiocytosine, 5-halouracil and cytosine, 5-propynyl uracil and cytosine, 6-azo uracil, cytosine and thymine, 5-uracil (pseudouracil), 4-thiouracil, 8-halo, 8-amino, 8-thiol, 8-thioalkyl, 8-hydroxyl and other 8-substituted adenines and guanines, 5-halo particularly 5-bromo, 5-trifluoromethyl and other 5-substituted uracils and cytosines, 7-methylguanine and 7-methyladenine, 8-azaguanine and 8-azaadenine, 7-deazaguanine and 7-deazaadenine and 3-deazaguanine and 3-deazaadenine. Further bases particularly useful for increasing the binding affinity of the oligomeric compounds of the invention include 5-substituted pyrimidines, 6-azapyrimidines and N-2, N-6 and O-6 substituted purines, including 2-aminopropyladenine, 5-propynyluracil and 5-propynylcytosine. 5-methylcytosine substitutions have been shown to increase nucleic acid duplex stability by 0.6-1.2\u00b0 C. and are presently preferred base substitutions, even more particularly when combined with 2\u2032-O-methoxyethyl sugar modifications.","Another modification of the oligonucleotides of the invention involves chemically linking to the oligonucleotide one or more moieties or conjugates, which enhance the activity, cellular distribution or cellular uptake of the oligonucleotide. Such moieties include but are not limited to lipid moieties such as a cholesterol moiety, cholic acid, a thioether, e.g., hexyl-S-tritylthiol, a thiocholesterol, an aliphatic chain, e.g., dodecandiol or undecyl residues, a phospholipid, e g., di-hexadecyl-rac-glycerol or triethylammonium 1,2-di-O-hexadecyl-rac-glycero-3-H-phosphonate, a polyamine or a polyethylene glycol chain, or adamantane acetic acid, a palmityl moiety, or an octadecylamine or hexylamino-carbonyl-oxycholesterol moiety, as disclosed in U.S. Pat. No. 6,303,374.","It is not necessary for all positions in a given oligonucleotide molecule to be uniformly modified, and in fact more than one of the aforementioned modifications may be incorporated in a single compound or even at a single nucleoside within an oligonucleotide.","It will be appreciated that oligonucleotides of the present invention may include further modifications for more efficient use as diagnostic agents and\/or to increase bioavailability, therapeutic efficacy and reduce cytotoxicity.","To enable cellular expression of the polynucleotides of the present invention, a nucleic acid construct according to the present invention may be used, which includes at least a coding region of one of the above nucleic acid sequences, and further includes at least one cis acting regulatory element. As used herein, the phrase \u201ccis acting regulatory element\u201d refers to a polynucleotide sequence, preferably a promoter, which binds a trans acting regulator and regulates the transcription of a coding sequence located downstream thereto.","Any suitable promoter sequence can be used by the nucleic acid construct of the present invention.","Preferably, the promoter utilized by the nucleic acid construct of the present invention is active in the specific cell population transformed. Examples of cell type-specific and\/or tissue-specific promoters include promoters such as albumin that is liver specific, lymphoid specific promoters [Calame et al., (1988) Adv. Immunol. 43:235-275]; in particular promoters of T-cell receptors [Winoto et al., (1989) EMBO J. 8:729-733] and immunoglobulins; [Banerji et al. (1983) Cell 33729-740], neuron-specific promoters such as the neurofilament promoter [Byrne et al. (1989) Proc. Natl. Acad. Sci. USA 86:5473-5477], pancreas-specific promoters [Edlunch et al. (1985) Science 230:912-916] or mammary gland-specific promoters such as the milk whey promoter (U.S. Pat. No. 4,873,316 and European Application Publication No. 264,166). The nucleic acid construct of the present invention can further include an enhancer, which can be adjacent or distant to the promoter sequence and can function in up regulating the transcription therefrom.","The nucleic acid construct of the present invention preferably further includes an appropriate selectable marker and\/or an origin of replication. Preferably, the nucleic acid construct utilized is a shuttle vector, which can propagate both in (wherein the construct comprises an appropriate selectable marker and origin of replication) and be compatible for propagation in cells, or integration in a gene and a tissue of choice. The construct according to the present invention can be, for example, a plasmid, a bacmid, a phagemid, a cosmid, a phage, a virus or an artificial chromosome.","Examples of suitable constructs include, but are not limited to, pcDNA3, pcDNA3.1 (+\/\u2212), pGL3, PzeoSV2 (+\/\u2212), pDisplay, pEF\/myc\/cyto, pCMV\/myc\/cyto each of which is commercially available from Invitrogen Co. (dot invitrogen dot com). Examples of retroviral vector and packaging systems are those sold by Clontech, San Diego, Calif., including Retro-X vectors pLNCX and pLXSN, which permit cloning into multiple cloning sites and the trasgene is transcribed from CMV promoter. Vectors derived from Mo-MuLV are also included such as pBabe, where the transgene will be transcribed from the 5\u2032LTR promoter.","Currently preferred in vivo nucleic acid transfer techniques include transfection with viral or non-viral constructs, such as adenovirus, lentivirus, Herpes simplex I virus, or adeno-associated virus (AAV) and lipid-based systems. Useful lipids for lipid-mediated transfer of the gene are, for example, DOTMA, DOPE, and DC-Chol [Tonkinson et al., Cancer Investigation, 14(1): 54-65 (1996)]. The most preferred constructs for use in gene therapy are viruses, most preferably adenoviruses, AAV, lentiviruses, or retroviruses. A viral construct such as a retroviral construct includes at least one transcriptional promoter\/enhancer or locus-defining element(s), or other elements that control gene expression by other means such as alternate splicing, nuclear RNA export, or post-translational modification of messenger. Such vector constructs also include a packaging signal, long terminal repeats (LTRs) or portions thereof, and positive and negative strand primer binding sites appropriate to the virus used, unless it is already present in the viral construct. In addition, such a construct typically includes a signal sequence for secretion of the peptide from a host cell in which it is placed. Preferably the signal sequence for this purpose is a mammalian signal sequence or the signal sequence of the polypeptide variants of the present invention. Optionally, the construct may also include a signal that directs polyadenylation, as well as one or more restriction sites and a translation termination sequence. By way of example, such constructs will typically include a 5\u2032 LTR, a tRNA binding site, a packaging signal, an origin of second-strand DNA synthesis, and a 3\u2032 LTR or a portion thereof. Other vectors can be used that are non-viral, such as cationic lipids, polylysine, and dendrimers.","Hybridization Assays","Detection of a nucleic acid of interest in a biological sample may optionally be effected by hybridization-based assays using an oligonucleotide probe (non-limiting examples of probes according to the present invention were previously described).","Traditional hybridization assays include PCR, RT-PCR, Real-time PCR, RNase protection, in-situ hybridization, primer extension, Southern blots (DNA detection), dot or slot blots (DNA, RNA), and Northern blots (RNA detection) (NAT type assays are described in greater detail below). More recently, PNAs have been described (Nielsen et al. 1999, Current Opin. Biotechnol. 10:71-75). Other detection methods include kits containing probes on a dipstick setup and the like.","Hybridization based assays which allow the detection of a variant of interest (i.e., DNA or RNA) in a biological sample rely on the use of oligonucleotides which can be 10, 15, 20, or 30 to 100 nucleotides long preferably from 10 to 50, more preferably from 40 to 50 nucleotides long.","Thus, the isolated polynucleotides (oligonucleotides) of the present invention are preferably hybridizable with any of the herein described nucleic acid sequences under moderate to stringent hybridization conditions.","Moderate to stringent hybridization conditions are characterized by a hybridization solution such as containing 10% dextrane sulfate, 1 M NaCl, 1% SDS and 5\u00d710cpm P labeled probe, at 65\u00b0 C., with a final wash solution of 0.2\u00d7SSC and 0.1% SDS and final wash at 65\u00b0 C. and whereas moderate hybridization is effected using a hybridization solution containing 10% dextrane sulfate, 1 M NaCl, 1% SDS and 5\u00d710cpm P labeled probe, at 65\u00b0 C., with a final wash solution of 1\u00d7SSC and 0.1% SDS and final wash at 50\u00b0 C.","More generally, hybridization of short nucleic acids (below 200 bp in length, e.g. 17-40 bp in length) can be effected using the following exemplary hybridization protocols which can be modified according to the desired stringency; (i) hybridization solution of 6\u00d7SSC and 1% SDS or 3 M TMACI, 0.01 M sodium phosphate (pH 6.8), 1 mM EDTA (pH 7.6), 0.5% SDS, 100 \u03bcg\/ml denatured salmon sperm DNA and 0.1% nonfat dried milk, hybridization temperature of 1-1.5\u00b0 C. below the T, final wash solution of 3 M TMACI, 0.01 M sodium phosphate (pH 6.8), 1 mM EDTA (pH 7.6), 0.5% SDS at 1-1.5\u00b0 C. below the T; (ii) hybridization solution of 6\u00d7SSC and 0.1% SDS or 3 M TMACI, 0.01 M sodium phosphate (pH 6.8), 1 mM EDTA (pH 7.6), 0.5% SDS, 100 \u03bcg\/ml denatured salmon sperm DNA and 0.1% nonfat dried milk, hybridization temperature of 2-2.5\u00b0 C. below the T, final wash solution of 3 M TMACI, 0.01 M sodium phosphate (pH 6.8), 1 mM EDTA (pH 7.6), 0.5% SDS at 1-1.5\u00b0 C. below the T, final wash solution of 6\u00d7SSC, and final wash at 22\u00b0 C.; (iii) hybridization solution of 6\u00d7SSC and 1% SDS or 3 M TMACI, 0.01 M sodium phosphate (pH 6.8), 1 mM EDTA (pH 7.6), 0.5% SDS, 100 \u03bcg\/ml denatured salmon sperm DNA and 0.1% nonfat dried milk, hybridization temperature.","The detection of hybrid duplexes can be carried out by a number of methods. Typically, hybridization duplexes are separated from unhybridized nucleic acids and the labels bound to the duplexes are then detected. Such labels refer to radioactive, fluorescent, biological or enzymatic tags or labels of standard use in the art. A label can be conjugated to either the oligonucleotide probes or the nucleic acids derived from the biological sample.","Probes can be labeled according to numerous well known methods. Non-limiting examples of radioactive labels include 3H, 14C, 32P, and 35S. Non-limiting examples of detectable markers include ligands, fluorophores, chemiluminescent agents, enzymes, and antibodies. Other detectable markers for use with probes, which can enable an increase in sensitivity of the method of the invention, include biotin and radio-nucleotides. It will become evident to the person of ordinary skill that the choice of a particular label dictates the manner in which it is bound to the probe.","For example, oligonucleotides of the present invention can be labeled subsequent to synthesis, by incorporating biotinylated dNTPs or rNTP, or some similar means (e.g., photo-cross-linking a psoralen derivative of biotin to RNAs), followed by addition of labeled streptavidin (e.g., phycoerythrin-conjugated streptavidin) or the equivalent. Alternatively, when fluorescently-labeled oligonucleotide probes are used, fluorescein, lissamine, phycoerythrin, rhodamine (Perkin Elmer Cetus), Cy2, Cy3, Cy3.5, Cy5, Cy5.5, Cy7, FluorX (Amersham) and others [e.g., Kricka et al. (1992), Academic Press San Diego, Calif.] can be attached to the oligonucleotides.","Those skilled in the art will appreciate that wash steps may be employed to wash away excess target DNA or probe as well as unbound conjugate. Further, standard heterogeneous assay formats are suitable for detecting the hybrids using the labels present on the oligonucleotide primers and probes.","It will be appreciated that a variety of controls may be usefully employed to improve accuracy of hybridization assays. For instance, samples may be hybridized to an irrelevant probe and treated with RNAse A prior to hybridization, to assess false hybridization.","Although the present invention is not specifically dependent on the use of a label for the detection of a particular nucleic acid sequence, such a label might be beneficial, by increasing the sensitivity of the detection. Furthermore, it enables automation. Probes can be labeled according to numerous well known methods.","As commonly known, radioactive nucleotides can be incorporated into probes of the invention by several methods. Non-limiting examples of radioactive labels include H, C, P, and S.","Those skilled in the art will appreciate that wash steps may be employed to wash away excess target DNA or probe as well as unbound conjugate. Further, standard heterogeneous assay formats are suitable for detecting the hybrids using the labels present on the oligonucleotide primers and probes.","It will be appreciated that a variety of controls may be usefully employed to improve accuracy of hybridization assays.","Probes of the invention can be utilized with naturally occurring sugar-phosphate backbones as well as modified backbones including phosphorothioates, dithionates, alkyl phosphonates and a-nucleotides and the like. Probes of the invention can be constructed of either ribonucleic acid (RNA) or deoxyribonucleic acid (DNA), and preferably of DNA.","NAT Assays","Detection of a nucleic acid of interest in a biological sample may also optionally be effected by NAT-based assays, which involve nucleic acid amplification technology, such as PCR for example (or variations thereof such as real-time PCR for example).","As used herein, a \u201cprimer\u201d defines an oligonucleotide which is capable of annealing to (hybridizing with) a target sequence, thereby creating a double stranded region which can serve as an initiation point for DNA synthesis under suitable conditions.","Amplification of a selected, or target, nucleic acid sequence may be carried out by a number of suitable methods. See generally Kwoh et al., 1990, Am. Biotechnol. Lab. 8:14 Numerous amplification techniques have been described and can be readily adapted to suit particular needs of a person of ordinary skill. Non-limiting examples of amplification techniques include polymerase chain reaction (PCR), ligase chain reaction (LCR), strand displacement amplification (SDA), transcription-based amplification, the q3 replicase system and NASBA (Kwoh et al., 1989, Proc. NatI. Acad. Sci. USA 86, 1173-1177; Lizardi et al., 1988, BioTechnology 6:1197-1202; Malek et al., 1994, Methods Mol. Biol., 28:253-260; and Sambrook et al., 1989, supra).","The terminology \u201camplification pair\u201d (or \u201cprimer pair\u201d) refers herein to a pair of oligonucleotides (oligos) of the present invention, which are selected to be used together in amplifying a selected nucleic acid sequence by one of a number of types of amplification processes, preferably a polymerase chain reaction. Other types of amplification processes include ligase chain reaction, strand displacement amplification, or nucleic acid sequence-based amplification, as explained in greater detail below. As commonly known in the art, the oligos are designed to bind to a complementary sequence under selected conditions.","In one particular embodiment, amplification of a nucleic acid sample from a patient is amplified under conditions which favor the amplification of the most abundant differentially expressed nucleic acid. In one preferred embodiment, RT-PCR is carried out on an mRNA sample from a patient under conditions which favor the amplification of the most abundant mRNA. In another preferred embodiment, the amplification of the differentially expressed nucleic acids is carried out simultaneously. It will be realized by a person skilled in the art that such methods could be adapted for the detection of differentially expressed proteins instead of differentially expressed nucleic acid sequences.","The nucleic acid (i.e. DNA or RNA) for practicing the present invention may be obtained according to well known methods.","Oligonucleotide primers of the present invention may be of any suitable length, depending on the particular assay format and the particular needs and targeted genomes employed. Optionally, the oligonucleotide primers are at least 12 nucleotides in length, preferably between 15 and 24 molecules, and they may be adapted to be especially suited to a chosen nucleic acid amplification system. As commonly known in the art, the oligonucleotide primers can be designed by taking into consideration the melting point of hybridization thereof with its targeted sequence (Sambrook et al., 1989, Molecular Cloning\u2014A Laboratory Manual, 2nd Edition, CSH Laboratories; Ausubel et al., 1989, in Current Protocols in Molecular Biology, John Wiley & Sons Inc., NewYork).","It will be appreciated that antisense oligonucleotides may be employed to quantify expression of a splice isoform of interest. Such detection is effected at the pre-mRNA level. Essentially the ability to quantitate transcription from a splice site of interest can be effected based on splice site accessibility. Oligonucleotides may compete with splicing factors for the splice site sequences. Thus, low activity of the antisense oligonucleotide is indicative of splicing activity.","The polymerase chain reaction and other nucleic acid amplification reactions are well known in the art (various non-limiting examples of these reactions are described in greater detail below). The pair of oligonucleotides according to this aspect of the present invention are preferably selected to have compatible melting temperatures (Tm), e.g., melting temperatures which differ by less than that 7\u00b0 C., preferably less than 5\u00b0 C., more preferably less than 4\u00b0 C., most preferably less than 3\u00b0 C., ideally between 3\u00b0 C. and 0\u00b0 C.","Polymerase Chain Reaction (PCR): The polymerase chain reaction (PCR), as described in U.S. Pat. Nos. 4,683,195 and 4,683,202 to Mullis and Mullis et al., is a method of increasing the concentration of a segment of target sequence in a mixture of genomic DNA without cloning or purification. This technology provides one approach to the problems of low target sequence concentration. PCR can be used to directly increase the concentration of the target to an easily detectable level. This process for amplifying the target sequence involves the introduction of a molar excess of two oligonucleotide primers which are complementary to their respective strands of the double-stranded target sequence to the DNA mixture containing the desired target sequence. The mixture is denatured and then allowed to hybridize. Following hybridization, the primers are extended with polymerase so as to form complementary strands. The steps of denaturation, hybridization (annealing), and polymerase extension (elongation) can be repeated as often as needed, in order to obtain relatively high concentrations of a segment of the desired target sequence.","The length of the segment of the desired target sequence is determined by the relative positions of the primers with respect to each other, and, therefore, this length is a controllable parameter. Because the desired segments of the target sequence become the dominant sequences (in terms of concentration) in the mixture, they are said to be \u201cPCR-amplified.\u201d","Ligase Chain Reaction (LCR or LAR): The ligase chain reaction [LCR; sometimes referred to as \u201cLigase Amplification Reaction\u201d (LAR)] has developed into a well-recognized alternative method of amplifying nucleic acids. In LCR, four oligonucleotides, two adjacent oligonucleotides which uniquely hybridize to one strand of target DNA, and a complementary set of adjacent oligonucleotides, which hybridize to the opposite strand are mixed and DNA ligase is added to the mixture. Provided that there is complete complementarity at the junction, ligase will covalently link each set of hybridized molecules. Importantly, in LCR, two probes are ligated together only when they base-pair with sequences in the target sample, without gaps or mismatches. Repeated cycles of denaturation, and ligation amplify a short segment of DNA. LCR has also been used in combination with PCR to achieve enhanced detection of single-base changes: see for example Segev, PCT Publication No. W09001069 A1 (1990). However, because the four oligonucleotides used in this assay can pair to form two short ligatable fragments, there is the potential for the generation of target-independent background signal. The use of LCR for mutant screening is limited to the examination of specific nucleic acid positions.","Self-Sustained Synthetic Reaction (3SR\/NASBA): The self-sustained sequence replication reaction (3SR) is a transcription-based in vitro amplification system that can exponentially amplify RNA sequences at a uniform temperature. The amplified RNA can then be utilized for mutation detection. In this method, an oligonucleotide primer is used to add a phage RNA polymerase promoter to the 5\u2032 end of the sequence of interest. In a cocktail of enzymes and substrates that includes a second primer, reverse transcriptase, RNase H, RNA polymerase and ribo- and deoxyribonucleoside triphosphates, the target sequence undergoes repeated rounds of transcription, cDNA synthesis and second-strand synthesis to amplify the area of interest. The use of 3SR to detect mutations is kinetically limited to screening small segments of DNA (e.g., 200-300 base pairs).","Q-Beta (Q\u03b2) Replicase: In this method, a probe which recognizes the sequence of interest is attached to the replicatable RNA template for Q\u03b2 replicase. A previously identified major problem with false positives resulting from the replication of unhybridized probes has been addressed through use of a sequence-specific ligation step. However, available thermostable DNA ligases are not effective on this RNA substrate, so the ligation must be performed by T4 DNA ligase at low temperatures (37 degrees C.). This prevents the use of high temperature as a means of achieving specificity as in the LCR, the ligation event can be used to detect a mutation at the junction site, but not elsewhere.","A successful diagnostic method must be very specific. A straight-forward method of controlling the specificity of nucleic acid hybridization is by controlling the temperature of the reaction. While the 3SR\/NASBA, and Q\u03b2 systems are all able to generate a large quantity of signal, one or more of the enzymes involved in each cannot be used at high temperature (i.e., >55 degrees C.). Therefore the reaction temperatures cannot be raised to prevent non-specific hybridization of the probes. If probes are shortened in order to make them melt more easily at low temperatures, the likelihood of having more than one perfect match in a complex genome increases. For these reasons, PCR and LCR currently dominate the research field in detection technologies.","The basis of the amplification procedure in the PCR and LCR is the fact that the products of one cycle become usable templates in all subsequent cycles, consequently doubling the population with each cycle. The final yield of any such doubling system can be expressed as: (1+X)=y, where \u201cX\u201d is the mean efficiency (percent copied in each cycle), \u201cn\u201d is the number of cycles, and \u201cy\u201d is the overall efficiency, or yield of the reaction. If every copy of a target DNA is utilized as a template in every cycle of a polymerase chain reaction, then the mean efficiency is 100%. If 20 cycles of PCR are performed, then the yield will be 2, or 1,048,576 copies of the starting material. If the reaction conditions reduce the mean efficiency to 85%, then the yield in those 20 cycles will be only 1.85, or 220,513 copies of the starting material. In other words, a PCR running at 85% efficiency will yield only 21% as much fmal product, compared to a reaction running at 100% efficiency. A reaction that is reduced to 50% mean efficiency will yield less than 1% of the possible product.","In practice, routine polymerase chain reactions rarely achieve the theoretical maximum yield, and PCRs are usually run for more than 20 cycles to compensate for the lower yield. At 50% mean efficiency, it would take 34 cycles to achieve the million-fold amplification theoretically possible in 20, and at lower efficiencies, the number of cycles required becomes prohibitive. In addition, any background products that amplify with a better mean efficiency than the intended target will become the dominant products.","Also, many variables can influence the mean efficiency of PCR, including target DNA length and secondary structure, primer length and design, primer and dNTP concentrations, and buffer composition, to name but a few. Contamination of the reaction with exogenous DNA (e.g., DNA spilled onto lab surfaces) or cross-contamination is also a major consideration. Reaction conditions must be carefully optimized for each different primer pair and target sequence, and the process can take days, even for an experienced investigator. The laboriousness of this process, including numerous technical considerations and other factors, presents a significant drawback to using PCR in the clinical setting. Indeed, PCR has yet to penetrate the clinical market in a significant way. The same concerns arise with LCR, as LCR must also be optimized to use different oligonucleotide sequences for each target sequence. In addition, both methods require expensive equipment, capable of precise temperature cycling.","Many applications of nucleic acid detection technologies, such as in studies of allelic variation, involve not only detection of a specific sequence in a complex background, but also the discrimination between sequences with few, or single, nucleotide differences. One method of the detection of allele-specific variants by PCR is based upon the fact that it is difficult for Taq polymerase to synthesize a DNA strand when there is a mismatch between the template strand and the 3\u2032 end of the primer. An allele-specific variant may be detected by the use of a primer that is perfectly matched with only one of the possible alleles; the mismatch to the other allele acts to prevent the extension of the primer, thereby preventing the amplification of that sequence. This method has a substantial limitation in that the base composition of the mismatch influences the ability to prevent extension across the mismatch, and certain mismatches do not prevent extension or have only a minimal effect.","A similar 3\u2032-mismatch strategy is used with greater effect to prevent ligation in the LCR. Any mismatch effectively blocks the action of the thermostable ligase, but LCR still has the drawback of target-independent background ligation products initiating the amplification. Moreover, the combination of PCR with subsequent LCR to identify the nucleotides at individual positions is also a clearly cumbersome proposition for the clinical laboratory.","The direct detection method according to various preferred embodiments of the present invention may be, for example a cycling probe reaction (CPR) or a branched DNA analysis.","When a sufficient amount of a nucleic acid to be detected is available, there are advantages to detecting that sequence directly, instead of making more copies of that target, (e.g., as in PCR and LCR). Most notably, a method that does not amplify the signal exponentially is more amenable to quantitative analysis. Even if the signal is enhanced by attaching multiple dyes to a single oligonucleotide, the correlation between the final signal intensity and amount of target is direct. Such a system has an additional advantage that the products of the reaction will not themselves promote further reaction, so contamination of lab surfaces by the products is not as much of a concern. Recently devised techniques have sought to eliminate the use of radioactivity and\/or improve the sensitivity in automatable formats. Two examples are the \u201cCycling Probe Reaction\u201d (CPR), and \u201cBranched DNA\u201d (bDNA).","Cycling probe reaction (CPR): The cycling probe reaction (CPR), uses a long chimeric oligonucleotide in which a central portion is made of RNA while the two termini are made of DNA. Hybridization of the probe to a target DNA and exposure to a thermostable RNase H causes the RNA portion to be digested. This destabilizes the remaining DNA portions of the duplex, releasing the remainder of the probe from the target DNA and allowing another probe molecule to repeat the process. The signal, in the form of cleaved probe molecules, accumulates at a linear rate. While the repeating process increases the signal, the RNA portion of the oligonucleotide is vulnerable to RNases that may carried through sample preparation.","Branched DNA: Branched DNA (bDNA), involves oligonucleotides with branched structures that allow each individual oligonucleotide to carry 35 to 40 labels (e.g., alkaline phosphatase enzymes). While this enhances the signal from a hybridization event, signal from non-specific binding is similarly increased.","The detection of at least one sequence change according to various preferred embodiments of the present invention may be accomplished by, for example restriction fragment length polymorphism (RFLP analysis), allele specific oligonucleotide (ASO) analysis, Denaturing\/Temperature Gradient Gel Electrophoresis (DGGE\/TGGE), Single-Strand Conformation Polymorphism (SSCP) analysis or Dideoxy fingerprinting (ddF).","The demand for tests which allow the detection of specific nucleic acid sequences and sequence changes is growing rapidly in clinical diagnostics. As nucleic acid sequence data for genes from humans and pathogenic organisms accumulates, the demand for fast, cost-effective, and easy-to-use tests for as yet mutations within specific sequences is rapidly increasing.","A handful of methods have been devised to scan nucleic acid segments for mutations. One option is to determine the entire gene sequence of each test sample (e.g., a bacterial isolate). For sequences under approximately 600 nucleotides, this may be accomplished using amplified material (e.g., PCR reaction products). This avoids the time and expense associated with cloning the segment of interest. However, specialized equipment and highly trained personnel are required, and the method is too labor-intense and expensive to be practical and effective in the clinical setting.","In view of the difficulties associated with sequencing, a given segment of nucleic acid may be characterized on several other levels. At the lowest resolution, the size of the molecule can be determined by electrophoresis by comparison to a known standard run on the same gel. A more detailed picture of the molecule may be achieved by cleavage with combinations of restriction enzymes prior to electrophoresis, to allow construction of an ordered map. The presence of specific sequences within the fragment can be detected by hybridization of a labeled probe, or the precise nucleotide sequence can be determined by partial chemical degradation or by primer extension in the presence of chain-terminating nucleotide analogs.","Restriction fragment length polymorphism (RFLP): For detection of single-base differences between like sequences, the requirements of the analysis are often at the highest level of resolution. For cases in which the position of the nucleotide in question is known in advance, several methods have been developed for examining single base changes without direct sequencing. For example, if a mutation of interest happens to fall within a restriction recognition sequence, a change in the pattern of digestion can be used as a diagnostic tool (e.g., restriction fragment length polymorphism [RFLP] analysis).","Single point mutations have been also detected by the creation or destruction of RFLPs. Mutations are detected and localized by the presence and size of the RNA fragments generated by cleavage at the mismatches. Single nucleotide mismatches in DNA heteroduplexes are also recognized and cleaved by some chemicals, providing an alternative strategy to detect single base substitutions, generically named the \u201cMismatch Chemical Cleavage\u201d (MCC). However, this method requires the use of osmium tetroxide and piperidine, two highly noxious chemicals which are not suited for use in a clinical laboratory.","RFLP analysis suffers from low sensitivity and requires a large amount of sample. When RFLP analysis is used for the detection of point mutations, it is, by its nature, limited to the detection of only those single base changes which fall within a restriction sequence of a known restriction endonuclease. Moreover, the majority of the available enzymes have 4 to 6 base-pair recognition sequences, and cleave too frequently for many large-scale DNA manipulations. Thus, it is applicable only in a small fraction of cases, as most mutations do not fall within such sites.","A handful of rare-cutting restriction enzymes with 8 base-pair specificities have been isolated and these are widely used in genetic mapping, but these enzymes are few in number, are limited to the recognition of G+C-rich sequences, and cleave at sites that tend to be highly clustered. Recently, endonucleases encoded by group I introns have been discovered that might have greater than 12 base-pair specificity, but again, these are few in number.","Allele specific oligonucleotide (ASO): If the change is not in a recognition sequence, then allele-specific oligonucleotides (ASOs), can be designed to hybridize in proximity to the mutated nucleotide, such that a primer extension or ligation event can bused as the indicator of a match or a mis-match. Hybridization with radioactively labeled allelic specific oligonucleotides (ASO) also has been applied to the detection of specific point mutations. The method is based on the differences in the melting temperature of short DNA fragments differing by a single nucleotide. Stringent hybridization and washing conditions can differentiate between mutant and wild-type alleles. The ASO approach applied to PCR products also has been extensively utilized by various researchers to detect and characterize point mutations in ras genes and gsp\/gip oncogenes. Because of the presence of various nucleotide changes in multiple positions, the ASO method requires the use of many oligonucleotides to cover all possible oncogenic mutations.","With either of the techniques described above (i.e., RFLP and ASO), the precise location of the suspected mutation must be known in advance of the test. That is to say, they are inapplicable when one needs to detect the presence of a mutation within a gene or sequence of interest.","Denaturing\/Temperature Gradient Gel Electrophoresis (DGGE\/TGGE): Two other methods rely on detecting changes in electrophoretic mobility in response to minor sequence changes. One of these methods, termed \u201cDenaturing Gradient Gel Electrophoresis\u201d (DGGE) is based on the observation that slightly different sequences will display different patterns of local melting when electrophoretically resolved on a gradient gel. In this manner, variants can be distinguished, as differences in melting properties of homoduplexes versus heteroduplexes differing in a single nucleotide can detect the presence of mutations in the target sequences because of the corresponding changes in their electrophoretic mobilities. The fragments to be analyzed, usually PCR products, are \u201cclamped\u201d at one end by a long stretch of G-C base pairs (30-80) to allow complete denaturation of the sequence of interest without complete dissociation of the strands. The attachment of a GC \u201cclamp\u201d to the DNA fragments increases the fraction of mutations that can be recognized by DGGE. Attaching a GC clamp to one primer is critical to ensure that the amplified sequence has a low dissociation temperature. Modifications of the technique have been developed, using temperature gradients, and the method can be also applied to RNA:RNA duplexes.","Limitations on the utility of DGGE include the requirement that the denaturing conditions must be optimized for each type of DNA to be tested. Furthermore, the method requires specialized equipment to prepare the gels and maintain the needed high temperatures during electrophoresis. The expense associated with the synthesis of the clamping tail on one oligonucleotide for each sequence to be tested is also a major consideration. In addition, long running times are required for DGGE. The long running time of DGGE was shortened in a modification of DGGE called constant denaturant gel electrophoresis (CDGE). CDGE requires that gels be performed under different denaturant conditions in order to reach high efficiency for the detection of mutations.","A technique analogous to DGGE, termed temperature gradient gel electrophoresis (TGGE), uses a thermal gradient rather than a chemical denaturant gradient. TGGE requires the use of specialized equipment which can generate a temperature gradient perpendicularly oriented relative to the electrical field. TGGE can detect mutations in relatively small fragments of DNA therefore scanning of large gene segments requires the use of multiple PCR products prior to running the gel.","Single-Strand Conformation Polymorphism (SSCP): Another common method, called \u201cSingle-Strand Conformation Polymorphism\u201d (SSCP) was developed by Hayashi, Sekya and colleagues and is based on the observation that single strands of nucleic acid can take on characteristic conformations in non-denaturing conditions, and these conformations influence electrophoretic mobility. The complementary strands assume sufficiently different structures that one strand may be resolved from the other. Changes in sequences within the fragment will also change the conformation, consequently altering the mobility and allowing this to be used as an assay for sequence variations.","The SSCP process involves denaturing a DNA segment (e.g., a PCR product) that is labeled on both strands, followed by slow electrophoretic separation on a non-denaturing polyacrylamide gel, so that intra-molecular interactions can form and not be disturbed during the run. This technique is extremely sensitive to variations in gel composition and temperature. A serious limitation of this method is the relative difficulty encountered in comparing data generated in different laboratories, under apparently similar conditions.","Dideoxy fingerprinting (ddF): The dideoxy fingerprinting (ddF) is another technique developed to scan genes for the presence of mutations. The ddF technique combines components of Sanger dideoxy sequencing with SSCP. A dideoxy sequencing reaction is performed using one dideoxy terminator and then the reaction products are electrophoresed on nondenaturing polyacrylamide gels to detect alterations in mobility of the termination segments as in SSCP analysis. While ddF is an improvement over SSCP in terms of increased sensitivity, ddF requires the use of expensive dideoxynucleotides and this technique is still limited to the analysis of fragments of the size suitable for SSCP (i.e., fragments of 200-300 bases for optimal detection of mutations).","In addition to the above limitations, all of these methods are limited as to the size of the nucleic acid fragment that can be analyzed. For the direct sequencing approach, sequences of greater than 600 base pairs require cloning, with the consequent delays and expense of either deletion sub-cloning or primer walking, in order to cover the entire fragment. SSCP and DGGE have even more severe size limitations. Because of reduced sensitivity to sequence changes, these methods are not considered suitable for larger fragments. Although SSCP is reportedly able to detect 90% of single-base substitutions within a 200 base-pair fragment, the detection drops to less than 50% for 400 base pair fragments. Similarly, the sensitivity of DGGE decreases as the length of the fragment reaches 500 base-pairs. The ddF technique, as a combination of direct sequencing and SSCP, is also limited by the relatively small size of the DNA that can be screened.","According to a presently preferred embodiment of the present invention the step of searching for any of the nucleic acid sequences described here, in tumor cells or in cells derived from a cancer patient is effected by any suitable technique, including, but not limited to, nucleic acid sequencing, polymerase chain reaction, ligase chain reaction, self-sustained synthetic reaction, Q\u03b2-Replicase, cycling probe reaction, branched DNA, restriction fragment length polymorphism analysis, mismatch chemical cleavage, heteroduplex analysis, allele-specific oligonucleotides, denaturing gradient gel electrophoresis, constant denaturant gel electrophoresis, temperature gradient gel electrophoresis and dideoxy fingerprinting.","Detection may also optionally be performed with a chip or other such device. The nucleic acid sample which includes the candidate region to be analyzed is preferably isolated, amplified and labeled with a reporter group. This reporter group can be a fluorescent group such as phycoerythrin. The labeled nucleic acid is then incubated with the probes immobilized on the chip using a fluidics station describe the fabrication of fluidics devices and particularly microcapillary devices, in silicon and glass substrates.","Once the reaction is completed, the chip is inserted into a scanner and patterns of hybridization are detected. The hybridization data is collected, as a signal emitted from the reporter groups already incorporated into the nucleic acid, which is now bound to the probes attached to the chip. Since the sequence and position of each probe immobilized on the chip is known, the identity of the nucleic acid hybridized to a given probe can be determined.","It will be appreciated that when utilized along with automated equipment, the above described detection methods can be used to screen multiple samples for a disease and\/or pathological condition both rapidly and easily.","Amino Acid Sequences and Peptides","The terms \u201cpolypeptide,\u201d \u201cpeptide\u201d and \u201cprotein\u201d are used interchangeably herein to refer to a polymer of amino acid residues. The terms apply to amino acid polymers in which one or more amino acid residue is an analog or mimetic of a corresponding naturally occurring amino acid, as well as to naturally occurring amino acid polymers. Polypeptides can be modified, e.g., by the addition of carbohydrate residues to form glycoproteins. The terms \u201cpolypeptide,\u201d \u201cpeptide\u201d and \u201cprotein\u201d include glycoproteins, as well as non-glycoproteins.","Polypeptide products can be biochemically synthesized such as by employing standard solid phase techniques. Such methods include but are not limited to exclusive solid phase synthesis, partial solid phase synthesis methods, fragment condensation, classical solution synthesis. These methods are preferably used when the peptide is relatively short (i.e., 10 kDa) and\/or when it cannot be produced by recombinant techniques (i.e., not encoded by a nucleic acid sequence) and therefore involves different chemistry.","Solid phase polypeptide synthesis procedures are well known in the art and further described by John Morrow Stewart and Janis Dillaha Young, Solid Phase Peptide Syntheses (2nd Ed., Pierce Chemical Company, 1984).","Synthetic polypeptides can optionally be purified by preparative high performance liquid chromatography [Creighton T. (1983) Proteins, structures and molecular principles. WH Freeman and Co. New York], after which their composition can be confirmed via amino acid sequencing.","In cases where large amounts of a polypeptide are desired, it can be generated using recombinant techniques such as described by Bitter et al., (1987) Methods in Enzymol. 153:516-544, Studier et al. (1990) Methods in Enzymol. 185:60-89, Brisson et al. (1984) Nature 310:511-514, Takamatsu et al. (1987) EMBO J. 6:307-311, Coruzzi et al. (1984) EMBO J. 3:1671-1680 and Brogli et al., (1984) Science 224:838-843, Gurley et al. (1986) Mol. Cell. Biol. 6:559-565 and Weissbach & Weissbach, 1988, Methods for Plant Molecular Biology, Academic Press, NY, Section VIII, pp 421-463.","The present invention also encompasses polypeptides encoded by the polynucleotide sequences of the present invention, as well as polypeptides according to the amino acid sequences described herein. The present invention also encompasses homologues of these polypeptides, such homologues can be at least 50%, at least 55%, at least 60%, at least 65%, at least 70%, at least 75%, at least 80%, at least 85%, at least 95% or more say 100% homologous to the amino acid sequences set forth below, as can be determined using BlastP software of the National Center of Biotechnology Information (NCBI) using default parameters, optionally and preferably including the following: filtering on (this option filters repetitive or low-complexity sequences from the query using the Seg (protein) program), scoring matrix is BLOSUM62 for proteins, word size is 3, E value is 10, gap costs are 11, 1 (initialization and extension), and number of alignments shown is 50. Optionally and preferably, nucleic acid sequence homology (identity) is determined using BlastN software of the National Center of Biotechnology Information (NCBI) using default parameters, which preferably include using the DUST filter program, and also preferably include having an E value of 10, filtering low complexity sequences and a word size of 11. Finally, the present invention also encompasses fragments of the above described polypeptides and polypeptides having mutations, such as deletions, insertions or substitutions of one or more amino acids, either naturally occurring or artificially induced, either randomly or in a targeted fashion.","It will be appreciated that peptides identified according the present invention may be degradation products, synthetic peptides or recombinant peptides as well as peptidomimetics, typically, synthetic peptides and peptoids and semipeptoids which are peptide analogs, which may have, for example, modifications rendering the peptides more stable while in a body or more capable of penetrating into cells. Such modifications include, but are not limited to N terminus modification, C terminus modification, peptide bond modification, including, but not limited to, CH2-NH, CH2-S, CH2-S\u2550O, O\u2550C\u2014NH, CH2-O, CH2-CH2, S\u2550C\u2014NH, CH\u2550CH or CF\u2550CH, backbone modifications, and residue modification. Methods for preparing peptidomimetic compounds are well known in the art and are specified. Further details in this respect are provided hereinunder.","Peptide bonds (\u2014CO\u2014NH\u2014) within the peptide may be substituted, for example, by N-methylated bonds (\u2014N(CH3)-CO\u2014), ester bonds (\u2014C(R)H\u2014C\u2014O\u2014O\u2014C(R)\u2014N\u2014), ketomethylen bonds (\u2014CO\u2014CH2-), \u03b1-aza bonds (\u2014NH\u2014N(R)\u2014CO\u2014), wherein R is any alkyl, e.g., methyl, carba bonds (\u2014CH2-NH\u2014), hydroxyethylene bonds (\u2014CH(OH)\u2014CH2-), thioamide bonds (\u2014CS\u2014NH\u2014), olefinic double bonds (\u2014CH\u2550CH\u2014), retro amide bonds (\u2014NH\u2014CO\u2014), peptide derivatives (\u2014N(R)\u2014CH2-CO\u2014), wherein R is the \u201cnormal\u201d side chain, naturally presented on the carbon atom.","These modifications can occur at any of the bonds along the peptide chain and even at several (2-3) at the same time.","Natural aromatic amino acids, Trp, Tyr and Phe, may be substituted for synthetic non-natural acid such as Phenylglycine, TIC, naphthylelanine (Nol), ring-methylated derivatives of Phe, halogenated derivatives of Phe or o-methyl-Tyr.","In addition to the above, the peptides of the present invention may also include one or more modified amino acids or one or more non-amino acid monomers (e.g. fatty acids, complex carbohydrates etc).","As used herein in the specification and in the claims section below the term \u201camino acid\u201d or \u201camino acids\u201d is understood to include the 20 naturally occurring amino acids; those amino acids often modified post-translationally in vivo, including, for example, hydroxyproline, phosphoserine and phosphothreonine; and other unusual amino acids including, but not limited to, 2-aminoadipic acid, hydroxylysine, isodesmosine, nor-valine, nor-leucine and ornithine. Furthermore, the term \u201camino acid\u201d includes both D- and L-amino acids.",{"@attributes":{"id":"p-0649","num":"0693"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"266pt","align":"center"}},"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Table 1 non-conventional or modified amino acids which can be used with the present"},{"entry":"invention."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Non-conventional amino acid","Code","Non-conventional amino acid","Code"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["\u03b1-aminobutyric acid","Abu","L-N-methylalanine","Nmala"]},{"entry":["\u03b1-amino-\u03b1-methylbutyrate","Mgabu","L-N-methylarginine","Nmarg"]},{"entry":["aminocyclopropane-","Cpro","L-N-methylasparagine","Nmasn"]},{"entry":["Carboxylate",{},"L-N-methylaspartic acid","Nmasp"]},{"entry":["aminoisobutyric acid","Aib","L-N-methylcysteine","Nmcys"]},{"entry":["aminonorbornyl-","Norb","L-N-methylglutamine","Nmgin"]},{"entry":["Carboxylate",{},"L-N-methylglutamic acid","Nmglu"]},{"entry":["Cyclohexylalanine","Chexa","L-N-methylhistidine","Nmhis"]},{"entry":["Cyclopentylalanine","Cpen","L-N-methylisolleucine","Nmile"]},{"entry":["D-alanine","Dal","L-N-methylleucine","Nmleu"]},{"entry":["D-arginine","Darg","L-N-methyllysine","Nmlys"]},{"entry":["D-aspartic acid","Dasp","L-N-methylmethionine","Nmmet"]},{"entry":["D-cysteine","Dcys","L-N-methylnorleucine","Nmnle"]},{"entry":["D-glutamine","Dgln","L-N-methylnorvaline","Nmnva"]},{"entry":["D-glutamic acid","Dglu","L-N-methylornithine","Nmorn"]},{"entry":["D-histidine","Dhis","L-N-methylphenylalanine","Nmphe"]},{"entry":["D-isoleucine","Dile","L-N-methylproline","Nmpro"]},{"entry":["D-leucine","Dleu","L-N-methylserine","Nmser"]},{"entry":["D-lysine","Dlys","L-N-methylthreonine","Nmthr"]},{"entry":["D-methionine","Dmet","L-N-methyltryptophan","Nmtrp"]},{"entry":["D-ornithine","Dorn","L-N-methyltyrosine","Nmtyr"]},{"entry":["D-phenylalanine","Dphe","L-N-methylvaline","Nmval"]},{"entry":["D-proline","Dpro","L-N-methylethylglycine","Nmetg"]},{"entry":["D-serine","Dser","L-N-methyl-t-butylglycine","Nmtbug"]},{"entry":["D-threonine","Dthr","L-norleucine","Nle"]},{"entry":["D-tryptophan","Dtrp","L-norvaline","Nva"]},{"entry":["D-tyrosine","Dtyr","\u03b1-methyl-aminoisobutyrate","Maib"]},{"entry":["D-valine","Dval","\u03b1-methyl-\u03b3-aminobutyrate","Mgabu"]},{"entry":["D-\u03b1-methylalanine","Dmala","\u03b1-methylcyclohexylalanine","Mchexa"]},{"entry":["D-\u03b1-methylarginine","Dmarg","\u03b1-methylcyclopentylalanine","Mcpen"]},{"entry":["D-\u03b1-methylasparagine","Dmasn","\u03b1-methyl-\u03b1-napthylalanine","Manap"]},{"entry":["D-\u03b1-methylaspartate","Dmasp","\u03b1-methylpenicillamine","Mpen"]},{"entry":["D-\u03b1-methylcysteine","Dmcys","N-(4-aminobutyl)glycine","Nglu"]},{"entry":["D-\u03b1-methylglutamine","Dmgln","N-(2-aminoethyl)glycine","Naeg"]},{"entry":["D-\u03b1-methylhistidine","Dmhis","N-(3-aminopropyl)glycine","Norn"]},{"entry":["D-\u03b1-methylisoleucine","Dmile","N-amino-\u03b1-methylbutyrate","Nmaabu"]},{"entry":["D-\u03b1-methylleucine","Dmleu","\u03b1-napthylalanine","Anap"]},{"entry":["D-\u03b1-methyllysine","Dmlys","N-benzylglycine","Nphe"]},{"entry":["D-\u03b1-methylmethionine","Dmmet","N-(2-carbamylethyl)glycine","Ngln"]},{"entry":["D-\u03b1-methylornithine","Dmorn","N-(carbamylmethyl)glycine","Nasn"]},{"entry":["D-\u03b1-methylphenylalanine","Dmphe","N-(2-carboxyethyl)glycine","Nglu"]},{"entry":["D-\u03b1-methylproline","Dmpro","N-(carboxymethyl)glycine","Nasp"]},{"entry":["D-\u03b1-methylserine","Dmser","N-cyclobutylglycine","Ncbut"]},{"entry":["D-\u03b1-methylthreonine","Dmthr","N-cycloheptylglycine","Nchep"]},{"entry":["D-\u03b1-methyltryptophan","Dmtrp","N-cyclohexylglycine","Nchex"]},{"entry":["D-\u03b1-methyltyrosine","Dmty","N-cyclodecylglycine","Ncdec"]},{"entry":["D-\u03b1-methylvaline","Dmval","N-cyclododeclglycine","Ncdod"]},{"entry":["D-\u03b1-methylalnine","Dnmala","N-cyclooctylglycine","Ncoct"]},{"entry":["D-\u03b1-methylarginine","Dnmarg","N-cyclopropylglycine","Ncpro"]},{"entry":["D-\u03b1-methylasparagine","Dnmasn","N-cycloundecylglycine","Ncund"]},{"entry":["D-\u03b1-methylasparatate","Dnmasp","N-(2,2-diphenylethyl)glycine","Nbhm"]},{"entry":["D-\u03b1-methylcysteine","Dnmcys","N-(3,3-diphenylpropyl)glycine","Nbhe"]},{"entry":["D-N-methylleucine","Dnmleu","N-(3-indolylyethyl) glycine","Nhtrp"]},{"entry":["D-N-methyllysine","Dnmlys","N-methyl-\u03b3-aminobutyrate","Nmgabu"]},{"entry":["N-methylcyclohexylalanine","Nmchexa","D-N-methylmethionine","Dnmmet"]},{"entry":["D-N-methylornithine","Dnmorn","N-methylcyclopentylalanine","Nmcpen"]},{"entry":["N-methylglycine","Nala","D-N-methylphenylalanine","Dnmphe"]},{"entry":["N-methylaminoisobutyrate","Nmaib","D-N-methylproline","Dnmpro"]},{"entry":["N-(1-methylpropyl)glycine","Nile","D-N-methylserine","Dnmser"]},{"entry":["N-(2-methylpropyl)glycine","Nile","D-N-methylserine","Dnmser"]},{"entry":["N-(2-methylpropyl)glycine","Nleu","D-N-methylthreonine","Dnmthr"]},{"entry":["D-N-methyltryptophan","Dnmtrp","N-(1-methylethyl)glycine","Nva"]},{"entry":["D-N-methyltyrosine","Dnmtyr","N-methyla-napthylalanine","Nmanap"]},{"entry":["D-N-methylvaline","Dnmval","N-methylpenicillamine","Nmpen"]},{"entry":["\u03b3-aminobutyric acid","Gabu","N-(p-hydroxyphenyl)glycine","Nhtyr"]},{"entry":["L-t-butylglycine","Tbug","N-(thiomethyl)glycine","Ncys"]},{"entry":["L-ethylglycine","Etg","Penicillamine","Pen"]},{"entry":["L-homophenylalanine","Hphe","L-\u03b1-methylalanine","Mala"]},{"entry":["L-\u03b1-methylarginine","Marg","L-\u03b1-methylasparagine","Masn"]},{"entry":["L-\u03b1-methylaspartate","Masp","L-\u03b1-methyl-t-butylglycine","Mtbug"]},{"entry":["L-\u03b1-methylcysteine","Mcys","L-methylethylglycine","Metg"]},{"entry":["L-\u03b1-methylglutamine","Mgln","L-\u03b1-methylglutamate","Mglu"]},{"entry":["L-\u03b1-methylhistidine","Mhis","L-\u03b1-methylhomo phenylalanine","Mhphe"]},{"entry":["L-\u03b1-methylisoleucine","Mile","N-(2-methylthioethyl)glycine","Nmet"]},{"entry":["D-N-methylglutamine","Dnmgln","N-(3-guanidinopropyl)glycine","Narg"]},{"entry":["D-N-methylglutamate","Dnmglu","N-(1-hydroxyethyl)glycine","Nthr"]},{"entry":["D-N-methylhistidine","Dnmhis","N-(hydroxyethyl)glycine","Nser"]},{"entry":["D-N-methylisoleucine","Dnmile","N-(imidazolylethyl)glycine","Nhis"]},{"entry":["D-N-methylleucine","Dnmleu","N-(3-indolylyethyl)glycine","Nhtrp"]},{"entry":["D-N-methyllysine","Dnmlys","N-methyl-\u03b3-aminobutyrate","Nmgabu"]},{"entry":["N-methylcyclohexylalanine","Nmchexa","D-N-methylmethionine","Dnmmet"]},{"entry":["D-N-methylornithine","Dnmorn","N-methylcyclopentylalanine","Nmcpen"]},{"entry":["N-methylglycine","Nala","D-N-methylphenylalanine","Dnmphe"]},{"entry":["N-methylaminoisobutyrate","Nmaib","D-N-methylproline","Dnmpro"]},{"entry":["N-(1-methylpropyl)glycine","Nile","D-N-methylserine","Dnmser"]},{"entry":["N-(2-methylpropyl)glycine","Nleu","D-N-methylthreonine","Dnmthr"]},{"entry":["D-N-methyltryptophan","Dnmtrp","N-(1-methylethyl)glycine","Nval"]},{"entry":["D-N-methyltyrosine","Dnmtyr","N-methyla-napthylalanine","Nmanap"]},{"entry":["D-N-methylvaline","Dnmval","N-methylpenicillamine","Nmpen"]},{"entry":["\u03b3-aminobutyric acid","Gabu","N-(p-hydroxyphenyl)glycine","Nhtyr"]},{"entry":["L-t-butylglycine","Tbug","N-(thiomethyl)glycine","Ncys"]},{"entry":["L-ethylglycine","Etg","Penicillamine","Pen"]},{"entry":["L-homophenylalanine","Hphe","L-\u03b1-methylalanine","Mala"]},{"entry":["L-\u03b1-methylarginine","Marg","L-\u03b1-methylasparagine","Masn"]},{"entry":["L-\u03b1-methylaspartate","Masp","L-\u03b1-methyl-t-butylglycine","Mtbug"]},{"entry":["L-\u03b1-methylcysteine","Mcys","L-methylethylglycine","Metg"]},{"entry":["L-\u03b1-methylglutamine","Mgln","L-\u03b1-methylglutamate","Mglu"]},{"entry":["L-\u03b1-methylhistidine","Mhis","L-\u03b1-methylhomophenylalanine","Mhphe"]},{"entry":["L-\u03b1-methylisoleucine","Mile","N-(2-methylthioethyl)glycine","Nmet"]},{"entry":["L-\u03b1-methylleucine","Mleu","L-\u03b1-methyllysine","Mlys"]},{"entry":["L-\u03b1-methylmethionine","Mmet","L-\u03b1-methylnorleucine","Mnle"]},{"entry":["L-\u03b1-methylnorvaline","Mnva","L-\u03b1-methylornithine","Morn"]},{"entry":["L-\u03b1-methylphenylalanine","Mphe","L-\u03b1-methylproline","Mpro"]},{"entry":["L-\u03b1-methylserine","mser","L-\u03b1-methylthreonine","Mthr"]},{"entry":["L-\u03b1-methylvaline","Mtrp","L-\u03b1-methyltyrosine","Mtyr"]},{"entry":["L-\u03b1-methylleucine","Mval Nnbhm","L-N-methylhomophenylalanine","Nmhphe"]},{"entry":["N-(N-(2,2-diphenylethyl)",{},"N-(N-(3,3-diphenylpropyl)"]},{"entry":["carbamylmethyl-glycine","Nnbhm","carbamylmethyl(1)glycine","Nnbhe"]},{"entry":["1-carboxy-1-(2,2-diphenyl","Nmbc"]},{"entry":"ethylamino)cyclopropane"},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},"Since the peptides of the present invention are preferably utilized in diagnostics which require the peptides to be in soluble form, the peptides of the present invention preferably include one or more non-natural or natural polar amino acids, including but not limited to serine and threonine which are capable of increasing peptide solubility due to their hydroxyl-containing side chain.","The peptides of the present invention are preferably utilized in a linear form, although it will be appreciated that in cases where cyclicization does not severely interfere with peptide characteristics, cyclic forms of the peptide can also be utilized.","The peptides of present invention can be biochemically synthesized such as by using standard solid phase techniques. These methods include exclusive solid phase synthesis well known in the art, partial solid phase synthesis methods, fragment condensation, classical solution synthesis. These methods are preferably used when the peptide is relatively short (i.e., 10 kDa) and\/or when it cannot be produced by recombinant techniques (i.e., not encoded by a nucleic acid sequence) and therefore involves different chemistry.","Synthetic peptides can be purified by preparative high performance liquid chromatography and the composition of which can be confirmed via amino acid sequencing.","In cases where large amounts of the peptides of the present invention are desired, the peptides of the present invention can be generated using recombinant techniques such as described by Bitter et al., (1987) Methods in Enzymol. 153:516-544, Studier et al. (1990) Methods in Enzymol. 185:60-89, Brisson et al. (1984) Nature 310:511-514, Takamatsu et al. (1987) EMBO J. 6:307-311, Coruzzi et al. (1984) EMBO J. 3:1671-1680 and Brogli et al., (1984) Science 224:838-843, Gurley et al. (1986) Mol. Cell. Biol. 6:559-565 and Weissbach & Weissbach, 1988, Methods for Plant Molecular Biology, Academic Press, New York, Section VIII, pp 421-463 and also as described above.","Antibodies","\u201cAntibody\u201d refers to a polypeptide ligand that is preferably substantially encoded by an immunoglobulin gene or immunoglobulin genes, or fragments thereof, which specifically binds and recognizes an epitope (e.g., an antigen). The recognized immunoglobulin genes include the kappa and lambda light chain constant region genes, the alpha, gamma, delta, epsilon and mu heavy chain constant region genes, and the myriad-immunoglobulin variable region genes. Antibodies exist, e.g., as intact immunoglobulins or as a number of well characterized fragments produced by digestion with various peptidases. This includes, e.g., Fab\u2032 and F(ab)\u2032fragments. The term \u201cantibody,\u201d as used herein, also includes antibody fragments either produced by the modification of whole antibodies or those synthesized de novo using recombinant DNA methodologies. It also includes polyclonal antibodies, monoclonal antibodies, chimeric antibodies, humanized antibodies, or single chain antibodies. \u201cFc\u201d portion of an antibody refers to that portion of an immunoglobulin heavy chain that comprises one or more heavy chain constant region domains, CH1, CH2 and CH3, but does not include the heavy chain variable region.","The functional fragments of antibodies, such as Fab, F(ab\u2032)2, and Fv that are capable of binding to macrophages, are described as follows: (1) Fab, the fragment which contains a monovalent antigen-binding fragment of an antibody molecule, can be produced by digestion of whole antibody with the enzyme papain to yield an intact light chain and a portion of one heavy chain; (2) Fab\u2032, the fragment of an antibody molecule that can be obtained by treating whole antibody with pepsin, followed by reduction, to yield an intact light chain and a portion of the heavy chain; two Fab\u2032 fragments are obtained per antibody molecule; (3) (Fab\u2032)2, the fragment of the antibody that can be obtained by treating whole antibody with the enzyme pepsin without subsequent reduction; F(ab\u2032)2 is a dimer of two Fab\u2032 fragments held together by two disulfide bonds; (4) Fv, defined as a genetically engineered fragment containing the variable region of the light chain and the variable region of the heavy chain expressed as two chains; and (5) Single chain antibody (\u201cSCA\u201d), a genetically engineered molecule containing the variable region of the light chain and the variable region of the heavy chain, linked by a suitable polypeptide linker as a genetically fused single chain molecule.","Methods of producing polyclonal and monoclonal antibodies as well as fragments thereof are well known in the art (See for example, Harlow and Lane, Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory, New York, 1988, incorporated herein by reference).","Antibody fragments according to the present invention can be prepared by proteolytic hydrolysis of the antibody or by expression in or mammalian cells (e.g. Chinese hamster ovary cell culture or other protein expression systems) of DNA encoding the fragment. Antibody fragments can be obtained by pepsin or papain digestion of whole antibodies by conventional methods. For example, antibody fragments can be produced by enzymatic cleavage of antibodies with pepsin to provide a 5S fragment denoted F(ab\u2032)2. This fragment can be further cleaved using a thiol reducing agent, and optionally a blocking group for the sulfhydryl groups resulting from cleavage of disulfide linkages, to produce 3.5S Fab\u2032 monovalent fragments. Alternatively, an enzymatic cleavage using pepsin produces two monovalent Fab\u2032 fragments and an Fc fragment directly. These methods are described, for example, by Goldenberg, U.S. Pat. Nos. 4,036,945 and 4,331,647, and references contained therein, which patents are hereby incorporated by reference in their entirety. See also Porter, R. R. [Biochem. J. 73: 119-126 (1959)]. Other methods of cleaving antibodies, such as separation of heavy chains to form monovalent light-heavy chain fragments, further cleavage of fragments, or other enzymatic, chemical, or genetic techniques may also be used, so long as the fragments bind to the antigen that is recognized by the intact antibody.","Fv fragments comprise an association of VH and VL chains. This association may be noncovalent, as described in Inbar et al. [Proc. Nat'l Acad. Sci. USA 69:2659-62 (19720]. Alternatively, the variable chains can be linked by an intermolecular disulfide bond or cross-linked by chemicals such as glutaraldehyde. Preferably, the Fv fragments comprise VH and VL chains connected by a peptide linker. These single-chain antigen binding proteins (sFv) are prepared by constructing a structural gene comprising DNA sequences encoding the VH and VL domains connected by an oligonucleotide. The structural gene is inserted into an expression vector, which is subsequently introduced into a host cell such as . The recombinant host cells synthesize a single polypeptide chain with a linker peptide bridging the two V domains. Methods for producing sFvs are described, for example, by [Whitlow and Filpula, Methods 2: 97-105 (1991); Bird et al., Science 242:423-426 (1988); Pack et al., Bio\/Technology 11:1271-77 (1993); and U.S. Pat. No. 4,946,778, which is hereby incorporated by reference in its entirety.","Another form of an antibody fragment is a peptide coding for a single complementarity-determining region (CDR). CDR peptides (\u201cminimal recognition units\u201d) can be obtained by constructing genes encoding the CDR of an antibody of interest. Such genes are prepared, for example, by using the polymerase chain reaction to synthesize the variable region from RNA of antibody-producing cells. See, for example, Larrick and Fry [Methods, 2: 106-10(1991)].","Humanized forms of non-human (e.g., murine) antibodies are chimeric molecules of immunoglobulins, immunoglobulin chains or fragments thereof (such as Fv, Fab, Fab\u2032, F(ab\u2032) or other antigen-binding subsequences of antibodies) which contain minimal sequence derived from non-human immunoglobulin. Humanized antibodies include human immunoglobulins (recipient antibody) in which residues from a complementary determining region (CDR) of the recipient are replaced by residues from a CDR of a non-human species (donor antibody) such as mouse, rat or rabbit having the desired specificity, affinity and capacity. In some instances, Fv framework residues of the human immunoglobulin are replaced by corresponding non-human residues. Humanized antibodies may also comprise residues which are found neither in the recipient antibody nor in the imported CDR or framework sequences. In general, the humanized antibody will comprise substantially all of at least one, and typically two, variable domains, in which all or substantially all of the CDR regions correspond to those of a non-human immunoglobulin and all or substantially all of the FR regions are those of a human immunoglobulin consensus sequence. The humanized antibody optimally also will comprise at least a portion of an immunoglobulin constant region (Fc), typically that of a human immunoglobulin [Jones et al., Nature, 321:522-525 (1986); Riechmann et al., Nature, 332:323-329 (1988); and Presta, Curr. Op. Struct. Biol., 2:593-596 (1992)].","Methods for humanizing non-human antibodies are well known in the art. Generally, a humanized antibody has one or more amino acid residues introduced into it from a source which is non-human. These non-human amino acid residues are often referred to as import residues, which are typically taken from an import variable domain. Humanization can be essentially performed following the method of Winter and co-workers [Jones et al., Nature, 321:522-525 (1986); Riechmann et al., Nature 332:323-327 (1988); Verhoeyen et al., Science, 239:1534-1536 (1988)], by substituting rodent CDRs or CDR sequences for the corresponding sequences of a human antibody. Accordingly, such humanized antibodies are chimeric antibodies (U.S. Pat. No. 4,816,567), wherein substantially less than an intact human variable domain has been substituted by the corresponding sequence from a non-human species. In practice, humanized antibodies are typically human antibodies in which some CDR residues and possibly some FR residues are substituted by residues from analogous sites in rodent antibodies.","Human antibodies can also be produced using various techniques known in the art, including phage display libraries [Hoogenboom and Winter, J. Mol. Biol., 227:381 (1991); Marks et al., J. Mol. Biol., 222:581 (1991)]. The techniques of Cole et al. and Boerner et al. are also available for the preparation of human monoclonal antibodies (Cole et al., Monoclonal Antibodies and Cancer Therapy, Alan R. Liss, p. 77 (1985) and Boerner et al., J. Immunol., 147(1):86-95 (1991)]. Similarly, human antibodies can be made by introduction of human immunoglobulin loci into transgenic animals, e.g., mice in which the endogenous immunoglobulin genes have been partially or completely inactivated. Upon challenge, human antibody production is observed, which closely resembles that seen in humans in all respects, including gene rearrangement, assembly, and antibody repertoire. This approach is described, for example, in U.S. Pat. Nos. 5,545,807; 5,545,806; 5,569,825; 5,625,126; 5,633,425; 5,661,016, and in the following scientific publications: Marks et al., Bio\/Technology 10,: 779-783 (1992); Lonberg et al., Nature 368: 856-859 (1994); Morrison, Nature 368 812-13 (1994); Fishwild et al., Nature Biotechnology 14, 845-51 (1996); Neuberger, Nature Biotechnology 14: 826 (1996); and Lonberg and Huszar, Intern. Rev. Immunol. 13, 65-93 (1995).","Preferably, the antibody of this aspect of the present invention specifically binds at least one epitope of the polypeptide variants of the present invention. As used herein, the term \u201cepitope\u201d refers to any antigenic determinant on an antigen to which the paratope of an antibody binds.","Epitopic determinants usually consist of chemically active surface groupings of molecules such as amino acids or carbohydrate side chains and usually have specific three dimensional structural characteristics, as well as specific charge characteristics.","Optionally, a unique epitope may be created in a variant due to a change in one or more post-translational modifications, including but not limited to glycosylation and\/or phosphorylation, as described below. Such a change may also cause a new epitope to be created, for example through removal of glycosylation at a particular site.","An epitope according to the present invention may also optionally comprise part or all of a unique sequence portion of a variant according to the present invention in combination with at least one other portion of the variant which is not contiguous to the unique sequence portion in the linear polypeptide itself, yet which are able to form an epitope in combination. One or more unique sequence portions may optionally combine with one or more other non-contiguous portions of the variant (including a portion which may have high homology to a portion of the known protein) to form an epitope.","Immunoassays","In another embodiment of the present invention, an immunoassay can be used to qualitatively or quantitatively detect and analyze markers in a sample. This method comprises: providing an antibody that specifically binds to a marker; contacting a sample with the antibody; and detecting the presence of a complex of the antibody bound to the marker in the sample.","To prepare an antibody that specifically binds to a marker, purified protein markers can be used. Antibodies that specifically bind to a protein marker can be prepared using any suitable methods known in the art.","After the antibody is provided, a marker can be detected and\/or quantified using any of a number of well recognized immunological binding assays. Useful assays include, for example, an enzyme immune assay (EIA) such as enzyme-linked immunosorbent assay (ELISA), a radioimmune assay (RIA), a Western blot assay, or a slot blot assay see, e.g., U.S. Pat. Nos. 4,366,241; 4,376,110; 4,517,288; and 4,837,168). Generally, a sample obtained from a subject can be contacted with the antibody that specifically binds the marker.","Optionally, the antibody can be fixed to a solid support to facilitate washing and subsequent isolation of the complex, prior to contacting the antibody with a sample. Examples of solid supports include but are not limited to glass or plastic in the form of, e.g., a microtiter plate, a stick, a bead, or a microbead. Antibodies can also be attached to a solid support.","After incubating the sample with antibodies, the mixture is washed and the antibody-marker complex formed can be detected. This can be accomplished by incubating the washed mixture with a detection reagent. Alternatively, the marker in the sample can be detected using an indirect assay, wherein, for example, a second, labeled antibody is used to detect bound marker-specific antibody, and\/or in a competition or inhibition assay wherein, for example, a monoclonal antibody which binds to a distinct epitope of the marker are incubated simultaneously with the mixture.","Throughout the assays, incubation and\/or washing steps may be required after each combination of reagents. Incubation steps can vary from about 5 seconds to several hours, preferably from about 5 minutes to about 24 hours. However, the incubation time will depend upon the assay format, marker, volume of solution, concentrations and the like. Usually the assays will be carried out at ambient temperature, although they can be conducted over a range of temperatures, such as 10\u00b0 C. to 40\u00b0 C.","The immunoassay can be used to determine a test amount of a marker in a sample from a subject. First, a test amount of a marker in a sample can be detected using the immunoassay methods described above. If a marker is present in the sample, it will form an antibody-marker complex with an antibody that specifically binds the marker under suitable incubation conditions described above. The amount of an antibody-marker complex can optionally be determined by comparing to a standard. As noted above, the test amount of marker need not be measured in absolute units, as long as the unit of measurement can be compared to a control amount and\/or signal.","Preferably used are antibodies which specifically interact with the polypeptides of the present invention and not with wild type proteins or other isoforms thereof, for example. Such antibodies are directed, for example, to the unique sequence portions of the polypeptide variants of the present invention, including but not limited to bridges, heads, tails and insertions described in greater detail below. Preferred embodiments of antibodies according to the present invention are described in greater detail with regard to the section entitled \u201cAntibodies\u201d.","Radio-immunoassay (RIA): In one version, this method involves precipitation of the desired substrate and in the methods detailed hereinbelow, with a specific antibody and radiolabelled antibody binding protein (e.g., protein A labeled with I) immobilized on a precipitable carrier such as agarose beads. The number of counts in the precipitated pellet is proportional to the amount of substrate.","In an alternate version of the RIA, a labeled substrate and an unlabelled antibody binding protein are employed. A sample containing an unknown amount of substrate is added in varying amounts. The decrease in precipitated counts from the labeled substrate is proportional to the amount of substrate in the added sample.","Enzyme linked immunosorbent assay (ELISA): This method involves fixation of a sample (e.g., fixed cells or a proteinaceous solution) containing a protein substrate to a surface such as a well of a microtiter plate. A substrate specific antibody coupled to an enzyme is applied and allowed to bind to the substrate. Presence of the antibody is then detected and quantitated by a colorimetric reaction employing the enzyme coupled to the antibody. Enzymes commonly employed in this method include horseradish peroxidase and alkaline phosphatase. If well calibrated and within the linear range of response, the amount of substrate present in the sample is proportional to the amount of color produced. A substrate standard is generally employed to improve quantitative accuracy.","Western blot: This method involves separation of a substrate from other protein by means of an acrylamide gel followed by transfer of the substrate to a membrane (e.g., nylon or PVDF). Presence of the substrate is then detected by antibodies specific to the substrate, which are in turn detected by antibody binding reagents. Antibody binding reagents may be, for example, protein A, or other antibodies. Antibody binding reagents may be radiolabelled or enzyme linked as described hereinabove. Detection may be by autoradiography, colorimetric reaction or chemiluminescence. This method allows both quantitation of an amount of substrate and determination of its identity by a relative position on the membrane which is indicative of a migration distance in the acrylamide gel during electrophoresis.","Immunohistochemical analysis: This method involves detection of a substrate in situ in fixed cells by substrate specific antibodies. The substrate specific antibodies may be enzyme linked or linked to fluorophores. Detection is by microscopy and subjective evaluation. If enzyme linked antibodies are employed, a colorimetric reaction may be required.","Fluorescence activated cell sorting (FACS): This method involves detection of a substrate in situ in cells by substrate specific antibodies. The substrate specific antibodies are linked to fluorophores. Detection is by means of a cell sorting machine which reads the wavelength of light emitted from each cell as it passes through a light beam. This method may employ two or more antibodies simultaneously.","Radio-Imaging Methods","These methods include but are not limited to, positron emission tomography (PET) single photon emission computed tomography (SPECT). Both of these techniques are non-invasive, and can be used to detect and\/or measure a wide variety of tissue events and\/or functions, such as detecting cancerous cells for example. Unlike PET, SPECT can optionally be used with two labels simultaneously. SPECT has some other advantages as well, for example with regard to cost and the types of labels that can be used. For example, U.S. Pat. No. 6,696,686 describes the use of SPECT for detection of breast cancer, and is hereby incorporated by reference as if fully set forth herein.","Display Libraries","According to still another aspect of the present invention there is provided a display library comprising a plurality of display vehicles (such as phages, viruses or bacteria) each displaying at least 6, at least 7, at least 8, at least 9, at least 10, 10-15, 12-17, 15-20, 15-30 or 20-50 consecutive amino acids derived from the polypeptide sequences of the present invention.","Methods of constructing such display libraries are well known in the art. Such methods are described in, for example, Young A C, et al., \u201cThe three-dimensional structures of a polysaccharide binding antibody to Cryptococcus neoformans and its complex with a peptide from a phage display library: implications for the identification of peptide mimotopes\u201d J Mol Biol 1997 Dec. 12; 274(4):622-34; Giebel L B et al. \u201cScreening of cyclic peptide phage libraries identifies ligands that bind streptavidin with high affinities\u201d Biochemistry 1995 Nov. 28; 34(47):15430-5; Davies E L et al., \u201cSelection of specific phage-display antibodies using libraries derived from chicken immunoglobulin genes\u201d J Immunol Methods 1995 Oct. 12; 186(1):125-35; Jones C R T al. \u201cCurrent trends in molecular recognition and bioseparation\u201d J Chromatogr A 1995 Jul. 14; 707(1):3-22; Deng S J et al. \u201cBasis for selection of improved carbohydrate-binding single-chain antibodies from synthetic gene libraries\u201d Proc Natl Acad Sci USA 1995 May 23; 92(11):4992-6; and Deng S J et al. \u201cSelection of antibody single-chain variable fragments with improved carbohydrate binding by phage display\u201d J Biol Chem 1994 Apr. 1; 269(13):9533-8, which are incorporated herein by reference.","The following sections relate to Candidate Marker Examples (first section) and to Experimental Data for these Marker Examples (second section). It should be noted that Table numbering is restarted within each section.","This section relates to examples of sequences according to the present invention, including illustrative methods of selection thereof.","Description of the Methodology Undertaken to Uncover the Biomolecular Sequences of the Present Invention","Human ESTs and cDNAs were obtained from GenBank versions 136 (Jun. 15, 2003 ftpdot ncbi dot nih dot gov\/genbank\/release dot notes\/gb136 dot release dot notes); NCBI genome assembly of April 2003; RefSeq sequences from June 2003; Genbank version 139 (December 2003); Human Genome from NCBI (Build 34) (from October 2003); and RefSeq sequences from December 2003. With regard to GenBank sequences, the human EST sequences from the EST (GBEST) section and the human mRNA sequences from the primate (GBPRI) section were used; also the human nucleotide RefSeq mRNA sequences were used (see for example dot ncbi dot nlm dot nih dot gov\/Genbank\/GenbankOverview dot html and for a reference to the EST section, see dot ncbi dot nlm dot nih dot gov\/dbEST\/; a general reference to dbEST, the EST database in GenBank, may be found in Boguski et al, Nat Genet. 1993 August; 4(4):332-3; all of which are hereby incorporated by reference as if fully set forth herein).","Novel splice variants were predicted using the LEADS clustering and assembly system as described in Sorek, R., Ast, G. & Graur, D. Alu-containing exons are alternatively spliced. Genome Res 12, 1060-7 (2002); U.S. Pat. No. 6,625,545; and U.S. patent application Ser. No. 10\/426,002, published as US20040101876 on May 27 2004; all of which are hereby incorporated by reference as if fully set forth herein. Briefly, the software cleans the expressed sequences from repeats, vectors and immunoglobulins. It then aligns the expressed sequences to the genome taking alternatively splicing into account and clusters overlapping expressed sequences into \u201cclusters\u201d that represent genes or partial genes.","These were annotated using the GeneCarta (Compugen, Tel-Aviv, Israel) platform. The GeneCarta platform includes a rich pool of annotations, sequence information (particularly of spliced sequences), chromosomal information, alignments, and additional information such as SNPs, gene ontology terms, expression profiles, functional analyses, detailed domain structures, known and predicted proteins and detailed homology reports.","A brief explanation is provided with regard to the method of selecting the candidates. However, it should be noted that this explanation is provided for descriptive purposes only, and is not intended to be limiting in any way. The potential markers were identified by a computational process that was designed to find genes and\/or their splice variants that are specifically expressed in cardiac tissue, as opposed to other types of tissues and also particularly as opposed to muscle tissue, by using databases of expressed sequences. Various parameters related to the information in the EST libraries, determined according to classification by library annotation, were used to assist in locating genes and\/or splice variants thereof that are specifically and\/or differentially expressed in heart tissues. The detailed description of the selection method and of these parameters is presented in Example 1 below.","In order to distinguish between differentially expressed gene products and constitutively expressed genes (i.e., house keeping genes), an algorithm based on an analysis of frequencies was configured. A specific algorithm for identification of transcripts specifically expressed in heart tissue is described hereinbelow.,","EST Analysis","ESTs were taken from the following main sources: libraries contained in Genbank version 136 (Jun. 15, 2003 ftp.ncbi.nih.gov\/genbank\/release.notes\/gb136.release.notes) and Genbank version 139 (December 2003); and from the LifeSeq library of Incyte Corporation (ESTs only; Wilmington, Del., USA). With regard to GenBank sequences, the human EST sequences from the EST (GBEST) section were used.","Library annotation\u2014EST libraries were manually classified according to:\n\n","The following rules were followed:","EST libraries originating from identical biological samples were considered as a single library.","EST libraries which included above-average levels of contamination, such as DNA contamination for example, were eliminated. The presence of such contamination was determined as follows. For each library, the number of unspliced ESTs that are not fully contained within other spliced sequences was counted. If the percentage of such sequences (as compared to all other sequences) was at least 4 standard deviations above the average for all libraries being analyzed, this library was tagged as being contaminated and was eliminated from further consideration in the below analysis (see also Sorek, R. & Safer, H. M. A novel algorithm for computational identification of contaminated EST libraries. Nucleic Acids Res 31, 1067-74 (2003) for further details).","Clusters (genes) having at least five sequences including at least two sequences from the tissue of interest were analyzed. Splice variants were identified by using the LEADS software package as described above.","For detection of heart tissue specific clusters, heart tissue libraries\/sequences were compared to the total number of libraries\/sequences in the cluster and in Genebank, and to the relevant numbers for muscle tissue libraries\/sequences. Statistical tools were employed to identify clusters that were heart tissue specific, both as compared to all other tissues and also in comparison to muscle tissue.","The algorithm\u2014for each tested tissue T and for each tested cluster the following were examined:","1. Each cluster includes at least 2 libraries from the tissue T. At least 3 clones (weighed\u2014as described above) from tissue T in the cluster;","2. The following equation was then used to determine heart tissue-specific expression as compared to expression in all tissue types for a particular cluster:",{"@attributes":{"id":"p-0704","num":"0751"},"maths":{"@attributes":{"id":"MATH-US-00001","num":"00001"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mfrac":[{"mi":["t","T"]},{"mrow":[{"mi":["n","t","m"],"mo":["-","-"]},{"mi":["N","T","M"],"mo":["-","-"]}]}],"mo":"\/"}}},"br":{}},"3. The following equation was then used to determine heart tissue-specific expression vs. expression in skeletal muscle tissue for a particular cluster:",{"@attributes":{"id":"p-0706","num":"0753"},"maths":{"@attributes":{"id":"MATH-US-00002","num":"00002"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mfrac":[{"mi":"t","munder":{"mi":["T","_"]}},{"mi":["m","M"]}],"mo":"\/"}}},"br":{}},"4. Fisher exact test P-values were computed for weighted clone counts to check that the counts are statistically significant according to the following function: F(t,T,n,N) which is the probability of a cluster actually being overexpressed in heart tissue, as compared to its overall level of expression. The P-value was preferably set to be less than about 1e-5, although optionally it could be set to be less than about 1e-3.","The results obtained are explained in greater detail for each marker below.","The following examples relate to specific actual marker examples. It should be noted that Table numbering is restarted within each example related to a particular Cluster, as indicated by the titles below.","This Section relates to Examples of sequences according to the present invention, including experiments involving these sequences, and illustrative, non-limiting examples of methods, assays and uses thereof. The materials and experimental procedures are explained first, as all experiments used them as a basis for the work that was performed.","The markers of the present invention were tested with regard to their expression in various heart and non-heart tissue samples. Unless otherwise noted, all experimental data relates to variants of the present invention, named according to the segment being tested (as expression was tested through RT-PCR as described). A description of the samples used in the panel is provided in Table 2 below. Tests were then performed as described in the Examples below.",{"@attributes":{"id":"p-0712","num":"0759"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"294pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Tissue samples in testing panel"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Lot no.","Source","Tissue","Pathology","Sex\/Age"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"5","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1-Am-Colon (C71)","071P10B","Ambion","Colon","PM","F\/43"]},{"entry":["2-B-Colon (C69)","A411078","Biochain","Colon","PM-Pool of 10","M&F"]},{"entry":["3-Cl-Colon (C70)","1110101","Clontech","Colon","PM-Pool of 3","M&F"]},{"entry":["4-Am-Small Intestine","091P0201A","Ambion","Small Intestine","PM","M\/75"]},{"entry":["5-B-Small Intestine","A501158","Biochain","Small Intestine","PM","M\/63"]},{"entry":["6-B-Rectum","A605138","Biochain","Rectum","PM","M\/25"]},{"entry":["7-B-Rectum","A610297","Biochain","Rectum","PM","M\/24"]},{"entry":["8-B-Rectum","A610298","Biochain","Rectum","PM","M\/27"]},{"entry":["9-Am-Stomach","110P04A","Ambion","Stomach","PM","M\/16"]},{"entry":["10-B-Stomach","A501159","Biochain","Stomach","PM","M\/24"]},{"entry":["11-B-Esophagus","A603814","Biochain","Esophagus","PM","M\/26"]},{"entry":["12-B-Esophagus","A603813","Biochain","Esophagus","PM","M\/41"]},{"entry":["13-Am-Pancreas","071P25C","Ambion","Pancreas","PM","M\/25"]},{"entry":["14-CG-Pancreas","CG-255-2","Ichilov","Pancreas","PM","M\/75"]},{"entry":["15-B-Lung","A409363","Biochain","Lung","PM","F\/26"]},{"entry":["16-Am-Lung (L93)","111P0103A","Ambion","Lung","PM","F\/61"]},{"entry":["17-B-Lung (L92)","A503204","Biochain","Lung","PM","M\/28"]},{"entry":["18-Am-Ovary (O47)","061P43A","Ambion","Ovary","PM","F\/16"]},{"entry":["19-B-Ovary (O48)","A504087","Biochain","Ovary","PM","F\/51"]},{"entry":["20-B-Ovary (O46)","A504086","Biochain","Ovary","PM","F\/41"]},{"entry":["21-Am-Cervix","101P0101A","Ambion","Cervix","PM","F\/40"]},{"entry":["22-B-Cervix","A408211","Biochain","Cervix","PM","F\/36"]},{"entry":["23-B-Cervix","A504089","Biochain","Cervix","PM-Pool of 5","M&F"]},{"entry":["24-B-Uterus","A411074","Biochain","Uterus","PM-Pool of 10","M&F"]},{"entry":["25-B-Uterus","A409248","Biochain","Uterus","PM","F\/43"]},{"entry":["26-B-Uterus","A504090","Biochain","Uterus","PM-Pool of 5","M&F"]},{"entry":["27-B-Bladder","A501157","Biochain","Bladder","PM","M\/29"]},{"entry":["28-Am-Bladder","071P02C","Ambion","Bladder","PM","M\/20"]},{"entry":["29-B-Bladder","A504088","Biochain","Bladder","PM-Pool of 5","M&F"]},{"entry":["30-Am-Placenta","021P33A","Ambion","Placenta","PB","F\/33"]},{"entry":["31-B-Placenta","A410165","Biochain","Placenta","PB","F\/26"]},{"entry":["32-B-Placenta","A411073","Biochain","Placenta","PB-Pool of 5","M&F"]},{"entry":["33-B-Breast (B59)","A607155","Biochain","Breast","PM","F\/36"]},{"entry":["34-Am-Breast (B63)","26486","Ambion","Breast","PM","F\/43"]},{"entry":["35-Am-Breast (B64)","23036","Ambion","Breast","PM","F\/57"]},{"entry":["36-Cl-Prostate (P53)","1070317","Clontech","Prostate","PB-Pool of 47","M&F"]},{"entry":["37-Am-Prostate (P42)","061P04A","Ambion","Prostate","PM","M\/47"]},{"entry":["38-Am-Prostate (P59)","25955","Ambion","Prostate","PM","M\/62"]},{"entry":["39-Am-Testis","111P0104A","Ambion","Testis","PM","M\/25"]},{"entry":["40-B-Testis","A411147","Biochain","Testis","PM","M\/74"]},{"entry":["41-Cl-Testis","1110320","Clontech","Testis","PB-Pool of 45","M&F"]},{"entry":["42-CG-Adrenal","CG-184-10","Ichilov","Adrenal","PM","F\/81"]},{"entry":["43-B-Adrenal","A610374","Biochain","Adrenal","PM","F\/83"]},{"entry":["44-B-Heart","A411077","Biochain","Heart","PB-Pool of 5","M&F"]},{"entry":["45-CG-Heart","CG-255-9","Ichilov","Heart","PM","M\/75"]},{"entry":["46-CG-Heart","CG-227-1","Ichilov","Heart","PM","F\/36"]},{"entry":["47-Am-Liver","081P0101A","Ambion","Liver","PM","M\/64"]},{"entry":["48-CG-Liver","CG-93-3","Ichilov","Liver","PM","F\/19"]},{"entry":["49-CG-Liver","CG-124-4","Ichilov","Liver","PM","F\/34"]},{"entry":["50-Cl-BM","1110932","Clontech","Bone Marrow","PM-Pool of 8","M&F"]},{"entry":["51-CGEN-Blood","WBC#5","CGEN","Blood",{},"M"]},{"entry":["52-CGEN-Blood","WBC#4","CGEN","Blood",{},"M"]},{"entry":["53-CGEN-Blood","WBC#3","CGEN","Blood",{},"M"]},{"entry":["54-CG-Spleen","CG-267","Ichilov","Spleen","PM","F\/25"]},{"entry":["55-CG-Spleen","111P0106B","Ambion","Spleen","PM","M\/25"]},{"entry":["56-CG-Spleen","A409246","Biochain","Spleen","PM","F\/12"]},{"entry":["56-CG-Thymus","CG-98-7","Ichilov","Thymus","PM","F\/28"]},{"entry":["58-Am-Thymus","101P0101A","Ambion","Thymus","PM","M\/14"]},{"entry":["59-B-Thymus","A409278","Biochain","Thymus","PM","M\/28"]},{"entry":["60-B-Thyroid","A610287","Biochain","Thyroid","PM","M\/27"]},{"entry":["61-B-Thyroid","A610286","Biochain","Thyroid","PM","M\/24"]},{"entry":["62-CG-Thyroid","CG-119-2","Ichilov","Thyroid","PM","F\/66"]},{"entry":["63-Cl-Salivary Gland","1070319","Clontech","Salivary Gland","PM-Pool of 24","M&F"]},{"entry":["64-Am-Kidney","111P0101B","Ambion","Kidney","PM-Pool of 14","M&F"]},{"entry":["65-Cl-Kidney","1110970","Clontech","Kidney","PM-Pool of 14","M&F"]},{"entry":["66-B-Kidney","A411080","Biochain","Kidney","PM-Pool of 5","M&F"]},{"entry":["67-CG-Cerebellum","CG-183-5","Ichilov","Cerebellum","PM","M\/74"]},{"entry":["68-CG-Cerebellum","CG-212-5","Ichilov","Cerebellum","PM","M\/54"]},{"entry":["69-B-Brain","A411322","Biochain","Brain","PM","M\/28"]},{"entry":["70-Cl-Brain","1120022","Clontech","Brain","PM-Pool of 2","M&F"]},{"entry":["71-B-Brain","A411079","Biochain","Brain","PM-Pool of 2","M&F"]},{"entry":["72-CG-Brain","CG-151-1","Ichilov","Brain","PM","F\/86"]},{"entry":["73-Am-Skeletal Muscle","101P013A","Ambion","Skeletal Muscle","PM","F\/28"]},{"entry":["74-Cl-Skeletal Muscle","1061038","Clontech","Skeletal Muscle","PM-Pool of 2","M&F"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]}}]}},"br":{}},"RNA preparation\u2014RNA was obtained from Clontech (Franklin Lakes, N.J. USA 07417, dot clontech dot com), BioChain Inst. Inc. (Hayward, Calif. 94545 USA dot biochain dot corn), ABS (Wilmington, Del. 19801, USA, dot absbioreagents dot com) or Ambion (Austin, Tex. 78744 USA,dot ambion dot com). Alternatively, RNA was generated from tissue samples using TRI-Reagent (Molecular Research Center), according to Manufacturer's instructions. Tissue and RNA samples were obtained from patients or from postmortem. Total RNA samples were treated with DNaseI (Ambion) and purified using RNeasy columns (Qiagen).","RT PCR\u2014Purified RNA (1 \u03bcg) was mixed with 150 ng Random Hexamer primers (Invitrogen) and 500 \u03bcM dNTP in a total volume of 15.6 \u03bcl. The mixture was incubated for 5 min at 65\u00b0 C. and then quickly chilled on ice. Thereafter, 5 \u03bcl of 5\u00d7 SuperscriptII first strand buffet (Invitrogen), 2.4 \u03bcl 0.1M DTT and 40 units RNasin (Promega) were added, and the mixture was incubated for 10 min at 25\u00b0 C., followed by further incubation at 42\u00b0 C. for 2 min. Then, 1 \u03bcl (200 units) of SuperscriptII (Invitrogen) was added and the reaction (final volume of 25 \u03bcl) was incubated for 50 min at 42\u00b0 C. and then inactivated at 70\u00b0 C. for 15 min. The resulting cDNA was diluted 1:20 in TE buffer (10 mM Tris pH=8, 1 mM EDTA pH=8).","Real-Time RT-PCR analysis\u2014cDNA (5 \u03bcl), prepared as described above, was used as a template in Real-Time PCR reactions using the SYBR Green I assay (PE Applied Biosystem) with specific primers and UNG Enzyme (Eurogentech or ABI or Roche). The amplification was effected as follows: 50\u00b0 C. for 2 min, 95\u00b0 C. for 10 min, and then 40 cycles of 95\u00b0 C. for 15 sec, followed by 60\u00b0 C. for 1 min. Detection was performed by using the PE Applied Biosystem SDS 7000. The cycle in which the reactions achieved a threshold level (Ct) of fluorescence was registered and was used to calculate the relative transcript quantity in the RT reactions. The relative quantity was calculated using the equation Q=efficiency^. The efficiency of the PCR reaction was calculated from a standard curve, created by using serial dilutions of several reverse transcription (RT) reactions To minimize inherent differences in the RT reaction, the resulting relative quantities were normalized to the geometric mean of the relative quantities of several housekeeping (HSKP) genes. Schematic summary of quantitative real-time PCR analysis is presented in . As shown, the x-axis shows the cycle number. The C=Threshold Cycle point, which is the cycle that the amplification curve crosses the fluorescence threshold that was set in the experiment. This point is a calculated cycle number in which PCR products signal is above the background level (passive dye ROX) and still in the Geometric\/Exponential phase (as shown, once the level of fluorescence crosses the measurement threshold, it has a geometrically increasing phase, during which measurements are most accurate, followed by a linear phase and a plateau phase; for quantitative measurements, the latter two phases do not provide accurate measurements). The y-axis shows the normalized reporter fluorescence. It should be noted that this type of analysis provides relative quantification.","The sequences of the housekeeping genes measured in all the examples on normal tissue samples panel were as follows:","RPL19 (GenBank Accession No. NM00098 (SEQ ID NO:437)), RPL19 Forward primer (SEQ ID NO:438): TGGCAAGAAGAAGGTCTGGTTAG RPL19Reverse primer (SEQ ID NO:439): TGATCAGCCCATCTTTGATGAG RPL19-amplicon (SEQ ID NO:440): TGGCAAGAAGAAGGTCTGGTTAGACCCCAATGAGACCAATGAAATCGCCAATGCCAACTCCCGTCAG CAGATCCGGAAGCTCATCAAAGATGGGCTGATCA","TATA box (GenBank Accession No. NM003194 (SEQ ID NO:441)), TATA box Forward primer (SEQ ID NO:442): CGGTTTGCTGCGGTAATCAT TATA box Reverse primer (SEQ ID NO:443): TTTCTTGCTGCCAGTCTGGAC TATA box-amplicon (SEQ ID NO:444): CGGTTTGCTGCGGTAATCATGAGGATAAGAGAGCCACGAACCACGGCACTGATTTTCAGTTCTGGGA AAATGGTGTGCACAGGAGCCAAGAGTGAAGAACAGTCCAGACTGGCAGCAAGAAA","Ubiquitin (GenBank Accession No. BC000449 (SEQ ID NO:445)) Ubiquitin Forward primer (SEQ ID NO:446): ATTTGGGTCGCGGTTCTTG Ubiquitin Reverse primer (SEQ ID NO:447): TGCCTTGACATTCTCGATGGT Ubiquitin-amplicon (SEQ ID NO:448): ATTTGGGTCGCGGTTCTTGTTTGTGGATCGCTGTGATCGTCACTTGACAATGCAGATCTTCGTGAAGAC TCTGACTGGTAAGACCATCACCCTCGAGGTTGAGCCCAGTGACACCATCGAGAATGTCAAGGCA SDHA (GenBank Accession No. NM004168 (SEQ ID NO:499))SDHA Forward primer (SEQ ID NO:450): TGGGAACAAGAGGGCATCTG SDHA Reverse primer (SEQ ID NO:451): CCACCACTGCATCAAATTCATG","SDHA-amplicon (SEQ ID NO:452) (SEQ ID NO:452): TGGGAACAAGAGGGCATCTGCTAAAGTTTCAGATTCCATTTCTGCTCAGTATCCAGTAGTGGATCATG AATTTGATGCAGTGGTGG","Cluster S67314 features 4 transcript(s) and 8 segment(s) of interest, the names for which are given in Tables 3 and 4, respectively, the sequences themselves are given at the end of the application. The selected protein variants are given in table 5.",{"@attributes":{"id":"p-0722","num":"0769"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Transcripts of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Transcript Name","SEQ ID NO"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"S67314_PEA_1_T4","1"]},{"entry":[{},"S67314_PEA_1_T5","2"]},{"entry":[{},"S67314_PEA_1_T6","3"]},{"entry":[{},"S67314_PEA_1_T7","4"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0723","num":"0770"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 4"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segments of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"98pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment Name","SEQ ID NO"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"S67314_PEA_1_node_0","65"]},{"entry":[{},"S67314_PEA_1_node_11","66"]},{"entry":[{},"S67314_PEA_1_node_13","67"]},{"entry":[{},"S67314_PEA_1_node_15","68"]},{"entry":[{},"S67314_PEA_1_node_17","69"]},{"entry":[{},"S67314_PEA_1_node_4","70"]},{"entry":[{},"S67314_PEA_1_node_10","71"]},{"entry":[{},"S67314_PEA_1_node_3","72"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0724","num":"0771"},"tables":{"@attributes":{"id":"TABLE-US-00006","num":"00006"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 5"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Proteins of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Protein Name","SEQ ID NO"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"S67314_PEA_1_P4","281"]},{"entry":[{},"S67314_PEA_1_P5","282"]},{"entry":[{},"S67314_PEA_1_P6","283"]},{"entry":[{},"S67314_PEA_1_P7","284"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"These sequences are variants of the known protein Fatty acid-binding protein, heart (SEQ ID NO:348) (SwissProt accession identifier FABH_HUMAN; known also according to the synonyms H-FABP; Muscle fatty acid-binding protein; M-FABP; Mammary-derived growth inhibitor; MDGI), referred to herein as the previously known protein.","Protein Fatty acid-binding protein, heart (SEQ ID NO:348) is known or believed to have the following function(s): FABP are thought to play a role in the intracellular transport of long-chain fatty acids and their acyl-CoA esters. The sequence for protein Fatty acid-binding protein, heart is given at the end of the application, as \u201cFatty acid-binding protein, heart amino acid sequence\u201d (SEQ ID NO:348). Known polymorphisms for this sequence are as shown in Table 6.",{"@attributes":{"id":"p-0727","num":"0774"},"tables":{"@attributes":{"id":"TABLE-US-00007","num":"00007"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 6"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations for Known Protein"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"154pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino",{}]},{"entry":["acid sequence","Comment"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"154pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1","V -> A"]},{"entry":["104","L -> K"]},{"entry":["124","C -> S"]},{"entry":["129","E -> Q"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"Protein Fatty acid-binding protein, heart (SEQ ID NO:348) localization is believed to be Cytoplasmic.","The following GO Annotation(s) apply to the previously known protein. The following annotation(s) were found: negative control of cell proliferation, which are annotation(s) related to Biological Process; and lipid binding, which are annotation(s) related to Molecular Function.","The GO assignment relies on information from one or more of the SwissProt\/TremBl Protein knowledgebase, available from <dot expasy dot ch\/sprot\/>; or Locuslink, available from <dot ncbi dot nlm dot nih dot gov\/projects\/LocusLink\/>.","The heart-selective diagnostic marker prediction engine provided the following results with regard to cluster S67314. Predictions were made for selective expression of transcripts of this cluster in heart tissue, according to the previously described methods. The numbers on the y-axis of  refer to weighted expression of ESTs in each category, as \u201cparts per million\u201d (ratio of the expression of ESTs for a particular cluster to the expression of all ESTs in that category, according to parts per million).","Overall, the following results were obtained as shown with regard to the histogram in , concerning the number of heart-specific clones in libraries\/sequences; as well as with regard to the histogram in , concerning the actual expression of oligonucleotides in various tissues, including heart.","This cluster was found to be selectively expressed in heart for the following reasons: in a comparison of the ratio of expression of the cluster in heart specific ESTs to the overall expression of the cluster in non-heart ESTs, which was found to be 13.8; the ratio of expression of the cluster in heart specific ESTs to the overall expression of the cluster in muscle-specific ESTs which was found to be 2.6; and fisher exact test P-values were computed both for library and weighted clone counts to check that the counts are statistically significant, and were found to be 1.10E-25.","One particularly important measure of specificity of expression of a cluster in heart tissue is the previously described comparison of the ratio of expression of the cluster in heart as opposed to muscle. This cluster was found to be specifically expressed in heart as opposed to non-heart ESTs as described above. However, many proteins have been shown to be generally expressed at a higher level in both heart and muscle, which is less desirable. For this cluster, as described above, the ratio of expression of the cluster in heart specific ESTs to the overall expression of the cluster in muscle-specific ESTs which was found to be 2.6, which clearly supports specific expression in heart tissue.","As noted above, cluster S67314 features 4 transcript(s), which were listed in Table 3 above. These transcript(s) encode for protein(s) which are variant(s) of protein Fatty acid-binding protein, heart (SEQ ID NO:348). A description of each variant protein according to the present invention is now provided.","Variant protein S67314_PEA1_P4 (SEQ ID NO:281) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) S67314_PEA1_T4 (SEQ ID NO:1). An alignment is given to the known protein (Fatty acid-binding protein, heart (SEQ ID NO:348)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between S67314_PEA1_P4 (SEQ ID NO:281) and FABH_HUMAN (SEQ ID NO:348):","1. An isolated chimeric polypeptide encoding for S67314_PEA1_P4 (SEQ ID NO:281), comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence MVDAFLGTWKLVDSKNFDDYMKSLGVGFATRQVASMTKPTTIIEKNGDILTLKTHSTFKNTEISFKLGVEF DETTADDRKVKSIVTLDGGKLVHLQKWDGQETTLVRELIDGKLIL corresponding to amino acids 1-116 of FABH_HUMAN (SEQ ID NO:348), which also corresponds to amino acids 1-116 of S67314_PEA1_P4 (SEQ ID NO:281), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence VRWATLELYLIGYYYCSFSQACSKKPSPPLRAVEAGTREWLWVRVVSGGNFLCSGFGLTQAGTQILPYRL HDCGQITFSKCNCKTGINNTNLVGLLGSL (SEQ ID NO:396) corresponding to amino acids 117-215 of S67314_PEA1_P4 (SEQ ID NO:281), wherein said firstand second amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of S67314_PEA1_P4 (SEQ ID NO:281), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VRWATLELYLIGYYYCSFSQACSKKPSPPLRAVEAGTREWLWVRVVSGGNFLCSGFLTQAGTQILPYRL HDCGQITFSKCNCKTGINNTNLVGLLGSL (SEQ ID NO:396) in S673141_P4 (SEQ ID NO:281).","Comparison report between S67314_PEA1_P4 (SEQ ID NO:281) and AAP35373 (SEQ ID NO:348):","1. An isolated chimeric polypeptide encoding for S67314_PEA1_P4 (SEQ ID NO:281), comprising a first amino acid sequence being at least 90% homologous to MVDAFLGTWKLVDSKNFDDYMKSLGVGFATRQVASMTKPTTIIEKNGDILTLKTHSTFKNTEISFKLGVEF DETTADDRKVKSIVTLDGGKLVHLQKWDGQETTLVRELIDGKLIL corresponding to amino acids 1-116 of AAP35373 (SEQ ID NO:348), which also corresponds to amino acids 1-116 of S67314_PEA1_P4 (SEQ ID NO:281), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence VRWATLELYLIGYYYCSFSQACSKKPSPPLRAVEAGTREWLWVRVVSGGNFLCSGFGLTQAGTQILPYRL HDCGQITFSKCNCKTGINNTNLVGLLGSL (SEQ ID NO:396) corresponding to amino acids 117-215 of S67314_PEA1_P4 (SEQ ID NO:281), wherein said first and second amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of S67314_PEA1_P4 (SEQ ID NO:281), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VRWATLELYLIGYYYCSFSQACSKKPSPPLRAVEAGTREWLWVRVVSGGNFLCSGFGLTQAGTQILPYRL HDCGQITFSKCNCKTGINNTNLVGLLGSL (SEQ ID NO:396) in S67314_PEA1_P4 (SEQ ID NO:281).","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: intracellularly. The protein localization is believed to be intracellular because neither of the trans-membrane region prediction programs predicted a trans-membrane region for this protein. In addition both signal-peptide prediction programs predict that this protein is a non-secreted protein.","Variant protein S67314_PEA1_P4 (SEQ ID NO:281) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 7, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein S67314_PEA1_P4 (SEQ ID NO:281) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0745","num":"0792"},"tables":{"@attributes":{"id":"TABLE-US-00008","num":"00008"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 7"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino acid","Alternative",{}]},{"entry":["sequence","amino acid(s)","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["53","K -> R","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein S67314_PEA1_P4 (SEQ ID NO:281) is encoded by the following transcript(s): S67314_PEA1_T4 (SEQ ID NO:1), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript S67314_PEA1_T4 (SEQ ID NO:1) is shown in bold; this coding portion starts at position 925 and ends at position 1569. The transcript also has the following SNPs as listed in Table 8 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein S67314_PEA1_P4 (SEQ ID NO:281) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0747","num":"0794"},"tables":{"@attributes":{"id":"TABLE-US-00009","num":"00009"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 8"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on nucleotide","Alternative",{}]},{"entry":["sequence","nucleic acid","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["580","T -> C","Yes"]},{"entry":["1082","A -> G","Yes"]},{"entry":["1670","A -> C","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein S67314_PEA1_P5 (SEQ ID NO:282) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) S67314_PEA1_T5 (SEQ ID NO:2). An alignment is given to the, known protein (Fatty acid-binding protein, heart (SEQ ID NO:348)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between S67314_PEA1_P5 (SEQ ID NO:282) and FABH_HUMAN (SEQ ID NO:348):","1. An isolated chimeric polypeptide encoding for S67314_PEA1_P5 (SEQ ID NO:282), comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence MVDAFLGTWKLVDSKNFDDYMKSLGVGFATRQVASMTKPTTIIEKNGDILTLKTHSTFKNTEISFKLGVEF DETTADDRKVKSIVTLDGGKLVHLQKWDGQETTLVRELIDGKLIL corresponding to amino acids 1-116 of FABH_HUMAN (SEQ ID NO:348), which also corresponds to amino acids 1-116 of S67314_PEA1_P5 (SEQ ID NO:282), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence DVLTAWPSIYRRQVKVLREDEITILPWHLQWSREKATKLLRPTLPSYNNHGWEELRVGKSIV (SEQ ID NO:397) corresponding to amino acids 117-178 of S67314_PEA1_P5 (SEQ ID NO:282), wherein said first and second amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of S67314_PEA1_P5 (SEQ ID NO:282), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence DVLTAWPSIYRRQVKVLREDEITILPWHLQWSREKATKLLRPTLPSYNNHGWEELRVGKSIV (SEQ ID NO:397) S67314_PEA1_P5 (SEQ ID NO:282).","Comparison report between S67314_PEA1_P5 (SEQ ID NO:282) and AAP35373 (SEQ ID NO:348):","1. An isolated chimeric polypeptide encoding for S67314_PEA1_P5 (SEQ ID NO:282), comprising a first amino acid sequence being at least 90% homologous to MVDAFLGTWKLVDSKNFDDYMKSLGVGFATRQVASMTKPTTIIEKNGDILTLKTHSTFKNTEISFKLGVEF DETTADDRKVKSIVTLDGGKLVHLQKWDGQETTLVRELIDGKLIL corresponding to amino acids 1-116 of AAP35373 (SEQ ID NO:348), which also corresponds to amino acids 1-116 of S67314_PEA1_P5 (SEQ ID NO:282), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence DVLTAWPSIYRRQVKVLREDEITILPWHLQWSREKATKLLRPTLPSYNNHGWEELRVGKSIV (SEQ ID NO:397) corresponding to amino acids 117-178 of S67314_PEA1_P5 (SEQ ID NO:282), wherein said first and second amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of S67314_PEA1_P5 (SEQ ID NO:282), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence DVLTAWPSIYRRQVKVLREDEITILPWHLQWSREKATKLLRPTLPSYNNHGWEELRVGKSIV (SEQ ID NO:397) in S67314_PEA1_P5 (SEQ ID NO:282).","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: intracellularly. The protein localization is believed to be intracellular because neither of the trans-membrane region prediction programs predicted a trans-membrane region for this protein. In addition both signal-peptide prediction programs predict that this protein is a non-secreted protein.","Variant protein S67314_PEA1_P5 (SEQ ID NO:282) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 9, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein S67314_PEA1_P5 (SEQ ID NO:282) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0757","num":"0804"},"tables":{"@attributes":{"id":"TABLE-US-00010","num":"00010"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 9"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino acid","Alternative",{}]},{"entry":["sequence","amino acid(s)","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["53","K -> R","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein S67314_PEA1_P5 (SEQ ID NO:282) is encoded by the following transcript(s):","S67314_PEA1_T5 (SEQ ID NO:2), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript S67314_PEA1_T5 (SEQ ID NO:2) is shown in bold; this coding portion starts at position 925 and ends at position 1458. The transcript also has the following SNPs as listed in Table 10 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein S67314_PEA1_P5 (SEQ ID NO:282) sequence provides support for the: deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0760","num":"0807"},"tables":{"@attributes":{"id":"TABLE-US-00011","num":"00011"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 10"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on nucleotide","Alternative",{}]},{"entry":["sequence","nucleic acid","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["580","T -> C","Yes"]},{"entry":["1082","A -> G","Yes"]},{"entry":["1326","A -> G","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein S67314_PEA1_P6 (SEQ ID NO:283) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) S67314_PEA1_T6 (SEQ ID NO:3). An alignment is given to the known protein (Fatty acid-binding protein, heart (SEQ ID NO:348)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between S67314_PEA1_P6 (SEQ ID NO:283) and FABH_HUMAN (SEQ ID NO:348):","1. An isolated chimeric polypeptide encoding for S67314_PEA1_P6 (SEQ ID NO:283), comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence MVDAFLGTWKLVDSKNFDDYMKSLGVGFATRQVASMTKPTTIIEKNGDILTLKTHSTFKNTEISFKLGVEF DETTADDRKVKSIVTLDGGKLVHLQKWDGQETTLVRELIDGKLIL corresponding to amino acids 1-116 of FABH_HUMAN (SEQ ID NO:348), which also corresponds to amino acids 1-116 of S67314_PEA1_P6 (SEQ ID NO:283), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence MEKLQLRNVK (SEQ ID NO:398) corresponding to amino acids 117-126 of S67314_PEA1_P6 (SEQ ID NO:283), wherein said first and second amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of S67314_PEA1_P6 (SEQ ID NO:283), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence MEKLQLRNVK (SEQ ID NO:398) in S67314_PEA1_P6 (SEQ ID NO:283).","Comparison report between S67314_PEA1_P6 (SEQ ID NO:283) and AAP35373 (SEQ ID NO:348):","1. An isolated chimeric polypeptide encoding for S67314_PEA1_P6 (SEQ ID NO:283), comprising a first amino acid sequence being at least 90% homologous to MVDAFLGTWKLVDSKNFDDYMKSLGVGFATRQVASMTKPTTIIEKNGDILTLKTHSTFKNTEISFKLGVEF DETTADDRKVKSIVTLDGGKLVHLQKWDGQETTLVRELIDGKLIL corresponding to amino acids 1-116 of AAP35373 (SEQ ID NO:348), which also corresponds to amino acids 1-116 of S67314_PEA1_P6 (SEQ ID NO:283), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence MEKLQLRNVK (SEQ ID NO:398) corresponding to amino acids 117-126 of S67314_PEA1_P6 (SEQ ID NO:283), wherein said first and second amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of S67314_PEA1_P6 (SEQ ID NO:283), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence MEKLQLRNVK (SEQ ID NO:398) in S67314_PEA1_P6 (SEQ ID NO:283).","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: intracellularly. The protein localization is believed to be intracellular because neither of the trans-membrane region prediction programs predicted a trans-membrane region for this protein. In addition both signal-peptide prediction programs predict that this protein is a non-secreted protein.","Variant protein S67314_PEA1_P6 (SEQ ID NO:283) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 11, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein S67314_PEA1_P6 (SEQ ID NO:283) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0770","num":"0817"},"tables":{"@attributes":{"id":"TABLE-US-00012","num":"00012"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 11"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino acid","Alternative",{}]},{"entry":["sequence","amino acid(s)","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["53","K -> R","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein S67314_PEA1_P6 (SEQ ID NO:283) is encoded by the following transcript(s): S67314_PEA1_T6 (SEQ ID NO:3), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript S67314_PEA1_T6 (SEQ ID NO:3) is shown in bold; this coding portion starts at position 925 and ends at position 1302. The transcript also has the following SNPs as listed in Table 12 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein S67314_PEA1_P6 (SEQ ID NO:283) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0772","num":"0819"},"tables":{"@attributes":{"id":"TABLE-US-00013","num":"00013"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 12"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on nucleotide","Alternative",{}]},{"entry":["sequence","nucleic acid","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["580","T -> C","Yes"]},{"entry":["1082","A -> G","Yes"]},{"entry":["1444","T -> C","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein S67314_PEA1_P7 (SEQ ID NO:284) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) S67314_PEA1_T7 (SEQ ID NO:4). An alignment is given to the known protein (Fatty acid-binding protein, heart (SEQ ID NO:348)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between S67314_PEA1_P7 (SEQ ID NO:284) and FABH_HUMAN (SEQ ID NO:348):","1. An isolated chimeric polypeptide encoding for S67314_PEA1_P7 (SEQ ID NO:284), comprising a first amino acid sequence being at least 90% homologous to MVDAFLGTWKLVDSKNFDDYMKSL corresponding to amino acids 1-24 of FABH_HUMAN (SEQ ID NO:348), which also corresponds to amino acids 1-24 of S67314_PEA1_P7 (SEQ ID NO:284), second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence AHILITFPLPS (SEQ ID NO:399) corresponding to amino acids 25-35 of S67314_PEA1_P7 (SEQ ID NO:284), and a third amino acid sequence being at least 90% homologous to GVGFATRQVASMTKPTTIIEKNGDILTLKTHSTFKNTEISFKLGVEFDETTADDRKVKSIVTLDGGKLVHLQ KWDGQETTLVRELIDGKLILTLTHGTAVCTRTYEKEA corresponding to amino acids 25-133 of FABH_HUMAN (SEQ ID NO:348), which also corresponds to amino acids 36-144 of S67314_PEA1_P7 (SEQ ID NO:284), wherein said first, second, third and fourth amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for an edge portion of S67314_PEA1_P7 (SEQ ID NO:284), comprising an amino acid sequence being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence encoding for AHILITFPLPS (SEQ ID NO:399), corresponding to S67314_PEA1_P7 (SEQ ID NO:284).","Comparison report between S67314_PEA_l_P7 (SEQ ID NO:284) and AAP35373 (SEQ ID NO:348):","1. An isolated chimeric polypeptide encoding for S67314_PEA1_P7 (SEQ ID NO:284), comprising a first amino acid sequence being at least 90% homologous to MVDAFLGTWKLVDSKNFDDYMKSL corresponding to amino acids 1-24 of AAP35373 (SEQ ID NO:348), which also corresponds to amino acids 1-24 of S67314_PEA1_P7 (SEQ ID NO:284), second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence AHILITFPLPS (SEQ ID NO:399) corresponding to amino acids 25-35 of S67314_PEA1_P7 (SEQ ID NO:284), and a third amino acid sequence being at least 90% homologous to GVGFATRQVASMTKPTTIIEKNGDILTLKTHSTFKNTEISFKLGVEFDETTADDRKVKSIVTLDGGKLVHLQ KWDGQETTLVRELIDGKLILTLTHGTAVCTRTYEKEA corresponding to amino acids 25-133 of AAP35373 (SEQ ID NO:348), which also corresponds to amino acids 36-144 of S67314_PEA1_P7 (SEQ ID NO:284), wherein said first, second and third amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for an edge portion of S67314_PEA1_P7 (SEQ ID NO:284), comprising an amino acid sequence being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence encoding for AHILITFPLPS (SEQ ID NO:399), corresponding to S67314_PEA1_P7 (SEQ ID NO:284).","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: intracellularly. The protein localization is believed to be intracellular because neither of the trans-membrane region prediction programs predicted a trans-membrane region for this protein. In addition both signal-peptide prediction programs predict that this protein is a non-secreted protein.","Variant protein S67314_PEA1_P7 (SEQ ID NO:284) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 13, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein S67314_PEA1_P7 (SEQ ID NO:284) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0782","num":"0829"},"tables":{"@attributes":{"id":"TABLE-US-00014","num":"00014"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 13"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino acid","Alternative",{}]},{"entry":["sequence","amino acid(s)","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["64","K -> R","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein S67314_PEA1_P7 (SEQ ID NO:284) is encoded by the following transcript(s): S67314_PEA1_T7 (SEQ ID NO:4), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript S67314_PEA1_T7 (SEQ ID NO:4) is shown in bold; this coding portion starts at position 925 and ends at position 1356. The transcript also has the following SNPs as listed in Table 14 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein S67314_PEA1_P7 (SEQ ID NO:284) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0784","num":"0831"},"tables":{"@attributes":{"id":"TABLE-US-00015","num":"00015"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 14"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on nucleotide","Alternative",{}]},{"entry":["sequence","nucleic acid","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["580","T -> C","Yes"]},{"entry":["1115","A -> G","Yes"]},{"entry":["2772","G -> A","Yes"]},{"entry":["2896","C -> A","Yes"]},{"entry":["2918","G -> C","Yes"]},{"entry":["3003","A -> G","Yes"]},{"entry":["3074","T -> G","Yes"]},{"entry":["1344","T -> C","Yes"]},{"entry":["1522","-> T","No"]},{"entry":["1540","-> A","No"]},{"entry":["1540","-> T","No"]},{"entry":["1578","G -> A","Yes"]},{"entry":["1652","G -> A","Yes"]},{"entry":["2263","G -> A","Yes"]},{"entry":["2605","T -> C","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"As noted above, cluster S67314 features 8 segment(s), which were listed in Table 4 above and for which the sequence(s) are given at the end of the application. These segment(s) are portions of nucleic acid sequence(s) which are described herein separately because they are of particular interest. A description of each segment according to the present invention is now provided.","Segment cluster S67314_PEA1_node0 (SEQ ID NO:65) according to the present invention is supported by 90 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): S67314_PEA1_T4 (SEQ ID NO:1), S67314_PEA1_T5 (SEQ ID NO:2), S67314_PEA1_T6 (SEQ ID NO:3) and S67314_PEA1_T7 (SEQ ID NO:4). Table 15 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0787","num":"0834"},"tables":{"@attributes":{"id":"TABLE-US-00016","num":"00016"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 15"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Segment"]},{"entry":[{},"Segment","ending"]},{"entry":["Transcript name","starting position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["S67314_PEA_1_T4 (SEQ ID NO: 1)","1","997"]},{"entry":["S67314_PEA_1_T5 (SEQ ID NO: 2)","1","997"]},{"entry":["S67314_PEA_1_T6 (SEQ ID NO: 3)","1","997"]},{"entry":["S67314_PEA_1_T7 (SEQ ID NO: 4)","1","997"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster S67314_PEA1_node11 (SEQ ID NO:66) according to the present invention is supported by 6 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): S67314_PEA1_T4 (SEQ ID NO:1). Table 16 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0789","num":"0836"},"tables":{"@attributes":{"id":"TABLE-US-00017","num":"00017"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 16"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Segment"]},{"entry":[{},"Segment","ending"]},{"entry":["Transcript name","starting position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["S67314_PEA_1_T4 (SEQ ID NO: 1)","1273","2110"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster S67314_PEA1_node13 (SEQ ID NO:67) according to the present invention is supported by 76 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): S67314_PEA1_T7 (SEQ ID NO:4). Table 17 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0791","num":"0838"},"tables":{"@attributes":{"id":"TABLE-US-00018","num":"00018"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 17"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Segment"]},{"entry":[{},"Segment","ending"]},{"entry":["Transcript name","starting position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["S67314_PEA_1_T7 (SEQ ID NO: 4)","1306","3531"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster S67314_PEA1_node15 (SEQ ID NO:68) according to the present invention is supported by 1 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): S67314_PEA1_T5 (SEQ ID NO:2). Table 18 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0793","num":"0840"},"tables":{"@attributes":{"id":"TABLE-US-00019","num":"00019"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 18"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Segment"]},{"entry":[{},"Segment","ending"]},{"entry":["Transcript name","starting position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["S67314_PEA_1_T5 (SEQ ID NO: 2)","1273","1733"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster S67314_PEA1_node17 (SEQ ID NO:69) according to the present invention is supported by 4 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): S67314_PEA1_T6 (SEQ ID NO:3). Table  below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0795","num":"0842"},"tables":{"@attributes":{"id":"TABLE-US-00020","num":"00020"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 19"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment",{}]},{"entry":[{},"starting","Segment"]},{"entry":["Transcript name","position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["S67314_PEA_1_T6 (SEQ ID NO: 3)","1273","1822"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster S67314_PEA1_node4 (SEQ ID NO:70) according to the present invention is supported by 101 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): S67314_PEA1_T4 (SEQ ID NO:1), S67314_PEA1_T5 (SEQ ID NO:2), S67314_PEA1_T6 (SEQ ID NO:3) and S67314_PEA1_T7 (SEQ ID NO:4). Table 20 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0797","num":"0844"},"tables":{"@attributes":{"id":"TABLE-US-00021","num":"00021"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 20"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment",{}]},{"entry":[{},"starting","Segment"]},{"entry":["Transcript name","position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["S67314_PEA_1_T4 (SEQ ID NO: 1)","998","1170"]},{"entry":["S67314_PEA_1_T5 (SEQ ID NO: 2)","998","1170"]},{"entry":["S67314_PEA_1_T6 (SEQ ID NO: 3)","998","1170"]},{"entry":["S67314_PEA_1_T7 (SEQ ID NO: 4)","1031","1203"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"According to an optional embodiment of the present invention, short segments related to the above cluster are also provided. These segments are up to about 120 bp in length, and so are included in a separate description.","Segment cluster S67314_PEA1_node10 (SEQ ID NO:71) according to the present invention is supported by 64 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): S67314_PEA1_T4 (SEQ ID NO:1), S67314_PEA1_T5 (SEQ ID NO:2), S67314_PEA1_T6 (SEQ ID NO:3) and S67314_PEA1_T7 (SEQ ID NO:4). Table 21 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0800","num":"0847"},"tables":{"@attributes":{"id":"TABLE-US-00022","num":"00022"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 21"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment",{}]},{"entry":[{},"starting","Segment"]},{"entry":["Transcript name","position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["S67314_PEA_1_T4 (SEQ ID NO: 1)","1171","1272"]},{"entry":["S67314_PEA_1_T5 (SEQ ID NO: 2)","1171","1272"]},{"entry":["S67314_PEA_1_T6 (SEQ ID NO: 3)","1171","1272"]},{"entry":["S67314_PEA_1_T7 (SEQ ID NO: 4)","1204","1305"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster S67314_PEA1_node3 (SEQ ID NO:72) according to the present invention is supported by 1 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): S67314_PEA1_T7 (SEQ ID NO:4). Table 22 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0802","num":"0849"},"tables":{"@attributes":{"id":"TABLE-US-00023","num":"00023"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 22"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Segment"]},{"entry":[{},"Segment","ending"]},{"entry":["Transcript name","starting position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["S67314_PEA_1_T7 (SEQ ID NO: 4)","998","1030"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}},"br":{}},{"@attributes":{"id":"p-0803","num":"0850"},"tables":{"@attributes":{"id":"TABLE-US-00024","num":"00024"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"280pt","align":"left"}},"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Sequence name: \/tmp\/EQOnMn6tqU\/R73CUVKUk5:FABH_HUMAN (SEQ ID NO: 348)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: S67314_PEA_1_P4 (SEQ ID NO: 281) \u00d7 FABH_HUMAN (SEQ ID NO: 348)"},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","1095.00","Escore:","0"]},{"entry":["Matching length:","115","Total length:","115"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"280pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00001","num":"00001"},"img":{"@attributes":{"id":"EMI-C00001","he":"31.75mm","wi":"84.92mm","file":"US07714100-20100511-C00001.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/EQ0nMn6tqU\/R73CUVKUk5:AAP35373 (SEQ ID NO: 348)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: S67314_PEA_1_P4 (SEQ ID NO: 281) \u00d7 AAP35373 (SEQ ID NO: 348)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","1107.00","Escore:","0"]},{"entry":["Matching length:","116","Total length:","116"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"280pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00002","num":"00002"},"img":{"@attributes":{"id":"EMI-C00002","he":"31.75mm","wi":"84.92mm","file":"US07714100-20100511-C00002.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/q14YPIBbdQ\/SeofJfCmJw:FABH_HUMAN (SEQ ID NO: 348)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: S67314_PEA_1_P5 (SEQ ID NO: 282) \u00d7 FABH_HUMAN (SEQ ID NO: 348)"},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","1095.00","Escore:","0"]},{"entry":["Matching length:","115","Total length:","115"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"280pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00003","num":"00003"},"img":{"@attributes":{"id":"EMI-C00003","he":"31.75mm","wi":"84.92mm","file":"US07714100-20100511-C00003.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/q14YPIBbdQ\/SeofJfCmJW:AAP35373 (SEQ ID NO: 348)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: S67314_PEA_P5 (SEQ ID NO: 282) \u00d7 AAP35373 (SEQ ID NO: 348)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","1107.00","Escore:","0"]},{"entry":["Matching length:","116","Total length:","116"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"280pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00004","num":"00004"},"img":{"@attributes":{"id":"EMI-C00004","he":"31.75mm","wi":"84.92mm","file":"US07714100-20100511-C00004.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/PXra2DxL1v\/Q8GTrzNMVX:FABH_HUMAN (SEQ ID NO: 348)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: S67314_PEA_1_P6 (SEQ ID NO: 283) \u00d7 FABH_HUMAN (SEQ ID NO: 348)"},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","1095.00","Escore:","0"]},{"entry":["Matching length:","115","Total length:","115"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"280pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00005","num":"00005"},"img":{"@attributes":{"id":"EMI-C00005","he":"31.75mm","wi":"84.92mm","file":"US07714100-20100511-C00005.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/PXra2DxL1v\/Q8GTrzNMVX:AAP35373 (SEQ ID NO: 348)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: S67314_PEA_1_P6 (SEQ ID NO: 283) \u00d7 AAP35373 (SEQ ID NO: 348)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","1107.00","Escore:","0"]},{"entry":["Matching length:","116","Total length:","116"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"280pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00006","num":"00006"},"img":{"@attributes":{"id":"EMI-C00006","he":"31.75mm","wi":"84.92mm","file":"US07714100-20100511-C00006.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/xYzWyViDom\/twDu3T69pd:FABH_HUMAN (SEQ ID NO: 348)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: S67314_PEA_1_P7 (SEQ ID NO: 284) \u00d7 FABH_HUMAN (SEQ ID NO: 348)"},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","1160.00","Escore:","0"]},{"entry":["Matching length:","132","Total length:","143"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","92.31","Total Percent Identity:","92.31"]},{"entry":["Gaps:","1"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"280pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00007","num":"00007"},"img":{"@attributes":{"id":"EMI-C00007","he":"31.75mm","wi":"84.92mm","file":"US07714100-20100511-C00007.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/xYzWyViDom\/twDu3T69pd:AAP35373 (SEQ ID NO: 348)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: S67314_PEA_1_P7 (SEQ ID NO: 284) \u00d7 AAP35373 (SEQ ID NO: 348)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","1172.00","Escore:","0"]},{"entry":["Matching length:","133","Total length:","144"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","92.36","Total Percent Identity:","92.36"]},{"entry":["Gaps:","1"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"280pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00008","num":"00008"},"img":{"@attributes":{"id":"EMI-C00008","he":"31.75mm","wi":"84.92mm","file":"US07714100-20100511-C00008.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}]}}},"Expression of FABH_HUMAN Fatty acid-binding protein transcripts detectable by or according to seg11, S67314 amplicon(s) and S67314 seg11F (SEQ ID NO:61) and S67314 seg11R (SEQ ID NO:62) primers was measured by real time PCR. In parallel the expression of four housekeeping genes\u2014RPL19 (GenBank Accession No. NM000981 (SEQ ID NO:437); RPL19 amplicon (SEQ ID NO:440)), TATA box (GenBank Accession No. NM003194 (SEQ ID NO:441); TATA amplicon (SEQ ID NO:444)), Ubiquitin (GenBank Accession No. BC000449 (SEQ ID NO:445); amplicon\u2014Ubiquitin-amplicon (SEQ ID NO:448)) and SDHA (GenBank Accession No. NM004168 (SEQ ID NO:449); amplicon\u2014SDHA-amplicon (SEQ ID NO:452)) was measured similarly. For each RT sample, the expression of the above amplicons was normalized to the geometric mean of the quantities of the housekeeping genes. The normalized quantity of each RT sample was then divided by the median of the quantities of the heart samples (Sample Nos. 44, 45, 46, Table 2, \u201cTissue samples in testing panel\u201d, above), to obtain a value of fold up-regulation for each sample relative to median of the heart.",{"@attributes":{"id":"p-0805","num":"0852"},"figref":"FIG. 5A"},"As is evident from , the expression of FABH_HUMAN Fatty acid-binding protein transcripts detectable by the above amplicon(s) in heart tissue samples was significantly higher than in most other samples (non heart tissue sample Nos. 1-11,13-21,23-26,28-43, 47-74, Table 2 above, \u201cTissue samples in testing panel\u201d).","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: S67314 seg11F (SEQ ID NO:61) forward primer; and S67314 seg11R (SEQ ID NO:62) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: S67314 seg11 (SEQ ID NO:63).",{"@attributes":{"id":"p-0809","num":"0856"},"tables":{"@attributes":{"id":"TABLE-US-00025","num":"00025"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"287pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["S67314 seg11F (SEQ ID NO: 61):",{}]},{"entry":"TCCCCTGAGAGCTGTAGAAGCT"},{"entry":{}},{"entry":"S67314 seg11R (SEQ ID NO: 62):"},{"entry":"CGGCCTGTGTGAGTCCAAA"},{"entry":{}},{"entry":"S67314 seg11 (SEQ ID NO: 63):"},{"entry":"TCCCCTGAGAGCTGTAGAAGCTGGGACAAGAGAGTGGTTGTGGGTCAGGGTGGTATCAGGTGGGAAT"},{"entry":{}},{"entry":"TTTCTGTGTAGTGGCTTTGGACTCACACAGGCCG"}]}}}}},"Expression of FABH_HUMAN Fatty acid-binding protein transcripts detectable by or according to seg15 node(s), S67314 seg15 amplicon(s) and S67314 seg15F and S67314 seg15R primers was measured by real time PCR. In parallel the expression of four housekeeping genes\u2014RPL19 (GenBank Accession No. NM000981 (SEQ ID NO:437); RPL19 amplicon (SEQ ID NO:440)), TATA box (GenBank Accession No. NM003194 (SEQ ID NO:441); TATA amplicon (SEQ ID NO:444)), Ubiquitin (GenBank Accession No. BC000449 (SEQ ID NO:445); amplicon\u2014Ubiquitin-amplicon (SEQ ID NO:448)) and SDHA (GenBank Accession No. NM004168 (SEQ ID NO:449); amplicon\u2014SDHA-amplicon (SEQ ID NO:452)), was measured similarly. For each RT sample, the expression of the above amplicons was normalized to the geometric mean of the quantities of the housekeeping genes. The normalized quantity of each RT sample was then divided by the median of the quantities of the heart samples (Sample Nos. 44-46, Table 2, above \u201cTissue samples in testing panel\u201d), to obtain a value of fold up-regulation for each sample relative to median of the heart.",{"@attributes":{"id":"p-0811","num":"0858"},"figref":"FIG. 5B"},"As is evident from , the expression of FABH_HUMAN Fatty acid-binding protein transcripts detectable by the above amplicon(s) in heart tissue samples was significantly higher than in most other samples (non-heart tissue sample Nos. 1-9, 11-21, 23-26, 28-43, 47-74 Table 2 above, \u201cTissue samples in testing panel\u201d).","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: S67314 seg15F Forward primer (SEQ ID NO:64); and S67314 seg15R Reverse primer (SEQ ID NO:274).","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: S67314 seg15.",{"@attributes":{"id":"p-0815","num":"0862"},"tables":{"@attributes":{"id":"TABLE-US-00026","num":"00026"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"294pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"147pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["S67314 seg15F (SEQ ID NO: 64) Forward primer:",{}]},{"entry":"TTCCTTGGCATCTCCAATGG"},{"entry":{}},{"entry":"S67314 seg15R (SEQ ID NO: 274) Reverse primer:"},{"entry":"GCCAACTCTCAGCTCCTCCC"},{"entry":{}},{"entry":"S67314 seg15 (SEQ ID NO: 275) Amplicon:"},{"entry":"TTCCTTGGCATCTCCAATGGAGTAGAGAGAAGGCAACAAAGCTTCTCAGACCCACATRACCGAGCTAT"},{"entry":{}},{"entry":"AACAACCATGGCTGGGAGGAGCTGAGAGTTGGC"}]}}}}},"Expression of FABH_HUMAN Fatty acid-binding protein transcripts detectable by or according to seg4 node(s), S67314 seg4 amplicon(s) and primers S67314seg4F and S67314seg4R was measured by real time PCR (this transcript corresponds to the known or WT protein). In parallel the expression of four housekeeping genes\u2014RPL19 (GenBank Accession No. NM000981 (SEQ ID NO:437); RPL19 amplicon (SEQ ID NO:440)), TATA box (GenBank Accession No. NM003194 (SEQ ID NO:441); TATA amplicon (SEQ ID NO:444)), Ubiquitin (GenBank Accession No. BC000449 (SEQ ID NO:445); amplicon\u2014Ubiquitin-amplicon (SEQ ID NO:448)) and SDHA (GenBank Accession No. NM004168 (SEQ ID NO:449); amplicon\u2014SDHA-amplicon (SEQ ID NO:452)), was measured similarly. For each RT sample, the expression of the above amplicons was normalized to the geometric mean of the quantities of the housekeeping genes. The normalized quantity of each RT sample was then divided by the median of the quantities of the heart samples (Sample Nos. 44-46, Table 2, above), to obtain a value of relative expression for each sample relative to median of the heart samples.",{"@attributes":{"id":"p-0817","num":"0864"},"figref":"FIG. 6"},"As is evident from , the expression of FABH_HUMAN Fatty acid-binding protein transcripts detectable by the above amplicon(s) in heart tissue samples was significantly higher than in the other samples (Sample Nos. 44-46 Table 2, \u201cTissue samples in testing panel\u201d).","Primer pairs are also optioanally and preferably ecompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: S67314seg4F forward primer (SEQ ID NO:276); and S67314seg4R reverse primer (SEQ ID NO:277).","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: S67314seg4.",{"@attributes":{"id":"p-0821","num":"0868"},"tables":{"@attributes":{"id":"TABLE-US-00027","num":"00027"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"287pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Forward primer 567314seg4F (SEQ ID NO: 276):",{}]},{"entry":"CCAAGCCTACCACAATCATCG"},{"entry":{}},{"entry":"Reverse primer S67314seg4R (SEQ ID NO: 277):"},{"entry":"CTCCACCCCCAACTTAAAGCT"},{"entry":{}},{"entry":"Amplicon S67314seg4 (SEQ ID NO: 278):"},{"entry":"CCAAGCCTACCACAATCATCGAAAAGAATGGGGACATTCTCACCCTAAAAACACACAGCACCTTCAA"},{"entry":{}},{"entry":"GAACACAGAGATCAGCTTTAAGTTGGGGGTGGAG"}]}}}}},"Cluster N56180 features 7 transcript(s) and 22 segment(s) of interest, the names for which are given in Tables 23 and 24, respectively, the sequences themselves are given at the end of the application. The selected protein variants are given in table 25.",{"@attributes":{"id":"p-0823","num":"0870"},"tables":{"@attributes":{"id":"TABLE-US-00028","num":"00028"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 23"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Transcripts of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"126pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Transcript Name","Seq ID No."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"126pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"N56180_T1","5"]},{"entry":[{},"N56180_T3","6"]},{"entry":[{},"N56180_T4","7"]},{"entry":[{},"N56180_T5","8"]},{"entry":[{},"N56180_T6","9"]},{"entry":[{},"N56180_T7","10"]},{"entry":[{},"N56180_T8","11"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0824","num":"0871"},"tables":{"@attributes":{"id":"TABLE-US-00029","num":"00029"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 24"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segments of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment Name","Seq ID No."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"N56180_node_2","73"]},{"entry":[{},"N56180_node_20","74"]},{"entry":[{},"N56180_node_22","75"]},{"entry":[{},"N56180_node_28","76"]},{"entry":[{},"N56180_node_34","77"]},{"entry":[{},"N56180_node_36","78"]},{"entry":[{},"N56180_node_4","79"]},{"entry":[{},"N56180_node_6","80"]},{"entry":[{},"N56180_node_0","81"]},{"entry":[{},"N56180_node_10","82"]},{"entry":[{},"N56180_node_12","83"]},{"entry":[{},"N56180_node_14","84"]},{"entry":[{},"N56180_node_16","85"]},{"entry":[{},"N56180_node_18","86"]},{"entry":[{},"N56180_node_24","87"]},{"entry":[{},"N56180_node_26","88"]},{"entry":[{},"N56180_node_29","89"]},{"entry":[{},"N56180_node_3","90"]},{"entry":[{},"N56180_node_31","91"]},{"entry":[{},"N56180_node_33","92"]},{"entry":[{},"N56180_node_35","93"]},{"entry":[{},"N56180_node_8","94"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0825","num":"0872"},"tables":{"@attributes":{"id":"TABLE-US-00030","num":"00030"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 25"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Proteins of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"126pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Protein Name","Seq ID No."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"N56180_P2","285"]},{"entry":[{},"N56180_P4","286"]},{"entry":[{},"N56180_P5","287"]},{"entry":[{},"N56180_P6","288"]},{"entry":[{},"N56180_P7","289"]},{"entry":[{},"N56180_P8","290"]},{"entry":[{},"N56180_P9","291"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"These sequences are variants of the known protein Calsequestrin, cardiac muscle isoform precursor (SEQ ID NO:349) (SwissProt accession identifier CAQ2_HUMAN; known also according to the synonyms Calsequestrin 2), referred to herein as the previously known protein.","Protein Calsequestrin, cardiac muscle isoform precursor (SEQ ID NO:349) is known or believed to have the following function(s): Calsequestrin is a high-capacity, moderate affinity, calcium-binding protein and thus acts as an internal calcium store in muscle. The release of calcium bound to calsequestrin through a calcium release channel triggers muscle contraction. The protein binds 40 to 50 moles of calcium. The sequence for protein Calsequestrin, cardiac muscle isoform precursor is given at the end of the application, as \u201cCalsequestrin, cardiac muscle isoform precursor amino acid sequence\u201d (SEQ ID NO:349). Known polymorphisms for this sequence are as shown in Table 26.",{"@attributes":{"id":"p-0828","num":"0875"},"tables":{"@attributes":{"id":"TABLE-US-00031","num":"00031"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 26"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations for Known Protein"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino",{}]},{"entry":["acid sequence","Comment"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["307","D -> H (in VTSIP). \/FTId = VAR_016075."]},{"entry":["67","Q -> P"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"Protein Calsequestrin, cardiac muscle isoform precursor (SEQ ID NO:349) localization is believed to be in the sarcoplasmic reticulum's terminal cisternae luminal spaces of cardiac and slow skeletal muscle cells.","The following GO Annotation(s) apply to the previously known protein. The following annotation(s) were found: striated muscle contraction; heart development; muscle development, which are annotation(s) related to Biological Process; calcium storage, which are annotation(s) related to Molecular Function; and smooth endoplasmic reticulum, which are annotation(s) related to Cellular Component.","The GO assignment relies on information from one or more of the SwissProt\/TremBl Protein knowledgebase, available from <dot expasy dot ch\/sprot\/>; or Locuslink, available from <dot ncbi dot nlm dot nih dot gov\/projects\/LocusLink\/>.","The heart-selective diagnostic marker prediction engine provided the following results with regard to cluster N56180. Predictions were made for selective expression of transcripts of this cluster in heart tissue, according to the previously described methods. The numbers on the y-axis of  refer to weighted expression of ESTs in each category, as \u201cparts per million\u201d (ratio of the expression of ESTs for a particular cluster to the expression of all ESTs in that category, according to parts per million).","Overall, the following results were obtained as shown with regard to the histogram in , concerning the number of heart-specific clones in libraries\/sequences; as well as with regard to the histogram in , concerning the actual expression of oligonucleotides in various tissues, including heart.","This cluster was found to be selectively expressed in heart for the following reasons: in a comparison of the ratio of expression of the cluster in heart specific ESTs to the overall expression of the cluster in non-heart ESTs was found to be 11; the ratio of expression of the cluster in heart specific ESTs to the overall expression of the cluster in muscle-specific ESTs was found to be 2.4; and fisher exact test P-values were computed both for library and weighted clone counts to check that the counts are statistically significant, and were found to be 4.70E-14.","One particularly important measure of specificity of expression of a cluster in heart tissue is the previously described comparison of the ratio of expression of the cluster in heart as opposed to muscle. This cluster was found to be specifically expressed in heart as opposed to non-heart ESTs as described above. However, many proteins have been shown to be generally expressed at a higher level in both heart and muscle, which is less desirable. For this cluster, as described above, the ratio of expression of the cluster in heart specific ESTs to the overall expression of the cluster in muscle-specific ESTs was found to be 2.4, which clearly supports specific expression in heart tissue.","As noted above, cluster N56180 features 7 transcript(s), which were listed in Table 23 above. These transcript(s) encode for protein(s) which are variant(s) of protein Calsequestin, cardiac muscle isoform precursor (SEQ ID NO:349). A description of each variant protein according to the present invention is now provided.","Variant protein N56180_P2 (SEQ ID NO:285) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) N56180_T1 (SEQ ID NO:5). An alignment is given to the known protein (Calsequestrin, cardiac muscle isoform precursor (SEQ ID NO:349)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between N56180_P2 (SEQ ID NO:285) and CAQ2_HUMAN (SEQ ID NO:349):","1. An isolated chimeric polypeptide encoding for N56180_P2 (SEQ ID NO:285), comprising a first amino acid sequence being at least 90% homologous to MKRTHLFIVGIYFLSSCRAEEGLNFPTYDGKDRVVSLSEKNFKQVLKKYDLLCLYYHEPVSSDKVTQKQF QLKEIVLELVAQVLEHKAIGFVMVDAKKEAKLAKKLGFDEEGSLYILKGDRTIEFDGEFAADVLVEFLLDL IEDPVEIISSKLEVQAFERIEDYIKLIGFFKSEDSEYYKAFEEAAEHFQPYIKFFATFDKGV corresponding to amino acids 1-203 of CAQ2_HUMAN (SEQ ID NO:349), which also corresponds to amino acids 1-203 of N56180_P2 (SEQ ID NO:285), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence LWLTPVIPTLWEADGGGLHEPWSWRPAWATWLQRNYL (SEQ ID NO:400) corresponding to amino acids 204-240 of N56180_P2 (SEQ ID NO:), wherein said first and second amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of N56180_P2 (SEQ ID NO:285), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence LWLTPVIPTLWEADGGGLHEPWSWRPAWATWLQRNYL (SEQ ID NO:400) in N56180_P2 (SEQ ID NO:285).","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: secreted or localized in the sarcoplasmic reticulum's terminal cistemae luminal spaces of cardiac and slow skeletal muscle cells like the WT protein. The protein localization is believed to be secreted because both signal-peptide prediction programs predict that this protein has a signal peptide, and neither trans-membrane region prediction program predicts that this protein has a trans-membrane region.","Variant protein N56180_P2 (SEQ ID NO:285) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 27, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein N56180_P2 (SEQ ID NO:285) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0843","num":"0890"},"tables":{"@attributes":{"id":"TABLE-US-00032","num":"00032"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 27"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino acid",{},"Previously"]},{"entry":["sequence","Alternative amino acid(s)","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["66","T -> A","Yes"]},{"entry":["76","V -> M","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein N56180_P2 (SEQ ID NO:285) is encoded by the following transcript(s): N56180_T1 (SEQ ID NO:5), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript N56180_T1 (SEQ ID NO:5) is shown in bold; this coding portion starts at position 242 and ends at position 961. The transcript also has the following SNPs as listed in Table 28 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein N56180_P2 (SEQ ID NO:285) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0845","num":"0892"},"tables":{"@attributes":{"id":"TABLE-US-00033","num":"00033"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 28"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on nucleotide",{},"Previously"]},{"entry":["sequence","Alternative nucleic acid","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["74","T ->","No"]},{"entry":["105","T -> C","Yes"]},{"entry":["2168","C -> G","Yes"]},{"entry":["2289","G -> T","No"]},{"entry":["2489","A -> C","No"]},{"entry":["2545","A ->","Yes"]},{"entry":["2638","A -> T","Yes"]},{"entry":["206","G -> A","Yes"]},{"entry":["221","G -> A","Yes"]},{"entry":["228","A -> C","Yes"]},{"entry":["437","A -> G","Yes"]},{"entry":["467","G -> A","Yes"]},{"entry":["1021","A ->","No"]},{"entry":["1521","C -> T","Yes"]},{"entry":["2018","C -> T","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein N56180_P4 (SEQ ID NO:286) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) N56180_T3 (SEQ ID NO:6). An alignment is given to the known protein (Calsequestrin, cardiac muscle isoform precursor (SEQ ID NO:349)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between N56180_P4 (SEQ ID NO:286) and CAQ2_HUMAN (SEQ ID NO:349):","1. An isolated chimeric polypeptide encoding for N56180_P4 (SEQ ID NO:286), comprising a first amino acid sequence being at least 90% homologous to MKRTHLFIVGIYFLSSCRAEEGLNFPTYDGKDRVVSLSEKNFKQVLKKYDLLCLYYHEPVSSDKVTQKQF QLKEIVLE corresponding to amino acids 1-78 of CAQ2_HUMAN (SEQ ID NO:349), which also corresponds to amino acids 1-78 of N56180_P4 (SEQ ID NO:286), second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence HWQISQWWLHFQTPREEGKMKLLELSESADGAAWKRWGGNSNTHRIQ (SEQ ID NO:401) corresponding to amino acids 79-125 of N56180_P4 (SEQ ID NO:286), and a third amino acid sequence being at least 90% homologous to LVAQVLEHKAIGFVMVDAKKEAKLAKKLGFDEEGSLYILKGDRTIEFDGEFAADVLVEFLLDLIEDPVEIIS SKLEVQAFERIEDYIKLIGFFKSEDSEYYKAFEEAAEHFQPYIKFFATFDKGVAKKLSLKMNEVDFYEPFMD EPIAIPNKPYTEEELVEFVKEHQRPTLRRLRPEEMFETWEDDLNGIHIVAFAEKSDPDGYEFLEILKQVARD NTDNPDLSILWIDPDDFPLLVAYWEKTFKIDLFRPQIGVVNVTDADSVWMEIPDDDDLPTAEELEDWIEDV LSGKINTEDDDEDDDDDDNSDEEDNDDSDDDDDE corresponding to amino acids 79-399 of CAQ2_HUMAN (SEQ ID NO:349), which also corresponds to amino acids 126-446 of N56180_P4 (SEQ ID NO:286), wherein said first, second and third amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for an edge portion of N56180_P4 (SEQ ID NO:286), comprising an amino acid sequence being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence encoding for HWQISQWWLHFQTPREEGKMKLLELSESADGAAWKRWGGNSNTHRIQ (SEQ ID NO:401), corresponding to N56180_P4 (SEQ ID NO:286).","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: secreted or localized in the sarcoplasmic reticulum's terminal cisternae luminal spaces of cardiac and slow skeletal muscle cells like the WT protein. The protein localization is believed to be secreted because both signal-peptide prediction programs predict that this protein has a signal peptide, and neither trans-membrane region prediction program predicts that this protein has a trans-membrane region.","Variant protein N56180_P4 (SEQ ID NO:286) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 29, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein N56180_P4 (SEQ ID NO:286) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0852","num":"0899"},"tables":{"@attributes":{"id":"TABLE-US-00034","num":"00034"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 29"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino acid",{},"Previously"]},{"entry":["sequence","Alternative amino acid(s)","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["115","W -> R","Yes"]},{"entry":["276","N ->","No"]},{"entry":["66","T -> A","Yes"]},{"entry":["76","V -> M","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein N56180_P4 (SEQ ID NO:286) is encoded by the following transcript(s): N56180_T3 (SEQ ID NO:6), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript N56180_T3 (SEQ ID NO:6) is shown in bold; this coding portion starts at position 242 and ends at position 1579. The transcript also has the following SNPs as listed in Table 30 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein N56180_P4 (SEQ ID NO:286) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0854","num":"0901"},"tables":{"@attributes":{"id":"TABLE-US-00035","num":"00035"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 30"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on nucleotide",{},"Previously"]},{"entry":["sequence","Alternative nucleic acid","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["74","T ->","No"]},{"entry":["105","T -> C","Yes"]},{"entry":["2064","C -> T","Yes"]},{"entry":["2214","C -> G","Yes"]},{"entry":["2335","G -> T","No"]},{"entry":["2535","A -> C","No"]},{"entry":["2591","A ->","Yes"]},{"entry":["2684","A -> T","Yes"]},{"entry":["206","G -> A","Yes"]},{"entry":["221","G -> A","Yes"]},{"entry":["228","A -> C","Yes"]},{"entry":["437","A -> G","Yes"]},{"entry":["467","G -> A","Yes"]},{"entry":["584","T -> C","Yes"]},{"entry":["1067","A ->","No"]},{"entry":["1567","C -> T","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein N56180_P5 (SEQ ID NO:287) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcipt(s) N56180_T4 (SEQ ID NO:7). An alignment is given to the known protein (Calsequestrin, cardiac muscle isoform precursor (SEQ ID NO:349)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between N56180_P5 (SEQ ID NO:287) and CAQ2_HUMAN (SEQ ID NO:349):","1. An isolated chimeric polypeptide encoding for N56180_P5 (SEQ ID NO:287), comprising a first amino acid sequence being at least 90% homologous to MKRTHLFIVGIYFLSSCRAEEGLNFPTYDGKDRVVSLSEKNFKQVLKKYDLLCLYYHEPVSSDKVTQKQF QLKEIVLELVAQVLEHKAIGFVMVDAKKEAKLAKKLGFDEEGSLYILKGDRTIEFDGEFAADVLVEFLLD corresponding to amino acids 1-140 of CAQ2_HUMAN (SEQ ID NO:349), which also corresponds to amino acids 1-140 of N56180_P5 (SEQ ID NO:287), and a second amino acid sequence being at least 90% homologous to VAKKLSLKMNEVDFYEPFMDEPIAIPNKPYTEEELVEFVKEHQRPTLRRLRPEEMFETWEDDLNGIHIVAF AEKSDPDGYEFLEILKQVARDNTDNPDLSILWIDPDDFPLLVAYWEKTFKIDLFRPQIGVVNVTDADSVW MEIPDDDDLPTAEELEDWIEDVLSGKINTEDDDEDDDDDDNSDEEDNDDSDDDDDE corresponding to amino acids 203-399 of CAQ2_HUMAN (SEQ ID NO:349), which also corresponds to amino acids 141-337 of N56180_P5 (SEQ ID NO:287), wherein said first and second amino acid sequences are contiguous and in a sequential order.","2. An isolated chimeric polypeptide encoding for an edge portion of N56180_P5 (SEQ ID NO:287), comprising a polypeptide having a length \u201cn\u201d, wherein \u201cn\u201d is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least two amino acids comprise DV, having a structure as follows: a sequence starting from any of amino acid numbers 140\u2212x to 140; and ending at any of amino acid numbers 141+((n\u22122)\u2212x), in which x varies from 0 to n\u22122.","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: secreted or localized in the sarcoplasmic reticulum's terminal cisternae luminal spaces of cardiac and slow skeletal muscle cells like the WT protein. The protein localization is believed to be secreted because both signal-peptide prediction programs predict that this protein has a signal peptide, and neither trans-membrane region prediction program predicts that this protein has a trans-membrane region.","Variant protein N56180_P5 (SEQ ID NO:287) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 31, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein N56180_P5 (SEQ ID NO:287) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0861","num":"0908"},"tables":{"@attributes":{"id":"TABLE-US-00036","num":"00036"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 31"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino acid",{},"Previously"]},{"entry":["sequence","Alternative amino acid(s)","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["167","N ->","No"]},{"entry":["66","T -> A","Yes"]},{"entry":["76","V -> M","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein N56180_P5 (SEQ ID NO:287) is encoded by the following transcript(s): N56180_T4 (SEQ ID NO:7), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript N56180_T4 (SEQ ID NO:7) is shown in bold; this coding portion starts at position 242 and ends at position 1252. The transcript also has the following SNPs as listed in Table 32 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein N56180_P5 (SEQ ID NO:287) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0863","num":"0910"},"tables":{"@attributes":{"id":"TABLE-US-00037","num":"00037"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 32"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on nucleotide",{},"Previously"]},{"entry":["sequence","Alternative nucleic acid","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["74","T ->","No"]},{"entry":["105","T -> C","Yes"]},{"entry":["1887","C -> G","Yes"]},{"entry":["2008","G -> T","No"]},{"entry":["2208","A -> C","No"]},{"entry":["2264","A ->","Yes"]},{"entry":["2357","A -> T","Yes"]},{"entry":["206","G -> A","Yes"]},{"entry":["221","G -> A","Yes"]},{"entry":["228","A -> C","Yes"]},{"entry":["437","A -> G","Yes"]},{"entry":["467","G -> A","Yes"]},{"entry":["740","A ->","No"]},{"entry":["1240","C -> T","Yes"]},{"entry":["1737","C -> T","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein N56180_P6 (SEQ ID NO:288) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) N56180_T5 (SEQ ID NO:8). An alignment is given to the known protein (Calsequestin, cardiac muscle isoform precursor (SEQ ID NO:349)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between N56180_P6 (SEQ ID NO:288) and CAQ2_HUMAN (SEQ ID NO:349):","1. An isolated chimeric polypeptide encoding for N56180_P6 (SEQ ID NO:288), comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence NETEAEQSYV (SEQ ID NO:402) corresponding to amino acids 1-10 of N56180_P6 (SEQ ID NO:288), second amino acid sequence being at least 90% homologous to RAEEGLNFPTYDGKDRVVSLSEKNFKQVLKKYDLLCLYYHEPVSSDKVTQKQFQLKEIVLELVAQVLEHK AIGFVMVDAKKEAKLAKKL corresponding to amino acids 18-106 of CAQ2_HUMAN (SEQ ID NO:349), which also corresponds to amino acids 11-99 of N56180_P6 (SEQ ID NO:288), a third (bridging) amino acid sequence comprising D, and a fourth amino acid sequence being at least 90% homologous to YKAFEEAAEHFQPYIKFFATFDKGVAKKLSLKMNEVDFYEPFMDEPIAIPNKPYTEEELVEFVKEHQRPTL RRLRPEEMFETWEDDLNGIHIVAFAEKSDPDGYEFLEILKQVARDNTDNPDLSILWIDPDDFPLLVAYWEK TFKIDLFRPQIGVVNVTDADSVWMEIPDDDDLPTAEELEDWIEDVLSGKINTEDDDEDDDDDDNSDEEDN DDSDDDDDE corresponding to amino acids 179-399 of CAQ2_HUMAN (SEQ ID NO:349), which also corresponds to amino acids 101-321 of N56180_P6 (SEQ ID NO:288), wherein said first, second, third and fourth amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a head of N56180_P6 (SEQ ID NO:288), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence NETEAEQSYV (SEQ ID NO:402) of N56180_P6 (SEQ ID NO:288).","3. An isolated polypeptide encoding for an edge portion of N56180_P6 (SEQ ID NO:288), comprising a polypeptide having a length \u201cn\u201d, wherein \u201cn\u201d is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least two amino acids comprise LDY having a structure as follows (numbering according to N56180_P6 (SEQ ID NO:288)): a sequence starting from any of amino acid numbers 99\u2212x to 99; and ending at any of amino acid numbers 101+((n\u22122)\u2212x), in which x varies from 0 to n\u22122.","Variant protein N56180_P6 (SEQ ID NO:288) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 33, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein N56180_P6 (SEQ ID NO:288) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0870","num":"0917"},"tables":{"@attributes":{"id":"TABLE-US-00038","num":"00038"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 33"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino acid","Alternative","Previously"]},{"entry":["sequence","amino acid(s)","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["151","N ->","No"]},{"entry":["59","T -> A","Yes"]},{"entry":["69","V -> M","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein N56180_P6 (SEQ ID NO:288) is encoded by the following transcript(s): N56180_T5 (SEQ ID NO:8), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript N56180_T5 (SEQ ID NO:8) is shown in bold; this coding portion starts at position 1 and ends at position 964. The transcript also has the following SNPs as listed in Table 34 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein N56180_P6 (SEQ ID NO:288) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0872","num":"0919"},"tables":{"@attributes":{"id":"TABLE-US-00039","num":"00039"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 34"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on nucleotide","Alternative","Previously"]},{"entry":["sequence","nucleic acid","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["176","A -> G","Yes"]},{"entry":["206","G -> A","Yes"]},{"entry":["452","A ->","No"]},{"entry":["952","C -> T","Yes"]},{"entry":["1449","C -> T","Yes"]},{"entry":["1599","C -> G","Yes"]},{"entry":["1720","G -> T","No"]},{"entry":["1920","A -> C","No"]},{"entry":["1976","A ->","Yes"]},{"entry":["2069","A -> T","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein N56180_P7 (SEQ ID NO:289) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) N56180_T6 (SEQ ID NO:9). An alignment is given to the known protein (Calsequestrin, cardiac muscle isoform precursor (SEQ ID NO:349)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between N56180_P7 (SEQ ID NO:289) and CAQ2_HUMAN (SEQ ID NO:349):","1. An isolated chimeric polypeptide encoding for N56180_P7 (SEQ ID NO:289), comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferable at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence MSSWLSAGSPSSLSV (SEQ ID NO:403) corresponding to amino acids 1-15 of N56180_P7 (SEQ ID NO:289), and a second amino acid sequence being at least 90% homologous to VAKKLSLKMNEVDFYEPFMDEPIAIPNKPYTEEELVEFVKEHQRPTLRRLRPEEMFETWEDDLNGIHIVAF AEKSDPDGYEFLEILKQVARDNTDNPDLSILWIDPDDFPLLVAYWEKTFKIDLFRPQIGVVNVTDADSVW MEIPDDDDLPTAEELEDWIEDVLSGKINTEDDDEDDDDDDNSDEEDNDDSDDDDDE corresponding to amino acids 203-399 of CAQ2_HUMAN (SEQ ID NO:349), which also corresponds to amino acids 16-212 of N56180_P7 (SEQ ID NO:289), wherein said first and second amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a head of N56180_P7 (SEQ ID NO:289), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence MSSWLSAGSPSSLSV (SEQ ID NO:403) of N56180_P7 (SEQ ID NO:289).","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: intracellularly. The protein localization is believed to be intracellular because neither of the trans-membrane region prediction programs predicted a trans-membrane region for this protein. In addition both signal-peptide prediction programs predict that this protein is a non-secreted protein.","Variant protein N56180_P7 (SEQ ID NO:289) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 35, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein N56180_P7 (SEQ ID NO:289) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0879","num":"0926"},"tables":{"@attributes":{"id":"TABLE-US-00040","num":"00040"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 35"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino acid","Alternative","Previously"]},{"entry":["sequence","amino acid(s)","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["42","N ->","No"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein N56180_P7 (SEQ ID NO:289) is encoded by the following transcript(s): N56180_T6 (SEQ ID NO:9), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript N56180_T6 (SEQ ID NO:9) is shown in bold; this coding portion starts at position 71 and ends at position 706. The transcript also has the following SNPs as listed in Table 36 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein N56180_P7 (SEQ ID NO:289) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0881","num":"0928"},"tables":{"@attributes":{"id":"TABLE-US-00041","num":"00041"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 36"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on nucleotide","Alternative","Previously"]},{"entry":["sequence","nucleic acid","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["194","A ->","No"]},{"entry":["694","C -> T","Yes"]},{"entry":["1191","C -> T","Yes"]},{"entry":["1341","C -> G","Yes"]},{"entry":["1462","G -> T","No"]},{"entry":["1662","A -> C","No"]},{"entry":["1718","A ->","Yes"]},{"entry":["1811","A -> T","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein N56180_P8 (SEQ ID NO:290) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) N56180_T7 (SEQ ID NO:10). An alignment is given to the known protein (Calsequestrin, cardiac muscle isoform precursor (SEQ ID NO:349)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between N56180_P8 (SEQ ID NO:290) and CAQ2_HUMAN (SEQ ID NO:349):","1. An isolated chimeric polypeptide encoding for N56180_P8 (SEQ ID NO:290), comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence MCRGYSTLLNPVS (SEQ ID NO:404) corresponding to amino acids 1-13 of N56180_P8 (SEQ ID NO:290), and a second amino acid sequence being at least 90% homologous to DGYEFLEILKQVARDNTDNPDLSILWIDPDDFPLLVAYWEKTFKIDLFRPQIGVVNVTDADSVWMEIPDDD DLPTAEELEDWIEDVLSGKINTEDDDEDDDDDDNSDEEDNDDSDDDDDE corresponding to amino acids 280-399 of CAQ2_HUMAN (SEQ ID NO:349), which also corresponds to amino acids 14-133 of N56180_P8 (SEQ ID NO:290), wherein said first and second amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a head of N56180_P8 (SEQ ID NO:290), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence MCRGYSTLLNPVS (SEQ ID NO:404) of N56180_P8 (SEQ ID NO:290).","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: intracellularly. The protein localization is believed to be intracellularly because neither of the trans-membrane region prediction programs predicted a trans-membrane region for this protein. In addition both signal-peptide prediction programs predict that this protein is a non-secreted protein.","Variant protein N56180_P8 (SEQ ID NO:290) is encoded by the following transcript(s): N56180_T7 (SEQ ID NO:10), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript N56180_T7 (SEQ ID NO:10) is shown in bold; this coding portion starts at position 97 and ends at position 495. The transcript also has the following SNPs as listed in Table 37 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein N56180_P8 (SEQ ID NO:290) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0888","num":"0935"},"tables":{"@attributes":{"id":"TABLE-US-00042","num":"00042"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 37"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on nucleotide","Alternative","Previously"]},{"entry":["sequence","nucleic acid","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["483","C -> T","Yes"]},{"entry":["980","C -> T","Yes"]},{"entry":["1130","C -> G","Yes"]},{"entry":["1251","G -> T","No"]},{"entry":["1451","A -> C","No"]},{"entry":["1507","A ->","Yes"]},{"entry":["1600","A -> T","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein N56180_P9 (SEQ ID NO:291) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) N56180_T8 (SEQ ID NO:11). An alignment is given to the known protein (Calsequestrin, cardiac muscle isoform precursor (SEQ ID NO:349)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between N56180_P9 (SEQ ID NO:291) and CAQ2_HUMAN (SEQ ID NO:349):","1. An isolated chimeric polypeptide encoding for N56180_P9 (SEQ ID NO:291), comprising a first amino acid sequence being at least 90% homologous to MKRTHLFIVGIYFLSSCRAEEGLNFPTYDGKDRVVSLSEKNFKQVLKKYDLLCLYYHEPVSSDKVTQKQF QLKEIVLELVAQVLEHKAIGFVMVDAKKEAKLAKKLGFDEEGSLYILKGDRTIEFDGEFAADVLVEFLLDL IEDPVEIISSKLEVQAFERIEDYIKLIGFFKSEDSEYYKAFEEAAEHFQPYIKFFATFDKGVAKKLSLKMNEV DFYEPFMDEPIAIPNKPYTEEELVEFVKEHQR corresponding to amino acids 1-246 of CAQ2_HUMAN (SEQ ID NO:349), which also corresponds to amino acids 1-246 of N56180_P9 (SEQ ID NO:291), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence SRNWTQ (SEQ ID NO:405) corresponding to amino acids 247-252 of N56180_P9 (SEQ ID NO:291), wherein said first and second amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of N56180_P9 (SEQ ID NO:291), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence SRNWTQ (SEQ ID NO:405) in N56180_P9 (SEQ ID NO:291).","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: secreted or localized in the sarcoplasmic reticulum's terminal cisternae luminal spaces of cardiac and slow skeletal muscle cells like the WT protein. The protein localization is believed to be secreted because both signal-peptide prediction programs predict that this protein has a signal peptide, and neither trans-membrane region prediction program predicts that this protein has a trans-membrane region.","Variant protein N56180_P9 (SEQ ID NO:291) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 38, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein N56180_P9 (SEQ ID NO:291) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0895","num":"0942"},"tables":{"@attributes":{"id":"TABLE-US-00043","num":"00043"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 38"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino acid","Alternative","Previously"]},{"entry":["sequence","amino acid(s)","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["229","N ->","No"]},{"entry":["66","T -> A","Yes"]},{"entry":["76","V -> M","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein N56180_P9 (SEQ ID NO:291) is encoded by the following transcript(s): N56180_T8 (SEQ ID NO:11), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript N56180_T8 (SEQ ID NO:11) is shown in bold; this coding portion starts at position 242 and ends at position 997. The transcript also has the following SNPs as listed in Table 39 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein N56180_P9 (SEQ ID NO:291) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0897","num":"0944"},"tables":{"@attributes":{"id":"TABLE-US-00044","num":"00044"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 39"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on nucleotide","Alternative","Previously"]},{"entry":["sequence","nucleic acid","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["74","T ->","No"]},{"entry":["105","T -> C","Yes"]},{"entry":["1153","G -> A","Yes"]},{"entry":["1170","G -> A","Yes"]},{"entry":["206","G -> A","Yes"]},{"entry":["221","G -> A","Yes"]},{"entry":["228","A -> C","Yes"]},{"entry":["437","A -> G","Yes"]},{"entry":["467","G -> A","Yes"]},{"entry":["926","A ->","No"]},{"entry":["1095","A ->","No"]},{"entry":["1095","A -> T","No"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"As noted above, cluster N56180 features 22 segment(s), which were listed in Table 24 above and for which the sequence(s) are given at the end of the application. These segment(s) are portions of nucleic acid sequence(s) which are described herein separately because they are of particular interest. A description of each segment according to the present invention is now provided.","Segment cluster N56180_node2 (SEQ ID NO:73) according to the present invention is supported by 36 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T1 (SEQ ID NO:5), N56180_T3 (SEQ ID NO:6), N56180_T4 (SEQ ID NO:7) and N56180_T8 (SEQ ID NO:11). Table 40 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0900","num":"0947"},"tables":{"@attributes":{"id":"TABLE-US-00045","num":"00045"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 40"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["N56180_T1 (SEQ ID NO: 5)","1","237"]},{"entry":["N56180_T3 (SEQ ID NO: 6)","1","237"]},{"entry":["N56180_T4 (SEQ ID NO: 7)","1","237"]},{"entry":["N56180_T8 (SEQ ID NO: 11)","1","237"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node20 (SEQ ID NO:74) according to the present invention is supported by 30 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T1 (SEQ ID NO:5), N56180_T3 (SEQ ID NO:6), N56180_T4 (SEQ ID NO:7), N56180_T5 (SEQ ID NO:8), N56180_T6 (SEQ ID NO:9) and N56180_T8 (SEQ ID NO:11). Table 41 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0902","num":"0949"},"tables":{"@attributes":{"id":"TABLE-US-00046","num":"00046"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 41"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["N56180_T1 (SEQ ID NO: 5)","943","1073"]},{"entry":["N56180_T3 (SEQ ID NO: 6)","989","1119"]},{"entry":["N56180_T4 (SEQ ID NO: 7)","662","792"]},{"entry":["N56180_T5 (SEQ ID NO: 8)","374","504"]},{"entry":["N56180_T6 (SEQ ID NO: 9)","116","246"]},{"entry":["N56180_T8 (SEQ ID NO: 11)","848","978"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node22 (SEQ ID NO:75) according to the present invention is supported by 3 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T8 (SEQ ID NO:11). Table 42 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0904","num":"0951"},"tables":{"@attributes":{"id":"TABLE-US-00047","num":"00047"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 42"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["N56180_T8 (SEQ ID NO: 11)","979","1259"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node28 (SEQ ID NO:76) according to the present invention is supported by 1 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T7 (SEQ ID NO:10). Table 43 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0906","num":"0953"},"tables":{"@attributes":{"id":"TABLE-US-00048","num":"00048"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 43"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["N56180_T7 (SEQ ID NO: 10)","1","136"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node34 (SEQ ID NO:77) according to the present invention is supported by 37 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T1 (SEQ ID NO:5), N56180_T3 (SEQ ID NO:6), N56180_T4 (SEQ ID NO:7), following transcript(s): N56180_T1 (SEQ ID NO:5), N56180_T3 (SEQ ID NO:6), N56180_T4 (SEQ ID NO:7), describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0908","num":"0955"},"tables":{"@attributes":{"id":"TABLE-US-00049","num":"00049"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 44"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["N56180_T1 (SEQ ID NO: 5)","1397","1644"]},{"entry":["N56180_T3 (SEQ ID NO: 6)","1443","1690"]},{"entry":["N56180_T4 (SEQ ID NO: 7)","1116","1363"]},{"entry":["N56180_T5 (SEQ ID NO: 8)","828","1075"]},{"entry":["N56180_T6 (SEQ ID NO: 9)","570","817"]},{"entry":["N56180_T7 (SEQ ID NO: 10)","359","606"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node36 (SEQ ID NO:78) according to the present invention is supported by 77 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T1 (SEQ ID NO:5), N56180_T3 (SEQ ID NO:6), N56180_T4 (SEQ ID NO:7), N56180T5 (SEQ ID NO:8), N56180_T(SEQ ID NO:9) and N56180_T(SEQ ID NO:10). Table 45 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0910","num":"0957"},"tables":{"@attributes":{"id":"TABLE-US-00050","num":"00050"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 45"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["N56180_T1 (SEQ ID NO: 5)","1655","2778"]},{"entry":["N56180_T3 (SEQ ID NO: 6)","1701","2824"]},{"entry":["N56180_T4 (SEQ ID NO: 7)","1374","2497"]},{"entry":["N56180_T5 (SEQ ID NO: 8)","1086","2209"]},{"entry":["N56180_T6 (SEQ ID NO: 9)","828","1951"]},{"entry":["N56180_T7 (SEQ ID NO: 10)","617","1740"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node4 (SEQ ID NO:79) according to the present invention is supported by 34 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T1 (SEQ ID NO:5), N56180_T3 (SEQ ID NO:6), N56180_T4 (SEQ ID NO:7), N56180_T5 (SEQ ID NO:8) and N56180_T8 (SEQ ID NO:11). Table 46 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0912","num":"0959"},"tables":{"@attributes":{"id":"TABLE-US-00051","num":"00051"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 46"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["N56180_T1 (SEQ ID NO: 5)","295","475"]},{"entry":["N56180_T3 (SEQ ID NO: 6)","295","475"]},{"entry":["N56180_T4 (SEQ ID NO: 7)","295","475"]},{"entry":["N56180_T5 (SEQ ID NO: 8)","34","214"]},{"entry":["N56180_T8 (SEQ ID NO: 11)","295","475"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node6 (SEQ ID NO:80) according to the present invention is supported by 1 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T3 (SEQ ID NO:6). Table 47 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0914","num":"0961"},"tables":{"@attributes":{"id":"TABLE-US-00052","num":"00052"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 47"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["N56180_T3 (SEQ ID NO: 6)","476","616"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"According to an optional embodiment of the present invention, short segments related to the above cluster are also provided. These segments are up to about 120 bp in length, and so are included in a separate description.","Segment cluster N56180_node0 (SEQ ID NO:81) according to the present invention is supported by 1 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T5 (SEQ ID NO:8). Table 48 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0917","num":"0964"},"tables":{"@attributes":{"id":"TABLE-US-00053","num":"00053"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 48"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Segment"]},{"entry":["Transcript name","Segment starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["N56180_T5 (SEQ ID NO: 8)","1","33"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node10 (SEQ ID NO:82) according to the present invention is supported by 24 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T1 (SEQ ID NO:5), N56180_T3 (SEQ ID NO:6), N56180_T4 (SEQ ID NO:7) and N56180_T8 (SEQ ID NO:11). Table 49 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0919","num":"0966"},"tables":{"@attributes":{"id":"TABLE-US-00054","num":"00054"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 49"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["N56180_T1 (SEQ ID NO: 5)","561","661"]},{"entry":["N56180_T3 (SEQ ID NO: 6)","702","802"]},{"entry":["N56180_T4 (SEQ ID NO: 7)","561","661"]},{"entry":["N56180_T8 (SEQ ID NO: 11)","561","661"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node12 (SEQ ID NO:83) according to the present invention is supported by 27 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T1 (SEQ ID NO:5), N56180_T3 (SEQ ID NO:6) and N56180_T8 (SEQ ID NO:11). Table 50 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0921","num":"0968"},"tables":{"@attributes":{"id":"TABLE-US-00055","num":"00055"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 50"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["N56180_T1 (SEQ ID NO: 5)","662","773"]},{"entry":["N56180_T3 (SEQ ID NO: 6)","803","914"]},{"entry":["N56180_T8 (SEQ ID NO: 11)","662","773"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node14 (SEQ ID NO:84) according to the present invention is supported by 26 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T1 (SEQ ID NO:5), N56180_T3 (SEQ ID NO:6), N56180_T5 (SEQ ID NO:8) and N56180_T8 (SEQ ID NO:11). Table 51 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0923","num":"0970"},"tables":{"@attributes":{"id":"TABLE-US-00056","num":"00056"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 51"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment "]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["N56180_T1 (SEQ ID NO: 5)","774","847"]},{"entry":["N56180_T3 (SEQ ID NO: 6)","915","988"]},{"entry":["N56180_T5 (SEQ ID NO: 8)","300","373"]},{"entry":["N56180_T8 (SEQ ID NO: 11)","774","847"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node16 (SEQ ID NO:85) according to the present invention is supported by 1 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T1 (SEQ ID NO:5). Table 52 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0925","num":"0972"},"tables":{"@attributes":{"id":"TABLE-US-00057","num":"00057"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 52"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Segment "]},{"entry":["Transcript name","Segment starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["N56180_T1 (SEQ ID NO: 5)","848","942"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node18 (SEQ ID NO:86) according to the present invention is supported by 1 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T6 (SEQ ID NO:9). Table 53 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0927","num":"0974"},"tables":{"@attributes":{"id":"TABLE-US-00058","num":"00058"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 53"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Segment"]},{"entry":["Transcript name","Segment starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["N56180_T6 (SEQ ID NO: 9)","1","115"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node24 (SEQ ID NO:87) according to the present invention is supported by 25 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T1 (SEQ ID NO:5), N56180_T3 (SEQ ID NO:6), N56180_T4 (SEQ ID NO:7), N56180_T5 (SEQ ID NO:8) and N56180_T6 (SEQ ID NO:9). Table 54 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0929","num":"0976"},"tables":{"@attributes":{"id":"TABLE-US-00059","num":"00059"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 54"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Segment"]},{"entry":["Transcript name","Segment starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["N56180_T1 (SEQ ID NO: 5)","1074","1119"]},{"entry":["N56180_T3 (SEQ ID NO: 6)","1120","1165"]},{"entry":["N56180_T4 (SEQ ID NO: 7)","793","838"]},{"entry":["N56180_T5 (SEQ ID NO: 8)","505","550"]},{"entry":["N56180_T6 (SEQ ID NO: 9)","247","292"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node26 (SEQ ID NO:88) according to the present invention is supported by 28 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T1 (SEQ ID NO:5), N56180_T3 (SEQ ID NO:6), N56180_T4 (SEQ ID NO:7), N56180_T5 (SEQ ID NO:8) and N56180_T6 (SEQ ID NO:9). Table 55 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0931","num":"0978"},"tables":{"@attributes":{"id":"TABLE-US-00060","num":"00060"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 55"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Segment"]},{"entry":["Transcript name","Segment starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["N56180_T1 (SEQ ID NO: 5)","1120","1174"]},{"entry":["N56180_T3 (SEQ ID NO: 6)","1166","1220"]},{"entry":["N56180_T4 (SEQ ID NO: 7)","839","893"]},{"entry":["N56180_T5 (SEQ ID NO: 8)","551","605"]},{"entry":["N56180_T6 (SEQ ID NO: 9)","293","347"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node29 (SEQ ID NO:89) according to the present invention is supported by 32 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T1 (SEQ ID NO:5), N56180_T3 (SEQ ID NO:6), N56180_T4 (SEQ ID NO:7), N56180_T5 (SEQ ID NO:8), N56180_T6 (SEQ ID NO:9) and N56180_T7 (SEQ ID NO:10). Table 56 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0933","num":"0980"},"tables":{"@attributes":{"id":"TABLE-US-00061","num":"00061"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 56"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["N56180_T1 (SEQ ID NO: 5)","1175","1275"]},{"entry":["N56180_T3 (SEQ ID NO: 6)","1221","1321"]},{"entry":["N56180_T4 (SEQ ID NO: 7)","894","994"]},{"entry":["N56180_T5 (SEQ ID NO: 8)","606","706"]},{"entry":["N56180_T6 (SEQ ID NO: 9)","348","448"]},{"entry":["N56180_T7 (SEQ ID NO: 10)","137","237"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node3 (SEQ ID NO:90) according to the present invention is supported by 36 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T1 (SEQ ID NO:5), N56180_T3 (SEQ ID NO:6), N56180_T4 (SEQ ID NO:7) and N56180_T8 (SEQ ID NO:11). Table 57 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0935","num":"0982"},"tables":{"@attributes":{"id":"TABLE-US-00062","num":"00062"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 57"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment "]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["N56180_T1 (SEQ ID NO: 5)","238","294"]},{"entry":["N56180_T3 (SEQ ID NO: 6)","238","294"]},{"entry":["N56180_T4 (SEQ ID NO: 7)","238","294"]},{"entry":["N56180_T8 (SEQ ID NO: 11)","238","294"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node31 (SEQ ID NO:91) according to the present invention is supported by 30 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T1 (SEQ ID NO:5), N56180_T3 (SEQ ID NO:6), N56180_T4 (SEQ ID NO:7), N56180_T5 (SEQ ID NO:8), N56180_T6 (SEQ ID NO:9) and N56180_T7 (SEQ ID NO:10). Table 58 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0937","num":"0984"},"tables":{"@attributes":{"id":"TABLE-US-00063","num":"00063"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 58"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["N56180_T1 (SEQ ID NO: 5)","1276","1350"]},{"entry":["N56180_T3 (SEQ ID NO: 6)","1322","1396"]},{"entry":["N56180_T4 (SEQ ID NO: 7)","995","1069"]},{"entry":["N56180_T5 (SEQ ID NO: 8)","707","781"]},{"entry":["N56180_T6 (SEQ ID NO: 9)","449","523"]},{"entry":["N56180_T7 (SEQ ID NO: 10)","238","312"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node33 (SEQ ID NO:92) according to the present invention is supported by 30 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T1 (SEQ ID NO:5), N56180_T3 (SEQ ID NO:6), N56180_T4 (SEQ ID NO:7), N56180_T5 (SEQ ID NO:8), N56180_T6 (SEQ ID NO:9) and N56180_T7 (SEQ ID NO:10). Table 59 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0939","num":"0986"},"tables":{"@attributes":{"id":"TABLE-US-00064","num":"00064"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 59"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["N56180_T1 (SEQ ID NO: 5)","1351","1396"]},{"entry":["N56180_T3 (SEQ ID NO: 6)","1397","1442"]},{"entry":["N56180_T4 (SEQ ID NO: 7)","1070","1115"]},{"entry":["N56180_T5 (SEQ ID NO: 8)","782","827"]},{"entry":["N56180_T6 (SEQ ID NO: 9)","524","569"]},{"entry":["N56180_T7 (SEQ ID NO: 10)","313","358"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node35 (SEQ ID NO:93) according to the present invention can be found in the following transcript(s): N56180_T1 (SEQ ID NO:5), N56180_T3 (SEQ ID NO:6), N56180_T4 (SEQ ID NO:7), N56180_T5 (SEQ ID NO:8), N56180_T6 (SEQ ID NO:9) and N56180_T7 (SEQ ID NO:10). Table 60 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0941","num":"0988"},"tables":{"@attributes":{"id":"TABLE-US-00065","num":"00065"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 60"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["N56180_T1 (SEQ ID NO: 5)","1645","1654"]},{"entry":["N56180_T3 (SEQ ID NO: 6)","1691","1700"]},{"entry":["N56180_T4 (SEQ ID NO: 7)","1364","1373"]},{"entry":["N56180_T5 (SEQ ID NO: 8)","1076","1085"]},{"entry":["N56180_T6 (SEQ ID NO: 9)","818","827"]},{"entry":["N56180_T7 (SEQ ID NO: 10)","607","616"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster N56180_node8 (SEQ ID NO:94) according to the present invention is supported by 25 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): N56180_T1 (SEQ ID NO:5), N56180_T3 (SEQ ID NO:6), N56180_T4 (SEQ ID NO:7), N56180_T5 (SEQ ID NO:8) and N56180_T8 (SEQ ID NO:11). Table 61 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0943","num":"0990"},"tables":{"@attributes":{"id":"TABLE-US-00066","num":"00066"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 61"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["N56180_T1 (SEQ ID NO: 5)","476","560"]},{"entry":["N56180_T3 (SEQ ID NO: 6)","617","701"]},{"entry":["N56180_T4 (SEQ ID NO: 7)","476","560"]},{"entry":["N56180_T5 (SEQ ID NO: 8)","215","299"]},{"entry":["N56180_T8 (SEQ ID NO: 11)","476","560"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}},"br":{}},{"@attributes":{"id":"p-0944","num":"0991"},"tables":{"@attributes":{"id":"TABLE-US-00067","num":"00067"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"266pt","align":"left"}},"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Sequence name: \/tmp\/QH4bp76Ojk\/sAp7DyaTKD:CAQ2_HUMAN (SEQ ID NO: 349)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: N56180_P2 (SEQ ID NO: 285) \u00d7 CAQ2_HUMAN (SEQ ID NO: 349)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","1955.00","Escore:","0"]},{"entry":["Matching length:","203","Total length:","203"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"266pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00009","num":"00009"},"img":{"@attributes":{"id":"EMI-C00009","he":"53.51mm","wi":"84.92mm","file":"US07714100-20100511-C00009.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/VtcMdCiEuz\/FlmsgLbcq4:CAQ2_HUMAN (SEQ ID NO: 349)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: N56180_P4 (SEQ ID NO: 286) \u00d7 CAQ2_HUMAN (SEQ ID NO: 349)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","3806.00","Escore:","0"]},{"entry":["Matching length:","399","Total length:","446"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","89.46","Total Percent Identity:","89.46"]},{"entry":["Gaps:","1"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"266pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00010","num":"00010"},"img":{"@attributes":{"id":"EMI-C00010","he":"64.35mm","wi":"85.01mm","file":"US07714100-20100511-C00010.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00011","num":"00011"},"img":{"@attributes":{"id":"EMI-C00011","he":"31.75mm","wi":"85.01mm","file":"US07714100-20100511-C00011.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/lRixkfCRfD\/JDL7BwYPJs:CAQ2_HUMAN (SEQ ID NO: 349)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: N56180_P5 (SEQ ID NO: 287) \u00d7 CAQ2_HUMAN (SEQ ID NO: 349)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","3202.00","Escore:","0"]},{"entry":["Matching length:","337","Total length:","399"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","84.46","Total Percent Identity:","84.46"]},{"entry":["Gaps:","1"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"266pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00012","num":"00012"},"img":{"@attributes":{"id":"EMI-C00012","he":"42.67mm","wi":"84.92mm","file":"US07714100-20100511-C00012.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00013","num":"00013"},"img":{"@attributes":{"id":"EMI-C00013","he":"42.59mm","wi":"84.92mm","file":"US07714100-20100511-C00013.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/rs5xPc26iA\/XlzfpEDJF7:CAQ2_HUMAN (SEQ ID NO: 349)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: N56180_P6 (SEQ ID NO: 288) \u00d7 CAQ2_HUMAN (SEQ ID NO: 349)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","2955.00","Escore:","0"]},{"entry":["Matching length:","314","Total length:","385"]},{"entry":["Matching Percent Similarity:","99.04","Matching Percent Identity:","99.04"]},{"entry":["Total Percent Similarity:","80.78","Total Percent Identity:","80.78"]},{"entry":["Gaps:","1"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"266pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00014","num":"00014"},"img":{"@attributes":{"id":"EMI-C00014","he":"42.59mm","wi":"84.92mm","file":"US07714100-20100511-C00014.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00015","num":"00015"},"img":{"@attributes":{"id":"EMI-C00015","he":"42.59mm","wi":"85.01mm","file":"US07714100-20100511-C00015.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/YOj6jtvAt2\/UVZXGVRVOx:CAQ2_HUMAN (SEQ ID NO: 349)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: N56180_P7 (SEQ ID NO: 289) \u00d7 CAQ2_HUMAN (SEQ ID NO: 349)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","1959.00","Escore:","0"]},{"entry":["Matching length:","197","Total length:","197"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"266pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00016","num":"00016"},"img":{"@attributes":{"id":"EMI-C00016","he":"42.59mm","wi":"84.92mm","file":"US07714100-20100511-C00016.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/kmYMCJ1GuB\/no5BPO2sjR:CAQ2_HUMAN (SEQ ID NO: 349)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: N56180_P8 (SEQ ID NO: 290) \u00d7 CAQ2_HUMAN (SEQ ID NO: 349)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","1187.00","Escore:","0"]},{"entry":["Matching length:","120","Total length:","120"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"266pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00017","num":"00017"},"img":{"@attributes":{"id":"EMI-C00017","he":"31.75mm","wi":"84.92mm","file":"US07714100-20100511-C00017.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/JIYFiyiYEk\/c42Jok7Lfq:CAQ2_HUMAN (SEQ ID NO: 349)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: N56180_P9 (SEQ ID NO: 291) \u00d7 CAQ2_HUMAN (SEQ ID NO: 349)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","2388.00","Escore:","0"]},{"entry":["Matching length:","246","Total length:","246"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"266pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00018","num":"00018"},"img":{"@attributes":{"id":"EMI-C00018","he":"53.59mm","wi":"84.92mm","file":"US07714100-20100511-C00018.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}]}}},"Expression of Calsequestrin, cardiac muscle isoform transcripts detectable by or according to seg6 (SEQ ID NO:335), N56180 amplicon(s) and N56180 seg6F (SEQ ID NO:279) and N56180 seg6R (SEQ ID NO:280) primers was measured by real time PCR. In parallel the expression of four housekeeping genes\u2014RPL19 (GenBank Accession No. NM000981 (SEQ ID NO:437); RPL19 amplicon (SEQ ID NO:440)), TATA box (GenBank Accession No. NM003194 (SEQ ID NO:441); TATA amplicon (SEQ ID NO:444)), Ubiquitin (GenBank Accession No. BC000449 (SEQ ID NO:445); amplicon\u2014Ubiquitin-amplicon (SEQ ID NO:448)) and SDHA (GenBank Accession No. NM004168 (SEQ ID NO:449); amplicon\u2014SDHA-amplicon (SEQ ID NO:452)) was measured similarly. For each RT sample, the expression of the above amplicons was normalized to the geometric mean of the quantities of the housekeeping genes. The normalized quantity of each RT sample was then divided by the median of the quantities of the heart samples (Sample Nos. 44, 45, 46, Table 2, above, \u201cTissue samples in testing panel\u201d), to obtain a value of fold up-regulation for each sample relative to median of the heart.",{"@attributes":{"id":"p-0946","num":"0993"},"figref":["FIG. 9","FIG. 9"]},"Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: N56180 seg6F (SEQ ID NO:279) forward primer; and N56180 seg6R (SEQ ID NO:280) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: N56180 seg6.",{"@attributes":{"id":"p-0949","num":"0996"},"tables":{"@attributes":{"id":"TABLE-US-00068","num":"00068"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"287pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["N56180 seg6F (SEQ ID NO: 279):",{}]},{"entry":"ATATCCCAGTGGTGGTTGCATT"},{"entry":{}},{"entry":"N56180 seg6R (SEQ ID NO: 280):"},{"entry":"CCCTCCCCAGCGTTTCC"},{"entry":{}},{"entry":"N56180 seg6 (SEQ ID NO: 335):"},{"entry":"ATATCCCAGTGGTGGTTGCATTTCCAAAGCCCAAGAGAGGAAGGCAAAATGAAGTTGCTGGAGTTGA"},{"entry":{}},{"entry":"GTGAATCTGCAGATGGAGCTGCGTGGAAACGCTGGGGAGGG"}]}}}},"br":{},"sub":["\u2014","\u2014","\u2014"]},{"@attributes":{"id":"p-0950","num":"0997"},"figref":["FIG. 10","FIG. 10"]},"Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: N56180 seg22F (SEQ ID NO:336) forward primer; and N56180 seg22R (SEQ ID NO:337) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: N56180 seg22.",{"@attributes":{"id":"p-0953","num":"1000"},"tables":{"@attributes":{"id":"TABLE-US-00069","num":"00069"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"294pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"147pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["N56180 seg22F (SEQ ID NO: 336):",{}]},{"entry":"TTGATACCACTTAGTGTAGCTCCAGC"},{"entry":{}},{"entry":"N56180 seg22R (SEQ ID NO: 337):"},{"entry":"TCAAGTAGTTGCTACAGACGCCA"},{"entry":{}},{"entry":"N56180 seg22 (SEQ ID NO: 361):"},{"entry":"TTGATACCACTTAGTGTAGCTCCAGCATGGATCAGCAAACTTTTTCTGTAAAGAACAAAATGGTAAAT"},{"entry":{}},{"entry":"ATTTCAGGTTCTGTGGGCCAGATGGCGTCTGTAGCAACTACTTGA"}]}}}}},"Cluster T10377 features 6 transcript(s) and 18 segment(s) of interest, the names for which are given in Tables 62 and 63, respectively, the sequences themselves are given at the end of the application. The selected protein variants are given in table 64.",{"@attributes":{"id":"p-0955","num":"1002"},"tables":{"@attributes":{"id":"TABLE-US-00070","num":"00070"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 62"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Transcripts of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"126pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Transcript Name","Seq ID No."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"T10377_T0","12"]},{"entry":[{},"T10377_T1","13"]},{"entry":[{},"T10377_T2","14"]},{"entry":[{},"T10377_T5","15"]},{"entry":[{},"T10377_T6","16"]},{"entry":[{},"T10377_T7","17"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0956","num":"1003"},"tables":{"@attributes":{"id":"TABLE-US-00071","num":"00071"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 63"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segments of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment Name","Seq ID No."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"T10377_node_0","95"]},{"entry":[{},"T10377_node_17","96"]},{"entry":[{},"T10377_node_19","97"]},{"entry":[{},"T10377_node_21","98"]},{"entry":[{},"T10377_node_27","99"]},{"entry":[{},"T10377_node_33","100"]},{"entry":[{},"T10377_node_12","101"]},{"entry":[{},"T10377_node_14","102"]},{"entry":[{},"T10377_node_16","103"]},{"entry":[{},"T10377_node_2","104"]},{"entry":[{},"T10377_node_23","105"]},{"entry":[{},"T10377_node_25","106"]},{"entry":[{},"T10377_node_29","107"]},{"entry":[{},"T10377_node_3","108"]},{"entry":[{},"T10377_node_31","109"]},{"entry":[{},"T10377_node_5","110"]},{"entry":[{},"T10377_node_8","111"]},{"entry":[{},"T10377_node_9","112"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0957","num":"1004"},"tables":{"@attributes":{"id":"TABLE-US-00072","num":"00072"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 64"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Proteins of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"126pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Protein Name","Seq ID No."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"T10377_P2","292"]},{"entry":[{},"T10377_P5","293"]},{"entry":[{},"T10377_P6","294"]},{"entry":[{},"T10377_P7","295"]},{"entry":[{},"T10377_P8","296"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The heart-selective diagnostic marker prediction engine provided the following results with regard to cluster T10377. Predictions were made for selective expression of transcripts of this cluster in heart tissue, according to the previously described methods. The numbers on the y-axis of  refer to weighted expression of ESTs in each category, as \u201cparts per million\u201d (ratio of the expression of ESTs for a particular cluster to the expression of all ESTs in that category, according to parts per million).","Overall, the following results were obtained as shown with regard to the histogram in , concerning the number of heart-specific clones in libraries\/sequences.","This cluster was found to be selectively expressed in heart for the following reasons: in a comparison of the ratio of expression of the cluster in heart specific ESTs to the overall expression of the cluster in non-heart ESTs, which was found to be 10.9. The expression level of this gene in muscle was negligible; and fisher exact test P-values were computed both for library and weighted clone counts to check that the counts are statistically significant, and were found to be 8.60E-15.","One particularly important measure of specificity of expression of a cluster in heart tissue is the previously described comparison of the ratio of expression of the cluster in heart as opposed to muscle. This cluster was found to be specifically expressed in heart as opposed to non-heart ESTs as described above. However, many proteins have been shown to be generally expressed at a higher level in both heart and muscle, which is less desirable. For this cluster, as described above, the expression level of this gene in muscle was negligible, which clearly supports specific expression in heart tissue.","As noted above, cluster T10377 features 6 transcript(s), which were listed in Table 62 above. A description of each variant protein according to the present invention is now provided.","Variant protein T10377_P2 (SEQ ID NO:292) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) T10377_T1 (SEQ ID NO:13) and T10377_T2 (SEQ ID NO:14). One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between T10377_P2 (SEQ ID NO:292) and Q96NF5 (SEQ ID NO:362):1. An isolated chimeric polypeptide encoding for T10377_P2 (SEQ ID NO:292), comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence MEISLVKCSE (SEQ ID NO:406) corresponding to amino acids 1-10 of T10377_P2 (SEQ ID NO:292), second amino acid sequence being at least 90% homologous to ANVCRLRLTVPPESPVPEQCEKKIERKEQLLDLSNGEPTRKLPQGVVYGVVRRSDQNQQKEMVVYGWST SQLKEEMNYIKDVRATLEKVRKRMYGDYDEMRQKIRQLTQELSVSHAQQEYLENHIQTQSSALDRFNAM NSALASDSIGLQKTLVDVTLENSNIKDQIRNLQQTYEASMDKLREKQRQLEVAQVENQLLKMKVESSQEA NAEVMREMTKKLYSQYEEKLQEEQRKHSAEKEALLEETNSFLK corresponding to amino acids 26-276 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 11-261 of T10377_P2 (SEQ ID NO:292), followed by A, and a third amino acid sequence being at least 90% homologous to IEEANKKMQAAEISLEEKDQRIGELDRLIERMEKERHQLQLQLLEHETEMSGELTDSDKERYQQLEEASAS LRERIRHLDDMVHCQQKKVKQMVEEIESLKKKLQQKQLLILQLLEKISFLEGENNELQSRLDYLTETQAKT EVETREIGVGCDLLPSQTGRTREIVMPSRNYTPYTRVLELTMKKTLT corresponding to amino acids 278-466 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 263-451 of T10377_P2 (SEQ ID NO:292), wherein said first, second, A, and third amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a head of T10377_P2 (SEQ ID NO:292), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence MEISLVKCSE (SEQ ID NO:406) of T10377_P2 (SEQ ID NO:292).","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: intracellularly. The protein localization is believed to be intracellular because neither of the trans-membrane region prediction programs predicted a trans-membrane region for this protein. In addition both signal-peptide prediction programs predict that this protein is a non-secreted protein.","Variant protein T10377_P2 (SEQ ID NO:292) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 65, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein T10377_P2 (SEQ ID NO:292) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0968","num":"1015"},"tables":{"@attributes":{"id":"TABLE-US-00073","num":"00073"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 65"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on",{},{}]},{"entry":["amino acid sequence","Alternative amino acid(s)","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["262","A -> V","No"]},{"entry":["30","C -> S","No"]},{"entry":["323","R -> G","No"]},{"entry":["36","R -> K","No"]},{"entry":["439","T ->","No"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein T10377_P2 (SEQ ID NO:292) is encoded by the following transcript(s): T10377_T1 (SEQ ID NO:13) and T10377_T2 (SEQ ID NO:14), for which the sequence(s) is\/are given at the end of the application.","The coding portion of transcript T10377_T1 (SEQ ID NO:13) is shown in bold; this coding portion starts at position 166 and ends at position 1518. The transcript also has the following SNPs as listed in Table 66 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein T10377_P2 (SEQ ID NO:292) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0971","num":"1018"},"tables":{"@attributes":{"id":"TABLE-US-00074","num":"00074"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 66"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on",{},{}]},{"entry":["nucleotide sequence","Alternative nucleic acid","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["152","A -> T","Yes"]},{"entry":["253","T -> A","No"]},{"entry":["272","G -> A","No"]},{"entry":["624","A -> G","Yes"]},{"entry":["786","G -> A","No"]},{"entry":["950","C -> T","No"]},{"entry":["1077","A -> G","No"]},{"entry":["1132","A -> G","No"]},{"entry":["1482","A ->","No"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"The coding portion of transcript T10377_T2 (SEQ ID NO:14) is shown in bold; this coding portion starts at position 270 and ends at position 1622. The transcript also has the following SNPs as listed in Table 67 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein T10377_P2 (SEQ ID NO:292) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0973","num":"1020"},"tables":{"@attributes":{"id":"TABLE-US-00075","num":"00075"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 67"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on",{},{}]},{"entry":["nucleotide sequence","Alternative nucleic acid","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["13","G -> T","Yes"]},{"entry":["26","G -> A","Yes"]},{"entry":["890","G -> A","No"]},{"entry":["1054","C -> T","No"]},{"entry":["1181","A -> G","No"]},{"entry":["1236","A -> G","No"]},{"entry":["1586","A ->","No"]},{"entry":["88","C -> T","Yes"]},{"entry":["115","G -> A","Yes"]},{"entry":["126","A -> G","Yes"]},{"entry":["212","A -> G","No"]},{"entry":["256","A -> T","Yes"]},{"entry":["357","T -> A","No"]},{"entry":["376","G -> A","No"]},{"entry":["728","A -> G","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein T10377_P5 (SEQ ID NO:293) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) T10377_T5 (SEQ ID NO:15). One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between T10377_P5 (SEQ ID NO:293) and Q96NF5 (SEQ ID NO:362):","1. An isolated chimeric polypeptide encoding for T10377_P5 (SEQ ID NO:293), comprising a first amino acid sequence being at least 90% homologous to MLRSTSTVTLLSGGAARTPGAPSRRANVCRLRLTVPPESPVPEQCEKKIERKEQLLDLSNGEPTRKLPQGV VYGVVRRSDQNQQKEMVVYGWSTSQLKEEMNYIKDVRATLEKVRKRMYGDYDEMRQKIRQLTQELSV SHAQQEYLENHIQTQSSALDRFNAMNSALASDSIGLQKTLVDVTLENSNIKDQIRNLQQTYEASMDKLRE KQRQLEVAQVENQLLKMKVESSQEANAEVMREMTKKLYSQYEEKLQEEQRKHSAEKEALLEETNSFLK corresponding to amino acids 1-276 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 1-276 of T10377_P5 (SEQ ID NO:293), followed by A, a second amino acid sequence being at least 90% homologous to IEEANKKMQAAEISLEEKDQRIGELDRLIERMEKERHQLQLQLLEHETEMSGELTDSDKERYQQLEEASAS LRERIRHLDDMVHCQQKKVKQMVE corresponding to amino acids 278-372 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 278-372 of T10377_P5 (SEQ ID NO:293), and a third amino acid sequence being at least 90% homologous to ENNELQSRLDYLTETQAKTEVETREIGVGCDLLPSQTGRTREIVMPSRNYTPYTRVLELTMKKTLT corresponding to amino acids 401-466 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 373-438 of T10377_P5 (SEQ ID NO:293), wherein said first, A, second, and third amino acid sequences are contiguous and in a sequential order.","2. An isolated chimeric polypeptide encoding for an edge portion of T10377_P5 (SEQ ID NO:293), comprising a polypeptide having a length \u201cn\u201d, wherein n is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least two amino acids comprise EE, having a structure as follows: a sequence starting from any of amino acid numbers 372\u2212x to 372; and ending at any of amino acid numbers 373+((n\u22122)\u2212x), in which x varies from 0 to n\u22122.","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: secreted. The protein localization is believed to be secreted because one of the two signal-peptide prediction programs (HMM:Signal peptide,NN:NO) predicts that this protein has a signal peptide.","Variant protein T10377_P5 (SEQ ID NO:293) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 68, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein T10377_P5 (SEQ ID NO:293) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0980","num":"1027"},"tables":{"@attributes":{"id":"TABLE-US-00076","num":"00076"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 68"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on",{},{}]},{"entry":["amino acid sequence","Alternative amino acid(s)","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["25","R -> G","No"]},{"entry":["277","A -> V","No"]},{"entry":["338","R -> G","No"]},{"entry":["426","T ->","No"]},{"entry":["45","C -> S","No"]},{"entry":["51","R -> K","No"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein T10377_P5 (SEQ ID NO:293) is encoded by the following transcript(s): T10377_T5 (SEQ ID NO:15), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript T10377_T5 (SEQ ID NO:15) is shown in bold; this coding portion starts at position 140 and ends at position 1453. The transcript also has the following SNPs as listed in Table 69 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein T10377_P5 (SEQ ID NO:293) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0982","num":"1029"},"tables":{"@attributes":{"id":"TABLE-US-00077","num":"00077"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 69"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on",{},{}]},{"entry":["nucleotide sequence","Alternative nucleic acid","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["13","G -> T","Yes"]},{"entry":["26","G -> A","Yes"]},{"entry":["969","C -> T","No"]},{"entry":["1096","A -> G","No"]},{"entry":["1151","A -> G","No"]},{"entry":["1417","A ->","No"]},{"entry":["88","C -> T","Yes"]},{"entry":["115","G -> A","Yes"]},{"entry":["126","A -> G","Yes"]},{"entry":["212","A -> G","No"]},{"entry":["272","T -> A","No"]},{"entry":["291","G -> A","No"]},{"entry":["643","A -> G","Yes"]},{"entry":["805","G -> A","No"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein T10377_P6 (SEQ ID NO:294) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) T10377_T6 (SEQ ID NO:16). One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between T10377_P6 (SEQ ID NO:294) and Q96NF5 (SEQ ID NO:362):","1. An isolated chimeric polypeptide encoding for T10377_P6 (SEQ ID NO:294), comprising a first amino acid sequence being at least 90% homologous to MLRSTSTVTLLSGGAARTPGAPSRRANVCRLRLTVPPESPVPEQCEKKIERKEQLLDLSNGEPTRKLPQGV VYGVVRRSDQNQQKEMVVYGWSTSQLKEEMNYIKDVRATLEKVRKRMYGDYDEMRQKIRQLTQELSV SHAQQEYLENHIQTQSSALDRFNAMNSALASDSIGLQKTLVDVTLENSNIKDQIRNLQQTYEASMDKLRE KQRQLEVAQVENQLLKMKVESSQEANAEVMREMTKKLYSQYEEKLQEEQRKHSAEKEALLEETNSFLK corresponding to amino acids 1-276 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 1-276 of T10377_P6 (SEQ ID NO:294), followed by A, a second amino acid sequence being at least 90% homologous to IEEANKKMQAAEISLEEKDQRIGELDRLIERMEKERHQLQLQLLEHETEMSGELTDSDKERYQQLEEASAS LRERIRHLDDMVHCQQKKVKQMVEEIESLKKKLQQKQLLILQLLEKISFLEGE corresponding to amino acids 278-401 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 278-401 of T10377_P6 (SEQ ID NO:294), and a third amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence PNRQDS (SEQ ID NO:407) corresponding to amino acids 402-407 of T10377_P6 (SEQ ID NO:294), wherein said first, A, second and third amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of T10377_P6 (SEQ ID NO:294), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence PNRQDS (SEQ ID NO:407) in T10377_P6 (SEQ ID NO:294).","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: secreted. The protein localization is believed to be secreted because one of the two signal-peptide prediction programs (HMM:Signal peptide,NN:NO) predicts that this protein has a signal peptide.","Variant protein T10377_P6 (SEQ ID NO:294) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 70, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein T10377_P6 (SEQ ID NO:294) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0989","num":"1036"},"tables":{"@attributes":{"id":"TABLE-US-00078","num":"00078"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 70"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on",{},{}]},{"entry":["amino acid sequence","Alternative amino acid(s)","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["25","R -> G","No"]},{"entry":["277","A -> V","No"]},{"entry":["338","R -> G","No"]},{"entry":["45","C -> S","No"]},{"entry":["51","R -> K","No"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein T10377_P6 (SEQ ID NO:294) is encoded by the following transcript(s): T10377_T6 (SEQ ID NO:16), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript T10377_T6 (SEQ ID NO:16) is shown in bold; this coding portion starts at position 140 and ends at position 1360. The transcript also has the following SNPs as listed in Table 71 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein T10377_P6 (SEQ ID NO:294) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0991","num":"1038"},"tables":{"@attributes":{"id":"TABLE-US-00079","num":"00079"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 71"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on",{},{}]},{"entry":["nucleotide sequence","Alternative nucleic acid","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["13","G -> T","Yes"]},{"entry":["26","G -> A","Yes"]},{"entry":["969","C -> T","No"]},{"entry":["1096","A -> G","No"]},{"entry":["1151","A -> G","No"]},{"entry":["1400","A ->","No"]},{"entry":["88","C -> T","Yes"]},{"entry":["115","G -> A","Yes"]},{"entry":["126","A -> G","Yes"]},{"entry":["212","A -> G","No"]},{"entry":["272","T -> A","No"]},{"entry":["291","G -> A","No"]},{"entry":["643","A -> G","Yes"]},{"entry":["805","G -> A","No"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein T10377_P7 (SEQ ID NO:295) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) T10377_T7 (SEQ ID NO:17). One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between T10377_P7 (SEQ ID NO:295) and Q96NF5 (SEQ ID NO:362):","1. An isolated chimeric polypeptide encoding for T10377_P7 (SEQ ID NO:295), comprising a first amino acid sequence being at least 90% homologous to MLRSTSTVTLLSGGAARTPGAPSRRANVCRLRLTVPPESPVPEQCEKKIERKEQLLDLSNGEPTRKLPQGV VYGVVRRSDQNQQKEMVVYGWSTSQLKEEMNYIKDVRATLEKVRKRMYGDYDEMRQKIRQLTQELSV SHAQQEYLENHIQTQSSALDRFNAMNSALASDSIGLQKTLVDVTLENSNIKDQIRNLQQTYEASMDKLRE KQRQLEVAQVENQLLKMKVESSQEANAEVMREMTKKLYSQYEEKLQEEQRKHSAEKEALLEETNSFLK corresponding to amino acids 1-276 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 1-276 of T10377_P7 (SEQ ID NO:295), followed by A, a second amino acid sequence being at least 90% homologous to IEEANKKMQAAEISLEEKDQRIGELDRLIERMEKERHQLQLQLLEHETEMSGELTDSDKERYQQLEEASAS LRERIRHLDDMVHCQQKKVKQMVEEI corresponding to amino acids 278-374 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 278-374 of T10377_P7 (SEQ ID NO:295), and a third amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence MSHELFSRFSLRLFGR (SEQ ID NO:408) corresponding to amino acids 375-390 of T10377_P7 (SEQ ID NO:295), wherein said first, A, second and third amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of T10377_P7 (SEQ ID NO:295), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence MSHELFSRFSLRLFGR (SEQ ID NO:408) in T10377_P7 (SEQ ID NO:295).","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: secreted. The protein localization is believed to be secreted because one of the two signal-peptide prediction programs (HMM:Signal peptide,NN:NO) predicts that this protein has a signal peptide.","Variant protein T10377_P7 (SEQ ID NO:295) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 72, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein T10377_P7 (SEQ ID NO:295) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-0998","num":"1045"},"tables":{"@attributes":{"id":"TABLE-US-00080","num":"00080"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 72"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on",{},{}]},{"entry":["amino acid sequence","Alternative amino acid(s)","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["25","R -> G","No"]},{"entry":["277","A -> V","No"]},{"entry":["338","R -> G","No"]},{"entry":["45","C -> S","No"]},{"entry":["51","R -> K","No"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein T10377_P7 (SEQ ID NO:295) is encoded by the following transcript(s): T10377_T7 (SEQ ID NO:17), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript T10377_T7 (SEQ ID NO:17) is shown in bold; this coding portion starts at position 140 and ends at position 1309. The transcript also has the following SNPs as listed in Table 73 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein T10377_P7 (SEQ ID NO:295) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-1000","num":"1047"},"tables":{"@attributes":{"id":"TABLE-US-00081","num":"00081"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 73"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on",{},{}]},{"entry":["nucleotide sequence","Alternative nucleic acid","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["13","G -> T","Yes"]},{"entry":["26","G -> A","Yes"]},{"entry":["969","C -> T","No"]},{"entry":["1096","A -> G","No"]},{"entry":["1151","A -> G","No"]},{"entry":["88","C -> T","Yes"]},{"entry":["115","G -> A","Yes"]},{"entry":["126","A -> G","Yes"]},{"entry":["212","A -> G","No"]},{"entry":["272","T -> A","No"]},{"entry":["291","G -> A","No"]},{"entry":["643","A -> G","Yes"]},{"entry":["805","G -> A","No"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}},"br":{}},"Comparison report between T10377_P8 (SEQ ID NO:296) and Q96NF5 (SEQ ID NO:362):","An isolated chimeric polypeptide encoding for T10377_P8 (SEQ ID NO:296), comprising a first amino acid sequence being at least 90% homologous to MEISLVKCSEANVCRLRLTVPPESPVPEQCEKKIERKEQLLDLSNGEPTRKLPQGVVYGVVRRSDQNQQKE MVVYGWSTSQLKEEMNYIKDVRATLEKVRKRMYGDYDEMRQKIRQLTQELSVSHAQQEYLENHIQTQS SALDRFNAMNSALASDSIGLQKTLVDVTLENSNIKDQIRNLQQTYEASMDKLREKQRQLEVAQVENQLLK MKVESSQEANAEVMREMTKKLYSQYEEKLQEEQRKHSAEKEALLEETNSFLK corresponding to amino acids 1-261 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 1-261 of T10377_P8 (SEQ ID NO:296), a second amino acid sequence comprising A, and a third amino acid sequence being at least 90% homologous to IEEANKKMQAAEISLEEKDQRIGELDRLIERMEKERHQLQLQLLEHETEMSGELTDSDKERYQQLEEASAS LRERIRHLDDMVHCQQKKVKQMVEEIESLKKKLQQKQLLILQLLEKISFLEGENNELQSRLDYLTETQAKT EVETREIGVGCDLLPSQTGRTREIVMPSRNYTPYTRVLELTMKKTLT corresponding to amino acids 263-451 of Q96NF5 (SEQ ID NO:362), which also corresponds to amino acids 263-451 of T10377_P8 (SEQ ID NO:296), wherein said first, second and third amino acid sequences are contiguous and in a sequential order.","The location of the protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The protein is believed to be located as follows with regard to the cell: secreted. The protein localization is believed to be secreted because one of the two signal-peptide prediction programs (HMM:Signal peptide,NN:NO) predicts that this protein has a signal peptide.","Protein T10377_P8 (SEQ ID NO:296) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 74, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in protein T10377_P8 (SEQ ID NO:296) sequence provides support for the deduced sequence of this protein according to the present invention).",{"@attributes":{"id":"p-1005","num":"1052"},"tables":{"@attributes":{"id":"TABLE-US-00082","num":"00082"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 74"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on",{},{}]},{"entry":["amino acid sequence","Alternative amino acid(s)","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["25","R -> G","No"]},{"entry":["277","V -> A","No"]},{"entry":["338","R -> G","No"]},{"entry":["45","C -> S","No"]},{"entry":["454","T ->","No"]},{"entry":["51","R -> K","No"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Protein T10377_P8 (SEQ ID NO:296) is encoded by the following transcript(s): T10377_T0 (SEQ ID NO:12), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript T10377_T0 (SEQ ID NO:12) is shown in bold; this coding portion starts at position 140 and ends at position 1537. The transcript also has the following SNPs as listed in Table 75 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in protein T10377_P8 (SEQ ID NO:296) sequence provides support for the deduced sequence of this protein according to the present invention).",{"@attributes":{"id":"p-1007","num":"1054"},"tables":{"@attributes":{"id":"TABLE-US-00083","num":"00083"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 75"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on",{},{}]},{"entry":["nucleotide sequence","Alternative nucleic acid","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["13","G -> T","Yes"]},{"entry":["26","G -> A","Yes"]},{"entry":["969","C -> T","No"]},{"entry":["1096","A -> G","No"]},{"entry":["1151","A -> G","No"]},{"entry":["1501","A ->","No"]},{"entry":["88","C -> T","Yes"]},{"entry":["115","G -> A","Yes"]},{"entry":["126","A -> G","Yes"]},{"entry":["212","A -> G","No"]},{"entry":["272","T -> A","No"]},{"entry":["291","G -> A","No"]},{"entry":["643","A -> G","Yes"]},{"entry":["805","G -> A","No"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"As noted above, cluster T10377 features 18 segment(s), which were listed in Table 63 above and for which the sequence(s) are given at the end of the application. These segment(s) are portions of nucleic acid sequence(s) which are described herein separately because they are of particular interest. A description of each segment according to the present invention is now provided.","Segment cluster T10377_node0 (SEQ ID NO:95) according to the present invention is supported by 25 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): T10377_T0 (SEQ ID NO:12), T10377_T2 (SEQ ID NO:14), T10377_T5 (SEQ ID NO:15), T10377_T6 (SEQ ID NO:16) and T10377_T7 (SEQ ID NO:17). Table 76 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1010","num":"1057"},"tables":{"@attributes":{"id":"TABLE-US-00084","num":"00084"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 76"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["T10377_T0 (SEQ ID NO: 12)","1","214"]},{"entry":["T10377_T2 (SEQ ID NO: 14)","1","214"]},{"entry":["T10377_T5 (SEQ ID NO: 15)","1","214"]},{"entry":["T10377_T6 (SEQ ID NO: 16)","1","214"]},{"entry":["T10377_T7 (SEQ ID NO: 17)","1","214"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster T10377_node17 (SEQ ID NO:96) according to the present invention is supported by 36 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): T10377_T0 (SEQ ID NO:12), T10377_T1 (SEQ ID NO:13), T10377_T2 (SEQ ID NO:14), T10377_T5 (SEQ ID NO:15), T10377_T6 (SEQ ID NO:16) and T10377_T7 (SEQ ID NO:17). Table 77 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1012","num":"1059"},"tables":{"@attributes":{"id":"TABLE-US-00085","num":"00085"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 77"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["T10377_T0 (SEQ ID NO: 12)","685","817"]},{"entry":["T10377_T1 (SEQ ID NO: 13)","666","798"]},{"entry":["T10377_T2 (SEQ ID NO: 14)","770","902"]},{"entry":["T10377_T5 (SEQ ID NO: 15)","685","817"]},{"entry":["T10377_T6 (SEQ ID NO: 16)","685","817"]},{"entry":["T10377_T7 (SEQ ID NO: 17)","685","817"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster T10377_node19 (SEQ ID NO:97) according to the present invention is supported by 38 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): T10377_T0 (SEQ ID NO:12), T10377_T1 (SEQ ID NO:13), T10377_T2 (SEQ ID NO:14), T10377_T5 (SEQ ID NO:15), T10377_T6 (SEQ ID NO:16) and T10377_T7 (SEQ ID NO:17). Table 78 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1014","num":"1061"},"tables":{"@attributes":{"id":"TABLE-US-00086","num":"00086"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 78"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["T10377_T0 (SEQ ID NO: 12)","818","943"]},{"entry":["T10377_T1 (SEQ ID NO: 13)","799","924"]},{"entry":["T10377_T2 (SEQ ID NO: 14)","903","1028"]},{"entry":["T10377_T5 (SEQ ID NO: 15)","818","943"]},{"entry":["T10377_T6 (SEQ ID NO: 16)","818","943"]},{"entry":["T10377_T7 (SEQ ID NO: 17)","818","943"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster T10377_node21 (SEQ ID NO:98) according to the present invention is supported by 42 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): T10377_T0 (SEQ ID NO:12), T10377_T1 (SEQ ID NO:13), T10377_T2 (SEQ ID NO:14), T10377_T5 (SEQ ID NO:15), T10377_T6 (SEQ ID NO:16) and T10377_T7 (SEQ ID NO:17). Table 79 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1016","num":"1063"},"tables":{"@attributes":{"id":"TABLE-US-00087","num":"00087"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 79"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["T10377_T0 (SEQ ID NO: 12)","944","1072"]},{"entry":["T10377_T1 (SEQ ID NO: 13)","925","1053"]},{"entry":["T10377_T2 (SEQ ID NO: 14)","1029","1157"]},{"entry":["T10377_T5 (SEQ ID NO: 15)","944","1072"]},{"entry":["T10377_T6 (SEQ ID NO: 16)","944","1072"]},{"entry":["T10377_T7 (SEQ ID NO: 17)","944","1072"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster T10377_node27 (SEQ ID NO:99) according to the present invention is supported by 1 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): T10377_T7 (SEQ ID NO:17). Table 80 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1018","num":"1065"},"tables":{"@attributes":{"id":"TABLE-US-00088","num":"00088"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 80"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["T10377_T7 (SEQ ID NO: 17)","1259","1418"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster T10377_node33 (SEQ ID NO:100) according to the present invention is supported by 103 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): T10377_T0 (SEQ ID NO:12), T10377_T1 (SEQ ID NO:13), T10377_T2 (SEQ ID NO:14), T10377_T5 (SEQ ID NO:15) and T10377_T6 (SEQ ID NO:16). Table 81 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1020","num":"1067"},"tables":{"@attributes":{"id":"TABLE-US-00089","num":"00089"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 81"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["T10377_T0 (SEQ ID NO: 12)","1444","2412"]},{"entry":["T10377_T1 (SEQ ID NO: 13)","1425","2393"]},{"entry":["T10377_T2 (SEQ ID NO: 14)","1529","2497"]},{"entry":["T10377_T5 (SEQ ID NO: 15)","1360","2328"]},{"entry":["T10377_T6 (SEQ ID NO: 16)","1343","2311"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"According to an optional embodiment of the present invention, short segments related to the above cluster are also provided. These segments are up to about 120 bp in length, and so are included in a separate description.","Segment cluster T10377_node12 (SEQ ID NO:101) according to the present invention is supported by 35 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): T10377_T0 (SEQ ID NO:12), T10377_T1 (SEQ ID NO:13), T10377_T2 (SEQ ID NO:14), T10377_T5 (SEQ ID NO:15), T10377_T6 (SEQ ID NO:16) and T10377_T7 (SEQ ID NO:17). Table 82 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1023","num":"1070"},"tables":{"@attributes":{"id":"TABLE-US-00090","num":"00090"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 82"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["T10377_T0 (SEQ ID NO: 12)","458","550"]},{"entry":["T10377_T1 (SEQ ID NO: 13)","439","531"]},{"entry":["T10377_T2 (SEQ ID NO: 14)","543","635"]},{"entry":["T10377_T5 (SEQ ID NO: 15)","458","550"]},{"entry":["T10377_T6 (SEQ ID NO: 16)","458","550"]},{"entry":["T10377_T7 (SEQ ID NO: 17)","458","550"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster T10377_node14 (SEQ ID NO:102) according to the present invention is supported by 28 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): T10377_T0 (SEQ ID NO:12), T10377_T1 (SEQ ID NO:13), T10377_T2 (SEQ ID NO:14), T10377_T5 (SEQ ID NO:15), T10377_T6 (SEQ ID NO:16) and T10377_T7 (SEQ ID NO:17). Table 83 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1025","num":"1072"},"tables":{"@attributes":{"id":"TABLE-US-00091","num":"00091"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 83"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["T10377_T0 (SEQ ID NO: 12)","551","664"]},{"entry":["T10377_T1 (SEQ ID NO: 13)","532","645"]},{"entry":["T10377_T2 (SEQ ID NO: 14)","636","749"]},{"entry":["T10377_T5 (SEQ ID NO: 15)","551","664"]},{"entry":["T10377_T6 (SEQ ID NO: 16)","551","664"]},{"entry":["T10377_T7 (SEQ ID NO: 17)","551","664"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster T10377_node16 (SEQ ID NO:103) according to the present invention can be found in the following transcript(s): T10377_T0 (SEQ ID NO:12), T10377_T1 (SEQ ID NO:13), T10377_T2 (SEQ ID NO:14), T10377_T5 (SEQ ID NO:15), T10377_T6 (SEQ ID NO:16) and T10377_T7 (SEQ ID NO:17). Table 84 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1027","num":"1074"},"tables":{"@attributes":{"id":"TABLE-US-00092","num":"00092"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 84"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["T10377_T0 (SEQ ID NO: 12)","665","684"]},{"entry":["T10377_T1 (SEQ ID NO: 13)","646","665"]},{"entry":["T10377_T2 (SEQ ID NO: 14)","750","769"]},{"entry":["T10377_T5 (SEQ ID NO: 15)","665","684"]},{"entry":["T10377_T6 (SEQ ID NO: 16)","665","684"]},{"entry":["T10377_T7 (SEQ ID NO: 17)","665","684"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster T10377_node2 (SEQ ID NO:104) according to the present invention is supported by 3 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): T10377_T1 (SEQ ID NO:13). Table 85 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1029","num":"1076"},"tables":{"@attributes":{"id":"TABLE-US-00093","num":"00093"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 85"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["T10377_T1 (SEQ ID NO: 13)","1","110"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster T10377_node23 (SEQ ID NO:105) according to the present invention is supported by 44 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): T10377_T0 (SEQ ID NO:12), T10377_T1 (SEQ ID NO:13), T10377_T2 (SEQ ID NO:14), T10377_T5 (SEQ ID NO:15), T10377_T6 (SEQ ID NO:16) and T10377_T7 (SEQ ID NO:17). Table 86 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1031","num":"1078"},"tables":{"@attributes":{"id":"TABLE-US-00094","num":"00094"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 86"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["T10377_T0 (SEQ ID NO: 12)","1073","1152"]},{"entry":["T10377_T1 (SEQ ID NO: 13)","1054","1133"]},{"entry":["T10377_T2 (SEQ ID NO: 14)","1158","1237"]},{"entry":["T10377_T5 (SEQ ID NO: 15)","1073","1152"]},{"entry":["T10377_T6 (SEQ ID NO: 16)","1073","1152"]},{"entry":["T10377_T7 (SEQ ID NO: 17)","1073","1152"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster T10377_node25 (SEQ ID NO:106) according to the present invention is supported by 50 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): T10377_T0 (SEQ ID NO:12), T10377_T1 (SEQ ID NO:13), T10377_T2 (SEQ ID NO:14), T10377_T5 (SEQ ID NO:15), T10377_T6 (SEQ ID NO:16) and T10377_T7 (SEQ ID NO:17). Table 87 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1033","num":"1080"},"tables":{"@attributes":{"id":"TABLE-US-00095","num":"00095"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 87"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["T10377_T0 (SEQ ID NO: 12)","1153","1258"]},{"entry":["T10377_T1 (SEQ ID NO: 13)","1134","1239"]},{"entry":["T10377_T2 (SEQ ID NO: 14)","1238","1343"]},{"entry":["T10377_T5 (SEQ ID NO: 15)","1153","1258"]},{"entry":["T10377_T6 (SEQ ID NO: 16)","1153","1258"]},{"entry":["T10377_T7 (SEQ ID NO: 17)","1153","1258"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster T10377_node29 (SEQ ID NO:107) according to the present invention is supported by 50 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): T10377_T0 (SEQ ID NO:12), T10377_T1 (SEQ ID NO:13), T10377_T2 (SEQ ID NO:14) and T10377_T6 (SEQ ID NO:16). Table 88 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1035","num":"1082"},"tables":{"@attributes":{"id":"TABLE-US-00096","num":"00096"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 88"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["T10377_T0 (SEQ ID NO: 12)","1259","1342"]},{"entry":["T10377_T1 (SEQ ID NO: 13)","1240","1323"]},{"entry":["T10377_T2 (SEQ ID NO: 14)","1344","1427"]},{"entry":["T10377_T6 (SEQ ID NO: 16)","1259","1342"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster T10377_node3 (SEQ ID NO:108) according to the present invention is supported by 4 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): T10377_T1 (SEQ ID NO:13) and T10377_T2 (SEQ ID NO:14). Table 89 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1037","num":"1084"},"tables":{"@attributes":{"id":"TABLE-US-00097","num":"00097"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 89"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["T10377_T1 (SEQ ID NO: 13)","111","195"]},{"entry":["T10377_T2 (SEQ ID NO: 14)","215","299"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster T10377_node31 (SEQ ID NO:109) according to the present invention is supported by 52 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): T10377_T0 (SEQ ID NO:12), T10377_T1 (SEQ ID NO:13), T10377_T2 (SEQ ID NO:14) and T10377_T5 (SEQ ID NO:15). Table 90 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1039","num":"1086"},"tables":{"@attributes":{"id":"TABLE-US-00098","num":"00098"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 90"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["T10377_T0 (SEQ ID NO: 12)","1343","1443"]},{"entry":["T10377_T1 (SEQ ID NO: 13)","1324","1424"]},{"entry":["T10377_T2 (SEQ ID NO: 14)","1428","1528"]},{"entry":["T10377_T5 (SEQ ID NO: 15)","1259","1359"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster T10377_node5 (SEQ ID NO:110) according to the present invention is supported by 30 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): T10377_T0 (SEQ ID NO:12), T10377_T1 (SEQ ID NO:13), T10377_T2 (SEQ ID NO:14), T10377_T5 (SEQ ID NO:15), T10377_T6 (SEQ ID NO:16) and T10377_T7 (SEQ ID NO:17). Table 91 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1041","num":"1088"},"tables":{"@attributes":{"id":"TABLE-US-00099","num":"00099"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 91"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["T10377_T0 (SEQ ID NO: 12)","215","301"]},{"entry":["T10377_T1 (SEQ ID NO: 13)","196","282"]},{"entry":["T10377_T2 (SEQ ID NO: 14)","300","386"]},{"entry":["T10377_T5 (SEQ ID NO: 15)","215","301"]},{"entry":["T10377_T6 (SEQ ID NO: 16)","215","301"]},{"entry":["T10377_T7 (SEQ ID NO: 17)","215","301"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster T10377_node8 (SEQ ID NO:111) according to the present invention is supported by 35 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): T10377_T0 (SEQ ID NO:12), T10377_T1 (SEQ ID NO:13), T10377_T2 (SEQ ID NO:14), T10377_T5 (SEQ ID NO:15), T10377_T6 (SEQ ID NO:16) and T10377_T7 (SEQ ID NO:17). Table 92 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1043","num":"1090"},"tables":{"@attributes":{"id":"TABLE-US-00100","num":"00100"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 92"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["T10377_T0 (SEQ ID NO: 12)","302","407"]},{"entry":["T10377_T1 (SEQ ID NO: 13)","283","388"]},{"entry":["T10377_T2 (SEQ ID NO: 14)","387","492"]},{"entry":["T10377_T5 (SEQ ID NO: 15)","302","407"]},{"entry":["T10377_T6 (SEQ ID NO: 16)","302","407"]},{"entry":["T10377_T7 (SEQ ID NO: 17)","302","407"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster T10377_node9 (SEQ ID NO:112) according to the present invention is supported by 35 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): T10377_T0 (SEQ ID NO:12), T10377_T1 (SEQ ID NO:13), T10377_T2 (SEQ ID NO:14), T10377_T5 (SEQ ID NO:15), T10377_T6 (SEQ ID NO:16) and T10377_T7 (SEQ ID NO:17). Table 93 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1045","num":"1092"},"tables":{"@attributes":{"id":"TABLE-US-00101","num":"00101"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 93"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["T10377_T0 (SEQ ID NO: 12)","408","457"]},{"entry":["T10377_T1 (SEQ ID NO: 13)","389","438"]},{"entry":["T10377_T2 (SEQ ID NO: 14)","493","542"]},{"entry":["T10377_T5 (SEQ ID NO: 15)","408","457"]},{"entry":["T10377_T6 (SEQ ID NO: 16)","408","457"]},{"entry":["T10377_T7 (SEQ ID NO: 17)","408","457"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-1046","num":"1093"},"tables":{"@attributes":{"id":"TABLE-US-00102","num":"00102"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment of: T10377_P2 (SEQ ID NO: 292) \u00d7 Q96NFS (SEQ ID NO: 362)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","4288.00","Escore:","0"]},{"entry":["Matching length:","441","Total length:","441"]},{"entry":["Matching Percent Similarity:","99.77","Matching Percent Identity:","99.77"]},{"entry":["Total Percent Similarity:","99.77","Total Percent Identity:","99.77"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00019","num":"00019"},"img":{"@attributes":{"id":"EMI-C00019","he":"42.59mm","wi":"84.92mm","file":"US07714100-20100511-C00019.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00020","num":"00020"},"img":{"@attributes":{"id":"EMI-C00020","he":"53.42mm","wi":"85.01mm","file":"US07714100-20100511-C00020.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Alignment of: T10377_P5 (SEQ ID NO: 293) \u00d7 Q96NF5 (SEQ ID NO: 362)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","4159.00","Escore:","0"]},{"entry":["Matching length:","438","Total length:","466"]},{"entry":["Matching Percent Similarity:","99.77","Matching Percent Identity:","99.77"]},{"entry":["Total Percent Similarity:","93.78","Total Percent Identity:","93.78"]},{"entry":["Gaps:","1"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00021","num":"00021"},"img":{"@attributes":{"id":"EMI-C00021","he":"64.43mm","wi":"85.09mm","file":"US07714100-20100511-C00021.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00022","num":"00022"},"img":{"@attributes":{"id":"EMI-C00022","he":"42.76mm","wi":"84.92mm","file":"US07714100-20100511-C00022.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Alignment of: T10377_P6 (SEQ ID NO: 294) \u00d7 Q96NF5 (SEQ ID NO: 362)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","3896.00","Escore:","0"]},{"entry":["Matching length:","403","Total length:","403"]},{"entry":["Matching Percent Similarity:","99.50","Matching Percent Identity:","99.50"]},{"entry":["Total Percent Similarity:","99.50","Total Percent Identity:","99.50"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00023","num":"00023"},"img":{"@attributes":{"id":"EMI-C00023","he":"53.59mm","wi":"85.01mm","file":"US07714100-20100511-C00023.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00024","num":"00024"},"img":{"@attributes":{"id":"EMI-C00024","he":"42.76mm","wi":"84.92mm","file":"US07714100-20100511-C00024.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Alignment of: T10377_P7 (SEQ ID NO: 295) \u00d7 Q96NF5 (SEQ ID NO: 362)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","3642.00","Escore:","0"]},{"entry":["Matching length:","376","Total length:","376"]},{"entry":["Matching Percent Similarity:","99.47","Matching Percent Identity:","99.47"]},{"entry":["Total Percent Similarity:","99.47","Total Percent Identity:","99.47"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00025","num":"00025"},"img":{"@attributes":{"id":"EMI-C00025","he":"42.67mm","wi":"85.01mm","file":"US07714100-20100511-C00025.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00026","num":"00026"},"img":{"@attributes":{"id":"EMI-C00026","he":"42.76mm","wi":"85.09mm","file":"US07714100-20100511-C00026.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Alignment of: T10377_P8 (SEQ ID NO: 296) \u00d7 Q96NF5 (SEQ ID NO: 362)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","4519.00","Escore:","0"]},{"entry":["Matching length:","465","Total length:","466"]},{"entry":["Matching Percent Similarity:","99","Matching Percent Identity:","99"]},{"entry":["Total Percent Similarity:","99","Total Percent Identity:","99"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00027","num":"00027"},"img":{"@attributes":{"id":"EMI-C00027","he":"42.67mm","wi":"85.01mm","file":"US07714100-20100511-C00027.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00028","num":"00028"},"img":{"@attributes":{"id":"EMI-C00028","he":"64.43mm","wi":"85.09mm","file":"US07714100-20100511-C00028.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}]}}},"Expression of Q96NF5 transcripts detectable by or according to junc25-31 node(s), T10377 amplicon(s) and T10377 junc25-31F (SEQ ID NO:363) and T10377 junc25-31R (SEQ ID NO:364) primers was measured by real time PCR. In parallel the expression of four housekeeping genes\u2014RPL19 (GenBank Accession No. NM000981 (SEQ ID NO:437); RPL19 amplicon (SEQ ID NO:440)), TATA box (GenBank Accession No. NM003194 (SEQ ID NO:441); TATA amplicon (SEQ ID NO:444)), Ubiquitin (GenBank Accession No. BC000449 (SEQ ID NO:445); amplicon\u2014Ubiquitin-amplicon (SEQ ID NO:448)) and SDHA (GenBank Accession No. NM004168 (SEQ ID NO:449); amplicon\u2014SDHA-amplicon (SEQ ID NO:452)), was measured similarly. For each RT sample, the expression of the above amplicons was normalized to the geometric mean of the quantities of the housekeeping genes. The normalized quantity of each RT sample was then divided by the median of the quantities of the heart samples (Sample Nos. 44, 45, 46, Table 2, above \u201cTissue samples in testing panel\u201d), to obtain a value of fold up-regulation for each sample relative to median of the heart.",{"@attributes":{"id":"p-1048","num":"1095"},"figref":"FIG. 12"},"As is evident from , the expression of Q96NF5 transcripts detectable by the above amplicon(s) in heart tissue samples was significantly higher than in most other samples (non-heart tissue sample Nos. 1-26, 28-43, 47-74 Table 2, above \u201cTissue samples in testing panel\u201d).","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: T10377 junc25-31F (SEQ ID NO:363) forward primer; and T10377 junc25-31R (SEQ ID NO:364) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: T10377 junc25-31 (SEQ ID NO:365).",{"@attributes":{"id":"p-1052","num":"1099"},"tables":{"@attributes":{"id":"TABLE-US-00103","num":"00103"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"287pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["T10377 junc25-31F (SEQ ID NO: 363):",{}]},{"entry":"AGCAGATGGTCGAGGAGAATAATG"},{"entry":{}},{"entry":"T10377 junc25-31R (SEQ ID NO: 364):"},{"entry":"ATCTCTCTGGTTTCCACTTCGG"},{"entry":{}},{"entry":"T10377 junc25-31 (SEQ ID NO: 365):"},{"entry":"AGCAGATGGTCGAGGAGAATAATGAACTACAAAGCAGGTTGGACTATTTAACAGAAACCCAGGCCA"},{"entry":{}},{"entry":"AGACCGAAGTGGAAACCAGAGAGAT"}]}}}}},"Expression of Q96NF5 transcripts detectable by or according to junc29-33 node(s), T10377 amplicon(s) and T10377 junc29-33F (SEQ ID NO:366) and T10377 junc29-33R (SEQ ID NO:367) primers was measured by real time PCR. In parallel the expression of four housekeeping genes\u2014RPL19 (GenBank Accession No. NM000981 (SEQ ID NO:437); RPL19 amplicon (SEQ ID NO:440)), TATA box (GenBank Accession No. NM003194 (SEQ ID NO:441); TATA amplicon (SEQ ID NO:444)), Ubiquitin (GenBank Accession No. BC000449 (SEQ ID NO:445); amplicon\u2014Ubiquitin-amplicon (SEQ ID NO:448)) and SDHA (GenBank Accession No. NM004168 (SEQ ID NO:449); amplicon\u2014SDHA-amplicon (SEQ ID NO:452)), was measured similarly. For each RT sample, the expression of the above amplicons was normalized to the geometric mean of the quantities of the housekeeping genes. The normalized quantity of each RT sample was then divided by the median of the quantities of the heart samples (Sample Nos. 44, 45, 46, Table 2, above \u201cTissue samples in testing panel\u201d), to obtain a value of fold up-regulation for each sample relative to median of the heart.",{"@attributes":{"id":"p-1054","num":"1101"},"figref":"FIG. 13"},"As is evident from , the expression of Q96NF5 transcripts detectable by the above amplicon(s) in heart tissue samples was significantly higher than in most other samples (non-heart tissue sample Nos. 1-26, 28-43, 47-74 Table 2 above \u201cTissue samples in testing panel\u201d).","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: T10377 junc29-33F (SEQ ID NO:366) forward primer; and T10377 junc29-33R (SEQ ID NO:367) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: T10377 junc29-33 (SEQ ID NO:368).",{"@attributes":{"id":"p-1058","num":"1105"},"tables":{"@attributes":{"id":"TABLE-US-00104","num":"00104"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"287pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["T10377 junc29-33F (SEQ ID NO: 366):",{}]},{"entry":"CTTTCTTAGAAGGAGAGCCAAACAG"},{"entry":{}},{"entry":"T10377 junc29-33R (SEQ ID NO: 367):"},{"entry":"CCTAAGTCAGAGTTTTCTTCATGGTTAAC"},{"entry":{}},{"entry":"T10377 junc29-33 (SEQ ID NO: 368):"},{"entry":"CTTTCTTAGAAGGAGAGCCAAACAGGCAGGACTCGTGAAATTGTGATGCCTTCTAGGAACTACACCC"},{"entry":{}},{"entry":"CATACACAAGAGTCCTGGAGTTAACCATGAAGAAAACTCTGACTTAGG"}]}}}}},"Expression of Q96NF5 transcripts detectable by or according to seg2-3 node(s), T10377 amplicon(s) and T10377 seg2-3F (SEQ ID NO:369) and T10377 seg2-3R (SEQ ID NO:370) primers was measured by real time PCR. In parallel the expression of four housekeeping genes\u2014RPL19 (GenBank Accession No. NM000981 (SEQ ID NO:437); RPL19 amplicon (SEQ ID NO:440)), TATA box (GenBank Accession No. NM003194 (SEQ ID NO:441); TATA amplicon (SEQ ID NO:444)), Ubiquitin (GenBank Accession No. BC000449 (SEQ ID NO:445); amplicon\u2014Ubiquitin-amplicon (SEQ ID NO:448)) and SDHA (GenBank Accession No. NM004168 (SEQ ID NO:449); amplicon\u2014SDHA-amplicon (SEQ ID NO:452)), was measured similarly. For each RT sample, the expression of the above amplicons was normalized to the geometric mean of the quantities of the housekeeping genes. The normalized quantity of each RT sample was then divided by the median of the quantities of the heart samples (Sample Nos. 44, 45, 46, Table 2, above \u201cTissue samples in testing panel\u201d), to obtain a value of fold up-regulation for each sample relative to median of the heart.",{"@attributes":{"id":"p-1060","num":"1107"},"figref":"FIG. 14"},"As is evident from , the expression of Q96NF5 transcripts detectable by the above amplicon(s) in heart tissue samples was significantly higher than in the skeletal muscle (non-heart tissue sample Nos. 1-9, 13-26, 28-43, 47-74 Table 2, \u201cTissue samples in testing samples\u201d).","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: T10377 seg2-3F (SEQ ID NO:369) forward primer; and T10377 seg2-3R (SEQ ID NO:370) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: T10377 seg2-3 (SEQ ID NO:371).",{"@attributes":{"id":"p-1064","num":"1111"},"tables":{"@attributes":{"id":"TABLE-US-00105","num":"00105"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"294pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"147pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["T10377 seg2-3F (SEQ ID NO: 369):",{}]},{"entry":"CTTCGCATTGTGCATAACACAA"},{"entry":{}},{"entry":"T10377 seg2-3R (SEQ ID NO: 370):"},{"entry":"GAAACTCGGATACACAATCTCCAGA"},{"entry":{}},{"entry":"T10377 seg2-3 (SEQ ID NO: 371):"},{"entry":"CTTCGCATTGTGCATAACACAAGCCCTGAACCAGCTGCTTTGGGAACCCCTGGGAATAAAGTGCCCTA"},{"entry":{}},{"entry":"CCTGCCTTTCAGGCACTGCCAAGCCTGGGGCATCTCTGGAGATTGTGTATCCGAGTTTC"}]}}}}},"Cluster Z24874 features 2 transcript(s) and 10 segment(s) of interest, the names for which are given in Tables 94 and 95, respectively, the sequences themselves are given at the end of the application. The selected protein variants are given in table 96.",{"@attributes":{"id":"p-1066","num":"1113"},"tables":{"@attributes":{"id":"TABLE-US-00106","num":"00106"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 94"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Transcripts of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Transcript Name","Seq ID No."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Z24874_PEA_2_T10","18"]},{"entry":[{},"Z24874_PEA_2_T11","19"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-1067","num":"1114"},"tables":{"@attributes":{"id":"TABLE-US-00107","num":"00107"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 95"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segments of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"98pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment Name","Seq ID No."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"98pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Z24874_PEA_2_node_21","113"]},{"entry":[{},"Z24874_PEA_2_node_4","114"]},{"entry":[{},"Z24874_PEA_2_node_0","115"]},{"entry":[{},"Z24874_PEA_2_node_10","116"]},{"entry":[{},"Z24874_PEA_2_node_12","117"]},{"entry":[{},"Z24874_PEA_2_node_13","118"]},{"entry":[{},"Z24874_PEA_2_node_14","119"]},{"entry":[{},"Z24874_PEA_2_node_16","120"]},{"entry":[{},"Z24874_PEA_2_node_3","121"]},{"entry":[{},"Z24874_PEA_2_node_6","122"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-1068","num":"1115"},"tables":{"@attributes":{"id":"TABLE-US-00108","num":"00108"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 96"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Proteins of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Protein Name","Seq ID No."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Z24874_PEA_2_P5","297"]},{"entry":[{},"Z24874_PEA_2_P6","298"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The heart-selective diagnostic marker prediction engine provided the following results with regard to cluster Z24874. Predictions were made for selective expression of transcripts of this cluster in heart tissue, according to the previously described methods. The numbers on the y-axis of  refer to weighted expression of ESTs in each category, as \u201cparts per million\u201d (ratio of the expression of ESTs for a particular cluster to the expression of all ESTs in that category, according to parts per million).","Overall, the following results were obtained as shown with regard to the histogram in , concerning the number of heart-specific clones in libraries\/sequences; as well as with regard to the histogram in , concerning the actual expression of oligonucleotides in various tissues, including heart.","This cluster was found to be selectively expressed in heart for the following reasons: in a comparison of the ratio of expression of the cluster in heart specific ESTs to the overall expression of the cluster in non-heart ESTs, which was found to be 16.7; the ratio of expression of the cluster in heart specific ESTs to the overall expression of the cluster in muscle-specific ESTs which was found to be 2.1; and fisher exact test P-values were computed both for library and weighted clone counts to check that the counts are statistically significant, and were found to be 3.20E-09.","One particularly important measure of specificity of expression of a cluster in heart tissue is the previously described comparison of the ratio of expression of the cluster in heart as opposed to muscle. This cluster was found to be specifically expressed in heart as opposed to non-heart ESTs as described above. However, many proteins have been shown to be generally expressed at a higher level in both heart and muscle, which is less desirable. For this cluster, as described above, the ratio of expression of the cluster in heart specific ESTs to the overall expression of the cluster in muscle-specific ESTs which was found to be 2.1, which clearly supports specific expression in heart tissue.","As noted above, cluster Z24874 features 2 transcript(s), which were listed in Table 94 above. A description of each variant protein according to the present invention is now provided.","Variant protein Z24874_PEA2_P5 (SEQ ID NO:297) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) Z24874_PEA2_T10 (SEQ ID NO:18). One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between Z24874_PEA2_P5 (SEQ ID NO:297) and Q9NP15 (SEQ ID NO:372):","1. An isolated chimeric polypeptide encoding for Z24874_PEA2_P5 (SEQ ID NO:297), comprising a first amino acid sequence being at least 90% homologous to MKLIVGIGGMTNGGKTTLTNSLLRALPNCCVIHQDDFFKPQDQIAVGEDGFKQWDVLESLDMEAMLDTV QAWLSSPQKFARAHGVSVQPEASDTHILLLEGFLLYSYKPLVDLYSRRYFLTVPYEECKWRRS corresponding to amino acids 1-132 of Q9NPI5 (SEQ ID NO:372), which also corresponds to amino acids 1-132 of Z24874_PEA2_P5 (SEQ ID NO:297), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence LPGRHEVPRGALP (SEQ ID NO:409) corresponding to amino acids 133-145 of Z24874_PEA2_P5 (SEQ ID NO:297), wherein said first and second amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of Z24874_PEA2_P5 (SEQ ID NO:297), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence LPGRHEVPRGALP (SEQ ID NO:409) in Z24874_PEA2_P5 (SEQ ID NO:297).","Comparison report between Z24874_PEA2_P5 (SEQ ID NO:297) and Q9NZK3 (SEQ ID NO:373):","1. An isolated chimeric polypeptide encoding for Z24874_PEA2_P5 (SEQ ID NO:297), comprising a first amino acid sequence being at least 90% homologous to MKLIVGIGGMTNGGKTTLTNSLLRALPNCCVIHQDDFFKPQDQIAVGEDGFKQWDVLESLDMEAMLDTV QAWLSSPQKFARAHGVSVQPEASDTHILLLEGFLLYSYKP corresponding to amino acids 1-109 of Q9NZK3 (SEQ ID NO:373), which also corresponds to amino acids 1-109 of Z24874_PEA2_P5 (SEQ ID NO:297), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence LVDLYSRRYFLTVPYEECKWRRSLPGRHEVPRGALP corresponding to amino acids 110-145 of Z24874_PEA2_P5 (SEQ ID NO:297), wherein said first and second amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of Z24874_PEA2_P5 (SEQ ID NO:297), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence LVDLYSRRYFLTVPYEECKWRRSLPGRHEVPRGALP in Z24874_PEA2_P5 (SEQ ID NO:297)","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: intracellularly. The protein localization is believed to be intracellularly because neither of the trans-membrane region prediction programs predicted a trans-membrane region for this protein. In addition both signal-peptide prediction programs predict that this protein is a non-secreted protein.","Variant protein Z24874_PEA2_P5 (SEQ ID NO:297) is encoded by the following transcript(s): Z24874_PEA2_T10 (SEQ ID NO:18), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript Z24874_PEA2_T10 (SEQ ID NO:18) is shown in bold; this coding portion starts at position 292 and ends at position 726. The transcript also has the following SNPs as listed in Table 97 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein Z24874_PEA2_P5 (SEQ ID NO:297) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-1083","num":"1130"},"tables":{"@attributes":{"id":"TABLE-US-00109","num":"00109"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 97"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on nucleotide",{},"Previously"]},{"entry":["sequence","Alternative nucleic acid","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1","G -> C","No"]},{"entry":["70","G -> A","Yes"]},{"entry":["504","C -> T","No"]},{"entry":["645","C -> T","Yes"]},{"entry":["954","C -> T","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein Z24874_PEA2_P6 (SEQ ID NO:298) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) Z24874_PEA2_T11 (SEQ ID NO:19). One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between Z24874_PEA2_P6 (SEQ ID NO:298) and Q9NPI5 (SEQ ID NO:372):","1. An isolated chimeric polypeptide encoding for Z24874_PEA2_P6 (SEQ ID NO:298), comprising a first amino acid sequence being at least 90% homologous to MKLIVGIGGMTNGGKTTLTNSLLRALPNCCVIHQDDFFKPQDQIAVGEDGFKQWDVLESLDMEAMLDTV QAWLSSPQKFARAHGVSVQPEASDTHILLLEGFLLYSY corresponding to amino acids 1-107 of Q9NPI5 (SEQ ID NO:372), which also corresponds to amino acids 1-107 of Z24874_PEA2_P6 (SEQ ID NO:298), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence NLPGRHEVPRGALP (SEQ ID NO:410) corresponding to amino acids 108-121 of Z24874_PEA2_P6 (SEQ ID NO:298), wherein said first and second amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of Z24874_PEA2_P6 (SEQ ID NO:298), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence NLPGRHEVPRGALP (SEQ ID NO:410) in Z24874_PEA2_P6 (SEQ ID NO:298).","Comparison report between Z24874_PEA2_P6 (SEQ ID NO:298) and Q9NZK3 (SEQ ID NO:373):","1. An isolated chimeric polypeptide encoding for Z24874_PEA2_P6 (SEQ ID NO:298), comprising a first amino acid sequence being at least 90% homologous to MKLIVGIGGMTNGGKTTLTNSLLRALPNCCVIHQDDFFKPQDQIAVGEDGFKQWDVLESLDMEAMLDTV QAWLSSPQKFARAHGVSVQPEASDTHILLLEGFLLYSY corresponding to amino acids 1-107 of Q9NZK3 (SEQ ID NO:372), which also corresponds to amino acids 1-107 of Z24874_PEA2_P6 (SEQ ID NO:298), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence NLPGRHEVPRGALP (SEQ ID NO:410) corresponding to amino acids 108-121 of Z24874_PEA2_P6 (SEQ ID NO:298), wherein said first and second amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of Z24874_PEA2_P6 (SEQ ID NO:298), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence NLPGRHEVPRGALP (SEQ ID NO:410) in Z24874_PEA2_P6 (SEQ ID NO:298).","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: intracellularly. The protein localization is believed to be intracellularly because neither of the trans-membrane region prediction programs predicted a trans-membrane region for this protein. In addition both signal-peptide prediction programs predict that this protein is a non-secreted protein.","Variant protein Z24874_PEA2_P6 (SEQ ID NO:298) is encoded by the following transcript(s): Z24874_PEA2_T11 (SEQ ID NO:19), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript Z24874_PEA2_T11 (SEQ ID NO:19) is shown in bold; this coding portion starts at position 292 and ends at position 654. The transcript also has the following SNPs as listed in Table 98 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein Z24874_PEA2_P6 (SEQ ID NO:298) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-1093","num":"1140"},"tables":{"@attributes":{"id":"TABLE-US-00110","num":"00110"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 98"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on nucleotide",{},"Previously"]},{"entry":["sequence","Alternative nucleic acid","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1","G -> C","No"]},{"entry":["70","G -> A","Yes"]},{"entry":["504","C -> T","No"]},{"entry":["882","C -> T","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"As noted above, cluster Z24874 features 10 segment(s), which were listed in Table 95 above and for which the sequence(s) are given at the end of the application. These segment(s) are portions of nucleic acid sequence(s) which are described herein separately because they are of particular interest. A description of each segment according to the present invention is now provided.","Segment cluster Z24874_PEA2_node21 (SEQ ID NO:113) according to the present invention is supported by 30 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): Z24874_PEA2_T10 (SEQ ID NO:18) and Z24874_PEA2_T11 (SEQ ID NO:19). Table 99 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1096","num":"1143"},"tables":{"@attributes":{"id":"TABLE-US-00111","num":"00111"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 99"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment",{}]},{"entry":[{},"starting","Segment"]},{"entry":["Transcript name","position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Z24874_PEA_2_T10 (SEQ ID NO: 18)","687","1027"]},{"entry":["Z24874_PEA_2_T11 (SEQ ID NO: 19)","615","955"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}},"br":{},"sub":["\u2014","\u2014","\u2014","\u2014"]},{"@attributes":{"id":"p-1097","num":"1144"},"tables":{"@attributes":{"id":"TABLE-US-00112","num":"00112"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 100"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment",{}]},{"entry":[{},"starting","Segment"]},{"entry":["Transcript name","position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Z24874_PEA_2_T10 (SEQ ID NO: 18)","138","317"]},{"entry":["Z24874_PEA_2_T11 (SEQ ID NO: 19)","138","317"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"According to an optional embodiment of the present invention, short segments related to the above cluster are also provided. These segments are up to about 120 bp in length, and so are included in a separate description.","Segment cluster Z24874_PEA2_node0 (SEQ ID NO:115) according to the present invention is supported by 3 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): Z24874_PEA2_T10 (SEQ ID NO:18) and Z24874_PEA2_T11 (SEQ ID NO:19). Table 101 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1100","num":"1147"},"tables":{"@attributes":{"id":"TABLE-US-00113","num":"00113"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 101"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Segment"]},{"entry":[{},"Segment","ending"]},{"entry":["Transcript name","starting position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Z24874_PEA_2_T10 (SEQ ID NO: 18)","1","77"]},{"entry":["Z24874_PEA_2_T11 (SEQ ID NO: 19)","1","77"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster Z24874_PEA2_node10 (SEQ ID NO:116) according to the present invention is supported by 25 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): Z24874_PEA2_T10 (SEQ ID NO:18) and Z24874_PEA2_T11 (SEQ ID NO:19). Table 102 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1102","num":"1149"},"tables":{"@attributes":{"id":"TABLE-US-00114","num":"00114"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 102"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Segment"]},{"entry":[{},"Segment","ending"]},{"entry":["Transcript name","starting position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Z24874_PEA_2_T10 (SEQ ID NO: 18)","409","457"]},{"entry":["Z24874_PEA_2_T11 (SEQ ID NO: 19)","409","457"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster Z24874_PEA2_node12 (SEQ ID NO:117) according to the present invention is supported by 26 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): Z24874_PEA2_T10 (SEQ ID NO:18) and Z24874_PEA2_T11 (SEQ ID NO:19). Table 103 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1104","num":"1151"},"tables":{"@attributes":{"id":"TABLE-US-00115","num":"00115"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 103"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment",{}]},{"entry":[{},"starting","Segment"]},{"entry":["Transcript name","position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Z24874_PEA_2_T10 (SEQ ID NO: 18)","458","524"]},{"entry":["Z24874_PEA_2_T11 (SEQ ID NO: 19)","458","524"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster Z24874_PEA2_node13 (SEQ ID NO:118) according to the present invention is supported by 21 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): Z24874_PEA2_T10 (SEQ ID NO:18) and Z24874_PEA2_T11 (SEQ ID NO:19). Table 104 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1106","num":"1153"},"tables":{"@attributes":{"id":"TABLE-US-00116","num":"00116"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 104"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment",{}]},{"entry":[{},"starting","Segment"]},{"entry":["Transcript name","position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Z24874_PEA_2_T10 (SEQ ID NO: 18)","525","561"]},{"entry":["Z24874_PEA_2_T11 (SEQ ID NO: 19)","525","561"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster Z24874_PEA2_node14 (SEQ ID NO:119) according to the present invention is supported by 20 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): Z24874_PEA2_T10 (SEQ ID NO:18) and Z24874_PEA2_T11 (SEQ ID NO:19). Table 105 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1108","num":"1155"},"tables":{"@attributes":{"id":"TABLE-US-00117","num":"00117"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 105"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment",{}]},{"entry":[{},"starting","Segment"]},{"entry":["Transcript name","position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Z24874_PEA_2_T10 (SEQ ID NO: 18)","562","614"]},{"entry":["Z24874_PEA_2_T11 (SEQ ID NO: 19)","562","614"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster Z24874_PEA2_node16 (SEQ ID NO:120) according to the present invention is supported by 17 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): Z24874_PEA2_T10 (SEQ ID NO:18). Table 106 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1110","num":"1157"},"tables":{"@attributes":{"id":"TABLE-US-00118","num":"00118"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 106"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment",{}]},{"entry":[{},"starting","Segment"]},{"entry":["Transcript name","position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Z24874_PEA_2_T10 (SEQ ID NO: 18)","615","686"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster Z24874_PEA2_node3 (SEQ ID NO:121) according to the present invention is supported by 8 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): Z24874_PEA2_T10 (SEQ ID NO:18) and Z24874_PEA2_T11 (SEQ ID NO:19). Table 107 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1112","num":"1159"},"tables":{"@attributes":{"id":"TABLE-US-00119","num":"00119"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 107"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment",{}]},{"entry":[{},"starting","Segment"]},{"entry":["Transcript name","position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Z24874_PEA_2_T10 (SEQ ID NO: 18)","78","137"]},{"entry":["Z24874_PEA_2_T11 (SEQ ID NO: 19)","78","137"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster Z24874_PEA2_node6 (SEQ ID NO:122) according to the present invention is supported by 23 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): Z24874_PEA2_T10 (SEQ ID NO:18) and Z24874_PEA2_T11 (SEQ ID NO:19). Table 108 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1114","num":"1161"},"tables":{"@attributes":{"id":"TABLE-US-00120","num":"00120"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 108"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Segment"]},{"entry":[{},"Segment","ending"]},{"entry":["Transcript name","starting position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Z24874_PEA_2_T10 (SEQ ID NO: 18)","318","408"]},{"entry":["Z24874_PEA_2_T11 (SEQ ID NO: 19)","318","408"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}},"br":{}},{"@attributes":{"id":"p-1115","num":"1162"},"tables":{"@attributes":{"id":"TABLE-US-00121","num":"00121"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"273pt","align":"left"}},"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Sequence name: \/tmp\/Ro5LG3OhE3\/oQvcWauNWJ:Q9NPI5 (SEQ ID NO: 372)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: Z24874_PEA_2_P5 (SEQ ID NO: 297) \u00d7 Q9NPI5 (SEQ ID NO: 372)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","1307.00","Escore:","0"]},{"entry":["Matching length:","132","Total length:","132"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"273pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00029","num":"00029"},"img":{"@attributes":{"id":"EMI-C00029","he":"31.75mm","wi":"84.92mm","file":"US07714100-20100511-C00029.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/Ro5LG3OhE3\/oQvcWauNWJ:Q9NZK3 (SEQ ID NO: 373)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: Z24874_PEA_2_P5 (SEQ ID NO: 297) \u00d7 Q9NZK3 (SEQ ID NO: 373)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","1070.00","Escore:","0"]},{"entry":["Matching length:","109","Total length:","109"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"273pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00030","num":"00030"},"img":{"@attributes":{"id":"EMI-C00030","he":"31.75mm","wi":"84.92mm","file":"US07714100-20100511-C00030.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/TxcClAWX3r\/LIzBcJ0ujT:Q9NPI5 (SEQ ID NO: 372)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: Z24874_PEA_2_P6 (SEQ ID NO: 298) \u00d7 Q9NPI5 (SEQ ID NO: 372)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","1048.00","Escore:","0"]},{"entry":["Matching length:","107","Total length:","107"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"273pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00031","num":"00031"},"img":{"@attributes":{"id":"EMI-C00031","he":"31.75mm","wi":"84.92mm","file":"US07714100-20100511-C00031.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/TxcClAWX3r\/LIzBcJ0ujT:Q9NZK3 (SEQ ID NO: 373)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: Z24874_PEA_2_P6 (SEQ ID NO: 298) \u00d7 Q9NZK3 (SEQ ID NO: 373)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","1048.00","Escore:","0"]},{"entry":["Matching length:","107","Total length:","107"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"273pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00032","num":"00032"},"img":{"@attributes":{"id":"EMI-C00032","he":"31.75mm","wi":"84.92mm","file":"US07714100-20100511-C00032.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}]}}},"Cluster HUMCDDANF features 2 transcript(s) and 7 segment(s) of interest, the names for which are given in Tables 109 and 110, respectively, the sequences themselves are given at the end of the application. The selected protein variants are given in table 111.",{"@attributes":{"id":"p-1117","num":"1164"},"tables":{"@attributes":{"id":"TABLE-US-00122","num":"00122"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 109"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Transcripts of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Transcript Name","Seq ID No."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"HUMCDDANF_T3","20"]},{"entry":[{},"HUMCDDANF_T4","21"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-1118","num":"1165"},"tables":{"@attributes":{"id":"TABLE-US-00123","num":"00123"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 110"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segments of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"98pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment Name","Seq ID No."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"HUMCDDANF_node_0","123"]},{"entry":[{},"HUMCDDANF_node_10","124"]},{"entry":[{},"HUMCDDANF_node_2","125"]},{"entry":[{},"HUMCDDANF_node_5","126"]},{"entry":[{},"HUMCDDANF_node_8","127"]},{"entry":[{},"HUMCDDANF_node_11","128"]},{"entry":[{},"HUMCDDANF_node_12","129"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-1119","num":"1166"},"tables":{"@attributes":{"id":"TABLE-US-00124","num":"00124"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 111"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Proteins of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Protein Name","Seq ID No."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"HUMCDDANF_P2","299"]},{"entry":[{},"HUMCDDANF_P3","300"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"These sequences are variants of the known protein Atrial natriuretic factor precursor (SEQ ID NO:350) (SwissProt accession identifier ANF_HUMAN; known also according to the synonyms ANF; Atrial natriuretic peptide; ANP; Prepronatriodilatin), referred to herein as the previously known protein; it contains Cardiodilatin-related peptide (CDP).","Protein Atrial natriuretic factor precursor (SEQ ID NO:350) is known or believed to have the following function(s): Atrial natriuretic factor (ANF) is a potent vasoactive substance synthesized in mammalian atria and is thought to play a key role in cardiovascular homeostasis; has a cGMP-stimulating activity. The sequence for protein Atrial natriuretic factor precursor is given at the end of the application, as \u201cAtrial natriuretic factor precursor amino acid sequence\u201d (SEQ ID NO:350). Known polymorphisms for this sequence are as shown in Table 112.",{"@attributes":{"id":"p-1122","num":"1169"},"tables":{"@attributes":{"id":"TABLE-US-00125","num":"00125"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 112"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations for Known Protein"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"161pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s)",{}]},{"entry":"on amino"},{"entry":["acid sequence","Comment"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["32","V -> M (in dbSNP: 5063). \/FTId = VAR_014579."]},{"entry":["152-153","Missing (in isoform 2). \/FTId = VAR_000594."]},{"entry":["65","E -> D"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"Protein Atrial natriuretic factor precursor (SEQ ID NO:350) localization is believed to be Secreted.","It has been investigated for clinical\/therapeutic use in humans, for example as a target for an antibody or small molecule, and\/or as a direct therapeutic; available information related to these investigations is as follows. Potential pharmaceutically related or therapeutically related activity or activities of the previously known protein are as follows: Aldosterone antagonist; Diuretic; Electrolyte absorption agonist. A therapeutic role for a protein represented by the cluster has been predicted. The cluster was assigned this field because there was information in the drug database or the public databases (e.g., described herein above) that this protein, or part thereof, is used or can be used for a potential therapeutic indication: Antihypertensive, diuretic; Antiasthma; Urological; Cardiostimulant, Antianaemic, Cardiovascular, Neuroprotective, Fertility enhancer, Male contraceptive, Hypolipaemic\/Antiatherosclerosis, Hepatoprotective and renal failure.","The following GO Annotation(s) apply to the previously known protein. The following annotation(s) were found: physiological processes; blood pressure regulation, which are annotation(s) related to Biological Process; hormone activity, which are annotation(s) related to Molecular Function; and extracellular, which are annotation(s) related to Cellular Component.","The GO assignment relies on information from one or more of the SwissProt\/TremBl Protein knowledgebase, available from <dot expasy dot ch\/sprot\/>; or Locuslink, available from <dot ncbi dot nlm dot nih dot gov\/projects\/LocusLink\/>.","The heart-selective diagnostic marker prediction engine provided the following results with regard to cluster HUMCDDANF. Predictions were made for selective expression of transcripts of this cluster in heart tissue, according to the previously described methods. The numbers on the y-axis of  refer to weighted expression of ESTs in each category, as \u201cparts per million\u201d (ratio of the expression of ESTs for a particular cluster to the expression of all ESTs in that category, according to parts per million).","Overall, the following results were obtained as shown with regard to the histogram in , concerning the number of heart-specific clones in libraries\/sequences; as well as with regard to the histogram in , concerning the actual expression of oligonucleotides in various tissues, including heart.","This cluster was found to be selectively expressed in heart for the following reasons: a comparison of the ratio of expression of the cluster in heart specific ESTs to the overall expression of the cluster in non-heart ESTs was found to be 56.3; The expression levels of this gene in muscle was negligible; and fisher exact test P-values were computed both for library and weighted clone counts to check that the counts are statistically significant, and were found to be 1.20E-249.","One particularly important measure of specificity of expression of a cluster in heart tissue is the previously described comparison of the ratio of expression of the cluster in heart as opposed to muscle. This cluster was found to be specifically expressed in heart as opposed to non-heart ESTs as described above. However, many proteins have been shown to be generally expressed at a higher level in both heart and muscle, which is less desirable. For this cluster, as described above, the expression levels of this gene in muscle was negligible, which clearly supports specific expression in heart tissue.","As noted above, cluster HUMCDDANF features 2 transcript(s), which were listed in Table 109 above. These transcript(s) encode for protein(s) which are variant(s) of protein Atrial natriuretic factor precursor (SEQ ID NO:350). A description of each variant protein according to the present invention is now provided.","Variant protein HUMCDDANF_P2 (SEQ ID NO:299) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) HUMCDDANF_T3 (SEQ ID NO:20). An alignment is given to the known protein (Atrial natriuretic factor precursor (SEQ ID NO:350)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between HUMCDDANF_P2 (SEQ ID NO:299) and ANF_HUMAN (SEQ ID NO:350):","1. An isolated chimeric polypeptide encoding for HUMCDDANF_P2 (SEQ ID NO:299), comprising a first amino acid sequence being at least 90% homologous to MPLEDEVVPPQVLSEPNEEAGAALSPLPEVPPWTGEVSPAQRDGGALGRGPWDSSDRSALLKSKLRALLT APRSLRRSSCFGGRMDRIGAQSGLGCNSFRY corresponding to amino acids 51-151 of ANF_HUMAN (SEQ ID NO:350), which also corresponds to amino acids 1-101 of HUMCDDANF_P2 (SEQ ID NO:299).","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: intracellularly. The protein localization is believed to be intracellular because neither of the trans-membrane region prediction programs predicted a trans-membrane region for this protein. In addition both signal-peptide prediction programs predict that this protein is a non-secreted protein.","Variant protein HUMCDDANF_P2 (SEQ ID NO:299) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 113, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein HUMCDDANF_P2 (SEQ ID NO:299) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-1137","num":"1184"},"tables":{"@attributes":{"id":"TABLE-US-00126","num":"00126"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 113"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on",{},{}]},{"entry":["amino acid sequence","Alternative amino acid(s)","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["20","A -> V","Yes"]},{"entry":["27","L -> F","Yes"]},{"entry":["74","S ->","No"]},{"entry":["76","R -> Q","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein HUMCDDANF_P2 (SEQ ID NO:299) is encoded by the following transcript(s): HUMCDDANF_T3 (SEQ ID NO:20), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript HUMCDDANF_T3 (SEQ ID NO:20) is shown in bold; this coding portion starts at position 381 and ends at position 683. The transcript also has the following SNPs as listed in Table 114 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein HUMCDDANF_P2 (SEQ ID NO:299) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-1139","num":"1186"},"tables":{"@attributes":{"id":"TABLE-US-00127","num":"00127"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 114"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on",{},{}]},{"entry":["nucleotide sequence","Alternative nucleic acid","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["199","C -> T","Yes"]},{"entry":["374","A -> G","No"]},{"entry":["771","T -> C","Yes"]},{"entry":["778","T -> C","Yes"]},{"entry":["809","C -> T","Yes"]},{"entry":["887","C -> G","No"]},{"entry":["968","A -> C","Yes"]},{"entry":["439","C -> T","Yes"]},{"entry":["458","C -> T","No"]},{"entry":["459","C -> T","Yes"]},{"entry":["602","C ->","No"]},{"entry":["607","G -> A","Yes"]},{"entry":["684","T -> C","Yes (short\/long variant)"]},{"entry":["711","A -> G","No"]},{"entry":["757","G -> T","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein HUMCDDANF_P3 (SEQ ID NO:300) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) HUMCDDANF_T4 (SEQ ID NO:21). An alignment is given to the known protein (Atrial natriuretic factor precursor (SEQ ID NO:350)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between HUMCDDANF_P3 (SEQ ID NO:300) and ANF_HUMAN (SEQ ID NO:350):","1. An isolated chimeric polypeptide encoding for HUMCDDANF_P3 (SEQ ID NO:300), comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence MSSFSTTT (SEQ ID NO:411) corresponding to amino acids 1-8 of HUMCDDANF_P3 (SEQ ID NO:300), and a second amino acid sequence being at least 90% homologous to NLLDHLEEKMPLEDEVVPPQVLSEPNEEAGAALSPLPEVPPWTGEVSPAQRDGGALGRGPWDSSDRSALL KSKLRALLTAPRSLRRSSCFGGRMDRIGAQSGLGCNSFRY corresponding to amino acids 42-151 of ANF_HUMAN (SEQ ID NO:350), which also corresponds to amino acids 9-118 of HUMCDDANF_P3 (SEQ ID NO:300), wherein said first and second amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a head of HUMCDDANF_P3 (SEQ ID NO:300), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence MSSFSTTT (SEQ ID NO:411) of HUMCDDANF_P3 (SEQ ID NO:300)","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: intracellularly. The protein localization is believed to be intracellularly because neither of the trans-membrane region prediction programs predicted a trans-membrane region for this protein. In addition both signal-peptide prediction programs predict that this protein is a non-secreted protein.","Variant protein HUMCDDANF_P3 (SEQ ID NO:300) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 115, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein HUMCDDANF_P3 (SEQ ID NO:300) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-1146","num":"1193"},"tables":{"@attributes":{"id":"TABLE-US-00128","num":"00128"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 115"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on",{},{}]},{"entry":["amino acid sequence","Alternative amino acid(s)","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["37","A -> V","Yes"]},{"entry":["44","L -> F","Yes"]},{"entry":["91","S ->","No"]},{"entry":["93","R -> Q","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein HUMCDDANF_P3 (SEQ ID NO:300) is encoded by the following transcript(s): HUMCDDANF_T4 (SEQ ID NO:21), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript HUMCDDANF_T4 (SEQ ID NO:21) is shown in bold; this coding portion starts at position 104 and ends at position 457. The transcript also has the following SNPs as listed in Table 116 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein HUMCDDANF_P3 (SEQ ID NO:300) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-1148","num":"1195"},"tables":{"@attributes":{"id":"TABLE-US-00129","num":"00129"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 116"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on",{},{}]},{"entry":["nucleotide sequence","Alternative nucleic acid","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["148","A -> G","No"]},{"entry":["213","C -> T","Yes"]},{"entry":["552","T -> C","Yes"]},{"entry":["583","C -> T","Yes"]},{"entry":["661","C -> G","No"]},{"entry":["742","A -> C","Yes"]},{"entry":["232","C -> T","No"]},{"entry":["233","C -> T","Yes"]},{"entry":["376","C ->","No"]},{"entry":["381","G -> A","Yes"]},{"entry":["458","T -> C","Yes (short\/long isoform)"]},{"entry":["485","A -> G","No"]},{"entry":["531","G -> T","Yes"]},{"entry":["545","T -> C","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"As noted above, cluster HUMCDDANF features 7 segment(s), which were listed in Table 110 above and for which the sequence(s) are given at the end of the application. These segment(s) are portions of nucleic acid sequence(s) which are described herein separately because they are of particular interest. A description of each segment according to the present invention is now provided.","Segment cluster HUMCDDANF_node0 (SEQ ID NO:123) according to the present invention is supported by 5 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMCDDANF_T3 (SEQ ID NO:20). Table 117 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1151","num":"1198"},"tables":{"@attributes":{"id":"TABLE-US-00130","num":"00130"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 117"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment starting",{}]},{"entry":["Transcript name","position","Segment ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMCDDANF_T3","1","353"]},{"entry":"(SEQ ID NO: 20)"},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMCDDANF_node10 (SEQ ID NO:124) according to the present invention is supported by 49 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMCDDANF_T3 (SEQ ID NO:20) and HUMCDDANF_T4 (SEQ ID NO:21). Table 118 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1153","num":"1200"},"tables":{"@attributes":{"id":"TABLE-US-00131","num":"00131"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 118"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment",{}]},{"entry":[{},"starting","Segment"]},{"entry":["Transcript name","position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMCDDANF_T3 (SEQ ID NO: 20)","813","940"]},{"entry":["HUMCDDANF_T4 (SEQ ID NO: 21)","587","714"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMCDDANF_node2 (SEQ ID NO:125) according to the present invention is supported by 41 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMCDDANF_T4 (SEQ ID NO:21). Table 119 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1155","num":"1202"},"tables":{"@attributes":{"id":"TABLE-US-00132","num":"00132"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 119"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment starting",{}]},{"entry":["Transcript name","position","Segment ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMCDDANF_T4","1","127"]},{"entry":"(SEQ ID NO: 21)"},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMCDDANF_node5 (SEQ ID NO:126) according to the present invention is supported by 62 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMCDDANF_T3 (SEQ ID NO:20) and HUMCDDANF_T4 (SEQ ID NO:21). Table 120 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1157","num":"1204"},"tables":{"@attributes":{"id":"TABLE-US-00133","num":"00133"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 120"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment",{}]},{"entry":[{},"starting","Segment"]},{"entry":["Transcript name","position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMCDDANF_T3 (SEQ ID NO: 20)","354","680"]},{"entry":["HUMCDDANF_T4 (SEQ ID NO: 21)","128","454"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMCDDANF_node8 (SEQ ID NO:127) according to the present invention is supported by 56 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMCDDANF_T3 (SEQ ID NO:20) and HUMCDDANF_T4 (SEQ ID NO:21). Table 121 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1159","num":"1206"},"tables":{"@attributes":{"id":"TABLE-US-00134","num":"00134"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 121"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment",{}]},{"entry":[{},"starting","Segment"]},{"entry":["Transcript name","position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMCDDANF_T3 (SEQ ID NO: 20)","681","812"]},{"entry":["HUMCDDANF_T4 (SEQ ID NO: 21)","455","586"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"According to an optional embodiment of the present invention, short segments related to the above cluster are also provided. These segments are up to about 120 bp in length, and so are included in a separate description.","Segment cluster HUMCDDANF_node11 (SEQ ID NO:128) according to the present invention can be found in the following transcript(s): HUMCDDANF_T3 (SEQ ID NO:20) and HUMCDDANF_T4 (SEQ ID NO:21). Table 122 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1162","num":"1209"},"tables":{"@attributes":{"id":"TABLE-US-00135","num":"00135"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 122"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment",{}]},{"entry":[{},"starting","Segment"]},{"entry":["Transcript name","position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMCDDANF_T3 (SEQ ID NO: 20)","941","951"]},{"entry":["HUMCDDANF_T4 (SEQ ID NO: 21)","715","725"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMCDDANF_node12 (SEQ ID NO:129) according to the present invention is supported by 36 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMCDDANF_T3 (SEQ ID NO:20) and HUMCDDANF_T4 (SEQ ID NO:21). Table 123 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1164","num":"1211"},"tables":{"@attributes":{"id":"TABLE-US-00136","num":"00136"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 123"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment",{}]},{"entry":[{},"starting","Segment"]},{"entry":["Transcript name","position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMCDDANF_T3 (SEQ ID NO: 20)","952","992"]},{"entry":["HUMCDDANF_T4 (SEQ ID NO: 21)","726","766"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}},"br":{}},{"@attributes":{"id":"p-1165","num":"1212"},"tables":{"@attributes":{"id":"TABLE-US-00137","num":"00137"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"287pt","align":"left"}},"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Sequence name: \/tmp\/3GyiZQyJ8L\/jYng3zFfcE:ANF_HUMAN (SEQ ID NO: 350)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: HUMCDDANF_P2 (SEQ ID NO: 299) \u00d7 ANF_HUMAN (SEQ ID NO: 350)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","988.00","Escore:","0"]},{"entry":["Matching length:","101","Total length:","101"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"287pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00033","num":"00033"},"img":{"@attributes":{"id":"EMI-C00033","he":"31.75mm","wi":"84.92mm","file":"US07714100-20100511-C00033.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/mnb7OPVCPP\/oTrSwgJLyB:ANF_HUMAN (SEQ ID NO: 350)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: HUMCDDANF_P3 (SEQ ID NO: 300) \u00d7 ANF_HUMAN (SEQ ID NO: 350)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","1076.00","Escore:","0"]},{"entry":["Matching length:","110","Total length:","110"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"287pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00034","num":"00034"},"img":{"@attributes":{"id":"EMI-C00034","he":"31.75mm","wi":"84.92mm","file":"US07714100-20100511-C00034.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}]}}},"Expression of Human cardiodilatin-atrial natriuretic factor (CDD-ANF) transcripts detectable by or according to junc2-5 node(s), HUHUMCDDANFjunc2-5F2R2 (SEQ ID NO:376) amplicon and primers HUMCDDANFjunc2-5F2 (SEQ ID NO:374) HUMCDDANFjunc2-5R2 (SEQ ID NO:375) was measured by real time PCR (this transcript relates to the known or WT protein). In parallel the expression of four housekeeping genes\u2014RPL19 (GenBank Accession No. NM000981 (SEQ ID NO:437); RPL19 amplicon (SEQ ID NO:440)), TATA box (GenBank Accession No. NM003194 (SEQ ID NO:441); TATA amplicon (SEQ ID NO:444)), Ubiquitin (GenBank Accession No. BC000449 (SEQ ID NO:445); amplicon\u2014Ubiquitin-amplicon (SEQ ID NO:448)) and SDHA (GenBank Accession No. NM004168 (SEQ ID NO:449); amplicon\u2014SDHA-amplicon (SEQ ID NO:452)) was measured similarly. For each RT sample, the expression of the above amplicons was normalized to the geometric mean of the quantities of the housekeeping genes. The normalized quantity of each RT sample was then divided by the quantity of heart sample no. 45 (Table 2, above), to obtain a value of relative expression for each sample relative to this heart sample.","As is evident from , the expression of Human cardiodilatin-atrial natriuretic factor (CDD-ANF) transcripts detectable by the above amplicon(s) in one of the heart tissue samples (Sample Nos. 45, Table 2, \u201cTissue samples in testing panel\u201d) was significantly higher than in the other samples, including other two heart samples. Sample 45 is from fibrotic heart, as opposed to heart samples 44 and 46 that are from normal hearts. (Note\u2014the product in samples 10 and 11 was found to be a non-specific product by inspecting the dissociation curve that was created in the real-time PCR experiment).","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: HUMCDDANFjunc2-5F2 (SEQ ID NO:374) forward primer; and HUMCDDANFjunc2-5R2 (SEQ ID NO:375) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: HUHUMCDDANFjunc2-5F2R2 (SEQ ID NO:376).",{"@attributes":{"id":"p-1170","num":"1217"},"tables":{"@attributes":{"id":"TABLE-US-00138","num":"00138"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"294pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"147pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Forward primer HUMCDDANFjunc2-5F2 (SEQ ID NO: 374):",{}]},{"entry":"CTTCTCCACCACCACCAATTTG"},{"entry":{}},{"entry":"Reverse primer HUMCDDANFjunc2-5R2 (SEQ ID NO: 375):"},{"entry":"GAGAGCAGCCCCCGCT"},{"entry":{}},{"entry":"Amplicon HUMCDDANFjunc2-5F2R2 (SEQ ID NO: 376):"},{"entry":"CTTCTCCACCACCACCAATTTGCTGGACCATTTGGAAGAAAAGATGCCTTTAGAAGATGAGGTCGTGC"},{"entry":{}},{"entry":"CCCCACAAGTGCTCAGTGAGCCGAATGAAGAAGCGGGGGCTGCTCTC"}]}}}}},"Cluster HUMTROPIA features 4 transcript(s) and 20 segment(s) of interest, the names for which are given in Tables 124 and 125, respectively, the sequences themselves are given at the end of the application. The selected protein variants are given in table 126.",{"@attributes":{"id":"p-1172","num":"1219"},"tables":{"@attributes":{"id":"TABLE-US-00139","num":"00139"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 124"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Transcripts of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"98pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Transcript Name","Seq ID No."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"HUMTROPIA_PEA_2_T10","22"]},{"entry":[{},"HUMTROPIA_PEA_2_T15","23"]},{"entry":[{},"HUMTROPIA_PEA_2_T3","24"]},{"entry":[{},"HUMTROPIA_PEA_2_T7","25"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-1173","num":"1220"},"tables":{"@attributes":{"id":"TABLE-US-00140","num":"00140"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 125"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segments of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment Name","Seq ID No."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"HUMTROPIA_PEA_2_node_0","130"]},{"entry":[{},"HUMTROPIA_PEA_2_node_10","131"]},{"entry":[{},"HUMTROPIA_PEA_2_node_22","132"]},{"entry":[{},"HUMTROPIA_PEA_2_node_23","133"]},{"entry":[{},"HUMTROPIA_PEA_2_node_11","134"]},{"entry":[{},"HUMTROPIA_PEA_2_node_14","135"]},{"entry":[{},"HUMTROPIA_PEA_2_node_15","136"]},{"entry":[{},"HUMTROPIA_PEA_2_node_16","137"]},{"entry":[{},"HUMTROPIA_PEA_2_node_20","138"]},{"entry":[{},"HUMTROPIA_PEA_2_node_21","139"]},{"entry":[{},"HUMTROPIA_PEA_2_node_24","140"]},{"entry":[{},"HUMTROPIA_PEA_2_node_25","141"]},{"entry":[{},"HUMTROPIA_PEA_2_node_29","142"]},{"entry":[{},"HUMTROPIA_PEA_2_node_30","143"]},{"entry":[{},"HUMTROPIA_PEA_2_node_31","144"]},{"entry":[{},"HUMTROPIA_PEA_2_node_32","145"]},{"entry":[{},"HUMTROPIA_PEA_2_node_4","146"]},{"entry":[{},"HUMTROPIA_PEA_2_node_5","147"]},{"entry":[{},"HUMTROPIA_PEA_2_node_8","148"]},{"entry":[{},"HUMTROPIA_PEA_2_node_9","149"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-1174","num":"1221"},"tables":{"@attributes":{"id":"TABLE-US-00141","num":"00141"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 126"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Proteins of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"98pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Protein Name","Seq ID No."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"HUMTROPIA_PEA_2_P5","301"]},{"entry":[{},"HUMTROPIA_PEA_2_P12","302"]},{"entry":[{},"HUMTROPIA_PEA_2_P17","303"]},{"entry":[{},"HUMTROPIA_PEA_2_P18","304"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"These sequences are variants of the known protein Troponin I, cardiac muscle (SwissProt accession identifier TRIC_HUMAN), referred to herein as the previously known protein and shown as SEQ ID NO:351.","Protein Troponin I, cardiac muscle (SEQ ID NO:351) is known or believed to have the following function(s): Troponin I is the inhibitory subunit of troponin, the thin filament regulatory complex which confers calcium-sensitivity to striated muscle actomyosin ATPase activity. Troponin I, cardiac muscle (SEQ ID NO:351) Binds to actin and tropomyosin. Defects in Troponin I, cardiac muscle (SEQ ID NO:351) are the cause of familial hypertrophic cardiomyopathy type 7 (CMH7) [MIM:191044]; also known as FHC type 7. CMH7 is an autosomal dominant disorder characterized by increased myocardial mass with myocyte and myofibrillar disarray. Defects in Troponin I, cardiac muscle (SEQ ID NO:351) are the cause of familial restrictive cardiomyopathy (RCM) [MIM:115210]. RCM is a heart muscle disorder characterized by impaired filling of the ventricles with reduced volume in the presence of normal or near normal wall thickness and systolic function. The disease may be associated with systemic disease but is most often idiopathic. The sequence for protein Troponin I, cardiac muscle is given at the end of the application, as \u201cTroponin I, cardiac muscle amino acid sequence\u201d (SEQ ID NO:351). Known polymorphisms for this sequence are as shown in Table 127.",{"@attributes":{"id":"p-1177","num":"1224"},"tables":{"@attributes":{"id":"TABLE-US-00142","num":"00142"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 127"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations for Known Protein"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"161pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s)",{}]},{"entry":"on amino"},{"entry":["acid sequence","Comment"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"161pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["81","P -> S (in CMH7). \/FTId = VAR_016078."]},{"entry":["143","L -> Q (in RCM). \/FTId = VAR_016079."]},{"entry":["144","R -> G (in CMH7). \/FTId = VAR_007603."]},{"entry":["144","R -> W (in RCM). \/FTId = VAR_016080."]},{"entry":["170","A -> T (in RCM). \/FTId = VAR_016081."]},{"entry":["177","K -> E (in RCM). \/FTId = VAR_016082."]},{"entry":["189","D -> H (in CMH7 and RCM). \/FTId = VAR_016083."]},{"entry":["191","R -> H (in RCM). \/FTId = VAR_016084."]},{"entry":["195","D -> N (in CMH7). \/FTId = VAR_016085."]},{"entry":["205","K -> Q (in CMH7). \/FTId = VAR_007604."]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"In addition to the above known polymorphisms, the present inventors have uncovered two new additional SNPs (shown with regard to SEQ ID NO:352 for the resultant amino acid sequence, and SEQ ID NO:353 for the nucleic acid sequence). This SNP is C-> (missing nucleotide \u201cC\u201d; will affect amino acid residues from 167 onwards). This will create a frame shift. A new protein will be formed. However, this SNP was located in a stretch of cytosine residues, which are known to be prone to errors in sequencing.","The previously known protein also has the following indication(s) and\/or potential therapeutic use(s): Cancer, lung, non-small cell; Cancer, breast; Cancer, sarcoma. It has been investigated for clinical\/therapeutic use in humans, for example as a target for an antibody or small molecule, and\/or as a direct therapeutic; available information related to these investigations is as follows. Potential pharmaceutically related or therapeutically related activity or activities of the previously known protein are as follows: Angiogenesis inhibitor; Epidermal growth factor antagonist; Fibroblast growth factor receptor antagonist. A therapeutic role for a protein represented by the cluster has been predicted. The cluster was assigned this field because there was information in the drug database or the public databases (e.g., described herein above) that this protein, or part thereof, is used or can be used for a potential therapeutic indication: Ophthalmological; Anticancer.","The following GO Annotation(s) apply to the previously known protein. The following annotation(s) were found: control of heart, which are annotation(s) related to Biological Process; and troponin complex, which are annotation(s) related to Cellular Component.","The GO assignment relies on information from one or more of the SwissProt\/TremBl Protein knowledgebase, available from <dot expasy dot ch\/sprot\/>; or Locuslink, available from <dot ncbi dot nlm dot nih dot gov\/projects\/LocusLink\/>.","The heart-selective diagnostic marker prediction engine provided the following results with regard to cluster HUMTROPIA. Predictions were made for selective expression of transcripts of this cluster in heart tissue, according to the previously described methods. The numbers on the y-axis of  refer to weighted expression of ESTs in each category, as \u201cparts per million\u201d (ratio of the expression of ESTs for a particular cluster to the expression of all ESTs in that category, according to parts per million).","Overall, the following results were obtained as shown with regard to the histogram in , concerning the number of heart-specific clones in libraries\/sequences; as well as with regard to the histogram in , concerning the actual expression of oligonucleotides in various tissues, including heart.","This cluster was found to be selectively expressed in heart for the following reasons: in a comparison of the ratio of expression of the cluster in heart specific ESTs to the overall expression of the cluster in non-heart ESTs, which was found to be 27.5. The expression level of this gene in muscle was negligible; and fisher exact test P-values were computed both for library and weighted clone counts to check that the counts are statistically significant, and were found to be 2.10E-88.","One particularly important measure of specificity of expression of a cluster in heart tissue is the previously described comparison of the ratio of expression of the cluster in heart as opposed to muscle. This cluster was found to be specifically expressed in heart as opposed to non-heart ESTs as described above. However, many proteins have been shown to be generally expressed at a higher level in both heart and muscle, which is less desirable. For this cluster, as described above, the expression level of this gene in muscle was negligible which clearly supports specific expression in heart tissue.","As noted above, cluster HUMTROPIA features 4 transcript(s), which were listed in Table 124 above. These transcript(s) encode for protein(s) which are variant(s) of protein Troponin I, cardiac muscle (SEQ ID NO:351). A description of each variant protein according to the present invention is now provided.","Variant protein HUMTROPIA_PEA2_P5 (SEQ ID NO:301) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) HUMTROPIA_PEA2_T3 (SEQ ID NO:24). An alignment is given to the known protein (Troponin I, cardiac muscle (SEQ ID NO:351)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between HUMTROPIA_PEA2_P5 (SEQ ID NO:301) and TRIC_HUMAN (SEQ ID NO:351):","1. An isolated chimeric polypeptide encoding for HUMTROPIA_PEA2_P5 (SEQ ID NO:301), comprising a first amino acid sequence being at least 90% homologous to MADGSSDAAREPRPAPAPIRRRSSNYRAYATEPHAKKKSKISASRKLQLKTLLLQIAKQELEREAEERRGE KGRALSTRCQPLELAGLGFAELQDLCRQLHARVDKVDEERYDIEAKVTKNITE corresponding to amino acids 1-124 of TRIC_HUMAN (SEQ ID NO:351), which also corresponds to amino acids 1-124 of HUMTROPIA_PEA2_P5 (SEQ ID NO:301), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence VGRMGSSGTFGVG (SEQ ID NO:412) corresponding to amino acids 125-137 of HUMTROPIA_PEA2_P5 (SEQ ID NO:301), wherein said first and second amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of HUMTROPIA_PEA2_P5 (SEQ ID NO:301), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VGRMGSSGTFGVG (SEQ ID NO:412) in HUMTROPIA_PEA2_P5 (SEQ ID NO:301).","The cellular location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: intracellularly. The protein localization is believed to be intracellular because neither of the trans-membrane region prediction programs predicted a trans-membrane region for this protein. In addition both signal-peptide prediction programs predict that this protein is a non-secreted protein.","Variant protein HUMTROPIA_PEA2_P5 (SEQ ID NO:301) is encoded by the following transcript(s): HUMTROPIA_PEA2_T3 (SEQ ID NO:24), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript HUMTROPIA_PEA2_T3 (SEQ ID NO:24) is shown in bold; this coding portion starts at position 148 and ends at position 558.","Variant protein HUMTROPIA_PEA2_P12 (SEQ ID NO:302) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) HUMTROPIA_PEA2_T15 (SEQ ID NO:23). An alignment is given to the known protein (Troponin I, cardiac muscle (SEQ ID NO:351)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between HUMTROPIA_PEA2_P12 (SEQ ID NO:302) and TRIC_HUMAN (SEQ ID NO:351):","1. An isolated chimeric polypeptide encoding for HUMTROPIA_PEA2_P12 (SEQ ID NO:302), comprising a first amino acid sequence being at least 90% homologous to MADGSSDA corresponding to amino acids 1-8 of TRIC_HUMAN (SEQ ID NO:351), which also corresponds to amino acids 1-8 of HUMTROPIA_PEA2_P12 (SEQ ID NO:302), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence KKSKISASRKLQLKTLLLQIAKQELEREAEERRGEKGRALSTRCQPLELAGL GFAELQDLCRQLHARVDKVDEERYDIEAKVTKNITEIADLTQKIFDLRGKFKRPTLRRVRISADAMMQALL GARAKESLDLRAHLKQVKKEDTEKENREVGDWRKNIDALSGMEGRKKKFES corresponding to amino acids 36-209 of TRIC_HUMAN (SEQ ID NO:351), which also corresponding to amino acids 9-182 of HUMTROPIA_PEA2_P12 (SEQ ID NO:302), wherein said first and second amino acid sequences are contiguous and in a sequential order.","2. An isolated chimeric polypeptide encoding for an edge portion of HUMTROPIA_PEA2_P12 (SEQ ID NO:302), comprising a polypeptide having a length \u201cn\u201d, wherein \u201cn\u201d is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least two amino acids comprise AK, having a structure as follows: a sequence starting from any of amino acid numbers 8\u2212x to 8; and ending at any of amino acid numbers 9+((n\u22122)\u2212x), in which x varies from 0 to n\u22122.","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: intracellularly. The protein localization is believed to be intracellular because neither of the trans-membrane region prediction programs predicted a trans-membrane region for this protein. In addition both signal-peptide prediction programs predict that this protein is a non-secreted protein.","Variant protein HUMTROPIA_PEA2_P12 (SEQ ID NO:302) is encoded by the following transcript(s): HUMTROPIA_PEA2_T15 (SEQ ID NO:23), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript HUMTROPIA_PEA2_T15 (SEQ ID NO:23) is shown in bold; this coding portion starts at position 148 and ends at position 693.","Variant protein HUMTROPIA_PEA2_P17 (SEQ ID NO:303) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) HUMTROPIA_PEA2_T7 (SEQ ID NO:25). An alignment is given to the known protein (Troponin I, cardiac muscle (SEQ ID NO:351)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between HUMTROPIA_PEA2_P17 (SEQ ID NO:303) and TRIC_HUMAN (SEQ ID NO:351):","1. An isolated chimeric polypeptide encoding for HUMTROPIA_PEA2_P17 (SEQ ID NO:303), comprising a first amino acid sequence being at least 90% homologous to MADGSSDAAREPRPAPAPIRRRSSNYRAYATEPHAK corresponding to amino acids 1-36 of TRIC_HUMAN (SEQ ID NO:351), which also corresponds to amino acids 1-36 of HUMTROPIA_PEA2_P17 (SEQ ID NO:303), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence VGRGFLGAEYRRRRDPRPWEWGEEPGLRRGRGLRGGASGAEFCRGSCSDW (SEQ ID NO:413) corresponding to amino acids 37-86 of HUMTROPIA_PEA2_P17 (SEQ ID NO:303), wherein said first and second amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of HUMTROPIA_PEA2_P17 (SEQ ID NO:303), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VGRGFLGAEYRRRRDPRPWEWGEEPGLRRGRGLRGGASGAEFCRGSCSDW (SEQ ID NO:413) in HUMTROPIA_PEA2_P17 (SEQ ID NO:303).","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: intracellularly. The protein localization is believed to be intracellular because neither of the trans-membrane region prediction programs predicted a trans-membrane region for this protein. In addition both signal-peptide prediction programs predict that this protein is a non-secreted protein.","Variant protein HUMTROPIA_PEA2_P17 (SEQ ID NO:303) is encoded by the following transcript(s): HUMTROPIA_PEA2_T7 (SEQ ID NO:25), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript HUMTROPIA_PEA2_T7 (SEQ ID NO:25) is shown in bold; this coding portion starts at position 148 and ends at position 405.","Variant protein HUMTROPIA_PEA2_P18 (SEQ ID NO:304) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) HUMTROPIA_PEA2_T10 (SEQ ID NO:22). An alignment is given to the known protein (Troponin I, cardiac muscle (SEQ ID NO:351)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between HUMTROPIA_PEA2_P18 (SEQ ID NO:304) and TRIC_HUMAN (SEQ ID NO:351):","1. An isolated chimeric polypeptide encoding for HUMTROPIA_PEA2_P18 (SEQ ID NO:304), comprising a first amino acid sequence being at least 90% homologous to MADGSSDA corresponding to amino acids 1-8 of TRIC_HUMAN (SEQ ID NO:351), which also corresponds to amino acids 1-8 of HUMTROPIA_PEA2_P18 (SEQ ID NO:304), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence VRAAG (SEQ ID NO:414) corresponding to amino acids 9-13 of HUMTROPIA_PEA2_P18 (SEQ ID NO:304), wherein said first and second amino acid sequences are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of HUMTROPIA_PEA2_P18 (SEQ ID NO:304), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VRAAG (SEQ ID NO:414) in HUMTROPIA_PEA2_P18 (SEQ ID NO:304).","The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: intracellularly. The protein localization is believed to be intracellularly because neither of the trans-membrane region prediction programs predicted a trans-membrane region for this protein. In addition both signal-peptide prediction programs predict that this protein is a non-secreted protein.","Variant protein HUMTROPIA_PEA2_P18 (SEQ ID NO:304) is encoded by the following transcript(s): HUMTROPIA_PEA2_T10 (SEQ ID NO:22), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript HUMTROPIA_PEA2_T10 (SEQ ID NO:22) is shown in bold; this coding portion starts at position 148 and ends at position 186.","As noted above, cluster HUMTROPIA features 20 segment(s), which were listed in Table 125 above and for which the sequence(s) are given at the end of the application. These segment(s) are portions of nucleic acid sequence(s) which are described herein separately because they are of particular interest. A description of each segment according to the present invention is now provided.","Segment cluster HUMTROPIA_PEA2_node0 (SEQ ID NO:130) according to the present invention is supported by 29 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMTROPIA_PEA2_T10 (SEQ ID NO:22), HUMTROPIA_PEA2_T15 (SEQ ID NO:23), HUMTROPIA_PEA2_T3 (SEQ ID NO:24) and HUMTROPIA_PEA2_T7 (SEQ ID NO:25). Table 128 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1213","num":"1260"},"tables":{"@attributes":{"id":"TABLE-US-00143","num":"00143"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 128"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMTROPIA_PEA_2_T10 (SEQ ID NO: 22)","1","158"]},{"entry":["HUMTROPIA_PEA_2_T15 (SEQ ID NO: 23)","1","158"]},{"entry":["HUMTROPIA_PEA_2_T3 (SEQ ID NO: 24)","1","158"]},{"entry":["HUMTROPIA_PEA_2_T7 (SEQ ID NO: 25)","1","158"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMTROPIA_PEA2_node10 (SEQ ID NO:131) according to the present invention is supported by 5 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMTROPIA_PEA2_T7 (SEQ ID NO:25). Table 129 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1215","num":"1262"},"tables":{"@attributes":{"id":"TABLE-US-00144","num":"00144"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 129"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"140pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMTROPIA_PEA_2_T7 (SEQ ID NO: 25)","256","660"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMTROPIA_PEA2_node22 (SEQ ID NO:132) according to the present invention is supported by 8 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMTROPIA_PEA2_T3 (SEQ ID NO:24). Table 130 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1217","num":"1264"},"tables":{"@attributes":{"id":"TABLE-US-00145","num":"00145"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 130"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"140pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMTROPIA_PEA_2_T3 (SEQ ID NO: 24)","520","1053"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMTROPIA_PEA2_node23 (SEQ ID NO:133) according to the present invention is supported by 49 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMTROPIA_PEA2_T10 (SEQ ID NO:22), HUMTROPIA_PEA2_T15 (SEQ ID NO:23), HUMTROPIA_PEA2_T3 (SEQ ID NO:24) and HUMTROPIA_PEA2_T7 (SEQ ID NO:25). Table  below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1219","num":"1266"},"tables":{"@attributes":{"id":"TABLE-US-00146","num":"00146"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 131"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["HUMTROPIA_PEA_2_T10 (SEQ ID NO: 22)","565","708"]},{"entry":["HUMTROPIA_PEA_2_T15 (SEQ ID NO: 23)","436","579"]},{"entry":["HUMTROPIA_PEA_2_T3 (SEQ ID NO: 24)","1054","1197"]},{"entry":["HUMTROPIA_PEA_2_T7 (SEQ ID NO: 25)","925","1068"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"According to an optional embodiment of the present invention, short segments related to the above cluster are also provided. These segments are up to about 120 bp in length, and so are included in a separate description.","Segment cluster HUMTROPIA_PEA2_node11 (SEQ ID NO:134) according to the present invention is supported by 28 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMTROPIA_PEA2_T10 (SEQ ID NO:22), HUMTROPIA_PEA2_T15 (SEQ ID NO:23), HUMTROPIA_PEA2_T3 (SEQ ID NO:24) and HUMTROPIA_PEA2_T7 (SEQ ID NO:25). Table 132 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1222","num":"1269"},"tables":{"@attributes":{"id":"TABLE-US-00147","num":"00147"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 132"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMTROPIA_PEA_2_T10 (SEQ ID NO: 22)","301","342"]},{"entry":["HUMTROPIA_PEA_2_T15 (SEQ ID NO: 23)","172","213"]},{"entry":["HUMTROPIA_PEA_2_T3 (SEQ ID NO: 24)","256","297"]},{"entry":["HUMTROPIA_PEA_2_T7 (SEQ ID NO: 25)","661","702"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMTROPIA_PEA2_node14 (SEQ ID NO:135) according to the present invention is supported by 37 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMTROPIA_PEA2_T10 (SEQ ID NO:22), HUMTROPIA_PEA2_T15 (SEQ ID NO:23), HUMTROPIA_PEA2_T3 (SEQ ID NO:24) and HUMTROPIA_PEA2_T7 (SEQ ID NO:25). Table 133 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1224","num":"1271"},"tables":{"@attributes":{"id":"TABLE-US-00148","num":"00148"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 133"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMTROPIA_PEA_2_T10 (SEQ ID NO: 22)","343","378"]},{"entry":["HUMTROPIA_PEA_2_T15 (SEQ ID NO: 23)","214","249"]},{"entry":["HUMTROPIA_PEA_2_T3 (SEQ ID NO: 24)","298","333"]},{"entry":["HUMTROPIA_PEA_2_T7 (SEQ ID NO: 25)","703","738"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMTROPIA_PEA2_node15 (SEQ ID NO:136) according to the present invention is supported by 42 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMTROPIA_PEA2_T10 (SEQ ID NO:22), HUMTROPIA_PEA2_T15 (SEQ ID NO:23), HUMTROPIA_PEA2_T3 (SEQ ID NO:24) and HUMTROPIA_PEA2_T7 (SEQ ID NO:25). Table 134 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1226","num":"1273"},"tables":{"@attributes":{"id":"TABLE-US-00149","num":"00149"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 134"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMTROPIA_PEA_2_T10 (SEQ ID NO: 22)","379","422"]},{"entry":["HUMTROPIA_PEA_2_T15 (SEQ ID NO: 23)","250","293"]},{"entry":["HUMTROPIA_PEA_2_T3 (SEQ ID NO: 24)","334","377"]},{"entry":["HUMTROPIA_PEA_2_T7 (SEQ ID NO: 25)","739","782"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMTROPIA_PEA2_node16 (SEQ ID NO:137) according to the present invention is supported by 40 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMTROPIA_PEA2_T10 (SEQ ID NO:22), HUMTROPIA_PEA2_T15 (SEQ ID NO:23), HUMTROPIA_PEA2_T3 (SEQ ID NO:24) and HUMTROPIA_PEA2_T7 (SEQ ID NO:25). Table 135 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1228","num":"1275"},"tables":{"@attributes":{"id":"TABLE-US-00150","num":"00150"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 135"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMTROPIA_PEA_2_T10 (SEQ ID NO: 22)","423","474"]},{"entry":["HUMTROPIA_PEA_2_T15 (SEQ ID NO: 23)","294","345"]},{"entry":["HUMTROPIA_PEA_2_T3 (SEQ ID NO: 24)","378","429"]},{"entry":["HUMTROPIA_PEA_2_T7 (SEQ ID NO: 25)","783","834"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMTROPIA_PEA2_node20 (SEQ ID NO:138) according to the present invention is supported by 44 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMTROPIA_PEA2_T10 (SEQ ID NO:22), HUMTROPIA_PEA2_T15 (SEQ ID NO:23), HUMTROPIA_PEA2_T3 (SEQ ID NO:24) and HUMTROPIA_PEA2_T7 (SEQ ID NO:25). Table 136 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1230","num":"1277"},"tables":{"@attributes":{"id":"TABLE-US-00151","num":"00151"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 136"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMTROPIA_PEA_2_T10 (SEQ ID NO: 22)","475","510"]},{"entry":["HUMTROPIA_PEA_2_T15 (SEQ ID NO: 23)","346","381"]},{"entry":["HUMTROPIA_PEA_2_T3 (SEQ ID NO: 24)","430","465"]},{"entry":["HUMTROPIA_PEA_2_T7 (SEQ ID NO: 25)","835","870"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMTROPIA_PEA2_node21 (SEQ ID NO:139) according to the present invention is supported by 44 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMTROPIA_PEA2_T10 (SEQ ID NO:22), HUMTROPIA_PEA2_T15 (SEQ ID NO:23), HUMTROPIA_PEA2_T3 (SEQ ID NO:24) and HUMTROPIA_PEA2_T7 (SEQ ID NO:25). Table 137 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1232","num":"1279"},"tables":{"@attributes":{"id":"TABLE-US-00152","num":"00152"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 137"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMTROPIA_PEA_2_T10 (SEQ ID NO: 22)","511","564"]},{"entry":["HUMTROPIA_PEA_2_T15 (SEQ ID NO: 23)","382","435"]},{"entry":["HUMTROPIA_PEA_2_T3 (SEQ ID NO: 24)","466","519"]},{"entry":["HUMTROPIA_PEA_2_T7 (SEQ ID NO: 25)","871","924"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMTROPIA_PEA2_node24 (SEQ ID NO:140) according to the present invention can be found in the following transcript(s): HUMTROPIA_PEA2_T10 (SEQ ID NO:22), HUMTROPIA_PEA2_T15 (SEQ ID NO:23), HUMTROPIA_PEA2_T3 (SEQ ID NO:24) and HUMTROPIA_PEA2_T7 (SEQ ID NO:25). Table 138 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1234","num":"1281"},"tables":{"@attributes":{"id":"TABLE-US-00153","num":"00153"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 138"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["HUMTROPIA_PEA_2_T10 (SEQ ID NO: 22)","709","726"]},{"entry":["HUMTROPIA_PEA_2_T15 (SEQ ID NO: 23)","580","597"]},{"entry":["HUMTROPIA_PEA_2_T3 (SEQ ID NO: 24)","1198","1215"]},{"entry":["HUMTROPIA_PEA_2_T7 (SEQ ID NO: 25)","1069","1086"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMTROPIA_PEA2_node25 (SEQ ID NO:141) according to the present invention can be found in the following transcript(s): HUMTROPIA_PEA2_T10 (SEQ ID NO:22), HUMTROPIA_PEA2_T15 (SEQ ID NO:23), HUMTROPIA_PEA2_T3 (SEQ ID NO:24) and HUMTROPIA_PEA2_T7 (SEQ ID NO:25). Table 139 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1236","num":"1283"},"tables":{"@attributes":{"id":"TABLE-US-00154","num":"00154"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 139"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["HUMTROPIA_PEA_2_T10 (SEQ ID NO: 22)","727","741"]},{"entry":["HUMTROPIA_PEA_2_T15 (SEQ ID NO: 23)","598","612"]},{"entry":["HUMTROPIA_PEA_2_T3 (SEQ ID NO: 24)","1216","1230"]},{"entry":["HUMTROPIA_PEA_2_T7 (SEQ ID NO: 25)","1087","1101"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMTROPIA_PEA2_node29 (SEQ ID NO:142) according to the present invention can be found in the following transcript(s): HUMTROPIA_PEA2_T10 (SEQ ID NO:22), HUMTROPIA_PEA2_T15 (SEQ ID NO:23), HUMTROPIA_PEA2_T3 (SEQ ID NO:24) and HUMTROPIA_PEA2_T7 (SEQ ID NO:25). Table 140 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1238","num":"1285"},"tables":{"@attributes":{"id":"TABLE-US-00155","num":"00155"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 140"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["HUMTROPIA_PEA_2_T10 (SEQ ID NO: 22)","742","761"]},{"entry":["HUMTROPIA_PEA_2_T15 (SEQ ID NO: 23)","613","632"]},{"entry":["HUMTROPIA_PEA_2_T3 (SEQ ID NO: 24)","1231","1250"]},{"entry":["HUMTROPIA_PEA_2_T7 (SEQ ID NO: 25)","1102","1121"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMTROPIA_PEA2_node30 (SEQ ID NO:143) according to the present invention can be found in the following transcript(s): HUMTROPIA_PEA2_T10 (SEQ ID NO:22), HUMTROPIA_PEA2_T15 (SEQ ID NO:23), HUMTROPIA_PEA2_T3 (SEQ ID NO:24) and HUMTROPIA_PEA2_T7 (SEQ ID NO:25). Table 141 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1240","num":"1287"},"tables":{"@attributes":{"id":"TABLE-US-00156","num":"00156"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 141"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["HUMTROPIA_PEA_2_T10 (SEQ ID NO: 22)","762","774"]},{"entry":["HUMTROPIA_PEA_2_T15 (SEQ ID NO: 23)","633","645"]},{"entry":["HUMTROPIA_PEA_2_T3 (SEQ ID NO: 24)","1251","1263"]},{"entry":["HUMTROPIA_PEA_2_T7 (SEQ ID NO: 25)","1122","1134"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMTROPIA_PEA2_node31 (SEQ ID NO:144) according to the present invention can be found in the following transcript(s): HUMTROPIA_PEA2_T10 (SEQ ID NO:22), HUMTROPIA_PEA2_T15 (SEQ ID NO:23), HUMTROPIA_PEA2_T3 (SEQ ID NO:24) and HUMTROPIA_PEA2_T7 (SEQ ID NO:25). Table 142 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1242","num":"1289"},"tables":{"@attributes":{"id":"TABLE-US-00157","num":"00157"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 142"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["HUMTROPIA_PEA_2_T10 (SEQ ID NO: 22)","775","798"]},{"entry":["HUMTROPIA_PEA_2_T15 (SEQ ID NO: 23)","646","669"]},{"entry":["HUMTROPIA_PEA_2_T3 (SEQ ID NO: 24)","1264","1287"]},{"entry":["HUMTROPIA_PEA_2_T7 (SEQ ID NO: 25)","1135","1158"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMTROPIA_PEA2_node32 (SEQ ID NO:145) according to the present invention is supported by 40 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMTROPIA_PEA2_T10 (SEQ ID NO:22), HUMTROPIA_PEA2_T15 (SEQ ID NO:23), HUMTROPIA_PEA2_T3 (SEQ ID NO:24) and HUMTROPIA_PEA2_T7 (SEQ ID NO:25). Table 143 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1244","num":"1291"},"tables":{"@attributes":{"id":"TABLE-US-00158","num":"00158"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 143"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["HUMTROPIA_PEA_2_T10 (SEQ ID NO: 22)","799","892"]},{"entry":["HUMTROPIA_PEA_2_T15 (SEQ ID NO: 23)","670","763"]},{"entry":["HUMTROPIA_PEA_2_T3 (SEQ ID NO: 24)","1288","1381"]},{"entry":["HUMTROPIA_PEA_2_T7 (SEQ ID NO: 25)","1159","1252"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMTROPIA_PEA2_node4 (SEQ ID NO:146) according to the present invention can be found in the following transcript(s): HUMTROPIA_PEA2_T10 (SEQ ID NO:22), HUMTROPIA_PEA2_T15 (SEQ ID NO:23), HUMTROPIA_PEA2_T3 (SEQ ID NO:24) and HUMTROPIA_PEA2_T7 (SEQ ID NO:25). Table 144 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1246","num":"1293"},"tables":{"@attributes":{"id":"TABLE-US-00159","num":"00159"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 144"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["HUMTROPIA_PEA_2_T10 (SEQ ID NO: 22)","159","171"]},{"entry":["HUMTROPIA_PEA_2_T15 (SEQ ID NO: 23)","159","171"]},{"entry":["HUMTROPIA_PEA_2_T3 (SEQ ID NO: 24)","159","171"]},{"entry":["HUMTROPIA_PEA_2_T7 (SEQ ID NO: 25)","159","171"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMTROPIA_PEA2_node5 (SEQ ID NO:147) according to the present invention is supported by 6 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMTROPIA_PEA2_T10 (SEQ ID NO:22). Table 145 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1248","num":"1295"},"tables":{"@attributes":{"id":"TABLE-US-00160","num":"00160"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 145"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMTROPIA_PEA_2_T10 (SEQ ID NO: 22)","172","216"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMTROPIA_PEA2_node8 (SEQ ID NO:148) according to the present invention is supported by 27 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMTROPIA_PEA2_T10 (SEQ ID NO:22), HUMTROPIA_PEA2_T3 (SEQ ID NO:24) and HUMTROPIA_PEA2_T7 (SEQ ID NO:25). Table 146 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1250","num":"1297"},"tables":{"@attributes":{"id":"TABLE-US-00161","num":"00161"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 146"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["HUMTROPIA_PEA_2_T10 (SEQ ID NO: 22)","217","266"]},{"entry":["HUMTROPIA_PEA_2_T3 (SEQ ID NO: 24)","172","221"]},{"entry":["HUMTROPIA_PEA_2_T7 (SEQ ID NO: 25)","172","221"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMTROPIA_PEA2_node9 (SEQ ID NO:149) according to the present invention is supported by 27 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMTROPIA_PEA2_T10 (SEQ ID NO:22), HUMTROPIA_PEA2_T3 (SEQ ID NO:24) and HUMTROPIA_PEA2_T7 (SEQ ID NO:25). Table 147 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1252","num":"1299"},"tables":{"@attributes":{"id":"TABLE-US-00162","num":"00162"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 147"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":[{},"starting","ending"]},{"entry":["Transcript name","position","position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["HUMTROPIA_PEA_2_T10 (SEQ ID NO: 22)","267","300"]},{"entry":["HUMTROPIA_PEA_2_T3 (SEQ ID NO: 24)","222","255"]},{"entry":["HUMTROPIA_PEA_2_T7 (SEQ ID NO: 25)","222","255"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}},"br":{}},{"@attributes":{"id":"p-1253","num":"1300"},"tables":{"@attributes":{"id":"TABLE-US-00163","num":"00163"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"273pt","align":"left"}},"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Sequence name: \/tmp\/p5CHmauP3N\/NVyK804uFt:TRIC_HUMAN (SEQ ID NO: 351)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: HUMTROPIA_PEA_2_P5 (SEQ ID NO: 301) \u00d7 TRIC_HUMAN (SEQ ID"},{"entry":"NO: 351)"},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","1183.00","Escore:","0"]},{"entry":["Matching length:","124","Total length:","124"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","99.19"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","99.19"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"273pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00035","num":"00035"},"img":{"@attributes":{"id":"EMI-C00035","he":"31.67mm","wi":"84.92mm","file":"US07714100-20100511-C00035.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/gCDnOSmn31\/GzfEmz5N5Z:TRIC_HUMAN (SEQ ID NO: 351)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: HUMTROPIA_PEA_2_P12 (SEQ ID NO: 302) \u00d7 TRIC_HUMAN (SEQ ID"},{"entry":"NO: 351)\u2003. ."},{"entry":"Alignment segment 1\/1:"},{"entry":"Alignment:"},{"entry":"\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003. \u2003\u2003\u2003\u2003\u2003\u2003. \u2003\u2003\u2003\u2003\u2003\u2003. \u2003\u2003\u2003\u2003\u2003\u2003. \u2003\u2003\u2003\u2003\u2003\u2003."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"70pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","873.00","Length:","209"]},{"entry":["Ratio:","4.823","Gaps:","1"]},{"entry":["Percent Similarity:","86.603","Percent Identity:","86.603"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"273pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"alignment_block:"},{"entry":"HUMTROPIA_PEA_2_P12 (SEQ ID NO: 302) \u00d7 Troponin\u2003. ."},{"entry":"Align seg 1\/1 to: Troponin from: 1 to: 209"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00036","num":"00036"},"img":{"@attributes":{"id":"EMI-C00036","he":"53.51mm","wi":"84.92mm","file":"US07714100-20100511-C00036.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/O8saIrMOll\/UU1NosjzB3:TRIC_HUMAN (SEQ ID NO: 351)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: HUMTROPIA_PEA_2_P17 (SEQ ID NO: 303) \u00d7 TRIC_HUMAN (SEQ ID"},{"entry":"NO: 351)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","344.00","Escore:","0"]},{"entry":["Matching length:","35","Total length:","35"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"273pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00037","num":"00037"},"img":{"@attributes":{"id":"EMI-C00037","he":"10.08mm","wi":"81.96mm","file":"US07714100-20100511-C00037.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: \/tmp\/shMGxspSCh\/hLCzvaPT2j:TRIC_HUMAN (SEQ ID NO: 351)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: HUMTROPIA_PEA_2_P18 (SEQ ID NO: 304) \u00d7 TRIC HUMAN (SEQ ID"},{"entry":"NO: 351)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","71.00","Escore:","0"]},{"entry":["Matching length:","9","Total length:","9"]},{"entry":["Matching Percent Similarity:","88.89","Matching Percent Identity:","88.89"]},{"entry":["Total Percent Similarity:","88.89","Total Percent Identity:","88.89"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"273pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00038","num":"00038"},"img":{"@attributes":{"id":"EMI-C00038","he":"7.54mm","wi":"81.96mm","file":"US07714100-20100511-C00038.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}]}}},"Expression of TRIC_HUMAN Troponin I, cardiac muscle transcripts detectable by or according to seg10 node(s), HUMTROPIA seg10 amplicon(s) (SEQ ID NO:379) and HUMTROPIA seg10F2 (SEQ ID NO:377) and HUMTROPIA seg10R2 (SEQ ID NO:378) primers was measured by real time PCR. In parallel the expression of four housekeeping genes\u2014Ubiquitin (GenBank Accession No. BC000449 (SEQ ID NO:445); amplicon\u2014Ubiquitin-amplicon (SEQ ID NO:448)) and SDHA (GenBank Accession No. NM004168 (SEQ ID NO:449); amplicon\u2014SDHA-amplicon (SEQ ID NO:452)), RPL19 (GenBank Accession No. NM000981 (SEQ ID NO:437); RPL19 amplicon (SEQ ID NO:440)), TATA box (GenBank Accession No. NM003194 (SEQ ID NO:441); TATA amplicon (SEQ ID NO:444)) was measured similarly. For each RT sample, the expression of the above amplicons was normalized to the geometric mean of the quantities of the housekeeping genes. The normalized quantity of each RT sample was then divided by the median of the quantities of the heart samples (Sample Nos. 44-46, Table 2, above \u201cTissue samples in testing panel\u201d), to obtain a value of fold up-regulation for each sample relative to median of the heart.",{"@attributes":{"id":"p-1255","num":"1302"},"figref":"FIG. 21A"},"As is evident from , the expression of TRIC_HUMAN Troponin I, cardiac muscle transcripts detectable by the above amplicon(s) in heart tissue samples was significantly higher than in most other samples (non-heart tissue sample Nos. 1-9, 11-26, 28-43, 47-74 Table 2 above \u201cTissue samples in testing panel\u201d).","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: HUMTROPIA seg10F2 forward primer (SEQ ID NO:377); and HUMTROPIA seg10R2 reverse primer (SEQ ID NO:378).","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: HUMTROPIA seg10(SEQ ID NO:379).",{"@attributes":{"id":"p-1259","num":"1306"},"tables":{"@attributes":{"id":"TABLE-US-00164","num":"00164"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"294pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"147pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["HUMTROPIA seg1 Forward primer (SEQ ID NO: 377):",{}]},{"entry":"TTGCAGAGGGTCATGCTCG"},{"entry":{}},{"entry":"HUMTROPIA seg1 Reverse primer (SEQ ID NO: 378):"},{"entry":"TCCTTTGGATAGGCACTTCCC"},{"entry":{}},{"entry":"HUMTROPIA seg1 Amplicon (SEQ ID NO: 379):"},{"entry":"TTGCAGAGGGTCATGCTCGGATTGGTGACAGCAGCCTGCGGGCGGAACTCCGTTGCCCTCGGACTTGC"},{"entry":{}},{"entry":"TTAGGGATAGATGGGAAGTGCCTATCCAAAGGA"}]}}}}},"Expression of TRIC_HUMAN Troponin I, cardiac muscle transcripts detectable by or according to seg22 node(s), HUMTROPIA seg22 amplicon(s) (SEQ ID NO:382) and HUMTROPIA seg22F (SEQ ID NO:380) and HUMTROPIA seg22R (SEQ ID NO:381) primers was measured by real time PCR. In parallel the expression of four housekeeping genes\u2014RPL19 (GenBank Accession No. NM000981 (SEQ ID NO:437); RPL19 amplicon (SEQ ID NO:440)), TATA box (GenBank Accession No. NM003194 (SEQ ID NO:441); TATA amplicon (SEQ ID NO:444)), Ubiquitin (GenBank Accession No. BC000449 (SEQ ID NO:445); amplicon\u2014Ubiquitin-amplicon (SEQ ID NO:448)) and SDHA (GenBank Accession No. NM004168 (SEQ ID NO:449); amplicon\u2014SDHA-amplicon (SEQ ID NO:452)), was measured similarly. For each RT sample, the expression of the above amplicons was normalized to the geometric mean of the quantities of the housekeeping genes. The normalized quantity of each RT sample was then divided by the median of the quantities of the heart samples (Sample Nos. 44-46, Table 2, above, \u201cTissue samples in testing panel\u201d), to obtain a value of fold up-regulation for each sample relative to median of the heart.",{"@attributes":{"id":"p-1261","num":"1308"},"figref":["FIG. 21B","FIG. 21B"]},"Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: HUMTROPIA seg22F forward (SEQ ID NO:380) primer; and HUMTROPIA seg22R (SEQ ID NO:381) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: HUMTROPIA seg22 (SEQ ID NO:382).",{"@attributes":{"id":"p-1264","num":"1311"},"tables":{"@attributes":{"id":"TABLE-US-00165","num":"00165"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"287pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["HUMTROPIA seg22 Forward primer (SEQ ID NO: 380):",{}]},{"entry":"GTGGGACGCATGGGCA"},{"entry":{}},{"entry":"HUMTROPIA seg22 Reverse primer (SEQ ID NO: 381):"},{"entry":"TTGTCCTGGGTCTCCTGGG"},{"entry":{}},{"entry":"HUMTROPIA seg22 Amplicon (SEQ ID NO: 382):"},{"entry":"GTGGGACGCATGGGCAGCTCGGGTACCTTCGGGGTAGGGTGAGATGGCTGGGACTTGGTCTCTGCCT"},{"entry":{}},{"entry":"GACCCCTTGCAGCTGCTTTTGGCTGCACATCCCAGGAGACCCAGGACAA"}]}}}}},"Expression of TRIC_HUMAN Troponin I, cardiac muscle transcripts detectable by or according to seg23-24-25 node(s), HUMTROPIA seg23-24-25 amplicon(s) and primers HUMTROPIA seg23-24-25F (SEQ ID NO:383) and HUMTROPIA seg23-24-25R (SEQ ID NO:384) was measured by real time PCR. This transcript relates to the known or WT protein (SEQ ID NO:351). In parallel the expression of four housekeeping genes\u2014RPL19 (GenBank Accession No. NM000981 (SEQ ID NO:437); RPL19 amplicon (SEQ ID NO:440)), TATA box (GenBank Accession No. NM003194 (SEQ ID NO:441); TATA amplicon (SEQ ID NO:444)), Ubiquitin (GenBank Accession No. BC000449 (SEQ ID NO:445); amplicon\u2014Ubiquitin-amplicon (SEQ ID NO:448)) and SDHA (GenBank Accession No. NM004168 (SEQ ID NO:449); amplicon\u2014SDHA-amplicon (SEQ ID NO:452)) was measured similarly. For each RT sample, the expression of the above amplicons was normalized to the geometric mean of the quantities of the housekeeping genes. The normalized quantity of each RT sample was then divided by the median of the quantities of the heart samples (Sample Nos. 44-46 Table 2, above), to obtain a value of relative expression for each sample relative to median of the heart samples.",{"@attributes":{"id":"p-1266","num":"1313"},"figref":"FIG. 22"},"As is evident from , the expression of TRIC_HUMAN Troponin I, cardiac muscle transcripts detectable by the above amplicon(s) in heart tissue samples was significantly higher than in the other samples (Sample Nos. 44-46 Table 2, \u201cTissue samples in testing panel\u201d).","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: HUMTROPIA seg23-24-25F (SEQ ID NO:383) forward primer; and HUMTROPIA seg23-24-25FR (SEQ ID NO:384) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: HUMTROPIA seg23-24-25 (SEQ ID NO:384).",{"@attributes":{"id":"p-1270","num":"1317"},"tables":{"@attributes":{"id":"TABLE-US-00166","num":"00166"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"287pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Forward primer HUMTROPIA seg23-24-25F (SEQ ID NO: 383):",{}]},{"entry":"AAGATCTTTGACCTTCGAGGCA"},{"entry":{}},{"entry":"Reverse primer HUMTROPIA seg23-24-25R (SEQ ID NO: 384):"},{"entry":"CTGCTTGAGGTGGGCCC"},{"entry":{}},{"entry":"Amplicon HUMTROPIA seg23-24-25 (SEQ ID NO: 385):"},{"entry":"AAGATCTTTGACCTTCGAGGCAAGTTTAAGCGGCCCACCCTGCGGAGAGTGAGGATCTCTGCAGATG"},{"entry":{}},{"entry":"CCATGATGCAGGCGCTGCTGGGGGCCCGGGCTAAGGAGTCCCTGGACCTGCGGGCCCACCTCAAGCA"},{"entry":{}},{"entry":"G"}]}}}}},"With regard to the variants of this cluster, the following should be noted. Sequence T7 (also referred to herein as HUMTROPIA_PEA2_T7 (SEQ ID NO:25) and troponin T7) has three open reading frames (ORFs) which are described in greater detail below.","The sequence in SEQ ID NO: 354 shows CDS-1 frame 1 from 148 to 406 length 259 (bp)=86 (aa) (similar to Troponin I N-ter)","MADGSSDAAREPRPAPAPIRRRSSNYRAYATEPHAKVGRGFLGAEYRRRRDPRPWEWGEEPGLRRGRGL RGGASGAEFCRGSCSDW*","The sequence in SEQ ID NO: 355 shows CDS-2 frame 1 from 628 to 1183 length 556 (bp)=185 (aa) (similar to Troponin I C-terminal portion)","MILPCSISPWQKKSKISASRKLQLKTLLLQIAKQELEREAEERRGEKGRALSTRCQPLELGLGFAELQDLC RQLHARVDKVDEERYDIEAKVTKNITEIADLTQKIFDLRGKFKRPTLRRVRISADAMMQALLGARAKESL DLRAHLKQVKKEDTEKENREVGDWRKNIDALSGMEGRKKKFES*","The sequence in SEQ ID NO: 356 shows CDS-3 frame 2 from 155 to 629 length 475 (bp)=158 (aa) (Not similar to Troponin I)","MGAAMRLGNLALHQPQSDAAPPTTALMPRSRTPRWDGASWGQSTGAGGIQDPGSGGRSQGCEGGGDYA EGLQGRSFAEGHARIGDSSLRAELRCPRTCLGIDGKCLSKGRDPDWWMGMRGVASRRLRAQVGRGPKSG PAGFAGGVLRSPPPSSPNPPP*","However, the presence of three ORFs could potentially complicate expression and also determination of expression of the desired protein. The first ORF starts at +1 of Troponin sequence (first \u201cATG\u201d is +1 to +3), and the second ORF starts at +8, encoding a 158 amino acid protein. Since the 2nd ATG is located very close to the first one, there is a possibility that it will be expressed as well.","In order to eliminate this possibility of expression of the long ORF, it is possible to optionally introduce two mutations (shown with regard to ):","1. \u201cc\u201d at position 57 to \u201ca\u201d","2. \u201cg\u201d at position 111 to \u201ca\u201d","Both mutations are silent, so the protein sequence will not change.","Cloning and expression verification of a Troponin variant HUMTROPIA_PEA2 T7 was performed as follows.","1. Full Length Validation","1.1. RNA Preparation","Human adult normal heart RNA pool (lot#A411077) was obtained from BioChain Inst. Inc. (Hayward, Calif. 94545 USA dot biochain dot com). Total RNA samples were treated with DNaseI (Ambion Cat # 1906).","1.2. RT PCR","Purified RNA (1 ug) was mixed with 150 ng Random Hexamer primers (Invitrogen Cat # 48190-011) and 500 uM dNTP (Takara, Cat # B9501-1) in a total volume of 15.6 ul DEPC-H2O (Beit Haemek, Cat # 01-852-1A). The mixture was incubated for 5 min at 65\u00b0 C. and then quickly chilled on ice. Thereafter, 5 ul of 5\u00d7 SuperscriptII first strand buffer (Invitrogen, Cat # Y00146), 2.4 ul 0.1M DTT (Invitrogen, Cat #Y00147) and 40 units RNasin (Promega, Cat # N251A) were added, and the mixture was incubated for 2 min at 42\u00b0 C. Then, 1 ul (200 units) of SuperscriptII (Invitrogen, Cat #18064-022) was added and the reaction was incubated for 50 min at 42\u00b0 C. and then inactivated at 70\u00b0 C. for 15 min. The resulting cDNA was diluted 1:20 in TE buffer (10 mM Tris pH=8, 1 mM EDTA pH=8).","1.3. RT-PCR Analysis","cDNA (5 ul), prepared as described above, was used as a template in PCR reactions. The amplification was done using AccuPower PCR PreMix (Bioneer, Korea, Cat# K2016), under the following conditions: 1 ul\u2014of each primer (10 uM)",{"@attributes":{"id":"p-1284","num":"1331"},"tables":{"@attributes":{"id":"TABLE-US-00167","num":"00167"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Tropfor","(SEQ ID NO: 357)","CCCTCACTGACCCTCCAAAC",{}]},{"entry":{}},{"entry":["TropRev","(SEQ ID NO: 358)","CTTCCCATCTATCCCTAAGC"]}]}}}},"br":{},"sub":"2","figref":["FIG. 34","FIG. 35"]},"It was concluded that the predicted Troponin variant is indeed a naturally expressed variant in a normal human tissue as shown in .","2. Cloning of Troponin Variant iInto Bacterial Expression Vector","The Troponin splice variant coding sequence was prepared for cloning by PCR amplification using the fragment described above as template and Platinum Pfx DNA polymerase (Invitrogen Cat # 11708021) under the following conditions: 5 \u03bcl\u2014Amplification \u00d710 buffer (Invitrogen Cat # 11708021); 2 \u03bcl\u2014PCR product from above; 1 \u03bcl\u2014dNTPs (10 mM each); 1 \u03bcl MgSO4 (50 mM) 5 \u03bcl enhancer solution (Invitrogen Cat # 11708021); 33 \u03bcl\u2014HO; 1 \u03bcl\u2014of each primer (10 \u03bcM) and 1.25 units of Taq polymerase [Platinum Pfx DNA polymerase (Invitrogen Cat # 11708021)] in a total reaction volume of 50 \u03bcl with a reaction program of 3 minutes at 94\u00b0 C.; 29 cycles of: [30 seconds at 94\u00b0 C., 30 seconds at 58\u00b0 C., 40 seconds at 68\u00b0 C.] and 7 minutes at 68\u00b0 C. The Primers listed below include specific sequences of the nucleotide sequence corresponding to the splice variant and NheI and HindIII restriction sites.",{"@attributes":{"id":"p-1287","num":"1334"},"tables":{"@attributes":{"id":"TABLE-US-00168","num":"00168"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Trop NheIfor",{},{}]},{"entry":["(SEQ ID NO: 359) -","ACAGCTAGCATGGCGGATGGGAGCAGC"]},{"entry":{}},{"entry":"TropHindIIIrev"},{"entry":["(SEQ ID NO: 360) -","CCTAAGCTTCACCAATCCGAGCATGAC"]}]}}}}},"The PCR product was then double digested with NheI and HindIII (New England Biolabs (UK) LTD), and inserted into pRSET-A (Invitrogen, Cat# V351-20), previously digested with the same enzymes, in-frame to an N-terminal 6His-tag, to give HisTroponin T7 pRSET (; (SEQ ID NO:386)). The coding sequence encodes for a protein having the 6His-tag at the N\u2032 end (6His residues in a row at one end of the protein), and 8 additional amino acids encoded by the pRSET vector.","The sequence of the Troponin insert in the final plasmid, as well as its flanking regions, were verified by sequencing and found to be identical to the desired sequences. The complete sequence of His Troponin T7 pRESTA is shown in  (SEQ ID NO:386).",{"@attributes":{"id":"p-1290","num":"1337"},"figref":"FIG. 38"},"3. Bacterial Cell Growth and Induction of Protein Expression","HisTroponin pRSETA DNA was transformed into competent BL21Gold cells (Stratagene Cat#230134). Ampicillin resistant transformants were screened and positive clones were further analyzed by restriction enzyme digestion and sequence verification.","Cells containing the HisTroponin T7 pRSET vector or empty pRSET vector (as negative control) were grown in LB medium, supplemented with Ampicillin (50 \u03bcg\/ml) and chloramphenicol (34 \u03bcg\/ml). Cells were grown until O.D.reaches 0.5. This value was reached in about 3 hours. 1 mM IPTG (Roche, Cat #724815) was added and the cells were grown at 37\u00b0 C. for additional 3 hours. 1 ml of each culture was removed for gel analysis at Tand T.","3.1. Coomassie Staining and Western Blotting Results","The time course of small-scale expression of Troponin in BL21Gold is demonstrated in -The expression of a recombinant protein with the appropriate molecular weight (11 kDa) was detected both by Coomassie staining () and by Western blot using anti His-antibodies (BD Clontech, Ref 631212) (). It was concluded that the protein encoded by Troponin variant T7 could be expressed in bacterial cells.","HUMTROPIA PEA 2 P17 (SEQ ID NO:303) is a Selective Marker for Cardiac Disease","As described above, the corresponding mRNA from HUMTROPIA_PEA2_T7 (SEQ ID NO:25) was found to be specifically differentially overexpressed in heart tissue, indicating that the corresponding protein (SEQ ID NO:303) could be a useful diagnostic marker for cardiac disease. As described in greater detail below, in order to demonstrate that HUMTROPIA_PEA2_P17 (SEQ ID NO:303) is a selective marker for cardiac disease, selected peptides from HUMTROPIA_PEA2_P17 (SEQ ID NO:303) were used to raise antibodies against the full length protein; such antibodies were shown to bind specifically to HUMTROPIA_PEA2_P17 (SEQ ID NO:303). Furthermore, the antibodies were also demonstrated to specifically differentially detect the presence of HUMTROPIA_PEA2_P17 (SEQ ID NO:303) in serum samples taken from heart patients as opposed to serum samples from patients without heart disease.","Thus, the experimental results demonstrate that HUMTROPIA_PEA2_P17 (SEQ ID NO:303) is a useful diagnostic tool for the diagnosis of heart disease and particularly heart disease which is characterized by damage of heart tissue, as (in a non-limiting example) for myocardial infarction. Damaged heart tissue results in the release of components which are normally located within the heart cell from the intracellular space to the bloodstream. Such components can then be detected and used to diagnose damage to the heart tissue. Thus, when located intracellularly, diagnostic agents for the heart need to be present at a sufficiently high (but not necessarily extremely high) level to be detected in the blood of a patient upon the occurrence of heart damage. Under such circumstances, high levels of overexpression are not required.","HUMTROPIA_PEA2_T7 Cloning, Expression and Purification","Cloning of Troponin Variant HUMTROPIA_PEA2_T7 (SEQ ID NO:25) into Bacterial Expression Vector","HUMTROPIA_PEA2_T7 (SEQ ID NO:25) was originally cloned into a pRSETA vector as described above and was then subcloned from the original construct, into a modified pTrcHisB vector via the Bgl II\/EcoRI restriction sites. The newly constructed subclone product was named pTrcHisB-JL-TNNI3-7 (SEQ ID NO:461). The sequence of the subclone was confirmed by use of the CEQ 8000 Genetic Analysis System (Beckman Coulter, Fullerton Calif., USA). This sequence is provided in , which shows the nucleic acid sequence of the modified pTrcHisB vector, cotaining SEQ ID NO:459 and the corresponding amino acid sequence, SEQ ID NO:460, featuring an additional His-Tag sequence for ease of purification and an additional vector-provided C-terminal sequence of the recombinant protein. The underlined sequence relates to the His-Tag sequence; double underlining indicates the actual sequence of HUMTROPIA_PEA2_P17 (SEQ ID NO:303); and italic type indicates the additional vector-provided C-terminal sequence of the recombinant protein.","Recombinant HUMTROPIA_PEA2_P17 (SEQ ID NO:460) Purification","The following protocol was used to initially purify recombinant HUMTROPIA_PEA2_P17 (SEQ ID NO:460) for use in immunization and screening. TOP10 cells were transformed with TrcHisB-JL-TNNI3-7 (SEQ ID NO:461) and were grown to ODof 0.6. Cells were induced with 1 mM IPTG final concentration for 4 hr. at 37\u00b0 C. Cells were centrifuged and the pellet was frozen. After thawing, the pellet was lysed with non-denaturing binding buffer (20 mM Tris.HCl pH 7.9, 0.5 M NaCl, 5 mM imidazole). Cells were sonicated, centrifuged again and the supernatant was discarded. The pellet was resuspended in denaturing binding buffer (20 mM Tris.HCl pH 7.9, 0.5 M NaCl, 5 mM imidazole, 6 M guanidine.HCl). Cells were sonicated and centrifuged twice more (in each case the supernatant was collected).","His-tagged HUMTROPIA_PEA2_P17 (SEQ ID NO:460) was bound to Nicharged Sephadex beads. The protein-resin slurry was transferred to the purification column. The column was washed with denaturing wash buffer (20 mM Tris.HCl pH 7.9, 0.5 M NaCl, 20 mM imidazole, 6 M urea). Bound HUMTROPIA_PEA2_P17 (SEQ ID NO:460) was eluted from the column with denaturing elution buffer (20 mM Tris.HCl pH 7.9, 0.5 M NaCl, 500 mM imidazole, 6 M urea). Fractions were analyzed by SDS-PAGE; protein-containing fractions were dialyzed several times against 100 mM AcOH\/NaOAc buffer, pH 4.0 containing 1 mM cysteine and 1 mM cystine; again against 100 mM AcOH\/NaOAc buffer, pH 4.0 without cysteine\/cystine; and again against 20 mM N(Bu)OAc buffer, pH 6.0. Yield and percent purity of HUMTROPIA_PEA2_P17 (SEQ ID NO:460) expression were determined by SDS-PAGE and Western blotting via the anti-His-tag G MAb (see  for gel results) and BCA assay for total protein content. Lanes are as follows: M: BioRad BMW (204, 115.4, 94.9, 54.3, 37.3, 29.1, 20.1, and 7.1 kD); lane 1: HUMTROPIA_PEA2_P17 (SEQ ID NO: 460)\/050225SM\/EC; lane 2: Control-HUMTROPIA_PEA2_P17 (SEQ ID NO:460) (non-purified); and lane 3: Control-HUMTROPIA_PEA2_P17 (SEQ ID NO:460) (mini-expression, 37\u00b0 3 hr). A yield of 74 mg of protein was determined, with an estimated purity of \u02dc90%.","Polyclonal Antibody Development","Peptide Synthesis","TNNI3-WT and the various TNNI3-7 peptide sequences were synthesized and used for reactivity testing of serum samples collected from each rabbit immunized with recombinant protein HUMTROPIA_PEA2_P17",{"@attributes":{"id":"p-1303","num":"1350"},"tables":{"@attributes":{"id":"TABLE-US-00169","num":"00169"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["(SEQ ID NO: 460):",{}]},{"entry":"(TNNI3-WT refers to the known or \u201cWT\u201d\u2003protein,"},{"entry":"Troponin I from cardiac muscle (SEQ ID NO: 351)):"},{"entry":"TNNI3-WT #1-36 (SEQ ID NO: 453):"},{"entry":"MADGS-SDAAR-EPRPA-PAPIR-RRSSN-YRAYA-TEPHA-K"},{"entry":{}},{"entry":"HUMTROPIA_PEA_2_P17 #37-86 (SEQ ID NO: 454):"},{"entry":"VGRGF-LGAEY-RRRRD-PRPWE-WGEEP-GLRRG-RGLRG-GASGA-"},{"entry":"EFCRG-SCSDW"},{"entry":{}},{"entry":"HUMTROPIA_PEA_2_P17 #37-53 (SEQ ID NO: 455):"},{"entry":"VGRGF-LGAEY-RRRRD-PR"},{"entry":{}},{"entry":"HUMTROPIA_PEA_2_P17 #53-68 (SEQ ID NO: 456):"},{"entry":"RPWE-WGEEP-GLRRG-RG"},{"entry":{}},{"entry":"HUMTROPIA_PEA_2_P17 #69-86, Cys(AcM)-79, 83"},{"entry":"(SEQ ID NO: 457):"},{"entry":"LRG-GASGA-EFCRG-SCSDW"}]}}}}},"Cys-modified HUMTROPIA_PEA2_P17 #36-86 Cys36 Cys(AcM)-79,83 peptide (SEQ ID NO:458) was conjugated at the N-terminus to Sulfolink\u2122 for Pab purification, and was used for polyclonal antibodies purification, as described below.","AcM=S-acetamidomethyl group","Anti-HUMTROPIA PEA 2 P17 (SEQ ID NO:303) Polyclonal Antibody Production","Rabbit Immunization and Bleeding Cycles","Two rabbits were immunized with recombinant protein HUMTROPIA_PEA2_P17 (SEQ ID NO:460) in order to generate the corresponding polyclonal antibodies. 200 \u03bcg and 100 \u03bcg of recombinant HUMTROPIA_PEA2_P17 (SEQ ID NO:460) were used for primary and secondary immunizations, respectively. After each primary immunization, a series of four-week secondary immunization cycles followed. Each cycle consisted of a secondary immunization on week 1, a bleed on week 3 and a re-immunization again on week 4. Immunization\/bleeding cycles were repeated in order to generate high titer serum. Approximately 18-20 ml sera were collected from each rabbit per cycle. Serum samples were then tested for reactivity to the WT-shared peptide (TNNI3-WT #1-36 (SEQ ID NO:453)), the unique peptide (HUMTROPIA_PEA2_P17 #37-86 (SEQ ID NO:454)), and three fragments of the unique peptide (HUMTROPIA_PEA2_P17 #37-53 (SEQ ID NO:455, unique-1), HUMTROPIA_PEA2_P17 #53-68 (SEQ ID NO:456, unique-2), and HUMTROPIA_PEA2_P17 #69-86 Cys(AcM)-79,83 (SEQ ID NO:457, unique-3)). The results are shown in Table 148 below.",{"@attributes":{"id":"p-1308","num":"1355"},"tables":{"@attributes":{"id":"TABLE-US-00170","num":"00170"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"161pt","align":"center"}}],"thead":{"row":{"entry":[{},"TABLE 148"]}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]},{"entry":[{},"Response to:"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Rabbit","(WT)","(unique)","(unique-1)","(unique-2)","(unique-3)"]},{"entry":[{},"#","#1-36","#37-86","#37-53","#53-68","#69-86"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"6","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Results:","R4172","+++++","+++","++++","\u2212","\u2212"]},{"entry":[{},"R4173","+++++","++","+","\u2212","\u2212"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}}]}}]}}},"While the largest response for both rabbit sera was to the portion of HUMTROPIA_PEA2_P17 (SEQ ID NO:303) that is shared with the known protein (SEQ ID NO: 351), it is clear that both sera responded to the first third of the unique peptide, though the response of rabbit R4173 was much lower than that of R4172. Bleeds that generated high titers were selected and used for further evaluation.","Anti-HUMTROPIA_PEA2_P17 (SEQ ID NO:303) PAb Purification","60 ml of serum collected from rabbits R4172 and R4173 at RT was treated with 2 M NaOAc pH 4.0 with stirring. pH was adjusted to 4.8, followed by addition to a final concentration of 7% v\/v caprylic acid. Treated serum was stirred at RT for 30 min., then cooled to 4\u00b0 C., and centrifuged at 8,000 g for 25 min. at 4\u00b0 C. Supernatant was collected and filtered through extra-thick glass fiber filter, then neutralized by dialysis against 1\u00d7PBS with 0.1% NaN. Caprylic acid treated serum was loaded onto a Sulfolink\u2122 affinity matrix that had been conjugated to N-terminally Cys-modified HUMTROPIA_PEA2_P17 #36-86 Cys36 Cys(AcM)-79,83 peptide (SEQ ID NO:458). PAb specific to this unique peptide resin was then eluted with 0.1 M glycine, pH 2.6, 150 mM NaCl, 76 mM ACA (\u03b5-aminocaproic acid).","Polyclonal Antibody Assay Development","The antibodies were used with an ELISA assay in order to determine their reactivity and specificity. Samples of the PAb were then tested for reactivity to either 1) recombinant HUMTROPIA_PEA2_P17 (SEQ ID NO:460) protein, 2) the WT-shared peptide (#1-36 (SEQ ID NO:453)), 3) the entire unique peptide (#37-86 (SEQ ID NO:454)), or 4-6) the three overlapping peptides comprising the unique peptide (SEQ ID NO:455-457), using the following conditions:","Plate Coat: 1000 ng\/ml recombinant HUMTROPIA_PEA2_P17 (SEQ ID NO:460) (050225SM\/EC), TNNI3-WT #1-36 (PC179-EH23-19; (SEQ ID NO:453)), HUMTROPIA_PEA2_P17 (SEQ ID NO:303) #37-86 (PC179-EH23-9; (SEQ ID NO:454)), HUMTROPIA_PEA2_P17 (SEQ ID NO:303) #37-53 (PC179-EH23-30; (SEQ ID NO:455)), HUMTROPIA_PEA2_P17 (SEQ ID NO:303) #53-68 (PC173-BG2-29; (SEQ ID NO:456)), or HUMTROPIA_PEA2_P17 #69-86 Cys(AcM)-79,83 (PC173-BG2-30; (SEQ ID NO:457)) in 1\u00d7PBS, 0.1% NaN; 2 hr. at RT",{"@attributes":{"id":"p-1313","num":"1360"},"tables":{"@attributes":{"id":"TABLE-US-00171","num":"00171"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"168pt","align":"left"}}],"thead":{"row":{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Sample:","0.01-10 \u03bcg\/ml Affinity-purified"]},{"entry":[{},{},"Anti-HUMTROPIA_PEA_2_P17"]},{"entry":[{},{},"(SEQ ID NO: 303) PAb (R4172\/R4173) in DB"]},{"entry":[{},"Tracer:","1:20,000 diluted goat anti-mouse (Fc) IgG-HRP or"]},{"entry":[{},{},"1:10,000 diluted goat anti-rabbit"]},{"entry":[{},{},"(Fc) IgG-HRP conjugate; 30 min. RT"]},{"entry":[{},"Substrate:","TMB Substrate for 15 min. at RT, then Acid Stopping"]},{"entry":[{},{},"Solution"]},{"entry":[{},"O.D.:","450 nm (average of duplicates) results were analyzed"]},{"entry":[{},{},"(results not shown)."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}}}},"From the results, it is possible to conclude the following about the Pab antibodies. The PAb responded strongly to the intact recombinant protein (as expected), and to the entire 50 aa unique peptide (SEQ ID NO:454), as well as to the first fragment of that unique region (i.e., #37-53; (SEQ ID NO:455)). There was also a strong response to the WT-shared region, likely due to imperfect affinity purification of the original rabbit serum on the unique peptide-resin column.","Serum Screening","As described in greater detail below, human serum from patients suffering from various heart diseases or conditions, along with normal control samples, were prepared and were subjected to a sandwich assay. The results show that the antibodies are able to detect the HUMTROPIA_PEA2_P17 (SEQ ID NO:303) of the present invention in both healthy and diseased serum samples, thereby confirming the utility of HUMTROPIA_PEA2_P17 (SEQ ID NO:303) as a diagnostic agent for heart disease in humans.",{"@attributes":{"id":"p-1316","num":"1363"},"tables":{"@attributes":{"id":"TABLE-US-00172","num":"00172"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"HUMTROPIA_PEA_2_P17 (SEQ ID NO: 303) (PAb: PAb Sandwich)"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"175pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Plate Coat:","5 \u03bcg\/ml Anti-HUMTROPIA_PEA_2_P17"]},{"entry":[{},"(SEQ ID NO: 303) PAb (R4172\/R4173) (050823HW)"]},{"entry":[{},"in 1X PBS, 0.1% NaN; 2 hr. at RT"]},{"entry":["Sample:","0-200 ng\/ml HUMTROPIA_PEA_2_P17"]},{"entry":[{},"(SEQ ID NO: 460) (050225SM\/EC) in DB with 1:4 NHS or"]},{"entry":[{},"1:4 dilution of human sample serum in DB; 1 hr. at RT"]},{"entry":["Tracer:","2 \u03bcg\/ml Anti-HUMTROPIA_PEA_2_P17"]},{"entry":[{},"(SEQ ID NO: 303) PAb-AP (R4172\/R4173)"]},{"entry":[{},"(050825HW) in DB; 60 min. at RT"]},{"entry":["Substrate:","PNPP Substrate for 1 hr. at RT"]},{"entry":["O.D.:","405 nm (average of duplicates)"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The results are shown in  graphically presented in .  include a description of the serum samples. The key to  is as follows:\n\n","True positives are defined as cardiovascular patient samples with assay signals greater than the median normal control+2 standard deviations (i.e., TP signal\u2267Median+2S.D.).","HUMTROPIA_PEA2_P17 (SEQ ID NO:303) was monitored in serum with a sandwich assays using the Pab antibodies on the top and bottom. Use of the PAb:PAb sandwich resulted in 69% sensitivity (though about 17% of the 69% were borderline positive values) and 90% specificity. Note that five of the true positive samples did, in fact, exhibit quite high signals (i.e., \u02dc2 O.D.). While these results are very encouraging for HUMTROPIA_PEA2_P17 (SEQ ID NO:303), the signal for the negative controls of HUMTROPIA_PEA2_P17 (SEQ ID NO:303) was relatively high (i.e., median control=0.512 O.D, cut off=0.628.).","Monoclonal Antibody Development","HUMTROPIA_PEA2_P17 (SEQ ID NO:303) Mouse Immunization, Bleeding, and Hybridoma Fusion","Five (5) Balb\/C mice were immunized subcutaneously twice each with 100 \u03bcg 1 mg\/ml of recombinant HUMTROPIA_PEA2_P17 (SEQ ID NO:460) immunogen. In the final boost, mice each were injected intraperitoneally with 150 \u03bcg of immunogen, prior to fusion with the myeloma cell line. Various dilutions of bleeds were evaluated for reactivity to the WT-shared peptide (TNNI3-WT #1-36 (SEQ ID NO:453)), the unique peptide (HUMTROPIA_PEA2_P17 #37-86 (SEQ ID NO:454)), and three fragments of the unique peptide (HUMTROPIA_PEA2_P17 #37-53 (SEQ ID NO:455, unique-1), HUMTROPIA_PEA2_P17 #53-68 (SEQ ID NO:456, unique-2), and HUMTROPIA_PEA2_P17 #69-86 Cys(AcM)-79,83 (SEQ ID NO:457, unique-3)). The results are shown in Table 149 below.",{"@attributes":{"id":"p-1321","num":"1370"},"tables":{"@attributes":{"id":"TABLE-US-00173","num":"00173"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"161pt","align":"center"}}],"thead":{"row":{"entry":[{},"TABLE 149"]}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]},{"entry":[{},"Response to:"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Mouse","(WT)","(unique)","(unique-1)","(unique-2)","(unique-3)"]},{"entry":[{},"#","#1-36","#37-86","#37-53","#53-68","#69-86"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"6","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Results:","#6","++++","+++","\u2212","+","\u2212"]},{"entry":[{},"#7","++++","+++","\u2212","\u2212","\u2212"]},{"entry":[{},"#8","+++++","++++","+++","++++","\u2212"]},{"entry":[{},"#9","+++++","++++","\u2212","++","\u2212"]},{"entry":[{},"#10\u2002","+++++","++++","\u2212","++","\u2212"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}}]}}]}}},"All five mice elicited very strong responses to the WT-shared peptide and strong responses to the entire 50 aa unique peptide. None of the mice elicited any response to the 3fragment of the unique region, and only one mouse, #8, elicited a strong response to the 1fragment. As for the 2peptide fragment, mouse #8 showed a very strong response, while mice #9 & #10 exhibited moderate responses. Mouse #6 showed a weak response to the 2fragment, but mouse #6 exhibited no response at all to this sequence.","Mouse #8 was selected for further fusion work. A total of five clones were developed.","Antibody Production","For each hybridoma cell line, a total of five mice were primed intraperitoneally with pristine, followed by injection of the hybridoma cell suspensions. Ascitic fluid was collected and pooled from all five mice for each clone.","Monoclonal Antibody Purification","Ascites samples containing each anti-HUMTROPIA_PEA2_P17 (SEQ ID NO:303) MAb (1B3, 1B4, 2E3, 5F7, and 11D8) were purified on recombinant Protein A-modified Sepharose Fast Flow resin, as follows: Samples first were diluted with an equal volume of Binding Buffer (12.7 mM sodium phosphate, pH 7.4, 187 mM NaCl, 0.125% NaN, 95.3 mM ACA), loaded onto Protein-A matrix and eluted with Elution Buffer (0.1 M citric acid, 76 mM ACA), pH 6.0, and then neutralized with \u201cProtein-A Neutralizing Buffer for pH 6\u201d (0.46 M Tris base, 2.3 M NaCl). Columns were then washed with Protein-A Elution Buffer, pH 3, and finally neutralized with \u201cProtein-A Neutralizing Buffer for pH 3\u201d (0.1.5 M Tris base, 0.75 M NaCl, 0.5% NaN). Purified MAbs were concentrated on Amicon YM-30 membranes and dialyzed into 1\u00d7PBS (10 mM sodium phosphate, pH 7.4, 150 mM NaCl), 0.1% w\/v NaN.","Monoclonal Antibody Biotinylation","Biotinylation was performed in Biochemistry by reacting a 20-fold molar excess of EZ-Link\u00ae NHS-LC-Biotin (Pierce Biotechnology Catalog #21336) with antibody for 1 hr. at RT in 0.1 M carbonate buffer, pH 9.5, followed by dialysis into 1\u00d7PBS buffer, pH 7.4, with ProClin (10 mM sodium phosphate, pH 7.4, 150 MM NaCl, 0.05% ProClin 300). Biotinylated antibodies (biotinylated MAbs 1B3, 1B4, 2E3, 5F7, and 11D8: Lot #060412SD) were transferred to Assay Development.","HUMTROPIA PEA 2 P17 (SEQ ID NO:303) Assay Development","Epitope Mapping","Samples of each MAb were then tested for reactivity to either 1) recombinant HUMTROPIA_PEA2_P17 (SEQ ID NO:460) protein, 2) the WT-shared peptide (#1-36; (SEQ ID NO:453)), 3) the entire unique peptide (#37-86; (SEQ ID NO:454)), or 4, 5, & 6) the three overlapping peptides comprising the unique peptide (SEQ ID NOs:455-457), using the following conditions:",{"@attributes":{"id":"p-1328","num":"1377"},"tables":{"@attributes":{"id":"TABLE-US-00174","num":"00174"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"168pt","align":"left"}}],"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Plate Coat:","1000 ng\/ml recombinant HUMTROPIA_PEA_2_P17"]},{"entry":[{},"(SEQ ID NO: 460) (050225SM\/EC), TNNI3-WT #1-36"]},{"entry":[{},"(PC179-EH23-19 (SEQ ID NO: 453)),"]},{"entry":[{},"HUMTROPIA_PEA_2_P17 #37-86"]},{"entry":[{},"(PC179-EH23-9 (SEQ ID NO: 454)),"]},{"entry":[{},"HUMTROPIA_PEA_2_P17 #37-53"]},{"entry":[{},"(PC179-EH23-30 (SEQ ID NO: 455)),"]},{"entry":[{},"HUMTROPIA_PEA_2_P17"]},{"entry":[{},"#53-68 (PC173-BG2-29 (SEQ ID NO: 456)), or"]},{"entry":[{},"HUMTROPIA_PEA_2_P17 #69-86 Cys(AcM)-79,83"]},{"entry":[{},"(PC173-BG2-30 (SEQ ID NO: 457)) in 1X PBS,"]},{"entry":[{},"0.1% NaN; 2 hr. at RT"]},{"entry":["Sample:","0.01-10 \u03bcg\/ml Anti-HUMTROPIA_PEA_2_P17"]},{"entry":[{},"(SEQ ID NO: 303) MAb (1B3, 1B4, 2E3, 5F7,"]},{"entry":[{},"or 11D8) in DB"]},{"entry":["Tracer:","1:20,000 diluted goat anti-mouse (Fc) IgG-HRP or "]},{"entry":[{},"1:10,000 diluted goat anti-rabbit (Fc) IgG-"]},{"entry":[{},"HRP conjugate; 30 min. RT"]},{"entry":["Substrate:","TMB Substrate for 15 min. at RT, then Acid Stopping"]},{"entry":[{},"Solution"]},{"entry":["O.D.:","450 nm (average of duplicates)"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}}}},"Results are shown in . The results show that MAbs 1B3, 2E3, and 5F7 all respond strongly to the intact recombinant protein (as expected), and to the entire 50 aa unique peptide (SEQ ID NO:454), as well as to the first fragment of that unique region (i.e., #37-53; SEQ ID NO:455), but not to the WT-shared peptide sequence (SEQ ID NO:453), thereby overcoming a drawback of the polyclonal antibodies as described above.","MAb 1B4 exhibits a significantly different epitope specificity. Like the above listed antibodies, 1B4 responds strongly to the recombinant protein and to the entire 50 aa unique peptide (SEQ ID NO:454), and does not respond to the WT-shared sequence (SEQ ID NO:453). Unlike the previous set of antibodies, however, the strong response of 1B4 to the unique region appears to be due to a moderate affinity for the second fragment of the unique sequence (SEQ ID NO:456).","MAb 11D8 responds moderately to the recombinant protein, but does not show any obvious affinity for any specific peptide fragments from the HUMTROPIA_PEA2_P17 (SEQ ID NO:303) protein sequence. It is possible that the epitope for this antibody is within the native variant sequence, but that the epitope is not composed of a single linear fragment of the protein, and, in fact, requires an intact 3-dimensional structure for effective binding.","Antibody Pairing","In order to develop the optimal sandwich assay for HUMTROPIA_PEA2_P17 (SEQ ID NO:303), each of the six antibodies was paired in all possible combinations (i.e., 6\u00d76 pairs, including the previously described polyclonal antibodies):",{"@attributes":{"id":"p-1333","num":"1382"},"tables":{"@attributes":{"id":"TABLE-US-00175","num":"00175"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"175pt","align":"left"}}],"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Plate Coat:","5 \u03bcg\/ml each MAb or PAb in 1X PBS,"]},{"entry":[{},"0.1% NaN; 2 hr. at RT"]},{"entry":["Sample:","0 or 100 ng\/ml recombinant HUMTROPIA_PEA_2_P17"]},{"entry":[{},"(SEQ ID NO: 460) (050225SM\/EC) in DB"]},{"entry":["Tracer:","2 \u03bcg\/ml MAb-biotin (followed by addition"]},{"entry":[{},"of 200 ng\/ml streptavidin-AP, for MAb detection only,"]},{"entry":[{},"in DB; 30 min. RT) or PAb-AP conjugate; 30 min. RT"]},{"entry":["Substrate:","PNPP Substrate; 1 hr. at RT"]},{"entry":["O.D.:","405 nm (average of duplicates)"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}}}},"Results are shown in . As shown, it is clear that only a limited number of pairs result in low background and relatively high dynamic range. These are 1B3:1B4, 1B3:PAb, 1B4:1B3, 1B4:2E3, and 1B4:5F7. Three pairs resulted in low background, but with only a moderate dynamic range: 2E3:PAb, 5F7:1B4, and 5F7:PAb. In many cases, use of 2E3 or PAb as a coat antibody (i.e., capture antibody) resulted in very high background signals. Use of 11D8 as a coat antibody resulted in no detectable capture of recombinant HUMTROPIA_PEA2_P17 (SEQ ID NO:460) at all.","The results shown in the lower right hand corner correspond to those demonstrated above for a PAb\/PAb sandwich; however, clearly the monoclonal antibodies achieved superior results in the sandwich assay.","Serum Screening","As described in greater detail below, human serum from patients suffering from various heart diseases or conditions, along with normal control samples, were prepared and were subjected to a sandwich assay. The results show that the antibodies are able to detect the HUMTROPIA_PEA2_P17 (SEQ ID NO:303) of the present invention in both healthy and diseased serum samples, thereby confirming the utility of HUMTROPIA_PEA2_P17 (SEQ ID NO:303) as a diagnostic agent for heart disease in humans.","TNNI3-WT (Commercial MAb:PAb Sandwich)",{"@attributes":{"id":"p-1337","num":"1386"},"tables":{"@attributes":{"id":"TABLE-US-00176","num":"00176"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"175pt","align":"left"}}],"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Plate Coat:","5 \u03bcg\/ml MAb 8E10 (HyTest; DPC79143) in"]},{"entry":[{},"1X PBS, 0.1% NaN; 2 hr. at RT"]},{"entry":["Sample:","0-159 ng\/ml TNNI3-WT Immulite\u2009\u00ae Troponin I Calibrator"]},{"entry":[{},"(LTI 3-X2 0004) with 1:4 NHS (normal human serum) or"]},{"entry":[{},"1:4 dilution of human sample serum in DB; 1 hr. at RT"]},{"entry":["Tracer:","2 \u03bcg\/ml Anti-Peptide 3 PAb-AP (BiosPacific;"]},{"entry":[{},"DPC66518) in DB; 30 min. at RT"]},{"entry":["Substrate:","PNPP Substrate for 1 hr. at RT"]},{"entry":["O.D.:","405 nm (average of duplicates)"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}}},"br":{},"sub":"\u2014"},{"@attributes":{"id":"p-1338","num":"1387"},"tables":{"@attributes":{"id":"TABLE-US-00177","num":"00177"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"175pt","align":"left"}}],"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Plate Coat:","5 \u03bcg\/ml Anti-HUMTROPIA_PEA_2_P17"]},{"entry":[{},"(SEQ ID NO: 303) PAb (R4172\/R4173) (050823HW)"]},{"entry":[{},"in 1X PBS, 0.1% NaN; 2 hr. at RT"]},{"entry":["Sample:","0-200 ng\/ml recombinant HUMTROPIA_PEA_2_P17"]},{"entry":[{},"(SEQ ID NO: 460) (050225SM\/EC) in DB with 1:4 NHS or"]},{"entry":[{},"1:4 dilution of human sample serum in DB; 1 hr. at RT"]},{"entry":["Tracer:","2 \u03bcg\/ml Anti-HUMTROPIA_PEA_2_P17"]},{"entry":[{},"(SEQ ID NO: 303) PAb-AP (R4172\/R4173)"]},{"entry":[{},"(050825HW) in DB; 60 min. at RT"]},{"entry":["Substrate:","PNPP Substrate for 1 hr. at RT"]},{"entry":["O.D.:","405 nm (average of duplicates)"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}}},"br":{},"sub":"\u2014"},{"@attributes":{"id":"p-1339","num":"1388"},"tables":{"@attributes":{"id":"TABLE-US-00178","num":"00178"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"175pt","align":"left"}}],"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Plate Coat:","5 \u03bcg\/ml Anti-HUMTROPIA_PEA_2_P17"]},{"entry":[{},"(SEQ ID NO: 303) 1B3 MAb (051221TQ) in 1X PBS,"]},{"entry":[{},"0.1% NaN; 2 hr. at RT"]},{"entry":["Sample:","0-200 ng\/ml recombinant HUMTROPIA_PEA_2_P17"]},{"entry":[{},"(SEQ ID NO: 460) (050225SM\/EC) in DB with 1:4 NHS or"]},{"entry":[{},"1:4 dilution of human sample serum in DB; 1 hr. at RT"]},{"entry":["Tracer:","2 \u03bcg\/ml Anti-HUMTROPIA_PEA_2_P17"]},{"entry":[{},"(SEQ ID NO: 303) PAb-AP (R4172\/R4173)"]},{"entry":[{},"(050825HW) in DB; 60 min. at RT"]},{"entry":["Substrate:","PNPP Substrate for 1 hr. at RT"]},{"entry":["O.D.:","405 nm (average of duplicates)"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}}},"br":{},"sub":"\u2014"},{"@attributes":{"id":"p-1340","num":"1389"},"tables":{"@attributes":{"id":"TABLE-US-00179","num":"00179"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"287pt","align":"left"}}],"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Plate Coat:","5 \u03bcg\/ml Anti-HUMTROPIA_PEA_2_P17 (SEQ ID NO: 303) 1B4 MAb (060127SMS) in 1X"]},{"entry":[{},"PBS, 0.1% NaN; 2 hr. at RT"]},{"entry":["Sample:","0-200 ng\/ml recombinant HUMTROPIA_PEA_2_P17 (SEQ ID NO: 460) (050225SM\/EC) in DB"]},{"entry":[{},"with 1:4 NHS or 1:4 dilution of human sample serum in DB; 1 hr. at RT"]},{"entry":["Tracer:","2 \u03bcg\/ml Anti-HUMTROPIA_PEA_2_P17 (SEQ ID NO: 303) 5F7 MAb-biotin-Lc25 (060412SD)"]},{"entry":[{},"in DB; 60 min. at RT; (followed by addition of 200 ng\/ml streptavidin-AP in DB; 30 min."]},{"entry":[{},"RT)"]},{"entry":["Substrate:","PNPP Substrate for 1 hr. at RT"]},{"entry":["O.D.:","405 nm (average of duplicates)"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}}},"br":{},"figref":"FIG. 45","ul":{"@attributes":{"id":"ul0009","list-style":"none"},"li":{"@attributes":{"id":"ul0009-0001","num":"0000"},"ul":{"@attributes":{"id":"ul0010","list-style":"none"},"li":["u: unknown","cvd: cardiovascular disease","N: normal controls"]}}}},"The results are shown in . True positives are defined as cardiovascular patient samples with assay signals greater than the median normal control+2 standard deviations (i.e., TP signal\u2267Median+2S.D.).","Use of the commercial TNNI3-WT antibody pair (i.e., 8E10 MAb: Anti-Peptide 3 PAb) in an ELISA format resulted in identification of 23 of the 42 cardiac patient samples (55% sensitivity), with six false positives (80% specificity). This is compared with the automated Immulite\u00ae results using the same antibody pair that gave 97.6% sensitivity and 100% specificity for this patient panel.","HUMTROPIA_PEA2_P17 (SEQ ID NO:303) was monitored in serum by three different sandwich assays: PAb:PAb, MAb:PAb, & MAb:MAb. Use of the PAb:PAb sandwich resulted in 69% sensitivity (though about 17% of the 69% were borderline positive values) and 90% specificity. Note that five of the true positive samples did, in fact, exhibit quite high signals (i.e., \u02dc2 O.D.).","Summary","HUMTROPIA_PEA2_P17 (SEQ ID NO:303) appears to be a specific molecular diagnostic marker of cardiac tissue. In order to check whether HUMTROPIA_PEA2_P17 (SEQ ID NO:303) protein expression parallels that of the corresponding mRNA, recombinant HUMTROPIA_PEA2_P17 (SEQ ID NO:303) protein was expressed, and both MAb and PAb specific for HUMTROPIA_PEA2_P17 (SEQ ID NO:303) were prepared and characterized.","Use of the optimized PAb:PAb sandwich assay formats for HUMTROPIA_PEA2_P17 (SEQ ID NO:303) protein led to sensitivities of 52-69% with specificities of 90-93% for cardiac patient serum samples. Use of optimized sandwich assay formats for HUMTROPIA_PEA2_P17 (SEQ ID NO:303) protein with MAb provided results with lower sensitivities and specificities as compared to PAb:PAb sandwich assay formats.","These assay development results suggests that HUMTROPIA_PEA2_P17 (SEQ ID NO:303) is present to a higher extent in serum samples derived from diseased patients as compared to serum derived from healthy individuals.","Cluster HUMSMCK features 5 transcript(s) and 14 segment(s) of interest, the names for which are given in Tables 150 and 151, respectively, the sequences themselves are given at the end of the application. The selected protein variants are given in table 152.",{"@attributes":{"id":"p-1348","num":"1400"},"tables":{"@attributes":{"id":"TABLE-US-00180","num":"00180"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 150"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Transcripts of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Transcript Name","Seq ID No."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"HUMSMCK_T5","26"]},{"entry":[{},"HUMSMCK_T6","27"]},{"entry":[{},"HUMSMCK_T7","28"]},{"entry":[{},"HUMSMCK_T9","29"]},{"entry":[{},"HUMSMCK_T11","30"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-1349","num":"1401"},"tables":{"@attributes":{"id":"TABLE-US-00181","num":"00181"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 151"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segments of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment Name","Seq ID No."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"HUMSMCK_node_0","150"]},{"entry":[{},"HUMSMCK_node_7","151"]},{"entry":[{},"HUMSMCK_node_12","152"]},{"entry":[{},"HUMSMCK_node_17","153"]},{"entry":[{},"HUMSMCK_node_22","154"]},{"entry":[{},"HUMSMCK_node_23","155"]},{"entry":[{},"HUMSMCK_node_25","156"]},{"entry":[{},"HUMSMCK_node_26","157"]},{"entry":[{},"HUMSMCK_node_28","158"]},{"entry":[{},"HUMSMCK_node_29","159"]},{"entry":[{},"HUMSMCK_node_32","160"]},{"entry":[{},"HUMSMCK_node_11","161"]},{"entry":[{},"HUMSMCK_node_14","162"]},{"entry":[{},"HUMSMCK_node_19","163"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-1350","num":"1402"},"tables":{"@attributes":{"id":"TABLE-US-00182","num":"00182"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 152"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Proteins of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"119pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Protein Name","Seq ID No.","Corresponding Transcript(s)"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMSMCK_P4","305","HUMSMCK_T5 (SEQ ID NO: 26)"]},{"entry":["HUMSMCK_P5","306","HUMSMCK_T6 (SEQ ID NO: 27)"]},{"entry":["HUMSMCK_P6","307","HUMSMCK_T7 (SEQ ID NO: 28);"]},{"entry":[{},{},"HUMSMCK_T11 (SEQ ID NO: 30)"]},{"entry":["HUMSMCK_P8","308","HUMSMCK_T9 (SEQ ID NO: 29)"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"These sequences are variants of the known protein Creatine kinase, sarcomeric mitochondrial precursor (SEQ ID NO:388) (SwissProt accession identifier KCRS_HUMAN; known also according to the synonyms EC 2.7.3.2; S-MtCK; Mib-CK; Basic-type mitochondrial creatine kinase), referred to herein as the previously known protein.","Protein Creatine kinase, sarcomeric mitochondrial precursor (SEQ ID NO:388) is known or believed to have the following function(s): Reversibly catalyzes the transfer of phosphate between ATP and various phosphogens (e.g. creatine phosphate). Creatine kinase isoenzymes play a central role in energy transduction in tissues with large, fluctuating energy demands, such as skeletal muscle, heart, brain and spermatozoa. The sequence for protein Creatine kinase, sarcomeric mitochondrial precursor is given at the end of the application, as \u201cCreatine kinase, sarcomeric mitochondrial precursor amino acid sequence\u201d (SEQ ID NO:388). Known polymorphisms for this sequence are as shown in Table 153.",{"@attributes":{"id":"p-1353","num":"1405"},"tables":{"@attributes":{"id":"TABLE-US-00183","num":"00183"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 153"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations for Known Protein"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"154pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino",{}]},{"entry":["acid sequence","Comment"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["74","S -> A"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"Protein Creatine kinase, sarcomeric mitochondrial precursor (SEQ ID NO:388) localization is believed to be Mitochondrial inner membrane; outer side.","The following GO Annotation(s) apply to the previously known protein. The following annotation(s) were found: energy pathways; muscle contraction, which are annotation(s) related to Biological Process; creatine kinase; transferase, transferring phosphorus-containing groups, which are annotation(s) related to Molecular Function; and mitochondrion, which are annotation(s) related to Cellular Component.","The GO assignment relies on information from one or more of the SwissProt\/TremBl Protein knowledgebase, available from <dot expasy dot ch\/sprot\/>; or Locuslink, available from <dot ncbi dot nlm dot nih dot gov\/projects\/LocusLink\/>.","The heart-selective diagnostic marker prediction engine provided the following results with regard to cluster HUMSMCK. Predictions were made for selective expression of transcripts of this cluster in heart tissue, according to the previously described methods. The numbers on the y-axis of  refer to weighted expression of ESTs in each category, as \u201cparts per million\u201d (ratio of the expression of ESTs for a particular cluster to the expression of all ESTs in that category, according to parts per million).","Overall, the following results were obtained as shown with regard to the histogram in , concerning the number of heart-specific clones in libraries\/sequences; as well as with regard to the histogram in , concerning the actual expression of oligonucleotides in various tissues, including heart.","This cluster was found to be selectively expressed in heart for the following reasons: in a comparison of the ratio of expression of the cluster in heart specific ESTs to the overall expression of the cluster in non-heart ESTs, which was found to be 18.1; the ratio of expression of the cluster in heart specific ESTs to the overall expression of the cluster in muscle-specific ESTs which was found to be 2.4; and fisher exact test P-values were computed both for library and weighted clone counts to check that the counts are statistically significant, and were found to be 3.60E-23.","One particularly important measure of specificity of expression of a cluster in heart tissue is the previously described comparison of the ratio of expression of the cluster in heart as opposed to muscle. This cluster was found to be specifically expressed in heart as opposed to non-heart ESTs as described above. However, many proteins have been shown to be generally expressed at a higher level in both heart and muscle, which is less desirable. For this cluster, as described above, the ratio of expression of the cluster in heart specific ESTs to the overall expression of the cluster in muscle-specific ESTs which was found to be 18.1, which clearly supports specific expression in heart tissue.","As noted above, cluster HUMSMCK features 5 transcript(s), which were listed in Table 150 above. These transcript(s) encode for protein(s) which are variant(s) of protein Creatine kinase, sarcomeric mitochondrial precursor (SEQ ID NO:388). A description of each variant protein according to the present invention is now provided.","Variant protein HUMSMCK_P4 (SEQ ID NO:305) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) HUMSMCK_T5 (SEQ ID NO:26). An alignment is given to the known protein (Creatine kinase, sarcomeric mitochondrial precursor (SEQ ID NO:388)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between HUMSMCK_P4 (SEQ ID NO:305) and KCRS_HUMAN_V1 (SEQ ID NO:347):","1. An isolated chimeric polypeptide encoding for HUMSMCK_P4 (SEQ ID NO:305), comprising a first amino acid sequence being at least 90% homologous to MASIFSKLLTGRNASLLFATMGTSVLTTGYLLNRQKVCAEVREQPRLFPPSADYPDLRKHNNCMAECLTP AIYAKLRNKVTPNGYTLDQCIQTGVDNPGHPFIKTVGMVAGDEESYEVFADLFDPVIKLRHNGYDPRVMK HTTDLDASKITQGQFDEHYVLSSRVRTGRSIRGLSLPPACTRAERREVENVAITALEGLKGDLAGRYYKLS EMTEQDQQRLIDDHFLFDKPVSPLLTCAGMARDWPDARGIWHNYDKTFLIWINEEDHTRVISMEKGGNM KRVFERFCRGLKEVERLIQERGWEFMWNERLGYILTCPSNLGTGLRAGVHVRIPKLSKDPRFSKILENLRL QKRGTGGVDTAAVADVYDISNIDRIGRSEV corresponding to amino acids 1-381 of KCRS_HUMAN_V1 (SEQ ID NO:347), which also corresponds to amino acids 1-381 of HUMSMCK_P4 (SEQ ID NO:305), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence TSLSLS (SEQ ID NO:415) corresponding to amino acids 382-387 of HUMSMCK_P4 (SEQ ID NO:305), wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of HUMSMCK_P4 (SEQ ID NO:305), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence TSLSLS (SEQ ID NO:415) in HUMSMCK_P4 (SEQ ID NO:305).","It should be noted that the known protein sequence (KCRS_HUMAN; SEQ ID NO:388) has one or more changes than the sequence given at the end of the application and named as being the amino acid sequence for KCRS_HUMAN_V1 (SEQ ID NO:347). These changes were previously known to occur and are listed in the table below.",{"@attributes":{"id":"p-1367","num":"1419"},"tables":{"@attributes":{"id":"TABLE-US-00184","num":"00184"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 154"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Changes to KCRS_HUMAN_V1 (SEQ ID NO: 347)"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"140pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino",{}]},{"entry":["acid sequence","Type of change"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["75","conflict"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: intracellularly. The protein localization is believed to be intracellular because of manual inspection of known protein localization and\/or gene structure.","Variant protein HUMSMCK_P4 (SEQ ID NO:305) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 155, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein HUMSMCK_P4 (SEQ ID NO:305) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-1370","num":"1422"},"tables":{"@attributes":{"id":"TABLE-US-00185","num":"00185"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 155"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino","Alternative","Previously"]},{"entry":["acid sequence","amino acid(s)","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["59","K ->","No"]},{"entry":["60","H ->","No"]},{"entry":["74","A -> S","Yes"]},{"entry":["117","E -> *","No"]},{"entry":["117","E ->","No"]},{"entry":["249","R ->","No"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein HUMSMCK_P4 (SEQ ID NO:305) is encoded by the following transcript(s): HUMSMCK_T5 (SEQ ID NO:26), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript HUMSMCK_T5 (SEQ ID NO:26) is shown in bold; this coding portion starts at position 1305 and ends at position 2465. The transcript also has the following SNPs as listed in Table 156 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein HUMSMCK_P4 (SEQ ID NO:305) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-1372","num":"1424"},"tables":{"@attributes":{"id":"TABLE-US-00186","num":"00186"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 156"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on nucleotide","Alternative","Previously"]},{"entry":["sequence","nucleic acid","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["223","A -> C","Yes"]},{"entry":["545","G -> T","Yes"]},{"entry":["1481","G ->","No"]},{"entry":["1482","C ->","No"]},{"entry":["1524","G -> T","Yes"]},{"entry":["1653","G ->","No"]},{"entry":["1653","G -> T","No"]},{"entry":["2050","G ->","No"]},{"entry":["2228","T -> C","No"]},{"entry":["2231","G -> A","No"]},{"entry":["2489","C -> T","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein HUMSMCK_P5 (SEQ ID NO:306) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) HUMSMCK_T6 (SEQ ID NO:27). An alignment is given to the known protein (Creatine kinase, sarcomeric mitochondrial precursor (SEQ ID NO:388)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between HUMSMCK_P5 (SEQ ID NO:306) and KCRS_HUMAN_V1 (SEQ ID NO:347):","1. An isolated chimeric polypeptide encoding for HUMSMCK_P5 (SEQ ID NO:306), comprising a first amino acid sequence being at least 90% homologous to MASIFSKLLTGRNASLLFATMGTSVLTTGYLLNRQKVCAEVREQPRLFPPSADYPDLRKHNNCMAECLTP AIYAKLRNKVTPNGYTLDQCIQTGVDNPGHPFIKTVGMVAGDEESYEVFADLFDPVIKLRHNGYDPRVMK HTTDLDASKITQGQFDEHYVLSSRVRTGRSIRGLSLPPACTRAERREVENVAITALEGLKGDLAGRYYKLS EMTEQDQQRLIDDHFLFDKPVSPLLTCAGMARDWPDARGIWHNYDKTFLIWINEEDHTRVISMEKGGNM KRVFERFCRGLKEVERLIQERGWEFMWNERLGYILTCPSNLGTGLRAGVHVRIPKLSK corresponding to amino acids 1-338 of KCRS_HUMAN_V1 (SEQ ID NO:347), which also corresponds to amino acids 1-338 of HUMSMCK_P5 (SEQ ID NO:306), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence VLLCAQWP (SEQ ID NO:416) corresponding to amino acids 339-346 of HUMSMCK_P5 (SEQ ID NO:306), wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of HUMSMCK_P5 (SEQ ID NO:306), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VLLCAQWP (SEQ ID NO:416) in HUMSMCK_P5 (SEQ ID NO:306).","It should be noted that the known protein sequence (KCRS_HUMAN (SEQ ID NO:388)) has one or more changes than the sequence given at the end of the application and named as being the amino acid sequence for KCRS_HUMAN_V1 (SEQ ID NO:347). These changes were previously known to occur and are listed in the table below.",{"@attributes":{"id":"p-1378","num":"1430"},"tables":{"@attributes":{"id":"TABLE-US-00187","num":"00187"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 157"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Changes to KCRS_HUMAN_V1 (SEQ ID NO: 347)"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"140pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino",{}]},{"entry":["acid sequence","Type of change"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["75","Conflict"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: intracellularly. The protein localization is believed to be intracellular because of manual inspection of known protein localization and\/or gene structure.","Variant protein HUMSMCK_P5 (SEQ ID NO:306) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 158, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein HUMSMCK_P5 (SEQ ID NO:306) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-1381","num":"1433"},"tables":{"@attributes":{"id":"TABLE-US-00188","num":"00188"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 158"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino acid","Alternative","Previously"]},{"entry":["sequence","amino acid(s)","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["59","K ->","No"]},{"entry":["60","H ->","No"]},{"entry":["74","A -> S","Yes"]},{"entry":["117","E -> *","No"]},{"entry":["117","E ->","No"]},{"entry":["249","R ->","No"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein HUMSMCK_P5 (SEQ ID NO:306) is encoded by the following transcript(s): HUMSMCK_T6 (SEQ ID NO:27), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript HUMSMCK_T6 (SEQ ID NO:27) is shown in bold; this coding portion starts at position 1305 and ends at position 2342. The transcript also has the following SNPs as listed in Table 159 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein HUMSMCK_P5 (SEQ ID NO:306) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-1383","num":"1435"},"tables":{"@attributes":{"id":"TABLE-US-00189","num":"00189"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 159"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on nucleotide","Alternative","Previously"]},{"entry":["sequence","nucleic acid","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["223","A -> C","Yes"]},{"entry":["545","G -> T","Yes"]},{"entry":["1481","G ->","No"]},{"entry":["1482","C ->","No"]},{"entry":["1524","G -> T","Yes"]},{"entry":["1653","G ->","No"]},{"entry":["1653","G -> T","No"]},{"entry":["2050","G ->","No"]},{"entry":["2228","T -> C","No"]},{"entry":["2231","G -> A","No"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein HUMSMCK_P6 (SEQ ID NO:307) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) HUMSMCK_T7 (SEQ ID NO:28) and HUMSMCK_T11 (SEQ ID NO:30). An alignment is given to the known protein (Creatine kinase, sarcomeric mitochondrial precursor (SEQ ID NO:388)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between HUMSMCK_P6 (SEQ ID NO:307) and KCRS_HUMAN_V1 (SEQ ID NO:347):","1. An isolated chimeric polypeptide encoding for HUMSMCK_P6 (SEQ ID NO:307), comprising a first amino acid sequence being at least 90% homologous to MASIFSKLLTGRNASLLFATMGTSVLTTGYLLNRQKVCAEVREQPRLFPPSADYPDLRKHNNCMAECLTP AIYAKLRNKVTPNGYTLDQCIQTGVDNPGHPFIKTVGMVAGDEESYEVFADLFDPVIKLRHNGYDPRVMK HTTDLDASKITQGQFDEHYVLSSRVRTGRSIRGLSLPPACTRAERREVENVAITALEGLKGDLAGRYYKLS EMTEQDQQRLID corresponding to amino acids 1-223 of KCRS_HUMAN_V1 (SEQ ID NO:347), which also corresponds to amino acids 1-223 of HUMSMCK_P6 (SEQ ID NO:307), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence A corresponding to amino acids 224-224 of HUMSMCK_P6 (SEQ ID NO:307), wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","It should be noted that the known protein sequence (KCRS_HUMAN (SEQ ID NO:388)) has one or more changes than the sequence given at the end of the application and named as being the amino acid sequence for KCRS_HUMAN_V1 (SEQ ID NO:347). These changes were previously known to occur and are listed in the table below.",{"@attributes":{"id":"p-1388","num":"1440"},"tables":{"@attributes":{"id":"TABLE-US-00190","num":"00190"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 160"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Changes to KCRS_HUMAN_V1 (SEQ ID NO: 347)"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"140pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino",{}]},{"entry":["acid sequence","Type of change"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["75","Conflict"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: intracellularly. The protein localization is believed to be intracellular because of manual inspection of known protein localization and\/or gene structure.","Variant protein HUMSMCK_P6 (SEQ ID NO:307) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 161, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein HUMSMCK_P6 (SEQ ID NO:307) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-1391","num":"1443"},"tables":{"@attributes":{"id":"TABLE-US-00191","num":"00191"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 161"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino acid","Alternative","Previously"]},{"entry":["sequence","amino acid(s)","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["59","K ->","No"]},{"entry":["60","H ->","No"]},{"entry":["74","A -> S","Yes"]},{"entry":["117","E -> *","No"]},{"entry":["117","E ->","No"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein HUMSMCK_P6 (SEQ ID NO:307) is encoded by the following transcript(s): HUMSMCK_T7 (SEQ ID NO:28) and HUMSMCK_T11 (SEQ ID NO:30), for which the sequence(s) is\/are given at the end of the application.","The coding portion of transcript HUMSMCK_T7 (SEQ ID NO:28) is shown in bold; this coding portion starts at position 1305 and ends at position 1976. The transcript also has the following SNPs as listed in Table 162 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein HUMSMCK_P6 (SEQ ID NO:307) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-1394","num":"1446"},"tables":{"@attributes":{"id":"TABLE-US-00192","num":"00192"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 162"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on nucleotide","Alternative","Previously"]},{"entry":["sequence","nucleic acid","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["223","A -> C","Yes"]},{"entry":["545","G -> T","Yes"]},{"entry":["1481","G ->","No"]},{"entry":["1482","C ->","No"]},{"entry":["1524","G -> T","Yes"]},{"entry":["1653","G ->","No"]},{"entry":["1653","G -> T","No"]},{"entry":["2142","T -> C","No"]},{"entry":["2145","G -> A","No"]},{"entry":["2398","C -> A","Yes"]},{"entry":["2521","G -> A","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"The coding portion of transcript HUMSMCK_T11 (SEQ ID NO:30) is shown in bold; this coding portion starts at position 1305 and ends at position 1976. The transcript also has the following SNPs as listed in Table 163 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein HUMSMCK_P6 (SEQ ID NO:307) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-1396","num":"1448"},"tables":{"@attributes":{"id":"TABLE-US-00193","num":"00193"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 163"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on nucleotide","Alternative","Previously"]},{"entry":["sequence","nucleic acid","known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["223","A -> C","Yes"]},{"entry":["545","G -> T","Yes"]},{"entry":["1481","G ->","No"]},{"entry":["1482","C ->","No"]},{"entry":["1524","G -> T","Yes"]},{"entry":["1653","G ->","No"]},{"entry":["1653","G -> T","No"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein HUMSMCK_P8 (SEQ ID NO:308) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) HUMSMCK_T9 (SEQ ID NO:29). An alignment is given to the known protein (Creatine kinase, sarcomeric mitochondrial precursor (SEQ ID NO:388)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between HUMSMCK_P8 (SEQ ID NO:308) and KCRS_HUMAN_V1 (SEQ ID NO:347):","1. An isolated chimeric polypeptide encoding for HUMSMCK_P8 (SEQ ID NO:308), comprising a first amino acid sequence being at least 90% homologous to MASIFSKLLTGRNASLLFATMGTSVLTTGYLLNRQKVCAEVREQPRLFPPSADYPDLRKHNNCMAECLTP AIYAKLRNKVTPNGYTLDQCIQTGVDNPGHPFIKTVGMVAGDEESYEVFADLFDPVIKLRHNGYDPRVMK HTTDLDASKITQGQFDEHYVLSSRVRTGRSIRGLSLPPACTRAERREVENVAITALEGLKGDLAGRYYKLS EMTEQDQQRLIDDHFLFDKPVSPLLTCAGMARDWPDARGIWHNYDKTFLIWINEEDHTRVISMEKGGNM KRVFERFCRGLKEV corresponding to amino acids 1-294 of KCRS_HUMAN_V1 (SEQ ID NO:347), which also corresponds to amino acids 1-294 of HUMSMCK_P8 (SEQ ID NO:308), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence RCYLRFLDIY (SEQ ID NO:417) corresponding to amino acids 295-304 of HUMSMCK_P8 (SEQ ID NO:308), wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of HUMSMCK_P8 (SEQ ID NO:308), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence RCYLRFLDIY (SEQ ID NO:417) in HUMSMCK_P8 (SEQ ID NO:308).","It should be noted that the known protein sequence (KCRS_HUMAN (SEQ ID NO:388)) has one or more changes than the sequence given at the end of the application and named as being the amino acid sequence for KCRS_HUMAN_V1 (SEQ ID NO:347). These changes were previously known to occur and are listed in the table below.",{"@attributes":{"id":"p-1402","num":"1454"},"tables":{"@attributes":{"id":"TABLE-US-00194","num":"00194"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 164"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Changes to KCRS_HUMAN_V1 (SEQ ID NO: 347)"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"140pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino",{}]},{"entry":["acid sequence","Type of change"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["75","Conflict"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: intracellularly. The protein localization is believed to be intracellular because of manual inspection of known protein localization and\/or gene structure.","Variant protein HUMSMCK_P8 (SEQ ID NO:308) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 165, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein HUMSMCK_P8 (SEQ ID NO:308) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-1405","num":"1457"},"tables":{"@attributes":{"id":"TABLE-US-00195","num":"00195"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 165"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino acid","Alternative",{}]},{"entry":["sequence","amino acid(s)","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["59","K ->","No"]},{"entry":["60","H ->","No"]},{"entry":["74","A -> S","Yes"]},{"entry":["117","E -> *","No"]},{"entry":["117","E ->","No"]},{"entry":["249","R ->","No"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein HUMSMCK_P8 (SEQ ID NO:308) is encoded by the following transcript(s): HUMSMCK_T9 (SEQ ID NO:29), for which the sequence(s) is\/are given at the end of the application. The coding portion of transcript HUMSMCK_T9 (SEQ ID NO:29) is shown in bold; this coding portion starts at position 1305 and ends at position 2216. The transcript also has the following SNPs as listed in Table 166 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein HUMSMCK_P8 (SEQ ID NO:308) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-1407","num":"1459"},"tables":{"@attributes":{"id":"TABLE-US-00196","num":"00196"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 166"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on nucleotide","Alternative",{}]},{"entry":["sequence","nucleic acid","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["223","A -> C","Yes"]},{"entry":["545","G -> T","Yes"]},{"entry":["1481","G ->","No"]},{"entry":["1482","C ->","No"]},{"entry":["1524","G -> T","Yes"]},{"entry":["1653","G ->","No"]},{"entry":["1653","G -> T","No"]},{"entry":["2050","G ->","No"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"As noted above, cluster HUMSMCK features 14 segment(s), which were listed in Table 151 above and for which the sequence(s) are given at the end of the application. These segment(s) are portions of nucleic acid sequence(s) which are described herein separately because they are of particular interest. A description of each segment according to the present invention is now provided.","Segment cluster HUMSMCK_node0 (SEQ ID NO:150) according to the present invention is supported by 38 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMSMCK_T5 (SEQ ID NO:26), HUMSMCK_T6 (SEQ ID NO:27), HUMSMCK_T7 (SEQ ID NO:28), HUMSMCK_T9 (SEQ ID NO:29) and HUMSMCK_T11 (SEQ ID NO:30). Table 167 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1410","num":"1462"},"tables":{"@attributes":{"id":"TABLE-US-00197","num":"00197"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 167"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMSMCK_T5 (SEQ ID NO: 26)","1","1284"]},{"entry":["HUMSMCK_T6 (SEQ ID NO: 27)","1","1284"]},{"entry":["HUMSMCK_T7 (SEQ ID NO: 28)","1","1284"]},{"entry":["HUMSMCK_T9 (SEQ ID NO: 29)","1","1284"]},{"entry":["HUMSMCK_T11 (SEQ ID NO: 30)","1","1284"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMSMCK_node7 (SEQ ID NO:151) according to the present invention is supported by 47 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMSMCK_T5 (SEQ ID NO:26), HUMSMCK_T6 (SEQ ID NO:27), HUMSMCK_T7 (SEQ ID NO:28), HUMSMCK_T9 (SEQ ID NO:29) and HUMSMCK_T11 (SEQ ID NO:30). Table 168 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1412","num":"1464"},"tables":{"@attributes":{"id":"TABLE-US-00198","num":"00198"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 168"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMSMCK_T5 (SEQ ID NO: 26)","1285","1456"]},{"entry":["HUMSMCK_T6 (SEQ ID NO: 27)","1285","1456"]},{"entry":["HUMSMCK_T7 (SEQ ID NO: 28)","1285","1456"]},{"entry":["HUMSMCK_T9 (SEQ ID NO: 29)","1285","1456"]},{"entry":["HUMSMCK_T11 (SEQ ID NO: 30)","1285","1456"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMSMCK_node12 (SEQ ID NO:152) according to the present invention is supported by 54 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMSMCK_T5 (SEQ ID NO:26), HUMSMCK_T6 (SEQ ID NO:27), HUMSMCK_T7 (SEQ ID NO:28), HUMSMCK_T9 (SEQ ID NO:29) and HUMSMCK_T11 (SEQ ID NO:30). Table 169 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1414","num":"1466"},"tables":{"@attributes":{"id":"TABLE-US-00199","num":"00199"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 169"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMSMCK_T5 (SEQ ID NO: 26)","1476","1655"]},{"entry":["HUMSMCK_T6 (SEQ ID NO: 27)","1476","1655"]},{"entry":["HUMSMCK_T7 (SEQ ID NO: 28)","1476","1655"]},{"entry":["HUMSMCK_T9 (SEQ ID NO: 29)","1476","1655"]},{"entry":["HUMSMCK_T11 (SEQ ID NO: 30)","1476","1655"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMSMCK_node17 (SEQ ID NO:153) according to the present invention is supported by 48 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMSMCK_T5 (SEQ ID NO:26), HUMSMCK_T6 (SEQ ID NO:27), HUMSMCK_T7 (SEQ ID NO:28), HUMSMCK_T9 (SEQ ID NO:29) and HUMSMCK_T11 (SEQ ID NO:30). Table 170 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1416","num":"1468"},"tables":{"@attributes":{"id":"TABLE-US-00200","num":"00200"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 170"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMSMCK_T5 (SEQ ID NO: 26)","1752","1973"]},{"entry":["HUMSMCK_T6 (SEQ ID NO: 27)","1752","1973"]},{"entry":["HUMSMCK_T7 (SEQ ID NO: 28)","1752","1973"]},{"entry":["HUMSMCK_T9 (SEQ ID NO: 29)","1752","1973"]},{"entry":["HUMSMCK_T11 (SEQ ID NO: 30)","1752","1973"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMSMCK_node22 (SEQ ID NO:154) according to the present invention is supported by 60 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMSMCK_T5 (SEQ ID NO:26), HUMSMCK_T6 (SEQ ID NO:27), HUMSMCK_T7 (SEQ ID NO:28), HUMSMCK_T9 (SEQ ID NO:29) and HUMSMCK_T11 (SEQ ID NO:30). Table 171 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1418","num":"1470"},"tables":{"@attributes":{"id":"TABLE-US-00201","num":"00201"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 171"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMSMCK_T5 (SEQ ID NO: 26)","2060","2183"]},{"entry":["HUMSMCK_T6 (SEQ ID NO: 27)","2060","2183"]},{"entry":["HUMSMCK_T7 (SEQ ID NO: 28)","1974","2097"]},{"entry":["HUMSMCK_T9 (SEQ ID NO: 29)","2060","2183"]},{"entry":["HUMSMCK_T11 (SEQ ID NO: 30)","1974","2097"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMSMCK_node23 (SEQ ID NO:155) according to the present invention is supported by 3 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMSMCK_T9 (SEQ ID NO:29) and HUMSMCK_T11 (SEQ ID NO:30). Table 172 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1420","num":"1472"},"tables":{"@attributes":{"id":"TABLE-US-00202","num":"00202"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 172"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMSMCK_T9 (SEQ ID NO: 29)","2184","2382"]},{"entry":["HUMSMCK_T11 (SEQ ID NO: 30)","2098","2296"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMSMCK_node25 (SEQ ID NO:156) according to the present invention is supported by 58 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMSMCK_T5 (SEQ ID NO:26), HUMSMCK_T6 (SEQ ID NO:27) and HUMSMCK_T7 (SEQ ID NO:28). Table 173 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1422","num":"1474"},"tables":{"@attributes":{"id":"TABLE-US-00203","num":"00203"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 173"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMSMCK_T5 (SEQ ID NO: 26)","2184","2318"]},{"entry":["HUMSMCK_T6 (SEQ ID NO: 27)","2184","2318"]},{"entry":["HUMSMCK_T7 (SEQ ID NO: 28)","2098","2232"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMSMCK_node26 (SEQ ID NO:157) according to the present invention is supported by 1 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMSMCK_T6 (SEQ ID NO:27). Table 174 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1424","num":"1476"},"tables":{"@attributes":{"id":"TABLE-US-00204","num":"00204"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 174"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMSMCK_T6 (SEQ ID NO: 27)","2319","2448"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMSMCK_node28 (SEQ ID NO:158) according to the present invention is supported by 59 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMSMCK_T5 (SEQ ID NO:26) and HUMSMCK_T7 (SEQ ID NO:28). Table 175 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1426","num":"1478"},"tables":{"@attributes":{"id":"TABLE-US-00205","num":"00205"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 75"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMSMCK_T5 (SEQ ID NO: 26)","2319","2444"]},{"entry":["HUMSMCK_T7 (SEQ ID NO: 28)","2233","2358"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMSMCK_node29 (SEQ ID NO:159) according to the present invention is supported by 3 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMSMCK_T5 (SEQ ID NO:26). Table 176 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1428","num":"1480"},"tables":{"@attributes":{"id":"TABLE-US-00206","num":"00206"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 176"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMSMCK_T5 (SEQ ID NO: 26)","2445","2820"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMSMCK_node32 (SEQ ID NO:160) according to the present invention is supported by 62 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMSMCK_T7 (SEQ ID NO:28). Table 177 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1430","num":"1482"},"tables":{"@attributes":{"id":"TABLE-US-00207","num":"00207"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 177"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMSMCK_T7 (SEQ ID NO: 28)","2359","2632"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"According to an optional embodiment of the present invention, short segments related to the above cluster are also provided. These segments are up to about 120 bp in length, and so are included in a separate description.","Segment cluster HUMSMCK_node11 (SEQ ID NO:161) according to the present invention can be found in the following transcript(s): HUMSMCK_T5 (SEQ ID NO:26), HUMSMCK_T6 (SEQ ID NO:27), HUMSMCK_T7 (SEQ ID NO:28), HUMSMCK_T9 (SEQ ID NO:29) and HUMSMCK_T11 (SEQ ID NO:30). Table 178 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1433","num":"1485"},"tables":{"@attributes":{"id":"TABLE-US-00208","num":"00208"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 178"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMSMCK_T5 (SEQ ID NO: 26)","1457","1475"]},{"entry":["HUMSMCK_T6 (SEQ ID NO: 27)","1457","1475"]},{"entry":["HUMSMCK_T7 (SEQ ID NO: 28)","1457","1475"]},{"entry":["HUMSMCK_T9 (SEQ ID NO: 29)","1457","1475"]},{"entry":["HUMSMCK_T11 (SEQ ID NO: 30)","1457","1475"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMSMCK_node14 (SEQ ID NO:162) according to the present invention is supported by 38 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMSMCK_T5 (SEQ ID NO:26), HUMSMCK_T6 (SEQ ID NO:27), HUMSMCK_T7 (SEQ ID NO:28), HUMSMCK_T9 (SEQ ID NO:29) and HUMSMCK_T11 (SEQ ID NO:30). Table 179 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1435","num":"1487"},"tables":{"@attributes":{"id":"TABLE-US-00209","num":"00209"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 179"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMSMCK_T5 (SEQ ID NO: 26)","1656","1751"]},{"entry":["HUMSMCK_T6 (SEQ ID NO: 27)","1656","1751"]},{"entry":["HUMSMCK_T7 (SEQ ID NO: 28)","1656","1751"]},{"entry":["HUMSMCK_T9 (SEQ ID NO: 29)","1656","1751"]},{"entry":["HUMSMCK_T11 (SEQ ID NO: 30)","1656","1751"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster HUMSMCK_node19 (SEQ ID NO:163) according to the present invention is supported by 47 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcript(s): HUMSMCK_T5 (SEQ ID NO:26), HUMSMCK_T6 (SEQ ID NO:27) and HUMSMCK_T9 (SEQ ID NO:29). Table 180 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1437","num":"1489"},"tables":{"@attributes":{"id":"TABLE-US-00210","num":"00210"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 180"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["HUMSMCK_T5 (SEQ ID NO: 26)","1974","2059"]},{"entry":["HUMSMCK_T6 (SEQ ID NO: 27)","1974","2059"]},{"entry":["HUMSMCK_T9 (SEQ ID NO: 29)","1974","2059"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-1438","num":"1490"},"tables":{"@attributes":{"id":"TABLE-US-00211","num":"00211"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"294pt","align":"left"}},"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Variant protein alignment to the previously known protein:"},{"entry":"Sequence name: KCRS_HUMAN_V1 (SEQ ID NO: 347)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: HUMSMCK_P4 (SEQ ID NO: 305) \u00d7 KCRS_HUMAN_V1 (SEQ ID NO: 347)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","3745.00","Escore:","0"]},{"entry":["Matching length:","381","Total length:","381"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"294pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00039","num":"00039"},"img":{"@attributes":{"id":"EMI-C00039","he":"42.67mm","wi":"85.01mm","file":"US07714100-20100511-C00039.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00040","num":"00040"},"img":{"@attributes":{"id":"EMI-C00040","he":"42.67mm","wi":"85.09mm","file":"US07714100-20100511-C00040.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: KCRS_HUMAN_V1 (SEQ ID NO: 347)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: HUMSMCK_P5 (SEQ ID NO: 306) \u00d7 KCRS_HUMAN_V1 (SEQ ID NO: 347)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","3344.00","Escore:","0"]},{"entry":["Matching length:","338","Total length:","338"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"294pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00041","num":"00041"},"img":{"@attributes":{"id":"EMI-C00041","he":"75.35mm","wi":"85.01mm","file":"US07714100-20100511-C00041.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: KCRS_HUMAN_V1 (SEQ ID NO: 347)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: HUMSMCK_P6 (SEQ ID NO: 307) \u00d7 KCRS_HUMAN_V1 (SEQ ID NO: 347)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","2176.00","Escore:","0"]},{"entry":["Matching length:","223","Total length:","223"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"294pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00042","num":"00042"},"img":{"@attributes":{"id":"EMI-C00042","he":"53.51mm","wi":"84.92mm","file":"US07714100-20100511-C00042.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Sequence name: KCRS_HUMAN_V1 (SEQ ID NO: 347)"},{"entry":"Sequence documentation:"},{"entry":"Alignment of: HUMSMCK_P8 (SEQ ID NO: 308) \u00d7 KCRS_HUMAN_V1 (SEQ ID NO: 347)\u2003. ."},{"entry":"Alignment segment 1\/1:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Quality:","2904.00","Escore:","0"]},{"entry":["Matching length:","294","Total length:","294"]},{"entry":["Matching Percent Similarity:","100.00","Matching Percent Identity:","100.00"]},{"entry":["Total Percent Similarity:","100.00","Total Percent Identity:","100.00"]},{"entry":["Gaps:","0"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"294pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Alignment:"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00043","num":"00043"},"img":{"@attributes":{"id":"EMI-C00043","he":"64.43mm","wi":"85.01mm","file":"US07714100-20100511-C00043.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}]}}},"Cluster H88495 features 7 transcript(s) and 22 segment(s) of interest, the names for which are given in Tables 181 and 182, respectively, the sequences themselves are given at the end of the application. The selected protein variants are given in table 183.",{"@attributes":{"id":"p-1440","num":"1492"},"tables":{"@attributes":{"id":"TABLE-US-00212","num":"00212"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 181"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Transcripts of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Transcript Name","Seq ID No."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"H88495_PEA_3_T3","31"]},{"entry":[{},"H88495_PEA_3_T4","32"]},{"entry":[{},"H88495_PEA_3_T5","33"]},{"entry":[{},"H88495_PEA_3_T6","34"]},{"entry":[{},"H88495_PEA_3_T7","35"]},{"entry":[{},"H88495_PEA_3_T8","36"]},{"entry":[{},"H88495_PEA_3_T9","37"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-1441","num":"1493"},"tables":{"@attributes":{"id":"TABLE-US-00213","num":"00213"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 182"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segments of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"98pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment Name","Seq ID No."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"H88495_PEA_3_node_0","164"]},{"entry":[{},"H88495_PEA_3_node_1","165"]},{"entry":[{},"H88495_PEA_3_node_4","166"]},{"entry":[{},"H88495_PEA_3_node_9","167"]},{"entry":[{},"H88495_PEA_3_node_13","168"]},{"entry":[{},"H88495_PEA_3_node_19","169"]},{"entry":[{},"H88495_PEA_3_node_21","170"]},{"entry":[{},"H88495_PEA_3_node_26","171"]},{"entry":[{},"H88495_PEA_3_node_2","172"]},{"entry":[{},"H88495_PEA_3_node_5","173"]},{"entry":[{},"H88495_PEA_3_node_6","174"]},{"entry":[{},"H88495_PEA_3_node_7","175"]},{"entry":[{},"H88495_PEA_3_node_8","176"]},{"entry":[{},"H88495_PEA_3_node_10","177"]},{"entry":[{},"H88495_PEA_3_node_11","178"]},{"entry":[{},"H88495_PEA_3_node_12","179"]},{"entry":[{},"H88495_PEA_3_node_14","180"]},{"entry":[{},"H88495_PEA_3_node_16","181"]},{"entry":[{},"H88495_PEA_3_node_18","182"]},{"entry":[{},"H88495_PEA_3_node_20","183"]},{"entry":[{},"H88495_PEA_3_node_23","184"]},{"entry":[{},"H88495_PEA_3_node_24","185"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-1442","num":"1494"},"tables":{"@attributes":{"id":"TABLE-US-00214","num":"00214"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 183"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Proteins of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Protein Name","Seq ID No.","Corresponding Transcript(s)"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["H88495_PEA_3_P15","309","H88495_PEA_3_T3"]},{"entry":[{},{},"(SEQ ID NO: 31);"]},{"entry":[{},{},"H88495_PEA_3_T4"]},{"entry":[{},{},"(SEQ ID NO: 32);"]},{"entry":[{},{},"H88495_PEA_3_T7"]},{"entry":[{},{},"(SEQ ID NO: 35)"]},{"entry":["H88495_PEA_3_P16","310","H88495_PEA_3_T5"]},{"entry":[{},{},"(SEQ ID NO: 33);"]},{"entry":[{},{},"H88495_PEA_3_T6"]},{"entry":[{},{},"(SEQ ID NO: 34)"]},{"entry":["H88495_PEA_3_P17","311","H88495_PEA_3_T8"]},{"entry":[{},{},"(SEQ ID NO: 36)"]},{"entry":["H88495_PEA_3_P18","312","H88495_PEA_3_T9"]},{"entry":[{},{},"(SEQ ID NO: 37)"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"These sequences are variants of the known protein Sarcoplasmic reticulum histidine-rich calcium-binding protein precursor (SEQ ID NO:389) (SwissProt accession identifier SRCH_HUMAN), referred to herein as the previously known protein.","Protein Sarcoplasmic reticulum histidine-rich calcium-binding protein precursor (SEQ ID NO:389) is known or believed to have the following function(s): May play a role in the regulation of calcium sequestration or release in the SR of skeletal and cardiac muscle. The sequence for protein Sarcoplasmic reticulum histidine-rich calcium-binding protein precursor is given at the end of the application, as \u201cSarcoplasmic reticulum histidine-rich calcium-binding protein precursor amino acid sequence\u201d (SEQ ID NO:389). Known polymorphisms for this sequence are as shown in Table 184.",{"@attributes":{"id":"p-1445","num":"1497"},"tables":{"@attributes":{"id":"TABLE-US-00215","num":"00215"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 184"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations for Known Protein"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino",{}]},{"entry":["acid sequence","Comment"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["96","S -> A. \/FTId = VAR_005623."]},{"entry":["204","Missing. \/FTId = VAR_011622."]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"Protein Sarcoplasmic reticulum histidine-rich calcium-binding protein precursor (SEQ ID NO:389) localization is believed to be Sarcoplasmic reticulum lumen.","The following GO Annotation(s) apply to the previously known protein. The following annotation(s) were found: muscle contraction, which are annotation(s) related to Biological Process; and calcium binding, which are annotation(s) related to Molecular Function.","The GO assignment relies on information from one or more of the SwissProt\/TremBl Protein knowledgebase, available from <dot expasy dot ch\/sprot\/>; or Locuslink, available from <dot ncbi dot nlm dot nih dot gov\/projects\/LocusLink\/>.","The heart-selective diagnostic marker prediction engine provided the following results with regard to cluster H88495. Predictions were made for selective expression of transcripts of this cluster in heart tissue, according to the previously described methods. The numbers on the y-axis of  refer to weighted expression of ESTs in each category, as \u201cparts per million\u201d (ratio of the expression of ESTs for a particular cluster to the expression of all ESTs in that category, according to parts per million).","Overall, the following results were obtained as shown with regard to the histogram in , concerning the number of heart-specific clones in libraries\/sequences; as well as with regard to the histogram in , concerning the actual expression of oligonucleotides in various tissues, including heart.","This cluster was found to be selectively expressed in heart for the following reasons: in a comparison of the ratio of expression of the cluster in heart specific ESTs to the overall expression of the cluster in non-heart ESTs, which was found to be 13.7; the ratio of expression of the cluster in heart specific ESTs to the overall expression of the cluster in muscle-specific ESTs which was found to be 2.3; and fisher exact test P-values were computed both for library and weighted clone counts to check that the counts are statistically significant, and were found to be 1.90E-06.","One particularly important measure of specificity of expression of a cluster in heart tissue is the previously described comparison of the ratio of expression of the cluster in heart as opposed to muscle. This cluster was found to be specifically expressed in heart as opposed to non-heart ESTs as described above. However, many proteins have been shown to be generally expressed at a higher level in both heart and muscle, which is less desirable. For this cluster, as described above, the ratio of expression of the cluster in heart specific ESTs to the overall expression of the cluster in muscle-specific ESTs which was found to be 13.7, which clearly supports specific expression in heart tissue.","As noted above, cluster H88495 features 7 transcript(s), which were listed in Table 181 above. These transcript(s) encode for protein(s) which are variant(s) of protein Sarcoplasmic reticulum histidine-rich calcium-binding protein precursor (SEQ ID NO:389). A description of each variant protein according to the present invention is now provided.","Variant protein H88495_PEA3_P15 (SEQ ID NO:309) according to the present invention has an amino acid sequence as given at the end of the application; it is encoded by transcript(s) H88495_PEA3_T3 (SEQ ID NO:31), H88495_PEA3_T4 (SEQ ID NO:32) and H88495_PEA3_T7 (SEQ ID NO:35). An alignment is given to the known protein (Sarcoplasmic reticulum histidine-rich calcium-binding protein precursor (SEQ ID NO:389)) at the end of the application. One or more alignments to one or more previously published protein sequences are given at the end of the application. A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between H88495_PEA3_P15 (SEQ ID NO:309) and SRCH_HUMAN_V1 (SEQ ID NO:346):","1. An isolated chimeric polypeptide encoding for H88495_PEA3_P15 (SEQ ID NO:309), comprising a first amino acid sequence being at least 90% homologous to MGHHRPWLHASVLWAGVASLLLPPAMTQQLRGDGLGFRNRNN corresponding to amino acids 1-42 of SRCH_HUMAN_V1 (SEQ ID NO:346), which also corresponds to amino acids 1-42 of H88495_PEA3_P15 (SEQ ID NO:309), a bridging amino acid N corresponding to amino acid 43 of H88495_PEA3_P15 (SEQ ID NO:309), a second amino acid sequence being at least 90% homologous to TGVAGLSEEASAELRHHLHSPRDHPDENKDVSTENGHHFWSHPDREKEDEDVAKEYGHLLPGHRSQDHK VGDEGVSGEEVFAEHGGQARGHRGHGSEDTEDSAEHRHHLPSHRSHSHQDEDEDEVVSSEHHHHILRHG HRGHDGEDDEGEEEEEEEEEEEEASTEYGHQAHRHRGHGSEEDEDVSDGHHHHGPSHRHQGHEEDDDDD DDDDDDDDDDDVSIEYRHQAHRHQGHGIEEDEDVSDGHHHRDPSHRHRSHEEDDNDDDDVSTEYGHQA HRHQDHRKEEVEAVSGEHHHHVPDDHRHQGHRDEEEDEDVSTERWHQGPQQHVHHGLVDEEEEEEEITVQF GHYVASHQPRGHKSDEEDFQDEYKTEVPHHHHHRVPREEDEEVSAELGHQAPSHRQSHQDEETGHGQRG SIKEMSHHPPGHTVVKDRSHLRKDDSEEEKEKEEDPGSHEEDDESSEQGEKGTHHGSRDQEDEEDEEEGH GLSLNQEEEEEEDKEEEEEEEDEERREERAEVGAPLSPDHSEEEEEEEEGLEEDEPRFTIIPNPLDRREEAGG ASSEEESGEDTGPQDAQEYGNYQPGSLCGYCSFCNRCTECESCHCDEENMGEHCDQCQ corresponding to amino acids 44-657 of SRCH_HUMAN_V1 (SEQ ID NO:346), which also corresponds to amino acids 44-657 of H88495_PEA3_P15 (SEQ ID NO:309), and a third amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95% homologous to a polypeptide having the sequence VRPHLTLKAPLGLRMHRDPLRTPSPKSWPLTQPLTPDATLTPQAILTPTLT (SEQ ID NO:418) corresponding to amino acids 658-708 of H88495_PEA3_P15 (SEQ ID NO:309), wherein said first amino acid sequence, bridging amino acid, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","2. An isolated polypeptide encoding for a tail of H88495_PEA3_P15 (SEQ ID NO:309), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95% homologous to the sequence VRPHLTLKAPLGLRMHRDPLRTPSPKSWPLTQPLTPDATLTPQAILTPTLT (SEQ ID NO:418) in H88495_PEA3_P15 (SEQ ID NO:309).","It should be noted that the known protein sequence (SRCH_HUMAN; SEQ ID NO:389) has one or more changes than the sequence given at the end of the application and named as being the amino acid sequence for SRCH_HUMAN_V1 (SEQ ID NO:346). These changes were previously known to occur and are listed in the table below.",{"@attributes":{"id":"p-1459","num":"1511"},"tables":{"@attributes":{"id":"TABLE-US-00216","num":"00216"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 185"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Changes to SRCH_HUMAN_V1 (SEQ ID NO: 346)"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"105pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"SNP position(s) on amino",{}]},{"entry":[{},"acid sequence","Type of change"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"97","Variant"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The location of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: secreted. The protein localization is believed to be secreted because both signal-peptide prediction programs predict that this protein has a signal peptide, and neither trans-membrane region prediction program predicts that this protein has a trans-membrane region.","Variant protein H88495_PEA3_P15 (SEQ ID NO:309) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 186, (given according to their position(s) on the amino acid sequence, with the alternative amino acid(s) listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein H88495_PEA3_P15 (SEQ ID NO:309) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-1462","num":"1514"},"tables":{"@attributes":{"id":"TABLE-US-00217","num":"00217"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 186"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position(s) on amino acid","Alternative",{}]},{"entry":["sequence","amino acid(s)","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["6","P -> L","No"]},{"entry":["6","P -> S","No"]},{"entry":["43","N -> S","Yes"]},{"entry":["96","A -> S","Yes"]},{"entry":["364","Q ->","No"]},{"entry":["580","D -> H","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein H88495_PEA3_P15 (SEQ ID NO:309) is encoded by the following transcript(s): H88495_PEA3_T3 (SEQ ID NO:31), H88495_PEA3_T4 (SEQ ID NO:32) and H88495_PEA3_T7 (SEQ ID NO:35), for which the sequence(s) is\/are given at the end of the application.","The coding portion of transcript H88495_PEA3_T3 (SEQ ID NO:31) is shown in bold; this coding portion starts at position 743 and ends at position 2866. The transcript also has the following SNPs as listed in Table 187 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein H88495_PEA3_P15 (SEQ ID NO:309) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-1465","num":"1517"},"tables":{"@attributes":{"id":"TABLE-US-00218","num":"00218"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 187"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP position on nucleotide","Alternative",{}]},{"entry":["sequence","nucleic acid","Previously known SNP?"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["223","A -> G","Yes"]},{"entry":["285","C -> T","Yes"]},{"entry":["362","A -> C","Yes"]},{"entry":["373","G -> C","Yes"]},{"entry":["628","A -> T","No"]},{"entry":["629","G -> T","No"]},{"entry":["758","C -> T","No"]},{"entry":["759","C -> T","No"]},{"entry":["847","G -> A","Yes"]},{"entry":["870","A -> G","Yes"]},{"entry":["958","G -> A","No"]},{"entry":["1028","G -> T","Yes"]},{"entry":["1321","A -> G","Yes"]},{"entry":["1834","G ->","No"]},{"entry":["1903","C -> T","Yes"]},{"entry":["2480","G -> C","Yes"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"The coding portion of transcript H88495_PEA3_T4 (SEQ ID NO:32) is shown in bold; this coding portion starts at position 743 and ends at position 2866. The transcript also has the following SNPs as listed in Table 188 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed; the last column indicates whether the SNP is known or not; the presence of known SNPs in variant protein H88495_PEA3_P15 (SEQ ID NO:309) sequence provides support for the deduced sequence of this variant protein according to the present invention).",{"@attributes":{"id":"p-1467","num":"1519"},"tables":{"@attributes":{"id":"TABLE-US-00219","num":"00219"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"
