---
title: Gene expression and pain
abstract: The present invention relates to double-stranded oligonucleotides, pharmaceutical compositions thereof, and use of such double-stranded oligonucleotides and pharmaceutical compositions to modulate nociceptive signaling in a cell or prevent and/or treat pain in a patient.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=07943591&OS=07943591&RS=07943591
owner: Adynxx, Inc.
number: 07943591
owner_city: San Francisco
owner_country: US
publication_date: 20080512
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"p":["The present application claims priority to U.S. Provisional Application No. 60\/917,583, filed May 11, 2007, which is incorporated by reference herein in its entirety.","The contents of the text file submitted electronically herewith are incorporated herein by reference in their entirety: A computer readable format copy of the Sequence Listing of the Sequence Listing (filename: ADDY 001 01US SeqList_ST25.txt, date recorded: Jul. 15, 2008, file size 44 kilobytes).","The present invention relates to double-stranded nucleic acids, termed oligonucleotide decoys, pharmaceutical compositions thereof, and the use of such oligonucleotide decoys and pharmaceutical compositions to modulate nociceptive signaling and to prevent and\/or treat pain.","Pain may be defined as an unpleasant sensory and emotional experience associated with actual or potential tissue damage, or described in terms of such damage. Chronic pain afflicts 40% of the U.S. population and is associated with numerous deleterious medical conditions. Persistent and highly debilitating, chronic pain is generally accompanied by weakness, sleeplessness, a lack of appetite, irritability and depression. Over time, the quality of life is profoundly affected and patients are often incapable of accomplishing the simple tasks of everyday life.","Currently used pain treatments apply a three-step pain ladder which recommends the administration of drugs as follows: non-opioids (e.g., aspirin, acetaminophen, etc.), then, as necessary, mild opioids (e.g., codeine) and finally strong opioids (e.g., morphine). Despite this arsenal of drugs, over 50% of patients with chronic pain are not effectively treated.","The ineffectiveness of current pain treatments is, inter alia, due to significant toxicity issues with existing drug therapies. Mild to severe toxicity is induced by all classes of pain drugs: non steroidal inflammatory drugs cause gastro-intestinal damage, coxibs are associated with heart failure, and opioids are responsible for numerous side effects including respiratory depression, sedation, digestive malfunctions and addiction.","Transcription factors are important factors in multiple signaling pathways and frequently control the concurrent expression of numerous genes. Many transcription factors are involved in the regulation of the expression of genes that are involved in pain including, but not limited to, POU factors, upstream stimulatory factors (USF), EGR1, cAMP-response element binding protein\/activating transcription factors (CREB\/ATF), activating protein 1 (AP1), serum response factor (SRF), promoter selective transcription factor (SP1) and the runt related transcription factor 1 (RUNX1).","Thus, there may be significant therapeutic potential in inhibiting transcription factors in order to monitor the expression of genes involved in pain. Accordingly, what is needed are selective, readily available non-toxic transcription factor inhibitors.","The present invention satisfies these and other needs by providing oligonucleotide decoys, e.g. double-stranded oligonucleotides, pharmaceutical compositions thereof, and use of such oligonucleotide decoys and pharmaceutical compositions to modulate nociceptive signaling and to prevent and\/or treat pain. Generally, the oligonucleotide decoys are transcription factor inhibitors.","In one aspect, oligonucleotide decoys comprising one or more transcription factor binding sites are provided. In certain embodiments, each transcription factor binding site binds to a transcription factor selected from the group consisting of POU1F1, POU2F, POU3F, POU4F1, POU5F1, USF, EGR1, CREB\/ATF, AP1, CEBP, SRF, ETS1, MEF2, SP1, RUNX, NFAT, ELK1, ternary complex factors, STAT, GATA1, ELF1, nuclear factor\u2014granulocyte\/macrophage a, HNF1, ZFHX3, IRF, TEAD1, TBP, NFY, caccc-box binding factors, KLF4, KLF7, IKZF, MAF, REST, HSF, KCNIP3 and PPAR transcription factors. In certain embodiments, the transcription factor that binds to a transcription factor binding site is a human transcription factor. In other embodiments, the transcription factor that binds to a transcription factor binding site is a non-human transcription factor (e.g., an avian, mammal (e.g., mouse, rat, dog, cat, horse, cow, etc.), or primate transcription factor).","In a related aspect, oligonucleotide decoys comprising two or more transcription factor binding sites are provided. In certain embodiments, each transcription factor binding site binds to a transcription factor selected from the group consisting of POU1F1, POU2F, POU3F, POU5F1, USF, EGR1, CREB\/ATF, AP1, CEBP, SRF, ETS1, MEF2, SP1, RUNX, NFAT, ELK1, ternary complex factors, STAT, GATA1, ELF1, nuclear factor\u2014granulocyte\/macrophage a, POU4F1, HNF1, ZFHX3, IRF, TEAD1, TBP, NFY, caccc-box binding factors, KLF4, KLF7, IKZF, MAF, REST, HSF, KCNIP3 and PPAR transcription factors. In certain embodiments, the relative position of the two transcription factor binding sites within the decoy modulates (e.g., increases) the binding affinity between a transcription factor and its transcription factor binding site, as compared to the binding affinity between the transcription factor and a decoy having a single transcription factor binding site. In certain embodiments, the relative position of the two transcription factor binding sites within the decoy promotes dimerization of transcription factors bound to the sites.","In certain embodiments, the oligonucleotide decoys comprise: (a) a sequence selected from the group consisting of SEQ ID NOs.: 1-40, 42, 45 and 47-53; or (b) a sequence having at least 50% identity with a sequence selected from the group consisting of SEQ ID NOs.: 1-40, 42, 45 and 47-53.","In certain embodiments, the oligonucleotide decoys can be provided as salts, hydrates, solvates or N-oxides derivatives.","In another aspect, pharmaceutical compositions comprising oligonucleotide decoys are provided. The pharmaceutical compositions generally comprise one or more oligonucleotide decoys and a pharmaceutically acceptable vehicle.","In another aspect, methods for treating or preventing pain are provided. The methods generally involve administering to a patient in need of such treatment or prevention a therapeutically effective amount of an oligonucleotide decoy of the invention, or a pharmaceutical composition thereof.","In another aspect, methods for modulating the transcription of a gene in a cell involved in nociceptive signaling, such as a dorsal root ganglion and\/or spinal cord neuron, are provided. The methods generally comprise administering to the cell an effective amount of an oligonucleotide decoy.","In another aspect, methods for modulating nociceptive signaling in a cell involved in nociceptive signaling, such as a dorsal root ganglion and\/or spinal cord neuron, are provided. The methods generally comprise administering to the cell an effective amount of an oligonucleotide decoy.","In yet another aspect, methods for monitoring the proteolytic degradation of proteins involved in nociceptive signaling in a cell are provided. The methods generally comprise administering to the cell an effective amount of an oligonucleotide decoy.","Definitions","\u201cBinding,\u201d as used in the context of transcription factors binding to oligonucleotide decoys, refers to a direct interaction (e.g. non-covalent bonding between the transcription factor and oligonucleotide decoy, including hydrogen-bonding, van der Waals bonding, etc.) between a transcription factor and an oligonucleotide decoy. Accordingly, an oligonucleotide that does not bind to a transcription factor does not directly interact with said transcription factor.","\u201cChronic\u201d refers to a period of time comprising months (e.g., at least two months) or years.","\u201cCompounds\u201d refers to double-stranded oligonucleotides, also referred to herein as oligonucleotide decoys. The compounds described herein may contain one or more chiral centers and\/or double bonds and therefore, may exist as stereoisomers, such as double-bond isomers (i.e., geometric isomers), enantiomers or diastereomers. Accordingly, the chemical structures depicted herein encompass all possible enantiomers and stereoisomers of the illustrated compounds including the stereoisomerically pure form (e.g. geometrically pure, enantiomerically pure or diastereomerically pure) and enantiomeric and stereoisomeric mixtures. Enantiomeric and stereoisomeric mixtures can be resolved into their component enantiomers or stereoisomers using separation techniques or chiral synthesis techniques well known to the skilled artisan. Compounds may also exist in several tautomeric forms including the enol form, the keto form and mixtures thereof. Accordingly, the chemical structures depicted herein encompass all possible tautomeric forms of compounds. Compounds described herein also include isotopically labeled compounds where one or more atoms have an atomic mass different from the atomic mass conventionally found in nature. Examples of isotopes that may be incorporated into the compounds of the invention include, but are not limited to, H, H, C, C, C, N, O, O, etc. Compounds may exist in unsolvated forms as well as solvated forms, including hydrated forms and as N-oxides. In general, compounds may be hydrated, solvated or N-oxides. Certain compounds may exist in multiple crystalline or amorphous forms. All physical forms are equivalent for the uses contemplated herein. Further, it should be understood, when partial structures of the compounds are illustrated, that brackets indicate the point of attachment of the partial structure to the rest of the molecule.","\u201cModulation of gene expression level\u201d refers to any change in gene expression level, including an induction or activation (e.g., an increase in gene expression), an inhibition or suppression (e.g., a decrease in gene expression), or a stabilization (e.g., prevention of the up-regulation or down-regulation of a gene that ordinarily occurs in response to a stimulus, such as a pain-inducing stimulus).","\u201cNociceptive signaling\u201d refers to molecular and cellular mechanisms involved in the detection of a noxious stimulus or of a potentially harmful stimulus, which leads to the perception of pain, including neurotransmitter synthesis and release, neurotransmitter-induced signaling, membrane depolarization, and related intra-cellular and inter-cellular signaling events.","\u201cOligonucleotide\u201d refers to any double-stranded, nucleic acid-containing polymer generally less than approximately 200 nucleotides (or 100 base pairs) and including, but not limited to, DNA, RNA and RNA-DNA hybrids. The term encompasses sequences that include any of the known base analogs of DNA and RNA including, but not limited to, 2,6-diaminopurine, 5-carboxymethylaminomethyl-2-thiouracil, 5-carboxymethylaminomethyluracil, dihydrouracil, inosine, uracil-5-oxyacetic acid, N6-isopentenyladenine, 1-methyladenine, N-uracil-5-oxyacetic acid methylester, queosine, 2-thiocytosine, 5-bromouracil, methylphosphonate, phosphorodithioate, ormacetal, 3\u2032-thioformacetal, nitroxide backbone, sulfone, sulfamate, morpholino derivatives, locked nucleic acid (LNA) derivatives, and\/or peptide nucleic acid (PNA) derivatives. In some embodiments, the oligonucleotide is composed of two complementary single-stranded oligonucleotides that are annealed together. In other embodiments, the oligonucleotide is composed of one single-stranded oligonucleotide that forms intramolecular base pairs to create a substantially double-stranded structure.","\u201cPain\u201d refers to an unpleasant sensory and emotional experience that is associated with actual or potential tissue damage or described in such terms. All of the different manifestations and qualities of pain, including mechanical pain (e.g., induced by a mechanical stimulus or by body motion), temperature-induced pain (e.g., pain induced by hot, warm and\/or cold temperatures), and chemically-induced pain (e.g., pain induced by a chemical). In certain embodiments, pain is chronic, sub-chronic, acute, or sub-acute. In certain embodiments, pain features hyperalgesia (i.e., an increased sensitivity to a painful stimulus) and\/or allodynia (i.e., a painful response to a usually non-painful stimulus). In certain embodiments, pain is pre-existing in a patient. In other embodiments, pain is iatrogenic, induced in a patient (e.g. post-operative pain).","\u201cPharmaceutically acceptable salt\u201d refers to a salt of a compound, which possesses the desired pharmacological activity of the parent compound. Such salts include, but are not limited to: (1) acid addition salts, formed with inorganic acids such as hydrochloric acid, hydrobromic acid, sulfuric acid, nitric acid, phosphoric acid, and the like; or formed with organic acids such as acetic acid, propionic acid, hexanoic acid, cyclopentanepropionic acid, glycolic acid, pyruvic acid, lactic acid, malonic acid, succinic acid, malic acid, maleic acid, fumaric acid, tartaric acid, citric acid, benzoic acid, 3-(4-hydroxybenzoyl) benzoic acid, cinnamic acid, mandelic acid, methanesulfonic acid, ethanesulfonic acid, 1,2-ethane-disulfonic acid, 2-hydroxyethanesulfonic acid, benzenesulfonic acid, 4-chlorobenzenesulfonic acid, 2-naphthalenesulfonic acid, 4-toluenesulfonic acid, camphorsulfonic acid, 4-methylbicyclo[2.2.2]-oct-2-ene-1-carboxylic acid, glucoheptonic acid, 3-phenylpropionic acid, trimethylacetic acid, tertiary butylacetic acid, lauryl sulfuric acid, gluconic acid, glutamic acid, hydroxynaphthoic acid, salicylic acid, stearic acid, muconic acid, and the like; or (2) salts formed when an acidic proton present in the parent compound is replaced by a metal ion, e.g., an alkali metal ion, an alkaline earth ion, or an aluminum ion; or coordinates with an organic base such as ethanolamine, diethanolamine, triethanolamine, N-methylglucamine and the like.","\u201cPharmaceutically acceptable vehicle\u201d refers to a diluent, adjuvant, excipient or carrier with which a compound of the invention is administered.","\u201cPatient\u201d includes any animal, including birds, mammals, primates, and humans.","\u201cPreventing\u201d or \u201cprevention\u201d refers to (1) a reduction in the risk of acquiring a disease or disorder (e.g. causing at least one of the clinical symptoms of a disease not to develop in a patient that may be exposed to or predisposed to the disease but does not yet experience or display symptoms of the disease), or (2) a reduction in the likely severity of a symptom associated with a disease or disorder (e.g., reducing the likely severity of at least one of the clinical symptoms of a disease in a patient that may be exposed to or predisposed to the disease but does not yet experience or display symptoms of the disease).","\u201cSub-acute\u201d refers to a period of time comprising hours (e.g., 1 h-24 h)","\u201cSub-chronic\u201d refers to a period of time comprising days or months (e.g. less than two months).","\u201cTreating\u201d or \u201ctreatment\u201d of any disease or disorder refers, in some embodiments, to ameliorating the disease or disorder (i.e., arresting or reducing the development of the disease or at least one of the clinical symptoms thereof). In other embodiments \u201ctreating\u201d or \u201ctreatment\u201d refers to ameliorating at least one physical parameter, which may not be discernible by the patient. In yet other embodiments, \u201ctreating\u201d or \u201ctreatment\u201d refers to inhibiting the disease or disorder, either physically, (e.g., stabilization of a discernible symptom), physiologically, (e.g., stabilization of a physical parameter) or both. In yet other embodiments, \u201ctreating\u201d or \u201ctreatment\u201d refers to delaying the onset of the disease or disorder.","\u201cTherapeutically effective amount\u201d means the amount of a compound that, when administered to a patient, is sufficient to effect such treatment of a particular disease or condition. The \u201ctherapeutically effective amount\u201d will vary depending on the compound, the disease, the severity of the disease, and the age, weight, etc., of the patient to be treated.","Reference will now be made in detail to preferred embodiments of the invention. While the invention will be described in conjunction with the preferred embodiments, it will be understood that it is not intended to limit the invention to those preferred embodiments. To the contrary, it is intended to cover alternatives, modifications, and equivalents as may be included within the spirit and scope of the invention as defined by the appended claims.","Oligonucleotide Decoys","The present invention relates to oligonucleotide decoys, pharmaceutical compositions thereof, and use of such oligonucleotide decoys and pharmaceutical compositions to modulate nociceptive signaling and to prevent and\/or treat pain.","In certain embodiments, the invention features oligonucleotide decoys comprising one or more (e.g., 1, 2, 3, 4, 5, etc.) transcription factor binding sites. In related embodiments, each transcription factor binding site binds to a transcription factor selected from the group consisting of POU1F1, POU2F, POU3F, POU4F1, POU5F1, USF, EGR1, CREB\/ATF, AP1, CEBP, SRF, ETS1, MEF2, SP1, RUNX, NFAT, ELK1, ternary complex factors, STAT, GATA1, ELF1, nuclear factor\u2014granulocyte\/macrophage a, HNF1, ZFHX3, IRF, TEAD1, TBP, NFY, caccc-box binding factors, KLF4, KLF7, IKZF, MAF, REST, HSF, KCNIP3 and PPAR transcription factors. In certain embodiments, transcription factor binding sites bind to two or more members of a family of closely-related transcription factors. Representative members of such transcription factor families can be selected from the group consisting of POU1F1, POU2F, POU3F, POU4F1, POU5F1, USF, EGR1, CREB\/ATF, AP1, CEBP, SRF, ETS1, MEF2, SP1, RUNX, NFAT, ELK1, ternary complex factors, STAT, GATA1, ELF1, nuclear factor-granulocyte\/macrophage a, HNF1, ZFHX3, IRF, TEAD1, TBP, NFY, caccc-box binding factors, KLF4, KLF7, IKZF, MAF, REST, HSF, KCNIP3 and PPAR transcription factors. Thus, in certain embodiments, an oligonucleotide decoy that binds to, e.g. EGR1, can also bind to one or more additional family members, e.g. EGR2, EGR3, EGR4.","In certain embodiments, the oligonucleotide decoys comprise two or more (e.g., 2, 3, 4, 5, etc.) transcription factor binding sites. In related embodiments, each transcription factor binding site binds to a transcription factor selected from the group consisting of POU1F1, POU2F, POU3F, POU4F1, POU5F1, USF, EGR1, CREB\/ATF, AP1, CEBP, SRF, ETS1, MEF2, SP1, RUNX, NFAT, ELK1, ternary complex factors, STAT, GATA1, ELF1, nuclear factor\u2014granulocyte\/macrophage a, HNF1, ZFHX3, IRF, TEAD1, TBP, NFY, caccc-box binding factors, KLF4, KLF7, IKZF, MAF, REST, HSF, KCNIP3 and PPAR transcription factors. In certain embodiments, the relative position of the two or more transcription factor binding sites within the decoy modulates (e.g., increases or decreases) the binding affinity between a target transcription factor (i.e., the transcription factor that a particular binding site is designed to bind to) and its transcription factor binding site, e.g. as compared to the binding affinity between the transcription factor and a decoy having a single transcription factor binding site (e.g., a consensus binding site) specific to the transcription factor. Thus, the relative position of the two transcription factor binding sites within an oligonucleotide decoy of the invention can increase the affinity of the oligonucleotide decoy for a target transcription factor (e.g., for one or more of the transcription factors targeted by the decoy). In certain embodiments, the increase in affinity of the oligonucleotide decoy for a target transcription factor is 1.2 fold or greater (e.g. about 1.2, 1.3, 1.4, 1.5, 1.6, 1.7, 1.8, 1.9, 2.0, 2.1, 2.2, 2.3, 2.4, 2.5, 2.6, 2.7, 2.8, 2.9, 3.0 fold, or more). In certain embodiments, the relative position of the two transcription factor binding sites within an oligonucleotide decoy promotes protein-protein interactions between transcription factors bound to the sites, e.g. homodimerization or heterodimerization of the transcription factors. In certain embodiments, such protein-protein interactions between transcription factors stablize their interactions, e.g., binding, to the oligonucleotide decoy, thereby increasing the binding affinity of the oligonucleotide decoy for one or more of the target transcription factors. In certain embodiments, a transcription factor that binds to a transcription factor binding site present in an oligonucleotide decoy is a human transcription factor. In other embodiments, the transcription factor that binds to a transcription factor binding site in an oligonucleotide decoy is a non-human, e.g., an avian, mammal (e.g., mouse, rat, dog, cat, horse, cow, etc.), or primate, transcription factor.","In certain embodiments, the transcription factor binding sites of an oligonucleotide decoy each bind to the same transcription factor, e.g. EGR1. In other embodiments, the transcription factor binding sites of an oligonucleotide decoy bind to different transcription factors, e.g. different members of a closely related family of transcription factors (e.g., different members of the EGR1 family) or a combination of transcription factors selected from the group consisting of POU1F1, POU2F, POU3F, POU4F1, POU5F1, USF, EGR1, CREB\/ATF, AP1, CEBP, SRF, ETS1, MEF2, SP1, RUNX, NFAT, ELK1, ternary complex factors, STAT, GATA1, ELF1, nuclear factor\u2014granulocyte\/macrophage a, HNF1, ZFHX3, IRF, TEAD1, TBP, NFY, caccc-box binding factors, KLF4, KLF7, IKZF, MAF, REST, HSF, KCNIP3 and PPAR transcription factors.","In certain embodiments, the transcription factor binding sites of an oligonucleotide decoy are separated from each other by a linker sequence. Linker sequences can be 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, or more base pairs in length. Typically, linker sequences will be two to five base pairs in length. In other embodiments, the transcription factor binding sites can be immediately adjacent to one another (e.g., no linker sequence is present) or overlapping. In cases where the transcription factor binding sites are overlapping, the transcription factor binding sites may share 1, 2, 3, 4, 5, or more base pairs. Alternatively, one or both of the transcription factor binding sites may be lacking base pairs that otherwise form part of a consensus binding sequence for the transcription factor(s) that bind to the site. In general, however, base pairs that are critical to the binding interaction between a transcription factor binding site and the transcription factors that bind to the site (e.g., base pairs that are essentially invariant in a consensus binding sequence for a particular transcription factor) are not shared or missing when transcription binding sequences are overlapping.","In certain embodiments, oligonucleotide decoys comprise flanking sequences located at each end of the decoy sequence. Flanking sequences can be 1, 2, 3, 4, 5, 6, or more base pairs in length. In general, flanking sequences are two to five base pairs in length. In preferred embodiments, 5\u2032 flanking sequences starts with a G\/C base pair and 3\u2032 flanking sequences terminate in a G\/C base pair. In preferred embodiments, flanking sequences do not form part of a transcription factor binding site and\/or do not interact with or bind to transcription factors. In other embodiments, flanking sequences form weak interactions with transcription factors bound to an adjacent transcription factor binding site.","In certain embodiments, oligonucleotide decoys are generally at least 10, 11, 12, 13, 14, 15, or more base pairs in length. In related embodiments, oligonucleotide decoys are generally less than 65, 60, 55, 50, or 45 base pairs in length. In preferred embodiments, oligonucleotide decoys are about 20 to 40 base pairs in length. In other embodiments, oligonucleotide decoys are about 20 to 35, 25 to 40, or 25 to 35 base pairs in length.","In certain embodiments, the oligonucleotide decoys comprise: (a) a sequence selected from the group consisting of SEQ ID NOs.: 1-40, 42, 45 and 47-53; or (b) a sequence having at least 50%, 55%, 60%, 65%, 70%, 75%, 80%, 85%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, or 99% identity with a sequence selected from the group consisting of SEQ ID NOs.: 1-40, 42, 45 and 47-53. In related embodiments, the oligonucleotide decoys comprise a sequence having at least 90% identity with a sequence selected from the group consisting of SEQ ID NOs.: 1-39, 42, 45 and 47-52. In other embodiments, the oligonucleotide decoys comprise a sequence having at least 85% identity with a sequence selected from the group consisting of SEQ ID NOs.: 1-17, 19-39, 42, 45 and 47-53. In other embodiments, the oligonucleotide decoys comprise a sequence having at least 80% identity with a sequence selected from the group consisting of SEQ ID NOs.: 1-5, 7-17, 19-39, 42, 45 and 47-53. In other embodiments, the oligonucleotide decoys comprise a sequence having at least 75% identity with a sequence selected from the group consisting of SEQ ID NOs.: 1-4, 7-9, 13, 15-17, 19-23, 26-39, 45, 48, 50, 51 and 53. In other embodiments, the oligonucleotide decoys comprise a sequence having at least 70% identity with a sequence selected from the group consisting of SEQ ID NOs.: 1-3, 7-9, 13, 15-17, 19-23, 26, 28, 30, 32, 34-36, 38-39 and 48. In other embodiments, the oligonucleotide decoys comprise a sequence having at least 65% identity with a sequence selected from the group consisting of SEQ ID NOs.: 2-3,9,13,15-16, 19-23, 26, 28, 30, 32, 34-36, 38 and 39. In other embodiments, the oligonucleotide decoys comprise a sequence having at least 60% identity with a sequence selected from the group consisting of SEQ ID NOs.: 2, 13, 15-16, 21, 23, 26, 30, 32, 34-36, 38 and 39. In still other embodiments, the oligonucleotide decoys comprise a sequence having at least 55% identity with a sequence selected from the group consisting of SEQ ID NOs.: 16, 23, 30, 32, 34, 35, 38 and 39. In still other embodiments, the oligonucleotide decoys comprise a sequence having at least 50% identity with a sequence selected from the group consisting of SEQ ID NOs.: 30, 32, 35, and 38.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (1):\n\n5\u2032-SnnnnATDBNddnnnnnATD. . . BNHHnnnnnnS-3\u2032 (SEQ ID NO.: 86)\u2003\u2003(1)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (1) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, or 7) nucleotides selected from the group consisting of d, d, n, n, n, n, and n. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of d, d, n, n, n, n, and nhave at least 70% identity to the nucleotide sequence of SEQ ID NO.: 1.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (2):\n\n5\u2032-SnnnnnYCVYRNGnncvydb. . . gyCVYRBGRnnnnnnS-3\u2032 (SEQ ID NO.: 87)\u2003\u2003(2)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (2) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8 or 9) nucleotides selected from the group consisting of n, n, c, v, y, d, b, g, and y. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of n, n, c, v, y, d, b, g, and yhave at least 60% identity to the nucleotide sequence of SEQ ID NO.: 2.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (3):\n\n5\u2032-SnnWWGSGKRGGMnnnwwwg. . . sgKRGGMDnnnnnS-3\u2032 (SEQ ID NO.: 88)\u2003\u2003(3)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (3) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8 or 9) nucleotides selected from the group consisting of n, n, n, w, w, w, g, s, and g.","In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of n, n, n, w, w, w, g, s, and ghave at least 65% identity to the nucleotide sequence of SEQ ID NO.: 3.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (4):\n\n5\u2032-SnnnnnnTKASSbmnnTKAS. . . SBMNnnnnS-3\u2032 (SEQ ID NO.: 89)\u2003\u2003(4)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (4) comprises a deletion of one or more (e.g. 1, 2, 3 or 4) nucleotides selected from the group consisting of b, m, n, and n. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of b, m, n, and nhave at least 75% identity to the nucleotide sequence of SEQ ID NO.: 4.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (5):\n\n5\u2032-SSnnnnTGASknhrrrrGAS. . . KNHrrnnnSS-3\u2032 (SEQ ID NO.: 90)\u2003\u2003(5)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (5) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6 or 7) nucleotides selected from the group consisting of k, n, h, r, r, r, and t. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of k, n, h, r, r, r, and thave at least 80% identity to the nucleotide sequence of SEQ ID NO.: 5.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (6):\n\n5\u2032-SnnnnwwwGATTKTssaaks. . . ngATTKTCSAAKSnnnS-3\u2032 (SEQ ID NO.: 91)\u2003\u2003(6)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (6) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7 or 8) nucleotides selected from the group consisting of s, s, a, a, k, s, n, and g. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of s, s, a, a, k, s, n, and ghave at least 85% identity to the nucleotide sequence of SEQ ID NO.: 6.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (7):\n\n5\u2032-SnnnnnggatrtCCATATTA. . . GGagatnnnnwwsS-3\u2032 (SEQ ID NO.: 92)\u2003\u2003(7)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (7) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16 or 17) nucleotides selected from the group consisting of g, g, a, t, r, t, a, g, a, t, n, n, n, n, w, wand s. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of g, g, a, t, r, t, a, a, a, t, n, n, n, n, w, wand shave at least 70% identity to the nucleotide sequence of SEQ ID NO.: 7.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (8):\n\n5\u2032-SnnnnCAGGAdddddddddT. . . CCATATTAGnnnnS-3\u2032 (SEQ ID NO.: 93)\u2003\u2003(8)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (8) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8 or 9) nucleotides selected from the group consisting of d, d, d, d, d, d, d, dand d. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of d, d, d, d, d, d, d, dand dhave at least 70% identity to the nucleotide sequence of SEQ ID NO.: 8.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (9):\n\n5\u2032-SnnnnCTAWAMWTAAnnnnc. . . tAWAAATAAAAnnnS-3\u2032 (SEQ ID NO.: 94)\u2003\u2003(9)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (9) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5 or 6) nucleotides selected from the group consisting of n, n, n, n, Cand t. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of n, n, n, n, cand thave at least 65% identity to the nucleotide sequence of SEQ ID NO.: 9.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (10):\n\n5\u2032-nnnnRRGSCsKrrnnnrrGS. . . CKRRNnnnnn-3\u2032 (SEQ ID NO.: 95)\u2003\u2003(10)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (10) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6 or 7) nucleotides selected from the group consisting of r, r, n, n, n, r, and r. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of n, n, n, n, cand thave at least 80% identity to the nucleotide sequence of SEQ ID NO.: 10.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (11):\n\n5\u2032-nnnnnGGCGGGGssssssss. . . sssCGGGCGGTTTAC-3\u2032 (SEQ ID NO.: 96)\u2003\u2003(11)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (11) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9 10 or 11) nucleotides selected from the group consisting of s, s, s, s, s, s, s, s, s, s, and s. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of s, s, s, s, s, s, s, s, s, s, and shave at least 80% identity to the nucleotide sequence of SEQ ID NO.: 11.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (12):\n\n5\u2032-SnnnnWGYGGtddddgWGYG. . . GTDDDDnnS-3\u2032 (SEQ ID NO.: 97)\u2003\u2003(12)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (12) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5 or 6) nucleotides selected from the group consisting of t, h, h, h, h, and g. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of t, h, h, h, h, and ghave at least 80% identity to the nucleotide sequence of SEQ ID NO.: 12.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (13):\n\n5\u2032-SnnnnTTGGGGTCATAnnnn. . . CACAGGAACCACAnnS-3\u2032 (SEQ ID NO.: 98)\u2003\u2003(13)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (13) comprises a deletion of one or more (e.g., 1, 2, 3 or 4) nucleotides selected from the group consisting of n, n, nand n. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of n, n, nand nhave at least 60% identity to the nucleotide sequence of SEQ ID NO.: 13.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (14):\n\n5\u2032-SnnnnnCHGGAHRynnncCG. . . GAHRYnnnnnnS-3\u2032 (SEQ ID NO.: 99)\u2003\u2003(14)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (14) comprises a deletion of one or more (e.g., 1, 2, 3, 4 or 5) nucleotides selected from the group consisting of y, n, n, nand c. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of y, n, n, nand chave at least 80% identity to the nucleotide sequence of SEQ ID NO.: 14.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (15):\n\n5\u2032-SnnMWWGGAAAAnndwwgga. . . aaanndwGGAAAAnnnnnnS-3\u2032 (SEQ ID NO.: 100)\u2003\u2003(15)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (15) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 or 15) nucleotides selected from the group consisting of n, n, d, w, w, g, g, a, a, a, a, n, n, dand w. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of n, n, d, w, w, g, g, a, a, a, a, n, n, dand whave at least 60% identity to the nucleotide sequence of SEQ ID NO.: 15.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (16):\n\n5\u2032-SnnnnnCACTTCC-yvmnnny. . . vCTTCCTGCnnnS-3\u2032 (SEQ ID NO.: 101)\u2003\u2003(16)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (16) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7 or 8) nucleotides selected from the group consisting of y, v, m, n, n, n, yand v. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of y, v, m, n, n, n, yand vhave at least 55% identity to the nucleotide sequence of SEQ ID NO.: 16.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (17):\n\n5\u2032-SnnnnnCTATAAATggcctA. . . TAAATGggggggS-3\u2032 (SEQ ID NO.: 102)\u2003\u2003(17)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (17) comprises a deletion of one or more (e.g., 1, 2, 3, 4 or 5) nucleotides selected from the group consisting of g, g, c, cand t. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of g, g, c, cand thave at least 70% identity to the nucleotide sequence of SEQ ID NO.: 17.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (18):\n\n5\u2032-SnnnnnnWWCGCGGwwggww. . . wCCGGWWnnnnnS-3\u2032 (SEQ ID NO.: 103)\u2003\u2003(18)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (18) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6 or 7) nucleotides selected from the group consisting of w, w, g, g, w, wand w. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of w, w, g, g, w, wand whave at least 90% identity to the nucleotide sequence of SEQ ID NO.: 18.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (19):\n\n5\u2032-SnnnTGCCTTATCTctnngg. . . GATAASnnnnS-3\u2032 (SEQ ID NO.: 104)\u2003\u2003(19)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (19) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5 or 6) nucleotides selected from the group consisting of c, t, n, n, gand g. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of c, t, n, n, gand ghave at least 65% identity to the nucleotide sequence of SEQ ID NO.: 19.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (20):\n\n5\u2032-SnnnnnTGAATwwgaggaaa. . . awwGCATGCnnS-3\u2032 (SEQ ID NO.: 105)\u2003\u2003(20)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (20) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11 or 12) nucleotides selected from the group consisting of w, w, g, a, g, g, a, a, a, a, wand w. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of w, w, g, a, g, g, a, a, a, a, wand whave at least 65% identity to the nucleotide sequence of SEQ ID NO.: 20.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (21):\n\n5\u2032-SnnnnGAGATTkcacnnnga. . . gatTKCACnnnnS-3\u2032 (SEQ ID NO.: 106)\u2003\u2003(21)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (21) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11 or 12) nucleotides selected from the group consisting of k, c, a, c, n, n, n, g, a, g, aand t. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of k, c, a, c, n, n, n, g, a, g, aand thave at least 60% identity to the nucleotide sequence of SEQ ID NO.: 21.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (22):\n\n5\u2032-SnnnnKCMTWAWArmwnrmw. . . KCMTWAWTnnnS-3\u2032 (SEQ ID NO.: 107)\u2003\u2003(22)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (22) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7 or 8) nucleotides selected from the group consisting of t, r, m, w, n, r, mand w. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of t, r, m, w, n, r, mand whave at least 65% identity to the nucleotide sequence of SEQ ID NO.: 22.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (23):\n\n5\u2032-SnnnAGKYAADNDThhhnnn. . . hhHYAADNDTWVMtgc-3\u2032 (SEQ ID NO.: 108)\u2003\u2003(23)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (23) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7 or 8) nucleotides selected from the group consisting of h, h, h, n, n, n, hand h. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of h, h, h, n, n, n, hand hhave at least 55% identity to the nucleotide sequence of SEQ ID NO.: 23.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (24):\n\n5\u2032-SnnnnAATAAtnnatTATTw. . . wnnnS-3\u2032 (SEQ ID NO.: 109)\u2003\u2003(24)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (24) comprises a deletion of one or more (e.g., 1, 2, 3, 4 or 5) nucleotides selected from the group consisting of t, n, n, aand t. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of t, n, n, aand thave at least 80% identity to the nucleotide sequence of SEQ ID NO.: 24.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (25):\n\n5\u2032-SnnnSDHWMSHkwwmcssdh. . . wmshKWWMCSnnnnS-3\u2032 (SEQ ID NO.: 110)\u2003\u2003(25)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (25) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12 or 13) nucleotides selected from the group consisting of k, w, w, m, c, s, s, d, h, w, m, sand h. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of k, w, w, m, c, s, s, d, h, w, m, sand hhave at least 80% identity to the nucleotide sequence of SEQ ID NO.: 25.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (26):\n\n5\u2032-SnnnykgykGAAyhbbnnny. . . hbbkGAATATCnnS-3\u2032 (SEQ ID NO.: 111)\u2003\u2003(26)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (26) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11 or 12) nucleotides selected from the group consisting of y, h, b, b, n, n, n, y, h, b, band k. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of y, h, b, b, n, n, n, y, h, b, band khave at least 60% identity to the nucleotide sequence of SEQ ID NO.: 26.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (27):\n\n5\u2032-SnnnTATAWwwnndntatAW. . . wwnnwWTAADWnnnnnS-3\u2032 (SEQ ID NO.: 112)\u2003\u2003(27)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (27) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13 or 14) nucleotides selected from the group consisting of w, w, n, n, d, n, t, a, t, w, w, n, n, and w. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of w, w, n, n, d, n, t, a, t, w, w, n, n, and whave at least 75% identity to the nucleotide sequence of SEQ ID NO.: 27.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (28):\n\n5\u2032-SnnnTATAAWWnnnnwwwAA. . . WWknnnnnS-3\u2032 (SEQ ID NO.: 113)\u2003\u2003(28)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (28) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6 or 7) nucleotides selected from the group consisting of n, n, n, n, w, wand w. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of n, n, n, n, w, wand whave at least 65% identity to the nucleotide sequence of SEQ ID NO.: 28.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (29):\n\n5\u2032-NnnCTGMKYKKYtmbyCAAT. . . sdnnnS-3\u2032 (SEQ ID NO.: 114)\u2003\u2003(29)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (29) comprises a deletion of one or more (e.g., 1, 2, 3 or 4) nucleotides selected from the group consisting of t, m, band y. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of t, m, band yhave at least 75% identity to the nucleotide sequence of SEQ ID NO.: 29.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (30):\n\n5\u2032-SnnTCTCYGATTGGYyhybn. . . nnyyhhvGATTGGYTCBYnS-3\u2032 (SEQ ID NO.: 115)\u2003\u2003(30)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (30) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11 or 12) nucleotides selected from the group consisting of y, h, y, b, n, n, n, y, y, h, hand v. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of y, h, y, b, n, n, n, y, y, h, hand vhave at least 50% identity to the nucleotide sequence of SEQ ID NO.: 30.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (31):\n\n5\u2032-SnnCACCCsassswssswCA. . . CCCannnS-3\u2032 (SEQ ID NO.: 116)\u2003\u2003(31)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (31) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9 or 10) nucleotides selected from the group consisting of s, a, s, s, s, w, s, s, sand w. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of s, a, s, s, s, w, s, s, sand whave at least 75% identity to the nucleotide sequence of SEQ ID NO.: 31.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (32):\n\n5\u2032-SnnCCTWTGCCTyyyyynnn. . . yyyyyGCCTCCTWSnnS-3\u2032 (SEQ ID NO.: 117)\u2003\u2003(32)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (32) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12 or 13) nucleotides selected from the group consisting of y, y, y, y, y, n, n, n, y, y, y, yand y. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of y, y, y, y, y, n, n, n, y, y, y, yand yhave at least 50% identity to the nucleotide sequence of SEQ ID NO.: 32.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (33):\n\n5\u2032-SnnnWWWGGGwdgnnwwwGG. . . GWDGnnnnS-3\u2032 (SEQ ID NO.: 118)\u2003\u2003(33)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (33) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7 or 8) nucleotides selected from the group consisting of w, d, g, n, n, w, wand w. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of w, d, g, n, n, w, wand whave at least 75% identity to the nucleotide sequence of SEQ ID NO.: 33.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (34):\n\n5\u2032-SwwwwwCACTCAGCwwwwcg. . . gwgwGGGWWgwwwwwS-3\u2032 (SEQ ID NO.: 119)\u2003\u2003(34)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (34) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9 or 10) nucleotides selected from the group consisting of w, w, w, w, c, g, g, w, gand w. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of w, w, w, w, c, g, g, w, gand whave at least 55% identity to the nucleotide sequence of SEQ ID NO.: 34.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (35):\n\n5\u2032-SnnWBYAGYACCDNRGHsAG. . . CNNHnnnWBYAGYACCDNRG. . . HSAGCNNHnnS-3\u2032 (SEQ ID NO.: 120)\u2003\u2003(35)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (35) comprises a deletion of one or more (e.g., 1, 2 or 3) nucleotides selected from the group consisting of n, nand n. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of n, nand nhave at least 50% identity to the nucleotide sequence of SEQ ID NO.: 35.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (36):\n\n5\u2032-SnnnnGARMAWksagknnnn. . . garmAWKSAGKnnnnS-3\u2032 (SEQ ID NO.: 121)\u2003\u2003(36)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (36) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12 or 13) nucleotides selected from the group consisting of k, s, a, g, k, n, n, n, n, g, a, rand m. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of k, s, a, g, k, n, n, n, n, g, a, rand mhave at least 60% identity to the nucleotide sequence of SEQ ID NO.: 36.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (37):\n\n5\u2032-SnnnnGARGCCSswgwnnnn. . . garGCCSSWGWnnnS-3\u2032 (SEQ ID NO.: 122)\u2003\u2003(37)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (37) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 or 11) nucleotides selected from the group consisting of s, w, g, w, n, n, n, n, g, aand r. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of s, w, g, w, n, n, n, n, g, aand rhave at least 75% identity to the nucleotide sequence of SEQ ID NO.: 37.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (38):\n\n5\u2032-sCGAAAGGACAAAssnvvnn. . . nsgdnnGGACAAAGGTCAs-3\u2032 (SEQ ID NO.: 123)\u2003\u2003(38)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (38) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9 or 10) nucleotides selected from the group consisting of s, s, n, v, v, n, n, n, sand g. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of s, s, n, v, v, n, n, n, sand ghave at least 50% identity to the nucleotide sequence of SEQ ID NO.: 38.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (39):\n\n5\u2032-SnnnARMRWWywmgnnarmr. . . wwyWMGAATTnnnnS-3\u2032 (SEQ ID NO.: 124)\u2003\u2003(39)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (39) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12 or 13) nucleotides selected from the group consisting of y, w, m, g, n, n, a, r, m, r, w, wand y. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of y, w, m, g, n, n, a, r, m, r, w, wand yhave at least 55% identity to the nucleotide sequence of SEQ ID NO.: 39.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (47):\n\n5\u2032-SnnnnnCACTTCCTGCnnnnnS-3\u2032 (SEQ ID NO.: 125)\u2003\u2003(47)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (47) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9 or 10) nucleotides selected from the group consisting of n, n, n, n, n, n, n, n, nand n. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of n, n, n, n, n, n, n, n, nand nhave at least 80% identity to the nucleotide sequence of SEQ ID NO.: 47.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (48):\n\n5\u2032-SnnnnnAGKYAADNDTWVMN. . . nnnnnS-3\u2032 (SEQ ID NO.: 126)\u2003\u2003(48)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (48) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9 or 10) nucleotides selected from the group consisting of n, n, n, n, n, n, n, n, nand n. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of n, n, n, n, n, n, n, n, nand nhave at least 70% identity to the nucleotide sequence of SEQ ID NO.: 48.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (49):\n\n5\u2032-SnnTCTCYGATTGGYTCBYnS-3\u2032 (SEQ ID NO.: 127)\u2003\u2003(49)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (49) comprises a deletion of one or more (e.g., 1, 2 or 3) nucleotides selected from the group consisting of n, nand n. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of n, nand nhave at least 80% identity to the nucleotide sequence of SEQ ID NO.: 49.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (50):\n\n5\u2032-SnnnnnCCTWTGCCTCCTWS. . . rrnnnS-3\u2032 (SEQ ID NO.: 128)\u2003\u2003(50)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (50) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9 or 10) nucleotides selected from the group consisting of n, n, n, n, n, r, r, n, nand n. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of n, n, n, n, n, r, r, n, nand nhave at least 75% identity to the nucleotide sequence of SEQ ID NO.: 50.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (51):\n\n5\u2032-SnnnnWBYAGYACCDNRGHS. . . AGCNNHnnnnS-3\u2032 (SEQ ID NO.: 129)\u2003\u2003(51)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (51) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7 or 8) nucleotides selected from the group consisting of n, n, n, n, n, n, nand n. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of n, n, n, n, n, n, nand nhave at least 75% identity to the nucleotide sequence of SEQ ID NO.: 51.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (52):\n\n5\u2032-SmrmWAGGNCAAAGGTCAnn. . . nnS-3\u2032 (SEQ ID NO.: 130)\u2003\u2003(52)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (52) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7 or 8) nucleotides selected from the group consisting of m, r, m, n, n, n, nand g. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of m, r, m, n, n, n, nand ghave at least 80% identity to the nucleotide sequence of SEQ ID NO.: 52.","In certain embodiments, an oligonucleotide decoy comprises a double-stranded sequence represented by formula (53):\n\n5\u2032-SscttgykgykGAATATcgn. . . nnnnS-3\u2032 (SEQ ID NO.: 131)\u2003\u2003(53)\n\n","In certain embodiments, an oligonucleotide decoy represented by formula (53) comprises a deletion of one or more (e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16 or 17) nucleotides selected from the group consisting of s, c, t, t, g, y, k, g, y, k, c, g, n, n, n, nand n. In certain embodiments, oligonucleotide decoys comprising a deletion of one or more nucleotides selected from the group consisting of s, C, t, t, g, y, k, g, y, k, c, g, n, n, n, nand nhave at least 75% identity to the nucleotide sequence of SEQ ID NO.: 53.","A double stranded oligonucleotide having a certain percent (e.g., 65%, 70%, 75%, 80%, 85%, 90%, 95%, or 99%) of sequence identity with another sequence means that, when aligned, that percentage determines the level of correspondence of bases arrangement in comparing the two sequences. This alignment and the percent homology or identity can be determined using any suitable software program known in the art that allows local alignment. The software program should be capable of finding regions of local identity between two sequences without the need to include the entire length of the sequences. In some embodiments, such program includes but is not limited to the EMBOSS Pairwise Alignment Algorithm (available from the European Bioinformatics Institute (EBI)), the ClustalW program (also available from the European Bioinformatics Institute (EBI)), or the BLAST program (BLAST Manual, Altschul et al., Natl Cent. Biotechnol. Inf., Natl Lib. Med. (NCIB NLM NIH), Bethesda, Md., and Altschul et al., (1997) NAR 25:3389 3402).","One skilled in the art will recognize that sequences encompassed by the invention include those that hybridize under stringent hybridization conditions with an exemplified sequence (e.g., SEQ ID NOs.: 1-42, 45, and 47-53). A nucleic acid is hybridizable to another nucleic acid when a single stranded form of the nucleic acid can anneal to the other single stranded nucleic acid under appropriate conditions of temperature and solution ionic strength. Hybridization conditions are well known in the art. In some embodiments, annealing may occur during a slow decrease of temperature from a denaturizing temperature (e.g. 100\u00b0 C.) to room temperature in a salt containing solvent (e.g., Tris-EDTA buffer).","Generally, the oligonucleotide decoys disclosed herein may be used to bind and, e.g., thereby inhibit, transcription factors that modulate the expression of genes involved nociceptive signaling and\/or a subject's (e.g., patient's) perception of pain. A oligonucleotide decoy disclosed herein designed to bind to a specific transcription factor has a nucleic acid sequence mimicking the endogenous genomics DNA sequence normally bound by the transcription factor. Accordingly, the oligonucleotide decoys disclosed herein inhibit a necessary step for gene expression. Further, the oligonucleotide decoys disclosed herein may bind to a number of different transcription factors. The oligonucleotide decoys disclosed herein may be chemically modified by methods well known to the skilled artisan (e.g., incorporation of phosphorothioate, methylphosphonate, phosphorodithioate, phosphoramidates, carbonate, thioether, siloxane, acetamidate or carboxymethyl ester linkages between nucleotides) to prevent degradation by nucleases within cells and extra-cellular fluids (e.g., serum, cerebrospinal fluid). Also, oligonucleotide decoys may be designed that form hairpin and dumbbell structures which also prevent or hinder nuclease degradation. Further, the oligonucleotide decoys may also be inserted as a portion of a larger plasmid capable of episomal maintenance or constitutive replication in the target cell in order to provide longer term, enhanced intracellular exposure to the decoy sequence and\/or reduce its degradation. Accordingly, any chemical modification or structural alteration known in the art to enhance oligonucleotide stability is within the scope of the present disclosure. In some embodiments, the oligonucleotide decoys disclosed herein may be attached, for example, to polyethylene glycol polymers, peptides (e.g., a protein translocation domain) or proteins which improve the therapeutic effect of oligonucleotide decoys. Such modified oligonucleotide decoys may preferentially traverse the cell membrane.","In certain embodiments, the oligonucleotide decoys are provided as salts, hydrates, solvates, or N-oxide derivatives. In certain embodiments, the oligonucleotide decoys are provided in solution (e.g., a saline solution having a physiologic pH) or in lyophilzed form. In other embodiments, the oligonucleotide decoys are provided in liposomes.","In certain embodiments, one or more oligonucleotide decoys are provided in a kit. In certain embodiments, the kit includes an instruction, e.g., for using said one or more oligonucleotide decoys. In certain embodiments, said instruction describes one or more of the methods of the present invention, e.g., a method for preventing or treating pain, a method of modulating gene expression in a cell, a method for modulating nociceptive signaling in a cell, a method for modulating protein degradation in a cell, etc. In certain embodiments, the oligonucleotide decoys provided in a kit are provided in lyophilized form. In certain related embodiments, a kit that comprises one or more lyophilized oligonucleotide decoys further comprises a solution (e.g., a pharmaceutically acceptable saline solution) that can be used to resuspend said one or more of the oligonucleotide decoys.","The double stranded oligonucleotides described herein may be made by conventional methods known in the art and thus are well within the ambit of the skilled artisan.","Pharmaceutical Compositions","The pharmaceutical compositions disclosed herein comprise a therapeutically effective amount of one or more oligonucleotide decoys, preferably, in purified form, together with a suitable amount of a pharmaceutically acceptable vehicle, so as to provide a form for proper administration to a patient. When administered to a patient, oligonucleotide decoys and pharmaceutically acceptable vehicles are preferably sterile. Water is a preferred vehicle when oligonucleotide decoys are administered intravenously. Saline solutions and aqueous dextrose and glycerol solutions can also be employed as liquid vehicles, particularly for injectable solutions. Suitable pharmaceutical vehicles include excipients such as starch, glucose, lactose, sucrose, gelatin, malt, rice, flour, chalk, silica gel, sodium stearate, glycerol monostearate, talc, sodium chloride, dried skim milk, glycerol, propylene, glycol, water, ethanol and the like. The present pharmaceutical compositions, if desired, can also contain minor amounts of wetting or emulsifying agents, or pH buffering agents. In addition, auxiliary, stabilizing, thickening, lubricating and coloring agents may be used.","Pharmaceutical compositions may be manufactured by means of conventional mixing, dissolving, granulating, dragee-making, levigating, emulsifying, encapsulating, entrapping or lyophilizing processes. Pharmaceutical compositions may be formulated in conventional manner using one or more physiologically acceptable carriers, diluents, excipients or auxiliaries, which facilitate processing of compounds disclosed herein into preparations which can be used pharmaceutically. Proper formulation is dependent upon the route of administration chosen.","The present pharmaceutical compositions can take the form of solutions, suspensions, emulsions, tablets, pills, pellets, capsules, capsules containing liquids, powders, sustained-release formulations, suppositories, aerosols, sprays, suspensions, or any other form suitable for use. Other examples of suitable pharmaceutical vehicles have been described in the art (see Remington's Pharmaceutical Sciences, Philadelphia College of Pharmacy and Science, 19th Edition, 1995).","Pharmaceutical compositions for oral delivery may be in the form of tablets, lozenges, aqueous or oily suspensions, granules, powders, emulsions, capsules, syrups, or elixirs, for example. Orally administered compositions may contain one or more optional agents, for example, sweetening agents such as fructose, aspartame or saccharin, flavoring agents such as peppermint, oil of wintergreen, or cherry coloring agents and preserving agents, to provide a pharmaceutically palatable preparation. Moreover, when in tablet or pill form, the compositions may be coated to delay disintegration and absorption in the gastrointestinal tract, thereby providing a sustained action over an extended period of time. Oral compositions can include standard vehicles such as mannitol, lactose, starch, magnesium stearate, sodium saccharine, cellulose, magnesium carbonate, etc. Such vehicles are preferably of pharmaceutical grade.","For oral liquid preparations such as, for example, suspensions, elixirs and solutions, suitable carriers, excipients or diluents include water, saline, alkyleneglycols (e.g. propylene glycol), polyalkylene glycols (e.g. polyethylene glycol), oils, alcohols, slightly acidic buffers between pH 4 and pH 6 (e.g. acetate, citrate, or ascorbate at between about 5 mM to about 50 mM), etc. Additionally, flavoring agents, preservatives, coloring agents, bile salts, acylcarnitines and the like may be added.","Compositions for administration via other routes may also be contemplated. For buccal administration, the compositions may take the form of tablets, lozenges, etc., formulated in conventional manner. Liquid drug formulations suitable for use with nebulizers and liquid spray devices and EHD aerosol devices will typically include a compound with a pharmaceutically acceptable vehicle. Preferably, the pharmaceutically acceptable vehicle is a liquid such as alcohol, water, polyethylene glycol or a perfluorocarbon. Optionally, another material may be added to alter the aerosol properties of the solution or suspension of compounds. Preferably, this material is liquid such as an alcohol, glycol, polyglycol or a fatty acid. Other methods of formulating liquid drug solutions or suspension suitable for use in aerosol devices are known to those of skill in the art (see, e.g., Biesalski, U.S. Pat. No. 5,112,598; Biesalski, U.S. Pat. No. 5,556,611). A compound may also be formulated in rectal or vaginal compositions such as suppositories or retention enemas, e.g. containing conventional suppository bases such as cocoa butter or other glycerides. In addition to the formulations described previously, a compound may also be formulated as a depot preparation. Such long acting formulations may be administered by implantation (for example, subcutaneously or intramuscularly) or by intramuscular injection. Thus, for example, a compound may be formulated with suitable polymeric or hydrophobic materials (for example, as an emulsion in an acceptable oil) or ion exchange resins, or as sparingly soluble derivatives, for example, as a sparingly soluble salt.","An oligonucleotide decoy may be included in any of the above-described formulations, or in any other suitable formulation, as a pharmaceutically acceptable salt, a solvate or hydrate. Pharmaceutically acceptable salts substantially retain the activity of the parent compound and may be prepared by reaction with appropriate bases or acids and tend to be more soluble in aqueous and other protic solvents than the corresponding parent form.","Therapeutic Uses","In certain embodiments, an oligonucleotide decoy and\/or pharmaceutical composition thereof is administered to a patient, such as an animal (e.g., a bird, mammal, primate, or human), suffering from pain including, but not limited to, mechanical pain (e.g., mechanical hyperalgesia and\/or allodynia), chemical pain, temperature pain, chronic pain, sub-chronic pain, acute pain, sub-acute pain, inflammatory pain, neuropathic pain, muscular pain, skeletal pain, post-surgery pain, arthritis pain, and diabetes pain. Further, in certain embodiments, the oligonucleotide decoys and\/or pharmaceutical compositions thereof are administered to a patient, such as an animal, as a preventative measure against pain including, but not limited to, post-operative pain, chronic pain, inflammatory pain, neuropathic pain, muscular pain, and skeletal pain. In certain embodiments, the oligonucleotide decoys and\/or pharmaceutical compositions thereof may be used for the prevention of one facet of pain while concurrently treating another symptom of pain.","Thus, in certain embodiments, the invention provides methods of treating pain in a patient comprising administering to a patient suffering from pain a therapeutically effective amount of an oligonucleotide decoy described herein. In related embodiments, methods of preventing pain in a patient are provided. Such methods comprise administering to a patient in need thereof (e.g., a patient likely to develop pain, e.g., post-operative pain) a therapeutically effective amount of an oligonucleotide decoy described herein. In certain embodiments, the oligonucleotide decoy is administered perineurally, epidurally\/peridurally, intrathecally, or intradermally.","In certain embodiments, the invention provides methods for treating or preventing pain in a patient comprising administering to a patient in need thereof a therapeutically effective amount of an oligonucleotide decoy, wherein the oligonucleotide decoy does not bind to the transcription factors AP1, ETS1 and STAT. In other embodiments, the invention provides methods for treating or preventing pain in a patient comprising administering to the patient in need thereof a therapeutically effective amount of one or more oligonucleotide decoys, wherein the oligonucleotide decoys bind to one or more transcription factors selected from the group consisting of AP1, ETS1, GATA and STAT transcription factors, provided that the pain is not lower back pain due to an intervertebral disc disorder.","In certain embodiments, the invention provides methods for modulating transcription of a gene present in a cell involved in nociceptive signaling and\/or the perception of pain in a patient. In certain embodiments, modulation comprises suppressing or repressing gene expression. In other embodiments, modulation comprises stabilizing gene expression. In still other embodiments, modulation comprises activating or inducing gene expression. In certain embodiments, the gene is involved in nociceptive signaling. Genes involved in nociceptive signaling include, but are not limited to, genes encoding membrane proteins (e.g., ion channels, membrane receptors, etc.), soluble signaling molecules (e.g., intracellular signaling molecules or neurotransmitters), synthetic enzymes (e.g. neurotransmitter synthesis enzymes), and transcription factors. Specific examples of such genes include, but are not limited to, BDKRB2, HTR3A, SCN9A, BDNF, GRM5, NOS1, GCH1, CDK5R1, CACNA1B, P2AR3 and PNMT.","In other embodiments, the invention provides methods for modulating nociceptive signaling in a cell. In certain embodiments, modulation comprises suppressing or repressing nociceptive signaling. In certain embodiments, modulating nociceptive signaling in a cell comprises modulating, e.g., increasing, proteolysis of a protein involved in nociceptive signaling in said cell. For instance, abnormally high proteasome activity has been linked to strong deficits of neuronal plasticity (i.e., a major cellular feature of pain). EGR1 is known to repress the expression of selected proteasome factors, thus limiting EGR1-dependent nociceptive signaling activity is relevant for treating pain. Further, neutrophines activate specific receptors in pain neurons that trigger nociceptive signalings. USF factors activate the expression of CGRP and Substance P, two major neurotrophins capable of inducing pain. Inhibiting USF factors is a potential approach to inhibit nociceptive signaling. In certain embodiments, modulation comprises activation of an inhibitor of nociceptive signaling.","In still other embodiments, the invention provided methods for modulating, e.g. increasing, proteolytic degradation of a protein involved in nociceptive signaling in a cell. In certain embodiments, modulation of protein degradation comprises stimulating proteosome function. In certain embodiments, the protein is involved in nociceptive signaling. Proteins involved in nociceptive signaling include, but are not limited to membrane proteins (e.g. ion channels, membrane receptors, etc.), soluble signaling molecules (e.g., intracellular signaling molecules or neurotransmitters), synthetic enzymes (e.g. neurotransmitter synthesis enzymes), and transcription factors. Specific examples of such proteins include, but are not limited to, BDKRB2, HTR3A, SCN9A, BDNF, GRM5, NOS1, GCH1, CDK5R1, CACNA1B, P2XR3 and PNMT.","In certain embodiments, the cell of the various methods is provided in vivo (e.g., in a patient suffering from pain or likely to suffer from pain). A cell provided in vivo can be located in different locations including, but not limited to, a dorsal root ganglia and\/or the spinal cord. In other embodiments, the cell of the various methods is provided in vitro (e.g., in a petri dish). The cell can be any cell involved in nociceptive signaling, including, but not limited to, a neuron (e.g. a pain neuron from dorsal root ganglia and\/or the spinal cord or from the sympathetic nervous system), a glial cell, a tissue supportive cell (e.g., fibroblast), an immune cell, or a cell from a cell line (e.g., a PC12 cell).","Methods of Administration and Dosage","The present methods for treatment or prevention of pain require administration of a oligonucleotide decoys, or pharmaceutical compositions thereof, to a patient in need of such treatment or prevention. The compounds and\/or pharmaceutical compositions thereof may be administered by any convenient route, for example, by infusion or bolus injection, by absorption through epithelial or mucocutaneous linings (e.g. oral mucosa, rectal and intestinal mucosa, etc.), or orally. Administration can be systemic or local. Various delivery systems are known, including, e.g., encapsulation in liposomes, microparticles, microcapsules, capsules, etc., that can be used to administer a compound and\/or pharmaceutical composition thereof. Methods of administration include, but are not limited to, intradermal, intramuscular, intraperitoneal, intravenous, subcutaneous, intranasal, epidural\/peridural, oral, sublingual, intranasal, intracerebral, intravaginal, transdermal, rectally, by inhalation or topically, particularly to the ears, nose, eyes, or skin. In certain embodiments, more than one oligonucleotide decoy is administered to a patient. The preferred mode of administration is left to the discretion of the practitioner, and will depend in-part upon the site of the medical condition.","In specific embodiments, it may be desirable to administer one or more oligonucleotide decoys locally to the area in need of treatment. This may be achieved, for example, and not by way of limitation, by local infusion during surgery, topical application (e.g., in conjunction with a wound dressing after surgery), by injection, by means of a catheter, by means of a suppository, or by means of an implant, said implant being of a porous, non-porous, or gelatinous material, including membranes, such as sialastic membranes, or fibers. In some embodiments, administration can be by direct injection at the site (e.g., former, current, or expected site) of pain.","In certain embodiments, it may be desirable to introduce one or more oligonucleotide decoys into the nervous system by any suitable route, including but not restricted to intraventricular, intrathecal, perineural and\/or epidural\/peridural injection. Intraventricular injection may be facilitated by an intraventricular catheter, for example, attached to a reservoir, such as an Ommaya reservoir.","Pulmonary administration can also be employed, e.g., by use of an inhaler or nebulizer, and formulation with an aerosolizing agent, or via perfusion in a fluorocarbon or synthetic pulmonary surfactant.","The amount of oligonucleotide decoy that will be effective in the treatment or prevention of pain in a patient will depend on the specific nature of the condition and can be determined by standard clinical techniques known in the art. In addition, in vitro or in vivo assays may optionally be employed to help identify optimal dosage ranges. The amount of a oligonucleotide decoy administered will, of course, be dependent on, among other factors, the subject being treated, the weight of the subject, the severity of the affliction, the manner of administration, and the judgment of the prescribing physician. In certain embodiments, a single dose of oligonucleotide decoy comprises about 5 \u03bcgs to 5 mgs, 50 \u03bcgs to 2.5 mgs, 100 \u03bcgs to 1 mg, 250 \u03bcgs to 750 \u03bcgs, or about 500 \u03bcgs of oligonucleotide decoy per kilogram of body weight.","Preferably, the dosage forms are adapted to be administered to a patient no more than twice per day, more preferably, only once per day. Dosing may be provided alone or in combination with other drugs and may continue as long as required for effective treatment or prevention of pain.","Combination Therapy","In certain embodiments, oligonucleotide decoys and\/or pharmaceutical compositions thereof can be used in combination therapy with at least one other therapeutic agent which may include but is not limited to an oligonucleotide decoy. The oligonucleotide decoy and\/or pharmaceutical composition thereof and the therapeutic agent can act additively or, more preferably, synergistically. In some embodiments, an oligonucleotide decoy and\/or a pharmaceutical composition thereof is administered concurrently with the administration of another therapeutic agent, including another oligonucleotide decoy. In other embodiments, an oligonucleotide decoy or a pharmaceutical composition thereof is administered prior or subsequent to administration of another therapeutic agent, including another oligonucleotide decoy.","Experimental Protocols","The invention is further defined by reference to the following experimental protocol. It will be apparent to those skilled in the art that many modifications, both to materials and methods, may be practiced without departing from the scope of the invention.","The experimental model consists of mimicking a pain situation by applying to neuronal cell lines, primary dorsal root ganglion (DRG), and\/or spinal cord neurons a combination of pro-inflammatory mediators (e.g., nerve growth factor, interleukin-1\u03b2, bradykinin, serotonin, substance P, etc.) known to trigger the modulation of pain genes. Pain genes expression profiling is realized by semi-quantitative Reverse Transcription-Polymerase Chain Reaction (sqRT-PCR) in several experimental situations, including but not restricted to, following pro-inflammatory mediator stimulation, with or without double stranded oligonucleotide treatment. An overview of the experiment is shown below:",{"@attributes":{"id":"p-0174","num":"0219"},"chemistry":{"@attributes":{"id":"CHEM-US-00001","num":"00001"},"img":{"@attributes":{"id":"EMI-C00001","he":"44.53mm","wi":"93.39mm","file":"US07943591-20110517-C00001.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"Cells are cultured in vitro and may be submitted to independent situations including but not limited to:\n\n","After treatment, cells are collected and the RNA is extracted. Pain gene expression levels are measured possibly by semi-quantitative RT-PCR and the expression profiles of each situation are compared to each other.","Oligonucleotide decoy treatment consists of transfecting one or more (concurrently or in a sequence at a time interval yet to be determined) oligonucleotide decoys of sequences selected from SEQ ID NOs.: 1-45 in neuronal cell lines, DRG, and\/or spinal cord neurons. Cell lines include, but are not limited to, PC12 cells (NGF-differentiated or not), SH-SY5Y cells, Weri cells, Hela, HEK293, F-11, NS20Y, and ND7\/23 cells, or any other cell line expressing one or more genes that may be selected (e.g. ACCN1-3, BDKRB1-2, BDNF, CACNA1G-H, CALCA, GRIN1, GRM1, GRM5, HTR1-3, NTRK1, P2RX3, PLC, PRKC, etc.). One or more transfection(s) is applied to the same set of cells, including or not including the same single (or set of) oligonucleotide decoys. Cells, either cell lines or primary neurons, are collected at a time after oligonucleotide decoy treatment (e.g., 24 or 48 hours post-treatment). The transfection efficiency is measured by following the uptake of a labeled oligonucleotide decoy, possibly with a dye such as fluoresceine. The efficiency is given in percentage of total cells that contain the labeled oligonucleotide decoy.","Cultured cells are collected after treatment with oligonucleotide decoy and their RNA is extracted. Extracted RNA is transformed into cDNA by reverse transcription. The amount of cDNA of each selected gene, which reflects the amount of endogenous mRNA, is measured by PCR. The same amount of PCR reaction product is loaded on an agarose gel saturated with ethidium bromide or any other suitable agent for DNA detection. Detection of DNA is performed under a UV lamp or any other suitable device and gels images are analyzed with quantification software. The amount of DNA produced during each PCR reaction is normalized on the amount of DNA produced by the control PCR reactions from housekeeping genes (e.g., ACTB, GAPDH) which reflects the total quantity of RNA initially present in cells. The comparison of ratio signal \/control values obtained for each gene with and without oligonucleotide decoy treatment(s) will give a relative measure of the impact of each oligonucleotide decoy on the level of expression of genes.","Control experiments with mismatched (e.g., SEQ ID NO.: 43 annealed to SEQ ID NO.: 46, referred to hereinafter as SEQ ID NO.: 43\/46), scrambled, and\/or mutated double-stranded oligonucleotides are performed in parallel to ensure the measured effect is specific to each oligonucleotide decoy. Cell viability after oligonucleotide decoy treatment may be measured.","The same approach may be used with current pain drugs such as nonsteroid anti-inflammatory drugs or coxibs to compare with oligonucleotide decoys.","In certain embodiments, oligonucleotide decoys produce an effect in the expression pattern(s), which includes, but is not limited to, an inhibition and\/or an induction of one or more gene(s) that may be involved in nociceptive signaling and\/or the perception of pain in a patient. In certain embodiments, the inhibited gene(s) may encode pro-pain factors, like receptors of pro-inflammatory mediators, and the activated genes may encode anti-pain factors, like opioids receptors.","Strand Annealing","For oligonucleotide decoys consisting of a pair of complementary strands, the complementary strands are annealed, at equimolar concentration, in a saline buffer, e.g., Tris-EDTA (TE). The standard procedure includes maintaining the solution of both strands at a high denaturizing temperature (e.g., 100\u00b0 C.) for a period of time which may vary depending on the complementary strands, followed by a slow decrease in temperature (e.g., 0.3-1\u00b0 C.\/min) until the solution reaches a low temperature of annealing (e.g., 20\u00b0 C.). The proper annealing of complementary strands may be verified by any suitable standard technique, including but not restricted to running samples of annealed oligonucleotides next to un-annealed ones on a non-denaturing polyacrylamide gel. For oligonucleotide decoys that are self-annealing, substantially the same protocol is followed.","Cell Culture","DRG and\/or spinal cord cells can be collected from an animal (e.g., a mammal, such as a rat or mouse) and the neurons can be freshly dissociated, using collagenase (e.g., collagenase type II) at 37\u00b0 C. Cells isolated in such fashion can be plated on suitable Petri dishes (e.g., collagen coated). Neurons are maintained in appropriate media culture (e.g., DMEM). Cell lines are thawed and maintained in adequate media and Petri dishes according to the supplier recommendations. Cells are typically incubated at 37\u00b0 C., 5% CO. Cell lines are cultured according to supplier recommendations.","The invention is further illustrated by the following examples which should not be construed as limiting.","Oligonucleotide decoys of the invention include, but are not limited to, sequences presented in Table 1. In general, the oligonucleotide decoy is generated by annealing the sequence provided in the table with a complementary sequence. To generate a mismatch double-stranded oligonucleotide, the sequence provided in the table can be annealed to a sequence that is only partially complementary. For example, SEQ ID NO.:43 can be annealed to SEQ ID NO.:46 to produce the mismatched sequence, SEQ ID NO.:43\/46, described in the following Examples.",{"@attributes":{"id":"p-0186","num":"0235"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"154pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}}],"thead":{"row":[{"entry":"TABLE\u20031"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["Oligonucleotide\u2003Sequences\u2003(5\u2032-3\u2032)","SEQ\u2003ID\u2003NO."]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["GGCTTATGCAAATTCGAATGCAAATTTGTCG","SEQ\u2003ID\u2003NO.:\u20031"]},{"entry":{}},{"entry":["CTAAGCCCACGTGACCATTGGCCAGGTGACCAG","SEQ\u2003ID\u2003NO.:\u20032"]},{"entry":["ATC",{}]},{"entry":{}},{"entry":["GTTATGCGTGGGCGATAATGCGGGGGCGTTATAG","SEQ\u2003ID\u2003NO.:\u20033"]},{"entry":{}},{"entry":["GCCTCCCTGAGCTCATTGACGTATCTCGG","SEQ\u2003ID\u2003NO.:\u20034"]},{"entry":{}},{"entry":["CGAATATGACTGAGAATGACTCAGATTTGC","SEQ\u2003ID\u2003NO.:\u20035"]},{"entry":{}},{"entry":["GGTTCTATGATTTTGGAATCGGATTGTGCAAAGA","SEQ\u2003ID\u2003NO.:\u20036"]},{"entry":["AGC",{}]},{"entry":{}},{"entry":["GCTTCAGGATGTCCATATTAGGAGATCTTGTTCG","SEQ\u2003ID\u2003NO.:\u20037"]},{"entry":{}},{"entry":["GGCCACAGGATGTAGGATGTCCATATTAGGATGC","SEQ\u2003ID\u2003NO.:\u20038"]},{"entry":{}},{"entry":["GTTCTCTAAAAATAAAAGGCTAAAAATAAAAGTCG","SEQ\u2003ID\u2003NO.:\u20039"]},{"entry":{}},{"entry":["ATTAGGGGCGGGGTCCGGGGCGGGGTATTA","SEQ\u2003ID\u2003NO.:\u200310"]},{"entry":{}},{"entry":["GTTATGGCGGGGCGGGGCGGGGCCGGGCGGTTTAC","SEQ\u2003ID\u2003NO.:\u200311"]},{"entry":{}},{"entry":["GGCAATGTGGTTTTAGTGTGGTTTTACGG","SEQ\u2003ID\u2003NO.:\u200312"]},{"entry":{}},{"entry":["GCCGTTTGGGGTCATAGAACCACAGGAACCACA","SEQ\u2003ID\u2003NO.:\u200313"]},{"entry":["CGG",{}]},{"entry":{}},{"entry":["CATTGCCCGGAAATGGACCGGATGTAATTTCC","SEQ\u2003ID\u2003NO.:\u200314"]},{"entry":{}},{"entry":["GTTCTTGGAAAATAAATGGAAAATAGTGGAAAATA","SEQ\u2003ID\u2003NO.:\u200315"]},{"entry":["AGTCG",{}]},{"entry":{}},{"entry":["CGTTCCCACTTCCTGCGACCACTTCCTGCCGGG","SEQ\u2003ID\u2003NO.:\u200316"]},{"entry":{}},{"entry":["CTGCACCTATAAATGGCCTATAAATGGGGATGC","SEQ\u2003ID\u2003NO.:\u200317"]},{"entry":{}},{"entry":["GCTTATTTCGCGGAAGGTTTCCCGGAAGTGGCG","SEQ\u2003ID\u2003NO.:\u200318"]},{"entry":{}},{"entry":["GCTGTGCCTTATCTCTTTGGGATAACTGGCG","SEQ\u2003ID\u2003NO.:\u200319"]},{"entry":{}},{"entry":["GCTTAATGAATAAGAGGAAAAATGCATGCTGG","SEQ\u2003ID\u2003NO.:\u200320"]},{"entry":{}},{"entry":["GTTCTGAGATTGCACGATGAGATTTCACAGTCG","SEQ\u2003ID\u2003NO.:\u200321"]},{"entry":{}},{"entry":["GTCCCGCATAAATAATGGCATCCTTAATCGCG","SEQ\u2003ID\u2003NO.:\u200322"]},{"entry":{}},{"entry":["GTGCAGGCAAGAGTAGAGACAGGCAAGAGTAGA","SEQ\u2003ID\u2003NO.:\u200323"]},{"entry":["TGC",{}]},{"entry":{}},{"entry":["CCGCCAATAATTAATTATTAAGGCC","SEQ\u2003ID\u2003NO.:\u200324"]},{"entry":{}},{"entry":["GCTTCGTTCCATTTCCGGTCTCGGTTTCCCCATTC","SEQ\u2003ID\u2003NO.:\u200325"]},{"entry":{}},{"entry":["GCTGCTGTGGAATATCGACCTGTGGAATATCGTG","SEQ\u2003ID\u2003NO.:\u200326"]},{"entry":{}},{"entry":["GCCGTATAAATGTGCTATAAAAGTTTTAAGACCG","SEQ\u2003ID\u2003NO.:\u200327"]},{"entry":["TGC",{}]},{"entry":{}},{"entry":["GCCGTATAAATGTGCTATAAAAGCCGTGC","SEQ\u2003ID\u2003NO.:\u200328"]},{"entry":{}},{"entry":["ATGCTGCGCTTTTCTCCAATCTGCGG","SEQ\u2003ID\u2003NO.:\u200329"]},{"entry":{}},{"entry":["CGTTCTCCGATTGGTCACGGACTCTCCGATTGGTC","SEQ\u2003ID\u2003NO.:\u200330"]},{"entry":["ACGGC",{}]},{"entry":{}},{"entry":["GCGCACCCCAGCCTGGCTCACCCACGCG","SEQ\u2003ID\u2003NO.:\u200331"]},{"entry":{}},{"entry":["GATCCTTTGCCTCCTTCGATCCTTTGCCTCCTTC","SEQ\u2003ID\u2003NO.:\u200332"]},{"entry":["AAG",{}]},{"entry":{}},{"entry":["GGTGTTTGGGAGAGCTTTGGGAGGATACG","SEQ\u2003ID\u2003NO.:\u200333"]},{"entry":{}},{"entry":["GCTAATCACTCAGCATTTCGGTGAGGGAAGTGA","SEQ\u2003ID\u2003NO.:\u200334"]},{"entry":["AAG",{}]},{"entry":{}},{"entry":["CCTTTCAGCACCACGGACAGCGCCAGCTTCAGCAC","SEQ\u2003ID\u2003NO.:\u200335"]},{"entry":["CACGGACAGCGCCTCG",{}]},{"entry":{}},{"entry":["GGATCGAACATGGAGTCAGTGAGAAATCAGGAT","SEQ\u2003ID\u2003NO.:\u200336"]},{"entry":["CGG",{}]},{"entry":{}},{"entry":["GGATCGAAGCCGGAGTCAAGGAGGCCCCTGATCGG","SEQ\u2003ID\u2003NO.:\u200337"]},{"entry":{}},{"entry":["CCGAAAGGACAAAGGTCAAGTCGAAAGGACAAAGG","SEQ\u2003ID\u2003NO.:\u200338"]},{"entry":["TCAG",{}]},{"entry":{}},{"entry":["CGGGAGAAAATTCGGGAACGTTCAAGAATTGTCGG","SEQ\u2003ID\u2003NO.:\u200339"]},{"entry":{}},{"entry":["GTTATGCGTGGGCGTAGATGCGGGGGCGTTATAG","SEQ\u2003ID\u2003NO.:\u200340"]},{"entry":{}},{"entry":["GATGCGTGGGCGTAGG","SEQ\u2003ID\u2003NO.:\u200341"]},{"entry":{}},{"entry":["GTATGCGTGGGCGGTGGGCGTAG","SEQ\u2003ID\u2003NO.:\u200342"]},{"entry":{}},{"entry":["GTTATGCGTTTGTAGATGCTTTCGTTATAG","SEQ\u2003ID\u2003NO.:\u200343"]},{"entry":{}},{"entry":["GTTATGCGTGGGCGATATAG","SEQ\u2003ID\u2003NO.:\u200344"]},{"entry":{}},{"entry":["GATGCGTGGGCGTTGACGTGGAAAATGC","SEQ\u2003ID\u2003NO.:\u200345"]},{"entry":{}},{"entry":["CTATTTCGAAACGATCTACATTGGCATAAC","SEQ\u2003ID\u2003NO.:\u200346"]},{"entry":{}},{"entry":["CGTTCCCACTTCCTGCGACCGG","SEQ\u2003ID\u2003NO.:\u200347"]},{"entry":{}},{"entry":["GGGTGAAGGCAAGAGTAGAGCGGCGG","SEQ\u2003ID\u2003NO.:\u200348"]},{"entry":{}},{"entry":["CGTTCTCCGATTGGTCACGCG","SEQ\u2003ID\u2003NO.:\u200349"]},{"entry":{}},{"entry":["GTACTCCCTTTGCCTCCTTCAACCGG","SEQ\u2003ID\u2003NO.:\u200350"]},{"entry":{}},{"entry":["CCTTATTCAGCACCACGGACAGCGCCATTCG","SEQ\u2003ID\u2003NO.:\u200351"]},{"entry":{}},{"entry":["GCGAAAGGACAAAGGTCAGGCGG","SEQ\u2003ID\u2003NO.:\u200352"]},{"entry":{}},{"entry":["GGCTTGCTGTGGAATATCGATGGTG","SEQ\u2003ID\u2003NO.:\u200353"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}}}},"SEQ ID NO.: 3, which is designed to bind EGR1 transcription factor, has a structure that is typical of class of oligonucleotide decoys of the invention. The structure of SEQ ID NO.: 3 includes, in order from 5\u2032 to 3\u2032, a 5\u2032 flanking sequence, a first transcription factor binding site, a linker sequence, a second transcription factor binding site, and a 3\u2032 flanking sequence. SEQ ID NO.: 40, which has 94% identity with SEQ ID NO.: 3 and the same basic structure, is predicted in silico to bind EGR1 better than SEQ ID NO.: 3. Pharmacological analysis of SEQ ID NO.: 40 was performed using a transcription factor ELISA kit specific for EGR1 binding detection. The sensitivity of transcription factor ELISA technology is ten times more sensitive than classical EMSA experiments, allowing detailed pharmacological studies of transcription factor decoys.","The proper annealing of forward and reverse strands of SEQ ID NO.: 40 was confirmed on a 2.5% agarose gel, as shown in . Binding experiments were conducted with the human form of EGR1 (hEGR1) present in nuclear extracts of TPA-stimulated K-562 cells. See, e.g. .","Quantitative competition ELISA using SEQ ID NO.: 40 and SEQ ID NO.: 41 show that SEQ ID NO.: 40 exhibits strong hEGR1 binding activity, as shown in . In our experimental context, an half-inhibition concentration (IC) value represents the concentration of competitor that gives 50% inhibition of the probe binding measured in absence of the competitor and, thus, is a measure of the relative affinities of sequences against each other. The results indicate that SEQ ID NO.: 40, which contains two EGR1 transcription factor binding sites, bares a relative affinity to hEGR1 similar to the consensus SEQ ID NO.: 41, which contains a single EGR1 transcription factor binding site, with ICof 215 nM and 250 nM, respectively.","We discovered that SEQ ID NO.: 42, which is 70% homologous to SEQ ID NO.: 3 but includes a specific fusion of the two EGR1 transcription factor binding sites present in SEQ ID NO.: 3, has an affinity for EGR1 two times higher than the single consensus sequence, SEQ ID NO.: 41, with an ICof 99 nM. See .","Crystal structure experiments studies have shown that a single EGR1 protein is able to bind its consensus binding sequence through three zinc finger domains. It is known that protein-protein interactions can directly change DNA binding activities, as proven for the AP1 factors c-jun and c-fos, where the c-jun:c-fos dimer binds to AP1 response elements five to thirty times better than c-jun:c-jun dimers. Without intending to be bound, we believe that the fusion of the two EGR1 transcription factor binding sites present in SEQ ID NO.: 42 induces protein-protein interactions between two EGR1 factors and thereby mutually increases their DNA binding affinity. In any event, the very high affinity of SEQ ID NO.: 42 for EGR1, as compare to known binding sequences, makes SEQ ID NO.: 42 particularly attractive as a pharmaceutical inhibitor of hEGR1.","The absence of non-specific oligonucleotides binding effect in our ELISA experiments was demonstrated by the lack of EGR1 binding to the mismatch sequence, SEQ ID NO.: 43\/46, as shown in . In addition, SP1 and WT1 transcription factors, which are structurally related to EGR1 and are able to bind GC-rich DNA sequences similar to the EGR1 consensus binding sequence, bound poorly to EGR1 oligonucleotide decoys. ELISA experiments detecting hSP1 binding demonstrated that SEQ ID NO.: 40 bound poorly to SP1 as compare to the SP 1-specific oligonucleotide decoy, SEQ ID NO.: 11, with an OD value 80% lower. See  (top panel). Furthermore, competition experiments demonstrated that SEQ ID NO.: 42 does not bind efficiently to hSP1, even at high excess concentrations, as shown in , top and bottom panels. A similar lack of affinity was observed for EGR1 oligonucleotide binding to hWT1. See , top and bottom panels.","Altogether, pharmacological experiments reveal that SEQ ID NO.: 42 is a powerful hEGR1 inhibitor compound as (i) it has a higher relative affinity for hEGR1 as compare to both the single consensus binding site decoy (SEQ ID NO.: 41) and the double consensus binding site decoy (SEQ ID NO.: 40) and, (ii) it is highly specific.","The capacity of SEQ ID NO.: 40 and SEQ ID NO.: 42 to inhibit hEGR1 transcriptional activity in human cells was measured through their effect on CDK5R1 gene expression. CDK5R1 is an activator of the CDK5 kinase. Both are up-regulated in pain neurons following peripheral inflammation and regulate nociceptive signaling, notably via phosphorylation of the capsaicin receptor, TRPV1. hEGR1 directly binds to the CDK5R1 promoter in human HL60 cells and controls its up-regulation following cell differentiation by 1,25-Dihydroxyvitamin D3. Segment of the natural CDK5R1 promoter used as a decoy in HL60 cells are already known to inhibit CDK5R1 expression. We assessed the efficiency of our decoy sequences to inhibit hEGR1 activity by measuring the level of inhibition of CDK5R1 they confer following HL60 cell differentiation. CDK5R1 mRNA expression level was measured by sq RT-PCR (see, e.g., ) and half-inhibition concentrations ICrefers to the decoy concentration needed to produce 50% inhibition of the maximum CDK5R1 mRNA expression level measured after 1,25-Dihydroxyvitamin D3 differentiation.","We confirmed the up-regulation of CDK5R1 mRNA expression level following 1,25-Dihydroxyvitamin D3 application, as well as the presence of hEGR1 in HL60 cells, as shown in . The high transfection yield (70%) of our decoy sequences into HL60 cells is illustrated in .  shows that we did not measure any significant difference in the number of dead cells between 1,25-Dihydroxyvitamin D3 treatment alone and combined with decoy sequences at concentrations up to 1 \u03bcM, demonstrating the lack of toxicity of EGR1 decoy in HL60 cells.","Dose response experiments from 250 nM to 2 \u03bcM conducted with our hEGR1 decoy sequences are displayed in . SEQ ID NO.: 40 and SEQ ID NO.: 41 have a similar IC, with values of 544 nM and 529 nM, respectively. This is directly consistent with the fact that the two sequences display roughly the same binding affinity for hEGR1. SEQ ID NO.: 42 is over three time more effective at inhibiting CDK5R1 mRNA expression than the other decoys, with an IC=150 nM, reflecting its higher affinity for hEGR1. Typical pictures of CDK5R1 mRNA expression detection on agarose gels illustrating the differential ICof SEQ ID NO.: 40 and SEQ ID NO.: 42 are displayed in . Those data reveal a direct relationship between the relative affinities of hEGR1 decoy sequences and their efficiency in a cellular context and further confirm the therapeutic potential of SEQ ID NO.: 42 as an hEGR1 inhibitor and for treating pain.","The specificity of SEQ ID NO: 42 activity was verified using two methods. First, we verified the absence of CDK5R1 expression inhibition from the mismatch sequence SEQ ID NO.: 43\/46, which indicates the lack of non-specific nucleotide exposure effects. See , left panel. Second, we confirmed the specificity of SEQ ID NO.: 42 activity by showing its lack of effect on the regulation of BCL2, a anti-apoptotic gene that lacks hEGR1 response element within its promoter and is not known to be regulated by hEGR1 in HL60 cells. Consistent with previous observations, we measured a down-regulation of BCL2 mRNA expression after HL60 differentiation, and this down-regulation was not altered by SEQ ID NO.:42 oligonucleotide decoy treatment. See , right panel.","PC12 are pheochromocytoma cells extensively used as a model to investigate pain signaling pathways because they express and regulate numerous pain genes in a fashion similar to endogenous pain neurons in response to pro-inflammatory mediators such as NGF or cAMP elevating compounds. We measured the effect of seq ID NO.: 42 decoy treatment on pain genes expression profile. We selected 11 pain genes based on (i) their critical roles in multiple pain syndromes, (ii) their different positions along pain signaling pathways and (iii) the strong parallel between the regulation of their expression between endogenous pain neurons and PC12 cells. They belong to four genes classes: ion channels (Scn9a, Cacna1b), membrane receptors (Grm5, Bdkrb2, P2rx3, Htr3a), signaling and neurotransmitter synthesis enzymes and related proteins (Cdk5r1, Gch1, Pnmt, Nos1) and neurotransmitter (Bdnf).","We obtained similar transfection yield in PC12 (80%) cells as compare to HL60 cells, as shown in .  displays the basal expression level of the selected pain genes normalized upon Gapdh expression level, with and without SEQ ID NO.: 42 decoy treatment. The results indicate that the basal expression of Bdkrb2, Htr3a and Scn9a is strongly inhibited by SEQ ID NO.: 42 treatment. Interestingly, all three of the genes encode membrane proteins\u2014two receptors and one ion channel. The absence of impact on the other genes expression level emphasizes the specificity of the EGR1 decoy treatment in PC12 cells. In further experiments, we treated PC12 cells with two pain mimicking stimuli that are known to mobilize EGR1\u2014NGF and forskolin. NGF induces the expression of EGR1 and forskolin acts as a permissive factor for EGR1 activity in PC12 cells. Twenty-four hours after NGF\/forskolin treatment of PC12 cells, we observed significant up-regulation of 7 of the 11 genes examined, including Bdnf, Grm5, Scn9a, Nos1, Gch1, Cdk5r1 and Pnmt. Our results are in agreement with several other studies showing the up-regulation of such pain genes in PC12 cells following NGF exposure. Treatment with SEQ ID NO.: 42 fully prevented the endogenous up-regulation of five of the genes, including Scn9a, Nos1, Gch1, Cdk5r1 and Pnmt. Interestingly, all of these genes except Scn9a encode enzymes-related proteins. Typical pictures of pain gene cDNA detection on agarose gels illustrating the two complementary effect of SEQ ID NO.: 42 treatment, basal expression inhibition and up-regulation block, are shown in . We verified the lack of non-specific oligonucleotides exposure effect in PC12 cells by showing the absence inhibition by the mismatch sequence SEQ ID NO.: 43\/46 on two pain genes, as shown in .","SEQ ID NO.: 42 inhibits the expression level of seven out of eleven pain genes on two different levels, basal transcription and pain-induced up-regulation. It is possible that the two effects operate on distinct classes of genes, as within our small scale experiments, basal transcription levels were inhibited among essentially only membrane proteins while under pain-like conditions the normal up-regulation of pain-associated genes was inhibited among essentially only genes encoding enzymes. The high proportion of genes regulated and their complementary qualities reflects the importance of EGR1 in pain and is in agreement with animal knockout and antisense studies demonstrating that in absence of EGR1, major pain syndromes are not maintained. From a therapeutic prospective, the interest of inhibiting EGR1 activity using SEQ ID NO.: 42 is the ability to concurrently modulate the expression of a high number of pain genes that are active at multiple steps of pain signaling pathways. For instance, a unique treatment with SEQ ID NO.: 42 would be sufficient to concurrently inhibit a receptor like BDRKD2 that perceive pain signals, an ion channel like SCN9A that relays pain signals within neurons, and a neurotransmitter synthesis enzyme like GCH1 that participate to its synaptic transmission between neurons, whereas normally a complex polypharmacy approach would be necessary to simultaneously affect such different targets. Altogether, the experimental data showing the strong inhibitory effect of SEQ ID NO.: 42 on EGR1-dependent pain gene expression reveals its therapeutic potential for pain treatment.","We analyzed several other oligonucleotide decoys sequences that target transcription factors with distinct roles, including (i) CREB\/ATF and NFAT, which are immediate early genes that are critical in pain gene expression plasticity and complement the role of EGR1, and (ii) AML1 and SP1 factors, which are critical in the maintenance of basal expression and tissue specific expression of numerous pain genes.",{"@attributes":{"id":"p-0202","num":"0251"},"figref":"FIG. 6A"},"The therapeutic potential of three of the sequences was assessed in PC12 cells, as described for SEQ ID NO.: 42. The presence of CREB\/ATF, NFAT and RUNX factors has been previously described in PC12 cells. Expression levels of pain genes before and after SEQ ID NO.: 4, SEQ ID NO.: 12, and SEQ ID NO.: 15 decoy treatment are shown in Table 2A.  illustrates the effect of oligonucleotide decoy treatment measured under pain-like conditions. Each sequence inhibited the expression of multiple genes under both basal and pain-like conditions. See Tables 2A and 2B, . For example, SEQ ID NO.: 4, which targets CREB\/ATF transcription factors, inhibited the basal expression level of Bdkrb2, Grm5, Htr3a, Pnmt and Nos1 and prevents the up-regulation of Scn9a, Cdk5r1, Pnmt and Nos1. We observed some overlap in the inhibition profiles of decoy sequences over the regulation of pain genes expression. Such redundancy is not surprising in the light of gene expression being controlled by scaffolds of transcription factors rather than by a single one and that all investigated factors are involved in pain signaling. In vivo, the respective involvement of each of transcription factor in the regulation of genes expression may depend on the type of pain neuron it is expressed in and in its global activity resulting from the integration of complex pain signaling pathways. Therefore, the therapeutic relevance of a particular decoy may depend on the pain syndrome, intensity and stage.","Some important pain genes like Scn9a, which is critical in the genesis of action potential in pain neurons (e.g. nonsense mutations in Scn9a generate insensitivity to pain), are very sensitive to transcription regulation. Scn9a up-regulation after NGF and forskolin treatment appears to implicate a transcriptional network that includes the three immediate early genes Egr1, Creb\/Atf and Nfat. If the activity of a single one of those factors is inhibited with one of our decoy sequences, the regulation is lost. This represents an important potential therapeutic advantage to the decoy approach as the expression of a given gene may be inhibited without the need to target all the transcription factors involved in its regulation.","Altogether, those experiments demonstrate that our decoy sequences have the potential to concurrently inhibit a high number of pain genes, a unique property for pain therapy.","Considering that a certain level of redundancy operates between transcription factor activities, we developed a composite decoy sequence, SEQ ID NO.: 45, for the concurrent inhibition of EGR1, CREB\/ATF and NFAT. The interest of such a sequence is the simultaneous inhibition of three major immediate early genes involved in neuronal plasticity and that integrate complementary signaling pathways critical for pain sensation. Signaling kinases like the MAPK\/ERK pathways, which are activated by numerous metabotropic pain receptors (e.g., the NGF receptors NTRK1\/NGFR), mobilize EGR1, while the calcium signaling pathways mobilized by calcium- and cationic-channels activate CREB and NFAT. The sequence of SEQ ID NO.: 45 includes, in 5\u2032 to 3\u2032 order, transcription factor binding sites for EGR1, CREB\/ATF and NFAT, each selected from the individual response elements of SEQ ID NO.: 3 (EGR1), SEQ ID NO.: 4 (CREB\/ATF), and SEQ ID NO.: 15 (NFAT).","The binding properties of SEQ ID NO.:45 for each factor are displayed FIG. A,B. Parallel ELISA competition experiments with SEQ ID NO.: 41 and SEQ ID NO.: 45 show that the relative binding affinity of this composite sequence for EGR1 is as high as the oligonucleotide decoy SEQ ID NO.: 41. Furthermore, the inhibition of hEGR1 activity in HL60 cells induced by SEQ ID NO.: 45 treatment matches the inhibition induced by SEQ ID NO.: 41 (), both having overlapping dose-response curves. These results are consistent with our prior observation that cell efficiency is directly linked to the relative affinity measured in ELISA experiments. Finally, further ELISA competition experiments show that, in plus of binding to hEGR1, SEQ ID NO.: 45 also specifically binds to hCREB\/hATF and hNFAT factors ().","We investigated the impact of SEQ ID NO.: 45 on pain gene expression in PC12 cells (table 2). Use of a composite sequence provides two benefits: (i) a potentially additive effect on the number and type of genes inhibited; and (ii) potentially greater inhibition of particular genes that are only partially inhibited by oligonucleotide decoys specific to single transcription factors. The additive effect was illustrated for SEQ ID NO.:45 by the differential inhibition among the composite, NFAT, and EGR1 decoys. For example, SEQ ID NO.: 42 does not inhibit Grm5 basal expression, while both SEQ ID NO.: 15 (NFAT) and SEQ ID NO.: 45 (composite) do. Similarly, in pain-like condition, SEQ ID NO.: 15 (NFAT) does not prevent Scn9a up-regulation after NGF and forskolin treatment, while SEQ ID NO.: 42 (EGR1) and SEQ ID NO.: 45 (composite) do.","The intensity effect appears strongly in the regulation of Bdrkb2 and Scn9a genes expression, as shown in . When one gene is inhibited by at least 2 transcription factors targeted by the composite sequence, the intensity of the inhibition is stronger than the inhibition conferred by the individual sequences. For instance, Bdrkb2 basal expression is individually inhibited by a factor of 5 by both SEQ ID NO.: 4 and SEQ ID NO.: 15, while it is inhibited by a factor of 10 by the composite oligonucleotide decoy, SEQ ID NO.: 45.",{"@attributes":{"id":"p-0210","num":"0259"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"301pt","align":"center"}},"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"Table 2A-Pain Genes Basal Expression"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"BDNF","NOS1","BDKRB2","P2RX3","Grm5","HTR3A"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"13"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"10","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"11","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"12","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"13","colwidth":"21pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Mean","SEM","Mean","SEM","Mean","SEM","Mean","SEM","Mean","SEM","Mean","SEM"]},{"entry":{"@attributes":{"namest":"1","nameend":"13","align":"center","rowsep":"1"}}},{"entry":["control","0.7","0.29","0.13","0.02","1.16","0.09","0.94","0.32","0.15","0.05","0.6","0.1"]},{"entry":["+ ID No. 4","0.54","0.14","0.06","0.01","0.28","0.02","0.72","0.39","0.04","0.01","0.01","0.00"]},{"entry":["+ ID No. 12","0.92","0.05","0.04","0.003","0.38","0.07","0.66","0.18","0.15","0.11","0.06","0.04"]},{"entry":["+ ID No. 15","0.63","0.04","0.12","0.05","0.22","0.03","0.59","0.20","0.03","0.01","0.02","0.01"]},{"entry":["+ ID No. 42","0.89","0.26","0.08","0.03","0.51","0.24","0.80","0.27","0.20","0.19","0.04","0.01"]},{"entry":["+ ID No. 45","0.626","0.08","0.021","0.005","0.05","0.002","1.038","0.21","0.048","0.01","0.06","0.02"]},{"entry":{"@attributes":{"namest":"1","nameend":"13","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"CACNA1b","SCN9Q","GCH1","CDKR1","PNMT"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"12"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"10","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"11","colwidth":"21pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Mean","SEM","Mean","SEM","Mean","SEM","Mean","SEM","Mean","SEM"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"11","align":"center","rowsep":"1"}}]},{"entry":[{},"control","1.08","0.21","0.33","0.06","0.76","0.10","0.68","0.11","0.51","0.10"]},{"entry":[{},"+ ID No. 4","1.02","0.15","0.17","0.15","1.07","0.30","0.98","0.12","0.05","0.01"]},{"entry":[{},"+ ID No. 12","0.84","0.19","0.04","0.01","0.98","0.22","0.29","0.02","0.06","0.02"]},{"entry":[{},"+ ID No. 15","0.30","0.13","0.11","0.002","1.03","0.05","0.98","0.40","0.08","0.01"]},{"entry":[{},"+ ID No. 42","1.03","0.29","0.11","0.01","1.04","0.08","0.86","0.23","0.41","0.17"]},{"entry":[{},"+ ID No. 45","0.826","0.06","0.045","0.02","1.025","0.14","0.578","0.06","0.07","0.02"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"11","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"301pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"Table 2B-NGF and Forskolin Stimulation"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"BDNF","NOS1","BDKRB2","P2RX3","Grm5","HTR3A"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"13"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"10","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"11","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"12","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"13","colwidth":"21pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Mean","SEM","Mean","SEM","Mean","SEM","Mean","SEM","Mean","SEM","Mean","SEM"]},{"entry":{"@attributes":{"namest":"1","nameend":"13","align":"center","rowsep":"1"}}},{"entry":["+ NGF + FSK","1.52","0.33","0.38","0.06","1.22","0.17","1.00","0.23","1.07","0.18","0.83","0.09"]},{"entry":["+ ID No: 4","1.22","0.20","0.19","0.03","0.80","0.22","0.89","0.34","0.98","0.11","0.53","0.13"]},{"entry":["+ ID No: 12","0.79","0.06","0.35","0.15","0.67","0.25","0.92","0.51","1.13","0.23","0.46","0.04"]},{"entry":["+ ID No: 15","1.55","0.52","0.10","0.01","1.10","0.19","1.32","0.31","0.81","0.17","0.28","0.09"]},{"entry":["+ ID No: 42","1.33","0.03","0.11","0.06","1.49","0.13","0.79","0.42","0.83","0.34","0.37","0.18"]},{"entry":["+ ID No: 45",{},{},"0.04","0.01","1.27","0.38","0.83","0.21","0.63","0.32","0.87","0.18"]},{"entry":{"@attributes":{"namest":"1","nameend":"13","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"CACNA1b","SCN9A","GCH1","CDKR1","PNMT"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"12"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"10","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"11","colwidth":"21pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Mean","SEM","Mean","SEM","Mean","SEM","Mean","SEM","Mean","SEM"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"11","align":"center","rowsep":"1"}}]},{"entry":[{},"+ NGF + FSK","0.91","0.14","0.87","0.11","1.81","0.35","1.25","0.26","1.34","0.18"]},{"entry":[{},"+ ID No: 4","0.80","0.13","0.19","0.06","2.18","0.69","0.65","0.27","0.09","0.05"]},{"entry":[{},"+ ID No: 12","1.49","0.40","0.38","0.16","1.59","0.25","0.84","0.18","0.26","0.09"]},{"entry":[{},"+ ID No: 15","1.04","0.14","1.00","0.28","1.95","0.21","0.80","0.05","0.17","0.06"]},{"entry":[{},"+ ID No: 42","1.36","0.46","0.28","0.09","0.99","0.12","0.66","0.20","0.58","0.18"]},{"entry":[{},"+ ID No: 45","1.63","0.40","0.10","0.01","0.68","0.32",{},{},"0.19","0.09"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"11","align":"center","rowsep":"1"}}]}]}}]}}},"Values are given as Mean and SEM, and represent the expression level in PC12 cells of each gene normalized on the Gapdh expression level. Units are arbitrary. Black cases represent experiments not done. n=2-4.","Inflammation is a major source of pain. It is a feature common to numerous pain syndromes, such as arthritic and post-operative pain. The Complete Freund Adjuvant model (CFA) is a well-characterized inflammatory pain model that is commonly used to reproduces features of human inflammatory pain. For instance, following inflammation in the hindpaw, animals develop a robust and long-lasting mechanical allodynia (i.e., a pain in response to a mechanical stimulus normally non-painful), a phenomenon that is a major source of pain and limitations for patients ambulation, breathing and feeding in a post-operative context.","In our experiments and accordingly to the literature, mechanical allodynia was measurable on the inflamed hindpaw at day 1 post-CFA and reached its maximum within 4 days, as shown in . Treatment with SEQ ID NO.: 42 resulted in an anti-allodynic tendency at day 1 post-CFA () and a robust reversal of allodynia at day 4 post-CFA for each stimulus force tested (). This is in agreement with EGR1 being involved in the maintenance of neuronal plasticity events, like neuronal sensitization and long-term potentiation, rather than their onset.","Altogether, those results indicate that SEQ ID NO.: 42 treatment has a robust anti-allodynic effect, demonstrating its therapeutic potential for treating pain in vivo. Particularly, SEQ ID NO.: 42 treatment is relevant in preventing the maintenance of long-lasting pain syndromes, e.g. chronic post-operative pain.","Cell Culture and Biological Reagents","HL60 (human peripheral blood, acute promyelocytic leukemia) and PC12 (rat adrenal gland, pheochromocytomal cells) cell lines were purchased from the UCSF Cell culture facility (CA, USA). HL-60 cells were grown in RPMI media 1640+L-Glutamine (Invitrogen, CA, USA) supplemented with 10% heat-inactivated fetal bovine serum and 1% penicillin-streptomycin (Invitrogen, CA, USA). Cells were splits into 6-well plates (BD Biosciences, USA) at about 200\u00d710cells\/well 24h before treatment with 1 \u03bcM 1,25-Dihydroxyvitamin D3 with or without decoy transfection as described previously. PC12 cells were grown in DMEM containing 1,000 mg\/L D-glucose, L-glutamine, 25 mM HEPES buffer, and 110 mg\/L sodium pyruvate (Invitrogen, CA, USA) and supplemented with 10% heat-inactivated fetal bovine serum, 5% heat-inactivated horse serum and 1% penicillin-streptomycin (Invitrogen, CA, USA). PC12 cells were split into CellBind 6-well plates (Corning, USA) 24 h before treatment with 100 nM NGF (Invitrogen, CA, USA) and 5 \u03bcM forskolin (Sigma-Aldrich, Mo., USA) with or without decoy transfection. All cells were grown at 37\u00b0 C. with 5% CO2. Dead cells counting were realized using Tryptan blue (Invitrogen, CA, USA) exclusion technique on a Malassez counting chamber.","Decoy Sequences Annealing","Forward and reverse strands for each decoy sequence were synthesized by Integrated DNA Technology (IA, USA) and resuspended in either 1\u00d7TE buffer, pH 7.4 or pH 8. Each strand pair was annealed in presence of 50 mM NaCl with a 7 min 95\u00b0 C. denaturation step and a slow cooling to 25\u00b0 C. at 0.5\u00b0 C.\/min. Annealing success was checked on a 2.5% agarose gel with ethidium bromide by observing the slower migration speeds of the duplexes versus a corresponding single strand.","Decoy Sequences Transfection","Transfections of decoy sequences were realized using Oligofectamine (Invitrogen, CA, USA) according to the manufacturer protocol. For HL60 experiments, decoy sequences transfections (250 nM, 500 nM, 1000 nM and 2000 nM) were immediately followed by 1,25-Dihydroxyvitamin D3 (1 \u03bcM) treatment. Cells were collected 48 h later and prepared for RNA extraction. For PC12 cells, NGF (100 ng\/ml) and forskolin (5 \u03bcM) were applied immediately after decoy sequences transfections (500 nM). Cells were collected 24 h after for RNA extraction.","For both cell lines, transfection yield was measured using SEQ ID NO.: 40 coupled to fluorescein (Integrated DNA Technology, IA, USA) 24 h post-transfection. The yield of transfection was calculated based upon the counting of fluorescent versus non-fluorescent cells observed under a fluorescent microscope.","Semi-quantitative Reverse Transcription and Polymerase Chain Reactions (sqRT-PCR)","Total RNA was extracted from cells using the RNeasy Plus kit (Qiagen, USA) that ensures removal of genomic DNA during RNA extraction. Equivalent RNA quantities are reverse transcripted into cDNA per condition, using either the First-strand cDNA synthesis kit (GE healthcare, NJ, USA) or the Superscript 1strand system (Invitrogen, CA, USA) and one-sixteenth of each RT was used per PCR reaction. PCR were realized in 20 \u03bcL total using the Promega master mix (Promega, WI, USA) with the following cycles: 95\u00b0 C. 1 min, 55\u00b0 C. 1 min, 72\u00b0 C. 1 min (25 cycles for housekeeping genes ACTB and Gapdh, 35 cycles for other genes for material detection in the linear detection range and before signal saturation). All primers used (see Table 3) have been previously described.",{"@attributes":{"id":"p-0220","num":"0269"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"70pt","align":"left"}}],"thead":{"row":[{"entry":"TABLE\u20033"},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Primer","Sequence\u20035\u2032-3\u2032","SEQ\u2003ID\u2003NO."]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["hACTB\u2003S","AAGAGAGGCATCCTCACCCT","SEQ\u2003ID\u2003NO.:\u200354"]},{"entry":{}},{"entry":["hACTB\u2003AS","TACATGGCTGGGGTGTTGAA","SEQ\u2003ID\u2003NO.:\u200355"]},{"entry":{}},{"entry":["hBCL2\u2003S","GGAAGTGAACATTTCGGTGAC","SEQ\u2003ID\u2003NO.:\u200356"]},{"entry":{}},{"entry":["hBCL2\u2003AS","GCCTCTCCTCACGTTCCC","SEQ\u2003ID\u2003NO.:\u200357"]},{"entry":{}},{"entry":["hCDK5R1\u2003S","GCCGTACAGAACAGCAAGAA","SEQ\u2003ID\u2003NO.:\u200358"]},{"entry":{}},{"entry":["hCDK5R1\u2003AS","GTCGGCATTTATCTGCAGCA","SEQ\u2003ID\u2003NO.:\u200359"]},{"entry":{}},{"entry":["rBdkrb2\u2003S","GAACATCTTTGTCCTCAGC","SEQ\u2003ID\u2003NO.:\u200360"]},{"entry":{}},{"entry":["rBdkrb2\u2003AS","CCGTCTGGACCTCCTTGAAC","SEQ\u2003ID\u2003NO.:\u200361"]},{"entry":{}},{"entry":["rBdnf\u2003S","GGCTTTGATGAGACCGGGTTC","SEQ\u2003ID\u2003NO.:\u200362"]},{"entry":[{},"CCT",{}]},{"entry":{}},{"entry":["rBdnf\u2003AS","GTAGGCCAAGTTGCCTTGTC","SEQ\u2003ID\u2003NO.:\u200363"]},{"entry":[{},"CGT",{}]},{"entry":{}},{"entry":["rCacna1b\u2003S","ATGCTGTTCTTCATCTACGC","SEQ\u2003ID\u2003NO.:\u200364"]},{"entry":{}},{"entry":["rCacna1b\u2003AS","TTGTCCATGATCACAGCAAC","SEQ\u2003ID\u2003NO.:\u200365"]},{"entry":{}},{"entry":["rEgr1\u2003S","AGATGATGCTGCTGAGCAAC","SEQ\u2003ID\u2003NO.:\u200366"]},{"entry":{}},{"entry":["rEgr1\u2003AS","AGTAAATGGGACTGCTGTCG","SEQ\u2003ID\u2003NO.:\u200367"]},{"entry":{}},{"entry":["rGapdh\u2003S","CCGCTGATGCCCCCATGTTTG","SEQ\u2003ID\u2003NO.:\u200368"]},{"entry":[{},"TGAT",{}]},{"entry":{}},{"entry":["rGapdh\u2003AS","GGCATGTCAGATCCACAACGG","SEQ\u2003ID\u2003NO.:\u200369"]},{"entry":[{},"ATAC",{}]},{"entry":{}},{"entry":["rGch1\u2003S","CCACGCCATGCAGTTCTTCA","SEQ\u2003ID\u2003NO.:\u200370"]},{"entry":[{},"CCA",{}]},{"entry":{}},{"entry":["rGch1\u2003AS","AGGCTGCAAGGCTTCTGTGAT","SEQ\u2003ID\u2003NO.:\u200371"]},{"entry":[{},"GGC",{}]},{"entry":{}},{"entry":["rGrm5\u2003S","GTGGCGGAGGCAGAGGAGAGC","SEQ\u2003ID\u2003NO.:\u200372"]},{"entry":{}},{"entry":["rGrm5\u2003AS","GTGGCCGCGGTGGACAACAT","SEQ\u2003ID\u2003NO.:\u200373"]},{"entry":{}},{"entry":["rHtr3a\u2003S","AATCAGGGCGAGTGGGAGC","SEQ\u2003ID\u2003NO.:\u200374"]},{"entry":{}},{"entry":["rHtr3a\u2003AS","GAGGACAGCTCTTGCAAGA","SEQ\u2003ID\u2003NO.:\u200375"]},{"entry":[{},"GGC",{}]},{"entry":{}},{"entry":["rNos1\u2003S","GAATACCAGCCTGATCCATGG","SEQ\u2003ID\u2003NO.:\u200376"]},{"entry":[{},"AAC",{}]},{"entry":{}},{"entry":["rNos1\u2003AS","TCCTCCAGGAGGGTGTCCACC","SEQ\u2003ID\u2003NO.:\u200377"]},{"entry":[{},"GCA",{}]},{"entry":{}},{"entry":["rP2rx3\u2003S","TGGCGTTCTGGGTATTAAGAT","SEQ\u2003ID\u2003NO.:\u200378"]},{"entry":[{},"CGG",{}]},{"entry":{}},{"entry":["rP2rx3\u2003AS","CAGTGGCCTGGTCACTGGCGA","SEQ\u2003ID\u2003NO.:\u200379"]},{"entry":{}},{"entry":["rCdk5r1\u2003S","GCTCTGCAGGGATGTTATC","SEQ\u2003ID\u2003NO.:\u200380"]},{"entry":[{},"TCC",{}]},{"entry":{}},{"entry":["rCdk5r1\u2003AS","CTTCTTGTCCTCCTGACCA","SEQ\u2003ID\u2003NO.:\u200381"]},{"entry":[{},"CTC",{}]},{"entry":{}},{"entry":["rPnmt\u2003S","CAGACTTCTTGGAGGTCAAC","SEQ\u2003ID\u2003NO.:\u200382"]},{"entry":[{},"CTG",{}]},{"entry":{}},{"entry":["rPnmt\u2003AS","TTATTAGGTGCCACTTCGG","SEQ\u2003ID\u2003NO.:\u200383"]},{"entry":[{},"GTG",{}]},{"entry":{}},{"entry":["rScn9a\u2003S","TTCATGACCTTGAGCAACCC","SEQ\u2003ID\u2003NO.:\u200384"]},{"entry":{}},{"entry":["rScn9a\u2003AS","TCTCTTCGAGTTCCTTCCTG","SEQ\u2003ID\u2003NO.:\u200385"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":"S = sense,"},{"entry":"AS = anti-sense,"},{"entry":"h = human,"},{"entry":"r = rat."}]}}}}},"12.5 \u03bcl of each PCR reaction was detected on 1% agarose gel (Invitrogen, CA, USA) with ethidium bromide (Fisher Scientific, PA, USA). Gel bands images were captured with a Fluor Chem SP gel imager system (Alpha Innotech, CA, USA) and analyzed using the Image J software (NIH, MD, USA). Expression levels were normalized on ACTB levels for HL60 experiments and Gapdh levels for PC12 experiments. Statistical significance was measured with the two-tails student t-test. Dose-responses curves were fitted with the exponential decay equation.","Transcription Factor ELISA Experiments","The affinity and specificity of decoy sequences for their transcription factor targets was measured with colorimetric transcription factor ELISA (Enzyme linked immuno adsorbent) kits (Panomics, CA, USA). Briefly, designated decoy sequences coupled to biotin were incubated for 30 minutes with nuclear protein extracts from TPA-stimulated K-562 cells expressing targeted transcription factors (Activemotif, CA, USA). The mixes of proteins and decoy sequences were loaded on 96-well plates coated with streptavidin provided in the kit. The quantity of transcription factor captured by each decoy sequence was revealed according to the supplier protocol using specific primary antibodies and secondary antibodies coupled to the horseradish peroxidase (HRP) enzyme. Reactions optical densities (ODs) were read at 450 nM with a Thermomax microplate reader (Molecular Device, CA, USA).","Experiments were conducted in 50 \u03bcl with 6.4 pmoles of biotin-coupled decoy sequence (probe) mixed with 10 \u03bcg of nuclear protein extract in the kit binding buffer. When the probe is incubated alone with the protein extracts, the resulting OD represents the binding activity of the probe for its target. When increasing concentration of competing, not biotinylated versions of the probe are added to the binding reaction, a reduction of OD values demonstrates binding specificity. The use of sequence variants as competitors allows measuring their relative affinities for the targeted factor as compared to the probe. The use of primary antibodies against several transcription factors (CREB\/ATF, WT1, NFATC1 from Santa Cruz Biotechnolgy, CA, USA, SP1 from emd biosciences, WI, USA and EGR1 from Panomics, CA, USA) allows detecting the relative specificity of decoy sequences for multiple factors. Competition curves were fitted with the exponential decay equation.","Behavioural Experiments","The plantar surface of left hind paw of Sprague-Dawley Rats (male, 250-300 g) was injected (30 G needle) with 150 \u03bcl of Complete Freund Adjuvant (CFA). Von Frey filaments of 1 g and 6 g were used to test for mechanical responsiveness (i.e., allodynia) of the hind paw. Briefly, each Von Frey filament was applied 5 times and the number of paw withdrawals was counted. Animals were habituated on a mesh floor 1 hour prior to testing. Basal mechanical sensitivity of animals was tested before SEQ ID NO.: 42 and CFA treatments. All experiments were conducted blinded.","SEQ ID NO.:42 was synthesized and HPLC-purified by Integrated DNA Technology (IA, USA). Decoy duplexes were annealed as described previously, in TE pH 8 at a 2 mM final concentration and injected intrathecally in rats with 13 nmoles\/injection (20 \u03bcl total, diluted 1:3, TE pH 8). The injection\/testing schedule was as follows:\n\n","Control animals are injected with only TE as a vehicle following the same schedule. For intrathecal injections, rats were anesthetized with 2% Isoflurane, their backs shaved and prepared with Betadine. Rats were then was placed on a bottle to keep the back arched. A 17 G \u00bd needle was slid rostrally along left side of L6 transverse process till it reached L5. The needle was then inserted between L5 and L6 until the intrathecal space was reached as indicated by tail twitch.","It will be apparent to those skilled in the art that many modifications, both to materials and methods, may be practiced without departing from the scope of this disclosure. Accordingly, the present embodiments are to be considered as illustrative and not restrictive, and the invention is not to be limited to the details given herein, but may be modified within the scope and equivalents of the appended claims.","All publications and patents cited herein are incorporated by reference in their entirety."],"BRFSUM":[{},{}],"heading":["DESCRIPTION OF THE TEXT FILE SUBMITTED ELECTRONICALLY","TECHNICAL FIELD","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","EXAMPLES","Example 1","Example 2","Affinity and Specificity of EGR1 Oligonucleotide Decoy Sequences","Example 3","Inhibition of hEGR1 Transcriptional Activity in Cells","Example 4","Inhibition of Pain Genes Expression","Example 5","Complementary Decoy Studies","Example 6","Composite Oligonucleotide Decoys","Example 7","Treatment of Pain In Vivo","Example 8","Materials and Methods"],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0020","num":"0019"},"figref":"FIG. 1","sub":["450 nm ","450 nm "]},{"@attributes":{"id":"p-0021","num":"0020"},"figref":"FIG. 2","sub":"50 "},{"@attributes":{"id":"p-0022","num":"0021"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0023","num":"0022"},"figref":"FIG. 4","sub":"50"},{"@attributes":{"id":"p-0024","num":"0023"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0025","num":"0024"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0026","num":"0025"},"figref":"FIG. 7","sub":"9"},{"@attributes":{"id":"p-0027","num":"0026"},"figref":"FIG. 8"}]},"DETDESC":[{},{}]}
