---
title: Polypeptides and polynucleotides, and uses thereof as a drug target for producing drugs and biologics
abstract: This invention relates to a novel target for production of immune and non-immune based therapeutics and for disease diagnosis. More particularly, the invention provides therapeutic antibodies against VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigens, which are predicted co-stimulatory family members and which are differentially expressed in cancers including, lung cancer, ovarian cancer, and colon cancer, and diagnostic and therapeutic usages. The use of these antibodies for modulating B7 costimulation and related therapies such as the treatment of autoimmunity are also provided. This invention further relates to the discovery of extracellular domains of VSIG1 and its variants, FXYD3 and its variants, ILDR1 and its variants, LOC253012 and its variants, AI216611 and its variants, and C1ORF32 and its variants which are suitable targets for immunotherapy, cancer therapy, and drug development.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08415455&OS=08415455&RS=08415455
owner: Compugen Ltd
number: 08415455
owner_city: Tel Aviv
owner_country: IL
publication_date: 20080903
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATIONS","REFERENCE TO SEQUENCE LISTING","FIELD OF THE INVENTION","BACKGROUND OF THE INVENTION","BRIEF SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","EXAMPLES","Example 1","Methods Used to Analyze the Expression of the RNA Encoding the Proteins of the Invention","Example 2","Description for Cluster AI581519","Example 3","Description for Cluster AA424839","Example 4","Description for Cluster AI216611","Example 5","Description for Cluster H686541","Example 6","Description for Cluster H190111","Example 7","Description for Cluster R31375","Example 8","Cloning of Full Length Transcripts Encoding VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 Fused to EGFP","Example 9","Determining Cell Localization of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 and FXYD3","Example 10","Cloning and Expression of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 and FXYD3 Extra Cellular Domain (ECD) Fused to Mouse Fc","Example 11","Protein Production of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 and FXYD3 Extra Cellular Domain (ECD) Fused to Mouse Fc","Example 12","Binding of the ECDs Fc-Fused Proteins of the Invention to Activated T Cells","Example 13","Binding of the ECDs Fc-Fused Proteins of the Invention to Lymphocytes and to CD4 Positive Cells","Example 14","Effect of the ECDs Fc-Fused Proteins of the Invention on T Cell Activation","Example 15","Interaction of the ECDs-Fc Fused Proteins of the Invention with Resting B Cells, Activated B Cells, and B Cell Derived Lymphoma Cell Lines","Example 16","Interaction of the ECD-Fc Fused Proteins of the Invention with Known B7","Example 17","Development of Mouse Monoclonal Anti-VSIG1, Anti-ILDR1, Anti-LOC253012, Anti-AI216611, Anti-C1ORF32 and Anti-FXYD3 Antibodies","Example 17","Development of Fully Human Anti-VSIG1, Anti-ILDR1, Anti-LOC253012, Anti-AI216611, Anti-C1ORF32 and Anti-FXYD3 Antibodies"],"p":["This Application is a national phase of, and claims priority from, PCT Application No. PCT\/US2008\/075122, filed on Sep. 3, 2008, which claims priority from U.S. Provisional Application Nos. 60\/969,865; 60\/969,799; 60\/969,780; 60\/969,806; 60\/969,769; 60\/969,788; all of which were filed on Sep. 4, 2007, and all of which are hereby incorporated by reference as if fully set forth herein.","The Sequence Listing is filed herewith electronically as a separate ASCII file entitled \u201c136PCTPatentInsequencelisting040808.txt\u201d, created on Sep. 3, 2008, having 427000 bytes in size; the contents of which are hereby incorporated by reference as if fully set forth herein.","This invention relates to the discovery of certain proteins that are differentially expressed in specific tissues and their use as therapeutic and diagnostic targets. More specifically the invention relates to a protein VSIG1 and its variants, FXYD3 and its variants, ILDR1 and its variants, LOC253012 and its variants, AI216611 and its variants, and C1ORF32 and its variants, which are differentially expressed by some cancers, and therefore are suitable targets for immunotherapy, cancer therapy, and drug development. This invention further relates to the discovery of extracellular domains of VSIG1 and its variants, FXYD3 and its variants, ILDR1 and its variants, LOC253012 and its variants, AI216611 and its variants, and C1ORF32 and its variants which are suitable targets for immunotherapy, cancer therapy, and drug development","Additionally, because some of the proteins of this invention, based on their B7-like structure, are believed to play a role in immune costimulation, the invention further relates to the use of these proteins, or drugs which modulate these proteins (agonistic and antagonistic), as immune modulators and for immune therapy, especially for treating cancer and immune related disorders such as cancers and autoimmune disorders. Also, the invention more specifically relates to therapeutic and diagnostic antibodies and therapies and diagnostic methods using same antibodies and antibody fragments that specifically bind to proteins of invention or a soluble or secreted portion thereof, especially the ectodomain.","Tumor antigens are ideally positioned as biomarkers and drug targets, and they play a critical role in the development of novel strategies for active and passive immunotherapy agents, to be used as stand-alone therapies or in conjunction with conventional therapies for cancer. Tumor antigens can be classified as either tumor-specific antigens (TSAs) where the antigens are expressed only in tumor cells and not in normal tissues, or tumor-associated antigens (TAAs) where the antigens are overexpressed in tumor cells but nonetheless also present at low levels in normal tissues.","TAAs and TSAs are validated as targets for passive (antibody) therapy as well as active immunotherapy using strategies to break immune tolerance and stimulate the immune system. The antigenic epitopes that are targeted by these therapeutic approaches are present at the cell surface, overexpressed in tumor cells compared to non-tumor cells, and are targeted by antibodies that block functional activity, inhibit cell proliferation, or induce cell death.","There are growing number of tumor-associated antigens against which monoclonal antibodies have been tested or are in use as treatment for cancer. The identification and molecular characterization of novel tumor antigens expressed by human malignancies is an active field in tumor immunology. Several approaches have been used to identify tumor-associated antigens as target candidates for immunotherapy, including high throughput bioinformatic approaches, based on genomics and proteomics. The identification of novel TAAs or TSAs expands the spectrum of tumor antigen targets available for immune recognition and provides new target molecules for the development of therapeutic agents for passive immunotherapy, including monoclonal antibodies, whether unmodified or armed. Such novel antigens may also point the way to more effective therapeutic vaccines for active or adoptive immunotherapy.","Cancer vaccination involves the administration of tumor antigens and is used to break immune tolerance and induce an active T-cell response to the tumor. Vaccine therapy includes the use of naked DNA, peptides, recombinant protein, and whole cell therapy, where the patient's own tumor cells are used as the source of the vaccine. With the identification of specific tumor antigens, vaccinations are more often carried out by dendritic cell therapy, whereby dendritic cells are loaded with the relevant protein or peptide, or transfected with vector DNA or RNA.","The major applications of anti-TAA antibodies for treatment of cancer are therapy with naked antibody, therapy with a drug-conjugated antibody, and fusion therapy with cellular immunity. Ever since their discovery, antibodies were envisioned as \u201cmagic bullets\u201d that would deliver toxic agents, such as drugs, toxins, enzymes and radioisotopes, specifically to the diseased site and leaving the non-target normal tissues unaffected. Indeed, antibodies, and in particular antibody fragments, can function as carriers of cytotoxic substances such as radioisotopes, drugs and toxins. Immunotherapy with such immunoconjugates is more effective than with the naked antibody.","In contrast to the overwhelming success of naked (such as Rituxan and Campath) and conjugated antibodies (such as Bexxar and Zevalin) in treating hematological malignancies, only modest success has been achieved in the immunotherapy of solid tumors. One of the major limitations in successful application of immunotherapy to solid tumors is the large molecular size of the intact immunoglobulin that results in prolonged serum half-life but in poor tumor penetration and uptake. Indeed, only a very small amount of administered antibody (as low as 0.01%) reaches the tumor. In addition to their size, antibodies encounter other impediments before reaching their target antigens expressed on the cell surface of solid tumors. Some of the barriers include poor blood flow in large tumors, permeability of vascular endothelium, elevated interstitial fluid pressure of tumor stroma, and heterogenous antigen expression.","With the advent of antibody engineering, small molecular weight antibody fragments exhibiting improved tumor penetration have been generated. Such antibody fragments are often conjugated to specific cytotoxic molecules and are designed to selectively deliver them to cancer cells. Still, solid tumors remain a formidable challenge for therapy, even with immunoconjugated antibody fragments.","The new wave of optimization strategies involves the use of biological modifiers to modulate the impediments posed by solid tumors. Thus, in combination to antibodies or their conjugated antibody fragments, various agents are being used to improve the tumor blood flow, enhance vascular permeability, lower tumor interstitial fluid pressure by modulating stromal cells and extracellular matrix components, upregulate expression of target antigens and improve penetration and retention of the therapeutic agent.","Immunotherapy with antibodies represents an exciting opportunity for combining with standard modalities, such as chemotherapy, as well as combinations with diverse biological agents to obtain a synergistic activity. Indeed, unconjugated mAbs are more effective when used in combination with other therapeutic agents, including other antibodies.","Another component of the immune system response to immunotherapy is the cellular response, specifically\u2014the T cell response and activation of cytotoxic T cells (CTLs). The efficiency of the immune system in mediating tumor regression depends on the induction of antigen-specific T-cell responses through physiologic immune surveillance, priming by vaccination, or following adoptive transfer of T-cells. Although a variety of tumor-associated antigens have been identified and many immunotherapeutic strategies have been tested, objective clinical responses are rare. The reasons for this include the inability of current immunotherapy approaches to generate efficient T-cell responses, the presence of regulatory cells that inhibit T-cell responses, and other escape mechanisms that tumors develop, such as inactivation of cytolytic T-cells through expression of negative costimulatory molecules. Effective immunotherapy for cancer will require the use of appropriate tumor-specific antigens; the optimization of the interaction between the antigenic peptide, the APC and the T cell; and the simultaneous blockade of negative regulatory mechanisms that impede immunotherapeutic effects.","T-cell activation plays a central role in driving both protective and pathogenic immune responses, and it requires the completion of a carefully orchestrated series of specific steps that can be preempted or disrupted by any number of critical events. Na\u00efve T cells must receive two independent signals from antigen-presenting cells (APC) in order to become productively activated. The first, Signal 1, is antigen-specific and occurs when T cell antigen receptors encounter the appropriate antigen-MHC complex on the APC. A second, antigen-independent signal (Signal 2) is delivered through a T cell costimulatory molecule that engages its APC-expressed ligand. In the absence of a costimulatory signal, T-cell activation is impaired or aborted, which may lead to a state of antigen-specific unresponsiveness (known as T-cell anergy), or may result in T-cell apoptotic death.","Costimulatory signals can be either stimulatory (positive costimulation) or inhibitory (negative costimulation or coinhibition). Positive costimulation is required for optimal activation of na\u00efve T cells, while negative costimulation is required for the acquisition of immunologic tolerance to self, as well as the termination of effector T cell functions. Costimulatory signals, particularly positive costimulatory signals, also play a role in the modulation of B cell activity. For example, B cell activation and the survival of germinal center B cells require T cell-derived signals in addition to stimulation by antigen.","Both positive and negative costimulatory signals play critical roles in the regulation of cell-mediated immune responses, and molecules that mediate these signals have proven to be effective targets for immunomodulation. Based on this knowledge, several therapeutic approaches that involve targeting of costimulatory molecules have been developed, and were shown to be useful for prevention and treatment of cancer and autoimmune diseases, as well as rejection of allogenic transplantation, each by turning on, or preventing the turning off, of immune responses in subjects with these pathological conditions.","Costimulatory molecule pairs usually consist of ligands expressed on APCs and their cognate receptors expressed on T cells. The well characterized B7\/CD28 and CD40\/CD40L costimulatory molecules are critical in primary T-cell activation. In recent years, several additional costimulatory molecules have been identified, that belong to the B7\/CD28 or the TNF\/TNF-R gene families. The effects of costimulatory TNFR family members can often be functionally, temporally, or spatially segregated from those of CD28 family members and from each other. The sequential and transient regulation of T cell activation\/survival signals by different costimulators may function to allow longevity of the response while maintaining tight control of T cell survival.","The B7 family consists of structurally related, cell-surface protein ligands, which bind to receptors on lymphocytes that regulate immune responses. Interaction of B7-family members with their respective costimulatory receptor, usually a member of the CD28-related family, augments immune responses, while interaction with coinhibitory receptors, such as CTLA4, attenuates immune responses. Members of the B7 family share 20-40% amino-acid identity and are structurally related, with the extracellular domain containing tandem domains related to variable and constant immunoglobulin domains.","There are currently seven known members of the family: B7.1 (CD80), B7.2 (CD86), B7-H1 (PD-L1), B7-H2 (ICOS-L), B7-DC (PD-L2), B7-H3, and B7-H4, each with unique, yet often overlapping functions. Clearly, each B7 molecule has developed its own indispensable niche in the immune system. As specific niches of B7 family members continue to be dissected, their diagnostic and therapeutic potential becomes ever more apparent. Many of the B7 superfamily members were initially characterized as T cell costimulatory molecules. However, more recently it has become clear they can also coinhibit T cell responses. Thus, B7 family members may have opposing effects on an immune response.","Central to the normal function of the immune system is its ability to distinguish between self and non-self, since failure to do so could provoke the onset of autoimmune disease. Most autoimmune disorders are known to involve autoreactive T cells and\/or autoantibodies. Thus, agents that are capable of inhibiting or eliminating autoreactive lymphocytes have a promising therapeutic potential. Furthermore, the use of agents that exhibit such immunosuppressive activity should also be beneficial in order to inhibit normal immune responses to alloantigens in patients receiving a transplant. Thus, novel agents that are capable of modulating costimulatory signals, without compromising the immune system's ability to defend against pathogens, are highly advantageous for treatment and prevention of such pathological conditions.","The importance of the B7 family members in regulating immune responses to self and allo-antigens was demonstrated by the development of immunodeficiency and autoimmune diseases in mice with mutations in B7-family genes. Accordingly, manipulation of the signals delivered by B7 ligands has shown potential in the treatment of autoimmunity, inflammatory diseases, and transplant rejection. This approach relies, at least partially, on the eventual deletion of auto- or allo-reactive T cells, presumably because in the absence of costimulation (which induces cell survival genes) T cells become highly susceptible to induction of apoptosis.","Harnessing the immune system to treat chronic diseases is a major goal of immunotherapy. Active and passive immunotherapies are proving themselves as effective therapeutic strategies. Passive immunotherapy, using monoclonal antibodies or receptor Fc-fusion proteins, has come of age and has shown great clinical success. A growing number of such therapeutic agents have been approved or are in clinical trials to prevent allograft rejection or to treat autoimmune diseases and cancer. Active immunotherapy (i.e. vaccines) has been effective against agents that normally cause acute self-limiting infectious diseases followed by immunity and has been at the forefront of efforts to prevent the infectious diseases that plague humankind. However, active immunotherapy has been much less effective against cancer or chronic infectious diseases primarily because these have developed strategies to escape normal immune responses. Among these are negative costimulators of the B7 family, such as B7-H1 and B7-H4, which are highly expressed in certain tumors, and afford local protection from immune cells-mediated attack.","The efficiency of the immune system in mediating tumor regression depends on the induction of antigen-specific T-cell responses through physiologic immune surveillance, priming by vaccination, or following adoptive transfer of T-cells. Although a variety of tumor-associated antigens have been identified and many immunotherapeutic strategies have been tested, objective clinical responses are rare. The reasons for this include the inability of current immunotherapy approaches to generate efficient T-cell responses, the presence of regulatory cells that inhibit T-cell responses, and other escape mechanisms that tumors develop, such as inactivation of cytolytic T-cells through expression of negative costimulatory molecules. Effective immunotherapy for cancer will require the use of appropriate tumor-specific antigens; the optimization of the interaction between the antigenic peptide, the APC and the T cell; and the simultaneous blockade of negative regulatory mechanisms that impede immunotherapeutic effects.","Costimulators of the B7 family play a critical role in activation and inhibition of antitumor immune responses. Novel agents targeting these molecules could find significant use in the modulation of immune responses and the improvement of cancer immunotherapy. Such agents could be administered in conjunction with tumor-specific antigens, as an adjuvant that serves to enhance the immune response to the antigen in the patient. In addition, such agents could be of use in other types of cancer immunotherapy, such as adoptive immunotherapy, in which tumor-specific T cell populations are expanded and directed to attack and kill tumor cells. Agents capable of augmenting such anti-tumor response have great therapeutic potential and may be of value in the attempt to overcome the obstacles to tumor immunotherapy.","Passive tumor immunotherapy uses the exquisite specificity and lytic capability of the immune system to target tumor specific antigens and treat malignant disease with a minimum of damage to normal tissue. Several approaches have been used to identify tumor-associated antigens as target candidates for immunotherapy. The identification of novel tumor specific antigens expands the spectrum of tumor antigen targets available for immune recognition and provides new target molecules for the development of therapeutic agents for passive immunotherapy, including monoclonal antibodies, whether unmodified or armed. Such novel antigens may also point the way to more effective therapeutic vaccines for active or adoptive immunotherapy.","Clinical development of costimulation blockade came to fruition with the approval of CTLA4Ig (abatacept) for rheumatoid arthritis. This soluble fusion protein, which acts as competitive inhibitor of the B7\/CD28 costimulatory pathway, is also in clinical trials for other immune diseases such as psoriasis and multiple sclerosis, and for transplant rejection. Promising results have also been obtained in a phase II clinical trial in kidney transplantation with belatacept, a re-engineered CTLA4Ig with enhanced binding affinity to its ligands, B7.1 and B7.2 (CD80 and CD86, respectively). Two fully human anti-CTLA4 monoclonal antibodies, Ipilimumab and tremelimumab, abrogate the CTLA4\/B7 inhibitory interaction, and are in clinical phase III for metastatic melanoma and other cancers, as well as HIV infection. Galiximab is a primatized monoclonal antibody targeting CD80, in Phase II for rheumatoid arthritis, psoriasis and Non-Hodgkin's lymphoma.","It is important to point out that strategies that use single agents to block costimulation have often proved to be insufficient. Given the diversity of the different costimulation molecules, future strategies may involve the simultaneous blockade of several selected pathways or combination therapy with conventional drugs, such as immunosuppressants for immune-related disorders or cytotoxic drugs for cancer.","Despite recent progress in the understanding of cancer biology and cancer treatment, as well as better understanding of the molecules involved in immune responses, the success rate for cancer therapy and for the treatment of autoimmune diseases remains low. Therefore, there is an unmet need for new therapies which can successfully treat both cancer and autoimmune disorders.","It is an object of the invention to provide novel therapeutic and diagnostic compositions containing at least one of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 proteins or one of the novel splice variants disclosed herein as well as to provide these novel VSIG1 splice variants; specifically ILDR1 splice variants; LOC253012 splice variants; AI216611 splice variants, C1ORF32 splice variants; and FXYD3 splice variants, and nucleic acid sequences encoding for same or fragments thereof especially the ectodomain or secreted forms of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins and\/or splice variants.","It is another object of the invention to use said proteins, splice variants and nucleic acid sequences as novel targets for development of drugs which specifically bind to the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins and\/or splice variants, and\/or drugs which agonize or antagonize the binding of other moieties to the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins and\/or splice variants.","It is still another object of the invention to provide drugs which modulate (agonize or antagonize) at least one VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 related biological activity. Such drugs include by way of example antibodies, small molecules, peptides, ribozymes, antisense molecules, siRNA's and the like. These molecules may directly bind or modulate an activity elicited by the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins or VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 DNA or portions or variants thereof or may indirectly modulate a VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 associated activity or binding of molecules to VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 and portions and variants thereof such as by modulating the binding of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 to its counterreceptor or endogenous ligand.","In more specific embodiments, the present invention provides novel splice variants of a known protein V-set and immunoglobulin domain containing 1 (SEQ ID NO:11) (RefSeq accession identifier NP872413, synonyms: RP5-889N15.1, 1700062D20Rik, GPA34, MGC44287, dJ889N15.1) or a polynucleotide encoding same, which can be used as diagnostic markers and\/or therapeutic agents which agonize or antagonize the binding of other moieties to the VSIG1 proteins and\/or which modulate (agonize or antagonize) at least one VSIG1 related biological activity.","According to one more specific embodiment, the novel splice variant is an isolated polynucleotide comprising a nucleic acid having a nucleic acid sequence as set forth in any one of AI581519_T10 (SEQ ID NO:9), AI581519_T11 (SEQ ID NO:10), or a sequence homologous thereto. According to another embodiment, the isolated polynucleotide is at least 95% homologous to any one of AI581519_T10 (SEQ ID NO:9), AI581519_T11 (SEQ ID NO:10).","According to yet another more specific embodiment, the novel splice variant is an isolated protein or polypeptide having an amino acid sequence as set forth in any one of AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), or a sequence homologous thereto. According to another embodiment, the isolated polypeptide is at least 95% homologous to any one of AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16).","It is another specific object of the invention to provide molecules and isolated polypeptides comprising the soluble ectodomain (ECD) of the VSIG1 proteins and fragments thereof as well as nucleic acid sequences encoding said soluble ectodomain, as well as fragments thereof and conjugates and the use thereof as therapeutics including their use in immunotherapy (promoting or inhibiting immune costimulation).","In more specific embodiments the present invention provides discrete portions of the VSIG1 proteins including different portions of the extracellular domain corresponding to residues 23-234 of the VSIG1 protein sequence contained in the sequence of AI581519_P3 (SEQ ID NO:11), corresponding to amino acid sequence depicted in SEQ ID NO:138, or residues 23-270 of the of the VSIG1 protein sequence contained in the sequence of AI581519_P4 (SEQ ID NO:12), corresponding to amino acid sequence depicted in SEQ ID NO:139, or residues 23-296 of the VSIG1 protein sequence contained in the sequence of AI581519_P5 (SEQ ID NO:13), corresponding to amino acid sequence depicted in SEQ ID NO:140, or residues 23-193 of the VSIG1 protein sequence contained in the sequence of AI581519_P7 (SEQ ID NO:14) corresponding to amino acid sequence depicted in SEQ ID NO:141, or residues 23-203 of the VSIG1 protein sequence contained in the of AI581519_P9 (SEQ ID NO:15) corresponding to amino acid sequence depicted in SEQ ID NO:142, or residues 23-231 of the VSIG1 protein sequence contained in the sequence of AI581519_P10 (SEQ ID NO:16), corresponding to amino acid sequence depicted in SEQ ID NO:143, or residues 26-293 of the VSIG1 protein sequence contained in the sequence of AI581519_P5 (SEQ ID NO:13), corresponding to amino acid sequence depicted in SEQ ID NO:302, or variants thereof possessing at least 80% sequence identity, more preferably at least 90% sequence identity therewith and even more preferably at least 95, 96, 97, 98 or 99% sequence identity therewith.","According to other more specific embodiments, the present invention provides novel splice variants of a known protein immunoglobulin-like domain containing receptor 1 (SEQ ID NO:21) (RefSeq accession identifier NP787120, also known as ILDR1alpha, ILDR1beta, ILDR1), or a polynucleotide encoding same, which can be used as diagnostic markers and\/or therapeutic agents which agonize or antagonize the binding of other moieties to the ILDR1 proteins and\/or which modulate (agonize or antagonize) at least one ILDR1 related biological activity.","In one specific embodiment, the novel splice variant is an isolated polynucleotide comprising a nucleic acid having a nucleic acid sequence as set forth in AA4248391_T7 (SEQ ID NO:20), or a sequence homologous thereto. According to another embodiment, the isolated polynucleotide is at least 95, 96, 97, 98 or 99% homologous to AA4248391_T7 (SEQ ID NO:20).","According to yet another specific embodiment, the novel splice variant is an isolated protein or polypeptide having an amino acid sequence as set forth in AA4248391_P11 (SEQ ID NO:24), or a sequence homologous there, i.e., which possesses at least 80, or 90% sequence identity therewith. According to another related embodiment, the isolated polypeptide is at least 95, 96, 97, 98 or 99% homologous to AA4248391_P11 (SEQ ID NO:24).","It is another embodiment of the invention to provide molecules and isolated polypeptides comprising the soluble ectodomain (ECD) of the ILDR1 proteins and fragments thereof as well as nucleic acid sequences encoding said soluble ectodomain, as well as fragments thereof and conjugates and the use thereof as therapeutics including their use in immunotherapy (promoting or inhibiting immune costimulation).","According to yet further embodiments the present invention provides discrete portions of the ILDR1 proteins including different portions of the extracellular domain corresponding to residues 24-162 of sequences AA424839_P3 (SEQ ID NO:22) and AA424839_P5 (SEQ ID NO:21), corresponding to amino acid sequence depicted in SEQ ID NO:75, or residues 24-457 of AA424839_P7 (SEQ ID NO:23), corresponding to amino acid sequence depicted in SEQ ID NO:76, or residues 24-105 of AA4248391_P11 (SEQ ID NO:24), corresponding to amino acid sequence depicted in SEQ ID NO:296, or residues 50-160 of AA4248391_P3 (SEQ ID NO:22), corresponding to amino acid sequence depicted in SEQ ID NO:301, or variants thereof possessing at least 80% sequence identity, more preferably at least 90% sequence identity therewith and even more preferably at least 95, 96, 97, 98 or 99% sequence identity therewith.","It is another embodiment of the invention to provide an isolated or purified soluble protein or nucleic acid sequence having or encoding the extracellular domain of the ILDR1 protein which optionally may be directly or indirectly attached to a non-ILDR1 protein or nucleic acid sequence such as a soluble immunoglobulin domain or fragment.","According to certain embodiments, the present invention provides novel splice variants of a known hypothetical protein LOC253012 isoform 1 (SEQ ID NO:35) (RefSeq accession identifier NP001034461) or a polynucleotide encoding same, and their use as diagnostic markers and\/or as therapeutic agents which agonize or antagonize the binding of other moieties to the LOC253012 proteins and\/or which modulate (agonize or antagonize) at least one LOC253012 related biological activity.","According to one embodiment, the novel LOC253012 splice variant is an isolated polynucleotide comprising a nucleic acid having a nucleic acid sequence as set forth in any one of H686541_T8 (SEQ ID NO:28), H686541_T15 (SEQ ID NO:29), H686541_T16 (SEQ ID NO:30), H686541_T17 (SEQ ID NO:31), H686541_T18 (SEQ ID NO:32), H686541_T19 (SEQ ID NO:33), or H686541_T20 (SEQ ID NO:34) or a sequence homologous thereto. According to another embodiment, the isolated polynucleotide is at least 95% homologous to any one of H686541_T8 (SEQ ID NO:28), H686541_T15 (SEQ ID NO:29), H686541_T16 (SEQ ID NO:30), H686541_T17 (SEQ ID NO:31), H686541_T18 (SEQ ID NO:32), H686541_T19 (SEQ ID NO:33), or H686541_T20 (SEQ ID NO:34).","According to yet another embodiment, the novel LOC253012 splice variant is an isolated protein or polypeptide having an amino acid sequence as set forth in any one of H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40) or a sequence homologous thereto. According to another embodiment, the isolated polypeptide is at least 95, 96, 97, 98 or 99% homologous to any one of H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40).","It is another object of the invention to provide molecules and isolated polypeptides comprising the soluble ectodomain (ECD) of the LOC253012 proteins and fragments thereof as well as nucleic acid sequences encoding said soluble ectodomain, as well as fragments thereof and conjugates and the use thereof as therapeutics including their use in immunotherapy (promoting or inhibiting immune costimulation).","According to yet further embodiments of the present invention there are discrete portions of the LOC253012 proteins including different portions of the extracellular domain corresponding to residues 38-349 of the sequence H686541_P2 (SEQ ID NO:35), corresponding to amino acid sequence depicted in SEQ ID NO:144, or residues 19-337 of the sequences H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), corresponding to amino acid sequence depicted in SEQ ID NO:145, or residues 1-335 of the sequences H686541_P5 (SEQ ID NO:36), corresponding to amino acid sequence depicted in SEQ ID NO:300, or variants thereof possessing at least 80% sequence identity, more preferably at least 90% sequence identity therewith and even more preferably at least 95, 96, 97, 98 or 99% sequence identity therewith.","It is another object of the invention to provide an isolated or purified soluble protein or nucleic acid sequence encoding having or encoding the extracellular domain of the LOC253012 protein which optionally may be directly or indirectly attached to a non-LOC253012 protein or nucleic acid sequence such as a soluble immunoglobulin domain or fragment.","According to certain embodiments, the present invention provides novel splice variants of AI216611, or a polynucleotide encoding same, which can be used as diagnostic markers and\/or therapeutic agents which agonize or antagonize the binding of other moieties to the AI216611 proteins and\/or which modulate (agonize or antagonize) at least one AI216611 related biological activity.","According to one embodiment, the novel AI216611 splice variant is an isolated polynucleotide comprising a nucleic acid having a nucleic acid sequence as set forth in AI216611_T1 (SEQ ID NO:42), or a sequence homologous thereto. According to another embodiment, the isolated polynucleotide is at least 95, 96, 97, 98 or 99% homologous to AI216611_T1 (SEQ ID NO:42).","According to yet another embodiment, the novel AI216611 splice variant is an isolated protein or polypeptide having an amino acid sequence as set forth in AI216611_P1 (SEQ ID NO:44) or a sequence homologous thereto. According to another embodiment, the isolated polypeptide is at least 95, 96, 97, 98 or 99% homologous to AI216611_P1 (SEQ ID NO:44).","It is another object of the invention to provide molecules and isolated polypeptides comprising the soluble ectodomain (ECD) of the AI216611 proteins and fragments thereof as well as nucleic acid sequences encoding said soluble ectodomain, as well as fragments thereof and conjugates and the use thereof as therapeutics including their use in immunotherapy (such as promoting or inhibiting immune costimulation). According to yet further embodiments of the present invention there are discrete portions of the AI216611 proteins including different portions of the extracellular domain corresponding to residues 29-147 of the sequence AI216611_P0 (SEQ ID NO:43) or AI216611_P1 (SEQ ID NO:44), corresponding to amino acid sequence depicted in SEQ ID NO:146, or residues 1-145 of the sequence AI216611_P0 (SEQ ID NO:43), corresponding to amino acid sequence depicted in SEQ ID NO:298, or variants thereof possessing at least 80% sequence identity, more preferably at least 90% sequence identity therewith and even more preferably at least 95, 96, 97, 98 or 99% sequence identity therewith.","It is another object of the invention to provide an isolated or purified soluble protein or nucleic acid sequence having or encoding the extracellular domain of the AI216611 protein which optionally may be directly or indirectly attached to a non-AI216611 protein or nucleic acid sequence such as a soluble immunoglobulin domain or fragment.","It is another object of the invention to provide vectors such as plasmids and recombinant viral vectors and host cells containing that express AI216611, its secreted or soluble form and\/or the ECD of the AI216611 protein and variants thereof or polypeptide conjugates containing any of the foregoing.","According to certain embodiments, the present invention provides novel splice variants of a known hypothetical protein LOC387597 (SEQ ID NO:47) (RefSeq accession identifier NP955383, synonyms: NP955383; LISCH-like; C1ORF32; RP4-782G3.2; dJ782G3.1) or a polynucleotide encoding same, which can be used as diagnostic markers and\/or therapeutic agents which agonize or antagonize the binding of other moieties to the C1ORF32 proteins and\/or which modulate (agonize or antagonize) at least one C1ORF32 related biological activity.","According to one embodiment, the novel LOC387597 splice variant is an isolated polynucleotide comprising a nucleic acid having a nucleic acid sequence as set forth in any one of H190111_T8 (SEQ ID NO:45), H190111_T9 (SEQ ID NO:46), or a sequence homologous thereto. According to another embodiment, the isolated polynucleotide is at least 95, 96, 97, 98 or 99% homologous to any one of H190111_T8 (SEQ ID NO:45), H190111_T9 (SEQ ID NO:46).","According to yet another embodiment, the novel splice LOC387597 variant is an isolated protein or polypeptide having an amino acid sequence as set forth in any one of H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50) or a sequence homologous thereto. According to another embodiment, the isolated polypeptide is at least 95, 96, 97, 98 or 99% homologous to any one of H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50).","It is another object of the invention to provide molecules and isolated polypeptides comprising the soluble ectodomain (ECD) of the C1ORF32 proteins and fragments thereof as well as nucleic acid sequences encoding said soluble ectodomain, as well as fragments thereof and conjugates and the use thereof as therapeutics including their use in immunotherapy (promoting or inhibiting immune costimulation).","According to yet further embodiments of the present invention there are discrete portions of the C1ORF32 proteins including different portions of the extracellular domain corresponding to residues 21-186 of the sequence H190111_P8 (SEQ ID NO:48), corresponding to amino acid sequence depicted in SEQ ID NO:147, or residues 21-169 of the sequence H190111_P9 (SEQ ID NO:50), corresponding to amino acid sequence depicted in SEQ ID NO:148, or residues 1-184 of the sequence H190111_P8 (SEQ ID NO:48), corresponding to amino acid sequence depicted in SEQ ID NO:299 or variants thereof possessing at least 80% sequence identity, more preferably at least 90% sequence identity therewith and even more preferably at least 95, 96, 97, 98 or 99% sequence identity therewith.","It is another object of the invention to provide an isolated or purified soluble protein or nucleic acid sequence encoding having or encoding the extracellular domain of the C1ORF32 protein which optionally may be directly or indirectly attached to a non-C1ORF32 protein or nucleic acid sequence such as a soluble immunoglobulin domain or fragment.","According to certain embodiments, the present invention provides novel splice variants of known protein FXYD3, FXYD domain-containing ion transport regulator 3 precursor (SEQ ID NO:70) (SwissProt accession identifier FXYD3_HUMAN; known also according to the synonyms Chloride conductance inducer protein Mat-8; Mammary tumor 8 kDa protein; Phospholemman-like) or a polynucleotide encoding same, which can be used as diagnostic markers and\/or therapeutic agents which agonize or antagonize the binding of other moieties to the FXYD3 proteins and\/or which modulate (agonize or antagonize) at least one FXYD3 related biological activity.","According to one embodiment, the novel FXYD3 splice variant is an isolated polynucleotide comprising a nucleic acid having a nucleic acid sequence as set forth in any one of R31375_T19 (SEQ ID NO:65), R31375_T25 (SEQ ID NO:66), R31375_T26 (SEQ ID NO:67), R31375_T29 (SEQ ID NO:68), R31375_T39 (SEQ ID NO:69), or a sequence homologous thereto. According to another embodiment, the isolated polynucleotide is at least 95, 96, 97, 98 or 99% homologous to any one of R31375_T19 (SEQ ID NO:65), R31375_T25 (SEQ ID NO:66), R31375_T26 (SEQ ID NO:67), R31375_T29 (SEQ ID NO:68), R31375_T39 (SEQ ID NO:69).","According to yet another embodiment, the novel FXYD3 splice variant is an isolated protein or polypeptide having an amino acid sequence as set forth in any one of R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74) or a sequence homologous thereto. According to another embodiment, the isolated polypeptide is at least 95, 96, 97, 98 or 99% homologous to any one of R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74).","It is another object of the invention to provide molecules and isolated polypeptides comprising the soluble ectodomain (ECD) of the FXYD3 proteins and fragments thereof as well as nucleic acid sequences encoding said soluble ectodomain, as well as fragments thereof and conjugates and the use thereof as therapeutics including their use in cancer immunotherapy.","According to yet further embodiments of the present invention there are discrete portions of the FXYD3 proteins including different portions of the extracellular domain corresponding to residues 21-36 of the sequence R31375_P0 (SEQ ID NO:70) or R31375_P31 (SEQ ID NO:73), corresponding to amino acid sequence depicted in SEQ ID NO:149, or residues 21-65 of the sequence R31375_P14 (SEQ ID NO:72), corresponding to amino acid sequence depicted in SEQ ID NO:150, or residues 21-25 of the sequence R31375_P33 (SEQ ID NO:74), corresponding to amino acid sequence depicted in SEQ ID NO:151, or residues 1-63 of the sequence R31375_P14 (SEQ ID NO:72), corresponding to amino acid sequence depicted in SEQ ID NO:297, or variants thereof possessing at least 80% sequence identity, more preferably at least 90% sequence identity therewith and even more preferably at least 95, 96, 97, 98 or 99% sequence identity therewith.","It is another object of the invention to provide an isolated or purified soluble protein or nucleic acid sequence encoding having or encoding the extracellular domain of any one of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 proteins which optionally may be directly or indirectly attached to a non-VSIG1, non-ILDR1, non-LOC253012, non-AI216611, non-C1ORF32 or non-FXYD3 protein or nucleic acid sequence, respectively, such as a soluble immunoglobulin domain or fragment.","It is another object of the invention to provide molecules and isolated polypeptides comprising edge portion, tail or head portion, of any one of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 novel variants of the invention, or a homologue or a fragment thereof as well as nucleic acid sequences encoding said edge portion, tail or head portion, as well as fragments thereof and conjugates and the use thereof as therapeutics and\/or for diagnostics.","It is further object of the invention to provide molecules and isolated polypeptides comprising a bridge, edge portion, tail or head portion, as depicted in any one of SEQ. ID NOs: 284-295, or a homologue or a fragment thereof as well as nucleic acid sequences encoding said edge portion, tail or head portion, as well as fragments thereof and conjugates and the use thereof as therapeutics and\/or for diagnostics.","It is another object of the invention to provide vectors such as plasmids and recombinant viral vectors and host cells containing the vectors that express any one of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3, its secreted or soluble form and\/or the ECD of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 protein and variants thereof or polypeptide conjugates containing any of the foregoing.","It is another object of the invention to use these vectors such as plasmids and recombinant viral vectors and host cells containing that express any one of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3, its secreted or soluble form and\/or the ECD of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 protein and variants thereof or polypeptide conjugates containing any of the foregoing to produce said VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 protein, fragments or variants thereof and\/or conjugates containing any one of the foregoing.","It is another object of the invention to provide pharmaceutical or diagnostic compositions containing any of the foregoing.","It is another object of the invention to provide and use compounds including VSIG1 ectodomain or fragments or variants thereof, which are suitable for treatment or prevention of cancer, autoimmune disorders, transplant rejection, graft versus host disease, and\/or for blocking or promoting immune costimulation mediated by the VSIG1, ILDR1, LOC253012, AI216611, FXYD3 or C1ORF32 polypeptide.","It is a specific object of the invention to develop novel monoclonal or polyclonal antibodies and antibody fragments and conjugates containing that specifically bind the full length VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen, selected from the group consisting of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), its secreted form and\/or the ECD thereof or conjugates or fragments thereof. These antibodies are potentially useful as therapeutics and\/or diagnostic agents (both in vitro and in vivo diagnostic methods). Included in particular are antibodies and fragments that are immune activating or immune suppressing such as antibodies or fragments that target cells via ADCC (antibody dependent cellular cytotoxicity) or CDC (complement dependent cytotoxicity) activities.","It is another object of the invention to provide diagnostic methods that include the use of any of the foregoing including by way of example immunohistochemical assay, radioimaging assays, in-vivo imaging, radioimmunoassay (RIA), ELISA, slot blot, competitive binding assays, fluorimetric imaging assays, Western blot, FACS, and the like. In particular this includes assays which use chimeric or non-human antibodies or fragments that specifically bind the intact VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 protein, selected from the group consisting of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), its soluble form, its ECD, and or conjugates, fragments or variants thereof.","It is another object of the invention to use novel therapeutically effective polyclonal or monoclonal antibodies against anyone of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen, selected from the group consisting of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), and fragments, conjugates, and variants thereof for treating conditions wherein the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen or its secreted or soluble form or ECD and\/or portions or variants thereof are differentially expressed including various cancers and malignancies including non-solid and solid tumors, sarcomas, hematological malignancies including but not limited to acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, cancer of the breast, prostate, lung, ovary, colon, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, brain and wherein the cancer is non-metastatic, invasive or metastatic.","It is another object of the invention to use novel therapeutically effective polyclonal or monoclonal antibodies against anyone of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen, selected from the group consisting of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), and fragments, conjugates and variants thereof for treating non-malignant disorders such as immune disorders including but not limited to autoimmune diseases, transplant rejection and graft versus host disease.","It is a specific object of the invention to use antibodies and antibody fragments against VSIG1 antigen, its secreted or soluble form or ECD and\/or variants, conjugates, or fragments thereof and fragments and variants thereof for treating and diagnosing lung cancer and\/or ovarian cancer, wherein this antigen is differentially expressed.","It is a specific embodiment of the invention to use antibodies and antibody fragments against ILDR1 antigen, its secreted or soluble form or ECD and\/or variants, conjugates, or fragments thereof and fragments and variants thereof for treating and diagnosing colon and\/or ovarian cancers wherein this antigen is differentially expressed.","It is a specific object of the invention to use antibodies and antibody fragments against LOC253012 or C1ORF32 antigen, its secreted or soluble form or ECD and\/or variants, conjugates, or fragments thereof and fragments and variants thereof for treating and diagnosing lung cancer, particularly small cell lung carcinoma, wherein this antigen is differentially expressed.","It is a specific object of the invention to use antibodies and antibody fragments against AI216611 antigen, its secreted or soluble form or ECD and\/or variants, conjugates, or fragments thereof and fragments and variants thereof for treating and diagnosing colon cancer, wherein this antigen is differentially expressed.","It is a specific object of the invention to use antibodies and antibody fragments against FXYD3 wild type antigen (R31375_P0 (SEQ ID NO:70)), or antibodies and antibody fragments against its secreted or soluble form or ECD and conjugates containing for treating and diagnosing ovarian cancer, wherein this antigen is differentially expressed.","It is another object of the invention to use antibodies and antibody fragments, and conjugates containing, against the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen, selected from the group consisting of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74) in modulating (enhancing or inhibiting) immunity including antibodies that activate or suppress the immune co-stimulation in particular B7 related immune costimulation and are capable of treating related therapeutic applications, through positive stimulation of T cell activity against cancer cells, and negative stimulation of T cell activity for the treatment of autoimmunity and other immune disorders.","It is another specific object of the invention to produce antibodies and antibody fragments against discrete portions of the VSIG1 proteins including different portions of the extracellular domain corresponding to residues 23-234 of the VSIG1 protein sequence contained in the sequence of AI581519_P3 (SEQ ID NO:11) corresponding to amino acid sequence depicted in SEQ ID NO:138, or residues 23-270 of the VSIG1 protein sequence contained in the sequence of AI581519_P4 (SEQ ID NO:12) corresponding to amino acid sequence depicted in SEQ ID NO:139, or residues 23-296 of the VSIG1 protein sequence contained in the sequence of AI581519_P5 (SEQ ID NO:13) corresponding to amino acid sequence depicted in SEQ ID NO:140, or residues 23-193 of the VSIG1 protein sequence contained in the sequence of AI581519_P7 (SEQ ID NO:14) corresponding to amino ac, id sequence depicted in SEQ ID NO:141, or residues 23-203 of the VSIG1 protein sequence contained in the sequence of AI581519_P9 (SEQ ID NO:15) corresponding to amino acid sequence depicted in SEQ ID NO:142, or residues 23-231 of the VSIG1 protein sequence contained in the sequence of AI581519_P10 (SEQ ID NO:16), corresponding to amino acid sequence depicted in SEQ ID NO:143, or residues 26-293 of the VSIG1 protein sequence contained in the sequence of AI581519_P5 (SEQ ID NO:13), corresponding to amino acid sequence depicted in SEQ ID NO:302.","It is another specific embodiment of the invention to produce antibodies and antibody fragments against discrete portions of the ILDR1 proteins including different portions of the extracellular domain corresponding to residues 24-162 of sequences AA424839_P3 (SEQ ID NO:22) and AA424839_P5 (SEQ ID NO:21), corresponding to amino acid sequence depicted in SEQ ID NO:75, residues 24-457 of AA424839_P7 (SEQ ID NO:23), corresponding to amino acid sequence depicted in SEQ ID NO:76, and residues 24-105 of AA4248391_P11 (SEQ ID NO:24), corresponding to amino acid sequence depicted in SEQ ID NO:296, or residues 50-160 of AA4248391_P3 (SEQ ID NO:22), corresponding to amino acid sequence depicted in SEQ ID NO:301 of the ILDR1 protein sequences disclosed herein.","It is another specific object of the invention to produce antibodies and antibody fragments against discrete portions of the LOC253012 proteins including different portions of the extracellular domain corresponding to residues 38-349 of the sequence H686541_P2 (SEQ ID NO:35), corresponding to amino acid sequence depicted in SEQ ID NO:144, or residues 19-337 of the sequences H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), or residues 1-335 of the sequences H686541_P5 (SEQ ID NO:36), corresponding to amino acid sequence depicted in SEQ ID NO:300, of the LOC253012 protein sequences disclosed herein.","It is another specific object of the invention to produce antibodies and antibody fragments against discrete portions of the AI216611 proteins including different portions of the extracellular domain corresponding to residues 29-147 of the sequence AI216611_P0 (SEQ ID NO:43) or AI216611_P1 (SEQ ID NO:44), corresponding to amino acid sequence depicted in SEQ ID NO:146, or residues 1-145 of the sequence AI216611_P0 (SEQ ID NO:43), corresponding to amino acid sequence depicted in SEQ ID NO:298 sequence disclosed herein.","It is another specific object of the invention to produce antibodies and antibody fragments against discrete portions of the C1ORF32 proteins including different portions of the extracellular domain corresponding to residues 21-186 of the C1ORF32 protein sequence contained in the sequence of H190111_P8 (SEQ ID NO:48), corresponding to amino acid sequence depicted in SEQ ID NO:147, or residues 21-169 of the C1ORF32 protein sequence contained in the sequence of H190111_P9 (SEQ ID NO:50), corresponding to amino acid sequence depicted in SEQ ID NO:148, or residues 1-184 of the sequence H190111_P8 (SEQ ID NO:48), corresponding to amino acid sequence depicted in SEQ ID NO:299.","It is another specific object of the invention to produce antibodies and antibody fragments against discrete portions of the FXYD3 proteins including different portions of the extracellular domain corresponding to residues 21-36 of the FXYD3 protein sequence contained in the sequence of R31375_P0 (SEQ ID NO:70) or R31375_P31 (SEQ ID NO:73), corresponding to amino acid sequence depicted in SEQ ID NO:149, or residues 21-65 of the FXYD3 protein sequence contained in the sequence of R31375_P14 (SEQ ID NO:72), corresponding to amino acid sequence depicted in SEQ ID NO:150, or residues 21-25 of the FXYD3 protein sequence contained in the sequence of R31375_P33 (SEQ ID NO:74), corresponding to amino acid sequence depicted in SEQ ID NO:151, or residues 1-63 of the sequence R31375_P14 (SEQ ID NO:72), corresponding to amino acid sequence depicted in SEQ ID NO:297.","It is a specific object of the invention to provide polyclonal and monoclonal antibodies and fragments thereof or an antigen binding fragment thereof comprising an antigen bindings site that binds specifically to the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 proteins, its soluble forms, the ECD thereof and\/or variants and fragments thereof.","It is a specific object of the invention to use such antibodies and fragments thereof for treatment or prevention of cancer and\/or for modulating (activating or blocking) the activity of the target in the immune co-stimulatory system.","It is a related object of the invention to select monoclonal and polyclonal antibodies and fragments thereof against VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 which are suitable for treatment or prevention of autoimmune disorders, transplant rejection, GVHD, and\/or for blocking or enhancing immune costimulation mediated by the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 polypeptide.","It is a specific object of the invention to use antibodies against anyone of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen, soluble form, ECD or fragment or variant thereof for the treatment and diagnosis of cancers including by way of example lung cancer, ovarian cancer, colon cancer, as well as other non-solid and solid tumors, sarcomas, hematological malignancies including but not limited to acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, cancer of the breast, prostate, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, brain and wherein the cancer is non-metastatic, invasive or metastatic.","With regard to lung cancer, the disease is selected from the group consisting of squamous cell lung carcinoma, lung adenocarcinoma, carcinoid, small cell lung cancer or non-small cell lung cancer.","It is another object of the invention to provide and use antibodies and antibody fragments against anyone of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen, its soluble form, or ECD and variants or fragments thereof as well as soluble polypeptides containing the ectodomain of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen or a portion thereof which are useful for immune modulation, including treatment of autoimmunity and preferably for treating an autoimmune disease selected from autoimmune diseases: Multiple sclerosis; Psoriasis; Rheumatoid arthritis; Systemic lupus erythematosus; Ulcerative colitis; Crohn's disease; immune disorders associated with graft transplantation rejection, benign lymphocytic angiitis, lupus erythematosus, Hashimoto's thyroiditis, primary myxedema, Graves' disease, pernicious anemia, autoimmune atrophic gastritis, Addison's disease, insulin dependent diabetes mellitus, good pasture's syndrome, myasthenia gravis, pemphigus, sympathetic ophthalmia, autoimmune uveitis, autoimmune hemolytic anemia, idiopathic thrombocytopenia, primary biliary cirrhosis, chronic action hepatitis, ulceratis colitis, Sjogren's syndrome, rheumatic disease, polymyositis, scleroderma, mixed connective tissue disease, inflammatory rheumatism, degenerative rheumatism, extra-articular rheumatism, collagen diseases, chronic polyarthritis, psoriasis arthropathica, ankylosing spondylitis, juvenile rheumatoid arthritis, periarthritis humeroscapularis, panarteriitis nodosa, progressive systemic scleroderma, arthritis uratica, dermatomyositis, muscular rheumatism, myositis, myogelosis and chondrocalcinosis.","It is another object of the invention to provide and use compounds including drugs such as small molecules, peptides, antibodies and fragments that bind anyone of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen, as well as ribozymes or antisense or siRNAs which target the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 nucleic acid sequence or fragments or variants thereof which are useful for treatment or prevention of cancer, autoimmune disorders, transplant rejection, GVHD, and\/or for blocking or enhancing immune costimulation mediated by the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 polypeptide.","It is another object of the invention to provide and use compounds including drugs such as small molecules, peptides, antibodies and fragments that bind the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen, as well as ribozymes or antisense or siRNAs which target the FXYD3 nucleic acid sequence or fragments or variants thereof which are useful for treatment or prevention of cancer.","It is a preferred object to provide therapeutic and diagnostic antibodies and fragments and conjugates containing useful in treating or diagnosing any of the foregoing that specifically bind to amino-acids residues 23-234 of the sequence AI581519_P3 (SEQ ID NO:11), corresponding to amino acid sequence depicted in SEQ ID NO:138, or residues 23-270 of the sequence AI581519_P4 (SEQ ID NO:12), corresponding to amino acid sequence depicted in SEQ ID NO:139, or residues 23-296 of the sequence AI581519_P5 (SEQ ID NO:13), corresponding to amino acid sequence depicted in SEQ ID NO:140, or residues 23-193 of the sequence AI581519_P7 (SEQ ID NO:14), corresponding to amino acid sequence depicted in SEQ ID NO:141, or residues 23-203 of the sequence AI581519_P9 (SEQ ID NO:15), corresponding to amino acid sequence depicted in SEQ ID NO:144, or residues 23-231 of the sequence AI581519_P10 (SEQ ID NO:16), corresponding to amino acid sequence depicted in SEQ ID NO:143, or residues 26-293 of the sequence AI581519_P5 (SEQ ID NO:13), corresponding to amino acid sequence depicted in SEQ ID NO:302 of the VSIG1 protein sequences disclosed herein.","It is a preferred embodiment to provide therapeutic and diagnostic antibodies and fragments and conjugates containing useful in treating or diagnosing any of the foregoing that specifically bind to amino-acids residues 24-162 of the ILDR1 protein sequence contained in the sequence of AA424839_P3 (SEQ ID NO:22) and AA424839_P5 (SEQ ID NO:21), corresponding to amino acid sequence depicted in SEQ ID NO:75, residues 24-457 of the ILDR1 protein sequence contained in the sequence of AA424839_P7 (SEQ ID NO:23), corresponding to amino acid sequence depicted in SEQ ID NO:76, and residues 24-105 of the ILDR1 protein sequence contained in the sequence of AA4248391_P11 (SEQ ID NO:24), corresponding to amino acid sequence depicted in SEQ ID NO:296, or residues 50-160 of AA4248391_P3 (SEQ ID NO:22), corresponding to amino acid sequence depicted in SEQ ID NO:301.","It is a preferred object to provide therapeutic and diagnostic antibodies and fragments and conjugates containing useful in treating or diagnosing any of the foregoing that specifically bind to amino-acids residues 38-349 of the LOC253012 protein sequence contained in the sequence of H686541_P2 (SEQ ID NO:35), corresponding to amino acid sequence depicted in SEQ ID NO:144, or residues 19-337 of the of the LOC253012 protein sequence contained in the sequences of H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40)), corresponding to amino acid sequence depicted in SEQ ID NO:145, or residues 1-335 of the sequences H686541_P5 (SEQ ID NO:36), corresponding to amino acid sequence depicted in SEQ ID NO:300.","It is a preferred object to provide therapeutic and diagnostic antibodies and fragments and conjugates containing useful in treating or diagnosing any of the foregoing that specifically bind to amino-acids residues 29-147 of the AI216611 protein sequence contained in the sequence of AI216611_P0 (SEQ ID NO:43) or AI216611_P1 (SEQ ID NO:44), corresponding to amino acid sequence depicted in SEQ ID NO:146, or residues 1-145 of the sequence AI216611_P0 (SEQ ID NO:43), corresponding to amino acid sequence depicted in SEQ ID NO:298.","It is a preferred object to provide therapeutic and diagnostic antibodies and fragments and conjugates containing useful in treating or diagnosing any of the foregoing that specifically bind to amino-acids residues 21-186 of the C1ORF32 protein sequence contained in the sequence of H190111_P8 (SEQ ID NO:48), corresponding to amino acid sequence depicted in SEQ ID NO:147, or residues 21-169 of the sequence of the C1ORF32 protein sequence contained in the sequence of H190111_P9 (SEQ ID NO:50), corresponding to amino acid sequence depicted in SEQ ID NO:149, or residues 1-184 of the sequence H190111_P8 (SEQ ID NO:48), corresponding to amino acid sequence depicted in SEQ ID NO:299.","It is a preferred object to provide therapeutic and diagnostic antibodies and fragments and conjugates containing useful in treating or diagnosing any of the foregoing that specifically bind to amino-acids residues 21-36 of the FXYD3 protein sequence contained in the sequence of R31375_P0 (SEQ ID NO:70), or R31375_P31 (SEQ ID NO:73), corresponding to amino acid sequence depicted in SEQ ID NO:149 or residues 21-65 of the FXYD3 protein sequence contained in the sequence of R31375_P14 (SEQ ID NO:72), corresponding to amino acid sequence depicted in SEQ ID NO:150, or residues or residues 21-25 of the FXYD3 protein sequence contained in the sequence of R31375_P33 (SEQ ID NO:74), corresponding to amino acid sequence depicted in SEQ ID NO:151, or residues 1-63 of the sequence R31375_P14 (SEQ ID NO:72), corresponding to amino acid sequence depicted in SEQ ID NO:297.","It is also a preferred object to provide antibodies and fragments thereof that bind to VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 and the specific residues above-identified and fragments thereof, wherein the antibody is a chimeric, humanized, fully human antibody and\/or is an antibody or antibody fragment having CDC or ADCC activities on target cells.","It is also a preferred object to provide chimeric and human antibodies and fragments thereof and conjugates containing that bind to VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 and the specific residues above-identified and fragments thereof.","It is another specific object of the invention to provide antibody fragments and conjugates containing useful in the foregoing therapies and related diagnostic methods including but not limited to Fab, F(ab\u2032)2, Fv or scFv fragment.","It is also an object of the invention to directly or indirectly attach the subject antibodies and fragments to markers and other effector moieties such as a detectable marker, or to an effector moiety such as an enzyme, a toxin, a therapeutic agent, or a chemotherapeutic agent.","In a preferred embodiment the inventive antibodies or fragments may be attached directly or indirectly to a radioisotope, a metal chelator, an enzyme, a fluorescent compound, a bioluminescent compound or a chemiluminescent compound.","It is also an object of the invention to provide pharmaceutical and diagnostic compositions that comprise a therapeutically or diagnostically effective form of an antibody or antibody fragment according to the invention.","It is another specific object of the invention to inhibit the growth of cells that express VSIG1 in a subject, comprising: administering to said subject an antibody that specifically binds to the antigen referred to herein as AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16) or VSIG1.","It is another specific object of the invention to provide methods for treating or preventing cancer, comprising administering to a patient an effective amount of a monoclonal antibody that specifically bind AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16) or VSIG1.","It is a more preferred object of the invention to use these antibodies for treating cancers selected from the group consisting of lung cancer, and ovarian cancer, and wherein the lung cancer or the ovarian cancer is non-metastatic, invasive or metastatic, wherein preferably the antibody has an antigen-binding region specific for the extracellular domain of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16).","It is another object of the invention to provide methods for treating or preventing autoimmune diseases, comprising administering to a patient an effective amount of a polyclonal or monoclonal antibody or fragment or a conjugate containing that specifically bind AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16).","It is another specific embodiment of the invention to inhibit the growth of cells that express ILDR1 in a subject, comprising: administering to said subject an antibody that specifically binds to the antigen referred to herein as AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24) or ILDR1.","It is another specific embodiment of the invention to provide methods for treating or preventing cancer, comprising administering to a patient an effective amount of a monoclonal antibody that specifically binds to AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24) or ILDR1.","It is a more preferred embodiment of the invention to use these antibodies for treating cancers selected from the group consisting of colon cancer or ovarian cancer, and wherein the colon cancer or the ovarian cancer is non-metastatic, invasive or metastatic wherein preferably the antibody has an antigen-binding region specific for the extracellular domain of AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23) or AA4248391_P11 (SEQ ID NO:24).","It is another embodiment of the invention to provide methods for treating or preventing autoimmune diseases, comprising administering to a patient an effective amount of a polyclonal or monoclonal antibody or fragment that specifically binds AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), or AA4248391_P11 (SEQ ID NO:24).","It is another specific object of the invention to inhibit the growth of cells that express LOC253012 in a subject, comprising: administering to said subject an antibody that specifically binds to the antigen referred to herein as H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40) or LOC253012.","It is another specific object of the invention to provide methods for treating or preventing cancer, comprising administering to a patient an effective amount of a monoclonal antibody that specifically bind H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40) or LOC253012.","It is a more preferred object of the invention to use these antibodies for treating cancers selected from the group consisting of lung cancer, especially small cell lung carcinoma, and wherein the lung cancer is non-metastatic, invasive or metastatic wherein preferably the antibody has an antigen-binding region specific for the extracellular domain of H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40).","It is another object of the invention to provide methods for treating or preventing autoimmune diseases, comprising administering to a patient an effective amount of a polyclonal or monoclonal antibody or fragment that specifically bind H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40).","It is another specific object of the invention to provide methods for treating or preventing cancer, comprising administering to a patient an effective amount of a monoclonal antibody that specifically binds to AI216611_P0 (SEQ ID NO:43) or AI216611_P1 (SEQ ID NO:44).","It is another object of the invention to provide methods for treating or preventing autoimmune diseases, comprising administering to a patient an effective amount of a polyclonal or monoclonal antibody or fragment or a conjugate containing that specifically bind AI216611_P0 (SEQ ID NO:43) or AI216611_P1 (SEQ ID NO:44).","It is another specific object of the invention to inhibit the growth of cells that express C1ORF32 in a subject, comprising: administering to said subject an antibody that specifically binds to the antigen referred to herein as H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), or C1ORF32.","It is another specific object of the invention to provide methods for treating or preventing cancer, comprising administering to a patient an effective amount of a monoclonal antibody that specifically binds to H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50) or C1ORF32.","It is a more preferred object of the invention to use these antibodies for treating cancers selected from the group consisting of lung cancer, particularly lung small cell carcinoma, and wherein the lung cancer is non-metastatic, invasive or metastatic, wherein preferably the antibody has an antigen-binding region specific for the extracellular domain of H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50).","It is another object of the invention to provide methods for treating or preventing autoimmune diseases, comprising administering to a patient an effective amount of a polyclonal or monoclonal antibody or fragment that specifically bind H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50).","It is another specific object of the invention to inhibit the growth of cells that express FXYD3 in a subject, comprising: administering to said subject an antibody that specifically binds to the antigen referred to herein as R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74).","It is another specific object of the invention to use part or all of the ectodomain of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 or its variants and conjugates containing for administration as an anti-cancer vaccine, for immunotherapy of cancer, including but not limited to ovarian cancer.","It is another specific object of the invention to provide methods for treating or preventing cancer, comprising administering to a patient an effective amount of a monoclonal antibody that specifically binds to R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74).","It is a more preferred object of the invention to use these antibodies for treating ovarian cancer, and wherein the ovarian cancer is non-metastatic, invasive or metastatic, wherein preferably the antibody has an antigen-binding region specific for the extracellular domain of R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74).","In another embodiment of the invention the cancer is selected from the group consisting of non-solid and solid tumors, sarcomas, hematological malignancies including but not limited to acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, cancer of the lung, ovary, breast, prostate, colon, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, brain and wherein the cancer may be non-metastatic, invasive or metastatic.","In a preferred embodiment the autoimmune diseases include Multiple sclerosis; Psoriasis; Rheumatoid arthritis; Systemic lupus erythematosus; Ulcerative colitis; Crohn's disease; immune disorders associated with graft transplantation rejection, benign lymphocytic angiitis, lupus erythematosus, Hashimoto's thyroiditis, primary myxedema, Graves' disease, pernicious anemia, autoimmune atrophic gastritis, Addison's disease, insulin dependent diabetes mellitis, good pasture's syndrome, myasthenia gravis, pemphigus, sympathetic ophthalmia, autoimmune uveitis, autoimmune hemolytic anemia, idiopathic thrombocytopenia, primary biliary cirrhosis, chronic action hepatitis, ulceratis colitis, Sjogren's syndrome, rheumatic disease, polymyositis, scleroderma, mixed connective tissue disease, inflammatory rheumatism, degenerative rheumatism, extra-articular rheumatism, collagen diseases, chronic polyarthritis, psoriasis arthropathica, ankylosing spondylitis, juvenile rheumatoid arthritis, periarthritis humeroscapularis, panarteriitis nodosa, progressive systemic scleroderma, arthritis uratica, dermatomyositis, muscular rheumatism, myositis, myogelosis and chondrocalcinosis.","It is a specific object of the invention to provide methods for treating or preventing rejection of any organ transplant and\/or graft versus host disease, comprising administering to a patient an effective amount of an antibody that specifically bind AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74). It is also preferred in the foregoing methods that the antibody possess an antigen-binding region specific for the extracellular domain of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), or AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74).","According to the present invention, each one of the following: the VSIG1 ectodomain, ILDR1 ectodomain, LOC253012 ectodomain, AI216611 ectodomain, C1ORF32 ectodomain or FXYD3 ectodomain of the present invention, antibodies and fragments that bind the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 of FXYD3 antigen, the compounds including drugs such as small molecules, peptides, as well as ribozymes or antisense or siRNAs which target the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 nucleic acid sequence or fragments or variants thereof which are useful for treatment or prevention of cancer, autoimmune disorders, transplant rejection, GVHD, and\/or for blocking or enhancing immune co-stimulation mediated by the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 polypeptide, can be used with simultaneous blockade of several co-stimulatory pathways or in combination therapy with conventional drugs, such as immunosuppressants or cytotoxic drugs for cancer.","It is another object of the invention to provide assays for detecting the presence of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), or AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73) or R31375_P33 (SEQ ID NO:74) protein in vitro or in vivo in a biological sample or individual comprising contacting the sample with an antibody having specificity for AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), or AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73) or R31375_P33 (SEQ ID NO:74) polypeptides, or a combination thereof, and detecting the binding of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), or AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73) or R31375_P33 (SEQ ID NO:74) protein in the sample.","It is another object of the invention to provide methods for detecting a disease, diagnosing a disease, monitoring disease progression or treatment efficacy or relapse of a disease, or selecting a therapy for a disease, comprising detecting expression of a AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), or AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73) or R31375_P33 (SEQ ID NO:74).","In a related object the detected diseases will include cancers such as lung cancer, ovarian cancer, colon cancer, as well as other non-solid and solid tumors, sarcomas, hematological malignancies including but not limited to acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, cancer of the breast, prostate, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, brain and wherein the cancer is non-metastatic, invasive or metastatic.","With regard to lung cancer, the disease is selected from the group consisting of non-metastatic, invasive or metastatic lung cancer; squamous cell lung carcinoma, lung adenocarcinoma, carcinoid, small cell lung cancer or non-small cell lung cancer; detection of overexpression in lung metastasis (vs. primary tumor); detection of overexpression in lung cancer, for example non small cell lung cancer, for example adenocarcinoma, squamous cell cancer or carcinoid, or large cell carcinoma; identification of a metastasis of unknown origin which originated from a primary lung cancer; assessment of a malignant tissue residing in the lung that is from a non-lung origin, including but not limited to: osteogenic and soft tissue sarcomas; colorectal, uterine, cervix and corpus tumors; head and neck, breast, testis and salivary gland cancers; melanoma; and bladder and kidney tumors; distinguishing between different types of lung cancer, therefore potentially affecting treatment choice (e.g. small cell vs. non small cell tumors); analysis of unexplained dyspnea and\/or chronic cough and\/or hemoptysis; differential diagnosis of the origin of a pleural effusion; diagnosis of conditions which have similar symptoms, signs and complications as lung cancer and where the differential diagnosis between them and lung cancer is of clinical importance including but not limited to: non-malignant causes of lung symptoms and signs, including but not limited to: lung lesions and infiltrates, wheeze, stridor, tracheal obstruction, esophageal compression, dysphagia, recurrent laryngeal nerve paralysis, hoarseness, phrenic nerve paralysis with elevation of the hemidiaphragm and Horner syndrome; or detecting a cause of any condition suggestive of a malignant tumor including but not limited to anorexia, cachexia, weight loss, fever, hypercalcemia, hypophosphatemia, hyponatremia, syndrome of inappropriate secretion of antidiuretic hormone, elevated ANP, elevated ACTH, hypokalemia, clubbing, neurologic-myopathic syndromes and thrombophlebitis.","With regard to ovarian cancer, the compounds of the present invention can be used in the diagnosis, treatment or prognostic assessment of non-metastatic, invasive or metastatic ovarian cancer; correlating stage and malignant potential; identification of a metastasis of unknown origin which originated from a primary ovarian cancer; differential diagnosis between benign and malignant ovarian cysts; diagnosing a cause of infertility, for example differential diagnosis of various causes thereof; detecting of one or more non-ovarian cancer conditions that may elevate serum levels of ovary related markers, including but not limited to: cancers of the endometrium, cervix, fallopian tubes, pancreas, breast, lung and colon; nonmalignant conditions such as pregnancy, endometriosis, pelvic inflammatory disease and uterine fibroids; diagnosing conditions which have similar symptoms, signs and complications as ovarian cancer and where the differential diagnosis between them and ovarian cancer is of clinical importance including but not limited to: non-malignant causes of pelvic mass, including, but not limited to: benign (functional) ovarian cyst, uterine fibroids, endometriosis, benign ovarian neoplasms and inflammatory bowel lesions; determining a cause of any condition suggestive of a malignant tumor including but not limited to anorexia, cachexia, weight loss, fever, hypercalcemia, skeletal or abdominal pain, paraneoplastic syndrome, or ascites.","In another related object the detected diseases will include autoimmune and neoplastic disorders selected from the group consisting of Multiple sclerosis; Psoriasis; Rheumatoid arthritis; Systemic lupus erythematosus; Ulcerative colitis; Crohn's disease; immune disorders associated with graft transplantation rejection, benign lymphocytic angiitis, lupus erythematosus, Hashimoto's thyroiditis, primary myxedema, Graves' disease, pernicious anemia, autoimmune atrophic gastritis, Addison's disease, insulin dependent diabetes mellitis, good pasture's syndrome, myasthenia gravis, pemphigus, sympathetic ophthalmia, autoimmune uveitis, autoimmune hemolytic anemia, idiopathic thrombocytopenia, primary biliary cirrhosis, chronic action hepatitis, ulceratis colitis, Sjogren's syndrome, rheumatic disease, polymyositis, scleroderma, mixed connective tissue disease, inflammatory rheumatism, degenerative rheumatism, extra-articular rheumatism, collagen diseases, chronic polyarthritis, psoriasis arthropathica, ankylosing spondylitis, juvenile rheumatoid arthritis, periarthritis humeroscapularis, panarteriitis nodosa, progressive systemic scleroderma, arthritis uratica, dermatomyositis, muscular rheumatism, myositis, myogelosis and chondrocalcinosis.","In another related object the detected diseases will include rejection of any organ transplant and\/or Graft versus host disease.","In a related aspect the foregoing assays will detect cells affected by the disease using the antibody that binds specifically to the AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), or AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73) or R31375_P33 (SEQ ID NO:74) protein wherein the assays may be effected in vitro or in vivo, and include RIA, ELISA, fluorimetric assays, FACS, slot blot, Western blot, immunohistochemical assays, radioimaging assays and the like. In some embodiments, this invention provides a method for diagnosing a disease in a subject, comprising detecting in the subject or in a sample obtained from said subject at least one polypeptide or polynucleotide selected from the group consisting of:","a polypeptide comprising an amino acid sequence as set forth in any one of SEQ ID NOs: 11-16, 21-34, 35-40, 43-44, 48-50, 70-76, 138-151, 296, 298-302;","a polypeptide comprising a bridge, edge portion, tail or head portion, of any one of SEQ. ID NOs: 284-295, or a homologue or a fragment thereof;","a polynucleotide comprising a nucleic acid sequence as set forth in any one of SEQ ID NOs: 1-10, 17-20, 25-34, 41-42, 45-46, 51-69;","a polynucleotide comprising a nucleic acid sequence encoding a polypeptide comprising a bridge, edge portion, tail or head portion, of any one of SEQ. ID NOs: 284-295;","an oligonucleotide having a nucleic acid sequence as set forth in SEQ. ID NOs: 187, 190, 193, 196, 199, 202, 205, 208, 211, 214, 217, 220, 223, 226, 229, 232, 235, 238, 241, 244, 247, 250, 253.","According to further embodiment, detecting a polypeptide of the invention comprises employing an antibody capable of specifically binding to at least one epitope of a polypeptide comprising an amino acid sequence of a polypeptide comprising a bridge, edge portion, tail, or head portion of any one of SEQ. ID NOs: 284-295. According to one embodiment, detecting the presence of the polypeptide or polynucleotide is indicative of the presence of the disease and\/or its severity and\/or its progress. According to another embodiment, a change in the expression and\/or the level of the polynucleotide or polypeptide compared to its expression and\/or level in a healthy subject or a sample obtained therefrom is indicative of the presence of the disease and\/or its severity and\/or its progress. According to a further embodiment, a change in the expression and\/or level of the polynucleotide or polypeptide compared to its level and\/or expression in said subject or in a sample obtained therefrom at earlier stage is indicative of the progress of the disease. According to still further embodiment, detecting the presence and\/or relative change in the expression and\/or level of the polynucleotide or polypeptide is useful for selecting a treatment and\/or monitoring a treatment of the disease.","According to one embodiment, detecting a polynucleotide of the invention comprises employing a primer pair, comprising a pair of isolated oligonucleotides capable of specifically hybridizing to at least a portion of a polynucleotide having a nucleic acid sequence as set forth in SEQ. ID NOs: 187, 190, 193, 196, 199, 202, 205, 208, 211, 214, 217, 220, 223, 226, 229, 232, 235, 238, 241, 244, 247, 250, 253, or polynucleotides homologous thereto.","According to another embodiment, detecting a polynucleotide of the invention comprises employing a primer pair, comprising a pair of isolated oligonucleotides as set forth in SEQ. ID NOs:185-186, 188-189, 191-192, 194-195, 197-198, 200-201, 203-204, 206-207, 209-210, 212-213, 215-216, 218-219, 221-222, 224-225, 227-228, 230-231, 233-234, 236-237, 239-240, 242-243, 245-246, 248-249, 251-252.","The invention also includes the following specific embodiments.","In one embodiment the invention includes an isolated polypeptide selected from AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA4248391_P11 (SEQ ID NO:24), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74) or a fragment or variant thereof that possesses at least 95, 96, 97, 98 or 99% sequence identity therewith.","In another embodiment the invention includes a fragment or conjugate comprising any one of the foregoing polypeptides.","In another embodiment the invention includes any one of the foregoing polypeptides fused to an immunoglobulin domain.","In another embodiment the invention includes any of the foregoing polypeptides attached to a detectable or therapeutic moiety.","In another embodiment the invention includes a nucleic acid sequence encoding any of the foregoing polypeptides.","In another embodiment the invention includes any of the nucleic acid sequences selected from AI581519_T10 (SEQ ID NO:9), AI581519_T11 (SEQ ID NO:10), AA4248391_T7 (SEQ ID NO:20), H686541_T8 (SEQ ID NO:28), H686541_T15 (SEQ ID NO:29), H686541_T16 (SEQ ID NO:30), H686541_T17 (SEQ ID NO:31), H686541_T18 (SEQ ID NO:32), H686541_T19 (SEQ ID NO:33), H686541_T20 (SEQ ID NO:34), AI216611_T1 (SEQ ID NO:42), H190111_T8 (SEQ ID NO:45), H190111_T9 (SEQ ID NO:46), R31375_T19 (SEQ ID NO:65); R31375_T25 (SEQ ID NO:66), R31375_T26 (SEQ ID NO:67), R31375_T29 (SEQ ID NO:68), R31375_T39 (SEQ ID NO:69), or a fragment or variant and conjugates containing that possesses at least 95, 96, 97, 98 or 99% sequence identity therewith.","In another embodiment the invention includes an isolated VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 ectodomain polypeptide, or fragment or conjugate thereof.","In another embodiment the invention includes any of the foregoing polypeptides, comprising a sequence of amino acid residues having at least 95, 96, 97, 98 or 99% sequence identity with amino acid residues 23-234 of AI581519_P3 (SEQ ID NO:11), corresponding to amino acid sequence depicted in SEQ ID NO:138, or amino acid residues 23-270 of AI581519_P4 (SEQ ID NO:12), corresponding to amino acid sequence depicted in SEQ ID NO:139, or amino acid residues 23-296 of AI581519_P5 (SEQ ID NO:13), corresponding to amino acid sequence depicted in SEQ ID NO:140, or amino acid residues 23-193 of AI581519_P7 (SEQ ID NO:14), corresponding to amino acid sequence depicted in SEQ ID NO:141, or amino acid residues 23-203 of AI581519_P9 (SEQ ID NO:15), corresponding to amino acid sequence depicted in SEQ ID NO:142, or amino acid residues 23-231 of AI581519_P10 (SEQ ID NO:16), corresponding to amino acid sequence depicted in SEQ ID NO:143, or residues 26-293 of AI581519_P5 (SEQ ID NO:13), corresponding to amino acid sequence depicted in SEQ ID NO:302, or amino acid residues 24-162 of AA424839_P3 (SEQ ID NO:22), or AA424839_P5 (SEQ ID NO:21), corresponding to amino acid sequence depicted in SEQ ID NO:75, or amino acid residues 24-456 of AA424839_P7 (SEQ ID NO:23), corresponding to amino acid sequence depicted in SEQ ID NO:76, or amino acid residues 24-105 of AA4248391_P11 (SEQ ID NO:24), corresponding to amino acid sequence depicted in SEQ ID NO:296, or residues 50-160 of AA4248391_P3 (SEQ ID NO:22), corresponding to amino acid sequence depicted in SEQ ID NO:301, or amino acid residues 38-349 of H686541_P2 (SEQ ID NO:35), corresponding to amino acid sequence depicted in SEQ ID NO:144, or residues 19-337 of H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), or H686541_P14 (SEQ ID NO:40), corresponding to amino acid sequence depicted in SEQ ID NO:145, or residues 1-335 of the sequences H686541_P5 (SEQ ID NO:36), corresponding to amino acid sequence depicted in SEQ ID NO:300, or amino acid residues 29-147 of the sequence AI216611_P0 (SEQ ID NO:43) or AI216611_P1 (SEQ ID NO:44), or residues 1-145 of the sequence AI216611_P0 (SEQ ID NO:43), corresponding to amino acid sequence depicted in SEQ ID NO:298, corresponding to amino acid sequence depicted in SEQ ID NO:146, or amino acid residues 21-186 of H190111_P8 (SEQ ID NO:48), corresponding to amino acid sequence depicted in SEQ ID NO:147, or residues 21-169 of H190111_P9 (SEQ ID NO:50), corresponding to amino acid sequence depicted in SEQ ID NO:148, or residues 1-184 of the sequence H190111_P8 (SEQ ID NO:48), corresponding to amino acid sequence depicted in SEQ ID NO:299, or amino acid residues 21-36 of R31375_P0 (SEQ ID NO:70) or R31375_P31 (SEQ ID NO:73), corresponding to amino acid sequence depicted in SEQ ID NO:149, or residues 21-65 of R31375_P14 (SEQ ID NO:72), corresponding to amino acid sequence depicted in SEQ ID NO:150, or residues 21-25 of R31375_P33 (SEQ ID NO:74), corresponding to amino acid sequence depicted in SEQ ID NO:151, or residues 1-63 of the sequence R31375_P14 (SEQ ID NO:72), corresponding to amino acid sequence depicted in SEQ ID NO:297.","In another embodiment the invention includes any of the foregoing polypeptides, comprising the extracellular domain of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73) or R31375_P33 (SEQ ID NO:74).","In another embodiment the invention includes any of the foregoing polypeptides, attached to a detectable or therapeutic moiety.","In another embodiment the invention includes any of the foregoing nucleic acid sequences encoding any one of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 ectodomain polypeptides and conjugates containing.","In another embodiment the invention includes an expression vector containing any of the foregoing nucleic acid sequences.","In another embodiment the invention includes a host cell comprising the foregoing expression vector or a virus containing a nucleic acid sequence encoding the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 ectodomain polypeptide, or fragment or conjugate thereof, wherein the cell expresses the polypeptide encoded by the DNA segment.","In another embodiment the invention includes a method of producing anyone of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 ectodomain polypeptides, or fragment or conjugate thereof, comprising culturing the foregoing host cell, wherein the cell expresses the polypeptide encoded by the DNA segment or nucleic acid and recovering said polypeptide.","In another embodiment the invention includes any of the foregoing isolated soluble VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 ectodomain wherein said polypeptide blocks or inhibits the interaction of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), or a fragment or variant thereof with a corresponding functional counterpart.","In another embodiment the invention includes the foregoing isolated soluble VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 ectodomains, wherein said polypeptide replaces or augments the interaction of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73) or R31375_P33 (SEQ ID NO:74), or a fragment or variant or conjugate thereof with a corresponding functional counterpart.","In another embodiment the invention includes a fusion protein comprising any of the foregoing isolated soluble VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 ectodomain joined to a non-VSIG1, non-ILDR1, non-LOC253012, non-AI216611, non-C1ORF32, non-FXYD3 protein sequence, correspondingly.","In another embodiment the invention includes any of the foregoing fusion proteins, wherein the non-VSIG1, non-ILDR1, non-LOC253012, non-AI216611, non-C1ORF32, non-FXYD3 protein is at least a portion of an immunoglobulin molecule.","In another embodiment the invention includes any of the foregoing fusion proteins, wherein a polyalkyl oxide moiety such as polyethylene glycol is attached to the polypeptide.","In another embodiment the invention includes any of the foregoing fusion proteins, wherein the immunoglobulin heavy chain constant region is an Fc fragment.","In another embodiment the invention includes any one of the protein sequences of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 and FXYD3 ECDs fused to mouse Fc, as set forth in any one of amino acid sequences as depicted in SEQ ID NOs: 103-108, or nucleic acid sequences encoding the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 and FXYD3 ECDs fused to mouse Fc. The invention further includes the nucleic acid sequences encoding the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 and FXYD3 ECDs fused to mouse Fc, as set forth in any one of nucleic acid sequences depicted in SEQ ID NOs:97-102.","In another embodiment the invention includes any of the foregoing fusion proteins wherein the immunoglobulin heavy chain constant region is an isotype selected from the group consisting of an IgG1, IgG2, IgG3, IgG4, IgM, IgE, IgA and IgD.","In another embodiment the invention includes any of the foregoing fusion proteins, wherein the polypeptide is fused to a VASP domain.","In another embodiment the invention includes any of the foregoing fusion proteins, wherein the fusion protein modulates lymphocyte activation.","In another embodiment the invention includes a pharmaceutical composition comprising any of the foregoing polynucleotide sequences and further comprising a pharmaceutically acceptable diluent or carrier.","In another embodiment the invention includes a pharmaceutical composition comprising the foregoing vector and further comprising a pharmaceutically acceptable diluent or carrier.","In another embodiment the invention includes a pharmaceutical composition comprising the foregoing host cell and further comprising a pharmaceutically acceptable diluent or carrier.","In another embodiment the invention includes a pharmaceutical composition comprising any of the foregoing VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 ectodomains and further comprising a pharmaceutically acceptable diluent or carrier.","In another embodiment the invention includes a pharmaceutical composition comprising any of the foregoing polypeptides and further comprising a pharmaceutically acceptable diluent or carrier.","In another embodiment the invention includes a pharmaceutical composition comprising the foregoing fusion protein and further comprising a pharmaceutically acceptable diluent or carrier.","In another embodiment the invention includes a method for treating or preventing cancer, comprising administering to a subject in need thereof a pharmaceutical composition comprising: a soluble molecule having the extracellular domain of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 polypeptide, or fragment or conjugate thereof; or polypeptide, comprising a sequence of amino acid residues having at least 95, 96, 97, 98 or 99% sequence identity with amino acid residues 23-234 of AI581519_P3 (SEQ ID NO:11), corresponding to amino acid sequence depicted in SEQ ID NO:138, or amino acid residues 23-270 of AI581519_P4 (SEQ ID NO:12), corresponding to amino acid sequence depicted in SEQ ID NO:139, or amino acid residues 23-296 of AI581519_P5 (SEQ ID NO:13), corresponding to amino acid sequence depicted in SEQ ID NO:140, or amino acid residues 23-193 of AI581519_P7 (SEQ ID NO:14), corresponding to amino acid sequence depicted in SEQ ID NO:141, or amino acid residues 23-203 of AI581519_P9 (SEQ ID NO:15), corresponding to amino acid sequence depicted in SEQ ID NO:142, or amino acid residues 23-231 of AI581519_P10 (SEQ ID NO:16), corresponding to amino acid sequence depicted in SEQ ID NO:143, or residues 26-293 of AI581519_P5 (SEQ ID NO:13), corresponding to amino acid sequence depicted in SEQ ID NO:302, or amino acid residues 24-162 of AA424839_P3 (SEQ ID NO:22), or AA424839_P5 (SEQ ID NO:21), corresponding to amino acid sequence depicted in SEQ ID NO:75, or amino acid residues 24-456 of AA424839_P7 (SEQ ID NO:23), corresponding to amino acid sequence depicted in SEQ ID NO:76, or amino acid residues 24-105 of AA4248391_P11 (SEQ ID NO:24), corresponding to amino acid sequence depicted in SEQ ID NO:296, or residues 50-160 of AA4248391_P3 (SEQ ID NO:22), corresponding to amino acid sequence depicted in SEQ ID NO:301, or amino acid residues 38-349 of H686541_P2 (SEQ ID NO:35), corresponding to amino acid sequence depicted in SEQ ID NO:144, or residues 19-337 of H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), or H686541_P14 (SEQ ID NO:40), corresponding to amino acid sequence depicted in SEQ ID NO:145, or residues 1-335 of the sequences H686541_P5 (SEQ ID NO:36), corresponding to amino acid sequence depicted in SEQ ID NO:300, or amino acid residues 29-147 of the sequence AI216611_P0 (SEQ ID NO:43) or AI216611_P1 (SEQ ID NO:44), or residues 1-145 of the sequence AI216611_P0 (SEQ ID NO:43), corresponding to amino acid sequence depicted in SEQ ID NO:298, corresponding to amino acid sequence depicted in SEQ ID NO:146, or amino acid residues 21-186 of H190111_P8 (SEQ ID NO:48), corresponding to amino acid sequence depicted in SEQ ID NO:147, or residues 21-169 of H190111_P9 (SEQ ID NO:50), corresponding to amino acid sequence depicted in SEQ ID NO:148, or residues 1-184 of the sequence H190111_P8 (SEQ ID NO:48), corresponding to amino acid sequence depicted in SEQ ID NO:299, or amino acid residues 21-36 of R31375_P0 (SEQ ID NO:70) or R31375_P31 (SEQ ID NO:73), corresponding to amino acid sequence depicted in SEQ ID NO:149, or residues 21-65 of R31375_P14 (SEQ ID NO:72), corresponding to amino acid sequence depicted in SEQ ID NO:150, or residues 21-25 of R31375_P33 (SEQ ID NO:74), corresponding to amino acid sequence depicted in SEQ ID NO:151, or residues 1-63 of the sequence R31375_P14 (SEQ ID NO:72), corresponding to amino acid sequence depicted in SEQ ID NO:297, or a nucleic acid sequence encoding the same.","In another embodiment the invention includes the foregoing method, wherein the cancer is selected from a group consisting of hematological malignancies such as acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, and soft or solid tumors such as cancer of breast, prostate, lung, ovary, colon, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, brain and wherein the cancer is non-metastatic, invasive or metastatic.","In another embodiment the invention includes the foregoing method wherein the cancer is selected from the group consisting of lung cancer, ovarian cancer or colon cancer, and wherein the lung cancer, the ovarian cancer or the colon cancer is non-metastatic, invasive or metastatic.","In another embodiment the invention includes a method for treating or preventing immune related conditions, such as autoimmune diseases or transplant rejection, comprising administering to a subject in need thereof a pharmaceutical composition comprising: a soluble molecule having the extracellular domain of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 polypeptide, or fragment or conjugate thereof; or polypeptide, comprising a sequence of amino acid residues having at least 95, 96, 97, 98 or 99% sequence identity with amino acid residues 23-234 of AI581519_P3 (SEQ ID NO:11), corresponding to amino acid sequence depicted in SEQ ID NO:138, or amino acid residues 23-270 of AI581519_P4 (SEQ ID NO:12), corresponding to amino acid sequence depicted in SEQ ID NO:139, or amino acid residues 23-296 of AI581519_P5 (SEQ ID NO:13), corresponding to amino acid sequence depicted in SEQ ID NO:140, or amino acid residues 23-193 of AI581519_P7 (SEQ ID NO:14), corresponding to amino acid sequence depicted in SEQ ID NO:141, or amino acid residues 23-203 of AI581519_P9 (SEQ ID NO:15), corresponding to amino acid sequence depicted in SEQ ID NO:142, or amino acid residues 23-231 of AI581519_P10 (SEQ ID NO:16), corresponding to amino acid sequence depicted in SEQ ID NO:143, or residues 26-293 of AI581519_P5 (SEQ ID NO:13), corresponding to amino acid sequence depicted in SEQ ID NO:302, or amino acid residues 24-162 of AA424839_P3 (SEQ ID NO:22), or AA424839_P5 (SEQ ID NO:21), corresponding to amino acid sequence depicted in SEQ ID NO:75, or amino acid residues 24-456 of AA424839_P7 (SEQ ID NO:23), corresponding to amino acid sequence depicted in SEQ ID NO:76, or amino acid residues 24-105 of AA4248391_P11 (SEQ ID NO:24), corresponding to amino acid sequence depicted in SEQ ID NO:296, or residues 50-160 of AA4248391_P3 (SEQ ID NO:22), corresponding to amino acid sequence depicted in SEQ ID NO:301, or amino acid residues 38-349 of H686541_P2 (SEQ ID NO:35), corresponding to amino acid sequence depicted in SEQ ID NO:144, or residues 19-337 of H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), or H686541_P14 (SEQ ID NO:40), corresponding to amino acid sequence depicted in SEQ ID NO:145, or residues 1-335 of the sequences H686541_P5 (SEQ ID NO:36), corresponding to amino acid sequence depicted in SEQ ID NO:300, or amino acid residues 29-147 of the sequence AI216611_P0 (SEQ ID NO:43) or AI216611_P1 (SEQ ID NO:44), or residues 1-145 of the sequence AI216611_P0 (SEQ ID NO:43), corresponding to amino acid sequence depicted in SEQ ID NO:298, corresponding to amino acid sequence depicted in SEQ ID NO:146, or amino acid residues 21-186 of H190111_P8 (SEQ ID NO:48), corresponding to amino acid sequence depicted in SEQ ID NO:147, or residues 21-169 of H190111_P9 (SEQ ID NO:50), corresponding to amino acid sequence depicted in SEQ ID NO:148, residues 1-184 of the sequence H190111_P8 (SEQ ID NO:48), corresponding to amino acid sequence depicted in SEQ ID NO:299, or residues 21-36 of R31375_P0 (SEQ ID NO:70) or R31375_P31 (SEQ ID NO:73), corresponding to amino acid sequence depicted in SEQ ID NO:149, or residues 21-65 of R31375_P14 (SEQ ID NO:72), corresponding to amino acid sequence depicted in SEQ ID NO:150, or residues 21-25 of R31375_P33 (SEQ ID NO:74), corresponding to amino acid sequence depicted in SEQ ID NO:151, or residues 1-63 of R31375_P14 (SEQ ID NO:72), corresponding to amino acid sequence depicted in SEQ ID NO:297, or a nucleic acid sequence encoding the same.","In another embodiment the invention includes the foregoing method, wherein the autoimmune diseases are selected from a group consisting of multiple sclerosis; psoriasis; rheumatoid arthritis; systemic lupus erythematosus; ulcerative colitis; Crohn's disease; immune disorders associated with graft transplantation rejection; benign lymphocytic angiitis, lupus erythematosus, Hashimoto's thyroiditis, primary myxedema, Graves' disease, pernicious anemia, autoimmune atrophic gastritis, Addison's disease, insulin dependent diabetes mellitis, Good pasture's syndrome, myasthenia gravis, pemphigus, sympathetic ophthalmia, autoimmune uveitis, autoimmune hemolytic anemia, idiopathic thrombocytopenia, primary biliary cirrhosis, chronic action hepatitis, ulceratis colitis, Sjogren's syndrome, rheumatic disease, polymyositis, scleroderma, mixed connective tissue disease, inflammatory rheumatism, degenerative rheumatism, extra-articular rheumatism, collagen diseases, chronic polyarthritis, psoriasis arthropathica, ankylosing spondylitis, juvenile rheumatoid arthritis, periarthritis humeroscapularis, panarteriitis nodosa, progressive systemic scleroderma, arthritis uratica, dermatomyositis, muscular rheumatism, myositis, myogelosis and chondrocalcinosis.","In another embodiment the invention includes the foregoing method, wherein the immune related disorders are selected from transplant rejection or graft versus host disease.","In another embodiment the invention includes an siRNA, antisense RNA, or ribozyme that binds the transcript encoding any one of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 polypeptides, selected from AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), or a fragment or a variant thereof, and inhibits its expression.","In another embodiment the invention includes a polyclonal or monoclonal antibody that specifically binds and\/or modulates an activity elicited by any one of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 polypeptides, selected from AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), or a fragment or a variant thereof and conjugates containing.","In another embodiment the invention includes a monoclonal or polyclonal antibody or an antigen binding fragment thereof comprising an antigen binding site that binds specifically to any one of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 polypeptides comprised in AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), or fragment or variant thereof that is at least 80% identical thereto.","In another embodiment the invention includes any of the foregoing antibodies or fragments thereof, wherein said antibody blocks or inhibits the interaction of one of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), or a fragment or variant thereof with a counterpart activity or function.","In another embodiment the invention includes any of the foregoing antibodies or fragments wherein said antibody replaces or augments the interaction of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), or a fragment or variant thereof with a counterpart function or activity.","In another embodiment the invention includes a method for modulating lymphocyte activity, comprising contacting a AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74) positive lymphocyte with a bioactive agent capable of modulating VSIG1-mediated, ILDR1-mediated, LOC253012-mediated, AI216611-mediated, C1ORF32-mediated, or FXYD3-mediated signaling in an amount effective to modulate at least one lymphocyte activity.","In another embodiment the invention includes the foregoing method, wherein said agent comprises an antagonist of VSIG1-mediated, ILDR1-mediated, LOC253012-mediated, AI216611-mediated, C1ORF32-mediated signaling, or FXYD3-mediated signaling and wherein said contacting inhibits the attenuation of lymphocyte activity mediated by such signaling.","In another embodiment the invention includes the foregoing method, wherein said contacting increases lymphocyte activity.","In another embodiment the invention includes the foregoing method wherein said antagonist comprises a blocking agent capable of interfering with the functional interaction of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 antigen and its counterpart.","In another embodiment the invention includes the foregoing antibody or fragment which is suitable for treatment or prevention of cancer by modulating the activity of any one of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins in a B7-like co-stimulatory system.","In another embodiment the invention includes the foregoing method wherein the administered antibody or fragment inhibits negative stimulation of T cell activity against cancer cells.","In another embodiment the invention includes any of the foregoing antibodies or fragments, wherein the cancer is selected from the group consisting of hematological malignancies such as acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, and soft tissue or solid tumors such as cancer of breast, prostate, lung, ovary, colon, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, brain and wherein the cancer is non-metastatic, invasive or metastatic.","In another embodiment the invention includes any of the foregoing antibodies or fragments, which are suitable for treatment or prevention of immune related disorders, such as autoimmune diseases or transplant rejection, by modulating the activity of anyone of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins in a B7-like co-stimulatory system.","In another embodiment the invention includes any of the foregoing antibodies or fragments, which are suitable for treating an autoimmune disease selected from multiple sclerosis; psoriasis; rheumatoid arthritis; Systemic lupus erythematosus; ulcerative colitis; Crohn's disease, immune disorders associated with graft transplantation rejection, benign lymphocytic angiitis, lupus erythematosus, Hashimoto's thyroiditis, primary myxedema, Graves' disease, pernicious anemia, autoimmune atrophic gastritis, Addison's disease, insulin dependent diabetes mellitus, Good pasture's syndrome, myasthenia gravis, pemphigus, sympathetic ophthalmia, autoimmune uveitis, autoimmune hemolytic anemia, idiopathic thrombocytopenia, primary biliary cirrhosis, chronic action hepatitis, Sjogren's syndrome, rheumatic disease, polymyositis, scleroderma, mixed connective tissue disease, inflammatory rheumatism, degenerative rheumatism, extra-articular rheumatism, collagen diseases, chronic polyarthritis, psoriasis arthropathica, ankylosing spondylitis, juvenile rheumatoid arthritis, periarthritis humeroscapularis, panarteriitis nodosa, progressive systemic scleroderma, arthritis uratica, dermatomyositis, muscular rheumatism, myositis, myogelosis and chondrocalcinosis.","In another embodiment the invention includes any of the foregoing antibodies or fragments, suitable for treating transplant rejection or graft versus host disease.","In another embodiment the invention includes any of the foregoing antibodies or fragments, that specifically binds to amino-acids: 23-234 of AI581519_P3 (SEQ ID NO:11), corresponding to amino acid sequence depicted in SEQ ID NO:138, or amino acid residues 23-270 of AI581519_P4 (SEQ ID NO:12), corresponding to amino acid sequence depicted in SEQ ID NO:139, or amino acid residues 23-296 of AI581519_P5 (SEQ ID NO:13), corresponding to amino acid sequence depicted in SEQ ID NO:140, or amino acid residues 23-193 of AI581519_P7 (SEQ ID NO:14), corresponding to amino acid sequence depicted in SEQ ID NO:141, or amino acid residues 23-203 of AI581519_P9 (SEQ ID NO:15), corresponding to amino acid sequence depicted in SEQ ID NO:142, or amino acid residues 23-231 of AI581519_P10 (SEQ ID NO:16), corresponding to amino acid sequence depicted in SEQ ID NO:143, or residues 26-293 of AI581519_P5 (SEQ ID NO:13), corresponding to amino acid sequence depicted in SEQ ID NO:302, or amino acid residues 24-162 of AA424839_P3 (SEQ ID NO:22), or AA424839_P5 (SEQ ID NO:21), corresponding to amino acid sequence depicted in SEQ ID NO:75, or amino acid residues 24-456 of AA424839_P7 (SEQ ID NO:23), corresponding to amino acid sequence depicted in SEQ ID NO:76, or amino acid residues 24-105 of AA4248391_P11 (SEQ ID NO:24), corresponding to amino acid sequence depicted in SEQ ID NO:296, or residues 50-160 of AA4248391_P3 (SEQ ID NO:22), corresponding to amino acid sequence depicted in SEQ ID NO:301, or amino acid residues 38-349 of H686541_P2 (SEQ ID NO:35), corresponding to amino acid sequence depicted in SEQ ID NO:144, or residues 19-337 of H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), or H686541_P14 (SEQ ID NO:40), corresponding to amino acid sequence depicted in SEQ ID NO:145, or residues 1-335 of the sequences H686541_P5 (SEQ ID NO:36), corresponding to amino acid sequence depicted in SEQ ID NO:300, or amino acid residues 29-147 of the sequence AI216611_P0 (SEQ ID NO:43) or AI216611_P1 (SEQ ID NO:44), or residues 1-145 of the sequence AI216611_P0 (SEQ ID NO:43), corresponding to amino acid sequence depicted in SEQ ID NO:298, corresponding to amino acid sequence depicted in SEQ ID NO:146, or amino acid residues 21-186 of H190111_P8 (SEQ ID NO:48), corresponding to amino acid sequence depicted in SEQ ID NO:147, or residues 21-169 of H190111_P9 (SEQ ID NO:50), corresponding to amino acid sequence depicted in SEQ ID NO:148, or residues 1-184 of the sequence H190111_P8 (SEQ ID NO:48), corresponding to amino acid sequence depicted in SEQ ID NO:299, or amino acid residues 21-36 of R31375_P0 (SEQ ID NO:70) or R31375_P31 (SEQ ID NO:73), corresponding to amino acid sequence depicted in SEQ ID NO:149, or residues 21-65 of R31375_P14 (SEQ ID NO:72), corresponding to amino acid sequence depicted in SEQ ID NO:150, or residues 21-25 of R31375_P33 (SEQ ID NO:74), corresponding to amino acid sequence depicted in SEQ ID NO:151, or residues 1-63 of the sequence R31375_P14 (SEQ ID NO:72), corresponding to amino acid sequence depicted in SEQ ID NO:297, or a variant or fragment or an epitope thereof.","In another embodiment the invention includes any of the foregoing antibodies or fragments, wherein the antigen binding site contains from about 3-7 contiguous or non-contiguous amino acids, more typically at least 5 contiguous or non-contiguous amino acids. These binding sites include conformational and non-conformational epitopes.","In another embodiment the invention includes any of the foregoing antibodies or fragments, wherein the antibody is a fully human antibody.","In another embodiment the invention includes any of the foregoing antibodies or fragments, wherein the antibody is a chimeric antibody.","In another embodiment the invention includes the foregoing antibodies or fragments wherein the antibody is a humanized or primatized antibody.","In another embodiment the invention includes any of the foregoing antibodies or fragments, wherein the fragment is selected from the group consisting of Fab, Fab\u2032, F(ab\u2032)2, F(ab\u2032), F(ab), Fv or scFv fragment and minimal recognition unit.","In another embodiment the invention includes any of the foregoing antibodies or fragments, wherein the antibody or fragment is coupled to a detectable marker, or to an effector moiety.","In another embodiment the invention includes any of the foregoing antibodies or fragments, wherein the effector moiety is an enzyme, a toxin, a therapeutic agent, or a chemotherapeutic agent.","In another embodiment the invention includes any of the foregoing antibodies or fragments, wherein the detectable marker is a radioisotope, a metal chelator, an enzyme, a fluorescent compound, a bioluminescent compound or a chemiluminescent compound.","In another embodiment the invention includes a pharmaceutical composition that comprises any of the foregoing antibodies or a fragment thereof.","In another embodiment the invention includes a pharmaceutical composition that comprises the foregoing antibodies or a fragment thereof.","In another embodiment the invention includes a method of inducing or enhancing an immune response, comprising administering to a patient in need thereof any of the foregoing antibodies or fragments and detecting induction or enhancement of said immune response.","In another embodiment the invention includes a method for potentiating a secondary immune response to an antigen in a patient, which method comprises administering effective amounts any of the foregoing antibodies or fragments.","In another embodiment the invention includes the foregoing method, wherein the antigen is preferably a cancer antigen, a viral antigen or a bacterial antigen, and the patient has preferably received treatment with an anticancer vaccine or a viral vaccine.","In another embodiment the invention includes a method of treating a patient with a VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 positive malignancy, comprising administering to the patient an effective amount of any of the foregoing antibodies or fragments.","In another embodiment the invention includes the foregoing method further comprising co-administering a chemotherapeutic agent.","In another embodiment the invention includes the foregoing method, wherein said malignancy is selected from a group consisting of hematological malignancies such as acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, and soft or solid tumors such as cancer of breast, prostate, lung, ovary, colon, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, brain and wherein the cancer is non-metastatic, invasive or metastatic.","In another embodiment the invention includes the foregoing method, wherein said malignancy is selected from the group consisting of lung cancer, ovarian cancer, colon cancer, and wherein the lung cancer, the ovarian cancer or the colon cancer is non-metastatic, invasive or metastatic.","In another embodiment the invention includes an assay for detecting the presence of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), or a fragment or variant thereof in a biological sample comprising contacting the sample with an antibody of any one of the foregoing, and detecting the binding of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), or a fragment or variant thereof in the sample.","In another embodiment the invention includes a method for detecting a disease, diagnosing a disease, monitoring disease progression or treatment efficacy or relapse of a disease, or selecting a therapy for a disease, comprising detecting expression of a AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), or a fragment or variant thereof.","In another embodiment the invention includes the foregoing method wherein detecting expression AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), or a fragment or variant thereof is performed in vivo or in vitro.","In another embodiment the invention includes the foregoing method, wherein the disease is selected from lung cancer, ovarian cancer, or colon cancer, and wherein the lung cancer, the ovarian cancer or the colon cancer is non-metastatic, invasive or metastatic.","In another embodiment the invention includes the foregoing method, wherein the disease is multiple sclerosis; psoriasis; rheumatoid arthritis; Systemic lupus erythematosus; ulcerative colitis; Crohn's disease; immune disorders associated with graft transplantation rejection, benign lymphocytic angiitis, lupus erythematosus, Hashimoto's thyroiditis, primary myxedema, Graves' disease, pernicious anemia, autoimmune atrophic gastritis, Addison's disease, insulin dependent diabetes mellitis, good pasture's syndrome, myasthenia gravis, pemphigus, sympathetic ophthalmia, autoimmune uveitis, autoimmune hemolytic anemia, idiopathic thrombocytopenia, primary biliary cirrhosis, chronic action hepatitis, ulceratis colitis, Sjogren's syndrome, rheumatic disease, polymyositis, scleroderma, mixed connective tissue disease, inflammatory rheumatism, degenerative rheumatism, extra-articular rheumatism, collagen diseases, chronic polyarthritis, psoriasis arthropathica, ankylosing spondylitis, juvenile rheumatoid arthritis, periarthritis humeroscapularis, panarteriitis nodosa, progressive systemic scleroderma, arthritis uratica, dermatomyositis, muscular rheumatism, myositis, myogelosis or chondrocalcinosis.","In another embodiment the invention includes a method of inhibiting growth of cells that express a polypeptide selected from AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), or a fragment or variant thereof in a subject, comprising: administering to said subject any of the foregoing antibodies or fragments.","In another embodiment the invention includes a method of treating or preventing cancer comprising the administration of a therapeutically effective amount of an antibody or binding fragment that specifically binds the AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), or a fragment or variant thereof that possesses at least 80% sequence identity therewith.","In another embodiment the invention includes the foregoing method, wherein the cancer is selected from a group consisting of hematological malignancies such as acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, and soft or solid tumors such as cancer of breast, prostate, lung, ovary, colon, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, brain and wherein the cancer is non-metastatic, invasive or metastatic.","In another embodiment the invention includes the foregoing method, wherein the cancer is selected from the group consisting of lung cancer, ovarian cancer, or colon cancer, and wherein the lung cancer, the ovarian cancer or the colon cancer is non-metastatic, invasive or metastatic.","In another embodiment the invention includes the foregoing method wherein the antibody is a human, humanized or chimeric antibody or antigen binding fragment.","In another embodiment the invention includes the foregoing method wherein the antibody or fragment is attached directly or indirectly to an effector moiety.","In another embodiment the invention includes the foregoing method, wherein the effector is selected from a drug, toxin, radionuclide, fluorophore and an enzyme.","In another embodiment the invention includes a method for treating or preventing an immune disorder, such as autoimmune or transplant related disease, comprising administering to a patient a therapeutically effective amount of an antibody that specifically binds to AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74),","or a fragment or variant thereof that possesses at least 80% sequence identity therewith.","In another embodiment the invention includes the foregoing method, wherein the antibody has an antigen-binding region specific for the extracellular domain of any one of said VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 polypeptides.","In another embodiment the invention includes the foregoing method, wherein the antibody or fragment modulates the B7\/co-stimulatory system in a manner that inhibits positive stimulation of T cell activity that created an autoimmune effect.","In another embodiment the invention includes the foregoing method, wherein the treatment is combined with a moiety useful for treating autoimmune or transplant rejection conditions.","In another embodiment the invention includes the foregoing method, wherein the moiety is a cytokine antibody, cytokine receptor antibody, drug, or another immunomodulatory agent.","In another embodiment the invention includes the foregoing method, wherein the autoimmune diseases are selected from a group consisting of multiple sclerosis; psoriasis; rheumatoid arthritis; systemic lupus erythematosus; ulcerative colitis; Crohn's disease; immune disorders associated with graft transplantation rejection, benign lymphocytic angiitis, lupus erythematosus, Hashimoto's thyroiditis, primary myxedema, Graves' disease, pernicious anemia, autoimmune atrophic gastritis, Addison's disease, insulin dependent diabetes mellitis, Good pasture's syndrome, myasthenia gravis, pemphigus, sympathetic ophthalmia, autoimmune uveitis, autoimmune hemolytic anemia, idiopathic thrombocytopenia, primary biliary cirrhosis, chronic action hepatitis, ulceratis colitis, Sjogren's syndrome, rheumatic disease, polymyositis, scleroderma, mixed connective tissue disease, inflammatory rheumatism, degenerative rheumatism, extra-articular rheumatism, collagen diseases, chronic polyarthritis, psoriasis arthropathica, ankylosing spondylitis, juvenile rheumatoid arthritis, periarthritis humeroscapularis, panarteriitis nodosa, progressive systemic scleroderma, arthritis uratica, dermatomyositis, muscular rheumatism, myositis, myogelosis and chondrocalcinosis.","In another embodiment the invention includes the foregoing method wherein the immune disorder is transplant rejection or graft versus host disease.","In another embodiment the invention includes a method of using an antibody or antigen binding fragment that specifically binds AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), or a fragment or variant thereof for in vivo imaging of tumors or inflammatory sites characterized by the differential expression of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), or a fragment or variant thereof.","In another embodiment the invention includes the foregoing method which is used in assessing cancer prognosis or a treatment protocol.","In another embodiment the invention includes a method for screening for a disease in a subject, comprising detecting in the subject or in a sample obtained from said subject a polypeptide having a sequence at least 85% homologous to the amino acid sequence as set forth in any one of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), or with a polypeptide having a sequence comprising the extracellular domain of any one of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74).","In another embodiment the invention includes the foregoing method wherein screening for a disease comprises detecting the presence or severity of the disease, disorder or condition, or prognosis of the subject, or treatment selection for said subject, or treatment monitoring of said subject.","In another embodiment the invention includes the foregoing method, wherein the disease is a cancer, selected from the group consisting of lung cancer, ovarian cancer, colon cancer, and wherein the lung cancer, the ovarian cancer and the colon cancer is non-metastatic, invasive or metastatic.","In another embodiment the invention includes the foregoing method wherein the disease is autoimmune disease and is selected from multiple sclerosis; psoriasis; rheumatoid arthritis; systemic lupus erythematosus; ulcerative colitis; Crohn's disease; immune disorders associated with graft transplantation rejection; benign lymphocytic angiitis, lupus erythematosus, Hashimoto's thyroiditis, primary myxedema, Graves' disease, pernicious anemia, autoimmune atrophic gastritis, Addison's disease, insulin dependent diabetes mellitis, Good pasture's syndrome, myasthenia gravis, pemphigus, sympathetic ophthalmia, autoimmune uveitis, autoimmune hemolytic anemia, idiopathic thrombocytopenia, primary biliary cirrhosis, chronic action hepatitis, ulceratis colitis, Sjogren's syndrome, rheumatic disease, polymyositis, scleroderma, mixed connective tissue disease, inflammatory rheumatism, degenerative rheumatism, extra-articular rheumatism, collagen diseases, chronic polyarthritis, psoriasis arthropathica, ankylosing spondylitis, juvenile rheumatoid arthritis, periarthritis humeroscapularis, panarteriitis nodosa, progressive systemic scleroderma, arthritis uratica, dermatomyositis, muscular rheumatism, myositis, myogelosis and chondrocalcinosis.","In another embodiment the invention includes the foregoing method, wherein the detection is conducted by immunoassay.","In another embodiment the invention includes the foregoing method, wherein the immunoassay utilizes an antibody which specifically interacts with the polypeptide having a sequence at least 85% homologous to the amino acid sequence as set forth in any one of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), or with a polypeptide having a sequence comprising the extracellular domain of any one of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), and R31375_P33 (SEQ ID NO:74).","In another embodiment the invention includes an antibody specific to AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), and R31375_P33 (SEQ ID NO:74), or a fragment or variant thereof that elicits apoptosis or lysis of cancer cells that express said protein.","In another embodiment the invention includes any of the foregoing antibodies or fragments, wherein said apoptosis or lysis activity involves CDC or ADCC activity of the antibody.","In another embodiment the invention includes any of the foregoing antibodies or fragments, wherein the cancer cells are selected from a group consisting of hematological malignancies such as acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, and soft or solid tumors such as cancer of breast, prostate, lung, ovary, colon, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, and brain.","In another embodiment the invention includes any of the foregoing antibodies or fragments, wherein the cancer cells are lung, ovarian or colon cancer cells.","In another embodiment the invention relates to any of the foregoing isolated soluble VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 ectodomain polypeptides, wherein said polypeptide or a fragment or variant thereof is used as an anti-cancer vaccine for cancer immunotherapy.","In another embodiment the invention relates to any an isolated polypeptide comprising an amino acid sequence having at least 80%, 85%, 90%, 95, 96, 97, 98 or 99%, 100% homologous to the sequence as that set forth in any one of SEQ. ID NOs: 284-295, or a fragment thereof.","In another embodiment the invention relates to any an isolated polynucleotide, comprising an amplicon having a nucleic acid sequence selected from the group consisting of 187, 190, 193, 196, 199, 202, 205, 208, 211, 214, 217, 220, 223, 226, 229, 232, 235, 238, 241, 244, 247, 250, 253, or polynucleotides homologous thereto.","In another embodiment the invention relates to any a primer pair, comprising a pair of isolated oligonucleotides capable of amplifying the above mentioned amplicon.","The primer pair, comprising a pair of isolated oligonucleotides having a sequence selected from the group consisting of SEQ. ID NOs: 185-186, 188-189, 191-192, 194-195, 197-198, 200-201, 203-204, 206-207, 209-210, 212-213, 215-216, 218-219, 221-222, 224-225, 227-228, 230-231, 233-234, 236-237, 239-240, 242-243, 245-246, 248-249, and 251-252.","A method for screening for a disease, disorder or condition in a subject, comprising detecting in the subject or in a sample obtained from said subject a polynucleotide having a sequence at least 85% homologous to the nucleic acid sequence as set forth in any one of SEQ ID NOs:1-10, 17-20, 25-34, 41-42, 45-46, 51-69, 187, 190, 193, 196, 199, 202, 205, 208, 211, 214, 217, 220, 223, 226, 229, 232, 235, 238, 241, 244, 247, 250, and 253.","The method as above, wherein screening for a disease comprises detecting the presence or severity of the disease, disorder or condition, or prognosis of the subject, or treatment selection for said subject, or treatment monitoring.","The method as above, wherein the disease is a cancer, selected from the group consisting of lung cancer, colon cancer and ovarian cancer, and wherein the lung cancer, colon cancer and ovarian cancer is non-metastatic, invasive or metastatic.","The method as above, wherein the disease is autoimmune disease.","The method as above, wherein the detection is performed using an oligonucleotide pair capable of hybridizing to at least a portion of a nucleic acid sequence at least 85% homologous to the nucleic acid sequence set forth in SEQ. ID NO: 1-10, 17-20, 25-34, 41-42, 45-46, 51-69, 187, 190, 193, 196, 199, 202, 205, 208, 211, 214, 217, 220, 223, 226, 229, 232, 235, 238, 241, 244, 247, 250, and 253.","The method as above wherein the detection is performed using an oligonucleotide pair as set forth in any one of SEQ. ID NOs: 185-186, 188-189, 191-192, 194-195, 197-198, 200-201, 203-204, 206-207, 209-210, 212-213, 215-216, 218-219, 221-222, 224-225, 227-228, 230-231, 233-234, 236-237, 239-240, 242-243, 245-246, 248-249, and 251-252.","The present invention relates to any one of the antigens referred to as VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3, and its corresponding nucleic acid sequence, and portions and variants thereof and conjugates containing and the use thereof as a therapeutic or diagnostic target. In particular the invention uses this antigen and discrete portions thereof as a drug target for therapeutic small molecules, peptides, antibodies, antisense RNAs, siRNAs, ribozymes, and the like. More particularly the invention relates to diagnostic and therapeutic polyclonal and monoclonal antibodies and fragments thereof that bind VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 and portions and variants thereof, especially those that target the ectodomain or portions or variants thereof particularly human or chimeric monoclonal antibodies, that bind specifically to the antigen AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P33 (SEQ ID NO:74), and variants thereof including those that promote or inhibit activities elicited by VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3, including those relating to modulation of immune costimulation, e.g. B7 related costimulation.","In certain embodiments, the antibodies of the invention are derived from particular heavy and light chain germline sequences and\/or comprise particular structural features such as CDR regions comprising particular amino acid sequences. The invention provides isolated antibodies, methods of making such antibodies, immunoconjugates and bispecific molecules comprising such antibodies and pharmaceutical and diagnostic compositions containing the antibodies, immunoconjugates or bispecific molecules of the invention.","The invention also relates to in vitro and in vivo methods of using the antibodies and fragments, to detect VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3, as well as to treat diseases associated with expression of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3, such as malignancies that differentially express VSIG1. The invention further relates to methods of using the antibodies and fragments, specific for VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 to treat autoimmune disorders and transplant and graft versus host disease. Accordingly, the invention also provides methods of using the anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, anti-FXYD3 antibodies of the invention and other drugs that modulate VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 to treat malignancies for example, in the treatment of lung cancer, ovarian cancer, colon cancer, non-solid and solid tumors, sarcomas, hematological malignancies including but not limited to acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, cancer of the breast, prostate, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, brain and wherein the cancer may be non-metastatic, invasive or metastatic. The invention further provides methods of using the anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, anti-FXYD3 antibodies of the invention and other drugs that modulate VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 to treat non-malignant disorders such as immune disorders including but not limited to autoimmune diseases, transplant rejection and graft versus host disease. Preferably these antibodies will possess ADCC or CDC activity against target cells such as cancer cells.","Also, the invention relates to the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 antigen and portions thereof including soluble polypeptide conjugates containing the ectodomain of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 and\/or the corresponding DNAs or vectors or cells expressing same for use in immunotherapy. Further the invention provides vectors, cells containing and use thereof for the expression of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 antigen, as well as discrete portions and variants thereof. Also, the invention provides non-antibody based VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 modulatory agents such as peptides, antisense RNAs, siRNAs, carbohydrates, and other small molecules that specifically bind and\/or modulate a VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 related activity.","In order that the present invention may be more readily understood, certain terms are first defined. Additional definitions are set forth throughout the detailed description.","The terms VSIG1 refers to the protein encoded by any one of the AI581519_TO (SEQ ID NO:1), AI581519_T1 (SEQ ID NO:2), AI581519_T2 (SEQ ID NO:3), AI581519_T3 (SEQ ID NO:4), AI581519_T4 (SEQ ID NO:5), AI581519_T5 (SEQ ID NO:6), AI581519_T6 (SEQ ID NO:7), AI581519_T8 (SEQ ID NO:8), AI581519_T10 (SEQ ID NO:9), AI581519_T11 (SEQ ID NO:10) transcripts reported herein, particularly to proteins as set forth in any one of AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), and variants thereof that are differentially expressed e.g., in cancers such as lung cancer and ovarian cancer, wherein the cancer may be non-metastatic, invasive or metastatic as well as non-malignant disorders such as immune disorders including but not limited to autoimmune diseases, transplant rejection and graft versus host disease.","The term ILDR1 refers to the to the protein encoded by any one of the AA424839_TO (SEQ ID NO:17), AA424839_T2 (SEQ ID NO:18), AA424839_T4 (SEQ ID NO:19), AA4248391_T7 (SEQ ID NO:20) transcripts reported herein, particularly to proteins as set forth in any one of AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), AA4248391_P11 (SEQ ID NO:24), and variants thereof that are differentially expressed e.g., in cancer such as colon cancer and ovarian cancer wherein the cancer may be non-metastatic, invasive or metastatic as well as non-malignant disorders such as immune disorders including but not limited to autoimmune diseases, transplant rejection and graft versus host disease.","The term LOC253012 refers to the protein encoded by any one of the H686541_TO (SEQ ID NO:25), H686541_T4 (SEQ ID NO:26), H686541_T5 (SEQ ID NO:27), H686541_T8 (SEQ ID NO:28), H686541_T15 (SEQ ID NO:29), H686541_T16 (SEQ ID NO:30), H686541_T17 (SEQ ID NO:31), H686541_T18 (SEQ ID NO:32), H686541_T19 (SEQ ID NO:33), H686541_T20 (SEQ ID NO:34) transcripts reported herein, particularly to proteins as set forth in any one of H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), and variants thereof that are differentially expressed e.g., in cancers such as lung cancer, especially small cell lung carcinoma, wherein the cancer may be non-metastatic, invasive or metastatic as well as non-malignant disorders such as immune disorders including but not limited to autoimmune diseases, transplant rejection and graft versus host disease.","The term AI216611 refers to the protein encoded by any one of the AI216611_TO (SEQ ID NO:41), AI216611_T1 (SEQ ID NO:42) transcripts reported herein, particularly to proteins as set forth in any one of AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), and variants thereof that are differentially expressed e.g., in cancers such as non-solid and solid tumors, sarcomas, hematological malignancies including but not limited to acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, cancer of the breast, prostate, lung, ovary, colon, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, and brain and wherein the cancer may be non-metastatic, invasive or metastatic as well as non-malignant disorders such as immune disorders including but not limited to autoimmune diseases, transplant rejection and graft versus host disease.","The terms C1ORF32 refers to the protein encoded by any one of the H190111_T8 (SEQ ID NO:45), H190111_T9 (SEQ ID NO:46) transcripts reported herein, particularly to proteins as set forth in any one of H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), and variants thereof that are differentially expressed e.g., in cancers such as lung cancer, particularly lung small cell carcinoma, wherein the cancer may be non-metastatic, invasive or metastatic as well as non-malignant disorders such as immune disorders including but not limited to autoimmune diseases, transplant rejection and graft versus host disease.","The term FXYD3 refers to the protein encoded by any one of the R31375_TO (SEQ ID NO:51); R31375_T1 (SEQ ID NO:52); R31375_T10 (SEQ ID NO:61); R31375_T11 (SEQ ID NO:62); R31375_T12 (SEQ ID NO:63); R31375_T13 (SEQ ID NO:64); R31375_T2 (SEQ ID NO:53); R31375_T3 (SEQ ID NO:54); R31375_T4 (SEQ ID NO:55); R31375_T5 (SEQ ID NO:56); R31375_T6 (SEQ ID NO:57); R31375_T7 (SEQ ID NO:58); R31375_T8 (SEQ ID NO:59); R31375_T9 (SEQ ID NO:60): R31375_T19 (SEQ ID NO:65); R31375_T25 (SEQ ID NO:66); R31375_T26 (SEQ ID NO:67); R31375_T29 (SEQ ID NO:68); R31375_T39 (SEQ ID NO:69) transcripts reported herein, particularly to proteins as set forth in any one of R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73), R31375_P31 (SEQ ID NO:73), and variants thereof that are differentially expressed e.g., in cancers such as ovarian cancer as well as other non-solid and solid tumors, sarcomas, hematological malignancies including but not limited to acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, cancer of the breast, prostate, lung, ovary, colon, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, brain and wherein the cancer may be non-metastatic, invasive or metastatic.","Preferably such VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 variants will possess at least 80% sequence identity therewith, more preferably at least 90% sequence identity therewith and even more preferably at least 95% sequence identity therewith.","Any one of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 proteins based on its domain structure is predicted to be an immune costimulatory protein, e.g., a B7 protein family member that is involved in B7 immune co-stimulation including for example T cell responses elicited against cancer cells and that elicit effects on immunity such as triggering of autoimmune effects.","The term the \u201csoluble ectodomain (ECD)\u201d or \u201cectodomain\u201d of VSIG1 refers to the polypeptide sequences below or the corresponding nucleic acid sequences (which does not comprise the signal peptide and the TM of VSIG1 protein):",{"@attributes":{"id":"p-0352","num":"0351"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"280pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"161pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[">AI581519_P3\u2003(SEQ\u2003IDS\u2003NO:\u200311)\u2003residues\u200323\u2003to\u2003234\u2003(SEQ\u2003ID\u2003NO:\u2003138)",{}]},{"entry":"QVTIPDGFVNVTVGSNVTLICIYTTTVASREQLSIQWSFFHKKEMEPISIYFS"},{"entry":{}},{"entry":"QGGQAVAIGQFKDRITGSNDPGNASITISHMQPADSGIYICDVNNPPDFLGQN"},{"entry":{}},{"entry":"QGILNVSVLVKPSKPLCSVQGRPETGHTISLSCLSALGTPSPVYYWHKLEGRDIV"},{"entry":{}},{"entry":"PVKENFNPTTGILVIGNLTNFEQGYYQCTAINRLGNSSCEIDLTSSHPEVG"},{"entry":{}},{"entry":">AI581519_P4\u2003(SEQ\u2003IDS\u2003NO:\u200312)\u2003residues\u200323\u2003to\u2003270\u2003(SEQ\u2003ID\u2003NO:\u2003139)"},{"entry":"QVTIPDGFVNVTVGSNVTLICIYTTTVASREQLSIQWSFFHKKEMEPISHSS"},{"entry":{}},{"entry":"CLSTEGMEEKAVGQCLKMTHVRDARGRCSWTSEIYFSQGGQAVAIGQFKDRI"},{"entry":{}},{"entry":"TGSNDPGNASITISHMQPADSGIYICDVNNPPDFLGQNQGILNVSVLVKPSKPL"},{"entry":{}},{"entry":"CSVQGRPETGHTISLSCLSALGTPSPVYYWHKLEGRDIVPVKENFNPTTGILVIG"},{"entry":{}},{"entry":"NLTNFEQGYYQCTAINRLGNSSCEIDLTSSHPEVG"},{"entry":{}},{"entry":">AI581519_P5\u2003(SEQ\u2003IDS\u2003NO:\u200313)\u2003residues\u200323\u2003to\u2003296\u2003(SEQ\u2003ID\u2003NO:\u2003140)"},{"entry":"QVTIPDGFVNVTVGSNVTLICIYTTTVASREQLSIQWSFFHKKEMEPISHSS"},{"entry":{}},{"entry":"CLSTEGMEEKAVGQCLKMTHVRDARGRCSWTSESPWEEGKWPDVEAVKGTL"},{"entry":{}},{"entry":"DGQQAELQIYFSQGGQAVAIGQFKDRITGSNDPGNASITISHMQPADSGIYICD"},{"entry":{}},{"entry":"VNNPPDFLGQNQGILNVSVLVKPSKPLCSVQGRPETGHTISLSCLSALGTPSPV"},{"entry":{}},{"entry":"YYWHKLEGRDIVPVKENFNPTTGILVIGNLTNFEQGYYQCTAINRLGNSSCEID"},{"entry":{}},{"entry":"LTSSHPEVG"},{"entry":{}},{"entry":">AI581519_P7\u2003(SEQ\u2003IDS\u2003NO:\u200314)\u2003residues\u200323\u2003To\u2003193\u2003(SEQ\u2003ID\u2003NO:\u2003141)"},{"entry":"QVTIPDGFVNVTVGSNVTLICIYTTTVASREQLSIQWSFFHKKEMEPISIYFS"},{"entry":{}},{"entry":"QGGQAVAIGQFKDRITGSNDPVKPSKPLCSVQGRPETGHTISLSCLSALGTPSP"},{"entry":{}},{"entry":"VYYWHKLEGRDIVPVKENFNPTTGILVIGNLTNFEQGYYQCTAINRLGNSSCEI"},{"entry":{}},{"entry":"DLTSSHPEVG"},{"entry":{}},{"entry":">AI581519_P9\u2003(SEQ\u2003IDS\u2003NO:\u200315)\u2003residues\u200323\u2003to\u2003203\u2003(SEQ\u2003ID\u2003NO:\u2003142)"},{"entry":"QVTIPDGFVNVTVGSNVTLICIYTTTVASREQLSIQWSFFHKKEMEPISIYFS"},{"entry":{}},{"entry":"QGGQAVAIGQFKDRITGSNDPGNASITISHMQPADSGIYICDVNNPPDFLGQN"},{"entry":{}},{"entry":"QGILNVSVLVKPSKPLCSVQGRPETGHTISLSCLSALGTPSPVYYWHKLEGRDIV"},{"entry":{}},{"entry":"PVKENFTNHRDFGHWKSDKF"},{"entry":{}},{"entry":">AI581519_P10\u2003(SEQ\u2003IDS\u2003NO:\u200316)\u2003residues\u200323\u2003To\u2003231\u2003(SEQ\u2003ID\u2003NO:\u2003143)"},{"entry":"QVTIPDGFVNVTVGSNVTLICIYTTTVASREQLSIQWSFFHKKEMEPISIYFS"},{"entry":{}},{"entry":"QGGQAVAIGQFKDRITGSNDPGNASITISHMQPADSGIYICDVNNPPDFLGQN"},{"entry":{}},{"entry":"QGILNVSVLVKPSKPLCSVQGRPETGHTISLSCLSALGTPSPVYYWHKLEGRDIV"},{"entry":{}},{"entry":"PVKENFNPTTGILVIGNLTNFEQGYYQCTAINRLGNSSCEIDLTSSRQ,"},{"entry":{}},{"entry":">AI581519_P5\u2003(SEQ\u2003IDS\u2003NO:\u200313)\u2003residues\u200326\u2003To\u2003293\u2003(SEQ\u2003ID\u2003NO:\u2003302)"},{"entry":"IPDGFVNVTVGSNVTLICIYTTTVASREQLSIQWSFFHKKEMEPISHSSCLS"},{"entry":{}},{"entry":"TEGMEEKAVSQCLKMTHARDARGRCSWTSESPWEEGKWPDVEAVKGTLDG"},{"entry":{}},{"entry":"QQAELQIYFSQGGQAVAIGQFKDRITGSNDPGNASITISHMQPADSGIYICDVN"},{"entry":{}},{"entry":"NPPDFLGQNQGILNVSVLVKPSKPLCSVQGRPETGHTISLSCLSALGTPSPVYY"},{"entry":{}},{"entry":"WHKLEGRDIVPVKENFNPTTGILVIGNLTNFEQGYYQCTAINRLGNSSCEIDLT"},{"entry":{}},{"entry":"SSHP,"}]}}}}},"and variants thereof possessing at least 80% sequence identity, more preferably at least 90% sequence identity therewith and even more preferably at least 95, 96, 97, 98 or 99% sequence identity therewith.","The term the \u201csoluble ectodomain (ECD)\u201d or \u201cectodomain\u201d of ILDR1 refers to the polypeptide sequences below or the corresponding nucleic acid sequences (which does not comprise the signal peptide and the TM of ILDR1 protein:",{"@attributes":{"id":"p-0355","num":"0354"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"336pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["residues\u200324-105\u2003of\u2003AA424839_1_P11\u2003(SEQ\u2003ID\u2003NO:\u200324):",{}]},{"entry":"SEQ\u2003ID\u2003NO:\u2003296"},{"entry":"ALSLGQDPSNDCNDNQREVRIVAQRRGQNEPVLGVDYRQRKITIQNRA"},{"entry":{}},{"entry":"DLVINEVMWWDHGVYYCTIEAPGDTSGDPDKEVK\u2003(SEQ\u2003ID\u2003NO:\u2003296);"},{"entry":{}},{"entry":"residues\u200324-162\u2003of\u2003AA424839_P3\u2003(SEQ\u2003ID\u2003NO:\u200322)\u2003and\u2003AA424839_P5\u2003(SEQ\u2003ID\u2003NO:\u200321):"},{"entry":"SEQ\u2003ID\u2003NO:\u200375"},{"entry":"LLVTVQHTERYVTLFASIILKCDYTTSAQLQDVVVTWRFKSFCKDPIFDYY"},{"entry":{}},{"entry":"SASYQAALSLGQDPSNDCNDNQREVRIVAQRRGQNEPVLGVDYRQRKITIQN"},{"entry":{}},{"entry":"RADLVINEVMWWDHGVYYCTIEAPGDTSGDPDKEVK:"},{"entry":{}},{"entry":"residues\u200324-457\u2003of\u2003AA424839_P7\u2003(SEQ\u2003ID\u2003NO:\u200323):"},{"entry":"SEQ\u2003ID\u2003NO:\u200376"},{"entry":"LLVTVQHTERYVTLFASIILKCDYTTSAQLQDVVVTVVRFKSFCKDPIFDYY"},{"entry":{}},{"entry":"SASYQAALSLGQDPSNDCNDNQREVRIVAQRRGQNEPVLGVDYRQRKITIQN"},{"entry":{}},{"entry":"PLARHRYMKQAQALGPQMMGKPLYWGADRSSQVSSYPMHPLLQRDLSLPSS"},{"entry":{}},{"entry":"LPQMPMTQTTNQPPIANGVLEYLEKELRNLNLAQPLPPDLKGRFGHPCSMLSS"},{"entry":{}},{"entry":"LGSEVVERRIIHLPPLIRDLSSSRRTSDSLHQQWLTPIPSRPWDLREGRSHHHYP"},{"entry":{}},{"entry":"DFHQELQDRGPKSWALERRELDPSWSGRHRSSRLNGSPIHWSDRDSLSDVPSS"},{"entry":{}},{"entry":"SEARWRPSHPPFRSRCQERPRRPSPRESTQRHGRRRRHRSYSPPLPSGLSSWSSE"},{"entry":{}},{"entry":"EDKERQPQSWRAHRRGSHSPHWPEEKPPSYRSLDITPGKNSRKKGSVERRSEK"},{"entry":{}},{"entry":"DSSHSGRSVVI;"},{"entry":{}},{"entry":"residues\u200350-160\u2003of\u2003AA424839_P3\u2003(SEQ\u2003ID\u2003NO:36):"},{"entry":"SEQ\u2003ID\u2003NO:\u2003301"},{"entry":"AQLQDVVVTWRFKSFCKDPIFDYYSASYQAALSLGQDPSNDCNDNQRE"},{"entry":{}},{"entry":"VRIVAQRRGQNEPVLGVDYRQRKITIQNRADLVINEVMWWDHGVYYCTIEAP"},{"entry":{}},{"entry":"GDTSGDPDKE,"}]}}}}},"and variants thereof possessing at least 80% sequence identity, more preferably at least 90% sequence identity therewith and even more preferably at least 95, 96, 97, 98 or 99% sequence identity therewith.","The term the \u201csoluble ectodomain (ECD)\u201d or \u201cectodomain\u201d of LOC253012 refers to the polypeptide sequences below or the corresponding nucleic acid sequences (which does not comprise the signal peptide and the TM of LOC253012 protein):",{"@attributes":{"id":"p-0358","num":"0357"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"182pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["H68654_1_P2\u2003(SEQ\u2003ID\u2003NO:\u200335)\u2003residues\u200338-349:",{}]},{"entry":"(SEQ\u2003ID\u2003NO:\u2003144)"},{"entry":"SHTVHGVRGQALYLPVHYGFHTPASDIQIIWLFERPHTMPKYLLGSVNKS"},{"entry":{}},{"entry":"VVPDLEYQHKFTMMPPNASLLINPLQFPDEGNYIVKVNIQGNGTLSASQKIQV"},{"entry":{}},{"entry":"TVDDPVTKPVVQIHPPSGAVEYVGNMTLTCHVEGGTRLAYQWLKNGRPVHT"},{"entry":{}},{"entry":"SSTYSFSPQNNTLHIAPVTKEDIGNYSCLVRNPVSEMESDIIMPIIYYGPYGLQV"},{"entry":{}},{"entry":"NSDKGLKVGEVFTVDLGEAILFDCSADSHPPNTYSWIRRTDNTTYIIKHGPRLE"},{"entry":{}},{"entry":"VASEKVAQKTMDYVCCAYNNITGRQDETHFTVIITSVGLEKLAQKGKSL;"},{"entry":{}},{"entry":"H68654_1_P5,\u2003(SEQ\u2003ID\u2003NO:\u200336)"},{"entry":{}},{"entry":"H68654_1_P7,\u2003(SEQ\u2003ID\u2003NO:\u200337)"},{"entry":{}},{"entry":"H68654_1_P12,\u2003(SEQ\u2003ID\u2003NO:\u200338)"},{"entry":{}},{"entry":"H68654_1_P13,\u2003(SEQ\u2003ID\u2003NO:\u200339)"},{"entry":{}},{"entry":"H68654_1_P14\u2003(SEQ\u2003ID\u2003NO:\u200340)\u2003residues\u200319-337:"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003145)"},{"entry":"GLKVTVPSHTVHGVRGQALYLPVHYGFHTPASDIQIIWLFERPHTMPKYLLGS"},{"entry":{}},{"entry":"VNKSVVPDLEYQHKFTMMPPNASLLINPLQFPDEGNYIVKVNIQGNGTLSASQ"},{"entry":{}},{"entry":"KIQVTVDDPVTKPVVQIHPPSGAVEYVGNMTLTCHVEGGTRLAYQWLKNGR"},{"entry":{}},{"entry":"PVHTSSTYSFSPQNNTLHIAPVTKEDIGNYSCLVRNPVSEMESDIIMPIIYYGPYG"},{"entry":{}},{"entry":"LQVNSDKGLKVGEVFTVDLGEAILFDCSADSHPPNTYSWIRRTDNTTYIIKHGP"},{"entry":{}},{"entry":"RLEVASEKVAQKTMDYVCCAYNNITGRQDETHFTVIITSVGLEKLAQKGKSL,"},{"entry":{}},{"entry":"H68654_1_P5\u2003(SEQ\u2003ID\u2003NO:\u200336)\u2003residues\u20031-335\u2003(SEQ\u2003ID\u2003NO:\u2003300):"},{"entry":"MWLKVFTTFLSFATGACSGLKVTVPSHTVHGVRGQALYLPVHYGFHTPASDI"},{"entry":{}},{"entry":"QIIWLFERPHTMPKYLLGSVNKSVVPDLEYQHKFTMMPPNASLLINPLQFPDE"},{"entry":{}},{"entry":"GNYIVKVNIQGNGTLSASQKIQVTVDDPVTKPVVQIHPPSGAVEYVGNMTLTC"},{"entry":{}},{"entry":"HVEGGTRLAYQWLKNGRPVHTSSTYSFSPQNNTLHIAPVTKEDIGNYSCLVRN"},{"entry":{}},{"entry":"PVSEMESDIIMPIIYYGPYGLQVNSDKGLKVGEVFTVDLGEAILFDCSADSHPPN"},{"entry":{}},{"entry":"TYSWIRRTDNTTYIIKHGPRLEVASEKVAQKTMDYVCCAYNNITGRQDETHFT"},{"entry":{}},{"entry":"VIITSVGLEKLAQKGK,"}]}}}}},"and variants thereof possessing at least 80% sequence identity, more preferably at least 90% sequence identity therewith and even more preferably at least 95, 96, 97, 98 or 99% sequence identity therewith.","The term the \u201csoluble ectodomain (ECD)\u201d or \u201cectodomain\u201d of AI216611 refers to the polypeptide sequences below or the corresponding nucleic acid sequences (which does not comprise the signal peptide and the TM of AI216611 protein):",{"@attributes":{"id":"p-0361","num":"0360"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"182pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[">AI216611_P0\u2003(SEQ\u2003ID\u2003NO:\u200343)\u2003From\u200329\u2003to\u2003147\u2003(SEQ\u2003ID\u2003NO:\u2003146)",{}]},{"entry":"LQSQGVSLYIPQATINATVKEDILLSVEYSCHGVPTIEWTYSSNWGTQKIV"},{"entry":{}},{"entry":"EWKPGTQANISQSHKDRVCTFDNGSIQLFSVGVRDSGYYVITVTERLGSSQFGT"},{"entry":{}},{"entry":"IVLHVSEILYEDLH,"},{"entry":{}},{"entry":">AI216611_P0\u2003(SEQ\u2003ID\u2003NO:\u200343)\u2003From\u20031\u2003to\u2003145\u2003(SEQ\u2003ID\u2003NO:\u2003298)"},{"entry":"MRPLPSGRRKTRGISLGLFALCLAAARCLQSQGVSLYIPQATINATVKEDI"},{"entry":{}},{"entry":"LLSVEYSCHGVPTIEWTYSSNWGTQKIVEWKPGTQANISQSHKDRVCTFDNGS"},{"entry":{}},{"entry":"IQLFSVGVRDSGYYVITVTERLGSSQFGTIVLHVSEILYED,"}]}}}}},"and variants thereof possessing at least 80% sequence identity, more preferably at least 90% sequence identity therewith and even more preferably at least 95, 96, 97, 98 or 99% sequence identity therewith.","The term the \u201csoluble ectodomain (ECD)\u201d or \u201cectodomain\u201d of C1ORF32 refers to the polypeptide sequences below or the corresponding nucleic acid sequences (which does not comprise the signal peptide and the TM of C1ORF32 protein):",{"@attributes":{"id":"p-0364","num":"0363"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"273pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"168pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[">H19011_1_P8\u2003(SEQ\u2003ID\u2003NO:\u200348)\u2003residues\u200321\u2003to\u2003186\u2003(SEQ\u2003ID\u2003NO:\u2003147)",{}]},{"entry":"LQVTVPDKKKVAMLFQPTVLRCHFSTSSHQPAVVQWKFKSYCQDRMGE"},{"entry":{}},{"entry":"SLGMSSTRAQSLSKRNLEWDPYLDCLDSRRTVRVVASKQGSTVTLGDFYRGREI"},{"entry":{}},{"entry":"TIVHDADLQIGKLMWGDSGLYYCIITTPDDLEGKNEGSLGLLVLGRTGLLADL"},{"entry":{}},{"entry":"LPSFAVEIMPE"},{"entry":{}},{"entry":">H19011_1_P9\u2003(SEQ\u2003ID\u2003NO:\u200350)\u2003residues\u200321\u2003to\u2003169\u2003(SEQ\u2003ID\u2003NO:\u2003148)"},{"entry":"LQVTVPDKKKVAMLFQPTVLRCHFSTSSHQPAVVQWKFKSYCQDRMGE"},{"entry":{}},{"entry":"SLGMSSTRAQSLSKRNLEWDPYLDCLDSRRTVRVVASKQGSTVTLGDFYRGREI"},{"entry":{}},{"entry":"TIVHDADLQIGKLMWGDSGLYYCIITTPDDLEGKNEGSLGLLVLEWV,"},{"entry":{}},{"entry":">H19011_1_P8\u2003(SEQ\u2003ID\u2003NO:\u200348)\u2003residues\u20031\u2003to\u2003184\u2003(SEQ\u2003ID\u2003NO:\u2003299)"},{"entry":"MDRVLLRWISLFWLTAMVEGLQVTVPDKKKVAMLFQPTVLRCHFSTSS"},{"entry":{}},{"entry":"HQPAVVQWKFKSYCQDRMGESLGMSSTRAQSLSKRNLEWDPYLDCLDSRRT"},{"entry":{}},{"entry":"VRVVASKQGSTVTLGDFYRGREITIVHDADLQIGKLMWGDSGLYYCIITTPDDL"},{"entry":{}},{"entry":"EGKNEDSVELLVLGRTGLLADLLPSFAVEIM,"}]}}}}},"and variants thereof possessing at least 80% sequence identity, more preferably at least 90% sequence identity therewith and even more preferably at least 95, 96, 97, 98 or 99% sequence identity therewith.","The term the \u201csoluble ectodomain (ECD)\u201d or \u201cectodomain\u201d of FXYD3 refers to the polypeptide sequences below or the corresponding nucleic acid sequences (which does not comprise the signal peptide and the TM of FXYD3 protein):",{"@attributes":{"id":"p-0367","num":"0366"},"tables":{"@attributes":{"id":"TABLE-US-00006","num":"00006"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":">R31375_PO\u2003(SEQ\u2003ID\u2003NO:\u200370);"},{"entry":"R31375_P31\u2003(SEQ\u2003ID\u2003NO:\u200373)"},{"entry":"From\u200321\u2003to\u200336\u2003(SEQ\u2003ID\u2003NO:\u2003149)"},{"entry":"NDLEDKNSPFYYDWHS"},{"entry":{}},{"entry":">R31375_P14\u2003(SEQ\u2003ID\u2003NO:\u200372)\u2003"},{"entry":"From\u200321\u2003to\u200365\u2003(SEQ\u2003ID\u2003NO:\u2003150)"},{"entry":"NDLEDKNSPFYYGAPYIFVKRMGGQMKRTQAGTEVPSTFLLDWHS"},{"entry":{}},{"entry":">R31375_P33\u2003(SEQ\u2003ID\u2003NO:\u200374)\u2003"},{"entry":"From\u200321\u2003to\u200325\u2003(SEQ\u2003ID\u2003NO:\u2003151)"},{"entry":"NDLED,"},{"entry":{}},{"entry":">R31375_P14\u2003(SEQ\u2003ID\u2003NO:\u200372)\u2003"},{"entry":"From\u20031\u2003to\u200363\u2003(SEQ\u2003ID\u2003NO:\u2003297)"},{"entry":"MQKVTLGLLVFLAGFPVLDANDLEDKNSPFYYGAPYIFVKRMGGQMKR"},{"entry":{}},{"entry":"TQAGTEVPSTFLLDW,"}]}}}}},"and variants thereof possessing at least 80% sequence identity, more preferably at least 90% sequence identity therewith and even more preferably at least 95, 96, 97, 98 or 99% sequence identity therewith.","The term \u201cimmune response\u201d refers to the action of, for example, lymphocytes, antigen presenting cells, phagocytic cells, granulocytes, and soluble macromolecules produced by the above cells or cells produced by the liver or spleen (including antibodies, cytokines, and complement) that results in selective damage to, destruction of, or elimination from the human body of invading pathogens, cells or tissues infected with pathogens, cancerous cells, or, in cases of autoimmunity or pathological inflammation, normal human cells or tissues.","A \u201csignal, transduction pathway\u201d refers to the biochemical relationship between a variety of signal transduction molecules that play a role in the transmission of a signal from one portion of a cell to another portion of a cell.","As used herein, the phrase \u201ccell surface receptor\u201d includes, for example, molecules and complexes of molecules capable of receiving a signal and the transmission of such a signal across the plasma membrane of a cell.","The term \u201cantibody\u201d as referred to herein includes whole polyclonal and monoclonal antibodies and any antigen binding fragment (i.e., \u201cantigen-binding portion\u201d) or single chains thereof. An \u201cantibody\u201d refers to a glycoprotein comprising at least two heavy (H) chains and two light (L) chains inter-connected by disulfide bonds, or an antigen binding portion thereof. Each heavy chain is comprised of a heavy chain variable region (abbreviated herein as VH) and a heavy chain constant region. The heavy chain constant region is comprised of three domains, CH1, CH2 and CH3. Each light chain is comprised of a light chain variable region (abbreviated herein as VL) and a light chain constant region. The light chain constant region is comprised of one domain, CL. The VH and VL regions can be further subdivided into regions of hypervariability, termed complementarity determining regions (CDR), interspersed with regions that are more conserved, termed framework regions (FR). Each VH and VL is composed of three CDRs and four FRs, arranged from amino-terminus to carboxy-terminus in the following order: FR1, CDR1, FR2, CDR2, FR3, CDR3, FR4. The variable regions of the heavy and light chains contain a binding domain that interacts with an antigen. The constant regions of the antibodies may mediate the binding of the immunoglobulin to host tissues or factors, including various cells of the immune system (e.g., effector cells) and the first component (Clq) of the classical complement system.","The term \u201cantigen-binding portion\u201d of an antibody (or simply \u201cantibody portion\u201d), as used herein, refers to one or more fragments of an antibody that retain the ability to specifically bind to an antigen (e.g., VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins or VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3). It has been shown that the antigen-binding function of an antibody can be performed by fragments of a full-length antibody. Examples of binding fragments encompassed within the term \u201cantigen-binding portion\u201d of an antibody include (i) a Fab fragment, a monovalent fragment consisting of the V Light, V Heavy, Constant light (CL) and CH1 domains; (ii) a F(ab\u2032).2 fragment, a bivalent fragment comprising two Fab fragments linked by a disulfide bridge at the hinge region; (iii) a Fd fragment consisting of the VH and CH1 domains; (iv) a Fv fragment consisting of the VL and VH domains of a single arm of an antibody, (v) a dAb fragment (Ward et al., (1989) Nature 341:544-546), which consists of a VH domain; and (vi) an isolated complementarity determining region (CDR). Furthermore, although the two domains of the Fv fragment, VL and VH, are coded for by separate genes, they can be joined, using recombinant methods, by a synthetic linker that enables them to be made as a single protein chain in which the VL and VH regions pair to form monovalent molecules (known as single chain Fv (scFv); see e.g., Bird et al. (1988) Science 242:423-426; and Huston et al. (1988) Proc. Natl. Acad. Sci. USA 85:5879-5883). Such single chain antibodies are also intended to be encompassed within the term \u201cantigen-binding portion\u201d of an antibody. These antibody fragments are obtained using conventional techniques known to those with skill in the art, and the fragments are screened for utility in the same manner as are intact antibodies.","An \u201cisolated antibody\u201d, as used herein, is intended to refer to an antibody that is substantially free of other antibodies having different antigenic specificities (e.g., an isolated antibody that specifically binds VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins or VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 is substantially free of antibodies that specifically bind antigens other than VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins or VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3, respectively. An isolated antibody that specifically binds VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins or VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 may, however, have cross-reactivity to other antigens, such as VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins or VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 molecules from other species, respectively. Moreover, an isolated antibody may be substantially free of other cellular material and\/or chemicals.","The terms \u201cmonoclonal antibody\u201d or \u201cmonoclonal antibody composition\u201d as used herein refer to a preparation of antibody molecules of single molecular composition. A monoclonal antibody composition displays a single binding specificity and affinity for a particular epitope.","The term \u201chuman antibody\u201d, as used herein, is intended to include antibodies having variable regions in which both the framework and CDR regions are derived from human germline immunoglobulin sequences. Furthermore, if the antibody contains a constant region, the constant region also is derived from human germline immunoglobulin sequences. The human antibodies of the invention may include amino acid residues not encoded by human germline immunoglobulin sequences (e.g., mutations introduced by random or site-specific mutagenesis in vitro or by somatic mutation in vivo). However, the term \u201chuman antibody\u201d, as used herein, is not intended to include antibodies in which CDR sequences derived from the germline of another mammalian species, such as a mouse, have been grafted onto human framework sequences.","The term \u201chuman monoclonal antibody\u201d refers to antibodies displaying a single binding specificity which have variable regions in which both the framework and CDR regions are derived from human germline immunoglobulin sequences. In one embodiment, the human monoclonal antibodies are produced by a hybridoma which includes a B cell obtained from a transgenic nonhuman animal, e.g., a transgenic mouse, having a genome comprising a human heavy chain transgene and a light chain transgene fused to an immortalized cell.","The term \u201crecombinant human antibody\u201d, as used herein, includes all human antibodies that are prepared, expressed, created or isolated by recombinant means, such as (a) antibodies isolated from an animal (e.g., a mouse) that is transgenic or transchromosomal for human immunoglobulin genes or a hybridoma prepared therefrom (described further below), (b) antibodies isolated from a host cell transformed to express the human antibody, e.g., from a transfectoma, (c) antibodies isolated from a recombinant, combinatorial human antibody library, and (d) antibodies prepared, expressed, created or isolated by any other means that involve splicing of human immunoglobulin gene sequences to other DNA sequences. Such recombinant human antibodies have variable regions in which the framework and CDR regions are derived from human germline immunoglobulin sequences. In certain embodiments, however, such recombinant human antibodies can be subjected to in vitro mutagenesis (or, when an animal transgenic for human Ig sequences is used, in vivo somatic mutagenesis) and thus the amino acid sequences of the VH and VL regions of the recombinant antibodies are sequences that, while derived from and related to human germline VH and VL sequences, may not naturally exist within the human antibody germline repertoire in vivo.","As used herein, \u201cisotype\u201d refers to the antibody class (e.g., IgM or IgG1) that is encoded by the heavy chain constant region genes.","The phrases \u201can antibody recognizing an antigen\u201d and \u201can antibody specific for an antigen\u201d are used interchangeably herein with the term \u201can antibody which binds specifically to an antigen.\u201d","As used herein, an antibody that \u201cspecifically binds to human VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins or VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 is intended to refer to an antibody that binds to human VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins or VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3, respectively, preferably one with a KD of 5\u00d710\u22128 M or less, more preferably 3\u00d710\u22128 M or less, and even more preferably 1\u00d7.10\u22129M or less.","The term \u201cK-assoc\u201d or \u201cKa\u201d, as used herein, is intended to refer to the association rate of a particular antibody-antigen interaction, whereas the term \u201cKdiss\u201d or \u201cKd,\u201d as used herein, is intended to refer to the dissociation rate of a particular antibody-antigen interaction. The term \u201cKD\u201d, as used herein, is intended to refer to the dissociation constant, which is obtained from the ratio of Kd to Ka (i.e., Kd\/Ka) and is expressed as a molar concentration (M). KD values for antibodies can be determined using methods well established in the art. A preferred method for determining the KD of an antibody is by using surface Plasmon resonance, preferably using a biosensor system such as a Biacore\u00ae system.","As used herein, the term \u201chigh affinity\u201d for an IgG antibody refers to an antibody having a KD of 10\u22128 M or less, more preferably 10\u22129M or less and even more preferably 10\u221210 M or less for a target antigen. However, \u201chigh affinity\u201d binding can vary for other antibody isotypes. For example, \u201chigh affinity\u201d binding for an IgM isotype refers to an antibody having a KD of 10\u22127 M or less, more preferably 10\u22128 M or less.","As used herein, the term \u201csubject\u201d includes any human or nonhuman animal. The term \u201cnonhuman animal\u201d includes all vertebrates, e.g., mammals and non-mammals, such as nonhuman primates, sheep, dogs, cats, horses, cows chickens, amphibians, reptiles, etc.","As used herein, the term \u201ctail\u201d refers to a peptide sequence at the end of an amino acid sequence that is unique to a splice variant according to the present invention. Therefore, a splice variant having such a tail may optionally be considered as a chimera, in that at least a first portion of the splice variant is typically highly homologous (often 100% identical) to a portion of the corresponding known protein, while at least a second portion of the variant comprises the tail.","As used herein, the term \u201chead\u201d refers to a peptide sequence at the beginning of an amino acid sequence that is unique to a splice variant according to the present invention. Therefore, a splice variant having such a head may optionally be considered as a chimera, in that at least a first portion of the splice variant comprises the head, while at least a second portion is typically highly homologous (often 100% identical) to a portion of the corresponding known protein.","As used herein, the term \u201can edge portion\u201d refers to a connection between two portions of a splice variant according to the present invention that were not joined in the wild type or known protein. An edge may optionally arise due to a join between the above \u201cknown protein\u201d portion of a variant and the tail, for example, and\/or may occur if an internal portion of the wild type sequence is no longer present, such that two portions of the sequence are now contiguous in the splice variant that were not contiguous in the known protein. A \u201cbridge\u201d may optionally be an edge portion as described above, but may also include a join between a head and a \u201cknown protein\u201d portion of a variant, or a join between a tail and a \u201cknown protein\u201d portion of a variant, or a join between an insertion and a \u201cknown protein\u201d portion of a variant.","In some embodiments, a bridge between a tail or a head or a unique insertion, and a \u201cknown protein\u201d portion of a variant, comprises at least about 10 amino acids, or in some embodiments at least about 20 amino acids, or in some embodiments at least about 30 amino acids, or in some embodiments at least about 40 amino acids, in which at least one amino acid is from the tail\/head\/insertion and at least one amino acid is from the \u201cknown protein\u201d portion of a variant. In some embodiments, the bridge may comprise any number of amino acids from about 10 to about 40 amino acids (for example, 10, 11, 12, 13 . . . 37, 38, 39, 40 amino acids in length, or any number in between).","It should be noted that a bridge cannot be extended beyond the length of the sequence in either direction, and it should be assumed that every bridge description is to be read in such manner that the bridge length does not extend beyond the sequence itself.","Furthermore, bridges are described with regard to a sliding window in certain contexts below. For example, certain descriptions of the bridges feature the following format: a bridge between two edges (in which a portion of the known protein is not present in the variant) may optionally be described as follows: a bridge portion of CONTIG-NAME_P1 (representing the name of the protein), comprising a polypeptide having a length \u201cn\u201d, wherein n is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least two amino acids comprise XX (2 amino acids in the center of the bridge, one from each end of the edge), having a structure as follows (numbering according to the sequence of CONTIG-NAME_P1): a sequence starting from any of amino acid numbers 49\u2212x to 49 (for example); and ending at any of amino acid numbers 50+((n\u22122)\u2212x) (for example), in which x varies from 0 to n\u22122. In this example, it should also be read as including bridges in which n is any number of amino acids between 10-50 amino acids in length. Furthermore, the bridge polypeptide cannot extend beyond the sequence, so it should be read such that 49\u2212x (for example) is not less than 1, nor 50+((n\u22122)\u2212x) (for example) greater than the total sequence length.","Various aspects of the invention are described in further detail in the following subsections.","Nucleic Acids","A \u201cnucleic acid fragment\u201d or an \u201coligonucleotide\u201d or a \u201cpolynucleotide\u201d are used herein interchangeably to refer to a polymer of nucleic acid residues. A polynucleotide sequence of the present invention refers to a single or double stranded nucleic acid sequences which is isolated and provided in the form of an RNA sequence, a complementary polynucleotide sequence (cDNA), a genomic polynucleotide sequence and\/or a composite polynucleotide sequences (e.g., a combination of the above).","Thus, the present invention encompasses nucleic acid sequences described hereinabove; fragments thereof, sequences hybridizable therewith, sequences homologous thereto [e.g., at least 90%, at least 95, 96, 97, 98 or 99% or more identical to the nucleic acid sequences set forth herein], sequences encoding similar polypeptides with different codon usage, altered sequences characterized by mutations, such as deletion, insertion or substitution of one or more nucleotides, either naturally occurring or man induced, either randomly or in a targeted fashion. The present invention also encompasses homologous nucleic acid sequences (i.e., which form a part of a polynucleotide sequence of the present invention), which include sequence regions unique to the polynucleotides of the present invention.","In cases where the polynucleotide sequences of the present invention encode previously unidentified polypeptides, the present invention also encompasses novel polypeptides or portions thereof, which are encoded by the isolated polynucleotide and respective nucleic acid fragments thereof described hereinabove.","Thus, the present invention also encompasses polypeptides encoded by the polynucleotide sequences of the present invention. The present invention also encompasses homologues of these polypeptides, such homologues can be at least 90%, at least 95, 96, 97, 98 or 99% or more homologous to the amino acid sequences set forth below, as can be determined using BlastP software of the National Center of Biotechnology Information (NCBI) using default parameters. Finally, the present invention also encompasses fragments of the above described polypeptides and polypeptides having mutations, such as deletions, insertions or substitutions of one or more amino acids, either naturally occurring or man induced, either randomly or in a targeted fashion.","As mentioned hereinabove, biomolecular sequences of the present invention can be efficiently utilized as tissue or pathological markers and as putative drugs or drug targets for treating or preventing a disease.","Oligonucleotides designed for carrying out the methods of the present invention for any of the sequences provided herein (designed as described above) can be generated according to any oligonucleotide synthesis method known in the art such as enzymatic synthesis or solid phase synthesis. Equipment and reagents for executing solid-phase synthesis are commercially available from, for example, Applied Biosystems. Any other means for such synthesis may also be employed; the actual synthesis of the oligonucleotides is well within the capabilities of one skilled in the art.","Oligonucleotides used according to this aspect of the present invention are those having a length selected from a range of about 10 to about 200 bases preferably about 15 to about 150 bases, more preferably about 20 to about 100 bases, most preferably about 20 to about 50 bases.","The oligonucleotides of the present invention may comprise heterocyclic nucleosides consisting of purines and the pyrimidines bases, bonded in a 3\u2032 to 5\u2032 phosphodiester linkage.","Preferable oligonucleotides are those modified in either backbone, internucleoside linkages or bases, as is broadly described hereinunder. Such modifications can oftentimes facilitate oligonucleotide uptake and resistivity to intracellular conditions.","Specific examples of preferred oligonucleotides useful according to this aspect of the present invention include oligonucleotides containing modified backbones or non-natural internucleoside linkages. Oligonucleotides having modified backbones include those that retain a phosphorus atom in the backbone, as disclosed in U.S. Pat. Nos. 4,469,863; 4,476,301; 5,023,243; 5,177,196; 5,188,897; 5,264,423; 5,276,019; 5,278,302; 5,286,717; 5,321,131; 5,399,676; 5,405,939; 5,453,496; 5,455,233; 5,466,677; 5,476,925; 5,519,126; 5,536,821; 5,541,306; 5,550,111; 5,563,253; 5,571,799; 5,587,361; and 5,625,050.","Preferred modified oligonucleotide backbones include, for example, phosphorothioates, chiral phosphorothioates, phosphorodithioates, phosphotriesters, aminoalkyl phosphotriesters, methyl and other alkyl phosphonates including 3\u2032-alkylene phosphonates and chiral phosphonates, phosphinates, phosphoramidates including 3\u2032-amino phosphoramidate and aminoalkylphosphoramidates, thionophosphoramidates, thionoalkylphosphonates, thionoalkylphosphotriesters, and boranophosphates having normal 3\u2032-5\u2032 linkages, 2\u2032-5\u2032 linked analogs of these, and those having inverted polarity wherein the adjacent pairs of nucleoside units are linked 3\u2032-5\u2032 to 5\u2032-3\u2032 or 2\u2032-5\u2032 to 5\u2032-2\u2032. Various salts, mixed salts and free acid forms can also be used.","Alternatively, modified oligonucleotide backbones that do not include a phosphorus atom therein have backbones that are formed by short chain alkyl or cycloalkyl internucleoside linkages, mixed heteroatom and alkyl or cycloalkyl internucleoside linkages, or one or more short chain heteroatomic or heterocyclic internucleoside linkages. These include those having morpholino linkages (formed in part from the sugar portion of a nucleoside); siloxane backbones; sulfide, sulfoxide and sulfone backbones; formacetyl and thioformacetyl backbones; methylene formacetyl and thioformacetyl backbones; alkene containing backbones; sulfamate backbones; methyleneimino and methylenehydrazino backbones; sulfonate and sulfonamide backbones; amide backbones; and others having mixed N, O, S and CH2 component parts, as disclosed in U.S. Pat. Nos. 5,034,506; 5,166,315; 5,185,444; 5,214,134; 5,216,141; 5,235,033; 5,264,562; 5,264,564; 5,405,938; 5,434,257; 5,466,677; 5,470,967; 5,489,677; 5,541,307; 5,561,225; 5,596,086; 5,602,240; 5,610,289; 5,602,240; 5,608,046; 5,610,289; 5,618,704; 5,623,070; 5,663,312; 5,633,360; 5,677,437; and 5,677,439.","Other oligonucleotides which can be used according to the present invention, are those modified in both sugar and the internucleoside linkage, i.e., the backbone, of the nucleotide units are replaced with novel groups. The base units are maintained for complementation with the appropriate polynucleotide target. An example for such an oligonucleotide mimetic, includes peptide nucleic acid (PNA). A PNA oligonucleotide refers to an oligonucleotide where the sugar-backbone is replaced with an amide containing backbone, in particular an aminoethylglycine backbone. The bases are retained and are bound directly or indirectly to aza nitrogen atoms of the amide portion of the backbone. United States patents that teach the preparation of PNA compounds include, but are not limited to, U.S. Pat. Nos. 5,539,082; 5,714,331; and 5,719,262, each of which is herein incorporated by reference. Other backbone modifications, which can be used in the present invention are disclosed in U.S. Pat. No. 6,303,374.","Oligonucleotides of the present invention may also include base modifications or substitutions. As used herein, \u201cunmodified\u201d or \u201cnatural\u201d bases include the purine bases adenine (A) and guanine (G), and the pyrimidine bases thymine (T), cytosine (C) and uracil (U). Modified bases include but are not limited to other synthetic and natural bases such as 5-methylcytosine (5-me-C), 5-hydroxymethyl cytosine, xanthine, hypoxanthine, 2-aminoadenine, 6-methyl and other alkyl derivatives of adenine and guanine, 2-propyl and other alkyl derivatives of adenine and guanine, 2-thiouracil, 2-thiothymine and 2-thiocytosine, 5-halouracil and cytosine, 5-propynyl uracil and cytosine, 6-azo uracil, cytosine and thymine, 5-uracil (pseudouracil), 4-thiouracil, 8-halo, 8-amino, 8-thiol, 8-thioalkyl, 8-hydroxyl and other 8-substituted adenines and guanines, 5-halo particularly 5-bromo, 5-trifluoromethyl and other 5-substituted uracils and cytosines, 7-methylguanine and 7-methyladenine, 8-azaguanine and 8-azaadenine, 7-deazaguanine and 7-deazaadenine and 3-deazaguanine and 3-deazaadenine. Further bases include those disclosed in U.S. Pat. No. 3,687,808, those disclosed in The Concise Encyclopedia Of Polymer Science and Engineering, pages 858-859, Kroschwitz, J. I., ed. John Wiley & Sons, 1990, those disclosed by Englisch et al., Angewandte Chemie, International Edition, 1991, 30, 613, and those disclosed by Sanghvi, Y. S., Chapter 15, Antisense Research and Applications, pages 289-302, Crooke, S. T. and Lebleu, B., ed., CRC Press, 1993. Such bases are particularly useful for increasing the binding affinity of the oligomeric compounds of the invention. These include 5-substituted pyrimidines, 6-azapyrimidines and N-2, N-6 and O-6 substituted purines, including 2-aminopropyladenine, 5-propynyluracil and 5-propynylcytosine. 5-methylcytosine substitutions have been shown to increase nucleic acid duplex stability by 0.6-1.2\u00b0 C. [Sanghvi Y S et al. (1993) Antisense Research and Applications, CRC Press, Boca Raton 276-278] and are presently preferred base substitutions, even more particularly when combined with 2\u2032-O-methoxyethyl sugar modifications.","Another modification of the oligonucleotides of the invention involves chemically linking to the oligonucleotide one or more moieties or conjugates, which enhance the activity, cellular distribution or cellular uptake of the oligonucleotide. Such moieties include but are not limited to lipid moieties such as a cholesterol moiety, cholic acid, a thioether, e.g., hexyl-5-tritylthiol, a thiocholesterol, an aliphatic chain, e.g., dodecandiol or undecyl residues, a phospholipid, e.g., di-hexadecyl-rac-glycerol or triethylammonium 1,2-di-O-hexadecyl-rac-glycero-3-H-phosphonate, a polyamine or a polyethylene glycol chain, or adamantane acetic acid, a palmityl moiety, or an octadecylamine or hexylamino-carbonyl-oxycholesterol moiety, as disclosed in U.S. Pat. No. 6,303,374.","It is not necessary for all positions in a given oligonucleotide molecule to be uniformly modified, and in fact more than one of the aforementioned modifications may be incorporated in a single compound or even at a single nucleoside within an oligonucleotide.","Peptides","The terms \u201cpolypeptide,\u201d \u201cpeptide\u201d and \u201cprotein\u201d are used interchangeably herein to refer to a polymer of amino acid residues. The terms apply to amino acid polymers in which one or more amino acid residue is an analog or mimetic of a corresponding naturally occurring amino acid, as well as to naturally occurring amino acid polymers. Polypeptides can be modified, e.g., by the addition of carbohydrate residues to form glycoproteins. The terms \u201cpolypeptide,\u201d \u201cpeptide\u201d and \u201cprotein\u201d include glycoproteins, as well as non-glycoproteins.","Polypeptide products can be biochemically synthesized such as by employing standard solid phase techniques. Such methods include exclusive solid phase synthesis, partial solid phase synthesis methods, fragment condensation, classical solution synthesis. These methods are preferably used when the peptide is relatively short (i.e., 10 kDa) and\/or when it cannot be produced by recombinant techniques (i.e., not encoded by a nucleic acid sequence) and therefore involves different chemistry.","Solid phase polypeptide synthesis procedures are well known in the art and further described by John Morrow Stewart and Janis Dillaha Young, Solid Phase Peptide Syntheses (2nd Ed., Pierce Chemical Company, 1984).","Synthetic polypeptides can be purified by preparative high performance liquid chromatography [Creighton T. (1983) Proteins, structures and molecular principles. WH Freeman and Co. N.Y.] and the composition of which can be confirmed via amino acid sequencing.","In cases where large amounts of a polypeptide are desired, it can be generated using recombinant techniques such as described by Bitter et al., (1987) Methods in Enzymol. 153:516-544, Studier et al. (1990) Methods in Enzymol. 185:60-89, Brisson et al. (1984) Nature 310:511-514, Takamatsu et al. (1987) EMBO J. 6:307-311, Coruzzi et al. (1984) EMBO J. 3:1671-1680 and Brogli et al., (1984) Science 224:838-843, Gurley et al. (1986) Mol. Cell. Biol. 6:559-565 and Weissbach & Weissbach, 1988, Methods for Plant Molecular Biology, Academic Press, NY, Section VIII, pp 421-463.","It will be appreciated that peptides identified according to the teachings of the present invention may be degradation products, synthetic peptides or recombinant peptides as well as peptidomimetics, typically, synthetic peptides and peptoids and semipeptoids which are peptide analogs, which may have, for example, modifications rendering the peptides more stable while in a body or more capable of penetrating into cells. Such modifications include, but are not limited to N terminus modification, C terminus modification, peptide bond modification, including, but not limited to, CH2-NH, CH2-S, CH2-S\u2550O, O\u2550C\u2014NH, CH2-O, CH2-CH2, S\u2550C\u2014NH, CH\u2550CH or CF\u2550CH, backbone modifications, and residue modification. Methods for preparing peptidomimetic compounds are well known in the art and are specified, for example, in Quantitative Drug Design, C.A. Ramsden Gd., Chapter 17.2, F. Choplin Pergamon Press (1992), which is incorporated by reference as if fully set forth herein. Further details in this respect are provided hereinunder.","Peptide bonds (\u2014CO\u2014NH\u2014) within the peptide may be substituted, for example, by N-methylated bonds (\u2014N(CH3)-CO\u2014), ester bonds (\u2014C(R)H\u2014C\u2014O\u2014O\u2014C(R)\u2014N\u2014), ketomethylen bonds (\u2014CO\u2014CH2\u2014), \u03b1-aza bonds (\u2014NH\u2014N(R)\u2014CO\u2014), wherein R is any alkyl, e.g., methyl, carba bonds (\u2014CH2-NH\u2014), hydroxyethylene bonds (\u2014CH(OH)\u2014CH\u2014), thioamide bonds (\u2014CS\u2014NH\u2014), olefinic double bonds (\u2014CH\u2550CH\u2014), retro amide bonds (\u2014NH\u2014CO\u2014), peptide derivatives (\u2014N(R)\u2014CH2-CO\u2014), wherein R is the \u201cnormal\u201d side chain, naturally presented on the carbon atom.","These modifications can occur at any of the bonds along the peptide chain and even at several (2-3) at the same time.","Natural aromatic amino acids, Trp, Tyr and Phe, may be substituted by synthetic non-natural acid such as Phenylglycine, TIC, naphthylelanine (Nol), ring-methylated derivatives of Phe, halogenated derivatives of Phe or o-methyl-Tyr.","In addition to the above, the peptides of the present invention may also include one or more modified amino acids or one or more non-amino acid monomers (e.g. fatty acids, complex carbohydrates etc).","As used herein in the specification and in the claims section below the term \u201camino acid\u201d or \u201camino acids\u201d is understood to include the 20 naturally occurring amino acids; those amino acids often modified post-translationally in vivo, including, for example, hydroxyproline, phosphoserine and phosphothreonine; and other unusual amino acids including, but not limited to, 2-aminoadipic acid, hydroxylysine, isodesmosine, nor-valine, nor-leucine and ornithine. Furthermore, the term \u201camino acid\u201d includes both D- and L-amino acids.","Since the peptides of the present invention are preferably utilized in therapeutics which require the peptides to be in soluble form, the peptides of the present invention preferably include one or more non-natural or natural polar amino acids, including but not limited to serine and threonine which are capable of increasing peptide solubility due to their hydroxyl-containing side chain.","The peptides of the present invention are preferably utilized in a linear form, although it will be appreciated that in cases where cyclization does not severely interfere with peptide characteristics, cyclic forms of the peptide can also be utilized.","The peptides of the present invention can be biochemically synthesized such as by using standard solid phase techniques. These methods include exclusive solid phase synthesis, partial solid phase synthesis methods, fragment condensation, classical solution synthesis. These methods are preferably used when the peptide is relatively short (i.e., 10 kDa) and\/or when it cannot be produced by recombinant techniques (i.e., not encoded by a nucleic acid sequence) and therefore involves different chemistry.","Solid phase peptide synthesis procedures are well known in the art and further described by John Morrow Stewart and Janis Dillaha Young, Solid Phase Peptide Syntheses (2nd Ed., Pierce Chemical Company, 1984).","Synthetic peptides can be purified by preparative high performance liquid chromatography [Creighton T. (1983) Proteins, structures and molecular principles. WH Freeman and Co. N.Y.] and the composition of which can be confirmed via amino acid sequencing.","In cases where large amounts of the peptides of the present invention are desired, the peptides of the present invention can be generated using recombinant techniques such as described by Bitter et al., (1987) Methods in Enzymol. 153:516-544, Studier et al. (1990) Methods in Enzymol. 185:60-89, Brisson et al. (1984) Nature 310:511-514, Takamatsu et al. (1987) EMBO J. 6:307-311, Coruzzi et al. (1984) EMBO J. 3:1671-1680 and Brogli et al., (1984) Science 224:838-843, Gurley et al. (1986) Mol. Cell. Biol. 6:559-565 and Weissbach & Weissbach, 1988, Methods for Plant Molecular Biology, Academic Press, NY, Section VIII, pp 421-463.","Expression Systems","To enable cellular expression of the polynucleotides of the present invention, a nucleic acid construct according to the present invention may be used, which includes at least a coding region of one of the above nucleic acid sequences, and further includes at least one cis acting regulatory element. As used herein, the phrase \u201ccis acting regulatory element\u201d refers to a polynucleotide sequence, preferably a promoter, which binds a trans acting regulator and regulates the transcription of a coding sequence located downstream thereto.","Any suitable promoter sequence can be used by the nucleic acid construct of the present invention.","Preferably, the promoter utilized by the nucleic acid construct of the present invention is active in the specific cell population transformed. Examples of cell type-specific and\/or tissue-specific promoters include promoters such as albumin that is liver specific [Pinkert et al., (1987) Genes Dev. 1:268-277], lymphoid specific promoters [Calame et al., (1988) Adv. Immunol. 43:235-275]; in particular promoters of T-cell receptors [Winoto et al., (1989) EMBO J. 8:729-733] and immunoglobulins; [Banerji et al. (1983) Cell 33729-740], neuron-specific promoters such as the neurofilament promoter [Byrne et al. (1989) Proc. Natl. Acad. Sci. USA 86:5473-5477], pancreas-specific promoters [Edlunch et al. (1985) Science 230:912-916] or mammary gland-specific promoters such as the milk whey promoter (U.S. Pat. No. 4,873,316 and European Application Publication No. 264,166). The nucleic acid construct of the present invention can further include an enhancer, which can be adjacent or distant to the promoter sequence and can function in up regulating the transcription therefrom.","The nucleic acid construct of the present invention preferably further includes an appropriate selectable marker and\/or an origin of replication. Preferably, the nucleic acid construct utilized is a shuttle vector, which can propagate both in (wherein the construct comprises an appropriate selectable marker and origin of replication) and be compatible for propagation in cells, or integration in a gene and a tissue of choice. The construct according to the present invention can be, for example, a plasmid, a bacmid, a phagemid, a cosmid, a phage, a virus or an artificial chromosome.","Examples of suitable constructs include, but are not limited to, pcDNA3, pcDNA3.1 (+\/\u2212), pGL3, PzeoSV2 (+\/\u2212), pDisplay, pEF\/myc\/cyto, pCMV\/myc\/cyto each of which is commercially available from Invitrogen Co. (www.invitrogen.com). Examples of retroviral vector and packaging systems are those sold by Clontech, San Diego, Calif., including Retro-X vectors pLNCX and pLXSN, which permit cloning into multiple cloning sites and the transgene is transcribed from CMV promoter. Vectors derived from Mo-MuLV are also included such as pBabe, where the transgene will be transcribed from the 5\u2032LTR promoter.","Currently preferred in vivo nucleic acid transfer techniques include transfection with viral or non-viral constructs, such as adenovirus, lentivirus, Herpes simplex I virus, or adeno-associated virus (AAV) and lipid-based systems. Useful lipids for lipid-mediated transfer of the gene are, for example, DOTMA, DOPE, and DC-Chol [Tonkinson et al., Cancer Investigation, 14(1): 54-65 (1996)]. The most preferred constructs for use in gene therapy are viruses, most preferably adenoviruses, AAV, lentiviruses, or retroviruses. A viral construct such as a retroviral construct includes at least one transcriptional promoter\/enhancer or locus-defining elements, or other elements that control gene expression by other means such as alternate splicing, nuclear RNA export, or post-translational modification of messenger. Such vector constructs also include a packaging signal, long terminal repeats (LTRs) or portions thereof, and positive and negative strand primer binding sites appropriate to the virus used, unless it is already present in the viral construct. In addition, such a construct typically includes a signal sequence for secretion of the peptide from a host cell in which it is placed. Preferably the signal sequence for this purpose is a mammalian signal sequence or the signal sequence of the polypeptides of the present invention. Optionally, the construct may also include a signal that directs polyadenylation, as well as one or more restriction sites and a translation termination sequence. By way of example, such constructs will typically include a 5\u2032 LTR, a tRNA binding site, a packaging signal, an origin of second-strand DNA synthesis, and a 3\u2032 LTR or a portion thereof. Other vectors can be used that are non-viral, such as cationic lipids, polylysine, and dendrimers.","Recombinant Expression Vectors and Host Cells","Another aspect of the invention pertains to vectors, preferably expression vectors, containing a nucleic acid encoding a protein of the invention, or derivatives, fragments, analogs or homologs thereof. As used herein, the term \u201cvector\u201d refers to a nucleic acid molecule capable of transporting another nucleic acid to which it has been linked. One type of vector is a \u201cplasmid\u201d, which refers to a circular double stranded DNA loop into which additional DNA segments can be ligated. Another type of vector is a viral vector, wherein additional DNA segments can be ligated into the viral genome. Certain vectors are capable of autonomous replication in a host cell into which they are introduced (e.g., bacterial vectors having a bacterial origin of replication and episomal mammalian vectors). Other vectors (e.g., non-episomal mammalian vectors) are integrated into the genome of a host cell upon introduction into the host cell, and thereby are replicated along with the host genome. Moreover, certain vectors are capable of directing the expression of genes to which they are operatively-linked. Such vectors are referred to herein as \u201cexpression vectors\u201d. In general, expression vectors of utility in recombinant DNA techniques are often in the form of plasmids. In the present specification, \u201cplasmid\u201d and \u201cvector\u201d can be used interchangeably as the plasmid is the most commonly used form of vector. However, the invention is intended to include such other forms of expression vectors, such as viral vectors (e.g., replication defective retroviruses, adenoviruses and adeno-associated viruses), which serve equivalent functions.","The recombinant expression vectors of the invention comprise a nucleic acid of the invention in a form suitable for expression of the nucleic acid in a host cell, which means that the recombinant expression vectors include one or more regulatory sequences, selected on the basis of the host cells to be used for expression, that is operatively-linked to the nucleic acid sequence to be expressed. Within a recombinant expression vector, \u201coperably-linked\u201d is intended to mean that the nucleotide sequence of interest is linked to the regulatory sequences in a manner that allows for expression of the nucleotide sequence (e.g., in an in vitro transcription\/translation system or in a host cell when the vector is introduced into the host cell).","The term \u201cregulatory sequence\u201d is intended to includes promoters, enhancers and other expression control elements (e.g., polyadenylation signals). Such regulatory sequences are described, for example, in Goeddel, Gene Expression Technology Methods in Enzymology 185, Academic Press, San Diego, Calif. (1990). Regulatory sequences include those that direct constitutive expression of a nucleotide sequence in many types of host cell and those that direct expression of the nucleotide sequence only in certain host cells (e.g., tissue-specific regulatory sequences). It will be appreciated by those skilled in the art that the design of the expression vector can depend on such factors as the choice of the host cell to be transformed, the level of expression of protein desired, etc. The expression vectors of the invention can be introduced into host cells to thereby produce proteins or peptides, including fusion proteins or peptides, encoded by nucleic acids as described herein.","The recombinant expression vectors of the invention can be designed for production of variant proteins in prokaryotic or eukaryotic cells. For example, proteins of the invention can be expressed in bacterial cells such as , insect cells (using baculovirus expression vectors) yeast cells or mammalian cells. Suitable host cells are discussed further in Goeddel, Gene Expression Technology: Methods in Enzymology 185, Academic Press, San Diego, Calif. (1990). Alternatively, the recombinant expression vector can be transcribed and translated in vitro, for example using T7 promoter regulatory sequences and T7 polymerase.","Expression of proteins in prokaryotes is most often carried out in with vectors containing constitutive or inducible promoters directing the expression of either fusion or non-fusion proteins. Fusion vectors add a number of amino acids to a protein encoded therein, to the amino or C terminus of the recombinant protein. Such fusion vectors typically serve three purposes: (i) to increase expression of recombinant protein; (ii) to increase the solubility of the recombinant protein; and (iii) to aid in the purification of the recombinant protein by acting as a ligand in affinity purification. Often, in fusion expression vectors, a proteolytic cleavage site is introduced at the junction of the fusion moiety and the recombinant protein to enable separation of the recombinant protein from the fusion moiety subsequent to purification of the fusion protein. Such enzymes, and their cognate recognition sequences, include Factor Xa, thrombin, PreScission, TEV and enterokinase. Typical fusion expression vectors include pGEX (Pharmacia Biotech Inc; Smith and Johnson, 1988. Gene 67: 31-40), pMAL (New England Biolabs, Beverly, Mass.) and pRIT5 (Pharmacia, Piscataway, N.J.) that fuse glutathione S-transferase (GST), maltose E binding protein, or protein A, respectively, to the target recombinant protein.","Examples of suitable inducible non-fusion expression vectors include pTrc (Amrann et al., (1988) Gene 69:301-315) and pET 11d (Studier et al., Gene Expression Technology: Methods in Enzymology 185, Academic Press, San Diego, Calif. (1990) 60-89)\u2014not accurate, pET11a-d have N terminal T7 tag.","One strategy to maximize recombinant protein expression in is to express the protein in a host bacterium with an impaired capacity to proteolytically cleave the recombinant protein. See, e.g., Gottesman, Gene Expression Technology: Methods in Enzymology 185, Academic Press, San Diego, Calif. (1990) 119-128. Another strategy is to alter the nucleic acid sequence of the nucleic acid to be inserted into an expression vector so that the individual codons for each amino acid are those preferentially utilized in (see, e.g., Wada, et al., 1992. Nucl. Acids Res. 20: 2111-2118). Such alteration of nucleic acid sequences of the invention can be carried out by standard DNA synthesis techniques. Another strategy to solve codon bias is by using BL21-codon plus bacterial strains (Invitrogen) or Rosetta bacterial strain (Novagen), these strains contain extra copies of rare tRNA genes.","In another embodiment, the expression vector encoding for the protein of the invention is a yeast expression vector. Examples of vectors for expression in yeast include pYepSec1 (Baldari, et al., 1987. EMBO J. 6: 229-234), pMFa (Kurjan and Herskowitz, 1982. Cell 30: 933-943), pJRY88 (Schultz et al., 1987. Gene 54: 113-123), pYES2 (Invitrogen Corporation, San Diego, Calif.), and picZ (InVitrogen Corp, San Diego, Calif.).","Alternatively, polypeptides of the present invention can be produced in insect cells using baculovirus expression vectors. Baculovirus vectors available for expression of proteins in cultured insect cells (e.g., SF9 cells) include the pAc series (Smith, et al., 1983. Mol. Cell. Biol. 3: 2156-2165) and the pVL series (Lucklow and Summers, 1989. Virology 170: 31-39).","In yet another embodiment, a nucleic acid of the invention is expressed in mammalian cells using a mammalian expression vector. Examples of mammalian expression vectors include pCDM8 (Seed, 1987. Nature 329: 840) and pMT2PC (Kaufman, et al., 1987. EMBO J. 6: 187-195), pIRESpuro (Clontech), pUB6 (Invitrogen), pCEP4 (Invitrogen) pREP4 (Invitrogen), pcDNA3 (Invitrogen). When used in mammalian cells, the expression vector's control functions are often provided by viral regulatory elements. For example, commonly used promoters are derived from polyoma, adenovirus 2, cytomegalovirus, Rous Sarcoma Virus, and simian virus 40. For other suitable expression systems for both prokaryotic and eukaryotic cells see, e.g., Chapters 16 and 17 of Sambrook, et al., Molecular Cloning: A Laboratory Manual. 2nd ed., Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1989.","In another embodiment, the recombinant mammalian expression vector is capable of directing expression of the nucleic acid preferentially in a particular cell type (e.g., tissue-specific regulatory elements are used to express the nucleic acid). Tissue-specific regulatory elements are known in the art. Non-limiting examples of suitable tissue-specific promoters include the albumin promoter (liver-specific; Pinkert, et al., 1987. Genes Dev. 1: 268-277), lymphoid-specific promoters (Calame and Eaton, 1988. Adv. Immunol. 43: 235-275), in particular promoters of T cell receptors (Winoto and Baltimore, 1989. EMBO J. 8: 729-733) and immunoglobulins (Banerji, et al., 1983. Cell 33: 729-740; Queen and Baltimore, 1983. Cell 33: 741-748), neuron-specific promoters (e.g., the neurofilament promoter; Byrne and Ruddle, 1989. Proc. Natl. Acad. Sci. USA 86: 5473-5477), pancreas-specific promoters (Edlund, et al., 1985. Science 230: 912-916), and mammary gland-specific promoters (e.g., milk whey promoter; U.S. Pat. No. 4,873,316 and European Application Publication No. 264,166). Developmentally-regulated promoters are also encompassed, e.g., the murine hox promoters (Kessel and Gruss, 1990. Science 249: 374-379) and the alpha-fetoprotein promoter (Campes and Tilghman, 1989. Genes Dev. 3: 537-546).","The invention further provides a recombinant expression vector comprising a DNA molecule of the invention cloned into the expression vector in an antisense orientation. That is, the DNA molecule is operatively-linked to a regulatory sequence in a manner that allows for expression (by transcription of the DNA molecule) of an RNA molecule that is antisense to mRNA encoding for protein of the invention. Regulatory sequences operatively linked to a nucleic acid cloned in the antisense orientation can be chosen that direct the continuous expression of the antisense RNA molecule in a variety of cell types, for instance viral promoters and\/or enhancers, or regulatory sequences can be chosen that direct constitutive, tissue specific or cell type specific expression of antisense RNA. The antisense expression vector can be in the form of a recombinant plasmid, phagemid or attenuated virus in which antisense nucleic acids are produced under the control of a high efficiency regulatory region, the activity of which can be determined by the cell type into which the vector is introduced. For a discussion of the regulation of gene expression using antisense genes see, e.g., Weintraub, et al., \u201cAntisense RNA as a molecular tool for genetic analysis,\u201d Reviews-Trends in Genetics, Vol. 1(1) 1986.","Another aspect of the invention pertains to host cells into which a recombinant expression vector of the invention has been introduced. The terms \u201chost cell\u201d and \u201crecombinant host cell\u201d are used interchangeably herein. It is understood that such terms refer not only to the particular subject cell but also to the progeny or potential progeny of such a cell. Because certain modifications may occur in succeeding generations due to either mutation or environmental influences, such progeny may not, in fact, be identical to the parent cell, but are still included within the scope of the term as used herein.","A host cell can be any prokaryotic or eukaryotic cell. For example, protein of the invention can be produced in bacterial cells such as , insect cells, yeast, plant or mammalian cells (such as Chinese hamster ovary cells (CHO) or COS or 293 cells). Other suitable host cells are known to those skilled in the art.","Vector DNA can be introduced into prokaryotic or eukaryotic cells via conventional transformation or transfection techniques. As used herein, the terms \u201ctransformation\u201d and \u201ctransfection\u201d are intended to refer to a variety of art-recognized techniques for introducing foreign nucleic acid (e.g., DNA) into a host cell, including calcium phosphate or calcium chloride co-precipitation, DEAE-dextran-mediated transfection, lipofection, or electroporation. Suitable methods for transforming or transfecting host cells can be found in Sambrook, et al. (Molecular Cloning: A Laboratory Manual. 2nd ed., Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1989), and other laboratory manuals.","For stable transfection of mammalian cells, it is known that, depending upon the expression vector and transfection technique used, only a small fraction of cells may integrate the foreign DNA into their genome. In order to identify and select these integrants, a gene that encodes a selectable marker (e.g., resistance to antibiotics) is generally introduced into the host cells along with the gene of interest. Various selectable markers include those that confer resistance to drugs, such as G418, hygromycin, puromycin, blasticidin and methotrexate. Nucleic acids encoding a selectable marker can be introduced into a host cell on the same vector as that encoding protein of the invention or can be introduced on a separate vector. Cells stably transfected with the introduced nucleic acid can be identified by drug selection (e.g., cells that have incorporated the selectable marker gene will survive, while the other cells die).","A host cell of the invention, such as a prokaryotic or eukaryotic host cell in culture, can be used to produce (i.e., express) protein of the invention. Accordingly, the invention further provides methods for producing proteins of the invention using the host cells of the invention. In one embodiment, the method comprises culturing the host cell of the present invention (into which a recombinant expression vector encoding protein of the invention has been introduced) in a suitable medium such that the protein of the invention is produced. In another embodiment, the method further comprises isolating protein of the invention from the medium or the host cell.","For efficient production of the protein, it is preferable to place the nucleotide sequences encoding the protein of the invention under the control of expression control sequences optimized for expression in a desired host. For example, the sequences may include optimized transcriptional and\/or translational regulatory sequences (such as altered Kozak sequences).","Protein Modifications","Fusion Proteins","According to the present invention, a fusion protein may be prepared from a protein of the invention by fusion with a portion of an immunoglobulin comprising a constant region of an immunoglobulin. More preferably, the portion of the immunoglobulin comprises a heavy chain constant region which is optionally and more preferably a human heavy chain constant region. The heavy chain constant region is most preferably an IgG heavy chain constant region, and optionally and most preferably is an Fc chain, most preferably an IgG Fc fragment that comprises CH2 and CH3 domains. Although any IgG subtype may optionally be used, the IgG1 subtype is preferred. The Fc chain may optionally be a known or \u201cwild type\u201d Fc chain, or alternatively may be mutated. Non-limiting, illustrative, exemplary types of mutations are described in US Patent Application No. 20060034852, published on Feb. 16, 2006, hereby incorporated by reference as if fully set forth herein. The term \u201cFc chain\u201d also optionally comprises any type of Fc fragment.","Several of the specific amino acid residues that are important for antibody constant region-mediated activity in the IgG subclass have been identified. Inclusion, substitution or exclusion of these specific amino acids therefore allows for inclusion or exclusion of specific immunoglobulin constant region-mediated activity. Furthermore, specific changes may result in aglycosylation for example and\/or other desired changes to the Fc chain. At least some changes may optionally be made to block a function of Fc which is considered to be undesirable, such as an undesirable immune system effect, as described in greater detail below.","Non-limiting, illustrative examples of mutations to Fc which may be made to modulate the activity of the fusion protein include the following changes (given with regard to the Fc sequence nomenclature as given by Kabat, from Kabat E A et al: Sequences of Proteins of Immunological Interest. US Department of Health and Human Services, NIH, 1991): 220C\u2192S; 233-238 ELLGGP\u2192EAEGAP; 265D\u2192A, preferably in combination with 434N\u2192A; 297N\u2192A (for example to block N-glycosylation); 318-322 EYKCK\u2192AYACA; 330-331AP\u2192SS; or a combination thereof (see for example M. Clark, \u201cChemical Immunol and Antibody Engineering\u201d, pp 1-31 for a description of these mutations and their effect). The construct for the Fc chain which features the above changes optionally and preferably comprises a combination of the hinge region with the CH2 and CH3 domains.","The above mutations may optionally be implemented to enhance desired properties or alternatively to block non-desired properties. For example, aglycosylation of antibodies was shown to maintain the desired binding functionality while blocking depletion of T-cells or triggering cytokine release, which may optionally be undesired functions (see M. Clark, \u201cChemical Immunol and Antibody Engineering\u201d, pp 1-31). Substitution of 331proline for serine may block the ability to activate complement, which may optionally be considered an undesired function (see M. Clark, \u201cChemical Immunol and Antibody Engineering\u201d, pp 1-31). Changing 330alanine to serine in combination with this change may also enhance the desired effect of blocking the ability to activate complement.","Residues 235 and 237 were shown to be involved in antibody-dependent cell-mediated cytotoxicity (ADCC), such that changing the block of residues from 233-238 as described may also block such activity if ADCC is considered to be an undesirable function.","Residue 220 is normally a cysteine for Fc from IgG1, which is the site at which the heavy chain forms a covalent linkage with the light chain. Optionally, this residue may be changed to a serine, to avoid any type of covalent linkage (see M. Clark, \u201cChemical Immunol and Antibody Engineering\u201d, pp 1-31).","The above changes to residues 265 and 434 may optionally be implemented to reduce or block binding to the Fc receptor, which may optionally block undesired functionality of Fc related to its immune system functions (see \u201cBinding site on Human IgG1 for Fc Receptors\u201d, Shields et al, Vol 276, pp 6591-6604, 2001).","The above changes are intended as illustrations only of optional changes and are not meant to be limiting in any way. Furthermore, the above explanation is provided for descriptive purposes only, without wishing to be bound by a single hypothesis.","Addition of Groups","If a protein according to the present invention is a linear molecule, it is possible to place various functional groups at various points on the linear molecule which are susceptible to or suitable for chemical modification. Functional groups can be added to the termini of linear forms of the protein of the invention. In some embodiments, the functional groups improve the activity of the protein with regard to one or more characteristics, including but not limited to, improvement in stability, penetration (through cellular membranes and\/or tissue barriers), tissue localization, efficacy, decreased clearance, decreased toxicity, improved selectivity, improved resistance to expulsion by cellular pumps, and the like. For convenience sake and without wishing to be limiting, the free N-terminus of one of the sequences contained in the compositions of the invention will be termed as the N-terminus of the composition, and the free C-terminal of the sequence will be considered as the C-terminus of the composition. Either the C-terminus or the N-terminus of the sequences, or both, can be linked to a carboxylic acid functional groups or an amine functional group, respectively.","Non-limiting examples of suitable functional groups are described in Green and Wuts, \u201cProtecting Groups in Organic Synthesis\u201d, John Wiley and Sons, Chapters 5 and 7, 1991, the teachings of which are incorporated herein by reference. Preferred protecting groups are those that facilitate transport of the active ingredient attached thereto into a cell, for example, by reducing the hydrophilicity and increasing the lipophilicity of the active ingredient, these being an example for \u201ca moiety for transport across cellular membranes\u201d.","These moieties can optionally and preferably be cleaved in vivo, either by hydrolysis or enzymatically, inside the cell. (Ditter et al., J. Pharm. Sci. 57:783 (1968); Ditter et al., J. Pharm. Sci. 57:828 (1968); Ditter et al., J. Pharm. Sci. 58:557 (1969); King et al., Biochemistry 26:2294 (1987); Lindberg et al., Drug Metabolism and Disposition 17:311 (1989); and Tunek et al., Biochem. Pharm. 37:3867 (1988), Anderson et al., Arch. Biochem. Biophys. 239:538 (1985) and Singhal et al., FASEB J. 1:220 (1987)). Hydroxyl protecting groups include esters, carbonates and carbamate protecting groups. Amine protecting groups include alkoxy and aryloxy carbonyl groups, as described above for N-terminal protecting groups. Carboxylic acid protecting groups include aliphatic, benzylic and aryl esters, as described above for C-terminal protecting groups. In one embodiment, the carboxylic acid group in the side chain of one or more glutamic acid or aspartic acid residue in a composition of the present invention is protected, preferably with a methyl, ethyl, benzyl or substituted benzyl ester, more preferably as a benzyl ester.","Non-limiting, illustrative examples of N-terminal protecting groups include acyl groups (\u2014CO\u2014R1) and alkoxy carbonyl or aryloxy carbonyl groups (\u2212CO\u2014O\u2014R1), wherein R1 is an aliphatic, substituted aliphatic, benzyl, substituted benzyl, aromatic or a substituted aromatic group. Specific examples of acyl groups include but are not limited to acetyl, (ethyl)-CO\u2014, n-propyl-CO\u2014, iso-propyl-CO\u2014, n-butyl-CO\u2014, sec-butyl-CO\u2014, t-butyl-CO\u2014, hexyl, lauroyl, palmitoyl, myristoyl, stearyl, oleoyl phenyl-CO\u2014, substituted phenyl-CO\u2014, benzyl-CO\u2014 and (substituted benzyl)-CO\u2014. Examples of alkoxy carbonyl and aryloxy carbonyl groups include CH3-O\u2014CO\u2014, (ethyl)-O\u2014CO\u2014, n-propyl-O\u2014CO\u2014, iso-propyl-O\u2014CO\u2014, n-butyl-O\u2014CO\u2014, sec-butyl-O\u2014CO\u2014, t-butyl-O\u2014CO\u2014, phenyl-O\u2014CO\u2014, substituted phenyl-O\u2014CO\u2014 and benzyl-O\u2014CO\u2014, (substituted benzyl)-O\u2014CO\u2014, Adamantan, naphtalen, myristoleyl, toluen, biphenyl, cinnamoyl, nitrobenzoy, toluoyl, furoyl, benzoyl, cyclohexane, norbornane, or Z-caproic. In order to facilitate the N-acylation, one to four glycine residues can be present in the N-terminus of the molecule.","The carboxyl group at the C-terminus of the compound can be protected, for example, by a group including but not limited to an amide (i.e., the hydroxyl group at the C-terminus is replaced with \u2014NH, \u2014NHRand \u2014NRR) or ester (i.e. the hydroxyl group at the C-terminus is replaced with \u2014OR). Rand Rare optionally independently an aliphatic, substituted aliphatic, benzyl, substituted benzyl, aryl or a substituted aryl group. In addition, taken together with the nitrogen atom, Rand Rcan optionally form a C4 to C8 heterocyclic ring with from about 0-2 additional heteroatoms such as nitrogen, oxygen or sulfur. Non-limiting suitable examples of suitable heterocyclic rings include piperidinyl, pyrrolidinyl, morpholino, thiomorpholino or piperazinyl. Examples of C-terminal protecting groups include but are not limited to \u2014NH, \u2014NHCH, \u2014N(CH), \u2014NH(ethyl), \u2014N(ethyl), \u2014N(methyl)(ethyl), \u2014NH(benzyl), \u2014N(C1-C4 alkyl)(benzyl), \u2014NH(phenyl), \u2014N(C1-C4 alkyl) (phenyl), \u2014OCH, \u2014O-(ethyl), \u2014O-(n-propyl), \u2014O-(n-butyl), \u2014O-(iso-propyl), \u2014O-(sec-butyl), \u2014O-(t-butyl), \u2014O-benzyl and \u2014O-phenyl.","Substitution by Peptidomimetic Moieties","A \u201cpeptidomimetic organic moiety\u201d can optionally be substituted for amino acid residues in the composition of this invention both as conservative and as non-conservative substitutions. These moieties are also termed \u201cnon-natural amino acids\u201d and may optionally replace amino acid residues, amino acids or act as spacer groups within the peptides in lieu of deleted amino acids. The peptidomimetic organic moieties optionally and preferably have steric, electronic or configurational properties similar to the replaced amino acid and such peptidomimetics are used to replace amino acids in the essential positions, and are considered conservative substitutions. However such similarities are not necessarily required. According to preferred embodiments of the present invention, one or more peptidomimetics are selected such that the composition at least substantially retains its physiological activity as compared to the native protein according to the present invention.","Peptidomimetics may optionally be used to inhibit degradation of the peptides by enzymatic or other degradative processes. The peptidomimetics can optionally and preferably be produced by organic synthetic techniques. Non-limiting examples of suitable peptidomimetics include D amino acids of the corresponding L amino acids, tetrazol (Zabrocki et al., J. Am. Chem. Soc. 110:5875-5880 (1988)); isosteres of amide bonds (Jones et al., Tetrahedron Lett. 29: 3853-3856 (1988)); LL-3-amino-2-propenidone-6-carboxylic acid (LL-Acp) (Kemp et al., J. Org. Chem. 50:5834-5838 (1985)). Similar analogs are shown in Kemp et al., Tetrahedron Lett. 29:5081-5082 (1988) as well as Kemp et al., Tetrahedron Lett. 29:5057-5060 (1988), Kemp et al., Tetrahedron Lett. 29:4935-4938 (1988) and Kemp et al., J. Org. Chem. 54:109-115 (1987). Other suitable but exemplary peptidomimetics are shown in Nagai and Sato, Tetrahedron Lett. 26:647-650 (1985); Di Maio et al., J. Chem. Soc. Perkin Trans., 1687 (1985); Kahn et al., Tetrahedron Lett. 30:2317 (1989); Olson et al., J. Am. Chem. Soc. 112:323-333 (1990); Garvey et al., J. Org. Chem. 56:436 (1990). Further suitable exemplary peptidomimetics include hydroxy-1,2,3,4-tetrahydroisoquinoline-3-carboxylate (Miyake et al., J. Takeda Res. Labs 43:53-76 (1989)); 1,2,3,4-tetrahydro-isoquinoline-3-carboxylate (Kazmierski et al., J. Am. Chem. Soc. 133:2275-2283 (1991)); histidine isoquinolone carboxylic acid (HIC) (Zechel et al., Int. J. Pep. Protein Res. 43 (1991)); (2S,3S)-methyl-phenylalanine, (2S, 3R)-methyl-phenylalanine, (2R,3S)-methyl-phenylalanine and (2R, 3R)-methyl-phenylalanine (Kazmierski and Hruby, Tetrahedron Lett. (1991)).","Exemplary, illustrative but non-limiting non-natural amino acids include beta-amino acids (beta3 and beta2), homo-amino acids, cyclic amino acids, aromatic amino acids, Pro and Pyr derivatives, 3-substituted Alanine derivatives, Glycine derivatives, ring-substituted Phe and Tyr Derivatives, linear core amino acids or diamino acids. They are available from a variety of suppliers, such as Sigma-Aldrich (USA) for example.","Chemical Modifications","In the present invention any part of a protein of the invention may optionally be chemically modified, i.e. changed by addition of functional groups. For example the side amino acid residues appearing in the native sequence may optionally be modified, although as described below alternatively other parts of the protein may optionally be modified, in addition to or in place of the side amino acid residues. The modification may optionally be performed during synthesis of the molecule if a chemical synthetic process is followed, for example by adding a chemically modified amino acid. However, chemical modification of an amino acid when it is already present in the molecule (\u201cin situ\u201d modification) is also possible.","The amino acid of any of the sequence regions of the molecule can optionally be modified according to any one of the following exemplary types of modification (in the peptide conceptually viewed as \u201cchemically modified\u201d). Non-limiting exemplary types of modification include carboxymethylation, acylation, phosphorylation, glycosylation or fatty acylation. Ether bonds can optionally be used to join the serine or threonine hydroxyl to the hydroxyl of a sugar. Amide bonds can optionally be used to join the glutamate or aspartate carboxyl groups to an amino group on a sugar (Garg and Jeanloz, Advances in Carbohydrate Chemistry and Biochemistry, Vol. 43, Academic Press (1985); Kunz, Ang. Chem. Int. Ed. English 26:294-308 (1987)). Acetal and ketal bonds can also optionally be formed between amino acids and carbohydrates. Fatty acid acyl derivatives can optionally be made, for example, by acylation of a free amino group (e.g., lysine) (Toth et al., Peptides: Chemistry, Structure and Biology, Rivier and Marshal, eds., ESCOM Publ., Leiden, 1078-1079 (1990)).","As used herein the term \u201cchemical modification\u201d, when referring to a protein or peptide according to the present invention, refers to a protein or peptide where at least one of its amino acid residues is modified either by natural processes, such as processing or other post-translational modifications, or by chemical modification techniques which are well known in the art. Examples of the numerous known modifications typically include, but are not limited to: acetylation, acylation, amidation, ADP-ribosylation, glycosylation, GPI anchor formation, covalent attachment of a lipid or lipid derivative, methylation, myristylation, pegylation, prenylation, phosphorylation, ubiquitination, or any similar process.","Other types of modifications optionally include the addition of a cycloalkane moiety to a biological molecule, such as a protein, as described in PCT Application No. WO 2006\/050262, hereby incorporated by reference as if fully set forth herein. These moieties are designed for use with biomolecules and may optionally be used to impart various properties to proteins.","Furthermore, optionally any point on a protein may be modified. For example, pegylation of a glycosylation moiety on a protein may optionally be performed, as described in PCT Application No. WO 2006\/050247, hereby incorporated by reference as if fully set forth herein. One or more polyethylene glycol (PEG) groups may optionally be added to O-linked and\/or N-linked glycosylation. The PEG group may optionally be branched or linear. Optionally any type of water-soluble polymer may be attached to a glycosylation site on a protein through a glycosyl linker.","Altered Glycosylation","Proteins of the invention may be modified to have an altered glycosylation pattern (i.e., altered from the original or native glycosylation pattern). As used herein, \u201caltered\u201d means having one or more carbohydrate moieties deleted, and\/or having at least one glycosylation site added to the original protein.","Glycosylation of proteins is typically either N-linked or O-linked. N-linked refers to the attachment of the carbohydrate moiety to the side chain of an asparagine residue. The tripeptide sequences, asparagine-X-serine and asparagine-X-threonine, where X is any amino acid except proline, are the recognition sequences for enzymatic attachment of the carbohydrate moiety to the asparagine side chain. Thus, the presence of either of these tripeptide sequences in a polypeptide creates a potential glycosylation site. O-linked glycosylation refers to the attachment of one of the sugars N-acetylgalactosamine, galactose, or xylose to a hydroxyamino acid, most commonly serine or threonine, although 5-hydroxyproline or 5-hydroxylysine may also be used.","Addition of glycosylation sites to proteins of the invention is conveniently accomplished by altering the amino acid sequence of the protein such that it contains one or more of the above-described tripeptide sequences (for N-linked glycosylation sites). The alteration may also be made by the addition of, or substitution by, one or more serine or threonine residues in the sequence of the original protein (for O-linked glycosylation sites). The protein's amino acid sequence may also be altered by introducing changes at the DNA level.","Another means of increasing the number of carbohydrate moieties on proteins is by chemical or enzymatic coupling of glycosides to the amino acid residues of the protein. Depending on the coupling mode used, the sugars may be attached to (a) arginine and histidine, (b) free carboxyl groups, (c) free sulfhydryl groups such as those of cysteine, (d) free hydroxyl groups such as those of serine, threonine, or hydroxyproline, (e) aromatic residues such as those of phenylalanine, tyrosine, or tryptophan, or (f) the amide group of glutamine. These methods are described in WO 87\/05330, and in Aplin and Wriston, CRC Crit. Rev. Biochem., 22: 259-306 (1981).","Removal of any carbohydrate moieties present on proteins of the invention may be accomplished chemically or enzymatically. Chemical deglycosylation requires exposure of the protein to trifluoromethanesulfonic acid, or an equivalent compound. This treatment results in the cleavage of most or all sugars except the linking sugar (N-acetylglucosamine or N-acetylgalactosamine), leaving the amino acid sequence intact.","Chemical deglycosylation is described by Hakimuddin et al., Arch. Biochem. Biophys., 259: 52 (1987); and Edge et al., Anal. Biochem., 118: 131 (1981). Enzymatic cleavage of carbohydrate moieties on proteins can be achieved by the use of a variety of endo- and exo-glycosidases as described by Thotakura et al., Meth. Enzymol., 138: 350 (1987).","Methods of Treatment","As mentioned hereinabove the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins or VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins and polypeptides of the present invention or nucleic acid sequence or fragments thereof especially the ectodomain or secreted forms of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins, as well as drugs which specifically bind to the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins and\/or splice variants, and\/or drugs which agonize or antagonize the binding of other moieties to the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins and\/or splice variants, and\/or drugs which modulate (agonize or antagonize) at least one VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 related biological activity (such drugs include by way of example antibodies, small molecules, peptides, ribozymes, antisense molecules, siRNA's and the like), can be used to treat cancer, including but not limited to non-solid and solid tumors, sarcomas, hematological malignancies including but not limited to acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, cancer of the breast, prostate, lung, ovary, colon, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, brain and wherein the cancer may be non-metastatic, invasive or metastatic.","The VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins or VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins and polypeptides of the present invention or nucleic acid sequence or fragments thereof especially the ectodomain or secreted forms of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins, as well as drugs which specifically bind to the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins and\/or splice variants, and\/or drugs which agonize or antagonize the binding of other moieties to the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 proteins and\/or splice variants, and\/or drugs which modulate (agonize or antagonize) at least one VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 related biological activity (such drugs include by way of example antibodies, small molecules, peptides, ribozymes, antisense molecules, siRNA's and the like), can be further used to treat non-malignant disorders such as immune disorders including but not limited to autoimmune diseases, transplant rejection and graft versus host disease, and\/or for blocking or promoting immune costimulation mediated by the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, FXYD3 polypeptide.","Thus, according to an additional aspect of the present invention there is provided a method of treating cancer, including but not limited to non-solid and solid tumors, sarcomas, hematological malignancies including but not limited to acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, cancer of the breast, prostate, lung, ovary, colon, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, brain and wherein the cancer may be non-metastatic, invasive or metastatic as well as non-malignant disorders such as immune disorders including but not limited to autoimmune diseases, transplant rejection and graft versus host disease, and\/or for blocking or promoting immune costimulation mediated by the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 polypeptide in a subject.","The subject according to the present invention is a mammal, preferably a human which is diagnosed with one of the disease, disorder or conditions described hereinabove, or alternatively is predisposed to at least one type of cancer, including but not limited to non-solid and solid tumors, sarcomas, hematological malignancies including but not limited to acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, cancer of the breast, prostate, lung, ovary, colon, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, brain, as well as non-malignant disorders such as immune disorders including but not limited to autoimmune diseases, transplant rejection and graft versus host disease.","As used herein the term \u201ctreating\u201d refers to preventing, curing, reversing, attenuating, alleviating, minimizing, suppressing or halting the deleterious effects of the above-described diseases, disorders or conditions.","Treating, according to the present invention, can be effected by specifically upregulating the expression of at least one of the polypeptides of the present invention in the subject.","Optionally, upregulation may be effected by administering to the subject at least one of the polypeptides of the present invention (e.g., recombinant or synthetic) or an active portion thereof, as described herein. However, since the bioavailability of large polypeptides may potentially be relatively small due to high degradation rate and low penetration rate, administration of polypeptides is preferably confined to small peptide fragments (e.g., about 100 amino acids). The polypeptide or peptide may optionally be administered in as part of a pharmaceutical composition, described in more detail below.","It will be appreciated that treatment of the above-described diseases according to the present invention may be combined with other treatment methods known in the art (i.e., combination therapy). Thus, treatment of malignancies using the agents of the present invention may be combined with, for example, radiation therapy, antibody therapy and\/or chemotherapy.","Alternatively or additionally, an upregulating method may optionally be effected by specifically upregulating the amount (optionally expression) in the subject of at least one of the polypeptides of the present invention or active portions thereof.","As is mentioned hereinabove and in the Examples section which follows, the biomolecular sequences of this aspect of the present invention may be used as valuable therapeutic tools in the treatment of diseases, disorders or conditions in which altered activity or expression of the wild-type gene product (known protein) is known to contribute to disease, disorder or condition onset or progression. For example, in case a disease is caused by overexpression of a membrane bound-receptor, a soluble variant thereof may be used as an antagonist which competes with the receptor for binding the ligand, to thereby terminate signaling from the receptor.","Anti-VSIG1, Anti-ILDR1, Anti-LOC253012, Anti-AI216611, Anti-C1ORF32, Anti-FXYD3 Antibodies","The antibodies of the invention including those having the particular germline sequences, homologous antibodies, antibodies with conservative modifications, engineered and modified antibodies are characterized by particular functional features or properties of the antibodies. For example, the antibodies bind specifically to human VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3. Preferably, an antibody of the invention binds to corresponding VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 with high affinity, for example with a KD of 10\u22128 M or less or 10\u22129M or less or even 10\u221210 M or less. The anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibodies of the invention preferably exhibit one or more of the following characteristics:","(i) binds to corresponding human VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 with a KD of 5\u00d710\u22128 M or less;","(ii) modulates (enhances or inhibits) B7 immune costimulation and related activities and functions such a T cell responses involved in antitumor immunity and autoimmunity. and\/or","(iii) binds to VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen expressed by cancer cells including for example lung cancer, ovarian cancer, colon cancer, but does not substantially bind to normal cells In addition, preferably these antibodies and conjugates thereof will be effective in eliciting selective killing of such cancer cells and for modulating immune responses involved in autoimmunity and cancer.","More preferably, the antibody binds to corresponding human VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen with a KD of 3\u00d710\u22128 M or less, or with a KD of 1\u00d710\u22129M or less, or with a KD of 0.1\u00d710\u22129M or less, or with a KD Of 0.05\u00d710\u22129M or less or with a KD of between 1\u00d710\u22129 and 1\u00d710\u221211 M.","Standard assays to evaluate the binding ability of the antibodies toward VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 are known in the art, including for example, ELISAs, Western blots and RIAs. Suitable assays are described in detail in the Examples. The binding kinetics (e.g., binding affinity) of the antibodies also can be assessed by standard assays known in the art, such as by Biacore analysis.","Upon production of anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibody sequences from antibodies can bind to VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 the VH and VL sequences can be \u201cmixed and matched\u201d to create other anti-VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 binding molecules of the invention. VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 binding of such \u201cmixed and matched\u201d antibodies can be tested using the binding assays described above. e.g., ELISAs). Preferably, when VH and VL chains are mixed and matched, a VH sequence from a particular VH\/VL pairing is replaced with a structurally similar VH sequence. Likewise, preferably a VL sequence from a particular VH\/VL pairing is replaced with a structurally similar VL sequence. For example, the VH and VL sequences of homologous antibodies are particularly amenable for mixing and matching.","Antibodies Having Particular Germline Sequences","In certain embodiments, an antibody of the invention comprises a heavy chain variable region from a particular germline heavy chain immunoglobulin gene and\/or a light chain variable region from a particular germline light chain immunoglobulin gene.","As used herein, a human antibody comprises heavy or light chain variable regions that is \u201cthe product of\u201d or \u201cderived from\u201d a particular germline sequence if the variable regions of the antibody are obtained from a system that uses human germline immunoglobulin genes. Such systems include immunizing a transgenic mouse carrying human immunoglobulin genes with the antigen of interest or screening a human immunoglobulin gene library displayed on phage with the antigen of interest. A human antibody that is \u201cthe product\u201d of or \u201cderived from\u201d a human germline immunoglobulin sequence can be identified as such by comparing the amino acid sequence of the human antibody to the amino acid sequences of human germline immunoglobulins and selecting the human germline immunoglobulin sequence that is closest in sequence (i.e., greatest % identity) to the sequence of the human antibody.","A human antibody that is \u201cthe product of\u201d or \u201cderived from\u201d a particular human germline immunoglobulin sequence may contain amino acid differences as compared to the germline sequence, due to, for example, naturally-occurring somatic mutations or intentional introduction of site-directed mutation. However, a selected human antibody typically is at least 90% identical in amino acids sequence to an amino acid sequence encoded by a human germline immunoglobulin gene and contains amino acid residues that identify the human antibody as being human when compared to the germline immunoglobulin amino acid sequences of other species (e.g., murine germline sequences). In certain cases, a human antibody may be at least 95, 96, 97, 98 or 99%, or even at least 96%, 97%, 98%, or 99% identical in amino acid sequence to the amino acid sequence encoded by the germline immunoglobulin gene. Typically, a human antibody derived from a particular human germline sequence will display no more than 10 amino acid differences from the amino acid sequence encoded by the human germline immunoglobulin gene. In certain cases, the human antibody may display no more than 5, or even no more than 4, 3, 2, or 1 amino acid difference from the amino acid sequence encoded by the germline immunoglobulin gene.","Homologous Antibodies","In yet another embodiment, an antibody of the invention comprises heavy and light chain variable regions comprising amino acid sequences that are homologous to isolated anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 amino acid sequences of preferred anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibodies, respectively, wherein the antibodies retain the desired functional properties of the parent anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibodies.","As used herein, the percent homology between two amino acid sequences is equivalent to the percent identity between the two sequences. The percent identity between the two sequences is a function of the number of identical positions shared by the sequences (i.e., % homology=# of identical positions\/total # of positions\u00d7100), taking into account the number of gaps, and the length of each gap, which need to be introduced for optimal alignment of the two sequences. The comparison of sequences and determination of percent identity between two sequences can be accomplished using a mathematical algorithm, as described in the non-limiting examples below.","The percent identity between two amino acid sequences can be determined using the algorithm of E. Meyers and W. Miller (Comput. Appl. Biosci., 4:11-17 (1988)) which has been incorporated into the ALIGN program (version 2.0), using a PAM120 weight residue table, a gap length penalty of 12 and a gap penalty of 4. In addition, the percent identity between two amino acid sequences can be determined using the Needleman and Wunsch (J. Mol. Biol. 48:444-453 (1970)) algorithm which has been incorporated into the GAP program in the GCG software package (available commercially), using either a Blossum 62 matrix or a PAM250 matrix, and a gap weight of 16, 14, 12, 10, 8, 6, or 4 and a length weight of 1, 2, 3, 4, 5, or 6.","Additionally or alternatively, the protein sequences of the present invention can further be used as a \u201cquery sequence\u201d to perform a search against public databases to, for example, identify related sequences. Such searches can be performed using the XBLAST program (version 2.0) of Altschul, et al. (1990) J Mol. Biol. 215:403-10. BLAST protein searches can be performed with the XBLAST program, score=50, wordlength=3 to obtain amino acid sequences homologous to the antibody molecules of the invention. To obtain gapped alignments for comparison purposes, Gapped BLAST can be utilized as described in Altschul et al., (1997) Nucleic Acids Res. 25(17):3389-3402. When utilizing BLAST and Gapped BLAST programs, the default parameters of the respective programs (e.g., XBLAST and NBLAST) can be used.","Antibodies with Conservative Modifications","In certain embodiments, an antibody of the invention comprises a heavy chain variable region comprising CDR1, CDR2 and CDR3 sequences and a light chain variable region comprising CDR1, CDR2 and CDR3 sequences, wherein one or more of these CDR sequences comprise specified amino acid sequences based on preferred anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibodies isolated and produced using methods herein, or conservative modifications thereof, and wherein the antibodies retain the desired functional properties of the anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibodies of the invention, respectively.","In various embodiments, the anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibody can be, for example, human antibodies, humanized antibodies or chimeric antibodies.","As used herein, the term \u201cconservative sequence modifications\u201d is intended to refer to amino acid modifications that do not significantly affect or alter the binding characteristics of the antibody containing the amino acid sequence. Such conservative modifications include amino acid substitutions, additions and deletions. Modifications can be introduced into an antibody of the invention by standard techniques known in the art, such as site-directed mutagenesis and PCR-mediated mutagenesis. Conservative amino acid substitutions are ones in which the amino acid residue is replaced with an amino acid residue having a similar side chain. Families of amino acid residues having similar side chains have been defined in the art. These families include amino acids with basic side chains (e.g., lysine, arginine, histidine), acidic side chains (e.g., aspartic acid, glutamic acid), uncharged polar side chains (e.g., glycine, asparagine, glutamine, serine, threonine, tyrosine, cysteine, tryptophan), nonpolar side chains (e.g., alanine, valine, leucine, isoleucine, proline, phenylalanine, methionine), beta-branched side chains (e.g., threonine, valine, isoleucine) and aromatic side chains (e.g., tyrosine, phenylalanine, tryptophan, histidine). Thus, one or more amino acid residues within the CDR regions of an antibody of the invention can be replaced with other amino acid residues from the same side chain family and the altered antibody can be tested for retained function (i.e., the functions set forth in (c) through (j) above) using the functional assays described herein.","Antibodies that Bind to the Same Epitope as Anti-VSIG1, Anti-ILDR1, Anti-LOC253012, Anti-AI216611, Anti-C1ORF32, or Anti-FXYD3 Antibodies of the Invention","In another embodiment, the invention provides antibodies that bind to preferred epitopes on human VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 which possess desired functional properties such as modulation of B7 co-stimulation and related functions. Other antibodies with desired epitope specificity may be selected and will have the ability to cross-compete for binding to VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen with the desired antibodies.","Engineered and Modified Antibodies","An antibody of the invention further can be prepared using an antibody having one or more of the VH and\/or VL sequences derived from an anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibody starting material to engineer a modified antibody, which modified antibody may have altered properties from the starting antibody. An antibody can be engineered by modifying one or more residues within one or both variable regions (i.e., VH and\/or VL), for example within one or more CDR regions and\/or within one or more framework regions. Additionally or alternatively, an antibody can be engineered by modifying residues within the constant regions, for example to alter the effector functions of the antibody.","One type of variable region engineering that can be performed is CDR grafting. Antibodies interact with target antigens predominantly through amino acid residues that are located in the six heavy and light chain complementarity determining regions (CDRs). For this reason, the amino acid sequences within CDRs are more diverse between individual antibodies than sequences outside of CDRs. Because CDR sequences are responsible for most antibody-antigen interactions, it is possible to express recombinant antibodies that mimic the properties of specific naturally occurring antibodies by constructing expression vectors that include CDR sequences from the specific naturally occurring antibody grafted onto framework sequences from a different antibody with different properties (see, e.g., Riechmann, L. et al. (1998) Nature 332:323-327; Jones, P. et al. (1986) Nature 321:522-525; Queen, C. et al. (1989) Proc. Natl. Acad. See. U.S.A. 86:10029-10033; U.S. Pat. No. 5,225,539 to Winter, and U.S. Pat. Nos. 5,530,101; 5,585,089; 5,693,762 and 6,180,370 to Queen et al.)","Suitable framework sequences can be obtained from public DNA databases or published references that include germline antibody gene sequences. For example, germline DNA sequences for human heavy and light chain variable region genes can be found in the \u201cVBase\u201d human germline sequence database (available on the Internet), as well as in Kabat, E. A., et al. (1991) Sequences of Proteins of Immunological Interest, Fifth Edition, U.S. Department of Health and Human Services, NIH Publication No. 91-3242; Tomlinson, I. M., et al. (1992) \u201cThe Repertoire of Human Germline VH Sequences Reveals about Fifty Groups of VH Segments with Different Hypervariable Loops\u201d J. Mol. Biol. 227:776-798; and Cox, J. P. L. et al. (1994) \u201cA Directory of Human Germ-line VH Segments Reveals a Strong Bias in their Usage\u201d Eur. J. Immunol. 24:827-836; the contents of each of which are expressly incorporated herein by reference.","Another type of variable region modification is to mutate amino acid residues within the VH and\/or VL CDR 1, CDR2 and\/or CDR3 regions to thereby improve one or more binding properties (e.g., affinity) of the antibody of interest. Site-directed mutagenesis or PCR-mediated mutagenesis can be performed to introduce the mutations and the effect on antibody binding, or other functional property of interest, can be evaluated in appropriate in vitro or in vivo assays. Preferably conservative modifications (as discussed above) are introduced. The mutations may be amino acid substitutions, additions or deletions, but are preferably substitutions. Moreover, typically no more than one, two, three, four or five residues within a CDR region are altered.","Engineered antibodies of the invention include those in which modifications have been made to framework residues within VH and\/or VL, e.g. to improve the properties of the antibody. Typically such framework modifications are made to decrease the immunogenicity of the antibody. For example, one approach is to \u201cbackmutate\u201d one or more framework residues to the corresponding germline sequence. More specifically, an antibody that has undergone somatic mutation may contain framework residues that differ from the germline sequence from which the antibody is derived. Such residues can be identified by comparing the antibody framework sequences to the germline sequences from which the antibody is derived.","In addition or alternative to modifications made within the framework or CDR regions, antibodies of the invention may be engineered to include modifications within the Fc region, typically to alter one or more functional properties of the antibody, such as serum half-life, complement fixation, Fc receptor binding, and\/or antigen-dependent cellular cytotoxicity. Furthermore, an antibody of the invention may be chemically modified (e.g., one or more chemical moieties can be attached to the antibody) or be modified to alter its glycosylation, again to alter one or more functional properties of the antibody. Such embodiments are described further below. The numbering of residues in the Fc region is that of the EU index of Kabat.","In one embodiment, the hinge region of CH1 is modified such that the number of cysteine residues in the hinge region is altered, e.g., increased or decreased. This approach is described further in U.S. Pat. No. 5,677,425 by Bodmer et al. The number of cysteine residues in the hinge region of CH1 is altered to, for example, facilitate assembly of the light and heavy chains or to increase or decrease the stability of the antibody.","In another embodiment, the Fc hinge region of an antibody is mutated to decrease the biological half life of the antibody. More specifically, one or more amino acid mutations are introduced into the CH2-CH3 domain interface region of the Fc-hinge fragment such that the antibody has impaired Staphylococcyl protein A (SpA) binding relative to native Fc-hinge domain SpA binding. This approach is described in further detail in U.S. Pat. No. 6,165,745 by Ward et al.","In another embodiment, the antibody is modified to increase its biological half life. Various approaches are possible. For example, one or more of the following mutations can be introduced: T252L, T254S, T256F, as described in U.S. Pat. No. 6,277,375 to Ward. Alternatively, to increase the biological half life, the antibody can be altered within the CH1 or CL region to contain a salvage receptor binding epitope taken from two loops of a CH2 domain of an Fc region of an IgG, as described in U.S. Pat. Nos. 5,869,046 and 6,121,022 by Presta et al.","In yet other embodiments, the Fc region is altered by replacing at least one amino acid residue with a different amino acid residue to alter the effector functions of the antibody. For example, one or more amino acids selected from amino acid residues 234, 235, 236, 237, 297, 318, 320 and 322 can be replaced with a different amino acid residue such that the antibody has an altered affinity for an effector ligand but retains the antigen-binding ability of the parent antibody. The effector ligand to which affinity is altered can be, for example, an Fc receptor or the C1 component of complement. This approach is described in further detail in U.S. Pat. Nos. 5,624,821 and 5,648,260, both by Winter et al.","In another example, one or more amino acids selected from amino acid residues 329, 331 and 322 can be replaced with a different amino acid residue such that the antibody has altered C1q binding and\/or reduced or abolished complement dependent cytotoxicity (CDC). This approach is described in further detail in U.S. Pat. No. 6,194,551 by Idusogie et al.","In another example, one or more amino acid residues within amino acid positions 231 and 239 are altered to thereby alter the ability of the antibody to fix complement. This approach is described further in PCT Publication WO 94\/29351 by Bodmer et al.","In yet another example, the Fc region is modified to increase the ability of the antibody to mediate antibody dependent cellular cytotoxicity (ADCC) and\/or to increase the affinity of the antibody for an Fcy receptor by modifying one or more amino acids at the following positions: 238, 239, 248, 249, 252, 254, 255, 256, 258, 265, 267, 268, 269, 270, 272, 276, 278, 280, 283, 285, 286, 289, 290, 292, 293, 294, 295, 296, 298, 301, 303, 305, 307, 309, 312, 315, 320, 322, 324, 326, 327, 329, 330, 331, 333, 334, 335, 337, 338, 340, 360, 373, 376, 378, 382, 388, 389, 398, 414, 416, 419, 430, 434, 435, 437, 438 or 439. This approach is described further in PCT Publication WO 00\/42072 by Presta. Moreover, the binding sites on human IgG1 for Fc grammar, Fc gamma RII, Fc gammaRIII and FcRn have been mapped and variants with improved binding have been described (see Shields, R. L. et al. (2001) J. Biol. Chem. 276:6591-6604). Specific mutations at positions 256, 290, 298, 333, 334 and 339 are shown to improve binding to FcyRIII. Additionally, the following combination mutants are shown to improve Fcgamma.RIII binding: T256A\/S298A, S298A\/E333A, S298A\/K224A and S298A\/E333A\/K334A.","In still another embodiment, the glycosylation of an antibody is modified. For example, an aglycoslated antibody can be made (i.e., the antibody lacks glycosylation). Glycosylation can be altered to, for example, increase the affinity of the antibody for antigen. Such carbohydrate modifications can be accomplished by, for example, altering one or more sites of glycosylation within the antibody sequence. For example, one or more amino acid substitutions can be made that result in elimination of one or more variable region framework glycosylation sites to thereby eliminate glycosylation at that site. Such aglycosylation may increase the affinity of the antibody for antigen. Such an approach is described in further detail in U.S. Pat. Nos. 5,714,350 and 6,350,861 by Co et al.","Additionally or alternatively, an antibody can be made that has an altered type of glycosylation, such as a hypofucosylated antibody having reduced amounts of fucosyl residues or an antibody having increased bisecting GlcNac structures. Such altered glycosylation patterns have been demonstrated to increase the ADCC ability of antibodies. Such carbohydrate modifications can be accomplished by, for example, expressing the antibody in a host cell with altered glycosylation machinery. Cells with altered glycosylation machinery have been described in the art and can be used as host cells in which to express recombinant antibodies of the invention to thereby produce an antibody with altered glycosylation. For example, the cell lines Ms704, Ms705, and Ms709 lack the fucosyltransferase gene, FUT8 (alpha(1,6)fucosyltransferase), such that antibodies expressed in the Ms704, Ms705, and Ms709 cell lines lack fucose on their carbohydrates. The Ms704, Ms705, and Ms709 FUT8.\u2212\/\u2212 cell lines are created by the targeted disruption of the FUT8 gene in CHO\/DG44 cells using two replacement vectors (see U.S. Patent Publication No. 20040110704 by Yamane et al. and Yamane-Ohnuki et al. (2004) Biotechnol Bioeng 87:614-22). As another example, EP 1,176,195 by Hanai et al. describes a cell line with a functionally disrupted FUT8 gene, which encodes a fucosyl transferase, such that antibodies expressed in such a cell line exhibit hypofucosylation by reducing or eliminating the alpha 1,6 bond-related enzyme. Hanai et al. also describe cell lines which have a low enzyme activity for adding fucose to the N-acetylglucosamine that binds to the Fc region of the antibody or does not have the enzyme activity, for example the rat myeloma cell line YB2\/0 (ATCC CRL 1662). PCT Publication WO 03\/035835 by Presta describes a variant CHO cell line, Lec13 cells, with reduced ability to attach fucose to Asn(297)-linked carbohydrates, also resulting in hypofucosylation of antibodies expressed in that host cell (see also Shields, R. L. et al. (2002) J. Biol. Chem. 277:26733-26740). PCT Publication WO 99\/54342 by Umana et al. describes cell lines engineered to express glycoprotein-modifying glycosyl transferases (e.g., beta(1,4)-N-acetylglucosaminyltransferase III (GnTIII)) such that antibodies expressed in the engineered cell lines exhibit increased bisecting GlcNac structures which results in increased ADCC activity of the antibodies (see also Umana et al. (1999) Nat. Biotech. 17:176-180). Alternatively, the fucose residues of the antibody may be cleaved off using a fucosidase enzyme. For example, the fucosidase alpha-L-fucosidase removes fucosyl residues from antibodies (Tarentino, A. L. et al. (1975) Biochem. 14:5516-23).","Another modification of the antibodies herein that is contemplated by the invention is pegylation. An antibody can be pegylated to, for example, increase the biological (e.g., serum) half life of the antibody. To pegylate an antibody, the antibody, or fragment thereof, typically is reacted with polyethylene glycol (PEG), such as a reactive ester or aldehyde derivative of PEG, under conditions in which one or more PEG groups become attached to the antibody or antibody fragment. Preferably, the pegylation is carried out via an acylation reaction or an alkylation reaction with a reactive PEG molecule (or an analogous reactive water-soluble polymer). As used herein, the term \u201cpolyethylene glycol\u201d is intended to encompass any of the forms of PEG that have been used to derivatize other proteins, such as mono (C1-C10) alkoxy- or aryloxy-polyethylene glycol or polyethylene glycol-maleimide. In certain embodiments, the antibody to be pegylated is an aglycosylated antibody. Methods for pegylating proteins are known in the art and can be applied to the antibodies of the invention. See for example, EP 0 154 316 by Nishimura et al. and EP 0 401 384 by Ishikawa et al.","Methods of Engineering Antibodies","As discussed above, the anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibodies having VH and VK sequences disclosed herein can be used to create new anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibodies, respectively, by modifying the VH and\/or VL sequences, or the constant regions attached thereto. Thus, in another aspect of the invention, the structural features of an anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibody of the invention, are used to create structurally related anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibodies that retain at least one functional property of the antibodies of the invention, such as binding to human VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3, respectively. For example, one or more CDR regions of one VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antibody or mutations thereof, can be combined recombinantly with known framework regions and\/or other CDRs to create additional, recombinantly-engineered, anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibodies of the invention, as discussed above. Other types of modifications include those described in the previous section. The starting material for the engineering method is one or more of the VH and\/or VK sequences provided herein, or one or more CDR regions thereof. To create the engineered antibody, it is not necessary to actually prepare (i.e., express as a protein) an antibody having one or more of the VH and\/or VK sequences provided herein, or one or more CDR regions thereof. Rather, the information contained in the sequences is used as the starting material to create a \u201csecond generation\u201d sequences derived from the original sequences and then the \u201csecond generation\u201d sequences is prepared and expressed as a protein.","Standard molecular biology techniques can be used to prepare and express altered antibody sequence.","Preferably, the antibody encoded by the altered antibody sequences is one that retains one, some or all of the functional properties of the anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibodies, respectively, produced by methods and with sequences provided herein, which functional properties include binding to VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen with a specific KD level or less and\/or modulating B7 costimulation and\/or selectively binding to desired target cells such as lung cancer, ovarian cancer, colon cancer, that express VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen.","The functional properties of the altered antibodies can be assessed using standard assays available in the art and\/or described herein.","In certain embodiments of the methods of engineering antibodies of the invention, mutations can be introduced randomly or selectively along all or part of an anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibody coding sequence and the resulting modified anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibodies can be screened for binding activity and\/or other desired functional properties.","Mutational methods have been described in the art. For example, PCT Publication WO 02\/092780 by Short describes methods for creating and screening antibody mutations using saturation mutagenesis, synthetic ligation assembly, or a combination thereof. Alternatively, PCT Publication WO 03\/074679 by Lazar et al. describes methods of using computational screening methods to optimize physiochemical properties of antibodies.","Nucleic Acid Molecules Encoding Antibodies of the Invention","Another aspect of the invention pertains to nucleic acid molecules that encode the antibodies of the invention. The nucleic acids may be present in whole cells, in a cell lysate, or in a partially purified or substantially pure form. A nucleic acid is \u201cisolated\u201d or \u201crendered substantially pure\u201d when purified away from other cellular components or other contaminants, e.g., other cellular nucleic acids or proteins, by standard techniques, including alkaline\/SDS treatment, CsCl banding, column chromatography, agarose gel electrophoresis and others well known in the art. See, F. Ausubel, et al., ed. (1987) Current Protocols in Molecular Biology, Greene Publishing and Wiley Interscience, New York. A nucleic acid of the invention can be, for example, DNA or RNA and may or may not contain intronic sequences. In a preferred embodiment, the nucleic acid is a cDNA molecule.","Nucleic acids of the invention can be obtained using standard molecular biology techniques. For antibodies expressed by hybridomas (e.g., hybridomas prepared from transgenic mice carrying human immunoglobulin genes as described further below), cDNAs encoding the light and heavy chains of the antibody made by the hybridoma can be obtained by standard PCR amplification or cDNA cloning techniques. For antibodies obtained from an immunoglobulin gene library (e.g., using phage display techniques), nucleic acid encoding the antibody can be recovered from the library.","Once DNA fragments encoding VH and VL segments are obtained, these DNA fragments can be further manipulated by standard recombinant DNA techniques, for example to convert the variable region genes to full-length antibody chain genes, to Fab fragment genes or to a scFv gene. In these manipulations, a VL- or VH-encoding DNA fragment is operatively linked to another DNA fragment encoding another protein, such as an antibody constant region or a flexible linker.","The term \u201coperatively linked\u201d, as used in this context, is intended to mean that the two DNA fragments are joined such that the amino acid sequences encoded by the two DNA fragments remain in-frame.","The isolated DNA encoding the VH region can be converted to a full-length heavy chain gene by operatively linking the VH-encoding DNA to another DNA molecule encoding heavy chain constant regions (CH1, CH2 and CH3). The sequences of human heavy chain constant region genes are known in the art (see e.g., Kabat, E. A., et al. (1991) Sequences of Proteins of Immunological Interest, Fifth Edition, U.S. Department of Health and Human Services, NIH Publication No. 91-3242) and DNA fragments encompassing these regions can be obtained by standard PCR amplification. The heavy chain constant region can be an IgG1, IgG2, IgG3, IgG4, IgA, IgE, IgM or IgD constant region, but most preferably is an IgG1 or IgG4 constant region. For a Fab fragment heavy chain gene, the VH-encoding DNA can be operatively linked to another DNA molecule encoding only the heavy chain CH1 constant region.","The isolated DNA encoding the VL region can be converted to a full-length light chain gene (as well as a Fab light chain gene) by operatively linking the VL-encoding DNA to another DNA molecule encoding the light chain constant region, CL. The sequences of human light chain constant region genes are known in the art (see e.g., Kabat, E. A., et al. (1991) Sequences of Proteins of Immunological Interest, Fifth Edition, U.S. Department of Health and Human Services, NIH Publication No. 91-3242) and DNA fragments encompassing these regions can be obtained by standard PCR amplification. The light chain constant region can be a kappa or lambda constant region, but most preferably is a kappa constant region.","To create a scFv gene, the VH- and VL-encoding DNA fragments are operatively linked to another fragment encoding a flexible linker, e.g., encoding the amino acid sequence (Gly4-Ser)3, such that the VH and VL sequences can be expressed as a contiguous single-chain protein, with the VL and VH regions joined by the flexible linker (see e.g., Bird et al. (1988) Science 242:423-426; Huston et al. (1988) Proc. Natl. Acad. Sci. USA 85:5879-5883; McCafferty et al., (1990) Nature 348:552-554).","Production of Anti-VSIG1, Anti-ILDR1, Anti-LOC253012, Anti-AI216611, Anti-C1ORF32, or Anti-FXYD3 Monoclonal Antibodies of the Invention","Monoclonal antibodies (mAbs) of the present invention can be produced by a variety of techniques, including conventional monoclonal antibody methodology e.g., the standard somatic cell hybridization technique of Kohler and Milstein (1975) Nature 256:495. Although somatic cell hybridization procedures are preferred, in principle, other techniques for producing monoclonal antibody can be employed e.g., viral or oncogenic transformation of B lymphocytes.","A preferred animal system for preparing hybridomas is the murine system. Hybridoma production in the mouse is a very well-established procedure. Immunization protocols and techniques for isolation of immunized splenocytes for fusion are known in the art. Fusion partners (e.g., murine myeloma cells) and fusion procedures are also known.","Chimeric or humanized antibodies of the present invention can be prepared based on the sequence of a murine monoclonal antibody prepared as described above. DNA encoding the heavy and light chain immunoglobulins can be obtained from the murine hybridoma of interest and engineered to contain non-murine (e.g., human) immunoglobulin sequences using standard molecular biology techniques. For example, to create a chimeric antibody, the murine variable regions can be linked to human constant regions using methods known in the art (see e.g., U.S. Pat. No. 4,816,567 to Cabilly et al.). To create a humanized antibody, the murine CDR regions can be inserted into a human framework using methods known in the art (see e.g., U.S. Pat. No. 5,225,539 to Winter, and U.S. Pat. Nos. 5,530,101; 5,585,089; 5,693,762 and 6,180,370 to Queen et al.).","In a preferred embodiment, the antibodies of the invention are human monoclonal antibodies. Such human monoclonal antibodies directed against VSIG1 can be generated using transgenic or transchromosomic mice carrying parts of the human immune system rather than the mouse system. These transgenic and transchromosomic mice include mice referred to herein as the HuMAb Mouse\u00ae and KM Mouse\u00ae respectively, and are collectively referred to herein as \u201chuman Ig mice.\u201d The HuMAb Mouse\u2122 (Medarex. Inc.) contains human immunoglobulin gene miniloci that encode unrearranged human heavy (.mu. and .gamma.) and .kappa. light chain immunoglobulin sequences, together with targeted mutations that inactivate the endogenous.mu. and .kappa. chain loci (see e.g., Lonberg, et al. (1994) Nature 368(6474): 856-859). Accordingly, the mice exhibit reduced expression of mouse IgM or .kappa., and in response to immunization, the introduced human heavy and light chain transgenes undergo class switching and somatic mutation to generate high affinity human IgGkappa. monoclonal (Lonberg, N. et al. (1994), supra; reviewed in Lonberg, N. (1994) Handbook of Experimental Pharmacology 113:49-101; Lonberg, N. and Huszar, D. (1995) Intern. Rev. Immunol. 13: 65-93, and Harding, F. and Lonberg, N. (1995) Ann. N.Y. Acad. Sci. 764:536-546). The preparation and use of the HuMab Mouse\u00ae, and the genomic modifications carried by such mice, is further described in Taylor, L. et al. (1992) Nucleic Acids Research 20:6287-6295; Chen, J. et al. (1993) International Immunology 5:647-656; Tuaillon et al. (1993) Proc. Natl. Acad. Sci. USA 90:3720-3724; Choi et al. (1993) Nature Genetics 4:117-123; Chen, J. et al. (1993) EMBO J. 12: 821-830; Tuaillon et al. (1994) J. Immunol. 152:2912-2920; Taylor, L. et al. (1994) International Immunology 6:579-591; and Fishwild, D. et al. (1996) Nature Biotechnology 14: 845-851, the contents of all of which are hereby specifically incorporated by reference in their entirety. See further, U.S. Pat. Nos. 5,545,806; 5,569,825; 5,625,126; 5,633,425; 5,789,650; 5,877,397; 5,661,016; 5,814,318; 5,874,299; and 5,770,429; all to Lonberg and Kay; U.S. Pat. No. 5,545,807 to Surani et al.; PCT Publication Nos. WO 92\/03918, WO 93\/12227, WO 94\/25585, WO 97\/13852, WO 98\/24884 and WO 99\/45962, all to Lonberg and Kay; and PCT Publication No. WO 01\/14424 to Korman et al.","In another embodiment, human antibodies of the invention can be raised using a mouse that carries human immunoglobulin sequences on transgenes and transchromosomes, such as a mouse that carries a human heavy chain transgene and a human light chain transchromosome. Such mice, referred to herein as \u201cKM Mice\u2122\u201d, are described in detail in PCT Publication WO 02\/43478 to Ishida et al.","Still further, alternative transgenic animal systems expressing human immunoglobulin genes are available in the art and can be used to raise anti-VSIG1 antibodies of the invention. For example, an alternative transgenic system referred to as the Xenomouse (Abgenix, Inc.) can be used; such mice are described in, for example, U.S. Pat. Nos. 5,939,598; 6,075,181; 6,114,598; 6, 150,584 and 6,162,963 to Kucherlapati et al.","Moreover, alternative transchromosomic animal systems expressing human immunoglobulin genes are available in the art and can be used to raise anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibodies of the invention. For example, mice carrying both a human heavy chain transchromosome and a human light chain transchromosome, referred to as \u201cTC mice\u201d can be used; such mice are described in Tomizuka et al. (2000) Proc. Natl. Acad. Sci. USA 97:722-727. Furthermore, cows carrying human heavy and light chain transchromosomes have been described in the art (Kuroiwa et al. (2002) Nature Biotechnology 20:889-894) and can be used to raise anti-VSIG1 antibodies of the invention.","Human monoclonal antibodies of the invention can also be prepared using phage display methods for screening libraries of human immunoglobulin genes. Such phage display methods for isolating human antibodies are established in the art. See for example: U.S. Pat. Nos. 5,223,409; 5,403,484; and 5,571,698 to Ladner et al.; U.S. Pat. Nos. 5,427,908 and 5,580,717 to Dower et al.; U.S. Pat. Nos. 5,969,108 and 6,172,197 to McCafferty et al.; and U.S. Pat. Nos. 5,885,793; 6,521,404; 6,544,73 1; 6,555,313; 6,582,915 and 6,593,081 to Griffiths et al.","Human monoclonal antibodies of the invention can also be prepared using SCID mice into which human immune cells have been reconstituted such that a human antibody response can be generated upon immunization. Such mice are described in, for example, U.S. Pat. Nos. 5,476,996 and 5,698,767 to Wilson et al.","Immunization of Human Ig Mice","When human Ig mice are used to raise human antibodies of the invention, such mice can be immunized with a purified or enriched preparation of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen and\/or recombinant VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXY, or an VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 fusion protein, as described by Lonberg, N. et al. (1994) Nature 368(6474): 856-859; Fishwild, D. et al. (1996) Nature Biotechnology 14: 845-851; and PCT Publication WO 98\/24884 and WO 01\/14424. Preferably, the mice will be 6-16 weeks of age upon the first infusion. For example, a purified or recombinant preparation (5-50. mu.g) of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen can be used to immunize the human Ig mice intraperitoneally.","Prior experience with various antigens by others has shown that the transgenic mice respond when initially immunized intraperitoneally (IP) with antigen in complete Freund's adjuvant, followed by every other week IP immunizations (up to a total of 6) with antigen in incomplete Freund's adjuvant. However, adjuvants other than Freund's are also found to be effective. In addition, whole cells in the absence of adjuvant are found to be highly immunogenic. The immune response can be monitored over the course of the immunization protocol with plasma samples being obtained by retroorbital bleeds. The plasma can be screened by ELISA (as described below), and mice with sufficient titers of anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 human immunoglobulin can be used for fusions. Mice can be boosted intravenously with antigen 3 days before sacrifice and removal of the spleen. It is expected that 2-3 fusions for each immunization may need to be performed. Between 6 and 24 mice are typically immunized for each antigen. Usually both HCo7 and HCo12 strains are used. In addition, both HCo7 and HCo12 transgene can be bred together into a single mouse having two different human heavy chain transgenes (HCo7\/HCo 12). Alternatively or additionally, the KM Mouse\u00ae strain can be used.","Generation of Hybridomas Producing Human Monoclonal Antibodies of the Invention","To generate hybridomas producing human monoclonal antibodies of the invention, splenocytes and\/or lymph node cells from immunized mice can be isolated and fused to an appropriate immortalized cell line, such as a mouse myeloma cell line. The resulting hybridomas can be screened for the production of antigen-specific antibodies. For example, single cell suspensions of splenic lymphocytes from immunized mice can be fused to one-sixth the number of P3X63-Ag8.653 nonsecreting mouse myeloma cells (ATCC, CRL 1580) with 50% PEG. Cells are plated at approximately 2\u00d710\u22125 in flat bottom microtiter plate, followed by a two week incubation in selective medium containing 20% fetal Clone Serum, 18% \u201c653\u201d conditioned media, 5% origen (IGEN), 4 mM L-glutamine, 1 mM sodium pyruvate, 5 mM HEPES, 0.055 mM 2-mercaptoethanol, 50 units\/ml penicillin, 50 mg\/ml streptomycin, 50 mg\/ml gentamycin and 1\u00d7HAT (Sigma; the HAT is added 24 hours after the fusion). After approximately two weeks, cells can be cultured in medium in which the HAT is replaced with HT. Individual wells can then be screened by ELISA for human monoclonal IgM and IgG antibodies. Once extensive hybridoma growth occurs, medium can be observed usually after 10-14 days. The antibody secreting hybridomas can be replated, screened again, and if still positive for human IgG, the monoclonal antibodies can be subcloned at least twice by limiting dilution. The stable subclones can then be cultured in vitro to generate small amounts of antibody in tissue culture medium for characterization.","To purify human monoclonal antibodies, selected hybridomas can be grown in two-liter spinner-flasks for monoclonal antibody purification. Supernatants can be filtered and concentrated before affinity chromatography with protein A-Sepharose (Pharmacia, Piscataway, N.J.). Eluted IgG can be checked by gel electrophoresis and high performance liquid chromatography to ensure purity. The buffer solution can be exchanged into PBS, and the concentration can be determined by OD280 using 1.43 extinction coefficient. The monoclonal antibodies can be aliquoted and stored at \u221280 degrees C.","Generation of Transfectomas Producing Monoclonal Antibodies of the Invention","Antibodies of the invention also can be produced in a host cell transfectoma using, for example, a combination of recombinant DNA techniques and gene transfection methods as is well known in the art (e.g., Morrison, S. (1985) Science 229:1202).","For example, to express the antibodies, or antibody fragments thereof, DNAs encoding partial or full-length light and heavy chains, can be obtained by standard molecular biology techniques (e.g., PCR amplification or cDNA cloning using a hybridoma that expresses the antibody of interest) and the DNAs can be inserted into expression vectors such that the genes are operatively linked to transcriptional and translational control sequences. In this context, the term \u201coperatively linked\u201d is intended to mean that an antibody gene is ligated into a vector such that transcriptional and translational control sequences within the vector serve their intended function of regulating the transcription and translation of the antibody gene. The expression vector and expression control sequences are chosen to be compatible with the expression host cell used. The antibody light chain gene and the antibody heavy chain gene can be inserted into separate vector or, more typically, both genes are inserted into the same expression vector. The antibody genes are inserted into the expression vector by standard methods (e.g., ligation of complementary restriction sites on the antibody gene fragment and vector, or blunt end ligation if no restriction sites are present). The light and heavy chain variable regions of the antibodies described herein can be used to create full-length antibody genes of any antibody isotype by inserting them into expression vectors already encoding heavy chain constant and light chain constant regions of the desired isotype such that the VH segment is operatively linked to the CH segments within the vector and the VK segment is operatively linked to the CL segment within the vector. Additionally or alternatively, the recombinant expression vector can encode a signal peptide that facilitates secretion of the antibody chain from a host cell. The antibody chain gene can be cloned into the vector such that the signal peptide is linked in-frame to the amino terminus of the antibody chain gene. The signal peptide can be an immunoglobulin signal peptide or a heterologous signal peptide (i.e., a signal peptide from a non-immunoglobulin protein).","In addition to the antibody chain genes, the recombinant expression vectors of the invention carry regulatory sequences that control the expression of the antibody chain genes in a host cell. The term \u201cregulatory sequence\u201d is intended to include promoters, enhancers and other expression control elements (e.g., polyadenylation signals) that control the transcription or translation of the antibody chain genes. Such regulatory sequences are described, for example, in Goeddel (Gene Expression Technology. Methods in Enzymology 185, Academic Press, San Diego, Calif. (1990)). It will be appreciated by those skilled in the art that the design of the expression vector, including the selection of regulatory sequences, may depend on such factors as the choice of the host cell to be transformed, the level of expression of protein desired, etc. Preferred regulatory sequences for mammalian host cell expression include viral elements that direct high levels of protein expression in mammalian cells, such as promoters and\/or enhancers derived from cytomegalovirus (CMV), Simian Virus 40 (SV40), adenovirus, (e.g., the adenovirus major late promoter (AdMLP) and polyoma. Alternatively, nonviral regulatory sequences may be used, such as the ubiquitin promoter or .beta.-globin promoter. Still further, regulatory elements composed of sequences from different sources, such as the SR alpha. promoter system, which contains sequences from the SV40 early promoter and the long terminal repeat of human T cell leukemia virus type 1 (Takebe, Y. et al. (1988) Mol. Cell. Biol. 8:466-472).","In addition to the antibody chain genes and regulatory sequences, the recombinant expression vectors of the invention may carry additional sequences, such as sequences that regulate replication of the vector in host cells (e.g., origins of replication) and selectable marker genes. The selectable marker gene facilitates selection of host cells into which the vector has been introduced (see, e.g., U.S. Pat. Nos. 4,399,216, 4,634,665 and 5,179,017, all by Axel et al.). For example, typically the selectable marker gene confers resistance to drugs, such as G418, hygromycin or methotrexate, on a host cell into which the vector has been introduced. Preferred selectable marker genes include the dihydrofolate reductase (DHFR) gene (for use in dhfr-host cells with methotrexate selection\/amplification) and the neo gene (for G418 selection).","For expression of the light and heavy chains, the expression vectors encoding the heavy and light chains is transfected into a host cell by standard techniques. The various forms of the term \u201ctransfection\u201d are intended to encompass a wide variety of techniques commonly used for the introduction of exogenous DNA into a prokaryotic or eukaryotic host cell, e.g., electroporation, calcium-phosphate precipitation, DEAE-dextran transfection and the like. Although it is theoretically possible to express the antibodies of the invention in either prokaryotic or eukaryotic host cells, expression of antibodies in eukaryotic cells, and most preferably mammalian host cells, is the most preferred because such eukaryotic cells, and in particular mammalian cells, are more likely than prokaryotic cells to assemble and secrete a properly folded and immunologically active antibody. Prokaryotic expression of antibody genes has been reported to be ineffective for production of high yields of active antibody (Boss, M. A. and Wood, C. R. (1985) Immunology Today 6:12-13).","Preferred mammalian host cells for expressing the recombinant antibodies of the invention include Chinese Hamster Ovary (CHO cells) (including dhfr-CHO cells, described in Urlaub and Chasin, (1980) Proc. Natl. Acad. Sci. USA 77:4216-4220, used with a DHFR selectable marker, e.g., as described in R. J. Kaufman and P. A. Sharp (1982) Mol. Biol. 159:601-621), NSO myeloma cells, COS cells and SP2 cells. In particular, for use with NSO myeloma cells, another preferred expression system is the GS gene expression system disclosed in WO 87\/04462, WO 89\/01036 and EP 338,841. When recombinant expression vectors encoding antibody genes are introduced into mammalian host cells, the antibodies are produced by culturing the host cells for a period of time sufficient to allow for expression of the antibody in the host cells or, more preferably, secretion of the antibody into the culture medium in which the host cells are grown. Antibodies can be recovered from the culture medium using standard protein purification methods.","Characterization of Antibody Binding to Antigen","Antibodies of the invention can be tested for binding to VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 by, for example, standard ELISA. Briefly, microtiter plates are coated with purified VSIG1 at 0.25. mu.g\/ml in PBS, and then blocked with 5% bovine serum albumin in PBS. Dilutions of antibody (e.g., dilutions of plasma from VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3-immunized mice) are added to each well and incubated for 1-2 hours at 37 degrees C. The plates are washed with PBS\/Tween and then incubated with secondary reagent (e.g., for human antibodies, a goat-anti-human IgG Fc-specific polyclonal reagent) conjugated to alkaline phosphatase for 1 hour at 37 degrees C. After washing, the plates are developed with pNPP substrate (1 mg\/ml), and analyzed at OD of 405-650. Preferably, mice which develop the highest titers will be used for fusions.","An ELISA assay as described above can also be used to screen for hybridomas that show positive reactivity with VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 immunogen. Hybridomas that bind with high avidity to VSIG1 are subcloned and further characterized. One clone from each hybridoma, which retains the reactivity of the parent cells (by ELISA), can be chosen for making a 5-10 vial cell bank stored at \u2212140 degrees C., and for antibody purification.","To purify anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibodies, selected hybridomas can be grown in two-liter spinner-flasks for monoclonal antibody purification. Supernatants can be filtered and concentrated before affinity chromatography with protein A-sepharose (Pharmacia, Piscataway, N.J.). Eluted IgG can be checked by gel electrophoresis and high performance liquid chromatography to ensure purity. The buffer solution can be exchanged into PBS, and the concentration can be determined by OD280 using 1.43 extinction coefficient. The monoclonal antibodies can be aliquoted and stored at \u221280 degrees C.","To determine if the selected anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 monoclonal antibodies bind to unique epitopes, each antibody can be biotinylated using commercially available reagents (Pierce, Rockford, Ill.). Competition studies using unlabeled monoclonal antibodies and biotinylated monoclonal antibodies can be performed using VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 coated-ELISA plates as described above. Biotinylated mAb binding can be detected with a strepavidin-alkaline phosphatase probe.","To determine the isotype of purified antibodies, isotype ELISAs can be performed using reagents specific for antibodies of a particular isotype. For example, to determine the isotype of a human monoclonal antibody, wells of microtiter plates can be coated with 1. mu.g\/ml of anti-human immunoglobulin overnight at 4 degrees C. After blocking with 1% BSA, the plates are reacted with 1 mug\/ml or less of test monoclonal antibodies or purified isotype controls, at ambient temperature for one to two hours. The wells can then be reacted with either human IgG1 or human IgM-specific alkaline phosphatase-conjugated probes. Plates are developed and analyzed as described above.","Anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 human IgGs can be further tested for reactivity with VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen, respectively, by Western blotting. Briefly, VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen can be prepared and subjected to sodium dodecyl sulfate polyacrylamide gel electrophoresis. After electrophoresis, the separated antigens are transferred to nitrocellulose membranes, blocked with 10% fetal calf serum, and probed with the monoclonal antibodies to be tested. Human IgG binding can be detected using anti-human IgG alkaline phosphatase and developed with BCIP\/NBT substrate tablets (Sigma Chem. Co., St. Louis, Mo.).","Conjugates or Immunoconjugates","The present invention encompasses conjugates for use in immune therapy comprising the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen and soluble portions thereof including the ectodomain or portions or variants thereof. For example the invention encompasses conjugates wherein the ECD of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen is attached to an immunoglobulin or fragment thereof. The invention contemplates the use thereof for promoting or inhibiting VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen activities such as immune costimulation and the use thereof in treating transplant, autoimmune, and cancer indications described herein.","In another aspect, the present invention features immunoconjugates comprising an anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibody, or a fragment thereof, conjugated to a therapeutic moiety, such as a cytotoxin, a drug (e.g., an immunosuppressant) or a radiotoxin. Such conjugates are referred to herein as \u201cimmunoconjugates\u201d. Immunoconjugates that include one or more cytotoxins are referred to as \u201cimmunotoxins.\u201d A cytotoxin or cytotoxic agent includes any agent that is detrimental to (e.g., kills) cells. Examples include taxol, cytochalasin B, gramicidin D, ethidium bromide, emetine, mitomycin, etoposide, tenoposide, vincristine, vinblastine, colchicin, doxorubicin, daunorubicin, dihydroxy anthracin dione, mitoxantrone, mithramycin, actinomycin D, 1-dehydrotestosterone, glucocorticoids, procaine, tetracaine, lidocaine, propranolol, and puromycin and analogs or homologs thereof. Therapeutic agents also include, for example, antimetabolites (e.g., methotrexate, 6-mercaptopurine, 6-thioguanine, cytarabine, 5-fluorouracil decarbazine), alkylating agents (e.g., mechlorethamine, thioepa chlorambucil, melphalan, carmustine (BSNU) and lomustine (CCNU), cyclothosphamide, busulfan, dibromomannitol, streptozotocin, mitomycin C, and cis-dichlorodiamine platinum (II) (DDP) cisplatin), anthracyclines (e.g., daunorubicin (formerly daunomycin) and doxorubicin), antibiotics (e.g., dactinomycin (formerly actinomycin), bleomycin, mithramycin, and anthramycin (AMC)), and anti-mitotic agents (e.g., vincristine and vinblastine).","Other preferred examples of therapeutic cytotoxins that can be conjugated to an antibody of the invention include duocarmycins, calicheamicins, maytansines and auristatins, and derivatives thereof. An example of a calicheamicin antibody conjugate is commercially available (Mylotarg\u2122; Wyeth).","Cytotoxins can be conjugated to antibodies of the invention using linker technology available in the art. Examples of linker types that have been used to conjugate a cytotoxin to an antibody include, but are not limited to, hydrazones, thioethers, esters, disulfides and peptide-containing linkers. A linker can be chosen that is, for example, susceptible to cleavage by low pH within the lysosomal compartment or susceptible to cleavage by proteases, such as proteases preferentially expressed in tumor tissue such as cathepsins (e.g., cathepsins B, C, D).","For further discussion of types of cytotoxins, linkers and methods for conjugating therapeutic agents to antibodies, see also Saito, G. et al. (2003) Adv. Drug Deliv. Rev. 55:199-215; Trail, P. A. et al. (2003) Cancer Immunol. Immunother. 52:328-337; Payne, G. (2003) Cancer Cell 3:207-212; Allen, T. M. (2002) Nat. Rev. Cancer 2:750-763; Pastan, I. and Kreitman, R. J. (2002) Curr. Opin. Investig. Drugs 3:1089-1091; Senter, P. D. and Springer, C. J. (2001) Adv. Drug Deliv. Rev. 53:247-264.","Antibodies of the present invention also can be conjugated to a radioactive isotope to generate cytotoxic radiopharmaceuticals, also referred to as radioimmunoconjugates. Examples of radioactive isotopes that can be conjugated to antibodies for use diagnostically or therapeutically include, but are not limited to, iodine 131, indium 111, yttrium 90 and lutetium 177. Method for preparing radioimmunconjugates are established in the art. Examples of radioimmunoconjugates are commercially available, including Zevalin\u2122 (IDEC Pharmaceuticals) and Bexxar\u2122 (Corixa Pharmaceuticals), and similar methods can be used to prepare radioimmunoconjugates using the antibodies of the invention.","The antibody conjugates of the invention can be used to modify a given biological response, and the drug moiety is not to be construed as limited to classical chemical therapeutic agents. For example, the drug moiety may be a protein or polypeptide possessing a desired biological activity. Such proteins may include, for example, an enzymatically active toxin, or active fragment thereof, such as abrin, ricin A, pseudomonas exotoxin, or diphtheria toxin; a protein such as tumor necrosis factor or interferon-.gamma.; or, biological response modifiers such as, for example, lymphokines, interleukin-1 (\u201cIL-1\u201d), interleukin-2 (\u201cIL-2\u201d), interleukin-6 (\u201cIL-6\u201d), granulocyte macrophage colony stimulating factor (\u201cGM-CSF\u201d), granulocyte colony stimulating factor (\u201cG-CSF\u201d), or other growth factors.","Techniques for conjugating such therapeutic moiety to antibodies are well known, see, e.g., Arnon et al., \u201cMonoclonal Antibodies For Immunotargeting Of Drugs In Cancer Therapy\u201d, in Monoclonal Antibodies And Cancer Therapy, Reisfeld et al. (eds.), pp. 243-56 (Alan R. Liss, Inc. 1985); Hellstrom et al., \u201cAntibodies For Drug Delivery\u201d, in Controlled Drug Delivery (2nd Ed.), Robinson et al. (eds.), pp. 623-53 (Marcel Dekker, Inc. 1987); Thorpe, \u201cAntibody Carriers Of Cytotoxic Agents In Cancer Therapy: A Review\u201d, in Monoclonal Antibodies '84: Biological And Clinical Applications, Pinchera et al. (eds.), pp. 475-506 (1985); \u201cAnalysis, Results, And Future Prospective Of The Therapeutic Use Of Radiolabeled Antibody In Cancer Therapy\u201d, in Monoclonal Antibodies For Cancer Detection And Therapy, Baldwin et al. (eds.), pp. 303-16 (Academic Press 1985), and Thorpe et al., \u201cThe Preparation And Cytotoxic Properties Of Antibody-Toxin Conjugates\u201d, Immunol. Rev., 62:119-58 (1982).","Bispecific Molecules","In another aspect, the present invention features bispecific molecules comprising an anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibody, or a fragment thereof, of the invention. An antibody of the invention, or antigen-binding portions thereof, can be derivatized or linked to another functional molecule, e.g., another peptide or protein (e.g., another antibody or ligand for a receptor) to generate a bispecific molecule that binds to at least two different binding sites or target molecules. The antibody of the invention may in fact be derivatized or linked to more than one other functional molecule to generate multispecific molecules that bind to more than two different binding sites and\/or target molecules; such multispecific molecules are also intended to be encompassed by the term \u201cbispecific molecule\u201d as used herein. To create a bispecific molecule of the invention, an antibody of the invention can be functionally linked (e.g., by chemical coupling, genetic fusion, noncovalent association or otherwise) to one or more other binding molecules, such as another antibody, antibody fragment, peptide or binding mimetic, such that a bispecific molecule results.","Accordingly, the present invention includes bispecific molecules comprising at least one first binding specificity for VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 and a second binding specificity for a second target epitope. In a particular embodiment of the invention, the second target epitope is an Fc receptor, e.g., human Fc gamma RI (CD64) or a human Fc alpha receptor (CD89). Therefore, the invention includes bispecific molecules capable of binding both to Fc gamma. R, Fc alpha R or Fc epsilon R expressing effector cells (e.g., monocytes, macrophages or polymorphonuclear cells (PMNs)), and to target cells expressing VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3, respectively. These bispecific molecules target VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 expressing cells to effector cell and trigger Fc receptor-mediated effector cell activities, such as phagocytosis of an VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 expressing cells, antibody dependent cell-mediated cytotoxicity (ADCC), cytokine release, or generation of superoxide anion.","In an embodiment of the invention in which the bispecific molecule is multispecific, the molecule can further include a third binding specificity, in addition to an anti-Fc binding specificity and an anti-6f binding specificity. In one embodiment, the third binding specificity is an anti-enhancement factor (EF) portion, e.g., a molecule which binds to a surface protein involved in cytotoxic activity and thereby increases the immune response against the target cell.","The \u201canti-enhancement factor portion\u201d can be an antibody, functional antibody fragment or a ligand that binds to a given molecule, e.g., an antigen or a receptor, and thereby results in an enhancement of the effect of the binding determinants for the Fc receptor or target cell antigen. The \u201canti-enhancement factor portion\u201d can bind an Fc receptor or a target cell antigen. Alternatively, the anti-enhancement factor portion can bind to an entity that is different from the entity to which the first and second binding specificities bind. For example, the anti-enhancement factor portion can bind a cytotoxic T-cell (e.g., via CD2, CD3, CD8, CD28, CD4, CD40, ICAM-1 or other immune cell that results in an increased immune response against the target cell).","In one embodiment, the bispecific molecules of the invention comprise as a binding specificity at least one antibody, or an antibody fragment thereof, including, e.g., an Fab, Fab\u2032, F(ab\u2032).sub.2, Fv, or a single chain Fv. The antibody may also be a light chain or heavy chain dimer, or any minimal fragment thereof such as a Fv or a single chain construct as described in Ladner et al. U.S. Pat. No. 4,946,778, the contents of which is expressly incorporated by reference.","In one embodiment, the binding specificity for an Fcy receptor is provided by a monoclonal antibody, the binding of which is not blocked by human immunoglobulin G (IgG). As used herein, the term \u201cIgG receptor\u201d refers to any of the eight.gamma.-chain genes located on chromosome 1. These genes encode a total of twelve transmembrane or soluble receptor isoforms which are grouped into three Fc.gamma. receptor classes: Fc gamma R1 (CD64), Fc gamma RII (CD32), and Fc gamma.RIII (CD 16). In one preferred embodiment, the Fc gamma. receptor a human high affinity Fc.gamma R1. The human Fc gammaRI is a 72 kDa molecule, which shows high affinity for monomeric IgG (10 8-10\u22129M.-1).","The production and characterization of certain preferred anti-Fc gamma. monoclonal antibodies are described by Fanger et al. in PCT Publication WO 88\/00052 and in U.S. Pat. No. 4,954,617, the teachings of which are fully incorporated by reference herein. These antibodies bind to an epitope of Fc.gamma.RI, FcyRII or FcyRIII at a site which is distinct from the Fc.gamma. binding site of the receptor and, thus, their binding is not blocked substantially by physiological levels of IgG. Specific anti-Fc.gamma.RI antibodies useful in this invention are mAb 22, mAb 32, mAb 44, mAb 62 and mAb 197. The hybridoma producing mAb 32 is available from the American Type Culture Collection, ATCC Accession No. HB9469. In other embodiments, the anti-Fcy receptor antibody is a humanized form of monoclonal antibody 22 (H22). The production and characterization of the H22 antibody is described in Graziano, R. F. et al. (1995) J. Immunol. 155 (10): 4996-5002 and PCT Publication WO 94\/10332. The H22 antibody producing cell line is deposited at the American Type Culture Collection under the designation HAO22CLI and has the accession no. CRL 11177.","In still other preferred embodiments, the binding specificity for an Fc receptor is provided by an antibody that binds to a human IgA receptor, e.g., an Fc-alpha receptor (Fc alpha.RI (CD89)), the binding of which is preferably not blocked by human immunoglobulin A (IgA). The term \u201cIgA receptor\u201d is intended to include the gene product of one alpha.-gene (Fc alpha.RI) located on chromosome 19. This gene is known to encode several alternatively spliced transmembrane isoforms of 55 to 10 kDa","Fc.alpha.RI (CD89) is constitutively expressed on monocytes\/macrophages, eosinophilic and neutrophilic granulocytes, but not on non-effector cell populations. Fc alpha R1 has medium affinity (Approximately 5\u00d710\u22127 M\u22121) for both IgA1 and IgA2, which is increased upon exposure to cytokines such as G-CSF or GM-CSF (Morton, H. C. et al. (1996) Critical Reviews in Immunology 16:423-440). Four FcaRI-specific monoclonal antibodies, identified as A3, A59, A62 and A77, which bind Fc.alpha.RI outside the IgA ligand binding domain, have been described (Monteiro, R. C. et al. (1992) J. Immunol. 148:1764).","Fc. alpha. RI and Fc gamma. R1 are preferred trigger receptors for use in the bispecific molecules of the invention because they are (1) expressed primarily on immune effector cells, e.g., monocytes, PMNs, macrophages and dendritic cells; (2) expressed at high levels (e.g., 5,000-100,000 per cell); (3) mediators of cytotoxic activities (e.g., ADCC, phagocytosis); (4) mediate enhanced antigen presentation of antigens, including self-antigens, targeted to them.","While human monoclonal antibodies are preferred, other antibodies which can be employed in the bispecific molecules of the invention are murine, chimeric and humanized monoclonal antibodies.","The bispecific molecules of the present invention can be prepared by conjugating the constituent binding specificities, e.g., the anti-FcR and anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 binding specificities, using methods known in the art. For example, each binding specificity of the bispecific molecule can be generated separately and then conjugated to one another. When the binding specificities are proteins or peptides, a variety of coupling or cross-linking agents can be used for covalent conjugation. Examples of cross-linking agents include protein A, carbodiimide, N-succinimidyl-5-acetyl-thioacetate (SATA), 5,5\u2032-dithiobis(2-nitrobenzoic acid) (DTNB), o-phenylenedimaleimide (oPDM), N-succinimidyl-3-(2-pyridyld-ithio)propionate (SPDP), and sulfosuccinimidyl 4-(N-maleimidomethyl) cyclohaxane-1-carboxylate (sulfo-SMCC) (see e.g., Karpovsky et al. (1984) J. Exp. Med. 160:1686; Liu, M A et al. (1985) Proc. Natl. Acad. Sci. USA 82:8648). Other methods include those described in Paulus (1985) Behring Ins. Mitt. No. 78, 118-132; Brennan et al. (1985) Science 229:81-83), and Glennie et al. (1987) J. Immunol. 139: 2367-2375). Preferred conjugating agents are SATA and sulfo-SMCC, both available from Pierce Chemical Co. (Rockford, Ill.).","When the binding specificities are antibodies, they can be conjugated via sulfhydryl bonding of the C-terminus hinge regions of the two heavy chains. In a particularly preferred embodiment, the hinge region is modified to contain an odd number of sulfhydryl residues, preferably one, prior to conjugation.","Alternatively, both binding specificities can be encoded in the same vector and expressed and assembled in the same host cell. This method is particularly useful where the bispecific molecule is a mAbXmAb, mAbXFab, FabXF(ab\u2032)2 or ligandXFab fusion protein. A bispecific molecule of the invention can be a single chain molecule comprising one single chain antibody and a binding determinant, or a single chain bispecific molecule comprising two binding determinants. Bispecific molecules may comprise at least two single chain molecules. Methods for preparing bispecific molecules are described for example in U.S. Pat. No. 5,260,203; U.S. Pat. No. 5,455,030; U.S. Pat. No. 4,881,175; U.S. Pat. No. 5,132,405; U.S. Pat. No. 5,091,513; U.S. Pat. No. 5,476,786; U.S. Pat. No. 5,013,653; U.S. Pat. No. 5,258,498; and U.S. Pat. No. 5,482,858.","Binding of the bispecific molecules to their specific targets can be confirmed by, for example, enzyme-linked immunosorbent assay (ELISA), radioimmunoassay (RIA), FACS analysis, bioassay (e.g., growth inhibition), or Western Blot assay. Each of these assays generally detects the presence of protein-antibody complexes of particular interest by employing a labeled reagent (e.g., an antibody) specific for the complex of interest. For example, the FcR-antibody complexes can be detected using e.g., an enzyme-linked antibody or antibody fragment which recognizes and specifically binds to the antibody-FcR complexes. Alternatively, the complexes can be detected using any of a variety of other immunoassays. For example, the antibody can be radioactively labeled and used in a radioimmunoassay (RIA) (see, for example, Weintraub, B., Principles of Radioimmunoassays, Seventh Training Course on Radioligand Assay Techniques, The Endocrine Society, March, 1986, which is incorporated by reference herein). The radioactive isotope can be detected by such means as the use of a gamma. counter or a scintillation counter or by autoradiography.","Pharmaceutical Compositions","In another aspect, the present invention provides a composition, e.g., a pharmaceutical composition, containing one or a combination of monoclonal antibodies, or antigen-binding portions thereof, of the present invention, formulated together with a pharmaceutically acceptable carrier. Such compositions may include one or a combination of (e.g., two or more different) antibodies, or immunoconjugates or bispecific molecules of the invention. For example, a pharmaceutical composition of the invention can comprise a combination of antibodies (or immunoconjugates or bispecifics) that bind to different epitopes on the target antigen or that have complementary activities.","As discussed supra, VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 the invention further embraces identifying other molecules such as small organic molecules, peptides, ribozymes, carbohydrates, glycoprotein, siRNAs, antisense RNAs and the like which specifically bind and\/or modulate (enhance or inhibit) an activity elicited by the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen, respectively. These molecules may be identified by known screening methods such as binding assays. Typically these assays will be high throughput and will screen a large library of synthesized or native compounds in order to identify putative drug candidates that bind and\/or modulate VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 related activities.","Specifically, the invention embraces the development of drugs containing the ectodomain of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen or a fragment or variant thereof or a corresponding nucleic acid sequence encoding. These conjugates may contain a targeting or other moiety such as an immunoglobulin domain. These conjugates may be expressed in known vector systems or cells or vectors containing the corresponding nucleic acid sequences may be used for cancer treatment and in immune therapy such as in the treatment of autoimmunity, transplant, GVHD, cancer, and other immune disorders or conditions.","Thus, the present invention features a pharmaceutical composition comprising a therapeutically effective amount of a therapeutic agent according to the present invention. According to the present invention the therapeutic agent could be any one of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 ectodomain, or a fragment or variant thereof, or a corresponding nucleic acid sequence encoding.","The pharmaceutical composition according to the present invention is further preferably used for the treatment of cancers including by way of example non-solid and solid tumors, sarcomas, hematological malignancies including but not limited to acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, cancer of the breast, prostate, lung, ovary, colon, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, brain and wherein the cancer may be non-metastatic, invasive or metastatic.","The pharmaceutical composition according to the present invention is further used for the treatment of autoimmunity and preferably for treating an autoimmune disease selected from: Multiple sclerosis; Psoriasis; Rheumatoid arthritis; Systemic lupus erythematosus; Ulcerative colitis; Crohn's disease; immune disorders associated with graft transplantation rejection, benign lymphocytic angiitis, lupus erythematosus, Hashimoto's thyroiditis, primary myxedema, Graves' disease, pernicious anemia, autoimmune atrophic gastritis, Addison's disease, insulin dependent diabetes mellitis, good pasture's syndrome, myasthenia gravis, pemphigus, sympathetic ophthalmia, autoimmune uveitis, autoimmune hemolytic anemia, idiopathic thrombocytopenia, primary biliary cirrhosis, chronic action hepatitis, ulceratis colitis, Sjogren's syndrome, rheumatic disease, polymyositis, scleroderma, mixed connective tissue disease, inflammatory rheumatism, degenerative rheumatism, extra-articular rheumatism, collagen diseases, chronic polyarthritis, psoriasis arthropathica, ankylosing spondylitis, juvenile rheumatoid arthritis, periarthritis humeroscapularis, panarteriitis nodosa, progressive systemic scleroderma, arthritis uratica, dermatomyositis, muscular rheumatism, myositis, myogelosis and chondrocalcinosis.","The pharmaceutical composition according to the present invention is preferably used for the treatment of for rejection of any organ transplant and\/or Graft versus host disease which might develop after bone marrow transplantation.","\u201cTreatment\u201d refers to both therapeutic treatment and prophylactic or preventative measures. Those in need of treatment include those already with the disorder as well as those in which the disorder is to be prevented. Hence, the mammal to be treated herein may have been diagnosed as having the disorder or may be predisposed or susceptible to the disorder. \u201cMammal\u201d for purposes of treatment refers to any animal classified as a mammal, including humans, domestic and farm animals, and zoo, sports, or pet animals, such as dogs, horses, cats, cows, etc. Preferably, the mammal is human.","The term \u201ctherapeutically effective amount\u201d refers to an amount of agent according to the present invention that is effective to treat a disease or disorder in a mammal.","The therapeutic agents of the present invention can be provided to the subject alone, or as part of a pharmaceutical composition where they are mixed with a pharmaceutically acceptable carrier.","Pharmaceutical compositions of the invention also can be administered in combination therapy, i.e., combined with other agents. For example, the combination therapy can include an anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibody or VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 modulating agent according to the present invention such as a soluble polypeptide conjugate containing the ectodomain of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen or a small molecule such as a peptide, ribozyme, siRNA, or other drug that binds VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 combined with at least one other therapeutic or immune modulatory agent. Examples of therapeutic agents that can be used in combination therapy are described in greater detail below in the section on uses of the antibodies of the invention.","As used herein, \u201cpharmaceutically acceptable carrier\u201d includes any and all solvents, dispersion media, coatings, antibacterial and antifungal agents, isotonic and absorption delaying agents, and the like that are physiologically compatible. Preferably, the carrier is suitable for intravenous, intramuscular, subcutaneous, parenteral, spinal or epidermal administration (e.g., by injection or infusion). Depending on the route of administration, the active compound, i.e., antibody, immunoconjugate, or bispecific molecule, may be coated in a material to protect the compound from the action of acids and other natural conditions that may inactivate the compound. The pharmaceutical compounds of the invention may include one or more pharmaceutically acceptable salts. A \u201cpharmaceutically acceptable salt\u201d refers to a salt that retains the desired biological activity of the parent compound and does not impart any undesired toxicological effects (see e.g., Berge, S. M., et al. (1977) J. Pharm. Sci. 66: 1-19). Examples of such salts include acid addition salts and base addition salts. Acid addition salts include those derived from nontoxic inorganic acids, such as hydrochloric, nitric, phosphoric, sulfuric, hydrobromic, hydroiodic, phosphorous and the like, as well as from nontoxic organic acids such as aliphatic mono- and dicarboxylic acids, phenyl-substituted alkanoic acids, hydroxy alkanoic acids, aromatic acids, aliphatic and aromatic sulfonic acids and the like. Base addition salts include those derived from alkaline earth metals, such as sodium, potassium, magnesium, calcium and the like, as well as from nontoxic organic amines, such as N,N\u2032-dibenzylethylenediamine, N-methylglucamine, chloroprocaine, choline, diethanolamine, ethylenediamine, procaine and the like.","A pharmaceutical composition of the invention also may include a pharmaceutically acceptable anti-oxidant. Examples of pharmaceutically acceptable antioxidants include: (1) water soluble antioxidants, such as ascorbic acid, cysteine hydrochloride, sodium bisulfate, sodium metabisulfite, sodium sulfite and the like; (2) oil-soluble antioxidants, such as ascorbyl palmitate, butylated hydroxyanisole (BHA), butylated hydroxytoluene (BHT), lecithin, propyl gallate, alpha-tocopherol, and the like; and (3) metal chelating agents, such as citric acid, ethylenediamine tetraacetic acid (EDTA), sorbitol, tartaric acid, phosphoric acid, and the like.","A pharmaceutical composition of the invention also may include a pharmaceutically acceptable anti-oxidant. Examples of pharmaceutically acceptable antioxidants include: (1) water soluble antioxidants, such as ascorbic acid, cysteine hydrochloride, sodium bisulfate, sodium metabisulfite, sodium sulfite and the like; (2) oil-soluble antioxidants, such as ascorbyl palmitate, butylated hydroxyanisole (BHA), butylated hydroxytoluene (BHT), lecithin, propyl gallate, alpha-tocopherol, and the like; and (3) metal chelating agents, such as citric acid, ethylenediamine tetraacetic acid (EDTA), sorbitol, tartaric acid, phosphoric acid, and the like. Examples of suitable aqueous and nonaqueous carriers that may be employed in the pharmaceutical compositions of the invention include water, ethanol, polyols (such as glycerol, propylene glycol, polyethylene glycol, and the like), and suitable mixtures thereof, vegetable oils, such as olive oil, and injectable organic esters, such as ethyl oleate. Proper fluidity can be maintained, for example, by the use of coating materials, such as lecithin, by the maintenance of the required particle size in the case of dispersions, and by the use of surfactants.","These compositions may also contain adjuvants such as preservatives, wetting agents, emulsifying agents and dispersing agents. Prevention of presence of microorganisms may be ensured both by sterilization procedures, supra, and by the inclusion of various antibacterial and antifungal agents, for example, paraben, chlorobutanol, phenol sorbic acid, and the like. It may also be desirable to include isotonic agents, such as sugars, sodium chloride, and the like into the compositions. In addition, prolonged absorption of the injectable pharmaceutical form may be brought about by the inclusion of agents which delay absorption such as aluminum monostearate and gelatin.","Pharmaceutically acceptable carriers include sterile aqueous solutions or dispersions and sterile powders for the extemporaneous preparation of sterile injectable solutions or dispersion. The use of such media and agents for pharmaceutically active substances is known in the art. Except insofar as any conventional media or agent is incompatible with the active compound, use thereof in the pharmaceutical compositions of the invention is contemplated. Supplementary active compounds can also be incorporated into the compositions.","Therapeutic compositions typically must be sterile and stable under the conditions of manufacture and storage. The composition can be formulated as a solution, microemulsion, liposome, or other ordered structure suitable to high drug concentration. The carrier can be a solvent or dispersion medium containing, for example, water, ethanol, polyol (for example, glycerol, propylene glycol, and liquid polyethylene glycol, and the like), and suitable mixtures thereof. The proper fluidity can be maintained, for example, by the use of a coating such as lecithin, by the maintenance of the required particle size in the case of dispersion and by the use of surfactants. In many cases, it will be preferable to include isotonic agents, for example, sugars, polyalcohols such as mannitol, sorbitol, or sodium chloride in the composition. Prolonged absorption of the injectable compositions can be brought about by including in the composition an agent that delays absorption, for example, monostearate salts and gelatin. Sterile injectable solutions can be prepared by incorporating the active compound in the required amount in an appropriate solvent with one or a combination of ingredients enumerated above, as required, followed by sterilization microfiltration. Generally, dispersions are prepared by incorporating the active compound into a sterile vehicle that contains a basic dispersion medium and the required other ingredients from those enumerated above. In the case of sterile powders for the preparation of sterile injectable solutions, the preferred methods of preparation are vacuum drying and freeze-drying (lyophilization) that yield a powder of the active ingredient plus any additional desired ingredient from a previously sterile-filtered solution thereof.","Sterile injectable solutions can be prepared by incorporating the active compound in the required amount in an appropriate solvent with one or a combination of ingredients enumerated above, as required, followed by sterilization microfiltration. Generally, dispersions are prepared by incorporating the active compound into a sterile vehicle that contains a basic dispersion medium and the required other ingredients from those enumerated above. In the case of sterile powders for the preparation of sterile injectable solutions, the preferred methods of preparation are vacuum drying and freeze-drying (lyophilization) that yield a powder of the active ingredient plus any additional desired ingredient from a previously sterile-filtered solution thereof.","The amount of active ingredient which can be combined with a carrier material to produce a single dosage form will vary depending upon the subject being treated, and the particular mode of administration. The amount of active ingredient which can be combined with a carrier material to produce a single dosage form will generally be that amount of the composition which produces a therapeutic effect. Generally, out of one hundred percent, this amount will range from about 0.01 percent to about ninety-nine percent of active ingredient, preferably from about 0.1 percent to about 70 percent, most preferably from about 1 percent to about 30 percent of active ingredient in combination with a pharmaceutically acceptable carrier.","Dosage regimens are adjusted to provide the optimum desired response (e.g., a therapeutic response). For example, a single bolus may be administered, several divided doses may be administered over time or the dose may be proportionally reduced or increased as indicated by the exigencies of the therapeutic situation. It is especially advantageous to formulate parenteral compositions in dosage unit form for ease of administration and uniformity of dosage. Dosage unit form as used herein refers to physically discrete units suited as unitary dosages for the subjects to be treated; each unit contains a predetermined quantity of active compound calculated to produce the desired therapeutic effect in association with the required pharmaceutical carrier. The specification for the dosage unit forms of the invention are dictated by and directly dependent on (a) the unique characteristics of the active compound and the particular therapeutic effect to be achieved, and (b) the limitations inherent in the art of compounding such an active compound for the treatment of sensitivity in individuals.","For administration of the antibody, the dosage ranges from about 0.0001 to 100 mg\/kg, and more usually 0.01 to 5 mg\/kg, of the host body weight. For example dosages can be 0.3 mg\/kg body weight, 1 mg\/kg body weight, 3 mg\/kg body weight, 5 mg\/kg body weight or 10 mg\/kg body weight or within the range of 1-10 mg\/kg. An exemplary treatment regime entails administration once per week, once every two weeks, once every three weeks, once every four weeks, once a month, once every 3 months or once every three to 6 months. Preferred dosage regimens for an anti-VSIG1 antibody of the invention include 1 mg\/kg body weight or 3 mg\/kg body weight via intravenous administration, with the antibody being given using one of the following dosing schedules: (i) every four weeks for six dosages, then every three months; (ii) every three weeks; (iii) 3 mg\/kg body weight once followed by 1 mg\/kg body weight every three weeks.","In some methods, two or more monoclonal antibodies with different binding specificities are administered simultaneously, in which case the dosage of each antibody administered falls within the ranges indicated. Antibody is usually administered on multiple occasions. Intervals between single dosages can be, for example, weekly, monthly, every three months or yearly. Intervals can also be irregular as indicated by measuring blood levels of antibody to the target antigen in the patient. In some methods, dosage is adjusted to achieve a plasma antibody concentration of about 1-1000 mug\/ml and in some methods about 25-300. mu.g\/ml.","Alternatively, antibody can be administered as a sustained release formulation, in which case less frequent administration is required. Dosage and frequency vary depending on the half-life of the antibody in the patient. In general, human antibodies show the longest half life, followed by humanized antibodies, chimeric antibodies, and nonhuman antibodies. The dosage and frequency of administration can vary depending on whether the treatment is prophylactic or therapeutic. In prophylactic applications, a relatively low dosage is administered at relatively infrequent intervals over a long period of time. Some patients continue to receive treatment for the rest of their lives. In therapeutic applications, a relatively high dosage at relatively short intervals is sometimes required until progression of the disease is reduced or terminated, and preferably until the patient shows partial or complete amelioration of symptoms of disease. Thereafter, the patient can be administered a prophylactic regime.","Actual dosage levels of the active ingredients in the pharmaceutical compositions of the present invention may be varied so as to obtain an amount of the active ingredient which is effective to achieve the desired therapeutic response for a particular patient, composition, and mode of administration, without being toxic to the patient. The selected dosage level will depend upon a variety of pharmacokinetic factors including the activity of the particular compositions of the present invention employed, or the ester, salt or amide thereof, the route of administration, the time of administration, the rate of excretion of the particular compound being employed, the duration of the treatment, other drugs, compounds and\/or materials used in combination with the particular compositions employed, the age, sex, weight, condition, general health and prior medical history of the patient being treated, and like factors well known in the medical arts.","A \u201ctherapeutically effective dosage\u201d of an anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32, or anti-FXYD3 antibody of the invention preferably results in a decrease in severity of disease symptoms, an increase in frequency and duration of disease symptom-free periods, an increase in lifespan, disease remission, or a prevention of impairment or disability due to the disease affliction. For example, for the treatment of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 positive tumors, e.g., lung tumors, ovarian tumors, and colon tumors, a \u201ctherapeutically effective dosage\u201d preferably inhibits cell growth or tumor growth by at least about 20%, more preferably by at least about 40%, even more preferably by at least about 60%, and still more preferably by at least about 80% relative to untreated subjects. The ability of a compound to inhibit tumor growth can be evaluated in an animal model system predictive of efficacy in human tumors. Alternatively, this property of a composition can be evaluated by examining the ability of the compound to inhibit, such inhibition in vitro by assays known to the skilled practitioner. A therapeutically effective amount of a therapeutic compound can decrease tumor size, or otherwise ameliorate symptoms in a subject. One of ordinary skill in the art would be able to determine such amounts based on such factors as the subject's size, the severity of the subject's symptoms, and the particular composition or route of administration selected.","A composition of the present invention can be administered via one or more routes of administration using one or more of a variety of methods known in the art. As will be appreciated by the skilled artisan, the route and\/or mode of administration will vary depending upon the desired results. Preferred routes of administration for antibodies of the invention include intravenous, intramuscular, intradermal, intraperitoneal, subcutaneous, spinal or other parenteral routes of administration, for example by injection or infusion. The phrase \u201cparenteral administration\u201d as used herein means modes of administration other than enteral and topical administration, usually by injection, and includes, without limitation, intravenous, intramuscular, intraarterial, intrathecal, intracapsular, intraorbital, intracardiac, intradermal, intraperitoneal, transtracheal, subcutaneous, subcuticular, intraarticular, subcapsular, subarachnoid, intraspinal, epidural and intrasternal injection and infusion.","Alternatively, an antibody or other VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 drug or molecule and their conjugates and combinations thereof that modulates a VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen activity according to the invention can be administered via a non-parenteral route, such as a topical, epidermal or mucosal route of administration, for example, intranasally, orally, vaginally, rectally, sublingually or topically.","The active compounds can be prepared with carriers that will protect the compound against rapid release, such as a controlled release formulation, including implants, transdermal patches, and microencapsulated delivery systems. Biodegradable, biocompatible polymers can be used, such as ethylene vinyl acetate, polyanhydrides, polyglycolic acid, collagen, polyorthoesters, and polylactic acid. Many methods for the preparation of such formulations are patented or generally known to those skilled in the art. See, e.g., Sustained and Controlled Release Drug Delivery Systems, J. R. Robinson, ed., Marcel Dekker, Inc., New York, 1978.","Therapeutic compositions can be administered with medical devices known in the art. For example, in a preferred embodiment, a therapeutic composition of the invention can be administered with a needles hypodermic injection device, such as the devices disclosed in U.S. Pat. No. 5,399,163; 5,383,851; 5,312,335; 5,064,413; 4,941,880; 4,790,824; or 4,596,556. Examples of well-known implants and modules useful in the present invention include: U.S. Pat. No. 4,487,603, which discloses an implantable micro-infusion pump for dispensing medication at a controlled rate; U.S. Pat. No. 4,486,194, which discloses a therapeutic device for administering medicaments through the skin; U.S. Pat. No. 4,447,233, which discloses a medication infusion pump for delivering medication at a precise infusion rate; U.S. Pat. No. 4,447,224, which discloses a variable flow implantable infusion apparatus for continuous drug delivery; U.S. Pat. No. 4,439,196, which discloses an osmotic drug delivery system having multi-chamber compartments; and U.S. Pat. No. 4,475,196, which discloses an osmotic drug delivery system. These patents are incorporated herein by reference. Many other such implants, delivery systems, and modules are known to those skilled in the art.","In certain embodiments, the antibodies or other VSIG1 related drugs of the invention can be formulated to ensure proper distribution in vivo. For example, the blood-brain barrier (BBB) excludes many highly hydrophilic compounds. To ensure that the therapeutic compounds of the invention cross the BBB (if desired), they can be formulated, for example, in liposomes. For methods of manufacturing liposomes, see, e.g., U.S. Pat. Nos. 4,522,811; 5,374,548; and 5,399,331. The liposomes may comprise one or more moieties which are selectively transported into specific cells or organs, thus enhance targeted drug delivery (see, e.g., V. V. Ranade (1989) J. Clin. Pharmacol. 29:685). Exemplary targeting moieties include folate or biotin (see, e.g., U.S. Pat. No. 5,416,016 to Low et al.); mannosides (Umezawa et al., (1988) Biochem. Biophys. Res. Commun. 153:1038); antibodies (P. G. Bloeman et al. (1995) FEBS Lett. 357:140; M. Owais et al. (1995) Antimicrob. Agents Chemother. 39:180); surfactant protein A receptor (Briscoe et al. (1995) Am. J. Physiol. 1233:134); p120 (Schreier et al. (1994) J. Biol. Chem. 269:9090); see also K. Keinanen; M. L. Laukkanen (1994) FEBS Lett. 346:123; J. J. Killion; I. J. Fidler (1994) Immunomethods 4:273.","Diagnostic Uses of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 Antigen and Corresponding Polynucleotides","According to some embodiments, the sample taken from a subject (patient) to perform the diagnostic assay according to the present invention is selected from the group consisting of a body fluid or secretion including but not limited to blood, serum, urine, plasma, prostatic fluid, seminal fluid, semen, the external secretions of the skin, respiratory, intestinal, and genitourinary tracts, tears, cerebrospinal fluid, sputum, saliva, milk, peritoneal fluid, pleural fluid, cyst fluid, secretions of the breast ductal system (and\/or lavage thereof), broncho alveolar lavage, lavage of the reproductive system and lavage of any other part of the body or system in the body; samples of any organ including isolated cells or tissues, wherein the cell or tissue can be obtained from an organ selected from, but not limited to lung, colon, ovarian and\/or breast tissue; stool or a tissue sample, or any combination thereof. In some embodiments, the term encompasses samples of in vivo cell culture constituents. Prior to be subjected to the diagnostic assay, the sample can optionally be diluted with a suitable eluant.","In some embodiments, the phrase \u201cmarker\u201d in the context of the present invention refers to a nucleic acid fragment, a peptide, or a polypeptide, which is differentially present in a sample taken from patients (subjects) having one of the herein-described diseases or conditions, as compared to a comparable sample taken from subjects who do not have one the above-described diseases or conditions.","In some embodiments, the term \u201cpolypeptide\u201d is to be understood to refer to a molecule comprising from at least 2 to several thousand or more amino acids. The term \u201cpolypeptide\u201d is to be understood to include, inter alia, native peptides (either degradation products, synthetically synthesized peptides or recombinant peptides), peptidomimetics, such as peptoids and semipeptoids or peptide analogs, which may comprise, for example, any desirable modification, including, inter alia, modifications rendering the peptides more stable while in a body or more capable of penetrating into cells, or others as will be appreciated by one skilled in the art. Such modifications include, but are not limited to N terminus modification, C terminus modification, peptide bond modification, backbone modifications, residue modification, or others. Inclusion of such peptides within the polypeptides of this invention may produce a polypeptide sharing identity with the polypeptides described herein, for example, those provided in the sequence listing.","In some embodiments, the phrase \u201cdifferentially present\u201d refers to differences in the quantity or quality of a marker present in a sample taken from patients having one of the herein-described diseases or conditions as compared to a comparable sample taken from patients who do not have one of the herein-described diseases or conditions. For example, a nucleic acid fragment may optionally be differentially present between the two samples if the amount of the nucleic acid fragment in one sample is significantly different from the amount of the nucleic acid fragment in the other sample, for example as measured by hybridization and\/or NAT-based assays. A polypeptide is differentially present between the two samples if the amount of the polypeptide in one sample is significantly different from the amount of the polypeptide in the other sample. It should be noted that if the marker is detectable in one sample and not detectable in the other, then such a marker can be considered to be differentially present. Optionally, a relatively low amount of up-regulation may serve as the marker, as described herein. One of ordinary skill in the art could easily determine such relative levels of the markers; further guidance is provided in the description of each individual marker below.","In some embodiments, the phrase \u201cdiagnostic\u201d means identifying the presence or nature of a pathologic condition. Diagnostic methods differ in their sensitivity and specificity. The \u201csensitivity\u201d of a diagnostic assay is the percentage of diseased individuals who test positive (percent of \u201ctrue positives\u201d). Diseased individuals not detected by the assay are \u201cfalse negatives.\u201d Subjects who are not diseased and who test negative in the assay are termed \u201ctrue negatives.\u201d The \u201cspecificity\u201d of a diagnostic assay is 1 minus the false positive rate, where the \u201cfalse positive\u201d rate is defined as the proportion of those without the disease who test positive. While a particular diagnostic method may not provide a definitive diagnosis of a condition, it suffices if the method provides a positive indication that aids in diagnosis.","In some embodiments, the phrase \u201cqualitative\u201d when in reference to differences in expression levels of a polynucleotide or polypeptide as described herein, refers to the presence versus absence of expression, or in some embodiments, the temporal regulation of expression, or in some embodiments, the timing of expression, or in some embodiments, any post-translational modifications to the expressed molecule, and others, as will be appreciated by one skilled in the art. In some embodiments, the phrase \u201cquantitative\u201d when in reference to differences in expression levels of a polynucleotide or polypeptide as described herein, refers to absolute differences in quantity of expression, as determined by any means, known in the art, or in other embodiments, relative differences, which may be statistically significant, or in some embodiments, when viewed as a whole or over a prolonged period of time, etc., indicate a trend in terms of differences in expression.","In some embodiments, the term \u201cdiagnosing\u201d refers to classifying a disease or a symptom, determining a severity of the disease, monitoring disease progression, forecasting an outcome of a disease and\/or prospects of recovery. The term \u201cdetecting\u201d may also optionally encompass any of the above.","Diagnosis of a disease according to the present invention can, in some embodiments, be affected by determining a level of a polynucleotide or a polypeptide of the present invention in a biological sample obtained from the subject, wherein the level determined can be correlated with predisposition to, or presence or absence of the disease. It should be noted that a \u201cbiological sample obtained from the subject\u201d may also optionally comprise a sample that has not been physically removed from the subject, as described in greater detail below.","In some embodiments, the term \u201clevel\u201d refers to expression levels of RNA and\/or protein or to DNA copy number of a marker of the present invention.","Typically the level of the marker in a biological sample obtained from the subject is different (i.e., increased or decreased) from the level of the same marker in a similar sample obtained from a healthy individual (examples of biological samples are described herein).","Numerous well known tissue or fluid collection methods can be utilized to collect the biological sample from the subject in order to determine the level of DNA, RNA and\/or polypeptide of the marker of interest in the subject.","Examples include, but are not limited to, fine needle biopsy, needle biopsy, core needle biopsy and surgical biopsy (e.g., brain biopsy), and lavage. Regardless of the procedure employed, once a biopsy\/sample is obtained the level of the marker can be determined and a diagnosis can thus be made.","Determining the level of the same marker in normal tissues of the same origin is preferably effected along-side to detect an elevated expression and\/or amplification and\/or a decreased expression, of the marker as opposed to the normal tissues.","In some embodiments, the term \u201ctest amount\u201d of a marker refers to an amount of a marker in a subject's sample that is consistent with a diagnosis of a particular disease or condition. A test amount can be either in absolute amount (e.g., microgram\/ml) or a relative amount (e.g., relative intensity of signals).","In some embodiments, the term \u201ccontrol amount\u201d of a marker can be any amount or a range of amounts to be compared against a test amount of a marker. For example, a control amount of a marker can be the amount of a marker in a patient with a particular disease or condition or a person without such a disease or condition. A control amount can be either in absolute amount (e.g., microgram\/ml) or a relative amount (e.g., relative intensity of signals).","In some embodiments, the term \u201cdetect\u201d refers to identifying the presence, absence or amount of the object to be detected.","In some embodiments, the term \u201clabel\u201d includes any moiety or item detectable by spectroscopic, photo chemical, biochemical, immunochemical, or chemical means. For example, useful labels include 32P, 35S, fluorescent dyes, electron-dense reagents, enzymes (e.g., as commonly used in an ELISA), biotin-streptavadin, dioxigenin, haptens and proteins for which antisera or monoclonal antibodies are available, or nucleic acid molecules with a sequence complementary to a target. The label often generates a measurable signal, such as a radioactive, chromogenic, or fluorescent signal, that can be used to quantify the amount of bound label in a sample. The label can be incorporated in or attached to a primer or probe either covalently, or through ionic, van der Waals or hydrogen bonds, e.g., incorporation of radioactive nucleotides, or biotinylated nucleotides that are recognized by streptavadin. The label may be directly or indirectly detectable. Indirect detection can involve the binding of a second label to the first label, directly or indirectly. For example, the label can be the ligand of a binding partner, such as biotin, which is a binding partner for streptavadin, or a nucleotide sequence, which is the binding partner for a complementary sequence, to which it can specifically hybridize. The binding partner may itself be directly detectable, for example, an antibody may be itself labeled with a fluorescent molecule. The binding partner also may be indirectly detectable, for example, a nucleic acid having a complementary nucleotide sequence can be a part of a branched DNA molecule that is in turn detectable through hybridization with other labeled nucleic acid molecules (see, e.g., P. D. Fahrlander and A. Klausner, Bio\/Technology 6:1165 (1988)). Quantitation of the signal is achieved by, e.g., scintillation counting, densitometry, or flow cytometry.","Exemplary detectable labels, optionally and preferably for use with immunoassays, include but are not limited to magnetic beads, fluorescent dyes, radiolabels, enzymes (e.g., horse radish peroxide, alkaline phosphatase and others commonly used in an ELISA), and calorimetric labels such as colloidal gold or colored glass or plastic beads. Alternatively, the marker in the sample can be detected using an indirect assay, wherein, for example, a second, labeled antibody is used to detect bound marker-specific antibody, and\/or in a competition or inhibition assay wherein, for example, a monoclonal antibody which binds to a distinct epitope of the marker are incubated simultaneously with the mixture.","\u201cImmunoassay\u201d is an assay that uses an antibody to specifically bind an antigen. The immunoassay is characterized by the use of specific binding properties of a particular antibody to isolate, target, and\/or quantify the antigen.","The phrase \u201cspecifically (or selectively) binds\u201d to an antibody or \u201cspecifically (or selectively) immunoreactive with,\u201d or \u201cspecifically interacts or binds\u201d when referring to a protein or peptide (or other epitope), refers, in some embodiments, to a binding reaction that is determinative of the presence of the protein in a heterogeneous population of proteins and other biologics. Thus, under designated immunoassay conditions, the specified antibodies bind to a particular protein at least two times greater than the background (non-specific signal) and do not substantially bind in a significant amount to other proteins present in the sample. Specific binding to an antibody under such conditions may require an antibody that is selected for its specificity for a particular protein. For example, polyclonal antibodies raised to seminal basic protein from specific species such as rat, mouse, or human can be selected to obtain only those polyclonal antibodies that are specifically immunoreactive with seminal basic protein and not with other proteins, except for polymorphic variants and alleles of seminal basic protein. This selection may be achieved by subtracting out antibodies that cross-react with seminal basic protein molecules from other species. A variety of immunoassay formats may be used to select antibodies specifically immunoreactive with a particular protein. For example, solid-phase ELISA immunoassays are routinely used to select antibodies specifically immunoreactive with a protein (see, e.g., Harlow & Lane, Antibodies, A Laboratory Manual (1988), for a description of immunoassay formats and conditions that can be used to determine specific immunoreactivity). Typically a specific or selective reaction will be at least twice background signal or noise and more typically more than 10 to 100 times background.","In another embodiment, this invention provides a method for detecting the polypeptides of this invention in a biological sample, comprising: contacting a biological sample with an antibody specifically recognizing a polypeptide according to the present invention and detecting said interaction; wherein the presence of an interaction correlates with the presence of a polypeptide in the biological sample.","In some embodiments of the present invention, the polypeptides described herein are non-limiting examples of markers for diagnosing a disease and\/or an indicative condition. Each marker of the present invention can be used alone or in combination, for various uses, including but not limited to, prognosis, prediction, screening, early diagnosis, determination of progression, therapy selection and treatment monitoring of a disease and\/or an indicative condition.","In a related object the detected diseases will include cancers such as non-solid and solid tumors, sarcomas, hematological malignancies including but not limited to acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, cancer of the breast, prostate, lung, ovary, colon, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, brain and wherein the cancer may be non-metastatic, invasive or metastatic.","In another related object the detected diseases will include autoimmune and neoplastic disorders selected from the group consisting of Multiple sclerosis; Psoriasis; Rheumatoid arthritis; Systemic lupus erythematosus; Ulcerative colitis; Crohn's disease; immune disorders associated with graft transplantation rejection, benign lymphocytic angiitis, lupus erythematosus, Hashimoto's thyroiditis, primary myxedema, Graves' disease, pernicious anemia, autoimmune atrophic gastritis, Addison's disease, insulin dependent diabetes mellitis, good pasture's syndrome, myasthenia gravis, pemphigus, sympathetic ophthalmia, autoimmune uveitis, autoimmune hemolytic anemia, idiopathic thrombocytopenia, primary biliary cirrhosis, chronic action hepatitis, ulceratis colitis, Sjogren's syndrome, rheumatic disease, polymyositis, scleroderma, mixed connective tissue disease, inflammatory rheumatism, degenerative rheumatism, extra-articular rheumatism, collagen diseases, chronic polyarthritis, psoriasis arthropathica, ankylosing spondylitis, juvenile rheumatoid arthritis, periarthritis humeroscapularis, panarteriitis nodosa, progressive systemic scleroderma, arthritis uratica, dermatomyositis, muscular rheumatism, myositis, myogelosis and chondrocalcinosis.","In another related object the detected diseases will include rejection of any organ transplant and\/or Graft versus host disease.","Each polypeptide\/polynucleotide of the present invention can be used alone or in combination, for various uses, including but not limited to, prognosis, prediction, screening, early diagnosis, determination of progression, therapy selection and treatment monitoring of disease and\/or an indicative condition, as detailed above.","Such a combination may optionally comprise any subcombination of markers, and\/or a combination featuring at least one other marker, for example a known marker. Furthermore, such a combination may optionally and preferably be used as described above with regard to determining a ratio between a quantitative or semi-quantitative measurement of any marker described herein to any other marker described herein, and\/or any other known marker, and\/or any other marker.","According to further embodiments of the present invention markers of the present invention might optionally be used alone or in combination with known markers for lung cancer, including but not limited to CEA, CA15-3, Beta-2-microglobulin, CA19-9, TPA, and\/or in combination with the known proteins for the variant marker as described herein.","According to further embodiments of the present invention markers of the present invention might optionally be used alone or in combination with known markers for ovarian cancer, including but not limited to CEA, CA125 (Mucin 16), CA72-4TAG, CA-50, CA 54-61, CA-195 and CA 19-9 in combination with CA-125, and\/or in combination with the known proteins for the variant marker as described herein.","According to further embodiments of the present invention markers of the present invention might optionally be used alone or in combination with known markers for colon cancer, including but not limited to CEA, CA19-9, CA50, and\/or in combination with the known proteins for the variant marker as described herein.","In some embodiments of the present invention, there are provided of methods, uses, devices and assays for the diagnosis of a disease or condition. Optionally a plurality of markers may be used with the present invention. The plurality of markers may optionally include a markers described herein, and\/or one or more known markers. The plurality of markers is preferably then correlated with the disease or condition. For example, such correlating may optionally comprise determining the concentration of each of the plurality of markers, and individually comparing each marker concentration to a threshold level. Optionally, if the marker concentration is above or below the threshold level (depending upon the marker and\/or the diagnostic test being performed), the marker concentration correlates with the disease or condition. Optionally and preferably, a plurality of marker concentrations correlates with the disease or condition.","Alternatively, such correlating may optionally comprise determining the concentration of each of the plurality of markers, calculating a single index value based on the concentration of each of the plurality of markers, and comparing the index value to a threshold level.","Also alternatively, such correlating may optionally comprise determining a temporal change in at least one of the markers, and wherein the temporal change is used in the correlating step.","Also alternatively, such correlating may optionally comprise determining whether at least \u201cX\u201d number of the plurality of markers has a concentration outside of a predetermined range and\/or above or below a threshold (as described above). The value of \u201cX\u201d may optionally be one marker, a plurality of markers or all of the markers; alternatively or additionally, rather than including any marker in the count for \u201cX\u201d, one or more specific markers of the plurality of markers may optionally be required to correlate with the disease or condition (according to a range and\/or threshold).","Also alternatively, such correlating may optionally comprise determining whether a ratio of marker concentrations for two markers is outside a range and\/or above or below a threshold. Optionally, if the ratio is above or below the threshold level and\/or outside a range, the ratio correlates with the disease or condition.","Optionally, a combination of two or more these correlations may be used with a single panel and\/or for correlating between a plurality of panels.","Optionally, the method distinguishes a disease or condition with a sensitivity of at least 70% at a specificity of at least 85% when compared to normal subjects. As used herein, sensitivity relates to the number of positive (diseased) samples detected out of the total number of positive samples present; specificity relates to the number of true negative (non-diseased) samples detected out of the total number of negative samples present. Preferably, the method distinguishes a disease or condition with a sensitivity of at least 80% at a specificity of at least 90% when compared to normal subjects. More preferably, the method distinguishes a disease or condition with a sensitivity of at least 90% at a specificity of at least 90% when compared to normal subjects. Also more preferably, the method distinguishes a disease or condition with a sensitivity of at least 70% at a specificity of at least 85% when compared to subjects exhibiting symptoms that mimic disease or condition symptoms.","A marker panel may be analyzed in a number of fashions well known to those of skill in the art. For example, each member of a panel may be compared to a \u201cnormal\u201d value, or a value indicating a particular outcome. A particular diagnosis\/prognosis may depend upon the comparison of each marker to this value; alternatively, if only a subset of markers is outside of a normal range, this subset may be indicative of a particular diagnosis\/prognosis. The skilled artisan will also understand that diagnostic markers, differential diagnostic markers, prognostic markers, time of onset markers, disease or condition differentiating markers, etc., may be combined in a single assay or device. Markers may also be commonly used for multiple purposes by, for example, applying a different threshold or a different weighting factor to the marker for the different purposes.","In one embodiment, the panels comprise markers for the following purposes: diagnosis of a disease; diagnosis of disease and indication if the disease is in an acute phase and\/or if an acute attack of the disease has occurred; diagnosis of disease and indication if the disease is in a non-acute phase and\/or if a non-acute attack of the disease has occurred; indication whether a combination of acute and non-acute phases or attacks has occurred; diagnosis of a disease and prognosis of a subsequent adverse outcome; diagnosis of a disease and prognosis of a subsequent acute or non-acute phase or attack; disease progression (for example for cancer, such progression may include for example occurrence or recurrence of metastasis).","The above diagnoses may also optionally include differential diagnosis of the disease to distinguish it from other diseases, including those diseases that may feature one or more similar or identical symptoms.","In certain embodiments, one or more diagnostic or prognostic indicators are correlated to a condition or disease by merely the presence or absence of the indicators. In other embodiments, threshold levels of a diagnostic or prognostic indicators can be established, and the level of the indicators in a patient sample can simply be compared to the threshold levels. The sensitivity and specificity of a diagnostic and\/or prognostic test depends on more than just the analytical \u201cquality\u201d of the test\u2014they also depend on the definition of what constitutes an abnormal result. In practice, Receiver Operating Characteristic curves, or \u201cROC\u201d curves, are typically calculated by plotting the value of a variable versus its relative frequency in \u201cnormal\u201d and \u201cdisease\u201d populations, and\/or by comparison of results from a subject before, during and\/or after treatment.","According to embodiments of the present invention, VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 protein, polynucleotide or a fragment thereof, may be featured as a biomarker for detecting disease and\/or an indicative condition, as detailed above.","According to still other embodiments, the present invention optionally and preferably encompasses any amino acid sequence or fragment thereof encoded by a nucleic acid sequence corresponding to VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 as described herein. Any oligopeptide or peptide relating to such an amino acid sequence or fragment thereof may optionally also (additionally or alternatively) be used as a biomarker.","In still other embodiments, the present invention provides a method for detecting a polynucleotide of this invention in a biological sample, using NAT based assays, comprising: hybridizing the isolated nucleic acid molecules or oligonucleotide fragments of at least about a minimum length to a nucleic acid material of a biological sample and detecting a hybridization complex; wherein the presence of a hybridization complex correlates with the presence of the polynucleotide in the biological sample. Non-limiting examples of methods or assays are described below.","The present invention also relates to kits based upon such diagnostic methods or assays.","Nucleic Acid Technology (NAT) Based Assays:","Detection of a nucleic acid of interest in a biological sample may also optionally be effected by NAT-based assays, which involve nucleic acid amplification technology, such as PCR for example (or variations thereof such as real-time PCR for example). As used herein, a \u201cprimer\u201d defines an oligonucleotide which is capable of annealing to (hybridizing with) a target sequence, thereby creating a double stranded region which can serve as an initiation point for DNA synthesis under suitable conditions. Amplification of a selected, or target, nucleic acid sequence may be carried out by a number of suitable methods known in the art. Non-limiting examples of amplification techniques include polymerase chain reaction (PCR), ligase chain reaction (LCR), strand displacement amplification (SDA), transcription-based amplification, the q3 replicase system and NASBA (Kwoh et al., 1989, Proc. Natl. Acad. Sci. USA 86, 1173-1177; Lizardi et al., 1988, BioTechnology 6:1197-1202; Malek et al., 1994, Methods Mol. Biol., 28:253-260; and Sambrook et al., 1989, supra). Non-limiting examples of Nucleic Acid Technology-based assay is selected from the group consisting of a PCR, Real-Time PCR, LCR, Self-Sustained Synthetic Reaction, Q-Beta Replicase, Cycling probe reaction, Branched DNA, RFLP analysis, DGGE\/TGGE, Single-Strand Conformation Polymorphism, Dideoxy fingerprinting, microarrays, Fluorescense In Situ Hybridization and Comparative Genomic Hybridization. The terminology \u201camplification pair\u201d (or \u201cprimer pair\u201d) refers herein to a pair of oligonucleotides (oligos) of the present invention, which are selected to be used together in amplifying a selected nucleic acid sequence by one of a number of types of amplification processes, preferably a polymerase chain reaction. As commonly known in the art, the oligos are designed to bind to a complementary sequence under selected conditions. In one particular embodiment, amplification of a nucleic acid sample from a patient is amplified under conditions which favor the amplification of the most abundant differentially expressed nucleic acid. In one preferred embodiment, RT-PCR is carried out on an mRNA sample from a patient under conditions which favor the amplification of the most abundant mRNA. In another preferred embodiment, the amplification of the differentially expressed nucleic acids is carried out simultaneously. It will be realized by a person skilled in the art that such methods could be adapted for the detection of differentially expressed proteins instead of differentially expressed nucleic acid sequences. The nucleic acid (i.e. DNA or RNA) for practicing the present invention may be obtained according to well known methods.","Oligonucleotide primers of the present invention may be of any suitable length, depending on the particular assay format and the particular needs and targeted genomes employed. Optionally, the oligonucleotide primers are at least 12 nucleotides in length, preferably between 15 and 24 molecules, and they may be adapted to be especially suited to a chosen nucleic acid amplification system. As commonly known in the art, the oligonucleotide primers can be designed by taking into consideration the melting point of hybridization thereof with its targeted sequence (Sambrook et al., 1989, Molecular Cloning\u2014A Laboratory Manual, 2nd Edition, CSH Laboratories; Ausubel et al., 1989, in Current Protocols in Molecular Biology, John Wiley & Sons Inc., N.Y.).","Immunoassays","In another embodiment of the present invention, an immunoassay can be used to qualitatively or quantitatively detect and analyze markers in a sample. This method comprises: providing an antibody that specifically binds to a marker; contacting a sample with the antibody; and detecting the presence of a complex of the antibody bound to the marker in the sample.","To prepare an antibody that specifically binds to a marker, purified protein markers can be used. Antibodies that specifically bind to a protein marker can be prepared using any suitable methods known in the art.","After the antibody is provided, a marker can be detected and\/or quantified using any of a number of well recognized immunological binding assays. Useful assays include, for example, an enzyme immune assay (EIA) such as enzyme-linked immunosorbent assay (ELISA), a radioimmune assay (RIA), a Western blot assay, or a slot blot assay see, e.g., U.S. Pat. Nos. 4,366,241; 4,376,110; 4,517,288; and 4,837,168). Generally, a sample obtained from a subject can be contacted with the antibody that specifically binds the marker.","Optionally, the antibody can be fixed to a solid support to facilitate washing and subsequent isolation of the complex, prior to contacting the antibody with a sample. Examples of solid supports include but are not limited to glass or plastic in the form of, e.g., a microtiter plate, a stick, a bead, or a microbead. Antibodies can also be attached to a solid support.","After incubating the sample with antibodies, the mixture is washed and the antibody-marker complex formed can be detected. This can be accomplished by incubating the washed mixture with a detection reagent. Alternatively, the marker in the sample can be detected using an indirect assay, wherein, for example, a second, labeled antibody is used to detect bound marker-specific antibody, and\/or in a competition or inhibition assay wherein, for example, a monoclonal antibody which binds to a distinct epitope of the marker are incubated simultaneously with the mixture.","Throughout the assays, incubation and\/or washing steps may be required after each combination of reagents. Incubation steps can vary from about 5 seconds to several hours, preferably from about 5 minutes to about 24 hours. However, the incubation time will depend upon the assay format, marker, volume of solution, concentrations and the like. Usually the assays will be carried out at ambient temperature, although they can be conducted over a range of temperatures, such as 10\u00b0 C. to 40\u00b0 C.","The immunoassay can be used to determine a test amount of a marker in a sample from a subject. First, a test amount of a marker in a sample can be detected using the immunoassay methods described above. If a marker is present in the sample, it will form an antibody-marker complex with an antibody that specifically binds the marker under suitable incubation conditions described above. The amount of an antibody-marker complex can optionally be determined by comparing to a standard. As noted above, the test amount of marker need not be measured in absolute units, as long as the unit of measurement can be compared to a control amount and\/or signal.","Radio-immunoassay (RIA): In one version, this method involves precipitation of the desired substrate and in the methods detailed herein below, with a specific antibody and radiolabeled antibody binding protein (e.g., protein A labeled with I125) immobilized on a precipitable carrier such as agarose beads. The number of counts in the precipitated pellet is proportional to the amount of substrate.","In an alternate version of the RIA, a labeled substrate and an unlabelled antibody binding protein are employed. A sample containing an unknown amount of substrate is added in varying amounts. The decrease in precipitated counts from the labeled substrate is proportional to the amount of substrate in the added sample.","Enzyme linked immunosorbent assay (ELISA): This method involves fixation of a sample (e.g., fixed cells or a proteinaceous solution) containing a protein substrate to a surface such as a well of a microtiter plate. A substrate specific antibody coupled to an enzyme is applied and allowed to bind to the substrate. Presence of the antibody is then detected and quantitated by a colorimetric reaction employing the enzyme coupled to the antibody. Enzymes commonly employed in this method include horseradish peroxidase and alkaline phosphatase. If well calibrated and within the linear range of response, the amount of substrate present in the sample is proportional to the amount of color produced. A substrate standard is generally employed to improve quantitative accuracy.","Western blot: This method involves separation of a substrate from other protein by means of an acrylamide gel followed by transfer of the substrate to a membrane (e.g., nylon or PVDF). Presence of the substrate is then detected by antibodies specific to the substrate, which are in turn detected by antibody binding reagents. Antibody binding reagents may be, for example, protein A, or other antibodies. Antibody binding reagents may be radiolabeled or enzyme linked as described hereinabove. Detection may be by autoradiography, colorimetric reaction or chemiluminescence. This method allows both quantitation of an amount of substrate and determination of its identity by a relative position on the membrane which is indicative of a migration distance in the acrylamide gel during electrophoresis.","Immunohistochemical analysis: This method involves detection of a substrate in situ in fixed cells by substrate specific antibodies. The substrate specific antibodies may be enzyme linked or linked to fluorophores. Detection is by microscopy and subjective evaluation. If enzyme linked antibodies are employed, a colorimetric reaction may be required.","Fluorescence activated cell sorting (FACS): This method involves detection of a substrate in situ in cells by substrate specific antibodies. The substrate specific antibodies are linked to fluorophores. Detection is by means of a cell sorting machine which reads the wavelength of light emitted from each cell as it passes through a light beam. This method may employ two or more antibodies simultaneously.","Radio-Imaging Methods","These methods include but are not limited to, positron emission tomography (PET) single photon emission computed tomography (SPECT). Both of these techniques are non-invasive, and can be used to detect and\/or measure a wide variety of tissue events and\/or functions, such as detecting cancerous cells for example. Unlike PET, SPECT can optionally be used with two labels simultaneously. SPECT has some other advantages as well, for example with regard to cost and the types of labels that can be used. For example, U.S. Pat. No. 6,696,686 describes the use of SPECT for detection of breast cancer, and is hereby incorporated by reference as if fully set forth herein.","Theranostics:","The term theranostics describes the use of diagnostic testing to diagnose the disease, choose the correct treatment regime according to the results of diagnostic testing and\/or monitor the patient response to therapy according to the results of diagnostic testing. Theranostic tests can be used to select patients for treatments that are particularly likely to benefit them and unlikely to produce side-effects. They can also provide an early and objective indication of treatment efficacy in individual patients, so that (if necessary) the treatment can be altered with a minimum of delay. For example: DAKO and Genentech together created HercepTest and Herceptin (trastuzumab) for the treatment of breast cancer, the first theranostic test approved simultaneously with a new therapeutic drug. In addition to HercepTest (which is an immunohistochemical test), other theranostic tests are in development which use traditional clinical chemistry, immunoassay, cell-based technologies and nucleic acid tests. PPGx's recently launched TPMT (thiopurine S-methyltransferase) test, which is enabling doctors to identify patients at risk for potentially fatal adverse reactions to 6-mercaptopurine, an agent used in the treatment of leukemia. Also, Nova Molecular pioneered SNP genotyping of the apolipoprotein E gene to predict Alzheimer's disease patients' responses to cholinomimetic therapies and it is now widely used in clinical trials of new drugs for this indication. Thus, the field of theranostics represents the intersection of diagnostic testing information that predicts the response of a patient to a treatment with the selection of the appropriate treatment for that particular patient.","Surrogate Markers:","A surrogate marker is a marker, that is detectable in a laboratory and\/or according to a physical sign or symptom on the patient, and that is used in therapeutic trials as a substitute for a clinically meaningful endpoint. The surrogate marker is a direct measure of how a patient feels, functions, or survives which is expected to predict the effect of the therapy. The need for surrogate markers mainly arises when such markers can be measured earlier, more conveniently, or more frequently than the endpoints of interest in terms of the effect of a treatment on a patient, which are referred to as the clinical endpoints. Ideally, a surrogate marker should be biologically plausible, predictive of disease progression and measurable by standardized assays (including but not limited to traditional clinical chemistry, immunoassay, cell-based technologies, nucleic acid tests and imaging modalities).","Surrogate endpoints were used first mainly in the cardiovascular area. For example, antihypertensive drugs have been approved based on their effectiveness in lowering blood pressure. Similarly, in the past, cholesterol-lowering agents have been approved based on their ability to decrease serum cholesterol, not on the direct evidence that they decrease mortality from atherosclerotic heart disease. The measurement of cholesterol levels is now an accepted surrogate marker of atherosclerosis. In addition, currently two commonly used surrogate markers in HIV studies are CD4+ T cell counts and quantitative plasma HIV RNA (viral load). In some embodiments of this invention, the polypeptide\/polynucleotide expression pattern may serve as a surrogate marker for a particular disease, as will be appreciated by one skilled in the art.","Uses and Methods of the Invention","The VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 drugs according to the invention, especially antibodies, particularly the human antibodies, antibody compositions, and soluble conjugates containing the ectodomain of the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen or a fragment or variant thereof, or a corresponding nucleic acid sequence or vector or cell expressing same and methods of the present invention have numerous in vitro and in vivo diagnostic and therapeutic utilities involving the diagnosis and treatment of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen related disorders and\/or disorders wherein modulation of immune co-stimulation e.g., involving B7-related immune costimulation involving VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen is therapeutically desirable. As noted these conditions include in particular cancers that differentially express the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen such as lung cancer, ovarian cancer, colon cancer, including invasive and metastatic forms thereof, and\/or autoimmune conditions wherein modulation of costimulation such as involving B7 is therapeutically desirable. The subject anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32 or anti-FXYD3 antibodies may prevent B7 mediated negative stimulation of T cell activity against cancer cells and\/or prevent positive stimulation of T cell activity. Such antibodies may be used in the treatment of conditions including cancers such non-solid and solid tumors, sarcomas, hematological malignancies including but not limited to acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, cancer of the breast, prostate, lung, ovary, colon, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, brain and wherein the cancer may be non-metastatic, invasive or metastatic as well as non-malignant disorders such as immune disorders including but not limited to transplant rejection and graft versus host disease, and autoimmune disorders such as afore-mentioned.","For example, these molecules can be administered to cells in culture, in vitro or ex vivo, or to human subjects, e.g., in vivo, to treat, prevent and to diagnose a variety of disorders. Preferred subjects include human patients having disorders mediated by cells expressing the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 antigen and cells that posses VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 activity. The methods are particularly suitable for treating human patients having a disorder associated with aberrant VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen expression using antibodies that specifically bind AI581519_P3 (SEQ ID NO:11), AI581519_P4 (SEQ ID NO:12), AI581519_P5 (SEQ ID NO:13), AI581519_P7 (SEQ ID NO:14), AI581519_P9 (SEQ ID NO:15), AI581519_P10 (SEQ ID NO:16), AA424839_P3 (SEQ ID NO:22), AA424839_P5 (SEQ ID NO:21), AA424839_P7 (SEQ ID NO:23), or AA4248391_P11 (SEQ ID NO:24), H686541_P2 (SEQ ID NO:35), H686541_P5 (SEQ ID NO:36), H686541_P7 (SEQ ID NO:37), H686541_P12 (SEQ ID NO:38), H686541_P13 (SEQ ID NO:39), H686541_P14 (SEQ ID NO:40), AI216611_P0 (SEQ ID NO:43), AI216611_P1 (SEQ ID NO:44), H190111_P8 (SEQ ID NO:48), H190111_P9 (SEQ ID NO:50), R31375_P0 (SEQ ID NO:70), R31375_P14 (SEQ ID NO:72), R31375_P31 (SEQ ID NO:73) or R31375_P33 (SEQ ID NO:74).","VSIG1, ILDR1, LOC253012, AI216611, C1ORF32, or FXYD3 drugs according to the invention, are administered together with another agent, the two can be administered in either order or simultaneously.","Given the specific binding of the antibodies of the invention for VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 the antibodies of the invention can be used to specifically detect VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 expression on the surface of cells and, moreover, can be used to purify VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen via immunoaffinity purification.","Furthermore, given the expression of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 on various tumor cells, the human antibodies, antibody compositions and methods of the present invention can be used to treat a subject with a tumorigenic disorder, e.g., a disorder characterized by the presence of tumor cells expressing VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen such as lung cancer and ovarian cancer, as mentioned.","In one embodiment, the antibodies (e.g., human monoclonal antibodies, multispecific and bispecific molecules and compositions) of the invention can be used to detect levels of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 or levels of cells which contain VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3, respectively, on their membrane surface, which levels can then be linked to certain disease symptoms. Alternatively, the antibodies can be used to inhibit or block VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 function which, in turn, can be linked to the prevention or amelioration of certain disease symptoms, thereby implicating VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3, respectively, as a mediator of the disease. This can be achieved by contacting a sample and a control sample with the anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32 or anti-FXYD3 antibody under conditions that allow for the formation of a complex between the corresponding antibody and VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3, respectively. Any complexes formed between the antibody and VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 are detected and compared in the sample and the control.","In another embodiment, the antibodies (e.g., human antibodies, multispecific and bispecific molecules and compositions) of the invention can be initially tested for binding activity associated with therapeutic or diagnostic use in vitro. For example, compositions of the invention can be tested using low cytometric assays.","The antibodies (e.g., human antibodies, multispecific and bispecific molecules, immunoconjugates and compositions) of the invention have additional utility in therapy and diagnosis of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3-related diseases. For example, the human monoclonal antibodies, the multispecific or bispecific molecules and the immunoconjugates can be used to elicit in vivo or in vitro one or more of the following biological activities: to inhibit the growth of and\/or kill a cell expressing VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3; to mediate phagocytosis or ADCC of a cell expressing VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 in the presence of human effector cells, or to block VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 ligand binding to VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3, respectively.","In a particular embodiment, the antibodies (e.g., human antibodies, multispecific and bispecific molecules and compositions) are used in vivo to treat, prevent or diagnose a variety of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3-related diseases. Examples of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3-related diseases include, among others, cancer, such as lung cancer, ovarian cancer, colon cancer, other non-solid and solid tumors, sarcomas, hematological malignancies including but not limited to acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, cancer of the breast, prostate, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, brain and wherein the cancer may be non-metastatic, invasive or metastatic. Additional examples of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3-related diseases include, among others, non-malignant disorders such as immune disorders including but not limited to autoimmune diseases, transplant rejection and graft versus host disease. Such disorders include by way of example autoimmune diseases selected from multiple sclerosis; psoriasis; rheumatoid arthritis; Systemic lupus erythematosus; Ulcerative colitis; Crohn's disease, immune disorders associated with graft transplantation rejection, benign lymphocytic angiitis, lupus erythematosus, Hashimoto's thyroiditis, primary myxedema, Graves' disease, pernicious anemia, autoimmune atrophic gastritis, Addison's disease, insulin dependent diabetes mellitis, Good pasture's syndrome, myasthenia gravis, pemphigus, sympathetic ophthalmia, autoimmune uveitis, autoimmune hemolytic anemia, idiopathic thrombocytopenia, primary biliary cirrhosis, chronic action hepatitis, ulceratis colitis, Sjogren's syndrome, rheumatic disease, polymyositis, scleroderma, mixed connective tissue disease, inflammatory rheumatism, degenerative rheumatism, extra-articular rheumatism, collagen diseases, chronic polyarthritis, psoriasis arthropathica, ankylosing spondylitis, juvenile rheumatoid arthritis, periarthritis humeroscapularis, panarteriitis nodosa, progressive systemic scleroderma, arthritis uratica, dermatomyositis, muscular rheumatism, myositis, myogelosis and chondrocalcinosis.","Suitable routes of administering the antibody compositions (e.g., human monoclonal antibodies, multispecific and bispecific molecules and immunoconjugates) of the invention in vivo and in vitro are well known in the art and can be selected by those of ordinary skill. For example, the antibody compositions can be administered by injection (e.g., intravenous or subcutaneous). Suitable dosages of the molecules used will depend on the age and weight of the subject and the concentration and\/or formulation of the antibody composition.","As previously described, human anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32 or anti-FXYD3 antibodies of the invention can be co-administered with one or other more therapeutic agents, e.g., an cytotoxic agent, a radiotoxic agent or an immunosuppressive agent. The antibody can be linked to the agent (as an immunocomplex) or can be administered separate from the agent. In the latter case (separate administration), the antibody can be administered before, after or concurrently with the agent or can be co-administered with other known therapies, e.g., an anti-cancer therapy, e.g., radiation. Such therapeutic agents include, among others, anti-neoplastic agents such as doxorubicin (adriamycin), cisplatin bleomycin sulfate, carmustine, chlorambucil, and cyclophosphamide hydroxyurea which, by themselves, are only effective at levels which are toxic or subtoxic to a patient. Cisplatin is intravenously administered as a 100 mg\/dose once every four weeks and adriamycin is intravenously administered as a 60-75 mg\/ml dose once every 21 days. Co-administration of the human anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32 or anti-FXYD3 antibodies, or antigen binding fragments thereof, of the present invention with chemotherapeutic agents provides two anti-cancer agents which operate via different mechanisms which yield a cytotoxic effect to human tumor cells. Such co-administration can solve problems due to development of resistance to drugs or a change in the antigenicity of the tumor cells which would render them unreactive with the antibody.","Target-specific effector cells, e.g., effector cells linked to compositions (e.g., human antibodies, multispecific and bispecific molecules) of the invention can also be used as therapeutic agents. Effector cells for targeting can be human leukocytes such as macrophages, neutrophils or monocytes. Other cells include eosinophils, natural killer cells and other IgG- or IgA-receptor bearing cells. If desired, effector cells can be obtained from the subject to be treated. The target-specific effector cells can be administered as a suspension of cells in a physiologically acceptable solution. The number of cells administered can be in the order of 10\u22128 to 10\u22129 but will vary depending on the therapeutic purpose. In general, the amount will be sufficient to obtain localization at the target cell, e.g., a tumor cell expressing VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 and to effect cell killing by, e.g., phagocytosis. Routes of administration can also vary.","Therapy with target-specific effector cells can be performed in conjunction with other techniques for removal of targeted cells. For example, anti-tumor therapy using the compositions (e.g., human antibodies, multispecific and bispecific molecules) of the invention and\/or effector cells armed with these compositions can be used in conjunction with chemotherapy. Additionally, combination immunotherapy may be used to direct two distinct cytotoxic effector populations toward tumor cell rejection. For example, anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32 or anti-FXYD3 antibodies linked to anti-Fc-gamma R1 or anti-CD3 may be used in conjunction with IgG- or IgA-receptor specific binding agents.","Bispecific and multispecific molecules of the invention can also be used to modulate FcgammaR or FcgammaR levels on effector cells, such as by capping and elimination of receptors on the cell surface. Mixtures of anti-Fc receptors can also be used for this purpose.","The compositions (e.g., human antibodies, multispecific and bispecific molecules and immunoconjugates) of the invention which have complement binding sites, such as portions from IgG1, -2, or -3 or IgM which bind complement, can also be used in the presence of complement. In one embodiment, ex vivo treatment of a population of cells comprising target cells with a binding agent of the invention and appropriate effector cells can be supplemented by the addition of complement or serum containing complement. Phagocytosis of target cells coated with a binding agent of the invention can be improved by binding of complement proteins. In another embodiment target cells coated with the compositions (e.g., human antibodies, multispecific and bispecific molecules) of the invention can also be lysed by complement. In yet another embodiment, the compositions of the invention do not activate complement.","The compositions (e.g., human antibodies, multispecific and bispecific molecules and immunoconjugates) of the invention can also be administered together with complement. Accordingly, within the scope of the invention are compositions comprising human antibodies, multispecific or bispecific molecules and serum or complement. These compositions are advantageous in that the complement is located in close proximity to the human antibodies, multispecific or bispecific molecules. Alternatively, the human antibodies, multispecific or bispecific molecules of the invention and the complement or serum can be administered separately.","Also within the scope of the present invention are kits comprising the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen or VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 conjugates or antibody compositions of the invention (e.g., human antibodies, bispecific or multispecific molecules, or immunoconjugates) and instructions for use. The kit can further contain one or more additional reagents, such as an immunosuppressive reagent, a cytotoxic agent or a radiotoxic agent, or one or more additional human antibodies of the invention (e.g., a human antibody having a complementary activity which binds to an epitope in the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen distinct from the first human antibody).","Accordingly, patients treated with antibody compositions of the invention can be additionally administered (prior to, simultaneously with, or following administration of a human antibody of the invention) with another therapeutic agent, such as a cytotoxic or radiotoxic agent, which enhances or augments the therapeutic effect of the human antibodies.","In other embodiments, the subject can be additionally treated with an agent that modulates, e.g., enhances or inhibits, the expression or activity of Fcy or Fcy receptors by, for example, treating the subject with a cytokine. Preferred cytokines for administration during treatment with the multispecific molecule include of granulocyte colony-stimulating factor (G-CSF), granulocyte-macrophage colony-stimulating factor (GM-CSF), interferon-.gamma. (IFN-.gamma.), and tumor necrosis factor (TNF).","The compositions (e.g., human antibodies, multispecific and bispecific molecules) of the invention can also be used to target cells expressing Fc gamma R or VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3, for example for labeling such cells. For such use, the binding agent can be linked to a molecule that can be detected. Thus, the invention provides methods for localizing ex vivo or in vitro cells expressing Fc receptors, such as FcgammaR, or VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen. The detectable label can be, e.g., a radioisotope, a fluorescent compound, an enzyme, or an enzyme co-factor.","In a particular embodiment, the invention provides methods for detecting the presence of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen in a sample, or measuring the amount of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen, respectively, comprising contacting the sample, and a control sample, with a human monoclonal antibody, or an antigen binding portion thereof, which specifically binds to VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3, respectively, under conditions that allow for formation of a complex between the antibody or portion thereof and VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3. The formation of a complex is then detected, wherein a difference complex formation between the sample compared to the control sample is indicative the presence of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen in the sample. As noted the invention in particular embraces assays for detecting VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen in vitro and in vivo such as immunoassays, radioimmunoassays, radioassays, radioimaging assays, ELISAs, Western blot, FACS, slot blot, immunohistochemical assays, and other assays well known to those skilled in the art.","In other embodiments, the invention provides methods for treating an VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 mediated disorder in a subject, e.g., cancer, such as non-solid and solid tumors, sarcomas, hematological malignancies including but not limited to acute lymphocytic leukemia, chronic lymphocytic leukemia, acute myelogenous leukemia, chronic myelogenous leukemia, multiple myeloma, Hodgkin's lymphoma, Non-Hodgkin's lymphoma, cancer of the breast, prostate, lung, ovary, colon, spleen, kidney, bladder, head and neck, uterus, testicles, stomach, cervix, liver, bone, skin, pancreas, brain and wherein the cancer may be non-metastatic, invasive or metastatic, as well as non-malignant disorders such as immune disorders including but not limited to transplant rejection and graft versus host disease, or an autoimmune disease selected from those aforementioned and methods of treating any condition wherein modulation of immune costimulation that involves VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 is therapeutically desirable using anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32 or anti-FXYD3 antibodies or soluble VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen conjugates or other drugs that target and modulate (promote or inhibit) one or more VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 biological activities.","By administering the anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32 or anti-FXYD3 antibody, soluble VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen conjugate or other drug that targets the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen or a portion thereof to a subject, the ability of VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen to induce such activities is inhibited or promoted and, thus, the associated disorder is treated. The soluble VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen or antigen conjugate or anti-VSIG1, anti-ILDR1, anti-LOC253012, anti-AI216611, anti-C1ORF32 or anti-FXYD3 antibody or fragment containing composition or other drug that targets and modulates VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3, can be administered alone or along with another therapeutic agent, such as a cytotoxic or a radiotoxic agent which acts in conjunction with or synergistically with the antibody composition to treat or prevent the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen mediated disease.","In yet another embodiment, immunoconjugates of the invention can be used to target compounds (e.g., therapeutic agents, labels, cytotoxins, radiotoxins immunosuppressants, etc.) to cells which have VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 cell surface receptors by linking such compounds to the antibody. Thus, the invention also provides methods for localizing ex vivo or in vivo cells expressing VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 (e.g., with a detectable label, such as a radioisotope, a fluorescent compound, an enzyme, or an enzyme co-factor). Alternatively, the immunoconjugates can be used to kill cells which have VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 cell surface receptors by targeting cytotoxins or radiotoxins to VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen.","The present invention is further illustrated by the following sequence characterization of a DNA transcript encoding the VSIG1, ILDR1, LOC253012, AI216611, C1ORF32 or FXYD3 antigen, its domains and expression data in normal and cancerous tissues as well as prophetic examples describing the manufacture of fully human antibodies thereto. This information and examples is illustrative and should not be construed as further limiting. The contents of all figures and all references, patents and published patent applications cited throughout this application are expressly incorporated herein by reference.","The targets of the present invention were tested with regard to their expression in various cancerous and non-cancerous tissue samples and\/or with regard to its expression in a wide panel of human samples which contains various types of immune cells, and hematological malignancies samples and cell lines, as well as several samples of normal tissues. The list of the blood specific RNA samples used for the qRT-PCR analysis is provided in Table 1 below. A description of the samples used in the normal tissue panel is provided in Table 2. A description of the samples used in the lung cancer testing panel is provided in Table 3 below. A description of the samples used in the ovary cancer testing panel is provided in Table 4 below. A description of the samples used in the colon cancer testing panel is provided in Table 5 below. The keys for the table 3, 4 and 5 are given in tables 31, 41, and 51, respectively. Tests were then performed as described in the \u201cMaterials and Experimental Procedures\u201d section below.",{"@attributes":{"id":"p-0735","num":"0734"},"tables":{"@attributes":{"id":"TABLE-US-00007","num":"00007"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"266pt","align":"center"}},"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Samples in blood specific panel"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Blood panel sample","Description","Organ\/Cell type","Tumor Type"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["1_PBMC2","PBMCs","blood-derived cells",{}]},{"entry":["2_PBMC3","PBMCs","blood-derived cells",{}]},{"entry":["3_Bcell1","B cells","blood-derived cells",{}]},{"entry":["4_Bcell2","B cells","blood-derived cells",{}]},{"entry":["5_J_Bcell","B cells","blood-derived cells",{}]},{"entry":["6_K_Bcells_act","Bcells activated","blood-derived cells",{}]},{"entry":["7_Tcell1","T cells","blood-derived cells",{}]},{"entry":["8_Tcell2","T cells","blood-derived cells",{}]},{"entry":["9_M_CD8","CD4+ T cells","blood-derived cells",{}]},{"entry":["10_G_CD4_unt","CD8+ T cells","blood-derived cells",{}]},{"entry":["11_H_CD4_Beads","CD4+ w Activation","blood-derived cells",{}]},{"entry":[{},"beads",{},{}]},{"entry":["12_I_CD4_Beads_IL12","CD4 w act.","blood-derived cells",{}]},{"entry":[{},"Beads + IL12",{},{}]},{"entry":["13_95_CD4+CD25\u2212","CD4+CD25\u2212","blood-derived cells",{}]},{"entry":["15_NK","NK cells","blood-derived cells",{}]},{"entry":["16_CD34+_1548","CD34+(PCBM1548)","blood-derived cells",{}]},{"entry":["17_CD34+_1028","CD34+(PCBM1028)","blood-derived cells",{}]},{"entry":["18_PMN","PMNs","blood-derived cells",{}]},{"entry":["19_A_Mono","Monocytes","blood-derived cells",{}]},{"entry":["20_B_Macro_imma","Macrophages","blood-derived cells",{}]},{"entry":[{},"immature",{},{}]},{"entry":["21_C_Macro_mat","Macrophages","blood-derived cells",{}]},{"entry":[{},"mature",{},{}]},{"entry":["22_D_DCs_immat","DCs immature","blood-derived cells",{}]},{"entry":["23_E_DCs_mat_LPS","DCs mature LPS","blood-derived cells",{}]},{"entry":["24_F_DCs_mat_CK","DCs mature CK","blood-derived cells",{}]},{"entry":["25_L_DCs + T","DCs + T cells","blood-derived cells",{}]},{"entry":["26_Lym1","13987A1","Lymph Node","Lymphoma"]},{"entry":["27_Lym2","43594B1","Muscle","lymphoma"]},{"entry":["28_Lym3","65493A1","Testis","Lymphoma"]},{"entry":["29_MalLym3","75894A1","Brain","Lymphoma"]},{"entry":["30_NonHod_SCLym","83325A1","Lymph Node","NHL Small"]},{"entry":[{},{},{},"Cell"]},{"entry":["31_NonHod_FolLym","76943A1(5 tubes)","Lymph Node","NHL Follicular"]},{"entry":["32_Lym_Fol_GI","CN_4_ASRBNA35",{},"NHL Follicular"]},{"entry":[{},{},{},"Grade I (Small"]},{"entry":[{},{},{},"Cell)"]},{"entry":["33_Lym_Fol_GII","CN_1_113GHA8J",{},"NHL Follicular"]},{"entry":[{},{},{},"Grade II"]},{"entry":[{},{},{},"(mixed Small"]},{"entry":[{},{},{},"& Large Cell)"]},{"entry":["34_Lym_Fol_GIII","CN_8_VXML6AXI",{},"NHL Follicular"]},{"entry":[{},{},{},"Grade III"]},{"entry":[{},{},{},"(Large Cell)"]},{"entry":["35_MalLym1","76218B1","Testis","NHL Large"]},{"entry":[{},{},{},"Cell"]},{"entry":["36_MalLym2","76102A1","Lymph Node","NHL Large"]},{"entry":[{},{},{},"Cell"]},{"entry":["37_Lym_DifBCell1","CN_2_4HDLNA2R",{},"NHL Diffuse"]},{"entry":[{},{},{},"Large B-Cell"]},{"entry":["38_Lym_DifBCell2","CN_3_4M4S7AAM",{},"NHL Diffuse"]},{"entry":[{},{},{},"Large B-Cell"]},{"entry":["39_Lym_DifBCell3","CN_5_HEODOAR2",{},"NHL Diffuse"]},{"entry":[{},{},{},"Large B-Cell"]},{"entry":["40_NonHod_Lym1","77332A1(5 tubes)","Colon","NHL Diffuse"]},{"entry":[{},{},{},"Large B-Cell"]},{"entry":["41_MalLym4","76161A1","Spleen","NHL Diffuse"]},{"entry":[{},{},{},"Large B-Cell"]},{"entry":["42_Lym_MantleCell1","CN_6_MAE47AOY",{},"NHL Mantle"]},{"entry":[{},{},{},"Cell"]},{"entry":["43_Lym_MantleCell2","CN_7_VJU9OAO9",{},"NHL Mantle"]},{"entry":[{},{},{},"Cell"]},{"entry":["44_NonHod_Lym2","95377A1(5 tubes)","Spleen","NHL"]},{"entry":["45_THP_1","THP-1","monocytes","AML cell line"]},{"entry":["46_KG_1","KG-1","myeloblast","AML cell line"]},{"entry":["47_BDCM","BDCM","B and DC like","AML cell line"]},{"entry":["48_CESS","CESS","lymphoblasts","AML cell line"]},{"entry":["49_HL60","HL60","myeloblast","AML cell line"]},{"entry":["50_K562","K562","lymphoblasts","CML cell line"]},{"entry":["51_Jurkat","Jurkat","T lymphoblasts","T ALL cell line"]},{"entry":["52_GA10","GA10","B lymphoblasts","Burkitts"]},{"entry":[{},{},{},"lymphoma cell"]},{"entry":[{},{},{},"line"]},{"entry":["53_RAMOS","RAMOS","B lymphoblasts","Burkitts"]},{"entry":[{},{},{},"lymphoma cell"]},{"entry":[{},{},{},"line"]},{"entry":["54_RAJI","RAJI","B lymphoblasts","Burkitts"]},{"entry":[{},{},{},"lymphoma cell"]},{"entry":[{},{},{},"line"]},{"entry":["55_Daudi","Daudi","B lymphoblasts","Burkitts"]},{"entry":[{},{},{},"lymphoma cell"]},{"entry":[{},{},{},"line"]},{"entry":["56_NL564","-NL564","B lymphoblasts","EBV"]},{"entry":[{},{},{},"transformed"]},{"entry":[{},{},{},"cell line"]},{"entry":["57_NL553","NL553","B lymphoblasts","EBV"]},{"entry":[{},{},{},"transformed"]},{"entry":[{},{},{},"cell line"]},{"entry":["58_SKW6.4","SKW6.4","B cells lymphoblasts","EBV"]},{"entry":[{},{},{},"transformed"]},{"entry":[{},{},{},"cell line"]},{"entry":["59_NCI_H929","NCI-H929","B lymphoblasts","Multiple"]},{"entry":[{},{},{},"Myeloma cell"]},{"entry":[{},{},{},"line"]},{"entry":["60_MC\/CAR","MC\/CAR","B lymphoblasts","Multiple"]},{"entry":[{},{},{},"Myeloma cell"]},{"entry":[{},{},{},"line"]},{"entry":["61_U266","U266","B lymphoblasts","Multiple"]},{"entry":[{},{},{},"Myeloma cell"]},{"entry":[{},{},{},"line"]},{"entry":["62_RPMI8226","RPMI8226","B lymphoblasts","Multiple"]},{"entry":[{},{},{},"Myeloma cell"]},{"entry":[{},{},{},"line"]},{"entry":["63_IM_9","IM-9","B lymphoblasts","Multiple"]},{"entry":[{},{},{},"Myeloma cell"]},{"entry":[{},{},{},"line"]},{"entry":["64_cereN","cerebellum normal","cerebellum normal",{}]},{"entry":["65_kidneyN1","kidney normal","kidney normal",{}]},{"entry":["66_kidneyN2","kidney normal","kidney normal",{}]},{"entry":["67_KidneyN3","kidney normal","kidney normal",{}]},{"entry":["68_colonN1","colon normal","colon normal",{}]},{"entry":["69_colonN2","colon normal","colon normal",{}]},{"entry":["70_stomN","stomach normal","stomach normal",{}]},{"entry":["71_liverN","liver normal","liver normal",{}]},{"entry":["72_lungN1","lung normal","lung normal",{}]},{"entry":["73_lungN2","lung normal","lung normal",{}]},{"entry":["74_small intestineN","small intestine","small intestine",{}]},{"entry":["75_brainN","brain normal mix","brain normal mix",{}]},{"entry":["76_heartN","heart normal mix","heart normal mix"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0736","num":"0735"},"tables":{"@attributes":{"id":"TABLE-US-00008","num":"00008"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"287pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Tissue samples in normal panel:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Sample id (GCI)\/case ","Tissue id","Sample id"]},{"entry":[{},{},"id (Asterand)","(GCI)\/Specimen","(Asterand)\/RNA id"]},{"entry":["sample name","Source","Lot no.","id (Asternd)","(GCI)"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":["1-(7)-Bc-Rectum","Biochain","A610297",{},{}]},{"entry":["2-(8)-Bc-Rectum","Biochain","A610298",{},{}]},{"entry":["3-GC-Colon","GCI","CDSUV","CDSUVNR3",{}]},{"entry":["4-As-Colon","Asterand","16364","31802","31802B1"]},{"entry":["5-As-Colon","Asterand","22900","74446","74446B1"]},{"entry":["6-GC-Small bowel","GCI","V9L7D","V9L7DN6Z",{}]},{"entry":["7-GC-Small bowel","GCI","M3GVT","M3GVTN5R",{}]},{"entry":["8-GC-Small bowel","GCI","196S2","196S2AJN",{}]},{"entry":["9-(9)-Am-Stomach","Ambion","110P04A",{},{}]},{"entry":["10-(10)-Bc-Stomach","Biochain","A501159",{},{}]},{"entry":["11-(11)-Bc-Esoph","Biochain","A603814",{},{}]},{"entry":["12-(12)-Bc-Esoph","Biochain","A603813",{},{}]},{"entry":["13-As-Panc","Asterand","8918","9442","9442C1"]},{"entry":["14-As-Panc","Asterand","10082","11134","11134B1"]},{"entry":["16-As-Liver","Asterand","7916","7203","7203B1"]},{"entry":["17-(28)-Am-Bladder","Ambion","071P02C",{},{}]},{"entry":["18-(29)-Bc-Bladder","Biochain","A504088",{},{}]},{"entry":["19-(64)-Am-Kidney","Ambion","111P0101B",{},{}]},{"entry":["20-(65)-Cl-Kidney","Clontech","1110970",{},{}]},{"entry":["21-(66)-Bc-Kidney","Biochain","A411080",{},{}]},{"entry":["22-GC-Kidney","GCI","N1EVZ","N1EVZN91",{}]},{"entry":["23-GC-Kidney","GCI","BMI6W","BMI6WN9F",{}]},{"entry":["25-(43)-Bc-Adrenal","Biochain","A610374",{},{}]},{"entry":["26-(16)-Am-Lung","Ambion","111P0103A",{},{}]},{"entry":["28-As-Lung","Asterand","9078","9275","9275B1"]},{"entry":["29-As-Lung","Asterand","6692","6161","6161A1"]},{"entry":["30-As-Lung","Asterand","7900","7180","7180F1"]},{"entry":["31-(75)-GC-Ovary","GCI","L629FRV1",{},{}]},{"entry":["32-(76)-GC-Ovary","GCI","DWHTZRQX",{},{}]},{"entry":["33-(77)-GC-Ovary","GCI","FDPL9NJ6",{},{}]},{"entry":["34-(78)-GC-Ovary","GCI","GWXUZN5M",{},{}]},{"entry":["36-GC-cervix","GCI","E2P2N","E2P2NAP4",{}]},{"entry":["38-(26)-Bc-Uterus","Biochain","A504090",{},{}]},{"entry":["39-(30)-Am-Placen","Ambion","021P33A",{},{}]},{"entry":["40-(32)-Bc-Placen","Biochain","A411073",{},{}]},{"entry":["41-GC-Breast","GCI","DHLR1",{},{}]},{"entry":["42-GC-Breast","GCI","TG6J6",{},{}]},{"entry":["43-GC-Breast","GCI","E6UDD","E6UDDNCF",{}]},{"entry":["44-(38)-Am-Prostate","Ambion","25955",{},{}]},{"entry":["45-Bc-Prostate","Biochain","A609258",{},{}]},{"entry":["46-As-Testis","Asterand","13071","19567","19567B1"]},{"entry":["47-As-Testis","Asterand","19671","42120","42120A1"]},{"entry":["49-GC-Artery","GCI","YGTVY","YGTVYAIN",{}]},{"entry":["50-TH-Blood-PBMC","Tel-","52497",{},{}]},{"entry":[{},"Hashomer",{},{},{}]},{"entry":["51-TH-Blood-PBMC","Tel-","31055",{},{}]},{"entry":[{},"Hashomer",{},{},{}]},{"entry":["52-TH-Blood-PBMC","Tel-","31058",{},{}]},{"entry":[{},"Hashomer",{},{},{}]},{"entry":["53-(54)-Ic-Spleen","Ichilov","CG-267",{},{}]},{"entry":["54-(55)-Am-Spleen","Ambion","111P0106B","54\u2014(55)-Am-Spleen","Ambion"]},{"entry":["56-(58)-Am-Thymus","Ambion","101P0101A",{},{}]},{"entry":["57-(60)-Bc-Thyroid","Biochain","A610287",{},{}]},{"entry":["58-(62)-Ic-Thyroid","Ichilov","CG-119-2",{},{}]},{"entry":["59-Gc-Sali gland","GCI","NNSMV","NNSMVNJC",{}]},{"entry":["60-(67)-Ic-Cerebellum","Ichilov","CG-183-5",{},{}]},{"entry":["61-(68)-Ic-Cerebellum","Ichilov","CG-212-5",{},{}]},{"entry":["62-(69)-Bc-Brain","Biochain","A411322",{},{}]},{"entry":["63-(71)-Bc-Brain","Biochain","A411079",{},{}]},{"entry":["64-(72)-Ic-Brain","Ichilov","CG-151-1",{},{}]},{"entry":["65-(44)-Bc-Heart","Biochain","A411077",{},{}]},{"entry":["66-(46)-Ic-Heart","Ichilov","CG-227-1",{},{}]},{"entry":["67-(45)-Ic-Heart","Ichilov","CG-255-9",{},{}]},{"entry":["(Fibrotic)",{},{},{},{}]},{"entry":["68-GC-Skel Mus","GCI","T8YZS","T8YZSN7O",{}]},{"entry":["69-GC-Skel Mus","GCI","Q3WKA","Q3WKANCJ",{}]},{"entry":["70-As-Skel Mus","Asterand","8774","8235","8235G1"]},{"entry":["71-As-Skel Mus","Asterand","8775","8244","8244A1"]},{"entry":["72-As-Skel Mus","Asterand","10937","12648","12648C1"]},{"entry":["73-As-Skel Mus","Asterand","6692","6166","6166A1"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0737","num":"0736"},"tables":{"@attributes":{"id":"TABLE-US-00009","num":"00009"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"392pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 3"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Lung cancer testing panel"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"11"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"8","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"9","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"10","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"11","colwidth":"35pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},"sample id",{},{},{},{},{},{},{}]},{"entry":[{},{},{},"(GCI)\/case id","TISSUE ID","RNA ID",{},{},{},{},{}]},{"entry":[{},{},{},"(Asterand)\/","(GCI)\/","(GCI)\/",{},{},{},{},{}]},{"entry":[{},"Source\/","sample","lot no.","specimen ID","Sample ID",{},"Diag","Specimen",{},{}]},{"entry":["Tissue","Delivery","name","(old samples)","(Asterand)","(Asterand)","Diag","remarks","location","Gr","TNM"]},{"entry":{"@attributes":{"namest":"1","nameend":"11","align":"center","rowsep":"1"}}},{"entry":["LC","GCI","1-GC-","7Z9V4","7Z9V4AYM",{},"Aden","BC",{},{},{}]},{"entry":[{},{},"BAC-SIA",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","2-GC-","ZW2AQ","ZW2AQARP",{},"Aden","BC",{},{},{}]},{"entry":[{},{},"BAC-SIB",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","72-(44)-","A501123",{},{},"AC",{},{},"2","UN"]},{"entry":[{},{},"Bc-BAC",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","4-GC-","3MOPL","3MOPLA79",{},"Aden",{},{},{},{}]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIA",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","5-GC-","KOJXD","KOJXDAV4",{},"Aden",{},{},{},{}]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIA",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","6-GC-","X2Q44","X2Q44A79",{},"Aden",{},{},{},{}]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIA",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","7-GC-","6BACZ","6BACZAP5",{},"Aden",{},{},{},{}]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIA",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","8-GC-","BS9AF","BS9AFA3E",{},"Aden",{},{},{},{}]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIA",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","9-GC-","UCLOA","UCLOAA9L",{},"Aden",{},{},{},{}]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIA",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","10-GC-","BVYK3","BVYK3A7Z",{},"Aden",{},{},{},{}]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIA",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","11-GC-","U4DM4","U4DM4AFZ",{},"Aden",{},{},{},{}]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIB",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","12-GC-","OWX5Y","OWX5YA3S",{},"Aden",{},{},{},{}]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIB",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","13-GC-","XYY96","XYY96A6B",{},"Aden",{},{},{},{}]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIIA",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","14-GC-","SO7B1","SO7B1AIJ",{},"Aden",{},{},{},{}]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIIA",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","15-GC-","QANSY","QANSYACD",{},"Aden",{},{},{},{}]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIIIA",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","16-(95)-","A610063",{},{},"Aden",{},{},"1","UN"]},{"entry":[{},{},"BC-Adeno",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","17-(89)-","A609077",{},{},"Aden",{},{},"2-3","UN"]},{"entry":[{},{},"Bc-Adeno",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","18-(76)-","A609218",{},{},"Aden",{},{},"3","UN"]},{"entry":[{},{},"Bc-Adeno",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","74-(2)-Bc-","A504118",{},{},"Aden",{},{},"1","UN"]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","75-(77)-","A608301",{},{},"Aden",{},{},"2","UN"]},{"entry":[{},{},"Bc-Adeno",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","76-(75)-","A609217",{},{},"Aden",{},{},"2","UN"]},{"entry":[{},{},"Bc-Adeno",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","78-(13)-","A504116",{},{},"Aden",{},{},"2-3","UN"]},{"entry":[{},{},"ABc-deno",{},{},{},{},{},{},{},{}]},{"entry":["LC","Ichilov","81-(14)-","CG-111",{},{},"Aden",{},{},"UN","UN"]},{"entry":[{},{},"Ic-Adeno",{},{},{},{},{},{},{},{}]},{"entry":["LC","Aster","19-As-Sq-","9220","9418","9418A1","SCC",{},{},"1","TXN0M0"]},{"entry":[{},{},"S0",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","20-GC-Sq-","U2QHS","U2QHSA2N",{},"SCC",{},{},{},{}]},{"entry":[{},{},"SIA",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","21-GC-Sq-","TRQR7","TRQR7ACD",{},"SCC",{},{},{},{}]},{"entry":[{},{},"SIB",{},{},{},{},{},{},{},{}]},{"entry":["LC","Aster","22-As-Sq-","17581","32603","32603B1","SCC",{},{},"3","T2N0M0"]},{"entry":[{},{},"SIB",{},{},{},{},{},{},{},{}]},{"entry":["LC","Aster","23-As-Sq-","18309","41454","41454B1","SCC",{},{},"2","T2N0MX"]},{"entry":[{},{},"SIB",{},{},{},{},{},{},{},{}]},{"entry":["LC","Aster","24-As-Sq-","9217","9415","9415B1","SCC",{},{},"2","T2N0M0"]},{"entry":[{},{},"SIB",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","25-GC-Sq-","RXQ1P","RXQ1PAEA",{},"SCC",{},{},{},{}]},{"entry":[{},{},"SIIB",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","26-GC-Sq-","KB5KH","KB5KHA6X",{},"SCC",{},{},{},{}]},{"entry":[{},{},"SIIB",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","27-GC-Sq-","LAYMB","LAYMBALF",{},"SCC",{},{},{},{}]},{"entry":[{},{},"SIIIA",{},{},{},{},{},{},{},{}]},{"entry":["LC","Ichilov","29-(25)-","CG-204",{},{},"SCC",{},{},"UN","UN"]},{"entry":[{},{},"Ic-Sq",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","30-(19)-","A408175",{},{},"SCC",{},{},"1","UN"]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","31-(78)-","A607125",{},{},"SCC",{},{},"2","UN"]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","32-(16)-","A409091",{},{},"SCC",{},{},"2","UN"]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","33-(80)-","A609163",{},{},"SCC",{},{},"2","UN"]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","34-(18)-","A503387",{},{},"SCC",{},{},"2-3","UN"]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","35-(81)-","A609076",{},{},"SCC",{},{},"3","UN"]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","82-(21)-","A503187",{},{},"SCC",{},{},"2","UN"]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","83-(17)-","A503183",{},{},"SCC",{},{},"2","UN"]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","84-(79)-","A609018",{},{},"SCC",{},{},"3","UN"]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","85-(22)-","A503386",{},{},"SCC",{},{},"UN","UN"]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","86-(20)-","A501121",{},{},"SCC",{},{},"UN","UN"]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","87-(88)-","A609219",{},{},"SCC",{},{},"UN","UN"]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","88-(100)-","A409017",{},{},"SCC",{},{},"UN","UN"]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{}]},{"entry":["LC","Ichilov","89-(24)-","CG-123",{},{},"SCC",{},{},"UN","UN"]},{"entry":[{},{},"Ic-Sq",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","36-GC-","AF8AL","AF8ALAAL",{},"LCC",{},{},{},{}]},{"entry":[{},{},"LCC-SIA",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","37-GC-","O62XU","O62XUA1X",{},"LCC",{},{},{},{}]},{"entry":[{},{},"LCC-SIB",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","38-GC-","OLOIM","OLOIMAS1",{},"LCC",{},{},{},{}]},{"entry":[{},{},"LCC-SIB",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","39-GC-","1ZWSV","1ZWSVAB9",{},"LCC",{},{},{},{}]},{"entry":[{},{},"LCC-SIIB",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","40-GC-","2YHOD","2YHODA1H",{},"LCC","NSCC . . .",{},{},{}]},{"entry":[{},{},"LCC-SIIB",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","41-GC-","38B4D","38B4DAQK",{},"LCC",{},{},{},{}]},{"entry":[{},{},"LCC-SIIB",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","90-(39)-","A504114",{},{},"LCC",{},{},"UN","UN"]},{"entry":[{},{},"Bc-LCC",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","91-(87)-","A609165",{},{},"LCC",{},{},"3","UN"]},{"entry":[{},{},"Bc-LCC",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","92-(38)-","A504113",{},{},"LCC",{},{},"UN","UN"]},{"entry":[{},{},"Bc-LCC",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","93-(82)-","A609170",{},{},"LCNC",{},{},"UN","UN"]},{"entry":[{},{},"Bc-LCC",{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","42-GC-","QPJQL","QPJQLAF6",{},"SMCC","NC",{},"3",{}]},{"entry":[{},{},"SCC-SIB",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","43-(32)-","A501391",{},{},"SMCC",{},{},{},"UN"]},{"entry":[{},{},"Bc-SCC",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","44-(30)-","A501389",{},{},"SMCC",{},{},"3","UN"]},{"entry":[{},{},"Bc-SCC",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","45-(83)-","A609162",{},{},"SMCC",{},{},"UN","UN"]},{"entry":[{},{},"Bc-SCC",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","46-(86)-","A608032",{},{},"SMCC",{},{},"3","UN"]},{"entry":[{},{},"Bc-SCC",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","47-(31)-","A501390",{},{},"SMCC",{},{},{},"UN"]},{"entry":[{},{},"Bc-SCC",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","48-(84)-","A609167",{},{},"SMCC",{},{},"UN","UN"]},{"entry":[{},{},"Bc-SCC",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","49-(85)-","A609169",{},{},"SMCC",{},{},"UN","UN"]},{"entry":[{},{},"Bc-SCC",{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","50-(33)-","A504115",{},{},"SMCC",{},{},{},"UN"]},{"entry":[{},{},"Bc-SCC",{},{},{},{},{},{},{},{}]},{"entry":["LN","Aster","51-As-","9078","9275","9275B1","Norm-L","PS",{},{},{}]},{"entry":[{},{},"N-PS",{},{},{},{},{},{},{},{}]},{"entry":["LN","Aster","52-As-","8757","8100","8100B1","Norm-L","PM","(Right),",{},{}]},{"entry":[{},{},"N-PM",{},{},{},{},{},"Lobe",{},{}]},{"entry":[{},{},{},{},{},{},{},{},"Inferior",{},{}]},{"entry":["LN","Aster","53-As-N-","6692","6161","6161A1","Norm-L","PM",{},{},{}]},{"entry":[{},{},"PM",{},{},{},{},{},{},{},{}]},{"entry":["LN","Aster","54-As-N-","7900","7180","7180F1","Norm-L","PM",{},{},{}]},{"entry":[{},{},"PM",{},{},{},{},{},{},{},{}]},{"entry":["LN","Aster","55-As-N-","8771","8163","8163A1","Norm-L","PM","(Left),",{},{}]},{"entry":[{},{},"PM",{},{},{},{},{},"Lobe",{},{}]},{"entry":[{},{},{},{},{},{},{},{},"Superior",{},{}]},{"entry":["LN","Aster","56-As-N-","13094","19763","19763A1","Norm-L","PM",{},{},{}]},{"entry":[{},{},"PM",{},{},{},{},{},{},{},{}]},{"entry":["LN","Aster","57-As-N-","19174","40654","40654A2","Norm-L","PM",{},{},{}]},{"entry":[{},{},"PM",{},{},{},{},{},{},{},{}]},{"entry":["LN","Aster","58-As-N-","13128","19642","19642A1","Norm-L","PM",{},{},{}]},{"entry":[{},{},"PM",{},{},{},{},{},{},{},{}]},{"entry":["LN","Aster","59-As-N-","14374","20548","20548C1","Norm-L","PM","(Right),",{},{}]},{"entry":[{},{},"PM",{},{},{},{},{},"Lobe",{},{}]},{"entry":[{},{},{},{},{},{},{},{},"Superior",{},{}]},{"entry":["LN","Amb","60-(99)-","36856",{},{},"N-PM","PM",{},{},{}]},{"entry":[{},{},"Am-NPM",{},{},{},{},{},{},{},{}]},{"entry":["LN","Amb","61-(96)-","36853",{},{},"N-PM","PM",{},{},{}]},{"entry":[{},{},"Am-NPM",{},{},{},{},{},{},{},{}]},{"entry":["LN","Amb","62-(97)-","36854",{},{},"N-PM","PM",{},{},{}]},{"entry":[{},{},"Am-NPM",{},{},{},{},{},{},{},{}]},{"entry":["LN","Amb","63-(93)-","111P0103A",{},{},"N-PM","PM-ICH",{},{},{}]},{"entry":[{},{},"Am-NPM",{},{},{},{},{},{},{},{}]},{"entry":["LN","Amb","64-(98)-","36855",{},{},"N-PM","PM",{},{},{}]},{"entry":[{},{},"Am-NPM",{},{},{},{},{},{},{},{}]},{"entry":["LN","Bioch","69-(91)-","A607257",{},{},"N-P2-","PM",{},{},{}]},{"entry":[{},{},"Bc-NPM",{},{},{},"PM",{},{},{},{}]},{"entry":["LN","Bioch","70-(90)-","A608152",{},{},"N-P2","PM",{},{},{}]},{"entry":[{},{},"Bc-NPM",{},{},{},"PM"]},{"entry":{"@attributes":{"namest":"1","nameend":"11","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"13"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"9","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"10","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"11","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"12","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"13","colwidth":"28pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},{},{},{},{},{},{},"# Cig.","# of Y.","# Y.",{}]},{"entry":[{},"Source\/","sample",{},{},{},{},{},"Smoking","Per","Use of","off",{}]},{"entry":["Tissue","Delivery","name","CS","Tum %","Gen","age","Ethnic B","Status","day","Tobacco","Tobacco","SmPY?"]},{"entry":{"@attributes":{"namest":"1","nameend":"13","align":"center","rowsep":"1"}}},{"entry":["LC","GCI","1-GC-","IA","80","F","63","WCAU","Prev U.","20","15","27","N"]},{"entry":[{},{},"BAC-SIA",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","2-GC-","IB","70","F","56","WCAU","Prev U.","15","28","10","Y"]},{"entry":[{},{},"BAC-SIB",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","72-(44)-",{},{},"F","61",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-BAC",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","4-GC-","IA","60","M","68","WCAU","Nev U.","\u2014","\u2014","\u2014","N"]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIA",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","5-GC-","IA","90","F","64","WCAU","Prev U.","15","40","\u20027","Y"]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIA",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","6-GC-","IA","85","M","58","WCAU","Prev U.","10","47","\u20020","Y"]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIA",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","7-GC-","IA","60","F","65","WCAU","Curr U.","\u20026","30","\u2014","Y"]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIA",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","8-GC-","IA","55","F","59","WCAU","Curr U.","20","40","\u2014","N"]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIA",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","9-GC-","IA","80","F","69","WCAU","Curr U.","30","52","\u2014","Y"]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIA",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","10-GC-","IA","60","F","60","WCAU","Curr U.","40","40","\u2014","N"]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIA",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","11-GC-","IB","65","F","68","WCAU","Prev U.","\u20025","\u20024","43","N"]},{"entry":[{},{},"Adeno-SIB",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","12-GC-","IB","90","M","69","WCAU","Curr U.","10","\u2014","\u2014",{}]},{"entry":[{},{},"Adeno-SIB",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","13-GC-","IIA","70","F","62","WCAU","Prev U.","\u20026","40","\u20026","N"]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIIA",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","14-GC-","IIA","70","M","56","WCAU","Curr U.","30","25","\u2014","Y"]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIIA",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","15-GC-","IIIA","65","F","61","WCAU","Curr U.","30","36","\u2014","Y"]},{"entry":[{},{},"Adeno-",{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SIIIA",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","16-(95)-",{},{},"F","54",{},{},{},{},{},{}]},{"entry":[{},{},"BC-Adeno",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","17-(89)-",{},{},"M","62",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-Adeno",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","18-(76)-",{},{},"M","57",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-Adeno",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","74-(2)-Bc-",{},{},"M","64",{},{},{},{},{},{}]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","75-(77)-",{},{},"M","44",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-Adeno",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","76-(75)-",{},{},"M","65",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-Adeno",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","78-(13)-",{},{},"M","64",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-Adeno",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Ichilov","81-(14)-Ic-",{},{},"M","68",{},{},{},{},{},{}]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Aster","19-As-Sq-","Occult","80","M","67","CAU","Curr U.","11-20","31-40",{},{}]},{"entry":[{},{},"S0",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","20-GC-Sq-","IA","55","F","68","WCAU","Prev U.","10","20","\u20020","N"]},{"entry":[{},{},"SIA",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","21-GC-Sq-","IB","75","M","62","WCAU","Prev U.","20","50","\u20020","Y"]},{"entry":[{},{},"SIB",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Aster","22-As-Sq-","IB","90","M","73","CAU","Prev U.",{},{},{},{}]},{"entry":[{},{},"SIB",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Aster","23-As-Sq-","IB","100","M","66","CAU","Prev U.","11-20","45",{},{}]},{"entry":[{},{},"SIB",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Aster","24-As-Sq-","IB","90","M","65","CAU","Curr U.","\u20026-10","41-50",{},{}]},{"entry":[{},{},"SIB",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","25-GC-Sq-","IIB","55","F","44","WCAU","Prev U.","20","20","\u20020","Y"]},{"entry":[{},{},"SIIB",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","26-GC-Sq-","IIB","65","M","68","WCAU","Prev U.","40","40","\u20020","Y"]},{"entry":[{},{},"SIIB",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","27-GC-Sq-","IIIA","65","F","58","WCAU","Prev U.","50","40","\u20021","Y"]},{"entry":[{},{},"SIIIA",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Ichilov","29-(25)-",{},{},"M","72",{},{},{},{},{},{}]},{"entry":[{},{},"Ic-Sq",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","30-(19)-",{},{},"M","78",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","31-(78)-",{},{},"M","62",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","32-(16)-",{},{},"F","68",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","33-(80)-",{},{},"M","74",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","34-(18)-",{},{},"M","63",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","35-(81)-",{},{},"M","53",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","82-(21)-",{},{},"M","52",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","83-(17)-",{},{},"M","57",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","84-(79)-",{},{},"M","67",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","85-(22)-",{},{},"M","48",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","86-(20)-",{},{},"M","64",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","87-(88)-",{},{},"M","64",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","88-(100)-",{},{},"M","64",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-Sq",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Ichilov","89-(24)-",{},{},"M","76",{},{},{},{},{},{}]},{"entry":[{},{},"Ic-Sq",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","36-GC-","IA","85","M","45","WCAU","Prev U.","45","33","\u20020","Y"]},{"entry":[{},{},"LCC-SIA",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","37-GC-","IB","75","F","60","WCAU","Prev U.","30","45","\u20020","Y"]},{"entry":[{},{},"LCC-SIB",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","38-GC-","IB","70","M","68","WCAU","Prev U.","\u2014","55","\u2014","Y"]},{"entry":[{},{},"LCC-SIB",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","39-GC-","IIB","50","M","51","WCAU","Prev U.","20","12","22","Y"]},{"entry":[{},{},"LCC-SIIB",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","40-GC-","IIB","95","M","62","WCAU","Prev U.","40","40","\u20020","Y"]},{"entry":[{},{},"LCC-SIIB",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","41-GC-","IIB","90","F","70","WCAU","Prev U.","30","50","\u2014","Y"]},{"entry":[{},{},"LCC-SIIB",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","90-(39)-",{},{},"F","35",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-LCC",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","91-(87)-",{},{},"F","47",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-LCC",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","92-(38)-",{},{},"M","58",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-LCC",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","93-(82)-",{},{},"M","68",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-LCC",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","GCI","42-GC-","IB","65","F","62","WCAU","Prev U.","20","35","\u2003\u2002\u20090.15","Y"]},{"entry":[{},{},"SCC-SIB",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","43-(32)-",{},{},"M","30",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-SCC",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","44-(30)-",{},{},"M","34",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-SCC",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","45-(83)-",{},{},"F","47",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-SCC",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","46-(86)-",{},{},"F","52",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-SCC",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","47-(31)-",{},{},"F","59",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-SCC",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","48-(84)-",{},{},"F","59",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-SCC",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","49-(85)-",{},{},"M","66",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-SCC",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LC","Bioch","50-(33)-",{},{},"M",{},{},{},{},{},{},{}]},{"entry":[{},{},"Bc-SCC",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LN","Aster","51-As-",{},{},"M","22","CAU","Nev U.",{},{},{},{}]},{"entry":[{},{},"N-PS",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LN","Aster","52-As-N-",{},{},"F","26","CAU","Nev U.",{},{},{},{}]},{"entry":[{},{},"PM",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LN","Aster","53-As-N-",{},{},"M","37","CAU","Nev U.",{},{},{},{}]},{"entry":[{},{},"PM",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LN","Aster","54-As-N-",{},{},"F","76","CAU","Prev U.",{},{},{},{}]},{"entry":[{},{},"PM",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LN","Aster","55-As-",{},{},"M","81","CAU","Prev U.","41 or","31-40",{},{}]},{"entry":[{},{},"N-PM",{},{},{},{},{},{},"more",{},{},{}]},{"entry":["LN","Aster","56-As-N-",{},{},"M","0","CAU","Prev U.","21-40","41-50",{},{}]},{"entry":[{},{},"PM",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LN","Aster","57-As-N-",{},{},"F","69","CAU","Curr U.","21-40","31-40",{},{}]},{"entry":[{},{},"PM",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LN","Aster","58-As-N-",{},{},"F","75","CAU",{},{},{},{},{}]},{"entry":[{},{},"PM",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LN","Aster","59-As-N-",{},{},"F","75","CAU",{},{},{},{},{}]},{"entry":[{},{},"PM",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LN","Amb","60-(99)-",{},{},"M","31",{},{},{},{},{},{}]},{"entry":[{},{},"Am-NPM",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LN","Amb","61-(96)-",{},{},"F","43",{},{},{},{},{},{}]},{"entry":[{},{},"Am-NPM",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LN","Amb","62-(97)-",{},{},"M","46",{},{},{},{},{},{}]},{"entry":[{},{},"Am-NPM",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LN","Amb","63-(93)-",{},{},"F","61",{},{},{},{},{},{}]},{"entry":[{},{},"Am-NPM",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LN","Amb","64-(98)-",{},{},"F","72",{},{},{},{},{},{}]},{"entry":[{},{},"Am-NPM",{},{},{},{},{},{},{},{},{},{}]},{"entry":["LN","Bioch","69-(91)-",{},{},"P2","24,",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-NPM",{},{},{},"29",{},{},{},{},{},{}]},{"entry":["LN","Bioch","70-(90)-",{},{},"P2","27,",{},{},{},{},{},{}]},{"entry":[{},{},"Bc-NPM",{},{},{},"28"]},{"entry":{"@attributes":{"namest":"1","nameend":"13","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"10"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"9","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"10","colwidth":"21pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Source\/","sample",{},{},"#","Recovery","Cause","Exc."]},{"entry":[{},"Tissue","Delivery","name","Smppl","DrAl","Dr","Type","of Death","Y."]},{"entry":{"@attributes":{"namest":"1","nameend":"10","align":"center","rowsep":"1"}}},{"entry":[{},"LC","GCI","1-GC-BAC-SIA","\u2014","Y","\u20020","Surg",{},"2001"]},{"entry":[{},"LC","GCI","2-GC-BAC-SIB","1","Y","\u20026","Surg",{},"2002"]},{"entry":[{},"LC","Bioch","72-(44)-Bc-BAC",{},{},{},{},{},{}]},{"entry":[{},"LC","GCI","4-GC-Adeno-SIA","\u2014","N","\u2014","Surg",{},"2001"]},{"entry":[{},"LC","GCI","5-GC-Adeno-SIA","1","N","\u20020","Surg",{},"2003"]},{"entry":[{},"LC","GCI","6-GC-Adeno-SIA","2","N","\u2014","Surg",{},"2004"]},{"entry":[{},"LC","GCI","7-GC-Adeno-SIA","1","N","\u2014","Surg",{},"2004"]},{"entry":[{},"LC","GCI","8-GC-Adeno-SIA","\u2014","N","\u2014","Surg",{},"2004"]},{"entry":[{},"LC","GCI","9-GC-Adeno-SIA","4","N","\u2014","Surg",{},"2005"]},{"entry":[{},"LC","GCI","10-GC-Adeno-SIA","\u2014","N","\u2014","Surg",{},"2002"]},{"entry":[{},"LC","GCI","11-GC-Adeno-SIB","\u2014","N","\u2014","Surg",{},"2003"]},{"entry":[{},"LC","GCI","12-GC-Adeno-SIB","\u2014","N","\u2014","Surg",{},"2002"]},{"entry":[{},"LC","GCI","13-GC-Adeno-SIIA","\u2014","Y","0","Surg",{},"2004"]},{"entry":[{},"LC","GCI","14-GC-Adeno-SIIA","1","N","\u2014","Surg",{},"2001"]},{"entry":[{},"LC","GCI","15-GC-Adeno-SIIIA","1","N","\u2014","Surg",{},"2004"]},{"entry":[{},"LC","Bioch","16-(95)-BC-Adeno",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","17-(89)-Bc-Adeno",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","18-(76)-Bc-Adeno",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","74-(2)-Bc-Adeno",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","75-(77)-Bc-Adeno",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","76-(75)-Bc-Adeno",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","78-(13)-Bc-Adeno",{},{},{},{},{},{}]},{"entry":[{},"LC","Ichilov","81-(14)-Ic-Adeno",{},{},{},{},{},{}]},{"entry":[{},"LC","Aster","19-As-Sq-S0",{},"O",{},"Surg",{},"2003"]},{"entry":[{},"LC","GCI","20-GC-Sq-SIA","\u2014","N","\u2014","Surg",{},"2004"]},{"entry":[{},"LC","GCI","21-GC-Sq-SIB","5","N","\u2014","Surg",{},"2005"]},{"entry":[{},"LC","Aster","22-As-Sq-SIB",{},"O",{},"Surg",{},"2004"]},{"entry":[{},"LC","Aster","23-As-Sq-SIB",{},"P",{},"Surg",{},"2005"]},{"entry":[{},"LC","Aster","24-As-Sq-SIB",{},"O",{},"Surg",{},"2002"]},{"entry":[{},"LC","GCI","25-GC-Sq-SIIB","2","N","\u2014","Surg",{},"2004"]},{"entry":[{},"LC","GCI","26-GC-Sq-SIIB","2","N","\u2014","Surg",{},"2004"]},{"entry":[{},"LC","GCI","27-GC-Sq-SIIIA","2","N","\u2014","Surg",{},"2004"]},{"entry":[{},"LC","Ichilov","29-(25)-Ic-Sq",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","30-(19)-Bc-Sq",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","31-(78)-Bc-Sq",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","32-(16)-Bc-Sq",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","33-(80)-Bc-Sq",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","34-(18)-Bc-Sq",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","35-(81)-Bc-Sq",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","82-(21)-Bc-Sq",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","83-(17)-Bc-Sq",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","84-(79)-Bc-Sq",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","85-(22)-Bc-Sq",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","86-(20)-Bc-Sq",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","87-(88)-Bc-Sq",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","88-(100)-Bc-Sq",{},{},{},{},{},{}]},{"entry":[{},"LC","Ichilov","89-(24)-Ic-Sq",{},{},{},{},{},{}]},{"entry":[{},"LC","GCI","36-GC-LCC-SIA","2","Y","28","Surg",{},"2004"]},{"entry":[{},"LC","GCI","37-GC-LCC-SIB","3","N","\u2014","Surg",{},"2004"]},{"entry":[{},"LC","GCI","38-GC-LCC-SIB","\u2014","N","\u2014","Surg",{},"2001"]},{"entry":[{},"LC","GCI","39-GC-LCC-SIIB","1","N","\u2014","Surg",{},"2004"]},{"entry":[{},"LC","GCI","40-GC-LCC-SIIB","2","Y","12","Surg",{},"2004"]},{"entry":[{},"LC","GCI","41-GC-LCC-SIIB","2","Y","13","Surg",{},"2002"]},{"entry":[{},"LC","Bioch","90-(39)-Bc-LCC",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","91-(87)-Bc-LCC",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","92-(38)-Bc-LCC",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","93-(82)-Bc-LCC",{},{},{},{},{},{}]},{"entry":[{},"LC","GCI","42-GC-SCC-SIB","2","N","\u2014","Surg",{},"2003"]},{"entry":[{},"LC","Bioch","43-(32)-Bc-SCC",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","44-(30)-Bc-SCC",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","45-(83)-Bc-SCC",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","46-(86)-Bc-SCC",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","47-(31)-Bc-SCC",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","48-(84)-Bc-SCC",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","49-(85)-Bc-SCC",{},{},{},{},{},{}]},{"entry":[{},"LC","Bioch","50-(33)-Bc-SCC",{},{},{},{},{},{}]},{"entry":[{},"LN","Aster","51-As-N-PS",{},"NU",{},"Surg",{},"2003"]},{"entry":[{},"LN","Aster","52-As-N-PM",{},"O",{},"Aut","CA","2003"]},{"entry":[{},"LN","Aster","53-As-N-PM",{},"C",{},"Aut","MCE","2002"]},{"entry":[{},"LN","Aster","54-As-N-PM",{},{},{},"Aut","CPulA","2002"]},{"entry":[{},"LN","Aster","55-As-N-PM",{},"O",{},"Aut","CA","2003"]},{"entry":[{},"LN","Aster","56-As-N-PM",{},"P",{},"Aut","IC",{}]},{"entry":[{},"LN","Aster","57-As-N-PM",{},"P",{},"Aut","CPulA","2005"]},{"entry":[{},"LN","Aster","58-As-N-PM",{},{},{},"Aut","CPulA","2004"]},{"entry":[{},"LN","Aster","59-As-N-PM",{},{},{},"Aut","CerA","2004"]},{"entry":[{},"LN","Amb","60-(99)-Am-NPM",{},{},{},{},{},{}]},{"entry":[{},"LN","Amb","61-(96)-Am-NPM",{},{},{},{},{},{}]},{"entry":[{},"LN","Amb","62-(97)-Am-NPM",{},{},{},{},{},{}]},{"entry":[{},"LN","Amb","63-(93)-Am-NPM",{},{},{},{},{},{}]},{"entry":[{},"LN","Amb","64-(98)-Am-NPM",{},{},{},{},{},{}]},{"entry":[{},"LN","Bioch","69-(91)-Bc-NPM",{},{},{},{},{},{}]},{"entry":[{},"LN","Bioch","70-(90)-Bc-NPM"]},{"entry":{"@attributes":{"namest":"1","nameend":"10","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0738","num":"0737"},"tables":{"@attributes":{"id":"TABLE-US-00010","num":"00010"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"105pt","align":"left"}}],"thead":{"row":[{"entry":[{},"TABLE 3_1"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"Key","Full Name"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"# Cig. Per day","Number of Cigarettes per"]},{"entry":[{},{},"day"]},{"entry":[{},"# Dr","Number of Drinks"]},{"entry":[{},"# of Y. Use of","Number of Years Using"]},{"entry":[{},"Tobacco","Tobacco"]},{"entry":[{},"# Y. off Tobacco","Number of Years Off"]},{"entry":[{},{},"Tobacco"]},{"entry":[{},"AC","Alveolus carcinoma"]},{"entry":[{},"Aden","ADENOCARCINOMA"]},{"entry":[{},"Amb","Ambion"]},{"entry":[{},"Aster","Asterand"]},{"entry":[{},"Aut","Autopsy"]},{"entry":[{},"BC","BRONCHIOLOALVEOLAR"]},{"entry":[{},{},"CARCINOMA"]},{"entry":[{},"Bioch","Biochain"]},{"entry":[{},"C","Current Use"]},{"entry":[{},"CA","Cardiac arrest"]},{"entry":[{},"CAU","Caucasian"]},{"entry":[{},"Cer A","Cerebrovascular accident"]},{"entry":[{},"CPul A","Cardiopulmonary arrest"]},{"entry":[{},"CS","Cancer Stage"]},{"entry":[{},"Curr U.","Current Use"]},{"entry":[{},"Diag","Diagnosis"]},{"entry":[{},"Dr Al","Drink Alcohol?"]},{"entry":[{},"Exc Y.","Excision Year"]},{"entry":[{},"Gen","Gender"]},{"entry":[{},"Gr","Grade"]},{"entry":[{},"Height","HT"]},{"entry":[{},"IC","Ischemic cardiomyopathy"]},{"entry":[{},"LC","Lung Cancer"]},{"entry":[{},"LCC","LARGE CELL"]},{"entry":[{},{},"CARCINOMA"]},{"entry":[{},"LCNC","Large Cell Neuroendocrine"]},{"entry":[{},{},"Carcinoma"]},{"entry":[{},"LN","Lung Normal"]},{"entry":[{},"MCE","Massive cerebral edema"]},{"entry":[{},"N","No"]},{"entry":[{},"NC","NEUROENDOCRINE"]},{"entry":[{},{},"CARCINOMA"]},{"entry":[{},"Nev. U.","Never Used"]},{"entry":[{},"Norm-L","Normal Lung"]},{"entry":[{},"N-P2-PM","Normal (Pool 2)- PM"]},{"entry":[{},"N-PM","Normal-PM"]},{"entry":[{},"NSCC . . .","NON-SMALL CELL"]},{"entry":[{},{},"CARCINOMA WITH"]},{"entry":[{},{},"SARCOMUTOUS"]},{"entry":[{},{},"TRANSFORMTAIO"]},{"entry":[{},"NU","Never used"]},{"entry":[{},"O","Occasional Use"]},{"entry":[{},"P","Previous Use"]},{"entry":[{},"P2","Pool 2"]},{"entry":[{},"Prev U.","Previous Use"]},{"entry":[{},"SCC","Squamous Cell Carcinoma"]},{"entry":[{},"Sm P Y?","Have people at home"]},{"entry":[{},{},"smoked in past 15 yr"]},{"entry":[{},"Sm ppl","If yes, how many?"]},{"entry":[{},"SMCC","SMALL CELL"]},{"entry":[{},{},"CARCINOMA"]},{"entry":[{},"SMOKE_GROWING_UP","Did people smoke at home"]},{"entry":[{},{},"while growing up"]},{"entry":[{},"Surg","Surgical"]},{"entry":[{},"Tum %","Tumor Percentage"]},{"entry":[{},"WCAU","White Caucasian"]},{"entry":[{},"Y","Yes"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}}}},{"@attributes":{"id":"p-0739","num":"0738"},"tables":{"@attributes":{"id":"TABLE-US-00011","num":"00011"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","orient":"land"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"637pt","align":"center"}},"thead":{"row":{"entry":"TABLE 4"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Tissue samples in ovary panel"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"21"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"8","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"10","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"11","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"12","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"13","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"14","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"15","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"16","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"17","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"18","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"19","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"20","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"21","colwidth":"35pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},"sample_id",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},{},"(GCI)\/case id","RNA ID",{},{},{},{},{},{},{},{},{},{},"Age",{},{},{},{},{}]},{"entry":[{},{},{},"(Asterand)\/","(GCI)\/",{},{},{},{},{},{},"Meno-",{},{},{},"at",{},"Oral","Oral",{},{}]},{"entry":[{},"Source\/","sample","lot no.","Sample ID",{},"C","Tumor",{},"Ethnic",{},"pausal","Mens","Preg","Preg","first",{},"Con","Con","Tubal","Recovery"]},{"entry":["Tissue","Delivery","name","(old samples)","(Asterand)","Diag","Stage","%","age","BG","CA125PRE","Status","Age","Times","Toterm","child","OC","Length","Unit","ligation","Type"]},{"entry":{"@attributes":{"namest":"1","nameend":"21","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"21"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"8","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"9","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"10","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"11","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"12","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"13","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"14","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"15","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"16","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"17","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"18","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"19","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"20","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"21","colwidth":"35pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["OVC","Asterand","1-As-SerSI","23074","71900A2","SA","I","80","49","CAU",{},"Pre-M",{},"2","1",{},{},{},{},{},"Surg"]},{"entry":["OVC","Asterand","2-As-SerSI","22653","70270A1","SA","I","90","69","WCAU",{},"Post-M",{},"1","1",{},{},{},{},{},"Surg"]},{"entry":["OVC","Asterand","3-As-SerSIB","18700","40771B1","SA","IB","100","62","WCAU",{},"Post-M",{},"3","3",{},{},{},{},{},"Surg"]},{"entry":["OVC","GOG","79-(32)-GO-","93-09-4901",{},"SPC","1B",{},"67",{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SerSIB",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","Asterand","4-As-SerSIB","17646","32667B1","SA","IB","100","68","W",{},"Post-M",{},"9","2",{},{},{},{},{},"Surg"]},{"entry":["OVC","Asterand","5-As-SerSIC","15644","22996A1","SA","IC","100","48","CAU",{},"M",{},"4","2",{},{},{},{},{},"Surg"]},{"entry":["OVC","Asterand","6-As-SerSIIA","18701","40773C1","SA","IIA","100","59","CAU",{},"Post-M",{},"1","1",{},{},{},{},{},"Surg"]},{"entry":["OVC","GCI","7-GC-SerSIIB","2O37O",{},"SA","IIB","75","43","WCAU","\u2014","Pre-M","12","0","0","\u20020","NO","\u2014",{},"NO","Surg"]},{"entry":["OVC","GCI","8-GC-SerSIIB","7B3DP",{},"SA","IIB","70","70","WCAU","\u2014","Post-M","14","5","3","20","YES","6","months","NO","Surg"]},{"entry":["OVC","GOG","80-(30)-GO-","2001-08-",{},"PSC","3A",{},"72",{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SerSIIIA","G011",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GOG","81-(70)-GO-","95-08-",{},"PSA","3B",{},"50",{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SerSIIIB","G069",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GOG","82-(5)-GO-","99-12-",{},"A","3C",{},"46",{},">500",{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SerSIIIC","G432",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","Asterand","9-As-SerSIIIC","13268","19832A1","SA","IIIC","90","48","C",{},"Post-M",{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC","GOG","83-(29)-GO-","2001-12-",{},"SA","3C",{},"50",{},"260",{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SerSIIIC","G035",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GCI","10-GC-SerSIIIC","3NTIS",{},"SA","IIIC","70","53","WCAU","70","Post-M","12","1","1","26","YES","3","months","NO","Surg"]},{"entry":["OVC","GCI","11-GC-SerSIIIC","CEJUS",{},"SA","IIIC","70","53","WCAU","4814","Pre-M","\u2014","2","2","30","NO","\u2014",{},"NO","Surg"]},{"entry":["OVC","GCI","12-GC-SerSIIIC","5NCLK",{},"SA","IIIC","70","54","WCAU","209","Post-M","13","2","2","21","YES","1","years","NO","Surg"]},{"entry":["OVC","ABS","84-(25)-AB-","N0021",{},"PSA","3C",{},"55","CAU",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SerSIIIC",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GCI","13-GC-SerSIIIC","1HI5H",{},"SA","IIIC","90","61","WCAU","34","Post-M","12","6","3","22","NO","\u2014",{},"NO","Surg"]},{"entry":["OVC","GCI","14-GC-SerSIIIC","7RMHZ",{},"SA","IIIC","80","63","WCAU","\u2014","Post-M","12","2","2","20","YES","10\u2002","years","NO","Surg"]},{"entry":["OVC","GCI","15-GC-SerSIIIC","4WAAB",{},"SA","IIIC","90","63","WCAU","\u2014","Post-M","11","2","1","29","YES","4","years","NO","Surg"]},{"entry":["OVC","GCI","16-GC-SerSIIIC","79Z67",{},"SA","IIIC","85","67","WCAU","\u2014","Post-M","12","6","5","24","YES","2","years","YES","Surg"]},{"entry":["OVC","GOG","85-(13)-GO-","94-05-7603",{},"APP","3C",{},"67",{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SerSIIIC",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GCI","17-GC-SerSIIIC","DDSNL",{},"SA","IIIC","70","68","WCAU","\u2014","Post-M","11","4","4","19","NO","\u2014",{},"NO","Surg"]},{"entry":["OVC","GCI","18-GC-SerSIV","DH8PH",{},"SA","IV","95","70","WCAU","\u2014","Post-M","13","4","3","20","NO","\u2014",{},"NO","Surg"]},{"entry":["OVC","BioChain","86-(33)-BC-Ser","A503175",{},"SPC",{},{},"41","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","BioChain","87-(14)-Bc-Ser","A501111",{},"A",{},{},"41","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","Biochain","88-(12)-Bc-Ser","A406023",{},"A",{},{},"45","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","Biochain","89-(11)-Bc-Ser","A407068",{},"A",{},{},"49","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","ABS","90-(4)-AB-Ser","ILS-7286",{},"PC","UN",{},"50","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","ABS","91-(6)-AB-Ser","A0106",{},"A","UN",{},"51","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","ABS","92-(3)-AB-Ser","ILS-1431",{},"PA","UN",{},"52","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","BioChain","93-(31)-Bc-Ser","A503176",{},"SPC",{},{},"52","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","ABS","94-(2)-AB-Ser","ILS-1408",{},"PA","UN",{},"53","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","ABS","95-(7)-AB-Ser","IND-00375",{},"A",{},{},"59","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","BioChain","96-(8)-Bc-Ser","A501113",{},"A",{},{},"60","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","Biochain","97-(10)-Bc-Ser","A407069",{},"A",{},{},"60","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","ABS","98-(1)-AB-Ser","ILS-1406",{},"PA","UN",{},"73","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GCI","19-GC-","E2WKF",{},"EA","IA","70","30","WCAU","\u2014","Pre-M","12","6","5","17","YES","6","years","NO","Surg"]},{"entry":[{},{},"EndoSIA",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GCI","20-GC-","5895C",{},"EA","IA","95","39","WCAU","\u2014","Pre-M","14","2","2","20","NO","\u2014",{},"NO","Surg"]},{"entry":[{},{},"EndoSIA",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GCI","21-GC-","533DX",{},"EA","IA","95","50","WCAU","190","Pre-M","11","0","\u2014","\u2014","YES","2","years","NO","Surg"]},{"entry":[{},{},"EndoSIA",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GCI","22-GC-","HZ2EY",{},"EA","IA","90","55","WCAU","1078","Pre-M","13","0","\u2014","\u2014","NO","\u2014",{},"NO","Surg"]},{"entry":[{},{},"EndoSIA",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GCI","23-GC-","RWOIV",{},"EA","IA","65","47","WCAU","1695","Pre-M","14","0","\u2014","\u2014","NO","\u2014",{},"NO","Surg"]},{"entry":[{},{},"EndoSIA",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GCI","24-GC-","1U52X",{},"EA","IIA","95","61","WCAU","275",{},"\u2014","\u2014","\u2014","\u2014",{},"\u2014",{},{},"Surg"]},{"entry":[{},{},"EndoSIIA",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GCI","25-GC-","A17WS",{},"EA","IIB","70","67","WCAU","78","Post-M","14","0","\u2014","\u2014","NO","\u2014",{},"NO","Surg"]},{"entry":[{},{},"EndoSIIB",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GCI","26-GC-","1VT3I",{},"EA","IIIC","90","50","WCAU","\u2014","Pre-M","12","2","2","24","YES","1","years","NO","Surg"]},{"entry":[{},{},"EndoSIIIC",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GCI","27-GC-","PZQXH",{},"EA","IIIC","80","52","WCAU","\u2014","Pre-M","11","0","\u2014","\u2014","YES","5","years","NO","Surg"]},{"entry":[{},{},"EndoSIIIC",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GCI","28-GC-","I8VHZ",{},"EA","IV","90","68","WCAU","\u2014","Post-M","\u2014","2","2","27","NO","\u2014",{},"NO","Surg"]},{"entry":[{},{},"EndoSIV",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GOG","99-(41)-GO-","98-03-G803",{},"Mixed . . .","2",{},"38",{},">35",{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SerMixSII",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GOG","100-(40)-GO-","95-11-G006",{},"PS &","3C",{},"49",{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SerMixSIIIC",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GOG","101-(37)-GO-","2002-05-",{},"MS &","3C",{},"56",{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SerMixSIIIC","G513",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GOG","102-(38)-GO-","2002-05-",{},"MS &","3C",{},"64",{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SerMixSIIIC","G509",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GOG","103-(34)-GO-","95-04-2002",{},"PEA","3C",{},"68",{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"SerMixSIIIC",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GOG","29-(21)-GO-","95-10-G020",{},"MC","IA",{},"44",{},{},">100",{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"MucSIA",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GCI","30-GC-","IMDA1",{},"MA","IC","70","41","WCAU","50","Pre-M","12","2","1","24","NO","\u2014",{},{},"Surg"]},{"entry":[{},{},"MucSIC",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","Asterand","31-As-","12742","18920A1","MA","IC","70","61","C",{},"Post-M",{},"3","3",{},{},{},{},{},"Surg"]},{"entry":[{},{},"MucSIC",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","ABS","32-(22)-AB-","A0139",{},"MC","IC",{},"72","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"MucSIC",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GCI","33-GC-","NJM4U",{},"MA","IIA","80","51","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":[{},{},"MucSIIA",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","ABS","34-(20)-AB-","USA-00273",{},"PMC","IIIA",{},"45","C",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"MucSIIIA",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GCI","35-GC-","RAFCW",{},"MA","IIIA","75","55","WCAU","95","Post-M","13","4","3","22","NO","\u2014",{},"NO","Surg"]},{"entry":[{},{},"MucSIIIA",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","Asterand","36-As-","23177","72888A1","MA","IIIC","60","52","C",{},"Pre-M",{},{},{},{},{},{},{},{},"Surg"]},{"entry":[{},{},"MucSIIIC",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","Asterand","37-As-","16103","29374B1","MA","IIIC","100","62","W",{},"Post-M",{},"1","1",{},{},{},{},{},"Surg"]},{"entry":[{},{},"MucSIIIC",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","BioChain","104-(19)-Bc-","A504085",{},"MA",{},{},"34","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"Muc",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","BioChain","105-(18)-Bc-","A504083",{},"MA",{},{},"45","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"Muc",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","BioChain","106-(17)-Bc-","A504084",{},"MA",{},{},"51","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"Muc",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","BioChain","107-(15)-Bc-","A407065",{},"C",{},{},"27","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"Car",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","Clontech","108-(16)-Cl-Car","1090387",{},"CNOS",{},{},"58","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GOG","109-(44)-GO-","2001-07-",{},"CCA","1A",{},"73",{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"ClearcellSIA","G084",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC","GOG","110-(43)-GO-","2001-10-",{},"CCA","3A",{},"74",{},"slightly",{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"ClearcellSIIIA","G002",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC_BT","GCI","38-GC-","SC656",{},"MBT","IA","75","40","WCAU","138","Pre-M","13","2","2","23","NO","\u2014",{},"YES","Surg"]},{"entry":[{},{},"MucBorderSIA",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC_BT","GCI","39-GC-","3D5FO",{},"MBT","IA","85","51","WCAU","19","?","15","0","\u2014","\u2014","NO","\u2014",{},"NO","Surg"]},{"entry":[{},{},"MucBorderSIA",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC_BT","GCI","40-GC-","7JP3F",{},"MBT","IA","75","56","WCAU","125","Post-M","14","3","3","19","YES","5","years","NO","Surg"]},{"entry":[{},{},"MucBorderSIA",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC_BT","ABS","111-(23)-AB-","VNM-00187",{},"MCLowM",{},{},"45","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"Border",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC_BT","GOG","112-(42)-GO-","98-08-G001",{},"EA of BM","1A",{},"46",{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"BorderSIA",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC_B","GOG","41-(62)-Go-","99-10-G442",{},"BMC",{},{},"32",{},"6",{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"BenMuc",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC_B","GCI","43-GC-","QLIKY",{},"BMC",{},"100","42","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":[{},{},"BenMuc",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC_B","Asterand","44-As-","16870","30534A1","BMC",{},"100","45","W",{},"Pre-M",{},"2","2",{},{},{},{},{},"Surg"]},{"entry":[{},{},"BenMuc",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC_B","GOG","45-(56)-GO-","99-01-G407",{},"BMC",{},{},"46",{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"BenMuc",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC_B","GCI","46-GC-","943EC",{},"BMC",{},"75","54","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":[{},{},"BenMuc",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC_B","GCI","47-GC-","JO8W7",{},"BMC",{},"50","56","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":[{},{},"BenMuc",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC_B","Asterand","48-As-BenSer","17016","30645B1","BSC","IA","100","38","C",{},"Pre-M",{},"2","2",{},{},{},{},{},"Surg"]},{"entry":["OVC_B","GOG","49-(64)-GO-","99-06-G039",{},"BSC",{},{},"57",{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"BenSer",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC_B","GCI","50-GC-BenSer","DQQ2F",{},"BSCF",{},"95","68","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_B","Asterand","51-As-BenSer","8786","8275A1","BSC",{},"100","80","CAU",{},"Post-M",{},"10\u2002","9",{},{},{},{},{},"Surg"]},{"entry":["OVC_NBM","Asterand","52-As-NBM","15690","23054A1","NO-BM",{},{},"52","CAU",{},"Pre-M",{},"10\u2002","3",{},{},{},{},{},"Surg"]},{"entry":["OVC_NBM","Asterand","53-As-NBM","16843","30488A1","NO-BM",{},{},"57","W",{},"Post-M",{},"4","2",{},{},{},{},{},"Surg"]},{"entry":["OVC_NBM","Asterand","54-As-NBM","16850","30496B1","NO-BM",{},{},"65","W",{},"Post-M",{},"2","2",{},{},{},{},{},"Surg"]},{"entry":["OVC_NBM","Asterand","55-As-NBM","16848","30499C1","NO-BM",{},{},"66","CAU",{},"Post-M",{},"9","2",{},{},{},{},{},"Surg"]},{"entry":["OVC_N","GCI","56-GC-NPS","WPU1U",{},"NO-PS",{},"0","32","WC",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_N","GCI","57-GC-NPS","Y9VHI",{},"NO-PS",{},"0","35","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_N","GCI","58-GC-NPS","76VM9",{},"NO-PS",{},"0","41","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_N","GCI","59-GC-NPS","DWHTZ",{},"NO-PS",{},"0","42","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_N","GCI","60-GC-NPS","SJ2R2",{},"NO-PS",{},"0","43","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_N","GCI","61-GC-NPS","9RQMN",{},"NO-PS",{},"0","45","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_N","GCI","62-GC-NPS","TOAE5",{},"NO-PS",{},"0","45","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_N","GCI","63-GC-NPS","TW9PM",{},"NO-PS",{},"0","46","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_N","GCI","64-GC-NPS","2VND2",{},"NO-PS",{},"0","46","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_N","GCI","65-GC-NPS","L629F",{},"NO-PS",{},"0","47","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_N","GCI","66-GC-NPS","XLB23",{},"NO-PS",{},"0","47","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_N","GCI","67-GC-NPS","IDUVY",{},"NO-PS",{},"0","47","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_N","GCI","68-GC-NPS","ZCXAD",{},"NO-PS",{},"0","48","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_N","GCI","69-GC-NPS","PEQ6C",{},"NO-PS",{},"0","49","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_N","GCI","70-GC-NPS","DD73B",{},"NO-PS",{},"0","49","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_N","GCI","71-GC-NPS","E2UF7",{},"NO-PS",{},"0","53","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_N","GCI","72-GC-NPS","GWXUZ",{},"NO-PS",{},"0","53","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_N","GCI","73-GC-NPS","4YG5P",{},"NO-PS",{},"0","55","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_N","GCI","74-GC-NPS","FDPL9",{},"NO-PS",{},"0","56","WCAU",{},{},{},{},{},{},{},{},{},{},"Surg"]},{"entry":["OVC_N","BioChain","75-(45)-","A503274",{},"NO-PS",{},{},"41","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"Bc-NPM",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC_N","BioChain","76-(46)-","A504086",{},"NO-PM",{},{},"41","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"Bc-NPM",{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC_N","Ichilov","77-(71)-Ic-NPM","CG-188-7",{},"NO-PM",{},{},"49",{},{},{},{},{},{},{},{},{},{},{},{}]},{"entry":["OVC_N","BioChain","78-(48)-","A504087",{},"NO-PM",{},{},"51","Asian",{},{},{},{},{},{},{},{},{},{},{}]},{"entry":[{},{},"Bc-NPM"]},{"entry":{"@attributes":{"namest":"1","nameend":"21","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0740","num":"0739"},"tables":{"@attributes":{"id":"TABLE-US-00012","num":"00012"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"168pt","align":"left"}}],"thead":{"row":[{"entry":"TABLE 4_1"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["Key","Full Name"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["A","Adenocarcinoma"]},{"entry":["APP","Adenocarcinoma from primary peritoneal"]},{"entry":["BMC","BENIGN MUCINOUS CYSTADENOMA"]},{"entry":["BSC","BENIGN SEROUS CYSTADENOMA"]},{"entry":["BSCF","BENIGN SEROUS CYSTADENOFIBROMA"]},{"entry":["C","Carcinoma"]},{"entry":["C Stage","Cancer stage"]},{"entry":["CAU","Caucasian"]},{"entry":["CCA","Clear cell adenocarcinoma"]},{"entry":["CNOS","Carcinoma NOS"]},{"entry":["EA","ENDOMETROID ADENOCARCINOMA"]},{"entry":["EA of","Endometroid adenocarcinoma of borderline"]},{"entry":["BM","malignancy"]},{"entry":["M","Menopausal"]},{"entry":["MA","MUCINOUS ADENOCARCINOMA"]},{"entry":["MBT","MUCINOUS BORDERLINE TUMOR"]},{"entry":["MC","Mucinous cystadenocarcinoma"]},{"entry":["MC Low","Mucinous cystadenocarcinoma with low malignant"]},{"entry":"M"},{"entry":["Mens.","Menstrual Age"]},{"entry":"Age"},{"entry":["Mixed . . .","Mixed epithelial cystadenocarcinoma with mucinous,"]},{"entry":[{},"endometrioid, squamous and papillary serous"]},{"entry":["MS &","Mixed serous and endometrioid adenocarcinoma"]},{"entry":"EA"},{"entry":["MS &","Mixed serous and endometrioid adenocarcinoma of"]},{"entry":["EAM","mullerian"]},{"entry":["NO-BM","NORMAL OVARY-BM"]},{"entry":["NO-PM","NORMAL OVARY-PM"]},{"entry":["NO-PS","NORMAL OVARY-PS"]},{"entry":["OC","Oral Contraceptive"]},{"entry":["OVC","Ovary Cancer"]},{"entry":["OVC_B","Ovary Benign"]},{"entry":["OVC_BT","Ovary Borderline Tumor"]},{"entry":["OVC_N","Ovary Normal"]},{"entry":["OVC_NBM","Ovary normal-benign matched"]},{"entry":["PA","Papillary adenocarcinoma"]},{"entry":["PC","Papillary cystadenocarcinoma"]},{"entry":["PEA","Papillary endometrioid adenocarcinoma"]},{"entry":["PMC","Papillary mucinous cystadenocarcinoma"]},{"entry":["Post-M","Post-menopausal"]},{"entry":["Pre-M","Pre-menopausal"]},{"entry":["PS & EC","Papillary serous and endometrioid"]},{"entry":[{},"cystadenocarcinoma"]},{"entry":["PSA","Papillary serous adenocarcinoma"]},{"entry":["PSC","Papillary serous carcinoma"]},{"entry":["SA","SEROUS ADENOCARCINOMA"]},{"entry":["SPC","Serous papillary cystadenocarcinoma"]},{"entry":["W","White"]},{"entry":["WCAU","WHITE\/CAUCASIAN"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}}}},{"@attributes":{"id":"p-0741","num":"0740"},"tables":{"@attributes":{"id":"TABLE-US-00013","num":"00013"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"441pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 5"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Colon cancer testing panel"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"10"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"8","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"9","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"10","colwidth":"21pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},"sample_id",{},{},{},{},{},{}]},{"entry":[{},{},{},"(GCI)\/case ",{},{},{},{},{},{}]},{"entry":[{},{},{},"id","TISSUE ID",{},{},{},{},{}]},{"entry":[{},{},{},"(Asterand)\/","(GCI)\/","Sample",{},{},"Speci-",{}]},{"entry":[{},"Source\/",{},"lot no.","specimen ID","ID",{},"Diag","men",{}]},{"entry":["Tissue","Delivery","sample name","(old samples","(Asterand)","(Asterand)","Diag","remarks","location","Gr"]},{"entry":{"@attributes":{"namest":"1","nameend":"10","align":"center","rowsep":"1"}}},{"entry":["CC","Asterand","1-As-AdenS0","18036","31312","31312B1","Aden",{},"Cec","3"]},{"entry":["CC","GCI","2-GC-","4QDH8","4QDH8ADT",{},"Aden",{},"DisC",{}]},{"entry":[{},{},"AdenoSI",{},{},{},{},{},{},{}]},{"entry":["CC","Ichilov","3-(7)-Ic-","CG-235",{},{},"AI",{},"Rectum","UN"]},{"entry":[{},{},"AdenoSI",{},{},{},{},{},{},{}]},{"entry":["CC","GCI","4-GC-","NTAI8","NTAI8AOU",{},"Aden",{},"Cec",{}]},{"entry":[{},{},"AdenoSI",{},{},{},{},{},{},{}]},{"entry":["CC","GCI","5-GC-","ARA7P","ARA7PAQA",{},"Aden",{},"Ret, ",{}]},{"entry":[{},{},"AdenoSI",{},{},{},{},{},"Low ",{}]},{"entry":[{},{},{},{},{},{},{},{},"Ant",{}]},{"entry":["CC","Ichilov","6-(20)-Ic-","CG-249",{},{},"UA",{},{},"3"]},{"entry":[{},{},"AdenoSIIA",{},{},{},{},{},{},{}]},{"entry":["CC","GCI","7-GC-","AFTS6","AFTS6AP6",{},"Aden",{},{},{}]},{"entry":[{},{},"AdenoSIIA",{},{},{},{},{},{},{}]},{"entry":["CC","GCI","8-GC-","5CYDK","5CYDKACS",{},"Aden",{},{},{}]},{"entry":[{},{},"AdenoSIIA",{},{},{},{},{},{},{}]},{"entry":["CC","GCI","9-GC-","XKSLS","XKSLSAF7",{},"Aden",{},{},{}]},{"entry":[{},{},"AdenoSIIA",{},{},{},{},{},{},{}]},{"entry":["CC","GCI","10-GC-","B4RU8","B4RU8A8Q",{},"Aden",{},{},{}]},{"entry":[{},{},"AdenoSIIA",{},{},{},{},{},{},{}]},{"entry":["CC","GCI","11-GC-","HB8EY","HB8EYA8I",{},"Aden",{},{},{}]},{"entry":[{},{},"AdenoSIIA",{},{},{},{},{},{},{}]},{"entry":["CC","Ichilov","12-(22)-","CG-229C",{},{},"Aden",{},{},"2"]},{"entry":[{},{},"AdenoSII",{},{},{},{},{},{},{}]},{"entry":["CC","GCI","13-GC-","X8C7X","X8C7XATL",{},"Aden",{},{},{}]},{"entry":[{},{},"AdenoSIIA",{},{},{},{},{},{},{}]},{"entry":["CC","GCI","14-GC-","HCP6K","HCP6KA8Z",{},"Aden",{},{},{}]},{"entry":[{},{},"AdenoSIIA",{},{},{},{},{},{},{}]},{"entry":["CC","GCI","15-GC-","ZX4X7","ZX4X7AXA",{},"Aden",{},{},{}]},{"entry":[{},{},"AdenoSIIA",{},{},{},{},{},{},{}]},{"entry":["CC","Asterand","16-As-","17915","31176","31176A1","Aden",{},{},"2-3"]},{"entry":[{},{},"AdenoSIIA",{},{},{},{},{},{},{}]},{"entry":["CC","Ichilov","17-(1)-Ic-","CG-335",{},{},"Aden",{},"Cec","2"]},{"entry":[{},{},"AdenoSIIA",{},{},{},{},{},{},{}]},{"entry":["CC","Asterand","19-As-","12772","18885","18885A1","Aden",{},"rectum","2"]},{"entry":[{},{},"AdenoSIIA",{},{},{},{},{},{},{}]},{"entry":["CC","GCI","20-GC-","JFYXP","JFYXPAMP",{},"Aden",{},{},{}]},{"entry":[{},{},"AdenoSIIA",{},{},{},{},{},{},{}]},{"entry":["CC","GCI","21-GC-","OJXW9","OJXW9ASR",{},"Aden",{},{},{}]},{"entry":[{},{},"AdenoSIIA",{},{},{},{},{},{},{}]},{"entry":["CC","Ichilov","22-(28)-Ic-","CG-284",{},{},"Aden",{},"sigma","2"]},{"entry":[{},{},"AdenoSIIA",{},{},{},{},{},{},{}]},{"entry":["CC","Ichilov","23-(10)-Ic-","CG-311",{},{},"Aden",{},"SigCol","1-2"]},{"entry":[{},{},"AdenoSIIA",{},{},{},{},{},{},{}]},{"entry":["CC","Ichilov","24-(14)-Ic-","CG-222",{},{},"WPAden",{},"Rectum",{}]},{"entry":[{},{},"AdenoSIII","(2)",{},{},{},{},{},{}]},{"entry":["CC","Ichilov","25-(23)-Ic-","CG-282",{},{},"MA",{},"sigma","UN"]},{"entry":[{},{},"AdenoSIII",{},{},{},{},{},{},{}]},{"entry":["CC","GCI","26-GC-","OTPI7","OTPI7AWY",{},"Aden",{},{},{}]},{"entry":[{},{},"AdenoSIII",{},{},{},{},{},{},{}]},{"entry":["CC","GCI","27-GC-","IG9NK","IG9NKAD3",{},"MA",{},{},{}]},{"entry":[{},{},"AdenoSIII",{},{},{},{},{},{},{}]},{"entry":["CC","GCI","28-GC-","53OM7","53OM7AGL",{},"Aden",{},{},{}]},{"entry":[{},{},"AdenoSIII",{},{},{},{},{},{},{}]},{"entry":["CC","GCI","29-GC-","BLUW6","BLUW6A6Y",{},"Aden",{},{},{}]},{"entry":[{},{},"AdenoSIII",{},{},{},{},{},{},{}]},{"entry":["CC","GCI","30-GC-","VZ6QA","VZ6QAAFA",{},"Aden",{},"REC-",{}]},{"entry":[{},{},"AdenoSIII",{},{},{},{},{},"TUM",{}]},{"entry":["CC","Ichilov","31-(6)-Ic-","CG-303",{},{},"Aden",{},{},"1-2"]},{"entry":[{},{},"AdenoSIII","(3)",{},{},{},{},{},{}]},{"entry":["CC","Ichilov","32-(2)-Ic-","CG-307",{},{},"Aden",{},"Cecum","2"]},{"entry":[{},{},"AdenoSIII",{},{},{},{},{},{},{}]},{"entry":["CC","Ichilov","33-(11)-Ic-","CG-337",{},{},"Aden",{},{},"1-2"]},{"entry":[{},{},"AdenoSIII",{},{},{},{},{},{},{}]},{"entry":["CC","Asterand","34-As-","18462","40971","40971A1","TA",{},"SigCol","2"]},{"entry":[{},{},"AdenoSIIIC",{},{},{},{},{},{},{}]},{"entry":["CC","Ichilov","35-(13)-Ic-","CG-290",{},{},"Aden",{},"RectCol","2"]},{"entry":[{},{},"AdenoSIV",{},{},{},{},{},{},{}]},{"entry":["CC","GCI","36-GC-","7D7QV","7D7QVAE6",{},"Aden",{},{},{}]},{"entry":[{},{},"AdenoSIV",{},{},{},{},{},{},{}]},{"entry":["CC","GCI","37-GC-","38U4V","38U4VAA4",{},"Aden",{},{},{}]},{"entry":[{},{},"AdenoSIV",{},{},{},{},{},{},{}]},{"entry":["CC","Ichilov","38-(9)-Ic-","CG-297",{},{},"Aden",{},"Rectum","2"]},{"entry":[{},{},"AdenoSIV",{},{},{},{},{},{},{}]},{"entry":["CC","Ichilov","71-(16)-Ic-","CG-278C",{},{},"Aden",{},{},"2"]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{}]},{"entry":["CC","Ichilov","72-(4)-Ic-","CG-276",{},{},"Carc",{},{},"3"]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{}]},{"entry":["CC","Ichilov","73-(17)-Ic-","CG-163",{},{},"Aden",{},"Rectum","2"]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{}]},{"entry":["CC","Ichilov","74-(5)-Ic-","CG-308",{},{},"Aden",{},"ColSig","2"]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{}]},{"entry":["CC","Ichilov","75-(72)-Ic-","CG-309",{},{},"Aden",{},{},"3"]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{}]},{"entry":["CC","Ichilov","76-(18)-Ic-","CG-22C",{},{},"Aden",{},{},"UN"]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{}]},{"entry":["CC","Ichilov","78-(21)-Ic-","CG-18C",{},{},"Aden",{},{},"UN"]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{}]},{"entry":["CC","Ichilov","79-(24)-Ic-","CG-12",{},{},"Aden",{},{},"UN"]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{}]},{"entry":["CC","Ichilov","80-(25)-Ic-","CG-2",{},{},"Aden",{},{},"UN"]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{}]},{"entry":["CC","biochain","82-(61)-Bc-","A606258",{},{},"Aden, Ulcer",{},{},"2"]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{}]},{"entry":["CC","biochain","83-(57)-Bc-","A609150",{},{},"Aden",{},{},"3"]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{}]},{"entry":["CC","biochain","84-(56)-Bc-","A609148",{},{},"Aden",{},{},"2"]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{}]},{"entry":["CC","biochain","85-(53)-Bc-","A609161",{},{},"Aden",{},{},"3"]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{}]},{"entry":["CC","biochain","86-(54)-Bc-","A609142",{},{},"Aden",{},{},"3"]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{}]},{"entry":["CC","biochain","87-(59)-Bc-","A609059",{},{},"Aden, Ulcer",{},{},"1"]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{}]},{"entry":["CC","biochain","88-(60)-Bc-","A609058",{},{},"Aden, ",{},{},"2"]},{"entry":[{},{},"Adeno",{},{},{},"Ulcer",{},{},{}]},{"entry":["CC","biochain","89-(55)-Bc-","A609144",{},{},"Aden",{},{},"3"]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{}]},{"entry":["CC","biochain","90-(58)-Bc-","A609152",{},{},"Aden",{},{},"1"]},{"entry":[{},{},"Adeno",{},{},{},{},{},{},{}]},{"entry":["CB","GCI","40-GC-Ben","IG3OY","IG3OYN7S",{},"TSAden",{},"RTCol",{}]},{"entry":["CB","GCI","41-GC-Ben","GKIEY","GKIEYAV4",{},"TSAdenHGD",{},"ProxTCol",{}]},{"entry":["CN","GCI","42-GC-NPS","AGVTC","AGVTCNK7",{},"NC","DIV",{},{}]},{"entry":["CN","Asterand","43-As-NPS","8956","9153","9153B1","NC",{},{},{}]},{"entry":["CN","GCI","44-GC-NPS","IG3OY","IG3OYN7S",{},"NC",{},"RTCol",{}]},{"entry":["CN","GCI","45-GC-NPS","K9OYX","K9OYXN4F",{},"NC","Divsw\/","LTCol",{}]},{"entry":[{},{},{},{},{},{},{},"FDIV",{},{}]},{"entry":["CN","Asterand","46-As-NPS","23024","74445","74445B1","NC","ChrDivs",{},{}]},{"entry":["CN","Asterand","47-As-NPS","23049","71410","71410B2","NC","ChrDivs",{},{}]},{"entry":["CN","GCI","48-GC-NPS","G7JJX","G7JJXAX7",{},"NC","Divsw\/","SigCol",{}]},{"entry":[{},{},{},{},{},{},{},"DIV...",{},{}]},{"entry":["CN","Asterand","49-As-NPS","22900","74446","74446B1","NC","ADw\/AF",{},{}]},{"entry":["CN","GCI","50-GC-NPS","XVPZ2","XVPZ2NDD",{},"NC","Div",{},{}]},{"entry":["CN","GCI","51-GC-NPS","CDSUV","CDSUVNR3",{},"NC","CU",{},{}]},{"entry":["CN","GCI","52-GC-NPS","GP5KH","GP5KHAOC",{},"NC","Div",{},{}]},{"entry":["CN","GCI","53-GC-NPS","YUZNR","YUZNRNDN",{},"NC","Divs","SigCol",{}]},{"entry":["CN","GCI","54-GC-NPS","28QN6","28QN6NI1",{},"NC","TSAden","RTCol",{}]},{"entry":["CN","GCI","55-GC-NPS","GV6N8","GV6N8NG9",{},"NC","Divs,",{},{}]},{"entry":["CN","GCI","56-GC-NPS","ZJ17R","ZJ17RNIH",{},"NC","TubAden","RTCol",{}]},{"entry":["CN","GCI","57-GC-NPS","2EEBJ","2EEBJN2Q",{},"NC","Div\/",{},{}]},{"entry":[{},{},{},{},{},{},{},"ChrInfl",{},{}]},{"entry":["CN","GCI","58-GC-NPS","68IX5","68IX5N1H",{},"NC","ChrDiv","LTCol",{}]},{"entry":["CN","GCI","59-GC-NPS","9GEGL","9GEGLN1V",{},"NC","ExtDivs","SigCol",{}]},{"entry":["CN","GCI","60-GC-NPS","PKU8O","PKU8OAJ3",{},"NC","Divs, ","SigCol",{}]},{"entry":[{},{},{},{},{},{},{},"ChrDiv...",{},{}]},{"entry":["CN","Asterand","61-As-NPS","22903","74452","74452B1","NC","MUw\/MI",{},{}]},{"entry":["CN","Asterand","62-As-NPS","16364","31802","31802B1","NC","UC",{},{}]},{"entry":["CN","biochain","63-(65)-","A607115",{},{},"N-PM","PM",{},{}]},{"entry":[{},{},"Bc-NPM",{},{},{},{},{},{},{}]},{"entry":["CN","Ambion","64-(71)-","071P10B",{},{},"N-PM","PM",{},{}]},{"entry":[{},{},"Am-NPM",{},{},{},{},{},{},{}]},{"entry":["CN","biochain","65-(66)-","A609262",{},{},"N-PM","PM",{},{}]},{"entry":[{},{},"Bc-NPM",{},{},{},{},{},{},{}]},{"entry":["CN","biochain","66-(63)-","A609260",{},{},"N-PM","PM",{},{}]},{"entry":[{},{},"Bc-NPM",{},{},{},{},{},{},{}]},{"entry":["CN","biochain","67-(62)-","A608273",{},{},"N-PM","PM",{},{}]},{"entry":[{},{},"Bc-NPM",{},{},{},{},{},{},{}]},{"entry":["CN","biochain","68-(64)-","A609261",{},{},"N-PM","PM",{},{}]},{"entry":[{},{},"Bc-NPM",{},{},{},{},{},{},{}]},{"entry":["CN","biochain","69-(41)-","A501156",{},{},"N-PM","PM",{},{}]},{"entry":[{},{},"Bc-NPM",{},{},{},{},{},{},{}]},{"entry":["CN","biochain","70-(67)-","A406029 +",{},{},"N-PMP10","PM",{},{}]},{"entry":[{},{},"Bc-NPM","A411078"]},{"entry":{"@attributes":{"namest":"1","nameend":"10","align":"center","rowsep":"1"}}},{"entry":{"@attributes":{"namest":"1","nameend":"10","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"10"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"9","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"10","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Source\/",{},{},{},{},"Tumor ","Gen-",{},{}]},{"entry":["Tissue","Delivery","sample name","TNM","CS","CS2","%","der","age","Ethnic B"]},{"entry":{"@attributes":{"namest":"1","nameend":"10","align":"center","rowsep":"1"}}},{"entry":["CC","Asterand","1-As-AdenS0","TXN0M0","0",{},"80","F","43","CAU"]},{"entry":["CC","GCI","2-GC-AdenoSI",{},"I","DukeA","85","F","44","WCAU"]},{"entry":["CC","Ichilov","3-(7)-Ic-AdenoSI",{},"I","DukeA",{},"F","66",{}]},{"entry":["CC","GCI","4-GC-AdenoSI",{},"I","DukeB1","80","M","53","WCAU"]},{"entry":["CC","GCI","5-GC-AdenoSI",{},"I","DukeB1","70","F","70","WCAU"]},{"entry":["CC","Ichilov","6-(20)-Ic-AdenoSIIA",{},"IIA","DukeB2",{},"M","36",{}]},{"entry":["CC","GCI","7-GC-AdenoSIIA",{},"IIA","DukeB2","75","M","39","WCAU"]},{"entry":["CC","GCI","8-GC-AdenoSIIA",{},"IIA","DukeB2","65","M","44","WCAU"]},{"entry":["CC","GCI","9-GC-AdenoSIIA",{},"IIA","DukeB2","65","M","48","WCAU"]},{"entry":["CC","GCI","10-GC-AdenoSIIA",{},"IIA","DukeB2","65","F","50","WCAU"]},{"entry":["CC","GCI","11-GC-AdenoSIIA",{},"IIA","DukeB2","65","M","53","WCAU"]},{"entry":["CC","Ichilov","12-(22)-AdenoSII",{},"II","DukeB",{},"F","55",{}]},{"entry":["CC","GCI","13-GC-AdenoSIIA",{},"IIA","DukeB2","90","M","56","WCAU"]},{"entry":["CC","GCI","14-GC-AdenoSIIA",{},"IIA","DukeB2.","80","M","58","WCAU"]},{"entry":["CC","GCI","15-GC-AdenoSIIA",{},"IIA","DukeB2","90","M","60","WCAU"]},{"entry":["CC","Asterand","16-As-AdenoSIIA","T3N0M0","IIA","DukeB2","60","F","64","CAU"]},{"entry":["CC","Ichilov","17-(1)-Ic-AdenoSIIA",{},"IIA","DukeB2",{},"F","66",{}]},{"entry":["CC","Asterand","19-As-AdenoSIIA","T3NXM0","IIA","DukeB2","60","F","67","CAU"]},{"entry":["CC","GCI","20-GC-AdenoSIIA",{},"IIA","DukeB2","60","F","68","WCAU"]},{"entry":["CC","GCI","21-GC-AdenoSIIA",{},"IIA","DukeB2","90","F","69","WCAU"]},{"entry":["CC","Ichilov","22-(28)-Ic-AdenoSIIA",{},"IIA","DukeB2",{},"F","72",{}]},{"entry":["CC","Ichilov","23-(10)-Ic-AdenoSIIA",{},"eIIA","DukeB2",{},"M","88",{}]},{"entry":["CC","Ichilov","24-(14)-Ic-AdenoSIII",{},"III","DukeC",{},"F","49",{}]},{"entry":["CC","Ichilov","25-(23)-Ic-AdenoSIII",{},"III","DukeC",{},"M","51",{}]},{"entry":["CC","GCI","26-GC-AdenoSIII",{},"III","DukeC2","70","F","54","WCAU"]},{"entry":["CC","GCI","27-GC-AdenoSIII",{},"III","DukeC2","90","F","54","WCAU"]},{"entry":["CC","GCI","28-GC-AdenoSIII",{},"III","DukeC2","75","F","61","WCAU"]},{"entry":["CC","GCI","29-GC-AdenoSIII",{},"III","DukeC2","85","F","64","WCAU"]},{"entry":["CC","GCI","30-GC-AdenoSIII",{},"III","DukeC2","60","M","67","WCAU"]},{"entry":["CC","Ichilov","31-(6)-Ic-AdenoSIII",{},"III","DukeC2.",{},"F","77",{}]},{"entry":["CC","Ichilov","32-(2)-Ic-AdenoSIII",{},"III","DukeC2.",{},"F","89",{}]},{"entry":["CC","Ichilov","33-(11)-Ic-AdenoSIII",{},"III","DukeC2.",{},"NA","NA",{}]},{"entry":["CC","Asterand","34-As-AdenoSIIIC",{},"IIIC",{},"76","F","68","CAU"]},{"entry":["CC","Ichilov","35-(13)-Ic-AdenoSIV",{},"IV","DukeD.",{},"M","47",{}]},{"entry":["CC","GCI","36-GC-AdenoSIV",{},"IV","DukeD","80","F","52","WCAU"]},{"entry":["CC","GCI","37-GC-AdenoSIV",{},"IV","DukeD","85","F","53","WCAU"]},{"entry":["CC","Ichilov","38-(9)-Ic-AdenoSIV",{},"IV","DukeD.",{},"M","62",{}]},{"entry":["CC","Ichilov","71-(16)-Ic-Adeno",{},{},"UN","50","F","60",{}]},{"entry":["CC","Ichilov","72-(4)-Ic-Adeno",{},{},"UN","75","M","64",{}]},{"entry":["CC","Ichilov","73-(17)-Ic-Adeno",{},{},"UN",{},"M","73",{}]},{"entry":["CC","Ichilov","74-(5)-Ic-Adeno",{},{},"UN",{},"F","80",{}]},{"entry":["CC","Ichilov","75-(72)-Ic-Adeno",{},{},"UN",{},"F","88",{}]},{"entry":["CC","Ichilov","76-(18)-Ic-Adeno",{},{},"UN",{},"NA","NA",{}]},{"entry":["CC","Ichilov","78-(21)-Ic-Adeno",{},{},"UN",{},"NA","NA",{}]},{"entry":["CC","Ichilov","79-(24)-Ic-Adeno",{},{},"UN",{},"NA","NA",{}]},{"entry":["CC","Ichilov","80-(25)-Ic-Adeno",{},{},"UN",{},"NA","NA",{}]},{"entry":["CC","biochain","82-(61)-Bc-Adeno",{},{},"UN",{},"M","41",{}]},{"entry":["CC","biochain","83-(57)-Bc-Adeno",{},{},"UN",{},"F","45",{}]},{"entry":["CC","biochain","84-(56)-Bc-Adeno",{},{},"UN","40","F","48",{}]},{"entry":["CC","biochain","85-(53)-Bc-Adeno",{},{},"UN",{},"F","53",{}]},{"entry":["CC","biochain","86-(54)-Bc-Adeno",{},{},"UN",{},"M","53",{}]},{"entry":["CC","biochain","87-(59)-Bc-Adeno",{},{},"UN",{},"M","58",{}]},{"entry":["CC","biochain","88-(60)-Bc-Adeno",{},{},"UN",{},"M","67",{}]},{"entry":["CC","biochain","89-(55)-Bc-Adeno",{},{},"UN",{},"M","68",{}]},{"entry":["CC","biochain","90-(58)-Bc-Adeno",{},{},"UN",{},"M","73",{}]},{"entry":["CB","GCI","40-GC-Ben",{},{},{},{},"F","48","WCAU"]},{"entry":["CB","GCI","41-GC-Ben",{},{},{},{},"F","75","WCAU"]},{"entry":["CN","GCI","42-GC-NPS",{},{},{},"0","M","45","WCAU"]},{"entry":["CN","Asterand","43-As-NPS",{},{},{},"0","F","46","CAU"]},{"entry":["CN","GCI","44-GC-NPS",{},{},{},"0","F","48","WCAU"]},{"entry":["CN","GCI","45-GC-NPS",{},{},{},"0","F","50","WCAU"]},{"entry":["CN","Asterand","46-As-NPS",{},{},{},"0","F","52","CAU"]},{"entry":["CN","Asterand","47-As-NPS",{},{},{},"0","F","52","CAU"]},{"entry":["CN","GCI","48-GC-NPS",{},{},{},"0","M","52","WCAU"]},{"entry":["CN","Asterand","49-As-NPS",{},{},{},"0","M","54","CAU"]},{"entry":["CN","GCI","50-GC-NPS",{},{},{},"0","F","55","WCAU"]},{"entry":["CN","GCI","51-GC-NPS",{},{},{},"0","M","55","WCAU"]},{"entry":["CN","GCI","52-GC-NPS",{},{},{},"0","F","57","WCAU"]},{"entry":["CN","GCI","53-GC-NPS",{},{},{},"0","F","57","WCAU"]},{"entry":["CN","GCI","54-GC-NPS",{},{},{},"0","M","59","WCAU"]},{"entry":["CN","GCI","55-GC-NPS",{},{},{},"0","F","61","WCAU"]},{"entry":["CN","GCI","56-GC-NPS",{},{},{},"0","M","61","WCAU"]},{"entry":["CN","GCI","57-GC-NPS",{},{},{},"0","F","66","WCAU"]},{"entry":["CN","GCI","58-GC-NPS",{},{},{},"0","F","66","WCAU"]},{"entry":["CN","GCI","59-GC-NPS",{},{},{},"0","M","68","WCAU"]},{"entry":["CN","GCI","60-GC-NPS",{},{},{},"0","F","69","WCAU"]},{"entry":["CN","Asterand","61-As-NPS",{},{},{},"0","M","71","CAU"]},{"entry":["CN","Asterand","62-As-NPS",{},{},{},"0","F","74","WCAU"]},{"entry":["CN","biochain","63-(65)-Bc-NPM",{},{},{},{},"M","24",{}]},{"entry":["CN","Ambion","64-(71)-Am-NPM",{},{},{},{},"F","34",{}]},{"entry":["CN","biochain","65-(66)-Bc-NPM",{},{},{},{},"M","58",{}]},{"entry":["CN","biochain","66-(63)-Bc-NPM",{},{},{},{},"M","61",{}]},{"entry":["CN","biochain","67-(62)-Bc-NPM",{},{},{},{},"M","66",{}]},{"entry":["CN","biochain","68-(64)-Bc-NPM",{},{},{},{},"F","68",{}]},{"entry":["CN","biochain","69-(41)-Bc-NPM",{},{},{},{},"M","78",{}]},{"entry":["CN","biochain","70-(67)-Bc-NPM",{},{},{},{},"F&","M",{}]},{"entry":[{},{},{},{},{},{},{},"M","(26-78)&F",{}]},{"entry":[{},{},{},{},{},{},{},{},"(53-77)."]},{"entry":{"@attributes":{"namest":"1","nameend":"10","align":"center","rowsep":"1"}}},{"entry":{"@attributes":{"namest":"1","nameend":"10","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"9"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"91pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"9","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Source\/",{},"Alcohol","Dr.","Alc.","Recovery","Exc."]},{"entry":[{},"Tissue","Delivery","sample name","Status","per day","Dur.","Type","Y."]},{"entry":{"@attributes":{"namest":"1","nameend":"9","align":"center","rowsep":"1"}}},{"entry":[{},"CC","Asterand","1-As-AdenS0","NU",{},{},"Auto","2004"]},{"entry":[{},"CC","GCI","2-GC-AdenoSI","Y","4",{},"Surg",{}]},{"entry":[{},"CC","Ichilov","3-(7)-Ic-AdenoSI",{},{},{},{},{}]},{"entry":[{},"CC","GCI","4-GC-AdenoSI","Y","\u2014",{},"Surg",{}]},{"entry":[{},"CC","GCI","5-GC-AdenoSI","Y","0",{},"Surg",{}]},{"entry":[{},"CC","Ichilov","6-(20)-Ic-AdenoSIIA",{},{},{},{},{}]},{"entry":[{},"CC","GCI","7-GC-AdenoSIIA","N","0",{},"Surg",{}]},{"entry":[{},"CC","GCI","8-GC-AdenoSIIA","N","\u2014",{},"Surg",{}]},{"entry":[{},"CC","GCI","9-GC-AdenoSIIA","Y","10\u2002",{},"Surg",{}]},{"entry":[{},"CC","GCI","10-GC-AdenoSIIA","N","\u2014",{},"Surg",{}]},{"entry":[{},"CC","GCI","11-GC-AdenoSIIA","N","\u2014",{},"Surg",{}]},{"entry":[{},"CC","Ichilov","12-(22)-AdenoSII",{},{},{},{},{}]},{"entry":[{},"CC","GCI","13-GC-AdenoSIIA","N","\u2014",{},"Surg",{}]},{"entry":[{},"CC","GCI","14-GC-AdenoSIIA","Y","4",{},"Surg",{}]},{"entry":[{},"CC","GCI","15-GC-AdenoSIIA","Y","5",{},"Surg",{}]},{"entry":[{},"CC","Asterand","16-As-AdenoSIIA","occ","1","21-30","Auto","2004"]},{"entry":[{},{},{},{},{},"drink\/week","years",{},{}]},{"entry":[{},"CC","Ichilov","17-(1)-Ic-AdenoSIIA",{},{},{},{},{}]},{"entry":[{},"CC","Asterand","19-As-AdenoSIIA","NU",{},{},"Surg","2004"]},{"entry":[{},"CC","GCI","20-GC-AdenoSIIA","Y","\u2014",{},"Surg",{}]},{"entry":[{},"CC","GCI","21-GC-AdenoSIIA","N","\u2014",{},"Surg",{}]},{"entry":[{},"CC","Ichilov","22-(28)-Ic-AdenoSIIA",{},{},{},{},{}]},{"entry":[{},"CC","Ichilov","23-(10)-Ic-AdenoSIIA",{},{},{},{},{}]},{"entry":[{},"CC","Ichilov","24-(14)-Ic-AdenoSIII",{},{},{},{},{}]},{"entry":[{},"CC","Ichilov","25-(23)-Ic-AdenoSIII",{},{},{},{},{}]},{"entry":[{},"CC","GCI","26-GC-AdenoSIII","N","\u2014",{},"Surg",{}]},{"entry":[{},"CC","GCI","27-GC-AdenoSIII","N","\u2014",{},"Surg",{}]},{"entry":[{},"CC","GCI","28-GC-AdenoSIII","N","\u2014",{},"Surg",{}]},{"entry":[{},"CC","GCI","29-GC-AdenoSIII","N","\u2014",{},"Surg",{}]},{"entry":[{},"CC","GCI","30-GC-AdenoSIII","Y","14\u2002",{},"Surg",{}]},{"entry":[{},"CC","Ichilov","31-(6)-Ic-AdenoSIII",{},{},{},{},{}]},{"entry":[{},"CC","Ichilov","32-(2)-Ic-AdenoSIII",{},{},{},{},{}]},{"entry":[{},"CC","Ichilov","33-(11)-Ic-AdenoSIII",{},{},{},{},{}]},{"entry":[{},"CC","Asterand","34-As-AdenoSIIIC","NU",{},{},"Surg","2005"]},{"entry":[{},"CC","Ichilov","35-(13)-Ic-AdenoSIV",{},{},{},{},{}]},{"entry":[{},"CC","GCI","36-GC-AdenoSIV","Y","3",{},"Surg",{}]},{"entry":[{},"CC","GCI","37-GC-AdenoSIV",{},"\u2014",{},"Surg",{}]},{"entry":[{},"CC","Ichilov","38-(9)-Ic-AdenoSIV",{},{},{},{},{}]},{"entry":[{},"CC","Ichilov","71-(16)-Ic-Adeno",{},{},{},{},{}]},{"entry":[{},"CC","Ichilov","72-(4)-Ic-Adeno",{},{},{},{},{}]},{"entry":[{},"CC","Ichilov","73-(17)-Ic-Adeno",{},{},{},{},{}]},{"entry":[{},"CC","Ichilov","74-(5)-Ic-Adeno",{},{},{},{},{}]},{"entry":[{},"CC","Ichilov","75-(72)-Ic-Adeno",{},{},{},{},{}]},{"entry":[{},"CC","Ichilov","76-(18)-Ic-Adeno",{},{},{},{},{}]},{"entry":[{},"CC","Ichilov","78-(21)-Ic-Adeno",{},{},{},{},{}]},{"entry":[{},"CC","Ichilov","79-(24)-Ic-Adeno",{},{},{},{},{}]},{"entry":[{},"CC","Ichilov","80-(25)-Ic-Adeno",{},{},{},{},{}]},{"entry":[{},"CC","biochain","82-(61)-Bc-Adeno",{},{},{},{},{}]},{"entry":[{},"CC","biochain","83-(57)-Bc-Adeno",{},{},{},{},{}]},{"entry":[{},"CC","biochain","84-(56)-Bc-Adeno",{},{},{},{},{}]},{"entry":[{},"CC","biochain","85-(53)-Bc-Adeno",{},{},{},{},{}]},{"entry":[{},"CC","biochain","86-(54)-Bc-Adeno",{},{},{},{},{}]},{"entry":[{},"CC","biochain","87-(59)-Bc-Adeno",{},{},{},{},{}]},{"entry":[{},"CC","biochain","88-(60)-Bc-Adeno",{},{},{},{},{}]},{"entry":[{},"CC","biochain","89-(55)-Bc-Adeno",{},{},{},{},{}]},{"entry":[{},"CC","biochain","90-(58)-Bc-Adeno",{},{},{},{},{}]},{"entry":[{},"CB","GCI","40-GC-Ben","Y","1",{},"Surg",{}]},{"entry":[{},"CB","GCI","41-GC-Ben","N","\u2014",{},"Surg",{}]},{"entry":[{},"CN","GCI","42-GC-NPS","N","\u2014",{},"Surg",{}]},{"entry":[{},"CN","Asterand","43-As-NPS","NU",{},{},"Surg","2002"]},{"entry":[{},"CN","GCI","44-GC-NPS","Y","1",{},"Surg",{}]},{"entry":[{},"CN","GCI","45-GC-NPS","N","\u2014",{},"Surg",{}]},{"entry":[{},"CN","Asterand","46-As-NPS","Occ",{},{},"Surg","2005"]},{"entry":[{},"CN","Asterand","47-As-NPS","occ",{},{},"Surg","2005"]},{"entry":[{},"CN","GCI","48-GC-NPS","N","\u2014",{},"Surg",{}]},{"entry":[{},"CN","Asterand","49-As-NPS","CurU",{},{},"Surg","2005"]},{"entry":[{},"CN","GCI","50-GC-NPS","N","\u2014",{},"Surg",{}]},{"entry":[{},"CN","GCI","51-GC-NPS","N","\u2014",{},"Surg",{}]},{"entry":[{},"CN","GCI","52-GC-NPS","Y","6",{},"Surg",{}]},{"entry":[{},"CN","GCI","53-GC-NPS","Y","1",{},"Surg",{}]},{"entry":[{},"CN","GCI","54-GC-NPS","Y","42\u2002",{},"Surg",{}]},{"entry":[{},"CN","GCI","55-GC-NPS","Y","3",{},"Surg",{}]},{"entry":[{},"CN","GCI","56-GC-NPS","Y","\u2014",{},"Surg",{}]},{"entry":[{},"CN","GCI","57-GC-NPS","Y","4",{},"Surg",{}]},{"entry":[{},"CN","GCI","58-GC-NPS","N","\u2014",{},"Surg",{}]},{"entry":[{},"CN","GCI","59-GC-NPS","N","\u2014",{},"Surg",{}]},{"entry":[{},"CN","GCI","60-GC-NPS","N","\u2014",{},"Surg",{}]},{"entry":[{},"CN","Asterand","61-As-NPS","Occ",{},{},"Surg","2005"]},{"entry":[{},"CN","Asterand","62-As-NPS","Occ",{},{},"Surg","2004"]},{"entry":[{},"CN","biochain","63-(65)-Bc-NPM",{},{},{},{},{}]},{"entry":[{},"CN","Ambion","64-(71)-Am-NPM",{},{},{},{},{}]},{"entry":[{},"CN","biochain","65-(66)-Bc-NPM",{},{},{},{},{}]},{"entry":[{},"CN","biochain","66-(63)-Bc-NPM",{},{},{},{},{}]},{"entry":[{},"CN","biochain","67-(62)-Bc-NPM",{},{},{},{},{}]},{"entry":[{},"CN","biochain","68-(64)-Bc-NPM",{},{},{},{},{}]},{"entry":[{},"CN","biochain","69-(41)-Bc-NPM",{},{},{},{},{}]},{"entry":[{},"CN","biochain","70-(67)-Bc-NPM"]},{"entry":{"@attributes":{"namest":"1","nameend":"9","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0742","num":"0741"},"tables":{"@attributes":{"id":"TABLE-US-00014","num":"00014"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"168pt","align":"left"}}],"thead":{"row":[{"entry":"TABLE 5_1"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["Key","Full Name"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["CC","Colon Cancer"]},{"entry":["CB","Colon Benign"]},{"entry":["CN","Colon Normal"]},{"entry":["WT","Weight"]},{"entry":["HT","Height"]},{"entry":["Aden","Adenocarcinoma"]},{"entry":["AI","Adenocarcinoma intramucosal"]},{"entry":["UA","Ulcerated adenocarcinoma"]},{"entry":["WP Aden","Well polypoid adeocarcinoma"]},{"entry":["MA","Mucinus adenocarcinoma"]},{"entry":["TA","Tubular adenocarcinoma"]},{"entry":["Carc","Carcinoma"]},{"entry":["TS Aden","TUBULOVILLOUS ADENOMA"]},{"entry":["TS Aden HGD","TUBULOVILLOUS ADENOMA with HIGH"]},{"entry":[{},"GRADE DYSPLASIA"]},{"entry":["NC","Normal Colon"]},{"entry":["N-PM","Normal PM"]},{"entry":["N-PM P10","Normal PM (Pool 10)"]},{"entry":["Diag","Diagnosis"]},{"entry":["Div","DIVERTICULITIS"]},{"entry":["Divs w\/F DIV","Diverticulosis with Focal DIVERTICULITIS"]},{"entry":["Chr Divs","Chronic diverticulosis"]},{"entry":["Divs","DIVERTICULOSIS WITH DIVERTICULITIS AND"]},{"entry":["w\/DIV . . .","FOCAL ABSCESS FORMATION; NO MALIGNANCY"]},{"entry":["AD w\/AF","Acute diverticulitis with abscess formation"]},{"entry":["CU","CECAL ULCERATION"]},{"entry":["Divs, PA","DIVERTICULOSIS AND PERICOLIC ABSCESS"]},{"entry":["Tub Aden","TUBULAR ADENOMA"]},{"entry":["Div\/Chr Infl","DIVERTICULOSIS\/CHRONIC INFLAMMATION"]},{"entry":["Chr Div","CHRONIC DIVERTICULITIS"]},{"entry":["Ext Divs","EXTENSIVE DIVERTICULOSIS"]},{"entry":["Divs,","DIVERTICULOSIS AND CHRONIC"]},{"entry":["Chr Div . . .","DIVERTICULITIS, SEROSAL FIBROSIS AND"]},{"entry":[{},"CHRONIC SEROSITIS"]},{"entry":["MU w\/MI","Mucosal ulceration with mural inflammation"]},{"entry":["UC","Ulcerative colitis"]},{"entry":["Cec","cecum"]},{"entry":["Dis C","DISTAL COLON"]},{"entry":["Ret, Low Ant","RETROSIGMOID, LOW ANTERIOR"]},{"entry":["Rect Col","Rectosigmoidal colon"]},{"entry":["Sig col","Sigmod colon"]},{"entry":["Col Sig","Colon Sigma"]},{"entry":["RT Col","RIGHT COLON"]},{"entry":["Prox T Col","PROXIMAL TRANSVERSE COLON"]},{"entry":["LT Col","Left Colon"]},{"entry":["Gr","Grade"]},{"entry":["CS","Cancer Stage"]},{"entry":["Ethnic B","Ethnic background"]},{"entry":["NU","Never Used"]},{"entry":["Occ","Occasion"]},{"entry":["Cur U","Current use"]},{"entry":["Dr. per day","Drinks per day"]},{"entry":["Alc. Dur.","Alcohol Duration"]},{"entry":["Auto.","Autopsy"]},{"entry":["Surg.","Surgical"]},{"entry":["Exc. Y.","Excision Year"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}}}},"Materials and Experimental Procedures Used to Obtain Expression Data RNA Preparation\u2014","RNA was obtained from ABS (Wilmington, Del. 19801, USA, http:\/\/www.absbioreagents.com), BioChain Inst. Inc. (Hayward, Calif. 94545 USA www.biochain.com), GOG for ovary samples\u2014Pediatic Cooperative Human Tissue Network, Gynecologic Oncology Group Tissue Bank, Children Hospital of Columbus (Columbus Ohio 43205 USA), Clontech (Franklin Lakes, N.J. USA 07417, www.clontech.com), Ambion (Austin, Tex. 78744 USA, http:\/\/www.ambion.com), Asternad (Detroit, Mich. 48202-3420, USA, www.asterand.com), and from Genomics Collaborative Inc. a Division of Seracare (Cambridge, Mass. 02139, USA, www.genomicsinc.com). Alternatively, RNA was generated from blood cells, cell lines or tissue samples using TRI-Reagent (Molecular Research Center), according to Manufacturer's instructions. Tissue and RNA samples were obtained from patients or from postmortem. Most total RNA samples were treated with DNaseI (Ambion).","RT PCR\u2014Purified RNA (2-10 \u03bcg) was mixed with 300-1500 ng Random Hexamer primers (Invitrogen) and 500 \u03bcM dNTP in a total volume of 31.2 to 156 \u03bcl. The mixture was incubated for 5 mM at 65\u00b0 C. and then quickly chilled on ice. Thereafter, 10-50 \u03bcl of 5\u00d7 SuperscriptII first strand buffer (Invitrogen), 4.8 to 24 \u03bcl 0.1M DTT and 80-400 units RNasin (Promega) were added, and the mixture was incubated for 10 min at 25\u00b0 C., followed by further incubation at 42\u00b0 C. for 2 mM. Then, 2-10 \u03bcl (400-2000 units) of SuperscriptII (Invitrogen) was added and the reaction (final volume of 50-2500) was incubated for 50 min at 42\u00b0 C. and then inactivated at 70\u00b0 C. for 15 min. The resulting cDNA was diluted 1:20 in TE buffer (10 mM Tris pH=8, 1 mM EDTA pH=8).","Real-Time RT-PCR analysis carried out as described below\u2014cDNA (5 \u03bcl), prepared as described above, was used as a template in Real-Time PCR reactions (final volume of 20 \u03bcl) using the SYBR Green I assay (PE Applied Biosystem) with specific primers and UNG Enzyme (Eurogentech or ABI or Roche). The amplification was effected as follows: 50\u00b0 C. for 2 min, 95\u00b0 C. for 10 min, and then 40 cycles of 95\u00b0 C. for 15 sec, followed by 60\u00b0 C. for 1 min, following by dissociation step. Detection was performed by using the PE Applied Biosystem SDS 7000. The cycle in which the reactions achieved a threshold level of fluorescence (Ct=Threshold Cycle, described in detail below) was registered and was used to calculate the relative transcript quantity in the RT reactions. The relative quantity was calculated using the equation Q=efficiency^-Ct. The efficiency of the PCR reaction was calculated from a standard curve, created by using different dilutions of several reverse transcription (RT) reactions. To minimize inherent differences in the RT reaction, the resulting relative quantities were normalized using a normalization factor calculated in the following way:","The expression of several housekeeping (HSKP) genes was checked on every panel. The relative quantity (Q) of each housekeeping gene in each sample, calculated as described above, was divided by the median quantity of this gene in all panel samples to obtain the \u201crelative Q rel to MED\u201d. Then, for each sample the median of the \u201crelative Q rel to MED\u201d of the selected housekeeping genes was calculated and served as normalization factor of this sample for further calculations. Schematic summary of quantitative real-time PCR analysis is presented in . As shown, the x-axis shows the cycle number. The CT=Threshold Cycle point, which is the cycle that the amplification curve crosses the fluorescence threshold that was set in the experiment. This point is a calculated cycle number in which PCR products signal is above the background level (passive dye ROX) and still in the Geometric\/Exponential phase (as shown, once the level of fluorescence crosses the measurement threshold, it has a geometrically increasing phase, during which measurements are most accurate, followed by a linear phase and a plateau phase; for quantitative measurements, the latter two phases do not provide accurate measurements). The y-axis shows the normalized reporter fluorescence. It should be noted that this type of analysis provides relative quantification.","For each RT sample, the expression of the specific amplicon was normalized to the normalization factor calculated from the expression of different house keeping genes as described in section above.","These house keeping genes are different for each panel. For colon panel\u2014HPRT1 (GenBank Accession No. NM000194 (SEQ ID NO: 118); amplicon\u2014HPRT1-amplicon (SEQ ID NO:181)), PBGD (GenBank Accession No. BC019323 (SEQ ID NO: 117); amplicon\u2014PBGD-amplicon (SEQ ID NO:178)), and G6PD (GenBank Accession No. NM000402 (SEQ ID NO: 119); G6PD amplicon (SEQ ID NO: 184)). For lung panel\u2014HPRT1 (GenBank Accession No. NM000194 (SEQ ID NO: 118); amplicon\u2014HPRT1-amplicon (SEQ ID NO:181)), PBGD (GenBank Accession No. BC019323 (SEQ ID NO: 117); amplicon\u2014PBGD-amplicon (SEQ ID NO:178)), SDHA (GenBank Accession No. NM004168 (SEQ ID NO: 116); amplicon\u2014SDHA-amplicon (SEQ ID NO:175)) and Ubiquitin (GenBank Accession No. BC000449 (SEQ ID NO: 115); amplicon\u2014Ubiquitin-amplicon (SEQ ID NO: 172)). For ovary panel\u2014SDHA (GenBank Accession No. NM004168 (SEQ ID NO: 116); amplicon\u2014SDHA-amplicon (SEQ ID NO:175)), HPRT1 (GenBank Accession No. NM000194 (SEQ ID NO: 118); amplicon\u2014HPRT1-amplicon (SEQ ID NO:181)) and G6PD (GenBank Accession No. NM000402 (SEQ ID NO: 119); G6PD amplicon (SEQ ID NO: 184)). For normal panel\u2014SDHA (GenBank Accession No. NM004168 (SEQ ID NO: 116); amplicon\u2014SDHA-amplicon (SEQ ID NO:175)), Ubiquitin (GenBank Accession No. BC000449 (SEQ ID NO: 115); amplicon\u2014Ubiquitin-amplicon (SEQ ID NO: 172)), and TATA box (GenBank Accession No. NM003194 (SEQ ID NO: 114); TATA amplicon (SEQ ID NO: 169)). For blood panel\u2014HSB1L_HUMAN (Accession No. Q9Y450) (SEQ ID NO: 109), DHSA_HUMAN (SEQ ID NO: 110) (Accession No P31040), SFRS4_HUMAN (SEQ ID NO: 111) (Accession No Q08170) and SLC25A3 (Accession No Q7Z7N7) (SEQ ID NO: 112).","The sequences of the housekeeping genes measured in all the examples of blood panel were as follows:",{"@attributes":{"id":"p-0751","num":"0750"},"tables":{"@attributes":{"id":"TABLE-US-00015","num":"00015"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"HSB1L_HUMAN\u2003(SEQ\u2003ID\u2003NO:\u2003109)\u2003(Accession\u2003No.\u2003Q9Y450)"},{"entry":"T05337_seg30-34F1-Forward\u2003primer\u2003(SEQ\u2003ID\u2003NO:\u2003152):"},{"entry":"GCTCCAGGCCATAAGGACTTC"},{"entry":{}},{"entry":"T05337_seg30-34R1\u2003(SEQ\u2003ID\u2003NO:\u2003153)-Reverse\u2003primer:"},{"entry":"CAGCTTCAAACTCTCCCCTGC"},{"entry":{}},{"entry":"Amplicon\u2003(SEQ\u2003ID\u2003NO:\u2003154):"},{"entry":"GCTCCAGGCCATAAGGACTTCATTCCAAATATGATTACAGGAGCAGCCCA"},{"entry":{}},{"entry":"GGCGGATGTAGCTGTTTTAGTTGTAGATGCCAGCAGGGGAGAGTTTGAAG"},{"entry":{}},{"entry":"CTG"},{"entry":{}},{"entry":"DHSA_HUMAN\u2003(SEQ\u2003ID\u2003NO:\u2003110)\u2003(Accession\u2003No\u2003P31040)"},{"entry":"M78124_seg45-48F1\u2003(SEQ\u2003ID\u2003NO:\u2003155)-Forward\u2003primer:"},{"entry":"TTCCTTGCCAGGACCTAGAG"},{"entry":{}},{"entry":"M78124_seg45-48R1-Reverse\u2003primer\u2003(SEQ\u2003ID\u2003NO:\u2003156):"},{"entry":"CATAAACCTTTCGCCTTGAC"},{"entry":{}},{"entry":"Amplicon\u2003(SEQ\u2003ID\u2003NO:\u2003157):"},{"entry":"TTCCTTGCCAGGACCTAGAGTTTGTTCAGTTCCACCCCACAGGCATATAT"},{"entry":{}},{"entry":"GGTGCTGGTTGTCTCATTACGGAAGGATGTCGTGGAGAGGGAGGCATTCT"},{"entry":{}},{"entry":"CATTAACAGTCAAGGCGAAAGGTTTATG"},{"entry":{}},{"entry":"SFRS4_HUMAN\u2003(SEQ\u2003ID\u2003NO:\u2003111)\u2003(Accession\u2003No\u2003Q08170)"},{"entry":"HUMSRP75Aseg30-33F1\u2003(SEQ\u2003ID\u2003NO:\u2003158)-\u2003Forward\u2003"},{"entry":"primer:"},{"entry":"AATTTGTCAAGTCGGTGCAGC"},{"entry":{}},{"entry":"HUMSRP75Aseg30-33R1\u2003(SEQ\u2003ID\u2003NO:\u2003159)-\u2003Reverse\u2003"},{"entry":"primer:"},{"entry":"TCACCCCTTCATTTTTGCGT"},{"entry":{}},{"entry":"Amplicon\u2003(SEQ\u2003ID\u2003NO:\u2003160):"},{"entry":"AATTTGTCAAGTCGGTGCAGCTGGCAAGACCTAAAGGATTATATGCGTCA"},{"entry":{}},{"entry":"GGCAGGAGAAGTGACTTATGCAGATGCTCACAAGGGACGCAAAAATGAAG"},{"entry":{}},{"entry":"GGGTGA"},{"entry":{}},{"entry":"SLC25A3\u2003(Accession\u2003No\u2003Q7Z7N7)\u2003(SEQ\u2003ID\u2003NO:\u2003112)"},{"entry":"SSMPCPseg24-25-29F1-\u2003Forward\u2003primer\u2003"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003161):"},{"entry":"CCCAAAATGTATAAGGAAGAAGGC"},{"entry":{}},{"entry":"SSMPCPseg24-25-29R1-\u2003Reverse\u2003primer\u2003"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003162):"},{"entry":"TTCAAAGCAGGCGAACTTCA"},{"entry":{}},{"entry":"Amplicon\u2003(SEQ\u2003ID\u2003NO:\u2003163):"},{"entry":"CAGCCAGGTTATGCCAACACTTTGAGGGATGCAGCTCCCAAAATGTATAA"},{"entry":{}},{"entry":"GGAAGAAGGCCTAAAAGCATTCTACAAGGGGGTTGCTCCTCTCTGGATGA"},{"entry":{}},{"entry":"GACAGATACCATACACCATGATGAAGTTCGCCTGCTTTGA"}]}}}}},"The sequences of the housekeeping genes measured in all the examples on normal tissue samples panel were as follows:",{"@attributes":{"id":"p-0753","num":"0752"},"tables":{"@attributes":{"id":"TABLE-US-00016","num":"00016"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"TATA\u2003box\u2003(GenBank\u2003Accession\u2003No.\u2003NM_003194"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003114)),"},{"entry":"TATA\u2003box\u2003Forward\u2003primer\u2003(SEQ\u2003ID\u2003NO:\u2003167):"},{"entry":"CGGTTTGCTGCGGTAATCAT"},{"entry":{}},{"entry":"TATA\u2003box\u2003Reverse\u2003primer\u2003(SEQ\u2003ID\u2003NO:\u2003168):"},{"entry":"TTTCTTGCTGCCAGTCTGGAC"},{"entry":{}},{"entry":"TATA\u2003box-amplicon\u2003(SEQ\u2003ID\u2003NO:\u2003169):"},{"entry":"CGGTTTGCTGCGGTAATCATGAGGATAAGAGAGCCACGAACCACG"},{"entry":{}},{"entry":"GCACTGATTTTCAGTTCTGGGAAAATGGTGTGCACAGGAGCCAAG"},{"entry":{}},{"entry":"AGTGAAGAACAGTCCAGACTGGCAGCAAGAAA"},{"entry":{}},{"entry":"Ubiquitin\u2003(GenBank\u2003Accession\u2003No.\u2003BC000449"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003115))"},{"entry":"Ubiquitn\u2003Forward\u2003primer\u2003(SEQ\u2003ID\u2003NO:\u2003170):"},{"entry":"ATTTGGGTCGCGGTTCTTG"},{"entry":{}},{"entry":"Ubiquitin\u2003Reverse\u2003primer\u2003(SEQ\u2003ID\u2003NO:\u2003171):"},{"entry":"TGCCTTGACATTCTCGATGGT"},{"entry":{}},{"entry":"Ubiquitin-amplicon\u2003(SEQ\u2003ID\u2003NO:\u2003172)"},{"entry":"ATTTGGGTCGCGGTTCTTGTTTGTGGATCGCTGTGATCGTCACTT"},{"entry":{}},{"entry":"GACAATGCAGATCTTCGTGAAGACTCTGACTGGTAAGACCATCAC"},{"entry":{}},{"entry":"CCTCGAGGTTGAGCCCAGTGACACCATCGAGAATGTCAAGGCA"},{"entry":{}},{"entry":"SDHA\u2003(GenBank\u2003Accession\u2003No.\u2003NM_004168"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003116))"},{"entry":"SDHA\u2003Forward\u2003primer\u2003(SEQ\u2003ID\u2003NO:\u2003173):"},{"entry":"TGGGAACAAGAGGGCATCTG"},{"entry":{}},{"entry":"SDHA\u2003Reverse\u2003primer\u2003(SEQ\u2003ID\u2003NO:\u2003174):"},{"entry":"CCACCACTGCATCAAATTCATG"},{"entry":{}},{"entry":"SDHA-amplicon\u2003(SEQ\u2003ID\u2003NO:\u2003175):"},{"entry":"TGGGAACAAGAGGGCATCTGCTAAAGTTTCAGATTCCATTTCTGC"},{"entry":{}},{"entry":"TCAGTATCCAGTAGTGGATCATGAATTTGATGCAGTGGTGG"}]}}}}},"The sequences for primers and amplicons of the housekeeping genes measured in all the cancer examples are listed below. For colon panel\u2014HPRT1, PBGD and G6PD were used. For lung panel\u2014PBGD, HPRT1, Ubiquitin and SDHA were used. For ovary panel\u2014HPRT1, SDHA and G6PD were used.",{"@attributes":{"id":"p-0755","num":"0754"},"tables":{"@attributes":{"id":"TABLE-US-00017","num":"00017"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"SDHA\u2003(GenBank\u2003Accession\u2003No.\u2003NM_004168"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003116):"},{"entry":"SDHA\u2003Forward\u2003primer\u2003(SEQ\u2003ID\u2003NO:\u2003173):"},{"entry":"TGGGAACAAGAGGGCATCTG"},{"entry":{}},{"entry":"SDHA\u2003Reverse\u2003primer\u2003(SEQ\u2003ID\u2003NO:\u2003174):"},{"entry":"CCACCACTGCATCAAATTCATG"},{"entry":{}},{"entry":"SDHA-amplicon\u2003(SEQ\u2003ID\u2003NO:\u2003175):"},{"entry":"TGGGAACAAGAGGGCATCTGCTAAAGTTTCAGATTCCATTTCTGCTC"},{"entry":{}},{"entry":"AGTATCCAGTAGTGGATCATGAATTTGATGCAGTGGTGG"},{"entry":{}},{"entry":"PBGD\u2003(GenBank\u2003Accession\u2003No.\u2003BC019323"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003117)),"},{"entry":"PBGD\u2003Forward\u2003primer\u2003(SEQ\u2003ID\u2003NO:\u2003176):"},{"entry":"TGAGAGTGATTCGCGTGGG"},{"entry":{}},{"entry":"PBGD\u2003Reverse\u2003primer\u2003(SEQ\u2003ID\u2003NO:\u2003177):"},{"entry":"CCAGGGTACGAGGCTTTCAAT"},{"entry":{}},{"entry":"PBGD-amplicon\u2003(SEQ\u2003ID\u2003NO:\u2003178):"},{"entry":"TGAGAGTGATTCGCGTGGGTACCCGCAAGAGCCAGCTTGCTCGCAT"},{"entry":{}},{"entry":"ACAGACGGACAGTGTGGTGGCAACATTGAAAGCCTCGTACCCTGG"},{"entry":{}},{"entry":"HPRT1\u2003(GenBank\u2003Accession\u2003No.\u2003NM_000194"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003118)),"},{"entry":"HPRT1\u2003Forward\u2003primer\u2003(SEQ\u2003ID\u2003NO:\u2003179):"},{"entry":"TGACACTGGCAAAACAATGCA"},{"entry":{}},{"entry":"HPRT1\u2003Reverse\u2003primer\u2003(SEQ\u2003ID\u2003NO:\u2003180):"},{"entry":"GGTCCTTTTCACCAGCAAGCT"},{"entry":{}},{"entry":"HPRT1-amplicon\u2003(SEQ\u2003ID\u2003NO:\u2003181):"},{"entry":"TGACACTGGCAAAACAATGCAGACTTTGCTTTCCTTGGTCAGGCAGT"},{"entry":{}},{"entry":"ATAATCCAAAGATGGTCAAGGTCGCAAGCTTGCTGGTGAAAAGGACC"},{"entry":{}},{"entry":"G6PD\u2003(GenBank\u2003Accession\u2003No.\u2003NM_000402"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003119))"},{"entry":"G6PD\u2003Forward\u2003primer\u2003(SEQ\u2003ID\u2003NO:\u2003182):"},{"entry":"gaggccgtcaccaagaacat"},{"entry":{}},{"entry":"G6PD\u2003Reverse\u2003primer\u2003(SEQ\u2003ID\u2003NO:\u2003183):"},{"entry":"ggacagccggtcagagctc"},{"entry":{}},{"entry":"G6PD-amplicon\u2003(SEQ\u2003ID\u2003NO:\u2003184):"},{"entry":"gaggccgtcaccaagaacattcacgagtcctgcatgagccagatagg"},{"entry":{}},{"entry":"ctggaaccgcatcatcgtggagaagcccttcgggagggacctgcaga"},{"entry":"gctctgaccggctgtcc"},{"entry":{}},{"entry":"Ubiquitin\u2003(GenBank\u2003Accession\u2003No.\u2003BC000449"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003115))"},{"entry":"Ubiquitin\u2003Forward\u2003primer\u2003(SEQ\u2003ID\u2003NO:\u2003170):"},{"entry":"ATTTGGGTCGCGGTTCTTG"},{"entry":{}},{"entry":"Ubiquitin\u2003Reverse\u2003primer\u2003(SEQ\u2003ID\u2003NO:\u2003171):"},{"entry":"TGCCTTGACATTCTCGATGGT"},{"entry":{}},{"entry":"Ubiquitin\u2003Amplicon\u2003(SEQ\u2003ID\u2003NO:\u2003172):"},{"entry":"ATTTGGGTCGCGGTTCTTGTTTGTGGATCGCTGTGATCGTCACTTGA"},{"entry":{}},{"entry":"CAATGCAGATCTTCGTGAAGACTCTGACTGGTAAGACCATCACCCTC"},{"entry":{}},{"entry":"GAGGTTGAGCCCAGTGACACCATCGAGAATGTCAAGGCA"}]}}}}},"Another methodology used to predict the expression pattern of the proteins of the invention was MED discovery engine:","MED is a platform for collection of public gene-expression data, normalization, annotation and performance of various queries. Expression data from the most widely used Affymetrix microarrays is downloaded from the Gene Expression Omnibus (GEO\u2014www.ncbi.nlm.nih.gov\/GEO). Data is multiplicatively normalized by setting the 95 percentile to a constant value (normalized expression=1200), and noise is filtered by setting the lower 30% to 0. Experiments are annotated, first automatically, and then manually, to identify tissue and condition, and chips are grouped according to this annotation, and cross verification of this grouping by comparing the overall expression pattern of the genes of each chip to the overall average expression pattern of the genes in this group. Each probeset in each group is assigned an expression value which is the median of the expressions of that probeset in all chips included in the group. The vector of expression of all probesets within a certain group is the virtual chip of that group, and the collection of all such virtual chips is a virtual panel. The panel (or sub-panels) can be queried to identify probesets with a required behavior (e.g. specific expression in a sub-set of tissues, or differential expression between disease and healthy tissues). These probesets are linked to LEADS contigs and to RefSeqs (http:\/\/www.ncbi.nlm.nih.gov\/RefSeq\/) by probe-level mapping, for further analysis.","The Affymetrix platforms that are downloaded are HG-U95A and the HG-U133 family (A,B, A2.0 and PLUS 2.0). Than three virtual panels were created: U95 and U133 Plus 2.0, based on the corresponding platforms, and U133 which uses the set of common probesets for HG-U133A, HG-U133A2.0 and HG-U133 PLUS 2.0+.","The results of the MED discovery engine are presented in scatter plots. The scatter plot is a compact representation of a given panel (collection of groups). The y-axis is the (normalized) expression and the x-axis describes the groups in the panel. For each group, the median expression is represented by a solid marker, and the expression values of the different chips in the group are represented by small dashes (\u201c-\u201d). The groups are ordered and marked as follows\u2014\u201cOther\u201d groups (e.g. benign, non-cancer diseases, etc.) with a triangle, Treated cells with a square, Normal with a circle, Matched with a cross, and Cancer with a diamond. The number of chips in each group is also written adjacent to it's name.","The present invention relates to VSIG1 polypeptides, novel splice variants and diagnostics and therapeutics based thereon.","According to the present invention, Cluster AI581519 (internal ID 72756422) features 10 transcripts and 2 segments of interest, the names for which are given in Tables 6 and 7, respectively. The selected protein variants are given in table 8.",{"@attributes":{"id":"p-0762","num":"0761"},"tables":{"@attributes":{"id":"TABLE-US-00018","num":"00018"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 6"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Transcripts of interest"},{"entry":"Transcript Name"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"161pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"AI581519_T0 (SEQ ID NO: 1)"]},{"entry":[{},"AI581519_T1 (SEQ ID NO: 2)"]},{"entry":[{},"AI581519_T2 (SEQ ID NO: 3)"]},{"entry":[{},"AI581519_T3 (SEQ ID NO: 4)"]},{"entry":[{},"AI581519_T4 (SEQ ID NO: 5)"]},{"entry":[{},"AI581519_T5 (SEQ ID NO: 6)"]},{"entry":[{},"AI581519_T6 (SEQ ID NO: 7)"]},{"entry":[{},"AI581519_T8 (SEQ ID NO: 8)"]},{"entry":[{},"AI581519_T10 (SEQ ID NO: 9)"]},{"entry":[{},"AI581519_T11 (SEQ ID NO: 10)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0763","num":"0762"},"tables":{"@attributes":{"id":"TABLE-US-00019","num":"00019"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 7"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segments of interest"},{"entry":"Segment Name"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"161pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"AI581519_N7 (SEQ ID NO: 120)"]},{"entry":[{},"AI581519_N9 (SEQ ID NO: 121)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0764","num":"0763"},"tables":{"@attributes":{"id":"TABLE-US-00020","num":"00020"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 8"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Proteins of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"147pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Protein Name","Corresponding Transcripts"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["AI581519_P3 (SEQ","AI581519_T0 (SEQ ID NO: 1); AI581519_T1"]},{"entry":["ID NO: 11)","(SEQ ID NO: 2); AI581519_T2 (SEQ ID NO: 3);"]},{"entry":[{},"AI581519_T3 (SEQ ID NO: 4); AI581519_T4"]},{"entry":[{},"(SEQ ID NO: 5)"]},{"entry":["AI581519_P4 (SEQ","AI581519_T5 (SEQ ID NO: 6)"]},{"entry":"ID NO: 12)"},{"entry":["AI581519_P5 (SEQ","AI581519_T6 (SEQ ID NO: 7)"]},{"entry":"ID NO: 13)"},{"entry":["AI581519_P7 (SEQ","AI581519_T8 (SEQ ID NO: 8)"]},{"entry":"ID NO: 14)"},{"entry":["AI581519_P9 (SEQ","AI581519_T10 (SEQ ID NO: 9)"]},{"entry":"ID NO: 15)"},{"entry":["AI581519_P10 (SEQ","AI581519_T11 (SEQ ID NO: 10)"]},{"entry":"ID NO: 16)"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"These sequences are variants of the known protein V-set and immunoglobulin domain containing 1 (RefSeq accession identifier NP872413, synonyms: RP5-889N15.1, 1700062D20Rik, GPA34, MGC44287, dJ889N15.1), referred to herein as the previously known protein.","VSIG1 is a V-set and immunoglobulin domain containing 1 protein also known as glycoprotein A34 (GPA34). This gene was originally identified as a transcript encoding a protein with similarity to the glycoprotein A33 (GPA33), a colon cancer antigen (Scanlan et al. Cancer Immunotherapy 6:2 2006), that has 32% identity to GPA33. The authors showed that A34 mRNA and protein expression is highly tissue-restricted, as it is expressed predominantly in stomach and testis. A34 mRNA and protein expression was also detected in gastric cancers, esophageal carcinomas, and ovarian cancers. In their studies they did not detect A34 in lung, breast or colon carcinomas (Scanlan et al. 2006, Cancer Immunity 6: 2).","A known wild type VSIG1 nucleic acid sequence has been reported in various patent and non-patent literature references. For example, the sequence of AI581519_P3 (SEQ ID NO:11) is disclosed in WO2004037999, referred there as glycoprotein A34 (GPA34). This PCT application contains the sequence for AI581519_P3 (SEQ ID NO:11) which encodes the A34 antigen disclosed herein. The corresponding antigen A34 is indicated to be expressed in some tested stomach cancers (29%), esophageal (63%) and to be expressed to a much lesser number extent (9%) on tested ovarian cancers. The authors suggest that this antigen may be used for therapy and may be a suitable target for antibody based cancer therapies.","WO9926972 discloses that this antigenic protein may also exhibit immune stimulating or immune suppressing activity such as for the treatment of various immune deficiencies and disorders (including severe combined immunodeficiency (SCID)), e.g., in regulating (up or down) growth and proliferation of T and\/or B lymphocytes, as well as effecting the cytolytic activity of NK cells and other cell populations.","In addition this same protein sequence, as depicted in AI581519_P3 (SEQ ID NO:11) herein, is mentioned in WO9960020 and US2002193567 wherein it is identified as Human secreted protein #62.","A sequence homologous to VSIG1 variant as depicted in AI581519_P4 (SEQ ID NO:12) herein, (containing 2 mismatches corresponding to known SNPs) is disclosed in WO2003027228 application, which purportedly discloses an extensive list of different alleged differentially expressed sequences.","Further, a sequence closely related to AI581519_P5 (SEQ ID NO:13), (containing 2 mismatches corresponding to known SNPs) is disclosed in PCT application WO2004100774, which also teaches many other purported differentially expressed sequences.","Still further, a sequence closely related to AI581519_P7 (SEQ ID NO:14), (containing 1 mismatch corresponding to known SNP) is disclosed in PCT application WO2004048550 which similarly contains an extensive listing of alleged differentially expressed sequences.","According to the present invention, VSIG1 is predicted to be a novel B7 member, based on the presence of an IgV and an IgC2 domain. A large portion of proteins having one domain of each Ig subtype are co-stimulatory molecules. Like other known B7 members, VSIG1 is also a type I membrane protein. In the present invention several alternative spliced variants of VSIG1 were identified, as described below, containing a unique region within the ectodomain. The new variants of VSIG1 were demonstrated in the present invention to be overexpressed in lung adenocarcinoma and ovarian cancer.","MED discovery engine described in Example 1 herein, was used to assess the expression of VSIG1 transcripts. Expression data for Affymetrix probe sets 234370_at representing the VSIG1 gene data is shown in . As evident from the scatter plot, presented in , the expression of VSIG1 transcripts detectable with the above probe sets was higher in lung cancer compared to normal lung samples.","As noted above, cluster AI581519 features 10 transcripts, which were listed in Table 6 above. These transcripts encode for proteins which are variants of protein V-set and immunoglobulin domain containing 1. A description of each variant protein according to the present invention is now provided.","Variant protein AI581519_P3 (SEQ ID NO:11) according to the present invention has an amino acid sequence encoded by transcripts AI581519_T0 (SEQ ID NO:1), AI581519_T1 (SEQ ID NO:2), AI581519_T2 (SEQ ID NO:3), AI581519_T3 (SEQ ID NO:4) and AI581519_T4 (SEQ ID NO:5).","The localization of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: membrane.","Variant protein AI581519_P3 (SEQ ID NO:11) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 9, (given according to their positions on the amino acid sequence, with the alternative amino acids listed).",{"@attributes":{"id":"p-0779","num":"0778"},"tables":{"@attributes":{"id":"TABLE-US-00021","num":"00021"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 9"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["23","Q \u2212> R"]},{"entry":["35","V \u2212>"]},{"entry":["51","S \u2212> F"]},{"entry":["90","I \u2212> V"]},{"entry":["127","Q \u2212>"]},{"entry":["146","S \u2212> R"]},{"entry":["176","K \u2212>"]},{"entry":["176","K \u2212> E"]},{"entry":["181","D \u2212> G"]},{"entry":["181","D \u2212> V"]},{"entry":["189","F \u2212> L"]},{"entry":["195","I \u2212> T"]},{"entry":["196","L \u2212>"]},{"entry":["202","T \u2212>"]},{"entry":["280","T \u2212>"]},{"entry":["288","S \u2212> G"]},{"entry":["322","P \u2212>"]},{"entry":["344","L \u2212> P"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The variant protein has the following domains, as determined by using InterPro. The domains are described in Table 10:",{"@attributes":{"id":"p-0781","num":"0780"},"tables":{"@attributes":{"id":"TABLE-US-00022","num":"00022"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 10"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"InterPro domains"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Analysis",{}]},{"entry":[{},"Domain description","type","Positions on protein"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"Immunoglobulin C2 type","HMMSmart","34-123, 152-218"]},{"entry":[{},"Immunoglobulin-like","ProfileScan","36-120, 140-227"]},{"entry":[{},"Immunoglobulin-like","HMMPfam","36-118, 154-213"]},{"entry":[{},"Immunoglobulin subtype","HMMSmart","28-137, 146-229"]},{"entry":[{},"Myelin P0 protein","FPrintScan","\u200235-59, 110-139"]},{"entry":[{},"Immunoglobulin V-set","HMMPfam","21-137"]},{"entry":[{},"Immunoglobulin V-type","HMMSmart","38-118"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},"Variant protein AI581519_P3 (SEQ ID NO:11) is encoded by the following transcripts: AI581519_T0 (SEQ ID NO:1), AI581519_T1 (SEQ ID NO:2), AI581519_T2 (SEQ ID NO:3), AI581519_T3 (SEQ ID NO:4) and AI581519_T4 (SEQ ID NO:5).","The coding portion of transcript AI581519_TO (SEQ ID NO:1) starts at position 171 and ends at position 1331. The transcript also has the following SNPs as listed in Table 11 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed (SEQ ID NO:11)).",{"@attributes":{"id":"p-0784","num":"0783"},"tables":{"@attributes":{"id":"TABLE-US-00023","num":"00023"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 11"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","238, 438, 696, 712, 1031, 1032, 2314, 2471"]},{"entry":[{},"T \u2212>","275, 757"]},{"entry":[{},"C \u2212> T","322, 452"]},{"entry":[{},"C \u2212> A","449, 608"]},{"entry":[{},"C \u2212>","549"]},{"entry":[{},"C \u2212> G","608"]},{"entry":[{},"A \u2212>","696, 776, 1010, 1136"]},{"entry":[{},"A \u2212> T","712"]},{"entry":[{},"T \u2212> C","735, 754, 1201"]},{"entry":[{},"G \u2212> A","1679, 1800, 1867"]},{"entry":[{},"T \u2212> A","2260\u2002"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The coding portion of transcript AI581519_T1 (SEQ ID NO:2) starts at position 171 and ends at position 1331. The transcript also has the following SNPs as listed in Table 12 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-0786","num":"0785"},"tables":{"@attributes":{"id":"TABLE-US-00024","num":"00024"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 12"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","238, 438, 696, 712, 1031, 1032, 2314, 2471"]},{"entry":[{},"T \u2212>","275, 757"]},{"entry":[{},"C \u2212> T","322, 452"]},{"entry":[{},"C \u2212> A","449, 608"]},{"entry":[{},"C \u2212>","549"]},{"entry":[{},"C \u2212> G","608"]},{"entry":[{},"A \u2212>","696, 776, 1010, 1136"]},{"entry":[{},"A \u2212> T","712"]},{"entry":[{},"T \u2212> C","735, 754, 1201"]},{"entry":[{},"G \u2212> A","1679, 1800, 1867"]},{"entry":[{},"T \u2212> A","2260\u2002"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The coding portion of transcript AI581519_T2 (SEQ ID NO:3) starts at position 171 and ends at position 1331. The transcript also has the following SNPs as listed in Table 13 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-0788","num":"0787"},"tables":{"@attributes":{"id":"TABLE-US-00025","num":"00025"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 13"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"147pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","238, 438, 696, 712, 1031, 1032, 1782"]},{"entry":[{},"T \u2212>","275, 757"]},{"entry":[{},"C \u2212> T","322, 452"]},{"entry":[{},"C \u2212> A","449, 608"]},{"entry":[{},"C \u2212>","549"]},{"entry":[{},"C \u2212> G","608"]},{"entry":[{},"A \u2212>","696, 776, 1010, 1136"]},{"entry":[{},"A \u2212> T","712"]},{"entry":[{},"T \u2212> C","735, 754, 1201"]},{"entry":[{},"G \u2212> A","1679\u2002"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The coding portion of transcript AI581519_T3 (SEQ ID NO:4) starts at position 171 and ends at position 1331. The transcript also has the following SNPs as listed in Table 14 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-0790","num":"0789"},"tables":{"@attributes":{"id":"TABLE-US-00026","num":"00026"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 14"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"147pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","238, 438, 696, 712, 1031, 1032, 1702"]},{"entry":[{},"T \u2212>","275, 757"]},{"entry":[{},"C \u2212> T","322, 452"]},{"entry":[{},"C \u2212> A","449, 608"]},{"entry":[{},"C \u2212>","549"]},{"entry":[{},"C \u2212> G","608"]},{"entry":[{},"A \u2212>","696, 776, 1010, 1136"]},{"entry":[{},"A \u2212> T","712"]},{"entry":[{},"T \u2212> C","735, 754, 1201"]},{"entry":[{},"G \u2212> A","1679\u2002"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The coding portion of transcript AI581519_T4 (SEQ ID NO:5) starts at position 171 and ends at position 1331. The transcript also has the following SNPs as listed in Table 15 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-0792","num":"0791"},"tables":{"@attributes":{"id":"TABLE-US-00027","num":"00027"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 15"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"140pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","238, 438, 696, 712, 1031, 1032"]},{"entry":[{},"T \u2212>","275, 757"]},{"entry":[{},"C \u2212> T","322, 452"]},{"entry":[{},"C \u2212> A","449, 608"]},{"entry":[{},"C \u2212>","549"]},{"entry":[{},"C \u2212> G","608"]},{"entry":[{},"A \u2212>","696, 776, 1010, 1136"]},{"entry":[{},"A \u2212> T","712"]},{"entry":[{},"T \u2212> C","735, 754, 1201"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"Variant protein AI581519_P4 (SEQ ID NO:12) according to the present invention has an amino acid sequence encoded by transcript AI581519_T5 (SEQ ID NO:6). Alignments to previously published protein sequences are shown in . A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","2. Comparison report between AI581519_P4 (SEQ ID NO:12) and known proteins NP872413 (SEQ ID NO: 11) and Q86XK7_HUMAN ():","A. An isolated chimeric polypeptide encoding for AI581519_P4 (SEQ ID NO:12), comprising a first amino acid sequence being at least 90% homologous to MVFAFWKVFLILSCLAGQVSVVQVTIPDGFVNVTVGSNVTLICIYTTTVASREQLS IQWSFFHKKEMEPIS corresponding to amino acids 1-71 of known proteins NP872413 and Q86XK7_HUMAN (SEQ ID NO: 11), which also corresponds to amino acids 1-71 of AI581519_P4 (SEQ ID NO:12), a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95, 96, 97, 98 or 99% homologous to a polypeptide having the sequence HSSCLSTEGMEEKAVGQCLKMTHVRDARGRCSWTSE (SEQ ID NO: 284) corresponding to amino acids 72-107 of AI581519_P4 (SEQ ID NO:12), and a third amino acid sequence being at least 90% homologous to IYFSQGGQAVAIGQFKDRITGSNDPGNASITISHMQPADSGIYICDVNNPPDFLG QNQGILNVSVLVKPSKPLCSVQGRPETGHTISLSCLSALGTPSPVYYWHKLEGRDI VPVKENFNPTTGILVIGNLTNFEQGYYQCTAINRLGNSSCEIDLTSSHPEVGIIVG ALIGSLVGAAIIISVVCFARNKAKAKAKERNSKTIAELEPMTKINPRGESEAMPRE DATQLEVTLPSSIHETGPDTIQEPDYEPKPTQEPAPEPAPGSEPMAVPDLDIELELE PETQSELEPEPEPEPESEPGVVVEPLSEDEKGVVKA corresponding to amino acids 72-387 of known proteins NP872413 and Q86XK7_HUMAN (SEQ ID NO: 11), which also corresponds to amino acids 108-423 of AI581519_P4 (SEQ ID NO:12), wherein said first amino acid sequence, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","C. An isolated polypeptide encoding for an edge portion of AI581519_P4 (SEQ ID NO:12), comprising an amino acid sequence being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95, 96, 97, 98 or 99% homologous to the sequence HSSCLSTEGMEEKAVGQCLKMTHVRDARGRCSWTSE (SEQ ID NO: 284) of AI581519_P4 (SEQ ID NO:12).","The localization of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: membrane.","Variant protein AI581519_P4 (SEQ ID NO:12) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 16, (given according to their positions on the amino acid sequence, with the alternative amino acids listed (SEQ ID NO:12)).",{"@attributes":{"id":"p-0799","num":"0798"},"tables":{"@attributes":{"id":"TABLE-US-00028","num":"00028"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 16"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["23","Q \u2212> R"]},{"entry":["35","V \u2212>"]},{"entry":["51","S \u2212> F"]},{"entry":["87","G \u2212> S"]},{"entry":["95","V \u2212> A"]},{"entry":["126","I \u2212> V"]},{"entry":["163","Q \u2212>"]},{"entry":["182","S \u2212> R"]},{"entry":["212","K \u2212>"]},{"entry":["212","K \u2212> E"]},{"entry":["217","D \u2212> G"]},{"entry":["217","D \u2212> V"]},{"entry":["225","F \u2212> L"]},{"entry":["231","I \u2212> T"]},{"entry":["232","L \u2212>"]},{"entry":["238","T \u2212>"]},{"entry":["316","T \u2212>"]},{"entry":["324","S \u2212> G"]},{"entry":["358","P \u2212>"]},{"entry":["380","L \u2212> P"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The variant protein has the following domains, as determined by using InterPro. The domains are described in Table 17:",{"@attributes":{"id":"p-0801","num":"0800"},"tables":{"@attributes":{"id":"TABLE-US-00029","num":"00029"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 17"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"InterPro domains"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Analysis",{}]},{"entry":[{},"Domain description","type","Positions on protein"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"Immunoglobulin C2 type","HMMSmart","34-159, 188-254"]},{"entry":[{},"Immunoglobulin subtype","HMMSmart","28-173, 182-265"]},{"entry":[{},"Immunoglobulin-like","ProfileScan","36-156, 176-263"]},{"entry":[{},"Immunoglobulin V-set","HMMPfam","21-173"]},{"entry":[{},"Immunoglobulin V-type","HMMSmart","38-154"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},"Variant protein AI581519_P4 (SEQ ID NO:12) is encoded by the AI581519_T5 (SEQ ID NO:6), for which the coding portion starts at position 171 and ends at position 1439. The transcript also has the following SNPs as listed in Table 18 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-0803","num":"0802"},"tables":{"@attributes":{"id":"TABLE-US-00030","num":"00030"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 18"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","238, 546, 804, 820, 1139, 1140, 2422, 2579"]},{"entry":[{},"T \u2212>","275, 865"]},{"entry":[{},"C \u2212> T","322, 560"]},{"entry":[{},"G \u2212> A","429, 1787, 1908, 1975"]},{"entry":[{},"T \u2212> C","454, 843, 862, 1309"]},{"entry":[{},"C \u2212> A","557, 716"]},{"entry":[{},"C \u2212>","657"]},{"entry":[{},"C \u2212> G","716"]},{"entry":[{},"A \u2212>","804, 884, 1118, 1244"]},{"entry":[{},"A \u2212> T","820"]},{"entry":[{},"T \u2212> A","2368\u2002"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"Variant protein AI581519_P5 (SEQ ID NO:13) according to the present invention has an amino acid sequence encoded by transcript AI581519_T6 (SEQ ID NO:7). Alignments to previously published protein sequences are shown in . A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","2. Comparison report between AI581519_P5 (SEQ ID NO:13) and known proteins NP872413 and Q86XK7_HUMAN (SEQ ID NO: 11) ():","A. An isolated chimeric polypeptide encoding for AI581519_P5 (SEQ ID NO:13), comprising a first amino acid sequence being at least 90% homologous to MVFAFWKVFLILSCLAGQVSVVQVTIPDGFVNVTVGSNVTLICIYTTTVASREQLS IQWSFFHKKEMEPIS corresponding to amino acids 1-71 of known proteins NP872413 and Q86XK7_HUMAN (SEQ ID NO: 11), which also corresponds to amino acids 1-71 of AI581519_P5 (SEQ ID NO:13), a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95, 96, 97, 98 or 99% homologous to a polypeptide having the sequence HSSCLSTEGMEEKAVGQCLKMTHVRDARGRCSWTSESPWEEGKWPDVEAVKG TLDGQQAELQ (SEQ ID NO: 285) corresponding to amino acids 72-133 of AI581519_P5 (SEQ ID NO:13), and a third amino acid sequence being at least 90% homologous to IYFSQGGQAVAIGQFKDRITGSNDPGNASITISHMQPADSGIYICDVNNPPDFLG QNQGILNVSVLVKPSKPLCSVQGRPETGHTISLSCLSALGTPSPVYYWHKLEGRDI VPVKENFNPTTGILVIGNLTNFEQGYYQCTAINRLGNSSCEIDLTSSHPEVGIIVG ALIGSLVGAAIIISVVCFARNKAKAKAKERNSKTIAELEPMTKINPRGESEAMPRE DATQLEVTLPSSIHETGPDTIQEPDYEPKPTQEPAPEPAPGSEPMAVPDLDIELELE PETQSELEPEPEPEPESEPGVVVEPLSEDEKGVVKA corresponding to amino acids 72-387 of known proteins NP872413 and Q86XK7_HUMAN (SEQ ID NO: 11), which also corresponds to amino acids 134-449 of AI581519_P5 (SEQ ID NO:13), wherein said first amino acid sequence, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","C. An isolated polypeptide encoding for an edge portion of AI581519_P5 (SEQ ID NO:13), comprising an amino acid sequence being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95, 96, 97, 98 or 99% homologous to the sequence HSSCLSTEGMEEKAVGQCLKMTHVRDARGRCSWTSESPWEEGKWPDVEAVKG TLDGQQAELQ (SEQ ID NO: 285) of AI581519_P5 (SEQ ID NO:13).","The localization of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: membrane.","Variant protein AI581519_P5 (SEQ ID NO:13) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 19, (given according to their positions on the amino acid sequence, with the alternative amino acids listed (SEQ ID NO:13)).",{"@attributes":{"id":"p-0810","num":"0809"},"tables":{"@attributes":{"id":"TABLE-US-00031","num":"00031"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 19"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["23","Q \u2212> R"]},{"entry":["35","V \u2212>"]},{"entry":["51","S \u2212> F"]},{"entry":["87","G \u2212> S"]},{"entry":["95","V \u2212> A"]},{"entry":["152","I \u2212> V"]},{"entry":["189","Q \u2212>"]},{"entry":["208","S \u2212> R"]},{"entry":["238","K \u2212>"]},{"entry":["238","K \u2212> E"]},{"entry":["243","D \u2212> G"]},{"entry":["243","D \u2212> V"]},{"entry":["251","F \u2212> L"]},{"entry":["257","I \u2212> T"]},{"entry":["258","L \u2212>"]},{"entry":["264","T \u2212>"]},{"entry":["342","T \u2212>"]},{"entry":["350","S \u2212> G"]},{"entry":["384","P \u2212>"]},{"entry":["406","L \u2212> P"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The variant protein has the following domains, as determined by using InterPro. The domains are described in Table 20:",{"@attributes":{"id":"p-0812","num":"0811"},"tables":{"@attributes":{"id":"TABLE-US-00032","num":"00032"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 20"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"InterPro domains"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Analysis",{}]},{"entry":[{},"Domain description","type","Positions on protein"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"Immunoglobulin subtype","HMMSmart","28-199, 208-291"]},{"entry":[{},"Immunoglobulin C2 type","HMMSmart","34-185, 214-280"]},{"entry":[{},"Immunoglobulin V-set","HMMPfam","\u200221-199"]},{"entry":[{},"Immunoglobulin-like","ProfileScan","202-289"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},"Variant protein AI581519_P5 (SEQ ID NO:13) is encoded by the following transcript AI581519_T6 (SEQ ID NO:7), for which the coding portion starts at position 171 and ends at position 1517. The transcript also has the following SNPs as listed in Table 21 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-0814","num":"0813"},"tables":{"@attributes":{"id":"TABLE-US-00033","num":"00033"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 21"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","238, 624, 882, 898, 1217, 1218, 2500, 2657"]},{"entry":[{},"T \u2212>","275, 943"]},{"entry":[{},"C \u2212> T","322, 638"]},{"entry":[{},"G \u2212> A","429, 1865, 1986, 2053"]},{"entry":[{},"T \u2212> C","454, 921, 940, 1387"]},{"entry":[{},"C \u2212> A","635, 794"]},{"entry":[{},"C \u2212>","735"]},{"entry":[{},"C \u2212> G","794"]},{"entry":[{},"A \u2212>","882, 962, 1196, 1322"]},{"entry":[{},"A \u2212> T","898"]},{"entry":[{},"T \u2212> A","2446\u2002"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"Variant protein AI581519_P7 (SEQ ID NO:14) according to the present invention is encoded by transcript AI581519_T8 (SEQ ID NO:8). Alignments to one or more previously published protein sequences are shown in . A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","2. Comparison report between AI581519_P7 (SEQ ID NO:14) and known proteins NP872413 and Q86XK7_HUMAN (SEQ ID NO: 11) ():","A. An isolated chimeric polypeptide encoding for AI581519_P7 (SEQ ID NO:14), comprising a first amino acid sequence being at least 90% homologous to MVFAFWKVFLILSCLAGQVSVVQVTIPDGFVNVTVGSNVTLICIYTTTVASREQLS IQWSFFHKKEMEPISIYFSQGGQAVAIGQFKDRITGSNDP corresponding to amino acids 1-96 of known proteins NP872413 and Q86XK7_HUMAN (SEQ ID NO: 11), which also corresponds to amino acids 1-96 of AI581519_P7 (SEQ ID NO:14), and a second amino acid sequence being at least 90% homologous to VKPSKPLCSVQGRPETGHTISLSCLSALGTPSPVYYWHKLEGRDIVPVKENFNPTT GILVIGNLTNFEQGYYQCTAINRLGNSSCEIDLTSSHPEVGIIVGALIGSLVGAAIII SVVCFARNKAKAKAKERNSKTIAELEPMTKINPRGESEAMPREDATQLEVTLPSS IHETGPDTIQEPDYEPKPTQEPAPEPAPGSEPMAVPDLDIELELEPETQSELEPEPE PEPESEPGVVVEPLSEDEKGVVKA corresponding to amino acids 138-387 of known proteins NP872413 and Q86XK7_HUMAN (SEQ ID NO: 11), which also corresponds to amino acids 97-446 of AI581519_P7 (SEQ ID NO:14), wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","C. An isolated chimeric polypeptide encoding for an edge portion of AI581519_P7 (SEQ ID NO:14), comprising a polypeptide having a length \u201cn\u201d, wherein n is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least two amino acids comprise PV, having a structure as follows: a sequence starting from any of amino acid numbers 96\u2212x to 96; and ending at any of amino acid numbers 97+((n\u22122)\u2212x), in which x varies from 0 to n\u22122.","The localization of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: membrane.","Variant protein AI581519_P7 (SEQ ID NO:14) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 22, (given according to their positions on the amino acid sequence, with the alternative amino acids listed (SEQ ID NO:14)).",{"@attributes":{"id":"p-0821","num":"0820"},"tables":{"@attributes":{"id":"TABLE-US-00034","num":"00034"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 22"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["23","Q \u2212> R"]},{"entry":["35","V \u2212>"]},{"entry":["51","S \u2212> F"]},{"entry":["90","I \u2212> V"]},{"entry":["105","S \u2212> R"]},{"entry":["135","K \u2212>"]},{"entry":["135","K \u2212> E"]},{"entry":["140","D \u2212> G"]},{"entry":["140","D \u2212> V"]},{"entry":["148","F \u2212> L"]},{"entry":["154","I \u2212> T"]},{"entry":["155","L \u2212>"]},{"entry":["161","T \u2212>"]},{"entry":["239","T \u2212>"]},{"entry":["247","S \u2212> G"]},{"entry":["281","P \u2212>"]},{"entry":["303","L \u2212> P"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The variant protein has the following domains, as determined by using InterPro. The domains are described in Table 23:",{"@attributes":{"id":"p-0823","num":"0822"},"tables":{"@attributes":{"id":"TABLE-US-00035","num":"00035"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 23"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"InterPro domains"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Analysis",{}]},{"entry":[{},"Domain description","type","Positions on protein"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"Immunoglobulin-like","ProfileScan","99-186"]},{"entry":[{},"Immunoglobulin C2 type","HMMSmart","111-177\u2002"]},{"entry":[{},"Immunoglobulin subtype","HMMSmart","28-188"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},"Variant protein AI581519_P7 (SEQ ID NO:14) is encoded by the transcript AI581519_T8 (SEQ ID NO:8), for which the coding portion starts at position 171 and ends at position 1208. The transcript also has the following SNPs as listed in Table 24 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-0825","num":"0824"},"tables":{"@attributes":{"id":"TABLE-US-00036","num":"00036"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 24"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","238, 438, 573, 589, 908, 909, 2191, 2348"]},{"entry":[{},"T \u2212>","275, 634"]},{"entry":[{},"C \u2212> T","322, 452"]},{"entry":[{},"C \u2212> A","449, 485"]},{"entry":[{},"C \u2212> G","485"]},{"entry":[{},"A \u2212>","573, 653, 887, 1013"]},{"entry":[{},"A \u2212> T","589"]},{"entry":[{},"T \u2212> C","612, 631, 1078"]},{"entry":[{},"G \u2212> A","1556, 1677, 1744"]},{"entry":[{},"T \u2212> A","2137\u2002"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"Variant protein AI581519_P9 (SEQ ID NO:15) according to the present invention has an amino acid sequence encoded by transcript AI581519_T10 (SEQ ID NO:9). Alignments to one or more previously published protein sequences are shown in . A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","2. Comparison report between AI581519_P9 (SEQ ID NO:15) and known proteins NP872413 and Q86XK7_HUMAN (SEQ ID NO: 11) ():","A. An isolated chimeric polypeptide encoding for AI581519_P9 (SEQ ID NO:15), comprising a first amino acid sequence being at least 90% homologous to MVFAFWKVFLILSCLAGQVSVVQVTIPDGFVNVTVGSNVTLICIYTTTVASREQLS IQWSFFHKKEMEPISIYFSQGGQAVAIGQFKDRITGSNDPGNASITISHMQPADSG IYICDVNNPPDFLGQNQGILNVSVLVKPSKPLCSVQGRPETGHTISLSCLSALGTP SPVYYWHKLEGRDIVPVKENF corresponding to amino acids 1-189 of known proteins NP872413 and Q86XK7_HUMAN (SEQ ID NO: 11), which also corresponds to amino acids 1-189 of AI581519_P9 (SEQ ID NO:15), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95, 96, 97, 98 or 99% homologous to a polypeptide having the sequence TNHRDFGHWKSDKF (SEQ ID NO: 286) corresponding to amino acids 190-203 of AI581519_P9 (SEQ ID NO:15), wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","C. An isolated polypeptide encoding for an edge portion of AI581519_P9 (SEQ ID NO:15), comprising an amino acid sequence being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95, 96, 97, 98 or 99% homologous to the sequence TNHRDFGHWKSDKF (SEQ ID NO: 286) of AI581519_P9 (SEQ ID NO:15).","The localization of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: membrane.","Variant protein AI581519_P9 (SEQ ID NO:15) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 25, (given according to their positions on the amino acid sequence, with the alternative amino acids listed (SEQ ID NO:15)).",{"@attributes":{"id":"p-0832","num":"0831"},"tables":{"@attributes":{"id":"TABLE-US-00037","num":"00037"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 25"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["23","Q \u2212> R"]},{"entry":["35","V \u2212>"]},{"entry":["51","S \u2212> F"]},{"entry":["90","I \u2212> V"]},{"entry":["127","Q \u2212>"]},{"entry":["146","S \u2212> R"]},{"entry":["176","K \u2212>"]},{"entry":["176","K \u2212> E"]},{"entry":["181","D \u2212> G"]},{"entry":["181","D \u2212> V"]},{"entry":["189","F \u2212> L"]},{"entry":["195","F \u2212>"]},{"entry":["202","K \u2212>"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The variant protein has the following domains, as determined by using InterPro. The domains are described in Table 26:",{"@attributes":{"id":"p-0834","num":"0833"},"tables":{"@attributes":{"id":"TABLE-US-00038","num":"00038"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 26"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"InterPro domains"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Analysis",{}]},{"entry":[{},"Domain description","type","Positions on protein"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"Immunoglobulin-like","HMMPfam","36-118"]},{"entry":[{},"Myelin P0 protein","FPrintScan","35-59, 110-139"]},{"entry":[{},"Immunoglobulin-like","ProfileScan","36-120"]},{"entry":[{},"Immunoglobulin subtype","HMMSmart","28-137"]},{"entry":[{},"Immunoglobulin V-set","HMMPfam","21-137"]},{"entry":[{},"Immunoglobulin V-type","HMMSmart","38-118"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},"Variant protein AI581519_P9 (SEQ ID NO:15) is encoded by the transcript AI581519_T10 (SEQ ID NO:9), for which the coding portion starts at position 171 and ends at position 779. The transcript also has the following SNPs as listed in Table 27 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-0836","num":"0835"},"tables":{"@attributes":{"id":"TABLE-US-00039","num":"00039"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 27"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","238, 438, 696, 712, 1029, 1030, 2312, 2469"]},{"entry":[{},"T \u2212>","275, 755"]},{"entry":[{},"C \u2212> T","322, 452"]},{"entry":[{},"C \u2212> A","449, 608"]},{"entry":[{},"C \u2212>","549"]},{"entry":[{},"C \u2212> G","608"]},{"entry":[{},"A \u2212>","696, 774, 1008, 1134"]},{"entry":[{},"A \u2212> T","712"]},{"entry":[{},"T \u2212> C","735, 752, 1199"]},{"entry":[{},"G \u2212> A","1677, 1798, 1865"]},{"entry":[{},"T \u2212> A","2258\u2002"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"Variant protein AI581519_P10 (SEQ ID NO:16) according to the present invention has an amino acid sequence as encoded by transcript AI581519_T11 (SEQ ID NO:10). Alignments to previously published protein sequences are shown in . A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","2. Comparison report between AI581519_P10 (SEQ ID NO:16) and known proteins NP872413 and Q86XK7_HUMAN (SEQ ID NO: 11) ():","A. An isolated chimeric polypeptide encoding for AI581519_P10 (SEQ ID NO:16), comprising a first amino acid sequence being at least 90% homologous to MVFAFWKVFLILSCLAGQVSVVQVTIPDGFVNVTVGSNVTLICIYTTTVASREQLS IQWSFFHKKEMEPISIYFSQGGQAVAIGQFKDRITGSNDPGNASITISHMQPADSG IYICDVNNPPDFLGQNQGILNVSVLVKPSKPLCSVQGRPETGHTISLSCLSALGTP SPVYYWHKLEGRDIVPVKENFNPTTGILVIGNLTNFEQGYYQCTAINRLGNSSCE IDLTSS corresponding to amino acids 1-229 of known proteins NP872413 and Q86XK7_HUMAN (SEQ ID NO: 11), which also corresponds to amino acids 1-229 of AI581519_P10 (SEQ ID NO:16), and a second amino acid sequence RQ (SEQ ID NO: 287) corresponding to amino acids 230-231 of AI581519_P10 (SEQ ID NO:16), wherein said first amino acid sequence, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","The localization of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: membrane.","Variant protein AI581519_P10 (SEQ ID NO:16) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 28, (given according to their positions on the amino acid sequence, with the alternative amino acids listed (SEQ ID NO:16)).",{"@attributes":{"id":"p-0842","num":"0841"},"tables":{"@attributes":{"id":"TABLE-US-00040","num":"00040"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 28"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["23","Q \u2212> R"]},{"entry":["35","V \u2212>"]},{"entry":["51","S \u2212> F"]},{"entry":["90","I \u2212> V"]},{"entry":["127","Q \u2212>"]},{"entry":["146","S \u2212> R"]},{"entry":["176","K \u2212>"]},{"entry":["176","K \u2212> E"]},{"entry":["181","D \u2212> G"]},{"entry":["181","D \u2212> V"]},{"entry":["189","F \u2212> L"]},{"entry":["195","I \u2212> T"]},{"entry":["196","L \u2212>"]},{"entry":["202","T \u2212>"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The variant protein has the following domains, as determined by using InterPro. The domains are described in Table 29:",{"@attributes":{"id":"p-0844","num":"0843"},"tables":{"@attributes":{"id":"TABLE-US-00041","num":"00041"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 29"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"InterPro domains"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Analysis",{}]},{"entry":[{},"Domain description","type","Positions on protein"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"Immunoglobulin C2 type","HMMSmart","34-123, 152-218"]},{"entry":[{},"Immunoglobulin subtype","HMMSmart","28-137, 146-229"]},{"entry":[{},"Immunoglobulin-like","HMMPfam","36-118, 154-113"]},{"entry":[{},"Myelin P0 protein","FPrintScan","\u200235-59, 110-139"]},{"entry":[{},"Immunoglobulin V-type","HMMSmart","38-118"]},{"entry":[{},"Immunoglobulin V-set","HMMPfam","21-137"]},{"entry":[{},"Immunoglobulin-like","ProfileScan","36-120, 140-227"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},"Variant protein AI581519_P10 (SEQ ID NO:16) is encoded by the transcript AI581519_T11 (SEQ ID NO:10), for which the coding portion starts at position 171 and ends at position 863. The transcript also has the following SNPs as listed in Table 30 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-0846","num":"0845"},"tables":{"@attributes":{"id":"TABLE-US-00042","num":"00042"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 30"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","238, 438, 696, 712, 889, 890, 2172, 2329"]},{"entry":[{},"T \u2212>","275, 757"]},{"entry":[{},"C \u2212> T","322, 452"]},{"entry":[{},"C \u2212> A","449, 608"]},{"entry":[{},"C \u2212>","549"]},{"entry":[{},"C \u2212> G","608"]},{"entry":[{},"A \u2212>","696, 776, 868, 994"]},{"entry":[{},"A \u2212> T","712"]},{"entry":[{},"T \u2212> C","735, 754, 1059"]},{"entry":[{},"G \u2212> A","1537, 1658, 1725"]},{"entry":[{},"T \u2212> A","2118\u2002"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"According to an optional embodiment of the present invention, short segments related to the above cluster are also provided. These segments are up to about 120 bp in length, and so are included in a separate description.","Segment cluster AI581519_N7 (SEQ ID NO:120) according to the present invention is supported by 6 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcripts: AI581519_T5 (SEQ ID NO:6) and AI581519_T6 (SEQ ID NO:7). Table 31 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0849","num":"0848"},"tables":{"@attributes":{"id":"TABLE-US-00043","num":"00043"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 31"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["AI581519_T5 (SEQ ID NO: 6)","384","491"]},{"entry":["AI581519_T6 (SEQ ID NO: 7)","384","491"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster AI581519_N9 (SEQ ID NO:121) according to the present invention is supported by 1 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcripts: AI581519_T6 (SEQ ID NO:7). Table 32 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0851","num":"0850"},"tables":{"@attributes":{"id":"TABLE-US-00044","num":"00044"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 32"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["AI581519_T6 (SEQ ID NO: 7)","492","569"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Expression of V-set and immunoglobulin domain containing 1 (VSIG1) AI581519 transcripts which are detectable by amplicon as depicted in sequence name AI581519_seg7 (SEQ ID NO:190) in normal and cancerous Ovary tissues, in normal and cancerous lung tissues and in different normal tissues","Expression of V-set and immunoglobulin domain containing 1 (VSIG1) transcripts detectable by or according to seg7\u2014AI581519_seg7 (SEQ ID NO:190) amplicon and primers AI581519_seg7F1 (SEQ ID NO: 188) and AI581519_seg7R1 (SEQ ID NO: 189) was measured by real time PCR on ovary panel, lung panel and normal panel. The samples used are detailed in Table 4, Table 3 and Table 2 above, respectively.","Ovary panel\u2014Non-detected samples (samples no. 80, 83, 100 and 109, Table 4) were assigned Ct value of 41 and were calculated accordingly. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1. The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 52-78, Table 4 above), to obtain a value of fold up-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-0855","num":"0854"},"figref":"FIG. 4"},"As is evident from , the expression of V-set and immunoglobulin domain containing 1 (VSIG1) transcripts detectable by the above amplicon in mucinous carcinoma and endometroid samples was significantly higher than in the non-cancerous samples (sample numbers 52-78, Table 4 above). Notably an over-expression of at least 40 fold was found in 4 out of 12 mucinous carcinoma samples and in 3 out of 10 endometroid samples.","Statistical analysis was applied to verify the significance of these results, as described below. Threshold of 40 fold over expression was found to differentiate between mucinous carcinoma and endometroid and normal samples with P value of 6.02e-003 and 1.54e-002, respectively as checked by exact Fisher test. The above values demonstrate statistical significance of the results.","Lung panel\u2014For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1. The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 51-64 and 69-70 Table 3 above), to obtain a value of fold up-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-0859","num":"0858"},"figref":"FIG. 5"},"As is evident from , the expression of V-set and immunoglobulin domain containing 1 (VSIG1) transcripts detectable by the above amplicon in adenocarcinoma samples was significantly higher than in the non-cancerous samples (sample numbers 51-64 and 69-70, Table 3 above). Notably an over-expression of at least 6 fold was found in 11 out of 23 adenocarcinoma samples.","Statistical analysis was applied to verify the significance of these results, as described below. The P value for the difference in the expression levels of V-set and immunoglobulin domain containing 1 (VSIG1) transcripts detectable by the above amplicon in Lung adenocarcinoma samples versus the normal tissue samples was determined by T test as 9.36e-003. Threshold of 6 fold over expression was found to differentiate between adenocarcinoma and normal samples with P value of 8.07e-004 as checked by exact Fisher test.","The above values demonstrate statistical significance of the results.","Normal panel\u2014For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1. The normalized quantity of each RT sample was then divided by the median of the quantities of the ovary samples (sample numbers 31, 32, 33 and 34, Table 2 above), to obtain a value of relative expression of each sample relative to median of the ovary samples, as shown in . The normalized quantity of each RT sample was also divided by the median of the quantities of the ovary samples (sample numbers 31-34, Table 2 above), to obtain a value of relative expression of each sample relative to median of the lung samples (sample numbers 26, 28, 29 and 30, Table 2 above), as shown in .","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: AI581519_seg7F1 (SEQ ID NO: 188) forward primer; and AI581519_seg7R1 (SEQ ID NO: 189) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: AI581519_seg7 (SEQ ID NO:190).",{"@attributes":{"id":"p-0866","num":"0865"},"tables":{"@attributes":{"id":"TABLE-US-00045","num":"00045"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>AI581519_seg7F1\u2003(SEQ\u2003ID\u2003NO:\u2003188):"},{"entry":"CACAGCTCGTGCCTCAGTACTG"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>AI581519_seg7R1\u2003(SEQ\u2003ID\u2003NO:\u2003189):"},{"entry":"AGCTACATCTTCCCCGAGCG"},{"entry":{}},{"entry":"Amplicon\u2003>AI581519_seg7\u2003(SEQ\u2003ID\u2003NO:\u2003190)"},{"entry":"CACAGCTCGTGCCTCAGTACTGAGGGTATGGAGGAAAAGGCAGTCGGTC"},{"entry":{}},{"entry":"AGTGTCTAAAAATGACGCACGTAAGAGACGCTCGGGGAAGATGTAGCT"}]}}}}},"Expression of V-set and immunoglobulin domain containing 1(VSIG1) AI581519 transcripts which are detectable by amplicon as depicted in sequence name AI581519_seg7-9 (SEQ ID NO: 187) in normal and cancerous ovary tissues and in normal and cancerous lung tissues","Expression of V-set and immunoglobulin domain containing 1(VSIG1) transcripts detectable by or according to seg7-9\u2014AI581519_seg7-9 (SEQ ID NO: 187) amplicon and primers AI581519_seg7-9F1 (SEQ ID NO: 185) and AI581519_seg7-9R1 (SEQ ID NO: 186) was measured by real time PCR on ovary panel and lung panel. The samples used are detailed in Table 4 and Table 3 above, respectively.","Ovary panel\u2014Non-detected sample (sample no. 40 Table 4) was assigned Ct value of 41 and was calculated accordingly. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1. The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 52-78, Table 4 above), to obtain a value of fold up-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-0870","num":"0869"},"figref":"FIG. 7"},"As is evident from , the expression of V-set and immunoglobulin domain containing 1(VSIG1) transcripts detectable by the above amplicon in mucinous carcinoma samples was significantly higher than in the non-cancerous samples (sample numbers 52-78, Table 4 above) and was higher in a few adenocarcinoma samples than in the non-cancerous samples. Notably an over-expression of at least 6 fold was found in 6 out of 9 mucinous carcinoma samples and in 9 out of 37 endometroid samples.","Statistical analysis was applied to verify the significance of these results, as described below. Threshold of 6 fold over expression was found to differentiate between mucinous carcinoma and normal samples with P value of 2.75e-004 as checked by exact Fisher test. Threshold of 6 fold over expression was found to differentiate between adenocarcinoma and normal samples with P value of 2.44e-002 as checked by exact Fisher test.","The above values demonstrate statistical significance of the results.","Lung panel\u2014For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1. The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 51-64 and 69-70, Table 3 above), to obtain a value of fold up-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-0875","num":"0874"},"figref":"FIG. 8"},"As is evident from , the expression of V-set and immunoglobulin domain containing 1(VSIG1) transcripts detectable by the above amplicon in adenocarcinoma samples was significantly higher than in the non-cancerous samples (sample numbers 51-64 and 69-70, Table 3 above) and was higher in a few non-small cell carcinoma samples than in the non-cancerous samples. Notably an over-expression of at least 17 fold was found in 8 out of 18 adenocarcinoma samples.","Statistical analysis was applied to verify the significance of these results, as described below. The P value for the difference in the expression levels of V-set and immunoglobulin domain containing 1(VSIG1) transcripts detectable by the above amplicon in Lung adenocarcinoma samples versus the normal tissue samples was determined by T test as 1.17e-002.","Threshold of 17 fold over expression was found to differentiate between adenocarcinoma and normal samples with P value of 2.41e-003 as checked by exact Fisher test.","The above values demonstrate statistical significance of the results.","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: AI581519_seg7-9F1 (SEQ ID NO: 185) forward primer; and AI581519_seg7-9R1 (SEQ ID NO: 186) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: AI581519_seg7-9 (SEQ ID NO: 187).",{"@attributes":{"id":"p-0882","num":"0881"},"tables":{"@attributes":{"id":"TABLE-US-00046","num":"00046"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>AI581519_seg7-9F1\u2003(SEQ\u2003ID\u2003NO:\u2003185):"},{"entry":"AATGACGCACGTAAGAGACGC"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>AI581519_seg7-9R1\u2003(SEQ\u2003ID\u2003NO:\u2003186):"},{"entry":"GAGTGCCCTTCACAGCCTCA"},{"entry":{}},{"entry":"Amplicon\u2003>AI581519_seg7-9\u2003(SEQ\u2003ID\u2003NO:\u2003187)"},{"entry":"AATGACGCACGTAAGAGACGCTCGGGGAAGATGTAGCTGGACCTCTGAG"},{"entry":{}},{"entry":"TCTCCTTGGGAGGAGGGGAAGTGGCCAGATGTTGAGGCTGTGAAGGGCA"},{"entry":{}},{"entry":"CTC"}]}}}}},"Expression of V-set and immunoglobulin domain containing 1(VSIG1) AI581519 transcripts which are detectable by amplicon as depicted in sequence name AI581519seg7-9 (SEQ ID NO: 196) in the blood-specific panel.","Expression of VSIG1 transcripts detectable by or according to seg7-9-AI581519seg7-9F3R3 (SEQ ID NO:196) amplicon and primers AI581519seg7-9F3 (SEQ ID NO:194) and AI581519seg7-9R3 (SEQ ID NO:195) was measured by real time PCR on blood panel. The samples used are detailed in Table 1 above. Non-detected samples (samples no. 28, 33, 83, 85, 90 and 63, Table 1) were assigned Ct value of 41 and were calculated accordingly. The samples used are detailed in Table 1 above. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1. The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 64, 69-72 and 74-76, Table 1 above), to obtain a value of relative expression of each sample relative to median of the normal samples.","The results of this analysis are depicted in the histogram in . Expression of the above-indicated VSIG1 transcript is high in CD8, CD4 untreated and CD4+CD25\u2212 samples but even higher in normal small intestine and normal stomach samples.","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: seg7-9F3 forward primer (SEQ ID NO:194); and seg7-9R3 reverse primer (SEQ ID NO:195).","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: seg7-9F3R3 (SEQ ID NO:196).",{"@attributes":{"id":"p-0888","num":"0887"},"tables":{"@attributes":{"id":"TABLE-US-00047","num":"00047"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>AI581519seg7-9F3"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003194)"},{"entry":"ATGACGCACGTAAGAGACGCTCG"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>AI581519seg7-9R3"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003195)"},{"entry":"GGAGTTCAGCCTGCTGTCCATCAAG"},{"entry":{}},{"entry":"Amplicon\u2003>AI581519seg7-9F3R3"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003196)"},{"entry":"ATGACGCACGTAAGAGACGCTCGGGGAAGATGTAGCTGGACCTCTGA"},{"entry":{}},{"entry":"GTCTCCTTGGGAGGAGGGGAAGTGGCCAGATGTTGAGGCTGTGAAGG"},{"entry":{}},{"entry":"GCACTCTTGATGGACAGCAGGCTGAACTCC"}]}}}}},"Expression of V-set and immunoglobulin domain containing 1(VSIG1) AI581519 transcripts which are detectable by amplicon as depicted in sequence name AI581519_junc7-11F2R2 (SEQ ID NO:193) in normal and cancerous lung tissues, normal and cancerous ovary tissues, different normal tissues and blood-specific panel.","Expression of VSIG1 transcripts detectable by or according to junc7-11F2R2-AI581519_junc7-11F2R2 (SEQ ID NO:193) amplicon and primers AI581519_junc7-11F2 (SEQ ID NO:191) and AI581519_junc7-11R2 (SEQ ID NO:192) was measured by real time PCR on lung panel, ovary panel, normal panel and blood panel. The samples used are detailed in Table 3, Table 4, Table 2 and Table 1 above, respectively. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1.","For lung panel\u2014Non-detected sample (sample no. 49, Table 3) was assigned Ct value of 41 and was calculated accordingly. The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (samples numbers 51-64, 69 and 70, Table 3 above), to obtain a value of fold up-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-0892","num":"0891"},"figref":"FIG. 10"},"As is evident from , the expression of VSIG1 transcripts detectable by the above amplicon in adenocarcinoma samples was significantly higher than in the non-cancerous samples (sample numbers 51-64, 69 and 70, Table 3 above). Notably an over-expression of at least 16 fold was found in 10 out of 23 adenocarcinoma samples.","Statistical analysis was applied to verify the significance of these results, as described below. The P value for the difference in the expression levels of VSIG1 transcripts detectable by the above amplicon in lung adenocarcinoma samples versus the normal tissue samples was determined by T test as 5.13e-003.","Threshold of 16 fold over expression was found to differentiate between adenocarcinoma and normal samples with P value of 1.80e-003 as checked by exact Fisher test.","The above values demonstrate statistical significance of the results.","For ovary panel\u2014Non-detected samples (samples no. 16, 23, 57, 60-62, 67, 68, 71-74, 77 and 78, Table 4) were assigned Ct value of 41 and were calculated accordingly. The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 52, 53, 55 and 57-67, Table 4 above), to obtain a value of fold up-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-0898","num":"0897"},"figref":"FIG. 11"},"As is evident from , the expression of VSIG1 transcripts detectable by the above amplicon in mucinous carcinoma samples was significantly higher than in the non-cancerous samples (sample numbers 52, 53, 55 and 57-67, Table 4 above) and was higher in a few adenocarcinoma samples than in the non-cancerous samples. Notably an over-expression of at least 25 fold was found in 6 out of 9 mucinous carcinoma samples and in 10 out of 37 adenocarcinoma samples.","Statistical analysis was applied to verify the significance of these results, as described below. Threshold of 25 fold over expression was found to differentiate between mucinous carcinoma samples and adenocarcinoma and normal samples with P value of 3.95e-004 and 1.88e-002, respectively, as checked by exact Fisher test. The above values demonstrate statistical significance of the results.","For normal panel\u2014Non-detected samples (samples no. 11-20, 28, 30, 32-34, 36, 38-40, 49 and 56, Table 2) were assigned Ct value of 41 and were calculated accordingly. The normalized quantity of each RT sample was then divided by the median of the quantities of the lung samples (sample numbers 26, and 28-30, Table 2 above), to obtain a value of relative expression of each sample relative to median of the lung samples, as shown in . The normalized quantity of each RT sample was then divided by the median of the quantities of the ovary samples (sample numbers 31-34, Table 2 above), to obtain a value of relative expression of each sample relative to median of the ovary samples, as shown in .","For blood panel\u2014Non-detected samples (samples no. 6, 15-19, 22-24, 27, 29, 40, 41, 46-50, 52-58, 60-64, 71 and 76, Table 1) were assigned Ct value of 41 and were calculated accordingly. The normalized quantity of each RT sample was then divided by the median of the quantities of the kidney normal samples (sample numbers 65-67, Table 1 above), to obtain a value of relative expression of each sample relative to median of the normal samples.","The results of this analysis are depicted in the histogram in . Expression of the above-indicated VSIG1 transcript is very high in CD8, CD4 untreated and CD4+CD25\u2212 samples, is high in several lymphomas but also very high in normal small intestine and normal stomach samples.","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: AI581519 junc7-11F2 (SEQ ID NO:191) forward primer; and AI581519 junc7-11R2 (SEQ ID NO:192) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: AI581519 junc7-11F2R2 (SEQ ID NO:193).",{"@attributes":{"id":"p-0906","num":"0905"},"tables":{"@attributes":{"id":"TABLE-US-00048","num":"00048"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>AI581519_junc7-11F2"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003191)"},{"entry":"GAAGATGTAGCTGGACCTCTGAGATTTA"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>AI581519_junc7-11R2"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003192)"},{"entry":"GTTGGACCCTGTAATTCGATCTTT"},{"entry":{}},{"entry":"Amplicon\u2003>AI581519_junc7-11F2R2"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003193)"},{"entry":"GAAGATGTAGCTGGACCTCTGAGATTTACTTTTCTCAAGGTGGACAAG"},{"entry":{}},{"entry":"CTGTAGCCATCGGGCAATTTAAAGATCGAATTACAGGGTCCAAC"}]}}}}},"The present invention relates to ILDR1 polypeptides, novel splice variants and diagnostics and therapeutics based thereon.","According to the present invention, Cluster AA424839 (internal ID 71418261) features 4 transcripts and 1 segment of interest, the names for which are given in Tables 33 and 34, respectively. The selected protein variants are given in table 35.",{"@attributes":{"id":"p-0909","num":"0908"},"tables":{"@attributes":{"id":"TABLE-US-00049","num":"00049"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 33"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Transcripts of interest"},{"entry":"Transcript Name"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"168pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"AA424839_T0 (SEQ ID NO: 17)"]},{"entry":[{},"AA424839_T2 (SEQ ID NO: 18)"]},{"entry":[{},"AA424839_T4 (SEQ ID NO: 19)"]},{"entry":[{},"AA424839_1_T7 (SEQ ID NO: 20)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0910","num":"0909"},"tables":{"@attributes":{"id":"TABLE-US-00050","num":"00050"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 34"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segments of interest"},{"entry":"Segment Name"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"168pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"AA424839_N18 (SEQ ID NO: 122)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0911","num":"0910"},"tables":{"@attributes":{"id":"TABLE-US-00051","num":"00051"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 35"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Proteins of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Protein Name","Corresponding Transcripts"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["AA424839_P3 (SEQ ID NO: 22)","AA424839_T0 (SEQ ID NO: 17)"]},{"entry":["AA424839_P5 (SEQ ID NO: 21)","AA424839_T2 (SEQ ID NO: 18)"]},{"entry":["AA424839_P7 (SEQ ID NO: 23)","AA424839_T4 (SEQ ID NO: 19)"]},{"entry":["AA424839_1_P11 (SEQ ID NO:","AA424839_1_T7 (SEQ ID NO: 20)"]},{"entry":"24)"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"These sequences are variants of the known protein immunoglobulin-like domain containing receptor 1 (RefSeq accession identifier NP787120 (SEQ ID NO: 21), also known as ILDR1alpha, ILDR1beta, MGC50831), referred to herein as the previously known protein.","ILDR1, denoted immunoglobulin-like domain containing receptor 1 (SEQ ID NO:21), was described by Hauge et al. (2004) BBRC 323: 970-978, that demonstrated differential expression of the transcripts encoding this protein in indolent follicular lymphoma (FL) and matched transformed diffuse large B cell lymphoma (DLBCL). The gene was identified using a cDNA subtraction strategy on patient-matched biopsies of FL and DLBCL. The protein was shown to contain a signal peptide and transmembrane domain, and an Ig domain in the extracellular portion, and it was found to be membrane bound protein, having 31% identity to lipolysis-stimulated remnant receptor (LSR).","According to the present invention, ILDR1 protein and ILDR1 splice variants were predicted to be novel B7\/CD28 members. According to the present invention, ILDR1 and ILDR1 splice variants were demonstrated to be overexpressed in ovarian cancer.","MED discovery engine described in Example 1 herein, was used to assess the expression of ILDR1 transcripts. Expression data for Affymetrix probe set 235583_at representing the ILDR1 gene data is shown in . As evident from the scatter plot, presented in , the expression of ILDR1 transcripts detectable with the above probe sets was higher in ovarian cancer compared to normal ovary samples. As evident from the scatter plot, presented in , the expression of ILDR1 transcripts detectable with the above probe sets was higher in colon cancer compared to normal colon samples.","As noted above, cluster AA424839 features 4 transcripts, which were listed in Table 33 above. These transcripts encode for proteins which are variants of protein immunoglobulin-like domain containing receptor 1 (SEQ ID NO:21). A description of each protein according to the present invention is now provided.","Variant protein AA424839_P3 (SEQ ID NO:22) according to the present invention has an amino acid sequence as encoded by transcript AA424839_TO (SEQ ID NO:17). Alignments to previously published protein sequences are shown in . A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","1. Comparison report between AA424839_P3 (SEQ ID NO:22) and known proteins Q86SU0_HUMAN (SEQ ID NO: 21) and NP787120 (SEQ ID NO: 21) ():","A. An isolated chimeric polypeptide encoding for AA424839_P3 (SEQ ID NO:22), comprising a first amino acid sequence being at least 90% homologous to MAWPKLPAPWLLLCTWLPAGCLSLLVTVQHTERYVTLFASIILKCDYTTSAQLQ DVVVTWRFKSFCKDPIFDYYSASYQAALSLGQDPSNDCNDNQREVRIVAQRRG QNEPVLGVDYRQRKITIQNRADLVINEVMWWDHGVYYCTIEAPGDTSGDPDKE VKLIVLHWLTVIFIILGALLLLLLIGVCWCQCCPQYCCCYIRCPCCPAHCCCPEE corresponding to amino acids 1-215 of known proteins Q86SU0_HUMAN (SEQ ID NO: 21) and NP787120 (SEQ ID NO: 21), which also corresponds to amino acids 1-215 of AA424839_P3 (SEQ ID NO:22), a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95, 96, 97, 98 or 99% homologous to a polypeptide having the sequence ALARHRYMKQAQALGPQMMGKPLYWGADRSSQVSSYPMHPLLQR (SEQ ID NO: 288) corresponding to amino acids 216-259 of AA424839_P3 (SEQ ID NO:22), and a third amino acid sequence being at least 90% homologous to DLSLPSSLPQMPMTQTTNQPPIANGVLEYLEKELRNLNLAQPLPPDLKGRFGHP CSMLSSLGSEVVERRIIHLPPLIRDLSSSRRTSDSLHQQWLTPIPSRPWDLREGRSH HHYPDFHQELQDRGPKSWALERRELDPSWSGRHRSSRLNGSPIHWSDRDSLSDV PSSSEARWRPSHPPFRSRCQERPRRPSPRESTQRHGRRRRHRSYSPPLPSGLSSWSS EEDKERQPQSWRAHRRGSHSPHWPEEKPPSYRSLDITPGKNSRKKGSVERRSEK DSSHSGRSVVI corresponding to amino acids 216-502 of known proteins Q86SU0_HUMAN (SEQ ID NO: 21) and NP787120 (SEQ ID NO: 21), which also corresponds to amino acids 260-546 of AA424839_P3 (SEQ ID NO:22), wherein said first amino acid sequence, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","B. An isolated polypeptide encoding for an edge portion of AA424839_P3 (SEQ ID NO:22), comprising an amino acid sequence being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95, 96, 97, 98 or 99% homologous to the sequence ALARHRYMKQAQALGPQMMGKPLYWGADRSSQVSSYPMHPLLQR (SEQ ID NO: 288) of AA424839_P3 (SEQ ID NO:22).","The localization of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: membrane.","Variant protein AA424839_P3 (SEQ ID NO:22) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 36, (given according to their positions on the amino acid sequence, with the alternative amino acids listed (SEQ ID NO:22)).",{"@attributes":{"id":"p-0923","num":"0922"},"tables":{"@attributes":{"id":"TABLE-US-00052","num":"00052"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 36"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["165","V \u2212> F"]},{"entry":["165","V \u2212> L"]},{"entry":["264","P \u2212> R"]},{"entry":["388","W \u2212> L"]},{"entry":["388","W \u2212> S"]},{"entry":["436","H \u2212> N"]},{"entry":["500","H \u2212> L"]},{"entry":["500","H \u2212> P"]},{"entry":["516","D \u2212> Y"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The variant protein has the following domains, as determined by using InterPro. The domains are described in Table 37.",{"@attributes":{"id":"p-0925","num":"0924"},"tables":{"@attributes":{"id":"TABLE-US-00053","num":"00053"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 37"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"InterPro domains"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Analysis",{}]},{"entry":[{},"Domain description","type","Positions on protein"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"Phospholipase A2","ScanRegExp","205-212"]},{"entry":[{},"Immunoglobulin subtype","HMMSmart","\u200230-166"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},"Variant protein AA424839_P3 (SEQ ID NO:22) is encoded by the transcript AA424839_T0 (SEQ ID NO:17), for which the coding portion starts at position 204 and ends at position 1841. The transcript also has the following SNPs as listed in Table 38 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-0927","num":"0926"},"tables":{"@attributes":{"id":"TABLE-US-00054","num":"00054"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 38"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"G \u2212> T","696, 1366, 1748, 1749"]},{"entry":[{},"G \u2212> C","696, 995, 1366"]},{"entry":[{},"C \u2212> G","994, 1529, 2305"]},{"entry":[{},"A \u2212> T","1442, 1702"]},{"entry":[{},"A \u2212> C","1442, 1702"]},{"entry":[{},"G \u2212> A","1502"]},{"entry":[{},"C \u2212> A","1509, 1529"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"Protein AA424839_P5 (SEQ ID NO:21) according to the present invention has an amino acid sequence as encoded by transcript AA424839_T2 (SEQ ID NO:18).","The localization of the protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The protein is believed to be located as follows with regard to the cell: membrane.","Protein AA424839_P5 (SEQ ID NO:21) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 39, (given according to their positions on the amino acid sequence, with the alternative amino acids listed (SEQ ID NO:21)).",{"@attributes":{"id":"p-0931","num":"0930"},"tables":{"@attributes":{"id":"TABLE-US-00055","num":"00055"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 39"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["165","V \u2212> F"]},{"entry":["165","V \u2212> L"]},{"entry":["220","P \u2212> R"]},{"entry":["344","W \u2212> L"]},{"entry":["344","W \u2212> S"]},{"entry":["392","H \u2212> N"]},{"entry":["456","H \u2212> L"]},{"entry":["456","H \u2212> P"]},{"entry":["472","D \u2212> Y"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The protein has the following domains, as determined by using InterPro. The domains are described in Table 40:",{"@attributes":{"id":"p-0933","num":"0932"},"tables":{"@attributes":{"id":"TABLE-US-00056","num":"00056"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 40"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"InterPro domains"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Analysis",{}]},{"entry":[{},"Domain description","type","Positions on protein"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"Phospholipase A2","ScanRegExp","205-212"]},{"entry":[{},"Immunoglobulin subtype","HMMSmart","\u200230-166"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},"Protein AA424839_P5 (SEQ ID NO:21) is encoded by the transcript AA424839_T2 (SEQ ID NO:18), for which the coding portion starts at position 204 and ends at position 1709. The transcript also has the following SNPs as listed in Table 41 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-0935","num":"0934"},"tables":{"@attributes":{"id":"TABLE-US-00057","num":"00057"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 41"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"G \u2212> T","696, 1234, 1616, 1617"]},{"entry":[{},"G \u2212> C","696, 863, 1234"]},{"entry":[{},"C \u2212> G","862, 1397, 2173"]},{"entry":[{},"A \u2212> T","1310, 1570"]},{"entry":[{},"A \u2212> C","1310, 1570"]},{"entry":[{},"G \u2212> A","1370"]},{"entry":[{},"C \u2212> A","1377, 1397"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The genomic structure of protein AA424839_P5 (SEQ ID NO:21) (number of exons relevant to the extra-cellular region of the protein, the length of these exons, the frame of the codon in which the introns are inserted and the location of the protein features and domains in the gene structure) is characteristic to the ligands of the B7\/co-stimulatory protein family, as given in table 42",{"@attributes":{"id":"p-0937","num":"0936"},"tables":{"@attributes":{"id":"TABLE-US-00058","num":"00058"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 42"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"genomic structure and protein features"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"98pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Exon","Exon","Amino-",{}]},{"entry":["number","Length","Acids","Protein feature on exon"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"98pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1","58","\u20021-19","Signal Peptide"]},{"entry":["2","171","20-76","Ig Domain"]},{"entry":["3","150","\u200277-126","Ig Domain"]},{"entry":["4","120","127-166","Ig Domain\/Trans-membrane"]},{"entry":[{},{},{},"region"]},{"entry":["5","147","167-215","Trans-membrane region"]},{"entry":["6","821","216-489"]},{"entry":["7","39","490-502"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein AA424839_P7 (SEQ ID NO:23) according to the present invention has an amino acid sequence as encoded by transcript AA424839_T4 (SEQ ID NO:19). Alignments to one or more previously published protein sequences are shown in . A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","1. Comparison report between AA424839_P7 (SEQ ID NO:23) and known proteins Q86SU0_HUMAN and NP787120 (SEQ ID NO: 21) ():","A. An isolated chimeric polypeptide encoding for AA424839_P7 (SEQ ID NO:23), comprising a first amino acid sequence being at least 90% homologous to MAWPKLPAPWLLLCTWLPAGCLSLLVTVQHTERYVTLFASIILKCDYTTSAQLQ DVVVTWRFKSFCKDPIFDYYSASYQAALSLGQDPSNDCNDNQREVRIVAQRRG QNEPVLGVDYRQRKITIQN corresponding to amino acids 1-126 of known proteins Q86SU0_HUMAN and NP787120 (SEQ ID NO: 21), which also corresponds to amino acids 1-126 of AA424839_P7 (SEQ ID NO:23), a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95, 96, 97, 98 or 99%, homologous to a polypeptide having the sequence PLARHRYMKQAQALGPQMMGKPLYWGADRSSQVSSYPMHPLLQR corresponding to amino acids 127-170 of AA424839_P7 (SEQ ID NO:23), and a third amino acid sequence being at least 90% homologous to DLSLPSSLPQMPMTQTTNQPPIANGVLEYLEKELRNLNLAQPLPPDLKGRFGHP CSMLSSLGSEVVERRIIHLPPLIRDLSSSRRTSDSLHQQWLTPIPSRPWDLREGRSH HHYPDFHQELQDRGPKSWALERRELDPSWSGRHRSSRLNGSPIHWSDRDSLSDV PSSSEARWRPSHPPFRSRCQERPRRPSPRESTQRHGRRRRHRSYSPPLPSGLSSWSS EEDKERQPQSWRAHRRGSHSPHWPEEKPPSYRSLDITPGKNSRKKGSVERRSEK DSSHSGRSVVI corresponding to amino acids 216-502 of known proteins Q86SU0_HUMAN and NP787120 (SEQ ID NO: 21), which also corresponds to amino acids 171-457 of AA424839_P7 (SEQ ID NO:23), wherein said first amino acid sequence, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","The localization of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: secreted.","Variant protein AA424839_P7 (SEQ ID NO:23) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 43, (given according to their positions on the amino acid sequence, with the alternative amino acids listed).",{"@attributes":{"id":"p-0943","num":"0942"},"tables":{"@attributes":{"id":"TABLE-US-00059","num":"00059"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 43"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["175","P \u2212> R"]},{"entry":["299","W \u2212> L"]},{"entry":["299","W \u2212> S"]},{"entry":["347","H \u2212> N"]},{"entry":["411","H \u2212> L"]},{"entry":["411","H \u2212> P"]},{"entry":["427","D \u2212> Y"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein AA424839_P7 (SEQ ID NO:23) is encoded by the transcript AA424839_T4 (SEQ ID NO:19), for which the coding portion starts at position 204 and ends at position 1574. The transcript also has the following SNPs as listed in Table 44 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-0945","num":"0944"},"tables":{"@attributes":{"id":"TABLE-US-00060","num":"00060"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 44"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"C \u2212> G","727, 1262, 2038"]},{"entry":[{},"G \u2212> C","\u2002728, 1099"]},{"entry":[{},"G \u2212> T","1099, 1481, 1482"]},{"entry":[{},"A \u2212> T","1175, 1435"]},{"entry":[{},"A \u2212> C","1175, 1435"]},{"entry":[{},"G \u2212> A","1235"]},{"entry":[{},"C \u2212> A","1242, 1262"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"Variant protein AA4248391_P11 (SEQ ID NO:24) according to the present invention has an amino acid sequence as encoded by transcript AA4248391_T7 (SEQ ID NO:20). Alignments to one or more previously published protein sequences are given in .","The localization of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: membrane.","Variant protein AA4248391_P11 (SEQ ID NO:24) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 45, (given according to their positions on the amino acid sequence, with the alternative amino acids listed (SEQ ID NO:24)).",{"@attributes":{"id":"p-0949","num":"0948"},"tables":{"@attributes":{"id":"TABLE-US-00061","num":"00061"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 45"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["108","V \u2212> F"]},{"entry":["108","V \u2212> L"]},{"entry":["207","P \u2212> R"]},{"entry":["331","W \u2212> L"]},{"entry":["331","W \u2212> S"]},{"entry":["379","H \u2212> N"]},{"entry":["443","H \u2212> L"]},{"entry":["443","H \u2212> P"]},{"entry":["459","D \u2212> Y"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The variant protein has the following domains, as determined by using InterPro. The domains are described in Table 46:",{"@attributes":{"id":"p-0951","num":"0950"},"tables":{"@attributes":{"id":"TABLE-US-00062","num":"00062"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 46"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"InterPro domains"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Analysis",{}]},{"entry":[{},"Domain description","type","Positions on protein"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"Phospholipase A2","ScanRegExp","148-155"]},{"entry":[{},"Immunoglobulin subtype","HMMSmart","\u200230-109"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},"Variant protein AA4248391_P11 (SEQ ID NO:24) is encoded by the transcript AA4248391_T7 (SEQ ID NO:20), for which the coding portion starts at position 204 and ends at position 1670. The transcript also has the following SNPs as listed in Table 47 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-0953","num":"0952"},"tables":{"@attributes":{"id":"TABLE-US-00063","num":"00063"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 47"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"G \u2212> T","525, 1195, 1577, 1578"]},{"entry":[{},"G \u2212> C","525, 824, 1195"]},{"entry":[{},"C \u2212> G","823, 1358, 2134"]},{"entry":[{},"A \u2212> T","1271, 1531"]},{"entry":[{},"A \u2212> C","1271, 1531"]},{"entry":[{},"G \u2212> A","1331"]},{"entry":[{},"C \u2212> A","1338, 1358"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"Segment cluster AA424839_N18 (SEQ ID NO:122) according to the present invention is supported by 10 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcripts: AA424839_TO (SEQ ID NO:17), AA424839_T2 (SEQ ID NO:18), AA424839_T4 (SEQ ID NO:19) and AA4248391_T7 (SEQ ID NO:20). Table 48 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-0955","num":"0954"},"tables":{"@attributes":{"id":"TABLE-US-00064","num":"00064"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 48"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["AA424839_T0 (SEQ ID NO: 17)","1173","1802"]},{"entry":["AA424839_T2 (SEQ ID NO: 18)","1041","1670"]},{"entry":["AA424839_T4 (SEQ ID NO: 19)","906","1535"]},{"entry":["AA424839_1_T7 (SEQ ID NO: 20)","1002","1631"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Expression of immunoglobulin-like domain containing receptor 1 (ILDR1) AA424839 transcripts which are detectable by amplicon as depicted in sequence name AA424839_seg18wt (SEQ ID NO:199) in normal and cancerous Ovary tissues and in different normal tissues","Expression of immunoglobulin-like domain containing receptor 1 (ILDR1) transcripts detectable by or according to seg18wt\u2014AA424839_seg18wt (SEQ ID NO:199) amplicon and primers AA424839_seg18wtF1 (SEQ ID NO:197) and AA424839_seg18wtR1 (SEQ ID NO:198) was measured by real time PCR on ovary panel and normal panel. The samples used are detailed in Table 4 and Table 2 above, respectively. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1.","Ovary panel\u2014The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 52-78, Table 4 above), to obtain a value of fold up-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-0959","num":"0958"},"figref":"FIG. 17"},"As is evident from , the expression of immunoglobulin-like domain containing receptor 1 (ILDR1) transcripts detectable by the above amplicon in serous carcinoma, mucinous carcinoma and adenocarcinoma samples was significantly higher than in the non-cancerous samples (sample numbers 52-78, Table 4 above). Notably an over-expression of at least 25 fold was found in 36 out of 37 adenocarcinoma samples, specifically in 17 out of 18 serous carcinoma samples, in 9 out of 9 mucinous carcinoma samples and in 10 out of 10 endometroid samples.","Statistical analysis was applied to verify the significance of these results, as described below. The P value for the difference in the expression levels of immunoglobulin-like domain containing receptor 1 (ILDR1) transcripts detectable by the above amplicon in Ovary adenocarcinoma samples, serous carcinoma samples mucinous carcinoma and endometriod versus the normal tissue samples was determined by T test as 3.85e-010, 6.21e-005, 1.10e-003 and 2.94e-004 respectively.","Threshold of 25 fold over expression was found to differentiate between adenocarcinoma, serous carcinoma, mucinous carcinoma, endometriod and normal samples with P value of 3.31e-017, 1.63e-011, 1.06e-008 and 2.87e-009, respectively, as checked by exact Fisher test.","The above values demonstrate statistical significance of the results.","Normal panel\u2014The normalized quantity of each RT sample was then divided by the median of the quantities of the ovary samples (sample numbers 31, 32, 33 and 34, Table 2 above), to obtain a value of relative expression of each sample relative to median of the ovary samples, as shown in .","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: AA424839_seg18wtF1 (SEQ ID NO:197) forward primer; and AA424839_seg18wtR1 (SEQ ID NO:198) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: AA424839_seg18wt (SEQ ID NO:199).",{"@attributes":{"id":"p-0967","num":"0966"},"tables":{"@attributes":{"id":"TABLE-US-00065","num":"00065"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>AA424839_seg18wtF1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003197)"},{"entry":"AGCCACCACCATTACCCTGA"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>AA424839_seg18wtR1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003198)"},{"entry":"TGCCTTCCACTCCACGATG"},{"entry":{}},{"entry":"Amplicon\u2003>AA424839_seg18wt"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003199)"},{"entry":"AGCCACCACCATTACCCTGATTTCCACCAGGAGCTCCAGGACCGGGG"},{"entry":{}},{"entry":"GCCAAAGTCTTGGGCATTGGAAAGAAGGGAGTTGGACCCATCGTGGA"},{"entry":{}},{"entry":"GTGGAAGGCA"}]}}}}},"Expression of immunoglobulin-like domain containing receptor 1 (ILDR1) AA424839 transcripts which are detectable by amplicon as depicted in sequence name AA424839_seg14-16 (SEQ ID NO: 202) in normal and cancerous ovary tissues or different normal tissues","Expression of ILDR1 transcripts detectable by or according to seg14-16-AA424839_seg14-16 (SEQ ID NO: 202) amplicon and primers AA424839_seg14-16E1 (SEQ ID NO:200) and AA424839_seg14-16R1 (SEQ ID NO:201) was measured by real time PCR on ovary panel or normal panel. The samples used are detailed in Table 4 and in table 2 above, respectively. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1.","For ovary panel\u2014the normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 52-55, 58, 59, 63-69 and 71-78, Table 4 above), to obtain a value of fold up-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-0971","num":"0970"},"figref":"FIG. 19"},"As is evident from , the expression of ILDR1 transcripts detectable by the above amplicon in serous carcinoma, mucinous carcinoma and adenocarcinoma samples was significantly higher than in the non-cancerous samples (sample numbers 52-55, 58, 59, 63-69 and 71-78, Table 4 above). Notably an over-expression of at least 14 fold was found in 33 out of 37 adenocarcinoma samples: 14 out of 18 serous carcinoma samples, in 9 out of 9 mucinous carcinoma samples and in 10 out of 10 adenocarcinoma samples.","Statistical analysis was applied to verify the significance of these results, as described below.","The P value for the difference in the expression levels of ILDR1 transcripts detectable by the above amplicon in ovary adenocarcinoma samples, ovary serous carcinoma samples, ovary mucinous carcinoma samples and ovary endometroid samples versus the normal tissue samples was determined by T test as 1.00e-008, 4.79e-004, 4.97e-004 and 6.93e-005, respectively.","Threshold of 14 fold over expression was found to differentiate between adenocarcinoma, serous carcinoma, mucinous carcinoma, endometriod and normal samples with P value of 3.78e-012, 2.03e-007, 6.99e-008 and 2.25e-008, respectively, as checked by exact Fisher test.","The above values demonstrate statistical significance of the results.","For normal panel\u2014The normalized quantity of each RT sample was then divided by the median of the quantities of the ovary samples (sample numbers 31-34, Table 2 above), to obtain a value of relative expression of each sample relative to median of the ovary samples.","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: AA424839_seg14-16F1 (SEQ ID NO:200) forward primer; and AA424839_seg14-16R1 (SEQ ID NO:201) reverse primer.","The results demonstrating the expression of ILDR1 AA424839 transcripts which are detectable by amplicon as depicted in sequence name AA424839_seg14-16 (SEQ ID NO: 202) in different normal tissues are presented in .","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: AA424839_seg14-16F1R1 (SEQ ID NO:202).",{"@attributes":{"id":"p-0981","num":"0980"},"tables":{"@attributes":{"id":"TABLE-US-00066","num":"00066"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>AA424839_seg14-16F1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003200)"},{"entry":"GCCACCGCTACATGAAGCA"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>AA424839_seg14-16R1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003201)"},{"entry":"CTGGACGGCAGGGACAAAT"},{"entry":{}},{"entry":"Amplicon\u2003>AA424839_seg14-16F1R1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003202)"},{"entry":"GCCACCGCTACATGAAGCAGGCCCAGGCCCTAGGTCCTCAGATGATGGG"},{"entry":{}},{"entry":"AAAACCCCTGTACTGGGGGGCGGACAGGAGCTCCCAGGTTTCATCTTAT"},{"entry":{}},{"entry":"CCAATGCACCCGCTGCTGCAGCGAGATTTGTCCCTGCCGTCCAG"}]}}}}},"Expression of immunoglobulin-like domain containing receptor 1 (ILDR1) AA424839 transcripts which are detectable by amplicon as depicted in sequence name AA424839_seg11-14F3R3 (SEQ ID NO:205) in the blood-specific panel.","Expression of ILDR1 transcripts detectable by or according to seg11-14-AA424839seg11-14F3R3 (SEQ ID NO: 205) amplicon and primers AA424839seg11-14F3 (SEQ ID NO: 203) and AA424839seg11-14R3 (SEQ ID NO: 204) was measured by real time PCR on blood panel. The samples used are detailed in Table 1 above. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1. The normalized quantity of each RT sample was then divided by the median of the quantities of the kidney normal samples (sample numbers 65-67, Table 1 above), to obtain a value of relative expression of each sample relative to median of the kidney normal samples.","The results of this analysis are depicted in the histogram in . Expression of the above-indicated ILDR1 transcript was seen in several lymphomas and cell lines, however the expression was as high as in kidney normal samples.","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: seg11-14F3 forward primer; and seg11-14R3 reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: seg11-14F3R3.",{"@attributes":{"id":"p-0987","num":"0986"},"tables":{"@attributes":{"id":"TABLE-US-00067","num":"00067"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>AA424839_seg11-14F3"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003203)"},{"entry":"TCCTCCTCCTGCTGCTGATTG"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>AA424839_seg11-14R3"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003204)"},{"entry":"TGGGCCTGCTTCATGTAGCG"},{"entry":{}},{"entry":"Amplicon\u2003>AA424839_seg11-14F3R3"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003205)"},{"entry":"TCCTCCTCCTGCTGCTGATTGGAGTGTGCTGGTGCCAGTGCTGTCCTCA"},{"entry":{}},{"entry":"GTATTGCTGCTGCTATATCCGCTGTCCCTGCTGTCCTGCCCACTGCTGC"},{"entry":{}},{"entry":"TGTCCTGAGGAAGCCCTGGCCCGCCACCGCTACATGAAGCAGGCCCA"}]}}}}},"The present invention relates in particular to a putative B7\/CD28 member referred to as AI216611 and diagnostics and therapeutics based thereon. According to the present invention, Cluster AI216611 (internal ID 70605934) features 2 transcripts and 3 segments of interest, the names for which are given in Tables 49 and 50, respectively. The selected proteins are given in table 51.",{"@attributes":{"id":"p-0989","num":"0988"},"tables":{"@attributes":{"id":"TABLE-US-00068","num":"00068"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 49"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Transcripts of interest"},{"entry":"Transcript Name"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"161pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"AI216611_T0 (SEQ ID NO: 41)"]},{"entry":[{},"AI216611_T1 (SEQ ID NO: 42)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0990","num":"0989"},"tables":{"@attributes":{"id":"TABLE-US-00069","num":"00069"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 50"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segments of interest"},{"entry":"Segment Name"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"161pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"AI216611_N2 (SEQ ID NO: 126)"]},{"entry":[{},"AI216611_N4 (SEQ ID NO: 127)"]},{"entry":[{},"AI216611_N6 (SEQ ID NO: 128)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0991","num":"0990"},"tables":{"@attributes":{"id":"TABLE-US-00070","num":"00070"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 51"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Proteins of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Protein Name","Corresponding Transcripts"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["AI216611_P0 (SEQ ID NO: 43)","AI216611_T0 (SEQ ID NO: 41)"]},{"entry":["AI216611_P1 (SEQ ID NO: 44)","AI216611_T1 (SEQ ID NO: 42)"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"AI216611 is an uncharacterized gene having no full length mRNA deposited in Genbank. The protein corresponding to AI216611_P0 appears in Celera's annotation of the human genome, based on computational analysis and translation of the genome (DNA sequence accession CH471065) (Venter, J. C et al., 2001 Science 291, 1304-1351). The protein corresponding to AI216611_P0 is also listed among other sequences disclosed in WO2003025148. However, this application does not characterize its function or more particularly teach that it is a B7\/CD28 costimulatory protein.","The protein corresponding to AI216611_P1 sequence is a novel protein, that is only partially similar (186 out of 199 amino acids are the same) to a polypeptide reported in WO205108415, assigned to Biogen-Idec, which purports that this polypeptide is a transmembrane protein that may be targeted in the treatment of hyperproliferative disorders. WO205108415 does not report a function of this polypeptide. More specifically, there is no indication that it is a B7\/CD28 costimulatory protein.","According to the present invention, AI216611 is predicted to be a novel B7\/CD28 family member based on the presence of an IgV domain, a characteristic structural feature of the B7\/CD28 family members. In addition, AI216611 is similar to the known CD28 family members in its exons' sizes and the position of the IgV and transmembrane domains within these exons. Like all known B7\/CD28 members, AI216611 is also a type I membrane protein. According to the present invention, two alternatively spliced transcripts of AI216611 are provided, each one containing a unique region within the intracellular region. The expression of AI216611 and its variants was demonstrated in the present invention to be downregulated in colon cancer, further supporting an immune costimulatory role.","As noted above, contig AI216611 features 2 transcripts, which were listed in Table 49 above. A description of each protein according to the present invention is now provided.","Protein AI216611_P0 (SEQ ID NO:43) according to the present invention has an amino acid sequence as encoded by transcript AI216611_T0 (SEQ ID NO:41).","The localization of the protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: membrane.","Protein AI216611_P0 (SEQ ID NO:43) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 52, (given according to their positions on the amino acid sequence, with the alternative amino acids listed (SEQ ID NO:43)).",{"@attributes":{"id":"p-0999","num":"0998"},"tables":{"@attributes":{"id":"TABLE-US-00071","num":"00071"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 52"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["27","R \u2212> L"]},{"entry":["27","R \u2212> P"]},{"entry":["76","Q \u2212> R"]},{"entry":["129","S \u2212> R"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The protein has the following domains, as determined by using InterPro. The domains are described in Table 53:",{"@attributes":{"id":"p-1001","num":"1000"},"tables":{"@attributes":{"id":"TABLE-US-00072","num":"00072"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 53"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"InterPro domains"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Analysis",{}]},{"entry":[{},"Domain description","type","Positions on protein"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"Immunoglobulin V-set","HMMPfam","33-129"]},{"entry":[{},"domain"]},{"entry":[{},"Immunoglobulin subtype","HMMSmart","40-129"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},"Protein AI216611_P0 (SEQ ID NO:43) is encoded by the transcript AI216611_TO (SEQ ID NO:41), for which the coding portion starts at position 1 and ends at position 600. The transcript also has the following SNPs as listed in Table 54 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1003","num":"1002"},"tables":{"@attributes":{"id":"TABLE-US-00073","num":"00073"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 54"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"G \u2212> T","80, 991"]},{"entry":[{},"G \u2212> C","80, 991"]},{"entry":[{},"C \u2212> T","387, 990\u2002"]},{"entry":[{},"C \u2212> G","387, 1205"]},{"entry":[{},"T \u2212> C","906, 1063, 1090"]},{"entry":[{},"A \u2212> G","227, 1109"]},{"entry":[{},"T \u2212> A","1409, 1448\u2002"]},{"entry":[{},"G \u2212> A","1447"]},{"entry":[{},"T \u2212> G","1448"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The genomic structure of protein AI216611_P0 (SEQ ID NO:43) (number of exons relevant to the extra-cellular region of the protein, the length of these exons, the frame of the codon in which the introns are inserted and the location of the protein features and domains in the gene structure) is characteristic to the receptors of the B7\/co-stimulatory protein family, as given in table 55",{"@attributes":{"id":"p-1005","num":"1004"},"tables":{"@attributes":{"id":"TABLE-US-00074","num":"00074"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 55"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"genomic structure and protein features"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"84pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Exon","Exon","Amino-",{}]},{"entry":["number","Length","Acids","Protein feature on exon"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"84pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1","91","\u20021-30","Signal Peptide"]},{"entry":["2","327","\u200231-139","Ig-like domain"]},{"entry":["3","141","140-186","Trans-membrane region"]},{"entry":["4","41","187-200"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},"Protein AI216611_P1 (SEQ ID NO:44) according to the present invention has an amino acid sequence as encoded by transcript AI216611_T1 (SEQ ID NO:42).","The localization of the protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: membrane.","Protein AI216611_P1 (SEQ ID NO:44) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 56, (given according to their positions on the amino acid sequence, with the alternative amino acids listed (SEQ ID NO:44)).",{"@attributes":{"id":"p-1009","num":"1008"},"tables":{"@attributes":{"id":"TABLE-US-00075","num":"00075"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 56"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["27","R \u2212> L"]},{"entry":["27","R \u2212> P"]},{"entry":["76","Q \u2212> R"]},{"entry":["129","S \u2212> R"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The protein has the following domains, as determined by using InterPro. The domains are described in Table 57:",{"@attributes":{"id":"p-1011","num":"1010"},"tables":{"@attributes":{"id":"TABLE-US-00076","num":"00076"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 57"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"InterPro domains"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Analysis",{}]},{"entry":[{},"Domain description","type","Positions on protein"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"Immunoglobulin V-set","HMMPfam","33-129"]},{"entry":[{},"domain"]},{"entry":[{},"Immunoglobulin subtype","HMMSmart","40-129"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},"Protein AI216611_P1 (SEQ ID NO:44) is encoded by the transcript AI216611_T1 (SEQ ID NO:42), for which the coding portion starts at position 1 and ends at position 597. The transcript also has the following SNPs as listed in Table 58 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1013","num":"1012"},"tables":{"@attributes":{"id":"TABLE-US-00077","num":"00077"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 58"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"G \u2212> T","\u200280, 1121"]},{"entry":[{},"G \u2212> C","\u200280, 1121"]},{"entry":[{},"C \u2212> T","387, 1120"]},{"entry":[{},"C \u2212> G","387, 1335"]},{"entry":[{},"T \u2212> C","1036, 1193, 1220"]},{"entry":[{},"A \u2212> G","227, 1239"]},{"entry":[{},"T \u2212> A","1549, 1578\u2002"]},{"entry":[{},"G \u2212> A","1577"]},{"entry":[{},"T \u2212> G","1578"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"As noted above, cluster AI216611 features 3 segments, which were listed in Table 50 above. These segments are portions of nucleic acid sequences which are described herein separately because they are of particular interest. A description of each segment according to the present invention is now provided.","Segment cluster AI216611_N2 (SEQ ID NO:126) according to the present invention is supported by 4 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcripts: AI216611_T0 (SEQ ID NO:41) and AI216611_T1 (SEQ ID NO:42). Table 59 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1016","num":"1015"},"tables":{"@attributes":{"id":"TABLE-US-00078","num":"00078"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 59"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["AI216611_T0 (SEQ ID NO: 41)","92","418"]},{"entry":["AI216611_T1 (SEQ ID NO: 42)","92","418"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster AI216611_N4 (SEQ ID NO:127) according to the present invention is supported by 4 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcripts: AI216611_T0 (SEQ ID NO:41) and AI216611_T1 (SEQ ID NO:42). Table 60 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1018","num":"1017"},"tables":{"@attributes":{"id":"TABLE-US-00079","num":"00079"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 60"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["AI216611_T0 (SEQ ID NO: 41)","419","559"]},{"entry":["AI216611_T1 (SEQ ID NO: 42)","419","559"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster AI216611_N6 (SEQ ID NO:128) according to the present invention is supported by 2 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcripts: AI216611_T0 (SEQ ID NO:41) and AI216611_T1 (SEQ ID NO:42). Table 61 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1020","num":"1019"},"tables":{"@attributes":{"id":"TABLE-US-00080","num":"00080"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 61"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["AI216611_T0 (SEQ ID NO: 41)","560","885"]},{"entry":["AI216611_T1 (SEQ ID NO: 42)","690","1015"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Expression of AI216611 transcripts which are detectable by amplicon as depicted in sequence name AI216611_junc4-6F2R2 (SEQ ID NO: 208) in normal and cancerous Colon tissues and in different normal tissues","Expression of AI216611 transcripts detectable by or according to junc4-6-AI216611_junc4-6F2R2 (SEQ ID NO: 208) amplicon and primers AI216611_junc4-6F2 (SEQ ID NO: 206) and AI216611_junc4-6R2 (SEQ ID NO: 207) was measured by real time PCR on colon panel and normal panel. The samples used are detailed in Table 5 and Table 2 above, respectively. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1.","Colon panel\u2014The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 42-70, Table 5 above). Then the reciprocal of this ratio was calculated, to obtain a value of fold down-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-1024","num":"1023"},"figref":"FIG. 22"},"As is evident from , the expression of AI216611 transcripts detectable by the above amplicon in cancer samples was significantly lower than in the non-cancerous samples (sample numbers 42-70, Table 5 above). Notably down regulation of at least 5 fold was found in 27 out of 55 adenocarcinoma samples.","Statistical analysis was applied to verify the significance of these results, as described below. The P value for the difference in the expression levels of AI216611 transcripts detectable by the above amplicon in Colon cancer samples versus the normal tissue samples was determined by T test as 2.39e-005.","Threshold of 5 fold down regulation was found to differentiate between cancer and normal samples with P value of 5.09e-007 as checked by exact Fisher test.","The above values demonstrate statistical significance of the results.","Normal panel\u2014Non-detected samples (samples no. 50, 52, 54 and 56, Table 2) were assigned Ct value of 41 and were calculated accordingly. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1. The normalized quantity of each RT sample was then divided by the median of the quantities of the colon samples (sample numbers 3, 4 and 5, Table 2 above), to obtain a value of relative expression of each sample relative to median of the colon samples, as shown in .","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: AI216611junc4-6F2 (SEQ ID NO: 206) forward primer; and AI216611junc4-6R2 (SEQ ID NO: 207) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: AI216611 junc4-6F2R2 (SEQ ID NO: 208).",{"@attributes":{"id":"p-1032","num":"1031"},"tables":{"@attributes":{"id":"TABLE-US-00081","num":"00081"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>AI216611_junc4-6F2"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003206):"},{"entry":"CCGCAGTATTAATCAGCCTCATG"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>AI216611j_unc4-6R2"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003207):"},{"entry":"AATCTCCTCAGTTGTGCTTTCTTTG"},{"entry":{}},{"entry":"Amplicon\u2003>AI216611_junc4-6F2R2"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003208)"},{"entry":"CCGCAGTATTAATCAGCCTCATGTGGGTTTGTAATAAGTGTGCATATA"},{"entry":{}},{"entry":"AATTTCAGAGGAAGAGAAGACACAAACTCAAAGAAAGCACAACTGAGG"},{"entry":{}},{"entry":"AGATT"}]}}}}},"Expression of AI216611 transcripts which are detectable by amplicon as depicted in sequence name AI216611_seg2WT (SEQ ID NO: 211) in normal and cancerous Colon tissues and in different normal tissues","Expression of AI216611 transcripts detectable by or according to seg2WT\u2014AI216611_seg2WT (SEQ ID NO: 211) amplicon and primers AI216611_seg2WTF1 (SEQ ID NO: 209) and AI216611_seg2WTR1 (SEQ ID NO: 210) was measured by real time PCR on colon panel and normal panel. The samples used are detailed in Table 5 and Table 2 above, respectively.","Colon panel\u2014Non-detected sample (sample no. 33, Table 5) was assigned Ct value of 41 and was calculated accordingly. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1. The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 42-70, Table 5 above). Then the reciprocal of this ratio was calculated, to obtain a value of fold down-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-1036","num":"1035"},"figref":"FIG. 24"},"As is evident from , the expression of AI216611 transcripts detectable by the above amplicon in cancer samples was significantly lower than in the non-cancerous samples (sample numbers 42-70, Table 5 above). Notably down regulation of at least 5 fold was found in 25 out of 55 adenocarcinoma samples.","Statistical analysis was applied to verify the significance of these results, as described below. Threshold of 5 fold down regulation was found to differentiate between cancer and normal samples with P value of 2.00e-006 as checked by exact Fisher test.","The above values demonstrate statistical significance of the results.","Normal panel\u2014For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of these house keeping genes as described in Example 1, herein. The normalized quantity of each RT sample was then divided by the median of the quantities of the colon samples (sample numbers 3, 4 and 5, Table 5 above), to obtain a value of relative expression of each sample relative to median of the colon samples, as shown in .","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: AI216611_seg2WTF1 (SEQ ID NO: 209) forward primer; and AI216611_seg2WTR1 (SEQ ID NO: 210) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: AI216611_seg2WT (SEQ ID NO: 211).",{"@attributes":{"id":"p-1043","num":"1042"},"tables":{"@attributes":{"id":"TABLE-US-00082","num":"00082"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>AI216611_seg2WTF1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003209)"},{"entry":"GAACGCAGAAGATCGTGGAGT"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>AI216611_seg2WTR1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003210)"},{"entry":"CTGAAGAGCTGGATGGAGCC"},{"entry":{}},{"entry":"Amplicon\u2003>AI216611_seg2WT"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003211)"},{"entry":"GAACGCAGAAGATCGTGGAGTGGAAACCAGGGACTCAGGCCAACAT"},{"entry":{}},{"entry":"CTCTCAAAGCCACAAGGACAGAGTCTGCACCTTTGACAACGGCTCC"},{"entry":{}},{"entry":"ATCCAGCTCTTCAG"}]}}}}},"Expression of AI216611 transcripts which are detectable by amplicon as depicted in sequence name AI216611junc4-6 (SEQ ID NO: 214) in the blood-specific panel.","Expression of AI216611 transcripts detectable by or according to junc4-6-junc4-6F4R4 (SEQ ID NO: 214) amplicon and primers junc4-6F4 (SEQ ID NO: 212) and junc4-6R4 (SEQ ID NO: 213) was measured by real time PCR on blood panel. The samples used are detailed in Table 1 above. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1. The normalized quantity of each RT sample was then divided by the median of the quantities of the kidney normal samples (sample numbers 65-67, Table 1 above), to obtain a value of relative expression of each sample relative to median of the kidney normal samples.","The results of this analysis are depicted in the histogram in . Expression of the above-indicated AI216611 transcript was much high in normal samples checked relative to the different blood specific samples in the panel.","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: junc4-6F4 (SEQ ID NO: 212) forward primer; and junc4-6R4 reverse primer (SEQ ID NO: 213).","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: junc4-6F4R4 (SEQ ID NO: 214).",{"@attributes":{"id":"p-1049","num":"1048"},"tables":{"@attributes":{"id":"TABLE-US-00083","num":"00083"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003AI216611junc4-6F4"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003\u2003212):"},{"entry":"CTGCACTTTGTCGCTGTCATC"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003AI216611junc4-6R4"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003213):"},{"entry":"CAATCTCCTCAGTTGTGCTTTCTTTG"},{"entry":{}},{"entry":"Amplicon\u2003AI216611junc4-6F4R4\u2003(SEQ\u2003ID\u2003NO:\u2003214):"},{"entry":"CTGCACTTTGTCGCTGTCATCCTTGCTTTTCTCGCTGCTGTGGCCGCAG"},{"entry":{}},{"entry":"TATTAATCAGCCTCATGTGGGTTTGTAATAAGTGTGCATATAAATTTCA"},{"entry":{}},{"entry":"GAGGAAGAGAAGACACAAACTCAAAGAAAGCACAACTGAGGAGATTG"}]}}}}},"Expression of AI216611 transcripts which are detectable by amplicon as depicted in sequence name AI216611junc2-4seg5 (SEQ ID NO: 217) in the blood-specific panel.","Expression of AI216611 transcripts detectable by or according to junc2-4seg5-junc2-4seg5F3R4 amplicon (SEQ ID NO: 217) and primers junc2-4seg5F3 (SEQ ID NO: 215) and junc2-4seg5R4 (SEQ ID NO: 216) was measured by real time PCR on blood panel. The samples used are detailed in Table 1 above. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1. The normalized quantity of each RT sample was then divided by the median of the quantities of the kidney normal samples (sample numbers 65-67, Table 1 above), to obtain a value of relative expression of each sample relative to median of the kidney normal samples.","The results of this analysis are depicted in the histogram in . Expression of the above-indicated AI216611 transcript was much high in normal samples checked relative to the different blood specific samples in the panel.","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: junc2-4seg5F3 (SEQ ID NO: 215) forward primer; and junc2-4seg5R4 (SEQ ID NO: 216) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: junc2-4seg5F3R4 (SEQ ID NO: 217).",{"@attributes":{"id":"p-1055","num":"1054"},"tables":{"@attributes":{"id":"TABLE-US-00084","num":"00084"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003AI216611\u2003junc2-4seg5F3"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003215):"},{"entry":"TGCTGCACGTCTCTGAGATCC"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003AI216611\u2003junc2-4seg5R4"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003216):"},{"entry":"CACCTCTGGCCTCAAAACCACTC"},{"entry":{}},{"entry":"Amplicon\u2003>AI216611\u2003junc2-4seg5F3R4"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003217)"},{"entry":"TGCTGCACGTCTCTGAGATCCTCTATGAAGACCTGCACTTTGTCGCTGT"},{"entry":{}},{"entry":"CATCCTTGCTTTTCTCGCTGCTGTGGCCGCAGTATTAATCAGCCTCATG"},{"entry":{}},{"entry":"TGGGTTTGTAATAAGTGTGCATATAAATTTCAGAGGAAGAGAAGACACA"},{"entry":{}},{"entry":"AACTCAAAGGTAACCCCCTGGGCCTTGTGATAATCCATGAGTGGTTTTG"},{"entry":{}},{"entry":"AGGCCAGAGGTG"}]}}}}},"Expression of AI216611 transcripts which are detectable by amplicon as depicted in sequence name AI216611_junc2-4seg5F2R2 (SEQ ID NO: 220) in normal and cancerous Colon tissues","Expression of AI216611 transcripts detectable by or according to junc2-4seg5F2R2\u2014AI216611_junc2-4seg5F2R2 (SEQ ID NO: 220) amplicon and primers AI216611_junc2-4seg5F2 (SEQ ID NO: 218) and AI216611_junc2-4seg5R2 (SEQ ID NO: 219) was measured by real time PCR on colon panel. The samples used are detailed in Table 5 above. Non-detected samples (samples no. 28, 33, 83, 85, 90 and 63, Table 5) were assigned Ct value of 41 and were calculated accordingly. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1. The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 42-62 and 64-70, Table 5 above). Then the reciprocal of this ratio was calculated, to obtain a value of fold down-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-1058","num":"1057"},"figref":"FIG. 28"},"As is evident from , the expression of AI216611 transcripts detectable by the above amplicon in cancer samples was significantly lower than in the non-cancerous samples (sample numbers 42-62 and 64-70, Table 5 above). Notably down regulation of at least 5 fold was found in 31 out of 55 adenocarcinoma samples.","Statistical analysis was applied to verify the significance of these results, as described below. The P value for the difference in the expression levels of AI216611 transcripts detectable by the above amplicon in Colon cancer samples versus the normal tissue samples was determined by T test as 5.29e-003.","Threshold of 5 fold down regulation was found to differentiate between cancer and normal samples with P value of 4.18e-008 as checked by exact Fisher test.","The above values demonstrate statistical significance of the results.","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: AI216611_junc2-4seg5F2 (SEQ ID NO: 218) forward primer; and AI216611_junc2-4seg5R2 (SEQ ID NO: 219) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: AI216611_junc2-4seg5F2R2 (SEQ ID NO: 220).",{"@attributes":{"id":"p-1065","num":"1064"},"tables":{"@attributes":{"id":"TABLE-US-00085","num":"00085"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>AI216611_junc2-4seg5F2"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003218)"},{"entry":"GCTGCACGTCTCTGAGATCCT"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>AI216611_junc2-4seg5R2"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003219)"},{"entry":"CACCTCTGGCCTCAAAACCA"},{"entry":{}},{"entry":"Amplicon\u2003>AI216611_junc2-4seg5F2R2"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003220)"},{"entry":"GCTGCACGTCTCTGAGATCCTCTATGAAGACCTGCACTTTGTCGCTGTC"},{"entry":{}},{"entry":"ATCCTTGCTTTTCTCGCTGCTGTGGCCGCAGTATTAATCAGCCTCATGT"},{"entry":{}},{"entry":"GGGTTTGTAATAAGTGTGCATATAAATTTCAGAGGAAGAGAAGACACAA"},{"entry":{}},{"entry":"ACTCAAAGGTAACCCCCTGGGCCTTGTGATAATCCATGAGTGGTTTTGA"},{"entry":{}},{"entry":"GGCCAGAGGTG"}]}}}}},"The present invention relates to LOC253012 polypeptides, novel splice variants and diagnostics and therapeutics based thereon.","According to the present invention, Cluster H686541 (internal ID 76432882) features 10 transcripts and 3 segments of interest, the names for which are given in Tables 62 and 63, respectively. The selected protein variants are given in table 64.",{"@attributes":{"id":"p-1068","num":"1067"},"tables":{"@attributes":{"id":"TABLE-US-00086","num":"00086"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 62"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Transcripts of interest"},{"entry":"Transcript Name"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"161pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"H68654_1_T0 (SEQ ID NO: 25)"]},{"entry":[{},"H68654_1_T4 (SEQ ID NO: 26)"]},{"entry":[{},"H68654_1_T5 (SEQ ID NO: 27)"]},{"entry":[{},"H68654_1_T8 (SEQ ID NO: 28)"]},{"entry":[{},"H68654_1_T15 (SEQ ID NO: 29)"]},{"entry":[{},"H68654_1_T16 (SEQ ID NO: 30)"]},{"entry":[{},"H68654_1_T17 (SEQ ID NO: 31)"]},{"entry":[{},"H68654_1_T18 (SEQ ID NO: 32)"]},{"entry":[{},"H68654_1_T19 (SEQ ID NO: 33)"]},{"entry":[{},"H68654_1_T20 (SEQ ID NO: 34)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-1069","num":"1068"},"tables":{"@attributes":{"id":"TABLE-US-00087","num":"00087"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 63"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segments of interest"},{"entry":"Segment Name"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"168pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"H68654_1_N3 (SEQ ID NO: 123)"]},{"entry":[{},"H68654_1_N7 (SEQ ID NO: 124)"]},{"entry":[{},"H68654_1_N12 (SEQ ID NO: 125)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-1070","num":"1069"},"tables":{"@attributes":{"id":"TABLE-US-00088","num":"00088"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 64"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Proteins of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Protein Name","Corresponding Transcripts"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["H68654_1_P2 (SEQ ID NO: 35)","H68654_1_T0 (SEQ ID NO: 25);"]},{"entry":[{},"H68654_1_T5 (SEQ ID NO: 27)"]},{"entry":["H68654_1_P5 (SEQ ID NO: 36)","H68654_1_T4 (SEQ ID NO: 26)"]},{"entry":["H68654_1_P7 (SEQ ID NO: 37)","H68654_1_T8 (SEQ ID NO: 28)"]},{"entry":["H68654_1_P12 (SEQ ID NO: 38)","H68654_1_T15 (SEQ ID NO: 29);"]},{"entry":[{},"H68654_1_T16 (SEQ ID NO: 30);"]},{"entry":[{},"H68654_1_T18 (SEQ ID NO: 32)"]},{"entry":["H68654_1_P13 (SEQ ID NO: 39)","H68654_1_T17 (SEQ ID NO: 31);"]},{"entry":[{},"H68654_1_T19 (SEQ ID NO: 33)"]},{"entry":["H68654_1_P14 (SEQ ID NO: 40)","H68654_1_T20 (SEQ ID NO: 34)"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"These sequences are variants of the known protein hypothetical protein LOC253012 isoform 1 (RefSeq accession identifier NP001034461 (SEQ ID NO: 35), NP937794 (SEQ ID NO: 36)), referred to herein as the previously known protein.","The known LOC253012 is a hypothetical protein that was computationally discovered during the secreted protein discovery initiative project (SPDI), a large-scale effort to identify novel human secreted and transmembrane proteins (Clark et al 2003, Genome Res. 13: 2265-2270). Its closest paralog that is experimentally validated is the hepatocyte cell adhesion molecule (Refseq accession NP689935) (evalue e-21).","LOC253012 antigen has been reported in European patent application Number EP162070, assigned to Genentech Inc. purports that LOC253012 (PRO346), corresponding to H686541_P2 (SEQ ID NO:35), is differentially expressed in a lung tumor samples. This patent specification alleges that the corresponding polypeptide can be used for developing antibodies against the disclosed PRO antigens, including PRO346. The patent application further suggests that antibodies to a genus of polypeptides reported therein that includes PRO346 may be used for treating and diagnosing cancer and specifically for diagnosing and treating lung cancer. The Genentech patent application does not teach, however, that LOC253012 (PRO346) is differentially expressed in small cell lung carcinoma. Also, there is no specific teaching in Genentech Inc. patent application that the anti-PRO346 antibodies can be used for treating small cell lung carcinomas and\/or for modulating co-stimulation of the APC\/T cell activity. Also, there is no teaching in the Genentech Inc. patent application that the PRO346 is an immune costimulatory protein or more specifically a B7 family member. There is no teaching in the Genentech Inc. patent application of the use of antibodies against the PRO346 antigen for modulating immune co-stimulation or particularly the B7 co-stimulatory pathway.","According to the present invention, LOC253012 was predicted to be a novel immune costimulatory protein and in particular a B7 co-stimulatory protein. The prediction was based on the presence of both an IgV domain and IgC2, a characteristic structural feature of the B7-family members. Like other B7 members, LOC253012 is also a type I membrane protein. LOC253012 and its variants were demonstrated in the present invention to be overexpressed in lung cancer.","MED discovery engine described in Example 1 herein, was used to assess the expression of LOC253012 transcripts. Expression data for Affymetrix probe set 242601_at representing the LOC253012 gene data is shown in . As evident from the scatter plot, presented in , the expression of LOC253012 transcripts detectable with the above probe sets was higher in lung cancer compared to normal lung samples.","As noted above, cluster H68654 features 10 transcripts, which were listed in Table 62 above. These transcripts encode for proteins which are variants of protein hypothetical protein LOC253012 isoform 1 (SEQ ID NO:35). A description of each variant protein according to the present invention is now provided.","Variant protein H686541_P2 (SEQ ID NO:35) according to the present invention has an amino acid sequence as encoded by transcripts H686541_T0 (SEQ ID NO:25) and H686541_T5 (SEQ ID NO:27).","The localization of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: membrane.","Variant protein H686541_P2 (SEQ ID NO:35) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 65, (given according to their positions on the amino acid sequence, with the alternative amino acids listed).",{"@attributes":{"id":"p-1080","num":"1079"},"tables":{"@attributes":{"id":"TABLE-US-00089","num":"00089"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 65"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["33","K \u2212> R"]},{"entry":["86","K \u2212> N"]},{"entry":["204","L \u2212> P"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The variant protein has the following domains, as determined by using InterPro. The domains are described in Table 66:",{"@attributes":{"id":"p-1082","num":"1081"},"tables":{"@attributes":{"id":"TABLE-US-00090","num":"00090"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 66"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"InterPro domains"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Analysis",{}]},{"entry":["Domain description","type","Positions on protein"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Immunoglobulin subtype","HMMSmart","38-141, 155-235, 255-333"]},{"entry":["Immunoglobulin-like","ProfileScan","149-233, 235-331"]},{"entry":["Immunoglobulin-like","HMMPfam","163-221"]},{"entry":["Immunoglobulin V-set","HMMPfam","\u200231-143"]},{"entry":["Immunoglobulin C2 type","HMMSmart","164-226"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"The coding portion of transcript H686541_T0 (SEQ ID NO:25) starts at position 79 and ends at position 1464. The transcript also has the following SNPs as listed in Table 67 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1084","num":"1083"},"tables":{"@attributes":{"id":"TABLE-US-00091","num":"00091"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 67"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","176, 1545"]},{"entry":[{},"G \u2212> A","336, 1754"]},{"entry":[{},"G \u2212> C","\u2002336"]},{"entry":[{},"T \u2212> C","564, 689\u2002"]},{"entry":[{},"A \u2212> T","1545"]},{"entry":[{},"G \u2212> T","1684"]},{"entry":[{},"T \u2212> A","1755"]},{"entry":[{},"T \u2212> G","1755"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The coding portion of transcript H686541_T5 (SEQ ID NO:27) starts at position 79 and ends at position 1464. The transcript also has the following SNPs as listed in Table 68 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1086","num":"1085"},"tables":{"@attributes":{"id":"TABLE-US-00092","num":"00092"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 68"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","176, 1545"]},{"entry":[{},"G \u2212> A","336, 1754"]},{"entry":[{},"G \u2212> C","\u2002336"]},{"entry":[{},"T \u2212> C","564, 689\u2002"]},{"entry":[{},"A \u2212> T","1545"]},{"entry":[{},"G \u2212> T","1684"]},{"entry":[{},"T \u2212> A","1755"]},{"entry":[{},"T \u2212> G","1755"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The genomic structure of protein H686541_P2 (SEQ ID NO: 35) (number of exons relevant to the extra-cellular region of the protein, the length of these exons, the frame of the codon in which the introns are inserted and the location of the protein features and domains in the gene structure) is characteristic to the ligands of the B7\/co-stimulatory protein family, as given in table 69.",{"@attributes":{"id":"p-1088","num":"1087"},"tables":{"@attributes":{"id":"TABLE-US-00093","num":"00093"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 69"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"genomic structure and protein features"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"84pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Exon","Exon","Amino-",{}]},{"entry":["number","Length","Acids","Protein feature on exon"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"84pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1","79","\u20021-26","Signal Peptide"]},{"entry":["2","351","\u200227-143","IgV domain"]},{"entry":["3","285","144-238","IgC2 domain"]},{"entry":["4","297","239-337","Ig-like domain"]},{"entry":["5","126","338-379","Trans-membrane region"]},{"entry":["6","25","380-387"]},{"entry":["7","38","388-400"]},{"entry":["8","74","401-425"]},{"entry":["9","110","426-462"]},{"entry":["10","1","462-462"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein H686541_P5 (SEQ ID NO:36) according to the present invention has an amino acid sequence encoded by transcripts H686541_T4 (SEQ ID NO:26).","The localization of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: membrane.","Variant protein H686541_P5 (SEQ ID NO:36) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 70, (given according to their positions on the amino acid sequence, with the alternative amino acids listed (SEQ ID NO:36)).",{"@attributes":{"id":"p-1092","num":"1091"},"tables":{"@attributes":{"id":"TABLE-US-00094","num":"00094"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 70"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["21","K \u2212> R"]},{"entry":["74","K \u2212> N"]},{"entry":["192","L \u2212> P"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The variant protein has the following domains, as determined by using InterPro. The domains are described in Table 71:",{"@attributes":{"id":"p-1094","num":"1093"},"tables":{"@attributes":{"id":"TABLE-US-00095","num":"00095"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 71"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"InterPro domains"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Analysis",{}]},{"entry":["Domain description","type","Positions on protein"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Immunoglobulin C2 type","HMMSmart","152-214"]},{"entry":["Immunoglobulin subtype","HMMSmart","26-129, 143-223, 243-321"]},{"entry":["Immunoglobulin-like","HMMPfam","151-209"]},{"entry":["Immunoglobulin V-set","HMMPfam","\u200219-131"]},{"entry":["Immunoglobulin-like","Profile","137-221, 223-319"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein H686541_P5 (SEQ ID NO:36) is encoded by the transcript H686541_T4 (SEQ ID NO:26), for which the coding portion starts at position 102 and ends at position 1451. The transcript also has the following SNPs as listed in Table 72 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1096","num":"1095"},"tables":{"@attributes":{"id":"TABLE-US-00096","num":"00096"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 72"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","163, 1532"]},{"entry":[{},"G \u2212> A","323, 1741"]},{"entry":[{},"G \u2212> C","\u2002323"]},{"entry":[{},"T \u2212> C","551, 676\u2002"]},{"entry":[{},"A \u2212> T","1532"]},{"entry":[{},"G \u2212> T","1671"]},{"entry":[{},"T \u2212> A","1742"]},{"entry":[{},"T \u2212> G","1742"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"Variant protein H686541_P7 (SEQ ID NO:37) according to the present invention has an amino acid sequence encoded by transcript H686541_T8 (SEQ ID NO:28). Alignment of H686541_P7 (SEQ ID NO:37) to one or more previously published protein sequences are shown in . A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","1. Comparison report between H686541_P7 (SEQ ID NO:37) and known proteins NP937794 and Q6UXI0_HUMAN (SEQ ID NO: 36) ():","A. An isolated chimeric polypeptide encoding for H686541_P7 (SEQ ID NO:37), comprising a first amino acid sequence being at least 90% homologous to MWLKVFTTFLSFATGACSGLKVTVPSHTVHGVRGQALYLPVHYGFHTPASDIQII WLFERPHTMPKYLLGSVNKSVVPDLEYQHKFTMMPPNASLLINPLQFPDEGNYI VKVNIQGNGTLSASQKIQVTVDDPVTKPVVQIHPPSGAVEYVGNMTLTCHVEG GTRLAYQWLKNGRPVHTSSTYSFSPQNNTLHIAPVTKEDIGNYSCLVRNPVSEM ESDIIMPIIYYGPYGLQVNSDKGLKVGEVFTVDLGEAILFDCSADSHPPNTYSWIR RTDNTTYIIKHGPRLEVASEKVAQKTMDYVCCAYNNITGRQDETHFTVIITSVGL EKLAQKGKSLSPLASITGISLFLIISMCLLFLWKKYQPYK corresponding to amino acids 1-367 of known proteins NP937794 and Q6UXI0_HUMAN (SEQ ID NO: 36), which also corresponds to amino acids 1-367 of H686541_P7 (SEQ ID NO:37), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95, 96, 97, 98 or 99% homologous to a polypeptide having the sequence GQKQNTGKLKHFQAMKMLWMTSEYMNLLLFQMFLVFPGSQAGLFQPLIVYRG KICTVQCMKLFSTSLPSSKTIQSELSWAKQYIRVKF (SEQ ID NO: 290) corresponding to amino acids 368-455 of H686541_P7 (SEQ ID NO:37), wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","B. An isolated polypeptide encoding for an edge portion of H686541_P7 (SEQ ID NO:37), comprising an amino acid sequence being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95, 96, 97, 98 or 99% homologous to the sequence GQKQNTGKLKHFQAMKMLWMTSEYMNLLLFQMFLVFPGSQAGLFQPLIVYRG KICTVQCMKLFSTSLPSSKTIQSELSWAKQYIRVKF (SEQ ID NO: 290) of H686541_P7 (SEQ ID NO:37).","2. Comparison report between H686541_P7 (SEQ ID NO:37) and known proteins NP001034461 (SEQ ID NO: 35) ):","A. An isolated chimeric polypeptide encoding for H686541_P7 (SEQ ID NO:37), comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95, 96, 97, 98 or 99%, homologous to a polypeptide having the sequence MWLKVFTTFLSFAT (SEQ ID NO: 289) corresponding to amino acids 1-14 of H686541_P7 (SEQ ID NO:37), a second amino acid sequence being at least 90% homologous to GACSGLKVTVPSHTVHGVRGQALYLPVHYGFHTPASDIQIIWLFERPHTMPKYL LGSVNKSVVPDLEYQHKFTMMPPNASLLINPLQFPDEGNYIVKVNIQGNGTLSA SQKIQVTVDDPVTKPVVQIHPPSGAVEYVGNMTLTCHVEGGTRLAYQWLKNGR PVHTSSTYSFSPQNNTLHIAPVTKEDIGNYSCLVRNPVSEMESDIIMPIIYYGPYGL QVNSDKGLKVGEVFTVDLGEAILFDCSADSHPPNTYSWIRRTDNTTYIIKHGPRL EVASEKVAQKTMDYVCCAYNNITGRQDETHFTVIITSVGLEKLAQKGKSLSPLAS ITGISLFLIISMCLLFLWKKYQPYK corresponding to amino acids 27-379 of known proteins NP001034461 (SEQ ID NO: 35), which also corresponds to amino acids 15-367 of H686541_P7 (SEQ ID NO:37), and a third amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95, 96, 97, 98 or 99% homologous to a polypeptide having the sequence GQKQNTGKLKHFQAMKMLWMTSEYMNLLLFQMFLVFPGSQAGLFQPLIVYRG KICTVQCMKLFSTSLPSSKTIQSELSWAKQYIRVKF (SEQ ID NO: 290) corresponding to amino acids 368-455 of H686541_P7 (SEQ ID NO:37), wherein said first amino acid sequence, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","B. An isolated polypeptide encoding for a head of H686541_P7 (SEQ ID NO:37), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95, 96, 97, 98 or 99% homologous to the sequence MWLKVFTTFLSFAT (SEQ ID NO: 289) of H686541_P7 (SEQ ID NO:37).","C. An isolated polypeptide encoding for an edge portion of H686541_P7 (SEQ ID NO:37), comprising an amino acid sequence being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95, 96, 97, 98 or 99% homologous to the sequence GQKQNTGKLKHFQAMKMLWMTSEYMNLLLFQMFLVFPGSQAGLFQPLIVYRG KICTVQCMKLFSTSLPSSKTIQSELSWAKQYIRVKF (SEQ ID NO: 290) of H686541_P7 (SEQ ID NO:37).","The localization of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: membrane.","Variant protein H686541_P7 (SEQ ID NO:37) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 73, (given according to their positions on the amino acid sequence, with the alternative amino acids listed (SEQ ID NO:37)).",{"@attributes":{"id":"p-1107","num":"1106"},"tables":{"@attributes":{"id":"TABLE-US-00097","num":"00097"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 73"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["21","K \u2212> R"]},{"entry":["74","K \u2212> N"]},{"entry":["192","L \u2212> P"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The variant protein has the following domains, as determined by using InterPro. The domains are described in Table 74:",{"@attributes":{"id":"p-1109","num":"1108"},"tables":{"@attributes":{"id":"TABLE-US-00098","num":"00098"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 74"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"InterPro domains"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Analysis",{}]},{"entry":["Domain description","type","Positions on protein"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Immunoglobulin subtype","HMMSmart","26-129, 143-223, 243-321"]},{"entry":["Immunoglobulin C2 type","HMMSmart","152-214"]},{"entry":["Immunoglobulin-like","ProfileScan","137-221, 223-319"]},{"entry":["Immunoglobulin V-set","HMMPfam","\u200219-131"]},{"entry":["Immunoglobulin-like","HMMPfam","151-209"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein H686541_P7 (SEQ ID NO:37) is encoded by the transcript H686541_T8 (SEQ ID NO:28), for which the coding portion starts at position 102 and ends at position 1466. The transcript also has the following SNPs as listed in Table 75 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1111","num":"1110"},"tables":{"@attributes":{"id":"TABLE-US-00099","num":"00099"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 75"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","163, 1507"]},{"entry":[{},"G \u2212> A","323, 1716"]},{"entry":[{},"G \u2212> C","\u2002323"]},{"entry":[{},"T \u2212> C","551, 676\u2002"]},{"entry":[{},"A \u2212> T","1507"]},{"entry":[{},"G \u2212> T","1646"]},{"entry":[{},"T \u2212> A","1717"]},{"entry":[{},"T \u2212> G","1717"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"Variant protein H686541_P12 (SEQ ID NO:38) according to the present invention has an amino acid sequence as encoded by transcripts H686541_T15 (SEQ ID NO:29), H686541_T16 (SEQ ID NO:30) and H686541_T18 (SEQ ID NO:32). Alignment of H686541_P12 (SEQ ID NO:38) to previously published protein sequences are shown in . A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","1. Comparison report between H686541_P12 (SEQ ID NO:38) and known proteins NP937794 and Q6UXI0_HUMAN (SEQ ID NO: 36) ():","A. An isolated chimeric polypeptide encoding for H686541_P12 (SEQ ID NO:38), comprising a first amino acid sequence being at least 90% homologous to MWLKVFTTFLSFATGACSGLKVTVPSHTVHGVRGQALYLPVHYGFHTPASDIQII WLFERPHTMPKYLLGSVNKSVVPDLEYQHKFTMMPPNASLLINPLQFPDEGNYI VKVNIQGNGTLSASQKIQVTVDDPVTKPVVQIHPPSGAVEYVGNMTLTCHVEG GTRLAYQWLKNGRPVHTSSTYSFSPQNNTLHIAPVTKEDIGNYSCLVRNPVSEM ESDIIMPIIYYGPYGLQVNSDKGLKVGEVFTVDLGEAILFDCSADSHPPNTYSWIR RTDNTTYIIKHGPRLEVASEKVAQKTMDYVCCAYNNITGRQDETHFTVIITSVGL EKLAQKGKSLSPLASITGISLFLIISMCLLFLWKKYQPYKVIKQKLEGRPETEYRKA QTFSGHEDALDDFGIYEFVAFPDVSGVSR corresponding to amino acids 1-413 of known proteins NP937794 and Q6UXI0_HUMAN (SEQ ID NO: 36), which also corresponds to amino acids 1-413 of H686541_P12 (SEQ ID NO:38), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95, 96, 97, 98 or 99% homologous to a polypeptide having the sequence VGFPSG (SEQ ID NO: 291) corresponding to amino acids 414-419 of H686541_P12 (SEQ ID NO:38), wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","B. An isolated polypeptide encoding for an edge portion of H686541_P12 (SEQ ID NO:38), comprising an amino acid sequence being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95, 96, 97, 98 or 99% homologous to the sequence VGFPSG (SEQ ID NO: 291) of H686541_P12 (SEQ ID NO:38).","2. Comparison report between H686541_P12 (SEQ ID NO:38) and known proteins NP001034461 (SEQ ID NO: 35) ():","A. An isolated chimeric polypeptide encoding for H686541_P12 (SEQ ID NO:38), comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95, 96, 97, 98 or 99%, homologous to a polypeptide having the sequence MWLKVFTTFLSFAT (SEQ ID NO: 289) corresponding to amino acids 1-14 of H686541_P12 (SEQ ID NO:38), a second amino acid sequence being at least 90% homologous to GACSGLKVTVPSHTVHGVRGQALYLPVHYGFHTPASDIQIIWLFERPHTMPKYL LGSVNKSVVPDLEYQHKFTMMPPNASLLINPLQFPDEGNYIVKVNIQGNGTLSA SQKIQVTVDDPVTKPVVQIHPPSGAVEYVGNMTLTCHVEGGTRLAYQWLKNGR PVHTSSTYSFSPQNNTLHIAPVTKEDIGNYSCLVRNPVSEMESDIIMPIIYYGPYGL QVNSDKGLKVGEVFTVDLGEAILFDCSADSHPPNTYSWIRRTDNTTYIIKHGPRL EVASEKVAQKTMDYVCCAYNNITGRQDETHFTVIITSVGLEKLAQKGKSLSPLAS ITGISLFLIISMCLLFLWKKYQPYKVIKQKLEGRPETEYRKAQTFSGHEDALDDFGI YEFVAFPDVSGVSR corresponding to amino acids 27-425 of known proteins NP001034461 (SEQ ID NO: 35), which also corresponds to amino acids 15-413 of H686541_P12 (SEQ ID NO:38), and a third amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95, 96, 97, 98 or 99% homologous to a polypeptide having the sequence VGFPSG (SEQ ID NO: 291) corresponding to amino acids 414-419 of H686541_P12 (SEQ ID NO:38), wherein said first amino acid sequence, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","B. An isolated polypeptide encoding for a head of H686541_P12 (SEQ ID NO:38), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95, 96, 97, 98 or 99% homologous to the sequence MWLKVFTTFLSFAT (SEQ ID NO: 289) of H686541_P12 (SEQ ID NO:38).","C. An isolated polypeptide encoding for an edge portion of H686541_P12 (SEQ ID NO:38), comprising an amino acid sequence being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95, 96, 97, 98 or 99% homologous to the sequence VGFPSG (SEQ ID NO: 291) of H686541_P12 (SEQ ID NO:38).","The localization of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: membrane.","Variant protein H686541_P12 (SEQ ID NO:38) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 76, (given according to their positions on the amino acid sequence, with the alternative amino acids listed (SEQ ID NO:38)).",{"@attributes":{"id":"p-1122","num":"1121"},"tables":{"@attributes":{"id":"TABLE-US-00100","num":"00100"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 76"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["21","K \u2212> R"]},{"entry":["74","K \u2212> N"]},{"entry":["192","L \u2212> P"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The variant protein has the following domains, as determined by using InterPro. The domains are described in Table 77.",{"@attributes":{"id":"p-1124","num":"1123"},"tables":{"@attributes":{"id":"TABLE-US-00101","num":"00101"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 77"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"InterPro domains"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Analysis",{}]},{"entry":["Domain description","type","Positions on protein"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Immunoglobulin subtype","HMMSmart","26-129, 143-223, 243-321"]},{"entry":["Immunoglobulin-like","HMMPfam","151-209"]},{"entry":["Immunoglobulin C2 type","HMMSmart","152-214"]},{"entry":["Immunoglobulin V-set","HMMPfam","\u200219-131"]},{"entry":["Immunoglobulin-like","ProfileScan","137-221, 223-319"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"The coding portion of transcript H686541_T15 (SEQ ID NO:29) starts at position 102 and ends at position 1358. The transcript also has the following SNPs as listed in Table 78 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1126","num":"1125"},"tables":{"@attributes":{"id":"TABLE-US-00102","num":"00102"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 78"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","163, 1586"]},{"entry":[{},"G \u2212> A","323, 1795"]},{"entry":[{},"G \u2212> C","\u2002323"]},{"entry":[{},"T \u2212> C","551, 676\u2002"]},{"entry":[{},"A \u2212> T","1586"]},{"entry":[{},"G \u2212> T","1725"]},{"entry":[{},"T \u2212> A","1796"]},{"entry":[{},"T \u2212> G","1796"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The coding portion of transcript H686541_ starts at position 102 and ends at position 1358. The transcript also has the following SNPs as listed in Table 79 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1128","num":"1127"},"tables":{"@attributes":{"id":"TABLE-US-00103","num":"00103"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 79"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","163, 1588"]},{"entry":[{},"G \u2212> A","323, 1797"]},{"entry":[{},"G \u2212> C","\u2002323"]},{"entry":[{},"T \u2212> C","551, 676\u2002"]},{"entry":[{},"A \u2212> T","1588"]},{"entry":[{},"G \u2212> T","1727"]},{"entry":[{},"T \u2212> A","1798"]},{"entry":[{},"T \u2212> G","1798"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The coding portion of transcript H686541_T18 (SEQ ID NO:32) starts at position 102 and ends at position 1358. The transcript also has the following SNPs as listed in Table 80 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1130","num":"1129"},"tables":{"@attributes":{"id":"TABLE-US-00104","num":"00104"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 80"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","\u2002163"]},{"entry":[{},"G \u2212> A","323, 2117"]},{"entry":[{},"G \u2212> C","\u2002323"]},{"entry":[{},"T \u2212> C","551, 676\u2002"]},{"entry":[{},"T \u2212> A","1603"]},{"entry":[{},"T \u2212> G","1603"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"Variant protein H686541_P13 (SEQ ID NO:39) according to the present invention has an amino acid sequence as encoded by transcripts H686541_T17 (SEQ ID NO:31) and H686541_T19 (SEQ ID NO:33). Alignments to one or more previously published protein sequences are shown in . A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","1. Comparison report between H686541_P13 (SEQ ID NO:39) and known proteins NP937794 and Q6UXI0_HUMAN (SEQ ID NO: 36) ():","A. An isolated chimeric polypeptide encoding for H686541_P13 (SEQ ID NO:39), comprising a amino acid sequence being at least 90% homologous to MWLKVFTTFLSFATGACSGLKVTVPSHTVHGVRGQALYLPVHYGFHTPASDIQII WLFERPHTMPKYLLGSVNKSVVPDLEYQHKFTMMPPNASLLINPLQFPDEGNYI VKVNIQGNGTLSASQKIQVTVDDPVTKPVVQIHPPSGAVEYVGNMTLTCHVEG GTRLAYQWLKNGRPVHTSSTYSFSPQNNTLHIAPVTKEDIGNYSCLVRNPVSEM ESDIIMPIIYYGPYGLQVNSDKGLKVGEVFTVDLGEAILFDCSADSHPPNTYSWIR RTDNTTYIIKHGPRLEVASEKVAQKTMDYVCCAYNNITGRQDETHFTVIITSVGL EKLAQKGKSLSPLASITGISLFLIISMCLLFLWKKYQPYKVIKQKLEGR corresponding to amino acids 1-376 of known proteins NP937794 and Q6UXI0_HUMAN (SEQ ID NO: 36), which also corresponds to amino acids 1-376 of H686541_P13 (SEQ ID NO:39), wherein said and first amino acid sequence are contiguous and in a sequential order.","2. Comparison report between H686541_P13 (SEQ ID NO:39) and known proteins NP001034461 (SEQ ID NO: 35) ():","A. An isolated chimeric polypeptide encoding for H686541_P13 (SEQ ID NO:39), comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95, 96, 97, 98 or 99%, homologous to a polypeptide having the sequence MWLKVFTTFLSFAT (SEQ ID NO: 289) corresponding to amino acids 1-14 of H686541_P13 (SEQ ID NO:39), and a second amino acid sequence being at least 90% homologous to GACSGLKVTVPSHTVHGVRGQALYLPVHYGFHTPASDIQIIWLFERPHTMPKYL LGSVNKSVVPDLEYQHKFTMMPPNASLLINPLQFPDEGNYIVKVNIQGNGTLSA SQKIQVTVDDPVTKPVVQIHPPSGAVEYVGNMTLTCHVEGGTRLAYQWLKNGR PVHTSSTYSFSPQNNTLHIAPVTKEDIGNYSCLVRNPVSEMESDIIMPIIYYGPYGL QVNSDKGLKVGEVFTVDLGEAILFDCSADSHPPNTYSWIRRTDNTTYIIKHGPRL EVASEKVAQKTMDYVCCAYNNITGRQDETHFTVIITSVGLEKLAQKGKSLSPLAS ITGISLFLIISMCLLFLWKKYQPYKVIKQKLEGR corresponding to amino acids 27-388 of known proteins NP001034461 (SEQ ID NO: 35), which also corresponds to amino acids 15-376 of H686541_P13 (SEQ ID NO:39), wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","B. An isolated polypeptide encoding for a head of H686541_P13 (SEQ ID NO:39), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95, 96, 97, 98 or 99% homologous to the sequence MWLKVFTTFLSFAT (SEQ ID NO: 289) of H686541_P13 (SEQ ID NO:39).","The localization of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: membrane.","Variant protein H686541_P13 (SEQ ID NO:39) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 81, (given according to their positions on the amino acid sequence, with the alternative amino acids listed (SEQ ID NO:39)).",{"@attributes":{"id":"p-1139","num":"1138"},"tables":{"@attributes":{"id":"TABLE-US-00105","num":"00105"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 81"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["21","K \u2212> R"]},{"entry":["74","K \u2212> N"]},{"entry":["192","L \u2212> P"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The variant protein has the following domains, as determined by using InterPro. The domains are described in Table 82:",{"@attributes":{"id":"p-1141","num":"1140"},"tables":{"@attributes":{"id":"TABLE-US-00106","num":"00106"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 82"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"InterPro domains"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Analysis",{}]},{"entry":["Domain description","type","Positions on protein"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Immunoglobulin subtype","HMMSmart","26-129, 143-223, 243-321"]},{"entry":["Immunoglobulin-like","ProfileScan","137-221, 223-319"]},{"entry":["Immunoglobulin C2 type","HMMSmart","152-214"]},{"entry":["Immunoglobulin V-set","HMMPfam","\u200219-131"]},{"entry":["Immunoglobulin-like","HMMPfam","151-209"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"The coding portion of transcript H686541_T17 (SEQ ID NO:31) starts at position 102 and ends at position 1229. The transcript also has the following SNPs as listed in Table 83 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1143","num":"1142"},"tables":{"@attributes":{"id":"TABLE-US-00107","num":"00107"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 83"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","163, 1626"]},{"entry":[{},"G \u2212> A","323, 1835"]},{"entry":[{},"G \u2212> C","\u2002323"]},{"entry":[{},"T \u2212> C","551, 676\u2002"]},{"entry":[{},"A \u2212> T","1626"]},{"entry":[{},"G \u2212> T","1765"]},{"entry":[{},"T \u2212> A","1836"]},{"entry":[{},"T \u2212> G","1836"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The coding portion of transcript H686541_T19 (SEQ ID NO:33) starts at position 102 and ends at position 1229. The transcript also has the following SNPs as listed in Table 84 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1145","num":"1144"},"tables":{"@attributes":{"id":"TABLE-US-00108","num":"00108"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 84"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","\u2002163"]},{"entry":[{},"G \u2212> A","323, 2211"]},{"entry":[{},"G \u2212> C","\u2002323"]},{"entry":[{},"T \u2212> C","551, 676\u2002"]},{"entry":[{},"T \u2212> A","1697"]},{"entry":[{},"T \u2212> G","1697"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"Variant protein H686541_P14 (SEQ ID NO:40) according to the present invention has an amino acid sequence as encoded by transcript H686541_T20 (SEQ ID NO:34). Alignments to previously published protein sequences are shown in . A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","1. Comparison report between H686541_P14 (SEQ ID NO:40) and known proteins NP937794 and Q6UXI0_HUMAN (SEQ ID NO: 36) ():","A. An isolated chimeric polypeptide encoding for H686541_P14 (SEQ ID NO:40), comprising a first amino acid sequence being at least 90% homologous to MWLKVFTTFLSFATGACSGLKVTVPSHTVHGVRGQALYLPVHYGFHTPASDIQII WLFERPHTMPKYLLGSVNKSVVPDLEYQHKFTMMPPNASLLINPLQFPDEGNYI VKVNIQGNGTLSASQKIQVTVDDPVTKPVVQIHPPSGAVEYVGNMTLTCHVEG GTRLAYQWLKNGRPVHTSSTYSFSPQNNTLHIAPVTKEDIGNYSCLVRNPVSEM ESDIIMPIIYYGPYGLQVNSDKGLKVGEVFTVDLGEAILFDCSADSHPPNTYSWIR RTDNTTYIIKHGPRLEVASEKVAQKTMDYVCCAYNNITGRQDETHFTVIITSVGL EKLAQKGKSLSPLASITGISLFLIISMCLLFLWKKYQPYKVIKQKLEGRPETEYRKA QTFSG corresponding to amino acids 1-389 of known proteins NP937794 and Q6UXI0_HUMAN (SEQ ID NO: 36), which also corresponds to amino acids 1-389 of H686541_P14 (SEQ ID NO:40), and a second amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95, 96, 97, 98 or 99% homologous to a polypeptide having the sequence FMLAAPSQREEEKKIWQGPGLLLCPHCNPHYHQY (SEQ ID NO: 292) corresponding to amino acids 390-423 of H686541_P14 (SEQ ID NO:40), wherein said first amino acid sequence and second amino acid sequence are contiguous and in a sequential order.","B. An isolated polypeptide encoding for an edge portion of H686541_P14 (SEQ ID NO:40), comprising an amino acid sequence being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95, 96, 97, 98 or 99% homologous to the sequence FMLAAPSQREEEKKIWQGPGLLLCPHCNPHYHQY (SEQ ID NO: 292) of H686541_P14 (SEQ ID NO:40).","2. Comparison report between H686541_P14 (SEQ ID NO:40) and known proteins NP001034461 (SEQ ID NO: 35) ():","A. An isolated chimeric polypeptide encoding for H686541_P14 (SEQ ID NO:40), comprising a first amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95, 96, 97, 98 or 99%, homologous to a polypeptide having the sequence MWLKVFTTFLSFAT (SEQ ID NO: 289) corresponding to amino acids 1-14 of H686541_P14 (SEQ ID NO:40), a second amino acid sequence being at least 90% homologous to GACSGLKVTVPSHTVHGVRGQALYLPVHYGFHTPASDIQIIWLFERPHTMPKYL LGSVNKSVVPDLEYQHKFTMMPPNASLLINPLQFPDEGNYIVKVNIQGNGTLSA SQKIQVTVDDPVTKPVVQIHPPSGAVEYVGNMTLTCHVEGGTRLAYQWLKNGR PVHTSSTYSFSPQNNTLHIAPVTKEDIGNYSCLVRNPVSEMESDIIMPIIYYGPYGL QVNSDKGLKVGEVFTVDLGEAILFDCSADSHPPNTYSWIRRTDNTTYIIKHGPRL EVASEKVAQKTMDYVCCAYNNITGRQDETHFTVIITSVGLEKLAQKGKSLSPLAS ITGISLFLIISMCLLFLWKKYQPYKVIKQKLEGRPETEYRKAQTFSG corresponding to amino acids 27-401 of known proteins NP001034461 (SEQ ID NO: 35), which also corresponds to amino acids 15-389 of H686541_P14 (SEQ ID NO:40), and a third amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95, 96, 97, 98 or 99% homologous to a polypeptide having the sequence FMLAAPSQREEEKKIWQGPGLLLCPHCNPHYHQY (SEQ ID NO: 292) corresponding to amino acids 390-423 of H686541_P14 (SEQ ID NO:40), wherein said first amino acid sequence, second amino acid sequence and third amino acid sequence are contiguous and in a sequential order.","B. An isolated polypeptide encoding for a head of H686541_P14 (SEQ ID NO:40), comprising a polypeptide being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95, 96, 97, 98 or 99% homologous to the sequence MWLKVFTTFLSFAT (SEQ ID NO: 289) of H686541_P14 (SEQ ID NO:40).","C. An isolated polypeptide encoding for an edge portion of H686541_P14 (SEQ ID NO:40), comprising an amino acid sequence being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95, 96, 97, 98 or 99% homologous to the sequence FMLAAPSQREEEKKIWQGPGLLLCPHCNPHYHQY (SEQ ID NO: 292) of H686541_P14 (SEQ ID NO:40).","The localization of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: membrane.","Variant protein H686541_P14 (SEQ ID NO:40) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 85, (given according to their positions on the amino acid sequence, with the alternative amino acids listed (SEQ ID NO:40)).",{"@attributes":{"id":"p-1156","num":"1155"},"tables":{"@attributes":{"id":"TABLE-US-00109","num":"00109"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 85"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["21","K \u2212> R"]},{"entry":["74","K \u2212> N"]},{"entry":["192","L \u2212> P"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The variant protein has the following domains, as determined by using InterPro. The domains are described in Table 86:",{"@attributes":{"id":"p-1158","num":"1157"},"tables":{"@attributes":{"id":"TABLE-US-00110","num":"00110"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 86"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"InterPro domains"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Analysis",{}]},{"entry":["Domain description","type","Positions on protein"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Immunoglobulin C2 type","HMMSmart","152-214"]},{"entry":["Immunoglobulin subtype","HMMSmart","26-129, 143-223, 243-321"]},{"entry":["Immunoglobulin-like","ProfileScan","137-221, 223-319"]},{"entry":["Immunoglobulin V-set","HMMPfam","\u200219-131"]},{"entry":["Immunoglobulin-like","HMMPfam","151-209"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein H686541_P14 (SEQ ID NO:40) is encoded by the transcript H686541_T20 (SEQ ID NO:34), for which the coding portion starts at position 102 and ends at position 1370. The transcript also has the following SNPs as listed in Table 87 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1160","num":"1159"},"tables":{"@attributes":{"id":"TABLE-US-00111","num":"00111"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 87"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A \u2212> G","163"]},{"entry":[{},"G \u2212> A","323"]},{"entry":[{},"G \u2212> C","323"]},{"entry":[{},"T \u2212> C","551, 676"]},{"entry":[{},"C \u2212> A","1491\u2002"]},{"entry":[{},"C \u2212> G","1491\u2002"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"As noted above, cluster H68654 features 3 segments, which were listed in Table 63. These segments are portions of nucleic acid sequences which are described herein separately because they are of particular interest. A description of each segment according to the present invention is now provided.","Segment cluster H686541_N3 (SEQ ID NO:123) according to the present invention is supported by 24 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcripts: H686541_TO (SEQ ID NO:25), H686541_T15 (SEQ ID NO:29), H686541_T16 (SEQ ID NO:30), H686541_T17 (SEQ ID NO:31), H686541_T18 (SEQ ID NO:32), H686541_T19 (SEQ ID NO:33), H686541_T20 (SEQ ID NO:34), H686541_T4 (SEQ ID NO:26), H686541_T5 (SEQ ID NO:27) and H686541_T8 (SEQ ID NO:28). Table 88 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1163","num":"1162"},"tables":{"@attributes":{"id":"TABLE-US-00112","num":"00112"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 88"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["H68654_1_T0 (SEQ ID NO: 25)","158","508"]},{"entry":["H68654_1_T15 (SEQ ID NO: 29)","145","495"]},{"entry":["H68654_1_T16 (SEQ ID NO: 30)","145","495"]},{"entry":["H68654_1_T17 (SEQ ID NO: 31)","145","495"]},{"entry":["H68654_1_T18 (SEQ ID NO: 32)","145","495"]},{"entry":["H68654_1_T19 (SEQ ID NO: 33)","145","495"]},{"entry":["H68654_1_T20 (SEQ ID NO: 34)","145","495"]},{"entry":["H68654_1_T4 (SEQ ID NO: 26)","145","495"]},{"entry":["H68654_1_T5 (SEQ ID NO: 27)","158","508"]},{"entry":["H68654_1_T8 (SEQ ID NO: 28)","145","495"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster H686541_N7 (SEQ ID NO:124) according to the present invention is supported by 20 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcripts: H686541_T0 (SEQ ID NO:25), H686541_T15 (SEQ ID NO:29), H686541_T16 (SEQ ID NO:30), H686541_T17 (SEQ ID NO:31), H686541_T18 (SEQ ID NO:32), H686541_T19 (SEQ ID NO:33), H686541_T20 (SEQ ID NO:34), H686541_T4 (SEQ ID NO:26), H686541_T5 (SEQ ID NO:27) and H686541_T8 (SEQ ID NO:28). Table 89 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1165","num":"1164"},"tables":{"@attributes":{"id":"TABLE-US-00113","num":"00113"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 89"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["H68654_1_T0 (SEQ ID NO: 25)","794","1090"]},{"entry":["H68654_1_T15 (SEQ ID NO: 29)","781","1077"]},{"entry":["H68654_1_T16 (SEQ ID NO: 30)","781","1077"]},{"entry":["H68654_1_T17 (SEQ ID NO: 31)","781","1077"]},{"entry":["H68654_1_T18 (SEQ ID NO: 32)","781","1077"]},{"entry":["H68654_1_T19 (SEQ ID NO: 33)","781","1077"]},{"entry":["H68654_1_T20 (SEQ ID NO: 34)","781","1077"]},{"entry":["H68654_1_T4 (SEQ ID NO: 26)","781","1077"]},{"entry":["H68654_1_T5 (SEQ ID NO: 27)","794","1090"]},{"entry":["H68654_1_T8 (SEQ ID NO: 28)","781","1077"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster H686541_N12 (SEQ ID NO:125) according to the present invention is supported by 18 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcripts: H686541_T0 (SEQ ID NO:25), H686541_T15 (SEQ ID NO:29), H686541_T16 (SEQ ID NO:30), H686541_T17 (SEQ ID NO:31), H686541_T18 (SEQ ID NO:32), H686541_T19 (SEQ ID NO:33), H686541_T20 (SEQ ID NO:34), H686541_T4 (SEQ ID NO:26), H686541_T5 (SEQ ID NO:27) and H686541_T8 (SEQ ID NO:28). Table 90 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1167","num":"1166"},"tables":{"@attributes":{"id":"TABLE-US-00114","num":"00114"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 90"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["H68654_1_T0 (SEQ ID NO: 25)","1091","1216"]},{"entry":["H68654_1_T15 (SEQ ID NO: 29)","1078","1203"]},{"entry":["H68654_1_T16 (SEQ ID NO: 30)","1078","1203"]},{"entry":["H68654_1_T17 (SEQ ID NO: 31)","1078","1203"]},{"entry":["H68654_1_T18 (SEQ ID NO: 32)","1078","1203"]},{"entry":["H68654_1_T19 (SEQ ID NO: 33)","1078","1203"]},{"entry":["H68654_1_T20 (SEQ ID NO: 34)","1078","1203"]},{"entry":["H68654_1_T4 (SEQ ID NO: 26)","1078","1203"]},{"entry":["H68654_1_T5 (SEQ ID NO: 27)","1091","1216"]},{"entry":["H68654_1_T8 (SEQ ID NO: 28)","1078","1203"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Expression of hypothetical protein LOC253012 H68654 transcripts which are detectable by amplicon as depicted in sequence name H68654_seg3WTF2R2 (SEQ ID NO: 226) in normal and cancerous Lung tissues and in different normal tissues","Expression of hypothetical protein LOC253012 transcripts detectable by or according to seg3F2R2_ H68654_seg3WTF2R2 (SEQ ID NO: 226) amplicon and primers H68654_seg3WTF2 (SEQ ID NO: 224) and H68654_seg3WTR2 (SEQ ID NO: 225) was measured by real time PCR on lung panel and normal panel. The samples used are detailed in Table 3 and Table 2 above, respectively. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1.","Lung panel\u2014The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 51-64 and 69-70, Table 3 above), to obtain a value of fold up-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-1171","num":"1170"},"figref":"FIG. 31"},"As is evident from , the expression of hypothetical protein LOC253012 transcripts detectable by the above amplicon in small cell carcinoma samples was significantly higher than in the non-cancerous samples (sample numbers 51-64 and 69-70, Table 3 above). Notably an over-expression of at least 80 fold was found in 7 out of 9 small cell carcinoma samples.","Statistical analysis was applied to verify the significance of these results, as described below.","The P value for the difference in the expression levels of hypothetical protein LOC253012 transcripts detectable by the above amplicon in Lung small cell carcinoma samples versus the normal tissue samples was determined by T test as 3.24e-003.","Threshold of 80 fold over expression was found to differentiate between small cell carcinoma and normal samples with P value of 7.49e-005 as checked by exact Fisher test.","The above values demonstrate statistical significance of the results.","Normal panel\u2014The normalized quantity of each RT sample was then divided by the median of the quantities of the lung samples (sample numbers 26, 28, 29 and 30, Table 2 above), to obtain a value of relative expression of each sample relative to median of the lung samples, as shown in .","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: H68654_seg3WTF2 (SEQ ID NO: 224) forward primer; and H68654_seg3WTR2 (SEQ ID NO: 225) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: H68654_seg3WTF2R2 (SEQ ID NO: 226).",{"@attributes":{"id":"p-1180","num":"1179"},"tables":{"@attributes":{"id":"TABLE-US-00115","num":"00115"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>H68654_seg3WTF2\u2003(SEQ\u2003ID\u2003NO:\u2003224):"},{"entry":"ATCACACACTGTCCATGGCGT"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>H68654_seg3WTR2\u2003(SEQ\u2003ID\u2003NO:\u2003225):"},{"entry":"GTCTCTCAAATAGCCATATGATCTGG"},{"entry":{}},{"entry":"Amplicon\u2003>H68654_seg3WTF2R2\u2003(SEQ\u2003ID\u2003NO:\u2003226)"},{"entry":"ATCACACACTGTCCATGGCGTCAGAGGTCAGGCCCTCTACCTACCCGTC"},{"entry":{}},{"entry":"CACTATGGCTTCCACACTCCAGCATCAGACATCCAGATCATATGGCTAT"},{"entry":{}},{"entry":"TTGAGAGAC"}]}}}}},"Expression of hypothetical protein LOC253012 H68654 transcripts which are detectable by amplicon as depicted in sequence name H68654_seg7-12WT (SEQ ID NO: 223) in normal and cancerous Lung tissues and in different normal tissues","Expression of hypothetical protein LOC253012 transcripts detectable by or according to seg7-12\u2014H68654_seg7-12WT (SEQ ID NO: 223) amplicon and primers H68654_seg7-12WTF1 (SEQ ID NO: 221) and H68654_seg7-12WTR1 (SEQ ID NO: 222) was measured by real time PCR on lung panel and normal panel. The samples used are detailed in Table 3 and Table 2 above, respectively. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1.","Lung panel\u2014Non-detected samples (samples no. 30, 41, 78 and 92, Table 3) were assigned Ct value of 41 and were calculated accordingly. The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 51-64 and 69-70, Table 3 above), to obtain a value of fold up-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-1184","num":"1183"},"figref":"FIG. 33"},"As is evident from , the expression of hypothetical protein LOC253012 transcripts detectable by the above amplicon in small cell carcinoma samples was significantly higher than in the non-cancerous samples (sample numbers 51-64 and 69-70, Table 3 above) and was higher in a few squamous cell carcinoma samples. Notably an over-expression of at least 25 fold was found in 6 out of 9 small cell carcinoma samples and in 4 out of 24 squamous cell carcinoma samples.","Statistical analysis was applied to verify the significance of these results, as described below. The P value for the difference in the expression levels of hypothetical protein LOC253012 transcripts detectable by the above amplicon in Lung small cell carcinoma samples and Lung squamous cell carcinoma samples versus the normal tissue samples was determined by T test as 1.24e-002 and 2.97e-002, respectively.","Threshold of 25 fold over expression was found to differentiate between small cell carcinoma and normal samples with P value of 4.74e-004 as checked by exact Fisher test.","The above values demonstrate statistical significance of the results.","Normal panel\u2014Non-detected sample (sample no. 16) was assigned Ct value of 41 and was calculated accordingly. The normalized quantity of each RT sample was then divided by the median of the quantities of the lung samples (sample numbers 26, 28, 29 and 30, Table 2 above), to obtain a value of relative expression of each sample relative to median of the lung samples, as shown in .","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: H68654_seg7-12WTF1 (SEQ ID NO: 221) forward primer; and H68654_seg7-12WTR1 (SEQ ID NO: 222) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: H68654_seg7-12WT (SEQ ID NO: 223).",{"@attributes":{"id":"p-1192","num":"1191"},"tables":{"@attributes":{"id":"TABLE-US-00116","num":"00116"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>H68654_seg7-12WTF1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003221):"},{"entry":"ATGGGCCTCGCTTAGAAGTTG"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>H68654_seg7-12WTR1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003222):"},{"entry":"TTCTGTGCAAGCTTCTCCAGTC"},{"entry":{}},{"entry":"Amplicon\u2003>H68654_seg7-12WT\u2003(SEQ\u2003ID\u2003NO:\u2003223):"},{"entry":"ATGGGCCTCGCTTAGAAGTTGCATCTGAGAAAGTAGCCCAGAAGACAAT"},{"entry":{}},{"entry":"GGACTATGTGTGCTGTGCTTACAACAACATAACCGGCAGGCAAGATGAA"},{"entry":{}},{"entry":"ACTCATTTCACAGTTATCATCACTTCCGTAGGACTGGAGAAGCTTGCAC"},{"entry":{}},{"entry":"AGAA"}]}}}}},"Expression of LOC253012 H68654 transcripts which are detectable by amplicon as depicted in sequence name H68654_seg3WTF2R2 (SEQ ID NO: 226) in the blood-specific panel.","Expression of LOC253012 transcripts detectable by or according to seg3-H68654seg3F2R2 (SEQ ID NO: 226) amplicon and primers H68654seg3F2 (SEQ ID NO: 224) and H68654seg3R2 (SEQ ID NO: 225) was measured by real time PCR on blood panel. The samples used are detailed in Table 1 above. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1. The normalized quantity of each RT sample was then divided by the median of the quantities of the kidney normal samples (sample numbers 65-67, Table 1 above), to obtain a value of relative expression of each sample relative to median of the kidney normal samples.","The results of this analysis are depicted in the histogram in . Expression of the above-indicated LOC253012 transcript is high in the kidney normal, colon normal and small intestine normal as in few of the different blood samples checked.","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: H68654_seg3WTF2 forward primer; and H68654_seg3WTR2 reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: H68654_seg3WTF2R2.",{"@attributes":{"id":"p-1198","num":"1197"},"tables":{"@attributes":{"id":"TABLE-US-00117","num":"00117"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>H68654_seg3WTF2:"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003224)"},{"entry":"ATCACACACTGTCCATGGCGT"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>H68654_seg3WTR2:"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003225)"},{"entry":"GTCTCTCAAATAGCCATATGATCTGG\u2003"},{"entry":{}},{"entry":"Amplicon\u2003>H68654_seg3WTF2R2"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003226)"},{"entry":"ATCACACACTGTCCATGGCGTCAGAGGTCAGGCCCTCTACCTACCCGT"},{"entry":{}},{"entry":"CCACTATGGCTTCCACACTCCAGCATCAGACATCCAGATCATATGGCT"},{"entry":{}},{"entry":"ATTTGAGAGAC"}]}}}}},"Expression of LOC253012 H68654 transcripts which are detectable by amplicon as depicted in sequence name H68654seg7-12 (SEQ ID NO: 223) in the blood-specific panel.","Expression of LOC253012 transcripts detectable by or according to seg7-12-H68654seg7-12WTF1R1 (SEQ ID NO: 223) amplicon and primers H68654seg7-12WTF1 (SEQ ID NO: 221) and H68654seg7-12WTR1 (SEQ ID NO: 222) was measured by real time PCR on blood panel. The samples used are detailed in Table 1 above. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1. The normalized quantity of each RT sample was then divided by the median of the quantities of the kidney normal samples (sample numbers 65-67, Table 1 above), to obtain a value of relative expression of each sample relative to median of the kidney normal samples.","The results of this analysis are depicted in the histogram in . Expression of the above-indicated LOC253012 transcript is higher in the kidney normal, colon normal and small intestine normal relative to the different blood samples checked.","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: seg7-12WTF1 forward primer (SEQ ID NO: 221); and seg7-12WTR1 reverse primer (SEQ ID NO: 222).","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: seg7-12WTF1R1 (SEQ ID NO: 223).",{"@attributes":{"id":"p-1204","num":"1203"},"tables":{"@attributes":{"id":"TABLE-US-00118","num":"00118"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>H68654_seg7-12WTF1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003221)"},{"entry":"ATGGGCCTCGCTTAGAAGTTG"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>H68654_seg7-12WTR1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003222)"},{"entry":"TTCTGTGCAAGCTTCTCCAGTC"},{"entry":{}},{"entry":"Amplicon\u2003>H68654_seg7-12WT"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003223)"},{"entry":"ATGGGCCTCGCTTAGAAGTTGCATCTGAGAAAGTAGCCCAGAAGACAAT"},{"entry":{}},{"entry":"GGACTATGTGTGCTGTGCTTACAACAACATAACCGGCAGGCAAGATGAA"},{"entry":{}},{"entry":"ACTCATTTCACAGTTATCATCACTTCCGTAGGACTGGAGAAGCTTGCAC"},{"entry":{}},{"entry":"AGAA"}]}}}}},"Expression of hypothetical protein LOC253012 H68654 transcripts which are detectable by amplicon as depicted in sequence name H68654_seg0-3 (SEQ ID NO: 229) in normal and cancerous Lung tissues Expression of LOC253012 transcripts detectable by or according to seg0-3-H68654_seg0-3 (SEQ ID NO: 229) amplicon and primers H68654_seg0-3F1 (SEQ ID NO: 227) and H68654_seg0-3R1 (SEQ ID NO: 228) was measured by real time PCR on lung panel. The samples used are detailed in Table 3 above. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1. The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 51-54, 56-64, 69 and 70, Table 3 above), to obtain a value of fold up-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-1206","num":"1205"},"figref":"FIG. 36"},"As is evident from , the expression of LOC253012 transcripts detectable by the above amplicon in squamous cell carcinoma and small cell carcinoma samples was significantly higher than in the non-cancerous samples (sample numbers 51-54, 56-64, 69 and 70, Table 3 above). Notably an over-expression of at least 5 fold was found in 6 out of 21 squamous cell carcinoma samples and in 8 out of 9 small cell carcinoma samples.","Statistical analysis was applied to verify the significance of these results, as described below. The P value for the difference in the expression levels of LOC253012 transcripts detectable by the above amplicon in lung squamous cell carcinoma samples and lung small cell carcinoma samples versus the normal tissue samples was determined by T test as 4.96e-002 and 1.05e-003, respectively.","Threshold of 5 fold over expression was found to differentiate between squamous cell carcinoma and small cell carcinoma and normal samples with P value of 2.79e-002 and 1.22e-005, respectively, as checked by exact Fisher test. The above values demonstrate statistical significance of the results.","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: H68654_seg0-3F1 (SEQ ID NO: 227) forward primer; and H68654_seg0-3R1 (SEQ ID NO: 228) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: H68654_seg0-3F1R1 (SEQ ID NO: 229).",{"@attributes":{"id":"p-1212","num":"1211"},"tables":{"@attributes":{"id":"TABLE-US-00119","num":"00119"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>H68654_seg0-3F1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003227)"},{"entry":"GCTTTCATGGAGCCCTTCG"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>H68654_seg0-3R1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003228)"},{"entry":"GCCTGACCTCTGACGCCA"},{"entry":{}},{"entry":"Amplicon\u2003>H68654_seg0-3F1R1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003229)"},{"entry":"GCTTTCATGGAGCCCTTCGGTGACACACTTGGGGTCTTTCAGTGCAAAA"},{"entry":{}},{"entry":"TATACCTCCTTCTCTTCGGTGCTTGCTCGGGGCTGAAGGTGACAGTGCC"},{"entry":{}},{"entry":"ATCACACACTGTCCATGGCGTCAGAGGTCAGGC"}]}}}}},"Expression of hypothetical protein LOC253012 H68654 transcripts which are detectable by amplicon as depicted in sequence name H68654_seg2-3 (SEQ ID NO: 232) in normal and cancerous Lung tissues","Expression of LOC253012 transcripts detectable by or according to seg2-3-H68654_seg2-3 (SEQ ID NO: 232) amplicon and primers H68654_seg2-3F1 (SEQ ID NO: 230) and H68654_seg2-3R1 (SEQ ID NO: 231) was measured by real time PCR on lung panel. The samples used are detailed in Table 3 above. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1. The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 51, 52, 54-64, 69 and 70, Table 3 above), to obtain a value of fold up-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-1215","num":"1214"},"figref":"FIG. 37"},"As is evident from , the expression of LOC253012 transcripts detectable by the above amplicon in small cell carcinoma samples was significantly higher than in the non-cancerous samples (sample numbers 51, 52, 54-64, 69 and 70, Table 3 above) Notably an over-expression of at least 8 fold was found in 7 out of 9 small cell carcinoma samples.","Statistical analysis was applied to verify the significance of these results, as described below. The P value for the difference in the expression levels of LOC253012 transcripts detectable by the above amplicon in Lung small cell carcinoma samples versus the normal tissue samples was determined by T test as 1.93e-002.","Threshold of 8 fold over expression was found to differentiate between small cell carcinoma and normal samples with P value of 1.04e-004 as checked by exact Fisher test.","The above values demonstrate statistical significance of the results.","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: H68654_seg2-3F1 (SEQ ID NO: 230) forward primer; and H68654_seg2-3R1 (SEQ ID NO: 231) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: H68654_seg2-3F1R1 (SEQ ID NO: 232).",{"@attributes":{"id":"p-1222","num":"1221"},"tables":{"@attributes":{"id":"TABLE-US-00120","num":"00120"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>H68654_seg2-3F1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003230)"},{"entry":"CTCTGCATTTGCCCCTTTAGA"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>H68654_seg2-3R1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003231)"},{"entry":"GATGGCACTGTCACCTTCAGC"},{"entry":{}},{"entry":"Amplicon\u2003>H68654_seg2-3F1R1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003232)"},{"entry":"CTCTGCATTTGCCCCTTTAGATTGTGAAATGTGGCTCAAGGTCTTCACA"},{"entry":{}},{"entry":"ACTTTCCTTTCCTTTGCAACAGGTGCTTGCTCGGGGCTGAAGGTGACAG"},{"entry":{}},{"entry":"TGCCATC"}]}}}}},"The present invention relates to C1ORF32 polypeptides, novel splice variants and diagnostics and therapeutics based thereon.","Cluster H190111 (internal ID 76432827) features 2 transcripts and 5 segments of interest, the names for which are given in Tables 91 and 92, respectively. The selected protein variants are given in table 93.",{"@attributes":{"id":"p-1225","num":"1224"},"tables":{"@attributes":{"id":"TABLE-US-00121","num":"00121"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 91"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Transcripts of interest"},{"entry":"Transcript Name"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"161pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"H19011_1_T8 (SEQ ID NO: 45)"]},{"entry":[{},"H19011_1_T9 (SEQ ID NO: 46)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-1226","num":"1225"},"tables":{"@attributes":{"id":"TABLE-US-00122","num":"00122"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 92"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segments of interest"},{"entry":"Segment Name"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"168pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"H19011_1_N13 (SEQ ID NO: 129)"]},{"entry":[{},"H19011_1_N8 (SEQ ID NO: 130)"]},{"entry":[{},"H19011_1_N10 (SEQ ID NO: 131)"]},{"entry":[{},"H19011_1_N11 (SEQ ID NO: 132)"]},{"entry":[{},"H19011_1_N12 (SEQ ID NO: 133)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-1227","num":"1226"},"tables":{"@attributes":{"id":"TABLE-US-00123","num":"00123"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 93"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Proteins of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Protein Name","Corresponding Transcripts"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["H19011_1_P8 (SEQ ID NO: 48)","H19011_1_T8 (SEQ ID NO: 45)"]},{"entry":["H19011_1_P9 (SEQ ID NO: 50)","H19011_1_T9 (SEQ ID NO: 46)"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"These sequences are variants of the known protein hypothetical protein LOC387597 (RefSeq accession identifier NP955383 (SEQ ID NO: 47), synonyms: C1ORF32, NP955383; LISCH-like; RP4-782G3.2; dJ782G3.1), referred to herein as the previously known protein.","C1ORF32 is a hypothetical protein that was computationally discovered during the annotation of chromosome 1 (Gregory S G et al. 2006, Nature 441 (7091) 315-321). Its closest annotated homolog belongs to the LISCH7 family, a subfamily of the immunoglobulin super family. One of the annotated members of this family is the lipolysis-stimulated lipoprotein receptor which has a probable role in the clearance of triglyceride-rich lipoprotein from blood (Swissprot annotation of accession Q86X29).","According to the present invention, C1ORF32 was predicted to be a novel B7\/CD28 member based on the presence of an IgV domain, in addition of its being a type I membrane protein, like other known B7 members. Also, two alternatively spliced variants of the present invention (H190111_P8 (SEQ ID NO:48) and H190111_P9 (SEQ ID NO:50)), which share only the first 5 exons with the wild type C1ORF32, are similar to the known B7 family members in their exons' sizes and the position of the IgV and transmembrane domains within these exons. In addition, C1ORF32 was shown in the present invention to be overexpressed in small cell lung cancer.","As noted above, cluster H19011 features 2 transcripts, which were listed in Table 91 above. These transcripts encode for proteins which are variants of protein hypothetical protein LOC387597 (SEQ ID NO:47). A description of each variant protein according to the present invention is now provided.","Variant protein H190111_P8 (SEQ ID NO:48) according to the present invention has an amino acid sequence as encoded by transcript H190111_T8 (SEQ ID NO:45). Alignments to one or more previously published protein sequences are shown in . A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between H190111_P8 (SEQ ID NO:48) and known proteins Q71H61_HUMAN and NP955383 (SEQ ID NO: 47) ():","A. An isolated chimeric polypeptide encoding for H190111_P8 (SEQ ID NO:48), comprising a first amino acid sequence being at least 90% homologous to MDRVLLRWISLFWLTAMVEGLQVTVPDKKKVAMLFQPTVLRCHFSTSSHQPAV VQWKFKSYCQDRMGESLGMSSTRAQSLSKRNLEWDPYLDCLDSRRTVRVVASK QGSTVTLGDFYRGREITIVHDADLQIGKLMWGDSGLYYCIITTPDDLEGKNE corresponding to amino acids 1-158 of known proteins Q71H61_HUMAN and NP955383 (SEQ ID NO: 47), which also corresponds to amino acids 1-158 of H190111_P8 (SEQ ID NO:48), a bridging amino acid G corresponding to amino acid 159 of H190111_P8 (SEQ ID NO:48), a second amino acid sequence being at least 90% homologous to S corresponding to amino acids 160-160 of known proteins Q71H61_HUMAN and NP955383 (SEQ ID NO: 47), which also corresponds to amino acids 160-160 of H190111_P8 (SEQ ID NO:48), bridging amino acids LG corresponding to amino acid 161-162 of H190111_P8 (SEQ ID NO:48), a third amino acid sequence being at least 90% homologous to LLVLGRTGLLADLLPSFAVEIMPEWVFVGLVLLGVFLFFVLVGICWCQCCPHSCC CYVRCPCCPDSC corresponding to amino acids 163-229 of known proteins Q71H61_HUMAN and NP955383 (SEQ ID NO: 47), which also corresponds to amino acids 163-229 of H190111_P8 (SEQ ID NO:48), a bridging amino acid W corresponding to amino acid 230 of H190111_P8 (SEQ ID NO:48), a fourth amino acid sequence being at least 90% homologous to CPQA corresponding to amino acids 231-234 of known proteins Q71H61_HUMAN and NP955383 (SEQ ID NO: 47), which also corresponds to amino acids 231-234 of H190111_P8 (SEQ ID NO:48), and a fifth amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95, 96, 97, 98 or 99% homologous to a polypeptide having the sequence CEYSDRWGDRAIERNVYLST (SEQ ID NO: 293) corresponding to amino acids 235-254 of H190111_P8 (SEQ ID NO:48), wherein said first amino acid sequence, bridging amino acid, second amino acid sequence, bridging amino acid, third amino acid sequence, bridging amino acid, fourth amino acid sequence and fifth amino acid sequence are contiguous and in a sequential order.","B. An isolated polypeptide encoding for an edge portion of H190111_P8 (SEQ ID NO:48), comprising an amino acid sequence being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95, 96, 97, 98 or 99% homologous to the sequence CEYSDRWGDRAIERNVYLST (SEQ ID NO: 293) of H190111_P8 (SEQ ID NO:48).","The localization of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: membrane.","Variant protein H190111_P8 (SEQ ID NO:48) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 94, (given according to their positions on the amino acid sequence, with the alternative amino acids listed (SEQ ID NO:48)). An example of such a deduced sequence, with alternative amino-acids, that was produced (using part of the SNPs below), is given under the name H190111_P8_V1 (SEQ ID NO:49).",{"@attributes":{"id":"p-1238","num":"1237"},"tables":{"@attributes":{"id":"TABLE-US-00124","num":"00124"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 94"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["159","G \u2212> D"]},{"entry":["161","L \u2212> V"]},{"entry":["162","G \u2212> E"]},{"entry":["202","V \u2212> D"]},{"entry":["202","V \u2212> G"]},{"entry":["230","W \u2212> C"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The variant protein has the following domains, as determined by using InterPro. The domains are described in Table 95:",{"@attributes":{"id":"p-1240","num":"1239"},"tables":{"@attributes":{"id":"TABLE-US-00125","num":"00125"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 95"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"InterPro domains"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Analysis",{}]},{"entry":[{},"Domain description","type","Positions on protein"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"LISCH7","HMMPfam","186-234"]},{"entry":[{},"IG","SMART","\u200227-166"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},"Variant protein H190111_P8 (SEQ ID NO:48) is encoded by the transcript H190111_T8 (SEQ ID NO:45), for which the coding portion starts at position 181 and ends at position 942. The transcript also has the following SNPs as listed in Table 96 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1242","num":"1241"},"tables":{"@attributes":{"id":"TABLE-US-00126","num":"00126"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 96"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"G \u2212> A","656"]},{"entry":[{},"C \u2212> G","661"]},{"entry":[{},"G \u2212> A","665"]},{"entry":[{},"T \u2212> A","785"]},{"entry":[{},"T \u2212> G","785"]},{"entry":[{},"G \u2212> C","870"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The genomic structure of protein H190111_P8 (SEQ ID NO:48) (number of exons relevant to the extra-cellular region of the protein, the length of these exons, the frame of the codon in which the introns are inserted and the location of the protein features and domains in the gene structure) is characteristic to the ligands of the B7\/co-stimulatory protein family, as given in table 97",{"@attributes":{"id":"p-1244","num":"1243"},"tables":{"@attributes":{"id":"TABLE-US-00127","num":"00127"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 97"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"genomic structure and protein features"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"84pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Exon","Exon","Amino-",{}]},{"entry":["number","Length","Acids","Protein feature on exon"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"84pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1","46","\u20021-15","Signal Peptide"]},{"entry":["2","333","\u200216-126","IgV domain"]},{"entry":["3","120","127-166","IgC2 domain"]},{"entry":["4","57","167-185"]},{"entry":["5","206","186-254","Trans-membrane region"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein H190111_P9 (SEQ ID NO:50) according to the present invention has an amino acid sequence as encoded by transcript H190111_T9 (SEQ ID NO:46). Alignments to one or more previously published protein sequences are shown in . A brief description of the relationship of the variant protein according to the present invention to each such aligned protein is as follows:","Comparison report between H190111_P9 (SEQ ID NO:50) and known proteins Q71H61_HUMAN and NP955383 (SEQ ID NO: 47) ():","A. An isolated chimeric polypeptide encoding for H190111_P9 (SEQ ID NO:50), comprising a first amino acid sequence being at least 90% homologous to MDRVLLRWISLFWLTAMVEGLQVTVPDKKKVAMLFQPTVLRCHFSTSSHQPAV VQWKFKSYCQDRMGESLGMSSTRAQSLSKRNLEWDPYLDCLDSRRTVRVVASK QGSTVTLGDFYRGREITIVHDADLQIGKLMWGDSGLYYCIITTPDDLEGKNE corresponding to amino acids 1-158 of known proteins Q71H61_HUMAN and NP955383 (SEQ ID NO: 47), which also corresponds to amino acids 1-158 of H190111_P9 (SEQ ID NO:50), a bridging amino acid G corresponding to amino acid 159 of H190111_P9 (SEQ ID NO:50), a second amino acid sequence being at least 90% homologous to S corresponding to amino acids 160-160 of known proteins Q71H61_HUMAN and NP955383 (SEQ ID NO: 47), which also corresponds to amino acids 160-160 of H190111_P9 (SEQ ID NO:50), bridging amino acids LG corresponding to amino acid 161-162 of H190111_P9 (SEQ ID NO:50), a third amino acid sequence being at least 90% homologous to LLVL corresponding to amino acids 163-166 of known proteins Q71H61_HUMAN and NP955383 (SEQ ID NO: 47), which also corresponds to amino acids 163-166 of H190111_P9 (SEQ ID NO:50), a fourth amino acid sequence being at least 90% homologous to EWVFVGLVLLGVFLFFVLVGICWCQCCPHSCCCYVRCPCCPDSC corresponding to amino acids 186-229 of known proteins Q71H61_HUMAN and NP955383 (SEQ ID NO: 47), which also corresponds to amino acids 167-210 of H190111_P9 (SEQ ID NO:50), a bridging amino acid W corresponding to amino acid 211 of H190111_P9 (SEQ ID NO:50), a fifth amino acid sequence being at least 90% homologous to CPQA corresponding to amino acids 231-234 of known proteins Q71H61_HUMAN and NP955383 (SEQ ID NO: 47), which also corresponds to amino acids 212-215 of H190111_P9 (SEQ ID NO:50), and a sixth amino acid sequence being at least 70%, optionally at least 80%, preferably at least 85%, more preferably at least 90% and most preferably at least 95, 96, 97, 98 or 99% homologous to a polypeptide having the sequence CEYSDRWGDRAIERNVYLST (SEQ ID NO: 293) corresponding to amino acids 216-235 of H190111_P9 (SEQ ID NO:50), wherein said first amino acid sequence, bridging amino acid, second amino acid sequence, bridging amino acid, third amino acid sequence, fourth amino acid sequence, bridging amino acid, fifth amino acid sequence and sixth amino acid sequence are contiguous and in a sequential order.","B. An isolated chimeric polypeptide encoding for an edge portion of H190111_P9 (SEQ ID NO:50), comprising a polypeptide having a length \u201cn\u201d, wherein n is at least about 10 amino acids in length, optionally at least about 20 amino acids in length, preferably at least about 30 amino acids in length, more preferably at least about 40 amino acids in length and most preferably at least about 50 amino acids in length, wherein at least two amino acids comprise LE, having a structure as follows: a sequence starting from any of amino acid numbers 166\u2212x to 166; and ending at any of amino acid numbers 167+((n\u22122)\u2212x), in which x varies from 0 to n\u22122.","C. An isolated polypeptide encoding for an edge portion of H190111_P9 (SEQ ID NO:50), comprising an amino acid sequence being at least 70%, optionally at least about 80%, preferably at least about 85%, more preferably at least about 90% and most preferably at least about 95, 96, 97, 98 or 99% homologous to the sequence CEYSDRWGDRAIERNVYLST (SEQ ID NO: 293) of H190111_P9 (SEQ ID NO:50).","The localization of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: membrane.","Variant protein H190111_P9 (SEQ ID NO:50) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 98, (given according to their positions on the amino acid sequence, with the alternative amino acids listed (SEQ ID NO:50)).",{"@attributes":{"id":"p-1252","num":"1251"},"tables":{"@attributes":{"id":"TABLE-US-00128","num":"00128"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 98"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["159","G \u2212> D"]},{"entry":["161","L \u2212> V"]},{"entry":["162","G \u2212> E"]},{"entry":["183","V \u2212> D"]},{"entry":["183","V \u2212> G"]},{"entry":["211","W \u2212> C"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"Variant protein H190111_P9 (SEQ ID NO:50) is encoded by the transcript H190111_T9 (SEQ ID NO:46), for which the coding portion starts at position 181 and ends at position 885. The transcript also has the following SNPs as listed in Table 99 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1254","num":"1253"},"tables":{"@attributes":{"id":"TABLE-US-00129","num":"00129"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 99"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"G \u2212> A","656"]},{"entry":[{},"C \u2212> G","661"]},{"entry":[{},"G \u2212> A","665"]},{"entry":[{},"T \u2212> A","728"]},{"entry":[{},"T \u2212> G","728"]},{"entry":[{},"G \u2212> C","813"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"As noted above, cluster H19011 features 5 segments, which were listed in Table 92 above. These segments are portions of nucleic acid sequences which are described herein separately because they are of particular interest. A description of each segment according to the present invention is now provided.","Segment cluster H190111_N13 (SEQ ID NO:129) according to the present invention is supported by 3 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcripts: H190111_T8 (SEQ ID NO:45) and H190111_T9 (SEQ ID NO:46). Table 100 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1257","num":"1256"},"tables":{"@attributes":{"id":"TABLE-US-00130","num":"00130"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 100"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["H19011_1_T8 (SEQ ID NO: 45)","884","1407"]},{"entry":["H19011_1_T9 (SEQ ID NO: 46)","827","1350"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"According to an optional embodiment of the present invention, short segments related to the above cluster are also provided. These segments are up to about 120 bp in length, and so are included in a separate description.","Segment cluster H190111_N8 (SEQ ID NO:130) according to the present invention is supported by 4 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcripts: H190111_T8 (SEQ ID NO:45). Table 101 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1260","num":"1259"},"tables":{"@attributes":{"id":"TABLE-US-00131","num":"00131"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 101"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["H19011_1_T8 (SEQ ID NO: 45)","680","736"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster H190111_N10 (SEQ ID NO:131) according to the present invention is supported by 5 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcripts: H190111_T8 (SEQ ID NO:45) and H190111_T9 (SEQ ID NO:46). Table 102 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1262","num":"1261"},"tables":{"@attributes":{"id":"TABLE-US-00132","num":"00132"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 102"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["H19011_1_T8 (SEQ ID NO: 45)","737","797"]},{"entry":["H19011_1_T9 (SEQ ID NO: 46)","680","740"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster H190111_N11 (SEQ ID NO:132) according to the present invention is supported by 3 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcripts: H190111_T8 (SEQ ID NO:45) and H190111_T9 (SEQ ID NO:46). Table 103 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1264","num":"1263"},"tables":{"@attributes":{"id":"TABLE-US-00133","num":"00133"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 103"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["H19011_1_T8 (SEQ ID NO: 45)","798","863"]},{"entry":["H19011_1_T9 (SEQ ID NO: 46)","741","806"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Segment cluster H190111_N12 (SEQ ID NO:133) according to the present invention is supported by 5 libraries. The number of libraries was determined as previously described. This segment can be found in the following transcripts: H190111_T8 (SEQ ID NO:45) and H190111_T9 (SEQ ID NO:46). Table 104 below describes the starting and ending position of this segment on each transcript.",{"@attributes":{"id":"p-1266","num":"1265"},"tables":{"@attributes":{"id":"TABLE-US-00134","num":"00134"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 104"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segment location on transcripts"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Segment","Segment"]},{"entry":["Transcript name","starting position","ending position"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["H19011_1_T8 (SEQ ID NO: 45)","864","883"]},{"entry":["H19011_1_T9 (SEQ ID NO: 46)","807","826"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Expression of C1ORF32, chromosome 1 open reading frame 32, H19011 transcripts which are detectable by amplicon as depicted in sequence name H19011_seg13F2R2 (SEQ ID NO: 235) in normal and cancerous Colon tissues, in normal and cancerous Lung tissues and in different normal tissues","Expression of C1ORF32, chromosome 1 open reading frame 32, transcripts detectable by or according to seg13\u2014H19011_seg13F2R2 (SEQ ID NO: 235) amplicon and primers H19011_seg13F2 (SEQ ID NO: 233) and H19011_seg13R2 (SEQ ID NO: 234) was measured by real time PCR on colon panel, lung panel and normal panel. The samples used are detailed in Table 5, Table 3 and Table 2 above, respectively. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1.","Colon panel\u2014The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 42-70, Table 5 above). Then the reciprocal of this ratio was calculated, to obtain a value of fold down-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-1270","num":"1269"},"figref":"FIG. 39"},"As is evident from , the expression of C1ORF32 transcripts detectable by the above amplicon in cancer samples was significantly lower than in the non-cancerous samples (sample numbers 42-70, Table 5 above). Notably down regulation of at least 6 fold was found in 17 out of 55 adenocarcinoma samples.","Statistical analysis was applied to verify the significance of these results, as described below. The P value for the difference in the expression levels of C1ORF32 transcripts detectable by the above amplicon in Colon cancer samples versus the normal tissue samples was determined by T test as 9.36e-004.","Threshold of 6 fold down regulation was found to differentiate between cancer and normal samples with P value of 2.67e-004 as checked by exact Fisher test.","The above values demonstrate statistical significance of the results.","Lung panel\u2014The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 51-64 and 69-70, Table 3 above), to obtain a value of fold up-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-1276","num":"1275"},"figref":"FIG. 40"},"As is evident from , the expression of C1ORF32 transcripts detectable by the above amplicon in small cell carcinoma samples was significantly higher than in the non-cancerous samples (sample numbers 51-64 and 69-70, Table 3 above). Notably an over-expression of at least 6 fold was found in 9 out of 9 small cell carcinoma samples.","Statistical analysis was applied to verify the significance of these results, as described below.","The P value for the difference in the expression levels of C1ORF32 transcripts detectable by the above amplicon in Lung small cell carcinoma samples versus the normal tissue samples was determined by T test as 3.43e-003.","Threshold of 6 fold over expression was found to differentiate between small cell carcinoma and normal samples with P value of 4.89e-007 as checked by exact Fisher test.","The above values demonstrate statistical significance of the results.","Normal panel\u2014The normalized quantity of each RT sample was then divided by the median of the quantities of the colon samples (sample numbers 3, 4 and 5, Table 2 above), to obtain a value of relative expression of each sample relative to median of the colon samples, as shown in . The normalized quantity of each RT sample was then divided by the median of the quantities of the lung samples (sample numbers 26, 28, 29 and 30, Table 2 above), to obtain a value of relative expression of each sample relative to median of the lung samples, as shown in .","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: H19011_seg13F2 (SEQ ID NO: 233) forward primer; and H19011_seg13R2 (SEQ ID NO: 234) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: H19011_seg13F2R2 (SEQ ID NO: 235).",{"@attributes":{"id":"p-1285","num":"1284"},"tables":{"@attributes":{"id":"TABLE-US-00135","num":"00135"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>H19011_seg13F2\u2003(SEQ\u2003ID\u2003NO:\u2003233):"},{"entry":"GTGAGTACAGTGACCGCTGGG"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>H19011_seg13R2\u2003(SEQ\u2003ID\u2003NO:\u2003234):"},{"entry":"GGAGAAGAGTCTGGAATGACCAA"},{"entry":{}},{"entry":"Amplicon\u2003>H19011_seg13F2R2\u2003(SEQ\u2003ID\u2003NO:\u2003235)"},{"entry":"GTGAGTACAGTGACCGCTGGGGAGACAGAGCGATCGAGAGAAATGTCTA"},{"entry":{}},{"entry":"CCTCTCTACCTGACAGCTGTGTGCGCTGGGTTCCTCCTCCACCTCCTGT"},{"entry":{}},{"entry":"CCTGCCACCCCCAAGATTGGTCATTCCAGACTCTTCTCC"}]}}}}},"Expression of C1ORF32, chromosome 1 open reading frame 32, H19011 transcripts which are detectable by amplicon as depicted in sequence name H19011_seg8-13F1R1 (SEQ ID NO: 238) in normal and cancerous Lung tissues","Expression of C1ORF32, chromosome 1 open reading frame 32, transcripts detectable by or according to seg8-13F1R1_ H19011_seg8-13F1R1 (SEQ ID NO: 238) amplicon and primers H19011_seg8-13F1 (SEQ ID NO: 236) and H19011_seg8-13R1 (SEQ ID NO: 237) was measured by real time PCR on lung panel. The samples used are detailed in Table 3 above. Samples that showed no detection of the amplicon (samples no. 1, 2, 4-10, 12-27, 29-35, 37-41, 51-64 and 69-70, Table 3) were assigned Ct value of 41 and were calculated accordingly. These samples showed a primer-dimer product with a characteristic dissociation curve and a significantly lower TM (this artefactual product was identified by its appearance in the negative control without RT sample). For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1. The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 51-64 and 69-70, Table 3 above), to obtain a value of fold up-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-1288","num":"1287"},"figref":"FIG. 42"},"As is evident from , the expression of C1ORF32 transcripts detectable by the above amplicon in small cell carcinoma samples was significantly higher than in the non-cancerous samples (sample numbers 51-64 and 69-70, Table 3 above). Notably an over-expression of at least 500 fold was found in 9 out of 9 small cell carcinoma samples.","Statistical analysis was applied to verify the significance of these results, as described below. The P value for the difference in the expression levels of C1ORF32 transcripts detectable by the above amplicon in Lung small cell carcinoma samples versus the normal tissue samples was determined by T test as 6.70e-003.","Threshold of 500 fold over expression was found to differentiate between small cell carcinoma and normal samples with P value of 4.89e-007 as checked by exact Fisher test.","The above values demonstrate statistical significance of the results.","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: H19011_seg8-13F1 (SEQ ID NO: 236) forward primer; and H19011_seg8-13R1 (SEQ ID NO: 237) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: H19011_seg8-13F1R1 (SEQ ID NO: 238).",{"@attributes":{"id":"p-1295","num":"1294"},"tables":{"@attributes":{"id":"TABLE-US-00136","num":"00136"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>H19011_seg8-13F1\u2003(SEQ\u2003ID\u2003NO:\u2003236):"},{"entry":"GCCCAGTTTTGCTGTGGAGA"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>H19011_seg8-13R1\u2003(SEQ\u2003ID\u2003NO:\u2003237):"},{"entry":"GGTAGACATTTCTCTCGATCGCTC"},{"entry":{}},{"entry":"Amplicon\u2003>H19011_seg8-13F1R1\u2003(SEQ\u2003ID\u2003NO:\u2003238)"},{"entry":"GCCCAGTTTTGCTGTGGAGATTATGCCAGAGTGGGTGTTTGTTGGCCTG"},{"entry":{}},{"entry":"GTGCTCCTGGGCGTCTTCCTCTTCTTCGTCCTGGTGGGGATCTGCTGGT"},{"entry":{}},{"entry":"GCCAGTGCTGCCCTCACAGCTGCTGCTGCTATGTCCGCTGCCCATGCTG"},{"entry":{}},{"entry":"CCCAGATTCCTGCTGGTGCCCTCAAGCCTGTGAGTACAGTGACCGCTGG"},{"entry":{}},{"entry":"GGAGACAGAGCGATCGAGAGAAATGTCTACC"}]}}}}},"Expression of C1ORF32, chromosome 1 open reading frame 32, H19011 transcripts which are detectable by amplicon as depicted in sequence name H19011_junc8-10seg13 (SEQ ID NO: 241) in normal and cancerous lung tissues, in normal and cancerous colon tissues, in different normal tissues and in the blood-specific panel.","Expression of C1ORF32 transcripts detectable by or according to junc8-10seg13\u2014H19011_junc8-10seg13 (SEQ ID NO: 241) amplicon and primers H19011_junc8-10seg13F1 (SEQ ID NO: 239) and H19011_junc8-10seg13R1 (SEQ ID NO: 240) was measured by real time PCR lung panel, colon panel, normal panel and blood panel. The samples used are detailed in Table 3, Table 5, Table 2 and Table 1 above, respectively. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1.","For lung panel\u2014Non-detected sample (sample no. 69, Table 3) was assigned Ct value of 41 and was calculated accordingly. The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 51, 53, 54, 56, 57, 59, 61, 62, 64 and 70, Table 3 above), to obtain a value of fold up-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-1299","num":"1298"},"figref":"FIG. 43"},"As is evident from , the expression of C1ORF32 transcripts detectable by the above amplicon in small cell carcinoma samples was significantly higher than in the non-cancerous samples (sample numbers 51, 53, 54, 56, 57, 59, 61, 62, 64 and 70, Table 3 above). Notably an over-expression of at least 7 fold was found in 9 out of 9 small cell carcinoma samples.","Statistical analysis was applied to verify the significance of these results, as described below.","The P value for the difference in the expression levels of C1ORF32 transcripts detectable by the above amplicon in Lung small cell carcinoma samples versus the normal tissue samples was determined by T test as 2.34e-003.","Threshold of 7 fold over expression was found to differentiate between small cell carcinoma and normal samples with P value of 1.08e-005 as checked by exact Fisher test.","The above values demonstrate statistical significance of the results.","For colon panel\u2014Non-detected sample (sample no. 79, Table 5) was assigned Ct value of 41 and was calculated accordingly. The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 42-62 and 65-70, Table 5 above). Then the reciprocal of this ratio was calculated, to obtain a value of fold down-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-1306","num":"1305"},"figref":"FIG. 44"},"As is evident from , the expression of C1ORF32 transcripts detectable by the above amplicon in cancer samples was significantly lower than in the non-cancerous samples (sample numbers 42-62 and 65-70, Table 5 above). Notably down regulation of at least 5 fold was found in 15 out of 36 adenocarcinoma samples.","Statistical analysis was applied to verify the significance of these results, as described below. Threshold of 5 fold down regulation was found to differentiate between cancer and normal samples with P value of 4.29e-004 as checked by exact Fisher test.","The above values demonstrate statistical significance of the results.","For normal panel\u2014Non-detected samples (samples no. 42 and 49, Table 2) were assigned Ct value of 41 and were calculated accordingly. The normalized quantity of each RT sample was then divided by the median of the quantities of the colon samples (sample numbers 4 and 5, Table 2 above), to obtain a value of relative expression of each sample relative to median of the colon samples, as shown in . The normalized quantity of each RT sample was then divided by the median of the quantities of the lung samples (sample numbers 26, 29 and 30, Table 2 above), to obtain a value of relative expression of each sample relative to median of the lung samples, as shown in .","For blood panel\u2014The normalized quantity of each RT sample was then divided by the median of the quantities of the kidney normal samples (sample numbers 65-67, Table 1 above), to obtain a value of relative expression of each sample relative to median of the kidney normal samples.","The results of this analysis are depicted in the histogram in . Expression of the above-indicated C1ORF32 transcript is high in one lymphoma sample (sample no. 33, Table 1) but in normal brain sample too.","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: H19011 junc8-10seg13F1 (SEQ ID NO: 239) forward primer; and H19011 junc8-10seg13R1 (SEQ ID NO: 240) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: H19011 junc8-10seg13F1R1 (SEQ ID NO: 241).",{"@attributes":{"id":"p-1315","num":"1314"},"tables":{"@attributes":{"id":"TABLE-US-00137","num":"00137"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>H19011_junc8-10seg13F1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003239)"},{"entry":"TGTGGAGATTATGCCAGAGTGG"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>H19011_junc8-10seg13R1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003240)"},{"entry":"GACATTTCTCTCGATCGCTCTGT"},{"entry":{}},{"entry":"Amplicon\u2003>H19011_junc8-10seg13F1R1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003241)"},{"entry":"TGTGGAGATTATGCCAGAGTGGGTGTTTGTTGGCCTGGTGCTCCTGGGC"},{"entry":{}},{"entry":"GTCTTCCTCTTCTTCGTCCTGGTGGGGATCTGCTGGTGCCAGTGCTGCC"},{"entry":{}},{"entry":"CTCACAGCTGCTGCTGCTATGTCCGCTGCCCATGCTGCCCAGATTCCTG"},{"entry":{}},{"entry":"CTGGTGCCCTCAAGCCTGTGAGTACAGTGACCGCTGGGGAGACAGAGCG"},{"entry":{}},{"entry":"ATCGAGAGAAATGTC"}]}}}}},"Expression of C1ORF32, chromosome 1 open reading frame 32, H19011 transcripts which are detectable by amplicon as depicted in sequence name H19011_junc6-10 (SEQ ID NO: 244) in normal and cancerous lung tissues and in normal and cancerous Colon tissues","Expression of C1ORF32 transcripts detectable by or according to junc6-10-H19011_junc6-10F1R1 (SEQ ID NO: 244) amplicon and primers H19011_junc6-10F1 (SEQ ID NO: 242) and H19011_junc6-10R1 (SEQ ID NO: 243) was measured by real time PCR on lung panel and colon panel. The samples used are detailed in Table 3 and Table 5 above, respectively. For each RT sample, the expression of the above amplicon was normalized to the normalization factor calculated from the expression of several house keeping genes as described in Example 1.","Lung panel\u2014The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 51-64, 69 and 70, Table 3 above). Then the reciprocal of this ratio was calculated, to obtain a value of fold down-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-1319","num":"1318"},"figref":"FIG. 47"},"As is evident from , the expression of C1ORF32 transcripts detectable by the above amplicon in non-small cell carcinoma samples, adenocarcinoma and squamous cell carcinoma was significantly lower than in the non-cancerous samples (sample numbers 51-64, 69 and 70, Table 3 above). Notably down regulation of at least 5 fold was found in 23 out of 39 non-small cell carcinoma samples especially in 8 out of 17 adenocarcinoma samples and in 12 out of 16 squamous cell carcinoma samples.","Statistical analysis was applied to verify the significance of these results, as described below. The P value for the difference in the expression levels of C1ORF32 transcripts detectable by the above amplicon lung non-small cell carcinoma, lung adenocarcinoma and lung squamous cell carcinoma samples, versus the normal tissue samples was determined by T test as 1.18e-003, 2.87e-002 and 3.55e-004, respectively.","Threshold of 5 fold down regulation was found to differentiate between lung non-small cell carcinoma, lung adenocarcinoma and lung squamous cell carcinoma samples and normal samples with P value of 1.59e-003, 3.54e-002 and 4.78e-004, respectively, as checked by exact Fisher test.","The above values demonstrate statistical significance of the results.","Colon panel\u2014The normalized quantity of each RT sample was then divided by the median of the quantities of the normal samples (sample numbers 42-70, Table 5 above). Then the reciprocal of this ratio was calculated, to obtain a value of fold down-regulation for each sample relative to median of the normal samples.",{"@attributes":{"id":"p-1325","num":"1324"},"figref":"FIG. 48"},"As is evident from , the expression of C1ORF32 transcripts detectable by the above amplicon in cancer samples was significantly lower than in the non-cancerous samples (sample numbers 42-70, Table 5 above). Notably down regulation of at least 9 fold was found in 23 out of 55 adenocarcinoma samples.","Statistical analysis was applied to verify the significance of these results, as described below.","Threshold of 9 fold down regulation was found to differentiate between cancer and normal samples with P value of 7.39e-006 as checked by exact Fisher test.","The above values demonstrate statistical significance of the results.","Primer pairs are also optionally and preferably encompassed within the present invention; for example, for the above experiment, the following primer pair was used as a non-limiting illustrative example only of a suitable primer pair: H19011 junc6-10F1 (SEQ ID NO: 242) forward primer; and H19011 junc6-10R1 (SEQ ID NO: 243) reverse primer.","The present invention also preferably encompasses any amplicon obtained through the use of any suitable primer pair; for example, for the above experiment, the following amplicon was obtained as a non-limiting illustrative example only of a suitable amplicon: H19011 junc6-10F1R1 (SEQ ID NO: 244).",{"@attributes":{"id":"p-1332","num":"1331"},"tables":{"@attributes":{"id":"TABLE-US-00138","num":"00138"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Forward\u2003Primer\u2003>H19011_junc6-10F1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003242)"},{"entry":"ACTCTATTACTGTATTATCACCACCCCAG"},{"entry":{}},{"entry":"Reverse\u2003Primer\u2003>H19011_junc6-10R1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003243)"},{"entry":"CCAACAAACACCCACTCCAAC"},{"entry":{}},{"entry":"Amplicon\u2003>H19011_junc6-10F1R1"},{"entry":"(SEQ\u2003ID\u2003NO:\u2003244)"},{"entry":"ACTCTATTACTGTATTATCACCACCCCAGATGACCTGGAGGGGAAAAA"},{"entry":{}},{"entry":"TGAGGGCTCACTGGGACTGCTGGTGTTGGAGTGGGTGTTTGTTGG"}]}}}}},"The present invention relates to a specific antigen FXYD3 and related diagnostic and therapeutics based thereon.","According to the present invention, Cluster R31375 (internal ID 72360301) features 19 transcripts and 4 segments of interest, the names for which are given in Tables 105 and 106, respectively. The selected protein variants are given in table 107.",{"@attributes":{"id":"p-1335","num":"1334"},"tables":{"@attributes":{"id":"TABLE-US-00139","num":"00139"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 105"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Transcripts of interest"},{"entry":"Transcript Name"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"161pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"R31375_T0 (SEQ ID NO: 51)"]},{"entry":[{},"R31375_T1 (SEQ ID NO: 52)"]},{"entry":[{},"R31375_T2 (SEQ ID NO: 53)"]},{"entry":[{},"R31375_T3 (SEQ ID NO: 54)"]},{"entry":[{},"R31375_T4 (SEQ ID NO: 55)"]},{"entry":[{},"R31375_T5 (SEQ ID NO: 56)"]},{"entry":[{},"R31375_T6 (SEQ ID NO: 57)"]},{"entry":[{},"R31375_T7 (SEQ ID NO: 58)"]},{"entry":[{},"R31375_T8 (SEQ ID NO: 59)"]},{"entry":[{},"R31375_T9 (SEQ ID NO: 60)"]},{"entry":[{},"R31375_T10 (SEQ ID NO: 61)"]},{"entry":[{},"R31375_T11 (SEQ ID NO: 62)"]},{"entry":[{},"R31375_T12 (SEQ ID NO: 63)"]},{"entry":[{},"R31375_T13 (SEQ ID NO: 64)"]},{"entry":[{},"R31375_T19 (SEQ ID NO: 65)"]},{"entry":[{},"R31375_T25 (SEQ ID NO: 66)"]},{"entry":[{},"R31375_T26 (SEQ ID NO: 67)"]},{"entry":[{},"R31375_T29 (SEQ ID NO: 68)"]},{"entry":[{},"R31375_T39 (SEQ ID NO: 69)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-1336","num":"1335"},"tables":{"@attributes":{"id":"TABLE-US-00140","num":"00140"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 106"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Segments of interest"},{"entry":"Segment Name"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"161pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"R31375_N30 (SEQ ID NO: 134)"]},{"entry":[{},"R31375_N33 (SEQ ID NO: 135)"]},{"entry":[{},"R31375_N34 (SEQ ID NO: 136)"]},{"entry":[{},"R31375_N37 (SEQ ID NO: 137)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-1337","num":"1336"},"tables":{"@attributes":{"id":"TABLE-US-00141","num":"00141"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 107"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Proteins of interest"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Protein Name","Corresponding Transcripts"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["R31375_P0 (SEQ","R31375_T0 (SEQ ID NO: 51); R31375_T1 (SEQ"]},{"entry":["ID NO: 70)","ID NO: 52); R31375_T10 (SEQ ID NO: 61);"]},{"entry":[{},"R31375_T11 (SEQ ID NO: 62); R31375_T12"]},{"entry":[{},"(SEQ ID NO: 63); R31375_T13 (SEQ ID NO:"]},{"entry":[{},"64); R31375_T2 (SEQ ID NO: 53); R31375_T3"]},{"entry":[{},"(SEQ ID NO: 54); R31375_T4 (SEQ ID NO: 55);"]},{"entry":[{},"R31375_T5 (SEQ ID NO: 56); R31375_T6 (SEQ"]},{"entry":[{},"ID NO: 57); R31375_T7 (SEQ ID NO: 58);"]},{"entry":[{},"R31375_T8 (SEQ ID NO: 59); R31375_T9 (SEQ"]},{"entry":[{},"ID NO: 60)"]},{"entry":["R31375_P14 (SEQ","R31375_T19 (SEQ ID NO: 65); R31375_T25"]},{"entry":["ID NO: 72)","(SEQ ID NO: 66); R31375_T26 (SEQ ID"]},{"entry":[{},"NO: 67)"]},{"entry":["R31375_P31 (SEQ","R31375_T29 (SEQ ID NO: 68)"]},{"entry":"ID NO: 73)"},{"entry":["R31375_P33 (SEQ","R31375_T39 (SEQ ID NO: 69)"]},{"entry":"ID NO: 74)"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"These sequences are variants of the known protein FXYD domain-containing ion transport regulator 3 precursor (SwissProt accession identifier FXYD3_HUMAN (SEQ ID NO: 70); known also according to the synonyms Chloride conductance inducer protein Mat-8; Mammary tumor 8 kDa protein; Phospholemman-like), referred to herein as the previously known protein.","FXYD3 was previously identified within a set of genes induced by the neu or Ha-Ras oncogenes in murine breast tumors, and was named Mat-8 (Mammary tumor, 8 kDa) (Morrison et al 1995). In normal tissues, FXYD3 is mainly expressed in the uterus, stomach, colon and skin (Morrison et al 1995). Its expression was found elevated in human primary breast tumors, as well as prostate carcinoma and pancreatic ductal adenocarcinoma (Grzmil et al 2004, Kayed et al 2006). Specific inhibition of its expression by siRNA in prostate cancer cell lines indicates a role in cellular proliferation (Grzmil et al 2004).","FXYD3 belongs to the FXYD family proteins. The seven known members of this family are all small membrane proteins that contain a common signature of 6 amino acids comprising the FXYD motif. Most FXYD proteins, including human FXYD3, are type I membrane proteins, containing a transmembrane domain and a cleavable signal peptide. However, the signal peptide of mouse FXYD3 is not cleaved and the signal peptide may act as a second transmembrane domain (Crambert et al 2005, Geering 2006). FXYD3, like other members of the FXYD family, interacts with Na\/K-ATPase and modulates its activity in a tissue-specific manner (Crambert et al 2005, Arimochi et al 2007, Geering 2006).","Two splice variant isoforms of FXYD3 have been previously described (Bibert et al 2006). These differ in a 26 amino acids in frame insertion after the transmembrane domain, and are differentially expressed during cell differentiation. Furthermore, both isoforms are able to stably associate with Na\/K-ATPase and play different functional roles in the regulation of the activities of this ATPase (Bibert et al 2006).","In addition, WO 2003101283 may be relevant to the present invention. This PCT application purports to disclose that R31375_P0 (SEQ ID NO:70) (wild-type FXYD3 nucleic acid coding sequence reported herein) is a differentially expressed nucleic acid which encodes a protein sequences that allegedly may be used as a diagnostic marker for human lung cancer.","Further, WO2003000012 purports to disclose a human breast cancer related protein referred to as protein #12 that seems to correspond to the R31375_ P0 (wild-type) disclosed herein. Also, U.S. Pat. No. 7,189,507 discloses a gene referred to as MAT8 in a long table of gene sequences that seems to correspond to R31375_P0 (SEQ ID NO:70). The table seems to suggest that this gene may be expressed in ovarian cancer.","Protein FXYD domain-containing ion transport regulator 3 precursor (SEQ ID NO:70) is known or believed to have the following functions: Induces a hyperpolarization-activated chloride current when expressed in oocytes. May be a modulator capable of activating endogenous oocyte channels. Known polymorphisms for this sequence are as shown in Table 108.",{"@attributes":{"id":"p-1345","num":"1344"},"tables":{"@attributes":{"id":"TABLE-US-00142","num":"00142"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u2003108"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino\u2003acid\u2003mutations\u2003for\u2003Known\u2003Protein"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP\u2003positions\u2003on",{}]},{"entry":["amino\u2003acid\u2003sequence","Comment"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["36-37","Missing"]},{"entry":{}},{"entry":["58","S\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003->"]},{"entry":[{},"SEWRSSGEQAGRGWGSPPLTTQLSPTG"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"Protein FXYD domain-containing ion transport regulator 3 precursor (SEQ ID NO:70) localization is believed to be Membrane; single-pass type I membrane protein (Potential).","The following GO Annotations apply to the previously known protein. The following annotations were found: chloride transport, which are annotations related to Biological Process; chloride channel activity, which are annotations related to Molecular Function; and integral to plasma membrane, which are annotations related to Cellular Component.","The GO assignment relies on information from one or more of the SwissProt\/TremBl Protein knowledge base, available from <http:\/\/www.expasy.ch\/sprot\/>; or Locuslink, available from <http:\/\/www.ncbi.nlm.nih.gov\/projects\/LocusLink\/>.","According to the present invention, novel FXYD3 splice variants were identified as membrane bound proteins that are predicted to be over expressed in cancers. FXYD3 is a type I membrane bound protein (Bibert et. alAl 2006) and according to the present invention 3 novel splice variants of this protein are provided. The novel splice variant referred herein as R31375_P14 (SEQ ID NO:72) has an additional in-frame exon in its extracellular region. The addition of this exon increases the length of the extracellular region by 298 amino acids, which comprises a significant addition to rather short extracellular domain of 18 amino acids of the wild type protein. The novel splice variant referred herein as R31375_P31 (SEQ ID NO:73) has an additional in-frame exon in the juxtamembrane domain of FXYD3, which adds 26 new amino acids to the intracellular region. The novel splice variant referred herein as R31375_P33 (SEQ ID NO:74) skips the 3rd coding exon of the wild type FXYD3 and, like R31375_P31 (SEQ ID NO:73), has the additional in-frame exon in the juxtamembrane domain. This causes the deletion of 8 amino acids in the ectodomain.","According to the present invention FXYD3 and R31375_P14 (SEQ ID NO:72) were shown to be overexpressed in ovarian cancer.","As noted above, cluster R31375 features 19 transcripts, which were listed in Table 105 above. These transcripts encode for proteins which are variants of protein FXYD domain-containing ion transport regulator 3 precursor (SEQ ID NO:70). A description of each variant protein according to the present invention is now provided.","Variant protein R31375_P0 (SEQ ID NO:70) according to the present invention has an amino acid sequence encoded by transcripts R31375_T0 (SEQ ID NO:51), R31375_T1 (SEQ ID NO:52), R31375_T10 (SEQ ID NO:61), R31375_T11 (SEQ ID NO:62), R31375_T12 (SEQ ID NO:63), R31375_T13 (SEQ ID NO:64), R31375_T2 (SEQ ID NO:53), R31375_T3 (SEQ ID NO:54), R31375_T4 (SEQ ID NO:55), R31375_T5 (SEQ ID NO:56), R31375_T6 (SEQ ID NO:57), R31375_T7 (SEQ ID NO:58), R31375_T8 (SEQ ID NO:59) and R31375_T9 (SEQ ID NO:60).","The localization of the variant protein was determined according to results from a number of different software programs and analyses, including analyses from SignalP and other specialized programs. The variant protein is believed to be located as follows with regard to the cell: membrane.","Variant protein R31375_P0 (SEQ ID NO:70) also has the following non-silent SNPs (Single Nucleotide Polymorphisms) as listed in Table 109, (given according to their positions on the amino acid sequence, with the alternative amino acids listed (SEQ ID NO:70)).",{"@attributes":{"id":"p-1355","num":"1354"},"tables":{"@attributes":{"id":"TABLE-US-00143","num":"00143"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 109"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino acid mutations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SNP positions on","Alternative amino"]},{"entry":["amino acid sequence","acids"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["3","K \u2212> R"]},{"entry":["19","D \u2212> A"]},{"entry":["19","D \u2212> V"]},{"entry":["50","A \u2212> P"]},{"entry":["75","E \u2212>"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The coding portion of transcript R31375_T0 (SEQ ID NO:51) starts at position 491 and ends at position 751. The transcript also has the following SNPs as listed in Table 110 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1357","num":"1356"},"tables":{"@attributes":{"id":"TABLE-US-00144","num":"00144"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 110"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"C \u2212> A","469"]},{"entry":[{},"A \u2212> G","498, 900, 1571"]},{"entry":[{},"A \u2212> T","546"]},{"entry":[{},"A \u2212> C","546"]},{"entry":[{},"T \u2212> A","607, 926\u2002"]},{"entry":[{},"T \u2212> G","607"]},{"entry":[{},"G \u2212> C","638"]},{"entry":[{},"G \u2212>","713"]},{"entry":[{},"C \u2212> T","792"]},{"entry":[{},"G \u2212> A","901, 1572"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The coding portion of transcript R31375_T1 (SEQ ID NO:52) starts at position 795 and ends at position 1055. The transcript also has the following SNPs as listed in Table 111 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1359","num":"1358"},"tables":{"@attributes":{"id":"TABLE-US-00145","num":"00145"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 111"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"G \u2212> A","513, 1205, 1876"]},{"entry":[{},"C \u2212> A","773"]},{"entry":[{},"A \u2212> G","802, 1204, 1875"]},{"entry":[{},"A \u2212> T","850"]},{"entry":[{},"A \u2212> C","850"]},{"entry":[{},"T \u2212> A","911, 1230"]},{"entry":[{},"T \u2212> G","911"]},{"entry":[{},"G \u2212> C","942"]},{"entry":[{},"G \u2212>","1017\u2002"]},{"entry":[{},"C \u2212> T","1096\u2002"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The coding portion of transcript R31375_T10 (SEQ ID NO:61) starts at position 1826 and ends at position 2086. The transcript also has the following SNPs as listed in Table 112 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1361","num":"1360"},"tables":{"@attributes":{"id":"TABLE-US-00146","num":"00146"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 112"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"147pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"C \u2212> A","\u2002469"]},{"entry":[{},"A \u2212> G","639, 984, 1504, 1833, 2235, 2906"]},{"entry":[{},"A \u2212> C","860, 1183, 1447, 1504, 1881"]},{"entry":[{},"C \u2212>","1063"]},{"entry":[{},"A \u2212> T","1183, 1447, 1881"]},{"entry":[{},"T \u2212> A","1942, 2261"]},{"entry":[{},"T \u2212> G","1942"]},{"entry":[{},"G \u2212> C","1973"]},{"entry":[{},"G \u2212>","2048"]},{"entry":[{},"C \u2212> T","2127"]},{"entry":[{},"G \u2212> A","2236, 2907"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The coding portion of transcript R31375_T11 (SEQ ID NO:62) starts at position 613 and ends at position 873. The transcript also has the following SNPs as listed in Table 113 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1363","num":"1362"},"tables":{"@attributes":{"id":"TABLE-US-00147","num":"00147"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 113"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"C \u2212> A","469"]},{"entry":[{},"A \u2212> G","620, 1022, 1693"]},{"entry":[{},"A \u2212> T","668"]},{"entry":[{},"A \u2212> C","668"]},{"entry":[{},"T \u2212> A","\u2002729, 1048"]},{"entry":[{},"T \u2212> G","729"]},{"entry":[{},"G \u2212> C","760"]},{"entry":[{},"G \u2212>","835"]},{"entry":[{},"C \u2212> T","914"]},{"entry":[{},"G \u2212> A","1023, 1694"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The coding portion of transcript R31375_T12 (SEQ ID NO:63) starts at position 711 and ends at position 971. The transcript also has the following SNPs as listed in Table 114 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1365","num":"1364"},"tables":{"@attributes":{"id":"TABLE-US-00148","num":"00148"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 114"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"C \u2212> A","469"]},{"entry":[{},"G \u2212>","545, 933"]},{"entry":[{},"G \u2212> C","545, 858"]},{"entry":[{},"C \u2212>","549"]},{"entry":[{},"A \u2212> G","718, 1120, 1791"]},{"entry":[{},"A \u2212> T","766"]},{"entry":[{},"A \u2212> C","766"]},{"entry":[{},"T \u2212> A","\u2002827, 1146"]},{"entry":[{},"T \u2212> G","827"]},{"entry":[{},"C \u2212> T","1012\u2002"]},{"entry":[{},"G \u2212> A","1121, 1792"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The coding portion of transcript R31375_T13 (SEQ ID NO:64) starts at position 1015 and ends at position 1275. The transcript also has the following SNPs as listed in Table 115 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1367","num":"1366"},"tables":{"@attributes":{"id":"TABLE-US-00149","num":"00149"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 115"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"G \u2212> A","513, 1425, 2096"]},{"entry":[{},"C \u2212> A","\u2002773"]},{"entry":[{},"G \u2212>","849, 1237"]},{"entry":[{},"G \u2212> C","849, 1162"]},{"entry":[{},"C \u2212>","\u2002853"]},{"entry":[{},"A \u2212> G","1022, 1424, 2095"]},{"entry":[{},"A \u2212> T","1070"]},{"entry":[{},"A \u2212> C","1070"]},{"entry":[{},"T \u2212> A","1131, 1450\u2002"]},{"entry":[{},"T \u2212> G","1131"]},{"entry":[{},"C \u2212> T","1316"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The coding portion of transcript R31375_T2 (SEQ ID NO:53) starts at position 678 and ends at position 938. The transcript also has the following SNPs as listed in Table 116 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1369","num":"1368"},"tables":{"@attributes":{"id":"TABLE-US-00150","num":"00150"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 116"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"G \u2212> A","513, 1088, 1759"]},{"entry":[{},"C \u2212> A","656"]},{"entry":[{},"A \u2212> G","685, 1087, 1758"]},{"entry":[{},"A \u2212> T","733"]},{"entry":[{},"A \u2212> C","733"]},{"entry":[{},"T \u2212> A","794, 1113"]},{"entry":[{},"T \u2212> G","794"]},{"entry":[{},"G \u2212> C","825"]},{"entry":[{},"G \u2212>","900"]},{"entry":[{},"C \u2212> T","979"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The coding portion of transcript R31375_T3 (SEQ ID NO:54) starts at position 572 and ends at position 832. The transcript also has the following SNPs as listed in Table 117 (given according to their position on the nucleotide sequence, with the alternative nucleic acid listed).",{"@attributes":{"id":"p-1371","num":"1370"},"tables":{"@attributes":{"id":"TABLE-US-00151","num":"00151"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 117"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Nucleic acid SNPs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SNP positions on"]},{"entry":[{},"Polymorphism","nucleotide sequence"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","r
