---
title: Polyvalent RNA-nanoparticle compositions
abstract: The present invention concerns nanoparticles functionalized with duplex RNA for a variety of uses, including but not limited to gene regulation. More specifically, the disclosure provides a new strategy for conjugating RNA to a nanoparticle to achieve increased stability and activity.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09139827&OS=09139827&RS=09139827
owner: NORTHWESTERN UNIVERSITY
number: 09139827
owner_city: Evanston
owner_country: US
publication_date: 20091124
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATIONS","STATEMENT OF GOVERNMENT INTEREST","FIELD OF THE INVENTION","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","Nanoparticles","Polynucleotide Attachment to a Nanoparticle","Polynucleotides","Polynucleotide Features","Polynucleotide Sequences and Hybridization","Modified Polynucleotides","Chimerics","EXAMPLES","Example 1","Preparation of RNase-free Nanoparticles","Example 2","Modification of RNase-free Nanoparticles","Example 3","Cellular Uptake of RNA-Nanoparticle Compositions","Example 4","Activity of RNA-Nanoparticle Compositions","Example 5","Orientation of Polynucleotide on Nanoparticle Surface"],"p":["This application claims priority benefit under 35 U.S.C. \u00a7119(e) of Provisional U.S. Patent Application No. 61\/117,449, filed Nov. 24, 2008, the disclosure of which is incorporated herein by reference in its entirety.","This invention was made with government support under Grant Number 5U54 CA119341 awarded by the National Institutes of Health, National Cancer Institute\/Centers of Cancer Nanotechnology Excellence (NCI\/CCNE) and Grant Number 5DP1 OD000285 awarded by the National Institutes of Health (NIH). The government has certain rights in the invention.","The present invention concerns nanoparticles functionalized with duplex RNA. The invention also provides a method for conjugating RNA to a nanoparticle.","RNA interference (RNAi) is a phenomenon wherein double-stranded RNA (dsRNA), when present in a cell, inhibits expression of a gene that has a sufficiently complementary sequence to a single strand in the double-stranded RNA. Inhibition of gene expression is caused by degradation of messenger RNA (mRNA) transcribed from the target gene [Sharp et al., Genes and Development 15: 485-490 (2001)]. The double-stranded RNA responsible for inducing RNAi is termed interfering RNA. The mechanism and cellular machinery through which dsRNA mediates RNAi has been investigated using both genetic and biochemical approaches. Biochemical analyses suggest that dsRNA introduced into the cytoplasm of a cell is first processed into RNA fragments 21-25 nucleotides long [Hammond et al., Nature 404: 293-296 (2000); Hamilton et al., Science 286: 950-952 (1999); Zamore et al., Cell 101: 25-33 (2000); Yang et al., Current Biology 10: 1191-1200 (2000); Parrish et al., Molecular Cell 6: 1077-1087 (2000)]. It has been shown in in vitro studies that these dsRNAs, termed small interfering RNAs (siRNA) are generated at least in one mechanism by the RNAse III-like enzyme Dicer [Hammond et al., Nature 404: 293-296 (2000)]. These siRNAs likely act as guides for mRNA cleavage, as the target mRNA is cleaved at a position in the center of the region hybridized to a particular siRNA [Sharp 2001]. Biochemical evidence suggests that the siRNA is part of a multicomponent nuclease complex termed the RNA-induced silencing complex (RISC) [Hammond et al., Nature 404: 293-296 (2000)]. One of the proteins of this complex, Argonaute2, has been identified as a product of the argonaute gene family [Sharp et al., Genes and Development 15: 485-490 (2001);]. This protein is essential for mouse development, and cells lacking Argonaute2 are unable to mount an experimental response to siRNAs. Mutations within a cryptic ribonuclease H domain within Argonaute2, as identified by comparison with the structure of an archeal Argonaute protein, inactivate RISC. Thus, Argonaute contributes \u201cSlicer\u201d activity to RISC, providing the catalytic engine for RNAi [Liu et al., Science 305(5689): 1437-1441 (2004)].","This gene family, which also contains the homolog rde-1 and related genes, the homolog qde-2, and the homolog arg-1, has been shown to be required for RNAi through genetic studies [Sharp et al., Genes and Development 15: 485-490 (2001); Hammond et al., Nature 404: 293-296 (2000); Hamilton et al., Science 286: 950-952 (1999)]. Genetic screens in have also identified the mut-7 gene as essential for RNAi. This gene bears resemblance to RNAse D, suggesting that its gene product acts in the mRNA degradation step of the reaction [Sharp et al., Genes and Development 15: 485-490 (2001)].","Over the past decade, researchers have designed, synthesized, studied, and applied polyvalent DNA-functionalized gold nanoparticles (DNA-Au NPs). [Mirkin et al., Nature 382: 607 (1996)]. These efforts have resulted in a new fundamental understanding of hybrid nanostructures [Demers et al., Anal. Chem. 72: 5535 (2000); Jin et al., J. Am. Chem. Soc. 125: 1643 (2003); Lytton-Jean et al., J. Am. Chem Soc 127: 12754-12754 (2005); Storhoff et al., J. Am. Chem. Soc. 122: 4640 (2000); You et al., Soft Matter 2: 190 (2006); Wang et al., Nanomed. 1: 413 (2006)], important and in certain cases commercially viable detection and diagnostic assays [Nam et al., Science 301: 1884 (2003); Stoeva et al., J. Am. Chem. Soc. 128: 8378 (2006); Liu et al., J. Am. Chem. Soc. 126: 12298 (2004); Faulds et al., Anal. Chem. 76: 412 (2004)], and the ability to program materials assembly through the use of DNA synthons [Mirkin et al., Nature 382: 607 (1996); Park et al., Nature 451: 553 (2008); Nykypanchuk et al., Nature, 451: 549 (2008)]. Polyvalent DNA-Au NPs have several unique properties, such as sharp and elevated melting temperatures [Jin et al., J. Am. Chem. Soc. 125: 1643 (2003)], enhanced binding properties [Lytton-Jean et al., J. Am. Chem Soc 127: 12754-12754 (2005)] (as compared with free strands of the same sequence) and distance-dependent optical properties [Elghanian et al., Science 277: 1078 (1997)]. In agreement with research on polyvalent molecular systems [Gestwicki et al., J. Am. Chem. Soc. 124: 14922 (2002)], the high surface DNA density and the ability of the nanoparticles to engage in multidentate interactions are the proposed origin of these unique properties.","Described herein is a nanoparticle composition that has an associated RNA monolayer imparting physical properties such that the composition is useful for genetic regulation. In contrast to other materials for introducing RNA to a cell, these compositions are not simply a delivery tool for RNA, but rather single entity agents that take advantage of the cooperative properties that result from the arrangement and high density of the surface ligands. The composition described herein enters cells without transfection agents and is resistant to degradation in a manner that enhances knockdown activity compared to conventional polymer carriers.","Thus, in some aspects a nanoparticle composition is provided comprising one or more ribonucleic acid (RNA) polynucleotides in association with a nanoparticle, wherein the RNA polynucleotide has a polypeptide interaction site, and wherein the RNA polynucleotide has a sequence that forms a duplex under conditions appropriate to form the duplex, the duplex having at least one domain in a single strand of the duplex sufficiently complementary to a sequence in a target polynucleotide to permit hybridization of the single strand to the target polynucleotide under appropriate conditions, and hybridization of the domain of the duplex to the sequence in the target polynucleotide creates a substrate site recognized by a polypeptide, and the RNA polynucleotide associated with the nanoparticle is in an orientation specific manner with respect to the polypeptide interaction site and the nanoparticle.","In some aspects a nanoparticle composition is provided in which the RNA polynucleotide is covalently associated with the nanoparticle. In other aspects, a nanoparticle composition is provided in which the RNA polynucleotide is not in covalent association with the nanoparticle.","In further aspects, a nanoparticle composition is provided wherein each RNA polynucleotide in association with the nanoparticle has an identical sequence. In other aspects, a nanoparticle composition is provided wherein at least two RNA polynucleotides in association with the nanoparticle have different sequences.","In some embodiments, a nanoparticle composition is provided wherein the duplex comprises a hairpin structure formed by the RNA polynucleotide.","The methods disclosed herein contemplate a nanoparticle composition further comprising an additional polynucleotide, the additional polynucleotide having a sequence sufficiently complementary to a sequence in the RNA polynucleotide to permit hybridization to the RNA polynucleotide under appropriate conditions to form the duplex.","In some aspects, a nanoparticle composition is provided wherein the additional polynucleotide is RNA. In other aspects, a nanoparticle composition is provided wherein the additional polynucleotide is deoxyribonucleic acid (DNA).","In further embodiments, a nanoparticle composition is provided wherein the additional polynucleotide is covalently associated with the nanoparticle. In some aspects, a nanoparticle composition is provided wherein the additional polynucleotide is not covalently associated with the nanoparticle.","In some embodiments, a nanoparticle composition is provided wherein the polypeptide interaction site is located proximal to the nanoparticle with respect to a midpoint in the RNA polynucleotide.","In further embodiments, a nanoparticle composition is provided wherein the polypeptide interaction site is located distal to the nanoparticle with respect to a midpoint in the RNA polynucleotide.","In some aspects of the methods, a nanoparticle composition is provided having surface density of the RNA of at least about 2 pmol\/cmto about 1000 pmol\/cm.","In various embodiments, a nanoparticle composition is provided wherein the polypeptide interaction site associates with a protein selected from the group consisting of RNase H, RNase D, RNase L, RNase III, Dicer, Argonaute, Argonaute2, and human immunodeficiency virus transactivating response RNA-binding protein (TRBP).","Further aspects of the methods provide a nanoparticle composition wherein the domain of the polynucleotide is about 10 nucleotides in length.","In some embodiments, a nanoparticle composition is provided wherein the RNA polynucleotide further includes a second domain sufficiently complementary to a second sequence in the target polynucleotide and hybridization of the second domain of the RNA polynucleotide to the second sequence in the target polynucleotide creates an additional substrate site recognized by a second polypeptide. In some aspects, a nanoparticle composition is provided wherein the second domain of the polynucleotide is about 10 nucleotides in length.","In some aspects, a nanoparticle composition is provided wherein the substrate site and the additional substrate site are the same. In other aspects, a nanoparticle composition is provided wherein the substrate site and the additional substrate site are different.","A nanoparticle composition is provided in some aspects wherein the RNA polynucleotide and the additional polynucleotide are complementary to each other over a length sufficient to allow hybridization. In other aspects, a nanoparticle composition is provided wherein the RNA polynucleotide and the additional polynucleotide are complementary to each other over their entire length.","In some embodiments, a nanoparticle composition is provided wherein the RNA polynucleotide is conjugated to the nanoparticle through a thiol linkage.","In some aspects, a nanoparticle composition is provided wherein the RNA polynucleotide has a half-life that is at least substantially the same as a half-life of identical RNA polynucleotide that is not associated with a nanoparticle. In other aspects, a nanoparticle composition is provided wherein the RNA polynucleotide has a half-life that is about 1-fold greater, about 2-fold greater, about 3-fold greater, about 4-fold greater, about 5-fold greater or more than a half-life of identical RNA that is not associated with a nanoparticle.","In various embodiments, a nanoparticle composition is provided wherein the RNA polynucleotide is about 5 to about 100 nucleotides in length. In some aspects, a nanoparticle composition is provided wherein the additional polynucleotide is about 5 to about 100 nucleotides in length.","In some embodiments, a nanoparticle composition is provided in which the nanoparticle is gold. In some aspects, a nanoparticle composition is provided in which the nanoparticle is silver.","In some aspects, a method of associating a RNA polynucleotide to a nanoparticle is provided comprising the step of aging a mixture of a thiolated RNA polynucleotide duplex with the nanoparticle in a series of solutions to associate the RNA polynucleotide to the nanoparticle, each solution comprising an increasing concentration of sodium chloride (NaCl) relative to a previous solution beginning with a first solution comprising about 0.1 M NaCl. In related aspects, the method further comprises the step of sonicating the mixture after the last aging step. In some aspects, the method further comprises the step of isolating the nanoparticles.","In some embodiments, a method of associating a RNA to a nanoparticle is provided comprising:","(a) mixing a thiolated RNA duplex with the nanoparticle in a solution comprising about 0.1 M sodium chloride (NaCl);","(b) aging the mixture in a series of salt solutions, each comprising an increasing concentration of NaCl relative to a previous solution;","(c) sonicating the mixture; and","(d) purifying the conjugated nanoparticle.","In various aspects of the method, the series of salt solutions ranges from about 0.1 M to about 0.3 M NaCl.","Some aspects of the method further comprise passivating the surface of the nanoparticle with oligo(ethylene glycol) thiol (OEG).","In some embodiments, a method of regulating expression of a target polynucleotide is provided comprising the step of hybridizing the target polynucleotide with the domain of a nanoparticle composition of the disclosure to form a substrate site for a polypeptide. In some aspects, the hybridizing results in degradation of the target polynucleotide. In various aspects, the polypeptide is selected from the group consisting of RNase H, RNase D, RNase L, RNase III, Dicer, Argonaute, Argonaute2, and TRBP.","Further aspects of the invention will become apparent from the detailed description provided below. However, it should be understood that the following detailed description and examples, while indicating preferred embodiments of the invention, are given by way of illustration only since various changes and modifications within the spirit and scope of the invention will become apparent to those skilled in the art from this detailed description.","To date, no methods have been developed for utilizing polyvalent particles and their unusual properties to load and transport RNA across cell membranes. Indeed, one must develop synthetic routes and materials that overcome one of the most challenging problems associated with RNA, most notably its chemical instability.","The ability to specifically inhibit expression of a target gene by RNAi has obvious benefits. For example, many diseases arise from the abnormal expression of a particular gene or group of genes. RNAi could be used to inhibit the expression of the deleterious gene and therefore alleviate symptoms of a disease or even provide a cure. For example, genes contributing to a cancerous state or to viral replication could be inhibited. In addition, mutant genes causing dominant genetic diseases such as myotonic dystrophy could be inhibited. Inflammatory diseases such as arthritis could also be treated by inhibiting such genes as cyclooxygenase or cytokines. Examples of targeted organs would include without limitation the liver, pancreas, spleen, skin, brain, prostate, heart etc. In addition, RNAi could be used to generate animals that mimic true genetic \u201cknockout\u201d animals to study gene function. Further description of contemplated uses and targets is provided below.","Drug discovery could also be facilitated by RNA technology. The RNA approach for target validation provides a quicker and less expensive approach to screen potential drug targets. Information for drug targeting is gained not only by inhibiting a potential drug target but also by determining whether an inhibited protein, and therefore the pathway, has significant phenotypic effects. For example, inhibition of LDL receptor expression should raise plasma LDL levels and, therefore, suggest that up-regulation of the receptor would be of therapeutic benefit. Expression arrays can be used to determine the responsive effect of inhibition on the expression of genes other than the targeted gene or pathway [Sharp et al., Genes and Development 15: 485-490 (2001)]. It will place the gene product within functional pathways and networks (interacting pathways).","It is disclosed herein that gold nanoparticles functionalized with RNA oligonucleotides take advantage of the ensemble properties that result from the surface functionalization of oligonucleotides, to increase the stability and efficacy of the bound RNA, while retaining the ability to act in the catalytic RNA interference pathway.","The synthesis of polyvalent RNA-nanoparticle compositions (RNA-Au NPs for those aspects wherein the nanoparticle is gold) requires that all components are free of nucleases, such as RNase, which degrades RNA ligands and results in unstable Au NP interactions by exposing the Au surface (as evidenced by aggregation). While conditions for preparing RNase free organic components and solutions are well established, provided herein are methods for creating RNase free inorganic gold nanoparticles.","It is to be noted that the terms \u201cpolynucleotide\u201d and \u201coligonucleotide\u201d are used interchangeably herein. It is also to be noted that the terms \u201cconjugated\u201d and \u201cfunctionalized\u201d are also used interchangeably herein.","As used herein, a \u201csubstrate site\u201d is a location on a polynucleotide (which is single- or double-stranded) that is recognized and acted upon by a polypeptide. As used herein, \u201cacted upon\u201d is understood to mean any enzymatic function performed by a polypeptide that recognizes and binds to the substrate site.","As used herein, a \u201cpolypeptide interaction site\u201d refers to a site on a polynucleotide (which is single- or double-stranded) that is recognized by a polypeptide. Recognition of the interaction site by a polypeptide, in various aspects, results in cleavage of the polynucleotide. In certain embodiments, the polypeptide itself that recognizes the polypeptide interaction site acts on the polynucleotide and in other embodiments, the polypeptide that recognizes the polynucleotide recognition site directs activity of one or more additional polypeptides to act on the polynucleotide.","As used herein, the term \u201ctarget\u201d or \u201ctarget polynucleotide\u201d refers to a polynucleotide against which a given RNA polynucleotide can be directed.","As used herein, \u201cRNA\u201d refers to a molecule comprising at least one ribonucleotide residue.","As used herein, \u201cduplex\u201d refers to a region in two complementary or substantially complementary polynucleotides that form base pairs with one another, either by Watson-Crick base pairing or any other manner that allows for a stabilized duplex between polynucleotide strands that are complementary or substantially complementary. For example, a polynucleotide strand having 21 nucleotide units can base pair with another polynucleotide of 21 nucleotide units, yet only 19 bases on each strand are complementary or substantially complementary, such that the \u201cduplex\u201d has 19 base pairs. The remaining bases may, for example, exist as 5\u2032 and 3\u2032 overhangs. Further, within the duplex, 100% complementarity is not required; substantial complementarity is allowable within a duplex. Substantial complementarity refers to 75% or greater complementarity. For example, a mismatch in a duplex consisting of 19 base pairs results in 94.7% complementarity, rendering the duplex substantially complementary.","Nanoparticles are thus provided which are functionalized to have a polynucleotide attached thereto. The size, shape and chemical composition of the nanoparticles contribute to the properties of the resulting polynucleotide-functionalized nanoparticle. These properties include for example, optical properties, optoelectronic properties, electrochemical properties, electronic properties, stability in various solutions, magnetic properties, and pore and channel size variation. Mixtures of nanoparticles having different sizes, shapes and\/or chemical compositions, as well as the use of nanoparticles having uniform sizes, shapes and chemical composition, and therefore a mixture of properties are contemplated. Examples of suitable particles include, without limitation, aggregate particles, isotropic (such as spherical particles), anisotropic particles (such as non-spherical rods, tetrahedral, and\/or prisms) and core-shell particles, such as those described in U.S. Pat. No. 7,238,472 and International Publication No. WO 2003\/08539, the disclosures of which are incorporated by reference in their entirety.","In one embodiment, the nanoparticle is metallic, and in various aspects, the nanoparticle is a colloidal metal. Thus, in various embodiments, nanoparticles of the invention include metal (including for example and without limitation, silver, gold, platinum, aluminum, palladium, copper, cobalt, indium, nickel, or any other metal amenable to nanoparticle formation), semiconductor (including for example and without limitation, CdSe, CdS, and CdS or CdSe coated with ZnS) and magnetic (for example, ferromagnetite) colloidal materials.","Also, as described in U.S. Patent Publication No 2003\/0147966, nanoparticles of the invention include those that are available commercially, as well as those that are synthesized, e.g., produced from progressive nucleation in solution (e.g., by colloid reaction) or by various physical and chemical vapor deposition processes, such as sputter deposition. See, e.g., HaVashi, Vac. Sci. Technol. A5(4):1375-84 (1987); Hayashi, Physics Today, 44-60 (1987); MRS Bulletin, January 1990, 16-47. As further described in U.S. Patent Publication No 2003\/0147966, nanoparticles contemplated are alternatively produced using HAuCland a citrate-reducing agent, using methods known in the art. See, e.g., Marinakos et al., Adv. Mater. 11:34-37(1999); Marinakos et al., Chem. Mater. 10: 1214-19(1998); Enustun & Turkevich, J. Am. Chem. Soc. 85: 3317(1963).","Nanoparticles can range in size from about 1 nm to about 250 nm in mean diameter, about 1 nm to about 240 nm in mean diameter, about 1 nm to about 230 nm in mean diameter, about 1 nm to about 220 nm in mean diameter, about 1 nm to about 210 nm in mean diameter, about 1 nm to about 200 nm in mean diameter, about 1 nm to about 190 nm in mean diameter, about 1 nm to about 180 nm in mean diameter, about 1 nm to about 170 nm in mean diameter, about 1 nm to about 160 nm in mean diameter, about 1 nm to about 150 nm in mean diameter, about 1 nm to about 140 nm in mean diameter, about 1 nm to about 130 nm in mean diameter, about 1 nm to about 120 nm in mean diameter, about 1 nm to about 110 nm in mean diameter, about 1 nm to about 100 nm in mean diameter, about 1 nm to about 90 nm in mean diameter, about 1 nm to about 80 nm in mean diameter, about 1 nm to about 70 nm in mean diameter, about 1 nm to about 60 nm in mean diameter, about 1 nm to about 50 nm in mean diameter, about 1 nm to about 40 nm in mean diameter, about 1 nm to about 30 nm in mean diameter, or about 1 nm to about 20 nm in mean diameter, about 1 nm to about 10 nm in mean diameter. In other aspects, the size of the nanoparticles is from about 5 nm to about 150 nm (mean diameter), from about 5 to about 50 nm, from about 10 to about 30 nm, from about 10 to 150 nm, from about 10 to about 100 nm, or about 10 to about 50 nm. The size of the nanoparticles is from about 5 nm to about 150 nm (mean diameter), from about 30 to about 100 nm, from about 40 to about 80 nm. The size of the nanoparticles used in a method varies as required by their particular use or application. The variation of size is advantageously used to optimize certain physical characteristics of the nanoparticles, for example, optical properties or the amount of surface area that can be derivatized as described herein.","Nanoparticles with polynucleotides attached thereto are thus provided wherein a duplex RNA is associated with the nanoparticle. In some aspects, the RNA that is associated with a nanoparticle is a small interfering RNA (siRNA). Association of the duplex RNA to the nanoparticles is contemplated through various means.","According to methods described herein, citrate-stabilized gold nanoparticles are synthesized and the particles are treated with 0.1% diethylpyrocarbonate (DEPC) for 12 hours with stirring, then autoclaved at 121\u00b0 C. for 60 minutes. In some aspects, treatment of the nanoparticles with DEPC is performed for about 1 hour. In various aspects, treatment of the nanoparticles with DEPC is performed for about 1.5, about 2, about 2.5, about 3, about 3.5, about 4, about 4.5, about 5, about 5.5, about 6, about 6.5, about 7, about 7.5, about 8, about 8.5, about 9, about 9.5, about 10, about 10.5, about 11, about 11.5, about 12, about 12.5, about 13, about 13.5, about 14, about 14.5, about 15, about 20, about 25, about 30, about 3 days, about 7 days or more.","In one embodiment, a single strand RNA polynucleotide is attached directly to the nanoparticle, optionally through a spacer as described herein, In those aspects where a single strand RNA polynucleotide is attached to the nanoparticle, the RNA polynucleotide comprises two portions that are sufficiently complementary to allow the two portions to hybridize to each other under appropriate conditions to form a hairpin structure.","In another embodiment, a duplex RNA is immobilized directly on a nanoparticle, optionally through a spacer as described herein, such that only one strand of the duplex RNA is attached directly to the nanoparticle.","In other embodiments, the polynucleotide attached to a nanoparticle is DNA. When DNA is attached to the nanoparticle, the DNA is comprised of a sequence that is sufficiently complementary to a single strand region of a RNA duplex such that hybridization of the DNA polynucleotide attached to a nanoparticle and the single strand region of the RNA duplex takes place, thereby associating the duplex RNA to the nanoparticle. In one aspect, the single strand region of the duplex RNA is an overhanging end. The DNA in various aspects is single stranded or double-stranded, as long as the double-stranded molecule also include a single strand region that hybridizes to the single strand region of the duplex RNA.","Polynucleotide Orientation","Sense vs. Antisense","In some aspects, the strand of RNA that is attached to the nanoparticle is the \u201csense\u201d strand and the complementary strand of the duplex RNA is hybridized to the sense strand but is not attached to the nanoparticle. In other aspects, the strand of RNA that is attached to the nanoparticle is the \u201cantisense\u201d strand, and the complementary strand of the duplex RNA is hybridized to the antisense strand but is not attached to the nanoparticle. As used herein, a \u201csense\u201d strand is a strand that is identical to a target polynucleotide and an \u201cantisense\u201d strand is a strand that is complementary to a target polynucleotide. Attachment of the sense strand or the antisense strand to the nanoparticle determines one aspect of the orientation of the double-stranded RNA to the nanoparticle.","It is demonstrated herein that a RNA duplex wherein the sense strand is attached to a nanoparticle and an antisense strand is hybridized to the sense strand but is not attached to a nanoparticle has higher activity (see Example 5) than a RNA duplex in which the antisense strand is attached to a nanoparticle and the sense strand is hybridized to the antisense strand but is not attached to the nanoparticle. Without being bound by theory, the orientation of the attachment of a RNA duplex to a nanoparticle (for example and without limitation, whether a sense strand or an antisense strand of a RNA duplex is attached to a nanoparticle) is contemplated to be important for presenting a substrate for a polypeptide contemplated by the present disclosure. In some aspects, the polypeptide is Dicer. In some aspects, the polypeptide is Argonaute.","Position of Polypeptide Interaction Site","In some embodiments, the disclosure contemplates that a polynucleotide attached to a nanoparticle is RNA. It is also contemplated that a polynucleotide is attached to a nanoparticle such that a protein interaction site located in a sequence in the RNA is either proximal or distal relative to the nanoparticle. In these aspects, \u201cproximal\u201d and \u201cdistal\u201d refer to a midpoint on the polynucleotide. For example, if a polynucleotide that is attached to a nanoparticle is 20 bases in length, then the midpoint is at a location 10 bases from the nanoparticle, and a protein interaction site can either be proximal or distal relative to the tenth base.","Immobilizing a RNA polynucleotide on a nanoparticle using methods described herein allows for the control of access to the duplex by a polypeptide of the disclosure.","In some embodiments, spacer sequences of varying length are utilized to vary the number of and the distance between the RNA polynucleotides on a nanoparticle thus controlling the rates of target polynucleotide degradation. Without being bound by theory, one can control the rate of target polynucleotide degradation by immobilizing a RNA polynucleotide on a nanoparticle such that the protein interaction site is in a proximal position as described above. This aspect, combined with a surface density aspect as described below, can allow or prevent access by a polypeptide of the disclosure to the protein interaction site.","Spacers are described in further detail herein below.","Spacers","In certain aspects, functionalized nanoparticles are contemplated which include those wherein an oligonucleotide is attached to the nanoparticle through a spacer. \u201cSpacer\u201d as used herein means a moiety that does not participate in modulating gene expression per se but which serves to increase distance between the nanoparticle and the functional oligonucleotide, or to increase distance between individual oligonucleotides when attached to the nanoparticle in multiple copies. Thus, spacers are contemplated being located between individual oligonucleotide in tandem, whether the oligonucleotides have the same sequence or have different sequences. In one aspect, the spacer when present is an organic moiety. In another aspect, the spacer is a polymer, including but not limited to a water-soluble polymer, a nucleic acid, a polypeptide, an oligosaccharide, a carbohydrate, a lipid, an ethylglycol, or combinations thereof.","In certain aspects, the spacer has a polynucleotide covalently bound to it which can bind to the nanoparticles. These polynucleotides are the same polynucleotides as described above. As a result of the binding of the spacer to the nanoparticles, the polynucleotide is spaced away from the surface of the nanoparticles and is more accessible for hybridization with its target. In instances wherein the spacer is a polynucleotide, the length of the spacer in various embodiments at least about 10 nucleotides, 10-30 nucleotides, or even greater than 30 nucleotides. The spacer may have any sequence which does not interfere with the ability of the polynucleotides to become bound to the nanoparticles or to the target polynucleotide. The spacers should not have sequences complementary to each other or to that of the oligonucleotides, but may be all or in part complementary to the target polynucleotide. In certain aspects, the bases of the polynucleotide spacer are all adenines, all thymines, all cytidines, all guanines, all uracils, or all some other modified base.","Surface Density","Nanoparticles as provided herein have a packing density of the polynucleotides on the surface of the nanoparticle that is, in various aspects, sufficient to result in cooperative behavior between nanoparticles and between polynucleotide strands on a single nanoparticle. In another aspect, the cooperative behavior between the nanoparticles increases the resistance of the polynucleotide to nuclease degradation. In yet another aspect, the uptake of nanoparticles by a cell is influenced by the density of polynucleotides associated with the nanoparticle. As described in PCT\/US2008\/65366, incorporated herein by reference in its entirety, a higher density of polynucleotides on the surface of a nanoparticle is associated with an increased uptake of nanoparticles by a cell.","A surface density adequate to make the nanoparticles stable and the conditions necessary to obtain it for a desired combination of nanoparticles and polynucleotides can be determined empirically. Generally, a surface density of at least 2 pmoles\/cmwill be adequate to provide stable nanoparticle-oligonucleotide compositions. In some aspects, the surface density is at least 15 pmoles\/cm. Methods are also provided wherein the polynucleotide is bound to the nanoparticle at a surface density of at least 2 pmol\/cm, at least 3 pmol\/cm, at least 4 pmol\/cm, at least 5 pmol\/cm, at least 6 pmol\/cm, at least 7 pmol\/cm, at least 8 pmol\/cm, at least 9 pmol\/cm, at least 10 pmol\/cm, at least about 15 pmol\/cm, at least about 20 pmol\/cm, at least about 25 pmol\/cm, at least about 30 pmol\/cm, at least about 35 pmol\/cm, at least about 40 pmol\/cm, at least about 45 pmol\/cm, at least about 50 pmol\/cm, at least about 55 pmol\/cm, at least about 60 pmol\/cm, at least about 65 pmol\/cm, at least about 70 pmol\/cm, at least about 75 pmol\/cm, at least about 80 pmol\/cm, at least about 85 pmol\/cm, at least about 90 pmol\/cm, at least about 95 pmol\/cm, at least about 100 pmol\/cm, at least about 125 pmol\/cm, at least about 150 pmol\/cm, at least about 175 pmol\/cm, at least about 200 pmol\/cm, at least about 250 pmol\/cm, at least about 300 pmol\/cm, at least about 350 pmol\/cm, at least about 400 pmol\/cm, at least about 450 pmol\/cm, at least about 500 pmol\/cm, at least about 550 pmol\/cm, at least about 600 pmol\/cm, at least about 650 pmol\/cm, at least about 700 pmol\/cm, at least about 750 pmol\/cm, at least about 800 pmol\/cm, at least about 850 pmol\/cm, at least about 900 pmol\/cm, at least about 950 pmol\/cm, at least about 1000 pmol\/cmor more.","Density of polynucleotides on the surface of a nanoparticle has been shown to modulate specific polypeptide interactions with the polynucleotide on the surface and\/or with the nanoparticle itself. Under various conditions, some polypeptides may be prohibited from interacting with polynucleotides associated with a nanoparticle based on steric hindrance caused by the density of polynucleotides. In aspects where interaction of polynucleotides with polypeptides that are otherwise precludes by steric hindrance is desirable, the density of polynucleotides on the nanoparticle surface is decreased to allow the polypeptide to interact with the polynucleotide.","Polynucleotide surface density has also been shown to modulate stability of the polynucleotide associated with the nanoparticle. In one embodiment, an RNA polynucleotide associated with a nanoparticle is provided wherein the RNA polynucleotide has a half-life that is at least substantially the same as the half-life of an identical RNA polynucleotide that is not associated with a nanoparticle. In other embodiments, the RNA polynucleotide associated with the nanoparticle has a half-life that is about 5% greater, about 10% greater, about 20% greater, about 30% greater, about 40% greater, about 50% greater, about 60% greater, about 70% greater, about 80% greater, about 90% greater, about 2-fold greater, about 3-fold greater, about 4-fold greater, about 5-fold greater, about 6-fold greater, about 7-fold greater, about 8-fold greater, about 9-fold greater, about 10-fold greater, about 20-fold greater, about 30-fold greater, about 40-fold greater, about 50-fold greater, about 60-fold greater, about 70-fold greater, about 80-fold greater, about 90-fold greater, about 100-fold greater, about 200-fold greater, about 300-fold greater, about 400-fold greater, about 500-fold greater, about 600-fold greater, about 700-fold greater, about 800-fold greater, about 900-fold greater, about 1000-fold greater, about 5000-fold greater, about 10,000-fold greater, about 50,000-fold greater, about 100,000-fold greater, about 200,000-fold greater, about 300,000-fold greater, about 400,000-fold greater, about 500,000-fold greater, about 600,000-fold greater, about 700,000-fold greater, about 800,000-fold greater, about 900,000-fold greater, about 1,000,000-fold greater or more than the half-life of an identical RNA polynucleotide that is not associated with a nanoparticle.","Methods of Attaching Polynucleotides","Polynucleotides contemplated for use in the methods include those bound to the nanoparticle through any means. Regardless of the means by which the polynucleotide is attached to the nanoparticle, attachment in various aspects is effected through a 5\u2032 linkage, a 3\u2032 linkage, some type of internal linkage, or any combination of these attachments.","In one aspect, the nanoparticles, the polynucleotides or both are functionalized in order to attach the polynucleotides to the nanoparticles. Methods to functionalize nanoparticles and polynucleotides are known in the art. For instance, polynucleotides functionalized with alkanethiols at their 3\u2032-termini or 5\u2032-termini readily attach to gold nanoparticles. See Whitesides, Proceedings of the Robert A. Welch Foundation 39th Conference On Chemical Research Nanophase Chemistry, Houston, Tex., pages 109-121 (1995). See also, Mucic et al. [Chem. Commun. 555-557 (1996)] which describes a method of attaching 3\u2032 thiol DNA to flat gold surfaces. The alkanethiol method can also be used to attach oligonucleotides to other metal, semiconductor and magnetic colloids and to the other types of nanoparticles described herein. Other functional groups for attaching oligonucleotides to solid surfaces include phosphorothioate groups (see, for example, U.S. Pat. No. 5,472,881 for the binding of oligonucleotide-phosphorothioates to gold surfaces), substituted alkylsiloxanes [(see, for example, Burwell, Chemical Technology, 4, 370-377 (1974) and Matteucci and Caruthers, J. Am. Chem. Soc., 103, 3185-3191 (1981) for binding of oligonucleotides to silica and glass surfaces, and Grabar et al., [Anal. Chem., 67, 735-743] for binding of aminoalkylsiloxanes and for similar binding of mercaptoaklylsiloxanes]. Polynucleotides with a 5\u2032 thionucleoside or a 3\u2032 thionucleoside may also be used for attaching oligonucleotides to solid surfaces. The following references describe other methods which may be employed to attached oligonucleotides to nanoparticles: Nuzzo et al., J. Am. Chem. Soc., 109, 2358 (1987) (disulfides on gold); Allara and Nuzzo, Langmuir, 1, 45 (1985) (carboxylic acids on aluminum); Allara and Tompkins, J. Colloid Interface Sci., 49, 410-421 (1974) (carboxylic acids on copper); Iler, The Chemistry Of Silica, Chapter 6, (Wiley 1979) (carboxylic acids on silica); Timmons and Zisman, J. Phys. Chem., 69, 984-990 (1965) (carboxylic acids on platinum); Soriaga and Hubbard, J. Am. Chem. Soc., 104, 3937 (1982) (aromatic ring compounds on platinum); Hubbard, Acc. Chem. Res., 13, 177 (1980) (sulfolanes, sulfoxides and other functionalized solvents on platinum); Hickman et al., J. Am. Chem. Soc., 111, 7271 (1989) (isonitriles on platinum); Maoz and Sagiv, Langmuir, 3, 1045 (1987) (silanes on silica); Maoz and Sagiv, Langmuir, 3, 1034 (1987) (silanes on silica); Wasserman et al., Langmuir, 5, 1074 (1989) (silanes on silica); Eltekova and Eltekov, Langmuir, 3, 951 (1987) (aromatic carboxylic acids, aldehydes, alcohols and methoxy groups on titanium dioxide and silica); Lec et al., J. Phys. Chem., 92, 2597 (1988) (rigid phosphates on metals).","U.S. patent application Ser. Nos. 09\/760,500 and 09\/820,279 and international application nos. PCT\/US01\/01190 and PCT\/US01\/10071 describe oligonucleotides functionalized with a cyclic disulfide. The cyclic disulfides in certain aspects have 5 or 6 atoms in their rings, including the two sulfur atoms. Suitable cyclic disulfides are available commercially or are synthesized by known procedures. Functionalization with the reduced forms of the cyclic disulfides is also contemplated. Functionalization with triple cyclic disulfide anchoring groups are described in PCT\/US2008\/63441, incorporated herein by reference in its entirety.","In certain aspects wherein cyclic disulfide functionalization is utilized, oligonucleotides are attached to a nanoparticle through one or more linkers. In one embodiment, the linker comprises a hydrocarbon moiety attached to a cyclic disulfide. Suitable hydrocarbons are available commercially, and are attached to the cyclic disulfides. The hydrocarbon moiety is, in one aspect, a steroid residue. Oligonucleotide-nanoparticle compositions prepared using linkers comprising a steroid residue attached to a cyclic disulfide are more stable compared to compositions prepared using alkanethiols or acyclic disulfides as the linker, and in certain instances, the oligonucleotide-nanoparticle compositions have been found to be 300 times more stable. In certain embodiments the two sulfur atoms of the cyclic disulfide are close enough together so that both of the sulfur atoms attach simultaneously to the nanoparticle. In other aspects, the two sulfur atoms are adjacent each other. In aspects where utilized, the hydrocarbon moiety is large enough to present a hydrophobic surface screening the surfaces of the nanoparticle.","In other aspects, a method for attaching oligonucleotides onto a surface is based on an aging process described in U.S. application Ser. No. 09\/344,667, filed Jun. 25, 1999; Ser. No. 09\/603,830, filed Jun. 26, 2000; Ser. No. 09\/760,500, filed Jan. 12, 2001; Ser. No. 09\/820,279, filed Mar. 28, 2001; Ser. No. 09\/927,777, filed Aug. 10, 2001; and in International application nos. PCT\/US97\/12783, filed Jul. 21, 1997; PCT\/US00\/17507, filed Jun. 26, 2000; PCT\/US01\/01190, filed Jan. 12, 2001; PCT\/US01\/10071, filed Mar. 28, 2001, the disclosures which are incorporated by reference in their entirety. The aging process provides nanoparticle-oligonucleotide compositions with enhanced stability and selectivity. The process comprises providing oligonucleotides, in one aspect, having covalently bound thereto a moiety comprising a functional group which can bind to the nanoparticles. The moieties and functional groups are those that allow for binding (i.e., by chemisorption or covalent bonding) of the oligonucleotides to nanoparticles. For example, oligonucleotides having an alkanethiol, an alkanedisulfide or a cyclic disulfide covalently bound to their 5\u2032 or 3\u2032 ends bind the oligonucleotides to a variety of nanoparticles, including gold nanoparticles.","Compositions produced by use of the \u201caging\u201d step have been found to be considerably more stable than those produced without the \u201caging\u201d step. Increased density of the oligonucleotides on the surfaces of the nanoparticles is achieved by the \u201caging\u201d step. The surface density achieved by the \u201caging\u201d step will depend on the size and type of nanoparticles and on the length, sequence and concentration of the oligonucleotides. A surface density adequate to make the nanoparticles stable and the conditions necessary to obtain it for a desired combination of nanoparticles and oligonucleotides can be determined empirically. Generally, a surface density of at least 2 picomoles\/cmwill be adequate to provide stable nanoparticle-oligonucleotide compositions. Regardless, various oligonucleotide densities are contemplated as disclosed herein.","An \u201caging\u201d step is incorporated into production of functionalized nanoparticles following an initial binding or oligonucleotides to a nanoparticle. In brief, the oligonucleotides are contacted with the nanoparticles in water for a time sufficient to allow at least some of the oligonucleotides to bind to the nanoparticles by means of the functional groups. Such times can be determined empirically. In one aspect, a time of about 12-24 hours is contemplated. Other suitable conditions for binding of the oligonucleotides can also be determined empirically. For example, a concentration of about 10-20 nM nanoparticles and incubation at room temperature is contemplated.","Next, at least one salt is added to the water to form a salt solution. The salt is any water-soluble salt, including, for example and without limitation, sodium chloride, magnesium chloride, potassium chloride, ammonium chloride, sodium acetate, ammonium acetate, a combination of two or more of these salts, or one of these salts in phosphate buffer. The salt is added as a concentrated solution, or in the alternative as a solid. In various embodiments, the salt is added all at one time or the salt is added gradually over time. By \u201cgradually over time\u201d is meant that the salt is added in at least two portions at intervals spaced apart by a period of time. Suitable time intervals can be determined empirically.","The ionic strength of the salt solution must be sufficient to overcome at least partially the electrostatic repulsion of the oligonucleotides from each other and, either the electrostatic attraction of the negatively-charged oligonucleotides for positively-charged nanoparticles, or the electrostatic repulsion of the negatively-charged oligonucleotides from negatively-charged nanoparticles. Gradually reducing the electrostatic attraction and repulsion by adding the salt gradually over time gives the highest surface density of oligonucleotides on the nanoparticles. Suitable ionic strengths can be determined empirically for each salt or combination of salts. In one aspect, a final concentration of sodium chloride of from about 0.01 M to about 1.0 M in phosphate buffer is utilized, with the concentration of sodium chloride being increased gradually over time. In another aspect, a final concentration of sodium chloride of from about 0.01 M to about 0.5 M, or about 0.1 M to about 0.3 M is utilized, with the concentration of sodium chloride being increased gradually over time.","After adding the salt, the oligonucleotides and nanoparticles are incubated in the salt solution for a period of time to allow additional oligonucleotides to bind to the nanoparticles to produce the stable nanoparticle-oligonucleotide compositions. An increased surface density of the oligonucleotides on the nanoparticles stabilizes the compositions, as has been described herein. The time of this incubation can be determined empirically. By way of example, in one aspect a total incubation time of about 24-48, wherein the salt concentration is increased gradually over this total time, is contemplated. This second period of incubation in the salt solution is referred to herein as the \u201caging\u201d step. Other suitable conditions for this \u201caging\u201d step can also be determined empirically. By way of example, an aging step is carried out with incubation at room temperature and pH 7.0.","The compositions produced by use of the \u201caging\u201d are in general more stable than those produced without the \u201caging\u201d step. As noted above, this increased stability is due to the increased density of the oligonucleotides on the surfaces of the nanoparticles which is achieved by the \u201caging\u201d step. The surface density achieved by the \u201caging\u201d step will depend on the size and type of nanoparticles and on the length, sequence and concentration of the oligonucleotides.","As used herein, \u201cstable\u201d means that, for a period of at least six months after the compositions are made, a majority of the oligonucleotides remain attached to the nanoparticles and the oligonucleotides are able to hybridize with nucleic acid and oligonucleotide targets under standard conditions encountered in methods of detecting nucleic acid and methods of nanofabrication.","The term \u201cnucleotide\u201d or its plural as used herein is interchangeable with modified forms as discussed herein and otherwise known in the art. In certain instances, the art uses the term \u201cnucleobase\u201d which embraces naturally-occurring nucleotides as well as modifications of nucleotides that can be polymerized. Thus, nucleotide or nucleobase means the naturally occurring nucleobases adenine (A), guanine (G), cytosine (C), thymine (T) and uracil (U) as well as non-naturally occurring nucleobases such as xanthine, diaminopurine, 8-oxo-N6-methyladenine, 7-deazaxanthine, 7-deazaguanine, N4,N4-ethanocytosin, N\u2032,N\u2032-ethano-2,6-diaminopurine, 5-methylcytosine (mC), 5-(C-C)-alkynyl-cytosine, 5-fluorouracil, 5-bromouracil, pseudoisocytosine, 2-hydroxy-5-methyl-4-tr-iazolopyridin, isocytosine, isoguanine, inosine and the \u201cnon-naturally occurring\u201d nucleobases described in Benner et al., U.S. Pat. No. 5,432,272 and Susan M. Freier and Karl-Heinz Altmann, 1997, Nucleic Acids Research, vol. 25: pp 4429-4443. The term \u201cnucleobase\u201d also includes not only the known purine and pyrimidine heterocycles, but also heterocyclic analogues and tautomers thereof. Further naturally and non-naturally occurring nucleobases include those disclosed in U.S. Pat. No. 3,687,808 (Merigan, et al.), in Chapter 15 by Sanghvi, in Antisense Research and Application, Ed. S. T. Crooke and B. Lebleu, CRC Press, 1993, in Englisch et al., 1991, Angewandte Chemie, International Edition, 30: 613-722 (see especially pages 622 and 623, and in the Concise Encyclopedia of Polymer Science and Engineering, J. I. Kroschwitz Ed., John Wiley & Sons, 1990, pages 858-859, Cook, Anti-Cancer Drug Design 1991, 6, 585-607, each of which are hereby incorporated by reference in their entirety). In various aspects, polynucleotides also include one or more \u201cnucleosidic bases\u201d or \u201cbase units\u201d which include compounds such as heterocyclic compounds that can serve like nucleobases, including certain \u201cuniversal bases\u201d that are not nucleosidic bases in the most classical sense but serve as nucleosidic bases. Universal bases include 3-nitropyrrole, optionally substituted indoles (e.g., 5-nitroindole), and optionally substituted hypoxanthine. Other desirable universal bases include, pyrrole, diazole or triazole derivatives, including those universal bases known in the art.","Polynucleotides may also include modified nucleobases. A \u201cmodified base\u201d is understood in the art to be one that can pair with a natural base (e.g., adenine, guanine, cytosine, uracil, and\/or thymine) and\/or can pair with a non-naturally occurring base. Exemplary modified bases are described in EP 1 072 679 and WO 97\/12896, the disclosures of which are incorporated herein by reference. Modified nucleobases include without limitation, 5-methylcytosine (5-me-C), 5-hydroxymethyl cytosine, xanthine, hypoxanthine, 2-aminoadenine, 6-methyl and other alkyl derivatives of adenine and guanine, 2-propyl and other alkyl derivatives of adenine and guanine, 2-thiouracil, 2-thiothymine and 2-thiocytosine, 5-halouracil and cytosine, 5-propynyl uracil and cytosine and other alkynyl derivatives of pyrimidine bases, 6-azo uracil, cytosine and thymine, 5-uracil (pseudouracil), 4-thiouracil, 8-halo, 8-amino, 8-thiol, 8-thioalkyl, 8-hydroxyl and other 8-substituted adenines and guanines, 5-halo particularly 5-bromo, 5-trifluoromethyl and other 5-substituted uracils and cytosines, 7-methylguanine and 7-methyladenine, 2-F-adenine, 2-amino-adenine, 8-azaguanine and 8-azaadenine, 7-deazaguanine and 7-deazaadenine and 3-deazaguanine and 3-deazaadenine. Further modified bases include tricyclic pyrimidines such as phenoxazine cytidine(1H-pyrimido[5,4-b][1,4]benzoxazin-2(3H)-one), phenothiazine cytidine (1H-pyrimido[5,4-b][1,4]benzothiazin-2(3H)-one), G-clamps such as a substituted phenoxazine cytidine (e.g. 9-(2-aminoethoxy)-H-pyrimido[5,4-b][1,4]benzox-azin-2(3H)-one), carbazole cytidine (2H-pyrimido[4,5-b]indol-2-one), pyridoindole cytidine (H-pyrido[3\u2032,2\u2032:4,5]pyrrolo[2,3-d]pyrimidin-2-one). Modified bases may also include those in which the purine or pyrimidine base is replaced with other heterocycles, for example 7-deaza-adenine, 7-deazaguanosine, 2-aminopyridine and 2-pyridone. Additional nucleobases include those disclosed in U.S. Pat. No. 3,687,808, those disclosed in The Concise Encyclopedia Of Polymer Science And Engineering, pages 858-859, Kroschwitz, J. I., ed. John Wiley & Sons, 1990, those disclosed by Englisch et al., 1991, Angewandte Chemie, International Edition, 30: 613, and those disclosed by Sanghvi, Y. S., Chapter 15, Antisense Research and Applications, pages 289-302, Crooke, S. T. and Lebleu, B., ed., CRC Press, 1993. Certain of these bases are useful for increasing the binding affinity and include 5-substituted pyrimidines, 6-azapyrimidines and N-2, N-6 and O-6 substituted purines, including 2-aminopropyladenine, 5-propynyluracil and 5-propynylcytosine. 5-methylcytosine substitutions have been shown to increase nucleic acid duplex stability by 0.6-1.2\u00b0 C. and are, in certain aspects combined with 2\u2032-O-methoxyethyl sugar modifications. See, U.S. Pat. Nos. 3,687,808, 4,845,205; 5,130,302; 5,134,066; 5,175,273; 5,367,066; 5,432,272; 5,457,187; 5,459,255; 5,484,908; 5,502,177; 5,525,711; 5,552,540; 5,587,469; 5,594,121, 5,596,091; 5,614,617; 5,645,985; 5,830,653; 5,763,588; 6,005,096; 5,750,692 and 5,681,941, the disclosures of which are incorporated herein by reference.","Methods of making polynucleotides of a predetermined sequence are well-known. See, e.g., Sambrook et al., Molecular Cloning: A Laboratory Manual (2nd ed. 1989) and F. Eckstein (ed.) Oligonucleotides and Analogues, 1st Ed. (Oxford University Press, New York, 1991). Solid-phase synthesis methods are preferred for both polyribonucleotides and polydeoxyribonucleotides (the well-known methods of synthesizing DNA are also useful for synthesizing RNA). Polyribonucleotides can also be prepared enzymatically. Non-naturally occurring nucleobases can be incorporated into the polynucleotide, as well. See, e.g., U.S. Pat. No. 7,223,833; Katz, J. Am. Chem. Soc., 74:2238 (1951); Yamane, et al., J. Am. Chem. Soc., 83:2599 (1961); Kosturko, et al., Biochemistry, 13:3949 (1974); Thomas, J. Am. Chem. Soc., 76:6032 (1954); Zhang, et al., J. Am. Chem. Soc., 127:74-75 (2005); and Zimmermann, et al., J. Am. Chem. Soc., 124:13684-13685 (2002).","Nanoparticles provided that are functionalized with a polynucleotide, or modified form thereof, generally comprise a polynucleotide from about 5 nucleotides to about 100 nucleotides in length. More specifically, nanoparticles are functionalized with polynucleotide that are about 5 to about 90 nucleotides in length, about 5 to about 80 nucleotides in length, about 5 to about 70 nucleotides in length, about 5 to about 60 nucleotides in length, about 5 to about 50 nucleotides in length about 5 to about 45 nucleotides in length, about 5 to about 40 nucleotides in length, about 5 to about 35 nucleotides in length, about 5 to about 30 nucleotides in length, about 5 to about 25 nucleotides in length, about 5 to about 20 nucleotides in length, about 5 to about 15 nucleotides in length, about 5 to about 10 nucleotides in length, and all polynucleotides intermediate in length of the sizes specifically disclosed to the extent that the polynucleotide is able to achieve the desired result. Accordingly, polynucleotides of 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100 or more nucleotides in length are contemplated.","Polynucleotides contemplated for attachment to a nanoparticle include those which modulate expression of a gene product expressed from a target polynucleotide. Accordingly, RNA polynucleotides which hybridize to a target polynucleotide and initiate an RNAse activity (for example RNAse H), triple helix forming polynucleotides which hybridize to double-stranded polynucleotides and inhibit transcription, and ribozymes which hybridize to a target polynucleotide and inhibit translation, are contemplated.","In various aspects, if a specific mRNA is targeted, a single nanoparticle-binding agent composition has the ability to bind to multiple copies of the same transcript. In one aspect, a nanoparticle is provided that is functionalized with identical polynucleotides, i.e., each polynucleotide has the same length and the same sequence. In other aspects, the nanoparticle is functionalized with two or more polynucleotides which are not identical, i.e., at least one of the attached polynucleotides differ from at least one other attached polynucleotide in that it has a different length and\/or a different sequence. In aspects wherein different polynucleotides are attached to the nanoparticle, these different polynucleotides bind to the same single target polynucleotide but at different locations, or substrate sites, or bind to different target polynucleotides which encode different gene products. Accordingly, in various aspects, a single nanoparticle-binding agent composition target more than one gene product. Polynucleotides are thus target-specific polynucleotides, whether at one or more specific regions in the target polynucleotide, or over the entire length of the target polynucleotide as the need may be to effect a desired level of inhibition of gene expression.","The present disclosure provides, in various embodiments, polyvalent RNA-nanoparticle compositions that are useful for gene regulation. Small interfering RNAs are double stranded RNA agents that have complementarity to (i.e., are able to hybridize with) a portion of the target RNA (generally messenger RNA (mRNA)). Generally, such complementarity is 100%, but can be less if desired, such as about 20%, about 25%, about 30%, about 35%, about 40%, about 45%, about 50%, about 55%, about 60%, about 70%, about 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, or 99%. For example, 19 bases out of 21 bases may be base-paired. Thus, it will be understood that an oligonucleotide used in the methods need not be 100% complementary to a desired target nucleic acid to be specifically hybridizable. Moreover, oligonucleotides may hybridize to each other over one or more segments such that intervening or adjacent segments are not involved in the hybridization event (e.g., a loop structure or hairpin structure). Percent complementarity between any given oligonucleotide can be determined routinely using BLAST programs (Basic Local Alignment Search Tools) and PowerBLAST programs known in the art (Altschul et al., 1990, J. Mol. Biol., 215: 403-410; Zhang and Madden, 1997, Genome Res., 7: 649-656).","In some aspects, where selection between various allelic variants is desired, 100% complementarity to the target gene is required in order to effectively discern the target sequence from the other allelic sequence. When selecting between allelic targets, choice of length is also an important factor because it is the other factor involved in the percent complementary and the ability to differentiate between allelic differences.","\u201cHybridization\u201d means an interaction between two or three strands of nucleic acids by hydrogen bonds in accordance with the rules of Watson-Crick DNA complementarity, Hoogstein binding, or other sequence-specific binding known in the art. Hybridization can be performed under different stringency conditions known in the art.","The term \u201cRNA\u201d includes duplexes of two separate strands, as well as single, triple, or quadruple stranded structures. An example of a single strand RNA is an RNA with a hairpin loop. The RNA sequence needs to be of sufficient length to bring the small interfering RNA and target RNA together through complementary base-pairing interactions. The RNA useful in the methods disclosed herein may be of varying lengths. The RNA, as disclosed herein, comprises a domain in a single strand of the duplex sufficiently complementary to a sequence in a target polynucleotide to permit hybridization of the single strand to the target polynucleotide under appropriate conditions, and hybridization of the domain of the duplex to the sequence in the target polynucleotide creates a substrate site recognized by a polypeptide. The length of this domain is generally greater than or equal to ten nucleotides and of sufficient length to hybridize with the target RNA; specifically 10-100 nucleotides; more specifically any integer between 10 and 80 nucleotides, such as 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, and 100. By \u201csufficient length\u201d is meant an oligonucleotide of greater than or equal to 10 nucleotides that is of a length great enough to provide the intended function under the expected condition.","RNA may be polymerized in vitro, recombinant RNA, contain chimeric sequences, or derivatives of these groups. The RNA contains, in various embodiments, ribonucleotides, deoxyribonucleotides, synthetic nucleotides, or any suitable combination such that expression of the target gene is inhibited.","A delivered RNA can stay within the cytoplasm or nucleus. The RNA can be delivered to a cell to inhibit expression an endogenous or exogenous nucleotide sequence or to affect a specific physiological characteristic not naturally associated with the cell.","A RNA can be delivered to a cell in order to produce a cellular change that is therapeutic. The delivery of RNA or other genetic material for therapeutic purposes (the art of improving health in an animal including treatment or prevention of disease) is called gene therapy. The RNA can be delivered either directly to the organism in situ or indirectly by transfer to a cell ex vivo that is then transplanted into the organism. Entry into the cell is required for the RNA to block the production of a protein or to decrease the amount of a RNA. Polynucleotide sequences contemplated by the present disclosure are further described below.","In some aspects, the disclosure provides methods of targeting specific nucleic acids. Any type of nucleic acid may be targeted, and the methods may be used, e.g., for therapeutic modulation of gene expression (See, e.g., PCT\/US2006\/022325, the disclosure of which is incorporated herein by reference). Examples of nucleic acids that can be targeted by the methods of the invention include but are not limited to genes (e.g., a gene associated with a particular disease), viral RNA, or mRNA, RNA, single-stranded nucleic acids.","The target nucleic acid may be in cells, tissue samples, or biological fluids, as also known in the art. See, e.g., Sambrook et al., Molecular Cloning: A Laboratory Manual (2nd ed. 1989) and B. D. Hames and S. J. Higgins, Eds., Gene Probes 1 (IRL Press, New York, 1995).","The terms \u201cstart codon region\u201d and \u201ctranslation initiation codon region\u201d refer to a portion of a mRNA or gene that encompasses contiguous nucleotides in either direction (i.e., 5\u2032 or 3\u2032) from a translation initiation codon. Similarly, the terms \u201cstop codon region\u201d and \u201ctranslation termination codon region\u201d refer to a portion of such a mRNA or gene that encompasses contiguous nucleotides in either direction (i.e., 5\u2032 or 3\u2032) from a translation termination codon. Consequently, the \u201cstart codon region\u201d (or \u201ctranslation initiation codon region\u201d) and the \u201cstop codon region\u201d (or \u201ctranslation termination codon region\u201d) are all regions which may be targeted effectively with the oligonucleotides on the functionalized nanoparticles.","Other target regions include the 5\u2032 untranslated region (5\u2032UTR), the portion of an mRNA in the 5\u2032 direction from the translation initiation codon, including nucleotides between the 5\u2032 cap site and the translation initiation codon of a mRNA (or corresponding nucleotides on the gene), and the 3\u2032 untranslated region (3\u2032UTR), the portion of a mRNA in the 3\u2032 direction from the translation termination codon, including nucleotides between the translation termination codon and 3\u2032 end of a mRNA (or corresponding nucleotides on the gene). The 5\u2032 cap site of a mRNA comprises an N7-methylated guanosine residue joined to the 5\u2032-most residue of the mRNA via a 5\u2032-5\u2032 triphosphate linkage. The 5\u2032 cap region of a mRNA is considered to include the 5\u2032 cap structure itself as well as the first 50 nucleotides adjacent to the cap site.","For prokaryotic target nucleic acid, in various aspects, the nucleic acid is RNA transcribed from genomic DNA. For eukaryotic target nucleic acid, the nucleic acid is an animal nucleic acid, a plant nucleic acid, a fungal nucleic acid, including yeast nucleic acid. As above, the target nucleic acid is a RNA transcribed from a genomic DNA sequence. In certain aspects, the target nucleic acid is a mitochondrial nucleic acid. For viral target nucleic acid, the nucleic acid is viral genomic RNA, or RNA transcribed from viral genomic DNA.","Methods for inhibiting gene product expression provided include those wherein expression of the target gene product is inhibited by at least about 5%, at least about 10%, at least about 15%, at least about 20%, at least about 25%, at least about 30%, at least about 35%, at least about 40%, at least about 45%, at least about 50%, at least about 55%, at least about 60%, at least about 65%, at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 96%, at least about 97%, at least about 98%, at least about 99%, or 100% compared to gene product expression in the absence of an oligonucleotide-functionalized nanoparticle. In other words, methods provided embrace those which results in essentially any degree of inhibition of expression of a target gene product.","The degree of inhibition is determined in vivo from a body fluid sample or from a biopsy sample or by imaging techniques well known in the art. Alternatively, the degree of inhibition is determined in a cell culture assay, generally as a predictable measure of a degree of inhibition that can be expected in vivo resulting from use of a specific type of nanoparticle and a specific oligonucleotide.","Modified polynucleotides are contemplated for functionalizing nanoparticles wherein both one or more sugar and\/or one or more internucleotide linkage of the nucleotide units in the polynucleotide is replaced with \u201cnon-naturally occurring\u201d groups. In one aspect, this embodiment contemplates a peptide nucleic acid (PNA). In PNA compounds, the sugar-backbone of a polynucleotide is replaced with an amide containing backbone. See, for example U.S. Pat. Nos. 5,539,082; 5,714,331; and 5,719,262, and Nielsen et al., Science, 1991, 254, 1497-1500, the disclosures of which are herein incorporated by reference.","Other linkages between nucleotides and unnatural nucleotides contemplated for the disclosed polynucleotides include those described in U.S. Pat. Nos. 4,981,957; 5,118,800; 5,319,080; 5,359,044; 5,393,878; 5,446,137; 5,466,786; 5,514,785; 5,519,134; 5,567,811; 5,576,427; 5,591,722; 5,597,909; 5,610,300; 5,627,053; 5,639,873; 5,646,265; 5,658,873; 5,670,633; 5,792,747; and 5,700,920; U.S. Patent Publication No. 20040219565; International Patent Publication Nos. WO 98\/39352 and WO 99\/14226; Mesmaeker et. al., Current Opinion in Structural Biology 5:343-355 (1995) and Susan M. Freier and Karl-Heinz Altmann, Nucleic Acids Research, 25:4429-4443 (1997), the disclosures of which are incorporated herein by reference.","Specific examples of polynucleotides include those containing modified backbones or non-natural internucleoside linkages. Polynucleotides having modified backbones include those that retain a phosphorus atom in the backbone and those that do not have a phosphorus atom in the backbone. Modified polynucleotides that do not have a phosphorus atom in their internucleoside backbone are considered to be within the meaning of \u201cpolynucleotide.\u201d","Modified polynucleotide backbones containing a phosphorus atom include, for example, phosphorothioates, chiral phosphorothioates, phosphorodithioates, phosphotriesters, aminoalkylphosphotriesters, methyl and other alkyl phosphonates including 3\u2032-alkylene phosphonates, 5\u2032-alkylene phosphonates and chiral phosphonates, phosphinates, phosphoramidates including 3\u2032-amino phosphoramidate and aminoalkylphosphoramidates, thionophosphoramidates, thionoalkylphosphonates, thionoalkylphosphotriesters, selenophosphates and boranophosphates having normal 3\u2032-5\u2032 linkages, 2\u2032-5\u2032 linked analogs of these, and those having inverted polarity wherein one or more internucleotide linkages is a 3\u2032 to 3\u2032, 5\u2032 to 5\u2032 or 2\u2032 to 2\u2032 linkage. Also contemplated are polynucleotides having inverted polarity comprising a single 3\u2032 to 3\u2032 linkage at the 3\u2032-most internucleotide linkage, i.e. a single inverted nucleoside residue which may be abasic (the nucleotide is missing or has a hydroxyl group in place thereof). Salts, mixed salts and free acid forms are also contemplated.","Representative United States patents that teach the preparation of the above phosphorus-containing linkages include, U.S. Pat. Nos. 3,687,808; 4,469,863; 4,476,301; 5,023,243; 5,177,196; 5,188,897; 5,264,423; 5,276,019; 5,278,302; 5,286,717; 5,321,131; 5,399,676; 5,405,939; 5,453,496; 5,455,233; 5,466,677; 5,476,925; 5,519,126; 5,536,821; 5,541,306; 5,550,111; 5,563,253; 5,571,799; 5,587,361; 5,194,599; 5,565,555; 5,527,899; 5,721,218; 5,672,697 and 5,625,050, the disclosures of which are incorporated by reference herein.","Modified polynucleotide backbones that do not include a phosphorus atom have backbones that are formed by short chain alkyl or cycloalkyl internucleoside linkages, mixed heteroatom and alkyl or cycloalkyl internucleoside linkages, or one or more short chain heteroatomic or heterocyclic internucleoside linkages. These include those having morpholino linkages; siloxane backbones; sulfide, sulfoxide and sulfone backbones; formacetyl and thioformacetyl backbones; methylene formacetyl and thioformacetyl backbones; riboacetyl backbones; alkene containing backbones; sulfamate backbones; methyleneimino and methylenehydrazino backbones; sulfonate and sulfonamide backbones; amide backbones; and others having mixed N, O, S and CHcomponent parts. In still other embodiments, polynucleotides are provided with phosphorothioate backbones and oligonucleosides with heteroatom backbones, and including \u2014CH\u2014NH\u2014O\u2014CH\u2014, \u2014CH\u2014N(CH)\u2014O\u2014CH\u2014, \u2014CH\u2014O\u2014N(CH)\u2014CH\u2014, \u2014CH\u2014N(CH)\u2014N(CH)\u2014CH\u2014 and \u2014O\u2014N(CH)\u2014CH\u2014CH\u2014 described in U.S. Pat. Nos. 5,489,677, and 5,602,240. See, for example, U.S. Pat. Nos. 5,034,506; 5,166,315; 5,185,444; 5,214,134; 5,216,141; 5,235,033; 5,264,562; 5,264,564; 5,405,938; 5,434,257; 5,466,677; 5,470,967; 5,489,677; 5,541,307; 5,561,225; 5,596,086; 5,602,240; 5,610,289; 5,602,240; 5,608,046; 5,610,289; 5,618,704; 5,623,070; 5,663,312; 5,633,360; 5,677,437; 5,792,608; 5,646,269 and 5,677,439, the disclosures of which are incorporated herein by reference in their entireties.","In various forms, the linkage between two successive monomers in the oligo consists of 2 to 4, desirably 3, groups\/atoms selected from \u2014CH\u2014, \u2014O\u2014, \u2014S\u2014, \u2014NRH\u2014, >C\u2550O, >C\u2550NRH, >C\u2550S, \u2014Si(R\u2033)\u2014, \u2014SO\u2014, \u2014S(O)\u2014, \u2014P(O)\u2014, \u2014PO(BH)\u2014, \u2014P(O,S)\u2014, \u2014P(S)\u2014, \u2014PO(R\u2033)\u2014, \u2014PO(OCH)\u2014, and \u2014PO(NHRH)\u2014, where RH is selected from hydrogen and C1-4-alkyl, and R\u2033 is selected from C1-6-alkyl and phenyl. Illustrative examples of such linkages are \u2014CH\u2014CH\u2014CH\u2014, \u2014CH\u2014CO\u2014CH\u2014, \u2014CH\u2014CHOH\u2014CH\u2014, \u2014O\u2014CH2-O\u2014, \u2014O\u2014CH2-CH2-, \u2014O\u2014CH2-CH=(including R5 when used as a linkage to a succeeding monomer), \u2014CH\u2014CH\u2014O\u2014, \u2014NRH\u2014CH\u2014CH\u2014, \u2014CH\u2014CH\u2014NRH\u2014, \u2014CH\u2014NRH\u2014CH\u2014, \u2014O\u2014CH\u2014CH\u2014NRH\u2014, \u2014NRH\u2014CO\u2014O\u2014, \u2014NRH\u2014CO\u2014NRH\u2014, \u2014NRH\u2014CS\u2014NRH\u2014, \u2014NRH\u2014C(\u2550NRH)\u2014NRH\u2014, \u2014NRH\u2014CO\u2014CH\u2014NRH\u2014O\u2014CO\u2014O\u2014, \u2014O\u2014CO\u2014CH\u2014O\u2014, \u2014O\u2014CH\u2014CO\u2014O\u2014, \u2014CH\u2014CO\u2014NRH\u2014, \u2014O\u2014CO\u2014NRH\u2014, \u2014NRH\u2014CO\u2014CH\u2014, \u2014O\u2014CH\u2014CO\u2014NRH\u2014, \u2014O\u2014CH\u2014CH\u2014NRH\u2014, \u2014CH\u2550N\u2014O\u2014, \u2014CH\u2014NRH\u2014O\u2014, \u2014CH\u2014O\u2014N=(including R5 when used as a linkage to a succeeding monomer), \u2014CH\u2014O\u2014NRH\u2014, \u2014CO\u2014NRH\u2014CH\u2014, \u2014CH\u2014NRH\u2014O\u2014, \u2014CH\u2014NRH\u2014CO\u2014, \u2014O\u2014NRH\u2014CH\u2014, \u2014O\u2014NRH, \u2014O\u2014CH\u2014S\u2014, \u2014S\u2014CH\u2014O\u2014, \u2014CH\u2014CH\u2014S\u2014, \u2014O\u2014CH\u2014CH\u2014S\u2014, \u2014S\u2014CH\u2014CH\u2550(including R5 when used as a linkage to a succeeding monomer), \u2014S\u2014CH\u2014CH\u2014, \u2014S\u2014CH\u2014CH\u2014O\u2014, \u2014S\u2014CH\u2014CH\u2014S\u2014, \u2014CH\u2014S\u2014CH\u2014, \u2014CHSO\u2014CH\u2014, \u2014CH\u2014SO\u2014CH\u2014, \u2014O\u2014SO\u2014O\u2014, \u2014O\u2014S(O)\u2014O\u2014, \u2014O\u2014S(O)\u2014CH\u2014, \u2014O\u2014S(O)\u2014NRH\u2014, \u2014NRH\u2014S(O)\u2014CH\u2014; \u2014O\u2014S(O)\u2014CH\u2014, \u2014O\u2014P(O)\u2014O\u2014, \u2014O\u2014P(O,S)\u2014O\u2014, \u2014O\u2014P(S)\u2014O\u2014, \u2014S\u2014P(O)\u2014O\u2014, \u2014S\u2014P(O,S)\u2014O\u2014, \u2014S\u2014P(S)\u2014O\u2014, \u2014O\u2014P(O)\u2014S\u2014, \u2014O\u2014P(O,S)\u2014S\u2014, \u2014O\u2014P(S)\u2014S\u2014, \u2014S\u2014P(O)\u2014S\u2014, \u2014S\u2014P(O,S)\u2014S\u2014, \u2014S\u2014P(S)\u2014S\u2014, \u2014O\u2014PO(R\u2033)\u2014O\u2014, \u2014O\u2014PO(OCH)\u2014O\u2014, \u2014O\u2014PO(OCHCH)\u2014O\u2014, \u2014O\u2014PO(OCHCHS\u2014R)\u2014O\u2014, \u2014O\u2014PO(BH)\u2014O\u2014, \u2014O\u2014PO(NHRN)\u2014O\u2014, \u2014O\u2014P(O)\u2014NRHH\u2014, \u2014NRH\u2014P(O)\u2014O\u2014, \u2014O\u2014P(O,NRH)\u2014O\u2014, \u2014CH\u2014P(O)\u2014O\u2014, \u2014O\u2014P(O)\u2014CH\u2014, and \u2014O\u2014Si(R\u2033)\u2014O\u2014; among which \u2014CH\u2014CO\u2014NRH\u2014, \u2014CH\u2014NRH\u2014O\u2014, \u2014S\u2014CH\u2014O\u2014, \u2014O\u2014P(O)\u2014O\u2014O\u2014P(\u2014O,S)\u2014O\u2014, \u2014O\u2014P(S)\u2014O\u2014, \u2014NRHP(O)\u2014O\u2014, \u2014O\u2014P(O,NRH)\u2014O\u2014, \u2014O\u2014PO(R\u2033)\u2014O\u2014, \u2014O\u2014PO(CH)\u2014O\u2014, and \u2014O\u2014PO(NHRN)\u2014O\u2014, where RH is selected form hydrogen and C1-4-alkyl, and R\u2033 is selected from C1-6-alkyl and phenyl, are contemplated. Further illustrative examples are given in Mesmaeker et. al., 1995, Current Opinion in Structural Biology, 5: 343-355 and Susan M. Freier and Karl-Heinz Altmann, 1997, Nucleic Acids Research, vol 25: pp 4429-4443.","Still other modified forms of polynucleotides are described in detail in U.S. Patent Application No. 20040219565, the disclosure of which is incorporated by reference herein in its entirety.","Modified polynucleotides may also contain one or more substituted sugar moieties. In certain aspects, polynucleotides comprise one of the following at the 2\u2032 position: OH; F; O\u2014, S\u2014, or N-alkyl; O\u2014, S\u2014, or N-alkenyl; O\u2014, S\u2014 or N-alkynyl; or O-alkyl-O-alkyl, wherein the alkyl, alkenyl and alkynyl may be substituted or unsubstituted Cto Calkyl or Cto Calkenyl and alkynyl. Other embodiments include O[(CH)O]CH, O(CH2)OCH, O(CH)NH, O(CH)CH, O(CH)ONH, and O(CH)ON[(CH)CH], where n and m are from 1 to about 10. Other polynucleotides comprise one of the following at the 2\u2032 position: C1 to C10 lower alkyl, substituted lower alkyl, alkenyl, alkynyl, alkaryl, aralkyl, O-alkaryl or O-aralkyl, SH, SCH, OCN, Cl, Br, CN, CF, OCF, SOCH, SOCH, ONO, NO, N, NH, heterocycloalkyl, heterocycloalkaryl, aminoalkylamino, polyalkylamino, substituted silyl, an RNA cleaving group, a reporter group, an intercalator, a group for improving the pharmacokinetic properties of a polynucleotide, or a group for improving the pharmacodynamic properties of a polynucleotide, and other substituents having similar properties. In one aspect, a modification includes 2\u2032-methoxyethoxy (2\u2032-O\u2014CHCHOCH, also known as 2\u2032-O-(2-methoxyethyl) or 2\u2032-MOE) (Martin et al., 1995, Helv. Chim. Acta, 78: 486-504) i.e., an alkoxyalkoxy group. Other modifications include 2\u2032-dimethylaminooxyethoxy, i.e., a O(CH)ON(CH)group, also known as 2\u2032-DMAOE, and 2\u2032-dimethylaminoethoxyethoxy (also known in the art as 2\u2032-O-dimethyl-amino-ethoxy-ethyl or 2\u2032-DMAEOE), i.e., 2\u2032-O\u2014CH\u2014O\u2014CH\u2014N(CH).","Still other modifications include 2\u2032-methoxy (2\u2032-O\u2014CH), 2\u2032-aminopropoxy (2\u2032-OCHCHCHNH), 2\u2032-allyl (2\u2032-CH\u2014CH\u2550CH), 2\u2032-O-allyl (2\u2032-O\u2014CH\u2014CH\u2550CH) and 2\u2032-fluoro (2\u2032-F). The 2\u2032-modification may be in the arabino (up) position or ribo (down) position. In one aspect, a 2\u2032-arabino modification is 2\u2032-F. Similar modifications may also be made at other positions on the polynucleotide, for example, at the 3\u2032 position of the sugar on the 3\u2032 terminal nucleotide or in 2\u2032-5\u2032 linked polynucleotides and the 5\u2032 position of 5\u2032 terminal nucleotide. Polynucleotides may also have sugar mimetics such as cyclobutyl moieties in place of the pentofuranosyl sugar. See, for example, U.S. Pat. Nos. 4,981,957; 5,118,800; 5,319,080; 5,359,044; 5,393,878; 5,446,137; 5,466,786; 5,514,785; 5,519,134; 5,567,811; 5,576,427; 5,591,722; 5,597,909; 5,610,300; 5,627,053; 5,639,873; 5,646,265; 5,658,873; 5,670,633; 5,792,747; and 5,700,920, the disclosures of which are incorporated by reference in their entireties herein.","In one aspect, a modification of the sugar includes Locked Nucleic Acids (LNAs) in which the 2\u2032-hydroxyl group is linked to the 3\u2032 or 4\u2032 carbon atom of the sugar ring, thereby forming a bicyclic sugar moiety. The linkage is in certain aspects a methylene (\u2014CH\u2014)n group bridging the 2\u2032 oxygen atom and the 4\u2032 carbon atom wherein n is 1 or 2. LNAs and preparation thereof are described in WO 98\/39352 and WO 99\/14226, the disclosures of which are incorporated herein by reference.","It is not necessary for all positions in a given compound to be uniformly modified, and in fact more than one of the aforementioned modifications may be incorporated in a single compound or even at a single nucleoside within an oligonucleotide. These \u201cchimeric\u201d antisense compounds typically contain at least one region including a modification as described herein, while the remainder of the oligonucleotide remains \u201cunmodified.\u201d","In certain aspects, the modification confers increased resistance to nuclease degradation, increased cellular uptake, increased stability and\/or increased binding affinity for the target nucleic acid. In other aspects the modification serves as a substrate for enzymes capable of cleaving RNA:DNA or RNA:RNA hybrids. By way of example, RNAse H is a cellular endonuclease which cleaves the RNA strand of an RNA:DNA duplex. Activation of RNase H, therefore, results in cleavage of the RNA target, thereby greatly enhancing the efficiency of oligonucleotide-mediated inhibition of gene expression. The cleavage of RNA:RNA hybrids can, in like fashion, be accomplished through the actions of endoribonucleases, such as RNAseL which cleaves both cellular and viral RNA. Cleavage of the RNA target can be routinely detected by gel electrophoresis and, if necessary, associated nucleic acid hybridization techniques known in the art.","Chimeric compounds may be formed as composite structures of two or more oligonucleotides, modified oligonucleotides, oligonucleosides and\/or oligonucleotide mimetics as described above. Such compounds have also been referred to in the art as hybrids or gapmers. See, for example, U.S. Pat. Nos. 5,013,830; 5,149,797; 5,220,007; 5,256,775; 5,366,878; 5,403,711; 5,491,133; 5,565,350; 5,623,065; 5,652,355; 5,652,356; and 5,700,922, the disclosures of which are incorporated herein by reference in their entireties.","Citrate-stabilized gold nanoparticles (13 nm) were prepared using published procedures [Frens, Nature Physical Science 241: 20-22 (1973)]. Following synthesis, particles were treated with 0.1% diethylpyrocarbonate (DEPC) for 12 hours with stiffing, then autoclaved at 121\u00b0 C. for 60 minutes. Importantly and quite surprisingly, the optical and physical properties of the nanoparticles are unaffected by this relatively extreme treatment, as measured by UV spectroscopy and transmission electron microscopy TEM analysis (). Subsequent ligand functionalization also was not affected by this treatment. Testing of RNase activity in these solutions using an RNaseAlert kit (Ambion) showed no detectable RNase activity as compared to controls or untreated particles ().","The resultant RNase-free nanoparticles were amenable to further modification by thiolated oligonucleotides using published procedures [Demers et al., Anal. Chem. 72: 5535 (2000)]. Without this pre-treatment, subsequent functionalization with RNA could not be accomplished, presumably due to rapid degradation of the RNA-based surface capping ligands. Duplexes composed of a 27-base RNA strand, and 25-base complement terminated with an ethylene glycol spacer and alkylthiol, were hybridized and added to the RNase free Au NPs, where they were allowed to chemisorb via the thiol-gold bond. For this work, the sequences were designed to target the firefly luciferase gene.","RNA oligonucleotides were synthesized using TOM-RNA reagents (Glen Research) and a MerMade 6 (Bioautomation) or commercially made (Integrated DNA Technologies). The oligonucleotides synthesized by non-commercial sources were purified using TOP-RNA cartridges (Varian). The sequences used for this study were: luciferase sense, 5\u2032-Phosphate rCrGrA rCrUrU rCrGrU rGrCrC rArGrA rGrUrC rUrUrU rCrGAC Spacer 18 Spacer 18-Thiol-3\u2032 (SEQ ID NO: 1), luciferase antisense, 5\u2032-rGrUrC rGrArA rArGrA rCrUrC rUrGrG rCrArC rGrArA rGrUrC rGrUrA-3\u2032 (SEQ ID NO: 2), Cy3 luciferase, 5\u2032-Cy3 rGrUrC rGrArA rArGrA rCrUrC rUrGrG rCrArC rGrArA rGrUrC rGrUrA-3\u2032 (SEQ ID NO: 3), Cy5 luciferase, 5\u2032-Cy5 rGrUrC rGrArA rArGrA rCrUrC rUrGrG rCrArC rGrArA rGrUrC rGrUrA-3\u2032 (SEQ ID NO: 4), luciferase dabcyl, 5\u2032-rCrGrA rCrUrU rCrGrU rGrCrC rArGrA rGrUrC rUrUrU rCrGA C-dabcyl-3\u2032 (SEQ ID NO: 5), Renilla luciferase sense, 5\u2032-Phosphate rGrGrA rGrGrA rCrGrC rUrCrC rArGrAr UrGrA rArArU rGrGGT Spacer 18 Spacer 18-Thiol-3\u2032 (SEQ ID NO: 6), Renilla luciferase antisense, 5\u2032 rArCrC rCrArU rUrUrC rArUrC rUrGrG rArGrC rGrUrC rCrUrG-3\u2032 (SEQ ID NO: 7).","Pre-formed, thiolated RNA duplexes (1000 nM) were incubated with RNase-free solution of Au NPs (10 nM) which had been adjusted with 0.1 M NaCl. The mixture was aged in solutions of increasing salt concentration (from 0.1 to 0.3 M NaCl) and sonicated to increase the coverage of oligoribonucleotides on the surface. Oligoethylene glycol (OEG) was added (30 \u03bcmol\/mL final concentration) 24 hrs after duplex addition and prevented particle precipitation in cell culture (). Addition at this concentration did not change duplex loading. After functionalization, the particles were purified by centrifugation at (13,000 rpm, 20 mins) at 4\u00b0 C., and resuspended in sterile phosphate buffered saline (PBS: 137 mM NaCl, 10 mM Phosphate, 2.7 mM KCl, pH 7.4). This process was repeated three times. Refrigerated centrifugation was necessary to prevent duplex dehybridization due to heating caused by the centrifugation process. To determine dsRNA loading, antisense strands were labeled with Cyanine 3 (Cy3) fluorescent dye. Fluorescence was measured using a Jobin Yvon Fluorolog FL3-22 with excitation at 550 nm and measuring emission from 560 to 620 nm in 1 nm increments and compared to a standard curve. The number of duplexes per particle was calculated by oxidizing the gold using 25 \u03bcM potassium cyanide, measuring the number of fluorescent antisense strands (indicative of formed duplexes), and dividing by the concentration of nanoparticles. Each RNA-gold nanoparticle composition contained 33\u00b14 RNA duplexes per 13 nm Au NP.","To prevent dehybridization of the RNA duplex, the salt concentration of the Au NP solution was adjusted to 0.1 M with NaCl prior to duplex addition. Subsequently, the salt concentration was increased from 0.1 to 0.3 M NaCl over 12 hours with brief ultrasonication following each addition to increase the coverage of oligonucleotides on the nanoparticle surface [Hurst et al., Anal Chem 78: 8313 (2006)]. To yield a more stable composition, RNA-functionalized particles were treated with 30 \u03bcmol\/mL of oligoethylene glycol-thiol (OEG-Thiol) as an additional surface passivating ligand (Scheme 1). OEG-Thiol passivation was found to stabilize these nanomaterials under cell culture conditions for extended periods of time.",{"@attributes":{"id":"p-0131","num":"0130"},"chemistry":{"@attributes":{"id":"CHEM-US-00001","num":"00001"},"img":{"@attributes":{"id":"EMI-C00001","he":"88.05mm","wi":"75.78mm","file":"US09139827-20150922-C00001.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"The ability of the compositions to enter cells was investigated by confocal microscopy using fluorescent (Cyanine 5, Cy5) compositions prepared as above. RNA-Au NPs were added to cultures of HeLa cells. Cells were grown on glass coverslips and treated with nanoparticles functionalized with fluorophore-labled RNA duplexes. After 6 hours of treatment, the coverslips were removed, washed with PBS, and fixed to a chamber filled with PBS mounted on a glass slide. All images were obtained by scanning confocal microscopy (Zeiss 510 LSM) at 63\u00d7 magnification and a 633 nm HeNe laser excitation source. Imaging studies reveal fluorescence throughout the cytoplasm of HeLa cells after 6 hours (). It is interesting to note that like DNA Au-NPs, RNA Au-NPs require no transfection agent to enter cells [Giljohann et al., Nano Lett. 7: 3818 (2007)]. Indeed, analytical flow cytometry confirmed uptake of the RNA-Au NPs in >99% of the cell population (). For the flow cytometry experiments, cells were treated with fluorescently-labeled (Cy5) RNA-nanoparticle compositions. Six hours post-transfection, cells were trypsinized to remove them from cell culture wells. Flow cytometry was performed using a DakoCytomation CyAn with excitation at 635 nm.","Having determined that RNA-Au NPs are internalized by cells, the intracellular activity of RNA-gold nanoparticle compositions was examined next. Protein knockdown studies were performed in HeLa cells using a transfected luciferase plasmid as a target for this model system. HeLa cells (ATCC) were grown in Eagle's minimal essential medium (EMEM), with 10% heat-inactivated fetal bovine serum (FBS) and maintained at 37\u00b0 C. in 5% CO. Cells were seeded in 96 well plates and grown for 1 day prior to transfection of a plasmid (psiCHECK 2, Promega) which contains both firefly luciferase and Renilla genes. Plasmid (0.2 \u03bcg per well) was added using Lipofectamine 2000 (Invitrogen) according to the manufacturer's recommendations. After plasmid introduction (24 hrs) the media was replaced with reduced serum media containing the functionalized RNA-Au NPs (3 nM nanoparticle concentration, \u2248100 nM RNA duplex concentration) directed against firefly luciferase. On day one of treatment, the cells were approximately 70% confluent. At the conclusion of experiments, replicate wells of treated cells were counted and measured for viability using a Guava EasyCyte Mini (Guava Technologies). Viability after incubation was >98% for cells treated with RNA Au NPs.","For comparison, the same number of luciferase RNA duplexes (100 nM) were transfected using the commercial agent Lipofectamine 2000 according to the manufacturer's recommended protocol. After 24 hours of treatment, the media was replaced with fresh EMEM. Cells were assayed for luciferase expression using a Dual-Glo (Promega) assay according to the manufacturer's protocols after the indicated number of days.","Quantification of luciferase expression was normalized to controls that had not been transfected and revealed that the nanoparticle agents down-regulate firefly luciferase in a dose and time dependent manner. Control (Renilla) expression is not affected by RNA-Au NPs designed against firefly luciferase, indicating that the knockdown is also sequence specific. Interestingly, the results of three independent experiments with the RNA-Au NPs show knockdown that exceeds that of the free RNA four days after treatment (73\u00b17% RNA-Au NPs vs. 33\u00b12% free, ).","The persistent knockdown of luciferase is the result of the stabilization of the RNA on the nanoparticle. To make this determination, Cy3-labeled RNA particles were diluted to a concentration of 5 nM in 90 \u03bcL of PBS in a 96-well plate. For the dabcyl-labeled molecular RNA, the concentration was 150 nM. The microplate was placed into a Photon Technology International FluoDia T70 fluorescence plate reader that was maintained at 37\u00b0 C. After allowing the sample to equilibrate (10 minutes), 10 \u03bcL of fetal bovine serum (FBS, Gemini Bioproducts) was added to bring the samples to 10% serum concentration. To prevent evaporation, the reaction was covered with 40 \u03bcL of mineral oil. The fluorescence of the sample (excitation=530 nm, emission=570 nm) was measured every 5 minutes for 48 hours. The base line fluorescence was determined from a sample treated with a 10 \u03bcL aliquot of PBS in place of FBS. The endpoint of the reaction was determined when no further increase in fluorescence as a function of time was observed. All samples were measured in triplicate.","In these stability experiments, compositions incubated in serum showed greatly enhanced stability relative to their molecular RNA counterparts. For example, in the presence of 10% serum, RNA-Au NP compositions had a 6-times greater half-life than molecular RNA duplexes (816\u00b159 mins vs. 133\u00b130 mins, ). These data indicate that nanoparticle conjugation provides significant protection from degradation in an extracellular context. Since the extracellular lifetime of RNA is of great importance for their storage, handling, and potential therapeutic application, nanoparticle conjugation may provide a significant advantage for the protection and delivery of functional RNA ligands. Importantly, this enhanced stability does not require chemical modifications to protect the integrity of the RNA.","The orientation of RNA immobilized on a nanoparticle can be controlled. The strategy of immobilizing the RNA substrates for Dicer enzymes allows for the control of access to the dupex. Different immobilization chemistries, monothiol or dithiol, and different lengths of spacer sequences may be utilized to vary the number of and the distance between the RNA duplexes thus controlling the rates of RNA degradation.","Experiments were performed to determine whether Dicer, an RNase responsible for initiating RNAi in this system, would be able to recognize and cleave these duplexes. In a typical enzyme kinetic experiment, RNA-Au NPs (approximately 5 nM) were mixed with Dicer (0.1 U\/mL final) in reaction buffer at 37\u00b0 C. The rate of degradation of RNA duplex was measured by monitoring the fluorescence increase every 72 seconds for at least 12 hours. To determine minimum and maximum fluorescence, samples containing no enzyme (minimum) or 3 mM potassium cyanide (KCN) to dissolve the gold (maximum) were used. The KCN oxidizes the gold nanoparticles, eliminating the quenching of the fluorophore-labeled strands.","The number of duplexes per nanoparticle was determined by fluorescently labeling one or both of the strands and comparing the fluorescence associated with a given concentration of RNA-functionalized nanoparticles with a standard curve generated using the same strands. Results from a representative loading study are shown in Table 1.",{"@attributes":{"id":"p-0141","num":"0140"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Determination of RNA duplex loading using two different RNA lading"},{"entry":"strategies. The conjugates contain fluorescein, either on the antisense strand"},{"entry":"(hybridized to the sense strand which is covalently linked to the nanoparticle),"},{"entry":"or in the case of the hairpin system, RNA is labeled with fluorescein on"},{"entry":"the 5\u2032 end."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Type of nanoparticle",{},"Number of"]},{"entry":[{},"composition","Location of fluorophore","duplexes"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":[{},"Two strands","5\u2032 end of the hybridized strand","91"]},{"entry":[{},"Single strand RNA","5\u2032 end of the hairpin","34"]},{"entry":[{},"Hairpin"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},"RNase III, a ribonuclease that is known to degrade dsRNAs, was compared to the activity of Dicer. Both RNase III and Dicer are active against the two systems tested as measured by increase in fluorescence above background (no enzyme added to reaction) over time.  shows the activity in the presence of RNase III.","When each system was treated with Dicer in a similar manner to RNase III, a commensurate increase in fluorescence was observed over time (). Additionally, the absolute difference in fluorescence above background was higher for Dicer compared to RNase III. These data suggest that these RNAs are more specifically recognized by Dicer than by a nonspecific enzyme like RNase III when densely immobilized on the nanoparticle surface. Furthermore, preferences in orientation can be observed for the immobilization of the sense versus antisense strand (). In the case where the sense strand is chemically attached to the particle, higher activity is observed. Without being bound by theory, this may reflect the ability of the antisense strand to act as the guide strand in the RNAi machinery. This difference is contemplated to be of use in modulating and tuning the RNAi response in cells."],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0038","num":"0037"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0039","num":"0038"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0040","num":"0039"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0041","num":"0040"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0042","num":"0041"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0043","num":"0042"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0044","num":"0043"},"figref":"FIG. 7"},{"@attributes":{"id":"p-0045","num":"0044"},"figref":"FIG. 8"}]},"DETDESC":[{},{}]}
