---
title: Biological markers predictive of anti-cancer response to insulin-like growth factor-1 receptor kinase inhibitors
abstract: The present invention provides diagnostic methods for predicting the effectiveness of treatment of a cancer patient with an IGF-1R kinase inhibitor. Methods are provided for predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor, comprising assessing whether the tumor cell expresses certain sensitivity or resistance biomarkers, or genomic classifiers. Improved methods for treating cancer patients with IGF-1R kinase inhibitors that incorporate this methodology are also provided.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08163509&OS=08163509&RS=08163509
owner: The Regents of the University of Colorado, A Body Corporate
number: 08163509
owner_city: Denver
owner_country: US
publication_date: 20091020
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATIONS","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","EXPERIMENTAL DETAILS","CONCLUSIONS","ABBREVIATIONS","INCORPORATION BY REFERENCE","EQUIVALENTS"],"p":["This application claims the benefit of U.S. Provisional Application No. 61\/196,885, filed on Oct. 20, 2008 and U.S. Provisional Application No. 61\/251,112 filed on Oct. 13, 2009.","Cancer is a generic name for a wide range of cellular malignancies characterized by unregulated growth, lack of differentiation, and the ability to invade local tissues and metastasize. These neoplastic malignancies affect, with various degrees of prevalence, every tissue and organ in the body. The present invention is directed to methods for diagnosing and treating cancer patients. In particular, the present invention is directed to methods for determining which patients will most benefit from treatment with an insulin-like growth factor-1 receptor (IGF-1R) kinase inhibitor.","IGF-1R belongs to the insulin receptor family that includes the Insulin Receptor (IR), IGF-1R (homodimer), IGF-1R\/IR (hybrid receptor), and IGF-2R (mannose 6-phosphate receptor). IGF-1R\/IR hybrids act as homodimers, preferentially binding and signaling with IGFs. IR exists in two isoforms: IR-B (traditional insulin receptor) and IR-A (a fetal form which is re-expressed in selected tumors and preferentially binds IGF-II). IGF-2R is a non-signaling receptor that acts as a \u201csink\u201d for IGF-II (Pollak M. N., et al. Nat Rev Cancer 2004 4:505-18). Six well-characterized insulin-like growth factor binding proteins (IGFBP-1 through -6) associate with IGF ligands to stabilize the IGFs and modulate their ability to bind the IGF-IR.","IGF-1R is a transmembrane RTK that binds primarily to IGF-1 but also to IGF-II and insulin with lower affinity. Binding of IGF-1 to its receptor results in receptor oligomerization, activation of tyrosine kinase, intermolecular receptor autophosphorylation, phosphorylation of cellular substrates, including IRS1 and Shc, leading to activation of the PI3K\/Akt and mitogen-activated protein kinase (MAPK) pathways (Adams T. E., et al. Cell Mol Life Sci 2000 57:1050-93; Pollak M. N., et al. Nat Rev Cancer 2004 4:505-18; Baserga R., Exp Cell Res 1999 253:1-6). The ligand-activated IGF-1R induces mitogenic activity in normal cells and plays an important role in abnormal growth. A major physiological role of the IGF-1 system is the promotion of normal growth and regeneration. Overexpressed IGF-1R (type 1 insulin-like growth factor receptor) can initiate mitogenesis and promote ligand-dependent neoplastic transformation. Furthermore, IGF-1R plays an important role in the establishment and maintenance of the malignant phenotype. Unlike the epidermal growth factor (EGF) receptor, no mutant oncogenic forms of the IGF-1R have been identified. However, several oncogenes have been demonstrated to affect IGF-1 and IGF-1R expression. A correlation between a reduction of IGF-1R expression and resistance to transformation has been seen. Exposure of cells to mRNA antisense to IGF-1R RNA prevents soft agar growth of several human tumor cell lines. IGF-1R abrogates progression into apoptosis, both in vivo and in vitro. It has also been shown that a decrease in the level of IGF-1R below wild-type levels causes apoptosis of tumor cells in vivo. The ability of IGF-1R disruption to cause apoptosis appears to be diminished in normal, non-tumorigenic cells.","The IGF-1 pathway has an important role in human tumor development. IGF-1R overexpression is frequently found in various tumors (breast, colon, lung, sarcoma) and is often associated with an aggressive phenotype. High circulating IGF1 concentrations are strongly correlated with prostate, lung and breast cancer risk. Furthermore, IGF-1R is required for establishment and maintenance of the transformed phenotype in vitro and in vivo (Baserga R. 1999, 253, 1-6). The kinase activity of IGF-1R is essential for the transforming activity of several oncogenes: EGFR, PDGFR, SV40 T antigen, activated Ras, Raf, and v-Src. The expression of IGF-1R in normal fibroblasts induces neoplastic phenotypes, which can then form tumors in vivo. IGF-1R expression plays an important role in anchorage-independent growth. IGF-1R has also been shown to protect cells from chemotherapy-, radiation-, and cytokine-induced apoptosis. Conversely, inhibition of endogenous IGF-1R by dominant negative IGF-1R, triple helix formation or antisense expression vector has been shown to repress transforming activity in vitro and tumor growth in animal models. The IGF-1R signaling pathway also appears to be a robust target in colorectal cancer (CRC), based upon data demonstrating overexpression of the receptor and ligands in CRC, association with a more malignant phenotype, chemotherapy resistance, and correlation with a poor prognosis (Saltz, L. B., et al. J Clin Oncol 2007; 25(30): 4793-4799; Tripkovic I., et al. Med Res. 2007 July; 38(5):519-25. Epub 2007 Apr. 26; Miyamoto S., et al. Clin Cancer Res. 2005 May 1; 11(9):3494-502; Nakamura M., et al. Clin Cancer Res. 2004 Dec. 15; 10(24):8434-41; Grothey A, et al. J Cancer Res Clin Oncol. 1999; 125(3-4):166-73).","It has been recognized that inhibitors of protein-tyrosine kinases are useful as selective inhibitors of the growth of mammalian cancer cells. For example, Gleevec\u2122 (also known as imatinib mesylate), a 2-phenylpyrimidine tyrosine kinase inhibitor that inhibits the kinase activity of the BCR-ABL fusion gene product, has been approved by the U.S. Food and Drug Administration for the treatment of CML. The 4-anilinoquinazoline compound Tarceva\u2122 (erlotinib HCl) has also been approved by the FDA, and selectively inhibits EGF receptor kinase with high potency. The development for use as anti-tumor agents of compounds that directly inhibit the kinase activity of IGF-1R, as well as antibodies that reduce IGF-1R kinase activity by blocking IGF-1R activation or antisense oligonucleotides that block IGF-1R expression, are areas of intense research effort (e.g. see Larsson, O. et al (2005) Brit. J. Cancer 92:2097-2101; Ibrahim, Y. H. and Yee, D. (2005) Clin. Cancer Res. 11:944s-950s; Mitsiades, C. S. et al. (2004) Cancer Cell 5:221-230; Camirand, A. et al. (2005) Breast Cancer Research 7:R570-R579 (DOI 10.1186\/bcr1028); Camirand, A. and Pollak, M. (2004) Brit. J. Cancer 90:1825-1829; Garcia-Echeverria, C. et al. (2004) Cancer Cell 5:231-239; Sachdev D, and Yee D., Mol Cancer Ther. 2007 January; 6(1):1-12; Hofmann F., and Garcia-Echeverria C., Drug Discov Today 2005 10:1041-7). Agents inhibiting the IGF-1R pathway have demonstrated anti-tumor efficacy in multiple human cancer models both in vitro and in vivo, particularly in pediatric models of Ewing's sarcoma and rhabdomyosarcoma (Manara M C, et al. Int J Oncol 2005 27:1605-16). Despite early hints of efficacy in patients with sarcoma, results to date of IGF-1R inhibitors in early clinical trials have not been impressive, indicating that patient selection strategies and rational combinations may be needed to move forward with this approach (Tolcher A. W., et al. Journal of Clinical Oncology, 2007 ASCO Annual Meeting Proceedings Part I. Vol 25, No. 18S (June 20 Supplement), 2007: 3002). Data acquired this far, has not indicated that activation, overexpression, or amplification of members of the IGF-1R pathway will predict responsiveness.","An anti-neoplastic drug would ideally kill cancer cells selectively, with a wide therapeutic index relative to its toxicity towards non-malignant cells. It would also retain its efficacy against malignant cells, even after prolonged exposure to the drug. Unfortunately, none of the current chemotherapies possess such an ideal profile. Instead, most possess very narrow therapeutic indexes. Furthermore, cancerous cells exposed to slightly sub-lethal concentrations of a chemotherapeutic agent will very often develop resistance to such an agent, and quite often cross-resistance to several other antineoplastic agents as well. Additionally, for any given cancer type one frequently cannot predict which patient is likely to respond to a particular treatment, even with newer gene-targeted therapies, such as protein-tyrosine kinase inhibitors, thus necessitating considerable trial and error, often at considerable risk and discomfort to the patient, in order to find the most effective therapy.","Thus, there is a need for more efficacious treatment for neoplasia and other proliferative disorders, and for more effective means for determining which tumors will respond to which treatment. Strategies for enhancing the therapeutic efficacy of existing drugs have involved changes in the schedule for their administration, and also their use in combination with other anticancer or biochemical modulating agents. Combination therapy is well known as a method that can result in greater efficacy and diminished side effects relative to the use of the therapeutically relevant dose of each agent alone. In some cases, the efficacy of the drug combination is additive (the efficacy of the combination is approximately equal to the sum of the effects of each drug alone), but in other cases the effect is synergistic (the efficacy of the combination is greater than the sum of the effects of each drug given alone). Target-specific therapeutic approaches are generally associated with reduced toxicity compared with conventional cytotoxic agents, and therefore lend themselves to use in combination regimens.","Several groups have investigated potential biomarkers to predict a patient's response to protein-tyrosine kinase inhibitors (see for example, PCT publications: WO 2004\/063709, WO 2005\/017493, WO 2004\/111273, and WO 2004\/071572; and US published patent applications: US 2005\/0019785, US 2007\/0065858, and US 2004\/0132097). However, no diagnostic or prognostic tests have yet emerged that can effectively guide practicing physicians in the treatment of their patients with such inhibitors, or can indicate to the physician which tumors will respond most favorable to a combination of such an inhibitor with a standard chemotherapy agent.","Thus, there remains a critical need for improved methods for determining the best mode of treatment for any given cancer patient. The present invention provides methods for determining which tumors will respond most effectively to treatment with IGF-1R kinase inhibitors based on whether the tumor cells express novel \u201csensitivity\u201d or \u201cresistance\u201d biomarkers, and for the incorporation of such determinations into more effective treatment regimens for cancer patients, whether such inhibitors are used as single agents or combined with other anti-cancer agents.","The present invention provides diagnostic methods for predicting the effectiveness of treatment of a cancer patient with an IGF-1R kinase inhibitor. These methods are based on the surprising discovery that the sensitivity of tumor cell growth to inhibition by IGF-1R kinase inhibitors is predicted by whether such tumor cells express certain \u201csensitivity\u201d or \u201cresistance\u201d biomarkers, or genomic classifiers.","Improved methods for treating cancer patients with IGF-1R kinase inhibitors that incorporate the above methodology are also provided. Thus, the present invention further provides a method for treating tumors or tumor metastases in a patient, comprising the steps of diagnosing a patient's likely responsiveness to an IGF-1R kinase inhibitor by assessing whether the tumor cells express high levels of sensitivity and\/or resistance biomarkers, or certain genomic classifiers, and administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor (e.g. OSI-906) where sensitivity to the inhibitor is predicted.","The term \u201ccancer\u201d in an animal refers to the presence of cells possessing characteristics typical of cancer-causing cells, such as uncontrolled proliferation, immortality, metastatic potential, rapid growth and proliferation rate, and certain characteristic morphological features. Often, cancer cells will be in the form of a tumor, but such cells may exist alone within an animal, or may circulate in the blood stream as independent cells, such as leukemic cells.","\u201cAbnormal cell growth\u201d, as used herein, unless otherwise indicated, refers to cell growth that is independent of normal regulatory mechanisms (e.g., loss of contact inhibition). This includes the abnormal growth of: (1) tumor cells (tumors) that proliferate by expressing a mutated tyrosine kinase or overexpression of a receptor tyrosine kinase; (2) benign and malignant cells of other proliferative diseases in which aberrant tyrosine kinase activation occurs; (4) any tumors that proliferate by receptor tyrosine kinases; (5) any tumors that proliferate by aberrant serine\/threonine kinase activation; and (6) benign and malignant cells of other proliferative diseases in which aberrant serine\/threonine kinase activation occurs.","The term \u201ctreating\u201d as used herein, unless otherwise indicated, means reversing, alleviating, inhibiting the progress of, or preventing, either partially or completely, the growth of tumors, tumor metastases, or other cancer-causing or neoplastic cells in a patient with cancer. The term \u201ctreatment\u201d as used herein, unless otherwise indicated, refers to the act of treating.","The phrase \u201ca method of treating\u201d or its equivalent, when applied to, for example, cancer refers to a procedure or course of action that is designed to reduce or eliminate the number of cancer cells in an animal, or to alleviate the symptoms of a cancer. \u201cA method of treating\u201d cancer or another proliferative disorder does not necessarily mean that the cancer cells or other disorder will, in fact, be eliminated, that the number of cells or disorder will, in fact, be reduced, or that the symptoms of a cancer or other disorder will, in fact, be alleviated. Often, a method of treating cancer will be performed even with a low likelihood of success, but which, given the medical history and estimated survival expectancy of an animal, is nevertheless deemed an overall beneficial course of action.","The term \u201ctherapeutically effective agent\u201d means a composition that will elicit the biological or medical response of a tissue, system, animal or human that is being sought by the researcher, veterinarian, medical doctor or other clinician.","The term \u201ctherapeutically effective amount\u201d or \u201ceffective amount\u201d means the amount of the subject compound or combination that will elicit the biological or medical response of a tissue, system, animal or human that is being sought by the researcher, veterinarian, medical doctor or other clinician.","The data presented in the Experimental Details section herein below demonstrate that tumor cells, such as CRC (colorectal cancer) cells, show a range of sensitivities to growth inhibition by an IGF-1R kinase inhibitor (e.g. PQIP, OSI-906). It is demonstrated that the degree of sensitivity of tumor cells to an IGF-1R kinase inhibitor can be assessed by determining the level of biomarkers expressed by a tumor cell, that are characteristic for cells that are either relatively sensitive (i.e. \u201csensitivity\u201d biomarker) or relatively resistant (\u201cresistance\u201d biomarker) to inhibition by an IGF-1R kinase inhibitor. For example, high levels of tumor cell expression of \u201csensitivity\u201d biomarkers such as Mitogen Activated Protein Kinase Kinase 6 (MAP2K6) and Aldehyde Dehydrogenase 1-A1 (ALDH1A1), correlate with high sensitivity to IGF-1R kinase inhibitors. Conversely, high levels of tumor cell expression of \u201cresistance\u201d biomarkers such as Metallothionein 1E (MT-1E) and Caldesmon (CALD1) correlate with low sensitivity to IGF-1R kinase inhibitors. Additional sensitivity or resistance biomarkers that can be used to predict the level of sensitivity of tumor cells to an IGF-1R kinase inhibitor are listed herein below, and in . Thus, these observations can form the basis of valuable new diagnostic methods for predicting the effects of IGF-1R kinase inhibitors on tumor growth, and give oncologists additional tools to assist them in choosing the most appropriate treatment for their patients. The data also indicates that tumor cells which are predicted to be sensitive to an IGF-1R kinase inhibitor by determining sensitivity or resistance biomarker expression are also likely to respond in a synergistic manner to treatment with a combination of an IGF-1R kinase inhibitor and an anti-cancer agent, wherein the anticancer agent is SN38, oxaliplatin, or 5-Fluorouracil. Furthermore, the data suggests that resistance biomarkers may also be useful for predicting which tumors develop resistance to IGF-1R kinase inhibitors.","Sensitivity biomarkers useful in any of the methods of this invention include expression products of the following genes: aldehyde dehydrogenase 1 family, member A1 (ALDH1A1, GeneID: 216); ring finger protein 128 (RNF128, GeneID: 79589); mitogen-activated protein kinase kinase 6 (MAP2K6, GeneID: 5608); quinolinate phosphoribosyltransferase (nicotinate-nucleotide pyrophosphorylase (carboxylating)) (QPRT, GeneID: 23475); interleukin 15 (IL15, GeneID: 3600); phospholipase D1, phosphatidylcholine-specific (PLD1, GeneID: 5337); hypothetical protein LOC157860 (LOC157860, GeneID: 157860); Muscleblind-like 2 () (MBNL2, GeneID: 10150); TBC1 domain family, member 8B (with GRAM domain) (TBC1D8B, GeneID: 54885); Galactokinase 2 (GALK2, GeneID: 2585); UDP-N-acetyl-alpha-D-galactosamine:polypeptide N-acetylgalactosaminyltransferase 4 (GalNAc-T4) (GALNT4, GeneID: 8693); PAN3 polyA specific ribonuclease subunit homolog () (PAN3, GeneID: 255967); dual-specificity tyrosine-(Y)-phosphorylation regulated kinase 2 (DYRK2, GeneID: 8445); pellino homolog 2 () (PELI2, GeneID: 57161); phosphoinositide-3-kinase, regulatory subunit 1 (p85 alpha) (PIK3R1, GeneID: 5295); phosphoglucomutase 2-like 1 (PGM2L1, GeneID: 283209); phosphorylase kinase, beta (PHKB, GeneID: 5257); Hypothetical protein LOC644009 (LOC644010, GeneID: 644010); CDC14 cell division cycle 14 homolog B ()\/\/\/CDC14 cell division cycle 14 homolog B () (CDC14B, GeneID: 8555); hypothetical protein LOC128977 (C22orf39, GeneID: 128977); solute carrier family 44, member 1 (SLC44A1, GeneID: 23446); hypothetical protein LOC202451 (LOC202451, GeneID: 202451); transmembrane protein 164\/\/\/similar to hypothetical protein FLJ22679 (TMEM164, GeneID: 84187); similar to cervical cancer suppressor-1 (ST20, GeneID: 400410 (also known as LOC400410)); hypothetical protein FLJ30596 (C5orf33, GeneID: 133686); lysosomal-associated membrane protein 2 (LAMP2, GeneID: 3920); protein phosphatase 1B (formerly 2C), magnesium-dependent, beta isoform (PPM1B, GeneID: 5495); Dehydrogenase\/reductase (SDR family) member 3 (DHRS3, GeneID: 9249); Leucine zipper and CTNNBIP1 domain containing (LZIC, GeneID: 84328); ubiquitin specific peptidase like 1 (USPL1, GeneID: 10208); golgi autoantigen, golgin subfamily b, macrogolgin (with transmembrane signal), 1 (GOLGB1, GeneID: 2804); chromosome 20 open reading frame 74 (C20orf74, GeneID: 57186); Coiled-coil domain containing 52 (CCDC52, GeneID: 152185); RAB40B, member RAS oncogene family (RAB40B, GeneID: 10966); frequently rearranged in advanced T-cell lymphomas 2 (FRAT2, GeneID: 23401); hsa-miR-224 (GeneID: 407009); hsa-miR-181a (GeneID: 406995); hsa-miR-194 (GeneID: 406969, 406970); hsa-miR-192 (GeneID: 406967); hsa-miR-215 (GeneID: 406997); hsa-miR-200b (GeneID:); hsa-miR-429 (GeneID: 554210); hsa-miR-200a (GeneID: 406983); hsa-miR-192* (GeneID: 406967); hsa-miR-200b* (GeneID: 406984); and hsa-miR-584 (GeneID: 693169).","Resistance biomarkers useful in any of the methods of this invention include expression products of the following genes: caldesmon 1 (CALD1, GeneID: 800); kelch-like 5 () (KLHL5, GeneID: 51088); metallothionein 1E (functional) (MT1E, GeneID: 4493); beta-1,3-N-acetylgalactosaminyltransferase 1 (globoside blood group) (B3GALNT1, GeneID: 8706); cysteine-rich, angiogenic inducer, 61 (CYR61, GeneID: 3491); metallothionein 1X (MT1X, GeneID: 4501); troponin T type 1 (skeletal, slow) (TNNT1, GeneID: 7138); metallothionein 1H-like protein\/\/\/hypothetical protein LOC650610 (MT1P2, GeneID: 645745 (also known as LOC645745)); metallothionein 1H (MT1H, GeneID: 4496); metallothionein 1F (functional) (MT1F, GeneID: 4494); metallothionein 2A (MT2A, GeneID: 4502); metallothionein 1M (MT1M, GeneID: 4499); MHC class I polypeptide-related sequence B (MICB, GeneID: 4277); collagen, type VI, alpha 1 (COL6A1, GeneID: 1291); myosin, light polypeptide 9, regulatory (MYL9, GeneID: 10398); metallothionein 1G (MT1G, GeneID: 4495); peripheral myelin protein 22 (PMP22, GeneID: 5376); chromosome 1 open reading frame 115 (C1orf115, GeneID: 79762); SWI\/SNF related, matrix associated, actin dependent regulator of chromatin, subfamily d, member 3 (SMARCD3, GeneID: 6604); clusterin (CLU, GeneID: 1191); shroom family member 2 (SHROOM2, GeneID: 357); purinergic receptor P2X, ligand-gated ion channel, 5 (P2RX5, GeneID: 5026); coiled-coil domain containing 109B (CCDC109B, GeneID: 55013); transmembrane protein 52 (TMEM52, GeneID: 339456); cyclin E2 (CCNE2, GeneID: 9134); transforming growth factor, beta 1 (Camurati-Engelmann disease) (TGFB1, GeneID: 7040); suppressor of cytokine signaling 3 (SOCS3, GeneID: 9021); NACHT, leucine rich repeat and PYD containing 11 (NLRP11, GeneID: 204801); glypican 1 (GPC1, GeneID: 2817); kinesin light chain 3 (KLC3, GeneID: 147700); breast carcinoma amplified sequence 4 (BCAS4, GeneID: 55653); establishment of cohesion 1 homolog 2 () (ESCO2, GeneID: 157570); Netrin 1 (NTN1, GeneID: 9423); MAD2 mitotic arrest deficient-like 2 (yeast) (MAD2L2, GeneID: 1045); tubulin tyrosine ligase-like family (TTLL7, GeneID: 79739), member 7; scavenger receptor class A, member 3 (SCARA3, GeneID: 51435); growth arrest and DNA-damage-inducible, beta (GADD45B, GeneID: 4616); immunoglobulin superfamily, member 4C (CADM4, GeneID: 199731 (also known as IGSF4C)); DDHD domain containing 1 (DDHD1, GeneID: 80821); BTG family, member 3 (BTG3, GeneID: 10950); kinesin family member 26A (KIF26A, GeneID: 26153); KIAA1622 (PPP4R4, GeneID: 57718); guanosine monophosphate reductase\/\/\/guanosine monophosphate reductase (GMPR, GeneID: 2766); storkhead box 1 (STOX1, GeneID: 219736); KIAA0672 gene product (RICH2, GeneID: 9912); MCM10 minichromosome maintenance deficient 10 () (MCM10, GeneID: 55388); DDHD domain containing 1 (DDHD1, GeneID: 80821); adenosine deaminase, RNA-specific, B1 (RED1 homolog rat) (ADARB1, GeneID: 104); CKLF-like MARVEL transmembrane domain containing 7 (CMTM7, GeneID: 112616); forkhead box F1 (FOXF1, GeneID: 2294); nuclear casein kinase and cyclin-dependent kinase substrate 1 (NUCKS1, GeneID: 64710); chromosome 11 open reading frame 63 (C11orf63, GeneID: 79864); acyl-CoA thioesterase 7 (ACOT7, GeneID: 11332); zinc finger protein 286 (ZNF286A, GeneID: 57335); amyloid beta (A4) precursor-like protein 1 (APLP1, GeneID: 333); ornithine aminotransferase (gyrate atrophy) (OAT, GeneID: 4942); pericentriolar material 1 (MBD1, GeneID: 4152 (also known as PCM1)); PRP40 pre-mRNA processing factor 40 homolog B () (PRPF40B, GeneID: 25766); solute carrier family 12 (potassium\/chloride transporters), member 4 (SLC12A4, GeneID: 6560); hypothetical protein FLJ38973 (C2orf69, GeneID: 205327); calcium and integrin binding family member 2 (CIB2, GeneID: 10518); integrin, alpha 7 (ITGA7, GeneID: 3679); BUB3 budding uninhibited by benzimidazoles 3 homolog (yeast) (BUB3, GeneID: 9184); chromosome 1 open reading frame 135 (C1orf135, GeneID: 79000); cell division cycle 27 (CDC27, GeneID: 996); docking protein 1, 62 kDa (downstream of tyrosine kinase 1) (DOK1, GeneID: 1796); adenosine kinase (ADK, GeneID: 132); Meis1, myeloid ecotropic viral integration site 1 homolog 3 (mouse) (MEIS3, GeneID: 56917); kringle containing transmembrane protein 2 (KREMEN2, GeneID: 79412); chromosome 21 open reading frame 91 (C21orf91, GeneID: 54149); solute carrier family 4, anion exchanger, member 3 (SLC4A3, GeneID: 6508); zinc finger protein 558 (ZNF558, GeneID: 148156); KIAA1442 protein (EBF4, GeneID: 57593 (also known as RP5-860F19.3)); MCM4 minichromosome maintenance deficient 4 () (MCM4, GeneID: 4173); mitogen-activated protein kinase 1 (MAPK1, GeneID: 5594); hsa-miR-886-3p (GeneID: 100126299); hsa-miR-521 (GeneID: 574494, 574481); and hsa-miR-432 (GeneID: 574451).","The NCBI GeneID numbers listed herein are unique identifiers of the biomarker gene from the NCBI Entrez Gene database record (National Center for Biotechnology Information (NCBI), U.S. National Library of Medicine, 8600 Rockville Pike, Building 38A, Bethesda, Md. 20894. The sequences of representative mRNAs expressed from the biomarker gene are also listed herein below. Proteins expressed from these mRNAs represent biomarker proteins that may be used in any of the methods of this invention.","Accordingly, the present invention provides a method of predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor, comprising: assessing the level of a sensitivity biomarker expressed by a tumor cell; and predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor, wherein high expression levels of tumor cell sensitivity biomarkers correlate with high sensitivity to inhibition by IGF-1R kinase inhibitors. Preferred examples of sensitivity biomarkers include Mitogen Activated Protein Kinase Kinase 6 (MAP2K6) and Aldehyde Dehydrogenase 1-A1 (ALDH1A1). Additional examples of sensitivity biomarkers that can be utilized in the methods of this invention include those listed herein, above, and in .","The present invention also provides a method of predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor, comprising: assessing the level of a resistance biomarker expressed by a tumor cell; and predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor, wherein high expression levels of tumor cell resistance biomarkers correlate with low sensitivity to inhibition by IGF-1R kinase inhibitors. Preferred examples of resistance biomarkers include Metallothionein 1E (MT-1E) and Caldesmon (CALD1). Additional examples of resistance biomarkers that can be utilized in the methods of this invention include those listed herein, above, and in .","Assessment of the level of a sensitivity or resistance biomarker expressed by a tumor cell in any of the methods of this invention will typically be determined relative to the expression level of said sensitivity or resistance biomarker in a control cell sample where sensitivity to inhibition by an IGF-1R kinase inhibitor is known, or can readily be determined (e.g. a tumor cell line such as those listed in ). Alternatively, a panel of tumor cell lines, each with a different level of expression of a sensitivity or resistance biomarker, and thus different sensitivity to inhibition by an IGF-1R kinase inhibitor, can be used construct a standard curve from which relative sensitivity to inhibition by an IGF-1R kinase inhibitor can be predicted. Expression levels of a sensitivity or resistance biomarker in a test tumor cell sample, or a control cell sample, may be determined relative to cell number, total protein or total RNA level, or the expression level of a housekeeping gene whose expression varies little or not at all from one cell to another, to give a \u201crelative expression level.\u201d. Comparison of biomarker expression levels in a test tumor cell sample versus a control cell sample may be performed by comparing such relative expression levels.","In the context of this invention, whether expression of biomarkers is defined as high or low may be determined relative to a control tumor cell(s) with a known biomarker expression level and sensitivity to IGF-1R inhibitor. For example, the CRC tumor cell lines listed in  herein may be used as control tumor cells. For example, of these cell lines, some are sensitive to the IGF-1R kinase inhibitors PQIP and OSI-906 (e.g. Colo205, HT29, and LS513), and express high levels of sensitivity biomarkers and low levels of resistance biomarkers. Other cell lines in  are relatively resistant to the IGF-1R kinase inhibitors PQIP and OSI-906, and express high levels of resistance biomarkers and low levels of sensitivity biomarkers (e.g. SW948, SW48, NCI-H508, HCT116, HCT15, SW480, RKO, HCT8, LoVo, LS123, T84, LS174T, LS180, SW1417, SW1116, SW837, SW1463, and SW403). Sensitivity of tumor cell growth to PQIP is defined as high if the tumor cell is inhibited with an IC50 of less than 0.5 \u03bcM, and low (i.e. relatively resistant) if the tumor cell is inhibited with an IC50 of greater than 5.0 \u03bcM. With other IGF-1R kinase inhibitors, particularly compounds of Formula I as described herein below, such as OSI-906, a qualitatively similar result is expected since they inhibit tumor cell growth by inhibiting the same signal transduction pathway as PQIP, although quantitatively the IC50 values may differ depending on the relative potency of the other inhibitor versus PQIP. OSI-906 and PQIP have almost identical IC50 values when tested on the same tumor cell type. Thus, sensitivity of tumor cell growth to OSI-906 is defined as high if the tumor cell is inhibited with an IC50 of less than or equal to 1.5 \u03bcM, and low (i.e. relatively resistant) if the tumor cell is inhibited with an IC50 of greater than 5.0 \u03bcM. The sensitivity range for other IGF-1R inhibitors may be different if the potency of the compound is different. However, the same two groups of sensitive and resistant tumor cells as described above, with their corresponding levels of sensitivity and resistance biomarker levels, can be used to predict whether new test tumors are sensitive to such other IGF-1R inhibitors by determining whether the sensitivity and\/or resistance biomarker levels of such test tumor cells are more similar to cell types in the sensitive or the resistant tumor cell groups. For example, sensitivity of tumor cell growth to antibody IGF-1R kinase inhibitors can thus readily be determined. One of skill in the art would also readily be able to identify additional tumor cell types with high or low biomarker expression levels that might also be used as control tumor cells.","It will be appreciated by those of skill in the art that a control cell sample need not be established for each assay as the assay is performed but rather, a baseline or control can be established by referring to a form of stored information regarding a previously determined control level for sensitive and resistant patients (responders and non-responders), such as a control level established by any of the methods described herein. Such a form of stored information can include, for example, but is not limited to, a reference chart, listing or electronic file of population or individual data regarding sensitive and resistant tumors\/patients, or any other source of data regarding control levels of expression of sensitivity or resistance biomarkers that is useful for the patient to be evaluated.","The present invention thus provides a method of predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor, comprising: assessing the level of a sensitivity biomarker expressed by a test tumor cell; comparing said level with the level of a sensitivity biomarker expressed by a control tumor cell of known sensitivity to an IGF-1R kinase inhibitor, and predicting the sensitivity of tumor cell growth to inhibition by the IGF-1R kinase inhibitor, wherein high expression levels of tumor cell sensitivity biomarkers correlate with high sensitivity to inhibition by IGF-1R kinase inhibitors.","The present invention also provides a method of predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor, comprising: assessing the level of a resistance biomarker expressed by a test tumor cell; comparing said level with the level of a resistance biomarker expressed by a control tumor cell of known sensitivity to an IGF-1R kinase inhibitor, and predicting the sensitivity of tumor cell growth to inhibition by the IGF-1R kinase inhibitor, wherein high expression levels of tumor cell resistance biomarkers correlate with low sensitivity to inhibition by IGF-1R kinase inhibitors.","Although many of the examples provided herein are directed to the IGF-1R kinase inhibitors PQIP or OSI-906, the methods of the present invention are not limited to the prediction of patients or tumors that will respond or not respond to this particular IGF-1R kinase inhibitor, but rather, can be used to predict patient's outcome to any IGF-1R kinase inhibitor, including inhibitors that are small molecules, peptides, antibodies, nucleic acids, or other types of inhibitors. In one embodiment, the small molecule IGF-1R kinase inhibitor may be one of a new class of relatively specific, orally-available, small-molecule compounds, as described by Formula I herein (see also US Published Patent Application US 2006\/0235031; e.g. OSI-906).","In any of the methods, compositions or kits of the invention described herein, the term \u201csmall molecule IGF-1R kinase inhibitor\u201d refers to a low molecular weight (i.e. less than 5000 Daltons; preferably less than 1000, and more preferably between 300 and 700 Daltons) organic compound that inhibits IGF-1R kinase by binding to the kinase domain of the enzyme. Examples of such compounds include IGF-1R kinase inhibitors of Formula (I) as described herein. The IGF-1R kinase inhibitor of Formula (I) can be any IGF-1R kinase inhibitor compound encompassed by Formula (I) that inhibits IGF-1R kinase upon administration to a patient. Examples of such inhibitors have been published in US Published Patent Application US 2006\/0235031, which is incorporated herein in its entirety, and include OSI-906 (cis-3-[8-amino-1-(2-phenyl-quinolin-7-yl)-imidazo[1,5-a]pyrazin-3-yl]-1-methyl-cyclobutanol), as used in the experiments described herein.","For any given sensitivity or resistance biomarker, the range of expression level between tumor cells that are relatively insensitive to IGF-1R kinase inhibitors and those that are sensitive, can readily be assessed by one of skill in the art, for example by testing on a panel of tumor cells as described herein (e.g. ), or by testing in tumor biopsies from patients whose tumor cells display a range of sensitivities to an IGF-1R kinase inhibitor (e.g. PQIP, OSI-906).","In addition, one of skill in the medical arts, particularly pertaining to the application of diagnostic tests and treatment with therapeutics, will recognize that biological systems are somewhat variable and not always entirely predictable, and thus many good diagnostic tests or therapeutics are occasionally ineffective. Thus, it is ultimately up to the judgement of the attending physician to determine the most appropriate course of treatment for an individual patient, based upon test results, patient condition and history, and his own experience. There may even be occasions, for example, when a physician will choose to treat a patient with an IGF-1R kinase inhibitor even when a tumor is not predicted to be particularly sensitive to IGF-1R kinase inhibitors, based on data from diagnostic tests or from other criteria, particularly if all or most of the other obvious treatment options have failed, or if some synergy is anticipated when given with another treatment. The fact that the IGF-1R kinase inhibitors as a class of compounds are relatively well tolerated compared to many other anti-cancer compounds, such as more traditional chemotherapy or cytotoxic agents used in the treatment of cancer, makes this a more viable option. Also, it should be noted that while the biomarkers disclosed herein predict which patients with tumors are likely to receive the most benefit from IGF-1R kinase inhibitors, it does not necessarily mean that patients with tumors which do not possess the optimal biomarker signature will receive no benefit, just that a more modest effect is to be anticipated.","Since diagnostic assays in biological systems are rarely infallible, this invention also provides additional embodiments wherein simultaneous assessment of the expression level in tumor cells of more than one biomarker level is utilized. In such embodiments (described below) there is likely to be a lower chance of a false prediction, compared to methods employing just a single biomarker expression level determination.","Accordingly, the present invention provides a method of predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor, comprising: assessing the level of two or more (or a panel of) sensitivity biomarkers expressed by a tumor cell; and predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor, wherein simultaneous high expression levels of all of the assessed tumor cell sensitivity biomarkers correlates with high sensitivity to inhibition by IGF-1R kinase inhibitors. In one preferred embodiment of this method the sensitivity biomarkers comprise Mitogen Activated Protein Kinase Kinase 6 (MAP2K6) and Aldehyde Dehydrogenase 1-A1 (ALDH1A1), wherein simultaneous high expression level of the two tumor cell sensitivity biomarkers correlates with high sensitivity to inhibition by IGF-1R kinase inhibitor. Note that in the latter embodiment a high expression level of both biomarkers is required to indicate high sensitivity.","The present invention also provides a method of predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor, comprising: assessing the level of two or more (or a panel of) resistance biomarkers expressed by a tumor cell; and predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor, wherein simultaneous low or undetectable expression levels of all of the assessed tumor cell resistance biomarkers correlates with high sensitivity to inhibition by IGF-1R kinase inhibitors. In one preferred embodiment of this method the resistance biomarkers comprise Metallothionein-1E (MT-1E) and Caldesmon (CALD1), wherein simultaneous low or undetectable expression level of the two tumor cell resistance biomarkers correlates with high sensitivity to inhibition by IGF-1R kinase inhibitor. Note that in the latter embodiment a low or undetectable expression of both biomarkers is required to indicate high sensitivity.","The present invention also provides a method of predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor, comprising: assessing the level of a sensitivity biomarker expressed by a tumor cell; assessing the level of a resistance biomarker expressed by, a tumor cell; and predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor, wherein a high ratio of sensitivity to resistance biomarker expression levels correlates with high sensitivity to inhibition by IGF-1R kinase inhibitors. In one preferred embodiment of this method the sensitivity biomarker comprises Mitogen Activated Protein Kinase Kinase 6 (MAP2K6) or Aldehyde Dehydrogenase 1-A1 (ALDH1A1) and the resistance biomarker comprises Metallothionein-1E (MT-1E) or Caldesmon (CALD1).","The present invention also provides a method of predicting the sensitivity of tumor growth to inhibition by an IGF-1R kinase inhibitor, comprising: assessing the level of two or more (or a panel of) sensitivity biomarkers expressed by cells of the tumor; and predicting the sensitivity of tumor growth to inhibition by an IGF-1R kinase inhibitor, wherein simultaneous high expression levels of all of the assessed tumor cell sensitivity biomarkers correlates with high sensitivity to inhibition by IGF-1R kinase inhibitors. In one preferred embodiment of this method the sensitivity biomarkers comprise Mitogen Activated Protein Kinase Kinase 6 (MAP2K6) and Aldehyde Dehydrogenase 1-A1 (ALDH1A1), wherein simultaneous high expression level of the two tumor cell sensitivity biomarkers correlates with high sensitivity to inhibition by IGF-1R kinase inhibitor. Note that in the latter embodiment a high expression level of both biomarkers is required to indicate high sensitivity.","The present invention also provides a method of predicting the sensitivity of tumor growth to inhibition by an IGF-1R kinase inhibitor, comprising: assessing the level of two or more (or a panel of) resistance biomarkers expressed by cells of the tumor; and predicting the sensitivity of tumor growth to inhibition by an IGF-1R kinase inhibitor, wherein simultaneous low or undetectable expression levels of all of the assessed tumor cell resistance biomarkers correlates with high sensitivity to inhibition by IGF-1R kinase inhibitors. In one preferred embodiment of this method the resistance biomarkers comprise Metallothionein 1E (MT-1E) and Caldesmon (CALD1), wherein simultaneous low or undetectable expression level of the two tumor cell resistance biomarkers correlates with high sensitivity to inhibition by IGF-1R kinase inhibitor. Note that in the latter embodiment a low or undetectable expression of both biomarkers is required to indicate high sensitivity.","The present invention also provides a method of predicting the sensitivity of tumor growth to inhibition by an IGF-1R kinase inhibitor, comprising: assessing the level of an sensitivity biomarker expressed by cells of the tumor; assessing the level of a resistance biomarker expressed by cells of the tumor; and predicting the sensitivity of tumor growth to inhibition by an IGF-1R kinase inhibitor, wherein a high ratio of sensitivity to resistance biomarker expression levels correlates with high sensitivity to inhibition by IGF-1R kinase inhibitors. In one preferred embodiment of this method the sensitivity biomarker comprises Mitogen Activated Protein Kinase Kinase 6 (MAP2K6) or Aldehyde Dehydrogenase 1-A1 (ALDH1A1) and the resistance biomarker comprises Metallothionein-1E (MT-1E) or Caldesmon (CALD1).","The present invention also provides a method of predicting whether a cancer patient is afflicted with a tumor that will respond effectively to treatment with an IGF-1R kinase inhibitor, comprising: assessing the level of two or more (or a panel of) sensitivity biomarkers expressed by cells of the tumor; and predicting if the tumor will respond effectively to treatment with an IGF-1R kinase inhibitor, wherein simultaneous high expression levels of all of the tumor cell sensitivity biomarkers correlates with a tumor that will respond effectively to treatment with an IGF-1R kinase inhibitor. In one preferred embodiment of this method the sensitivity biomarkers comprise Mitogen Activated Protein Kinase Kinase 6 (MAP2K6) and Aldehyde Dehydrogenase 1-A1 (ALDH1A1), wherein simultaneous high expression level of the two tumor cell sensitivity biomarkers correlates with a tumor that will respond effectively to treatment with an IGF-1R kinase inhibitor. Note that in the latter embodiment a high expression level of both biomarkers is required to indicate a tumor that will respond effectively to treatment with an IGF-1R kinase inhibitor.","The present invention also provides a method of predicting whether a cancer patient is afflicted with a tumor that will respond effectively to treatment with an IGF-1R kinase inhibitor, comprising: assessing the level of two or more (or a panel of) resistance biomarkers expressed by cells of the tumor; and predicting if the tumor will respond effectively to treatment with an IGF-1R kinase inhibitor, wherein simultaneous low or undetectable expression levels of all of the tumor cell resistance biomarkers correlates with a tumor that will respond effectively to treatment with an IGF-1R kinase inhibitor. In one preferred embodiment of this method the resistance biomarkers comprise Metallothionein (MT) and Caldesmon (CALD1), wherein simultaneous low or undetectable expression level of the two tumor cell resistance biomarkers correlates with a tumor that will respond effectively to treatment with an IGF-1R kinase inhibitor. Note that in the latter embodiment a low or undetectable expression of both biomarkers is required to indicate a tumor that will respond effectively to treatment with an IGF-1R kinase inhibitor.","The present invention also provides a method of predicting whether a cancer patient is afflicted with a tumor that will respond effectively to treatment with an IGF-1R kinase inhibitor, comprising: assessing the level of an sensitivity biomarker expressed by cells of the tumor; assessing the level of a resistance biomarker expressed by cells of the tumor; and predicting if the tumor will respond effectively to treatment with an IGF-1R kinase inhibitor, wherein a high ratio of sensitivity to resistance biomarker expression levels correlates with a tumor that will respond effectively to treatment with an IGF-1R kinase inhibitor. In one preferred embodiment of this method the sensitivity biomarker comprises Mitogen Activated Protein Kinase Kinase 6 (MAP2K6) or Aldehyde Dehydrogenase 1-A1 (ALDH1A1) and the resistance biomarker comprises Metallothionein 1E (MT-1E) or Caldesmon (CALD1).","The present invention also provides a method of predicting whether a cancer patient is afflicted with a tumor that will respond effectively to treatment with an IGF-1R kinase inhibitor, comprising: obtaining a sample of the patient's tumor; assessing the level of a sensitivity biomarker expressed by cells of the tumor; determining whether said level is statistically more similar to the level of the same sensitivity biomarker of tumor cells that are known to be sensitive to the IGF-1R kinase inhibitor or to the level of the same sensitivity biomarker of tumor cells that are known to be resistant to the IGF-1R kinase inhibitor; and predicting that the tumor will respond effectively to treatment with an IGF-1R kinase inhibitor if the level of the sensitivity biomarker is statistically more similar to the level of the sensitivity biomarker of tumor cells that are known to be sensitive to the IGF-1R kinase inhibitor, wherein the sensitivity biomarker is selected from any of those listed herein.","The present invention also provides a method of predicting whether a cancer patient is afflicted with a tumor that will respond effectively to treatment with an IGF-1R kinase inhibitor, comprising: obtaining a sample of the patient's tumor; assessing the level of a resistance biomarker expressed by cells of the tumor; determining whether said level is statistically more similar to the level of the same resistance biomarker of tumor cells that are known to be sensitive to the IGF-1R kinase inhibitor or to the level of the same resistance biomarker of tumor cells that are known to be resistant to the IGF-1R kinase inhibitor; and predicting that the tumor will respond effectively to treatment with an IGF-1R kinase inhibitor if the level of the resistance biomarker is statistically more similar to the level of the resistance biomarker of tumor cells that are known to be sensitive to the IGF-1R kinase inhibitor, wherein the resistance biomarker is selected from any of those listed herein.","The present invention also provides a method of treating a cancer patient with an IGF-1R kinase inhibitor, comprising: predicting whether the patient is afflicted with a tumor that will respond effectively to treatment with an IGF-1R kinase inhibitor, by: obtaining a sample of the patients tumor; assessing the level of a sensitivity biomarker expressed by cells of the tumor; determining whether said level is statistically more similar to the level of the same sensitivity biomarker of tumor cells that are known to be sensitive to the IGF-1R kinase inhibitor or to the level of the same sensitivity biomarker of tumor cells that are known to be resistant to the IGF-1R kinase inhibitor; and predicting that the tumor will respond effectively to treatment with an IGF-1R kinase inhibitor if the level of the sensitivity biomarker is statistically more similar to the level of the sensitivity biomarker of tumor cells that are known to be sensitive to the IGF-1R kinase inhibitor; and treating the patient with a therapeutically effective amount of an IGF-1R kinase inhibitor if the patient is afflicted with a tumor that is predicted to respond effectively to treatment with an IGF-1R kinase inhibitor; wherein the sensitivity biomarker is selected from any of those listed herein.","The present invention also provides a method of treating a cancer patient with an IGF-1R kinase inhibitor, comprising: predicting whether the patient is afflicted with a tumor that will respond effectively to treatment with an IGF-1R kinase inhibitor, by: obtaining a sample of the patients tumor; assessing the level of a resistance biomarker expressed by cells of the tumor; determining whether said level is statistically more similar to the level of the same resistance biomarker of tumor cells that are known to be sensitive to the IGF-1R kinase inhibitor or to the level of the same resistance biomarker of tumor cells that are known to be resistant to the IGF-1R kinase inhibitor; and predicting that the tumor will respond effectively to treatment with an IGF-1R kinase inhibitor if the level of the resistance biomarker is statistically more similar to the level of the resistance biomarker of tumor cells that are known to be sensitive to the IGF-1R kinase inhibitor; and treating the patient with a therapeutically effective amount of an IGF-1R kinase inhibitor if the patient is afflicted with a tumor that is predicted to respond effectively to treatment with an IGF-1R kinase inhibitor; wherein the resistance biomarker is selected from any of those listed herein.","In any of the methods described herein, the \u201ctumor cells that are known to be sensitive to the IGF-1R kinase inhibitor\u201d or \u201cthe tumor cells that are known to be resistant to the IGF-1R kinase inhibitor\u201d, which are used as a reference or control for comparison with the tumor cells of a patient's tumor, may be of the same tumor type (e.g. CRC, NSCLC, breast cancer etc) as the cells of the tumor from the patient being examined, or they may be from a tumor type that has similar characteristics with respect to sensitivity or resistance biomarker expression levels and their correlation with sensitivity of the tumor cells to growth inhibition by an IGF-1R kinase inhibitor.","In any of the methods described herein, in \u201cdetermining whether said level is statistically more similar to the level of the same sensitivity biomarker of tumor cells that are known to be sensitive to the IGF-1R kinase inhibitor or to the level of the same sensitivity biomarker of tumor cells that are known to be resistant to the IGF-1R kinase inhibitor\u201d, the biomarker expression level of the tumor cells \u201cknown to be sensitive\u201d or \u201cknown to be resistant\u201d may be a level corresponding to that from one sensitive or one resistant tumor cell type respectively (e.g. one of the CRC tumor cell lines used in the experimental section herein), or the level may be an average value deduced from panels of sensitive or resistant tumor cell types.","The present invention also provides a method for treating tumors or tumor metastases in a patient, comprising administering to said patient simultaneously or sequentially a therapeutically effective amount of a combination of a synthetic, cell permeable miRNA mimic of hsa-miR-224 or hsa-miR-181, and an IGF-1R kinase inhibitor. In this method the IGF-1R kinase inhibitor may be a small molecule IGF-1R kinase inhibitor, such as for example an IGF-1R kinase inhibitor of Formula (I), such as PQIP or OSI-909, or may be an anti-IGF-1R antibody, such as those described herein. In one embodiment, the patient is a human in need of treatment for cancer.","The present invention also provides a pharmaceutical composition comprising a synthetic, cell permeable miRNA mimic of hsa-miR-224 or hsa-miR-181, and an IGF-1R kinase inhibitor, in a pharmaceutically acceptable carrier. In this composition, the IGF-1R kinase inhibitor may be a small molecule IGF-1R kinase inhibitor, such as for example an IGF-1R kinase inhibitor of Formula (I), such as PQIP or OSI-909, or may be an anti-IGF-1R antibody, such as those described herein.","The present invention also provides a kit comprising one or more containers, comprising a synthetic, cell permeable miRNA mimic of hsa-miR-224 or hsa-miR-181, and an IGF-1R kinase inhibitor. In this kit, the IGF-1R kinase inhibitor may be a small molecule IGF-1R kinase inhibitor, such as for example an IGF-1R kinase inhibitor of Formula (I), such as PQIP or OSI-909, or may be an anti-IGF-1R antibody, such as those described herein.","In the context of this invention, a \u201csynthetic, cell permeable miRNA mimic\u201d is an agent that is capable of entering the tumor cells of a patient, and producing a similar sensitizing effect with respect to inhibitors of IGF-1R kinase as observed herein for the miRNA mimic of hsa-miR-224 (see Experimental section). In one embodiment this agent may comprise a vector capable of expressing hsa-miR-224 or hsa-miR-181 in the tumor cells of the patient (e.g. an adeno-associated viral (AAV) vector; e.g. see Wang, Z., et al (2005) Nat. Biotechnol. 23, 321-328.). In another embodiment, the agent may comprise an oligonucleotide modified to be resistant to cell and\/or blood degradative enzymes (e.g. a phosphorothioate or 2\u2032-0-methoxyethyl modified oligonucleotide, with for example 2-O-methyl RNA bases at both 5\u2032 and 3\u2032 ends), that may be delivered to tumor cells via for example a complex with liposomes and\/or a cell permeable peptide (e.g. see Torchilin V P. (2006) Adv Drug Deliv Rev. 2006 Dec. 1; 58 (14):1532-55). Such agents are readily prepared by methods known in the art. For example, fully phosphorothioated oligonucleotides may be synthesized using \u03b2-cyanoethylphosphor-amidite chemistry on a DNA synthesizer (e.g. see Agrawal S, et al. Proc Natl Acad Sci USA 1997; 94:2620-2625). After the synthesis, oligonucleotides can be deprotected using standard protocols, and purified by high-performance liquid chromatography.","The genes coding for examples of sensitivity or resistance molecular biomarkers that can be used in the practice of the methods of the invention are described herein, above, and in . The sensitivity or resistance molecular biomarkers can include any product expressed by these genes, e.g. expressed mRNA or protein, splice variants, polymorphic variants etc. Thus, the biomarkers include mRNAs expressed by these biomarker genes as listed in the sequence list herein, below, or mRNAs that hybridize under stringent conditions to the complement of these nucleic acids, wherein the stringent conditions comprise, for example, incubating at 42\u00b0 C. in a solution comprising 50% formamide, 5\u00d7SSC, and 1% SDS and washing at 65\u00b0 C. in a solution comprising 0.2\u00d7SSC and 0.1% SDS, or polypeptides encoded by any of these mRNAs. In an additional embodiment where the tumor is present in a non-human patient the biomarker is an animal homologue of the human gene product (e.g. from dog, mouse, rat, rabbit, cat, monkey, ape, etc.).","As described herein, this invention provides methods using different biomarkers to predict tumor sensitivity to inhibition by IGF-1R kinase inhibitors. Each of these methods have potential advantages and disadvantages, and while the preferred method will ultimately depend on individual patient circumstances, the use of multiple diagnostic methods will likely improve one's ability to predict the likely outcome of a therapeutic regimen comprising use of an IGF-1R kinase inhibitor. Therefore, this invention provides the method for treating tumors or tumor metastases in a patient comprising the initial use, either simultaneously or sequentially, of two or more of any of the diagnostic methods as described hereinfor for predicting sensitivity to inhibition by IGF-1R kinase inhibitors, followed by administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor if one or more of the diagnostic methods indicate that the patient is potentially responsive to an IGF-1R kinase inhibitor.","Factors to be considered in determining the preferred diagnostic method for predicting tumor sensitivity to inhibition by IGF-1R kinase inhibitors include both inherent characteristics of each of the methods and technical considerations affecting the use of the methods. For example, as described above, certain sensitivity or resistance biomarkers are preferred over others in diagnostic methods using a single biomarker. An improved ability to correctly predict sensitivity to IGF-1R kinase inhibitors may be achieved by employing two or more biomarker determinations in the diagnostic method.","Determination of sensitivity or resistance biomarker level can be assessed by a number of different approaches, including direct analysis of proteins that segregate as sensitivity or resistance biomarkers. An advantage of this approach is that markers are read directly. However, this approach also requires sufficient quantities of tissue in order to perform an analysis (e.g. immunohistochemistry). Sufficient quantities of tissue may be difficult to obtain from certain procedures such as FNA (fine needle aspiration). Core biopsies provide larger amounts of tissue, but are sometimes not routinely performed during diagnoses. Alternatively, these biomarkers could be evaluated based upon the expression level of their encoding RNA transcripts using a quantitative PCR based approach. An advantage of this approach is that very few tumor cells are required for this measurement, and it is very likely that sufficient material may be obtained via an FNA. However, here the transcript levels for a given biomarker may be derived from both tumor cells as well as infiltrating stromal cells from the tumor. Given that stromal cells might also express sensitivity or resistance biomarkers, this may obscure detection of the IGF-1R kinase inhibitor sensitivity status for the tumor cells. Use of in situ hybridization (e.g. FISH) or tissue microdisection may be useful here to overcome this potential limitation.","In the methods described herein the tumor cell will typically be from a patient diagnosed with cancer, a precancerous condition, or another form of abnormal cell growth, and in need of treatment. The cancer may be lung cancer (e.g. non-small cell lung cancer (NSCLC)), pancreatic cancer, head and neck cancer, gastric cancer, breast cancer, colon cancer, ovarian cancer, or any of a variety of other cancers described herein below. The cancer is preferably one known to be potentially treatable with an IGF-1R kinase inhibitor.","In the methods of this invention, sensitivity or resistance biomarker expression level can be assessed relative to a control molecule whose expression level remains relatively constant in different tumor cells (e.g. a \u201chousekeeping\u201d gene, such as GAPDH, \u03b2-actin, tubulin, or the like). Biomarker expression level can also be assessed relative to another type of tumor cell biomarker (i.e. sensitivity compared to resistance), or to the biomarker level in non-tumor cells of the same tissue, or another cell or tissue source used as an assay reference.","In the methods of this invention, the level of an sensitivity or resistance biomarker expressed by a tumor cell can be assessed by using any of the standard bioassay procedures known in the art for determination of the level of expression of a gene, including for example ELISA, RIA, immunoprecipitation, immunoblotting, immunofluorescence microscopy, RT-PCR, in situ hybridization, cDNA microarray, or the like, as described in more detail below.","In the methods of this invention, the expression level of a tumor cell sensitivity or resistance biomarker is preferably assessed by assaying a tumor biopsy. However, in an alternative embodiment, expression level of the tumor cell biomarker can be assessed in bodily fluids or excretions containing detectable levels of biomarkers originating from the tumor or tumor cells. Bodily fluids or excretions useful in the present invention include blood, urine, saliva, stool, pleural fluid, lymphatic fluid, sputum, ascites, prostatic fluid, cerebrospinal fluid (CSF), or any other bodily secretion or derivative thereof. By blood it is meant to include whole blood, plasma, serum or any derivative of blood. Assessment of tumor sensitivity or resistance biomarkers in such bodily fluids or excretions can sometimes be preferred in circumstances where an invasive sampling method is inappropriate or inconvenient.","In the methods of this invention, the tumor cell can be a lung cancer tumor cell (e.g. non-small cell lung cancer (NSCLC)), a pancreatic cancer tumor cell, a breast cancer tumor cell, a head and neck cancer tumor cell, a gastric cancer tumor cell, a colon cancer tumor cell, an ovarian cancer tumor cell, or a tumor cell from any of a variety of other cancers as described herein below. The tumor cell is preferably of a type known to or expected to express IGF-1R kinase, as do all tumor cells from solid tumors. The IGF-1R kinase can be wild type or a mutant form.","In the methods of this invention, the IGF-1R kinase inhibitor can be any IGF-1R kinase inhibitor as described herein below, including pharmacologically acceptable salts or polymorphs thereof.","The following methods represent additional specific embodiments of the methods of the invention.","The present invention provides a method of predicting the sensitivity of tumor growth to inhibition by an IGF-1R kinase inhibitor, comprising: assessing the level of an sensitivity biomarker expressed by cells of the tumor; and predicting the sensitivity of tumor growth to inhibition by an IGF-1R kinase inhibitor, wherein high expression levels of tumor cell sensitivity biomarkers correlate with high sensitivity of tumor growth to inhibition by IGF-1R kinase inhibitors.","The present invention provides a method of predicting the sensitivity of tumor growth to inhibition by an IGF-1R kinase inhibitor, comprising: assessing the level of a resistance biomarker expressed by cells of the tumor; and predicting the sensitivity of tumor growth to inhibition by an IGF-1R kinase inhibitor, wherein high expression levels of tumor cell resistance biomarkers correlate with low sensitivity of tumor growth to inhibition by IGF-1R kinase inhibitors.","The present invention provides a method of predicting whether a cancer patient is afflicted with a tumor that will respond effectively to treatment with an IGF-1R kinase inhibitor, comprising: assessing the level of an sensitivity biomarker expressed by cells of the tumor; and predicting if the tumor will respond effectively to treatment with an IGF-1R kinase inhibitor, wherein high expression levels of tumor cell sensitivity biomarkers correlate with a tumor that will respond effectively to treatment with an IGF-1R kinase inhibitor.","In the methods of this invention, the tumor can be a lung cancer tumor (e.g. non-small cell lung cancer (NSCLC)), a pancreatic cancer tumor, a breast cancer tumor, a head and neck cancer tumor, a gastric cancer tumor, a colon cancer tumor, an ovarian cancer tumor, or a tumor from any of a variety of other cancers as described herein below. The tumor is preferably of a type whose cells are known to or expected to express IGF-1R kinase, as do all solid tumors. The IGF-1R kinase can be wild type or a mutant form.","The present invention provides a method of predicting whether a cancer patient is afflicted with a tumor that will respond effectively to treatment with an IGF-1R kinase inhibitor, comprising: assessing the level of a resistance biomarker expressed by cells of the tumor; and predicting if the tumor will respond effectively to treatment with an IGF-1R kinase inhibitor, wherein high expression levels of tumor cell resistance biomarkers correlate with a tumor that will respond less effectively to treatment with an IGF-1R kinase inhibitor.","The present invention provides a method of predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor comprising: determining the tumor cell level of a sensitivity biomarker polypeptide; determining the tumor cell level of a control polypeptide; comparing the tumor cell level of the sensitivity biomarker polypeptide to the tumor cell level of the control polypeptide; wherein a high ratio of tumor cell biomarker polypeptide to tumor cell control polypeptide indicates a high predicted sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor. For this method, examples of useful sensitivity biomarker polypeptides include Mitogen Activated Protein Kinase Kinase 6 (MAP2K6) and Aldehyde Dehydrogenase 1-A1 (ALDH1A1).","The present invention provides a method of predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor comprising: determining the tumor cell level of a sensitivity biomarker polynucleotide that encodes a polypeptide; determining the tumor cell level of a control polynucleotide; comparing the tumor cell level of the sensitivity biomarker polynucleotide that encodes a polypeptide to the tumor cell level of the control polynucleotide; wherein a high ratio of tumor cell biomarker polynucleotide to tumor cell control polynucleotide indicates a high predicted sensitivity of, tumor cell growth to inhibition by an IGF-1R kinase inhibitor. For this method examples of polypeptides encoded by the sensitivity biomarker polynucleotide include Mitogen Activated Protein Kinase Kinase 6 (MAP2K6) and Aldehyde Dehydrogenase 1-A1 (ALDH1A1).","The present invention provides a method of predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor comprising: determining the tumor cell level of a resistance biomarker polypeptide; determining the tumor cell level of a control polypeptide; comparing the tumor cell level of the resistance biomarker polypeptide to the tumor cell level of the control polypeptide; wherein a low ratio of tumor cell biomarker polypeptide to tumor cell control polypeptide indicates a high predicted sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor. For this method, examples of useful resistance biomarker polypeptides include Metallothionein 1E (MT-1E) and Caldesmon (CALD1).","The present invention provides a method of predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor comprising: determining the tumor cell level of a resistance biomarker polynucleotide that encodes an polypeptide; determining the tumor cell level of a control polynucleotide; comparing the tumor cell level of the resistance biomarker polynucleotide that encodes an polypeptide to the tumor cell level of the control polynucleotide; wherein a low ratio of tumor cell biomarker polynucleotide to tumor cell control polynucleotide indicates a high predicted sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor. For this method, examples of useful polypeptides encoded by the biomarker polynucleotide include Metallothionein 1E (MT-1E) and Caldesmon (CALD1).","The present invention provides a method of predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor comprising: determining the tumor cell level of a sensitivity biomarker polypeptide; determining a non-tumor cell level of a sensitivity biomarker polypeptide; comparing the tumor cell level of the sensitivity biomarker polypeptide to the non-tumor cell level of the sensitivity biomarker polypeptide; wherein a high ratio of tumor cell biomarker polypeptide to non-tumor cell biomarker polypeptide indicates a high predicted sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor. For this method, examples of useful sensitivity biomarker polypeptide include Mitogen Activated Protein Kinase Kinase 6 (MAP2K6) and Aldehyde Dehydrogenase 1-A1 (ALDH1A1).","The present invention provides a method of predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor comprising: determining the tumor cell level of a sensitivity biomarker polynucleotide that encodes an polypeptide; determining a non-tumor cell level of a sensitivity biomarker polynucleotide that encodes an polypeptide; comparing the tumor cell level of the sensitivity biomarker polynucleotide that encodes an polypeptide to the non-tumor cell level of the sensitivity biomarker polynucleotide that encodes an polypeptide; wherein a high ratio of tumor cell biomarker polynucleotide to non-tumor cell biomarker polynucleotide indicates a high predicted sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor. For this method, examples of useful polypeptides encoded by the sensitivity biomarker polynucleotide include Mitogen Activated Protein Kinase Kinase 6 (MAP2K6) and Aldehyde Dehydrogenase 1-A1 (ALDH1A1).","The present invention provides a method of predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor comprising: determining the tumor cell level of a resistance biomarker polypeptide; determining a non-tumor cell level of a resistance biomarker polypeptide; comparing the tumor cell level of the resistance biomarker polypeptide to the non-tumor cell level of the resistance biomarker polypeptide; wherein a low ratio of tumor cell biomarker polypeptide to non-tumor cell biomarker polypeptide indicates a high predicted sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor. For this method, examples of useful resistance biomarker polypeptides include Metallothionein 1E (MT-1E) and Caldesmon (CALD1).","The present invention provides a method of predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor comprising: determining the tumor cell level of a resistance biomarker polynucleotide that encodes an polypeptide; determining a non-tumor cell level of a resistance biomarker polynucleotide that encodes an polypeptide; comparing the tumor cell level of the resistance biomarker polynucleotide that encodes an polypeptide to the non-tumor cell level of the resistance biomarker polynucleotide that encodes an polypeptide; wherein a low ratio of tumor cell biomarker polynucleotide to non-tumor cell biomarker polynucleotide indicates a high predicted sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor. For this method, examples of useful polypeptides encoded by the biomarker polynucleotide include Metallothionein 1E (MT-1E) and Caldesmon (CALD1).","The present invention provides a method of predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor comprising: determining the tumor cell level of a sensitivity biomarker polypeptide; determining the tumor cell level of a resistance biomarker polypeptide; comparing the level of the sensitivity biomarker polypeptide to the level of the resistance biomarker polypeptide; wherein a high ratio of sensitivity biomarker polypeptide to resistance biomarker polypeptide indicates a high predicted sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor. For this method, examples of useful sensitivity biomarker polypeptides include Mitogen Activated Protein Kinase Kinase 6 (MAP2K6) and Aldehyde Dehydrogenase 1-A1 (ALDH1A1). For this method, examples of useful resistance biomarker polypeptides include Metallothionein 1E (MT-1E) and Caldesmon (CALD1).","The present invention provides a method of predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor comprising: determining the tumor cell level of a sensitivity biomarker polynucleotide that encodes a polypeptide; determining the tumor cell level of a resistance biomarker polynucleotide that encodes a polypeptide; (c) comparing the level of the sensitivity biomarker polynucleotide to the level of the resistance biomarker polynucleotide; wherein a high ratio of sensitivity biomarker polynucleotide to resistance biomarker polynucleotide indicates a predicted high sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor. For this method, examples of useful polypeptides encoded by the sensitivity biomarker polynucleotide include Mitogen Activated Protein Kinase Kinase 6 (MAP2K6) and Aldehyde Dehydrogenase 1-A1 (ALDH1A1). For this method, examples of useful polypeptides encoded by the resistance biomarker polynucleotide include Metallothionein 1E (MT-1E) and Caldesmon (CALD1).","The present invention provides a method of assessing whether a cancer patient is afflicted with a cancer that will respond effectively to treatment with an IGF-1R kinase inhibitor, the method comprising comparing: the level of expression of a resistance biomarker in a patient sample; and the normal level of expression of the biomarker in a control non-cancer sample, wherein a significant increase in the level of expression of the resistance biomarker in the patient sample over the normal level is an indication that the patient is afflicted with a cancer which is less likely to respond effectively to treatment with an IGF-1R kinase inhibitor. For this method, examples of useful resistance biomarkers include Metallothionein 1E (MT-1E) and Caldesmon (CALD1), and nucleic acids encoding for these proteins.","The present invention provides a method of assessing whether a cancer patient is afflicted with a cancer that will respond effectively to treatment with an IGF-1R kinase inhibitor; the method comprising comparing: the level of expression of an sensitivity biomarker in a patient sample; and the normal level of expression of the biomarker in a control non-cancer sample, wherein a significant decrease in the level of expression of the sensitivity biomarker in the patient sample over the normal level is an indication that the patient is afflicted with a cancer which is less likely to respond effectively to treatment with an IGF-1R kinase inhibitor. For this method, examples of useful sensitivity biomarkers include Mitogen Activated Protein Kinase Kinase 6 (MAP2K6) and Aldehyde Dehydrogenase 1-A1 (ALDH1A1), and nucleic acids encoding for these proteins.","The present invention provides a method of assessing whether a cancer patient is afflicted with a cancer that will respond effectively to treatment with an IGF-1R kinase inhibitor, the method comprising comparing: the level of expression of an sensitivity biomarker in a patient sample; and the level of expression of a resistance biomarker in a patient sample, wherein a high ratio of the level of expression of the sensitivity biomarker to the level of expression of the resistance biomarker is an indication that the patient is afflicted with a cancer which is likely to respond effectively to treatment with an IGF-1R kinase inhibitor. For this method, examples of useful sensitivity biomarkers include Mitogen Activated Protein Kinase Kinase 6 (MAP2K6) and Aldehyde Dehydrogenase 1-A1 (ALDH1A1), and nucleic acids encoding for these proteins. For this method, examples of useful resistance biomarkers include Metallothionein 1E (MT-1E) and Caldesmon (CALD1), and nucleic acids encoding for these proteins.","In any of the above methods referring to a patient sample, an example of such a sample can be a tumor biopsy.","The present invention provides a method of determining whether in a human subject a tumor will be responsive to treatment with an IGF-1R kinase inhibitor, comprising: (a) collecting a sample of a bodily substance containing human nucleic acid or protein, said nucleic acid or protein having originated from cells of the human subject, (b) determining quantitatively or semi-quantitatively in the sample a level of expression for a sensitivity biomarker protein or a sensitivity biomarker mRNA; and (c) comparing the expression level in (b) to the level of biomarker expression in a normal control, or to the level of a control polypeptide or nucleic acid in the sample, wherein reduced expression of the sensitivity biomarker protein or the sensitivity biomarker mRNA, with respect to the control level, indicates the presence in the human subject of a tumor which is less likely to respond effectively to treatment with an IGF-1R kinase inhibitor.","The present invention provides a method of determining whether in a human subject a tumor will be responsive to treatment with an IGF-1R kinase inhibitor, comprising: (a) collecting a sample of a bodily substance containing human nucleic acid or protein, said nucleic acid or protein having originated from cells of the human subject, (b) determining quantitatively or semi-quantitatively in the sample a level of expression for a resistance biomarker protein or a resistance biomarker mRNA; and (c) comparing the expression level in (b) to the level of biomarker expression in a normal control, or to the level of a control polypeptide or nucleic acid in the sample, wherein increased expression of the resistance biomarker protein or the resistance biomarker mRNA, with respect to the control level, indicates the presence in the human subject of a tumor which is less likely to respond effectively to treatment with an IGF-1R kinase inhibitor.","The present invention provides a method of determining the likelihood that a patient with a tumor will show relatively long survival benefit from therapy with an IGF-1R kinase inhibitor, comprising determining the level of a sensitivity biomarker in the cells of the tumor, comparing said level with the level of sensitivity biomarker expression in a non-tumor control, or to the level of a control polypeptide or nucleic acid in the tumor sample, and determining whether the cells of the tumor contain a relatively high level of the sensitivity biomarker, a high level being indicative that a patient with a tumor will show relatively long survival benefit from therapy with an IGF-1R kinase inhibitor.","The present invention provides a method of determining the likelihood that a patient with a tumor will show relatively long survival benefit from therapy with an IGF-1R kinase inhibitor, comprising determining the level of a resistance biomarkers in the cells of the tumor, comparing said level with the level of resistance biomarker expression in a non-tumor control, or to the level of a control polypeptide or nucleic acid in the tumor sample, and determining whether the cells of the tumor contain a relatively low level of the resistance biomarker, a low level being indicative that a patient with a tumor will show relatively long survival benefit from therapy with an IGF-1R kinase inhibitor.","The present invention provides a method for determining for a patient with a tumor the likelihood that said patient will show relatively long survival benefit from therapy with an IGF-1R kinase inhibitor, comprising: determining the level of a sensitivity biomarker in the cells of the tumor, comparing said level with the level of sensitivity biomarker expression in a non-tumor control, or to the level of a control polypeptide or nucleic acid in the tumor sample, and determining whether the cells of the tumor contain a relatively high level of the sensitivity biomarkers; determining the level of a resistance biomarker in the cells of the tumor, comparing said level with the level of resistance biomarker expression in a non-tumor control, or to the level of a control polypeptide or nucleic acid in the tumor sample, and determining whether the cells of the tumor contain a relatively low level of the resistance biomarker, wherein a high level of the sensitivity biomarker and a low level of the resistance biomarker is indicative that a patient with a tumor will show relatively long survival benefit from therapy with an IGF-1R kinase inhibitor.","The present invention provides a method of determining a prognosis for survival for a patient with a neoplastic condition in response to therapy with an IGF-1R kinase inhibitor, comprising: measuring the level of an sensitivity biomarker associated with neoplastic cells, and comparing said level of sensitivity biomarker to a non-neoplastic sensitivity biomarker reference level, or to the level of a control polypeptide or nucleic acid associated with the neoplastic cells, wherein a decreased level of sensitivity biomarker associated with the neoplastic cells correlates with decreased survival of said patient.","The present invention provides a method of determining a prognosis for survival for a patient with a neoplastic condition in response to therapy with an IGF-1R kinase inhibitor, comprising: measuring the level of an resistance biomarker associated with neoplastic cells, and comparing said level of resistance biomarker to a non-neoplastic resistance biomarker reference level, or to the level of a control polypeptide or nucleic acid associated with the neoplastic cells, wherein an increased level of resistance biomarker associated with the neoplastic cells correlates with decreased survival of said patient.","The present invention also provides methods of predicting which tumor cells will be inhibited in a synergistic manner by treatment with a combination of an IGF-1R inhibitor and an anti-cancer agent, wherein the anticancer agent is SN38, oxaliplatin, or 5-Fluorouracil, by determining tumor cell sensitivity or resistance biomarker expression. Tumor cells predicted to be sensitive to an IGF-1R kinase inhibitor will also be inhibited in a synergistic manner by treatment with a combination of an IGF-1R inhibitor and an anti-cancer agent. Thus any of the methods described herein for predicting the sensitivity of tumor cells to an IGF-1R kinase inhibitor will also predict which tumor cells will be inhibited in a synergistic manner by treatment with a combination of an IGF-1R inhibitor and an anti-cancer agent, wherein the anticancer agent is SN38, oxaliplatin, or 5-Fluorouracil.","The present invention thus provides a method of predicting whether tumor cells of a patient will be inhibited in a synergistic manner by treatment with a combination of an IGF-1R inhibitor and an anti-cancer agent, wherein the anticancer agent is SN38, oxaliplatin, or 5-Fluorouracil, comprising: assessing the level of a resistance biomarker expressed by the tumor cell; and predicting the likelihood that tumor cell growth inhibition by the IGF-1R kinase inhibitor anti-cancer agent combination will be synergistic, wherein low levels of resistance biomarker expression by tumor cells correlates with a high probability that inhibition by the IGF-1R kinase inhibitor anti-cancer agent combination will be synergistic.","The present invention also provides a method of predicting whether tumor cells of a patient will be inhibited in a synergistic manner by treatment with a combination of an IGF-1R inhibitor and an anti-cancer agent, wherein the anticancer agent is SN38, oxaliplatin, or 5-Fluorouracil, comprising: assessing the level of a sensitivity biomarker expressed by the tumor cell; and predicting the likelihood that tumor cell growth inhibition by the IGF-1R kinase inhibitor anti-cancer agent combination will be synergistic, wherein high levels of resistance biomarker expression by tumor cells correlates with a high probability that inhibition by the IGF-1R kinase inhibitor anti-cancer agent combination will be synergistic.","The present invention also provides a method for treating tumors or tumor metastases in a patient with an IGF-1R kinase inhibitor chemotherapy combination, comprising the steps of: predicting whether the tumor cells of the patient will be inhibited in a synergistic manner by treatment with a combination of an IGF-1R kinase inhibitor and an anti-cancer agent, wherein the anticancer agent is SN38, oxaliplatin, or 5-Fluorouracil, by assessing the level of a resistance biomarker expressed by the tumor cells, wherein low levels of resistance biomarker expression by tumor cells correlates with a high probability that inhibition by the IGF-1R kinase inhibitor chemotherapy combination will be synergistic, and administering to said patient a therapeutically effective amount of a combination of an IGF-1R kinase inhibitor and one of SN38, oxaliplatin, or 5-Fluorouracil if the patient is diagnosed to be potentially responsive to the IGF-1R kinase inhibitor chemotherapy combination in a synergistic manner. In one embodiment of this method the IGF-1R kinase inhibitor administered to said patient is OSI-906. In another embodiment the tumor cells are NSCL, colon, breast, ovarian, head and neck, or pancreatic tumor cells.","The present invention also provides a method for treating tumors or tumor metastases in a patient with an IGF-1R kinase inhibitor chemotherapy combination, comprising the steps of: predicting whether the tumor cells of the patient will be inhibited in a synergistic manner by treatment with a combination of an IGF-1R kinase inhibitor and an anti-cancer agent, wherein the anticancer agent is SN38, oxaliplatin, or 5-Fluorouracil, by assessing the level of a sensitivity biomarker expressed by the tumor cells, wherein high levels of sensitivity biomarker expression by tumor cells correlates with a high probability that inhibition by the IGF-1R kinase inhibitor chemotherapy combination will be synergistic, and administering to said patient a therapeutically effective amount of a combination of an IGF-1R kinase inhibitor and one of SN38, oxaliplatin, or 5-Fluorouracil if the patient is diagnosed to be potentially responsive to the IGF-1R kinase inhibitor chemotherapy combination in a synergistic manner. In one embodiment the IGF-1R kinase inhibitor administered to said patient is OSI-906. In another embodiment the tumor cells are NSCL, colon, breast, ovarian, head and neck, or pancreatic tumor cells.","The present invention also provides a method of predicting whether tumor cells have developed resistance to inhibition by an IGF-1R kinase inhibitor, comprising: assessing the level of a resistance biomarker expressed by a tumor cell; and predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor, wherein high levels of the resistance biomarker expression by tumor cells correlates with low sensitivity to inhibition by IGF-1R kinase inhibitors, and thus resistance to inhibition by an IGF-1R kinase inhibitor. In one embodiment of this method the resistance biomarker is selected from caldesmon 1 (CALD1, GeneID: 800); kelch-like 5 () (KLHL5, GeneID: 51088); metallothionein 1E (functional) (MT1E, GeneID: 4493); beta-1,3-N-acetylgalactosaminyltransferase 1 (globoside blood group) (B3GALNT1, GeneID: 8706); cysteine-rich, angiogenic inducer, 61 (CYR61, GeneID: 3491); metallothionein 1X (MT1X, GeneID: 4501); troponin T type 1 (skeletal, slow) (TNNT1, GeneID: 7138); metallothionein 1H-like protein\/\/\/hypothetical protein LOC650610 (MT1P2, GeneID: 645745); metallothionein 1H (MT1H, GeneID: 4496); metallothionein 1F (functional) (MT1F, GeneID: 4494); metallothionein 2A (MT2A, GeneID: 4502); metallothionein 1M (MT1M, GeneID: 4499); MHC class I polypeptide-related sequence B (MICB, GeneID: 4277); collagen, type VI, alpha 1 (COL6A1, GeneID: 1291); myosin, light polypeptide 9, regulatory (MYL9, GeneID: 10398); and metallothionein 1G (MT1G, GeneID: 4495). The method can also be incorporated into a treatment regimen such that the method is used to predict if resistance to an IGF-1R kinase inhibitor has developed prior to administration of further doses of the IGF-1R kinase inhibitor being administered. If resistance has not developed then further doses of the IGF-1R kinase inhibitor are indicated, assuming that goals for efficacy and lack of toxicity are being met. If resistance has developed then further doses of the IGF-1R kinase inhibitor may be contra-indicated, and the physician may choose to treat with another anti-cancer agent or treatment.","The data presented in the Experimental Details section herein below, using the K-TSP algorithm as a discriminative classifier, identified three human gene pairs, (PROM1 (GeneID: 8842), MT1E (GeneID: 4493)), (LY75 (GeneID: 4065), OXCT1 (GeneID: 5019)) and (HSD17B2 (GeneID: 3294), CALD1 (GeneID: 800)), that can be utilized in predicting the sensitivity of tumor cell growth (e.g. colorectal tumor cells) to IGF-1R kinase inhibitors (e.g. PQIP, OSI-906). Thus, if the expression of the first member of each of these gene pairs is greater than the expression of the second member, tumor cell growth is likely to be sensitive to an IGF-1R kinase inhibitor.","The present invention thus provides a method of predicting the sensitivity of tumor cell growth to an IGF-1R kinase inhibitor, comprising: assessing the level of the gene MT1E expressed by the tumor cells; assessing the level of the gene PROM1 expressed by the tumor cells; determining whether the tumor cells express a higher level of PROM1 than MT1E; and predicting that tumor cell growth is likely to be sensitive to an IGF-1R kinase inhibitor if the tumor cells express a higher level of PROM1 than MT1E. This method may be utilized to select a cancer patient who is predicted to benefit from therapeutic administration of an IGF-1R kinase inhibitor, by applying it to a sample of the cells of a tumor of the patient (e.g. a tumor biopsy, or circulating tumor cells isolated from a blood sample), either alone, or in addition to other diagnostic tests to predict response to administration of an IGF-1R kinase inhibitor. The present invention thus provides a method of identifying patients with cancer who are most likely to benefit from treatment with an IGF-1R kinase inhibitor, comprising: obtaining a sample of the patient's tumor; assessing the level of the gene MT1E expressed by the tumor cells; assessing the level of the gene PROM1 expressed by the tumor cells; determining whether the tumor cells express a higher level of PROM1 than MT1E; and identifying the patient as one most likely to benefit from treatment with an IGF-1R kinase inhibitor if the tumor cells express a higher level of PROM1 than MT1E.","The present invention thus provides a method of predicting the sensitivity of tumor cell growth to an IGF-1R kinase inhibitor, comprising: assessing the level of the gene OXCT1 expressed by the tumor cells; assessing the level of the gene LY75 expressed by the tumor cells; determining whether the tumor cells express a higher level of LY75 than OXCT1; and predicting that tumor cell growth is likely to be sensitive to an IGF-1R kinase inhibitor if the tumor cells express a higher level of LY57 than OXCT1. This method may be utilized to select a cancer patient who is predicted to benefit from therapeutic administration of an IGF-1R kinase inhibitor, by applying it to a sample of the cells of a tumor of the patient (e.g. a tumor biopsy, or circulating tumor cells isolated from a blood sample), either alone, or in addition to other diagnostic tests to predict response to administration of an IGF-1R kinase inhibitor. The present invention thus provides a method of identifying patients with cancer who are most likely to benefit from treatment with an IGF-1R kinase inhibitor, comprising: obtaining a sample of the patient's tumor; assessing the level of the gene OXCT1 expressed by the tumor cells; assessing the level of the gene LY75 expressed by the tumor cells; determining whether the tumor cells express a higher level of LY75 than OXCT1; and identifying the patient as one most likely to benefit from treatment with an IGF-1R kinase inhibitor if the tumor cells express a higher level of LY75 than OXCT1.","The present invention thus provides a method of predicting the sensitivity of tumor cell growth to an IGF-1R kinase inhibitor, comprising: assessing the level of the gene CALD1 expressed by the tumor cells; assessing the level of the gene HSD17B2 expressed by the tumor cells; determining whether the tumor cells express a higher level of HSD17B2 than CALD1; and predicting that tumor cell growth is likely to be sensitive to an IGF-1R kinase inhibitor if the tumor cells express a higher level of HSD17B2 than CALD1. This method may be utilized to select a cancer patient who is predicted to benefit from therapeutic administration of an IGF-1R kinase inhibitor, by applying it to a sample of the cells of a tumor of the patient (e.g. a tumor biopsy, or circulating tumor cells isolated from a blood sample), either alone, or in addition to other diagnostic tests to predict response to administration of an IGF-1R kinase inhibitor. The present invention thus provides a method of identifying patients with cancer who are most likely to benefit from treatment with an IGF-1R kinase inhibitor, comprising: obtaining a sample of the patient's tumor; assessing the level of the gene CALD1 expressed by the tumor cells; assessing the level of the gene HSD17B2 expressed by the tumor cells; determining whether the tumor cells express a higher level of HSD17B2 than CALD1; and identifying the patient as one most likely to benefit from treatment with an IGF-1R kinase inhibitor if the tumor cells express a higher level of HSD17B2 than CALD1.","Determination of the gene expression level for each of the genes of the three gene pairs, (PROM1, MT1E), (LY75, OXCT1) and (HSD17B2, CALD1), can be accomplished by any of the methods known in the art for assessing the expression level of genes, as for example described herein for biomarkers. In a preferred embodiment, the level of mRNA expressed by the gene is determined. In an alternative embodiment, the level of protein expressed by the gene is determined.","The data presented in the Experimental Details section herein below demonstrates that tumor cells, such as CRC (colorectal cancer) cells, show a range of sensitivities to growth inhibition by an IGF-1R kinase inhibitor (e.g. PQIP, OSI-906) and that the degree of sensitivity of tumor cells to an IGF-1R kinase inhibitor can be assessed by determining the presence of mutant K-RAS in the tumor cells, such that the presence of mutant K-RAS is indicative that the cells are likely to have low sensitivity, or be relatively resistant, to growth inhibition by an IGF-1R kinase inhibitor, or conversely, the absence of mutant K-RAS (i.e. wild type K-RAS) is indicative that the cells are likely to have high sensitivity to growth inhibition by an IGF-1R kinase inhibitor.","The present invention thus provides a method of predicting the sensitivity of tumor cell growth to an IGF-1R kinase inhibitor, comprising: determining whether the tumor cells possess a mutant K-RAS gene; and predicting that tumor cell growth is likely to be sensitive to an IGF-1R kinase inhibitor if the tumor cells do not possess a mutant K-RAS gene. This method may be utilized to select a cancer patient who is predicted to benefit from therapeutic administration of an IGF-1R kinase inhibitor, by applying it to a sample of the cells of a tumor of the patient (e.g. a tumor biopsy, or circulating tumor cells isolated from a blood sample), either alone, or in addition to other diagnostic tests to predict response to administration of an IGF-1R kinase inhibitor. The present invention thus provides a method of identifying patients with cancer who are most likely to benefit from treatment with an IGF-1R kinase inhibitor, comprising: obtaining a sample of the patient's tumor; determining whether the tumor cells possess a mutant K-RAS gene; and identifying the patient as one most likely to benefit from treatment with an IGF-1R kinase inhibitor if the tumor cells do not possess a mutant K-RAS gene. The mutant K-RAS gene may be a K-RAS gene (GeneID: 3845) with an activating mutation at any one of seven known KRAS point mutations in codons 12 and 13 as follows: Gly12Asp (GGT>GAT), Gly12Ala (GGT>GCT), Gly12Val (GGT>GTT), Gly12Ser (GGT>AGT), Gly12Arg (GGT>CGT), Gly12Cys (GGT>TGT), and Gly13Asp (GGC>GAC)). Inherent in this method is the recognition that expression of a mutant KRAS gene in tumor cells correlates with lower sensitivity of the tumor cells to growth inhibition by an IGF-1R kinase inhibitor than tumor cells that have wild type KRAS.","In any of the methods described herein the IGF-1R kinase inhibitor may be a small molecule kinase inhibitor, such as a compound of Formula I as disclosed herein, such as for example OSI-906, or an anti-IGF-1R antibody. The tumor cells of the patient may be NSCL, colon, CRC, breast, ovarian, head and neck, or pancreatic tumor cells. In one embodiment, one or more additional anti-cancer agents may be co-administered simultaneously or sequentially with the IGF-1R kinase inhibitor. Such additional anti-cancer agents may for example comprise an EGFR kinase inhibitor (e.g. erlotinib, an anti-EGFR antibody), or any of the other anti-cancer agents described herein.","The data presented in the Experimental Details section herein below also demonstrates that tumor cells, such as CRC (colorectal cancer) cells, displaying an increase in IGF-1R gene copy number (or genomic gain, i.e. unbalanced genomic gain) are predicted to be especially responsive to treatment with IGF-1R kinase inhibitors (e.g. PQIP, OSI-906), and therefore patients having tumors with such cells are the best candidates for the use of this line of therapy. In contrast, patients having tumors with little or no gain in copy number of the IGF-1R gene are predicted to have a poorer outcome to treatment with IGF-1R kinase inhibitors, as their tumor cells are more resistant to inhibition. IGF-1R gene copy number thus represents an additional \u201csensitivity\u201d biomarker that can be used to predict the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor. Assessment or quantitation of the expression level or amount of this biomarker (i.e. IGF-1R gene copy number) in tumor cells can thus be used to assist with patient selection for potential treatment with an IGF-1R kinase inhibitor.","The present invention thus provides a method of predicting the sensitivity of tumor cell growth to an IGF-1R kinase inhibitor, comprising: assessing IGF-1R gene copy number in the tumor cells; determining if there is an increased IGF-1R gene copy number (i.e. unbalanced gain when normalized to ploidy); and predicting that tumor cell growth is likely to be sensitive to an IGF-1R kinase inhibitor if the tumor cells have increased IGF-1R gene copy number. The present invention also provides a method of identifying patients with cancer who are most likely to benefit from treatment with an IGF-1R kinase inhibitor, comprising: obtaining a sample of the patient's tumor; assessing IGF-1R gene copy number in the tumor cells; determining if there is an increased IGF-1R gene copy number relative to ploidy; and identifying the patient as one most likely to benefit from treatment with an IGF-1R kinase inhibitor if the tumor cells have increased IGF-1R gene copy number.","In the studies presented herein, IGF-1R gene copy number was studied by FISH because this method presents several advantages, although the practice of the present invention is not limited to this technique. FISH is DNA-based and can be successfully performed in fresh or preserved paraffin-embedded tumor samples. The technology is well established, has short turn-around in clinical cytogenetics and molecular pathology laboratories, and an IGF-1R FISH probe is available, and could readily be made commercially available. These results thus support the routine use of IGF-1R-FISH analysis and related techniques (e.g. CISH, SISH) for selecting patients for IGF-1R kinase inhibitor (e.g. OSI-906) therapy.","The methods and test kits provided by the present invention are extremely useful for patients with any cancer that can be treated with IGF-1R kinase inhibitors, such as NSCLC, CRC, breast cancer, ovarian cancer, head and neck cancer, pancreatic cancer etc. Such patients might, as a result of the methods provided herein, be spared from side effects and financial costs of an ineffective therapy in the event that they do not have genomic gain affecting the IGF-1R locus.","The copy number of genes in tumor cells according to the invention can be measured, for example in FISH assays, e.g. in nuclei. Such tests, as well as other detection methods, can be performed in primary tumors, metastatic tumors, locally recurring tumors, sputum, bronchial lavage, ascites, spinal fluid, or other tumoral settings. The markers can be measured in tumor specimens that are fresh, frozen, fixed or otherwise preserved. Quantitation of the gene copy number according to the present invention can also be accomplished using a variety of other types of hybridization assays well known in the art, and described herein, e.g. CISH (Chromogenic In Situ Hybridization), SISH (Silver In Situ Hybridization), PCR based techniques, genomic arrays including SNPs.","The nucleotide sequence of the human IGF-1R gene (GeneID: 3480) is known in the art and can be found under GenBank Accession No. NC000015 (incorporated herein by reference), for example. Nucleotide probes are also known in the art and available for use as probes to detect IGF-1R genes. For example, probes for detecting both IGF-1R and chromosome 15 centromere sequences are available, as described herein.","In the methods of this invention, the level or amount of IGF-1R gene copy number in the tumor cell sample may be compared to a control level of IGF-1R gene copy number selected from: (i) a control level that has been correlated with sensitivity to an IGF-1R kinase inhibitor (e.g. PQIP, OSI-906); and (ii) a control level that has been correlated with resistance to the IGF-1R kinase inhibitor. A patient is selected as being predicted to benefit from therapeutic administration of an IGF-1R kinase inhibitor, if the level of IGF-1R gene copy number in the patient's tumor cells is statistically similar to or greater than the control level of IGF-1R gene copy number that has been correlated with sensitivity to the IGF-1R kinase inhibitor, or if the level of IGF-1R gene copy number in the patient's tumor cells is statistically greater than the level of IGF-1R gene copy number that has been correlated with resistance to the IGF-1R kinase inhibitor. A patient is selected as being predicted to not benefit from therapeutic administration of an IGF-1R kinase inhibitor, if the level of IGF-1R gene copy number in the patient's tumor cells is statistically less than the control level of IGF-1R gene copy number that has been correlated with sensitivity to the IGF-1R kinase inhibitor, or if the level of IGF-1R gene copy number in the patient's tumor cells is statistically similar to or less than the level of IGF-1R gene copy number that has been correlated with resistance to the IGF-1R kinase inhibitor. The IGF-1R kinase inhibitor sensitive or resistant tumor cells listed in  are examples of cells that may be used for control levels of IGF-1R gene copy number. Preferred resistant tumor cells for determining control levels of IGF-1R gene copy number include for example HCT116, HCT15, HCT8, LS174T, and RKO. Preferred sensitive tumor cells for determining control levels of IGF-1R gene copy number include for example Colo205, HT29, CaCo2, and LS513.","More specifically, according to the present invention, a \u201ccontrol level\u201d is a control level of IGF-1R gene copy number, which can include a level that is correlated with sensitivity to the IGF-1R kinase inhibitor or a level that is correlated with resistance to the IGF-1R kinase inhibitor. Therefore, it can be determined, as compared to the control or baseline level of IGF-1R gene copy number, whether a patient sample is more likely to be sensitive to or resistant to the IGF-1R kinase inhibitor therapy (e.g., a good responder or responder (one who will benefit from the therapy), or a poor responder or non-responder (one who will not benefit or will have little benefit from the therapy)).","The method for establishing a control level of IGF-1R gene copy number is selected based on the sample type, the tissue or organ from which the sample is obtained, and the status of the patient to be evaluated. Preferably, the method is the same method that will be used to evaluate the sample in the patient. In a preferred embodiment, the control level is established using the same cell type as the cell to be evaluated. In a preferred embodiment, the control level is established from control samples that are from patients or cell lines known to be resistant or sensitive to an IGF-1R kinase inhibitor. In one aspect, the control samples are obtained from a population of matched individuals. According to the present invention, the phrase \u201cmatched individuals\u201d refers to a matching of the control individuals on the basis of one or more characteristics which are suitable for the type of cell or tumor growth to be evaluated. For example, control individuals can be matched with the patient to be evaluated on the basis of gender, age, race, or any relevant biological or sociological factor that may affect the baseline of the control individuals and the patient (e.g., preexisting conditions, consumption of particular substances, levels of other biological or physiological factors). To establish a control level, samples from a number of matched individuals are obtained and evaluated in the same manner as for the test samples. The number of matched individuals from whom control samples must be obtained to establish a suitable control level (e.g., a population) can be determined by those of skill in the art, but should be statistically appropriate to establish a suitable baseline for comparison with the patient to be evaluated (i.e., the test patient). The values obtained from the control samples are statistically processed using any suitable method of statistical analysis to establish a suitable baseline level using methods standard in the art for establishing such values.","It will be appreciated by those of skill in the art that a control level need not be established for each assay as the assay is performed but rather, a baseline or control can be established by referring to a form of stored information regarding a previously determined control level for sensitive and resistant patients (responders and non-responders), such as a control level established by any of the above-described methods. Such a form of stored information can include, for example, but is not limited to, a reference chart, listing or electronic file of population or individual data regarding sensitive and resistant tumors\/patients, or any other source of data regarding control level IGF-1R gene copy number that is useful for the patient to be evaluated. For example, one can use the guidelines established above and further described in the Experimental section for establishing increased IGF-1R gene copy number, which have already been correlated with responsiveness to an IGF-1R kinase inhibitor, to rate a given patient sample.","The invention thus provides a method to select a cancer patient who is predicted to benefit or not benefit from therapeutic administration of an IGF-1R kinase inhibitor, comprising: a) detecting in a sample of tumor cells from a patient a level of a biomarker selected from the group consisting of: i) an expression level or amount of IGF-1R gene copy number; ii) an expression level of a sensitivity biomarker; and iii) an expression level of a resistance biomarker; b) comparing the level of the biomarker in the tumor cell sample to a control level of the biomarker selected from the group consisting of: i) a control level of the biomarker that has been correlated with sensitivity to the IGF-1R kinase inhibitor; and ii) a control level of the biomarker that has been correlated with resistance to the IGF-1R kinase inhibitor; and c) selecting the patient as being predicted to benefit from therapeutic administration of the IGF-1R kinase inhibitor, if the level of the biomarker in the patient's tumor cells is statistically similar to or greater than the control level of the biomarker that has been correlated with sensitivity to the IGF-1R kinase inhibitor, or if the level of the biomarker in the patient's tumor cells is statistically greater than the level of the biomarker that has been correlated with resistance to the IGF-1R kinase inhibitor; or d) selecting the patient as being predicted to not benefit from therapeutic administration of the IGF-1R kinase inhibitor, if the level of the biomarker in the patient's tumor cells is statistically less than the control level of the biomarker that has been correlated with sensitivity to the IGF-1R kinase inhibitor, or if the level of the biomarker in the patient's tumor cells is statistically similar to or less than the level of the biomarker that has been correlated with resistance to the IGF-1R kinase inhibitor. In one embodiment the step of detecting in (a)(i) or (a)(ii) is performed using a nucleotide probe that hybridizes to the IGF-1R gene. In this embodiment, the step of detecting further comprises using a nucleotide probe that hybridizes to chromosome 15 centromere sequences. Alternatively, a chimeric nucleotide probe that hybridizes to the IGF-1R gene and to chromosome 15 centromere sequences may be used.","The invention further provides a method of treating tumors or tumor metastases in a patient, comprising (a) performing the steps of a method to select a cancer patient who is predicted to benefit or not benefit from therapeutic administration of an IGF-1R kinase inhibitor, and (b) administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor (e.g. OSI-906) if the patient is predicted to benefit from an IGF-1R kinase inhibitor, or administering to said patient an alternative therapy or no therapy if the patient is not predicted to benefit from administration of an IGF-1R kinase inhibitor.","The invention further provides a method of predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor, comprising: assessing the level of IGF-1R gene copy number in a tumor cell; and predicting the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor, wherein increased levels of tumor cell IGF-1R gene copy number correlate with high sensitivity to inhibition by IGF-1R kinase inhibitors.","The invention further provides a method for treating tumors or tumor metastases in a patient, comprising the steps of: diagnosing a patient's likely responsiveness to an IGF-1R kinase inhibitor by assessing the IGF-1R gene copy number of the tumor cells of the patient, wherein increased levels of IGF-1R gene copy number in the tumor cells correlates with high sensitivity to inhibition by IGF-1R kinase inhibitors, and administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor if the patient is diagnosed to be potentially responsive to an IGF-1R kinase inhibitor. In one embodiment the IGF-1R kinase inhibitor administered to said patient is OSI-906. In another embodiment of the method the tumor cells are NSCL, colon, breast, ovarian, head and neck, or pancreatic tumor cells.","Another embodiment of the invention includes an assay kit for performing any of the methods of the present invention. The assay kit can include a means for detecting in a sample of tumor cells a level of IGF-1R gene copy number, or other biomarker. The assay kit preferably also includes one or more controls. The controls could include: (i) a control sample for detecting sensitivity to the IGF-1R kinase inhibitor being evaluated for use in a patient; (ii) a control sample for detecting resistance to the IGF-1R kinase inhibitor; (iii) information containing a predetermined control level of particular biomarker to be measured with regard to IGF-1R kinase inhibitor sensitivity or resistance (e.g., a predetermined control level of IGF-1R gene copy number, or other biomarker, that has been correlated with sensitivity to the IGF-1R kinase inhibitor or resistance to IGF-1R kinase inhibitor).","The data presented in the Experimental Details section herein below demonstrates that several of the classifiers of sensitivity of tumor cell growth to IGF-1R kinase inhibitors can be integrated together to develop a signature of sensitivity to such inhibitors. These classifiers, or characteristics, of tumor cells that have high sensitivity include the following five classifiers: higher expression level of the PROM1 gene than the MT1E gene; higher expression level of the LY75 gene than the OXCT1 gene; higher expression level of the HSD17B2 gene than the CALD1 gene; IGF-1R gene copy number (i.e. unbalanced gain when normalized to ploidy); and the absence of a mutant K-RAS gene. For an integrated genomic classifier, in order for a tumor cell to predict as sensitive to an IGF-1R kinase inhibitor, at least four of these classifiers must be present in the tumor cells. This integrated genomic classifier was able to correctly predict the IGF-1R kinase inhibitor sensitivity of test tumor cell lines with 89% success rate, and of test human tumor explants with 100% success rate, a superior result to that achieved with any of the individual predictors.","Accordingly, the invention further provides a method of identifying patients with cancer who are most likely to benefit from treatment with an IGF-1R kinase inhibitor, comprising: (1) obtaining a sample of the patient's tumor, (2) determining if tumor cells of the sample exhibit the following classifiers of tumor cells that are more likely to be sensitive to growth inhibition by an IGF-1R kinase inhibitor: (a) higher expression level of the PROM1 gene than the MT1E gene; (b) higher expression level of the LY75 gene than the OXCT1 gene; (c) higher expression level of the HSD17B2 gene than the CALD1 gene; (d) increased IGF-1R gene copy number (i.e. unbalanced gain when normalized to ploidy); (e) the absence of a mutant K-RAS gene; and (3) identifying the patient as one most likely to benefit from treatment with an IGF-1R kinase inhibitor if at least four of the five assessed characteristics are present in the tumor cells. In step 1, obtaining a sample of the patient's tumor may be accomplished, for example, by performing a tumor biopsy, or isolating circulating tumor cells from a blood sample.","The invention further provides the use of the preceding method as part of a treatment regimen in order to determine which patients would most benefit from administration of an IGF-1R kinase inhibitor. Accordingly, the invention further provides a method for treating cancer in a patient, comprising the steps of: (A) diagnosing a patient's likely responsiveness to an IGF-1R kinase inhibitor by determining if the patient has a tumor that is likely to respond to treatment with an IGF-1R kinase inhibitor, by (1) obtaining a sample of the patient's tumor, (2) determining if tumor cells of the sample exhibit the following classifiers of tumor cells that are more likely to be sensitive to growth inhibition by an IGF-1R kinase inhibitor: (a) higher expression level of the PROM1 gene than the MT1E gene; (b) higher expression level of the LY75 gene than the OXCT1 gene; (c) higher expression level of the HSD17B2 gene than the CALD1 gene; (d) increased IGF-1R gene copy number (i.e. unbalanced gain when normalized to ploidy); (e) the absence of a mutant K-RAS gene; and (3) identifying the patient as having a tumor that is likely to likely to respond to treatment with an IGF-1R kinase inhibitor if at least four of the five assessed characteristics are present; and (B) administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor if the patient is diagnosed to be potentially responsive to an IGF-1R kinase inhibitor.","The invention further provides a method for treating cancer in a patient, comprising the steps of: (A) diagnosing a patient's likely responsiveness to an IGF-1R kinase inhibitor by determining if the patient has a tumor that is likely to respond to treatment with an IGF-1R kinase inhibitor, by: obtaining a sample of the patient's tumor, determining if tumor cells of the sample exhibit the following classifiers of tumor cells that are more likely to be sensitive to growth inhibition by an IGF-1R kinase inhibitor: (a) higher expression level of the PROM1 gene than the MT1E gene; (b) higher expression level of the LY75 gene than the OXCT1 gene; (c) higher expression level of the HSD17B2 gene than the CALD1 gene; (d) increased levels of IGF-1R gene copy number relative to ploidy; (e) the absence of a mutant K-RAS gene; and identifying the patient as having a tumor that is likely to respond to treatment with an IGF-1R kinase inhibitor if at least four of the five assessed classifiers are present in the tumor cells; and (B) administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor if the patient is diagnosed to be potentially responsive to an IGF-1R kinase inhibitor.","The invention further provides an alternative integrated genomic classifier, wherein in order for a tumor cell to predict as sensitive to an IGF-1R kinase inhibitor, at least three of the five classifiers must be present. The invention further provides an integrated genomic classifier, wherein in order for a tumor cell to predict as sensitive to an IGF-1R kinase inhibitor, at least two of the five classifiers must be present. The invention thus provides the corresponding methods using these alternative integrated genomic classifiers, for identifying patients with cancer who are most likely to benefit from treatment with an IGF-1R kinase inhibitor, and methods for treating cancer in a patient, as described above for the integrated genomic classifier.","The invention further provides a method for treating cancer in a patient, comprising the steps of: (A) diagnosing a patient's likely responsiveness to an IGF-1R kinase inhibitor by determining if the patient has a tumor that is likely to respond to treatment with an IGF-1R kinase inhibitor by: obtaining a sample of the patient's tumor; assessing the level of the gene MT1E expressed by the tumor cells; assessing the level of the gene PROM1 expressed by the tumor cells; determining whether the tumor cells express a higher level of PROM1 than MT1E; and identifying the patient as likely to benefit from treatment with an IGF-1R kinase inhibitor if the tumor cells express a higher level of PROM1 than MT1E, and (B) administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor if the patient is diagnosed to be potentially responsive to an IGF-1R kinase inhibitor.","The invention further provides a method for treating cancer in a patient, comprising the steps of: (A) diagnosing a patient's likely responsiveness to an IGF-1R kinase inhibitor by determining if the patient has a tumor that is likely to respond to treatment with an IGF-1R kinase inhibitor by: obtaining a sample of the patient's tumor; assessing the level of the gene OXCT1 expressed by the tumor cells; assessing the level of the gene LY75 expressed by the tumor cells; determining whether the tumor cells express a higher level of LY75 than OXCT1; and identifying the patient as likely to benefit from treatment with an IGF-1R kinase inhibitor if the tumor cells express a higher level of LY75 than OXCT1, and (B) administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor if the patient is diagnosed to be potentially responsive to an IGF-1R kinase inhibitor.","The invention further provides a method for treating cancer in a patient, comprising the steps of: (A) diagnosing a patient's likely responsiveness to an IGF-1R kinase inhibitor by determining if the patient has a tumor that is likely to respond to treatment with an IGF-1R kinase inhibitor by: obtaining a sample of the patient's tumor; assessing the level of the gene CALD1 expressed by the tumor cells; assessing the level of the gene HSD17B2 expressed by the tumor cells; determining whether the tumor cells express a higher level of HSD17B2 than CALD1; and identifying the patient as likely to benefit from treatment with an IGF-1R kinase inhibitor if the tumor cells express a higher level of HSD17B2 than CALD1, and (B) administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor if the patient is diagnosed to be potentially responsive to an IGF-1R kinase inhibitor.","The invention further provides a method for treating cancer in a patient, comprising the steps of: (A) diagnosing a patient's likely responsiveness to an IGF-1R kinase inhibitor by determining if the patient has a tumor that is likely to respond to treatment with an IGF-1R kinase inhibitor by: obtaining a sample of the patient's tumor; assessing IGF-1R gene copy number in the tumor cells; determining if there is an increased IGF-1R gene copy number relative to ploidy; and identifying the patient as likely to benefit from treatment with an IGF-1R kinase inhibitor if the tumor cells have increased IGF-1R gene copy number, and (B) administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor if the patient is diagnosed to be potentially responsive to an IGF-1R kinase inhibitor.","The invention further provides a method for treating cancer in a patient, comprising the steps of: (A) diagnosing a patient's likely responsiveness to an IGF-1R kinase inhibitor by determining if the patient has a tumor that is likely to respond to treatment with an IGF-1R kinase inhibitor by: obtaining a sample of the patient's tumor; determining whether the tumor cells possess a mutant K-RAS gene; and identifying the patient as likely to benefit from treatment with an IGF-1R kinase inhibitor if the tumor cells do not possess a mutant K-RAS gene, and (B) administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor if the patient is diagnosed to be potentially responsive to an IGF-1R kinase inhibitor.","For assessment of tumor cell sensitivity or resistance biomarker expression, patient samples containing tumor cells, or proteins or nucleic acids produced by these tumor cells, may be used in the methods of the present invention. In these embodiments, the level of expression of the biomarker can be assessed by assessing the amount (e.g. absolute amount or concentration) of the marker in a tumor cell sample, e.g., a tumor biopsy obtained from a patient, or other patient sample containing material derived from the tumor (e.g. blood, serum, urine, or other bodily fluids or excretions as described herein above). The cell sample can, of course, be subjected to a variety of well-known post-collection preparative and storage techniques (e.g., nucleic acid and\/or protein extraction, fixation, storage, freezing, ultrafiltration, concentration, evaporation, centrifugation, etc.) prior to assessing the amount of the marker in the sample. Likewise, tumor biopsies may also be subjected to post-collection preparative and storage techniques, e.g., fixation.","In the methods of the invention, one can detect expression of biomarker proteins having at least one portion which is displayed on the surface of tumor cells which express it. It is a simple matter for the skilled artisan to determine whether a marker protein, or a portion thereof, is exposed on the cell surface. For example, immunological methods may be used to detect such proteins on whole cells, or well known computer-based sequence analysis methods may be used to predict the presence of at least one extracellular domain (i.e. including both secreted proteins and proteins having at least one cell-surface domain). Expression of a marker protein having at least one portion which is displayed on the surface of a cell which expresses it may be detected without necessarily lysing the tumor cell (e.g. using a labeled antibody which binds specifically with a cell-surface domain of the protein).","Expression of a biomarkers described in this invention may be assessed by any of a wide variety of well known methods for detecting expression of a transcribed nucleic acid or protein. Non-limiting examples of such methods include immunological methods for detection of secreted, cell-surface, cytoplasmic, or nuclear proteins, protein purification methods, protein function or activity assays, nucleic acid hybridization methods, nucleic acid reverse transcription methods, and nucleic acid amplification methods.","In one embodiment, expression of a biomarker is assessed using an antibody (e.g. a radio-labeled, chromophore-labeled, fluorophore-labeled, or enzyme-labeled antibody), an antibody derivative (e.g. an antibody conjugated with a substrate or with the protein or ligand of a protein-ligand pair {e.g. biotin-streptavidin}), or an antibody fragment (e.g. a single-chain antibody, an isolated antibody hypervariable domain, etc.) which binds specifically with a biomarker protein or fragment thereof, including a biomarker protein which has undergone either all or a portion of post-translational modifications to which it is normally subjected in the tumor cell (e.g. glycosylation, phosphorylation, methylation etc.).","In another embodiment, expression of a biomarker is assessed by preparing mRNA\/cDNA (i.e. a transcribed polynucleotide) from cells in a patient sample, and by hybridizing the mRNA\/cDNA with a reference polynucleotide which is a complement of a biomarker nucleic acid, or a fragment thereof. cDNA can, optionally, be amplified using any of a variety of polymerase chain reaction methods prior to hybridization with the reference polynucleotide. Expression of biomarkers can likewise be detected using quantitative PCR to assess the level of expression of the biomarker(s). Alternatively, any of the many known methods of detecting mutations or variants (e.g. single nucleotide polymorphisms, deletions, etc.) of a biomarker of the invention may be used to detect occurrence of a biomarker in a patient.","In a related embodiment, a mixture of transcribed polynucleotides obtained from the sample is contacted with a substrate having fixed thereto a polynucleotide complementary to or homologous with at least a portion (e.g. at least 7, 10, 15, 20, 25, 30, 40, 50, 100, 500, or more nucleotide residues) of a biomarker nucleic acid. If polynucleotides complementary to or homologous with are differentially detectable on the substrate (e.g. detectable using different chromophores or fluorophores, or fixed to different selected positions), then the levels of expression of a plurality of biomarkers can be assessed simultaneously using a single substrate (e.g. a \u201cgene chip\u201d microarray of polynucleotides fixed at selected positions). When a method of assessing biomarker expression is used which involves hybridization of one nucleic acid with another, it is preferred that the hybridization be performed under stringent hybridization conditions.","When a plurality of biomarkers of the invention are used in the methods of the invention, the level of expression of each biomarker in a patient sample can be compared with the normal level of expression of each of the plurality of biomarkers in non-cancerous samples of the same type (or with biomarker levels in a control cell), either in a single reaction mixture (i.e. using reagents, such as different fluorescent probes, for each biomarker) or in individual reaction mixtures corresponding to each of the biomarkers.","The level of expression of a biomarker in normal (i.e. non-cancerous) human tissue can be assessed in a variety of ways. In one embodiment, this normal level of expression is assessed by assessing the level of expression of the biomarker in a portion of cells which appears to be non-cancerous, and then comparing this normal level of expression with the level of expression in a portion of the tumor cells. Alternately, and particularly as further information becomes available as a result of routine performance of the methods described herein, population-average values for normal expression of the biomarkers of the invention may be used. In other embodiments, the \u2018normal\u2019 level of expression of a biomarker may be determined by assessing expression of the biomarker in a patient sample obtained from a non-cancer-afflicted patient, from a patient sample obtained from a patient before the suspected onset of cancer in the patient, from archived patient samples, and the like.","An exemplary method for detecting the presence or absence of a biomarker protein or nucleic acid in a biological sample involves obtaining a biological sample (e.g. a tumor-associated body fluid) from a test subject and contacting the biological sample with a compound or an agent capable of detecting the polypeptide or nucleic acid (e.g., mRNA, cDNA). The detection methods of the invention can thus be used to detect mRNA, protein, or cDNA, for example, in a biological sample in vitro as well as in vivo. For example, in vitro techniques for detection of mRNA include Northern hybridizations and in situ hybridizations. In vitro techniques for detection of a biomarker protein include enzyme linked immunosorbent assays (ELISAs), Western blots, immunoprecipitations and immunofluorescence. In vitro techniques for detection of genomic DNA include Southern hybridizations. In vivo techniques for detection of mRNA include polymerase chain reaction (PCR), Northern hybridizations and in situ hybridizations. Furthermore, in vivo techniques for detection of a biomarker protein include introducing into a subject a labeled antibody directed against the protein or fragment thereof. For example, the antibody can be labeled with a radioactive marker whose presence and location in a subject can be detected by standard imaging techniques.","A general principle of such diagnostic and prognostic assays involves preparing a sample or reaction mixture that may contain a biomarker, and a probe, under appropriate conditions and for a time sufficient to allow the biomarker and probe to interact and bind, thus forming a complex that can be removed and\/or detected in the reaction mixture. These assays can be conducted in a variety of ways.","For example, one method to conduct such an assay would involve anchoring the biomarker or probe onto a solid phase support, also referred to as a substrate, and detecting target biomarker\/probe complexes anchored on the solid phase at the end of the reaction. In one embodiment of such a method, a sample from a subject, which is to be assayed for presence and\/or concentration of biomarker, can be anchored onto a carrier or solid phase support. In another embodiment, the reverse situation is possible, in which the probe can be anchored to a solid phase and a sample from a subject can be allowed to react as an unanchored component of the assay.","There are many established methods for anchoring assay components to a solid phase. These include, without limitation, biomarker or probe molecules which are immobilized through conjugation of biotin and streptavidin. Such biotinylated assay components can be prepared from biotin-NHS (N-hydroxy-succinimide) using techniques known in the art (e.g., biotinylation kit, Pierce Chemicals, Rockford, Ill.), and immobilized in the wells of streptavidin-coated 96 well plates (Pierce Chemical). In certain embodiments, the surfaces with immobilized assay components can be prepared in advance and stored.","Other suitable carriers or solid phase supports for such assays include any material capable of binding the class of molecule to which the biomarker or probe belongs. Well-known supports or carriers include, but are not limited to, glass, polystyrene, nylon, polypropylene, nylon, polyethylene, dextran, amylases, natural and modified celluloses, polyacrylamides, gabbros, and magnetite.","In order to conduct assays with the above mentioned approaches, the non-immobilized component is added to the solid phase upon which the second component is anchored. After the reaction is complete, uncomplexed components may be removed (e.g., by washing) under conditions such that any complexes formed will remain immobilized upon the solid phase. The detection of biomarker\/probe complexes anchored to the solid phase can be accomplished in a number of methods outlined herein.","In one embodiment, the probe, when it is the unanchored assay component, can be labeled for the purpose of detection and readout of the assay, either directly or indirectly, with detectable labels discussed herein and which are well-known to one skilled in the art.","It is also possible to'directly detect biomarker\/probe complex formation without further manipulation or labeling of either component (biomarker or probe), for example by utilizing the technique of fluorescence energy transfer (i.e. FET, see for example, Lakowicz et al., U.S. Pat. No. 5,631,169; Stavrianopoulos, et al., U.S. Pat. No. 4,868,103). A fluorophore label on the first, \u2018donor\u2019 molecule is selected such that, upon excitation with incident light of appropriate wavelength, its emitted fluorescent energy will be absorbed by a fluorescent label on a second \u2018acceptor\u2019 molecule, which in turn is able to fluoresce due to the absorbed energy. Alternately, the \u2018donor\u2019 protein molecule may simply utilize the natural fluorescent energy of tryptophan residues. Labels are chosen that emit different wavelengths of light, such that the \u2018acceptor\u2019 molecule label may be differentiated from that of the \u2018donor\u2019. Since the efficiency of energy transfer between the labels is related to the distance separating the molecules, spatial relationships between the molecules can be assessed. In a situation in which binding occurs between the molecules, the fluorescent emission of the \u2018acceptor\u2019 molecule label in the assay should be maximal. An FET binding event can be conveniently measured through standard fluorometric detection means well known in the art (e.g., using a fluorimeter).","In another embodiment, determination of the ability of a probe to recognize a biomarker can be accomplished without labeling either assay component (probe or biomarker) by utilizing a technology such as real-time Biomolecular Interaction Analysis (BIA) (see, e.g., Sjolander, S. and Urbaniczky, C., 1991, Anal. Chem. 63:2338-2345 and Szabo et al., 1995, Curr. Opin. Struct. Biol. 5:699-705). As used herein, \u201cBIA\u201d or \u201csurface plasmon resonance\u201d is a technology for studying biospecific interactions in real time, without labeling any of the interactants (e.g., BIAcore). Changes in the mass at the binding surface (indicative of a binding event) result in alterations of the refractive index of light near the surface (the optical phenomenon of surface plasmon resonance (SPR)), resulting in a detectable signal which can be used as an indication of real-time reactions between biological molecules.","Alternatively, in another embodiment, analogous diagnostic and prognostic assays can be conducted with biomarker and probe as solutes in a liquid phase. In such an assay, the complexed biomarker and probe are separated from uncomplexed components by any of a number of standard techniques, including but not limited to: differential centrifugation, chromatography, electrophoresis and immunoprecipitation. In differential centrifugation, biomarker\/probe complexes may be separated from uncomplexed assay components through a series of centrifugal steps, due to the different sedimentation equilibria of complexes based on their different sizes and densities (see, for example, Rivas, G., and Minton, A. P., 1993, Trends Biochem Sci. 18(8):284-7). Standard chromatographic techniques may also be utilized to separate complexed molecules from uncomplexed ones. For example, gel filtration chromatography separates molecules based on size, and through the utilization of an appropriate gel filtration resin in a column format, for example, the relatively larger complex may be separated from the relatively smaller uncomplexed components. Similarly, the relatively different charge properties of the biomarker\/probe complex as compared to the uncomplexed components may be exploited to differentiate the complex from uncomplexed components, for example through the utilization of ion-exchange chromatography resins. Such resins and chromatographic techniques are well known to one skilled in the art (see, e.g., Heegaard, N. H., 1998, J. Mol. Recognit. Winter 11(1-6):141-8; Hage, D. S., and Tweed, S. A. J. Chromatogr B Biomed Sci Appl 1997 Oct. 10; 699(1-2):499-525). Gel electrophoresis may also be employed to separate complexed assay components from unbound components (see, e.g., Ausubel et al., ed., Current Protocols in Molecular Biology, John Wiley & Sons, New York, 1987-1999). In this technique, protein or nucleic acid complexes are separated based on size or charge, for example. In order to maintain the binding interaction during the electrophoretic process, non-denaturing gel matrix materials and conditions in the absence of reducing agent are typically preferred. Appropriate conditions to the particular assay and components thereof will be well known to one skilled in the art.","In a particular embodiment, the level of biomarker mRNA can be determined both by in situ and by in vitro formats in a biological sample using methods known in the art. The term \u201cbiological sample\u201d is intended to include tissues, cells, biological fluids and isolates thereof, isolated from a subject, as Well as tissues, cells and fluids present within a subject. Many expression detection methods use isolated RNA. For in vitro methods, any RNA isolation technique that does not select against the isolation of mRNA can be utilized for the purification of RNA from tumor cells (see, e.g., Ausubel et al., ed., Current Protocols in Molecular Biology, John Wiley & Sons, New York 1987-1999). Additionally, large numbers of tissue samples can readily be processed using techniques well known to those of skill in the art, such as, for example, the single-step RNA isolation process of Chomczynski (1989, U.S. Pat. No. 4,843,155).","The isolated mRNA can be used in hybridization or amplification assays that include, but are not limited to, Southern or Northern analyses, polymerase chain reaction analyses and probe arrays. One preferred diagnostic method for the detection of mRNA levels involves contacting the isolated mRNA with a nucleic acid molecule (probe) that can hybridize to the mRNA encoded by the gene being detected. The nucleic acid probe can be, for example, a full-length cDNA, or a portion thereof, such as an oligonucleotide of at least 7, 15, 30, 50, 100, 250 or 500 nucleotides in length and sufficient to specifically hybridize under stringent conditions to a mRNA or genomic DNA encoding a biomarker of the present invention. Other suitable probes for use in the diagnostic assays of the invention are described herein. Hybridization of an mRNA with the probe indicates that the biomarker in question is being expressed.","In one format, the mRNA is immobilized on a solid surface and contacted with a probe, for example by running the isolated mRNA on an agarose gel and transferring the mRNA from the gel to a membrane, such as nitrocellulose. In an alternative format, the probe(s) are immobilized on a solid surface and the mRNA is contacted with the probe(s), for example, in an Affymetrix gene chip array. A skilled artisan can readily adapt known mRNA detection methods for use in detecting the level of mRNA encoded by the biomarkers of the present invention.","An alternative method for determining the level of mRNA biomarker in a sample involves the process of nucleic acid amplification, e.g., by RT-PCR (the experimental embodiment set forth in Mullis, 1987, U.S. Pat. No. 4,683,202), ligase chain reaction (Barany, 1991, Proc. Natl. Acad. Sci. USA, 88:189-193), self sustained sequence replication (Guatelli et al., 1990, Proc. Natl. Acad. Sci. USA 87:1874-1878), transcriptional amplification system (Kwoh et al., 1989, Proc. Natl. Acad. Sci. USA 86:1173-1177), Q-Beta Replicase (Lizardi et al., 1988, Bio\/Technology 6:1197), rolling circle replication (Lizardi et al., U.S. Pat. No. 5,854,033) or any other nucleic acid amplification method, followed by the detection of the amplified molecules using techniques well known to those of skill in the art. These detection schemes are especially useful for the detection of nucleic acid molecules if such molecules are present in very low numbers. As used herein, amplification primers are defined as being a pair of nucleic acid molecules that can anneal to 5\u2032 or 3\u2032 regions of a gene (plus and minus strands, respectively, or vice-versa) and contain a short region in between. In general, amplification primers are from about 10 to 30 nucleotides in length and flank a region from about 50 to 200 nucleotides in length. Under appropriate conditions and with appropriate reagents, such primers permit the amplification of a nucleic acid molecule comprising the nucleotide sequence flanked by the primers.","For in situ methods, mRNA does not need to be isolated from the tumor cells prior to detection. In such methods, a cell or tissue sample is prepared\/processed using known histological methods. The sample is then immobilized on a support, typically a glass slide, and then contacted with a probe that can hybridize to mRNA that encodes the biomarker.","As an alternative to making determinations based on the absolute expression level of the biomarker, determinations may be based on the normalized expression level of the biomarker. Expression levels are normalized by correcting the absolute expression level of a biomarker by comparing its expression to the expression of a gene that is not a biomarker, e.g., a housekeeping gene that is constitutively expressed. Suitable genes for normalization include housekeeping genes such as the actin gene, or a tumor cell-specific gene that is expressed at a constant level in the tumor cell type of interest. This normalization allows the comparison of the expression level in one sample, e.g., a patient sample, to another sample, e.g., a non-tumor sample, a control sample, or between samples from different sources.","Alternatively, the expression level can be provided as a relative expression level. To determine a relative expression level of a biomarker (e.g. a resistance biomarker), the level of expression of the biomarker is determined for 10 or more samples of normal versus cancer cell isolates (or resistant cell versus sensitive cell isolates), preferably 50 or more samples, prior to the determination of the expression level for the sample in question. The mean expression level of each of the genes assayed in the larger number of samples is determined and this is used as a baseline expression level for the biomarker. The expression level of the biomarker determined for the test sample (absolute level of expression) is then divided by the mean expression value obtained for that biomarker. This provides a relative expression level.","In another embodiment of the present invention, a biomarker protein is detected. A preferred agent for detecting biomarker protein of the invention is an antibody capable of binding to such a protein or a fragment thereof, preferably an antibody with a detectable label. Antibodies can be polyclonal, or more preferably, monoclonal. An intact antibody, or a fragment or derivative thereof (e.g., Fab or F(ab\u2032).sub.2) can be used. The term \u201clabeled\u201d, with regard to the probe or antibody, is intended to encompass direct labeling of the probe or antibody by coupling (i.e., physically linking) a detectable substance to the probe or antibody, as well as indirect labeling of the probe or antibody by reactivity with another reagent that is directly labeled. Examples of indirect labeling include detection of a primary antibody using a fluorescently labeled secondary antibody and end-labeling of a DNA probe with biotin such that it can be detected with fluorescently labeled streptavidin.","Proteins from tumor cells can be isolated using techniques that are well known to those of skill in the art. The protein isolation methods employed can, for example, be such as those described in Harlow and Lane (Harlow and Lane, 1988, Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.).","A variety of formats can be employed to determine whether a sample contains a protein that binds to a given antibody. Examples of such formats include, but are not limited to, enzyme immunoassay (EIA), radioimmunoassay (RIA), Western blot analysis and enzyme linked immunoabsorbent assay (ELISA). A skilled artisan can readily adapt known protein\/antibody detection methods for use in determining whether tumor cells express a biomarker of the present invention.","In one format, antibodies, or antibody fragments or derivatives, can be used in methods such as Western blots or immunofluorescence techniques to detect the expressed proteins. In such uses, it is generally preferable to immobilize either the antibody or proteins on a solid support. Suitable solid phase supports or carriers include any support capable of binding an antigen or an antibody. Well-known supports or carriers include glass, polystyrene, polypropylene, polyethylene, dextran, nylon, amylases, natural and modified celluloses, polyacrylamides, gabbros, and magnetite.","One skilled in the art will know many other suitable carriers for binding antibody or antigen, and will be able to adapt such support for use with the present invention. For example, protein isolated from tumor cells can be run on a polyacrylamide gel electrophoresis and immobilized onto a solid phase support such as nitrocellulose. The support can then be washed with suitable buffers followed by treatment with the detectably labeled antibody. The solid phase support can then be washed with the buffer a second time to remove unbound antibody. The amount of bound label on the solid support can then be detected by conventional means.","For ELISA assays, specific binding pairs can be of the immune or non-immune type. Immune specific binding pairs are exemplified by antigen-antibody systems or hapten\/anti-hapten systems. There can be mentioned fluorescein\/anti-fluorescein, dinitrophenyl\/anti-dinitrophenyl, biotin\/anti-biotin, peptide\/anti-peptide and the like. The antibody member of the specific binding pair can be produced by customary methods familiar to those skilled in the art. Such methods involve immunizing an animal with the antigen member of the specific binding pair. If the antigen member of the specific binding pair is not immunogenic, e.g., a hapten, it can be covalently coupled to a carrier protein to render it immunogenic. Non-immune binding pairs include systems wherein the two components share a natural affinity for each other but are not antibodies. Exemplary non-immune pairs are biotin-streptavidin, intrinsic factor-vitamin B, folic acid-folate binding protein and the like.","A variety of methods are available to covalently label antibodies with members of specific binding pairs. Methods are selected based upon the nature of the member of the specific binding pair, the type of linkage desired, and the tolerance of the antibody to various conjugation chemistries. Biotin can be covalently coupled to antibodies by utilizing commercially available active derivatives. Some of these are biotin-N-hydroxy-succinimide which binds to amine groups on proteins; biotin hydrazide which binds to carbohydrate moieties, aldehydes and carboxyl groups via a carbodiimide coupling; and biotin maleimide and iodoacetyl biotin which bind to sulfhydryl groups. Fluorescein can be coupled to protein amine groups using fluorescein isothiocyanate. Dinitrophenyl groups can be coupled to protein amine groups using 2,4-dinitrobenzene sulfate or 2,4-dinitrofluorobenzene. Other standard methods of conjugation can be employed to couple monoclonal antibodies to a member of a specific binding pair including dialdehyde, carbodiimide coupling, homofunctional crosslinking, and heterobifunctional crosslinking. Carbodiimide coupling is an effective method of coupling carboxyl groups on one substance to amine groups on another. Carbodiimide coupling is facilitated by using the commercially available reagent 1-ethyl-3-(dimethyl-aminopropyl)-carbodiimide (EDAC).","Homobifunctional crosslinkers, including the bifunctional imidoesters and bifunctional N-hydroxysuccinimide esters, are commercially available and are employed for coupling amine groups on one substance to amine groups on another. Heterobifunctional crosslinkers are reagents which possess different functional groups. The most common commercially available heterobifunctional crosslinkers have an amine reactive N-hydroxysuccinimide ester as one functional group, and a sulfhydryl reactive group as the second functional group. The most common sulfhydryl reactive groups are maleimides, pyridyl disulfides and active halogens. One of the functional groups can be a photoactive aryl nitrene, which upon irradiation reacts with a variety of groups.","The detectably-labeled antibody or detectably-labeled member of the specific binding pair is prepared by coupling to a reporter, which can be a radioactive isotope, enzyme, fluorogenic, chemiluminescent or electrochemical materials. Two commonly used radioactive isotopes are I and H. Standard radioactive isotopic labeling procedures include the chloramine T, lactoperoxidase and Bolton-Hunter methods for I and reductive methylation for H. The term \u201cdetectably-labeled\u201d refers to a molecule labeled in such a way that it can be readily detected by the intrinsic enzymic activity of the label or by the binding to the label of another component, which can itself be readily detected.","Enzymes suitable for use in this invention include, but are not limited to, horseradish peroxidase, alkaline phosphatase, (3-galactosidase, glucose oxidase, luciferases, including firefly and renilla, \u03b2-lactamase, urease, green fluorescent protein (GFP) and lysozyme. Enzyme labeling is facilitated by using dialdehyde, carbodiimide coupling, homobifunctional crosslinkers and heterobifunctional crosslinkers as described above for coupling an antibody with a member of a specific binding pair.","The labeling method chosen depends on the functional groups available on the enzyme and the material to be labeled, and the tolerance of both to the conjugation conditions. The labeling method used in the present invention can be one of, but not limited to, any conventional methods currently employed including those described by Engvall and Pearlmann, Immunochemistry 8, 871 (1971), Avrameas and Ternynck, Immunochemistry 8, 1175 (1975), Ishikawa et al., J. Immunoassay 4(3):209-327 (1983) and Jablonski, Anal. Biochem. 148:199 (1985).","Labeling can be accomplished by indirect methods such as using spacers or other members of specific binding pairs. An example of this is the detection of a biotinylated antibody with unlabeled streptavidin and biotinylated enzyme, with streptavidin and biotinylated enzyme being added either sequentially or simultaneously. Thus, according to the present invention, the antibody used to detect can be detectably-labeled directly with a reporter or indirectly with a first member of a specific binding pair. When the antibody is coupled to a first member of a specific binding pair, then detection is effected by reacting the antibody-first member of a specific binding complex with the second member of the binding pair that is labeled or unlabeled as mentioned above.","Moreover, the unlabeled detector antibody can be detected by reacting the unlabeled antibody with a labeled antibody specific for the unlabeled antibody. In this instance \u201cdetectably-labeled\u201d as used above is taken to mean containing an epitope by which an antibody specific for the unlabeled antibody can bind. Such an anti-antibody can be labeled directly or indirectly using any of the approaches discussed above. For example, the anti-antibody can be coupled to biotin which is detected by reacting with the streptavidin-horseradish peroxidase system discussed above.","In one embodiment of this invention biotin is utilized. The biotinylated antibody is in turn reacted with streptavidin-horseradish peroxidase complex. Orthophenylenediamine, 4-chloro-naphthol, tetramethylbenzidine (TMB), ABTS, BTS or ASA can be used to effect chromogenic detection.","In one immunoassay format for practicing this invention, a forward sandwich assay is used in which the capture reagent has been immobilized, using conventional techniques, on the surface of a support. Suitable supports used in assays include synthetic polymer supports, such as polypropylene, polystyrene, substituted polystyrene, e.g. aminated or carboxylated polystyrene, polyacrylamides, polyamides, polyvinylchloride, glass beads, agarose, or nitrocellulose.","The invention also encompasses kits for detecting the presence of a biomarker protein or nucleic acid in a biological sample. Such kits can be used to determine if a subject is suffering from or is at increased risk of developing a tumor that is less susceptible to inhibition by IGF-1R kinase inhibitors. For example, the kit can comprise a labeled compound or agent capable of detecting a biomarker protein or nucleic acid in a biological sample and means for determining the amount of the protein or mRNA in the sample (e.g., an antibody which binds the protein or a fragment thereof, or an oligonucleotide probe which binds to DNA or mRNA encoding the protein). Kits can also include instructions for interpreting the results obtained using the kit.","For antibody-based kits, the kit can comprise, for example: (1) a first antibody (e.g., attached to a solid support) which binds to a biomarker protein; and, optionally, (2) a second, different antibody which binds to either the protein or the first antibody and is conjugated to a detectable label.","For oligonucleotide-based kits, the kit can comprise, for example: (1) an oligonucleotide, e.g., a detectably labeled oligonucleotide, which hybridizes to a nucleic acid sequence encoding a biomarker protein or (2) a pair of primers useful for amplifying a biomarker nucleic acid molecule. The kit can also comprise, e.g., a buffering agent, a preservative, or a protein stabilizing agent. The kit can further comprise components necessary for detecting the detectable label (e.g., an enzyme or a substrate). The kit can also contain a control sample or a series of control samples which can be assayed and compared to the test sample. Each component of the kit can be enclosed within an individual container and all of the various containers can be within a single package, along with instructions for interpreting the results of the assays performed using the kit.","The present invention further provides a method for treating tumors or tumor metastases in a patient, comprising the steps of diagnosing a patient's likely responsiveness to an IGF-1R kinase inhibitor using any of the methods described herein for determining the expression level of tumor cell sensitivity and\/or resistance biomarkers, and administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor. For this method, an example of a preferred IGF-1R kinase inhibitor would be one with similar characteristics (e.g. selectivity, potency) to PQIP, (e.g. OSI-906, including pharmacologically acceptable salts or polymorphs thereof). In this method one or more additional anti-cancer agents or treatments can be co-administered simultaneously or sequentially with the IGF-1R kinase inhibitor, as judged to be appropriate by the administering physician given the prediction of the likely responsiveness of the patient to an IGF-1R kinase inhibitor, in combination with any additional circumstances pertaining to the individual patient.","It will be appreciated by one of skill in the medical arts that the exact manner of administering to said patient of a therapeutically effective amount of an IGF-1R kinase inhibitor following a diagnosis of a patient's likely responsiveness to an IGF-1R kinase inhibitor will be at the discretion of the attending physician. The mode of administration, including dosage, combination with other anti-cancer agents, timing and frequency of administration, and the like, may be affected by the diagnosis of a patient's likely responsiveness to an IGF-1R kinase inhibitor, as well as the patient's condition and history. Thus, even patients diagnosed with tumors predicted to be relatively insensitive to IGF-1R kinase inhibitors may still benefit from treatment with such inhibitors, particularly in combination with other anti-cancer agents, or agents that may alter a tumor's sensitivity to IGF-1R kinase inhibitors.","The present invention further provides a method for treating tumors or tumor metastases in a patient, comprising the steps of diagnosing a patient's likely responsiveness to an IGF-1R kinase inhibitor by assessing whether the tumor cells are sensitive to inhibition by an IGF-1R kinase inhibitor, by for example any of the methods described herein for determining the expression level of tumor cell sensitivity and\/or resistance biomarkers, identifying the patient as one who is likely to demonstrate an effective response to treatment with an IGF-1R kinase inhibitor, and administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor. In one embodiment the IGF-1R kinase inhibitor used for treatment comprises OSI-906.","The present invention also provides a method for inhibiting tumor cell growth in a patient, comprising the steps of diagnosing a patient's likely responsiveness to an IGF-1R kinase inhibitor by using any of the methods described herein to predict the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor, identifying the patient as one who is likely to demonstrate an effective response to treatment with an IGF-1R kinase inhibitor, and administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor. In one embodiment the IGF-1R kinase inhibitor used for treatment comprises OSI-906.","The present invention also provides a method for treating tumors or tumor metastases in a patient, comprising the steps of: diagnosing a patient's likely responsiveness to an IGF-1R kinase inhibitor by assessing the level of a resistance biomarker expressed by the tumor cells of the patient, wherein the resistance biomarker is selected from any of those listed herein (e.g. see ), and wherein high levels of expression of the biomarker by tumor cells correlates with low sensitivity to inhibition by IGF-1R kinase inhibitors, and administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor if the level of the resistance biomarker expressed by the tumor cells of the patient is low, implying that the patient is potentially responsive to an IGF-1R kinase inhibitor. In one embodiment the IGF-1R kinase inhibitor used for treatment comprises OSI-906.","The present invention also provides a method for treating tumors or tumor metastases in a patient, comprising the steps of: diagnosing a patient's likely responsiveness to an IGF-1R kinase inhibitor by assessing the level of a sensitivity biomarker expressed by the tumor cells of the patient, wherein the sensitivity biomarker is selected from any of those listed herein (e.g. see ), and wherein high levels of expression of the biomarker by tumor cells correlates with high sensitivity to inhibition by IGF-1R kinase inhibitors, and administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor if the level of the sensitivity biomarker expressed by the tumor cells of the patient is high, implying that the patient is potentially responsive to an IGF-1R kinase inhibitor. In one embodiment the IGF-1R kinase inhibitor used for treatment comprises OSI-906.","The present invention also provides a method for treating tumors or tumor metastases in a patient, comprising the steps of: diagnosing a patient's likely responsiveness to an IGF-1R kinase inhibitor by assessing the level of any of the resistance biomarkers listed herein (e.g. see ) expressed by the tumor cells of the patient, wherein high levels of resistance biomarker expression by tumor cells correlates with low sensitivity to inhibition by IGF-1R kinase inhibitors, and administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor if the patient is diagnosed to be potentially responsive to an IGF-1R kinase inhibitor (i.e. if low levels of resistance biomarkers predict high sensitivity to inhibition by IGF-1R kinase inhibitors). In one embodiment the IGF-1R kinase inhibitor used for treatment comprises OSI-906. In another embodiment the resistance biomarker is selected from caldesmon 1 (CALD1, GeneID: 800); kelch-like 5 () (KLHL5, GeneID: 51088); metallothionein 1E (functional) (MT1E, GeneID: 4493); beta-1,3-N-acetylgalactosaminyltransferase 1 (globoside blood group) (B3GALNT1, GeneID: 8706); cysteine-rich, angiogenic inducer, 61 (CYR61, GeneID: 3491); metallothionein 1X (MT1X, GeneID: 4501); troponin T type 1 (skeletal, slow) (TNNT1, GeneID: 7138); metallothionein 1H-like protein\/\/\/hypothetical protein LOC650610 (MT1P2, GeneID: 645745); metallothionein 1H (MT1H, GeneID: 4496); metallothionein IF (functional) (MT1F, GeneID: 4494); metallothionein 2A (MT2A, GeneID: 4502); metallothionein 1M (MT1M, GeneID: 4499); MHC class I polypeptide-related sequence B (MICB, GeneID: 4277); and collagen, type VI, alpha 1 (COL6A1, GeneID: 1291). In the above methods, mRNA or protein expressed by the biomarker gene may be determined.","The present invention also provides a method for treating tumors or tumor metastases in a patient, comprising the steps of: diagnosing a patient's likely responsiveness to an IGF-1R kinase inhibitor by assessing the level of any of the sensitivity biomarkers listed herein (e.g. see ) expressed by the tumor cells of the patient, wherein high levels of sensitivity biomarker expression by tumor cells correlates with high sensitivity to inhibition by IGF-1R kinase inhibitors, and administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor if the patient is diagnosed to be potentially responsive to an IGF-1R kinase inhibitor (i.e. if high levels of sensitivity biomarkers predict high sensitivity to inhibition by IGF-1R kinase inhibitors). In one embodiment the IGF-1R kinase inhibitor used for treatment comprises OSI-906. In another embodiment the sensitivity biomarker is selected from aldehyde dehydrogenase 1 family, member A1 (ALDH1A1, GeneID: 216); ring finger protein 128 (RNF128, GeneID: 79589); mitogen-activated protein kinase kinase 6 (MAP2K6, GeneID: 5608); and quinolinate phosphoribosyltransferase (nicotinate-nucleotide pyrophosphorylase (carboxylating)) (QPRT, GeneID: 23475). In the above methods, mRNA or protein expressed by the biomarker gene may be determined.","The present invention further provides a method for treating tumors or tumor metastases in a patient, comprising the steps of diagnosing a patient's likely responsiveness to an IGF-1R kinase inhibitor by any of the methods described herein for determining the expression level of tumor cell sensitivity and\/or resistance biomarkers, identifying the patient as one who is less likely or not likely to demonstrate an effective response to treatment with an IGF-1R kinase inhibitor, and treating said patient with an anti-cancer therapy other than an IGF-1R kinase inhibitor.","The present invention also provides for any of the methods of treatment with an IGF-1R kinase inhibitor described herein, the method as described but including prior to the step of administering to the patient an IGF-1R kinase inhibitor, an additional step of assessment of the level of IGF-1 and\/or IGF-2 (i.e. insulin-like growth factors 1 and\/or 2) in the tumor of the patient. Since IGF-1R has been reported to be activated only upon ligand (i.e. IGF-1 and\/or IGF-2) binding, if there is no IGF-1R ligand present in a tumor, then even if one or more of the methods of the instant invention predict that it should be sensitive to inhibition by IGF-1R kinase inhibitors, the tumor cells cannot under such circumstances be relying on the IGF-1R signaling pathway for growth and survival, and thus an IGF-1R kinase inhibitor would probably not be an effective treatment. Many tumors have been found to express elevated levels of IGF-1 and\/or IGF-2 (Pollack, M. N. et al. (2004) Nature Reviews Cancer 4:505-518), which could originate from the tumor cells themselves, from stromal cells present in the tumor, or via the vascular system from non-tumor cells (e.g. liver cells). Assessment of the level of IGF-1 and\/or IGF-2 can be performed by any method known in the art, such as for example any of the methods described herein for assessment of biomarkers levels, e.g. immunoassay determination of IGF-1 and\/or IGF-2 protein levels; determination of IGF-1 and\/or IGF-2 mRNA transcript levels. In an alternative embodiment, the of step of assessment of the level of IGF-1 and\/or IGF-2 (i.e. insulin-like growth factors 1 and\/or 2) in the tumor of the patient can be replaced with a step of assessment of the level of IGF-1 and\/or IGF-2 (i.e. insulin-like growth factors 1 and\/or 2) in the blood or serum of the patient. This alternative, though not a direct measure of the level of IGF-1 and\/or IGF-2 in the tumor, can give an indication of the potential availability of ligand to the IGF-1R in the tumor, and is a simpler and less expensive test. The potential disadvantage of this indirect assessment of IGF-1 and\/or IGF-2 is that it may not give a true indication of the levels of ligand in the tumor if IGF-1 and\/or IGF-2 is produced locally in the tumor, either by the tumor cells themselves, or by stromal cells within the tumor.","Accordingly, the invention provides a method for treating tumors or tumor metastases in a patient, comprising the steps of: diagnosing a patient's likely responsiveness to an IGF-1R kinase inhibitor, by assessing the level of a sensitivity and\/or resistance biomarker expressed by a tumor cell, wherein high expression levels of a sensitivity biomarker correlates with high sensitivity to inhibition by IGF-1R kinase inhibitors, and wherein a high expression levels of an resistance biomarker correlates with low sensitivity to inhibition by IGF-1R kinase inhibitors; assessing the level of IGF-1 and\/or IGF-2 in the tumor (or blood or serum) of the patient; and administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor if the patient is diagnosed to be potentially responsive to an IGF-1R kinase inhibitor, and the tumor is determined to have IGF-1 and\/or IGF-2 (or blood or serum levels indicate the potential availability of IGF-1 and\/or IGF-2 to the tumor cells). In the context of this method, where more than one biomarker is assessed for predicting sensitivity to IGF-1R kinase inhibitors, the patient is diagnosed to be potentially responsive to an IGF-1R kinase inhibitor if at least one of the biomarkers indicates potential sensitivity.","The present invention also provides a method for treating tumors or tumor metastases in a patient, comprising the steps of: diagnosing a patient's likely responsiveness to an IGF-1R kinase inhibitor, by assessing the level of a resistance and\/or sensitivity biomarker indicative of whether the tumor cells are sensitive to inhibition by an IGF-1R kinase inhibitor; assessing the level of IGF-1 and\/or IGF-2 in the tumor of the patient; and administering to said patient a therapeutically effective amount of an IGF-1R kinase inhibitor if the patient is diagnosed to be potentially responsive to an IGF-1R kinase inhibitor, and the tumor is determined to have IGF-1 and\/or IGF-2.","The present invention further provides a method of identifying a sensitivity biomarker whose expression level is predictive of the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor, comprising: (a) measuring the expression level of a candidate sensitivity biomarker in a panel of tumor cells that displays a range of sensitivities to an IGF-1R kinase inhibitor, and (b) identifying a correlation between the expression level of said candidate sensitivity biomarker in the tumor cells and the sensitivity of tumor cell growth to inhibition by the IGF-1R kinase inhibitor, wherein a correlation of high levels of the sensitivity biomarker with high sensitivity of tumor cell growth to inhibition by the IGF-1R kinase inhibitor indicates that the expression level of said sensitivity biomarker is predictive of the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor. In one embodiment of this method the panel of tumor cells is a panel of tumor cell lines. In an alternative embodiment the panel of tumor cells is a panel of primary tumor cells, prepared from tumor samples derived from patients or experimental animal models. In an additional embodiment the panel of tumor cells is a panel of tumor cell lines in mouse xenografts, wherein tumor cell growth can for example be determined by monitoring a molecular marker of growth or a gross measurement of tumor growth, e.g. tumor dimensions or weight.","The present invention further provides a method of identifying a resistance biomarker whose expression level is predictive of the sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor, comprising: (a) measuring the expression level of a candidate resistance biomarker in a panel of tumor cells that displays a range of sensitivities to an IGF-1R kinase inhibitor, and (b) identifying a correlation between the expression level of said candidate resistance biomarker in the tumor cells and the sensitivity of tumor cell growth to inhibition by the IGF-1R kinase inhibitor, wherein a correlation of high levels of the resistance biomarker with low sensitivity of tumor cell growth to inhibition by the IGF-1R kinase inhibitor indicates that the expression level of said resistance biomarker is predictive of the lack of sensitivity of tumor cell growth to inhibition by an IGF-1R kinase inhibitor. In one embodiment of this method the panel of tumor cells is a panel of tumor cell lines. In an alternative embodiment the panel of tumor cells is a panel of primary tumor cells, prepared from tumor samples derived from patients or experimental animal models. In an additional embodiment the panel of tumor cells is a panel of tumor cell lines in mouse xenografts, wherein tumor cell growth can for example be determined by monitoring a molecular marker of growth or a gross measurement of tumor growth, e.g. tumor dimensions or weight.","The present invention further provides a method of identifying an sensitivity biomarker that is diagnostic for more effective treatment of a neoplastic condition with an IGF-1R kinase inhibitor, comprising: (a) measuring the level of a candidate sensitivity biomarker in neoplastic cell-containing samples from patients with a neoplastic condition, and (b) identifying a correlation between the level of said candidate sensitivity biomarker in the sample from the patient with the effectiveness of treatment of the neoplastic condition with an IGF-1R kinase inhibitor, wherein a correlation of high levels of the sensitivity biomarker with more effective treatment of the neoplastic condition with an IGF-1R kinase inhibitor indicates that said sensitivity biomarker is diagnostic for more effective treatment of the neoplastic condition with an IGF-1R kinase inhibitor.","The present invention further provides a method of identifying a resistance biomarker that is diagnostic for less effective treatment of a neoplastic condition with an IGF-1R kinase inhibitor, comprising: (a) measuring the level of a candidate resistance biomarker in neoplastic cell-containing samples from patients with a neoplastic condition, and (b) identifying a correlation between the level of said candidate resistance biomarker in the sample from the patient with the effectiveness of treatment of the neoplastic condition with an IGF-1R kinase inhibitor, wherein a correlation of high levels of the resistance biomarker with less effective treatment of the neoplastic condition with an IGF-1R kinase inhibitor indicates that said resistance biomarker is diagnostic for less effective treatment of the neoplastic condition with an IGF-1R kinase inhibitor.","The effectiveness of treatment in the preceding methods can for example be determined by measuring the decrease in size of tumors present in the patients with the neoplastic condition, or by assaying a molecular determinant of the degree of proliferation of the tumor cells.","The present invention provides a method of identifying an sensitivity biomarker that is diagnostic for increased survival of a patient with a neoplastic condition when treated with an IGF-1R kinase inhibitor, comprising: (a) measuring the level of the candidate sensitivity biomarker in neoplastic cell-containing samples from patients with a neoplastic condition, and (b) identifying a correlation between the level of said candidate sensitivity biomarker in the sample from the patient with the survival of that patient when treated with an IGF-1R kinase inhibitor, wherein the correlation of an sensitivity biomarker with survival in said patients indicates said sensitivity biomarker is diagnostic for increased survival of a patient with said neoplastic condition when treated with an IGF-1R kinase inhibitor.","The present invention provides a method of identifying a resistance biomarker that is diagnostic for decreased survival of a patient with a neoplastic condition when treated with an IGF-1R kinase inhibitor, comprising: (a) measuring the level of the candidate resistance biomarker in neoplastic cell-containing samples from patients with a neoplastic condition, and (b) identifying an inverse correlation between the level of said candidate resistance biomarker in the sample from the patient with the survival of that patient when treated with an IGF-1R kinase inhibitor, wherein the inverse correlation of a resistance biomarker with survival in said patients indicates said resistance biomarker is diagnostic for decreased survival of a patient with said neoplastic condition when treated with an IGF-1R kinase inhibitor.","The present invention further provides any of the methods described herein for treating tumors or tumor metastases in a patient comprising administering to the patient a therapeutically effective amount of an IGF-1R kinase inhibitor, and in addition, simultaneously or sequentially, one or more other cytotoxic, chemotherapeutic or anti-cancer agents, or compounds that enhance the effects of such agents.","In the context of this invention, additional other cytotoxic, chemotherapeutic or anti-cancer agents, or compounds that enhance the effects of such agents, include, for example: alkylating agents or agents with an alkylating action, such as cyclophosphamide (CTX; e.g. CYTOXAN\u00ae), chlorambucil (CHL; e.g. LEUKERAN\u00ae), cisplatin (CisP; e.g. PLATINOL\u00ae) busulfan (e.g. MYLERAN\u00ae), melphalan, carmustine (BCNU), streptozotocin, triethylenemelamine (TEM), mitomycin C, and the like; anti-metabolites, such as methotrexate (MTX), etoposide (VP16; e.g. VEPESID\u00ae), 6-mercaptopurine (6MP), 6-thiocguanine (6TG), cytarabine (Ara-C), 5-fluorouracil (5-FU), capecitabine (e.g. XELODA\u00ae), dacarbazine (DTIC), and the like; antibiotics, such as actinomycin D, doxorubicin (DXR; e.g. ADRIAMYCIN\u00ae), daunorubicin (daunomycin), bleomycin, mithramycin and the like; alkaloids, such as vinca alkaloids such as vincristine (VCR), vinblastine, and the like; and other antitumor agents, such as paclitaxel (e.g. TAXOL\u00ae) and pactitaxel derivatives, the cytostatic agents, glucocorticoids such as dexamethasone (DEX; e.g. DECADRON\u00ae) and corticosteroids such as prednisone, nucleoside enzyme inhibitors such as hydroxyurea, amino acid depleting enzymes such as asparaginase, leucovorin and other folic acid derivatives, and similar, diverse antitumor agents. The following agents may also be used as additional agents: arnifostine (e.g. ETHYOL\u00ae), dactinomycin, mechlorethamine (nitrogen mustard), streptozocin, cyclophosphamide, lomustine (CCNU), doxorubicin lipo (e.g. DOXIL\u00ae), gemcitabine (e.g. GEMZAR\u00ae), daunorubicin lipo (e.g. DAUNOXOME\u00ae), procarbazine, mitomycin, docetaxel (e.g. TAXOTERE\u00ae), aldesleukin, carboplatin, oxaliplatin, cladribine, camptothecin, CPT 11 (irinotecan), 10-hydroxy 7-ethyl-camptothecin (SN38), floxuridine, fludarabine, ifosfamide, idarubicin, mesna, interferon beta, interferon alpha, mitoxantrone, topotecan, leuprolide, megestrol, melphalan, mercaptopurine, plicamycin, mitotane, pegaspargase, pentostatin, pipobroman, plicamycin, tamoxifen, teniposide, testolactone, thioguanine, thiotepa, uracil mustard, vinorelbine, chlorambucil.","The present invention further provides any of the methods described herein for treating tumors or tumor metastases in a patient comprising administering to the patient a therapeutically effective amount of an IGF-1R kinase inhibitor, and in addition, simultaneously or sequentially, one or more anti-hormonal agents. As used herein, the term \u201canti-hormonal agent\u201d includes natural or synthetic organic or peptidic compounds that act to regulate or inhibit hormone action on tumors.","Antihormonal agents include, for example: steroid receptor antagonists, anti-estrogens such as tamoxifen, raloxifene, aromatase inhibiting 4(5)-imidazoles, other aromatase inhibitors, 42-hydroxytamoxifen, trioxifene, keoxifene, LY 117018, onapristone, and toremifene (e.g. FARESTON\u00ae); anti-androgens such as flutamide, nilutamide, bicalutamide, leuprolide, and goserelin; and pharmaceutically acceptable salts, acids or derivatives of any of the above; agonists and\/or antagonists of glycoprotein hormones such as follicle stimulating hormone (FSH), thyroid stimulating hormone (TSH), and luteinizing hormone (LH) and LHRH (leuteinizing hormone-releasing hormone); the LHRH agonist goserelin acetate, commercially available as ZOLADEX\u00ae (AstraZeneca); the LHRH antagonist D-alaninamide N-acetyl-3-(2-naphthalenyl)-D-alanyl-4-chloro-D-phenylalanyl-3-(3-pyridinyl)-D-alanyl-L-seryl-N6-(3-pyridinylcarbonyl)-L-lysyl-N6-(3-pyridinylcarbonyl)-D-lysyl-L-leucyl-N6-(1-methylethyl)-L-lysyl-L-proline (e.g ANTIDEi\u00ae, Ares-Serono); the LHRH antagonist ganirelix acetate; the steroidal anti-androgens cyproterone acetate (CPA) and megestrol acetate, commercially available as MEGACE\u00ae (Bristol-Myers Oncology); the nonsteroidal anti-androgen flutamide (2-methyl-N-[4,20-nitro-3-(trifluoromethyl)phenylpropanamide), commercially available as EULEXIN\u00ae (Schering Corp.); the non-steroidal anti-androgen nilutamide, (5,5-dimethyl-3-[4-nitro-3-(trifluoromethyl-4\u2032-nitrophenyl)-4,4-dimethyl-imidazolidine-dione); and antagonists for other non-permissive receptors, such as antagonists for RAR, RXR, TR, VDR, and the like.","The use of the cytotoxic and other anticancer agents described above in chemotherapeutic regimens is generally well characterized in the cancer therapy arts, and their use herein falls under the same considerations for monitoring tolerance and effectiveness and for controlling administration routes and dosages, with some adjustments. For example, the actual dosages of the cytotoxic agents may vary depending upon the patient's cultured cell response determined by using histoculture methods. Generally, the dosage will be reduced compared to the amount used in the absence of additional other agents.","Typical dosages of an effective cytotoxic agent can be in the ranges recommended by the manufacturer, and where indicated by in vitro responses or responses in animal models, can be reduced by up to about one order of magnitude concentration or amount. Thus, the actual dosage will depend upon the judgment of the physician, the condition of the patient, and the effectiveness of the therapeutic method based on the in vitro responsiveness of the primary cultured malignant cells or histocultured tissue sample, or the responses observed in the appropriate animal models.","The present invention further provides any of the methods described herein for treating tumors or tumor metastases in a patient comprising administering to the patient a therapeutically effective amount of an IGF-1R kinase inhibitor, and in addition, simultaneously or sequentially, one or more angiogenesis inhibitors.","Anti-angiogenic agents include, for example: VEGFR inhibitors, such as SU-5416 and SU-6668 (Sugen Inc. of South San Francisco, Calif., USA), or as described in, for example International Application Nos. WO 99\/24440, WO 99\/62890, WO 95\/21613, WO 99\/61422, WO 98\/50356, WO 99\/10349, WO 97\/32856, WO 97\/22596, WO 98\/54093, WO 98\/02438, WO 99\/16755, and WO 98\/02437, and U.S. Pat. Nos. 5,883,113, 5,886,020, 5,792,783, 5,834,504 and 6,235,764; VEGF inhibitors such as IM862 (Cytran Inc. of Kirkland, Wash., USA); angiozyme, a synthetic ribozyme from Ribozyme (Boulder, Colo.) and Chiron (Emeryville, Calif.); and antibodies to VEGF, such as bevacizumab (e.g. AVASTIN\u2122, Genentech, South San Francisco, Calif.), a recombinant humanized antibody to VEGF; integrin receptor antagonists and integrin antagonists, such as to \u03b1\u03b2, \u03b1\u03b2and \u03b1\u03b2integrins, and subtypes thereof, e.g. cilengitide (EMD 121974), or the anti-integrin antibodies, such as for example \u03b1\u03b2specific humanized antibodies (e.g. VITAXIN\u00ae); factors such as IFN-alpha (U.S. Pat. Nos. 41,530,901, 4,503,035, and 5,231,176); angiostatin and plasminogen fragments (e.g. kringle 1-4, kringle 5, kringle 1-3 (O'Reilly, M. S. et al. (1994) Cell 79:315-328; Cao et al. (1996) J. Biol. Chem. 271: 29461-29467; Cao et al. (1997) J. Biol. Chem. 272:22924-22928); endostatin (O'Reilly, M. S. et al. (1997) Cell 88:277; and International Patent Publication No. WO 97\/15666); thrombospondin (TSP-1; Frazier, (1991) Curr. Opin. Cell Biol. 3:792); platelet factor 4 (PF4); plasminogen activator\/urokinase inhibitors; urokinase receptor antagonists; heparinases; fumagillin analogs such as TNP-4701; suramin and suramin analogs; angiostatic steroids; bFGF antagonists; flk-1 and flt-1 antagonists; anti-angiogenesis agents such as MMP-2 (matrix-metalloproteinase 2) inhibitors and MMP-9 (matrix-metalloproteinase 9) inhibitors. Examples of useful matrix metalloproteinase inhibitors are described in International Patent Publication Nos. WO 96\/33172, WO 96\/27583, WO 98\/07697, WO 98\/03516, WO 98\/34918, WO 98\/34915, WO 98\/33768, WO 98\/30566, WO 90\/05719, WO 99\/52910, WO 99\/52889, WO 99\/29667, and WO 99\/07675, European Patent Publication Nos. 818,442, 780,386, 1,004,578, 606,046, and 931,788; Great Britain Patent Publication No. 9912961, and U.S. Pat. Nos. 5,863,949 and 5,861,510. Preferred MMP-2 and MMP-9 inhibitors are those that have little or no activity inhibiting MMP-1. More preferred, are those that selectively inhibit MMP-2 and\/or MMP-9 relative to the other matrix-metalloproteinases (i.e. MMP-1, MMP-3, MMP-4, MMP-5, MMP-6, MMP-7, MMP-8, MMP-10, MMP-11, MMP-12, and MMP-13).","The present invention further provides any of the methods described herein for treating tumors or tumor metastases in a patient comprising administering to the patient a therapeutically effective amount of an IGF-1R kinase inhibitor, and in addition, simultaneously or sequentially, one or more tumor cell pro-apoptotic or apoptosis-stimulating agents.","The present invention further provides any of the methods described herein for treating tumors or tumor metastases in a patient comprising administering to the patient a therapeutically effective amount of an IGF-1R kinase inhibitor, and in addition, simultaneously or sequentially, one or more signal transduction inhibitors.","Signal transduction inhibitors include, for example: erbB2 receptor inhibitors, such as organic molecules, or antibodies that bind to the erbB2 receptor, for example, trastuzumab (e.g. HERCEPTIN\u00ae); inhibitors of other protein tyrosine-kinases, e.g. imitinib (e.g. GLEEVEC\u00ae); EGFR kinase inhibitors (see herein below); ras inhibitors; raf inhibitors; MEK inhibitors; mTOR inhibitors, including mTOR inhibitors that bind to and directly inhibits both mTORC1 and mTORC2 kinases; mTOR inhibitors that are dual PI3K\/mTOR kinase inhibitors, such as for example the compound PI-103 as described in Fan, Q-W et al (2006) Cancer Cell 9:341-349 and Knight, Z. A. et al. (2006) Cell 125:733-747; mTOR inhibitors that are dual inhibitors of mTOR kinase and one or more other PIKK (or PIK-related) kinase family members. Such members include MEC1, TEL1, RAD3, MEI-41, DNA-PK, ATM, ATR, TRRAP, PI3K, and PI4K kinases; cyclin dependent kinase inhibitors; protein kinase C inhibitors; PI-3 kinase inhibitors; and PDK-1 inhibitors (see Dancey, J. and Sausville, E. A. (2003) Nature Rev. Drug Discovery 2:92-313, for a description of several examples of such inhibitors, and their use in clinical trials for the treatment of cancer).","EGFR inhibitors include, for example: [6,7-bis(2-methoxyethoxy)-4-quinazolin-4-yl]-(3-ethynylphenyl)amine (also known as OSI-774, erlotinib, or TARCEVA\u2122 (erlotinib HCl); OSI Pharmaceuticals\/Genentech\/Roche) (U.S. Pat. No. 5,747,498; International Patent Publication No. WO 01\/34574, and Moyer, J. D. et al. (1997) Cancer Res. 57:4838-4848); CI-1033 (formerly known as PD183805; Pfizer) (Sherwood et al., 1999, Proc. Am. Assoc. Cancer Res. 40:723); PD-158780 (Pfizer); AG-1478 (University of California); CGP-59326 (Novartis); PKI-166 (Novartis); EKB-569 (Wyeth); GW-2016 (also known as GW-572016 or lapatinib ditosylate; GSK); gefitinib (also known as ZD1839 or IRESSA\u2122; Astrazeneca) (Woodburn et al., 1997, Proc. Am. Assoc. Cancer Res. 38:633); and antibody-based EGFR kinase inhibitors. A particularly preferred low molecular weight EGFR kinase inhibitor that can be used according to the present invention is [6,7-bis(2-methoxyethoxy)-4-quinazolin-4-yl]-(3-ethynylphenyl)amine (i.e. erlotinib), its hydrochloride salt (i.e. erlotinib HCl, TARCEVA\u2122), or other salt forms (e.g. erlotinib mesylate). Antibody-based EGFR kinase inhibitors include any anti-EGFR antibody or antibody fragment that can partially or completely block EGFR activation by its natural ligand. Non-limiting examples of antibody-based EGFR kinase inhibitors include those described in Modjtahedi, H., et al., 1993, Br. J. Cancer 67:247-253; Teramoto, T., et al., 1996, Cancer 77:639-645; Goldstein et al., 1995, Clin. Cancer Res. 1:1311-1318; Huang, S. M., et al., 1999, Cancer Res. 15:59(8):1935-40; and Yang, X., et al., 1999, Cancer Res. 59:1236-1243. Thus, the EGFR kinase inhibitor can be the monoclonal antibody Mab E7.6.3 (Yang, X. D. et al. (1999) Cancer Res. 59:1236-43), or Mab C225 (ATCC Accession No. HB-8508), or an antibody or antibody fragment having the binding specificity thereof. Suitable monoclonal antibody EGFR kinase inhibitors include, but are not limited to, IMC-C225 (also known as cetuximab or ERBITUX\u2122; Imclone Systems), ABX-EGF (Abgenix), EMD 72000 (Merck KgaA, Darmstadt), RH3 (York Medical Bioscience Inc.), and MDX-447 (Medarex\/Merck KgaA).","EGFR kinase inhibitors also include, for example multi-kinase inhibitors that have activity on EGFR kinase, i.e. inhibitors that inhibit EGFR kinase and one or more additional kinases. Examples of such compounds include the EGFR and HER2 inhibitor CI-1033 (formerly known as PD183805; Pfizer); the EGFR and HER2 inhibitor GW-2016 (also known as GW-572016 or lapatinib ditosylate; GSK); the EGFR and JAK 2\/3 inhibitor AG490 (a tyrphostin); the EGFR and HER2 inhibitor ARRY-334543 (Array BioPharma); BIBW-2992, an irreversible dual EGFR\/HER2 kinase inhibitor (Boehringer Ingelheim Corp.); the EGFR and HER2 inhibitor EKB-569 (Wyeth); the VEGF-R2 and EGFR inhibitor ZD6474 (also known as ZACTIMA\u2122; AstraZeneca Pharmaceuticals), and the EGFR and HER2 inhibitor BMS-599626 (Bristol-Myers Squibb).","ErbB2 receptor inhibitors include, for example: ErbB2 receptor inhibitors, such as GW-282974 (Glaxo Wellcome plc), monoclonal antibodies such as AR-209 (Aronex Pharmaceuticals Inc. of The Woodlands, Tex., USA) and 2B-1 (Chiron), and erbB2 inhibitors such as those described in International Publication Nos. WO 98\/02434, WO 99\/35146, WO 99\/35132, WO 98\/02437, WO 97\/13760, and WO 95\/19970, and U.S. Pat. Nos. 5,587,458, 5,877,305, 6,465,449 and 6,541,481.","The present invention further provides any of the methods described herein for treating tumors or tumor metastases in a patient comprising administering to the patient a therapeutically effective amount of an IGF-1R kinase inhibitor, and in addition, simultaneously or sequentially, an anti-HER2 antibody or an immunotherapeutically active fragment thereof.","The present invention further provides any of the methods described herein for treating tumors or tumor metastases in a patient comprising administering to the patient a therapeutically effective amount of an IGF-1R kinase inhibitor, and in addition, simultaneously or sequentially, one or more additional anti-proliferative agents.","Additional antiproliferative agents include, for example: Inhibitors of the enzyme farnesyl protein transferase and inhibitors of the receptor tyrosine kinase PDGFR, including the compounds disclosed and claimed in U.S. Pat. Nos. 6,080,769, 6,194,438, 6,258,824, 6,586,447, 6,071,935, 6,495,564, 6,150,377, 6,596,735 and 6,479,513, and International Patent Publication WO 01\/40217, and FGFR kinase inhibitors.","Examples of PDGFR kinase inhibitors that can be used according to the present invention include Imatinib (GLEEVEC\u00ae; Novartis); SU-12248 (sunitib malate, SUTENT\u00ae; Pfizer); Dasatinib (SPRYCEL\u00ae; BMS; also known as BMS-354825); Sorafenib (NEXAVAR\u00ae; Bayer; also known as Bay-43-9006); AG-13736 (Axitinib; Pfizer); RPR127963 (Sanofi-Aventis); CP-868596 (Pfizer\/OSI Pharmaceuticals); MLN-518 (tandutinib; Millennium Pharmaceuticals); AMG-706 (Motesanib; Amgen); ARAVA\u00ae (leflunomide; Sanofi-Aventis; also known as SU101), and OSI-930 (OSI Pharmaceuticals); Additional preferred examples of low molecular weight PDGFR kinase inhibitors that are also FGFR kinase inhibitors that can be used according to the present invention include XL-999 (Exelixis); SU6668 (Pfizer); CHIR-258\/TKI-258 (Chiron); RO4383596 (Hoffmann-La Roche) and BIBF-1120 (Boehringer Ingelheim).","Examples of FGFR kinase inhibitors that can be used according to the present invention include RO-4396686 (Hoffmann-La Roche); CHIR-258 (Chiron; also known as TKI-258); PD 173074 (Pfizer); PD 166866 (Pfizer); ENK-834 and ENK-835 (both Enkam Pharmaceuticals A\/S); and SU5402 (Pfizer). Additional preferred examples of low molecular weight FGFR kinase inhibitors that are also PDGFR kinase inhibitors that can be used according to the present invention include XL-999 (Exelixis); SU6668 (Pfizer); CHIR-258\/TKI-258 (Chiron); RO4383596 (Hoffmann-La Roche), and BIBF-1120 (Boehringer Ingelheim).","The present invention further provides any of the methods described herein for treating tumors or tumor metastases in a patient comprising administering to the patient a therapeutically effective amount of an IGF-1R kinase inhibitor, and in addition, simultaneously or sequentially, a COX II (cyclooxygenase II) inhibitor. Examples of useful COX-II inhibitors include alecoxib (e.g. CELEBREX\u2122), valdecoxib, and rofecoxib.","The present invention further provides any of the methods described herein for treating tumors or tumor metastases in a patient comprising administering to the patient a therapeutically effective amount of an IGF-1R kinase inhibitor, and in addition, simultaneously or sequentially, treatment with radiation or a radiopharmaceutical.","The source of radiation can be either external or internal to the patient being treated. When the source is external to the patient, the therapy is known as external beam radiation therapy (EBRT). When the source of radiation is internal to the patient, the treatment is called brachytherapy (BT). Radioactive atoms for use in the context of this invention can be selected from the group including, but not limited to, radium, cesium-137, iridium-192, americium-241, gold-198, cobalt-57, copper-67, technetium-99, iodine-123, iodine-131, and indium-111. Where the IGF-1R kinase inhibitor according to this invention is an antibody, it is also possible to label the antibody with such radioactive isotopes.","Radiation therapy is a standard treatment for controlling unresectable or inoperable tumors and\/or tumor metastases. Improved results have been seen when radiation therapy has been combined with chemotherapy. Radiation therapy is based on the principle that high-dose radiation delivered to a target area will result in the death of reproductive cells in both tumor and normal tissues. The radiation dosage regimen is generally defined in terms of radiation absorbed dose (Gy), time and fractionation, and must be carefully defined by the oncologist. The amount of radiation a patient receives will depend on various considerations, but the two most important are the location of the tumor in relation to other critical structures or organs of the body, and the extent to which the tumor has spread. A typical course of treatment for a patient undergoing radiation therapy will be a treatment schedule over a 1 to 6 week period, with a total dose of between 10 and 80 Gy administered to the patient in a single daily fraction of about 1.8 to 2.0 Gy, 5 days a week. In a preferred embodiment of this invention there is synergy when tumors in human patients are treated with the combination treatment of the invention and radiation. In other words, the inhibition of tumor growth by means of the agents comprising the combination of the invention is enhanced when combined with radiation, optionally with additional chemotherapeutic or anticancer agents. Parameters of adjuvant radiation therapies are, for example, contained in International Patent Publication WO 99\/60023.","The present invention further provides any of the methods described herein for treating tumors or tumor metastases in a patient comprising administering to the patient a therapeutically effective amount of an IGF-1R kinase inhibitor, and in addition, simultaneously or sequentially, treatment with one or more agents capable of enhancing antitumor immune responses.","Agents capable of enhancing antitumor immune responses include, for example: CTLA4 (cytotoxic lymphocyte antigen 4) antibodies (e.g. MDX-CTLA4), and other agents capable of blocking CTLA4. Specific CTLA4 antibodies that can be used in the present invention include those described in U.S. Pat. No. 6,682,736.","In the context of this invention, an \u201ceffective amount\u201d of an agent or therapy is as defined above. A \u201csub-therapeutic amount\u201d of an agent or therapy is an amount less than the effective amount for that agent or therapy, but when combined with an effective or sub-therapeutic amount of another agent or therapy can produce a result desired by the physician, due to, for example, synergy in the resulting efficacious effects, or reduced side effects.","As used herein, the term \u201cpatient\u201d preferably refers to a human in need of treatment with an IGF-1R kinase inhibitor for any purpose, and more preferably a human in need of such a treatment to treat cancer, or a precancerous condition or lesion. However, the term \u201cpatient\u201d can also refer to non-human animals, preferably mammals such as dogs, cats, horses, cows, pigs, sheep and non-human primates, among others, that are in need of treatment with an IGF-1R kinase inhibitor.","In a preferred embodiment, the patient is a human in need of treatment for cancer, a precancerous condition or lesion, or other forms of abnormal cell growth. The cancer is preferably any cancer or tumor treatable, either partially or completely, by administration of an IGF-1R kinase inhibitor. The cancer or tumor may be, for example, lung cancer, non small cell lung (NSCL) cancer, bronchioloalveolar cell lung cancer, bone cancer, pancreatic cancer, skin cancer, cancer of the head or neck, cutaneous or intraocular melanoma, uterine cancer, ovarian cancer, rectal cancer, cancer of the anal region, stomach cancer, gastric cancer, colon cancer, colorectal cancer (CRC), breast cancer, uterine cancer, carcinoma of the fallopian tubes, carcinoma of the endometrium, carcinoma of the cervix, carcinoma of the vagina, carcinoma of the vulva, Hodgkin's Disease, cancer of the esophagus, cancer of the small intestine, cancer of the endocrine system, cancer of the thyroid gland, cancer of the parathyroid gland, cancer of the adrenal gland, sarcoma of soft tissue, cancer of the urethra, cancer of the penis, prostate cancer, cancer of the bladder, cancer of the kidney or ureter, renal cell carcinoma, carcinoma of the renal pelvis, mesothelioma, hepatocellular cancer, biliary cancer, chronic or acute leukemia, lymphocytic lymphomas, neoplasms of the central nervous system (CNS), spinal axis tumors, brain stem glioma, glioblastoma multiforme, astrocytomas, schwannomas, ependymomas, medulloblastomas, meningiomas, squamous cell carcinomas, pituitary adenomas, including refractory versions of any of the above cancers, or a combination of one or more of the above cancers. The precancerous condition or lesion includes, for example, the group consisting of oral leukoplakia, actinic keratosis (solar keratosis), precancerous polyps of the colon or rectum, gastric sensitivity dysplasia, adenomatous dysplasia, hereditary nonpolyposis colon cancer syndrome (HNPCC), Barrett's esophagus, bladder dysplasia, and precancerous cervical conditions.","The term \u201crefractory\u201d as used herein is used to define a cancer for which treatment (e.g. chemotherapy drugs, biological agents, and\/or radiation therapy) has proven to be ineffective. A refractory cancer tumor may shrink, but not to the point where the treatment is determined to be effective. Typically however, the tumor stays the same size as it was before treatment (stable disease), or it grows (progressive disease).","For purposes of the present invention, \u201cco-administration of\u201d and \u201cco-administering\u201d an IGF-1R kinase inhibitor with an additional anti-cancer agent (both components referred to hereinafter as the \u201ctwo active agents\u201d) refer to any administration of the two active agents, either separately or together, where the two active agents are administered as part of an appropriate dose regimen designed to obtain the benefit of the combination therapy. Thus, the two active agents can be administered either as part of the same pharmaceutical composition or in separate pharmaceutical compositions. The additional agent can be administered prior to, at the same time as, or subsequent to administration of the IGF-1R kinase inhibitor, or in some combination thereof. Where the IGF-1R kinase inhibitor is administered to the patient at repeated intervals, e.g., during a standard course of treatment, the additional agent can be administered prior to, at the same time as, or subsequent to, each administration of the IGF-1R kinase inhibitor, or some combination thereof, or at different intervals in relation to the IGF-1R kinase inhibitor treatment, or in a single dose prior to, at any time during, or subsequent to the course of treatment with the IGF-1R kinase inhibitor.","The IGF-1R kinase inhibitor will typically be administered to the patient in a dose regimen that provides for the most effective treatment of the cancer (from both efficacy and safety perspectives) for which the patient is being treated, as known in the art, and as disclosed, e.g. in International Patent Publication No. WO 01\/34574. In conducting the treatment method of the present invention, the IGF-1R kinase inhibitor can be administered in any effective manner known in the art, such as by oral, topical, intravenous, intra-peritoneal, intramuscular, intra-articular, subcutaneous, intranasal, intra-ocular, vaginal, rectal, or intradermal routes, depending upon the type of cancer being treated, the type of IGF-1R kinase inhibitor being used (for example, small molecule, antibody, RNAi, ribozyme or antisense construct), and the medical judgement of the prescribing physician as based, e.g., on the results of published clinical studies.","The amount of IGF-1R kinase inhibitor administered and the timing of IGF-1R kinase inhibitor administration will depend on the type (species, gender, age, weight, etc.) and condition of the patient being treated, the severity of the disease or condition being treated, and on the route of administration. For example, small molecule IGF-1R kinase inhibitors can be administered to a patient in doses ranging from 0.001 to 100 mg\/kg of body weight per day or per week in single or divided doses, or by continuous infusion (see for example, International Patent Publication No. WO 01\/34574). In particular, compounds such as Compound 66, or similar compounds, can be administered to a patient in doses ranging from 5-200 mg per day, or 100-1600 mg per week, in single or divided doses, or by continuous infusion. A preferred dose is 150 mg\/day. Antibody-based IGF-1R kinase inhibitors, or antisense, RNAi or ribozyme constructs, can be administered to a patient in doses ranging from 0.1 to 100 mg\/kg of body weight per day or per week in single or divided doses, or by continuous infusion. In some instances, dosage levels below the lower limit of the aforesaid range may be more than adequate, while in other cases still larger doses may be employed without causing any harmful side effect, provided that such larger doses are first divided into several small doses for administration throughout the day.","The IGF-1R kinase inhibitors and other additional agents can be administered either separately or together by the same or different routes, and in a wide variety of different dosage forms. For example, the IGF-1R kinase inhibitor is preferably administered orally or parenterally. Where the IGF-1R kinase inhibitor is Compound 66, or a similar such compound, oral administration is preferable. Both the IGF-1R kinase inhibitor and other additional agents can be administered in single or multiple doses.","The IGF-1R kinase inhibitor can be administered with various pharmaceutically acceptable inert carriers in the form of tablets, capsules, lozenges, troches, hard candies, powders, sprays, creams, salves, suppositories, jellies, gels, pastes, lotions, ointments, elixirs, syrups, and the like. Administration of such dosage forms can be carried out in single or multiple doses. Carriers include solid diluents or fillers, sterile aqueous media and various non-toxic organic solvents, etc. Oral pharmaceutical compositions can be suitably sweetened and\/or flavored.","The IGF-1R kinase inhibitor can be combined together with various pharmaceutically acceptable inert carriers in the form of sprays, creams, salves, suppositories, jellies, gels, pastes, lotions, ointments, and the like. Administration of such dosage forms can be carried out in single or multiple doses. Carriers include solid diluents or fillers, sterile aqueous media, and various non-toxic organic solvents, etc.","All formulations comprising proteinaceous IGF-1R kinase inhibitors should be selected so as to avoid denaturation and\/or degradation and loss of biological activity of the inhibitor.","Methods of preparing pharmaceutical compositions comprising an IGF-1R kinase inhibitor are known in the art, and are described, e.g. in International Patent Publication No. WO 01\/34574. In view of the teaching of the present invention, methods of preparing pharmaceutical compositions comprising an IGF-1R kinase inhibitor will be apparent from the above-cited publications and from other known references, such as Remington's Pharmaceutical Sciences, Mack Publishing Company, Easton, Pa., 18edition (1990).","For oral administration of IGF-1R kinase inhibitors, tablets containing one or both of the active agents are combined with any of various excipients such as, for example, micro-crystalline cellulose, sodium citrate, calcium carbonate, dicalcium phosphate and glycine, along with various disintegrants such as starch (and preferably corn, potato or tapioca starch), alginic acid and certain complex silicates, together with granulation binders like polyvinyl pyrrolidone, sucrose, gelatin and acacia. Additionally, lubricating agents such as magnesium stearate, sodium lauryl sulfate and talc are often very useful for tableting purposes. Solid compositions of a similar type may also be employed as fillers in gelatin capsules; preferred materials in this connection also include lactose or milk sugar as well as high molecular weight polyethylene glycols. When aqueous suspensions and\/or elixirs are desired for oral administration, the IGF-1R kinase inhibitor may be combined with various sweetening or flavoring agents, coloring matter or dyes, and, if so desired, emulsifying and\/or suspending agents as well, together with such diluents as water, ethanol, propylene glycol, glycerin and various like combinations thereof.","For parenteral administration of either or both of the active agents, solutions in either sesame or peanut oil or in aqueous propylene glycol may be employed, as well as sterile aqueous solutions comprising the active agent or a corresponding water-soluble salt thereof. Such sterile aqueous solutions are preferably suitably buffered, and are also preferably rendered isotonic, e.g., with sufficient saline or glucose. These particular aqueous solutions are especially suitable for intravenous, intramuscular, subcutaneous and intraperitoneal injection purposes. The oily solutions are suitable for intra-articular, intramuscular and subcutaneous injection purposes. The preparation of all these solutions under sterile conditions is readily accomplished by standard pharmaceutical techniques well known to those skilled in the art. Any parenteral formulation selected for administration of proteinaceous IGF-1R kinase inhibitors should be selected so as to avoid denaturation and loss of biological activity of the inhibitor.","Additionally, it is possible to topically administer either or both of the active agents, by way of, for example, creams, lotions, jellies, gels, pastes, ointments, salves and the like, in accordance with standard pharmaceutical practice. For example, a topical formulation comprising an IGF-1R kinase inhibitor in about 0.1% (w\/v) to about 5% (w\/v) concentration can be prepared.","For veterinary purposes, the active agents can be administered separately or together to animals using any of the forms and by any of the routes described above. In a preferred embodiment, the IGF-1R kinase inhibitor is administered in the form of a capsule, bolus, tablet, liquid drench, by injection or as an implant. As an alternative, the IGF-1R kinase inhibitor can be administered with the animal feedstuff, and for this purpose a concentrated feed additive or premix may be prepared for a normal animal feed. Such formulations are prepared in a conventional manner in accordance with standard veterinary practice.","As used herein, the term \u201cIGF-1R kinase inhibitor\u201d refers to any IGF-1R kinase inhibitor that is currently known in the art or that will be identified in the future, and includes any chemical entity that, upon administration to a patient, results in inhibition of a biological activity specifically associated with activation of the IGF-1 receptor in the patient, and resulting from the binding to IGF-1R of its natural ligand(s). Such IGF-1R kinase inhibitors include any agent that can block IGF-1R activation and the downstream biological effects of IGF-1R activation that are relevant to treating cancer in a patient. Such an inhibitor can act by binding directly to the intracellular domain of the receptor and inhibiting its kinase activity. Alternatively, such an inhibitor can act by occupying the ligand binding site or a portion thereof of the IGF-1 receptor, thereby making the receptor inaccessible to its natural ligand so that its normal biological activity is prevented or reduced. Alternatively, such an inhibitor can act by modulating the dimerization of IGF-1R polypeptides, or interaction of IGF-1R polypeptide with other proteins, or enhance ubiquitination and endocytotic degradation of IGF-1R. An IGF-1R kinase inhibitor can also act by reducing the amount of IGF-1 available to activate IGF-1R, by for example antagonizing the binding of IGF-1 to its receptor, by reducing the level of IGF-1, or by promoting the association of IGF-1 with proteins other than IGF-1R such as IGF binding proteins (e.g. IGFBP3). IGF-1R kinase inhibitors include but are not limited to low molecular weight inhibitors, antibodies or antibody fragments, antisense constructs, small inhibitory RNAs (i.e. RNA interference by dsRNA; RNAi), and ribozymes. In a preferred embodiment, the IGF-1R kinase inhibitor is a small organic molecule or an antibody that binds specifically to the human IGF-1R.","IGF-1R kinase inhibitors include, for example imidazopyrazine IGF-1R kinase inhibitors, quinazoline IGF-1R kinase inhibitors, pyrido-pyrimidine IGF-1R kinase inhibitors, pyrimido-pyrimidine IGF-1R kinase inhibitors, pyrrolo-pyrimidine IGF-1R kinase inhibitors, pyrazolo-pyrimidine IGF-1R kinase inhibitors, phenylamino-pyrimidine IGF-1R kinase inhibitors, oxindole IGF-1R kinase inhibitors, indolocarbazole IGF-1R kinase inhibitors, phthalazine IGF-1R kinase inhibitors, isoflavone IGF-1R kinase inhibitors, quinalone IGF-1R kinase inhibitors, and tyrphostin IGF-1R kinase inhibitors, and all pharmaceutically acceptable salts and solvates of such IGF-1R kinase inhibitors.","Additional examples of IGF-1R kinase inhibitors include those in International Patent Publication No. WO 05\/097800, that describes 6,6-bicyclic ring substituted heterobicyclic protein kinase inhibitors, International Patent Publication No. WO 05\/037836, that describes imidazopyrazine IGF-1R kinase inhibitors, International Patent Publication Nos. WO 03\/018021 and WO 03\/018022, that describe pyrimidines for treating IGF-1R related disorders, International Patent Publication Nos. WO 02\/102804 and WO 02\/102805, that describe cyclolignans and cyclolignans as IGF-1R inhibitors, International Patent Publication No. WO 02\/092599, that describes pyrrolopyrimidines for the treatment of a disease which responds to an inhibition of the IGF-1R tyrosine kinase, International Patent Publication No. WO 01\/72751, that describes pyrrolopyrimidines as tyrosine kinase inhibitors, and in International Patent Publication No. WO 00\/71129, that describes pyrrolotriazine inhibitors of kinases, and in International Patent Publication No. WO 97\/28161, that describes pyrrolo[2,3-d]pyrimidines and their use as tyrosine kinase inhibitors, Parrizas, et al., which describes tyrphostins with in vitro and in vivo IGF-1R inhibitory activity (Endocrinology, 138:1427-1433 (1997)), International Patent Publication No. WO 00\/35455, that describes heteroaryl-aryl ureas as IGF-1R inhibitors, International Patent Publication No. WO 03\/048133, that describes pyrimidine derivatives as modulators of IGF-1R, International Patent Publication No. WO 03\/024967, WO 03\/035614, WO 03\/035615, WO 03\/035616, and WO 03\/035619, that describe chemical compounds with inhibitory effects towards kinase proteins, International Patent Publication No. WO 03\/068265, that describes methods and compositions for treating hyperproliferative conditions, International Patent Publication No. WO 00\/17203, that describes pyrrolopyrimidines as protein kinase inhibitors, Japanese Patent Publication No. JP 07\/133280, that describes a cephem compound, its production and antimicrobial composition, Albert, A. et al., 11: 1540-1547 (1970), which describes pteridine studies and pteridines unsubstituted in the 4-position, and A. Albert et al., 44: 1-5 (1969) which describes a synthesis of pteridines (unsubstituted in the 4-position) from pyrazines, via 3-4-dihydropteridines.","IGF-1R kinase inhibitors particularly useful in this invention include compounds represented by Formula (I) (see below), as described in US Published Patent Application US 2006\/0235031, where their preparation is described in detail. PQIP (cis-3-[3-(4-Methyl-piperazin-1-yl)-cyclobutyl]1-(2-phenyl-quinolin-7-yl)-imidazo[1,5-a]pyrazin-8-ylamine) and OSI-906 (cis-3-[8-amino-1-(2-phenyl-quinolin-7-yl)-imidazo[1,5-a]pyrazin-3-yl]-1-methyl-cyclobutanol) represents IGF-1R kinase inhibitors according to Formula (I).","OSI-906 has the structure as follows:",{"@attributes":{"id":"p-0265","num":"0264"},"chemistry":{"@attributes":{"id":"CHEM-US-00001","num":"00001"},"img":{"@attributes":{"id":"EMI-C00001","he":"55.96mm","wi":"47.84mm","file":"US08163509-20120424-C00001.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"PQIP has the structure as follows:",{"@attributes":{"id":"p-0267","num":"0266"},"chemistry":{"@attributes":{"id":"CHEM-US-00002","num":"00002"},"img":{"@attributes":{"id":"EMI-C00002","he":"75.95mm","wi":"64.69mm","file":"US08163509-20120424-C00002.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"An IGF-1R kinase inhibitor of Formula (I), as described in US Published Patent Application US 2006\/0235031, is represented by the formula:",{"@attributes":{"id":"p-0269","num":"0268"},"chemistry":{"@attributes":{"id":"CHEM-US-00003","num":"00003"},"img":{"@attributes":{"id":"EMI-C00003","he":"25.15mm","wi":"49.11mm","file":"US08163509-20120424-C00003.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"or a pharmaceutically acceptable salt thereof, wherein:","X, and Xare each independently N or C-(E);","Xis N, C-(E), or N-(E);","X, X, X, and Xare each independently N or C;\n\n","Qis",{"@attributes":{"id":"p-0275","num":"0275"},"chemistry":{"@attributes":{"id":"CHEM-US-00004","num":"00004"},"img":{"@attributes":{"id":"EMI-C00004","he":"26.25mm","wi":"33.87mm","file":"US08163509-20120424-C00004.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"X, X, X, X, X, and Xare each independently N, C-(E), or N\u2014O;","wherein at least one of X, X, X, X, X, and Xis N or N\u2014O;","Ris absent, Calkyl, cycloCalkyl, bicycloCalkyl, aryl, heteroaryl, aralkyl, heteroaralkyl, heterocyclyl, heterobicycloCalkyl, spiroalkyl, or heterospiroalkyl, any of which is optionally substituted by one or more independent Gsubstituents;","E, E, G, and Gare each independently halo, \u2014CF, \u2014OCF, \u2014OR, \u2014NRR(R), \u2014C(\u2550O)R, \u2014COR, \u2014CONRR, \u2014NO, \u2014CN, \u2014S(O)R, \u2014SONRR, \u2014NRC(\u2550O)R, \u2014NRC(\u2550O)OR, \u2014NRC(\u2550O)NRR, \u2014NRS(O)R, \u2014C(\u2550S)OR, \u2014C(\u2550O)SR, \u2014NRC(\u2550NR)NRR, \u2014NRC(\u2550NR)OR, \u2014NRC(\u2550NR)SR, \u2014OC(\u2550O)OR, \u2014OC(\u2550O)NRR, \u2014OC(\u2550O)SR, \u2014SC(\u2550O)OR, \u2014SC(\u2550O)NRR, Calkyl, Calkenyl, Calkynyl, CalkoxyCalkyl, CalkoxyCalkenyl, CalkoxyCalkynyl, CalkylthioCalkyl, CalkylthioCalkenyl, CalkylthioCalkynyl, cycloCalkyl, cycloCalkenyl, cycloCalkylCalkyl, cycloCalkenylCalkyl, cycloCalkylCalkenyl, cycloCalkenylCalkenyl, cycloCalkylCalkynyl, cycloCalkenylCalkynyl, heterocyclyl-Calkyl, heterocyclyl-Calkenyl, or heterocyclyl-Calkynyl, any of which is optionally substituted with one or more independent halo, oxo, \u2014CF, \u2014OR, \u2014NRR(R), \u2014C(\u2550O)R, \u2014COR, \u2014C(\u2550O)NRR, \u2014NO, \u2014CN, \u2014S(\u2550O)R, \u2014SONRR, \u2014NRC(\u2550O)R, \u2014NRC(\u2550O)OR, \u2014NRC(\u2550O)NRR, \u2014NRS(O)R, C(\u2550S)OR, \u2014C(\u2550O)SR, \u2014NRC(\u2550NR)NRR, \u2014NRC(\u2550NR)OR, \u2014NRC(\u2550NR)SR, \u2014OC(\u2550O)OR, \u2014OC(\u2550O)NRR, \u2014OC(\u2550O)SR, \u2014SC(\u2550O)OR, or \u2014SC(\u2550O)NRRsubstituents;","or E, E, or Goptionally is \u2014(W)\u2014(Y)\u2014R;","or E, E, G, or Goptionally independently is aryl-Calkyl, aryl-Calkenyl, aryl-Calkynyl, hetaryl-Calkyl, hetaryl-Calkenyl, or hetaryl-Calkynyl, any of which is optionally substituted with one or more independent halo, \u2014CF, \u2014OCF, \u2014OR, \u2014NRR(R), \u2014C(O)R, \u2014COR, \u2014C(\u2550O)NRR, \u2014NO, \u2014CN, \u2014S(O)R, \u2014SONRR, \u2014NRC(\u2550O)R, \u2014NRC(\u2550O)OR, \u2014NRC(\u2550O)NRR, \u2014NRS(O)R, \u2014C(\u2550S)OR, \u2014C(\u2550O)SR, \u2014NRC(\u2550NR)NRR, \u2014NRC(\u2550NR)OR, \u2014NRC(\u2550NR)SR, \u2014OC(\u2550O)OR, \u2014OC(\u2550O)NRR, \u2014OC(\u2550O)SR, \u2014SC(\u2550O)OR, or \u2014SC(\u2550O)NRRsubstituents;","Gis halo, oxo, \u2014CF, \u2014OCF, \u2014OR, \u2014NRR(R), \u2014C(O)R, \u2014COR, \u2014C(\u2550O)NRR, \u2014NO, \u2014CN, \u2014S(O)R, \u2014SONRR, NR(C\u2550O)R, NRC(\u2550O)OR, NRC(\u2550O)NRR, NRS(O)R, \u2014C(\u2550S)OR, \u2014C(\u2550O)SR, \u2014NRC(\u2550NR)NRR, \u2014NRC(\u2550NR)OR, \u2014NRC(\u2550NR)SR, \u2014OC(\u2550O)OR, \u2014OC(\u2550)NRR, \u2014OC(\u2550)SR, \u2014SC(\u2550O)OR, \u2014SC(\u2550O)NRR, \u2014P(O)OROR, Calkylidene, Calkyl, Calkenyl, Calkynyl, CalkoxyCalkyl, CalkoxyCalkenyl, CalkoxyCalkynyl, CalkylthioCalkyl, CalkylthioCalkenyl, CalkylthioCalkynyl, cycloCalkyl, cycloCalkyl, cycloCalkenyl, cycloCalkylCalkyl, cycloCalkenylCalkyl, cycloCalkylCalkenyl, cycloCalkenylCalkenyl, cycloCalkylCalkynyl, cycloCalkenylCalkynyl, heterocyclyl-Calkyl, heterocyclyl-Calkenyl, or heterocyclyl-Calkynyl, any of which is optionally substituted with one or more independent halo, oxo, \u2014CF, \u2014OCF, \u2014OR, \u2014NRR(R), \u2014C(O)R, \u2014COR, \u2014C(\u2550O)NRR, \u2014NO, \u2014CN, \u2014S(O)R, \u2014SONRR, \u2014NRC(\u2550O)R, \u2014NRC(\u2550O)OR, \u2014NRC(\u2550O)NRR, \u2014NRS(O)R, \u2014C(\u2550S)OR, \u2014C(\u2550O)SR, \u2014NRC(\u2550NR)NRR, \u2014NRC(\u2550NR)OR, \u2014NRC(\u2550NR)SR, \u2014OC(\u2550O)OR, \u2014OC(\u2550O)NRR, \u2014OC(\u2550O)SR, \u2014SC(\u2550O)OR, \u2014P(O)OROR, or \u2014SC(\u2550O)NRRsubstituents;","or Gis aryl-Calkyl, aryl-Calkenyl, aryl-Calkynyl, hetaryl-Calkyl, hetaryl-Calkenyl, or hetaryl-Calkynyl, any of which is optionally substituted with one or more independent halo, \u2014CF, \u2014OCF, \u2014OR, \u2014NRR(R), \u2014C(O)R, \u2014COR, \u2014(\u2550O)NRR, \u2014NO, \u2014CN, \u2014S(O)R, \u2014SONRR, \u2014NRC(\u2550O)R, \u2014NRC(\u2550O)OR, \u2014NRC(\u2550O)NRR, \u2014NRS(O)R, \u2014C(\u2550S)OR, \u2014C(\u2550O)SR, \u2014NRC(\u2550NR)NRR, \u2014NRC(\u2550NR)OR, \u2014NRC(\u2550NR)SR, \u2014OC(\u2550O)OR, \u2014OC(\u2550O)NRR, \u2014OC(\u2550O)SR, \u2014SC(\u2550O)OR, \u2014P(O)OROR, or \u2014SC(\u2550O)NRRsubstituents;","or Gis C, taken together with the carbon to which it is attached forms a C\u2550C double bond which is substituted with Rand G;","R, R, R, R, R, R, R, R, R, R, R, R, R, R, R, and Rare each independently Calkyl, Calkenyl, Calkynyl, CalkoxyCalkyl, CalkoxyCalkenyl, CalkoxyCalkynyl, CalkylthioCalkyl, CalkylthioCalkenyl, CalkylthioCalkynyl, cycloCalkyl, cycloCalkenyl, cycloCalkylCalkyl, cycloCalkenylCalkyl, cycloCalkylCalkenyl, cycloCalkenylCalkenyl, cycloCalkylCalkynyl, cycloCalkenylCalkynyl, heterocyclyl-Calkyl, heterocyclyl-Calkenyl, heterocyclyl-Calkynyl, aryl-Calkyl, aryl-Calkenyl, or aryl-Calkynyl, hetaryl-Calkyl, hetaryl-Calkenyl, or hetaryl-Calkynyl, any of which is optionally substituted by one or more independent Gsubstituents;","or in the case of \u2014NRR(R)or \u2014NRR(R)or \u2014NRR(R)or \u2014NRR(R)or \u2014NRR(R)or \u2014NRR(R), then Rand R, or Rand R, or Rand R, respectfully, are optionally taken together with the nitrogen atom to which they are attached to form a 3-10 membered saturated or unsaturated ring, wherein said ring is optionally substituted by one or more independent Gsubstituents and wherein said ring optionally includes one or more heteroatoms other than the nitrogen to which Rand R, or Rand R, or Rand Rare attached;","Wand Yare each independently \u2014O\u2014, \u2014NR\u2014, \u2014S(O)\u2014, \u2014CRR\u2014, \u2014N(C(O)OR)\u2014, \u2014N(C(O)R)\u2014, \u2014N(SOR)\u2014, \u2014CHO\u2014, \u2014CHS\u2014, \u2014CHN(R)\u2014, \u2014CH(NR)\u2014, \u2014CHN(C(O)R)\u2014, \u2014CHN(C(O)OR)\u2014, \u2014CHN(SOR)\u2014, \u2014CH(NHR)\u2014, \u2014CH(NHC(O)R)\u2014, \u2014CH(NHSOR)\u2014, \u2014CH(NHC(O)OR)\u2014, \u2014CH(OC(O)R)\u2014, \u2014CH(OC(O)NHR)\u2014, \u2014CH\u2550CH\u2014, \u2014C\u2261CH\u2014, \u2014C(\u2550NOR)\u2014, \u2014C(O)\u2014, \u2014CH(OR)\u2014, \u2014C(O)N(R)\u2014, \u2014N(R)C(O)\u2014, \u2014N(R)S(O)\u2014, \u2014N(R)S(O)\u2014 \u2014OC(O)N(R)\u2014, \u2014N(R)C(O)N(R)\u2014, \u2014NRC(O)O\u2014, \u2014S(O)N(R)\u2014, \u2014S(O)N(R)\u2014, \u2014N(C(O)R)S(O)\u2014, \u2014N(C(O)R)S(O)\u2014, \u2014N(R)S(O)N(R)\u2014, \u2014N(R)S(O)N(R)\u2014, \u2014C(O)N(R)C(O)\u2014, \u2014S(O)N(R)C(O)\u2014, \u2014S(O)N(R)C(O)\u2014, \u2014OS(O)N(R)\u2014, \u2014OS(O)N(R)\u2014, \u2014N(R)S(O)O\u2014, \u2014N(R)S(O)O\u2014, \u2014N(R)S(O)C(O)\u2014, \u2014N(R)S(O)C(O)\u2014, \u2014SON(C(O)R)\u2014, \u2014SON(C(O)R)\u2014, \u2014N(R)SON(R)\u2014, \u2014N(R)SON(R)\u2014, \u2014C(O)O\u2014, \u2014N(R)P(OR)O\u2014, \u2014N(R)P(OR)\u2014, \u2014N(R)P(O)(OR)O\u2014, \u2014N(R)P(O)(OR)\u2014, \u2014N(C(O)R)P(OR)O\u2014, \u2014N(C(O)R)P(OR)\u2014, \u2014N(C(O)R)P(O)(OR)O\u2014, \u2014N(C(O)R)P(OR)\u2014, \u2014CH(R)S(O)\u2014, \u2014CH(R)S(O)\u2014, \u2014CH(R)N(C(O)OR)\u2014, \u2014CH(R)N(C(O)R)\u2014, \u2014CH(R)N(SOR)\u2014, \u2014CH(R)O\u2014, \u2014CH(R)S\u2014, \u2014CH(R)N(R)\u2014, \u2014CH(R)N(C(O)R)\u2014, \u2014CH(R)N(C(O)OR)\u2014, \u2014CH(R)N(SOR)\u2014, \u2014CH(R)C(\u2550NOR)\u2014, \u2014CH(R)C(O)\u2014, \u2014CH(R)CH(OR)\u2014, \u2014CH(R)C(O)N(R)\u2014, \u2014CH(R)N(R)C(O)\u2014, \u2014CH(R)N(R)S(O)\u2014, \u2014CH(R)N(R)S(O)\u2014, \u2014CH(R)OC(O)N(R)\u2014, \u2014CH(R)N(R)C(O)N(R)\u2014, \u2014CH(R)NRC(O)O\u2014, \u2014CH(R)S(O)N(R)\u2014, \u2014CH(R)S(O)N(R)\u2014, \u2014CH(R)N(C(O)R)S(O)\u2014, \u2014CH(R)N(C(O)R)S(O)\u2014, \u2014CH(ON(R)S(O)N(R)\u2014, \u2014CH(R)N(R)S(O)N(R)\u2014, \u2014CH(R)C(O)N(R)C(O)\u2014, \u2014CH(R)S(O)N(R)C(O)\u2014, \u2014CH(R)S(O)N(R)C(O)\u2014, \u2014CH(R)OS(O)N(R)\u2014, \u2014CH(R)OS(O)N(R)\u2014, \u2014CH(R)N(R)S(O)O\u2014, \u2014CH(R)N(R)S(O)O\u2014, \u2014CH(R)N(R)S(O)C(O)\u2014, \u2014CH(R)N(R)S(O)C(O)\u2014, \u2014CH(R)SON(C(O)R)\u2014, \u2014CH(R)SON(C(O)R)\u2014, \u2014CH(R)N(R)SON(R)\u2014, \u2014CH(R)N(R)SON(R)\u2014, \u2014CH(R)C(O)O\u2014, \u2014CH(R)N(R)P(OR)O\u2014, \u2014CH(R)N(R)P(OR)\u2014, \u2014CH(R)N(R)P(O)(OR)O\u2014, \u2014CH(R)N(R)P(O)(OR)\u2014, \u2014CH(R)N(C(O)R)P(OR)O\u2014, \u2014CH(R)N(C(O)R)O(OR)\u2014, \u2014CH(R)N(C(O)R)P(O)(OR)O\u2014, or \u2014CH(R)N(C(O)R)P(OR)\u2014;","R, R, G, and Gare each independently Calkyl, Calkenyl, Calkynyl, CalkoxyCalkyl, CalkoxyCalkenyl, CalkoxyCalkynyl, CalkylthioCalkyl, CalkylthioCalkenyl, CalkylthioCalkynyl, cycloCalkyl, cycloCalkenyl, cycloCalkylCalkyl, cycloCalkenylCalkyl, cycloCalkylCalkenyl, cycloCalkenylCalkenyl, cycloCalkylCalkynyl, cycloCalkenylCalkynyl, heterocyclyl-Calkyl, heterocyclyl-Calkenyl, heterocyclyl-Calkynyl, aryl-Calkyl, aryl-Calkenyl, aryl-Calkynyl, hetaryl-Calkyl, hetaryl-Calkenyl, or hetaryl-Calkynyl, any of which is optionally substituted with one or more independent halo, \u2014CF, \u2014OCF, \u2014OR, \u2014NRR, \u2014C(O)R, \u2014COR, \u2014CONRR, \u2014NO, \u2014CN, \u2014S(O)R, \u2014SONRR, \u2014NRC(\u2550O)R, \u2014NRC(\u2550O)OR, \u2014NRC(\u2550O)NRR, \u2014NRS(O)R, \u2014C(\u2550S)OR, \u2014C(\u2550O)SR, \u2014NRC(\u2550NR)NRR, \u2014NRC(\u2550NR)OR, \u2014NRC(\u2550NR)SR, \u2014OC(\u2550O)OR, \u2014OC(\u2550O)NRR, \u2014OC(\u2550O)SR, \u2014SC(\u2550O)OR, \u2014P(O)OROR, or \u2014SC(\u2550O)NRRsubstituents;","or Rwith Rare optionally taken together with the carbon atom to which they are attached to form a 3-10 membered saturated or unsaturated ring, wherein said ring is optionally substituted with one or more independent Rsubstituents and wherein said ring optionally includes one or more heteroatoms;","R, R, and Rare each independently acyl, Calkyl, Calkenyl, aryl, heteroaryl, heterocyclyl or cycloCalkyl, any of which is optionally substituted by one or more independent Gsubstituents;","Ris Calkyl, Calkenyl, Calkynyl, aryl, heteroaryl, cycloCalkyl, heteroclycl, cycloCalkenyl, or heterocycloalkenyl, any of which is optionally substituted by one or more independent Gsubstituents;","Ris halo, \u2014OR, \u2014SH, \u2014NRR, \u2014COR, \u2014C(\u2550O)NRR, \u2014NO, \u2014CN, \u2014S(O)R, \u2014SONRR, Calkyl, Calkenyl, Calkynyl, CalkoxyCalkyl, CalkoxyCalkenyl, CalkoxyCalkynyl, CalkylthioCalkyl, CalkylthioCalkenyl, CalkylthioCalkynyl, cycloCalkyl, cycloCalkenyl, cycloCalkylCalkyl, cycloCalkenylCalkyl, cycloCalkylCalkenyl, cycloCalkenylCalkenyl, cycloCalkylCalkynyl, cycloCalkenylCalkynyl, heterocyclyl-Calkyl, heterocyclyl-Calkenyl, or heterocyclyl-Calkynyl, any of which is optionally substituted with one or more independent halo, cyano, nitro, \u2014OR, \u2014SONRR, or \u2014NRRsubstituents;","or Ris aryl-Calkyl, aryl-Calkenyl, aryl-Calkynyl, hetaryl-Calkyl, hetaryl-Calkenyl, hetaryl-Calkynyl, mono(Calkyl)aminoCalkyl, di(Calkyl)aminoCalkyl, mono(aryl)aminoCalkyl, di(aryl)aminoCalkyl, or \u2014N(Calkyl)-Calkyl-aryl, any of which is optionally substituted with one or more independent halo, cyano, nitro, \u2014OR, Calkyl, Calkenyl, Calkynyl, haloCalkyl, haloCalkenyl, haloCalkynyl, \u2014COOH, Calkoxycarbonyl, \u2014C(\u2550O)NRR, \u2014SONRR, or \u2014NRRsubstituents;","or in the case of \u2014NRR, Rand Rare optionally taken together with the nitrogen atom to which they are attached to form a 3-10 membered saturated or unsaturated ring, wherein said ring is optionally substituted with one or more independent halo, cyano, hydroxy, nitro, Calkoxy, \u2014SONRR, or \u2014NRRsubstituents, and wherein said ring optionally includes one or more heteroatoms other than the nitrogen to which Rand Rare attached;","R, R, R, R, R, and Rare each independently Calkyl, Calkenyl, Calkynyl, CalkoxyCalkyl, CalkoxyCalkenyl, CalkoxyCalkynyl, CalkylthioCalkyl, CalkylthioCalkenyl, CalkylthioCalkynyl, cycloCalkyl, cycloCalkenyl, cycloCalkylCalkyl, cycloCalkenylCalkyl, cycloCalkylCalkenyl, cycloCalkenylCalkenyl, cycloCalkylCalkynyl, cycloCalkenylCalkynyl, heterocyclyl-Calkyl, heterocyclyl-Calkenyl, heterocyclyl-Calkynyl, Calkylcarbonyl, Calkenylcarbonyl, Calkynylcarbonyl, Calkoxycarbonyl, CalkoxycarbonylCalkyl, monoCalkylaminocarbonyl, diCalkylaminocarbonyl, mono(aryl)aminocarbonyl, di(aryl)aminocarbonyl, or Calkyl(aryl)aminocarbonyl, any of which is optionally substituted with one or more independent halo, cyano, hydroxy, nitro, Calkoxy, \u2014SON(Calkyl)(Calkyl), or \u2014N(Calkyl)(Calkyl) substituents;","or R, R, R, R, R, and Rare each independently aryl-Calkyl, aryl-Calkenyl, aryl-Calkynyl, hetaryl-Calkyl, hetaryl-Calkenyl, hetaryl-Calkynyl, mono(Calkyl)aminoCalkyl, di(Calkyl)aminoCalkyl, mono(aryl)aminoCalkyl, di(aryl)aminoCalkyl, or \u2014N(Calkyl)-Calkyl-aryl, any of which is optionally substituted with one or more independent halo, cyano, nitro, \u2014O(Calkyl), Calkyl, Calkenyl, Calkynyl, haloCalkyl, haloCalkenyl, haloCalkynyl, \u2014COOH, Calkoxycarbonyl, \u2014CON(Calkyl)(Calkyl), \u2014SON(Calkyl)(Calkyl), or \u2014N(Calkyl)(Calkyl) substituents;","n, m, j1, j1a, j2a, j4, j4a, j5a, j7, and j8 are each independently 0, 1, or 2; and aa and bb are each independently 0 or 1.","Additional, specific examples of IGF-1R kinase inhibitors that can be used according to the present invention include h7C10 (Centre de Recherche Pierre Fabre), an IGF-1 antagonist; EM-164 (ImmunoGen Inc.), an IGF-1R modulator; CP-751871 (figitumumab; Pfizer Inc.), an IGF-1 antagonist; lanreotide (Ipsen), an IGF-1 antagonist; IGF-1R oligonucleotides (Lynx Therapeutics Inc.); IGF-1 oligonucleotides (National Cancer Institute); IGF-1R protein-tyrosine kinase inhibitors in development by Novartis (e.g. NVP-AEW541, Garcia-Echeverria, C. et al. (2004) Cancer Cell 5:231-239; or NVP-ADW742, Mitsiades, C. S. et al. (2004) Cancer Cell 5:221-230); IGF-1R protein-tyrosine kinase inhibitors (Ontogen Corp); OSI-906 (OSI Pharmaceuticals); AG-1024 (Camirand, A. et al. (2005) Breast Cancer Research 7:R570-R579 (DOI 10.1186\/bcr1028); Camirand, A. and Pollak, M. (2004) Brit. J. Cancer 90:1825-1829; Pfizer Inc.), an IGF-1 antagonist; the tyrphostins AG-538 and 1-OMe-AG 538; BMS-536924, a small molecule inhibitor of IGF-1R; PNU-145156E (Pharmacia & Upjohn SpA), an IGF-1 antagonist; BMS 536924, a dual IGF-1R and 1R kinase inhibitor (Bristol-Myers Squibb); AEW541 (Novartis); GSK621659A (Glaxo Smith-Kline); INSM-18 (Insmed); and XL-228 (Exelixis).","Antibody-based IGF-1R kinase inhibitors include any anti-IGF-1R antibody or antibody fragment that can partially or completely block IGF-1R activation by its natural ligand. Antibody-based IGF-1R kinase inhibitors also include any anti-IGF-1 antibody or antibody fragment that can partially or completely block IGF-1R activation. Non-limiting examples of antibody-based IGF-1R kinase inhibitors include those described in Larsson, O. et al (2005) Brit. J. Cancer 92:2097-2101 and Ibrahim, Y. H. and Yee, D. (2005) Clin. Cancer Res. 11:944s-950s, or being developed by Imclone (e.g. A12) or Schering-Plough Research Institute (e.g. 19D12; or as described in US Patent Application Publication Nos. US 2005\/0136063 A1 and US 2004\/0018191 A1). The IGF-1R kinase inhibitor can be a monoclonal antibody, or an antibody or antibody fragment having the binding specificity thereof.","Additional antibody-based IGF-1R kinase inhibitors can be raised according to known methods by administering the appropriate antigen or epitope to a host animal selected, e.g., from pigs, cows, horses, rabbits, goats, sheep, and mice, among others. Various adjuvants known in the art can be used to enhance antibody production.","Although antibodies useful in practicing the invention can be polyclonal, monoclonal antibodies are preferred. Monoclonal antibodies against IGF-1R can be prepared and isolated using any technique that provides for the production of antibody molecules by continuous cell lines in culture. Techniques for production and isolation include but are not limited to the hybridoma technique originally described by Kohler and Milstein (Nature, 1975, 256: 495-497); the human B-cell hybridoma technique (Kosbor et al., 1983, Immunology Today 4:72; Cote et al., 1983, Proc. Nati. Acad. Sci. USA 80: 2026-2030); and the EBV-hybridoma technique (Cole et al, 1985, Monoclonal Antibodies and Cancer Therapy, Alan R. Liss, Inc., pp. 77-96).","Alternatively, techniques described for the production of single chain antibodies (see, e.g., U.S. Pat. No. 4,946,778) can be adapted to produce anti-IGF-1R single chain antibodies. Antibody-based IGF-1R kinase inhibitors useful in practicing the present invention also include anti-IGF-1R antibody fragments including but not limited to F(ab\u2032).sub.2 fragments, which can be generated by pepsin digestion of an intact antibody molecule, and Fab fragments, which can be generated by reducing the disulfide bridges of the F(ab\u2032).sub.2 fragments. Alternatively, Fab and\/or scFv expression libraries can be constructed (see, e.g., Huse et al., 1989, Science 246: 1275-1281) to allow rapid identification of fragments having the desired specificity to IGF-1R.","Techniques for the production and isolation of monoclonal antibodies and antibody fragments are well-known in the art, and are described in Harlow and Lane, 1988, Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory, and in J. W. Goding, 1986, Monoclonal Antibodies: Principles and Practice, Academic Press, London. Humanized anti-IGF-1R antibodies and antibody fragments can also be prepared according to known techniques such as those described in Vaughn, T. J. et al., 1998, Nature Biotech. 16:535-539 and references cited therein, and such antibodies or fragments thereof are also useful in practicing the present invention.","IGF-1R kinase inhibitors for use in the present invention can alternatively be based on antisense oligonucleotide constructs. Anti-sense oligonucleotides, including anti-sense RNA molecules and anti-sense DNA molecules, would act to directly block the translation of IGF-1R mRNA by binding thereto and thus preventing protein translation or increasing mRNA degradation, thus decreasing the level of IGF-1R kinase protein, and thus activity, in a cell. For example, antisense oligonucleotides of at least about 15 bases and complementary to unique regions of the mRNA transcript sequence encoding IGF-1R can be synthesized, e.g., by conventional phosphodiester techniques and administered by e.g., intravenous injection or infusion. Methods for using antisense techniques for specifically inhibiting gene expression of genes whose sequence is known are well known in the art (e.g. see U.S. Pat. Nos. 6,566,135; 6,566,131; 6,365,354; 6,410,323; 6,107,091; 6,046,321; and 5,981,732).","Small inhibitory RNAs (siRNAs) can also function as IGF-1R kinase inhibitors for use in the present invention. IGF-1R gene expression can be reduced by contacting the tumor, subject or cell with a small double stranded RNA (dsRNA), or a vector or construct causing the production of a small double stranded RNA, such that expression of IGF-1R is specifically inhibited (i.e. RNA interference or RNAi). Methods for selecting an appropriate dsRNA or dsRNA-encoding vector are well known in the art for genes whose sequence is known (e.g. see Tuschi, T., et al. (1999) Genes Dev. 13(24):3191-3197; Elbashir, S. M. et al. (2001) Nature 411:494-498; Hannon, G. J. (2002) Nature 418:244-251; McManus, M. T. and Sharp, P. A. (2002) Nature Reviews Genetics 3:737-747; Bremmelkamp, T. R. et al. (2002) Science 296:550-553; U.S. Pat. Nos. 6,573,099 and 6,506,559; and International Patent Publication Nos. WO 01\/36646, WO 99\/32619, and WO 01\/68836).","Ribozymes can also function as IGF-1R kinase inhibitors for use in the present invention. Ribozymes are enzymatic RNA molecules capable of catalyzing the specific cleavage of RNA. The mechanism of ribozyme action involves sequence specific hybridization of the ribozyme molecule to complementary target RNA, followed by endonucleolytic cleavage. Engineered hairpin or hammerhead motif ribozyme molecules that specifically and efficiently catalyze endonucleolytic cleavage of IGF-1R mRNA sequences are thereby useful within the scope of the present invention. Specific ribozyme cleavage sites within any potential RNA target are initially identified by scanning the target molecule for ribozyme cleavage sites, which typically include the following sequences, GUA, GUU, and GUC. Once identified, short RNA sequences of between about 15 and 20 ribonucleotides corresponding to the region of the target gene containing the cleavage site can be evaluated for predicted structural features, such as secondary structure, that can render the oligonucleotide sequence unsuitable. The suitability of candidate targets can also be evaluated by testing their accessibility to hybridization with complementary oligonucleotides, using, e.g., ribonuclease protection assays.","Both antisense oligonucleotides and ribozymes useful as IGF-1R kinase inhibitors can be prepared by known methods. These include techniques for chemical synthesis such as, e.g., by solid phase phosphoramadite chemical synthesis. Alternatively, anti-sense RNA molecules can be generated by in vitro or in vivo transcription of DNA sequences encoding the RNA molecule. Such DNA sequences can be incorporated into a wide variety of vectors that incorporate suitable RNA polymerase promoters such as the T7 or SP6 polymerase promoters. Various modifications to the oligonucleotides of the invention can be introduced as a means of increasing intracellular stability and half-life. Possible modifications include but are not limited to the addition of flanking sequences of ribonucleotides or deoxyribonucleotides to the 5\u2032 and\/or 3\u2032 ends of the molecule, or the use of phosphorothioate or 2\u2032-O-methyl rather than phosphodiesterase linkages within the oligonucleotide backbone.","In the context of the methods of treatment of this invention, IGF-1R kinase inhibitors are used as a composition comprised of a pharmaceutically acceptable carrier and a non-toxic therapeutically effective amount of an IGF-1R kinase inhibitor compound (including pharmaceutically acceptable salts thereof).","The term \u201cpharmaceutically acceptable salts\u201d refers to salts prepared from pharmaceutically acceptable non-toxic bases or acids. When a compound of the present invention is acidic, its corresponding salt can be conveniently prepared from pharmaceutically acceptable non-toxic bases, including inorganic bases and organic bases. Salts derived from such inorganic bases include aluminum, ammonium, calcium, copper (cupric and cuprous), ferric, ferrous, lithium, magnesium, manganese (manganic and manganous), potassium, sodium, zinc and the like salts. Particularly preferred are the ammonium, calcium, magnesium, potassium and sodium salts. Salts derived from pharmaceutically acceptable organic non-toxic bases include salts of primary, secondary, and tertiary amines, as well as cyclic amines and substituted amines such as naturally occurring and synthesized substituted amines. Other pharmaceutically acceptable organic non-toxic bases from which salts can be formed include ion exchange resins such as, for example, arginine, betaine, caffeine, choline, N\u2032,N\u2032-dibenzylethylenediamine, diethylamine, 2-diethylaminoethanol, 2-dimethylaminoethanol, ethanolamine, ethylenediamine, N-ethylmorpholine, N-ethylpiperidine, glucamine, glucosamine, histidine, hydrabamine, isopropylamine, lysine, methylglucamine, morpholine, piperazine, piperidine, polyamine resins, procaine, purines, theobromine, triethylamine, trimethylamine, tripropylamine, tromethamine and the like.","When a compound used in the present invention is basic, its corresponding salt can be conveniently prepared from pharmaceutically acceptable non-toxic acids, including inorganic and organic acids. Such acids include, for example, acetic, benzenesulfonic, benzoic, camphorsulfonic, citric, ethanesulfonic, fumaric, gluconic, glutamic, hydrobromic, hydrochloric, isethionic, lactic, maleic, malic, mandelic, methanesulfonic, mucic, nitric, pamoic, pantothenic, phosphoric, succinic, sulfuric, tartaric, p-toluenesulfonic acid and the like. Particularly preferred are citric, hydrobromic, hydrochloric, maleic, phosphoric, sulfuric and tartaric acids.","Pharmaceutical compositions used in the present invention comprising an IGF-1R kinase inhibitor compound (including pharmaceutically acceptable salts thereof) as active ingredient, can include a pharmaceutically acceptable carrier and optionally other therapeutic ingredients or adjuvants. Other therapeutic agents may include those cytotoxic, chemotherapeutic or anti-cancer agents, or agents which enhance the effects of such agents, as listed above. The compositions include compositions suitable for oral, rectal, topical, and parenteral (including subcutaneous, intramuscular, and intravenous) administration, although the most suitable route in any given case will depend on the particular host, and nature and severity of the conditions for which the active ingredient is being administered. The pharmaceutical compositions may be conveniently presented in unit dosage form and prepared by any of the methods well known in the art of pharmacy.","In practice, the IGF-1R kinase inhibitor compounds (including pharmaceutically acceptable salts thereof) of this invention can be combined as the active ingredient in intimate admixture with a pharmaceutical carrier according to conventional pharmaceutical compounding techniques. The carrier may take a wide variety of forms depending on the form of preparation desired for administration, e.g. oral or parenteral (including intravenous). Thus, the pharmaceutical compositions of the present invention can be presented as discrete units suitable for oral administration such as capsules, cachets or tablets each containing a predetermined amount of the active ingredient. Further, the compositions can be presented as a powder, as granules, as a solution, as a suspension in an aqueous liquid, as a non-aqueous liquid, as an oil-in-water emulsion, or as a water-in-oil liquid emulsion. In addition to the common dosage forms set out above, an IGF-1R kinase inhibitor compound (including pharmaceutically acceptable salts of each component thereof) may also be administered by controlled release means and\/or delivery devices. The combination compositions may be prepared by any of the methods of pharmacy. In general, such methods include a step of bringing into association the active ingredients with the carrier that constitutes one or more necessary ingredients. In general, the compositions are prepared by uniformly and intimately admixing the active ingredient with liquid carriers or finely divided solid carriers or both. The product can then be conveniently shaped into the desired presentation.","An IGF-1R kinase inhibitor compound (including pharmaceutically acceptable salts thereof) used in this invention, can also be included in pharmaceutical compositions in combination with one or more other therapeutically active compounds. Other therapeutically active compounds may include those cytotoxic, chemotherapeutic or anti-cancer agents, or agents which enhance the effects of such agents, as listed above.","Thus in one embodiment of this invention, the pharmaceutical composition can comprise an IGF-1R kinase inhibitor compound in combination with an anticancer agent, wherein said anti-cancer agent is a member selected from the group consisting of alkylating drugs, antimetabolites, microtubule inhibitors, podophyllotoxins, antibiotics, nitrosoureas, hormone therapies, kinase inhibitors, activators of tumor cell apoptosis, and antiangiogenic agents.","The pharmaceutical carrier employed can be, for example, a solid, liquid, or gas. Examples of solid carriers include lactose, terra alba, sucrose, talc, gelatin, agar, pectin, acacia, magnesium stearate, and stearic acid. Examples of liquid carriers are sugar syrup, peanut oil, olive oil, and water. Examples of gaseous carriers include carbon dioxide and nitrogen.","In preparing the compositions for oral dosage form, any convenient pharmaceutical media may be employed. For example, water, glycols, oils, alcohols, flavoring agents, preservatives, coloring agents, and the like may be used to form oral liquid preparations such as suspensions, elixirs and solutions; while carriers such as starches, sugars, microcrystalline cellulose, diluents, granulating agents, lubricants, binders, disintegrating agents, and the like may be used to form oral solid preparations such as powders, capsules and tablets. Because of their ease of administration, tablets and capsules are the preferred oral dosage units whereby solid pharmaceutical carriers are employed. Optionally, tablets may be coated by standard aqueous or nonaqueous techniques.","A tablet containing the composition used for this invention may be prepared by compression or molding, optionally with one or more accessory ingredients or adjuvants. Compressed tablets may be prepared by compressing, in a suitable machine, the active ingredient in a free-flowing form such as powder or granules, optionally mixed with a binder, lubricant, inert diluent, surface active or dispersing agent. Molded tablets may be made by molding in a suitable machine, a mixture of the powdered compound moistened with an inert liquid diluent. Each tablet preferably contains from about 0.05 mg to about 5 g of the active ingredient and each cachet or capsule preferably contains from about 0.05 mg to about 5 g of the active ingredient.","For example, a formulation intended for the oral administration to humans may contain from about 0.5 mg to about 5 g of active agent, compounded with an appropriate and convenient amount of carrier material that may vary from about 5 to about 95 percent of the total composition. Unit dosage forms will generally contain between from about 1 mg to about 2 g of the active ingredient, typically 25 mg, 50 mg, 100 mg, 200 mg, 300 mg, 400 mg, 500 mg, 600 mg, 800 mg, or 1000 mg.","Pharmaceutical compositions used in the present invention suitable for parenteral administration may be prepared as solutions or suspensions of the active compounds in water. A suitable surfactant can be included such as, for example, hydroxypropylcellulose. Dispersions can also be prepared in glycerol, liquid polyethylene glycols, and mixtures thereof in oils. Further, a preservative can be included to prevent the detrimental growth of microorganisms.","Pharmaceutical compositions used in the present invention suitable for injectable use include sterile aqueous solutions or dispersions. Furthermore, the compositions can be in the form of sterile powders for the extemporaneous preparation of such sterile injectable solutions or dispersions. In all cases, the final injectable form must be sterile and must be effectively fluid for easy syringability. The pharmaceutical compositions must be stable under the conditions of manufacture and storage; thus, preferably should be preserved against the contaminating action of microorganisms such as bacteria and fungi. The carrier can be a solvent or dispersion medium containing, for example, water, ethanol, polyol (e.g., glycerol, propylene glycol and liquid polyethylene glycol), vegetable oils, and suitable mixtures thereof.","Pharmaceutical compositions for the present invention can be in a form suitable for topical sue such as, for example, an aerosol, cream, ointment, lotion, dusting powder, or the like. Further, the compositions can be in a form suitable for use in transdermal devices. These formulations may be prepared, utilizing an IGF-1R kinase inhibitor compound (including pharmaceutically acceptable salts thereof), via conventional processing methods. As an example, a cream or ointment is prepared by admixing hydrophilic material and water, together with about 5 wt % to about 10 wt % of the compound, to produce a cream or ointment having a desired consistency.","Pharmaceutical compositions for this invention can be in a form suitable for rectal administration wherein the carrier is a solid. It is preferable that the mixture forms unit dose suppositories. Suitable carriers include cocoa butter and other materials commonly used in the art. The suppositories may be conveniently formed by first admixing the composition with the softened or melted carrier(s) followed by chilling and shaping in molds.","In addition to the aforementioned carrier ingredients, the pharmaceutical formulations described above may include, as appropriate, one or more additional carrier ingredients such as diluents, buffers, flavoring agents, binders, surface-active agents, thickeners, lubricants, preservatives (including anti-oxidants) and the like. Furthermore, other adjuvants can be included to render the formulation isotonic with the blood of the intended recipient. Compositions containing an IGF-1R kinase inhibitor compound (including pharmaceutically acceptable salts thereof) may also be prepared in powder or liquid concentrate form.","Dosage levels for the compounds used for practicing this invention will be approximately as described herein, or as described in the art for these compounds. It is understood, however, that the specific dose level for any particular patient will depend upon a variety of factors including the age, body weight, general health, sex, diet, time of administration, route of administration, rate of excretion, drug combination and the severity of the particular disease undergoing therapy.","Many alternative experimental methods known in the art may be successfully substituted for those specifically described herein in the practice of this invention, as for example described in many of the excellent manuals and textbooks available in the areas of technology relevant to this invention (e.g. Using Antibodies, A Laboratory Manual, edited by Harlow, E. and Lane, D., 1999, Cold Spring Harbor Laboratory Press, (e.g. ISBN 0-87969-544-7); Roe B. A. et. al. 1996, DNA Isolation and Sequencing (Essential Techniques Series), John Wiley & Sons (e.g. ISBN 0-471-97324-0); Methods in Enzymology: Chimeric Genes and Proteins\u201d, 2000, ed. J. Abelson, M. Simon, S. Emr, J. Thorner. Academic Press; Molecular Cloning: a Laboratory Manual, 2001, 3Edition, by Joseph Sambrook and Peter MacCallum, (the former Maniatis Cloning manual) (e.g. ISBN 0-87969-577-3); Current Protocols in Molecular Biology, Ed. Fred M. Ausubel, et. al. John Wiley & Sons (e.g. ISBN 0-471-50338-X); Current Protocols in Protein Science, Ed. John E. Coligan, John Wiley & Sons (e.g. ISBN 0-471-11184-8); and Methods in Enzymology: Guide to protein Purification, 1990, Vol. 182, Ed. Deutscher, M. P., Academic Press, Inc. (e.g. ISBN 0-12-213585-7)), or as described in the many university and commercial websites devoted to describing experimental methods in molecular biology.","This invention will be better understood from the Experimental Details that follow. However, one skilled in the art will readily appreciate that the specific methods and results discussed are merely illustrative of the invention as described more fully in the claims which follow thereafter, and are not to be considered in any way limited thereto.","Introduction","Colorectal cancer (CRC) represents a major health burden, and is the second-leading cause of cancer deaths in the U.S. In the past decade, the median survival among patients with metastatic CRC (mCRC) has increased, primarily due to the introduction of irinotecan, oxaliplatin and signal transduction modulators targeting the vascular endothelial growth factor (VEGF) and epidermal growth factor receptor (EGFR) pathways (Goldberg R. M., The Oncologist 2006; 11(9):981-7; Cunningham D., et al. The New England Journal of Medicine 2004; 351(4):337-45; Hurwitz H., et al. The New England Journal of Medicine, 2004; 350(23):2335-42; Van Cutsem E., et al. J Clin Oncol 2007; 25(13):1658-64). Increasingly, patients are receiving all of the above agents first- or second-line, so that for patients receiving subsequent salvage therapy, the median progression-free survival (PFS) is 8-10 weeks (Saltz L B, et al. J Clin Oncol 2007; 25(30): 4793-4799). For this population of CRC patients, options are limited and thus new agents are needed that can either induce tumor regression or disease stabilization. The IGF-1R signaling pathway appears to be a robust target in colorectal cancer (CRC), based upon data demonstrating overexpression of the receptor and ligands in CRC, association with a more malignant phenotype, chemotherapy resistance, and correlation with a poor prognosis (Saltz, L. B., et al. J Clin Oncol 2007; 25(30): 4793-4799; Tripkovic I., et al. Med Res. 2007 July; 38(5):519-25. Epub 2007 Apr. 26; Miyamoto S., et al. Clin Cancer Res. 2005 May 1; 11(9):3494-502; Nakamura M., et al. Clin Cancer Res. 2004 Dec. 15; 10(24):8434-41; Grothey A, et al. J Cancer Res Clin Oncol. 1999; 125(3-4):166-73).","Patient selection is also an emerging area in CRC, where the development of EGFR-targeted antibodies has been fraught with controversies surrounding the most appropriate markers for selecting patients and predicting efficacy. These agents were initially developed using EGFR immunohistochemistry (IHC) as a selection tool, whereas the results of large randomized studies indicate that we have only begun to understand the determinants of response to this class of drugs (Khambata-Ford S., et al. J Clin Oncol. 2007 Aug. 1; 25(22):3230-7; Li\u00e8vre A, et al. Cancer Res. 2006 Apr. 15; 66(8):3992-5; Italiano A., et al. Ann Surg Oncol. 2007 Nov. 7; [Epub ahead of print]; Chung K. Y., et al. J Clin Oncol. 2005 Mar. 20; 23(9):1803-10. Epub 2005 Jan. 27). The strategy taken herein to identify biomarkers that can be used to select patients for treatment with IGF-1R inhibitors is to identify potential predictive markers before, or in parallel with, early clinical development of IGF-1R inhibitors in CRC. Such an approach has also been utilized recently in the development of dasatanib and the proapoptotic ligand, Apo2L\/TRAIL (Coldren C. D., et al. Mol Cancer Res 2006; 4(8):521-8; Witta S. E., et al. Cancer Res. 2006; 66(2):944-50; Frederick B. A., et al. Molecular cancer therapeutics 2007; 6(6):1683-91; Huang F., et al. Cancer Res. 2007 Mar. 1; 67(5):2226-38; Wagner K. W., et al. Nat Med. 2007 September; 13(9):1070-7. Epub 2007 Sep. 2), and has resulted in valuable biological insights, including the relationship of epithelial-to-mesenchymal transition (EMT), the breast cancer \u201ctriple-negative\u201d phenotype, and death receptor O-glycosylation to responsiveness to EGFR or IGF-1R tyrosine kinase inhibitors, dasatinib, and Apo2L\/TRAIL, respectively (Witta S. E., et al. Cancer Res. 2006; 66(2):944-50; Frederick B. A., et al. Molecular cancer therapeutics 2007; 6(6):1683-91; Huang F., et al. Cancer Res. 2007 Mar. 1; 67(5):2226-38; Wagner K. W., et al. Nat Med. 2007 September; 13(9):1070-7. Epub 2007 Sep. 2).","Here we demonstrate that sensitivity of CRC tumor cells to IGF-1 receptor inhibition is predicted by various \u201csensitivity\u201d biomarkers. Conversely insensitivity to IGF-1 receptor inhibition is predicted by various \u201cresistance\u201d biomarkers. Since many newly developed targeted agents may be eventually incorporated clinically into traditional chemotherapy regimens, we also examined the effects of an IGF-1R inhibitor in combination with standard chemotherapy agents (5-flurouracil, SN38 and oxaliplatin) in colon cancer cell lines. It was found that the \u201csensitivity\u201d or \u201cresistance\u201d biomarkers not only predict which tumor cells will be sensitive to the IGF-1R inhibitor, but also predict which will respond in a synergistic manner when treated with a combination of an IGF-1R inhibitor and a chemotherapeutic agent.","PQIP Studies","Materials and Methods","IGF-1R Inhibitor Compound: IGF-1R inhibitor compound PQIP was provided by OSI Pharmaceuticals, (Melville, N.Y.). PQIP (cis-3-[3-(4-Methyl-piperazin-1-yl)-cyclobutyl]1-(2-phenyl-quinolin-7-yl)-imidazo[1,5-a]pyrazin-8-ylamine) is a 1,3-disubstituted-8-amino-imidazopyrazine derivative synthesized by the methods described in patent application number WO 2005\/097800 A1. Compound identity and purity (>99%) were verified by H and C nuclear magnetic resonance, mass spectrometry (MS), and high-performance liquid chromatography using Bruker Advance 400, WatersMicromass ZQ, and Waters LC Module I Plus instruments, respectively, as well as by elemental analysis. PQIP was dissolved in DMSO as a 10 mmol\/L stock solution for use in biochemical or cellular assays in vitro.","Cell Lines and Culture. Twenty-eight human colon cancer cell lines, were obtained from American Type Culture Collection (Manassas, Va.). Cells were grown in RPMI medium supplemented with 10% fetal bovine serum, 1% non-essential amino acids, 1% penicillin\/streptomycin and were maintained at 37\u00b0 C. in an incubator under an atmosphere containing 5% CO. The cells were routinely screened for the presence of mycoplasma (MycoAlert, Cambrex Bio Science, Baltimore, Md.) and were exposed to PQIP when they reached approximately 70% confluence.","Microarray Analysis: Cells were plated at 2\u00d7106 in 6-well plates 24 h prior to harvest. After 24 hours cells were rinsed twice with PBS, and RNA was prepared using a RNeasy Plus mini kit (Qiagen, Valencia, Calif.). RNA stabilization, isolation, and microarray sample labeling were carried out using standard methods for reverse transcription and one round of in vitro transcription. HG-U133 set microarrays were hybridized with 10 \u03bcg of cRNA and processed according to the protocols of the manufacturer (Affymetrix). Hybridization signals and detection calls were generated in BioConductor, using the germa and affy software packages. Microarray data was analyzed using BRB ArrayTools v3.2 developed by Dr. Richard Simon and Amy Peng Lam. Multidimensional scaling, using centered correlation was performed and 110 genes were found to be significant at p<0.005.","RT-PCR analyses: Expression levels of genes of interest from array analyses were assessed using RT-PCR. Total RNA was isolated from cells using the RNeasy mini kit (Qiagen, Valencia, Calif.), cDNA synthesized from one mg of total RNA using the Taqman reverse transcription kit (Applied Biosystems, Foster City, Calif.), and expression levels detected from 100 ng of cDNA using Power SYBR Green detection chemistry (Applied Biosystems, Foster City Calif.), all according to the manufacturer's directions. Samples were run on an Applied Biosystems Step One Plus (Applied Biosystems, Foster City Calif.).","shRNA Knockdown: The pRS-shE2F6 gene-specific shRNA expression cassettes, along with control shRNA plasmids including the original pRS vector (TR20003, were purchased from OriGene (Rockville Md.). The sequence of the metallothionein 2A-specific 29mer shRNA is GTAAAGAACGCGACTTCCACAAACCTGGA SEQ ID NO:114. Stable clones were generated by transfecting HCT116 cells in 6-well dishes with 1 \u03bcg of each of the shRNA plasmids using Fugene 6 (Roche, Basel Switzerland), according to manufacturer's recommendations. Seventy-two hours after transfection, the cells were placed under selection with 2.0 \u03bcg\/mL of puromycin, splitting 1:5 when the cells reached confluency. Multiple clones from the same transfection were pooled and grown under puromycin selection. Successful knockdown of specific genes and gene products was confirmed by semi-quantitative RT-PCR and immunoblotting with specific antibodies.","Preparation of cytogenetic slides with metaphase and interphase cells: Cells in culture for nine colorectal cancer (CRC) cell lines were subjected to mitotic arrest and were harvested after hypotonization in 0.075 M KCl for 20 min at 37\u00b0 C. followed by fixation using methanol\/acetic acid (3:1). Following three changes of fixative, four slides were dropped for each cell line while checking for optimal cell density, chromosome spreading, encapsulation\/residual cytoplasm, and chromosome morphology.","Preparation of BAC clone as FISH probe: A FISH probe encompassing the IGF-1R gene was derived from the clone RP11-262P8. The IFG1R DNA was labeled with SpectrumRed conjugated dUTPS using the Vysis Nick translation kit (Abbott Molecular), according to manufacturer's instructions. The labeled DNA was ethanol precipitated using herring sperm DNA (1:50 v\/v) as carrier and human Cot-1 DNA (1:10 v\/v) for blocking repetitive sequences. The DNA pellet was diluted in 10 \u03bcl of hybridization mix (50% formamide\/10% dextran sulphate\/2\u00d7SSC).","Fluorescent in situ Hybridization Assays: Dual-color FISH assays were performed on the prepared slides of colorectal cancer cell lines using 120 ng of SpectrumRed-labeled IGF1R and 0.3 \u03bcl of SpectrumGreen-labeled CEP15 (Abbott Molecular) per 113 mmhybridization area according to standard protocol in the laboratory. The slides were first washed in 70% acetic acid for 20-30 sec, then incubated in 0.008% pepsin\/0.01 M HCl at 37\u00b0 C. for 3-5 min, in 1% formaldehyde for 10 min and dehydrated in a graded ethanol series. The probe mix was applied to the selected hybridization areas, which were covered with glass cover slips and sealed with rubber cement. DNA co-denaturation was performed for 9 min at 85\u00b0 C. and hybridization was allowed to occur at 37\u00b0 C. for 40-48 hours. Post-hybridization washes were performed with 2\u00d7SSC\/0.3% NP-40 at 72\u00b0 C. and 2\u00d7SSC for 2 min at room temperature and dehydrated in a graded ethanol series. Chromatin was counterstained with DAPI (0.3 \u03bcg\/ml in Vectashield Mounting Medium, Vector Laboratories). Analysis was performed on epifluorescence microscope using single interference filter sets for green (FITC), red (Texas red), and blue (DAPI) as well as dual (red\/green) and triple (blue, red, green) band pass filters. Approximately 20 metaphase spreads and 100 interphase nuclei were analyzed in each cell line and in this setting we roughly estimated the ploidy and identified the chromosomes harboring homologous sequences to the IGF1R\/CEP15 probe set. For documentation, images were captured using a CCD camera and merged using dedicated software (CytoVision, AI).","Immunoprecipitation and Immunoblotting: HT29 and HCT116 CRC cell lines were seeded into 6-well plates, allowed to attach for 24 hours and serum starved overnight. Cells were then exposed to IGF-2 (100 ng\/mL) for 10 minutes with or without a 3 hour PQIP (0.4 \u03bcmol\/L) pretreatment. After treatment cells were rinsed with PBS and scraped into RIPA lysis buffer containing protease inhibitors, EDTA, NaF, and sodium orthovanadate. Total protein was quantified using the BioRad Dc Protein Assay (BioRad, Hercules, Calif.). Total protein (30 \u03bcg) was electrophoresed on a 4-20% gradient SDS-polyacrylimide gel then electrophoretically transferred to Immobilon-P (Millipore, Bedford, Mass.). Membranes were blocked for 1 hour in 5% non-fat dry milk (BioRad, Hercules, Calif.) in TBS-Tween (0.1%) prior to overnight incubation at 4\u00b0 C. with the appropriate primary antibody. Blots were then washed 3\u00d720 minutes in TBS-Tween (0.1%) and were incubated with the appropriate secondary anti-rabbit or anti-mouse IgG1 horseradish peroxidase-linked antibody at 1:20,000 (Jackson ImmunoResearch, West Grove, Pa.) for one hour at room temperature. After three additional washes, the blots were developed by Immobilon Western Chemiluminescent HRP substrate (Millipore, Billerica, Mass.). Anti-IGF1R\u03b2 rabbit polyclonal (Cell Signaling Technology, Danvers, Mass.) was used for immunoprecipitation at a 1:50 dilution. The following primary antibodies were used for immunoblotting (all from Cell Signaling Technology, Danvers, Mass.). Anti-phosphorylated Akt (T308) rabbit polyclonal, anti-phosphorylated S6 ribosomal protein (S235\/236) rabbit mAb, anti-phosphorylated p44\/42 MAPK (Thr202\/Tyr204) rabbit mAb, and anti-phosphorylated tyrosine mouse mAb at 1:1.000. Anti-Akt (pan) rabbit mAb, anti-S6 ribosomal protein rabbit mAb, anti-p44\/42 MAP Kinase rabbit mAb, anti Pan-Actin rabbit polyclonal, anti-IGF1R\u03b2 rabbit polyclonal, anti-PARP rabbit polyclonal, and anti-cyclin D1 rabbit polyclonal at 1:5000.","Cytotoxicity and Combination Effects: HT29, HCT116, RKO, and LS513 CRC cell lines were exposed to 0-10 nM SN-38 (7-Ethyl-10-hydroxy-camptothecin), 0-10 \u03bcM Oxaliplatin, or 0-50 \u03bcM 5-Fluorouracil (5-FU) for 72 hours. HT29, HCT116, RKO, and LS513 CRC cell lines were exposed to PQIP in combination with SN-38, Oxaliplatin or 5-FU. In all combinations, cells were seeded in 96-well flat bottomed plates and left overnight. PQIP and chemotherapy were both added for 72 hours. The results of the combinations were analyzed by the Chou and Talalay method with Calcusyn (Biosoft, Ferguson, Mo.).","Detection of IGF1R using Meso Scale Discovery assays (MSD): HT29 and HCT116 CRC cell lines were seeded into 6-well plates, allowed to attach for 24 hours and exposed to PQIP and chemotherapy (SN38, or Oxaliplatin) alone or in combination for 72 h. Following exposure, cells were rinsed with PBS and scraped into RIPA lysis buffer containing protease inhibitors, EDTA, NaF, and sodium orthovanadate. Total protein was quantified using the BioRad Dc Protein Assay (BioRad, Hercules, Calif.). Fifty micrograms of protein in 25 \u03bcL was added per well for MSD detection of phosphorylated IGF-1R according to manufacturer's instructions (Meso Scale Discovery, Gaithersburg, Md.).","Results","CRC cell lines exhibit a range of sensitivities to the IGF-1R kinase inhibitor PQIP: The sensitivity of 26 colorectal cancer (CRC) cell lines to growth inhibition by the IGF-1R kinase inhibitor PQIP was tested using a sulforhodamine B (SRB) assay (see  for characteristics of the cell lines tested.). The CRC cell lines were treated with a range of concentrations of PQIP (from 0.05-5 \u03bcM) for 72 hours. As shown in  the majority of the cell lines failed to reach an IC50 up to 5 \u03bcmol\/L, but a clear distinction could be made between the cell lines that were sensitive (i.e. cell lines Colo205, HT29, Colo320, and LS513, with IC50<0.5 \u03bcmol\/L) and the cell lines that were resistant (HCT116, HCT15, SW480, RKO, LS1034, CaCo2, HCT8, LoVo, LS123, T84, LS174T, LS180, SW1417, SW1116, SW48, NCI-H508, SW948, SW837, SW1463, SW403, with IC50>5 umol\/L). Several cell lines were of intermediate sensitivity (e.g. SW620, Colo201, SK-CO-1).","Differential Gene Expression Between PQIP-Sensitive and PQIP-Resistant CRC Cell Lines: To determine the genes associated with sensitivity or resistance to PQIP treatment, the gene expression profiles of PQIP-untreated samples of the 4 most sensitive (S) and the 5 most resistant (R) CRC cell lines were examined. This gene profile was generated by analyzing the differential expression of PQIP-resistant and PQIP-sensitive CRC cell lines. RNA was prepared from the nine cell lines using Affymetrix Human Gene 1.0 ST arrays. Using previously published methods, we were able to identify 110 genes that were significant at the p<0.005 level of univariate significance, using a two-sample T-test (). Top scoring genes (i.e. p<0.0015) included caldesmon (CALD1), several metallothioneins (e.g. MT-1E), aldehyde dehydrogenase (ALDH1A1), and mitogen activated protein kinase kinase 6 (MAP2K6).","Caldesmon was expressed at high levels in R tumor cells (61-fold increased in R cells, p=0.0002; ). Caldesmon is an actin-binding protein that has recently been shown to play a critical role in regulating the formation and dynamics of podosomes and invadopodia, cell adhesion structures that protrude from the plasma membrane and degrade the extracellular matrix (ECM), thus promoting cancer cell invasion (Linder, S. and Aepfelbacher, M. Trends Cell Biol 2003 13: 376-385; Yoshio, T., et al. FEBS lett. 2007 581: 3777-3782; Koji-Owada, M., et al. Proc Natl Acad Sci USA 1984 81: 3133-3137). Interestingly, caldesmon is a negative regulator of the formation podosomes and invadopodia by sequestering actin, thus indicating that CRC cell lines with an invasive\/malignant phenotype are more responsive to IGF-1R inhibition.","Also overrepresented in R compared to S cell lines are the metallothioneins (e.g. MT-1E), a family of ubiquitous, low molecular weight intracellular proteins that bind and detoxify heavy metal ions (Kagi, J. H. R. Meth Enzymol 1991 205: 613-626; Bauman, J. W. et al. Toxicol Appl Pharmacol 1991 110: 347-354) (; MT-1E RT-PCR,  (RT-PCR)). This family are among the highest ranked (15-36-fold; p=9.2\u00d710\u22120.0001) group of differentially expressed genes (; MT-1E RT-PCR, ). Metallothioneins can be induced by a variety of stimuli, are involved in other cellular functions such development, differentiation proliferation, and carcinogenesis, and have been associated with a poor prognosis and metastasis in cancer (Bauman, J. W. et al. Toxicol Appl Pharmacol 1991 110: 347-354; Bruewer, M, et al. World J Surg 2002 26: 726-731; Sens, M A, et al. Am J Pathol 2001 159: 21-26; Cherian, M. G., et al. Mut Res 2003 533: 201-209; Jin, R., et al. Br J Cancer 2000 83: 319-323; Shmid, K. W., et al. Arch A Pathol Anat Histopathol 1993 422: 153-159). They may also play a functional role in cancer drug resistance, though the mechanism for this remains poorly understood (Surowiak, P, et al. Histol Histopathol 2005 20: 1037-1044; Saga, Y, et al. Int J Urol 2004 11: 407-415).","An example of a gene highly expressed in S tumor cell lines, is the aldehyde dehydrogenase gene, ALDH1A1 (83-fold; p=0.002; , C (RT-PCR)), an enzyme involved in the metabolism of alcohol and interestingly, the oxidation of all-trans retinal to all-trans retinoic acid (Molotkov A, and Duester G. J Biol Chem. 2003 Sep. 19; 278(38):36085-90. Epub 2003 Jul. 7; Luo P, Wang A, et al. Stem Cells. 2007 October; 25(10):2628-37. Epub 2007 Jul. 12; Rice K. L., et al. Leuk Res. 2007 Dec. 12; [Epub ahead of print]). ALDH1A1 has also been associated with drug resistance. When ALDH1A1 is knocked down in lung cancer cell lines they become sensitive to 4-hydroperoxycyclophosphamide (4-HC) (Moreb, J. S., et al. Cancer Chemother Pharmacol. 2007 January; 59(1):127-36. Epub 2006 Apr. 14). This is the opposite of what is seen here, in that ALDH1A1 is up regulated in the PQIP sensitive lines.","Another example of a gene highly expressed in S tumor cell lines, is mitogen-activated protein kinase kinase 6 (MAP2K6). MAP2K6 is part of the p38 MAPK pathway and acts to directly activate p38. MAP2K6 has been shown to be activated following treatment with cisplatin in numerous cancer cell lines. This activation then activates p38 MAPK, which in turn leads to increase in cell killing following cisplatin treatment. If you block p38 activation the cancer cells become resistant. This shows that MAP2K6 can play a role in sensitivity to anti-cancer agents. (Losa J. H., et al. Oncogene 2003 Jun. 26; 22(26):3998-4006).","shRNA Knockdown of Selected Genes: To characterize the potential significance of these genes, we selectively reduced their expression in S and R cell lines using stably transfected shRNAs and examined their effects on the S or R phenotype. To characterize the potential functionality that the selected genes may play in the sensitivity or resistance to PQIP, resistant and sensitive CRC cell lines were stably transfected with specific shRNA, treated with increasing concentrations of PQIP and proliferation assays performed to observe, if when manipulated, these genes alter responsiveness (i.e. the S or R phenotype). Two resistant cell lines HCT116 and SW480 were transfected with metallothionein 2A (MT-2A) shRNA and were exposed to increasing concentrations of PQIP. RT-PCR indicated that a greater than 80% reduction in gene expression was achieved in two separate clones and a nearly complete reduction in protein expression. Both of these transfected cell lines were assayed for proliferation as described above. As shown in  HCT116 cells transfected with two different shRNA constructs to reduce MT2A expression demonstrated a left shift in sensitivity when compared to the parental cell line or the non-targeted scramble shRNA control. However the transfected SW480 cell lines showed a very modest if any left shift in sensitivity when compared to parental or the non-targeted scramble shRNA control (). HCT116 and SW480 cell lines transfected with caldesmon shRNA, to reduce the expression of calsesmon, the other gene highly upregulated in the PQIP resistant CRC cell lines, showed no left shift in sensitivity when compared to parental or the non-targeted scramble shRNA control (data not shown). HT29 tumor cells were transfected with shRNA for ALDH1A1, which is up regulated in the PQIP sensitive cell lines. When ALDH1A1 is knocked down in HT29 and exposed to PQIP at increasing concentrations, no left shift in sensitivity is observed (data not shown).","It appears that MT2A and not caldesmon increases the sensitivity to PQIP in the PQIP-resistant CRC cell line HCT116. Thus MT2A, in addition to being be a predictive marker for PQIP resistance, when down-modulated can increase sensitivity to IGF-1R kinase inhibitors in resistant cell lines, indicative of a causative relationship with resistance. This may lead to the ability to target MTs in patients with resistant tumors to increase their sensitivity to PQIP. So far no genes associated with sensitivity, when modulated show any right shift toward resistance.","Gene Expression Analysis Post PQIP Exposure: In order to identify genes that are modulated following exposure to PQIP, 5 sensitive and 5 resistant CRC cell lines were treated with 400 nM PQIP for 72 hours, total RNA was purified, and gene array analysis was performed as described previously. The gene expression profiles following PQIP exposure were compared to the respective untreated profiles. To characterize changes in gene expression profile, one PQIP resistant (HCT116) and one PQIP sensitive (HT29) tumor cell type was examined, and caldesmon, metallothioneins, ALDH1A1, and MAP2K6 expression levels were analyzed.",{"@attributes":{"id":"p-0354","num":"0354"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Fold change of post exposure PQIP to untreated."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Gene","HCT116","HT29"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"MT1E","1.2","12.5"]},{"entry":[{},"MT1X","2.0","16.0"]},{"entry":[{},"MT1H","1.8","11.0"]},{"entry":[{},"MT1F","1.6","10.8"]},{"entry":[{},"MT2A","1.0","8.1"]},{"entry":[{},"MT1M","1.5","15.2"]},{"entry":[{},"MT1G","1.1","4.5"]},{"entry":[{},"CALD1","1.9","1.5"]},{"entry":[{},"ALDH1A1","2.3","0.56"]},{"entry":[{},"MAP2K6","0.7","0.87"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},"As seen in Table 1, in both HCT116 and HT29 overall the expression of MTs is higher post PQIP exposure, with a much higher expression in the HT29 CRC cell line. These increases may be due to that fact that the cells that remain following PQIP exposure are more resistant to PQIP and therefore have higher expression of MTs. Since some of the MTs are also modestly upregulated following PQIP treatment in the resistant cells, this may show that PQIP may actually be able to modulate MT expression and MTs could be used as a surrogate biomarker in patients following PQIP treatment","Caldesmon was also slightly higher in both CRC cell lines. ALDH1A1, which had higher expression in the untreated PQIP sensitive CRC cell line HT29, decreased expression post PQIP exposure However in the HCT116 PQIP resistant CRC cell line the expression of ALDH1A1 increased following PQIP exposure. Expression of MAP2K6 did not change much following PQIP exposure.","Fluorescent in situ hybridization of IGF1R in Colon Cancer Cell Lines: Recent studies have shown EGFR gene copy number detection by FISH can predict outcome of patients treated with cetuximab and chemotherapy (Hirsch F R, et. al. J Clin Oncol. 2008 Jul. 10; 26(20):3351-7). Gene copy number of IGF-1R in CRC cell lines was therefore examined to see if sensitivity to PQIP can be predicted in a similar fashion. Dual-color FISH assays were performed on the prepared slides of the colorectal cancer cell lines. Metaphase spread and interphase nuclei from the cell lines RK0 and Colo205 hybridized with the IGF1R (red)\/CEP15 (green) probe set are exemplified in . Detailed results of interphase and metaphase analyses are presented in , , and , for 25 CRC cell lines. The ploidy for each cell line was roughly estimated based on chromosome count in the metaphases analyzed. None of these CRC cell lines showed IGF1R gene amplification or significant loss. However, interestingly, all four lines that are sensitive to PQIP showed slightly increased IGF-1R copy numbers (unbalanced gains when normalized to ploidy), i.e. COLO205, HT29, CaCo2, and LS513. By contrast, only 2 of 16 R cell lines exhibited unbalanced gains. Based on the results it does not appear that the any of the CRC cell lines have increased gene amplification or significant loss. However, IGF-1R gene copy number (unbalanced gains when normalized to ploidy) does appear to be a statistically significant predictor of sensitivity to PQIP.","Characterizing IGF1R Pathway of Colon Cancer Cell Lines by Immunohistochemistry: Previous studies have shown that combination of gene copy number and protein expression by IHC can predict the outcome of NSCLC patients to gefitinib treatment (44). There did not however appear to be any consistent correlation between EGFR expression and sensitivity to EGFR targeted agents (e.g. gefitinib, erlotinib). The IGF-1R pathway was thus evaluated by IHC to determine if over expression of any part of this pathway can be predictive of sensitivity. IGF1R and IGF2R protein expression was evaluated by IHC using established methods. Table 2 shows the scores (0-4) of IGF1R and IGF2R protein expression on a panel of CRC cell lines. There does not appear to be a correlation between IGF1R or IGF2R expression and sensitivity to PQIP, similar to results reported previously with the EGFR pathway.",{"@attributes":{"id":"p-0359","num":"0359"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"IHC scores for IGF1R and IGF2R on CRC cell lines."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Cell Line","IGF2R","IGF1R"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"HCT15","3+","2+"]},{"entry":[{},"HCT116","2+","2+"]},{"entry":[{},"LS180","1+","2+"]},{"entry":[{},"SW620","2+","2+"]},{"entry":[{},"HCT8","2+","2+"]},{"entry":[{},"RKO","2+","2+"]},{"entry":[{},"SW480","2+"]},{"entry":[{},"HT29","2+","2+"]},{"entry":[{},"COLO201","2+","3+"]},{"entry":[{},"COLO205","2+","3+"]},{"entry":[{},"LS513","2+","2+"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},"Effect of PQIP on receptor phosphorylation and pathway activation: Since PQIP is expected to inhibit autophosphorylation of IGF1R, we tested its ability to inhibit IGF-2 mediated activation of the receptor and pathway. One PQIP sensitive and one PQIP resistant CRC cell line were chosen for immunoprecipitation and immunoblotting analysis. Following 18 h serum starvation, HT29 and HCT116 CRC cell lines were stimulated or unstimulated for 10 minutes with 100 ng\/mL of IGF-2 with or without a 3 h pre-exposure to PQIP (0.4 \u03bcmol\/L). In both sensitive HT29 and resistant HCT116 CRC cell lines, PQIP almost fully inhibits IGF-2 induced tyrosine autophosphorylation of IGF1R (). In these CRC cell lines PQIP similarly inhibits phosphorylation of Akt, S6 ribosomal protein with or without IGF-2 stimulation. However, IGF-2 induced phosphorylation of p42\/44 MAPK is only inhibited in the sensitive HT29 and not in resistant HCT116 CRC cell line ().","As expected, in both HT29 and HCT116 cell lines IGF1R, Akt, and S6 ribosomal protein are all successfully inhibited with addition of PQIP. Interestingly, PQIP inhibits ERK in the sensitive HT29 cells only, suggesting that resistance in HCT116 may be explained by dependence on growth factor receptors other than IGF1R. Similar results were previously reported in GEO CRC cells, showing inhibition of Akt, but not ERK (Ji Q. S., et. al. Mol Cancer Ther. 2007 August; 6(8):2158-67. Epub 2007 August 1).","Combination treatment of PQIP with Chemotherapy in Colon Cancer Cell Lines: IGF1R is overexpressed in CRC and has been associated with resistance to chemotherapy. Therefore, we investigated the effect of PQIP in combination with standard chemotherapy for CRC. Four CRC cell lines including, two PQIP sensitive and two PQIP resistant, were chosen for combinations. HT29 and LS513 CRC cell lines were classified as sensitive, with IC's around 0.3 \u03bcmol\/L. HCT116 and RKO CRC cell lines were classified as resistant, with IC>5 \u03bcmol\/L. Concentrations were chosen based on single agent proliferation curves for each compound. Cells were exposed to PQIP and chemotherapy (SN38, oxaliplatin, or 5-Fluorouracil) concurrently for 72 h and combinations were assessed using the SRB assay. Sensitive HT29 and LS513 CRC cell line demonstrate synergy, with nearly all combination index (CI) values between 0.2-1.0 (). The resistant HCT116 and RKO CRC cell lines showed mostly additivity and some modest synergy (CI values 0.6-1.4) in nearly all combinations tested, displaying reduced sensitivity to the combination (). It thus appears that biomarkers which predict tumor cell sensitivity will also indicate which cells will respond in a synergistic manner to an IGF-1R kinase inhibitor (e.g. PQIP) and chemotherapy (i.e. SN38, oxaliplatin, or 5-Fluorouracil).","Additional combinatorial experiments involving analysis of IGF1R protein levels were carried out using PQIP sensitive HT29 and PQIP resistant HCT116 CRC cell lines. Cells were exposed to PQIP (0.4 \u03bcmol\/L) and chemotherapy (SN38 0.004 \u03bcmol\/L, or Oxaliplatin 1.0 \u03bcmol\/L) alone or in combination for 72 h. Protein was extracted and analyzed for phospho\/total IGF1R levels using a Meso Scale Discovery (MSD; Gaithersburg, Md. 20877) assay. As shown in , phosphorylated IGF1R was inhibited upon exposure to PQIP alone and in both combinations. SN38 or Oxaliplatin alone exhibited no significant effect on IGF1R phosphorylation ().","It has been shown that chemotherapy, specifically oxaliplatin, in HT29 and HCT116 induces an increase in expression levels of both total and phosphorylated Akt and ERK. Despite this, we were able to achieve a synergistic effect on proliferation indicating that inhibition of these prosurvival pathways is not an absolute requirement for the action of these compounds. Previous studies have shown that other compounds can increase phosphorylated ERK levels and this may be reflective of sustained versus transient ERK activation which has been associated with proapoptotic cell death (46).","Due to the synergistic effect with these compounds, further investigation was done on the chemotherapy combinations using MSD assays. Interestingly, the basal levels of phosphorylated IGF1R are higher in the sensitive HT29 cells than in the resistant HCT116 cells. In addition, all combinations display a similar decrease in phosphorylated IGF1R with PQIP alone. This suggests again that the synergy seen is independent of phosphorylated IGF1R and may be by other mechanisms, such as apoptosis.","OSIP-906 Studies","Materials and Methods","IGF-1R Inhibitor Compound: IGF-1R inhibitor compound OSI-906 was provided by OSI Pharmaceuticals, (Melville, N.Y.). OSIP-906 (cis-3-[8-amino-1-(2-phenyl-quinolin-7-yl)-imidazo[1,5-a]pyrazin-3-yl]-1-methyl-cyclobutanol) is synthesized by the methods described in patent application number WO 2005\/097800.","Cell Lines and Culture: Twenty-seven human colon cancer cell lines, were obtained from American Type Culture Collection (Manassas, Va.). The GEO cells were a generous gift from Dr. Fortunato Ciardiello (Cattedra di Oncologia Medica, Dipartimento Medico-Chirurgico di Internistica Clinica e Sperimentale \u201cF Magrassi e A Lanzara,\u201d Seconda Universit\u00e0 degli Studi di Napoli, Naples). All cells except GEO were grown in RPMI medium supplemented with 10% fetal bovine serum, 1% non-essential amino acids, 1% penicillin\/streptomycin and were maintained at 37\u00b0 C. in an incubator under an atmosphere containing 5% CO. GEO cells were grown in DMEM\/F12 supplemented with 10% fetal bovine serum, 1% non-essential amino acids, 1% penicillin\/streptomycin. The cells were routinely screened for the presence of mycoplasma (MycoAlert, Cambrex Bio Science, Baltimore, Md.) and were exposed to OSI-906 when they reached approximately 70% confluence. OSI-906 was provided by OSI Pharmaceuticals, (Boulder, Colo.) and prepared as a 10 mM stock solution in DMSO.","Fluorescent in situ Hybridization (FISH): Dual-color FISH assays were performed on the prepared slides of the CRC cell lines using 120 ng of Spectrum Red-labeled IGF-1R (University of Colorado Cancer Center Cytogenetics Lab) and 0.3 ul of Spectrum Green-labeled CEP15 (Abbott Molecular, Abbott Park, Ill.) per 113 mmhybridization area according to standard protocol in the laboratory (1). The slides were first washed in 70% acetic acid for 20-30 sec, then incubated in 0.008% pepsin\/0.01 M HCl at 37\u00b0 C. for 3-5 min, in 1% formaldehyde for 10 min and dehydrated in a graded ethanol series. The probe mix was applied to the selected hybridization areas, which were covered with glass cover slips and sealed with rubber cement. DNA co-denaturation was performed for 9 min at 85\u00b0 C. and hybridization was allowed to occur at 37\u00b0 C. for 40-48 hours. Post-hybridization washes were performed with 2\u00d7SSC\/0.3% NP-40 at 72\u00b0 C. and 2\u00d7SSC for 2 min at room temperature (RT) and dehydrated in a graded ethanol series (Cappuzzo F, et al. Br J Cancer 2008; 99(1):83-9). Chromatin was counterstained with DAPI (0.3 \u03bcg\/ml in Vectashield Mounting Medium, Vector Laboratories, Burlingame, Calif.). Analysis was performed on an epifluorescence microscope using single interference filter sets for green (FITC), red (Texas Red), and blue (DAPI) as well as dual (red\/green) and triple (blue, red, green) band pass filters. Approximately 20 metaphase spreads and 100 interphase nuclei were analyzed in each cell line, ploidy was assessed along with identification of the chromosomes harboring homologous sequences to the IGF-1R\/CEP15 probe set. For documentation, images were captured using a CCD camera and merged using dedicated software (CytoVision, AI, San Jose, Calif.).","Immunoblotting: Cells were seeded into 6-well plates 24 hr prior to treatment and new media was added with or without drug for an additional 72 hr. After treatment, cells were scraped into RIPA buffer containing protease inhibitors, EDTA, NaF, and sodium orthovanadate. The total protein in samples was determined using the BioRad DProtein Assay (BioRad, Hercules, Calif.). Thirty micrograms of total protein was loaded onto a 10% gradient gel, electrophoresed, and then transferred to PVDF using the I-Blot (Invitrogen, Carlsbad, Calif.). The membranes were blocked for one hour at RT with 5% nonfat dry milk in TBS containing tween-20 (0.1%) prior to overnight incubation at 4\u00b0 C. with the following primary antibodies: pIGF-1R, IGF-1R, pSHC, SHC, pIRS-1, IRS-1, pAKT, AKT, pERK, ERK, pS6RP, S6RP, pPI3K, and PI3K. (all from Cell Signaling, Beverly, Mass.). After the primary antibody, blots were washed 3\u00d720 minutes in TBS-Tween (0.1%), incubated with the appropriate secondary anti-rabbit or anti-mouse IgG horseradish peroxidase-linked antibody at 1:20,000 (Jackson ImmunoResearch, West Grove, Pa.) for one hour at RT, washed three times and developed using the Immobilon Western Chemiluminescent HRP substrate (Millipore, Billerica, Mass.). Immunoblot experiments were performed in triplicate for each antibody.","Immunohistochemistry (IHC): IGF-1R and downstream effector protein expression was assessed by IHC using the following antibodies. IGF-1R (Ventana medical systems, Tucson, Ariz.), IGF-2R (Santa Cruz, Santa Cruz, Calif.), phospho-ERK (Cell Signaling, Beverly, Mass.), IGF2, (ABCAM, Cambridge, Mass.). The staining procedures were performed according to the antibody manufacturer's recommendations. For scoring of proteins, a staining index calculated as percent of stained tumor cells\u00d7average staining intensity graded from 0 to 4 was used, resulting in an index value between 0 and 400. Consistent with previous reports, samples with a staining index of 200 or higher were predefined as protein-positive (Jimeno A., et al. Cancer Res 2008; 68(8):284:1-9) The scoring was performed by pathologists who were blinded to the exposure data.","KRAS\/BRAF\/PI3K mutation analyses. For both human primary tumor explants and CRC cell lines DNA was isolated using the Qiagen DNA extraction kit. KRAS mutations were analyzed by one of two methods. The human primary tumor explants were assayed by the University of Colorado Cancer Center Pathology Core with the DxS Scorpion method (DxS, Manchester, UK) using the manufacturer's instructions. Briefly, template DNA was analyzed for a set of seven known KRAS point mutations in codons 12 and 13 (i.e. Gly12Asp (GGT>GAT), Gly12Ala (GGT>GCT), Gly12Val (GGT>GTT), Gly12Ser (GGT>AGT), Gly12Arg (GGT>CGT), Gly12Cys (GGT>TGT), and Gly13Asp (GGC>GAC)) using the THERASCREEN\u00ae KRAS Mutation Detection kit (DxS Ltd., Manchester, UK). Reactions and analysis were performed on a Lightcycler 480 real-time PCR instrument (LC480) that was calibrated using a dye calibration kit provided by the kit manufacturer. Reactions were performed on a 96-well plate in 20 \u03bcl reactions using approximately 60 ng of each DNA template. Sample DNA was amplified with eight separate primer sets (one for the wild-type sequence and one for each of seven different point mutations) with an internal Scorpion reporter probe. Cycle cross point (CP) values were calculated using the LC480 Fit-point software suite, and the control Cp was subtracted from the Cp of each mutation specific primer set (). Because there may be spurious low level amplification in the absence of mutant template, amplification products are often visible at later cycle numbers for most of the primer sets. To avoid false-positive results due to, background amplification, the assay is considered valid only if the control Cp value is less than or equal to 35 cycles. Cp thresholds are calculated to compensate for this background amplification. Mutations are called when the Cp is less than the statistically-set 5% confidence-value threshold (Franklin W A H J, Sugita M, Bemis L, Jimeno A, Messersmith W A KRAS mutation: Comparison of testing methods and tissue sampling techniques in colon cancer. J Mol Diagn 2009).","The CRC cell lines were analyzed for KRAS mutations by the University of Colorado Cancer Center Pathology Core with a high resolution melting temperature method using custom primers and the Roche LC480 real time PCR machine (Mannheim, Germany). Briefly, template DNA was tested by High Resolution Melting (HRM) analysis using a Lightcycler 480 real-time PCR instrument (Roche Applied Science, Indianapolis, Ind.). Approximately 60 ng of tumor template DNA, wild type control DNA and mutant control DNA were amplified on the Lightcycler 480 instrument using HRM master mix (Roche cat #04909631001), with the RASO1 and RASA2 primers and 1.75 mM MgCl2 in a 10 \u03bcl on a 96 well plate, using a 2-step cycling program (95\u00b0 melting, 72\u00b0 annealing and extension) for 45 cycles. PCR products were analyzed by HRM with 25 data acquisitions per degree of temperature increase, from 40\u00b0 to 90\u00b0 C. Lightcycler 480 Gene Scanning software using the known wild-type control samples for baseline calculation was used for these analyses (3). BRAF and PI3K mutations were analyzed by PCR amplification and direct sequencing of the products as described previously (Jhawer M, et al. Cancer Res 2008; 68(6):1953-61). Primers used were F, AACACATTTCAAGCCCCAAA SEQ ID NO:115 and R, GAAACTGGTTTCAAAATATTCGTT SEQ ID NO:116 for amplification of exon 15 of BRAF; F, GCTTTTTCTGTAAATCATCTGTG SEQ ID NO:117 and R, CTGAG-ATCAGCCAAATTCAGT SEQ ID NO:118 for exon 9 of PIK3CA; and F, CATTTGCTCCAAACTGACCA SEQ ID NO:119 and R, TACTCCAAAGCCTCTTGCTC SEQ ID NO:120 (for codon 1023 mutation) and F, ACATTCGAAA-GACCCTAGCC SEQ ID NO:121 and R, CAATTCCTATGCAATCGGTCT SEQ ID NO:122 (for codon 1047 mutation) for exon 20 of PIK3CA.","Gene expression profiles: Cells were plated at 2\u00d710in 6-well plates 24 h prior to harvest. After 24-72 hours cells were rinsed twice with PBS, and RNA was prepared using a RNeasy Plus mini kit (Qiagen, Valencia, Calif.). RNA stabilization, isolation, and microarray sample labeling were carried out using standard methods for reverse transcription and one round of in vitro transcription. Total RNAs isolated from CRC cell lines and tumor xenografts were hybridized on Affymetrix U133 Plus 2.0 gene arrays at least in duplicates. This gene array has \u02dc54,000 probes comprising \u02dc20,000 genes. Sample preparation and processing procedure was performed as described in the Affymetrix GeneChip\u00ae Expression Analysis Manual (Affymetrix Inc., Santa Clara, Calif.). In addition, CRC cell line gene expression profiles were obtained from the GlaxoSmithKline (GSK) genomic profiling data via the NCI cancer Bioinformatics Grid (caBIG\u00ae). These data were also profiled using Affymetrix U133 Plus 2.0 gene arrays in triplicates. To integrate the data generated from our lab and GSK, absolute intensity signals from the microarray gene expression profiles were extracted and probe sets representing the same gene were collapsed based on maximum values. Next, the gene expression levels were converted to a rank-based matrix and standardized (mean=0, standard deviation=1) for each microarray. Using this pre-processing method, the same cell lines from different data sets were clustered based on their gene expression profiles. Data analyses were performed on this rank-based matrix.","shRNA knockdown: The pRS-shE2F6 gene-specific shRNA expression cassettes, along with control shRNA plasmids including the original pRS vector (TR20003, were purchased from OriGene (Rockville, Md.). The sequence of the metallothionein 2A-specific 29mer shRNA is GTAAAGAACGCGACTTCCACA-AACCTGGA SEQ ID NO:123. Stable clones were generated by transfecting HCT116 cells in 6-well dishes with 1 \u03bcg of each of the shRNA plasmids using Eugene 6 (Roche, Basel Switzerland), according to manufacturer's recommendations. Seventy-two hours after transfection, the cells were placed under selection with 2.0 \u03bcg\/mL of puromycin, splitting 1:5 when the cells reached confluency. Multiple clones from the same transfection were pooled and grown under puromycin selection. Successful knockdown of specific genes and gene products was confirmed by semi-quantitative RT-PCR and immunoblotting with specific antibodies.","Gene set enrichment analysis: Gene set analysis was performed using the GSEA software Version 2.0.1 obtained from the Broad Institute (Subramanian A. et al, Proc Natl Acad Sci USA. 2005 Oct. 25; 102(43):15545-50. Epub 2005 September 30; PMID: 16199517). Gene set permutations were performed 1000 times for each analysis. We used the nominal p-value and Normalized Enrichment Score (NES) to sort the pathways enriched in each phenotype. We used the 199 pathways defined by Kyoto Encyclopedia of Genes and Genomes (KEGG) database as the gene set in this study (PMID: 18077471; Kanehisa M., et al., Nucleic Acids Res. 2008 January; 36 (Database issue):D480-4. Epub 2007 December 12). Human pathway annotations were downloaded from KEGG (August 2007 release). The KEGG human pathways used in this study include metabolism, genetic information processing, environmental information processing, cellular processes and human diseases. One hundred and sixty-six gene sets passed the gene set size filter criteria (min=10, max=500).","K-TSP classifier: We used the K-TSP algorithm (PMID: 16105897; Kanehisa M. et al., Nucleic Acids Res. 2008 January; 36 (Database issue):D480-4. Epub 2007 Dec. 12.) to construct a discriminative classifier in predicting tumors sensitive to OSI-906. In brief, the algorithm exploits the information contained in the rank-based matrix by focusing on \u201cmarker gene pairs\u201d (i, j) for which there is a significant difference in the probability of the event (R<R) across the N samples from class Y=1 (OSI-906 sensitive) to Y=\u22121 (OSI-906 resistant), where the event (R<R) is equivalent to the rank of gene i is less than the rank of gene j if and only if gene i is expressed less then gene j (relative expression). Here, the quantities of interest are p(m)=Prob(R<R|Y=m), m=(1, \u22121), i.e., the probabilities of observing R<Rin each class. These probabilities are estimated by the relative frequencies of occurrences of R<Rwithin profiles and over samples. Let \u0394denote the \u201cscore\u201d of gene pair (i, j), where \u0394=|p(1)\u2212p(\u22121)|. A score \u0394is computed for every pair of genes i, j\u03b5{1, . . . , P}, i\u2260j. Gene pairs with high scores are viewed as most informative for classification. Using an internal leave-one-out cross-validation, the final k-TSP classifier utilizes the k disjoint pairs of genes, which achieve the k best scores from the training set. In this study, maximum number of pairs (kmax) was fixed as 10.","In vivo Growth Studies: Five to six-week-old female athymic nude mice (Harlan Sprague Dawley) were used. The research protocol was approved by the University of Colorado Denver, Animal Care and Use Committee. Mice were caged in groups of five and kept on 12-h light\/dark cycle and provided with sterilized food and water ad libitum. All of the studies were conducted in accordance with the NIH guidelines for the care and use of laboratory animals, and animals were housed in a facility accredited by the American Association for Accreditation of Laboratory Animal Care. Animals were allowed to acclimate for at least 7 days before any handling.","The primary CRC xenografts were generated according to published methodology (Rubio-Viqueira B, et al. Clin Cancer Res 2006; 12(15):4652-61.). Briefly, surgical specimens of patients operated at the University of Colorado Hospital were reimplanted s.c. to 5 mice for each patient. Tumors were let to grow to a size of 1000-1500 mmat which point were harvested, divided, and transplanted to another 5 mice to maintain the tumor bank. After a subsequent growth passage, tumors were excised and propagated to cohorts of \u226725 mice, for treatment. Tumors from this cohort were allowed to grow until reaching \u02dc150-300 mm, at which time they were evenly distributed by size in the two treatment groups. Tumors from this treatment stage were treated for 28 days with either vehicle control (25 mM tartaric acid), or OSI-906 (40 mg\/kg). Mice were monitored daily for signs of toxicity and were weighed twice weekly. Tumor size was evaluated two times per week by caliper measurements using the following formula: tumor volume=[length\u00d7width]\/0.52. Relative tumor growth index was calculated by relative tumor growth of treated mice divided by relative tumor growth of control mice since the initiation of therapy (T\/C).","For cell line xenografts mice were allowed to acclimate as above. All CRC cells were harvested in exponential phase growth and resuspended in a 1:1 mixture of serum-free RPMI 1640 and Matrigel (BD Biosciences). Five to 10 million cells per mouse were injected s.c. into the flank using a 23-gauge needle. Mice were monitored daily for signs of toxicity and were weighed twice weekly. Tumor size was evaluated two times per week by caliper measurements using the following formula: tumor volume=[length\u00d7width2]\/0.52. When tumors reached 150-300 mm3 mice were randomized into two groups with at least 10 tumors per group. Mice were treated for 14 days with either vehicle or OSI-906 as above.","Statistical methods. Significance levels for comparison of groups were calculated using GraphPad Prism Software (La Jolla, Calif.). Differences were considered significant at P<0.05.",{"@attributes":{"id":"p-0383","num":"0383"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"308pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2b"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"miRNAs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"35pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},{},"SLS,",{},{}]},{"entry":["NCBI",{},"NCBI","Sanger ID:","Sanger","Sanger:","Sanger:"]},{"entry":["Official","NCBI Official","Gene ID","Stem-loop","miRBase","Mature","Minor"]},{"entry":["Symbol","Full Name","#.","sequence (SLS)","Accession #","sequence","sequence"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"35pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["MIR224","microRNA 224","407009","hsa-miR-224","MI0000301","hsa-miR-",{}]},{"entry":[{},{},{},{},{},"224"]},{"entry":["MIR181A1","microRNA 181a-1","406995","hsa-miR-181a1","MI0000289","hsa-miR-"]},{"entry":[{},{},{},"hsa-miR-181a2",{},"181a"]},{"entry":["MIR194-1","microRNA 194-1","406969","hsa-miR-194-1","MI0000488","hsa-miR-"]},{"entry":["MIR194-2","microRNA 194-2","406970","hsa-miR-194-2","MI0000732","194"]},{"entry":["MIR584","microRNA 584","693169","hsa-miR-584","MI0003591","hsa-miR-"]},{"entry":[{},{},{},{},{},"584"]},{"entry":["MIR215","microRNA 215","406997","hsa-miR-215","MI0000291","hsa-miR-"]},{"entry":[{},{},{},{},{},"215"]},{"entry":["MIR429","microRNA 429","554210","hsa-miR-429","MI0001641","hsa-miR-"]},{"entry":[{},{},{},{},{},"429"]},{"entry":["MIR200A","microRNA 200a","406983","hsa-miR-200a","MI0000737","hsa-miR-"]},{"entry":[{},{},{},{},{},"200a"]},{"entry":["MIR200B","microRNA 200b","406984","hsa-miR-200b","MI0000342","hsa-miR-","hsa-miR-"]},{"entry":[{},{},{},{},{},"200b","200b*"]},{"entry":["MIR886","microRNA 886","100126299","hsa-miR-886","MI0005527","hsa-miR-"]},{"entry":[{},{},{},{},{},"886-3p"]},{"entry":["MIR521-1","microRNA 521-1","574494","hsa-miR-521-1","MI0003176","hsa-miR-"]},{"entry":["MIR521-2","microRNA 521-2","574481","hsa-miR-521-2","MI0003163","521"]},{"entry":["MIR432","microRNA 432","574451","hsa-miR-432","MI0003133","hsa-miR-"]},{"entry":[{},{},{},{},{},"432"]},{"entry":["MIR192","microRNA 192","406967","hsa-miR-192","MI0000234","hsa-miR-","hsa-miR-"]},{"entry":[{},{},{},{},{},"192","192*"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}}]}}]}}},"miRNA profiling. miRNA profiling was performed at Dharmacon (Lafayette, Colo.) using their proprietary miRNA probe set and a two color high-density array. The miRNA patterns in OSI906 sensitive (S) and resistant (R) CRC cell lines were confirmed using qRT-PCR. Total RNA was extracted from the following eight colon cancer cell lines: LS513, COLO205, HT29, GEO, HCT-15, RKO, HCT-8, and SW480 using Qiagen's miRNeasy Mini Kit. Next, reverse transcription was performed using Qiagen's miScript Reverse Transcription Kit, followed by Real-time qPCR using miRNA specific primers and the Step One Plus Real-Time PCR Software. The experiment was repeated three times in order to confirm the results. In order to assess the functional significance of miRNA expression with regard to OSI906 sensitivity, transfection was performed on one sensitive cell line with miR-181a and miR-224 antagonists and one resistant cell line with miR-181a and miR-224 mimic. The cells were transfected in 60 mm dishes for 24 hours, after which the samples were plated into 96-well plates, and treated with different doses of OSI906 (5 \u03bcM-0.075 \u03bcM). The results were analyzed using SRB. The remainder of the samples transfected with miR-224 was plated in 60 mm dishes and qRT-PCR was performed. The miRNAs profiled, and their encoding genes, are described in Table 2b. The sequences and other details of the probed miRNAs can be found at any of a number of online databases, including the Sanger miRNA database \u201cmiRBase\u201d; and the NCBI Entrez Gene database (National Center for Biotechnology Information (NCBI), U.S. National Library of Medicine, 8600 Rockville Pike, Building 38A, Bethesda, Md. 20894.","Results and Discussion","Assessment of responsiveness of a panel of CRC cell lines to OSI-906. In order to assess CRC cell lines with differential sensitivity to OSI-906, a panel of 27 CRC cell lines were exposed to increasing concentrations of OSI-906 and proliferation was measured using the SRB assay as described previously (Pitts, T M, et al. Mol Cancer Ther 2009; 8(2):342-9). As depicted in  there was a wide range of sensitivity of the CRC cell lines to OSI-906, the majority of the cell lines failing to reach an IC50 up to 5 \u03bcmol\/L, but a clear distinction could be made between the cell lines that were sensitive and the cell lines that were resistant. For classification, a sensitive cell line was classified as one with an IC\u22661.5 \u03bcmol\/L, whereas resistant cell lines had ICof >5 \u03bcmol\/L. Six CRC cell lines from this panel met the criteria as being sensitive (i.e. Colo205, GEO, LS513, HT29, CaCo2 and LS1034), and the remaining 21 were resistant (i.e. Colo201, SK-CO-1, SW948, SW48, NCI-H508, HCT116, HCT15, SW480, RKO, HCT8, LoVo, LS123, T84, LS174T, LS180, SW1417, SW1116, SW837, SW1463, SW620 and SW403).","Fluorescent in situ hybridization (FISH) analysis of CRC cell lines for IGF-1R gene copy number. Since previous studies have suggested that increased EGFR copy number may be predictive for EGFR-directed agents, we assessed IGF-1R gene copy number in the panel of cell lines (Hirsch F R, et al. J Clin Oncol 2008; 26(20):3351-7). Using an IGF-1R specific probe, twenty-seven CRC cell lines with differing sensitivities to the IGF-1R inhibitor OSI-906 were subjected to FISH analysis. Although IGF-1R gene amplification was not observed in any of the CRC cell lines, several of them displayed an unbalanced gain (based upon ploidy) of IGF-1R copy number, with a trend (p=0.17) towards a relationship between the presence of unbalanced gain and sensitivity (Table 3). Representative spreads\/interphase nuclei are depicted in  for a sensitive (18A) and resistant (18B) cell line.","Assessment of KRAS\/BRAF\/PI3K gene mutation status by sequencing. Since tumors with mutant KRAS\/BRAF or PI3K demonstrate resistance to EGFR-based therapies, we characterized the KRAS\/BRAF\/PI3K mutational status of the CRC cell lines (Sartore-Bianchi A, et al. Cancer Res 2009; 69(5):1851-7; Di Nicolantonio F, et al. J Clin Oncol 2008; 26(35):5705-12; Perrone F, et al. Ann Oncol 2009; 20(1):84-90; Jimeno A, et al. Cancer J 2009; 15(2):110-3). Although the correlation observed between KRAS status and OSI-906 sensitivity did not reach statistical significance, there was a trend (p=0.3442) towards KRAS wild-type tumors being more sensitive, and KRAS mutants being more resistant. There was no relationship between either BRAF or PI3K mutation status and responsiveness to OSI-906.",{"@attributes":{"id":"p-0389","num":"0389"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Baseline FISH analysis of OSI-906 sensitive and resistant CRC"},{"entry":"cell lines demonstrating IGF-IR gain based on ploidy."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Cell Line","IGF1R\/Ploidy","S\/I\/R to OSI-906"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"CoLo205","Gain - High","S"]},{"entry":[{},"GEO","Balanced","S"]},{"entry":[{},"HT29","Gain - High","S"]},{"entry":[{},"LS513","Gain - Low","S"]},{"entry":[{},"CaCo2","Gain - Low","S"]},{"entry":[{},"LS1034","Gain - High","S"]},{"entry":[{},"CoLo201","Balanced","R"]},{"entry":[{},"SK-CO-1","Gain - High","R"]},{"entry":[{},"SW948","Balanced","R"]},{"entry":[{},"SW620","Balanced","R"]},{"entry":[{},"T-84","Gain - Low","R"]},{"entry":[{},"HCT116","Balanced","R"]},{"entry":[{},"HCT15","Balanced","R"]},{"entry":[{},"HCT8","Balanced","R"]},{"entry":[{},"LoVo","Gain - High","R"]},{"entry":[{},"LS123","Gain - Low","R"]},{"entry":[{},"LS174T","Balanced","R"]},{"entry":[{},"LS180","Balanced","R"]},{"entry":[{},"NCI-H508","Balanced","R"]},{"entry":[{},"RKO","Balanced","R"]},{"entry":[{},"SW1116","Gain Low","R"]},{"entry":[{},"SW1417","Balanced","R"]},{"entry":[{},"SW1463","Loss","R"]},{"entry":[{},"SW403","Balanced","R"]},{"entry":[{},"SW48","Balanced","R"]},{"entry":[{},"SW480","Balanced","R"]},{"entry":[{},"SW837","Balanced","R"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},"Identification of genes that were differentially expressed in OSI-906 sensitive versus OSI-906-resistant CRC cell lines. To initially identify genes that correlated with sensitivity to OSI-906, we analyzed the basal gene expression profiles of the four most sensitive and the four most resistant CRC cell lines. Using the two-sample t-test, 139 genes were identified as differentially expressed in OSI-906 sensitive and OSI-906 resistant CRC cell lines. To characterize these potential biomarkers we decided to focus the top scoring genes (p<0.002) for the initial studies. For example, caldesmon (61-fold up regulated), an actin-binding protein has recently been shown to play a critical role in regulating the formation and dynamics of podosomes and invadopodia, cell adhesion structures that protrude from the plasma membrane and degrade the extracellular matrix (ECM), thus promoting cancer cell invasion (Linder S, et al.: Trends Cell Biol 2003; 13(7):376-85). Interestingly, caldesmon is a negative regulator of the formation podosomes and invadopodia, indicating that CRC cell lines with an invasive\/malignant phenotype are more responsive to IGF-1R inhibition. Metallothioneins (MT) as a group were also up-regulated in the resistant CRC cell lines versus the sensitive. MT are a family of ubiquitous, low molecular weight intracellular proteins that bind and detoxify heavy metal ions (Kagi J H. Methods Enzymol 1991; 205:613-26; Bauman J W, et al. Toxicol Appl Pharmacol 1991; 110(2):347-54). This family represented the highest ranked group (9-36-fold increase) of differentially expressed genes. MT can be induced by a variety of stimuli, are involved in other cellular functions such development, differentiation proliferation, and carcinogenesis, and have been associated with a poor prognosis and metastasis in cancer (Bruewer M, et al. World J Surg 2002; 26(6):726-31; Sens M A, et al. Am J Pathol 2001; 159(1):21-6; Cherian M G, et al. Mutat Res 2003; 533(1-2):201-9; Jin R, et al. Br J Cancer 2000; 83(3):319-23). MT may also play a functional role in cancer drug resistance, though the mechanism for this remains poorly understood (Surowiak P, et al. Histol Histopathol 2005; 20(4):1037-44; Saga Y, et al. Int J Urol 2004; 11(6):407-15). Several genes were also up-regulated in the sensitive lines. Of interest was the aldehyde hehydrogenease (ALDH1A1, 83 fold) and the mitogen-activated protein kinase kinase 6 (MAP2K6, 11 fold). ALDH1A1 is an enzyme involved in the metabolism of alcohol and interestingly, the oxidation of all-trans retinal to all-trans retinoic acid (Molotkov A, Duester G. Genetic evidence that J Biol Chem 2003; 278(38):36085-90; Rice K L, et al. Leuk Res 2008; 32(6):873-83). MAP2K6 is a protein that phosphorylates and activates p38 MAP kinase in response to inflammatory cytokines or environmental stress. As an essential component of p38 MAP kinase mediated signal transduction pathway, this gene is involved in many cellular processes such as stress induced cell cycle arrest, transcription activation invasion\/migration and apoptosis (Abdollahi T, et al. Apoptosis 2005; 10(6):1383-93; Hsieh Y H, et al. Cancer Res 2007; 67(9):4320-7; Timofeev O, et al. Cell Cycle 2005; 4(1):118-20).","Training and validation of K-TSP classifier for predicting OSI-906 sensitivity. A main goal of this project was to develop a classifier to predict sensitivity to OSI-906. To this end we used both cell lines and xenografts treated with OSI-906 as a training set. We used baseline gene array from the 4 most sensitive and 4 of the most resistant cell lines grown both in vitro and in vivo. In this study we used the K-TSP algorithm as a discriminative classifier (Tan A, et al. Bioinformatics 2005; 2:3896-904), which has been proven in various studies. Using an internal leave-one-out cross-validation, the final k-TSP classifier utilizes the k disjoint pairs of genes, which achieve the k best scores from the training set. In this study, maximum number of pairs (kmax) was fixed as 10. Using four sensitive and resistant cell lines to OSI-906, the k-TSP classifier identified three gene pairs as the final classifier: (PROM1, MT1E), (LY75, OXCT1) and (HSD17B2, CALD1). Interestingly two of these genes, MT1E and CALD1 also appeared in the gene analysis above. In addition to the gene pairs we integrated KRAS mutational (WT) status and IGF-1R FISH (unbalanced gain) analysis to develop a signature of sensitivity to the IGF-1R inhibitor.  shows diagrammatically how this prediction is used. In order for a tumor to predict as sensitive it must meet 4 of the 5 classifiers. From the training data, the integrated genomic classifier achieved an estimated leave-one-out cross-validation of 85.8%. Using this integrated genomic classifier, we validated its prediction in the 18 CRC cell lines not used in the training set. In this validation set, the integrated genomic classifier correctly predicted the OSI-906 sensitivity on 89% (16\/18) cell lines.","shRNA knockdown of potential predictive biomarker genes. In order to determine whether any of the highly ranked genes had a functional role in mediating responsiveness to OSI-906 we performed knockdown experiments with shRNA. CRC cell lines were transfected with shRNA for the genes listed above (Caldesmon, Metallothionein 2A (MT2A), MT1E, ALDH1A1, and MAP2K6) and the phenotype was analyzed by exposing the CRC cell lines to increasing concentrations of OSI-906. As depicted in , shRNA knockdown of MT2A resulted in a robust decrease in MT2A RNA and protein in OSI-906 resistant HCT116 and SW480 cells. These cells, when exposed to increasing concentrations of OSI-906 exhibited a \u201cleft-shift\u201d in sensitivity (IC>5 \u03bcmol\/L to IC=5 \u03bcmol\/L). In contrast the SW480 cells did not exhibit the same \u201cleft shift\u201d . Similar shRNA knockdown of the other genes noted above, did not demonstrate a functional role in mediating sensitivity (ALDH1A1, MAP2K6) or resistance (CALD1, MT1E) to OSI-906 (data not shown).","K-TSP, FISH, KRAS status can predict sensitivity to OSI-906 in human tumor explants. To further validate the predictive classifier we predicted the sensitivity of 5 human tumor explants prior to treating with OSI-906. The classifier predicted three explants as resistant and two as sensitive. The overall accuracy for the predictor in the human tumor explants was 100%. Following treatment with OSI-906, the two that predicted as sensitive were indeed sensitive (TGI=21-28%) and the three that predicted as resistant were resistant (TGI=130-200%).  show representative graphs of two resistant explants and two sensitive explants.","IGF-1R pathway analysis by immunoblotting and IHC. To determine if, by analyzing downstream effectors of the IGF-1R pathway, a pattern of responsiveness existed, CRC cell lines were subjected to immunoblotting and IHC at baseline. As depicted in , none of the major components of the IGF-1R pathway appeared to predict sensitivity to OSI-906. For example, activated (phosphorylated) IGF-1R, ERK, AKT, IRS-1, mTOR, PI3K, Shc, and S6 kinase were variably present at baseline in both sensitive and resistant cell lines (3A). Similarly, baseline expression of phosphorylated ERK, AKT, IGF-1R, IGF-2R, and total IGF by IHC did not correspond to sensitivity to OSI-906. Table 4 shows IHC scores of the CRC cell lines","Pathway analysis of OSI-906-sensitive and -resistant CRC cell lines at baseline and post OSI-906 exposure. To determine whether any particular pathway was associated with responsiveness to OSI-906, pathway enrichment analysis was performed on the CRC cell lines pre-exposure. Sixty-eight KEGG pathways were up regulated in sensitive versus resistant CRC cell lines. Table 5 depicts the top 20 pathways that were overrepresented in the sensitive cell lines, including the Wnt, hedgehog, basal cell carcinoma, and p53 signaling pathways. Within these pathways core genes included several frizzled receptors and their associated wnt ligands. In the p53 pathway, not only was the p53 gene up regulated, but the IGF-1R ligand, IGF-1 was also over expressed. In the OSI-906-resistant cell lines there were also several unique pathways that were up regulated, including ErbB, MAPK, and VEGF signaling as well as those involving cell adhesion, such as cell adhesion molecules (CAM), and focal adhesion kinase. Within these and other pathways several core genes may be involved in conferring resistance to OSI-906.","In order to determine which pathways\/genes were modulated following exposure to OSI-906, pathway analysis was performed as before on the sensitive and resistant CRC cell lines. In the OSI-906 sensitive cell lines the thyroid cancer, pancreatic cancer and the notch signaling pathways were up regulated. Core genes involved in these pathways include; TP53, ErbB2, STAT3 and CDKN2. Determining pathways and genes that are modulated in resistant cell lines can be extremely beneficial in finding rationale drug combinations that may be able to sensitize tumors to treatments. Pathway analysis of the OSI-906 resistant cell lines demonstrated that several pathways including colorectal cancer, VEGF, and Fc epsilon R1 signaling pathways were all up regulated following exposure. Interestingly genes involved in these pathways include several MAPKs, AKTs and FZD receptors.","miRNA profiling analysis. Following miRNA profiling a heat map was generated showing the differentially expressed miRNAs in OSI-906 sensitive and resistant CRC cell lines (). Two miRNAs, hsa-miR-224 and hsa-miR-181a were shown to be upregulated in the OSI-906 sensitive CRC cell lines. We decided to look at these two miRNAs because they have been shown previously (Prueitt R L, et al. Prostate 2008; 68(10:1152-64) to possibly be associated with metallothionein expression and they show an inverse relationship ().","In order to determine if hsa-miR-224 had any effect on proliferation when exposed to OSI-906, a resistant CRC cell line (RKO) was transfected with hsa-miR-224 mimic and proliferation was assessed by SRB assay. As can be seen in  when hsa-miR-224 mimic is transfected into RKO cells an increase in the inhibition of proliferation is seen indicating the hsa-miR-224 may have an effect on the sensitivity to OSI-906. To see if the opposite is true we transfected an OSI-906 sensitive (HT29) cell line with hsa-miR-224 antagonist and proliferation was assessed by the SRB assay. As can be seen in , hsa-miR-224 antagonist does not affect proliferation.","Several other miRNAs were also upregulated, but to a lesser extent, in the OSI-906 sensitive CRC cell lines. These included hsa-miR-194, hsa-miR-192, hsa-miR-215, hsa-miR-200b, hsa-miR-429, hsa-miR-200a, hsa-miR-192*, hsa-miR-200b*, hsa-miR-584. Three miRNAs were upregulated in the OSI-906 resistant CRC cell line, i.e. hsa-miR-886-3p, hsa-miR-521, and hsa-miR-432.","These upregulated miRNAs, in tumor cell lines that are either sensitive or resistant to IGF-1R kinase inhibitors (e.g. OSI-906, anti-IGF-1R antibodies), provide additional biomarkers that may be utilized to predict sensitivity of tumor cell lines to IGF-1R kinase inhibitors. Additionally, the hsa-miR-224 mimic enhancement of the sensitivity of a resistant tumor cell line to inhibition by an IGF-1R kinase inhibitor enables new therapeutic options comprising combinations of an IGF-1R kinase with agents possessing similar hsa-miR-224 mimetic activity.","A broad range of sensitivity to OSI-906 was observed among the 27 CRC cell lines in vitro that was recapitulated by the 4 sensitive and 4 most resistant cell lines in vivo. To develop the K-TSP classifier, we utilized both in vitro and in vivo baseline gene array for a given cell line in the training set so that differences relating purely to the tumor microenvironment would be minimized. Our integrative genomic classifier differs from other published drug-response gene-expression signatures in two important aspects, first, as opposed to other studies where the gene signatures remain unvalidated, we validated our integrative genomic signature on a set of independent direct-human CRC explants as an intermediate step before moving into the clinic, and second, the classifier is comprised of the relative expression of 3 gene pairs, KRAS mutation status, and IGF-1R FISH analyses, all of which can be readily applied to clinical settings (Zha J, et al. Mol. Cancer Ther. 2009 August; 8(8):2110-21. Epub 2009 Aug. 11; Watters J W, Roberts C J. Mol Cancer Ther. 2006 October; 5(10):2444-9; Potti A, et al. Nat Med. 2006 November; 12(11): 1294-300. Epub 2006 Oct. 22). In summary, these data demonstrate that an integrated approach to the development of predictive biomarkers in the early clinical development of IGF-1R kinase inhibitors is feasible.",{"@attributes":{"id":"p-0402","num":"0402"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 4"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"IHC baseline patterns"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"8"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["ID","% (+) cells","0","1+","2+","3+","4+","H Score"]},{"entry":{"@attributes":{"namest":"1","nameend":"8","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"8"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"8","colwidth":"28pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},{},{},"pMAPK",{},{}]},{"entry":["LS 513","5","95",{},"2","3",{},"13"]},{"entry":["LS180","1","99",{},"1",{},{},"2"]},{"entry":["HT29","1","99",{},{},"1",{},"3"]},{"entry":["Colo201","1","99",{},{},"1",{},"3"]},{"entry":["SW480","90","10",{},{},"10","80","350"]},{"entry":["HCT8","20","80","2","2","16",{},"54"]},{"entry":["Colo205","1","99","1",{},{},{},"1"]},{"entry":["RKO","20","80","5","5","5","5","50"]},{"entry":["HCT116","1","99","1",{},{},{},"1"]},{"entry":["SW620","90","10","90",{},{},{},"90"]},{"entry":["HCT15","1","99",{},"1",{},{},"2"]},{"entry":["SW1463","10","90",{},{},"10",{},"30"]},{"entry":["NCIH508","1","99","1",{},{},{},"1"]},{"entry":["SW48","90","10",{},"10","80",{},"260"]},{"entry":["SW403","70","30",{},"10","60",{},"200"]},{"entry":["SW1417","30","70",{},{},"30",{},"90"]},{"entry":["LOVO","10","90","2","3","5",{},"23"]},{"entry":["CaCo2","2","98",{},"1","1",{},"5"]},{"entry":["SW837","5","95",{},"1","4",{},"14"]},{"entry":["LS123","30","70",{},"5","5","20","105"]},{"entry":["GEO","40","60",{},{},"10","30","150"]},{"entry":["LS1034","10","90",{},{},"4","6","36"]},{"entry":["SKCO1","40","60",{},{},"20","20","140"]},{"entry":["SW1116","20","80",{},{},"18","2","62"]},{"entry":["T84",{},{},{},{},{},{},"0"]},{"entry":["SW948","100",{},"100",{},{},{},"100"]},{"entry":["LS174T","100",{},"60","40",{},{},"140"]},{"entry":[{},{},{},{},{},"IGF 2"]},{"entry":["LS 513","100",{},{},{},"95","5","305"]},{"entry":["LS180","100",{},"80",{},"20",{},"140"]},{"entry":["HT29","100",{},{},"90","10",{},"210"]},{"entry":["Colo201","100",{},{},"99","1",{},"201"]},{"entry":["SW480","100",{},"100",{},{},{},"100"]},{"entry":["HCT8","100",{},{},{},"100",{},"300"]},{"entry":["Colo205","100",{},{},{},{},"100","400"]},{"entry":["RKO","100",{},"10","30","30","30","280"]},{"entry":["HCT116","100",{},{},"80",{},"20","240"]},{"entry":["SW620","100",{},{},{},"100",{},"300"]},{"entry":["HCT15","100",{},{},{},"10","90","390"]},{"entry":["SW1463","100",{},{},{},"90","10","310"]},{"entry":["NCIH508","100",{},{},{},"100",{},"300"]},{"entry":["SW48","100",{},{},"99","1",{},"201"]},{"entry":["SW403","100",{},{},{},"60","40","340"]},{"entry":["SW1417","100",{},{},{},{},"100","400"]},{"entry":["LOVO","100",{},{},"99","1",{},"201"]},{"entry":["CaCo2","100",{},{},{},{},"100","400"]},{"entry":["SW837","100",{},{},{},"100",{},"300"]},{"entry":["LS123","100",{},{},{},{},"100","400"]},{"entry":["GEO","100",{},{},{},"100",{},"300"]},{"entry":["LS1034","100",{},{},{},{},"100","400"]},{"entry":["SKCO1","100",{},{},"10","80","10","300"]},{"entry":["SW1116","100",{},{},"100",{},{},"200"]},{"entry":["T84","100",{},{},{},"100",{},"300"]},{"entry":["SW948","100",{},{},{},{},"100","400"]},{"entry":["LS174T","100",{},{},{},"100",{},"300"]},{"entry":[{},{},{},{},{},"IGF 2r"]},{"entry":["LS 513","100",{},"98",{},"1","1","105"]},{"entry":["LS180","100",{},"100",{},{},{},"100"]},{"entry":["HT29","100",{},"100",{},{},{},"100"]},{"entry":["Colo201","100",{},{},"100",{},{},"200"]},{"entry":["SW480","100",{},{},"100",{},{},"200"]},{"entry":["HCT8","100",{},{},"99","1",{},"201"]},{"entry":["Colo205","100",{},{},"100",{},{},"200"]},{"entry":["RKO","100",{},{},"100",{},{},"200"]},{"entry":["HCT116","100",{},"90","10",{},{},"110"]},{"entry":["SW620","100",{},"80","20",{},{},"120"]},{"entry":["HCT15","100",{},{},"90","10",{},"210"]},{"entry":["SW1463","100",{},{},{},"80","20","320"]},{"entry":["NCIH508","100",{},{},"95",{},"5","210"]},{"entry":["SW48","100",{},{},"100",{},{},"200"]},{"entry":["SW403","100",{},{},"95","5",{},"205"]},{"entry":["SW1417","100",{},{},{},"95","5","305"]},{"entry":["LOVO","100",{},{},"100",{},{},"200"]},{"entry":["CaCo2","100",{},{},"95","5",{},"205"]},{"entry":["SW837","100",{},{},"100",{},{},"200"]},{"entry":["LS123","100",{},{},"100",{},{},"200"]},{"entry":["GEO","100",{},{},"95","5",{},"205"]},{"entry":["LS1034","65",{},{},"60","5",{},"135"]},{"entry":["SKCO1","100",{},{},"90","10",{},"210"]},{"entry":["SW1116","100",{},{},{},"90","10","310"]},{"entry":["T84","100",{},"80","15","5",{},"125"]},{"entry":["SW948","100",{},"100",{},{},{},"100"]},{"entry":["LS174T","100",{},{},"100",{},{},"200"]},{"entry":[{},{},{},{},{},"IGF 1r"]},{"entry":["LS 513","100",{},{},"100",{},{},"200"]},{"entry":["LS180","100",{},"99","1",{},{},"101"]},{"entry":["HT29","100",{},"99","1",{},{},"101"]},{"entry":["Colo201","100",{},{},{},"100",{},"300"]},{"entry":["SW480","100",{},{},"50","50",{},"250"]},{"entry":["HCT8","100",{},"99","1",{},{},"101"]},{"entry":["Colo205","100",{},{},{},"100",{},"300"]},{"entry":["RKO","100",{},{},"100",{},{},"200"]},{"entry":["HCT116","100",{},"100",{},{},{},"100"]},{"entry":["SW620","100",{},"99","1",{},{},"101"]},{"entry":["HCT15","100",{},"90","10",{},{},"110"]},{"entry":["SW1463","100",{},{},{},"100",{},"300"]},{"entry":["NCIH508","100",{},{},{},{},"100","400"]},{"entry":["SW48","100",{},{},{},{},"100","400"]},{"entry":["SW403","100",{},{},{},{},{},"100"]},{"entry":["SW1417","100",{},{},{},"90","10","310"]},{"entry":["LOVO","100",{},{},{},{},"100","400"]},{"entry":["CaCo2","100",{},{},{},{},"100","400"]},{"entry":["SW837","100",{},{},{},{},"100","400"]},{"entry":["LS123","100",{},{},{},"90","10","310"]},{"entry":["GEO","70",{},{},"65","5",{},"145"]},{"entry":["LS1034","100",{},{},"80","10","10","230"]},{"entry":["SKCO1","100",{},{},{},{},"100","400"]},{"entry":["SW1116","100",{},{},{},"10","90","390"]},{"entry":["T84","100",{},{},"10","90",{},"290"]},{"entry":["SW948","100",{},{},{},{},"100","400"]},{"entry":["LS174T","100",{},{},{},"100",{},"300"]},{"entry":[{},{},{},{},{},"pIGF1r"]},{"entry":["LS 513","60","40","20","20","10","10","130"]},{"entry":["LS180","80","20","5","5","5","5","50"]},{"entry":["HT29","40","60","0","10","10","20","130"]},{"entry":["Colo201","90","10","0","10","20","60","320"]},{"entry":["SW480","80","20","50","10","15","5","135"]},{"entry":["HCT8","100","0","10","10","70","10","280"]},{"entry":["Colo205","100","0","0","0","10","90","390"]},{"entry":["RKO","90","10","0","0","5","5","35"]},{"entry":["HCT116","100","0","40","40","10","10","190"]},{"entry":["SW620","80","20","60","10","10","0","110"]},{"entry":["HCT15","90","10","0","20","0","70","320"]},{"entry":["SW1463","100","0","90","0","10","0","120"]},{"entry":["NCIH508","60","10","20","0","10","60","290"]},{"entry":["SW48","100","0","0","10","10","80","370"]},{"entry":["SW403","100","0","90","10","0","0","110"]},{"entry":["SW1417","100","0","0","0","20","80","380"]},{"entry":["LOVO","100","0","0","10","20","70","360"]},{"entry":["CaCo2","100","0","0","0","0","100","400"]},{"entry":["SW837","100","0","0","50","10","40","290"]},{"entry":["LS123","100","0","0","0","80","20","320"]},{"entry":["GEO","100","0","0","0","80","20","320"]},{"entry":["LS1034","100","0","80","0","0","20","160"]},{"entry":["SKCO1","100","0","0","0","10","90","390"]},{"entry":["SW1116","60","40","10","10","0","40","190"]},{"entry":"T84"},{"entry":"SW948"},{"entry":"LS174T"},{"entry":{"@attributes":{"namest":"1","nameend":"8","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0403","num":"0403"},"tables":{"@attributes":{"id":"TABLE-US-00006","num":"00006"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 5"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"List of pathways enriched in either OSI-906 sensitive"},{"entry":"or resistant cell lines pre and post exposure."},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["UP IN PRE SEN (UP IN","UP IN PRE RES (DOWN IN SEN"]},{"entry":["SEN BEFORE EXPOSURE)","PRE EXPOSURE"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["Aminoacyl-tRNA biosynthesis","Leukocyte transendothelial"]},{"entry":["Ribosome","migration"]},{"entry":["Glycosphingolipid biosynthesis - ","Epithelial cell signaling in"]},{"entry":["ganglioseries",{"i":"Hellcobacter pylori "}]},{"entry":["Basal cell carcinoma","Pathogenic "]},{"entry":["Methionine metabolism","infection - EPEC"]},{"entry":["Heparan sulfate biosynthesis","Cell adhesion molecules (CAMs)"]},{"entry":["RNA polymerase","Pathogenic "]},{"entry":["Selenoamino acid metabolism","infection - EHEC"]},{"entry":["Cysteine metabolism","Tight junction"]},{"entry":["Purine metabolism","Regulation of actin cytoskeleton"]},{"entry":["Hedgehog signaling pathway","Focal adhesion"]},{"entry":["Wnt signaling pathway","MAPK signaling pathway"]},{"entry":["Glycine, serine and threonine","Cholera - Infection"]},{"entry":["metabolism","Glutathione metabolism"]},{"entry":["Naphthalene and anthracene","Sphingolipid metabolism"]},{"entry":["degradation","Type I diabetes mellitus"]},{"entry":["Porphyrin and chlorophyll","Toll-like receptor signaling"]},{"entry":["metabolism","pathway"]},{"entry":["p53 signaling pathway","Pentose phosphate pathway"]},{"entry":["Keratan sulfate biosynthesis","Metabolism of xenobiotics by"]},{"entry":["Androgen and estrogen","cytochrome P450"]},{"entry":["metabolism","ErbB signaling pathway"]},{"entry":["Glycosylphosphatidylinositol(GPI)-","Phosphatidylinositol signaling"]},{"entry":["anchor biosynthesis","system"]},{"entry":["Renin-angiotensin system","VEGF signaling pathway"]},{"entry":[{},"Glycerophospholipid metabolism"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":[{},"UP IN PRE RES (DOWN IN SEN"]},{"entry":["UP IN POST SEN","PRE EXPOSURE"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["gamma-Hexachlorocyclohexane","Leukocyte transendothelial"]},{"entry":["degradation","migration"]},{"entry":["Linoleic acid metabolism","Epithelial cell signaling in"]},{"entry":["Thyroid cancer",{"i":"Hellcobacter pylori "}]},{"entry":["Glycosphingolipid biosynthesis - ","Pathogenic "]},{"entry":["neo-lactoseries","infection - EPEC"]},{"entry":["Keratan sulfate biosynthesis","Cell adhesion molecules (CAMs)"]},{"entry":["Glycan structures - biosynthesis 2","Pathogenic "]},{"entry":["Androgen and estrogen metabolism","infection - EHEC"]},{"entry":["Maturity onset diabetes of the","Tight junction"]},{"entry":["young","Regulation of actin cytoskeleton"]},{"entry":["Glycosphingolipid biosynthesis - ","Focal adhesion"]},{"entry":["lactoseries","MAPK signaling pathway"]},{"entry":["Bisphenol A degradation","Cholera - infection"]},{"entry":["Sphingolipid metabolism","Glutathione metabolism"]},{"entry":["Axon guidance","Sphingolipid metabolism"]},{"entry":["O-Glycan biosynthesis","Type I diabetes mellitus"]},{"entry":["Glycan structures - biosynthesis 1","Toll-like receptor signaling"]},{"entry":["Notch signaling pathway","pathway"]},{"entry":["ABC transporters - General","Pentose phosphate pathway"]},{"entry":["Leukocyte transendothelial","Metabolism of xenoblotics by"]},{"entry":["migration","cytochrome P450"]},{"entry":["N-Glycan biosynthesis","ErbB signaling pathway"]},{"entry":["Glycan structures - degradation","Phosphatidylinositol signaling"]},{"entry":["Pancreatic cancer","system"]},{"entry":[{},"VEGF signaling pathway"]},{"entry":[{},"Glycerophospholipid metabolism"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"EGF, epidermal growth factor; EMT, epithelial to mesenchymal transition; NSCLC, non-small cell lung carcinoma; HNSCC, head and neck squamous cell carcinoma; CRC, colorectal cancer; MBC, metastatic breast cancer; EGFR, epidermal growth factor receptor; ErbB3, \u201cv-erb-b2 erythroblastic leukemia viral oncogene homolog 3\u201d, also known as HER-3; pHER3, phosphorylated HER3; Erk kinase, Extracellular signal-regulated protein kinase, also known as mitogen-activated protein kinase; pErk, phosphorylated Erk; Brk, Breast tumor kinase (also known as protein tyrosine kinase 6 (PTK6)); LC, liquid chromatography; MS, mass spectrometry; IGF-1, insulin-like growth factor-1; IGF-1R or IGFR, insulin-like growth factor-1 receptor; TGF\u03b1, transforming growth factor alpha; HB-EGF, heparin-binding epidermal growth factor; LPA, lysophosphatidic acid; TGF\u03b1, transforming growth factor alpha; IC, half maximal inhibitory concentration; RT, room temperature; pY, phosphotyrosine; pPROTEIN, phospho-PROTEIN, \u201cPROTEIN\u201d can be any protein that can be phosphorylated, e.g. EGFR, ERK, HER3, S6 etc; wt, wild-type; PI3K, phosphatidyl inositol-3 kinase; GAPDH, Glyceraldehyde 3-phosphate dehydrogenase, PMID, PubMed Unique Identifier; NCBI, National Center for Biotechnology Information.","All patents, published patent applications and other references disclosed herein are hereby expressly incorporated herein by reference.","Those skilled in the art will recognize, or be able to ascertain, using no more than routine experimentation, many equivalents to specific embodiments of the invention described specifically herein. Such equivalents are intended to be encompassed in the scope of the following claims."],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE FIGURES","p":[{"@attributes":{"id":"p-0014","num":"0013"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0015","num":"0014"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0016","num":"0015"},"figref":["FIG. 3","FIGS. 14 and 15"]},{"@attributes":{"id":"p-0017","num":"0016"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0018","num":"0017"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0019","num":"0018"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0020","num":"0019"},"figref":"FIG. 7"},{"@attributes":{"id":"p-0021","num":"0020"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0022","num":"0021"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0023","num":"0022"},"figref":"FIG. 10"},{"@attributes":{"id":"p-0024","num":"0023"},"figref":"FIG. 11"},{"@attributes":{"id":"p-0025","num":"0024"},"figref":"FIG. 12"},{"@attributes":{"id":"p-0026","num":"0025"},"figref":"FIG. 13"},{"@attributes":{"id":"p-0027","num":"0026"},"figref":"FIG. 14"},{"@attributes":{"id":"p-0028","num":"0027"},"figref":"FIG. 15"},{"@attributes":{"id":"p-0029","num":"0028"},"figref":"FIG. 16"},{"@attributes":{"id":"p-0030","num":"0029"},"figref":"FIG. 17"},{"@attributes":{"id":"p-0031","num":"0030"},"figref":"FIG. 18"},{"@attributes":{"id":"p-0032","num":"0031"},"figref":"FIG. 19"},{"@attributes":{"id":"p-0033","num":"0032"},"figref":"FIG. 20"},{"@attributes":{"id":"p-0034","num":"0033"},"figref":"FIG. 21"},{"@attributes":{"id":"p-0035","num":"0034"},"figref":"FIG. 22"},{"@attributes":{"id":"p-0036","num":"0035"},"figref":"FIG. 23"},{"@attributes":{"id":"p-0037","num":"0036"},"figref":"FIG. 24"},{"@attributes":{"id":"p-0038","num":"0037"},"figref":"FIG. 25"},{"@attributes":{"id":"p-0039","num":"0038"},"figref":"FIG. 26"}]},"DETDESC":[{},{}]}
