---
title: Methods and kits for quantitative methyltransferase and demethylase measurements
abstract: The invention provides methods and kits for characterizing the activity of a methyltransferase or demethylase. The method involves enzymatically methylating or demethylating in vitro a substrate that is a peptide fragment of a full-length polypeptide, and then non-enzymatically methylating the peptide substrate with methyl groups that differ in molecular weight from the enzymatically added or removed methyl groups. Typically, deuterated or C formaldehyde is used to non-enzymatically methylate the substrate. The fully methylated substrate is then characterized by mass spectrometry to determine the ratio of enzymatically produced nonmethyl, monomethyl, and dimethyl residues on the peptide.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08367366&OS=08367366&RS=08367366
owner: The Board of Trustees of the University of Arkansas
number: 08367366
owner_city: Little Rock
owner_country: US
publication_date: 20101204
---

{"@attributes":{"id":"description"},"GOVINT":[{},{}],"heading":["GOVERNMENT SUPPORT","BACKGROUND","SUMMARY","DETAILED DESCRIPTION","EXAMPLE 1","Reductive Dimethylation of Lysine Residues","EXAMPLE 2","Nonenzmymatic Trimethylation of Lysine","REFERENCES"],"p":["This invention was made with government support under grants P20RR015569, P20RR016460 and R01DA025755 awarded by the United States National Institutes of Health. The government has certain rights in the invention.","As a result of the sequencing of the human genome, it has become apparent that the complexity of an organism does not necessarily correlate with the size of its genome. This has sparked an interest in discovering exactly how genes are regulated on levels other than primary genomic sequence. Epigenetics has arisen as a field that addresses this concern by focusing on post-transcriptional methods of gene expression control including DNA methylation, histone post-translational modifications (PTMs) and non-coding RNAs (1). Modifications made to proteins post-translationally can affect their function, location or longevity. One set of highly modified proteins important to gene expression are histones. DNA is wrapped around histones, which form nucleosomes before being condensed into chromatin and ultimately chromosomes.","Histone PTMs have been shown to be very important to gene expression. Some modifications serve to signal the recruitment of chromatin modifying enzymes while some serve to alter the interaction between the histones and DNA allowing or prohibiting the access of transcription machinery (2). Some more common histone modifications include methylation, acetylation and phosphorylation. Of these common modifications, methylation is, by far, the most complex. Both lysine and arginine residues can be modified by mono- or di-methylation and lysine residues can be tri-methylated as well. These varying states of methylation can be associated with both active and inactive genes. The complexity and importance of methylation, on histones in particular, has stirred much interest in the enzymes capable of adding (methyltransferases) and removing (demethylases) these modifications.","The first demethylase was discovered in 2004 and was termed Lysine Specific Demethylase 1 or LSD1 (3). LSD1 is a flavin-dependent amine oxidase that can remove mono- and di-methyl marks from H3K4 primarily, H3K9 under certain conditions and some non-histone substrates such as p53 (4-6). It has been shown to be part of many protein complexes including CoREST, NuRD, and AR\/ER (7). LSD1 is also associated with gene repression and has been suggested to be important in initiating myc-induced transcription in cancers (3, 8-11).","Structural and biochemical studies have led to the development of numerous LSD1 inactivators that have the potential to be therapeutic tools, much like the successful deacetylase inhibitors currently in use (12). In addition, the mechanism of LSD1 indicates that it is an excellent candidate for suicide inactivators. Many monoamine oxidase (MAO) inhibitors have been suggested as potential LSD1 suicide inactivators (13). Several different assays are used to study the activity of LSD1 in the presence and absence of these various inhibitors in order to determine their efficiency.","With the importance of histone methylation and demethylation in normal gene regulation and aberrant gene regulation in cancer and other diseases, improved tools to characterize the activity and specificity of enzymes catalyzing methylation and demethylation of histones and other proteins are needed.","The invention involves methods and kit for characterizing the activity of a methylase or demethylase or enzyme mixture containing a methylase or demethylase. The methods involve enzymatically methylating and\/or demethylating a peptide substrate that is a peptide fragment of a full-length polypeptide in vitro. Typically, this step adds or removes methyl groups of natural isotope distribution. After enzymatically methylating and\/or demethylating the peptide substrate, the peptide substrate is chemically fully methylated with, e.g., deuterated formaldehyde and reductant. This results in a peptide with enzymatically added or removed methyl groups that have H isotopic hydrogen and non-enzymatically added methyl groups that differ in molecular mass because they are deuterated. Mass spectrometry is then used to determine the masses of the product peptides, from which it can be determined how many of the methyl groups on the product peptide are deuterated and how many are not, and therefore whether the methyl groups were added (or removed) enzymatically, or were added by nonenzymatic methylation with deuterated methyl groups. Importantly, the method also allows one to quantitatively determine the ratio of nonmethyl, monomethyl, and dimethyl, and optionally trimethyl, species after enzymatic methylation or demethylation.","Other methods can determine whether a peptide or a particular residue on a peptide is methylated. But they cannot quantitatively determine the ratio of methylated species (nonmethylated, monomethyl, dimethyl, and optionally trimethyl residues) on a peptide or at a particular residue of the peptide. The present method can. This is because peptide species having nonmethyl, monomethyl, and dimethyl lysine residues, are different chemically, and thus in mass spectrometry can ionize with different efficiencies. But after fully methylating each species with a methyl reagent having normatural isotopic composition, the peptides are all fully methylated and are chemically identical. Thus, they ionize with equal efficiency. But they differ in their molecular weight depending on whether the peptide were nonmethylated, monomethylated, or dimethylated before being fully methylated nonenzymatically, and therefore they can be distinguished and quantitatively compared by mass spectrometry.","Applicants have developed a quantitative assay for differentiating between polypeptides having no methyl, monomethyl, and dimethyl groups on particular lysine residues and optionally trimethyl groups on lysine residues. The method studies enzymatic methylation or demethylation of a peptide, followed by non-enzymatic methylation, followed by mass spectrometry to differentiate between various lysine methylation states. This method accounts for methyl states within a sample, allowing sample-to-sample relative comparison. We have termed our assay MassSQUIRM (Mass Spectrometric Quantitation Using Isotopic Reductive Methylation).","In reductive methylation, lysine residues are chemically di-methylated using the following reactions to alkylate and then reduce amines:\n\nR\u2014NH+HC\u2550OR\u2014N\u2550CH+HO\u2192R\u2014NH\u2014CH\u2003\u2003(1)\n\nR\u2014NH\u2014CH+HC\u2550OR\u2014N\u2014CHCHOH\u2192R\u2014N(CH)\u2003\u2003(2)\n","In the present method, we use a combination of methylation, isotopic labeling and mass spectrometry to quantitatively measure the activity of methylases or demethylases and the identity of lysine residues that are methylated or demethylated, and the number of methyl groups added or removed to each residue.","Thus, one embodiment provides a method for characterizing the activity of an enzyme or enzyme mixture comprising a methyltransferase or demethylase, the method comprising: first, (a) incubating the enzyme or enzyme mixture in vitro with a polypeptide substrate under conditions that allow methylation and\/or demethylation of the polypeptide substrate to add methyl groups to or remove methyl groups from one or more lysine residues of the polypeptide substrate to generate an enzymatically methylated and\/or demethylated polypeptide, wherein the polypeptide substrate is a peptide fragment of a full-length protein. The method, second, comprises (b) reacting the enzymatically methylated and\/or demethylated polypeptide with one or more substances to non-enzymatically react with the polypeptide to add one or more methyl groups to the polypeptide to generate a fully dimethylated or trimethylated polypeptide that is fully dimethylated or trimethylated at least one lysine residue; wherein the methyl groups non-enzymatically added to the polypeptide differ in molecular weight from the added or removed methyl groups of the enzymatically methylated and\/or demethylated protein or peptide. And the method, third, comprises (c) subjecting the fully methylated polypeptide to mass spectrometry to quantitatively determine a ratio of enzymatically produced nonmethyl, monomethyl, and dimethyl and optionally trimethyl lysine species at least one lysine residue.","Another embodiment of the invention provides a kit for characterizing the activity of a methyl transferase on a substrate, the kit comprising: (a) (i) a polypeptide substrate for a methyltransferase and\/or demethylase, wherein the polypeptide substrate is a peptide fragment of a full-length protein or (ii) a methyltransferase or demethylase; and (b) a methylation reagent for nonenzymatic methylation selected from the group consisting of formaldehyde, iodomethane, and a reductant capable of reducing a Schiff base in vitro; and optionally (c) S-adenosyl methionine; wherein the methylation reagent or the polypeptide substrate have a non-natural isotopic composition, or if neither has a non-natural isotopic composition then S-adenosyl methionine is a component of the kit and has a non-natural isotopic composition.","As an example of the method, a polypeptide substrate with a lysine residue, which is not methylated in the original substrate, is acted upon by a methylase with S-adenosyl methionine (SAM) having the normal natural isotopic composition as the methyl donor for the methylase. This results in addition of up to two methyl groups on a particular lysine residue. Some fraction of the substrate has one methyl group added, some none, and some two. Reductive methylation of the product peptide with deuterated formaldehyde donor after it is acted upon by the enzyme produces a peptide mixture where all peptides have the lysine residue dimethylated. But for the fraction that had no methyl groups after enzyme action, the methyl groups are \u201cheavy\u201d (deuterated), while for the fraction that was dimethylated by enzyme action both methyl groups are light, and for the fraction that was monomethylated by enzyme action, one methyl is light and one is heavy.","In one embodiment of the method, the mass spectrometry is MALDI mass spectrometry. MALDI does not fragment molecules; it just shows their molecular weight.","In many cases, it is known which residue on a peptide is acted upon by the enzyme, or the peptide has only one lysine or arginine residue and thus only one residue could be methylated. In these cases, MALDI provides all the information needed.","In other cases, it may be unknown which residue is methylated or two or more residues may be methylated and it may be desired to know the ratio of methylated species for each residue. In these cases, the residue methylated and the ratio of enzymatically produced nonmethyl, monomethyl, and dimethyl and optionally trimethyl species at each residue can be determined by tandem mass spectrometry. In this method, the polypeptide is fragmented in one step of the mass spectrometry, and the fragments are analyzed in a second step. Knowing the amino acid sequence of the polypeptide, from the masses of the fragments produced it is possible to identify the amino acid compositions of the fragments and which residue is methylated, as well as the ratio of enzymatically produced nonmethyl, monomethyl, and dimethyl, and optionally trimethyl species at each of the methylated residues.","Thus, one embodiment further comprises determining amino acid positions of enzymatically added or removed methyl groups on the polypeptide.","In another embodiment, the method comprises quantitatively determining a ratio of enzymatically produced nonmethyl, monomethyl, and dimethyl and optionally trimethyl lysine species at a first lysine residue identified by position and quantitatively determining a ratio of enzymatically produced nonmethyl, monomethyl, and dimethyl and optionally trimethyl lysine species at a second lysine residue identified by position.","In one embodiment of the methods, the mass spectrometry comprises MALDI mass spectrometry.","In another embodiment, the mass spectrometry comprises tandem mass spectrometry.","The substrate peptide can be methylated (prior to the tested enzymatic reaction) in some embodiments. This is necessary if enzymatic demethylation is studied. The methylation of the substrate may be at a different position than the position where enzymatic methylation and\/or demethylation is to be studied, or it may be at the position that is to be enzymatically demethylated or further methylated.","In other embodiments, the polypeptide substrate is acetylated. For instance, lysine at the 7th position of a peptide may be acetylated and the enzymatic methylation of lysine at the 4th position may be studied.","In one embodiment, the enzyme or enzyme mixture comprises lysine specific demethylase 1 (LSD1).","In particular embodiments, the enzyme or enzyme mixture comprises a methyltransferase. In other embodiments, it comprises a demethylase.","Histones are key proteins whose methylation status affect gene regulation. In one embodiment, the polypeptide substrate is a polypeptide of a histone. In one embodiment, it is a polypeptide of histone H3.","In one embodiment, the polypeptide substrate comprises H3K4 or H3K9 (i.e., lysine 4 or lysine 9 of histone H3).","In one embodiment of the method, step (b) comprises reacting the enzymatically methylated and\/or demethylated polypeptide with formaldehyde non-enzymatically to generate a formaldehyde-reacted polypeptide and reducing the formaldehyde reacted polypeptide to generate a fully dimethylated polypeptide.","In one embodiment of the method, step (a) comprises incubating the enzyme or enzyme mixture in vitro with two or more polypeptide substrates differing in their acetylation state but identical in their amino acid sequence, and the method comprises comparing activity of the enzyme or enzyme mixture on the two or more polypeptide substrates.","More generally, in one embodiment of the method, step (a) comprises incubating the enzyme or enzyme mixture in vitro with two or more polypeptide substrates differing in post-translational modifications but identical in their amino acid sequence, and the method comprises comparing activity of the enzyme or enzyme mixture on the two or more polypeptide substrates. The post-translational modifications may be, for example, acetylation or methylation. It may also be phosphorylation or glycosylation.","In particular embodiments where step (a) comprises incubating the enzyme or enzyme mixture in vitro with two or more polypeptide substrates differing in post-translational modifications but identical in their amino acid sequence, the two or more polypeptide substrates differ in methylation at one or more amino acid residues.","In particular embodiments, the polypeptide substrate comprises a synthetic ligand and the method further comprises before step (c) purifying the enzymatically acetylated and\/or deacetylated polypeptide by contacting the polypeptide with a solid substrate coupled to a receptor for the ligand. For instance, the ligand may be biotin and the peptide may be purified by capturing it with avidin or streptavidin.","In a particular embodiment, the enzyme or enzyme mixture comprises a methyltransferase, and step (a) comprises incubating the methyltransferase in vitro with S-adenosyl methionine and the polypeptide substrate to generate an enzymatically methylated polypeptide.","In a particular embodiment, the enzyme or enzyme mixture comprises a demethylase, and step (a) comprises incubating the demethylase in vitro with a partially or fully methylated polypeptide substrate to generate an enzymatically demethylated polypeptide.","In one embodiment, the enzymatically added or removed methyl groups are of natural isotopic composition and the non-enzymatically added methyl groups contain at least one deuterium. In another embodiment, the enzymatically added or removed methyl groups are of natural isotopic composition and the non-enzymatically added methyl groups contain C.","In a particular embodiment, the polypeptide substrate is enzymatically methylated and\/or demethylated on two or more amino acid residues.","One embodiment of the invention provides a kit for characterizing the activity of a methyl transferase on a substrate, the kit comprising: (a) (i) a polypeptide substrate for a methyltransferase and\/or demethylase, wherein the polypeptide substrate is a peptide fragment of a full-length protein or (ii) a methyltransferase or demethylase; and (b) a methylation reagent for nonenzymatic methylation selected from the group consisting of formaldehyde, iodomethane, and a reductant capable of reducing a Schiff base in vitro; and optionally (c) S-adenosyl methionine; wherein the methylation reagent or the polypeptide substrate have a non-natural isotopic composition, or if neither has a non-natural isotopic composition then S-adenosyl methionine is a component of the kit and has a non-natural isotopic composition.","The kit may also have both an enzyme and a polypeptide substrate, i.e., it may comprise (a) (i) a polypeptide substrate for a methyltransferase and\/or demethylase, wherein the polypeptide substrate is a peptide fragment of a full-length protein and (ii) a methyltransferase or demethylase.","In specific embodiments, the kit comprises a polypeptide substrate.","In specific embodiments, the kit comprises a methyltransferase or demethylase.","In one embodiment, the kit comprises d-formaldehyde.","In one embodiment, the polypeptide substrate comprises a synthetic ligand. In a more specific embodiment, the kit further comprises a solid substrate coupled to a receptor for the ligand.","In one embodiment, the kit contains two or more polypeptide substrates that differ in post-translational modifications but are identical in their amino acid sequence.","In a particular embodiment, the kit comprises S-adenosyl methionine.","In a particular embodiment, the polypeptide substrate is a substrate for methylation and\/or demethylation by the methyltransferase and\/or demethylase at two or more amino acid residues.","In one embodiment the kit comprises a polypeptide substrate for a methyltransferase and\/or demethylase, wherein the polypeptide substrate is a peptide fragment of a full-length protein. In one embodiment the kit comprises a methyltransferase or demethylase;","The invention will now be illustrated by the following examples.","Materials and Methods","MassSQUIRM Lysine residues were chemically dimethylated using a reductive methylation technique adapted from Rayment, et al (21). Briefly, 5 \u03bcg of synthetic histone H3 peptide (ARTKQTARKSTGGKAPRKQLC (SEQ ID NO:1)) (the superscript 1 refers to the alanine residue being the first residue of the H3 protein) was resuspended in 50 mM sodium phosphate pH 7.4 then 0.12 mg of borane dimethylamine (Sigma) and 3.2 mM isotopically light formaldehyde (Sigma) or isotopically heavy d-formaldehyde (Cambridge Isotope Laboratories) were sequentially added. This reaction was incubated for 2 hours at 4\u00b0 C. Fresh aliquots of borane dimethylamine and formaldehyde were added and the reaction was again incubated at 4\u00b0 C. for two hours. A final aliquot of borane dimethylamine was added and the reaction was incubated at 4\u00b0 C. for \u02dc16 hours. The reaction was quenched with 80 mM Tris-Cl, pH 7.5. Peptides were incubated with POROS R220 micron beads (Applied Biosystems), collected with a CZipTip\u2122 (Millipore) and spotted for MALDI analysis in 2,5-dihydroxybenzoic acid. Mass spectra of peptides were collected with a MALDI-prOTOF mass spectrometer (PerkinElmerSciex) (61, 62). Spectra were viewed and peak areas extracted using MoverZ\u2122 software (Genomic Solutions). Reaction products were verified by MSwith a Thermo LTQ XL mass spectrometer coupled to a NanoLC-2D\u2122 liquid chromatography system (Eksigent). Un-, mono- and dimethylated K27 versions of synthetic histone H3 peptide (SKAARKSAPSTGG (SEQ ID NO:2)) were used for dynamic range experiments.","Demethylase Assay. The demethylase reaction consisted of 0.25 \u03bcg H3K4me2-biotin peptide (ARTKme2QTARKSTGGKAPRKQLYKbiotin) plus 15.6, 31.3, 62.5, 125, 250 or 500 ng recombinant LSD1 (BPS Biosciences) in the following reaction buffer: 50 mM Tris-Cl pH 8.5, 50 mM KCl, 5 mM MgCl, 5% glycerol. The reaction proceeded for 2 hr at 37\u00b0 C. Reaction products were collected with POROS R2 20 micron beads for 15 minutes at room temperature, loaded into a CZipTip, eluted in 40 \u03bcL 70% acetonitrile\/0.1% TFA, lyophilized, and subjected to MassSQUIRM as described above.","LSD1 Inhibition Assay. Inhibition experiments were performed in triplicate under demethylase assay conditions as described above using 0.25 \u03bcg of H3K4me2-bio peptide alone, peptide with 125 ng GST-LSD1 (prepared in-house), and peptide with LSD1 and 16.7 mM phenylethylhydrazine inhibitor (Chem Services). As described above, peptides were isolated with POROS R220 micron beads and ZipTips prior to Mass SQUIRM analysis.","Results","Reductive Methylation is an Efficient Method for Modifying Lysine Residues. We chose to use reductive methylation to address the issue of differential ionization of methylated peptides. Ordinarily, methylation is seen in mass spectrometric data as an addition of 14 Da with mono-methylation (one open circle, ) and 28 Da with di-methylation (two open circles, ). It might seem logical to compare the peak areas of the three monoisotopic peaks seen in  in order to quantify their abundance. This would lead us to believe that the unmethylated peptide (corresponding to the solid line monoisotopic peak) is in higher abundance than the mono-methylated (dotted line) and di-methylated (dashed line) peptides. This, however, is not necessarily a correct assumption. It is possible that the unmethylated peptide is differentially ionized as compared to the others and is thus recorded by the mass spectrometer detector at a higher level. For this reason, we cannot compare these three peaks in a quantitative manner.","Reductive methylation is a reaction involving formaldehyde that results in dimethylated lysine residues. Using this technique, we can convert all three peptides in  to the same chemical species and thus cause them to ionize identically (). Unfortunately, this causes them to all be contained in a single monoisotopic peak, from which we cannot decipher the original methylation state of each individual peptide.","To solve this dilemma, we chose to use deuterated formaldehyde in our reactions. Reductive methylation performed with heavy formaldehyde results in the addition of up to two deuterated methyl groups to lysine residues and the N-terminus. Pre-existing methyl lysines will be isotopically light (open circles), while methyl groups added by reductive methylation will be isotopically heavy (closed circles) (). Peptides with dimethylated lysines will be separated by 2 or 4 Da, depending on the number of deuterated methyl groups added, but will ionize identically (). Once peptides are converted to the dimethylation state by reductive methylation, we can compare their peak areas in a quantitative manner.","To determine the efficiency of the reductive methylation reaction using both heavy and light formaldehyde, we subjected a synthetic peptide, containing four unmodified lysine residues, to both forms of the reaction. We found that the reaction is \u02dc100% efficient when using light or heavy formaldehyde (). Reductive methylation also occurs on the N-terminus of the peptide as is shown in . We observed an addition of ten methyl groups to the peptide (two for each lysine and two on the N-terminal residue) totaling 140 Da with light formaldehyde and 160 Da with heavy formaldehyde.","It is prudent to take into consideration the dynamic range capabilities of both your experiment and the instrument which will be used for analysis because a linear response is needed for accurate quantification. To determine the dynamic range of our assay, we used three different forms of a synthetic peptide: un-, mono-, and di-methylated. When the peptides were added at equal amounts, the mass spectrum showed nonequivalent levels of peptide ionization\u2014illustrating that modified peptides do not necessarily ionize equally (). To study the dynamic range, the amount of each peptide added to produce a mass spectrum with equivalent peptide ionization was empirically determined (, 1:1 spectrum). The un- and di-methylated peptides were mixed at a ratio of 1:1. The mono-methylated peptide was mixed with the others at the following ratios: 1:1, 1:2, 1:4, 1:8, 1:16 and 1:32 (). The observed ratio was determined by diving the monoisotopic areas under the mono-methyl peak by the un- and di-methylated peaks. Using this method, we determined a dynamic range of 1:8 (, squares). To check the dynamic range following reductive methylation chemistry, we incubated the same peptides in the same ratios but exposed them to MassSQUIRM analysis (). We determined the peak ratios as mentioned above with added compensation for isotopic overlap as described in the next section. We determined the dynamic range following reductive methylation to be 1:8 as well (, diamonds).","Development of a Method for Compensating for Isotopic Overlap when Using MassSQUIRM. When using mass spectrometry for relative quantification, it is most common to compare the lowest mass, or monoisotopic, peak between peptides. In addition to the monoisotopic peak, a given peptide will show a series of peaks representing naturally occurring isotopes, which are termed the isotopic envelope. When two peptides are similar in mass and are being relatively compared, the isotopic envelopes can overlap and complicate quantification. As can be seen in , one caveat with the use of deuterated formaldehyde in reductive methylation is that some isotopic overlap occurs between the different peptides. For example, the second monoisotopic peak in  (1 open and 1 closed circle) contains some overlap from the isotopic envelope of the first peptide (2 open circles). In order to compensate for this overlap, we first determined the peak area (A) ratio of Cand Cisotopes relative to the monoisotopic peak for the peptide with light dimethylation ().\n\n\u2003\u2003(3)\n\nA\u2003\u2003(4)\n","This gave us the ratio of peptide existing in these isotopic states (r& r) specific to our experiment and mass spectrometer. We then used this information to determine the following formulas for quantifying the amount of peptide existing in each modification state in a sample:\n\nH3K4me2\u2003\u2003(5)\n\nH3K4me=()\u2003\u2003(6)\n\nH3K4()\u2212[(())]\u2003\u2003(7)\n","As an example, we have shown an LSD1 treated H3K4 peptide following heavy reductive methylation (0.25 \u03bcg LSD1+0.25 \u03bcg H3K4me2 from ) that exists in multiple states of methylation (). This method was used to generate all quantitative data presented in this paper.","LSD1 Activity can be Quantitatively Measured Using MassSQUIRM. Once we had optimized the heavy reductive methylation reaction, we decided to use it to determine the activity of a demethylase. Even though LSD1 has been extensively studied, a reliable method for quantitatively studying its demethylase activity has proven elusive. For this reason, we chose to use LSD1 to test the ability of MassSQUIRM as a method for determining demethylase activity in vitro. Histone demethylase assays were performed using a synthetic H3K4me2 peptide and varying concentrations of LSD1. Samples were then subjected to MassSQUIRM (). The resulting mass spectra showed an increase in LSD1 activity as LSD1 concentration was increased from zero to 0.5 \u03bcg. A shift of peptide methylation state from completely di-methylated to 97% un-methylated was observed when using 0.5 \u03bcg LSD1 (). These results indicate that MassSQUIRM is an appropriate method for measuring the activity of LSD1 by quantitatively following methylation levels in vitro.","To emphasize the importance of our assay, we repeated the above experiment using phenylethylhydrazine, an MAO inhibitor known to inhibit LSD1 activity. We chose to use 0.125 \u03bcg LSD1 for this assay because it yielded a mixed population of modifications in our initial experiments (). Triplicate samples of peptide alone, peptide with LSD1 and peptide with LSD1 with phenylethylhydrazine were subjected to histone demethylation assays. Following these assays, samples were analyzed by MassSQUIRM. Upon addition of LSD1, we observed a shift of spectra to greater mass, indicating a mixed population of methylation states (, middle relative to bottom panel). When samples were treated with LSD1 and its inhibitor, the shift disappeared (, top panel) indicating inhibition of LSD1 activity. Samples containing peptide alone (, bottom panel) were used to determine the experimental rand rvalues used to quantify the results. Under these conditions, phenylethylhydrazine inhibits LSD1 activity by 96% ().","Discussion","The recent discovery of lysine demethylases has led to an overwhelming amount of correspondences in a very short amount of time (22-38).","Currently, demethylase activity can only be measured semi-quantitatively by methods such as autoradiography, formaldehyde release and western blotting. Mass spectrometry, as shown here, can provide an excellent method for attaining quantitative measurements of post-translational modifications. Because lysine methylation can occur in three different states, it is possible to have four different populations (un-, mono-, di- and tri-) in a single sample. Deciphering, quantifying and comparing these different methyl states can be challenging. An assay with the ability to follow each of these different methyl states quantitatively within a population does not currently exist. Most currently available mass spectrometric methods used to quantify changes in PTM status of a peptide are complex and expensive (39). In this work, we present a straightforward and inexpensive mass spectrometry-based method for quantifying the activity of demethylases acting on mono- and di-methyl lysines.","Since the initial discovery of LSD1, more than twenty human demethylase enzymes have been identified (7). Several efforts to classify these enzymes in detail are currently underway. Many demethylases have already been implicated as potential cancer diagnostic and prognostic indicators while some have been implicated as potential targets for therapies (40). Since, most histone demethylase mechanisms involve redox chemistry, they are prime candidates for suicide inactivators (13). We have shown that phenylethylhydrazine, an MAO inhibitor, serves to inhibit LSD1 activity by 96% (). Further studies of this inactivator could lead to its use in inhibiting LSD1 in some cancers. Our assay can be used to further classify the effect of this drug on different diseases involving LSD1, possibly leading to rapid turnaround for therapeutic use.","We chose to test our assay on LSD1, specifically, due to the existence of a wealth of literature suggesting that it is important in many diseases and the availability of a number of LSD1 inhibitors (3, 12, 41-45). Overexpression of LSD1 in prostate cancer, poorly differentiated neuroblastoma and estrogen receptor (ER)-negative breast cancer has been associated with aggressive forms of these diseases (5, 40, 46, 47). There have also been some promising results indicating that LSD1 inhibitors, in combination with other drugs, lead to slower growth of colon cancer in mouse xenograft models (48). All signs point to LSD1 being an excellent target for drug development.","LSD1 exists as a component of several complexes, many of which contain histone deacetylase enzymes (28, 49-51). Deacetylase inhibitors have been used successfully in clinical trials and could be used in combination with demethylase inhibitors to provide potent treatment for some diseases (52). Interestingly, when LSD1 interacts with androgen receptor, its specificity changes from H3K4 to H3K9 although this interaction has not been verified in vitro (5). Current assays exist that allow the study of interactions between LSD1 and its associating proteins, yet these assays merely represent a qualitative measure of the effect of these interactions on LSD1 activity. When studying protein complexes, it is common to combine interacting proteins in vitro and perform enzymatic assays to try and reconstruct the optimal complex. These results are used to design inhibitors that may affect a component of a complex that is not necessarily the ultimate target but that would have the same biological effect (53). For this reason, our assay will be useful in designing new drugs to target diseases that involve LSD1 and its various complexes.","Although LSD1 is the most commonly studied demethylase enzyme to date, there are several other demethylases that have not been as well characterized. Some have already been shown to be important factors in human disease (40). Members of the JMJD1 family of lysine demethylases remove mono- and di-methyl marks from H3K9 resulting in removal of a repressive mark (54). Thus, these enzymes are most likely associated with activating genes but limited analysis of them has been performed. More extensive studies of these enzymes will be undertaken before more definite conclusions can be made about their role in human disease.","We present MassSQUIRM as an inexpensive and quantitative method for comprehensive study of the activity of demethylases involved in mono- and di-methylation. MassSQUIRM offers quantitation not only of the product of the reactions of these enzymes but also their intermediates. This assay will be a powerful tool in studying the mechanism of LSD1 and possibly its interacting partners. It will also serve as a useful tool in classifying many newly discovered lysine demethylase enzymes such as PHF8 and could be used for certain methyltransferase enzymes (25, 27, 55-60). MassSQUIRM is the first assay of its kind to offer a quantitative method for studying LSD1 activity, thus its impact on the field has the potential to be quite extensive.","Example 1 shows nonenzymatic methylation of lysine residues to dimethyl lysine. Some lysine residues are biologically trimethylated. In order to quantitatively compare trimethyl lysines to monomethyl, dimethyl, and unmethylated lysines, it is necessary to nonenzymatically methylate lysines all the way to trimethyl lysine. The formaldehyde method does not do this. A stronger methylation method is needed. In this example, iodomethane is used to nonenzymatically trimethylate lysine residues. As is described above, with this nonenzymatic methylation reaction, one can quantitatively compare the extent of enzymatic methylation or demethylation by enzymatically methylating or demethylating a peptide, and then nonenzymatically methylating the peptide to trimethyl lysines, where the nonenzymatically added methyl groups differ in molecular weight (isotopic composition) from the enzymatically added or removed methyl groups.","The lysine trimethylation reaction was modified from a method described by Stewart et al (63). Briefly, 50 \u03bcg of histone H3 peptide (Abcam), amino acids 23-34, was lyophilized in 4\u00d740 mm glass reaction vessels (made in house from glass tubing). This vessel was placed in a 10 mm\u00d77 inch glass NMR tube (Fisher Scientific) and a narrow constriction was formed near the top of the tube with a Bunsen burner. The lower portion of this tube was submerged in liquid nitrogen and the tube was evacuated with a vacuum pump. Nitrogen gas was introduced to the NMR tube and was evacuated. This process was repeated three times to remove any water vapor or oxygen in the NMR tube. Next, 100 uL of iodomethane (Sigma) was added to lyophilized peptide while streaming nitrogen gas across the top of the NMR tube. The tube was evacuated again and sealed under vacuum at the constriction point. The NMR tube containing reaction vessels were placed in a dry block heater and incubated for 15 hours at 75\u00b0 C. The next day the NMR tube was broken and the reaction vessel was removed. Peptides were dissolved in 150 \u03bcL of 2% triethylamine and transferred to a 1.5 mL microcentrifuge tube. This tube was placed in a dry block heater for 1 hour at 90\u00b0 C. then lyophilized. Peptides are analyzed by mass spectrometry to quantify un-, mono-, di- and trimethylation levels on lysines.",{"@attributes":{"id":"p-0076","num":"0000"},"ul":{"@attributes":{"id":"ul0001","list-style":"none"},"li":["1. Goldberg, A. D., Allis, C. D., and Bernstein, E. (2007) Epigenetics: a landscape takes shape, 128, 635-638.","2. Kouzarides, T. (2007) Chromatin modifications and their function, 128, 693-705.","3. Shi, Y., Lan, F., Matson, C., Mulligan, P., Whetstine, J. R., Cole, P. A., and Casero, R. A. (2004) Histone demethylation mediated by the nuclear amine oxidase homolog LSD1, 119, 941-953.","4. Formeris, F., Binda, C., Vanoni, M. A., Battaglioli, E., and Mattevi, A. (2005) Human histone demethylase LSD1 reads the histone code, 280, 41360-41365.","5. Metzger, E., Wissmann, M., Yin, N., Muller, J. M., Schneider, R., Peters, A. H., Gunther, T., Buettner, R., and Schule, R. (2005) LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription, 437, 436-439.","6. Nicholson, T. B., and Chen, T. (2009) LSD1 demethylates histone and non-histone proteins, 4, 129-132.","7. Mosammaparast, N., and Shi, Y. Reversal of histone methylation: biochemical and molecular mechanisms of histone demethylases, 79, 155-179.","8. Amente, S., Bertoni, A., Morano, A., Lania, L., Avvedimento, E. V., and Majello, B. LSD1-mediated demethylation of histone H3 lysine 4 triggers Myc-induced transcription, 29, 3691-3702.","9. Lakowski, B., Roelens, I., and Jacob, S. (2006) CoREST-like complexes regulate chromatin modification and neuronal gene expression, 29, 227-239.","10. Shi, Y. J., Matson, C., Lan, F., Iwase, S., Baba, T., and Shi, Y. (2005) Regulation of LSD1 histone demethylase activity by its associated factors, 19, 857-864.","11. Lim, S., Metzger, E., Schule, R., Kirfel, J., and Buettner, R. Epigenetic regulation of cancer growth by histone demethylases, ","12. Culhane, J. C., and Cole, P. A. (2007) LSD1 and the chemistry of histone demethylation, 11, 561-568.","13. Cole, P. A. (2008) Chemical probes for histone-modifying enzymes, 4, 590-597.","14. Chellappan, S. P. (2009) 2nd ed., Humana Press, New York, N.Y.","15. Yamane, K., Tateishi, K., Klose, R. J., Fang, J., Fabrizio, L. A., Erdjument-Bromage, H., Taylor-Papadimitriou, J., Tempst, P., and Zhang, Y. (2007) PLU-1 is an H3K4 demethylase involved in transcriptional repression and breast cancer cell proliferation, 25, 801-812.","16. Bordeaux, J., Welsh, A., Agarwal, S., Killiam, E., Baquero, M., Hanna, J., Anagnostou, V., and Rimm, D. Antibody validation, 48, 197-209.","17. Connor, C., Cheung, I., Simon, A., Jakovcevski, M., Weng, Z., and Akbarian, S. A simple method for improving the specificity of anti-methyl histone antibodies, 5.","18. Cloos, P. A., Christensen, J., Agger, K., Maiolica, A., Rappsilber, J., Antal, T., Hansen, K. H., and Helin, K. (2006) The putative oncogene GASC1 demethylates tri- and dimethylated lysine 9 on histone H3, 442, 307-311.","19. Smith, C. M., Gafken, P. R., Zhang, Z., Gottschling, D. E., Smith, J. B., and Smith, D. L. (2003) Mass spectrometric quantification of acetylation at specific lysines within the amino-terminal tail of histone H4, 316, 23-33.","20. Tackett, A. J., Dilworth, D. J., Davey, M. J., O'Donnell, M., Aitchison, J. D., Rout, M. P., and Chait, B. T. (2005) Proteomic and genomic characterization of chromatin complexes at a boundary, 169, 35-47.","21. Rayment, I., Rypniewski, W. R., Schmidt-Base, K., Smith, R., Tomchick, D. R., Benning, M. M., Winkelmann, D. A., Wesenberg, G., and Holden, H. M. (1993) Three-dimensional structure of myosin subfragment-1: a molecular motor, 261, 50-58.","22. Ciccone, D. N., Su, H., Hevi, S., Gay, F., Lei, H., Bajko, J., Xu, G., Li, E., and Chen, T. (2009) KDM1B is a histone H3K4 demethylase required to establish maternal genomic imprints, 461, 415-418.","23. Greer, E. L., Maures, T. J., Hauswirth, A. G., Green, E. M., Leeman, D. S., Maro, G. S., Han, S., Banko, M. R., Gozani, O., and Brunet, A. Members of the H3K4 trimethylation complex regulate lifespan in a germline-dependent manner in ","24. Katz, D. J., Edwards, T. M., Reinke, V., and Kelly, W. G. (2009) A LSD1 demethylase contributes to germline immortality by reprogramming epigenetic memory, 137, 308-320.","25. Liu, W., Tanasa, B., Tyurina, O. V., Zhou, T. Y., Gassmann, R., Liu, W. T., Ohgi, K. A., Benner, C., Garcia-Bassets, I., Aggarwal, A. K., Desai, A., Dorrestein, P. C., Glass, C. K., and Rosenfeld, M. G. PHF8 mediates histone H4 lysine 20 demethylation events involved in cell cycle progression, ","26. Metzger, E., Imhof, A., Patel, D., Kahl, P., Hoffmeyer, K., Friedrichs, N., Muller, J. M., Greschik, H., Kirfel, J., Ji, S., Kunowska, N., Beisenherz-Huss, C., Gunther, T., Buettner, R., and Schule, R. Phosphorylation of histone H3T6 by PKCbeta(I) controls demethylation at histone H3K4, 464, 792-796.","27. Qi, H. H., Sarkissian, M., Hu, G. Q., Wang, Z., Bhattacharjee, A., Gordon, D. B., Gonzales, M., Lan, F., Ongusaha, P. P., Huarte, M., Yaghi, N. K., Lim, H., Garcia, B. A., Brizuela, L., Zhao, K., Roberts, T. M., and Shi, Y. Histone H4K20\/H3K9 demethylase PHF8 regulates zebrafish brain and craniofacial development, 466, 503-507.","28. Wang, Y., Zhang, H., Chen, Y., Sun, Y., Yang, F., Yu, W., Liang, J., Sun, L., Yang, X., Shi, L., Li, R., Li, Y., Zhang, Y., Li, Q., Yi, X., and Shang, Y. (2009) LSD1 is a subunit of the NuRD complex and targets the metastasis programs in breast cancer, 138, 660-672.","29. Lan, F., Bayliss, P. E., Rinn, J. L., Whetstine, J. R., Wang, J. K., Chen, S., Iwase, S., Alpatov, R., Issaeva, I., Canaani, E., Roberts, T. M., Chang, H. Y., and Shi, Y. (2007) A histone H3 lysine 27 demethylase regulates animal posterior development, 449, 689-694.","30. Li, F., Huarte, M., Zaratiegui, M., Vaughn, M. W., Shi, Y., Martienssen, R., and Cande, W. Z. (2008) Lid2 is required for coordinating H3K4 and H3K9 methylation of heterochromatin and euchromatin, 135, 272-283.","31. Okada, Y., Scott, G., Ray, M. K., Mishina, Y., and Zhang, Y. (2007) Histone demethylase JHDM2A is critical for Tnp1 and Prm1 transcription and spermatogenesis, 450, 119-123.","32. Perillo, B., Ombra, M. N., Bertoni, A., Cuozzo, C., Sacchetti, S., Sasso, A., Chiariotti, L., Malorni, A., Abbondanza, C., and Avvedimento, E. V. (2008) DNA oxidation as triggered by H3K9me2 demethylation drives estrogen-induced gene expression, 319, 202-206.","33. Tateishi, K., Okada, Y., Kallin, E. M., and Zhang, Y. (2009) Role of Jhdm2a in regulating metabolic gene expression and obesity resistance, 458, 757-761.","34. De Santa, F., Totaro, M. G., Prosperini, E., Notarbartolo, S., Testa, G., and Natoli, G. (2007) The histone H3 lysine-27 demethylase Jmjd3 links inflammation to inhibition of polycomb-mediated gene silencing, 130, 1083-1094.","35. Huang, J., Sengupta, R., Espejo, A. B., Lee, M. G., Dorsey, J. A., Richter, M., Opravil, S., Shiekhattar, R., Bedford, M. T., Jenuwein, T., and Berger, S. L. (2007) p53 is regulated by the lysine demethylase LSD1, 449, 105-108.","36. Lan, F., Collins, R. E., De Cegli, R., Alpatov, R., Horton, J. R., Shi, X., Gozani, O., Cheng, X., and Shi, Y. (2007) Recognition of unmethylated histone H3 lysine 4 links BHC80 to LSD1-mediated gene repression, 448, 718-722.","37. Lee, M. G., VIIIa, R., Trojer, P., Norman, J., Yan, K. P., Reinberg, D., Di Croce, L., and Shiekhattar, R. (2007) Demethylation of H3K27 regulates polycomb recruitment and H2A ubiquitination, 318, 447-450.","38. Tahiliani, M., Mei, P., Fang, R., Leonor, T., Rutenberg, M., Shimizu, F., Li, J., Rao, A., and Shi, Y. (2007) The histone H3K4 demethylase SMCX links REST target genes to X-linked mental retardation, 447, 601-605.","39. Schulze, W. X., and Usadel, B. Quantitation in mass-spectrometry-based proteomics, 61, 491-516.","40. Lim, S., Janzer, A., Becker, A., Zimmer, A., Schule, R., Buettner, R., and Kirfel, J. Lysine-specific demethylase 1 (LSD1) is highly expressed in ER-negative breast cancers and a biomarker predicting aggressive biology, 31, 512-520.","41. Culhane, J. C., Wang, D., Yen, P. M., and Cole, P. A. Comparative analysis of small molecules and histone substrate analogues as LSD1 lysine demethylase inhibitors, 132, 3164-3176.","42. Yang, M., Culhane, J. C., Szewczuk, L. M., Jalili, P., Ball, H. L., Machius, M., Cole, P. A., and Yu, H. (2007) Structural basis for the inhibition of the LSD1 histone demethylase by the antidepressant trans-2-phenylcyclopropylamine, 46, 8058-8065.","43. Yang, M., Culhane, J. C., Szewczuk, L. M., Gocke, C. B., Brautigam, C. A., Tomchick, D. R., Machius, M., Cole, P. A., and Yu, H. (2007) Structural basis of histone demethylation by LSD1 revealed by suicide inactivation, 14, 535-539.","44. Szewczuk, L. M., Culhane, J. C., Yang, M., Majumdar, A., Yu, H., and Cole, P. A. (2007) Mechanistic analysis of a suicide inactivator of histone demethylase LSD1, 46, 6892-6902.","45. Culhane, J. C., Szewczuk, L. M., Liu, X., Da, G., Marmorstein, R., and Cole, P. A. (2006) A mechanism-based inactivator for histone demethylase LSD1, 128, 4536-4537.","46. Kahl, P., Gullotti, L., Heukamp, L. C., Wolf, S., Friedrichs, N., Vorreuther, R., Solleder, G., Bastian, P. J., Ellinger, J., Metzger, E., Schule, R., and Buettner, R. (2006) Androgen receptor coactivators lysine-specific histone demethylase 1 and four and a half LIM domain protein 2 predict risk of prostate cancer recurrence, 66, 11341-11347.","47. Schulte, J. H., Lim, S., Schramm, A., Friedrichs, N., Koster, J., Versteeg, R., Ora, I., Pajtler, K., Klein-Hitpass, L., Kuhfittig-Kulle, S., Metzger, E., Schule, R., Eggert, A., Buettner, R., and Kirfel, J. (2009) Lysine-specific demethylase 1 is strongly expressed in poorly differentiated neuroblastoma: implications for therapy, 69, 2065-2071.","48. Huang, Y., Stewart, T. M., Wu, Y., Baylin, S. B., Marton, L. J., Perkins, B., Jones, R. J., Woster, P. M., and Casero, R. A., Jr. (2009) Novel oligoamine analogues inhibit lysine-specific demethylase 1 and induce reexpression of epigenetically silenced genes, 15, 7217-7228.","49. Wang, J., Scully, K., Zhu, X., Cai, L., Zhang, J., Prefontaine, G. G., Krones, A., Ohgi, K. A., Zhu, P., Garcia-Bassets, I., Liu, F., Taylor, H., Lozach, J., Jayes, F. L., Korach, K. S., Glass, C. K., Fu, X. D., and Rosenfeld, M. G. (2007) Opposing LSD1 complexes function in developmental gene activation and repression programmes, 446, 882-887.","50. Lee, M. G., Norman, J., Shilatifard, A., and Shiekhattar, R. (2007) Physical and functional association of a trimethyl H3K4 demethylase and Ring6a\/MBLR, a polycomb-like protein, 128, 877-887.","51. Lee, M. G., Wynder, C., Cooch, N., and Shiekhattar, R. (2005) An essential role for CoREST in nucleosomal histone 3 lysine 4 demethylation, 437, 432-435.","52. Beumer, J. H., and Tawbi, H. Role of Histone Deacetylases and Their Inhibitors in Cancer Biology and Treatment, ","53. Mayers, M., Ruderman, E. M., and Perlman, H. (2009) Intracellular signal pathways: potential for therapies, 11, 378-385.","54. Yamane, K., Toumazou, C., Tsukada, Y., Erdjument-Bromage, H., Tempst, P., Wong, J., and Zhang, Y. (2006) JHDM2A, a JmjC-containing H3K9 demethylase, facilitates transcription activation by androgen receptor, 125, 483-495.","55. Fortschegger, K., de Graaf, P., Outchkourov, N. S., van Schaik, F. M., Timmers, H. T., and Shiekhattar, R. PHF8 targets histone methylation and RNA polymerase II to activate transcription, 30, 3286-3298.","56. Loenarz, C., Ge, W., Coleman, M. L., Rose, N. R., Cooper, C. D., Klose, R. J., Ratcliffe, P. J., and Schofield, C. J. PHF8, a gene associated with cleft lip\/palate and mental retardation, encodes for an Nepsilon-dimethyl lysine demethylase, 19, 217-222.","57. Suganuma, T., and Workman, J. L. Features of the PHF8\/KIAA1718 histone demethylase, ","58. Yu, L., Wang, Y., Huang, S., Wang, J., Deng, Z., Zhang, Q., Wu, W., Zhang, X., Liu, Z., Gong, W., and Chen, Z. Structural insights into a novel histone demethylase PHF8, 20, 166-173.","59. Yue, W. W., Hozjan, V., Ge, W., Loenarz, C., Cooper, C. D., Schofield, C. J., Kavanagh, K. L., Oppermann, U., and McDonough, M. A. Crystal structure of the PHF8 Jumonji domain, an Nepsilon-methyl lysine demethylase, 584, 825-830.","60. Zhu, Z., Wang, Y., Li, X., Xu, L., Wang, X., Sun, T., Dong, X., Chen, L., Mao, H., Yu, Y., Li, J., Chen, P. A., and Chen, C. D. PHF8 is a histone H3K9me2 demethylase regulating rRNA synthesis, 20, 794-801.","61. Collom, S. L., Jamakhandi, A. P., Tackett, A. J., Radominska-Pandya, A., and Miller, G. P. (2007) CYP2E1 active site residues in substrate recognition sequence 5 identified by photoaffinity labeling and homology modeling, 459, 59-69.","62. Gradolatto, A., Rogers, R. S., Lavender, H., Taverna, S. D., Allis, C. D., Aitchison, J. D., and Tackett, A. J. (2008) Yta7 Regulates Histone Gene Expression, 179, 291-304.","63. Stewart N A, Pham V T, Choma C T, Kaplan H. (2002) Improved peptide detection with matrix-assisted laser desorption\/ionization mass spectrometry by trimethylation of amino groups, 16: 1448-1453."]}},"All patents, patent documents, and other references cited are hereby incorporated by reference."],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0015","num":"0014"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0016","num":"0015"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0017","num":"0016"},"figref":["FIG. 3","FIG. 1C"],"sub":["1 ","2 ","1","2","3"]},{"@attributes":{"id":"p-0018","num":"0017"},"figref":"FIG. 4","sub":["1 ","2 "]},{"@attributes":{"id":"p-0019","num":"0018"},"figref":"FIG. 5","sub":["1 ","2 "]}]},"DETDESC":[{},{}]}
