---
title: AAV capsid library and AAV capsid proteins
abstract: Recombinant adeno-associated viral (AAV) capsid proteins are provided. Methods for generating a library of recombinant adeno-associated viral capsid proteins are also provided.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08574583&OS=08574583&RS=08574583
owner: The Board of Trustees of the Leland Stanford Junior University
number: 08574583
owner_city: Palo Alto
owner_country: US
publication_date: 20111115
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATIONS","STATEMENT REGARDING GOVERNMENT INTEREST","REFERENCE TO SEQUENCE LISTING, TABLE OR COMPUTER PROGRAM","TECHNICAL FIELD","BACKGROUND","BRIEF SUMMARY","BRIEF DESCRIPTION OF THE SEQUENCES","DETAILED DESCRIPTION","I. Definitions","II. Chimeric AAV Capsid","III. Generation of a Library of Novel AAV Capsids","IV. Examples","Example 1","AAV Capsid Library Generation","Example 2","In Vitro Transduction with Recombinant AAV-DJ Vectors","Example 3","In Vivo Studies","Example 4","In Vivo Studies"],"p":["This application is a divisional application of currently pending U.S. application Ser. No. 12\/538,791, which is a continuation of U.S. application Ser. No. 11\/731,314 filed on Mar. 30, 2007 now issued as U.S. Pat. No. 7,588,772, which claims priority to U.S. Provisional Application Ser. No. 60\/787,371, filed on Mar. 30, 2006. Each of the aforementioned applications is incorporated herein by reference in its entirety.","This invention was made with Government support under contracts HL064274 & HL066948 awarded by the National Institutes of Health. The Government has certain rights in this invention.","A \u201cSequence Listing\u201d is submitted with this application in the form of a text file, created 10 Aug. 2009, and named \u201c586008243US01SEQLIST.txt\u201d (27,932 bytes), the contents of which are incorporated herein by reference in their entirety.","The subject matter described herein relates to libraries of recombinant adeno-associated viral (AAV) plasmids or viruses with varying capsid nucleotide sequences and to methods of generating the libraries. The subject matter also relates to nucleotide sequences isolated from the libraries and to the AAV capsid proteins encoded by these sequences. The subject matter also relates to plasmids and viruses comprising the identified sequences, which preferably provide a high transduction efficiency and a low level of neutralization by the human immune system.","Multiple recombinant gene transfer vectors based on different types of viruses have been developed and tested in clinical trials in recent years. Gene transfer vectors based on adeno-associated virus (AAV), i.e., AAV vectors, have become favored vectors because of characteristics such as an ability to transduce different types of dividing and non-dividing cells of different tissues and the ability to establish stable, long-term transgene expression. While vectors based on other viruses, such as adenoviruses and retroviruses may posses certain desirable characteristics, the use of other vectors has been associated with toxicity or some human diseases. These side effects have not been detected with gene transfer vectors based on AAV (Manno et al., 12(3):342 (2006)). Additionally, the technology to produce and purify AAV vectors without undue effort has been developed.","At least 11 AAV serotypes have been identified, cloned, sequenced, and converted into vectors, and at least 100 new AAV variants have been isolated from non-primates, primates and humans. However, the majority of preclinical data to date that involves AAV vectors has been generated with vectors that are based on the human AAV-2 serotype, which is considered the AAV prototype.","There are several disadvantages to the currently used AAV-2 vectors. For example, a number of clinically relevant cell types and tissues are not efficiently transduced with these vectors. Also, a large percentage of the human population is immune to AAV-2 due to prior exposure to wildtype AAV-2 virus. It has been estimated that up to 96% of all humans are seropositive for AAV-2, and up to 67% of the seropositive individuals carry neutralizing anti-AAV-2 antibodies which could eliminate or greatly reduce transduction by AAV-2 vectors. Moreover, AAV-2 has been reported to cause a cell mediated immune response in patients when given systemically (Manno et al., 12(3):342 (2006)).","Methods of overcoming the limitations of AAV-2 vectors have been proposed. For example, randomly mutagenizing the nucleotide sequence encoding the AAV-2 capsid by error-prone PCR has been proposed as a method of generating AAV-2 mutants that are able to escape the neutralizing antibodies that affect wildtype AAV-2. However, it is expected that it will be difficult to generate significantly improved AAV-2 variants with single random point mutations, as the naturally occurring serotypes have only about 85% homology at the most in the capsid nucleotide sequence.","Methods of using a mixture of AAV serotype constructs for AAV vectors have also been developed. The resulting chimeric vectors possess capsid proteins from different serotypes, and ideally, thus have properties of the different serotypes used. However, the ratio of the different capsid proteins is different from vector to vector and cannot be consistently controlled or reproduced (due to lack of genetic templates), which is unacceptable for clinical use and not satisfactory for experimental use.","A third approach at modifying the AAV-2 capsid are peptide insertion libraries, in which randomized oligonucleotides encoding up to 7 amino acids are incorporated into a defined location within the AAV-2 capsid. The display of these peptides on the AAV-2 capsid surface can then be exploited to re-target the particles to cells or tissues that are otherwise refractory to infection with the wildtype AAV-2 virus. However, because knowledge of the atomic capsid structure is a prerequisite for this type of AAV modification, this method is currently restricted to AAV serotype 2. Moreover, peptide insertion libraries typically cannot address the issues of AAV particle immunogenicity or transduction efficiency.","Thus, there remains a need for new AAV vectors and a method of generating new AAV vectors. In particular, there is a need for AAV based vectors that can be used efficiently with a variety of cell types and tissues and that do not react with a pre-existing anti-AAV human immunity that could neutralize or inactivate the vectors. There also remains a need for vectors that transduce different cell types in vivo and in vitro and that offer a more restricted biodistribution or a more promiscuous biodistribution, depending on what may be required. In particular, there remains a need for vectors capable of transducing a variety of cells types, such as hematopoietic stem cells or embryonic stem cells.","The foregoing examples of the related art and limitations related therewith are intended to be illustrative and not exclusive. Other limitations of the related art will become apparent to those of skill in the art upon a reading of the specification and a study of the drawings.","The following aspects and embodiments thereof described and illustrated below are meant to be exemplary and illustrative, not limiting in scope.","In one aspect, recombinant capsid proteins and methods for generating recombinant capsid proteins are provided. The capsid proteins include regions or domains that are derived from different serotypes of AAV. The AAV serotypes may be human or non-human. Recombinant AAV comprising the capsid proteins and plasmids encoding the capsid proteins are also provided.","In one aspect, a capsid protein comprises an individual amino acid or an amino acid sequence from a first AAV serotype, and from at least a second AAV serotype.","In one embodiment, the capsid protein additionally comprises a sequence of amino acid residues from a contiguous sequence of amino acids from a third AAV serotype.","In another embodiment, the sequences of amino acids in the first sequence, in the second sequence, and in the third or further sequence, are each a contiguous sequence of amino acids from the first AAV serotype, the second AAV serotype, the third and\/or further AAV serotypes. In another embodiment, the contiguous sequence of amino acids forms a conserved set of amino acid residues, the conserved set having at least about 70% sequence identity, more preferably at least about 80%, still more preferably at least about 85%, and still more preferably at least about 90% or 95% sequence identity with the AAV serotype from a contiguous sequence in its respective AAV serotype.","In one embodiment, the first AAV serotype is AAV-2 and the second AAV serotype is AAV-8 or AAV-9.","In another aspect, a capsid protein comprises an amino acid sequence comprising a first sequence of amino acid residues of a first AAV serotype, a second sequence of amino acid residues of a second AAV serotype, and a third sequence of amino acid residues of a third AAV serotype.","In one embodiment, the first AAV serotype is AAV-2, the second AAV serotype is AAV-8, and the third AAV serotype is AAV-9.","In a preferred embodiment, a capsid protein comprises an amino acid sequence having at least about 80% sequence identity to the amino acid sequence of SEQ ID NO: 1. In another embodiment, the capsid protein is encoded by a nucleotide sequence having at least about 80% sequence identity to the nucleotide sequence of SEQ ID NO: 2.","A viral particle comprising a capsid protein sequence as described above, is contemplated in another embodiment.","In another aspect, a plasmid comprising a sequence selected from the group consisting of (i) sequences having at least 80% sequence identity to SEQ ID NO:2 and (ii) SEQ ID NO: 2 is provided.","In yet another aspect, a recombinant AAV vector is provided, the vector comprising a capsid protein having an amino acid sequence selected from the group of sequences consisting of (i) sequences having at least 80% sequence identity to SEQ ID NO:1 and (ii) SEQ ID NO: 1.","In still another aspect, a method of expressing a gene of interest in a mammal is provided. The method comprises introducing a recombinant AAV vector into a mammal, the recombinant AAV vector encoding for a gene of interest which is encapsidated into a capsid protein having an amino acid sequence selected from the group of sequences consisting of (i) sequences having at least 80% sequence identity to SEQ ID NO:1 and (ii) SEQ ID NO:1.","In still another aspect, a method of generating a library of recombinant AAV plasmids is disclosed, the method comprising: isolating AAV capsid nucleotide sequences from two or more serotypes of AAV; digesting the AAV capsid nucleotide sequences into fragments; reassembling the fragments using PCR to form PCR products; and cloning the re-assembled PCR products into plasmids to generate a library of recombinant AAV plasmids.","In one embodiment, the method includes isolating AAV capsid nucleotide sequences from human AAV serotypes and non-human AAV serotypes. Exemplary serotypes include AAV-2, AAV-8, and AAV-9.","In another embodiment, the method comprises transfecting cells with the plasmids to produce a viral library, preferably an AAV viral library.","In one embodiment, the transfection includes transfecting into 293 kidney cells with a helper Adenovirus.","In another embodiment, the method additionally includes, after the transfecting, passaging the viral library in a selected cell type in the presence of a stringent condition, and selecting AAV capsids that survive the passaging. Passaging can be for several or multiple passages, for example from between 2-5 or 2-10 passages.","In one embodiment, a stringent condition comprises the presence of human immune globulin.","In another aspect, a library prepared according to the methods described above is disclosed. In one embodiment the library is comprised of plasmids of shuffled full-length capsid genes and in another embodiment the library is comprised of viral particles obtained by transfecting all or a portion of the plasmid library into a selected cell, optionally in combination with an adenoviral helper plasmid.","In addition to the exemplary aspects and embodiments described above, further aspects and embodiments will become apparent by reference to the drawings and by study of the following descriptions.","SEQ ID NO:1 is an amino acid sequence of a novel recombinant VP1 capsid protein, referred to herein as AAV-DJ.","SEQ ID NO:2 is a nucleotide sequence encoding the protein AAV-DJ.","SEQ ID NO:3 is the amino acid sequence of the capsid protein of AAV-2.","SEQ ID NO:4 is the amino acid sequence of the capsid protein of AAV-8.","SEQ ID NO:5 is the amino acid sequence of the capsid protein of AAV-9.","SEQ ID NOS:6-15 are artificial primers.","The practice of the subject matter described herein will employ, unless otherwise indicated, conventional techniques of molecular biology, microbiology, cell biology and recombinant DNA, which are within the skill of the art. See, e.g., Sambrook, Fritsch, and Maniatis, MOLECULAR CLONING: A LABORATORY MANUAL, 2nd edition (1989); CURRENT PROTOCOLS IN MOLECULAR BIOLOGY, (F. M. Ausubel et al. eds., 1987); the series METHODS IN ENZYMOLOGY (Academic Press, Inc.); PCR 2: A PRACTICAL APPROACH (M. J. McPherson, B. D. Hames and G. R. Taylor eds., 1995) and ANIMAL CELL CULTURE (R. I. Freshney. Ed., 1987).","As used in this specification and the appended claims, the singular forms \u201ca,\u201d \u201can\u201d and \u201cthe\u201d include plural references unless the content clearly dictates otherwise.","A polynucleotide is typically composed of a specific sequence of four nucleotide bases: adenine (A); cytosine (C); guanine (G): and thymine (T) (uracil (U) for thymine (T) when the polynucleotide is RNA). Thus, the term polynucleotide sequence is the alphabetical representation of a polynucleotide molecule. This alphabetical representation can be input into databases in a computer having a central processing unit and used for bioinformatics applications such as functional genomics and homology searching.","An \u201cisolated polynucleotide\u201d molecule is a nucleic acid molecule separate and discrete from the whole organism with which the molecule is found in nature; or a nucleic acid molecule devoid, in whole or part, of sequences normally associated with it in nature; or a sequence, as it exists in nature, but having heterologous sequences in association therewith.","Techniques for determining nucleic acid and amino acid \u201csequence identity\u201d also are known in the art. Typically, such techniques include determining the nucleotide sequence of the mRNA for a gene and\/or determining the amino acid sequence encoded thereby, and comparing these sequences to a second nucleotide or amino acid sequence. In general, \u201cidentity\u201d refers to an exact nucleotide-to-nucleotide or amino acid-to-amino acid correspondence of two polynucleotides or polypeptide sequences, respectively. Two or more sequences (polynucleotide or amino acid) can be compared by determining their \u201cpercent identity.\u201d The percent identity of two sequences, whether nucleic acid or amino acid sequences, is the number of exact matches between two aligned sequences divided by the length of the shorter sequences and multiplied by 100. Percent identity may also be determined, for example, by comparing sequence information using the advanced BLAST computer program, including version 2.2.9, available from the National Institutes of Health. The BLAST program is based on the alignment method of Karlin and Altschul. 87:2264-2268 (1990) and as discussed in Altschul, et al., 215:403-410 (1990); Karlin And Altschul, 90:5873-5877 (1993); and Altschul et al., 25:3389-3402 (1997). Briefly, the BLAST program defines identity as the number of identical aligned symbols (i.e., nucleotides or amino acids), divided by the total number of symbols in the shorter of the two sequences. The program may be used to determine percent identity over the entire length of the proteins being compared. Default parameters are provided to optimize searches with short query sequences in, for example, blastp with the program. The program also allows use of an SEG filter to mask-off segments of the query sequences as determined by the SEG program of Wootton and Federhen, 17:149-163 (1993). Ranges of desired degrees of sequence identity are approximately 80% to 100% and integer values therebetween. Typically, the percent identities between a disclosed sequence and a claimed sequence are at least 80%, at least 85%, at least 90%, at least 95%, or at least 98%.","Alternatively, the degree of sequence similarity between polynucleotides can be determined by hybridization of polynucleotides under conditions that form stable duplexes between homologous regions, followed by digestion with single-stranded-specific nuclease(s), and size determination of the digested fragments. Two DNA, or two polypeptide sequences are \u201csubstantially homologous\u201d to each other when the sequences exhibit at least about 80-85%, preferably 85-90%, more preferably 90-95%, and most preferably 98-100% sequence identity to the reference sequence over a defined length of the molecules, as determined using the methods above. As used herein, substantially homologous also refers to sequences showing complete identity to the specified DNA or polypeptide sequence. DNA sequences that are substantially homologous can be identified in a Southern hybridization experiment under, for example, stringent conditions, as defined for that particular system. Defining appropriate hybridization conditions is within the skill of the art. See, e.g., Sambrook et al., supra; DNA Cloning, supra; Nucleic Acid Hybridization, supra.","In one aspect, capsid proteins with regions or domains or individual amino acids that are derived from two or more different serotypes of AAV are provided. In one embodiment, described below, a capsid protein comprised of a first region that is derived from a first AAV serotype, a second region that is derived from a to second AAV serotype, and a third region that is derived from a third AAV serotype is provided. The AAV serotypes may be human AAV serotypes or non-human AAV serotypes, such as bovine, avian, and caprine AAV serotypes. In particular, non-primate mammalian AAV serotypes, such as AAV sequences from rodents (e.g., mice, rats, rabbits, and hamsters) and carnivores (e.g., dogs, cats, and raccoons), may be used. By including individual amino acids or regions from multiple AAV serotypes in one capsid protein, capsid proteins that have multiple desired properties that are separately derived from the multiple AAV serotypes may be obtained.","In one embodiment, a capsid protein, referred to herein as \u201cAAV-DJ\u201d, that has an amino acid sequence comprising a first region that is derived from a first AAV serotype (AAV-2), a second region that is derived from a second AAV serotype (AAV-8), and a third region that is derived from a third AAV serotype (AAV-9), is provided. The AAV-DJ capsid protein was identified from a library of capsid proteins, the library generated using a method described below (Example 1). It will be appreciated that the AAV-DJ protein is merely exemplary of the beneficial capsid proteins that can be obtained from a library generated according to the teachings herein, where the beneficial capsid proteins preferably have multiple desired properties that are derived from multiple AAV serotypes.","The amino acid sequence of AAV-DJ is shown in SEQ ID NO: 1, and the nucleotide sequence encoding AAV-DJ is shown in SEQ ID NO: 2.  show an alignment of the amino acid sequences of AAV-DJ and of the capsid proteins of AAV-2 (SEQ ID NO:3), AAV-5 (SEQ ID NO:4), and AAV-9 (SEQ ID NO:5). The five boxes numbered 1-5 in  represent the five known loops on the exterior of the AAV-2 capsid which are likely to be involved in capsid binding to cellular receptors and recognized by neutralizing antibodies. The alignment in  show that the N-terminus of AAV-DJ is identical to the N-terminus of the AAV-2 capsid protein and that the C-terminus of AAV-DJ is identical to the C-terminus of the AAV-8 capsid protein. The loop 1 region of AAV-DJ is identical to the loop 1 region of AAV-9. The loop 2, 3, and 5 regions of AAV-DJ are identical to the corresponding regions of AAV-8. The loop 4 region of AAV-DJ is a hybrid of the loop 4 regions of AAV-2 and AAV-8, with parts of the AAV-DJ loop 4 region being identical to parts of the loop 4 region of AAV-2, parts of the AAV-DJ loop 4 region being identical to parts of the loop 4 region of AAV-8, and parts of the loop 4 region of AAV-DJ being identical to both parts of the loop 4 region of AAV-2 and of AAV-8.","AAV-DJ has four mismatches to the two T cell epitopes in AAV-2 which have recently been identified as being involved in an anti-AAV cytotoxic T lymphocyte (CTL) response in humans. Thus, recombinant AAV vectors that include the AAV-DJ capsid protein or a derivative thereof are likely less immunogenic in humans than AAV-2 vectors that include the AAV-2 capsid.","Studies were conducted to confirm that infectious viral particles can be formed with AAV-DJ as the capsid. In a first study, the AAV-DJ nucleotide sequence was inserted into an AAV helper plasmid that also expresses the AAV-2 rep gene (Example 2). 293 kidney cells were then co-transfected with the AAV helper plasmid and an adenoviral helper plasmid, as well as a gfp-expressing vector plasmid. For comparison, two different versions of an AAV-2 helper were used (designated AAV-2 \u201cold\u201d and AAV-2 \u201cnew\u201d) which differ in the expression levels of viral proteins. Three days after the co-transfection, Western blotting (with 303.9 (Rep) and B1 (capsid protein)) of the 293 cell extracts revealed the presence of presence of Rep and capsid proteins at levels comparable to those found in cells co-transfected with plasmids expressing the AAV-2, AAV-8, or AAV-9 capsid proteins (blot not shown).","In another study, particle infectivity and ability to avoid neutralization by human immune globulin (IVIG) of AAV-DJ clone was compared to wildtypes AAV-2, AAV-8, and AAV-9. Two different versions of an AAV-2 helper were used (designated AAV-2 old and AAV-2 new) which differ in the expression levels of viral proteins. Recombinant AAVs with either the AAV-DJ, AAV-2, AAV-8, or AAV-9 capsids were produced by triple transfecting cells with a plasmid encoding gfp flanked by AAV inverted terminal repeats (ITRs), a plasmid encoding adenoviral helper genes, and a plasmid encoding the AAV-2 Rep gene and either the AAV-DJ, AAV-2, AAV-8, or AAV-9 capsid protein, and then freeze-thaw lysing the cells. Each virus-containing lysate was then neutralized using a high dose (1:1 volume) of two different batches of human immune globulin (IVIG1 and IVIG2) ( (293 cells);  (Huh-7 cells)), or three decreasingly lower doses (1:2 (high), 1:10 (med), and 1:25 (low) antiserum\/virus) of the two different batches of human immune globulin (IVIG1 and IVIG2), or a monoclonal A20 antibody (, 293 cells), or a polyclonal anti-AAV-8 serum (\u201cA8\u201d). A20 is a monoclonal antibody that was raised against assembled AAV-2 capsids and anti-AAV-8 is a polyclonal rabbit serum raised against assembled AAV-8 capsids. Lysates treated with PBS were used as a control. The virus-containing lysates were neutralized by incubating the lysates with the human immune globulin or antibody for a period of time (one hour at room temperature (20-25\u00b0 C.)) and then infecting cells in the presence of helper adenovirus. The remaining activity of the viruses after the neutralization period was determined by titrating gfp expression units on the cells.","The results for the 293 cells are shown in  and for the Huh-7 cells in . In the absence of IVIG1, IVIG2, and A20, the AAV-DJ virus was at least as infectious on 293 cells as AAV-2 and several fold more infectious than AAV-2 on Huh-7 cells. The data also shows that the AAV-DJ virus and AAV-8 were able to partially escape neutralization by IVIG, while AAV-2 was not. AAV-9 had intermediate IVIG results relative to AAV-DJ\/AAV-8 and AAV-2, and was neutralized at high IVIG doses. AAV-2 was neutralized by the A20 antibody, but the A20 antibody did not significantly affect AAV-DJ, AAV-8, or AAV-9. The polyclonal anti-AAV-8 antiserum neutralized all four capsids at high or medium doses, whereas AAV-2 and AAV-DJ partially escaped neutralization at the low dose.","In summary, it was found that the AAV-DJ virus was more infectious to Huh-7 cells than the previously known most efficient AAV on Huh-7 cells (AAV-2) even in the presence of high concentrations of human immune globulin. Also, the AAV-DJ virus was found to have improved resistance to neutralization by human immune globulin relative to AAV-2. Such resistance is reasonable, given that the AAV-DJ capsid was selected from a library partially based on its ability to produce virus that resist neutralization by human immune globulin. However, the improved resistance of the AAV-DJ virus to the A20 antibody was surprising and unexpected, because (i) it was not part of the selection scheme described below that was used to isolate AAV-DJ; and (ii) AAV-DJ shares substantial identity to AAV-2, which is neutralized by the A20 antibody.","In yet another study using human melanoma cell, in vitro infectivity of gfp-expressing vectors from the AAV-DJ capsid gene was compared to the in vitro infectivity of eight commonly used wildtype AAVs, AAV-1, AAV-2, AAV-3, AAV-4, AAV-5, AAV-6, AAV-8, or AAV-9. The melanoma cells were infected with 2\u00d710recombinant AAV particles of each serotype and gfp expression was visualized three days later. The quantative results, expressed as gfp expression in IU\/mL, from virus titration on the melanoma cells (in 96-well plates) are shown in . The AAV-DJ vector was superior to the wildtype vectors, and, notably, substantially better than AAV-2.","A number of cell lines were infected with ten-fold serial dilutions of each serotype, or AAV-DJ or the DJ heparin mutant DJ\/8, discussed below, expressing a gfp reporter gene. Vector preparations were normalized to contain 2\u00d710total (vector DNA-containing) particles per mL prior to infection. Three days later, gfp-expressing cells were counted and infectious titers determined, taking into account the dilution factor. As seen in Table 1, AAV-DJ vectors showed the highest infectivity on all tested cell lines, and ratios of total to infectious particles were frequently far below 500, highlighting the extreme efficiency of AAV-DJ in vitro, and suggesting its particular usefulness for ex vivo gene transfer applications.",{"@attributes":{"id":"p-0070","num":"0069"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"315pt","align":"center"}},"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"In vitro infectivity of AAV-DJ and wildtype vectors"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"238pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Ratio of Total to Infectious AAV particles(\u00d710)"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"12"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"10","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"11","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"12","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"AAV","AAV","AAV","AAV","AAV","AAV","AAV","AAV","AAV","AAV"]},{"entry":["Cell line","Tissue","1","2","3","4","5","6","8","9","DJ","DJ\/8"]},{"entry":{"@attributes":{"namest":"1","nameend":"12","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"12"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"8","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"9","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"10","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"11","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"12","colwidth":"28pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Huh-7","hu liver","4","0.5","20","2000","400","5","70","7000","0.1","300"]},{"entry":["293","hu kidney","2","0.5","20","700","400","10","70","700","0.1","200"]},{"entry":["HeLa","hu cervix","70","2","100","2000","30","200","1000","2000","0.3","1000"]},{"entry":["HepG2","hu liver","2000","50","300","20000","3000","1000","20000","nd","4","10000"]},{"entry":["Hep1A","mu liver","10","2","1000","200","2000","200","1000","20000","0.5","2000"]},{"entry":["911","hu retina","6","1","9","500","700","6000","1000","nd","0.2","400"]},{"entry":["CHO","ha ovary","10","10","70","700","3","20","100","1000","0.04","200"]},{"entry":["COS","si kidney","3","1","3","30","20","7","50","200","0.2","300"]},{"entry":["MeWo","hu skin","2","0.2","1","70","3","2","20","100","0.007","20"]},{"entry":["NIH3T3","mu fibrobl.","200","20","700","700","7000","200","7000","nd","4","20000"]},{"entry":["A549","hu lung","70","10","50","nd","2000","100","2000","7000","1","20000"]},{"entry":["HT1180","hu fibrobl.","50","10","100","7000","3000","30","2000","10000","3","5000"]},{"entry":{"@attributes":{"namest":"1","nameend":"12","align":"center","rowsep":"1"}}},{"entry":{"@attributes":{"namest":"1","nameend":"12","align":"left","id":"FOO-00001"},"sup":"1"}},{"entry":{"@attributes":{"namest":"1","nameend":"12","align":"left","id":"FOO-00002"},"sup":["2","7"]}}]}}]}}},"Vectors prepared with the AAV-DJ capsid were also tested in vivo for expression of a gene of interest. In a first study, recombinant human factor IX (FIX)-expressing AAVs with either the AAV-DJ, AAV-2, AAV-8, or AAV-9 capsids were produced by a triple transfection technique described in Example 3. Doses of 5\u00d710, 2\u00d710, and 1\u00d710(low, medium, and high, respectively) recombinant viral particles were injected peripherally into immunocompetent mice (C57\/BL6) and plasma hFIX was monitored for up to four months after injection. The FIX protein plasma levels were quantified by ELISA, and the results are shown in .","In , the shading represents 1-100% normal hFIX levels in humans (0.05 to 5 \u03bcg\/mL). FIX levels over 1% are considered therapeutic in hemophilics. As seen, the AAV-8, -9 or -DJ vectors exceeded the 100% level already at the lowest dose. A dose-dependent expression from the AAV-DJ capsid at levels equivalent to AAV-8 and -9, the best two naturally identified AAVs reported in liver thus far, was observed. All three viruses readily outperformed the AAV-2 prototype at any dose and expressed over 100% of normal hFIX levels from intravenous injection of 5\u00d710particles, whereas AAV-2 expression was over 100% of normal hFIX levels only at a dose of 1\u00d710.","In another study, recombinant human alpha-1-antitrypsin (hAAT)-expressing AAVs were prepared, from the AAV-DJ, AAV-2, AAV-8, or AAV-9 capsids. The hAAT gene was under an RSV promoter. Mice (C57\/BL6) were injected via tail vein infusions of 2\u00d710particles and plasma levels of hAAT were determined via specific ELISA 3, 7, and 14 days after injection. Results are shown in . AAV-8, AAV-9, and AAV-DJ expressed efficiently and equally outperformed the vector with an AAV-2 capsid.","In another in vivo study, liver transduction in the presence of human serum was quantified, to assess the ability of AAV-DJ to evade neutralization in vivo. As described in Example 4, mice were passively immunized with 4 or 20 mg IVIG prior to infusion of hFIX-expressing AAV-2, -8, -9, or -DJ. Plasma hFIX levels for each AAV serotype are shown in  as percent corresponding virus level in control mice treated with phosphate-buffered saline rather than IVIG as a function of time post infusion.  shows the results for mice immunized with 4 mg IVIG and  shows the results for mice immunized with 20 mg IVIG. AAV-2 expression was completely abolished, however transduction with AAV-DJ, -8 or -9 was inhibited in a dose-dependent manner, with AAV-DJ showing intermediate resistance at the high, and efficient evasion (similar to AAV-8 and AAV-9) at the low IVIG dose (). These results were confirmed with a second independent IVIG batch from another vendor (Carimune 12%, Behring AG, data not shown).","In another study, also described in Example 4, the feasibility to repeatedly administer the different viruses to mice was assessed, to evaluate capsid cross-neutralization. Results are shown in . No gene expression upon re-infusion of any of the capsids into animals already treated with the same serotype was observed. However, AAV-8 and -9 also efficiently blocked each other, substantiating previous data (Gao, G. et al., 78:6381-6388 (2004)). This result might argue against the use of vectors based on these wildtypes in re-administration protocols, albeit they could be combined with AAV-2. In contrast, primary infusion of AAV-DJ allowed subsequent expression (up to 18%) from AAV-2, -8 or -9, likely due to the fact that AAV-DJ only shares a limited number of epitopes with each wildtype virus. In the reverse experiment, AAV-DJ vectors were inhibited in animals immunized with AAV-8 or -9, while giving detectable expression in AAV-2-treated mice. This implied a stronger or broader immune response from primary infusion of serotypes 8 or 9. AAV-DJ was more resistant to the corresponding mouse sera in culture, as seen in . Less cross-reactivity between AAV-8 and -9 was noted.","AAV-DJ, as well as other recombinant protein capsids identified in the library discussed below, retained a heparin binding domain (HBD) from the AAV-2 parent. This domain functions in binding to the primary AAV-2 receptor heparin sulfate proteoglycan (Summerford, C. et al., 72:1438-1445 (1998)). To investigate the role of the AAV-DJ HBD, two crucial arginine residues (Kern, A. et al., 77:11072-11081 (2003)) were mutated to the respective residues in AAV-8 or -9, as shown in , and are referred to herein as AAV-DJ\/8 and AAV-DJ\/9. Table 1 above includes data on the mutant AAV-DJ\/8, and shows that gfp expression was reduced by several orders of magnitude, and was as low as that observed with serotypes AAV-8 or AAV-9.","The infectivity drop (Table 1) was shown to correlate with a reduced binding to cells. As seen in , a titration of infectious particles on 293 kidney cells illustrated the role of the HBD for infection in culture, as seen by the reduction in infectivity in the HBD mutants AAV-DJ\/8 and AAV-DJ\/9. Additional mutants were prepared and tested, and are identified herein as AAV-2\/8 (HBD negative), AAV-8\/2 (HBD positive), and AAV-9\/2 (HBD positive). Cell binding assays, shown in , confirmed the role of the HBD for attachment to cultured cells, where the drop in binding with the mutants correlated with the transduction data in . The HBD-positive AAV-8 and AAV-9 mutants bound several fold more efficiently than AAV-2 on HeLa cells (), but transduced less efficiently. Thus, cell attachment alone cannot explain the unusual infectivity of AAV-DJ. Instead, a synergistic effect from sharing beneficial properties from all AAV-DJ parents is contemplated, resulting in enhancement of multiple steps in AAV-DJ transduction.","The HBD also was shown to influence biodistribution, as shown in Table 2. AAV-8 and -9 (HBD-negative) demonstrated an unrestricted tropism, readily transducing all tested tissues at 1\u00d710particles per mouse. In striking contrast, AAV-2 and likewise AAV-DJ (both HBD-positive) were restricted to liver and, to a lesser extent, heart, kidney and spleen, and near or below detection limit in all other tissues. Quantification of double-stranded vector DNA (using liver as an internal standard in each group) showed that AAV-DJ transduced lung, brain, pancreas and gut about 2- to 4-fold less efficiently than wildtypes 8 or 9. The effect of the HBD on viral tropism was best exemplified by comparing AAV-DJ to the DJ\/8 mutant: HBD deletion alleviated the liver restriction and expanded transduction to all nonhepatic tissues, identical to AAV-8 and -9, and including the brain. These findings corroborate and explain a series of reports on wide tissue dissemination of vectors based on HBD-negative natural serotypes (AAV-1 and -4 to \u22129) in mice, dogs and monkeys, in contrast to the HBD-positive AAV-2. Notably, AAV-DJ also transduced nonhepatic tissues at the maximum dose of 7\u00d710particles, but still to a lesser extent than the HBD-negative viruses, in particular AAV-9. Even at this dose, brain and also lung transduction remained marginal.",{"@attributes":{"id":"p-0079","num":"0078"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"322pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Relative transduction of non-hepatic tissues with AAV vectors"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"10"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"10","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Lung","Heart","Kidney","Spleen","Brain","Pancreas","Gut","Muscle"]},{"entry":{"@attributes":{"namest":"1","nameend":"10","align":"center","rowsep":"1"}}},{"entry":["AAV-2","1e12","nd","0.7 \u00b1 0.1","0.8 \u00b1 0.1","0.2 \u00b1 0.0","nd","nd","nd","nd"]},{"entry":[{},"7e12","nd","1.5 \u00b1\u2009.03","2.0 \u00b1 0.3","1.0 \u00b1 0.2","nd","nd","nd","nd"]},{"entry":["AAV-8","1e12","0.5 \u00b1 0.0","1.2 \u00b1 0.2","0.9 \u00b1 0.2","0.3 \u00b1 0.0","0.2 \u00b1 0.0","0.2 \u00b1 0.0","0.3 \u00b1 0.0","0.7 \u00b1 0.1"]},{"entry":[{},"7e12","2.5 \u00b1 0.3","2.5 \u00b1 0.2","2.6 \u00b1 0.3","1.5 \u00b1 0.2","1.5 \u00b1 0.2","1.2 \u00b1 0.2","1.2 \u00b1 0.2","1.9 \u00b1 0.2"]},{"entry":["AAV-9","1e12","0.7 \u00b1 0.1","1.3 \u00b1 0.2","1.1 \u00b1 0.2","0.4 \u00b1 0.0","0.2 \u00b1 0.0","0.2 \u00b1 0.0","0.3 \u00b1 0.0","0.8 \u00b1 0.1"]},{"entry":[{},"7e12","2.6 \u00b1 0.3","3.6 \u00b1 0.4","3.8 \u00b1 0.4","1.5 \u00b1 0.2","1.8 \u00b1 0.2","1.3 \u00b1 0.2","1.9 \u00b1 0.2","3.0 \u00b1 0.3"]},{"entry":{}},{"entry":["AAV-DJ","1e12 7e12",{"chemistry":{"@attributes":{"id":"CHEM-US-00001","num":"00001"},"img":{"@attributes":{"id":"EMI-C00001","he":"5.93mm","wi":"9.31mm","file":"US08574583-20131105-C00001.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"1.3 \u00b1 0.2 2.3 \u00b1 0.2","0.8 \u00b1 0.2 2.1 \u00b1 0.2","0.5 \u00b1 0.1 1.5 \u00b1 0.2",{"chemistry":{"@attributes":{"id":"CHEM-US-00002","num":"00002"},"img":{"@attributes":{"id":"EMI-C00002","he":"6.01mm","wi":"9.31mm","file":"US08574583-20131105-C00002.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},{"chemistry":{"@attributes":{"id":"CHEM-US-00003","num":"00003"},"img":{"@attributes":{"id":"EMI-C00003","he":"5.93mm","wi":"9.31mm","file":"US08574583-20131105-C00003.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},{"chemistry":{"@attributes":{"id":"CHEM-US-00004","num":"00004"},"img":{"@attributes":{"id":"EMI-C00004","he":"5.93mm","wi":"9.31mm","file":"US08574583-20131105-C00004.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},{"chemistry":{"@attributes":{"id":"CHEM-US-00005","num":"00005"},"img":{"@attributes":{"id":"EMI-C00005","he":"5.93mm","wi":"9.31mm","file":"US08574583-20131105-C00005.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}]},{"entry":{}},{"entry":["AAV-DJ\/8","1e12","0.6 \u00b1 0.0","1.3 \u00b1 0.2","0.8 \u00b1 0.2","0.2 \u00b1 0.0","0.2 \u00b1 0.0","0.1 \u00b1 0.0","0.2 \u00b1 0.0","0.7 \u00b1 0.1"]},{"entry":[{},"7e12","2.6 \u00b1 0.3","2.5 \u00b1 0.3","2.3 \u00b1 0.3","1.6 \u00b1 0.3","1.8 \u00b1 0.2","1.2 \u00b1 0.2","1.3 \u00b1 0.2","2.0 \u00b1 0.2"]},{"entry":{"@attributes":{"namest":"1","nameend":"10","align":"center","rowsep":"1"}}},{"entry":"Vector copy numbers (per diploid genomic equivalent) were determined via Phosphoimager scan analyses of Southern Blots. At least three independent mice were analysed per dose. Copy numbers are shown in percent (rounded to one decimal, plus standard deviations) relative to those in liver within each group, allowing comparison between vectors and doses. For AAV-2, most signals were below the detection limit of the Southern Blot analyses (~0.03 copies of double-stranded AAV DNA per cell), preventing calculation of relative transduction in these cases (nd, not determined). Grey shadows highlight doses\/tissues where relative AAV-DJ transduction differed by at least 2-fold from serotypes 8 and 9, as well as the AAV-DJ HBD mutant."}]}}]}},"br":{}},"It will be appreciated that conservative amino acid substitutions may be to the protein of SEQ ID NO:1, to achieve proteins having, for example, 60%, 70%, 80%, 90%, or 95% sequence identity to SEQ ID NO:1, and preferably with retention of activity of the native sequence. Conservative amino acid substitutions, as known in the art and as referred to herein, involve substituting amino acids in a protein with amino acids having similar side chains in terms of, for example, structure, size and\/or chemical properties. For example, the amino acids within each of the following groups may be interchanged with other amino acids in the same group: amino acids having aliphatic side chains, including glycine, alanine, valine, leucine and isoleucine; amino acids having non-aromatic, hydroxyl-containing side chains, such as serine and threonine; amino acids having acidic side chains, such as aspartic acid and glutamic acid; amino acids having amide side chains, including glutamine and asparagine; basic amino acids, including lysine, arginine and histidine; amino acids having aromatic ring side chains, including phenylalanine, tyrosine and tryptophan; and amino acids having sulfur-containing side chains, including cysteine and methionine. Additionally, amino acids having acidic side chains, such as aspartic acid and glutamic acid, are considered interchangeable herein with amino acids having amide side chains, such as asparagine and glutamine.","In one embodiment, the recombinant AAV capsid protein is comprised of a first sequence of amino acid residues from a first AAV serotype, and at least a second sequence of amino acid residues from a second AAV serotype. The first sequence is, in the embodiment, a conserved set of amino acids from a contiguous sequence of amino acids from the first AAV serotype. The second sequence is a conserved set of amino acids from a contiguous sequence of amino acids from the second AAV serotype. A \u201cconserved set\u201d of amino acids refers to a contiguous sequence of amino acids that is identical or closely homologous to a sequence of amino acids in the AAV serotype. In one embodiment, close homology intends at least about 80% sequence identity. A contiguous sequence of amino acids in such a conserved set may be anywhere from 2 to 500, 2 to 400, 2 to 300, 2 to 200, 2 to 100, or 2 to 50 amino acid residues in length.","It will also be appreciated that the recombinant vectors described herein are contemplated for use in methods of expressing a gene of interest in a variety of cells and in a mammal. Transduction into cells lines in addition to the cell lines described herein, for example in Table 1, are exemplary, and other cells lines, particularly stem cells, are contemplated. In terms of in vivo use, the method preferably comprises introducing a recombinant AAV into the mammal, the recombinant AAV encoding the gene of interest and comprising a capsid protein having an amino acid sequence selected from the group of sequences consisting of (i) sequences having 80% sequence identity to SEQ ID NO:1 and (ii) SEQ ID NO: 1. The vector expressing a gene of interest is introduced to the mammal, typically by injection, intravenously, subcutaneously, parenterally, or the like. The gene of interest can be any gene, and many suitable genes for expression for therapeutic or non-therapeutic purposes are readily identified by a skilled artisan. The nucleotide sequence of the gene of interest is typically \u201coperably linked\u201d to one or more other nucleotide sequences, including but not limited to the gene for a selected capsid protein, a promoter, and enhancer, and the like.","A gene is \u201coperably linked\u201d to another nucleotide sequence when it is placed in a functional relationship with another nucleotide sequence. For example, if a coding sequence is operably linked to a promoter sequence, this generally means that the promoter may promote transcription of the coding sequence. Operably linked means that the DNA sequences being linked are typically contiguous and, where necessary to join two protein coding regions, contiguous and in reading frame. However, since enhancers may function when separated from the promoter by several kilobases and intronic sequences may be of variable length, some nucleotide sequences may be operably linked but not contiguous. Additionally, as defined herein, a nucleotide sequence is intended to refer to a natural or synthetic linear and sequential array of nucleotides and\/or nucleosides, and derivatives thereof. The terms \u201cencoding\u201d and \u201ccoding\u201d refer to the process by which a nucleotide sequence, through the mechanisms of transcription and translation, provides the information to a cell from which a series of amino acids can be assembled into a specific amino acid sequence to produce a polypeptide.","In another aspect, a method of generating a library of novel AAV capsids is provided. Embodiments of this aspect include a method of isolating a recombinant AAV plasmid that includes a novel AAV capsid. These embodiments will now be discussed with reference to .",{"@attributes":{"id":"p-0085","num":"0084"},"figref":["FIG. 8","FIG. 8"],"b":["402","404","406"]},"As shown in step , the full length PCR products are then PCR amplified, preferably with Platinum Pfx polymerase, using primers that bind to the signature regions that are contained in the full length PCR products because they were present in the original primers used to isolate the capsid nucleic acid sequences. The PCR products from step  are then cloned into a conventional plasmid, as shown in step  to provide a library of novel AAV capsid genes. In one embodiment, the capsid genes are cloned into an ITR-rep-containing AAV plasmid, to subsequently create the actual viral library.",{"@attributes":{"id":"p-0087","num":"0086"},"figref":["FIG. 9","FIG. 8"],"b":["502","504","506","508"]},"In a supporting study (Example 1), the approach outline in  was used to generate a library. In brief, the capsid gene from eight different AAV serotypes (AAV-2, AAV-4. AAV-5, AAV-8, AAV-9, avian AAV, bovine AAV, and caprine AAV) was fragmented, and the PCR products from step  were blunt cloned into the pCR4-TOPO plasmid, available from Invitrogen. Twenty-four (24) subclones were sequenced to confirm that capsid sequences that are a hybrid of different serotypes were created. Sequences from all eight of the serotypes were represented in the subclones. Typically, the hybrid capsid sequences included sequences from at least two, and often, more than six, of the serotypes. The capsid sequences in the pCR4-TOPO plasmid were then subcloned into a plasmid comprising the AAV-2 rep gene, as well as the two AAV-2 ITRs, that was then used to transform bacteria. It is estimated that approximately a library of 3\u00d710hybrid AAV capsid gene variants were obtained from a single reaction and from 10 plates of bacteria. Up-scaling (including plating on 100 plates of bacteria) resulted in a plasmid library of approximately 6.9\u00d710clones. This plasmid library was then co-transfected into 293 human embryonic kidney cells together with an adenoviral helper plasmid, to produce a viral library of hybrid AAV particles.","This library of AAV capsid variants was then co-infected with wildtype Adenovirus-5 helpervirus and successfully amplified in several cell lines, including human kidney 293 cells, human hepatoma Huh-7 cells, and mouse fibroblast NIH3T3 cells. Successful amplification of the viral library was confirmed by Western blots of whole cell extracts using the B1 antibody which recognizes an eight amino acid epitope that is largely conserved over most known AAV serotypes, and thus should be present in the majority of the hybrid AAVs described herein. Replicating AAV particles were detected in all of the tested cell lines for up to five consecutive passages. Whole freeze-thaw cell extracts were used for infecting fresh cells each time. To date, the viral library has also been successfully passaged six times in primary human hepatocytes, which are notoriously difficult to infect with vectors based on wiidtype AAVs.","The viral library was also amplified in human Huh-7 cells in the presence of human immune globulin (IVIG). It was found that the specific IVIG used (IVIG Gamimune\u00aeN 10% from Bayer) contained abundant neutralizing antibodies against AAV-2 and AAV-3, as well as some antibodies against AAV-1, AAV-4, AAV-5, and AAV-6. Thus, amplification in human Huh-7 cells in the presence of IVIG provided a selective pressure for AAV hybrids comprising domains from different serotypes since selecting for a high efficiency infection of Huh-7 cells favors AAV-2 domains, while selecting for escape from IVIG neutralization favors AAV-8 and AAV-9 domains. The selection was successful, as it was found that with increasing passages of the library, an increasing tolerance to IVIG was achieved. After the fourth passage, surviving virus could be amplified in the presence of 500 \u03bcL IVIG, while after the first passage, surviving virus could only be amplified in the presence of approximately 10 \u03bcA IVIG.","After the 5passage, the hybrid capsid sequences were PCR amplified and blunt cloned in pCR4-TOPO. The capsid sequences from 96 colonies were sequenced and found to be identical. The hybrid capsid sequence is the AAV-DJ sequence described above.","In summary, a plasmid library was created using DNA Family Shuffling (Crameri, et al., 391: 288-291 (1998)) of parental AAV capsid genes. Subsequently, a viral library was generated, by transfecting the plasmid library into cells together with an adenoviral helper plasmid. This second viral library was then subjected to selection pressure, to isolate specific candidates. From those, selected shuffled capsid genes were isolated and subcloned into an AAV helper plasmid, to make recombinant AAV vectors comprising the hybrid capsid. More particularly, DNA Family shuffling was used to create a complex library of hybrid particles from eight different wildtypes. Serial amplification on human cells enriched hybrids from a multitude of AAV serotypes, typically containing an AAV-2 heparin binding domain (HBD). More stringent selection with pooled human antisera yielded a single AAV-2-8-9 chimera, referred to herein as AAV-DJ. Recombinant AAV-DJ vectors were superior to natural AAVs in cultured cells and outperformed the AAV-2 prototype in tissue in vivo. Vectors with an AAV-DJ capsid were superior in vitro and gave a robust and specific in vivo performance, and provided an ability to evade humoral neutralization by human serum.","The following examples are illustrative in nature and are in no way intended to be limiting.","A. Plasmids for AAV Capsid Library Generation","Plasmids containing full-length capsid (cap) genes of seven different AAV serotypes were obtained (AAV-2, -4, -5, -8, -9, avian and bovine AAV). Goat AAV was partly synthesized (GeneArt, Regensburg, Germany) as a 888 nt fragment (nt 1023 to 1910). This subclone spans the entire right half of the goat AAV capsid protein, which comprises all 42 reported differences between goat AAV and AAV-5. The other seven cap genes were initially amplified via PCR and subcloned into pBlueScript II SK (Stratagene). The purpose was to flank all cap genes with sites for the unique restriction enzymes Pac I (5\u2032) or Asc I (3\u2032), to facilitate later cloning of \u201cshuffled\u201d cap genes into a wildtype AAV plasmid (see below). All primers also contained either a Hind III (5\u2032) or a Spe I (3\u2032) site, to allow directed cloning into pBlueScript (none of the four restriction enzymes cuts in any parental cap gene). A 20 nt signature region was inserted between the two restriction sites in each primer, to provide conserved primer binding sites for later PCR amplification of shuffled genes. Together, the sequence of the forward primers was 5\u2032 GGACTC AAGCTT  TTAATTAA CAGGT ATG 3\u2032 (SEQ ID NO:6; Hind III site in bold, Pac I site in italics\/bold, signature region underlined) directly attached at the 3\u2032 end to the first 22 nt of each cap gene following its ATG start codon. Likewise, the reverse primer was 5\u2032 CGTGAG ACTAGT  GGCGCGCC TTA 3\u2032 (SEQ ID NO:7; Spe I site in bold, Acs I site in italics\/bold, signature region underlined) directly attached at the 3\u2032 end to the last 22 nt of each cap gene up to the TAA stop codon.","In parallel, a wildtype cap recipient plasmid was engineered to contain the AAV-2 packaging elements (ITRs) flanking the AAV-2 rep gene (encoding AAV replication proteins), together with Pac I and Asc I sites for cap cloning, and the AAV-2 polyadenylation site. Therefore, AAV-2 rep (nt 191 to 2189) was PCR amplified using primers containing Bgl II sites and then subcloned into pTRUF3 (carrying AAV-2 ITRs with adjacent Bgl II sites). The forward primer used was 5\u2032 CGAACC AGATCT TCCTGTATTAGAGGTCACGTGAG 3\u2032 (SEQ ID NO:8; Bgl II site in bold, AAV-2 nt 191 underlined), and the reverse primer was 5\u2032 GGTAGC AGATCT GTTCGACCGCAGCCTTTCGAATGTCCGG  GATTA GGCGCGCC CTGGACTC  CATTTGTTCAAAGATGC 3\u2032 (SEQ ID NO:9; Bgl II site in bold, polyadenylation signal underlined, Asc I site in italics\/bold, Pac I site in italics\/bold\/underlined, AAV-2 rep stop codon in italics\/underlined). Note that this changed the AAV-2 Swa I site (downstream of rep stop codon) into a Pac I site.","B. DNA Family Shuffling of AAV Capsid Genes","For DNA shuffling of AAV capsid genes, a 2-step protocol was used where the parental genes were first fragmented using DNase I enzyme and then reassembled into a full-length gene via primer-less PCR. This was followed by a second PCR including primers binding outside of the cap genes, allowing their subcloning into the wildtype recipient ITR\/rep plasmid. Initially, all cap genes were isolated from the subclones via Hind III\/Spe I digestion (Eco RI for goat AAV) and then reaction conditions were optimized as follows. Various DNAse I concentrations and incubation times were tested, aiming to obtain a pool of fragments between 0.2 and 1.0 kb in size. Optimal conditions found were: 1 \u03bcg per cap gene. 1 \u03bcL 1:200 pre-diluted DNase I (10 U\/\u03bcL, Roche), 50 mM Tris Cl pH 7.4, 1 mM MgCl, total volume of 50 \u03bcL. The reaction was incubated for 2 min at room temperature and then stopped by heat inactivating at 80\u00b0 C. for 10 min. Fragments of the desired sizes were isolated by running the entire reaction on a 1% agarose gel (total final volume \u02dc60 \u03bcl). The re-assembly PCR reaction was then optimized by testing various DNA polymerases (Pfx Platinum, Stratagene; DeepVent, NEB; Taq, Amersham) and respective conditions. Best results were obtained using PuReTaq Ready-To-Go PCR Beads (Amersham) and the following conditions: 25 \u03bcL purified cap fragments, program: 4 min 95\u00b0 C., 40 cycles (1 min 95\u00b0 C., 1 min 50\u00b0 C., 3 min 72\u00b0 C.), 10 min 72\u00b0 C., 10 min 4\u00b0 C. Agarose gel (1%) analysis of 1 \u03bcL from this reaction typically showed a smear up to 5 kb and no distinct bands. The same three polymerases as above were then evaluated for the primer-containing second PCR, and the following conditions were found optimal: 1 \u03bcL Pfx Platinum, 2 \u03bcL product from first PCR, 1 mM MgSO4, 1 \u03bcg of each primer (see below), 0.3 mM each dNTP, total volume 50 \u03bcL, program: 5 min 94\u00b0 C., 40 cycles (30 sec 94\u00b0 C., 1 min 55\u00b0 C., 3 minutes 68\u00b0 C.), 10 min 68\u00b0 C., 10 min 4\u00b0 C. The primers used bound to the 20 nt signature regions described in the previous chapter. This reaction gave a distinct \u02dc2.2 kb full-length cap band (1% agarose gel), which was purified (60 \u03bcL total) and cloned (4 \u03bcL) using the Zero Blunt TOPO PCR cloning kit (with electro-competent TOP10 cells) (Invitrogen, Carlsbad, Calif., USA). This intermediate cloning step significantly enhanced the yield of shuffled cap genes, as compared to efforts to directly clone the PCR product via conventional means (data not shown). The shuffled cap genes were then released from the TOPO plasmid via Pac I and Asc I double digestion and cloned into the appropriately digested ITR\/rep recipient plasmid. Performing all these reactions under minimal conditions (volumes and amounts), a library of approximately 3\u00d710bacterial colonies was obtained. Up-scaling of each step (including final plating on 100\u00d715 cm plates) resulted in a final library of \u02dc6.9\u00d710plasmid clones. Its integrity, genetic diversity and functionality was confirmed by DNA sequencing and small scale expression studies. From the latter, it was determined by extrapolation that the viral library (below) retained >90% viability.","C. Selective In Vitro Amplification of the Capsid Library","A viral library was prepared by transfecting 50\u00d7 T225 flasks of 293 cells with 50 \u03bcg plasmid per flask from the bacterial library, together with 25 \u03bcg of an adenoviral helper plasmid. The resulting hybrid viruses were concentrated, purified and titrated as described for recombinant AAV. The final library had a particle titer (viral genomes) of 8.2\u00d710\/mL. Various amounts of purified shuffled AAV were then incubated with different cell lines (in 6 cm dishes), together with also varying amounts of helper Adenovirus type 5. Ideally, the Adenovirus would lyse the cells within three days, giving the AAV sufficient time to replicate. The AAV amounts were adjusted to obtain minimal signals in Western blot analyses of cell extracts. This helped to optimize the stringency of the library in each amplification round, by ensuring that a single viral genome was delivered to each cell, and subsequently packaged into the capsid expressed from its own genome.","In one set of experiments, the library was additionally subjected to IVIG pressure during amplification. Therefore, various volumes of the library and IVIG (Gamimune\u00aeN 10%, Bayer, Elkhardt, Ind., USA) were mixed and incubated for 1 hour at 37\u00b0 C., and then added to the cells. After overnight incubation, the cells were washed and super-infected with Adenovirus. The wash step was included to avoid helper virus inactivation by the IVIG. As before. AAV amplification was controlled by Western blotting after each round, and only supernatants giving minimal expression were used for subsequent infections. The increasing IVIG resistance of the library during consecutive passages allowed continuous escalation of the IVIG doses. All amplification experiments comprised five infection cycles (Adenovirus was heat inactivated between each and then added fresh, to avoid uncontrolled amplification). Finally, viral DNA was purified from the supernatant (DNA Extractor Kit, Wako, Japan), and AAV cap genes were PCR amplified (DeepVent Polymerase), using primers 5\u2032 GATCTGGTCAATGTGGATTTGGATG 3\u2032 (SEQ ID NO:10; binding in AAV-2 rep upstream of the Pac I site used for cap cloning) and 5\u2032 GACCGCAGCCTTTCGAATGTCCG 3\u2032 (SEQ ID NO:11; binding downstream of the Asc I site and polyadenylation signal). The resulting blunt-ended cap genes were subcloned using the Zero Blunt TOPO PCR cloning kit for Sequencing (Invitrogen) and DNA was prepared from individual clones (96 per cell line\/amplification round).","To assemble full-length cap sequences, T3 and T7 primers were used to obtain the 5\u2032 and 3\u2032 ends of each clone, and then individual primers (not shown) were designed to acquire the remaining sequence. Alignments (DNA and protein) with the eight parental cap genes were performed using BLAST and Vector NTI 10\/AlignX software (Invitrogen).","D. AAV Protein Analyses","Western blot and immunofluorescence analyses were carried out as reported (Grimm, D. et al., 102:2412-2419 (2003)) using the monoclonal B1 antibody for detection of immobilized AAV capsid proteins, useful because its eight amino acid epitope is largely conserved-across known AAV serotypes.","A. Helper Plasmid Cloning and Vector Particle Production","Helper plasmids expressing wildtype AAV-2, -8 or -9 cap together with AAV-2 rep genes, as well as AAV-2-based vector plasmids expressing the hFIX gene from a liver-specific or the EF1\u03b1 promoter, were previously described (Nakai, H. et al., 79:214-224 (2005)); Gao, G. et al., 78:6381-6388 (2004)). Two self-complementary vector plasmids expressing either the gfp gene from a CMV promoter, or the hAAT gene from an RSV (Rous Sarcoma Virus) promoter, were prepared using conventional techniques.","For cloning of helper plasmids expressing shuffled cap genes, the entire AAV-8 cap gene was removed from the AAV-8 helper construct by cutting with Swa I and Pme I (both create blunt ends; Swa I cuts 9 nt upstream of the VP1 start codon, Pme I cuts 53 nt downstream of the polyadenylation signal). The novel cap genes were amplified from the respective TOPO constructs (see above) via PCR, using the forward primer 5\u2032  CAGGT 3\u2032 (SEQ ID NO:12; the underlined nt restored the Swa I site to maintain correct reading frames) directly attached at the 3\u2032 end to the first 25 nt of each cap gene, which for AAV-DJ was: ATGGCTGCCGATGGTTATCTTCCAG (SEQ ID NO:13; identical in AAV-2, -8 and -9). The reverse primer was 5\u2032  AATTCGCCCTTCGCAGAGACCAAAGTTCAACTGAAACGAATCAACCGG  GATTAACAGGCAA 3\u2032 (SEQ ID NO:14; nt restoring the Pme I site are underlined, the polyadenylation signal is shown in bold) directly attached at the 3\u2032 end to the last (3\u2032) 23 nt of the shuffled capsid genes, which for AAV-DJ was: TTACAGATTACGGGTGAGGTAAC, 3\u2032-5\u2032 orientation, SEQ ID NO:15). PCRs were performed using DeepVent DNA Polymerase (NEB), creating blunt ends allowing straight-forward insertion into the linearized AAV-8 helper plasmid. Insert junctions and correct orientation were confirmed via DNA sequencing (Biotech Core). Vector production and particle titration (dot blot) were performed as previously described (Nakai, H. et al., 79:214-224 (2005)). Yields for all vectors including AAV-DJ and the HBD mutants typically exceeded 6\u00d710total physical particles per 50\u00d7 T225 flasks (2\u00d710cells).","B. In Vitro Transduction","All transformed cell lines were maintained in DMEM (Gibco) containing 10% fetal calf serum, 2 mM L-glutamine and 50 IU\/ml of each penicillin and streptomycin at 37\u00b0 C. in 5% CO. Fresh primary human hepatocytes (in 6-well plates without Matrigel) were obtained from Admet (Durham, N.C., USA) and maintained in Hepatocyte Basal Medium (Cambrex, Walkersville, Md., USA) with recommended supplements. Titration of gfp-expressing recombinant AAV particles was performed in 96-well plates, following normalization of each virus stock to 2\u00d710particles\/mL. For in vitro neutralization studies, 50 \u03bcL per vector preparation were incubated with serial 10-fold dilutions of two batches of IVIG (designated IVIG1 and IVIG2) or mouse sera (following a 1 hour heat inactivation at 56\u00b0 C.) for 1 hour at 37\u00b0 C. prior to titration on cells. Titers of neutralizing antibodies were calculated as reported (Grimm, D. et al., 102:2412-2419 (2003)).","Recombinant AAVs with either the AAV-DJ, AAV-2, AAV-8, or AAV-9 capsids were produced by triple transfecting cells with a plasmid encoding the human factor IX (FIX) gene under the control of an adjacent liver-specific promoter and flanked by AAV inverted terminal repeats (ITRs), a plasmid encoding adenoviral helper genes, and a plasmid encoding the AAV-2 rep gene and either the AAV-DJ, AAV-2, AAV-8, or AAV-9 capsid protein. The liver-specific promoter that was used was a human alpha-1-antitrypsin (hAAT) promoter fused with an apolipoprotein E (ApoE) enhancer and an HCR (hepatic locus control region). The AAV ITRs that were used were derived from AAV-2 and are identical to each other.","Doses of 5\u00d710, 2\u00d710, and 1\u00d710(low, medium, and high, respectively) recombinant viral particles were injected into mice via tail vein infusions with a volume of 300 microliters infused over a period of about 10 seconds. The mice were bled at 1, 3, and 8 days after the infusions. The FIX protein plasma levels were quantified by ELISA, and the results are shown in .","A. Expression Studies in Mice","Wildtype female C57\/BL6 mice (6 to 8 weeks old, 20 to 25 grams) were purchased from Jackson Laboratory (Bar Harbor, Me., USA). Recombinant AAV expressing the hFIX or hAAT genes were administered in 300 \u03bcL 1\u00d7PBS via tail vein infusion. Blood was collected at the indicated timepoints via retroorbital bleeding, and plasma hFIX or hAAT levels were determined via ELISA as previously described (Nakai, H. et al., 79:214-224 (2005); Grimm, D. et al., 441:537-541 (2006)). Results for the hAAT-expressing vectors are shown in .","Genomic DNA was extracted from mouse tissues and analyzed via Southern blotting, using hFIX- or hAAT-specific probes, as previously reported (Nakai, H. et al., 76:11343-11349 (2002)).","B. Immunologic In Vivo Assays","For passive immunization studies, mice (n=4 per group) were injected intravenously (tail vein) with 40 \u03bcL (low dose) or 200 \u03bcL (high dose) of IVIG (100 mg\/mL) diluted in 1\u00d7PBS to a total volume of 300 \u03bcL, and 24 hours later infused (tail vein) with 2\u00d710recombinant hFIX-expressing AAV. Plasma hFIX levels per virus and timepoint are shown in  as percent of corresponding levels in control mice (PBS instead of IVIG).","For cross-neutralization studies, mice were immunized against individual AAV serotypes by peripheral infusion of 1\u00d710recombinant hAAT-expressing particles. Three weeks later, mouse serum was collected for in vitro neutralization assays, before the mice were re-infused with 1\u00d710(5\u00d710for AAV-2) recombinant hFIX-expressing AAV. Sera was taken from the mice at the time of re-injection (H), as well as from a parallel group of mice injected with a lower dose (L) of 2\u00d710particles. Neutralizing antibody titers (NAb) against the wildtype AAVs or AAV-DJ were determined. Results are shown in .","While a number of exemplary aspects and embodiments have been discussed above, those of skill in the art will recognize certain modifications, permutations, additions and sub-combinations thereof. It is therefore intended that the following appended claims and claims hereafter introduced are interpreted to include all such modifications, permutations, additions and sub-combinations as are within their true spirit and scope."],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0035","num":"0034"},"figref":["FIG. 1A","FIG. 1B"]},{"@attributes":{"id":"p-0036","num":"0035"},"figref":["FIGS. 2A-2C","FIGS. 2A","FIG. 2B","FIGS. 2A-2B","FIG. 2C"],"b":"2"},{"@attributes":{"id":"p-0037","num":"0036"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0038","num":"0037"},"figref":["FIGS. 4A-4C","FIG. 4A","FIG. 4B","FIG. 4C"],"sup":["10 ","11 ","12 "]},{"@attributes":{"id":"p-0039","num":"0038"},"figref":"FIG. 5","sup":"11"},{"@attributes":{"id":"p-0040","num":"0039"},"figref":["FIGS. 6A-6B","FIG. 6A","FIG. 6B"]},{"@attributes":{"id":"p-0041","num":"0040"},"figref":"FIG. 6C"},{"@attributes":{"id":"p-0042","num":"0041"},"figref":["FIG. 6D","FIG. 6C"],"sup":"10 "},{"@attributes":{"id":"p-0043","num":"0042"},"figref":"FIG. 7A"},{"@attributes":{"id":"p-0044","num":"0043"},"figref":["FIG. 7B","FIG. 7A"]},{"@attributes":{"id":"p-0045","num":"0044"},"figref":["FIGS. 7C-7D","FIG. 7C","FIG. 7D","FIG. 7A"]},{"@attributes":{"id":"p-0046","num":"0045"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0047","num":"0046"},"figref":"FIG. 9"}]},"DETDESC":[{},{}]}
