---
title: Downregulation of SINE/ALU retrotransposon transcription to induce or restore proliferative capacity and/or pluripotency to a stem cell
abstract: In certain embodiments methods are provided for inducing and/or restoring and/or maintaining a non-senescent phenotype, or aspects thereof (e.g., proliferative capacity and/or pluripotency) in a mammalian cell. The methods typically involve reducing the level or activity of SINE/Alu retrotransposon transcripts in the cell in an amount sufficient to induce or restore proliferative capacity and/or pluripotency to said mammalian cell.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09617514&OS=09617514&RS=09617514
owner: BUCK INSTITUTE FOR RESEARCH ON AGING
number: 09617514
owner_city: Novato
owner_country: US
publication_date: 20111020
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATIONS","STATEMENT OF GOVERNMENTAL SUPPORT","BACKGROUND","SUMMARY","DEFINITIONS","DETAILED DESCRIPTION","EXAMPLES","Example 1","SINE\/Alu Retrotransposon-Dependent Chromatin Changes Induce Ex-Vivo Aging of Human Adult Stem Cells","Example 2","Supplemental Methods","Example 3","Protein Complex Assembled on piRNA Derived from Alu Retrotransposal Transcript Indicates Putative Participation of retroRNA in the Cell Cycle, DNA Repair and Chromatin Assembly"],"p":["This application is a 371 National Phase of PCT\/US2011\/057140, filed on Oct. 20, 2011, which claims priority to and benefit of U.S. Ser. No. 61\/406,954, filed on Oct. 26, 2010 and to U.S. Ser. No. 61\/455,808, filed on Oct. 26, 2010 both of which are incorporated herein by reference in their entirety for all purposes.","This invention was made with government support under Grant No: DE019608 awarded by the by the National Institutes of Health. The government has certain rights in this invention.","Adult stem cells are extremely important for long-term tissue homeostasis throughout life. Their self-renewing proliferative capacity involves numerous tightly coordinated processes to ensure preservation of genome integrity during cell division. The regulatory mechanisms underlying their aging are less well defined. Nonetheless, global gene expression studies of stem cells purified from young and old mice have implicated the involvement of epigenetic regulation in higher-order chromatin dynamics. These studies have suggested coordinated age-dependent regulation of chromosomal regions, chromatin remodeling activities and lineage specification genes (Chambers et al. (2007) 5: e201; Rossi et al. (2007) 42: 385-390; Rossi et al. (2005) 102: 9194-9199).","All cells are constantly challenged by exogenous and endogenous sources of DNA damage; depending on the nature of the damage, they activate different DNA damage repair mechanisms (Sinclair et al. (2004) 164: 396-414). In parallel, cells also activate checkpoint pathways, which delay cell cycle progression until genome integrity has been restored (Shiloh (2001) 11: 71-77). One aspect of the stem cell hypothesis of aging postulates that the gradual and coordinated age-related loss of DNA damage repair capacity results in DNA damage accumulation over time. This damage would pose a significant threat to adult stem cell survival by altering proliferation and differentiation patterns, ultimately triggering cellular senescence. Therefore, the ability of adult stem cells to monitor and faithfully repair DNA damage is key to the prevention of aging and neoplastic transformations.","Little is known about the precise relationship between chromatin and DNA-repair factors. More than 50% of the human genome consists of retrotransposons (Lander et al. (2001) 409: 860-921). Their epigenetic makeup is poorly understood and inadequately annotated at the genomic level, due to a high degree of sequence conservation. In fact, many retrotransposons are derived from ancestral RNA genes and might represent genetically active sequences that encode different types of RNA with yet unknown functions (McClintock (1956) 21: 197-216). However, clear evidence exists that the retrotransposal portion of the genome profoundly influences the organization, integrity, and evolution of the host's genome and transcriptome (Han et al. (2004) 429: 268-274; Kazazian (2004) 303: 1626-1632). A growing body of evidence demonstrates that, during mammalian evolution, a large number of ancient retroelements acquired regulatory or structural functions.","The majority of retrotransposons are expressed in extraordinarily complex patterns in a cell- or tissue-specific manner, and potentially provide a rich source of non-protein coding RNAs to guide the trajectories of cellular differentiation and multicellular development (Amaral et al. (2008) 319: 1787-1789; Birney et al. (2007) 447: 799-816; Denoeud et al. (2007) 17: 746-759; Dinger et al. (2008) 18: 1433-1445; Dinger et al. (2008) 40: 151-159; Emanuelsson et al. (2007) 17: 886-897; Faulkner et al. (2009) 41: 563-571; Lowe et al. (2007) 104: 8005-8010; Mattick et al., (2009) 31: 51-59; Mercer et al. (2008) 105: 716-721; Mikkelsen et al., ed. (2007) --; Rozowsky et al. (2007) 17: 732-745; Trinklein et al. (2007) 17: 720-731). Recent studies have proven that retrotransposon transcriptional activities trigger and guide the processes of (i) assembly of centromeric chromatin, (ii) gene transcription, (iii) compartmentalization of chromatin and, (iv) nuclear organization of chromatin insulation during X-chromosome inactivation. Retrotransposons also serve a distinct function in non-random chromosomal translocations in tumors (Allen et al. (2004) 11: 816-821; Chueh et al. (2005) 14: 85-93; Lei and Corces (2006) 124: 886-888; Lei and Corces (2006) 38: 936-941; Lin et al. (2009) 139: 1069-1083; Lunyak (2008) 20: 281-287; Lunyak et al. (2007) 317: 248-251; Mattick et al. (2009) 31: 51-59; Navarro et al. (2009) 2: 8).","There is also a considerable amount of tissue-specific, development-specific, and disease-related variability in DNA methylation and covalent modifications of chromatin within the retrotransposal portion of the genome (Kondo and Issa (2003) 278: 27658-27662; Estecio et al. (2007) 2: e399). A genome-wide study by Martens (Martens et al. (2005) 24: 800-812) demonstrates that LINEs, SINE\/Alus, and other interspersed retrotransposons have variable degrees of H3K9, H3K27, and H4K20 histone methylation, raising the possibility that posttranscriptional modifications (PTM) of retrotransposal chromatin can influence diverse cellular processes.","Efficient repair of DNA double-strand breaks and authentic genome maintenance at the chromatin level are fundamental to faithful human adult stem cell self-renewal. Stem cell aging can be linked to deficiencies in these two processes. In one example, we report that \u02dc65% of naturally occurring repairable damage in self-renewing adult stem cells occurs in transposable elements. Upregulation of transcriptional activity from SINE\/Alu retrotransposons interferes with the recruitment of condensin I and cohesin complexes in pericentric chromatin, resulting in the loss of efficient DNA repair and, in turn, senescence. Stable knockdown of generic SINE\/Alu transcripts in senescent human adult stem cells reinstates the cells self-renewing properties and unexpectedly increases their plasticity as manifested by upregulation of Nanog and Oct4. Our results demonstrate the functional significance of SINE\/Alu retrotransposons and provide mechanistic insight into their novel role in mediating crosstalk between chromatin, DNA repair and aging of human adult stem cells.","In certain embodiments, methods are provided for restoring a non-senescent phenotype, or aspects of a non-senescent phenotype to a senescent cell (e.g., a senescent adult stem cell). In certain embodiments, methods are provided for maintaining a non-senescent phenotype, or aspects of a non-senescent phenotype in a cell (e.g., a senescent adult stem cell). In certain embodiments methods are provided for inducing and\/or restoring and\/or maintaining a non-senescent phenotype, or aspects thereof (e.g., proliferative capacity and\/or pluripotency) in a mammalian cell. The methods typically involve reducing the level or activity of SINE\/Alu retrotransposon transcripts in the cell in an amount sufficient to induce or restore proliferative capacity and\/or pluripotency to said mammalian cell.","In certain embodiments methods of transdifferentiating a mammalian cell from a first cell type or lineage into a second cell type or lineage are provided. The method typically involves transforming a differentiated cell into a pluripotent cell (or restoring pluripotency to a stem cell or a progenitor cell) by one of the methods described herein (e.g., by reducing the level or activity of SINE\/Alu retrotransposon transcripts in the cell); culturing the cell under conditions that induce or permit differentiation of the cell; selecting cells that have differentiated into the second cell type or lineage; and culturing the cells of said second cell type or lineage. In certain embodiments the first cell type is a mesodermal cell type. In certain embodiments the first cell type is an ectodermal cell type. In certain embodiments the first cell type is an endodermal cell type. In certain embodiments the first cell type or lineage is a mesodermal cell type and the second cell type or lineage is a neuroectodermal cell type. In certain embodiments the first cell type is an ectodermal cell type and the second lineage is a mesodermal or an endodermal cell type. In certain embodiments, the first cell type is an endodermal cell type and the second lineage is an ectodermal or mesodermal cell type. In certain embodiments the first cell type is an adipocyte or a bone marrow cell. In certain embodiments the second cell type is a cell type selected from the group consisting of a blood cell, a fetal cell, an epithelial cell, an adipocyte, a smooth muscle cell, a nerve cell, a pancreatic beta cell, and a cardiomyocyte. In certain embodiments the culturing the cell under conditions that induce or permit differentiation of said cell comprises culturing said cells in a medium lacking or having a reduced quantity of leukemia inhibitory factor (LIF) and\/or contacting (or culturing) the cell with one or more reagents (e.g., retinoic acid, PDGF, insulin, Arctigenin, ATRA (vitamin A), boswellic acid, bromelain and other proteolytic enzymes, CAPE, flavonoids (including apigenin, luteolin, quercetin, genistein, and daidzein), emodin, EPA and DHA, monoterpenes, resveratrol, 1,25-D3 (vitamin D3)) that induce differentiation.","Micro-RNAs are single-stranded RNAs of typically 22-nucleotides that are processed from \u02dc70-nucleotide hairpin RNA precursors by the Rnase III nuclease, Dicer. Similar to siRNAs, miRNAs can silence gene activity through destruction of homologous mRNA in plants or blocking its translation in plants and animals.","shRNA or short hairpin RNA is an RNA molecule that contains a sense strand, antisense strand, and a short loop sequence between the sense and antisense fragments. Due to the complementarity of the sense and antisense fragments in their sequence, such RNA molecules tend to form hairpin-shaped double-stranded RNA (dsRNA). shRNA is cloned into a vector, allowing for expression by a pol III type promoter. The expressed shRNA is then exported into the cytoplasm where it is processed by dicer into siRNA which then get incorporated into the siRNA induced silencing complex (RISC).","Small Interfering RNA (siRNA) are typically 21-23 nucleotide double-stranded RNA molecules. Once incorporated into the RNA-induced silencing complex (RISC) they facilitate the cleavage and degradation of its recognized mRNA.","Piwi-interacting RNA (piRNA) is class of small non-coding RNA molecules that is expressed in, or can be introduced into animal cells (see, e.g., Seto et al. (2007) 26(5): 603-609; Siomi et al. (2011) 12:246-258). piRNAs form RNA-protein complexes through interactions with piwi proteins. These piRNA complexes have been linked to both epigenetic and post-transcriptional gene silencing of retrotransposons and other genetic elements.","By \u201cpluripotency\u201d and \u201cpluripotent stem cells\u201d it is meant that such cells have the ability to differentiate into all types of cells in an organism. Pluripotent cells are characterized by the expression of one or more pluripotency markers known by one of ordinary skill in the art. Such markers include, but are not limited to Alkaline Phosphatase, SSEA3, SSEA4, Sox2, Oct3\/4, Nanog, TRA160, TRA181, TDGF 1, Dnmt3b, FoxD3, GDF3, Cyp26a1, TERT, and zfp42. In certain embodiments pluripontent cells are capable of forming or contributing to ectoderm, mesoderm, or endoderm tissues in a living organism.","The terms \u201cprimary cells\u201d, \u201cprimary cell lines\u201d, and \u201cprimary cultures\u201d are used interchangeably herein to refer to cells and cell cultures that have been derived from a subject and allowed to grow in vitro for a limited number of passages, i.e. splittings, of the culture. For example primary cultures are cultures that may have been passaged 0 times, 1 time, 2 times, 4 times, 5 times, 10 times, or 15 times, but not enough times go through the crisis stage. Typically, the primary adipose cells of the present invention are maintained for fewer than 10 passages in vitro prior to use.","The terms \u201cindividual,\u201d \u201csubject,\u201d \u201chost,\u201d and \u201cpatient,\u201d are used interchangeably herein and refer to any mammalian subject for whom diagnosis, treatment, or therapy is desired, particularly humans.","By \u201cadipose\u201d is meant any fat tissue. The adipose tissue may be brown or white adipose tissue, derived from subcutaneous, omental\/visceral, mammary, gonadal, or other adipose tissue site. In certain embodiments the adipose is subcutaneous white adipose tissue or visceral adipose tissue or a lipoaspirate sample. The adipose tissue may be from any organism having fat tissue. Preferably, the adipose tissue is mammalian, most preferably the adipose tissue is human. A convenient source of adipose tissue is from liposuction surgery, however, the source of adipose tissue need not be so limited.","It was discovered that a significant (e.g., \u02dc65% in certain instances) of naturally occurring repairable damage in self-renewing adult stem cells occurs in transposable elements. Upregulation of transcriptional activity from SINE\/Alu retrotransposons interferes with the recruitment of condensin I and cohesin complexes in pericentric chromatin, resulting in the loss of efficient DNA repair and, in turn, senescence.","It was demonstrated that stable knockdown of generic SINE\/Alu retrotransposon transcripts in senescent human adult stem cells reinstates the cells' self-renewing properties and unexpectedly increases their plasticity (e.g., as manifested by upregulation of the pluripotency markers Nanog and Oct4). The results presented herein demonstrate the functional significance of SINE\/Alu retrotransposons and provide mechanistic insight into their novel role in mediating crosstalk between chromatin, DNA repair and aging of human adult stem cells.","In view of this discovery, methods are provided for inducing and\/or restoring and\/or maintaining proliferative capacity and\/or pluripotency in a mammalian cell. More generally, the methods reduce the rate of onset of senescence or prevent senescence, or rescuing a cell from senescence. In various embodiments the methods induce or restore a pluripotent phenotype to a differentiated and\/or senescent cell. In various embodiments the methods typically involve downregulating or inhibiting the level or activity of SINE\/Alu retrotransposon transcripts in the cell (see, e.g., ).","As demonstrated herein, such down regulation\/inhibition rescues cells (e.g., adult stem cells) from senescence restoring pluripotency and\/or proliferative capacity (see, e.g.,  F, , , and ). Such down regulation affords a number of uses. For example, it is believed the viable lifetime of stem cell populations (e.g., stem cell lines) can be extended (perhaps indefinitely) by down regulating (e.g., stably down-regulating) SINE\/Alu retrotransposon transcripts in those cells. When\/if desired, the stem cells can be induced to differentiate into embryoid bodies, precursors or particular lineages, or terminally differentiated cells according to standard methods well known to those of skill in the art.","Accordingly cells (including previously terminally differentiated mammalian cells, progenitor cells, stem cells, stem cell lines, and induced pluripotent stem cells (iPSCs)) are provided where the cells have SINE\/Alu retrotransposon transcripts downregulated. In various embodiments the cells include normal (non-senescent) stem cells, stem cells that have been \u201crescued\u201d from senescence by downregulating SINE\/Alu retrotransposon transcript production or activity, induced pluripotent stem cells (IPSCs) that contain a construct that downregulates SINE\/Alu retrotransposon transcript level or activity, embryonic stem cells, and the like. In certain embodiments cells are excluded that have been treated in a manner to form IPSCs (e.g., cells are excluded in which one or more of Nanog, and\/or LIN-28, and\/or Oct3\/4, and\/or Sox2, and\/or Klf4, and\/or c-myc are directly upregulated and\/or that contain heterologous constructs that express Oct3\/4, and\/or Sox2, and\/or Klf4, and\/or c-myc).","It was also demonstrated that cells in which pluripotency has been induced and\/or restored according to the methods described herein (e.g., inhibition of Alu retrotransposon) can be subsequently induced to differentiate using standard methods well known to those of skill in the art (e.g., withdrawal of LIF from culture media) (see, e.g., , and ). This provides further evidence that the methods described herein induce or restore pluripotency to a cell. Accordingly, in certain embodiments, differentiated cells comprising cells that have been induced to differentiate from pluripotent cells generated according to the methods described herein are contemplated.","In addition, in certain embodiments, methods of transdifferentiating cells are contemplated. The transdifferentiation methods typically comprise inducing or restoring a cell to a pluripotent phenotype according to the methods described herein (e.g., by inhibiting the Alu retrotransposon or other components of the pathway), and then culturing the resulting pluripotent cells under conditions that allow or induce differentiation. Cells that differentiate into a desired cell type (e.g., pancreatic beta cells, motoneurons, hematopoietic progenitor cells, neural cells, dopaminergic neurons, adipocytes, cardiomyocytes, and the like) and\/or lineage are then selected and can be subsequently cultured (e.g., expanded in culture) or directly utilized.","In certain embodiments the methods described herein can be used to restore pluripotency and\/or proliferative capacity to a cell (e.g., to a cell that has committed to a differentiation pathway). In certain embodiments the methods described herein can be used to restore to a lesser senescent state or to a non-senescent state a cell that shows one or more indications of senescence.","In various embodiments stem cell comprises a cell selected from the group consisting of an embryonic stem cell, a cord blood stem cell, an adult stem cell, and an IPSC. In various embodiments the mammalian stem cell is a stem cell derived from a tissue selected from the group consisting of human adipose tissue, human bone marrow, human neurological tissue, human smooth muscle, human adipose tissue, human cardiomyocytes, human endothelial tissue, human epithelial tissue, human pancreatic tissue, human bone or cartilage and the like.","In certain embodiments the cell is one that has committed to differentiation to a cell type selected from the group consisting of ectoderm, mesoderm, and endoderm. In certain embodiments the cell is one that has committed to differentiation to a cell type selected from the group consisting of human adipose cells, human blood cells, human nerve cells, human smooth muscle cells, human adipocytes, human chondrocytes, human osteoclasts and hosteoblasts, human cardiomyocytes, human endothelial cells, and human epithelial cells. In certain embodiments the cell comprises a non-renewing progenitor cell. In certain embodiments the mammalian cell comprises a terminally differentiated cell.","Downregulating\/Inhibiting SINE\/Alu Retrotransposon Transcripts","It was a surprising discovery that inhibition of SINE\/Alu retrotransposon transcripts can restore proliferative capacity and\/or pluripotency to a senescent stem cell or can maintain proliferative capacity and\/or pluripotency in a non-senescent stem cell. Any of a variety of methods to inhibit SINE\/Alu retrotransposon transcripts can be used.","In various embodiments SINE\/Alu retrotransposon transcripts can be reduced\/inhibited using inhibitory RNAs. Suitable inhibitory RNAs include, but are not limited to siRNAs, shRNAs, miRNAs, dicer-substrate 27-mer duplexes, single-stranded interfering RNA, and the like.","siRNAs typically refer to a double-stranded interfering RNA unless otherwise noted. In various embodiments, suitable siRNA molecules to inhibit SINE\/Alu retrotransposon transcripts include double-stranded ribonucleic acid molecules comprising two nucleotide strands, each strand having about 19 to about 28 nucleotides (i.e. about 19, 20, 21, 22, 23, 24, 25, 26, 27, or 28 nucleotides). Thus, the phrase \u201cinterfering RNA having a length of 19 to 49 nucleotides\u201d when referring to a double-stranded interfering RNA means that the antisense and sense strands independently have a length of about 19 to about 49 nucleotides, including interfering RNA molecules where the sense and antisense strands are connected by a linker molecule.","In addition to siRNA molecules, other interfering RNA molecules and RNA-like molecules can to inhibit SINE\/Alu retrotransposon transcripts. Examples of other interfering RNA molecules that can to inhibit SINE\/Alu retrotransposon transcripts include, but are not limited to short hairpin RNAs (shRNAs), single-stranded siRNAs, microRNAs (miRNAs), and dicer-substrate 27-mer duplexes. Examples of RNA-like molecules that can to inhibit SINE\/Alu retrotransposon transcripts include, but are not limited to siRNA, single-stranded siRNA, microRNA, piwiRNA, and shRNA molecules containing one or more chemically modified nucleotides, one or more non-nucleotides, one or more deoxyribonucleotides, and\/or one or more non-phosphodiester linkages. Typically, all RNA or RNA-like molecules that can interact with SINE\/Alu retrotransposon transcripts RISC and participate in RISC-related changes in gene expression can be referred to as \u201cinterfering RNAs\u201d or \u201cinterfering RNA molecules.\u201d SiRNAs, single-stranded siRNAs, shRNAs, miRNAs, and dicer-substrate 27-mer duplexes are, therefore, subsets of \u201cinterfering RNAs\u201d or \u201cinterfering RNA molecules.\u201d","Using the known nucleotide sequences for SINE\/Alu retrotransposon transcript(s), suitable siRNAs can readily be produced. In various embodiments siRNA that inhibit SINE\/Alu retrotransposon transcripts can comprise partially purified RNA, substantially pure RNA, synthetic RNA, recombinantly produced RNA, as well as altered RNA that differs from naturally-occurring RNA by the addition, deletion, substitution and\/or alteration of one or more nucleotides. Such alterations can include, for example, addition of non-nucleotide material, such as to the end(s) of the siRNA or to one or more internal nucleotides of the siRNA, including modifications that make the siRNA resistant to nuclease digestion.","In various embodiments one or both strands of the siRNA can comprise a 3\u2032 overhang. As used herein, a \u201c3\u2032 overhang\u201d refers to at least one unpaired nucleotide extending from the 3\u2032-end of an RNA strand. Thus in one embodiment, the siRNA comprises at least one 3\u2032 overhang of from 1 to about 6 nucleotides (which includes ribonucleotides or deoxynucleotides) in length, from 1 to about 5 nucleotides in length, from 1 to about 4 nucleotides in length, or about 2 to about 4 nucleotides in length.","In an illustrative embodiment in which both strands of the siRNA molecule comprise a 3\u2032 overhang, the length of the overhangs can be the same or different for each strand. In certain embodiments the 3\u2032 overhang is present on both strands of the siRNA, and is one, two, or three nucleotides in length. For example, each strand of the siRNA can comprise 3\u2032 overhangs of dithymidylic acid (\u201cTT\u201d) or diuridylic acid (\u201cuu\u201d).","In order to enhance the stability of the siRNA, the 3\u2032 overhangs can be also stabilized against degradation. In one embodiment, the overhangs are stabilized by including purine nucleotides, such as adenosine or guanosine nucleotides. In certain embodiments, substitution of pyrimidine nucleotides by modified analogues, e.g., substitution of uridine nucleotides in the 3\u2032 overhangs with 2\u2032-deoxythymidine, is tolerated and does not affect the efficiency of RNAi degradation. In particular, it is believed the absence of a 2\u2032 hydroxyl in the 2\u2032-deoxythymidine can significantly enhance the nuclease resistance of the 3\u2032 overhang.","In certain embodiments, the siRNA comprises the sequence AA(N19)TT (SEQ ID NO:3), AA(N21)TT (SEQ ID NO:4), NA(N21) (SEQ ID NO:5), and the like, where N is any nucleotide. In various embodiments these siRNA comprise approximately 30%-70% GC, and preferably comprise approximately 50% G\/C. The sequence of the sense siRNA strand corresponds to (N19)TT (SEQ ID NO:6) or N21 (SEQ ID NO:7) (i.e., positions 3 to 23), respectively. In the latter case, the 3\u2032 end of the sense siRNA is converted to TT. The rationale for this sequence conversion is to generate a symmetric duplex with respect to the sequence composition of the sense and antisense strand 3\u2032 overhangs. The antisense RNA strand is then synthesized as the complement to positions 1 to 21 of the sense strand.","Because position 1 of the 23-nt sense strand in these embodiments is not recognized in a sequence-specific manner by the antisense strand, the 3\u2032-most nucleotide residue of the antisense strand can be chosen deliberately. However, the penultimate nucleotide of the antisense strand (complementary to position 2 of the 23-nt sense strand in either embodiment) is generally complementary to the targeted sequence.","In another illustrative embodiment, the siRNA comprises the sequence NAR(N17)YNN (SEQ ID NO:8), where R is a purine (e.g., A or G) and Y is a pyrimidine (e.g., C or U\/T). The respective 21-nt sense and antisense RNA strands of this embodiment therefore generally begin with a purine nucleotide. Such siRNA can be expressed from pol III expression vectors without a change in targeting site, as expression of RNAs from pol III promoters is only believed to be efficient when the first transcribed nucleotide is a purine.","In various embodiments the siRNA of the invention can be targeted to any stretch of approximately 10-30, or 15-25, or 19-25 contiguous nucleotides in any of the target mRNA sequences (the \u201ctarget sequence\u201d). Techniques for selecting target sequences for siRNA are given, for example, in Tuschl et al., \u201cThe siRNA User Guide,\u201d revised May 6, 2004. The \u201csiRNA User Guide\u201d is available on the world wide web at a website maintained by Dr. Thomas Tuschl, and can be found by accessing the website of Rockefeller University and searching with the keyword \u201csiRNA.\u201d In addition, the \u201csiRNA User Guide\u201d can be located by performing a google search for \u201csiRNA User Guide\u201d and can also be found at \u201cwww.rockefeller.edu\/labheads\/tusch1\/sirna.html. Techniques for selecting target sequences for siRNA and miRNA can also be found in Sioud (2008) (), Humana Press.","In certain embodiments the sense strand of the present siRNA comprises a nucleotide sequence identical to any contiguous stretch of about 19 to about 25 nucleotides in the target SINE\/Alu retrotransposon transcript(s).","The SINE\/Alu retrotransposon transcript silencing siRNAs can be obtained using a number of techniques known to those of skill in the art. For example, the siRNA can be chemically synthesized or recombinantly produced using methods known in the art, such as the in vitro system described in U.S. published application US 2002\/0086356.","In certain embodiments the siRNAs are chemically synthesized using appropriately protected ribonucleoside phosphoramidites and a conventional DNA\/RNA synthesizer. The siRNAs can be synthesized as two separate, complementary RNA molecules, or as a single RNA molecule with two complementary regions. Commercial suppliers of synthetic RNA molecules or synthesis reagents include Proligo (Hamburg, Germany), Dharmacon Research (Lafayette, Colo., USA), Pierce Chemical (part of Perbio Science, Rockford, Ill., USA), Glen Research (Sterling, Va., USA), ChemGenes (Ashland, Mass., USA) and Cruachem (Glasgow, UK). Custom siRNA can be obtained from commercial suppliers (see, e.g., Thermo Fisher Scientific, Lafayette Colo.; Qiagen, Valencia, Calif.; Applied Biosystems, Foster City, Calif.; and the like).","In certain embodiments siRNA can also be expressed from recombinant circular or linear DNA plasmids using any suitable promoter. Suitable promoters for expressing siRNA from a plasmid include, for example, the U6 or H1 RNA pol III promoter sequences and the cytomegalovirus promoter. Selection of other suitable promoters is within the skill in the art. The recombinant plasmids can also comprise inducible or regulatable promoters for expression of the siRNA in a particular tissue or in a particular intracellular environment.","The siRNA expressed from recombinant plasmids can either be isolated from cultured cell expression systems by standard techniques, or can be expressed intracellularly at or near the target area(s) in vivo. The use of recombinant plasmids to deliver siRNA to cells in vivo is discussed in more detail below.","siRNA can be expressed from a recombinant plasmid either as two separate, complementary RNA molecules, or as a single RNA molecule with two complementary regions. Selection of plasmids suitable for expressing siRNAs, methods for inserting nucleic acid sequences for expressing the siRNA into the plasmid, and methods of delivering the recombinant plasmid to the cells of interest are within the skill in the art (see, e.g., Tuschl (2002) 20: 446-448; Brummelkamp et al. (2002) 296: 550 553; Miyagishi et al. (2002) 20: 497-500; Paddison et al. (2002) 16: 948-958; Lee et al. (2002) 20: 500-505; Paul et al. (2002) 20: 505-508, and the like).","In one illustrative embodiment, a plasmid comprising nucleic acid sequences for expressing an siRNA for inhibiting SINE\/Alu retrotransposon transcripts comprises a sense RNA strand coding sequence in operable connection with a polyT termination sequence under the control of a human U6 RNA promoter, and an antisense RNA strand coding sequence in operable connection with a polyT termination sequence under the control of a human U6 RNA promoter. The plasmid is ultimately intended for use in producing a recombinant adeno-associated viral vector comprising the same nucleic acid sequences for expressing the siRNA.","As used herein, \u201cin operable connection with a polyT termination sequence\u201d means that the nucleic acid sequences encoding the sense or antisense strands are adjacent to the polyT termination signal in the 5\u2032 direction or sufficiently close so that during transcription of the sense or antisense sequences from the plasmid, the polyT termination signals act to terminate transcription after the desired product is transcribed.","As used herein, \u201cunder the control\u201d of a promoter means that the nucleic acid sequences encoding the sense or antisense strands are located 3\u2032 of the promoter, so that the promoter can initiate transcription of the sense or antisense coding sequences.","In certain embodiments the siRNA can be delivered as a small hairpin RNA or short hairpin RNA (shRNA). shRNA is a sequence of RNA that makes a tight hairpin turn that can be used to silence gene expression via RNA interference. In typical embodiments, shRNA uses a vector introduced into cells and utilizes the U6 promoter to ensure that the shRNA is always expressed. This vector is usually passed on to daughter cells, allowing the gene silencing to be inherited. The shRNA hairpin structure is cleaved by the cellular machinery into siRNA, which is then bound to the RNA-induced silencing complex (RISC). This complex binds to and cleaves mRNAs that match the siRNA that is bound to it.","In certain embodiments the sense sequence of the shRNA will be from about 19 to about 30, more nucleotides (e.g. about 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, or 30 nucleotides) in length, more typically from about 19 to about 22 nucleotides in length, the antisense sequence will be from about 19 to about 30, more typically from 19 to about 22 nucleotides (e.g. about 19, 20, 21 or 22 nucleotides), in length, and the loop region will be from about 3 to about 19 nucleotides (e.g., about 3, 4, 5, etc., . . . up to about 19) nucleotides in length. In some embodiments, the sense and antisense sequences are the same length, i.e. the shRNA will form a symmetrical hairpin, but this is not necessarily the case. In some cases, the sense or antisense strand may be shorter than its complementary strand, and an asymmetric hairpin is formed. Further, while in some instances the base pairing between the sense and antisense sequences is exact, this also need not be the case. In other words, some mismatch between the sequences may be tolerated, or even desired, e.g. to decrease the strength of the hydrogen bonding between the two strands. However, in one illustrative embodiment, the sense and antisense sequences are the same length, and the base pairing between the two is exact and does not contain any mismatches. The shRNA molecule can also comprise a 5\u2032-terminal phosphate group that can be chemically modified. In addition, the loop portion of the shRNA molecule can comprise, for example, nucleotides, non-nucleotides, linker molecules, conjugate molecules, etc.","The shRNA\/siRNA\/piRNA described herein targets and causes the RNAi-mediated degradation of SINE\/Alu retrotransposon transcripts, or alternative splice forms, or participates in genomic silencing via (PIWI RNA pathways). Thus, in certain embodiments, methods are provided for inhibiting SINE\/Alu retrotransposon transcripts in a cell, comprising administering an effective amount of an SINE\/Alu retrotransposon transcript siRNA\/shRNA\/piRNA to the cell, such that the target mRNA is degraded.","In various embodiments the siRNA\/shRNA\/piRNA can be expressed from recombinant viral vectors introduced into the subject cells. The recombinant viral vectors comprise sequences encoding the siRNA\/shRNA and any suitable promoter for expressing the siRNA\/shRNA\/piRNA sequences. Suitable promoters include, but are not limited to, the U6 or H1 RNA pol III promoter sequences and the cytomegalovirus promoter. Selection of other suitable promoters is within the skill in the art. The recombinant viral vectors can also comprise inducible or regulatable promoters for expression of the siRNA\/shRNA in a particular tissue or in a particular intracellular environment.","The siRNA\/shRNA\/piRNA can be expressed from a recombinant viral vector either as two separate, complementary RNA molecules, or as a single RNA molecule with two complementary regions.","Any viral vector capable of accepting the coding sequences for the siRNA\/shRNA\/piRNA molecule(s) to be expressed can be used, for example vectors derived from adenovirus (AV); adeno-associated virus (AAV); retroviruses (e.g. lentiviruses (LV), Rhabdoviruses, murine leukemia virus); herpes virus, and the like. The tropism of the viral vectors can also be modified by pseudotyping the vectors with envelope proteins or other surface antigens from other viruses. For example, an AAV vector can be pseudotyped with surface proteins from vesicular stomatitis virus (VSV), rabies, Ebola, Mokola, and the like.","Selection of recombinant viral vectors suitable for use in methods for inserting nucleic acid sequences for expressing the siRNA\/shRNA\/piRNA into the vector, and methods of delivering the viral vector to the cells of interest are within the skill in the art (see, e.g., Domburg (1995) 2: 301-310; Eglitis (1988) 6: 608-614; Miller (1990) 1: 5-14; Anderson (1998) 392: 25-30, and the like).","In certain embodiments suitable viral vectors include, but are not limited to lentiviral vectors. In one illustrative embodiment, lentiviral shRNA constructs to knockdown genetic SINE\/Alu transcript are designed. The shRNA sense and anti-sense strands are chemically synthesized and the strands are annealed with equal amounts of each other creating restriction site specific overhangs for cloning, and ligated into a vector (e.g., a HindIII and BglII digested, gel purified pENTR\/pTER+ vector). Equal amounts of each construct is mixed with pLenti-CMV-GFP DEST vector in LR Clonase reaction to recombine cloned shRNA production elements into a destination vector according to the manufacturer's instructions (Invitrogen). The produced lentiviral plasmid is transformed into Stbl3 cells (Invitrogen) for amplification.","In certain embodiments suitable viral vectors include those derived from AV and AAV. In one illustrative embodiment, the siRNA\/shRNA\/piRNA is expressed as two separate, complementary single-stranded RNA molecules from a recombinant AAV vector comprising, for example, either the U6 or H1 RNA promoters, or the cytomegalovirus (CMV) promoter. A suitable AV vector for expressing the siRNA, a method for constructing the recombinant AV vector, and a method for delivering the vector into target cells, are described in Xia et al. (2002) 20: 1006 1010.","Suitable AAV vectors for expressing the siRNA\/shRNA, methods for constructing the recombinant AV vector, and methods for delivering the vectors into target cells are also described in Samulski et al. (1987) 61: 3096-3101; Fisher et al. (1996) 70: 520-532; Samulski et al. (1989) 63: 3822-3826; U.S. Pat. Nos. 5,252,479 and 5,139,941; International Patent Application Nos. WO 1994\/013788; and WO 1993\/024641, and the like.","Sources of Cells.","Stem cells (e.g., adult stem cells, embryonic stem cells, cord stem cells, IPSCs, etc.) can be obtained according to standard methods well known to those of skill in the art. In certain embodiments, stem cells are also commercially available.","Differentiating Stem Cells.","In certain embodiments methods are provided for differentiating stem cells in which level or activity of SINE\/Alu retrotransposon transcripts is reduced\/inhibited. It is believed these stem cells can be differentiated into embryoid bodies or terminally differentiated cells using standard differentiation methods well known to those of skill in the art.","For example, in certain embodiments, removal of leukemia inhibitory factor (LIF) can result in the differentiation of the modified stem cells described herein (e.g., stem cells in which SINE\/Alu retrotransposon transcripts are reduced\/inhibited) into embryoid bodies.","Methods of differentiating stem cells are well known to those of skill in the art. For example, extract from infarcted myocardium contains biochemical factors that induce cardiomyocyte differentiation from bone marrow-mesenchymal stem cells (BM-MSCs) (see, e.g., Ge et al. (2009) 381(3): 317-321). Bone morphogenic factors (BMP4, BMP7, and BMP8b) can increase differentiation of human germ cells from human ES cells (see e.g., Gonsalves et al. (2006) 15(6): 831-837). Primary cell cultures derived from adipose tissue or skeletal muscle differentiate into adipocytes when cultured in high glucose (see, e.g., Aguiari et al. (2008) 105(4): 1226-1231). Interleukin-27 induces differentiation in hematopoietic stem cells (see, e.g., Seita et al. (2008) 111(4): 1903-1912).","With respect to neural cell differentiation, the neurotrophin family is one of the most important inducible signals for the differentiation. Among them, nerve growth factor is well known to induce neurogenesis, and neurotrophin 3 is involved in oligodendrocyte development. Another important differentiation signal family is the ciliary neurotrophic factor (CNTF)-leukemia inhibitory factor (LIF) cytokine family, which plays a pivotal role in regulating gliogenesis in the developing mammalian central nervous system. In addition, AICAR (5-aminoimidazole-4-carboxamide-1-\u03b2-D-ribofuranoside;) induces astroglial differentiation of neural stem cells (see, e.g., Zang et al. (2008) 283(10): 6201-6208 and references therein).","A number of natural compounds are also known to induce differentiation of stem cells in vitro. Such compounds include, but are not limited to retinoic acid, PDGF, insulin, Arctigenin, ATRA (vitamin A), boswellic acid, bromelain and other proteolytic enzymes, CAPE, flavonoids (including apigenin, luteolin, quercetin, genistein, and daidzein), emodin, EPA and DHA, monoterpenes, resveratrol, 1,25-D3 (vitamin D3), and the like.","It is believed the pluripotent cells produced using the methods described herein can be subsequently induced to differentiate using methods well known to induce differentiation if iPSCs. In this regard it is noted that methods of differentiating induced pluripotent cells into CD34CD43 hematopoietic progenitors and CD31CD43endothelial cells are described by Choi et al. (2009) 27: 559. These cells can be further separated into phenotypically defined subsets of primitive hematopoietic cells in a pattern of differentiation resembling that of ES cells. Methods of differentiating induced pluripotent cells into pancreatic insulin-producing cells are described by Zhang et al. (2009) 19: 429. After first generating PDX-1 positive progenitor cells, human iPS cells were further differentiated into pancreatic cells expressing MafA, Glut2, insulin, and in some cases, amylase and C-peptide. Functional cardiomyocytes demonstrating sarcomeric organization and expressing cardiac markers including Nkx2.5, cardiac Troponin T, atrial natriuretic factor, and myosin heavy and light chains, have also been derived from human iPS cells and are indistinguishable from those generated from ES cells (see, e.g., Zhang et al. (2009) 104:e30). Electrophysiology experiments revealed that like ES cells, iPS cells differentiate into nodal-, atrial-, and ventricular-like phenotypes, and are responsive to the canonical cardiomyocyte beta-adrenergic signaling pathway. Both human ES and iPS cells were efficiently converted to neural cells using two inhibitors of TGF-beta\/Smad signaling, Noggin and the drug SB431542 (Chambers et al. (2009) 27: 275). The synergistic action of these two inhibitors resulted in Pax6 primitive neural cells that could then be further differentiated into neural crest, anterior CNS, somatic motoneurons, and dopaminergic neurons.","These approaches are intended to be illustrative and not limiting. Other methods of inducing differentiation of stem cells are well known to those of skill in the art.","In certain embodiments cells and compositions comprising cells that have been differentiated into embryoid bodies or further differentiated (e.g., terminally differentiated) are also contemplated. Such differentiated cells include but are not limited to embryoid bodies and\/or progenitor cells and\/or terminally differentiated cells that are differentiated into lineages for cardiomyocytes, blood cells, epithelial cells, osteoblasts, osteoclasts, chondrocytes, adipocytes, smooth muscle cells, nerve cells (neurons), glial cells, pancreatic \u03b2-cells, motoneurons, and the like.","The following examples are offered to illustrate, but not to limit the claimed invention.","In this example, we report mechanistic aspects of ex-vivo adult mesenchymal stem cell aging. Using genome-wide comparative analysis of repairable and persistent DNA damage in human adult stem cells as they traverse into the state of cellular senescence, we were able to uncover a new, unexpected functional role of retrotransposons in the complex pathways of cellular aging. We present evidence that a majority of repairable DNA damage in self-renewing human adult stem cells is distributed non-randomly and is enriched at retrotransposal repeats. Our data suggests that the misregulation of transcriptional activity of at least one class of human retrotransposons (SINE\/Alu repeats) impairs the assembly of cohesin and condensin complexes in pericentric chromatin sites of \u03b3H2AX enrichment, thus interfering with cellular DNA repair machinery. The inability of adult stem cells to resolve pericentric DNA damage results in exit from self-renewal to senescence. We provide evidence that this event is reversible, and suggest a new mechanistic model wherein the loss of appropriate control of transcriptional activity in the retrotransposal portion of the human genome can directly or indirectly influence complex molecular events leading to the cessation of adult stem cell self-renewal and triggering their senescence.","Results","Replicative Senescence of Human Adult Mesenchymal Stem Cells","Human adult adipose derived mesenchymal stem cells (hADSCs) were isolated from subcutaneous abdominal fat as described in Experimental Procedures, and characterized based on cell surface antigens (). After four weeks in the culture, the cells became more uniform as demonstrated by FACS analysis. More than 99.3% and 99.9% of the cells expressed mesenchymal stem cell MSC-specific cell type markers CD105 and CD44 (Dominici et al. (2006) 8: 315-317), respectively, but did not express markers of hematopoietic stem cells (CD45) or endothelial progenitor cells (CD31) (). hADSC can be induced to differentiate along several mesenchymal tissue lineages, including adipocytes, osteoblasts, myocytes, and chondrocytes (Erickson et al. (2002) 290: 763-769; Zuk et al. (2001) 7: 211-228). Thus, hADSCs in these experiments bore strong resemblance to mesenchymal stem cells (MSCs).","The self-renewing (SR) capacity of hADSCs remained consistent until population doubling (PD17), after which they displayed characteristic phenotypes of \u201cold age\u201d with prolonged ex vivo passages shown in  (and also described by (Bonab et al. (2006) 7: 14; Fehrer et al. (2007) 6: 745-757; Kern et al. (2006) 24: 1294-1301). The culture's morphological abnormalities are typical of the Hayflick model of cellular aging (Juckett (1987) 38: 49-71). By PD37, hADSC cultures accumulated non-dividing giant cells expressing the enzyme lysosomal pH6 senescence-associated J3-galactosidase (SA-J3-Gal) (Dimri et al. (1995) 92: 9363-9367), as shown in . Cells self-renewed poorly due to a decrease in the number of dividing cells as determined by incorporation of bromodeoxyuridine (BrdU) and [H] thymidine into DNA ( and ).","To determine if the hADSCs limited expansion capacity was due to senescence associated with persistent DNA damage accumulation, we performed immunostaining experiments with antibodies against key mediators of DNA damage response (DDR), which facilitate checkpoint activation and repair. In particular, we used a phosphorylated form of histone variant H2AX (\u03b3H2AX) (Shiloh (2003) 3: 155-168) and p53 binding protein-1 (53BP1) (Aguilera and Gomez-Gonzalez (2008) 9: 204-217; Shiloh (2003) 3: 155-168; Stewart (2009) 8: 1532-1538) to provide evidence of the presence of molecular characteristics of cells bearing DNA double-strand breaks (DSB) in both self-renewing and senescent hADSC cultures. As hADSCs approached senescence, both \u03b3H2AX and 53BP1 localized (), forming characteristic foci, named persistent DNA damage foci (previously reported by Rodier et al. (2009) 11: 973-979). Persistent \u03b3H2AX\/53BP1 foci formation upon cellular senescence has been associated with the presence of unresolved DSB, as determined by colocalization with several DNA repair factors (d'Adda di Fagagna et al. (2003) 426: 194-198). Occurrences of \u03b3H2AX\/53BP1 foci formations were very rare in self-renewing ADSCs, and their formation increased as cultured hADSCs approached senescence (SEN hADSCs) ().","Combined, these results reveal that senescence-associated effects in hADSCs are not restricted to senescent passages, but are continuously acquired from the onset of their ex-vivo expansion, similar to phenomena previously reported for human fibroblasts and human hematopoietic stem cells.","We also noticed that senescent hADSCs contained activated forms of the DNA damage checkpoint kinases CHK1 and CHK2, phosphorylated on S345 and T68, respectively (). The phosphorylation of these sites by ATM\/ATR is required for full execution of DNA-damage-induced cell-cycle arrest in human somatic cells (d'Adda di Fagagna et al. (2003) 426: 194-198; Sedelnikova et al. (2008) 7: 89-100; Tanaka et al. (2006) 39: 313-323). To further delineate the state of these cells, we performed differential (SR vs. SEN hADSCs) transcriptional analysis of the 96 genes involved in multiple aspects of the cell cycle (Human Cell Cycle qPCR array), described in detail in Experimental Procedures. Genes that were statistically significantly downregulated with a p value <0.05 in SEN hADSCs are shown in Table. 1, and include genes involved in cell cycle regulation, DNA replication, and mitosis, suggesting that senescent hADSC cultures follow a DDR program directly or indirectly related to the formation of senescence-associated DNA-damage foci. The causal factors that might mediate this process in human adult stem cells are as yet unknown.","Genome-Wide ChIP-Seq Location Analysis of \u03b3H2AX in Senescent hADSCs.","To identify genomic loci directly engaged in DDR in SEN hADSCs, we performed genome-wide profiling of hADSC chromatin using chromatin immunoprecipitation with an antibody against the \u03b3H2AX modified histone, followed by next generation sequencing (ChIP-seq) on the ABI SOLiD platform (). Asynchronous SR samples of hADSCs were used in similar experiments to establish an overall representation of \u03b3H2AX modified chromatin, if any, to define the locations of repairable DNA damage which does not affect the SR properties of the cells. ChIP-seq was performed on four replicates each of SR and SEN hADSC cultures. Genomic mapping of the resulting sequence tags (Table 2) was followed by outlier removal, noise reduction, and merging of the data from replicate experiments. Details of the algorithms developed and approaches used for ChIP-seq data analyses can be found in Supplemental Methods. The ChIP-seq experimental protocol and data merging we employed were supported by the highly consistent mapping results seen for the replicate experiments. Having mapped and processed the ChIP-seq tags, we evaluated tag counts with respect to an empirically-derived count threshold (Supplemental Methods) for approximately mono-nucleosomal-sized windows of 200 bp in order to identify individual \u03b3H2AX modified nucleosomes on the human genome. The resulting genomic distributions of \u03b3H2AX modified nucleosomes were compared for SR and SEN cells () in order to assess the positional and quantitative changes in \u03b3H2AX modified chromatin associated with these phenotypes.","Majority of \u03b3H2AX Modified Chromatin Maps to the Retrotransposal Portion of the Human Genome","Past studies reported that topological constraints on chromatin structures upon DNA replication and transcription pose the biggest danger to DNA integrity (reviewed in Shrivastav et al. (2008) 18: 134-147). Altogether, there are four potential sources of damage in aging adult stem cells: transient DSB generated by inducible gene transcription, DNA origin firing (Ju et al. (2006) 312: 1798-1802; Ju and Rosenfeld (2006) 5: 2557-2560; Rampakakis and Zannis-Hadjopoulos (2009) 37: 5714-5724), DNA damage resulting from impediments of replication forks due to collision of replication and transcription machineries, and difficulties arising from particularities in the replication of centromeric and telomeric regions of the genome (Dalal and Bui (2010) 22: 392-402; Dotiwala et al. (2010) 20: 328-332; Morris and Moazed (2007) 128: 647-650; Schoeftner and Blasco (2009) 28: 2323-2336). Recent studies of \u03b3H2AX distribution in the genome of cycling cells revealed \u201cfragile\u201d genomic locations (Szilard et al. (2010) 17: 299-305), suggesting that mapping sites of \u03b3H2AX enrichment could be fruitful to pinpoint at-risk genomic elements in other genomes. Since mechanistic aspects of \u201cfragile sites\u201d are linked to regulation of cell cycle checkpoints and DNA repair (Durkin and Glover (2007) 41: 169-192), one can expect that the gradual accumulation of unresolved DNA damage sites at these locations in asynchronously cycling cells can result in the triggering of cellular senescence.","Our mapping experiments revealed that \u03b3H2AX modified chromatin was distributed non-randomly and, unexpectedly, represented a small fraction of total human chromatin in asynchronously cycling SR hADSCs: \u02dc1.4% (200,977 \u03b3H2AX modified nucleosomes), and it represented an even smaller portion in SEN hADSCs: \u02dc1.2% of total chromatin (173,877 \u03b3H2AX modified nucleosomes). The majority of \u03b3H2AX modifications (SR: 65.4%, SEN: 65.2%) mapped to transposable elements (TEs), most of which were SINE\/Alu, L1, or LTR retrotransposons (). In addition to the overall abundance of retrotransposons bearing \u03b3H2AX modified nucleosomes, this genomic element also showed the greatest difference in relative frequencies of \u03b3H2AX marks for SR versus SEN hADSCs (). SINE\/Alu elements were the most enriched for \u03b3H2AX modified nucleosomes in SR cells compared to SEN cells, whereas L1s had the greatest relative increase of \u03b3H2AX modified nucleosomes in SEN cells (). Apart from genic sequences, the retrotransposal portion of the genome is known to impede the progression of replication machinery (Scheifele et al. (2009) 106: 13927-13932; Voineagu et al. (2008) 105: 9936-9941; de la Loza et al. (2009) (), 8: 620-626), suggesting that a portion of \u03b3H2AX detected in asynchronously dividing cells (SR sample) might be caused by replication-fork pausing or collapse as described previously in yeast (Admire et al. (2006) 20: 159-173; Cha and Kleckner (2002) 297: 602-606; de la Loza et al. (2009) (), 8: 620-626; Deshpande and Newlon (1996) 272: 1030-1033).","Our data suggest a role for retrotransposons in DNA damage-mediated cellular response related to normal cell cycle progression (Curcio et al. (2007) 27: 8874-8885; Scheifele et al. (2009) 106: 13927-13932), and imply that these elements may contribute to the aging phenotype in hADSCs (Rudin and Thompson (2001) 30: 64-71; Wang et al. (1999) 433: 147-157). Our data further corroborate the observation that the retrotransposal portion of the genome is particularly sensitive to DSB, a critical finding considering that this type of DNA breakage has been recently linked directly to genomic rearrangements in cancer (Eickbush (2002) 31: 126-127; Gosselin et al. (2009) 69: 7917-7925; Lin et al. (2009) 139: 1069-1083; Rudin and Thompson (2001) 30: 64-71; Weinstock et al. (2006) () 5: 1065-1074).","Size and Distribution of \u03b3H2AX Chromatin in SR and SEN hADSCs","Previous studies have indicated that \u03b3H2AX forms within minutes of DSB formation and can cover large (thousands to millions of base pairs) regions of chromatin (Redon et al. (2003) 4: 678-684; Rogakou et al. (1999) 146: 905-916). Therefore, we investigated the continuity of \u03b3H2AX chromatin in both asynchronously dividing SR and SEN hADSCs.","We developed a maximum segment-based algorithm to identify contiguous genomic regions that have anomalously high densities of \u03b3H2AX modified sites (Supplemental Methods). Such \u03b3H2AX-dense regions were taken to represent individual \u03b3H2AX chromatin clusters, and the distributions of the clusters were evaluated for SR and SEN cells. There was a single dominant \u03b3H2AX cluster size peak in both SR and SEN cells, which corresponded to approximately mono-nucleosomal-sized fragments that probably marked single DSB sites (). The total amount of \u03b3H2AX mono-nucleosomal chromatin did not change significantly upon cellular senescence of adult stromal cells (): there was \u02dc21 MB of mononucleosomal sized chromatin in each cell type and the difference between SR and SEN cells was only \u02dc0.6%. Our computational analysis did not reveal a particular genomic category that is significantly enriched for mono-nucleosomal damage accumulation upon cell cycle arrest in SEN hADSCs (Table 3). However, a majority of the mono-nucleosomal \u03b3H2AX sites (67.3% in SR and 65.8% in SEN) were associated with the retrotransposal portion of the genome, with SINEs contributing 34.9% and 30.0% and LINEs accounting for 32.4% and 35.78% of these sites in SR and SEN hADSCs, respectively (Table 3).","The \u03b3H2AX cluster size distributions were largely coincident for the majority of the clusters, but began to diverge substantially in tails of the distributions, which were occupied by the largest clusters (). This difference between the SR versus SEN \u03b3H2AX cluster size distributions was used to delineate mid-size and large clusters. Relative entropy values, calculated as a measurement of the difference between the two distributions (Supplemental Methods), were calculated over a range of cluster length threshold values. The cluster length threshold value that maximized the relative entropy between the SR versus SEN \u03b3H2AX cluster size distributions was taken as a cut-off between mid-size and large clusters ().","In contrast to what was observed for mono-nucleosomal sites, there were substantial differences in the amount of, as well as the genomic features occupied by, large \u03b3H2AX clusters. The total amount of large \u03b3H2AX clusters dropped more than three-fold from the SR (37,467,892 bp) to the SEN (11,243,429 bp) samples (). In SR cells, 47.9% of large cluster chromatin was associated with the genic portion of the genome including: (i) gene promoters, (ii) introns, (iii) exons and, (iv) intron-exon junctions (Table 3). In SEN non-replicating samples, large \u03b3H2AX clusters were shifted to intergenic regions, with a decrease in genic sequences to 35.2%.","We hypothesized that this \u03b3H2AX deposition might result from topological problems arising from replication fork collision with transcription machinery. Our data support such a hypothesis since the dynamic of large clusters was shifted to intergenic regions in non-replicating SEN samples, with drastic decreases in exonic and intronic sequences. It is tempting to speculate that large genic and intergenic clusters of \u03b3H2AX might represent hot spots not only for chromosome fragility at the G2-M transition (SEN samples), but also for sister chromatid entanglement during the S phase of SR hADSCs.","\u03b3H2AX Chromatin and Gene Density","Transcriptional activity may represent an obstacle for progressing replication forks, resulting in their collapse and the accumulation of DSB (reviewed in Zegerman and Diffley (2009) () 8: 1077-1088). Therefore, functional genic regions such as exons and promoters are possibly relatively enriched with \u03b3H2AX modified nucleosomes in SR hADSCs, in contrast to SEN cells that have ceased their replication activity. Also, one may expect that gene-dense human chromosomes would be prone to \u03b3H2AX accumulation, and if this damage were not resolved during checkpoint activation, perhaps due to wear-and-tear of the DNA repair machinery, this could trigger the senescent phenotype. If this were the case, one would expect to observe \u03b3H2AX enrichment at similar locations within the chromatin of SR and SEN cells, and the amounts of \u03b3H2AX chromatin would be positively correlated with gene density in both phenotypes.","To test these two hypotheses, we evaluated the overlap between \u03b3H2AX chromatin sites in SR and SEN cells and compared the gene density of human chromosomes to the density of \u03b3H2AX sequence tags in SR and SEN cells. The overlap of \u03b3H2AX chromatin sites between cell types was small: \u02dc5%. Gene density and \u03b3H2AX tag density were significantly positively correlated in SR samples, and negatively correlated in SEN samples ( and Table 4).","This difference in correlations for SR versus SEN cells is consistent with the transcription versus replication complex conflict model explaining the excess of genic-region-specific damage in SR hADSCs, and points out that the majority of damage in SEN cells is not related to unrepaired damage accumulation caused by the collision of replication forks and transcriptional complexes.","Considering that about 50% of \u03b3H2AX modifications are associated with the retrotransposal portion of the genome (), and it has been previously reported that the two most abundant categories of the retrotransposons, such as SINES and LINE, are differentially enriched in gene-dense and gene-poor chromatin (intergenic), respectively, with a bias toward GC-rich DNA (Lander et al. (2001) 409: 860-921; Mouse Genome Sequencing Consortium, Waterston et al. (2002) 420: 520-562), we next investigated the correlation between \u03b3H2AX accumulation and the GC-content of human chromosomes. GC-rich genomic areas are also prone to the accumulation of DNA damage (Cha and Kleckner (2002) 297: 602-606), but there is no a priori reason to expect a difference in GC-related damage accumulation between SR and SEN cells. Indeed, when chromosomal GC content and \u03b3H2AX tag density were compared, as was done for gene density, we observed significantly positive correlations for both SR and SEN hADSCs ( and Table 4). The positive correlation of GC content and \u03b3H2AX tag density in SEN cells stood in contrast to the negative correlation between gene density and \u03b3H2AX tag density in the same cells. This result is unexpected, since it is known that GC content is positively correlated with gene density. To tease apart these two genomic features, we performed a partial correlation analysis between gene density and \u03b3H2AX tag density in SEN cells by controlling for GC content. When partial correlation was performed in this way, the correlation between gene density and \u03b3H2AX tag density in SEN cells became more highly negative and statistically significant (Table 4).","Our data indicate a redistribution of \u03b3H2AX modified nucleosomes from genic regions in SR to non-genic regions in SEN hADSCs. This is consistent with what is seen when the relative enrichments of \u03b3H2AX modified nucleosomes are compared for SR and SEN cells genome-wide (). The redistribution of \u03b3H2AX during senescence may also explain the excess of SINE\/Alu-associated \u03b3H2AX modified nucleosomes in SR cells, since SINE\/Alus are enriched in and around genes, along with the excess of L1 \u03b3H2AX modified nucleosomes in SEN cells, as L1s are primarily intergenic ().","Promoter Distribution of \u03b3H2AX","Previously, transcriptional activation has been reported to require the formation of transient DSB in the promoters of inducible genes (Ju et al. (2006) 312: 1798-1802). Later it was suggested that transcriptional misregulation upon cellular aging might stem from the inability to successfully repair DSB (Tower (2006) 4: 101-103). Under these conditions, the transient DSB in SR hADSCs would become persistent DNA damage sites in SEN cells.","We observed significant changes in \u03b3H2AX accumulation within proximal promoter regions of protein-coding genes between SR and SEN cells. Consistent with the overall genomic distribution of \u03b3H2AX, there was a twofold increase in \u03b3H2AX accumulation in SR cell promoters than seen for SEN cells; \u02dc3.9% of SR cell promoter sequences were occupied by \u03b3H2AX compared to 2.0% for SEN cells (Table 5). Furthermore, there was relatively little overlap in genes with \u03b3H2AX-modified promoters between SR and SEN cells (). For both SR and SEN hADSCs, \u03b3H2AX showed periodic enrichment at specific positions from \u22122 kb to +2 kb relative to transcription start sites (TSS) (). These enrichment patterns are consistent with nucleosome phasing around TSS in both SR and SEN cells; however, the distributions were shifted, suggesting a repositioning of \u03b3H2AX modified nucleosomes relative to TSS in SEN samples.","We evaluated genes with \u03b3H2AX modified promoters in SEN hADSCs (Table 6) in order to assess their potential impact on cellular function related to the aging phenotype. There were a number of over-represented gene ontology (GO) functional categories among this set of genes, including the related cell cycle (GO:0007049) and cell proliferation (GO:0008283) categories (Table 7). Genes in these categories have demonstrated effects on cell cycle progression and cell division (Table 8). When we performed a similar analysis for the group of cell cycle regulator genes (Table 1 and data not shown), we did not observe any correlation between senescence-associated downregulation of transcriptional activity of the genes and \u03b3H2AX accumulation in their promoters.","Our data suggest that down-regulation of tested cell cycle regulators in SEN hADSCs at the transcriptional level is not related to accumulation of DSB as previously proposed (Tower (2006) 4: 101-103).","Peri-Telomeric Distribution of \u03b3H2AX Chromatin and Human Adult Stem Cell Senescence","Previous studies have reported that DSBs causing senescence in somatic cells arise directly from dysfunctional telomeres (Celli and de Lange (2005) 7: 712-718; d'Adda di Fagagna et al. (2003) 426: 194-198; Takai et al. (2003) 13: 1549-1556). It was speculated that telomeric erosion might provide the reservoir of persistent DNA damage signal resulting in sustained p53 activation and manifestation of the SEN phenotype.","To ascertain whether dysfunctional telomeres are directly engaged in DDR in SEN hADSCs, we investigated the chromosomal enrichment of \u03b3H2AX in the telomeric and peri-telomeric areas of the human genome.","Telomeric regions are highly repetitive and have not been extensively characterized at the sequence level; therefore, we could not unambiguously map telomeric \u03b3H2AX ChIP-seq tags to the human genome. To evaluate \u03b3H2AX accumulation at telomeres, we therefore analyzed the counts of ChIP-seq tags that bore a specific telomeric repeat sequence motif (TTAGGG\/AATCCC, (SEQ ID NO:9)) among all sequencing tags (mapped uniquely and unmapped due to their repetitive nature). SR cell samples had an average of 1.66% of such telomeric tags while SEN cells had 1.63%. These percentages did not reveal an over-representation of telomeric tags, and the difference between the two cell types was not significant.","We defined peri-telomeres as 100 kb regions from the end of the sequenced human chromosomes, and evaluated 15 Mb of genomic DNA extending into the chromosome arms for SR and SEN hADSCs (). Overall, SR samples showed an enrichment of \u03b3H2AX over the entire region, while SEN samples were depleted for \u03b3H2AX. However, for both asynchronously cycling adult stem cells and for their SEN counterparts there was a marked decrease in the \u03b3H2AX density approaching and into the peri-telomeric regions (). In fact, the total fraction of \u03b3H2AX modified chromatin found in peri-telomeres was quite small (SR=0.07%, SEN=0.05%; ), but this could be due to the relatively small portion of the genome made up of peri-telomeres as defined here. To evaluate this possibility, we computed the genomic enrichment of \u03b3H2AX modified chromatin in peri-telomeres within SR and SEN cells as the fraction of modified chromatin normalized by the fraction of the genome occupied by peri-telomeric sequences. When peri-telomeric regions were evaluated in this way, they showed 1.5-fold (SR) and 2.1-fold (SEN) depletions in \u03b3H2AX modified nucleosomes ().","Our data support the notion that, contrary to what has been observed in human somatic cells (d'Adda di Fagagna et al. (2003) 426: 194-198), the aging of human adult stem cells might not be directly triggered by extensive peri-telomeric damage due to shortening of telomeres. This observation further corroborates findings previously reported (Sedelnikova et al. (2004) 6: 168-170). However, close inspection revealed a number of individual chromosomes with significant telomeric accumulations of \u03b3H2AX. The peri-telomeric \u03b3H2AX distribution on these chromosomes was markedly asymmetrical with respect to the chromosome arms (). In SR cells, chromosomes 10, 12, 17, and 18 had \u03b3H2AX enriched telomeres, as did chromosomes 4, 7, and 18 in SEN hADSCs. Despite the differences in peri-telomeric \u03b3H2AX accumulation between SR and SEN cells, the chromosome arm asymmetry was always preserved. The relative levels of chromosome arm-specific \u03b3H2AX peri-telomeric accumulation were identical for SR and SEN cells ().","Pericentric distribution of \u03b3H2AX","Compelling evidence indicates that pericentric chromatin is of critical importance for the regulation of kinetochore assembly, sister chromatid cohesion, spindle attachment, and chromosome segregation (Blower and Karpen (2001) 3: 730-739; Bernard et al. (2001) 294: 2539-2542; Schueler and Sullivan (2006) 7: 301-313). Data in yeast also point to a causal relationship between fork stalling and chromosomal DSB in areas known as \u201creplication slow zones\u201d (RSZ) (Cha and Kleckner (2002) 297: 602-606), many of which are associated with pericentric and centromeric chromatin. Also, spindle assembly checkpoint (SAC) acts semi-redundantly with the DNA damage checkpoint to enact long-term cell-cycle arrest in the presence of unresolved DSB, thereby preventing the segregation of the damaged chromatin (Dotiwala et al. (2010) 20: 328-332; Elledge (1996) 274: 1664-1672; Harrison and Haber (2006) 40: 209-235; Lim et al. (2009) 19: 325-333). Therefore, we further investigated whether human pericentric chromatin shows signs of fragility upon aging of adult stem cells, and whether or not it is resistant to \u03b3H2AX accumulation in cycling cells.","In our data, the distributions of \u03b3H2AX modified chromatin at peri-centromeric regions were markedly different from those seen for peri-telomeres. While peri-centromeres also contained a small fraction of the overall \u03b3H2AX modified chromatin across the genome (SR=0.78%, SEN=0.82%; ), there was substantially more \u03b3H2AX modified chromatin in peri-centromeres than expected based on the fraction of the genome they occupy (). Similar to chromosome-specific damage in peri-telomers, there were a number of individual chromosomes with highly enriched levels of peri-centromeric \u03b3H2AX accumulation (). Overall peri-centromeres exhibited 2.0- and 2.1-fold enrichments for SR and SEN cells respectively (), thus indicating intrinsic susceptibility (\u201cfragility\u201d) of these chromosomal regions. Peri-centromeres were not only enriched for \u03b3H2AX, but they were also one of the few genomic features that showed relatively greater \u03b3H2AX presence in SEN cells (, ). Chromosomes 6, 14, 15, 16 and 21 demonstrated the most significant \u03b3H2AX enrichment in SEN cells ().","Interestingly, our data did not reveal uniform and gradual accumulation of \u03b3H2AX in pericentric chromatin on the same chromosomes upon senescence, suggesting that damage accumulation in senescence is not necessarily a result of simply passing on a cell-cycle-associated defect in the DNA repair of these regions.","CENP-A and CREST Colocalization with Persistent DNA Damage Foci in Senescent ADSCs","An increase in pericentric \u03b3H2AX accumulation only on a subset of human chromosomes in SEN hADSCs could be indicative of: i) unresolved DSB (possibly due to a change in the characteristics of their pericentric heterochromatin and the molecular machinery responsible for its repair), and\/or, ii) defects in the assembly of kenetochore that provokes spindle-generated DNA damage, resulting in SAC (Guerrero et al. (2010) 107: 4159-4164). Both of these events, acting separately or synergistically, might trigger senescence of human adult stem cells.","To uncouple these two possible scenarios, we first evaluated colocalization of central to kinetochore assembly centromeric histone H3 variant CENP-A with persistent \u03b3H2AX\/53BP1 foci in SEN hADSCs. SR and SEN hADSCs were immunolabeled with antibodies to the 53BP1 and CENP-A proteins and studied by confocal fluorescent microscopy. In nearly 75% of the cases, CENP-A colocalized with large 53BP1\/\u03b3H2AX in SEN hADSCs (, B, , B). Similar overlapping immunostaining patterns were observed when human anti-kinetochore serum (\u03b1-CREST) and anti-53BP1 antibodies or \u03b1-CREST and anti-\u03b3H2AX antibodies were assessed by confocal microscopy (, D). As expected, we did not observe (detectable by our method) colocalization of centromeric regions with either 53BP1 () or \u03b3H2AX (data not shown) in interphase of SR hADSCs, since no prolonged cell cycle arrest was triggered in these cells (, ) nor \u03b3H2AX\/53BP1 foci were formed (). Our data indicate that centromeric regions are embedded within persistent \u03b3H2AX\/53BP1 DNA damage foci, formed upon senescence of hADSCs, and yet no defects in centromeric chromatin CENP-A incorporation\/inner kinetochore assembly are observed. CENP-A is required to recruit many other centromere and kinetochore proteins (McClelland et al. (2007) 26: 5033-5047), with the exception of proteins located in adjacent heterochromatin domains, such as heterochromatic protein 1 (HP1) (Blower and Karpen (2001) 3: 730-739). We cannot rule out a possibility that defects in the other kinetochore\/spindle components acting downstream of CENP-A affect the assembly of mitotic spindle, and that such deficiencies result in increased levels of \u03b3H2AX in peri-centromeric regions due to inappropriate chromosome segregation and cell cycle arrest (Dalton et al. (2007) 67: 11487-11492; Guerrero et al. (2010) 107: 4159-4164; Quignon et al. (2007) 26: 165-172). In fact, we observed significant transcriptional downregulation upon hADSCs senescence of KNTC1 (hROD), a component of tension-sensitive kinetochore complex, known as the RZZ (Table 1) (Chan et al. (2000) 2: 944-947).","Senescence-Associated \u03b3H2AX Foci are Sites of Active Pol-III Transcription","There is a body of literature suggesting that pericentric chromatin is essential for the establishment and function of centromeres (Bernard et al. (1994) 214: 373-380; Dalal and Bui (2010) 22: 392-402). In fission yeast, native pericentric chromosomal regions are permissive to RNA Pol-III (Chen et al. (2008) 451: 734-737; Scott et al. (2007) 2: e1099), indicating a complex relationship between heterochromatic assembly, maturation of centromeric chromatin, and transcription. Mammalian chromatin modifying activities associated with cellular aging, such as sirtuins, have been previously reported to contribute not only to the formation of facultative heterochromatin (Vaquero et al. (2007) Nature 450, 440-444), but to also be involved in the mediation of repression at constitutive heterochromatic regions such as pericentromeric chromatin. Importantly, the generation of DSB by oxidative stress leads to increased transcription of pericentric satellite repeat DNA in a model of mammalian embryonic stem cells (Oberdoerffer et al. (2008) 135: 907-918).","With this in mind, we investigated the possibility of transcriptional activity associated with persistent \u03b3H2AX\/53BP1 foci in SEN hADSCs. We combined the transcriptional assay of 5-fluorouridine (5-FUr) incorporation with detection of the 53BP1 and promyelocytic leukemia (PML) proteins upon senescence of hADSCs (described in Experimental Procedures and Casafont et al. (2006) 140: 453-462). Nuclear incorporation of a halogenated nucleotide analog into nascent RNA was visualized in the whole nucleus by immunocytochemistry with an anti-BrdU antibody (Boisvert et al. (2000) 148: 283-292). We chose a specific nuclear compartment, PML body, since it has been suggested to play a role in aspects of transcriptional regulation and\/or protein segregation (Carbone et al. (2002) 21: 1633-1640; Dellaire et al. (2006) 175: 55-66; Dellaire et al. (2009) 8: 3750-3769). We observed a close juxtapositioning of PML nuclear bodies and senescence-associated persistent 53BP1 foci (). Nascent RNA transcripts visibly decorated the periphery of the PML nuclear body and showed clear colocalization with the 53BP1 signal (). These observations are consistent with previous reports indicating a possible involvement of RNA in 53BP1 foci formation after IR-induced damage in NIH3T3 and Hela cells (Pryde et al. (2005) 118: 2043-2055).","It has been proposed that transcriptional control through retroelements may facilitate pericentric transcription (Ugarkovic (2005) 6: 1035-1039). The large clusters of \u03b3H2AX chromatin in our experiments colocalized with pericentric regions known to be enriched for SINE\/Alu retrotransposal repeats (, Table 3, and Schueler and Sullivan (2006) 7: 301-313). Considering that transcriptional activity of SINE\/Alu repeats was largely upregulated in SEN hADSCs () and driven by the Pol-III transcriptional complex (Deininger (1989) (Washington, D.C., American Society for Microbiology)), we examined the dynamic formation of the persistent 53BP1\/\u03b3H2AX foci and FUr-incorporation in response to the inhibition of Pol-III transcription by the drug tagetin (Allen et al. (2004) 11: 816-821; Wang et al. (2003) 14: 2425-2435).","Senescent hADSCs were either cultured in the presence of 10 tM inhibitor of Pol III transcriptional activity, tagetin, for 2 hrs at 37\u00b0 C. (+tagetin) or in the absence of the inhibitor treatment (-tagetin). Nuclear RNA was labeled by addition of 2 mM FUr to the culture for 10 min at 37\u00b0 C. After fixation, cells were immunolabelled with anti-BrdU antibody (red) to detect FUr incorporation sites in combination with anti-53BP1. Double labeling experiment revealed FUr incorporation sites exclusively localized with persistent DNA damage sites throughout entire depth of z-stack images. Tagetin inhibition of Pol III dependent transcription resulted in complete disappearance of FUr incorporation, and loss of compaction of the DNA damage sites as detected by more defuse 53BP1 staining.","Our data revealed that SEN hADSCs treated with the inhibitor of Pol-III tagetin showed impaired 53BP1 focus-forming capacity. Further, comparative strand-specific RT-PCR analysis of a number of SINE\/Alu sequences associated with DNA damage in pericentric areas of chromosomes 10 (designated MIR and Alu) and 21 (designated AluSx and AluJb) revealed that upregulation of Alu transcriptional activity correlates with the presence of persistent DNA damage in SEN samples, as shown for the Alu repeat in the vicinity of the 7.6 kb \u03b3H2AX cluster. Our data demonstrate that, despite the fact that the transcriptional activity of the chosen repeats (AluJb, AluSx, and Alu) is recorded regardless of the state of the hADSC. Alu transcription is upregulated on chromosome 10 where damage is observed only upon cellular senescence. No transcriptional activity from the MIR element was recorded in either the SR or SEN state of hADSCs. These data also support our observation that \u03b3H2AX\/53PB1 foci in SEN hADSC are centers of Pol-III dependent transcriptional activity (). To our surprise, by using a conventional ChIP assay, we observed an increase in the recruitment of a Pol-III transcriptional complex (designated TFIIIC) to the MIR repeat during senescence even in the absence of the MIR-recorded transcription (). This could be due to an ability of TFIIIC to not only be essential for loading PoI-III machinery (Young et al. (1991) 252: 542-546; Kundu et al. (1999) 19: 1605-1615; Noma et al. (2006) 125: 859-872), but also for its ability to participate in the structural organization of chromatin cohesion and condensation within the eukaryotic nucleus as recently demonstrated in yeast (Iwasaki et al. (2010) 21: 254-265).","The presence of large tracts of satellite DNA limits the selection of strand-specific primers for reverse transcription due to a higher degree of sequence complementation, thus preventing further meticulous assessment of multiple, densely-packed SINE\/Alu repeats within and near these genomic loci. Nevertheless, our data, together with multiple reports from yeast and mammals (Lunyak et al. (2007) 317: 248-251; Noma et al. (2006) 125: 859-872; Oki and Kamakaka (2005) 19: 707-716; Scott et al. (2006) 16: 119-129; Willoughby et al. (2000) 275: 759-768), suggest a general role for the Pol-III transcription in genome organization, and support the hypothesis that, similar to centromeric tRNA genes in yeast (Iwasaki et al. (2010) 21: 254-265), pericentric SINE\/Alu retrotransposons in humans may be integral to centromere functioning in self-renewing hADSCs, and upon senescence of hADSCs.","Impaired Cohesin and Condensin I Association with Pericentric \u03b3H2AX Foci in Senescent hADSCs","Multiple lines of evidence indicate that the processes of chromosome repair and segregation are directly linked through cohesin, an evolutionarily-conserved protein complex (Heidinger-Pauli et al. (2008) 31: 47-56; Kim et al. (2002) 99: 1241-1246; Sjogren and Nasmyth (2001) 11: 991-995; Strom et al. (2004) 16: 1003-1015; Unal et al. (2004) 16: 991-1002). Specific association of cohesin with SINE\/Alu repeats was also previously reported (Hakimi et al. (2002) 418L 994-998). The Scc1\/Rad21\/Mcd1 subunit of this complex is central to cohesin function and has been shown to be necessary for sister chromatid cohesion and kinetochore function in vertebrate cells (Sonoda et al. (2001) 1: 759-770), as well as G1 and G2-M DNA damage checkpoints (Jessberger (2009) 28: 2491-2493). Reportedly, the cohesin complex becomes enriched at DSB sites and facilitates DNA repair by homologous recombination (HR) (Bekker-Jensen et al. (2006) 173: 195-206; Potts et al. (2006) 25: 3377-3388). Inefficient postreplicative repair of DSB can arise from a deficiency in either cohesin loading or the conversion of cohesin to its cohesive state by Eco1 (ctf7) (Strom et al. (2004) 16: 1003-1015; Unal et al. (2004) 16: 991-1002; Unal et al. (2007) 317: 245-248). In addition, one cannot overlook the importance of the condensin-related pathway in these events. Condensin loads cohesin onto the chromosome and is known to form two different complexes: condensin I and II (Samoshkin et al. (2009) 4: e6831). Importantly, the condensin I complex has been implicated in the function of pericentric heterochromatin. The depletion of the Cap-H subunit of the condensin I complex results in defects associated with alterations in the structural integrity of centromere-proximal heterochromatin in (Oliveira et al. (2005) 25: 8971-8984). It has been demonstrated that Cap-H depletion does not affect CENP-A incorporation into the centric chromatin or kinetochore assembly, but does result in severe depletion of cohesin Scc1\/Rad21\/Mcd1, triggering cell cycle defects.","We hypothesized that SINE\/Alu transcription in pericentric chromatin can affect condensin and cohesin loading, thus blocking DNA repair by HR. This inability to repair DNA damage will directly or indirectly result in the cessation of the SR capacity of hADSCs by triggering cellular senescence. Therefore, we investigated the recruitment components of the cohesin complex Scc1\/Rad21\/Mcd1, condensin I complex (Cap-H), and histone acetylase Eco1 to the SINE\/Alu repeats in the vicinity of a persistent pericentric \u03b3H2AX cluster on chromosome 10. Data obtained by conventional ChIP analysis demonstrated a statistically significant loss in the recruitment of Scc1\/Rad21\/Mcd1 and Cap-H to the MIR and Alu repeats during senescence of hADSCs (SEN red bars in ) when compared to the status of the same genomic locations in SR cells (SR blue bar in ). Surprisingly, the recruitment of histone acetylase Eco1 was largely correlated with the formation of persistent \u03b3H2AX clusters in senescence, and appeared to be Scc1\/Rad21\/Mcd1-independent. This suggests another, not yet identified, role for the Eco1 protein in DDR, different from its previously reported participation in the modulation of cohesiveness of Scc1\/Rad21\/Mcd1 (Hakimi et al. (2002) 418L 994-998; Strom et al. (2004) 16: 1003-1015; Unal et al. (2004) 16: 991-1002; Unal et al. (2007) 317: 245-248).","Together these data further support the correlation between an increase in SINE\/Alu transcription, and, (i) DNA damage accumulation, (ii) defects in pericentric recruitments of condensin I (Cap-H) and cohesin (Scc1\/Rad21\/Mcd1), and (iii) senescence of hADSCs.","Functional Significance of the SINE\/Alu Transcripts in Senescence of hADSCs","SINE\/Alu elements occupy 6% of the human genome, significantly outnumbering other families of pseudogenes generated by retrotransposition (Weiner et al., 1986). The human SINE\/Alu retrotransposon is a tandem repeat of two B1 elements connected by an A-rich linker (), and the secondary structure of its RNA was previously reported (Sinnett et al. (1991) 266: 8675-8678). In order to evaluate the functional significance of SINE\/Alu transcription in establishing and\/or mediating cellular senescence upon ex-vivo aging, SEN hADSCs were genetically manipulated to stably express shRNA, targeting the generic SINE\/Alu transcript. We took advantage of the common sequence feature of SINE\/Alu retrotransposons to generate a number of lenti-virus-delivered shRNA constructs carrying GFP for assessment of transduction efficiency (; Experimental Procedures).","Knockdown of the generic SINE\/Alu transcript in transduced hADSCs stably expressing shRNA against different portions of SINE\/Alu was demonstrated by Northern Blot hybridization () with a transduction efficiency of nearly 99% (). hADSCs transduced with lentiGFP expressing sh-132Alu RNA demonstrated a near complete knockdown of the generic SINE\/Alu transcript. hADSCs transduced with lentiGFP (control) or lentiGFP sh-193Alu exhibited little or no change at the SINE\/Alu transcription level. Surprisingly, SEN hADSCs lines stably expressing sh-132Alu RNA had a dramatically altered morphology and exhibited an increase in proliferation as detected in [H] thymidine uptake DNA synthesis experiments (). In contrast, SEN hADSCs transduced with lentiGFP (control), or treated with polybrene alone (PB), did not show any significant changes in their proliferation rate when compared with wild type (wt) SEN hADSCs (). Faithful re-establishment of DNA synthesis upon knockdown of the SINE\/Alu transcript was further recorded 96 hours after transduction. Unexpectedly, when compared to wt SEN hADSCs, prior SEN hADSCs lines stably expressing sh-132Alu RNA demonstrated statistically significant upregulation of pluripotency factors Nanog and Oct4 as detected by qPCR analysis (). The levels of Nanog and Oct4 in wt SEN hADSCs were below the threshold of the qPCR method.","In summary, this finding raises the possibility that SINE\/Alu RNA serves as an unexpected but key component of the machinery that controls stem cell pluripotency. Consistently, we observed that prior SEN hADSC with stable knockdown for generic SINE\/Alu RNA were viable, and after 7 days in culture had formed cell aggregates (embryoid-like structures) which tested positive for alkaline phosphatase activity, a marker of induced pluripotency ().","Discussion","Repetitive elements are implicated in changing\/controlling properties of the chromatin in many systems, but few studies have addressed their functional significance in the context of cellular aging, in particular, in human adult stem cells. Several observations have linked persistent DDR to the manifestation of cellular senescence phenotype and ultimately to cell aging. Although cellular senescence is thought to be naturally irreversible cell-cycle arrest induced by DDR signaling, it is still unclear what determines the inability of cells with activated DDR to promptly and properly fix DNA lesions and restore their proliferative capacity. However, what has become clear from the cumulative studies of different models of cellular senescence is that these determinants can vary among cell types and in intensity, duration and nature of the DNA damage.","This raises the possibility that specific genomic regions\/elements exist within cells where DNA damage is less easily repaired, possibly in a manner which is affected by age-dependent changes in chromatin structure and\/or chromosomal organization. These functional elements have not been previously identified. In this study we have addressed this question.","Central to this working hypothesis was our initial effort to perform a location analysis of \u03b3H2AX histone modification, which appears on the nucleosomes adjacent to the sites of DNA lesions. By using ChIP-seq analysis with single nucleosome resolution we were able to map locations of repairable DNA damage, which did not affect the proliferative capacity of hADSCs, and perform a comparative analysis of the dynamic of this damage in the onset of cellular senescence. Based on the distribution of \u03b3H2AX modifications, our data allowed us to conclude that neither repairable nor persistent DNA damage occurs randomly in SR or SEN hADSCs, and both types of damage are biased toward the transposable portion of the genome. Increased transcriptional activity of SINE\/Alu retrotransposons in the aging human adult stem cells significantly correlates with the formation of persistent DNA damage foci, indicating a role in interfering with DNA repair. We provide a mechanistic explanation for retrotransposons' transcriptional interference with DDR: defective recruitment of the cohesin and condensin I complexes to pericentric chromatin upon their activation. Both of these complexes are critical to chromatin organization and damage repair of these genomic regions. We propose a novel model whereby SINE\/Alu transposons act by altering chromatin structure in the vicinity of DNA lesions, thus blocking their repair (). This event is integral to establishing\/mediating persistent DDR upon ex-vivo aging of human adult stem cells.","Although it is still an open question what regulates a transcriptional activity of SINE\/Alu retrotransposons, it is tempting to speculate that this might be due to the loss of epigenetic control for DNA methylation with ex-vivo aging of hADSCs. However, we cannot exclude the possibility that upregulation of the SINE\/Alu transcripts is not directly related to an increase in their transcriptional activity, but rather results from senescence-associated deficiencies in their posttranscriptional processing, turn over or even degradation. Whatever their nature, our finding represents the first example of SINE\/Alu transcription's role in the altering the properties of pericentric chromatin in conjunction with persistent DDR and cellular senescence of human adult stem cells.","Importantly, our results demonstrate a functional significance of SINE\/Alu transcripts in the mechanism of ex-vivo aging of hADSC, and challenge the dogma that cellular senescence, and ultimately cellular aging, is an irreversible process. Since a depletion of generic SINE\/Alu transcripts alone is sufficient to reinstate the proliferative capacity of hADSCs, and to increase the plasticity of previously niche-restricted human adult stem cells, it is plausible to suggest that SINE\/Alu retrotransposons are playing an important role in mesenchymal niche specification through yet unknown, but interconnected with adult stem cell aging, pathways.","Experimental Procedures","Antibodies","Primary antibodies used were: b-actin (Abcam #ab62760), 53BP1 (Bethyl Laboratories #A300-273A), BrdU (BD, 7580), CAP-H (Bethyl Laboratories #A300-603A), CD31 (Invitrogen #HMCD3101), CD44 (Invitrogen #HMCD4401), CD45 (Invitrogen #HMCD4501), CD105 (Invitrogen #HMCD10520), Phospho-cdc2 (Tyr15) (Cell Signaling Technologies #9111), Phospho-Chk1 (Ser345) (CST #2341), Phospho-Chk2 (Thr68) (CST #2661), CENP-A (Abcam #ab13939), human anti-centromeric autoantibody, a-CREST (Antibodies Incorporated #15-235), Eco1 (Bethyl Laboratories #A300-312A), \u03b3H2AX (Millipore #05-636), PML (Santa Cruz Biotechnology #sc-966), and Rad21 (Abcam #ab992). Secondary antibodies were AlexaFlour\u00ae conjugated donkey antibodies (Invitrogen).","Human ADSC Isolation and Expansion","Human adipose derived stem cells were isolated from human subcutaneous white adipose tissue collected during liposuction procedures. The lipoaspirate was suspended in Hank s Buffered Salt Solution (HBSS), 3.5% Bovine Serum Albumin (BSA), 1% Collagenase, type II (Sigma) in 1:3 w\/v ratio and shaken at 37\u00b0 C. for 50 min The cells were filtered through a 70 \u03bcm mesh cell strainer (BD Falcon #352350), treated with Red Blood Cell Lysis buffer (150 mM NHCl, 10 mM KHCO, 0.1 mM EDTA, pH 7.3), and expanded ex-vivo in DMEM\/F12 complete medium (DMEM\/F12, 10% FBS, 100 U\/ml penicillin, 100 \u03bcg\/ml streptomycin, 2.5 \u03bcg\/ml amphotericin B; Invitrogen) in 10% CO2 at 37\u00b0 C. and passaged at 80% confluency, changing medium every 72-96 hours. Cumulative population doublings were calculated by summing the population doublings (PD=log(N\/N)\u00d73.33, where Nis the number of cells plated in the flask and N is the number of cells harvested at this passage) across multiple passages as a function of the number of days it was grown in culture.","Surface Marker Characterization","5\u00d710cells each were incubated for 30 min on ice in the dark with fluorochrome-conjugated antibodies (CD31, CD44, CD45 and CD105; Invitrogen) in PBS with 1% BSA (Sigma), washed and analyzed in a Guava EasyCyte Mini System (Guava Technologies, Millipore). Data analysis was done with FlowJo software (Tree Star, Ashland, Oreg.).","Assessment of Cellular Senescence and Pluripotency Phenotype","Senescence-associated J3-galactosidase activity assay was done as described in manufacturer's kit (BioVision). Cellular phenotype towards induced pluripotent cells was assayed with alkaline phosphatase staining kit (Stemgent).","Immunofluorescence","1-3\u00d710cells\/well in 4-well slides were fixed with 4% paraformaldehyde and permeabilized with PBS, 0.5% Triton X-100. The blocking and antibody incubations were carried out in 4% normal donkey serum (NDS; Jackson Immunochemicals) in PBS. The nuclei were counter-stained with 100 ng\/ml 4\u2032,6-diamidino-2-phenylindole (DAPI; Sigma), and the slides were mounted in ProLong\u00ae Gold antifade aqueous mounting medium (Invitrogen).","Epifluorescence images were acquired on an Olympus BX60 fluorescence microscope with Spotfire 3.2.4 software (Diagnostics Instruments). Confocal images (z-series slice thickness 0.39 \u03bcm) were acquired on Zeiss LSM 510 NLO with 488 nm Argon, 543 nm HeNe, and Coherent Chameleon 2-photon lasers using a 63\u00d7 planapo objective and 0.08\u00d70.08\u00d70.39 \u03bcm voxel dimensions. Image stacks were deconvolved using Huygens Professional 3.4.0 (Scientific Volume Imaging, Netherlands) and visualized in Bitplane Imaris 6.3.1 (Bitplane Inc., Saint Paul, Minn.).","5-Fluorouridine Labeling for RNA Transcript Detection In Situ","Senescent hADSCs, 1\u00d710cells\/well in 4-well slides, were treated with 2 mM 5-Fluorouridine (FUr; Sigma) for 10 minutes. The nuclei were exposed with ice-cold CSK buffer (100 mM KCl, 300 mM sucrose, 10 mM Pipes pH 6.8, 3 mM MgCl2, 1 mM EGTA, 1.2 mM phenylmethylsulfonyl fluoride) with 0.5% Triton X-100 and fixed with 4% paraformaldehyde. Immunostaining was performed as described above for immunoflourescence.","Expression Profiling","Total RNAs from \u02dc1.5\u00d710SR and SEN ADSCs were prepared by TRIZOL\u00ae method (Invitrogen). Human Cell Cycle qRCR Array (SABiosciences #PAHS-020C-2) was used for the profiling (7500 Fast PCR System, Applied Biosystems). Three RNA samples from SR hADSCs were matched to RNA samples for their replicatively SEN counterparts. Statistical significance of the transcriptional differences was calculated by T-test as described in SABiosciences manual and \u201cwww.sabiosciences.com\/RTPCR\u201d. The model-based expression values were calculated and filtered for the genes with higher or lower than two-fold differences in expression between SEN and SR samples. Categories of the genes were annotated using the PANTHER (protein analysis through evolutionary relationships) classification system.","Preparation of Nucleosomal Lysates","3\u00d710SR and SEN hADSCs were washed with PBS, and crosslinked in 1% formaldehyde. The cells were scraped from the plate in CSK buffer with 0.5% Triton X-100, washed, and sonicated in microccocal nuclease buffer (50 mM Tris-HCl, pH 7.9, 5 mM CaCl; NEB) with a Covaris S2 Sonicator (frequency sweep at 1% duty cycle, 5 intensity and 50 cycles per burst for 12 cycles of 10 sec each at 4\u00b0 C.). Sonicates were cenrtrifuged to get rid of the cellular debris, and supernatants were incubated with 25U\/tl of microccocal nuclease (NEB).","ChIP-Seq: Nucleosomal ChIP and SOLiD Library Preparation","Nucleosomal lysates were adjusted to antibody Binding buffer (25 mM Tris-HCl pH 7.9, 0.15% SDS, 1% Triton X-100, 150 mM NaCl), and immunoprecipitated with 10 \u03bcg \u03b3H2AX antibody overnight at 4\u00b0 C. Protein A Sepharose-beads were blocked with 10 \u03bcg\/ml glycogen, 10 \u03bcg\/ml lysozyme, 5 \u03bcg\/mltRNA and 20 \u03bcg\/ml BSA for 1 hour at 4\u00b0 C. before binding to the immuno-complexes for 1 hour at 4\u00b0 C. Beads were washed with Binding buffer and again with the same buffer containing 0.5M NaCl and finally in TE (10 mM Tris-HCl pH 7.6, 1 mM EDTA). Immuno-complexes bound to beads were re-suspended in TE buffer, de-crosslinked, and purified by Proteinase K treatment, phenol:chloroform extraction, and isopropanol precipitation. DNA fragments were prepared for adapter ligation by filling-in ends by DNA polymerase I (Klenow fragment) and phosphorylating 5\u2032 ends of PCR primers by Polynucleotide kinase (NEB), and ligated to 30-fold molar excess of SOLID\u2122 System 2.0 (Applied Biosystems) library adapters according to manufacturer's protocol. DNA libraries were amplified by PCR using cloned Pfu DNA Polymerase (Stratagene, Agilent Technologies. Mononucleosomal size DNA fragments were size-selected in a 2% agarose gel, cut around 200 bp size, and purified with QIAQUICK\u00ae Gel Extraction Kit (Qiagen). Emulsion PCR (ePCR) for sequencing bead preparations were done with 0.05 and 0.1 pg\/\u03bcl of library DNA for each sample. Samples were sequenced on SOLiD\u2122 System 2.0 (Applied Biosystems) according to manufacturer's protocol.","RT PCR Analysis of Retrotransposal Transcription and qPCR","Genomic coordinates of the SINE\/Alu elements tested in RT-PCR were from the March 2006 Assembly (NCBI36\/hgl8) of the Human Genome Browser at UCSC (genome.ucsc.edu); MIR: chr10:41922815-41922906, Alu: chr10:41928992-41929118, AluJb: chr21:10141132-10141429 and AluSx: chr21:10145344-10145644. 100 ng of total RNA was used with the RT2 First Strand Kit (SABiosciences) per reaction. The primers for first strand synthesis are at locations outside of the SINE\/Alu element sequences (external or reverse primers, Table 9) and forward primers within the SINE\/Alu element sequence (internal forward primers, Table 9). RPL13A, GAPDH was used as a positive control.","For real-time quantitative PCR, 1 mg of each total RNA was used for first strand cDNA synthesis with SUPERSCRIPT\u00ae III First-Strand Synthesis System for RT-PCR (Invitrogen) with random priming. qPCR was performed using RT2 SYBR Green qPCR MasterMix (SABiosciences) and run in LIGHTCYCLER\u00ae 480 II (Roche). qPCR primers are listed in Table 9. Data analysis of relative gene expression was done by 2-DCt method.","Lentiviral shRNA Constructs","Lentiviral shRNA constructs to knockdown genetic SINE\/Alu transcript were designed as follows: oligonucleotides Lenti sh-132 Alu RNA: 5\u2032-GAT CCC CCC ACC ACG CCC GGC TAA TTT TCA AGA GAA ATT AGC CGG GCG TGG TGG TTT TTG GAA A-3\u2032 (SEQ ID NO:10) and Lenti sh-193 Alu RNA 5\u2032-GAT CCC CCC CGG GTT CAA GCG ATT CTT TCA AGA GAA GAA TCG CTT GAA CCC GGG TTT TTG GAA A-3\u2032 (SEQ ID NO:11), were annealed with equal amounts of their complementary strands, creating restriction site specific overhangs for cloning, and ligated into HindIII and BglII digested, gel purified pENTR\/pTER+vector (Campeau et al., 2009). The constructs were confirmed by sequencing (sense strand sequence is shown above). Equal amounts of each constructs was mixed with pLenti-CMV-GFP DEST vector (Campeau et al., 2009) in LR Clonase reaction to recombine cloned shRNA production elements into a destination vector according to manufacturer's instructions (Invitrogen). The produced lentiviral plasmid was transformed into Stb13 cells (Invitrogen) for amplification.","Lentiviral Production and Transduction","293T cells were grown in DMEM complete medium (DMEM, high glucose, 10% FBS, 0.1M nonessential amino acid, 6 mM L-glutamine, 1 mM pyruvate, 100 U\/ml penicillin, 100 \u03bcg\/ml streptomycin) for lentiviral production, and transfected for 12 hours with pLenti sh-132Alu or pLenti sh-193Alu and pCgpV, pRSV-Rev, and pCMV-VSV-G helper plasmids (Allele Biotechnology) in 2:1:1:1 molar ratio using Lipofectamine 2000 (Invitrogen) according to standard protocol (Campeau et al., 2009). Medium was collected at 48 and 72 hours and filtered (0.45 \u03bcm). Virus was precipitated with PEG and frozen in aliquots (\u221280\u00b0 C.). Lentiviral transductions were done in complete medium with 5 \u03bcg\/ml Polybrene (Santa Cruz Biotechnology) for 12 hours. Viral titers were determined by comparing GFP positive cells counts to total population.","Proliferation Index","For each condition in [H]-thymidine uptake assay, 10,000 cells were treated with 1 \u03bcCi [H]-thymidine (Perkin-Elmer, Boston, Mass.) in DMEM\/F12 complete for 24 hours. DNA was isolated from harvested cells and quantified with NanoDrop (ND-1000; NanoDrop Technologies Inc). [H]-thymidine uptake into cellular DNA was measured with liquid scintillation counter (LS 6500; Beckman Instruments). Lentiviral transduction was done as above, and untreated and Polybrene treated controls were carried out in parallel.","Northern Hybridization","RNA was isolated from SR and SEN hADSCs (with or without lentiviral transduction) with mirVANA (Ambion, Invitrogen) kit, and 2 mg of total RNA\/lane was run on a 7M Urea, 6% polyacrylamide, TBE. Gel was stained with ethidium bromide, photographed and electroblotted onto Hybond\u2122-N+(Amersham, GE Healthcare). Hybridizations were performed in 6\u00d7SSC, 4\u00d7Denhardts\u2032, 0.1% SDS at 37\u00b0 C. Oligonucleotide probes were labeled with Biotin-16-dUTP (Roche) by terminal transferase (NEB). Northern was visualized with streptavidin-HRP (Invitrogen) using ECL Plus Western Blotting Detection Reagent (Amersham, GE Healthcare) and Amersham Hyperfilm (GE Healthcare). Oligonucleotide used as probes were: SINE\/Alu 132:5\u2032-CCA CCA CGC CCG GCT AAT TT-3\u2032 (SEQ ID NO:12) and SINE\/Alu 90: 5\u2032-CGC GCG CCA CCA CGC CCG GCT AAT TTT TGT ATT TTT AGT AGA GAC GGG GTT TCA CCA TGT TGG CC-3\u2032 (SEQ ID NO:13).","Outlier Removal.","\u03b3H2AX ChIP-seq tags were mapped to the March 2006 human genome reference sequence (NCBI build 36.1, UCSC hg18) using the SOLiD\u2122 System Analysis Pipeline Tool (Corona Lite). After tag-to-genome mapping of the ChIP-Seq data for each biological replicate experiment, tag counts were compared between replicates for identical genomic positions across all chromosomes. To do this, tag counts for each individual nucleotide position in each replicate experiment were determined. Then nucleotide positions that have a tag count of 0 in either replicate were eliminated from consideration. This is because if a position has a tag count of 0 in both data sets, it will not be considered in the subsequent analysis since there is no signal there. In addition, if a position has a 0 tag count in one replicate and a nonzero tag count in the other replicate, then it is considered to be an unreliable position. For nucleotide positions that have non-zero tag counts in both replicates, linear regression was used to compare the tag counts over all nucleotide positions. Based on the regression line and the data distribution around the line, 95% confidence bounds are determined using the Working-Hotelling formula as follows: For each value of x, an expected value of y\u2212E(y)\u2014is estimated based on the linear regression model E(y)=mx+b. Then for each estimated value of E(y), a standard deviation\u2212s(y)\u2014is computed based on the data distribution around E(y) for the corresponding value of x. For each value of E(y), the upper and lower confidence values are computed as E(y)\u00b1w\u00b7s(y) where the constant w is determined based on the F distribution at the confidence level \u03b1 with 2 and n\u22122 degrees of freedom: w=\u221a(2\u00b7F[(1\u2212\u03b1),2,n\u22122]). This regression analysis and confidence bound inference is done for each individual chromosome. Nucleotide positions that fall outside of the confidence bounds determined in this way are removed from further consideration. Replicate experiments were highly consistent.","Noise Reduction and Data Merging.","After removal of outliers, noise reduction was performed in order to empirically determine a tag count threshold value th above which an individual nucleosome sized genomic position (i.e. mono-nucleosome) is considered to be modified. A window size of 200 bp was chosen for this analysis based on 147 bp of DNA in a single nucleosome plus linker sequence. The average tag count in 200 bp windows\u2014\u03bb\u2014is computed as the total number of tags in the genome n divided by the genome length l times the window size of 200: \u03bb=(n\/l)\u00b7200. The Poisson distribution is parameterized by the 200 value in order to model the background noise and determine the tag count signal threshold th for modified nucleosome positions:",{"@attributes":{"id":"p-0193","num":"0192"},"maths":{"@attributes":{"id":"MATH-US-00001","num":"00001"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mi":"th","mo":"=","mrow":{"mi":"min","mo":"\u2062","mrow":{"mo":["{","}"],"mrow":{"mrow":{"mi":"\u03c4","mo":["\u2062","\u2062","\u2062"],"mstyle":[{"mtext":":"},{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}}],"mrow":{"munderover":{"mo":"\u2211","mrow":{"mi":["i","\u03c4"],"mo":"="},"mi":"\u221e"},"mo":"\u2062","mrow":{"mfrac":{"msup":{"mi":["\u03bb","i"]},"mrow":{"mi":"i","mo":"!"}},"mo":"\u2062","msup":{"mi":"\u2147","mrow":{"mo":"-","mi":"\u03bb"}}}}},"mo":"\u2264","mi":"P"}}}}}},"br":{}},"Maximal-Segment Based Clustering Algorithm for Continuous Modified Nucleosome Positions.","In addition to the analysis of mono-nucleosome size fragments of modified genomic DNA described previously, we also investigated clusters of adjacent co-located modified nucleosomes. Clusters were operationally defined as contiguous genomic regions where the number of modified mono-nucleosome size fragments is significantly greater than the average genomic background level of modified positions. To identify and demarcate such clusters, we applied the Maximal Segment algorithm. To do this, we first devised a binary scoring scheme that characterizes mono-nucleosome size fragments (200 bp) as either modified or unmodified. This procedure is used to define a binary genome-wide map of nucleosome scores. Then the Maximal Segment algorithm was applied to the genomic map of binary nucleosome scores to define clusters. The details of the Maximal Segment algorithm are presented elsewhere (Ruzzo and Tompa (1999) 234-241). Below, we describe our scoring scheme for individual nucleosome positions.","Binary Nucleosome Scoring Scheme","A binary scoring scheme is implemented in such a way as to assign the log likelihood that an individual mono-nucleosome size fragment either modified or unmodified. The scores are assigned as: s=ln(q\/p) where p is the density of modified nucleosome positions over the whole genome computed as the number of modified positions divided by the total number of positions and q is the density of modified nucleosome positions in real clusters. There is no way to directly calculate q since the clusters are unknown a priori. In order to estimate q, a Poisson distribution is parameterized with p, the density of modified nucleosome positions over the whole genome. Then, given a confidence level P, a threshold value of q can be chosen using the Poisson distribution as shown:",{"@attributes":{"id":"p-0198","num":"0197"},"maths":{"@attributes":{"id":"MATH-US-00002","num":"00002"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mi":"q","mo":"=","mrow":{"mi":"min","mo":"\u2062","mrow":{"mo":["{","}"],"mrow":{"mrow":{"mi":"\u03c4","mo":["\u2062","\u2062","\u2062"],"mstyle":[{"mtext":":"},{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}}],"mrow":{"munderover":{"mo":"\u2211","mrow":{"mi":["i","\u03c4"],"mo":"="},"mi":"\u221e"},"mo":"\u2062","mrow":{"mfrac":{"msup":{"mi":["\u03bb","i"]},"mrow":{"mi":"i","mo":"!"}},"mo":"\u2062","msup":{"mi":"\u2147","mrow":{"mo":"-","mi":"\u03bb"}}}}},"mo":"\u2264","mi":"P"}}}}}},"br":{}},"This scoring scheme results in an assignment of a single positive score to all modified nucleosome positions and a single negative score to all unmodified positions. This approach is taken based on the proof that log likelihood ratios of this kind are optimal scores for the identification of contiguous genomic segments (Karlin and Altschul (1990) 87: 2264-2268). In addition, the value of q chosen for the scoring scheme allows for control over cluster selection in the sense that the density of modified nucleosomes per cluster will be greater than or equal to the value of q.","Relative Entropy to Define Large Clusters.","After getting the size distributions of contiguous modified nucleosome regions (i.e. clusters), we classified the clusters into 3 groups: 1) mononucleosomal clusters, 2) mid-sized clusters and 3) large clusters. In order to set a size threshold between mid-sized and large clusters, we calculated the relative entropies for different thresholds between the size distributions in self-renewing and in senescent cells (). The threshold is set as the one which gives the maximal relative entropy between the two distributions (). Thus, the difference between fractions of mid-sized and large clusters in the two cell types is maximized.","Comparison with Genomic Features.","The tag-to-genome mapping and the quality control procedures described above (see Outlier removal and Noise reduction and data merging sections) yielded maps of nucleosome positions in the human genome that can be confidently considered to be modified by \u00e3H2AX for self-renewing (SR) and senescent (SEN) hADSCs.","These maps were used to compare the locations of \u00e3H2AX modified nucleosomes to various genomic features of interest within and between SR and SEN cells. For instance, we compared the locations of modified nucleosomes in the genome to the locations of transcription start sites, exons, introns, repetitive sequence elements, pericentric regions and peri-telomeric regions. These comparisons were done using the UCSC Table Browser tool (Karolchik et al. (2004) 32: D493-D496), which is a part of the UCSC Genome Browser suite of tools and genome annotations (Karolchik et al. (2003) 31: 51-54). In addition, custom Perl scripts were used to modify UCSC Genome Browser tracks, such as the gene annotation tracks, to pull out specific features of interest including transcription start site locations and exon\/intron boundaries for further analysis.",{"@attributes":{"id":"p-0205","num":"0204"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}}],"thead":{"row":[{"entry":"TABLE 1"},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":[{},{},{},"Fold up-\/",{}]},{"entry":[{},{},"Gene","down-"]},{"entry":["Biological process","Gene name","Symbol","regulation","P value"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["cell cycle\/cell","cyclin B1","CCNB1","\u22122.66","0.0006"]},{"entry":["cycle control","cyclin B2","CCNB2","\u22123.10","0.0005"]},{"entry":[{},"Cyclin D2","CCND2","2.55","0.0001"]},{"entry":[{},"cyclin F","CCNF","\u22122.40","0.008"]},{"entry":[{},"cell division cycle 2, G1","CDC2","\u22124.51","0.0001"]},{"entry":[{},"to S and G2 to M"]},{"entry":[{},"cell division cycle 20","CDC20","\u22123.15","0.0001"]},{"entry":[{},"homolog ()"]},{"entry":[{},"cyclin-dependent kinase 2","CDK2","\u22122.75","0.0012"]},{"entry":[{},"cyclin-dependent kinase","CDKN3","\u22123.28","0.0029"]},{"entry":[{},"inhibitor 3 (CDK2-"]},{"entry":[{},"associated dual"]},{"entry":[{},"specificity phosphatase)"]},{"entry":[{},"CHK1 checkpoint","CHEK1","\u22123.00","0.008"]},{"entry":[{},"homolog ()"]},{"entry":[{},"retinoblastoma-like 1","RBL1","\u22122.56","0.0012"]},{"entry":[{},"(p107)"]},{"entry":["DNA replication","MCM2","MCM2","\u22123.29","0.0001"]},{"entry":[{},"minichromosome"]},{"entry":[{},"mantenacne deficient 2,"]},{"entry":[{},"mitotin ()"]},{"entry":[{},"MCM3","MCM3","\u22122.33","0.0184"]},{"entry":[{},"minichromosome"]},{"entry":[{},"maintenance deficient 3"]},{"entry":[{},"()"]},{"entry":[{},"MCM4","MCM4","\u22123.44","0.0022"]},{"entry":[{},"minichromosome"]},{"entry":[{},"maintenance deficient 4"]},{"entry":[{},"()"]},{"entry":[{},"MCM5","MCM5","\u22123.57","0.0001"]},{"entry":[{},"minichromosome"]},{"entry":[{},"maintenance deficient 5,"]},{"entry":[{},"cell division cycle 46 ()"]},{"entry":[{},"replication protein A3,","RPA3","\u22122.52","0.0014"]},{"entry":[{},"14 kDa"]},{"entry":[{},"proliferating cell nuclear","PCNA","\u22122.53","0.0009"]},{"entry":[{},"antigen"]},{"entry":["Chromosome","MAD2 mitotic arrest","MAD2L1","\u22124.37","0.0021"]},{"entry":["segregation","deficient-like 1 (yeast)"]},{"entry":[{},"kinetochore associated 1","KNTC1","\u22123.22","0.0049"]},{"entry":["DNA repair","breast cancer 1, early","BRCA1","\u22123.83","0.0032"]},{"entry":[{},"onset"]},{"entry":[{},"breast cancer 2, early","BRCA2","\u22123.32","0.0011"]},{"entry":[{},"onset"]},{"entry":[{},"RAD51 homolog (RecA","RAD51","\u22123.75","0.0005"]},{"entry":[{},"homolog, ) ()"]},{"entry":["mRNA","transcription factor Dp-1","TFDP1","\u22122.21","0.0097"]},{"entry":"transcription"},{"entry":"regulation"},{"entry":["inhibition of","baculoviral IAP repeat","BIRC5","\u22123.70","0.002"]},{"entry":["apoptosis","containing 5 (surviving)"]},{"entry":["stress response","G-2 and S-phase","GTSE1","\u22124.51","0.0017"]},{"entry":[{},"expressed 1"]},{"entry":["Biological process","CDC28 protein kinase","CKS1B","\u22122.35","0.0382"]},{"entry":["unclassified","regulatory subunit 1B"]},{"entry":[{},"CDC28 protein kinase","CKS2","\u22122.42","0.001"]},{"entry":[{},"regulatory subunit 2"]},{"entry":[{},"antigen identified by","MKI67","\u22124.04","0.0053"]},{"entry":[{},"monoclonal antibody Ki-"]},{"entry":[{},"67"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0206","num":"0205"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"\u03b3H2AX ChIP-seq tag and mapping data."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"70pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Experiment","Total tags","Mapped tags","% Mapped tags"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]},{"entry":[{},"SR1","25650863","6427476","25.058%"]},{"entry":[{},"SR2","22864646","6740098","29.478%"]},{"entry":[{},"SR3","23172888","7153067","30.868%"]},{"entry":[{},"SR4","20554101","6610682","32.162%"]},{"entry":[{},"SEN1","32114970","7915256","24.647%"]},{"entry":[{},"SEN2","28469416","7569634","26.589%"]},{"entry":[{},"SEN3","26694829","4911085","18.397%"]},{"entry":[{},"SEN4","23942531","4780551","19.967%"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0207","num":"0206"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Fractions of \u03b3H2AX clusters occupied by genomic features."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Mono-",{},{}]},{"entry":[{},"nucleosomal","Mid-size","Large"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Features","SR","SEN","SR","SEN","SR","SEN"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"Genic Regions"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["All genic","41.5%","41.0%","42.9%","39.7%","47.9%","35.2%"]},{"entry":["regions",{},{},{},{},{},{}]},{"entry":["Promoter","0.75%","0.63%","0.83%","0.61%","0.94%","0.56%"]},{"entry":["Exon","2.86%","2.43%","3.20%","2.14%","4.02%","1.78%"]},{"entry":["Intron","37.9%","38.0%","38.9%","36.9%","42.9%","32.9%"]},{"entry":["Exon-intron","0.19%","0.16%","0.21%","0.14%","0.28%","0.11%"]},{"entry":["Junction",{},{},{},{},{},{}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"Intergenic Regions"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["All Intergenic","59.6%","59.9%","57.8%","61.4%","52.5%","66.8%"]},{"entry":["regions",{},{},{},{},{},{}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"All Transposable Elements (TEs)"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":["All TEs","72.7%","70.6%","69.6%","70.2%","64.4%","65.6%"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"Retrotransposons"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["All","67.3%","65.8%","66.0%","65.8%","63.5%","61.4%"]},{"entry":["Retro-",{},{},{},{},{},{}]},{"entry":["transposons",{},{},{},{},{},{}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"SINE Elements"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["All SINEs","34.9%","30.0%","36.3%","30.9%","37.5%","28.6%"]},{"entry":["Alu","23.4%","17.8%","22.6%","18.8%","20.8%","16.9%"]},{"entry":["MIR","10.4%","11.2%","12.7%","11.2%","15.8%","11.0%"]},{"entry":["SINE-other","1.06%","0.97%","1.03%","0.88%","0.91%","0.71%"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"LINE Elements"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["All LINEs","32.4%","35.8%","29.7%","34.9%","26.0%","32.8%"]},{"entry":["L1","20.3%","23.6%","17.2%","22.8%","12.9%","21.9%"]},{"entry":["L2","10.7%","10.5%","10.9%","10.4%","11.5%","9.19%"]},{"entry":["LINE-other","1.42%","1.68%","1.55%","1.65%","1.64%","1.75%"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"Other Transposable Elements"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["LTR","20.8%","17.7%","18.6%","17.1%","14.9%","15.9%"]},{"entry":["DNA","8.50%","9.15%","8.75%","9.35%","8.70%","8.57%"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0208","num":"0207"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 4"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"\u03b3H2AX tag density compared to chromosome gene density"},{"entry":"and GC content"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"119pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"98pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":["SR","SEN"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Slope","Correlation","P-value","Slope","Correlation","P-value"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Correlation: \u03b3H2AX tag density \u00d7 chromosome gene"},{"entry":"density [R]"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":["\u2003\u2002\u2009\u20030.48","0.64","5.2E\u221204","\u22120.19","\u22120.22","0.16"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Correlation: \u03b3H2AX tag density \u00d7 chromosome"},{"entry":"GC content [R]"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":["2366","0.67","2.6E\u221204","2429","0.54","3.9E\u221203"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Partial correlation: \u03b3H2AX tag density \u00d7 chromosome"},{"entry":"gene density controlling for GC content [r]"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["N\/A","0.49","0.01","N\/A","\u22120.65","1.1E\u221204"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0209","num":"0208"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 5"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Length fractions of genomic features intersected with clusters of \u03b3H2AX"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Mono-nucleosomal","Mid-size","Large"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Features","SR","SEN","SR","SEN","SR","SEN"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}},{"entry":["TSS","0.51%","0.43%","2.29%","1.36%","1.12%","0.20%"]},{"entry":["Exon","0.64%","0.55%","2.90%","1.56%","1.58%","0.21%"]},{"entry":["Intron","0.46%","0.46%","1.91%","1.45%","0.91%","0.21%"]},{"entry":["Junction","0.63%","0.53%","2.81%","1.48%","1.60%","0.19%"]},{"entry":["Intergenic","0.67%","0.68%","2.65%","2.26%","1.04%","0.40%"]},{"entry":["peri-","0.09%","0.09%","0.39%","0.40%","0.73%","0.52%"]},{"entry":["centromere",{},{},{},{},{},{}]},{"entry":["peri-","0.31%","0.21%","1.42%","0.75%","0.29%","0.26%"]},{"entry":["telomere",{},{},{},{},{},{}]},{"entry":["TE","0.75%","0.73%","2.90%","2.35%","1.16%","0.36%"]},{"entry":["intergenic","0.76%","0.74%","2.88%","2.43%","1.04%","0.39%"]},{"entry":["TE",{},{},{},{},{},{}]},{"entry":["genic TE","0.73%","0.71%","2.94%","2.21%","1.36%","0.31%"]},{"entry":["Alu","0.67%","0.51%","2.61%","1.74%","1.04%","0.25%"]},{"entry":["MIR","0.69%","0.75%","3.42%","2.43%","1.84%","0.39%"]},{"entry":["SINE-other","0.77%","0.71%","3.02%","2.08%","1.16%","0.27%"]},{"entry":["L1","0.49%","0.57%","1.68%","1.78%","0.54%","0.28%"]},{"entry":["L2","0.79%","0.78%","3.26%","2.50%","1.49%","0.36%"]},{"entry":["LINE-other","0.65%","0.77%","2.87%","2.45%","1.31%","0.42%"]},{"entry":["LTR","0.81%","0.70%","2.93%","2.17%","1.02%","0.33%"]},{"entry":["DNA","0.65%","0.70%","2.69%","2.31%","1.16%","0.34%"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0210","num":"0209"},"tables":{"@attributes":{"id":"TABLE-US-00006","num":"00006"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 6"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"List of genes with promoter \u03b3H2AX accumulation in SR "},{"entry":"or SEN cells."},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["NM_001001933","LHX8","NM_152418","WDR21C"]},{"entry":["NM_139240","C1orf105","NM_052832","SLC26A7"]},{"entry":["NM_015658","NOC2L","NM_015435","RNF19A"]},{"entry":["NM_198317","KLHL17","NM_030788","TM7SF4"]},{"entry":["NM_148902","TNFRSF18","NM_139166","ABRA"]},{"entry":["NM_080875","MIB2","NM_177531","PKHD1L1"]},{"entry":["NM_033467","MMEL1","NM_021021","SNTB1"]},{"entry":["NM_152492","CCDC27","NM_078480","PUF60"]},{"entry":["NM_207370","GPR153","NM_201384","PLEC1"]},{"entry":["NM_181864","ACOT7","NM_024531","GPR172A"]},{"entry":["NM_006786","UTS2","NM_003923","FOXH1"]},{"entry":["NM_005026","PIK3CD","NM_001029976","ZNF16"]},{"entry":["NM_198544","APITD1","NR_023392","ZNF252"]},{"entry":["NM_001127325","MAD2L2","NM_021240","DMRT3"]},{"entry":["NM_001009611","PRAMEF4","NM_005511","MLANA"]},{"entry":["NM_001018001","KIAA1026","NM_000170","GLDC"]},{"entry":["NM_207348","SLC25A34","NM_144966","FREM1"]},{"entry":["NM_001089591","UQCRHL","NM_017645","FAM29A"]},{"entry":["NM_153213","ARHGEF19","NM_002172","IFNA14"]},{"entry":["NM_017940","NBPF1","NM_078487","CDKN2B"]},{"entry":["NM_012387","PADI4","NM_016410","CHMP5"]},{"entry":["NM_014589","PLA2G2E","NM_007343","PRSS3"]},{"entry":["NM_207334","FAM43B","NM_015297","KIAA1045"]},{"entry":["NM_001113347","ECE1","NM_002732","PRKACG"]},{"entry":["NM_002167","ID3","NM_001007470","TRPM3"]},{"entry":["NM_054016","FUSIP1","NM_021154","PSAT1"]},{"entry":["NM_178122","C1orf201","NM_030940","ISCA1"]},{"entry":["NM_015627","LDLRAP1","NM_005226","S1PR3"]},{"entry":["NM_198137","CATSPER4","NM_145006","SUSD3"]},{"entry":["NM_145345","UBXN11","NM_177995","PTPDC1"]},{"entry":["NM_001039775","AIM1L","NM_003837","FBP2"]},{"entry":["NM_152365","C1orf172","NM_000507","FBP1"]},{"entry":["NM_005281","GPR3","NM_173200","NR4A3"]},{"entry":["NM_001990","EYA3","NM_207299","PRG-3"]},{"entry":["NM_001269","RCC1","NM_000035","ALDOB"]},{"entry":["NM_004437","EPB41","NM_018112","TMEM38B"]},{"entry":["NM_002379","MATN1","NM_006687","ACTL7A"]},{"entry":["NM_004814","SNRNP40","NM_173521","C9orf84"]},{"entry":["NM_023009","MARCKSL1","NM_033051","SLC46A2"]},{"entry":["NM_005268","GJB5","NM_138424","KIF12"]},{"entry":["NM_017629","EIF2C4","NM_017418","1-Dec"]},{"entry":["NM_172313","CSF3R","NM_198188","ASTN2"]},{"entry":["NM_017850","C1orf109","NM_001011649","CDK5RAP2"]},{"entry":["NM_001113482","MANEAL","NM_012164","FBXW2"]},{"entry":["NR_003929","LOC728448","NM_001735","C5"]},{"entry":["NM_152373","ZNF684","NM_007018","CEP110"]},{"entry":["NM_144990","SLFNL1","NM_001004450","OR1B1"]},{"entry":["NM_007102","GUCA2B","NR_003071","SNORD90"]},{"entry":["NM_144626","TMEM125","NM_030978","ARPC5L"]},{"entry":["NM_182517","C1orf210","NM_005833","RABEPK"]},{"entry":["NM_001005417","B4GALT2","NM_006195","PBX3"]},{"entry":["NR_000024","SNORD46","NM_001011703","FAM125B"]},{"entry":["NM_153274","BEST4","NM_203305","FAM102A"]},{"entry":["NM_005897","IPP","NM_032728","PPAPDC3"]},{"entry":["NM_004474","FOXD2","NR_003050","SNORD62A"]},{"entry":["NM_032110","DMRTA2","NM_018956","C9orf9"]},{"entry":["NM_004153","ORC1L","NM_152572","C9orf98"]},{"entry":["NM_001010978","LDLRAD1","NM_002003","FCN1"]},{"entry":["NM_152607","C1orf177","NM_178469","LCN8"]},{"entry":["NM_015888","HOOK1","NM_016219","MAN1B1"]},{"entry":["NM_001083592","ROR1","NM_080877","SLC34A3"]},{"entry":["NM_152489","UBE2U","NM_052817","MID2"]},{"entry":["NM_001924","GADD45A","NM_003662","PIR"]},{"entry":["NM_003838","FPGT","NM_007220","CA5B"]},{"entry":["NM_001112808","TNNI3K","NM_001080975","REPS2"]},{"entry":["NM_001002912","C1orf173","NM_006089","SCML2"]},{"entry":["NM_012152","LPAR3","NM_001037343","CDKL5"]},{"entry":["NM_016620","ZNF644","NR_023358","SCARNA9L"]},{"entry":["NM_001127215","GFI1","NM_001415","EIF2S3"]},{"entry":["NM_007358","MTF2","NM_177404","MAGEB1"]},{"entry":["NM_001114106","SLC44A3","NM_004229","MED14"]},{"entry":["NM_080630","COL11A1","NM_001356","DDX3X"]},{"entry":["NM_001122961","C1orf194","NM_017776","ZNF673"]},{"entry":["NM_172212","CSF1","NR_015378","LOC401588"]},{"entry":["NM_032414","PROK1","NM_001039891","ZNF674"]},{"entry":["NM_001025197","CHI3L2","NM_153280","UBA1"]},{"entry":["NM_024102","WDR77","NM_007130","ZNF41"]},{"entry":["NM_000677","ADORA3","NM_001114123","ELK1"]},{"entry":["NM_033020","TRIM33","NM_007137","ZNF81"]},{"entry":["NM_001002810","PDE4DIP","NM_001037735","ZNF630"]},{"entry":["NM_144697","C1orf51","NM_203475","PORCN"]},{"entry":["NM_182578","THEM5","NM_000377","WAS"]},{"entry":["NM_016190","CRNN","NM_002049","GATA1"]},{"entry":["NM_178354","LCE1F","NM_017602","OTUD5"]},{"entry":["NM_001128600","LCE6A","NM_007075","WDR45"]},{"entry":["NM_001024209","SPRR2E","NM_015698","GPKOW"]},{"entry":["NM_000427","LOR","NM_020717","SHROOM4"]},{"entry":["NM_005979","S100A13","NM_005448","BMP15"]},{"entry":["NM_004515","ILF2","NM_001010862","SPIN3"]},{"entry":["NM_173852","KRTCAP2","NM_138737","HEPH"]},{"entry":["NM_001039517","C1orf104","NM_014725","STARD8"]},{"entry":["NM_006912","RIT1","NM_207320","OTUD6A"]},{"entry":["NM_006365","C1orf61","NM_002565","P2RY4"]},{"entry":["NM_021948","BCAN","NM_004312","ARR3"]},{"entry":["NM_001004469","OR10J5","NM_007363","NONO"]},{"entry":["NM_001013661","VSIG8","NR_002226","INGX"]},{"entry":["NM_012337","CCDC19","NM_001013627","NHSL2"]},{"entry":["NM_001001734","ATP1A4","NM_017669","ERCC6L"]},{"entry":["NM_052931","SLAMF6","NR_003255","TSIX"]},{"entry":["NM_003037","SLAMF1","NM_015975","TAF9B"]},{"entry":["NM_017625","ITLN1","NM_153252","BRWD3"]},{"entry":["NM_030916","PVRL4","NM_000307","POU3F4"]},{"entry":["NM_053282","SH2D1B","NM_033048","CPXCR1"]},{"entry":["NM_178550","C1orf110","NM_006729","DIAPH2"]},{"entry":["NM_018417","ADCY10","NM_022144","TNMD"]},{"entry":["NM_001460","FMO2","NM_021637","TMEM35"]},{"entry":["NM_001531","MR1","NM_022053","NXF2"]},{"entry":["NM_005562","LAMC2","NM_182541","TMEM31"]},{"entry":["NM_002597","PDC","NM_017416","IL1RAPL2"]},{"entry":["NM_005298","GPR25","NM_138382","RIPPLY1"]},{"entry":["NM_003281","TNNI1","NM_018301","RBM41"]},{"entry":["NM_001048230","ADORA1","NM_173494","CXorf41"]},{"entry":["NM_018208","ETNK2","NM_002764","PRPS1"]},{"entry":["NM_030952","NUAK2","NM_001522","GUCY2F"]},{"entry":["NM_181644","MFSD4","NM_001004308","ZCCHC16"]},{"entry":["NM_134325","SLC26A9","NM_001031855","LONRF3"]},{"entry":["NR_004389","SNORA16B","NM_001152","SLC25A5"]},{"entry":["NM_206594","ESRRG","NM_181777","UBE2A"]},{"entry":["NR_001587","AURKAPS1","NM_017544","NKRF"]},{"entry":["NM_024709","C1orf115","NM_004541","NDUFA1"]},{"entry":["NM_014184","CNIH4","NM_006978","RNF113A"]},{"entry":["NM_033445","HIST3H2A","NM_006777","ZBTB33"]},{"entry":["NM_004578","RAB4A","NM_003588","CUL4B"]},{"entry":["NM_152379","C1orf131","NM_001008222","ZDHHC9"]},{"entry":["NM_022051","EGLN1","NM_001077188","HS6ST2"]},{"entry":["NM_032435","KIAA1804","NM_004484","GPC3"]},{"entry":["NM_021186","ZP4","NM_021796","PLAC1"]},{"entry":["NM_024804","ZNF669","NM_001013403","LOC347487"]},{"entry":["NM_175911","OR2L13","NM_004065","CDR1"]},{"entry":["NM_019044","CCDC93","NM_173078","SLITRK4"]},{"entry":["NM_152670","C2orf51","NM_001011544","MAGEA11"]},{"entry":["NM_001040649","ACP1","NM_005342","HMGB3"]},{"entry":["NM_015677","SH3YL1","NM_004224","GPR50"]},{"entry":["NM_175722","TPO","NM_153478","CSAG1"]},{"entry":["NM_138799","MBOAT2","NM_005367","MAGEA12"]},{"entry":["NM_033090","GREB1","NM_007150","ZNF185"]},{"entry":["NM_012344","NTSR2","NM_001001344","ATP2B3"]},{"entry":["NM_182828","GDF7","NM_001025243","IRAK1"]},{"entry":["NM_181713","UBXN2A","NR_000011","SNORA70"]},{"entry":["NM_001040710","C2orf84","NM_004728","DDX21"]},{"entry":["NM_153759","DNMT3A","NM_020397","CAMK1D"]},{"entry":["NM_017877","C2orf18","NM_022365","DNAJC1"]},{"entry":["NM_007046","EMILIN1","NM_020824","ARHGAP21"]},{"entry":["NM_032434","ZNF512","NM_014915","ANKRD26"]},{"entry":["NM_020744","MTA3","NM_052997","ANKRD30A"]},{"entry":["NM_022436","ABCG5","NM_003421","ZNF37A"]},{"entry":["NM_001003937","TSPYL6","NM_145312","ZNF485"]},{"entry":["NM_014562","OTX1","NM_014696","GPRIN2"]},{"entry":["NM_006759","UGP2","NM_001098512","PRKG1"]},{"entry":["NM_152792","ASPRV1","NM_022079","HERC4"]},{"entry":["NM_001692","ATP6V1B1","NM_030625","TET1"]},{"entry":["NM_020459","PAIP2B","NM_014767","SPOCK2"]},{"entry":["NM_144579","SFXN5","NM_001114133","SYNPO2L"]},{"entry":["NM_003960","NAT8","NR_002724","MBL1P1"]},{"entry":["NM_001615","ACTG2","NM_032372","DYDC2"]},{"entry":["NM_021196","SLC4A5","NM_138812","DYDC1"]},{"entry":["NM_016170","TLX2","NM_006829","C10orf116"]},{"entry":["NM_021103","TMSB10","NM_148978","PANK1"]},{"entry":["NM_031283","TCF7L1","NM_006413","RPP30"]},{"entry":["NM_017952","PTCD3","NM_004523","KIF11"]},{"entry":["NR_003503","GGT8P","NM_015385","SORBS1"]},{"entry":["NM_002371","MAL","NM_020349","ANKRD2"]},{"entry":["NM_013434","KCNIP3","NM_138413","C10orf65"]},{"entry":["NM_001037228","LOC285033","NM_001009997","C10orf62"]},{"entry":["NM_001079","ZAP70","NM_031212","SLC25A28"]},{"entry":["NM_138798","MITD1","NM_003988","PAX2"]},{"entry":["NM_001025108","AFF3","NM_013274","POLL"]},{"entry":["NM_173343","IL1R2","NM_002779","PSD"]},{"entry":["NM_173205","IL1F7","NM_002925","RGS10"]},{"entry":["NM_173161","IL1F10","NM_024834","C10orf119"]},{"entry":["NM_173842","IL1RN","NR_003570","FLJ46361"]},{"entry":["NM_012184","FOXD4L1","NM_001105574","HMX3"]},{"entry":["NM_002830","PTPN4","NM_001007793","BUB3"]},{"entry":["NM_032740","SFT2D3","NM_153442","GPR26"]},{"entry":["NM_138770","CCDC74A","NM_000375","UROS"]},{"entry":["NM_144586","LYPD1","NM_018180","DHX32"]},{"entry":["NM_019845","RPRM","NM_001380","DOCK1"]},{"entry":["NM_001009959","ERMN","NM_006426","DPYSL4"]},{"entry":["NM_002349","LY75","NM_003577","UTF1"]},{"entry":["NM_173162","KCNH7","NM_001098483","C10orf125"]},{"entry":["NM_000523","HOXD13","NM_145651","SCGB1C1"]},{"entry":["NM_012086","GTF3C3","NM_001039490","CD151"]},{"entry":["NM_199440","HSPD1","NM_138567","SYT8"]},{"entry":["NM_001228","CASP8","NM_001013254","LSP1"]},{"entry":["NM_001102659","LOC200726","NM_199292","TH"]},{"entry":["NM_005210","CRYGB","NM_014555","TRPM5"]},{"entry":["NM_000634","IL8RA","NM_001039165","MRGPRE"]},{"entry":["NM_007127","VIL1","NM_003141","TRIM21"]},{"entry":["NM_001105537","ZNF142","NM_001004137","OR52M1"]},{"entry":["NM_003936","CDK5R2","NM_001004757","OR51Q1"]},{"entry":["NM_001005176","SP140","NR_002777","TRIMP1"]},{"entry":["NM_030926","ITM2C","NM_176875","CCKBR"]},{"entry":["NM_031313","ALPPL2","NM_012192","FXC1"]},{"entry":["NM_005199","CHRNG","NM_016229","CYB5R2"]},{"entry":["NM_007120","UGT1A4","NM_001003745","OR10A3"]},{"entry":["NM_018410","HJURP","NM_005418","ST5"]},{"entry":["NM_001005853","OR6B2","NM_001031853","INSC"]},{"entry":["NM_001080835","LOC643905","NM_017508","SOX6"]},{"entry":["NM_005301","GPR35","NM_000525","KCNJ11"]},{"entry":["NM_015148","PASK","NM_002478","MYOD1"]},{"entry":["NM_002712","PPP1R7","NM_194285","SPTY2D1"]},{"entry":["NM_175727","IL5RA","NM_003986","BBOX1"]},{"entry":["NM_020873","LRRN1","NM_001031854","ACCSL"]},{"entry":["NM_033337","CAV3","NM_001031730","TSPAN18"]},{"entry":["NM_014850","SRGAP3","NM_000256","MYBPC3"]},{"entry":["NM_033084","FANCD2","NM_001004727","OR4X2"]},{"entry":["NM_003178","SYN2","NM_001005201","OR8H3"]},{"entry":["NM_005037","PPARG","NM_001004064","OR8J3"]},{"entry":["NM_025265","TSEN2","NM_001004746","OR5T2"]},{"entry":["NM_012298","CAND2","NM_001005210","LRRC55"]},{"entry":["NM_001007073","RPL32","NM_000062","SERPING1"]},{"entry":["NM_024827","HDAC11","NM_145008","YPEL4"]},{"entry":["NM_001080423","GRIP2","NM_005838","GLYAT"]},{"entry":["NM_020839","WDR48","NM_001004706","OR4D11"]},{"entry":["NM_001337","CX3CR1","NM_152716","PATL1"]},{"entry":["NM_005201","CCR8","NM_178031","TMEM132A"]},{"entry":["NM_001098414","ZNF621","NM_152718","VWCE"]},{"entry":["NM_020707","HHATL","NM_021727","FADS3"]},{"entry":["NM_173658","ZNF660","NM_020238","INCENP"]},{"entry":["NM_145044","ZNF501","NM_006473","TAF6L"]},{"entry":["NM_013270","TSP50","NM_199337","TMEM179B"]},{"entry":["NM_182702","TESSP2","NM_198897","FIBP"]},{"entry":["NM_001008269","TMEM89","NM_178864","NPAS4"]},{"entry":["NM_000884","IMPDH2","NM_003793","CTSF"]},{"entry":["NM_001640","APEH","NM_014578","RHOD"]},{"entry":["NM_007024","TMEM115","NM_017857","SSH3"]},{"entry":["NM_152397","IQCF1","NM_000695","ALDH3B2"]},{"entry":["NM_144641","PPM1M","NM_002335","LRP5"]},{"entry":["NM_020163","SEMA3G","NM_005247","FGF3"]},{"entry":["NM_002218","ITIH4","NM_000803","FOLR2"]},{"entry":["NM_205853","MUSTN1","NM_033388","ATG16L2"]},{"entry":["NM_001126128","PROK2","NM_004154","P2RY6"]},{"entry":["NM_001105580","GABRR3","NM_021200","PLEKHB1"]},{"entry":["NM_001042459","FILIP1L","NM_004041","ARRB1"]},{"entry":["NM_032787","GPR128","NM_015516","TSKU"]},{"entry":["NM_014981","MYH15","NM_182833","GDPD4"]},{"entry":["NM_001008273","TAGLN3","NM_153696","PSMAL"]},{"entry":["NM_001025073","C3orf17","NM_001105522","LOC729384"]},{"entry":["NM_152538","IGSF11","NM_033135","PDGFD"]},{"entry":["NM_005191","CD80","NM_002906","RDX"]},{"entry":["NM_175924","ILDR1","NM_015191","SIK2"]},{"entry":["NM_182628","CCDC37","NM_001101389","LOC644672"]},{"entry":["NM_013336","SEC61A1","NM_152315","FAM55A"]},{"entry":["NM_153330","DNAJB8","NM_001077263","TMPRSS13"]},{"entry":["NM_020187","C3orf37","NM_001467","SLC37A4"]},{"entry":["NM_001042384","CEP63","NM_024791","PDZD3"]},{"entry":["NM_004189","SOX14","NM_004205","USP2"]},{"entry":["NM_001002026","CLDN18","NM_001005197","OR8D4"]},{"entry":["NM_016161","A4GNT","NM_001002918","OR8D2"]},{"entry":["NM_004766","COPB2","NM_017425","SPA17"]},{"entry":["NM_080862","SPSB4","NM_025004","CCDC15"]},{"entry":["NM_139209","GRK7","NM_022112","P53AIP1"]},{"entry":["NM_013308","GPR171","NM_138342","GLB1L2"]},{"entry":["NM_014575","SCHIP1","NM_003044","SLC6A12"]},{"entry":["NM_000882","IL12A","NM_001039029","LRTM2"]},{"entry":["NM_173084","TRIM59","NM_020996","FGF6"]},{"entry":["NM_004122","GHSR","NM_001769","CD9"]},{"entry":["NM_130770","HTR3C","NM_020400","LPAR5"]},{"entry":["NM_001100120","ECE2","NM_002075","GNB3"]},{"entry":["NM_001102416","KNG1","NM_002543","OLR1"]},{"entry":["NM_130834","OPA1","NM_020853","KIAA1467"]},{"entry":["NM_152531","C3orf21","NM_005504","BCAT1"]},{"entry":["NM_138461","TM4SF19","NM_001098531","RAPGEF3"]},{"entry":["NM_182524","ZNF595","NM_014554","SENP1"]},{"entry":["NM_001039127","ZNF718","NM_012272","PRPF40B"]},{"entry":["NM_017733","PIGG","NM_001971","ELA1"]},{"entry":["NM_022042","SLC26A1","NM_175053","KRT74"]},{"entry":["NM_003023","SH3BP2","NM_153633","HOXC4"]},{"entry":["NM_198229","RGS12","NM_006163","NFE2"]},{"entry":["NM_001113361","TBC1D14","NM_020370","GPR84"]},{"entry":["NM_001001290","SLC2A9","NM_006741","PPP1R1A"]},{"entry":["NM_004787","SLIT2","NM_021191","NEUROD4"]},{"entry":["NM_147183","KCNIP4","NM_002206","ITGA7"]},{"entry":["NM_015187","KIAA0746","NM_002429","MMP19"]},{"entry":["NM_018302","C4orf19","NM_022465","IKZF4"]},{"entry":["NM_207406","BEND4","NM_012064","MIP"]},{"entry":["NM_198353","KCTD8","NR_003046","SNORD59B"]},{"entry":["NM_000809","GABRA4","NM_148897","SDR9C7"]},{"entry":["NM_032622","LNX1","NM_024779","PIP4K2C"]},{"entry":["NM_032495","HOPX","NM_001478","B4GALNT1"]},{"entry":["NM_020368","UTP3","NM_000075","CDK4"]},{"entry":["NM_000584","IL8","NM_015026","MON2"]},{"entry":["NM_033214","GK2","NM_022496","ACTR6"]},{"entry":["NM_001003810","HNRNPD","NM_016053","CCDC53"]},{"entry":["NM_001098540","HPSE","NM_006825","CKAP4"]},{"entry":["NM_032717","AGPAT9","NM_002920","RFX4"]},{"entry":["NM_138980","MAPK10","NM_203436","ASCL4"]},{"entry":["NM_178135","HSD17B13","NM_057169","GIT2"]},{"entry":["NM_198281","GPRIN3","NM_002710","PPP1CC"]},{"entry":["NM_183049","TMSL3","NM_001034025","ERP29"]},{"entry":["NM_000671","ADH5","NM_181578","RFC5"]},{"entry":["NM_000670","ADH4","NM_002859","PXN"]},{"entry":["NM_000667","ADH1A","NM_015918","POP5"]},{"entry":["NM_000673","ADH7","NM_004276","CABP1"]},{"entry":["NM_033430","PDE5A","NM_000545","HNF1A"]},{"entry":["NM_152778","MFSD8","NM_016237","ANAPC5"]},{"entry":["NM_000910","NPY2R","NM_207437","DNAH10"]},{"entry":["NM_005651","TDO2","NM_144669","GLT1D1"]},{"entry":["NM_005038","PPID","NM_183044","RNF6"]},{"entry":["NM_017923","1-Mar","NM_017826","SOHLH2"]},{"entry":["NM_006792","MORF4","NM_001009814","KIAA0564"]},{"entry":["NR_003612","FAM92A3","NM_002498","NEK3"]},{"entry":["NM_207352","CYP4V2","NM_001011724","HNRNPA1L2"]},{"entry":["NM_000128","F11","NM_022843","PCDH20"]},{"entry":["NM_013232","PDCD6","NM_006346","PIBF1"]},{"entry":["NM_024786","ZDHHC11","NM_014305","TGDS"]},{"entry":["NM_001044","SLC6A3","NM_019616","F7"]},{"entry":["NM_020227","PRDM9","NM_000504","F10"]},{"entry":["NM_018356","C5orf22","NM_198235","RNASE1"]},{"entry":["NM_005983","SKP2","NM_201535","NDRG2"]},{"entry":["NM_001007527","LMBRD2","NM_001012264","RNASE13"]},{"entry":["NM_018034","WDR70","NM_007192","SUPT16H"]},{"entry":["NM_152403","EGFLAM","NM_001344","DAD1"]},{"entry":["NM_000163","GHR","NM_000359","TGM1"]},{"entry":["NM_022483","C5orf28","NM_138452","DHRS1"]},{"entry":["NM_022132","MCCC2","NM_032594","INSM2"]},{"entry":["NM_004291","CARTPT","NM_032352","BRMS1L"]},{"entry":["NM_153217","TMEM174","NM_003317","NKX2-1"]},{"entry":["NM_001004441","ANKRD34B","NM_001202","BMP4"]},{"entry":["NM_173061","CAST","NM_015589","SAMD4A"]},{"entry":["NM_001962","EFNA5","NM_014992","DAAM1"]},{"entry":["NM_022828","YTHDC2","NM_004857","AKAP5"]},{"entry":["NM_014350","TNFAIP8","NM_001039465","SFRS5"]},{"entry":["NM_153223","CEP120","NM_015351","TTC9"]},{"entry":["NM_001017974","P4HA2","NM_018228","C14orf115"]},{"entry":["NM_021982","SEC24A","NM_002632","PGF"]},{"entry":["NM_145282","LOC153328","NM_004452","ESRRB"]},{"entry":["NM_015564","LRRTM2","NM_024496","C14orf4"]},{"entry":["NM_001033112","PAIP2","NM_033426","KIAA1737"]},{"entry":["NM_016459","MGC29506","NM_145870","GSTZ1"]},{"entry":["NM_001077693","ECSCR","NM_015859","GTF2A1"]},{"entry":["NM_032289","PSD2","NM_001275","CHGA"]},{"entry":["NM_018911","PCDHA8","NM_022151","MOAP1"]},{"entry":["NM_031857","PCDHA9","NM_006215","SERPINA4"]},{"entry":["NM_031883","PCDHAC2","NR_003234","SNORD113-6"]},{"entry":["NM_018936","PCDHB2","NR_003196","SNORD114-4"]},{"entry":["NM_031947","SLC25A2","NR_003316","SNORD116-1"]},{"entry":["NM_004576","PPP2R2B","NR_003324","SNORD116-3"]},{"entry":["NM_001040174","HTR4","NR_003330","SNORD116-15"]},{"entry":["NM_024577","SH3TC2","NR_003311","SNORD115-17"]},{"entry":["NM_133263","PPARGC1B","NR_003313","SNORD115-20"]},{"entry":["NM_001804","CDX1","NR_003496","HBII-52-27"]},{"entry":["NM_130899","FAM71B","NR_003350","SNORD115-35"]},{"entry":["NM_001037333","CYFIP2","NR_003353","SNORD115-38"]},{"entry":["NM_007017","SOX30","NR_003355","SNORD115-40"]},{"entry":["NM_000816","GABRG2","NM_017762","MTMR10"]},{"entry":["NM_199246","CCNG1","NM_001103184","FMN1"]},{"entry":["NM_145266","NUDCD2","NM_005135","SLC12A6"]},{"entry":["NM_001102609","LOC133874","NM_170675","MEIS2"]},{"entry":["NM_001029886","PFN3","NM_152453","TMCO5A"]},{"entry":["NM_019057","FLJ10404","NM_033286","C15orf23"]},{"entry":["NM_001024649","CANX","NM_133639","RHOV"]},{"entry":["NM_201627","TRIM41","NM_020857","VPS18"]},{"entry":["NR_002592","SNORD96A","NM_016642","SPTBN5"]},{"entry":["NM_138296","PTCRA","NM_022473","ZFP106"]},{"entry":["NM_018303","EXOC2","NM_000119","EPB42"]},{"entry":["NM_004568","SERPINB6","NM_004245","TGM5"]},{"entry":["NM_205864","CAGE1","NM_014985","CEP152"]},{"entry":["NM_153005","RIOK1","NM_016194","GNB5"]},{"entry":["NR_004855","HULC","NM_130810","DYX1C1"]},{"entry":["NM_153003","OFCC1","NM_012182","FOXB1"]},{"entry":["NM_003220","TFAP2A","NM_004663","RAB11A"]},{"entry":["NM_016462","TMEM14C","NM_016166","PIAS1"]},{"entry":["NM_207582","HERV-FRD","NM_022369","STRA6"]},{"entry":["NM_138574","HDGFL1","NM_020447","C15orf17"]},{"entry":["NM_014722","FAM65B","NM_173469","UBE2Q2"]},{"entry":["NM_170610","HIST1H2BA","NM_003978","PSTPIP1"]},{"entry":["NM_006355","TRIM38","NM_022566","MESDC1"]},{"entry":["NM_003523","HIST1H2BE","NM_201651","SLC28A1"]},{"entry":["NM_003532","HIST1H3E","NM_000326","RLBP1"]},{"entry":["NM_001732","BTN1A1","NM_198925","SEMA4B"]},{"entry":["NM_003511","HIST1H2AL","NM_002569","FURIN"]},{"entry":["NM_005322","HIST1H1B","NM_018668","VPS33B"]},{"entry":["NM_003447","ZNF165","NM_006011","ST8SIA2"]},{"entry":["NM_021253","TRIM39","NM_001102612","LOC145814"]},{"entry":["NM_005803","FLOT1","NM_016310","POLR3K"]},{"entry":["NR_003948","HCG22","NM_016541","GNG13"]},{"entry":["NM_000595","LTA","NM_207419","C1QTNF8"]},{"entry":["NR_002971","SNORA38","NM_024164","TPSB2"]},{"entry":["NM_025258","C6orf27","NM_023076","UNKL"]},{"entry":["NM_006929","SKIV2L","NM_004970","IGFALS"]},{"entry":["NM_033554","HLA-DPA1","NR_002327","SNORA10"]},{"entry":["NM_005453","ZBTB22","NM_174903","RNF151"]},{"entry":["NM_018679","TCP11","NM_005262","GFER"]},{"entry":["NM_153487","MDGA1","NM_022372","GBL"]},{"entry":["NM_031460","KCNK17","NM_182563","C16orf79"]},{"entry":["NM_024807","TREML2","NM_001374","DNASE1L2"]},{"entry":["NM_018643","TREM1","NM_001919","DCI"]},{"entry":["NM_002630","PGC","NM_001089","ABCA3"]},{"entry":["NM_000322","PRPH2","NM_007108","TCEB2"]},{"entry":["NM_003192","TBCC","NM_172229","KREMEN2"]},{"entry":["NM_023932","DLK2","NR_002169","OR1F2"]},{"entry":["NM_021572","ENPP5","NM_138440","VASN"]},{"entry":["NM_005588","MEP1A","NM_000833","GRIN2A"]},{"entry":["NM_000847","GSTA3","NM_170664","OTOA"]},{"entry":["NM_004282","BAG2","NM_173806","C16orf65"]},{"entry":["NM_183227","RAB23","NM_006043","HS3ST2"]},{"entry":["NM_078485","COL9A1","NM_032038","SPNS1"]},{"entry":["NM_014989","RIMS1","NM_006319","CDIPT"]},{"entry":["NM_002395","ME1","NM_152458","ZNF785"]},{"entry":["NM_016230","CYB5R4","NM_001080417","ZNF629"]},{"entry":["NM_000865","HTR1E","NM_000887","ITGAX"]},{"entry":["NM_001042493","C6orf162","NM_016633","ERAF"]},{"entry":["NM_020320","RARS2","NM_145186","ABCC11"]},{"entry":["NM_002042","GABRR1","NM_017839","LPCAT2"]},{"entry":["NM_015891","CDC40","NM_005949","MT1F"]},{"entry":["NM_001013734","RFPL4B","NM_005950","MT1G"]},{"entry":["NM_173560","RFX6","NM_153837","GPR114"]},{"entry":["NM_001029858","SLC35F1","NM_170776","GPR97"]},{"entry":["NM_020755","SERINC1","NM_001297","CNGB1"]},{"entry":["NM_033515","ARHGAP18","NM_001080492","KLKBL4"]},{"entry":["NM_022121","PERP","NM_020465","NDRG4"]},{"entry":["NM_020340","KIAA1244","NM_001796","CDH8"]},{"entry":["NM_015439","CCDC28A","NM_173815","CES8"]},{"entry":["NM_033071","SYNE1","NM_014329","EDC4"]},{"entry":["NM_014892","RBM16","NM_002801","PSMB10"]},{"entry":["NM_207118","GTF2H5","NM_012320","PLA2G15"]},{"entry":["NM_138810","TAGAP","NM_032178","SLC7A6OS"]},{"entry":["NM_016098","BRP44L","NM_018667","SMPD3"]},{"entry":["NR_002787","LOC154449","NM_138612","HAS3"]},{"entry":["NM_001098201","GPER","NM_030579","CYB5B"]},{"entry":["NM_007353","GNA12","NM_006927","ST3GAL2"]},{"entry":["NR_015343","LOC389458","NM_138994","CNTNAP4"]},{"entry":["NM_014038","BZW2","NM_014861","ATP2C2"]},{"entry":["NR_022006","KIAA0087","NM_031476","CRISPLD2"]},{"entry":["NM_000823","GHRHR","NM_152287","ZNF276"]},{"entry":["NM_006658","C7orf16","NM_145068","TRPV3"]},{"entry":["NM_207173","NPSR1","NM_144611","CYB5D2"]},{"entry":["NM_181791","GPR141","NM_001140","ALOX15"]},{"entry":["NM_001105282","FAM183B","NM_003562","SLC25A11"]},{"entry":["NM_021223","MYL7","NM_020795","NLGN2"]},{"entry":["NM_013389","NPC1L1","NM_001251","CD68"]},{"entry":["NR_002990","SNORA5B","NM_153007","ODF4"]},{"entry":["NM_021116","ADCY1","NM_001010855","PIK3R6"]},{"entry":["NM_000790","DDC","NM_004822","NTN1"]},{"entry":["NM_182546","VSTM2A","NM_139162","SMCR7"]},{"entry":["NM_016220","ZNF107","NM_144775","SMCR8"]},{"entry":["NM_015852","ZNF117","NM_004618","TOP3A"]},{"entry":["NM_031468","CALN1","NM_007148","RNF112"]},{"entry":["NM_002835","PTPN12","NM_001080837","SEBOX"]},{"entry":["NM_012395","PFTK1","NM_003593","FOXN1"]},{"entry":["NM_019004","ANKIB 1","NM_031934","RAB34"]},{"entry":["NM_004912","KRIT1","NR_000014","SNORD42A"]},{"entry":["NM_006528","TFPI2","NM_020791","TAOK1"]},{"entry":["NM_018842","BAIAP2L1","NM_015986","CRLF3"]},{"entry":["NM_003496","TRRAP","NM_015544","TMEM98"]},{"entry":["NM_057096","CYP3A43","NM_001094","ACCN1"]},{"entry":["NM_001185","AZGP1","NM_145272","C17orf50"]},{"entry":["NM_006833","COPS6","NM_024864","MRM1"]},{"entry":["NM_003227","TFR2","NM_000458","HNF1B"]},{"entry":["NM_002291","LAMB1","NM_003673","TCAP"]},{"entry":["NM_182597","C7orf53","NM_021724","NR1D1"]},{"entry":["NM_003391","WNT2","NM_019010","KRT20"]},{"entry":["NM_016087","WNT16","NM_031961","KRTAP9-2"]},{"entry":["NM_002851","PTPRZ1","NM_001524","HCRT"]},{"entry":["NM_001024613","FEZF1","NM_001991","EZH1"]},{"entry":["NM_003117","SPAM1","NM_080863","ASB16"]},{"entry":["NM_006193","PAX4","NM_145663","DBF4B"]},{"entry":["NR_002187","LOC286016","NM_005892","FMNL1"]},{"entry":["NM_001869","CPA2","NM_199282","ARHGAP27"]},{"entry":["NM_001105543","PLXNA4","NM_175882","IMP5"]},{"entry":["NM_002825","PTN","NM_001007532","STH"]},{"entry":["NM_001085429","TMEM213","NM_024320","ATAD4"]},{"entry":["NM_032295","SLC37A3","NM_152244","SNX11"]},{"entry":["NM_176817","TAS2R38","NM_000023","SGCA"]},{"entry":["NM_019841","TRPV5","NM_001267","CHAD"]},{"entry":["NM_001031690","FAM131B","NM_004655","AXIN2"]},{"entry":["NM_175571","GIMAP8","NM_000346","SOX9"]},{"entry":["NM_014020","TMEM176B","NM_000835","GRIN2C"]},{"entry":["NM_144727","CRYGN","NM_015353","KCTD2"]},{"entry":["NM_001040633","PRKAG2","NM_001005849","SUMO2"]},{"entry":["NM_033225","CSMD1","NM_001005619","ITGB4"]},{"entry":["NM_002052","GATA4","NM_180990","ZACN"]},{"entry":["NM_000237","LPL","NM_006640","9-Sep"]},{"entry":["NM_001114137","EPB49","NM_001042573","FLJ21865"]},{"entry":["NM_144962","PEBP4","NM_178543","ENPP7"]},{"entry":["NM_007257","PNMA2","NM_024110","CARD14"]},{"entry":["NM_171982","TRIM35","NM_181483","C18orf1"]},{"entry":["NM_013959","NRG1","NM_003799","RNMT"]},{"entry":["NM_152413","GOT1L1","NM_020805","KLHL14"]},{"entry":["NR_003129","RNF5P1","NM_032980","DTNA"]},{"entry":["NR_001569","tMDC","NM_018170","RPRD1A"]},{"entry":["NM_014420","DKK4","NM_001008239","C18orf25"]},{"entry":["NM_002027","FNTA","NM_001101654","CXXC1"]},{"entry":["NM_006269","RP1","NM_018696","ELAC1"]},{"entry":["NM_138969","SDR16C5","NM_012397","SERPINB13"]},{"entry":["NM_001007070","SDCBP","NM_001037331","C18orf62"]},{"entry":["NM_032466","ASPH","NM_001025101","MBP"]},{"entry":["NM_000370","TTPA","NM_014913","ADNP2"]},{"entry":["NM_002604","PDE7A","NM_198969","AES"]},{"entry":["NM_144650","ADHFE1","NM_001017392","SFRS14"]},{"entry":["NM_013257","SGK3","NM_182577","ODF3L2"]},{"entry":["NM_025170","PREX2","NM_005860","FSTL3"]},{"entry":["NM_006540","NCOA2","NM_002777","PRTN3"]},{"entry":["NM_007332","TRPA1","NM_001001975","ATP5D"]},{"entry":["NM_004770","KCNB2","NM_005883","APC2"]},{"entry":["NM_024721","ZFHX4","NM_003021","SGTA"]},{"entry":["NM_198584","CA13","NM_021231","C19orf29"]},{"entry":["NM_020209","SHD","NM_001103176","CNPY1"]},{"entry":["NM_001013706","LSDP5","NM_014671","UBE3C"]},{"entry":["NM_005817","M6PRBP1","NM_001040032","C8orf74"]},{"entry":["NM_015318","ARHGEF18","NM_031439","SOX7"]},{"entry":["NM_020533","MCOLN1","NM_001127505","ASAH1"]},{"entry":["NM_006949","STXBP2","NM_003155","STC1"]},{"entry":["NM_001042462","TRAPPC5","NM_015254","KIF13B"]},{"entry":["NM_002002","FCER2","NM_021631","FKSG2"]},{"entry":["NM_001004456","OR1M1","NR_003671","LOC728024"]},{"entry":["NM_001005193","OR7G2","NM_003012","SFRP1"]},{"entry":["NM_003451","ZNF177","NM_001556","IKBKB"]},{"entry":["NM_003259","ICAM5","NM_000749","CHRNB3"]},{"entry":["NM_001611","ACP5","NM_001005365","POTEA"]},{"entry":["NM_006397","RNASEH2A","NM_144651","PXDNL"]},{"entry":["NM_006563","KLF1","NM_005098","MSC"]},{"entry":["NM_004907","IER2","NM_000971","RPL7"]},{"entry":["NM_003765","STX10","NM_172037","RDH10"]},{"entry":["NM_024323","C19orf57","NM_024699","ZFAND1"]},{"entry":["NM_001007525","NWD1","NM_152284","CHMP4C"]},{"entry":["NM_015260","SIN3B","NM_001100393","RALYL"]},{"entry":["NM_000923","PDE4C","NM_001083588","E2F5"]},{"entry":["NM_004750","CRLF1","NM_019098","CNGB3"]},{"entry":["NM_004720","LPAR2","NM_002380","MATN2"]},{"entry":["NM_021047","ZNF253","NM_001029860","FBXO43"]},{"entry":["NM_033468","ZNF257","NM_152888","COL22A1"]},{"entry":["NM_001098626","ZNF98","NM_207414","FLJ43860"]},{"entry":["NM_021175","HAMP","NM_177458","LYNX1"]},{"entry":["NM_015302","KIAA0841","NM_201379","PLEC1"]},{"entry":["NM_001863","COX6B1","NM_001129888","CYHR1"]},{"entry":["NM_052948","SNX26","NM_025251","KIAA1688"]},{"entry":["NM_004646","NPHS1","NM_021951","DMRT1"]},{"entry":["NM_020951","ZNF529","NM_005059","RLN2"]},{"entry":["NM_152605","ZNF781","NM_001099223","IFT74"]},{"entry":["NM_001042507","LGALS7B","NM_145005","C9orf72"]},{"entry":["NM_020862","LRFN1","NM_004512","IL11RA"]},{"entry":["NM_016941","DLL3","NM_020944","GBA2"]},{"entry":["NM_182704","SELV","NM_016042","EXOSC3"]},{"entry":["NM_152361","EID2B","NM_153237","C9orf71"]},{"entry":["NM_033543","CEACAM21","NM_032171","CEP78"]},{"entry":["NM_001817","CEACAM4","NM_004938","DAPK1"]},{"entry":["NM_198850","PHLDB3","NM_001912","CTSL1"]},{"entry":["NM_182498","ZNF428","NM_001023564","CTSL3"]},{"entry":["NM_198478","NKPD1","NM_033014","OGN"]},{"entry":["NM_033258","GNG8","NM_000136","FANCC"]},{"entry":["NM_019855","CABP5","NM_014930","ZNF510"]},{"entry":["NM_014959","CARD8","NM_002486","NCBP1"]},{"entry":["NM_031485","GRWD1","NM_014788","TRIM14"]},{"entry":["NM_177973","SULT2B1","NM_197977","ZNF189"]},{"entry":["NM_000979","RPL18","NM_001017998","GNG10"]},{"entry":["NM_145807","NTN5","NM_032303","HSDL2"]},{"entry":["NM_006179","NTF4","NM_021218","C9orf80"]},{"entry":["NM_031886","KCNA7","NM_005118","TNFSF15"]},{"entry":["NM_001040031","CD37","NM_001244","TNFSF8"]},{"entry":["NM_020309","SLC17A7","NM_002077","GOLGA1"]},{"entry":["NM_017432","PTOV1","NM_001100877","PHYHD1"]},{"entry":["NM_152899","IL4I1","NM_014581","OBP2B"]},{"entry":["NM_052884","SIGLEC11","NM_015837","FCN2"]},{"entry":["NM_001030049","KLK3","NM_001080849","DNLZ"]},{"entry":["NM_004917","KLK4","NM_024718","C9orf86"]},{"entry":["NM_001012964","KLK6","NM_004479","FUT7"]},{"entry":["NM_016543","SIGLEC7","NM_000718","CACNA1B"]},{"entry":["NM_053003","SIGLEC12","NM_001727","BRS3"]},{"entry":["NM_001462","FPR2","NM_005044","PRKX"]},{"entry":["NM_173857","VN1R4","NM_001042480","GEMIN8"]},{"entry":["NR_003699","ZNF525","NM_005089","ZRSR2"]},{"entry":["NM_001110514","EBF4","NM_004057","S100G"]},{"entry":["NM_024958","NRSN2","NM_001024666","SH3KBP1"]},{"entry":["NM_001040007","RSPO4","NM_152577","ZNF645"]},{"entry":["NM_198994","TGM6","NM_031442","TMEM47"]},{"entry":["NM_014731","ProSAPiP1","NM_152856","RBM10"]},{"entry":["NM_025220","ADAM33","NM_019056","NDUFB11"]},{"entry":["NM_024960","PANK2","NM_175723","SSX5"]},{"entry":["NM_012409","PRND","NM_003173","SUV39H1"]},{"entry":["NM_170774","RASSF2","NM_013271","PCSK1N"]},{"entry":["NM_144773","PROKR2","NM_000084","CLCN5"]},{"entry":["NM_003434","ZNF133","NM_031407","HUWE1"]},{"entry":["NM_016652","CRNKL1","NM_152424","FAM123B"]},{"entry":["NM_153675","FOXA2","NM_207422","FLJ44635"]},{"entry":["NM_001052","SSTR4","NM_152694","ZCCHC5"]},{"entry":["NM_005492","CST8","NM_003022","SH3BGRL"]},{"entry":["NM_032609","COX4I2","NM_001105243","PCDH19"]},{"entry":["NM_001011718","XKR7","NM_001031834","RAB40AL"]},{"entry":["NM_182658","C20orf185","NM_001113490","AMOT"]},{"entry":["NM_021215","RPRD1B","NM_007231","SLC6A14"]},{"entry":["NR_002911","SNORA71A","NM_145800","6-Sep"]},{"entry":["NM_003064","SLPI","NM_012084","GLUD2"]},{"entry":["NM_181502","SPINLW1","NM_001042751","STAG2"]},{"entry":["NM_147198","WFDC9","NM_001114937","SH2D1A"]},{"entry":["NM_172131","WFDC10B","NM_001078172","FAM127B"]},{"entry":["NM_052951","DNTTIP1","NM_006359","SLC9A6"]},{"entry":["NM_080752","ZSWIM3","NM_000074","CD40LG"]},{"entry":["NM_080608","C20orf165","NM_032512","PDZD4"]},{"entry":["NM_172113","EYA2","NM_024332","BRCC3"]},{"entry":["NM_005985","SNAI1","NM_001818","AKR1C4"]},{"entry":["NM_199129","TMEM189","NM_153256","C10orf47"]},{"entry":["NM_199203","TMEM189-","NM_153244","C10orf111"]},{"entry":[{},"UBE2V1",{},{}]},{"entry":["NM_018431","DOK5","NR_023388","PRINS"]},{"entry":["NM_080617","CBLN4","NM_017433","MYO3A"]},{"entry":["NM_017495","RBM38","NM_173576","MKX"]},{"entry":["NR_003505","PPP4R1L","NM_021738","SVIL"]},{"entry":["NM_207034","EDN3","NM_003591","CUL2"]},{"entry":["NM_152757","C20orf200","NM_153368","GJD4"]},{"entry":["NM_017896","C20orf11","NM_001004297","OR13A1"]},{"entry":["NM_172109","KCNQ2","NM_020945","WDFY4"]},{"entry":["NM_005975","PTK6","NM_001079516","ASAH2B"]},{"entry":["NM_033405","PRIC285","NM_194298","SLC16A9"]},{"entry":["NM_012384","GMEB2","NM_030759","NRBF2"]},{"entry":["NM_182482","BAGE2","NM_201262","DNAJC12"]},{"entry":["NM_182484","BAGE5","NM_024045","DDX50"]},{"entry":["NM_001187","BAGE","NM_015634","KIAA1279"]},{"entry":["NR_003087","ABCC13","NM_018649","H2AFY2"]},{"entry":["NM_022136","SAMSN1","NM_020799","STAMBPL1"]},{"entry":["NM_001338","CXADR","NR_002779","NUDT9P1"]},{"entry":["NM_181600","KRTAP13-4","NM_032373","PCGF5"]},{"entry":["NM_181615","KRTAP20-1","NM_173497","HECTD2"]},{"entry":["NM_175858","KRTAP11-1","NM_003061","SLIT1"]},{"entry":["NM_005806","OLIG2","NM_018425","PI4K2A"]},{"entry":["NM_058182","FAM165B","NM_182639","HPS1"]},{"entry":["NM_130436","DYRK1A","NM_001278","CHUK"]},{"entry":["NM_033173","B3GALT5","NM_005029","PITX3"]},{"entry":["NM_182832","PLAC4","NM_015916","CALHM2"]},{"entry":["NM_002462","MX1","NM_001129742","CALHM3"]},{"entry":["NM_207629","ABCG1","NM_000681","ADRA2A"]},{"entry":["NM_003226","TFF3","NM_002313","ABLIM1"]},{"entry":["NM_018961","UBASH3A","NR_003251","INPP5F"]},{"entry":["NM_018964","SLC37A1","NM_014468","VENTX"]},{"entry":["NM_003307","TRPM2","NM_001109","ADAM8"]},{"entry":["NM_198699","KRTAP10-12","NM_000773","CYP2E1"]},{"entry":["NM_003343","UBE2G2","NM_025092","ATHL1"]},{"entry":["NM_032261","C21orf56","NM_001012416","KRTAP5-6"]},{"entry":["NM_006272","S100B","NM_002555","SLC22A18"]},{"entry":["NM_053006","TSSK2","NM_001005161","OR52B4"]},{"entry":["NM_003776","MRPL40","NM_001004753","OR51F2"]},{"entry":["NM_003325","HIRA","NM_001005567","OR51B5"]},{"entry":["NM_007310","COMT","NM_001005288","OR51I1"]},{"entry":["NM_002650","PI4KA","NM_130389","TRIM34"]},{"entry":["NM_004173","SLC7A4","NM_198185","OVCH2"]},{"entry":["NM_007128","VPREB1","NM_001004461","OR10A6"]},{"entry":["NM_002073","GNAZ","NM_001025389","AMPD3"]},{"entry":["NM_001007468","SMARCB1","NM_012250","RRAS2"]},{"entry":["NR_001283","TOP1P2","NM_016451","COPB1"]},{"entry":["NM_001008566","TPST2","NM_000922","PDE3B"]},{"entry":["NM_001886","CRYBA4","NM_001741","CALCA"]},{"entry":["NM_012399","PITPNB","NM_000728","CALCB"]},{"entry":["NM_013387","UCRC","NM_006292","TSG101"]},{"entry":["NM_004861","GAL3ST1","NM_032781","PTPN5"]},{"entry":["NM_014227","SLC5A4","NM_213599","ANO5"]},{"entry":["NM_001098535","RFPL3","NM_177553","GAS2"]},{"entry":["NR_001450","RFPL3S","NM_001018080","FSHB"]},{"entry":["NM_182486","C1QTNF6","NM_001127612","PAX6"]},{"entry":["NM_032561","C22orf23","NM_001752","CAT"]},{"entry":["NM_181773","APOBEC3H","NM_002391","MDK"]},{"entry":["NM_153497","MAP3K7IP1","NM_016223","PACSIN3"]},{"entry":["NM_005297","MCHR1","NM_024783","AGBL2"]},{"entry":["NM_003932","ST13","NM_001005272","OR4A5"]},{"entry":["NM_152513","MEI1","NM_001003750","OR8I2"]},{"entry":["NM_145733","3-Sep","NM_198183","UBE2L6"]},{"entry":["NM_007326","CYB5R3","NM_001085468","CTNND1"]},{"entry":["NM_001012986","LOC388910","NM_001005212","OR9Q1"]},{"entry":["NM_001123225","C22orf41","NM_001079807","PGA3"]},{"entry":["NM_152228","TAS1R3","NM_004111","FEN1"]},{"entry":["NM_001003808","C1orf222","NM_022830","TUT1"]},{"entry":["NM_207306","KIAA0495","NM_012200","B3GAT3"]},{"entry":["NM_181866","ACOT7","NM_198334","GANAB"]},{"entry":["NM_001042665","PLEKHG5","NR_002560","SNORD31"]},{"entry":["NM_004401","DFFA","NR_002561","SNORD30"]},{"entry":["NM_001103170","AADACL3","NR_002559","SNORD29"]},{"entry":["NM_032341","DDI2","NM_017490","MARK2"]},{"entry":["NM_178840","C1orf64","NM_014067","MACROD1"]},{"entry":["NM_004070","CLCNKA","NM_024650","C11orf80"]},{"entry":["NM_152376","UBXN10","NM_020441","CORO1B"]},{"entry":["NM_001077195","ZNF436","NM_145200","CABP4"]},{"entry":["NM_018202","TMEM57","NM_004055","CAPN5"]},{"entry":["NM_203401","STMN1","NM_021827","CCDC81"]},{"entry":["NM_024869","GRRP1","NM_001101321","UBTFL2"]},{"entry":["NR_003066","SNORD85","NM_004621","TRPC6"]},{"entry":["NM_001856","COL16A1","NM_004347","CASP5"]},{"entry":["NM_001042771","LCK","NM_052889","CARD16"]},{"entry":["NM_206837","C1orf102","NM_152434","CWF19L2"]},{"entry":["NM_022756","C1orf149","NM_207430","C11orf88"]},{"entry":["NM_003819","PABPC4","NM_001082970","C11orf57"]},{"entry":["NM_148960","CLDN19","NM_012459","TIMM8B"]},{"entry":["NM_173484","KLF17","NM_198971","HINFP"]},{"entry":["NM_020365","EIF2B3","NM_012101","TRIM29"]},{"entry":["NM_000778","CYP4A11","NR_003125","LOC85391"]},{"entry":["NM_057176","BSND","NM_001005188","OR6X1"]},{"entry":["NM_152377","C1orf87","NM_001004474","OR10S1"]},{"entry":["NM_032852","ATG4C","NM_001005468","OR8B2"]},{"entry":["NM_203464","AK3L1","NM_001005196","OR8B4"]},{"entry":["NR_003042","SNORD45C","NM_001005195","OR8B12"]},{"entry":["NM_005263","GFI1","NM_014987","IGSF9B"]},{"entry":["NM_005665","EVI5","NM_002234","KCNA5"]},{"entry":["NM_002061","GCLM","NM_001039916","ZNF384"]},{"entry":["NM_139156","AMPD2","NM_007273","PHB2"]},{"entry":["NM_147148","GSTM4","NM_000319","PEX5"]},{"entry":["NM_139053","EPS8L3","NM_015509","NECAP1"]},{"entry":["NM_004696","SLC16A4","NM_020661","AICDA"]},{"entry":["NM_004185","WNT2B","NM_016523","KLRF1"]},{"entry":["NM_006699","MAN1A2","NM_205852","CLEC12B"]},{"entry":["NM_006099","PIAS3","NM_030817","APOLD1"]},{"entry":["NM_005399","PRKAB2","NM_001013698","C12orf69"]},{"entry":["NM_212551","LYSMD1","NM_001175","ARHGDIB"]},{"entry":["NM_178431","LCE3A","NM_032918","RERG"]},{"entry":["NM_207308","NUP210L","NM_004570","PIK3C2G"]},{"entry":["NM_170782","KCNN3","NM_018638","ETNK1"]},{"entry":["NM_000157","GBA","NM_001004329","DBX2"]},{"entry":["NM_032323","TMEM79","NM_015401","HDAC7"]},{"entry":["NM_001004476","OR10K2","NM_015086","DDN"]},{"entry":["NM_001639","APCS","NM_021044","DHH"]},{"entry":["NM_015331","NCSTN","NM_006337","MCRS1"]},{"entry":["NM_005149","TBX19","NM_000424","KRT5"]},{"entry":["NM_018186","C1orf112","NR_003716","HOTAIR"]},{"entry":["NM_033418","C1orf156","NM_014212","HOXC11"]},{"entry":["NM_152663","RALGPS2","NM_014620","HOXC4"]},{"entry":["NM_173533","TDRD5","NR_003084","HOXC5"]},{"entry":["NM_052966","FAM129A","NM_153693","HOXC6"]},{"entry":["NM_006469","IVNS1ABP","NM_005538","INHBC"]},{"entry":["NM_002113","CFHR1","NM_014770","AGAP2"]},{"entry":["NM_001994","F13B","NM_002076","GNS"]},{"entry":["NM_001024594","C1orf53","NM_020525","IL22"]},{"entry":["NM_004767","GPR37L1","NM_032735","BEST3"]},{"entry":["NM_002832","PTPN7","NM_005447","RASSF9"]},{"entry":["NM_201348","PRELP","NM_001037671","C12orf74"]},{"entry":["NM_020439","CAMK1G","NM_018838","NDUFA12"]},{"entry":["NM_052843","OBSCN","NM_001093","ACACB"]},{"entry":["NM_005999","TSNAX","NM_024072","DDX54"]},{"entry":["NM_019090","KIAA1383","NM_194286","KIAA1853"]},{"entry":["NM_080738","EDARADD","NM_001014336","IL31"]},{"entry":["NM_152666","PLD5","NM_019887","DIABLO"]},{"entry":["NM_006642","SDCCAG8","NM_178314","RILPL1"]},{"entry":["NM_014812","CEP170","NR_002979","SNORA49"]},{"entry":["NM_031844","HNRNPU","NM_175066","DDX51"]},{"entry":["NM_001004492","OR2B11","NM_001007537","RP11-45B20.2"]},{"entry":["NM_001005487","OR13G1","NM_002097","GTF3A"]},{"entry":["NM_001005286","OR6F1","NM_145657","GSX1"]},{"entry":["NM_001005522","OR2T8","NM_178007","STARD13"]},{"entry":["NM_001004695","OR2T33","NM_005584","MAB21L1"]},{"entry":["NM_001004693","OR2T10","NM_001004127","ALG11"]},{"entry":["NM_001042521","MGC13057","NM_001005918","ATP7B"]},{"entry":["NM_013388","PREB","NM_001005501","OR4K2"]},{"entry":["NM_007266","GPN1","NM_001004480","OR11H6"]},{"entry":["NM_017910","TRMT61B","NM_138376","TTC5"]},{"entry":["NM_005102","FEZ2","NM_017807","OSGEP"]},{"entry":["NM_002759","EIF2AK2","NM_019852","METTL3"]},{"entry":["NM_148962","OXER1","NM_016609","SLC22A17"]},{"entry":["NM_001101330","LOC728819","NM_138460","CMTM5"]},{"entry":["NM_001042385","PREPL","NM_006177","NRL"]},{"entry":["NM_178313","SPTBN1","NM_006263","PSME1"]},{"entry":["NM_001039348","EFEMP1","NM_014169","CHMP4A"]},{"entry":["NM_144709","PUS10","NM_203355","CTAGES"]},{"entry":["NM_138458","WDR92","NM_020937","FANCM"]},{"entry":["NM_006507","REG1B","NM_018139","C14orf104"]},{"entry":["NM_017750","RETSAT","NM_006832","FERMT2"]},{"entry":["NM_018271","THNSL2","NM_022571","GPR135"]},{"entry":["NM_020151","STARD7","NM_018373","SYNJ2BP"]},{"entry":["NM_012214","MGAT4A","NM_014982","PCNX"]},{"entry":["NM_201557","FHL2","NM_001024674","LIN52"]},{"entry":["NM_032411","C2orf40","NM_004755","RPS6KA5"]},{"entry":["NM_032528","ST6GAL2","NM_000624","SERPINA5"]},{"entry":["NM_176825","SULT1C2","NM_152592","C14orf49"]},{"entry":["NM_000575","IL1A","NR_003237","SNORD113-9"]},{"entry":["NM_025181","SLC35F5","NM_001311","CRIP1"]},{"entry":["NM_005270","GLI2","NM_144599","NIPA1"]},{"entry":["NM_004805","POLR2D","NR_003317","SNORD116-2"]},{"entry":["NM_032357","CCDC115","NR_003298","SNORD115-6"]},{"entry":["NM_018328","MBD5","NR_003311","SNORD115-17"]},{"entry":["NM_004522","KIF5C","NR_003314","SNORD115-22"]},{"entry":["NM_052917","GALNT13","NR_003495","HBII-52-24"]},{"entry":["NM_006593","TBR1","NR_003347","SNORD115-32"]},{"entry":["NM_004490","GRB14","NR_003498","HBII-52-45"]},{"entry":["NM_021193","HOXD12","NM_001042495","SLC12A6"]},{"entry":["NM_001077197","PDE11A","NM_175741","C15orf55"]},{"entry":["NM_145739","OSBPL6","NM_207445","C15orf54"]},{"entry":["NM_000885","ITGA4","NM_001080791","C15orf57"]},{"entry":["NM_002500","NEUROD1","NM_015540","RPAP1"]},{"entry":["NM_005019","PDE1A","NM_014994","MAPKBP1"]},{"entry":["NM_000090","COL3A1","NM_173088","CAPN3"]},{"entry":["NM_014585","SLC40A1","NM_016396","CTDSPL2"]},{"entry":["NM_144708","ANKAR","NM_003758","EIF3J"]},{"entry":["NM_022353","OSGEPL1","NM_152448","C15orf43"]},{"entry":["NM_016192","TMEFF2","NM_207581","DUOXA2"]},{"entry":["NM_138395","MARS2","NM_000338","SLC12A1"]},{"entry":["NM_014670","BZW1","NM_014701","SECISBP2L"]},{"entry":["NM_006139","CD28","NM_153374","LYSMD2"]},{"entry":["NM_014929","FASTKD2","NM_001104554","PAQR5"]},{"entry":["NM_020989","CRYGC","NM_201526","ISLR"]},{"entry":["NM_000599","IGFBP5","NM_015162","ACSBG1"]},{"entry":["NM_001557","IL8RB","NM_003847","PEX11A"]},{"entry":["NM_005381","NCL","NM_020211","RGMA"]},{"entry":["NM_002601","PDE6D","NM_152449","LYSMD4"]},{"entry":["NM_001631","ALPI","NM_006453","TBL3"]},{"entry":["NM_017974","ATG16L1","NM_004209","SYNGR3"]},{"entry":["NR_003006","SCARNA6","NM_001103175","CCDC64B"]},{"entry":["NM_021027","UGT1A9","NM_004220","ZNF213"]},{"entry":["NR_004428","EGO","NM_016292","TRAP1"]},{"entry":["NM_015869","PPARG","NM_004380","CREBBP"]},{"entry":["NM_015199","ANKRD28","NM_001127448","ABAT"]},{"entry":["NM_054110","GALNTL2","NM_002761","PRM1"]},{"entry":["NM_017897","OXSM","NM_014153","ZC3H7A"]},{"entry":["NM_001025068","ARPP-21","NM_015161","ARL6IP1"]},{"entry":["NM_175888","ZNF620","NM_001002911","GPR139"]},{"entry":["NM_016305","SS18L2","NM_003460","ZP2"]},{"entry":["NM_001295","CCR1","NM_013302","EEF2K"]},{"entry":["NM_003965","CCRL2","NM_181078","IL21R"]},{"entry":["NM_138615","DHX30","NM_024516","C16orf53"]},{"entry":["NM_005051","QARS","NM_001114380","ITGAL"]},{"entry":["NM_080865","GPR62","NR_002966","SNORA30"]},{"entry":["NM_018313","PBRM1","NM_033226","ABCC12"]},{"entry":["NM_015224","C3orf63","NM_001006610","SIAH1"]},{"entry":["NM_002841","PTPRG","NM_144602","C16orf78"]},{"entry":["NM_182920","ADAMTS9","NM_031885","BBS2"]},{"entry":["NM_015123","FRMD4B","NM_002990","CCL22"]},{"entry":["NM_001080393","GLT8D4","NR_002978","SNORA46"]},{"entry":["NM_001080448","EPHA6","NR_003079","SNORD111"]},{"entry":["NM_014648","DZIP3","NM_020995","HPR"]},{"entry":["NM_005944","CD200","NM_001105663","NUDT7"]},{"entry":["NM_001085357","BTLA","NM_021197","WFDC1"]},{"entry":["NM_173799","TIGIT","NM_024735","FBXO31"]},{"entry":["NM_005694","COX17","NM_006822","RAB40B"]},{"entry":["NM_016372","GPR175","NM_182705","FAM101B"]},{"entry":["NM_007283","MGLL","NM_022463","NXN"]},{"entry":["NM_007354","C3orf27","NR_003073","SNORD91B"]},{"entry":["NM_004164","RBP2","NM_018553","C17orf85"]},{"entry":["NM_152616","TRIM42","NM_198501","SMTNL2"]},{"entry":["NM_023915","GPR87","NM_001100812","CXCL16"]},{"entry":["NM_000902","MME","NM_001024937","MINK1"]},{"entry":["NM_207015","NAALADL2","NM_019013","FAM64A"]},{"entry":["NM_014693","ECE2","NM_003985","TNK1"]},{"entry":["NM_005787","ALG3","NM_152766","C17orf61"]},{"entry":["NM_004366","CLCN2","NM_032356","LSMD1"]},{"entry":["NM_006232","POLR2H","NM_004217","AURKB"]},{"entry":["NM_004443","EPHB3","NM_201432","GAS7"]},{"entry":["NM_018138","TBCCD1","NM_001372","DNAH9"]},{"entry":["NM_181573","RFC4","NM_016113","TRPV2"]},{"entry":["NM_001114980","TP63","NM_016078","FAM18B"]},{"entry":["NM_178335","CCDC50","NM_016231","NLK"]},{"entry":["NM_015274","MAN2B2","NM_015584","POLDIP2"]},{"entry":["NM_001105662","USP17","NM_144683","DHRS13"]},{"entry":["NM_031911","C1QTNF7","NM_005408","CCL13"]},{"entry":["NM_001775","CD38","NM_017559","FNDC8"]},{"entry":["NM_182592","YIPF7","NM_001104588","SLFN11"]},{"entry":["NM_173536","GABRG1","NM_032875","FBXL20"]},{"entry":["NM_006587","CORIN","NM_199334","THRA"]},{"entry":["NM_018475","TMEM165","NM_033187","KRTAP4-3"]},{"entry":["NM_002703","PPAT","NM_033060","KRTAP4-1"]},{"entry":["NM_001012763","GNRHR","NM_021013","KRT34"]},{"entry":["NM_016323","HERC5","NM_032387","WNK4"]},{"entry":["NM_145244","DDIT4L","NM_001466","FZD2"]},{"entry":["NM_002006","FGF2","NM_152343","C17orf46"]},{"entry":["NM_007080","LSM6","NM_005486","TOM1L1"]},{"entry":["NM_183375","ESSPL","NM_138363","CCDC45"]},{"entry":["NM_004564","PET112L","NM_014960","ARSG"]},{"entry":["NM_001013415","FBXW7","NM_139177","SLC39A11"]},{"entry":["NM_000824","GLRB","NM_022036","GPRCSC"]},{"entry":["NM_201592","GPM6A","NM_178233","OTOP3"]},{"entry":["NM_138464","LOC116349","NR_004397","SNORD1C"]},{"entry":["NM_004174","SLC9A3","NM_199167","CLUL1"]},{"entry":["NM_018140","CEP72","NM_005433","YES1"]},{"entry":["NM_007030","TPPP","NM_173211","TGIF1"]},{"entry":["NM_182632","SLC6A18","NM_173464","L3MBTL4"]},{"entry":["NM_001001702","FLJ33360","NM_003826","NAPG"]},{"entry":["NM_001369","DNAH5","NM_172241","CTAGE1"]},{"entry":["NR_003921","PMCHL1","NM_002647","PIK3C3"]},{"entry":["NM_178140","PDZD2","NR_002970","SNORA37"]},{"entry":["NM_001042625","CAPSL","NM_005024","SERPINB10"]},{"entry":["NM_173489","HEATR7B2","NM_152721","DOK6"]},{"entry":["NM_001005473","PLCXD3","NM_006566","CD226"]},{"entry":["NM_175921","C5orf51","NM_130760","MADCAM1"]},{"entry":["NM_153361","MGC42105","NM_006830","UQCR"]},{"entry":["NM_021147","CCNO","NM_198532","C19orf35"]},{"entry":["NM_181523","PIK3R1","NM_024333","FSD1"]},{"entry":["NR_003014","SNORA47","NM_153359","MGC24975"]},{"entry":["NM_005779","LHFPL2","NM_174918","C19orf59"]},{"entry":["NM_005654","NR2F1","NR_002931","CLEC4GP1"]},{"entry":["NM_001002796","MCTP1","NM_012335","MYO1F"]},{"entry":["NM_031952","SPATA9","NM_001004699","OR2Z1"]},{"entry":["NM_004772","C5orf13","NM_004461","FARSA"]},{"entry":["NM_001077654","TNFAIP8","NM_080864","RLN3"]},{"entry":["NM_001048252","CTXN3","NM_032207","C19orf44"]},{"entry":["NM_002154","HSPA4","NM_145046","CALR3"]},{"entry":["NM_017415","KLHL3","NM_002248","KCNN1"]},{"entry":["NM_013983","NRG2","NM_025249","KIAA1683"]},{"entry":["NM_018931","PCDHB11","NM_001080409","ZNF99"]},{"entry":["NM_018929","PCDHGC5","NM_001105570","NUDT19"]},{"entry":["NM_001079812","DIAPH1","NM_019849","SLC7A10"]},{"entry":["NM_181677","PPP2R2B","NM_021902","FXYD1"]},{"entry":["NM_054023","SCGB3A2","NM_207392","KRTDAP"]},{"entry":["NM_133371","MYOZ3","NM_014364","GAPDHS"]},{"entry":["NM_052860","ZNF300","NM_021232","PRODH2"]},{"entry":["NR_002168","PPP1R2P3","NM_144689","ZNF420"]},{"entry":["NM_021911","GABRB2","NM_003890","FCGBP"]},{"entry":["NM_001129891","LOC-","NM_198540","B3GNT8"]},{"entry":[{},"100131897",{},{}]},{"entry":["NM_144769","FOXI1","NM_004363","CEACAM5"]},{"entry":["NM_001008220","CPLX2","NM_002780","PSG4"]},{"entry":["NM_016290","UIMC1","NM_001005376","PLAUR"]},{"entry":["NM_197975","BTNL3","NM_001033719","ZNF404"]},{"entry":["NM_001085401","C6orf201","NM_001081563","DMPK"]},{"entry":["NM_145649","GCNT2","NM_004943","DMWD"]},{"entry":["NM_003529","HIST1H3A","NM_001425","EMP3"]},{"entry":["NR_003504","MGC22265","NM_182575","IZUMO1"]},{"entry":["NM_003535","HIST1H3J","NM_014475","DHDH"]},{"entry":["NM_001509","GPX5","NM_001459","FLT3LG"]},{"entry":["NM_006510","TRIM27","NM_001079907","ZNF331"]},{"entry":["NM_021904","GABBR1","NM_001037731","DEFB116"]},{"entry":["NM_001077511","TCF19","NM_153269","C20orf96"]},{"entry":["NM_001105564","CCHCR1","NM_080725","SRXN1"]},{"entry":["NM_005514","HLA-B","NM_001083910","SIRPB1"]},{"entry":["NM_002904","RDBP","NM_001039508","SIRPG"]},{"entry":["NM_022107","GPSM3","NR_003684","SNORD119"]},{"entry":["NM_001077516","SLC39A7","NM_000490","AVP"]},{"entry":["NM_002931","RING1","NM_198798","ANKRD5"]},{"entry":["NM_002263","KIFC1","NM_130811","SNAP25"]},{"entry":["NM_007104","RPL10A","NM_001195","BFSP1"]},{"entry":["NM_032115","KCNK16","NM_002509","NKX2-2"]},{"entry":["NM_014623","MEA1","NM_178312","GGTLC1"]},{"entry":["NM_152882","PTK7","NM_024893","C20orf39"]},{"entry":["NM_170609","CRISP1","NR_004846","LOC10013486"]},{"entry":["NM_014464","TINAG","NM_002110","HCK"]},{"entry":["NM_019036","HMGCLL1","NM_031483","ITCH"]},{"entry":["NM_183050","BCKDHB","NM_006404","PROCR"]},{"entry":["NR_003038","SNHG5","NM_080748","ROMO1"]},{"entry":["NM_018064","AKIRIN2","NM_213632","C20orf132"]},{"entry":["NM_002043","GABRR2","NR_003018","SNORA71D"]},{"entry":["NM_015323","KIAA0776","NR_003239","SNHG11"]},{"entry":["NM_005190","CCNC","NM_000022","ADA"]},{"entry":["NM_014028","OSTM1","NM_198139","SEMG1"]},{"entry":["NM_006016","CD164","NM_001048226","DBNDD2"]},{"entry":["NM_153048","FYN","NM_021035","ZNFX1"]},{"entry":["NM_002269","KPNA5","NM_080829","FAM65C"]},{"entry":["NM_024581","FAM184A","NM_173485","TSHZ2"]},{"entry":["NM_152730","C6orf170","NM_177980","CDH26"]},{"entry":["NM_001446","FABP7","NM_080606","BHLHE23"]},{"entry":["NM_030963","RNF146","NM_080823","SRMS"]},{"entry":["NM_014702","KIAA0408","NM_017806","LIME1"]},{"entry":["NM_001079823","LAMA2","NM_003906","MCM3AP"]},{"entry":["NM_005021","ENPP3","NM_058181","C21orf57"]},{"entry":["NM_053278","TAAR8","NM_021219","JAM2"]},{"entry":["NM_078488","VNN2","NM_181614","KRTAP19-7"]},{"entry":["NM_052831","C6orf192","NM_000454","SOD1"]},{"entry":["NM_005923","MAP3K5","NM_170737","KCNJ15"]},{"entry":["NM_001100166","PHACTR2","NM_004915","ABCG1"]},{"entry":["NM_001080951","PLAGL1","NM_198693","KRTAP10-2"]},{"entry":["NM_138785","C6orf72","NM_198690","KRTAP10-9"]},{"entry":["NM_017909","RMND1","NM_198692","KRTAP10-11"]},{"entry":["NM_001007466","TULP4","NM_197966","BID"]},{"entry":["NM_020823","TMEM181","NM_002688","5-Sep"]},{"entry":["NM_003057","SLC22A1","NR_003714","DKFZp434P21"]},{"entry":["NM_021977","SLC22A3","NM_002430","MN1"]},{"entry":["NM_024492","LPAL2","NM_030641","APOL6"]},{"entry":["NM_000301","PLG","NM_013356","SLC16A8"]},{"entry":["NM_031409","CCR6","NM_004810","GRAP2"]},{"entry":["NM_018974","UNC93A","NM_024053","CENPM"]},{"entry":["NM_144781","PDCD2","NM_058238","WNT7B"]},{"entry":["NM_020144","PAPOLB","NM_032257","ZMYND12"]},{"entry":["NM_032172","USP42","NM_002440","MSH4"]},{"entry":["NM_019005","MIOS","NM_031936","GPR61"]},{"entry":["NM_019029","CPVL","NM_001042552","TATDN3"]},{"entry":["NM_175887","PRR15","NM_007357","COG2"]},{"entry":["NM_032639","PLEKHA8","NM_001004698","OR2W5"]},{"entry":["NM_006774","INMT","NM_175735","LYG2"]},{"entry":["NM_016489","NT5C3","NM_001106","ACVR2B"]},{"entry":["NM_015283","DPY19L1","NM_007246","KLHL2"]},{"entry":["NM_002192","INHBA","NM_016604","JMJD1B"]},{"entry":["NM_015052","HECW1","NM_005565","LCP2"]},{"entry":["NM_001013398","IGFBP3","NM_014068","PSORS1C1"]},{"entry":["NR_003595","MGC26484","NM_004289","NFE2L3"]},{"entry":["NM_198570","VWC2","NM_018697","LANCL2"]},{"entry":["NM_030798","WBSCR16","NR_015374","LOC401463"]},{"entry":["NM_000927","ABCB1","NM_014637","MTFR1"]},{"entry":["NM_006980","MTERF","NM_172060","EYA1"]},{"entry":["NM_001742","CALCR","NM_199160","LHX6"]},{"entry":["NM_001099400","SGCE","NM_012364","OR1Q1"]},{"entry":["NM_015395","TECPR1","NM_000216","KAL1"]},{"entry":["NM_015545","PTCD1","NM_001099857","IKBKG"]},{"entry":["NM_000777","CYP3A5","NM_001002295","GATA3"]},{"entry":["NM_018275","C7orf43","NM_018294","CWF19L1"]},{"entry":["NM_013439","PILRA","NM_003054","SLC18A2"]},{"entry":["NM_145030","C7orf47","NM_001128202","C10orf122"]},{"entry":["NM_022574","GIGYF1","NM_030754","SAA2"]},{"entry":["NM_021930","RINT1","NM_012194","C11orf41"]},{"entry":["NM_007356","LAMB4","NM_199418","PRCP"]},{"entry":["NM_015723","PNPLA8","NM_145018","C11orf32"]},{"entry":["NM_182529","THAP5","NM_001009562","LST-3TM12"]},{"entry":["NM_001662","ARF5","NM_152590","IFLTD1"]},{"entry":["NM_012133","COPG2","NM_020782","KLHDC5"]},{"entry":["NM_012450","SLC13A4","NM_015319","TENC1"]},{"entry":["NM_003852","TRIM24","NM_201550","LRRC10"]},{"entry":["NM_130840","ATP6V0A4","NR_002801","DKFZp686A1627"]},{"entry":["NM_016019","LUC7L2","NM_016350","NIN"]},{"entry":["NM_022750","PARP12","NM_001756","SERPINA6"]},{"entry":["NM_001080392","KIAA1147","NM_004918","TCL1B"]},{"entry":["NM_018980","TAS2R5","NM_005926","MFAP1"]},{"entry":["NM_004668","MGAM","NM_152450","FAM81A"]},{"entry":["NR_002140","OR6W1P","NM_002558","P2RX1"]},{"entry":["NM_001004685","OR2F2","NM_000442","PECAM1"]},{"entry":["NM_001001659","OR2A14","NM_001040185","ZNF765"]},{"entry":["NM_002889","RARRES2",{},{}]},{"entry":["NM_198285","WDR86"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0211","num":"0210"},"tables":{"@attributes":{"id":"TABLE-US-00007","num":"00007"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Supplemental Table 7. Gene Ontology (GO) analysis of genes"},{"entry":"with promoter \u03b3H2AX accumulation in SEN cells."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["GO term","Name","P-value"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["GO: 0043170","Biopolymer metabolic process","2.83E\u221226"]},{"entry":["GO: 0006139","Nucleobase, nucleoside, nucleotide and","1.43E\u221221"]},{"entry":[{},"nucleic acid metabolic process"]},{"entry":["GO: 0007165","Signal transduction","4.91E\u221219"]},{"entry":["GO: 0019538","Protein metabolic process","7.67E\u221217"]},{"entry":["GO: 0019222","Regulation of metabolic process","1.25E\u221216"]},{"entry":["GO: 0031323","Regulation of cellular metabolic process","3.47E\u221216"]},{"entry":["GO: 0016070","RNA metabolic process","1.35E\u221214"]},{"entry":["GO: 0010468","Regulation of gene expression","1.35E\u221214"]},{"entry":["GO: 0044267","Cellular protein metabolic process","1.11E\u221213"]},{"entry":["GO: 0044260","Cellular macromolecule metabolic process","1.14E\u221213"]},{"entry":["GO: 0019219","Regulation of nucleobase, nucleoside,","3.05E\u221213"]},{"entry":[{},"nucleotide and nucleic acid metabolic"]},{"entry":[{},"process"]},{"entry":["GO: 0006350","transcription","3.16E\u221213"]},{"entry":["GO: 0045449","Regulation of transcription","1.15E\u221212"]},{"entry":["GO: 0048522","Positive regulation of cellular process","1.78E\u221211"]},{"entry":["GO: 0048518","Positive regulation of biological process","3.02E\u221211"]},{"entry":["GO: 0032774","RNA biosynthetic process","5.03E\u221211"]},{"entry":["GO: 0006351","Transcription DNA dependent","5.85E\u221211"]},{"entry":["GO: 0007275","Multicellular organismal development","6.55E\u221211"]},{"entry":["GO: 0051234","Establishment of localization","1.16E\u221210"]},{"entry":["GO: 0048519","Negative regulation of biological process","3.29E\u221210"]},{"entry":["GO: 0006355","Regulation of transcription, DNA dependent","3.35E\u221210"]},{"entry":["GO: 0006810","transport","3.75E\u221210"]},{"entry":["GO: 0048731","System development","5.44E\u221210"]},{"entry":["GO: 0051252","Regulation of RNA metabolic process","6.32E\u221210"]},{"entry":["GO: 0048523","Negative regulation of cellular process","9.86E\u221210"]},{"entry":["GO: 0048856","Anatomical structure development","1.25E\u221209"]},{"entry":["GO: 0051641","Cellular localization","4.91E\u221209"]},{"entry":["GO: 0035556","Intracellular signaling cascade","1.35E\u221208"]},{"entry":["GO: 0022607","Cellular component assembly","2.00E\u221208"]},{"entry":["GO: 0051649","Establishment of cellular localization","2.15E\u221208"]},{"entry":["GO: 0006366","Transcription from RNA polymerase II","2.38E\u221208"]},{"entry":[{},"promoter"]},{"entry":["GO: 0007166","Cell surface receptor linked signal","2.81E\u221208"]},{"entry":[{},"transduction"]},{"entry":["GO: 0009966","Regulation of signal transduction","4.65E\u221208"]},{"entry":["GO: 0006464","Protein modification process","5.35E\u221208"]},{"entry":["GO: 0006996","Organelle organization and biogenesis","7.78E\u221208"]},{"entry":["GO: 0065003","Macromolecular complex assembly","9.38E\u221208"]},{"entry":["GO: 0007049","Cell cycle","1.23E\u221207"]},{"entry":["GO: 0009058","Biosynthetic process","3.04E\u221207"]},{"entry":["GO: 0046907","Intracellular transport","4.31E\u221207"]},{"entry":["GO: 0048468","Cell development","5.91E\u221207"]},{"entry":["GO:","Macromolecule localization","6.70E\u221207"]},{"entry":["GO 0008283","Cell proliferation","8.91E\u221207"]},{"entry":["GO: 0045934","Negative regulation of nucleobase,","1.07E\u221206"]},{"entry":[{},"nucleoside, nucleotide and nucleic acid"]},{"entry":[{},"metabolic process"]},{"entry":["GO: 0009892","Negative regulation of metabolic process","1.99E\u221206"]},{"entry":["GO: 0031324","Negative regulation of cellular metabolic","2.53E\u221206"]},{"entry":[{},"process"]},{"entry":["GO: 0006950","Response to stress","2.98E\u221206"]},{"entry":["GO: 0006357","Regulation of transcription from RNA","3.77E\u221206"]},{"entry":[{},"polymerase II promoter"]},{"entry":["GO: 0008104","Protein localization","4.31E\u221206"]},{"entry":["GO: 0016481","Negative regulation of transcription","7.94E\u221206"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0212","num":"0211"},"tables":{"@attributes":{"id":"TABLE-US-00008","num":"00008"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"287pt","align":"center"}},"thead":{"row":{"entry":"TABLE 8"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Cell Cycle (GO: 0007049) and Cell Proliferation (GO: 0008283) genes with"},{"entry":"promoter \u03b3H2AX accumulation in SEN cells."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"133pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Gene",{},{}]},{"entry":["GO term","symbol","Gene name","Function*"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["GO: 0007049","INHBA","inhibin, beta A","The inhibin beta A subunit joins the alpha"]},{"entry":[{},{},"(activin A, activin","subunit to form a pituitary FSH secretion"]},{"entry":[{},{},"AB alpha","inhibitor. Inhibin has been shown to regulate"]},{"entry":[{},{},"polypeptide)","gonadal stromal cell proliferation negatively"]},{"entry":[{},{},{},"and to have tumor-suppressor activity."]},{"entry":["GO: 0007049","GFI1","growth factor","This gene encodes a nuclear zinc finger"]},{"entry":[{},{},"independent 1","protein that functions as a transcriptional"]},{"entry":[{},{},{},"repressor. This protein plays a role in diverse"]},{"entry":[{},{},{},"developmental contexts, including"]},{"entry":[{},{},{},"hematopoiesis and oncogenesis. It functions"]},{"entry":[{},{},{},"as part of a complex along with other"]},{"entry":[{},{},{},"cofactors to control histone modifications"]},{"entry":[{},{},{},"that lead to silencing of the target gene"]},{"entry":[{},{},{},"promoters."]},{"entry":["GO: 0007049","CD28","CD28 molecule","CD28 costimulation is essential for CD4"]},{"entry":["GO: 0008283",{},{},"(MIM 186940)-positive T-cell proliferation,"]},{"entry":[{},{},{},"survival, interleukin-2 (IL2; MIM147680)"]},{"entry":[{},{},{},"production, and T-helper type-2 (Th2)"]},{"entry":[{},{},{},"development."]},{"entry":["GO: 0007049","STMN1","stathmin","This gene belongs to the stathmin family of"]},{"entry":[{},{},"1\/oncoprotein 18","genes. It encodes a ubiquitous cytosolic"]},{"entry":[{},{},{},"phosphoprotein proposed to function as an"]},{"entry":[{},{},{},"intracellular relay integrating regulatory"]},{"entry":[{},{},{},"signals of the cellular environment. The"]},{"entry":[{},{},{},"encoded protein is involved in the regulation"]},{"entry":[{},{},{},"of the microtubule filament system by"]},{"entry":[{},{},{},"destabilizing microtubules."]},{"entry":["GO: 0007049","CUL2","cullin 2","Core component of multiple cullin-RING-"]},{"entry":["GO: 0008283",{},{},"based ECS (ElonginB\/C-CUL2\/5-SOCS-box"]},{"entry":[{},{},{},"protein) E3 ubiquitin-protein ligase"]},{"entry":[{},{},{},"complexes, which mediate the ubiquitination"]},{"entry":[{},{},{},"of target proteins. May serve as a rigid"]},{"entry":[{},{},{},"scaffold in the complex and may contribute"]},{"entry":[{},{},{},"to catalysis through positioning of the"]},{"entry":[{},{},{},"substrate and the ubiquitin-conjugating"]},{"entry":[{},{},{},"enzyme."]},{"entry":["GO: 0007049","GAS7","growth arrest-","Growth arrest-specific 7 is expressed"]},{"entry":[{},{},"specific 7","primarily in terminally differentiated brain"]},{"entry":[{},{},{},"cells and predominantly in mature cerebellar"]},{"entry":[{},{},{},"Purkinje neurons. GAS7 plays a putative"]},{"entry":[{},{},{},"role in neuronal development."]},{"entry":["GO: 0007049","HERC5","hect domain and","This gene is a member of the HERC family"]},{"entry":[{},{},"RLD 5","of ubiquitin ligases and encodes a protein"]},{"entry":[{},{},{},"with a HECT domain and five RCC1"]},{"entry":[{},{},{},"repeats. Pro-inflammatory cytokines"]},{"entry":[{},{},{},"upregulate expression of this gene in"]},{"entry":[{},{},{},"endothelial cells. The protein localizes to the"]},{"entry":[{},{},{},"cytoplasm and perinuclear region and"]},{"entry":[{},{},{},"functions as an interferon-induced E3"]},{"entry":[{},{},{},"protein ligase that mediates ISGylation of"]},{"entry":[{},{},{},"protein targets."]},{"entry":["GO: 0008283","GLI2","GLI-Kruppel","This gene encodes a protein which belongs"]},{"entry":[{},{},"family member","to the C2H2-type zinc finger protein"]},{"entry":[{},{},"GLI2","subclass of the Gli family. Gli family zinc"]},{"entry":[{},{},{},"finger proteins are mediators of Sonic"]},{"entry":[{},{},{},"hedgehog (Shh) signaling and they are"]},{"entry":[{},{},{},"implicated as potent oncogenes in the"]},{"entry":[{},{},{},"embryonal carcinoma cell. The protein"]},{"entry":[{},{},{},"encoded by this gene localizes to the"]},{"entry":[{},{},{},"cytoplasm and activates patched Drosophila"]},{"entry":[{},{},{},"homolog (PTCH) gene expression. It is also"]},{"entry":[{},{},{},"thought to play a role during embryogenesis."]},{"entry":[{},{},{},"The encoded protein is associated with"]},{"entry":[{},{},{},"several phenotypes - Greig"]},{"entry":[{},{},{},"cephalopolysyndactyly syndrome, Pallister-"]},{"entry":[{},{},{},"Hall syndrome, preaxial polydactyly type"]},{"entry":[{},{},{},"IV, postaxial polydactyly types A1 and B."]},{"entry":["GO: 0008283","TCF19","transcription factor","Potential trans-activating factor that could"]},{"entry":[{},{},"19 (SC1)","play an important role in the transcription of"]},{"entry":[{},{},{},"genes required for the later stages of cell"]},{"entry":[{},{},{},"cycle progression."]},{"entry":["GO: 0008283","CD40LG","CD40 ligand (TNF","The protein encoded by this gene is"]},{"entry":[{},{},"superfamily,","expressed on the surface of T cells. It"]},{"entry":[{},{},"member 5, hyper-","regulates B cell function by engaging CD40"]},{"entry":[{},{},"IgM syndrome)","on the B cell surface. A defect in this gene"]},{"entry":[{},{},{},"results in an inability to undergo"]},{"entry":[{},{},{},"immunoglobulin class switch and is"]},{"entry":[{},{},{},"associated with hyper-IgM syndrome."]},{"entry":["GO: 0008283","CALCA","calcitonin\/calcitonin-","This gene encodes the peptide hormones"]},{"entry":[{},{},"related","calcitonin, calcitonin gene-related peptide"]},{"entry":[{},{},"polypeptide, alpha","and katacalcin by tissue-specific alternative"]},{"entry":[{},{},{},"RNA splicing of the gene transcripts and"]},{"entry":[{},{},{},"cleavage of inactive precursor proteins."]},{"entry":[{},{},{},"Calcitonin is involved in calcium regulation"]},{"entry":[{},{},{},"and acts to regulate phosphorus metabolism."]},{"entry":["GO: 0008283","TNFSF15","tumor necrosis","The protein encoded by this gene is a"]},{"entry":[{},{},"factor (ligand)","cytokine that belongs to the tumor necrosis"]},{"entry":[{},{},"superfamily,","factor (TNF) ligand family. This cytokine"]},{"entry":[{},{},"member 15","is a ligand for receptor TNFRSF25 and"]},{"entry":[{},{},{},"decoy receptor TNFRSF21\/DR6. It can"]},{"entry":[{},{},{},"activate NF-kappaB and MAP kinases, and"]},{"entry":[{},{},{},"acts as an autocrine factor to induce"]},{"entry":[{},{},{},"apoptosis in endothelial cells. This cytokine"]},{"entry":[{},{},{},"is also found to inhibit endothelial cell"]},{"entry":[{},{},{},"proliferation, and thus may function as an"]},{"entry":[{},{},{},"angiogenesis inhibitor."]},{"entry":["GO: 0008283","ADRA2A","adrenergic, alpha-","Alpha-2-adrenergic receptors are members"]},{"entry":[{},{},"2A-, receptor","of the G protein-coupled receptor"]},{"entry":[{},{},{},"superfamily. These receptors have a critical"]},{"entry":[{},{},{},"role in regulating neurotransmitter release"]},{"entry":[{},{},{},"from sympathetic nerves and from"]},{"entry":[{},{},{},"adrenergic neurons in the central nervous"]},{"entry":[{},{},{},"system."]},{"entry":["GO: 0008283","IL8RB","interleukin 8","The protein encoded by this gene is a"]},{"entry":[{},{},"receptor, beta","member of the G-protein-coupled receptor"]},{"entry":[{},{},{},"family. This protein is a receptor for"]},{"entry":[{},{},{},"interleukin 8 (IL8). It binds to IL8 with high"]},{"entry":[{},{},{},"affinity, and transduces the signal through a"]},{"entry":[{},{},{},"G-protein activated second messenger"]},{"entry":[{},{},{},"system. This receptor also binds to"]},{"entry":[{},{},{},"chemokine (C\u2014X\u2014C motif) ligand 1"]},{"entry":[{},{},{},"(CXCL1\/MGSA), a protein with melanoma"]},{"entry":[{},{},{},"growth stimulating activity, and has been"]},{"entry":[{},{},{},"shown to be a major component required for"]},{"entry":[{},{},{},"serum-dependent melanoma cell growth."]},{"entry":[{},{},{},"This receptor mediates neutrophil migration"]},{"entry":[{},{},{},"to sites of inflammation."]},{"entry":["GO: 0008283","TNFSF8","tumor necrosis","The protein encoded by this gene is a"]},{"entry":[{},{},"factor (ligand)","cytokine that belongs to the tumor necrosis"]},{"entry":[{},{},"superfamily,","factor (TNF) ligand family. This cytokine is"]},{"entry":[{},{},"member 8","a ligand for TNFRSF8\/CD30, which is a cell"]},{"entry":[{},{},{},"surface antigen and a marker for Hodgkin"]},{"entry":[{},{},{},"lymphoma and related hematologic"]},{"entry":[{},{},{},"malignancies. This cytokine was shown to"]},{"entry":[{},{},{},"enhance cell proliferation of some"]},{"entry":[{},{},{},"lymphoma cell lines, while to induce cell"]},{"entry":[{},{},{},"death and reduce cell proliferation of other"]},{"entry":[{},{},{},"lymphoma cell lines."]},{"entry":["GO: 0008283","FLT3LG","fms-related","Stimulates the proliferation of early"]},{"entry":[{},{},"tyrosine kinase 3","hematopoietic cells. Synergizes well with a"]},{"entry":[{},{},"ligand","number of other colony stimulating factors"]},{"entry":[{},{},{},"and interleukins."]},{"entry":["GO: 0008283","CD164","CD164 molecule,","CD164 is a type I integral transmembrane"]},{"entry":[{},{},"sialomucin","sialomucin that functions as an adhesion"]},{"entry":[{},{},{},"receptor."]},{"entry":["GO: 0008283","PLG","plasminogen","The protein encoded by this gene is a"]},{"entry":[{},{},{},"secreted blood zymogen that is activated by"]},{"entry":[{},{},{},"proteolysis and converted to plasmin and"]},{"entry":[{},{},{},"angiostatin."]},{"entry":["GO: 0008283","FABP7","fatty acid binding","The protein encoded by this gene is a brain"]},{"entry":[{},{},"protein 7, brain","fatty acid binding protein."]},{"entry":["GO: 0008283","EVI5","ecotropic viral","Functions as a regulator of cell cycle"]},{"entry":[{},{},"integration site 5","progression by stabilizing the FBXO5"]},{"entry":[{},{},{},"protein and promoting cyclin-A"]},{"entry":[{},{},{},"accumulation during interphase. May play a"]},{"entry":[{},{},{},"role in cytokinesis."]},{"entry":["GO: 0008283","IL1A","interleukin 1,","The protein encoded by this gene is a"]},{"entry":[{},{},"alpha","member of the interleukin 1 cytokine family."]},{"entry":[{},{},{},"This cytokine is a pleiotropic cytokine"]},{"entry":[{},{},{},"involved in various immune responses,"]},{"entry":[{},{},{},"inflammatory processes, and hematopoiesis."]},{"entry":["GO: 0008283","RERG","RAS-like,","RERG, a member of the RAS superfamily of"]},{"entry":[{},{},"estrogen-regulated,","GTPases, inhibits cell proliferation and"]},{"entry":[{},{},"growth inhibitor","tumor formation."]},{"entry":["GO: 0008283","EIF2AK2","eukaryotic","Following activation by double-stranded"]},{"entry":[{},{},"translation","RNA in the presence of ATP, the kinase"]},{"entry":[{},{},"initiation factor 2-","becomes autophosphorylated and can"]},{"entry":[{},{},"alpha kinase 2","catalyze the phosphorylation of the"]},{"entry":[{},{},{},"translation initiation factor EIF2S1, which"]},{"entry":[{},{},{},"leads to an inhibition of the initiation of"]},{"entry":[{},{},{},"protein synthesis."]},{"entry":["GO: 0008283","EMP3","epithelial","Probably involved in cell proliferation and"]},{"entry":[{},{},"membrane protein 3","cell-cell interactions."]},{"entry":["GO: 0008283","CRIP1","cysteine-rich","Cysteine-rich intestinal protein (CRIP)"]},{"entry":[{},{},"protein 1","belongs to the LIM\/double zinc finger"]},{"entry":[{},{},"(intestinal)","protein family. CRIP may be involved in"]},{"entry":[{},{},{},"intestinal zinc transport."]},{"entry":["GO: 0008283","REG1B","regenerating islet-","This gene encodes a protein secreted by the"]},{"entry":[{},{},"derived 1 beta","exocrine pancreas that is highly similar to"]},{"entry":[{},{},"(pancreatic stone","the REG1A protein."]},{"entry":[{},{},"protein, pancreatic"]},{"entry":[{},{},"thread protein)"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0213","num":"0212"},"tables":{"@attributes":{"id":"TABLE-US-00009","num":"00009"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"273pt","align":"center"}},"thead":{"row":{"entry":"SUPPLEMENTAL\u2003TABLE\u20039"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"RT-PCR\u2003and\u2003qPCR\u2003primers"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"196pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SEQ\u2003"]},{"entry":["Primer","Sequence","ID\u2003NO"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["MIRm","Forward:\u20035\u2032-AGGATTTAAAGCTCTCTCTGCAGG-3\u2032","14"]},{"entry":[{},"Reverse:\u20035\u2032-ATGACTGAACTCTAAGATAAAGATCACAGC-3\u2032","15"]},{"entry":{}},{"entry":["Alu","Forward:\u20035\u2032-AGACAATCCTGGCCAACTTGG-3\u2032","16"]},{"entry":[{},"Reverse:\u20035\u2032-GCATTCCTGGACTGTGATGTGG-3\u2032","17"]},{"entry":{}},{"entry":["AluJb","Forward:\u20035\u2032-TGTGTGCCTGTAGTCCTAGCTACTAGG-3\u2032","18"]},{"entry":[{},"Reverse:\u20035\u2032-TTCAGGTTAGAGCTCTGAAGTCACG-3\u2032","19"]},{"entry":{}},{"entry":["AluSx","Forward:\u20035\u2032-TCTGCTCGGGAGGCTGAGG-3\u2032","20"]},{"entry":[{},"Reverse:\u20035\u2032-CCACCCACGAAGAATACATTTGC-3\u2032","21"]},{"entry":{}},{"entry":["RPL13A","Forward:\u20035\u2032-AAGGTGTTTGACGGCATCC-3\u2032","22"]},{"entry":[{},"Reverse:\u20035\u2032-GTTCTTCTCGGCCTGTTTCC-3\u2032","23"]},{"entry":{}},{"entry":["Oct4","Forward:\u20035\u2032-CTTCGCAAGCCCTCATTTC-3\u2032","24"]},{"entry":[{},"Reverse:\u20035\u2032-GAGAAGGCGAAATCCGAAG-3\u2032","25"]},{"entry":{}},{"entry":["Nanog","Forward:\u20035\u2032-CCCCAGCCTTTACTCTTCCT-3\u2032","26"]},{"entry":[{},"Reverse:\u20035\u2032-CTGGTTGCTCCAGGTTGAAT-3\u2032","27"]},{"entry":{}},{"entry":["b-actin\u2003","Forward:\u20035\u2032-CAACTCCATCATGAAGTGTGAC-3\u2032","28"]},{"entry":[{},"Reverse:\u20035\u2032-GCCATGCCAATCTCATCTTG-3\u2032","29"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Recent genome wide analysis indicates that thousands of short Interspersed elements (SINEs) are present in constrained non-exonic elements (Lowe and Haussler, PNAS 2007) suggesting that these repeated retrotransposal elements might play a unique and yet to be discovered role in shaping and\/or specializing the genomic landscape as well as gene regulation during mammalian evolution. Here we report LC-MS\/MS RNA-affinity complex isolation using piRNA derived from Alu retrotransposal RNA. Our data indicate the role of piAlu RNAs in DNA repair, cell cycle, and chromatin regulation.","Recently, we have demonstrated that a majority of the repairable DNA damage sites in self-renewing human adult stem cells is associated with the retrotransposal portion of the genome, in particular, with Alu retrotransposons (Wang et al. (2011) 10: 3016-3030). Our group has shown that the up-regulation of transcriptional activity from Alu can be triggered by genotoxic stress-induced damage and can be recorded in human retinal pigment epithelium (RPE) and human adult stem cells (Id.) upon in-vivo and ex-vivo aging, respectively. Evidence indicates that increased accumulation of Alu RNA within cells is directly linked to Dicer-1 deficiency (Kaneko et al. (2011) 471: 325-330), suggesting that Alu transcript accumulation might not necessarily be due to increased transcriptional activity from Alu retrotransposons, but rather may result from the absence of concomitant Alu transcript processing, leading to cytotoxicity. Our recent data indicates that the accumulation of unprocessed Alu transcripts triggers chromatin deterioration, loss of DNA repair in pericentromeric areas eliciting the persistent DNA damage response and, ultimately, cellular senescence. Our data also indicate that human adult stem cells stably expressing an shRNA against Alu transcripts override cellular senescence and reinstate their DNA repair capacity (Wang et al. (2011) 10: 3016-3030), suggesting that RNA interference (RNAi) machinery is involved in these events. This data also suggests two equally possible mechanisms through which an shRNA against Alu might mediate the observed function: 1) through the PTGS Dicer-dependent pathway via the cytoplasmic degradation of unprocessed Alu RNA or 2) through facilitating transcriptional silencing by recruiting either the Dicer-dependent or Piwi-dependent arms of the RNAi pathway to act directly on the chromatin as shown in . Both of these pathways are plausible and either could depend on the assembly of single or multiple protein complexes that are capable of cross-talk with DNA-damage-sensing\/repair and chromatin and\/or centromeric maintenance pathways. Interestingly, a recent study indicated that transcripts generated from telomere-repeat-encoded RNA (TERRA) interact with heterochromatin protein 1 (HP1), trimethlyated histone H3 Lysine 9 (H3K9me3), core components of the Shelterin complex as well as members of the DNA-damage-sensing pathway (Deng et al. (2009) 35: 403-413).","Similarly we believe that the portion of Alu RNA, which is functionally relevant to overriding the senescent phenotype of hADSCs (Wang et al. (2011) 10: 3016-3030), can mediate a broad array of downstream effects associated with retrotransposal transcription. Since the cytoplasmic role of Alu RNA in the assembly of signal recognition particles has been previously reported (He et al. (1994) 107(Pt 4): 903-912), we have focused our efforts specifically on the elucidation of nuclear complexes assembled on Alu RNA. Using an unbiased RNA affinity assay coupled with mass spectrometry, we provide evidence for the composition of molecular complexes assembled on processed Alu RNA transcripts. Our data suggests the pathways and molecular processes through which processed intermediates of Alu RNA may participate in a multitude of nuclear regulations within human adult stem cells.","Results","Alu RNA and its Homology to Human PIWI-Interacting RNA (piRNA)","Previously it was reported that Alu RNA participates in the cytoplasmic assembly of the signal recognition particle (SRP) in mammalian cells (He et al. (1994) 107(Pt 4): 903-912). The mammalian SRP is composed of a single RNA, the 7SL RNA, and six proteins (Walter and Blobel (1981) 91: 557-561). SRP9 and SRP14 bind to the 5\u2032 end of the RNA (Alu-domain), which functions in translational arrest (Siegel and Walter (1988) 13: 314-316; Siegel and Walter (1986) 320:81-84). However, this functional shRNA reported in Wang et al. (2011) 10: 3016-3030 has no complementarity to 7SL RNA. In addition, the region of Alu that corresponds to the shRNA is more highly conserved between repeats than the rest of the element (). This conservation is consistent with the ability of the shRNA to effectively knock down transcription from multiple repeated Alu copies, including members of distinct subfamilies. Taken together, these data illuminate the functional relevance of the Alu shRNA sequence analyzed here, and indicate that this sequence may represent a processed Alu RNA species that functions in the nucleus. We queried the Alu shRNA sequence analyzed here against previously reported PIWI interacting RNA (piRNA) sequences previously reported in Girard et al. (2006) 442: 199-202.","The exact sequence of the shRNA was identified as a PIWI-interacting RNA, pir-52207 and pir-57460 (Id.) (NCBI accession #DQ585095 (tgcctgtaat cccagctact caggaggctg, (SEQ ID NO:30)), and is referred to herein as piALU. This suggests that Alu RNA is subjected to processing by an analog of the human PIWI complex, the molecular function of which is unknown, but nevertheless indicates that this piAlu transcript, derived from processed Alu transcripts, might function in the nucleus.","Alu RNA Interaction with Nuclear Factors","We attempted to obtain a comprehensive list of proteins that interact, both directly and indirectly, with piALU. For this purpose, we engaged an unbiased RNA-affinity assay using synthetic biotinylated RNA oligos encompassing Alu nucleotides 140-183, shown in , with nuclear extracts from self-renewing human adult stem cells. This portion of Alu is conserved among different types of Alu retrotransposons: AluSx, AluJb, AluJo, and AluY ().","Our piALU RNA affinity assay was combined with a comprehensive LC-MS\/MS (Liquid Chromatography-Electrospray Tandem Mass Spectrometry) analysis of interacting proteins. Since a nuclear RNA surveillance pathway exists that protects from transcriptional noise, we added a control hexanucleotide repeat control RNA \u201cbait\u201d, described in Materials and Methods, to avoid non-sequence-specific interactions. The experimental procedure was previously reported by Deng et al. (2009) 35: 403-413 and is outlined in . Our comparative analysis of RNA-protein binding by PAGE depicted the presence of multiple piALU probe-specific bands () that were excised for LC-MS\/MS identification. An exhaustive list of the proteins identified is shown in Supplementary Table 1.","Alu RNA Interactions with Major Chromatin Modifiers, DNA Repair Complexes, Centromeric Chromatin\/Kinetochore Assembly Factors and Transcriptional Factors","Our mass spectrometry analysis of unbiased piALU RNA-affinity precipitants revealed the presence four major protein categories such as chromatin modifiers, DNA repair complexes, centromeric chromatin\/kinetochore assembly and transcriptional factors (). The abundance of each component is represented as a color gradient fading from true hues to black for singly represented peptides. Similar to protein interactions with telomeric RNA repeats (TERRA) (Id.), piALU binding proteins include DNA damage response proteins DNA-PKC, PARP and hnRNPM, suggesting that piALU RNA may be integral to DNA repair complexes.","To assess the relevance of piALU RNA to biological processes, we applied a Gene Ontology (GO) (The Gene Ontology Consortium (2010) 38(suppl 1): D331-D335; Barrell et al. (2009) 37:D396-D403) analysis to the list of interacting proteins. All together, 19 cellular processes were significantly (3.12\u00d710\u2266P\u22661.5\u00d710) enriched among Alu-interacting proteins including: DNA repair, organelle organization, cell cycle, chromatin remodeling and transcription (). These broader categories encompass nucleotide excision repair, DSB repair via homologous recombination, DNA recombination, V(D)J recombination, response to stress, chromosome segregation, cell cycle check-points, regulation of cell cycle checkpoint arrest, interphase of mitotic cycle and mitosis (). Complexes related to DNA duplex unwinding, regulation of transcription and transcriptional elongation from RNA polymerase II promoters were also significantly enriched.","Several nuclear proteins were associated with multiple GO terms (n=12) designated to cellular components including the centrosome, nucleoplasm, kinetochore, spindle, microtubule cytoskeleton, centromeric and telomeric regions of chromosomes, as well as the cohesin and DNA-directed RNA polymerase complexes ().","These data suggest a possible role for Alu RNA in the regulation of chromatin organization in centromeric\/pericentromeric regions as well as DNA repair processes and the transcriptional regulation of genes. All of these cellular processes have been proven to be indispensable to the regulation of cell cycle progression.","Protein-Protein Interaction Network of the piALU RNA-Affinity Complex","Understanding the cellular function(s) of an isolated protein complex(es) is facilitated by the ability to correctly uncover and annotate all functional interactions among LC-MS\/MS identified proteins. To build a protein-protein interaction network among piALU RNA-affinity isolated proteins, we used the online database Search Tool for the Retrieval of Interacting Genes (STRING) version 9.0. This tool provides uniquely comprehensive coverage and ease of access to both experimental and predicted interaction information (Szklarczyk et al. (2011) 39: D561-D568). The STRING model of our LC-MS\/MS data of nuclear piALU RNA-protein interactions is shown in . Only high confidence level (experimentally reported) evidence of interactions\/associations in curated databases in conjunction with the Markov clustering algorithm (MCL) were used to build the network and assign proteins into families (stronger associations are represented by thicker lines in  and ) (Enright et al. (2002) 30: 1575-1584). Strikingly, 9 out of the 64 Alu RNA interacting partners are involved in chromosome segregation and include SGOL1, SGOL2, CENPE, CENPF, MRE11A, CDCA5 with 4 more (PBRM1, CENPL, CDC27 and MAP3K8) directly linked to mitosis, while the addition of PIN4 and KIF15 depicts a complex protein network regulating spindle assembly. This observation suggests that piALU transcripts might form a molecular connection between centromere cohesion and microtubule dynamics at the kinetochore. The previously reported loss of outer kinetochore proteins CENP-E and CENP-F upon knockdown of Sgo (Salic et al. (2004) 118: 567-578), together with our recent observation that the accumulation of unprocessed Alu RNA blocks the recruitment the cohesin complex to centromeres and triggers cellular senescence in adult stem cells (Wang et al. (2011) 10: 3016-3030), suggests that Alu RNA in its processed form may be an RNA adaptor in spindle microtubule dynamics.","Data shown in  also indicate that 34 out of the 64 analyzed piALU RNA interacting partners are somehow involved in the regulation of the cell cycle. The presence of centrosomal proteins CEP192, CEP110, CEP152 and CEP135 points to the intriguing possibility that Alu RNA intermediates play a role in this cellular organelle and together with CUL4A, GMNN and CTD1 (), may participate in regulating interphase and the G\/M phase transition.","Excitingly, piALU RNA also pulled down components of the nucleotide excision repair (ERCC3, DDB1, MRE11A), non-homologous end joining (LIG4, XRCC6) and homologous recombination (PALB2, RAD50) DNA repair pathways as well as the BAZ1B protein, suggesting a putative role for piALU RNA in double-strand break repair. The assembly of the DNA damage repair complexes described herein may be a critical event in adult stem cell aging (Kaneko et al. (2011) 471: 325-330; Wang et al. (2011) 10: 3016-3030). It is tempting to speculate that a lack of proper assembly\/function of these complexes together or separately with chromatin remodeling components (CHD6, SMARCA2, BAZ1A, ARID1A, 1N080, ASXL1 and KDM2A) may explain the gradual loss of DNA damage repair triggering the persistent DNA damage response and cellular senescence (Rodier et al. (2009) 11: 973-979; Rodier et al. (2011) 124: 68-81).","Discussion","A recent genome-wide ChIP-RNA sequencing project revealed that over a third of all known large conserved ncRNAs can be pulled down with just four different chromatin modifying proteins (Khalil et al. (2009) 106: 11667-11672), suggesting that most long ncRNAs associate with chromatin modifiers. The piALU RNA interactome presented herein also includes a variety of chromatin modifiers, possibly elucidating another level of complexity in ncRNA\/chromatin modifier protein complexes that depend on the presence of a shorter, processed form of the Alu transcript. The full-length sequence of Alu RNA can form four distinct stem-loop structures, which may act as individual nucleating centers for the formation of distinct RNA-protein complexes (). Only one of the four stem-loops was used as a synthetic RNA \u201cbait\u201d, since it encompasses the previously reported human PIWI-interacting RNA (Girard et al. (2006) 442: 199-202) and is capable of nucleating protein complex(es) that are directly or indirectly responsible for reinstating self-renewal to senescent adult stem cells (Wang et al. (2011) 10: 3016-3030).","Indeed, in accord with our previous data, the observed interactions supports the relevance of these nuclear piALU RNA-protein complex(es) to cell cycle control, DNA repair, chromatin maintenance, and the function of centromeres\/pericentromeres in human adult stem cells. This data is currently lacking functional assessment, but nevertheless may hold the key to understanding how retrotransposal transcription\/processing regulates human adult stem cell aging and senescence ex-vivo.","The data reported in this Example points to interesting possible involvements of short RNA species derived from Alu retrotransposons in the regulation of cell cycle control, RNA-pol II transcription and the maintenance of chromatin architecture. It appears that this participation is intimately linked to DNA repair processes.","Material and Methods","Human ADSC Isolation and Expansion","Human adipose derived stem cells (hADSCs) were isolated from human subcutaneous white adipose tissue collected during liposuction procedures. The lipoaspirate was suspended in Hank's Buffered Salt Solution (HBSS), 3.5% Bovine Serum Albumin (BSA), 1% Collagenase, type II (Sigma) in 1:3 w\/v ratio and shaken at 37\u00b0 C. for 50 min. The cells were filtered through a 70 \u03bcm mesh cell strainer (BD Falcon #352350), treated with Red Blood Cell Lysis buffer (150 mM NHCl, 10 mM KHCO, 0.1 mM EDTA, pH 7.3), and expanded ex-vivo in DMEM\/F12 complete medium (DMEM\/F12, 10% FBS, 100 U\/ml penicillin, 100 \u03bcg\/ml streptomycin, 2.5 \u03bcg\/ml amphotericin B; Invitrogen) in 10% COat 37\u00b0 C. and passaged at 80% confluency, changing medium every 72-96 hours.","Nuclear Chromatin Extraction","Nuclear and cytoplasmic proteins were isolated from self-renewing hADSCs using the ProteoJET\u2122 Cytoplasmic and Nuclear Protein Extraction Kit (Fermentas #K0311), according to the manufacturer's protocol.","RNA Secondary Structure Folding","5\u2032 biotinylated single stranded synthetic RNA probes corresponding to the 140-183 nucleotide region of full-length Alu, (5\u2032-\/Bio\/-GCC GGG CG UG AUG GUG GGC GCC UGU AGU CCC AGC UAC UCG GGA GGC-3\u2032, SEQ ID NO:31) as well as a synthetic random octamer repeat 5\u2032-\/Bio\/-(CACUGA)(SEQ ID NO:32) as a linear control, were purchased from Integrated DNA Technologies (IDT). These oligos were purified via RNase-Free HPLC. 8 \u03bcg of each oligo were heated in a PCR tube with 15 ml of 2\u00d7 Folding Buffer (40 mM HEPES pH 7.9, 0.4 mM EDTA, 2 M NaCl) and 9 ml DEPC water with 50 U\/ml SUPERase-In RNAse inhibitor (Ambion, #AM2694). The RNA oligonucleotides were heated to 78\u00b0 C. to induce a linear conformation, then the reaction temperature was reduced to and held at 54\u00b0 C. to accommodate and retain desired secondary structure formation (mimicking the secondary stem-loop structure of the full length RNA), based on calculations by UNAFold software (IDT). The control repeat RNA 5\u2032-\/Bio\/-(CACUGA)(SEQ ID NO:32) is not capable of forming a secondary structure.","Protein Pulldown (Unbiased RNA Affinity Assay)","100 \u03bcl of streptavidin-coated Dynabeads (Dynabeads M-270 Streptavidin, #65306) per reaction were washed three times in 1\u00d7D150 buffer (20 mM HEPES pH 7.9, 20% glycerol, 0.2 mM EDTA, 150 mM NaCl, 0.05% NP-40, 1 mM PMSF, 10 mM \u03b2-mercaptoethanol) before a final high-salt wash with 1\u00d7 Folding Buffer (containing 1 M NaCl). 30 \u03bcl of folded RNA solution was added to each reaction tube and incubated at RT for 15 min, then washed twice with 100 \u03bcl of low-salt Folding Buffer (containing 150 mM NaCl). Bead-bound RNA probes were incubated for 30 min at RT with nuclear extracts from self-renewing hADSCs, along with 1 \u03bcl of 0.01M PMSF and 3 \u03bcl SUPERase-In. After incubation, the Dynabeads were washed five times in 1\u00d7D150 buffer, leaving only directly and indirectly interacting partners attached to the beads. As a negative control, nuclear extracts were also incubated with Dynabeads alone to control for any interaction between nuclear molecules and the beads themselves.","Denaturing PAGE Analysis of Affinity Purified Protein Complexes.","XT Sample buffer 12 (BioRad #161-0791) and XT Reducing Agent (BioRad #161-0792) were added to 10 \u03bcg of protein. After denaturation for 5 min at 100\u00b0 C., protein samples were loaded into a precast 4-12% NuPAGE Novex 4-12% Bis-Tris gel (Invitrogen, #NP0321BOX), and run out at 150 V for two hours. Gels were stained with ProteoSilver Plus Silver Stain Kit (Sigma, #Prot-SIL1). A SeeBlue Plus2 Pre-Stained Standard (Invitrogen, cat #LC5925) was used for visualization and approximation of molecular weights of protein samples. Bands unique to the Alu probe lane were excised and the analyzed by LC-MS\/MS for protein content.","Sample Preparation for MS Analysis","The gel samples were first rinsed with acetonitrile, then reduced using DTT followed by alkylation using iodoacetamide. We rinsed gel samples with three alternating washes of 50 mM ammonium bicarbonate and acetonitrile, then cooled and resuspended each gel slice in trypsin (5.5 \u03bcg\/mL in 50 mM ammonium bicarbonate\/10% acetonitrile) and incubated at 37\u00b0 C. for 24 hours for digestion of proteins. We extracted peptides with one rinse of 50 mM ammonium bicarbonate\/10% acetonitrile followed by one rinse of 50% acetonitrile\/0.1% formic acid, and prepared samples for mass spectrometry by lyophilization and rehydration in 20 \u03bcL 5% acetonitrile\/0.2% formic acid.","Identification of Protein Partners by Mass Spectrometry High Resolution LC-MS\/MS","High resolution LC-MS\/MS analysis was carried out on an LTQ-Orbitrap XL mass spectrometer (Thermo Fisher Scientific) as previously described ENREF 45 (Lopez et al. (2011) 10: 133-142). Briefly, we loaded excised and digested samples into 96-well plates for mass spectrometry analysis on a LTQ-Orbitrap XL (Thermo Fisher Scientific) instrument as previously described (Lopez et al. (2011) 10: 133-142). We injected 10 \u03bcL of each re-constituted sample using a Thermo Scientific EASY-nLC Autosampler. Reverse phase chromatographic separations were carried out using Hypersil GOLD\u2122 C18\u2122 3 \u03bcm media packed into a fused silica 75 \u03bcm inner diameter, 20 cm long column running at 250 nL\/min. A gradient was produced using 5-40% acetonitrile, 0.2% formic acid over 150 minutes. The LTQ-Orbitrap was run in a top 8 configuration at 60K resolution for a full scan, with monoisotopic precursor selection enabled, and +1, and unassigned charge state rejected. The analysis on the LTQ-Orbitrap instrument was carried out with CID fragmentation.","LC-MS\/MS Data Analysis and Protein Identification","LC-MS\/MS data analysis, protein identification and peaklist generation were performed using Proteome Discoverer (Thermo Fisher Scientific) algorithm incorporating the SEQUEST\u00ae search engine and Percolator\u2122 (Kall et al. (2007) 4: 923-925) as previously described (Lopez et al. (2011) 10: 133-142). MS\/MS data were searched using 10 ppm mass accuracy on precursor m\/z and a 0.5 Da window on fragment ions. Fully enzymatic tryptic searches with up to three missed cleavage sites were allowed. Oxidized methionines were searched as a variable modification and alkylated cysteines were searched as a fixed modification. Human databases were downloaded from NCBI and supplemented with common contaminants. We filtered peptides for each charge state to a false discovery rate (FDR) of 1%, and then grouped peptides into proteins using Occam's razor logic.","Protein Interaction Network Analysis","All highly represented members of the RNA-protein complex which were isolated were analyzed using STRING 9.0 software (Szklarczyk et al. (2011) 39: D561-D568). The obtained interaction network was subsequently analyzed with BINGO2 plug-in to determine statistical enrichments for 14 Gene Ontology (GO) categories (The Gene Ontology Consortium (2010) 38(suppl 1) D331-D335; Barrell et al. (2009) 37:D396-D403).","It is understood that the examples and embodiments described herein are for illustrative purposes only and that various modifications or changes in light thereof will be suggested to persons skilled in the art and are to be included within the spirit and purview of this application and scope of the appended claims. All publications, patents, and patent applications cited herein are hereby incorporated by reference in their entirety for all purposes."],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0020","num":"0019"},"figref":["FIGS. 1A-1C","FIG. 1A","FIG. 1B","FIG. 1C","FIG. 8"]},{"@attributes":{"id":"p-0021","num":"0020"},"figref":["FIGS. 2A-2G","FIG. 2A","FIG. 2B","FIG. 2C","FIG. 2D","FIG. 2E","FIG. 2F","FIG. 2G"]},{"@attributes":{"id":"p-0022","num":"0021"},"figref":["FIGS. 3A-3E","FIG. 3A","FIG. 3B","FIG. 3C","FIG. 3D","FIG. 3E"]},{"@attributes":{"id":"p-0023","num":"0022"},"figref":["FIG. 4","FIG. 13"]},{"@attributes":{"id":"p-0024","num":"0023"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0025","num":"0024"},"figref":["FIGS. 6A-6G","FIG. 6A","FIG. 6B","FIG. 6C","FIG. 6D","FIG. 6E","FIG. 6F"],"sup":["3","3"],"b":"6"},{"@attributes":{"id":"p-0026","num":"0025"},"figref":["FIGS. 7A-7B","FIG. 7A","FIG. 7B"],"sup":["3","3","3"]},{"@attributes":{"id":"p-0027","num":"0026"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0028","num":"0027"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0029","num":"0028"},"figref":"FIG. 10"},{"@attributes":{"id":"p-0030","num":"0029"},"figref":["FIGS. 11A-11C","FIG. 11A","FIG. 11B","FIG. 11C","FIG. 11A"]},{"@attributes":{"id":"p-0031","num":"0030"},"figref":"FIG. 12"},{"@attributes":{"id":"p-0032","num":"0031"},"figref":["FIG. 13","FIGS. 4C and 4D"]},{"@attributes":{"id":"p-0033","num":"0032"},"figref":"FIG. 14"},{"@attributes":{"id":"p-0034","num":"0033"},"figref":["FIGS. 15A and 15B","FIG. 15A","FIG. 15B"]},{"@attributes":{"id":"p-0035","num":"0034"},"figref":"FIG. 16"},{"@attributes":{"id":"p-0036","num":"0035"},"figref":"FIG. 17"},{"@attributes":{"id":"p-0037","num":"0036"},"figref":"FIG. 18"},{"@attributes":{"id":"p-0038","num":"0037"},"figref":["FIGS. 19A and 19B","FIG. 19A","FIG. 19B"]},{"@attributes":{"id":"p-0039","num":"0038"},"figref":["FIGS. 20A-20E","FIG. 20A","FIG. 20B","FIG. 20C","FIG. 20D","FIG. 20E"]},{"@attributes":{"id":"p-0040","num":"0039"},"figref":"FIG. 21"},{"@attributes":{"id":"p-0041","num":"0040"},"figref":["FIG. 22A","FIG. 22A","FIG. 22B","FIG. 22C","FIG. 22D"]},{"@attributes":{"id":"p-0042","num":"0041"},"figref":"FIG. 23"},{"@attributes":{"id":"p-0043","num":"0042"},"figref":["FIG. 24A","FIG. 24C"],"sub":["10","10"],"b":"24"},{"@attributes":{"id":"p-0044","num":"0043"},"figref":"FIG. 25"}]},"DETDESC":[{},{}]}
