---
title: Reducing levels of nicotinic alkaloids in plants
abstract: Two genes, A622 and NBB1, can be influenced to achieve a decrease of nicotinic alkaloid levels in plants. In particular, suppression of one or both of A622 and NBB1 may be used to decrease nicotine in tobacco plants.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09029656&OS=09029656&RS=09029656
owner: 22nd Century Limited, LLC
number: 09029656
owner_city: Clarence
owner_country: US
publication_date: 20110408
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATION","FIELD OF THE INVENTION","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","EXAMPLE 1","Preparation of pRNAi-A622 Vector for Reducing Alkaloid Content by Down-Regulating A622 Expression","EXAMPLE 2","Suppression of A622 in Tobacco BY-2 Cells","EXAMPLE 3","Construction of an Inducible A622 RNAi Vector","EXAMPLE 4","Suppression of A622 in Tobacco Hairy Roots","EXAMPLE 5","Identification of NBB1 as a Gene Regulated by the NIC Loci","EXAMPLE 6","Characterization of NBB1","EXAMPLE 7","Phylogenetic Analysis of NBB1","EXAMPLE 8","Preparation of NBB1 Suppression Construct","EXAMPLE 9","NBB1 Suppression in Tobacco BY-2 Cells","EXAMPLE 10","NBB1 Suppression in Tobacco Hairy Roots","EXAMPLE 11","NBB1 Suppression in Transgenic Tobacco Plants"],"p":["This application is continuation of U.S. application Ser. No. 11\/579,661, filed Feb. 28, 2006, which is a U.S. national stage of PCT\/IB2006\/001741, filed under 35 U.S.C. \u00a7371, published in English, and claims the benefit of U.S. Provisional Application No. 60\/656,636, filed Feb. 28, 2005. Each application is incorporated herein in its entirety.","The present invention relates to the field of molecular biology and the down-regulation of alkaloid synthesis. More specifically, the invention relates to methodology and constructs for reducing nicotinic alkaloids in a plant, particularly but not exclusively in a tobacco plant.","Presently, several methods exist for reducing nicotinic alkaloids, such as nicotine, in plants. A low-nicotine strain of tobacco has been employed, for instance, as breeding stock for low-nicotine cultivars. Legg et al., 10:212 (1970). Genetic engineering methods also can be used to reduce nicotine levels. For example, U.S. Pat. Nos. 5,369,023, and 5,260,205 discuss decreasing nicotine levels via antisense targeting of an endogenous putrescine methyl transferase (PMT) sequence. Voelckel et al., 11:121-126 (2001). The tobacco quinolate phosphoribosyl transferase (QPT) gene has been cloned, Sinclair et al., 44: 603-617 (2000), and its antisense suppression provided significant nicotine reductions in transgenic tobacco plants. Xie et al., 30: 17-37 (2004). See also U.S. Pat. Nos. 6,586,661 and 6,423,520.","Several nicotine biosynthesis enzymes are known. For instance, see Hashimoto et al., 37; 25-37 (1998); Reichers & Timko, 41:387-401 (1999); Imanishi et al., 38:1101-1111 (1998). Still, there is a continuing need for additional genetic engineering methods for further reducing nicotinic alkaloids. When only PMT is down-regulated in tobacco, for example, nicotine is reduced but anatabine increases by about 2-to-6-fold. Chintapakorn & Hamill, 53: 87-105 (2003); Steppuhn, et al., 2(8): e217: 1074-1080 (2004). When only QPT is down-regulated, a fair amount of alkaloids remain. See U.S. Plant Variety Certificate No. 200100039.","Reducing total alkaloid content in tobacco would increase the value of tobacco as a biomass resource. When grown under conditions that maximize biomass, such as high density and multiple cuttings, tobacco can yield more than 8 tons dry weight per acre, which is comparable with other crops used for biomass. Large-scale growing and processing of conventional tobacco biomass has several drawbacks, however. For example, significant time and energy is spent extracting, isolating, and disposing tobacco alkaloids because conventional tobacco biomass, depending on the variety, contains about 1 to about 5 percent alkaloids. On a per acre basis, conventional tobacco biomass contains approximately as much as 800 pounds of alkaloids. Also, people handling tobacco may suffer from overexposure to nicotine, commonly referred to as \u201cgreen tobacco disease.\u201d","Reduced-alkaloid tobacco is more amenable for non-traditional purposes, such as biomass and derived products. For example, it is advantageous to use reduced-alkaloid tobacco for producing ethanol and protein co-products. U.S. published application No. 2002\/0197688. Additionally, alkaloid-free tobacco or fractions thereof may be used as a forage crop, animal feed, or a human nutritive source. Id.","Beyond these benefits associated with reducing nicotine, more successful methods are needed to assist smokers in quitting smoking. Nicotine replacement therapy (NRT) is not very effective as a smoking cessation treatment because its success rate is less than 20 percent after 6 to 12-months from the end of the nicotine replacement period. Bohadana et al., 160:3128-3134 (2000); Croghan et al., 5:181-187 (2003); Stapleton et al., 90:31-42 (1995). Nicotine-reduced or nicotine-free tobacco cigarettes have assisted smokers in quitting smoking successfully, by weaning the smoker from nicotine yet allowing the smoker to perform the smoking ritual. Additionally, denicotinized cigarettes relieve craving and other smoking withdrawal symptoms. See Rose, 184: 274-285 (2006) and Rose et al., 8:89-101 (2006).","Accordingly, there is a continuing need to identify additional genes whose expression can be affected to decrease nicotinic alkaloid content.","Two genes, A622 and NBB1, can be influenced to achieve a decrease of nicotinic alkaloid levels in plants. In particular, suppression of one or both of A622 and NBB1 may be used to decrease nicotine in tobacco plants.","Accordingly, in one aspect, the invention provides an isolated nucleic acid molecule comprising a nucleotide sequence selected from (a) a nucleotide sequence set forth in SEQ ID NO: 3; (b) a nucleotide sequence that encodes a polypeptide having the amino acid sequence set forth in SEQ ID NO: 4; and (c) a nucleotide sequence that differs from the nucleotide sequences of (a) or (b) due to degeneracy of the genetic code and encodes a polypeptide with NBB1 expression.","In another aspect, the invention provides an isolated nucleic acid molecule comprising a nucleotide sequence selected from (a) a nucleotide sequence set forth in SEQ ID NO: 1; (b) a nucleotide sequence that encodes a polypeptide having the amino acid sequence set forth in SEQ ID NO: 2; and (c) a nucleotide sequence that differs from the nucleotide sequences of (a) or (b) due to degeneracy of the genetic code and encodes a polypeptide with A622 expression, wherein said nucleotide sequence is operatively linked to a heterologous promoter.","In another aspect, the invention provides a method for reducing an alkaloid in a plant, comprising decreasing NBB1 and A622 expression.","In another aspect, the invention provides a transgenic plant having reduced A622 expression and alkaloid content, as well as a tobacco plant having reduced NBB1 expression and alkaloid content. The invention provides also a genetically engineered plant having reduced nicotine and anatabine content.","In another aspect, the invention provides a reduced-nicotine tobacco product made from a tobacco plant having reduced A622 or NBB1 expression.","The present inventors identified two genes, A622 and NBB1, that can be influenced to achieve a decrease of nicotinic alkaloid levels in plants, including but not limited to tobacco. While A622 was identified previously by Hibi et al. 6: 723-735 (1994), the present inventors discovered a role for A622, heretofore unknown, in the context of decreasing nicotine biosynthesis. The field was wholly unaware of NBB1, before the inventor's discovery, and they also elucidated a role for NBB1 in an approach, according to the present invention, for reducing nicotinic alkaloid content in plants.","Accordingly, the present invention encompasses both methodology and constructs for reducing nicotinic alkaloid content in a plant, by suppressing A622 or NBB1 expression. That is, nicotinic alkaloid levels can be reduced by suppressing one or both of A622 and NBB1. Pursuant to this aspect of the invention, a plant or any part thereof is transformed with a nucleotide sequence, expression of which suppresses at least one of A622 and NBB1 and reduces nicotinic alkaloid content.","In another aspect of the invention, nicotine can be further suppressed in a plant by concurrently suppressing expression of any known enzyme in the nicotine biosynthesis pathway, such as QPT or PMT, and at least one of A622 and NBB1. In addition to decreasing nicotine, for example, the present invention provides a means for concurrently reducing anatabine. Thus, anatabine levels can be lowered by suppressing a nicotine biosynthesis gene, such as QPT, and at least one of A622 and NBB1.","By means of affecting A622 and\/or NBB1 expression, to the ends of reducing nicotinic alkaloid content in a plant, numerous reduced-alkaloid plants and by-products may be obtained, in keeping with the present invention. For example, a tobacco plant having suppressed A622 or NBB1 expression may be used for producing reduced-nicotine cigarettes, which may find use as a smoking cessation product. Likewise, reduced-nicotine tobacco may be used as a forage crop, animal feed, or a source for human nutrition.","Other objects, features and advantages of the present invention will become apparent from the following detailed description. It should be understood, however, that the detailed description and the specific examples, while indicating preferred embodiments of the invention, are given by way of illustration only, since changes and modifications within the spirit and scope of the invention may become apparent to those of skill in the art from this detailed description.","Definitions","The technical terms employed in this specification are commonly used in biochemistry, molecular biology and agriculture; hence, they are understood by those skilled in the field to which this invention belongs. Those technical terms can be found, for example in: MC: A LM3rd ed., vol. 1-3, ed. Sambrook and Russel, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 2001; CPMB, ed. Ausubel et al., Greene Publishing Associates and Wiley-Interscience, New York, 1988 (with periodic updates); SPMB: A CMCPMB5ed., vol. 1-2, ed. Ausubel et al., John Wiley & Sons, Inc., 2002; GA: A LM, vol. 1-2, ed. Green et al., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1997.","Methodology involving plant biology techniques are described herein and are described in detail in methodology treatises such as MPMB: A LCM, ed. Maliga et al., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1995. Various techniques using PCR are described, for example, in Innis et al., PCR P: A GMA, Academic Press, San Diego, 1990 and in Dieffenbach and Dveksler, PCR P: A LM2ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 2003. PCR-primer pairs can be derived from known sequences by known techniques such as using computer programs intended for that purpose, e.g., Primer, Version 0.5, 1991, Whitehead Institute for Biomedical Research, Cambridge, Mass. Methods for chemical synthesis of nucleic acids are discussed, for example, in Beaucage & Caruthers, 22:1859-1862 (1981), and Matteucci & Caruthers, 103:3185 (1981).","Restriction enzyme digestions, phosphorylations, ligations and transformations were done as described in Sambrook et al., MC: A LM2ed. (1989), Cold Spring Harbor Laboratory Press. All reagents and materials used for the growth and maintenance of bacterial cells were obtained from Aldrich Chemicals (Milwaukee, Wis.), DIFCO Laboratories (Detroit, Mich.), Invitrogen (Gaithersburg, Md.), or Sigma Chemical Company (St. Louis, Mo.) unless otherwise specified.","A622 expression is controlled by the NIC1 and NIC2 gene loci in tobacco plants. Hibi et al., 6: 723-735 (1994). It has been reported that A622 exhibits the same expression pattern as PMT. Shoji, T. et al., 41:9:1072-1076 (2000a); Shoji, T., et al., 50:427-440 (2002). Both A622 and PMT are expressed specifically in roots, particularly in the cortex and endodermis of the apical parts of the roots and root hairs Shoji et al. (2002). Moreover, A622 and PMT have a common pattern of expression in response to NIC regulation and methyl-jasmonate stimulus. A622 is induced in the roots of in response to wounding of aerial tissues. Cane et al., 32, 305-320 (2005). In , A622 is induced in wounded leaves under conditions that result in QPT induction. Sinclair et al., 31:721-9 (2004).","The nucleic acid sequence of A662 (SEQ ID NO: 1) has been determined. Hibi et al. (1994), supra. The protein encoded by this nucleic acid sequence (SEQ ID NO: 2) resembles isoflavone reductases (IFR) and contains an NADPH-binding motif. A622 shows homology to TP7, a tobacco phenylcoumaran benzylic ether reductase (PCBER) involved in lignin biosynthesis. Shoji et al. (2002), supra. No PCBER activity was observed, however, when A622 expressed in was assayed with two different substrates.","Based on co-regulation of A622 and PMT and the similarity of A622 to IFR, A622 was proposed to function as a reductase in the final steps of nicotinic alkaloid synthesis. Hibi et al. (1994); Shoji, et al. (2000a). No IFR activity was observed, however, when the protein was expressed in bacteria (id.). The function of A622 was unknown previously, and there was no understanding heretofore that A622 plays a role in nicotine synthesis.","A622 expression refers to biosynthesis of a gene product encoded by SEQ ID NO: 1. A622 suppression refers to the reduction of A622 expression. A622 suppression has an ability to down-regulate nicotinic alkaloid content in a plant or a plant cell.","An alkaloid is a nitrogen-containing basic compound found in plants and produced by secondary metabolism. A nicotinic alkaloid is nicotine or an alkaloid that is structurally related to nicotine and that is synthesized from a compound produced in the nicotine biosynthesis pathway. In the case of tobacco, nicotinic alkaloid content and total alkaloid content are used synonymously.","Illustrative alkaloids include but are not limited to nicotine, nornicotine, anatabine, anabasine, anatalline, N-methylanatabine, N-methylanabasine, myosmine, anabaseine, N\u2032-formylnornicotine, nicotyrine, and cotinine. Other very minor alkaloids in tobacco leaf are reported, for example, in Hecht, S. S. et al., 12: 92-98 (1979); Tso, T. C., . Ideals Inc., Beltsville, Md. (1990). The chemical structures of several alkaloids are presented, for example, in Felpin et al., 66: 6305-6312 (2001).","Nicotine is the primary alkaloid in along with 50-60 percent of other species of . Based on alkaloid accumulation in the leaves, nornicotine, anatabine, and anabasine are the other foremost alkaloids in . Anatabine is usually not the primary alkaloid in any species but does accumulate to relatively higher amounts in 3 species; anabasine is the primary alkaloid in four species. Nornicotine is the primary alkaloid in 30 to 40 percent of species. Depending on the variety, about 85 to about 95 percent of total alkaloids in is nicotine. Bush, L. P., , Coresta 285-291 (1999); Hoffmann, et al., 41:1-52, (1994).","In the present invention, nicotinic alkaloid content can be reduced in a genetically engineered plant by down-regulating at least one of A622 and NBB1. Additionally, a nicotinic alkaloid content can be lowered by down-regulating a nicotine biosynthesis enzyme, such as QPT or PMT, and at least one of A622 and NBB1.","Anatabine is a nicotinic alkaloid. Previous studies have demonstrated that PMT suppression reduces nicotine content but increases putrescine and anatabine levels. Chintapakorn & Hamill, 53: 87-105 (2003); Sato et al., 98, 367-372. (2001); Steppuhn, A., et al., 2(8): e217: 1074-1080 (2004). For the purposes of the present invention, anatabine content can be lowered in a genetically engineered plant by down-regulating at least one of A622 and NBB1. Anatabine levels can be lowered further by down-regulating a nicotine biosynthesis enzyme, such as QPT, and at least one of A622 and NBB1.","A BY-2 Tobacco Cell is a cell line established in 1960s by Japan Tobacco Co., Ltd. from a tobacco variety Bright Yellow-2. Since this cell line grows very fast in tissue culture, it is easy to grow on a large scale and is amenable for genetic manipulation. A BY-2 tobacco cell is widely used as a model plant cell line in basic research. When cultured in a standard medium, a BY-2 tobacco cell does not produce nicotinic alkaloids. Addition of jasmonate into the culture medium induces formation of nicotinic alkaloids.","Complementary DNA (cDNA) is a single-stranded DNA molecule that is formed from an mRNA template by the enzyme reverse transcriptase. Those skilled in the art also use \u201ccDNA\u201d to denote to a double stranded DNA molecule that includes such a single-stranded DNA molecule and its complementary DNA strand. Typically, a primer complementary to portions of mRNA is employed for the initiation of a reverse transcription process that yields a cDNA.","Expression refers to the biosynthesis of a gene product. In the case of a structural gene, for example, expression involves transcription of the structural gene into mRNA and the translation of the mRNA into one or more polypeptides.","Gene refers to a polynucleotide sequence that comprises control and coding sequences necessary for the production of a polypeptide or precursor. The polypeptide can be encoded by a full-length coding sequence or by any portion of the coding sequence. A gene may constitute an uninterrupted coding sequence or it may include one or more introns, bound by the appropriate splice junctions. Moreover, a gene may contain one or more modifications in either the coding or the untranslated regions that could affect the biological activity or the chemical structure of the expression product, the rate of expression, or the manner of expression control. Such modifications include, but are not limited to, mutations, insertions, deletions, and substitutions of one or more nucleotides. In this regard, such modified genes may be referred to as \u201cvariants\u201d of the \u201cnative\u201d gene.","Genetically engineered (GE) encompasses any methodology for introducing a nucleic acid or specific mutation into a host organism. For example, a tobacco plant is genetically engineered when it is transformed with a polynucleotide sequence that suppresses expression of a gene, such as A622 or NBB1, and thereby reduces nicotine levels. In contrast, a tobacco plant that is not transformed with a polynucleotide sequence that suppresses expression of a target gene is a control plant and is referred to as a \u201cnon-transformed\u201d plant.","In the present context, the \u201cgenetically engineered\u201d category includes \u201ctransgenic\u201d plants and plant cells (see definition, infra), as well as plants and plant cells produced by means of targeted mutagenesis effected, for example, through the use of chimeric RNA\/DNA oligonucleotides, as described by Beetham et al., 96: 8774-8778 (1999) and Zhu et al., loc. cit. at 8768-8773, or so-called \u201crecombinagenic olionucleobases,\u201d as described in PCT application WO 03\/013226. Likewise, a genetically engineered plant or plant cell may be produced by the introduction of a modified virus, which, in turn, causes a genetic modification in the host, with results similar to those produced in a transgenic plant, as described herein. See, e.g., U.S. Pat. No. 4,407,956. Additionally, a genetically engineered plant or plant cell may be the product of any native approach (i.e., involving no foreign nucleotide sequences), implemented by introducing only nucleic acid sequences derived from the host plant species or from a sexually compatible plant species. See, e.g., U.S. published application No. 2004\/0107455.","A genomic library is a collection of clones that contains at least one copy of essentially every DNA sequence in the genome.","The NBB1 sequence was identified by cDNA microarray prepared from a -derived cDNA library, pursuant to the protocol of Katoh et al., 79 (Ser. B): 151-154 (2003). NBB1 also is controlled by the nicotine biosynthesis regulatory loci, NIC1 and NIC2. NBB1 and PMT have the same pattern of expression in tobacco plants. That NBB1 is involved in nicotine biosynthesis is indicated by the fact that NBB1, like PMT and A622, is under the control of the NIC genes and exhibits a similar pattern of expression.","NBB1 expression refers to biosynthesis of a gene product encoded by SEQ ID NO: 3. NBB1 suppression refers to the reduction of NBB1 expression. NBB1 suppression has an ability to down-regulate nicotinic alkaloid content.","NIC1 and NIC2 loci are two independent genetic loci in , formerly designated as A and B. Mutations nic1 and nic2 reduce expression levels of nicotine biosynthesis enzymes and nicotine content, generally the nicotine content of wild type>homozygous nic2>homozygous nic1>homozygous nic1 and homozygous nic2 plants. Legg & Collins, 13:287 (1971); Hibi et al., 6: 723-735 (1994); Reed & Jelesko, 167:1123 (2004).","Nicotine is the major alkaloid in and some other species in the genus. Other plants have nicotine-producing ability, including, for example, and genera in the , and and genera in the ","Plant is a term that encompasses whole plants, plant organs (e.g. leaves, stems, roots, etc.), seeds, and plant cells and progeny of the same. Plant material includes, without limitation, seeds suspension cultures, embryos, meristematic regions, callus tissues, leaves, roots and shoots, gametophytes, sporophytes, pollen, and microspores. The class of plants which can be used in the present invention is generally as broad as the class of higher plants amenable to transformation techniques, including both monocotyledonous and dicotyledonous plants. A preferred plant is a plant having nicotine-producing ability of the and genera in the or the and genera in the . A particularly preferred plant is ","Protein refers to a polymer of amino acid residues.","Reduced-nicotine plant encompasses a genetically engineered plant that contains less than half, preferably less than 25%, and more preferably less than 20% or less than 10% of the nicotine content of a non-transgenic control plant of the same type. A reduced-nicotine plant also includes a genetically engineered plant that contains less total alkaloids compared with a control plant.","A structural gene refers to a DNA sequence that is transcribed into messenger RNA (mRNA) which is then translated into a sequence of amino acids characteristic of a specific polypeptide. \u201cMessenger RNA (mRNA)\u201d denotes an RNA molecule that contains the coded information for the amino acid sequence of a protein.","Sequence identity or \u201cidentity\u201d in the context of two nucleic acid or polypeptide sequences includes reference to the residues in the two sequences which are the same when aligned for maximum correspondence over a specified region. When percentage of sequence identity is used in reference to proteins it is recognized that residue positions which are not identical often differ by conservative amino acid substitutions, where amino acid residues are substituted for other amino acid residues with similar chemical properties, such as charge and hydrophobicity, and therefore do not change the functional properties of the molecule. Where sequences differ in conservative substitutions, the percent sequence identity may be adjusted upwards to correct for the conservative nature of the substitution. Sequences which differ by such conservative substitutions are said to have \u201csequence similarity\u201d or \u201csimilarity.\u201d Means for making this adjustment are well-known to those of skill in the art. Typically this involves scoring a conservative substitution as a partial rather than a full mismatch, thereby increasing the percentage sequence identity. Thus, for example, where an identical amino acid is given a score of 1 and a non-conservative substitution is given a score of zero, a conservative substitution is given a score between zero and 1. The scoring of conservative substitutions is calculated, for example, according to the algorithm of Meyers & Miller, 4: 11-17 (1988), as implemented in the program PC\/GENE (Intelligenetics, Mountain View, Calif., USA).","Use in this description of a percentage of sequence identity denotes a value determined by comparing two optimally aligned sequences over a comparison window, wherein the portion of the polynucleotide sequence in the comparison window may comprise additions or deletions (i.e., gaps) as compared to the reference sequence (which does not comprise additions or deletions) for optimal alignment of the two sequences. The percentage is calculated by determining the number of positions at which the identical nucleic acid base or amino acid residue occurs in both sequences to yield the number of matched positions, dividing the number of matched positions by the total number of positions in the window of comparison, and multiplying the result by 100 to yield the percentage of sequence identity.","Sequence identity has an art-recognized meaning and can be calculated using published techniques. See CMB, Lesk, ed. (Oxford University Press, 1988), B: IGP, Smith, ed. (Academic Press, 1993), CASD, PI, Griffin & Griffin, eds., (Humana Press, 1994), SAIMB, Von Heinje ed., Academic Press (1987), SAP, Gribskov & Devereux, eds. (Macmillan Stockton Press, 1991), and Carillo & Lipton, 48: 1073 (1988). Methods commonly employed to determine identity or similarity between two sequences include but are not limited to those disclosed in GHC, Bishop, ed., (Academic Press, 1994) and Carillo & Lipton, supra. Methods to determine identity and similarity are codified in computer programs. Preferred computer program methods to determine identity and similarity between two sequences include but are not limited to the GCG program package (Devereux et al., 12: 387 (1984)), BLASTP, BLASTN, FASTA (Atschul et al., 215: 403 (1990)), and FASTDB (Brutlag et al., 6: 237 (1990)).","Tobacco refers to any plant in the genus that produces nicotinic alkaloids. Tobacco also refers to products comprising material produced by a plant, and therefore includes, for example, cigarettes, cigars, chewing tobacco, snuff and cigarettes made from GE reduced-nicotine tobacco for use in smoking cessation. Examples of species include but are not limited to , and ","Tobacco-specific nitrosamines (TSNAs) are a class of carcinogens that are predominantly formed in tobacco during curing, processing, and smoking. Hoffman, D., et al., 58, 1841-4 (1977); Wiernik A et al., Recent Adv. Tob. Sci, (1995), 21: 39-80. TSNAs, such as 4-(N-nitrosomethylamino)-1-(3-pyridyl)-1-butanone (NNK), N\u2032-nitrosonornicotine (NNN), N\u2032-nitrosoanatabine (NAT), and N\u2032-nitrosoanabasine (NAB), are formed by N-nitrosation of nicotine and other minor alkaloids, such as nornicotine, anatabine, and anabasine. Reducing nicotinic alkaloids reduces the level of TSNAs in tobacco and tobacco products.","Tobacco hairy roots refers to tobacco roots that have T-DNA from an Ri plasmid of integrated in the genome and grow in culture without supplementation of auxin and other pytohormones. Tobacco hairy roots produce nicotinic alkaloids as roots of a tobacco plant do.","Transgenic plant refers to a plant that comprises a nucleic acid sequence that also is present per se in another organism or species or that is optimized, relative to host codon usage, from another organism or species.","A transgenic plant may be produced by any genetic transformation methodology. Suitable transformation methods include, for example, -mediated transformation, particle bombardment, electroporation, polyethylene glycol fusion, transposon tagging, and site-directed mutagenesis. Identification and selection of a trans genic plant are well-known techniques, the details of which need not be repeated here.","A variant is a nucleotide or amino acid sequence that deviates from the standard, or given, nucleotide or amino acid sequence of a particular gene or protein. The terms \u201cisoform,\u201d \u201cisotype,\u201d and \u201canalog\u201d also refer to \u201cvariant\u201d forms of a nucleotide or an amino acid sequence. An amino acid sequence that is altered by the addition, removal or substitution of one or more amino acids, or a change in nucleotide sequence, may be considered a \u201cvariant\u201d sequence. The variant may have \u201cconservative\u201d changes, wherein a substituted amino acid has similar structural or chemical properties, e.g., replacement of leucine with isoleucine. A variant may have \u201cnonconservative\u201d changes, e.g., replacement of a glycine with a tryptophan. Analogous minor variations may also include amino acid deletions or insertions, or both. Guidance in determining which amino acid residues may be substituted, inserted, or deleted may be found using computer programs well known in the art such as Vector NTI Suite (InforMax, MD) software. \u201cVariant\u201d may also refer to a \u201cshuffled gene\u201d such as those described in Maxygen-assigned patents.","The present invention is not limited to the particular methodology, protocols, vectors, and reagents, etc., described here, as these may vary. Furthermore, this specification employs the above-discussed terminology for the purpose of describing particular embodiments only and not to limit the scope of the invention.","The singular forms \u201ca,\u201d \u201can,\u201d and \u201cthe\u201d include plural reference unless the context clearly dictates otherwise. Thus, a reference to \u201ca gene\u201d is a reference to one or more genes and encompasses equivalents thereof that are known to the skilled person, and so forth.","Polynucleotide Sequences","Nicotinic alkaloid biosynthesis genes have been identified in several plant species, exemplified by plants. Accordingly, the present invention embraces any nucleic acid, gene, polynucleotide, DNA, RNA, mRNA, or cDNA molecule that is isolated from the genome of a plant species that down-regulates nicotinic alkaloid biosynthesis.","For example, suppression of at least one of A622 and NBB1, may be used to down-regulate nicotine content in a plant. Additionally, nicotinic alkaloid levels can be reduced further by suppressing expression of a nicotine biosynthesis gene, such as at least one of QPT and PMT, and at least one of A622 and NBB1. Plants with suppression of multiple genes may be obtained by regeneration of plants from plant cells genetically engineered for suppression of multiple genes or by crossing a first plant genetically engineered for suppression of a nicotine biosynthesis gene with a second plant genetically engineered for suppression of at least one of A622 and NBB1.","In one aspect, the invention provides an isolated nucleic acid molecule comprising SEQ ID NO: 1; polynucleotide sequences encoding a polypeptide set forth in SEQ ID NO: 2; polynucleotide sequences which hybridize to SEQ ID NO: 1 and encode an A622 polypeptide; and polynucleotide sequences which differ from SEQ ID NO: 1 due to the degeneracy of the genetic code. A peptide encoded by SEQ ID NO: 1 is a further aspect of the invention and is set forth in SEQ ID NO: 2.","In another aspect, the invention provides an isolate nucleic acid molecule comprising SEQ ID NO: 3; polynucleotide sequences encoding a polypeptide set forth in SEQ ID NO: 4; polynucleotide sequences which hybridize to SEQ ID NO: 3 and encode an NBB1 polypeptide; and polynucleotide sequences which differ from SEQ ID NO: 3 due to the degeneracy of the genetic code. A peptide encoded by SEQ ID NO: 3 is a further aspect of the invention and is set forth in SEQ ID NO: 4","The invention further provides nucleic acids that are complementary to SEQ ID NO: 1 or 3, as well as a nucleic acid, comprising at least 15 contiguous bases, that hybridizes to SEQ ID NO: 1 or 3 under moderate or high stringency conditions, as described below. For the purposes of this description, the category of nucleic acids that hybridize to SEQ ID NO: 3 is exclusive of a nucleic acid having the sequence of SEQ ID NO: 559, disclosed in published international application WO 03\/097790, and of any fragment thereof.","In a further embodiment, a siRNA molecule of the invention comprises a polynucleotide sequence that suppresses expression of either of SEQ ID NO. 1 or 3, although the sequences set forth in SEQ ID NO: 1 or 3 are not limiting. A siRNA molecule of the invention can comprise any contiguous A622 or NBB1 sequence, e.g., about 15 to about 25 or more, or about 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, or 25 or more contiguous nucleotides. In this context, too, the category of siRNA molecules is exclusive of a molecule having the nucleotide sequence of the aforementioned SEQ ID NO: 559 in WO 03\/097790, as well as any fragment thereof.","By \u201cisolated\u201d nucleic acid molecule(s) is intended a nucleic acid molecule, DNA or RNA, which has been removed from its native environment. For example, recombinant DNA molecules contained in a DNA construct are considered isolated for the purposes of the present invention. Further examples of isolated DNA molecules include recombinant DNA molecules maintained in heterologous host cells or DNA molecules that are purified, partially or substantially, in solution. Isolated RNA molecules include in vitro RNA transcripts of the DNA molecules of the present invention. Isolated nucleic acid molecules, according to the present invention, further include such molecules produced synthetically.","Nucleic acid molecules of the present invention may be in the form of RNA, such as mRNA, or in the form of DNA, including, for instance, cDNA and genomic DNA obtained by cloning or produced synthetically. The DNA or RNA may be double-stranded or single-stranded. Single-stranded DNA may be the coding strand, also known as the sense strand, or it may be the non-coding strand, also called the anti-sense strand.","Unless otherwise indicated, all nucleotide sequences determined by sequencing a DNA molecule herein were determined using an automated DNA sequencer (such as the Model 373 from Applied Biosystems, Inc.). Therefore, as is known in the art for any DNA sequence determined by this automated approach, any nucleotide sequence determined herein may contain some errors. Nucleotide sequences determined by automation are typically at least about 95% identical, more typically at least about 96% to at least about 99.9% identical to the actual nucleotide sequence of the sequenced DNA molecule. The actual sequence can be more precisely determined by other approaches including manual DNA sequencing methods well known in the art. As is also known in the art, a single insertion or deletion in a determined nucleotide sequence compared to the actual sequence will cause a frame shift in translation of the nucleotide sequence such that the predicted amino acid sequence encoded by a determined nucleotide sequence may be completely different from the amino acid sequence actually encoded by the sequenced DNA molecule, beginning at the point of such an insertion or deletion.","In another aspect, the invention provides an isolated nucleic acid molecule comprising a polynucleotide that hybridizes, under stringent hybridization conditions, to a portion of the polynucleotide in a nucleic acid molecule of the invention, as described above. By a polynucleotide that hybridizes to a \u201cportion\u201d of a polynucleotide is intended a polynucleotide, either DNA or RNA, hybridizing to at least about 15 nucleotides, and more preferably at least about 20 nucleotides, and still more preferably at least about 30 nucleotides, and even more preferably more than 30 nucleotides of the reference polynucleotide.","For the purpose of the invention, two sequences hybridize when they form a double-stranded complex in a hybridization solution of 6\u00d7SSC, 0.5% SDS, 5\u00d7Denhardt's solution and 100 \u03bcg of non-specific carrier DNA. See Ausubel et al., supra, at section 2.9, supplement 27 (1994). Sequences may hybridize at \u201cmoderate stringency,\u201d which is defined as a temperature of 60\u00b0 C. in a hybridization solution of 6\u00d7SSC, 0.5% SDS, 5\u00d7Denhardt's solution and 100 \u03bcg of non-specific carrier DNA. For \u201chigh stringency\u201d hybridization, the temperature is increased to 68\u00b0 C. Following the moderate stringency hybridization reaction, the nucleotides are washed in a solution of 2\u00d7SSC plus 0.05% SDS for five times at room temperature, with subsequent washes with 0.1\u00d7SSC plus 0.1% SDS at 60\u00b0 C. for 1 h. For high stringency, the wash temperature is increased to 68\u00b0 C. For the purpose of the invention, hybridized nucleotides are those that are detected using 1 ng of a radiolabeled probe having a specific radioactivity of 10,000 cpm\/ng, where the hybridized nucleotides are clearly visible following exposure to X-ray film at \u221270\u00b0 C. for no more than 72 hours.","The present application is directed to such nucleic acid molecules which are at least 60%, 65%, 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, 99% or 100% identical to a nucleic acid sequence described in of SEQ ID NO: 1 or 3, Preferred are nucleic acid molecules which are at least 95%, 96%, 97%, 98%, 99% or 100% identical to the nucleic acid sequence shown in of SEQ ID NO: 1 or 3. Differences between two nucleic acid sequences may occur at the 5\u2032 or 3\u2032 terminal positions of the reference nucleotide sequence or anywhere between those terminal positions, interspersed either individually among nucleotides in the reference sequence or in one or more contiguous groups within the reference sequence.","As a practical matter, whether any particular nucleic acid molecule is at least 95%, 96%, 97%, 98% or 99% identical to a reference nucleotide sequence refers to a comparison made between two molecules using standard algorithms well known in the art and can be determined conventionally using publicly available computer programs such as the BLASTN algorithm. See Altschul et al., 25: 3389-3402 (1997).","The heterologous sequence utilized in the antisense methods of the present invention may be selected so as to produce an RNA product complementary to an entire A622 or NBB1 mRNA sequence, or to a portion thereof. The sequence may be complementary to any contiguous sequence of the natural messenger RNA, that is, it may be complementary to the endogenous mRNA sequence proximal to the 5\u2032-terminus or capping site, downstream from the capping site, between the capping site and the initiation codon and may cover all or only a portion of the non-coding region, may bridge the non-coding and coding region, be complementary to all or part of the coding region, complementary to the 3\u2032-terminus of the coding region, or complementary to the 3\u2032-untranslated region of the mRNA.","Suitable antisense sequences may be from at least about 13 to about 15 nucleotides, at least about 16 to about 21 nucleotides, at least about 20 nucleotides, at least about 30 nucleotides, at least about 50 nucleotides, at least about 75 nucleotides, at least about 100 nucleotides, at least about 125 nucleotides, at least about 150 nucleotides, at least about 200 nucleotides, or more. In addition, the sequences may be extended or shortened on the 3\u2032 or 5\u2032 ends thereof.","The particular antisense sequence and the length of the antisense sequence will vary, depending, for example, upon the degree of inhibition desired and the stability of the antisense sequence. Generally available techniques and the information provided in this specification can guide the selection of appropriate A622 or NBB1 antisense sequences. With reference to SEQ ID NO: 1 or 3 herein, an oligonucleotide of the invention may be a continuous fragment of A622 or NBB1 cDNA sequence in antisense orientation, of any length that is sufficient to achieve the desired effects when transformed into a recipient plant cell.","The present invention may contemplate sense co-suppression of one or both of A622 and NBB1. Sense polynucleotides employed in carrying out the present invention are of a length sufficient to suppress, when expressed in a plant cell, the native expression of the plant A622 or NBB1 protein in that plant cell. Such sense polynucleotides may be essentially an entire genomic or complementary nucleic acid encoding the A622 or NBB1 enzyme, or a fragment thereof, with such fragments typically being at least 15 nucleotides in length. Techniques are generally available for ascertaining the length of sense DNA that results in suppression of the expression of a native gene in a cell.","In an alternate embodiment of the present invention, plant cells are transformed with a nucleic acid construct containing a polynucleotide segment encoding an enzymatic RNA molecule (a \u201cribozyme\u201d), which enzymatic RNA molecule is directed against (i.e., cleaves) the mRNA transcript of DNA encoding A622 or NBB1, as described herein. Ribozymes contain substrate binding domains that bind to accessible regions of the target mRNA, and domains that catalyze the cleavage of RNA, preventing translation and protein production. The binding domains may comprise antisense sequences complementary to the target mRNA sequence; the catalytic motif may be a hammerhead motif or other motifs, such as the hairpin motif.","Ribozyme cleavage sites within an RNA target may initially be identified by scanning the target molecule for ribozyme cleavage sites (e.g., GUA, GUU or GUC sequences). Once identified, short RNA sequences of 15, 20, 30, or more ribonucleotides corresponding to the region of the target gene containing the cleavage site may be evaluated for predicted structural features.","The suitability of candidate targets also may be evaluated by testing their accessibility to hybridization with complimentary oligonucleotides, using ribonuclease protection assays as are known in the art. DNA encoding enzymatic RNA molecules may be produced in accordance with known techniques. For example, see Cech et al., U.S. Pat. No. 4,987,071; Keene et al., U.S. Pat. No. 5,559,021; Donson et al., U.S. Pat. No. 5,589,367; Torrence et al., U.S. Pat. No. 5,583,032; Joyce, U.S. Pat. No. 5,580,967; Gold et al., U.S. Pat. No. 5,595,877; Wagner et al., U.S. Pat. Nos. 5,591,601; and 5,622,854.","Production of such an enzymatic RNA molecule in a plant cell and disruption of A622 or NBB1 protein production reduces protein activity in plant cells, in essentially the same manner as production of an antisense RNA molecule; that is, by disrupting translation of mRNA in the cell which produces the enzyme. The term \u201cribozyme\u201d describes an RNA-containing nucleic acid that functions as an enzyme, such as an endoribonuclease, and may be used interchangeably with \u201cenzymatic RNA molecule.\u201d","The present invention further includes nucleic acids encoding ribozymes, nucleic acids that encode ribozymes and that have been inserted into an expression vector, host cells containing such vectors, and methodology employing ribozymes to decrease A622 and NBB1 expression in plants.","In one embodiment, the present invention provides double-stranded nucleic acid molecules of that mediate RNA interference gene silencing. In another embodiment, the siNA molecules of the invention consist of duplex nucleic acid molecules containing about 15 to about 30 base pairs between oligonucleotides comprising about 15 to about 30 (e.g., about 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, or 30) nucleotides. In yet another embodiment, siNA molecules of the invention comprise duplex nucleic acid molecules with overhanging ends of about 1 to about 32 (e.g., about 1, 2, or 3) nucleotides, for example, about 21-nucleotide duplexes with about 19 base pairs and 3\u2032-terminal mononucleotide, dinucleotide, or trinucleotide overhangs. In yet another embodiment, siNA molecules of the invention comprise duplex nucleic acid molecules with blunt ends, where both ends are blunt, or alternatively, where one of the ends is blunt.","An siNA molecule of the present invention may comprise modified nucleotides while maintaining the ability to mediate RNAi. The modified nucleotides can be used to improve in vitro or in vivo characteristics such as stability, activity, and\/or bioavailability. For example, a siNA molecule of the invention can comprise modified nucleotides as a percentage of the total number of nucleotides present in the siNA molecule. As such, a siNA molecule of the invention can generally comprise about 5% to about 100% modified nucleotides (e.g., about 5%, 10%, 15%, 20%, 25%, 30%, 35%, 40%, 45%, 50%, 55%, 60%, 65%, 70%, 75%, 80%, 85%, 90%, 95% or 100% modified nucleotides). The actual percentage of modified nucleotides present in a given siNA molecule will depend on the total number of nucleotides present in the siNA. If the siNA molecule is single stranded, the percent modification can be based upon the total number of nucleotides present in the single stranded siNA molecules. Likewise, if the siNA molecule is double stranded, the percent modification can be based upon the total number of nucleotides present in the sense strand, antisense strand, or both the sense and antisense strands.","For example, A622 and NBB1 expression may be decreased through genetic engineering methods that are well known in the art. Expression can be reduced by introducing a nucleic acid construct that results in expression of an RNA comprising a portion of a sequence encoding A622 or NBB1. The portion of the sequence may be in the sense or antisense orientation. The portion of the sequence may be present in inverted repeats capable of forming a double-stranded RNA region. Expression may be reduced by introducing a nucleic acid construct encoding an enzymatic RNA molecule (i.e., a \u201cribozyme\u201d), which enzymatic RNA molecule is directed against (i.e., cleaves) the mRNA transcript of DNA encoding A622 or NBB1. Expression may be reduced by introducing a nucleic acid comprising a portion of an A622 or NBB1 sequence that causes targeted in situ mutagenesis of an endogenous gene, resulting in its inactivation.","In one embodiment of the present invention, plant cells are transformed with a nucleic acid construct containing a mutant allele of one or both of A622 and NBB1 that comprises a polynucleotide sequence that suppresses expression of one or both of A622 and NBB1. Mutant alleles according to the invention may arise from antisense sequence suppression of one or both of A622 and NBB1, or sense co-suppression of one or both of A622 and NBB1, as described herein. Thus, a mutant allele according to the invention may comprises an antisense nucleic acid sequence that expresses a short interfering RNA that suppresses expression of one or both of A622 and NBB1 or expression of a gene product encoded by SEQ ID NO: 1 or SEQ ID NO: 3. In an alternative embodiment, the mutant allele may comprise a sense nucleic acid sequence that suppresses expression of one or both of A622 and NBB1 or expression of a gene product encoded by SEQ ID NO: 1 or SEQ ID NO: 3. In yet another embodiment, the mutant allele according to the invention comprises a nucleic acid sequence that encodes a ribozyme which cleaves one or both of A622 and NBB1 transcripts. In a preferred embodiment, mutant alleles are at least 60%, 65%, 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% identical to a nucleic acid sequence described in of SEQ ID NO: 1 or 3. Preferred are mutant alleles which are at least 95%, 96%, 97%, 98%, 99% or 100% identical to the nucleic acid sequence set forth in SEQ ID NO: 1 or 3. Differences between two mutant alleles may occur at the 5\u2032 or 3\u2032 terminal positions of the reference nucleotide sequence or anywhere between those terminal positions, interspersed either individually among nucleotides in the reference sequence or in one or more contiguous groups within the reference sequence.","Sequence Analysis","Methods of alignment of sequences for comparison are well-known in the art. Optimal alignment of sequences for comparison may be conducted by the local homology algorithm of Smith and Waterman, 2: 482 (1981); by the homology alignment algorithm of Needleman and Wunsch, 48: 443 (1970); by the search for similarity method of Pearson and Lipman, . USA 85: 2444 (1988); by computerized implementations of these algorithms, including, but not limited to: CLUSTAL in the PC\/Gene program by Intelligenetics, Mountain View, Calif.; GAP, BESTFIT, BLAST, FASTA, and TFASTA in the Wisconsin Genetics Software Package, Genetics Computer Group (GCG), 575 Science Dr., Madison, Wis., USA; the CLUSTAL program is well described by Higgins and Sharp, 73: 237-244 (1988); Higgins and Sharp, CABIOS 5: 151-153 (1989); Corpet et al., 16: 10881-90 (1988); Huang et al., 8: 155-65 (1992), and Pearson et al., 24: 307-331 (1994).","The BLAST family of programs that can be used for database similarity searches includes: BLASTN for nucleotide query sequences against nucleotide database sequences; BLASTX for nucleotide query sequences against protein database sequences; BLASTP for protein query sequences against protein database sequences; TBLASTN for protein query sequences against nucleotide database sequences; and TBLASTX for nucleotide query sequences against nucleotide database sequences. See, , Chapter 19, Ausubel et al., Eds., Greene Publishing and Wiley-Interscience, New York (1995); Altschul et al., 215:403-410 (1990); and, Altschul et al., 25:3389-3402 (1997).","Software for performing BLAST analyses is publicly available, e.g., through the National Center for Biotechnology Information. This algorithm involves first identifying high scoring sequence pairs (HSPs) by identifying short words of length W in the query sequence, which either match or satisfy some positive-valued threshold score T when aligned with a word of the same length in a database sequence. T is referred to as the neighborhood word score threshold. These initial neighborhood word hits act as seeds for initiating searches to find longer HSPs containing them. The word hits are then extended in both directions along each sequence for as far as the cumulative alignment score can be increased. Cumulative scores are calculated using, for nucleotide sequences, the parameters M (reward score for a pair of matching residues; always >0) and N (penalty score for mismatching residues; always <0). For amino acid sequences, a scoring matrix is used to calculate the cumulative score. Extension of the word hits in each direction are halted when: the cumulative alignment score falls off by the quantity X from its maximum achieved value; the cumulative score goes to zero or below, due to the accumulation of one or more negative-scoring residue alignments; or the end of either sequence is reached. The BLAST algorithm parameters W, T, and X determine the sensitivity and speed of the alignment. The BLASTN program (for nucleotide sequences) uses as defaults a wordlength (W) of 11, an expectation (E) of 10, a cutoff of 100, M=5, N=\u22124, and a comparison of both strands. For amino acid sequences, the BLASTP program uses as defaults a wordlength (W) of 3, an expectation (E) of 10, and the BLOSUM62 scoring matrix. See Henikoff & Henikoff, 89:10915 (1998).","In addition to calculating percent sequence identity, the BLAST algorithm also performs a statistical analysis of the similarity between two sequences (see, e.g., Karlin & Altschul, 90:5873-5877 (1993)). One measure of similarity provided by the BLAST algorithm is the smallest sum probability (P(N)), which provides an indication of the probability by which a match between two nucleotide or amino acid sequences would occur by chance.","Multiple alignment of the sequences can be performed using the CLUSTAL method of alignment (Higgins & Sharp, 5:151-153 (1989)) with the default parameters (GAP PENALTY=10, GAP LENGTH PENALTY=10). Default parameters for pairwise alignments using the CLUSTAL method are KTUPLE 1, GAP PENALTY=3, WINDOW=5 and DIAGONALS SAVED=5.","The following running parameters are preferred for determining alignments and similarities using BLASTN that contribute to the E values and percentage identity for polynucleotide sequences: Unix running command: blastall -p blastn -d embldb -e 10-G0 -E0 -r 1 -v 30 -b 30 -i queryseq -o results; the parameters are: -p Program Name [String]; -d Database [String]; -e Expectation value (E) [Real]; -G Cost to open a gap (zero invokes default behavior) [Integer]; -E Cost to extend a gap (zero invokes default behavior) [Integer]; -r Reward for a nucleotide match (blastn only) [Integer]; -v Number of one-line descriptions (V) [Integer]; -b Number of alignments to show (B) [Integer]; -i Query File [File In]; and -o BLAST report Output File [File Out] Optional.","The \u201chits\u201d to one or more database sequences by a queried sequence produced by BLASTN, FASTA, BLASTP or a similar algorithm, align and identify similar portions of sequences. The hits are arranged in order of the degree of similarity and the length of sequence overlap. Hits to a database sequence generally represent an overlap over only a fraction of the sequence length of the queried sequence.","The BLASTN, FASTA and BLASTP algorithms also produce \u201cExpect\u201d values for alignments. The Expect value (E) indicates the number of hits one can \u201cexpect\u201d to see over a certain number of contiguous sequences by chance when searching a database of a certain size. The Expect value is used as a significance threshold for determining whether the hit to a database, such as the preferred EMBL database, indicates true similarity. For example, an E value of 0.1 assigned to a polynucleotide hit is interpreted as meaning that in a database of the size of the EMBL database, one might expect to see 0.1 matches over the aligned portion of the sequence with a similar score simply by chance. By this criterion, the aligned and matched portions of the polynucleotide sequences then have a probability of 90% of being the same. For sequences having an E value of 0.01 or less over aligned and matched portions, the probability of finding a match by chance in the EMBL database is 1% or less using the BLASTN or FASTA algorithm.","According to one embodiment, \u201cvariant\u201d polynucleotides, with reference to each of the polynucleotides of the present invention, preferably comprise sequences having the same number or fewer nucleic acids than each of the polynucleotides of the present invention and producing an E value of 0.01 or less when compared to the polynucleotide of the present invention. That is, a variant polynucleotide is any sequence that has at least a 99% probability of being the same as the polynucleotide of the present invention, measured as having an E value of 0.01 or less using the BLASTN, FASTA, or BLASTP algorithms set at parameters described above. Alternatively, variant polynucleotides of the present invention hybridize to the polynucleotide sequence of SEQ ID NO: 1 or 3, or complements, reverse sequences, or reverse complements of those sequences, under stringent conditions.","The present invention also encompasses polynucleotides that differ from the disclosed sequences but that, as a consequence of the degeneracy of the genetic code, encode a polypeptide which is the same as that encoded by a polynucleotide of the present invention. Thus, polynucleotides comprising sequences that differ from the polynucleotide sequences recited in SEQ ID NO: 1 or 3 or complements, reverse sequences, or reverse complements thereof, as a result of conservative substitutions are contemplated by and encompassed within the present invention. Additionally, polynucleotides comprising sequences that differ from the polynucleotide sequences recited in SEQ ID NO: 1 or 3, or complements, reverse complements or reverse sequences thereof, as a result of deletions and\/or insertions totaling less than 10% of the total sequence length are also contemplated by and encompassed within the present invention.","In addition to having a specified percentage identity to an inventive polynucleotide sequence, variant polynucleotides preferably have additional structure and\/or functional features in common with the inventive polynucleotide. In addition to sharing a high degree of similarity in their primary structure to polynucleotides of the present invention, polynucleotides having a specified degree of identity to, or capable of hybridizing to an inventive polynucleotide preferably have at least one of the following features: (i) they contain an open reading frame or partial open reading frame encoding a polypeptide having substantially the same functional properties as the polypeptide encoded by the inventive polynucleotide; or (ii) they have domains in common. For example, a variant polynucleotide may encode a polypeptide having the ability to suppress A622 or NBB1.","Nucleic Acid Constructs","In accordance with one aspect of the invention, a sequence that reduces nicotinic alkaloid biosynthesis is incorporated into a nucleic acid construct that is suitable for plant transformation. For example, such a nucleic acid construct can be used to decrease at least one of A622 or NBB1 gene expression in plants. Additionally, an inventive nucleic acid construct may decrease one or both of A622 and NBB1 expression, as well as a polynucleotide sequence encoding a nicotine biosynthesis enzyme.","Accordingly, nucleic acid constructs are provided that comprise a sequence that down-regulates nicotinic alkaloid biosynthesis, under the control of a transcriptional initiation region operative in a plant, so that the construct can generate RNA in a host plant cell.","Recombinant DNA constructs may be made using standard techniques. For example, the DNA sequence for transcription may be obtained by treating a vector containing said sequence with restriction enzymes to cut out the appropriate segment. The DNA sequence for transcription may also be generated by annealing and ligating synthetic oligonucleotides or by using synthetic oligonucleotides in a polymerase chain reaction (PCR) to give suitable restriction sites at each end. The DNA sequence then is cloned into a vector containing suitable regulatory elements, such as upstream promoter and downstream terminator sequences.","Suitable Regulatory Elements","Promoter connotes a region of DNA upstream from the start of transcription that is involved in recognition and binding of RNA polymerase and other proteins to initiate transcription. A \u201cconstitutive promoter\u201d is one that is active throughout the life of the plant and under most environmental conditions. Tissue-specific, tissue-preferred, cell type-specific, and inducible promoters constitute the class of \u201cnon-constitutive promoters.\u201d \u201cOperably linked\u201d refers to a functional linkage between a promoter and a second sequence, where the promoter sequence initiates and mediates transcription of the DNA sequence corresponding to the second sequence. In general, \u201coperably linked\u201d means that the nucleic acid sequences being linked are contiguous.","Promoters useful for expression of a nucleic acid sequence introduced into a cell to reduce expression of A622 or NBB1 may be constitutive promoters, or tissue-specific, tissue-preferred, cell type-specific, and inducible promoters. Preferred promoters include promoters which are active in root tissues, such as the tobacco RB7promoter (Hsu et al. 44:9-19 (1995); U.S. Pat. No. 5,459,252) and promoters that are activated under conditions that result in elevated expression of enzymes involved in nicotine biosynthesis such as the tobacco RD2 promoter (U.S. Pat. No. 5,837,876), PMT promoters (Shoji T. et al., 41:831-839 (2000b); WO 2002\/038588) or an A622 promoter (Shoji T. et al., 50:427-440 (2002)).","The vectors of the invention may also contain termination sequences, which are positioned downstream of the nucleic acid molecules of the invention, such that transcription of mRNA is terminated, and polyA sequences added. Exemplary of such terminators are the cauliflower mosaic virus (CaMV) 35S terminator and the nopaline synthase gene (Tnos) terminator. The expression vector also may contain enhancers, start codons, splicing signal sequences, and targeting sequences.","Expression vectors of the invention may also contain a selection marker by which transformed plant cells can be identified in culture. The marker may be associated with the heterologous nucleic acid molecule, i.e., the gene operably linked to a promoter. As used herein, the term \u201cmarker\u201d refers to a gene encoding a trait or a phenotype that permits the selection of, or the screening for, a plant or plant cell containing the marker. Usually, the marker gene will encode antibiotic or herbicide resistance. This allows for selection of transformed cells from among cells that are not transformed or transfected.","Examples of suitable selectable markers include adenosine deaminase, dihydrofolate reductase, hygromycin-B-phosphotransferase, thymidne kinase, xanthine-guanine phospho-ribosyltransferase, glyphosate and glufosinate resistance, and amino-glycoside 3\u2032-O-phosphotranserase (kanamycin, neomycin and G418 resistance). These markers include resistance to G418, hygromycin, bleomycin, kanamycin, and gentamicin. The construct may also contain the selectable marker gene Bar that confers resistance to herbicidal phosphinothricin analogs like ammonium gluphosinate. Thompson et al., 9: 2519-2523 (1987). Other suitable selection markers are known as well.","Visible markers such as green florescent protein (GFP) may be used. Methods for identifying or selecting transformed plants based on the control of cell division have also been described. See WO 2000\/052168 and WO 2001\/059086.","Replication sequences, of bacterial or viral origin, may also be included to allow the vector to be cloned in a bacterial or phage host. Preferably, a broad host range prokaryotic origin of replication is used. A selectable marker for bacteria may be included to allow selection of bacterial cells bearing the desired construct. Suitable prokaryotic selectable markers also include resistance to antibiotics such as kanamycin or tetracycline.","Other DNA sequences encoding additional functions may also be present in the vector, as is known in the art. For instance, when is the host, T-DNA sequences may be included to facilitate the subsequent transfer to and incorporation into plant chromosomes.","Plants for Genetic Engineering","The present invention comprehends the genetic manipulation of a plant to suppress nicotinic alkaloid synthesis via introducing a polynucleotide sequence that down-regulates expression of a gene, such as A622 and NBB1, that regulates nicotinic alkaloid synthesis. The result is a plant having reduced nicotinic alkaloid levels.","In this description, \u201cplant\u201d denotes any nicotinic alkaloid containing plant material that can be genetically manipulated, including but not limited to differentiated or undifferentiated plant cells, protosomes, whole plants, plant tissues, or plant organs, or any component of a plant such as a leaf, stem, root, bud, tuber, fruit, rhizome, or the like.","Illustrative plants that can be engineered in accordance with the invention include but are not limited to tobacco, potato, tomato, egg plant, green pepper, and ","Plant Transformation and Selection","Constructs according to the invention may be used to transform any plant cell, using a suitable transformation technique. Both monocotyledonous and dicotyledonous angiosperm or gymnosperm plant cells may be transformed in various ways known to the art. For example, see Klein et al., 4: 583-590 (1993); Bechtold et al., 316:1194-1199 (1993); Bent et al., 204:383-396 (1986); Paszowski et al., 3: 2717-2722 (1984); Sagi et al., 13: 262-266 (1994). species such as and can be used, for example, in accordance with Nagel et al., 67: 325 (1990). Additionally, plants may be transformed by or transformation. Broothaerts et al., 433:629-633 (2005).","For example, may be transformed with a plant expression vector via, e.g., electroporation, after which the is introduced to plant cells via, e.g., the well known leaf-disk method. Additional methods for accomplishing this include, but are not limited to, electroporation, particle gun bombardment, calcium phosphate precipitation, and polyethylene glycol fusion, transfer into germinating pollen grains, direct transformation (Lorz et al., 199: 179-182 (1985)), and other methods known to the art. If a selection marker, such as kanamycin resistance, is employed, it makes it easier to determine which cells have been successfully transformed.","The transformation methods discussed above are known to be useful for transforming dicots. Additionally, de la Pena et al., 325: 274-276 (1987), Rhodes et al., 240: 204-207 (1988), and Shimamato et al., 328: 274-276 (1989) have transformed cereal monocots using . Also see Bechtold et al., 316 (1994), illustrating vacuum infiltration for -mediated transformation.","For the purposes of this description, a plant or plant cell may be transformed with a plasmid comprising one or more sequences, each operably linked to a promoter. For example, an illustrative vector may comprise a QPT sequence operably linked to a promoter. Likewise, the plasmid may comprise a QPT sequence operably linked to a promoter and an A622 sequence operably linked to a promoter. Alternatively, a plant or plant cell may be transformed with more than one plasmid. For example, a plant or plant cell may be transformed with a first plasmid comprising a QPT sequence operably linked to a promoter, which is distinct from a second plasmid comprising an A622 or NBB1 sequence. Of course, the first and second plasmids or portions thereof are introduced into the same plant cell","Genetically engineered plants of the invention may be produced by conventional breeding. For example, a genetically engineered plant having down-regulated QPT and A622 activity may be produced by crossing a transgenic plant having reduced QPT expression with a transgenic plant having reduced A622 expression. Following successive rounds of crossing and selection, a genetically engineered plant having down-regulated QPT and A622 activity can be selected.","The presence of a protein, polypeptide, or nucleic acid molecule in a particular cell can be measured to determine if, for example, a cell has been successfully transformed or transfected.","Marker genes may be included within pairs of recombination sites recognized by specific recombinases such as cre or flp to facilitate removal of the marker after selection. See U.S. published application No. 2004\/0143874.","Transgenic plants without marker genes may be produced using a second plasmid comprising a nucleic acid encoding the marker, distinct from a first plasmid that comprises an A622 or NBB1 sequence. The first and second plasmids or portions thereof are introduced into the same plant cell, such that the selectable marker gene that is transiently expressed, transformed plant cells are identified, and transformed plants are obtained in which the A622 or NBB1 sequence is stably integrated into the genome and the selectable marker gene is not stably integrated. See U.S. published application No. 2003\/0221213. The first plasmid that comprises an A622 or NBB1 sequence may optionally be a binary vector with a T-DNA region that is completely made up of nucleic acid sequences present in wild-type non-transgenic or sexually compatible species.","Plant cells may be transformed with nucleic acid constructs of the present invention without the use of a selectable or visible marker and transgenic plant tissue and transgenic regenerated plants may be identified by detecting the presence of the introduced construct by PCR or other methods of detection of specific nucleic acid sequences. Identification of transformed plant cells may be facilitated by recognition of differences in the growth rate or a morphological feature of said transformed plant cell compared to the growth rate or a morphological feature of a non-transformed plant cell that is cultured under similar conditions (see WO 2004\/076625).","Methods of regenerating a transgenic plant from a transformed cell or culture vary according to the plant species but are based on known methodology. For example, methods for regenerating of transgenic tobacco plants are well-known.","For the purposes of the present description, genetically engineered plants are selected that have down-regulated expression of at least one of A622 and NBB1. Additionally, the inventive genetically engineered plants may have down-regulated expression of a nicotine biosynthesis gene, such as QPT or PMT, and at least one of A622 and NBB1.","Nicotine serves as a natural pesticide which helps protect tobacco plants from damage by pests and susceptibility of conventionally bred or transgenic low-nicotine tobacco to insect damage has been reported to increase. Legg, P. D., et al., 13:287-291 (1971); Voelckel, C., et al., 11:121-126 (2001); Steppuhn, A., et al., 2(8): e217: 1074-1080 (2004). It may therefore be desirable to additionally transform reduced-nicotine plants produced by the present methods with a transgene that will confer additional insect protection, such as gene encoding a Bt insecticidal protein, proteinase inhibitor, or biotin-binding protein. A transgene conferring additional insect protection may be introduced by crossing a transgenic reduced-nicotine plant with a second transgenic plant containing a gene encoding an insect resistance protein.","Quantifying Nicotinic Alkaloid Content","Transgenic plants of the invention are characterized by decreased nicotinic alkaloid content. Decreased nicotinic alkaloid content in the genetically engineered plant is preferably achieved via decreased expression of a nicotine biosynthesis pathway gene, such as A622 or NBB1.","In describing a plant of the invention, the phrase \u201creduced-nicotine or nicotinic alkaloid content\u201d refers to a quantitative reduction in the amount of nicotinic alkaloid in the plant when compared with a non-transformed control plant. A quantitative decrease in nicotinic alkaloid levels can be assayed by several methods, as for example by quantification based on gas-liquid chromatography, high performance liquid chromatography, radio-immunoassays, and enzyme-linked immunosorbent assays. In the present invention, nicotinic alkaloid levels were measured by gas-liquid chromatography equipped with a capillary column and an FID detector, as described in Hibi, N. et al., 100: 826-835 (1992).","Reduced-Nicotinic-Alkaloid Products","The present invention provides a transgenic plant having reduced-nicotinic-alkaloid levels. For example, the instant invention contemplates reducing nicotine levels by suppressing at least one of A622 and NBB1 expression. Following selection of a transgenic plant having suppressed A622 or NBB1 expression and reduced-nicotine content, a variety of products may be made from such a plant.","Because the invention provides a method for reducing alkaloids, TSNAs may also be reduced because there is a significant, positive correlation between alkaloid content in tobacco and TSNA accumulation. For example, a significant correlation coefficient between anatabine and NAT was 0.76. Djordjevic et al., 37: 752-756 (1989). TSNAs are a class of carcinogens that are predominantly formed in tobacco during curing, processing, and smoking. However, TSNAs are present in small quantities in growing tobacco plants or fresh cut tobacco. Hecht & Hoffman, 58, 1841-4 (1977); Wiernik et al., 21: 39-80 (1995). Nitrosamines, containing the organic functional group, N\u2014N\u2550O, are formed from the facile addition of an N\u2550O group by a nitrosating agent to a nitrogen of a secondary or tertiary amine. This particular class of carcinogens is found only in tobacco although they could potentially occur in other nicotine-containing products.","TSNAs are considered to be among the most prominent carcinogens in cigarette smoke and their carcinogenic properties are well documented. See Hecht, S. 424:127-42 (1999); Hecht, S. 11, 559-603 (1998); Hecht, S., et al., 8, 273-294 (1989). TSNAs have been cited as causes of oral cancer, esophageal cancer, pancreatic cancer, and lung cancer (Hecht & Hoffman, 54-61 (1991)). In particular, TSNAs have been implicated as the causative agent in the dramatic rise of adenocarcinoma associated with cigarette smoking and lung cancer (Hoffmann et al., 26, 199-211 (1996)).","The four TSNAs considered to be the most important by levels of exposure and carcinogenic potency and reported to be possibly carcinogenic to humans are N\u2032-nitrosonornicotine (NNN), 4-methylnitrosoamino-1-(3-pyridyl)-1-butanone (NNK), N\u2032-nitrosoanatabine (NAT) and N\u2032-nitrosoanabasine (NAB) Reviewed in . Lyon (France) Vol 37, pp. 205-208 (1985). These TSNAs are formed by N-nitrosation of nicotine and of the minor alkaloids that include nornicotine, anatabine, and anabasine.","The following levels of alkaloid compounds have been reported for mainstream smoke of non-filter cigarettes (measured in \u03bcg\/cigarette); nicotine: 100-3000, nornicotine: 5-150, anatabine: 5-15, Anabasine: 5-12 (Hoffmann at al., 14:7:767-790 (2000)). Mainstream smoke of U.S. cigarettes, with or without filter tips, contain (measured in ng\/cigarette): 9-180 ng NNK, 50-500 ng NNN, 3-25 ng NAB and 55-300 ng NAT. Hoffmann, et al., 41:1-52 (1994). It is important to note that the levels of these TSNAs in sidestream smoke are 5-10 fold above those in mainstream smoke. Hoffmann, et al (1994).","Xie et al. (2004) reported that Vector 21-41, which is a GE reduced-nicotine tobacco by the down-regulation of QPT, has a total alkaloid level of about 2300 ppm, which is less than 10 percent of the wild-type tobacco. Mainstream smoke from the Vector 21-41 cigarettes had less than 10 percent of NNN, NAT, NAB, and NNK compared to such levels of a standard full flavor cigarette produced from wild-type tobacco.","The strategy for reducing TSNAs by reducing the corresponding tobacco alkaloid precursors is currently the main focus of agricultural tobacco research. Siminszky et al., 102(41) 14919-14924 (2005). Thus, to reduce formation of all TSNAs there is an urgent need to reduce the precursor nicotinic alkaloids as much as possible by genetic engineering.","Among others, U.S. Pat. Nos. 5,803,081, 6,135,121, 6,805,134, 6,907,887 and 6,959,712 and U.S. Published Application Nos. 2005\/0034365 and 2005\/0072047 discuss methods to reduce tobacco-specific nitrosamines (TSNAs).","A reduced-nicotine tobacco product may be in the form of leaf tobacco, shredded tobacco, cut tobacco and tobacco fractions. A reduced-nicotine tobacco product may include cigarette tobacco, cigar tobacco, snuff, chewing tobacco, pipe tobacco, and cigarettes made from GE reduced-nicotine tobacco for use in smoking cessation.","Reduced-nicotine tobacco may also be used to produce reconstituted tobacco (Recon). Recon is produced from tobacco stems and\/or smaller leaf particles by a process that closely resembles typical paper making. This process entails processing the various tobacco portions that are to be made into Recon and cutting the tobacco into a size and shape that resembles cut rag tobacco made from whole leaf tobacco. This cut recon then gets mixed with cut-rag tobacco and is ready for cigarette making.","In addition to traditional tobacco products, such as cigarette and cigar tobacco, reduced-nicotine tobacco can be used as source for protein, fiber, ethanol, and animal feeds. See U.S. published application No. 2002\/0197688. For example, reduced-nicotine tobacco may be used as a source of Rubisco (ribulose bisphosphate carboxylase-oxygenase or fraction 1 protein) because unlike other plants, tobacco-derived Rubisco can be readily extracted in crystalline form. With the exception of slightly lower levels of methionine, Rubisco's content of essential amino acids equals or exceeds that of the FAO Provisional Pattern. Ershoff, B. H., et al. 157:626-630 (1978); Wildman, S. G. 73:243-250 (2002)).","For biofuels to replace a sizable portion of the world's dependence on non-renewable energy sources, co-products, such as Rubisco, are required to help defray the cost of producing this renewable energy. Greene et al., ; National Resources Defense Counsel (2004). Thus, the greater reduction in nicotinic alkaloids in tobacco, the greater the likelihood of a successful tobacco biomass system.","Specific examples are presented below of methods for identifying sequences encoding enzymes involved in nicotine, as well as for introducing the target gene to produce plant transformants. They are meant to be exemplary and not as limitations on the present invention.","The plasmid pHANNIBAL, see Wesley et al., 27: 581-590 (2001), was modified to produce plasmid pHANNIBAL-X as shown in . A SacI restriction site between the ampicillin resistance gene (Amp) and 35S promoter was eliminated by SacI cutting and subsequent DNA blunting and ligation. The multi-cloning sites (MCS) were modified as follows. A Barn H I restriction site was added to the MCS between the promoter and Pdk intron by inserting an adaptor (5\u2032 TCGAACGGGATCCCGCCGCTCGAGCGG) between the XhoI and EcoRI sites. A Bam H1 site was eliminated from and a Sac I site was inserted into the MCS between the intron and terminator by inserting an adaptor (5\u2032 GATCAGCTCTAGAGCCGAGCTCGC) between the BamHI and XbaI sites.","A plant RNAi binary vector was prepared using pHANNIBAL-X using the scheme diagramed in , in which distinct \u201csense\u201d and \u201cantisense\u201d fragments are first obtained by the addition of specific restriction sites to the ends of a segment of the gene of interest, and then the sense and antisense fragments are inserted in the desired orientations in the modified pHANNIBAL-X plasmid.","The DNA segment containing the sense and antisense fragments and the intervening Pdk intron was substituted for the GUS coding region of pBI121 (Wesley et al., 2001,) to produce an RNAi binary vector.","The 814 bp-1160 bp region of the A622 cDNA was used as the dsRNA forming region (sense chain, antisense chain). PCR was performed using A622 cDNA cloned in pcDNAII as the template and primers with additional bases encoding the indicated restriction enzyme sites, and the target DNA fragment was collected and TA cloned to a pGEM-T vector.","The sequences of the primers used were:",{"@attributes":{"id":"p-0160","num":"0159"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Sense\u2003chain\u2003A622\u2003F814-XhoI-A622\u2003R1160-KpnI"},{"entry":"A622\u2003F814-XhoI"},{"entry":"5\u2032 CCGCTCGAGCGGTCAGAGGAAGATATTCTCCA\u20033\u2032"},{"entry":{}},{"entry":"A622\u2003R1160-KpnI"},{"entry":"5\u2032 GGGGTACCCCTGGAATAAGACGAAAAATAG\u20033\u2032"},{"entry":{}},{"entry":"Antisense\u2003chain\u2003A622\u2003F814-XbaI-A622\u2003R1160-ClaI"},{"entry":"A622\u2003F814-XbaI"},{"entry":"5\u2032 GCTCTAGAGCTCAGAGGAAGATATTCTCCA\u20033\u2032"},{"entry":{}},{"entry":"A622\u2003R1160-ClaI"},{"entry":"5\u2032 CCATCGATGGTGGAATAAGACGAAAAATAG\u20033\u2032"}]}}}}},"Recombination with the modified pHANNIBAL-X was performed starting with the sense chain followed by the antisense chain. The TA cloned DNA fragments were cut with the appropriate restriction enzymes, collected, and ligated to pHANNIBAL-X which was cut with the same restriction enzymes. The resulting plasmid contains a DNA sequence with inverted repeats of the A622 fragment separated by the Pdk intron.","The RNAi region was excised from the pHANNIBAL-X with the incorporated sense and antisense chains by treating with BamH I and Sac I and ligated to pBI121 from which the GUS coding region had been removed by similar treatment to produce the binary vector pRNAi-A622 for plant transformation, which contains a T-DNA segment () that contains an nptII selectable marker cassette and the A622 RNAi cassette.","While tobacco BY-2 cell cultures do not normally synthesize nicotinic alkaloids, methyl jasmonate treatment induces expression of genes for known enzymes in the nicotine biosynthesis pathway and elicits formation of nicotinic alkaloids.","In order to deduce the function of A622, RNAi strain cultured cells were prepared in which mRNA from pRNAi-A622 was expressed in cultured tobacco BY-2 cells to suppress A622.","Agrotransformation","The vector (pRNAi-A622) was transformed into strain EH105, which was used to transform tobacco BY-2 cells. The methods for infecting and selecting the tobacco BY-2 cells were as follows.","Four ml of BY-2 cells which had been cultured for 7 days in 100 ml of modified LS medium, see Imanishi et al., 38: 1101-1111 (1998), were subcultured into 100 ml modified LS medium, and cultured for 4 days.","One hundred microliters of solution, which had been cultured for 1 day in YEB medium, were added to the 4 ml of cells that had been cultured for 4 days, and the two were cultured together for 40 hours in the dark at 27\u00b0 C.","After culture, the cells were washed twice with modified LS medium to remove the ","The washed cells were spread on modified LS selection medium containing kanamycin (50 mg\/l) and carbenicillin (250 mg\/l), and transformed cells were selected.","After having been cultured for about 2 weeks in the dark at 27\u00b0 C., the transformed cells were transferred to a fresh modified LS selection medium, and cultured in the dark for 1 week at 27\u00b0 C.","The transformed cells were then grown in a suspension culture in the dark at 27\u00b0 C. for 1 week in 30 ml of liquid modified LS medium.","1 ml of the cultured transformed cells was subcultured to 100 ml of modified LS medium. The transformed cells were subcultured every 7 days in the same way as wild-type cells.","Alkaloid Synthesis","10 ml each of transformed BY-2 cells which had been cultured for 7 days and cultured tobacco cells which had been transformed using a green fluorescent protein (GFP) expression vector as the control were washed twice with modified LS medium containing no 2,4-D, and, after addition of modified LS medium containing no 2,4-D to a total of 100 ml, were suspension cultured at 27\u00b0 C. for 12 hours.","After addition of 100 \u03bcl of methyljasmonate (MeJa) which had been diluted to 50 \u03bcM with DMSO, the cells were suspension cultured for 48 hours at 27\u00b0 C.","Jasmonate treated cells were filtered, collected, and freeze dried. Sulfuric acid, 3 ml of 0.1 N, was added to 50 mg of the freeze-dried sample. The mixture was sonicated for 15 minutes, and filtered. A 28% ammonium solution was added to 1 ml of the filtrate, and centrifuged for 10 minutes at 15000 rpm.","One ml of the supernatant was added to an Extrelut-1 column (Merck) and let sit for 5 minutes. This was eluted with 6 ml of chloroform. The eluate was then dried under reduced pressure at 37\u00b0 C. with an evaporator (Taitec Concentrator TC-8).","The dried sample was dissolved in 50 \u03bcl of ethanol solution containing 0.1% dodecane. A gas chromatograph (GC-14B) equipped with a capillary column and an FID detector was used to analyze the samples. A RESTEC Rtx-5Amine column (Restec) was used as the capillary column. The column temperature was maintained at 100\u00b0 C. for 10 min, elevated to 150\u00b0 C. at 25\u00b0 C.\/min, held at 150\u00b0 C. for 1 min, elevated to 170\u00b0 C. at 1\u00b0 C.\/min, held at 170\u00b0 C. for 2 min, elevated to 300\u00b0 C. at 30\u00b0 C.\/min, and then held at 300\u00b0 C. for 10 min. Injection and detector temperature was 300\u00b0 C. One \u03bcl of each sample was injected, and nicotinic alkaloids were quantified by the internal standard method.","As shown in , in the transgenic BY-2 lines in which the A622 gene expression was suppressed by RNAi (A3, A21 A33 and A43 lines), jasmonate elicitation did not result in high accumulation of anatabine (the major alkaloid in elicited cultured cells), anatalline, nicotine, or anabasine, compared with control cell lines.","RNA Expression","To determine whether the reduction of alkaloid accumulation in the A622-RNAi lines is specifically related to reduction of A622 expression, rather than an indirect effect on the levels of expression of genes for known enzymes in the nicotine biosynthesis pathway, the levels of expression of A622 and other genes was measured in methyl jasmonate treated lines, transgenic lines, and control lines.","Total RNAs were isolated from wild-type and transgenic BY-2 cell lines which were treated with 50 \u03bcM MeJA for 48 h. RNA levels of specific genes were determined by RT-PCR. RNA was extracted using RNeasy Plant mini kit (Qiagen) according to the manufacture's instructions. cDNA was synthesized using random hexamers and SuperScript First-Strand Synthesis System for RT-PCR (Invitrogen). RT-PCR was performed with 5 ng of the cDNA as a template using a TaKaRa ExTaq (Takara Bio) under the following conditions: for detection of A622, NBB1, AO, QS, QPT, ODC, 22 cycles of 94\u00b0 C. for 30 sec, 57\u00b0 C. for 30 sec, and 72\u00b0 C. for 30 sec, for detection of PMT, 24 cycles of 94\u00b0 C. for 1 min, 52\u00b0 C. for 30 sec, and 72\u00b0 C. for 1 min.",{"@attributes":{"id":"p-0181","num":"0180"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"A622\u2003primers:"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"140pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"A622-07F","5\u2032 ATGGTTGTATCAGAGAAAAG"]},{"entry":[{},"A622-05R","5\u2032 CCTTCTGCCTCTATCATCCTCCTG"]},{"entry":[{},{}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"NBBI\u2003primers:"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"140pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"NBBI-01F","5\u2032ATGTTTCCGCTCATAATTCTG"]},{"entry":[{},"NBBI-1365","5\u2032TCTTCGCCCATGGCTTTTCGGTCT"]},{"entry":[{},{}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"AO\u2003primers:"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"140pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"AO\u2003RT-1","5\u2032 CAAAACCAGATCGCTTGGTC"]},{"entry":[{},"AO\u2003RT-2","5\u2032 CACAGCACTTACACCACCTT"]},{"entry":[{},{}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"QS\u2003primers:"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"140pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"QS\u2003RT-1","5\u2032 CGGTGGAGCAAAAGTAAGTG"]},{"entry":[{},"QS\u2003RT-2","5\u2032 GAAACGGAACAATCAAAGCA"]},{"entry":[{},{}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"QPT\u2003primers:"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"140pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"QPTRT-1","5\u2032 TCACTGCTACAGTGCATCCT"]},{"entry":[{},"QPTRT-2","5\u2032 TTAGAGCTTTGCCGACACCT"]},{"entry":[{},{}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"ODC\u2003primers:"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"140pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"ODC\u2003RT-1","5\u2032 CGTCTCATTCCACATCGGTAGC"]},{"entry":[{},"ODC\u2003RT-2","5\u2032 GGTGAGTAACAATGGCGGAAGT"]},{"entry":[{},{}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"PMT\u2003primers:"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"140pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"PMT\u2003RT-1","5\u2032 GCCATGATAATGGCAACGAG"]},{"entry":[{},"PMT\u2003RT-2","5\u2032 TTAGCAGCGAGATAAGGGAA"]}]}}]}}},"As shown in , A622 is not induced in A622-silenced lines. Other genes for known nicotine biosynthetic pathway enzymes are induced. These results provide evidence that A622 is included in the nicotinic alkaloid biosynthesis pathway, and demonstrate that the nicotinic alkaloid content and particularly the nicotine content of plant cells having nicotine-producing ability can be reduced by down-regulating A622 expression.","Constitutive suppression of A622 expression in tobacco hairy roots significantly inhibited root growth, precluding analysis of nicotinic alkaloids. To circumvent this, an estradiol-inducible gene expression system (XVE system) was developed. The XVE system produces RNAi hairpin molecules and target genes are suppressed only after addition of an inducer (beta-estradiol) into the culture medium.","The RNAi region containing A622 sense and antisense DNA fragments was excised from the pHANNIBAL-X plasmid with Xho I and Xba I, and ligated into pBluescript KS which had been digested with Xho I and Xba I. The RNAi region was then excised with Xho I and Spe I, and was subcloned between the XhoI and SpeI sites in the MCS of the XVE vector pER8 (Zuo J. et al, 24: 265-273 (2000)) to produce the binary vector pXVE-A622RNAi.","The T-DNA region of pXVE-A622RNAi (See ) contains a cassette for estradiol-inducible expression of the chimeric transcription factor XVE, an hpt selectable marker cassette, and a cassette in which expression of the A622 RNAi is under the control of the LexA-46 promoter, which is activated by XVE.","The binary vector pXVE-A622RNAi was introduced to strain 15834 by electroporation. cv. Petit Havana SR1 plants were transformed by using a leaf-disc method, as described by Kanegae et al., 105(2):483-90. (1994). Hygromycin resistance (15 mg\/L in B5 medium) was used to select transformed roots. Transgenic hairy roots were grown at 27\u00b0 C. in the dark.","Transgenic hairy roots carrying the T-DNA from pXVE-A622RNAi were grown in the B5 medium for 10 days and then gene silencing was induced by addition of 17-beta-estradiol (2 \u03bcM) for 4 days. RT-PCR analysis showed that A622 expression was efficiently suppressed in tobacco hairy root lines A8 and A10 transformed with the estradiol-inducible A622 suppression construct after the estradiol addition. See .","Total RNA was extracted from hairy roots by using RNeasy Plant mini kit (Qiagen). cDNA was synthesized by using random hexamers and SuperScript First-Strand Synthesis System for RT-PCR (Invitrogen). RT-PCR was carried with 2.5 ng of the cDNA as a template using a TaKaRa ExTaq (Takara Bio) under the following conditions: for detection of A622, 22 cycles of 94\u00b0 C. for 30 sec, 57\u00b0 C. for 30 sec, and 72\u00b0 C. for 30 sec; for detection of \u03b1-tubulin, 24 cycles of 94\u00b0 C. for 1 min, 52\u00b0 C. for 30 sec, and 72\u00b0 C. for 1 min.",{"@attributes":{"id":"p-0189","num":"0188"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"Primers\u2003for\u2003A622\u2003detection;\u2003(SEQ\u2003ID\u2003NOS\u200311-12)"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["A622-07F","5\u2032 ATGGTTGTATCAGAGAAAAG"]},{"entry":["A622-05R","5\u2032 CCTTCTGCCTCTATCATCCTCCTG"]},{"entry":{}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Primers\u2003for\u2003\u03b1-tubulin\u2003detection;"},{"entry":"(SEQ\u2003ID\u2003NOS\u200325-26)"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Tub\u2003RT-1","5\u2032 AGTTGGAGGAGGTGATGATG"]},{"entry":["Tub\u2003RT-2","5\u2032 TATGTGGGTCGCTCAATGTC"]}]}}]}}},"Nicotine contents were measured in hairy root lines transformed with an inducible A622 suppression construct without (\u2212) and after induction of suppression with estradiol (+). The RT-PCR graph in  shows that A622 expression was already partially suppressed before estradiol induction. This is especially true with the #8 line. Nicotine contents varied among the A622 suppressed lines but were lower in the A622 suppressed lines than in the wild-type hairy roots.","A cDNA micro-array prepared from a -derived cDNA library (Katoh et al., ., Vol. 79, Ser. B, No. 6, pp. 151-154 (2003)) was used to search for novel genes which are controlled by the nicotine biosynthesis regulatory NIC loci.",{"@attributes":{"id":"p-0192","num":"0191"},"i":["N. sylvestris ","N. tabacum "],"sup":"2"},"Roots of eight-week-old plantlets were harvested, immediately frozen with liquid nitrogen, and kept at \u221280\u00b0 C. until use. Total RNA was isolated using Plant RNeasy Mini kit (Qiagen) from the frozen roots, and mRNA was purified using GenElute mRNA Miniprep kit (Sigma). cDNA was synthesized from 0.4 \u03bcg of the purified mRNA by using LabelStar Array Kit (Qiagen) in the presence of Cy3 or Cy5-dCTP (Amersham). cDNA hybridization to the microarray slides and post-hybridization washes were performed using a Lucida Pro hybrid-machine (Amersham). Microarrays were scanned using an FLA-8000 scanner (Fujifilm). Acquired array images were quantified for signal intensity with ArrayGauge software (Fujifilm). cDNA probes from wild type and niclnic2 tobacco were labeled with Cy3 and Cy5 in reciprocal pair-wise combinations. Hybridization signals were normalized by accounting for the total signal intensity of dyes. cDNA clones which hybridized to wild-type probes more than twice as strongly compared to nic1nic2 probes were identified, and these included NBB1.","Full-length NBB1 cDNA was obtained by 5\u2032- and 3\u2032-RACE from total RNA of by using a SMART RACE cDNA Amplification Kit (Clontech).","The nucleotide sequence of the NBB1 cDNA insert was determined on both strands using an ABI PRISM\u00ae 3100 Genetic Analyzer (Applied Biosystems) and a BigDye\u00ae Terminator v3.1 Cycle Sequencing Kit (Applied Biosystems). The nucleotide sequence of NBB1 is set forth in SEQ ID NO: 3. The amino acid sequence encoded by the nucleotide sequence is set forth in SEQ ID NO: 4. The protein sequence includes a FAD-binding motif. A putative vacuolar signal peptide is located at N-terminus.","NBB1 expression was investigated in tobacco plants by Northern blot analysis.","RNA was extracted from plant bodies which had been treated with methyljasmonate vapor, using cv. Burley 21 (abbreviated below as WT) and mutants nic1, nic2 and nic1 nic2 having mutations introduced in the Burley 21 background. Cultivation was in a sterile sealed environment, and the plants were raised for 2 months at 25\u00b0 C. with 150 \u03bcmole photons\/mof light (16 h light, 8 h dark) on \u00bd\u00d7B5 medium (3% sucrose, 0.3% gellan gum). Methyl jasmonate treatment was accomplished by adding 0.5 mL of 100 \u03bcM methyl jasmonate to an Agripot container (Kirin, Tokyo) with a solid medium capacity of 80 cmand a gas capacity of 250 cmcontaining the plants. The treatment times were set at 0 h and 24 h. The root parts and leaf parts (2through 6leaves from a plant body with a total of 7 to 10 leaves) were collected from the plant bodies and immediately stored frozen using liquid nitrogen.","RNA was extracted using an RNeasy Midi Kit (Qiagen) according to the manufacturer's protocol. However, polyvinyl pyrrolidine was added to a concentration of 1% to the RLT. The column operation was performed twice to increase the purity of the RNA.","RNA blotting was carried out according to the ordinary methods given by Sambrook and Russell (Sambrook, J. et al., , Cold Spring Harbor Laboratory, Chapter 7 (2001)).","The sequence fragment from 1278 bp through the end (1759 bp) of the NBB1 nucleotide sequence (SEQ ID NO: 3) was used as the probe template. The template was prepared by amplification from the cDNA clone using PCR using the following primers:",{"@attributes":{"id":"p-0201","num":"0200"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"182pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"primer\u20031:\u2003GGAAAACTAACAACGGAATCTCT"]},{"entry":[{},{}]},{"entry":[{},"primer\u20032:\u2003GATCAAGCTATTGCTTTCCCT"]}]}}}}},"The probe was labeled with P using a Bcabest labeling kit (Takara) according to the manufacturer's instructions. Hybridization was accomplished using ULTRAhyb (Ambion) as the buffer according to the manufacturer's protocol.","PMT probe was prepared from a PMT sequence cloned into a pcDNAII vector in (Hibi et al., 1994). The plasmid was extracted and purified from the using a QIAprep Spin Miniprep Kit (Qiagen), treated with the restriction enzymes XbaI and HindIII by ordinary methods, and run through agarose gel electrophoresis, and about 1.5 kb DNA fragments were collected. A QIAquick Gel Extraction Kit (Qiagen) was used for collection. The collected DNA fragments were P labeled by the same methods used for the NBB1 probe, and hybridized. The results are shown in .","As  clearly shows, NBB1 and PMT have the same pattern of expression in tobacco plants. Evidence that NBB1 is involved in nicotine biosynthesis is that, like PMT and A622, NBB1 is under the control of the NIC genes, and it exhibits a similar pattern of expression to PMT and A622.","NBB1 polypeptide has 25% identity and 60% homology to the berberine bridge enzyme (BBE). (Dittrich H. et al., , Vol. 88, 9969-9973 (1991)). An alignment of the NBB1 polypeptide with EcBBE is shown in .","A phylogenetic tree was constructed using the sequences of NBB1 polypeptide and plant BBE-like polypeptides (based on Carter and Thornburg, 134, 460-469 (2004). The phylogenetic analysis was performed using neighbor-joining method with the CLUSTAL W program. Numbers indicate bootstrap values from 1,000 replicates. The sequences used were: EcBBE, California poppy BBE (GenBank accession no. AF005655); PsBBE, opium poppy () probable reticuline oxidase (AF025430); BsBBE, barberry () BBE (AF049347); VuCPRD2, cowpea () drought-induced protein (AB056448); NspNEC5, sp. Nectarin V (AF503441\/AF503442); HaCHOX, sunflower () carbohydrate oxidase (AF472609); LsCHOX, lettuce () carbohydrate oxidase (AF472608); and 27 genes (At1g01980, At1g11770, At1g26380, At1g26390, At1g26400, At1g26410, At1g26420, At1g30700, At1g30710, At1g30720, At1g30730, At1g30740, At1g30760, At1g34575, At2g34790, At2g34810, At4g20800, At4g20820, At4g20830, At4g20840, At4g20860, At5g44360, At5g44380, At5g44390, At5g44400, At5g44410, and At5g44440).","The results are shown in . The three known BBEs form a separate Glade and are underlined and indicated as \u201cTrue BBEs.\u201d The sequence of NBB1 is not highly similar to any of the BBE or BBE-like proteins, and is separated from the other sequences at the base of the tree. The only other BBE like protein described from the genus , nectarin V, a protein described in nectar of the a hybrid ornamental , Carter and Thornburg (2004), clusters with the cowpea drought-induced protein and several putative BBE-like proteins from . Because the nectar of the ornamental tobacco lacks alkaloids and nectarin V has glucose oxidase activity, it was concluded that nectarin V is involved in antimicrobial defense in flowers and is not likely to have any role in alkaloid synthesis. Id.","The 342-bp DNA fragment of the NBB1 cDNA was amplified by PCR and cloned into pGEM-T vector using the following primers.",{"@attributes":{"id":"p-0209","num":"0208"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"189pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Antisense\u2003chain\u2003(SEQ\u2003ID\u2003NOS\u200329-30)"]},{"entry":[{},"NBB1-20F-EcoRI"]},{"entry":[{},"5\u2032 CCGGAATTCGCACAGTGGAATGAAGAGGACG\u20033\u2032"]},{"entry":[{},{}]},{"entry":[{},"NBB1-18R-XhoI"]},{"entry":[{},"5\u2032 CCGCTCGAGGCGTTGAACCAAGCATAGGAGG\u20033\u2032"]},{"entry":[{},{}]},{"entry":[{},"Sense\u2003chain\u2003(SEQ\u2003ID\u2003NOS\u200331-32)"]},{"entry":[{},"NBB1-16F-ClaI"]},{"entry":[{},"5\u2032 CCATCGATGCACAGTGGAATGAAGAGGACG\u20033\u2032"]},{"entry":[{},{}]},{"entry":[{},"NBB1-19R-XbaI"]},{"entry":[{},"5\u2032 GCTCTAGAGCGTTGAACCAAGCATAGGAGG\u20033\u2032"]}]}}}}},"Resultant PCR products were digested with EcoRI and XhoI for the antisense insertion, and with ClaI and XbaI for sense chain insertion. The sense DNA fragment was subcloned into the pHANNIBAL-X, followed by insertion of the antisense fragment. The resulting plasmid contained a inverted repeat of the NBB1 fragment, separated by the Pdk intron.","The RNAi region was excised from the pHANNIBAL-X with BamH I and Sac I, and ligated into pBI121 to replace the GUS coding region and to produce the binary vector pRNAi-NBB1. The T-DNA region of pHANNIBAL-NBB1 3\u2032 (See ) contains an nptII selectable marker cassette and cassette for expression of a hairpin RNAi with a double-stranded region corresponding to a 3\u2032 fragment of NBB1.","Methyl jasmonate treatment of Tobacco BY-2 cells induces NBB1 expression in addition to expression of genes for known enzymes in the nicotine biosynthesis pathway. The effects of NBB1 suppression was tested in BY-2 cells.","The vector pRNAi-NBB1 was introduced into strain EHA105, which was used to transform tobacco BY-2 cells. BY-2 cells were cultured in 100 ml of modified LS medium. cells (100 \u03bcl) in YEB medium were added to 4 ml of BY-2 cells and cultured for 40 hours in the dark at 27\u00b0 C. After infected tobacco cells were washed twice with modified LS medium, washed tobacco cells were spread on modified LS agar medium containing kanamycin (50 mg\/l) and carbenicillin (250 mg\/l). After 2 weeks in the dark at 27\u00b0 C., growing tobacco calluses were transferred to fresh LS selection medium with the same antibiotics, and cultured in the dark at 27\u00b0 C. for one more week. Growing tobacco cells were transferred to liquid modified LS medium without antibiotics. The transformed tobacco cells were subcultured at a 7-day intervals.","Cultured tobacco cells were cultured with modified LS medium without 2,4-D, at 27\u00b0 C. for 12 hours. After 100 \u03bcl of methyljasmonate (MeJA) dissolved in DMSO was added to 100 mL of the tobacco suspension culture to give a final concentration of 50 \u03bcM, tobacco cells were cultured for an additional 48 hours. MeJA-treated cells were filtered, collected, and freeze dried. Sulfuric acid, 3 ml of 0.1 N, was added to 100 mg of the freeze-dried sample. The mixture was sonicated for 15 minutes, and filtered. A 28% ammonium solution was added to 1 ml of the filtrate, and centrifuged for 10 minutes at 15000 rpm. One ml of the supernatant was added to an Extrelut-1 column (Merck) and eluted with 6 ml of chloroform. The eluate was then dried under reduced pressure at 37\u00b0 C. with an evaporator (Taitec Concentrator TC-8). The dried sample was dissolved in 50 \u03bcl of ethanol solution containing 0.1% dodecane. A gas chromatograph (GC-14B) equipped with a capillary column and an FID detector was used to analyze the samples. A RESTEC Rtx-5Amine column (Restec) was used as the capillary column. The column temperature was maintained at 100\u00b0 C. for 10 min, elevated to 150\u00b0 C. at 25\u00b0 C.\/min, held at 150\u00b0 C. for 1 min, elevated to 170\u00b0 C. at 1\u00b0 C.\/min, held at 170\u00b0 C. for 2 min, elevated to 300\u00b0 C. at 30\u00b0 C.\/min, and then held at 300\u00b0 C. for 10 min. Injection and detector temperature was 300\u00b0 C. One \u03bcl of each sample was injected, and nicotinic alkaloids were quantified by the internal standard method.","Accumulation of nicotinic alkaloids following methyl jasmonate elicitation was greatly reduced in NBB1-suppressed BY-2 cell lines (N37 and N40) compared with wild-type tobacco cells (See ).","To determine whether the reduction of alkaloid accumulation in the NBB1-RNAi lines is specifically related to reduction of NBB1 expression, rather than an indirect effect on the levels of expression of genes for known enzymes in the nicotine biosynthesis pathway, the levels of expression of NBB1 and other genes was measured in methyl jasmonate treated lines transgenic and control lines.","Total RNAs were isolated from wild-type and transgenic BY-2 cell lines which had been treated with 50 uM MeJA for 48 h. RNA levels of specific genes were determined by RT-PCR. The results are shown in .","In NBB1-silenced lines induction of NBB1 is not observed, but induction of known genes of the nicotine biosynthetic pathway still occurs, as well as induction of A622. Note also that induction of NBB1 is not affected in A662-suppressed lines.","These results demonstrate that NBB1 reductase is included in the nicotinic alkaloid biosynthesis pathway, and that the nicotinic alkaloid content, and particularly, the nicotine content of plant cells having nicotine-producing ability can be decreased by down-regulating NBB1 expression.","Tobacco SR-1 hairy roots accumulate nicotine as the major alkaloid. The effect of NBB1 suppression on alkaloid accumulation in hairy roots was studied.","The binary vector pRNAi-NBB1 3\u2032 was introduced into strain 15834 by electroporation. cv. Petit Havana SR1 plants were transformed by using a leaf-disc method, as described above for suppression of A622 in tobacco hairy roots. Hairy roots were selected and cultured and alkaloids were extracted, purified, and analyzed as described above.","When NBB1 expression was suppressed by RNAi, transgenic root lines (HN6, HN19, HN20 and HN29) contained highly reduced levels of nicotine compared with the control cell line, as well as reduced levels of anatabine (See ).","Transgenic hairy roots carrying pHANNIBAL-NBB1 3\u2032 were grown in B5 medium for two weeks, and gene expression was analyzed by RT-PCR. NBB1 expression was specifically suppressed in four transgenic lines (See ). NBB1 Expression of other genes for enzymes in the nicotine biosynthetic pathway was not affected.","The results demonstrate that reduced-nicotine accumulation results from reduced NBB1 expression, not a lack of expression of genes for known enzymes of the nicotine biosynthesis pathway.","Two attB-NBB1 fragments were amplified by PCR from the NBB1 cDNA in pGEM-T vector using a primer set of NBB1-aatB1 and attB1 adapter, and a set of NBB1-attB2 and attB2 adapter.",{"@attributes":{"id":"p-0226","num":"0225"},"tables":{"@attributes":{"id":"TABLE-US-00006","num":"00006"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Gene-specific\u2003primers:\u2003(SEQ\u2003ID\u2003NOS\u200333-34)"},{"entry":"NBB1-attB1"},{"entry":"5\u2032 AAAAAGCAGGCTTCGAAGGAGATAGAACCATGGTTCCGCTCATA"},{"entry":"ATTCTGATCAGCTT"},{"entry":{}},{"entry":"NBB1-attB2"},{"entry":"5\u2032 AGAAAGCTGGGTCTTCACTGCTATACTTGTGCTCTTGA"},{"entry":{}},{"entry":"Adapter\u2003primers:\u2003(SEQ\u2003ID\u2003NOS\u200335-36)"},{"entry":"attB1\u2003adapter"},{"entry":"5\u2032 GGGGACAAGTTTGTACAAAAAAGCAGGCT"},{"entry":{}},{"entry":"attB2\u2003adapter"},{"entry":"5\u2032 GGGGACCACTTTGTACAAGAAAGCTGGGT"}]}}}}},"The PCR conditions used were those recommended by the manufacture. An entry clone pDONR221-NBB1-1 was created by BP recombination reactions between the attB-NBB1 PCR products and pDONR221 (Invitrogen).","The NBB1 ORF was transferred from the pDONR221-NBB1-1 vector to a GATEWAY binary vector pANDA 35HK which was designed to express a dsRNA with GUS partial fragment under the CaMV35S promoter (Dr. Ko Shimamoto, NAIST) by LR reaction. The resultant NBB1 RNAi vector is referred to as pANDA-NBB1full.","The T-DNA of pANDA-NBB1full (See ) contains an nptII selectable marker cassette, an NBB1 RNAi cassette in which the full length coding region of NBB1 is present in inverted repeats separated by a GUS linker, and an hpt selectable marker cassette.","The binary vector pANDA-NBB1full was introduced to strain EHA105 by electroporation. cv. Petit Havana SR1 plants were transformed by using a leaf-disk method, basically as described by Kanegae et al., 105, 483-490 (1994). Hygromycin resistance (30 mg\/L in MS medium) was used for selection. Transgenic plants were regenerated from the leaf discs as described and grown at 27\u00b0 C. under continuous light in a growth chamber.","Leaf tissue was collected from TO generation plants grown for 36 days. Alkaloids were extracted, purified, and analyzed as described above. Levels of nicotine in the leaves of plants from lines transformed with the NBB1 suppression vector pANDA-NBB1full were reduced compared to wild type (See ). Leaves of transgenic lines (#6, #14 and #22) contained levels of nicotine only about 16% of the level in leaves of wild type plants."],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0016","num":"0015"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0017","num":"0016"},"figref":"FIG. 2A"},{"@attributes":{"id":"p-0018","num":"0017"},"figref":"FIG. 2B"},{"@attributes":{"id":"p-0019","num":"0018"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0020","num":"0019"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0021","num":"0020"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0022","num":"0021"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0023","num":"0022"},"figref":"FIG. 7"},{"@attributes":{"id":"p-0024","num":"0023"},"figref":"FIG. 8A"},{"@attributes":{"id":"p-0025","num":"0024"},"figref":"FIG. 8B"},{"@attributes":{"id":"p-0026","num":"0025"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0027","num":"0026"},"figref":"FIG. 10","i":"Eschscholzia californica "},{"@attributes":{"id":"p-0028","num":"0027"},"figref":"FIG. 11"},{"@attributes":{"id":"p-0029","num":"0028"},"figref":"FIG. 12"},{"@attributes":{"id":"p-0030","num":"0029"},"figref":"FIG. 13","i":"A. rhizogenes"},{"@attributes":{"id":"p-0031","num":"0030"},"figref":"FIG. 14"},{"@attributes":{"id":"p-0032","num":"0031"},"figref":"FIG. 15"},{"@attributes":{"id":"p-0033","num":"0032"},"figref":"FIG. 16","i":"Nicotiana tabacum "}]},"DETDESC":[{},{}]}
