---
title: Biosynthetic pathway for heterologous expression of a nonribosomal peptide synthetase drug and analogs
abstract: The present invention is directed to the biosynthetic pathway for a nonribosomal peptide synthetase (NRPS) derived drug and analogs thereof. The invention also discloses polynucleotide sequences useful for heterologous expression in a convenient microbial host for the synthesis of the NRPS derived drug.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08815562&OS=08815562&RS=08815562
owner: Colorado State University Research Foundation
number: 08815562
owner_city: Fort Collins
owner_country: US
publication_date: 20110415
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATIONS","STATEMENT OF GOVERNMENT INTEREST","SEQUENCE LISTING","FIELD OF THE INVENTION","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","Enzymes in Et743 Synthesis","Synthesis of ET743","EXAMPLES","Example 1","Collection of ","Example 2","High-Throughput Sequencing of ","Example 3","Secondary Metabolite Identification","Example 4","Metagenomic Sequencing and Phylogenetics"],"p":["This application is a U.S. National Phase of International Application No. PCT\/US2011\/032796 filed Apr. 15, 2011, incorporated herein by reference, which claims priority of U.S. Provisional Patent Application Ser. No. 61\/324,639, filed Apr. 15, 2010, the disclosure of which is incorporated herein in its entirety.","This invention was made with government support under Grant Numbers 2R01DC04173, 3R01DC02148, 1 R01 AI080935-01, and 2RO1CA085419, awarded by the National Institutes of Health (NIH). The government has certain rights in the invention.","This application contains, as a separate part of the disclosure, a Sequence Listing in computer-readable form (filename: 45345A_SeqListing.txt; created: Apr. 15, 2011; 200,632 bytes\u2014ASCII text file) which is incorporated by reference in its entirety.","The present invention is directed to the biosynthetic pathway for a nonribosomal peptide synthetase (NRPS) drug and analogs thereof. The invention also discloses polynucleotide sequences useful for heterologous expression in a microbial host for the synthesis of the NRPS drug.","Nonribosomal peptide synthetases (NRPSs) are large multidomain enzymes responsible for the biosynthesis of many pharmacologically important bioactive compounds of great structural diversity [Marahiel et al., . (Washington, D.C.) 97: 2651-2673 (1997); Schwarzer et al., 20: 275-287 (2003); Cane et al., 282, 63-68 (1998)]. Prominent examples are the antibiotics penicillin, vancomycin, and actinomycin D, the immunosuppressant cyclosporine A, the siderophore enterobactin, and the antitumor drug bleomycin. NRPSs are organized into distinct modules, each of them responsible for the incorporation of one amino acid into the nascent peptide chain. A module can be further subdivided into catalytic domains, which are responsible for the coordinated recognition and activation [adenylation (A) domain] [Stachelhaus et al., 6: 493-505 (1999)], covalent binding and transfer [peptidyl carrier protein (PCP) domain] [Stachelhaus et al., 3: 913-921 (1996)], and incorporation [condensation (C) domain] of a certain substrate amino acid into the peptide chain [Stachelhaus et al., 273: 22773-22781 (1998)]. In addition to these so-called core domains, optional domains catalyze the modification of incorporated residues, i.e., by epimerization (E) or N-methylation (MT) domains [Walsh et al., 5: 525-534 (2001)]. Product release is normally effected by a thioesterase (Te) domain, catalyzing the formation of linear, cyclic, or branched cyclic products, representative for the class of NRPSs [Trauger et al., 407: 215-218 (2000)].","Because of the modular organization of NRPSs and the colinearity between biosynthetic template and product, the NRP assembly line mechanism accommodates an enormous potential for biocombinatorial approaches. Crucial for such approaches is a profound knowledge about the substrate selectivity of catalytic domains and the determinants of selective communication between modules. In this context, little is known about the intermolecular communication between NRPSs within the same biosynthetic complex [Hahn et al., () 101(44): 15585-15590 (2004)].","ET743 (Trabectedin) is a tetrahydroisoquinoline natural product with potent activity as a chemotherapeutic originally isolated from the tunicate [Zewail-Foote et al., 42: 2493-2497 (1999)]. This drug has been commercialized by PharmaMar in Europe as Yondelis\u00ae for patients with soft tissue sarcoma and a new drug application has been filed with the Food and Drug Administration after completion of a Phase III clinical trial with coadministration of Doxil. Improved clinical outcome was observed as compared to Doxil alone [Chuk et al., 14: 794-799 (2009)]. ET743 has been found to have a unique mechanisms of action including low nM cytotoxicity [Izbicka et al., 9: 981 (1998)], sequence specific alteration of DNA transcription [D'Incalci, 7: 210 (2002); Minuzzo, 97: 6780 (2000); Seaman et al., 120: 13028-13041 (1998)], and induced DNA breakage [Takebayashi, 96: 7196 (1999)] attributed to the ability of ET743 to alkylate the minor groove of DNA [Pommier et al., 35: 13303-13309 (1996); Zewail-Foote et al., 42: 2493-2497 (1999)].","Obtaining sufficient amounts of ET743 has presented a challenge since it was first isolated in 0.0001% yield from the natural source [Rinehart et al., The Journal of Organic Chemistry 55: 4512-4515 (1990)]. Aquaculture has proven to be viable [Fusetani N (ed): Drugs from the Sea. Basel. Karger. 2000. pp120-133. Chapter: Dominick Mendola Aquacultural Production of Bryostatin 1 and ecteinascidin 743; Fusetani, Drugs from the Sea. (2000); Carballo, 31: 481 (2000)], although not an economical method for supplying ET743 for clinical trials and commercial use [Cuevas, 26: 322 (2009)]. Total synthesis of ET743 was first reported [Corey et al., 118: 9202-9203 (1996)] and further routes of synthesis have been published [Endo et al., 124: 6552-6554 (2002), Chen et al., 128: 87-89 (2005), Zheng et al., 45: 1754 (2006), and Fishlock et al., 73: 9594-9600 (2008)]. Commercial production by PharmaMar has used a semi-synthetic scheme in which cyanosafracin B is transformed into ET743 over eight steps [Cuevas et al., 2: 2545-2548 (2000)] as safracin B can be cultured on the kilogram scale from the wild-type producer [Ikeda, . Series B 36: 1290 (1983)].","The similarity of ET743 to three other bacterial derived natural products, safracin () [Ikeda, . Series B 36: 1290 (1983)], saframycin () [Arai et al., 36: 1025 (1980)], and saframycin Mx1 () [Irschik, . Series B 41: 993 (1988)] has been put forth as evidence that ET743 is of prokaryotic origin, likely from a bacterium that is closely associated to [Piel, 13: 39 (2006)] (). Evidence for such a \u201csymbiont hypothesis\u201d has been generated for a diverse set of natural products isolated from macroscopic sources including bryostatin [Lopanik et al., 139: 131 (2004); Sudek et al., 70: 67-74 (2007); Lopanik et al., & 15: 1175 (2008)], rhizoxin [Partida-Martinez, 437: 884 (2005); Scherlach et al., 128: 11529-11536 (2006); Partida-Martinez et al., 57: 2583-2590 (2007)], and onnamide\/pederin [Piel, 99: 14002 (2002); Piel, 101: 16222 (2004); Schmidt, 4: 466 (2008)].","A common biosynthetic logic has been elucidated for the three known tetrahydroisoquinoline pathways, and was used to identify an ET743 biosynthetic scheme (). All three described biosynthetic pathways [Velasco et al., 56: 144 (2005); Li et al., 190: 251-263 (2008); Pospiech et al., 141: 1793 (1995)] consist of three nonribosomal peptide synthetase (NRPS) modules and numerous tailoring enzymes [Fischbach et al., 106: 3468-3496. (2006)]. Each module contains one of the three core domains, an adenylation (A) domain, a condensation (C) domain, and a thiolation (T) domain, that act in concert to polymerize amino acid building blocks through amide linkages. Two of the three pathways are initiated in an uncharacterized manner by an acyl ligase (AL) domain and T-domain. All three NRPS trimodules are terminated by a rare RE-domain, which utilizes NAD(P)H to release the enzyme bound intermediate in reduced form as an activated aldehyde [Kopp et al., 128: 16478 (2006)]. Disclosed herein are polynucleotide and polypeptide sequences derived from the microbiome of that are involved in a biosynthetic pathway to synthesize ET743.","Accordingly, the present disclosure provides a protein comprising the polypeptide sequence of SEQ ID NO: 1 (EtuA1), the polypeptide sequence of SEQ ID NO: 2 (EtuA2), the polypeptide sequence of SEQ ID NO: 3 (EtuA3), the polypeptide sequence of SEQ ID NO: 4 (EtuD1), the polypeptide sequence of SEQ ID NO: 5 (EtuD2), the polypeptide sequence of SEQ ID NO: 6 (EtuD3), the polypeptide sequence of SEQ ID NO: 7 (EtuF1), the polypeptide sequence of SEQ ID NO: 8 (EtuF2), the polypeptide sequence of SEQ ID NO: 9 (EutF3), the polypeptide sequence of SEQ ID NO: 10 (EtuH), the polypeptide sequence of SEQ ID NO: 11 (EtuM1), the polypeptide sequence of SEQ ID NO: 12 (EtuM2), the polypeptide sequence of SEQ ID NO: 13 (EtuN1), the polypeptide sequence of SEQ ID NO: 14 (EtuN2), the polypeptide sequence of SEQ ID NO: 15 (EtuN3), the polypeptide sequence of SEQ ID NO: 16 (EtuO), the polypeptide sequence of SEQ ID NO: 17 (EtuP1), the polypeptide sequence of SEQ ID NO: 18 (EtuP2), the polypeptide sequence of SEQ ID NO: 19 (EtuR1), the polypeptide sequence of SEQ ID NO: 20 (EtuR2): the polypeptide sequence of SEQ ID NO: 21 (EtuR3), the polypeptide sequence of SEQ ID NO: 22 (EtuT), the polypeptide sequence of SEQ ID NO: 23 (EtuU1), the polypeptide sequence of SEQ ID NO: 24 (EtuU2), or the polypeptide sequence of SEQ ID NO: 25 (EtuU3)","Also provided is a polypeptide having a sequence that is 90% identical to the polypeptide sequence of SEQ ID NO: 1 (EtuA1), SEQ ID NO: 2 (EtuA2), SEQ ID NO: 3 (EtuA3), SEQ ID NO: 4 (EtuD1), SEQ ID NO: 5 (EtuD2), SEQ ID NO: 6 (EtuD3), SEQ ID NO: 7 (EtuF1), SEQ ID NO: 8 (EtuF2), SEQ ID NO: 9 (EutF3), SEQ ID NO: 10 (EtuH), SEQ ID NO: 11 (EtuM1), SEQ ID NO: 12 (EtuM2), SEQ ID NO: 13 (EtuN1), SEQ ID NO: 14 (EtuN2), SEQ ID NO: 15 (EtuN3), SEQ ID NO: 16 (EtuO), SEQ ID NO: 17 (EtuP1), SEQ ID NO: 18 (EtuP2), SEQ ID NO: 19 (EutR1), SEQ ID NO: 20 (EtuR2): SEQ ID NO: 21 (EtuR3), SEQ ID NO: 22 (EtuT), SEQ ID NO: 23 (EtuU1), SEQ ID NO: 24 (EtuU2), and SEQ ID NO: 25 (EtuU3).","In other embodiments, a polypeptide that is 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98% or 99% identical to a polypeptide sequence of SEQ ID NO: 1 (EtuA1), SEQ ID NO: 2 (EtuA2), SEQ ID NO: 3 (EtuA3), SEQ ID NO: 4 (EtuD1), SEQ ID NO: 5 (EtuD2), SEQ ID NO: 6 (EtuD3), SEQ ID NO: 7 (EtuF1), SEQ ID NO: 8 (EtuF2), SEQ ID NO: 9 (EutF3), SEQ ID NO: 10 (EtuH), SEQ ID NO: 11 (EtuM1), SEQ ID NO: 12 (EtuM2), SEQ ID NO: 13 (EtuN1), SEQ ID NO: 14 (EtuN2), SEQ ID NO: 15 (EtuN3), SEQ ID NO: 16 (EtuO), SEQ ID NO: 17 (EtuP1), SEQ ID NO: 18 (EtuP2), SEQ ID NO: 19 (EutR1), SEQ ID NO: 20 (EtuR2); SEQ ID NO: 21 (EtuR3), SEQ ID NO: 22 (EtuT), SEQ ID NO: 23 (EtuU1), SEQ ID NO: 24 (EtuU2), and SEQ ID NO: 25 (EtuU3) is provided.","Also provided by the present disclosure is a polynucleotide (SEQ ID NO: 57) encoding the polypeptide sequences disclosed herein.","In another embodiment, the present disclosure provides a polynucleotide comprising the sequence of SEQ ID NO: 26 encoding EtuA1, the polynucleotide sequence of SEQ ID NO: 27 encoding EtuA2, the polynucleotide sequence of SEQ ID NO: 28 encoding EtuA3, the polynucleotide sequence of SEQ ID NO: 29 encoding EtuD1, the polynucleotide sequence of SEQ ID NO: 30 encoding EtuD2, the polynucleotide sequence of SEQ ID NO: 31 encoding EtuD3, the polynucleotide sequence of SEQ ID NO: 32 encoding EtuF1, the polynucleotide sequence of SEQ ID NO: 33 encoding EtuF2, the polynucleotide sequence of SEQ ID NO: 34 encoding EtuF3, the polynucleotide sequence of SEQ ID NO: 35 encoding EtuH, the polynucleotide sequence of SEQ ID NO:36 encoding EtuM1, the polynucleotide sequence of SEQ ID NO: 37 encoding EtuM2, the polynucleotide sequence of SEQ ID NO: 38 encoding EtuN1, the polynucleotide sequence of SEQ ID NO: 39 encoding EtuN2, the polynucleotide sequence of SEQ ID NO: 40 encoding EtuN3, the polynucleotide sequence of SEQ ID NO: 41 encoding EtuO, the polynucleotide sequence of SEQ ID NO: 42 encoding EtuP1, the polynucleotide sequence of SEQ ID NO: 43 encoding EtuP2, the polynucleotide sequence of SEQ ID NO: 44 encoding EtuR1, the polynucleotide sequence of SEQ ID NO: 45 encoding EtuR2, the polynucleotide sequence of SEQ ID NO:46 encoding EtuR3, the polynucleotide sequence of SEQ ID NO: 47 encoding EtuT, the polynucleotide sequence of SEQ ID NO: 48 encoding EtuU1, the polynucleotide sequence of SEQ ID NO: 49 encoding EtuU2, and the polynucleotide sequence of SEQ ID NO: 50 encoding EtuU3.","Also provided is a polynucleotide that is 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98% or 99% identical to the polynucleotide sequence set out in SEQ ID NO: 26, SEQ ID NO: 27, SEQ ID NO: 28, SEQ ID NO: 29, SEQ ID NO: 30, SEQ ID NO: 31, SEQ ID NO: 32, SEQ ID NO: 33, SEQ ID NO: 34, SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37, SEQ ID NO: 38, SEQ ID NO: 39, SEQ ID NO: 40, SEQ ID NO: 41, SEQ ID NO: 42, SEQ ID NO: 43, SEQ ID NO: 44, SEQ ID NO: 45, SEQ ID NO: 46, SEQ ID NO: 47, SEQ ID NO: 48, SEQ ID NO: 49, or SEQ ID NO: 50.","In another embodiment, the present disclosure provides an expression vector comprising a polynucleotide comprising the sequence of SEQ ID NO: 26 encoding EtuA1, the polynucleotide sequence of SEQ ID NO: 27 encoding EtuA2, the polynucleotide sequence of SEQ ID NO: 28 encoding EtuA3, the polynucleotide sequence of SEQ ID NO: 29 encoding EtuD1, the polynucleotide sequence of SEQ ID NO: 30 encoding EtuD2, the polynucleotide sequence of SEQ ID NO: 31 encoding EtuD3, the polynucleotide sequence of SEQ ID NO: 32 encoding EtuF1, the polynucleotide sequence of SEQ ID NO: 33 encoding EtuF2, the polynucleotide sequence of SEQ ID NO: 34 encoding EtuF3, the polynucleotide sequence of SEQ ID NO: 35 encoding EtuH, the polynucleotide sequence of SEQ ID NO:36 encoding EtuM1, the polynucleotide sequence of SEQ ID NO: 37 encoding EtuM2, the polynucleotide sequence of SEQ ID NO: 38 encoding EtuN1, the polynucleotide sequence of SEQ ID NO: 39 encoding EtuN2, the polynucleotide sequence of SEQ ID NO: 40 encoding EtuN3, the polynucleotide sequence of SEQ ID NO: 41 encoding EtuO, the polynucleotide sequence of SEQ ID NO: 42 encoding EtuP1, the polynucleotide sequence of SEQ ID NO: 43 encoding EtuP2, the polynucleotide sequence of SEQ ID NO: 44 encoding EtuR1, the polynucleotide sequence of SEQ ID NO: 45 encoding EtuR2, the polynucleotide sequence of SEQ ID NO:46 encoding EtuR3, the polynucleotide sequence of SEQ ID NO: 47 encoding EtuT, the polynucleotide sequence of SEQ ID NO: 48 encoding EtuU1, the polynucleotide sequence of SEQ ID NO: 49 encoding EtuU2, and the polynucleotide sequence of SEQ ID NO: 50 encoding EtuU3.","Also provided is an expression vector comprising a polynucleotide that is 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98% or 99% identical to the polynucleotide sequence set out in SEQ ID NO: 26, SEQ ID NO: 27, SEQ ID NO: 28, SEQ ID NO: 29, SEQ ID NO: 30, SEQ ID NO: 31, SEQ ID NO: 32, SEQ ID NO: 33, SEQ ID NO: 34, SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37, SEQ ID NO: 38, SEQ ID NO: 39, SEQ ID NO: 40, SEQ ID NO: 41, SEQ ID NO: 42, SEQ ID NO: 43, SEQ ID NO: 44, SEQ ID NO: 45, SEQ ID NO: 46, SEQ ID NO: 47, SEQ ID NO: 48, SEQ ID NO: 49, or SEQ ID NO: 50.","The disclosure further provides a polynucleotide comprising one or more of SEQ ID NO: 26, SEQ ID NO: 27, SEQ ID NO: 28, SEQ ID NO: 29, SEQ ID NO: 30, SEQ ID NO: 31, SEQ ID NO: 32, SEQ ID NO: 33, SEQ ID NO: 34, SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37, SEQ ID NO: 38, SEQ ID NO: 39, SEQ ID NO: 40, SEQ ID NO: 41, SEQ ID NO: 42, SEQ ID NO: 43, SEQ ID NO: 44, SEQ ID NO: 45, SEQ ID NO:46, SEQ ID NO: 47, SEQ ID NO: 48, SEQ ID NO: 49, and SEQ ID NO: 50. Also provided is a polynucleotide comprising one or more sequences that is 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98% or 99% identical to the polynucleotide sequence set out in SEQ ID NO: 26, SEQ ID NO: 27, SEQ ID NO: 28, SEQ ID NO: 29, SEQ ID NO: 30, SEQ ID NO: 31, SEQ ID NO: 32, SEQ ID NO: 33, SEQ ID NO: 34, SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37, SEQ ID NO: 38, SEQ ID NO: 39, SEQ ID NO: 40, SEQ ID NO: 41, SEQ ID NO: 42, SEQ ID NO: 43, SEQ ID NO: 44, SEQ ID NO: 45, SEQ ID NO: 46, SEQ ID NO: 47, SEQ ID NO: 48, SEQ ID NO: 49, or SEQ ID NO: 50","The disclosure further provides an expression vector comprising a polynucleotide comprising one or more of SEQ ID NO: 26, SEQ ID NO: 27, SEQ ID NO: 28, SEQ ID NO: 29, SEQ ID NO: 30, SEQ ID NO: 31, SEQ ID NO: 32, SEQ ID NO: 33, SEQ ID NO: 34, SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37, SEQ ID NO: 38, SEQ ID NO: 39, SEQ ID NO: 40, SEQ ID NO: 41, SEQ ID NO: 42, SEQ ID NO: 43, SEQ ID NO: 44, SEQ ID NO: 45, SEQ ID NO:46, SEQ ID NO: 47, SEQ ID NO: 48, SEQ ID NO: 49, and SEQ ID NO: 50. Also provided is an expression vector comprising a polynucleotide comprising one or more sequences that is 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98% or 99% identical to the polynucleotide sequence set out in SEQ ID NO: 26, SEQ ID NO: 27, SEQ ID NO: 28, SEQ ID NO: 29, SEQ ID NO: 30, SEQ ID NO: 31, SEQ ID NO: 32, SEQ ID NO: 33, SEQ ID NO: 34, SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37, SEQ ID NO: 38, SEQ ID NO: 39, SEQ ID NO: 40, SEQ ID NO: 41, SEQ ID NO: 42, SEQ ID NO: 43, SEQ ID NO: 44, SEQ ID NO: 45, SEQ ID NO:46, SEQ ID NO: 47, SEQ ID NO: 48, SEQ ID NO: 49, or SEQ ID NO: 50","In some embodiments, a transformed host cell is provided, wherein the host cell is transformed with a heterologous polynucleotide comprising the polynucleotide sequence set out in SEQ ID NO: 26, SEQ ID NO: 27, SEQ ID NO: 28, SEQ ID NO: 29, SEQ ID NO: 30, SEQ ID NO: 31, SEQ ID NO: 32, SEQ ID NO: 33, SEQ ID NO: 34, SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37, SEQ ID NO: 38, SEQ ID NO: 39, SEQ ID NO: 40, SEQ ID NO: 41, SEQ ID NO: 42, SEQ ID NO: 43, SEQ ID NO: 44, SEQ ID NO: 45, SEQ ID NO:46, SEQ ID NO: 47, SEQ ID NO: 48, SEQ ID NO: 49, or SEQ ID NO: 50. In another embodiment, a host cell is provided which is transformed with a polynucleotide that is 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98% or 99% identical to the polynucleotide sequence set out in the polynucleotide sequence set out in SEQ ID NO: 26, SEQ ID NO: 27, SEQ ID NO: 28, SEQ ID NO: 29, SEQ ID NO: 30, SEQ ID NO: 31, SEQ ID NO: 32, SEQ ID NO: 33, SEQ ID NO: 34, SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37, SEQ ID NO: 38, SEQ ID NO: 39, SEQ ID NO: 40, SEQ ID NO: 41, SEQ ID NO: 42, SEQ ID NO: 43, SEQ ID NO: 44, SEQ ID NO: 45, SEQ ID NO:46, SEQ ID NO: 47, SEQ ID NO: 48, SEQ ID NO: 49, or SEQ ID NO: 50. In yet another embodiment, a transformed host cell is provided, wherein the host cell is transformed with one or more heterologous polynucleotides comprising the polynucleotide sequence set out in SEQ ID NO: 26, SEQ ID NO: 27, SEQ ID NO: 28, SEQ ID NO: 29, SEQ ID NO: 30, SEQ ID NO: 31, SEQ ID NO: 32, SEQ ID NO: 33, SEQ ID NO: 34, SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37, SEQ ID NO: 38, SEQ ID NO: 39, SEQ ID NO: 40, SEQ ID NO: 41, SEQ ID NO: 42, SEQ ID NO: 43, SEQ ID NO: 44, SEQ ID NO: 45, SEQ ID NO:46, SEQ ID NO: 47, SEQ ID NO: 48, SEQ ID NO: 49, and SEQ ID NO: 50. In still another embodiment, a transformed host cell is provided comprising a polynucleotide that is 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98% or 99% identical to one or more polynucleotide sequences set out in the polynucleotide sequences set out in SEQ ID NO: 26, SEQ ID NO: 27, SEQ ID NO: 28, SEQ ID NO: 29, SEQ ID NO: 30, SEQ ID NO: 31, SEQ ID NO: 32, SEQ ID NO: 33, SEQ ID NO: 34, SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37, SEQ ID NO: 38, SEQ ID NO: 39, SEQ ID NO: 40, SEQ ID NO: 41, SEQ ID NO: 42, SEQ ID NO: 43, SEQ ID NO: 44, SEQ ID NO: 45, SEQ ID NO:46, SEQ ID NO: 47, SEQ ID NO: 48, SEQ ID NO: 49, and SEQ ID NO: 50.","Also provided is a host cell transformed with a polynucleotide encoding (i) EtuA1 (SEQ ID NO: 1) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 1 having EtuA1 activity, and (ii) EtuA2 (SEQ ID NO:2) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 2 having EtuA2 activity. In various embodiments, the host cell is further transformed with a polynucleotide encoding EtuF3 (SEQ ID NO: 3) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 3 having EtuF3 activity. In still further embodiments, the host cell of is further transformed with a polynucleotide encoding EtuO (SEQ ID NO: 16) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 16 having EtuO activity. In still other embodiments, the host cell is further transformed with a polynucleotide encoding EtuA3 (SEQ ID NO: 3) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 3 having EtuA3 activity. In other embodiments, host cell is further transformed with a polynucleotide encoding (i) EtuH (SEQ ID NO: 10) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 10 having EtuH activity (ii) EtuM2 (SEQ ID NO: 12) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 12 having EtuM2 activity (iii) EtuM1 (SEQ ID NO:11) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 11 having EtuM1 activity.","Also provided is a host cell transformed with a polynucleotide encoding (i) EtuA1 (SEQ ID NO: 1) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 1 having EtuA1 activity, and (ii) EtuA2 (SEQ ID NO:2) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 2 having EtuA2 activity, the host cell further transformed with a polynucleotide encoding EtuA3 (SEQ ID NO: 3) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 3 having EtuA3 activity. In various embodiments, the host cell is further transformed with a polynucleotide encoding EtuF3 (SEQ ID NO: 9) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 9 having EtuF3 activity. In still other embodiments, the host cell is further transformed with a polynucleotide encoding EtuO (SEQ ID NO: 16) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 16 having EtuO activity. In still other embodiments, the host cell is further transformed with a polynucleotide encoding EtuH (SEQ ID NO: 10) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 10 having EtuH activity (ii) EtuM2 (SEQ ID NO: 12) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 12 having EtuM2 activity (iii) EtuM1 (SEQ ID NO: 11) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 11 having EtuM1 activity.","Also provided is a host cell transformed with a polynucleotide encoding EtuO (SEQ ID NO: 16) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 16 having EtuO activity. In various embodiments, the host cell is further transformed with a polynucleotide encoding (i) EtuF3 (SEQ ID NO: 9) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 9 having EtuF3 activity (ii) EtuA1 (SEQ ID NO: 1) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 1 having EtuA1 activity (iii) EtuA2 (SEQ ID NO: 2) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 2, having EtuA2 activity. In other aspects, the host cell is further transformed with a polynucleotide encoding EtuA3 (SEQ ID NO: 3) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 3 having EtuA3 activity. In still other aspects, the host cell is further transformed with a polynucleotide encoding EtuH (SEQ ID NO: 10) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 10 having EtuH activity (ii) EtuM2 (SEQ ID NO: 12) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 12 having EtuM2 activity (iii) EtuM1 (SEQ ID NO: 11) or a protein 90%, 91%, 92%, 93, 94%, 95%, 96%, 97%, 98% or 99% identical to SEQ ID NO: 11 having EtuM1 activity.","The present disclosure additionally provides a method for synthesizing a nonribosomal peptide synthetase (NRPS) derived drug, an analog of an NRPS derived drug, or a metabolic intermediate in the NRPS derived drug synthetic pathway in a host cell comprising the step of culturing a host cell of the present disclosure under conditions suitable to produce the nonribosomal peptide synthetase (NRPS) derived drug and\/or analog. In some aspects, the NRPS derived drug is ET743.","In various embodiments, the host cell is a prokaryote. In some aspects, the prokaryote is selected from the group consisting of , and ","In some embodiments, the host cell comprises one or more heterologous polynucleotides encoded on an expression vector. In some aspects, the one or more heterologous polynucleotides are present in a single expression vector. In further aspects, the expression vector comprises a sequence that encodes the polypeptide sequence as set forth in SEQ ID NO: 4.","In some embodiments, the at least two heterologous polynucleotides are encoded on separate expression vectors. In some aspects, the at least two of the heterologous polynucleotides are encoded on a single expression vector.","Disclosed herein is a biosynthetic pathway for production of a NRPS derived drug, ET743. Knowledge of this pathway enables economical access to the drug and analogs through heterologous expression [Tang et al., 287: 640-642 (2000); Wenzel et al., 16: 594 (2005)] and pathway engineering [Nguyen et al., 103: 17462 (2006)]. The disclosure of the polynucleotide and polypeptide sequences herein began with collection of the tunicate . Presence of ET743 in the sample collected was verified by mass spectrometry as a surrogate for the presence of the presumed symbiont. Two rounds of metagenomic Roche 454 FLX\u2122 Platinum sequencing were performed and after multiple rounds of gap filling by PCR and Sanger sequencing, assembly and annotation, a putative gene cluster for ET743 biosynthesis was identified. Key proteins in the proposed biosynthetic pathway were heterologously expressed and activity was determined in vitro with mass spectrometry and radioassays, confirming key steps at the initiation, cyclization, and termination of ET743 biosynthesis. This work has broad implications for accessing other unknown natural products from complex assemblages. In one embodiment, the present disclosure provides a method for directly accessing this potent drug through fermentation.","Accordingly, the present disclosure provides a polynucleotide sequence encoding the one or more polypeptides in the pathway for production of Etu disclosed herein. Those polypeptides include the polypeptide sequence of SEQ ID NO: 1 (EtuA1), the polypeptide sequence of SEQ ID NO: 2 (EtuA2), the polypeptide sequence of SEQ ID NO: 3 (EtuA3), the polypeptide sequence of SEQ ID NO: 4 (EtuD1), the polypeptide sequence of SEQ ID NO: 5 (EtuD2), the polypeptide sequence of SEQ ID NO: 6 (EtuD3), the polypeptide sequence of SEQ ID NO: 7 (EtuF1), the polypeptide sequence of SEQ ID NO: 8 (EtuF2), the polypeptide sequence of SEQ ID NO: 9 (EutF3), the polypeptide sequence of SEQ ID NO: 10 (EtuH), the polypeptide sequence of SEQ ID NO: 11 (EtuM1), the polypeptide sequence of SEQ ID NO: 12 (EtuM2), the polypeptide sequence of SEQ ID NO: 13 (EtuN1), the polypeptide sequence of SEQ ID NO: 14 (EtuN2), the polypeptide sequence of SEQ ID NO: 15 (EtuN3), the polypeptide sequence of SEQ ID NO: 16 (EtuO), the polypeptide sequence of SEQ ID NO: 17 (EtuP1), the polypeptide sequence of SEQ ID NO: 18 (EtuP2), the polypeptide sequence of SEQ ID NO: 19 (EutR1), the polypeptide sequence of SEQ ID NO: 20 (EtuR2): the polypeptide sequence of SEQ ID NO: 21 (EtuR3), the polypeptide sequence of SEQ ID NO: 22 (EtuT), the polypeptide sequence of SEQ ID NO: 23 (EtuU1), the polypeptide sequence of SEQ ID NO: 24 (EtuU2), or the polypeptide sequence of SEQ ID NO: 25 (EtuU3). Polypeptides contemplated also include those comprising an polypeptide that is 90%, 91%, 92%, 93%, 94%,","A polynucleotide sequence is also provided that encodes the polypeptide sequence of SEQ ID NO: 1 (EtuA1), the polypeptide sequence of SEQ ID NO: 2 (EtuA2), the polypeptide sequence of SEQ ID NO: 3 (EtuA3), the polypeptide sequence of SEQ ID NO: 4 (EtuD1), the polypeptide sequence of SEQ ID NO: 5 (EtuD2), the polypeptide sequence of SEQ ID NO: 6 (EtuD3), the polypeptide sequence of SEQ ID NO: 7 (EtuF1), the polypeptide sequence of SEQ ID NO: 8 (EtuF2), the polypeptide sequence of SEQ ID NO: 9 (EutF3), the polypeptide sequence of SEQ ID NO: 10 (EtuH), the polypeptide sequence of SEQ ID NO: 11 (EtuM1), the polypeptide sequence of SEQ ID NO: 12 (EtuM2), the polypeptide sequence of SEQ ID NO: 13 (EtuN1), the polypeptide sequence of SEQ ID NO: 14 (EtuN2), the polypeptide sequence of SEQ ID NO: 15 (EtuN3), the polypeptide sequence of SEQ ID NO: 16 (EtuO), the polypeptide sequence of SEQ ID NO: 17 (EtuP1), the polypeptide sequence of SEQ ID NO: 18 (EtuP2), the polypeptide sequence of SEQ ID NO: 19 (EutR1), the polypeptide sequence of SEQ ID NO: 20 (EtuR2): the polypeptide sequence of SEQ ID NO: 21 (EtuR3), the polypeptide sequence of SEQ ID NO: 22 (EtuT), the polypeptide sequence of SEQ ID NO: 23 (EtuU1), the polypeptide sequence of SEQ ID NO: 24 (EtuU2), or the polypeptide sequence of SEQ ID NO: 25 (EtuU3).","A polynucleotide sequence is also provided that encodes a polypeptide that is 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, or 99% identical to the polypeptide sequence of SEQ ID NO: 1 (EtuA1), the polypeptide sequence of SEQ ID NO: 2 (EtuA2), the polypeptide sequence of SEQ ID NO: 3 (EtuA3), the polypeptide sequence of SEQ ID NO: 4 (EtuD1), the polypeptide sequence of SEQ ID NO: 5 (EtuD2), the polypeptide sequence of SEQ ID NO: 6 (EtuD3), the polypeptide sequence of SEQ ID NO: 7 (EtuF1), the polypeptide sequence of SEQ ID NO: 8 (EtuF2), the polypeptide sequence of SEQ ID NO: 9 (EutF3), the polypeptide sequence of SEQ ID NO: 10 (EtuH), the polypeptide sequence of SEQ ID NO: 11 (EtuM1), the polypeptide sequence of SEQ ID NO: 12 (EtuM2), the polypeptide sequence of SEQ ID NO: 13 (EtuN1), the polypeptide sequence of SEQ ID NO: 14 (EtuN2), the polypeptide sequence of SEQ ID NO: 15 (EtuN3), the polypeptide sequence of SEQ ID NO: 16 (EtuO), the polypeptide sequence of SEQ ID NO: 17 (EtuP1), the polypeptide sequence of SEQ ID NO: 18 (EtuP2), the polypeptide sequence of SEQ ID NO: 19 (EutR1), the polypeptide sequence of SEQ ID NO: 20 (EtuR2): the polypeptide sequence of SEQ ID NO: 21 (EtuR3), the polypeptide sequence of SEQ ID NO: 22 (EtuT), the polypeptide sequence of SEQ ID NO: 23 (EtuU1), the polypeptide sequence of SEQ ID NO: 24 (EtuU2), or the polypeptide sequence of SEQ ID NO: 25 (EtuU3).","Exemplary polynucleotides encoding polypeptides of the disclosure are set out in Table 1.",{"@attributes":{"id":"p-0038","num":"0037"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"thead":{"row":[{"entry":[{},"TABLE 1"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},{},"Polypeptide Sequence","Polynucleotide"]},{"entry":[{},"Etu Gene","ID Number","Sequence ID Number"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"A1","1","26"]},{"entry":[{},"A2","2","27"]},{"entry":[{},"A3","3","28"]},{"entry":[{},"D1","4","29"]},{"entry":[{},"D2","5","30"]},{"entry":[{},"D3","6","31"]},{"entry":[{},"F1","7","32"]},{"entry":[{},"F2","8","33"]},{"entry":[{},"F3","9","34"]},{"entry":[{},"H","10","35"]},{"entry":[{},"M1","11","36"]},{"entry":[{},"M2","12","37"]},{"entry":[{},"N1","13","38"]},{"entry":[{},"N2","14","39"]},{"entry":[{},"N3","15","40"]},{"entry":[{},"O","16","41"]},{"entry":[{},"P1","17","42"]},{"entry":[{},"P2","18","43"]},{"entry":[{},"Rl","19","44"]},{"entry":[{},"R2","20","45"]},{"entry":[{},"R3","21","46"]},{"entry":[{},"T","22","47"]},{"entry":[{},"U1","23","48"]},{"entry":[{},"U2","24","49"]},{"entry":[{},"U3","25","50"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},"Accordingly, the disclosure provides a polynucleotide comprising the sequence of SEQ ID NO: 26 encoding EtuA1, the polynucleotide sequence of SEQ ID NO: 27 encoding EtuA2, the polynucleotide sequence of SEQ ID NO: 28 encoding EtuA3, the polynucleotide sequence of SEQ ID NO: 29 encoding EtuD1, the polynucleotide sequence of SEQ ID NO: 30 encoding EtuD2, the polynucleotide sequence of SEQ ID NO: 31 encoding EtuD3, the polynucleotide sequence of SEQ ID NO: 32 encoding EtuF1, the polynucleotide sequence of SEQ ID NO: 33 encoding EtuF2, the polynucleotide sequence of SEQ ID NO: 34 encoding EtuF3, the polynucleotide sequence of SEQ ID NO: 35 encoding EtuH, the polynucleotide sequence of SEQ ID NO:36 encoding EtuM1, the polynucleotide sequence of SEQ ID NO: 37 encoding EtuM2, the polynucleotide sequence of SEQ ID NO: 38 encoding EtuN1, the polynucleotide sequence of SEQ ID NO: 39 encoding EtuN2, the polynucleotide sequence of SEQ ID NO: 40 encoding EtuN3, the polynucleotide sequence of SEQ ID NO: 41 encoding EtuO, the polynucleotide sequence of SEQ ID NO: 42 encoding EtuP1, the polynucleotide sequence of SEQ ID NO: 43 encoding EtuP2, the polynucleotide sequence of SEQ ID NO: 44 encoding EtuR1, the polynucleotide sequence of SEQ ID NO: 45 encoding EtuR2, the polynucleotide sequence of SEQ ID NO:46 encoding EtuR3, the polynucleotide sequence of SEQ ID NO: 47 encoding EtuT, the polynucleotide sequence of SEQ ID NO: 48 encoding EtuU1, the polynucleotide sequence of SEQ ID NO: 49 encoding EtuU2, and the polynucleotide sequence of SEQ ID NO: 50 encoding EtuU3.","Also provided is a polynucleotide that is 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98% or 99% identical to the polynucleotide sequence set out in SEQ ID NO: 26, SEQ ID NO: 27, SEQ ID NO: 28, SEQ ID NO: 29, SEQ ID NO: 30, SEQ ID NO: 31, SEQ ID NO: 32, SEQ ID NO: 33, SEQ ID NO: 34, SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37, SEQ ID NO: 38, SEQ ID NO: 39, SEQ ID NO: 40, SEQ ID NO: 41, SEQ ID NO: 42, SEQ ID NO: 43, SEQ ID NO: 44, SEQ ID NO: 45, SEQ ID NO:46, SEQ ID NO: 47, SEQ ID NO: 48, SEQ ID NO: 49, or SEQ ID NO: 50.","Also provided by the present disclosure is a host cell transformed with a heterologous polynucleotide comprising any one or more of the polynucleotide sequences comprising the sequence of SEQ ID NO: 26 encoding EtuA1, the polynucleotide sequence of SEQ ID NO: 27 encoding EtuA2, the polynucleotide sequence of SEQ ID NO: 28 encoding EtuA3, the polynucleotide sequence of SEQ ID NO: 29 encoding EtuD1, the polynucleotide sequence of SEQ ID NO: 30 encoding EtuD2, the polynucleotide sequence of SEQ ID NO: 31 encoding EtuD3, the polynucleotide sequence of SEQ ID NO: 32 encoding EtuF1, the polynucleotide sequence of SEQ ID NO: 33 encoding EtuF2, the polynucleotide sequence of SEQ ID NO: 34 encoding EtuF3, the polynucleotide sequence of SEQ ID NO: 35 encoding EtuH, the polynucleotide sequence of SEQ ID NO:36 encoding EtuM1, the polynucleotide sequence of SEQ ID NO: 37 encoding EtuM2, the polynucleotide sequence of SEQ ID NO: 38 encoding EtuN1, the polynucleotide sequence of SEQ ID NO: 39 encoding EtuN2, the polynucleotide sequence of SEQ ID NO: 40 encoding EtuN3, the polynucleotide sequence of SEQ ID NO: 41 encoding EtuO, the polynucleotide sequence of SEQ ID NO: 42 encoding EtuP1, the polynucleotide sequence of SEQ ID NO: 43 encoding EtuP2, the polynucleotide sequence of SEQ ID NO: 44 encoding EtuR1, the polynucleotide sequence of SEQ ID NO: 45 encoding EtuR2, the polynucleotide sequence of SEQ ID NO:46 encoding EtuR3, the polynucleotide sequence of SEQ ID NO: 47 encoding EtuT, the polynucleotide sequence of SEQ ID NO: 48 encoding EtuU1, the polynucleotide sequence of SEQ ID NO: 49 encoding EtuU2, and the polynucleotide sequence of SEQ ID NO: 50 encoding EtuU3. Host cells are also provided comprising one or more polynucleotides that is\/are 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98% or 99% identical to one or more polynucleotide sequences set out in the polynucleotide sequences set out in SEQ ID NO: 26, SEQ ID NO: 27, SEQ ID NO: 28, SEQ ID NO: 29, SEQ ID NO: 30, SEQ ID NO: 31, SEQ ID NO: 32, SEQ ID NO: 33, SEQ ID NO: 34, SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37, SEQ ID NO: 38, SEQ ID NO: 39, SEQ ID NO: 40, SEQ ID NO: 41, SEQ ID NO: 42, SEQ ID NO: 43, SEQ ID NO: 44, SEQ ID NO: 45, SEQ ID NO:46, SEQ ID NO: 47, SEQ ID NO: 48, SEQ ID NO: 49, and SEQ ID NO: 50. Additionally, host cells are provided comprising a polynucleotide encoding one or more polypeptides selected from the group consisting of the polypeptide sequence of SEQ ID NO: 1 (EtuA1), the polypeptide sequence of SEQ ID NO: 2 (EtuA2), the polypeptide sequence of SEQ ID NO: 3 (EtuA3), the polypeptide sequence of SEQ ID NO: 4 (EtuD1), the polypeptide sequence of SEQ ID NO: 5 (EtuD2), the polypeptide sequence of SEQ ID NO: 6 (EtuD3), the polypeptide sequence of SEQ ID NO: 7 (EtuF1), the polypeptide sequence of SEQ ID NO: 8 (EtuF2), the polypeptide sequence of SEQ ID NO: 9 (EutF3), the polypeptide sequence of SEQ ID NO: 10 (EtuH), the polypeptide sequence of SEQ ID NO: 11 (EtuM1), the polypeptide sequence of SEQ ID NO: 12 (EtuM2), the polypeptide sequence of SEQ ID NO: 13 (EtuN1), the polypeptide sequence of SEQ ID NO: 14 (EtuN2), the polypeptide sequence of SEQ ID NO: 15 (EtuN3), the polypeptide sequence of SEQ ID NO: 16 (EtuO), the polypeptide sequence of SEQ ID NO: 17 (EtuP1), the polypeptide sequence of SEQ ID NO: 18 (EtuP2), the polypeptide sequence of SEQ ID NO: 19 (EutR1), the polypeptide sequence of SEQ ID NO: 20 (EtuR2): the polypeptide sequence of SEQ ID NO: 21 (EtuR3), the polypeptide sequence of SEQ ID NO: 22 (EtuT), the polypeptide sequence of SEQ ID NO: 23 (EtuU1), the polypeptide sequence of SEQ ID NO: 24 (EtuU2), or the polypeptide sequence of SEQ ID NO: 25 (EtuU3) In still other aspects, a host cell is provided comprising a polynucleotide encoding one or more polypeptides that is 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, or 99% identical to the polypeptide sequence of SEQ ID NO: 1 (EtuA1), the polypeptide sequence of SEQ ID NO: 2 (EtuA2), the polypeptide sequence of SEQ ID NO: 3 (EtuA3), the polypeptide sequence of SEQ ID NO: 4 (EtuD1), the polypeptide sequence of SEQ ID NO: 5 (EtuD2), the polypeptide sequence of SEQ ID NO: 6 (EtuD3), the polypeptide sequence of SEQ ID NO: 7 (EtuF1), the polypeptide sequence of SEQ ID NO: 8 (EtuF2), the polypeptide sequence of SEQ ID NO: 9 (EutF3), the polypeptide sequence of SEQ ID NO: 10 (EtuH), the polypeptide sequence of SEQ ID NO: 11 (EtuM1), the polypeptide sequence of SEQ ID NO: 12 (EtuM2), the polypeptide sequence of SEQ ID NO: 13 (EtuN1), the polypeptide sequence of SEQ ID NO: 14 (EtuN2), the polypeptide sequence of SEQ ID NO: 15 (EtuN3), the polypeptide sequence of SEQ ID NO: 16 (EtuO), the polypeptide sequence of SEQ ID NO: 17 (EtuP1), the polypeptide sequence of SEQ ID NO: 18 (EtuP2), the polypeptide sequence of SEQ ID NO: 19 (EutR1), the polypeptide sequence of SEQ ID NO: 20 (EtuR2): the polypeptide sequence of SEQ ID NO: 21 (EtuR3), the polypeptide sequence of SEQ ID NO: 22 (EtuT), the polypeptide sequence of SEQ ID NO: 23 (EtuU1), the polypeptide sequence of SEQ ID NO: 24 (EtuU2), and or the polypeptide sequence of SEQ ID NO: 25 (EtuU3).","It will be understood by those of skill in the art that a host cell can be transformed with 1, 2, 3, 4, 5, 6, 7, 8, 9 or all of the polynucleotides identified in Table 1. In various aspects, a transformed host cell comprises one, two, three or more expression vectors. Suitable host cells are known to those of ordinary skill in the art, and include without limitation prokaryotic host cells.","\u201cHeterologous,\u201d as used herein, means of different natural origin or of synthetic origin. For example and without limitation, if a host cell is transformed with a polynucleotide sequence that does not occur in the untransformed host cell, that nucleic acid sequence is said to be heterologous with respect to the host cell. The transforming nucleic acid optionally includes, in various embodiments, a heterologous promoter, heterologous coding sequence, and\/or heterologous termination sequence. Alternatively, the transforming polynucleotide in another embodiment, is completely heterologous or includes any possible combination of heterologous and endogenous polynucleotide sequences. Similarly, \u201cheterologous\u201d refers to a polynucleotide sequence derived from and inserted into the same natural, original cell type, but which is present in a non-natural state, e.g., a different copy number, or under the control of different regulatory elements. The term \u201cheterologous\u201d applies to cells, including plant and bacterial cells, and also to plasmids, plastids, and viruses.","As used herein, \u201canalog\u201d means that a described compound shares a structural or functional characteristic with a parent compound from which it was derived or designed.","It is understood that the term \u201cintermediate\u201d as used means that a compound may be subjected to further processing steps, for example and without limitation as disclosed herein for the synthesis of ET743. The term \u201cintermediate\u201d is used interchangeably with the term \u201cmetabolic intermediate\u201d herein.  sets out a pathway for production of ET743 with various intermediates in the synthetic pathway. Intermediates are contemplated without limitation for their immediate use, for isolation and storage for use at a later time, for use in the production of analogs, and as semi-synthetic precursors of the NRPS not limited to ET743. In some embodiments, inter mediates are contemplated for the production of semi-synthetic compounds that have antimicrobial, anticancer, and\/or anti-inflammatory activity. The person of skill in the art will appreciate that intermediates as described herein are also useful in themselves, or useful for production of, for example, ET743 by their modification in an in vitro systems incorporating one or more isolated polypeptides as described herein and in  that allow for production of ET743. Alternatively, intermediates are used to in vivo systems wherein a transformed host cell as described here is capable of taking up the intermediate and further processing the intermediate to provide ET743.","In an embodiment, the present disclosure provides a method for synthesizing a nonribosomal peptide synthetase (NRPS) derived drug and\/or analog and\/or intermediate in a host cell comprising the step of culturing the host cell as described herein under conditions suitable to produce the nonribosomal peptide synthetase (NRPS) derived drug and\/or analog. Illustrative host cells include prokaryotic, yeast, avian, fungal, insect, mammalian, and plant cells. Prokaryotic host cells contemplated herein include without limitation , and host cells contemplated herein include without limitation laboratory and\/or industrial strains of cells, such as BL21 or K12-derived strains (e.g., C600, DH1\u03b1, DH5\u03b1, HB101, INV1, JM109, TB1, TG1, and X-1Blue). Such strains are available from the ATCC or from commercial vendors such as BD Biosciences Clontech (Palo Alto, Calif.) and Stratagene (La Jolla, Calif.). For detailed descriptions of nucleic acid manipulation techniques, see Ausubel et al., eds., Current Protocols in Molecular Biology, Wiley Interscience, 2006, and Sambrook and Russell, Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Press, NY, 2001, each of which is incorporated herein by reference in its entirety.","As used herein, \u201cexpression vector\u201d refers to a polynucleotide sequence capable of directing expression of a particular polynucleotide sequence in an appropriate host cell, including a promoter operably linked to the polynucleotide sequence of interest, which is optionally operably linked to 3\u2032 sequences, such as 3\u2032 regulatory sequences or termination signals. In some aspects, an expression vector also includes sequences required for proper translation of the polynucleotide sequence if any such sequences are needed.","Methods are also provided wherein a host cell transformed with one or more expression vectors, encoding one or more polypeptides of the present disclosure, produces an intermediate compound in the synthesis of a NRPS drug. In some aspects of the method, the intermediate is isolated. In various other aspects of the method, the intermediate is used to complete synthesis of the NRPS drug. The synthesis method is, in various aspects, completed in the same host cell. In other aspects, the synthesis method is completed in a different host cell. In further aspects, methods are provided wherein the different host cell is transformed with one or more expression vectors encoding one or more polypeptides that are useful for the completion of said synthesis. The one or more polypeptides are, in some aspects, any of the polypeptides described herein, and in some aspects are heterologous polypeptides that catalyze the same or similar steps in the biosynthetic pathway. It will be understood by those of ordinary skill in the art that polypeptides from genetically related or unrelated organisms may have the same or similar enzymatic capabilities as those disclosed herein. Accordingly, it is contemplated that such polypeptides may be used in combination with the polypeptides described herein in the synthesis of a NRPS drug and\/or analog and\/or intermediate.","In some embodiments, at least two heterologous polynucleotides are encoded on separate expression vectors. In some embodiments, at least two of the heterologous polynucleotides are encoded on a single expression vector.","It is noted here that, as used in this specification and the appended claims, the singular forms \u201ca,\u201d \u201can,\u201d and \u201cthe\u201d include plural reference unless the context clearly dictates otherwise.","EtuA1, EtuA2, EtuA3 are Three Predicted NRPSs with Catalytic Domains Bearing Predicted Amino Acid Specificity Motifs.","Sequence analysis and deep annotation revealed that biosynthetic pathway architecture is non-collinear (as with SafA-B) and is represented by EtuA3\u2192EtuA1\u2192EtuA2 EtuA3 (AL-T-C-A-T) contains the AL-T starter module that is common to the saframycin, and saframycin Mx1 metabolic systems. The role of this module was elucidated for the saframycin biosynthetic pathway, where acylation of the precursor is required for further chain extension, cyclization and RE processing (Pictet-Spenglerase). The NRPS A-domain, based upon the amino acid specificity motif, was predicted to utilize cysteine with 100% sequence identity to the top three cysteine A domain sequence motifs. EtuA3 specificity is, therefore, unique to the Etu biosynthetic pathway and a key differentiator compared to other characterized tetrahydroisoquinoline systems, which all utilize alanine. EtuA1 (C-A-T) has the greatest homology to SafA module 1 by BLASTx; however, the protein sequence identity and similarity are relatively low (29\/54) compared to the other NRPSs in the pathway. An A-domain selectivity motif cannot be identified in EtuA1. Based on structural analysis of ET-743, a glycolic acid unit may be loaded and activated by the EtuA1 A-domain. Loading of hydroxy acids and formation of esters by NRPS modules have been characterized previously. This extender unit represents another key difference relative to characterized tetrahydroisoquinoline antibiotics, for which a conserved core motif (7\/8 amino acid identity) is both predicted and observed to select glycine. EtuA2 (C-A-T-RE) contains the same A-domain specificity motif for the final NRPS module as all known tetrahydroisoquinoline biosynthetic pathways. As verified in the saframycin biosynthetic system, the EtuA2 homolog SfmC iteratively extends two 3H-4O-Me-5Me-Tyr residues. The terminal EtuR2 RE domain serves as a key marker of the pathway and was examined biochemically to assess its activity in elaborating the tetrahydroisoquinoline core molecule","DNA Processing Enzymes are Unusual to Observe in a Natural Product Biosynthetic Pathway.","It is hypothesized EtuD1-3 may have a role in repairing damage induced by ET-743 given its mechanism of action. EtuD1 appears to be a homolog of the TatD Mg2+ dependent DNAse while EtuD2 shows similarity to a DNA polymerase III subunit \u03b4\u2032, which has been characterized as part of the DNA-enzyme assembly complex. EtuD3 is a homolog of the 5\u2032\u21923\u2032 exonuclease domain from DNA polymerase I.","Fatty Acid Processing Enzymes.","Pathway components that mediate production of essential cofactors or substrates are often encoded within biosynthetic gene clusters21. EtuF1 and EtuF2 appear to represent subunits of an acetyl-CoA carboxylase. These enzymes transform acetyl-CoA to malonyl-CoA for fatty acid biosynthesis, and may supply substrate for synthesis of the fatty acid for EtuA3 AL. EtuF3 appears to be a penicillin acylase40. It is proposed that this key enzyme may act to release the predicted fatty acid modified intermediate of ET-743 after formation of the tetradepsipeptide and Pictet-Spengler cyclization () prior to further processing into mature intermediates that are isolable from the tunicate.","Generation of 3-hydroxy-4-O-methyl-5-methyl-tyrosine (Hydroxylase and Methyltransferases).","ET-743 is derived from at least two units of the unusual amino acid 3H-40-Me-5Me-Tyr. The intermediate may be generated through 3-hydroxylation, 4-O-methylation, and 5-methylation of tyrosine. EtuH, an SfmD homolog, is predicted to hydroxylate tyrosine at the 3-position, whereas EtuM1, a SacF homolog, may be a SAM-dependent methyltransferase and a candidate for C-methylation at the 5-position. SafC, an EtuM2 homolog, has been characterized in vitro as a catechol 4-O-methyltransferase41. Biochemical studies in the saframycin pathway revealed that SfmD (EtuH homolog), SfmM2 (EtuM1 homolog) and SmfM3 (EtuM2 homolog) form a minimal unit for 3H-4O-Me-5Me-Tyr production thus diverting tyrosine to secondary metabolism.","EtuO is an FAD-Dependent Monooxygenase that Shows High Similarity to SfmO2 and SacJ.","EtuO may catalyze modification of the tetrahydroisoquinoline to produce the hydroxylated species based on previous work involving sacJ gene disruption ( -). In vitro biochemical characterization of this enzyme will require synthesis of an advanced intermediate to determine its precise activity.","Regulatory Enzymes.","EtuR1 has significant similarity (59%) to S29x, a protein previously shown to have a role in host-symbiont interactions between Amoeba proteus and the symbiotic Gram-negative X-bacteria. This fascinating protein is excreted from the bacterium, and localized to the A. proteus nucleus. The role of S29x in host-symbiont interactions is unclear with no other homologs characterized. The presence of a homolog to a characterized symbiont-derived gene in the Etu cluster suggests that regulated host-symbiont interactions may be involved in ET-743 biosynthesis. BLASTx analysis of EtuR2 shows (34\/58%) identity\/similarity to a MerR family transcriptional regulator. This class of regulators has been found in diverse classes of bacteria and responds to toxic effectors including heavy metals and antibiotics45. EtuR3 resembles the TraR\/DksA transcriptional regulator that functions as a DNAK suppressor protein.","EtuT Appears to be a Drug Transporter Protein.","Members of this superfamily are commonly represented in natural product biosynthetic pathways and could serve as part of a resistance\/export mechanism for ET-743.","ET743 is derived from an unusual amino acid precursor 3-hydroxy-5-methyl-O-methyl-tyrosine (3h5mOmY). Previously, gene cassettes have been described and characterized for the generation of this key and unusual intermediate. As shown in , the pathway provided herein includes 3-hydroxylation of tyrosine, 4-O methylation of tyrosine, followed by 5-methylation of tyrosine.","The overall scheme for ET743 biosynthesis is also depicted in . Using cell-free extracts, Kerr and Miranda identified tyrosine and cysteine as biosynthetic building blocks, but incorrect coupling [Kerr et al., 58: 1618 (1995)]. Tunicate was harvested, lysed, and an active-cell free extract was fed radioactive tyrosine and cysteine. Based on this experiment, it was established that both tyrosine and cysteine are utilized in ET743 biosynthesis, although their biosynthetic scheme is incorrect given subsequent development of the model for NRPS biosynthesis.","Without being bound by a specific mechanism, ET743 synthesis begins with assembly of the key subunit 3H-4O-Me-5Me-Tyr (7) (). This non-proteinogenic amino acid is formed by 3-hydroxylation of (4), 4-O methylation of (5), and 5-methylation (6) catalyzed by EtuH, EtuM2, and EtuM141, respectively. Next, the fatty acid CoA ligase of EtuA3 loads a fatty acid (8) onto the T domain. This fatty acid is condensed with cysteine that is activated and loaded by the C-A-T module of EtuA3 (9). Cysteine condenses with a T-loaded glycolate on EtuA1 to form the acylated-depsipeptide (10). Based on Koketsu's model, (10) is reductively released by the EtuA2 RE-domain as an aldehydo-depsipeptide (11) from the EtuA1 T-domain. Such a \u201creach-back\u201d model is not unprecedented in natural product biosynthesis48. EtuA2 loaded with 3H-4O-Me-5Me-Tyr (7) is then condensed with (11) to form the cyclic aldehydo-tridepsipeptide (12) through the presumed Pictet-Spenglerase activity of the EtuA2 C domain. Intermediate (12) is released from the EtuA2 T by the RE-domain activity as an aldehyde (13). Based on Koketsu's model it is proposed that EtuA2 catalyzes a second Pictet-Spengler reaction between another unit of 3H-4O-Me-5Me-Tyr (7) and (13). The protein-bound tetradepsipeptide (14) is then reductively released to form aldehyde (15) that may undergo a further enzyme-catalyzed Pictet-Spengler reaction to form the fatty-acid bound carbinolamine pre-ET-743 (16). The penicillin acylase EtuF3 (16) is then proposed to cleave the fatty acid unit, which may serve to sequester substrate in the EtuA2 active site during repeated loading\/release, forming pre-ET-743 (17). Proposed intermediates ET-583 (18), ET-597 (19), ET-596 (20), and ET-594 (21) have all been isolated, characterized30, and all except ET-596 (20) have been confirmed by our secondary metabolite analysis (FIG. S3). It is further proposed that pre-ET-743 (17) is hydroxylated by EtuO, acetylation and formation of the thioether ring are both catalyzed by unknown enzymes\/mechanisms and intermediates to form ET-583 (18). An unidentified N-methyltransferase acts on ET-583 (18) to generate ET-597 (19). In accordance with Sakai, it is also proposed that a transamination reaction proceeds on (19) to produce ET-596 (20). Another unknown protein catalyzes formation of a methylene dioxybridge in the A ring to generate ET-594 (21). Since compounds (18-21) are isolable, and tryptophan analogs of ET-743 have also been observed, it is reasonable to propose that the final subunit to complete biosynthesis of the drug is added at a late stage, perhaps by formation of an imine to the \u03b2-carbonyl and the new tyrosine analog. In both ET-743 total synthesis and semi-synthesis schemes, the \u03b1-ketone (21) is transformed to the final tetrahydroisoquinoline ring system by addition of 4-O-methyl-tyrosine under mild conditions 8,10. Further processing steps are hypothetical, with neither enzyme nor intermediate identified. It is proposed that another tyrosine analog, 4-O-methyl-tyrosine (22) is condensed with ET-594 (21). The proposed imine intermediate (24) may then undergo another Pictet-Spengler-type reaction to form the final ring system (25). It is unknown if this unusual cyclization reaction and reduction is catalyzed by EtuA2 or an additional enzyme. The mechanism by which the proposed thioester of ET-743 is released from the proposed enzyme as ET-743 (1) remains to be established. Full validation of this proposed pathway will require synthesis of the predicted enzyme substrates, and direct biochemical analysis.",{"@attributes":{"id":"p-0068","num":"0067"},"i":["Ecteinascidia turbinata ","E. turbinata ","Marine Biology ","Bugula neritina","E. turbinata ","Proceedings of the National Academy of Sciences ","The Journal of Organic Chemistry ","The Journal of Organic Chemistry ","Medicinal Research Reviews "],"figref":"FIG. 4","sup":"+ "},"The samples collected were then prepared for high throughput sequencing and subjected to two rounds of Roche 454 FLX\u2122 Platinum sequencing. Of the more than 90,000 strains identified in the sample, the best candidate for ET743 production was identified. Previously, Moss and coworkers had examined 16S rRNA genes from and found that three bacterial species dominated, with one accounting for >50% of clones examined and assigned as a \u03b4-proteobacteria, (AY054370). In-situ hybridization was used to identify specific intracellular localization of the bacteria and it was also identified in the larvae\u2014suggesting that a symbiosis or other close relationship may be occurring [Moss et al., 143: 99 (2003)]. collected from multiple sites was later shown to contain five persistent strains of bacteria [Perez-Matos et al., 92(2): 155 (2007)]. Again, was found across all sites, and was the only identified bacteria present in both studies. An 11.2 kb contig was assembled from DNA sequencing reads and was found to match the 16S rRNA gene with 99% identity. The presence of this sequence in the pool is strong evidence that the ET743 biosynthetic pathway should be present within the metagenomic DNA.","Based on the similar biosynthetic origins between saframycin [Li et al., 190: 251-263 (2008)] (GenBank Accession Number DQ838002), saframycin Mx1 [Pospiech et al., 141:1793 (1995)] (GenBank Accession Number MXU24657), and safracin [Velasco et al., 56: 144 (2005)] (GenBank Accession Number AY061859), the identified biosynthetic genes in these pathways were identified from the total metagenomic DNA which was translated in all six reading frames and blastP was used to search against the identified biosynthetic genes from those pathways.  depicts the 19 kb assembled contig containing the 10 putative ET743 biosynthetic genes, nearest matches, and proposed functions. The contig was extended and confirmed by Sanger sequencing. The left-hand limit of the cluster is contemplated to be identified herein, as in the negative direction a conserved gene DNA polymerase III delta subunit is present as determined by four out of the top five hits from NCBI BLAST. Interestingly, three out of the top five are identified as coming from marine \u03b3-proteobacteria.","Field-collected tunicate samples of from the Florida Keys were confirmed to contain ET-743 and related metabolites using high-resolution, high-mass accuracy, liquid chromatography-Fourier transform ion cyclotron resonance mass spectrometry (LC-FTICR-MS). Known biosynthetic precursors were identified from the tunicate by extracted ion chromatograms at \u00b120 ppm, including the M+Hand (M\u2212H2O)+Hfor ET-743 (1), ET-597 (19), ET-594 (21) and ET-583 (18). Confirmation by LC-MS\/MS was performed on-line with FTICR-MS and an iontrap-mass spectrometer (IT-MS). Since all four compounds identified had previously been characterized by MS\/MS, assignment of product ions was straightforward as observed fragmentation was consistent between earlier studies using fast atom bombardment (FAB)-collision induced dissociation (CID) [Sakai et al., J. Am. Chem. Soc. 118: 9017-9023 (1996)], and work by the inventors with electrospray ionization (ESI)-(CID) on FTICR and IT instruments. The presence of both ET-743 and presumed precursors indicated that ET-743 biosynthesis occurred within the field-collected animal, and thus that the producing symbiont was present.","Briefly, tunicate samples were deproteinized with methanol (2:1 ratio, 1 hr at \u221220\u00b0 C.), the protein was removed by centrifugation (14,000 RCF\u00d7G), and the supernatant was concentrated 5-fold. Fifty microliters of this sample was analyzed on a Luna C18 100 \u212b 2\u00d7250 mm 5 \u03bcm column (Phenomenex). The following gradient was generated on an Agilent 1100 HPLC: 0 (98.2), 10 (98.2), 95 (2.98), 100 (2.98) and 105 (98.2). Values are given as Time (% A, % B). Time is given in minutes with the total run time being 120 minutes at a flow rate of 0.2 ml\/min. A column heater was operated at 50\u00b0 C. The flow was diverted for the first 10 minutes of the run. Buffer A consisted of 0.1% formic acid in DDI water and Buffer B consisted of 0.1% FA in acetonitrile.","FTICR MS was performed on an APEX-Q (Apollo II ion source, 7T magnet, Bruker Daltonics). Data were gathered by ESI in positive ion mode (2,400 V, m\/z 150-1,000, transient 128K, 1 scan\/spectrum) with external ion accumulation (0.33 s), dynamic trapping, and 1 ICR cell fill per spectrum. External calibration utilized HP-mix (Agilent). For FTICR MS\/MS experiments auto-MS\/MS was selected with Q-isolation (10 m\/z, 5 precursor ions, collision energy of \u221216 to \u221221 V). A peak list of possible ET-743 related metabolites was used for precursor ion selection. Data were processed in Data Analysis (Bruker Daltonics) and MS\/MS spectra were interpreted manually. Metabolite peaks were detected over multiple samples and runs. Iontrap-MS\/MS was performed with HPLC conditions as above, except with a Surveyor HPLC (ThermoFisher). An LTQ Deca XP Ion trap MS (ThermoFisher) was employed for data-dependent MS\/MS (1 precursor ion scan, 400-1800 m\/z, 7 MS\/MS events, isolation width 3 m\/z, normalized collision energy 35%). Data analysis was performed in Excalibur version 3.0 (Thermo) and MS\/MS spectra were interpreted manually.","454 and 16S rRNA gene library construction and sequencing methods. Metagenomic DNA was extracted from frozen samples using a DNeasy Tissue kit (Qiagen). DNA was used to prepare a 16S rRNA gene targeted amplicon library using primers TGCTGCCTCCCGTAGGAGT (SEQ ID NO: 51) and AGAGTTTGATCCTGGCTCAG (SEQ ID NO: 52) and a random shotgun 454 FLX library. Sequencing was performed on a Roche\/454 Life Sciences FLX Sequencer. Later, a second shotgun library was prepared using the 454 Titanium upgrade. Tunicate raw sequencing reads from the first FLX run were assembled using the 454 Newbler assembler (v2.0.00.20). The second 454 Titanium sequencing run was assembled together with the first sequencing run data producing a second assembly (Newbler v2.0.01.14).","NRPS module identification. Reads\/contigs were filtered by protein homology to the saframycin, saframycin Mx1, and safracin NRPS genes characterized in (DQ838002), (U24657), and (AY061859) using BLASTx\/tBLASTn searches. Primers were designed from the ends of filtered sequences (VectorNTI 9, Informax) and PCR reactions were designed based on the location of the BLAST hit on the reference sequences. Sequencing of positive reactions with linked sequences was performed with Sequencher 4.9, Gene Codes. Flanking sequence from high interest contigs was obtained by restriction-site PCR (RS-PCR) including two rounds of PCR using a semi-degenerate primer in conjunction with nested primers of known sequence [Ragin et al., Int. J. Cancer 110: 701-709 (2004)].","Analysis of the Metagenomic Population.","Classification of the raw reads and total assembly was performed with MG-RAST [Meyer et al., BMC Bioinform. 9: 386-386 (2008)]. Sequences were classified by protein homology to a manually curated database (the SEED). The 16S rRNA gene amplicon sequencing run was analyzed by assembling the raw reads with an identity threshold of 95%. The assembled contigs were submitted to the Ribosomal Database Project (RDP) for classification [Wang et al., Ap. Environ. Microbiol 73: 5261-5267 (2007)]. Multiple sequence alignment (MSA) for 16S rRNA gene sequences was performed by [DeSantis et al., Ap. Environ. Microbiol 72: 5069-5072 (2006); DeSantis et al., Nuc. Acids Res. 34: W394-399 (2006)]. Default parameters were used except for minimum length (300) and minimum % identity (50%). Formatting was changed to \u201cremove common alignment gap characters\u201d to provide an equal length MSA. The correct tree-building model was selected [Keane et al., BMC Evol. Biol. 6: 29-29 (2006)], and assembled using the maximum likelihood method with the HYK nucleotide substitution matrix and additional parameters selected by Modelgenerator (Phym1 v2.4.4) [Guindon et al., Sys. Biol. 52: 696-704 (2003)]. The cladeorgram was displayed (FigTree v1.3.1) using midpoint rooting and colored with clade annotation. Gene-finding was performed on the NRPS contig, 16S contig (contig00422), and random contigs from the total shotgun assembly (AMIgene with manual curation) [Bocs et al., Nucleic Acids Research 31: 3723-3726 (2003)]. Relative Synonymous Codon Usage (RSCU) analysis and CAI analysis was performed with codonW [Wang et al., Proc. Nat. Acad. Sci. USA 104: 7612-7616 (2007)]. Further phylogenetic classification of the NRPS contig and contig00422 was performed using the Na\u00efve Bayesian Classification Tool using the 3- and 6-mer setting [Hicks et al., ACS Chem. Biol. 1:93-102 (2006)].","Based on identification of ET-743 from field-collected tunicates, total hologenomic DNA from samples was prepared. This DNA was used to prepare a 16S rRNA gene amplicon library and a random shotgun fragment library for 454 based FLX pyrosequencing. Raw reads from the first shotgun sequencing run, and an assembly of these data were filtered using relatedness of the translated protein sequences to the saframycin and safracin non-ribosomal peptide synthetases (NRPSs) (MXU24657, DQ838002, AY061859) using BLASTx and tBLASTn. Linkage of these sequences was performed using a combination of traditional PCR and restriction-site PCR (RS-PCR) yielding six contigs of high interest containing NRPS domains for biosynthesis of ET-74331. A second sequencing run combined with the first generated another assembly of 839,923 reads with an average read length of 332 bp, bearing 77,754 total contigs, and 15,097 contigs larger than 500 bp. A 22 kb contig was identified that linked 4\/6 of the high interest contigs from the first assembly and extended this putative NRPS-containing contig to >35 kb using RS-PCR. This DNA fragment was PCR amplified and Sanger sequenced for confirmation. Twenty-six ET-743 biosynthetic genes were identified in this contig and annotated with proposed function using BLASTx against the NCBI NR database, Genbank HQ609499). The individual genes appear to be of bacterial origin, suggesting that the cluster is not derived from the tunicate genome. In addition to the 35 kb putative NRPS contig, sequences containing ribosomal RNA (rRNA) fragments were identified. One of these rRNA sequences was located in a large contig (contig00422) that was extended to >26 kb with RS-PCR. Contig00422 (Genbank HQ542106) contains a full 16S rRNA gene, which aligns (>99% identity) to the 16S rRNA gene reported previously for (AY054370) (DQ494516) [Moss et al., Mar. Biol. 143: 99-110 (2003); D'Agostino et al., Int. J. Gynecol. Cancer. 16: 71-76 (2006)].","Taxonomic classification of the raw reads and of the total assembly was performed using the Metagenomic Rapid Annotations with Subsystems pipeline (MG-RAST) [Meyer et al., BMC Bioinform. 9: 386-394 (2008)]. Results from both sets were consistent, with \u02dc40% of the classified sequences being of eukaryotic origin (mainly [sea squirt\/tunicate]) and the remaining 60% being largely proteobacterial sequence (>90%) of which there were two major populations: \u03b1-proteobacterial (largely 78-85%) and \u03b3-proteobacterial (10-17%). 16S rRNA gene amplicon sequencing runs identified 30 variants but only three significant ones (>1% of the total reads). The largest population of 16S rRNA gene reads was classified as (\u02dc78%), consistent with the classification of shotgun reads by MG-RAST. This 16S rRNA gene variant aligns to contig09113 from the shotgun sequence assembly, found previously in two of the three tunicate sampling sites from the Caribbean (clone 2j, DQ494507) [Parez-Matos et al., Antonie van Leeuwenhoek 92: 155-164 (2007)]. The second most abundant 16S rRNA gene variant is an unclassified \u03b3-proteobacterium (\u02dc19%) that aligns to contig00422 and represents in the sample. A third small population of 16S rRNA gene reads was identified as unclassified bacteria and corresponded to one read from the shotgun sequencing runs (also identified previously) [Moss et al., Mar. Biol. 143: 99-110 (2003)]. These three variants account for >97% of the 16S rRNA gene sequencing reads. None of these three strains form a close phylogenetic relationship with , or , producers of the three tetrahydroisoquinoline antibiotics whose pathways have been previously characterized.","The putative ET-743 35 kb biosynthetic gene cluster was then linked to the 16s contig00422 by evaluating the codon usage bias. Bacteria typically do not employ synonymous codons equally and this can be exploited as a unique marker [Sharp et al., ISME J. 1: 693-702 (2007)]. A Relative Synonymous Codon Usage (RSCU) analysis was performed using the annotated NRPS contig and contig00422 as well as ORFs identified in several contigs chosen at random. The RSCU score is the observed frequency of a codon divided by the frequency expected for equal usage of all synonymous codons, thereby making it a measure of non-randomness [Sharp et al., ISME J. 1: 693-702 (2007)]. RSCU scores for each codon were similar between the genes on the contig bearing the presumed NRPS biosynthetic genes and the 16S rRNA gene-containing contig00422, but varied compared to RSCU scores from genes located in the random contigs from the total assembly. The extremely low GC content of the contig bearing the putative ET-743 NRPS genes (\u02dc23%) closely matches the GC content (26%) of the contig bearing the 16S rRNA gene corresponding to , providing another strong marker of genetic linkage. On the other hand, appear to have uniformly high GC content (54%-70%) according to current whole genome sequencing data, indicating that the contig containing NRPS genes is unlikely to be linked to this organism. The only fully sequenced and annotated tunicate genome, , is 35% GC.(NZ_AABS00000000) To account for GC bias in codon usage random genes from the low GC bacterium (\u02dc29%) str. Okra were included. A comparison of the mean RSCU values for each codon revealed that only 12\/60 values differed significantly (p<0.05) between the putative ET-743 NRPS and contig00422 genes while 18\/60 differed between the putative NRPS genes and random genes from . The significant differences between genes are most evident in the codons encoding isoleucine (AUU, AUC, AUA), lysine (AAA, AAG), aspartic acid (GAU, GAC), glutamic acid (GAA, GAG) and arginine (CGU, CGC, CGA, CGG, AGA, AGG). 49\/60 codons differed significantly between the putative NRPS genes and random tunicate metagenome genes. In addition to RSCU analysis the contig containing the 26 predicted ET-743 pathway genes was used in a correspondence analysis using codonW to generate a codon adaptive index (CAI). This index was then used as a reference for comparison with the same genes used in the RSCU analysis. Although all CAI scores differed significantly from the NRPS contig CAI score (p<0.05), the CAI score and random gene CAI scores differed to a larger degree. The contig bearing the NRPS genes and contig00422 was also analyzed with the Na\u00efve Bayesian Classifier (NBC) tool, a composition-based metagenome fragment classifier that uses N-mer frequency profiles [Yamamoto et al., J. Bacteriol. 190: 1308-1316 (2008)]. NBC analysis based on 3- and 6-mer profiles resulted in high confidence classification of both contigs as \u03b3-proteobacteria\/Enterobacteriaceae. This same 16S rRNA gene sequence has now been linked to collections from the Mediterranean, Caribbean, and Florida Keys. Taken together, the sequence contig bearing NRPS module genes are derived from the same organism as contig00422 ().","Biochemical Confirmation of a Key Enzymatic Activity.","Briefly, the biochemical reaction of compound (26) to (27) was performed as described by Koketsu [Koketsu et al., Nat. Chem. Biol 6: 408-411]. Reactions took place in reaction buffer with either no enzyme, EtuA RE-domain, or SfmC. Cofactors were then added from concentrated stocks. Compound (26) was then added in DMF followed by incubation overnight at room temperature. LC-FTICR MS was then run to monitor the reaction products.","Specifically, the transformation of thioester-bound acylated-depsipeptide (10) to the aldehydo-didepsipeptide (11) is a key enzymatic step thought to be catalyzed by the EtuA2 RE domain. Therefore, the excised EtuA2 RE domain was cloned and overexpressed to test this activity To make the SfmC over-expression construct, the sfmC gene was amplified using genomic DNA from NRRL 11002 as template and SfmC_F (5\u2032-GCAGAATTCGTGACCCGGCACGAGCC-3\u2032, NdeI site underlined (SEQ ID NO: 53) and SfmC_R (5\u2032-TTTGGATCCTCATCGCTCCTCCTCCAGCGTGC-3\u2032, HindIII site underlined SEQ ID NO: 54) as primers. The PCR product was digested with NdeI and HindIII and cloned to the same sites of pET-28a to generate pET28a-sfmC. PfuTurbo\u00ae DNA Polymerase (Stratagene) was used in sfmC cloning.","To make the over-expression construct for the RE domain of EtuA3, the RE coding sequence (1,251 bp) was amplified via PCR using the metagenomic DNA mixture as template and EtA3RE_F (5\u2032-GCAGAATTCACCTTGCAAAAAGAAGGAATTG-3\u2032, NdeI site underlined (SEQ ID NO: 55) and EtA3RE_R (5\u2032-CGCGGATCTTATATTTTTTTCGGATGAGGAAAG-3\u2032, XhoI site underlined (SEQ ID NO: 56) as primers, digested with NdeI and XhoI, and further cloned to the same sites on pET-28a to generate pET28a-RE. KOD DNA Polymerase (Novagen) was used in the cloning of the EtuA3 RE domain.","The N-His-tagged RE domain protein and the N-His-tagged SfmC protein expression constructs were separately transformed into BL21 (DE3)+pRare. The two strains were grown at 37\u00b0 C. in 0.5 L TB medium to an ODof approximately 0.8 in 2 L flasks. The cultures were cooled to 18\u00b0 C., and isopropyl \u03b2-D-thiogalactopyranoside was added to a final concentration of 0.2 mM and grown 12-16 hours with shaking. The cells were harvested by centrifugation and frozen at \u221280\u00b0 C. Cell pellets were thawed to 4\u00b0 C. and resuspended in 5\u00d7 volume of lysis buffer (20 mM HEPES, pH 7.8, 300 mM NaCl, 20 mM imidazole, 1 mM Tris(2-carboxyethyl) phosphine (TCEP), approximately 20 mg CelLytic Express (Sigma-Aldrich)) before lysis via sonication. Centrifugation at 40,000\u00d7g for 30 min provided clear lysates. Proteins were purified using affinity chromatography with Nickel-NTA resin (Qiagen). Briefly, after filtration of the supernatant through 0.45 \u03bcm membrane, the solution was loaded onto a 1 mL gravity flow column. The column was washed with 10 column volumes of wash buffer (20 mM HEPES, pH 7.8, 300 mM NaCl, 50 mM imidazole, 1.0 mM TCEP, 10% glycerol) and eluted with 20 mM HEPES, pH 7.8, 300 mM NaCl, 400 mM imidazole, 1.0 mM TCEP, 10% glycerol. Fractions were pooled, concentrated, and loaded onto a PD10-desalting column (GE Healthcare Life Sciences) equilibrated with storage buffer (20 mM HEPES, pH 7.4, 150 mM NaCl, 1.0 mM TCEP, 20% glycerol). Fractions were combined, concentrated, frozen, and stored at \u221280\u00b0 C. Protein concentrations were calculated using A280 and predicted protein extinction coefficients. Proteins were approximately 80% and 95% pure by SDS-PAGE with yields of 1 mg\/L for EtuA2 RE and 3 mg\/L for SfmC.","Koketsu and coworkers had shown the same activity for the matching saframycin substrate analogs (25\u219226) with the SfmC A-T-RE-tridomain [Koketsu et al., Nat. Chem. Biol 6: 408-411]. Therefore, the known saframycin substrate analog (25) was synthesized and transformed to the previously characterized saframycin aldehydo-dipeptide (26) by the EtuA2 RE domain. As a positive control, substrate (25) was converted with high efficiency to (26) by purified apo-SfmC (C-A-T-RE). The differential activity was expected as (25), while clearly an acceptable substrate, is missing the cysteine-derived thiol and has a glycine in place of the glycolic acid compared to the native substrate (10). Experimental data was in agreement with a synthetic authentic standard aldehydo-dipeptide (27). Confirmation of RE enzyme activity links the predicted biochemical scheme to demonstrated function in the ET-743 biosynthetic pathway.","Metaproteomics to Identify ET-743 Biosynthetic Proteins.","Briefly, tunicate protein samples were precipitated with acetone then resolubilized, reduced, alkylated, diluted, and digested with trypsin. The sample was fractionated over 40 minutes into 20 fractions using SCX chromatography. The 20 peptide fractions were analyzed once on an LTQ-Orbitrap XL interfaced with a nanoLC 2D system. Peptides were separated on a capillary column in-house packed with C18 resin after loading on a C18 trap column. LC eluent was introduced into the instrument via a chip-based nanoelectrospray source in positive ion mode. The LTQ-orbitrap was operated in data-dependent mode. The 20 peptide fractions were also analyzed in duplicate on a Solarix 12T hybrid Q-FTICR interfaced with a nanoLC system. Peptides were separated on a capillary column packed in-house with C18 resin after loading on a C18 trap column. The FTICR operated in data-dependent mode. All scans were collected in the profile mode and peak picking was performed from the profile mode spectra. Bioinformatics analysis was performed with the Transproteomic Pipeline and all assigned Etu peptides were validated by comparison with synthetic peptide standards.","Specifically, total tryptic peptides from the field-collected sample were fractionated by strong-cation-exchange chromatography, desalted, and then analyzed by reverse phase nano-LC MS\/MS. Datasets were collected on LTQ-Orbitrap and 12T Q-FTICR mass spectrometers, with high-resolution\/mass-accuracy MS 1 spectra (and MS2 for FTICR only). Data were processed in Trans Proteomic Pipeline49 with four distinct search engines (X!tandem, OMSSA, Inspect, and Spectrast) and the peptide and protein prophet probability models with false discovery rates at the protein level of 0.6-0.9%. The database searched consisted of a six-frame translation of the total metagenome assembly filtered to contain all possible polypeptides >60 amino acids in length (SI). Sequence length-based cutoffs were utilized rather than ORF prediction due to the short length of many metagenomic contigs derived from the 454 metagenomic sequencing. Filtering resulted in a six-fold reduction in total sequence length versus the unfiltered six-frame translation. A 60 amino acid cut-off represents a 0.2% chance of any random sequence producing a translation without a stop codon appearing. Based upon 23S\/16S analysis the closest fully sequenced organisms to the four principle constituents of the assemblage were included to assign homologous proteins derived from genes that may have been incompletely sequenced in the metagenomic analysis (tunicate: NZ_AABS00000000, \u03b1-proteobacteria: DSS-3 NC 003911, \u03b3-proteobacteria: RSA 331 NC010115, unknown bacteria: subsp. SC str. PG1 NC005364). Reversed sequences for all proteins were included as decoys in the search database.","A total of 289 proteins were identified at a probability >95% from interprophet pooled analysis of all four search engines prior to Protein Prophet analysis. Three of the proteins identified were derived from the Etu pathway with two identified by Orbitrap and one by FTICR and Orbitrap MS. The penicillin acylase EtuF3 was identified with two unique peptides, 3+ TIQHEIELSDIGPIINNLIQEN115NQINKK (N115=deamidated) and 2+ RPIELR, and the protein was identified in 3\/4 search engines providing a total protein probability of 99.99%. The bacterial symbiont protein EtuR1 was identified with two unique peptides, 2+ GSNIHYDLENDHNDYEK and 3+ GSNIHYDLENDHNDYEK, identified by 3\/4 search engines at the protein level with a combined protein probability of 100.00%. The EtuM1 SAM dependent methyltransferase was identified by one unique peptide, 2+ LLDVGGGTAINAIALAK and 2\/4 search engines at the protein level with a probability of 99.16%. All identified Etu peptides were validated by comparison with synthetic peptide standards by LC elution time (\u00b12 minutes on the same nano-LC system), and MS\/MS fragmentation spectra. Detailed spectral information was obtained. These three biosynthetic proteins identified with high probabilities and confidence levels by multiple search algorithms and comparison with authentic standards strongly suggests that ET-743 biosynthetic genes are expressed in the tunicate microbial symbiont assemblage.","The overall constitution of the collected assemblage proteome was also characterized using a BlastP analysis of metagenomic contigs, restricted to tunicate and bacterial sequences. The 289 proteins identified represent the minimum number of distinct DNA sequences that fully represent the assigned dataset, however as many as 391 distinct DNA sequences can be assigned to this same set of peptides (e.g., multiple copy genes, homologous genes, shared peptides). Of the maximum possible 391 proteins 283 were tunicate derived, 91 were bacterial derived, and 17 produced no significant similarity. Of the 91 bacterial proteins identified 71 appeared to be of proteobacterial origin and 20 could not be assigned at the family level. Of the 71 proteobacterial proteins identified 24 were \u03b1-proteobacterial, 26 were \u03b3-proteobacterial origin, and 21 could not be assigned beyond proteobacterial. Apparent distribution of predominant species (tunicate, \u03b1-proteobacterial, \u03b3-proteobacterial) roughly correlates with metagenomics\/16S rRNA genes with higher amounts of tunicate proteins observed than would be expected from a direct correlation of DNA and protein levels. More \u03b3-proteobacterial proteins were observed compared to \u03b1-proteobacterial assuming DNA is proportional to expression level.","The inability to culture the vast majority of bacterial and fungal symbionts (outside of their natural host or environmental niche) that produce secondary metabolites have limited access to a huge genetic diversity relating to untapped chemical resources for therapeutic and other industrial applications. This includes complex marine (e.g., sponge, tunicates, dinoflagellates) and terrestrial (e.g., plant-microbe, biofilm, insect-gut, human-gut) microbial consortia where the presence of large populations of diverse microorganisms and their corresponding genomes that bear natural product gene clusters remain unexplored. This new source of metabolic and chemical diversity will lead to important new basic knowledge, and also contribute to ongoing drug discovery efforts against many disease indications.","In these studies, several steps were taken to obtain evidence for identification of the ET-743 biosynthetic pathway and the corresponding producing microbial symbiont. First, the presence of the ET-743 natural product and intermediates were used as markers for the producing bacterium in the tunicate\/microbial consortium. Second, codon usage similarity between the biosynthetic gene cluster and a contig containing a 16S rRNA gene sequence support as the bacterial producer of ET-743. Direct functional analysis of a key biosynthetic enzyme confirmed its predicted catalytic assignment in the pathway. Finally, symbiont-derived expression of three ET-743 biosynthetic enzymes was confirmed by metaproteomic and bioinformatic analysis, enabling the direct correlation between natural product, the Etu gene cluster, and predicted biosynthetic proteins. This tiered strategy provides a general approach for future efforts to characterize orphan and target natural product biosynthetic systems from complex marine and terrestrial microbial assemblages including animal-microbe symbiont consortia, and dinoflagellates.","While the present invention has been described in terms of various embodiments and examples, it is understood that variations and improvements will occur to those skilled in the art. Therefore, only such limitations as appear in the claims should be placed on the invention."],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0030","num":"0029"},"figref":"FIG. 1","i":["Streptomyces lavendulae","Myxococcus xanthus","Pseudomonas fluorescens"]},{"@attributes":{"id":"p-0031","num":"0030"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0032","num":"0031"},"figref":"FIG. 3"}]},"DETDESC":[{},{}]}
