---
title: Membrane-integrated viral DNA-packaging motor protein connector biosensor for DNA sequencing and other uses
abstract: Compositions and methods are disclosed that exploit the unprecedented modification of double-stranded DNA virus DNA-packaging motor protein connector polypeptides to render them capable of stable incorporation into lipid membranes as a self-assembled homodocamer that forms an aperture through which conductance can occur when an electrical potential is applied across the membrane. The aperture permits use of the modified protein as a biosensor, for dsDNA sequencing, SNP detection and highly sensitive affinity capture and fingerprinting of analytes, and also finds use in electropotential-driven solute translocation, such as for liposomal loading to form therapeutic nanoparticles (e.g., gene delivery) and bioreactors, and for other uses. The aperture can further be used in optical detection of dsDNA or other acceptor labeled analytes in a fluorophore donor labeled single pore channel.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09234238&OS=09234238&RS=09234238
owner: 
number: 09234238
owner_city: 
owner_country: 
publication_date: 20110426
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATION","STATEMENT OF GOVERNMENT INTEREST","STATEMENT REGARDING SEQUENCE LISTING","BACKGROUND","BRIEF SUMMARY","DETAILED DESCRIPTION","EXAMPLES","Example 1","Assembly of Conductive Channel-Containing Membrane Using Isolated Viral DNA-Packaging Motor Connector Protein","Example 2","Incorporation of Isolated Viral DNA-Packaging Motor Connector Protein into Liposomes to Form Conductive Channel-Containing Membrane","Example 3","Re-Engineering of dsDNA Viral DNA-Packaging Motor Protein Connectors for Nucleic Acid Translocation","Example 4","Modified Membrane-Incorporated Viral DNA-Packaging Motor Protein Connector Conductive Channel as a Stochastic Sensing Machine for Combined Analyte Capture and Fingerprinting in Single Molecule Analysis","Example 5","Sensing and Fingerprinting of Double-Stranded DNA ","Example 6","Ions as Analytes","Example 7","DNA as Analyte","Example 8","Specifically Binding Polypeptide (Antibody) as Analyte","Example 9","Fluorescence Resonance Energy Transfer (FRET) for DNA Sequencing"],"p":["This application is a continuation in part of international patent application PCT\/US09\/62826, which was filed on Oct. 30, 2009, which claims the benefit of U.S. Provisional Application Ser. No. 61\/109,669, filed Oct. 30, 2008.","This invention was made with government support under Grant No. PN2 EY 018230, awarded by the NIH Nanomedicine Development Center\/NIH Roadmap for Medical Research\/National Institutes of Health, and Grants No. R01-GM59944, EB 03730 awarded by the National Institutes of Health. The government has certain rights in this invention.","The Sequence Listing associated with this application is provided in text format in lieu of a paper copy, and is hereby incorporated by reference into the specification entirely. The name of the text file containing the Sequence Listing is CIP_PXG_SEQUENCE_LISTING.txt. The text file is 102 KB, was created on Apr. 15, 2011, and is being submitted electronically via EFS-Web to the U.S. Patent Office, concurrent with the filing of the specification.","The Sequence Listing text file submitted herein does not include any new matter which goes beyond the disclosure of the application as filed.","1. Technical Field","The present invention embodiments relate generally to the field of biosensors. More specifically, the compositions and methods described herein relate to an engineered viral DNA-packaging motor protein connector that can be incorporated into a lipid membrane to form an electroconductive aperture (or to form a fluorescence excitation aperture), for use in DNA sequencing and other applications.","2. Description of the Related Art","Highly sensitive detection and characterization of minute quantities of chemicals and biochemicals represent desirable goals of modern analytical technologies. Robust molecular sensing devices would find uses in a wide range of biomedical, industrial, environmental, forensic, security and other contexts, for example, in the detection and identification of pathogens and chemicals at extremely low concentrations for disease diagnosis and environmental monitoring, in high throughput DNA sequencing and other genomics applications, and elsewhere.","Analytical methodologies have been described that employ intermolecular affinity binding interactions, typically non-covalent in nature, to detect binding or \u201ccapture\u201d of an analyte of interest by a specific affinity ligand, for instance, including detection of bacterial, viral, parasitic or other microbial pathogens or pathogen-associated antigens, and detection of antibodies, cancer markers, and other analytes (e.g., Kittigul et al., 1998 September; 59(3):352-6; Cordiano et al., 1995 Jan. 13; 178(1):121-30; Olson et al., 1990 Nov. 6; 134(1):71-9; Nerurkar et al., 1984 July; 20(1):109-14; Jia et al., 2009 October; 161(1):38-43; He et al., 2007 May; 14(5):617-23; Xu et al., 2006 August; 44(8):2872-8; Che et al., 2004 June; 42(6):2629-35; Hunt et al., Brown et al., 2001 September; 65(3):208-13; Loa et al., 2000 July-September; 44(3):498-506; Lubenko et al., 2000 September; 10(3):213-8; Chanteau et al., 2000 April; 4(4):377-83; Brinker et al., 1998 April; 36(4):1064-9; Vyse et al., 1997 January; 63(1-2):93-101; Peterson et al., 1997 January; 35(1):208-12; Lairmore et al., 1993 June; 9(6):565-71; Heller et al., 1993 October; 37(1-2):127-33; van Loon et al., 1992 February; 108(1):165-74; Wolf-Rogers et al., 1990 Oct. 19; 133(2):191-8; Barsoum et al., 1990 July; 71(1):107-13; Hierholzer et al., 1989 June; 27(6):1243-9; Hurley et al., 1986; 7(4):309-36; Wolff et al., 53:2560-65 (1993); see generally, e.g., Harlow and Lane, , Cold Spring Harbor Laboratory, 1988; Weir, D. M., 1986, Blackwell Scientific, Boston, Mass.).","Beyond detection of the presence of an analyte following its involvement in an affinity binding interaction, sophisticated technologies are emerging that permit characterization of the analyte, often by comparing a single- or multiparameter physicochemical profile of the analyte to type-characteristic profiles generated using one or more known reference standards, and hence referred to as \u201cfingerprinting\u201d techniques. (e.g., Li et al., 2009 23(22):3533-3542; Ali et al., 2009; Leski et al., 18, 2009; Weinkopff et al., 18, 2009; Song et al., 2009 9(11):3090-9; Ortea et al., 2009 57(13):5665-72; Amini, 2009(1):11-6; Shi et al., 2009 32(1):142-6; Sun et al., 2009 1216(5):830-6; Yin et al., 2003 93(8):1006-13; Roy et al., 2008 14(20):6610-7; Pei et al., 2008 33(14):1662-8; Arthur, 2008, 141:257-70; Zhao et al., 2008 26(1):43-9; Woo et al., . 2008 80(7):2419-25; Damodaran et al., 2007 5(3-4):152-7; Fellstr\u00f6m et al., 2008 72(2):133-40; Song et al., 2006 1:4556-9; De Vuyst et al., 2008 125(1):79-90).","The use of transmembrane channels has been demonstrated in stochastic analyte detection (Bayley et al., 2001 13:225-230), an electrochemical approach relying on the real-time observation of individual binding events between single substrate molecules and a receptor, as evidenced by altered (e.g., decreased or increased in a statistically significant manner) electrical conductance by the channel (receptor) as a result of substrate (analyte) binding. A wide range of processes, such as the transport of DNA, RNA, pharmaceutical agents, peptides, proteins, and polymers, have been studied by such approaches, for example, using electrophysiological measurements of individual membrane channels (Thieffry et al., 1988 7:1449; Hinnah et al., 2002 83:899; Alcayaga et al., 1992 . 311:246-50; Benz et al., 1986 165:978; Movileanu et al., 2000 18:1091).","For instance, the transient blockade of ionic current through the alpha-hemolysin (\u03b1-HL) channel, a bacterial transmembrane pore-forming protein, has been used to measure the length of single-stranded DNA or RNA (Kasianowicz et al. 93, 13770-13773 (1996)). Subsequently, DNA hairpin molecules have been used to decelerate the DNA translocation rate through the alpha-hemolysin (\u03b1-HL) pore, to demonstrate the ability of a transmembrane ion channel to discriminate between single nucleotide polymorphisms (Vercoutere et al., 2001 19:248). Detection of base pair stacking and strand orientation within the pore have also been investigated (Vercoutere et al., 2003 31:1311; Howorka et al., 2001 19:636; deGuzman et al., 2006 34:6425). The channel of \u03b1-HL with a covalently attached adapter molecule has been shown to discriminate the nucleotides A, T, G, and C (Clarke et al., 2009 4:265).","Other protein channels that have been investigated include alamethicin for detection of polyethylene glycol (Bezrukov, 2000 174:1-13), and the reengineered MspA protein from for translocation of ssDNA (Butler et al., 2008 105:20647). Most studies involving nucleic acid transport through nanopores have focused on \u03b1-HL. However, the limiting lumen diameter of \u03b1-HL (1.5 nm) and other channels has restricted their DNA and RNA applications to translocation of single-stranded nucleic acid (Song, 1996 274:1859). A similar limitation was also reported for the MspA nanopore (Butler et al., 2008).","In a small number of other membrane pore systems, evidence of double-stranded DNA (dsDNA) transport across the membrane has been presented (Szabo et al., 2002 12:127; Mobasheri et al. 2002 31:389; Carneiro et al., 2003 1612:144), but these systems are not robust and represent poor candidates for widespread use such as biomedical applications, due to their undesirable voltage gating properties and the associated signal fluctuation. For this reason, their potential is considered limited and researchers have switched instead to fabricating synthetic metal or silicon nanopores for potential use in DNA sequencing (Smeets et al, 2006 6:89; Wang et al., 2001 19:622; Iqbal et al., 2007 2:243). Such synthetic nanopores, however, suffer from shortcomings due to difficulties in reliably producing replicated structures having consistent properties from batch to batch, and also lack versatility with regard to the ability to engineer modifications to pore structures and\/or to serve as substrates for modification by a wide range of chemical conjugation. As a result, the search for superior alternatives to currently available protein nanopores is still ongoing.","Clearly there is a need for improved compositions and methods that would provide a versatile membrane conductive channel platform for sensitively detecting and characterizing a wide range of analytes, having a lumen capable of accommodating dsDNA, that can be reliably and reproducibly assembled, that is not susceptible to voltage gating under working conditions, and that can be readily modified to feature a wide variety of specific affinity receptors for use in the detection and characterization of different analytes. The presently disclosed invention embodiments fulfill such a need, and offer other related advantages.","With regards to optical detection based DNA sequence or other applications within membrane integrated biosensors, current procedures also have their limitations. For example, batch DNA sequencing using fluorescent dyes have been developed by different labs (Shendure J A, et al 2008 January; Chapter 7: Unit 7.1; Bayley H. 2006 December; 10(6):628-37; Korlach J, et al 2008 September; 27(9):1072-83. PMCID:PMC2582155; Soni G V, Meller A. 2007 November; 53(11):1996-2001). The traditional optical detection for DNA sequencing normally uses fluorescent 2\u2032,3\u2032-dedeoxy nucleotide as the chain terminator; therefore, the traditional approach is limited by the number of base pairs one procedure can achieve.","An improved composition and methods that would provide a membrane integrated reaction pore for high throughput, optical detection based application is in need. This disclosure further presents embodiments that fulfill the concept of a single pore based optical biosensor system.","The present invention provides, in certain embodiments a conductive channel-containing membrane, comprising (a) a membrane layer; and (b) an isolated viral DNA-packaging motor connector protein that is incorporated into the membrane layer to form an aperture through which conductance can occur when an electrical potential is applied across the membrane. In certain embodiments the viral DNA-packaging motor connector protein is artificial. In certain embodiments the viral DNA-packaging motor connector protein comprises a homododecamer of viral DNA-packaging motor connector protein polypeptide subunits, wherein each of said subunits comprises (a) an aperture domain that comprises an isolated viral connector protein polypeptide having an amino terminus and a carboxy terminus; and (b) either or both of (i) at least one flexibility domain that comprises a polypeptide of 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17 or 18 contiguous uncharged amino acids and that is fused to at least one of the amino terminus and the carboxy terminus of (a), and (ii) at least one affinity\/alignment domain. In certain embodiments the viral DNA-packaging motor connector protein comprises a homododecamer of viral DNA-packaging motor connecting protein polypeptide subunits, wherein each of said subunits comprises (a) an aperture domain that comprises an isolated viral connector protein polypeptide having an amino terminus and a carboxy terminus; and (b) either or both of (i) at least one flexibility domain that comprises a polypeptide of 4, 5, 6, 7, 8, 9, 10, 11, 12, 13 or 14 contiguous uncharged amino acids and that is fused to at least one of the amino terminus and the carboxy terminus of (a), and (ii) at least one affinity\/alignment domain that comprises a polypeptide of formula X-X-X-X-X-X-X-Xand that is fused to the flexibility domain, wherein each Xis independently either any uncharged amino acid or no amino acid, each Xis independently any uncharged amino acid, Xis a positively charged amino acid selected from lysine, arginine and histidine, Xis a negatively charged amino acid selected from glutamic acid and aspartic acid, and Xis a positively charged amino acid selected from lysine, arginine and histidine.","In certain embodiments the viral DNA-packaging motor connector protein comprises a homododecamer of viral DNA-packaging motor connector protein polypeptide subunits, wherein each of said subunits comprises (a) an aperture domain that comprises an isolated viral connector protein polypeptide having an amino terminus and a carboxy terminus; and (b) either or both of (i) at least one flexibility domain that comprises a polypeptide of sequence Gly-Gly-Gly-Gly-Gly-Gly as set forth in SEQ ID NO:23 and that is fused to the carboxy terminus of (a), and (ii) at least one affinity\/alignment domain that is fused to the flexibility domain. In certain embodiments the affinity\/alignment domain comprises a polypeptide that is selected from (i) a Strep-II tag as set forth in SEQ ID NO:22 [WSHPQRFEK], (ii) a polyhistidine polypeptide tag of 3, 4, 5, 6, 7, 8, 9, 10, 11 or 12 contiguous histidine residues, (iii) a polyarginine polypeptide of 3, 4, 5, 6, 7, 8, 9, 10, 11 or 12 contiguous arginine residues, (iv) an HIV Tat polypeptide of sequence YGRKKRRQRR [SEQ ID NO:39], and (v) a peptide tag of sequence DRATPY [SEQ ID NO:40]. In certain embodiments the viral DNA-packaging motor connector protein comprises a homododecamer of viral DNA-packaging motor connector protein polypeptide subunits, wherein each of said subunits is selected from (i) C-His6-gp10\/K234A as set forth in SEQ ID NO:41, (ii) C-His6-gp10\/K234C as set forth in SEQ ID NO:42, (iii) C-His6-gp10\/CS\/CS\/K234C as set forth in SEQ ID NO:43, (iv) \u03941-14\/gp10-Strep-II as set forth in SEQ ID NO:44, and (v) Gp10\/\u0394285-309-Strep-II as set forth I SEQ ID NO:45.","In certain embodiments the viral DNA-packaging motor connector protein comprises a homododecamer of viral DNA-packaging motor connector protein polypeptide subunits that each comprise a polypeptide selected from (i) all or a transmembrane aperture-forming portion of bacteriophage phi29 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:1 [Acc. No. ACE96033] (ii) all or a transmembrane aperture-forming portion of phage T4 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:7 [Acc. No. NP049782], (iii) all or a transmembrane aperture-forming portion of phage lambda DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in any one of SEQ ID NOS:8-11 [Acc. Nos. gi 549295, gi 6723246, gi 15837315, gi 16764273], (iv) all or a transmembrane aperture-forming portion of phage SPP1 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:12 [Acc. No. P54309], (v) all or a transmembrane aperture-forming portion of phage P22 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:13 [Acc. No. AAA72961], (vi) all or a transmembrane aperture-forming portion of phage P2 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:14 [Acc. No. NP046757], (vii) all or a transmembrane aperture-forming portion of phage P3 DNA-packaging motor connector protein polypeptide, (viii) all or a transmembrane aperture-forming portion of phage T3 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:15 [Acc. No. CAA35152], (ix) all or a transmembrane aperture-forming portion of phage T5 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NOS:16-19 (Accession numbers AAX12078, YP006980; AAS77191; AAU05287), and (x) all or a transmembrane aperture-forming portion of phage T7 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:20 [Acc. No. NP041995].","In certain embodiments the viral DNA-packaging motor connector protein comprises a homododecamer of viral DNA-packaging motor connector protein polypeptide subunits that each comprise a polypeptide that comprises all or a transmembrane aperture-forming portion of a double-stranded DNA bacteriophage DNA-packaging motor connector protein. In certain embodiments the viral connector protein polypeptide is selected from (i) all or a transmembrane aperture-forming portion of bacteriophage phi29 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:1 [Acc. No. ACE96033] (ii) all or a transmembrane aperture-forming portion of phage T4 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:7 [Acc. No. NP049782], (iii) all or a transmembrane aperture-forming portion of phage lambda DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in any one of SEQ ID NOS:8-11 [Acc. Nos. gi 549295, gi 6723246, gi 15837315, gi 16764273], (iv) all or a transmembrane aperture-forming portion of phage SPP1 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:12 [Acc. No. P54309], (v) all or a transmembrane aperture-forming portion of phage P22 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:13 [Acc. No. AAA72961], (vi) all or a transmembrane aperture-forming portion of phage P2 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:14 [Acc. No. NP046757], (vii) all or a transmembrane aperture-forming portion of phage P3 DNA-packaging motor connector protein polypeptide, (viii) all or a transmembrane aperture-forming portion of phage T3 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:15 [Acc. No. CAA35152], (ix) all or a transmembrane aperture-forming portion of phage T5 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NOS:16-19 (Accession numbers AAX12078, YP006980; AAS77191; AAU05287), and (x) all or a transmembrane aperture-forming portion of phage T7 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:20 [Acc. No. NP041995].","In certain embodiments the viral DNA-packaging motor connector protein comprises a homododecamer of viral DNA-packaging motor connector protein polypeptide subunits that each comprise a polypeptide that comprises all or a transmembrane aperture-forming portion of bacteriophage phi29 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:1 [Acc. No. ACE96033]. In certain embodiments the viral connector protein polypeptide comprises all or a transmembrane aperture-forming portion of bacteriophage phi29 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:1 [Acc. No. ACE96033]. In certain embodiments the viral DNA-packaging motor connector protein comprises a detectable label, which may in certain further embodiments be selected from a colorimetric indicator, a GCMS tag compound, a fluorescent indicator, a luminescent indicator, a phosphorescent indicator, a radiometric indicator, a dye, an enzyme, a substrate of an enzyme, an energy transfer molecule, a quantum dot, a metal particle and an affinity label. In certain further embodiments the affinity label is selected from avidin, streptavidin, biotin, an aptamer, an antibody, a lectin, an oligosaccharide, a nucleic acid, an enzyme, a metal ion-binding polypeptide, a Strep-II tag as set forth in SEQ ID NO:22 [WSHPQRFEK], a polyhistidine polypeptide tag of 3, 4, 5, 6, 7, 8, 9, 10, 11 or 12 contiguous histidine residues, a Strep-I tag, a FLAG\u00ae peptide tag, a Myc peptide tag, glutathione-S-transferase, maltose binding protein, protein A, protein G, HIV Tat polypeptide [SEQ ID NO:39], a peptide having the amino acid sequence DRATPY [SEQ ID NO:40], glutaredoxin-2, and a phage-displayed peptide that specifically binds an affinity ligand. In certain further embodiments the antibody is selected from an intact immunoglobulin, a single-chain antibody, an scFv, a Fab and a (Fab)\u2032.","In certain embodiments relating to the above described conductive channel-containing membrane, the membrane layer comprises a lipid layer. In a further embodiment the lipid layer comprises amphipathic lipids, which in certain still further embodiments comprise phospholipids and the lipid layer comprises a lipid bilayer. In certain other embodiments the lipid layer is selected from a planar membrane layer and a liposome. In certain embodiments the liposome is selected from a multilamellar liposome and a unilamellar liposome. In certain other embodiments the incorporated viral DNA-packaging motor connector protein is mobile in the membrane layer. In certain other embodiments the conductive channel-containing membrane is capable of translocating double-stranded DNA through the aperture when the electrical potential is applied. In certain embodiments conductance occurs without voltage gating when the electrical potential is applied.","According to certain other embodiments of the present invention, there is provided a method of making a conductive channel-containing membrane, comprising (a) preparing dried amphipathic lipids on a solid substrate by contacting a first solution comprising amphipathic lipids and an organic solvent with the solid substrate and substantially removing the solvent; and (b) resuspending the dried amphipathic lipids in a second solution that comprises an aqueous solvent, an osmotic agent and a plurality of isolated viral DNA-packaging motor connector protein subunit polypeptides that are capable of self-assembly into a homododecameric viral DNA-packaging motor connector protein, to obtain a membrane that comprises a lipid bilayer in which is incorporated the viral DNA-packaging motor connector protein under conditions and for a time sufficient for said connector protein to form an aperture through which conductance can occur when an electrical potential is applied across the membrane, and thereby making a conductive channel-containing membrane.","In certain other embodiments there is provided a method of making a conductive channel-containing membrane, comprising (a) substantially removing solvent from a mixture comprising amphipathic lipids and at least one solvent, to obtain dried amphipathic lipids; and (b) resuspending the dried amphipathic lipids in a second solution that comprises an aqueous solvent, an osmotic agent and a plurality of isolated viral DNA-packaging motor connector protein subunit polypeptides that are capable of self-assembly into a homododecameric viral DNA-packaging motor connector protein, to obtain a membrane that comprises a lipid bilayer in which is incorporated the viral DNA-packaging motor connector protein under conditions and for a time sufficient for said connector protein to form an aperture through which conductance can occur when an electrical potential is applied across the membrane, and thereby making a conductive channel-containing membrane.","In certain further embodiments of the above described methods, the amphipathic lipids comprise phospholipids. In certain further embodiments the phospholipids comprise one or more phospholipids selected from phosphatidylcholine, phosphatidylethanolamine, phosphatidylserine, phosphatidylinositol, phosphatidylglycerol, cardiolipin, 1,2-diphytanoyl-sn glycerol-3-phosphocholine, and 1,2-dioleoyl-sn-glycero-3-phosphocholine. In certain embodiments the organic solvent comprises at least one solvent selected from chloroform, methanol, ethanol, n-propanol, isopropanol, n-butanol, isobutanol, pyridine, and diisopropyl ether. In certain embodiments the osmotic agent comprises at least one agent that is selected from sucrose or another disaccharide, glycerol, mannitol and dextran. In certain embodiments the DNA-packaging motor connector protein subunit comprises any of the above described DNA-packaging motor connector protein subunit polypeptides. In certain embodiments the lipid bilayer is present in a liposome, which in certain further embodiments is selected from a multilamellar liposome and a unilamellar liposome. In certain embodiments the incorporated viral DNA-packaging motor connector protein is mobile in the membrane layer. In certain embodiments the viral DNA-packaging motor connector protein is capable of translocating double-stranded DNA through the aperture when electrical potential is applied to the membrane. In certain embodiments conductance occurs in the conductive channel-containing membrane without voltage gating when electrical potential is applied. In certain further embodiments the applied electrical potential is selected from (i) a potential that is between \u2212100 mV and 100 mV, (ii) a potential that is between \u2212400 mV and 400 mV, (iii) a potential that is between \u2212300 mV and 300 mV, (iv) a potential that is between \u2212200 mV and 200 mV, (v) a potential that is between \u2212150 mV and 150 mV, (vi) a potential that is between \u221275 mV and 75 mV, and (vii) a potential that is between \u221250 mV and 50 mV.","According to certain other embodiments of the present invention there is provided a method of concentrating nucleic acid molecules on one side of a conductive channel-containing membrane that comprises a first side and a second side, the method comprising (a) making a conductive channel-containing membrane by a method comprising (i) substantially removing solvent from a mixture comprising amphipathic lipids and at least one solvent, to obtain dried amphipathic lipids; and (ii) resuspending the dried amphipathic lipids in a second solution that comprises an aqueous solvent, an osmotic agent and a plurality of isolated viral DNA-packaging motor connector protein subunit polypeptides that are capable of self-assembly into a homododecameric viral DNA-packaging motor connector protein, to obtain a membrane that comprises a lipid bilayer in which is incorporated the viral DNA-packaging motor connector protein under conditions and for a time sufficient for said connector protein to form an aperture through which conductance can occur when an electrical potential is applied across the membrane, and thereby making a conductive channel-containing membrane; and (b) contacting the conductive channel-containing membrane of (a) with one or a plurality of nucleic acid molecules and with an electrical potential that is applied across the membrane, under conditions and for a time sufficient for electrophoretic translocation of the nucleic acid through the aperture of the connector protein, and thereby concentrating nucleic acid molecules on one side of the conductive channel-containing membrane. In certain further embodiments nucleic acid translocation causes accumulation of the nucleic acid on one side of the membrane and against a nucleic acid concentration gradient. In another embodiment there is provided a nucleic acid-containing liposome that comprises a conductive channel-containing membrane and nucleic acid molecules that are concentrated on one side of the membrane, wherein the liposome is produced according to the method just described. In certain further embodiments the liposome is a nanoparticle, and in certain other embodiments the liposome is a bioreactor. According to certain embodiments there is provided a method for delivering a nucleic acid to a cell, comprising introducing to the cell one or a plurality of the liposomes described above. In one embodiment the cell is introduced to the one or plurality of liposomes in vitro, and in another embodiment the cell is introduced to the one or plurality of liposomes in vivo.","In another embodiment of the present invention there is provided an isolated protein, comprising a homododecamer of viral DNA-packaging motor connector protein polypeptide subunits, wherein each of said subunits comprises a fusion protein which comprises (a) an aperture domain that comprises an isolated viral connector protein polypeptide having an amino terminus and a carboxy terminus; (b) at least one flexibility domain that comprises a polypeptide of 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17 or 18 contiguous uncharged amino acids and that is fused to at least one of the amino terminus and the carboxy terminus of (a); and (c) at least one affinity\/alignment domain.","In another embodiment there is provided an isolated protein, comprising a homododecamer of viral DNA-packaging motor connector protein polypeptide subunits, wherein each of said subunits comprises a fusion protein which comprises (a) an aperture domain that comprises an isolated viral connector protein polypeptide having an amino terminus and a carboxy terminus; (b) at least one flexibility domain that comprises a polypeptide of 4, 5, 6, 7, 8, 9, 10, 11, 12, 13 or 14 contiguous uncharged amino acids and that is fused to at least one of the amino terminus and the carboxy terminus of (a); and (c) at least one affinity\/alignment domain that comprises a polypeptide of formula X-X-X-X-X-X-X-Xand that is fused to the flexibility domain, wherein each Xis independently either any uncharged amino acid or no amino acid, each Xis independently any uncharged amino acid, Xis a positively charged amino acid selected from lysine, arginine and histidine, Xis a negatively charged amino acid selected from glutamic acid and aspartic acid, and Xis a positively charged amino acid selected from lysine, arginine and histidine.","In one embodiment there is provided an isolated protein, comprising a homododecamer of viral DNA-packaging motor connector protein polypeptide subunits, wherein each of said subunits comprises a fusion protein which comprises (a) an aperture domain that comprises an isolated viral connector protein polypeptide having an amino terminus and a carboxy terminus; (b) at least one flexibility domain that comprises a polypeptide of sequence Gly-Gly-Gly-Gly-Gly-Gly as set forth in SEQ ID NO:23 and that is fused to the carboxy terminus of (a); and (c) at least one affinity\/alignment domain that is fused to the flexibility domain. In certain further embodiments the affinity\/alignment domain comprises a polypeptide that is selected from (i) a Strep-II tag as set forth in SEQ ID NO:22 [WSHPQRFEK], (ii) a polyhistidine polypeptide tag of 3, 4, 5, 6, 7, 8, 9, 10, 11 or 12 contiguous histidine residues, (iii) a polyarginine polypeptide of 3, 4, 5, 6, 7, 8, 9, 10, 11 or 12 contiguous arginine residues, (iv) an HIV Tat polypeptide of sequence YGRKKRRQRRR [SEQ ID NO:39] and (v) a peptide tag of sequence DRATPY [SEQ ID NO:40].","In another embodiment there is provided an isolated protein, comprising a homododecamer of viral DNA-packaging motor connector protein polypeptide subunits, wherein each of said subunits comprises a polypeptide that is selected from (i) C-His6-gp10\/K234A as set forth in SEQ ID NO:41, (ii) C-His6-gp10\/K234C as set forth in SEQ ID NO:42, (iii) C-His6-gp10\/CS\/CS\/K234C as set forth in SEQ ID NO:43, (iv) \u03941-14\/gp10-Strep-II as set forth in SEQ ID NO:44, and (v) Gp10\/\u0394285-309-Strep-II as set forth I SEQ ID NO:45.","In certain embodiments the aperture domain comprises a polypeptide that is selected from (i) all or a transmembrane aperture-forming portion of bacteriophage phi29 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:1 [Acc. No. ACE96033] (ii) all or a transmembrane aperture-forming portion of phage T4 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:7 [Acc. No. NP049782], (iii) all or a transmembrane aperture-forming portion of phage lambda DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in any one of SEQ ID NOS:8-11 [Acc. Nos. gi 549295, gi 6723246, gi 15837315, gi 16764273], (iv) all or a transmembrane aperture-forming portion of phage SPP1 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:12 [Acc. No. P54309], (v) all or a transmembrane aperture-forming portion of phage P22 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:13 [Acc. No. AAA72961], (vi) all or a transmembrane aperture-forming portion of phage P2 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:14 [Acc. No. NP046757], (vii) all or a transmembrane aperture-forming portion of phage P3 DNA-packaging motor connector protein polypeptide, (viii) all or a transmembrane aperture-forming portion of phage T3 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:15 [Acc. No. CAA35152], (ix) all or a transmembrane aperture-forming portion of phage T5 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NOS:16-19 (Accession numbers AAX12078, YP006980; AAS77191; AAU05287), and (x) all or a transmembrane aperture-forming portion of phage T7 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:20 [Acc. No. NP041995].","In certain embodiments the aperture domain comprises a polypeptide that comprises all or a transmembrane aperture-forming portion of a double-stranded DNA bacteriophage DNA-packaging motor connector protein. In certain embodiments the aperture domain comprises a polypeptide that is (i) all or a transmembrane aperture-forming portion of bacteriophage phi29 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:1 [Acc. No. ACE96033] (ii) all or a transmembrane aperture-forming portion of phage T4 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:7 [Acc. No. NP049782], (iii) all or a transmembrane aperture-forming portion of phage lambda DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in any one of SEQ ID NOS:8-11 [Acc. Nos. gi 549295, gi 6723246, gi 15837315, gi 16764273], (iv) all or a transmembrane aperture-forming portion of phage SPP1 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:12 [Acc. No. P54309], (v) all or a transmembrane aperture-forming portion of phage P22 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:13 [Acc. No. AAA72961], (vi) all or a transmembrane aperture-forming portion of phage P2 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:14 [Acc. No. NP046757], (vii) all or a transmembrane aperture-forming portion of phage P3 DNA-packaging motor connector protein polypeptide, (viii) all or a transmembrane aperture-forming portion of phage T3 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:15 [Acc. No. CAA35152], (ix) all or a transmembrane aperture-forming portion of phage T5 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NOS:16-19 (Accession numbers AAX12078, YP006980; AAS77191; AAU05287), and (x) all or a transmembrane aperture-forming portion of phage T7 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:20 [Acc. No. NP041995].","In certain embodiments the aperture domain comprises a polypeptide that comprises all or a transmembrane aperture-forming portion of a bacteriophage phi29 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:1 [Acc. No. ACE96033]. In certain embodiments the viral connector protein polypeptide comprises all or a transmembrane aperture-forming portion of bacteriophage phi29 DNA-packaging motor connector protein polypeptide having the amino acid sequence set forth in SEQ ID NO:1 [Acc. No. ACE96033]. In certain embodiments the above described isolated protein is capable of (i) self-assembly into a dodecameric viral connector protein, and (ii) packaging viral dsDNA following incorporation into a viral procapsid. In certain embodiments the isolated protein comprises at least one detectable label, which in certain further embodiments is selected from a colorimetric indicator, a GCMS tag compound, a fluorescent indicator, a luminescent indicator, a phosphorescent indicator, a radiometric indicator, a dye, an enzyme, a substrate of an enzyme, an energy transfer molecule, a quantum dot, a metal particle and an affinity label. In certain further embodiments the affinity label is selected from avidin, streptavidin, biotin, an aptamer, an antibody, a lectin, an oligosaccharide, a nucleic acid, an enzyme, a metal ion-binding polypeptide, a Strep-II tag as set forth in SEQ ID NO:22 [WSHPQRFEK], a polyhistidine polypeptide tag of 3, 4, 5, 6, 7, 8, 9, 10, 11 or 12 contiguous histidine residues, a Strep-I tag, a FLAG\u00ae peptide tag, a Myc peptide tag, glutathione-S-transferase, maltose binding protein, protein A, protein G, an HIV Tat polypeptide of sequence YGRKKRRQRRR [SEQ ID NO:39], a peptide having the amino acid sequence DRATPY [SEQ ID NO:40], glutaredoxin-2, and a phage-displayed peptide that specifically binds an affinity ligand. In certain embodiments the antibody is selected from an intact immunoglobulin, a single-chain antibody, an scFv, a Fab and a (Fab)\u20322.","In another embodiment of the present invention there is provided a method for detecting presence of an analyte molecule, comprising (a) contacting a test solution containing the analyte molecule with a conductive channel-containing membrane which comprises a membrane layer and incorporated therein one or a plurality of isolated viral DNA-packaging motor connector proteins that are capable of forming an aperture through which conductance can occur when an electrical potential is applied across the membrane, and that each comprise a homododecamer of viral DNA-packaging motor connector protein polypeptide subunits, wherein each of said subunits comprises (1) an aperture domain that comprises an isolated viral connector protein polypeptide having an amino terminus and a carboxy terminus, and (2) either or both of (i) at least one flexibility domain and (ii) at least one affinity\/alignment domain, under conditions and for a time sufficient for specific binding of the analyte molecule to the affinity\/alignment domain; and (b) determining, at one or a plurality of time points prior to the step of contacting and at one or a plurality of time points after the step of contacting, a conductance signal that results from the applied electrical potential, wherein an alteration in the conductance signal after the step of contacting relative to the conductance signal prior to the step of contacting indicates binding of the analyte molecule to the connector protein, and therefrom detecting presence of the analyte molecule. In a further embodiment the alteration in the conductance signal indicates binding of the analyte molecule to the affinity\/alignment domain.","Turning to another embodiment of the present invention, there is provided a method for identifying an analyte, comprising (a) contacting a test solution containing the analyte molecule with a conductive channel-containing membrane which comprises a membrane layer and incorporated therein one or a plurality of isolated viral DNA-packaging motor connector proteins that are capable of forming an aperture through which conductance can occur when an electrical potential is applied across the membrane, and that each comprise a homododecamer of viral DNA-packaging motor connector protein polypeptide subunits, wherein each of said subunits comprises (1) an aperture domain that comprises an isolated viral connector protein polypeptide having an amino terminus and a carboxy terminus, and (2) either or both of (i) at least one flexibility domain and (ii) at least one affinity\/alignment domain, under conditions and for a time sufficient for specific binding of the analyte molecule to the affinity\/alignment domain; (b) determining, at one or a plurality of time points prior to the step of contacting and at one or a plurality of time points after the step of contacting, a conductance signal that results from the applied electrical potential and therefrom generating a conductance signal profile, wherein an alteration in the conductance signal after the step of contacting relative to the conductance signal prior to the step of contacting indicates binding of the analyte molecule to the connector protein; and (c) comparing the conductance signal profile from (b) to a reference conductance signal profile for the analyte, and therefrom identifying the analyte molecule. In certain further embodiments, the alteration in the conductance signal indicates binding of the analyte molecule to the affinity\/alignment domain.","According to certain embodiments of the above described methods, the step of contacting is repeated one or a plurality of times. In certain embodiments the step of comparing comprises one or more of (i) comparing conductance signal amplitude from the conductance signal profile of (b) to conductance signal amplitude in the reference conductance signal profile for the analyte, and (ii) comparing conductance signal duration from the conductance signal profile of (b) to conductance signal duration in the reference conductance signal profile for the analyte. In certain embodiments of the above described methods, the applied electrical potential results in ionic migration along an electrochemical gradient in the aperture domain. In certain embodiments the analyte comprises a nucleic acid molecule. In certain embodiments the analyte comprises a nucleic acid molecule and the step of comparing comprises identifying at least one nucleotide that is present in the nucleic acid molecule. In certain embodiments the method comprises determining a nucleic acid sequence of the nucleic acid molecule. In certain embodiments the method comprises identifying a single nucleotide polymorphism in the nucleic acid molecule. In certain embodiments voltage gating is not present, which in certain embodiments is not present when the applied electrical potential is selected from (i) a potential that is between \u2212100 mV and 100 mV, (ii) a potential that is between \u2212400 mV and 400 mV, (iii) a potential that is between \u2212300 mV and 300 mV, (iv) a potential that is between \u2212200 mV and 200 mV, (v) a potential that is between \u2212150 mV and 150 mV, (vi) a potential that is between \u221275 mV and 75 mV, and (vii) a potential that is between \u221250 mV and 50 mV. In certain embodiments of the just-described methods, the conductive channel-containing membrane is the conductive channel-containing membrane described above. In certain embodiments of the just-described methods, the conductive channel-containing membrane is made according to the methods described above. In certain embodiments of the just-described methods, the isolated viral DNA-packaging motor connector protein comprises the isolated protein as described above.","Yet another embodiment of this disclosure provides an optical channel membrane, comprising (a) a membrane layer, and (b) an isolated viral DNA packaging motor connector protein, said viral DNA packaging motor connector protein is labeled with at least one donor fluorophore and is incorporated into the membrane layer to form a aperture through which fluorescence excitation is captured when an analyte labeled with at least one acceptor fluorophore passes through the membrane layer.","In certain embodiments of this disclosure the optical channel membrane is also a conductive channel membrane, which comprises (a) a membrane layer, and (b) an isolated viral DNA packaging motor connector protein, said viral DNA packaging motor connector protein is labeled with at least one donor fluorophore and is incorporated into the membrane layer to form a aperture through which fluorescence detection is coupled with the current detection when (i) an analyte labeled with at least one acceptor fluorophore passes through the membrane; and (ii) an electrical potential is applied across the membrane.","In the embodiments that contain the optical channel membrane the connector protein is labeled by Quantum Dots (QDs), for example but not limited to QD525, Alexa Fluor\u2122 488, QD 655 or QD 705 Alexa Fluor\u2122700. The analyte is labeled by the corresponding acceptor that is to be excited by said QDs. The examples of acceptor choices are, not limited to, Cy3\u2122, Alexa Fluor\u2122 546, Tetramethylrhodamine, TAMRA\u2122, Cy5.5, Alexa Fluor\u2122 700, IRDye\u00ae 700, or Alexa Fluor\u2122 750, Alexa Fluor\u2122790, Dy750.","Another embodiment of this disclosure provide a method of making an optical channel membrane, comprising: (a) preparing dried amphipathic lipids on a solid substrate by contacting a first solution comprising amphipathic lipids and an organic solvent with the solid substrate and substantially removing the solvent; (b) resuspending the dried amphipathic lipids in a second solution that comprises an aqueous solvent, an osmotic agent and a plurality of isolated viral DNA-packaging motor connector protein subunit polypeptides that are capable of self-assembly into a homododecameric viral DNA-packaging motor connector protein, to obtain a membrane that comprises a lipid bilayer in which is incorporated the viral DNA-packaging motor connector protein; and (c) attaching FRET donors to either N or C terminus of the connector subunits under conditions and for a time sufficient for said connector protein to form an aperture through which fluorescence excitation occurs and is being captured when an analyte labeled with a corresponding acceptor passes through the membrane, thereby making an optical detection membrane.","This disclosure also provides a method of making an optical and conductive channel membrane for synchronous detection of both optical and electrical signals, comprising: (a) preparing dried amphipathic lipids on a solid substrate by contacting a first solution comprising amphipathic lipids and an organic solvent with the solid substrate and substantially removing the solvent; (b) resuspending the dried amphipathic lipids in a second solution that comprises an aqueous solvent, an osmotic agent and a plurality of isolated viral DNA-packaging motor connector protein subunit polypeptides that are capable of self-assembly into a homododecameric viral DNA-packaging motor connector protein, to obtain a membrane that comprises a lipid bilayer in which is incorporated the viral DNA-packaging motor connector protein; and (c) attaching FRET donors to either N or C terminus of the connector subunits under conditions and for a time sufficient for said connector protein to form an aperture through which (i) fluorescence excitation occurs and is being captured when an analyte labeled with a corresponding acceptor passes through the membrane; and (ii) conductance occurs when an electrical potential is applied across the membrane, thereby making an optical and conductive channel containing membrane.","These and other aspects and embodiments of the invention will be evident upon reference to the following detailed description and attached drawings. All of the U.S. patents, U.S. patent application publications, U.S. patent applications, foreign patents, foreign patent applications and non-patent publications referred to in this specification and\/or listed in the Application Data Sheet, are incorporated herein by reference in their entirety, as if each was incorporated individually.","Certain embodiments of the invention disclosed herein are based on the surprising discovery that an isolated double-stranded DNA bacteriophage DNA-packaging motor connector protein can, unexpectedly, be modified and incorporated into a membrane layer such as a phospholipid bilayer membrane, to form an aperture through which conductance can occur when an electrical potential is applied across the membrane, and through which a wide variety of analyte molecules including double-stranded DNA (dsDNA) can be translocated.","These viral DNA-packaging motor protein connectors have previously been characterized as components of dsDNA viral procapsid assembly involving protein-protein and protein-RNA interactions (e.g., Simpson et al., 2000 408:745; Robinson et al., 2006 34:2698), but prior to the present disclosure they have never been regarded as having the potential to act as integral components of lipid membranes. The herein described embodiments thus exploit heretofore unrecognized molecular attributes of such connector proteins to provide transmembrane aperture-forming biosensors having a number of advantageous properties.","An exemplary unmodified viral DNA-packaging motor connector protein from bacteriophage phi29 has been purified and its three-dimensional structure has been crystallographically characterized (e.g., Guasch et al., 1998 430:283; see also Marais et al., 2008 16:1267). DNA-packaging motor connector proteins of other dsDNA viruses (e.g., T4, lambda, P22, P2, T3, T5 and T7), despite sharing little sequence homology with, and differing in molecular weight from, the phi29 connector, exhibit significant underlying structural similarities (e.g., Bazinet et al., 1985 39:109-29). Accordingly, a number of preferred embodiments as described herein refer to the phi29 DNA-packaging motor connector protein (e.g., Genbank Acc. No. ACE96033, [SEQ ID NO:1]) and\/or to polypeptide subunits thereof including fragments, variants and derivatives thereof (e.g., Acc. Nos. gi 29565762, gi 31072023, gi 66395194, gi 29565739, gi 157738604, [SEQ ID NOS:2-6]), but the invention is not intended to be so limited.","Instead, in certain other embodiments the use of an isolated viral DNA-packaging motor connector protein from other dsDNA viruses is contemplated, including without limitation the isolated viral DNA-packaging motor connector protein from any of phage lambda, P2, P3, P22, T3, T4, T5, SPP1 and T7, or another isolated dsDNA virus DNA-packaging motor connector protein (e.g., T4 (Acc. No. NP049782)(Driedonks et al., 1981 152:641), lambda (Acc. Nos. gi 549295, gi 6723246, gi 15837315, gi 16764273)(Kochan et al., 1984 174:433), SPP1 (Acc. No. P54309), P22 (Acc. No. AAA72961)(Cingolani et al., 2002 139:46), P2 (Acc. No. NP046757, P3 (Nutter et al., 1972 10(3):560-2), T3 (Acc. No. CAA35152)(Carazo et al., 1986 94:105), T5 (Accession numbers AAX12078, YP006980; AAS77191; AAU05287), T7 (Acc. No. NP041995)(Cerritelli et al., 1996 285:299; Agirrezabala et al., 2005 347:895)) [SEQ ID NOS:7-20]).","Without wishing to be bound by theory, it is believed in this regard that like the phi29 DNA-packaging motor connector protein exemplified herein, these and other dsDNA virus packaging motor connector proteins, which have been substantially structurally characterized, can be modified according to the teachings herein to obtain an isolated DNA-packaging motor connector protein that can be incorporated into a membrane layer to form an aperture through which conductance can occur when an electrical potential is applied across the membrane. Accordingly, disclosure herein with respect to the phi29 connector protein is intended, for certain embodiments, to be illustrative of related embodiments that are contemplated using any of such other isolated dsDNA viral DNA-packaging motor connector proteins, which may be modified for use in such embodiments according to the teachings found herein.","As described in greater detail herein, isolated DNA-packaging motor connector protein polypeptides, including such polypeptides that have been artificially engineered to possess unprecedented properties of membrane incorporation (e.g., stable transmembrane integration in a membrane layer) and functional electroconductive transmembrane aperture formation, exhibit unpredicted and advantageously superior traits to provide a new class of electroconductive biosensors, and in particular embodiments, biosensors that are capable of translocating double-stranded DNA (dsDNA) across a membrane layer in response to an electrical potential. Certain embodiments therefore contemplate the use of these biosensors for DNA sequencing.","Briefly and by way of background, the genome of linear dsDNA viruses is packaged into a preformed procapsid (Black, 43, 267-292 (1989), Guo, () 5(1), 1-3 (1994), Guo et al., 64, 886-903 (2007), Rao et al., . (2008). This entropically unfavorable process is accomplished by an ATP-driven motor (Guo et al. 197, 229-236 (1987), Chemla et al., 122, 683-692 (2005), Hwang et al., 35, 2796-2803 (1996), Sabanayagam et al., 93, L17-L19 (2007)). In bacteriophage phi29, the motor uses one ATP to package 2 bp (Guo et al., 1987) or 2.5 bp of DNA (Moffitt et al., 457, 446-4U2 (2009). The protein hub of this motor is a truncated cone structure, termed a connector (), that allows dsDNA to enter during maturation and exit during infection (Kochan et al., 174, 433-447 (1984), Rishovd et al., 245, 11-17 (1998), Simpson et al., D57, 1260-1269 (2001), Guasch et al., 315, 663-676 (2002), Agirrezabala et al., 347, 895-902 (2005). The connector has a central channel () formed by twelve GP10 protein subunits. While the connector proteins of viruses share little sequence homology and vary in molecular weight, there is significant underlying structural similarity (Bazinet & King, 39, 109-129 (1985)). By demonstrating viral DNA packaging and procapsid conversion to infectious virions, phi29 DNA packaging motor was the first to be assembled in vitro in a defined system and remains one of the most well studied (Guo et al., 83, 3505-3509 (1986)). The motor utilizes six pRNA (packaging RNA) molecules (Guo et al., 236, 690-694 (1987), Guo et al., 2, 149-155 (1998), Zhang, et al., 2, 141-147 (1998), Shu et al., 26, 527-537 (2007)) to gear the machine ().","As described herein for the first time, phi29 and other isolated dsDNA viral DNA-packaging motor protein connectors, including engineered and mutated versions thereof such as fusion proteins that retain their aperture domain and comprise either or both of an affinity\/alignment domain and a flexibility domain, may be usefully incorporated into membrane layers to form apertures permitting their use as conductive channels when an electrical potential is applied across the membrane. Modified isolated double-stranded DNA virus DNA-packaging motor protein connectors such as the phi29 connector may be engineered to have desired structures for use in the presently disclosed embodiments (Jiminez et al., 1986 232:1113: Donate et al., 1994 3:2378; Bradley et al., 309: 1868-1871 (2005); Schueler-Furman et al., 310:638 (2005); Dietz et al., 103:1244 (2006); Dodson et al., 450:176 (2007); Qian et al., 450:259 (2007)), where protein crystallographic structural data are readily available (e.g., Simpson et al., D57, 1260-1269 (2001), Guasch et al., 315, 663-676 (2002), Cai et al., 4, 8-18 (2008), Guo et al., 5, 856-863 (2005). Furthermore, the procedures for large scale production and purification of phi29 connector have been developed (Guo et al., 2005; Ibanez et al., 12, 2351-2365 (1984), Robinson et al., 34, 2698-2709 (2006), Xiao et al., 3, 100-107 (2009).","Embodiments described herein will accordingly find use in a variety of molecular analytical contexts, including, for example, sensitive detection and characterization of chemical and biochemical analytes for biomedical, clinical, industrial, chemical, pharmaceutical, environmental, forensic, national security, toxicological and other purposes, including any situation where rapid, specific and exquisitely sensitive detection and\/or characterization of an analyte (e.g., preferably a soluble analyte that is provided in solution) may be desired. Expressly contemplated are embodiments in which the presently disclosed compositions and methods are used for DNA sequencing as described in greater detail below, including dsDNA sequencing, high-throughput DNA sequencing, genomics, SNP detection, molecular diagnostics and other DNA sequencing applications.","Exemplary analytes thus include nucleic acids such as DNA and RNA (including dsDNA and dsRNA), including for the detection and identification of single nucleotide polymorphisms (SNPs) and\/or mutations in such nucleic acids, and\/or nucleic acid sequence determination. Other exemplary analytes that may be detected and\/or characterized using the herein described compositions and methods include other biopolymers (e.g., proteins, glycoproteins, peptides, glycopeptides, oligosaccharides, polysaccharides, lipids, glycolipids, phospholipids, etc.) and other biomolecules (e.g., soluble mediators, cofactors, vitamins, bioactive lipids, metabolites, and the like), drugs and other pharmaceutical and pharmacological agents, including natural and synthetic compounds, food and cosmetics agents such as flavorants, odorants, preservatives, antioxidants, antimicrobial agents, stabilizers, carriers, excipients, modifying agents and the like, natural and synthetic toxins, dyes, and other compounds.","Accordingly and in certain embodiments, any analyte for which detection and\/or characterization is desired may be used, where it will be recognized from the disclosure herein that the analyte is preferably soluble in a solvent that does not compromise the integrity of the particular membrane layer in which the isolated viral DNA-packaging motor connector protein is incorporated to form an aperture through which conductance can occur when an electrical potential is applied across the membrane. Analyte selection may thus vary as a function of the composition of the particular membrane layer being used, which may therefore influence solvent selection. Those skilled in the art will be familiar with criteria to be employed for selecting a solvent that is compatible with a membrane layer of any particular composition. In preferred embodiments, the membrane layer comprises a phospholipid bilayer and the solvent in which the analyte is provided comprises an aqueous solvent, e.g., a solvent that comprises water.","The presently disclosed compositions and methods may include in certain embodiments the practice of measuring electrical conductance, across a membrane layer in which the herein-described isolated viral DNA-packaging motor connector protein is incorporated, by adapting established electrophysiology instrumentation and methodologies. For example, and as described in greater detail below, modifications may be made to patch-clamp or planar membrane techniques for generating transmembrane potentials and measuring conductance across such membranes. Exemplary descriptions of such methodologies may also be found, for example, in Kasianowicz et al., 1996 93:13770; Gu et al., 1999 398:686; Kasianowicz et al., 2001 73:2268; Henrickson et al., 2000 85:3057; Hromada et al., 2008 8:602; Robertson et al., 2007 104:8207; U.S. Pat. Nos. 6,267,872; 6,746,594; and 6,936,433. Those familiar with the art will appreciate general methodologic approaches from these and similar references, and it will be further understood that advantages as described herein may be derived in part from the present disclosure, for the first time, of novel conductive channel-containing membranes that comprise an incorporated isolated viral DNA-packaging motor connector protein that forms an aperture having a larger lumen than many heretofore described protein nanopores, and that functions as a conductive channel across a wide range of applied electrical potentials without undesirable interruptions in conductivity due to voltage gating behavior, which is seen with many previously described protein nanopore channels. Thus, for example by way of illustration and not limitation, using known electrophysiology methodologies such as those referenced above and described herein, conductance occurs in the presently disclosed conductive channel-containing membrane without voltage gating when an electrical potential is applied that may in selected embodiments be between \u2212100 mV and 100 mV, between \u2212400 mV and 400 mV, between \u2212300 mV and 300 mV, between \u2212200 mV and 200 mV, between \u2212150 mV and 150 mV, between \u221275 mV and 75 mV, between \u221250 mV and 50 mV, or within another voltage range as may vary according to the particular conductance conditions that are employed, as will be apparent to the skilled person based on the present disclosure.","Further, and according to non-limiting theory, the present conductive channel-containing membranes offer advantageous detection sensitivities that may derive in part from the aperture formed by the present viral DNA-packaging motor protein connectors, and also provide advantageous analyte characterization capabilities that may derive in part from the stable membrane incorporation of a protein conductive channel that can be engineered or mutated to have desired functional properties such as any of a wide variety of analyte-accessible affinity interaction domains by which to engage analyte in a specific binding interaction.","For methods of detecting an analyte, these and related embodiments contemplate sensitivity that is obtained by observation of an altered (e.g., increased or decreased in a statistically significant manner) level of conductance through the aperture in the conductive channel-containing membrane across which electrical potential is applied, when the incorporated connector protein is engaged in a specific binding interaction with the analyte, relative to the level of conductance when no such specific binding interaction is present. Accordingly in certain preferred embodiments there is provided a method for detecting presence of an analyte molecule, comprising (a) contacting a test solution containing the analyte molecule with a conductive channel-containing membrane which comprises a membrane layer and incorporated therein one or a plurality of isolated viral DNA-packaging motor connector proteins that are capable of forming an aperture through which conductance can occur when an electrical potential is applied across the membrane, and that each comprise a homododecamer of viral DNA-packaging motor connector protein polypeptide subunits, wherein each of said subunits comprises (1) an aperture domain that comprises an isolated viral connector protein polypeptide having an amino terminus and a carboxy terminus, and (2) either or both of (i) at least one flexibility domain and (ii) at least one affinity\/alignment domain, under conditions and for a time sufficient for specific binding of the analyte molecule to the affinity\/alignment domain; and (b) determining, at one or a plurality of time points prior to the step of contacting and at one or a plurality of time points after the step of contacting, a conductance signal that results from the applied electrical potential, wherein an alteration in the conductance signal after the step of contacting relative to the conductance signal prior to the step of contacting indicates binding of the analyte molecule to the connector protein, and therefrom detecting presence of the analyte molecule. In certain related further embodiments, the alteration in the conductance signal indicates binding of the analyte molecule to the affinity\/alignment domain.","By way of illustration and not limitation, the aperture formed by the membrane-incorporated connector protein is believed to be at least partially obstructed or occluded when analyte is present and is bound to the channel-conductive membrane, resulting in altered, and typically decreased, conductance across the membrane. In the absence of bound analyte, no such constraint is placed on conductance through the channel, such that altered levels of conductance may be readily detected when bound and unbound states of the analyte are observed. As shown below in the Examples, the present conductive channel-containing membrane is believed according to non-limiting theory to afford exquisite sensitivity in the detection of analyte by permitting observation of such an analyte binding-associated alteration in conductance at the level of a single analyte molecule binding to a single membrane-incorporated isolated viral DNA-packaging motor connector protein that has formed a conductive transmembrane aperture. In certain other embodiments, conductance signals may be detected from multiple conductive channels formed by multiple transmembrane apertures of the connectors described herein.","In certain conceptually related embodiments, information beyond the mere detection of the absence or presence of an analyte may be obtained, where a conductance signal profile is generated, for instance, by collecting a record that reflects the amplitude of conductance, including altered conductance as described herein at one or a plurality of time points, and the duration of conductance, including altered conductance as described herein, at one or a plurality of timepoints. Such a signal profile may reflect any number of properties of the analyte in the course of its interaction with the connector protein(s), for example, binding affinity and\/or binding avidity (e.g., if the analyte is multivalent), and also potentially, physicochemical properties of the analyte such as relative molecular mass, charge, and\/or hydrophobicity, which may vary as a function of the particular analyte, the particular connector protein, the membrane composition, the solvent conditions, the applied electrical potential, and other factors.","In certain particularly preferred embodiments, a reference conductance signal profile may be generated using an analyte of known molecular structure, to which reference profile a conductance signal profile that is generated by the interaction of a test solution containing an analyte for which structural information is desired can be compared.","Hence, in these and related embodiments there is provided a method for identifying an analyte, comprising (a) contacting a test solution containing the analyte molecule with a conductive channel-containing membrane which comprises a membrane layer and incorporated therein one or a plurality of isolated viral DNA-packaging motor connector proteins that are capable of forming an aperture through which conductance can occur when an electrical potential is applied across the membrane, and that each comprise a homododecamer of viral DNA-packaging motor connector protein polypeptide subunits, wherein each of said subunits comprises (1) an aperture domain that comprises an isolated viral connector protein polypeptide having an amino terminus and a carboxy terminus, and (2) either or both of (i) at least one flexibility domain and (ii) at least one affinity\/alignment domain, under conditions and for a time sufficient for specific binding of the analyte molecule to the affinity\/alignment domain; (b) determining, at one or a plurality of time points prior to the step of contacting and at one or a plurality of time points after the step of contacting, a conductance signal that results from the applied electrical potential and therefrom generating a conductance signal profile, wherein an alteration in the conductance signal after the step of contacting relative to the conductance signal prior to the step of contacting indicates binding of the analyte molecule to the connector protein; and (c) comparing the conductance signal profile from (b) to a reference conductance signal profile for the analyte, and therefrom identifying the analyte molecule.","In certain further embodiments the alteration in the conductance signal indicates binding of the analyte molecule to the affinity\/alignment domain. In any of the above described methods for detecting an analyte and for identifying an analyte, the step of contacting may be repeated one or a plurality of times, for instance, to generate a conductance signal profile. As noted above, comparison of a reference conductance signal profile for a known analyte to a test conductance signal profile for a test analyte may permit identification of the test analyte, an approach that may be employed, for example, in the characterization of nucleic acid molecules, such as identifying one or more single nucleotide polymorphisms in a nucleic acid molecule, or sequencing a nucleic molecule (including dsDNA) by determining a series of characteristic alterations in conductance as a linear DNA molecule is translocated through the aperture. In these and related embodiments it will be appreciated that based on the disclosure herein, the skilled person can, readily and without undue experimentation, determine conditions (e.g., selection of membrane, connector protein, solution conditions, applied electrical potential, etc.) under which a reference conductance signal profile can be generated, and therefrom generate a test conductance signal profile to which it can be compared.","Thus in certain embodiments the step of comparing comprises one or more of (i) comparing conductance signal amplitude from the test conductance signal profile to conductance signal amplitude in the reference conductance signal profile for the analyte, and (ii) comparing conductance signal duration from the test conductance signal profile to conductance signal duration in the reference conductance signal profile for the analyte. In certain embodiments, for example those in which complete or partial translocation of the analyte through the aperture is achieved, the applied electrical potential may result in ionic migration along an electrochemical gradient in the aperture domain. Thus in certain related embodiments of the instant methods, the analyte comprises a nucleic acid molecule, and in certain other related embodiments the step of comparing comprises identifying at least one nucleotide that is present in the nucleic acid molecule, which method in certain further embodiments comprises determining a nucleic acid sequence of the nucleic acid molecule.","It will be appreciated that certain preferred embodiments as described herein relate to an isolated DNA-packaging motor connector protein that comprises a dodecamer of chimeric or fusion polypeptide subunits, i.e., a non-naturally occurring polypeptide that is the product of recombinant genetic engineering techniques with which those skilled in the art will be familiar. In these and related embodiments, the fusion polypeptide comprises (i) an aperture domain, which may be all or an aperture-forming portion of a double-stranded DNA virus DNA-packaging motor protein connector polypeptide such as those discussed above and elsewhere herein (e.g., any polypeptide set forth in SEQ ID NOS:1-20 or other related DNA-packaging motor protein polypeptide, including variants, fragments and derivatives); and (ii) either or both of at least one flexibility domain and at least one affinity\/alignment domain. Exemplary connectors may comprise homododecamers of viral DNA-packaging motor protein connector polypeptide subunits such as those having the amino acid sequences set forth in SEQ ID NOS:31-35 and 41-45.","Typically, the flexibility domain and\/or the affinity\/alignment domain may be present as a peptide sequence attached to the C-terminus and\/or the N-terminus of the aperture domain, but the invention is not intended to be so limited and also contemplates inclusion of the flexibility and\/or affinity\/alignment domains at other locations within the connector polypeptide sequence. As noted elsewhere herein, detailed crystallographic structural data are available for the instant viral DNA-packaging motor protein connector polypeptides, and sophisticated three-dimensional structural protein modeling software programs are well known in the art for predicting the effects of structural modifications to proteins.","According to non-limiting theory, isolated DNA-packaging motor connector proteins as disclosed herein may be stabilized for retention and functional incorporation in membrane layers such as phospholipid bilayers, by modifying those portions of the connector polypeptides that interact with, respectively, hydrophilic phospholipid polar head groups and hydrophobic phospholipid fatty acyl chains, in a manner that energetically favors integration of the connector in the membrane to form a transmembrane aperture. Compositions and methods for introducing proteins to membranes, and for determining their incorporation into membranes, and further for ascertaining their disposition in the membranes as integral or transmembrane proteins, are known in the art and exemplified herein, as also are methodologies for determining functional incorporation of such proteins as electroconductive transmembrane channels.","Hence, and as presented herein, for example, a C-terminal modification comprising a flexibility domain of uncharged amino acids such as 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18 or more contiguous uncharged amino acids (e.g., Gly, SEQ ID NO:23), and C-terminally attached thereto an affinity\/alignment domain such as a peptide of formula I:\n\nX-X-X-X-X-X-X-X,\u2003\u2003[I]\n","wherein each Xis independently either any uncharged amino acid or no amino acid, each Xis independently any uncharged amino acid, Xis a positively charged amino acid selected from lysine, arginine and histidine, Xis a negatively charged amino acid selected from glutamic acid and aspartic acid, and Xis a positively charged amino acid selected from lysine, arginine and histidine, for example, the peptide WSHPQFEK (SEQ ID NO:22) sometimes referred to as a \u201cStrep-II\u201d tag, is believed advantageously to influence the physicochemical properties of a dsDNA viral DNA-packaging motor protein connector in a manner that favors its retention as a transmembrane channel in a lipid bilayer, and that also advantageously exposes the tag to the surrounding milieu for interaction with solutes such as analytes. Thus, for instance and as a non-limiting illustrative example, under suitable conditions such as are provided by a buffered solution at or near physiological pH and with an electrical potential applied across the membrane, charge interactions may favor engagement of DNA in solution by the conductive channel and electrodiffusive forces may promote DNA translocation through the conductive channel aperture, which as noted herein can accommodate dsDNA.","The aperture domain may thus be provided by any dsDNA viral DNA-packaging motor protein connector polypeptide as described herein, and from available structural characterization of such proteins in view of the present disclosure regarding membrane incorporation of the modified connectors and their functional testing both for membrane integration and for establishment of an aperture through which conductance can occur when a membrane potential is applied, it will be appreciated that an aperture-forming portion includes a sufficient portion of such connector polypeptide subunits as may be needed for an assembled connector to exhibit these properties. As noted above, in certain embodiments a flexibility domain of about 4-18 contiguous uncharged amino acids may be advantageously fused to a viral DNA-packaging motor protein connector sequence as provided herein.","As also noted above, an affinity\/alignment domain may be any peptide or polypeptide domain that can be fused to the viral DNA-packaging motor protein connector polypeptide subunit sequence, with or without an intervening flexibility domain being present, that promotes retention of the connector in the membrane layer and\/or that provides an affinity interaction domain such as a receptor, ligand, binding site, counter-receptor or the like, which may be used to engage in specific binding of an analyte and\/or to encourage analyte interaction with the conductive channel and\/or both.","An exemplary affinity\/alignment domain described for use herein in certain embodiments is the \u201cStrep-II\u201d peptide tag, WSHPQFEK [SEQ ID NO:22], by way of non-limiting example situated in a fusion protein construct as a C-terminal affinity tag, (e.g., Schmidt et al., 1993; 6:109-22; Schmidt et al., 1996; 255:753-6.) or any of a number of other known peptide or polypeptide or other structures that have been identified as being capable of participating in specific affinity binding interactions with a cognate binding partner, such as, e.g., an antibody or an antigen-binding portion thereof (as also described below), the \u201cAviD\u201d tag having the sequence DRATPY [SEQ ID NO:40] (Gaj et al., 2007 56:54), glutaredoxin-2 (Lundberg et al., 2006 45:37), or other known moieties such as avidin, streptavidin, lectins, antibodies, receptors, cell adhesion molecule recognition components, lectins, metal ion-binding polypeptides (e.g., polyhistidines such as hexahistidine or a polypeptide tag of 3, 4, 5, 7, 8, 9, 10, 11 or 12 contiguous histidine residues), a polyarginine polypeptide of 3, 4, 5, 6, 7, 8, 9, 10, 11 or 12 contiguous arginine residues, a FLAG peptide tag, a Myc peptide tag having an antibody-defined epitope, glutathione-S-transferase (GST), maltose binding protein, immunoglobulin constant-region-binding protein A or protein G, HIV Tat peptide (e.g., SEQ ID NO:39), a phage-displayed peptide that specifically binds an affinity ligand, an aptamer, or any other of a large number of well known affinity-interacting molecules that are known to the art, such as those described in the references cited herein. Other receptors, binding domains, ligands, counterreceptors and the like, all or parts of which may be used to produce affinity\/alignment domains, include those disclosed in WO\/2005\/097997, WO\/2002\/056910, WO\/2005\/017148, and WO\/2005\/037989. Certain embodiments contemplate viral DNA-packaging motor connector proteins as provided herein that further comprise a detectable label, which, it will be appreciated, may be any of a wide variety of chemical, biochemical, radiochemical or nanomolecular moieties that permits detection using established techniques and instrumentation. Exemplary detectable labels are known in the art and include those described in WO\/2007\/075253 and US\/2008\/0176209.","As described in greater detail in the examples below, according to certain preferred embodiments the presently described conductive channel-containing membrane may be formed by incorporating the isolated viral DNA-packaging motor connector protein as provided herein, including connectors comprised of polypeptide subunits which comprise fusion proteins having an aperture domain and either or both of a flexibility domain and an affinity\/alignment domain as described herein, into a liposomal membrane. The membrane typically comprises amphipathic lipids such as phospholipids (e.g., one or more of phosphatidylserine, phosphatidylethanolamine, phosphatidylcholine, phosphatidylinositol, phosphatidylglycerol, cardiolipin, 1,2-diphytanoyl-sn glycerol-3-phosphocholine, 1,2-dioleoyl-sn-glycero-3-phosphocholine, or other phospholipids with which the skilled artisan will be familiar), which tend to form bilayers when exposed under appropriate conditions to an aqueous milieu.","It will be appreciated that as described below, viral DNA-packaging motor protein connectors may be effectively incorporated into membranes by first providing amphipathic lipids from which solvent (e.g., an organic solvent such as one or more of chloroform, methanol, ethanol, n-propanol, isopropanol, n-butanol, isobutanol, pyridine, diisopropyl ether) has been substantially removed (e.g., such that little or no residual solvent can be detected by visual inspection, and preferably such that the lipid preparation is regarded as dry), and then resuspending the dried amphipathic lipids in a solution that comprises an aqueous solvent, an osmotic agent, and a plurality of isolated viral DNA-packaging motor protein connector subunit polypeptides that are capable of self-assembly into homododecameric connector proteins. Without wishing to be bound by theory, it is believed that the inclusion of the osmotic agent advantageously facilitates functional DNA-packaging motor connector incorporation into the membrane as conductive channels, by influencing the formation and size of, and\/or intermolecular dynamics within, substantially spherical liposomes, although it is recognized that other methods for producing conductive channel-containing membranes should not be excluded. Non-limiting examples of osmotic agents that may be used in these and related embodiments include sucrose or other disaccharides, glycerol, mannitol and dextran.","In certain embodiments the liposome comprising the conductive channel-containing membrane may be used in liposomal form, for example, as a vehicle for delivery to cells in vitro or in vivo of nucleic acid molecules that have been concentrated or accumulated in the liposome, such as by electric potential-driven translocation, against the nucleic acid concentration gradient that would otherwise limit the nucleic acid concentration that can be achieved according to recognized equilibrium principles, through the connector apertures, to obtain nucleic acid-containing liposomes. Such liposomes may advantageously find use as therapeutic agents, instance in gene therapy and related strategies. A wide range of formulations are available for in vitro and in vivo therapeutic administration of nucleic acid-containing liposomes, and may be modified for use with liposomes produced according to the present disclosure. See, e.g., WO\/2002\/034236; WO\/2002\/036767; WO\/2003\/094963; WO\/2005\/034979; WO\/2005\/120461; WO\/2000\/03683; Lasic, 1997, CRC Press, Boca Raton, Fla.; WO 96\/40964; WO 1998\/51278; WO 2009\/086558; US 2007\/0042031; US 2006\/0240093; US 2006\/0083780; US 2004\/0142025. Multilamellar and unilamellar liposomes are contemplated, with unilamellar liposomes being preferred in certain embodiments.","Also contemplated are embodiments in which the membrane-bound compartments defined by the presently described conductive channel-containing membrane may be advantageously employed as bioreactors. As will be appreciated by those skilled in the art, according to the present teachings the herein described conductive channel may be used to selectively translocate any of a large number of analytes, biomolecules, synthetic small molecules, ionic species and\/or other solutes, depending on the properties of such compounds and of the particular conductive channel being employed (e.g., the nature of the affinity\/alignment domain, the connector structure, the membrane composition, the connector orientation, the applied potential, etc.). Accordingly, within the membrane-bound compartment, which may be liposomal or nanoparticulate or one side of a planar membrane within a suitable apparatus, one or more desired molecular species can be accumulated, (e.g., at desired concentrations that may be electrodiffusively accumulated against a concentration gradient that would otherwise limit the concentrations attainable as a function of equilibrium dynamics) under conditions that favor a desired biomolecular or biochemical reaction such as a reaction useful for nucleic acid sequencing, SNP determination, nucleic acid amplification, nucleic acid synthesis, ligation or cleavage, macromolecular assembly, analyte detection or any other desired biochemical or synthetically engineered bioreaction.","As also described herein, certain embodiments contemplate incorporation of the herein described isolated viral DNA-packaging motor protein connectors into membrane layers to form liposomes, which may then donate membrane-integrated connectors to planar membrane systems, such as planar bilayer membrane (BLM) systems, by way of artificial membrane fusion manipulations according to art-known methodologies. Accordingly, conductive channel-containing membranes as described herein for the first time may be configured as desired for a particular purpose and\/or to accommodate use with certain instrumentation, including but not limited to liposomal and\/or nanoparticle (including carrier particle) delivery, planar membrane layers, microfluidic chambers, micropore and nanopore electroconductivity chambers, patch-clamp apparatus, fluorescence labeled analyte detection\/characterization, and any other configuration compatible for use with the instant conductive channel-containing membrane as may be adapted based on the present disclosure and in view of knowledge in the art. As also described elsewhere herein, certain preferred embodiments contemplate the use of the present conductive channel-containing membrane for nucleic acid sequencing and SNP identification, including DNA sequencing and preferably including dsDNA sequencing, and in particular embodiments including real-time fingerprinting of dsDNA as it is translocated through the presently described conductive channel, by potential-driven translocation accompanied by detection and recording of altered conductance amplitude and\/or duration over a plurality of time points to generate a conductance profile that can be compared to a reference conductance profile. In certain embodiments, an independent optical detection of dsDNA or a synchronous optical and electrical detection of DNA translocation is achieved by the combination of both conductance and optical single molecule detection system.","Antibodies. As noted above, also contemplated by certain embodiments of the present invention are affinity\/alignment domains that comprise an antibody, including anti-analyte binding molecules that are peptides, polypeptides and other molecules that specifically bind to an analyte the detection and\/or analysis of which is desired. Such binding molecules can be used in a method for detecting presence of an analyte, or in a method for identifying an analyte (for example, in a test solution), as described herein. An antibody, or another affinity\/alignment domain as provided herein, is said to specifically bind to a particular cognate antigen or analyte (e.g., the analyte of interest) if it reacts (e.g., binds) at a detectable level with the antigen\/analyte but does not react detectably with structurally distinct or unrelated molecules. Preferred binding molecules thus include antibodies, which may be, for example, polyclonal, monoclonal, single chain, chimeric, humanized, anti-idiotypic, or CDR-grafted immunoglobulins, or antigen-binding fragments thereof, such as proteolytically generated or recombinantly produced immunoglobulin F(ab\u2032), Fab, Fab\u2032, Fv, and\/or Fd fragments, single domain antibodies (\u201cdAbs\u201d; Holt et al., 2003 21:484) and diabodies (Hudson et al., 1999 231:177). An antibody according to the present invention may belong to any immunoglobulin class, for example IgG, IgE, IgM, IgD, or IgA. It may be obtained from or derived from an animal, for example, fowl (e.g., chicken) or a mammal, which includes but is not limited to a mouse, rat, hamster, rabbit, or other rodent, a cow, horse, sheep, goat, camel, human, or other primate. The antibody may be an internalizing antibody, or the antibody may be modified so that it may be easily transported across a cell membrane.","For purposes of identifying and preparing a desired antibody or affinity\/alignment domain as provided herein, the binding properties of an antibody (used here in exemplary and non-limiting fashion as illustrative of affinity\/alignment domains) to its specific antigen (used here in exemplary and non-limiting fashion as illustrative of analytes) may generally be assessed using conventional immunodetection methods including, for example, an enzyme-linked immunosorbent assay (ELISA), immunoprecipitation, radioimmunoassays, immunoblotting and the like, which may be readily performed by those having ordinary skill in the art. A skilled artisan will also be familiar with such immunodetection methods which, when used to detect an antibody that binds to a conformational epitope of an antigen\/ligand, may preferably avoid any reagent or condition which could potentially denature the antigen and thus alter or destroy the ligand conformational epitope.","Methods well known in the art and described herein may be used to generate antibodies, including polyclonal antisera or monoclonal antibodies, that are specific for a particular antigen as may be desired. Antibodies also may be produced as genetically engineered immunoglobulins (Ig) or Ig fragments designed to have desirable properties. For example, by way of illustration and not limitation, antibodies may include a recombinant IgG that is a chimeric fusion protein having at least one variable (V) region domain from a first mammalian species and at least one constant region domain from a second, distinct mammalian species (see, e.g., Morrison et al., 81:6851-55 (1984); Shin et al., 178:459-76 (1989); Walls et al., 21:2921-29 (1993); U.S. Pat. No. 5,482,856). Most commonly, a chimeric antibody has murine variable region sequences and human constant region sequences. Such a murine\/human chimeric immunoglobulin may be \u201chumanized\u201d by grafting the complementarity determining regions (CDRs) derived from a murine antibody, which confer binding specificity for an antigen, into human-derived V region framework regions and human-derived constant regions (see, e.g., Jones et al., 321:522-25 (1986); Riechmann et al., 332:323-27 (1988); Padlan et al., 9:133-39 (1995); Chothia et al., 342:377-383 (1989); Bajorath et al., 2:95-103 (1995); EP-0578515-A3). Fragments of these molecules may be generated by proteolytic digestion, or optionally, by proteolytic digestion followed by mild reduction of disulfide bonds and alkylation. Alternatively, such fragments may also be generated by recombinant genetic engineering techniques (e.g., Harris, W. J., Adair, J. R., (.) 1997 Antibody Therapeutics, CRC Press, Boca Raton, Fla.).","An antibody that is immunospecific (or an affinity\/alignment domain that is specific) or that specifically binds to a cognate antigen (or analyte) as provided herein reacts at a detectable level with the antigen\/analyte and not with molecules having distinct or unrelated structures, preferably with an affinity constant, K, of greater than or equal to about 10M, more preferably of greater than or equal to about 10M, more preferably of greater than or equal to about 10M, and still more preferably of greater than or equal to about 10M. Affinity of an antibody for its cognate antigen is also commonly expressed as a dissociation constant K, and an antibody specifically binds to the antigen if it binds with a Kof less than or equal to 10M, less than or equal to about 10M, less than or equal to about 10M, less than or equal to 10M, or less than or equal to 10M. Affinities of binding partners or antibodies can be readily determined using conventional techniques, for example, those described by Scatchard et al. (51:660 (1949)) or by surface plasmon resonance (BIAcore, Biosensor, Piscataway, N.J.). See, e.g., Wolff et al., 53:2560-2565 (1993), or may be determined according to other methodologies as described herein.","Antibodies may generally be prepared by any of a variety of techniques known to those skilled in the art. See, e.g., Harlow et al., , Cold Spring Harbor Laboratory (1988). In one such technique, an animal is immunized with an immunogenic form of the antigen\/analyte, for instance, as a hapten on a suitable carrier according to established methodologies, as an antigen to generate polyclonal antisera. Suitable animals include, for example, rabbits, sheep, goats, pigs, cattle, and may also include smaller mammalian species, such as mice, rats, and hamsters, or other species.","An immunogen may comprise a purified or partially purified antigen or analyte of interest, or may be comprised of cells expressing the antigen (e.g., for an antigen which is a polypeptide or a polynucleotide or a metabolite) or to which the antigen has been introduced in a manner that enhances its immunogenicity. Peptide or polypeptide antigens may be generated using standard recombinant genetic methodologies, or by proteolytic cleavage of naturally occurring proteins, or may be chemically synthesized. Peptides may be isolated by techniques known in the art such as polyacrylamide gel electrophoresis or any of a variety of other separation methods such as liquid chromatography or other suitable methodologies.","For raising antibodies to antigens that are polypeptides or peptides, peptides useful as immunogens typically may have an amino acid sequence of at least 4 or 5 consecutive amino acids from the polypeptide sequence, and preferably have at least 6, 7, 8, 9, 10, 11, 12, 14, 15, 16, 18, 19 or 20 consecutive amino acids of the polypeptide sequence. Certain other preferred peptide immunogens may comprise 21-25, 26-30, 31-35, 36-40, 41-50 or more consecutive amino acids of a polypeptide sequence. Polypeptides or peptides useful for immunization may also be selected by analyzing the primary, secondary, and tertiary structure of a polypeptide antigen\/analyte of interest according to methods known to those skilled in the art, in order to determine amino acid sequences more likely to generate an antigenic response in a host animal. See, e.g., Novotny, 1991 28:201-207; Berzofsky, 1985 229:932-40; Chang et al. 117:863-68 (1995); Kolaskar et al. 261:31-42 (1999)). Preferably, the polypeptide or peptide comprises a sufficient number of amino acids to fold in a manner that approximates the conformation of the polypeptide\/analyte in its biologically active form.","Immunogens may be prepared and animals immunized according to methods well known in the art. See, e.g., Harlow et al., , Cold Spring Harbor Laboratory (1988). The immune response may be monitored by periodically bleeding the animal, separating the sera out of the collected blood, and analyzing the sera in an immunoassay, such as an ELISA or Ouchterlony diffusion assay, or the like, to determine the specific antibody titer. Once an antibody titer is established, the animals may be bled periodically to accumulate the polyclonal antisera. Polyclonal antibodies that bind specifically to the antigen may then be purified from such antisera, for example, by affinity chromatography using protein A or protein G, which specifically binds to a constant region (heavy or light chain) of the antibody(ies) to be purified, or using the antigen\/analyte, immobilized on a suitable solid support.","Monoclonal antibodies that specifically bind to the antigen, and hybridomas, which are immortal eukaryotic cell lines that produce monoclonal antibodies having the desired binding specificity, may also be prepared, for example, using the technique of Kohler and Milstein (256:495-497; 19766:511-519 (1975)) and improvements thereto with which a skilled artisan will be familiar. An animal\u2014for example, a rat, hamster, or a mouse\u2014is immunized with an immunogen; lymphoid cells that include antibody-forming cells, typically spleen cells, are obtained from the immunized animal; and such cells may be immortalized by fusion with a selection agent-sensitized myeloma (e.g., plasmacytoma) cell fusion partner.","Monoclonal antibodies may be isolated from the supernatants of hybridoma cultures or isolated from a mouse that has been treated (e.g., pristane-primed) to promote formation of ascites fluid containing the monoclonal antibody. Antibodies may be purified by affinity chromatography using an appropriate ligand selected based on particular properties of the monoclonal antibody (e.g., heavy or light chain isotype, binding specificity, etc.). Examples of a suitable ligand, immobilized on a solid support, include Protein A, Protein G, an anti-constant region (light chain or heavy chain) antibody, an anti-idiotype antibody and the antigen\/analyte for which specific antibodies are desired. Human monoclonal antibodies may be generated by any number of techniques with which those having ordinary skill in the art will be familiar.","Antibodies and other affinity\/alignment domain polypeptides may also be identified and isolated using well known phage display techniques, for example, from human immunoglobulin phage libraries, from rabbit immunoglobulin phage libraries, and\/or from chicken immunoglobulin phage libraries (see, e.g., Winter et al., 1994 12:433-55; Burton et al., 1994 57:191-280; U.S. Pat. No. 5,223,409; Huse et al., 1989 246:1275-81; Schlebusch et al., 1997 16:47-52 and references cited therein; Rader et al., 275:13668-76 (2000); Popkov et al., 325:325-35 (2003); Andris-Widhopf et al., 242:159-31 (2000)), or by other methodologies such as ribosome display (e.g., Hanes et al., 1998 95:14130) or yeast display (e.g., Colby et al., 2004 388:348) or the like. Antibodies isolated from non-human species or non-human immunoglobulin libraries may be genetically engineered according to methods described herein and known in the art, for example to optimize affinity, or to chimerize, or to \u201chumanize\u201d the antibody or fragment thereof.","In certain embodiments, a B cell from an immunized animal that is producing an antibody having desired specificity is selected and the light chain and heavy chain variable regions are cloned from the B cell according to molecular biology techniques known in the art (WO 92\/02551; U.S. Pat. No. 5,627,052; Babcook et al., 93:7843-48 (1996)) and described herein. Preferably B cells from an immunized animal are isolated from the spleen, lymph node, or peripheral blood sample by selecting a cell that is producing an antibody that specifically binds to the antigen\/analyte of interest. B cells may also be isolated from humans, for example, from a peripheral blood sample.","An antibody fragment may also be any synthetic or genetically engineered protein that acts like an antibody in that it binds to a specific antigen to form a complex. For example, antibody fragments include isolated fragments consisting of the light chain variable region; \u201cFv\u201d fragments consisting of the variable regions of the heavy and light chains; recombinant single chain polypeptide molecules in which light and heavy variable regions are connected by a peptide linker (scFv proteins); and minimal recognition units consisting of the amino acid residues that mimic the hypervariable region. Such an antibody fragment preferably comprises at least one variable region domain. (see, e.g., Bird et al., 242:423-26 (1988); Huston et al., 85:5879-5883 (1988); EP-B1-0318554; U.S. Pat. No. 5,132,405; U.S. Pat. No. 5,091,513; and U.S. Pat. No. 5,476,786).","In certain embodiments, an antibody that specifically binds to an antigen\/analyte may be an antibody that is expressed as an intracellular protein. Such intracellular antibodies are also referred to as intrabodies and may comprise an Fab fragment, or preferably comprise a scFv fragment (see, e.g., Lecerf et al., 98:4764-49 (2001)). The framework regions flanking the CDR regions can be modified to improve expression levels and solubility of an intrabody in an intracellular reducing environment (see, e.g., Worn et al., 275:2795-803 (2000)). An intrabody may be directed to a particular cellular location or organelle, for example by constructing a vector that comprises a polynucleotide sequence encoding the variable regions of an intrabody that may be operatively fused to a polynucleotide sequence that encodes a particular target antigen within the cell (see, e.g., Graus-Porta et al., 15:1182-91 (1995); Lener et al., 267:1196-205 (2000)). An intrabody may be introduced into a cell by a variety of techniques available to the skilled artisan including via a gene therapy vector, or a lipid mixture (e.g., Provectin\u2122 manufactured by Imgenex Corporation, San Diego, Calif.), or according to photochemical internalization methods.","The polynucleotides encoding an antibody or fragment thereof that specifically bind an antigen\/analyte of interest, as described herein, may be propagated and expressed according to any of a variety of well-known procedures for nucleic acid excision, ligation, transformation, and transfection using any number of known expression vectors. Thus, in certain embodiments expression of an antibody fragment may be preferred in a prokaryotic host, such as (see, e.g., Pluckthun et al., 1989 178:497-515). In certain other embodiments, expression of the antibody or a fragment thereof may be in a eukaryotic host cell, including yeast (e.g., , and ), fungi (e.g., cells such as those of ) animal cells (including mammalian cells) or plant cells. Examples of suitable animal cells include, but are not limited to, myeloma, COS, CHO, or hybridoma cells. Examples of plant cells include tobacco, corn, soybean, and rice cells. Antibodies that specifically bind to the antigen\/analyte of interest may be screened in assays for determining antibody affinity as described above, such as assays for determining Kd of an antibody.","Viral DNA-Packaging Motor Connector Protein Polypeptides","The polypeptides of the present invention include mutant viral DNA-packaging motor connector protein-derived polypeptides and fusion proteins having amino acid sequence regions that are identical or similar to sequences known in the art, or fragments or portions thereof. For example by way of illustration and not limitation, a mutant bacteriophage phi29 viral DNA-packaging motor connector protein [e.g., SEQ ID NO:1, Genbank Acc. No. ACE96033] or an engineered bacteriophage phi29 viral DNA-packaging motor connector protein-derived polypeptide fusion protein (e.g., SEQ ID NOS:31-35 and 41-45) is contemplated for use according to the instant invention, as are polypeptides having at least 80% similarity (preferably a 80% identity) and more preferably 90% similarity (more preferably a 90% identity) to the reported polypeptide and still more preferably a 95% similarity (still more preferably a 95% identity) to the herein disclosed polypeptides and to portions of such polypeptides, wherein such portions of a mutant or engineered phi29 viral DNA-packaging motor connector protein-derived polypeptide generally contain at least 150, 175, 200, 225, 250, 275 or more amino acids and more preferably at least 240, 260, 280, 285, 290, 295, 296, 297, 298, 299, 300, 301, 302, 303, 304, 305, 306, 307, 308, 309, 310, 311, 312, 313, 314, 315, 316, 317, 318, 319, 320, 321, 322, 323, 324, 325, 326, 327, 328, 329, 330 or more amino acids.","In like fashion, certain other embodiments contemplate other mutant double-stranded DNA bacteriophage virus motor connector proteins such as mutated forms of phage T4 DNA-packaging motor connector protein polypeptide [SEQ ID NO:7], lambda phage DNA-packaging motor connector protein polypeptide [SEQ ID NOS:8-11] (Accession numbers gi549295, gi6723246, gi15837315, gi16764273, phage SPP1 DNA-packaging motor connector protein polypeptide [SEQ ID NO:12] (Accession number P54309), phage P22 DNA-packaging motor connector protein polypeptide [SEQ ID NO:13] (Accession number AAA72961), phage P2 DNA-packaging motor connector protein polypeptide [SEQ ID NO:14] (Accession number NP046757), phage P3 DNA-packaging motor connector protein polypeptide (Nutter et al., 1972 10(3):560-2), phage T3 DNA-packaging motor connector protein polypeptide [SEQ ID NO:15] (Accession number CAA35152, phage T5 DNA-packaging motor connector protein polypeptide [SEQ ID NOS:16-19] (Accession numbers AAX12078, YP006980; AAS77191; AAU05287), and phage T7 DNA-packaging motor connector protein polypeptide [SEQ ID NO:20] (Accession number NP041995).","Certain embodiments of the present invention relate to viral DNA-packaging motor connector protein polypeptides, mutant viral DNA-packaging motor connector protein polypeptides and engineered viral DNA-packaging motor connector polypeptide fusion proteins, and certain embodiments relate to constructs encoding viral DNA-packaging motor connector proteins such as mutant viral DNA-packaging motor connector protein polypeptides and engineered viral DNA-packaging motor connector polypeptide fusion proteins, for example viral DNA-packaging motor connector protein fusion polypeptides containing a motor protein connector aperture domain as provided herein and also containing one or both of a flexibility domain and an affinity\/alignment domain as also provided herein, and in particular to methods for making a conductive channel-containing membrane using polypeptide products of recombinant constructs encoding such proteins that are capable of self-assembly into a homododecameric viral DNA-packaging motor connector protein and that may be expressed, for example and in certain related embodiments, as fragments, analogs and derivatives of such polypeptides.","Variants, fragments, derivatives, truncations, and the like of such mutant viral DNA-packaging motor connector protein polypeptides are also contemplated and are discussed in greater detail below, including routine methodologies for their production, structural characterization (e.g., for the presence of an aperture domain and either or both of a flexibility domain and an affinity alignment domain as provided herein) and functional testing (e.g., for their aperture-forming properties when incorporated into lipid membranes, as also described herein). According to certain preferred embodiments, a mutant viral DNA-packaging motor connector polypeptide for use in the compositions and methods disclosed herein may comprise an amino acid sequence that has at least 80 percent, 85 percent, 90 percent, 95 percent, 96, 97, 98, 99 or greater percent sequence identity to a double-stranded DNA bacteriophage DNA-packaging motor connector protein polypeptide such as a viral DNA-packaging motor connector protein polypeptide subunit as provided herein, for example by way illustration and not limitation, a polypeptide that comprises the amino acid sequence set forth in any one of SEQ ID NOS:1-20, 31-35 and 41-45.","The terms \u201cfragment,\u201d \u201cderivative\u201d and \u201canalog\u201d when referring to viral DNA-packaging motor connector proteins or polypeptides, refers to any mutant viral DNA-packaging motor connector protein-derived polypeptide described herein, or a fusion protein comprising such polypeptide, that retains essentially the same biological function or activity as such polypeptide. Thus, an analog includes a proprotein which can be activated by cleavage of the proprotein portion to produce an active viral DNA-packaging motor connector polypeptide, which in preferred embodiments may be incorporated into a membrane layer to form an aperture through which conductance can occur when an electrical potential is applied across the membrane and\/or may be capable of self-assembly into a homododecameric viral DNA-packaging motor connector protein such as may form such an aperture to obtain a conductive channel-containing membrane.","A fragment, derivative or analog of a viral DNA-packaging motor connector protein-derived polypeptide described herein, including polypeptides or fusion proteins or domains or portions thereof encoded by the cDNAs referred to herein and for which nucleotide coding sequences may be known to the art and\/or can be deduced from the polypeptide sequences disclosed herein, may be (i) one in which one or more of the amino acid residues are substituted with a conserved or non-conserved amino acid residue (preferably a conserved amino acid residue) and such substituted amino acid residue may or may not be one encoded by the genetic code, or (ii) one in which one or more of the amino acid residues includes a substituent group, or (iii) one in which additional amino acids are fused to the mutant viral DNA-packaging motor connector protein-derived polypeptide, including amino acids that are employed for detection or specific functional alteration of the mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide or a proprotein sequence. Such fragments, derivatives and analogs are deemed to be within the scope of those skilled in the art from the teachings herein.","As known in the art \u201csimilarity\u201d between two polypeptides is determined by comparing the amino acid sequence and conserved amino acid substitutes thereto of the polypeptide to the sequence of a second polypeptide. Fragments or portions of the nucleic acids encoding polypeptides according to the presently disclosed embodiments may be used to synthesize full-length nucleic acids encoding a mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide. As used herein, \u201c% identity\u201d refers to the percentage of identical amino acids situated at corresponding amino acid residue positions when two or more polypeptide are aligned and their sequences analyzed using a gapped BLAST algorithm (e.g., Altschul et al., 1997 25:3389) which weights sequence gaps and sequence mismatches according to the default weightings provided by the National Institutes of Health\/NCBI database (National Center for Biotechnology Information, Bethesda, Md.).","The term \u201cisolated\u201d means that the material is removed from its original environment (e.g., the natural environment if it is naturally occurring). For example, a naturally occurring nucleic acid or polypeptide present in a living animal or intact naturally occurring virus is not isolated, but the same nucleic acid or polypeptide, separated from some or all of the co-existing materials in the natural system, is isolated. Such nucleic acids could be part of a vector and\/or such nucleic acids or polypeptides could be part of a composition, and still be isolated in that such vector or composition is not part of its natural environment.","The term \u201cgene\u201d means the segment of DNA involved in producing a polypeptide chain; it includes regions preceding and following the coding region \u201cleader and trailer\u201d as well as intervening sequences (introns) between individual coding segments (exons).","As described herein, certain invention embodiments provide mutant or engineered viral DNA-packaging motor connector protein-derived polypeptides and fusion proteins encoded by nucleic acids that have the mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide coding sequence fused in frame to an additional fusion polypeptide encoding sequence to provide for expression of a mutant or an engineered viral DNA-packaging motor connector protein-derived polypeptide sequence fused to an additional functional fusion polypeptide sequence that permits, for example by way of illustration and not limitation, detection, functional alteration, isolation and\/or purification of the resulting fusion protein.","Such fusion proteins may permit functional alteration of a mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide by containing additional polypeptide sequences that influence behavior of the fusion product, for example, by altering the availability of protein surface-exposed charged and\/or hydrophobic amino acid side chains in such a manner as to alter (e.g., increase or decrease in a statistically relevant manner relative to an appropriate control) interactions with a membrane such as with amphipathic membrane components such as phospholipids in a membrane bilayer. According to non-limiting theory, in these and related embodiments the mutant or engineered viral DNA-packaging motor connector protein-derived fusion polypeptide may exhibit more stable incorporation into the membrane and\/or may form an aperture through which conductance can occur when an electrical potential is applied across the membrane in a manner that provides an aperture having desirable properties, such as increased aperture dimensions and\/or altered distribution of hydrophilic, hydrophobic, neutral and\/or charged amino acid side chains and\/or preservation of the conductive aperture structure despite truncation of the viral DNA-packaging motor connector protein sequence or addition of polypeptide domains (e.g., a flexibility domain or an affinity\/alignment domain as provided herein) that are not found in the wild-type DNA-packaging motor connector protein.","Determination of the three-dimensional structures of representative polypeptides (e.g., mutant or engineered DNA-packaging motor connector protein subunits as provided herein, for instance, a polypeptide fusion protein having a flexibility domain and\/or an affinity\/alignment domain as provided herein) may be made through routine methodologies such that substitution, addition, deletion or insertion of one or more amino acids with selected natural or non-natural amino acids can be virtually modeled for purposes of determining whether a so derived structural variant retains the space-filling properties of presently disclosed species. See, for instance, Donate et al., 1994 3:2378; Bradley et al., 309: 1868-1871 (2005); Schueler-Furman et al., 310:638 (2005); Dietz et al., 103:1244 (2006); Dodson et al., 450:176 (2007); Qian et al., 450:259 (2007). Some additional non-limiting examples of computer algorithms that may be used for these and related embodiments, such as for rational design of flexibility domains or affinity\/alignment domains as provided herein, include Desktop Molecular Modeler (See, for example, Agboh et al., 279, 40: 41650-57 (2004)), which allows for determining atomic dimensions from space-filling models (van der Waals radii) of energy-minimized conformations; GRID, which seeks to determine regions of high affinity for different chemical groups, thereby enhancing binding, Monte Carlo searches, which calculate mathematical alignment, and CHARMM (Brooks et al. (1983) 4:187-217) and AMBER (Weiner et al (1981) 106: 765), which assess force field calculations, and analysis (see also, Eisenfield et al. (1991) 261:C376-386; Lybrand (1991) 46:49-54; Froimowitz (1990) 8:640-644; Burbam et al. (1990) 7:99-111; Pedersen (1985) 61:185-190; and Kini et al. (1991) 9:475-488).","In one embodiment, the present invention provides truncated components (e.g., fragments of a mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide) for use in a mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide fusion protein, and in another embodiment the invention provides nucleic acids encoding a mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide fusion protein having such truncated components.","A truncated molecule may be any molecule that comprises less than a full length version of the molecule, for example, a truncated viral DNA-packaging motor protein polypeptide subunit. Truncated molecules provided by the present invention may include truncated biological polymers, and in preferred embodiments of the invention such truncated molecules may be truncated nucleic acid molecules or truncated polypeptides. Truncated nucleic acid molecules have less than the full length nucleotide sequence of a known or described nucleic acid molecule, where such a known or described nucleic acid molecule may be a naturally occurring, a synthetic or a recombinant nucleic acid molecule, so long as one skilled in the art would regard it as a full length molecule. Thus, for example, truncated nucleic acid molecules that correspond to a gene sequence contain less than the full length gene where the gene comprises coding and non-coding sequences, promoters, enhancers and other regulatory sequences, flanking sequences and the like, and other functional and non-functional sequences that are recognized as part of the gene. In another example, truncated nucleic acid molecules that correspond to a mRNA sequence contain less than the full length mRNA transcript, which may include various translated and non-translated regions as well as other functional and non-functional sequences.","In other preferred embodiments, truncated molecules are polypeptides that comprise less than the full length amino acid sequence of a particular protein or polypeptide component. As used herein \u201cdeletion\u201d has its common meaning as understood by those familiar with the art, and may refer to molecules that lack one or more of a portion of a sequence from either terminus or from a non-terminal region, relative to a corresponding full length molecule, for example, as in the case of truncated molecules provided herein. Truncated molecules that are linear biological polymers such as nucleic acid molecules or polypeptides may have one or more of a deletion from either terminus of the molecule or a deletion from a non-terminal region of the molecule, where such deletions may be deletions of 1-1500 contiguous nucleotide or amino acid residues, preferably 1-500 contiguous nucleotide or amino acid residues and more preferably 1-300 contiguous nucleotide or amino acid residues, including deletions of 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31-40, 41-50, 51-74, 75-100, 101-150, 151-200, 201-250 or 251-299 contiguous nucleotide or amino acid residues. In certain particularly preferred embodiments truncated nucleic acid molecules may have a deletion of 3-75 contiguous nucleotides. In certain other particularly preferred embodiments truncated polypeptide molecules may have a deletion of 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31-40, or 41-50 contiguous amino acids.","Alterations of the native amino acid sequence may be accomplished by any of a number of conventional methods. Mutations can be introduced at particular loci by synthesizing oligonucleotides containing a mutant sequence, flanked by restriction sites enabling ligation to fragments of the native sequence. Following ligation, the resulting reconstructed sequence encodes an analog having the desired amino acid insertion, substitution, or deletion.","Alternatively, oligonucleotide-directed site-specific mutagenesis procedures can be employed to provide an altered gene wherein predetermined codons can be altered by substitution, deletion or insertion. Exemplary methods of making such alterations are disclosed by Walder et al. (42:133, 1986); Bauer et al. (37:73, 1985); Craik (, Jan. 1985, 12-19); Smith et al. (, Jan. 1985, 12-19); Smith et al. (, Plenum Press, 1981); Kunkel (82:488, 1985); Kunkel et al. (154:367, 1987); and U.S. Pat. Nos. 4,518,584 and 4,737,462.","As an example, modification of DNA may be performed by site-directed mutagenesis of DNA encoding the protein combined with the use of DNA amplification methods using primers to introduce and amplify alterations in the DNA template, such as PCR splicing by overlap extension (SOE). Site-directed mutagenesis is typically effected using a phage vector that has single- and double-stranded forms, such as M13 phage vectors, which are well-known and commercially available. Other suitable vectors that contain a single-stranded phage origin of replication may be used (see, e.g., Veira et al., 15:3, 1987). In general, site-directed mutagenesis is performed by preparing a single-stranded vector that encodes the protein of interest. An oligonucleotide primer that contains the desired mutation within a region of homology to the DNA in the single-stranded vector is annealed to the vector followed by addition of a DNA polymerase, such as DNA polymerase I (Klenow fragment), which uses the double stranded region as a primer to produce a heteroduplex in which one strand encodes the altered sequence and the other the original sequence. The heteroduplex is introduced into appropriate bacterial cells and clones that include the desired mutation are selected. The resulting altered DNA molecules may be expressed recombinantly in appropriate host cells to produce the modified protein.","Equivalent DNA constructs that encode various additions or substitutions of amino acid residues or sequences, or deletions of terminal or internal residues or sequences not needed for biological activity are also encompassed by certain embodiments of the invention. For example, according to certain contemplated embodiments, sequences encoding Cys residues that are not desirable or essential for biological activity can be altered to cause the Cys residues to be deleted or replaced with other amino acids, preventing formation of incorrect or undesirable intramolecular disulfide bridges upon renaturation.","Modification of the polypeptide may be effected by any means known to those of skill in the relevant art. The preferred methods herein rely on modification of DNA encoding the fusion protein and expression of the modified DNA. DNA encoding one of the mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide fusions discussed above may be mutagenized using standard methodologies, including those described below. For example, cysteine residues that may otherwise facilitate multimer formation or that may mediate interactions with reactive groups present on analytes being detected by the herein described conductive channel-containing membrane or that may promote particular molecular conformations, can be deleted from a polypeptide or replaced, e.g., cysteine residues that are responsible for undesired aggregate formation. If necessary, the identity of cysteine residues that contribute to such aggregate formation may be determined empirically, by deleting and\/or replacing a cysteine residue and ascertaining whether the resulting protein forms aggregates in solutions containing physiologically acceptable buffers and salts. In addition, fragments of mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide fusions may be constructed and used, which in certain preferred embodiments includes fragments that comprise a transmembrane aperture-forming portion of such polypeptides. Design, production and testing of such fragments may be achieved based on the present disclosure and using art-known methodologies, including, for example, computer-modeled polypeptide design, recombinant expression, functional incorporation into membrane layers, and conductance testing, as discussed herein.","Conservative substitutions of amino acids are well known and may be made generally without altering the biological activity of the resulting mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide molecule. For example, such substitutions are generally made by interchanging within the groups of polar residues, charged residues, hydrophobic residues, small residues, and the like. If necessary, such substitutions may be determined empirically merely by testing the resulting modified protein for the ability to alter (i.e., increase or decrease in a statistically significant manner) e.g., dodecamer self-assembly, incorporation into a membrane layer, conductive aperture formation, and\/or analyte detection, in in vitro assays, such as those described herein.","The present invention further relates to nucleic acids which hybridize to mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide encoding polynucleotide sequences as provided herein, or their complements, as will be readily apparent to those familiar with the art, if there is at least 70%, preferably 80-85%, more preferably at least 90%, and still more preferably at least 95%, 96%, 97%, 98% or 99% identity between the sequences. The present invention particularly relates to nucleic acids which hybridize under stringent conditions to the mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide encoding nucleic acids referred to herein. As used herein, the term \u201cstringent conditions\u201d means hybridization will occur only if there is at least 90-95% and preferably at least 97% identity between the sequences. The nucleic acids which hybridize to mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide encoding nucleic acids referred to herein, in preferred embodiments, encode polypeptides which retain substantially the same biological function or activity (e.g., dodecameric self-assembly, incorporation into membrane layers, conductive aperture formation, ability to detect analyte) as the mutant or engineered viral DNA-packaging motor connector protein-derived polypeptides described herein.","As used herein, to \u201chybridize\u201d under conditions of a specified stringency is used to describe the stability of hybrids formed between two single-stranded nucleic acid molecules. Stringency of hybridization is typically expressed in conditions of ionic strength and temperature at which such hybrids are annealed and washed. Typically \u201chigh\u201d, \u201cmedium\u201d and \u201clow\u201d stringency encompass the following conditions or equivalent conditions thereto: high stringency: 0.1\u00d7SSPE or SSC, 0.1% SDS, 65\u00b0 C.; medium stringency: 0.2\u00d7SSPE or SSC, 0.1% SDS, 50\u00b0 C.; and low stringency: 1.0\u00d7SSPE or SSC, 0.1% SDS, 50\u00b0 C. As known to those having ordinary skill in the art, variations in stringency of hybridization conditions may be achieved by altering the time, temperature and\/or concentration of the solutions used for prehybridization, hybridization and wash steps, and suitable conditions may also depend in part on the particular nucleotide sequences of the probe used, and of the blotted, proband nucleic acid sample. Accordingly, it will be appreciated that suitably stringent conditions can be readily selected without undue experimentation where a desired selectivity of the probe is identified, based on its ability to hybridize to one or more certain proband sequences while not hybridizing to certain other proband sequences.","The nucleic acids of the present invention, also referred to herein as polynucleotides, may be in the form of RNA or in the form of DNA, which DNA includes cDNA, genomic DNA, and synthetic DNA. The DNA may be double-stranded or single-stranded, and if single stranded may be the coding strand or non-coding (anti-sense) strand. A coding sequence which encodes a mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide for use according to certain invention embodiments may contain sequence regions that are identical to portions of the viral DNA-packaging motor connector protein coding sequences known in the art, or may for such reasons have a different coding sequence, which, as a result of the redundancy or degeneracy of the genetic code, encodes the same regions of such a viral DNA-packaging motor connector protein polypeptide.","The nucleic acids which encode mutant or engineered viral DNA-packaging motor connector protein-derived polypeptides for use according to certain invention embodiments may include, but are not limited to: only the coding sequence for the mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide; the coding sequence for the mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide and additional coding sequence; the coding sequence for the mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide (and optionally additional coding sequence) and non-coding sequence, such as introns or non-coding sequences 5\u2032 and\/or 3\u2032 of the coding sequence for the mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide, which for example may further include but need not be limited to one or more regulatory nucleic acid sequences that may be a regulated or regulatable promoter, enhancer, other transcription regulatory sequence, repressor binding sequence, translation regulatory sequence or any other regulatory nucleic acid sequence. Thus, the term \u201cnucleic acid encoding\u201d or \u201cpolynucleotide encoding\u201d a mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide encompasses a nucleic acid which include's only coding sequence for a mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide as well as a nucleic acid which includes additional coding and\/or non-coding sequence(s).","Nucleic acids and oligonucleotides for use as described herein can be synthesized by any method known to those of skill in this art (see, e.g., WO 93\/01286, U.S. application Ser. No. 07\/723,454; U.S. Pat. No. 5,218,088; U.S. Pat. No. 5,175,269; U.S. Pat. No. 5,109,124). Identification of oligonucleotides and nucleic acid sequences for use in the presently disclosed embodiments involves methods well known in the art. For example, the desirable properties, lengths and other characteristics of useful oligonucleotides are well known. In certain embodiments, synthetic oligonucleotides and nucleic acid sequences may be designed that resist degradation by endogenous host cell nucleolytic enzymes by containing such linkages as: phosphorothioate, methylphosphonate, sulfone, sulfate, ketyl, phosphorodithioate, phosphoramidate, phosphate esters, and other such linkages that have proven useful in antisense applications (see, e.g., Agrwal et al., 28:3539-3542 (1987); Miller et al., 93:6657-6665 (1971); Stec et al., 26:2191-2194 (1985); Moody et al., 12:4769-4782 (1989); Uznanski et al., . (1989); Letsinger et al., 40:137-143 (1984); Eckstein, 54:367-402 (1985); Eckstein, 14:97-100 (1989); Stein In: , Cohen, Ed, Macmillan Press, London, pp. 97-117 (1989); Jager et al., 27:7237-7246 (1988)).","Host organisms include those organisms in which recombinant production of mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide products encoded by the recombinant constructs of certain presently disclosed invention embodiments may occur, such as bacteria (for example, ), yeast (for example, and ), insect cells and mammals, including in vitro and in vivo expression. Host organisms thus may include organisms for the construction, propagation, expression or other steps in the production of the compositions provided herein. Presently preferred host organisms are bacterial strains.","The DNA construct encoding the desired mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide is introduced into a plasmid for expression in an appropriate host. In certain preferred embodiments, the host is a bacterial host. The sequence encoding the ligand or nucleic acid binding domain is preferably codon-optimized for expression in the particular host. Thus, for example, if a mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide is expressed in bacteria, the codons would be optimized for bacterial usage. For small coding regions, the gene can be synthesized as a single oligonucleotide. For larger proteins, splicing of multiple oligonucleotides, mutagenesis, or other techniques known to those in the art may be used. The sequences of nucleotides in the plasmids that are regulatory regions, such as promoters and operators, are operationally associated with one another for transcription. The sequence of nucleotides encoding a mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide may also include DNA encoding a secretion signal, whereby the resulting peptide is a precursor protein. The resulting processed protein may be recovered from the periplasmic space or the fermentation medium.","In preferred embodiments, the DNA plasmids also include a transcription terminator sequence. As used herein, a \u201ctranscription terminator region\u201d is a sequence that signals transcription termination. The entire transcription terminator may be obtained from a protein-encoding gene, which may be the same or different from the inserted mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide encoding gene or the source of the promoter. Transcription terminators are optional components of the expression systems herein, but are employed in preferred embodiments.","The plasmids used herein include a promoter in operative association with the DNA encoding the protein or polypeptide of interest and are designed for expression of proteins in a suitable host as described above (e.g., bacterial, murine or human) depending upon the desired use of the plasmid. Suitable promoters for expression of proteins and polypeptides herein are widely available and are well known in the art. Inducible promoters or constitutive promoters that are linked to regulatory regions are preferred. Such promoters include, but are not limited to, the T7 phage promoter and other T7-like phage promoters, such as the T3, T5 and SP6 promoters, the trp, lpp, and lac promoters, such as the lacUV5, from ; the P10 or polyhedrin gene promoter of baculovirus\/insect cell expression systems (see, e.g., U.S. Pat. Nos. 5,243,041, 5,242,687, 5,266,317, 4,745,051, and 5,169,784) and inducible promoters from other eukaryotic expression systems. For expression of the proteins such promoters are inserted in a plasmid in operative linkage with a control region such as the lac operon.","Preferred promoter regions are those that are inducible and functional in . Examples of suitable inducible promoters and promoter regions include, but are not limited to: the lac operator responsive to isopropyl \u03b2-D-thiogalactopyranoside (IPTG; see Nakamura et al., 18:1109-1117, 1979); the metallothionein promoter metal-regulatory-elements responsive to heavy-metal (e.g., zinc) induction (see, e.g., U.S. Pat. No. 4,870,009 to Evans et al.); the phage T7lac promoter responsive to IPTG (see, e.g., U.S. Pat. No. 4,952,496; and Studier et al., 185:60-89, 1990) and the TAC promoter.","The plasmids may optionally include a selectable marker gene or genes that are functional in the host. A selectable marker gene includes any gene that confers a phenotype on bacteria that allows transformed bacterial cells to be identified and selectively grown from among a vast majority of untransformed cells. Suitable selectable marker genes for bacterial hosts, for example, include the ampicillin resistance gene (Amps), tetracycline resistance gene (Tcr) and the kanamycin resistance gene (Kang).","The plasmids may also include DNA encoding a signal for secretion of the operably linked protein. Secretion signals suitable for use are widely available and are well known in the art. Prokaryotic and eukaryotic secretion signals functional in may be employed. The presently preferred secretion signals include, but are not limited to, those encoded by the following genes: ompA, ompT, ompF, ompC, beta-lactamase, and alkaline phosphatase, and the like (von Heijne, 184:99-105, 1985). In addition, the bacterial pelB gene secretion signal (Lei et al., 169:4379, 1987), the phoA secretion signal, and the cek2 functional in insect cell may be employed. The most preferred secretion signal is the ompA secretion signal. Other prokaryotic and eukaryotic secretion signals known to those of skill in the art may also be employed (see, e.g., von Heijne, 184:99-105, 1985), as may also be a fusion domain comprising the YebF carrier protein (Zhang et al., 2006 24:100). Using the methods described herein, one of skill in the art can substitute secretion signals that are functional in yeast, insect or mammalian cells to secrete proteins from those cells.","Preferred plasmids for transformation of cells include the pET expression vectors (e.g., pET-11a, pET-12a-c, pET-15b; see U.S. Pat. No. 4,952,496; available from Novagen, Madison, Wis.). Other preferred plasmids include the pKK plasmids, particularly pKK 223-3, which contains the tac promoter (Brosius et al., 81:6929, 1984; Ausubel et al., 5,122,463, 5,173,403, 5,187,153, 5,204,254, 5,212,058, 5,212,286, 5,215,907, 5,220,013, 5,223,483, and 5,229,279). Plasmid pKK has been modified by replacement of the ampicillin resistance gene with a kanamycin resistance gene. (Available from Pharmacia; obtained from pUC4K, see, e.g., Vieira et al. (19:259-268, 1982; and U.S. Pat. No. 4,719,179.) Baculovirus vectors, such as pBlueBac (also called pJVETL and derivatives thereof), particularly pBlueBac III (see, e.g., U.S. Pat. Nos. 5,278,050, 5,244,805, 5,243,041, 5,242,687, 5,266,317, 4,745,051, and 5,169,784; available from Invitrogen, San Diego) may also be used for expression of the polypeptides in insect cells. Other plasmids include the pIN-IIIompA plasmids (see U.S. Pat. No. 4,575,013; see also Duffaud et al., 153:492-507, 1987), such as pIN-IIIompA2.","Preferably, the DNA molecule is replicated in bacterial cells, preferably in . The preferred DNA molecule also includes a bacterial origin of replication, to ensure the maintenance of the DNA molecule from generation to generation of the bacteria. In this way, large quantities of the DNA molecule can be produced by replication in bacteria. Preferred bacterial origins of replication include, but are not limited to, the f1-ori and col E1 origins of replication. Preferred hosts contain chromosomal copies of DNA encoding T7 RNA polymerase operably linked to an inducible promoter, such as the lacUV promoter (see U.S. Pat. No. 4,952,496). Such hosts include, but are not limited to, lysogens strains HMS174(DE3)pLysS, BL21(DE3)pLysS, HMS174(DE3) and BL21(DE3). Strain BL21(DE3) is preferred. The pLys strains provide low levels of T7 lysozyme, a natural inhibitor of T7 RNA polymerase.","The DNA molecules provided may also contain a gene coding for a repressor protein. The repressor protein is capable of repressing the transcription of a promoter that contains sequences of nucleotides to which the repressor protein binds. The promoter can be derepressed by altering the physiological conditions of the cell. For example, the alteration can be accomplished by adding to the growth medium a molecule that inhibits the ability to interact with the operator or with regulatory proteins or other regions of the DNA or by altering the temperature of the growth media. Preferred repressor proteins include, but are not limited to the lacI repressor responsive to IPTG induction, the temperature sensitive \u03bb cl857 repressor, and the like. The lacI repressor is preferred.","In general, recombinant constructs of the subject embodiments will also contain elements necessary for transcription and translation. In particular, such elements are preferred where the recombinant expression construct containing nucleic acid sequences encoding mutant or engineered viral DNA-packaging motor connector protein-derived polypeptides is intended for expression in a host cell or organism. In certain embodiments of the present invention, cell type preferred or cell type specific expression of a mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide encoding gene may be achieved by placing the gene under regulation of a promoter. The choice of the promoter will depend upon the cell type to be transformed and the degree or type of control desired. Promoters can be constitutive or active and may further be cell type specific, tissue specific, individual cell specific, event specific, temporally specific or inducible. Cell-type specific promoters and event type specific promoters are preferred. Examples of constitutive or nonspecific promoters include the SV40 early promoter (U.S. Pat. No. 5,118,627), the SV40 late promoter (U.S. Pat. No. 5,118,627), CMV early gene promoter (U.S. Pat. No. 5,168,062), and adenovirus promoter. In addition to viral promoters, cellular promoters are also amenable within the context of this invention. In particular, cellular promoters for the so-called housekeeping genes are useful. Viral promoters are preferred, because generally they are stronger promoters than cellular promoters. Promoter regions have been identified in the genes of many eukaryotes including higher eukaryotes, such that suitable promoters for use in a particular host can be readily selected by those skilled in the art.","Inducible promoters may also be used. These promoters include MMTV LTR (PCT WO 91\/13160), inducible by dexamethasone; metallothionein promoter, inducible by heavy metals; and promoters with cAMP response elements, inducible by cAMP. By using an inducible promoter, the nucleic acid sequence encoding a mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide may be delivered to a cell by the subject invention expression construct and will remain quiescent until the addition of the inducer. This allows further control on the timing of production of the gene product.","Event-type specific promoters are active or up-regulated only upon the occurrence of an event, such as tumorigenicity or viral infection. The HIV LTR is a well known example of an event-specific promoter. The promoter is inactive unless the tat gene product is present, which occurs upon viral infection. Some event-type promoters are also tissue-specific.","Additionally, promoters that are coordinately regulated with a particular cellular gene may be used. For example, promoters of genes that are coordinately expressed may be used when expression of a particular mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide-encoding gene is desired in concert with expression of one or more additional endogenous or exogenously introduced genes.","In addition to the promoter, repressor sequences, negative regulators, or tissue-specific silencers may be inserted to reduce non-specific expression of mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide encoding genes in certain situations, such as, for example, a host or host cell in which it may be desired transiently and\/or cell type-specifically or tissue-specifically or site-specifically to alter transmembrane electroconductivity as part of an experimental or therapeutic strategy. Multiple repressor elements may be inserted in the promoter region. Repression of transcription is independent on the orientation of repressor elements or distance from the promoter. One type of repressor sequence is an insulator sequence. Such sequences inhibit transcription (Dunaway et al., 17: 182-9, 1997; Gdula et al., 93:9378-83, 1996, Chan et al., 70: 5312-28, 1996; Scott and Geyer, 14:6258-67, 1995; Kalos and Fournier, 15:198-207, 1995; Chung et al., 74: 505-14, 1993) and will silence background transcription.","Repressor elements have also been identified in the promoter regions of the genes for type II (cartilage) collagen, choline acetyltransferase, albumin (Hu et al., 3(9):577-588, 1992), phosphoglycerate kinase (PGK-2) (Misuno et al., 119(2):293-297, 1992), and in the 6-phosphofructo-2-kinase\/fructose-2,6-bisphosphatase gene. (Lemaigre et al., 11(2):1099-1106.) Furthermore, the negative regulatory element Tse-1 has been identified in a number of liver specific genes, and has been shown to block cAMP response element-(CRE) mediated induction of gene activation in hepatocytes. (Boshart et al., 61(5):905-916, 1990).","In preferred embodiments, elements that increase the expression of the desired product are incorporated into the construct. Such elements include internal ribosome binding sites (IRES; Wang and Siddiqui, 203:99, 1995; Ehrenfeld and Semler, 203:65, 1995; Rees et al., 20:102, 1996; Sugimoto et al., 12:694, 1994). IRES increase translation efficiency. As well, other sequences may enhance expression. For some genes, sequences especially at the 5\u2032 end inhibit transcription and\/or translation. These sequences are usually palindromes that can form hairpin structures. Any such sequences in the nucleic acid to be delivered are generally deleted. Expression levels of the transcript or translated product are assayed to confirm or ascertain which sequences affect expression. Transcript levels may be assayed by any known method, including Northern blot hybridization, RNase probe protection and the like. Protein levels may be assayed by any known method, including ELISA, western blot, immunocytochemistry or other well known techniques.","Other elements may be incorporated into the mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide encoding constructs of the present invention. In preferred embodiments, the construct includes a transcription terminator sequence, including a polyadenylation sequence, splice donor and acceptor sites, and an enhancer. Other elements useful for expression and maintenance of the construct in mammalian cells or other eukaryotic cells may also be incorporated (e.g., origin of replication). Because the constructs are conveniently produced in bacterial cells, elements that are necessary for, or that enhance, propagation in bacteria are incorporated. Such elements include an origin of replication, a selectable marker and the like.","As provided herein, an additional level of controlling the expression of nucleic acids encoding mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide delivered to cells using the constructs of the present invention embodiments may be provided by simultaneously delivering two or more differentially regulated nucleic acid constructs. The use of such a multiple nucleic acid construct approach may permit coordinated regulation of viral DNA-packaging motor connector protein expression. Those familiar with the art will appreciate that multiple levels of regulated gene expression may be achieved in a similar manner by selection of suitable regulatory sequences, including but not limited to promoters, enhancers and other well known gene regulatory elements.","The present invention also relates in certain embodiments to vectors, and to constructs prepared from known vectors that include nucleic acids of the present invention, and in particular to \u201crecombinant expression constructs\u201d that include any nucleic acids encoding mutant or engineered viral DNA-packaging motor connector protein-derived polypeptides according to certain invention embodiments as provided herein; and to host cells which are genetically engineered with vectors and\/or constructs of the invention. Mutant or engineered viral DNA-packaging motor connector protein-derived polypeptides can be expressed in virtually any host cell under the control of appropriate promoters, depending on the nature of the construct (e.g., type of promoter, as described above), and on the nature of the desired host cell (e.g., whether postmitotic terminally differentiated or actively dividing; e.g., whether the expression construct occurs in a host cell as an episome or is integrated into the host cell genome). Appropriate cloning and expression vectors for use with prokaryotic and eukaryotic hosts are described by Sambrook, et al., , Third Edition, Cold Spring Harbor, N.Y., (2001).","Typically, the constructs are derived from plasmid vectors. A preferred construct is a modified pNASS vector (Clontech, Palo Alto, Calif.), which has nucleic acid sequences encoding an ampicillin resistance gene, a polyadenylation signal and a T7 promoter site. Other suitable mammalian expression vectors are well known (see, e.g., Ausubel et al., 1995; Sambrook et al., supra; see also, e.g., catalogues from Invitrogen, San Diego, Calif.; Novagen, Madison, Wis.; Pharmacia, Piscataway, N.J.; and others). Presently preferred constructs may be prepared that include a dihydrofolate reductase (DHFR) encoding sequence under suitable regulatory control, for promoting enhanced production levels of the mutant or engineered viral DNA-packaging motor connector protein-derived polypeptides, which levels result from gene amplification following application of an appropriate selection agent (e.g., methotrexate).","Generally, recombinant expression vectors will include origins of replication and selectable markers permitting transformation of the host cell, and a promoter derived from a highly-expressed gene to direct transcription of a downstream structural sequence, as described above. The heterologous structural sequence is assembled in appropriate phase with translation initiation and termination sequences. Thus, for example, the mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide encoding nucleic acids as provided herein may be included in any one of a variety of expression vector constructs as a recombinant expression construct for expression in a host cell.","The appropriate DNA sequence(s) may be inserted into the vector by a variety of procedures. In general, the DNA sequence is inserted into an appropriate restriction endonuclease site(s) by procedures known in the art. Standard techniques for cloning, DNA isolation, amplification and purification, for enzymatic reactions involving DNA ligase, DNA polymerase, restriction endonucleases and the like, and various separation techniques are those known and commonly employed by those skilled in the art. A number of standard techniques are described, for example, in Ausubel et al. (2004 , John Wiley & Sons, Inc., Boston, Mass.); Sambrook et al. (2001 , Third Ed., Cold Spring Harbor Laboratory, Plainview, N.Y.); Maniatis et al. (1982 Molecular Cloning, Cold Spring Harbor Laboratory, Plainview, N.Y.); Glover (Ed.) (1985 , IRL Press, Oxford, UK); Hames and Higgins (Eds.), (1985 , IRL Press, Oxford, UK); and elsewhere.","The DNA sequence in the expression vector is operatively linked to at least one appropriate expression control sequences (e.g., a constitutive promoter or a regulated promoter) to direct mRNA synthesis. Representative examples of such expression control sequences include promoters of eukaryotic cells or their viruses, as described above. Promoter regions can be selected from any desired gene using CAT (chloramphenicol transferase) vectors or other vectors with selectable markers. Eukaryotic promoters include CMV immediate early, HSV thymidine kinase, early and late SV40, LTRs from retrovirus, and mouse metallothionein-I. Selection of the appropriate vector and promoter is well within the level of ordinary skill in the art, and preparation of certain particularly preferred recombinant expression constructs comprising at least one promoter or regulated promoter operably linked to a nucleic acid encoding a mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide is described herein.","Transcription of the DNA encoding the polypeptides of the present invention by higher eukaryotes may be increased by inserting an enhancer sequence into the vector. Enhancers are cis-acting elements of DNA, usually about from 10 to 300 bp that act on a promoter to increase its transcription. Examples including the SV40 enhancer on the late side of the replication origin by 100 to 270, a cytomegalovirus early promoter enhancer, the polyoma enhancer on the late side of the replication origin, and adenovirus enhancers.","As provided herein, in certain embodiments the vector may be a viral vector such as a retroviral vector. (Miller et al., 1989 7:980; Coffin and Varmus, 1996 Retroviruses, NY.) For example, retroviruses from which the retroviral plasmid vectors may be derived include, but are not limited to, Moloney Murine Leukemia Virus, spleen necrosis virus, retroviruses such as Rous Sarcoma Virus, Harvey Sarcoma virus, avian leukosis virus, gibbon ape leukemia virus, human immunodeficiency virus, adenovirus, Myeloproliferative Sarcoma Virus, and mammary tumor virus.","Retroviruses are RNA viruses which can replicate and integrate into the genome of a host cell via a DNA intermediate. This DNA intermediate, or provirus, may be stably integrated into the host cell DNA. According to certain embodiments of the present invention, an expression construct may comprise a retrovirus into which a foreign gene that encodes a foreign protein is incorporated in place of normal retroviral RNA. When retroviral RNA enters a host cell coincident with infection, the foreign gene is also introduced into the cell, and may then be integrated into host cell DNA as if it were part of the retroviral genome. Expression of this foreign gene within the host results in expression of the foreign protein.","Most retroviral vector systems which have been developed for gene therapy are based on murine retroviruses. Such retroviruses exist in two forms, as free viral particles referred to as virions, or as proviruses integrated into host cell DNA. The virion form of the virus contains the structural and enzymatic proteins of the retrovirus (including the enzyme reverse transcriptase), two RNA copies of the viral genome, and portions of the source cell plasma membrane containing viral envelope glycoprotein. The retroviral genome is organized into four main regions: the Long Terminal Repeat (LTR), which contains cis-acting elements necessary for the initiation and termination of transcription and is situated both 5\u2032 and 3\u2032 of the coding genes, and the three coding genes gag, pol, and env. These three genes gag, pol, and env encode, respectively, internal viral structures, enzymatic proteins (such as integrase), and the envelope glycoprotein (designated gp70 and p15e) which confers infectivity and host range specificity of the virus, as well as the \u201cR\u201d peptide of undetermined function.","Separate packaging cell lines and vector producing cell lines have been developed because of safety concerns regarding the uses of retroviruses, including their use in expression constructs as provided by the present invention. Briefly, this methodology employs the use of two components, a retroviral vector and a packaging cell line (PCL). The retroviral vector contains long terminal repeats (LTRs), the foreign DNA to be transferred and a packaging sequence (y). This retroviral vector will not reproduce by itself because the genes which encode structural and envelope proteins are not included within the vector genome. The PCL contains genes encoding the gag, pol, and env proteins, but does not contain the packaging signal \u201cy\u201d. Thus, a PCL can only form empty virion particles by itself. Within this general method, the retroviral vector is introduced into the PCL, thereby creating a vector-producing cell line (VCL). This VCL manufactures virion particles containing only the retroviral vector's (foreign) genome, and therefore has previously been considered to be a safe retrovirus vector for therapeutic use.","\u201cRetroviral vector construct\u201d refers to an assembly which is, within preferred embodiments of the invention, capable of directing the expression of a sequence(s) or gene(s) of interest, such as mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide encoding nucleic acid sequences. Briefly, the retroviral vector construct must include a 5\u2032 LTR, a tRNA binding site, a packaging signal, an origin of second strand DNA synthesis and a 3\u2032 LTR. A wide variety of heterologous sequences may be included within the vector construct, including for example, a sequence which encodes a protein (e.g., a desired supplementary gene or a replacement gene), or which is itself useful as a transcribed molecule (e.g., as a ribozyme or antisense sequence).","Retroviral vector constructs of the present invention may be readily constructed from a wide variety of retroviruses, including for example, B, C, and D type retroviruses as well as spumaviruses and lentiviruses (see, e.g., RNA Tumor Viruses, Second Edition, Cold Spring Harbor Laboratory, 1985). Such retroviruses may be readily obtained from depositories or collections such as the American Type Culture Collection (\u201cATCC\u201d; Rockville, Md.), or isolated from known sources using commonly available techniques. Any of the above retroviruses may be readily utilized in order to assemble or construct retroviral vector constructs, packaging cells, or producer cells of the present invention given the disclosure provided herein, and standard recombinant techniques (e.g., Sambrook et al, 2d ed., Cold Spring Harbor Laboratory Press, 1989; Kunkle, 82:488, 1985).","Suitable promoters for use in viral vectors generally may include, but are not limited to, the retroviral LTR; the SV40 promoter; and the human cytomegalovirus (CMV) promoter described in Miller, et al., 7:980-990 (1989), or any other promoter (e.g., cellular promoters such as eukaryotic cellular promoters including, but not limited to, the histone, pol III, and \u03b2-actin promoters). Other viral promoters which may be employed include, but are not limited to, adenovirus promoters, thymidine kinase (TK) promoters, and B19 parvovirus promoters. The selection of a suitable promoter will be apparent to those skilled in the art from the teachings contained herein, and may be from among either regulated promoters or promoters as described above.","As described above, the retroviral plasmid vector is employed to transduce packaging cell lines to form producer cell lines. Examples of packaging cells which may be transfected include, but are not limited to, the PE501, PA317, \u03c8-2, \u03c8-AM, PA12, T19-14X, VT-19-17-H2, \u03c8CRE, \u03c8CRIP, GP+E-86, GP+envAm12, and DAN cell lines as described in Miller, 1:5-14 (1990). The vector may transduce the packaging cells through any means known in the art. Such means include, but are not limited to, electroporation, the use of liposomes, and CaPOprecipitation. In one alternative, the retroviral plasmid vector may be encapsulated into a liposome, or coupled to a lipid, and then administered to a host.","The producer cell line generates infectious retroviral vector particles which include the nucleic acid sequence(s) encoding the mutant or engineered viral DNA-packaging motor connector protein-derived polypeptides or fusion proteins. Such retroviral vector particles then may be employed, to transduce eukaryotic cells, either in vitro or in vivo. The transduced eukaryotic cells will express the nucleic acid sequence(s) encoding the mutant or engineered polypeptide or fusion protein. Eukaryotic cells which may be transduced include, in preferred embodiments, smooth muscle cells (e.g., vascular smooth muscle cells including arterial smooth muscle cells and venous smooth muscle cells, gastrointestinal tract smooth muscle cells, respiratory tract smooth muscle cells, urogenital tract smooth muscle cells), fibroblasts, myofibroblasts, chondrocytes, pericytes, glial cells, glioma cells, macrophages, and endothelial cells, but the invention is not intended to be so limited such that cells to be transduced may in certain embodiments also include, for example, embryonic stem cells, as well as hematopoietic stem cells, hepatocytes, fibroblasts, circulating peripheral blood mononuclear and polymorphonuclear cells including myelomonocytic cells, lymphocytes, myoblasts, tissue macrophages, dendritic cells, Kupffer cells, lymphoid and reticuloendothelia cells of the lymph nodes and spleen, keratinocytes, endothelial cells, and bronchial epithelial cells.","As another example of an embodiment of the invention in which a viral vector is used to prepare the recombinant mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide encoding expression construct, in one preferred embodiment, host cells transduced by a recombinant viral construct directing the expression of mutant or engineered polypeptides may produce viral particles containing expressed mutant or engineered polypeptides that are derived from portions of a host cell membrane incorporated by the viral particles during viral budding.","In another embodiment, the present invention relates to host cells containing the above described recombinant mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide encoding expression constructs. Host cells are genetically engineered (transduced, transformed or transfected) with the vectors and\/or expression constructs of these and related invention embodiments which may be, for example, a cloning vector, a shuttle vector or an expression construct. The vector or construct may be, for example, in the form of a plasmid, a viral particle, a phage, etc. The engineered host cells can be cultured in conventional nutrient media modified as appropriate for activating promoters, selecting transformants or amplifying particular genes such as genes encoding mutant or engineered viral DNA-packaging motor connector protein-derived polypeptides. The culture conditions for particular host cells selected for expression, such as temperature, pH and the like, will be readily apparent to the ordinarily skilled artisan.","The host cell can be a higher eukaryotic cell, such as a mammalian cell, or a lower eukaryotic cell, such as a yeast cell, or the host cell can be a prokaryotic cell, such as a bacterial cell. Representative examples of appropriate host cells according to the present invention include, but need not be limited to, bacterial cells, such as ; fungal cells, such as yeast; insect cells, such as S2 and Sf9; animal cells, such as CHO, COS or 293 cells; adenoviruses; plant cells, or any suitable cell already adapted to in vitro propagation or so established de novo. The selection of an appropriate host is deemed to be within the scope of those skilled in the art from the teachings herein.","Various mammalian cell culture systems can also be employed to express recombinant protein. Examples of mammalian expression systems include the COS-7 lines of monkey kidney fibroblasts, described by Gluzman, 23:175 (1981), and other cell lines capable of expressing a compatible vector, for example, the C127, 3T3, CHO, HeLa and BHK cell lines. Mammalian expression vectors will comprise an origin of replication, a suitable promoter and enhancer, and also any necessary ribosome binding sites, polyadenylation site, splice donor and acceptor sites, transcriptional termination sequences, and 5\u2032 flanking nontranscribed sequences, for example as described herein regarding the preparation of mutant or engineered viral DNA-packaging motor connector protein-derived polypeptide encoding expression constructs. DNA sequences derived from the SV40 splice, and polyadenylation sites may be used to provide the required nontranscribed genetic elements. Introduction of the construct into the host cell can be effected by a variety of methods with which those skilled in the art will be familiar, including but not limited to, for example, calcium phosphate transfection, DEAE-Dextran mediated transfection, or electroporation (Davis et al., 1986 ).","As used herein and in the appended claims, the singular forms \u201ca,\u201d \u201cand,\u201d and \u201cthe\u201d include plural referents unless the context clearly dictates otherwise. Thus, for example, reference to \u201ca polypeptide\u201d or \u201cthe polypeptide\u201d includes reference to one or more polypeptides (i.e., a plurality of polypeptides) and equivalents thereof known to those skilled in the art, and so forth, unless clearly indicated otherwise. Reference throughout this specification to \u201cone embodiment,\u201d or \u201can embodiment,\u201d or \u201cin another embodiment,\u201d or \u201cin some embodiments\u201d means that a particular referent feature, structure, or characteristic described in connection with the embodiment is included in at least one embodiment. Thus, the appearance of the phrases \u201cin one embodiment,\u201d or \u201cin an embodiment,\u201d or \u201cin another embodiment\u201d in various places throughout this specification are not necessarily all referring to the same embodiment. Furthermore, the particular features, structures, or characteristics may be combined in any suitable manner in one or more embodiments.","Throughout this specification, unless the context requires otherwise, the words \u201ccomprise\u201d, \u201ccomprises\u201d and \u201ccomprising\u201d will be understood to imply the inclusion of a stated step or element or group of steps or elements but not the exclusion of any other step or element or group of steps or elements. By \u201cconsisting of\u201d is meant including, and limited to, whatever follows the phrase \u201cconsisting of.\u201d Thus, the phrase \u201cconsisting of\u201d indicates that the listed elements are required or mandatory, and that no other elements may be present. By \u201cconsisting essentially of\u201d is meant including any elements listed after the phrase, and limited to other elements that do not interfere with or contribute to the activity or action specified in the disclosure for the listed elements. Thus, the phrase \u201cconsisting essentially of\u201d indicates that the listed elements are required or mandatory, but that no other elements are required and may or may not be present depending upon whether or not they affect the activity or action of the listed elements.","The practice of the present invention will employ, unless indicated specifically to the contrary, conventional methods of chemistry, biochemistry, organic chemistry, molecular biology, microbiology, recombinant DNA techniques, genetics, immunology, cell biology, stem cell protocols, cell culture and transgenic biology that are within the skill of the art, many of which are described below for the purpose of illustration. Such techniques are explained fully in the literature. See, e.g., Sambrook, et al., (3Edition, 2001); Sambrook, et al., (2Edition, 1989); Maniatis et al., (1982); Ausubel et al., (John Wiley and Sons, updated July 2008); , Greene Pub. Associates and Wiley-Interscience; Glover, , vol. I & II (IRL Press, Oxford, 1985); Anand, , (Academic Press, New York, 1992); Guthrie and Fink, (Academic Press, New York, 1991); (N. Gait, Ed., 1984); (B. Hames & S. Higgins, Eds., 1985); (B. Hames & S. Higgins, Eds., 1984); (R. Freshney, Ed., 1986); Perbal, (1984); Fire et al., (Cambridge University Press, Cambridge, 2005); Schepers, (Wiley-VCH, 2005); Engelke, (): & (DNA Press, 2003); Gott, (Methods in Molecular Biology; Human Press, Totowa, N.J., 2004); Sohail, (CRC, 2004); Clarke and Sanseau, & (Nuts & Bolts series; DNA Press, 2006); (IRL Press, 1986); the treatise, (Academic Press, Inc., N.Y.); (J. H. Miller and M. P. Calos eds., 1987, Cold Spring Harbor Laboratory); Harlow and Lane, , (Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1998); (Mayer and Walker, eds., Academic Press, London, 1987); , Volumes I-IV (D. M. Weir and C C Blackwell, eds., 1986); Roitt, 6th Edition, (Blackwell Scientific Publications, Oxford, 1988); (Methods in Molecular Biology) (Kurstad Turksen, Ed., 2002); (Methods in Molecular Biology) (K. Turksen, Ed., 2006); (Methods in Molecular Biology) (K. Turksen, Ed., 2006); (Methods in Molecular Biology) (K. Turksen Ed., 2006); (Methods in Molecular Biology) (D. J. Prockop, D. G. Phinney, and B. A. Bunnell Eds., 2008); (Methods in Molecular Medicine) (C. A. Klug, and C. T. Jordan Eds., 2001); (Methods in Molecular Biology) (K. D. Bunting Ed., 2008) (Methods in Molecular Biology) (L. P. Weiner Ed., 2008); Hogan et al., (2Edition, 1994); Nagy et al., (3Edition, 2002), and (), 4th Ed., (Univ. of Oregon Press, Eugene, Oreg., 2000).","This example describes redesign of the dsDNA phage phi29 DNA-packaging motor connector protein to include distinct regions of hydrophilicity.","Materials.","The phospholipids 1,2-diphytanoyl-sn glycerol-3-phosphocholine (DPhPC) and 1,2-Dioleoyl-sn-Glycero-3-Phosphocholine (DOPC) were purchased from Avanti Polar Lipids (Alabaster, Ala.). N-(7-nitrobenz-2-oxa-1,3-diazol-4-yl)-1,2-dihexadecanoyl-sn-glycero-3-phosphoethanolamine, triethylammonium salt (NBD-PE) were purchased from Invitrogen. n-Decane and chloroform were purchased from Fisher and TEDIA, respectively.","Reengineering of Phi29 Connector.","The construction of the plasmid for the expression of the connector protein and the assembly of the dodecameric connector have been reported previously (Guo et al., 5, 856-863 (2005)). The subsequent terminal modifications of the connectors were also essentially as described except for the use of the specified flexibility and affinity\/alignment domains as disclosed herein (Cai et al., 4, 8-18 (2008); Sun et al., 34(19), 5482-5490 (2006); Robinson et al., 34, 2698-2709 (2006)). Briefly, the modification of one of the plasmids was, for example, achieved by a two-step PCR. First, the primer pair F1-R1 was used to amplify the GP10 gene The first PCR product was used as a template for a second step PCR with primer pair F1 and R2, which contained affinity Tags (His-tag and\/or Strep-II tag, WSHPQFEK, SEQ ID NO:22) as well as the restriction sites for NdeI and XhoI, respectively. The second PCR product was digested with NdeI\/XhoI and ligated into the NdeI\/XhoI sites of the vector pET-21a(+) (Novagen) to generate the plasmid.","Expression and Purification of the Connector.","Plasmid pETgp10-C-strep-II or -C-Hiswas transformed into the strain HMS174 (DE3) for protein expression. A volume of 10 mL of the culture was incubated overnight at 37\u00b0 C. in Luria-Bertani (LB) medium containing 100 \u03bcg\/mL ampicillin and agitated at 250 rpm. A volume of 5 mL of the culture was inoculated into 500 mL medium and induced with 0.5 mM IPTG when the cell density reached 0.5-0.6 unit at OD. Cells were harvested 3 hours after IPTG induction by centrifugation at 5000\u00d7g for 20 min in a Beckman JS-7.5 rotor and then stored at \u221270\u00b0 C. before use.","The Strep-II-tagged connector was purified by affinity chromatography with Strep-Tactin (IBA, St. Louis, Mo.). Cells were resuspended with Buffer W (15% glycerol, 0.5 M NaCl, 1 mM EDTA, 100 mM Tris-HCl, pH 8.0), and the cleared lysate was loaded onto a Strep-Tactin Sepharose Column and washed with Buffer W. The Strep-II-tagged connector was eluted by buffer E (15% glycerol, 0.5 M NaCl, 1 mM EDTA, 2.5 mM desthiobiotin, 100 mM Tris-HCl, pH 8.0). The His-tagged connector was purified with Nickel affinity chromatography (Novagen) essentially as described in Robinson et al. (2006).","In Vitro DNA Packaging Assay Using Phi29 Motor.","The purification of procapsids, GP16, and DNA-GP3 (Lee et al., 69, 5024-5032 (1995); Guo et al., 83, 3505-3509 (1986)), and the procedure for DNA packaging using the phi29 DNA packaging motor in vitro have been previously described (Guo et al., 1986; Lee et al., 69, 5018-5023 (1995). Briefly, a volume of 10 \u03bcL of 0.3 \u03bcg\/\u03bcL of purified normal procapsids or reengineered procapsid C-strep were mixed with 100 ng of pRNA in TMS (100 mM Tris-HCl, pH 8.0, 10 mM MgCl, 100 mM NaCl) for 30 min at room temperature. The presence of Mg in TMS buffer promoted binding of pRNA to the procapsid. These pRNA-enriched procapsids were mixed with 3 \u03bcL reaction buffer (10 mM ATP\/6 mM spermidine\/3 mM \u03b2-mercaptoethanol in TMS), 100 ng of DNA-gp3, and 6 \u03bcL of 0.5 \u03bcg\/\u03bcL DNA packaging enzyme gp16. The mixtures were then incubated for 30 min at room temperature. DNA packaging efficiency was verified by agarose gel electrophoresis using a DNase protection assay (Guo et al., 185, 395-400 (1991)). The packaged DNA, which was protected by the procapsid, was shown on the gel.","The biological activity of the DNA-filled capsid was tested for its ability to be converted into infectious phi29 virion. After 30 min of DNA packaging, the neck, tail, and morphogenic proteins were added to complete the assembly of the infectious virions, which were then assayed by standard plaque formation (Guo et al., 1986; Lee et al., 69, 5018-5023 (1995)).","Fluorescence Labeling of the Connector.","The connector was labeled using a Fluoro Tag\u2122 FITC conjugation kit (Sigma, St. Louis, Mo.). Buffer E was changed to a sodium carbonate-bicarbonate buffer using column chromatography. FITC solution was added drop wise into the connector solution and incubated for 2 hours with gentle stirring. Free FITC was removed by column chromatography, and FITC-labeled connector was eluted with PBS. FITC-connector was identified by SDS-PAGE. Labeling efficiency was determined from fluorescein\/protein molar ratio and measured by UV-Vis spectrophotometry.","Preparation of Giant Lipid Vesicles Containing the Reengineered Connector.","To prepare the fluorescent giant lipid vesicles, 1 mL of 1 mg\/mL DOPC or DPhPC and 1% (molar ratio) NBD-PE were mixed in a vial. Chloroform was evaporated by a gentle stream of nitrogen gas, and the lipid vial was further dried in a desiccator overnight. To rehydrate the lipid film, 2 mL of 200 mM-300 mM sucrose was used as exemplary osmotic agent to bud vesicles off the glass and into the solution. The vial was then covered with parafilm and stored overnight. An aliquot was taken from the middle of the solution and then transferred into a Petri dish. After settling, the vesicles were observed with epi-fluorescence microscopy ().","Incorporation of the viral DNA-packaging motor protein connector into giant vesicles was accomplished as described above, except the NBD-PE was omitted. A volume of 100 \u03bcL of FITC-labeled reengineered connectors was added to the above dehydrated lipid with a final lipid:connector mole ratio of 75:1 (or as low as 4000:1 to 16000:1 for BLM experiments) ().","Insertion of the Connector into Planar Bilayer Lipid Membrane.","A two-step approach was used to incorporate the connector into the planar bilayer lipid membrane (BLM). The first step was the preparation of unilamellar lipid vesicles containing the reengineered connector as described above. The next step was to fuse the extruded liposome into a planar BLM (). The fluidity of the lipid bilayer was demonstrated by FRAP (Fluorescence Recovery After Photobleaching) (). An excitation light was focused continuously on the bilayer to bleach the dye. The photobleached area appeared dark. But after the light was switched off, the appearance of fluorescence gradually recovered due to the return by diffusion of the fluorescent lipid into the photobleached area.","A standard BLM chamber (BCH-1A from Eastern Scientific LLC, Rockville, Md.) was utilized to form horizontal BLMs. A thin Teflon film with an aperture of 70-120 \u03bcm (TP-01 from Eastern Sci. LLC) or 180-250 \u03bcm (TP-02 from Eastern Sci. LLC) in diameter was used as a partition to separate the chamber into cis- (working volume 250 \u03bcL) and trans- (working volume 2.5 mL) compartments. After the aperture was pre-painted twice with 0.5 \u03bcL 3% (w\/v) DPhPC n-decane solution to ensure the complete coating of the entire edge of the aperture, these compartments were filled with conducting buffers (5 mM Tris\/pH 7.9, TMS, or 5 mM HEPES\/pH 7.9, with varying concentration of NaCl or KCl).","Formation of the bilayer membrane on the partition was a precursor step for viral DNA-packaging motor protein connector insertion into the bilayer (). The occurrence of successful viral DNA-packaging motor protein connector insertions was about 47-83% in the studies described here (approximately 280 separate BLM experiments).","For single conductance measurements, the giant liposome\/connector complex prepared earlier was extruded using a polycarbonate membrane with pore size of 200 nm or 400 nm to generate small unilamellar liposomes. This liposome stock solution was further diluted by 10-20 fold for the BLM experiments before use. For insertion of viral DNA-packaging motor protein connectors, 0.5-2 \u03bcL of the diluted liposome solution was loaded into the cis-chamber.","Conductance was measured in two ways: the first was derived at specific but constant holding potentials, and the second from the slope of the current trace induced by a scanning potential starting at \u2212100 mV and ramping to 100 mV after incorporation of phi29 GP10 viral DNA-packaging motor protein connector into the lipid membrane ().","Q-PCR Analysis.","For Q-PCR analysis, the connector\/liposome complexes were added to the cis-side (working volume 500 \u03bcL). 141-bp DNA was added to the trans-side with a final concentration of 25 nM. As a negative control, the DNA was added without the addition of connector\/liposome complexes. A potential of \u221295 mV was applied and samples were collected from the cis-side at 30 min intervals for Q-PCR analysis. DNA concentration was determined by a DU530 UV\/Vis spectrometer (Beckman Coulter, Fullerton, Calif.). Absolute quantification was used to determine the copy number of DNA in samples collected. Standard curves were constructed using the 141-bp DNA with 10 fold dilution of known concentration (). Each dilution was assayed in triplicate. iQ\u2122 SYBR Green Supermix (Bio-Rad, Hercules, Calif.) was used for the Q-PCR reaction. Q-PCR was carried out in the iCycler iQ\u2122 multicolor real-time PCR detection system (Bio-Rad). The sequences for forward and reverse oligonucleotide primers corresponding to the DNA template were 5\u2032-TAA TAC GAC TCA CTA TTA GAA CGG CAT CAA GGT GAA CTC AAG ATT TTG TAT GTT GGG GAT TA-3\u2032 [SEQ ID NO:48] and 5\u2032-AAG AAC GGC ATC AAG GTG AAC TTC AAG ATA ATT GAC AGC AGG CAA TCA AC-3\u2032 [SEQ ID NO:49], respectively (oligos were purchased from Integrated DNA Technologies, Inc. (\u201cIDT\u201d), Coralville, Iowa).","This example describes insertion into liposomes and a lipid bilayer of the modified viral DNA-packaging motor connector protein that was designed and prepared as described above, including in Example 1, and characterization of the resulting conductive channel. The presence of the channel across the lipid bilayer that was formed by the connector protein was confirmed by single channel conductance measurements and translocation of dsDNA.","Separation and Detection of Liposome\/Connector Complex by Sucrose Gradient Sedimentation.","A 5-20% linear sucrose gradient sedimentation in TMS (50 mM Tris, pH 8.0, 100 mM NaCl, 10 mM MgCl) (Guo et al., 1986) was performed to separate the liposome\/connector complexes from the free connector. A 0.1 mL sample was loaded at the top of the 5-mL centrifugation tube. After spinning in a Beckman L-80 ultracentrifuge at 27,000 rpm for 30 min at 20\u00b0 C. in a SW55 rotor, fractions were collected from the bottom of the tube and analyzed by 10% SDS-PAGE ().","Filtration.","The proteoliposomes were filtered with a 0.45 \u03bcm cellulose acetate membrane (Life Science Products, Inc.) to separate the free connector from the liposome\/connector complex. A volume of 500 \u03bcL of connector-incorporating DOPC liposome vesicles in 200 mM sucrose solution was added to the filtration tube and filled with the same solution. Less than 200 \u03bcL of liquid was retained after spinning at 3000\u00d7g for 15 min. This microfiltration process was repeated five times. The liposomes\/connector complexes retained on the top of the filter were imaged by fluorescence microscopy ().","Preparation of dsDNA for Translocation Experiments.","A 35-bp dsDNA was prepared by annealing two single stranded DNA, 5\u2032-TTA TAG GGA TAG TTG TAA GCT AAA GAA TAC GTT AC-3\u2032 (Integrated DNA Technologies, Inc. (\u201cIDT\u201d), Coralville, Iowa) and 5\u2032-GTA ACG TAT TCT TTA GCT TAC AAC TAT CCC TAT AA-3\u2032 (IDT). The annealing was performed at 65\u00b0 C. for 3 min and the sample was incubated at room temperature for 2 hrs. A 16% PAGE gel was used to purify the dsDNA. After releasing the DNA from the gel, it was concentrated by ethanol precipitation.","A linear 5.5 kilo-base plasmid DNA was prepared from an in-house constructed circular plasmid, Cx43, with a blunt-end cut by an endonuclease, EcoRV. After purification with a QIAGEN MiniElute Reaction Cleaning-up kit (QIAGEN), the DNA was added to trans-side directly for DNA translocation experiments.","Electrophysiological Measurements.","A pair of Ag\/AgCl electrodes connected directly to the head-stage of a current amplifier were used to measure the current traces across the bilayer lipid membrane, and the trace was recorded using an Axopatch 200B patch clamp amplifier coupled with the Axon DigiData 1322A or Axon DigiData 1440 analog-digital converter (Axon Instruments, Inc., Union City, Calif.). All the voltages reported are those of the trans-compartment. Data were low-pass filtered at a frequency of 1 kHz and acquired at sampling frequency of 2 kHz, if not specified. The PClamp 9.1 software (Axon Instruments) was used to collect the data, and the software Clampfit was used for data analysis.","Double-Stranded DNA Translocation Experiments.","In the DNA translocation experiments, DNA was added to the trans-side, if not specifically stated. C-Histagged connector (viral DNA-packaging motor protein connector having C-terminal hexahistidine modification) was used to evaluate translocation events. TMS\/1M NaCl buffer was used as DNA translocation buffer. Two methods were used to add DNA to the chamber for the translocation experiments. Method 1: DNA was added under a voltage of 0 mV after connector insertion. When the voltage was switched back, the DNA moved towards connector channels by free diffusion of DNA and applied voltage; Method 2: DNA was premixed with buffer completely in the chamber before connector insertion. The DNA movement relied mainly on applied voltage. All experiments were performed using Method 1 unless otherwise specified.","Modifying the Phi29 DNA-Packaging Motor Connector Protein.","In general, membrane pores and ion channels contain a hydrophobic domain which anchors the protein in the membrane. Analysis of the surface charge of the phi29 connector revealed that its central surface region exhibited slight hydrophobicity compared with the two flanking layers at the wide and narrow ends, respectively (FIG. )(Simpson et al., 2001 57:1260-69; Guasch et al., 2002 315:663-676). To facilitate connector purification, an affinity\/alignment domain comprising a C-terminal His (His) or Strep-II (WSHPQFEK) tag was inserted just downstream of a flexibility domain comprising a six glycine linker for improved affinity tag flexibility. Six-glycine linkers were included to provide end-flexibility (). After purification to homogeneity, it was found that the modified DNA-packaging motor protein connector polypeptide, GP10, self-assembled into the motor protein connector dodecameric structure with similar morphology to the 12-fold symmetric wild type connector (), as observed by transmission electron microscopy, TEM (FIG. D,E). The existence of a native and authentic motor configuration was verified through its competency to package the double-stranded DNA after incorporation into the procapsid () and to assemble the resulting DNA-filled capsid into the infectious phi29 virion ().","Reconstituting the Connector into Liposomes.","A procedure for reconstituting the connector into liposomes was developed by co-incubation of the connector with the lipid in the presence of sucrose as an exemplary osmotic agent. Such incubation provided an opportunity for the hydrophobic layer of the connector to interact with the hydrophobic domain of the lipid molecules. The dehydration-rehydration method (Lasic, D. D., CRC Press LLC., (1997), Boca Raton, Fla.) led to the production of giant liposomes up to 50 \u03bcm in diameter (). The insertion of the connector protein into the lipid membrane was confirmed by fluorescence microscopy, filtration assay, and sedimentation analysis (). The presence of the connector in the membrane was visible with fluorescence microscopy, showing a clear fluorescent ring around the liposome (). The fluorescent ring was very similar in appearance to the liposome generated with fluorescent lipids NBD-PE (). No fluorescent ring was observed when the fluorescently-tagged connector was mixed non-specifically with the non-connector inserted liposome (). The free connectors were removed by filtration using a membrane with a pore size of 0.45 \u03bcm or by 5-20% sucrose gradient ultracentrifugation ().","Incorporating the Connector into Planar Lipid Membranes.","Since none of the above experiments could distinguish between loose attachment of the connector to the bilayer surface and tight incorporation of it into the bilayer membrane to form a channel via retention of its aperture when integrated into the bilayer, a single channel conductance assay was performed. Results showed that direct incubation of the connector protein with liposomes or with a planar lipid bilayer did not lead to channel formation in the bilayer membrane (). Connector insertion into the bilayer only occurred when the connector protein-reconstituted proteoliposomes were fused into the bilayer () The channel insertion was observed through a discrete step-wise increase in conductance as shown in a continuous current trace (), under either positive or negative transmembrane voltage.","Typically, the insertion of one connector into the bilayer resulted in an increase in the current of approximately 65 pA (equivalent to 1.6 nS) at a potential of \u221240 mV in the presence of a solution of 5 mM Tris (pH 7.9)\/0.5 M NaCl. Occasionally a 130.9 pA jump, attributed to membrane insertion of two connectors, was also observed (). Similar results were obtained when the channel conductance was measured in TMS\/1 M NaCl (). In this case, the occurrence of simultaneous insertion of two connectors and three connectors was 4.7% and 1.9%, respectively.","For conductance measurements, a plot of the I-V curve was obtained under different voltages (). The average conductance per single pore was 1.57\u00b10.16 nS\/pore (a total of 38 inserted connectors) in a 5 mM Tris\/0.5 M NaCl () and 3.21\u00b10.51 nS\/pore (a total of 213 inserted connectors) in a TMS\/1 M NaCl buffer (). As a comparison, the conductance measurements were also performed for connector channels under a ramp voltage (FIG. F,G]). The slopes from the fitted curves representing the conductance of formed channels in the 5 mM Tris\/0.5 M NaCl were 1.59 nS\/single pore, 3.40 nS\/two pores, and 4.98 nS\/three pores, respectively. When NaCl concentration increased to 1 M (TMS buffer with 1 M NaCl), the slope of the curve was 3.12 nS\/single pore. A buffer of 5 mM HEPES\/1 M KCl was also used for conductance measurements (Table 1).",{"@attributes":{"id":"p-0226","num":"0225"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Comparison of Single Channel Conductance from the GP-10"},{"entry":"Connector (phi29 motor protein connector) and \u03b1-Hemolysin"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Pore","Cross",{}]},{"entry":[{},"diameter","section area","Conductance (nS\/pore) "]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Proteins","(nm)","(nm)","at 0.5M NaCl","at 1M KCl"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Connector","3.6","10.2","1.57 \u00b1 0.16","4.84 \u00b1 0.15\u2009"]},{"entry":["\u03b1-HL","1.5","1.8","0.31 \u00b1 0.05","0.94 \u00b1 0.01"]},{"entry":["Ratio","2.4","5.7","5.1","5.1"]},{"entry":"(Connector\/"},{"entry":"\u03b1-HL)"},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"left","id":"FOO-00001"},"sup":"a "}},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"left","id":"FOO-00002"},"sup":"b","i":"Chem. Biol. "}},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"left","id":"FOO-00003"},"sup":"c","i":["Nanotechnol. ","Nat. Biotechnol. "]}}]}}]}}},"The channel conductance of the connector was compared with that of alpha-hemolysin (\u03b1-HL) using solutions of different ionic strength (Table 1). It has been reported that the diameter of the narrow end of the connector channel is 3.6 nm, while the channel formed by \u03b1-HL has a diameter of only 1.5 nm (Song et al., 1996 274:1859). Therefore, the ratio of the cross-sectional area of the channels between the connector and \u03b1-HL was 5.7. The ratio of measured conductance of the connector to \u03b1-HL was 5.1 (Table 1).","Since the conductance of a channel is proportional to its cross-sectional area, it can be concluded that the cross-sectional area of the aperture formed by the viral DNA-packaging motor protein connector in the buffer solutions was approximately 5.1 fold greater than that of \u03b1-HL, which compared well with the ratio of cross-sectional areas from the crystal data of both proteins. Moreover, compared with other transmembrane proteins or ion channel proteins with larger channels, e.g., Streptolysin (Gilbert et al., 1999 97:647), Kir (Lopatin et al., 1996 71:682), VDAC (Szabo et al., 1998 12:495) and bacterial porins (Iqbal et al., 2007 2:243), the connector channel had additional advantages. For example, the viral DNA-packaging motor protein connector channels were stable and, unlike the previously described transmembrane channels, did not exhibit voltage gating under the reported conditions. The channel conductance was uniform, demonstrating a linear response to applied voltages between \u2212100 mV and 100 mV (FIG. F,G).","Translocation of Double-Stranded DNA.","Both linear and circular plasmid Cx43 DNA (5.5 kb) were used to examine the translocation of dsDNA through the aperture formed by the viral DNA-packaging motor protein connector channel. In the case of the linear DNA plasmid, DNA translocation induced numerous current blockades which led to the current jump of single connector insertion transiently reduced by 25-45% (). Similar results were also found in translocation experiments of a 35-bp dsDNA (). However, when the linear Cx43 was added to the cis-chamber, no such blockades were observed until the voltage was switched to positive potential (). The short-lived blockades could be attributed to the occurrence of DNA translocations. In contrast, in the absence of DNA, the current trace was quiescent (, ). Occasionally, unspecific blockades were observed with a minimum detectable time. These unspecific blockades rarely occurred compared to DNA translocation events. They were usually characterized with detectable time very close to the limit of sampling frequency. (, G). When circular plasmid dsDNA Cx43 was used, no translocation of the circular plasmid was observed (, upper left). Interestingly, when the same amount of circular plasmid digested by DNase I was added to the chamber, a burst of transient blockades occurred (lower left). The same results were also observed when the linear Cx43 digested by DNase I was used ( lower right). All the above results confirmed that only the linear dsDNA passed through the connector channels.","Occasionally, blockade events were observed in the range of 5-15% ( right) with a dwell time from several to hundreds of milliseconds. These events were attributed to non-specific blockades other than DNA translocation because they were found to occur in the absence of DNA (). The non-specific blockade events could be due to interactions of viral DNA-packaging motor protein connector apertures (conductive channels) with lipid or lipid micelles because their occurrence was observed to increased after the addition of more liposomes to the chamber. The occurrence of the non-specific blockades could be minimized when the diluted connector reconstituted liposomes were used and\/or when lower transmembrane voltage was applied ( left). Interestingly, simultaneous blockade events were occasionally detected (, upper right, and ). These events were recorded under multiple pore conditions. A continuous current trace recording events before and after addition of DNA was also shown (). On one occasion, a burst of DNA blockades was observed after the insertion of a third viral DNA-packaging motor protein connector. In comparison, when DNA was premixed with buffer before connector insertion, the DNA blockades were observed immediately after the first insertion occurred (). This result indicated a lack of a stirring facility in the DNA chamber leading to a delay in DNA translocation.","The blockade rates were affected by two factors: DNA concentration and transmembrane voltage. In the presence of 45 pM DNA under 3 viral DNA-packaging motor protein connector insertions, the blockade rate was approximately 0.8-1 blockades\/s ( upper right). Under the same number of viral DNA-packaging motor protein connector insertions, when 4 \u03bcM of DNA was placed in the chamber, the blockade rate was approximately 5-5.8 blockades\/s (). For the linear Cx43 DNA, the blockade rate increased as the ramping voltage was applied ().","To calculate the dwell time (\u03c4) for DNA translocation events, blockade episodes greater than 32% were grouped, since this percentage of blockade seemed consistent with the ratio of cross sectional area between dsDNA and the pore. A histogram of these events can be seen in . It should also be noted that 6 (under \u221275 mV) and 20 (under \u221240 mV) individual outlying events scattered between 120 ms to 9800 ms were not included in the graph for clarity. The dwell time distribution under \u221240 mV appeared to be broader than that under \u221275 mV. The average dwell time for DNA blockades under the \u221275 mV and under \u221240 mV was 9.2 ms and 22.1 ms respectively (only the events less than 50 ms under \u221240 mV were used for the calculation). As a comparison, the distribution of dwell time for the 35-bp dsDNA were also included (). The average of the dwell times in this case was 0.53 ms. Therefore, it can be concluded that the dwell time of DNA translocation was affected by applied voltage and the size of DNA.","To verify the passage of dsDNA through the apertures formed by the viral DNA-packaging motor protein connectors to create conductive channels, quantitative PCR (Q-PCR) was used to quantify the translocation of 141-bp DNA under a constant voltage. DNA was added to the trans-side and samples were taken from the cis-side for quantification at 30 minute intervals. For comparison, control experiments were performed in the absence of viral DNA-packaging motor protein connectors (). Experiments with connector insertions in bilayer lipid membranes (BLMs) showed an increase in the number of DNA molecules in the cis-chamber over time (N=9 experiments). In contrast, in the absence of connectors the number of DNA molecules in the cis-chamber remained undetectable over the 90-minute time course (N=4 experiments). Moreover, the DNA translocation rate was affected by the number of inserted viral DNA-packaging motor protein connectors ().","To verify that the increase of the DNA copy number in the cis-chamber was due to DNA translocation through the conductive channels formed by the apertures of the viral DNA-packaging motor protein connectors rather than membrane leakage, three additional experiments were carried out under the conditions of known leakage of BLM or partitions. When leaking occurred, the copy number of DNA per ul of solution in the cis-chamber was approximately 10-10fold higher than those experiments without leakage ().","Double-stranded DNA viral DNA-packaging motor protein connectors' pore size and conductance are modulated through engineering to vary aperture dimensions, surface hydropathy profile, freedom of movement of flexibility domain and\/or affinity\/alignment domain interaction with membrane bilayers and nature of the particular affinity binding site, all as described above. The electrophysiological, chemical and mechanical properties of the modified connector as incorporated into a membrane layer to form a conductive channel are characterized under different conditions, such as ionic strength, pH, and temperature. The orientation of the connector channel for DNA directional transmembrane transportation is determined as described herein, and condensed dsDNA loading into giant liposomes is performed under applied transmembrane electrical potentials. Unidirectional transmembrane translocation of nucleic acids into liposomal lumens precedes delivery of genetic information to recipient cells in vitro and\/or in vivo, and also precedes generation of liposomal or other membrane bounded bioreactors.","Pairs of His tag\/Anti-His tag antibody, Strep tag\/Streptavidin, Biotin\/Streptavidin, Strep tag\/Anti-strep antibody, and His-tag\/Nickel-NTA pairs are used as model systems to demonstrate the membrane-incorporated modified dsDNA viral DNA-packaging motor protein connector transmembrane aperture's utility as a conductive channel-based single molecule detector. The connector is modified by fusing at the N- and\/or C-termini with an affinity\/alignment domain that is a single chain antibody or with a peptide having desired affinity-binding specificity following identification in phage-display or with an RNA aptamer and is designed as described above to be capable of both analyte capture (i.e., detectable alteration of conductance upon specific analyte binding) and analyte fingerprinting (i.e., amplitude- and\/or duration-dependent generation of a conductance profile of altered conductance over a plurality of timepoints in response to a test analyte and under defined conditions that is compared to a reference conductance profile for known analytes, to characterize the test analyte). The combination of analyte detection by capture and analyte characterization by fingerprinting provides sensitivity enhancement (i.e., ability to detect and\/or characterize that is greater, in a statistically significant manner, than would be the ability if only capture functionality was present) for identification of molecules at very low concentration. Detection and characterization of analytes occurs at earlier stages of disease for diagnostic purposes, and sensitivity of environmental surveillance for analytes is enhanced.","Various mutations are made in the polypeptide sequences that influence the dimensions, hydropathy, charge and conductivity properties, and solvent accessibility of the aperture domain (lumen) and\/or the N- and\/or C-termini, including modifications to provide different flexibility domains and\/or affinity\/alignment domains, of the DNA-packaging motor protein connector that is capable of forming a transmembrane conductive channel are made, to adjust the channel properties for enhancing sensitivity and resolution of analyte detection and characterization. Translocations of several types and shapes of nucleic acid polymers are characterized through conductance measurements including generation of conductance profiles such as profiles over a plurality of timepoints of the amplitude and duration of altered conductance. Detection of DNA mismatch mutations in cancer cells is tested by contacting cancer cell-derived DNA with the conductive channel-containing membrane. Procedures for reducing DNA translocation speed are developed by varying assay conditions (e.g., temperature, pH, ionic conditions, magnitude of applied electrical potential) and by varying modified DNA-packaging motor connector composition. The conductive channel is derivatized with channel-modifying chemicals in a controlled manner to slow the transmembrane passage (translocation) of dsDNA and conductance profiles are generated by which single nucleotides are recognized with high precision. The conductive channel-containing membrane is incorporated into a high throughput dsDNA sequencing apparatus.","The conductance of a single modified viral DNA-packaging motor protein connector in a bilayer membrane (BLM) increased with the salt concentration in a strong linear relationship (). In addition, the conductance of the connector in BLM appeared to be ion dependent, with the conductance in KCl higher than that in NaCl, when the same salt concentration was used (, C) Such properties of the connector channel in BLM make it possible to test the buffer concentrations and to differentiate ion species, as well as to tune the conductance.","Linear and circular plasmid dsDNA (5.5 kb) were both used to examine translocation of dsDNA through the conductive channel formed by a membrane-incorporated modified viral DNA-packaging motor protein connector, as also described above in Examples 1 and 2. Linear DNA translocation induced numerous current blockades by about 30% (). When circular plasmid dsDNA was used, no translocation was observed (). However, after digestion by DNase I, both linear and circular plasmid DNA gave a burst of transient blockades (, D). The results indicated that only linear dsDNA passed through the connector channels, while circular DNA did not. The DNA translocation was confirmed by Quantitative PCR of the DNA samples that passed through the pore using the SYBR Green kit ().","The conductive channel connector aperture in BLM differentiated the sizes of linear dsDNA based on their characteristic dwell times, and differentiated linear dsDNA from folded dsDNA. When a 5 kbp dsDNA translocated through the conductive channel under an applied potential, different types of current blockade events were observed (). The persistence length of dsDNA in the presence of 1M ionic strength is \u02dc150 bp (Baumann et al., 1997 94:6185), and DNA translocation has been observed through synthetic nanopores (Storm et al., 2005 71(5 Pt 1):51903; Chen et al., 2004 4:2293). Similar observations for the herein described modified phi29 DNA-packaging motor protein connector channel were demonstrated by discrete levels in current blockade arising from a single translocation event (Single linear events type I, II and III) (). The double (and triple) events were attributed to two (and three) DNA molecules translocating through two (and three) channels simultaneously (). Similar events were observed using 500 bp dsDNA as the analyte.","One-way traffic (unidirectional) DNA translocation was observed through a modified phi29 DNA-packaging motor protein connector channel in a BLM. A single modified DNA-packaging motor protein connector was inserted in a BLM with DNA premixed in both the cis- and the trans-chambers. When the applied voltage was alternated between positive and negative potentials, DNA translocations were only observable in one direction (either positive only or negative only) as shown in . According to non-limiting theory it appeared that the DNA could only pass through the conductive channel aperture formed by the connector in one direction, an interpretation that was consistent with the results of dsDNA translocation experiments performed with multiple connector insertions ().","Further modifying the amino acid sequence of the modified phi29 DNA-packaging motor protein connector channel resulted in alterations of the channel conductance. When the lysine at position 234 inside the connector tunnel (aperture) was replaced by cysteine (K234C) [SEQ ID NO:42] or alanine (K234A) [SEQ ID NO:41], the current jump induced by a single modified connector inserted into the BLM was smaller than that of the wild-type connector ().","This example describes generation of a conductance profile \u201cfingerprint\u201d of specifically captured polypeptide analyte molecules (antibody) following their interaction with the affinity\/alignment domain of a modified DNA-packaging motor protein connector as described herein.","The phi29 DNA-packaging motor protein connector was re-engineered as described above with a His-tag as an affinity\/alignment domain at its C-terminus. The re-engineered connector was inserted in a BLM and showed a typical current jump under an applied potential under assay conditions essentially as described above. When a specific anti-His-tag antibody was added in the sample chamber, a step-wise decrease in the current was observed, with each small step being about 20% of the total current jump for a single connector pore (). The step-wise decrease in current jump was due to the sequential binding of one antibody molecule to each of the twelve His-tags on the dodecameric connector, which sequential binding decreased the overall size of the aperture forming the conductive channel. This approach is contemplated for single molecule identification by the combination of (i) capturing analyte through this type of antibody-antigen specific binding interaction and (ii) \u201cfingerprinting\u201d the analyte by generating an analyte-characteristic conductance profile (e.g., a blockade pattern such as the one shown in ).","This example uses custom designed highly sensitive single molecule dual channel imaging system to detect dual labeled dsDNA system.","A customized highly sensitive imaging system in single fluorophore detection is established in our lab. The apparatus has synchronous optical and electrical design setup, using solid state channel and single protein channel. In this device, the silicon chip containing a SiN membrane is replaced by the BCH Chamber (Eastern Scientific LLC) containing the Teflon membrane. The refractive indices of the buffer used in the cis chamber, nw is about 1.33. Since the glass refractive index, ng is 1.5, a solution containing 7M CsCl and 10 mM Tris, pH of 8.5 with a index of nCs=1.41 is placed between the water buffer and the glass for total internal reflection.","To solve the problem for single pore sensing, which is the feasibility to control the number of connectors in the BLM, we had to quantify and monitor the number of connectors inserted in the BLM. To stop any further insertion of connectors into the membrane, an automated patch clamp was incorporated into our system.","This automated system has several advantages: (1) It is automated; (2) the cis- and trans-chambers are miniaturized with provisions for efficient solution exchange; (3) single liposomes containing single connector can be patched efficiently. This system was originally designed for the patching single cells. Preliminary data demonstrated the successful patch of liposome\/connector complexes to form connector channel in the lipid membrane or cell membrane (data not shown).","In this system, buffer removal and sample exchange is monitored and controlled at will using the automated computer system. As soon as the desired number of connector insertion reaches the level in interest, immediate removal of the buffer and\/or sample followed by thorough washing stops additional connector insertions. A single connector-reconstituted liposome is patched from the inside as a whole cell and is converted into a membrane by suction with automatic force control. A Pipette is moved between the monitor system and the imaging microfluidic chamber. A custom design pipette tip ensures the presence of a stable pore, and more importantly, the connector incorporation process does not depend on vesicle fusion. Hence, the membrane pore system is monitored and assembled under control with high success rates under a wide range of experimental conditions.","The Use of Fluorescence Resonance Energy Transfer (FRET) for Detection of DNA","For a given dye pair, the detectable FRET event ranges only from 1 to 10 nm. Using single-molecule FRET with objective type TIRF for detection will narrow the depth of focus and provide better spatial resolution in z-direction when a fluorescently labeled DNA chain is passing through the pore. The connector or the active motor embedded in the membrane will be labeled with donor fluorophores at the N- or C terminus of the connector as described previously. The possible donor\/acceptor pairs are listed in Table 2.",{"@attributes":{"id":"p-0252","num":"0251"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Proposed donor and acceptor for Detection via FRET"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"98pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Choice of Donor","Choice of Acceptor"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"QD525","Cy3\u2009\u2122"]},{"entry":[{},"Alexa Fluor\u2009\u2122 488","Alexa Fluor\u2009\u2122 546"]},{"entry":[{},{},"Tetramethylrhodamine"]},{"entry":[{},{},"TAMRA\u2009\u2122"]},{"entry":[{},"QD655","Cy5.5"]},{"entry":[{},{},"Alexa Fluor\u2009\u2122 700"]},{"entry":[{},{},"IRDye\u2009\u00ae 700"]},{"entry":[{},"QD705","Alexa Fluor\u2009\u2122 750"]},{"entry":[{},"Alexa Fluor\u2009\u2122 700","Alexa Fluor\u2009\u2122 790"]},{"entry":[{},{},"Dy750"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"Quantum dots (QD) have been applied in single-molecule FRET. QDs as donors have the following advantages: 1) very bright and long-lasting without photobleaching; 2) narrow fluorescence emission spectra for reducing the possibility of cross-talk with acceptor emission; 3) broad excitation spectra, thus different QDs can be excited with same laser wavelength in UV or short visible spectra to avoid the use of multiple lasers in the experiment and eliminate cross excitation of acceptors; 4) various functionalized QDs and excitation filter sets are commercially available, providing versatilities in connector labeling.","Attaching FRET Donors to the Channel","Label the Connector with Donors:","Cysteine residue or a biotin group will be introduced to the connector for incorporating the donors at the N- or C-terminal ends of the 12 copies of the connector subunits. We have previously shown that some terminal modifications have little or no effect on channel formation or its ability to package dsDNA69, thus verifying its authenticity. As the connector is composed of 12 subunits of gp10 protein, there are a total of 12 re-engineered tags on each connector molecule. The QD525 and QD655 will be attached to the terminus of the same connector by mixing them at a 1:1 molar ratio during labeling via strep\/biotin or SH-Maleimide interactions.","Label the DNA with Two Acceptors Corresponding to Two Donors","The proof of concept for distinguishing two different nucleotides that pass through the connector pore is shown by detecting two different fluorescence signals. Short dsDNA chains is constructed using customized fluorescent DNA oligos with two dyes attached between 2 to 50 bp (). Acceptors to be excited by donors QD525 and QD655 are listed in Table 2. The design of labeling the DNA with acceptors as a proof of concept is shown in . Near UV Laser is used to excite the QDs, and FRET is detected synchronously with the electrical signals. Proper dichroic mirrors and band pass filters are used to exclude the signals from the donor emissions while transmitting the acceptor emissions with high efficiency. The distance for FRET is about 1-10 nm, corresponding to a range of 1-30 bp. The closer the dyes are, the stronger the FRET signal. The dual-labeled short dsDNA chains () is used to provide empirical information of how close between two dyes the system can discriminate ().","In the case emission spectrum of Alexa546 and the excitation spectrum of Cy5.5 slightly overlaps, very small energy transfer might occur if they are within close proximity in DNA. Under those conditions Alexa790 is used to replace Cy5.5 as the acceptor for donor QD705 (table 2). Alexa790 is an IR dye and will not have substantial overlap with the emission spectra of Alexa546 ().","The various embodiments described above can be combined to provide further embodiments. All of the U.S. patents, U.S. patent application publications, U.S. patent application, foreign patents, foreign patent application and non-patent publications referred to in this specification and\/or listed in the Application Data Sheet are incorporated herein by reference, in their entirety. Aspects of the embodiments can be modified, if necessary to employ concepts of the various patents, application and publications to provide yet further embodiments.","These and other changes can be made to the embodiments in light of the above-detailed description. In general, in the following claims, the terms used should not be construed to limit the claims to the specific embodiments disclosed in the specification and the claims, but should be construed to include all possible embodiments along with the full scope of equivalents to which such claims are entitled. Accordingly, the claims are not limited by the disclosure."],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE SEVERAL VIEWS OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0045","num":"0044"},"figref":"FIG. 1","i":["Acta Cryst D","J. Mol. Biol. ","J. Nanosci. Nanotechnol. "]},{"@attributes":{"id":"p-0046","num":"0045"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0047","num":"0046"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0048","num":"0047"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0049","num":"0048"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0050","num":"0049"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0051","num":"0050"},"figref":"FIG. 7"},{"@attributes":{"id":"p-0052","num":"0051"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0053","num":"0052"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0054","num":"0053"},"figref":"FIG. 10"},{"@attributes":{"id":"p-0055","num":"0054"},"figref":"FIG. 11","sub":"T "},{"@attributes":{"id":"p-0056","num":"0055"},"figref":"FIG. 12"},{"@attributes":{"id":"p-0057","num":"0056"},"figref":["FIG. 13","FIG. 8"]},{"@attributes":{"id":"p-0058","num":"0057"},"figref":"FIG. 14"},{"@attributes":{"id":"p-0059","num":"0058"},"figref":"FIG. 15"},{"@attributes":{"id":"p-0060","num":"0059"},"figref":"FIG. 16"},{"@attributes":{"id":"p-0061","num":"0060"},"figref":"FIG. 17","sup":["nd ","rd "]},{"@attributes":{"id":"p-0062","num":"0061"},"figref":"FIG. 18"},{"@attributes":{"id":"p-0063","num":"0062"},"figref":"FIG. 19"},{"@attributes":{"id":"p-0064","num":"0063"},"figref":"FIG. 20"},{"@attributes":{"id":"p-0065","num":"0064"},"figref":"FIG. 21"},{"@attributes":{"id":"p-0066","num":"0065"},"figref":"FIG. 22"}]},"DETDESC":[{},{}]}
