---
title: Neuropilin antagonists
abstract: Novel anti-NRP1 antibodies and variants thereof having unique structural and functional characteristics are disclosed. Also provided are uses of the antibodies in research, diagnostic and therapeutic applications.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08211429&OS=08211429&RS=08211429
owner: Genetech, Inc.
number: 08211429
owner_city: South San Francisco
owner_country: US
publication_date: 20110630
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"p":["This application is a divisional application of U.S. application Ser. No. 12\/107,544, now U.S. Pat. No. 7,994,286,filed Apr. 22, 2008 which is a continuation application of PCT Application No. PCT\/US2006\/043516 filed Nov. 8, 2006, which claims the benefit of priority under 35 USC 119(e) to U.S. provisional application Nos. 60\/734,798 filed Nov. 8, 2005 and 60\/820,561, filed Jul. 27, 2006, the entire contents of which are incorporated herein by reference.","The present invention relates generally to compositions and methods associated with neuropilin (NRP) activities. More particularly, the invention pertains to compositions and methods for modulating vacscular formation and maintenance mediated by neuropilin-1 (NRP1) receptor. This invention further relates to methods for the screening of substances therapeutically useful for preventing or treating conditions and diseases associated with angiogenesis.","Development of a vascular system is a fundamental requirement for many physiological and pathological processes. Actively growing tissues such as embryos and tumors require adequate blood supply. They satisfy this need by producing pro-angiogenic factors, which promote new blood vessel formation and maintenance via a process generally referred to as angiogenesis. Vascular formation is a complex but orderly biological event involving all or many of the following steps: a) Endothelial cells (ECs) proliferate from existing ECs or differentiate from progenitor cells; b) ECs migrate and coalesce to form cord-like structures; c) vascular cords then undergo tubulogenesis to form vessels with a central lumen d) existing cords or vessels send out sprouts to form secondary vessels; e) primitive vascular plexus undergo further remodeling and reshaping; and f) peri-endothelial cells are recruited to encase the endothelial tubes, providing maintenance and modulatory functions to the vessels; such cells including pericytes for small capillaries, smooth muscle cells for larger vessels, and myocardial cells in the heart. Hanahan, D. 277:48-50 (1997); Hogan, B. L. & Kolodziej, P. A. 3:513-23 (2002); Lubarsky, B. & Krasnow, M. A. 112:19-28 (2003).","It is now well established that angiogenesis is implicated in the pathogenesis of a variety of disorders. These include solid tumors and metastasis, atherosclerosis, retrolental fibroplasia, hemangiomas, chronic inflammation, intraocular neovascular diseases such as proliferative retinopathies, e.g., diabetic retinopathy, age-related macular degeneration (AMD), neovascular glaucoma, immune rejection of transplanted corneal tissue and other tissues, rheumatoid arthritis, and psoriasis. Folkman et al., J. Biol. Chem., 267:10931-10934 (1992); Klagsbrun et al., Annu. Rev. Physiol. 53:217-239 (1991); and Garner A., \u201cVascular diseases\u201d, In: Pathobiology of Ocular Disease. A Dynamic Approach, Garner A., Klintworth G K, eds., 2nd Edition (Marcel Dekker, NY, 1994), pp 1625-1710.","In the case of tumor growth, angiogenesis appears to be crucial for the transition from hyperplasia to neoplasia, and for providing nourishment for the growth and metastasis of the tumor. Folkman et al., 339:58 (1989). The neovascularization allows the tumor cells to acquire a growth advantage and proliferative autonomy compared to the normal cells. A tumor usually begins as a single aberrant cell which can proliferate only to a size of a few cubic millimeters due to the distance from available capillary beds, and it can stay \u2018dormant\u2019 without further growth and dissemination for a long period of time. Some tumor cells then switch to the angiogenic phenotype to activate endothelial cells, which proliferate and mature into new capillary blood vessels. These newly formed blood vessels not only allow for continued growth of the primary tumor, but also for the dissemination and recolonization of metastatic tumor cells. Accordingly, a correlation has been observed between density of microvessels in tumor sections and patient survival in breast cancer as well as in several other tumors. Weidner et al., 324:1-6 (1991); Horak et al., 340:1120-1124 (1992); Macchiarini et al., 340:145-146 (1992). The precise mechanisms that control the angiogenic switch is not well understood, but it is believed that neovascularization of tumor mass results from the net balance of a multitude of angiogenesis stimulators and inhibitors (Folkman, 1995, 1(1):27-31).","The process of vascular development is tightly regulated. To date, a significant number of molecules, mostly secreted factors produced by surrounding cells, have been shown to regulate EC differentiation, proliferation, migration and coalescence into cord-like structures. For example, vascular endothelial growth factor (VEGF) has been identified as the key factor involved in stimulating angiogenesis and in inducing vascular permeability. Ferrara et al., 18:4-25 (1997). The finding that the loss of even a single VEGF allele results in embryonic lethality points to an irreplaceable role played by this factor in the development and differentiation of the vascular system. Furthermore, VEGF has been shown to be a key mediator of neovascularization associated with tumors and intraocular disorders. Ferrara et al., . supra. The VEGF mRNA is overexpressed by the majority of human tumors examined. Berkman et al., 91:153-159 (1993); Brown et al., 26:86-91 (1995); Brown et al., 53:4727-4735 (1993); Mattern et al., 73:931-934 (1996); Dvorak et al., 146:1029-1039 (1995).","Also, the concentration levels of VEGF in eye fluids are highly correlated to the presence of active proliferation of blood vessels in patients with diabetic and other ischemia-related retinopathies. Aiello et al., 331:1480-1487 (1994). Furthermore, studies have demonstrated the localization of VEGF in choroidal neovascular membranes in patients affected by AMD. Lopez et al., 37:855-868 (1996).","Anti-VEGF neutralizing antibodies suppress the growth of a variety of human tumor cell lines in nude mice (Kim et al., 362:841-844 (1993); Warren et al., 95:1789-1797 (1995); Borgstrom et al., 56:4032-4039 (1996); Melnyk et al., 56:921-924 (1996)) and also inhibit intraocular angiogenesis in models of ischemic retinal disorders. Adamis et al., 114:66-71 (1996). Therefore, anti-VEGF monoclonal antibodies or other inhibitors of VEGF action are promising candidates for the treatment of tumors and various intraocular neovascular disorders. Such antibodies are described, for example, in EP 817,648 published Jan. 14, 1998; and in WO98\/45331 and WO98\/45332, both published Oct. 15, 1998. One of the anti-VEGF antibodies, bevacizumab, has been approved by the FDA for use in combination with a chemotherapy regimen to treat metastatic colorectal cancer (CRC) and non-samll cell lung cancer (NSCLC). And bevacizumab is being investigated in many ongoing clinical trials for treating various cancer indications.","During development of the nervous system, neurons send out cable-like axons that migrate over long distances in order to reach their targets. See review by Carmeliet and Tessier-Lavigne (2005) 436:193-200. At the leading tip of a growing axon is a highly motile, sensory structure called growth cone. Through dynamic cycles of extension and retraction of filopodial extensions, the growth cone continually senses and asseses from a myriad of guidance cues in its spatial environment, and accurately selects a correct track for extension towards its final target.","Over the past decade, considerable progress has been made in understanding axon guidance mechanisms. See review by Dickson (2002) 298:1959-64. Guidance cues come in four varieties: attractants and repellents; which may act either at short range (i.e., cell- or matrix-associated) or at longer range (i.e., diffusible). So far, four major families of axon guidance molecules have been identified: the netrins, semaphorins, ephrins and slits. See review by Huber et al (2003) 26:509-63.","The semaphorins (Sema), also called collapsins, belong to a large family of phylogenetically conserved secreted and membrane-associated proteins. Members of the semaphorin family are capable of mediating both repulsive and attractive axon guidance events during neural development. Raper (2000) 10:88-94. The more than thirty semaphorins identified to date all share a conserved N-terminal Sema domain of about 500 amino acids. Semaphorin members are classified into eight subfamilies depending on their structural similarities and species of origin. For more details on unified nomenclature for semaphorins, see Semaphorin Nomenclature Committee (1999) 97:551-552.","The neuropilin (NRP) family is comprised of two homologous proteins, neuropilin-1 (NRP1) and neuropilin-2 (NRP2). NRP1 was first identified as a type I 130-kDa transmembrane glycoprotein expressed in growth cones of growing axons. NRP2 was subsequently identified by expression cloning. Fujisawa and Kitsukawa (1998) 8:587-592. NRPs are found to be receptors for a subset of semaphorins, the class 3 semaphorins. It was suggested that NRPs function as non-signaling co-receptors along with another semaphorin receptor family, plexins.","Although initially described as a mediator of axon guidance, NRPs have also been found to play critical roles in vascular development. Carmeliet and Tessier-Lavigne (2005). It is identified as an isoform-specific VEGF receptor expressed on tumor and endothelial cells, prompting considerable efforts to understand the role of NRPs in vascular and tumor biology. Soker et al (1998) 92:735-745; Klagsbrun et al (2002) 515:33-48. Genetic studies have provided strong evidence that Nrp1 is required for vascular morphogenesis. Loss of Nrp1 function results in vascular remodeling and branching defects, a phenotype that can be further enhanced by the loss of Nrp2 function. Kawasaki et al. (1999) 126:4895-4902; Takashima et al. (2002) 99:3657-3662. These results suggest that early in development Nrp1 and Nrp2 may have overlapping functions. However, the expression of each Nrp is partitioned later in development, with Nrp1 being expressed primarily in arteries, and Nrp2 in veins and lymphatic vessels. Yuan et al (2002) 129:4797-4806; Herzog et al. (2001) 109:115-119. Notably, loss of Nrp2 function alone specifically impairs lymphatic development.","As Nrp1 is expressed in many other cell types during development, the role of vascular Nrp1 was addressed through the generation of an EC-specific knock-out, which resulted in similar vascular defects to those seen in the null allele. Gu et al. (2003) 5:45-57. Interestingly, this study also showed that Sema3A binding to NRP1 is not required for vascular development. In another study, defects were observed in the guidance of endothelial tip cells in the developing hindbrain in Nrp1 KO embryos. Gerhardt et al. (2004) 231:503-509.","Despite the extensive studies in NRP1\u2032s role in vascular development, it remains unclear as to whether NRP1 exerts its vascular function exclusively via the VEGF-VEGF Receptor 2 (VEGFR2) pathway, as an enhancer for VEGF binding to VEGFR2 and thereby for VEGFR2 signaling, or via a signaling pathway independent of VEGFR2, or a combination of both.","Monoclonal antibodies can be manufactured using recombinant DNA technology. Widespread use has been made of monoclonal antibodies, particularly those derived from rodents, however nonhuman antibodies are frequently antigenic in humans. The art has attempted to overcome this problem by constructing \u201cchimeric\u201d antibodies in which a nonhuman antigen-binding domain is coupled to a human constant domain (Cabilly et al., U.S. Pat. No. 4,816,567). The isotype of the human constant domain may be selected to tailor the chimeric antibody for participation in antibody-dependent cellular cytotoxicity (ADCC) and complement-dependent cytotoxicity. In a further effort to resolve the antigen binding functions of antibodies and to minimize the use of heterologous sequences in human antibodies, humanized antibodies have been generated for various antigens in which substantially less than an intact human variable domain has been substituted at regions by the corresponding sequence from a non-human species. For example, rodent residues have been substituted for the corresponding segments of a human antibody. In practice, humanized antibodies are typically human antibodies in which some complementarity determining region (CDR) residues and possibly some framework region (FR) residues are substituted by residues from analogous sites in rodent antibodies. Jones et al. (1986) 321:522-525; Riechmann et al. (1988) 332:323-327; Verhoeyen et al. (1988) 239:1534-1536.","Prior to administering a therapeutic antibody to human, preclinical studies in nonhuman mammals are generally desired to evaluate the efficacy and\/or toxicity of the antibody. Ideally, the antibodies subject to these studies are capable of recognizing and reacting with high potency to a target antigen endogenous to the host animal such as mouse or nonhuman primate.","Phage display technology has provided a powerful tool for generating and selecting novel proteins that bind to a ligand, such as an antigen. Using the technique of phage display, large libraries of protein variants can be generated and rapidly sorted for those sequences that bind to a target antigen with high affinity. Nucleic acids encoding variant polypeptides are fused to a nucleic acid sequence encoding a viral coat protein, such as the gene III protein or the gene VIII protein. Monovalent phage display systems where the nucleic acid sequence encoding the protein or polypeptide is fused to a nucleic acid sequence encoding a portion of the gene III protein have been developed. (Bass, S. (1990) 8:309; Lowman and Wells (1991) 3:205). In a monovalent phage display system, the gene fusion is expressed at low levels and wild type gene III proteins are also expressed so that infectivity of the particles is retained. Methods of generating peptide libraries and screening those libraries have been disclosed in many patents (e.g., U.S. Pat. Nos. 5,723,286, 5,432,018, 5,580,717, 5,427,908 and 5,498,530).","The demonstration of expression of peptides on the surface of filamentous phage and the expression of functional antibody fragments in the periplasm of was important in the development of antibody phage display libraries. (Smith et al. (1985) 228:1315; Skerra and Pluckthun (1988) 240:1038). Libraries of antibodies or antigen binding polypeptides have been prepared in a number of ways including by altering a single gene by inserting random DNA sequences or by cloning a family of related genes. Methods for displaying antibodies or antigen binding fragments using phage display have been described in U.S. Pat. Nos. 5,750,373, 5,733,743, 5,837,242, 5,969,108, 6,172,197, 5,580,717, and 5,658,727. The library is then screened for expression of antibodies or antigen binding proteins with desired characteristics.","Phage display technology has several advantages over conventional hybridoma and recombinant methods for preparing antibodies with the desired characteristics. This technology allows the development of large libraries of antibodies with diverse sequences in less time and without the use of animals. Preparation of hybridomas or preparation of humanized antibodies can easily require several months of preparation. In addition, since no immunization is required, phage antibody libraries can be generated for antigens which are toxic or have low antigenicity (Hogenboom (1988) 4:1-20). Phage antibody libraries can also be used to generate and identify novel therapeutic antibodies.","Phage display libraries have been used to generate human antibodies from immunized, non-immunized humans, germ line sequences, or na\u00efve B cell Ig repertories (Barbas & Burton(1996) 14:230; Griffiths et al. (1994) 13:3245; Vaughan et al. (1996) 14:309; Winter EP 0368 684 B1). Na\u00efve, or nonimmune, antigen binding libraries have been generated using a variety of lymphoidal tissues. Some of these libraries are commercially available, such as those developed by Cambridge Antibody Technology and Morphosys (Vaughan et al.(1996) 14:309; Knappik et al. (1999) 296:57). However, many of these libraries have limited diversity.","The ability to identify and isolate high affinity antibodies from a phage display library is important in isolating novel antibodies for therapeutic use. Isolation of high affinity antibodies from a library is dependent on the size of the library, the efficiency of production in bacterial cells and the diversity of the library. See, for e.g., Knappik et al. (1999) 296:57. The size of the library is decreased by inefficiency of production due to improper folding of the antibody or antigen binding protein and the presence of stop codons. Expression in bacterial cells can be inhibited if the antibody or antigen binding domain is not properly folded. Expression can be improved by mutating residues in turns at the surface of the variable\/constant interface, or at selected CDR residues. (Deng et al. (1994) 269:9533, Ulrich et al. (1995) 92:11907-11911; Forsberg et al. (1997) 272:12430). The sequence of the framework region is a factor in providing for proper folding when antibody phage libraries are produced in bacterial cells.","Generating a diverse library of antibodies or antigen binding proteins is also important to isolation of high affinity antibodies. Libraries with diversification in limited CDRs have been generated using a variety of approaches. See, e.g., Tomlinson (2000) 18:989-994. CDR3 regions are of interest in part because they often are found to participate in antigen binding CDR3 regions on the heavy chain vary greatly in size, sequence and structural conformation.","Others have also generated diversity by randomizing CDR regions of the variable heavy and light chains using all 20 amino acids at each position. It was thought that using all 20 amino acids would result in a large diversity of sequences of variant antibodies and increase the chance of identifying novel antibodies. (Barbas (1994) 91:3809; Yelton, D E (1995) 155:1994; Jackson, J. R. (1995) 154:3310 and Hawkins, R E (1992) 226:889).","The present invention provides novel anti-NRP1 antibodies capable of modulating at least one neuropilin mediated biological activity. Preferably, the anti-NRP1 antibodies are antagonist antibodies capable of inhibiting at least one neuropilin mediated biological activity. More specifically, the present invention provides methods of generating anti-NRP1 antibodies from a designed human synthetic antibody phage library, and novel function blocking anti-NRP1 antibodies generated thereof The anti-NRP1 antibodies of the invention fall into two classes, depending on where they bind to on NRP1: anti-NRP1antibodies (including YW64.3 and variants thereof) are those that bind to the CUB domains (a1a2) of NRP1; and anti-NRP1antibodies (including YW107.4 and variants thereof) are those that bind to the coagulation factor V\/VIII domains (b1b2) of NRP1.","In one aspect, the anti-NRP1antibodies of the invention can be selected from the \u201cYW64\u201d antibody clones as shown in Table III, having the identified partial CDR sequences and binding affinities to murine and human NRP1s. The anti-NRP1antibody of the invention preferably comprises a light chain variable domain comprising the following CDR amino acid sequences: CDRL1 (RASQSISSYLA; SEQ ID NO:123), CDRL2 (GASSRAS; SEQ ID NO:124) and CDRL3 (QQYMSVPIT; SEQ ID NO:125). For example, the anti-NRP1antibody comprises a light chain variable domain sequence of SEQ ID NO:3. The anti-NRP1antibody of the invention preferably comprises a heavy chain variable domain comprising the following CDR amino acid sequences: CDRH1 (GFSFSSEPIS; SEQ ID NO:126), CDRH2 (SSITGKNGYTYYADSVKG; SEQ ID NO:127) and CDRH3 (WGKKVYGMDV; SEQ ID NO:128). For example, the anti-NRP1antibody comprises a heavy chain variable domain sequence of SEQ ID NO:4. More preferably, the anti-NRP1antibody of the invention is the YW64.3 antibody comprising a light chain variable domain sequence of SEQ ID NO:3 and a heavy chain variable domain sequence of SEQ ID NO:4.","In another aspect, the anti-NRP1antibodies of the invention can be selected from the \u201cYW107.4\u201d antibody clones as shown in Table IV, having the identified partial CDR sequences and binding affinities to murine and human NRP1s. The anti-NRP1antibody of the invention preferably comprises a light chain variable domain comprising the following CDR amino acid sequences: CDRL1 (RASQYFSSYLA; SEQ ID NO:129), CDRL2 (GASSRAS; SEQ ID NO:130) and CDRL3 (QQYLGSPPT; SEQ ID NO:131). For example, the anti-NRP1antibody comprises a light chain variable domain sequence of SEQ ID NO:5. The anti-NRP1antibody of the invention preferably comprises a heavy chain variable domain comprising the following CDR amino acid sequences: CDRH1 (GFTFSSYAMS; SEQ ID NO:132), CDRH2 (SQISPAGGYTNYADSVKG; SEQ ID NO:133) and CDRH3 (ELPYYRMSKVMDV; SEQ ID NO:134). For example, the anti-NRP1antibody comprises a heavy chain variable domain sequence of SEQ ID NO:6. More preferably, the anti-NRP1antibody of the invention is the YW107.4.87 antibody comprising a light chain variable domain sequence of SEQ ID NO:5 and a heavy chain variable domain sequence of SEQ ID NO:6.","Also provided by the present invention are uses of the anti-NRP1 antibodies for treating angiogenesis associated disorders such as cancer. In one preferred embodiment, the anti-NRP1 antibodies of invention are used in combination with an anti-VEGF antibody. Preferably, the anti-VEGF antibody is capable of binding to the same VEGF epitope as the antibody A4.6.1. More preferably, the anti-VEGF antibody is bevacizumab or ranibizumab.","The present invention relates to novel compositions and methods for modulating NRP mediated biological activities.","Definitions","\u201cNeuropilin\u201d or NRP refers collectively to neuropilin-1 (NRP1), neuropilin-2 (NRP2) and their isoforms and variants, as described in Rossignol et al. (2000) 70:211-222. Neuropilins are 120 to 130 kDa non-tyrosine kinase receptors. There are multiple NRP-1 and NRP-2 splice variants and soluble isoforms. The basic structure of neuropilins comprises five domains: three extracellular domains (a1a2, b1b2 and c), a transmembrane domain, and a cytoplasmic domain. The a1a2 domain is homologous to complement components C1r and C1s (CUB), which generally contains four cysteine residues that form two disculfid bridges. The b1b2 domain is homologous to coagulation factors V and VIII. The central portion of the c domain is designated as MAM due to its homology to meprin, A5 and receptor tyrosine phosphotase \u03bc proteins. The a1a2 and b1b2 domains are responsible for ligand binding, whereas the c domain is critical for homodimerization or heterodimerization. Gu et al. (2002) 277:18069-76; He and Tessier-Lavigne (1997) 90:739-51.","\u201cNeuropilin mediated biological activity\u201d refers in general to physiological or pathological events in which neuropilin-1 and\/or neuropilin-2 plays a substantial role. Non-limiting examples of such activities are axon guidance during embryonic nervous system development or neuron-regeneration, angiogenesis (including vascular modeling), tumorgenesis and tumor metastasis.","The term \u201cantibody\u201d is used in the broadest sense and specifically covers monoclonal antibodies (including full length monoclonal antibodies), polyclonal antibodies, multispecific antibodies (e.g., bispecific antibodies), and antibody fragments so long as they exhibit the desired biological activity.","The term \u201cmonoclonal antibody\u201d as used herein refers to an antibody obtained from a population of substantially homogeneous antibodies, i.e., the individual antibodies comprising the population are identical except for possible naturally occurring mutations that may be present in minor amounts. Monoclonal antibodies are highly specific, being directed against a single antigenic site. Furthermore, in contrast to conventional (polyclonal) antibody preparations which typically include different antibodies directed against different determinants (epitopes), each monoclonal antibody is directed against a single determinant on the antigen. The modifier \u201cmonoclonal\u201d indicates the character of the antibody as being obtained from a substantially homogeneous population of antibodies, and is not to be construed as requiring production of the antibody by any particular method. For example, the monoclonal antibodies to be used in accordance with the present invention may be made by the hybridoma method first described by Kohler et al. (1975) 256:495, or may be made by recombinant DNA methods (see, e.g., U.S. Pat. No. 4,816,567). The \u201cmonoclonal antibodies\u201d may also be isolated from phage antibody libraries using the techniques described in Clackson et al. (1991) 352:624-628 and Marks et al. (1991) 222:581-597, for example.","The monoclonal antibodies herein specifically include \u201cchimeric\u201d antibodies (immunoglobulins) in which a portion of the heavy and\/or light chain is identical with or homologous to corresponding sequences in antibodies derived from a particular species or belonging to a particular antibody class or subclass, while the remainder of the chain(s) is identical with or homologous to corresponding sequences in antibodies derived from another species or belonging to another antibody class or subclass, as well as fragments of such antibodies, so long as they exhibit the desired biological activity (U.S. Pat. No. 4,816,567; and Morrison et al. (1984) 81:6851-6855).","\u201cHumanized\u201d forms of non-human (e.g., murine) antibodies are chimeric antibodies which contain minimal sequence derived from non-human immunoglobulin. For the most part, humanized antibodies are human immunoglobulins (recipient antibody) in which residues from a hypervariable region of the recipient are replaced by residues from a hypervariable region of a non-human species (donor antibody) such as mouse, rat, rabbit or nonhuman primate having the desired specificity, affinity, and capacity. In some instances, Fv framework region (FR) residues of the human immunoglobulin are replaced by corresponding non-human residues. Furthermore, humanized antibodies may comprise residues which are not found in the recipient antibody or in the donor antibody. These modifications are made to further refine antibody performance. In general, the humanized antibody will comprise substantially all of at least one, and typically two, variable domains, in which all or substantially all of the hypervariable loops correspond to those of a non-human immunoglobulin and all or substantially all of the FR regions are those of a human immunoglobulin sequence. The humanized antibody optionally also will comprise at least a portion of an immunoglobulin constant region (Fc), typically that of a human immunoglobulin. For further details, see Jones et al. (1986) 321:522-525; Riechmann et al. (1988) 332:323-329; and Presta (1992) 2:593-596.","A \u201cspecies-dependent antibody\u201d is one which has a stronger binding affinity for an antigen from a first mammalian species than it has for a homologue of that antigen from a second mammalian species. Normally, the species-dependent antibody \u201cbinds specifically\u201d to a human antigen (i.e. has a binding affinity (K) value of no more than about 1\u00d710M, preferably no more than about 1\u00d710M and most preferably no more than about 1\u00d710M) but has a binding affinity for a homologue of the antigen from a second nonhuman mammalian species which is at least about 50 fold, or at least about 500 fold, or at least about 1000 fold, weaker than its binding affinity for the human antigen. The species-dependent antibody can be any of the various types of antibodies as defined above, but preferably is a humanized or human antibody.","As used herein, \u201cantibody mutant\u201d or \u201cantibody variant\u201d refers to an amino acid sequence variant of the species-dependent antibody wherein one or more of the amino acid residues of the species-dependent antibody have been modified. Such mutants necessarily have less than 100% sequence identity or similarity with the species-dependent antibody. In a preferred embodiment, the antibody mutant will have an amino acid sequence having at least 75% amino acid sequence identity or similarity with the amino acid sequence of either the heavy or light chain variable domain of the species-dependent antibody, more preferably at least 80%, more preferably at least 85%, more preferably at least 90%, and most preferably at least 95%. Identity or similarity with respect to this sequence is defined herein as the percentage of amino acid residues in the candidate sequence that are identical (i.e same residue) or similar (i e amino acid residue from the same group based on common side-chain properties, see below) with the species-dependent antibody residues, after aligning the sequences and introducing gaps, if necessary, to achieve the maximum percent sequence identity. None of N-terminal, C-terminal, or internal extensions, deletions, or insertions into the antibody sequence outside of the variable domain shall be construed as affecting sequence identity or similarity.","An \u201cisolated\u201d antibody is one which has been identified and separated and\/or recovered from a component of its natural environment. Contaminant components of its natural environment are materials which would interfere with diagnostic or therapeutic uses for the antibody, and may include enzymes, hormones, and other proteinaceous or nonproteinaceous solutes. In preferred embodiments, the antibody will be purified (1) to greater than 95% by weight of antibody as determined by the Lowry method, and most preferably more than 99% by weight, (2) to a degree sufficient to obtain at least 15 residues of N-terminal or internal amino acid sequence by use of a spinning cup sequenator, or (3) to homogeneity by SDS-PAGE under reducing or nonreducing conditions using Coomassie blue or, preferably, silver stain. Isolated antibody includes the antibody in situ within recombinant cells since at least one component of the antibody's natural environment will not be present. Ordinarily, however, isolated antibody will be prepared by at least one purification step.","As used herein, \u201cantibody variable domain\u201d refers to the portions of the light and heavy chains of antibody molecules that include amino acid sequences of Complementarity Determining Regions (CDRs; ie., CDR1, CDR2, and CDR3), and Framework Regions (FRs). Vrefers to the variable domain of the heavy chain. Vrefers to the variable domain of the light chain. According to the methods used in this invention, the amino acid positions assigned to CDRs and FRs may be defined according to Kabat (Sequences of Proteins of Immunological Interest (National Institutes of Health, Bethesda, Md., 1987 and 1991)) Amino acid numbering of antibodies or antigen binding fragments is also according to that of Kabat.","As used herein, the term \u201cComplementarity Determining Regions (CDRs; ie., CDR1, CDR2, and CDR3) refers to the amino acid residues of an antibody variable domain the presence of which are necessary for antigen binding. Each variable domain typically has three CDR regions identified as CDR1, CDR2 and CDR3. Each complementarity determining region may comprise amino acid residues from a \u201ccomplementarity determining region\u201d as defined by Kabat (i.e. about residues 24-34 (L1), 50-56 (L2) and 89-97 (L3) in the light chain variable domain and 31-35 (H1), 50-65 (H2) and 95-102 (H3) in the heavy chain variable domain; Kabat et al., 5th Ed. Public Health Service, National Institutes of Health, Bethesda, Md. (1991)) and\/or those residues from a \u201chypervariable loop\u201d (i.e. about residues 26-32 (L1), 50-52 (L2) and 91-96 (L3) in the light chain variable domain and 26-32 (H1), 53-55 (H2) and 96-101 (H3) in the heavy chain variable domain; Chothia and Lesk (1987) 196:901-917). In some instances, a complementarity determining region can include amino acids from both a CDR region defined according to Kabat and a hypervariable loop. For example, the CDRH1 of the heavy chain of antibody 4D5 includes amino acids 26 to 35.","\u201cFramework regions\u201d (hereinafter FR) are those variable domain residues other than the CDR residues. Each variable domain typically has four FRs identified as FR1, FR2, FR3 and FR4. If the CDRs are defined according to Kabat, the light chain FR residues are positioned at about residues 1-23 (LCFR1), 35-49 (LCFR2), 57-88 (LCFR3), and 98-107 (LCFR4) and the heavy chain FR residues are positioned about at residues 1-30 (HCFR1), 36-49 (HCFR2), 66-94 (HCFR3), and 103-113 (HCFR4) in the heavy chain residues. If the CDRs comprise amino acid residues from hypervariable loops, the light chain FR residues are positioned about at residues 1-25 (LCFR1), 33-49 (LCFR2), 53-90 (LCFR3), and 97-107 (LCFR4) in the light chain and the heavy chain FR residues are positioned about at residues 1-25 (HCFR1), 33-52 (HCFR2), 56-95 (HCFR3), and 102-113 (HCFR4) in the heavy chain residues. In some instances, when the CDR comprises amino acids from both a CDR as defined by Kabat and those of a hypervariable loop, the FR residues will be adjusted accordingly. For example, when CDRH1 includes amino acids H26-H35, the heavy chain FR1 residues are at positions 1-25 and the FR2 residues are at positions 36-49.","As used herein, \u201ccodon set\u201d refers to a set of different nucleotide triplet sequences used to encode desired variant amino acids. A set of oligonucleotides can be synthesized, for example, by solid phase synthesis, including sequences that represent all possible combinations of nucleotide triplets provided by the codon set and that will encode the desired group of amino acids. A standard form of codon designation is that of the IUB code, which is known in the art and described herein. A codon set typically is represented by 3 capital letters in italics, eg. NNK, NNS, XYZ, DVK and the like. A \u201cnon-random codon set\u201d, as used herein, thus refers to a codon set that encodes select amino acids that fulfill partially, preferably completely, the criteria for amino acid selection as described herein. Synthesis of oligonucleotides with selected nucleotide \u201cdegeneracy\u201d at certain positions is well known in that art, for example the TRIM approach (Knappek et al. (1999) 296:57-86); Garrard & Henner (1993) 128:103). Such sets of oligonucleotides having certain codon sets can be synthesized using commercial nucleic acid synthesizers (available from, for example, Applied Biosystems, Foster City, Calif.), or can be obtained commercially (for example, from Life Technologies, Rockville, Md.). Therefore, a set of oligonucleotides synthesized having a particular codon set will typically include a plurality of oligonucleotides with different sequences, the differences established by the codon set within the overall sequence. Oligonucleotides, as used according to the invention, have sequences that allow for hybridization to a variable domain nucleic acid template and also can, but does not necessarily, include restriction enzyme sites useful for, for example, cloning purposes.","An \u201cFv\u201d fragment is an antibody fragment which contains a complete antigen recognition and binding site. This region consists of a dimer of one heavy and one light chain variable domain in tight association, which can be covalent in nature, for example in scFv. It is in this configuration that the three CDRs of each variable domain interact to define an antigen binding site on the surface of the V-Vdimer Collectively, the six CDRs or a subset thereof confer antigen binding specificity to the antibody. However, even a single variable domain (or half of an Fv comprising only three CDRs specific for an antigen) has the ability to recognize and bind antigen, although usually at a lower affinity than the entire binding site.","The \u201cFab\u201d fragment contains a variable and constant domain of the light chain and a variable domain and the first constant domain (CH1) of the heavy chain. F(ab\u2032)antibody fragments comprise a pair of Fab fragments which are generally covalently linked near their carboxy termini by hinge cysteines between them. Other chemical couplings of antibody fragments are also known in the art.","\u201cSingle-chain Fv\u201d or \u201cscFv\u201d antibody fragments comprise the Vand Vdomains of antibody, wherein these domains are present in a single polypeptide chain. Generally the Fv polypeptide further comprises a polypeptide linker between the Vand Vdomains, which enables the scFv to form the desired structure for antigen binding. For a review of scFv, see Pluckthun in , Vol 113, Rosenburg and Moore eds. Springer-Verlag, New York, pp. 269-315 (1994).","The term \u201cdiabodies\u201d refers to small antibody fragments with two antigen-binding sites, which fragments comprise a heavy chain variable domain (V) connected to a light chain variable domain (V) in the same polypeptide chain (Vand V). By using a linker that is too short to allow pairing between the two domains on the same chain, the domains are forced to pair with the complementary domains of another chain and create two antigen-binding sites. Diabodies are described more fully in, for example, EP 404,097; WO 93\/11161; and Hollinger et al. (1993) 90:6444-6448.","The expression \u201clinear antibodies\u201d refers to the antibodies described in Zapata et al. (1995 8(10):1057-1062). Briefly, these antibodies comprise a pair of tandem Fd segments (V-C1-V-C1) which, together with complementary light chain polypeptides, form a pair of antigen binding regions. Linear antibodies can be bispecific or monospecific.","As used herein, \u201clibrary\u201d refers to a plurality of antibody or antibody fragment sequences (for example, polypeptides of the invention), or the nucleic acids that encode these sequences, the sequences being different in the combination of variant amino acids that are introduced into these sequences according to the methods of the invention.","\u201cPhage display\u201d is a technique by which variant polypeptides are displayed as fusion proteins to at least a portion of coat protein on the surface of phage, e.g., filamentous phage, particles. A utility of phage display lies in the fact that large libraries of randomized protein variants can be rapidly and efficiently sorted for those sequences that bind to a target antigen with high affinity. Display of peptide and protein libraries on phage has been used for screening millions of polypeptides for ones with specific binding properties. Polyvalent phage display methods have been used for displaying small random peptides and small proteins through fusions to either gene III or gene VIII of filamentous phage. Wells and Lowman (1992) 3:355-362, and references cited therein. In a monovalent phage display, a protein or peptide library is fused to a gene III or a portion thereof, and expressed at low levels in the presence of wild type gene III protein so that phage particles display one copy or none of the fusion proteins. Avidity effects are reduced relative to polyvalent phage so that sorting is on the basis of intrinsic ligand affinity, and phagemid vectors are used, which simplify DNA manipulations. Lowman and Wells (1991) 3:205-0216.","A \u201cphagemid\u201d is a plasmid vector having a bacterial origin of replication, e.g., ColE1, and a copy of an intergenic region of a bacteriophage. The phagemid may be used on any known bacteriophage, including filamentous bacteriophage and lambdoid bacteriophage. The plasmid will also generally contain a selectable marker for antibiotic resistance. Segments of DNA cloned into these vectors can be propagated as plasmids. When cells harboring these vectors are provided with all genes necessary for the production of phage particles, the mode of replication of the plasmid changes to rolling circle replication to generate copies of one strand of the plasmid DNA and package phage particles. The phagemid may form infectious or non-infectious phage particles. This term includes phagemids which contain a phage coat protein gene or fragment thereof linked to a heterologous polypeptide gene as a gene fusion such that the heterologous polypeptide is displayed on the surface of the phage particle.","The term \u201cphage vector\u201d means a double stranded replicative form of a bacteriophage containing a heterologous gene and capable of replication. The phage vector has a phage origin of replication allowing phage replication and phage particle formation. The phage is preferably a filamentous bacteriophage, such as an M13, fl, fd, Pf3 phage or a derivative thereof, or a lambdoid phage, such as lambda, 21, phi80, phi81, 82, 424, 434, etc., or a derivative thereof.","As used herein, \u201csolvent accessible position\u201d refers to a position of an amino acid residue in the variable regions of the heavy and light chains of a source antibody or antigen binding fragment that is determined, based on structure, ensemble of structures and\/or modeled structure of the antibody or antigen binding fragment, as potentially available for solvent access and\/or contact with a molecule, such as an antibody-specific antigen. These positions are typically found in the CDRs and on the exterior of the protein. The solvent accessible positions of an antibody or antigen binding fragment, as defined herein, can be determined using any of a number of algorithms known in the art. Preferably, solvent accessible positions are determined using coordinates from a 3-dimensional model of an antibody, preferably using a computer program such as the Insightll program (Accelrys, San Diego, Calif.). Solvent accessible positions can also be determined using algorithms known in the art (e.g., Lee and Richards (1971) 55, 379 and Connolly (1983) 16, 548). Determination of solvent accessible positions can be performed using software suitable for protein modeling and 3-dimensional structural information obtained from an antibody. Software that can be utilized for these purposes includes SYBYL Biopolymer Module software (Tripos Associates). Generally and preferably, where an algorithm (program) requires a user input size parameter, the \u201csize\u201d of a probe which is used in the calculation is set at about 1.4 Angstrom or smaller in radius. In addition, determination of solvent accessible regions and area methods using software for personal computers has been described by Pacios (1994) 18(4): 377-386.","An \u201cangiogenic factor or agent\u201d is a growth factor which stimulates the development of blood vessels, e.g., promote angiogenesis, endothelial cell growth, stabiliy of blood vessels, and\/or vasculogenesis, etc. For example, angiogenic factors, include, but are not limited to, e.g., VEGF and members of the VEGF family, P1GF, PDGF family, fibroblast growth factor family (FGFs), TIE ligands (Angiopoietins), ephrins, Del-1, fibroblast growth factors: acidic (aFGF) and basic (bFGF), Follistatin, Granulocyte colony-stimulating factor (G-CSF), Hepatocyte growth factor (HGF) \/scatter factor (SF), Interleukin-8 (IL-8), Leptin, Midkine,neuropilins, Placental growth factor, Platelet-derived endothelial cell growth factor (PD-ECGF), Platelet-derived growth factor, especially PDGF-BB or PDGFR-beta, Pleiotrophin (PTN), Progranulin, Proliferin, Transforming growth factor-alpha (TGF-alpha), Transforming growth factor-beta (TGF-beta), Tumor necrosis factor-alpha (TNF-alpha), etc. It would also include factors that accelerate wound healing, such as growth hormone, insulin-like growth factor-1 (IGF-I), VIGF, epidermal growth factor (EGF), CTGF and members of its family, and TGF-alpha and TGF-beta. See, e.g., Klagsbrun and D'Amore (1991) 53:217-39; Streit and Detmar (2003) 22:3172-3179; Ferrara & Alitalo (1999) 5(12):1359-1364; Tonini et al. (2003) 22:6549-6556 (e.g., Table 1 listing known angiogenic factors); and, Sato (2003) 8:200-206.","An \u201canti-angiogenesis agent\u201d or \u201cangiogenesis inhibitor\u201d refers to a small molecular weight substance, an polynucleotide, an polypeptide, an isolated protein, a recombinant protein, an antibody, or conjugates or fusion proteins thereof, that inhibits angiogenesis, vasculogenesis, or undesirable vascular permeability, either directly or indirectly. It should be understood that theanti-angiogenesis agent includes those agents that bind and block the angiogenic activity of the angiogenic factor or its receptor. For example, an anti-angiogenesis agent is an antibody or other antagonist to an angiogenic agent as defined above, e.g., antibodies to VEGF-A or to the VEGF-A receptor (e.g., KDR receptor or Flt-1 receptor), anti-PDGFR inhibitors such as Gleevec\u2122 (Imatinib Mesylate). Anti-angiogensis agents also include native angiogenesis inhibitors, e.g., angiostatin, endostatin, etc. See, e.g., Klagsbrun and D'Amore (1991) 53:217-39; Streit and Detmar (2003) 22:3172-3179 (e.g., Table 3 listing anti-angiogenic therapy in malignant melanoma); Ferrara & Alitalo (1999) 5(12):1359-1364; Tonini et al. (2003) 22:6549-6556 (e.g., Table 2 listing known antiangiogenic factors); and, Sato (2003) 8:200-206 (e.g., Table 1 listing anti-angiogenic agents used in clinical trials).","The term \u201cVEGF\u201d or \u201cVEGF-A\u201d as used herein refers to the 165-amino acid human vascular endothelial cell growth factor and related 121-, 189-, and 206-amino acid human vascular endothelial cell growth factors, as described by Leung et al. (1989) 246:1306, and Houck et al. (1991) 5:1806, together with the naturally occurring allelic and processed forms thereof The term \u201cVEGF\u201d also refers to VEGFs from non-human species such as mouse, rat or primate. Sometimes the VEGF from a specific species are indicated by terms such as hVEGF for human VEGF, mVEGF for murine VEGF, and etc. The term \u201cVEGF\u201d is also used to refer to truncated forms of the polypeptide comprising amino acids 8 to 109 or 1 to 109 of the 165-amino acid human vascular endothelial cell growth factor. Reference to any such forms of VEGF may be identified in the present application, e.g., by \u201cVEGF (8-109),\u201d \u201cVEGF (1-109)\u201d or \u201cVEGF.\u201d The amino acid positions for a \u201ctruncated\u201d native VEGF are numbered as indicated in the native VEGF sequence. For example, amino acid position 17 (methionine) in truncated native VEGF is also position 17 (methionine) in native VEGF. The truncated native VEGF has binding affinity for the KDR and Flt-1 receptors comparable to native VEGF.","An \u201canti-VEGF antibody\u201d is an antibody that binds to VEGF with sufficient affinity and specificity. Preferably, the anti-VEGF antibody of the invention can be used as a therapeutic agent in targeting and interfering with diseases or conditions wherein the VEGF activity is involved. An anti-VEGF antibody will usually not bind to other VEGF homologues such as VEGF-B or VEGF-C, nor other growth factors such as P1GF, PDGF or bFGF. A preferred anti-VEGF antibody is a monoclonal antibody that binds to the same epitope as the monoclonal anti-VEGF antibody A4.6.1 produced by hybridoma ATCC HB 10709. More preferably the anti-VEGF antibody is a recombinant humanized anti-VEGF monoclonal antibody generated according to Presta et al. (1997) 57:4593-4599, including but not limited to the antibody known as bevacizumab (BV; Avastin\u2122)","The anti-VEGF antibody \u201cBevacizumab (BV)\u201d, also known as \u201crhuMAb VEGF\u201d or Avastin\u00ae, is a recombinant humanized anti-VEGF monoclonal antibody generated according to Presta et al. (1997) 57:4593-4599. It comprises mutated human IgG1 framework regions and antigen-binding complementarity-determining regions from the murine anti-hVEGF monoclonal antibody A.4.6.1 that blocks binding of human VEGF to its receptors. Approximately 93% of the amino acid sequence of Bevacizumab, including most of the framework regions, is derived from human IgG1, and about 7% of the sequence is derived from the murine antibody A4.6.1. Bevacizumab has a molecular mass of about 149,000 daltons and is glycosylated.","A \u201cVEGF antagonist\u201d refers to a molecule capable of neutralizing, blocking, inhibiting, abrogating, reducing or interfering with VEGF activities including its binding to one or more VEGF receptors. VEGF antagonists include anti-VEGF antibodies and antigen-binding fragments thereof, receptor molecules and derivatives which bind specifically to VEGF thereby sequestering its binding to one or more receptors, anti-VEGF receptor antibodies and VEGF receptor antagonists such as small molecule inhibitors of the VEGFR tyrosine kinases.","\u201cTreatment\u201d refers to both therapeutic treatment and prophylactic or preventative measures. Those in need of treatment include those already with the disorder as well as those in which the disorder is to be prevented.","A \u201cdisorder\u201d is any condition that would benefit from treatment with the antibody. For example, mammals who suffer from or need prophylaxis against abnormal angiogenesis (excessive, inappropriate or uncontrolled angiogenesis) or vascular permeability. This includes chronic and acute disorders or diseases including those pathological conditions which predispose the mammal to the disorder in question. Non-limiting examples of disorders to be treated herein include malignant and benign tumors; non-leukemias and lymphoid malignancies; neuronal, glial, astrocytal, hypothalamic and other glandular, macrophagal, epithelial, stromal and blastocoelic disorders; and inflammatory, angiogenic and immunologic disorders.","Abnormal angiogenesis occurs when new blood vessels either grow excessively, insufficiently or inappropriately (e.g., the location, timing or onset of the angiogenesis being undesired from a medical standpoint) in a diseased state or such that it causes a diseased state. Excessive, inappropriate or uncontrolled angiogenesis occurs when there is new blood vessel growth that contributes to the worsening of the diseased state or causes a diseased state, such as in cancer, especially vascularized solid tumors and metastatic tumors (including colon, lung cancer (especially small-cell lung cancer), or prostate cancer), diseases caused by ocular neovascularisation, especially diabetic blindness, retinopathies, primarily diabetic retinopathy or age-related macular degeneration (AMD), psoriasis, psoriatic arthritis, haemangioblastoma such as haemangioma; inflammatory renal diseases, such as glomerulonephritis, especially mesangioproliferative glomerulonephritis, haemolytic uremic syndrome, diabetic nephropathy or hypertensive nephrosclerosis; various imflammatory diseases, such as arthritis, especially rheumatoid arthritis, inflammatory bowel disease, psorsasis, sarcoidosis, arterial arteriosclerosis and diseases occurring after transplants, endometriosis or chronic asthma and more than 70 other conditions. The new blood vessels can feed the diseased tissues, destroy normal tissues, and in the case of cancer, the new vessels can allow tumor cells to escape into the circulation and lodge in other organs (tumor metastases). Insufficient angiogenesis occurs when there is inadequate blood vessels growth that contributes to the worsening of a diseased state, e.g., in diseases such as coronary artery disease, stroke, and delayed wound healing. Further, ulcers, strokes, and heart attacks can result from the absence of angiogenesis that normally required for natural healing. The present invention contemplates treating those patients that are at risk of developing the above-mentioned illnesses.","Other patients that are candidates for receiving the antibodies or polypeptides of this invention have, or are at risk for developing, abnormal proliferation of fibrovascular tissue, acne rosacea, acquired immune deficiency syndrome, artery occlusion, atopic keratitis, bacterial ulcers, Bechets disease, blood borne tumors, carotid obstructive disease, choroidal neovascularization, chronic inflammation, chronic retinal detachment, chronic uveitis, chronic vitritis, contact lens overwear, corneal graft rejection, corneal neovascularization, corneal graft neovascularization, Crohn's disease, Eales disease, epidemic keratoconjunctivitis, fungal ulcers, Herpes simplex infections, Herpes zoster infections, hyperviscosity syndromes, Kaposi's sarcoma, leukemia, lipid degeneration, Lyme's disease, marginal keratolysis, Mooren ulcer, Mycobacteria infections other than leprosy, myopia, ocular neovascular disease, optic pits, Osler-Weber syndrome (Osler-Weber-Rendu, osteoarthritis, Pagets disease, pars planitis, pemphigoid, phylectenulosis, polyarteritis, post-laser complications, protozoan infections, pseudoxanthoma elasticum, pterygium keratitis sicca, radial keratotomy, retinal neovascularization, retinopathy of prematurity, retrolental fibroplasias, sarcoid, scleritis, sickle cell anemia, Sogrens syndrome, solid tumors, Stargarts disease, Steven's Johnson disease, superior limbic keratitis, syphilis, systemic lupus, Terrien's marginal degeneration, toxoplasmosis, trauma, tumors of Ewing sarcoma, tumors of neuroblastoma, tumors of osteosarcoma, tumors of retinoblastoma, tumors of rhabdomyosarcoma, ulcerative colitis, vein occlusion, Vitamin A deficiency and Wegeners sarcoidosis, undesired angiogenesis associated with diabetes, parasitic diseases, abnormal wound healing, hypertrophy following surgery, injury or trauma, inhibition of hair growth, inhibition of ovulation and corpus luteum formation, inhibition of implantation and inhibition of embryo development in the uterus.","Anti-angiogenesis therapies are useful in the general treatment of graft rejection, lung inflammation, nephrotic syndrome, preeclampsia, pericardial effusion, such as that associated with pericarditis, and pleural effusion, diseases and disorders characterized by undesirable vascular permeability, e.g., edema associated with brain tumors, ascites associated with malignancies, Meigs' syndrome, lung inflammation, nephrotic syndrome, pericardial effusion, pleural effusion, permeability associated with cardiovascular diseases such as the condition following myocardial infarctions and strokes and the like.","Other angiogenesis-dependent diseases according to this invention include angiofibroma (abnormal blood of vessels which are prone to bleeding), neovascular glaucoma (growth of blood vessels in the eye), arteriovenous malformations (abnormal communication between arteries and veins), nonunion fractures (fractures that will not heal), atherosclerotic plaques (hardening of the arteries), pyogenic granuloma (common skin lesion composed of blood vessels), scleroderma (a form of connective tissue disease), hemangioma (tumor composed of blood vessels), trachoma (leading cause of blindness in the third world), hemophilic joints, vascular adhesions and hypertrophic scars (abnormal scar formation).","The terms \u201ccancer\u201d and \u201ccancerous\u201d refer to or describe the physiological condition in mammals that is typically characterized by unregulated cell growth. Examples of cancer include but are not limited to, carcinoma, lymphoma, blastoma, sarcoma, and leukemia. More particular examples of such cancers include squamous cell cancer, lung cancer (including small-cell lung cancer, non-small cell lung cancer, adenocarcinoma of the lung, and squamous carcinoma of the lung), cancer of the peritoneum, hepatocellular cancer, gastric or stomach cancer (including gastrointestinal cancer), pancreatic cancer, glioblastoma, cervical cancer, ovarian cancer, liver cancer, bladder cancer, hepatoma, breast cancer, colon cancer, colorectal cancer, endometrial or uterine carcinoma, salivary gland carcinoma, kidney or renal cancer, liver cancer, prostate cancer, vulval cancer, thyroid cancer, hepatic carcinoma and various types of head and neck cancer, as well as B-cell lymphoma (including low grade\/follicular non-Hodgkin's lymphoma (NHL); small lymphocytic (SL) NHL; intermediate grade\/follicular NHL; intermediate grade diffuse NHL; high grade immunoblastic NHL; high grade lymphoblastic NHL; high grade small non-cleaved cell NHL; bulky disease NHL; mantle cell lymphoma; AIDS-related lymphoma; and Waldenstrom's Macroglobulinemia); chronic lymphocytic leukemia (CLL); acute lymphoblastic leukemia (ALL); Hairy cell leukemia; chronic myeloblastic leukemia; and post-transplant lymphoproliferative disorder (PTLD), as well as abnormal vascular proliferation associated with phakomatoses, edema (such as that associated with brain tumors), and Meigs' syndrome.","The term \u201canti-neoplastic composition\u201d refers to a composition useful in treating cancer comprising at least one active therapeutic agent, e.g., \u201canti-cancer agent.\u201d Examples of therapeutic agents (anti-cancer agents) include, but are limited to, e.g., chemotherapeutic agents, growth inhibitory agents, cytotoxic agents, agents used in radiation therapy, anti-angiogenesis agents, apoptotic agents, anti-tubulin agents, and other-agents to treat cancer, such as anti-HER-2 antibodies, anti-CD20 antibodies, an epidermal growth factor receptor (EGFR) antagonist (e.g., a tyrosine kinase inhibitor), HER1\/EGFR inhibitor (e.g., erlotinib (Tarceva\u2122), platelet derived growth factor inhibitors (e.g., Gleevec\u2122 (Imatinib Mesylate)), a COX-2 inhibitor (e.g., celecoxib), interferons, cytokines, antagonists (e.g., neutralizing antibodies) that bind to one or more of the following targets ErbB2, ErbB3, ErbB4, PDGFR-beta, BlyS, APRIL, BCMA or VEGF receptor(s), TRAIL\/Apo2, and other bioactive and organic chemical agents, etc. Combinations thereof are also included in the invention.","The term \u201ccytotoxic agent\u201d as used herein refers to a substance that inhibits or prevents the function of cells and\/or causes destruction of cells. The term is intended to include radioactive isotopes (e.g., I, I, Yand Re), chemotherapeutic agents, and toxins such as enzymatically active toxins of bacterial, fungal, plant or animal origin, or fragments thereof","A \u201cchemotherapeutic agent\u201d is a chemical compound useful in the treatment of cancer. Examples of chemotherapeutic agents include is a chemical compound useful in the treatment of cancer. Examples of chemotherapeutic agents include alkylating agents such as thiotepa and CYTOXAN\u00ae cyclosphosphamide; alkyl sulfonates such as busulfan, improsulfan and piposulfan; aziridines such as benzodopa, carboquone, meturedopa, and uredopa; ethylenimines and methylamelamines including altretamine, triethylenemelamine, trietylenephosphoramide, triethiylenethiophosphoramide and trimethylolomelamine; acetogenins (especially bullatacin and bullatacinone); a camptothecin (including the synthetic analogue topotecan); bryostatin; callystatin; CC-1065 (including its adozelesin, carzelesin and bizelesin synthetic analogues); cryptophycins (particularly cryptophycin 1 and cryptophycin 8); dolastatin; duocarmycin (including the synthetic analogues, KW-2189 and CB1-TM1); eleutherobin; pancratistatin; a sarcodictyin; spongistatin; nitrogen mustards such as chlorambucil, chlornaphazine, cholophosphamide, estramustine, ifosfamide, mechlorethamine, mechlorethamine oxide hydrochloride, melphalan, novembichin, phenesterine, prednimustine, trofosfamide, uracil mustard; nitrosureas such as carmustine, chlorozotocin, fotemustine, lomustine, nimustine, and ranimnustine; antibiotics such as the enediyne antibiotics (e.g., calicheamicin, especially calicheamicin gammall and calicheamicin omegall (see, e.g., Agnew (1994) 33:183-186); dynemicin, including dynemicin A; bisphosphonates, such as clodronate; an esperamicin; as well as neocarzinostatin chromophore and related chromoprotein enediyne antiobiotic chromophores), aclacinomysins, actinomycin, authramycin, azaserine, bleomycins, cactinomycin, carabicin, carminomycin, carzinophilin, chromomycinis, dactinomycin, daunorubicin, detorubicin, 6-diazo-5-oxo-L-norleucine, ADRIAMYCIN\u00ae doxorubicin (including morpholino-doxorubicin, cyanomorpholino-doxorubicin, 2-pyrrolino-doxorubicin and deoxydoxorubicin), epirubicin, esorubicin, idarubicin, marcellomycin, mitomycins such as mitomycin C, mycophenolic acid, nogalamycin, olivomycins, peplomycin, potfiromycin, puromycin, quelamycin, rodorubicin, streptonigrin, streptozocin, tubercidin, ubenimex, zinostatin, zorubicin; anti-metabolites such as methotrexate and 5-fluorouracil (5-FU); folic acid analogues such as denopterin, methotrexate, pteropterin, trimetrexate; purine analogs such as fludarabine, 6-mercaptopurine, thiamiprine, thioguanine; pyrimidine analogs such as ancitabine, azacitidine, 6-azauridine, carmofur, cytarabine, dideoxyuridine, doxifluridine, enocitabine, floxuridine; androgens such as calusterone, dromostanolone propionate, epitiostanol, mepitiostane, testolactone; anti-adrenals such as aminoglutethimide, mitotane, trilostane; folic acid replenisher such as frolinic acid; aceglatone; aldophosphamide glycoside; aminolevulinic acid; eniluracil; amsacrine; bestrabucil; bisantrene; edatraxate; defofamine; demecolcine; diaziquone; elfornithine; elliptinium acetate; an epothilone; etoglucid; gallium nitrate; hydroxyurea; lentinan; lonidainine; maytansinoids such as maytansine and ansamitocins; mitoguazone; mitoxantrone; mopidanmol; nitraerine; pentostatin; phenamet; pirarubicin; losoxantrone; podophyllinic acid; 2-ethylhydrazide; procarbazine; PSK\u00ae polysaccharide complex (JHS Natural Products, Eugene, Oreg.); razoxane; rhizoxin; sizofiran; spirogermanium; tenuazonic acid; triaziquone; 2,2\u2032,2\u2033-trichlorotriethylamine; trichothecenes (especially T-2 toxin, verracurin A, roridin A and anguidine); urethan; vindesine; dacarbazine; mannomustine; mitobronitol; mitolactol; pipobroman; gacytosine; arabinoside (\u201cAra-C\u201d); cyclophosphamide; thiotepa; taxoids, e.g., TAXOL\u00ae paclitaxel (Bristol-Myers Squibb Oncology, Princeton, N.J.), ABRAXANE\u2122 Cremophor-free, albumin-engineered nanoparticle formulation of paclitaxel (American Pharmaceutical Partners, Schaumberg, Ill.), and TAXOTERE\u00ae doxetaxel (Rh\u00f4ne-Poulenc Rorer, Antony, France); chloranbucil; GEMZAR\u00ae gemcitabine; 6-thioguanine; mercaptopurine; methotrexate; platinum analogs such as cisplatin and carboplatin; vinblastine; platinum; etoposide (VP-16); ifosfamide; mitoxantrone; vincristine; NAVELBINE\u00ae vinorelbine; novantrone; teniposide; edatrexate; daunomycin; aminopterin; xeloda; ibandronate; irinotecan (Camptosar, CPT-11) (including the treatment regimen of irinotecan with 5-FU and leucovorin); topoisomerase inhibitor RFS 2000; difluorometlhylornithine (DMFO); retinoids such as retinoic acid; capecitabine; combretastatin; leucovorin (LV); oxaliplatin, including the oxaliplatin treatment regimen (FOLFOX); inhibitors of PKC-alpha, Raf, H-Ras, EGFR (e.g., erlotinib (Tarceva\u2122)) and VEGF-A that reduce cell proliferation and pharmaceutically acceptable salts, acids or derivatives of any of the above.","Also included in this definition are anti-hormonal agents that act to regulate or inhibit hormone action on tumors such as anti-estrogens and selective estrogen receptor modulators (SERMs), including, for example, tamoxifen (including NOLVADEX\u00ae tamoxifen), raloxifene, droloxifene, 4-hydroxytamoxifen, trioxifene, keoxifene, LY117018, onapristone, and FARESTON\u2022toremifene;","aromatase inhibitors that inhibit the enzyme aromatase, which regulates estrogen production in the adrenal glands, such as, for example, 4(5)-imidazoles, aminoglutethimide, MEGASE\u00ae megestrol acetate, AROMASIN\u00ae exemestane, formestanie, fadrozole, RIVISOR\u00ae vorozole, FEMARA\u00ae letrozole, and ARIMIDEX\u00ae anastrozole; and anti-androgens such as flutamide, nilutamide, bicalutamide, leuprolide, and goserelin; as well as troxacitabine (a 1,3-dioxolane nucleoside cytosine analog); antisense oligonucleotides, particularly those which inhibit expression of genes in signaling pathways implicated in abherant cell proliferation, such as, for example, PKC-alpha, Raf and H-Ras; ribozymes such as a VEGF expression inhibitor (e.g., ANGIOZYME\u00ae ribozyme) and a HER2 expression inhibitor; vaccines such as gene therapy vaccines, for example, ALLOVECTIN\u00ae vaccine, LEUVECTIN\u00ae vaccine, and VAXID\u00ae vaccine; PROLEUKIN\u00ae rIL-2; LURTOTECAN\u00ae topoisomerase 1 inhibitor; ABARELIX\u00ae rmRH; Vinorelbine and Esperamicins (see U.S. Pat. No. 4,675,187), and pharmaceutically acceptable salts, acids or derivatives of any of the above.","The term \u201cprodrug\u201d as used in this application refers to a precursor or derivative form of a pharmaceutically active substance that is less cytotoxic to tumor cells compared to the parent drug and is capable of being enzymatically activated or converted into the more active parent form. See, e.g., Wilman (1986) \u201cProdrugs in Cancer Chemotherapy\u201d 14, pp. 375-382, 615th Meeting Belfast and Stella et al. (1985). \u201cProdrugs: A Chemical Approach to Targeted Drug Delivery,\u201d , Borchardt et al, (ed.), pp. 247-267, Humana Press. The prodrugs of this invention include, but are not limited to, phosphate-containing prodrugs, thiophosphate-containing prodrugs, sulfate-containing prodrugs, peptide-containing prodrugs, D-amino acid-modified prodrugs, glycosylated prodrugs, \u03b2-lactam-containing prodrugs, optionally substituted phenoxyacetamide-containing prodrugs or optionally substituted phenylacetamide-containing prodrugs, 5-fluorocytosine and other 5-fluorouridine prodrugs which can be converted into the more active cytotoxic free drug. Examples of cytotoxic drugs that can be derivatized into a prodrug form for use in this invention include, but are not limited to, those chemotherapeutic agents described above.","An \u201cisolated\u201d nucleic acid molecule is a nucleic acid molecule that is identified and separated from at least one contaminant nucleic acid molecule with which it is ordinarily associated in the natural source of the antibody nucleic acid. An isolated nucleic acid molecule is other than in the form or setting in which it is found in nature. Isolated nucleic acid molecules therefore are distinguished from the nucleic acid molecule as it exists in natural cells. However, an isolated nucleic acid molecule includes a nucleic acid molecule contained in cells that ordinarily express the antibody where, for example, the nucleic acid molecule is in a chromosomal location different from that of natural cells.","The expression \u201ccontrol sequences\u201d refers to DNA sequences necessary for the expression of an operably linked coding sequence in a particular host organism. The control sequences that are suitable for prokaryotes, for example, include a promoter, optionally an operator sequence, and a ribosome binding site. Eukaryotic cells are known to utilize promoters, polyadenylation signals, and enhancers.","Nucleic acid is \u201coperably linked\u201d when it is placed into a functional relationship with another nucleic acid sequence. For example, DNA for a presequence or secretory leader is operably linked to DNA for a polypeptide if it is expressed as a preprotein that participates in the secretion of the polypeptide; a promoter or enhancer is operably linked to a coding sequence if it affects the transcription of the sequence; or a ribosome binding site is operably linked to a coding sequence if it is positioned so as to facilitate translation. Generally, \u201coperably linked\u201d means that the DNA sequences being linked are contiguous, and, in the case of a secretory leader, contiguous and in reading phase. However, enhancers do not have to be contiguous. Linking is accomplished by ligation at convenient restriction sites. If such sites do not exist, the synthetic oligonucleotide adaptors or linkers are used in accordance with conventional practice.","As used herein, the expressions \u201ccell,\u201d \u201ccell line,\u201d and \u201ccell culture\u201d are used interchangeably and all such designations include progeny. Thus, the words \u201ctransformants\u201d and \u201ctransformed cells\u201d include the primary subject cell and cultures derived therefrom without regard for the number of transfers. It is also understood that all progeny may not be precisely identical in DNA content, due to deliberate or inadvertent mutations. Mutant progeny that have the same function or biological activity as screened for in the originally transformed cell are included. Where distinct designations are intended, it will be clear from the context.","Production of Anti-NRP1 Antibodies","The invention herein provides novel anti-NRP1 antibodies. Exemplary methods for generating antibodies are described in more detail in the following sections.","The novel anti-NRP1 antibodies are selected using the NRP1 antigen derived from a mammalian species. Preferably the antigen is human NRP1 (hNRP1). However, NRPs from other species such as murine NRP1 (mNRP1) can also be used as the target antigen. The NRP antigens from various mammalian species may be isolated from natural sources. In other embodiments, the antigen is produced recombinantly or made using other synthetic methods known in the art.","The antibody selected will normally have a sufficiently strong binding affinity for the NRP1 antigen. For example, the antibody may bind hNRP1 with a Kvalue of no more than about 5 nM, preferably no more than about 2 nM, and more preferably no more than about 500 \u03bcM. Antibody affinities may be determined by a surface plasmon resonance based assay (such as the BlAcore assay as described in Examples); enzyme-linked immunoabsorbent assay (ELISA); and competition assays (e.g. RIA's), for example.","Also, the antibody may be subject to other biological activity assays, e.g., in order to evaluate its effectiveness as a therapeutic. Such assays are known in the art and depend on the target antigen and intended use for the antibody. Examples include the HUVEC inhibition assay (as described in the Examples below); tumor cell growth inhibition assays (as described in WO 89\/06692, for example); antibody-dependent cellular cytotoxicity (ADCC) and complement-mediated cytotoxicity (CDC) assays (U.S. Pat. No. 5,500,362); and agonistic activity or hematopoiesis assays (see WO 95\/27062).","To screen for antibodies which bind to a particular epitope on the antigen of interest, a routine cross-blocking assay such as that described in , Cold Spring Harbor Laboratory, Ed Harlow and David Lane (1988), can be performed. Alternatively, epitope mapping, e.g. as described in Champe et al. (1995) 270:1388-1394, can be performed to determine whether the antibody binds an epitope of interest.","Generation of Novel Anti-NRP1 Antibodies from Synthetic Antibody Phage Libraries","In a preferred embodiment, the invention provides a method for generating and selecting novel anti-NRP1 antibodies using a unique phage display approach. The approach involves generation of synthetic antibody phage libraries based on single framework template, design of sufficient diversities within variable domains, display of polypeptides having the diversified variable domains, selection of candidate antibodies with high affinity to target NRP1 antigen, and isolation of the selected antibodies.","Details of the phage display methods can be found, for example, in WO03\/102157 published Dec. 11, 2003.","In one aspect, the antibody libraries used in the invention can be generated by mutating the solvent accessible and\/or highly diverse positions in at least one CDR of an antibody variable domain. Some or all of the CDRs can be mutated using the methods provided herein. In some embodiments, it may be preferable to generate diverse antibody libraries by mutating positions in CDRH1, CDRH2 and CDRH3 to form a single library or by mutating positions in CDRL3 and CDRH3 to form a single library or by mutating positions in CDRL3 and CDRH1, CDRH2 and CDRH3 to form a single library.","A library of antibody variable domains can be generated, for example, having mutations in the solvent accessible and\/or highly diverse positions of CDRH1, CDRH2 and CDRH3. Another library can be generated having mutations in CDRL1, CDRL2 and CDRL3. These libraries can also be used in conjunction with each other to generate binders of desired affinities. For example, after one or more rounds of selection of heavy chain libraries for binding to a target antigen, a light chain library can be replaced into the population of heavy chain binders for further rounds of selection to increase the affinity of the binders.","Preferably, a library is created by substitution of original amino acids with variant amino acids in the CDRH3 region of the variable region of the heavy chain sequence. The resulting library can contain a plurality of antibody sequences, wherein the sequence diversity is primarily in the CDRH3 region of the heavy chain sequence.","In one aspect, the library is created in the context of the humanized antibody 4D5 sequence, or the sequence of the framework amino acids of the humanized antibody 4D5 sequence. Preferably, the library is created by substitution of at least residues 95-100a of the heavy chain with amino acids encoded by the DVK codon set, wherein the DVK codon set is used to encode a set of variant amino acids for every one of these positions. An example of an oligonucleotide set that is useful for creating these substitutions comprises the sequence (DVK). In some embodiments, a library is created by substitution of residues 95-100a with amino acids encoded by both DVK and NNK codon sets. An example of an oligonucleotide set that is useful for creating these substitutions comprises the sequence (DVK)(NNK). In another embodiment, a library is created by substitution of at least residues 95-100a with amino acids encoded by both DVK and NNK codon sets. An example of an oligonucleotide set that is useful for creating these substitutions comprises the sequence (DVK)(NNK). Another example of an oligonucleotide set that is useful for creating these substitutions comprises the sequence (NNK). Other examples of suitable oligonucleotide sequences can be determined by one skilled in the art according to the criteria described herein.","In another embodiment, different CDRH3 designs are utilized to isolate high affinity binders and to isolate binders for a variety of epitopes. The range of lengths of CDRH3 generated in this library is 11 to 13 amino acids, although lengths different from this can also be generated. H3 diversity can be expanded by using NNK, DVK and NVK codon sets, as well as more limited diversity at N and\/or C-terminal","Diversity can also be generated in CDRH1 and CDRH2. The designs of CDR-H1 and H2 diversities follow the strategy of targeting to mimic natural antibodies repertoire as described with modification that focus the diversity more closely matched to the natural diversity than previous design.","For diversity in CDRH3, multiple libraries can be constructed separately with different lengths of H3 and then combined to select for binders to target antigens. The multiple libraries can be pooled and sorted using solid support selection and solution sorting methods as described previously and herein below. Multiple sorting satrategies may be employed. For example, one variation involves sorting on target bound to a solid, followed by sorting for a tag that may be present on the fusion polypeptide (eg. anti-gD tag) and followed by another sort on target bound to solid. Alternatively, the libraries can be sorted first on target bound to a solid surface, the eluted binders are then sorted using solution phase binding with decreasing concentrations of target antigen. Utilizing combinations of different sorting methods provides for minimization of selection of only highly expressed sequences and provides for selection of a number of different high affinity clones.","High affinity binders for the target NRP1 antigen can be isolated from the libraries. Limiting diversity in the H1\/H2 region decreases degeneracy about 10to 10fold and allowing more H3 diversity provides for more high affinity binders. Utilizing libraries with different types of diversity in CDRH3 (eg. utilizing DVK or NVT) provides for isolation of binders that may bind to different epitopes of a target antigen.","Of the binders isolated from the pooled libraries as described above, it has been discovered that affinity may be further improved by providing limited diversity in the light chain. Light chain diversity is generated in this embodiment as follows in CDRL1: amino acid position 28 is encoded by RDT; amino acid position 29 is encoded by RKT; amino acid position 30 is encoded by RVW; amino acid position 31 is encoded by ANW; amino acid position 32 is encoded by THT; optionally, amino acid position 33 is encoded by CTG; in CDRL2: amino acid position 50 is encoded by KBG; amino acid position 53 is encoded by AVC; and optionally, amino acid position 55 is encoded by GMA ; in CDRL3: amino acid position 91 is encoded by TMT or SRT or both; amino acid position 92 is encoded by DMC; amino acid position 93 is encoded by RVT; amino acid position 94 is encoded by NHT; and amino acid position 96 is encoded by TWT or YKG or both.","In another embodiment, a library or libraries with diversity in CDRH1, CDRH2 and CDRH3 regions is generated. In this embodiment, diversity in CDRH3 is generated using a variety of lengths of H3 regions and using primarily codon sets XYZ and NNK or NNS. Libraries can be formed using individual oligonucleotides and pooled or oligonucleotides can be pooled to form a subset of libraries. The libraries of this embodiment can be sorted against target bound to solid. Clones isolated from multiple sorts can be screened for specificity and affinity using ELISA assays. For specificity, the clones can be screened against the desired target antigens as well as other nontarget antigens. Those binders to the target NRP1 antigen can then be screened for affinity in solution binding competition ELISA assay or spot competition assay. High affinity binders can be isolated from the library utilizing XYZ codon sets prepared as described above. These binders can be readily produced as antibodies or antigen binding fragments in high yield in cell culture.","In some embodiments, it may be desirable to generate libraries with a greater diversity in lengths of CDRH3 region. For example, it may be desirable to generate libraries with CDRH3 regions ranging from about 7 to 19 amino acids.","High affinity binders isolated from the libraries of these embodiments are readily produced in bacterial and eukaryotic cell culture in high yield. The vectors can be designed to readily remove sequences such as gD tags, viral coat protein component sequence, and\/or to add in constant region sequences to provide for production of full length antibodies or antigen binding fragments in high yield.","A library with mutations in CDRH3 can be combined with a library containing variant versions of other CDRs, for example CDRL1, CDRL2, CDRL3, CDRH1 and\/or CDRH2. Thus, for example, in one embodiment, a CDRH3 library is combined with a CDRL3 library created in the context of the humanized 4D5 antibody sequence with variant amino acids at positions 28, 29, 30,31, and\/or 32 using predetermined codon sets. In another embodiment, a library with mutations to the CDRH3 can be combined with a library comprising variant CDRH1 and\/or CDRH2 heavy chain variable domains. In one embodiment, the CDRH1 library is created with the humanized antibody 4D5 sequence with variant amino acids at positions 28, 30, 31, 32 and 33. A CDRH2 library may be created with the sequence of humanized antibody 4D5 with variant amino acids at positions 50, 52, 53, 54, 56 and 58 using the predetermined codon sets.","Anti-NRP1 Antibody Mutants","The novel anti-NRP1 antibody generated from phage libraries can be further modified to generate antibody mutants with improved physical, chemical and or biological properties over the parent antibody. Where the assay used is a biological activity assay, the antibody mutant preferably has a biological activity in the assay of choice which is at least about 10 fold better, preferably at least about 20 fold better, more preferably at least about 50 fold better, and sometimes at least about 100 fold or 200 fold better, than the biological activity of the parent antibody in that assay. For example, an anti-NRP1 antibody mutant preferably has a binding affinity for NRP1 which is at least about 10 fold stronger, preferably at least about 20 fold stronger, more preferably at least about 50 fold stronger, and sometimes at least about 100 fold or 200 fold stronger, than the binding affinity of the parent anti-NRP1 antibody.","To generate the antibody mutant, one or more amino acid alterations (e.g. substitutions) are introduced in one or more of the hypervariable regions of the parent antibody. Alternatively, or in addition, one or more alterations (e.g. substitutions) of framework region residues may be introduced in the parent antibody where these result in an improvement in the binding affinity of the antibody mutant for the antigen from the second mammalian species. Examples of framework region residues to modify include those which non-covalently bind antigen directly (Amit et al. (1986) 233:747-753); interact with\/effect the conformation of a CDR (Chothia et al. (1987) 196:901-917); and\/or participate in the V-Vinterface (EP 239 400B1). In certain embodiments, modification of one or more of such framework region residues results in an enhancement of the binding affinity of the antibody for the antigen from the second mammalian species. For example, from about one to about five framework residues may be altered in this embodiment of the invention. Sometimes, this may be sufficient to yield an antibody mutant suitable for use in preclinical trials, even where none of the hypervariable region residues have been altered. Normally, however, the antibody mutant will comprise additional hypervariable region alteration(s).","The hypervariable region residues which are altered may be changed randomly, especially where the starting binding affinity of the parent antibody is such that such randomly produced antibody mutants can be readily screened.","One useful procedure for generating such antibody mutants is called \u201calanine scanning mutagenesis\u201d (Cunningham and Wells (1989) 244:1081-1085). Here, one or more of the hypervariable region residue(s) are replaced by alanine or polyalanine residue(s) to affect the interaction of the amino acids with the antigen from the second mammalian species. Those hypervariable region residue(s) demonstrating functional sensitivity to the substitutions then are refined by introducing further or other mutations at or for the sites of substitution. Thus, while the site for introducing an amino acid sequence variation is predetermined, the nature of the mutation per se need not be predetermined The ala-mutants produced this way are screened for their biological activity as described herein.","Normally one would start with a conservative substitution such as those shown below under the heading of \u201cpreferred substitutions\u201d. If such substitutions result in a change in biological activity (e.g. binding affinity), then more substantial changes, denominated \u201cexemplary substitutions\u201d in the following table, or as further described below in reference to amino acid classes, are introduced and the products screened.","Preferred Substitutions:",{"@attributes":{"id":"p-0113","num":"0112"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"thead":{"row":[{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"Original","Exemplary","Preferred"]},{"entry":[{},"Residue","Substitutions","Substitutions"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Ala (A)","val; leu; ile","val"]},{"entry":[{},"Arg (R)","lys; aln; asn","lys"]},{"entry":[{},"Asn (N)","gln; his; lys; arg","gln"]},{"entry":[{},"Asp (D)","glu","glu"]},{"entry":[{},"Cys (C)","ser","ser"]},{"entry":[{},"Gln (Q)","asn","asn"]},{"entry":[{},"Glu (E)","asp","asp"]},{"entry":[{},"Gly (G)","pro; ala","ala"]},{"entry":[{},"His (H)","asn; gln; lys; arg","arg"]},{"entry":[{},"Ile (I)","leu; val; met; ala; phe;","leu"]},{"entry":[{},{},"norleucine",{}]},{"entry":[{},"Leu (L)","norleucine; ile; val; met; ala;","ile"]},{"entry":[{},{},"phe",{}]},{"entry":[{},"Lys (K)","arg; gln; asn","arg"]},{"entry":[{},"Met (M)","leu; phe; ile","leu"]},{"entry":[{},"Phe (F)","leu; val; ile; ala; tyr","leu"]},{"entry":[{},"Pro (P)","ala","ala"]},{"entry":[{},"Ser (S)","thr","thr"]},{"entry":[{},"Thr (T)","ser","ser"]},{"entry":[{},"Trp (W)","tyr; phe","tyr"]},{"entry":[{},"Tyr (Y)","trp; phe; thr; ser","phe"]},{"entry":[{},"Val (V)","ile; leu; met; phe; ala;","leu"]},{"entry":[{},{},"norleucine"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}}}},"Even more substantial modifications in the antibodies biological properties are accomplished by selecting substitutions that differ significantly in their effect on maintaining (a) the structure of the polypeptide backbone in the area of the substitution, for example, as a sheet or helical conformation, (b) the charge or hydrophobicity of the molecule at the target site, or (c) the bulk of the side chain. Naturally occurring residues are divided into groups based on common side-chain properties:","(1) hydrophobic: norleucine, met, ala, val, leu, ile;","(2) neutral hydrophilic: cys, ser, thr, asn, gln;","(3) acidic: asp, glu;","(4) basic: his, lys, arg;","(5) residues that influence chain orientation: gly, pro; and","(6) aromatic: trp, tyr, phe.","Non-conservative substitutions will entail exchanging a member of one of these classes for another class.","In another embodiment, the sites selected for modification are affinity matured using phage display (see above).","Nucleic acid molecules encoding amino acid sequence mutants are prepared by a variety of methods known in the art. These methods include, but are not limited to, oligonucleotide-mediated (or site-directed) mutagenesis, PCR mutagenesis, and cassette mutagenesis of an earlier prepared mutant or a non-mutant version of the parent antibody. The preferred method for making mutants is site directed mutagenesis (see, e.g., Kunkel (1985) 82:488).","In certain embodiments, the antibody mutant will only have a single hypervariable region residue substituted. In other embodiments, two or more of the hypervariable region residues of the parent antibody will have been substituted, e.g. from about two to about ten hypervariable region substitutions.","Ordinarily, the antibody mutant with improved biological properties will have an amino acid sequence having at least 75% amino acid sequence identity or similarity with the amino acid sequence of either the heavy or light chain variable domain of the parent antibody, more preferably at least 80%, more preferably at least 85%, more preferably at least 90%, and most preferably at least 95%. Identity or similarity with respect to this sequence is defined herein as the percentage of amino acid residues in the candidate sequence that are identical (i.e same residue) or similar (i e. amino acid residue from the same group based on common side-chain properties, see above) with the parent antibody residues, after aligning the sequences and introducing gaps, if necessary, to achieve the maximum percent sequence identity. None of N-terminal, C-terminal, or internal extensions, deletions, or insertions into the antibody sequence outside of the variable domain shall be construed as affecting sequence identity or similarity.","Following production of the antibody mutant, the biological activity of that molecule relative to the parent antibody is determined As noted above, this may involve determining the binding affinity and\/or other biological activities of the antibody. In a preferred embodiment of the invention, a panel of antibody mutants is prepared and screened for binding affinity for the antigen such as NRP1 or a fragment thereof One or more of the antibody mutants selected from this initial screen are optionally subjected to one or more further biological activity assays to confirm that the antibody mutant(s) with enhanced binding affinity are indeed useful, e.g. for preclinical studies.","The antibody mutant(s) so selected may be subjected to further modifications, oftentimes depending on the intended use of the antibody. Such modifications may involve further alteration of the amino acid sequence, fusion to heterologous polypeptide(s) and\/or covalent modifications such as those elaborated below. With respect to amino acid sequence alterations, exemplary modifications are elaborated above. For example, any cysteine residue not involved in maintaining the proper conformation of the antibody mutant also may be substituted, generally with serine, to improve the oxidative stability of the molecule and prevent aberrant cross linking. Conversely, cysteine bond(s) may be added to the antibody to improve its stability (particularly where the antibody is an antibody fragment such as an Fv fragment). Another type of amino acid mutant has an altered glycosylation pattern. This may be achieved by deleting one or more carbohydrate moieties found in the antibody, and\/or adding one or more glycosylation sites that are not present in the antibody. Glycosylation of antibodies is typically either N-linked or O-linked. N-linked refers to the attachment of the carbohydrate moiety to the side chain of an asparagine residue. The tripeptide sequences asparagine-X-serine and asparagine-X-threonine, where X is any amino acid except proline, are the recognition sequences for enzymatic attachment of the carbohydrate moiety to the asparagine side chain. Thus, the presence of either of these tripeptide sequences in a polypeptide creates a potential glycosylation site. O-linked glycosylation refers to the attachment of one of the sugars N-aceylgalactosamine, galactose, or xylose to a hydroxyamino acid, most commonly serine or threonine, although 5-hydroxyproline or 5-hydroxylysine may also be used. Addition of glycosylation sites to the antibody is conveniently accomplished by altering the amino acid sequence such that it contains one or more of the above-described tripeptide sequences (for N-linked glycosylation sites). The alteration may also be made by the addition of, or substitution by, one or more serine or threonine residues to the sequence of the original antibody (for O-linked glycosylation sites).","Vectors, Host Cells and Recombinant Methods","The anti-NRP1 antibody of the invention can be produced recombinantly, using techniques and materials readily obtainable.","For recombinant production of an anti-NRP1 antibody, the nucleic acid encoding it is isolated and inserted into a replicable vector for further cloning (amplification of the DNA) or for expression. DNA encoding the antibody is readily isolated or synthethized using conventional procedures (e.g., by using oligonucleotide probes that are capable of binding specifically to DNAs encoding the heavy and light chains of the antibody). Many vectors are available. The vector components generally include, but are not limited to, one or more of the following: a signal sequence, an origin of replication, one or more marker genes, an enhancer element, a promoter, and a transcription termination sequence.","(i) Signal Sequence Component","The antibody of this invention may be produced recombinantly not only directly, but also as a fusion polypeptide with a heterologous polypeptide, which is preferably a signal sequence or other polypeptide having a specific cleavage site at the N-terminus of the mature protein or polypeptide. The heterologous signal sequence selected preferably is one that is recognized and processed (i.e., cleaved by a signal peptidase) by the host cell. For prokaryotic host cells that do not recognize and process the native antibody signal sequence, the signal sequence is substituted by a prokaryotic signal sequence selected, for example, from the group of the alkaline phosphatase, penicillinase, lpp, or heat-stable enterotoxin II leaders. For yeast secretion the native signal sequence may be substituted by, e.g., the yeast invertase leader, a factor leader (including and \u03b1-factor leaders), or acid phosphatase leader, the glucoamylase leader, or the signal described in WO 90\/13646. In mammalian cell expression, mammalian signal sequences as well as viral secretory leaders, for example, the herpes simplex gD signal, are available.","The DNA for such precursor region is ligated in reading frame to DNA encoding the antibody.","(ii) Origin of Replication Component","Both expression and cloning vectors contain a nucleic acid sequence that enables the vector to replicate in one or more selected host cells. Generally, in cloning vectors this sequence is one that enables the vector to replicate independently of the host chromosomal DNA, and includes origins of replication or autonomously replicating sequences. Such sequences are well known for a variety of bacteria, yeast, and viruses. The origin of replication from the plasmid pBR322 is suitable for most Gram-negative bacteria, the 2\u03bc plasmid origin is suitable for yeast, and various viral origins (SV40, polyoma, adenovirus, VSV or BPV) are useful for cloning vectors in mammalian cells. Generally, the origin of replication component is not needed for mammalian expression vectors (the SV40 origin may typically be used only because it contains the early promoter).","(iii) Selection Gene Component","Expression and cloning vectors may contain a selection gene, also termed a selectable marker. Typical selection genes encode proteins that (a) confer resistance to antibiotics or other toxins, e.g., ampicillin, neomycin, methotrexate, or tetracycline, (b) complement auxotrophic deficiencies, or (c) supply critical nutrients not available from complex media, e.g., the gene encoding D-alanine racemase for ","One example of a selection scheme utilizes a drug to arrest growth of a host cell. Those cells that are successfully transformed with a heterologous gene produce a protein conferring drug resistance and thus survive the selection regimen. Examples of such dominant selection use the drugs neomycin, mycophenolic acid and hygromycin.","Another example of suitable selectable markers for mammalian cells are those that enable the identification of cells competent to take up the antibody nucleic acid, such as DHFR, thymidine kinase, metallothionein-I and -II, preferably primate metallothionein genes, adenosine deaminase, ornithine decarboxylase, etc.","For example, cells transformed with the DHFR selection gene are first identified by culturing all of the transformants in a culture medium that contains methotrexate (Mtx), a competitive antagonist of DHFR. An appropriate host cell when wild-type DHFR is employed is the Chinese hamster ovary (CHO) cell line deficient in DHFR activity.","Alternatively, host cells (particularly wild-type hosts that contain endogenous DHFR) transformed or co-transformed with DNA sequences encoding antibody, wild-type DHFR protein, and another selectable marker such as aminoglycoside 3\u2032-phosphotransferase (APH) can be selected by cell growth in medium containing a selection agent for the selectable marker such as an aminoglycosidic antibiotic, e.g., kanamycin, neomycin, or G418. See U.S. Pat. No. 4,965,199.","A suitable selection gene for use in yeast is the trpl gene present in the yeast plasmid YRp7 (Stinchcomb et al. (1979) 282:39). The trpl gene provides a selection marker for a mutant strain of yeast lacking the ability to grow in tryptophan, for example, ATCC No. 44076 or PEP4-1. Jones (1977) 85:12. The presence of the trpl lesion in the yeast host cell genome then provides an effective environment for detecting transformation by growth in the absence of tryptophan. Similarly, Leu2-deficient yeast strains (ATCC 20,622 or 38,626) are complemented by known plasmids bearing the Leu2 gene.","In addition, vectors derived from the 1.6 \u03bcm circular plasmid pKD1 can be used for transformation of yeasts. Alternatively, an expression system for large-scale production of recombinant calf chymosin was reported for . Van den Berg (1990) 8:135. Stable multi-copy expression vectors for secretion of mature recombinant human serum albumin by industrial strains of have also been disclosed. Fleer et al. (1991) 9:968-975.","(iv) Promoter Component","Expression and cloning vectors usually contain a promoter that is recognized by the host organism and is operably linked to the antibody nucleic acid. Promoters suitable for use with prokaryotic hosts include the phoA promoter, \u03b2-lactamase and lactose promoter systems, alkaline phosphatase, a tryptophan (tip) promoter system, and hybrid promoters such as the tac promoter. However, other known bacterial promoters are suitable. Promoters for use in bacterial systems also will contain a Shine-Dalgarno (S.D.) sequence operably linked to the DNA encoding the antibody.","Promoter sequences are known for eukaryotes. Virtually all eukaryotic genes have an AT-rich region located approximately 25 to 30 bases upstream from the site where transcription is initiated. Another sequence found 70 to 80 bases upstream from the start of transcription of many genes is a CNCAAT region where N may be any nucleotide. At the 3\u2032 end of most eukaryotic genes is an AATAAA sequence that may be the signal for addition of the poly A tail to the 3\u2032 end of the coding sequence. All of these sequences are suitably inserted into eukaryotic expression vectors.","Examples of suitable promoting sequences for use with yeast hosts include the promoters for 3-phosphoglycerate kinase or other glycolytic enzymes, such as enolase, glyceraldehyde-3-phosphate dehydrogenase, hexokinase, pyruvate decarboxylase, phosphofructokinase, glucose-6-phosphate isomerase, 3-phosphoglycerate mutase, pyruvate kinase, triosephosphate isomerase, phosphoglucose isomerase, and glucokinase.","Other yeast promoters, which are inducible promoters having the additional advantage of transcription controlled by growth conditions, are the promoter regions for alcohol dehydrogenase 2, isocytochrome C, acid phosphatase, degradative enzymes associated with nitrogen metabolism, metallothionein, glyceraldehyde-3-phosphate dehydrogenase, and enzymes responsible for maltose and galactose utilization. Suitable vectors and promoters for use in yeast expression are further described in EP 73,657. Yeast enhancers also are advantageously used with yeast promoters.","Antibody transcription from vectors in mammalian host cells is controlled, for example, by promoters obtained from the genomes of viruses such as polyoma virus, fowlpox virus, adenovirus (such as Adenovirus 2), bovine papilloma virus, avian sarcoma virus, cytomegalovirus, a retrovirus, hepatitis-B virus and most preferably Simian Virus 40 (SV40), from heterologous mammalian promoters, e.g., the actin promoter or an immunoglobulin promoter, from heat-shock promoters, provided such promoters are compatible with the host cell systems.","The early and late promoters of the SV40 virus are conveniently obtained as an SV40 restriction fragment that also contains the SV40 viral origin of replication. The immediate early promoter of the human cytomegalovirus is conveniently obtained as a HindIII E restriction fragment. A system for expressing DNA in mammalian hosts using the bovine papilloma virus as a vector is disclosed in U.S. Pat. No. 4,419,446. A modification of this system is described in U.S. Pat. No. 4,601,978. See also Reyes et al. (1982) 297:598-601 on expression of human \u03b2-interferon cDNA in mouse cells under the control of a thymidine kinase promoter from herpes simplex virus. Alternatively, the rous sarcoma virus long terminal repeat can be used as the promoter.","(v) Enhancer Element Component","Transcription of a DNA encoding the antibody of this invention by higher eukaryotes is often increased by inserting an enhancer sequence into the vector. Many enhancer sequences are now known from mammalian genes (globin, elastase, albumin, \u03b1-fetoprotein, and insulin). Typically, however, one will use an enhancer from a eukaryotic cell virus. Examples include the SV40 enhancer on the late side of the replication origin (bp 100-270), the cytomegalovirus early promoter enhancer, the polyoma enhancer on the late side of the replication origin, and adenovirus enhancers. See also Yaniv (1982) 297:17-18 on enhancing elements for activation of eukaryotic promoters. The enhancer may be spliced into the vector at a position 5\u2032 or 3\u2032 to the antibody-encoding sequence, but is preferably located at a site 5\u2032 from the promoter.","(vi) Transcription Termination Component","Expression vectors used in eukaryotic host cells (yeast, fungi, insect, plant, animal, human, or nucleated cells from other multicellular organisms) will also contain sequences necessary for the termination of transcription and for stabilizing the mRNA. Such sequences are commonly available from the 5\u2032 and, occasionally 3\u2032, untranslated regions of eukaryotic or viral DNAs or cDNAs. These regions contain nucleotide segments transcribed as polyadenylated fragments in the untranslated portion of the mRNA encoding the antibody. One useful transcription termination component is the bovine growth hormone polyadenylation region. See WO94\/11026 and the expression vector disclosed therein.","(vii) Selection and Transformation of Host Cells","Suitable host cells for cloning or expressing the DNA in the vectors herein are the prokaryote, yeast, or higher eukaryote cells described above. Suitable prokaryotes for this purpose include eubacteria, such as Gram-negative or Gram-positive organisms, for example, Enterobacteriaceae such as e.g., , e.g., , e.g., , and , as well as such as (e.g., 41P disclosed in DD 266,710 published 12 Apr. 1989), Pseudomonas such as , and . One preferred cloning host is 294 (ATCC 31,446), although other strains such as B, X1776 (ATCC 31,537), and W3110 (ATCC 27,325) are suitable. These examples are illustrative rather than limiting.","In addition to prokaryotes, eukaryotic microbes such as filamentous fungi or yeast are suitable cloning or expression hosts for antibody-encoding vectors. , or common baker's yeast, is the most commonly used among lower eukaryotic host microorganisms. However, a number of other genera, species, and strains are commonly available and useful herein, such as hosts such as, e.g., (ATCC 12,424), (ATCC 16,045), (ATCC 24,178), (ATCC 56,500), (ATCC 36,906), , and (EP 402,226); (EP 183,070); (EP 244,234); such as and filamentous fungi such as, e.g., , and hosts such as and ","Suitable host cells for the expression of glycosylated antibody are derived from multicellular organisms. Examples of invertebrate cells include plant and insect cells. Numerous baculoviral strains and variants and corresponding permissive insect host cells from hosts such as (caterpillar), (mosquito), (mosquito), (fruitfly), and have been identified. A variety of viral strains for transfection are publicly available, e.g., the L-1 variant of NPV and the Bm-5 strain of NPV, and such viruses may be used as the virus herein according to the present invention, particularly for transfection of cells. Plant cell cultures of cotton, corn, potato, soybean, petunia, tomato, and tobacco can also be utilized as hosts.","However, interest has been greatest in vertebrate cells, and propagation of vertebrate cells in culture (tissue culture) has become a routine procedure. Examples of useful mammalian host cell lines are monkey kidney CV1 line transformed by SV40 (COS-7, ATCC CRL 1651); human embryonic kidney line (293 or 293 cells subcloned for growth in suspension culture, Graham et al. (1977) 36:59) ; baby hamster kidney cells (BHK, ATCC CCL 10); Chinese hamster ovary cells\/-DHFR (CHO, Urlaub et al. (1980) 77:4216) ; mouse sertoli cells (TM4, Mather (1980) 23:243-251); monkey kidney cells (CV1 ATCC CCL 70); African green monkey kidney cells (VERO-76, ATCC CRL-1587); human cervical carcinoma cells (HELA, ATCC CCL 2); canine kidney cells (MDCK, ATCC CCL 34); buffalo rat liver cells (BRL 3A, ATCC CRL 1442); human lung cells (W138, ATCC CCL 75); human liver cells (Hep G2, HB 8065); mouse mammary tumor (MMT 060562, ATCC CCL51); TRI cells (Mather et al. (1982) 383:44-68); MRC 5 cells; FS4 cells; and a human hepatoma line (Hep G2).","Host cells are transformed with the above-described expression or cloning vectors for antibody production and cultured in conventional nutrient media modified as appropriate for inducing promoters, selecting transformants, or amplifying the genes encoding the desired sequences.","(viii) Culturing the Host Cells","The host cells used to produce the antibody of this invention may be cultured in a variety of media. Commercially available media such as Ham's F10 (Sigma), Minimal Essential Medium ((MEM), (Sigma), RPMI-1640 (Sigma), and Dulbecco's Modified Eagle's Medium ((DMEM), Sigma) are suitable for culturing the host cells. In addition, any of the media described in Ham et al. (1979) 58:44, Barnes et al. (1980) 102:255, U.S. Pat. Nos. 4,767,704; 4,657,866; 4,927,762; 4,560,655; or 5,122,469; WO 90\/03430; WO 87\/00195; or U.S. Pat. Re. 30,985 may be used as culture media for the host cells. Any of these media may be supplemented as necessary with hormones and\/or other growth factors (such as insulin, transferrin, or epidermal growth factor), salts (such as sodium chloride, calcium, magnesium, and phosphate), buffers (such as HEPES), nucleotides (such as adenosine and thymidine), antibiotics (such as GENTAMYCIN\u2122 drug), trace elements (defined as inorganic compounds usually present at final concentrations in the micromolar range), and glucose or an equivalent energy source. Any other necessary supplements may also be included at appropriate concentrations that would be known to those skilled in the art. The culture conditions, such as temperature, pH, and the like, are those previously used with the host cell selected for expression, and will be apparent to the ordinarily skilled artisan.","(ix) Antibody Purification","When using recombinant techniques, the antibody can be produced intracellularly, in the periplasmic space, or directly secreted into the medium. If the antibody is produced intracellularly, as a first step, the particulate debris, either host cells or lysed fragments, is removed, for example, by centrifugation or ultrafiltration. Carter et al. (1992) 10:163-167 describe a procedure for isolating antibodies which are secreted to the periplasmic space of . Briefly, cell paste is thawed in the presence of sodium acetate (pH 3.5), EDTA, and phenylmethylsulfonylfluoride (PMSF) over about 30 min. Cell debris can be removed by centrifugation. Where the antibody is secreted into the medium, supernatants from such expression systems are generally first concentrated using a commercially available protein concentration filter, for example, an Amicon or Millipore Pellicon ultrafiltration unit. A protease inhibitor such as PMSF may be included in any of the foregoing steps to inhibit proteolysis and antibiotics may be included to prevent the growth of adventitious contaminants","The antibody composition prepared from the cells can be purified using, for example, hydroxylapatite chromatography, gel electrophoresis, dialysis, and affinity chromatography, with affinity chromatography being the preferred purification technique. The suitability of protein A as an affinity ligand depends on the species and isotype of any immunoglobulin Fc domain that is present in the antibody. Protein A can be used to purify antibodies that are based on human \u03b31, \u03b32, or \u03b34 heavy chains (Lindmark et al. (1983) 62:1-13). Protein G is recommended for all mouse isotypes and for human \u03b33 (Guss et al. (1986) 5:15671575). The matrix to which the affinity ligand is attached is most often agarose, but other matrices are available. Mechanically stable matrices such as controlled pore glass or poly(styrenedivinyl)benzene allow for faster flow rates and shorter processing times than can be achieved with agarose. Where the antibody comprises a C3 domain, the Bakerbond ABX\u2122 resin (J. T. Baker, Phillipsburg, N.J.) is useful for purification. Other techniques for protein purification such as fractionation on an ion-exchange column, ethanol precipitation, Reverse Phase HPLC, chromatography on silica, chromatography on heparin SEPHAROSE\u2122 chromatography on an anion or cation exchange resin (such as a polyaspartic acid column), chromatofocusing, SDS-PAGE, and ammonium sulfate precipitation are also available depending on the antibody to be recovered.","Following any preliminary purification step(s), the mixture comprising the antibody of interest and contaminants may be subjected to low pH hydrophobic interaction chromatography using an elution buffer at a pH between about 2.5-4.5, preferably performed at low salt concentrations (e.g., from about 0-0.25M salt).","Pharmaceutical Formulations","Therapeutic formulations of the antibody are prepared for storage by mixing the antibody having the desired degree of purity with optional physiologically acceptable carriers, excipients or stabilizers (16th edition, Osol, A. Ed. (1980)), in the form of lyophilized formulations or aqueous solutions. Acceptable carriers, excipients, or stabilizers are nontoxic to recipients at the dosages and concentrations employed, and include buffers such as phosphate, citrate, and other organic acids; antioxidants including ascorbic acid and methionine; preservatives (such as octadecyldimethylbenzyl ammonium chloride; hexamethonium chloride; benzalkonium chloride, benzethonium chloride; phenol, butyl or benzyl alcohol; alkyl parabens such as methyl or propyl paraben; catechol; resorcinol; cyclohexanol; 3-pentanol; and m-cresol); low molecular weight (less than about 10 residues) polypeptide; proteins, such as serum albumin, gelatin, or immunoglobulins; hydrophilic polymers such as polyvinylpyrrolidone; amino acids such as glycine, glutamine, asparagine, histidine, arginine, or lysine; monosaccharides, disaccharides, and other carbohydrates including glucose, mannose, or dextrins; chelating agents such as EDTA; sugars such as sucrose, mannitol, trehalose or sorbitol; salt-forming counter-ions such as sodium; metal complexes (e.g., Zn-protein complexes); and\/or non-ionic surfactants such as TWEEN\u2122, PLURONICS\u2122 or polyethylene glycol (PEG).","The formulation herein may also contain more than one active compound as necessary for the particular indication being treated, preferably those with complementary activities that do not adversely affect each other. For example, it may be desirable to further provide an immunosuppressive agent. Such molecules are suitably present in combination in amounts that are effective for the purpose intended.","The active ingredients may also be entrapped in microcapsule prepared, for example, by coacervation techniques or by interfacial polymerization, for example, hydroxymethylcellulose or gelatin-microcapsule and poly-(methylmethacylate) microcapsule, respectively, in colloidal drug delivery systems (for example, liposomes, albumin microspheres, microemulsions, nano-particles and nanocapsules) or in macroemulsions. Such techniques are disclosed in 16th edition, Osol, A. Ed. (1980).","The formulations to be used for in vivo administration must be sterile. This is readily accomplished by filtration through sterile filtration membranes.","Sustained-release preparations may be prepared. Suitable examples of sustained-release preparations include semipermeable matrices of solid hydrophobic polymers containing the antibody, which matrices are in the form of shaped articles, e.g., films, or microcapsule. Examples of sustained-release matrices include polyesters, hydrogels (for example, poly(2-hydroxyethyl-methacrylate), or poly(vinylalcohol)), polylactides (U.S. Pat. No. 3,773,919), copolymers of L-glutamic acid and \u03b3 ethyl-L-glutamate, non-degradable ethylene-vinyl acetate, degradable lactic acid-glycolic acid copolymers such as the LUPRON DEPOT\u2122 (injectable microspheres composed of lactic acid-glycolic acid copolymer and leuprolide acetate), and poly-D-(\u2212)-3-hydroxybutyric acid. While polymers such as ethylene-vinyl acetate and lactic acid-glycolic acid enable release of molecules for over 100 days, certain hydrogels release proteins for shorter time periods. When encapsulated antibodies remain in the body for a long time, they may denature or aggregate as a result of exposure to moisture at 37\u00b0 C., resulting in a loss of biological activity and possible changes in immunogenicity. Rational strategies can be devised for stabilization depending on the mechanism involved. For example, if the aggregation mechanism is discovered to be intermolecular S\u2014S bond formation through thio-disulfide interchange, stabilization may be achieved by modifying sulfhydryl residues, lyophilizing from acidic solutions, controlling moisture content, using appropriate additives, and developing specific polymer matrix compositions.","Therapeutic Uses","It is contemplated that the antibody of the present invention may be used to treat a mammal. In one embodiment, the antibody is administered to a nonhuman mammal for the purposes of obtaining preclinical data, for example. Exemplary nonhuman mammals to be treated include nonhuman primates, dogs, cats, rodents and other mammals in which preclinical studies are performed. Such mammals may be established animal models for a disease to be treated with the antibody or may be used to study toxicity of the antibody of interest. In each of these embodiments, dose escalation studies may be performed in the mammal. Where the antibody is an anti-NRP1 antibody, it may be administered to a host rodent in a solid tumor model, for example.","In addition, or in the alternative, the antibody is used to treat a human, e.g. a patient suffering from a disease or disorder who could benefit from administration of the antibody.","The present invention encompasses antiangiogenic cancer therapy, a novel cancer treatment strategy aimed at inhibiting the development of tumor blood vessels required for providing nutrients to support tumor growth. Because angiogenesis is involved in both primary tumor growth and metastasis, the antiangiogenic treatment provided by the invention is capable of inhibiting the neoplastic growth of tumor at the primary site as well as preventing metastasis of tumors at the secondary sites, therefore allowing attack of the tumors by other therapeutics. Examples of cancer to be treated herein include, but are not limited to, carcinoma, lymphoma, blastoma, sarcoma, and leukemia. More particular examples of such cancers include squamous cell cancer, lung cancer (including small-cell lung cancer, non-small cell lung cancer, adenocarcinoma of the lung, and squamous carcinoma of the lung), cancer of the peritoneum, hepatocellular cancer, gastric or stomach cancer (including gastrointestinal cancer), pancreatic cancer, glioblastoma, cervical cancer, ovarian cancer, liver cancer, bladder cancer, hepatoma, breast cancer, colon cancer, colorectal cancer, endometrial or uterine carcinoma, salivary gland carcinoma, kidney or renal cancer, liver cancer, prostate cancer, vulval cancer, thyroid cancer, hepatic carcinoma and various types of head and neck cancer, as well as B-cell lymphoma (including low grade\/follicular non-Hodgkin's lymphoma (NHL); small lymphocytic (SL) NHL; intermediate grade\/follicular NHL; intermediate grade diffuse NHL; high grade immunoblastic NHL; high grade lymphoblastic NHL; high grade small non-cleaved cell NHL; bulky disease NHL; mantle cell lymphoma; AIDS-related lymphoma; and Waldenstrom's Macroglobulinemia); chronic lymphocytic leukemia (CLL); acute lymphoblastic leukemia (ALL); Hairy cell leukemia; chronic myeloblastic leukemia; and post-transplant lymphoproliferative disorder (PTLD), as well as abnormal vascular proliferation associated with phakomatoses, edema (such as that associated with brain tumors), and Meigs' syndrome. More particularly, cancers that are amenable to treatment by the antibodies of the invention include breast cancer, colorectal cancer, rectal cancer, non-small cell lung cancer, non-Hodgkins lymphoma (NHL), renal cell cancer, prostate cancer, liver cancer, pancreatic cancer, soft-tissue sarcoma, kaposi's sarcoma, carcinoid carcinoma, head and neck cancer, melanoma, ovarian cancer, mesothelioma, and multiple myeloma. More preferably, the methods of the invention are used to treat colorectal cancer in a human patient.","It is contemplated that when used to treat various diseases such as tumors, the antibodies of the invention can be combined with other therapeutic agents suitable for the same or similar diseases. When used for treating cancer, antibodies of the present invention may be used in combination with conventional cancer therapies, such as surgery, radiotherapy, chemotherapy or combinations thereof.","In certain aspects, other therapeutic agents useful for combination cancer therapy with the antibody of the invention include other anti-angiogenic agents. Many anti-angiogenic agents have been identified and are known in the arts, including those listed by Carmeliet and Jain (2000).","In one aspect, the antibody of the invention is used in combination with a VEGF antagonist or a VEGF receptor antagonist such as anti-VEGF antibodies, VEGF variants, soluble VEGF receptor fragments, aptamers capable of blocking VEGF or VEGFR, neutralizing anti-VEGFR antibodies, inhibitors of VEGFR tyrosine kinases and any combinations thereof Alternatively, or in addition, two or more anti-NRP1 antibodies may be co-administered to the patient. In a more preferred embodiment, the anti-NRP1or anti-NRPantibody of the invention is used in combination with an anti-VEGF antibody to generate additive or synergistic effects. Preferred anti-VEGF antibodies include those that bind to the same epitope as the anti-hVEGF antibody A4.6.1. More preferably the anti-VEGF antibody is bevacizumab or ranibizumab.","In some other aspects, other therapeutic agents useful for combination tumor therapy with the antibody of the invention include antagonist of other factors that are involved in tumor growth, such as EGFR, ErbB2 (also known as Her2) ErbB3, ErbB4, or TNF. Preferably, the anti-NRP1 antibody of the invention can be used in combination with small molecule receptor tyrosine kinase inhibitors (RTKIs) that target one or more tyrosine kinase receptors such as VEGF receptors, FGF receptors, EGF receptors and PDGF receptors. Many therapeutic small molecule RTKIs are known in the art, including, but are not limited to, vatalanib (PTK787), erlotinib (TARCEVA\u00ae), OSI-7904, ZD6474 (ZACTIMA\u00ae), ZD6126 (ANG453), ZD1839, sunitinib (SUTENT\u00ae), semaxanib (SU5416), AMG706, AG013736, Imatinib (GLEEVEC\u00ae), MLN-518, CEP-701, PKC- 412, Lapatinib (GSK572016), bortezomib (VELCADE\u00ae), AZD2171, sorafenib (NEXAVAR\u00ae), XL880, and CHIR-265.","The anti-NRP1 antibody of the invention, either alone or in combination with a second therpateutic agent (such as an anti-VEGF antibody) can be further used in combination with one or more chemotherapeutic agents. A variety of chemotherapeutic agents may be used in the combined treatment methods of the invention. An exemplary and non-limiting list of chemotherapeutic agents contemplated is provided herein under \u201cDefinition\u201d.","When the anti-NRP1 antibody is co-administered with a second therapeutic agent, the second therapeutic agent may be administered first, followed by the anti-NRP1 antibody. However, simultaneous administration or administration of the anti-NRP1 antibody first is also contemplated. Suitable dosages for the second therapeutic agent are those presently used and may be lowered due to the combined action (synergy) of the agent and anti-NRP1 antibody.","For the prevention or treatment of disease, the appropriate dosage of antibody will depend on the type of disease to be treated, the severity and course of the disease, whether the antibody is administered for preventive or therapeutic purposes, previous therapy, the patient's clinical history and response to the antibody, and the discretion of the attending physician. The antibody is suitably administered to the patient at one time or over a series of treatments.","Depending on the type and severity of the disease, about 1 \u03bcg\/kg to 50 mg\/kg (e.g. 0.1-20 mg\/kg) of antibody is an initial candidate dosage for administration to the patient, whether, for example, by one or more separate administrations, or by continuous infusion. A typical daily dosage might range from about 1 \u03bcg\/kg to about 100 mg\/kg or more, depending on the factors mentioned above. For repeated administrations over several days or longer, depending on the condition, the treatment is sustained until a desired suppression of disease symptoms occurs. However, other dosage regimens may be useful. In a preferred aspect, the antibody of the invention is administered every two to three weeks, at a dose ranged from about 5 mg\/kg to about 15 mg\/kg. More preferably, such dosing regimen is used in combination with a chemotherapy regimen as the first line therapy for treating metastatic colorectal cancer. In some aspects, the chemotherapy regimen involves the traditional high-dose intermittent administration. In some other aspects, the chemotherapeutic agents are administered using smaller and more frequent doses without scheduled breaks (\u201cmetronomic chemotherapy\u201d). The progress of the therapy of the invention is easily monitored by conventional techniques and assays.","The antibody composition will be formulated, dosed, and administered in a fashion consistent with good medical practice. Factors for consideration in this context include the particular disorder being treated, the particular mammal being treated, the clinical condition of the individual patient, the cause of the disorder, the site of delivery of the agent, the method of administration, the scheduling of administration, and other factors known to medical practitioners. The \u201ctherapeutically effective amount\u201d of the antibody to be administered will be governed by such considerations, and is the minimum amount necessary to prevent, ameliorate, or treat a disease or disorder. The antibody need not be, but is optionally formulated with one or more agents currently used to prevent or treat the disorder in question. The effective amount of such other agents depends on the amount of antibody present in the formulation, the type of disorder or treatment, and other factors discussed above. These are generally used in the same dosages and with administration routes as used hereinbefore or about from 1 to 99% of the heretofore employed dosages. Generally, alleviation or treatment of a disease or disorder involves the lessening of one or more symptoms or medical problems associated with the disease or disorder. In the case of cancer, the therapeutically effective amount of the drug can accomplish one or a combination of the following: reduce the number of cancer cells; reduce the tumor size; inhibit (i.e., to decrease to some extent and\/or stop) cancer cell infiltration into peripheral organs; inhibit tumor metastasis; inhibit, to some extent, tumor growth; and\/or relieve to some extent one or more of the symptoms associated with the cancer. To the extent the drug may prevent growth and\/or kill existing cancer cells, it may be cytostatic and\/or cytotoxic. In some embodiments, a composition of this invention can be used to prevent the onset or reoccurrence of the disease or disorder in a subject or mammal.","Non-Therapeutic Uses","The antibodies of the invention may be used as affinity purification agents. In this process, the antibodies are immobilized on a solid phase such a Sephadex resin or filter paper, using methods well known in the art. The immobilized antibody is contacted with a sample containing the antigen to be purified, and thereafter the support is washed with a suitable solvent that will remove substantially all the material in the sample except the antigen to be purified, which is bound to the immobilized antibody. Finally, the support is washed with another suitable solvent, such as glycine buffer, pH 5.0, that will release the antigen from the antibody.","The antibodies of this invention may also be useful in diagnostic assays, e.g., for detecting expression of an antigen of interest in specific cells, tissues, or serum.","For diagnostic applications, the antibody typically will be labeled with a detectable moiety. Numerous labels are available which can be generally grouped into the following categories:","(a) Radioisotopes, such as S, C, I, H, and I. The antibody can be labeled with the radioisotope using the techniques described in , Volumes 1 and 2, Coligen et al. (1991) Ed. Wiley-Interscience, New York, N.Y., Pubs. for example and radioactivity can be measured using scintillation counting.","(b) Fluorescent labels such as rare earth chelates (europium chelates) or fluorescein and its derivatives, rhodamine and its derivatives, dansyl, Lissamine, phycoerythrin and Texas Red are available. The fluorescent labels can be conjugated to the antibody using the techniques disclosed in , supra, for example. Fluorescence can be quantified using a fluorimeter.","(c) Various enzyme-substrate labels are available and U.S. Pat. No. 4,275,149 provides a review of some of these. The enzyme generally catalyzes a chemical alteration of the chromogenic substrate which can be measured using various techniques. For example, the enzyme may catalyze a color change in a substrate, which can be measured spectrophotometrically. Alternatively, the enzyme may alter the fluorescence or chemiluminescence of the substrate. Techniques for quantifying a change in fluorescence are described above. The chemiluminescent substrate becomes electronically excited by a chemical reaction and may then emit light which can be measured (using a chemiluminometer, for example) or donates energy to a fluorescent acceptor. Examples of enzymatic labels include luciferases (e.g., firefly luciferase and bacterial luciferase; U.S. Pat. No. 4,737,456), luciferin, 2,3-dihydrophthalazinediones, malate dehydrogenase, urease, peroxidase such as horseradish peroxidase (HRPO), alkaline phosphatase, \u03b2-galactosidase, glucoamylase, lysozyme, saccharide oxidases (e.g., glucose oxidase, galactose oxidase, and glucose-6-phosphate dehydrogenase), heterocyclic oxidases (such as uricase and xanthine oxidase), lactoperoxidase, microperoxidase, and the like. Techniques for conjugating enzymes to antibodies are described in O'Sullivan et al. (1981) Methods for the Preparation of Enzyme-Antibody Conjugates for use in Enzyme Immunoassay, in . (ed J. Langone & H. Van Vunakis), Academic press, New York 73:147-166.","Examples of enzyme-substrate combinations include, for example:","(i) Horseradish peroxidase (HRPO) with hydrogen peroxidase as a substrate, wherein the hydrogen peroxidase oxidizes a dye precursor (e.g., orthophenylene diamine (OPD) or 3,3\u2032,5,5\u2032-tetramethyl benzidine hydrochloride (TMB));","(ii) alkaline phosphatase (AP) with para-Nitrophenyl phosphate as chromogenic substrate; and","(iii) \u03b2-D-galactosidase (\u03b2-D-Gal) with a chromogenic substrate (e.g., p-nitrophenyl-\u03b2-D-galactosidase) or fluorogenic substrate 4-methylumbelliferyl-\u03b2-D-galactosidase.","Numerous other enzyme-substrate combinations are available to those skilled in the art. For a general review of these, see U.S. Pat. Nos. 4,275,149 and 4,318,980.","Sometimes, the label is indirectly conjugated with the antibody. The skilled artisan will be aware of various techniques for achieving this. For example, the antibody can be conjugated with biotin and any of the three broad categories of labels mentioned above can be conjugated with avidin, or vice versa. Biotin binds selectively to avidin and thus, the label can be conjugated with the antibody in this indirect manner. Alternatively, to achieve indirect conjugation of the label with the antibody, the antibody is conjugated with a small hapten (e.g., digoxin) and one of the different types of labels mentioned above is conjugated with an anti-hapten antibody (e.g., anti-digoxin antibody). Thus, indirect conjugation of the label with the antibody can be achieved.","In another embodiment of the invention, the antibody need not be labeled, and the presence thereof can be detected using a labeled antibody which binds to the antibody.","The antibodies of the present invention may be employed in any known assay method, such as competitive binding assays, direct and indirect sandwich assays, and immunoprecipitation assays. Zola, , pp. 147-158 (CRC Press, Inc. 1987).","Competitive binding assays rely on the ability of a labeled standard to compete with the test sample analyze for binding with a limited amount of antibody. The amount of antigen in the test sample is inversely proportional to the amount of standard that becomes bound to the antibodies. To facilitate determining the amount of standard that becomes bound, the antibodies generally are insolubilized before or after the competition, so that the standard and analyze that are bound to the antibodies may conveniently be separated from the standard and analyze which remain unbound.","Sandwich assays involve the use of two antibodies, each capable of binding to a different immunogenic portion, or epitope, of the protein to be detected. In a sandwich assay, the test sample analyze is bound by a first antibody which is immobilized on a solid support, and thereafter a second antibody binds to the analyze, thus forming an insoluble three-part complex. See, e.g., U.S. Pat No. 4,376,110. The second antibody may itself be labeled with a detectable moiety (direct sandwich assays) or may be measured using an anti-immunoglobulin antibody that is labeled with a detectable moiety (indirect sandwich assay). For example, one type of sandwich assay is an ELISA assay, in which case the detectable moiety is an enzyme.","For immunohistochemistry, the tumor sample may be fresh or frozen or may be embedded in paraffin and fixed with a preservative such as formalin, for example.","The antibodies may also be used for in vivo diagnostic assays. Generally, the antibody is labeled with a radionuclide (such as In, Tc, C, I, I, H, P or S) or a dye so that the tumor can be localized using immunoscintiography.","In one embodiment, a method of detecting NRP1 in a biological sample (e.g., tissue, blood, sera, spinal fluid) or a prepared biological sample can comprise the step of contacting an antibody of this invention with the sample and observing the anti-NRP1 antibody bound to the NRP1 in the sample or determining the amount of the anti-NRP1 antibody bound to NRP1 in the sample. In another embodiment, a method of detecting NRP1 in a subject comprises the step of administering an antibody of this invention to the subject and observing the anti-NRP1 antibody bound to the NRP1 in the subject or determining the amount of the anti-NRP1 antibody bound to NRP1 in the subject (e.g., human, mouse, rabbit, rat, etc.).","Diagnostic Kits","As a matter of convenience, the antibody of the present invention can be provided in a kit, i.e., a packaged combination of reagents in predetermined amounts with instructions for performing the diagnostic assay. Where the antibody is labeled with an enzyme, the kit will include substrates and cofactors required by the enzyme (e.g., a substrate precursor which provides the detectable chromophore or fluorophore). In addition, other additives may be included such as stabilizers, buffers (e.g., a block buffer or lysis buffer) and the like. The relative amounts of the various reagents may be varied widely to provide for concentrations in solution of the reagents which substantially optimize the sensitivity of the assay. Particularly, the reagents may be provided as dry powders, usually lyophilized, including excipients which on dissolution will provide a reagent solution having the appropriate concentration.","Articles of Manufacture","In another embodiment of the invention, an article of manufacture containing materials useful for the treatment of the disorders described above is provided. The article of manufacture comprises a container and a label. Suitable containers include, for example, bottles, vials, syringes, and test tubes. The containers may be formed from a variety of materials such as glass or plastic. The container holds a composition which is effective for treating the condition and may have a sterile access port (for example the container may be an intravenous solution bag or a vial having a stopper pierceable by a hypodermic injection needle). The active agent in the composition is the antibody. The label on, or associated with, the container indicates that the composition is used for treating the condition of choice. The article of manufacture may further comprise a second container comprising a pharmaceutically-acceptable buffer, such as phosphate-buffered saline, Ringer's solution and dextrose solution. It may further include other materials desirable from a commercial and user standpoint, including other buffers, diluents, filters, needles, syringes, and package inserts with instructions for use.","The following examples are intended merely to illustrate the practice of the present invention and are not provided by way of limitation. The disclosures of all patent and scientific literatures cited herein are expressly incorporated in their entirety by reference.","A. Generation of Antibody Phage Libraries","A variety of methods are known in the art for generating antibody libraries displayed on phage or displayed using other technologies. One of the advantages of these libraries, as compared to conventional hybridoma technologies, is that they are well suited for generating cross-species functional antibodies, because they allow in vitro selection and screening without the introduction of immune tolerance. Smith (1985) 228:1315-1317; Bradbury and Marks (2004) 290:29-49.","In general, two types of combinatorial antibody libraries have been developed, distinguished by the source of repertoires. Most libraries to date are \u201cnatural\u201d antibody libraries which use the natural repertoires as the source for its diversity, where the genes as message RNA of immune cells from na\u00efve or immunized animals or human are amplified and cloned into vector for phage display or other display technology, such as ribosome or yeast display. The natural antibodies usually have multiple frameworks, which together with variable CDRs sequences and the recombination of light chain and heavy chain made up the diversity of the library. The size of the library determines the performance of the libraries since the repertoires are in general larger than the library size. The synthetic library, on the other hand, is a new branch of library where the diversity is designed and built into the library with synthetic DNA. Single or multiple frameworks have been used. For single framework library, the source of the diversity solely depends on the degeneracy of synthetic DNA designed to create the diverse CDR loops. Both the diversity design and the size of the libraries are critical for the library performance, which can be measured by the affinity of the antibodies found from the libraries. Knappik et al. (2000) 296:57-86; Sheets et al. (1998) 95: 6157-6162; de Haard et al. (1999) 274:18218-18230.","By introducing synthetic diversity at solvent-exposed positions within the variable heavy chain complementarity-determining regions (CDRs), Lee et al. developed a phage-displayed synthetic antibody library built on a single human framework (VL, VH). Sidhu et al. (2004) 338:299-310; Lee et al. (2004) 340:1073-1093; Carter et al. (1992) 89:4285-4289. This \u201cVH library\u201d was displayed as a bivalent antigen-binding fragment (Fab\u20322) and used tailored codons to mimic the natural diversity observed in human immunoglobulin. Lee et al. (2004) 284:119-132. While the VH library has performed well as measured by the affinity and function of derived antibodies, further modifications in library design can be desirable in generating functional antibodies of certain antigen targets of interest. The recent availability of trinucleotides for oligonucleotide synthesis enabled the ability to increase amino acid diversity without increasing DNA diversity. A new \u201cVH\/VL library\u201d was thus generated as described herein, wherein the highly variable positions in CDR-L3 were further diversified, since CDR-H3 and CDR-L3 form the inner sphere of the antigen-binding site. Mian et al. (1991) 217:133-151.","Materials and Methods","The VH\/VL na\u00efve library template with consensus CDR-L1, -L2, -L3, -H1 and -H2 was generated using oligonucleotide-directed mutagenesis on phagemid pV0350-4 with stop codons on CDR-H3 and displaying bivalently on the surfaces of M13 bacteriophage particles. Lee et al. (2004) 340:1073-1093. Phage-displayed libraries were constructed using Kunkel mutagenesis method as described (Kunkel et al. (1991) 204:125-139), with a mixture of mutagenic oligonucleotides designed to introduce mutations at the designed sites in CDR-L3, H1, H2 and H3 and repair CDR-H3 stop codons. The mutagenesis reactions (\u02dc10 \u03bcg DNA) were electroporated into SS320 cells (\u02dc10cells), as described (Sidhu et al. (2004)).","Results","The VH\/VL library described here utilized the trastuzumab (HERCEPTIN\u00ae)derived VLand VHframework as used for the VH library, which has been shown to display well on bacteriophage, express well in , and which can rapidly be converted to a full length IgG that expresses well in mammalian cells. Lee et al. (2004) 340:1073-1093; Carter et al. (1992) 89:4285-4289. The libraries were displayed on the phage surface as a bivalent Fab (Fab\u20322) by being fused to the phage coat protein P3. This bivalent display was intended to increase the apparent binding affinities to immobilized antigens and help to improve the recovery of rare or low-affinity phage antibody clones.","To avoid potential biases inherited from the Herceptin\u00ae-derived CDR sequences maintained in the light chain of the VH library, consensus kappa I CDR sequences were introduced into the template for the VH\/VL library. Consensus CDR residues are determined by selecting the most prevalent amino acids existing in natural human antibodies. The stop codons, previously employed in the heavy chain of the VH library to ensure mutagenesis in all 3 CDRs, were similarly replaced with consensus subgroup III sequences for CDR-H1 and CDR-H2. The consensus CDR sequences represent the most prevalent amino acid in each position. CDR-H3 plays a dominant role in antigen recognition, thus several stop codons were placed in H3 to ensure functional antibody clones from the libraries were different from each other. Xu et al. (2000) 13: 37-45. The presence of human consensus CDR sequences was expected to allow partially mutated variants (not all targeted CDRs are changed) to be displayed and to remain potentially functional in binding. In this way the VH\/VL design has the advantage of increasing the ratio of functional phage antibody clones in the library. The CDR sequences used were SISSYL for CDR-L1 (positions 28-33), GASSRA for CDR-L2 (positions 50-55), YYSSPL for CDR-L3 (positions 91-96), FTFSSYAMS for CDR-H1 (positions 27-35), and RISPSGGSTY for CDR-H2 (positions 50-58), and WXXXRPXXMDY for CDR-H3 (positions 95-102, X is a stop codon) as shown in Table I. The prevalence of each position in human antibodies is also shown in Table I.","Diversity in the VH\/VL library was introduced into a subset of CDR positions based on their high solvent exposure and\/or especially high variability among natural antibody sequences. Positions chosen for mutagenesis and the diversity that was introduced are shown in Table II. For example, in CDR-H1, position 27, 28, 30, 31, 32, 33 and 34 were chosen to diversify. For the VH\/VL library design, degenerate oligo codons or trinucletides were used to guide the diversity in each position so that the most prevalent amino acids would be represented. For example in CDR-H1 (position 30), serine represents about 50% of natural diversity, so a mixture of trinucletides (X1) that have ca. 52% serine and 2.5% of other amino acids except cysteine was used.","CDR-H3 and CDR-L3 form the center of the antigen-binding site and therefore show the highest frequency of antigen contacts in structurally known antibody-antigen complexes. Chothia et al. (1989) 342:877-883. Five residues in CDR-L3 (Table II) with the highest variability were randomized Overall CDR-H3 is the most diverse in terms of length, sequence, and structure and is a key component of the diversity in natural antibodies. Xu and Davis (2000) 13:37-45; Wu et al. (1993) 16:1-7. Thus 12 sub-libraries were constructed with different CDR-H3 lengths varying from 9 to 20 amino acids. Combined, these sub-libraries cover approximately 90% of CDR-H3 length variation in natural antibodies. Oligonucleotides encoding CDR-H3 were synthesized using trinucleotide codons. This enabled us to easily delete cysteine (rare in CDR-H3), and to boost levels of glycine, tyrosine and serine, the most abundant residues in CDR-H3. Mian et al. (1991) 217:133-151. Codon X7, a trinucleotide mixture of about 15.6% each serine, tyrosine, and glycine, with 3.1% each of the remaining amino acids except cysteine, was used for each position in CDR-H3 (theoretical calculations for all trinucleotides mixture). Different combinations of trinucleotides were also used in selected positions of CDR-H1, H2, H3, and L3. As shown in Table II and Supplemental Table I, Codons X1 to X6 have a high percentage of serine, tyrosine, or glycine. X1 has 52.5% of serine, X2 has 52.5% tyrosine, X3 has 10% tyrosine, glycine or serine, X4 has 28.8% glycine, X5 has 19.2% tyrosine, glycine or serine, and X6 has 20% tyrosine or serine. The VH\/VL antibody phage library was estimated to have approximately 10variants displayed.",{"@attributes":{"id":"p-0203","num":"0202"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"center"}},"thead":{"row":{"entry":"TABLE I"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Consensus sequence of CDR-L1, CDR-L2, CDR-L3, CDR-H1 and CDR-H2 in the template"},{"entry":"of VH\/VL library. Consensus CDR residues are determined by selecting the most prevalent "},{"entry":"amino acids existing in natural human antibodies. The prevelance (%) of each residue in "},{"entry":"human antibodies at a given position is shown, which is calculated from the alignment of "},{"entry":"approximately 1600 human light chain sequences and 3500 human heavy chain sequences"},{"entry":"in the Kabat database (Kabat et al. (1977)    252: 6609-6616)."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"112pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},"Prevalence in natural"]},{"entry":["CDRs","Positions","Residues","human antibodies (%)"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["CDR-L1","28","S","33"]},{"entry":[{},"29","I","40"]},{"entry":[{},"30","S","55"]},{"entry":[{},"31","S","44"]},{"entry":[{},"32","Y","67"]},{"entry":[{},"33","L","94"]},{"entry":["CDR-L2","50","G","25"]},{"entry":[{},"51","A","79"]},{"entry":[{},"52","S","95"]},{"entry":[{},"53","S","36"]},{"entry":[{},"54","R","60"]},{"entry":[{},"55","A","45"]},{"entry":["CDR-L3","91","Y","54"]},{"entry":[{},"92","Y","23"]},{"entry":[{},"93","S","46"]},{"entry":[{},"94","S","24"]},{"entry":[{},"95","P","80"]},{"entry":[{},"96","L","22"]},{"entry":["CDR-H1","27","F","45"]},{"entry":[{},"28","T","54"]},{"entry":[{},"29","F","73"]},{"entry":[{},"30","S","68"]},{"entry":[{},"31","S","50"]},{"entry":[{},"32","Y","64"]},{"entry":[{},"33","A","22"]},{"entry":[{},"34","M","46"]},{"entry":[{},"35","S","34"]},{"entry":["CDR-H2","50","R","17"]},{"entry":[{},"51","I","84"]},{"entry":[{},"52","S","26"]},{"entry":[{},"\u200252a","P","29"]},{"entry":[{},"53","S","24"]},{"entry":[{},"54","G","37"]},{"entry":[{},"55","G","53"]},{"entry":[{},"56","S","28"]},{"entry":[{},"57","T","56"]},{"entry":[{},"58","Y","32"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0204","num":"0203"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"273pt","align":"center"}},"thead":{"row":{"entry":"TABLE II"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Designed diversity of CDR-L3, CDR-H1, CDR-H2 and CDR-H3 for VH\/VL library."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"147pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Design diversity",{}]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"112pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Residues encoded (%)","Natural diversity"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"8"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["CDRs","Positions","Codon","Y","G","S","others (-Cys)","coverage (%)"]},{"entry":{"@attributes":{"namest":"1","nameend":"8","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"8"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"56pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["CDR-L3","Y91",{"i":"TAC"},"100","\u2014","\u2014","\u2014","77"]},{"entry":[{},{},{"i":"MGC"},"\u2014","\u2014","\u2014","R\/S(50)"]},{"entry":[{},"Y92",{"b":"X5"},"19.2","19.2","19.2","2.5","100"]},{"entry":[{},"S93",{"b":"X1"},"2.5","2.5","52.5","2.5","100"]},{"entry":[{},"S94",{"b":"X6"},"20","3.3","20","3.3","100"]},{"entry":[{},"L96",{"i":"NTC"},"\u2014","\u2014","\u2014","F\/I\/L\/V(25)","45"]},{"entry":["CDR-H1","F27",{"i":"TWC"},"50","\u2014","\u2014","F(50)","65"]},{"entry":[{},"T28",{"i":"ASC"},"\u2014","\u2014","50","T(50)","90"]},{"entry":[{},"S30",{"i":"ASC"},"\u2014","\u2014","50","T(50)","86"]},{"entry":[{},"S31",{"b":"X1"},"2.5","2.5","52.5","2.5","100"]},{"entry":[{},"Y32",{"b":"X2"},"52.5","2.5","2.5","2.5","100"]},{"entry":[{},"A33",{"b":"X7"},"15.6","15.6","15.6","3.1","100"]},{"entry":[{},"M34",{"i":"ATS"},"\u2014","\u2014","\u2014","M\/I(50)","67"]},{"entry":["CDR-H2","R50",{"b":"X3"},"5","5","5","5\u2002\u2009","100"]},{"entry":[{},"S52",{"b":"X6"},"20","3.3","20","3.3","100"]},{"entry":[{},"P52a",{"i":"CCT"},"\u2014","\u2014","\u2014","P(100)","100"]},{"entry":[{},{},{"b":"X7"},"15.6","15.6","15.6","3.1"]},{"entry":[{},"S53",{"b":"X7"},"15.6","15.6","15.6","3.1","100"]},{"entry":[{},"G54",{"i":"RRC"},"\u2014","\u2014","25","D\/G\/N(25)","81"]},{"entry":[{},"S56",{"i":"DMT"},"16.6","\u2014","16.6","A\/D\/N\/T(16.6)","81"]},{"entry":[{},"Y58",{"i":"DAC"},"33.3","\u2014","\u2014","D\/N(33.3)","70"]},{"entry":["CDR-H3","95",{"b":"X4"},"3.8","28.8","3.8","3.8","100"]},{"entry":[{},"96",{"b":"X4"},"3.8","28.8","3.8","3.8","100"]},{"entry":[{},"97~100k",{"b":"(X7)4~15"},"15.6","15.6","15.6","3.1",">98"]},{"entry":[{},"100l",{"b":"X7"},"15.6","15.6","15.6","3.1","100"]},{"entry":[{},{},{"i":"GBT"},"\u2014","33.3","\u2014","A\/V(33.3)"]},{"entry":[{},"100m",{"i":"TTC"},"\u2014","\u2014","\u2014","F(100)","89"]},{"entry":[{},{},{"i":"ATG"},"\u2014","\u2014","\u2014","M(100)"]},{"entry":[{},"101",{"i":"GAT"},"\u2014","\u2014","\u2014","D(100)","92"]},{"entry":[{},"102",{"i":"TAC"},"100","\u2014","\u2014","\u2014","67"]},{"entry":[{},{},{"i":"GTC"},"\u2014","\u2014","\u2014","V(100)"]},{"entry":{"@attributes":{"namest":"1","nameend":"8","align":"center","rowsep":"1"}}},{"entry":"CDR positions chosen for randomization in CDR-L3, CDR-H1, CDR-H2 and CDR-H3 are listed with consensus residues in the library template. Designed diversity is either a group of residues encoded by a tailored degenerate codon (italics) or 19 amino acids without cysteine encoded by mixtures of trinucleotides codon (bold text) so that the percentage of amino acid types encoded at each position was close to or higher than 50% of amino acid types found in the database. For particular positions, all 19 amino acids without cysteine are introduced using trinucleotides codon mixtures with different bias toward Tyr (Y), Gly (G) and Ser (S)."}]}}]}},"br":[{},{}]},"Library sorting and screening to identify anti NRP1 antibodies\u2014Human and murine NRP1 constructs (1-641aa) were cloned into mammalian expression vector, and expressed in CHO cells. Truncated form of NRP-1, a1a2 and b1b2 domains, were expressed in baculovirus. NUNC 96 well Maxisorp immunoplates were coated overnight at 4\u00b0 with target antigen (10 ug\/ml) and were blocked for 1 hr at room temperature with phage blocking buffer PBST (PBS and 1% BSA and 0.05% Tween 20). The antibody phage libraries were added to antigen plates and incubated overnight at RT. The following day antigen-coated plates were washed 10 times with PBT (PBS with 0.05% T-20), and bound phage were eluted with 50 mM HCl and 500 mM NaCl for 30 minutes and neutralized with equal volume of 1M Tris base pH7.5. Recovered phage was amplified in XL-1 Blue cells. During the subsequent selection rounds, incubation of antibody phage with the antigen-coated plates was reduced to 2-3 hours, and the stringency of plate washing was gradually increased.","Anti-NRP1 antibodies binding affinities, specificity and flow cytometry analysis\u2014Phage antibody ICvalues were determined using competitive phage-binding ELISA as described. Lee et al. (2004) 340:1073-1093. Competition curves were fit with a four-parameter non-linear regression curve-fitting program (Kaleidagraph, Synergy Software) to determine the ICvalues which were calculated as the concentration of antigen in solution binding stage that inhibited 50% of the phage-displayed antibody from binding to immobilized antigen.","Clones of interest were then reformatted into IgGs by cloning Vand Vregion of individual clones into LPG3 and LPG4 vector respectively (Carter et al. (1992) 89:4285-4289), transiently expressed in mammalian cells, and purified with protein A columns. For binding affinity determinations of anti-NRP1 IgGs, Surface Plasmon Resonance (SPR) measurement with a BIAcore\u2122-3000 instrument was used. Anti-NRP1 IgGs were coupled to activated CM5 biosensor chips to achieve approximately 500 response units (RU), followed by blocking un-reacted groups with 1M ethanolamine. For kinetic measurements, two-fold serial dilutions of Neuropilin (0.7 to 500 nM) were injected in PBST buffer at 25\u00b0 C. with a flow rate of 30 \u03bcl\/min. Association rates (k) and dissociation rates (k) were calculated using a simple one-to-one Langmuir binding model (BIAcore Evaluation Software version 3.2). The equilibrium dissociation constant (Kd) was calculated as the ratio k\/k.","For binding specificity tests, 10 \u03bcg\/ml of IgGs in PBST buffer was incubated with 2 \u03bcg\/ml antigen-coated 96-well Maxisorp plates for at least 1 hr, and the plates were washed with PBT buffer. Bound antibodies were detected with anti-human antibody HRP conjugates, developed with TMB substrate for approximately 5 minutes, quenched with 1M HPO, and read spectrophotometrically at 450 nm","For flow cytometry analysis, HUVEC cells were detached from the tissue culture flasks with cell dissociation buffer. Dissociated cells were washed in PBS and re-suspended in PBS containing 2% Fetal Bovine Serum (FACS buffer). Cells were incubated with 10 ug\/ml anti-NRP1 antibodies or control antibody (anti-IgE) in FACS buffer on ice for 30 minutes. Cells were then washed twice in PBS, and stained in FACS buffer with PE-Fab\u20322 goat anti-human IgG, Fc specific antibody on ice for 30 minutes. Following two PBS washes, the cells were re-suspended in 200 \u03bcl FACS buffer and analyzed by flow cytometry (FACS caliber, Benton Dickenson, Mountain View, Calif.) using Cell-Quest software.","Results","The 12 VH\/VL sub-libraries were individually panned against immobilized CHO cell expressed hNRP1 (a1a2b1b2)-Fc protein for the first round of selection. Eluted phages from each sub-library were amplified and then combined for second round of selection. Since hNRP1-Fc was used as the antigen, the pooled phage was pre-absorbed with excess irrelevant Fc fusion protein after the first round of panning to minimize the recovery of anti-Fc phage antibodies. After the fourth round of panning, 95 randomly picked phage clones were evaluated for the ability to specifically bind to NRP1. Ninety percent of clones were positive for hNRP1 binding and 40% of positive clones bound both human and murine NRP1.","These phage clones were sequenced, and 10 unique clones were chosen for further characterization. All bound to both human and murine NRP1 with an ICbelow 70 nM in the phage ELISA. Clone YW64.3 bound human and murine NRP1 with an ICof 0.5 and 3.4 nM, respectively (Table III). Sequences from the 10 clones reflect the VH\/VL library design with variable CDR-H3 lengths and varied changes distributed throughout CDR-H1, H2 and L3 yet some consensus CDR sequences from the library template are retained. Clones 64.3 and 64.29 have changes in all 4 CDRs and also have the best binding affinities to both human and murine NRP1 (Table III).",{"@attributes":{"id":"p-0212","num":"0211"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"441pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u2003III"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Partial\u2003CDR\u2003sequences\u2003and\u2003binding\u2003affinity\u2003(phage\u2003IC50)\u2003for\u2003anti-NRP1\u2003"},{"entry":"antibodies.\u2003Only\u2003CDR\u2003amino\u2003acid\u2003sequences\u2003at\u2003randomized\u2003positions"},{"entry":"\u2003\u2003of\u200311\u2003different\u2003phage\u2003clones\u2003initially\u2003identified\u2003from\u2003VH\/VL"},{"entry":"\u2003library\u2003are\u2003shown.\u2003The\u2003affinities\u2003as\u2003IC\u2003values\u2003to\u2003hNRP1\u2003and\u2003"},{"entry":"mNRP1\u2003were\u2003measured\u2003with\u2003competitive\u2003phage\u2003ELISA."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"11"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"8","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"9","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"10","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"11","colwidth":"63pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Partial","SEQ","Partial","SEQ","Partial","SEQ","Partial","SEQ",{},{}]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"10"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"8","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"9","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"10","colwidth":"119pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"CDR-L3","ID","CDR-H1","ID","CDR-H2","ID","CDR-H3","ID","IC50\u2003(nM)"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"11"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"8","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"9","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"10","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"11","colwidth":"63pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Clone","(91-96)","NO:","(27-34)","NO:","(50-58)","NO:","(95-102)","NO:","mNRP\u20031","hNRP\u20031"]},{"entry":{"@attributes":{"namest":"1","nameend":"11","align":"center","rowsep":"1"}}},{"entry":["YW64.3","YMSVPI","\u20037","FSFSSEPI","18","SITGKNGYY","29","WGKKVYG----MDV","40","3.4\u2003\u00b1 0.9","0.48\u2003\u00b1 0.02"]},{"entry":{}},{"entry":["YW\u200364.4","YYSSPL","\u20038","FTFSSYAM","19","SIAGSGGYY","30","WGGSNGSG---FDY","41","\u200335\u2003\u00b1 9.6","\u2003\u200324\u2003\u00b1 7.4"]},{"entry":{}},{"entry":["YW\u200364.14","YYSSPL","\u20039","YSFSSHMM","20","SIYPPGGYY","31","WGSRSPG----MDV","42","\u200311\u2003\u00b1 3.4","\u20035.2\u2003\u00b1 0.9"]},{"entry":{}},{"entry":["YW\u200364.23","YYSSPL","10","FTFSSYAM","21","TIIPHGGYY","32","WAKRSYG----MDV","43","\u200346\u2003\u00b1 21","\u2003\u200347\u2003\u00b1 14"]},{"entry":{}},{"entry":["YW\u200364.28","YYSSPL","11","FTFSSYAM","22","WISPLNGYY","33","WGRRYIG----MDV","44","7.9\u2003\u00b1 1","\u20031.2\u2003\u00b1 0.2"]},{"entry":{}},{"entry":["YW\u200364.29","RYSVPI","12","FTFSSYQL","23","SIF-SGGYY","34","YGNHV------MDV","45","9.9\u2003\u00b1 4.2","\u20032.7\u2003\u00b1 0.1"]},{"entry":{}},{"entry":["YW\u200364.30","YYSSPL","13","FTFSSYAM","24","SISRGDGYY","35","WAGGSA-----MDV","46","\u200360\u2003\u00b1 11","\u2003\u200350\u2003\u00b1 19"]},{"entry":{}},{"entry":["YW\u200364.47","YYSSPL","14","FTFSQYSI","25","TIYPFGGYY","36","FGQSYYGGSYAMDV","47","\u200367\u2003\u00b1 25","\u2003\u200321\u2003\u00b1 6"]},{"entry":{}},{"entry":["YW\u200364.53","YYSSPL","15","FTFTSRTM","26","SIS-SGGYY","37","WESYYG-----MDV","48","\u200316\u2003\u00b1 2.9","\u2003\u2003\u20034\u2003\u00b1 0.6"]},{"entry":{}},{"entry":["YW\u200364.55","YYSSPL","16","FTFSSYAM","27","SIYSTGGYY","38","WGYPG------MDV","49","\u200317\u2003\u00b1 5.8","\u20033.9\u2003\u00b1 1"]},{"entry":{}},{"entry":["YW\u2003107.4","YYSSPL","17","FTFSSYAM","28","QISPAGGYN","39","ELPYYRMSKV-MDV","50","\u200338\u2003\u00b1 20","\u20036.1\u2003\u00b1 1"]},{"entry":{"@attributes":{"namest":"1","nameend":"11","align":"center","rowsep":"1"}}}]}}]}}},"Domain-truncated variants of hNRP1 were used to identify the binding epitope for these clones and all were mapped to the a1a2 domain of hNRP1. To select phage antibodies that bound to the b1b2 domain of NRP1 and potentially block the binding of VEGF, we initiated a new panning process using baculovirus expressed hNRP1 (b1b2)-His as an immobilized antigen. Following 4 rounds of selection, only one unique clone, YW107.4 was identified that bound both human and murine NRP1. YW107.4 bound to human and murine NRP1 with an ICof 6 and 38 nM in the Phage ELISA (Table III).","Characterization of selected anti-NRP1 IgG\u2014Selected anti-NRP1 clones from Table III were reformatted into full length human IgG1, expressed in CHO cells and purified for further characterization. The anti-NRP1 phage antibodies YW64.3 and YW107.4 bound specifically to human and murine NRP1 and did not bind to human or murine NRP-2, ErbB2-ECD or BSA. . Each of the other eight-phage antibodies showed similar specificity. By surface plasmon resonance, immobilized YW64.3 and YW107.4 IgG did not interact with these antigens at concentrations up to 500 nM. However, both YW64.3 and YW107.4 bound human NRP1 with a Kd of 0.9 and 5 nM as well as murine NRP1 with a Kd of 7.8 and 11 nM, respectively. Although these antibodies were selected using plate-immobilized antigen, FACS analysis demonstrated all purified IgGs also bound to HUVEC cells which express hNRP1 endogenously (e.g. YW64.3 and YW107.4 shown in ).","C. Affinity maturation of Anti-NRP1 Antibodies","YW107.4 bound to the b1b2 domain of both human and murine NRP1 with a phage ICof 6 and 38 nM, respectively (Table III). To improve potency in vivo, this clone was affinity-matured using human NRP1-His.","Materials and Methods","To generate the library template for affinity maturation of clone YW107.4, the GCN4 leucine zipper of the parental phagemid was first removed using Kunkel mutagenesis to provide a monovalent display Fab format. A stop codon was incorporated in CDR-L3. A soft randomization strategy was used for affinity maturation, which introduced the mutation rate of approximately 50% at the selected positions by the mutagenic DNA synthesizing with 70-10-10-10 mixtures of bases favoring the wild type nucleotides. Gallop et al. (1994) 37:1233-1251. Three different libraries with combinations of CDR loops, L1\/L2\/L3, L3\/H1\/H2 and L3\/H3 randomization, were generated through soft randomizing selected residues at positions 28-32 of CDR-L1, 50 and 53-55 of CDR-L2, 91, 92, 93, 94 and 96 of CDR-L3, 28-35 of CDR-H1, 50-58 of CDR-H2, and 95-100 of CDR-H3.","For selecting affinity-matured clones, phage libraries were subjected to plate sorting for the first round and followed by four rounds of solution phase sorting as described. Lee et al. (2004) 340:1073-1093. At the first round of plate sorting, three libraries were added to hNRP1-coated plate separately for 1 hr at 37\u00b0 C. After that, four rounds of solution phase sorting were performed to enhance the efficiency of affinity-based selection with increasing stringency as follow: round 2 (5 nM biotinylated hNRP1), round 3 (1 nM biotinylated hNRP1), round 4 (0.5 nM biotinylated hNRP1 and 250 nM non-biotinylated hNRP1 competitor at 37\u00b0 C. for 1 hr) and round 5 (0.5 nM biotinylated hNRP1 and 500 nM non-biotinylated hNRP1 competitor at 37\u00b0 C. for 3 hr). During the selection process, the reaction without biotinylated hNRP1 was included and served as background phage binding for calculating the enrichment of each round of panning.","After five rounds of panning, a high-throughput single-point competitive phage ELISA was used to rapidly screen for high-affinity clones as described. Sidhu et al. (2004) 338:299-310. Clones with low ratio of the absorbance at 450 nm in the presence of 5 nM hNRP1 to that in the absence of hNRP1 were chosen for further characterization.","Results","Three different CDR combinations, L1\/L2\/L3, L3\/H1\/H2 and L3\/H3, were targeted for randomization using a \u2018soft randomization\u2019 strategy that maintains a wild-type sequence bias such that selected positions are mutated only 50 percent of the time. Gallop et al. (1994) 37:1233-1251. For affinity maturation, the monovalent Fab was displayed on phage rather than bi-valent Fab to reduce potential avidity during selection. Stop codons were introduced at CDR-L3 in each sub-library. Off-rate selection strategies (see Methods) were employed to improve the affinity of YW107.4, since it already possessed a relatively high association rate constant (2.2\u00d710), but the dissociation rate constant (1.1\u00d710) was relatively fast (Table V).","In the first round of selection, all 3 CDR soft-randomized libraries were panned against immobilized hNRP1 followed by subsequent rounds with a solution-phase sorting strategy to limit target concentration and enhance affinity-based selection. The concentration of biotinylated-hNRP1 was gradually reduced from 5 to 0.5 nM and a 500-fold excess of non-biotinylated hNRP1 was added to compete for fast off-rate binders. The mixture was also incubated at 37\u00b0 C. for up to 2 hrs.","The L1\/L2\/L3 library showed significant enrichment following round 5. Ninety-six clones were randomly picked, sequenced and then affinities ranked. Twenty-three unique phage clones were selected and purified for further characterization. Most clones had improved affinity for hNRP1 as determined by phage competition ELISA. Surprisingly, the affinity for mNRP1 was also improved despite being omitted from the selection process, suggesting that the clones bound to a conserved epitope. Selected clones had 4 to 7 changes in the 3 light chain CDRs; positions 28 and 30 in CDR-L1 tended to be substituted with Tyrosine and Histidine, respectively, whereas positions 92, 93, and 96 in CDR-L3 were more diverse (Table IV).","Clones with the highest affinity for both human and murine NRP1 (YW107.4.18, YW107.4.38, YW107.4.52, YW107.4.63, YW107.4.76a and YW107.4.87) were reformatted and expressed as full-length antibodies. All 6 IgGs had improved affinity for hNRP1 and maintained complete blocking of VEGF-A binding. Affinities for human and murine NRP1 ranged from 0.4 to 1.8 nM (Table V). The dissociation rate constant of YW107.4.87 was improved leading to an overall improvement in affinity of about 10-fold for both human and murine NRP1; no binding was observed to human or murine NRP2 (). YW107.4.87 also showed improved binding to cell surface NRP1 ().",{"@attributes":{"id":"p-0223","num":"0222"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"350pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u2003IV"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"CDR\u2003sequences\u2003and\u2003binding\u2003affinity\u2003(phage\u2003IC50)\u2003for\u2003"},{"entry":"affinity\u2003improved\u2003YW107.4\u2003clones.\u2003The\u2003deduced\u2003amino\u2003acids\u2003"},{"entry":"sequences\u2003of\u2003affinity-matured\u2003YW107.4\u2003clones\u2003at"},{"entry":"\u2003randomized\u2003light\u2003chain\u2003CDR\u2003positions\u2003are\u2003shown.\u2003Fab-"},{"entry":"phage\u2003IC\u2003values\u2003of\u2003individual\u2003clone\u2003against\u2003hNRP1\u2003and"},{"entry":"\u2003mNRP1\u2003from\u2003competition\u2003phage\u2003ELISA\u2003were"},{"entry":"\u2003used\u2003to\u2003compare\u2003affinity\u2003improvement\u2003with"},{"entry":"the\u2003YW107.4."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"9"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"8","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"9","colwidth":"98pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Partial","SEQ","Partial","SEQ","Partial","SEQ",{},{}]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"8"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"8","colwidth":"154pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"CDR-L1","ID","CDR-L2","ID","CDR-L3","ID","IC50\u2003(nM)"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"9"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"8","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"9","colwidth":"98pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Clone","(28-32)","NO:","(50-55)","NO:","(92-95)","NO:","mNRP1","hNRP1"]},{"entry":{"@attributes":{"namest":"1","nameend":"9","align":"center","rowsep":"1"}}},{"entry":["YW107.4","SISSY","51","GASSRA","75","YSSPL","\u200399","\u200338\u2003\u00b1 20","\u20036.1\u2003\u00b1 1"]},{"entry":{}},{"entry":["YW107.4.33a","YISSY","52","GASRRA","76","IGSPI","100","1.5\u2003\u00b1 0.4","0.31\u2003\u00b1 0.04"]},{"entry":{}},{"entry":["YW107.4.53","YISSY","53","GASRRE","77","LNSPL","101","1.7\u2003\u00b1 0.1","0.30\u2003\u00b1 0.02"]},{"entry":{}},{"entry":["YW107.4.66","YISSY","54","GASSRA","78","IVSPL","102","2.6\u2003\u00b1 0.6","0.45\u2003\u00b1 0.02"]},{"entry":{}},{"entry":["YW107.4.76a","YISSY","55","GASRRA","79","LRSPH","103","0.8\u2003\u00b1 0.4","0.23\u2003\u00b1 0.01"]},{"entry":{}},{"entry":["YW107.4.78","YISSY","56","GASSRE","80","LSSPI","104","1.7\u2003\u00b1 0.2","0.30\u2003\u00b1 0.02"]},{"entry":{}},{"entry":["YW107.4.85","YISSY","57","GASSGE","81","IISPI","105","\u2003\u20034\u2003\u00b1 0.7","0.44\u2003\u00b1 0.06"]},{"entry":{}},{"entry":["YW107.4.51","RISSY","58","GASRRE","82","KLSPL","106","\u200312\u2003\u00b1 6","\u2003\u2003\u20032\u2003\u00b1 0.5"]},{"entry":{}},{"entry":["YW107.4.58","YISSY","59","GASSRA","83","KSSPR","107","3.5\u2003\u00b1 1.5","0.73\u2003\u00b1 0.28"]},{"entry":{}},{"entry":["YW107.4.42","WIHSY","60","GASSSA","84","YSSPL","108","1.2\u2003\u00b1 2.2","0.81\u2003\u00b1 0.09"]},{"entry":{}},{"entry":["YW107.4.59","RIHSY","61","GASSRA","85","YISPL","109","\u200326\u2003\u00b1 14","\u20031.8\u2003\u00b1 0.7"]},{"entry":{}},{"entry":["YW107.4.33b","YIHSY","62","GASRRA","86","YGTPH","110","7.8\u2003\u00b1 2","0.58\u2003\u00b1 0.07"]},{"entry":{}},{"entry":["YW107.4.54","PLHSY","63","GASSRA","87","YRSPL","111","2.7\u2003\u00b1 1","0.84\u2003\u00b1 0.05"]},{"entry":{}},{"entry":["YW107.4.63","YLSSY","64","GASSSE","88","ISVPL","112","0.8\u2003\u00b1 0.1","0.15\u2003\u00b1 0.01"]},{"entry":{}},{"entry":["YW107.4.38","YLSSY","65","GASSRA","89","LRSPI","113","1.4\u2003\u00b1 0.3","0.28\u2003\u00b1 0.01"]},{"entry":{}},{"entry":["YW107.4.18","YFSSY","66","GASTHE","90","IRSPL","114","1.1\u2003\u00b1 0.3","0.21\u2003\u00b1 0.02"]},{"entry":{}},{"entry":["YW107.4.52","YFSSY","67","GASTLA","91","IRSPL","115","1.1\u2003\u00b1 0.3","0.22\u2003\u00b1 0.04"]},{"entry":{}},{"entry":["YW107.4.87","YFSSY","68","GASSRA","92","LGSPP","116","1.5\u2003\u00b1 0.2","0.21\u2003\u00b1 0.04"]},{"entry":{}},{"entry":["YW107.4.55","LTHSY","69","GASSRA","93","YSSPL","117","\u2003\u20032\u2003\u00b1 2.4","0.93\u2003\u00b1 0.05"]},{"entry":{}},{"entry":["YW107.4.20","RTHSY","70","GASSRA","94","YGSPH","118","\u200333","0.45\u2003\u00b1 0.06"]},{"entry":{}},{"entry":["YW107.4.12","YTHSY","71","GASSRA","95","YSSPI","119","3.4\u2003\u00b1 0.6","0.10\u2003\u00b1 0.02"]},{"entry":{}},{"entry":["YW107.4.17","YTHSY","72","GASSRA","96","YSSPV","120","\u2003\u20036\u2003\u00b1 1","0.24\u2003\u00b1 0.02"]},{"entry":{}},{"entry":["YW107.4.41","WTHSY","73","GASRLE","97","FISPH","121","5.5\u2003\u00b1 2.3","0.43\u2003\u00b1 0.07"]},{"entry":{}},{"entry":["YW107.4.76b","WVHSY","74","GASSRA","98","YGTPI","122","3.9\u2003\u00b1 0.8","0.34\u2003\u00b1 0.01"]},{"entry":{"@attributes":{"namest":"1","nameend":"9","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0224","num":"0223"},"tables":{"@attributes":{"id":"TABLE-US-00006","num":"00006"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"280pt","align":"center"}},"thead":{"row":{"entry":"TABLE V"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Binding kinetic analysis of anti-NRP1 IgGs at 25\u00b0 C. The binding affinities "},{"entry":"of anti-NRP1 IgGs are measured on BIAcore-3000 instrument using IgG-immobilized "},{"entry":"biosensor chip with dilutions of hNRP1 or mNRP1 flowing through at 25\u00b0 C. "},{"entry":"(See Materials and Methods). Each measurement has an error in Kd approximately \u00b125%."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"119pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"119pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"murine NRP1","human NRP1"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Clone","kon\/10(Ms)","koff\/10 (s)","Kd(nM)","kon\/10(Ms)","koff\/10 (s)","Kd(nM)"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}},{"entry":["YW64.3","0.6\u2002","4.5","7.8","1.3\u2002","1.2\u2002","0.9"]},{"entry":["YW107.4","1.4\u2002","15\u2002\u2009","11\u2002\u2009","2.2\u2002","11\u2003\u2002\u2009","5\u2009\u2002"]},{"entry":["YW107.4.18","1.4\u2002","2.9","2.1","1.8\u2002","1.8\u2002","1\u2009\u2002"]},{"entry":["YW107.4.38","0.84","2.7","3.2","0.94","1.7\u2002","1.8"]},{"entry":["YW107.4.52","1.3\u2002","1.6","1.2","1.4\u2002","1.3\u2002","0.9"]},{"entry":["YW107.4.63","1.6\u2002","1.3","0.8","1.8\u2002","1.5\u2002","0.8"]},{"entry":["YW107.4.76a","0.42","13\u2002\u2009","31\u2002\u2009","1.2\u2002","1.7\u2002","1.4"]},{"entry":["YW107.4.87","1.6\u2002","2\u2009","1.3","1.9\u2002","0.77","0.4"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}}]}}]}}},"The antibody sequences of YW64.3 and YW107.4.87 are shown in . For the purpose of this application and in the following descriptions of anti-NRP1 antibody activities, \u201cYW64.3\u201d and \u201canti-NRP\u201d are used interchangeably; and \u201cYW107.4.87\u201d and \u201canti-NRP\u201d are used interchangeably.","Materials and Methods","Cell Cultures","HUVEC, HUAEC, and HMVECs were purchased from Cambrex and cultured in EGM-2 medium (Cambrex). Cells and tissue cultures are maintained at 37\u00b0 C. in a 5% CO, 95% humidity incubator.","DRG and Hippocampal Collapse Assays","Collapse assays on axons from mouse E12.5 DRG were performed as described in He and Tessier-Lavigne (1997) 90:739-751. Briefly, DRG explants were plated on laminin coated 8-chamber slides (Nunc) and cultured in N3-F12 medium with 50 ng\/ml NGF overnight at 37\u00b0 C. in a 5% CO, 95% humidity incubator. Human Semaphorin-3A (Met-1 to Val-771) was cloned into the eukaryotic expression vector pRK5 with a C-terminal hexahistidine fusion tag. The protein was transiently expressed in CHO cells and purified by NiNTA affinity chromatography. N-terminal sequencing was used to confirm the protein visible as a 90 kDa band on a Coomassie stained SDS-PAGE gel. Purified Sema3A was added at \u02dc8 ng\/ml in the presence or absence of inhibitors, and explants were incubated at 37\u00b0 C. for 30 min to induce collapse. For visualization, growth cones were fixed in 4% PFA and 15% sucrose, stained with rhodamin-phalloidin (Molecular Probes) at 1:40 in PBS for 30 min, and then washed and mounted with Fluoromount G (Fisher). To perform the hippocampal collapse assay, E17 mouse brains were dissected in chilled PBS and horizontally sectioned into 250 \u03bcm thick slices using a tissue chopper (Mcllwain). The dentate gyms was further subdissected from selected sections of hippocampus using fine tungsten needles. Explants were plated on laminin coated 8-chamber slides (Nunc) and cultured in Neurobasal medium supplemented with B27 (Gibco Life Technologies) overnight at 37\u00b0 C. To induce collapse, explants were incubated with control or anti-NRP1 antibodies (10 \u03bcg\/ml) in the presence of mock transfected or Sema3F transfected COS cell conditioned media for 30 minutes at 37\u00b0 C.","Cell Migration Assay","Cell migration assays were performed using a modified Boyden chamber assay with the 8 \u03bcm pore size Falcon 24-multiwell insert system (BD Biosciences). The plates were pre-coated with 8 \u03bcg\/ml Laminin (Invitrogen) for 16 hrs at 37\u00b0 C. in both the upper and lower chambers. HUVECs (80-90% confluent) were harvested with trypsin, counted, centrifuged and resuspended in EBM-2 with 0.1% BSA at a concentration of 5\u00d710cells\/ml. 100 \u03bcl cells were added into the upper chamber of the transwell system. Inhibitors were added to the upper chamber with the cells immediately before addition of the stimuli to the lower chamber (10 ng\/ml of VEGF in most cases), which contained 500 \u03bcl EBM-2 with 0.1% BSA. The plates were then placed at 37\u00b0 C. overnight. To cease the assay, cells on the upper face of the membrane were removed using a sponge swab and cells on the lower face were fixed with methanol and stained with Cytox green (Molecular Probes). The number of migrated cells on the lower face of the membrane was counted using AxiPhot fluorescent microscope and the results were analyzed with the ImageJ program (NIH, http:\/\/rsb.info.nih.gov\/ij\/).","Bead Outgrowth Assay","Dextran-coated Cytodex 3 microcarrier beads (Amersham Pharmacia) were incubated with subconfluent HUVECs at a concentration of 400 cells per bead in 1 ml of EGM-2 medium, for 4 hrs at 37\u00b0 C. in a 5% CO2 incubator with gentle shake every 20 min. Beads were then transferred to a 25-cmtissue culture flask (BD Biosciences) and incubated in 5 ml of EGM-2 for 12-16 hrs at 37\u00b0 C. HUVEC-coated beads were then washed three times with 1 ml EGM-2 and resuspended in 2.5 m\/ml of fibrinogen (Sigma) in PBS at a density of 200 beads\/ml. 0.5 ml of fibrinogen\/bead solution was transferred into one well of a 24-well tissue culture plate containing 0.625 units thrombin (Sigma) to induce clotting. The fibrinogen\/bead solution was incubated for 5 min at room temperature, then for 20 min at 37\u00b0 C. in a 5% CO2 incubator. After the clotting was complete, lml of EGM-2 was added to each well and the clot was equilibrated for 30 min at 37\u00b0 C. The medium was removed, and lml of skin fibroblast cells (Detroit 551) at a concentration of 2000\/ml in EGM-2 medium were plated on the top of the clot. Different antibodies were then added to each well and the assay was monitored for 8 days with change in medium every 2-3 days. 10\u00d7 resolution images of the beads were captured by an inverted microscope, and concentric circles spaced at 100, 200, and 300 \u03bcm were digitally drawn around the bead in each image. The number of vessels crossing each line was counted and the average was taken at each distance and condition.","FACS Analysis","Confluent HUVECs were incubated with control or anti-NRP1 antibodies at 10 \u03bcg\/ml for 5 min, 2 hrs or 20 hrs at 37\u00b0 C. in a 5% CO2 incubator. Cells were harvested with enzyme free cell dissociation buffer (Gibco), neutralized with equal volume of FACS buffer (1\u00d7PBS, 2% FBS, 2 mM EDTA, 0.1% sodium azide) and transferred into Pro-bind U-bottom 96-well assay plates (Falcon) at 500,000 cells per well. Cell pellets were collected by centrifuged at 2 k rpm, and resuspended in 25 \u03bcl staining buffer (FACS buffer with 5% normal mouse serum, 2% normal rat serum and 10% 10 \u03bcg\/ml human IgG) containing appropriate biotinylated antibody at 1:100, followed by a 2 hr incubation at 4\u00b0 C. Antibodies were biotinylated using the FluoReporter mini-biotin-xx protein labeling kit (Molecular Probes). Cells were then washed twice with FACS buffer, and incubated in 25 \u03bcl FACS buffer containing 1:1000 streptavidin-PE (BD Biosciences) for 20 min at 4\u00b0 C. Finally, cells were washed and resuspended in FACS buffer, and analyzed with the FacsCalibur system (BD Biosciences).","Cell Adhesion Assay","Subconfluent HUVECs were pre-incubated in 100 \u03bcl Medium 199 with control or anti-NRP1 antibodies for 30 min at 37\u00b0 C., then plated on NUNC maxisorp flat bottom 96-well plates (eBioscience) coated with 1 \u03bcg\/ml Fibronectin (Roche) at 10,000 cells per well. Plates were centrifuged for 1 min at 140 g to synchronize contract of cells with substrate, and incubated at 37\u00b0 C. for 30 min. Plates were then washed 3 times with PBS, frozen at \u221280\u00b0 C. for two hours. Cell density was determined with the CyQuant kit (Molecular Probes).","Mouse Neonatal Retinal Vascular Assay","Neonatal CD1 mice (same litters) were administered via i.p. with different antibodies at a concentration of 10 mg\/kg. The injections were performed on postnatal day 1 and 3 for postnatal day 5 (P5) studies, and on day 1, 3 and 5 for P8 studies. Eyes were collected and fixed with 4% PFA in PBS for 16 hrs at 4\u00b0 C., followed by PBS washes. The dissected retinas were block with 10% mouse serum in PBSt (PBS, 1% Triton X-100) for 3 hrs, then incubated overnight at 4\u00b0 C. with biotinylated isolectin B4 (Bandeiraea simplicifolia; Sigma) 25 \u03bcg\/ml in PBLEC (1% Triton X-100, 0.1 mM CaCl, 0.1 mM MgCl, 0.1 mM MnCl, in PBS pH 6.8). Retinas were then washed 4 times in PBSt, and incubated with Alexa 488 streptavidin (1:200; Molecular Probes) overnight at 4\u00b0 C. After staining was completed, retinas were washed 4 times in PBSt, post fixed with 4% PFA in PBS, followed by another 4 washes in PBS. Images of flat mounted retinas were captured by confocal fluorescence microscopy.","Mouse Skin Vessel Permeability Assay","5-7 week C57BL6J female mice were injected with 150 \u03bcl 0.5% Evan's blue solution i.v. The areas of the back and flanks of the animals were shaved to remove hair below the neck and above the hip, and the shaved area was divided into 4 injection zones. One hour after the Evan's blue injection, 20 \u03bcl of PBS containing BSA (7.5 \u03bcg\/ml) or hVEGF (7.5 \u03bcg\/ml) with or without antibody (0.5 mg\/ml) were injected i.d. randomly on any one of four zones, one injection for each zone. One hour after the i.d. injection, the animals were sacrificed and the skin containing the 4 zones was dissected out. A digital image of the skin was taken. A 8 mm skin biopsy punch was used to cut out skin samples of same area on the injection sites, and skin samples were incubated in formamide solution at 55\u00b0 C. for 48 hrs to extract the Evan's blue dye from the tissues. The absorbance of the solution was then measured with a spectrometer at 605 nm","Cell Proliferation Assay","Cell proliferation was analyzed with the Cell Proliferation ELISA kit (Roche). Briefly, 90% confluent HUVECs were harvested and resuspended in assay media (DMEM:F12 50:50, 1.5% FBS), and plated at 3,000 cells per well into black 96-well tissue culture plate (ViewPlate-96, Perkin Elmer) pre-coated with 1% gelatin. Cells were incubated at 37\u00b0 C. for 16 hrs, changed into fresh assay media and incubated for another 8 hrs. VEGF (20 ng\/ml) and different antibodies were added to the culture and the cells were incubated for 20 hrs. BrdU labeling solution was added to a final concentration of 10 mM and the cells were incubated for another 24 hrs at 37\u00b0 C. BrdU incorporation was determined by chemiluminescence immunoassay.","VEGFR2 Phosphorylation Assay","Total VEGFR2 and phospho-VEGFR2 were determined with DuoSet IC ELISA kits (R&D). Briefly, subconfluent HUVECs were rinsed with PBS and lysed in the provided lysis buffer. Cell lysates were centrifuged at 14 kg for 5 min at 4\u00b0 C., and supernatants were transferred to clean tubes. ELISA assays were then carried out as directed.","Immunoblotting","Confluent HUVECs were starved for 16 hrs in EBM-2 with 0.2% FBS and 0.1% BSA, and then changed into EMB-2 with 0.1% BSA, followed by a 90 min incubation at 37\u00b0 C. Control and anti-NRP1 antibodies were added at 50 \u03bcg\/ml, and cells were incubated for 30 min at 37\u00b0 C. Cells were stimulated with VEGF (20 ng\/ml) for 10 min at 37\u00b0 C., washed twice with ice-cold PBS and lysed in lysis buffer containing 20 mM Tris 7.5, 150 mM NaCl, 1% Triton X-100, phosphatase inhibitor cocktail I and II (Sigma) and complete protease inhibitor tablet (Roche). Aliquots of lysate were subjected to electrophoresis in 4-20% Novex Tris-Glycine SDS gels (Invitrogen), followed by electrotransfer to Invitrolon PVDF membrane (Invitrogen). The primary antibodies used in this study were anti-Erk, anti-phospho-Erk, anti-Akt, anti-phospho-Akt, anti-p38, anti-phospho-p38, anti-Src and anti-phospho-Src (Tyr416), all purchased from Cell Signaling Technology.","Results","A. Selective Actions of Anti-NRP1and Anti-NRP1on Sema3A Function and VEGF Binding, Respectively","Anti-NRP1 antibodies were tested for their ability to block binding of VEGFto NRP1. Anti-NRP1strongly blocked VEGF binding to NRP1, whereas anti-NRP1did not (). These results suggests that the CUB domains (a1-a2) of NRP1 are not necessary for VEGF binding. Gu et al. (2002) 277:18069-18076.","Next, anti-NRP1 antibodies were tested for their effects on Sema3A function. It has been previously suggested that Sema3A and VEGFmay share overlapping binding domains in the N-terminal region of the b1 domain, and thus may compete for binding to NRP1. Gu et al. (2002) 277:18069-18076; Miao et al. (1999) 146:233-242. Therefore, it seemed possible that a single anti-NRP1 mAb would block both Sema3A and VEGF binding. Anti-NRP1 mAbs of this invention were tested for their abilities to block Sema3A-induced axon growth cone collapse. Dorsal root ganglia (DRG) were dissected from mouse E12.5 embryos and cultured to establish sensory neuron growth cones that were responsive to Sema3A\/NRP1-dependent collapse. He and Tessier-Lavigne (1997) 90:739-751. Adding Sema3A to these cultures caused growth cones to retract their actin rich structures. However, if anti-NRP1 was added to the wells at the same time as Sema3A, collapse was completely blocked. In contrast, anti-NRP1 had no effect on Sema3A-induced collapse ().","These two antibodies thus have functionally distinct actions: anti-NRP1 blocks Sema3A function but does not interfere with VEGF binding to NRP1; whereas anti-NRP1 blocks VEGF binding to NRP1 without effect on Sema3A function. Further studies showed that neither antibody blocks Sema3F\/NRP2-dependent collapse of E17 hippocampal growth cones (), consistent with the observation that neither antibody binds NRP2. The antibodies thus provide selective tools to dissect the role of NRP1 in vascular biology, and means for selectively blocking NRP1 activities.","B. Both Anti-NRP1and Anti-NRP1Reduce VEGF-Dependent Endothelial Cell Migration and Bead Outgrowth","The anti-NRP1 mAbs of the invention were tested in VEGF-driven endothelial cell (EC) migration. Using a transwell system, human umbilical vein endothelial cells (HUVECs) were introduced into the top chamber, while VEGF was added to the bottom chamber to promote EC migration. ECs that had migrated to the bottom chamber were then fixed, stained, and quantified (). Anti-VEGF antibodies were used as a positive control to block VEGF-driven EC migration in this and subsequent experiments (a cross-species reactive anti-VEGF antibody, B20.4.1, was used in all experiments unless stated otherwise; Liang et al. (2006) 281:951-961.","Anti-NRP1 mAbs were added to cells in the top chamber just prior to the addition of VEGF. Interestingly, both anti-NRP1and anti-NRP1significantly reduced endothelial cell migration, with anti-NRP1providing a stronger block of migration (). Similar results were obtained using two other types of EC lines, HUAECs and HAECs.","To further dissect the role of NRP1 and anti-NRP1 mAbs in more complex EC functions, an in vitro system of angiogenic sprouting was employed. Nakatsu et al. (2003) 66:102-112. In this assay, ECs coated on beads sprout over a seven days period, resulting in multiple well-defined vessel structures protruding from each bead. Adding either anti-NRP1 mAbs to these cultures resulted in a reduction in vessel length, and in the case of anti-NRP1a decrease in the number of sprouts was also observed. This observation was confirmed by quantification (). Anti-VEGF was used as a positive control and completely blocked sprouting in this assay. The results confirmed strong effects for each anti-NRP1 mAb in reducing EC migration and outgrowth of vessels in this in vitro assay.","C. NRP1 is Necessary for Vascular Remodeling in the Mouse Retina","Phenotypes observed in the Nrp1 knock-out mice are consistent with a defect in vascular remodeling. Gu et al. (2003) 5:45-57; Kawasaki et al. (1999) 126:895-4902; Takashima et al. (2002) 99:3657-3662. Next, the role of NRP1 in vascular remodeling was tested by analyzing the effects of systemic treatment with the anti-NRP1 mAbs on the developing mouse retina, which is an ideal model to investigate the stereotyped events of vascular sprouting, remodeling, and maturation. Dorrell and Friedlander (2006) 25:277-95. At birth, an astrocytic network is in place to guide ECs sprouting from central retinal artery located near the optic nerve head (ONH). After one day of postnatal life (P1) the retinal vasculature has developed a morphogenic furrow in the nerve fiber layer (NFL), lying superficial to the ganglion cell layer, that begins to extend toward the edge of the retina in a concentric pattern, reaching halfway by P5 (). Over the next three days the furrow continues to extend to the edge of the retina, while the vascular plexus closest to the ONH undergoes stereotyped remodeling, consisting of thinning of the vascular plexus into a refined capillary network in the NFL (), and sprouting of vessels into deeper layers of the retina ().","Antibodies were injected into neonatal mice beginning at P1, followed by injections every other day until retinas were collected at P5 and P8. To visualize the vasculature, retinas were stained with isolectin B4 and compared. While the refined capillary network of the IgG control group is well established between P5 and P8, this vascular remodeling is completely inhibited by both anti-NRP1 mAbs. This difference was quantified by comparing the vascular density from representative images taken of the NFL vascular network in regions adjacent to the ONH (). Because treatment with either anti-NRP1 mAb results in a strong inhibition of vascular remodeling, vascular density is significantly higher in both anti-NRP1 and anti-NRP1 treated retinas as compared to controls. Interestingly, the developmental furrow continues to extend in both anti-NRP1 mAb treated retinas, with only a slight inhibition of extension in the anti-NRP1 treated animals (). This suggests that development of the retina is not generally inhibited, but that there is instead a specific block in vascular remodeling when treating with either anti-NRP1 mAb.","In contrast to the anti-NRP1 mAbs, anti-VEGF treatment resulted in a reduction of retinal vascular density as compared to control (). At a qualitative level, vessels in anti-VEGF treated retinas have a reduced complexity at P8, a trend that is present but less obvious at P5. These data suggest that anti-VEGF treatment results in either block of initial sprouting and\/or vascular regression. Interestingly, systemic delivery of anti-VEGF does not significantly reduce the extension of the vascular furrow (). This is likely a consequence of poor antibody diffusion to the extending tip cells due to the lack of vessel lumens in this outer region. Gerhardt et al. (2003) 161:1163-1177.","Evaluating P8 retinas also allowed further investigation of the effects of antibody treatment on angiogenic sprouting. Between P5 and P8, vessels begin to sprout from the NFL vascular network into deeper vascular layers resulting in the formation of the outer plexiform layer (OPL) vascular network, which is superficial to the outer nuclear layer and is the deepest vascular bed in the retina (). Later in development, sprouting from collaterals that gave rise to the OPL results in an intermediate vascular bead, termed the inner plexiform layer (IPL). Images taken from the OPL at P8 show a complete inhibition of sprouting by both anti-NRP1 mAbs, and by anti-VEGF treatments.","The different phenotypes observed in retinas taken from animals systemically treated with either anti-NRP1 or anti-NRP1 , as compared to anti-VEGF, suggest that NRP1 may regulate EC function by mechanisms other than enhancing VEGFR2 signaling.","D. Blocking NRP1 Function has Little Effect on VEGFR2 Signaling","Having observed that NRP1 is required for endothelial cell migration in response to VEGF, we investigated the requirement of NRP1 in EC proliferation and vascular permeability\u2014two defining cellular activities induced by VEGF. Remarkably, treatment with either anti-NRP1 or anti-NRP1had no effect on VEGF-induced permeability, whereas anti-VEGF provided a strong block (). A similar trend was observed when testing for VEGF-induced EC proliferation, with anti-NRP1 showing no block of proliferation, and anti-NRP1only a slight dose-responsive reduction (). These results support previously published data showing that siRNA knockdown of NRP1 in ECs does not effect proliferation induced by VEGF (Murga et al. (2005) 105:1992-1999), and suggests that NRP1's primary role in VEGF-driven EC behaviors is to mediate cell migration.","The effect of anti-NRP1 antibodies on VEGFR2 signaling was studied. VEGF binds to the second and third extracellular IgG domains of VEGFR2, and activates the receptor by triggering auto-phosphorylation of several tyrosine residues in the intracellular domain. In contrast to anti-VEGF, which completely blocked VEGFR2 phosphorylation induced by VEGF in HUVECs, anti-NRP1 did not significantly change VEGFR2 phosphorylation levels, whereas anti-NRP1resulted in only a modest reduction ().","Rather than regulating VEGFR2 phosphorylation level directly, NRP1 may act to modulate specific VEGFR2 pathways. To address this possibility, the effect of anti-NRP1 treatment on downstream signaling events mediated by VEGFR2 was examined. VEGFR2 has been shown to induce EC proliferation through activation of the mitogen activated protein kinases Erk1\/2 (Rousseau et al. (1997) 15:2169-2177; Takahashi et al. Shibuya (1999) 18:2221-2230), and to regulate EC survival and vascular permeability through the PI3-Kinase\/Akt pathway. Chen et al. (2005) 11:1188-1196; Gerber et al. (1998) 273:13313-13316; Six et al. (2002) 532:67-69. Consistent with the observation that anti-NRP1 mAb treatments did not significantly change VEGF-induced EC proliferation or vascular permeability, anti-NRP1 and anti-NRP1incubation did not affect VEGF-induced phosphorylation of Erk1\/2 or Akt. On the other hand, inhibiting NRP1 function with anti-NRP1 antibodies strongly reduced EC migration. It is possible that NRP1 specifically regulates VEGFR2 pathways required for cell motility, such as the p38 MAP kinase pathway, which has been shown to be required for VEGF-driven actin reorganization and cell migration in ECs. Rousseau et al. (1997) 15:2169-2177. Anti-NRP1 and anti-NRP1treatments both led to a slight reduction in p38 phosphorylation level in HUVECs (), lending support to this hypothesis. However, the mild decrease in p38 phosphorylation alone is unlikely to account for the strong reduction we observed in the migration and sprouting assays as described in previous sections (), or explain the qualitatively different phenotypes observed between the anti-NRP1 mAbs and anti-VEGF mAb treatments in the retinal vascular remodeling experiments ().","Materials and Methods","The human cell lines, H1299 and SK-MES-1 non-small-cell lung carcinomas were obtained from ATCC (Rockville, Md.). For SK-MES-1, each HRLN female nude mouse received a 1 mmtumor fragment s.c. implant in the flank. For H1299, 1\u00d710tumor cells were injected s.c. in the flank of HRLN female nude mice. Tumor growth was monitored twice weekly by caliper measurements. When tumors reach an average size of 80-120 mm, mice were sorted to give nearly identical group mean tumor sizes, and treatment was started. This was considered day 1 of each study. All treatments were body-weight adjusted at 0.2 ml\/20 g. Tumor volume (mm)=width\u00d7length\/2. Percent tumor growth inhibition (% TGI)=(median tumor volume of the control arm\u2212median tumor volume of the treatment arm\/median tumor volume of the control arm)\u00d7100.% TGI is only measured as long as all animals remain on they study (for SK-MES-1=day 22; for H1299=day 15). Time to endpoint (TTE)=log(endpoint volume, mm)\u2212b\/m; were b is the intercept and m is the slope of the line obtained by linear regression of the log-transformed tumor growth data. Percent tumor growth delay (% TGD)=(median TTE for a treatment arm\u2212median TTE for the control arm\/median TTE for the control arm)\u00d7100.","Mice were anesthetized with Avertin (1.3% tribromoethanol and 0.8% amyl alcohol; Sigma-Aldrich). FITC-labeled Lycopersicon esculentum lectin (150 \u03bcg in 150 \u03bcl of 0.9% NaCl; Vector Laboratories) was injected i.v. 3 minutes before systemic perfusion. The vasculature was perfused transcardially with 4% paraformaldehyde (PFA) in PBS for 2 to 3 min. Tumors were removed and post-fixed by immersion in the same fixative for 16 hrs, followed by an incubation in 30% sucrose overnight for cryoprotection, then embedded in OCT and frozen. Sections (30 \u03bcm) were cut and mounted onto glass slides, rehydrated and blocked in PBSt (PBS, 0.5% Triton X-100) with 5% normal goat serum for 2 hrs at room temperature. The sections were incubated overnight with anti-NRP1 (1:500) or rat anti-mouse PDGFR\u03b2 (1:1000; clone APBS, eBioscience) at 4\u00b0 C. Sections were then washed 4 times in PBSt, and incubated with secondary antibodies, Alexa 488 goat anti-human IgG and Alexa 568 goat anti-rat IgG (1:200; Molecular Probes), for 4 hrs at room temperature. After staining was completed, sections were washed 4 times in PBSt, post fixed with 4% PFA in PBS, followed by another 4 washes in PBS. The final wash solution was removed, and 1-2 drops of Fluoromount-G (SouthernBiotech) were added. Glass coverslips were placed over samples, and images were captured with a Zeiss Axiophot fluorescence microscope.","Results","Blocking the VEGF pathway has been proven to reduce neovascularization in mouse tumor models and in human cancers. Ferrara and Kerbel (2005) 438:967-974. However, it is believed that some tumors are less dependent on VEGF for vessel formation, or may become insensitive to anti-VEGF therapies. Jain et al. (2006) 3:24-40; Kerbel et al. (2001) 20:79-86. To determine if NRP1 blockade could enhance the tumor growth inhibition provided by blocking VEGF, we selected several xenograft models known to exhibit varying sensitivity to anti-VEGF therapy. Since mouse stromal VEGF, in addition to tumor derived VEGF, has been shown to impact tumor growth, we used anti-VEGF mAbs that recognize both murine and human VEGF. Liang et al. (2006) 281:951-961.","These experiments were designed to test the effect of blocking NRP1 alone and in combination with anti-VEGF. Also included were single agent anti-VEGF and isotype control antibody. SK-MES-1 is a NSCLC xenograft model that expresses NRP1 primarily in vascular and stromal tissue, with an intermediate level of expression in tumor cells. In this model, anti-VEGF provided a 52% tumor growth inhibition (TGI), single agent anti-NRP1caused a 37% TGI, and anti-NRP1 had no significant effect on TGI ().","Strikingly, the additive effect was observed when either anti-NRP1 antibody was used in combination with anti-VEGF. As depicted in , anti-NRP1 used in combination with anti-VEGF increased the TGI from about 52% to about 70%; and anti-NRP1used in combination with anti-VEGF increased the TGI to about 77% (). Similar results were obtained in the H1299 NSCLC xenograft model, which also expresses NRP1 in vascular and stromal tissue at high levels, but in tumor cells to a lesser extent. Single agent anti-NRP1showed 39% TGI, anti-VEGF showed 28% TGI, and the combination showed 51% TGI ().","Animals in the SK-MES-1 model were dosed to day 35 and followed to day 60 to examine the delay in tumor growth (animals were removed from the study when tumor sizes exceeded 1500 mm; no animals were removed as a result of toxicity). The Kaplan-Meier plot shows a significant effect of both anti-NRP1 mAb combination arms in delaying tumor growth as compared to single agent arms (). Measurements of tumor growth delay (TGD) show no single agent delay for anti-NRP1 ; 24% TGD for anti-NRP1 ; 60% TGD for anti-VEGF; 93% TGD for anti-NRP1 +anti-VEGF combination arm; and 96% for anti-NRP1+anti-VEGF combination arm.","Similar results to those seen in the NSCLC models were also observed in a murine model of breast cancer, termed founder 5 (Fo5), in which tumors generated by MMTV-Her2 transgenic expression were dissected and implanted in nude mice. Finkle et al. (2004) 10:2499-2511. In this model, NRP1 is primarily expressed on vessels and adjacent stroma, and anti-VEGF is again only partially effective at reducing tumor growth (). Growth inhibition was similar to that seen in the NSCLC models, however to focus on the histological changes in the context of anti-NRP1 and anti-VEGF treatment, animals were harvested on the same day early in the experiment (dashed line; ).","Using the Fo5 model for vascular histology studies, we observed that control tumors showed very large unorganized vessels that generally lack pericyte coverage as determined by PDGFR-\u03b2 immunohistochemistry. Treatment with anti-VEGF alone resulted in a decrease in vessel diameter and a general reduction in vascular density. In addition, most, if not all vessels that remained in anti-VEGF treated tumors showed close association of pericytes. Similar results were observed with other pericyte markers.","Vessels from tumors treated with either anti-NRP1 mAb alone appear similar in morphology to anti-NRP1 treated vessels in neonatal mouse retinas, with vessels appearing flat and unorganized, maintaining a plexus-like appearance. On the other hand, similar to control treated tumor vessels, tumors treated with either anti-NRP1 mAb also lack significant pericyte coverage. Most interestingly, tumors that were treated with anti-VEGF and either anti-NRP1 or anti-NRP1 , showed a combined phenotype, with vessel reduction similar to anti-VEGF treated tumors, but with remaining vessels appearing less refined and lacking close pericyte association.","These morphological observations are correlated with the quantified changes in vascular desensity. As shown in , treatment with both anti-NRP1 antibodies and anti-VEGF decreased vascular density (10A). On the other hand, only anti-VEGF seems to increase perictye staining in tumor samples tested, as compared to control. Overall, pericyte\/vascular ratio measuring relative pericyte coverage shows that anti-VEGF treated tumors had the highest ratio, which is decreased by anti-NRP1and anti-NRP1 treatment.","The findings that anti-NRP1 (anti-NRP1 and anti-NRP1) and anti-VEGF may be acting by blocking different EC functions and\/or signaling pathways raised the possibility that combining these antibodies in tumor models could have an additive or synergistic effect in reducing tumor growth. Indeed, a profound additive effect was observed when combining anti-NRP1 with anti-VEGF (). Especially in the case of anti-NRP 1 , the results suggest a synergistic, rather than additive effect, since in multiple tumor models the mean tumor volume following single agent anti-NRP1treatment was not different from control, but the antibody significantly enhanced tumor response when combined with anti-VEGF, compared to anti-VEGF treatment alone.","In the case of anti-NRP1, single agent treatment resulted in a significant reduction in tumor growth. Interestingly, these effects are only slightly less potent than anti-VEGF alone. However, the reduction in tumor growth from the combination arms for anti-NRP1 was similar to the anti-NRP1 combination arms. The single agent effect of anti-NRP1may be due to its relatively stronger inhibitory activity in reducing sprouting and in vivo angiogenesis, compared to anti-NRP1. On the other hand, both antibodies were equally potent in blocking vascular remodeling in the mouse retinal development assay and inhibiting pericyte association with vessels in the Fo5 tumor model.","It has been proposed that vascular remodeling in the retina takes place in the absence of close pericyte associations, suggesting that pericytes serve to stabilize immature blood vessels ending the plasticity period of vessel remodeling. Benjamin et al. (1998) 125:1591-1598. Findings of this invention suggest that NRP1 is required for this intricate process of vessel remodeling, consisting of the morphogenesis of the vascular plexus into fine capillaries, followed by further maturation.","The idea of pericytes stabilizing tumor vessels, thus giving rise to anti-VEGF therapy resistance as a consequence of vessels losing their VEGF-dependence, has been proposed and tested experimentally. Bergers et al. (2003) 111:1287-1295; Erber et al. (2004) 18:338-340. In these studies, it was observed that blocking both the function of VEGF (EC ligand) and PDGF (pericyte ligand) results in further disruption of tumor vasculature. Subsequently, it has been shown in imaging studies that blocking VEGF function alone in tumors results in a significant amount of vascular regression, but the remaining vessels become \u201cnormalized\u201d with close pericyte associations, possibly a consequence of immature vessels undergoing vascular remodeling and maturation. Inai et al. (2004) 165:35-52.","Based on the observations obtained in the developing retina, it can be postulated that blocking NRP1 function in tumors would also reduce vascular remodeling of tumor vessels, subsequently inhibiting vessel maturation, therefore keeping vessels in a state of VEGF-dependence. This alone may not have a significant effect on reducing tumor growth, as most tumor vessels are already inherently disorganized. Baluk et al. (2005) 15:102-111. This prediction is consistent with the observation that, although anti-NRP1 strongly inhibits vascular remodeling, we do not see a single agent effect with this antibody in reducing tumor growth. However, it was speculated that blocking vascular remodeling, and the subsequent maturation, in combination with anti-VEGF therapy may render the remaining tumor vessels more susceptible to regression. The experiments of the present invention showed that vessels remaining in tumors treated with either anti-NRP1 mAb in combination with anti-VEGF indeed lack pericyte association. These results suggest that the additive effect we observe in blocking both NRP1 and VEGF function may arise as a consequence of rendering remaining vessels in anti-VEGF treated tumors more vulnerable to regression.","Based on the results obtained in developing retina and xenograft tumor models, it is postulated that blocking NRP1 function in newly formed vessels inhibits vessels from undergoing remodeling and subsequent maturation, thus rendering these vessels more dependent on VEGF for survival (see the model in ). Consistent with this model, we observed a significant decrease in vascular density of the developing retina in neonate mice treated with both anti-NRP1 and anti-VEGF at half the dose of single agent alone (single dose=10 mg\/kg, combination dose=5 mg\/kg of each antibody). See , as compared to .","Moreover, it is possible that NRP1 may also be required not only for endothelial but also pericyte function. It has recently been reported that bFGF induces upregulation of NRP1 on smooth muscle cells, resulting in the ability of VEGF to induce smooth muscle cell migration. Liu et al. (2005) 32:206-212. In the present experiment, expression of NRP1 was seen on human smooth muscle cells, as well as on pericytes in various murine and human tumors (for example, expression of NRP1 extends to areas adjacent to lectin positive stain in the Fo5 model).","In conclusion, the experiments described herein demonstrate that blocking NRP1 and VEGF function in tumor models results in an additive effect in reducing tumor growth. Also presented are evidences that NRP1 may also be acting through mechanisms other than VEGFR2 signaling."],"BRFSUM":[{},{}],"heading":["FIELD OF THE INVENTION","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","MODES FOR CARRYING OUT THE INVENTION","EXAMPLES","Example 1","Generation of Anti-NRP1 Antibodies","Example 2","Biological Activities of Anti-NRP1 Antibodies","Example 3","Tumor Inhibitory Activities of Anti-NRP1 Antibodies"],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0030","num":"0029"},"figref":"FIG. 1","b":["1","1"]},{"@attributes":{"id":"p-0031","num":"0030"},"figref":"FIG. 2","sup":"B","b":["2","2"]},{"@attributes":{"id":"p-0032","num":"0031"},"figref":"FIG. 3","sup":["A","B"],"b":"3"},{"@attributes":{"id":"p-0033","num":"0032"},"figref":"FIG. 4","sub":"165 "},{"@attributes":{"id":"p-0034","num":"0033"},"figref":"FIG. 5","b":"5","sup":"A "},{"@attributes":{"id":"p-0035","num":"0034"},"figref":"FIG. 6","b":"6","sup":"\u221211"},{"@attributes":{"id":"p-0036","num":"0035"},"figref":"FIG. 7","b":["7","7","7","7","7"]},{"@attributes":{"id":"p-0037","num":"0036"},"figref":"FIG. 8","b":["8","8","8","8"],"sup":["A","B","A "]},{"@attributes":{"id":"p-0038","num":"0037"},"figref":"FIG. 9","b":["9","9"]},{"@attributes":{"id":"p-0039","num":"0038"},"figref":"FIG. 10","b":["10","10","10"]},{"@attributes":{"id":"p-0040","num":"0039"},"figref":"FIG. 11","b":["11","11"]}]},"DETDESC":[{},{}]}
