---
title: Method for enriching a prokaryotic DNA
abstract: A method is described for enriching procaryotic DNA, said method including the steps of contacting at least one procaryotic DNA with at least one protein or polypeptide which is capable of specifically binding to non-methylated CpG motifs, and separating the protein/polypeptide-DNA complex. Moreover, the application relates to a kit for carrying out said method.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08288115&OS=08288115&RS=08288115
owner: SIRS-Lab GmbH
number: 08288115
owner_city: Jena
owner_country: DE
publication_date: 20111005
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["RELATED APPLICATIONS","TECHNICAL FIELD OF THE INVENTION","BACKGROUND OF THE INVENTION","SUMMARY AND DETAILED DESCRIPTION","EXAMPLE 1","Prior Art Method of Detection","EXAMPLE 2","Carrying Out the Method According to the Invention"],"p":["This application is a division of U.S. application Ser. No. 10\/528,235, now U.S. Pat. No. 8,062,854, filed Dec. 11, 2007, which application is a 371 of international application no. PCT\/EP2003\/008825 filed Aug. 8, 2003, which application claims the benefit of EP application no. 02020904.5 filed Sep. 18, 2002. All applications are hereby fully incorporated by reference.","The invention relates to a method of enriching procaryotic DNA as well as to a kit for carrying out said method.","Infections caused by bacteria are one of the most frequent causes of inflammatory diseases. For the prognosis of the clinical course as well as, in particular, for timely selection of suitable therapeutic measures, early detection of the bacterial pathogens is of decisive importance.","In the detection of bacterial pathogens, use is made, above all, of different methods of cultivating cells. However, methods of molecular biology which are based on the detection of pathogen-specific nucleic acids have also become more important recently. In addition to the high specificity of these methods, mention must be made of the little time required as an essential advantage over conventional methods. However, the sensitivity of the detection of procaryotic DNA directly from body fluids and from test material that has not been pre-treated has hitherto been much too low as compared to the culture of microorganisms. An amount of nucleic acids of bacteria sufficient to detect pathogens directly from the test material that has not been pre-treated is achieved, if at all, in the region of the 16S-mRNA molecules. However, this requires that the bacteria to be detected be present in the metabolic phases and express sufficient 16S-mRNA.","This is usually not the case, in particular in patients who are subject to antibiotic therapy. Moreover, certain pathogenicity factors of bacteria are not expressed every time, although the corresponding genes are present in the bacterial genome. Therefore, the detection of the pathogenicity factors and resistance of bacteria at the chromosomal level is indispensable for the diagnosis of septic disease states.","This applies even more because, at this level, a distinction can also be made between pathogenic and commensal bacteria.","Most frequently, the detection of pathogen-specific nucleic acids is effected by amplification of the procaryotic DNA by means of the polymerase chain reaction (PCR) or the ligase chain reaction (LCR), respectively. The high specificity and fast availability of the results is contrasted by the susceptibility to interference or by strongly inhibiting factors of clinical samples.","In a conventional PCR detection method, successful detection of pathogens in the blood requires isolation of total DNA from at least 1 to 5 ml of blood. However, the total DNA concentration is then too high to be employed directly in a PCR reaction.","Things are different with regard to the blood culture for detection of sepsis pathogens. In this case, the lower detection limit is less than 10 bacteria per ml. This detection limit is presently achieved only by PCR protocols whose target sequence is in the 16S-RNA region and which are therefore dependent on the expression of said target sequence. Greater diagnostic reliability can be expected of PCR protocols which have their target sequences in the chromosome of the microorganisms. The expression behavior of different genes can be considerably changed or limited, especially under the influence of an ongoing antibiotic therapy, even if the antibiotic used is ultimately not effective. This situation is often found particularly in intensive therapy wards, where most patients receive antibiotic treatment, thus not allowing to grow any relevant bacteria from blood cultures or other samples for this reason.","Due to insufficient sensitivity, the detection of pathogen-specific nucleic acids, without an amplification step by direct detection of procaryotic DNA (probe technique, FISH technique), is of diagnostic importance only at a sufficiently high germ number in the test material.","The essential problems of the detection of procaryotic DNA for identification of bacterial pathogens in body fluids consist, beside PCR-inhibiting ingredients in the test material, mainly in the excess of eucaryotic DNA versus procaryotic DNA. In this connection, competitive processes in DNA analysis as well as the low quantity of procaryotic DNA can be regarded as a hindrance to a qualitative and quantitative detection of pathogens.","The usual methods of DNA isolation enrich the total DNA of a body fluid so that the ratio of host DNA to microbial DNA may be between 1:10and 1:10. This difference makes the difficulty in detecting microbial DNA in body fluids quite clear.","Therefore, it is an object of the present invention to provide a method of isolating and\/or enriching microbial DNA, in test samples having a high content of eucaryotic DNA from patients with infections, for quick and easy detection of pathogens, said detection enabling early diagnosis of infections caused by bacterial pathogens.","According to the invention, this object is achieved by a method of enriching procaryotic DNA, comprising the steps of\n\n","In this case, the term procaryotic DNA relates to both viral and bacterial DNA. Said DNA may be purified and dissolved again or may be present directly in the original source (e.g. body fluid, such as blood, serum, etc.).","Separation may be effected by means of different methods of isolating or enriching DNA protein complexes or DNA polypeptide complexes that are well-known to the person skilled in the art. In doing so, use will be made preferably of methods in which the DNA-binding protein is immobilized to a carrier matrix in order to enrich the DNA from the sample solution.","According to a preferred embodiment, the separation is followed by a step of separating the DNA and the protein\/polypeptide. This may be effected, for example, by conventional methods of DNA purification which are known to the person skilled in the art. In the most simple case, the separation is based on the change in pH value or in the salt concentration (e.g. to 1 M NaCl) of the medium\/buffer or on the addition of chaotropic reagents, etc.; i.e. suitable parameters which lead to the separation of the protein-DNA-complex. Such methods are known to the person skilled in the art.","According to a further preferred embodiment, the protein or the polypeptide is coupled to a carrier. This embodiment represents a particularly simple way of enriching procaryotic DNA, because the separation from the solution is particularly easy, for example by means of physical removal (e.g. by centrifugation) of the charged carrier(s) from the solution.","As the solution of the procaryotic DNA, any suitable solvent is basically suitable. However, the method is particularly useful for enriching procaryotic DNA from solutions which contain different biomolecular species, in particular different types of DNA. The invention preferably relates to a method of separating and enriching procaryotic or viral DNA and eucaryotic DNA from a mixture of procaryotic or viral DNA. In doing so, for example, the procaryotic DNA which is present in body fluids is separated from the eucaryotic DNA, by specific binding to the protein or to the polypeptide, and enriched. The procaryotic DNA enriched in this way facilitates detection of procaryotic pathogens with the help of molecular biology methods and can contribute to the diagnosis of diseases caused by pathogenic pathogens.","In particular, the embodiment according to which the DNA-binding protein or polypeptide is immobilized to the surface of a carrier is suitable for adsorption of procaryotic DNA from body fluids, preferably from blood. Moreover, this approach allows removal of microbial DNA, which is present in blood or other body fluids, from said fluids. The body fluid (e.g. whole blood, serum or liquor) purified in this way from the microbial DNA, which is also capable in itself of initiating severe inflammatory reactions in patients, can then be fed back into the body.","Body fluids in the sense of the invention are understood to be all fluids originating from the body of a mammal, including humans, in which disease pathogens may occur, such as blood, urine, liquor, pleural, pericardial, peritoneal as well as synovial fluids. The description of the invention referring to human blood is not to be construed as limitative, but only as an exemplary application.","Proteins or polypeptides in the sense of the invention are understood to be all eucaryotic and procaryotic proteins which are capable of specifically binding procaryotic DNA. Proteins or polypeptides which are capable of specifically binding non-methylated CpG-motifs are particularly suitable for this purpose.","Bacterial pathogens are preferably understood to be pathogens of sepsis, but also any other bacterial pathogens of infections. They may differ from commensal pathogens, which are sometimes also found in test samples from patients, but do not have any pathogenic significance.","In isolating the total DNA from infected body liquids, the ratio of host-DNA to pathogen-DNA may be, in many cases, 1:10to 1:10and less. Through the specific binding of procaryotic DNA to the protein or polypeptide having such selective properties, the method according to the invention enables enrichment by 3 exponential units and more.","The protein or the polypeptide may be coupled directly or indirectly to the carrier. The type of coupling depends on the carrier and the carrier material. Suitable carriers include, in particular, membranes, microparticles and resins, or similar materials for affinity matrices. Suitable materials for binding of the protein or of the polypeptide, as well as\u2014depending on the type of material\u2014for carrying out such binding, are well-known to the person skilled in the art. For indirect coupling, such specific antibodies against the protein or polypeptide are suitable, for example, which are in turn bound to the carrier by known methods.","One application of the method according to the invention consists in enriching procaryotic DNA. A further application consists in the separation of procaryotic DNA from a mixture of eucaryotic and procaryotic DNA by binding of the procaryotic DNA to a specific protein or polypeptide which has been immobilized to a matrix. The mixture of the body's own DNA and procaryotic DNA is contacted with the affinity matrix by means of suitable methods and, in doing so, the procaryotic DNA is bound to the immobilized protein; the eucaryotic DNA passes, for example, through a separating column and may be collected separately. Affinity matrices may be, for example, polymeric polysaccharides, such as agaroses, other biopolymers, synthetic polymers, or carriers having a silicate backbone, such as porous glasses or other solid or flexible carriers on which the DNA-binding protein or polypeptide is immobilized. After separation of procaryotic DNA from eucaryotic DNA has been effected, the affinity matrix is rinsed with a suitable reagent, so that either the binding protein with the coupled procaryotic DNA is separated from the matrix and\/or the procaryotic DNA is separated from the binding protein and is available for further process steps in a sufficient amount.","A further application of the method according to the invention consists in the separation and enrichment of procaryotic DNA from eucaryotic DNA by binding of the procaryotic DNA to a specific protein which has been immobilized on microparticles. In this connection, all microparticles which allow the DNA-binding protein or polypeptide to be immobilized are suitable. Such microparticles may consist of latex, plastics (e.g. styrofoam, polymer), metal or ferromagnetic substances. Furthermore, use may also be made of fluorescent microparticles, such as those available from the Luminex company, for example. After the procaryotic DNA has been bound to the proteins immobilized on microparticles, said microparticles are separated from the mixture of substances by suitable methods, such as filtration, centrifugation, precipitation, sorting by measuring the intensity of fluorescence, or by magnetic methods. After separation from the microparticles, the procaryotic DNA is available for further processing.","Another application of the method according to the invention consists in the separation and enrichment of procaryotic DNA from eucaryotic DNA by binding of the procaryotic DNA to a specific protein or polypeptide, which is subsequently separated from other ingredients of the mixture by electrophoresis.","A further application of the method according to the invention consists in the separation and enrichment of procaryotic DNA from eucaryotic DNA by binding of the procaryotic DNA to the protein or polypeptide. Said protein is subsequently bound to corresponding antibodies. The antibodies may be bound to solid or flexible substrates, such as glass, plastics, silicon, microparticles, membranes, or may be present in solution. After binding of the procaryotic DNA to the protein or the polypeptide and binding of the latter to the specific antibody, separation from the substance mixture is effected by methods known to the person skilled in the art.","As protein or polypeptide, any protein or polypeptide is particularly suitable which binds procaryotic DNA with non-methylated CpG motifs, for example. For this purpose, specific antibodies or antisera against procaryotic DNA are suitable, for example. Their preparation and isolation are known to the person skilled in the art.","Procaryotic DNA differs from eucaryotic DNA, for example, by the presence of non-methylated CpG motifs. Thus, the protein\/polypeptide is conveniently a protein which specifically recognizes and binds non-methylated CpG motifs. Conveniently, this also includes a specific antibody or a corresponding antiserum. According to a further preferred embodiment, the protein or polypeptide is a protein or polypeptide encoded by the TLR9 gene or by the CGBP gene.","This embodiment of the invention is based on the finding that eucaryotic DNA and procaryotic DNA differ in their content of CpG motifs. In the procaryotic DNA, cytosine-guanosine-dinucleotides (CpG motifs) are present in an excess of 20 times that of eucaryotic DNA. In procaryotic DNA, these motifs are non-methylated, whereas they are methylated for the most part in eucaryotic DNA, which further enhances the difference. Non-methylated CpG motifs are non-methylated deoxycytidylate-deoxyguanylate-dinucleotides within the procaryotic genome or within fragments thereof.","Secondly, this preferred embodiment of the invention is based on the finding that there are proteins or polypeptides which bind specifically to non-methylated CpG motifs of the DNA. The binding property of these proteins\/polypeptides is used, according to the invention, in order to bind procaryotic DNA, on the one hand, and thus to enrich it, on the other hand, from a sample mostly containing eucaryotic DNA.","An application for isolating cDNA, which uses the presence of methylated CpG motifs in eucaryotic DNA was described by Cross et al. Nature Genetics 6 (1994) 236-244. The immunostimulatory application of single-stranded oligodeoxyribonucleotides (ODN) with the corresponding CpG motifs has been shown several times (Hacker et al., Immunology 105 (2002) 245-251, U.S. Pat. No. 6,239,116). As recognition molecules of the procaryotic CpG motifs, two receptor proteins have been identified so far. Toll-like-receptor 9 is known from WO 02\/06482 as a molecule recognizing non-methylated CpG motifs. Voo et al. Molecular and Cellular Biology (2000) 2108-2121 describe a further receptor protein, i.e. the human CpG-binding protein (hCGBP), which is used in an analytic approach as a recognition molecule for detecting non-methylated CpG motifs in procaryotic DNA. In both publications, the CpG-binding proteins are not used for isolating or enriching procaryotic DNA.","A protein or polypeptide which is encoded by cDNA having a sequence with a homology of at least 80%, preferably at least 90%, and particularly preferably at least 95%, to the sequence according to gene bank access no.: NM-014593 (SEQ. ID No. 1) (version NM-014593 1, GI: 7656974; NCBI database) is particularly suitable. These are proteins or polypeptids which correspond to CGBP or are derived therefrom and which specifically recognize and bind CpG motifs. A method of enriching prokaryotic DNA also comprises a protein or polypeptide which is encoded by cDNA with a sequence having a homology of at least 80%, preferably at least 90%, to the sequence according to gene bank access no.: NM-001101654 (SEQ. ID No. 2) (version NM-001101654 1, GI: 156142181; NCBI database).","According to a further preferred embodiment, the protein or polypeptide is encoded by cDNA having a sequence with a homology of at least 80%, preferably at least 90%, to the sequence according to gene bank access no. AB045180 (SEQ. ID No. 3) (coding sequence of the TLR9 gene; NCBI database, version AB045180.1; GI: 11761320) or a fragment thereof, preferably cDNA having a homology of at least 80%, particularly preferably 90%, to transcript variant A (gene bank access no. NM-138688 (SEQ. ID No. 4); version NM-017442.1; GI: 20302169; NCBI database) or transcript variant B (gene bank access no. NM-017442 (SEQ. ID No. 5); version NM-138688.1; GI: 20302170; NCBI database).","Moreover, the invention relates to a method of purifying body fluids to remove procaryotic DNA. In this connection, it is convenient for the separation to be effected extracorporally, under sterile conditions, to allow the body fluids to be fed back into the body again, so that the body's own immune system is assisted in eliminating infections by removing the procaryotic DNA contained in said body fluids.","Any suitable chemical, mechanical or electrochemical processes may be considered for the extracorporal removal of procaryotic DNA from body fluids. Further, the combination with other extracorporal therapeutic methods, such as hemoperfusion, heart-lung machine or endotoxin absorbers, represents a further convenient application. This enumeration does not represent a limitation of the methods.","According to a particularly preferred embodiment, the invention relates to a method of detecting procaryotic DNA. In this case, the enrichment of the procaryotic DNA is followed by a step of amplifying said procaryotic DNA, for which all common methods of amplification are suitable (PCR, LCR; LM-PCR, etc.).","Moreover, the invention relates to a kit for enriching procaryotic DNA by means of one of the above-described methods, said kit containing at least the protein\/polypeptide, preferably further reagents suitable to carry out said method.","According to a preferred embodiment, said kit contains, in addition to the protein\/polypeptide, at least one set of primers, which are suitable to amplify genomic DNA of certain procaryonts under standard conditions.","The invention has the advantage that, by specific binding of non-methylated procaryotic DNA rich in CpG motifs to proteins with specific affinity for such structures, procaryotic DNA from the total DNA of an infected host is successfully concentrated and thus the sensitivity of detection of pathogen DNA in body fluids is strongly enhanced.","The possibilities of separating procaryotic DNA from eucaryotic DNA using a specifically binding protein are no more time-consuming than known methods of isolating total DNA. However, the following detection can then be effected only via a PCR reaction. A nested PCR will not be required in most cases, which makes it possible to save a considerable amount of time in diagnostics.","The invention will be explained in more detail below by means of examples, without limiting it thereto.","Fresh, heparinized human blood, which contains streptococcus pyogenes with 103\/ml colony-forming units as pathogens, is used for detection of pathogens. The DNA is isolated by means of absorption to DNA-binding matrix using commercial kits for isolation of total DNA from body fluids according to modified instructions from the manufacturer. For this purpose, 200 \u03bcl of the total lysis buffer, which contains proteinase K and SDS, is added to 100 \u03bcl of infected blood in Eppendorf tubes. The mixture is incubated at 37\u00b0 C. for 30 min. and then heated to 95\u00b0 C. for 20 min. After cooling, 20 \u03bcl of mutanolysine are added and incubated at 37\u00b0 C. for another 60 min. After centrifugation, the supernatant is applied to the centrifugal columns using DNA-binding matrix and the DNA is purified according to the manufacturer's instructions. The purified DNA is placed in a final volume of 100 \u03bcl of 0.01 mol tris buffer, pH 7.5, or in an equal amount of elution buffer from the manufacturer. For detection of pathogens, primers are selected to identify the streptolysin O gene (slo).\n\n",{"@attributes":{"id":"p-0049","num":"0051"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"right"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Forward\u2003primer\u20031:","(SEQ.\u2003ID\u2003No.\u20036)"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"189pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"5\u2032-AGCATACAAGCAAATTTTTTACACCG\u2003"]},{"entry":[{},{}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"right"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Reverse\u2003primer\u20032:","(SEQ.\u2003ID\u2003No.\u20037)"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"189pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"5\u2032-GTTCTGTTATTGACACCCGCAATT"]}}}]}},"ul":{"@attributes":{"id":"ul0003","list-style":"none"},"li":["Primer Concentration 1 mg\/ml","Starting material: 5 \u03bcl isolated DNA\n    \n    ","Reaction:"]}},{"@attributes":{"id":"p-0050","num":"0059"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"left"}}],"thead":{"row":{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"1 x","5 min 95\u00b0 C."]},{"entry":[{},"40 cycles each at","30 sec. 95\u00b0 C."]},{"entry":[{},{},"30 sec. 51\u00b0 C."]},{"entry":[{},{},"3 min 72\u00b0 C."]},{"entry":[{},"1 x","7 min 72\u00b0 C."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}}}},"The results of the PCR of streptococci-DNA in human blood are shown in . 10 \u03bcl of the 25 \u03bcl of starting material were separated. 1) PCR starting material containing 5 \u03bcl template DNA; 2) starting material containing 5 \u03bcl template, at a dilution of 1:10. 3) positive control: 0.2 \u03bcl of streptococci-DNA as template in the absence of eucaryotic DNA from blood. ST) molecular weight standard","Result: The primary PCR does not result in a visible PCR product. Therefore, a 2. PCR (nested PCR) was carried out as below.","2. PCR (Nested): Amplification of a 348 by Fragment Contained in the Above Slo-Fragment.",{"@attributes":{"id":"p-0053","num":"0062"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"126pt","align":"right"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Forward\u2003primer\u20033:","(SEQ.\u2003ID\u2003No.\u20038)"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"5\u2032-CCTTCCTAATAATCCTGCGGATGT-3\u2032"]},{"entry":[{},{}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"126pt","align":"right"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Reverse\u2003primer\u20034:","(SEQ.\u2003ID\u2003No.\u20039)"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"5\u2032-CTGAAGGTAGCATTAG\u2003TCTTTGAT\u2003AACG-3\u2032"]}}}]}},"ul":{"@attributes":{"id":"ul0005","list-style":"none"},"li":["Primer concentration: 1 mg\/ml","Starting material: 5 \u03bcl from PCR1, sample 1, \n    \n    "]}},{"@attributes":{"id":"p-0054","num":"0069"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"left"}}],"thead":{"row":{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"1 x","5 min 95\u00b0 C."]},{"entry":[{},"50 cycles each at","30 sec. 95\u00b0 C."]},{"entry":[{},{},"30 sec. 54\u00b0 C."]},{"entry":[{},{},"3 min 72\u00b0 C."]},{"entry":[{},"1 x","7 min 72\u00b0 C."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}}}},{"@attributes":{"id":"p-0055","num":"0070"},"figref":["FIG. 2","FIG. 1","FIG. 1"]},"Result: In the nested PCR, the desired slo-DNA fragment is amplified at a pathogen number of 100 streptococci cells per 100 \u03bcl blood (sample 1). At 5 \u03bcl template DNA in the 1PCR (), this corresponds to about 5 to 10 template molecules. At a dilution of 1:10 (sample 2), sensitivity is exhausted (0.5 to 1 template molecules).","The DNA is dissolved from a cell lysate as described above for the previous PCR methods. The difference is that between 1 ml and 5 ml of test material are employed.","Three milliliters of fresh, heparinized or citrate-added human blood, which contains streptococcus pyogenes with 102\/ml colony-forming units as pathogens, is used for detection of pathogens. The DNA is isolated by means of lysis buffers which contain SDS and proteinase K, using commercial kits to isolate total DNA from body fluids according to modified instructions from the manufacturer. For this purpose, 6 ml of the total lysis buffer, which contains proteinase K and SDS, is added to 6 ml of infected blood. The mixture is incubated at 37\u00b0 C. for 30 min. and then heated to 95\u00b0 C. for 20 min. After cooling, 200 \u03bcg of mutanolysine are added and incubated at 37\u00b0 C. for another 60 min. After centrifugation, the mixture is precipitated with ethanol at a final concentration of 70%, and upon centrifugation, the pellet is washed with 2 ml of 70% ethanol. The ethanol residue is removed in a vacuum centrifuge and the precipitated DNA is collected in 500 \u03bcl TE buffer. The DNA is then applied to a column which contains 0.5 ml of sepharose and is immobilized on the 1 mg of TLR9. The column is washed with 5 volumes of TE buffer. Elution is carried out with chaotropic ions at a high concentration, e.g. with 0.7 ml of a 6 mole NaJ or KSCN solution. This eluate can then be applied directly to a commercial DNA-isolating centrifugal column, and the CpG-enriched DNA may be isolated according to instructions, as in the initial example, to a small volume of between 20 \u03bcl and 100 \u03bcl and employed for further analysis, such as pathogen PCR."],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"p":[{"@attributes":{"id":"p-0046","num":"0047"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0047","num":"0048"},"figref":["FIG. 2","FIG. 1"]}]},"DETDESC":[{},{}]}
