---
title: Polypeptides and their use
abstract: The present invention provides polypeptides that bind to inorganic solid surfaces, structures comprising such polypeptides, and methods of making such structures.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09493513&OS=09493513&RS=09493513
owner: University of Washington Through Its Center for Commercialization
number: 09493513
owner_city: Seattle
owner_country: US
publication_date: 20121024
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS REFERENCE","STATEMENT OF U.S. GOVERNMENT INTEREST","BACKGROUND","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","Example 1","Controlling Self-Assembly of Engineered Peptides on Graphite by Rational Mutation","Results","Discussion","Conclusion","Methods","REFERENCES FOR EXAMPLE 1","SUPPLEMENTAL REFERENCES","Example 2","Controlling the Surface Chemistry of Graphite by Engineered Self-Assembled Peptides","Abstract","Introduction","Supporting Information","REFERENCES FOR EXAMPLE 2","Example 3","Spatially Modulated Biomolecular Doping of Single-Layer Graphene by Self-Assembled Peptide Nanostructures","Abstract","Introduction","Results","REFERENCES FOR EXAMPLE 3","Supplemental Information for Example 3","Supplemental References for Example 3","Example 4","Biofunctionalized Peptide Nanowires on Graphite for the Self-Assembly of Hierarchical Nanostructures","Abstract","Results","REFERENCES FOR EXAMPLE 4","Supplemental Information for Example 4","Immobilization of Streptavidin and Streptavidin-Quantum Dots on Peptide Nanowires","Example 5","Selective Detection of Proteins via Self-Coassembled Mixed Peptide Functionalized Graphene Bio-Sensors","Abstract","Introduction","Materials and Methods","Results and Discussion","Conclusions","REFERENCES FOR EXAMPLE 5","Example 6","Self-Assembled Peptides Modulating Electrical Properties of Single-Layer Two-Dimensional Materials","Abstract","Methods","REFERENCES FOR EXAMPLE 6","Example 7"],"p":["This application claims priority to U.S. Provisional Patent Application Ser. No. 61\/550,721 filed Oct. 24, 2011, incorporated by reference herein it its entirety.","This work was funded by grant number DMR-0520567 and DMR-0706655, awarded by the National Science Foundation. The U.S. government has certain rights in the invention.","Over the last several decades, self-assembly has become a viable approach to create nanostructures by allowing control over feature size and organization at the molecular and mesoscopic length scales. In organisms, proteins often display exquisite self-assemblythrough molecular recognition, determined by their rich chemistry and wide range of conformations. Proteins also control solid interfaces in hard tissues, e.g., bones, spicules, shells, and teeth, where they initiate nucleation or regulate specific mineral growth to form intricate solid structures. It is, therefore, desirable to use proteins as molecular building blocks to control practical bio-solid interfaces. Earlier studies have used proteins to form organized nanostructures on solid surfaces, e.g., assembly of bacterial surface-layer proteins, amyloids, and de novo designed peptides on practical solids. As in all protein self-assembly, molecular recognition of solids must be governed by specific, non-covalent, interactions inherent in their sequence. Correlation between primary amino acid sequences and their molecular interactions that lead to self-assembly on solid surfaces has not been established due to the complexities of protein\/solid systems. Therefore, there have been no universal method to create proteins\/peptides which can self-assemble into long-range ordered nanostructures or confluent ordered film on various materials. Furthermore, the interaction of ordered proteins or peptides with nano-materials has been unrevealed and uncontrolled. Still further, there have been no peptides found to form long-range ordered structures on graphene, the single atomic layer of graphite, or other atomic single layer materials.","In a first aspect, the present invention provides polypeptides of the general formula:",{"@attributes":{"id":"p-0006","num":"0005"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"right"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"(SEQ\u2003ID\u2003NO:\u20031)"},{"entry":"Z1-Z2-R1-R2-R3-T-R5-R6-R7-R8-R9-R10-R11-R12\u2003,"}]}}}},"br":{}},"Z1 is absent or is a molecular tag;","Z2 is absent or is any sequence of 1-88 amino acids;","R1 is selected from the group consisting of I, T, A, V, and L;","R2 is selected from the group consisting of M, Q, A, V, L, and I;","R3 is selected from the group consisting of V, S, T, I, L, and A;","R5 is selected from the group consisting of E, K, D, N, T, S and A;","R6 is selected from the group consisting of S and P;","R7 is selected from the group consisting of S and P;","R8 is selected from the group consisting of D, R, N, and A;","R9 is selected from the group consisting of W, F, F-phenyl, Amino-F, A, and Y;","R10 is selected from the group consisting of S, D, and R;","R11 is selected from the group consisting of S, D, and R; and","R12 is selected from the group consisting of W, F, A, and Y.","In a second aspect, the present invention provides polypeptides comprising an amino acid sequence according to the general formula:",{"@attributes":{"id":"p-0021","num":"0020"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"147pt","align":"right"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200336)"]},{"entry":[{},"D1-D2-D3,"]}]}}}}},"wherein D1 is a domain of about 3-5 amino acids, wherein at least two of the amino acids are hydrophobic and non-aromatic;","wherein D2 is a domain of about 5 amino acids, wherein at least three of the amino acids are hydrophilic;","wherein D3 is a domain of about 4 amino acids, wherein at least two of the amino acids in D3 have an aromatic ring;","wherein the polypeptide is capable of binding to an inorganic solid surface, such as graphite, graphene, MoS, MoSe, WSe, WS, or BN.","In a further aspect, the present invention provides compositions comprising two or more different polypeptides of the invention.","In another aspect, the present invention provides isolated nucleic acids encoding the polypeptide of embodiment of the invention. In a further aspect, the present invention provides recombinant expression vectors comprising the isolated nucleic acid of the invention operatively linked to a promoter. In a still further aspect, the invention provides recombinant host cells comprising the recombinant expression vector of any embodiment of the invention.","In a further aspect, the present invention provides structures, comprising:","(a) an inorganic solid surface; and","(b) a polypeptide array bound to the solid surface, wherein the polypeptide array comprises the polypeptide or composition of any embodiment or combination of embodiment of the invention. In various embodiments, the inorganic solid surface is selected from the group consisting of graphite, graphene, MoS, MoSe, WSe, WS, or BN, or combinations thereof.","The present invention further provides methods for making a structure, comprising contacting an inorganic solid surface with the polypeptide or composition of any of embodiment or combination of embodiments of the invention, under conditions suitable to promote self-assembly of the polypeptides into a patterned array on the surface.","In another aspect, the present invention provides methods for designing polypeptides that bind to inorganic solid surfaces, comprising:","(a) providing a polypeptide according to the general formula D1-D2-D3 (SEQ ID NO: 36),\n\n","(b) modifying amino acids in 1, 2, or all 3 domains to produce a modified polypeptide; and","(c) testing the modified polypeptide for altered binding to inorganic solid surfaces.","All references cited are herein incorporated by reference in their entirety. Within this application, unless otherwise stated, the techniques utilized may be found in any of several well-known references such as: (Sambrook, et al., 1989, Cold Spring Harbor Laboratory Press), (Methods in Enzymology, Vol. 185, edited by D. Goeddel, 1991. Academic Press, San Diego, Calif.), \u201cGuide to Protein Purification\u201d in (M. P. Deutshcer, ed., (1990) Academic Press, Inc.); (Innis, et al. 1990. Academic Press, San Diego, Calif.), 2Ed. (R. I. Freshney. 1987. Liss, Inc. New York, N.Y.), , pp. 109-128, ed. E. J. Murray, The Humana Press Inc., Clifton, N.J.), and the Ambion 1998 Catalog (Ambion, Austin, Tex.).","As used herein, the singular forms \u201ca\u201d, \u201can\u201d and \u201cthe\u201d include plural referents unless the context clearly dictates otherwise. \u201cAnd\u201d as used herein is interchangeably used with \u201cor\u201d unless expressly stated otherwise.","As used herein, the amino acid residues are abbreviated as follows: alanine (Ala; A), asparagine (Asn; N), aspartic acid (Asp; D), arginine (Arg; R), cysteine (Cys; C), glutamic acid (Glu; E), glutamine (Gln; Q), glycine (Gly; G), histidine (H is; H), isoleucine (Ile; I), leucine (Leu; L), lysine (Lys; K), methionine (Met; M), phenylalanine (Phe; F), proline (Pro; P), serine (Ser; S), threonine (Thr; T), tryptophan (Trp; W), tyrosine (Tyr; Y), and valine (Val; V).","All embodiments within and between different aspects of the invention can be combined unless the context clearly dictates otherwise.","In a first aspect, the present invention provides polypeptide comprising an amino acid sequence according to the general formula I:",{"@attributes":{"id":"p-0073","num":"0075"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"147pt","align":"right"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200336)"]},{"entry":[{},"D1-D2-D3\u2003,"]}]}}}}},"wherein D1 is a domain of about 3-5 amino acids, wherein at least two of the amino acids are hydrophobic and non-aromatic;","wherein D2 is a domain of about 5 amino acids, wherein at least three of the amino acids are hydrophilic;","wherein D3 is a domain of about 4 amino acids, wherein at least two of the amino acids in D3 have an aromatic ring;","wherein the polypeptide is capable of binding to an inorganic solid surface, such as graphite, graphene, MoS, MoSe, WSe, WS, or BN.","The polypeptides can be of any suitable length between 12 and 100 amino acid residues. In various embodiments, the polypeptides range in length between 12-90, 12-80, 12-70, 12-60, 12-50, 12-40, 12-30, 12-25, 12-20, 12-15, 12-14, 12-13, or 12 amino acids.","The polypeptides of the invention have been shown by the inventors to be capable of binding to inorganic solid surfaces, such as graphite, graphene, MoS, MoSe, WSe, WS, and BN, and to have a variety of activities related to diffusion across the surface and self-directed assembly on such inorganic atomically flat solid surfaces. Thus, the polypeptides can be used, for example, to control practical bio-solid interfaces and to prepare a wide variety of patterned arrays for a wide variety of uses. By using bio-combinatorial selection, the inventors have identified a large number of binding polypeptides, referred to as \u201cgraphene-binding polypeptides\u201d (GrBPs); it will be understood that polypeptides disclosed here may have other binding specificities, as the polypeptides were derived by mutational analysis of polypeptides originally identified as graphene-binding polypeptides. The inventors have demonstrated herein that many such GrBPs, can self-assemble into long-range ordered nanostructures on single layer atomic materials (SLAM). Through the bio-combinatorial selection process disclosed herein, the inventors have discovered that the GrBPs of the invention possess 3 domains: hydrophobic, hydrophilic, and aromatic domains. As disclosed in more detail below, the 3-domain structure allows control of the surface chemistry of ordered peptide films, more flexibly and controllably, on graphite and graphene. For example, domain I (D1) can be changed from hydrophobic to hydrophilic by alteration of amino-acids, leading to arbitrarily change the hydrophobicity of peptide-coated graphite and graphene surfaces.","As demonstrated in the examples that follow, the polypeptides of the invention possess a broad area of activities relating to binding to, diffusing across (i.e. surface movement of the polypeptides), and assembling on inorganic solid surfaces, as well as possessing a variety of chemical activities (including but not limited to modifying inorganic surface hydrophobicity or contact angle), doping effects (including but not limited to injecting holes or electrons into the inorganic surface\/single layer atomic materials (SLAM), enzymatic activities and additional functionalities.","The 3-domain structure of the polypeptides of the invention permits control of polypeptide self-assembly various material surfaces via rational design of amino-acid sequences. The alteration of amino acids in each domain enables us to control self-assembly of peptides on solid surfaces. For example, modification of primary amino acid sequence based on the 3-domain concept creates new polypeptides that can self-assemble into long-range ordered nanostructures on various atomic single layer materials, such as graphite, graphene, MoS, MoSe, WSe, WS, and BN. This technique can be applicable for other chalcogenides. In one non-limiting example (described in more detail in the examples that follow), the location of the charged amino acids in a starting polypeptide sequence () is in domain 2. By changing the location of charged amino acids from domain 2 to domain 3 (M6: IMVTASSAYDDY (SEQ ID NO: 2), negatively charged), the resulting peptides can form ordered structures on MoS. In another non-limiting example, (M8: IMVTASSAYRRY (SEQ ID NO: 3), positively charged in domain 3) can form ordered structures on boron nitride (BN). The inventors have thus discovered, for the first time, such polypeptides forming ordered nanostructures on atomic single layers, graphene and MoS.","Based on these techniques, the electrical and optical properties of atomic single layer materials can be modified in a controlled manner, as demonstrated in the examples that follow. In one embodiment, the polypeptide nanostructures permit spatially doping charge carriers into inorganic surfaces, including but not limited to graphene and MoS. This is often called as spatial modulation doping in conventional semiconductor technology. Since single-layer graphene and MoSare respectively zero-gap and direct-gap semiconductor, they are useful for electronics, where control of charge carrier density in semiconductor materials is essential for practical applications. The inventors have demonstrated that the polypeptides can be used to modify the carrier charge density with randomly attached adsorbates on graphene. Using ordered polypeptide nanowires (, for example), the spatially local carrier density of graphene was modified in a controlled manner, as described in the examples that follow. In the case of MoS, its optical properties have been modified using the polypeptides of the invention. Since MoS2 has bandgap, it can emit light. Using the polypeptides, the color of emitted light from MoScan be modified by controlling the interaction between the polypeptides and MoS. This modulation doping of atomic single layers has never been achieved before the present invention. The present invention is thus of great value, for example, in nano-opto-electronic applications with bio-materials and biomolecular systems.","Any suitable hydrophobic, hydrophilic, or aromatic ring containing amino acid can be used, whether naturally occurring or non-naturally occurring. In one embodiment, the at least two amino acids in domain D3 that have an aromatic ring are selected from the group consisting of Y, F, and W. In another embodiment, the at least two hydrophobic amino acids in D1 are selected from the group consisting of I, L, A, and V. In a further embodiment, the at least three hydrophilic amino acids in D2 are selected from the group consisting of T, S, E, and D.","In another aspect, the present invention provides polypeptides comprising or consisting of the general formula II:",{"@attributes":{"id":"p-0085","num":"0087"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"right"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"(SEQ\u2003ID\u2003NO:\u20031)"]},{"entry":[{},"Z1-Z2-R1-R2-R3-T-R5-R6-R7-R8-R9-R10-R11-R12,"]}]}}}},"br":{}},"Z1 is absent or is a molecular tag;","Z2 is absent or is any sequence of 1-88 amino acids;","R1 is selected from the group consisting of I, T, A, V, and L;","R2 is selected from the group consisting of M, Q, A, V, L, and I;","R3 is selected from the group consisting of V, S, T, I, L, and A;","R5 is selected from the group consisting of E, K, D, N, T, S and A;","R6 is selected from the group consisting of S and P;","R7 is selected from the group consisting of S and P;","R8 is selected from the group consisting of D, R, N, and A;","R9 is selected from the group consisting of W, F, F-phenyl, Amino-F, A, and Y;","R10 is selected from the group consisting of S, D, and R;","R11 is selected from the group consisting of S, D, and R; and","R12 is selected from the group consisting of W, F, A, and Y.","In one embodiment, Z1 is absent. In another embodiment, Z1 is present. As used herein, a \u201cmolecular tag\u201d is any moiety that can be linked to polypeptide and which is optionally capable of further binding to a target of interest (directly or indirectly). Z1 can be present as part of a fusion protein with the remainder of the polypeptide, or may be otherwise bound to the polypeptide (covalently or non-covalently). In one embodiment, the molecular tag is a reporter molecule (including but not limited to fluorescent proteins, fluorescent dyes, etc.), and can be used to monitor location of the polypeptides, such as when bound to a surface. In another embodiment, the molecular tag is a chemical group of interest. In another embodiment, the molecular tag is capable of further binding to a target of interest (directly or indirectly). Non-limiting examples of such molecular tags include antibodies, epitope tags, protein domains, either member of a receptor\/ligand pair or fragments thereof retaining binding activity, and other molecules with specific binding affinities to a target of interest. A non-limiting example of a binding pair is biotin\/avidin; thus, in one non-limiting embodiment Z1 can comprise biotin. In another non-limiting embodiment, Z1 comprises dihydroxyphenylalanine (DOPA). In another non-limiting embodiment, Z1 comprises any tumor-specific molecular tag. In one embodiment of a tumor-specific molecular tag, Z1 comprises the peptide EPIHRSTLTALL (SEQ ID NO; 4). Other non-limiting binding pair examples include an antibody, fragment thereof, or epitope specific for a target of interest, peptides that bind to targets of interest (including but not limited to cell-specific binding peptides, anti-microbial peptides, peptides that bind to another solid, tumor cell-binding peptides, and other antigen\/antibody pairs. Epitope tags, such as a his-tag, and antibodies directed against the epitope tag (or fragments thereof) are further examples of molecular tags. Many further such examples will be readily apparent to those of skill in the art. In one embodiment the molecular tag is a polypeptide present as a fusion with the polypeptide of the invention. Any molecular tag suitable for a given purpose can be used.","In one embodiment, Z2 is absent. In another embodiment, Z2 is present and is any sequence of 1-88 amino acids. While not being bound by any specific mechanism of action, the inventors believe that the N-terminus of the polypeptides of the invention are free for rational control of intermolecular interactions as well as further functionalization, and predictable display of specific chemistry. In various embodiments when Z2 is present, it is 1-80, 1-70, 1-10, 1-50, 1-40, 1-30, 1-20, 1-10, 1-9, 1-8, 1-7, 1-6, 1-5, 1-4, 1-3, 1-2, 2-80, 2-70, 2-10, 2-50, 2-40, 2-30, 2-20, 2-10, 2-9, 2-8, 2-7, 2-6, 2-5, 2-4, 2-3, 1, 2, or 3 amino acids. In one embodiment, Z2 is present and is 1, 2, or 3 hydrophilic amino acids; in another embodiment, Z2 is present as 1, 2, or 3 serine residues. In various further embodiments, Z2 is present as any 1, 2, or 3 amino acids (including amino acid analogues). In other embodiments, Z2 can be selected from the group consisting of: H, W, C, SS, and VV.","In another embodiment of the polypeptides according to general formula II:","Z1 and Z2 are as defined above;","R1 is selected from the group consisting of I, T, A, and L;","R2 is selected from the group consisting of M, Q, and I;","R3 is selected from the group consisting of V, S, T, and A;","R5 is selected from the group consisting of E, K, D, N, and A;","R6 is selected from the group consisting of S and P;","R7 is selected from the group consisting of S and P;","R8 is selected from the group consisting of D, R, N, and A;","R9 is selected from the group consisting of W, F, F-phenyl, Amino-F, A, and Y;","R10 is selected from the group consisting of S, D, and R;","R11 is selected from the group consisting of S, D, and R; and","R12 is selected from the group consisting of W, F, A, and Y.","In another embodiment of the polypeptides according to general formula II:","Z1 and Z2 are as defined above;","R1 is I;","R2 is selected from the group consisting of M and Q;","R3 is V;","R5 is selected from the group consisting of E, K, and A;","R6 is S;","R7 is S;","R8 is selected from the group consisting of D, R, and A;","R9 is selected from the group consisting of W and Y;","R10 is S;","R11 is S; and","R12 is selected from the group consisting of W and Y.","In a further preferred embodiment of any of the above embodiments, one, two, or all three of the following are true:","R1 is I;","R2 is M; and","R3 is V.","In a further preferred embodiment of any of the above embodiments, one, two, three, or all four of the following are true:","R5 is E;","R6 is S;","R7 is S; and","R8 is D.","In a further preferred embodiment of any of the above embodiments, one two, three, or all four of the following are true:","R5 is A;","R8 is A;","R10 is D or R; and","R11 is D or R. This embodiment is particularly useful for molybdenum binding and\/or boron nitride binding polypeptides, as exemplified by the GrBP-5 and GrBP-8 mutants (see examples for details). In this embodiment, the polypeptides are of the general formula:","Z1 and Z2 are as defined above;","R1 is selected from the group consisting of I, T, A, V, and L;","R2 is selected from the group consisting of M, Q, A, V, L, and I;","R3 is selected from the group consisting of V, S, T, I, L, and A;","R5 is A;","R6 is selected from the group consisting of S and P;","R7 is selected from the group consisting of S and P;","R8 is A;","R9 is selected from the group consisting of W, F, F-phenyl, Amino-F, and Y;","R10 is selected from the group consisting of D, and R;","R11 is selected from the group consisting of D, and R; and","R12 is selected from the group consisting of W, F, and Y. In a further embodiment of the molybdenum binding polypeptides:","Z1 and Z2 are as defined above;","R1 is I;","R2 is M;","R3 is V;","R5 is A;","R6 is S;","R7 is S;","R8 is A;","R9 is Y;","R10 is selected from the group consisting of D, and R;","R11 is selected from the group consisting of D, and R; and","R12 is Y.","In a further preferred embodiment any of the above embodiments, one or more of the following is true:","R10 is S and R11 is S;","R9 is Y and R12 is Y;","R9 is F or W and R12 is F or W; and","R10 is D or R and R11 is D or R.","In another embodiment of the polypeptides according to general formula II:","Z1 is as defined above;","Z2 is any 1 or 2 amino acids;","R1 is I;","R2 is selected from the group consisting of M and Q;","R3 is V;","R5 is selected from the group consisting of E, K, and A;","R6 is S;","R7 is S;","R8 is selected from the group consisting of D, R, and A;","R9 is selected from the group consisting of W and Y;","R10 is S;","R11 is S; and","R12 is selected from the group consisting of W and Y.","Peptides according to this embodiment and the following two embodiments are shown to have enzymatic and other unique activities, as described in the examples that follow. In another embodiment:","Z1 is as defined above;","Z2 is selected from the group consisting of H, W, C, SS, and VV;","R1 is I;","R2 is selected from the group consisting of M and Q;","R3 is V;","R5 is selected from the group consisting of E, K, and A;","R6 is S;","R7 is S;","R8 is selected from the group consisting of D, R, and A;","R9 is selected from the group consisting of W and Y;","R10 is S;","R11 is S; and","R12 is selected from the group consisting of W and Y. In a further embodiment of the polypeptides according to general formula II:","Z1 is as defined above;","Z2 is selected from the group consisting of H, W, C, SS, and VV; and","R1 is I;","R2 M;","R3 is V;","R5 is E;","R6 is S;","R7 is S;","R8 is D;","R9 is Y;","R10 is S;","R11 is S; and","R12 is Y.","In another preferred embodiment, the polypeptide comprises or consists of a polypeptide selected from the group consisting of:",{"@attributes":{"id":"p-0212","num":"0214"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"right"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"(SEQ\u2003ID\u2003NO:\u20035)"},{"entry":"IMVTESSDYSSY\u2003(GrBP5)"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u20036)"},{"entry":"IMVTESSDASSA\u2003(GrBP5-M1)"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u20037)"},{"entry":"IMVTESSDWSSW\u2003(GrBP5-M2);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u20038)"},{"entry":"IMVTKSSRFSSF\u2003(GrBP5-M3);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u20039)"},{"entry":"TQSTKSSRYSSY\u2003(GrBP5-M4);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200310)"},{"entry":"IMVTESSRYSSY\u2003(GrBP5-M5);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u20032)"},{"entry":"IMVTASSAYDDY\u2003(GrBP5-M6);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200311)"},{"entry":"IMVTASSAYRDY;"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u20033)"},{"entry":"IMVTASSAYRRY\u2003(GrBP5-M8);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200312)"},{"entry":"IMVTASSDYSSY\u2003(GrBP5-12);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200313)"},{"entry":{"u":"H"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200314)"},{"entry":{"u":"W"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200315)"},{"entry":{"u":"VV"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200316)"},{"entry":{"u":"SS"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200317)"},{"entry":"LIATESSDYSSY\u2003(GrBP5\u2003hydrophobic);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200318)"},{"entry":{"u":"AQT"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200319)"},{"entry":"IMVTASSAYSSY\u2003(GrBP\u20035\u2003neutral);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200320)"},{"entry":{"u":"Bio"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200321)"},{"entry":"IMVTEDYSSY\u2003(Rigid\u2003GrBP5);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200322)"},{"entry":{"u":"C"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200323)"},{"entry":{"u":"DOPA"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200324)"},{"entry":"IMVTESSD(Nonnatural\u2003F)SSY\u2003(AminoF-GrBP5);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200325)"},{"entry":"IMVTESSD(F-Phenyl)SSY\u2003(F-Phenyl-GrBP5);\u2003"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200326)"},{"entry":"IMVTESSDYSSY\u2003(D-GrBP5),\u2003"},{"entry":"where\u2003all\u2003residues\u2003are\u2003D\u2003amino\u2003acids;"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200327)"},{"entry":{"u":"EPIHRSTLTALL"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200328)"},{"entry":"IMVTSSWSSW\u2003(GrBP5\u2003neutral\u2003WSSW);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200329)"},{"entry":"IMVTESSDFSSF;"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200330)"},{"entry":"TQSTESSDYSSY;\u2003"},{"entry":"and"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200335)"},{"entry":"IMVTDSSAYSSY\u2003(GRP5-M10)."}]}}}}},"The polypeptides of the invention can be made by any suitable technique, including but not limited to recombinant DNA technology and standard polypeptide synthetic techniques. The polypeptides may comprise D amino acids, L amino acids, or combinations thereof, and may further contain modifications as deemed appropriate for a given use.","In another embodiment, the present invention provides compositions, comprising two or more different polypeptides of the invention. As shown in the examples that follow, polypeptides of the invention that have different properties can be combined as appropriate for a given use to provide additional functionality to resulting surfaces\/devices. In one non-limiting example, the composition comprises a combination of SSIMVTESSDYSSY (SEQ ID NO: 16) (SS-GrBP5) and one of the other polypeptides of the invention (including but not limited to GrBP5) that comprises a molecular tag. As shown in the examples that follow, SS-GrBP5 possesses anti-fouling properties; combinations of it with a tagged polypeptide of interest for a given purpose can be used to create sensors capable of selective detection of a target of the molecular tag in a complex solution (including but not limited to blood samples). In one embodiment, the polypeptides are not chemically bound to each other and may be present in any ratio suitable for a given purpose, including but not limited to between 10:1 and 1:10 of one polypeptide (such as SS-GrBP5) to another polypeptide (such as molecular tagged GrBP5); in various further embodiments, the polypeptides are present in a ratio of 9:1 to 1:9; 8:1 to 1:8; 7:1 to 1:7; 6:1 to 1:6; 5:1 to 1:5; 4:1 to 1:4; 3:1 to 1:3: 2:1 to 1:2; or 1:1. As will be understood by those of skill in the art, this embodiment includes any combination of two or more polypeptides disclosed herein. Thus, in various embodiments, the two or more polypeptides can be two or more polypeptides according to any of polypeptide formulae disclosed herein, or embodiments thereof, as well as:",{"@attributes":{"id":"p-0215","num":"0217"},"tables":{"@attributes":{"id":"TABLE-US-00006","num":"00006"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"right"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"(SEQ\u2003ID\u2003NO:\u20035)"},{"entry":"IMVTESSDYSSY\u2003(GrBP5)\u2003"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u20036)"},{"entry":"IMVTESSDASSA\u2003(GrBP5-M1)"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u20037)"},{"entry":"IMVTESSDWSSW\u2003(GrBP5-M2);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u20038)"},{"entry":"IMVTKSSRFSSF\u2003(GrBP5-M3);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u20039)"},{"entry":"TQSTKSSRYSSY\u2003(GrBP5-M4);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200310)"},{"entry":"IMVTESSRYSSY\u2003(GrBP5-M5);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u20032)"},{"entry":"IMVTASSAYDDY\u2003(GrBP5-M6);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200311)"},{"entry":"IMVTASSAYRDY;"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u20033)"},{"entry":"IMVTASSAYRRY\u2003(GrBP5-M8);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200312)"},{"entry":"IMVTASSDYSSY\u2003(GrBP5-12);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200313)"},{"entry":{"u":"H"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200314)"},{"entry":{"u":"W"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200315)"},{"entry":{"u":"VV"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200316)"},{"entry":{"u":"SS"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200317)"},{"entry":"LIATESSDYSSY\u2003(GrBP5\u2003hydrophobic);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200318)"},{"entry":{"u":"AQT"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200319)"},{"entry":"IMVTASSAYSSY\u2003(GrBP\u20035\u2003neutral);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200320)"},{"entry":{"u":"Bio"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200321)"},{"entry":"IMVTEDYSSY\u2003(Rigid\u2003GrBP5);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200322)"},{"entry":{"u":"C"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200323)"},{"entry":{"u":"DOPA"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200324)"},{"entry":"IMVTESSD(Nonnatural\u2003F)SSY\u2003(AminoF-GrBP5);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200325)"},{"entry":"IMVTESSD(F-Phenyl)SSY\u2003(F-Phenyl-GrBP5);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200326)"},{"entry":"IMVTESSDYSSY\u2003(D-GrBP5);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200327)"},{"entry":{"u":"EPIHRSTLTALL"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200328)"},{"entry":"IMVTSSWSSW\u2003(GrBP5\u2003neutral\u2003WSSW);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200329)"},{"entry":"IMVTESSDFSSF;"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200330)"},{"entry":"TQSTESSDYSSY;"},{"entry":"and"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200335)"},{"entry":"IMVTDSSAYSSY\u2003(GRPS-M10)"}]}}}}},"and any modifications thereof, such as any of the recited polypeptides further comprising a molecular tag. As disclosed in the examples that follow, the different polypeptides disclosed herein possess different binding activities (i.e.: binding to different inorganic solids and\/or different binding strengths to a given inorganic solid), aggregation, and\/or diffusion properties, while some are shown to have anti-fouling capabilities or to be capable of modifying the wettability of the inorganic solid surface. Based on the teachings herein, one of skill in the art can determine which polypeptides to combine depending on the specific goal to be achieved.","In another embodiment, two or more polypeptides of the invention are provided as a fusion protein. In this embodiment, the fusion protein has a general formula of:\n\n12,\n","wherein P1 and P2 are independently a polypeptide of the invention, and Z is an optional linker amino acid sequence between the polypeptides. In this embodiment, P1 and P2 can be the same polypeptide or different polypeptides. The linker amino acid sequence can include any number of residues between P1 and P2, or may be absent.","In a second aspect, the present invention provides isolated nucleic acid encoding the polypeptide of embodiment or combination of embodiments of the invention. The isolated nucleic acids can be used, for example, for recombinant production of the polypeptides of the invention. The isolated nucleic acid sequence may comprise RNA or DNA. As used herein, \u201cisolated nucleic acids\u201d are those that have been removed from their normal surrounding nucleic acid sequences in the genome or in cDNA sequences. Such isolated nucleic acid sequences may comprise additional sequences useful for promoting expression and\/or purification of the encoded protein, including but not limited to polyA sequences, modified Kozak sequences, and sequences encoding epitope tags, export signals, and secretory signals, nuclear localization signals, and plasma membrane localization signals. It will be apparent to those of skill in the art, based on the teachings herein, what nucleic acid sequences will encode the polypeptides of the invention.","In a third aspect, the present invention provides recombinant expression vectors comprising the isolated nucleic acid of the invention operatively linked to a suitable control sequence. \u201cRecombinant expression vector\u201d includes vectors that operatively link a nucleic acid coding region or gene to any control sequences capable of effecting expression of the gene product. \u201cControl sequences\u201d operably linked to the nucleic acid sequences of the invention are nucleic acid sequences capable of effecting the expression of the nucleic acid molecules. The control sequences need not be contiguous with the nucleic acid sequences, so long as they function to direct the expression thereof. Thus, for example, intervening untranslated yet transcribed sequences can be present between a promoter sequence and the nucleic acid sequences and the promoter sequence can still be considered \u201coperably linked\u201d to the coding sequence. Other such control sequences include, but are not limited to, polyadenylation signals, termination signals, and ribosome binding sites. Such expression vectors can be of any type known in the art, including but not limited plasmid and viral-based expression vectors. The construction of expression vectors for use in transfecting prokaryotic and eukaryotic cells is well known in the art, and thus can be accomplished via standard techniques. (See, for example, Sambrook, Fritsch, and Maniatis, in: Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Laboratory Press, 1989, pp. 109-128, ed. E. J. Murray, The Humana Press Inc., Clifton, N.J.), and the Ambion 1998 Catalog (Ambion, Austin, Tex.). The expression vector must be replicable in the host organisms either as an episome or by integration into host chromosomal DNA. In a preferred embodiment, the expression vector comprises a plasmid. However, the invention is intended to include other expression vectors that serve equivalent functions, such as viral vectors.","In a fourth aspect, the present invention provides host cells that have been transfected with the recombinant expression vectors disclosed herein, wherein the host cells can be either prokaryotic or eukaryotic. The cells can be transiently or stably transfected. Such transfection of expression vectors into prokaryotic and eukaryotic cells can be accomplished via any technique known in the art, including but not limited to standard bacterial transformations, calcium phosphate co-precipitation, electroporation, or liposome mediated-, DEAE dextran mediated-, polycationic mediated-, or viral mediated transfection. (See, for example, (Sambrook, et al., 1989, Cold Spring Harbor Laboratory Press; 2Ed. (R. I. Freshney. 1987. Liss, Inc. New York, N.Y.). A method of producing a polypeptide according to the invention is an additional part of the invention. The method comprises the steps of (a) culturing a host according to this aspect of the invention under conditions conducive to the expression of the polypeptide, and (b) optionally, recovering the expressed polypeptide. The expressed polypeptide can be recovered from the cell free extract, but preferably they are recovered from the culture medium. Methods to recover polypeptide from cell free extracts or culture medium are well known to the man skilled in the art.","In a fifth aspect, the present invention provides a structure, comprising:","(a) an inorganic solid surface; and","(b) a polypeptide array bound to the solid surface, wherein the polypeptide array comprises the polypeptide of any embodiment or combination of embodiments of the invention. As described above, the polypeptides of the invention have been show by the inventors to be capable of binding to inorganic solid surfaces and capable of self-assembly into long-range ordered nanostructures on various atomic single layer materials, including but not limited to graphite, graphene, MoS, MoSe, WSe, WS, or BN, as well as other chalcogenides.","In a preferred embodiment, the inorganic solid surface is selected from the group consisting of graphite, graphene, MoS, MoSe, WSe, WS, and BN, or combinations there. In a further embodiment, the polypeptide array comprises a patterned array on the solid surface; such patterned array may include, but is not limited to, an ordered polypeptide film, a porous confluent film, peptide clusters, nanowires, quantum dots, metallic and insulator nanoparticles (NPs, such as Au and silica, glass), and nanoscale p-n junctions, or combinations of thereof.","In a further preferred embodiment, the inorganic solid surface is a single layer material.","In another further preferred embodiment, the inorganic solid surface is part of a device selected from the group consisting of protein chips, peptide-molecular circuits, field effect transistors, designer proteins, semiconductor structures, implantable medical devices (including but not limited to cardiovascular devices and bone-contacting devices), bio-fuel cells, and biosensors for any use (including but not limited to cancer detection, insulin monitoring, neural activity monitoring, etc.).","In one embodiment, the polypeptides comprise or consist of an amino acid sequence according to the general formula I:",{"@attributes":{"id":"p-0229","num":"0231"},"tables":{"@attributes":{"id":"TABLE-US-00007","num":"00007"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"147pt","align":"right"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200336)"]},{"entry":[{},"D1-D2-D3,"]}]}}}}},"wherein D1 is a domain of about 3-5 amino acids, wherein at least two of the amino acids are hydrophobic and non-aromatic;","wherein D2 is a domain of about 5 amino acids, wherein at least three of the amino acids are hydrophilic;","wherein D3 is a domain of about 4 amino acids, wherein at least two of the amino acids in D3 have an aromatic ring;","wherein the polypeptide is capable of binding to an inorganic solid surface, such as graphite, graphene, MoS, MoSe, WSe, WS, or BN.","The polypeptides can be of any suitable length between 12 and 100 amino acid residues. In various embodiments, the polypeptides range in length between 12-90, 12-80, 12-70, 12-60, 12-50, 12-40, 12-30, 12-25, 12-20, 12-15, 12-14, 12-13, or 12 amino acids.","Any suitable hydrophobic, hydrophilic, or aromatic ring containing amino acid can be used, whether naturally occurring or non-naturally occurring. In one embodiment, the at least two amino acids in domain D3 that have an aromatic ring are selected from the group consisting of Y, F, and W. In another embodiment, the at least two hydrophobic amino acids in D1 are selected from the group consisting of I, L, A, and V. In a further embodiment, the at least three hydrophilic amino acids in D2 are selected from the group consisting of T, S, E, and D.","In another aspect, the present invention provides polypeptides comprising or consisting of the general formula II:",{"@attributes":{"id":"p-0237","num":"0239"},"tables":{"@attributes":{"id":"TABLE-US-00008","num":"00008"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"right"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"(SEQ\u2003ID\u2003NO:\u20031)"]},{"entry":[{},"Z1-Z2-R1-R2-R3-T-R5-R6-R7-R8-R9-R10-R11-R12,"]}]}}}}},"Z1 is absent or is a molecular tag;","Z2 is absent or is any sequence of 1-88 amino acids;","R1 is selected from the group consisting of I, T, A, V, and L;","R2 is selected from the group consisting of M, Q, A, V, L, and I;","R3 is selected from the group consisting of V, S, T, I, L, and A;","R5 is selected from the group consisting of E, K, D, N, T, S and A;","R6 is selected from the group consisting of S and P;","R7 is selected from the group consisting of S and P;","R8 is selected from the group consisting of D, R, N, and A;","R9 is selected from the group consisting of W, F, F-phenyl, Amino-F, A, and Y;","R10 is selected from the group consisting of S, D, and R;","R11 is selected from the group consisting of S, D, and R; and","R12 is selected from the group consisting of W, F, A, and Y.","In one embodiment, Z1 is absent. In another embodiment, Z1 is present. As used herein, a \u201cmolecular tag\u201d is any moiety that can be linked to polypeptide and which is optionally capable of further binding to a target of interest (directly or indirectly). Z1 can be present as part of a fusion protein with the remainder of the polypeptide, or may be otherwise bound to the polypeptide (covalently or non-covalently). In one embodiment, the molecular tag is a reporter molecule (including but not limited to fluorescent proteins, fluorescent dyes, etc.), and can be used to monitor location of the polypeptides, such as when bound to a surface. In another embodiment, the molecular tag is a chemical group of interest. In another embodiment, the molecular tag is capable of further binding to a target of interest (directly or indirectly). Non-limiting examples of such molecular tags include antibodies, epitope tags, protein domains, either member of a receptor\/ligand pair or fragments thereof retaining binding activity, and other molecules with specific binding affinities to a target of interest. A non-limiting example of a binding pair is biotin\/avidin; thus, in one non-limiting embodiment Z1 can comprise biotin. In another non-limiting embodiment, Z1 comprises DOPA. In another non-limiting embodiment, Z1 comprises the peptide EPIHRSTLTALL (SEQ ID NO: 4), a tumor-specific molecular tag. Other non-limiting binding pair examples include an antibody, fragment thereof, or epitope specific for a target of interest, peptides that bind to targets of interest (including but not limited to cell-specific binding peptides, anti-microbial peptides, peptides that bind to another solid, tumor cell-binding peptides, and other antigen\/antibody pairs. Epitope tags, such as a his-tag, and antibodies directed against the epitope tag (or fragments thereof) are further examples of molecular tags. Many further such examples will be readily apparent to those of skill in the art. In one embodiment the molecular tag is a polypeptide present as a fusion with the polypeptide of the invention. Any molecular tag suitable for a given purpose can be used.","In one embodiment, Z2 is absent. In another embodiment, Z2 is present and is any sequence of 1-88 amino acids. In various embodiments when Z2 is present, it is 1-80, 1-70, 1-10, 1-50, 1-40, 1-30, 1-20, 1-10, 1-9, 1-8, 1-7, 1-6, 1-5, 1-4, 1-3, 1-2, 2-80, 2-70, 2-10, 2-50, 2-40, 2-30, 2-20, 2-10, 2-9, 2-8, 2-7, 2-6, 2-5, 2-4, 2-3, 1, 2, or 3 amino acids. In one embodiment, Z2 is present and is 1, 2, or 3 hydrophilic amino acids; in another embodiment, Z2 is present as 1, 2, or 3 serine residues. In various further embodiments, Z2 is present as any 1, 2, or 3 amino acids (including amino acid analogues). In other embodiments, Z2 can be selected from the group consisting of: H, W, C, SS, and VV.","In another embodiment of the polypeptides according to general formula II:","Z1 and Z2 are as defined above;","R1 is selected from the group consisting of I, T, A, and L;","R2 is selected from the group consisting of M, Q, and I;","R3 is selected from the group consisting of V, S, T, and A;","R5 is selected from the group consisting of E, K, D, N, and A;","R6 is selected from the group consisting of S and P;","R7 is selected from the group consisting of S and P;","R8 is selected from the group consisting of D, R, N, and A;","R9 is selected from the group consisting of W, F, F-phenyl, Amino-F, A, and Y;","R10 is selected from the group consisting of S, D, and R;","R11 is selected from the group consisting of S, D, and R; and","R12 is selected from the group consisting of W, F, A, and Y.","In another embodiment of the polypeptides according to general formula II:","Z1 and Z2 are as defined above;","R1 is I;","R2 is selected from the group consisting of M and Q;","R3 is V;","R5 is selected from the group consisting of E, K, and A;","R6 is S;","R7 is S;","R8 is selected from the group consisting of D, R, and A;","R9 is selected from the group consisting of W and Y;","R10 is S;","R11 is S; and","R12 is selected from the group consisting of W and Y.","In a further preferred embodiment of any of the above embodiments, one, two, or all three of the following are true:","R1 is I;","R2 is M; and","R3 is V.","In a further preferred embodiment of any of the above embodiments, one, two, three, or all four of the following are true:","R5 is E;","R6 is S;","R7 is S; and","R8 is D.","In a further preferred embodiment of any of the above embodiments, one two, three, or all four of the following are true:","R5 is A;","R8 is A;","R10 is D or R; and","R11 is D or R. This embodiment is particularly useful for molybdenum binding and\/or boron nitride binding polypeptides, as exemplified by the GrBP-5 and GrBP-8 mutants (see examples for details). In this embodiment, the polypeptides are of the general formula:","Z1 and Z2 are as defined above;","R1 is selected from the group consisting of I, T, A, V, and L;","R2 is selected from the group consisting of M, Q, A, V, L, and I;","R3 is selected from the group consisting of V, S, T, I, L, and A;","R5 is A;","R6 is selected from the group consisting of S and P;","R7 is selected from the group consisting of S and P;","R8 is A;","R9 is selected from the group consisting of W, F, F-phenyl, Amino-F, and Y;","R10 is selected from the group consisting of D, and R;","R11 is selected from the group consisting of D, and R; and","R12 is selected from the group consisting of W, F, and Y. In a further embodiment of the molybdenum binding polypeptides:","Z1 and Z2 are as defined above;","R1 is I;","R2 is M;","R3 is V;","R5 is A;","R6 is S;","R7 is S;","R8 is A;","R9 is Y;","R10 is selected from the group consisting of D, and R;","R11 is selected from the group consisting of D, and R; and","R12 is Y.","In a further preferred embodiment any of the above embodiments, one or more of the following is true:","R10 is S and R11 is S;","R9 is Y and R12 is Y;","R9 is F or W and R12 is F or W; and R10 is D or R and R11 is D or R.","In another embodiment of the polypeptides according to general formula II:","Z1 is as defined above;","Z2 is any 1 or 2 amino acids;","R1 is I;","R2 is selected from the group consisting of M and Q;","R3 is V;","R5 is selected from the group consisting of E, K, and A;","R6 is S;","R7 is S;","R8 is selected from the group consisting of D, R, and A;","R9 is selected from the group consisting of W and Y;","R10 is S;","R11 is S; and","R12 is selected from the group consisting of W and Y.","Polypeptides according to this embodiment and the following two embodiments are shown to have enzymatic and other unique activities, as described in the examples that follow. In another embodiment:","Z1 is as defined above;","Z2 is selected from the group consisting of H, W, C, SS, and VV;","R1 is I;","R2 is selected from the group consisting of M and Q;","R3 is V;","R5 is selected from the group consisting of E, K, and A;","R6 is S;","R7 is S;","R8 is selected from the group consisting of D, R, and A;","R9 is selected from the group consisting of W and Y;","R10 is S;","R11 is S; and","R12 is selected from the group consisting of W and Y. In a further embodiment of the polypeptides according to general formula II:","Z1 is as defined above;","Z2 is selected from the group consisting of H, W, C, SS, and VV; and","R1 is I;","R2 is M;","R3 is V;","R5 is E;","R6 is S;","R7 is S;","R8 is D;","R9 is Y;","R10 is S;","R11 is S; and","R12 is Y.","In another preferred embodiment, the polypeptide comprises or consists of a polypeptide selected from the group consisting of:",{"@attributes":{"id":"p-0363","num":"0365"},"tables":{"@attributes":{"id":"TABLE-US-00009","num":"00009"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"right"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"(SEQ\u2003ID\u2003NO:\u20035)"},{"entry":"IMVTESSDYSSY\u2003(GrBP5)\u2003"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u20036)"},{"entry":"IMVTESSDASSA\u2003(GrBP5-M1)\u2003"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u20037)"},{"entry":"IMVTESSDWSSW\u2003(GrBP5-M2);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u20038)"},{"entry":"IMVTKSSRFSSF\u2003(GrBP5-M3);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u20039)"},{"entry":"TQSTKSSRYSSY\u2003(GrBP5-M4);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200310)"},{"entry":"IMVTESSRYSSY\u2003(GrBP5-M5);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u20032)"},{"entry":"IMVTASSAYDDY\u2003(GrBP5-M6);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200311)"},{"entry":"IMVTASSAYRDY;"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u20033)"},{"entry":"IMVTASSAYRRY\u2003(GrBP5-M8);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200312)"},{"entry":"IMVTASSDYSSY\u2003(GrBP5-12);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200313)"},{"entry":{"u":"H"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200314)"},{"entry":{"u":"W"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200315)"},{"entry":{"u":"VV"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200316)"},{"entry":{"u":"SS"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200317)"},{"entry":"LIATESSDYSSY\u2003(GrBPS\u2003hydrophobic);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200318)"},{"entry":{"u":"AQT"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200319)"},{"entry":"IMVTASSAYSSY\u2003(GrBP\u20035\u2003neutral);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200320)"},{"entry":{"u":"Bio"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200321)"},{"entry":"IMVTEDYSSY\u2003(Rigid\u2003GrBPS);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200322)"},{"entry":{"u":"C"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200323)"},{"entry":{"u":"DOPA"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200324)"},{"entry":"IMVTESSD(Nonnatural\u2003F)SSY\u2003(AminoF-GrBPS);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200325)"},{"entry":"IMVTESSD(F-Phenyl)SSY\u2003(F-Phenyl-GrBPS);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200326)"},{"entry":"IMVTESSDYSSY\u2003(D-GrBPS);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200327)"},{"entry":{"u":"EPIHRSTLTALL"}},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200328)"},{"entry":"IMVTSSWSSW\u2003(GrBPS\u2003neutral\u2003WSSW);"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200329)"},{"entry":"IMVTESSDFSSF;"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200330)"},{"entry":"TQSTESSDYSSY;"},{"entry":"and"},{"entry":{}},{"entry":"(SEQ\u2003ID\u2003NO:\u200335)"},{"entry":"IMVTDSSAYSSY\u2003(GRPS-M10)."}]}}}}},"The 3-domain structure of the polypeptides of the invention permits control of polypeptide self-assembly various material surfaces via rational design of amino-acid sequences. The alteration of amino acids in each domain enables us to control self-assembly of peptides on solid surfaces. For example, modification of primary amino acid sequence based on the 3-domain concept creates new polypeptides that can self-assemble into long-range ordered nanostructures on various atomic single layer materials, such as MoS, WSe, and BN. This technique can be applicable for other chalcogenides. In one non-limiting example (described in more detail in the examples that follow), the location of the charged amino acids in a starting polypeptide sequence () is in domain 2. By changing the location of charged amino acids from domain 2 to domain 3 (M6: IMVTASSAYDDY (SEQ ID NO: 2), negatively charged), the resulting peptides can form ordered structures on MoS. In another non-limiting example, (M8: IMVTASSAYRRY (SEQ ID NO: 3), positively charged in domain 3) can form ordered structures on boron nitride (BN). The inventors have thus discovered, for the first time, such polypeptides forming ordered nanostructures on atomic single layers, graphene and MoS.","Based on these techniques, the electrical and optical properties of atomic single layer materials can be modified in a controlled manner, as demonstrated in the examples that follow. In one embodiment, the polypeptide nanostructures permit spatially doping charge carriers into inorganic surfaces, including but not limited to graphene and MoS. This is often called as spatial modulation doping in conventional semiconductor technology. Since single-layer graphene and MoSare respectively zero-gap and direct-gap semiconductor, they are useful for electronics, where control of charge carrier density in semiconductor materials is essential for practical applications. The inventors have demonstrated that the polypeptides can be used to modify the carrier charge density with randomly attached adsorbates on graphene. Using ordered polypeptide nanowires (, for example), the spatially local carrier density of graphene was modified in a controlled manner, as described in the examples that follow. In the case of MoS, its optical properties have been modified using the polypeptides of the invention. Since MoS2 has bandgap, it can emit light. Using the polypeptides, the color of emitted light from MoScan be modified by controlling the interaction between the polypeptides and MoS. This modulation doping of atomic single layers has never been achieved before the present invention. The present invention is thus of great value, for example, in nano-opto-electronic applications with bio-materials and biomolecular systems.","In another embodiment, the polypeptides comprising two or more different polypeptides of the invention. As shown in the examples that follow, polypeptides of the invention that have different properties can be combined as appropriate for a given use to provide additional functionality to resulting surfaces\/devices. In one non-limiting example, the composition comprises a combination of SSIMVTESSDYSSY (SEQ ID NO: 16) (SS-GrBP5) and one of the other polypeptides of the invention (including but not limited to GrBP5) that comprises a molecular tag. As shown in the examples that follow, SS-GrBP5 possesses anti-fouling properties; combinations of it with a tagged polypeptide of interest for a given purpose can be used to create sensors capable of selective detection of a target of the molecular tag in a complex solution (including but not limited to blood samples). In one embodiment, the polypeptides are not chemically bound to each other and may be present in any ratio suitable for a given purpose, including but not limited to between 10:1 and 1:10 of one polypeptide (such as SS-GrBP5) to another polypeptide (such as molecular tagged GrBP5); in various further embodiments, the polypeptides are present in a ratio of 9:1 to 1:9; 8:1 to 1:8; 7:1 to 1:7; 6:1 to 1:6; 5:1 to 1:5; 4:1 to 1:4; 3:1 to 1:3: 2:1 to 1:2; or 1:1. As will be understood by those of skill in the art, this embodiment includes any combination of two or more polypeptides disclosed herein. Thus, in various embodiments, the two or more polypeptides can be two or more polypeptides according to general formulae I or II above, or embodiments thereof, as well as",{"@attributes":{"id":"p-0367","num":"0369"},"tables":{"@attributes":{"id":"TABLE-US-00010","num":"00010"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"right"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"(SEQ\u2003ID\u2003NO:\u20035)"},{"entry":"IMVTESSDYSSY\u2003(GrBP5)\u2003"},{"entry":"(SEQ\u2003ID\u2003NO:\u20036)"},{"entry":"IMVTESSDASSA\u2003(GrBP5-M1)"},{"entry":"(SEQ\u2003ID\u2003NO:\u20037)"},{"entry":"IMVTESSDWSSW\u2003(GrBP5-M2);"},{"entry":"(SEQ\u2003ID\u2003NO:\u20038)"},{"entry":"IMVTKSSRFSSF\u2003(GrBP5-M3);"},{"entry":"(SEQ\u2003ID\u2003NO:\u20039)"},{"entry":"TQSTKSSRYSSY\u2003(GrBP5-M4);"},{"entry":"(SEQ\u2003ID\u2003NO:\u200310)"},{"entry":"IMVTESSRYSSY\u2003(GrBP5-M5);"},{"entry":"(SEQ\u2003ID\u2003NO:\u20032)"},{"entry":"IMVTASSAYDDY\u2003(GrBP5-M6);"},{"entry":"(SEQ\u2003ID\u2003NO:\u200311)"},{"entry":"IMVTASSAYRDY;"},{"entry":"(SEQ\u2003ID\u2003NO:\u20033)"},{"entry":"IMVTASSAYRRY\u2003(GrBP5-M8);"},{"entry":"(SEQ\u2003ID\u2003NO:\u200312)"},{"entry":"IMVTASSDYSSY\u2003(GrBP5-12);"},{"entry":"(SEQ\u2003ID\u2003NO:\u200313)"},{"entry":{"u":"H"}},{"entry":"(SEQ\u2003ID\u2003NO:\u200314)"},{"entry":{"u":"W"}},{"entry":"(SEQ\u2003ID\u2003NO:\u200315)"},{"entry":{"u":"VV"}},{"entry":"(SEQ\u2003ID\u2003NO:\u200316)"},{"entry":{"u":"SS"}},{"entry":"(SEQ\u2003ID\u2003NO:\u200317)"},{"entry":"LIATESSDYSSY\u2003(GrBP5\u2003hydrophobic);"},{"entry":"(SEQ\u2003ID\u2003NO:\u200318)"},{"entry":{"u":"AQT"}},{"entry":"(SEQ\u2003ID\u2003NO:\u200319)"},{"entry":"IMVTASSAYSSY\u2003(GrBP\u20035\u2003neutral);"},{"entry":"(SEQ\u2003ID\u2003NO:\u200320)"},{"entry":{"u":"Bio"}},{"entry":"(SEQ\u2003ID\u2003NO:\u200321)"},{"entry":"IMVTEDYSSY\u2003(Rigid\u2003GrBP5);"},{"entry":"(SEQ\u2003ID\u2003NO:\u200322)"},{"entry":{"u":"C"}},{"entry":"(SEQ\u2003ID\u2003NO:\u200323)"},{"entry":{"u":"DOPA"}},{"entry":"(SEQ\u2003ID\u2003NO:\u200324)"},{"entry":"IMVTESSD(Nonnatural\u2003F)SSY\u2003(AminoF-GrBP5);"},{"entry":"(SEQ\u2003ID\u2003NO:\u200325)"},{"entry":"IMVTESSD(F-Phenyl)SSY\u2003(F-Phenyl-GrBP5);"},{"entry":"(SEQ\u2003ID\u2003NO:\u200326)"},{"entry":"IMVTESSDYSSY\u2003(D-GrBP5);"},{"entry":"(SEQ\u2003ID\u2003NO:\u200327)"},{"entry":{"u":"EPIHRSTLTALL"}},{"entry":"(SEQ\u2003ID\u2003NO:\u200328)"},{"entry":"IMVTSSWSSW\u2003(GrBP5\u2003neutral\u2003WSSW);"},{"entry":"(SEQ\u2003ID\u2003NO:\u200329)"},{"entry":"IMVTESSDFSSF;"},{"entry":"(SEQ\u2003ID\u2003NO:\u200330)"},{"entry":"TQSTESSDYSSY;"},{"entry":"(SEQ\u2003ID\u2003NO:\u200335)"},{"entry":"IMVTDSSAYSSY\u2003(GRP5-M10);"}]}}}}},"and any modifications thereof, such as any of the recited polypeptides further comprising a molecular tag. As disclosed in the examples that follow, the different polypeptides disclosed herein possess different binding activities (i.e.: binding to different inorganic solids and\/or different binding strengths to a given inorganic solid), aggregation, and\/or diffusion properties, while some are shown to have anti-fouling capabilities or to be capable of modifying the wettability of the inorganic solid surface. Based on the teachings herein, one of skill in the art can determine which polypeptides to combine depending on the specific goal to be achieved.","In various non-limiting embodiments:","M6: IMVTASSAYDDY (SEQ ID NO: 2) (or M6 with a molecular tag) can form ordered structures on MoS;","M8: IMVTASSAYRRY (SEQ ID NO: 3) (or M8 with a molecular tag) can form ordered structures on boron nitride (BN);","GrBP5, SS-GrBP5, M2, M4, and M5 (with or without a molecular tag) can form ordered structures on graphene;","Inorganic surface wettability can be controlled using polypeptide arrays of M1, M2, M3, M4, M5, M6, M7, M8, (with or without a molecular tag), or combinations thereof;","Use of any of the polypeptides (or combinations thereof) with a molecular tag to form chemically addressable polypeptide nanowires on an inorganic surface on which the polypeptide(s) used form ordered structures. By controlling the growth conditions, through both time and concentration of polypeptide(s), it is possible to form variety of polypeptide nanostructures, i.e., O-D clusters, 1-D wires and 2-D films, with defined number density and highly precise spacing at nanoscale dimensions. Attachment of the molecular tag to such arrays can be used as the basis for organizing other metallic nanoparticles for optical, electrical or magnetic nanodevices fabricated on inorganic solids; and","Providing a mixed monolayer of SS-GrBP5 combined with GrBP-5 (with a molecular tag) on graphene to produce a sensor capable of selective detection of a target of the molecular tag in a complex solution. Numerous other non-limiting embodiments are disclosed in the examples that follow. As discussed above, the structures of the invention permit the electrical and optical properties of atomic single layer materials to be modified in a controlled manner. In one embodiment, the polypeptide nanostructures (including but not limited to nanowires quantum dots, and metallic and insulator nanoparticles (NPs, such as Au and silica, glass)) permit spatially doping charge carriers into inorganic surfaces, including but not limited to graphite, graphene, MoS, MoSe, WSe, WS, and BN. Since single-layer graphene and MoSare respectively zero-gap and direct-gap semiconductor, they are useful for electronics, where control of charge carrier density in semiconductor materials is essential for practical applications. In another embodiment, the surface comprises MoS, Since MoS2 has bandgap, it can emit light; the structures of the invention provide for modified color of emitted light from MoSby controlling the interaction between the polypeptides and MoS. Other non-limiting embodiments are described in the examples that follow.","The self-assembled polypeptides, SAPs, discovered here offer opportunities for new fundamental studies of ordered bio-molecular surface structures and their atomistic details via computational modeling and more detailed and quantitative experimental structural analysis. Biologically engineered inorganic solid surfaces created by SAPs may enable future peptide-based hybrid molecular technologies such as protein chips, peptide-molecular circuits, and designer multifunctional proteins and enzymes genetically engineered to perform diverse, addressable functions. Further, the inventors have shown that the polypeptides can self-assemble into long range ordered structures, such as nanowires, quantum dots, and\/or metallic and insulator nanoparticles (NPs, such as Au and silica, glass), on single-layer inorganic solid surfaces, can form nano-scale electronic p-n junctions on inorganic solid surfaces, and can modulate the electrical conductivity and photoluminescence of inorganic solid surfaces such as single-layer MoS. Thus, the structures of the present invention can realize diverse and complex electronics of two dimensional materials, such as bio-nano electronics.","Surface functionalization of solid surfaces with biological materials has been done by synthetic chemicals linked with proteins or other biological materials. However, these conventional techniques have difficulties in forming long-range ordered structures on solid surfaces and their covalent bonding to solid surfaces easily distract the inherent physical properties of solid surfaces. This is more critical in the case of atomic single layers because of their extremely small thickness and volume. The polypeptides of the invention interact with surfaces through pi-pi coupling with aromatic rings rather than such distractive covalent bonds. Furthermore, the rational design of amino acid sequence based on the three domain concept allows control of the assembly and surface (physical) topology, surface chemistry, and interaction of the polypeptides with the surface, leading to doping effects.","In a sixth aspect, the present invention provides methods for making the structures of the invention, comprising contacting an inorganic solid surface with one or more polypeptides of the invention under conditions suitable to promote self-assembly of the polypeptides into a patterned array on the surface. Such patterned array may include, but is not limited to, an ordered polypeptide film, a porous confluent film, peptide clusters, nanowires, quantum dots, metallic and insulator nanoparticles (NPs, such as Au and silica, glass), nanoscale p-n junctions, and combinations thereof. Any suitable conditions can be used to promote self-assembly of the polypeptides on the inorganic solid surface, and will depend at least on the polypeptide(s) used, the inorganic solid surface used, and the structure to be prepared. It is well within the level of skill in the art to determine appropriate conditions, based on the extensive teachings herein. For example, the use of different pH and buffer conditions can be used to modify polypeptide binding to the inorganic solid surface, surface diffusion, and\/or self-assembly, as shown in the examples that follow.","In a preferred embodiment, the inorganic solid surface is selected from the group consisting of graphite, graphene, MoS, MoSe, WSe, WS, or BN, or combinations there. In a further preferred embodiment, the inorganic solid surface is a single layer material. In another further preferred embodiment, the inorganic solid surface is part of a device selected from the group consisting of protein chips, peptide-molecular circuits, field effect transistors, designer proteins, semiconductor structures, implantable medical devices (including but not limited to cardiovascular devices and bone-contacting devices), bio-fuel cells, and biosensors for any use (including but not limited to cancer detection, insulin monitoring, neural activity monitoring, etc.).","In a seventh aspect, the present invention provides methods for designing polypeptides that bind to inorganic solid surfaces, comprising:","(a) providing a polypeptide according to the general formula D1-D2-D3 (SEQ ID NO: 36),\n\n","(b) modifying amino acids in 1, 2, or all 3 domains to produce a modified polypeptide; and","(c) testing the modified polypeptide for altered binding to inorganic solid surfaces.","The methods of this aspect of the invention utilize the polypeptides designed herein, and their 3-domain structure, as the basis for design of modified polypeptides that can be used to produce polypeptides with altered inorganic solid surface binding characteristics of interest. The polypeptides of this aspect are as described herein. Modifying amino acids in the peptides can be accomplished using any suitable technique, such as through replacement with a different amino acid during chemical synthesis, or by modifying a coding region using standard techniques where recombinant production is desired. Any suitable methods for testing binding of the polypeptides to inorganic solid surfaces can be used, including but not limited to those disclosed in the examples that follow.","In one non-limiting embodiment, at least one amino acid is modified in D1 to alter D1 from hydrophobic to hydrophilic. In one embodiment, one or both of the hydrophobic amino acids in D1 are modified to hydrophilic amino acids. In this embodiment, one can change the hydrophobicity of peptide-coated surfaces in a controlled manner.","In another non-limiting embodiment, at least one amino acid modification is made in D2 and at least one amino acid modification is made in D3, such that a charged residue is only found in D3. Such peptides can form ordered structures on boron nitride (BN) or MoS, as described herein. For example, in one non-limiting embodiment, at least two amino acid modifications are made in D3, such that D3 comprises or consists of at least two amino acids having an aromatic ring and at least two charged amino acids.","As demonstrated in the examples, the ability to control electronic and optical properties of a man-made matter (single layer atomic materials, the latest in nanotechnology) using bio-combinatorially selected and rationally engineered represents for the first time, the opening of a new area of technology with tremendous potential in practical engineering and medicine. By doing so, it is demonstrated that inanimate matter can now be manipulated to change its intrinsic physical and chemical functionalities using an animate (biological) entity, a simple sequence design of a peptide. The extraordinary potential posed by this invention is schematically described below:\n\n","Since the polypeptide (containing just 12 amino acids in the examples, and hence several hundred atoms) and the single layer material (several hundred atoms) are comparable in size, GEPI can affect the structure and, hence, the intrinsic function of SLAM, and vice versa, i.e., surface atomic structures of the inorganic SLAM material, including chemistry and crystallographic symmetry, affects\/controls the conformation on the polypeptide on the surface and its function (i.e., assembly and displayed chemistry).","The surface-bound polypeptides may further bind to other entities, including but not limited to DNA, RNA, and proteins, for use in diagnostic and therapeutic applications.","The results demonstrated herein also show that quantum dots (QDs) can be directed assembled on, for example, peptide nanowires described. Similarly, metallic and insulator nanoparticles (NPs, such as Au and silica, glass) can also be directed assembled. Clearly, these further demonstrate the significance of the invention, since NPs and QDs add enormously to the functionality of these GEPIU\/SLAM systems.","In organisms, interfaces in hard tissues are controlled by proteins via molecular recognition of specific mineral faces in, e.g., bones, spicules, shells, and teeth,where they initiate nucleation and regulate growth of specific phases to form intricate solid architectures.Inspired by biology, self-assembly of proteins onto solid surfaces is an enabling methodology for developing molecular surface coatings for a wide range of biological applications, e.g., biocompatible implants,controlled biofilms,and, more recently, the development of molecular biosensors.In these applications, protein functions on solids, such as molecular recognitionand self-assembly,derives from inherently rich chemistry and complex molecular conformations coded by their amino acid sequences. Understanding the relationship between protein sequences and surface functions during self-assembly would establish them as highly programmable molecular constructs to tailor structure and chemistry of bio-solid interfaces. To this end, unique 2D organizations of proteins have been exploited on solid surfaces using, e.g., bacterial surface-layer proteins,linear amyloid structuresand ordered films of de novo designed peptides.However, the correlation between amino acid sequences and detailed 2D molecular ordering remains largely unknown, due to complex biomolecule-solid and intermolecular interactions that accompany surface growth processes.","As a novel approach, we demonstrate that short dodecapeptides selected by phage display are capable of self-assembly on graphite and form long-range ordered biomolecular nanostructures. Using atomic force microscopy and contact angle studies, we identify three amino-acid domains along the primary sequence that steer peptide ordering and lead to nanostructures with uniformly displayed residues. The peptides are further engineered via simple mutations to control fundamental interfacial processes, including initial binding, surface aggregation and growth kinetics, and intermolecular interactions. Tailoring short peptides via their primary sequence offers versatile control over molecular self-assembly, resulting in well-defined surface properties essential in building engineered, chemically rich, bio-solid interfaces.","In this molecular self-assembly study, we utilize GrBP5 (IMVTESSDYSSY (SEQ ID NO: 5), ), the strongest solid binding member of the biocombinatorially selected sixty different sequences, designated as the wild-type (WT) peptide. A combinatorial library of \u02dc10random 12-mer peptides fused to the minor coat protein (pIII) of M13 phage was used to select sequences with affinity to graphite flakes. Four selection rounds were carried out in the panning experiment, where each round consisted of (i) Panning the phage library against graphite powder, (ii) Rinsing unbound phage, (iii) Elution of specifically bound phages and (iv) Amplification of the enriched selection library. Affinity of the final selected clones was then quantified by spectrophotometric absorbance of depleted phage solutions after long exposure to graphite (see SI, Supplementary Methods S1-S3). We first characterize assembly of WT GrBP5 on highly ordered pyrolytic graphite (HOPG) substrate, incubated with 2 \u03bcM peptide solution in distilled water for three hours and scanned with an atomic force microscope (AFM) in air. The images reveal the unique capability of GrBP5 to form uniformly ordered molecular structures over several micrometers in dimension on the HOPG surface (). The peptide film displays highly ordered nanostructures that display six-fold symmetry; seen as discrete maxima in the fast Fourier transform (FFT) power spectra taken by using the AFM image (). The symmetry in the peptide film is likely to be guided by the molecular recognition of the underlying (0001) hexagonal graphite lattice, leading to preferred growth along specific crystallographic directions during the assembly process. The measured height of \u02dc1.4 nm implies that the film is monomolecular thick, well below the longest dimension of a stretched single peptide (with a stretched end-to-end distance of \u02dc4.2 nm). The considerably short height implies that peptides conform into a more compact, folded, structure during molecular ordering on graphite.","To understand the formation of peptide nanostructures, graphite surfaces were exposed to GrBP5 in a time-lapsed series of experiments (). Initially, upon 10 minutes of exposure to HOPG, peptides form discrete clusters \u02dc10-50 nm in diameter with an average height of \u02dc1.2 nm. At 60 minutes, two distinct phases are present, revealing that surface bound peptides undergo a dynamic morphological transformation from an amorphous (disordered) to an ordered state. The thicker, amorphous phase (AP) is \u02dc1.8 nm in height, and the thinner, flat, ordered phase (OP) is \u02dc1.3 nm in height. Phase-lag imaging by AFM signify a large difference between AP and OP structures (not shown). In the AP, surface-bound peptides likely crowd together randomly to form a topologically rough, disordered and porous structure (60 min) By 180 minutes, the disordered phase has fully transformed into a flat \u02dc1.3-nm thick OP monolayer with ordered morphology covering the HOPG. Pseudo three-dimensional renderings of the AFM images () better highlight the higher topography of disordered molecules among the ordered regions, showing a distinctive color for the OP (red strips) from the higher AP regions (yellow porous film). As seen in , growing OP edges are dotted with peaks (in yellow, separated by \u02dc30-nm average distance), suggesting that loosely ordered peptides are captured and incorporated into growing OP structures (also confirmed by in situ AFM experiments, not shown).","In contrast to the previous observations of peptide assembly on other atomically flat solid surfaces in the literature,we discovered in this work that the two phases formed by the peptides coexist until an eventual full ordering of a GrBP5 monolayer. This unique disordered-to-ordered peptide phase transformation takes place at about 60% total surface coverage as shown in , where percent total coverage of disordered and ordered peptides are plotted against total surface coverage (See SI, Supplementary Methods S4). Depicted in , the assembly process of GrBP5 can be divided into two broad regimes: (i) Surface aggregation, comprised of binding to and clustering of peptides on graphite, and (ii) A gradual densification and ordering process, accompanied by a phase transformation that occurs at \u02dc60% total peptide coverage.","To demonstrate that the surface processes of GrBP5 self-assembly can be interrogated through rational mutations of the peptide, we first classify the sequence into three chemically distinct domains, as depicted in : (I) hydrophobic (IMV), (II) hydrophilic (TESSD) and (III) aromatic (YSSY). Aromatic residues such as tyrosine (Y) are known to strongly interact with graphitic surfaces through a coupling of \u03c0-electrons.Two of the four residues at the C-terminus of GrBP5 are aromatic-containing tyrosines (Y), defining YSSY as Domain-III. Located at the C-terminus of the peptide, this aromatic domain may function as an anchor for initial binding and possible diffusion during the aggregation regime of GrBP5 on graphite. On the other hand, intermolecular interactions necessary for long-range order may be driven by the amphiphilic tail comprised of hydrophobic Domain-I and hydrophilic Domain-II. Domain-I contains three purely aliphatic residues isoleucine (I), methionine (M) and valine (V), located at the N-terminus, while the Domain-II spans residues 4-8: threonine (T), glutamic acid (E), two serines (S), and aspartic acid (D) in the center of the peptide. The prevalence of amphiphilic motifs in many self-assembling molecular systemsleads us to assume that the tail component of the peptide may play a key role in the AP to OP ordering transformation. Overall, therefore, we hypothesize that each domain can be correlated to the observed framework of the three regimes in the interfacial processes, i.e., binding, diffusion and self-assembly () which were then individually addressed using five mutants (labeled M1 through M5) generated specifically for this purpose.","To test the function of the presumed binding domain, first the aromatic content of Domain-III was knocked out by replacing both of the tyrosine residues at the C-terminus with alanine, A. This mutant peptide, named M1 (IMVTESSDASSA (SEQ ID NO: 6)), is expected to maintain minimal interactions with graphite by displaying only methyl groups. Not unexpectedly, AFM analysis demonstrates that the HOPG surfaces remain bare even after exposure to 1 \u03bcM of M1 for up to 3 hours (). From this simple mutation, the binding capability of GrBP5 to graphite is largely eliminated, indicating that the roles of hydrophobic residues in Domain-I and hydrophilic residues in Domain-II are not sufficient to promote an interaction with graphite. More importantly, these results show that Domain-III (YSSY (SEQ ID NO: 31)), provides anchoring of the peptide to the surface, and can be addressed independently of the remaining sequence to alter peptide binding and, possibly, surface diffusion leading to aggregation.","To examine the binding characteristics of the peptide further, the tyrosine residues in Domain-III were replaced with either tryptophan (W) or phenylalanine (F), two other natural amino acids containing aromatic moieties. It is possible that peptides containing W, which contains an extra indole-ring, provides a more conformal and stable i-interaction with graphite surfaces, giving it a higher affinity over Y. On the other hand, peptides including F, lacking an OHgroup, may have a weaker affinity towards graphite. Thus, the designed two mutant sequences, M2 (IMVTESSDWSSW (SEQ ID NO: 7)) and M3 (IMVTESSDFSSF (SEQ ID NO: 8)) may have different binding and aggregation from those of GrBP5. Systematic time-lapsed AFM experiments show that the type of aromatic residues in the anchoring domain significantly influences the formation of peptide nanostructures on graphite (). M2 and M3 display either a highly porous disordered film or only fine peptide clusters, respectively. The effect of aromatic residues on binding, as well as unbinding, is evident when the initial deposition rates of peptides, D, are quantified and compared among samples prepared at the earliest exposure times: 5 seconds for WT, 10 min for M2 and M3 (See SI, Supplementary Methods S7). Such an analysis reveals that WT peptides arrive at the surface with an estimated D of \u02dc4235\u00b143 s, while M2 arrives at \u02dc73\u00b19 sand M3 at 22\u00b18 sover a 1 \u03bcmarea (). Thus, there is a \u02dc60\u00d7 discrepancy in initial binding between WT and M2, while M3 remains the slowest.","In addition to binding, the anchoring domain may also have a fundamental role in cluster formation during the initial aggregation stages of interfacial processes on solid surfaces. Measuring the number and size of peptide cluster features on graphite over time provides means to quantitatively track peptide surface kinetics in response to Domain-III mutations. In this scheme, surface aggregation is manifested as a decrease in the number density of clusters while the cluster size increases on the average. AFM analyses of early adsorption by WT and M2 reveal that, as total area coverage increases over time, the number of WT clusters decreases at a rate of \u02dc1 scompared to a rate of \u02dc1.3\u00d710sfor M2 over an area of 1 \u03bcm; a 75-fold difference (determined from the initial slope from ). While the number of clusters decreases, as expected, the average cluster size increases over time for both WT and M2 peptides (); clusters of WT grow 24\u00d7 faster than those of M2. These observations signify that WT has a higher aggregation rate than M2. In contrast, the number of clusters in M3 increases over time while the size remains the same, implying limited diffusion and, hence, lack of aggregation. The \u02dc58\u00d7 discrepancy in deposition rate between WT and M2 might explain the higher aggregation rate of WT over M2. The relative binding and aggregation rates of mutant peptides can, therefore, be ranked in decreasing order as: WT, highest binding and aggregation; M2, low binding and low aggregation; M3, low binding and no aggregation; and M1, no binding.","The aggregation behavior of peptides is likely to be the key in determining their final, ordered nanostructures on graphite. This phenomenon was further studied by exposing graphite samples to three concentrations of peptide solutions for 3 hours each, as shown in . Upon this incubation period, WT formed long-range ordered self-assembled peptide nanostructures in all three conditions, i.e., 0.1, 1.0 and 5.0 \u03bcM. The mutant M2, with the second highest rate of aggregation, remains disordered at 0.1 \u03bcM and 1.0 \u03bcM concentrations; interestingly, however, at 5.0 \u03bcM it forms very finely ordered structures after 3 hours of incubation (See ). Since the density of peptide clusters increases monotonically with total surface coverage, a critical density of clusters is likely required for ordering. This is apparent for the WT peptide, where the film undergoes transformation from AP to OP at \u02dc60% total surface coverage (). At 60%, half of the adsorbed WT peptide exists in the ordered phase, indicating threshold coverage for transformation. M2, on the other hand, remains entirely disordered at the same total surface coverage. Since M2 eventually orders at near 100% coverage, it likely undergoes transformation at a higher total coverage than WT. We speculate that the slower aggregation kinetics of M2 impedes peptide clusters from crowding on the surface and results in finer AP features, defining the small dimension of the ordered features observed at 5 \u03bcM. On the other hand, M3 remains too sparsely clustered at all three concentrations used here, never reaching a coverage threshold and remaining discretely bound even after 3 hours and at the highest concentrations. The trends in peptide aggregation rates were also verified by quantifying differences in values of peptide affinity constants, K, which are quantitatively estimated using Langmuir-like treatments (, and Supplementary Methods S9). Here, the Y-containing WT displayed the highest K of 3.78 \u03bcM, while the WSSW mutant had a K of 1.17 \u03bcM; an affinity to graphite of about one third of the WT value. Lastly, the mutant M3 shows a significant loss of affinity with a low K of \u02dc0.1 \u03bcM. The high K of WT peptide may be an indication that the highly ordered structures are the most stable on graphite surfaces due to favorable intermolecular interactions as well as their strong surface binding.","To probe the domain that directs the ordering seen in the WT GrBP5 on graphite, Domain-I at the N-terminus was mutated by modifying its hydrophobic nature. For this purpose, both negative and positive sequence knock-outs were prepared. In the design of a negative knock-out sequence, we replaced IMV with three similarly sized hydrophilic amino acids: threonine (T), glutamine (Q) and serine (S). The resulting sequence of mutant M4, therefore, is entirely hydrophilic. In contrast to the highly ordered structures of WT peptide on the surface (), the M4 peptide forms highly porous and disordered structures (). Next, the positive knock-out mutant M5 was designed to restore the hydrophobic characteristics of Domain-I and, therefore, the amphiphilic nature of the overall tail, presuming this mutant might allow the formation of ordered structures on graphite. Here IMV was replaced with three other aliphatic amino acids; leucine (L), isoleucine (I) and alanine (A) which results in slightly higher hydropathy than that of WT. As expected, the mutant M5 is also found to bind strongly to graphite (not shown), and, upon assembly, maintains ordering similar to that by WT (). This result, therefore, proves that the hydrophobic nature of Domain-I, and the amphiphilic tail it forms with Domain-II, are essential for long-range order of the dodecapeptide on graphite.","These hydropathic mutations can also be utilized to control the chemical characteristics of the graphite surface, such as wettability. For this, contact angle (\u03b8) of aqueous droplets were measured on HOPG coated with confluent peptide films, each at a comparable surface coverage ( insets, see Methods and Materials). The ordered WT and M5 films display an average contact angle of 65.3\u00b10.8\u00b0 and 88.9\u00b10.7\u00b0, respectively, indicating that peptides with hydrophobic Domain-I exposed display surfaces with low wettability. In contrast, graphite covered by disordered peptides, e.g., the case of M4, exhibit a significant drop in \u03b8with a contact angle of 34.7\u00b11.2\u00b0, forming a wetting surface. A simple knock out of hydrophobic Domain-I in M4, therefore, considerably lowers \u03b8as compared to those in WT and M5. These results demonstrate that the chemical properties of graphite surface are tunable by rational amino acid mutations, governed by specific peptide conformations brought about by the self-assembly processes.","The correlation between sequence and self-assembly, as established, provides key insight into the transformation from disordered to ordered nanostructures of the peptide, GrBP5, on graphite. An analysis of the amphiphilic behavior of our peptides, manifested by a change in concurrent wetting angle and film height over time, permits us to propose a unique transition in the molecular conformation during assembly on the solid surface. Contact angle measurements of samples scanned by AFM containing AP peptide reveal that the \u03b8values for WT, M4 and M5 decrease linearly with surface coverage, becoming hydrophilic. The linear trend of cos(\u03b8) shown in (blue dotted line) for all three peptides implies that the AP displays chemistries, and conformations, are similar in both discrete and confluent states.As seen in , ordered films present a shift to greater hydrophobicity. In the low coverage regime, the terminal hydrophobic amino acid domain of the WT peptide is likely first buried towards graphite while hydrophilic residues are exposed to water, as depicted in . When coverage reaches a certain threshold, however, a drastic shift is observed from hydrophilic towards more hydrophobic values (black arrow in ). This phenomenon suggests that the hydrophobic domain emerges from its buried state and is, then, exposed to water, effectively switching the amphiphile's conformation. The plausible changes in the molecular structure of peptides between the disordered and ordered states are concurrent with the height changes of the peptides, as determined by AFM (). Here, the height of WT and M5 peptide films exhibit a drastic contraction by 33% and 50%, respectively. The range of height changes observed for the OP is only \u02dc0.1 nm, while it is \u02dc1.0 nm for the AP. Thus, the switch in surface wetting properties, accompanied by a physical molecular contraction, leads us to believe that amphiphilic peptides undergo folding to reach their energetically favored, and uniform, final state in the ordered phase on the surface. The purely hydrophilic M4 peptide, by contrast, exhibits no folding, or ordering, that can be measured by AFM or contact angle, as plotted in and schematically illustrated in ","Detailed investigation of the molecular self-assembly of a graphite-binding peptide, GrBP5, on graphite studied by AFM imaging revealed a strong correlation between the amino acid composition and sequence to the resulting self-assembled nanostructures. Formation of peptide nanostructures on graphite involves first the formation of a disordered film that eventually transforms into an ordered structure. This is accompanied by height changes in the film as well as various wetting characteristics, as determined by contact angle measurements. Analysis of the sequence reveals three distinct chemical domains which play a role in the binding, diffusion, and assembled organization of the peptide on graphite: aromatic, hydrophilic, and hydrophobic. Mutations of the aromatic domain at the C-terminus significantly modifies the binding characteristics of GrBP5. Tyrosine (Y) residues replaced by alanine (A) largely eliminates the ability for the peptide to bind to graphite, while replacing tyrosine with tryptophan (W) or phenylalanine (F) tunes the peptide's affinity to graphite from strong to weak or moderate binding, respectively. Mutation of the tail at the N-terminus from hydrophobic to hydrophilic eliminates the amphiphilic character of the peptide, disrupting intermolecular interactions on the surface, and prevents bound peptides from forming long range ordered nanostructures. Disordered and ordered nanostructures of peptide mutants on graphite display either hydrophilic or hydrophobic domains, respectively, effectively tuning the contact angle and, thus, giving the film wetting or non-wetting characteristics.","A wide-range of surface phenomena exhibited by peptides during assembly, e.g., binding, clustering, and ordering, share large similarities with well-established epitaxial growth processes of atomic systems on surfaces, e.g., in molecular beam epitaxy (MBE),the foundation for modern semiconducting devices. As demonstrated (), the growth behavior of peptides can be controlled by varying the peptide concentration and incubation time, which effectively changes their rate of arrival to the surface and growth on graphite. Unique to peptides, however, a simple sequence of amino acids, as found here, can be further engineered to contain programmable segments for independently controlling multiple surface and intermolecular interactions. The ability to address peptide domains provides an opportunity for the predictable control over biomolecular self-assembly in the formation of complex, novel nanoarchitectures i.e., nanoislands (M2), nanowires (WT), and amorphous and ordered confluent films (M5 and WT, respectively). The SAPs, with the capability to form ordered nanostructures with controlled surface chemistry, therefore, have the potential to be the foundation of future peptide-based hybrid molecular technologies such as protein chips,peptide-molecular circuits,and designer multifunctional proteinsand enzymes,that can be genetically engineered to perform diverse, addressable functions.","Peptide Synthesis.","Peptides were prepared on an automated solid-phase peptide synthesizer (CS336X, CSBio Inc., Menlo Park, Calif.) employing standard batch wise Fmoc chemistry procedures as reported previously.Peptides were verified by MALDI-TOF mass spectrometry. The monomeric state of peptides in solution was also verified via size-exclusion chromatography (See SI, Supplementary Methods S12).","Sample Preparation for Microscopy.","For ex situ imaging, it was essential to prevent the reorganization of peptide structures during the drying process. We found that removing incubation solutions by applying a flash freeze and freeze-dry technique, common in biological electron microscopy specimen preparation,preserved surfaces adequately. Freshly cleaved HOPG surfaces are mounted on a nickel specimen puck (Ted Pella, Inc., Redding, Calif.) and incubated with 50 \u03bcL peptide under experimental conditions in a modified scintillation vial. The vial contains a centered hollow glass column which elevates the sample to minimize contact with warm elements during the freeze-dry process. When incubation is finished, the vial is immediately placed in a \u221280 C deep freezer which freezes within 10-15 seconds. Samples are then immediately placed inside a glass jar and surrounded with crushed ice. The jar is placed in a liquid Nbath and transferred to a standard freeze-drier (Virtis Benchtop K, SP Industries, Inc., Warminster, Pa.) and immediately placed in a vacuum with a \u221280 C condensing plate to sublime frozen incubation solutions. The drying rate was \u02dc6 \u03bcL\/hr. All images in this study, except for , were prepared using this method. Surfaces in were prepared by wicking incubation solution with a KimWipe. For verification, we performed extensive time and concentration experiments using our 5 mutants and found (see ) agreement in coverage, height and density trends. In all, \u02dc60 samples were prepared in this study for reproducibility.","Atomic Force Microscopy.","A Digital Instruments (Veeco, Santa Barbara, Calif.) Multimode Nanoscope IIIa scanning probe microscope equipped with high frequency NanoSensors PPP-NCHR (NanoandMore USA, Lady's Island, S.C., USA) non-contact probes, with a 42 N\/m spring constant, at a 4V amplitude set point.","AFM Image Processing.","Large \u2018wavy\u2019 topographic features coming from HOPG surfaces observed by AFM were removed by image subtraction to allow large area coverage analysis of peptides. Specifically, GWYDDION (Czech Metrology Institute, Czech Republic) image filters and simple image operations were applied to the raw AFM data. First, images were corrected for tilt by a first order plane subtraction while fast scan lines were normalized by aligning their median z-offset. Next, an erosion filter was applied to AFM images using a neighborhood of 10-15 pixels, where peptide features are removed to create a secondary image containing only the topography of the bare surface. This secondary image is then directly subtracted from the original AFM image, yielding a background subtracted image for surface coverage analysis. To quantify surface coverage values, IMAGEJ (NIH, Bethesda, Md., USA) is used to determine the threshold depth in the image where full lateral surface features are measured. This threshold is then used to form a binary image.","To calculate coverage values for mixed images comprised of both disordered and ordered peptides (e.g., ), a particle analyzer was used to distinguish peptide features via SPIP (Image Metrology A\/S, Denmark). Ordered peptides generally occupy higher area coverage over disordered peptides and can be quantified separately for independent coverage values based on particle size.","Height measurements of discretely bound, amorphous and ordered phases of peptide in  were measured by AFM height image histogram analysis. Peaks in the histogram arise from a predominant number of pixels at certain heights. For peptides on HOPG, bare surface pixels form a prominent peak due to the atomically flat nature of graphite. Particles observed on the surface, likely clustered or monomeric peptides, are tip convoluted and fail to reflect the true maximum height of interest in the histogram so image filtering is necessary. To address this, dilation filtering was used to enhance the total number of pixels representing the maximum heights of bound particles. This is reflected by an upward shift in peptide peak height on the histogram, where overall height is measured with respect to the bare graphite peak from the original image. These values agree with individual cross sectional measurements. In confluent films, no filtering was used since pixels mainly come from the flat film and form a dominant peak. For images with two phases present, height was measured from each phase independently.","Contact Angle Study.","For coverage normalized \u03b8values, 50 mmHOPG samples were freshly cleaved and immediately incubated with 80 \u03bcL of appropriate peptide solution in water ranging from 10 mins to 7 hrs. The drop was then wicked off using a tissue and the sample was dried under a gentle stream of N. Samples were equilibrated in air for 30 min prior to contact angle measurements. Static sessile contact angles were measured by an FTA1000B Goniometer (First Ten Angstroms, Inc., Portsmouth, Va.) with a digital camera and auto-capture software system by the vendor after 2 \u03bcL of peptide solution is dropped, performed in duplicate for each surface. The peptide solutions were placed on graphite with the same concentrations as used for their original assembly to prevent desorption. The samples were then dried with nitrogen and measured for coverage by AFM. AFM images were obtained from at least four different locations (4 \u03bcm\u00d70.5 \u03bcm sized scans each) on all samples by the methods described above. (See SI, Supplementary Methods S11)","S1. Supplementary Methods: Combinatorial Selection of Graphite Binding Peptides by Phage Display","The Ph.D-12 phage display library, based on a combinatorial library of random 12-mer peptides fused to the minor coat protein(pIII) of M13 phage (New England BioLabs), was used to select peptide sequences against graphite powder (crystalline, \u02dc300 mesh, 99%, 10129, Alfa Aesar). The 12-mer phage display library has an estimated diversity of 2.7\u00d710different random clones. To enrich the graphite binding clones, four selection rounds were carried out in the panning experiment. Prior to exposing powders to the phage library, graphite was cleaned by exposure to increasingly polar solvents in the presence of ultrasonication first in a methanol\/acetone mixture, then in isopropanol, and lastly in DI water. The graphite powder was then dried under vacuum. 10 mg of graphite powder was exposed to a volume of 10 \u03bcL of phage display library and incubated in potassium phosphate\/sodium carbonate buffer (PC, 55 mM KHPO, 45 mM NaCOand 200 mM NaCl, pH 7.4), containing 0.1% detergent (combination of 20% (w\/v) tween 20 and 20% (w\/v) tween 80) for 30 min. on a rotator at room temperature and then washed twice before overnight incubation. Following overnight incubation, to remove the nonspecifically- or weakly-bound phage, the graphite powder was then washed ten times using PC buffer with increasing detergent concentrations from 0.1% to 0.5% (v\/v), at pH 7.2. The bound phage were then eluted from the surface in a stepwise manner by applying an elution buffer consisting of 0.2 M Glycine-HCl pH 2.2 (Sigma Aldrich, St. Louis, Mo.) for 15 min. Eluted phage were then transferred to an early log phase ER2738 culture (\u02dcOD: 0.5) and amplified for 4.5 h. The amplified phage were isolated by polyethylene glycol (PEG) precipitation. Purification of phage includes three main PEG-NaCl (20% (w\/v) PEG-8000 (Sigma) and 2.5 M NaCl (Sigma)) steps followed by re-suspension of phage precipitate in PC buffer with decreased volume. In the first step, 1:6 ratio PEG-NaCl (\u02dc40 mL) was added into supernatant that is recovered after centrifugation of and incubated overnight at 40\u00b0 C. to precipitate phage. Following, phage pellet was obtained by centrifugation and re-suspended in 5 mL PC. In the second and third steps, the same procedure was followed with decreased incubation times (2 hours and 10 min) and PC buffer volume (1 mL and 200 \u03bcL) In the subsequent three selection rounds, graphite powder was exposed to the phage obtained from each previous round to enrich the phage pool in favor of the strongest-binding clones. Following the four rounds of panning, amino acid sequences of the 60 phage selected clones were amplified and identified by DNA sequencing. The single-stranded DNA of selected phage plaques were isolated by a QIAprep Spin M13 Kit (Qiagen, Valencia, Calif.) and amplified via PCR in the presence of dye-labeled terminators (Big dye terminator v3.1, Applied Biosystems, Carlsbad, Calif.). PCR products were purified by Shephadex G-50 column precipitation. A 96 gIII primer or 5\u2032-OH CCC TC TAGTTA GCG TAA CG-3\u2032 (SEQ ID NO: 32) primer was used for the amplification of ssDNA. The selected sequences of DNA from clones were analyzed by an Applied Biosystems 310 Avant genetic analyzer.","S2. Supplementary Methods: Characterization of Graphite Binding Affinity by Spectrophotometric Quantification of Phage","The selected clones, following sequencing, were then tested individually for their binding affinities. To measure clone binding affinity, we adopted spectrophotometry to quantify both the amplified phage starting solutions as well as their depletion due to reaction with graphite powder. Phage concentrations in the range 10-10pfu\/ml can be accurately determined by analyzing the broad optical absorption peak located from 260 nm to 280 nm, with a slight maximum at 269 nm. Since this peak reflects the nucleotide content of the particular phage, a molar extinction coefficient (9.006\u00d710M-1cm-1) has been previously derived (3). This treatment was modified to reflect the genome size of M13KE used in our study, 7222 bp, as calculated below.",{"@attributes":{"id":"p-0420","num":"0426"},"maths":{"@attributes":{"id":"MATH-US-00001","num":"00001"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mrow":{"mi":["Phage","particles","ml"],"mo":["\u2062","\u2062","\u2062","\u2062"],"mstyle":[{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}},{"mtext":"\/"}]},"mo":"=","mfrac":{"mrow":[{"mrow":{"mo":["(",")"],"mrow":{"mrow":[{"mi":["Measured","A"],"mo":["\u2062","\u2062","\u2062","\u2062"],"mstyle":[{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}},{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}}],"mn":"269"},{"mi":["Measured","A"],"mo":["\u2062","\u2062","\u2062","\u2062"],"mstyle":[{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}},{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}}],"mn":"320"}],"mo":"-"}},"mo":["\u00d7","\u00d7"],"mn":"6","msup":{"mn":["10","16"]}},{"mn":"7222","mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.6em","height":"0.6ex"}}},"mrow":{"mo":["(",")"],"mrow":{"mi":["Number","of","nucleotides","in","phage","genome"],"mo":["\u2062","\u2062","\u2062","\u2062","\u2062","\u2062","\u2062","\u2062","\u2062","\u2062"],"mstyle":[{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}},{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}},{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}},{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}},{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}}]}}}]}}}}},"To quantify the relative binding strength of all selected clones, both phage concentration and powder weight were optimized to maximize the absorbance change upon phage reaction with graphite. First, a phage working concentration was found by forming an optimization curve (via absorbance of 10, 10and 10pfu\/ml phage solutions) and choosing the greatest value from the linear regime, 10pfu\/mL. Next, analogously, graphite powders of incremental weights (0.5, 1.0, 2.0 and 5.0 mg) were each exposed to 10pfu\/ml phage solution to form a second optimization curve, establishing a working powder weight of 5 mg. Using these conditions, all selected clones were individually exposed to 5 mg of graphite powder in PC buffer and incubated for 24 hours on a rotator at room temperature to allow phage-inorganic powder surface interactions to occur. After incubation, the reacted graphite powder was pelleted by centrifugation at 5,000 rpm for 3 min and the supernatant (unbound phage) was used for spectrophotometric measurements. Relative binding affinities were calculated from the measured absorption intensities of depleted (unbound) phage solutions as a percentage of the original absorption intensity from starting solutions. Each experimental set was performed in triplicates and repeated twice, including a positive control of only 10pfu\/ml selected phage clones with no graphite powder and a negative control that has no random insertion sequence (M13KO7 purchased from NEB, N0315S). Finally, the graphite binding peptides (GrBPs) were categorized and grouped as strong, moderate and weak binders.","S3. Supplementary Methods: Verification of Clone Affinity by Atomic Force Microscopy (AFM)","The strongest (GrBP5: IMVTESSDYSSY (SEQ ID NO: 5)) and weakest (GrBP6: THPLPIHANELT (SEQ ID NO: 34)) selected binders were verified quantitatively via AFM imaging and Langmuir modeling on highly ordered pyrolytic graphite (HOPG). Peptide sequences were synthesized and characterized for their affinity by drop coating HOPG samples with various concentrations of peptide for three hours. Samples were then scanned with high resolution tapping force AFM and coverage was quantified to form curves. From this analysis, GrBP5 was found to have an equilibrium constant (Keq) of 2.23 \u03bcM-1, which was about five-fold greater than 0.43 \u03bcM-1 Keq of GrBP6. Interestingly, while GrBP6 contains His residues that may be involved with graphite binding, it lacks the cooperative interactions of GrBP5 that forms more highly covered ordered surfaces. The hydrophilic central region of GrBP5 may be related to the evolution of the three domains of GrBP5 from the phage screening process. It is likely that the central hydrophilic region of the peptide (IMVTESSDYSSY (SEQ ID NO: 5)) allows a higher chance for selection, since soluble phages are preferable in the aqueous screening media.","S7. Supplementary Methods: Quantification of Deposition Rate D","The deposition rate of peptides onto the surface is defined by the relation: D(t)=s*F(t), where D(t) is the deposition rate, s is the sticking coefficient of the peptide, and F(t) is the flux rate of peptides hitting the surface, resulting from their bulk diffusion in solution. Since peptides used in this study are chemically similar, their size and hydrodynamic properties should result in similar F(t) trends. Therefore, measuring D(t) should directly reflect their surface affinity from varied s values of different peptide sequences. The adsorption probability s is also a measure of peptide lifetime on the surface, including both binding and unbinding events. To quantify peptide arrival rates over time in our study, coverage values at 10 minutes were divided by assuming a peptide surface area of 3.14 nmwith a radius of roughly 1 nm, established by molecular simulations to be detailed elsewhere. Coverage values were measured over ten separate 1 \u03bcm2 regions on the surface using the methods outlined previously. Counts were then calculated as per second intervals. Errors in the main text were derived by taking the error in coverage from by the area of peptides, as was done to calculate the F value.","S8. Supplementary Methods: Quantification of Particle Density Change Over Time","The change in number density of particles over time was determined by quantifying the number of features in each image over time. Methods described previously were used to measure the population of particles, defined as isolated features on the surface, over a combined minimum scanning area of 16 \u03bcm(3-4 2 \u03bcm\u00d72 \u03bcm AFM images). For WT-GrBP5, images at 1 hour were already interconnected so earlier adsorption times, below 10 mins, were required for measurable coarsening rates. Rates were measured from the linear region of the curves, which was up to \u02dc900 secs. for WT-GrBP5 and 10800 secs. for Mutant 2.","S9. Supplementary Methods: Determining Peptide Binding Affinity (K) on HOPG by AFM","WT GrBP5 and its five mutants were assessed for affinity towards HOPG by exposing surfaces to 0.5, 1.0 and 5.0 \u03bcM concentrations of 50 \u03bcL of each peptide for 3 hours. After three hours, samples were immediately flash frozen in a \u221280\u00b0 C. freezer (observed to take \u02dc10 seconds) and freeze-dried using the protocol described in the M&M section of the main text. The results permit surface coverage quantification, and yield exponential behaviors which can be modeled and quantified by fitting to an adsorption model for an affinity constant K. To extract K values, the binding affinities of peptides to graphite were estimated from a relationship between the observed surface coverage of peptides by AFM and the peptide concentrations used for incubation. Here, we employed a simple adsorption equation based on reference 4 which allows us to approximate the binding affinity.",{"@attributes":{"id":"p-0426","num":"0432"},"maths":{"@attributes":{"id":"MATH-US-00002","num":"00002"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mi":"\u03b8","mo":"=","mrow":{"msub":{"mi":["\u03b8","max"]},"mo":"\u00d7","mfrac":{"mrow":[{"mi":["K","C"],"mo":"\u00d7"},{"mn":"1","mo":"+","mrow":{"mi":["U","C"],"mo":"\u00d7"}}]}}}}},"br":[{},{}],"sub":"max "},"Synthesis was carried out on a preloaded support resin using HBTU activation chemistry, while 20% piperidine in DMF was employed to afford the Fmoc deprotection and monitored by UV absorbance at 301 nm. Following solid-phase synthesis, the peptides were cleaved off the support and side chain deprotected by stirring the resin-bound peptide in a cocktail (containing either 87.5:5:5:2.5 TFA\/thioanisole\/HO\/phenol\/ethanedithiol or 94:1:2.5:2.5 TFA\/triisopropylsilane\/HO\/EDT, depending on the peptide) under N2 atmosphere for 2-3 hrs. The resin was removed by filtration, and each of the peptides was precipitated with cold ether to yield crude peptide product that was lyophilized (Virtis Benchtop K, SP Industries, Inc., Warminster, Pa.). Peptides were reconstituted using various ratios of DI H2O and acetonitrile. Purification by reverse-phase HPLC for peptides employed, first, an isocratic (0% B for 2 min) and, then, a linear gradient of 1%\/min for analytical and 0.5%\/min for semiprep scales at 1 and 10 mL\/min flow rates, respectively. Retention times spanned 30-50 mins depending on the peptide in semi-preparative HPLC. Analytical peaks were isolated by auto-threshold collection (Waters Deltaprep 600, analytical mode) and peptides were verified by MALDI-TOF mass spectrometry with reflectron (RETOF-MS) on an Autoflex II (Bruker Daltonics, Billerica, Mass.) mass spectrometer in positive-ion mode. The observed M\/Z fractions were subsequently collected manually from a scaled semi-preparative separation (Waters Deltaprep 600, semiprep. mode).","S11. Supplementary Methods: Pendant Drop Analysis of Peptide Solutions","Pendant drop shape analysis, used to determine the liquid-air interfacial tension of solutions, relies on the distortion, by gravity, of a drop suspended from a syringe needle of a known diameter. By using the Young-Laplace equation interfacial tension is derived from the total volume of the drop, its vertical size, contact angle with the needle, and liquid density. In this case all of the geometric parameters were automatically obtained from the drop images using vendor-provided software (First Ten Angstroms, Inc, Portsmouth, Va.) and the density of the liquid is assumed to be that of water. The results indicate no statistically significant differences in the interfacial tensions of various peptide solutions used in our experiments.","S12. Supplementary Methods: Size-Exclusion Chromatography of GrBP5","Solution state of GrBP5 was characterized by size-exclusion chromatography (7.8\u00d7300 mm, Ultrahydrogel 250, Waters Corporation, Milford, Mass.), at a constant 1 mL\/min flow rate using DI Water. DI water alone showed characteristic solvent front. Peptide calibration solution (Bruker Daltonics, Billerica, Mass.), demonstrated mass range and resolution of the SEC column, while GrBP5 solution showing one major peak in generally 1200-1400 mass region of the calibration solution (data not shown). Minor peak existed as \u02dc10% of total peak area while GrBP5 peak is quantified as \u02dc90%, making up the majority of the solution.",{"@attributes":{"id":"p-0430","num":"0000"},"ul":{"@attributes":{"id":"ul0007","list-style":"none"},"li":["1. Lowenstam, H.; Weiner, S., . Oxford University Press: Oxford, 1989.","2. Sarikaya, M.; Aksay, I., & . AIP:College Park, 1996.","3. Mann, S., . Oxford University Press: Oxford, 2001.","4. Addadi, L.; Weiner, S., Interactions between Acidic Proteins and Crystals\u2014Stereochemical Requirements in Biomineralization. 1985, 82, 4110-4114.","5. Mann, S., Molecular Recognition in Biomineralization. 1988, 332 (6160), 119-124.","6. Cha, J. N.; Shimizu, K.; Zhou, Y.; Christiansen, S. C.; Chmelka, B. F.; Stucky, G. D.; Morse, D. E., Silicatein Filaments and Subunits From a Marine Sponge Direct the Polymerization of Silica and Silicones in vitro. 1999, 96, 361-365.","7. Ratner, B. D.; Hoffman, A. S.; Schoen, F. J.; Lemons, J. E., 2nd ed.; Elsevier Academic Press: San Diego, 2004.","8. Chen, C. S.; Mrksich, M.; Huang, S.; Whitesides, G. M.; Ingber, D. E., Geometric Control of Cell Life and Death. 1997, 276, 1425-1428.","9. Templin, M. F.; Stoll, D.; Schrenk, M.; Traub, P. C.; Vohringer, C. F.; Joos, T. O., Protein Microarray Technology. 2002, 20, 160-167.","10. Pauling, L., Molecular architecture and Biological Reactions. 1946, 24, 1375-1377.","11. Bradley, D. E., Ultrastructure of Bacteriophages and Bacteriocins. 1967, 31, 230-314.","12. Gyorvary, E. S.; Stein, O.; Pum, D.; Sleytr, U. B., Self-assembly and Recrystallization of Bacterial S-layer Proteins at Silicon Supports Imaged in Real Time by Atomic Force Microscopy. . () 2003, 212, 300-306.","13. Sneer, R.; Weygand, M. J.; Kjaer, K.; Tirrell, D. A.; Rapaport, H., Parallel \u03b2-Sheet Assemblies at Interfaces. 2004, 5, 747-750.","14. Zhang, F.; Du, H. N.; Zhang, Z. X.; Ji, L. N.; Li, H. T.; Tang, L.; Wang, H. B.; Fan, C. H.; Xu, H. J.; Zhang, Y., et al., Epitaxial Growth of Peptide Nanofilaments on Inorganic Surfaces: Effects of Interfacial Hydrophobicity\/Hydrophilicity. 2006, 45, 3611-3613.","15. Harper, J. D.; Wong, S. S.; Lieber, C. M.; Lansbury, P. T., Observation of Metastable A-\u03b2 Amyloid Protofibrils by Atomic Force Microscopy. 1997, 4, 119-125.","16. Brown, C. L.; Aksay, I. A.; Saville, D. A.; Hecht, M. H., Template-Directed Assembly of a de novo Designed Protein. 2002, 124, 6846-6848.","17. Grigoryan, G.; Kim, Y. H.; Acharya, R.; Axelrod, K.; Jain, R. M.; Willis, L.; Drndic, M.; Kikkawa, J. M.; DeGrado, W. F., Computational Design of Virus-Like Protein Assemblies on Carbon Nanotube Surfaces. 2011, 332, 1071-1076.","18. Sarikaya, M.; Tamerler, C.; Jen, A. K. Y.; Schulten, K.; Baneyx, F., Molecular Biomimetics: Nanotechnology through Biology. 2003, 2, 577-585.","19. Whaley, S. R.; English, D. S.; Hu, E. L.; Barbara, P. F.; Belcher, A. M., Selection of Peptides with Semiconductor Binding Specificity for Directed Nanocrystal Assembly. 2000, 405, 665-668.","20. Sano, K. I.; Shiba, K., A Hexapeptide Motif that Electrostatically Binds to the Surface of Titanium. 2003, 125, 14234-14235.","21. So, C. R.; Kulp, J. L.; Oren, E. E.; Zareie, H.; Tamerler, C.; Evans, J. S.; Sarikaya, M., Molecular Recognition and Supramolecular Self-Assembly of a Genetically Engineered Gold Binding Peptide on Au{1112009, 3, 1525-1531.","22. Wang, S. Q.; Humphreys, E. S.; Chung, S. Y.; Delduco, D. F.; Lustig, S. R.; Wang, H.; Parker, K. N.; Rizzo, N. W.; Subramoney, S.; Chiang, Y. M., et al., Peptides with Selective Affinity for Carbon Nanotubes. 2003, 2, 196-200.","23. Naik, R. R.; Brott, L. L.; Clarson, S. J.; Stone, M. O., Silica-precipitating peptides isolated from a combinatorial phage display peptide library. 2002, 2, 95-100.","24. Brown, S., Metal-recognition by repeating polypeptides. 1997, 15, 269-272.","25. So, C. R.; Tamerler, C.; Sarikaya, M., Adsorption, Diffusion, and Self-Assembly of an Engineered Gold-Binding Peptide on Au(111) Investigated by Atomic Force Microscopy. 2009, 48, 5174-5177.","26. Hnilova, M. et al. Effect of Molecular Conformations on the Adsorption Behavior of Gold-Binding Peptides. 2008, 24, 12440-12445.","27. Kowalewski, T.; Holtzman, D. M., In situ atomic force microscopy study of Alzheimer's \u03b2-amyloid peptide on different substrates: New insights into mechanism of (3-sheet formation. 1999, 96, 3688-3693.","28. Xie, H.; Becraft, E. J.; Baughman, R. H.; Dalton, A. B.; Dieckmann, G. R., Ranking the Affinity of Aromatic Residues for Carbon Nanotubes by Using Designed Surfactant Peptides. 2008, 14, 139-151.","29. Tomasio, S. M.; Walsh, T. R., Modeling the Binding Affinity of Peptides for Graphitic Surfaces. Influences of Aromatic Content and Interfacial Shape. 2009, 113, 8778-8785.","30. Whitehouse, C.; Fang, J. Y.; Aggeli, A.; Bell, M.; Brydson, R.; Fishwick, C. W. G.; Henderson, J. R.; Knobler, C. M.; Owens, R. W.; Thomson, N. H., et al., Adsorption and Self-Assembly of Peptides on Mica Substrates. 2005, 44, 1965-1968.","31. Sowerby, S. J.; Petersen, G. B.; Holm, N. G., Primordial Coding of Amino Acids by Adsorbed Purine Bases. 2002, 32, 35-46.","32. Schwartz, D. K., Mechanisms and Kinetics of Self-Assembled Monolayer Formation. 2001, 52, 107-137.","33. Barth, J. V.; Costantini, G.; Kern, K., Engineering Atomic and Molecular Nanostructures at Surfaces. 2005, 437, 671-679.","34. Jensen, P.; Barabasi, A. L.; Larralde, H.; Havlin, S.; Stanley, H. E., Deposition, Diffusion and Aggregation of Atoms on Surfaces\u2014a Model for Nanostructure Growth. 1994, 50, 15316-15329.","35. Zhu, H.; Bilgin, M.; Bangham, R.; Hall, D.; Casamayor, A.; Bertone, P.; Lan, N.; Jansen, R.; Bidlingmaier, S.; Houfek, T., et al., Global Analysis of Protein Activities using Proteome Chips. 2001, 293, 2101-2105.","36. Langer, R.; Peppas, N. A., Advances in Biomaterials, Drug Delivery, and Bionanotechnology. 2003, 49, 2990-3006.","37. Joachim, C.; Gimzewski, J. K.; Aviram, A., Electronics using Hybrid-Molecular and Mono-Molecular Devices. 2000, 408, 541-548.","38. Baneyx, F.; Mujacic, M., Recombinant Protein Folding and Misfolding in 2004, 22, 1399-1408.","39. Siegel, J. B.; Zanghellini, A.; Lovick, H. M.; Kiss, G.; Lambert, A. R.; Clair, J. L. S.; Gallaher, J. L.; Hilvert, D.; Gelb, M. H.; Stoddard, B. L., et al., Computational Design of an Enzyme Catalyst for a Stereoselective Bimolecular Diels-Alder Reaction. 2010, 329, 309-313.","40. Gross, H., 1 ed.; Springer: New York, 1987.","41. Engel, A., Biological Applications of Scanning Probe Microscopes. 1991, 20, 79-108.","42. Kyte, J. & Doolittle, R. F. A Simple Method for Displaying the Hydropathic Character of a Pro-tein. 1982, 157, 105-132.","43. Cassie, A. B. D. & Baxter, S. Wettability of Porous Surfaces. 1944, 40, 0546-0550."]}},{"@attributes":{"id":"p-0431","num":"0000"},"ul":{"@attributes":{"id":"ul0008","list-style":"none"},"li":["1. S. Donatan, H. Yazici, H. Bermek, M. Sarikaya, C. Tamerler, M. Urgen, 29, 14 (2009).","2. M. Gungormus, H. Fong, I. W. Kim, J. S. Evans, C. Tamerler, M. Sarikaya, 9, 966 (2008).","3. C. F. Barbas, D. R. Burton, J. K. Scott, G. J. Silverman, 2001.","4. I. Langmuir, 38, 2221 (1917)."]}},"The systematic control over surface chemistry is a long-standing challenge in biomedical and nano-technological applications for graphitic materials. As a novel approach, we utilize graphite-binding dodecapeptides that self-assemble into dense domains to form monolayer thick long-range ordered films on graphite. Specifically, the peptides are rationally designed through their amino acid sequences to predictably display hydrophilic and hydrophobic characteristics while maintaining their self-assembly capabilities on the solid substrate. The peptides are observed to maintain a high tolerance for sequence modification, allowing the control over surface chemistry via their amino acid sequence. Furthermore, through a single step co-assembly of two different designed peptides, we predictably and precisely tune the wettability of the resulting functionalized graphite surfaces from 44 to 83 degrees. The modular molecular structures and predictable behavior of short peptides demonstrated here give rise to a novel platform for functionalizing graphitic materials that offers numerous advantages, including non-invasive modification of the substrate, biocompatible processing in an aqueous environment, and simple fusion with other functional biological molecules.","Controlling interfacial properties of materials through surface functionalization of solid substrates has been a major challenge in medicine and nanotechnology for the last two decades.The precise display of function and chemistry is particularly critical for engineering bio-inorganic interfaces, where the orientation and density of the immobilized molecules may have direct bearing on the performance of the resulting assembly, such as, for example, the specific activity of immobilized enzymes.Biocombinatorially selected (through phage or cell-surface display) and genetically engineered solid-binding peptides offer a versatile platform for bridging the bio\/inorganic divide.These strong-binding (k\u02dc50 nM to 1 \u03bcM), material-specific 7-14 amino acid long sequences possess a wealth of chemical diversity and modular capacity through mutation and targeted chemical modification, providing unique opportunities for tuning binding, chemical properties and display.Rather than covalent bonding, prevalent in synthetic linkers, such as silanes, thiols, and phosphanates,short peptides bind through weak forces at multiple positions at the peptide\/solid interface with the advantage of assembling and functioning in aqueous solutions.As biocompatible coatings, therefore, solid-binding peptides are particularly well suited for applications in medical and biological fields because they are produced and function under biological conditions, and have not shown any toxicity in cell culture studies.","Graphite, graphene and carbon nano-tubes (CNT) have been employed for both biomedical and nanotechnological applications due to their anti-microbial activity,high conductivity, optical transparency, and surface sensitive properties.These graphitic materials have been employed for biosensing applications in particular due to their excellent electronic properties, resulting from delocalized it bonds on the surface.A number of functionalization routes through covalent bonding, i.e. the introduction of carboxylic groups, have been employed to control the interface with graphitic materials.In parallel, however, to preserve the intrinsic properties of these materials, methods of non-covalent functionalization via \u03c0-\u03c0 stacking using aromatic chemistry have been used to assemble functional molecules.","A non-invasive approach using peptides could present a biocompatible alternative to controlling the surface properties of graphite. Here, we demonstrate precise control over the hydropathy of graphite through single-step self-assembly of peptides and their engineered mutants. The graphite binding peptides are modular and can be designed to self-assemble into stable monomolecular films on graphite, exposing predictable surface chemistry through the display of specific amino acids. The highly oriented pyrolytic graphite (HOPG) has an atomically flat surface which is ideally suited to demonstrate, using atomic force microscopy and contact angle measurements, the display of tailored chemistries on graphite through controlling the binding and assembly of the designed peptides, in the absence of roughness effects.","The dodecapeptide used in this work, GrBP5-WT (Sequence: IMVTESSDYSSY (SEQ ID NO: 5), affinity constant: K=3.78 \u03bcM)is unique among graphite- and CNT-binding peptide sequences, as it forms long-range ordered, uniform, and crystallographic molecular nanostructures on HOPG (), which can be controlled through sequence mutation. The abbreviation WT denotes the original, unmodified, sequence of the peptide, which we call \u201cwild type\u201d. We have found that the self-assembly of GrBP5-WT arises from a combination of binding through the aromatic rings of tyrosine residues at the C-terminus, and ordering through intermolecular interactions among hydrophobic tail domains (). The formation of ordered morphology, apparent from , and evidenced by the FFT, seems to be a result of both lattice matching with the underlying HOPG, which results in six-fold symmetry, and assembly conditions, such as concentration, which, along with intermolecular interactions, determines the size of features.Replacing the hydrophobic residues at the N-terminus with hydrophilic ones, therefore, inhibits formation of the ordered phase (OP) and causes the peptide film to remain in the amorphous phase (AP) because of the lack of intermolecular interactions (). Furthermore, it was found that an ordered film of GrBP5-WT displays hydrophobic property on the graphite surface (). These findings motivated us to hypothesize that ordered structures of GrBP5-WT leave the N-terminus amino-acids free for rational control of intermolecular interactions as well as further functionalization, and predictable display of specific chemistry. Based on this hypothesis, we aim here to demonstrate that the wettability of graphite can be controlled by varying the N-terminal sequence of the GrBP5-WT peptide. Specifically, we design two mutants (i.e., variants of the WT peptide) which exhibit hydrophobic or hydrophilic properties while retaining their ordered structure and predictable display capability. The hydrophobic mutant GrBP5-Phob is produced by substitution of the three N-terminal amino-acids of the wild type with a more hydrophobic LIA sequence (Table 1). The hydrophilic hybrid mutant SS-GrBP5 (1), on the other hand, is designed by the addition of two hydrophilic serine residues to the N-terminus of the wild type peptide.",{"@attributes":{"id":"p-0437","num":"0488"},"tables":{"@attributes":{"id":"TABLE-US-00011","num":"00011"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u20031\u2003"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Peptide\u2003sequences,\u2003weights,\u2003and\u2003hydropathy\u2003indices"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Mol.",{}]},{"entry":["Peptide\u2003","Sequence","Mass","G.R.A.V.Y."]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["GrBP5-WT","IMVTESSDYSSY","1381.4","-0.242"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u20035)",{},{}]},{"entry":{}},{"entry":["GrBP5-Phob","LIATESSDYSSY","1335.3","-0.283"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200317)",{},{}]},{"entry":{}},{"entry":["GrBP5-Phil","TQSTESSDYSSY","1354.3","-1.542"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200330)",{},{}]},{"entry":{}},{"entry":["SS-GrBP5","SSIMVTESSDYSSY","1555.6","-0.321"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200316)"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"left","id":"FOO-00001"},"sup":"[a]"}}]}}]}}},"To characterize the wettability of the peptide-functionalized graphite surfaces, freshly prepared HOPG substrates were incubated in 1 \u03bcM aqueous peptide solutions (Table 1) for several time intervals, resulting in samples of peptide films on the graphite surface that range from sparse to near-confluent monolayers. The wettability of these films was quantified by contact angle goniometry, and the coverage was determined by atomic force microscopy (AFM) (See supporting information for detailed procedures). The plot of coverage vs. contact angle values exhibits a coinciding linear correlation below 70% coverage for all peptides (). Above 70%, they also display linear relationships but with slopes that vary greatly with sequence ().","To quantitatively compare the wettability of different peptides, we applied Cassie's Law,which describes the contact angle, cos \u03b8, of a macroscopic droplet on a chemically heterogeneous surface via the relation: cos \u03b8=\u03c6cos \u03b8+\u03c6cos \u03b8(\u03c6: coverage and \u03b8: contact angle for graphite and peptides, respectively). By fitting this equation to the experimental data, effective contact angles are extrapolated for fully covered surfaces of each peptide, \u03b8, (Table 2). Between 0 and 70% coverage, all peptides display little difference in \u03b8(about 28\u00b0\u00b14\u00b0). The \u03b8values above 70% coverage of GrBP5-WT and GrBP5-Phob, however, are drastically different.",{"@attributes":{"id":"p-0440","num":"0491"},"tables":{"@attributes":{"id":"TABLE-US-00012","num":"00012"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Projected contact angles of a theoretical 100% peptide-covered sample for"},{"entry":"each peptide and phase, and corresponding measured roughness values."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"\u03b8 AP","AP","\u03b8 OP","OP"]},{"entry":[{},"(070%","Roug.\/","(>70%","Rough.\/"]},{"entry":["Peptide","coverage)","nm","coverage)","nm"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":["GrBP5-","30.0\u00b0 \u00b1 2.5\u00b0\u2002","0.92","64.0\u00b0 \u00b1 1.0\u00b0","0.39"]},{"entry":["WT",{},{},{},{}]},{"entry":["GrBP5-","23.0\u00b0 \u00b1 11.5\u00b0","0.53","87.0\u00b0 \u00b1 0.5\u00b0","0.36"]},{"entry":["Phob",{},{},{},{}]},{"entry":["GrBP5-","32.0\u00b0 \u00b1 12.0\u00b0","0.72","No Order","No Order"]},{"entry":["Phil",{},{},{},{}]},{"entry":["SS-","25.5\u00b0 \u00b1 5.5\u00b0\u2002","1.20","36.0\u00b0 \u00b1 1.0\u00b0","0.34"]},{"entry":"GrBP5"},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},"These results are also reflected in the corresponding AFM experiments () where peptides are observed to form one of the two phases on the surface: either a long-range ordered phase (OP) at high coverage, exhibiting six-fold symmetry, or a sparse amorphous phase (AP) at low coverage without recognizable crystallographic symmetry in the formed film. There is an effective transition threshold from AP to OP at about 70% coverage. Ordered peptide films also exhibit more uniform and narrow distributions of thickness as measured by AFM, which is apparent from the average roughness values in Table 2. Only GrBP5-Phil, with no hydrophobic tail, remained disordered under all incubation conditions.","The similarity in the \u03b8contact angles of all peptides in the AP below 70% coverage may indicate that the exposed amino acid domain is conserved among mutants and is predominantly hydrophilic. This is most likely due to the random alignment of peptides displaying polar residue- and a serine-rich domain in the central portion of the peptide (Table 1). Conversely, the contrast between the contact angle contribution of GrBP5-WT (64.0\u00b0 and GrBP5-Phob (87.0\u00b0) in the OP implies that the three amino-acids at the N-terminus are uniformly exposed towards the solution, since they are the only ones that differ between the two peptide sequences.","Remarkably, both the binding and ordering capabilities of the peptides were retained in the SS-GrBP5 mutant (). Even in the ordered state, the SS-GrBP5 mutant has a contact angle contribution close to that of the AP (Table 2), indicating that hydrophilic residues are displayed in both of its phases of the peptide films. The difference in the contribution of the two phases, about 9\u00b0, confirms a transition from AP to OP, seen clearly in , on surfaces functionalized by SS-GrBP5.","The uniform display of N-terminal residues by self-assembling peptides forming confluent ordered films on graphite results in a wide range of wettability values. It is, therefore, plausible to further tune the contact angle through a simultaneous high coverage, single-step, co-assembly of peptides with varying wettability. We chose the two mutants that retained their assembly capabilities and exhibited the widest range of \u03b8, i.e., 36.0\u00b0 and 87.0\u00b0, to achieve precise control over the wettability of graphite surfaces at a constant coverage. For this, 1 \u03bcM aqueous solutions of GrBP5-Phob and the SS-GrBP5 were mixed in appropriate proportions and incubated on HOPG for 3 hours, assuming the complete solubility of the two peptides solutions. The AFM images and contact angle versus fraction coverage dependencies are shown in . While the peptide coverage of these films remained at around 80%, the contact angles of functionalized surfaces varied from 44.0\u00b0 to 83.0\u00b0. The linear nature of the plot and the uniformity of the AFM images indicate that GrBP5-Phob is homogeneously dispersed within the film formed by SS-GrBP5. Moreover, by adding a third term to Cassie's equation, we were able to predict the cosine of the contact angle resulting from a given mixed monolayer based on the \u03b8values from Table 2 (). The agreement between the prediction and the data is quite close (Coefficient of determination, based on sum of squares of error is R=0.96). The small discrepancy in the predicted and measured values could result from a slight variation in the binding affinities of the two peptides, whereby SS-GrBP5 is present on the surface in slightly higher proportion than the GrBP5-Phob under the same incubation conditions.","Controlling surface wettability through self-assembled peptides provides a novel approach for engineering biomolecule\/graphite interfaces. Mixed self-assembled peptide films prepared in water can be used, for example, in the development of bio-sensors with optimized chemical properties and biocompatibility. The intermolecular interactions among different peptides in the ordered phase can be tailored to form novel, complex nanostructures with spatially controlled structural and biofunctional properties. Furthermore, the ease with which displayed amino acid domains are introduced into short peptides permits development of biomolecular constructs with designer proteins,peptide domains,and chemical groupsto further control functionality of graphitic surfaces. This inherently biocompatible and non-covalent molecular immobilization approach is suitable for a variety of potential applications of graphite, and graphitic materials, in nanobiotechnology.","1) Normalization of peptide coverage and application Cassie's Law. Young's equation indicates that the surface energy varies linearly with the cosine of the contact angle:",{"@attributes":{"id":"p-0447","num":"0498"},"maths":{"@attributes":{"id":"MATH-US-00003","num":"00003"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mrow":{"mi":["cos","\u03b8"],"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}}},"mo":"=","mfrac":{"mrow":{"msub":[{"mi":["\u03b3","SV"]},{"mi":["\u03b3","SL"]}],"mo":"-"},"msub":{"mi":["\u03b3","LV"]}}}}}},"It is possible to model the contact angle of a more complex surface by linear addition of the weighted contributions of their constituents, as is shown by Cassie's law.\n\ncos \u03b8=\u03c6cos \u03b8+\u03c6cos \u03b8\n\nwhere \u03c6and \u03b8are the coverage and surface energy associated with each constituent respectively, it is possible, therefore, to normalize the contact angles with respect to coverage of each peptide, and to extrapolate the hydropathy of a theoretical fully covered surface, as long as the two displayed phases have uniform and distinct properties and no superhydrophobic effect is taking place.\n","Samples of varying coverage were prepared to contain either the ordered or disordered phases as determined by AFM and subjected to contact angle measurement. The relationship between the coverage and cosine of the contact angle was linear over the range of coverages produced, indicating that no superhydrophobicity was occurring, justifying the assumptions of Cassie's law.","2) Liquid surface tension data. Interfacial tension was calculated from at least 3 pendant drop shapes. The results indicated that there was no significant difference in the interfacial energies of various peptide solutions and that the contact angle measurements, therefore, could be compared directly.",{"@attributes":{"id":"p-0451","num":"0000"},"ul":{"@attributes":{"id":"ul0009","list-style":"none"},"li":["1. Chen, D.; Wang, G.; Li, J. H. Interfacial bioelectrochemistry: Fabrication, properties and applications of functional nanostructured biointerfaces. 2007, 111, 2351-2367.","2. Nel, A. E.; Madler, L.; Velegol, D.; Xia, T.; Hoek, E. M. V.; Somasundaran, P.; Klaessig, F.; Castranova, V.; Thompson, M. Understanding Biophysicochemical Interactions at the Nano-bio Interface. 2009, 8, 543-557.","3. Cha, T.; Guo, A.; Zhu, X. Y. Enzymatic Activity on a Chip: The Critical Role of Protein Orientation. 2005, 5, 416-419.","4. Tamerler, C.; Khatayevich, D.; Gungormus, M.; Kacar, T.; Oren, E. E.; Hnilova, M.; Sarikaya, M. Molecular Biomimetics: GEPI-Based Biological Routes to Technology. 2010, 94, 78-94.","5. Shiba, K. Exploitation of Peptide Motif Sequences and their Use in Nanobiotechnology. 2010, 21, 412-425.","6. Peelle, B. R.; Krauland, E. M.; Wittrup, K. D.; Belcher, A. M. Design Criteria for Engineering Inorganic Material-Specific Peptides. 2005, 21, 6929-6933.","7. Meyers, S. R.; Khoo, X. J.; Huang, X.; Walsh, E. B.; Grinstaff, M. W.; Kenan, D. J. The Development of Peptide-Based Interfacial Biomaterials for Generating Biological Functionality on the Surface of Bioinert Materials. 2009, 30, 277-286.","8. Sarikaya, M.; Tamerler, C.; Jen, A. K. Y.; Schulten, K.; Baneyx, F. Molecular Biomimetics: Nanotechnology Through Biology. 2003, 2, 577-585.","9. Wei, J. H.; Kacar, T.; Tamerler, C.; Sarikaya, M.; Ginger, D. S, Nanopatterning Peptides as Bifunctional Inks for Templated Assembly. 2009, 5, 689-693.","10. Ulman, A. Formation and Structure of Self-Assembled Monolayers. 1996, 96, 1533-1554.","11. Gawalt, E. S; Avaltroni, M. J.; Koch, N.; Schwartz, J. Self-Assembly and Bonding of Alkanephosphonic Acids on the Native Oxide Surface of Titanium. 2001, 17, 5736-5738.","12. Hnilova, M.; Oren, O. O.; Seker, U. O. S.; Wilson, B. R.; Collino, S.; Evans, J. S.; Tamerler, C.; Sarikaya, M. Effect of Molecular Conformations on the Adsorption Behavior of Gold-Binding Peptides. 2008, 24, 12440-12445.","13. Tamerler, C.; Sarikaya, M. Molecular Biomimetics: Nanotechnology and Bionanotechnology Using Genetically Engineered Peptides. 2009, 367 1705-1726.","14. Khatayevich, D.; Gungormus, M.; Yazici, H.; So, C.; Cetinel, S.; Ma, H.; Jen, A.; Tamerler, C.; Sarikaya, M. Biofunctionalization of Materials for Implants Using Engineered Peptides. 2010, 6, 4634-4641.","15. Yuca, E.; Karatas, A. Y.; Seker, U. O, S.; Gungormus, M.; Dinler-Doganay, G.; Sarikaya, M.; Tamerler, C. In Vitro Labeling of Hydroxyapatite Minerals by an Engineered Protein. 2011, 108, 1021-1030.","16. Hu, W. B.; Peng, C.; Luo, W. J.; Lv, M.; Li, X. M.; Li, D.; Huang, Q.; Fan, C. H. Graphene-Based Antibacterial Paper. 2010, 4, 4317-4323.","17. Balandin, A. A.; Ghosh, S.; Bao, W. Z.; Calizo, I.; Teweldebrhan, D.; Miao, F.; Lau, C. N. Superior Thermal Conductivity of Single-Layer Graphene. 2008, 8, 902-907.","18. Castro Neto, A. H.; Guinea, F.; Peres, N. M. R.; Novoselov, K. S.; Geim, A. K. The Electronic Properties of Graphene. 3 2009, 81, 109-162.","19. Dai, H. J. Carbon Nanotubes: Synthesis, Integration, and Properties. 2002, 35, 1035-1044.","20. Novoselov, K. S.; Geim, A. K.; Morozov, S. V.; Jiang, D.; Zhang, Y.; Dubonos, S. V.; Grigorieva, I. V.; Firsov, A. A. Electric Field Effect in Atomically Thin Carbon Films. 2004, 306, 666-669.","21. Gorton, L. ; Elsevier Science: Amsterdam, Netherlands, 2005.","22. Du, D.; Zou, Z. X.; Shin, Y. S.; Wang, J.; Wu, H.; Engelhard, M. H.; Liu, J.; Aksay, I. A.; Lin, Y. H. Sensitive Immunosensor for Cancer Biomarker Based on Dual Signal Amplification Strategy of Graphene Sheets and Multienzyme Functionalized Carbon Nanospheres. 2010, 82, 2989-2995.","23. Lin, Y. H.; Lu, F.; Tu, Y.; Ren, Z. F. Glucose Biosensors Based on Carbon Nanotube Nanoelectrode Ensembles. 2004, 4, 191-195.","24. Li, D.; Muller, M. B.; Gilje, S.; Kaner, R. B.; Wallace, G. G. Processable Aqueous Dispersions of Graphene Nanosheets. 2008, 3, 101-105.","25. Niyogi, S.; Bekyarova, E.; Itkis, M. E.; McWilliams, J. L.; Hamon, M. A.; Haddon, R. C. Solution Properties of Graphite and Graphene. 2006, 128, 7720-7721.","26. Choi, E. Y.; Han, T. H.; Hong, J. H.; Kim, J. E.; Lee, S. H.; Kim, H. W.; Kim, S. O, Noncovalent Functionalization of Graphene with End-Functional Polymers. 2010, 20, 1907-1912.","27. Ghosh, A.; Rao, K. V.; Voggu, R.; George, S. J. Non-Covalent Functionalization, Solubilization of Graphene and Single-Walled Carbon Nanotubes with Aromatic Donor and Acceptor Molecules. 2010, 488, 198-201.","28. Yu, X.; Wang, Z. Q.; Jiang, Y. G.; Zhang, X. Surface Gradient Material: From Superhydrophobicity to Superhydrophilicity. 2006, 22, 4483-4486.","29. Yang, H.; Fung, S.; Sun, W.; Mikkelsen, S.; Pritzker, M.; Chen, P. Ionic-Complementary Peptide-Modified Highly Ordered Pyrolytic Graphite Electrode for Biosensor Application. 2008, 24, 964-971.","30. Cui, Y.; Kim, S. N.; Jones, S. E.; Wissler, L. L.; Naik, R. R.; McAlpine, M. C. Chemical Functionalization of Graphene Enabled by Phage Displayed Peptides. 2010, 10, 4559-4565.","31. Wang, Q. H.; Hersam, M. C. Room-Temperature Molecular-Resolution Characterization of Self-Assembled Organic Monolayers on Epitaxial Graphene. 2009, 1, 206-211.","32. Kim, S. N.; Kuang, Z.; Slocik, J. M.; Jones, S. E.; Cui, Y.; Farmer, B. L.; McAlpine, M. C.; Naik, R. R. Preferential Binding of Peptides to Graphene Edges and Planes. 2011, 133, 14480-14483.","33. Kase, D.; Kulp, J. L.; Yudasaka, M.; Evans, J. S.; Iijima, S.; Shiba, K. Affinity Selection of Peptide Phage Libraries Against Single-Wall Carbon Nanohorns Identifies a Peptide Aptamer with Conformational Variability. 2004, 20, 8939-8941.","34. Tomasio, S. M.; Walsh, T. R. Modeling the Binding Affinity of Peptides for Graphitic Surfaces. Influences of Aromatic Content and Interfacial Shape. 2009, 113, 8778-8785.","35. Han, T. H.; Lee, W. J.; Lee, D. H., Kim, J. E., Choi, E., Kim, S. O. Peptide\/Graphene Hybrid Assembly into Core\/Shell Nanowires. 2010, 22, 2060-2064.","36. So, C. R.; Hayamizu, Y.; Yazici, H.; Gresswell, C.; Khatayevich, D.; Tamerler, C.; Sarikaya M. Controlling Self-Assembly of Engineered Peptides on Graphite by Rational Mutation. 2012, 6, 1648-1656","37. Cassie, A. B. D.; Baxter, S. Wettability of Porous Surfaces. 1944, 40, 0546-0550.","38. Kacar, T.; Zin. M. T.; So, C. R.; Wilson, B. R.; Ma, H.; Gul-Karaguler, N.; Jen, A. K-Y.; Sarikaya, M.; Tamerler, C. Directed Self-Immobilization of Alkaline Phosphatase on Micro-Patterned Substrates Via Genetically Fused Metal-Binding Peptide. 2009, 103, 696-705.","39. Hall Sedlak, R.; Hnilova, M.; Gachelet, E.; Przybylal, L.; Dranow, D.; Gonen, T.; Sarikaya, M.; Tamerler, C.; Traxler, B. An Engineered DNA-Binding Protein Self-assembles Metallic Nanostructures. 2010, 11, 2108-2112.","40. Hnilova, M.; Khatayevich, D.; Carlson, A.; Oren, E. E.; Gresswell, C.;, S. Zheng; Ohuchi, F.; Sarikaya, M.; Tamerler, C. Single-Step Fabrication of Patterned Gold Film Array by an Engineered Multi-Functional Peptide. 2012, 365, 97-102."]}},"We show that two-dimensional supramolecular patterns of engineered dodecapeptides modify the electrical conductivity of graphene via biomolecular doping. Observed conductivity of graphene field effect transistors covered by self-assembled peptides indicates a formation of electronic junctions. Peptides also modify the carrier mobility in the graphene. While ordered peptides maintain the carrier mobility of graphene, randomly adsorbed peptides significantly reduce it. Engineered peptides have significant promise as novel molecular tools to link electronics of single layer atomic materials with biology.","Self-assembly of biomolecules on nanomaterials potentially offers a novel bottom-up technology to form bio-nano hybrid systems toward development of bio-nanoelectronic devices. In particular, proteins are auspicious building blocks for the assembly of hierarchical structures due to their diversity of conformation and chemistry based on simple amino acid sequences. Graphene can be an ideal platform for the self-assembly because of the atomically flat two-dimensional (2D) surface and rich physical properties, e.g., linear energy dispersion, high carrier mobility, and high mechanical strength. Recent studies on graphene electronics have been focused on the surface functionalization modifying electronic characteristics of graphene directly through interactions with adsorbed molecules, i.e. chemical doping. Although not yet explored, doping strategies via molecular self-assembly may be extended to proteins having both characteristics of the doping and their spatial control by using their exquisite nanostructures. Even more intriguing is the ability to control interfaces between biomolecule and the single layer atomic material via the molecular recognition of graphene by engineered peptides. Interface control is the key to develop novel bioelectronic sensors with high sensitivity and intricate spatial configurations at the nanoscale. For design of the bio\/nano interfaces, one needs to probe electrical interactions that arise between the peptides and graphene, where the conformation and chemistry of the peptides are expected to play the most significant role.","Peptides can be a versatile molecular tool to address the challenge of self-assembly and molecular doping on graphene as they are simpler (shorter in size and simpler in chemistry) than natural proteins, and yet they have a wide-range of chemistry and molecular conformations inherent in their amino acid sequences. Engineered peptides have been shown to self-assemble into supramolecular structures on atomically flat surfaces, such as on mica, Au(111) and graphite. These peptides exhibit long-range ordered structures template and directed by the atomic lattice of the underlying surfaces, indicating that peptides have a potential to directly functionalize graphene via molecular recognition to solid surfaces. Similar to the epitaxial crystal growth, peptides forms ordered nanostructures on solid surfaces through a series of surface phenomena: binding, diffusion, and ordering. Unlike randomly immobilized biomolecules on solid surfaces, long-range ordered peptides preserve the sufficient energy balance essential among their binding, diffusion, and conformational changes. Recently, peptides have been demonstrated to modify graphene conductivity, although they do not form ordered structures on a graphene. Uniform alignment and conformations of peptides in their ordered structures could be the key to controllably modify electrical properties of bio\/nano interfaces.","In this work, we use to form supramolecular ordered structures and investigate its effect on the electronic properties of the graphene. We demonstrate that dodecapeptides of the present invention self-assemble into 2D supramolecular network modifying electric properties of graphene that are correlated to their morphology on the graphene. The abrupt boundaries of ordered peptide supramolecules create electronic junctions via spatial biomolecular doping, manifested as a self-assembled electronic network. We also utilize GrBP5 mutants without the ability to form ordered structures on the graphene, as control experiments. While ordered peptides of GrBP5 do not change the mobility of graphene, randomly absorbed mutant peptides significantly reduce it. The biomolecular doping of graphene, spatially defined by peptide supramolecular nanostructures, represents the first step in integrating biology with nano-electronics towards realizing fully self-assembled bio-nanoelectronics systems.","To study the self-assembly of GrBP5 on graphene, mechanically exfoliated graphene are incubated with peptides in degased DI water (500 nM for 20 minutes, Supplementary Method). After the incubation, the sample is quickly dried by nitrogen blow. Atomic force microscopy (AFM) imaging revealed that peptides self-assemble into ordered structures, \u201csupramolecular nanowires\u201d, () with six-fold symmetry on single-layer graphene (). The longitudinal length of nanowires ranges up to micrometers, while the observed average thickness is 1.10 nm with a minimal width of about 12.00 nm (). Since the length of the straightened (non-folded) GrBP5 molecule is about 3 nm, the observations suggest that the peptides have folded into packed conformation forming a monomolecular thickness throughout their self-assembled nanostructures, similar to that seen with graphite. It is also worth to mention that there are no peptides observed on SiO, indicating that peptides bind specifically to the graphene.","Peptide self-assembly may provide a novel way to modify electrical properties of graphene, where ordered peptides locally dope graphene with charge carriers. To verify this hypothesis, investigation was geared towards correlating the assembled morphologies of the peptides and the measured conductivities of the graphene field effect transistor (GFET). As depicted in , special care was taken for reliable characterization of graphene conductivity induced by the ordered peptides. First, GFETs are fabricated by direct soldering of indium contacts to an exfoliated graphene on a SiO\/Si wafer. We employed this resist-free process instead of a conventional lithography, because of the difficulty to completely remove residues from the resists that disrupt peptide self-assembly into long-range ordered structures. Second, we directly place a droplet of peptide aqueous solution on GFETs to prevent contamination. In the solution process, water molecules are known to significantly change the conductivity of the graphene on SiOin the presence of oxygen. To prevent the water effect, we utilize deoxygenated DI water prepared by bubbling with nitrogen or argon gases (Supplementary Method). All the conductivity measurements, therefore, have been performed in argon atmosphere to eliminate the water effect.","In this work, we characterized conductivities of more than ten GFETs before and after the peptide incubation. Each GFET has various morphologies of peptides controlled by the incubation time and concentration of peptide solutions. AFM images exhibit a gradual growth of supramolecular peptides from disordered islands to nanowires with increasing coverage (), and conductivities of GFETs are clearly modulated by peptides (). The gate-voltage was cyclically swept over a range from 0V to maximum, from maximum to minimum, and back to 0V. All as-prepared GFETs have symmetric conductivity curves with a charge neutral point (CNP) at around 20V, i.e., the voltage with minimum conductivity. GFETs with peptide coverage of 26, 40, and 48% display two distinct dips in their conductivities. As coverage increases, the left-dip gradually disappears and the right-dip becomes dominant. At 88% coverage, the CNP shifts up to +85V, indicating significant hole doping by peptides. The observation of two dips is unique in our supramolecular peptides on graphene compared with previous reports on the chemical doping of graphene, where a CNP monotonically shifts without displaying another dip. In these published cases, potassium atoms or organic molecules randomly deposit on graphene surface. On the other hand, two minima in conductivity have also been observed in GFETs with micro-patterned chemical dopantswhere graphene is selectively covered partially by dopants and other regions remain uncovered. Based on these previous findings, our observations indicate that the two conductivity minima, on the left and the right, can be attributed to uncovered and peptide-covered regions in graphene, respectively. This result suggests formation of electronic junctions in graphene, where charge carriers of holes accumulate underneath peptides via biomolecular doping. The first time observation that there are two minima (i.e., electronic junction) in graphene functionalized by self-assembled peptides is highly significant as they form without the use of a top-down technique such as lithography shown in the literature. Additionally, the conductivity shows no hysteresis over the cyclic sweep of the gate voltage, which indicates that: (1) Peptides are rigidly immobilized and stable even under the electric field, and (2) In our measurement conditions, water effects were not observed, which usually appears as a hysteresis in the cyclic measurement of the conductivity.","AFM images in  show that disordered (islands) and ordered peptides (nanowires) coexist at low coverage of 26 and 40%, while nanowires are dominant at 48% coverage, a possible threshold. Here, a question arises as to which of the structures (random islands or ordered nanowires) of the peptides mainly contribute to the formation of two dips observed in the graphene conductivity. To address this question, we performed a control experiment with a mutant peptide, M2, which only forms disordered, or random, nanostructures on graphene. The mutant M2 has slightly modified sequence (IMVTESSDWSSW (SEQ ID NO: 7)), where two tyrosines (Y) in the anchoring domain of the wild-type GrBP5 sequence (WT) are replaced by tryptophans (W). In a previous study, it was found that the two tyrosines play an essential role in the binding of the peptide to graphite substrate and that the mutant M2 has random adsorption on graphite due to relatively small surface diffusion caused by the two tryptophans. In this work, AFM studies show that M2 is again randomly absorbed on graphene (). In the subsequent conductivity measurement of GFETs with M2, a single dip, rather than two dips, was observed and the CNP shifts up to 75V as the coverage increases to 92%. This result conclusively confirms that the random adsorption of peptides does not form abrupt boundaries between the covered and uncovered regions, resulting in an averaged doping of the graphene with holes. Thus, one could conclude that the observed two dips in  are originated from the ordered peptides creating abrupt boundaries at the edges of the ordered nanostructures.","To interrogate the effects of the peptides more quantitatively, we determine the charge carrier density and mobility from  and  (Supplementary Methods). From the shifts of CNPs, the charge carrier density induced by peptides onto graphene can be estimated over a range of coverage (). First to notice is that both the WT and M2 show monotonic increase of the hole density as the peptide coverage on the graphene surface increases. WT has slightly higher carrier density than that of M2; this effect might be caused either by relatively strong interactions of WT peptides with graphene or high density of WT peptides on graphene. In contrast, the mobility of the graphene vs. the peptide coverage shows clear distinct effects between WT and M2 (). The mobility was estimated from the slope of the linear region on the left side of conductivity curves. In the case of WT, the mobility decreases from 2,900 cm\/Vs of as-prepared GFETs to 500 cm\/Vs at the low coverage of 20%. This observation indicates that peptide islands may induce charge impurities into graphene. Such a reduction in mobility has been previously observed by the depositing potassium atoms, self-assembled monolayers, and micelleson graphene. As the coverage of peptides increases, the mobility recovers gradually. This trend implies that the growth of ordered structures significantly reduces the carrier scattering in graphene. Enhancements of the carrier mobility have been recently observed by several studies employing high dielectric media on graphene, such as a high-\u03ba materialand ionic solutions, where the dielectric media partially screens charge impurities in graphene. Our observations with the WT peptides can also be interpreted as being caused by the screening effect from the densely-packed peptides, which are uniformly aligned on the graphene substrate. In contrast, the mobility change by M2 shows a monotonic decrease as the coverage of disordered peptides increases. This observation is similar to the reduction of the mobility by deposited materials, as seen in previous studies, 24, 25 indicating that randomly adsorbed peptides induce charge impurities which results in carrier scattering even at the high coverage regime.","This trend has meaningful correlations with the peptide morphology on the graphene. Analyzing the AFM images closely, one notices that the height of peptides vs. the coverage () shows a tendency that is different in WT than in M2. While M2 shows a monotonic increase of the height as the coverage increases, WT reveals threshold coverage at around 50%, above which value the height and its standard deviation decreases. The non-monotonic change of the height and its distribution could be an indication that WT forms ordered nanostructures with a close-packed organization. In contrast, the monotonic change in the M2 could indicate that M2 peptides form random conformations resulting in crowding, rather than ordering, on the surface. In , the mobility change caused by peptide vs. the peptide coverage is plotted. The mobility change is calculated by \u0394\u03bc=(\u03bc\u2212\u03bc)\/n, where \u03bc, \u03bc, and n are the mobility of as-prepared GFETs, mobility modified by peptides, and charge carrier density induced by peptides, respectively. The mobility change is normalized by the carrier density over the peptide coverage, which roughly provides us information of how highly an induced carrier by peptides reduces the mobility of holes in a graphene. This mobility change shows a similar threshold of the coverage to the one in the height change. While M2 shows a nearly-constant change of mobilities over the peptide coverage, WT shows a monotonic change of the mobility reduction. Interestingly, the fitted line of WT intersects the one of M2 at around the threshold coverage of 50%. The reduction of mobilities by WT is higher than M2 in the low coverage regime, but it becomes less in the high coverage regime. These observations suggest that the electrical interactions of WT peptides with the graphene cause the carrier scattering more than M2 at the low coverage regime, but the scattering becomes weaker once WT peptides form ordered structures above the threshold coverage.","Although there is no experimental work evaluating electrical interactions of every amino acid to the graphene in a quantitative manner, both of the aromatic and charged amino acids are promising candidates with meaningful effects on the graphene conductivity as suggested by previous works including carbon nanotubes. Our peptides in this work have two aromatic amino acids (W and Y) and two charged amino acids (D and E). W and Y have isoelectric points (IP) of 5.7 and 5.9, respectively, which possibly cause moderate electric interactions with graphene in the neutral condition of water. On the other hand, charged amino acids, aspartic acid (D) and glutamic acid (E), have respectively 2.8 and 3.2, and their electrical interactions can be stronger than the aromatic amino acids. In our previous work, we found that the aromatic amino acids are the binding site to the graphite. Thus, the distance of the aromatic residues of W and Y to the graphene could be similar. However the distance of the charged amino acids could be varied by the conformation of peptides significantly, which might be the major difference between WT and M2. Ordered structures of peptides may periodically place these charged amino acids, leading the less carrier scattering.","Our approach demonstrated here to modify local electronic properties of graphenes by self-assembled peptides can be used in a wide range of potential fundamental research and practical implementations. For example, a supramolecular array of peptides can provide two-dimensional scaffolds for functional proteins in forming nanoscale bioelectronic circuits on graphenes without a significant disturbance of the carrier mobility, which may facilitate new applications in biosensing or bioenergy generation. The change of the graphene mobility by absorbed peptides can be useful to detect conformation changes of peptides or proteins on graphenes.",{"@attributes":{"id":"p-0464","num":"0000"},"ul":{"@attributes":{"id":"ul0010","list-style":"none"},"li":["1. Kowalewski, T. and Holtzman, D. M. In situ atomic force microscopy study of Alzheimer's \u03b2-amyloid peptide on different substrates: New insights into mechanism of \u03b2-sheet formation. 96, 3688 (1999).","2. Zhang, F. et al. Epitaxial growth of peptide nanofilaments on inorganic surfaces: effects of interfacial hydrophobicity\/hydrophilicity. 45, 3611 (2006).","3. So, C. R. et al. Molecular recognition and supramolecular self-assembly of a genetically engineered gold binding peptide on Au {1113, 1525 (2009).","4. So, C. R. et al. Controlled self-assembly of peptides on graphite via rational mutation. (2011).","5. Katz, E. and Willner I. Integrated nanoparticle-biomolecule hybrid systems: synthesis, properties, and applications. 43, 6042 (2004).","6. Offenh\u00e4usser, A. and Rinaldi, R. (Springer-Verlag, New York, 2009).","7. Novoselov, K. S. et al. Electric field effect in atomically thin carbon films. 306, 666 (2004).","8. Zhang, Y., Tan, J. W., Stormer, H. L. and Kim, P. Experimental observation of the quantum Hall effect and Berry's phase in graphene. 438, 201 (2005).","9. Bolotin, K. I. et al. Ultrahigh electron mobility in suspended graphene. 146, 351 (2008).","10. Lee, C., Wei, X., Kysar, J. W. and Hone, J. Measurement of the elastic properties and intrinsic strength of monolayer graphene, 321, 385 (2008).","11. Schedin, F. et al. Detection of individual gas molecules adsorbed on graphene. 6, 652 (2007).","12. Dong, X. et al. Doping single-layer graphene with aromatic molecules. 5, 1422 (2009).","13. Wang, Q. H. and Hersam, M. C. Room-temperature molecular-resolution characterization of self-assembled organic monolayers on epitaxial graphene. 1, 206 (2009).","14. Pollard, A. J. et al. Supramolecular assemblies formed on an epitaxial graphene superstructure, 49, 1794 (2010).","15. Wang, X., Tabakman, S. M. and Dai, H. Atomic layer deposition of metal oxides on pristine and functionalized. 130, 8152 (2008).","16. Barth, J. V., Costantini, G. and Kern, K. Engineering atomic and molecular nanostructures at surfaces. 437, 671 (2005).","17. Cui, Y. et al. Chemical functionalization of graphene enabled by phage displayed peptides. 10, 4559 (2010).","18. Sarikaya, M. et al. Molecular biomimetics: nanotechnology through biology. 2, 577 (2003).","19. Girita, C. O. and Zettl, A., Soldering to a single atomic layer. 91, 193512 (2007).","20. Levesque, P. L. et al. Probing charge transfer at surfaces using graphene transistors. 11, 132 (2011).","21. Chen, J. H. et al. Charged-impurity scattering in graphene. 4, 377 (2008).","22. Farmer, D. B., Lin, Y. M., Afzali-Ardakani, A. and Avouris, P. Behavior of a chemically doped graphene junction. 94, 213106 (2009).","23. Cheng, H. C. et al. high-quality graphene p-n junctions via resist-free fabrication and solution-based noncovalent functionalization. 5, 2051 (2011).","24. Lee, B. et al. Modification of electronic properties of graphene with self-assembled monolayers. 10, 2427 (2010).","25. Shih, C. J., et al. Understanding surfactant\/graphene interactions using a graphene field effect transistor: relating molecular structure to hysteresis and carrier mobility. 22, 8579 (2012).","26. Chen, F.; Xia, J.; Ferry, D. K.; Tao, N. Dielectric screening enhanced performance in graphene FET. 2009, 9, 2571.","27. Chen, F.; Xia, J.; Tao, N. Ionic screening of charged-impurity scattering in graphene. 2009, 9, 1621.","28. Allen, B. L.; Kichambare, D. P.; Star, A. Carbon nanotube field-effect-transistor-based biosensors. 2007, 19, 1439.","29. Tomasio, S. M. and Walsh, T. R. Modeling the binding affinity of peptides for graphitic surfaces. influences of aromatic content and interfacial shape, 113, 8778 (2009).","30. Rajesh, C., Majumder, C., Mizuseki, H. and Kawazoe, Y. A theoretical study on the interaction of aromatic amino acids with graphene and single walled carbon nanotube. 130, 124911 (2009)."]}},"S1. Peptide Synthesis","Peptide synthesis was carried out on a preloaded support resin using HBTU activation chemistry, while 20% piperidine in DMF was employed to afford the Fmoc deprotection and monitored by UV absorbance at 301 nm. Following solid-phase synthesis, the peptides were cleaved off the support and side chain deprotected by stirring the resin-bound peptide in a cocktail under Natmosphere for 2-3 hours. This cocktail contained either TFA\/thioanisole\/HO\/phenol\/ethanedithiol (87.5:5:5:2.5) or TFA\/triisopropylsilane\/HO\/EDT (94:1:2.5:2.5), depending on the peptide. The resin was removed by filtration, and each of the peptides was precipitated with cold ether to yield crude peptide product that was lyophilized (Virtis Benchtop K, SP Industries, Inc., Warminster, Pa.). Peptides were reconstituted using various ratios of DI water and acetonitrile. Purification by reverse-phase HPLC for peptides employed, first, an isocratic (0% B for 2 min) and, then, a linear gradient of 1%\/min for analytical and 0.5%\/min for semiprep scales at 1 and 10 mL\/min flow rates, respectively. Retention times spanned 30-50 mins depending on the peptide in semi-preparative HPLC. Analytical peaks were isolated by autothreshold collection (Waters Deltaprep 600, analytical mode) and peptides were verified by MALDI-TOF mass spectrometry with reflectron (RETOF-MS) on an Autoflex II (Bruker Daltonics, Billerica, Mass.) mass spectrometer in positive-ion mode. The observed M\/Z fractions were subsequently collected manually from a scaled semi-preparative separation (Waters Deltaprep 600, semiprep. mode).","S2. Sample Preparation for Atomic Force Microscope Measurements","Graphene was mechanically exfoliated from natural graphite flakes (3763, Asbury Carbons) on a Si wafer with 300 nm SiO(1). The number of graphene layers was confirmed by Raman spectroscopy and atomic force microscope (AFM). Single-layer MoSwas also mechanically exfoliated from a bulk MoS(Moly Disulfide, SPI Supplies). Single MoSlayers were confirmed by AFM and photoluminescence (PL) measurements. After exfoliation, samples were incubated with peptides in DI water. A sample for  was incubated under the condition of 500 nM peptide concentration for 20 minutes. All samples for  were prepared under the condition of 1 \u03bcM for 1 hour. After the incubation, samples were gently dried with nitrogen blow.","S3. Characterization of GrBP5 and its Mutants, M6 and M8, on Bulk Graphite and MoS","To roughly estimate binding affinities of each peptide (WT, M6 and M8) to surfaces, we characterized peptides on bulk graphite and MoSby AFM. Flakes of graphite and MoS(same suppliers in the section 2) on Si wafers with 300 nm-thick SiOwere incubated with peptides in DI wafer for 3 hours. The concentrations of peptides were 0.1, 1, and 5 \u03bcM. After the incubation, samples were gently dried under Nitrogen gas. Interestingly, the results show all peptides capable of forming long-range ordered nanowires at 0.1 \u03bcM on both graphite and MoS, except for M8 on graphite, which instead forms fiber-like structures with loose alignment (data not shown for graphite; MoSdata shown in ). At higher concentrations, all peptides show domains with aligned structures inside, except for M8 on graphite with randomly distributed domains. The distinctive behavior of M8 on graphite was also observed in its binding affinity. The binding affinities of peptides to graphite and MoSwere estimated from a relationship between the observed surface coverage of peptides by AFM and the peptide concentrations used for incubation. () Here, we employed a simple adsorption equation which allows us to roughly estimate the binding affinity, as described in Example 1 Supporting information.","The binding affinities obtained by fitting the adsorption equation with experimental data show that M8 has the exceptionally large binding affinity to graphite. On the other hand, other peptides, WT and M6, have relatively weak binding affinities. In fact, aligned peptide nanowires on graphite were already observed in previous reports, which characterized surface morphology of peptides on graphite at various conditions (14,16). These report suggested that binding, diffusion and intermolecular interactions of peptides play important roles during their self-assembly process into long-range ordered structures. It also suggested that strong interactions between peptides and the surface results in suppressed surface-diffusion rates and, therefore, inhibits peptide self-assembly into ordered structures. In light of such findings, our observations indicate that M8 has strong interactions with graphite surfaces and therefore an apparent suppressed surface diffusion. On the other hand, WT and M6 may have optimal interactions with the surface, readily forming self-assembled structures. Also notable is the contrast in peptide binding specificity to particular materials that arises from modified primary amino acid sequences. While M6 has weak but comparable binding affinities to both graphite and MoS, WT and M8 show opposing trends in binding affinity to graphite and MoS. While WT binds strongly to MoS, M8 binds weakly. In the case of graphite, however, this tendency is reversed. The opposing material selectivity may be due to the difference in electric potential of graphite and MoSat the interface between surface and peptide solution. WT and M8, with their opposing mean charge, might therefore show contrasting tendencies in their material selectivity. At the same time, they may have a higher sensitivity to the electric potential of surfaces over M6. Although these hypotheses need to be further investigated, the significant effect on peptide binding affinity after rational modification of the amino acid sequence sheds light on tuning supramolecular peptide self-assembly on multiple materials.","S4. Fabrication of Field Effect Transistors (FET) and Electrical Characterization","Graphene was exfoliated from natural graphite flakes (3763, Asbury Carbons) on a doped Si wafer, covered with 300 nm SiO, by the conventional mechanical exfoliation method (1). Single layers of graphene were identified by Raman spectroscopy and AFM. In this study, the cleanliness of graphene surfaces was vital in order to observe supramolecular assembly of peptides. In conventional lithographic fabrication of graphene FETs (GFET), resists often remain on graphene surfaces even after rigorous cleaning processes. Such contamination was found to cause undesired doping effects in the ASL and also disrupt the self-assembly of peptides. To avoid such contamination, we utilized micro-soldering of Indium (24). After the mechanical exfoliation of graphene, we immediately formed indium electrodes on graphene. The Indium electrodes were also manually covered by PMMA (495, MicroChem Corp.) to minimize their electrochemical disturbance to graphene and self-assembly of peptides during incubation. Peptides were immobilized by placing 20 \u03bcL droplets of peptide solution at various concentrations onto a GFET chip. Samples were kept under humidity control (>80%) to prevent drying during their incubation. Then, GFETs were rinsed with DI water promptly and dried with nitrogen flow. The details of the incubation conditions for the imaged GFETs in  are as follows; Sample 1: \u03b8=26%, 50 nM for 20 minutes; Sample 2: \u03b8=40%, 50 nM for 60 minutes; Sample 3: \u03b8=50%, 200 nM for 90 minutes; Sample 4: \u03b8=88%, 500 nM for 20 minutes.","Electric measurements were carried out on two-probe measurements with a semiconductor characterization system (4200-SCS, Keithley) and a probe station (Singatone Corp., Gilroy, Calif.). The conductivity of GFETs in  was normalized using the dimensions of GFETs. The typical size of the graphene sheet between two electric contacts is 40 \u03bcm in width and 20 \u03bcm in length. All electrical measurement was performed under Argon atmosphere to eliminate instability caused by moisture. In gate response measurements, the typical source-drain voltage was 5 mV. In our study, contact resistance of indium electrodes was small compared with resistance of graphene, because of the large contact area, typically 40 \u03bcm by 10 \u03bcm. Typically, resistance of GFET is in the order of kilo ohms, while contact resistance is sub ohms Plots of sheet resistance vs. gate voltages also show small contribution of contact resistance (not shown). Before the incubation with peptides, as-prepared GFETs shows charge neutral point (CNP) at around 20V on average. The CNPs depends on a sample and range from 15V to 25V (not shown). During the conductivity measurement, the gate voltages were cyclically swept from minimal to maximal voltage, and then, to minimal voltage again. Even after incubation with peptides, GFETs did not show hysteresis in cyclic gate voltage measurements. It is worth to note that GFETs incubated with DI water showed shifts of CNP (10V after 10-minuites incubation). However, GFETs incubated with degassed DI water (bubbled with Ar gas for 30 minutes) did not show significant CNP shifts. It indicates that oxygen dissolved in water might cause doping. In this work, we used degassed peptide solution (bubbled with Ar gas for 30 minutes) in the incubation with GFETs to eliminate excessive doping.","In the case of MoS, due to the difficulty of mechanically exfoliating large single-layer flakes, we employed electron beam lithography to fabricate electrodes over the previous micro-soldering approach. For the lithography, we used PMMA (495, MicroChem. Corp.) as a resist, and exposed it using a scanning electron microscope (SEM7000, JEOL). After development in MIBK:IPA 1:3 solution (developer, Microchem) for 3 min, we sputtered Ti\/Pt (5 nm\/25 nm) as electrodes, and lifted off by boiled acetone and isopropyl alcohol. All FET devices were cleaned by following three cleaning processes; 1) Rinsing with boiled acetone for 30 minutes, 2) Cleaning with UV\/ozone (UV\/Ozone ProCleaner, BioForce Nanosciences Inc.) for 1 minute, 3) Current induced annealing (ref) under argon gas atmosphere. We confirmed that we can regenerate the conductivity of all three FET devices using this cleaning process even after the incubation with peptides. In this study, we utilized a same FET platform to directly compare modification of MoSconductivity by peptides, M6 and M8. All peptides were incubated with the MoSFET under a condition of 1 \u03bcM for 1 hour. After the incubation, peptide solution was gently dried by nitrogen. The conductivity measurement was performed under argon gas. Source-drain voltage was 100 mV for gate response measurements.","S5. Estimation of Charge Carrier Density and Field-Effect Mobility","The charge carrier density in single-layer graphene was estimated from Vobserved in  using a simple relationship",{"@attributes":{"id":"p-0473","num":"0592"},"maths":{"@attributes":{"id":"MATH-US-00004","num":"00004"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mrow":{"mi":"n","mo":"=","mrow":{"msub":{"mi":["C","i"]},"mo":"\u00d7","mrow":{"msub":{"mi":["V","cnp"]},"mo":"\/","mi":"e"}}},"mo":","}}},"br":{},"sub":"i "},{"@attributes":{"id":"p-0474","num":"0593"},"maths":{"@attributes":{"id":"MATH-US-00005","num":"00005"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mstyle":{"mspace":{"@attributes":{"width":"1.1em","height":"1.1ex"}}},"mo":"\u2062","mrow":{"mrow":{"mo":["(",")"],"mrow":{"mrow":[{"mrow":[{"mrow":{"msub":{"mi":["C","i"]},"mo":"=","mrow":{"msub":{"mi":"\u025b","mn":"2"},"mo":"\u2062","mrow":{"msub":{"mi":["\u025b","r"]},"mo":"\/","mi":"d"}}},"mo":";"},{"munder":{"mi":["\u025b","r"]},"mo":"=","mn":"3.9"}],"mo":"="},{"mi":"d","mo":"=","mrow":{"mn":"300","mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}},"mi":"nm"}}],"mo":","}},"mo":","}}}},"br":{},"sub":"cnp ","figref":"FIG. 12"},{"@attributes":{"id":"p-0475","num":"0594"},"maths":{"@attributes":{"id":"MATH-US-00006","num":"00006"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mrow":{"mi":"\u03bc","mo":"=","mrow":{"mrow":[{"mo":["[","]"],"mrow":{"mrow":[{"mo":"\u2146","msub":{"mi":["I","ds"]}},{"mo":"\u2146","msub":{"mi":["V","s"]}}],"mo":"\/"}},{"mo":["[","]"],"mrow":{"mi":"L","mo":"\/","mrow":{"mo":["(",")"],"mrow":{"mi":"W","mo":["\u2062","\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"msub":[{"mi":["C","i"]},{"mi":["V","ds"]}]}}}}],"mo":"\u00d7"}},"mo":","}}},"br":[{},{}],"sub":["ds ","ds","g "],"figref":"FIG. 12B"},"Various peptide nanostructures observed in this study exhibited a height distribution dependent on their lateral size. We analyzed an AFM image with peptide nanostructures on graphene using a particle analysis module in SPIP (Image Metrology A\/S). The AFM image shows peptide nanowires to have a linear shape and lateral length of greater than a micron, as well as particle-like islands with a lateral diameter of less than 100 nm. An increase of lateral size from islands to nanowires correlates monotonically with the height of the nanostructures (not shown). This tendency was also observed in the peptide morphology on actual GFET devices (not shown).","S7. Height Analysis of Peptides on Graphene and MoS","To quantitatively compare the thickness of peptide structures on ASLs, we further analyzed the height histograms of AFM images in . AFM images and their corresponding height histograms for peptides WT, M6 and M8 on graphene and MoSwere obtained using SPIP (not shown). Each histogram was fitted with Gaussian peaks to derive the average thickness of peptide structures. In this analysis, most images are best fit with three Gaussian peaks, except for WT on a graphene. The presence of three individual peaks in the histogram may indicate that AFM images contain distributions of structures on ASL surfaces with two general phases, disordered and ordered. In the case of WT on graphene, the AFM image does not contain disordered peptides. The fitting results clearly show that peptides have a similar thickness of 1 nm on both graphene and MoS, with the exception of M8 on MoS, which has a thickness of 1.7 nm. This observation implies that the distinct effects of M6 and M8 on the PL and conductivity of MoSin  originates from a conformational difference between M6 and M8 on MoS.","S8. Raman Measurements for Graphene and Peptide Modified Graphene","Mechanically exfoliated single-layer graphene on Si wafers were verified by a Raman microscope. Recently, the doping of graphene by organic molecules has been characterized by Raman spectroscopy (13). We characterized Raman spectra of graphene before and after incubation with the GrBP5:WT peptide (not shown). Raman spectroscopy was performed using a Renishaw Raman microscope with the 514 nm excitation laser. The excitation power was 100 \u03bcW. The Si peak at 520 cmwas used as a reference for wavenumber calibration. Ten graphene samples were used for the Raman characterization of the peptide doping effect. The well characterized graphene samples were then incubated with 1 \u03bcM peptide solutions of GrBP5:WT for 1 hour. GrBP5:WT shows 5 cmshift in the G-band and 1 cmshift in the 2D-band after the incubation on the average. In the previous report (13), organic molecules act as hole dopants showed positive shift in both 2D and G band. GrBP5:WT also demonstrated positive peak shifts in both 2D- and D-band, and it supports the results of the conductivity measurements () showing hole doping by peptides.",{"@attributes":{"id":"p-0479","num":"0000"},"ul":{"@attributes":{"id":"ul0011","list-style":"none"},"li":["1. K. S, Novoselov et al., Electric field effect in atomically thin carbon films. 306, 666 (2004).","2. Y. Zhang, J. W. Tan, H. L. Stormer, P. Kim, Experimental observation of the quantum Hall effect and Berry's phase in graphene. 438, 201 (2005).","3. K. I. Bolotin et al., Ultrahigh electron mobility in suspended graphene. 146, 351 (2008).","4. A. Splendiani et al., Emerging photoluminescence in monolayer MoS10, 1271 (2010).","5. K. F. Mak et al., Atomically thin MoS: a new direct-gap semiconductor. 105, 136805 (2010).","6. K. S, Novoselov et al., Two-dimensional atomic crystals. 102, 10451 (2005).","7. B. Radisavljevic, A. Radenovic, J. Brivio, V. Giacometti, A. K is, Single-layer MoStransistors. 6, 147 (2011).","8. Q. H. Wang, M. C. Hersam, Room-temperature molecular-resolution characterization of self-assembled organic monolayers on epitaxial graphene. 1, 206 (2009).","9. A. J. Pollard et al., Supramolecular assemblies formed on an epitaxial graphene superstructure, 49, 1794 (2010).","10. X. Wang, S. M. Tabakman, H. Dai, Atomic layer deposition of metal oxides on pristine and functionalized. 130, 8152 (2008).","11. F. Schedin et al., Detection of individual gas molecules adsorbed on graphene. 6, 652 (2007).","12. J.-H. Chen et al., Charged-impurity scattering in graphene. 4, 377 (2008).","13. X. Dong et al., Doping single-layer graphene with aromatic molecules. 5, 1422 (2009).","14. T. Kowalewski, D. M. Holtzman, In situ atomic force microscopy study of Alzheimer's \u03b2-amyloid peptide on different substrates: New insights into mechanism of b-sheet formation. 96, 3688 (1999).","15. C. So et al., Molecular recognition and supramolecular self-assembly of a genetically engineered gold binding peptide on Au{1113, 1525 (2009).","16. C. So et al., The molecular mechanism of self-assembled peptides on graphite. (2011)","17. S. Brown, M. Sarikaya, E. A. Johnson, A genetic analysis of crystal growth. 299, 725-735 (2000).","18. S. R. Whaley et al., Selection of peptides with semiconductor binding specificity for directed nanocrystal assembly. 405, 665 (2000).","19. M. Sarikaya et al., Molecular biomimetics: nanotechnology through biology. 2, 577 (2003).","20. D. B. Farmer, Y. M. Lin, A. Afzali-Ardakani, P. Avouris, Behavior of a chemically doped graphene junction. 94, 213106 (2009).","21. H. C. Cheng et al., high-quality graphene p-n junctions via resist-free fabrication and solution-based noncovalent functionalization. 5, 2051 (2011).","22. C. Rajesh, C. Majumder, H. Mizuseki, Y. Kawazoe, A theoretical study on the interaction of aromatic amino acids with graphene and single walled carbon nanotube. 130, 124911 (2009).","23. S. Regot, et al., Distributed biological computation with multicellular engineered networks. 469, 207 (2011).","24. C. O. Girit, A. Zettl, Soldering to a single atomic layer. 91, 193512 (2007).","25. J. Yan, Y. Zhang, P. Kim, A. Pinczuk, Electric field effect tuning of electron-phonon coupling in graphene. 98, 166802 (2007)."]}},"Peptide GrBP5 has been shown above to have a strong affinity to graphite and form high aspect ratio nanostructures that are symmetrically aligned along specific crystal orientations of the surface. More importantly, explicit domains of amino acids along the primary sequence of the GrBP5 dodecapeptide (IMVTESSDYSSY (SEQ ID NO: 5)) govern peptide-surface and peptide-peptide interactions, resulting in growth of structures where the N-terminus of peptides are uniformly exposed from the substrate into the solution. Utilizing GrBP5, here we demonstrate two accomplishments: Formation of discrete nm-wide, micron-long peptide nanowires (PNWs) on graphite, and targeted assembly of quantum dots (QDs) on these functional molecular scaffolds.","We first attach biological functionality into GrBP5 by appending biotin to its N-terminus (bio-GrBP5: ). Discrete nano-sized clusters of bio-GrBP5 that form at low concentrations, over time, transform into high aspect ratio nanowires of uniform orientation. As we demonstrate here, utilizing highly specific biotin-streptavidin (SA) interactions, it is then possible to target SA-functionalized entities such as quantum dots onto nanowires.","By controlling peptide deposition conditions, e.g., time and concentration, discrete molecular nanowires can be assembled at ultralow concentrations (10s of nM) which display biotin on graphite, as schematically displayed in . Upon exposure to these conditions, long spanning peptide nanowires (PNWs) are successfully formed using 50 nM peptides (suspended in deionized water) as imaged by AFM in . The PNWs are oriented uniaxially, as seen by Fast Fourier Transform (FFT) in , with a minimum width of \u02dc17 nm () and lengths spanning at least 4 \u03bcm, spaced \u02dc16 nm apart. The maximum height of PNWs, measured by AFM, is \u02dc1.8\u00b10.4 nm (), This value, on the average, is higher than the structures formed by WT GrBP5, i.e., without the biotin, by about 3 \u212broughly the same size of free molecular biotin (\u02dc6\u00d79\u00d72 \u212b) and may indicate that biotin is displayed by the NWs. Interestingly, as seen in , PNW widths are measured to increase at 2 nm intervals, suggesting that new wires grow co-axially along existing wires. This also reveals the size for single peptide nanowires without the convoluting effects of the AFM tip.","A variety of peptide nanostructures form at 10, 50 and 125 nM concentrations upon 5 hours of incubation time under ambient conditions, as shown in . From these quasi-equilibrium states, peptide affinity (K) can be calculated by surface coverage measurements (), yielding a value of 20 \u03bcM. While assembly at 125 nM solution yields confluent ordered films, similar to previous observations, the lowest concentration yields sparse peptide clusters, \u02dc1.5 nm in height, and 20-30 nm in diameter. The height of the clusters are reminiscent of the disordered phase from previous studies of GrBP5 film formation,seen immediately preceding the ordering transition. Interestingly, isolated high aspect ratio nanowires form at 50 nM with uniform spacing, signifying a threshold concentration for wire formation.","To understand the transition from discrete peptide clusters to individual molecular wires, we perform time-lapsed AFM of the adsorption process using the solutions containing 50 nM concentrations at different time spans (3, 5 and 7 hours, ). In previous observations of the confluent film formation, the transition from disorder to order was observed to correspond with a decrease in height of the assembled peptide by up to \u02dc50%. During the particle-to-wire transition observed here, a similar trend occurs with a less abrupt height change of \u02dc30% as peptide coverage increases (). Concurrent with the NW formation, the spacing narrows from \u02dc50 nm to \u02dc16 nm upon 5 and 7 hours of assembly, respectively. Thus, at low concentrations, peptides initially exist as discrete clusters that reach a certain height and size, and eventually grow over time to form nanowires likely templated by the graphite surface lattice.","Functional peptide nanowires offer a novel molecular scaffold to immobilize biomolecules via specific recognition of its accessible tail, which is programmable through the known peptide sequence. To demonstrate this, here we first utilize available biotin from peptide NW formation and, taking advantage of the highly specific biotin-SA interaction, incubate SA proteins to assess the functionality of biotin displayed on the nanowires (depicted in ). To immobilize SA, functional nanowires are first formed on graphite (), followed by exposure to a solution of SA in buffer for 1 minute. As seen in , nanowires are uniformly coated with SA and are spaced roughly 30 nm apart on the wire. The observed height of coated bio-GrBP5 PNWs significantly increased from 1.8\u00b10.4 nm to 5.8\u00b10.9 nm, indicating that SA molecules are bound to the nanowires. The average height of immobilized SA on NWs is, therefore, \u02dc4 nm, in good agreement with the dimension characterized by protein crystallography (45\u00d745\u00d753 \u212b). Thus, these results indicate that protein dimension, and structure of SA could be largely preserved when bound to PNWs.","To demonstrate further utility, chemically functionalized self-assembled PNWs allow targeted assembly of nanoparticles onto patterns on graphite. SA-coated quantum dots (SA-QDs) are widely employed in both nanotechnology and nanomedicine.We used SA-QDs here to demonstrate the functionality of bio-GrBP5 as a molecular scaffold through the highly ordered NW architecture. Upon exposure to SA-QD solution, particles target bio-GrBP5 wires and self-organize into 2D patterns via specific biotin-SA molecular recognition. SA-QD decorated NWs span several micrometers on the surface, represented in with the longest measuring 3-\u03bcm or longer (). AFM height measurements of the PNW-immobilized particles averaged \u02dc7.5 nm; this is indicative of the presence of PNWs (\u02dc1.8 nm in height) underlying immobilized SA-QD particles (). The resulting 5.7 nm increase in height, on the average, over the measured height of bare PNWs (\u02dcL8 nm) agrees well with the literature values of free standing SA-QDs which reportedly varies from 4.7\u00b10.7 nm to 5.5\u00b10.6 nm. As indicated in the histogram in , the height of SA-QDs adsorbed onto bare graphite is determined to be about 6 nm in height, also in agreement with SA-QD measurements on PNWs. In the absence of PNWs, however, SA-QDs aggregate uncontrollably on the graphite substrate forming large, mostly irregular, 2D islands (). As further confirmation of QDs on PNWs, immobilized QDs are excited by a 514 nm laser to measure their photoluminescence (PL) (See , Supporting Information). The PL spectrum shows a large peak at the expected emission wavelength of 605 nm for the specific Cd\u2014Se QDs used (). To estimate the PL yield per QD, the integrated intensity of the emission peak is divided by the number of particles determined by AFM analysis (Supporting Information Methods); the resulting values are listed in . The yield from SA-QD particles on wires is found to be, on average, \u02dc3\u00d7 greater than their emission on bare graphite, indicating that the increased distance between QDs and graphite by PNWs may effectively suppress the quenching of the PL signal.","In conclusion, we demonstrate a method to form chemically addressable peptide nanowires on graphite which then can be used as a molecular scaffold for the hierarchical assembly of quantum dots. Specifically, at low peptide concentrations (a few nM) discrete PNWs form on atomically flat graphite through a transformation of peptide nanoclusters into elongated nanostructures, guided by the surface lattice. In fact, by controlling the growth conditions, through both time and concentration, it is possible to form variety of peptide nanostructures, i.e., 0-D clusters, 1-D wires and 2-D films, with defined number density and highly precise spacing at nanoscale dimensions. Furthermore, bio-functional moieties, such as biotin, can be attached to the N-terminus of the peptides and which then become available for interaction with the protein SA and SA-functionalized QDs upon self-assembly on graphite. Chemically addressable nanowires, via functionalization of the GrBP5 tail, can be used as the basis for organizing other metallic nanoparticles, e.g., via Cysteine- or Histidine-tags, for future optical, electrical or magnetic nanodevices fabricated on graphite and graphene.",{"@attributes":{"id":"p-0488","num":"0000"},"ul":{"@attributes":{"id":"ul0012","list-style":"none"},"li":["[1] a) M. Grzelczak, J. Vermant, E. M. Furst, L. M. Liz-Marzan, 2010, 4, 3591; b) S. Srivastava, N. A. Kotov, Soft Matter 2009, 5, 1146.","[2] Z. Y. Tang, N. A. Kotov, 2005, 17, 951.","[3] a) I. W. Hamley, 2003, 14, R39; b) Y. Lin, A. Boker, J. B. He, K. Sill, H. Q. Xiang, C. Abetz, X. F. Li, J. Wang, T. Emrick, S. Long, Q. Wang, A. Balazs, T. P. Russell, 2005, 434, 55; c) S. O. Kim, H. H. Solak, M. P. Stoykovich, N. J. Ferrier, J. J. de Pablo, P. F. Nealey, 2003, 424, 411; d) W. A. Lopes, H. M. Jaeger, 2001, 414, 735.","[4] a) H. Bui, C. Onodera, C. Kidwell, Y. Tan, E. Graugnard, W. Kuang, J. Lee, W. B. Knowlton, B. Yurke, W. L. Hughes, 2010, 10, 3367; b) H. Yan, J. Sharma, R. Chhabra, Y. Liu, Y. G. Ke, 2006, 45, 730; c) J. D. Le, Y. Pinto, N. C. Seeman, K. Musier-Forsyth, T. A. Taton, R. A. Kiehl, 2004, 4, 2343; d) T. H. LaBean, H. Y. Li, S. H. Park, J. H. Reif, H. Yan, 2004, 126, 418.","[5] a) Y. Huang, C. Y. Chiang, S. K. Lee, Y. Gao, E. L. Hu, J. De Yoreo, A. M. Belcher, 2005, 5, 1429; b) T. Scheibel, R. Parthasarathy, G. Sawicki, X. M. Lin, H. Jaeger, S. L. Lindquist, 2003, 100, 4527; c) G. Grigoryan, Y. H. Kim, R. Acharya, K. Axelrod, R. M. Jain, L. Willis, M. Drndic, J. M. Kikkawa, W. F. DeGrado, 2011, 332, 1071; d) R. A. McMillan, C. D. Paavola, J. Howard, S. L. Chan, N. J. Zaluzec, J. D. Trent, 2002, 1, 247; e) S. Sotiropoulou, Y. Sierra-Sastre, S. S. Mark, C. A. Batt, 2008, 20, 821.","[6] L. Adler-Abramovich, D. Aronov, P. Beker, M. Yevnin, S. Stempler, L. Buzhansky, G. Rosenman, E. Gazit, 2009, 4, 849.","[7] C. Whitehouse, J. Y. Fang, A. Aggeli, M. Bell, R. Brydson, C. W. G. Fishwick, J. R. Henderson, C. M. Knobler, R. W. Owens, N. H. Thomson, D. A. Smith, N. Boden, 2005, 44, 1965.","[8] H. Y. Bai, K. Xu, Y. J. Xu, H. Matsui, 2007, 46, 3319.","[9] Z. H. Su, Y. A. Lin, G. H. Xiao, E. Balizan, G. Kaur, Z. W. Niu, Q. A. Wang, 2011, 27, 1398.","[10] a) Q. Gu, C. D. Cheng, R. Gonela, S. Suryanarayanan, S. Anabathula, K. Dai, D. T. Haynie, 2006, 17, R14; b) N. Nuraje, I. A. Banerjee, R. I. MacCuspie, L. T. Yu, H. Matsui, 2004, 126, 8088.","[11] a) C. R. So, Y. Hayamizu, H. Yazici, C. Gresswell, D. Khatayevich, C. Tamerler, M. Sarikaya, 2012, 6, 1648; b) C. R. So, J. L. Kulp, E. E. Oren, H. Zareie, C. Tamerler, J. S. Evans, M. Sarikaya, 2009, 3, 1525; c) F. Zhang, H. N. Du, Z. X. Zhang, L. N. Ji, H. T. Li, L. Tang, H. B. Wang, C. H. Fan, H. J. Xu, Y. Zhang, J. Hu, H. Y. Hu, J. H. He, 2006, 45, 3611; d) C. L. Brown, I. A. Aksay, D. A. Saville, M. H. Hecht, 2002, 124, 6846.","[12] J. V. Barth, J. Weckesser, G. Trimarchi, M. Vladimirova, A. De Vita, C. Z. Cai, H. Brune, P. Gunter, K. Kern, 2002, 124, 7991.","[13] A. L. Weisenhorn, F. J. Schmitt, W. Knoll, P. K. Hansma, 1992, 42, 1125.","[14] A. M. Belcher, C. B. Mao, C. E. Flynn, A. Hayhurst, R. Sweeney, J. F. Qi, G. Georgiou, B. Iverson, 2003, 100, 6946.","[15] T. Q. Vu, V. P. Pattani, C. F. Li, T. A. Desai, 2008, 10, 367."]}},"bio-GrBP5 peptide nanowires were first formed on graphite and verified under an AFM. Next, either streptavidin (from 50677, Sigma-Aldrich, St. Louis, Mo.) or streptavidin-Qdot conjugates (SA-QDs) were introduced to the same surface for immobilization to nanowires. For this, a 1 \u03bcM solution of SA or Cd\u2014Se Qdot 605-streptayidin conjugate (Q10101MP, Life Technologies Co., Grand Island, N.Y.) was diluted 1:1 with deionized water at room temperature for a 500 nM solution of SA or SA-QD. On a pre-assembled surface with bio-GrBP5 peptide nanowires, 500 nM SA or SA-QD solution was incubated on graphite at room temperature for 1 minute. Excess SA or SA-QD solution was removed after 1 minute by pipette. In order to prevent salt formation from the buffer, 100 \u03bcL of deionized water was drop incubated on graphite after SA or SA-QD incubation and removed by pipetting. After this rinsing step, the remaining liquid on graphite surface was dried off with a stream of nitrogen.","Photoluminescence Measurements.","For photoluminescence measurements, a Raman microscope (InVia, Renishaw, Hoffman Estates, Ill.) equipped with an inverted optical microscope (DMIRBE, Leica Microsystems, Wetzlar, Germany) was used. The excitation source used was an Argon Ion laser at 514 nm wavelength with a spot size of less than 1 \u03bcm (using a 100\u00d7 objective). Laser output power was 10 mW, set to 10% power by microscope software prior to data acquisition.","In the present study we demonstrate selective detection of a model protein against a background of serum protein using a graphene sensor functionalized via co-assembled, multifunctional, self-assembling peptides, which simultaneously display the probe and prevent non-specific adsorption. In particular we demonstrate a graphene field effect transistor sensor which can detect streptavidin against a background of serum bovine albumin at less than 50 ng\/ml. We have also developed a regeneration protocol, which allows us to utilize each sensor for over 12 experiments. This peptide based functionalization platform is particularly well suited to biological applications due to the ease of fusion of peptides with proteinaceous probes, and biocompatible nature. This system can be applied to a variety of bio-sensing problems, such as the detection of cancer markers under clinical conditions.","Detection of molecular recognition events ex situ has been a well-established approach to drug discovery, and molecular biology research since the 1960s.[1] More recently, it has been applied in clinical research and practice for diagnosing and monitoring of diseases, including several types of cancer, through the detection of biomarkers in blood, tissues, or urine of the patient. [2] Among the techniques used to detect such markers are the surface plasmon resonance spectroscopy (SPR),[3, 4] quartz crystal microbalance (QCM),[5] various immunoassays, including electrochemical assays, and others.[6-9] The applicability of these techniques to clinical practice remains limited, however, since the markers are present in very low concentrations, against a background of serum or tissue proteins. In the last several years, single layer material field effect transistor (SLaM-FET) sensors have been employed for ultrasensitive detection of molecular binding.[10, 11] Graphene and carbon nano-tubes in particular have been used successfully to achieve very low detection limit,[12-15] with some designs reported to detect femtogram per milliliter concentrations of analyte.[16] Such sensitivity is possible due to graphene's excellent electronic properties, resulting from delocalized it bonds on the surface, and a band gap which is very sensitive to doping.[17-19]","In order to capture the target, probes must be immobilized on the sensor surface in a predictable, non-perturbative fashion. A number of functionalization routes through covalent bonding, i.e. the introduction of carboxylic groups, have been employed to control the interface with graphitic materials.[20, 21] However, to preserve the intrinsic properties of these materials, methods of non-covalent functionalization via \u03c0-\u03c0 it stacking using aromatic chemistry have been used.[22, 23] In addition to these successful chemical functionalization techniques, non-invasive approaches using peptides have been demonstrated as a biocompatible alternative to controlling the surface properties of graphitic materials. Biocombinatorially selected through phage or cell-surface display, engineered solid-binding peptides offer a versatile platform for bridging the bio\/inorganic divide. [24-26] These strong-binding (kd 50 nM to 1 \u03bcM), material-specific 7-14 amino acid long sequences possess a wealth of chemical diversity and modular capacity through mutation and targeted chemical modification, providing unique opportunities for tuning binding, chemical properties, and display.[27, 28] Instead of covalent bonding, prevalent in synthetic linkers, short peptides bind through weak forces at multiple positions at the peptide\/solid interface enabling them to assemble and function in aqueous solutions.[29, 30] Solid-binding peptides are particularly well suited for applications in medical and biological fields because they are produced and function under biological conditions, and have not shown any toxicity in cell culture studies.[31, 32] Various graphite,[33] graphene-,[34-36] and CNT-binding [37, 38] peptide sequences and poly amino acids have been identified in literature through combinatorial display and other means. They have been employed for applications such as bioinorganic nano-structure formation,[39] as well as non-specific control of surface chemistry.[33] The dodecapeptide GrBP5-WT (Sequence: IMVTESSDYSSY (SEQ ID NO: 5), affinity constant: Ka=3.78 \u03bcM-1) is unique among graphite- and CNT-binding peptide sequences identified so far, as it forms long-range ordered, uniform, and crystallographic molecular nanostructures on graphitic materials (), which can be controlled through sequence mutation. GrBP5-WT is modular and can be designed to expose predictable surface chemistry through the display of specific amino acids. Moreover, it has been shown that two mutants of GrBP5-WT can be simultaneously assembled to display a combination of properties. Multi-functionality is critical to the application of a biomarker sensor in clinical practice, since in addition to possessing sensitivity, it must be capable of discriminating for the target against a background of proteins present in the sample. It is necessary to simultaneously impart the targeting and the anti-fouling capabilities to the sensor. In the present study, we demonstrate selective detection of a model protein against a background of serum protein using a graphene sensor functionalized via co-assembled, multifunctional, self-assembling peptides, which simultaneously display the probe and prevent non-specific adsorption. In particular we employ a 4-probe graphene field effect transistor (gFET) sensor (, B, C), functionalized using simultaneously co-assembled mutants of GrBP5-WT, displaying biotin, or hydrophilic residues at the N-terminus (Table 3) to detect streptavidin (SA) against a background of bovine serum albumin (BSA). We also developed a regeneration protocol, which allows us to utilize a single sensor for over 12 experiments. Through this study we establish a methodology for single-step bio-functionalization of graphitic sensors toward sensitive detection of biomolecules, which can be applied to a variety of analytes in complex solutions.",{"@attributes":{"id":"p-0494","num":"0651"},"tables":{"@attributes":{"id":"TABLE-US-00013","num":"00013"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u20033\u2003"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Peptide\u2003sequences\u2003weights\u2003and\u2003hydropathy\u2003indices"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Molecule","Sequence","Mol.\u2003Mass","G.R.A.V.Y.[a]"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["GrBP5-WT","IMVTESSDYSSY","1381.4","-0.242"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u20035)",{},{}]},{"entry":{}},{"entry":["Bio-GrBP5","bioIMVTESSDYSSY","1624.3","Aprox."]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200320)",{},"-0.185"]},{"entry":{}},{"entry":["SS-GrBP5","SSIMVTESSDYSSY","1555.6","-0.321"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200316)"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},"Sensor Construction","Graphene was prepared by exfoliation method [18] on SiO2 wafers, which were pre-treated with acidic piranha solution (75% sulfuric acid, 25% hydrogen peroxide). Orientation markers were made via indium micro-soldering, [42] and a PMMA coating was applied to the wafer by spin-coating for 1 min at 1000 RPM. The patterns ( A,B) were made via electron beam lithography on JOEL 7000 SEM (JOEL Ltd., Japan). The pattern was developed and then extended by hand to lengthen the electrodes. The electrodes were made by sputtering 2 nm of titanium as adhesion layer, followed by 46 nm of platinum, and 2 nm of titanium, as insulating layer using Gatan Precision Etching Coating System Model 682 (Gatan Inc., USA). The PMMA was removed in boiling acetone, and the device annealed in a tube furnace under a 60% argon\/40% hydrogen atmosphere at 450 Co for 1 hour. The terminals of indium solder were added at the ends of the electrodes. The contacts were current-annealed under nitrogen atmosphere by cycling currents of up to 1 mA at up to 60 volts through the device using Agilent U2722A USB Modular Source Measure Unit (Agilent. USA), until the resistance of the device remained constant between cycles. The devices were re-cleaned between experiments by boiling in acetone for 1 hour, followed by current-annealing.","Peptides and Proteins","The peptides were produced by solid-state synthesis using a CSBio 336 s automated peptide synthesizer (CSBio, USA) on Wang resin via Fmoc chemistry and HBTU activation. The crude peptides were purified by reverse phase high performance liquid chromatography to >98% purity (Gemini 10 \u03bcm C18 110A column) The purified peptides were verified by mass spectroscopy (MS) using a MALDI-TOF mass spectrometer (Bruker Daltonics Inc., USA). The amino acid sequences and the physio-chemical properties of the peptides used in the study are shown in Table 3. Streptavidin and bovine serum albumin were purchased from Sigma-Aldrich, USA, and used as received. All of the solutions were prepared with deionized water.","Experimental Setup","The device was connected to the Agilent U2722A USB Modular Source Measure Unit (Agilent, USA) in a four probe configuration, with current kept constant between R1 and R2 terminals, and voltage measured between R3 and R4 terminals (). A 20 \u03bcl drop of water was placed on the sensor and the resistance was allowed to equilibrate. The device was maintained in a hydration chamber at 100% humidity to prevent evaporation. Analyte was added to the static drop on the sensor in the appropriate concentrations, so that the total volume of the drop never exceeded 35 \u03bcl. Data was collected from two distinct devices and each experiment was reproduced at least once. See supporting information for additional data.","Atomic Force Microscopy","To complement the biosensor results, certain states of each experiment were reproduced on highly-oriented pyrolytic graphite (HOPG) and imaged by atomic force microscopy (AFM). The peptide solutions, at concentrations identical to those used for sensing experiments, were placed on HOPG and allowed to bind for a similar amount of time (10-90 minutes). The samples were washed by dilution, dried and immediately imaged using a Digital Instruments Nanoscope-IIIa Multimode AFM (Veeco, USA) under tapping mode. The images were taken in at least 3 different areas of the sample. The similarity between peptide binding to HOPG and peptide binding to graphene was demonstrated by AFM (not shown).","In order to test the capabilities of the graphene field effect transistor sensor, as well as to ensure that our peptide-based functionalization scheme appropriately displayed the probe, we created a biotin-graphite binding peptide (bio-GrBP5) fusion sequence (Table 3). By sequential introduction of increasing concentrations of the peptide to the sensor, we were able to determine the ultimate sensitivity of our system, and to create a dense, uniform monolayer of self-assembled biotinilated peptide (, C). We then introduced streptavidin to the system in sequentially increasing concentrations to determine if the sensitivity significantly diminishes with distance from the sensor. The AFM images (), along with the sensogram () demonstrate binding of streptavidin to the biotinilated sensor. The height difference of about 2 nm is observed between the peptide only and the SA added surfaces. Moreover, we found that the sensitivity of our device is between 20 and 50 ng\/ml, both in the bare and functionalized state. As a control, we also tested the binding of SA to the un-functionalized sensor, finding that no detectable binding occurs in the range of concentrations used (not shown). The differences in morphology in the two AFM images indicates that there is a certain amount of instability introduced into the peptide by the analyte, however, the coverage remains above 85% after two hours of testing, which is well beyond the time required for measurements.","Non-specific adsorption of background proteins is a major concern when testing for biomarkers in clinically relevant samples. Serum albumin (BSA) constitutes about 50% of the blood serum proteins, and is, therefore, a good model protein to serve as the background against which SA may be detected.  demonstrates the adsorption of BSA on graphite after washing (corresponding sensogram ). The protein is present on the entire surface and agglomerates into fibers as the result of drying. The sensogram shows robust detection of binding at 50 ng\/ml, followed by increased binding with overall resistance shift of almost 200 k\u03a9 at 10 \u03bcg\/ml, which is higher than any observed in other experiments, probably due to the formation of a multi-layer film. In order to prevent non-specific adsorption of BSA, we employed the SS-GrBP5 mutant (Table 3), identified in an earlier study as one capable of assembling into ordered monolayers and presenting hydrophilic chemistry (36\u2032 contact angle at 100% coverage).[41] Such a contact angle value is similar to those achieved in literature by self-assembled monolayer polyethylene glycol anti-fouling systems (about 32\u00b0).[32, 43, 44] The contact angle value has been strongly linked to anti-fouling properties in a variety of systems.[45] We, therefore, hypothesized that our self-assembled peptide monolayers, which are dense and exhibit similar contact angles, would also be anti-fouling. To test this hypothesis, we assembled SS-GrBP5 peptide () on the sensor surface, and introduced BSA to the system (). The binding of BSA was significantly impaired by the SS-GrBP5 mutant, resulting in no detectable binding at less than 10 \u03bcg\/ml concentrations of BSA. The AFM image of the surface after incubation with 10 \u03bcg\/ml BSA for 90 minutes () shows a small amount of protein present sparsely on the surface and very low degradation of the underlying peptide monolayer. The added stability, as compared to the bio-GrBP5 mutant, is likely the result of the more amphiphilic nature of the SS-GrBP5 monolayer, which remains robustly oriented between the hydrophobic graphite and water. As a control experiment, we introduced BSA to a monolayer of bio-GrBP5 (not shown). The sensogram show significant binding of BSA.","By combining the anti-fouling properties of SS-GrBP5 with the biotinylated probe peptide, we created a sensor capable of selective detection of streptavidin in a bovine serum albumin solution. A mixed peptide monolayer consisting of 25% bio-GrBP5 and 75% SS-GrBP5 was formed in a single step from a solution with an overall concentration of 10 \u03bcg\/ml. The AFM of the resulting monolayer () shows that the peptides self-assembled into the dense, ordered structure with no discernible segregation, leading to the likely conclusion that the peptides are miscible. Guided by the results in , we introduced 5 \u03bcg\/ml BSA to the sensor and detected no binding (). We then introduced sequentially increasing concentrations of SA to the sensor, while maintaining the levels of BSA, and were able to reliably detect streptavidin at 100 ng\/ml, although some signal was present at even lower concentrations (). The AFM image of the graphite surface at the final conditions () showed less coverage by SA than the biotin-only surface (). The overall stability of the bio-GrBP5 appears to have been improved by co-assembly with SS-GrBP5, displaying very high coverage and density. SS-GrBP5 also seems to retain its anti-fouling function despite a 25% reduction in coverage, owing to the fact that the overall order of the monolayer is maintained by the self-assembling sequence of bio-GrBP5 inclusion. Since the above experiment was carried out under the stop-flow conditions, some of the streptavidin may have been deactivated by the excess bio-GrBP5 still left in solution. To eliminate this variable and to find the region of linear response for our sensor, we repeated the experiment, washing the sensor by dilution after immobilizing bio-GrBP5, prior to introducing streptavidin.  shows the sensogram, and the corresponding signal versus concentration plot for the experiment. The Langmuir fit shows a KD value of about 1.5 nM-9, which is higher than the established literature values.[46] This may indicate that biotin is not as readily accessible on the co-assembled surface, possibly because it is surrounded by a large numbers of the anti-fouling SS-GrBP5 peptides, which are slightly larger that bio-GrBP5.","The sensor devices used in this study were regenerated and reused. We observed some sensor degradation over the course of about 12 experiments, with the noise levels increasing from near 2 k\u03a9 to about 10 k\u03a9. However, it seems that the sensitivity of the device is not significantly affected between trials, meaning that it can be used to produce comparable data several times. This is a significant feature for practical application of graphene based sensors, and represents another advantage of peptide functionalization over covalent approaches.","In this study, we demonstrated a system for simultaneous passivation and functionalization of the graphene sensor using the GrBP5 modular graphite-binding peptide family, allowing for selective detection of streptavidin against a serum albumin background. The same system can be extended to other, more efficient sensor architectures, and other single layer materials systems, such as the semi-conducting MoS2 or WS2, through rational mutation of the GrBP5-WT peptide. Our ability to control the groups displayed by the peptide monolayers, as well as the possibility of creating mixed monolayers, allows for the construction of multi-functional sensors capable of selective detection of multiple probes in a complex environment. Since a variety of short peptidic probes have been identified for many disease biomarkers through phage and cell-surface display, the next step for this technology would be to create a diagnosis and monitoring system via genetic fusion of the two sequences. However, the system is not limited to clinical applications. Specific ligand receptor interactions are of critical importance to all areas of molecular biology. Using our system, a selective biosensor can be created for many of the ligand-receptor pairs under investigation, by displaying one of the constituents on the graphene surface in a predictable and robust fashion. More broadly, this study adds another tool to the peptide-based bionanotechnology toolkit, taking advantage of Nature's way of putting together materials at the nano-scale.",{"@attributes":{"id":"p-0504","num":"0000"},"ul":{"@attributes":{"id":"ul0013","list-style":"none"},"li":["[1] Yalow R S, Berson S A. IMMUNOASSAY OF ENDOGENOUS PLASMA INSULIN 1N MA N. Journal of Clinical Investigation. 1960; 39:1157-75.","[2] Ludwig J A, Weinstein J N. Biomarkers in cancer staging, prognosis and treatment selection. Nature Reviews Cancer. 2005; 5:845-56.","[3] Vaisocherova H, Yang W, Zhang Z, Cao Z Q, Cheng G, Piliarik M, et al. Ultralow fouling and functionalizable surface chemistry based on a zwitterionic polymer enabling sensitive and specific protein detection in undiluted blood plasma. Analytical Chemistry. 2008; 80:7894-901.","[4] Chou S F, Hsu W L, Hwang J M, Chen C Y. Development of an immunosensor for human ferritin, a nonspecific tumor marker, based on surface plasmon resonance. Biosensors & Bioelectronics. 2004; 19:999-1005.","[5] Di Natale C, Macagnano A, Martinelli E, Paolesse R, D'Arcangelo G, Roscioni C, et al. Lung cancer identification by the analysis of breath by means of an array of non-selective gas sensors. Biosensors & Bioelectronics. 2003; 18:1209-18.","[6] Yu X, Munge B, Patel V, Jensen G, Bhirde A, Gong J D, et al. Carbon nanotube amplification strategies for highly sensitive immunodetection of cancer biomarkers. Journal of the American Chemical Society. 2006; 128:11199-205.","[7] Liu X, Dai Q, Austin L, Coutts J, Knowles G, Zou J H, et al. A one-step homogeneous immunoassay for cancer biomarker detection using gold nanoparticle probes coupled with dynamic light scattering. Journal of the American Chemical Society. 2008; 130:2780-+.","[8] Mani V, Chikkaveeraiah B V, Patel V, Gutkind J S, Rusling J F. Ultrasensitive Immunosensor for Cancer Biomarker Proteins Using Gold Nanoparticle Film Electrodes and Multienzyme-Particle Amplification. Acs Nano. 2009; 3:585-94.","[9] Liu G D, Lin Y Y, Wang J, Wu H, Wai C M, Lin Y H. Disposable electrochemical immunosensor diagnosis device based on nanoparticle probe and immunochromatographic strip. Analytical Chemistry.","[10] Curreli M, Zhang R, Ishikawa F N, Chang H K, Cote R J, Zhou C, et al. Real-Time, Label-Free Detection of Biological Entities Using Nanowire-Based FETs. Ieee Transactions on Nanotechnology. 2008; 7:651-67.","[11] Alam M M, Wang J, Guo Y Y, Lee S P, Tseng H R. Electrolyte-gated transistors based on conducting polymer nanowire junction arrays. Journal of Physical Chemistry B. 2005; 109:12777-84.","[12] Ohno Y, Maehashi K, Matsumoto K. Label-Free Biosensors Based on Aptamer-Modified Graphene Field-Effect Transistors. Journal of the American Chemical Society. 2010; 132:18012-3.","[13] Stine R, Robinson J T, Sheehan P E, Tamanaha C R. Real-Time DNA Detection Using Reduced Graphene Oxide Field Effect Transistors. Advanced Materials. 2010; 22:5297-300.","[14] Someya T, Small J, Kim P, Nuckolls C, Yardley J T. Alcohol vapor sensors based on single-walled carbon nanotube field effect transistors. Nano Letters. 2003; 3:877-81.","[15] Page T R, Hayamizu Y, So C R, Sarikaya M. Electrical detection of biomolecular adsorption on sprayed graphene sheets. Biosensors & Bioelectronics. 2012; 33:304-8.","[16] Zhang B, Li Q, Cui T H. Ultra-sensitive suspended graphene nanocomposite cancer sensors with strong suppression of electrical noise. Biosensors & Bioelectronics. 2012; 31:105-9.","[17] Castro Neto A H, Guinea F, Peres N M R, Novoselov K S, Geim A K. The electronic properties of graphene. Reviews of Modern Physics. 2009; 81:109-62.","[18] Novoselov K S, Geim A K, Morozov S V, Jiang D, Zhang Y, Dubonos S V, et al. Electric field effect in atomically thin carbon films. Science. 2004; 306:666-9.","[19] Balandin A A, Ghosh S, Bao W Z, Calizo I, Teweldebrhan D, Miao F, et al. Superior thermal conductivity of single-layer graphene. Nano Letters. 2008; 8:902-7.","[20] Li D, Muller M B, Gilje S, Kaner R B, Wallace G G. Processable aqueous dispersions of graphene nanosheets. Nature Nanotechnology. 2008; 3:101-5.","[21] Niyogi S, Bekyarova E, Itkis M E, McWilliams J L, Hamon M A, Haddon R C. Solution properties of graphite and graphene. Journal of the American Chemical Society. 2006; 128:7720-1.","[22] Choi E Y, Han T H, Hong J H, Kim J E, Lee S H, Kim H W, et al. Noncovalent functionalization of graphene with end-functional polymers. Journal of Materials Chemistry. 2010; 20:1907-12.","[23] Ghosh A, Rao K V, Voggu R, George S J. Non-covalent functionalization, solubilization of graphene and single-walled carbon nanotubes with aromatic donor and acceptor molecules. Chemical Physics Letters. 2010; 488:198-201.","[24] Shiba K. Exploitation of peptide motif sequences and their use in nanobiotechnology. Current Opinion in Biotechnology. 2010; 21:412-25.","[25] Tamerler C, Khatayevich D, Gungormus M, Kacar T, Oren E E, Hnilova M, et al. Molecular Biomimetics: GEPI-Based Biological Routes to Technology. Biopolymers. 2010; 94:78-94.","[26] Peelle B R, Krauland E M, Wittrup K D, Belcher A M. Design criteria for engineering inorganic material-specific peptides. Langmuir. 2005; 21:6929-33.","[27] Sarikaya M, Tamerler C, Jen A K Y, Schulten K, Baneyx F. Molecular biomimetics: nanotechnology through biology. Nature Materials. 2003; 2:577-85.","[28] Wei J H, Kacar T, Tamerler C, Sarikaya M, Ginger D S. Nanopatterning Peptides as Bifunctional Inks for Templated Assembly. Small. 2009; 5:689-93.","[29] Hnilova M, Oren E E, Seker UO S, Wilson B R, Collino S, Evans J S, et al. Effect of Molecular Conformations on the Adsorption Behavior of Gold-Binding Peptides. Langmuir. 2008; 24:12440-5.","[30] Tamerler C, Sarikaya M. Molecular biomimetics: nanotechnology and bionanotechnology using genetically engineered peptides. Philosophical Transactions of the Royal Society a-Mathematical Physical and Engineering Sciences. 2009; 367:1705-26.","[31] Meyers S R, Khoo X J, Huang X, Walsh E B, Grinstaff M W, Kenan D J. The development of peptide-based interfacial biomaterials for generating biological functionality on the surface of bioinert materials. Biomaterials. 2009; 30:277-86.","[32] Khatayevich D, Gungormus M, Yazici H, So C, Cetinel S, Ma H, et al. Biofunctionalization of materials for implants using engineered peptides. Acta Biomaterialia. 2010; 6:4634-41.","[33] Yang H, Fung S Y, Sun W, Mikkelsen S, Pritzker M, Chen P. Ionic-complementary peptide-modified highly ordered pyrolytic graphite electrode for biosensor application. Biotechnology Progress. 2008; 24:964-71.","[34] Cui Y, Kim S N, Jones S E, Wissler L L, Naik R R, McAlpine M C. Chemical Functionalization of Graphene Enabled by Phage Displayed Peptides. Nano Letters. 2010; 10:4559-65.","[35] Wang Q H, Hersam M C. Room-temperature molecular-resolution characterization of self-assembled organic monolayers on epitaxial graphene. Nature Chemistry. 2009; 1:206-11.","[36] Kim S N, Kuang Z F, Slocik J M, Jones S E, Cui Y, Farmer B L, et al. Preferential Binding of Peptides to Graphene Edges and Planes. Journal of the American Chemical Society. 2011; 133:14480-3.","[37] Kase D, Kulp J L, Yudasaka M, Evans J S, Iijima S, Shiba K. Affinity selection of peptide phage libraries against single-wall carbon nanohorns identifies a peptide aptamer with conformational variability. Langmuir. 2004; 20:8939-41.","[38] Tomasio S M, Walsh T R. Modeling the Binding Affinity of Peptides for Graphitic Surfaces. Influences of Aromatic Content and Interfacial Shape. Journal of Physical Chemistry C. 2009; 113:8778-85.","[39] Han T H, Lee W J, Lee D H, Kim J E, Choi E Y, Kim S O. Peptide\/Graphene Hybrid Assembly into Core\/Shell Nanowires. Advanced Materials. 2010; 22:2060-+.","[40] So C R, Hayamizu Y, Yazici H, Gresswell C, Khatayevich D, Tamerler C, et al. Controlling Self-Assembly of Engineered Peptides on Graphite by Rational Mutation. Acs Nano. 2012; 6:1648-56.","[41] Khatayevich D, So C R, Hayamizu Y, Gresswell C, Sarikaya M. Controlling the Surface Chemistry of Graphite by Engineered Self-Assembled Peptides. Langmuir. 2012; 28:8589-93.","[42] Girit C O, Zettl A. Soldering to a single atomic layer. Applied Physics Letters. 2007; 91.","[43] Harder P, Grunze M, Dahint R, Whitesides G M, Laibinis P E. Molecular conformation in oligo(ethylene glycol)-terminated self-assembled monolayers on gold and silver surfaces determines their ability to resist protein adsorption. Journal of Physical Chemistry B. 1998; 102:426-36.","[44] Faucheux N, Schweiss R, Lutzow K, Werner C, Groth T. Self-assembled monolayers with different terminating groups as model substrates for cell adhesion studies. Biomaterials. 2004; 25:2721-30.","[45] Menzies K L, Jones L. The Impact of Contact Angle on the Biocompatibility of Biomaterials. Optometry and Vision Science. 2010; 87:387-99.","[46] Holmberg A, Blomstergren A, Nord O, Lukacs M, Lundeberg J, Uhlen M. The biotin-streptavidin interaction can be reversibly broken using water at elevated temperatures. Electrophoresis. 2005; 26:501-10."]}},"Single atomic layers of two-dimensional materials, such as graphene and MoS, are attractive as a novel platform to integrate heterogeneous nano-scale components towards a diversity of applications in the next generation electronics. Furthermore, the controllability of their electrical property via chemical doping with heterogeneous molecules attached on the surface gains increased interest. There have been various studies in the assembly of nano-materials on graphene, such as nano-particles, organic molecules, and bio-molecules. In order to realize more complex electronics with the two-dimensional materials, it is crucial to assemble sophisticated nano-structures forming discrete electronic junctions on a single layer of two-dimensional materials. However, such organic molecules can form only confluent monolayer film on graphene. Furthermore, there has been no study of molecular self-assembly on MoS2 single layers. Here we demonstrate that bio-combinatorially selected and rationally designed peptides can self-assemble into long range ordered structures, i.e., nanowires, on single-layer graphene and MoS. The self-assembled peptides form nano-scale electronic p-n junctions in graphene. Furthermore, peptides modulate the electrical conductivity and photoluminescence of single-layer MoS. Self-assembled peptides can potentially realize diverse and complex electronics of two dimensional materials, such as bio-nano electronics.","Graphene has attracted wide interests due to its rich physics and extraordinarily high carrier mobility. Recently, single-layer MoS, another two-dimensional material, has been found to have a direct band gap exhibiting appreciable photoluminescence and attractive switching property in a field effect transistor. Of particular interest in those single layer materials is energy-band engineering via chemical doping, in which the charge carrier density is varied by adsorbed chemical substances. Whereas there has been no report of chemical doping in single-layer MoS, chemical doping in graphene has been reported using adsorbed gases or molecules as dopants. In principle, the chemical doping is capable of forming spatially abrupt electronic junctions with nano-scale patterns of dopants on single layer materials. Aiming to form electronic junctions for realization of more sophisticated graphene-based electronics, lithographic, imprinting, and micro-drawing techniques have been utilized to fabricate patterned dopants on graphene. In contrast to those top-down processes, molecules self-assembled into long-range ordered nanostructures could offer a scalable bottom-up approach to create nano-scale electronic junctions on graphene. However, those molecules that have been utilized as dopants have established only confluent films on surfaces, mainly due to their simple molecular structures. Hence, we need to find more complex molecules having broad chemical diversity able to form discrete nanostructures, i.e., two-dimensional patterns of ordered molecules, on single layer materials. Furthermore, the complex molecules need the ability of doping effect with single layer materials.","Utilizing GrBP5 as a starting point, we rationally designed its derivatives by simple substitutions of several amino acids in the primary sequence. The challenging task in the new design is to create a peptide capable of forming ordered structures on MoS. Our strategy of the rational design is to control the peptide self-assembly and their doping effect through smart control of the location and the sign of charged amino acids. The GrBP5 contains charged and aromatic amino acids (colored in ). The charged amino acids, Aspartate (D) and Glutamate (E), are negatively charged under the condition of pH 7. Based on previous studies indicating a strong binding affinity of aromatic amino acids to graphite surfaces via pi-electrons, we assumed that YSSY (SEQ ID NO: 31) containing two Tyrosines (Y) could be the binding site to graphene. When peptides bound on a graphene, the separation between charged amino acids and aromatic amino acids in the sequence could be monotonically related to the actual distance of the charged amino acids from the graphene surface. Based on the above assumption, we synthesized GrBP5 mutant6 (M6), which was designed to have a stronger electric interaction with graphene than GrBP5 wild type (WT), due to the negatively charged amino acids in the middle of two tyrosines. In the same manner, M8 has been synthesized to have opposite charges ().",{"@attributes":{"id":"p-0508","num":"0710"},"figref":["FIG. 26","FIG. 26","FIG. 26","FIG. 26","FIGS. 26"],"i":["e ","f"]},"The conductivity of graphene was clearly modulated by peptides. Two-terminal indium electrodes were contacted on a mechanically exfoliated graphene by micro-soldering technique to persevere the cleanness of graphene, which is essential for the peptide self-assembly. All electrical measurement was performed under Argon gas atmosphere to eliminate instability caused by moisture. To investigate the correlation between the morphology of peptide nanostructures on graphene and the graphene conductivity, we utilized various concentrations of the GrBP5 WT from 50 nM to 500 nM to tune the peptides self-assembly and the coverage on graphene. The conductivity of graphene with 500 nM WT shows significant peak shift in the gate response (). The bare graphene shows charge neutral point (CNP) at 30V typically. After the peptide incubation, the peak is shifted to 80V indicating hole doping via the peptides. Interestingly, the field-effect mobility of holes in graphene shows slight change from 3500 to 2500 cmVsafter peptides incubation. The observation of a weak disruption in the mobility is consistent with previous reports on the chemically doped graphene with organic molecules. The AFM image of the actual FET device shows highly covered graphene surface by ordered peptides (). The peptides formed a number of nanowires with a close separation. When a graphene FET was incubated with 50 nM GrBP5, the conductivity of graphene revealed double peaks in the gate response (). Each CNP peak at 30V and 80V can be attributed to bare graphene region and peptide coated region based on the original location of those bare and peptide coated graphene in . Supporting this interpretation, the AFM image shows well-defined peptide nanowires on graphene (). The energy separation between n- and p-type regions is estimated as 200 meV, which is sufficiently larger than thermal energy at room temperature. The formation of PN junction enabled by peptides is scalable and very simple because we use only aqueous droplet on a graphene FET. Other derivatives, M6 and M8, also modulated the graphene conductivity via doping. M6 shows hole doping in the gate response. Not as expected, the M8, which has opposite charges from M6 and WT, did not dope graphene with electrons but holes. It is probably because of the absence of the water molecules.","Similar to graphene, the interaction of peptides with single layer MoSis highly correlated with the peptide sequence. Interestingly, on MoS, only M6 formed long range ordered structures like nanowires (), but not WT. The single layer MoSwas prepared in the same manner as graphene. The number of the layers was confirmed by both the AFM and PL. The width and height of the nanowires are 11 nm and 1.5 nm, respectively (). The WT and M8 forms only isolated islands with high coverage (). It indicates that the location of the charged amino acids in the sequence is highly correlated with the surface behavior of peptides on graphene including binding, diffusion and self-assembly. To investigate the peptide effect on an optical process in MoS, we performed PL measurements before and after peptide incubation. We focus here M6 and M8 since M6 is the only peptide forming ordered structures and M8 is its pair with opposite sign of charges. While WT shows the large peak shift of 9 meV in the PL among our peptides. On the other hand, M8 does not show any peak shift. Note that the peak width of the both peptide case is broadened from 64 meV to 76 meV after peptide incubation.","Conductivity measurements performed by single layer MoStransistors shows a consistent result with the PL measurement. Due to the difficulty to produce a large sheet of mechanically exfoliated MoS2, we employed electron beam lithography to fabricate electrodes on the single layer MoS, instead of the micro-soldering. In the gate response, M6 shows significant doping effect as the threshold voltage is shifted from \u221210V to 15V (). M8 does not show the threshold voltage shift. It corresponds to the peak shifts in PL. However both peptides show similarly slower slope than the bare graphene, which indicates decrease of the hole mobility. This is related with the observation of the peak broadening in the PL measurement.","Peptide Synthesis.","Peptides were prepared on an automated solid-phase peptide synthesizer (CS336X, CSBio Inc., Menlo Park, Calif.) employing standard batch wise Fmoc chemistry procedures as reported previously.Peptides were verified by MALDI-TOF mass spectrometry as well as size-exclusion chromatography to verify peptide mass and dispersion in water. (See SI appendix, S7)","Sample Preparations.","Graphenes were mechanically exfoliated on a Si wafer with 300-nm thick SiO2 and incubated with 50 \u03bcL peptide under experimental condition. When incubation is finished, the substrate is gently dried with nitrogen gas. All samples in this study were prepared using this method.","We used a mechanically exfoliated graphene on a Si-wafer with 300 nm-thick SiO2. The exfoliated graphene was incubated with an aqueous solution of peptides (0.25 \u03bcM concentration for 20 minutes). After the incubation, the sample was gently dried by nitrogen gas.","Atomic Force Microscopy.","A Digital Instruments (Veeco, Santa Barbara, Calif.) Multimode Nanoscope Ma scanning probe microscope equipped with high frequency NanoSensors PPP\u2014NCHR (NanoandMore USA, Lady's Island, S.C., USA) non-contact probes, with a 42 N\/m spring constant, at a 4V amplitude setpoint.","Conductivity Measurements.","Room Temperature","Photoluminescence Measurements.","Room Temperatures",{"@attributes":{"id":"p-0516","num":"0000"},"ul":{"@attributes":{"id":"ul0014","list-style":"none"},"li":["1. H. A. Lowenstam and S. Weiner, On Biomineralization (Oxford University Press, Oxford, UK, 1989).","2. M. Sarikaya, Biomimetics: Materials fabrication through biology, Proc. Natl. Acad. Sci. USA, 96, 14183 (1999).","3. S. Mann, Molecular recognition in biomineralization, Nature, 332, 119 (1988).","4. I Weissbuch, L. Addadi, M. Lahav, and L. Leiserowitz, Molecular recognition at crystal interfaces, Science 253, 637 (1991).","5. M. Sarikaya, C. Tamerler, A. K. Y. Jen, K. Schulten, F. Baneyx, Molecular biomimetics: nanotechnology through biology. Nat. Mater., 2, 577 (2003).","6. S. W. Lee, C. B. Mao, C. E. Flynn, A. M. Belcher, Ordering of Quantum Dots Using Genetically Engineered Viruses. Science, 296, 892 (2002).","7. M. B. Dickerson, K. H. Sandhage, and R. R. Naik, Protein- and Peptide-Directed Syntheses of Inorganic Materials. Chem. Reviews, 108, 4935 (2008).","8. K. Sano, H. Sasaki, and K. Shiba, Utilization of the Pleiotropy of a Peptidic Aptamer To Fabricate Heterogeneous Nanodot-Containing Multilayer Nanostructures. J. Am. Chem. Soc., 128, 1717 (2006).","9. C. L. Chen and N. L. Rosi, Peptide-Based Methods for the Preparation of Nanostructured Inorganic Materials. Angew. Chem. Intl. Ed., 49, 1924 (2010).","10. C. Toniolo, A. Bianco, M. Maggini, G. Scorrano, M. Prato et al., A Bioactive Fullerene Peptide. J. Med. Chem. 37, 4558 (1994).","11. D. Pantarotto, C. D. Partidos, R. Graff, J. Hoebeke, J-P., Briand, M. Prato, A. Bianco, Synthesis, Structural Characterization, and Immunological Properties of Carbon Nanotubes Functionalized with Peptides. J. Am. Chem. Soc. 125, 6160 (2003).","12. D. Kase, J. L. Kulp, M. Yudasaka, J. S. Evans, S. Iijima, K. Shiba, Affinity Selection of Peptide Phage Libraries against Single-Wall Carbon Nanohorns Identifies a Peptide Aptamer with Conformational Variability. Langmuir 20, 8939 (2004).","13. B. L. Allen, P. D. Kichambare, A. Star, Carbon Nanotube Field-Effect-Transistor-Based Biosensors. Adv. Mater. 9, 1439 (2006).","14. K. S, Novoselov, A. K. Geim, S. V. Morozov, D. Jiang, Y. Zhang, S. V. Dubonos, I. V. Grigorieva, A. A. Firsov, Science 306, 666 (2004).","15. Y. Zhang, J. W. Tan, H. L. Stormer, P. Kim, Experimental observation of the quantum Hall effect and Berry's phase in graphene. Nature 438, 201 (2005).","16. K. I. Bolotin, K. J. Sikes, Z. Jiang, M. Klima, G. Fudenberg, J. Hone, P. Kim, H. L. Stormer, Ultrahigh electron mobility in suspended graphene. Solid State Commun 146, 351 (2008).","17. M. Y. Han, B. Oezyilmaz, Y. Zhang, P. Kim, Energy Band Gap Engineering of Graphene Nanoribbons. Phys. Rev. Lett. 98, 206805 (2007).","18. Y. Zhang, T.-Ta Tang, C. Girit, Z. Hao, M. C. Martin, A. Zettl, M. F. Crommie, Y. R. Shen, F. Wang, Direct observation of a widely tunable bandgap in bilayer graphene. Nature 459, 820 (2009).","19. X. Dong, D. Fu, W. Fang, Y. Shi, P. C., L.-J. Li, Doping Single-Layer Graphene with Aromatic Molecules. Small 5, 1422 (2009).","20. A. Das, S. Pinsana, B. Chakraborty, S. Piscanec, S. K. Saha, U. V. Waghmare, K. S. Novoselov, H. R. Krishnamurthy, A. K. Geim, A. C. Ferrari, A. K. Sood, Monitoring dopants by Raman scattering in an electrochemically top-gated graphene transistor, Nature Nanotech. 3, 210 (2008).","21. G. P. Smith, Filamentous fusion phage: novel expression vectors that display cloned antigens on the virion surface. Science, 228, 1315 (1985).","22. S. Brown, Engineered iron oxide adhesion mutants of the phage-lambda receptor, Proc. Natl. Acad. Sci. U.S.A. 89, 8651 (1992).","23. See supporting on-line material.","24. S. D. M. Tomasio and T. R. Walsh, Atomistic modeling of the interaction between peptides and carbon nanotubes. Molecular Physics, 105, 221 (2007).","25. C. Rajesh, C. Majumder, H. Mizuseki, and Y. A. Kawazoe, A theoretical study on the interaction of aromatic amino acids with graphene and single walled carbon nanotube. J. Chem. Phys. 130, 124911 (2009).","26. C. So, C. Tamerler and M. Sarikaya, Adsorption, Diffusion, and Self-Assembly of an Engineered Gold-Binding Peptide on Au(111) Investigated by Atomic Force Microscopy. Angew Chem. Intl. Ed. 48, 5174 (2009).","27. C. So, J. L. Kulp, E. E., Oren, H. Zareie, C. Tamerler, J. S. Evans, and M. Sarikaya, Molecular Recognition and Supramolecular Self-Assembly of a Genetically Engineered Gold Binding Peptide on Au {111}. ACS Nano, 3, 1525 (2009).","28. C. H. Lui, L. Liu, K. F. Mak, G. W. Flynn, T. F. Heinz, Ultraflat graphene, Nature 462, 339 (2009).","29. G. M. Whitesides, J. P. Mathias, and C. T. Seto, Molecular self-assembly and nanochemistry: a chemical strategy for the synthesis of nanostructures. Science 254, 1312 (1991).","30. L. A. Bumm, J. J. Arnold, M. T. Cygan, T. D. Dunbar, T. P. Burgin, L. D. Jones, J. M. Tour, and P. S. Weiss, Are single molecular wires conducting? Science, 271, 1705 (1996)."]}},"Using the methods demonstrated in the various examples above, various mutations of the WT GrBP5 have been shown capable of binding and ordering on variety of single layer atomic materials (SLAMs) with different physical characteristics, such as crystal structure and conducting. These include: semiconducting, metallic and insulator characteristics.","Representative SLAMs include:",{"@attributes":{"id":"p-0518","num":"0000"},"ul":{"@attributes":{"id":"ul0015","list-style":"none"},"li":{"@attributes":{"id":"ul0015-0001","num":"0000"},"ul":{"@attributes":{"id":"ul0016","list-style":"none"},"li":["Metallic: Graphene or doped graphene","Semiconductors: MoS2, MoSe2, WS2, WSe2, -- Insulator: BN."]}}}},"Exemplary of such ordering are provided in . One such study involved analysis of M8 (IMVTASSAYRRY) (SEQ ID NO:3) on BN (Boron Nitride). Similar to the assembly of specific peptides on other atomically flat materials or single layer materials, here assembly of M8 on BN is shown as observed by atomic force microscopy imaging. The assembly of the peptide M8 was accomplished using a 1 mM\/ml upon 1 hr of incubation (followed by rinsing and drying). Despite the fact that M8 peptide does not form long range ordered structures on any of the other four substrates, except partial ordering on WSe2, this peptides assembly on BN is much more prominent. Having basic amino acid Arginine \u201cRR\u201d replacing the two Serine residues at the same position, while removing the acidic residues E and D (and replacing them with Alanine) makes this peptide more positively charged and results in the binding and assembly of the peptide on BN. In summary, the M8 mutation is specific for BN substrate. Similar studies were done for other mutants on other inorganic solid surfaces (See  and Figure Legends). A summary of exemplary studies on the polypeptides is shown in Table 4 below. Each peptide has been tested for binding activity to one or more SLAM, as well as diffusion, assembly, chemical activity (such as modifying SLAM hydrophobicity or contact angle), and electrical activity (such as doping effects, including but not limited to injecting holes or electrons into the SLAM. The presence of such activities is noted by a \u201cY\u201d, or \u201cYY\u201d (for very strong activity), the absence of such activity is noted by a \u201cN\u201d, and a \u201c?\u201d indicates that the activity has not been tested.",{"@attributes":{"id":"p-0520","num":"0752"},"tables":{"@attributes":{"id":"TABLE-US-00014","num":"00014"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"49pt","align":"center"}}],"thead":{"row":[{"entry":"TABLE\u20034"},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}},{"entry":["Peptide","Sequence","Binding","Diffusing","Assembling","Chemical","Electronic"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["GrBP5-WT","IMVTESSDYSSY","Y","Y","YY","Y","Y"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u20035)",{},{},{},{},{}]},{"entry":{}},{"entry":["GrBP5-M1","IMVTESSDASSA","N","N","N","N","N"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u20036)",{},{},{},{},{}]},{"entry":{}},{"entry":["GrBP5-M2","IMVTESSDWSSW","Y","Y","N","Y","Y"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u20037)",{},{},{},{},{}]},{"entry":{}},{"entry":["GrBP5-M3","IMVTKSSRFSSF","Y","N","N","Y",{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u20038)",{},{},{},{},{}]},{"entry":{}},{"entry":["GrBP5-M4","TQSTKSSRYSSY","Y","Y","N","Y","Y"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u20039)",{},{},{},{},{}]},{"entry":{}},{"entry":["GrBP5-M5","IMVTESSRYSSY","Y","Y","Y","Y","Y"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200310)",{},{},{},{},{}]},{"entry":{}},{"entry":["GrBP5-M6","IMVTASSAYDDY","Y","Y","Y","Y","YY"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u20032)",{},{},{},{},{}]},{"entry":{}},{"entry":["GrBP5-M8","IMVTASSAYRRY","Y","Y","N","Y","YY"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u20033)",{},{},{},{},{}]},{"entry":{}},{"entry":["GrBP5-M10","IMVTDSSAYSSY","Y","Y","?","Y","Y"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200335)",{},{},{},{},{}]},{"entry":{}},{"entry":["GrBP5-12","IMVTASSDYSSY","Y","Y","?","Y","Y"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200312)",{},{},{},{},{}]},{"entry":{}},{"entry":["HGrBP5","HIMVTESSDYSSY","Y","Y","Y","YY","Y"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200313)",{},{},{},{},{}]},{"entry":{}},{"entry":["WGrBP5","WIMVTESSDYSSY","Y","Y","Y","YY","Y"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200314)",{},{},{},{},{}]},{"entry":{}},{"entry":["SSGrBP5","SSIMVTESSDYSSY","Y","Y","YY","Y","Y"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200316)",{},{},{},{},{}]},{"entry":{}},{"entry":["VVGrBP5","VVIMVTESSDYSSY","Y","Y","?","Y","Y"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200315)",{},{},{},{},{}]},{"entry":{}},{"entry":["GrBP5","LIATESSDYSSY","Y","Y","Y","YY","Y"]},{"entry":["hydrophobic","(SEQ\u2003ID\u2003NO:\u200317)",{},{},{},{},{}]},{"entry":{}},{"entry":["GrBP\u20035","AQTTESSDYSSY","Y","Y","N","YY","Y"]},{"entry":["hydrophilic","(SEQ\u2003ID\u2003NO:\u200318)",{},{},{},{},{}]},{"entry":{}},{"entry":["GrBP\u20035","IMVTASSAYSSY","Y","?","?","?","Y"]},{"entry":["neutral","(SEQ\u2003ID\u2003NO:\u200319)",{},{},{},{},{}]},{"entry":{}},{"entry":["Bio-GrBP5","Bio-IMVTESSDYSSY","Y","Y","YY","YY","Y"]},{"entry":["(Biotin)","(SEQ\u2003ID\u2003NO:\u200320)",{},{},{},{},{}]},{"entry":{}},{"entry":["Rigid\u2003GrBP5","IMVTEPPDYSSY","Y","?","?","?","Y"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200321)",{},{},{},{},{}]},{"entry":{}},{"entry":["Cys-GrBP5","CIMVTESSDYSSY","Y","Y","Y","YY","Y"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200322)",{},{},{},{},{}]},{"entry":{}},{"entry":["DOPA-GrBP5","DOPA-IMVTESSDYSSY","Y","Y","Y","YY","Y"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200323)",{},{},{},{},{}]},{"entry":{}},{"entry":["AminoF-GrBP5","IMVTESSD(NonnaturalF)SSY","Y","?","?","?","?"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200324)",{},{},{},{},{}]},{"entry":{}},{"entry":["F-Phenyl-","IMVTESSD(F-Phenyl)SSY","Y","?","?","?","?"]},{"entry":["GrBP5","(SEQ\u2003ID\u2003NO:\u200325)",{},{},{},{},{}]},{"entry":{}},{"entry":["D-GrBP5","IMVTESSDYSSY","Y","Y","YY","Y","Y"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200326)",{},{},{},{},{}]},{"entry":{}},{"entry":["AntibodyBP-","EPIHRSTLTALL-SS-","Y","Y","Y","YY","N"]},{"entry":["SS-GrBP5","IMVTESSDYSSY",{},{},{},{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200327)"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}}]}}}}}],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":["BRIEF DESCRIPTION OF THE FIGURES","Example 1","Figure Legends"],"p":[{"@attributes":{"id":"p-0036","num":"0038"},"figref":"FIG. 1","sup":"42 "},{"@attributes":{"id":"p-0037","num":"0039"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0038","num":"0040"},"figref":"FIG. 3","sup":"2"},{"@attributes":{"id":"p-0039","num":"0041"},"figref":"FIG. 4","sup":"2"},{"@attributes":{"id":"p-0040","num":"0042"},"figref":"FIG. 5","sub":["CA","CA "]},{"@attributes":{"id":"p-0041","num":"0043"},"figref":"FIG. 6","sub":"cA","sup":["42 ","2"]},{"@attributes":{"id":"p-0042","num":"0044"},"figref":"FIG. 7"},{"@attributes":{"id":"p-0043","num":"0045"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0044","num":"0046"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0045","num":"0047"},"figref":"FIG. 10"},{"@attributes":{"id":"p-0046","num":"0048"},"figref":"FIG. 11","sub":"2 "},{"@attributes":{"id":"p-0047","num":"0049"},"figref":"FIG. 12"},{"@attributes":{"id":"p-0048","num":"0050"},"figref":"FIG. 13"},{"@attributes":{"id":"p-0049","num":"0051"},"figref":["FIG. 14","FIGS. 3 and 4"]},{"@attributes":{"id":"p-0050","num":"0052"},"figref":"FIG. 15"},{"@attributes":{"id":"p-0051","num":"0053"},"figref":"FIG. 16"},{"@attributes":{"id":"p-0052","num":"0054"},"figref":"FIG. 17"},{"@attributes":{"id":"p-0053","num":"0055"},"figref":"FIG. 18"},{"@attributes":{"id":"p-0054","num":"0056"},"figref":"FIG. 19"},{"@attributes":{"id":"p-0055","num":"0057"},"figref":"FIG. 20"},{"@attributes":{"id":"p-0056","num":"0058"},"figref":"FIG. 21"},{"@attributes":{"id":"p-0057","num":"0059"},"figref":"FIG. 22"},{"@attributes":{"id":"p-0058","num":"0060"},"figref":"FIG. 23"},{"@attributes":{"id":"p-0059","num":"0061"},"figref":"FIG. 24","sub":"2 "},{"@attributes":{"id":"p-0060","num":"0062"},"figref":"FIG. 25","sub":"2 "},{"@attributes":{"id":"p-0061","num":"0063"},"figref":"FIG. 26","sub":"2"},{"@attributes":{"id":"p-0062","num":"0064"},"figref":"FIG. 27"},{"@attributes":{"id":"p-0063","num":"0065"},"figref":"FIG. 28","sub":["2 ","2","2 ","2 ","2"]},{"@attributes":{"id":"p-0064","num":"0066"},"figref":"FIG. 29"},{"@attributes":{"id":"p-0065","num":"0067"},"figref":"FIG. 30"},{"@attributes":{"id":"p-0066","num":"0068"},"figref":"FIG. 31"},{"@attributes":{"id":"p-0067","num":"0069"},"figref":"FIG. 32"}]},"DETDESC":[{},{}]}
