---
title: Nucleic acids and proteins and methods for making and using them
abstract: The invention provides polypeptides, including enzymes, structural proteins and binding proteins, polynucleotides encoding these polypeptides, and methods of making and using these polynucleotides and polypeptides. Polypeptides, including enzymes and antibodies, and nucleic acids of the invention can be used in industrial, experimental, food and feed processing, nutritional and pharmaceutical applications, e.g., for food and feed supplements, colorants, neutraceuticals, cosmetic and pharmaceutical needs.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08962800&OS=08962800&RS=08962800
owner: BP Corporation North America Inc.
number: 08962800
owner_city: Naperville
owner_country: US
publication_date: 20120220
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATIONS","FIELD OF THE INVENTION","BACKGROUND","SUMMARY","DETAILED DESCRIPTION","DEFINITIONS"],"p":["This application is a divisional of U.S. patent application Ser. No. 11\/817,403, filed Aug. 29, 2007, now U.S. Pat. No. 8,119,385; which is the U.S. national phase, pursuant to 35 U.S.C. \u00a7371, of international application number PCT\/US2006\/007642, filed Mar. 3, 2006, designating the United States and published on Sep. 14, 2006 as publication number WO 2006\/096527, which claims priority under 35 USC \u00a7119(e)(1) of prior U.S. provisional application No. 60\/658,984, filed Mar. 4, 2005, all of which are hereby incorporated by reference.","This invention relates to molecular and cellular biology and biochemistry. In one aspect, the invention provides polypeptides, including enzymes, structural proteins and binding proteins (e.g., ligands, receptors), polynucleotides encoding these polypeptides, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, structural proteins and binding proteins, including thermostable and thermotolerant activity, and polynucleotides encoding these enzymes, structural proteins and binding proteins and making and using these polynucleotides and polypeptides. The polypeptides of the invention can be used in a variety of pharmaceutical, agricultural and industrial contexts, including the manufacture of cosmetics and nutraceuticals.","Additionally, the polypeptides of the invention can be used in food processing, brewing, bath additives, alcohol production, peptide synthesis, enantioselectivity, hide preparation in the leather industry, waste management and animal degradation, silver recovery in the photographic industry, medical treatment, silk degumming, biofilm degradation, biomass conversion to ethanol, biodefense, antimicrobial agents and disinfectants, personal care and cosmetics, biotech reagents, in corn wet milling and pharmaceuticals such as digestive aids and anti-inflammatory (anti-phlogistic) agents.","The invention provides isolated and recombinant polypeptides, including enzymes, structural proteins and binding proteins, polynucleotides encoding these polypeptides, and methods of making and using these polynucleotides and polypeptides. The polypeptides of the invention, and the polynucleotides encoding the polypeptides of the invention, encompass many classes of enzymes, structural proteins and binding proteins. In one aspect, the enzymes and proteins of the invention include, e.g. aldolases, alpha-galactosidases, amidases, e.g. secondary amidases, amylases, catalases, carotenoid pathway enzymes, dehalogenases, endoglucanases, epoxide hydrolases, esterases, hydrolases, glucosidases, glycosidases, inteins, isomerases, laccases, lipases, monooxygenases, nitroreductases, nitrilases, P450 enzymes, pectate lyases, phosphatases, phospholipases, phytases, polymerases and xylanases. The invention also provides isolated and recombinant polypeptides, including enzymes, structural proteins and binding proteins, polynucleotides encoding these polypeptides, having the activities described in Table 1, Table 2 or Table 3, below. The enzymes and proteins of the invention have utility in a variety of applications.","The invention provides isolated or recombinant nucleic acids comprising a nucleic acid sequence having at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more, or complete (100%) sequence identity to an exemplary nucleic acid of the invention, e.g., including SEQ ID NO:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO:11, SEQ ID NO:13, SEQ ID NO:15, SEQ ID NO:17, SEQ ID NO:19, SEQ ID NO:21, SEQ ID NO:23, SEQ ID NO:25, and all nucleic acids disclosed in the SEQ ID listing, which include all odd numbered SEQ ID NO:s from SEQ ID NO:1 through SEQ ID NO:26,897, over a region of at least about 10, 15, 20, 25, 30, 35, 40, 45, 50, 75, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000, 1050, 1100, 1150, 1200, 1250, 1300, 1350, 1400, 1450, 1500, 1550, 1600, 1650, 1700, 1750, 1800, 1850, 1900, 1950, 2000, 2050, 2100, 2200, 2250, 2300, 2350, 2400, 2450, 2500, or more residues, encodes at least one polypeptide having an enzyme, structural or binding activity, and the sequence identities are determined by analysis with a sequence comparison algorithm or by a visual inspection. In one aspect, the enzymes and proteins of the invention include, e.g. aldolases, alpha-galactosidases, amidases, e.g. secondary amidases, amylases, catalases, carotenoid pathway enzymes, dehalogenases, endoglucanases, epoxide hydrolases, esterases, hydrolases, glucosidases, glycosidases, inteins, isomerases, laccases, lipases, monooxygenases, nitroreductases, nitrilases, P450 enzymes, pectate lyases, phosphatases, phospholipases, phytases, polymerases and xylanases. In another aspect, the isolated and recombinant polypeptides of the invention, including enzymes, structural proteins and binding proteins, and polynucleotides encoding these polypeptides, of the invention have activity as described in Table 1, Table 2 or Table 3, below.","In one aspect, the invention also provides isolated or recombinant nucleic acids with a common novelty in that they are all derived from a common source, e.g., an environmental source, mixed environmental sources or mixed cultures. The invention provides isolated or recombinant nucleic acids isolated from a common source, e.g. an environmental source, mixed environmental sources or mixed cultures comprising a polynucleotide of the invention, e.g., an exemplary sequence of the invention, including SEQ ID NO:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO:11, SEQ ID NO:13, SEQ ID NO:15, SEQ ID NO:17, SEQ ID NO:19, SEQ ID NO:21, SEQ ID NO:23, SEQ ID NO:25, and all nucleic acids disclosed in the SEQ ID listing, which include all odd numbered SEQ ID NO:s from SEQ ID NO:1 through SEQ ID NO:26,897, over a region of at least about 10, 15, 20, 25, 30, 35, 40, 45, 50, 75, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000, 1050, 1100, 1150, 1200, 1250, 1300, 1350, 1400, 1450, 1500, 1550, 1600, 1650, 1700, 1750, 1800, 1850, 1900, 1950, 2000, 2050, 2100, 2200, 2250, 2300, 2350, 2400, 2450, 2500, or more residues, encodes at least one polypeptide having an enzyme, structural or binding activity, and the sequence identities are determined by analysis with a sequence comparison algorithm or by a visual inspection. In one aspect, the enzymes and proteins of the invention include, e.g. aldolases, alpha-galactosidases, amidases, e.g. secondary amidases, amylases, catalases, carotenoid pathway enzymes, dehalogenases, endoglucanases, epoxide hydrolases, esterases, hydrolases, glucosidases, glycosidases, inteins, isomerases, laccases, lipases, monooxygenases, nitroreductases, nitrilases, P450 enzymes, pectate lyases, phosphatases, phospholipases, phytases, polymerases and xylanases. In another aspect, the isolated and recombinant polypeptides of the invention, including enzymes, structural proteins and binding proteins, and polynucleotides encoding these polypeptides, of the invention have activity as described in Table 1, Table 2 or Table 3, below.","In alternative aspects, the isolated or recombinant nucleic acid encodes a polypeptide comprising an exemplary sequence of the invention, e.g., including sequences as set forth in SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, SEQ ID NO:14, SEQ ID NO:16, SEQ ID NO:18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, and all polypeptides disclosed in the SEQ ID listing, which include all even numbered SEQ ID NO:s from SEQ ID NO:2 through SEQ ID NO: 26,898. In one aspect these polypeptides have an enzyme, structural or binding activity. In one aspect, the enzymes and proteins of the invention include, e.g. aldolases, alpha-galactosidases, amidases, e.g. secondary amidases, amylases, catalases, carotenoid pathway enzymes, dehalogenases, endoglucanases, epoxide hydrolases, esterases, hydrolases, glucosidases, glycosidases, inteins, isomerases, laccases, lipases, monooxygenases, nitroreductases, nitrilases, P450 enzymes, pectate lyases, phosphatases, phospholipases, phytases, polymerases and xylanases. In another aspect, the isolated and recombinant polypeptides of the invention, including enzymes, structural proteins and binding proteins, and polynucleotides encoding these polypeptides, of the invention have activity as described in Table 1, Table 2 or Table 3, below.","In alternative aspects, the enzyme, structural or binding activity comprises a recombinase activity, a helicase activity, a DNA replication activity, a DNA recombination activity, an isomerase, a trans-isomerase activity or topoisomerase activity, a methyl transferase activity, an aminotransferase activity, a uracil-5-methyl transferase activity, a cysteinyl tRNA synthetase activity, a hydrolase, an esterase activity, a phosphoesterase activity, an acetylmuramyl pentapeptide phosphotransferase activity, a glycosyltransferase activity, an acetyltransferase activity, an acetylglucosamine phosphate transferase activity, a centromere binding activity, a telomerase activity or a transcriptional regulatory activity, a heat shock protein activity, a protease activity, a proteinase activity, a peptidase activity, a carboxypeptidase activity, an endonuclease activity, an exonuclease activity, a RecB family exonuclease activity, a polymerase activity, a carbamoyl phosphate synthetase activity, a methyl-thioadenine synthetase activity, an oxidoreductase activity, an Fe\u2014S oxidoreductase activity, a flavodoxin reductase activity, a permease activity, a thymidylate activity, a dehydrogenase activity, a pyrophosphorylase activity, a coenzyme metabolism activity, a dinucleotide-utilizing enzyme activity, a molybdopterin or thiamine biosynthesis activity, a beta-lactamase activity, a ligand binding activity, an ion transport activity, an ion metabolism activity, a tellurite resistance protein activity, an inorganic ion transport activity, a nucleotide transport activity, a nucleotide metabolism activity, an actin or myosin activity, a lipase activity or a lipid acyl hydrolase (LAH) activity, a cell envelop biogenesis activity, an outer membrane synthesis activity, a ribosomal structure synthesis activity, a translational processing activity, a transcriptional initiation activity, a TATA-binding activity, a signal transduction activity, an energy metabolism activity, an ATPase activity, an information storage and\/or processing activity, and\/or any of the polypeptides activities as set forth in Table 1, Table 2 or Table 3, below.","In one aspect, the sequence comparison algorithm is a BLAST version 2.2.2 algorithm where a filtering setting is set to blastall -p blastp -d \u201cnr pataa\u201d-F F, and all other options are set to default.","Another aspect of the invention is an isolated or recombinant nucleic acid including at least 10, 15, 20, 25, 30, 35, 40, 45, 50, 75, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000, 1050, 1100, 1150, 1200, 1250, 1300, 1350, 1400, 1450, 1500, 1550, 1600, 1650, 1700, 1750, 1800, 1850, 1900, 1950, 2000, 2050, 2100, 2200, 2250, 2300, 2350, 2400, 2450, 2500, or more consecutive bases of a nucleic acid sequence of the invention, sequences substantially identical thereto, and the sequences complementary thereto.","In one aspect, the isolated or recombinant nucleic acid encodes a polypeptide having a enzyme, structural or binding activity, that is thermostable. The polypeptide can retain activity under conditions comprising a temperature range of between about 37\u00b0 C. to about 95\u00b0 C.; between about 55\u00b0 C. to about 85\u00b0 C., between about 70\u00b0 C. to about 95\u00b0 C., or, between about 90\u00b0 C. to about 95\u00b0 C.","In another aspect, the isolated or recombinant nucleic acid encodes a polypeptide having an enzyme, structural or binding activity, which is thermotolerant. The polypeptide can retain activity after exposure to a temperature in the range from greater than 37\u00b0 C. to about 95\u00b0 C. or anywhere in the range from greater than 55\u00b0 C. to about 85\u00b0 C. The polypeptide can retain activity after exposure to a temperature in the range between about 1\u00b0 C. to about 5\u00b0 C., between about 5\u00b0 C. to about 15\u00b0 C., between about 15\u00b0 C. to about 25\u00b0 C., between about 25\u00b0 C. to about 37\u00b0 C., between about 37\u00b0 C. to about 95\u00b0 C., between about 55\u00b0 C. to about 85\u00b0 C., between about 70\u00b0 C. to about 75\u00b0 C., or between about 90\u00b0 C. to about 95\u00b0 C., or more. In one aspect, the polypeptide retains activity after exposure to a temperature in the range from greater than 90\u00b0 C. to about 95\u00b0 C. at about pH 4.5.","The invention provides isolated or recombinant nucleic acids comprising a sequence that hybridizes under stringent conditions to a nucleic acid comprising a sequence of the invention, e.g., an exemplary sequence of the invention, including SEQ ID NO:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO:11, SEQ ID NO:13, SEQ ID NO:15, SEQ ID NO:17, SEQ ID NO:19, SEQ ID NO:21, SEQ ID NO:23, SEQ ID NO:25, and all nucleic acids disclosed in the SEQ ID listing, which include all odd numbered SEQ ID NO:s from SEQ ID NO:1 through SEQ ID NO:26,897, or fragments or subsequences thereof. In one aspect, the nucleic acid encodes a polypeptide having a enzyme, structural or binding activity. The nucleic acid can be at least about 10, 15, 20, 25, 30, 35, 40, 45, 50, 75, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000, 1050, 1100, 1150, 1200 or more residues in length or the full length of the gene or transcript. In one aspect, the stringent conditions include a wash step comprising a wash in 0.2\u00d7SSC at a temperature of about 65\u00b0 C. for about 15 minutes.","The invention provides a nucleic acid probe for identifying a nucleic acid encoding a polypeptide having a enzyme, structural or binding activity, wherein the probe comprises at least about 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000 or more, consecutive bases of a sequence comprising a sequence of the invention, or fragments or subsequences thereof, wherein the probe identifies the nucleic acid by binding or hybridization. The probe can comprise an oligonucleotide comprising at least about 10 to 50, about 20 to 60, about 30 to 70, about 40 to 80, or about 60 to 100 consecutive bases of a sequence comprising a sequence of the invention, or fragments or subsequences thereof.","The invention provides a nucleic acid probe for identifying a nucleic acid encoding a polypeptide having a enzyme, structural or binding activity, wherein the probe comprises a nucleic acid comprising a sequence at least about 10, 15, 20, 30, 40, 50, 60, 70, 80, 90, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000 or more residues having at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more, or complete (100%) sequence identity to a nucleic acid of the invention. In one aspect, the sequence identities are determined by analysis with a sequence comparison algorithm or by visual inspection. In alternative aspects, the probe can comprise an oligonucleotide comprising at least about 10 to 50, about 20 to 60, about 30 to 70, about 40 to 80, or about 60 to 100 consecutive bases of a nucleic acid sequence of the invention, or a subsequence thereof.","The invention provides an amplification primer pair for amplifying a nucleic acid encoding a polypeptide having a enzyme, structural or binding activity, wherein the primer pair is capable of amplifying a nucleic acid comprising a sequence of the invention, or fragments or subsequences thereof. One or each member of the amplification primer sequence pair can comprise an oligonucleotide comprising at least about 10 to 50, or more, consecutive bases of the sequence, or about 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30 or more consecutive bases of the sequence.","The invention provides amplification primer pairs, wherein the primer pair comprises a first member having a sequence as set forth by about the first (the 5\u2032) 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36 or more residues of a nucleic acid of the invention, and a second member having a sequence as set forth by about the first (the 5\u2032) 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36 or more residues of the complementary strand of the first member.","The invention provides polypeptide-, enzyme-, protein-, e.g. structural or binding protein-encoding nucleic acids generated by amplification, e.g., polymerase chain reaction (PCR), using an amplification primer pair of the invention. The invention provides polypeptide-, enzyme-, protein-, e.g. structural or binding protein-encoding nucleic acids generated by amplification, e.g., polymerase chain reaction (PCR), using an amplification primer pair of the invention. The invention provides methods of making a polypeptide, enzyme, protein, e.g. structural or binding protein, by amplification, e.g., polymerase chain reaction (PCR), using an amplification primer pair of the invention. In one aspect, the amplification primer pair amplifies a nucleic acid from a library, e.g., a gene library, such as an environmental library.","The invention provides methods of amplifying a nucleic acid encoding a polypeptide having an enzyme, structural or binding activity, comprising amplification of a template nucleic acid with an amplification primer sequence pair capable of amplifying a nucleic acid sequence of the invention, or fragments or subsequences thereof.","The invention provides expression cassettes comprising a nucleic acid of the invention or a subsequence thereof. In one aspect, the expression cassette can comprise the nucleic acid that is operably linked to a promoter. The promoter can be a viral, bacterial, mammalian or plant promoter. In one aspect, the plant promoter can be a potato, rice, corn, wheat, tobacco or barley promoter. The promoter can be a constitutive promoter. The constitutive promoter can comprise CaMV35S. In another aspect, the promoter can be an inducible promoter. In one aspect, the promoter can be a tissue-specific promoter or an environmentally regulated or a developmentally regulated promoter. Thus, the promoter can be, e.g., a seed-specific, a leaf-specific, a root-specific, a stem-specific or an abscission-induced promoter. In one aspect, the expression cassette can further comprise a plant or plant virus expression vector.","The invention provides cloning vehicles comprising an expression cassette (e.g., a vector) of the invention or a nucleic acid of the invention. The cloning vehicle can be a viral vector, a plasmid, a phage, a phagemid, a cosmid, a fosmid, a bacteriophage or an artificial chromosome. The viral vector can comprise an adenovirus vector, a retroviral vector or an adeno-associated viral vector. The cloning vehicle can comprise a bacterial artificial chromosome (BAC), a plasmid, a bacteriophage P1-derived vector (PAC), a yeast artificial chromosome (YAC), or a mammalian artificial chromosome (MAC).","The invention provides transformed cell comprising a nucleic acid of the invention or an expression cassette (e.g., a vector) of the invention, or a cloning vehicle of the invention. In one aspect, the transformed cell can be a bacterial cell, a mammalian cell, a fungal cell, a yeast cell, an insect cell or a plant cell. In one aspect, the plant cell can be a cereal, a potato, wheat, rice, corn, tobacco or barley cell.","The invention provides transgenic non-human animals comprising a nucleic acid of the invention or an expression cassette (e.g., a vector) of the invention. In one aspect, the animal is a mouse, a rat, a pig, a goat or a sheep.","The invention provides transgenic plants comprising a nucleic acid of the invention or an expression cassette (e.g., a vector) of the invention. The transgenic plant can be a cereal plant, a corn plant, a potato plant, a tomato plant, a wheat plant, an oilseed plant, a rapeseed plant, a soybean plant, a rice plant, a barley plant or a tobacco plant.","The invention provides transgenic seeds comprising a nucleic acid of the invention or an expression cassette (e.g., a vector) of the invention. The transgenic seed can be a cereal plant, a corn seed, a wheat kernel, an oilseed, a rapeseed, a soybean seed, a palm kernel, a sunflower seed, a sesame seed, a peanut or a tobacco plant seed.","The invention provides an antisense oligonucleotide comprising a nucleic acid sequence complementary to or capable of hybridizing under stringent conditions to a nucleic acid of the invention. The invention provides methods of inhibiting the translation of a polypeptide, enzyme, protein, e.g. structural or binding protein message in a cell comprising administering to the cell or expressing in the cell an antisense oligonucleotide comprising a nucleic acid sequence complementary to or capable of hybridizing under stringent conditions to a nucleic acid of the invention. In one aspect, the antisense oligonucleotide is between about 10 to 50, about 20 to 60, about 30 to 70, about 40 to 80, or about 60 to 100 bases in length, e.g., 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, 100 or more bases in length.","The invention provides methods of inhibiting the translation of a polypeptide, enzyme, protein, e.g. structural or binding protein message in a cell comprising administering to the cell or expressing in the cell an antisense oligonucleotide comprising a nucleic acid sequence complementary to or capable of hybridizing under stringent conditions to a nucleic acid of the invention. The invention provides double-stranded inhibitory RNA (RNAi, or RNA interference) molecules (including small interfering RNA, or siRNAs, for inhibiting transcription, and microRNAs, or miRNAs, for inhibiting translation) comprising a subsequence of a sequence of the invention. In one aspect, the RNAi is about 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, 100 or more duplex nucleotides in length. The invention provides methods of inhibiting the expression of a polypeptide, enzyme, protein, peptide, e.g. structural or binding protein in a cell comprising administering to the cell or expressing in the cell a double-stranded inhibitory RNA (iRNA, including small interfering RNA, or siRNAs, for inhibiting transcription, and microRNAs, or miRNAs, for inhibiting translation), wherein the RNA comprises a subsequence of a sequence of the invention.","The invention provides isolated or recombinant polypeptides encoded by a nucleic acid of the invention. In alternative aspects, the polypeptide can have a sequence as set forth in SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, etc., and all polypeptides disclosed in the SEQ ID listing, which include all even numbered SEQ ID NO:s from SEQ ID NO:2 through SEQ ID NO:26,898 (the exemplary sequences of the invention), or subsequences thereof, including fragments having enzymatic and\/or substrate binding activity. The polypeptide can have an enzyme, structural or binding activity.","In alternative aspects, the enzyme, structural or binding activity comprises a recombinase activity, a helicase activity, a DNA replication activity, a DNA recombination activity, an isomerase, a trans-isomerase activity or topoisomerase activity, a methyl transferase activity, an aminotransferase activity, a uracil-5-methyl transferase activity, a cysteinyl tRNA synthetase activity, a hydrolase, an esterase activity, a phosphoesterase activity, an acetylmuramyl pentapeptide phosphotransferase activity, a glycosyltransferase activity, an acetyltransferase activity, an acetylglucosamine phosphate transferase activity, a centromere binding activity, a telomerase activity or a transcriptional regulatory activity, a heat shock protein activity, a protease activity, a proteinase activity, a peptidase activity, a carboxypeptidase activity, an endonuclease activity, an exonuclease activity, a RecB family exonuclease activity, a polymerase activity, a carbamoyl phosphate synthetase activity, a methyl-thioadenine synthetase activity, an oxidoreductase activity, an Fe\u2014S oxidoreductase activity, a flavodoxin reductase activity, a permease activity, a thymidylate activity, a dehydrogenase activity, a pyrophosphorylase activity, a coenzyme metabolism activity, a dinucleotide-utilizing enzyme activity, a molybdopterin or thiamine biosynthesis activity, a beta-lactamase activity, a ligand binding activity, an ion transport activity, an ion metabolism activity, a tellurite resistance protein activity, an inorganic ion transport activity, a nucleotide transport activity, a nucleotide metabolism activity, an actin or myosin activity, a lipase activity or a lipid acyl hydrolase (LAH) activity, a cell envelop biogenesis activity, an outer membrane synthesis activity, a ribosomal structure synthesis activity, a translational processing activity, a transcriptional initiation activity, a TATA-binding activity, a signal transduction activity, an energy metabolism activity, an ATPase activity, an information storage and\/or processing activity, and\/or any of the polypeptides activities as set forth in Table 1, Table 2 or Table 3, below.","Exemplary polypeptide or peptide sequences of the invention include SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, etc., and all polypeptides disclosed in the SEQ ID listing, which include all even numbered SEQ ID NO:s from SEQ ID NO:2 through SEQ ID NO:26,898, and subsequences thereof and variants thereof. Exemplary polypeptides also include fragments of at least about 10, 15, 20, 25, 30, 35, 40, 45, 50, 75, 80, 85, 90, 95, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600 or more residues in length, or over the full length of an enzyme. Exemplary polypeptide or peptide sequences of the invention include sequence encoded by a nucleic acid of the invention. Exemplary polypeptide or peptide sequences of the invention include polypeptides or peptides specifically bound by an antibody of the invention.","In one aspect, the polypeptide, enzyme, protein, e.g. structural or binding protein, is thermostable. The polypeptide, enzyme, protein, e.g. structural or binding protein can retain activity under conditions comprising a temperature range of between about 1\u00b0 C. to about 5\u00b0 C., between about 5\u00b0 C. to about 15\u00b0 C., between about 15\u00b0 C. to about 25\u00b0 C., between about 25\u00b0 C. to about 37\u00b0 C., between about 37\u00b0 C. to about 95\u00b0 C., between about 55\u00b0 C. to about 85\u00b0 C., between about 70\u00b0 C. to about 75\u00b0 C., or between about 90\u00b0 C. to about 95\u00b0 C., or more. In another aspect, the polypeptide, enzyme, protein, e.g. structural or binding protein can be thermotolerant. The polypeptide, enzyme, protein, e.g. structural or binding protein can retain activity after exposure to a temperature in the range from greater than 37\u00b0 C. to about 95\u00b0 C., or in the range from greater than 55\u00b0 C. to about 85\u00b0 C. In one aspect, the polypeptide, enzyme, protein, e.g. structural or binding protein can retain activity after exposure to a temperature in the range from greater than 90\u00b0 C. to about 95\u00b0 C. at pH 4.5.","Another aspect of the invention provides an isolated or recombinant polypeptide or peptide including at least 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95 or 100 or more consecutive bases of a polypeptide or peptide sequence of the invention, sequences substantially identical thereto, and the sequences complementary thereto. The peptide can be, e.g., an immunogenic fragment, a motif (e.g., a binding site), a signal sequence, a prepro sequence or an active site.","The invention provides isolated or recombinant nucleic acids comprising a sequence encoding a polypeptide, enzyme, protein, e.g. structural or binding protein having any of the activities as set forth in Tables 1, 2 or 3, and a signal sequence, wherein the nucleic acid comprises a sequence of the invention. In one aspect, the isolated or recombinant polypeptide can comprise the polypeptide of the invention comprising a heterologous signal sequence or a heterologous preprosequence, such as a heterologous enzyme or non-enzyme signal sequence. The invention provides isolated or recombinant nucleic acids comprising a sequence encoding a polypeptide, enzyme, protein, e.g. structural or binding protein having any of the activities as set forth in Tables 1, 2 or 3, wherein the sequence does not contain a signal sequence and the nucleic acid comprises a sequence of the invention. In one aspect, the invention provides an isolated or recombinant polypeptide comprising a polypeptide of the invention lacking all or part of a signal sequence.","In one aspect, the invention provides chimeric proteins comprising a first domain comprising a signal sequence of the invention and at least a second domain. The protein can be a fusion protein. The second domain can comprise an enzyme. The enzyme can be a non-enzyme.","The invention provides chimeric polypeptides comprising at least a first domain comprising signal peptide (SP), a prepro sequence and\/or a catalytic domain (CD) of the invention and at least a second domain comprising a heterologous polypeptide or peptide, wherein the heterologous polypeptide or peptide is not naturally associated with the signal peptide (SP), prepro sequence and\/or catalytic domain (CD). In one aspect, the heterologous polypeptide or peptide is not an enzyme. The heterologous polypeptide or peptide can be amino terminal to, carboxy terminal to or on both ends of the signal peptide (SP), prepro sequence and\/or catalytic domain (CD).","The invention provides isolated or recombinant nucleic acids encoding a chimeric polypeptide, wherein the chimeric polypeptide comprises at least a first domain comprising signal peptide (SP), a prepro domain and\/or a catalytic domain (CD) of the invention and at least a second domain comprising a heterologous polypeptide or peptide, wherein the heterologous polypeptide or peptide is not naturally associated with the signal peptide (SP), prepro domain and\/or catalytic domain (CD).","The invention provides isolated or recombinant signal sequences (e.g., signal peptides) consisting of or comprising a sequence as set forth in residues 1 to 14, 1 to 15, 1 to 16, 1 to 17, 1 to 18, 1 to 19, 1 to 20, 1 to 21, 1 to 22, 1 to 23, 1 to 24, 1 to 25, 1 to 26, 1 to 27, 1 to 28, 1 to 28, 1 to 30, 1 to 31, 1 to 32, 1 to 33, 1 to 34, 1 to 35, 1 to 36, 1 to 37, 1 to 38, 1 to 40, 1 to 41, 1 to 42, 1 to 43, 1 to 44, 1 to 45, 1 to 46 or 1 to 47, of a polypeptide of the invention, including the exemplary polypeptides of the invention (including SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, etc., and all polypeptides disclosed in the SEQ ID listing, which include all even numbered SEQ ID NO:s from SEQ ID NO:2 through SEQ ID NO:26,898). In one aspect, the invention provides signal sequences comprising the first 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70 or more amino terminal residues of a polypeptide of the invention.","In one aspect, the enzyme, structural or binding activity comprises a specific activity at about 37\u00b0 C. in the range from about 1 to about 1200 units per milligram of protein, or, about 100 to about 1000 units per milligram of protein. In another aspect, the polypeptide, enzyme, protein, e.g. structural or binding protein activity comprises a specific activity from about 100 to about 1000 units per milligram of protein, or, from about 500 to about 750 units per milligram of protein. Alternatively, the enzyme, structural or binding activity comprises a specific activity at 37\u00b0 C. in the range from about 1 to about 750 units per milligram of protein, or, from about 500 to about 1200 units per milligram of protein. In one aspect, the enzyme, structural or binding activity comprises a specific activity at 37\u00b0 C. in the range from about 1 to about 500 units per milligram of protein, or, from about 750 to about 1000 units per milligram of protein. In another aspect, the enzyme, structural or binding activity comprises a specific activity at 37\u00b0 C. in the range from about 1 to about 250 units per milligram of protein. Alternatively, the enzyme, structural or binding activity comprises a specific activity at 37\u00b0 C. in the range from about 1 to about 100 units per milligram of protein.","In another aspect, thermotolerance comprises retention of at least half of the specific activity of the enzyme, structural or binding protein at 37\u00b0 C. after being heated to the elevated temperature. Alternatively, thermotolerance can comprise retention of specific activity at 37\u00b0 C. in the range from about 1 to about 1200 units per milligram of protein, or, from about 500 to about 1000 units per milligram of protein, after being heated to the elevated temperature. In another aspect, thermotolerance can comprise retention of specific activity at 37\u00b0 C. in the range from about 1 to about 500 units per milligram of protein after being heated to the elevated temperature.","The invention provides the isolated or recombinant polypeptide of the invention, wherein the polypeptide comprises at least one glycosylation site. In one aspect, glycosylation can be an N-linked glycosylation. In one aspect, the polypeptide can be glycosylated after being expressed in a or a ","In one aspect, the polypeptide, enzyme, protein, e.g. structural or binding protein can retain activity under conditions comprising about pH 6.5, pH 6, pH 5.5, pH 5, pH 4.5 or pH 4. In another aspect, the polypeptide, enzyme, protein, e.g. structural or binding protein can retain activity under conditions comprising about pH 7, pH 7.5 pH 8.0, pH 8.5, pH 9, pH 9.5, pH 10, pH 10.5 or pH 11. In one aspect, the polypeptide can retain an enzyme, structural or binding activity after exposure to conditions comprising about pH 6.5, pH 6, pH 5.5, pH 5, pH 4.5 or pH 4. In another aspect, the polypeptide can retain enzyme, structural or binding activity after exposure to conditions comprising about pH 7, pH 7.5 pH 8.0, pH 8.5, pH 9, pH 9.5, pH 10, pH 10.5 or pH 11.","In one aspect, the polypeptide, enzyme, protein, e.g. structural or binding protein of the invention has activity at under alkaline conditions, e.g., the alkaline conditions of the gut, e.g., the small intestine. In one aspect, the polypeptide, enzyme, protein, e.g. structural or binding protein can retain activity after exposure to the acidic pH of the stomach.","The invention provides protein preparations comprising a polypeptide of the invention, wherein the protein preparation comprises a liquid, a solid or a gel.","The invention provides heterodimers comprising a polypeptide of the invention and a second protein or domain. The second member of the heterodimer can be a different enzyme, a different enzyme or another protein. In one aspect, the second domain can be a polypeptide and the heterodimer can be a fusion protein. In one aspect, the second domain can be an epitope or a tag. In one aspect, the invention provides homodimers comprising a polypeptide of the invention.","The invention provides immobilized polypeptides having enzyme, structural or binding activity, wherein the polypeptide comprises a polypeptide of the invention, a polypeptide encoded by a nucleic acid of the invention, or a polypeptide comprising a polypeptide of the invention and a second domain. In one aspect, the polypeptide can be immobilized on a cell, a metal, a resin, a polymer, a ceramic, a glass, a microelectrode, a graphitic particle, a bead, a gel, a plate, an array or a capillary tube.","The invention provides arrays comprising an immobilized nucleic acid of the invention. The invention provides arrays comprising an antibody of the invention.","The invention provides isolated or recombinant antibodies that specifically bind to a polypeptide of the invention or to a polypeptide encoded by a nucleic acid of the invention. These antibodies of the invention can be a monoclonal or a polyclonal antibody. The invention provides hybridomas comprising an antibody of the invention, e.g., an antibody that specifically binds to a polypeptide of the invention or to a polypeptide encoded by a nucleic acid of the invention. The invention provides nucleic acids encoding these antibodies.","The invention provides method of isolating or identifying a polypeptide having enzyme, structural or binding activity comprising the steps of: (a) providing an antibody of the invention; (b) providing a sample comprising polypeptides; and (c) contacting the sample of step (b) with the antibody of step (a) under conditions wherein the antibody can specifically bind to the polypeptide, thereby isolating or identifying a polypeptide having an enzyme, structural or binding activity.","The invention provides methods of making an anti-polypeptide, anti-enzyme, or anti-protein, e.g. anti-structural or anti-binding protein, antibody comprising administering to a non-human animal a nucleic acid of the invention or a polypeptide of the invention or subsequences thereof in an amount sufficient to generate a humoral immune response, thereby making an anti-polypeptide, anti-enzyme, or anti-protein, e.g. anti-structural or anti-binding protein, antibody. The invention provides methods of making an anti-polypeptide, anti-enzyme, or anti-protein, e.g. anti-structural or anti-binding protein, immune comprising administering to a non-human animal a nucleic acid of the invention or a polypeptide of the invention or subsequences thereof in an amount sufficient to generate an immune response.","The invention provides methods of producing a recombinant polypeptide comprising the steps of: (a) providing a nucleic acid of the invention operably linked to a promoter; and (b) expressing the nucleic acid of step (a) under conditions that allow expression of the polypeptide, thereby producing a recombinant polypeptide. In one aspect, the method can further comprise transforming a host cell with the nucleic acid of step (a) followed by expressing the nucleic acid of step (a), thereby producing a recombinant polypeptide in a transformed cell.","The invention provides methods for identifying a polypeptide having enzyme, structural or binding activity comprising the following steps: (a) providing a polypeptide of the invention; or a polypeptide encoded by a nucleic acid of the invention; (b) providing an enzyme, structural or binding activity substrate; and (c) contacting the polypeptide or a fragment or variant thereof of step (a) with the substrate of step (b) and detecting a decrease in the amount of substrate or an increase in the amount of a reaction product, wherein a decrease in the amount of the substrate or an increase in the amount of the reaction product detects a polypeptide having a enzyme, structural or binding activity.","The invention provides methods for identifying a polypeptide, enzyme, protein, e.g. structural or binding protein, substrate comprising the following steps: (a) providing a polypeptide of the invention; or a polypeptide encoded by a nucleic acid of the invention; (b) providing a test substrate; and (c) contacting the polypeptide of step (a) with the test substrate of step (b) and detecting a decrease in the amount of substrate or an increase in the amount of reaction product, wherein a decrease in the amount of the substrate or an increase in the amount of a reaction product identifies the test substrate as a polypeptide, enzyme, protein, e.g. structural or binding protein, substrate.","The invention provides methods of determining whether a test compound specifically binds to a polypeptide comprising the following steps: (a) expressing a nucleic acid or a vector comprising the nucleic acid under conditions permissive for translation of the nucleic acid to a polypeptide, wherein the nucleic acid comprises a nucleic acid of the invention, or, providing a polypeptide of the invention; (b) providing a test compound; (c) contacting the polypeptide with the test compound; and (d) determining whether the test compound of step (b) specifically binds to the polypeptide.","The invention provides methods for identifying a modulator of a enzyme, structural or binding activity comprising the following steps: (a) providing a polypeptide of the invention or a polypeptide encoded by a nucleic acid of the invention; (b) providing a test compound; (c) contacting the polypeptide of step (a) with the test compound of step (b) and measuring an activity of the polypeptide, enzyme, protein, e.g. structural or binding protein, wherein a change in the enzyme, structural or binding activity measured in the presence of the test compound compared to the activity in the absence of the test compound provides a determination that the test compound modulates the enzyme, structural or binding activity. In one aspect, the enzyme, structural or binding activity can be measured by providing a polypeptide, enzyme, protein, e.g. structural or binding protein, substrate and detecting a decrease in the amount of the substrate or an increase in the amount of a reaction product, or, an increase in the amount of the substrate or a decrease in the amount of a reaction product. A decrease in the amount of the substrate or an increase in the amount of the reaction product with the test compound as compared to the amount of substrate or reaction product without the test compound identifies the test compound as an activator of enzyme, structural or binding activity. An increase in the amount of the substrate or a decrease in the amount of the reaction product with the test compound as compared to the amount of substrate or reaction product without the test compound identifies the test compound as an inhibitor of enzyme, structural or binding activity.","The invention provides computer systems comprising a processor and a data storage device wherein said data storage device has stored thereon a polypeptide sequence or a nucleic acid sequence of the invention (e.g., a polypeptide encoded by a nucleic acid of the invention). In one aspect, the computer system can further comprise a sequence comparison algorithm and a data storage device having at least one reference sequence stored thereon. In another aspect, the sequence comparison algorithm comprises a computer program that indicates polymorphisms. In one aspect, the computer system can further comprise an identifier that identifies one or more features in said sequence. The invention provides computer readable media having stored thereon a polypeptide sequence or a nucleic acid sequence of the invention. The invention provides methods for identifying a feature in a sequence comprising the steps of: (a) reading the sequence using a computer program which identifies one or more features in a sequence, wherein the sequence comprises a polypeptide sequence or a nucleic acid sequence of the invention; and (b) identifying one or more features in the sequence with the computer program. The invention provides methods for comparing a first sequence to a second sequence comprising the steps of: (a) reading the first sequence and the second sequence through use of a computer program which compares sequences, wherein the first sequence comprises a polypeptide sequence or a nucleic acid sequence of the invention; and (b) determining differences between the first sequence and the second sequence with the computer program. The step of determining differences between the first sequence and the second sequence can further comprise the step of identifying polymorphisms. In one aspect, the method can further comprise an identifier that identifies one or more features in a sequence. In another aspect, the method can comprise reading the first sequence using a computer program and identifying one or more features in the sequence.","The invention provides methods for isolating or recovering a nucleic acid encoding a polypeptide, enzyme, protein, e.g. structural or binding protein, from an environmental sample comprising the steps of: (a) providing an amplification primer sequence pair for amplifying a nucleic acid encoding a polypeptide, enzyme, protein, e.g. structural or binding protein, wherein the primer pair is capable of amplifying a nucleic acid of the invention; (b) isolating a nucleic acid from the environmental sample or treating the environmental sample such that nucleic acid in the sample is accessible for hybridization to the amplification primer pair; and, (c) combining the nucleic acid of step (b) with the amplification primer pair of step (a) and amplifying nucleic acid from the environmental sample, thereby isolating or recovering a nucleic acid encoding a polypeptide, enzyme, protein, e.g. structural or binding protein from an environmental sample. One or each member of the amplification primer sequence pair can comprise an oligonucleotide comprising an amplification primer sequence pair of the invention, e.g., having at least about 10 to 50 consecutive bases of a sequence of the invention.","The invention provides methods for isolating or recovering a nucleic acid encoding a polypeptide, enzyme, protein, e.g. structural or binding protein from an environmental sample comprising the steps of: (a) providing a polynucleotide probe comprising a nucleic acid of the invention or a subsequence thereof; (b) isolating a nucleic acid from the environmental sample or treating the environmental sample such that nucleic acid in the sample is accessible for hybridization to a polynucleotide probe of step (a); (c) combining the isolated nucleic acid or the treated environmental sample of step (b) with the polynucleotide probe of step (a); and (d) isolating a nucleic acid that specifically hybridizes with the polynucleotide probe of step (a), thereby isolating or recovering a nucleic acid encoding a polypeptide, enzyme, protein, e.g. structural or binding protein from an environmental sample. The environmental sample can comprise a water sample, a liquid sample, a soil sample, an air sample or a biological sample. In one aspect, the biological sample can be derived from a bacterial cell, a protozoan cell, an insect cell, a yeast cell, a plant cell, a fungal cell or a mammalian cell.","The invention provides methods of generating a variant of a nucleic acid encoding a polypeptide having an enzyme, structural or binding activity comprising the steps of: (a) providing a template nucleic acid comprising a nucleic acid of the invention; and (b) modifying, deleting or adding one or more nucleotides in the template sequence, or a combination thereof, to generate a variant of the template nucleic acid. In one aspect, the method can further comprise expressing the variant nucleic acid to generate a variant the polypeptide, enzyme, protein, e.g. structural or binding protein. The modifications, additions or deletions can be introduced by a method comprising error-prone PCR, shuffling, oligonucleotide-directed mutagenesis, assembly PCR, sexual PCR mutagenesis, in vivo mutagenesis, cassette mutagenesis, recursive ensemble mutagenesis, exponential ensemble mutagenesis, site-specific mutagenesis, gene reassembly, Gene Site Saturation Mutagenesis (GSSM), synthetic ligation reassembly (SLR) or a combination thereof. In another aspect, the modifications, additions or deletions are introduced by a method comprising recombination, recursive sequence recombination, phosphothioate-modified DNA mutagenesis, uracil-containing template mutagenesis, gapped duplex mutagenesis, point mismatch repair mutagenesis, repair-deficient host strain mutagenesis, chemical mutagenesis, radiogenic mutagenesis, deletion mutagenesis, restriction-selection mutagenesis, restriction-purification mutagenesis, artificial gene synthesis, ensemble mutagenesis, chimeric nucleic acid multimer creation and a combination thereof.","In one aspect, the method can be iteratively repeated until a polypeptide, enzyme, protein, e.g. structural or binding protein having an altered or different activity or an altered or different stability from that of a polypeptide encoded by the template nucleic acid is produced. In one aspect, the variant the polypeptide, enzyme, protein, e.g. structural or binding protein is thermotolerant, and retains some activity after being exposed to an elevated temperature. In another aspect, the variant the polypeptide, enzyme, protein, e.g. structural or binding protein has increased glycosylation as compared to the polypeptide, enzyme, protein, e.g. structural or binding protein encoded by a template nucleic acid. Alternatively, the variant the polypeptide, enzyme, protein, e.g. structural or binding protein has an enzyme, structural or binding activity under a high temperature, wherein the polypeptide, enzyme, protein, e.g. structural or binding protein encoded by the template nucleic acid is not active under the high temperature. In one aspect, the method can be iteratively repeated until a polypeptide, enzyme, protein, e.g. structural or binding protein coding sequence having an altered codon usage from that of the template nucleic acid is produced. In another aspect, the method can be iteratively repeated until a polypeptide, enzyme, protein, e.g. structural or binding protein gene having higher or lower level of message expression or stability from that of the template nucleic acid is produced.","The invention provides methods for modifying codons in a nucleic acid encoding a polypeptide having an enzyme, structural or binding activity to increase its expression in a host cell, the method comprising the following steps: (a) providing a nucleic acid of the invention encoding a polypeptide having an enzyme, structural or binding activity; and, (b) identifying a non-preferred or a less preferred codon in the nucleic acid of step (a) and replacing it with a preferred or neutrally used codon encoding the same amino acid as the replaced codon, wherein a preferred codon is a codon over-represented in coding sequences in genes in the host cell and a non-preferred or less preferred codon is a codon under-represented in coding sequences in genes in the host cell, thereby modifying the nucleic acid to increase its expression in a host cell.","The invention provides methods for modifying codons in a nucleic acid encoding a polypeptide having an enzyme, structural or binding activity; the method comprising the following steps: (a) providing a nucleic acid of the invention; and, (b) identifying a codon in the nucleic acid of step (a) and replacing it with a different codon encoding the same amino acid as the replaced codon, thereby modifying codons in a nucleic acid encoding a polypeptide, enzyme, protein, e.g. structural or binding protein.","The invention provides methods for modifying codons in a nucleic acid encoding a polypeptide having an enzyme, structural or binding activity to increase its expression in a host cell, the method comprising the following steps: (a) providing a nucleic acid of the invention encoding a polypeptide, enzyme, protein, e.g. structural or binding protein, polypeptide; and, (b) identifying a non-preferred or a less preferred codon in the nucleic acid of step (a) and replacing it with a preferred or neutrally used codon encoding the same amino acid as the replaced codon, wherein a preferred codon is a codon over-represented in coding sequences in genes in the host cell and a non-preferred or less preferred codon is a codon under-represented in coding sequences in genes in the host cell, thereby modifying the nucleic acid to increase its expression in a host cell.","The invention provides methods for modifying a codon in a nucleic acid encoding a polypeptide having an enzyme, structural or binding activity to decrease its expression in a host cell, the method comprising the following steps: (a) providing a nucleic acid of the invention; and (b) identifying at least one preferred codon in the nucleic acid of step (a) and replacing it with a non-preferred or less preferred codon encoding the same amino acid as the replaced codon, wherein a preferred codon is a codon over-represented in coding sequences in genes in a host cell and a non-preferred or less preferred codon is a codon under-represented in coding sequences in genes in the host cell, thereby modifying the nucleic acid to decrease its expression in a host cell. In one aspect, the host cell can be a bacterial cell, a fungal cell, an insect cell, a yeast cell, a plant cell or a mammalian cell.","The invention provides methods for producing a library of nucleic acids encoding a plurality of modified polypeptides, enzymes, proteins, e.g. structural or binding proteins, active sites or substrate binding sites, wherein the modified active sites or substrate binding sites are derived from a first nucleic acid comprising a sequence encoding a first active site or a first substrate binding site the method comprising the following steps: (a) providing a first nucleic acid encoding a first active site or first substrate binding site, wherein the first nucleic acid sequence comprises a sequence that hybridizes under stringent conditions to a nucleic acid of the invention, and the nucleic acid encodes a polypeptide, enzyme, protein, e.g. structural or binding protein, active site or a polypeptide, enzyme, protein, e.g. structural or binding protein, substrate binding site; (b) providing a set of mutagenic oligonucleotides that encode naturally-occurring amino acid variants at a plurality of targeted codons in the first nucleic acid; and, (c) using the set of mutagenic oligonucleotides to generate a set of active site-encoding or substrate binding site-encoding variant nucleic acids encoding a range of amino acid variations at each amino acid codon that was mutagenized, thereby producing a library of nucleic acids encoding a plurality of modified the polypeptide, enzyme, protein, e.g. structural or binding protein, active sites or substrate binding sites. In one aspect, the method comprises mutagenizing the first nucleic acid of step (a) by a method comprising an optimized directed evolution system, Gene Site Saturation Mutagenesis (GSSM), synthetic ligation reassembly (SLR), error-prone PCR, shuffling, oligonucleotide-directed mutagenesis, assembly PCR, sexual PCR mutagenesis, in vivo mutagenesis, cassette mutagenesis, recursive ensemble mutagenesis, exponential ensemble mutagenesis, site-specific mutagenesis, gene reassembly, and a combination thereof. In another aspect, the method comprises mutagenizing the first nucleic acid of step (a) or variants by a method comprising recombination, recursive sequence recombination, phosphothioate-modified DNA mutagenesis, uracil-containing template mutagenesis, gapped duplex mutagenesis, point mismatch repair mutagenesis, repair-deficient host strain mutagenesis, chemical mutagenesis, radiogenic mutagenesis, deletion mutagenesis, restriction-selection mutagenesis, restriction-purification mutagenesis, artificial gene synthesis, ensemble mutagenesis, chimeric nucleic acid multimer creation and a combination thereof.","The invention provides methods for making a small molecule comprising the steps of: (a) providing a plurality of biosynthetic enzymes capable of synthesizing or modifying a small molecule, wherein one of the enzymes comprises an enzyme encoded by a nucleic acid of the invention; (b) providing a substrate for at least one of the enzymes of step (a); and, (c) reacting the substrate of step (b) with the enzymes under conditions that facilitate a plurality of biocatalytic reactions to generate a small molecule by a series of biocatalytic reactions.","The invention provides methods for modifying a small molecule comprising the steps: (a) providing a enzyme encoded by a nucleic acid of the invention; (b) providing a small molecule; and, (c) reacting the enzyme of step (a) with the small molecule of step (b) under conditions that facilitate an enzymatic reaction catalyzed by the enzyme, thereby modifying a small molecule by an enzymatic reaction. In one aspect, the method comprises providing a plurality of small molecule substrates for the enzyme of step (a), thereby generating a library of modified small molecules produced by at least one enzymatic reaction catalyzed by the enzyme. In one aspect, the method further comprises a plurality of additional enzymes under conditions that facilitate a plurality of biocatalytic reactions by the enzymes to form a library of modified small molecules produced by the plurality of enzymatic reactions. In one aspect, the method further comprises the step of testing the library to determine if a particular modified small molecule that exhibits a desired activity is present within the library. The step of testing the library can further comprises the steps of systematically eliminating all but one of the biocatalytic reactions used to produce a portion of the plurality of the modified small molecules within the library by testing the portion of the modified small molecule for the presence or absence of the particular modified small molecule with a desired activity, and identifying at least one specific biocatalytic reaction that produces the particular modified small molecule of desired activity.","The invention provides methods for determining a functional fragment of a polypeptide, enzyme, protein, e.g. structural or binding protein, comprising the steps of: (a) providing a polypeptide, enzyme, protein, e.g. structural or binding protein, wherein the enzyme comprises a polypeptide of the invention, or a polypeptide encoded by a nucleic acid of the invention, or a subsequence thereof; and (b) deleting a plurality of amino acid residues from the sequence of step (a) and testing the remaining subsequence for an enzyme, structural or binding activity, thereby determining a functional fragment of a polypeptide, enzyme, protein, e.g. structural or binding protein. In one aspect, the polypeptide, enzyme, protein, e.g. structural or binding protein activity is measured by providing a polypeptide, enzyme, protein, e.g. structural or binding protein, substrate and detecting a decrease in the amount of the substrate or an increase in the amount of a reaction product.","The invention provides methods for whole cell engineering of new or modified phenotypes by using real-time metabolic flux analysis, the method comprising the following steps: (a) making a modified cell by modifying the genetic composition of a cell, wherein the genetic composition is modified by addition to the cell of a nucleic acid of the invention; (b) culturing the modified cell to generate a plurality of modified cells; (c) measuring at least one metabolic parameter of the cell by monitoring the cell culture of step (b) in real time; and, (d) analyzing the data of step (c) to determine if the measured parameter differs from a comparable measurement in an unmodified cell under similar conditions, thereby identifying an engineered phenotype in the cell using real-time metabolic flux analysis. In one aspect, the genetic composition of the cell can be modified by a method comprising deletion of a sequence or modification of a sequence in the cell, or, knocking out the expression of a gene. In one aspect, the method can further comprise selecting a cell comprising a newly engineered phenotype. In another aspect, the method can comprise culturing the selected cell, thereby generating a new cell strain comprising a newly engineered phenotype.","The invention provides methods of increasing thermotolerance or thermostability of a polypeptide, enzyme, protein, e.g. structural or binding protein, polypeptide, the method comprising glycosylating a polypeptide, enzyme, protein, e.g. structural or binding protein, wherein the polypeptide, enzyme, protein, e.g. structural or binding protein comprises at least thirty contiguous amino acids of a polypeptide of the invention; or a polypeptide encoded by a nucleic acid sequence of the invention, thereby increasing thermotolerance or thermostability of the polypeptide, enzyme, protein, e.g. structural or binding protein. In one aspect, the polypeptide, enzyme, protein, e.g. structural or binding protein specific activity can be thermostable or thermotolerant at a temperature in the range from greater than about 37\u00b0 C. to about 95\u00b0 C.","The invention provides methods for overexpressing a recombinant polypeptide, enzyme, protein, e.g. structural or binding protein, in a cell comprising expressing a vector comprising a nucleic acid comprising a nucleic acid of the invention or a nucleic acid sequence of the invention, wherein the sequence identities are determined by analysis with a sequence comparison algorithm or by visual inspection, wherein overexpression is effected by use of a high activity promoter, a dicistronic vector or by gene amplification of the vector.","The invention provides methods of making a transgenic plant comprising the following steps: (a) introducing a heterologous nucleic acid sequence into the cell, wherein the heterologous nucleic sequence comprises a nucleic acid sequence of the invention, thereby producing a transformed plant cell; and (b) producing a transgenic plant from the transformed cell. In one aspect, the step (a) can further comprise introducing the heterologous nucleic acid sequence by electroporation or microinjection of plant cell protoplasts. In another aspect, the step (a) can further comprise introducing the heterologous nucleic acid sequence directly to plant tissue by DNA particle bombardment. Alternatively, the step (a) can further comprise introducing the heterologous nucleic acid sequence into the plant cell DNA using an host. In one aspect, the plant cell can be a potato, corn, rice, wheat, tobacco, or barley cell.","The invention provides methods of expressing a heterologous nucleic acid sequence in a plant cell comprising the following steps: (a) transforming the plant cell with a heterologous nucleic acid sequence operably linked to a promoter, wherein the heterologous nucleic sequence comprises a nucleic acid of the invention; (b) growing the plant under conditions wherein the heterologous nucleic acids sequence is expressed in the plant cell. The invention provides methods of expressing a heterologous nucleic acid sequence in a plant cell comprising the following steps: (a) transforming the plant cell with a heterologous nucleic acid sequence operably linked to a promoter, wherein the heterologous nucleic sequence comprises a sequence of the invention; (b) growing the plant under conditions wherein the heterologous nucleic acids sequence is expressed in the plant cell.","The invention provides feeds or foods comprising a polypeptide of the invention, or a polypeptide encoded by a nucleic acid of the invention. In one aspect, the invention provides a food, feed, a liquid, e.g., a beverage (such as a fruit juice or a beer), a bread or a dough or a bread product, or a beverage precursor (e.g., a wort), comprising a polypeptide of the invention. The invention provides food or nutritional supplements for an animal comprising a polypeptide of the invention, e.g., a polypeptide encoded by the nucleic acid of the invention.","In one aspect, the polypeptide in the food or nutritional supplement can be glycosylated. The invention provides edible enzyme delivery matrices comprising a polypeptide of the invention, e.g., a polypeptide encoded by the nucleic acid of the invention. In one aspect, the delivery matrix comprises a pellet. In one aspect, the polypeptide can be glycosylated. In one aspect, the polypeptide, enzyme, protein, e.g. structural or binding protein activity is thermotolerant. In another aspect, the polypeptide, enzyme, protein, e.g. structural or binding protein activity is thermostable.","The invention provides a food, a feed or a nutritional supplement comprising a polypeptide of the invention. The invention provides methods for utilizing a polypeptide, enzyme, protein, e.g. structural or binding protein, as a nutritional supplement in an animal diet, the method comprising: preparing a nutritional supplement containing a polypeptide, enzyme, protein, e.g. structural or binding protein, comprising at least thirty contiguous amino acids of a polypeptide of the invention; and administering the nutritional supplement to an animal. The animal can be a human, a ruminant or a monogastric animal. The polypeptide, enzyme, protein, e.g. structural or binding protein can be prepared by expression of a polynucleotide encoding the polypeptide, enzyme, protein, e.g. structural or binding protein in an organism selected from the group consisting of a bacterium, a yeast, a plant, an insect, a fungus and an animal. The organism can be selected from the group consisting of an sp., sp. and sp.","The invention provides edible enzyme delivery matrix comprising thermostable recombinant polypeptide, enzyme, protein, e.g. structural or binding protein of the invention. The invention provides methods for delivering a polypeptide, enzyme, protein, e.g. structural or binding protein, supplement to an animal, the method comprising: preparing an edible enzyme delivery matrix in the form of pellets comprising a granulate edible carrier and thermostable recombinant polypeptide, enzyme, protein, e.g. structural or binding protein, wherein the pellets readily disperse the polypeptide, enzyme, protein, e.g. structural or binding protein contained therein into aqueous media, and administering the edible enzyme delivery matrix to the animal. The recombinant polypeptide, enzyme, protein, e.g. structural or binding protein can comprise a polypeptide of the invention. The polypeptide, enzyme, protein, e.g. structural or binding protein can be glycosylated to provide thermostability at pelletizing conditions. The delivery matrix can be formed by pelletizing a mixture comprising a grain germ and a polypeptide, enzyme, protein, e.g. structural or binding protein. The pelletizing conditions can include application of steam. The pelletizing conditions can comprise application of a temperature in excess of about 80\u00b0 C. for about 5 minutes and the enzyme retains a specific activity of at least 350 to about 900 units per milligram of enzyme.","In one aspect, invention provides a pharmaceutical composition comprising a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention, or a polypeptide encoded by a nucleic acid of the invention. In one aspect, the pharmaceutical composition acts as a digestive aid.","The details of one or more aspects of the invention are set forth in the accompanying drawings and the description below. Other features, objects, and advantages of the invention will be apparent from the description and drawings, and from the claims.","All publications, patents, patent applications, GenBank sequences and ATCC deposits, cited herein are hereby expressly incorporated by reference for all purposes.","Like reference symbols in the various drawings indicate like elements.","The invention provides isolated and recombinant polypeptides, including enzymes, structural proteins and binding proteins, polynucleotides encoding these polypeptides, and methods of making and using these polynucleotides and polypeptides. The polypeptides of the invention, and the polynucleotides encoding the polypeptides of the invention, encompass many classes of enzymes, structural proteins and binding proteins. In one aspect, the enzymes and proteins of the invention comprise, e.g. aldolases, alpha-galactosidases, amidases, e.g. secondary amidases, amylases, catalases, carotenoid pathway enzymes, dehalogenases, endoglucanases, epoxide hydrolases, esterases, hydrolases, glucosidases, glycosidases, inteins, isomerases, laccases, lipases, monooxygenases, nitroreductases, nitrilases, P450 enzymes, pectate lyases, phosphatases, phospholipases, phytases, polymerases and xylanases, which are more specifically described below. The invention also provides isolated and recombinant polypeptides, including enzymes, structural proteins and binding proteins, polynucleotides encoding these polypeptides, having the activities described in Table 1, Table 2 or Table 3, below.","Aldolases","In one aspect, the invention provides aldolases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having an aldolase activity, including thermostable and thermotolerant aldolase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides. In one aspect, the aldolase activity comprises catalysis of the formation of a carbon-carbon bond. In one aspect, the aldolase activity comprises an aldol condensation. The aldol condensation can have an aldol donor substrate comprising an acetaldehyde and an aldol acceptor substrate comprising an aldehyde. The aldol condensation can yield a product of a single chirality. In one aspect, the aldolase activity is enantioselective. The aldolase activity can comprise a 2-deoxyribose-5-phosphate aldolase (DERA) activity. The aldolase activity can comprise catalysis of the condensation of acetaldehyde as donor and a 2(R)-hydroxy-3-(hydroxy or mercapto)-propionaldehyde derivative to form a 2-deoxysugar. The aldolase activity can comprise catalysis of the condensation of acetaldehyde as donor and a 2-substituted acetaldehyde acceptor to form a 2,4,6-trideoxyhexose via a 4-substituted-3-hydroxybutanal intermediate. The aldolase activity can comprise catalysis of the generation of chiral aldehydes using two acetaldehydes as substrates. The aldolase activity can comprises enantioselective assembling of chiral \u03b2,\u03b4-dihydroxyheptanoic acid side chains. The aldolase activity can comprise enantioselective assembling of the core of [R\u2014(R*,R*)]-2-(4-fluorophenyl)-b,d-dihydroxy-5-(1-methylethyl)-3-phenyl-4-(phenylamino)-carbonyl]-1H-pyrrole-1-heptanoic acid (Atorvastatin, or LIPITOR\u2122), rosuvastatin (CRESTOR\u2122) and\/or fluvastatin (LESCOL\u2122). The aldolase activity can comprise, with an oxidation step, synthesis of a 3R,5S-6-chloro-2,4,6-trideoxy-erythro-hexonolactone.","Alpha-Galactosidases","In one aspect, the invention provides alpha-galactosidases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having an alpha-galactosidase activity, including thermostable and thermotolerant alpha-galactosidase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","An alpha galactosidase hydrolyses the non-reducing terminal alpha 1-3,4,6 linked galactose from poly- and oligosaccharides. These saccharides are commonly found in legumes and are difficult to digest. As such, alpha-galactosidases can be used as a digestive aid to break down raffinose, stachyose, and verbascose, found in such foods as beans and other gassy foods.","Amidases","In one aspect, the invention provides amidases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having an amidase activity, including thermostable and thermotolerant amidase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides. In one aspect, the amidases of the invention are used in the removal of arginine, phenylalanine or methionine from the N-terminal end of peptides in peptide or peptidomimetic synthesis. In one aspect, the enzyme of the invention, e.g. an amidase, is selective for the L, or \u201cnatural\u201d enantiomer of the amino acid derivatives and is therefore useful for the production of optically active compounds. These reactions can be performed in the presence of the chemically more reactive ester functionality, a step which is very difficult to achieve with nonenzymatic methods. The enzyme is also able to tolerate high temperatures (at least 70\u00b0 C.), and high concentrations of organic solvents (>40% DMSO), both of which cause a disruption of secondary structure in peptides, which enables cleavage of otherwise resistant bonds.","Secondary Amidases","In one aspect, the invention provides secondary amidases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having a secondary amidase activity, including thermostable and thermotolerant secondary amidase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","Secondary amidases include a variety of useful enzymes including peptidases, proteases, and hydantoinases. This class of enzymes can be used in a range of commercial applications. For example, secondary amidases can be used to: 1) increase flavor in food, in particular cheese (known as enzyme ripened cheese); 2) promote bacterial and fungal killing; 3) modify and de-protect fine chemical intermediates 4) synthesize peptide bonds; 5) and carry out chiral resolutions. Particularly, there is a need in the art for an enzyme capable of hydrolyzing Cephalosporin C.","Amylases","In one aspect, the invention provides amylases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having an amylase activity, including thermostable and thermotolerant amylase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","In one aspect, the polypeptides of the invention can be used as amylases, for example, alpha amylases or glucoamylases, to catalyze the hydrolysis of starch into sugars. In one aspect, the invention is directed to polypeptides having thermostable amylase activity, such as alpha amylases or glucoamylase activity, e.g., a 1,4-alpha-D-glucan glucohydrolase activity. In one aspect, the polypeptides of the invention can be used as amylases, for example, alpha amylases or glucoamylases, to catalyze the hydrolysis of starch into sugars, such as glucose. The invention is also directed to nucleic acid constructs, vectors, and host cells comprising the nucleic acid sequences of the invention as well as recombinant methods for producing the polypeptides of the invention. The invention is also directed to the use of amylases of the invention in starch conversion processes, including production of high fructose corn syrup (HFCS), ethanol, dextrose, and dextrose syrups.","Commercially, glucoamylases are used to further hydrolyze cornstarch, which has already been partially hydrolyzed with an alpha-amylase. The glucose produced in this reaction may then be converted to a mixture of glucose and fructose by a glucose isomerase enzyme. This mixture, or one enriched with fructose, is the high fructose corn syrup commercialized throughout the world. In general, starch to fructose processing consists of four steps: liquefaction of granular starch, saccharification of the liquefied starch into dextrose, purification, and isomerization to fructose. The object of a starch liquefaction process is to convert a concentrated suspension of starch polymer granules into a solution of soluble shorter chain length dextrins of low viscosity.","The amylases of the invention can be used in automatic dish wash (ADW) products and laundry detergent. In ADW products, the amylase will function at pH 10-11 and at 45-60\u00b0 C. in the presence of calcium chelators and oxidative conditions. For laundry, activity at pH 9-10 and 40\u00b0 C. in the appropriate detergent matrix will be required. Amylases are also useful in textile desizing, brewing processes, starch modification in the paper and pulp industry and other processes described in the art.","Amylases can be used commercially in the initial stages (liquefaction) of starch processing; in wet corn milling; in alcohol production; as cleaning agents in detergent matrices; in the textile industry for starch desizing; in baking applications; in the beverage industry; in oilfields in drilling processes; in inking of recycled paper and in animal feed. Amylases are also useful in textile desizing, brewing processes, starch modification in the paper and pulp industry and other processes.","Carotenoid Pathway Enzymes","The invention provides novel enzymes, and the polynucleotides encoding them, involved in carotenoid (such as lycopenes and luteins), astaxanthin and\/or isoprenoid synthesis. The invention also provides novel genes in the carotenoid, astaxanthin and isoprenoid biosynthetic pathways comprising at least one enzyme of the invention. For example, alternative aspects, the invention provides one or more nucleic acid coding sequences (CDSs, or ORFs) encoding all, or at least one, enzyme(s) involved in a desired biosynthetic pathway for carotenoids, astaxanthins and\/or isoprenoids. The nucleic acid coding sequence(s) can be expressed through an expression plasmid, vector, engineered virus or any episomal expression system, or, can be integrated into the genome of the host cell. In one aspect, the enzyme(s) involved in the biosynthetic pathway system comprise a novel combination of enzymes. In another aspect, the enzyme(s) involved in the biosynthetic pathway system comprise at least one novel enzyme of the invention\u2014where nucleic acids used in the system encode a novel enzyme of the invention.","Carotenoids are natural pigments which have antioxidant and anti-carcinogenic activity. They are free radical scavengers, and as such, strong antioxidants. Carotenoids have a conjugated backbone structure and are very rigid molecules, having a backbone consisting of 9 to 11 alternating single\/double bonds and have very similar electro-optical properties as polyacetylene. Astaxanthins are abundant naturally occurring carotenoids. They contain an internal unit similar to beta-carotene but have two terminal carbonyl and hydroxyl functionalities. These compounds are useful for food and feed supplements, colorants, neutraceuticals, cosmetic and pharmaceutical needs. Isoprenoids are compounds biosynthesized from or containing isoprene (unsaturated branched chain five-carbon hydrocarbon) units, including terpenes, carotenoids, fat soluble vitamins, ubiquinone, rubber, and some steroids. Biosynthetic pathways for carotenoids, astaxanthins and isoprenoids are known; most of these published pathways are derived from one organism or a combination of genes from a few species.","Catalases","In one aspect, the invention provides catalases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having a catalase activity, including thermostable and thermotolerant catalase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","In processes where hydrogen peroxide is a by-product, catalases of the invention can be used to destroy or detect hydrogen peroxide, e.g., in production of glyoxylic acid and in glucose sensors. Also, in processes where hydrogen peroxide is used as a bleaching or antibacterial agent, catalases of the invention can be used to destroy residual hydrogen peroxide, e.g. in contact lens cleaning, in bleaching steps in pulp and paper production, and in the pasteurization of dairy products. Further, such catalases of the invention can be used as catalysts for oxidation reactions, e.g. epoxidation and hydroxylation.","Dehalogenases","In one aspect, the invention provides dehalogenases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having a dehalogenase activity, including thermostable and thermotolerant dehalogenase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","Environmental pollutants consist of a large quantity and variety of chemicals; many of these are toxic, environmental hazards that were designated in 1979 as priority pollutants by the U.S. Environmental Protection Agency. Microbial and enzymatic biodegradation is one method for the elimination of these pollutants. Accordingly, methods have been designed to treat commercial wastes and to bioremediate polluted environments via microbial and related enzymatic processes. Unfortunately, many chemical pollutants are either resistant to microbial degradation or are toxic to potential microbial-degraders when present in high concentrations and certain combinations.","Dehalogenases, e.g. haloalkane dehalogenases, of the invention can cleave carbon-halogen bonds in haloalkanes and halocarboxylic acids by hydrolysis, thus converting them to their corresponding alcohols. This reaction can be used for detoxification involving haloalkanes, such as ethylchloride, methylchloride, and 1,2-dichloroethane (e.g., detoxification of toxic composition, e.g., pesticides, poisons, chemical warfare agents and the like comprising haloalkanes).","The present invention provides a number of dehalogenase enzymes useful in bioremediation having improved enzymatic characteristics. The polynucleotides and polynucleotide products of the invention are useful in, for example, groundwater treatment involving transformed host cells containing a polynucleotide or polypeptide of the invention (e.g., the bacteria Xanthobacter autotrophicus) and the haloalkane 1,2-dichlorethane as well as removal of polychlorinated biphenyls (PCB's) from soil sediment.","The haloalkane dehalogenase of the invention are useful in carbon-halide reduction efforts. The enzymes of the invention initiate the degradation of haloalkanes. Alternatively, host cells containing a dehalogenase polynucleotide or polypeptide of the invention can feed on the haloalkanes and produce the detoxifying enzyme.","Endoglucanases","In one aspect, the invention provides endoglucanases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having an endoglucanase activity, including thermostable and thermotolerant endoglucanase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","In one aspect, the enzymes of the invention have a glucanase, e.g., an endoglucanase, activity, e.g., catalyzing hydrolysis of internal endo-\u03b2-1,4- and\/or \u03b2-1,3-glucanase linkages. In one aspect, the endoglucanase activity (e.g., endo-1,4-beta-D-glucan 4-glucano hydrolase activity) comprises hydrolysis of 1,4- and\/or \u03b2-1,3-beta-D-glycosidic linkages in cellulose, cellulose derivatives (e.g., carboxy methyl cellulose and hydroxy ethyl cellulose) lichenin, beta-1,4 bonds in mixed beta-1,3 glucans, such as cereal beta-D-glucans or xyloglucans and other plant material containing cellulosic parts.","Endoglucanases of the invention (e.g., endo-beta-1,4-glucanases, EC 3.2.1.4; endo-beta-1,3(1)-glucanases, EC 3.2.1.6; endo-beta-1,3-glucanases, EC 3.2.1.39) can hydrolyze internal \u03b1-1,4- and\/or \u03b2-1,3-glucosidic linkages in cellulose and glucan to produce smaller molecular weight glucose and glucose oligomers. Glucans are polysaccharides formed from 1,4-\u03b2- and\/or 1,3-glycoside-linked D-glucopyranose. Endoglucanases of the invention can be used in the food industry, for baking and fruit and vegetable processing, breakdown of agricultural waste, in the manufacture of animal feed, in pulp and paper production, textile manufacture and household and industrial cleaning agents. Endoglucanases are produced by fungi and bacteria.","Beta-glucans are major non-starch polysaccharides of cereals. The glucan content can vary significantly depending on variety and growth conditions. The physicochemical properties of this polysaccharide are such that it gives rise to viscous solutions or even gels under oxidative conditions. In addition glucans have high water-binding capacity. All of these characteristics present problems for several industries including brewing, baking, animal nutrition. In brewing applications, the presence of glucan results in wort filterability and haze formation issues. In baking applications (especially for cookies and crackers), glucans can create sticky doughs that are difficult to machine and reduce biscuit size. In addition, this carbohydrate is implicated in rapid rehydration of the baked product resulting in loss of crispiness and reduced shelf-life. For monogastric animal feed applications with cereal diets, beta-glucan is a contributing factor to viscosity of gut contents and thereby adversely affects the digestibility of the feed and animal growth rate. For ruminant animals, these beta-glucans represent substantial components of fiber intake and more complete digestion of glucans would facilitate higher feed conversion efficiencies. It is desirable for animal feed endoglucanases to be active in the animal stomach.","Endoglucanases of the invention can be used in the digestion of cellulose, a beta-1,4-linked glucan found in all plant material. Cellulose is the most abundant polysaccharide in nature. Enzymes of the invention that digest cellulose have utility in the pulp and paper industry, in textile manufacture and in household and industrial cleaning agents.","Epoxide Hydrolases","In one aspect, the invention provides epoxide hydrolases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having an epoxide hydrolase activity, including thermostable and thermotolerant epoxide hydrolase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides. The polypeptides of the invention can be used as epoxide hydrolases to catalyze the hydrolysis of epoxides and arene oxides to their corresponding diols.","Epoxide hydrolases catalyze the hydrolysis of epoxides and arene oxides to their corresponding diols. Epoxide hydrolases from microbial sources are highly versatile biocatalysts for the asymmetric hydrolysis of epoxides on a preparative scale. Besides kinetic resolution, which furnishes the corresponding vicinal diol and remaining non-hydrolyzed epoxide in nonracemic form, enantioconvergent processes are possible. These are highly attractive as they lead to the formation of a single enantiomeric diol from a racemic oxirane.","Microsomal epoxide hydrolases are biotransformation enzymes that catalyze the conversion of a broad array of xenobiotic epoxide substrates to more polar diol metabolites, see, e.g., Omiecinski (2000) Toxicol. Lett. 112-113:365-370. Microsomal epoxide hydrolases catalyze the addition of water to epoxides in a two-step reaction involving initial attack of an active site carboxylate on the oxirane to give an ester intermediate followed by hydrolysis of the ester. Soluble epoxide hydrolase play a role in the biosynthesis of inflammation mediators.","Epoxide hydrolases of the invention can be used in the detoxification of epoxides or in the biosynthesis of hormones. Additionally, epoxide hydrolases of the invention can efficiently process several substrates, leading to enantiomerically enriched-epoxides (the unreacted enantiomer) and\/or to the corresponding vicinal diols.","Esterases","In one aspect, the invention provides esterases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having an esterase activity, including thermostable and thermotolerant esterase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","Many esterases are known and have been discovered in a broad variety of organisms, including bacteria, yeast and higher animals and plants. A principal example of esterases are the lipases, which are used in the hydrolysis of lipids, acidolysis (replacement of an esterified fatty acid with a free fatty acid) reactions, transesterification (exchange of fatty acids between triglycerides) reactions, and in ester synthesis. The major industrial applications for lipases include: the detergent industry, where they are employed to decompose fatty materials in laundry stains into easily removable hydrophilic substances; the food and beverage industry where they are used in the manufacture of cheese, the ripening and flavoring of cheese, as antistaling agents for bakery products, and in the production of margarine and other spreads with natural butter flavors; in waste systems; and in the pharmaceutical industry where they are used as digestive aids.","Alternatively, esterases of the invention can be used in detergent compositions. In one aspect, the esterase can be a nonsurface-active esterase. In another aspect, the esterase can be a surface-active esterase. The esterase can be formulated in a non-aqueous liquid composition, a cast solid, a granular form, a particulate form, a compressed tablet, a gel form, a paste or a slurry form.","In another aspect, the invention provides fabrics or clothing comprising an esterase of the invention. In another aspect, esterases of the invention are used to treat a lipid-containing fabric.","In another aspect, the invention provides foods and drinks comprising an esterase of the invention. The invention also provides cheeses comprising an esterase of the invention. Additionally, the invention provides methods for the manufacture of cheese comprising the following steps: (a) providing a polypeptide having an esterase activity, wherein the polypeptide comprises a polypeptide of the invention, or, a polypeptide encoded by a nucleic acid of the invention; (b) providing a cheese precursor; and (c) contacting the polypeptide of step (a) with the precursor of step (b) under condition wherein the esterase can catalyze cheese manufacturing processes. In one aspect, the method can comprise the process of ripening and flavoring of cheese.","In another aspect, the invention provides margarines and spreads comprising an enzyme of the invention. The invention provides methods for production of margarine or other spreads with natural butter flavors comprising the following steps: (a) providing a polypeptide having an esterase activity, wherein the polypeptide comprises a polypeptide of the invention, or, a polypeptide encoded by a nucleic acid of the invention; (b) providing a margarine or a spread precursor; and (c) contacting the polypeptide of step (a) with the precursor of step (b) under condition wherein the esterase can catalyze processes involved in margarine or spread production.","The invention provides methods for treating solid or liquid waste products comprising the following steps: (a) providing a polypeptide having an esterase activity, wherein the polypeptide comprises a polypeptide of the invention, or, a polypeptide encoded by a nucleic acid of the invention; (b) providing a solid or a liquid waste; and (c) contacting the polypeptide of step (a) and the waste of step (b) under conditions wherein the polypeptide can treat the waste. The invention provides solid or liquid waste products comprising a polypeptide of the invention.","The invention provides methods for aiding digestion in a mammal comprising (a) providing a polypeptide having an esterase activity, wherein the polypeptide comprises a polypeptide of the invention, or, a polypeptide encoded by a nucleic acid of the invention; (b) providing a composition comprising a substrate for the polypeptide of step (a); (c) feeding or administering to the mammal the polypeptide of step (a) with a feed or food comprising a substrate for the polypeptide of step (a), thereby helping digestion in the mammal. In one aspect, the mammal is a human.","The invention provides pharmaceutical compositions comprising a polypeptide and\/or a nucleic acid of the invention, e.g., a pharmaceutical composition for use as a digestive aid in a mammal comprising a polypeptide having an esterase activity, wherein the polypeptide comprises a polypeptide of the invention, or, a polypeptide encoded by a nucleic acid of the invention. In one aspect, the mammal comprises a human. The enzymes of the invention are used in the manufacture of medicaments.","The invention provides bakery products comprising a polypeptide of the invention. The invention provides antistaling agents for bakery products comprising a polypeptide having an esterase activity, wherein the polypeptide comprises a polypeptide of the invention, or, a polypeptide encoded by a nucleic acid of the invention.","The invention provides methods for hydrolyzing, breaking up or disrupting a ester-comprising composition comprising the following steps: (a) providing a polypeptide of the invention having an esterase activity, or a polypeptide encoded by a nucleic acid of the invention; (b) providing a composition comprising a protein; and (c) contacting the polypeptide of step (a) with the composition of step (b) under conditions wherein the esterase hydrolyzes, breaks up or disrupts the ester-comprising composition.","Alternatively, the invention provides methods for liquefying or removing ester-comprising compositions comprising the following steps: (a) providing a polypeptide of the invention having an esterase activity, or a polypeptide encoded by a nucleic acid of the invention; (b) providing a composition comprising a protein; and (c) contacting the polypeptide of step (a) with the composition of step (b) under conditions wherein esterase removes or liquefies the ester-comprising compositions.","Hydrolases","In one aspect, the invention provides hydrolases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having a hydrolase activity, e.g., an esterase, acylase, lipase, phospholipase or protease activity, including thermostable and thermotolerant hydrolase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides. The hydrolase activities of the polypeptides and peptides of the invention include esterase activity, lipase activity (hydrolysis of lipids), acidolysis reactions (to replace an esterified fatty acid with a free fatty acid), transesterification reactions (exchange of fatty acids between triglycerides), ester synthesis, ester interchange reactions, phospholipase activity (e.g., phospholipase A, B, C and D activity, patatin activity, lipid acyl hydrolase (LAH) activity) and protease activity (hydrolysis of peptide bonds). The polypeptides of the invention can be used in a variety of pharmaceutical, agricultural and industrial contexts, including the manufacture of cosmetics and nutraceuticals.","In one aspect, the polypeptides of the invention are used in the biocatalytic synthesis of structured lipids (lipids that contain a defined set of fatty acids distributed in a defined manner on the glycerol backbone), including cocoa butter alternatives (CBA), lipids containing poly-unsaturated fatty acids (PUFAs), diacylglycerides, e.g., 1,3-diacyl glycerides (DAGs), monoglycerides, e.g., 2-monoglycerides (MAGs) and triacylglycerides (TAGs). In one aspect, the polypeptides of the invention are used to modify oils, such as fish, animal and vegetable oils, and lipids, such as poly-unsaturated fatty acids. The hydrolases of the invention having lipase activity can modify oils by hydrolysis, alcoholysis, esterification, transesterification and\/or interesterification. The methods of the invention can use lipases with defined regio-specificity or defined chemoselectivity in biocatalytic synthetic reactions. In another aspect, the polypeptides of the invention are used to synthesize enantiomerically pure chiral products.","Additionally, the polypeptides of the invention can be used in food processing, brewing, bath additives, alcohol production, peptide synthesis, enantioselectivity, hide preparation in the leather industry, waste management and animal degradation, silver recovery in the photographic industry, medical treatment, silk degumming, biofilm degradation, biomass conversion to ethanol, biodefense, antimicrobial agents and disinfectants, personal care and cosmetics, biotech reagents, in increasing starch yield from corn wet milling and pharmaceuticals such as digestive aids and anti-inflammatory (anti-phlogistic) agents.","The major industrial applications for hydrolases, e.g., esterases, lipases, phospholipases and proteases, include the detergent industry, where they are employed to decompose fatty materials in laundry stains into easily removable hydrophilic substances; the food and beverage industry where they are used in the manufacture of cheese, the ripening and flavoring of cheese, as antistaling agents for bakery products, and in the production of margarine and other spreads with natural butter flavors; in waste systems; and in the pharmaceutical industry where they are used as digestive aids.","Oils and fats an important renewable raw material for the chemical industry. They are available in large quantities from the processing of oilseeds from plants like rice bran oil, rapeseed (canola), sunflower, olive, palm or soy. Other sources of valuable oils and fats include fish, restaurant waste, and rendered animal fats. These fats and oils are a mixture of triglycerides or lipids, i.e. fatty acids (FAs) esterified on a glycerol scaffold. Each oil or fat contains a wide variety of different lipid structures, defined by the FA content and their regiochemical distribution on the glycerol backbone. These properties of the individual lipids determine the physical properties of the pure triglyceride. Hence, the triglyceride content of a fat or oil to a large extent determines the physical, chemical and biological properties of the oil. The value of lipids increases greatly as a function of their purity. High purity can be achieved by fractional chromatography or distillation, separating the desired triglyceride from the mixed background of the fat or oil source. However, this is costly and yields are often limited by the low levels at which the triglyceride occurs naturally. In addition, the purity of the product is often compromised by the presence of many structurally and physically or chemically similar triglycerides in the oil.","An alternative to purifying triglycerides or other lipids from a natural source is to synthesize the lipids. The products of such processes are called structured lipids because they contain a defined set of fatty acids distributed in a defined manner on the glycerol backbone. The value of lipids also increases greatly by controlling the fatty acid content and distribution within the lipid. Lipases can be used to affect such control.","Phospholipases are enzymes that hydrolyze the ester bonds of phospholipids. Corresponding to their importance in the metabolism of phospholipids, these enzymes are widespread among prokaryotes and eukaryotes. The phospholipases affect the metabolism, construction and reorganization of biological membranes and are involved in signal cascades. Several types of phospholipases are known which differ in their specificity according to the position of the bond attacked in the phospholipid molecule. Phospholipase A1 (PLA1) removes the 1-position fatty acid to produce free fatty acid and 1-lyso-2-acylphospholipid. Phospholipase A2 (PLA2) removes the 2-position fatty acid to produce free fatty acid and 1-acyl-2-lysophospholipid. PLA1 and PLA2 enzymes can be intra- or extra-cellular, membrane-bound or soluble. Intracellular PLA2 is found in almost every mammalian cell. Phospholipase C (PLC) removes the phosphate moiety to produce 1,2 diacylglycerol and phospho base. Phospholipase D (PLD) produces 1,2-diacylglycerophosphate and base group. PLC and PLD are important in cell function and signaling. Patatins are another type of phospholipase thought to work as a PLA.","In general, enzymes, including hydrolases such as esterases, lipases and proteases, are active over a narrow range of environmental conditions (temperature, pH, etc.), and many are highly specific for particular substrates. The narrow range of activity for a given enzyme limits its applicability and creates a need for a selection of enzymes that (a) have similar activities but are active under different conditions or (b) have different substrates. For instance, an enzyme capable of catalyzing a reaction at 50\u00b0 C. may be so inefficient at 35\u00b0 C., that its use at the lower temperature will not be feasible. For this reason, laundry detergents generally contain a selection of proteolytic enzymes (e.g., polypeptides of the invention), allowing the detergent to be used over a broad range of wash temperature and pH. In view of the specificity of enzymes and the growing use of hydrolases in industry, research, and medicine, there is an ongoing need in the art for new enzymes and new enzyme inhibitors.","Glucosidases","In one aspect, the invention provides glucosidases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having a glucosidase activity, including thermostable and thermotolerant glucosidase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","Alpha-glucosidases of the invention can catalyze the hydrolysis of starches into sugars. Alpha-glucosidases can hydrolyze terminal non-reducing 1, 4 or 1,6 linked \u03b1-D-glucose residues in starch, with release of \u03b1-D-glucose.","Alpha-glucosidases of the invention can be used commercially in the stages liquefaction and saccharification of starch processing; in wet corn milling; in alcohol production; as cleaning agents in detergent matrices; in the textile industry for starch desizing; in baking applications; in the beverage industry; in oilfields in drilling processes; in inking of recycled paper and in animal feed. Alpha-glucosidases of the invention are also useful in textile desizing, brewing processes, starch modification in the paper and pulp industry and other processes.","Glycosidases","In one aspect, the invention provides glycosidases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having a glycosidase activity, including thermostable and thermotolerant glycosidase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides. Glycosidase enzymes of the invention can have more specific activity as glucosidases, \u03b1-galactosidases, \u03b2-galactosidases, \u03b2-mannosidases, \u03b2-mannanases, endoglucanases, and pullulanases.","\u03b1-galactosidases of the invention can catalyze the hydrolysis of galactose groups on a polysaccharide backbone or hydrolyze the cleavage of di- or oligosaccharides comprising galactose. \u03b2-mannanases of the invention can catalyze the hydrolysis of mannose groups internally on a polysaccharide backbone or hydrolyze the cleavage of di- or oligosaccharides comprising mannose groups. \u03b2-mannosidases of the invention can hydrolyze non-reducing, terminal mannose residues on a mannose-containing polysaccharide and the cleavage of di- or oligosaccaharides comprising mannose groups.","Guar gum is a branched galactomannan polysaccharide composed of \u03b2-1,4 linked mannose backbone with a-1,6 linked galactose sidechains. The enzymes required for the degradation of guar are \u03b2-mannanase, \u03b2-mannosidase and \u03b1-galactosidase. \u03b2-mannanase hydrolyses the mannose backbone internally and \u03b2-mannosidase hydrolyses non-reducing, terminal mannose residues. \u03b1-galactosidase hydrolyses \u03b1-linked galactose groups.","Galactomannan polysaccharides and the enzymes of the invention that degrade them have a variety of applications. Guar is commonly used as a thickening agent in food and is utilized in hydraulic fracturing in oil and gas recovery. Consequently, galactomannanases are industrially relevant for the degradation and modification of guar. Furthermore, a need exists for thermostable galactomannases that are active in extreme conditions associated with oil drilling and well stimulation.","There are other applications for these enzymes in various industries, such as in the beet sugar industry. 20-30% of the domestic U.S. sucrose consumption is sucrose from sugar beets. Raw beet sugar can contain a small amount of raffinose when the sugar beets are stored before processing and rotting begins to set in. Raffinose inhibits the crystallization of sucrose and also constitutes a hidden quantity of sucrose. Thus, there is merit to eliminating raffinose from raw beet sugar. \u03b1-Galactosidase has also been used as a digestive aid to break down raffinose, stachyose, and verbascose in such foods as beans and other gassy foods.","\u03b2-Galactosidases of the invention can be used for the production of lactose-free dietary milk products. Additionally, \u03b2-galactosidases of the invention can be used for the enzymatic synthesis of oligosaccharides via transglycosylation reactions.","Pullulanase is well known as a debranching enzyme of pullulan and starch. The enzyme of the invention can hydrolyze \u03b1-1,6-glucosidic linkages on these polymers. Starch degradation for the production or sweeteners (glucose or maltose) is a very important industrial application of this enzyme. The degradation of starch is developed in two stages. The first stage involves the liquefaction of the substrate with \u03b1-amylase, and the second stage, or saccharification stage, is performed by \u03b2-amylase with pullalanase added as a debranching enzyme, to obtain better yields.","Endoglucanases of the invention can be used in a variety of industrial applications. For instance, the endoglucanases of the invention can hydrolyze the internal \u03b2-1,4-glycosidic bonds in cellulose, which may be used for the conversion of plant biomass into fuels and chemicals. Endoglucanases of the invention also have applications in detergent formulations, the textile industry, in animal feed, in waste treatment, oil drilling and well stimulation, and in the fruit juice and brewing industry for the clarification and extraction of juices.","Inteins","In one aspect, the invention provides inteins, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In another aspect, the invention provides a chimeric protein comprising at least three domains, wherein the first domain comprises at least one enzyme domain or a binding protein domain, the second domain comprises at least one intein domain and a third domain comprising a detectable moiety domain, at least one intein domain is positioned between at least one enzyme or binding protein and at least one detectable moiety domain, and the intein domain has at least one cleavage or splicing activity.","In one aspect, the detectable moiety domain comprises a detectable peptide or polypeptide. The detectable peptide or a polypeptide can be a fluorescent peptide or polypeptide. The detectable peptide or a polypeptide can be a bioluminescent or a chemiluminescent peptide or polypeptide. In one aspect, the bioluminescent or chemiluminescent polypeptide comprises a green fluorescent protein (GFP), an aequorin, an obelin, a mnemiopsin or a berovin. In one aspect, the detectable moiety domain comprises an enzyme that generates a detectable signal. The enzyme that generates a detectable signal can comprise an alpha-galactosidase, an antibiotic (e.g., chloramphenicol acetyltransferase) or a kinase. The detectable moiety domain can comprise a radioactive isotope.","In one aspect, the chimeric protein is a recombinant fusion protein. In one aspect, the intein domain splicing activity results in cleavage of the enzyme domain from the intein domain and detectable domain. The intein domain splicing activity can result in cleavage of the enzyme domain from the intein domain and detectable domain and cleavage of the detectable domain from the intein domain. In one aspect, the intein domain splicing activity results in cleavage of the detectable domain from the intein domain. In one aspect, the intein domain has only splicing activity. The intein domain can have only cleaving activity.","In one aspect, at least one domain is separated from another domain by a linker. The linker can be a flexible linker. The intein domain can be separated from the detectable moiety domain and the enzyme domain by a linker.","Isomerases","In one aspect, the invention provides isomerases, e.g. xylose isomerases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having an isomerase activity, e.g. xylose isomerase activity, including thermostable and thermotolerant isomerase activity, e.g. xylose isomerase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","In one aspect, the invention provides xylose isomerase enzymes, polynucleotides encoding the enzymes, methods of making and using these polynucleotides and polypeptides. The polypeptides of the invention can be used in a variety of agricultural and industrial contexts. For example, the polypeptides of the invention can be used for converting glucose to fructose or for manufacturing high content fructose syrups in large quantities. Other examples include use of the polypeptides of the invention in confectionary, brewing, alcohol and soft drinks production, and in diabetic foods and sweeteners.","Laccases","In one aspect, the invention provides laccases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having a laccase activity, including thermostable and thermotolerant laccase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","In one aspect, the invention provides methods of depolymerizing lignin, e.g., in a pulp or paper manufacturing process, using a polypeptide of the invention. In another aspect, the invention provides methods for oxidizing products that can be mediators of laccase-catalyzed oxidation reactions, e.g., 2,2-azinobis-(3-ethylbenzthiazoline-6-sulfonate) (ABTS), 1-hydroxybenzotriazole (HBT), 2,2,6,6-tetramethylpiperidin-1-yloxy (TEMPO), dimethoxyphenol, dihydroxyfumaric acid (DHF) and the like.","Laccases are a subclass of the multicopper oxidase super family of enzymes, which includes ascorbate oxidases and the mammalian protein, ceruloplasmin. Laccases are one of the oldest known enzymes and were first implicated in the oxidation of urushiol and laccol. In one aspect, reactions catalyzed by laccases of the invention comprises the oxidation of phenolic substrates. The major target application has been in the delignification of wood fibers during the preparation of pulp.","Lipases","In one aspect, the invention provides lipases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having a lipase activity, including thermostable and thermotolerant lipase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","In one aspect, the lipases of the invention can be used in a variety of pharmaceutical, agricultural and industrial contexts, including the manufacture of cosmetics and nutraceuticals. In one aspect, the lipases of the invention are used in the biocatalytic synthesis of structured lipids (lipids that contain a defined set of fatty acids distributed in a defined manner on the glycerol backbone), including cocoa butter alternatives (CBA), lipids containing poly-unsaturated fatty acids (PUFAs), diacylglycerides, e.g., 1,3-diacyl glycerides (DAGs), monoglycerides, e.g., 2-monoglycerides (MAGs) and triacylglycerides (TAGs). In one aspect, the polypeptides of the invention are used to modify oils, such as fish, animal and vegetable oils, and lipids, such as poly-unsaturated fatty acids. The lipases of the invention can modify oils by hydrolysis, alcoholysis, esterification, transesterification and\/or interesterification. The methods of the invention use lipases with defined regio-specificity or defined chemoselectivity in biocatalytic synthetic reactions. In another aspect, the polypeptides of the invention are used to synthesize enantiomerically pure chiral products.","The invention provides lipase enzymes, polynucleotides encoding the enzymes, methods of making and using these polynucleotides and polypeptides. The polypeptides of the invention can be used in a variety of pharmaceutical, agricultural and industrial contexts, including the manufacture of cosmetics and nutraceuticals. In one aspect, the polypeptides of the invention are used in the biocatalytic synthesis of structured lipids (lipids that contain a defined set of fatty acids distributed in a defined manner on the glycerol backbone), including cocoa butter alternatives, poly-unsaturated fatty acids (PUFAs), 1,3-diacyl glycerides (DAGs), 2-monoglycerides (MAGs) and triacylglycerides (TAGs), such as 1,3-dipalmitoyl-2-oleoylglycerol (POP), 1,3-distearoyl-2-oleoylglycerol (SOS), 1-palmitoyl-2-oleoyl-3-stearoylglycerol (POS) or 1-oleoyl-2,3-dimyristoylglycerol (OMM), long chain polyunsaturated fatty acids such as arachidonic acid, docosahexaenoic acid (DHA) and eicosapentaenoic acid (EPA).","In one aspect, the invention provides synthesis (using lipases of the invention) of a triglyceride mixture composed of POS (Palmitic-Oleic-Stearic), POP (Palmitic-Oleic-Palmitic) and SOS (Stearic-Oleic-Stearic) from glycerol. This synthesis uses free fatty acids versus fatty acid esters. In one aspect, this reaction can be performed in one pot with sequential addition of fatty acids using crude glycerol and free fatty acids and fatty acid esters. In one aspect, stearate and palmitate are mixed together to generate mixtures of DAGs. In one aspect, the diacylglycerides are subsequently acylated with oleate to give components of cocoa butter equivalents. In alternative aspects, the proportions of POS, POP and SOS can be varied according to: stearate to palmitate ratio; selectivity of enzyme for palmitate versus stearate; or enzyme enantioselectivity (could alter levels of POS\/SOP). One-pot synthesis of cocoa butter equivalents or other cocoa butter alternatives is possible using this aspect of the invention.","In one aspect, lipases that exhibit regioselectivity and\/or chemoselectivity are used in the structure synthesis of lipids or in the processing of lipids. Thus, the methods of the invention use lipases with defined regio-specificity or defined chemoselectivity (e.g., a fatty acid specificity) in a biocatalytic synthetic reaction. For example, the methods of the invention can use lipases with SN1, SN2 and\/or SN3 regio-specificity, or combinations thereof. In one aspect, the methods of the invention use lipases that exhibit regioselectivity for the 2-position of a triacylglyceride (TAG). This SN2 regioselectivity can be used in the synthesis of a variety of structured lipids, e.g., triacylglycerides (TAGs), including 1,3-DAGs and components of cocoa butter.","The methods and compositions (lipases) of the invention can be used in the biocatalytic synthesis of structured lipids, and the production of nutraceuticals (e.g., polyunsaturated fatty acids and oils), various foods and food additives (e.g., emulsifiers, fat replacers, margarines and spreads), cosmetics (e.g., emulsifiers, creams), pharmaceuticals and drug delivery agents (e.g., liposomes, tablets, formulations), and animal feed additives (e.g., polyunsaturated fatty acids, such as linoleic acids) comprising lipids made by the structured synthesis methods of the invention or processed by the methods of the invention","In one aspect, lipases of the invention can act on fluorogenic fatty acid (FA) esters, e.g., umbelliferyl FA esters. In one aspect, profiles of FA specificities of lipases made or modified by the methods of the invention can be obtained by measuring their relative activities on a series of umbelliferyl FA esters, such as palmitate, stearate, oleate, laurate, PUFA, butyrate.","The methods and compositions (lipases) of the invention can be used to synthesize enantiomerically pure chiral products. In one aspect, the methods and compositions (lipases) of the invention can be used to prepare a D-amino acid and corresponding esters from a racemic mix. For example, D-aspartic acid can be prepared from racemic aspartic acid. In one aspect, optically active D-homophenylalanine and\/or its esters are prepared. The enantioselectively synthesized D-homophenylalanine can be starting material for many drugs, such as Enalapril, Lisinopril, and Quinapril, used in the treatment of hypertension and congestive heart failure. The D-aspartic acid and its derivatives made by the methods and compositions of the invention can be used in pharmaceuticals, e.g., for the inhibition of arginiosuccinate synthetase to prevent or treat sepsis or cytokine-induced systemic hypotension or as immunosuppressive agents. The D-aspartic acid and its derivatives made by the methods and compositions of the invention can be used as taste modifying compositions for foods, e.g., as sweeteners (e.g., ALITAME\u2122). For example, the methods and compositions (lipases) of the invention can be used to synthesize an optical isomer S(+) of 2-(6-methoxy-2-naphthyl)propionic acid from a racemic (R,S) ester of 2-(6-methoxy-2-naphthyl)propionic acid.","In one aspect, the methods and compositions (lipases) of the invention can be used to for stereoselectively hydrolyzing racemic mixtures of esters of 2-substituted acids, e.g., 2-aryloxy substituted acids, such as R-2-(4-hydroxyphenoxy)propionic acid, 2-arylpropionic acid, ketoprofen to synthesize enantiomerically pure chiral products.","The methods and compositions (lipases) of the invention can be used to hydrolyze oils, such as fish, animal and vegetable oils, and lipids, such as poly-unsaturated fatty acids. In one aspect, the polypeptides of the invention are used process fatty acids (such as poly-unsaturated fatty acids), e.g., fish oil fatty acids, for use in or as a feed additive. Addition of poly-unsaturated fatty acids PUFAs to feed for dairy cattle has been demonstrated to result in improved fertility and milk yields. Fish oil contains a high level of PUFAs and therefore is a potentially inexpensive source for PUFAs as a starting material for the methods of the invention. The biocatalytic methods of the invention can process fish oil under mild conditions, thus avoiding harsh conditions utilized in some processes. Harsh conditions may promote unwanted isomerization, polymerization and oxidation of the PUFAs. In one aspect, the methods of the invention comprise lipase-catalyzed total hydrolysis of fish-oil or selective hydrolysis of PUFAs from fish oil to provide a mild alternative that would leave the high-value PUFAs intact. In one aspect, the methods further comprise hydrolysis of lipids by chemical or physical splitting of the fat.","In one aspect, the lipases and methods of the invention are used for the total hydrolysis of fish oil. Lipases can be screened for their ability to catalyze the total hydrolysis of fish oil under different conditions using. In alternative aspects, a single or multiple lipases are used to catalyze the total splitting of the fish oil. Several lipases of the invention may need to be used, owing to the presence of the PUFAs. In one aspect, a PUFA-specific lipase of the invention is combined with a general lipase to achieve the desired effect.","The methods and compositions (lipases) of the invention can be used to catalyze the partial or total hydrolysis of other oils, e.g. olive oils, that do not contain PUFAs.","The methods and compositions (lipases) of the invention can be used to catalyze the hydrolysis of PUFA glycerol esters. These methods can be used to make feed additives. In one aspect, lipases of the invention catalyze the release of PUFAs from simple esters and fish oil. Standard assays and analytical methods can be utilized.","The methods and compositions (lipases) of the invention can be used to selectively hydrolyze saturated esters over unsaturated esters into acids or alcohols. The methods and compositions (lipases) of the invention can be used to treat latexes for a variety of purposes, e.g., to treat latexes used in hair fixative compositions to remove unpleasant odors. The methods and compositions (lipases) of the invention can be used in the treatment of a lipase deficiency in an animal, e.g., a mammal, such as a human. The methods and compositions (lipases) of the invention can be used to prepare lubricants, such as hydraulic oils. The methods and compositions (lipases) of the invention can be used in making and using detergents. The methods and compositions (lipases) of the invention can be used in processes for the chemical finishing of fabrics, fibers or yarns. In one aspect, the methods and compositions (lipases) of the invention can be used for obtaining flame retardancy in a fabric using, e.g., a halogen-substituted carboxylic acid or an ester thereof, i.e. a fluorinated, chlorinated or bromated carboxylic acid or an ester thereof.","Monooxygenases","In one aspect, the invention provides monooxygenases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having a monooxygenase activity, including thermostable and thermotolerant monooxygenase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","In one aspect, the monooxygenases of the invention have commercial utility as biocatalysts for use in the synthesis of aromatic and aliphatic esters and their derivatives, such as acids and alcohols. In one aspect, the monooxygenases of the invention are used in the catalysis of sulfoxidation reactions. In one aspect, the invention provides Baeyer-Villiger monooxygenases, polynucleotides encoding the Baeyer-Villiger monooxygenases, and methods of using these Baeyer-Villiger monooxygenases and polynucleotides. In one aspect, the invention provides methods of producing chiral synthetic intermediates using Baeyer-Villiger monooxygenases.","In one aspect, the monooxygenase activity comprises catalysis of sulfoxidation reactions. The monooxygenase activity can comprise an asymmetric sulfoxidation reaction. The monooxygenase activity can be enantiospecific. In one aspect, it can generate a substantially chiral product.","In one aspect, the monooxygenase activity comprises generation of an ester or a lactone having at least one of the following structures:",{"@attributes":{"id":"p-0175","num":"0174"},"chemistry":{"@attributes":{"id":"CHEM-US-00001","num":"00001"},"img":{"@attributes":{"id":"EMI-C00001","he":"16.68mm","wi":"42.93mm","file":"US08962800-20150224-C00001.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"wherein: R, R, Rand Rare each independently selected from \u2014H, substituted or unsubstituted alkyl, alkenyl, alkynyl, aryl, heteroaryl, cycloalkyl, and heterocyclic; wherein the substituted groups are substituted with one or more of lower alkyl, hydroxy, alkoxy, mercapto, cycloalkyl, heterocyclic, aryl, heteroaryl, aryloxy, and halogen, or two or more of R, R, Rand Rmay together form cyclic moieties, and, R\u2032 is selected from substituted or unsubstituted alkylene, alkenylene, alkynylene, arylene, heteroarylene, cycloalkylene, and heterocyclic; wherein the substitutions are substituted with one or more of lower alkyl, hydroxy, alkoxy, mercapto, cycloalkyl, heterocyclic, aryl, heteroaryl, aryloxy, and halogen.","In one aspect, the monooxygenase activity comprises oxidation of a cycloalkanone to produce a chiral lactone. The cycloalkanone can comprise a cyclobutanone, a cyclopentanone, a cyclohexanone, a 2-methylcyclopentanone, a 2-methylcyclohexanone, a cyclohex-2-ene-1-one, a 2-(cyclohex-1-enyl)cyclohexanone, a 1,2-cyclohexanedione, a 1,3-cyclohexanedione or a 1,4-cyclohexanedione.","In one aspect, the monooxygenase activity comprises a chlorophenol 4-monooxygenase activity or a xylene monooxygenase activity.","The invention provides a pharmaceutical composition comprising a polypeptide of the invention.","The invention provides a method for converting a ketone to its corresponding ester comprising contacting the ketone with a polypeptide of the invention under conditions wherein the polypeptide catalyzes the conversion of the ketone to its corresponding ester. In one aspect, the polypeptide has an monooxygenase activity that is enantiospecific to generate a substantially chiral product. In one aspect, the ester is an aromatic or an aliphatic ester.","The invention provides a method for converting a cycloaliphatic ketone to its corresponding lactone comprising contacting the cycloaliphatic ketone with a polypeptide of the invention under conditions wherein the polypeptide catalyzes the conversion of the cycloaliphatic ketone to its corresponding lactone. In one aspect, the polypeptide has an monooxygenase activity that is enantiospecific to generate a substantially chiral product. In one aspect, the ester or lactone has at least one of the following structures:",{"@attributes":{"id":"p-0182","num":"0181"},"chemistry":{"@attributes":{"id":"CHEM-US-00002","num":"00002"},"img":{"@attributes":{"id":"EMI-C00002","he":"16.68mm","wi":"42.93mm","file":"US08962800-20150224-C00002.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"wherein: R, R, Rand Rare each independently selected from \u2014H, substituted or unsubstituted alkyl, alkenyl, alkynyl, aryl, heteroaryl, cycloalkyl, and heterocyclic; wherein the substituted groups are substituted with one or more of lower alkyl, hydroxy, alkoxy, mercapto, cycloalkyl, heterocyclic, aryl, heteroaryl, aryloxy, and halogen, or two or more of R, R, Rand Rmay together form cyclic moieties, and, R\u2032 is selected from substituted or unsubstituted alkylene, alkenylene, alkynylene, arylene, heteroarylene, cycloalkylene, and heterocyclic; wherein the substitutions are substituted with one or more of lower alkyl, hydroxy, alkoxy, mercapto, cycloalkyl, heterocyclic, aryl, heteroaryl, aryloxy, and halogen.","Nitroreductases","In one aspect, the invention provides nitroreductases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having a nitroreductase activity, including thermostable and thermotolerant nitroreductase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","Nitroreductases can catalyze the six-electron reduction of nitro compounds to the corresponding amines. Amines have a variety of applications as synthons and advanced pharmaceutical intermediates. There are markets for both aromatic amines and chiral aliphatic amines.","Nitroreductases of the invention fall in to two main classes. These are the oxygen-sensitive and oxygen-insensitive nitroreductases. The oxygen-sensitive enzyme can catalyze nitroreduction only under anaerobic conditions. A nitro anion radical is formed by a one-electron transfer and is immediately reoxidized in the presence of oxygen thus generating a futile cycle whereby reducing equivalents are consumed without nitroreduction. On the other hand the oxygen-insensitive nitroreductases catalyze nitroreduction in a series of two electron transfers, first via the nitroso and then the hydroxylamine intermediates before forming the amine.","Nitrilases","In one aspect, the invention provides nitrilases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having a nitrilase activity, including thermostable and thermotolerant nitrilase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","Nitrilases of the invention can be used for hydrolyzing a nitrile to a carboxylic acid. In one embodiment, the conditions of the reaction comprise aqueous conditions. In another embodiment, the conditions comprise a pH of about 8.0 and\/or a temperature from about 37\u00b0 C. to about 45\u00b0 C. Nitrilases of the invention can also be used for hydrolyzing a cyanohydrin moiety or an aminonitrile moiety of a molecule. Alternatively, the nitrilases of the invention can be used for making a chiral \u03b1-hydroxy acid molecule, a chiral amino acid molecule, a chiral \u03b2-hydroxy acid molecule, or a chiral gamma-hydroxy acid molecule. In one embodiment, the chiral molecule is an (R)-enantiomer. In another embodiment, the chiral molecule is an (S)-enantiomer. In one embodiment of the invention, one particular enzyme can have R-specificity for one particular substrate and the same enzyme can have S-specificity for a different particular substrate.","In one aspect, nitrilases of the invention can be used for making a composition or an intermediate thereof, wherein the nitrilase of the invention hydrolyzes a cyanohydrin or a aminonitrile moiety. In one embodiment, the composition or intermediate thereof comprises (S)-2-amino-4-phenyl butanoic acid. In a further embodiment, the composition or intermediate thereof comprises an L-amino acid. In a further embodiment, the composition comprises a food additive or a pharmaceutical drug.","In another aspect, nitrilases of the invention can be used for making an (R)-ethyl 4-cyano-3-hydroxybutyric acid, wherein the nitrilase of the invention acts upon a hydroxyglutaryl nitrile and selectively produces an (R)-enantiomer, so as to make (R)-ethyl 4-cyano-3-hydroxybutyric acid. In one embodiment, the ee is at least 95% or at least 99%. In another embodiment, the hydroxyglutaryl nitrile comprises 1,3-di-cyano-2-hydroxy-propane or 3-hydroxyglutaronitrile.","In another aspect, nitrilases of the invention can be used for making an (S)-ethyl 4-cyano-3-hydroxybutyric acid, wherein the nitrilase of the invention acts upon a hydroxyglutaryl nitrile and selectively produces an (S)-enantiomer, so as to make (S)-ethyl 4-cyano-3-hydroxybutyric acid.","In another aspect, the nitrilases of the invention can be used for making a (R)-mandelic acid, wherein the nitrilase of the invention acts upon a mandelonitrile to produce a (R)-mandelic acid. In one embodiment, the (R)-mandelic acid comprises (R)-2-chloromandelic acid. In another embodiment, the (R)-mandelic acid comprises an aromatic ring substitution in the ortho-, meta-, or para-positions; a 1-naphthyl derivative of (R)-mandelic acid, a pyridyl derivative of (R)-mandelic acid or a thienyl derivative of (R)-mandelic acid or a combination thereof.","In another aspect, the nitrilases of the invention can be used for making a (S)-mandelic acid, wherein the nitrilase of the invention acts upon a mandelonitrile to produce a (S)-mandelic acid. In one embodiment, the (S)-mandelic acid comprises (S)-methyl benzyl cyanide and the mandelonitrile comprises (S)-methoxy-benzyl cyanide. In one embodiment, the (S)-mandelic acid comprises an aromatic ring substitution in the ortho-, meta-, or para-positions; a 1-naphthyl derivative of (S)-mandelic acid, a pyridyl derivative of (S)-mandelic acid or a thienyl derivative of (S)-mandelic acid or a combination thereof.","In yet another aspect, the nitrilases of the invention can be used for making a (S)-phenyl lactic acid derivative or a (R)-phenyllactic acid derivative, wherein the nitrilase of the invention acts upon a phenyllactonitrile and selectively produces an (S)-enantiomer or an (R)-enantiomer, thereby producing an (S)-phenyl lactic acid derivative or an (R)-phenyl lactic acid derivative.","P450 Enzymes","In one aspect, the invention provides P450 enzymes, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having a P450 enzymatic activity, including thermostable and thermotolerant P450 enzymatic activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","P450s are oxidative enzymes that are widespread in nature and polypeptides of the invention having P450 activity can be used in processes such as detoxifying xenobiotics, catabolism of unusual carbon sources and biosynthesis of secondary metabolites (e.g., detoxification of toxic composition, e.g., pesticides, poisons, chemical warfare agents and the like). These oxygenases activate molecular oxygen using an iron-heme center and utilize a redox electron shuttle to support the epoxidation reaction.","In one aspect, the P450 activity comprises a monooxygenation reaction. In one aspect, the P450 activity comprises catalysis of incorporation of oxygen into a substrate. In one aspect, the P450 activity can further comprise hydroxylation of aliphatic or aromatic carbons. In another aspect, the P450 activity can comprise epoxidation. Alternatively, the P450 activity can comprise N-, O-, or S-dealkylation. In one aspect, the P450 activity can comprise dehalogenation. In another aspect the P450 activity can comprise oxidative deamination. Alternatively, the P450 activity can comprise N-oxidation or N-hydroxylation. In one aspect, the P450 activity can comprise sulphoxide formation.","In one aspect, the epoxidase activity further comprises an alkene substrate. The epoxidase activity can further comprise production of a chiral product. In one aspect, the epoxidase activity can be enantioselective.","Pectate Lyases","In one aspect, the invention provides pectate lyases, e.g. pectinases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having a pectate lyase, e.g. a pectinase activity, including thermostable and thermotolerant pectate lyase, e.g. a pectinase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","The pectate lyases, e.g. pectinases, of the invention can be used to catalyze the beta-elimination or hydrolysis of pectin and\/or polygalacturonic acid, such as 1,4-linked alpha-D-galacturonic acid. They can be used in variety of industrial applications, e.g., to treat plant cell walls, such as those in cotton or other natural fibers. In another exemplary industrial application, the polypeptides of the invention can be used in textile scouring.","In one aspect, pectate lyase activity comprises catalysis of beta-elimination (trans-elimination) or hydrolysis of pectin or polygalacturonic acid (pectate). The pectate lyase activity can comprise the breakup or dissolution of plant cell walls. The pectate lyase activity can comprise beta-elimination (trans-elimination) or hydrolysis of 1,4-linked alpha-D-galacturonic acid. The pectate lyase activity can comprise catalysis of beta-elimination (trans-elimination) or hydrolysis of methyl-esterified galacturonic acid. The pectate lyase activity can be exo-acting or endo-acting. In one aspect, the pectate lyase activity is endo-acting and acts at random sites within a polymer chain to give a mixture of oligomers. In one aspect, the pectate lyase activity is exo-acting and acts from one end of a polymer chain and produces monomers or dimers. The pectate lyase activity can catalyze the random cleavage of alpha-1,4-glycosidic linkages in pectic acid (polygalacturonic acid) by trans-elimination or hydrolysis. The pectate lyase activity can comprise activity the same or similar to pectate lyase (EC 4.2.2.2), poly(1,4-alpha-D-galacturonide) lyase, polygalacturonate lyase (EC 4.2.2.2), pectin lyase (EC 4.2.2.10), polygalacturonase (EC 3.2.1.15), exo-polygalacturonase (EC 3.2.1.67), exo-polygalacturonate lyase (EC 4.2.2.9) or exo-poly-alpha-galacturonosidase (EC 3.2.1.82). The pectate lyase activity can comprise beta-elimination (trans-elimination) or hydrolysis of galactan to galactose or galactooligomers. The pectate lyase activity can comprise beta-elimination (trans-elimination) or hydrolysis of a plant fiber. The plant fiber can comprise cotton fiber, hemp fiber or flax fiber.","The pectate lyases, e.g. pectinases, of the invention can be used for hydrolyzing, breaking up or disrupting a pectin- or pectate (polygalacturonic acid)-comprising composition, for liquefying or removing a pectin or pectate (polygalacturonic acid) from a composition. Alternatively, the pectate lyases, e.g. pectinases, of the invention can be used in detergent compositions. In one aspect, the pectate lyase is a nonsurface-active pectate lyase or a surface-active pectate lyase. The pectate lyase can be formulated in a non-aqueous liquid composition, a cast solid, a granular form, a particulate form, a compressed tablet, a gel form, a paste or a slurry form.","In one aspect, the pectate lyases, e.g. pectinases, of the invention can be used for washing an object. In another aspect, textiles or fabrics comprise a polypeptide of the invention, or a polypeptide encoded by a nucleic acid of the invention, wherein the polypeptide has pectate lyase, e.g. pectinase activity. Additionally, the pectate lyases, e.g. pectinases, of the invention can be used for fiber, thread, textile or fabric scouring. In one aspect, the pectate lyase is an alkaline active and thermostable pectate lyase. The desizing and scouring treatments can be combined in a single bath. The method can further comprise addition of an alkaline and thermostable amylase. The desizing or scouring treatments can comprise conditions of between about pH 8.5 to pH 10.0 and temperatures of at about 40\u00b0 C. The method can further comprise addition of a bleaching step. The desizing, scouring and bleaching treatments can be done simultaneously or sequentially in a single-bath container. The bleaching treatment can comprise hydrogen peroxide or at least one peroxy compound that can generate hydrogen peroxide when dissolved in water, or combinations thereof, and at least one bleach activator. The fiber, thread, textile or fabric can comprise a cellulosic material. The cellulosic material can comprise a crude fiber, a yarn, a woven or knit textile, a cotton, a linen, a flax, a ramie, a rayon, a hemp, a jute or a blend of natural or synthetic fibers.","Alternatively, the pectate lyases, e.g. pectinases, of the invention can be used in feeds or foods. For example, the pectate lyases, e.g. pectinases, of the invention can be used to improve the extraction of oil from an oil-rich plant material. In one aspect, the oil-rich plant material comprises an oil-rich seed. The oil can be a soybean oil, an olive oil, a rapeseed (canola) oil or a sunflower oil.","In another aspect, the pectate lyases, e.g. pectinases, of the invention can be used for preparing a fruit or vegetable juice, syrup, puree or extract. In yet another aspect, the pectate lyases, e.g. pectinases, of the invention can used for treating a paper or a paper or wood pulp. Alternatively, the invention provides papers or paper products or paper pulps comprising a pectate lyase of the invention, or a polypeptide encoded by a nucleic acid of the invention.","In yet another aspect, the invention provides pharmaceutical compositions comprising a polypeptide of the invention, or a polypeptide encoded by a nucleic acid of the invention, wherein the polypeptide has pectate lyase, e.g. pectinase activity. The pharmaceutical composition can act as a digestive aid.","Alternatively, the invention provides oral care products comprising a polypeptide of the invention, or a polypeptide encoded by a nucleic acid of the invention, wherein the polypeptide has pectate lyase, e.g. pectinase activity. The oral care product can comprise a toothpaste, a dental cream, a gel or a tooth powder, an odontic, a mouth wash, a pre- or post brushing rinse formulation, a chewing gum, a lozenge or a candy.","Phosphatases","In one aspect, the invention provides phosphatases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having a phosphatase activity, including thermostable and thermotolerant phosphatase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","Phosphatases are a group of enzymes that remove phosphate groups from organophosphate ester compounds. There are numerous phosphatases, including alkaline phosphatases, phosphodiesterases and phytases.","Alkaline phosphatases are widely distributed enzymes and are composed of a group of enzymes which hydrolyze organic phosphate ester bonds at alkaline pH.","Phosphodiesterases are capable of hydrolyzing nucleic acids by hydrolyzing the phosphodiester bridges of DNA and RNA. The classification of phosphodiesterases depends upon which side of the phosphodiester bridge is attacked. The 3\u2032 enzymes specifically hydrolyze the ester linkage between the 3\u2032 carbon and the phosphoric group whereas the 5\u2032 enzymes hydrolyze the ester linkage between the phosphoric group and the 5\u2032 carbon of the phosphodiester bridge. The best known of the class 3\u2032 enzymes is a phosphodiesterase from the venom of the rattlesnake or from a rustle's viper, which hydrolyses all the 3\u2032 bonds in either RNA or DNA liberating nearly all the nucleotide units as nucleotide 5\u2032 phosphates. This enzyme requires a free 3\u2032 hydroxyl group on the terminal nucleotide residue and proceeds stepwise from that end of the polynucleotide chain. This enzyme and all other nucleases which attack only at the ends of the polynucleotide chains are called exonucleases. The 5\u2032 enzymes are represented by a phosphodiesterase from bovine spleen, also an exonuclease, which hydrolyses all the 5\u2032 linkages of both DNA and RNA and thus liberates only nucleoside 3\u2032 phosphates. It begins its attack at the end of the chain having a free 3\u2032 hydroxyl group.","Phytase enzymes remove phosphate from phytic acid (inositol hexaphosphoric acid), a compound found in plants such as corn, wheat and rice. The enzyme has commercial use for the treatment of animal feed, making the inositol of the phytic acid available for animal nutrition. Phytases are used to improve the utilization of natural phosphorus in animal feed. Use of phytase as a feed additive enables the animal to metabolize a larger degree of its cereal feed's natural mineral content thereby reducing or altogether eliminating the need for synthetic phosphorus additives. More important than the reduced need for phosphorus additives is the corresponding reduction of phosphorus in pig and chicken waste. Many European countries severely limit the amount of manure that can be spread per acre due to concerns regarding phosphorus contamination of ground water.","Alkaline phosphatases hydrolyze monophosphate esters, releasing an organic phosphate and the cognate alcohol compound. It is non-specific with respect to the alcohol moiety and it is this feature which accounts for the many uses of this enzyme.","Phospholipases","In one aspect, the invention provides phospholipases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having a phospholipase activity, including thermostable and thermotolerant phospholipase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","Phospholipases are enzymes that hydrolyze the ester bonds of phospholipids. Corresponding to their importance in the metabolism of phospholipids, these enzymes are widespread among prokaryotes and eukaryotes. The phospholipases affect the metabolism, construction and reorganization of biological membranes and are involved in signal cascades. Several types of phospholipases are known which differ in their specificity according to the position of the bond attacked in the phospholipid molecule. Phospholipase A1 (PLA1) removes the 1-position fatty acid to produce free fatty acid and 1-lyso-2-acylphospholipid. Phospholipase A2 (PLA2) removes the 2-position fatty acid to produce free fatty acid and 1-acyl-2-lysophospholipid. PLA1 and PLA2 enzymes can be intra- or extra-cellular, membrane-bound or soluble. Intracellular PLA2 is found in almost every mammalian cell. Phospholipase C (PLC) removes the phosphate moiety to produce 1,2 diacylglycerol and phospho base. Phospholipase D (PLD) produces 1,2-diacylglycerophosphate and base group. PLC and PLD are important in cell function and signaling. PLD had been the dominant phospholipase in biocatalysis. Patatins are another type of phospholipase, thought to work as a PLA.","The invention provides methods for cleaving a glycerolphosphate ester linkage comprising the following steps: (a) providing a polypeptide having a phospholipase activity, wherein the polypeptide comprises an amino acid sequence of the invention, or the polypeptide is encoded by a nucleic acid of the invention; (b) providing a composition comprising a glycerolphosphate ester linkage; and, (c) contacting the polypeptide of step (a) with the composition of step (b) under conditions wherein the polypeptide cleaves the glycerolphosphate ester linkage. In one aspect, the conditions comprise between about pH 5 to about 5.5, or, between about pH 4.5 to about 5.0. In one aspect, the conditions comprise a temperature of between about 40\u00b0 C. and about 70\u00b0 C. In one aspect, the composition comprises a vegetable oil. In one aspect, the composition comprises an oilseed phospholipid. In one aspect, the cleavage reaction can generate a water extractable phosphorylated base and a diglyceride.","Phospholipases of the invention can be used in oil degumming, wherein the phospholipase is used under conditions wherein the phospholipase can cleave ester linkages in an oil, thereby degumming the oil. In one aspect, the oil is a vegetable oil. In another aspect, the vegetable oil comprises oilseed. The vegetable oil can comprise palm oil, rapeseed oil, corn oil, soybean oil, canola oil, sesame oil, peanut oil or sunflower oil. In one aspect, the method further comprises addition of a phospholipase of the invention, another phospholipase, another enzyme, or a combination thereof.","In another aspect of the invention, phospholipases of the invention can be used for converting a non-hydratable phospholipid to a hydratable form or for caustic refining of a phospholipid-containing composition. In the latter use, the polypeptide of the invention can be added before caustic refining and the composition comprising the phospholipid can comprise a plant and the polypeptide can be expressed transgenically in the plant, the polypeptide having a phospholipase activity can be added during crushing of a seed or other plant part, or, the polypeptide having a phospholipase activity is added following crushing or prior to refining. The polypeptide can be added during caustic refining and varying levels of acid and caustic can be added depending on levels of phosphorous and levels of free fatty acids. The polypeptide can be added after caustic refining: in an intense mixer or retention mixer prior to separation; following a heating step; in a centrifuge; in a soapstock; in a washwater; or, during bleaching or deodorizing steps.","In yet another aspect, the phospholipases of the invention can be used for purification of a phytosterol or a triterpene. The phytosterol or a triterpene can comprise a plant sterol. The plant sterol can be derived from a vegetable oil. The vegetable oil can comprise a coconut oil, canola oil, cocoa butter oil, corn oil, cottonseed oil, linseed oil, olive oil, palm oil, peanut oil, oil derived from a rice bran, safflower oil, sesame oil, soybean oil or a sunflower oil. The method can comprise use of nonpolar solvents to quantitatively extract free phytosterols and phytosteryl fatty-acid esters. The phytosterol or a triterpene can comprise a \u03b2-sitosterol, a campesterol, a stigmasterol, a stigmastanol, a \u03b2-sitostanol, a sitostanol, a desmosterol, a chalinasterol, a poriferasterol, a clionasterol or a brassicasterol.","In one embodiment, the phospholipases of the invention can be used for refining a crude oil. The polypeptide can have a phospholipase activity is in a water solution that is added to the composition. The water level can be between about 0.5 to 5%. The process time can be less than about 2 hours, less than about 60 minutes, less than about 30 minutes, less than 15 minutes, or less than 5 minutes. The hydrolysis conditions can comprise a temperature of between about 25\u00b0 C.-70\u00b0 C. The hydrolysis conditions can comprise use of caustics. The hydrolysis conditions can comprise a pH of between about pH 3 and pH 10, between about pH 4 and pH 9, or between about pH 5 and pH 8. The hydrolysis conditions can comprise addition of emulsifiers and\/or mixing after the contacting of step (c). The methods can comprise addition of an emulsion-breaker and\/or heat to promote separation of an aqueous phase. The methods can comprise degumming before the contacting step to collect lecithin by centrifugation and then adding a PLC, a PLC and\/or a PLA to remove non-hydratable phospholipids. The methods can comprise water degumming of crude oil to less than 10 ppm for edible oils and subsequent physical refining to less than about 50 ppm for biodiesel oils. The methods can comprise addition of acid to promote hydration of non-hydratable phospholipids.","Phytases","In one aspect, the invention provides phytases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having a phytase activity, including thermostable and thermotolerant phytase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","Conversion of phytate to inositol and inorganic phosphorous can be catalyzed by phytase enzymes. Phytases such as phytase #EC 3.1.3.8 are capable of catalyzing the hydrolysis of myo-inositol hexaphosphate to D-myo-inositol 1,2,4,5,6-pentaphosphate and orthophosphate. Other phytases hydrolyze inositol pentaphosphate to tetra-, tri-, and lower phosphates. Acid phosphatases are enzymes that catalytically hydrolyze a wide variety of phosphate esters. For example, #EC 3.1.3.2 enzymes catalyze the hydrolysis of orthophosphoric monoesters to orthophosphate products.","Phytases of the invention can be used in producing phytase as a feed additive, e.g. for monogastric animals, fish, poultry, ruminants and other non-ruminants. Phytases of the invention can also be used for producing animal feed from certain industrial processes, e.g., wheat and corn waste products. In one aspect, the wet milling process of corn produces glutens sold as animal feeds. The addition of phytase improves the nutritional value of the feed product.","Phytases of the invention may also be used in dietary aids or in pharmaceutical compositions, for reducing pollution and increasing nutrient availability in an environment or environmental sample by degrading environmental phytic acid, for liberating minerals from phytates in plant materials either in vitro, i.e., in feed treatment processes, or in vivo, i.e., by administering the enzymes to animals.","Polymerases","In one aspect, the invention provides polymerases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having a polymerase activity, including thermostable and thermotolerant polymerase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","The polymerase enzymes of the invention can have different polymerase activities at various high temperatures. In one aspect, the polymerase activity comprises addition of deoxynucleotides at the 3\u2032 hydroxyl end of a polynucleotide. The invention also provides kits, e.g., diagnostic kits, and methods for performing various amplification reactions, e.g., polymerase chain reactions, transcription amplifications, ligase chain reactions, self-sustained sequence replication or Q Beta replicase amplifications.","In one aspect, the polymerase activity comprises addition of nucleotides at the 3\u2032 hydroxyl end of a nucleic acid. The polymerase activity can comprise a 5\u2032\u21923\u2032 polymerase activity, a 3\u2032\u21925\u2032 exonuclease activity or a 5\u2032\u21923\u2032 exonuclease activity or all or a combination thereof. In one aspect, the polymerase activity comprises only a 5\u2032\u21923\u2032 polymerase activity, but not a 3\u2032\u21925\u2032 exonuclease activity or a 5\u2032\u21923\u2032 exonuclease activity. In another aspect, the polymerase activity can comprise a 5\u2032\u21923\u2032 polymerase activity and a 3\u2032\u21925\u2032 exonuclease activity, but not a 5\u2032\u21923\u2032 exonuclease activity. Alternatively, the polymerase activity can comprise a 5\u2032\u21923\u2032 polymerase activity and a 5\u2032\u21923\u2032 exonuclease activity, but not a 3\u2032\u21925\u2032 exonuclease activity. The polymerase activity can comprise addition of dUTP or dITP. The polymerase activity can comprise addition of a modified or a non-natural nucleotide to a polynucleotide, such as an analog of guanine, cytosine, thymine, adenine or uracil, e.g., a 2-aminopurine, an inosine or a 5-methylcytosine.","In one aspect, the polymerase activity can comprise strand displacement properties. In one aspect, the polymerase activity comprises reverse transcriptase activity.","Proteases","In one aspect, the invention provides proteases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having a protease activity, including thermostable and thermotolerant protease activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","Proteases of the invention can be carbonyl hydrolases which act to cleave peptide bonds of proteins or peptides. Proteolytic enzymes are ubiquitous in occurrence, found in all living organisms, and are essential for cell growth and differentiation. The extracellular proteases are of commercial value and find multiple applications in various industrial sectors. Industrial applications of proteases include food processing, brewing, alcohol production, peptide synthesis, enantioselectivity, hide preparation in the leather industry, waste management and animal degradation, silver recovery in the photographic industry, medical treatment, silk degumming, biofilm degradation, biomass conversion to ethanol, biodefense, antimicrobial agents and disinfectants, personal care and cosmetics, biotech reagents and in increasing starch yield from corn wet milling. Additionally, proteases are important components of laundry detergents and other products. Within biological research, proteases are used in purification processes to degrade unwanted proteins. It is often desirable to employ proteases of low specificity or mixtures of more specific proteases to obtain the necessary degree of degradation.","Proteases are classified according to their catalytic mechanisms. The International Union of Biochemistry and Molecular Biology (IUBMB) recognizes four mechanistic classes: (1) the serine proteases; (2) the cysteine proteases; (3) the aspartic proteases; and (4) the metalloproteases. In addition, the IUBMB recognizes a class of endopeptidases (oligopeptidases) of unknown catalytic mechanism. The serine proteases have alkaline pH optima, the metalloproteases are optimally active around neutrality, and the cysteine and aspartic enzymes have acidic pH optima. Serine proteases class comprises two distinct families: the chymotrypsin family, which includes the mammalian enzymes such as chymotrypsin, trypsin, elastase, or kallikrein, and the subtilisin family, which include the bacterial enzymes such as subtilisin. Serine proteases are used for a variety of industrial purposes, such as laundry detergents to aid in the removal of proteinaceous stains. In the food processing industry, serine proteases are used to produce protein-rich concentrates from fish and livestock, and in the preparation of dairy products.","The proteases of the invention can be used in a variety of diagnostic, therapeutic, and industrial contexts. The proteases of the invention can be used as, e.g., an additive for a detergent, for processing foods and for chemical synthesis utilizing a reverse reaction. Additionally, the proteases of the invention can be used in food processing, brewing, bath additives, alcohol production, peptide synthesis, enantioselectivity, hide preparation in the leather industry, waste management and animal degradation, silver recovery in the photographic industry, medical treatment, silk degumming, biofilm degradation, biomass conversion to ethanol, biodefense, antimicrobial agents and disinfectants, personal care and cosmetics, biotech reagents, in increasing starch yield from corn wet milling and pharmaceuticals such as digestive aids and anti-inflammatory (anti-phlogistic) agents.","Xylanases","In one aspect, the invention provides xylanases, polynucleotides encoding them, and methods of making and using these polynucleotides and polypeptides. In one aspect, the invention is directed to polypeptides, e.g., enzymes, having a xylanase activity, including thermostable and thermotolerant xylanase activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides.","Xylanases (e.g., endo-1,4-beta-xylanase, EC 3.2.1.8) of the invention can hydrolyze internal \u03b2-1,4-xylosidic linkages in xylan to produce smaller molecular weight xylose and xylo-oligomers. Xylans are polysaccharides formed from 1,4-\u03b2-glycoside-linked D-xylopyranoses. Xylanases of the invention are of considerable commercial value, being used in the food industry, for baking and fruit and vegetable processing, breakdown of agricultural waste, in the manufacture of animal feed and in pulp and paper production.","Arabinoxylanase are major non-starch polysaccharides of cereals representing 2.5-7.1% w\/w depending on variety and growth conditions. The physicochemical properties of this polysaccharide are such that it gives rise to viscous solutions or even gels under oxidative conditions. In addition, arabinoxylans have high water-binding capacity and may have a role in protein foam stability. All of these characteristics present problems for several industries including brewing, baking, animal nutrition and paper manufacturing. In brewing applications, the presence of xylan results in wort filterability and haze formation issues. In baking applications (especially for cookies and crackers), these arabinoxylans create sticky doughs that are difficult to machine and reduce biscuit size. In addition, this carbohydrate is implicated in rapid rehydration of the baked product resulting in loss of crispiness and reduced shelf-life. For monogastric animal feed applications with cereal diets, arabinoxylan is a major contributing factor to viscosity of gut contents and thereby adversely affects the digestibility of the feed and animal growth rate. For ruminant animals, these polysaccharides represent substantial components of fiber intake and more complete digestion of arabinoxylans would facilitate higher feed conversion efficiencies.","Xylanases are currently used as additives (dough conditioners) in dough processing for the hydrolysis of water soluble arabinoxylan. In baking applications (especially for cookies and crackers), arabinoxylan creates sticky doughs that are difficult to machine and reduce biscuit size. In addition, this carbohydrate is implicated in rapid rehydration of the baked product resulting in loss of crispiness and reduced shelf-life.","The enhancement of xylan digestion in animal feed may improve the availability and digestibility of valuable carbohydrate and protein feed nutrients. For monogastric animal feed applications with cereal diets, arabinoxylan is a major contributing factor to viscosity of gut contents and thereby adversely affects the digestibility of the feed and animal growth rate. For ruminant animals, these polysaccharides represent substantial components of fiber intake and more complete digestion would facilitate higher feed conversion efficiencies. It is desirable for animal feed xylanases to be active in the animal stomach. This requires a feed enzyme to have high activity at 37\u00b0 C. and at low pH for monogastrics (pH 2-4) and near neutral pH for ruminants (pH 6.5-7). The enzyme should also possess resistance to animal gut xylanases and stability at the higher temperatures involved in feed pelleting. As such, there is a need in the art for xylanase feed additives for monogastric feed with high specific activity, activity at 35-40\u00b0 C. and pH 2-4, half life greater than 30 minutes in SGF and a half-life >5 minutes at 85\u00b0 C. in formulated state. For ruminant feed, there is a need for xylanase feed additives that have a high specific activity, activity at 35-40\u00b0 C. and pH 6.5-7.0, half life greater than 30 minutes in SRF and stability as a concentrated dry powder.","In one aspect, the xylanases of the invention are also used in improving the quality and quantity of milk protein production in lactating cows, increasing the amount of soluble saccharides in the stomach and small intestine of pigs, improving late egg production efficiency and egg yields in hens. Additionally, xylanases of the inventions can be used in biobleaching and treatment of chemical pulps, biobleaching and treatment of wood or paper pulps, in reducing lignin in wood and modifying wood, as feed additives and\/or supplements or in manufacturing cellulose solutions. Detergent compositions comprising xylanases of the invention are used for fruit, vegetables and\/or mud and clay compounds.","In another aspect, xylanases of the invention can be used in compositions for the treatments and\/or prophylaxis of coccidiosis. In yet another aspect, xylanases of the invention can be used in the production of water soluble dietary fiber, in improving the filterability, separation and production of starch, the beverage industry in improving filterability of wort or beer, in reducing viscosity of plant material, or in increasing viscosity or gel strength of food products such as jam, marmalade, jelly, juice, paste, soup, salsa, etc. Xylanases of the invention may also be used in hydrolysis of hemicellulose for which it is selective, particularly in the presence of cellulose. In addition, xylanases of the invention can also be used in the production of ethanol, in transformation of a microbe that produces ethanol, in production of oenological tannins and enzymatic composition, in stimulating the natural defenses of plants, in production of sugars from hemicellulose substrates, in the cleaning of fruit, vegetables, mud or clay containing soils, in cleaning beer filtration membranes, and in killing or inhibiting microbial cells.","Table 1, below, lists the various EC (Enzyme Commission) Numbers along with the corresponding mode of action for each enzyme class, subclass and sub-subclass. Enzyme nomenclature is based upon the recommendations of the Nomenclature Committee of the International Union of Biochemistry and Molecular Biology (IUBMB). Table 2, below, lists the various EC Numbers along with the corresponding name given to each enzyme class, subclass and sub-subclass. Tables 1 and 2 list exemplary enzymatic activities of polypeptides of the invention, as can be determined by sequence identity (e.g., homology); and in one embodiment a sequence of the invention comprises an enzyme having at least 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more sequence identity (homology) to an enzyme encoded by an exemplary sequence of the invention, including all odd numbered SEQ ID NO:1 to SEQ ID NO:26,897, or an exemplary polypeptide of the invention, including all even numbered SEQ ID NO:2 to SEQ ID NO:26,898, and with an exemplary function as listed in Table 1 or Table 2.","Table 3, below, contains the exemplary SEQ ID NO:s of the invention, and the closest hit (BLAST) information for the polynucleotides and polypeptides of the invention. This information includes the closest hit organism, accession number, definition of the closest hit, EC number, percentage amino acid identity and the percent nucleotide identity, along with the Evalue for the closest hits. The information contained in Table 3 identifies exemplary activities of polypeptides of the invention, based on sequence identity (homology). In one embodiment a sequence of the invention comprises an enzyme with at least 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more sequence identity (homology) to an enzyme as listed in Table 3.",{"@attributes":{"id":"p-0242","num":"0241"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 1"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"EC (Enzyme Commission) Numbers with the corresponding mode of"},{"entry":"action for each enzyme class, subclass and sub-subclass"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"175pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1.\u2014.\u2014.\u2014","Oxidoreductases."]},{"entry":["1.1.\u2014.\u2014","Acting on the CH\u2014OH group of donors."]},{"entry":["1.1.1.\u2014","With NAD(+) or NADP(+) as acceptor."]},{"entry":["1.1.2.\u2014","With a cytochrome as acceptor."]},{"entry":["1.1.3.\u2014","With oxygen as acceptor."]},{"entry":["1.1.4.\u2014","With a disulfide as acceptor."]},{"entry":["1.1.5.\u2014","With a quinone or similar compound as acceptor."]},{"entry":["1.1.99.\u2014","With other acceptors."]},{"entry":["1.2.\u2014.\u2014","Acting on the aldehyde or oxo group of donors."]},{"entry":["1.2.1.\u2014","With NAD(+) or NADP(+) as acceptor."]},{"entry":["1.2.2.\u2014","With a cytochrome as acceptor."]},{"entry":["1.2.3.\u2014","With oxygen as acceptor."]},{"entry":["1.2.4.\u2014","With a disulfide as acceptor."]},{"entry":["1.2.7.\u2014","With an iron-sulfur protein as acceptor."]},{"entry":["1.2.99.\u2014","With other acceptors."]},{"entry":["1.3.\u2014.\u2014","Acting on the CH\u2014CH group of donors."]},{"entry":["1.3.1.\u2014","With NAD(+) or NADP(+) as acceptor."]},{"entry":["1.3.2.\u2014","With a cytochrome as acceptor."]},{"entry":["1.3.3.\u2014","With oxygen as acceptor."]},{"entry":["1.3.5.\u2014","With a quinone or related compound as acceptor."]},{"entry":["1.3.7.\u2014","With an iron-sulfur protein as acceptor."]},{"entry":["1.3.99.\u2014","With other acceptors."]},{"entry":["1.4.\u2014.\u2014","Acting on the CH\u2014NH(2) group of donors."]},{"entry":["1.4.1.\u2014","With NAD(+) or NADP(+) as acceptor."]},{"entry":["1.4.2.\u2014","With a cytochrome as acceptor."]},{"entry":["1.4.3.\u2014","With oxygen as acceptor."]},{"entry":["1.4.4.\u2014","With a disulfide as acceptor."]},{"entry":["1.4.7.\u2014","With an iron-sulfur protein as acceptor."]},{"entry":["1.4.99.\u2014","With other acceptors."]},{"entry":["1.5.\u2014.\u2014","Acting on the CH\u2014NH group of donors."]},{"entry":["1.5.1.\u2014","With NAD(+) or NADP(+) as acceptor."]},{"entry":["1.5.3.\u2014","With oxygen as acceptor."]},{"entry":["1.5.4.\u2014","With a disulfide as acceptor."]},{"entry":["1.5.5.\u2014","With a quinone or similar compound as acceptor."]},{"entry":["1.5.8.\u2014","With a flavin as acceptor."]},{"entry":["1.5.99.\u2014","With other acceptors."]},{"entry":["1.6.\u2014.\u2014","Acting on NADH or NADPH."]},{"entry":["1.6.1.\u2014","With NAD(+) or NADP(+) as acceptor."]},{"entry":["1.6.2.\u2014","With a heme protein as acceptor."]},{"entry":["1.6.3.\u2014","With a oxygen as acceptor."]},{"entry":["1.6.4.\u2014","With a disulfide as acceptor."]},{"entry":["1.6.5.\u2014","With a quinone or similar compound as"]},{"entry":[{},"acceptor."]},{"entry":["1.6.6.\u2014","With a nitrogenous group as acceptor."]},{"entry":["1.6.8.\u2014","With a flavin as acceptor."]},{"entry":["1.6.99.\u2014","With other acceptors."]},{"entry":["1.7.\u2014.\u2014","Acting on other nitrogenous compounds as"]},{"entry":[{},"donors."]},{"entry":["1.7.1.\u2014","With NAD(+) or NADP(+) as acceptor."]},{"entry":["1.7.2.\u2014","With a cytochrome as acceptor."]},{"entry":["1.7.3.\u2014","With oxygen as acceptor."]},{"entry":["1.7.7.\u2014","With an iron-sulfur protein as acceptor."]},{"entry":["1.7.99.\u2014","With other acceptors."]},{"entry":["1.8.\u2014.\u2014","Acting on a sulfur group of donors."]},{"entry":["1.8.1.\u2014","With NAD(+) or NADP(+) as acceptor."]},{"entry":["1.8.2.\u2014","With a cytochrome as acceptor."]},{"entry":["1.8.3.\u2014","With oxygen as acceptor."]},{"entry":["1.8.4.\u2014","With a disulfide as acceptor."]},{"entry":["1.8.5.\u2014","With a quinone or similar compound as"]},{"entry":[{},"acceptor."]},{"entry":["1.8.7.\u2014","With an iron-sulfur protein as acceptor."]},{"entry":["1.8.98.\u2014","With other, known, acceptors."]},{"entry":["1.8.99.\u2014","With other acceptors."]},{"entry":["1.9.\u2014.\u2014","Acting on a heme group of donors."]},{"entry":["1.9.3.\u2014","With oxygen as acceptor."]},{"entry":["1.9.6.\u2014","With a nitrogenous group as acceptor."]},{"entry":["1.9.99.\u2014","With other acceptors."]},{"entry":["1.10.\u2014.\u2014","Acting on diphenols and related"]},{"entry":[{},"substances as donors."]},{"entry":["1.10.1.\u2014","With NAD(+) or NADP(+) as acceptor."]},{"entry":["1.10.2.\u2014","With a cytochrome as acceptor."]},{"entry":["1.10.3.\u2014","With oxygen as acceptor."]},{"entry":["1.10.99.\u2014","With other acceptors."]},{"entry":["1.11.\u2014.\u2014","Acting on a peroxide as acceptor"]},{"entry":[{},"(peroxidases)."]},{"entry":["1.12.\u2014.\u2014","Acting on hydrogen as donor."]},{"entry":["1.12.1.\u2014","With NAD(+) or NADP(+) as acceptor."]},{"entry":["1.12.2.\u2014","With a cytochrome as acceptor."]},{"entry":["1.12.5.\u2014","With a quinone or similar compound as"]},{"entry":[{},"acceptor."]},{"entry":["1.12.7.\u2014","With an iron-sulfur protein as acceptor."]},{"entry":["1.12.98.\u2014","With other known acceptors."]},{"entry":["1.12.99.\u2014","With other acceptors."]},{"entry":["1.13.\u2014.\u2014","Acting on single donors with"]},{"entry":[{},"incorporation of molecular oxygen."]},{"entry":["1.13.11.\u2014","With incorporation of two atoms of oxygen."]},{"entry":["1.13.12.\u2014","With incorporation of one atom of oxygen."]},{"entry":["1.14.\u2014.\u2014","Acting on paired donors, with incorporation or"]},{"entry":[{},"reduction of molecular oxygen"]},{"entry":["1.14.11.\u2014","With 2-oxoglutarate as one donor, and"]},{"entry":[{},"incorporation of one atom each of oxygen into both donors"]},{"entry":["1.14.12.\u2014","With NADH or NADPH as one donor, and"]},{"entry":[{},"incorporation of two atoms of oxygen into one donor"]},{"entry":["1.14.13.\u2014","With NADH or NADPH as one donor, and"]},{"entry":[{},"incorporation of one atom of oxygen"]},{"entry":["1.14.14.\u2014","With reduced flavin or flavoprotein as one"]},{"entry":[{},"donor, and incorporation of one atom of oxygen"]},{"entry":["1.14.15.\u2014","With a reduced iron-sulfur protein as one"]},{"entry":[{},"donor, and incorporation of one atom of oxygen"]},{"entry":["1.14.16.\u2014","With reduced pteridine as one donor, and"]},{"entry":[{},"incorporation of one atom of oxygen"]},{"entry":["1.14.17.\u2014","With reduced ascorbate as one donor, and"]},{"entry":[{},"incorporation of one atom of oxygen"]},{"entry":["1.14.18.\u2014","With another compound as one donor, and"]},{"entry":[{},"incorporation of one incorporation of one atom of"]},{"entry":[{},"oxygen"]},{"entry":["1.14.19.\u2014","With oxidation of a pair of donors resulting"]},{"entry":[{},"in the reduction of molecular oxygen to two molecules"]},{"entry":[{},"of water"]},{"entry":["1.14.20.\u2014","With 2-oxoglutarate as one donor, and the"]},{"entry":[{},"other dehydrogenated."]},{"entry":["1.14.21.\u2014","With NADH or NADPH as one donor, and"]},{"entry":[{},"the other dehydrogenated."]},{"entry":["1.15.\u2014.\u2014","Acting on superoxide as acceptor."]},{"entry":["1.16.\u2014.\u2014","Oxidizing metal ions."]},{"entry":["1.16.1.\u2014","With NAD(+) or NADP(+) as acceptor."]},{"entry":["1.16.3.\u2014","With oxygen as acceptor."]},{"entry":["1.16.8.\u2014","With flavin as acceptor."]},{"entry":["1.17.\u2014.\u2014","Acting on CH or CH(2) groups."]},{"entry":["1.17.1.\u2014","With NAD(+) or NADP(+) as acceptor."]},{"entry":["1.17.3.\u2014","With oxygen as acceptor."]},{"entry":["1.17.4.\u2014","With a disulfide as acceptor."]},{"entry":["1.17.5.\u2014","With a quinone or similar compound as"]},{"entry":[{},"acceptor"]},{"entry":["1.17.99.\u2014","With other acceptors."]},{"entry":["1.18.\u2014.\u2014","Acting on iron-sulfur proteins as donors."]},{"entry":["1.18.1.\u2014","With NAD(+) or NADP(+) as acceptor."]},{"entry":["1.18.6.\u2014","With dinitrogen as acceptor."]},{"entry":["1.18.96.\u2014","With other, known, acceptors."]},{"entry":["1.18.99.\u2014","With H(+) as acceptor."]},{"entry":["1.19.\u2014.\u2014","Acting on reduced flavodoxin as donor."]},{"entry":["1.19.6.\u2014","With dinitrogen as acceptor."]},{"entry":["1.20.\u2014.\u2014","Acting on phosphorus or arsenic in"]},{"entry":[{},"donors."]},{"entry":["1.20.1.\u2014","Acting on phosphorus or arsenic in"]},{"entry":[{},"donors, with NAD(P)(+) as acceptor"]},{"entry":["1.20.4.\u2014","Acting on phosphorus or arsenic in"]},{"entry":[{},"donors, with disulfide as acceptor"]},{"entry":["1.20.98.\u2014","Acting on phosphorus or arsenic in"]},{"entry":[{},"donors, with other, known acceptors"]},{"entry":["1.20.99.\u2014","Acting on phosphorus or arsenic in"]},{"entry":[{},"donors, with other acceptors"]},{"entry":["1.21.\u2014.\u2014","Acting on x-H and y-H to form an x-y"]},{"entry":[{},"bond."]},{"entry":["1.21.3.\u2014","With oxygen as acceptor."]},{"entry":["1.21.4.\u2014","With a disulfide as acceptor."]},{"entry":["1.21.99.\u2014","With other acceptors."]},{"entry":["1.97.\u2014.\u2014","Other oxidoreductases."]},{"entry":["2.\u2014.\u2014.\u2014","Transferases."]},{"entry":["2.1.\u2014.\u2014","Transferring one-carbon groups."]},{"entry":["2.1.1.\u2014","Methyltransferases."]},{"entry":["2.1.2.\u2014","Hydroxymethyl-, formyl- and related"]},{"entry":[{},"transferases."]},{"entry":["2.1.3.\u2014","Carboxyl- and carbamoyltransferases."]},{"entry":["2.1.4.\u2014","Amidinotransferases."]},{"entry":["2.2.\u2014.\u2014","Transferring aldehyde or ketone residues."]},{"entry":["2.2.1.\u2014","Transketolases and transaldolases."]},{"entry":["2.3.\u2014.\u2014","Acyltransferases."]},{"entry":["2.3.1.\u2014","Transferring groups other than amino-"]},{"entry":[{},"acyl groups."]},{"entry":["2.3.2.\u2014","Aminoacyltransferases."]},{"entry":["2.3.3.\u2014","Acyl groups converted into alkyl on"]},{"entry":[{},"transfer."]},{"entry":["2.4.\u2014.\u2014","Glycosyltransferases."]},{"entry":["2.4.1.\u2014","Hexosyltransferases."]},{"entry":["2.4.2.\u2014","Pentosyltransferases."]},{"entry":["2.4.99.\u2014","Transferring other glycosyl groups."]},{"entry":["2.5.\u2014.\u2014","Transferring alkyl or aryl groups, other"]},{"entry":[{},"than methyl groups."]},{"entry":["2.6.\u2014.\u2014","Transferring nitrogenous groups."]},{"entry":["2.6.1.\u2014","Transaminases (aminotransferases)."]},{"entry":["2.6.3.\u2014","Oximinotransferases."]},{"entry":["2.6.99.\u2014","Transferring other nitrogenous groups."]},{"entry":["2.7.\u2014.\u2014","Transferring phosphorous-containing groups."]},{"entry":["2.7.1.\u2014","Phosphotransferases with an alcohol group as"]},{"entry":[{},"acceptor."]},{"entry":["2.7.2.\u2014","Phosphotransferases with a carboxyl group as"]},{"entry":[{},"acceptor."]},{"entry":["2.7.3.\u2014","Phosphotransferases with a nitrogenous group"]},{"entry":[{},"as acceptor."]},{"entry":["2.7.4.\u2014","Phosphotransferases with a phosphate group as"]},{"entry":[{},"acceptor."]},{"entry":["2.7.6.\u2014","Diphosphotransferases."]},{"entry":["2.7.7.\u2014","Nucleotidyltransferases."]},{"entry":["2.7.8.\u2014","Transferases for other substituted phosphate"]},{"entry":[{},"groups."]},{"entry":["2.7.9.\u2014","Phosphotransferases with paired acceptors."]},{"entry":["2.8.\u2014.\u2014","Transferring sulfur-containing groups."]},{"entry":["2.8.1.\u2014","Sulfurtransferases."]},{"entry":["2.8.2.\u2014","Sulfotransferases."]},{"entry":["2.8.3.\u2014","CoA-transferases."]},{"entry":["2.8.4.\u2014","Transferring alkylthio groups."]},{"entry":["2.9.\u2014.\u2014","Transferring selenium-containing groups."]},{"entry":["2.9.1.\u2014","Selenotransferases."]},{"entry":["3.\u2014.\u2014.\u2014","Hydrolases."]},{"entry":["3.1.\u2014.\u2014","Acting on ester bonds."]},{"entry":["3.1.1.\u2014","Carboxylic ester hydrolases."]},{"entry":["3.1.2.\u2014","Thiolester hydrolases."]},{"entry":["3.1.3.\u2014","Phosphoric monoester hydrolases."]},{"entry":["3.1.4.\u2014","Phosphoric diester hydrolases."]},{"entry":["3.1.5.\u2014","Triphosphoric monoester hydrolases."]},{"entry":["3.1.6.\u2014","Sulfuric ester hydrolases."]},{"entry":["3.1.7.\u2014","Diphosphoric monoester hydrolases."]},{"entry":["3.1.8.\u2014","Phosphoric triester hydrolases."]},{"entry":["3.1.11.\u2014","Exodeoxyribonucleases producing 5\u2032-"]},{"entry":[{},"phosphomonoesters."]},{"entry":["3.1.13.\u2014","Exoribonucleases producing 5\u2032-"]},{"entry":[{},"phosphomonoesters."]},{"entry":["3.1.14.\u2014","Exoribonucleases producing 3\u2032-"]},{"entry":[{},"phosphomonoesters."]},{"entry":["3.1.15.\u2014","Exonucleases active with either ribo- or"]},{"entry":[{},"deoxyribonucleic acid and producing 5\u2032-"]},{"entry":[{},"phosphomonoesters"]},{"entry":["3.1.16.\u2014","Exonucleases active with either ribo- or"]},{"entry":[{},"deoxyribonucleic acid producing 3\u2032-phosphomonoesters"]},{"entry":["3.1.21.\u2014","Endodeoxyribonucleases producing 5\u2032-"]},{"entry":[{},"phosphomonoesters."]},{"entry":["3.1.22.\u2014","Endodeoxyribonucleases producing"]},{"entry":[{},"other than 5\u2032-phosphomonoesters."]},{"entry":["3.1.25.\u2014","Site-specific endodeoxyribonucleases"]},{"entry":[{},"specific for altered bases."]},{"entry":["3.1.26.\u2014","Endoribonucleases producing 5\u2032-"]},{"entry":[{},"phosphomonoesters."]},{"entry":["3.1.27.\u2014","Endoribonucleases producing other"]},{"entry":[{},"than 5\u2032-phosphomonoesters."]},{"entry":["3.1.30.\u2014","Endoribonucleases active with either"]},{"entry":[{},"ribo- or deoxyribonucleic and producing 5\u2032-"]},{"entry":[{},"phosphomonoesters"]},{"entry":["3.1.31.\u2014","Endoribonucleases active with either"]},{"entry":[{},"ribo- or deoxyribonucleic and producing 3\u2032-"]},{"entry":[{},"phosphomonoesters"]},{"entry":["3.2.\u2014.\u2014","Glycosylases."]},{"entry":["3.2.1.\u2014","Glycosidases, i.e. enzymes hydrolyzing"]},{"entry":[{},"O- and S-glycosyl compounds"]},{"entry":["3.2.2.\u2014","Hydrolyzing N-glycosyl compounds."]},{"entry":["3.3.\u2014.\u2014","Acting on ether bonds."]},{"entry":["3.3.1.\u2014","Thioether and trialkylsulfonium"]},{"entry":[{},"hydrolases."]},{"entry":["3.3.2.\u2014","Ether hydrolases."]},{"entry":["3.4.\u2014.\u2014","Acting on peptide bonds (peptide"]},{"entry":[{},"hydrolases)."]},{"entry":["3.4.11.\u2014","Aminopeptidases."]},{"entry":["3.4.13.\u2014","Dipeptidases."]},{"entry":["3.4.14.\u2014","Dipeptidyl-peptidases and tripeptidyl-"]},{"entry":[{},"peptidases."]},{"entry":["3.4.15.\u2014","Peptidyl-dipeptidases."]},{"entry":["3.4.16.\u2014","Serine-type carboxypeptidases."]},{"entry":["3.4.17.\u2014","Metallocarboxypeptidases."]},{"entry":["3.4.18.\u2014","Cysteine-type carboxypeptidases."]},{"entry":["3.4.19.\u2014","Omega peptidases."]},{"entry":["3.4.21.\u2014","Serine endopeptidases."]},{"entry":["3.4.22.\u2014","Cysteine endopeptidases."]},{"entry":["3.4.23.\u2014","Aspartic endopeptidases."]},{"entry":["3.4.24.\u2014","Metalloendopeptidases."]},{"entry":["3.4.25.\u2014","Threonine endopeptidases."]},{"entry":["3.4.99.\u2014","Endopeptidases of unknown catalytic"]},{"entry":[{},"mechanism."]},{"entry":["3.5.\u2014.\u2014","Acting on carbon-nitrogen bonds, other"]},{"entry":[{},"than peptide bonds."]},{"entry":["3.5.1.\u2014","In linear amides."]},{"entry":["3.5.2.\u2014","In cyclic amides."]},{"entry":["3.5.3.\u2014","In linear amidines."]},{"entry":["3.5.4.\u2014","In cyclic amidines."]},{"entry":["3.5.5.\u2014","In nitriles."]},{"entry":["3.5.99.\u2014","In other compounds."]},{"entry":["3.6.\u2014.\u2014","Acting on acid anhydrides."]},{"entry":["3.6.1.\u2014","In phosphorous-containing anhydrides."]},{"entry":["3.6.2.\u2014","In sulfonyl-containing anhydrides."]},{"entry":["3.6.3.\u2014","Acting on acid anhydrides; catalyzing"]},{"entry":[{},"transmembrane movement of substances"]},{"entry":["3.6.4.\u2014","Acting on acid anhydrides; involved in"]},{"entry":[{},"cellular and subcellular movement"]},{"entry":["3.6.5.\u2014","Acting on GTP; involved in cellular and"]},{"entry":[{},"subcellular movement."]},{"entry":["3.7.\u2014.\u2014","Acting on carbon-carbon bonds."]},{"entry":["3.7.1.\u2014","In ketonic substances."]},{"entry":["3.8.\u2014.\u2014","Acting on halide bonds."]},{"entry":["3.8.1.\u2014","In C-halide compounds."]},{"entry":["3.9.\u2014.\u2014","Acting on phosphorus-nitrogen bonds."]},{"entry":["3.10.\u2014.\u2014","Acting on sulfur-nitrogen bonds."]},{"entry":["3.11.\u2014.\u2014","Acting on carbon-phosphorus bonds."]},{"entry":["3.12.\u2014.\u2014","Acting on sulfur-sulfur bonds."]},{"entry":["3.13.\u2014.\u2014","Acting on carbon-sulfur bonds."]},{"entry":["4.\u2014.\u2014.\u2014","Lyases."]},{"entry":["4.1.\u2014.\u2014","Carbon-carbon lyases."]},{"entry":["4.1.1.\u2014","Carboxy-lyases."]},{"entry":["4.1.2.\u2014","Aldehyde-lyases."]},{"entry":["4.1.3.\u2014","Oxo-acid-lyases."]},{"entry":["4.1.99.\u2014","Other carbon-carbon lyases."]},{"entry":["4.2.\u2014.\u2014","Carbon-oxygen lyases."]},{"entry":["4.2.1.\u2014","Hydro-lyases."]},{"entry":["4.2.2.\u2014","Acting on polysaccharides."]},{"entry":["4.2.3.\u2014","Acting on phosphates."]},{"entry":["4.2.99.\u2014","Other carbon-oxygen lyases."]},{"entry":["4.3.\u2014.\u2014","Carbon-nitrogen lyases."]},{"entry":["4.3.1.\u2014","Ammonia-lyases."]},{"entry":["4.3.2.\u2014","Lyases acting on amides, amidines, etc."]},{"entry":["4.3.3.\u2014","Amine-lyases."]},{"entry":["4.3.99.\u2014","Other carbon-nitrogen-lyases."]},{"entry":["4.4.\u2014.\u2014","Carbon-sulfur lyases."]},{"entry":["4.5.\u2014.\u2014","Carbon-halide lyases."]},{"entry":["4.6.\u2014.\u2014","Phosphorus-oxygen lyases."]},{"entry":["4.99.\u2014.\u2014","Other lyases."]},{"entry":["5.\u2014.\u2014.\u2014","Isomerases."]},{"entry":["5.1.\u2014.\u2014","Racemases and epimerases."]},{"entry":["5.1.1.\u2014","Acting on amino acids and derivatives."]},{"entry":["5.1.2.\u2014","Acting on hydroxy acids and derivatives."]},{"entry":["5.1.3.\u2014","Acting on carbohydrates and derivatives."]},{"entry":["5.1.99.\u2014","Acting on other compounds."]},{"entry":["5.2.\u2014.\u2014","Cis-trans-isomerases."]},{"entry":["5.3.\u2014.\u2014","Intramolecular oxidoreductases."]},{"entry":["5.3.1.\u2014","Interconverting aldoses and ketoses."]},{"entry":["5.3.2.\u2014","Interconverting keto- and enol-groups."]},{"entry":["5.3.3.\u2014","Transposing C\u2550C bonds."]},{"entry":["5.3.4.\u2014","Transposing S\u2014S bonds."]},{"entry":["5.3.99.\u2014","Other intramolecular oxidoreductases."]},{"entry":["5.4.\u2014.\u2014","Intramolecular transferases (mutases)."]},{"entry":["5.4.1.\u2014","Transferring acyl groups."]},{"entry":["5.4.2.\u2014","Phosphotransferases (phosphomutases)."]},{"entry":["5.4.3.\u2014","Transferring amino groups."]},{"entry":["5.4.4.\u2014","Transferring hydroxy groups."]},{"entry":["5.4.99.\u2014","Transferring other groups."]},{"entry":["5.5.\u2014.\u2014","Intramolecular lyases."]},{"entry":["5.99.\u2014.\u2014","Other isomerases."]},{"entry":["6.\u2014.\u2014.\u2014","Ligases."]},{"entry":["6.1.\u2014.\u2014","Forming carbon-oxygen bonds."]},{"entry":["6.1.1.\u2014","Ligases forming aminoacyl-tRNA and"]},{"entry":[{},"related compounds."]},{"entry":["6.2.\u2014.\u2014","Forming carbon-sulfur bonds."]},{"entry":["6.2.1.\u2014","Acid--thiol ligases."]},{"entry":["6.3.\u2014.\u2014","Forming carbon-nitrogen bonds."]},{"entry":["6.3.1.\u2014","Acid--ammonia (or amide) ligases (amide"]},{"entry":[{},"synthases)."]},{"entry":["6.3.2.\u2014","Acid--D-amino-acid ligases (peptide"]},{"entry":[{},"synthases)."]},{"entry":["6.3.3.\u2014","Cyclo-ligases."]},{"entry":["6.3.4.\u2014","Other carbon--nitrogen ligases."]},{"entry":["6.3.5.\u2014","Carbon--nitrogen ligases with glutamine as"]},{"entry":[{},"amido-N-donor."]},{"entry":["6.4.\u2014.\u2014","Forming carbon-carbon bonds."]},{"entry":["6.5.\u2014.\u2014","Forming phosphoric ester bonds."]},{"entry":["6.6.\u2014.\u2014","Forming nitrogen-metal bonds."]},{"entry":["6.6.1.\u2014","Forming nitrogen-metal bonds."]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0243","num":"0242"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 2"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"EC Numbers with the corresponding name given to each enzyme"},{"entry":"class, subclass and sub-subclass."},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"ENZYME: 1.\u2014.\u2014.\u2014"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"168pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"1.1.1.1","Alcohol dehydrogenase."]},{"entry":[{},"1.1.1.2","Alcohol dehydrogenase (NADP+)."]},{"entry":[{},"1.1.1.3","Homoserine dehydrogenase."]},{"entry":[{},"1.1.1.4","(R,R)-butanediol dehydrogenase."]},{"entry":[{},"1.1.1.5","Acetoin dehydrogenase."]},{"entry":[{},"1.1.1.6","Glycerol dehydrogenase."]},{"entry":[{},"1.1.1.7","Propanediol-phosphate dehydrogenase."]},{"entry":[{},"1.1.1.8","Glycerol-3-phosphate dehydrogenase"]},{"entry":[{},{},"(NAD+)."]},{"entry":[{},"1.1.1.9","D-xylulose reductase."]},{"entry":[{},"1.1.1.10","L-xylulose reductase."]},{"entry":[{},"1.1.1.11","D-arabinitol 4-dehydrogenase."]},{"entry":[{},"1.1.1.12","L-arabinitol 4-dehydrogenase."]},{"entry":[{},"1.1.1.13","L-arabinitol 2-dehydrogenase."]},{"entry":[{},"1.1.1.14","L-iditol 2-dehydrogenase."]},{"entry":[{},"1.1.1.15","D-iditol 2-dehydrogenase."]},{"entry":[{},"1.1.1.16","Galactitol 2-dehydrogenase."]},{"entry":[{},"1.1.1.17","Mannitol-1-phosphate 5-"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.18","Inositol 2-dehydrogenase."]},{"entry":[{},"1.1.1.19","L-glucuronate reductase."]},{"entry":[{},"1.1.1.20","Glucuronolactone reductase."]},{"entry":[{},"1.1.1.21","Aldehyde reductase."]},{"entry":[{},"1.1.1.22","UDP-glucose 6-dehydrogenase."]},{"entry":[{},"1.1.1.23","Histidinol dehydrogenase."]},{"entry":[{},"1.1.1.24","Quinate dehydrogenase."]},{"entry":[{},"1.1.1.25","Shikimate dehydrogenase."]},{"entry":[{},"1.1.1.26","Glyoxylate reductase."]},{"entry":[{},"1.1.1.27","L-lactate dehydrogenase."]},{"entry":[{},"1.1.1.28","D-lactate dehydrogenase."]},{"entry":[{},"1.1.1.29","Glycerate dehydrogenase."]},{"entry":[{},"1.1.1.30","3-hydroxybutyrate dehydrogenase."]},{"entry":[{},"1.1.1.31","3-hydroxyisobutyrate dehydrogenase."]},{"entry":[{},"1.1.1.32","Mevaldate reductase."]},{"entry":[{},"1.1.1.33","Mevaldate reductase (NADPH)."]},{"entry":[{},"1.1.1.34","Hydroxymethylglutaryl-CoA reductase"]},{"entry":[{},{},"(NADPH)."]},{"entry":[{},"1.1.1.35","3-hydroxyacyl-CoA dehydrogenase."]},{"entry":[{},"1.1.1.36","Acetoacetyl-CoA reductase."]},{"entry":[{},"1.1.1.37","Malate dehydrogenase."]},{"entry":[{},"1.1.1.38","Malate dehydrogenase (oxaloacetate-"]},{"entry":[{},{},"decarboxylating)."]},{"entry":[{},"1.1.1.39","Malate dehydrogenase"]},{"entry":[{},{},"(decarboxylating)."]},{"entry":[{},"1.1.1.40","Malate dehydrogenase (oxaloacetate-"]},{"entry":[{},{},"decarboxylating) (NADP+)."]},{"entry":[{},"1.1.1.41","Isocitrate dehydrogenase (NAD+)."]},{"entry":[{},"1.1.1.42","Isocitrate dehydrogenase (NADP+)."]},{"entry":[{},"1.1.1.43","Phosphogluconate 2-dehydrogenase."]},{"entry":[{},"1.1.1.44","Phosphogluconate dehydrogenase"]},{"entry":[{},{},"(decarboxylating)."]},{"entry":[{},"1.1.1.45","L-gulonate 3-dehydrogenase."]},{"entry":[{},"1.1.1.46","L-arabinose 1-dehydrogenase."]},{"entry":[{},"1.1.1.47","Glucose 1-dehydrogenase."]},{"entry":[{},"1.1.1.48","Galactose 1-dehydrogenase."]},{"entry":[{},"1.1.1.49","Glucose-6-phosphate 1-dehydrogenase."]},{"entry":[{},"1.1.1.50","3-alpha-hydroxysteroid dehydrogenase (B-"]},{"entry":[{},{},"specific)."]},{"entry":[{},"1.1.1.51","3(or 17)beta-hydroxysteroid dehydrogenase."]},{"entry":[{},"1.1.1.52","3-alpha-hydroxycholanate dehydrogenase."]},{"entry":[{},"1.1.1.53","3-alpha(or 20-beta)-hydroxysteroid"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.54","Allyl-alcohol dehydrogenase."]},{"entry":[{},"1.1.1.55","L-acetaldehyde reductase (NADPH)."]},{"entry":[{},"1.1.1.56","Ribitol 2-dehydrogenase."]},{"entry":[{},"1.1.1.57","Fructuronate reductase."]},{"entry":[{},"1.1.1.58","Tagaturonate reductase."]},{"entry":[{},"1.1.1.59","3-hydroxypropionate dehydrogenase."]},{"entry":[{},"1.1.1.60","2-hydroxy-3-oxopropionate reductase."]},{"entry":[{},"1.1.1.61","4-hydroxybutyrate dehydrogenase."]},{"entry":[{},"1.1.1.62","Estradiol 17-beta-dehydrogenase."]},{"entry":[{},"1.1.1.63","Testosterone 17-beta-dehydrogenase."]},{"entry":[{},"1.1.1.64","Testosterone 17-beta-dehydrogenase"]},{"entry":[{},{},"(NADP+)."]},{"entry":[{},"1.1.1.65","Pyridoxine 4-dehydrogenase."]},{"entry":[{},"1.1.1.66","Omega-hydroxydecanoate dehydrogenase."]},{"entry":[{},"1.1.1.67","Mannitol 2-dehydrogenase."]},{"entry":[{},"1.1.1.69","Gluconate 5-dehydrogenase."]},{"entry":[{},"1.1.1.71","Alcohol dehydrogenase (NAD(P)+)."]},{"entry":[{},"1.1.1.72","Glycerol dehydrogenase (NADP+)."]},{"entry":[{},"1.1.1.73","Octanol dehydrogenase."]},{"entry":[{},"1.1.1.75","(R)-aminopropanol dehydrogenase."]},{"entry":[{},"1.1.1.76","(S,S)-butanediol dehydrogenase."]},{"entry":[{},"1.1.1.77","Lactaldehyde reductase."]},{"entry":[{},"1.1.1.78","D-lactaldehyde dehydrogenase."]},{"entry":[{},"1.1.1.79","Glyoxylate reductase (NADP+)."]},{"entry":[{},"1.1.1.80","Isopropanol dehydrogenase (NADP+)."]},{"entry":[{},"1.1.1.81","Hydroxypyruvate reductase."]},{"entry":[{},"1.1.1.82","Malate dehydrogenase (NADP+)."]},{"entry":[{},"1.1.1.83","D-malate dehydrogenase (decarboxylating)."]},{"entry":[{},"1.1.1.84","Dimethylmalate dehydrogenase."]},{"entry":[{},"1.1.1.85","3-isopropylmalate dehydrogenase."]},{"entry":[{},"1.1.1.86","Ketol-acid reductoisomerase."]},{"entry":[{},"1.1.1.87","Homoisocitrate dehydrogenase."]},{"entry":[{},"1.1.1.88","Hydroxymethylglutaryl-CoA reductase."]},{"entry":[{},"1.1.1.90","Aryl-alcohol dehydrogenase."]},{"entry":[{},"1.1.1.91","Aryl-alcohol dehydrogenase (NADP+)."]},{"entry":[{},"1.1.1.92","Oxaloglycolate reductase"]},{"entry":[{},{},"(decarboxylating)."]},{"entry":[{},"1.1.1.93","Tartrate dehydrogenase."]},{"entry":[{},"1.1.1.94","Glycerol-3-phosphate dehydrogenase"]},{"entry":[{},{},"(NAD(P)+)."]},{"entry":[{},"1.1.1.95","Phosphoglycerate dehydrogenase."]},{"entry":[{},"1.1.1.96","Diiodophenylpyruvate reductase."]},{"entry":[{},"1.1.1.97","3-hydroxybenzyl-alcohol dehydrogenase."]},{"entry":[{},"1.1.1.98","(R)-2-hydroxy-fatty-acid dehydrogenase."]},{"entry":[{},"1.1.1.99","(S)-2-hydroxy-fatty-acid dehydrogenase."]},{"entry":[{},"1.1.1.100","3-oxoacyl-[acyl-carrier-protein]"]},{"entry":[{},{},"reductase."]},{"entry":[{},"1.1.1.101","Acylglycerone-phosphate reductase."]},{"entry":[{},"1.1.1.102","3-dehydrosphinganine reductase."]},{"entry":[{},"1.1.1.103","L-threonine 3-dehydrogenase."]},{"entry":[{},"1.1.1.104","4-oxoproline reductase."]},{"entry":[{},"1.1.1.105","Retinol dehydrogenase."]},{"entry":[{},"1.1.1.106","Pantoate 4-dehydrogenase."]},{"entry":[{},"1.1.1.107","Pyridoxal 4-dehydrogenase."]},{"entry":[{},"1.1.1.108","Carnitine 3-dehydrogenase."]},{"entry":[{},"1.1.1.110","Indolelactate dehydrogenase."]},{"entry":[{},"1.1.1.111","3-(imidazol-5-yl)lactate dehydrogenase."]},{"entry":[{},"1.1.1.112","Indanol dehydrogenase."]},{"entry":[{},"1.1.1.113","L-xylose 1-dehydrogenase."]},{"entry":[{},"1.1.1.114","Apiose 1-reductase."]},{"entry":[{},"1.1.1.115","Ribose 1-dehydrogenase (NADP+)."]},{"entry":[{},"1.1.1.116","D-arabinose 1-dehydrogenase."]},{"entry":[{},"1.1.1.117","D-arabinose 1-dehydrogenase"]},{"entry":[{},{},"(NAD(P)+)."]},{"entry":[{},"1.1.1.118","Glucose 1-dehydrogenase (NAD+)."]},{"entry":[{},"1.1.1.119","Glucose 1-dehydrogenase (NADP+)."]},{"entry":[{},"1.1.1.120","Galactose 1-dehydrogenase (NADP+)."]},{"entry":[{},"1.1.1.121","Aldose 1-dehydrogenase."]},{"entry":[{},"1.1.1.122","D-threo-aldose 1-dehydrogenase."]},{"entry":[{},"1.1.1.123","Sorbose 5-dehydrogenase (NADP+)."]},{"entry":[{},"1.1.1.124","Fructose 5-dehydrogenase (NADP+)."]},{"entry":[{},"1.1.1.125","2-deoxy-D-gluconate 3-dehydrogenase."]},{"entry":[{},"1.1.1.126","2-dehydro-3-deoxy-D-gluconate 6-"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.127","2-dehydro-3-deoxy-D-gluconate 5-"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.128","L-idonate 2-dehydrogenase."]},{"entry":[{},"1.1.1.129","L-threonate 3-dehydrogenase."]},{"entry":[{},"1.1.1.130","3-dehydro-L-gulonate 2-dehydrogenase."]},{"entry":[{},"1.1.1.131","Mannuronate reductase."]},{"entry":[{},"1.1.1.132","GDP-mannose 6-dehydrogenase."]},{"entry":[{},"1.1.1.133","dTDP-4-dehydrorhamnose reductase."]},{"entry":[{},"1.1.1.134","dTDP-6-deoxy-L-talose 4-"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.135","GDP-6-deoxy-D-talose 4-dehydrogenase."]},{"entry":[{},"1.1.1.136","UDP-N-acetylglucosamine 6-"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.137","Ribitol-5-phosphate 2-dehydrogenase."]},{"entry":[{},"1.1.1.138","Mannitol 2-dehydrogenase (NADP+)."]},{"entry":[{},"1.1.1.140","Sorbitol-6-phosphate 2-dehydrogenase."]},{"entry":[{},"1.1.1.141","15-hydroxyprostaglandin dehydrogenase"]},{"entry":[{},{},"(NAD+)."]},{"entry":[{},"1.1.1.142","D-pinitol dehydrogenase."]},{"entry":[{},"1.1.1.143","Sequoyitol dehydrogenase."]},{"entry":[{},"1.1.1.144","Perillyl-alcohol dehydrogenase."]},{"entry":[{},"1.1.1.145","3-beta-hydroxy-delta(5)-steroid"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.146","11-beta-hydroxysteroid dehydrogenase."]},{"entry":[{},"1.1.1.147","16-alpha-hydroxysteroid dehydrogenase."]},{"entry":[{},"1.1.1.148","Estradiol 17-alpha-dehydrogenase."]},{"entry":[{},"1.1.1.149","20-alpha-hydroxysteroid dehydrogenase."]},{"entry":[{},"1.1.1.150","21-hydroxysteroid dehydrogenase"]},{"entry":[{},{},"(NAD+)."]},{"entry":[{},"1.1.1.152","3-alpha-hydroxy-5-beta-androstane-17-"]},{"entry":[{},{},"one 3-alpha-dehydrogenase."]},{"entry":[{},"1.1.1.153","Sepiapterin reductase."]},{"entry":[{},"1.1.1.154","Ureidoglycolate dehydrogenase."]},{"entry":[{},"1.1.1.155","Homoisocitrate dehydrogenase."]},{"entry":[{},"1.1.1.156","Glycerol 2-dehydrogenase (NADP+)."]},{"entry":[{},"1.1.1.157","3-hydroxybutyryl-CoA dehydrogenase."]},{"entry":[{},"1.1.1.158","UDP-N-acetylmuramate dehydrogenase."]},{"entry":[{},"1.1.1.159","7-alpha-hydroxysteroid dehydrogenase."]},{"entry":[{},"1.1.1.160","Dihydrobunolol dehydrogenase."]},{"entry":[{},"1.1.1.161","Cholestanetetraol 26-dehydrogenase."]},{"entry":[{},"1.1.1.162","Erythrulose reductase."]},{"entry":[{},"1.1.1.163","Cyclopentanol dehydrogenase."]},{"entry":[{},"1.1.1.164","Hexadecanol dehydrogenase."]},{"entry":[{},"1.1.1.165","2-alkyn-1-ol dehydrogenase."]},{"entry":[{},"1.1.1.166","Hydroxycyclohexanecarboxylate"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.167","Hydroxymalonate dehydrogenase."]},{"entry":[{},"1.1.1.168","2-dehydropantolactone reductase (A-"]},{"entry":[{},{},"specific)."]},{"entry":[{},"1.1.1.169","2-dehydropantoate 2-reductase."]},{"entry":[{},"1.1.1.170","Sterol-4-alpha-carboxylate 3-"]},{"entry":[{},{},"dehydrogenase (decarboxylating)."]},{"entry":[{},"1.1.1.172","2-oxoadipate reductase."]},{"entry":[{},"1.1.1.173","L-rhamnose 1-dehydrogenase."]},{"entry":[{},"1.1.1.174","Cyclohexane-1,2-diol dehydrogenase."]},{"entry":[{},"1.1.1.175","D-xylose 1-dehydrogenase."]},{"entry":[{},"1.1.1.176","12-alpha-hydroxysteroid"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.177","Glycerol-3-phosphate 1-dehydrogenase"]},{"entry":[{},{},"(NADP+)."]},{"entry":[{},"1.1.1.178","3-hydroxy-2-methylbutyryl-CoA"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.179","D-xylose 1-dehydrogenase (NADP+)."]},{"entry":[{},"1.1.1.181","Cholest-5-ene-3-beta,7-alpha-diol 3-"]},{"entry":[{},{},"beta-dehydrogenase."]},{"entry":[{},"1.1.1.183","Geraniol dehydrogenase."]},{"entry":[{},"1.1.1.184","Carbonyl reductase (NADPH)."]},{"entry":[{},"1.1.1.185","L-glycol dehydrogenase."]},{"entry":[{},"1.1.1.186","dTDP-galactose 6-dehydrogenase."]},{"entry":[{},"1.1.1.187","GDP-4-dehydro-D-rhamnose reductase."]},{"entry":[{},"1.1.1.188","Prostaglandin-F synthase."]},{"entry":[{},"1.1.1.189","Prostaglandin-E(2) 9-reductase."]},{"entry":[{},"1.1.1.190","Indole-3-acetaldehyde reductase"]},{"entry":[{},{},"(NADH)."]},{"entry":[{},"1.1.1.191","Indole-3-acetaldehyde reductase"]},{"entry":[{},{},"(NADPH)."]},{"entry":[{},"1.1.1.192","Long-chain-alcohol dehydrogenase."]},{"entry":[{},"1.1.1.193","5-amino-6-(5-"]},{"entry":[{},{},"phosphoribosylamino)uracil reductase."]},{"entry":[{},"1.1.1.194","Coniferyl-alcohol dehydrogenase."]},{"entry":[{},"1.1.1.195","Cinnamyl-alcohol dehydrogenase."]},{"entry":[{},"1.1.1.196","15-hydroxyprostaglandin-D"]},{"entry":[{},{},"dehydrogenase (NADP+)."]},{"entry":[{},"1.1.1.197","15-hydroxyprostaglandin dehydrogenase"]},{"entry":[{},{},"(NADP+)."]},{"entry":[{},"1.1.1.198","(+)-borneol dehydrogenase."]},{"entry":[{},"1.1.1.199","(S)-usnate reductase."]},{"entry":[{},"1.1.1.200","Aldose-6-phosphate reductase"]},{"entry":[{},{},"(NADPH)."]},{"entry":[{},"1.1.1.201","7-beta-hydroxysteroid dehydrogenase"]},{"entry":[{},{},"(NADP+)."]},{"entry":[{},"1.1.1.202","1,3-propanediol dehydrogenase."]},{"entry":[{},"1.1.1.203","Uronate dehydrogenase."]},{"entry":[{},"1.1.1.205","IMP dehydrogenase."]},{"entry":[{},"1.1.1.206","Tropine dehydrogenase."]},{"entry":[{},"1.1.1.207","(\u2212)-menthol dehydrogenase."]},{"entry":[{},"1.1.1.208","(+)-neomenthol dehydrogenase."]},{"entry":[{},"1.1.1.209","3(or 17)-alpha-hydroxysteroid"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.210","3-beta(or 20-alpha)-hydroxysteroid"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.211","Long-chain-3-hydroxyacyl-CoA"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.212","3-oxoacyl-[acyl-carrier-protein]"]},{"entry":[{},{},"reductase (NADH)."]},{"entry":[{},"1.1.1.213","3-alpha-hydroxysteroid dehydrogenase"]},{"entry":[{},{},"(A-specific)."]},{"entry":[{},"1.1.1.214","2-dehydropantolactone reductase (B-"]},{"entry":[{},{},"specific)."]},{"entry":[{},"1.1.1.215","Gluconate 2-dehydrogenase."]},{"entry":[{},"1.1.1.216","Farnesol dehydrogenase."]},{"entry":[{},"1.1.1.217","Benzyl-2-methyl-hydroxybutyrate"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.218","Morphine 6-dehydrogenase."]},{"entry":[{},"1.1.1.219","Dihydrokaempferol 4-reductase."]},{"entry":[{},"1.1.1.220","6-pyruvoyltetrahydropterin 2\u2032-reductase."]},{"entry":[{},"1.1.1.221","Vomifoliol 4\u2032-dehydrogenase."]},{"entry":[{},"1.1.1.222","(R)-4-hydroxyphenyllactate"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.223","Isopiperitenol dehydrogenase."]},{"entry":[{},"1.1.1.224","Mannose-6-phosphate 6-reductase."]},{"entry":[{},"1.1.1.225","Chlordecone reductase."]},{"entry":[{},"1.1.1.226","4-hydroxycyclohexanecarboxylate"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.227","(\u2212)-borneol dehydrogenase."]},{"entry":[{},"1.1.1.228","(+)-sabinol dehydrogenase."]},{"entry":[{},"1.1.1.229","Diethyl 2-methyl-3-oxosuccinate"]},{"entry":[{},{},"reductase."]},{"entry":[{},"1.1.1.230","3-alpha-hydroxyglycyrrhetinate"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.231","15-hydroxyprostaglandin-I"]},{"entry":[{},{},"dehydrogenase (NADP+)."]},{"entry":[{},"1.1.1.232","15-hydroxyicosatetraenoate"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.233","N-acylmannosamine 1-dehydrogenase."]},{"entry":[{},"1.1.1.234","Flavanone 4-reductase."]},{"entry":[{},"1.1.1.235","8-oxocoformycin reductase."]},{"entry":[{},"1.1.1.236","Tropinone reductase."]},{"entry":[{},"1.1.1.237","Hydroxyphenylpyruvate reductase."]},{"entry":[{},"1.1.1.238","12-beta-hydroxysteroid dehydrogenase."]},{"entry":[{},"1.1.1.239","3-alpha-(17-beta)-hydroxysteroid"]},{"entry":[{},{},"dehydrogenase (NAD+)."]},{"entry":[{},"1.1.1.240","N-acetylhexosamine 1-dehydrogenase."]},{"entry":[{},"1.1.1.241","6-endo-hydroxycineole dehydrogenase."]},{"entry":[{},"1.1.1.243","Carveol dehydrogenase."]},{"entry":[{},"1.1.1.244","Methanol dehydrogenase."]},{"entry":[{},"1.1.1.245","Cyclohexanol dehydrogenase."]},{"entry":[{},"1.1.1.246","Pterocarpin synthase."]},{"entry":[{},"1.1.1.247","Codeinone reductase (NADPH)."]},{"entry":[{},"1.1.1.248","Salutaridine reductase (NADPH)."]},{"entry":[{},"1.1.1.250","D-arabinitol 2-dehydrogenase."]},{"entry":[{},"1.1.1.251","Galactitol-1-phosphate 5-"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.252","Tetrahydroxynaphthalene reductase."]},{"entry":[{},"1.1.1.254","(S)-carnitine 3-dehydrogenase."]},{"entry":[{},"1.1.1.255","Mannitol dehydrogenase."]},{"entry":[{},"1.1.1.256","Fluoren-9-ol dehydrogenase."]},{"entry":[{},"1.1.1.257","4-(hydroxymethyl)benzenesulfonate"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.258","6-hydroxyhexanoate dehydrogenase."]},{"entry":[{},"1.1.1.259","3-hydroxypimeloyl-CoA"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.260","Sulcatone reductase."]},{"entry":[{},"1.1.1.261","Glycerol-1-phosphate dehydrogenase"]},{"entry":[{},{},"(NAD(P)+)."]},{"entry":[{},"1.1.1.262","4-hydroxythreonine-4-phosphate"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.263","1,5-anhydro-D-fructose reductase."]},{"entry":[{},"1.1.1.264","L-idonate 5-dehydrogenase."]},{"entry":[{},"1.1.1.265","3-methylbutanal reductase."]},{"entry":[{},"1.1.1.266","dTDP-4-dehydro-6-deoxyglucose"]},{"entry":[{},{},"reductase."]},{"entry":[{},"1.1.1.267","1-deoxy-D-xylulose-5-phosphate"]},{"entry":[{},{},"reductoisomerase."]},{"entry":[{},"1.1.1.268","2-(R)-hydroxypropyl-CoM"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.269","2-(S)-hydroxypropyl-CoM"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.270","3-keto-steroid reductase."]},{"entry":[{},"1.1.1.271","GDP-L-fucose synthase."]},{"entry":[{},"1.1.1.272","(R)-2-hydroxyacid dehydrogenase."]},{"entry":[{},"1.1.1.273","Vellosimine dehydrogenase."]},{"entry":[{},"1.1.1.274","2,5-didehydrogluconate reductase."]},{"entry":[{},"1.1.1.275","(+)-trans-carveol dehydrogenase."]},{"entry":[{},"1.1.1.276","Serine 3-dehydrogenase."]},{"entry":[{},"1.1.1.277","3-beta-hydroxy-5-beta-steroid"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.278","3-beta-hydroxy-5-alpha-steroid"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.1.279","(R)-3-hydroxyacid-ester dehydrogenase."]},{"entry":[{},"1.1.1.280","(S)-3-hydroxyacid-ester dehydrogenase."]},{"entry":[{},"1.1.1.281","GDP-4-dehydro-6-deoxy-D-mannose"]},{"entry":[{},{},"reductase."]},{"entry":[{},"1.1.1.282","Quinate\/shikimate dehydrogenase."]},{"entry":[{},"1.1.2.2","Mannitol dehydrogenase (cytochrome)."]},{"entry":[{},"1.1.2.3","L-lactate dehydrogenase (cytochrome)."]},{"entry":[{},"1.1.2.4","D-lactate dehydrogenase (cytochrome)."]},{"entry":[{},"1.1.2.5","D-lactate dehydrogenase (cytochrome c-"]},{"entry":[{},{},"553)."]},{"entry":[{},"1.1.3.3","Malate oxidase."]},{"entry":[{},"1.1.3.4","Glucose oxidase."]},{"entry":[{},"1.1.3.5","Hexose oxidase."]},{"entry":[{},"1.1.3.6","Cholesterol oxidase."]},{"entry":[{},"1.1.3.7","Aryl-alcohol oxidase."]},{"entry":[{},"1.1.3.8","L-gulonolactone oxidase."]},{"entry":[{},"1.1.3.9","Galactose oxidase."]},{"entry":[{},"1.1.3.10","Pyranose oxidase."]},{"entry":[{},"1.1.3.11","L-sorbose oxidase."]},{"entry":[{},"1.1.3.12","Pyridoxine 4-oxidase."]},{"entry":[{},"1.1.3.13","Alcohol oxidase."]},{"entry":[{},"1.1.3.14","Catechol oxidase (dimerizing)."]},{"entry":[{},"1.1.3.15","(S)-2-hydroxy-acid oxidase."]},{"entry":[{},"1.1.3.16","Ecdysone oxidase."]},{"entry":[{},"1.1.3.17","Choline oxidase."]},{"entry":[{},"1.1.3.18","Secondary-alcohol oxidase."]},{"entry":[{},"1.1.3.19","4-hydroxymandelate oxidase."]},{"entry":[{},"1.1.3.20","Long-chain-alcohol oxidase."]},{"entry":[{},"1.1.3.21","Glycerol-3-phosphate oxidase."]},{"entry":[{},"1.1.3.23","Thiamine oxidase."]},{"entry":[{},"1.1.3.24","L-galactonolactone oxidase."]},{"entry":[{},"1.1.3.25","Cellobiose oxidase."]},{"entry":[{},"1.1.3.27","Hydroxyphytanate oxidase."]},{"entry":[{},"1.1.3.28","Nucleoside oxidase."]},{"entry":[{},"1.1.3.29","N-acylhexosamine oxidase."]},{"entry":[{},"1.1.3.30","Polyvinyl-alcohol oxidase."]},{"entry":[{},"1.1.3.37","D-arabinono-1,4-lactone oxidase."]},{"entry":[{},"1.1.3.38","Vanillyl-alcohol oxidase."]},{"entry":[{},"1.1.3.39","Nucleoside oxidase (H(2)O(2)-forming)"]},{"entry":[{},"1.1.3.40","D-mannitol oxidase."]},{"entry":[{},"1.1.3.41","Xylitol oxidase."]},{"entry":[{},"1.1.4.1","Vitamin-K-epoxide reductase"]},{"entry":[{},{},"(warfarin-sensitive)."]},{"entry":[{},"1.1.4.2","Vitamin-K-epoxide reductase"]},{"entry":[{},{},"(warfarin-insensitive)."]},{"entry":[{},"1.1.5.2","Quinoprotein glucose dehydrogenase."]},{"entry":[{},"1.1.99.1","Choline dehydrogenase."]},{"entry":[{},"1.1.99.2","2-hydroxyglutarate dehydrogenase."]},{"entry":[{},"1.1.99.3","Gluconate 2-dehydrogenase"]},{"entry":[{},{},"(acceptor)."]},{"entry":[{},"1.1.99.4","Dehydrogluconate dehydrogenase."]},{"entry":[{},"1.1.99.5","Glycerol-3-phosphate dehydrogenase."]},{"entry":[{},"1.1.99.6","D-2-hydroxy-acid dehydrogenase."]},{"entry":[{},"1.1.99.7","Lactate--malate transhydrogenase."]},{"entry":[{},"1.1.99.8","Alcohol dehydrogenase (acceptor)."]},{"entry":[{},"1.1.99.9","Pyridoxine 5-dehydrogenase."]},{"entry":[{},"1.1.99.10","Glucose dehydrogenase (acceptor)."]},{"entry":[{},"1.1.99.11","Fructose 5-dehydrogenase."]},{"entry":[{},"1.1.99.12","Sorbose dehydrogenase."]},{"entry":[{},"1.1.99.13","Glucoside 3-dehydrogenase."]},{"entry":[{},"1.1.99.14","Glycolate dehydrogenase."]},{"entry":[{},"1.1.99.16","Malate dehydrogenase (acceptor)."]},{"entry":[{},"1.1.99.18","Cellobiose dehydrogenase (acceptor)."]},{"entry":[{},"1.1.99.19","Uracil dehydrogenase."]},{"entry":[{},"1.1.99.20","Alkan-1-ol dehydrogenase (acceptor)."]},{"entry":[{},"1.1.99.21","D-sorbitol dehydrogenase (acceptor)."]},{"entry":[{},"1.1.99.22","Glycerol dehydrogenase (acceptor)."]},{"entry":[{},"1.1.99.23","Polyvinyl-alcohol dehydrogenase"]},{"entry":[{},{},"(acceptor)."]},{"entry":[{},"1.1.99.24","Hydroxyacid--oxoacid"]},{"entry":[{},{},"transhydrogenase."]},{"entry":[{},"1.1.99.25","Quinate dehydrogenase"]},{"entry":[{},{},"(pyrroloquinoline-quinone)."]},{"entry":[{},"1.1.99.26","3-hydroxycyclohexanone"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.1.99.27","(R)-pantolactone dehydrogenase"]},{"entry":[{},{},"(flavin)."]},{"entry":[{},"1.1.99.28","Glucose--fructose oxidoreductase."]},{"entry":[{},"1.1.99.29","Pyranose dehydrogenase (acceptor)."]},{"entry":[{},"1.1.99.30","2-oxo-acid reductase."]},{"entry":[{},"1.2.1.1","Formaldehyde dehydrogenase"]},{"entry":[{},{},"(glutathione)."]},{"entry":[{},"1.2.1.2","Formate dehydrogenase."]},{"entry":[{},"1.2.1.3","Aldehyde dehydrogenase (NAD+)."]},{"entry":[{},"1.2.1.4","Aldehyde dehydrogenase (NADP+)."]},{"entry":[{},"1.2.1.5","Aldehyde dehydrogenase (NAD(P)+)."]},{"entry":[{},"1.2.1.7","Benzaldehyde dehydrogenase"]},{"entry":[{},{},"(NADP+)."]},{"entry":[{},"1.2.1.8","Betaine-aldehyde dehydrogenase."]},{"entry":[{},"1.2.1.9","Glyceraldehyde-3-phosphate dehydrogenase"]},{"entry":[{},{},"(NADP+)."]},{"entry":[{},"1.2.1.10","Acetaldehyde dehydrogenase (acetylating)."]},{"entry":[{},"1.2.1.11","Aspartate-semialdehyde dehydrogenase."]},{"entry":[{},"1.2.1.12","Glyceraldehyde-3-phosphate dehydrogenase"]},{"entry":[{},{},"(phosphorylating)."]},{"entry":[{},"1.2.1.13","Glyceraldehyde-3-phosphate dehydrogenase"]},{"entry":[{},{},"(NADP(+)) (phosphorylating)."]},{"entry":[{},"1.2.1.15","Malonate-semialdehyde dehydrogenase."]},{"entry":[{},"1.2.1.16","Succinate-semialdehyde dehydrogenase"]},{"entry":[{},{},"(NAD(P)+)."]},{"entry":[{},"1.2.1.17","Glyoxylate dehydrogenase (acylating)."]},{"entry":[{},"1.2.1.18","Malonate-semialdehyde dehydrogenase"]},{"entry":[{},{},"(acetylating)."]},{"entry":[{},"1.2.1.19","Aminobutyraldehyde dehydrogenase."]},{"entry":[{},"1.2.1.20","Glutarate-semialdehyde dehydrogenase."]},{"entry":[{},"1.2.1.21","Glycolaldehyde dehydrogenase."]},{"entry":[{},"1.2.1.22","Lactaldehyde dehydrogenase."]},{"entry":[{},"1.2.1.23","2-oxoaldehyde dehydrogenase (NAD+)."]},{"entry":[{},"1.2.1.24","Succinate-semialdehyde dehydrogenase."]},{"entry":[{},"1.2.1.25","2-oxoisovalerate dehydrogenase (acylating)."]},{"entry":[{},"1.2.1.26","2,5-dioxovalerate dehydrogenase."]},{"entry":[{},"1.2.1.27","Methylmalonate-semialdehyde"]},{"entry":[{},{},"dehydrogenase (acylating)."]},{"entry":[{},"1.2.1.28","Benzaldehyde dehydrogenase (NAD+)."]},{"entry":[{},"1.2.1.29","Aryl-aldehyde dehydrogenase."]},{"entry":[{},"1.2.1.30","Aryl-aldehyde dehydrogenase (NADP+)."]},{"entry":[{},"1.2.1.31","L-aminoadipate-semialdehyde"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.2.1.32","Aminomuconate-semialdehyde"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.2.1.33","(R)-dehydropantoate dehydrogenase."]},{"entry":[{},"1.2.1.36","Retinal dehydrogenase."]},{"entry":[{},"1.2.1.38","N-acetyl-gamma-glutamyl-phosphate"]},{"entry":[{},{},"reductase."]},{"entry":[{},"1.2.1.39","Phenylacetaldehyde dehydrogenase."]},{"entry":[{},"1.2.1.40","3-alpha,7-alpha,12-alpha-"]},{"entry":[{},{},"trihydroxycholestan-26-al 26-oxidoreductase."]},{"entry":[{},"1.2.1.41","Glutamate-5-semialdehyde dehydrogenase."]},{"entry":[{},"1.2.1.42","Hexadecanal dehydrogenase (acylating)."]},{"entry":[{},"1.2.1.43","Formate dehydrogenase (NADP+)."]},{"entry":[{},"1.2.1.44","Cinnamoyl-CoA reductase."]},{"entry":[{},"1.2.1.45","4-carboxy-2-hydroxymuconate-6-"]},{"entry":[{},{},"semialdehyde dehydrogenase."]},{"entry":[{},"1.2.1.46","Formaldehyde dehydrogenase."]},{"entry":[{},"1.2.1.47","4-trimethylammoniobutyraldehyde"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.2.1.48","Long-chain-aldehyde dehydrogenase."]},{"entry":[{},"1.2.1.49","2-oxoaldehyde dehydrogenase"]},{"entry":[{},{},"(NADP+)."]},{"entry":[{},"1.2.1.50","Long-chain-fatty-acyl-CoA reductase."]},{"entry":[{},"1.2.1.51","Pyruvate dehydrogenase (NADP+)."]},{"entry":[{},"1.2.1.52","Oxoglutarate dehydrogenase"]},{"entry":[{},{},"(NADP+)."]},{"entry":[{},"1.2.1.53","4-hydroxyphenylacetaldehyde"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.2.1.54","Gamma-guanidinobutyraldehyde"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.2.1.57","Butanal dehydrogenase."]},{"entry":[{},"1.2.1.58","Phenylglyoxylate dehydrogenase"]},{"entry":[{},{},"(acylating)."]},{"entry":[{},"1.2.1.59","Glyceraldehyde-3-phosphate"]},{"entry":[{},{},"dehydrogenase (NAD(P)(+)) (phosphorylating)."]},{"entry":[{},"1.2.1.60","5-carboxymethyl-2-hydroxymuconic-"]},{"entry":[{},{},"semialdehyde dehydrogenase."]},{"entry":[{},"1.2.1.61","4-hydroxymuconic-semialdehyde"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.2.1.62","4-formylbenzenesulfonate"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.2.1.63","6-oxohexanoate dehydrogenase."]},{"entry":[{},"1.2.1.64","4-hydroxybenzaldehyde"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.2.1.65","Salicylaldehyde dehydrogenase."]},{"entry":[{},"1.2.1.66","Mycothiol-dependent formaldehyde"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.2.1.67","Vanillin dehydrogenase."]},{"entry":[{},"1.2.1.68","Coniferyl-aldehyde dehydrogenase."]},{"entry":[{},"1.2.1.69","Fluoroacetaldehyde dehydrogenase."]},{"entry":[{},"1.2.2.1","Formate dehydrogenase (cytochrome)."]},{"entry":[{},"1.2.2.2","Pyruvate dehydrogenase (cytochrome)."]},{"entry":[{},"1.2.2.3","Formate dehydrogenase (cytochrome c-"]},{"entry":[{},{},"553)."]},{"entry":[{},"1.2.2.4","Carbon-monoxide dehydrogenase"]},{"entry":[{},{},"(cytochrome b-561)."]},{"entry":[{},"1.2.3.1","Aldehyde oxidase."]},{"entry":[{},"1.2.3.3","Pyruvate oxidase."]},{"entry":[{},"1.2.3.4","Oxalate oxidase."]},{"entry":[{},"1.2.3.5","Glyoxylate oxidase."]},{"entry":[{},"1.2.3.6","Pyruvate oxidase (CoA-acetylating)."]},{"entry":[{},"1.2.3.7","Indole-3-acetaldehyde oxidase."]},{"entry":[{},"1.2.3.8","Pyridoxal oxidase."]},{"entry":[{},"1.2.3.9","Aryl-aldehyde oxidase."]},{"entry":[{},"1.2.3.11","Retinal oxidase."]},{"entry":[{},"1.2.3.13","4-hydroxyphenylpyruvate oxidase."]},{"entry":[{},"1.2.4.1","Pyruvate dehydrogenase (acetyl-"]},{"entry":[{},{},"transferring)."]},{"entry":[{},"1.2.4.2","Oxoglutarate dehydrogenase (succinyl-"]},{"entry":[{},{},"transferring)."]},{"entry":[{},"1.2.4.4","3-methyl-2-oxobutanoate dehydrogenase (2-"]},{"entry":[{},{},"methylpropanoyl-transferring)."]},{"entry":[{},"1.2.7.1","Pyruvate synthase."]},{"entry":[{},"1.2.7.2","2-oxobutyrate synthase."]},{"entry":[{},"1.2.7.3","2-oxoglutarate synthase."]},{"entry":[{},"1.2.7.4","Carbon-monoxide dehydrogenase"]},{"entry":[{},{},"(ferredoxin)."]},{"entry":[{},"1.2.7.5","Aldehyde ferredoxin oxidoreductase."]},{"entry":[{},"1.2.7.6","Glyceraldehyde-3-phosphate dehydrogenase"]},{"entry":[{},{},"(ferredoxin)."]},{"entry":[{},"1.2.7.7","3-methyl-2-oxobutanoate dehydrogenase"]},{"entry":[{},{},"(ferredoxin)."]},{"entry":[{},"1.2.7.8","Indolepyruvate ferredoxin oxidoreductase."]},{"entry":[{},"1.2.7.9","2-oxoglutarate ferredoxin oxidoreductase."]},{"entry":[{},"1.2.99.2","Carbon-monoxide dehydrogenase"]},{"entry":[{},{},"(acceptor)."]},{"entry":[{},"1.2.99.3","Aldehyde dehydrogenase (pyrroloquinoline-"]},{"entry":[{},{},"quinone)."]},{"entry":[{},"1.2.99.4","Formaldehyde dismutase."]},{"entry":[{},"1.2.99.5","Formylmethanofuran dehydrogenase."]},{"entry":[{},"1.2.99.6","Carboxylate reductase."]},{"entry":[{},"1.2.99.7","Aldehyde dehydrogenase (FAD-"]},{"entry":[{},{},"independent)."]},{"entry":[{},"1.3.1.1","Dihydrouracil dehydrogenase (NAD+)."]},{"entry":[{},"1.3.1.2","Dihydropyrimidine dehydrogenase"]},{"entry":[{},{},"(NADP+)."]},{"entry":[{},"1.3.1.3","Cortisone beta-reductase."]},{"entry":[{},"1.3.1.4","Cortisone alpha-reductase."]},{"entry":[{},"1.3.1.5","Cucurbitacin delta(23)-reductase."]},{"entry":[{},"1.3.1.6","Fumarate reductase (NADH)."]},{"entry":[{},"1.3.1.7","Meso-tartrate dehydrogenase."]},{"entry":[{},"1.3.1.8","Acyl-CoA dehydrogenase (NADP+)."]},{"entry":[{},"1.3.1.9","Enoyl-[acyl-carrier-protein] reductase"]},{"entry":[{},{},"(NADH)."]},{"entry":[{},"1.3.1.10","Enoyl-[acyl-carrier-protein] reductase"]},{"entry":[{},{},"(NADPH, B-specific)."]},{"entry":[{},"1.3.1.11","2-coumarate reductase."]},{"entry":[{},"1.3.1.12","Prephenate dehydrogenase."]},{"entry":[{},"1.3.1.13","Prephenate dehydrogenase (NADP+)."]},{"entry":[{},"1.3.1.14","Orotate reductase (NADH)."]},{"entry":[{},"1.3.1.15","Orotate reductase (NADPH)."]},{"entry":[{},"1.3.1.16","Beta-nitroacrylate reductase."]},{"entry":[{},"1.3.1.17","3-methyleneoxindole reductase."]},{"entry":[{},"1.3.1.18","Kynurenate-7,8-dihydrodiol dehydrogenase."]},{"entry":[{},"1.3.1.19","Cis-1,2-dihydrobenzene-1,2-diol"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.3.1.20","Trans-1,2-dihydrobenzene-1,2-diol"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.3.1.21","7-dehydrocholesterol reductase."]},{"entry":[{},"1.3.1.22","Cholestenone 5-alpha-reductase."]},{"entry":[{},"1.3.1.23","Cholestenone 5-beta-reductase."]},{"entry":[{},"1.3.1.24","Biliverdin reductase."]},{"entry":[{},"1.3.1.25","1,6-dihydroxycyclohexa-2,4-diene-1-"]},{"entry":[{},{},"carboxylate dehydrogenase."]},{"entry":[{},"1.3.1.26","Dihydrodipicolinate reductase."]},{"entry":[{},"1.3.1.27","2-hexadecenal reductase."]},{"entry":[{},"1.3.1.28","2,3-dihydro-2,3-dihydroxybenzoate"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.3.1.29","Cis-1,2-dihydro-1,2-dihydroxynaphthalene"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.3.1.30","Progesterone 5-alpha-reductase."]},{"entry":[{},"1.3.1.31","2-enoate reductase."]},{"entry":[{},"1.3.1.32","Maleylacetate reductase."]},{"entry":[{},"1.3.1.33","Protochlorophyllide reductase."]},{"entry":[{},"1.3.1.34","2,4-dienoyl-CoA reductase (NADPH)."]},{"entry":[{},"1.3.1.35","Phosphatidylcholine desaturase."]},{"entry":[{},"1.3.1.36","Geissoschizine dehydrogenase."]},{"entry":[{},"1.3.1.37","Cis-2-enoyl-CoA reductase (NADPH)."]},{"entry":[{},"1.3.1.38","Trans-2-enoyl-CoA reductase (NADPH)."]},{"entry":[{},"1.3.1.39","Enoyl-[acyl-carrier-protein] reductase"]},{"entry":[{},{},"(NADPH, A-specific)."]},{"entry":[{},"1.3.1.40","2-hydroxy-6-oxo-6-phenylhexa-2,4-"]},{"entry":[{},{},"dienoate reductase."]},{"entry":[{},"1.3.1.41","Xanthommatin reductase."]},{"entry":[{},"1.3.1.42","12-oxophytodienoate reductase."]},{"entry":[{},"1.3.1.43","Cyclohexadienyl dehydrogenase."]},{"entry":[{},"1.3.1.44","Trans-2-enoyl-CoA reductase (NAD+)."]},{"entry":[{},"1.3.1.45","2\u2032-hydroxyisoflavone reductase."]},{"entry":[{},"1.3.1.46","Biochanin-A reductase."]},{"entry":[{},"1.3.1.47","Alpha-santonin 1,2-reductase."]},{"entry":[{},"1.3.1.48","15-oxoprostaglandin 13-oxidase."]},{"entry":[{},"1.3.1.49","Cis-3,4-dihydrophenanthrene-3,4-diol"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.3.1.51","2\u2032-hydroxydaidzein reductase."]},{"entry":[{},"1.3.1.52","2-methyl-branched-chain-enoyl-CoA"]},{"entry":[{},{},"reductase."]},{"entry":[{},"1.3.1.53","(3S,4R)-3,4-dihydroxycyclohexa-1,5-"]},{"entry":[{},{},"diene-1,4-dicarboxylate dehydrogenase."]},{"entry":[{},"1.3.1.54","Precorrin-6A reductase."]},{"entry":[{},"1.3.1.56","Cis-2,3-dihydrobiphenyl-2,3-diol"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.3.1.57","Phloroglucinol reductase."]},{"entry":[{},"1.3.1.58","2,3-dihydroxy-2,3-dihydro-p-cumate"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.3.1.59","1,6-dihydroxy-5-methylcyclohexa-2,4-"]},{"entry":[{},{},"dienecarboxylate dehydrogenase."]},{"entry":[{},"1.3.1.60","Dibenzothiophene dihydrodiol"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.3.1.61","Terephthalate 1,2-cis-dihydrodiol"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.3.1.62","Pimeloyl-CoA dehydrogenase."]},{"entry":[{},"1.3.1.63","2,4-dichlorobenzoyl-CoA reductase."]},{"entry":[{},"1.3.1.64","Phthalate 4,5-cis-dihydrodiol"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.3.1.65","5,6-dihydroxy-3-methyl-2-oxo-1,2,5,6-"]},{"entry":[{},{},"tetrahydroquinoline dehydrogenase."]},{"entry":[{},"1.3.1.66","Cis-dihydroethylcatechol"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.3.1.67","Cis-1,2-dihydroxy-4-methylcyclohexa-"]},{"entry":[{},{},"3,5-diene-1-carboxylate dehydrogenase."]},{"entry":[{},"1.3.1.68","1,2-dihydroxy-6-methylcyclohexa-3,5-"]},{"entry":[{},{},"dienecarboxylate dehydrogenase."]},{"entry":[{},"1.3.1.69","Zeatin reductase."]},{"entry":[{},"1.3.1.70","Delta(14)-sterol reductase."]},{"entry":[{},"1.3.1.71","Delta(24(24(1)))-sterol reductase."]},{"entry":[{},"1.3.1.72","Delta(24)-sterol reductase."]},{"entry":[{},"1.3.1.73","1,2-dihydrovomilenine reductase."]},{"entry":[{},"1.3.1.74","2-alkenal reductase."]},{"entry":[{},"1.3.1.75","Divinyl chlorophyllide a 8-vinyl-"]},{"entry":[{},{},"reductase."]},{"entry":[{},"1.3.1.76","Precorrin-2 dehydrogenase."]},{"entry":[{},"1.3.2.3","Galactonolactone dehydrogenase."]},{"entry":[{},"1.3.3.1","Dihydroorotate oxidase."]},{"entry":[{},"1.3.3.2","Lathosterol oxidase."]},{"entry":[{},"1.3.3.3","Coproporphyrinogen oxidase."]},{"entry":[{},"1.3.3.4","Protoporphyrinogen oxidase."]},{"entry":[{},"1.3.3.5","Bilirubin oxidase."]},{"entry":[{},"1.3.3.6","Acyl-CoA oxidase."]},{"entry":[{},"1.3.3.7","Dihydrouracil oxidase."]},{"entry":[{},"1.3.3.8","Tetrahydroberberine oxidase."]},{"entry":[{},"1.3.3.9","Secologanin synthase."]},{"entry":[{},"1.3.3.10","Tryptophan alpha,beta-oxidase."]},{"entry":[{},"1.3.5.1","Succinate dehydrogenase (ubiquinone)."]},{"entry":[{},"1.3.7.1","6-hydroxynicotinate reductase."]},{"entry":[{},"1.3.7.2","15,16-dihydrobiliverdin:ferredoxin"]},{"entry":[{},{},"oxidoreductase."]},{"entry":[{},"1.3.7.3","Phycoerythrobilin:ferredoxin"]},{"entry":[{},{},"oxidoreductase."]},{"entry":[{},"1.3.7.4","Phytochromobilin:ferredoxin"]},{"entry":[{},{},"oxidoreductase."]},{"entry":[{},"1.3.7.5","Phycocyanobilin:ferredoxin"]},{"entry":[{},{},"oxidoreductase."]},{"entry":[{},"1.3.99.1","Succinate dehydrogenase."]},{"entry":[{},"1.3.99.2","Butyryl-CoA dehydrogenase."]},{"entry":[{},"1.3.99.3","Acyl-CoA dehydrogenase."]},{"entry":[{},"1.3.99.4","3-oxosteroid 1-dehydrogenase."]},{"entry":[{},"1.3.99.5","3-oxo-5-alpha-steroid 4-dehydrogenase."]},{"entry":[{},"1.3.99.6","3-oxo-5-beta-steroid 4-dehydrogenase."]},{"entry":[{},"1.3.99.7","Glutaryl-CoA dehydrogenase."]},{"entry":[{},"1.3.99.8","2-furoyl-CoA dehydrogenase."]},{"entry":[{},"1.3.99.10","Isovaleryl-CoA dehydrogenase."]},{"entry":[{},"1.3.99.11","Dihydroorotate dehydrogenase."]},{"entry":[{},"1.3.99.12","2-methylacyl-CoA dehydrogenase."]},{"entry":[{},"1.3.99.13","Long-chain-acyl-CoA dehydrogenase."]},{"entry":[{},"1.3.99.14","Cyclohexanone dehydrogenase."]},{"entry":[{},"1.3.99.15","Benzoyl-CoA reductase."]},{"entry":[{},"1.3.99.16","Isoquinoline 1-oxidoreductase."]},{"entry":[{},"1.3.99.17","Quinoline 2-oxidoreductase."]},{"entry":[{},"1.3.99.18","Quinaldate 4-oxidoreductase."]},{"entry":[{},"1.3.99.19","Quinoline-4-carboxylate 2-"]},{"entry":[{},{},"oxidoreductase."]},{"entry":[{},"1.3.99.20","4-hydroxybenzoyl-CoA reductase."]},{"entry":[{},"1.3.99.21","(R)-benzylsuccinyl-CoA"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.4.1.1","Alanine dehydrogenase."]},{"entry":[{},"1.4.1.2","Glutamate dehydrogenase."]},{"entry":[{},"1.4.1.3","Glutamate dehydrogenase (NAD(P)+)."]},{"entry":[{},"1.4.1.4","Glutamate dehydrogenase (NADP+)."]},{"entry":[{},"1.4.1.5","L-amino-acid dehydrogenase."]},{"entry":[{},"1.4.1.7","Serine 2-dehydrogenase."]},{"entry":[{},"1.4.1.8","Valine dehydrogenase (NADP+)."]},{"entry":[{},"1.4.1.9","Leucine dehydrogenase."]},{"entry":[{},"1.4.1.10","Glycine dehydrogenase."]},{"entry":[{},"1.4.1.11","L-erythro-3,5-diaminohexanoate"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.4.1.12","2,4-diaminopentanoate dehydrogenase."]},{"entry":[{},"1.4.1.13","Glutamate synthase (NADPH)."]},{"entry":[{},"1.4.1.14","Glutamate synthase (NADH)."]},{"entry":[{},"1.4.1.15","Lysine dehydrogenase."]},{"entry":[{},"1.4.1.16","Diaminopimelate dehydrogenase."]},{"entry":[{},"1.4.1.17","N-methylalanine dehydrogenase."]},{"entry":[{},"1.4.1.18","Lysine 6-dehydrogenase."]},{"entry":[{},"1.4.1.19","Tryptophan dehydrogenase."]},{"entry":[{},"1.4.1.20","Phenylalanine dehydrogenase."]},{"entry":[{},"1.4.2.1","Glycine dehydrogenase (cytochrome)."]},{"entry":[{},"1.4.3.1","D-aspartate oxidase."]},{"entry":[{},"1.4.3.2","L-amino-acid oxidase."]},{"entry":[{},"1.4.3.3","D-amino-acid oxidase."]},{"entry":[{},"1.4.3.4","Amine oxidase (flavin-containing)."]},{"entry":[{},"1.4.3.5","Pyridoxamine-phosphate oxidase."]},{"entry":[{},"1.4.3.6","Amine oxidase (copper-containing)."]},{"entry":[{},"1.4.3.7","D-glutamate oxidase."]},{"entry":[{},"1.4.3.8","Ethanolamine oxidase."]},{"entry":[{},"1.4.3.10","Putrescine oxidase."]},{"entry":[{},"1.4.3.11","L-glutamate oxidase."]},{"entry":[{},"1.4.3.12","Cyclohexylamine oxidase."]},{"entry":[{},"1.4.3.13","Protein-lysine 6-oxidase."]},{"entry":[{},"1.4.3.14","L-lysine oxidase."]},{"entry":[{},"1.4.3.15","D-glutamate(D-aspartate) oxidase."]},{"entry":[{},"1.4.3.16","L-aspartate oxidase."]},{"entry":[{},"1.4.3.19","Glycine oxidase."]},{"entry":[{},"1.4.4.2","Glycine dehydrogenase (decarboxylating)."]},{"entry":[{},"1.4.7.1","Glutamate synthase (ferredoxin)."]},{"entry":[{},"1.4.99.1","D-amino-acid dehydrogenase."]},{"entry":[{},"1.4.99.2","Taurine dehydrogenase."]},{"entry":[{},"1.4.99.3","Amine dehydrogenase."]},{"entry":[{},"1.4.99.4","Aralkylamine dehydrogenase."]},{"entry":[{},"1.4.99.5","Glycine dehydrogenase (cyanide-"]},{"entry":[{},{},"forming)."]},{"entry":[{},"1.5.1.1","Pyrroline-2-carboxylate reductase."]},{"entry":[{},"1.5.1.2","Pyrroline-5-carboxylate reductase."]},{"entry":[{},"1.5.1.3","Dihydrofolate reductase."]},{"entry":[{},"1.5.1.5","Methylenetetrahydrofolate dehydrogenase"]},{"entry":[{},{},"(NADP+)."]},{"entry":[{},"1.5.1.6","Formyltetrahydrofolate dehydrogenase."]},{"entry":[{},"1.5.1.7","Saccharopine dehydrogenase (NAD+, L-"]},{"entry":[{},{},"lysine-forming)."]},{"entry":[{},"1.5.1.8","Saccharopine dehydrogenase (NADP+, L-"]},{"entry":[{},{},"lysine-forming)."]},{"entry":[{},"1.5.1.9","Saccharopine dehydrogenase (NAD+, L-"]},{"entry":[{},{},"glutamate-forming)."]},{"entry":[{},"1.5.1.10","Saccharopine dehydrogenase (NADP+,"]},{"entry":[{},{},"L-glutamate-forming)."]},{"entry":[{},"1.5.1.11","D-octopine dehydrogenase."]},{"entry":[{},"1.5.1.12","1-pyrroline-5-carboxylate dehydrogenase."]},{"entry":[{},"1.5.1.15","Methylenetetrahydrofolate dehydrogenase"]},{"entry":[{},{},"(NAD+)."]},{"entry":[{},"1.5.1.16","D-lysopine dehydrogenase."]},{"entry":[{},"1.5.1.17","Alanopine dehydrogenase."]},{"entry":[{},"1.5.1.18","Ephedrine dehydrogenase."]},{"entry":[{},"1.5.1.19","D-nopaline dehydrogenase."]},{"entry":[{},"1.5.1.20","Methylenetetrahydrofolate reductase"]},{"entry":[{},{},"(NADPH)."]},{"entry":[{},"1.5.1.21","Delta(1)-piperideine-2-carboxylate"]},{"entry":[{},{},"reductase."]},{"entry":[{},"1.5.1.22","Strombine dehydrogenase."]},{"entry":[{},"1.5.1.23","Tauropine dehydrogenase."]},{"entry":[{},"1.5.1.24","N(5)-(carboxyethyl)ornithine synthase."]},{"entry":[{},"1.5.1.25","Thiomorpholine-carboxylate"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.5.1.26","Beta-alanopine dehydrogenase."]},{"entry":[{},"1.5.1.27","1,2-dehydroreticulinium reductase"]},{"entry":[{},{},"(NADPH)."]},{"entry":[{},"1.5.1.28","Opine dehydrogenase."]},{"entry":[{},"1.5.1.29","FMN reductase."]},{"entry":[{},"1.5.1.30","Flavin reductase."]},{"entry":[{},"1.5.1.31","Berberine reductase."]},{"entry":[{},"1.5.1.32","Vomilenine reductase."]},{"entry":[{},"1.5.1.33","Pteridine reductase."]},{"entry":[{},"1.5.1.34","6,7-dihydropteridine reductase."]},{"entry":[{},"1.5.3.1","Sarcosine oxidase."]},{"entry":[{},"1.5.3.2","N-methyl-L-amino-acid oxidase."]},{"entry":[{},"1.5.3.4","N(6)-methyl-lysine oxidase."]},{"entry":[{},"1.5.3.5","(S)-6-hydroxynicotine oxidase."]},{"entry":[{},"1.5.3.6","(R)-6-hydroxynicotine oxidase."]},{"entry":[{},"1.5.3.7","L-pipecolate oxidase."]},{"entry":[{},"1.5.3.10","Dimethylglycine oxidase."]},{"entry":[{},"1.5.3.11","Polyamine oxidase."]},{"entry":[{},"1.5.3.12","Dihydrobenzophenanthridine oxidase."]},{"entry":[{},"1.5.4.1","Pyrimidodiazepine synthase."]},{"entry":[{},"1.5.5.1","Electron-transferring-flavoprotein"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.5.8.1","Dimethylamine dehydrogenase."]},{"entry":[{},"1.5.8.2","Trimethylamine dehydrogenase."]},{"entry":[{},"1.5.99.1","Sarcosine dehydrogenase."]},{"entry":[{},"1.5.99.2","Dimethylglycine dehydrogenase."]},{"entry":[{},"1.5.99.3","L-pipecolate dehydrogenase."]},{"entry":[{},"1.5.99.4","Nicotine dehydrogenase."]},{"entry":[{},"1.5.99.5","Methylglutamate dehydrogenase."]},{"entry":[{},"1.5.99.6","Spermidine dehydrogenase."]},{"entry":[{},"1.5.99.8","Proline dehydrogenase."]},{"entry":[{},"1.5.99.9","Methylenetetrahydromethanopterin"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.5.99.11","5,10-methylenetetrahydromethanopterin"]},{"entry":[{},{},"reductase."]},{"entry":[{},"1.5.99.12","Cytokinin dehydrogenase."]},{"entry":[{},"1.6.1.1","NAD(P)(+) transhydrogenase (B-specific)."]},{"entry":[{},"1.6.1.2","NAD(P)(+) transhydrogenase (AB-"]},{"entry":[{},{},"specific)."]},{"entry":[{},"1.6.2.2","Cytochrome-b5 reductase."]},{"entry":[{},"1.6.2.4","NADPH--hemoprotein reductase."]},{"entry":[{},"1.6.2.5","NADPH--cytochrome-c2 reductase."]},{"entry":[{},"1.6.2.6","Leghemoglobin reductase."]},{"entry":[{},"1.6.3.1","NAD(P)H oxidase."]},{"entry":[{},"1.6.5.3","NADH dehydrogenase (ubiquinone)."]},{"entry":[{},"1.6.5.4","Monodehydroascorbate reductase (NADH)."]},{"entry":[{},"1.6.5.5","NADPH:quinone reductase."]},{"entry":[{},"1.6.5.6","p-benzoquinone reductase (NADPH)."]},{"entry":[{},"1.6.5.7","2-hydroxy-1,4-benzoquinone reductase."]},{"entry":[{},"1.6.6.9","Trimethylamine-N-oxide reductase."]},{"entry":[{},"1.6.99.1","NADPH dehydrogenase."]},{"entry":[{},"1.6.99.2","NAD(P)H dehydrogenase (quinone)."]},{"entry":[{},"1.6.99.3","NADH dehydrogenase."]},{"entry":[{},"1.6.99.5","NADH dehydrogenase (quinone)."]},{"entry":[{},"1.6.99.6","NADPH dehydrogenase (quinone)."]},{"entry":[{},"1.7.1.1","Nitrate reductase (NADH)."]},{"entry":[{},"1.7.1.2","Nitrate reductase (NAD(P)H)."]},{"entry":[{},"1.7.1.3","Nitrate reductase (NADPH)."]},{"entry":[{},"1.7.1.4","Nitrite reductase (NAD(P)H)."]},{"entry":[{},"1.7.1.5","Hyponitrite reductase."]},{"entry":[{},"1.7.1.6","Azobenzene reductase."]},{"entry":[{},"1.7.1.7","GMP reductase."]},{"entry":[{},"1.7.1.9","Nitroquinoline-N-oxide reductase."]},{"entry":[{},"1.7.1.10","Hydroxylamine reductase (NADH)."]},{"entry":[{},"1.7.1.11","4-"]},{"entry":[{},{},"(dimethylamino)phenylazoxybenzene reductase."]},{"entry":[{},"1.7.1.12","N-hydroxy-2-acetamidofluorene"]},{"entry":[{},{},"reductase."]},{"entry":[{},"1.7.2.1","Nitrite reductase (NO-forming)."]},{"entry":[{},"1.7.2.2","Nitrite reductase (cytochrome;"]},{"entry":[{},{},"ammonia-forming)."]},{"entry":[{},"1.7.2.3","Trimethylamine-N-oxide reductase"]},{"entry":[{},{},"(cytochrome c)."]},{"entry":[{},"1.7.3.1","Nitroethane oxidase."]},{"entry":[{},"1.7.3.2","Acetylindoxyl oxidase."]},{"entry":[{},"1.7.3.3","Urate oxidase."]},{"entry":[{},"1.7.3.4","Hydroxylamine oxidase."]},{"entry":[{},"1.7.3.5","3-aci-nitropropanoate oxidase."]},{"entry":[{},"1.7.7.1","Ferredoxin--nitrite reductase."]},{"entry":[{},"1.7.7.2","Ferredoxin--nitrate reductase."]},{"entry":[{},"1.7.99.1","Hydroxylamine reductase."]},{"entry":[{},"1.7.99.4","Nitrate reductase."]},{"entry":[{},"1.7.99.5","5,10-methylenetetrahydrofolate"]},{"entry":[{},{},"reductase (FADH(2))."]},{"entry":[{},"1.7.99.6","Nitrous-oxide reductase."]},{"entry":[{},"1.7.99.7","Nitric-oxide reductase."]},{"entry":[{},"1.7.99.8","Hydroxylamine oxidoreductase."]},{"entry":[{},"1.8.1.2","Sulfite reductase (NADPH)."]},{"entry":[{},"1.8.1.3","Hypotaurine dehydrogenase."]},{"entry":[{},"1.8.1.4","Dihydrolipoyl dehydrogenase."]},{"entry":[{},"1.8.1.5","2-oxopropyl-CoM reductase"]},{"entry":[{},{},"(carboxylating)."]},{"entry":[{},"1.8.1.6","Cystine reductase."]},{"entry":[{},"1.8.1.7","Glutathione-disulfide reductase."]},{"entry":[{},"1.8.1.8","Protein-disulfide reductase."]},{"entry":[{},"1.8.1.9","Thioredoxin-disulfide reductase."]},{"entry":[{},"1.8.1.10","CoA-glutathione reductase."]},{"entry":[{},"1.8.1.11","Asparagusate reductase."]},{"entry":[{},"1.8.1.12","Trypanothione-disulfide reductase."]},{"entry":[{},"1.8.1.13","Bis-gamma-glutamylcystine reductase."]},{"entry":[{},"1.8.1.14","CoA-disulfide reductase."]},{"entry":[{},"1.8.1.15","Mycothione reductase."]},{"entry":[{},"1.8.2.1","Sulfite dehydrogenase."]},{"entry":[{},"1.8.2.2","Thiosulfate dehydrogenase."]},{"entry":[{},"1.8.3.1","Sulfite oxidase."]},{"entry":[{},"1.8.3.2","Thiol oxidase."]},{"entry":[{},"1.8.3.3","Glutathione oxidase."]},{"entry":[{},"1.8.3.4","Methanethiol oxidase."]},{"entry":[{},"1.8.3.5","Prenylcysteine oxidase."]},{"entry":[{},"1.8.4.1","Glutathione--homocystine"]},{"entry":[{},{},"transhydrogenase."]},{"entry":[{},"1.8.4.2","Protein-disulfide reductase"]},{"entry":[{},{},"(glutathione)."]},{"entry":[{},"1.8.4.3","Glutathione--CoA-glutathione"]},{"entry":[{},{},"transhydrogenase."]},{"entry":[{},"1.8.4.4","Glutathione--cystine transhydrogenase."]},{"entry":[{},"1.8.4.5","Methionine-S-oxide reductase."]},{"entry":[{},"1.8.4.6","Protein-methionine-S-oxide reductase."]},{"entry":[{},"1.8.4.7","Enzyme-thiol transhydrogenase"]},{"entry":[{},{},"(glutathione-disulfide)."]},{"entry":[{},"1.8.4.8","Phosphoadenylyl-sulfate reductase"]},{"entry":[{},{},"(thioredoxin)."]},{"entry":[{},"1.8.4.9","Adenylyl-sulfate reductase"]},{"entry":[{},{},"(glutathione)."]},{"entry":[{},"1.8.4.10","Adenylyl-sulfate reductase"]},{"entry":[{},{},"(thioredoxin)."]},{"entry":[{},"1.8.5.1","Glutathione dehydrogenase (ascorbate)."]},{"entry":[{},"1.8.7.1","Sulfite reductase (ferredoxin)."]},{"entry":[{},"1.8.98.1","CoB--CoM heterodisulfide reductase."]},{"entry":[{},"1.8.99.1","Sulfite reductase."]},{"entry":[{},"1.8.99.2","Adenylyl-sulfate reductase."]},{"entry":[{},"1.8.99.3","Hydrogensulfite reductase."]},{"entry":[{},"1.9.3.1","Cytochrome-c oxidase."]},{"entry":[{},"1.9.6.1","Nitrate reductase (cytochrome)."]},{"entry":[{},"1.9.99.1","Iron--cytochrome-c reductase."]},{"entry":[{},"1.10.1.1","Trans-acenaphthene-1,2-diol"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.10.2.1","L-ascorbate--cytochrome-b5 reductase."]},{"entry":[{},"1.10.2.2","Ubiquinol--cytochrome-c reductase."]},{"entry":[{},"1.10.3.1","Catechol oxidase."]},{"entry":[{},"1.10.3.2","Laccase."]},{"entry":[{},"1.10.3.3","L-ascorbate oxidase."]},{"entry":[{},"1.10.3.4","O-aminophenol oxidase."]},{"entry":[{},"1.10.3.5","3-hydroxyanthranilate oxidase."]},{"entry":[{},"1.10.3.6","Rifamycin-B oxidase."]},{"entry":[{},"1.10.99.1","Plastoquinol--plastocyanin reductase."]},{"entry":[{},"1.11.1.1","NADH peroxidase."]},{"entry":[{},"1.11.1.2","NADPH peroxidase."]},{"entry":[{},"1.11.1.3","Fatty-acid peroxidase."]},{"entry":[{},"1.11.1.5","Cytochrome-c peroxidase."]},{"entry":[{},"1.11.1.6","Catalase."]},{"entry":[{},"1.11.1.7","Peroxidase."]},{"entry":[{},"1.11.1.8","Iodide peroxidase."]},{"entry":[{},"1.11.1.9","Glutathione peroxidase."]},{"entry":[{},"1.11.1.10","Chloride peroxidase."]},{"entry":[{},"1.11.1.11","L-ascorbate peroxidase."]},{"entry":[{},"1.11.1.12","Phospholipid-hydroperoxide glutathione"]},{"entry":[{},{},"peroxidase."]},{"entry":[{},"1.11.1.13","Manganese peroxidase."]},{"entry":[{},"1.11.1.14","Diarylpropane peroxidase."]},{"entry":[{},"1.12.1.2","Hydrogen dehydrogenase."]},{"entry":[{},"1.12.1.3","Hydrogen dehydrogenase (NADP+)."]},{"entry":[{},"1.12.2.1","Cytochrome-c3 hydrogenase."]},{"entry":[{},"1.12.5.1","Hydrogen:quinone oxidoreductase."]},{"entry":[{},"1.12.7.2","Ferredoxin hydrogenase."]},{"entry":[{},"1.12.98.1","Coenzyme F420 hydrogenase."]},{"entry":[{},"1.12.98.2","5,10-methenyltetrahydromethanopterin"]},{"entry":[{},{},"hydrogenase."]},{"entry":[{},"1.12.98.3","Methanosarcina-phenazine hydrogenase."]},{"entry":[{},"1.12.99.6","Hydrogenase (acceptor)."]},{"entry":[{},"1.13.11.1","Catechol 1,2-dioxygenase."]},{"entry":[{},"1.13.11.2","Catechol 2,3-dioxygenase."]},{"entry":[{},"1.13.11.3","Protocatechuate 3,4-dioxygenase."]},{"entry":[{},"1.13.11.4","Gentisate 1,2-dioxygenase."]},{"entry":[{},"1.13.11.5","Homogentisate 1,2-dioxygenase."]},{"entry":[{},"1.13.11.6","3-hydroxyanthranilate 3,4-dioxygenase."]},{"entry":[{},"1.13.11.8","Protocatechuate 4,5-dioxygenase."]},{"entry":[{},"1.13.11.9","2,5-dihydroxypyridine 5,6-dioxygenase."]},{"entry":[{},"1.13.11.10","7,8-dihydroxykynurenate 8,8a-"]},{"entry":[{},{},"dioxygenase."]},{"entry":[{},"1.13.11.11","Tryptophan 2,3-dioxygenase."]},{"entry":[{},"1.13.11.12","Lipoxygenase."]},{"entry":[{},"1.13.11.13","Ascorbate 2,3-dioxygenase."]},{"entry":[{},"1.13.11.14","2,3-dihydroxybenzoate 3,4-dioxygenase."]},{"entry":[{},"1.13.11.15","3,4-dihydroxyphenylacetate 2,3-"]},{"entry":[{},{},"dioxygenase."]},{"entry":[{},"1.13.11.16","3-carboxyethylcatechol 2,3-dioxygenase."]},{"entry":[{},"1.13.11.17","Indole 2,3-dioxygenase."]},{"entry":[{},"1.13.11.18","Sulfur dioxygenase."]},{"entry":[{},"1.13.11.19","Cysteamine dioxygenase."]},{"entry":[{},"1.13.11.20","Cysteine dioxygenase."]},{"entry":[{},"1.13.11.22","Caffeate 3,4-dioxygenase."]},{"entry":[{},"1.13.11.23","2,3-dihydroxyindole 2,3-dioxygenase."]},{"entry":[{},"1.13.11.24","Quercetin 2,3-dioxygenase."]},{"entry":[{},"1.13.11.25","3,4-dihydroxy-9,10-secoandrosta-"]},{"entry":[{},{},"1,3,5(10)-triene-9,17-dione 4,5-dioxygenase."]},{"entry":[{},"1.13.11.26","Peptide-tryptophan 2,3-dioxygenase."]},{"entry":[{},"1.13.11.27","4-hydroxyphenylpyruvate dioxygenase."]},{"entry":[{},"1.13.11.28","2,3-dihydroxybenzoate 2,3-dioxygenase."]},{"entry":[{},"1.13.11.29","Stizolobate synthase."]},{"entry":[{},"1.13.11.30","Stizolobinate synthase."]},{"entry":[{},"1.13.11.31","Arachidonate 12-lipoxygenase."]},{"entry":[{},"1.13.11.32","2-nitropropane dioxygenase."]},{"entry":[{},"1.13.11.33","Arachidonate 15-lipoxygenase."]},{"entry":[{},"1.13.11.34","Arachidonate 5-lipoxygenase."]},{"entry":[{},"1.13.11.35","Pyrogallol 1,2-oxygenase."]},{"entry":[{},"1.13.11.36","Chloridazon-catechol dioxygenase."]},{"entry":[{},"1.13.11.37","Hydroxyquinol 1,2-dioxygenase."]},{"entry":[{},"1.13.11.38","1-hydroxy-2-naphthoate 1,2-dioxygenase."]},{"entry":[{},"1.13.11.39","Biphenyl-2,3-diol 1,2-dioxygenase."]},{"entry":[{},"1.13.11.40","Arachidonate 8-lipoxygenase."]},{"entry":[{},"1.13.11.41","2,4\u2032-dihydroxyacetophenone dioxygenase."]},{"entry":[{},"1.13.11.42","Indoleamine-pyrrole 2,3-dioxygenase."]},{"entry":[{},"1.13.11.43","Lignostilbene alpha-beta-dioxygenase."]},{"entry":[{},"1.13.11.44","Linoleate diol synthase."]},{"entry":[{},"1.13.11.45","Linoleate 11-lipoxygenase."]},{"entry":[{},"1.13.11.46","4-hydroxymandelate synthase."]},{"entry":[{},"1.13.11.47","3-hydroxy-4-oxoquinoline 2,4-"]},{"entry":[{},{},"dioxygenase."]},{"entry":[{},"1.13.11.48","3-hydroxy-2-methylquinolin-4-one 2,4-"]},{"entry":[{},{},"dioxygenase."]},{"entry":[{},"1.13.11.49","Chlorite O(2)-lyase."]},{"entry":[{},"1.13.11.50","Acetylacetone-cleaving enzyme."]},{"entry":[{},"1.13.12.1","Arginine 2-monooxygenase."]},{"entry":[{},"1.13.12.2","Lysine 2-monooxygenase."]},{"entry":[{},"1.13.12.3","Tryptophan 2-monooxygenase."]},{"entry":[{},"1.13.12.4","Lactate 2-monooxygenase."]},{"entry":[{},"1.13.12.5","Renilla-luciferin 2-monooxygenase."]},{"entry":[{},"1.13.12.6","Cypridina-luciferin 2-monooxygenase."]},{"entry":[{},"1.13.12.7","Photinus-luciferin 4-monooxygenase (ATP-"]},{"entry":[{},{},"hydrolyzing)."]},{"entry":[{},"1.13.12.8","Watasenia-luciferin 2-monooxygenase."]},{"entry":[{},"1.13.12.9","Phenylalanine 2-monooxygenase."]},{"entry":[{},"1.13.12.11","Methylphenyltetrahydropyridine N-"]},{"entry":[{},{},"monooxygenase."]},{"entry":[{},"1.13.12.12","Apo-beta-carotenoid-14\u2032,13\u2032-dioxygenase."]},{"entry":[{},"1.13.12.13","Oplophorus-luciferin 2-monooxygenase."]},{"entry":[{},"1.13.99.1","Inositol oxygenase."]},{"entry":[{},"1.13.99.3","Tryptophan 2\u2032-dioxygenase."]},{"entry":[{},"1.14.11.1","Gamma-butyrobetaine dioxygenase."]},{"entry":[{},"1.14.11.2","Procollagen-proline dioxygenase."]},{"entry":[{},"1.14.11.3","Pyrimidine-deoxynucleoside 2\u2032-"]},{"entry":[{},{},"dioxygenase."]},{"entry":[{},"1.14.11.4","Procollagen-lysine 5-dioxygenase."]},{"entry":[{},"1.14.11.6","Thymine dioxygenase."]},{"entry":[{},"1.14.11.7","Procollagen-proline 3-dioxygenase."]},{"entry":[{},"1.14.11.8","Trimethyllysine dioxygenase."]},{"entry":[{},"1.14.11.9","Naringenin 3-dioxygenase."]},{"entry":[{},"1.14.11.10","Pyrimidine-deoxynucleoside 1\u2032-"]},{"entry":[{},{},"dioxygenase."]},{"entry":[{},"1.14.11.11","Hyoscyamine (6S)-dioxygenase."]},{"entry":[{},"1.14.11.12","Gibberellin-44 dioxygenase."]},{"entry":[{},"1.14.11.13","Gibberellin 2-beta-dioxygenase."]},{"entry":[{},"1.14.11.14","6-beta-hydroxyhyoscyamine"]},{"entry":[{},{},"epoxidase."]},{"entry":[{},"1.14.11.15","Gibberellin 3-beta-dioxygenase."]},{"entry":[{},"1.14.11.16","Peptide-aspartate beta-dioxygenase."]},{"entry":[{},"1.14.11.17","Taurine dioxygenase."]},{"entry":[{},"1.14.11.18","Phytanoyl-CoA dioxygenase."]},{"entry":[{},"1.14.11.19","Leucocyanidin oxygenase."]},{"entry":[{},"1.14.11.20","Desacetoxyvindoline 4-hydroxylase."]},{"entry":[{},"1.14.11.21","Clavaminate synthase."]},{"entry":[{},"1.14.12.1","Anthranilate 1,2-dioxygenase"]},{"entry":[{},{},"(deaminating, decarboxylating)."]},{"entry":[{},"1.14.12.3","Benzene 1,2-dioxygenase."]},{"entry":[{},"1.14.12.4","3-hydroxy-2-"]},{"entry":[{},{},"methylpyridinecarboxylate dioxygenase."]},{"entry":[{},"1.14.12.5","5-pyridoxate dioxygenase."]},{"entry":[{},"1.14.12.7","Phthalate 4,5-dioxygenase."]},{"entry":[{},"1.14.12.8","4-sulfobenzoate 3,4-dioxygenase."]},{"entry":[{},"1.14.12.9","4-chlorophenylacetate 3,4-"]},{"entry":[{},{},"dioxygenase."]},{"entry":[{},"1.14.12.10","Benzoate 1,2-dioxygenase."]},{"entry":[{},"1.14.12.11","Toluene dioxygenase."]},{"entry":[{},"1.14.12.12","Naphthalene 1,2-dioxygenase."]},{"entry":[{},"1.14.12.13","2-chlorobenzoate 1,2-dioxygenase."]},{"entry":[{},"1.14.12.14","2-aminobenzenesulfonate 2,3-"]},{"entry":[{},{},"dioxygenase."]},{"entry":[{},"1.14.12.15","Terephthalate 1,2-dioxygenase."]},{"entry":[{},"1.14.12.16","2-hydroxyquinoline 5,6-dioxygenase."]},{"entry":[{},"1.14.12.17","Nitric oxide dioxygenase."]},{"entry":[{},"1.14.12.18","Biphenyl 2,3-dioxygenase."]},{"entry":[{},"1.14.13.1","Salicylate 1-monooxygenase."]},{"entry":[{},"1.14.13.2","4-hydroxybenzoate 3-"]},{"entry":[{},{},"monooxygenase."]},{"entry":[{},"1.14.13.3","4-hydroxyphenylacetate 3-"]},{"entry":[{},{},"monooxygenase."]},{"entry":[{},"1.14.13.4","Melilotate 3-monooxygenase."]},{"entry":[{},"1.14.13.5","Imidazoleacetate 4-monooxygenase."]},{"entry":[{},"1.14.13.6","Orcinol 2-monooxygenase."]},{"entry":[{},"1.14.13.7","Phenol 2-monooxygenase."]},{"entry":[{},"1.14.13.8","Dimethylaniline monooxygenase (N-"]},{"entry":[{},{},"oxide-forming)."]},{"entry":[{},"1.14.16.4","Tryptophan 5-monooxygenase."]},{"entry":[{},"1.14.16.5","Glyceryl-ether monooxygenase."]},{"entry":[{},"1.14.16.6","Mandelate 4-monooxygenase."]},{"entry":[{},"1.14.17.1","Dopamine beta-monooxygenase."]},{"entry":[{},"1.14.17.3","Peptidylglycine monooxygenase."]},{"entry":[{},"1.14.17.4","Aminocyclopropanecarboxylate oxidase."]},{"entry":[{},"1.14.18.1","Monophenol monooxygenase."]},{"entry":[{},"1.14.18.2","CMP-N-acetylneuraminate"]},{"entry":[{},{},"monooxygenase."]},{"entry":[{},"1.14.19.1","Stearoyl-CoA 9-desaturase."]},{"entry":[{},"1.14.19.2","Acyl-[acyl-carrier-protein] desaturase."]},{"entry":[{},"1.14.19.3","Linoleoyl-CoA desaturase."]},{"entry":[{},"1.14.20.1","Deacetoxycephalosporin-C synthase."]},{"entry":[{},"1.14.21.1","(S)-stylopine synthase."]},{"entry":[{},"1.14.21.2","(S)-cheilanthifoline synthase."]},{"entry":[{},"1.14.21.3","Berbamunine synthase."]},{"entry":[{},"1.14.21.4","Salutaridine synthase."]},{"entry":[{},"1.14.21.5","(S)-canadine synthase."]},{"entry":[{},"1.14.99.1","Prostaglandin-endoperoxide synthase."]},{"entry":[{},"1.14.99.2","Kynurenine 7,8-hydroxylase."]},{"entry":[{},"1.14.99.3","Heme oxygenase (decyclizing)."]},{"entry":[{},"1.14.99.4","Progesterone monooxygenase."]},{"entry":[{},"1.14.99.7","Squalene monooxygenase."]},{"entry":[{},"1.14.99.9","Steroid 17-alpha-monooxygenase."]},{"entry":[{},"1.14.99.10","Steroid 21-monooxygenase."]},{"entry":[{},"1.14.99.11","Estradiol 6-beta-monooxygenase."]},{"entry":[{},"1.14.99.12","Androst-4-ene-3,17-dione"]},{"entry":[{},{},"monooxygenase."]},{"entry":[{},"1.14.99.14","Progesterone 11-alpha-monooxygenase."]},{"entry":[{},"1.14.99.15","4-methoxybenzoate monooxygenase (O-"]},{"entry":[{},{},"demethylating)."]},{"entry":[{},"1.14.99.19","Plasmanylethanolamine desaturase."]},{"entry":[{},"1.14.99.20","Phylloquinone monooxygenase (2,3-"]},{"entry":[{},{},"epoxidizing)."]},{"entry":[{},"1.14.99.21","Latia-luciferin monooxygenase"]},{"entry":[{},{},"(demethylating)."]},{"entry":[{},"1.14.99.22","Ecdysone 20-monooxygenase."]},{"entry":[{},"1.14.99.23","3-hydroxybenzoate 2-monooxygenase."]},{"entry":[{},"1.14.99.24","Steroid 9-alpha-monooxygenase."]},{"entry":[{},"1.14.99.26","2-hydroxypyridine 5-monooxygenase."]},{"entry":[{},"1.14.99.27","Juglone 3-monooxygenase."]},{"entry":[{},"1.14.99.28","Linalool 8-monooxygenase."]},{"entry":[{},"1.14.99.29","Deoxyhypusine monooxygenase."]},{"entry":[{},"1.14.99.30","Carotene 7,8-desaturase."]},{"entry":[{},"1.14.99.31","Myristoyl-CoA 11-(E) desaturase."]},{"entry":[{},"1.14.99.32","Myristoyl-CoA 11-(Z) desaturase."]},{"entry":[{},"1.14.99.33","Delta(12)-fatty acid"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.14.99.34","Monoprenyl isoflavone epoxidase."]},{"entry":[{},"1.14.99.35","Thiophene-2-carbonyl-CoA"]},{"entry":[{},{},"monooxygenase."]},{"entry":[{},"1.14.99.36","Beta-carotene 15,15\u2032-"]},{"entry":[{},{},"monooxygenase."]},{"entry":[{},"1.14.99.37","Taxadiene 5-alpha-hydroxylase."]},{"entry":[{},"1.15.1.1","Superoxide dismutase."]},{"entry":[{},"1.15.1.2","Superoxide reductase."]},{"entry":[{},"1.16.1.1","Mercury(II) reductase."]},{"entry":[{},"1.16.1.2","Diferric-transferrin reductase."]},{"entry":[{},"1.16.1.3","Aquacobalamin reductase."]},{"entry":[{},"1.16.1.4","Cob(II)alamin reductase."]},{"entry":[{},"1.16.1.5","Aquacobalamin reductase"]},{"entry":[{},{},"(NADPH)."]},{"entry":[{},"1.16.1.6","Cyanocobalamin reductase"]},{"entry":[{},{},"(cyanide-eliminating)."]},{"entry":[{},"1.16.1.7","Ferric-chelate reductase."]},{"entry":[{},"1.16.1.8","[Methionine synthase] reductase."]},{"entry":[{},"1.16.3.1","Ferroxidase."]},{"entry":[{},"1.16.8.1","Cob(II)yrinic acid a,c-diamide"]},{"entry":[{},{},"reductase."]},{"entry":[{},"1.17.1.1","CDP-4-dehydro-6-deoxyglucose"]},{"entry":[{},{},"reductase."]},{"entry":[{},"1.17.1.2","4-hydroxy-3-methylbut-2-enyl"]},{"entry":[{},{},"diphosphate reductase."]},{"entry":[{},"1.17.1.3","Leucoanthocyanidin reductase."]},{"entry":[{},"1.17.1.4","Xanthine dehydrogenase."]},{"entry":[{},"1.17.1.5","Nicotinate dehydrogenase."]},{"entry":[{},"1.17.3.1","Pteridine oxidase."]},{"entry":[{},"1.17.3.2","Xanthine oxidase."]},{"entry":[{},"1.17.3.3","6-hydroxynicotinate"]},{"entry":[{},{},"dehydrogenase."]},{"entry":[{},"1.17.4.1","Ribonucleoside-diphosphate"]},{"entry":[{},{},"reductase."]},{"entry":[{},"1.17.4.2","Ribonucleoside-triphosphate"]},{"entry":[{},{},"reductase."]},{"entry":[{},"1.17.4.3","4-hydroxy-3-methylbut-2-en-1-yl"]},{"entry":[{},{},"diphosphate synthase."]},{"entry":[{},"1.17.5.1","Phenylacetyl-CoA dehydrogenase."]},{"entry":[{},"1.17.99.1","4-cresol dehydrogenase"]},{"entry":[{},{},"(hydroxylating)."]},{"entry":[{},"1.17.99.2","Ethylbenzene hydroxylase."]},{"entry":[{},"1.18.1.1","Rubredoxin--NAD(+) reductase."]},{"entry":[{},"1.18.1.2","Ferredoxin--NADP(+) reductase."]},{"entry":[{},"1.18.1.3","Ferredoxin--NAD(+) reductase."]},{"entry":[{},"1.18.1.4","Rubredoxin--NAD(P)(+) reductase."]},{"entry":[{},"1.18.6.1","Nitrogenase."]},{"entry":[{},"1.19.6.1","Nitrogenase (flavodoxin)."]},{"entry":[{},"1.20.1.1","Phosphonate dehydrogenase."]},{"entry":[{},"1.20.4.1","Arsenate reductase (glutaredoxin)."]},{"entry":[{},"1.20.4.2","Methylarsonate reductase."]},{"entry":[{},"1.20.98.1","Arsenate reductase (azurin)."]},{"entry":[{},"1.20.99.1","Arsenate reductase (donor)."]},{"entry":[{},"1.21.3.1","Isopenicillin-N synthase."]},{"entry":[{},"1.21.3.2","Columbamine oxidase."]},{"entry":[{},"1.21.3.3","Reticuline oxidase."]},{"entry":[{},"1.21.3.4","Sulochrin oxidase ((+)-bisdechlorogeodin-"]},{"entry":[{},{},"forming)."]},{"entry":[{},"1.21.3.5","Sulochrin oxidase ((\u2212)-bisdechlorogeodin-"]},{"entry":[{},{},"forming)."]},{"entry":[{},"1.21.3.6","Aureusidin synthase."]},{"entry":[{},"1.21.4.1","D-proline reductase (dithiol)."]},{"entry":[{},"1.21.4.2","Glycine reductase."]},{"entry":[{},"1.21.4.3","Sarcosine reductase."]},{"entry":[{},"1.21.4.4","Betaine reductase."]},{"entry":[{},"1.21.99.1","Beta-cyclopiazonate dehydrogenase."]},{"entry":[{},"1.97.1.1","Chlorate reductase."]},{"entry":[{},"1.97.1.2","Pyrogallol hydroxytransferase."]},{"entry":[{},"1.97.1.3","Sulfur reductase."]},{"entry":[{},"1.97.1.4","[Formate acetyltransferase] activating"]},{"entry":[{},{},"enzyme."]},{"entry":[{},"1.97.1.8","Tetrachloroethene reductive dehalogenase."]},{"entry":[{},"1.97.1.9","Selenate reductase."]},{"entry":[{},"1.97.1.10","Thyroxine 5\u2032-deiodinase."]},{"entry":[{},"1.97.1.11","Thyroxine 5-deiodinase."]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"ENZYME: 2.\u2014.\u2014.\u2014"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"168pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"2.1.1.1","Nicotinamide N-methyltransferase."]},{"entry":[{},"2.1.1.2","Guanidinoacetate N-methyltransferase."]},{"entry":[{},"2.1.1.3","Thetin--homocysteine S-methyltransferase."]},{"entry":[{},"2.1.1.4","Acetylserotonin O-methyltransferase."]},{"entry":[{},"2.1.1.5","Betaine--homocysteine S-methyltransferase."]},{"entry":[{},"2.1.1.6","Catechol O-methyltransferase."]},{"entry":[{},"2.1.1.7","Nicotinate N-methyltransferase."]},{"entry":[{},"2.1.1.8","Histamine N-methyltransferase."]},{"entry":[{},"2.1.1.9","Thiol S-methyltransferase."]},{"entry":[{},"2.1.1.10","Homocysteine S-methyltransferase."]},{"entry":[{},"2.1.1.11","Magnesium protoporphyrin IX"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.12","Methionine S-methyltransferase."]},{"entry":[{},"2.1.1.13","Methionine synthase."]},{"entry":[{},"2.1.1.14","5-methyltetrahydropteroyltriglutamate--"]},{"entry":[{},{},"homocysteine S-methyltransferase."]},{"entry":[{},"2.1.1.15","Fatty-acid O-methyltransferase."]},{"entry":[{},"2.1.1.16","Methylene-fatty-acyl-phospholipid"]},{"entry":[{},{},"synthase."]},{"entry":[{},"2.1.1.17","Phosphatidylethanolamine N-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.18","Polysaccharide O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.19","Trimethylsulfonium--"]},{"entry":[{},{},"tetrahydrofolate N-methyltransferase."]},{"entry":[{},"2.1.1.20","Glycine N-methyltransferase."]},{"entry":[{},"2.1.1.21","Methylamine--glutamate N-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.22","Carnosine N-methyltransferase."]},{"entry":[{},"2.1.1.25","Phenol O-methyltransferase."]},{"entry":[{},"2.1.1.26","Iodophenol O-methyltransferase."]},{"entry":[{},"2.1.1.27","Tyramine N-methyltransferase."]},{"entry":[{},"2.1.1.28","Phenylethanolamine N-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.29","tRNA (cytosine-5-)-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.31","tRNA (guanine-N(1)-)-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.32","tRNA (guanine-N(2)-)-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.33","tRNA (guanine-N(7)-)-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.34","tRNA (guanosine-2\u2032-O-)-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.35","tRNA (uracil-5-)-methyltransferase."]},{"entry":[{},"2.1.1.36","tRNA (adenine-N(1)-)-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.37","DNA (cytosine-5-)-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.38","O-demethylpuromycin O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.39","Inositol 3-methyltransferase."]},{"entry":[{},"2.1.1.40","Inositol 1-methyltransferase."]},{"entry":[{},"2.1.1.41","Sterol 24-C-methyltransferase."]},{"entry":[{},"2.1.1.42","Luteolin O-methyltransferase."]},{"entry":[{},"2.1.1.43","Histone-lysine N-methyltransferase."]},{"entry":[{},"2.1.1.44","Dimethylhistidine N-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.45","Thymidylate synthase."]},{"entry":[{},"2.1.1.46","Isoflavone 4\u2032-O-methyltransferase."]},{"entry":[{},"2.1.1.47","Indolepyruvate C-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.48","rRNA (adenine-N(6)-)-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.49","Amine N-methyltransferase."]},{"entry":[{},"2.1.1.50","Loganate O-methyltransferase."]},{"entry":[{},"2.1.1.51","rRNA (guanine-N(1)-)-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.52","rRNA (guanine-N(2)-)-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.53","Putrescine N-methyltransferase."]},{"entry":[{},"2.1.1.54","Deoxycytidylate C-methyltransferase."]},{"entry":[{},"2.1.1.55","tRNA (adenine-N(6)-)-methyltransferase."]},{"entry":[{},"2.1.1.56","mRNA (guanine-N(7)-)-methyltransferase."]},{"entry":[{},"2.1.1.57","mRNA (nucleoside-2\u2032-O-)-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.59","[Cytochrome c]-lysine N-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.60","Calmodulin-lysine N-methyltransferase."]},{"entry":[{},"2.1.1.61","tRNA (5-methylaminomethyl-2-"]},{"entry":[{},{},"thiouridylate)-methyltransferase."]},{"entry":[{},"2.1.1.62","mRNA (2\u2032-O-methyladenosine-N(6)-)-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.63","Methylated-DNA--[protein]-cysteine S-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.64","3-demethylubiquinone-9 3-O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.65","Licodione 2\u2032-O-methyltransferase."]},{"entry":[{},"2.1.1.66","rRNA (adenosine-2\u2032-O-)-methyltransferase."]},{"entry":[{},"2.1.1.67","Thiopurine S-methyltransferase."]},{"entry":[{},"2.1.1.68","Caffeate O-methyltransferase."]},{"entry":[{},"2.1.1.69","5-hydroxyfuranocoumarin 5-O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.70","8-hydroxyfuranocoumarin 8-O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.71","Phosphatidyl-N-methylethanolamine N-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.72","Site-specific DNA-methyltransferase"]},{"entry":[{},{},"(adenine-specific)."]},{"entry":[{},"2.1.1.74","Methylenetetrahydrofolate--tRNA-(uracil-"]},{"entry":[{},{},"5-)-methyltransferase (FADH(2)-oxidizing)."]},{"entry":[{},"2.1.1.75","Apigenin 4\u2032-O-methyltransferase."]},{"entry":[{},"2.1.1.76","Quercetin 3-O-methyltransferase."]},{"entry":[{},"2.1.1.77","Protein-L-isoaspartate(D-aspartate) O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.78","Isoorientin 3\u2032-O-methyltransferase."]},{"entry":[{},"2.1.1.79","Cyclopropane-fatty-acyl-phospholipid"]},{"entry":[{},{},"synthase."]},{"entry":[{},"2.1.1.80","Protein-glutamate O-methyltransferase."]},{"entry":[{},"2.1.1.82","3-methylquercitin 7-O-methyltransferase."]},{"entry":[{},"2.1.1.83","3,7-dimethylquercitin 4\u2032-O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.84","Methylquercetagetin 6-O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.85","Protein-histidine N-methyltransferase."]},{"entry":[{},"2.1.1.86","Tetrahydromethanopterin S-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.87","Pyridine N-methyltransferase."]},{"entry":[{},"2.1.1.88","8-hydroxyquercitin 8-O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.89","Tetrahydrocolumbamine 2-O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.90","Methanol--5-"]},{"entry":[{},{},"hydroxybenzimidazolylcobamide Co-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.91","Isobutyraldoxime O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.92","Bergaptol O-methyltransferase."]},{"entry":[{},"2.1.1.93","Xanthotoxol O-methyltransferase."]},{"entry":[{},"2.1.1.94","11-O-demethyl-17-O-"]},{"entry":[{},{},"deacetylvindoline O-methyltransferase."]},{"entry":[{},"2.1.1.95","Tocopherol O-methyltransferase."]},{"entry":[{},"2.1.1.96","Thioether S-methyltransferase."]},{"entry":[{},"2.1.1.97","3-hydroxyanthranilate 4-C-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.98","Diphthine synthase."]},{"entry":[{},"2.1.1.99","16-methoxy-2,3-dihydro-3-"]},{"entry":[{},{},"hydroxytabersonine N-methyltransferase."]},{"entry":[{},"2.1.1.100","Protein-S-isoprenylcysteine O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.101","Macrocin O-methyltransferase."]},{"entry":[{},"2.1.1.102","Demethylmacrocin O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.103","Phosphoethanolamine N-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.104","Caffeoyl-CoA O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.105","N-benzoyl-4-hydroxyanthranilate"]},{"entry":[{},{},"4-O-methyltransferase."]},{"entry":[{},"2.1.1.106","Tryptophan 2-C-methyltransferase."]},{"entry":[{},"2.1.1.107","Uroporphyrin-III C-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.108","6-hydroxymellein O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.109","Demethylsterigmatocystin 6-O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.110","Sterigmatocystin 7-O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.111","Anthranilate N-methyltransferase."]},{"entry":[{},"2.1.1.112","Glucuronoxylan 4-O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.113","Site-specific DNA-"]},{"entry":[{},{},"methyltransferase (cytosine-N(4)-specific)."]},{"entry":[{},"2.1.1.114","Hexaprenyldihydroxybenzoate"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.115","(RS)-1-benzyl-1,2,3,4-"]},{"entry":[{},{},"tetrahydroisoquinoline N-methyltransferase."]},{"entry":[{},"2.1.1.116","3\u2032-hydroxy-N-methyl-(S)-coclaurine 4\u2032-O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.117","(S)-scoulerine 9-O-methyltransferase."]},{"entry":[{},"2.1.1.118","Columbamine O-methyltransferase."]},{"entry":[{},"2.1.1.119","10-hydroxydihydrosanguinarine 10-O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.120","12-hydroxydihydrochelirubine 12-O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.121","6-O-methylnorlaudanosoline 5\u2032-O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.122","(S)-tetrahydroprotoberberine N-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.123","[Cytochrome-c]-methionine S-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.124","[Cytochrome-c]-arginine N-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.125","Histone-arginine N-methyltransferase."]},{"entry":[{},"2.1.1.126","[Myelin basic protein]-arginine N-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.127","[Ribulose-bisphosphate carboxylase]-"]},{"entry":[{},{},"lysine N-methyltransferase."]},{"entry":[{},"2.1.1.128","(RS)-norcoclaurine 6-O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.129","Inositol 4-methyltransferase."]},{"entry":[{},"2.1.1.130","Precorrin-2 C(20)-methyltransferase."]},{"entry":[{},"2.1.1.131","Precorrin-3B C(17)-methyltransferase."]},{"entry":[{},"2.1.1.132","Precorrin-6Y C(5,15)-methyltransferase"]},{"entry":[{},{},"(decarboxylating)."]},{"entry":[{},"2.1.1.133","Precorrin-4 C(11)-methyltransferase."]},{"entry":[{},"2.1.1.136","Chlorophenol O-methyltransferase."]},{"entry":[{},"2.1.1.137","Arsenite methyltransferase."]},{"entry":[{},"2.1.1.139","3\u2032-demethylstaurosporine O-"]},{"entry":[{},{},"methyltransferase."]},{"entry":[{},"2.1.1.140","(S)-coclaurine-N-methyltransferase."]},{"entry":[{},"2.1.1.141","Jasmonate O-methyltransferase."]},{"entry":[{},"2.1.1.142","Cycloartenol 24-C-methyltransferase."]},{"entry":[{},"2.1.1.143","24-methylenesterol C-methyltransferase."]},{"entry":[{},"2.1.1.144","Trans-aconitate 2-methyltransferase."]},{"entry":[{},"2.1.1.145","Trans-aconitate 3-methyltransferase."]},{"entry":[{},"2.1.1.146","(Iso)eugenol O-methyltransferase."]},{"entry":[{},"2.1.1.147","Corydaline synthase."]},{"entry":[{},"2.1.1.148","Thymidylate synthase (FAD)."]},{"entry":[{},"2.1.1.149","Myricetin O-methyltransferase."]},{"entry":[{},"2.1.1.150","Isoflavone 7-O-methyltransferase."]},{"entry":[{},"2.1.1.151","Cobalt-factor II C(20)-methyltransferase."]},{"entry":[{},"2.1.1.152","Precorrin-6A synthase (deacetylating)."]},{"entry":[{},"2.1.2.1","Glycine hydroxymethyltransferase."]},{"entry":[{},"2.1.2.2","Phosphoribosylglycinamide"]},{"entry":[{},{},"formyltransferase."]},{"entry":[{},"2.1.2.3","Phosphoribosylaminoimidazolecarboxamide"]},{"entry":[{},{},"formyltransferase."]},{"entry":[{},"2.1.2.4","Glycine formimidoyltransferase."]},{"entry":[{},"2.1.2.5","Glutamate formimidoyltransferase."]},{"entry":[{},"2.1.2.7","D-alanine 2-"]},{"entry":[{},{},"hydroxymethyltransferase."]},{"entry":[{},"2.1.2.8","Deoxycytidylate 5-"]},{"entry":[{},{},"hydroxymethyltransferase."]},{"entry":[{},"2.1.2.9","Methionyl-tRNA formyltransferase."]},{"entry":[{},"2.1.2.10","Aminomethyltransferase."]},{"entry":[{},"2.1.2.11","3-methyl-2-oxobutanoate"]},{"entry":[{},{},"hydroxymethyltransferase."]},{"entry":[{},"2.1.3.1","Methylmalonyl-CoA"]},{"entry":[{},{},"carboxytransferase."]},{"entry":[{},"2.1.3.2","Aspartate carbamoyltransferase."]},{"entry":[{},"2.1.3.3","Ornithine carbamoyltransferase."]},{"entry":[{},"2.1.3.5","Oxamate carbamoyltransferase."]},{"entry":[{},"2.1.3.6","Putrescine carbamoyltransferase."]},{"entry":[{},"2.1.3.7","3-hydroxymethylcephem"]},{"entry":[{},{},"carbamoyltransferase."]},{"entry":[{},"2.1.3.8","Lysine carbamoyltransferase."]},{"entry":[{},"2.1.4.1","Glycine amidinotransferase."]},{"entry":[{},"2.1.4.2","Scyllo-inosamine-4-phosphate"]},{"entry":[{},{},"amidinotransferase."]},{"entry":[{},"2.2.1.1","Transketolase."]},{"entry":[{},"2.2.1.2","Transaldolase."]},{"entry":[{},"2.2.1.3","Formaldehyde transketolase."]},{"entry":[{},"2.2.1.4","Acetoin--ribose-5-phosphate"]},{"entry":[{},{},"transaldolase."]},{"entry":[{},"2.2.1.5","2-hydroxy-3-oxoadipate synthase."]},{"entry":[{},"2.2.1.6","Acetolactate synthase."]},{"entry":[{},"2.2.1.7","1-deoxy-D-xylulose-5-phosphate"]},{"entry":[{},{},"synthase."]},{"entry":[{},"2.2.1.8","Fluorothreonine transaldolase."]},{"entry":[{},"2.3.1.1","Amino-acid N-acetyltransferase."]},{"entry":[{},"2.3.1.2","Imidazole N-acetyltransferase."]},{"entry":[{},"2.3.1.3","Glucosamine N-acetyltransferase."]},{"entry":[{},"2.3.1.4","Glucosamine 6-phosphate N-"]},{"entry":[{},{},"acetyltransferase."]},{"entry":[{},"2.3.1.5","Arylamine N-acetyltransferase."]},{"entry":[{},"2.3.1.6","Choline O-acetyltransferase."]},{"entry":[{},"2.3.1.7","Carnitine O-acetyltransferase."]},{"entry":[{},"2.3.1.8","Phosphate acetyltransferase."]},{"entry":[{},"2.3.1.9","Acetyl-CoA C-acetyltransferase."]},{"entry":[{},"2.3.1.10","Hydrogen-sulfide S-acetyltransferase."]},{"entry":[{},"2.3.1.11","Thioethanolamine S-acetyltransferase."]},{"entry":[{},"2.3.1.12","Dihydrolipoyllysine-residue"]},{"entry":[{},{},"acetyltransferase."]},{"entry":[{},"2.3.1.13","Glycine N-acyltransferase."]},{"entry":[{},"2.3.1.14","Glutamine N-phenylacetyltransferase."]},{"entry":[{},"2.3.1.15","Glycerol-3-phosphate O-acyltransferase."]},{"entry":[{},"2.3.1.16","Acetyl-CoA C-acyltransferase."]},{"entry":[{},"2.3.1.17","Aspartate N-acetyltransferase."]},{"entry":[{},"2.3.1.18","Galactoside O-acetyltransferase."]},{"entry":[{},"2.3.1.19","Phosphate butyryltransferase."]},{"entry":[{},"2.3.1.20","Diacylglycerol O-acyltransferase."]},{"entry":[{},"2.3.1.21","Carnitine O-palmitoyltransferase."]},{"entry":[{},"2.3.1.22","2-acylglycerol O-acyltransferase."]},{"entry":[{},"2.3.1.23","1-acylglycerophosphocholine O-"]},{"entry":[{},{},"acyltransferase."]},{"entry":[{},"2.3.1.24","Sphingosine N-acyltransferase."]},{"entry":[{},"2.3.1.25","Plasmalogen synthase."]},{"entry":[{},"2.3.1.26","Sterol O-acyltransferase."]},{"entry":[{},"2.3.1.27","Cortisol O-acetyltransferase."]},{"entry":[{},"2.3.1.28","Chloramphenicol O-acetyltransferase."]},{"entry":[{},"2.3.1.29","Glycine C-acetyltransferase."]},{"entry":[{},"2.3.1.30","Serine O-acetyltransferase."]},{"entry":[{},"2.3.1.31","Homoserine O-acetyltransferase."]},{"entry":[{},"2.3.1.32","Lysine N-acetyltransferase."]},{"entry":[{},"2.3.1.33","Histidine N-acetyltransferase."]},{"entry":[{},"2.3.1.34","D-tryptophan N-acetyltransferase."]},{"entry":[{},"2.3.1.35","Glutamate N-acetyltransferase."]},{"entry":[{},"2.3.1.36","D-amino-acid N-acetyltransferase."]},{"entry":[{},"2.3.1.37","5-aminolevulinate synthase."]},{"entry":[{},"2.3.1.38","[Acyl-carrier-protein] S-acetyltransferase."]},{"entry":[{},"2.3.1.39","[Acyl-carrier-protein] S-"]},{"entry":[{},{},"malonyltransferase."]},{"entry":[{},"2.3.1.40","Acyl-[acyl-carrier-protein]--phospholipid"]},{"entry":[{},{},"O-acyltransferase."]},{"entry":[{},"2.3.1.41","3-oxoacyl-[acyl-carrier-protein] synthase."]},{"entry":[{},"2.3.1.42","Glycerone-phosphate O-acyltransferase."]},{"entry":[{},"2.3.1.43","Phosphatidylcholine--sterol O-"]},{"entry":[{},{},"acyltransferase."]},{"entry":[{},"2.3.1.44","N-acetylneuraminate 4-O-"]},{"entry":[{},{},"acetyltransferase."]},{"entry":[{},"2.3.1.45","N-acetylneuraminate 7-O(or 9-O)-"]},{"entry":[{},{},"acetyltransferase."]},{"entry":[{},"2.3.1.46","Homoserine O-succinyltransferase."]},{"entry":[{},"2.3.1.47","8-amino-7-oxononanoate synthase."]},{"entry":[{},"2.3.1.48","Histone acetyltransferase."]},{"entry":[{},"2.3.1.49","Deacetyl-[citrate-(pro-3S)-lyase] S-"]},{"entry":[{},{},"acetyltransferase."]},{"entry":[{},"2.3.1.50","Serine C-palmitoyltransferase."]},{"entry":[{},"2.3.1.51","1-acylglycerol-3-phosphate O-"]},{"entry":[{},{},"acyltransferase."]},{"entry":[{},"2.3.1.52","2-acylglycerol-3-phosphate O-"]},{"entry":[{},{},"acyltransferase."]},{"entry":[{},"2.3.1.53","Phenylalanine N-acetyltransferase."]},{"entry":[{},"2.3.1.54","Formate C-acetyltransferase."]},{"entry":[{},"2.3.1.56","Aromatic-hydroxylamine O-"]},{"entry":[{},{},"acetyltransferase."]},{"entry":[{},"2.3.1.57","Diamine N-acetyltransferase."]},{"entry":[{},"2.3.1.58","2,3-diaminopropionate N-"]},{"entry":[{},{},"oxalyltransferase."]},{"entry":[{},"2.3.1.59","Gentamicin 2\u2032-N-acetyltransferase."]},{"entry":[{},"2.3.1.60","Gentamicin 3\u2032-N-acetyltransferase."]},{"entry":[{},"2.3.1.61","Dihydrolipoyllysine-residue"]},{"entry":[{},{},"succinyltransferase."]},{"entry":[{},"2.3.1.62","2-acylglycerophosphocholine O-"]},{"entry":[{},{},"acyltransferase."]},{"entry":[{},"2.3.1.63","1-alkylglycerophosphocholine O-"]},{"entry":[{},{},"acyltransferase."]},{"entry":[{},"2.3.1.64","Agmatine N(4)-"]},{"entry":[{},{},"coumaroyltransferase."]},{"entry":[{},"2.3.1.65","Glycine N-choloyltransferase."]},{"entry":[{},"2.3.1.66","Leucine N-acetyltransferase."]},{"entry":[{},"2.3.1.67","1-alkylglycerophosphocholine O-"]},{"entry":[{},{},"acetyltransferase."]},{"entry":[{},"2.3.1.68","Glutamine N-acyltransferase."]},{"entry":[{},"2.3.1.69","Monoterpenol O-acetyltransferase."]},{"entry":[{},"2.3.1.70","CDP-acylglycerol O-"]},{"entry":[{},{},"arachidonoyltransferase."]},{"entry":[{},"2.3.1.71","Glycine N-benzoyltransferase."]},{"entry":[{},"2.3.1.72","Indoleacetylglucose--inositol O-"]},{"entry":[{},{},"acyltransferase."]},{"entry":[{},"2.3.1.73","Diacylglycerol--sterol O-"]},{"entry":[{},{},"acyltransferase."]},{"entry":[{},"2.3.1.74","Naringenin-chalcone synthase."]},{"entry":[{},"2.3.1.75","Long-chain-alcohol O-fatty-"]},{"entry":[{},{},"acyltransferase."]},{"entry":[{},"2.3.1.76","Retinol O-fatty-acyltransferase."]},{"entry":[{},"2.3.1.77","Triacylglycerol--sterol O-"]},{"entry":[{},{},"acyltransferase."]},{"entry":[{},"2.3.1.78","Heparan-alpha-glucosaminide N-"]},{"entry":[{},{},"acetyltransferase."]},{"entry":[{},"2.3.1.79","Maltose O-acetyltransferase."]},{"entry":[{},"2.3.1.80","Cysteine-S-conjugate N-"]},{"entry":[{},{},"acetyltransferase."]},{"entry":[{},"2.3.1.81","Aminoglycoside N(3\u2032)-"]},{"entry":[{},{},"acetyltransferase."]},{"entry":[{},"2.3.1.82","Aminoglycoside N(6\u2032)-"]},{"entry":[{},{},"acetyltransferase."]},{"entry":[{},"2.3.1.83","Phosphatidylcholine--dolichol O-"]},{"entry":[{},{},"acyltransferase."]},{"entry":[{},"2.3.1.84","Alcohol O-acetyltransferase."]},{"entry":[{},"2.3.1.85","Fatty-acid synthase."]},{"entry":[{},"2.3.1.86","Fatty-acyl-CoA synthase."]},{"entry":[{},"2.3.1.87","Aralkylamine N-acetyltransferase."]},{"entry":[{},"2.3.1.88","Peptide alpha-N-acetyltransferase."]},{"entry":[{},"2.3.1.89","Tetrahydrodipicolinate N-acetyltransferase."]},{"entry":[{},"2.3.1.90","Beta-glucogallin O-galloyltransferase."]},{"entry":[{},"2.3.1.91","Sinapoylglucose--choline O-"]},{"entry":[{},{},"sinapoyltransferase."]},{"entry":[{},"2.3.1.92","Sinapoylglucose--malate O-"]},{"entry":[{},{},"sinapoyltransferase."]},{"entry":[{},"2.3.1.93","13-hydroxylupinine O-tigloyltransferase."]},{"entry":[{},"2.3.1.94","Erythronolide synthase."]},{"entry":[{},"2.3.1.95","Trihydroxystilbene synthase."]},{"entry":[{},"2.3.1.96","Glycoprotein N-palmitoyltransferase."]},{"entry":[{},"2.3.1.97","Glycylpeptide N-tetradecanoyltransferase."]},{"entry":[{},"2.3.1.98","Chlorogenate--glucarate O-"]},{"entry":[{},{},"hydroxycinnamoyltransferase."]},{"entry":[{},"2.3.1.99","Quinate O-hydroxycinnamoyltransferase."]},{"entry":[{},"2.3.1.100","Myelin-proteolipid O-"]},{"entry":[{},{},"palmitoyltransferase."]},{"entry":[{},"2.3.1.101","Formylmethanofuran--"]},{"entry":[{},{},"tetrahydromethanopterin N-formyltransferase."]},{"entry":[{},"2.3.1.102","N(6)-hydroxylysine O-acetyltransferase."]},{"entry":[{},"2.3.1.103","Sinapoylglucose--sinapoylglucose O-"]},{"entry":[{},{},"sinapoyltransferase."]},{"entry":[{},"2.3.1.104","1-alkenylglycerophosphocholine O-"]},{"entry":[{},{},"acyltransferase."]},{"entry":[{},"2.3.1.105","Alkylglycerophosphate 2-O-"]},{"entry":[{},{},"acetyltransferase."]},{"entry":[{},"2.3.1.106","Tartronate O-"]},{"entry":[{},{},"hydroxycinnamoyltransferase."]},{"entry":[{},"2.3.1.107","17-O-deacetylvindoline O-"]},{"entry":[{},{},"acetyltransferase."]},{"entry":[{},"2.3.1.108","Tubulin N-acetyltransferase."]},{"entry":[{},"2.3.1.109","Arginine N-succinyltransferase."]},{"entry":[{},"2.3.1.110","Tyramine N-feruloyltransferase."]},{"entry":[{},"2.3.1.111","Mycocerosate synthase."]},{"entry":[{},"2.3.1.112","D-tryptophan N-malonyltransferase."]},{"entry":[{},"2.3.1.113","Anthranilate N-malonyltransferase."]},{"entry":[{},"2.3.1.114","3,4-dichloroaniline N-malonyltransferase."]},{"entry":[{},"2.3.1.115","Isoflavone-7-O-beta-glucoside 6\u2033-O-"]},{"entry":[{},{},"malonyltransferase."]},{"entry":[{},"2.3.1.116","Flavonol-3-O-beta-glucoside O-"]},{"entry":[{},{},"malonyltransferase."]},{"entry":[{},"2.3.1.117","2,3,4,5-tetrahydropyridine-2,6-"]},{"entry":[{},{},"dicarboxylate N-succinyltransferase."]},{"entry":[{},"2.3.1.118","N-hydroxyarylamine O-acetyltransferase."]},{"entry":[{},"2.3.1.119","Icosanoyl-CoA synthase."]},{"entry":[{},"2.3.1.121","1-alkenylglycerophosphoethanolamine O-"]},{"entry":[{},{},"acyltransferase."]},{"entry":[{},"2.3.1.122","Trehalose O-mycolyltransferase."]},{"entry":[{},"2.3.1.123","Dolichol O-acyltransferase."]},{"entry":[{},"2.3.1.125","1-alkyl-2-acetylglycerol O-"]},{"entry":[{},{},"acyltransferase."]},{"entry":[{},"2.3.1.126","Isocitrate O-"]},{"entry":[{},{},"dihydroxycinnamoyltransferase."]},{"entry":[{},"2.3.1.127","Ornithine N-benzoyltransferase."]},{"entry":[{},"2.3.1.128","Ribosomal-protein-alanine N-"]},{"entry":[{},{},"acetyltransferase."]},{"entry":[{},"2.3.1.129","Acyl-[acyl-carrier-protein]--UDP-N-"]},{"entry":[{},{},"acetylglucosamine O-acyltransferase."]},{"entry":[{},"2.3.1.130","Galactarate O-"]},{"entry":[{},{},"hydroxycinnamoyltransferase."]},{"entry":[{},"2.3.1.131","Glucarate O-"]},{"entry":[{},{},"hydroxycinnamoyltransferase."]},{"entry":[{},"2.3.1.132","Glucarolactone O-"]},{"entry":[{},{},"hydroxycinnamoyltransferase."]},{"entry":[{},"2.3.1.133","Shikimate O-"]},{"entry":[{},{},"hydroxycinnamoyltransferase."]},{"entry":[{},"2.3.1.134","Galactolipid O-acyltransferase."]},{"entry":[{},"2.3.1.135","Phosphatidylcholine--retinol O-"]},{"entry":[{},{},"acyltransferase."]},{"entry":[{},"2.3.1.136","Polysialic-acid O-acetyltransferase."]},{"entry":[{},"2.3.1.137","Carnitine O-octanoyltransferase."]},{"entry":[{},"2.3.1.138","Putrescine N-"]},{"entry":[{},{},"hydroxycinnamoyltransferase."]},{"entry":[{},"2.3.1.139","Ecdysone O-acyltransferase."]},{"entry":[{},"2.3.1.140","Rosmarinate synthase."]},{"entry":[{},"2.3.1.141","Galactosylacylglycerol O-"]},{"entry":[{},{},"acyltransferase."]},{"entry":[{},"2.3.1.142","Glycoprotein O-fatty-"]},{"entry":[{},{},"acyltransferase."]},{"entry":[{},"2.3.1.143","Beta-glucogallin--"]},{"entry":[{},{},"tetrakisgalloylglucose O-galloyltransferase."]},{"entry":[{},"2.3.1.144","Anthranilate N-benzoyltransferase."]},{"entry":[{},"2.3.1.145","Piperidine N-piperoyltransferase."]},{"entry":[{},"2.3.1.146","Pinosylvin synthase."]},{"entry":[{},"2.3.1.147","Glycerophospholipid arachidonoyl-"]},{"entry":[{},{},"transferase (CoA-independent)."]},{"entry":[{},"2.3.1.148","Glycerophospholipid acyltransferase"]},{"entry":[{},{},"(CoA-dependent)."]},{"entry":[{},"2.3.1.149","Platelet-activating factor"]},{"entry":[{},{},"acetyltransferase."]},{"entry":[{},"2.3.1.150","Salutaridinol 7-O-acetyltransferase."]},{"entry":[{},"2.3.1.151","Benzophenone synthase."]},{"entry":[{},"2.3.1.152","Alcohol O-cinnamoyltransferase."]},{"entry":[{},"2.3.1.153","Anthocyanin 5-aromatic"]},{"entry":[{},{},"acyltransferase."]},{"entry":[{},"2.3.1.154","Propionyl-CoA C(2)-"]},{"entry":[{},{},"trimethyltridecanoyltransferase."]},{"entry":[{},"2.3.1.155","Acetyl-CoA C-myristoyltransferase."]},{"entry":[{},"2.3.1.156","Phloroisovalerophenone synthase."]},{"entry":[{},"2.3.1.157","Glucosamine-1-phosphate N-"]},{"entry":[{},{},"acetyltransferase."]},{"entry":[{},"2.3.1.158","Phospholipid:diacylglycerol acyltransferase."]},{"entry":[{},"2.3.1.159","Acridone synthase."]},{"entry":[{},"2.3.1.160","Vinorine synthase."]},{"entry":[{},"2.3.1.161","Lovastatin nonaketide synthase."]},{"entry":[{},"2.3.1.162","Taxadien-5-alpha-ol O-acetyltransferase."]},{"entry":[{},"2.3.1.163","10-hydroxytaxane O-acetyltransferase."]},{"entry":[{},"2.3.1.164","Isopenicillin-N N-acyltransferase."]},{"entry":[{},"2.3.1.165","6-methylsalicylic acid synthase."]},{"entry":[{},"2.3.1.166","2-alpha-hydroxytaxane 2-O-"]},{"entry":[{},{},"benzoyltransferase."]},{"entry":[{},"2.3.1.167","10-deacetylbaccatin III 10-O-"]},{"entry":[{},{},"acetyltransferase."]},{"entry":[{},"2.3.1.168","Dihydrolipoyllysine-residue (2-"]},{"entry":[{},{},"methylpropanoyl)transferase."]},{"entry":[{},"2.3.1.169","CO-methylating acetyl-CoA synthase."]},{"entry":[{},"2.3.2.1","D-glutamyltransferase."]},{"entry":[{},"2.3.2.2","Gamma-glutamyltransferase."]},{"entry":[{},"2.3.2.3","Lysyltransferase."]},{"entry":[{},"2.3.2.4","Gamma-glutamylcyclotransferase."]},{"entry":[{},"2.3.2.5","Glutaminyl-peptide cyclotransferase."]},{"entry":[{},"2.3.2.6","Leucyltransferase."]},{"entry":[{},"2.3.2.7","Aspartyltransferase."]},{"entry":[{},"2.3.2.8","Arginyltransferase."]},{"entry":[{},"2.3.2.9","Agaritine gamma-glutamyltransferase."]},{"entry":[{},"2.3.2.10","UDP-N-acetylmuramoylpentapeptide-lysine"]},{"entry":[{},{},"N(6)-alanyltransferase."]},{"entry":[{},"2.3.2.11","Alanylphosphatidylglycerol synthase."]},{"entry":[{},"2.3.2.12","Peptidyltransferase."]},{"entry":[{},"2.3.2.13","Protein-glutamine gamma-"]},{"entry":[{},{},"glutamyltransferase."]},{"entry":[{},"2.3.2.14","D-alanine gamma-glutamyltransferase."]},{"entry":[{},"2.3.2.15","Glutathione gamma-"]},{"entry":[{},{},"glutamylcysteinyltransferase."]},{"entry":[{},"2.3.3.1","Citrate (Si)-synthase."]},{"entry":[{},"2.3.3.2","Decylcitrate synthase."]},{"entry":[{},"2.3.3.3","Citrate (Re)-synthase."]},{"entry":[{},"2.3.3.4","Decylhomocitrate synthase."]},{"entry":[{},"2.3.3.5","2-methylcitrate synthase."]},{"entry":[{},"2.3.3.6","2-ethylmalate synthase."]},{"entry":[{},"2.3.3.7","3-ethylmalate synthase."]},{"entry":[{},"2.3.3.8","ATP citrate synthase."]},{"entry":[{},"2.3.3.9","Malate synthase."]},{"entry":[{},"2.3.3.10","Hydroxymethylglutaryl-CoA synthase."]},{"entry":[{},"2.3.3.11","2-hydroxyglutarate synthase."]},{"entry":[{},"2.3.3.12","3-propylmalate synthase."]},{"entry":[{},"2.3.3.13","2-isopropylmalate synthase."]},{"entry":[{},"2.3.3.14","Homocitrate synthase."]},{"entry":[{},"2.3.3.15","Sulfoacetaldehyde"]},{"entry":[{},{},"acetyltransferase."]},{"entry":[{},"2.4.1.1","Phosphorylase."]},{"entry":[{},"2.4.1.2","Dextrin dextranase."]},{"entry":[{},"2.4.1.4","Amylosucrase."]},{"entry":[{},"2.4.1.5","Dextransucrase."]},{"entry":[{},"2.4.1.7","Sucrose phosphorylase."]},{"entry":[{},"2.4.1.8","Maltose phosphorylase."]},{"entry":[{},"2.4.1.9","Inulosucrase."]},{"entry":[{},"2.4.1.10","Levansucrase."]},{"entry":[{},"2.4.1.11","Glycogen (starch) synthase."]},{"entry":[{},"2.4.1.12","Cellulose synthase (UDP-forming)."]},{"entry":[{},"2.4.1.13","Sucrose synthase."]},{"entry":[{},"2.4.1.14","Sucrose-phosphate synthase."]},{"entry":[{},"2.4.1.15","Alpha,alpha-trehalose-phosphate"]},{"entry":[{},{},"synthase (UDP-forming)."]},{"entry":[{},"2.4.1.16","Chitin synthase."]},{"entry":[{},"2.4.1.17","Glucuronosyltransferase."]},{"entry":[{},"2.4.1.18","1,4-alpha-glucan branching"]},{"entry":[{},{},"enzyme."]},{"entry":[{},"2.4.1.19","Cyclomaltodextrin"]},{"entry":[{},{},"glucanotransferase."]},{"entry":[{},"2.4.1.20","Cellobiose phosphorylase."]},{"entry":[{},"2.4.1.21","Starch synthase."]},{"entry":[{},"2.4.1.22","Lactose synthase."]},{"entry":[{},"2.4.1.23","Sphingosine beta-"]},{"entry":[{},{},"galactosyltransferase."]},{"entry":[{},"2.4.1.24","1,4-alpha-glucan 6-alpha-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.25","4-alpha-glucanotransferase."]},{"entry":[{},"2.4.1.26","DNA alpha-glucosyltransferase."]},{"entry":[{},"2.4.1.27","DNA beta-glucosyltransferase."]},{"entry":[{},"2.4.1.28","Glucosyl-DNA beta-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.29","Cellulose synthase (GDP-forming)."]},{"entry":[{},"2.4.1.30","1,3-beta-oligoglucan"]},{"entry":[{},{},"phosphorylase."]},{"entry":[{},"2.4.1.31","Laminaribiose phosphorylase."]},{"entry":[{},"2.4.1.32","Glucomannan 4-beta-"]},{"entry":[{},{},"mannosyltransferase."]},{"entry":[{},"2.4.1.33","Alginate synthase."]},{"entry":[{},"2.4.1.34","1,3-beta-glucan synthase."]},{"entry":[{},"2.4.1.35","Phenol beta-glucosyltransferase."]},{"entry":[{},"2.4.1.36","Alpha,alpha-trehalose-phosphate"]},{"entry":[{},{},"synthase (GDP-forming)."]},{"entry":[{},"2.4.1.37","Fucosylgalactoside 3-alpha-"]},{"entry":[{},{},"galactosyltransferase."]},{"entry":[{},"2.4.1.38","Beta-N-"]},{"entry":[{},{},"acetylglucosaminylglycopeptide beta-1,4-"]},{"entry":[{},{},"galactosyltransferase."]},{"entry":[{},"2.4.1.39","Steroid N-"]},{"entry":[{},{},"acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.40","Glycoprotein-fucosylgalactoside"]},{"entry":[{},{},"alpha-N-acetylgalactosaminyltransferase."]},{"entry":[{},"2.4.1.41","Polypeptide N-"]},{"entry":[{},{},"acetylgalactosaminyltransferase."]},{"entry":[{},"2.4.1.43","Polygalacturonate 4-alpha-"]},{"entry":[{},{},"galacturonosyltransferase."]},{"entry":[{},"2.4.1.44","Lipopolysaccharide 3-alpha-"]},{"entry":[{},{},"galactosyltransferase."]},{"entry":[{},"2.4.1.45","2-hydroxyacylsphingosine 1-beta-"]},{"entry":[{},{},"galactosyltransferase."]},{"entry":[{},"2.4.1.46","1,2-diacylglycerol 3-beta-"]},{"entry":[{},{},"galactosyltransferase."]},{"entry":[{},"2.4.1.47","N-acylsphingosine"]},{"entry":[{},{},"galactosyltransferase."]},{"entry":[{},"2.4.1.48","Heteroglycan alpha-"]},{"entry":[{},{},"mannosyltransferase."]},{"entry":[{},"2.4.1.49","Cellodextrin phosphorylase."]},{"entry":[{},"2.4.1.50","Procollagen galactosyltransferase."]},{"entry":[{},"2.4.1.52","Poly(glycerol-phosphate) alpha-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.53","Poly(ribitol-phosphate) beta-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.54","Undecaprenyl-phosphate"]},{"entry":[{},{},"mannosyltransferase."]},{"entry":[{},"2.4.1.56","Lipopolysaccharide N-"]},{"entry":[{},{},"acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.57","Phosphatidylinositol alpha-"]},{"entry":[{},{},"mannosyltransferase."]},{"entry":[{},"2.4.1.58","Lipopolysaccharide"]},{"entry":[{},{},"glucosyltransferase I."]},{"entry":[{},"2.4.1.60","Abequosyltransferase."]},{"entry":[{},"2.4.1.62","Ganglioside galactosyltransferase."]},{"entry":[{},"2.4.1.63","Linamarin synthase."]},{"entry":[{},"2.4.1.64","Alpha,alpha-trehalose phosphorylase."]},{"entry":[{},"2.4.1.65","3-galactosyl-N-acetylglucosaminide"]},{"entry":[{},{},"4-alpha-L-fucosyltransferase."]},{"entry":[{},"2.4.1.66","Procollagen glucosyltransferase."]},{"entry":[{},"2.4.1.67","Galactinol--raffinose"]},{"entry":[{},{},"galactosyltransferase."]},{"entry":[{},"2.4.1.68","Glycoprotein 6-alpha-L-"]},{"entry":[{},{},"fucosyltransferase."]},{"entry":[{},"2.4.1.69","Galactoside 2-alpha-L-"]},{"entry":[{},{},"fucosyltransferase."]},{"entry":[{},"2.4.1.70","Poly(ribitol-phosphate) N-"]},{"entry":[{},{},"acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.71","Arylamine glucosyltransferase."]},{"entry":[{},"2.4.1.73","Lipopolysaccharide glucosyltransferase II."]},{"entry":[{},"2.4.1.74","Glycosaminoglycan galactosyltransferase."]},{"entry":[{},"2.4.1.75","UDP-galacturonosyltransferase."]},{"entry":[{},"2.4.1.78","Phosphopolyprenol glucosyltransferase."]},{"entry":[{},"2.4.1.79","Galactosylgalactosylglucosylceramide"]},{"entry":[{},{},"beta-D-acetylgalactosaminyltransferase."]},{"entry":[{},"2.4.1.80","Ceramide glucosyltransferase."]},{"entry":[{},"2.4.1.81","Flavone 7-O-beta-glucosyltransferase."]},{"entry":[{},"2.4.1.82","Galactinol--sucrose galactosyltransferase."]},{"entry":[{},"2.4.1.83","Dolichyl-phosphate beta-D-"]},{"entry":[{},{},"mannosyltransferase."]},{"entry":[{},"2.4.1.85","Cyanohydrin beta-glucosyltransferase."]},{"entry":[{},"2.4.1.86","Glucosaminylgalactosylglucosylceramide"]},{"entry":[{},{},"beta-galactosyltransferase."]},{"entry":[{},"2.4.1.87","N-acetyllactosaminide 3-alpha-"]},{"entry":[{},{},"galactosyltransferase."]},{"entry":[{},"2.4.1.88","Globoside alpha-N-"]},{"entry":[{},{},"acetylgalactosaminyltransferase."]},{"entry":[{},"2.4.1.90","N-acetyllactosamine synthase."]},{"entry":[{},"2.4.1.91","Flavonol 3-O-glucosyltransferase."]},{"entry":[{},"2.4.1.92","(N-acetylneuraminyl)-"]},{"entry":[{},{},"galactosylglucosylceramide N-"]},{"entry":[{},{},"acetylgalactosaminyltransferase."]},{"entry":[{},"2.4.1.94","Protein N-acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.95","Bilirubin-glucuronoside"]},{"entry":[{},{},"glucuronosyltransferase."]},{"entry":[{},"2.4.1.96","Sn-glycerol-3-phosphate 1-"]},{"entry":[{},{},"galactosyltransferase."]},{"entry":[{},"2.4.1.97","1,3-beta-D-glucan phosphorylase."]},{"entry":[{},"2.4.1.99","Sucrose:sucrose fructosyltransferase."]},{"entry":[{},"2.4.1.100","2,1-fructan:2,1-fructan 1-"]},{"entry":[{},{},"fructosyltransferase."]},{"entry":[{},"2.4.1.101","Alpha-1,3-mannosyl-glycoprotein 2-beta-"]},{"entry":[{},{},"N-acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.102","Beta-1,3-galactosyl-O-glycosyl-"]},{"entry":[{},{},"glycoprotein beta-1,6-N-"]},{"entry":[{},{},"acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.103","Alizarin 2-beta-glucosyltransferase."]},{"entry":[{},"2.4.1.104","O-dihydroxycoumarin 7-O-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.105","Vitexin beta-glucosyltransferase."]},{"entry":[{},"2.4.1.106","Isovitexin beta-glucosyltransferase."]},{"entry":[{},"2.4.1.109","Dolichyl-phosphate-mannose--protein"]},{"entry":[{},{},"mannosyltransferase."]},{"entry":[{},"2.4.1.110","tRNA-queuosine beta-"]},{"entry":[{},{},"mannosyltransferase."]},{"entry":[{},"2.4.1.111","Coniferyl-alcohol glucosyltransferase."]},{"entry":[{},"2.4.1.112","Alpha-1,4-glucan-protein synthase"]},{"entry":[{},{},"(UDP-forming)."]},{"entry":[{},"2.4.1.113","Alpha-1,4-glucan-protein synthase"]},{"entry":[{},{},"(ADP-forming)."]},{"entry":[{},"2.4.1.114","2-coumarate O-beta-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.115","Anthocyanidin 3-O-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.116","Cyanidin-3-rhamnosylglucoside 5-O-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.117","Dolichyl-phosphate beta-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.118","Cytokinin 7-beta-glucosyltransferase."]},{"entry":[{},"2.4.1.119","Dolichyl-diphosphooligosaccharide--"]},{"entry":[{},{},"protein glycotransferase."]},{"entry":[{},"2.4.1.120","Sinapate 1-glucosyltransferase."]},{"entry":[{},"2.4.1.121","Indole-3-acetate beta-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.122","Glycoprotein-N-acetylgalactosamine"]},{"entry":[{},{},"3-beta-galactosyltransferase."]},{"entry":[{},"2.4.1.123","Inositol 3-alpha-galactosyltransferase."]},{"entry":[{},"2.4.1.125","Sucrose--1,6-alpha-glucan 3(6)-alpha-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.126","Hydroxycinnamate 4-beta-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.127","Monoterpenol beta-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.128","Scopoletin glucosyltransferase."]},{"entry":[{},"2.4.1.129","Peptidoglycan glycosyltransferase."]},{"entry":[{},"2.4.1.130","Dolichyl-phosphate-mannose--"]},{"entry":[{},{},"glycolipid alpha-mannosyltransferase."]},{"entry":[{},"2.4.1.131","Glycolipid 2-alpha-"]},{"entry":[{},{},"mannosyltransferase."]},{"entry":[{},"2.4.1.132","Glycolipid 3-alpha-"]},{"entry":[{},{},"mannosyltransferase."]},{"entry":[{},"2.4.1.133","Xylosylprotein 4-beta-"]},{"entry":[{},{},"galactosyltransferase."]},{"entry":[{},"2.4.1.134","Galactosylxylosylprotein 3-beta-"]},{"entry":[{},{},"galactosyltransferase."]},{"entry":[{},"2.4.1.135","Galactosylgalactosylxylosylprotein 3-"]},{"entry":[{},{},"beta-glucuronosyltransferase."]},{"entry":[{},"2.4.1.136","Gallate 1-beta-glucosyltransferase."]},{"entry":[{},"2.4.1.137","Sn-glycerol-3-phosphate 2-alpha-"]},{"entry":[{},{},"galactosyltransferase."]},{"entry":[{},"2.4.1.138","Mannotetraose 2-alpha-N-"]},{"entry":[{},{},"acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.139","Maltose synthase."]},{"entry":[{},"2.4.1.140","Alternansucrase."]},{"entry":[{},"2.4.1.141","N-acetylglucosaminyldiphosphodolichol"]},{"entry":[{},{},"N-acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.142","Chitobiosyldiphosphodolichol beta-"]},{"entry":[{},{},"mannosyltransferase."]},{"entry":[{},"2.4.1.143","Alpha-1,6-mannosyl-glycoprotein 2-"]},{"entry":[{},{},"beta-N-acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.144","Beta-1,4-mannosyl-glycoprotein 4-beta-"]},{"entry":[{},{},"N-acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.145","Alpha-1,3-mannosyl-glycoprotein 4-"]},{"entry":[{},{},"beta-N-acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.146","Beta-1,3-galactosyl-O-glycosyl-"]},{"entry":[{},{},"glycoprotein beta-1,3-N-"]},{"entry":[{},{},"acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.147","Acetylgalactosaminyl-O-glycosyl-"]},{"entry":[{},{},"glycoprotein beta-1,3-N-"]},{"entry":[{},{},"acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.148","Acetylgalactosaminyl-O-glycosyl-"]},{"entry":[{},{},"glycoprotein beta-1,6-N-"]},{"entry":[{},{},"acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.149","N-acetyllactosaminide beta-1,3-N-"]},{"entry":[{},{},"acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.150","N-acetyllactosaminide beta-1,6-N-"]},{"entry":[{},{},"acetylglucosaminyl-transferase."]},{"entry":[{},"2.4.1.152","4-galactosyl-N-acetylglucosaminide 3-"]},{"entry":[{},{},"alpha-L-fucosyltransferase."]},{"entry":[{},"2.4.1.153","Dolichyl-phosphate alpha-N-"]},{"entry":[{},{},"acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.154","Globotriosylceramide beta-1,6-N-"]},{"entry":[{},{},"acetylgalactosaminyl-transferase."]},{"entry":[{},"2.4.1.155","Alpha-1,6-mannosyl-glycoprotein 6-"]},{"entry":[{},{},"beta-N-acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.156","Indolylacetyl-myo-inositol"]},{"entry":[{},{},"galactosyltransferase."]},{"entry":[{},"2.4.1.157","1,2-diacylglycerol 3-glucosyltransferase."]},{"entry":[{},"2.4.1.158","13-hydroxydocosanoate 13-beta-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.159","Flavonol-3-O-glucoside L-"]},{"entry":[{},{},"rhamnosyltransferase."]},{"entry":[{},"2.4.1.160","Pyridoxine 5\u2032-O-beta-D-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.161","Oligosaccharide 4-alpha-D-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.162","Aldose beta-D-fructosyltransferase."]},{"entry":[{},"2.4.1.163","Beta-galactosyl-N-"]},{"entry":[{},{},"acetylglucosaminylgalactosylglucosyl-ceramide beta-"]},{"entry":[{},{},"1,3-acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.164","Galactosyl-N-"]},{"entry":[{},{},"acetylglucosaminylgalactosylglucosyl-ceramide beta-"]},{"entry":[{},{},"1,6-N-acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.165","N-"]},{"entry":[{},{},"acetylneuraminylgalactosylglucosylceramide beta-1,4-"]},{"entry":[{},{},"N-acetylgalactosaminyltransferase."]},{"entry":[{},"2.4.1.166","Raffinose--raffinose alpha-"]},{"entry":[{},{},"galactosyltransferase."]},{"entry":[{},"2.4.1.167","Sucrose 6(F)-alpha-galactosyltransferase."]},{"entry":[{},"2.4.1.168","Xyloglucan 4-glucosyltransferase."]},{"entry":[{},"2.4.1.170","Isoflavone 7-O-glucosyltransferase."]},{"entry":[{},"2.4.1.171","Methyl-ONN-azoxymethanol beta-D-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.172","Salicyl-alcohol beta-D-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.173","Sterol 3-beta-glucosyltransferase."]},{"entry":[{},"2.4.1.174","Glucuronylgalactosylproteoglycan 4-beta-"]},{"entry":[{},{},"N-acetylgalactosaminyltransferase."]},{"entry":[{},"2.4.1.175","Glucuronosyl-N-acetylgalactosaminyl-"]},{"entry":[{},{},"proteoglycan 4-beta-N-"]},{"entry":[{},{},"acetylgalactosaminyltransferase."]},{"entry":[{},"2.4.1.176","Gibberellin beta-D-glucosyltransferase."]},{"entry":[{},"2.4.1.177","Cinnamate beta-D-glucosyltransferase."]},{"entry":[{},"2.4.1.178","Hydroxymandelonitrile"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.179","Lactosylceramide beta-1,3-"]},{"entry":[{},{},"galactosyltransferase."]},{"entry":[{},"2.4.1.180","Lipopolysaccharide N-"]},{"entry":[{},{},"acetylmannosaminouronosyltransferase."]},{"entry":[{},"2.4.1.181","Hydroxyanthraquinone"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.182","Lipid-A-disaccharide synthase."]},{"entry":[{},"2.4.1.183","Alpha-1,3-glucan synthase."]},{"entry":[{},"2.4.1.184","Galactolipid galactosyltransferase."]},{"entry":[{},"2.4.1.185","Flavanone 7-O-beta-glucosyltransferase."]},{"entry":[{},"2.4.1.186","Glycogenin glucosyltransferase."]},{"entry":[{},"2.4.1.187","N-"]},{"entry":[{},{},"acetylglucosaminyldiphosphoundecaprenol N-acetyl-"]},{"entry":[{},{},"beta-D-mannosaminyltransferase."]},{"entry":[{},"2.4.1.188","N-"]},{"entry":[{},{},"acetylglucosaminyldiphosphoundecaprenol"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.189","Luteolin 7-O-glucuronosyltransferase."]},{"entry":[{},"2.4.1.190","Luteolin-7-O-glucuronide 7-O-"]},{"entry":[{},{},"glucuronosyltransferase."]},{"entry":[{},"2.4.1.191","Luteolin-7-O-diglucuronide 4\u2032-O-"]},{"entry":[{},{},"glucuronosyltransferase."]},{"entry":[{},"2.4.1.192","Nuatigenin 3-beta-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.193","Sarsapogenin 3-beta-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.194","4-hydroxybenzoate 4-O-beta-D-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.195","Thiohydroximate beta-D-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.196","Nicotinate glucosyltransferase."]},{"entry":[{},"2.4.1.197","High-mannose-oligosaccharide beta-"]},{"entry":[{},{},"1,4-N-acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.198","Phosphatidylinositol N-"]},{"entry":[{},{},"acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.199","Beta-mannosylphosphodecaprenol-"]},{"entry":[{},{},"mannooligosaccharide 6-mannosyltransferase."]},{"entry":[{},"2.4.1.201","Alpha-1,6-mannosyl-glycoprotein 4-"]},{"entry":[{},{},"beta-N-acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.202","2,4-dihydroxy-7-methoxy-2H-1,4-"]},{"entry":[{},{},"benzoxazin-3(4H)-one 2-D-glucosyltransferase."]},{"entry":[{},"2.4.1.203","Trans-zeatin O-beta-D-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.205","Galactogen 6-beta-"]},{"entry":[{},{},"galactosyltransferase."]},{"entry":[{},"2.4.1.206","Lactosylceramide 1,3-N-acetyl-beta-"]},{"entry":[{},{},"D-glucosaminyltransferase."]},{"entry":[{},"2.4.1.207","Xyloglucan:xyloglucosyl transferase."]},{"entry":[{},"2.4.1.208","Diglucosyl diacylglycerol synthase."]},{"entry":[{},"2.4.1.209","Cis-p-coumarate glucosyltransferase."]},{"entry":[{},"2.4.1.210","Limonoid glucosyltransferase."]},{"entry":[{},"2.4.1.211","1,3-beta-galactosyl-N-"]},{"entry":[{},{},"acetylhexosamine phosphorylase."]},{"entry":[{},"2.4.1.212","Hyaluronan synthase."]},{"entry":[{},"2.4.1.213","Glucosylglycerol-phosphate synthase."]},{"entry":[{},"2.4.1.214","Glycoprotein 3-alpha-L-"]},{"entry":[{},{},"fucosyltransferase."]},{"entry":[{},"2.4.1.215","Cis-zeatin O-beta-D-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"2.4.1.216","Trehalose 6-phosphate"]},{"entry":[{},{},"phosphorylase."]},{"entry":[{},"2.4.1.217","Mannosyl-3-phosphoglycerate"]},{"entry":[{},{},"synthase."]},{"entry":[{},"2.4.1.218","Hydroquinone glucosyltransferase."]},{"entry":[{},"2.4.1.219","Vomilenine glucosyltransferase."]},{"entry":[{},"2.4.1.220","Indoxyl-UDPG glucosyltransferase."]},{"entry":[{},"2.4.1.221","Peptide-O-fucosyltransferase."]},{"entry":[{},"2.4.1.222","O-fucosylpeptide 3-beta-N-"]},{"entry":[{},{},"acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.223","Glucuronyl-galactosyl-proteoglycan"]},{"entry":[{},{},"4-alpha-N-acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.224","Glucuronosyl-N-"]},{"entry":[{},{},"acetylglucosaminyl-proteoglycan 4-alpha-N-"]},{"entry":[{},{},"acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.225","N-acetylglucosaminyl-proteoglycan"]},{"entry":[{},{},"4-beta-glucuronosyltransferase."]},{"entry":[{},"2.4.1.226","N-acetylgalactosaminyl-"]},{"entry":[{},{},"proteoglycan 3-beta-glucuronosyltransferase."]},{"entry":[{},"2.4.1.227","Undecaprenyldiphospho-"]},{"entry":[{},{},"muramoylpentapeptide beta-N-"]},{"entry":[{},{},"acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.228","Lactosylceramide 4-alpha-"]},{"entry":[{},{},"galactosyltransferase."]},{"entry":[{},"2.4.1.229","[Skp1-protein]-hydroxyproline N-"]},{"entry":[{},{},"acetylglucosaminyltransferase."]},{"entry":[{},"2.4.1.230","Kojibiose phosphorylase."]},{"entry":[{},"2.4.1.231","Alpha,alpha-trehalose phosphorylase"]},{"entry":[{},{},"(configuration-retaining)."]},{"entry":[{},"2.4.1.232","Initiation-specific alpha-1,6-"]},{"entry":[{},{},"mannosyltransferase."]},{"entry":[{},"2.4.2.1","Purine-nucleoside phosphorylase."]},{"entry":[{},"2.4.2.2","Pyrimidine-nucleoside phosphorylase."]},{"entry":[{},"2.4.2.3","Uridine phosphorylase."]},{"entry":[{},"2.4.2.4","Thymidine phosphorylase."]},{"entry":[{},"2.4.2.5","Nucleoside ribosyltransferase."]},{"entry":[{},"2.4.2.6","Nucleoside deoxyribosyltransferase."]},{"entry":[{},"2.4.2.7","Adenine phosphoribosyltransferase."]},{"entry":[{},"2.4.2.8","Hypoxanthine"]},{"entry":[{},{},"phosphoribosyltransferase."]},{"entry":[{},"2.4.2.9","Uracil phosphoribosyltransferase."]},{"entry":[{},"2.4.2.10","Orotate phosphoribosyltransferase."]},{"entry":[{},"2.4.2.11","Nicotinate"]},{"entry":[{},{},"phosphoribosyltransferase."]},{"entry":[{},"2.4.2.12","Nicotinamide"]},{"entry":[{},{},"phosphoribosyltransferase."]},{"entry":[{},"2.4.2.14","Amidophosphoribosyltransferase."]},{"entry":[{},"2.4.2.15","Guanosine phosphorylase."]},{"entry":[{},"2.4.2.16","Urate-ribonucleotide phosphorylase."]},{"entry":[{},"2.4.2.17","ATP phosphoribosyltransferase."]},{"entry":[{},"2.4.2.18","Anthranilate"]},{"entry":[{},{},"phosphoribosyltransferase."]},{"entry":[{},"2.4.2.19","Nicotinate-nucleotide diphosphorylase"]},{"entry":[{},{},"(carboxylating)."]},{"entry":[{},"2.4.2.20","Dioxotetrahydropyrimidine"]},{"entry":[{},{},"phosphoribosyltransferase."]},{"entry":[{},"2.4.2.21","Nicotinate-nucleotide--"]},{"entry":[{},{},"dimethylbenzimidazole phosphoribosyltransferase."]},{"entry":[{},"2.4.2.22","Xanthine phosphoribosyltransferase."]},{"entry":[{},"2.4.2.23","Deoxyuridine phosphorylase."]},{"entry":[{},"2.4.2.24","1,4-beta-D-xylan synthase."]},{"entry":[{},"2.4.2.25","Flavone apiosyltransferase."]},{"entry":[{},"2.4.2.26","Protein xylosyltransferase."]},{"entry":[{},"2.4.2.27","dTDP-dihydrostreptose--streptidine-6-"]},{"entry":[{},{},"phosphate dihydrostreptosyltransferase."]},{"entry":[{},"2.4.2.28","S-methyl-5-thioadenosine phosphorylase."]},{"entry":[{},"2.4.2.29","Queuine tRNA-ribosyltransferase."]},{"entry":[{},"2.4.2.30","NAD(+) ADP-ribosyltransferase."]},{"entry":[{},"2.4.2.31","NAD(P)(+)--arginine ADP-"]},{"entry":[{},{},"ribosyltransferase."]},{"entry":[{},"2.4.2.32","Dolichyl-phosphate D-xylosyltransferase."]},{"entry":[{},"2.4.2.33","Dolichyl-xylosyl-phosphate--protein"]},{"entry":[{},{},"xylosyltransferase."]},{"entry":[{},"2.4.2.34","Indolylacetylinositol arabinosyltransferase."]},{"entry":[{},"2.4.2.35","Flavonol-3-O-glycoside xylosyltransferase."]},{"entry":[{},"2.4.2.36","NAD(+)--diphthamide ADP-"]},{"entry":[{},{},"ribosyltransferase."]},{"entry":[{},"2.4.2.37","NAD(+)--dinitrogen-reductase ADP-D-"]},{"entry":[{},{},"ribosyltransferase."]},{"entry":[{},"2.4.2.38","Glycoprotein 2-beta-D-xylosyltransferase."]},{"entry":[{},"2.4.2.39","Xyloglucan 6-xylosyltransferase."]},{"entry":[{},"2.4.2.40","Zeatin O-beta-D-xylosyltransferase."]},{"entry":[{},"2.4.99.1","Beta-galactoside alpha-2,6-sialyltransferase."]},{"entry":[{},"2.4.99.2","Monosialoganglioside sialyltransferase."]},{"entry":[{},"2.4.99.3","Alpha-N-acetylgalactosaminide alpha-2,6-"]},{"entry":[{},{},"sialyltransferase."]},{"entry":[{},"2.4.99.4","Beta-galactoside alpha-2,3-sialyltransferase."]},{"entry":[{},"2.4.99.5","Galactosyldiacylglycerol alpha-2,3-"]},{"entry":[{},{},"sialyltransferase."]},{"entry":[{},"2.4.99.6","N-acetyllactosaminide alpha-2,3-"]},{"entry":[{},{},"sialyltransferase."]},{"entry":[{},"2.4.99.7","(Alpha-N-acetylneuraminyl-2,3-beta-"]},{"entry":[{},{},"galactosyl-1,3)-N-acetyl-galactosaminide 6-alpha-"]},{"entry":[{},{},"sialyltransferase."]},{"entry":[{},"2.4.99.8","Alpha-N-acetylneuraminate alpha-2,8-"]},{"entry":[{},{},"sialyltransferase."]},{"entry":[{},"2.4.99.9","Lactosylceramide alpha-2,3-"]},{"entry":[{},{},"sialyltransferase."]},{"entry":[{},"2.4.99.10","Neolactotetraosylceramide alpha-2,3-"]},{"entry":[{},{},"sialyltransferase."]},{"entry":[{},"2.4.99.11","Lactosylceramide alpha-2,6-N-"]},{"entry":[{},{},"sialyltransferase."]},{"entry":[{},"2.5.1.1","Dimethylallyltranstransferase."]},{"entry":[{},"2.5.1.2","Thiamine pyridinylase."]},{"entry":[{},"2.5.1.3","Thiamine-phosphate diphosphorylase."]},{"entry":[{},"2.5.1.4","Adenosylmethionine cyclotransferase."]},{"entry":[{},"2.5.1.5","Galactose-6-sulfurylase."]},{"entry":[{},"2.5.1.6","Methionine adenosyltransferase."]},{"entry":[{},"2.5.1.7","UDP-N-acetylglucosamine 1-"]},{"entry":[{},{},"carboxyvinyltransferase."]},{"entry":[{},"2.5.1.8","tRNA isopentenyltransferase."]},{"entry":[{},"2.5.1.9","Riboflavin synthase."]},{"entry":[{},"2.5.1.10","Geranyltranstransferase."]},{"entry":[{},"2.5.1.11","Trans-octaprenyltranstransferase."]},{"entry":[{},"2.5.1.15","Dihydropteroate synthase."]},{"entry":[{},"2.5.1.16","Spermidine synthase."]},{"entry":[{},"2.5.1.17","Cob(I)yrinic acid a,c-diamide"]},{"entry":[{},{},"adenosyltransferase."]},{"entry":[{},"2.5.1.18","Glutathione transferase."]},{"entry":[{},"2.5.1.19","3-phosphoshikimate 1-"]},{"entry":[{},{},"carboxyvinyltransferase."]},{"entry":[{},"2.5.1.20","Rubber cis-polyprenylcistransferase."]},{"entry":[{},"2.5.1.21","Farnesyl-diphosphate"]},{"entry":[{},{},"farnesyltransferase."]},{"entry":[{},"2.5.1.22","Spermine synthase."]},{"entry":[{},"2.5.1.23","Sym-norspermidine synthase."]},{"entry":[{},"2.5.1.24","Discadenine synthase."]},{"entry":[{},"2.5.1.25","tRNA-uridine"]},{"entry":[{},{},"aminocarboxypropyltransferase."]},{"entry":[{},"2.5.1.26","Alkylglycerone-phosphate synthase."]},{"entry":[{},"2.5.1.27","Adenylate dimethylallyltransferase."]},{"entry":[{},"2.5.1.28","Dimethylallylcistransferase."]},{"entry":[{},"2.5.1.29","Farnesyltranstransferase."]},{"entry":[{},"2.5.1.30","Trans-hexaprenyltranstransferase."]},{"entry":[{},"2.5.1.31","Di-trans,poly-cis-"]},{"entry":[{},{},"decaprenylcistransferase."]},{"entry":[{},"2.5.1.32","Geranylgeranyl-diphosphate"]},{"entry":[{},{},"geranylgeranyltransferase."]},{"entry":[{},"2.5.1.33","Trans-pentaprenyltranstransferase."]},{"entry":[{},"2.5.1.34","Tryptophan dimethylallyltransferase."]},{"entry":[{},"2.5.1.35","Aspulvinone dimethylallyltransferase."]},{"entry":[{},"2.5.1.36","Trihydroxypterocarpan"]},{"entry":[{},{},"dimethylallyltransferase."]},{"entry":[{},"2.5.1.38","Isonocardicin synthase."]},{"entry":[{},"2.5.1.39","4-hydroxybenzoate"]},{"entry":[{},{},"nonaprenyltransferase."]},{"entry":[{},"2.5.1.41","Phosphoglycerol"]},{"entry":[{},{},"geranylgeranyltransferase."]},{"entry":[{},"2.5.1.42","Geranylgeranylglycerol-phosphate"]},{"entry":[{},{},"geranylgeranyltransferase."]},{"entry":[{},"2.5.1.43","Nicotianamine synthase."]},{"entry":[{},"2.5.1.44","Homospermidine synthase."]},{"entry":[{},"2.5.1.45","Homospermidine synthase (spermidine-"]},{"entry":[{},{},"specific)."]},{"entry":[{},"2.5.1.46","Deoxyhypusine synthase."]},{"entry":[{},"2.5.1.47","Cysteine synthase."]},{"entry":[{},"2.5.1.48","Cystathionine gamma-synthase."]},{"entry":[{},"2.5.1.49","O-acetylhomoserine"]},{"entry":[{},{},"aminocarboxypropyltransferase."]},{"entry":[{},"2.5.1.50","Zeatin 9-aminocarboxyethyltransferase."]},{"entry":[{},"2.5.1.51","Beta-pyrazolylalanine synthase."]},{"entry":[{},"2.5.1.52","L-mimosine synthase."]},{"entry":[{},"2.5.1.53","Uracilylalanine synthase."]},{"entry":[{},"2.5.1.54","3-deoxy-7-phosphoheptulonate synthase."]},{"entry":[{},"2.5.1.55","3-deoxy-8-phosphooctulonate synthase."]},{"entry":[{},"2.5.1.56","N-acetylneuraminate synthase."]},{"entry":[{},"2.5.1.57","N-acylneuraminate-9-phosphate synthase."]},{"entry":[{},"2.5.1.58","Protein farnesyltransferase."]},{"entry":[{},"2.5.1.59","Protein geranylgeranyltransferase type I."]},{"entry":[{},"2.5.1.60","Protein geranylgeranyltransferase type II."]},{"entry":[{},"2.5.1.61","Hydroxymethylbilane synthase."]},{"entry":[{},"2.5.1.62","Chlorophyll synthase."]},{"entry":[{},"2.5.1.63","Adenosyl-fluoride synthase."]},{"entry":[{},"2.5.1.64","2-succinyl-6-hydroxy-2,4-cyclohexadiene-"]},{"entry":[{},{},"1-carboxylate synthase."]},{"entry":[{},"2.6.1.1","Aspartate transaminase."]},{"entry":[{},"2.6.1.2","Alanine transaminase."]},{"entry":[{},"2.6.1.3","Cysteine transaminase."]},{"entry":[{},"2.6.1.4","Glycine transaminase."]},{"entry":[{},"2.6.1.5","Tyrosine transaminase."]},{"entry":[{},"2.6.1.6","Leucine transaminase."]},{"entry":[{},"2.6.1.7","Kynurenine--oxoglutarate transaminase."]},{"entry":[{},"2.6.1.8","2,5-diaminovalerate transaminase."]},{"entry":[{},"2.6.1.9","Histidinol-phosphate transaminase."]},{"entry":[{},"2.6.1.11","Acetylornithine transaminase."]},{"entry":[{},"2.6.1.12","Alanine--oxo-acid transaminase."]},{"entry":[{},"2.6.1.13","Ornithine--oxo-acid transaminase."]},{"entry":[{},"2.6.1.14","Asparagine--oxo-acid transaminase."]},{"entry":[{},"2.6.1.15","Glutamine--pyruvate transaminase."]},{"entry":[{},"2.6.1.16","Glutamine--fructose-6-phosphate"]},{"entry":[{},{},"transaminase (isomerizing)."]},{"entry":[{},"2.6.1.17","Succinyldiaminopimelate transaminase."]},{"entry":[{},"2.6.1.18","Beta-alanine--pyruvate transaminase."]},{"entry":[{},"2.6.1.19","4-aminobutyrate transaminase."]},{"entry":[{},"2.6.1.21","D-alanine transaminase."]},{"entry":[{},"2.6.1.22","(S)-3-amino-2-methylpropionate"]},{"entry":[{},{},"transaminase."]},{"entry":[{},"2.6.1.23","4-hydroxyglutamate transaminase."]},{"entry":[{},"2.6.1.24","Diiodotyrosine transaminase."]},{"entry":[{},"2.6.1.26","Thyroid-hormone transaminase."]},{"entry":[{},"2.6.1.27","Tryptophan transaminase."]},{"entry":[{},"2.6.1.28","Tryptophan--phenylpyruvate"]},{"entry":[{},{},"transaminase."]},{"entry":[{},"2.6.1.29","Diamine transaminase."]},{"entry":[{},"2.6.1.30","Pyridoxamine--pyruvate transaminase."]},{"entry":[{},"2.6.1.31","Pyridoxamine--oxaloacetate"]},{"entry":[{},{},"transaminase."]},{"entry":[{},"2.6.1.32","Valine--3-methyl-2-oxovalerate"]},{"entry":[{},{},"transaminase."]},{"entry":[{},"2.6.1.33","dTDP-4-amino-4,6-dideoxy-D-glucose"]},{"entry":[{},{},"transaminase."]},{"entry":[{},"2.6.1.34","UDP-2-acetamido-4-amino-2,4,6-"]},{"entry":[{},{},"trideoxyglucose transaminase."]},{"entry":[{},"2.6.1.35","Glycine--oxaloacetate transaminase."]},{"entry":[{},"2.6.1.36","L-lysine 6-transaminase."]},{"entry":[{},"2.6.1.37","2-aminoethylphosphonate--pyruvate"]},{"entry":[{},{},"transaminase."]},{"entry":[{},"2.6.1.38","Histidine transaminase."]},{"entry":[{},"2.6.1.39","2-aminoadipate transaminase."]},{"entry":[{},"2.6.1.40","(R)-3-amino-2-methylpropionate--"]},{"entry":[{},{},"pyruvate transaminase."]},{"entry":[{},"2.6.1.41","D-methionine--pyruvate transaminase."]},{"entry":[{},"2.6.1.42","Branched-chain-amino-acid"]},{"entry":[{},{},"transaminase."]},{"entry":[{},"2.6.1.43","Aminolevulinate transaminase."]},{"entry":[{},"2.6.1.44","Alanine--glyoxylate transaminase."]},{"entry":[{},"2.6.1.45","Serine--glyoxylate transaminase."]},{"entry":[{},"2.6.1.46","Diaminobutyrate--pyruvate"]},{"entry":[{},{},"transaminase."]},{"entry":[{},"2.6.1.47","Alanine--oxomalonate transaminase."]},{"entry":[{},"2.6.1.48","5-aminovalerate transaminase."]},{"entry":[{},"2.6.1.49","Dihydroxyphenylalanine transaminase."]},{"entry":[{},"2.6.1.50","Glutamine--scyllo-inositol"]},{"entry":[{},{},"transaminase."]},{"entry":[{},"2.6.1.51","Serine--pyruvate transaminase."]},{"entry":[{},"2.6.1.52","Phosphoserine transaminase."]},{"entry":[{},"2.6.1.54","Pyridoxamine-phosphate transaminase."]},{"entry":[{},"2.6.1.55","Taurine-2-oxoglutarate transaminase."]},{"entry":[{},"2.6.1.56","1D-1-guanidino-3-amino-1,3-dideoxy-"]},{"entry":[{},{},"scyllo-inositol transaminase."]},{"entry":[{},"2.6.1.57","Aromatic-amino-acid transaminase."]},{"entry":[{},"2.6.1.58","Phenylalanine(histidine) transaminase."]},{"entry":[{},"2.6.1.59","dTDP-4-amino-4,6-dideoxygalactose"]},{"entry":[{},{},"transaminase."]},{"entry":[{},"2.6.1.60","Aromatic-amino-acid--glyoxylate"]},{"entry":[{},{},"transaminase."]},{"entry":[{},"2.6.1.62","Adenosylmethionine--8-amino-7-"]},{"entry":[{},{},"oxononanoate transaminase."]},{"entry":[{},"2.6.1.63","Kynurenine--glyoxylate transaminase."]},{"entry":[{},"2.6.1.64","Glutamine--phenylpyruvate transaminase."]},{"entry":[{},"2.6.1.65","N(6)-acetyl-beta-lysine transaminase."]},{"entry":[{},"2.6.1.66","Valine--pyruvate transaminase."]},{"entry":[{},"2.6.1.67","2-aminohexanoate transaminase."]},{"entry":[{},"2.6.1.68","Ornithine(lysine) transaminase."]},{"entry":[{},"2.6.1.70","Aspartate--phenylpyruvate transaminase."]},{"entry":[{},"2.6.1.71","Lysine--pyruvate 6-transaminase."]},{"entry":[{},"2.6.1.72","D-4-hydroxyphenylglycine transaminase."]},{"entry":[{},"2.6.1.73","Methionine--glyoxylate transaminase."]},{"entry":[{},"2.6.1.74","Cephalosporin-C transaminase."]},{"entry":[{},"2.6.1.75","Cysteine-conjugate transaminase."]},{"entry":[{},"2.6.1.76","Diaminobutyrate--2-oxoglutarate"]},{"entry":[{},{},"transaminase."]},{"entry":[{},"2.6.1.77","Taurine--pyruvate aminotransferase."]},{"entry":[{},"2.6.3.1","Oximinotransferase."]},{"entry":[{},"2.6.99.1","dATP(dGTP)--DNA purinetransferase."]},{"entry":[{},"2.7.1.1","Hexokinase."]},{"entry":[{},"2.7.1.2","Glucokinase."]},{"entry":[{},"2.7.1.3","Ketohexokinase."]},{"entry":[{},"2.7.1.4","Fructokinase."]},{"entry":[{},"2.7.1.5","Rhamnulokinase."]},{"entry":[{},"2.7.1.6","Galactokinase."]},{"entry":[{},"2.7.1.7","Mannokinase."]},{"entry":[{},"2.7.1.8","Glucosamine kinase."]},{"entry":[{},"2.7.1.10","Phosphoglucokinase."]},{"entry":[{},"2.7.1.11","6-phosphofructokinase."]},{"entry":[{},"2.7.1.12","Gluconokinase."]},{"entry":[{},"2.7.1.13","Dehydrogluconokinase."]},{"entry":[{},"2.7.1.14","Sedoheptulokinase."]},{"entry":[{},"2.7.1.15","Ribokinase."]},{"entry":[{},"2.7.1.16","Ribulokinase."]},{"entry":[{},"2.7.1.17","Xylulokinase."]},{"entry":[{},"2.7.1.18","Phosphoribokinase."]},{"entry":[{},"2.7.1.19","Phosphoribulokinase."]},{"entry":[{},"2.7.1.20","Adenosine kinase."]},{"entry":[{},"2.7.1.21","Thymidine kinase."]},{"entry":[{},"2.7.1.22","Ribosylnicotinamide kinase."]},{"entry":[{},"2.7.1.23","NAD(+) kinase."]},{"entry":[{},"2.7.1.24","Dephospho-CoA kinase."]},{"entry":[{},"2.7.1.25","Adenylyl-sulfate kinase."]},{"entry":[{},"2.7.1.26","Riboflavin kinase."]},{"entry":[{},"2.7.1.27","Erythritol kinase."]},{"entry":[{},"2.7.1.28","Triokinase."]},{"entry":[{},"2.7.1.29","Glycerone kinase."]},{"entry":[{},"2.7.1.30","Glycerol kinase."]},{"entry":[{},"2.7.1.31","Glycerate kinase."]},{"entry":[{},"2.7.1.32","Choline kinase."]},{"entry":[{},"2.7.1.33","Pantothenate kinase."]},{"entry":[{},"2.7.1.34","Pantetheine kinase."]},{"entry":[{},"2.7.1.35","Pyridoxal kinase."]},{"entry":[{},"2.7.1.36","Mevalonate kinase."]},{"entry":[{},"2.7.1.37","Protein kinase."]},{"entry":[{},"2.7.1.38","Phosphorylase kinase."]},{"entry":[{},"2.7.1.39","Homoserine kinase."]},{"entry":[{},"2.7.1.40","Pyruvate kinase."]},{"entry":[{},"2.7.1.41","Glucose-1-phosphate"]},{"entry":[{},{},"phosphodismutase."]},{"entry":[{},"2.7.1.42","Riboflavin phosphotransferase."]},{"entry":[{},"2.7.1.43","Glucuronokinase."]},{"entry":[{},"2.7.1.44","Galacturonokinase."]},{"entry":[{},"2.7.1.45","2-dehydro-3-deoxygluconokinase."]},{"entry":[{},"2.7.1.46","L-arabinokinase."]},{"entry":[{},"2.7.1.47","D-ribulokinase."]},{"entry":[{},"2.7.1.48","Uridine kinase."]},{"entry":[{},"2.7.1.49","Hydroxymethylpyrimidine kinase."]},{"entry":[{},"2.7.1.50","Hydroxyethylthiazole kinase."]},{"entry":[{},"2.7.1.51","L-fuculokinase."]},{"entry":[{},"2.7.1.52","Fucokinase."]},{"entry":[{},"2.7.1.53","L-xylulokinase."]},{"entry":[{},"2.7.1.54","D-arabinokinase."]},{"entry":[{},"2.7.1.55","Allose kinase."]},{"entry":[{},"2.7.1.56","1-phosphofructokinase."]},{"entry":[{},"2.7.1.58","2-dehydro-3-deoxygalactonokinase."]},{"entry":[{},"2.7.1.59","N-acetylglucosamine kinase."]},{"entry":[{},"2.7.1.60","N-acylmannosamine kinase."]},{"entry":[{},"2.7.1.61","Acyl-phosphate--hexose"]},{"entry":[{},{},"phosphotransferase."]},{"entry":[{},"2.7.1.62","Phosphoramidate--hexose"]},{"entry":[{},{},"phosphotransferase."]},{"entry":[{},"2.7.1.63","Polyphosphate--glucose"]},{"entry":[{},{},"phosphotransferase."]},{"entry":[{},"2.7.1.64","Inositol 3-kinase."]},{"entry":[{},"2.7.1.65","Scyllo-inosamine 4-kinase."]},{"entry":[{},"2.7.1.66","Undecaprenol kinase."]},{"entry":[{},"2.7.1.67","1-phosphatidylinositol 4-kinase."]},{"entry":[{},"2.7.1.68","1-phosphatidylinositol-4-phosphate"]},{"entry":[{},{},"5-kinase."]},{"entry":[{},"2.7.1.69","Protein-N(pi)-phosphohistidine--sugar"]},{"entry":[{},{},"phosphotransferase."]},{"entry":[{},"2.7.1.71","Shikimate kinase."]},{"entry":[{},"2.7.1.72","Streptomycin 6-kinase."]},{"entry":[{},"2.7.1.73","Inosine kinase."]},{"entry":[{},"2.7.1.74","Deoxycytidine kinase."]},{"entry":[{},"2.7.1.76","Deoxyadenosine kinase."]},{"entry":[{},"2.7.1.77","Nucleoside phosphotransferase."]},{"entry":[{},"2.7.1.78","Polynucleotide 5\u2032-hydroxy-kinase."]},{"entry":[{},"2.7.1.79","Diphosphate--glycerol phosphotransferase."]},{"entry":[{},"2.7.1.80","Diphosphate--serine phosphotransferase."]},{"entry":[{},"2.7.1.81","Hydroxylysine kinase."]},{"entry":[{},"2.7.1.82","Ethanolamine kinase."]},{"entry":[{},"2.7.1.83","Pseudouridine kinase."]},{"entry":[{},"2.7.1.84","Alkylglycerone kinase."]},{"entry":[{},"2.7.1.85","Beta-glucoside kinase."]},{"entry":[{},"2.7.1.86","NADH kinase."]},{"entry":[{},"2.7.1.87","Streptomycin 3\u2033-kinase."]},{"entry":[{},"2.7.1.88","Dihydrostreptomycin-6-phosphate 3\u2032-alpha-"]},{"entry":[{},{},"kinase."]},{"entry":[{},"2.7.1.89","Thiamine kinase."]},{"entry":[{},"2.7.1.90","Diphosphate--fructose-6-phosphate 1-"]},{"entry":[{},{},"phosphotransferase."]},{"entry":[{},"2.7.1.91","Sphinganine kinase."]},{"entry":[{},"2.7.1.92","5-dehydro-2-deoxygluconokinase."]},{"entry":[{},"2.7.1.93","Alkylglycerol kinase."]},{"entry":[{},"2.7.1.94","Acylglycerol kinase."]},{"entry":[{},"2.7.1.95","Kanamycin kinase."]},{"entry":[{},"2.7.1.99","[Pyruvate dehydrogenase (lipoamide)] kinase."]},{"entry":[{},"2.7.1.100","S-methyl-5-thioribose kinase."]},{"entry":[{},"2.7.1.101","Tagatose kinase."]},{"entry":[{},"2.7.1.102","Hamamelose kinase."]},{"entry":[{},"2.7.1.103","Viomycin kinase."]},{"entry":[{},"2.7.1.104","Diphosphate--protein phosphotransferase."]},{"entry":[{},"2.7.1.105","6-phosphofructo-2-kinase."]},{"entry":[{},"2.7.1.106","Glucose-1,6-bisphosphate synthase."]},{"entry":[{},"2.7.1.107","Diacylglycerol kinase."]},{"entry":[{},"2.7.1.108","Dolichol kinase."]},{"entry":[{},"2.7.1.109","[Hydroxymethylglutaryl-CoA reductase"]},{"entry":[{},{},"(NADPH)] kinase."]},{"entry":[{},"2.7.1.110","Dephospho-[reductase kinase] kinase."]},{"entry":[{},"2.7.1.112","Protein-tyrosine kinase."]},{"entry":[{},"2.7.1.113","Deoxyguanosine kinase."]},{"entry":[{},"2.7.1.114","AMP--thymidine kinase."]},{"entry":[{},"2.7.1.115","[3-methyl-2-oxobutanoate dehydrogenase"]},{"entry":[{},{},"(lipoamide)] kinase."]},{"entry":[{},"2.7.1.116","[Isocitrate dehydrogenase (NADP+)] kinase."]},{"entry":[{},"2.7.1.117","[Myosin light-chain] kinase."]},{"entry":[{},"2.7.1.118","ADP--thymidine kinase."]},{"entry":[{},"2.7.1.119","Hygromycin-B kinase."]},{"entry":[{},"2.7.1.120","Caldesmon kinase."]},{"entry":[{},"2.7.1.121","Phosphoenolpyruvate--glycerone"]},{"entry":[{},{},"phosphotransferase."]},{"entry":[{},"2.7.1.122","Xylitol kinase."]},{"entry":[{},"2.7.1.123","Calcium\/calmodulin-dependent protein"]},{"entry":[{},{},"kinase."]},{"entry":[{},"2.7.1.124","[Tyrosine 3-monooxygenase] kinase."]},{"entry":[{},"2.7.1.125","Rhodopsin kinase."]},{"entry":[{},"2.7.1.126","[Beta-adrenergic-receptor] kinase."]},{"entry":[{},"2.7.1.127","Inositol-trisphosphate 3-kinase."]},{"entry":[{},"2.7.1.128","[Acetyl-CoA carboxylase] kinase."]},{"entry":[{},"2.7.1.129","[Myosin heavy-chain] kinase."]},{"entry":[{},"2.7.1.130","Tetraacyldisaccharide 4\u2032-kinase."]},{"entry":[{},"2.7.1.131","[Low-density lipoprotein receptor]"]},{"entry":[{},{},"kinase."]},{"entry":[{},"2.7.1.132","Tropomyosin kinase."]},{"entry":[{},"2.7.1.134","Inositol-tetrakisphosphate 1-kinase."]},{"entry":[{},"2.7.1.135","[Tau protein] kinase."]},{"entry":[{},"2.7.1.136","Macrolide 2\u2032-kinase."]},{"entry":[{},"2.7.1.137","Phosphatidylinositol 3-kinase."]},{"entry":[{},"2.7.1.138","Ceramide kinase."]},{"entry":[{},"2.7.1.140","Inositol-tetrakisphosphate 5-kinase."]},{"entry":[{},"2.7.1.141","[RNA-polymerase]-subunit kinase."]},{"entry":[{},"2.7.1.142","Glycerol-3-phosphate--glucose"]},{"entry":[{},{},"phosphotransferase."]},{"entry":[{},"2.7.1.143","Diphosphate-purine nucleoside kinase."]},{"entry":[{},"2.7.1.144","Tagatose-6-phosphate kinase."]},{"entry":[{},"2.7.1.145","Deoxynucleoside kinase."]},{"entry":[{},"2.7.1.146","ADP-specific phosphofructokinase."]},{"entry":[{},"2.7.1.147","ADP-specific glucokinase."]},{"entry":[{},"2.7.1.148","4-(cytidine 5\u2032-diphospho)-2-C-methyl-"]},{"entry":[{},{},"D-erythritol kinase."]},{"entry":[{},"2.7.1.149","1-phosphatidylinositol-5-phosphate 4-"]},{"entry":[{},{},"kinase."]},{"entry":[{},"2.7.1.150","1-phosphatidylinositol-3-phosphate 5-"]},{"entry":[{},{},"kinase."]},{"entry":[{},"2.7.1.151","Inositol-polyphosphate multikinase."]},{"entry":[{},"2.7.1.153","Phosphatidylinositol-4,5-bisphosphate 3-"]},{"entry":[{},{},"kinase."]},{"entry":[{},"2.7.1.154","Phosphatidylinositol-4-phosphate 3-"]},{"entry":[{},{},"kinase."]},{"entry":[{},"2.7.1.155","Diphosphoinositol-pentakisphosphate"]},{"entry":[{},{},"kinase."]},{"entry":[{},"2.7.1.156","Adenosylcobinamide kinase."]},{"entry":[{},"2.7.2.1","Acetate kinase."]},{"entry":[{},"2.7.2.2","Carbamate kinase."]},{"entry":[{},"2.7.2.3","Phosphoglycerate kinase."]},{"entry":[{},"2.7.2.4","Aspartate kinase."]},{"entry":[{},"2.7.2.6","Formate kinase."]},{"entry":[{},"2.7.2.7","Butyrate kinase."]},{"entry":[{},"2.7.2.8","Acetylglutamate kinase."]},{"entry":[{},"2.7.2.10","Phosphoglycerate kinase (GTP)."]},{"entry":[{},"2.7.2.11","Glutamate 5-kinase."]},{"entry":[{},"2.7.2.12","Acetate kinase (diphosphate)."]},{"entry":[{},"2.7.2.13","Glutamate 1-kinase."]},{"entry":[{},"2.7.2.14","Branched-chain-fatty-acid kinase."]},{"entry":[{},"2.7.3.1","Guanidinoacetate kinase."]},{"entry":[{},"2.7.3.2","Creatine kinase."]},{"entry":[{},"2.7.3.3","Arginine kinase."]},{"entry":[{},"2.7.3.4","Taurocyamine kinase."]},{"entry":[{},"2.7.3.5","Lombricine kinase."]},{"entry":[{},"2.7.3.6","Hypotaurocyamine kinase."]},{"entry":[{},"2.7.3.7","Opheline kinase."]},{"entry":[{},"2.7.3.8","Ammonia kinase."]},{"entry":[{},"2.7.3.9","Phosphoenolpyruvate--protein"]},{"entry":[{},{},"phosphotransferase."]},{"entry":[{},"2.7.3.10","Agmatine kinase."]},{"entry":[{},"2.7.3.11","Protein-histidine pros-kinase."]},{"entry":[{},"2.7.3.12","Protein-histidine tele-kinase."]},{"entry":[{},"2.7.4.1","Polyphosphate kinase."]},{"entry":[{},"2.7.4.2","Phosphomevalonate kinase."]},{"entry":[{},"2.7.4.3","Adenylate kinase."]},{"entry":[{},"2.7.4.4","Nucleoside-phosphate kinase."]},{"entry":[{},"2.7.4.6","Nucleoside-diphosphate kinase."]},{"entry":[{},"2.7.4.7","Phosphomethylpyrimidine kinase."]},{"entry":[{},"2.7.4.8","Guanylate kinase."]},{"entry":[{},"2.7.4.9","dTMP kinase."]},{"entry":[{},"2.7.4.10","Nucleoside-triphosphate--adenylate"]},{"entry":[{},{},"kinase."]},{"entry":[{},"2.7.4.11","(Deoxy)adenylate kinase."]},{"entry":[{},"2.7.4.12","T(2)-induced deoxynucleotide kinase."]},{"entry":[{},"2.7.4.13","(Deoxy)nucleoside-phosphate kinase."]},{"entry":[{},"2.7.4.14","Cytidylate kinase."]},{"entry":[{},"2.7.4.15","Thiamine-diphosphate kinase."]},{"entry":[{},"2.7.4.16","Thiamine-phosphate kinase."]},{"entry":[{},"2.7.4.17","3-phosphoglyceroyl-phosphate--"]},{"entry":[{},{},"polyphosphate phosphotransferase."]},{"entry":[{},"2.7.4.18","Farnesyl-diphosphate kinase."]},{"entry":[{},"2.7.4.19","5-methyldeoxycytidine-5\u2032-phosphate"]},{"entry":[{},{},"kinase."]},{"entry":[{},"2.7.4.20","Dolichyl-diphosphate--polyphosphate"]},{"entry":[{},{},"phosphotransferase."]},{"entry":[{},"2.7.4.21","Inositol-hexakisphosphate kinase."]},{"entry":[{},"2.7.6.1","Ribose-phosphate diphosphokinase."]},{"entry":[{},"2.7.6.2","Thiamine diphosphokinase."]},{"entry":[{},"2.7.6.3","2-amino-4-hydroxy-6-"]},{"entry":[{},{},"hydroxymethyldihydropteridine diphosphokinase."]},{"entry":[{},"2.7.6.4","Nucleotide diphosphokinase."]},{"entry":[{},"2.7.6.5","GTP diphosphokinase."]},{"entry":[{},"2.7.7.1","Nicotinamide-nucleotide"]},{"entry":[{},{},"adenylyltransferase."]},{"entry":[{},"2.7.7.2","FMN adenylyltransferase."]},{"entry":[{},"2.7.7.3","Pantetheine-phosphate adenylyltransferase."]},{"entry":[{},"2.7.7.4","Sulfate adenylyltransferase."]},{"entry":[{},"2.7.7.5","Sulfate adenylyltransferase (ADP)."]},{"entry":[{},"2.7.7.6","DNA-directed RNA polymerase."]},{"entry":[{},"2.7.7.7","DNA-directed DNA polymerase."]},{"entry":[{},"2.7.7.8","Polyribonucleotide nucleotidyltransferase."]},{"entry":[{},"2.7.7.9","UTP--glucose-1-phosphate"]},{"entry":[{},{},"uridylyltransferase."]},{"entry":[{},"2.7.7.10","UTP--hexose-1-phosphate"]},{"entry":[{},{},"uridylyltransferase."]},{"entry":[{},"2.7.7.11","UTP--xylose-1-phosphate"]},{"entry":[{},{},"uridylyltransferase."]},{"entry":[{},"2.7.7.12","UDP-glucose--hexose-1-phosphate"]},{"entry":[{},{},"uridylyltransferase."]},{"entry":[{},"2.7.7.13","Mannose-1-phosphate guanylyltransferase."]},{"entry":[{},"2.7.7.14","Ethanolamine-phosphate"]},{"entry":[{},{},"cytidylyltransferase."]},{"entry":[{},"2.7.7.15","Choline-phosphate cytidylyltransferase."]},{"entry":[{},"2.7.7.18","Nicotinate-nucleotide adenylyltransferase."]},{"entry":[{},"2.7.7.19","Polynucleotide adenylyltransferase."]},{"entry":[{},"2.7.7.21","tRNA cytidylyltransferase."]},{"entry":[{},"2.7.7.22","Mannose-1-phosphate guanylyltransferase"]},{"entry":[{},{},"(GDP)."]},{"entry":[{},"2.7.7.23","UDP-N-acetylglucosamine"]},{"entry":[{},{},"diphosphorylase."]},{"entry":[{},"2.7.7.24","Glucose-1-phosphate"]},{"entry":[{},{},"thymidylyltransferase."]},{"entry":[{},"2.7.7.25","tRNA adenylyltransferase."]},{"entry":[{},"2.7.7.27","Glucose-1-phosphate adenylyltransferase."]},{"entry":[{},"2.7.7.28","Nucleoside-triphosphate-aldose 1-"]},{"entry":[{},{},"phosphate nucleotidyltransferase."]},{"entry":[{},"2.7.7.30","Fucose-1-phosphate guanylyltransferase."]},{"entry":[{},"2.7.7.31","DNA nucleotidylexotransferase."]},{"entry":[{},"2.7.7.32","Galactose-1-phosphate"]},{"entry":[{},{},"thymidylyltransferase."]},{"entry":[{},"2.7.7.33","Glucose-1-phosphate cytidylyltransferase."]},{"entry":[{},"2.7.7.34","Glucose-1-phosphate"]},{"entry":[{},{},"guanylyltransferase."]},{"entry":[{},"2.7.7.35","Ribose-5-phosphate adenylyltransferase."]},{"entry":[{},"2.7.7.36","Aldose-1-phosphate adenylyltransferase."]},{"entry":[{},"2.7.7.37","Aldose-1-phosphate"]},{"entry":[{},{},"nucleotidyltransferase."]},{"entry":[{},"2.7.7.38","3-deoxy-manno-octulosonate"]},{"entry":[{},{},"cytidylyltransferase."]},{"entry":[{},"2.7.7.39","Glycerol-3-phosphate"]},{"entry":[{},{},"cytidylyltransferase."]},{"entry":[{},"2.7.7.40","D-ribitol-5-phosphate"]},{"entry":[{},{},"cytidylyltransferase."]},{"entry":[{},"2.7.7.41","Phosphatidate cytidylyltransferase."]},{"entry":[{},"2.7.7.42","[Glutamate--ammonia-ligase]"]},{"entry":[{},{},"adenylyltransferase."]},{"entry":[{},"2.7.7.43","N-acylneuraminate cytidylyltransferase."]},{"entry":[{},"2.7.7.44","Glucuronate-1-phosphate"]},{"entry":[{},{},"uridylyltransferase."]},{"entry":[{},"2.7.7.45","Guanosine-triphosphate"]},{"entry":[{},{},"guanylyltransferase."]},{"entry":[{},"2.7.7.46","Gentamicin 2\u2033-nucleotidyltransferase."]},{"entry":[{},"2.7.7.47","Streptomycin 3\u2033-adenylyltransferase."]},{"entry":[{},"2.7.7.48","RNA-directed RNA polymerase."]},{"entry":[{},"2.7.7.49","RNA-directed DNA polymerase."]},{"entry":[{},"2.7.7.50","mRNA guanylyltransferase."]},{"entry":[{},"2.7.7.51","Adenylylsulfate--ammonia"]},{"entry":[{},{},"adenylyltransferase."]},{"entry":[{},"2.7.7.52","RNA uridylyltransferase."]},{"entry":[{},"2.7.7.53","ATP adenylyltransferase."]},{"entry":[{},"2.7.7.54","Phenylalanine adenylyltransferase."]},{"entry":[{},"2.7.7.55","Anthranilate adenylyltransferase."]},{"entry":[{},"2.7.7.56","tRNA nucleotidyltransferase."]},{"entry":[{},"2.7.7.57","N-methylphosphoethanolamine"]},{"entry":[{},{},"cytidylyltransferase."]},{"entry":[{},"2.7.7.58","(2,3-dihydroxybenzoyl)adenylate"]},{"entry":[{},{},"synthase."]},{"entry":[{},"2.7.7.59","[Protein-PII] uridylyltransferase."]},{"entry":[{},"2.7.7.60","2-C-methyl-D-erythritol 4-phosphate"]},{"entry":[{},{},"cytidylyltransferase."]},{"entry":[{},"2.7.7.61","Holo-ACP synthase."]},{"entry":[{},"2.7.7.62","Adenosylcobinamide-phosphate"]},{"entry":[{},{},"guanylyltransferase."]},{"entry":[{},"2.7.8.1","Ethanolaminephosphotransferase."]},{"entry":[{},"2.7.8.2","Diacylglycerol"]},{"entry":[{},{},"cholinephosphotransferase."]},{"entry":[{},"2.7.8.3","Ceramide cholinephosphotransferase."]},{"entry":[{},"2.7.8.4","Serine-phosphoethanolamine synthase."]},{"entry":[{},"2.7.8.5","CDP-diacylglycerol--glycerol-3-phosphate"]},{"entry":[{},{},"3-phosphatidyltransferase."]},{"entry":[{},"2.7.8.6","Undecaprenyl-phosphate galactose"]},{"entry":[{},{},"phosphotransferase."]},{"entry":[{},"2.7.8.7","Holo-[acyl-carrier-protein] synthase."]},{"entry":[{},"2.7.8.8","CDP-diacylglycerol--serine O-"]},{"entry":[{},{},"phosphatidyltransferase."]},{"entry":[{},"2.7.8.9","Phosphomannan"]},{"entry":[{},{},"mannosephosphotransferase."]},{"entry":[{},"2.7.8.10","Sphingosine cholinephosphotransferase."]},{"entry":[{},"2.7.8.11","CDP-diacylglycerol--inositol 3-"]},{"entry":[{},{},"phosphatidyltransferase."]},{"entry":[{},"2.7.8.12","CDP-glycerol glycerophosphotransferase."]},{"entry":[{},"2.7.8.13","Phospho-N-acetylmuramoyl-pentapeptide-"]},{"entry":[{},{},"transferase."]},{"entry":[{},"2.7.8.14","CDP-ribitol ribitolphosphotransferase."]},{"entry":[{},"2.7.8.15","UDP-N-acetylglucosamine--dolichyl-"]},{"entry":[{},{},"phosphate N-acetylglucosaminephosphotransferase."]},{"entry":[{},"2.7.8.17","UDP-N-acetylglucosamine--lysosomal-"]},{"entry":[{},{},"enzyme N-acetylglucosaminephosphotransferase."]},{"entry":[{},"2.7.8.18","UDP-galactose--UDP-N-acetylglucosamine"]},{"entry":[{},{},"galactose phosphotransferase."]},{"entry":[{},"2.7.8.19","UDP-glucose--glycoprotein glucose"]},{"entry":[{},{},"phosphotransferase."]},{"entry":[{},"2.7.8.20","Phosphatidylglycerol--membrane-"]},{"entry":[{},{},"oligosaccharide glycerophosphotransferase."]},{"entry":[{},"2.7.8.21","Membrane-oligosaccharide"]},{"entry":[{},{},"glycerophosphotransferase."]},{"entry":[{},"2.7.8.22","1-alkenyl-2-acylglycerol choline"]},{"entry":[{},{},"phosphotransferase."]},{"entry":[{},"2.7.8.23","Carboxyvinyl-carboxyphosphonate"]},{"entry":[{},{},"phosphorylmutase."]},{"entry":[{},"2.7.8.24","Phosphatidylcholine synthase."]},{"entry":[{},"2.7.8.25","Triphosphoribosyl-dephospho-CoA"]},{"entry":[{},{},"synthase."]},{"entry":[{},"2.7.8.26","Adenosylcobinamide-GDP"]},{"entry":[{},{},"ribazoletransferase."]},{"entry":[{},"2.7.9.1","Pyruvate, phosphate dikinase."]},{"entry":[{},"2.7.9.2","Pyruvate, water dikinase."]},{"entry":[{},"2.7.9.3","Selenide, water dikinase."]},{"entry":[{},"2.7.9.4","Alpha-glucan, water dikinase."]},{"entry":[{},"2.8.1.1","Thiosulfate sulfur-transferase."]},{"entry":[{},"2.8.1.2","3-mercaptopyruvate sulfur-transferase."]},{"entry":[{},"2.8.1.3","Thiosulfate--thiol sulfur-transferase."]},{"entry":[{},"2.8.1.4","tRNA sulfur-transferase."]},{"entry":[{},"2.8.1.5","Thiosulfate--dithiol sulfur-transferase."]},{"entry":[{},"2.8.1.6","Biotin synthase."]},{"entry":[{},"2.8.1.7","Cysteine desulfurase."]},{"entry":[{},"2.8.2.1","Aryl sulfotransferase."]},{"entry":[{},"2.8.2.2","Alcohol sulfotransferase."]},{"entry":[{},"2.8.2.3","Amine sulfotransferase."]},{"entry":[{},"2.8.2.4","Estrone sulfotransferase."]},{"entry":[{},"2.8.2.5","Chondroitin 4-sulfotransferase."]},{"entry":[{},"2.8.2.6","Choline sulfotransferase."]},{"entry":[{},"2.8.2.7","UDP-N-acetylgalactosamine-4-sulfate"]},{"entry":[{},{},"sulfotransferase."]},{"entry":[{},"2.8.2.8","[Heparan sulfate]-glucosamine N-"]},{"entry":[{},{},"sulfotransferase."]},{"entry":[{},"2.8.2.9","Tyrosine-ester sulfotransferase."]},{"entry":[{},"2.8.2.10","Renilla-luciferin sulfotransferase."]},{"entry":[{},"2.8.2.11","Galactosylceramide sulfotransferase."]},{"entry":[{},"2.8.2.13","Psychosine sulfotransferase."]},{"entry":[{},"2.8.2.14","Bile-salt sulfotransferase."]},{"entry":[{},"2.8.2.15","Steroid sulfotransferase."]},{"entry":[{},"2.8.2.16","Thiol sulfotransferase."]},{"entry":[{},"2.8.2.17","Chondroitin 6-sulfotransferase."]},{"entry":[{},"2.8.2.18","Cortisol sulfotransferase."]},{"entry":[{},"2.8.2.19","Triglucosylalkylacylglycerol"]},{"entry":[{},{},"sulfotransferase."]},{"entry":[{},"2.8.2.20","Protein-tyrosine sulfotransferase."]},{"entry":[{},"2.8.2.21","Keratan sulfotransferase."]},{"entry":[{},"2.8.2.22","Arylsulfate sulfotransferase."]},{"entry":[{},"2.8.2.23","[Heparan sulfate]-glucosamine 3-"]},{"entry":[{},{},"sulfotransferase 1."]},{"entry":[{},"2.8.2.24","Desulfoglucosinolate sulfotransferase."]},{"entry":[{},"2.8.2.25","Flavonol 3-sulfotransferase."]},{"entry":[{},"2.8.2.26","Quercetin-3-sulfate 3\u2032-sulfotransferase."]},{"entry":[{},"2.8.2.27","Quercetin-3-sulfate 4\u2032-sulfotransferase."]},{"entry":[{},"2.8.2.28","Quercetin-3,3\u2032-bissulfate 7-"]},{"entry":[{},{},"sulfotransferase."]},{"entry":[{},"2.8.2.29","[Heparan sulfate]-glucosamine 3-"]},{"entry":[{},{},"sulfotransferase 2."]},{"entry":[{},"2.8.2.30","[Heparan sulfate]-glucosamine 3-"]},{"entry":[{},{},"sulfotransferase 3."]},{"entry":[{},"2.8.3.1","Propionate CoA-transferase."]},{"entry":[{},"2.8.3.2","Oxalate CoA-transferase."]},{"entry":[{},"2.8.3.3","Malonate CoA-transferase."]},{"entry":[{},"2.8.3.5","3-oxoacid CoA-transferase."]},{"entry":[{},"2.8.3.6","3-oxoadipate CoA-transferase."]},{"entry":[{},"2.8.3.7","Succinate--citramalate CoA-transferase."]},{"entry":[{},"2.8.3.8","Acetate CoA-transferase."]},{"entry":[{},"2.8.3.9","Butyrate--acetoacetate CoA-transferase."]},{"entry":[{},"2.8.3.10","Citrate CoA-transferase."]},{"entry":[{},"2.8.3.11","Citramalate CoA-transferase."]},{"entry":[{},"2.8.3.12","Glutaconate CoA-transferase."]},{"entry":[{},"2.8.3.13","Succinate--hydroxymethylglutarate CoA-"]},{"entry":[{},{},"transferase."]},{"entry":[{},"2.8.3.14","5-hydroxypentanoate CoA-transferase."]},{"entry":[{},"2.8.3.15","Succinyl-CoA:(R)-benzylsuccinate CoA-"]},{"entry":[{},{},"transferase."]},{"entry":[{},"2.8.3.16","Formyl-CoA transferase."]},{"entry":[{},"2.8.3.17","Cinnamoyl-CoA:phenyllactate CoA-"]},{"entry":[{},{},"transferase."]},{"entry":[{},"2.8.4.1","Coenzyme-B sulfoethylthiotransferase."]},{"entry":[{},"2.9.1.1","L-seryl-tRNA(Sec) selenium transferase."]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"ENZYME: 3.\u2014.\u2014.\u2014"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"168pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"3.1.1.1","Carboxylesterase."]},{"entry":[{},"3.1.1.2","Arylesterase."]},{"entry":[{},"3.1.1.3","Triacylglycerol lipase."]},{"entry":[{},"3.1.1.4","Phospholipase A(2)."]},{"entry":[{},"3.1.1.5","Lysophospholipase."]},{"entry":[{},"3.1.1.6","Acetylesterase."]},{"entry":[{},"3.1.1.7","Acetylcholinesterase."]},{"entry":[{},"3.1.1.8","Cholinesterase."]},{"entry":[{},"3.1.1.10","Tropinesterase."]},{"entry":[{},"3.1.1.11","Pectinesterase."]},{"entry":[{},"3.1.1.13","Sterol esterase."]},{"entry":[{},"3.1.1.14","Chlorophyllase."]},{"entry":[{},"3.1.1.15","L-arabinonolactonase."]},{"entry":[{},"3.1.1.17","Gluconolactonase."]},{"entry":[{},"3.1.1.19","Uronolactonase."]},{"entry":[{},"3.1.1.20","Tannase."]},{"entry":[{},"3.1.1.21","Retinyl-palmitate esterase."]},{"entry":[{},"3.1.1.22","Hydroxybutyrate-dimer hydrolase."]},{"entry":[{},"3.1.1.23","Acylglycerol lipase."]},{"entry":[{},"3.1.1.24","3-oxoadipate enol-lactonase."]},{"entry":[{},"3.1.1.25","1,4-lactonase."]},{"entry":[{},"3.1.1.26","Galactolipase."]},{"entry":[{},"3.1.1.27","4-pyridoxolactonase."]},{"entry":[{},"3.1.1.28","Acylcarnitine hydrolase."]},{"entry":[{},"3.1.1.29","Aminoacyl-tRNA hydrolase."]},{"entry":[{},"3.1.1.30","D-arabinonolactonase."]},{"entry":[{},"3.1.1.31","6-phosphogluconolactonase."]},{"entry":[{},"3.1.1.32","Phospholipase A(1)."]},{"entry":[{},"3.1.1.33","6-acetylglucose deacetylase."]},{"entry":[{},"3.1.1.34","Lipoprotein lipase."]},{"entry":[{},"3.1.1.35","Dihydrocoumarin hydrolase."]},{"entry":[{},"3.1.1.36","Limonin-D-ring-lactonase."]},{"entry":[{},"3.1.1.37","Steroid-lactonase."]},{"entry":[{},"3.1.1.38","Triacetate-lactonase."]},{"entry":[{},"3.1.1.39","Actinomycin lactonase."]},{"entry":[{},"3.1.1.40","Orsellinate-depside hydrolase."]},{"entry":[{},"3.1.1.41","Cephalosporin-C deacetylase."]},{"entry":[{},"3.1.1.42","Chlorogenate hydrolase."]},{"entry":[{},"3.1.1.43","Alpha-amino-acid esterase."]},{"entry":[{},"3.1.1.44","4-methyloxaloacetate esterase."]},{"entry":[{},"3.1.1.45","Carboxymethylenebutenolidase."]},{"entry":[{},"3.1.1.46","Deoxylimonate A-ring-lactonase."]},{"entry":[{},"3.1.1.47","1-alkyl-2-acetylglycerophosphocholine"]},{"entry":[{},{},"esterase."]},{"entry":[{},"3.1.1.48","Fusarinine-C ornithinesterase."]},{"entry":[{},"3.1.1.49","Sinapine esterase."]},{"entry":[{},"3.1.1.50","Wax-ester hydrolase."]},{"entry":[{},"3.1.1.51","Phorbol-diester hydrolase."]},{"entry":[{},"3.1.1.52","Phosphatidylinositol deacylase."]},{"entry":[{},"3.1.1.53","Sialate O-acetylesterase."]},{"entry":[{},"3.1.1.54","Acetoxybutynylbithiophene deacetylase."]},{"entry":[{},"3.1.1.55","Acetylsalicylate deacetylase."]},{"entry":[{},"3.1.1.56","Methylumbelliferyl-acetate deacetylase."]},{"entry":[{},"3.1.1.57","2-pyrone-4,6-dicarboxylate lactonase."]},{"entry":[{},"3.1.1.58","N-acetylgalactosaminoglycan"]},{"entry":[{},{},"deacetylase."]},{"entry":[{},"3.1.1.59","Juvenile-hormone esterase."]},{"entry":[{},"3.1.1.60","Bis(2-ethylhexyl)phthalate esterase."]},{"entry":[{},"3.1.1.61","Protein-glutamate methylesterase."]},{"entry":[{},"3.1.1.63","11-cis-retinyl-palmitate hydrolase."]},{"entry":[{},"3.1.1.64","All-trans-retinyl-palmitate hydrolase."]},{"entry":[{},"3.1.1.65","L-rhamnono-1,4-lactonase."]},{"entry":[{},"3.1.1.66","5-(3,4-diacetoxybut-1-ynyl)-2,2\u2032-"]},{"entry":[{},{},"bithiophene deacetylase."]},{"entry":[{},"3.1.1.67","Fatty-acyl-ethyl-ester synthase."]},{"entry":[{},"3.1.1.68","Xylono-1,4-lactonase."]},{"entry":[{},"3.1.1.70","Cetraxate benzylesterase."]},{"entry":[{},"3.1.1.71","Acetylalkylglycerol acetylhydrolase."]},{"entry":[{},"3.1.1.72","Acetylxylan esterase."]},{"entry":[{},"3.1.1.73","Feruloyl esterase."]},{"entry":[{},"3.1.1.74","Cutinase."]},{"entry":[{},"3.1.1.75","Poly(3-hydroxybutyrate) depolymerase."]},{"entry":[{},"3.1.1.76","Poly(3-hydroxyoctanoate)"]},{"entry":[{},{},"depolymerase."]},{"entry":[{},"3.1.1.77","Acyloxyacyl hydrolase."]},{"entry":[{},"3.1.1.78","Polyneuridine-aldehyde esterase."]},{"entry":[{},"3.1.1.79","Hormone-sensitive lipase."]},{"entry":[{},"3.1.2.1","Acetyl-CoA hydrolase."]},{"entry":[{},"3.1.2.2","Palmitoyl-CoA hydrolase."]},{"entry":[{},"3.1.2.3","Succinyl-CoA hydrolase."]},{"entry":[{},"3.1.2.4","3-hydroxyisobutyryl-CoA hydrolase."]},{"entry":[{},"3.1.2.5","Hydroxymethylglutaryl-CoA hydrolase."]},{"entry":[{},"3.1.2.6","Hydroxyacylglutathione hydrolase."]},{"entry":[{},"3.1.2.7","Glutathione thiolesterase."]},{"entry":[{},"3.1.2.10","Formyl-CoA hydrolase."]},{"entry":[{},"3.1.2.11","Acetoacetyl-CoA hydrolase."]},{"entry":[{},"3.1.2.12","S-formylglutathione hydrolase."]},{"entry":[{},"3.1.2.13","S-succinylglutathione hydrolase."]},{"entry":[{},"3.1.2.14","Oleoyl-[acyl-carrier-protein] hydrolase."]},{"entry":[{},"3.1.2.15","Ubiquitin thiolesterase."]},{"entry":[{},"3.1.2.16","[Citrate-(pro-3S)-lyase] thiolesterase."]},{"entry":[{},"3.1.2.17","(S)-methylmalonyl-CoA hydrolase."]},{"entry":[{},"3.1.2.18","ADP-dependent short-chain-acyl-CoA"]},{"entry":[{},{},"hydrolase."]},{"entry":[{},"3.1.2.19","ADP-dependent medium-chain-acyl-CoA"]},{"entry":[{},{},"hydrolase."]},{"entry":[{},"3.1.2.20","Acyl-CoA hydrolase."]},{"entry":[{},"3.1.2.21","Dodecanoyl-[acyl-carrier protein]"]},{"entry":[{},{},"hydrolase."]},{"entry":[{},"3.1.2.22","Palmitoyl-protein hydrolase."]},{"entry":[{},"3.1.2.23","4-hydroxybenzoyl-CoA thioesterase."]},{"entry":[{},"3.1.2.24","2-(2-hydroxyphenyl)benzenesulfinate"]},{"entry":[{},{},"hydrolase."]},{"entry":[{},"3.1.2.25","Phenylacetyl-CoA hydrolase."]},{"entry":[{},"3.1.3.1","Alkaline phosphatase."]},{"entry":[{},"3.1.3.2","Acid phosphatase."]},{"entry":[{},"3.1.3.3","Phosphoserine phosphatase."]},{"entry":[{},"3.1.3.4","Phosphatidate phosphatase."]},{"entry":[{},"3.1.3.5","5\u2032-nucleotidase."]},{"entry":[{},"3.1.3.6","3\u2032-nucleotidase."]},{"entry":[{},"3.1.3.7","3\u2032(2\u2032),5\u2032-bisphosphate nucleotidase."]},{"entry":[{},"3.1.3.8","3-phytase."]},{"entry":[{},"3.1.3.9","Glucose-6-phosphatase."]},{"entry":[{},"3.1.3.10","Glucose-1-phosphatase."]},{"entry":[{},"3.1.3.11","Fructose-bisphosphatase."]},{"entry":[{},"3.1.3.12","Trehalose-phosphatase."]},{"entry":[{},"3.1.3.13","Bisphosphoglycerate phosphatase."]},{"entry":[{},"3.1.3.14","Methylphosphothioglycerate phosphatase."]},{"entry":[{},"3.1.3.15","Histidinol-phosphatase."]},{"entry":[{},"3.1.3.16","Phosphoprotein phosphatase."]},{"entry":[{},"3.1.3.17","[Phosphorylase] phosphatase."]},{"entry":[{},"3.1.3.18","Phosphoglycolate phosphatase."]},{"entry":[{},"3.1.3.19","Glycerol-2-phosphatase."]},{"entry":[{},"3.1.3.20","Phosphoglycerate phosphatase."]},{"entry":[{},"3.1.3.21","Glycerol-1-phosphatase."]},{"entry":[{},"3.1.3.22","Mannitol-1-phosphatase."]},{"entry":[{},"3.1.3.23","Sugar-phosphatase."]},{"entry":[{},"3.1.3.24","Sucrose-phosphatase."]},{"entry":[{},"3.1.3.25","Inositol-1(or 4)-monophosphatase."]},{"entry":[{},"3.1.3.26","4-phytase."]},{"entry":[{},"3.1.3.27","Phosphatidylglycerophosphatase."]},{"entry":[{},"3.1.3.28","ADP-phosphoglycerate phosphatase."]},{"entry":[{},"3.1.3.29","N-acylneuraminate-9-phosphatase."]},{"entry":[{},"3.1.3.31","Nucleotidase."]},{"entry":[{},"3.1.3.32","Polynucleotide 3\u2032-phosphatase."]},{"entry":[{},"3.1.3.33","Polynucleotide 5\u2032-phosphatase."]},{"entry":[{},"3.1.3.34","Deoxynucleotide 3\u2032-phosphatase."]},{"entry":[{},"3.1.3.35","Thymidylate 5\u2032-phosphatase."]},{"entry":[{},"3.1.3.36","Phosphoinositide 5-phosphatase."]},{"entry":[{},"3.1.3.37","Sedoheptulose-bisphosphatase."]},{"entry":[{},"3.1.3.38","3-phosphoglycerate phosphatase."]},{"entry":[{},"3.1.3.39","Streptomycin-6-phosphatase."]},{"entry":[{},"3.1.3.40","Guanidinodeoxy-scyllo-inositol-4-"]},{"entry":[{},{},"phosphatase."]},{"entry":[{},"3.1.3.41","4-nitrophenylphosphatase."]},{"entry":[{},"3.1.3.42","[Glycogen-synthase-D] phosphatase."]},{"entry":[{},"3.1.3.43","[Pyruvate dehydrogenase (lipoamide)]-"]},{"entry":[{},{},"phosphatase."]},{"entry":[{},"3.1.3.44","[Acetyl-CoA carboxylase]-"]},{"entry":[{},{},"phosphatase."]},{"entry":[{},"3.1.3.45","3-deoxy-manno-octulosonate-8-"]},{"entry":[{},{},"phosphatase."]},{"entry":[{},"3.1.3.46","Fructose-2,6-bisphosphate 2-"]},{"entry":[{},{},"phosphatase."]},{"entry":[{},"3.1.3.47","[Hydroxymethylglutaryl-CoA"]},{"entry":[{},{},"reductase (NADPH)]-phosphatase."]},{"entry":[{},"3.1.3.48","Protein-tyrosine-phosphatase."]},{"entry":[{},"3.1.3.49","[Pyruvate kinase]-phosphatase."]},{"entry":[{},"3.1.3.50","Sorbitol-6-phosphatase."]},{"entry":[{},"3.1.3.51","Dolichyl-phosphatase."]},{"entry":[{},"3.1.3.52","[3-methyl-2-oxobutanoate"]},{"entry":[{},{},"dehydrogenase (lipoamide)]-phosphatase."]},{"entry":[{},"3.1.3.53","[Myosin light-chain]-phosphatase."]},{"entry":[{},"3.1.3.54","Fructose-2,6-bisphosphate 6-"]},{"entry":[{},{},"phosphatase."]},{"entry":[{},"3.1.3.55","Caldesmon-phosphatase."]},{"entry":[{},"3.1.3.56","Inositol-polyphosphate 5-phosphatase."]},{"entry":[{},"3.1.3.57","Inositol-1,4-bisphosphate 1-"]},{"entry":[{},{},"phosphatase."]},{"entry":[{},"3.1.3.58","Sugar-terminal-phosphatase."]},{"entry":[{},"3.1.3.59","Alkylacetylglycerophosphatase."]},{"entry":[{},"3.1.3.60","Phosphoenolpyruvate phosphatase."]},{"entry":[{},"3.1.3.62","Multiple inositol-polyphosphate"]},{"entry":[{},{},"phosphatase."]},{"entry":[{},"3.1.3.63","2-carboxy-D-arabinitol-1-phosphatase."]},{"entry":[{},"3.1.3.64","Phosphatidylinositol-3-phosphatase."]},{"entry":[{},"3.1.3.66","Phosphatidylinositol-3,4-bisphosphate"]},{"entry":[{},{},"4-phosphatase."]},{"entry":[{},"3.1.3.67","Phosphatidylinositol-3,4,5-"]},{"entry":[{},{},"trisphosphate 3-phosphatase."]},{"entry":[{},"3.1.3.68","2-deoxyglucose-6-phosphatase."]},{"entry":[{},"3.1.3.69","Glucosylglycerol 3-phosphatase."]},{"entry":[{},"3.1.3.70","Mannosyl-3-phosphoglycerate"]},{"entry":[{},{},"phosphatase."]},{"entry":[{},"3.1.3.71","2-phosphosulfolactate phosphatase."]},{"entry":[{},"3.1.3.72","5-phytase."]},{"entry":[{},"3.1.3.73","Alpha-ribazole phosphatase."]},{"entry":[{},"3.1.4.1","Phosphodiesterase I."]},{"entry":[{},"3.1.4.2","Glycerophosphocholine phosphodiesterase."]},{"entry":[{},"3.1.4.3","Phospholipase C."]},{"entry":[{},"3.1.4.4","Phospholipase D."]},{"entry":[{},"3.1.4.11","Phosphoinositide phospholipase C."]},{"entry":[{},"3.1.4.12","Sphingomyelin phosphodiesterase."]},{"entry":[{},"3.1.4.13","Serine-ethanolaminephosphate"]},{"entry":[{},{},"phosphodiesterase."]},{"entry":[{},"3.1.4.14","[Acyl-carrier-protein] phosphodiesterase."]},{"entry":[{},"3.1.4.15","Adenylyl-[glutamate--ammonia ligase]"]},{"entry":[{},{},"hydrolase."]},{"entry":[{},"3.1.4.16","2\u2032,3\u2032-cyclic-nucleotide 2\u2032-"]},{"entry":[{},{},"phosphodiesterase."]},{"entry":[{},"3.1.4.17","3\u2032,5\u2032-cyclic-nucleotide phosphodiesterase."]},{"entry":[{},"3.1.4.35","3\u2032,5\u2032-cyclic-GMP phosphodiesterase."]},{"entry":[{},"3.1.4.37","2\u2032,3\u2032-cyclic-nucleotide 3\u2032-"]},{"entry":[{},{},"phosphodiesterase."]},{"entry":[{},"3.1.4.38","Glycerophosphocholine"]},{"entry":[{},{},"cholinephosphodiesterase."]},{"entry":[{},"3.1.4.39","Alkylglycerophosphoethanolamine"]},{"entry":[{},{},"phosphodiesterase."]},{"entry":[{},"3.1.4.40","CMP-N-acylneuraminate"]},{"entry":[{},{},"phosphodiesterase."]},{"entry":[{},"3.1.4.41","Sphingomyelin phosphodiesterase D."]},{"entry":[{},"3.1.4.42","Glycerol-1,2-cyclic-phosphate 2-"]},{"entry":[{},{},"phosphodiesterase."]},{"entry":[{},"3.1.4.43","Glycerophosphoinositol"]},{"entry":[{},{},"inositolphosphodiesterase."]},{"entry":[{},"3.1.4.44","Glycerophosphoinositol"]},{"entry":[{},{},"glycerophosphodiesterase."]},{"entry":[{},"3.1.4.45","N-acetylglucosamine-1-phosphodiester"]},{"entry":[{},{},"alpha-N-acetylglucosaminidase."]},{"entry":[{},"3.1.4.46","Glycerophosphodiester phosphodiesterase."]},{"entry":[{},"3.1.4.48","Dolichylphosphate-glucose"]},{"entry":[{},{},"phosphodiesterase."]},{"entry":[{},"3.1.4.49","Dolichylphosphate-mannose"]},{"entry":[{},{},"phosphodiesterase."]},{"entry":[{},"3.1.4.50","Glycosylphosphatidylinositol"]},{"entry":[{},{},"phospholipase D."]},{"entry":[{},"3.1.4.51","Glucose-1-phospho-D-"]},{"entry":[{},{},"mannosylglycoprotein phosphodiesterase."]},{"entry":[{},"3.1.5.1","dGTPase."]},{"entry":[{},"3.1.6.1","Arylsulfatase."]},{"entry":[{},"3.1.6.2","Steryl-sulfatase."]},{"entry":[{},"3.1.6.3","Glycosulfatase."]},{"entry":[{},"3.1.6.4","N-acetylgalactosamine-6-sulfatase."]},{"entry":[{},"3.1.6.6","Choline-sulfatase."]},{"entry":[{},"3.1.6.7","Cellulose-polysulfatase."]},{"entry":[{},"3.1.6.8","Cerebroside-sulfatase."]},{"entry":[{},"3.1.6.9","Chondro-4-sulfatase."]},{"entry":[{},"3.1.6.10","Chondro-6-sulfatase."]},{"entry":[{},"3.1.6.11","Disulfoglucosamine-6-sulfatase."]},{"entry":[{},"3.1.6.12","N-acetylgalactosamine-4-sulfatase."]},{"entry":[{},"3.1.6.13","Iduronate-2-sulfatase."]},{"entry":[{},"3.1.6.14","N-acetylglucosamine-6-sulfatase."]},{"entry":[{},"3.1.6.15","N-sulfoglucosamine-3-sulfatase."]},{"entry":[{},"3.1.6.16","Monomethyl-sulfatase."]},{"entry":[{},"3.1.6.17","D-lactate-2-sulfatase."]},{"entry":[{},"3.1.6.18","Glucuronate-2-sulfatase."]},{"entry":[{},"3.1.7.1","Prenyl-diphosphatase."]},{"entry":[{},"3.1.7.2","Guanosine-3\u2032,5\u2032-bis(diphosphate) 3\u2032-"]},{"entry":[{},{},"diphosphatase."]},{"entry":[{},"3.1.7.3","Monoterpenyl-diphosphatase."]},{"entry":[{},"3.1.8.1","Aryldialkylphosphatase."]},{"entry":[{},"3.1.8.2","Diisopropyl-fluorophosphatase."]},{"entry":[{},"3.1.11.1","Exodeoxyribonuclease I."]},{"entry":[{},"3.1.11.2","Exodeoxyribonuclease III."]},{"entry":[{},"3.1.11.3","Exodeoxyribonuclease (lambda-"]},{"entry":[{},{},"induced)."]},{"entry":[{},"3.1.11.4","Exodeoxyribonuclease (phage Sp3-"]},{"entry":[{},{},"induced)."]},{"entry":[{},"3.1.11.5","Exodeoxyribonuclease V."]},{"entry":[{},"3.1.11.6","Exodeoxyribonuclease VII."]},{"entry":[{},"3.1.13.1","Exoribonuclease II."]},{"entry":[{},"3.1.13.2","Exoribonuclease H."]},{"entry":[{},"3.1.13.3","Oligonucleotidase."]},{"entry":[{},"3.1.13.4","Poly(A)-specific ribonuclease."]},{"entry":[{},"3.1.14.1","Yeast ribonuclease."]},{"entry":[{},"3.1.15.1","Venom exonuclease."]},{"entry":[{},"3.1.16.1","Spleen exonuclease."]},{"entry":[{},"3.1.21.1","Deoxyribonuclease I."]},{"entry":[{},"3.1.21.2","Deoxyribonuclease IV (phage-T(4)-"]},{"entry":[{},{},"induced)."]},{"entry":[{},"3.1.21.3","Type I site-specific deoxyribonuclease."]},{"entry":[{},"3.1.21.4","Type II site-specific deoxyribonuclease."]},{"entry":[{},"3.1.21.5","Type III site-specific deoxyribonuclease."]},{"entry":[{},"3.1.21.6","CC-preferring endodeoxyribonuclease."]},{"entry":[{},"3.1.21.7","Deoxyribonuclease V."]},{"entry":[{},"3.1.22.1","Deoxyribonuclease II."]},{"entry":[{},"3.1.22.2","Aspergillus deoxyribonuclease K(1)."]},{"entry":[{},"3.1.22.4","Crossover junction endoribonuclease."]},{"entry":[{},"3.1.22.5","Deoxyribonuclease X."]},{"entry":[{},"3.1.25.1","Deoxyribonuclease (pyrimidine dimer)."]},{"entry":[{},"3.1.26.1","Physarum polycephalum ribonuclease."]},{"entry":[{},"3.1.26.2","Ribonuclease alpha."]},{"entry":[{},"3.1.26.3","Ribonuclease III."]},{"entry":[{},"3.1.26.4","Ribonuclease H."]},{"entry":[{},"3.1.26.5","Ribonuclease P."]},{"entry":[{},"3.1.26.6","Ribonuclease IV."]},{"entry":[{},"3.1.26.7","Ribonuclease P4."]},{"entry":[{},"3.1.26.8","Ribonuclease M5."]},{"entry":[{},"3.1.26.9","Ribonuclease (poly-(U)-specific)."]},{"entry":[{},"3.1.26.10","Ribonuclease IX."]},{"entry":[{},"3.1.26.11","Ribonuclease Z."]},{"entry":[{},"3.1.27.1","Ribonuclease T(2)."]},{"entry":[{},"3.1.27.2","Bacillus subtilis ribonuclease."]},{"entry":[{},"3.1.27.3","Ribonuclease T(1)."]},{"entry":[{},"3.1.27.4","Ribonuclease U(2)."]},{"entry":[{},"3.1.27.5","Pancreatic ribonuclease."]},{"entry":[{},"3.1.27.6","Enterobacter ribonuclease."]},{"entry":[{},"3.1.27.7","Ribonuclease F."]},{"entry":[{},"3.1.27.8","Ribonuclease V."]},{"entry":[{},"3.1.27.9","tRNA-intron endonuclease."]},{"entry":[{},"3.1.27.10","rRNA endonuclease."]},{"entry":[{},"3.1.30.1","Aspergillus nuclease S(1)."]},{"entry":[{},"3.1.30.2","Serratia marcescens nuclease."]},{"entry":[{},"3.1.31.1","Micrococcal nuclease."]},{"entry":[{},"3.2.1.1","Alpha-amylase."]},{"entry":[{},"3.2.1.2","Beta-amylase."]},{"entry":[{},"3.2.1.3","Glucan 1,4-alpha-glucosidase."]},{"entry":[{},"3.2.1.4","Cellulase."]},{"entry":[{},"3.2.1.6","Endo-1,3(4)-beta-glucanase."]},{"entry":[{},"3.2.1.7","Inulinase."]},{"entry":[{},"3.2.1.8","Endo-1,4-beta-xylanase."]},{"entry":[{},"3.2.1.10","Oligo-1,6-glucosidase."]},{"entry":[{},"3.2.1.11","Dextranase."]},{"entry":[{},"3.2.1.14","Chitinase."]},{"entry":[{},"3.2.1.15","Polygalacturonase."]},{"entry":[{},"3.2.1.17","Lysozyme."]},{"entry":[{},"3.2.1.18","Exo-alpha-sialidase."]},{"entry":[{},"3.2.1.20","Alpha-glucosidase."]},{"entry":[{},"3.2.1.21","Beta-glucosidase."]},{"entry":[{},"3.2.1.22","Alpha-galactosidase."]},{"entry":[{},"3.2.1.23","Beta-galactosidase."]},{"entry":[{},"3.2.1.24","Alpha-mannosidase."]},{"entry":[{},"3.2.1.25","Beta-mannosidase."]},{"entry":[{},"3.2.1.26","Beta-fructofuranosidase."]},{"entry":[{},"3.2.1.28","Alpha,alpha-trehalase."]},{"entry":[{},"3.2.1.31","Beta-glucuronidase."]},{"entry":[{},"3.2.1.32","Xylan endo-1,3-beta-xylosidase."]},{"entry":[{},"3.2.1.33","Amylo-alpha-1,6-glucosidase."]},{"entry":[{},"3.2.1.35","Hyaluronoglucosaminidase."]},{"entry":[{},"3.2.1.36","Hyaluronoglucuronidase."]},{"entry":[{},"3.2.1.37","Xylan 1,4-beta-xylosidase."]},{"entry":[{},"3.2.1.38","Beta-D-fucosidase."]},{"entry":[{},"3.2.1.39","Glucan endo-1,3-beta-D-glucosidase."]},{"entry":[{},"3.2.1.40","Alpha-L-rhamnosidase."]},{"entry":[{},"3.2.1.41","Pullulanase."]},{"entry":[{},"3.2.1.42","GDP-glucosidase."]},{"entry":[{},"3.2.1.43","Beta-L-rhamnosidase."]},{"entry":[{},"3.2.1.44","Fucoidanase."]},{"entry":[{},"3.2.1.45","Glucosylceramidase."]},{"entry":[{},"3.2.1.46","Galactosylceramidase."]},{"entry":[{},"3.2.1.47","Galactosylgalactosylglucosylceramidase."]},{"entry":[{},"3.2.1.48","Sucrose alpha-glucosidase."]},{"entry":[{},"3.2.1.49","Alpha-N-acetylgalactosaminidase."]},{"entry":[{},"3.2.1.50","Alpha-N-acetylglucosaminidase."]},{"entry":[{},"3.2.1.51","Alpha-L-fucosidase."]},{"entry":[{},"3.2.1.52","Beta-N-acetylhexosaminidase."]},{"entry":[{},"3.2.1.53","Beta-N-acetylgalactosaminidase."]},{"entry":[{},"3.2.1.54","Cyclomaltodextrinase."]},{"entry":[{},"3.2.1.55","Alpha-N-arabinofuranosidase."]},{"entry":[{},"3.2.1.56","Glucuronosyl-disulfoglucosamine"]},{"entry":[{},{},"glucuronidase."]},{"entry":[{},"3.2.1.57","Isopullulanase."]},{"entry":[{},"3.2.1.58","Glucan 1,3-beta-glucosidase."]},{"entry":[{},"3.2.1.59","Glucan endo-1,3-alpha-glucosidase."]},{"entry":[{},"3.2.1.60","Glucan 1,4-alpha-"]},{"entry":[{},{},"maltotetraohydrolase."]},{"entry":[{},"3.2.1.61","Mycodextranase."]},{"entry":[{},"3.2.1.62","Glycosylceramidase."]},{"entry":[{},"3.2.1.63","1,2-alpha-L-fucosidase."]},{"entry":[{},"3.2.1.64","2,6-beta-fructan 6-levanbiohydrolase."]},{"entry":[{},"3.2.1.65","Levanase."]},{"entry":[{},"3.2.1.66","Quercitrinase."]},{"entry":[{},"3.2.1.67","Galacturan 1,4-alpha-galacturonidase."]},{"entry":[{},"3.2.1.68","Isoamylase."]},{"entry":[{},"3.2.1.70","Glucan 1,6-alpha-glucosidase."]},{"entry":[{},"3.2.1.71","Glucan endo-1,2-beta-glucosidase."]},{"entry":[{},"3.2.1.72","Xylan 1,3-beta-xylosidase."]},{"entry":[{},"3.2.1.73","Licheninase."]},{"entry":[{},"3.2.1.74","Glucan 1,4-beta-glucosidase."]},{"entry":[{},"3.2.1.75","Glucan endo-1,6-beta-glucosidase."]},{"entry":[{},"3.2.1.76","L-iduronidase."]},{"entry":[{},"3.2.1.77","Mannan 1,2-(1,3)-alpha-mannosidase."]},{"entry":[{},"3.2.1.78","Mannan endo-1,4-beta-mannosidase."]},{"entry":[{},"3.2.1.80","Fructan beta-fructosidase."]},{"entry":[{},"3.2.1.81","Agarase."]},{"entry":[{},"3.2.1.82","Exo-poly-alpha-galacturonosidase."]},{"entry":[{},"3.2.1.83","Kappa-carrageenase."]},{"entry":[{},"3.2.1.84","Glucan 1,3-alpha-glucosidase."]},{"entry":[{},"3.2.1.85","6-phospho-beta-galactosidase."]},{"entry":[{},"3.2.1.86","6-phospho-beta-glucosidase."]},{"entry":[{},"3.2.1.87","Capsular-polysaccharide endo-1,3-alpha-"]},{"entry":[{},{},"galactosidase."]},{"entry":[{},"3.2.1.88","Beta-L-arabinosidase."]},{"entry":[{},"3.2.1.89","Arabinogalactan endo-1,4-beta-"]},{"entry":[{},{},"galactosidase."]},{"entry":[{},"3.2.1.91","Cellulose 1,4-beta-cellobiosidase."]},{"entry":[{},"3.2.1.92","Peptidoglycan beta-N-acetylmuramidase."]},{"entry":[{},"3.2.1.93","Alpha,alpha-phosphotrehalase."]},{"entry":[{},"3.2.1.94","Glucan 1,6-alpha-isomaltosidase."]},{"entry":[{},"3.2.1.95","Dextran 1,6-alpha-isomaltotriosidase."]},{"entry":[{},"3.2.1.96","Mannosyl-glycoprotein endo-beta-N-"]},{"entry":[{},{},"acetylglucosaminidase."]},{"entry":[{},"3.2.1.97","Glycopeptide alpha-N-"]},{"entry":[{},{},"acetylgalactosaminidase."]},{"entry":[{},"3.2.1.98","Glucan 1,4-alpha-maltohexaosidase."]},{"entry":[{},"3.2.1.99","Arabinan endo-1,5-alpha-L-arabinosidase."]},{"entry":[{},"3.2.1.100","Mannan 1,4-mannobiosidase."]},{"entry":[{},"3.2.1.101","Mannan endo-1,6-alpha-mannosidase."]},{"entry":[{},"3.2.1.102","Blood-group-substance endo-1,4-beta-"]},{"entry":[{},{},"galactosidase."]},{"entry":[{},"3.2.1.103","Keratan-sulfate endo-1,4-beta-"]},{"entry":[{},{},"galactosidase."]},{"entry":[{},"3.2.1.104","Steryl-beta-glucosidase."]},{"entry":[{},"3.2.1.105","Strictosidine beta-glucosidase."]},{"entry":[{},"3.2.1.106","Mannosyl-oligosaccharide glucosidase."]},{"entry":[{},"3.2.1.107","Protein-glucosylgalactosylhydroxylysine"]},{"entry":[{},{},"glucosidase."]},{"entry":[{},"3.2.1.108","Lactase."]},{"entry":[{},"3.2.1.109","Endogalactosaminidase."]},{"entry":[{},"3.2.1.110","Mucinaminylserine mucinaminidase."]},{"entry":[{},"3.2.1.111","1,3-alpha-L-fucosidase."]},{"entry":[{},"3.2.1.112","2-deoxyglucosidase."]},{"entry":[{},"3.2.1.113","Mannosyl-oligosaccharide 1,2-alpha-"]},{"entry":[{},{},"mannosidase."]},{"entry":[{},"3.2.1.114","Mannosyl-oligosaccharide 1,3-1,6-alpha-"]},{"entry":[{},{},"mannosidase."]},{"entry":[{},"3.2.1.115","Branched-dextran exo-1,2-alpha-"]},{"entry":[{},{},"glucosidase."]},{"entry":[{},"3.2.1.116","Glucan 1,4-alpha-maltotriohydrolase."]},{"entry":[{},"3.2.1.117","Amygdalin beta-glucosidase."]},{"entry":[{},"3.2.1.118","Prunasin beta-glucosidase."]},{"entry":[{},"3.2.1.119","Vicianin beta-glucosidase."]},{"entry":[{},"3.2.1.120","Oligoxyloglucan beta-glycosidase."]},{"entry":[{},"3.2.1.121","Polymannuronate hydrolase."]},{"entry":[{},"3.2.1.122","Maltose-6\u2032-phosphate glucosidase."]},{"entry":[{},"3.2.1.123","Endoglycosylceramidase."]},{"entry":[{},"3.2.1.124","3-deoxy-2-octulosonidase."]},{"entry":[{},"3.2.1.125","Raucaffricine beta-glucosidase."]},{"entry":[{},"3.2.1.126","Coniferin beta-glucosidase."]},{"entry":[{},"3.2.1.127","1,6-alpha-L-fucosidase."]},{"entry":[{},"3.2.1.128","Glycyrrhizinate beta-glucuronidase."]},{"entry":[{},"3.2.1.129","Endo-alpha-sialidase."]},{"entry":[{},"3.2.1.130","Glycoprotein endo-alpha-1,2-"]},{"entry":[{},{},"mannosidase."]},{"entry":[{},"3.2.1.131","Xylan alpha-1,2-glucuronosidase."]},{"entry":[{},"3.2.1.132","Chitosanase."]},{"entry":[{},"3.2.1.133","Glucan 1,4-alpha-maltohydrolase."]},{"entry":[{},"3.2.1.134","Difructose-anhydride synthase."]},{"entry":[{},"3.2.1.135","Neopullulanase."]},{"entry":[{},"3.2.1.136","Glucuronoarabinoxylan endo-1,4-"]},{"entry":[{},{},"beta-xylanase."]},{"entry":[{},"3.2.1.137","Mannan exo-1,2-1,6-alpha-"]},{"entry":[{},{},"mannosidase."]},{"entry":[{},"3.2.1.139","Alpha-glucuronidase."]},{"entry":[{},"3.2.1.140","Lacto-N-biosidase."]},{"entry":[{},"3.2.1.141","4-alpha-D-{(1->4)-alpha-D-"]},{"entry":[{},{},"glucano}trehalose trehalohydrolase."]},{"entry":[{},"3.2.1.142","Limit dextrinase."]},{"entry":[{},"3.2.1.143","Poly(ADP-ribose) glycohydrolase."]},{"entry":[{},"3.2.1.144","3-deoxyoctulosonase."]},{"entry":[{},"3.2.1.145","Galactan 1,3-beta-galactosidase."]},{"entry":[{},"3.2.1.146","Beta-galactofuranosidase."]},{"entry":[{},"3.2.1.147","Thioglucosidase."]},{"entry":[{},"3.2.1.148","Ribosylhomocysteinase."]},{"entry":[{},"3.2.1.149","Beta-primeverosidase."]},{"entry":[{},"3.2.1.150","Oligoxyloglucan reducing-end-"]},{"entry":[{},{},"specific cellobiohydrolase."]},{"entry":[{},"3.2.1.151","Xyloglucan-specific endo-beta-1,4-"]},{"entry":[{},{},"glucanase."]},{"entry":[{},"3.2.2.1","Purine nucleosidase."]},{"entry":[{},"3.2.2.2","Inosine nucleosidase."]},{"entry":[{},"3.2.2.3","Uridine nucleosidase."]},{"entry":[{},"3.2.2.4","AMP nucleosidase."]},{"entry":[{},"3.2.2.5","NAD(+) nucleosidase."]},{"entry":[{},"3.2.2.6","NAD(P)(+) nucleosidase."]},{"entry":[{},"3.2.2.7","Adenosine nucleosidase."]},{"entry":[{},"3.2.2.8","Ribosylpyrimidine nucleosidase."]},{"entry":[{},"3.2.2.9","Adenosylhomocysteine nucleosidase."]},{"entry":[{},"3.2.2.10","Pyrimidine-5\u2032-nucleotide"]},{"entry":[{},{},"nucleosidase."]},{"entry":[{},"3.2.2.11","Beta-aspartyl-N-acetylglucosaminidase."]},{"entry":[{},"3.2.2.12","Inosinate nucleosidase."]},{"entry":[{},"3.2.2.13","1-methyladenosine nucleosidase."]},{"entry":[{},"3.2.2.14","NMN nucleosidase."]},{"entry":[{},"3.2.2.15","DNA-deoxyinosine glycosylase."]},{"entry":[{},"3.2.2.16","Methylthioadenosine nucleosidase."]},{"entry":[{},"3.2.2.17","Deoxyribodipyrimidine endonucleosidase."]},{"entry":[{},"3.2.2.19","[Protein ADP-ribosylarginine] hydrolase."]},{"entry":[{},"3.2.2.20","DNA-3-methyladenine glycosylase I."]},{"entry":[{},"3.2.2.21","DNA-3-methyladenine glycosylase II."]},{"entry":[{},"3.2.2.22","rRNA N-glycosylase."]},{"entry":[{},"3.2.2.23","DNA-formamidopyrimidine glycosylase."]},{"entry":[{},"3.2.2.24","ADP-ribosyl-[dinitrogen reductase]"]},{"entry":[{},{},"hydrolase."]},{"entry":[{},"3.3.1.1","Adenosylhomocysteinase."]},{"entry":[{},"3.3.1.2","Adenosylmethionine hydrolase."]},{"entry":[{},"3.3.2.1","Isochorismatase."]},{"entry":[{},"3.3.2.2","Alkenylglycerophosphocholine hydrolase."]},{"entry":[{},"3.3.2.3","Epoxide hydrolase."]},{"entry":[{},"3.3.2.4","Trans-epoxysuccinate hydrolase."]},{"entry":[{},"3.3.2.5","Alkenylglycerophosphoethanolamine"]},{"entry":[{},{},"hydrolase."]},{"entry":[{},"3.3.2.6","Leukotriene-A(4) hydrolase."]},{"entry":[{},"3.3.2.7","Hepoxilin-epoxide hydrolase."]},{"entry":[{},"3.3.2.8","Limonene-1,2-epoxide hydrolase."]},{"entry":[{},"3.4.11.1","Leucyl aminopeptidase."]},{"entry":[{},"3.4.11.2","Membrane alanyl aminopeptidase."]},{"entry":[{},"3.4.11.3","Cystinyl aminopeptidase."]},{"entry":[{},"3.4.11.4","Tripeptide aminopeptidase."]},{"entry":[{},"3.4.11.5","Prolyl aminopeptidase."]},{"entry":[{},"3.4.11.6","Aminopeptidase B."]},{"entry":[{},"3.4.11.7","Glutamyl aminopeptidase."]},{"entry":[{},"3.4.11.9","Xaa-Pro aminopeptidase."]},{"entry":[{},"3.4.11.10","Bacterial leucyl aminopeptidase."]},{"entry":[{},"3.4.11.13","Clostridial aminopeptidase."]},{"entry":[{},"3.4.11.14","Cytosol alanyl aminopeptidase."]},{"entry":[{},"3.4.11.15","Aminopeptidase Y."]},{"entry":[{},"3.4.11.16","Xaa-Trp aminopeptidase."]},{"entry":[{},"3.4.11.17","Tryptophanyl aminopeptidase."]},{"entry":[{},"3.4.11.18","Methionyl aminopeptidase."]},{"entry":[{},"3.4.11.19","D-stereospecific aminopeptidase."]},{"entry":[{},"3.4.11.20","Aminopeptidase Ey."]},{"entry":[{},"3.4.11.21","Aspartyl aminopeptidase."]},{"entry":[{},"3.4.11.22","Aminopeptidase I."]},{"entry":[{},"3.4.11.23","PepB aminopeptidase."]},{"entry":[{},"3.4.13.3","Xaa-His dipeptidase."]},{"entry":[{},"3.4.13.4","Xaa-Arg dipeptidase."]},{"entry":[{},"3.4.13.5","Xaa-methyl-His dipeptidase."]},{"entry":[{},"3.4.13.7","Glu-Glu dipeptidase."]},{"entry":[{},"3.4.13.9","Xaa-Pro dipeptidase."]},{"entry":[{},"3.4.13.12","Met-Xaa dipeptidase."]},{"entry":[{},"3.4.13.17","Non-stereospecific dipeptidase."]},{"entry":[{},"3.4.13.18","Cytosol nonspecific dipeptidase."]},{"entry":[{},"3.4.13.19","Membrane dipeptidase."]},{"entry":[{},"3.4.13.20","Beta-Ala-His dipeptidase."]},{"entry":[{},"3.4.13.21","Dipeptidase E."]},{"entry":[{},"3.4.14.1","Dipeptidyl-peptidase I."]},{"entry":[{},"3.4.14.2","Dipeptidyl-peptidase II."]},{"entry":[{},"3.4.14.4","Dipeptidyl-peptidase III."]},{"entry":[{},"3.4.14.5","Dipeptidyl-peptidase IV."]},{"entry":[{},"3.4.14.6","Dipeptidyl-dipeptidase."]},{"entry":[{},"3.4.14.9","Tripeptidyl-peptidase I."]},{"entry":[{},"3.4.14.10","Tripeptidyl-peptidase II."]},{"entry":[{},"3.4.14.11","Xaa-Pro dipeptidyl-peptidase."]},{"entry":[{},"3.4.15.1","Peptidyl-dipeptidase A."]},{"entry":[{},"3.4.15.4","Peptidyl-dipeptidase B."]},{"entry":[{},"3.4.15.5","Peptidyl-dipeptidase Dcp."]},{"entry":[{},"3.4.16.2","Lysosomal Pro-X carboxypeptidase."]},{"entry":[{},"3.4.16.4","Serine-type D-Ala-D-Ala"]},{"entry":[{},{},"carboxypeptidase."]},{"entry":[{},"3.4.16.5","Carboxypeptidase C."]},{"entry":[{},"3.4.16.6","Carboxypeptidase D."]},{"entry":[{},"3.4.17.1","Carboxypeptidase A."]},{"entry":[{},"3.4.17.2","Carboxypeptidase B."]},{"entry":[{},"3.4.17.3","Lysine carboxypeptidase."]},{"entry":[{},"3.4.17.4","Gly-X carboxypeptidase."]},{"entry":[{},"3.4.17.6","Alanine carboxypeptidase."]},{"entry":[{},"3.4.17.8","Muramoylpentapeptide"]},{"entry":[{},{},"carboxypeptidase."]},{"entry":[{},"3.4.17.10","Carboxypeptidase E."]},{"entry":[{},"3.4.17.11","Glutamate carboxypeptidase."]},{"entry":[{},"3.4.17.12","Carboxypeptidase M."]},{"entry":[{},"3.4.17.13","Muramoyltetrapeptide"]},{"entry":[{},{},"carboxypeptidase."]},{"entry":[{},"3.4.17.14","Zinc D-Ala-D-Ala carboxypeptidase."]},{"entry":[{},"3.4.17.15","Carboxypeptidase A2."]},{"entry":[{},"3.4.17.16","Membrane Pro-X carboxypeptidase."]},{"entry":[{},"3.4.17.17","Tubulinyl-Tyr carboxypeptidase."]},{"entry":[{},"3.4.17.18","Carboxypeptidase T."]},{"entry":[{},"3.4.17.19","Carboxypeptidase Taq."]},{"entry":[{},"3.4.17.20","Carboxypeptidase U."]},{"entry":[{},"3.4.17.21","Glutamate carboxypeptidase II."]},{"entry":[{},"3.4.17.22","Metallocarboxypeptidase D."]},{"entry":[{},"3.4.18.1","Cathepsin X."]},{"entry":[{},"3.4.19.1","Acylaminoacyl-peptidase."]},{"entry":[{},"3.4.19.2","Peptidyl-glycinamidase."]},{"entry":[{},"3.4.19.3","Pyroglutamyl-peptidase I."]},{"entry":[{},"3.4.19.5","Beta-aspartyl-peptidase."]},{"entry":[{},"3.4.19.6","Pyroglutamyl-peptidase II."]},{"entry":[{},"3.4.19.7","N-formylmethionyl-peptidase."]},{"entry":[{},"3.4.19.9","Gamma-glutamyl hydrolase."]},{"entry":[{},"3.4.19.11","Gamma-D-glutamyl-meso-"]},{"entry":[{},{},"diaminopimelate peptidase."]},{"entry":[{},"3.4.19.12","Ubiquitinyl hydrolase 1."]},{"entry":[{},"3.4.21.1","Chymotrypsin."]},{"entry":[{},"3.4.21.2","Chymotrypsin C."]},{"entry":[{},"3.4.21.3","Metridin."]},{"entry":[{},"3.4.21.4","Trypsin."]},{"entry":[{},"3.4.21.5","Thrombin."]},{"entry":[{},"3.4.21.6","Coagulation factor Xa."]},{"entry":[{},"3.4.21.7","Plasmin."]},{"entry":[{},"3.4.21.9","Enteropeptidase."]},{"entry":[{},"3.4.21.10","Acrosin."]},{"entry":[{},"3.4.21.12","Alpha-lytic endopeptidase."]},{"entry":[{},"3.4.21.19","Glutamyl endopeptidase."]},{"entry":[{},"3.4.21.20","Cathepsin G."]},{"entry":[{},"3.4.21.21","Coagulation factor VIIa."]},{"entry":[{},"3.4.21.22","Coagulation factor IXa."]},{"entry":[{},"3.4.21.25","Cucumisin."]},{"entry":[{},"3.4.21.26","Prolyl oligopeptidase."]},{"entry":[{},"3.4.21.27","Coagulation factor XIa."]},{"entry":[{},"3.4.21.32","Brachyurin."]},{"entry":[{},"3.4.21.34","Plasma kallikrein."]},{"entry":[{},"3.4.21.35","Tissue kallikrein."]},{"entry":[{},"3.4.21.36","Pancreatic elastase."]},{"entry":[{},"3.4.21.37","Leukocyte elastase."]},{"entry":[{},"3.4.21.38","Coagulation factor XIIa."]},{"entry":[{},"3.4.21.39","Chymase."]},{"entry":[{},"3.4.21.41","Complement subcomponent C1r."]},{"entry":[{},"3.4.21.42","Complement subcomponent C1s."]},{"entry":[{},"3.4.21.43","Classical-complement-pathway C3\/C5"]},{"entry":[{},{},"convertase."]},{"entry":[{},"3.4.21.45","Complement factor I."]},{"entry":[{},"3.4.21.46","Complement factor D."]},{"entry":[{},"3.4.21.47","Alternative-complement-pathway C3\/C5"]},{"entry":[{},{},"convertase."]},{"entry":[{},"3.4.21.48","Cerevisin."]},{"entry":[{},"3.4.21.49","Hypodermin C."]},{"entry":[{},"3.4.21.50","Lysyl endopeptidase."]},{"entry":[{},"3.4.21.53","Endopeptidase La."]},{"entry":[{},"3.4.21.54","Gamma-renin."]},{"entry":[{},"3.4.21.55","Venombin AB."]},{"entry":[{},"3.4.21.57","Leucyl endopeptidase."]},{"entry":[{},"3.4.21.59","Tryptase."]},{"entry":[{},"3.4.21.60","Scutelarin."]},{"entry":[{},"3.4.21.61","Kexin."]},{"entry":[{},"3.4.21.62","Subtilisin."]},{"entry":[{},"3.4.21.63","Oryzin."]},{"entry":[{},"3.4.21.64","Endopeptidase K."]},{"entry":[{},"3.4.21.65","Thermomycolin."]},{"entry":[{},"3.4.21.66","Thermitase."]},{"entry":[{},"3.4.21.67","Endopeptidase So."]},{"entry":[{},"3.4.21.68","T-plasminogen activator."]},{"entry":[{},"3.4.21.69","Protein C (activated)."]},{"entry":[{},"3.4.21.70","Pancreatic endopeptidase E."]},{"entry":[{},"3.4.21.71","Pancreatic elastase II."]},{"entry":[{},"3.4.21.72","IgA-specific serine endopeptidase."]},{"entry":[{},"3.4.21.73","U-plasminogen activator."]},{"entry":[{},"3.4.21.74","Venombin A."]},{"entry":[{},"3.4.21.75","Furin."]},{"entry":[{},"3.4.21.76","Myeloblastin."]},{"entry":[{},"3.4.21.77","Semenogelase."]},{"entry":[{},"3.4.21.78","Granzyme A."]},{"entry":[{},"3.4.21.79","Granzyme B."]},{"entry":[{},"3.4.21.80","Streptogrisin A."]},{"entry":[{},"3.4.21.81","Streptogrisin B."]},{"entry":[{},"3.4.21.82","Glutamyl endopeptidase II."]},{"entry":[{},"3.4.21.83","Oligopeptidase B."]},{"entry":[{},"3.4.21.84","Limulus clotting factor C."]},{"entry":[{},"3.4.21.85","Limulus clotting factor B."]},{"entry":[{},"3.4.21.86","Limulus clotting enzyme."]},{"entry":[{},"3.4.21.87","Omptin."]},{"entry":[{},"3.4.21.88","Repressor lexA."]},{"entry":[{},"3.4.21.89","Signal peptidase I."]},{"entry":[{},"3.4.21.90","Togavirin."]},{"entry":[{},"3.4.21.91","Flavivirin."]},{"entry":[{},"3.4.21.92","Endopeptidase Clp."]},{"entry":[{},"3.4.21.93","Proprotein convertase 1."]},{"entry":[{},"3.4.21.94","Proprotein convertase 2."]},{"entry":[{},"3.4.21.95","Snake venom factor V activator."]},{"entry":[{},"3.4.21.96","Lactocepin."]},{"entry":[{},"3.4.21.97","Assemblin."]},{"entry":[{},"3.4.21.98","Hepacivirin."]},{"entry":[{},"3.4.21.99","Spermosin."]},{"entry":[{},"3.4.21.100","Pseudomonalisin."]},{"entry":[{},"3.4.21.101","Xanthomonalisin."]},{"entry":[{},"3.4.21.102","C-terminal processing peptidase."]},{"entry":[{},"3.4.21.103","Physarolisin."]},{"entry":[{},"3.4.22.1","Cathepsin B."]},{"entry":[{},"3.4.22.2","Papain."]},{"entry":[{},"3.4.22.3","Ficain."]},{"entry":[{},"3.4.22.6","Chymopapain."]},{"entry":[{},"3.4.22.7","Asclepain."]},{"entry":[{},"3.4.22.8","Clostripain."]},{"entry":[{},"3.4.22.10","Streptopain."]},{"entry":[{},"3.4.22.14","Actinidain."]},{"entry":[{},"3.4.22.15","Cathepsin L."]},{"entry":[{},"3.4.22.16","Cathepsin H."]},{"entry":[{},"3.4.22.24","Cathepsin T."]},{"entry":[{},"3.4.22.25","Glycyl endopeptidase."]},{"entry":[{},"3.4.22.26","Cancer procoagulant."]},{"entry":[{},"3.4.22.27","Cathepsin S."]},{"entry":[{},"3.4.22.28","Picornain 3C."]},{"entry":[{},"3.4.22.29","Picornain 2A."]},{"entry":[{},"3.4.22.30","Caricain."]},{"entry":[{},"3.4.22.31","Ananain."]},{"entry":[{},"3.4.22.32","Stem bromelain."]},{"entry":[{},"3.4.22.33","Fruit bromelain."]},{"entry":[{},"3.4.22.34","Legumain."]},{"entry":[{},"3.4.22.35","Histolysain."]},{"entry":[{},"3.4.22.36","Caspase-1."]},{"entry":[{},"3.4.22.37","Gingipain R."]},{"entry":[{},"3.4.22.38","Cathepsin K."]},{"entry":[{},"3.4.22.39","Adenain."]},{"entry":[{},"3.4.22.40","Bleomycin hydrolase."]},{"entry":[{},"3.4.22.41","Cathepsin F."]},{"entry":[{},"3.4.22.42","Cathepsin O."]},{"entry":[{},"3.4.22.43","Cathepsin V."]},{"entry":[{},"3.4.22.44","Nuclear-inclusion-a endopeptidase."]},{"entry":[{},"3.4.22.45","Helper-component proteinase."]},{"entry":[{},"3.4.22.46","L-peptidase."]},{"entry":[{},"3.4.22.47","Gingipain K."]},{"entry":[{},"3.4.22.48","Staphopain."]},{"entry":[{},"3.4.22.49","Separase."]},{"entry":[{},"3.4.22.50","V-cath endopeptidase."]},{"entry":[{},"3.4.22.51","Cruzipain."]},{"entry":[{},"3.4.22.52","Calpain-1."]},{"entry":[{},"3.4.22.53","Calpain-2."]},{"entry":[{},"3.4.23.1","Pepsin A."]},{"entry":[{},"3.4.23.2","Pepsin B."]},{"entry":[{},"3.4.23.3","Gastricsin."]},{"entry":[{},"3.4.23.4","Chymosin."]},{"entry":[{},"3.4.23.5","Cathepsin D."]},{"entry":[{},"3.4.23.12","Nepenthesin."]},{"entry":[{},"3.4.23.15","Renin."]},{"entry":[{},"3.4.23.16","HIV-1 retropepsin."]},{"entry":[{},"3.4.23.17","Pro-opiomelanocortin converting enzyme."]},{"entry":[{},"3.4.23.18","Aspergillopepsin I."]},{"entry":[{},"3.4.23.19","Aspergillopepsin II."]},{"entry":[{},"3.4.23.20","Penicillopepsin."]},{"entry":[{},"3.4.23.21","Rhizopuspepsin."]},{"entry":[{},"3.4.23.22","Endothiapepsin."]},{"entry":[{},"3.4.23.23","Mucorpepsin."]},{"entry":[{},"3.4.23.24","Candidapepsin."]},{"entry":[{},"3.4.23.25","Saccharopepsin."]},{"entry":[{},"3.4.23.26","Rhodotorulapepsin."]},{"entry":[{},"3.4.23.28","Acrocylindropepsin."]},{"entry":[{},"3.4.23.29","Polyporopepsin."]},{"entry":[{},"3.4.23.30","Pycnoporopepsin."]},{"entry":[{},"3.4.23.31","Scytalidopepsin A."]},{"entry":[{},"3.4.23.32","Scytalidopepsin B."]},{"entry":[{},"3.4.23.34","Cathepsin E."]},{"entry":[{},"3.4.23.35","Barrierpepsin."]},{"entry":[{},"3.4.23.36","Signal peptidase II."]},{"entry":[{},"3.4.23.38","Plasmepsin I."]},{"entry":[{},"3.4.23.39","Plasmepsin II."]},{"entry":[{},"3.4.23.40","Phytepsin."]},{"entry":[{},"3.4.23.41","Yapsin 1."]},{"entry":[{},"3.4.23.42","Thermopsin."]},{"entry":[{},"3.4.23.43","Prepilin peptidase."]},{"entry":[{},"3.4.23.44","Nodavirus endopeptidase."]},{"entry":[{},"3.4.23.45","Memapsin 1."]},{"entry":[{},"3.4.23.46","Memapsin 2."]},{"entry":[{},"3.4.23.47","HIV-2 retropepsin."]},{"entry":[{},"3.4.23.48","Plasminogen activator Pla."]},{"entry":[{},"3.4.24.1","Atrolysin A."]},{"entry":[{},"3.4.24.3","Microbial collagenase."]},{"entry":[{},"3.4.24.6","Leucolysin."]},{"entry":[{},"3.4.24.7","Interstitial collagenase."]},{"entry":[{},"3.4.24.11","Neprilysin."]},{"entry":[{},"3.4.24.12","Envelysin."]},{"entry":[{},"3.4.24.13","IgA-specific metalloendopeptidase."]},{"entry":[{},"3.4.24.14","Procollagen N-endopeptidase."]},{"entry":[{},"3.4.24.15","Thimet oligopeptidase."]},{"entry":[{},"3.4.24.16","Neurolysin."]},{"entry":[{},"3.4.24.17","Stromelysin 1."]},{"entry":[{},"3.4.24.18","Meprin A."]},{"entry":[{},"3.4.24.19","Procollagen C-endopeptidase."]},{"entry":[{},"3.4.24.20","Peptidyl-Lys metalloendopeptidase."]},{"entry":[{},"3.4.24.21","Astacin."]},{"entry":[{},"3.4.24.22","Stromelysin 2."]},{"entry":[{},"3.4.24.23","Matrilysin."]},{"entry":[{},"3.4.24.24","Gelatinase A."]},{"entry":[{},"3.4.24.25","Vibriolysin."]},{"entry":[{},"3.4.24.26","Pseudolysin."]},{"entry":[{},"3.4.24.27","Thermolysin."]},{"entry":[{},"3.4.24.28","Bacillolysin."]},{"entry":[{},"3.4.24.29","Aureolysin."]},{"entry":[{},"3.4.24.30","Coccolysin."]},{"entry":[{},"3.4.24.31","Mycolysin."]},{"entry":[{},"3.4.24.32","Beta-lytic metalloendopeptidase."]},{"entry":[{},"3.4.24.33","Peptidyl-Asp metalloendopeptidase."]},{"entry":[{},"3.4.24.34","Neutrophil collagenase."]},{"entry":[{},"3.4.24.35","Gelatinase B."]},{"entry":[{},"3.4.24.36","Leishmanolysin."]},{"entry":[{},"3.4.24.37","Saccharolysin."]},{"entry":[{},"3.4.24.38","Gametolysin."]},{"entry":[{},"3.4.24.39","Deuterolysin."]},{"entry":[{},"3.4.24.40","Serralysin."]},{"entry":[{},"3.4.24.41","Atrolysin B."]},{"entry":[{},"3.4.24.42","Atrolysin C."]},{"entry":[{},"3.4.24.43","Atroxase."]},{"entry":[{},"3.4.24.44","Atrolysin E."]},{"entry":[{},"3.4.24.45","Atrolysin F."]},{"entry":[{},"3.4.24.46","Adamalysin."]},{"entry":[{},"3.4.24.47","Horrilysin."]},{"entry":[{},"3.4.24.48","Ruberlysin."]},{"entry":[{},"3.4.24.49","Bothropasin."]},{"entry":[{},"3.4.24.50","Bothrolysin."]},{"entry":[{},"3.4.24.51","Ophiolysin."]},{"entry":[{},"3.4.24.52","Trimerelysin I."]},{"entry":[{},"3.4.24.53","Trimerelysin II."]},{"entry":[{},"3.4.24.54","Mucrolysin."]},{"entry":[{},"3.4.24.55","Pitrilysin."]},{"entry":[{},"3.4.24.56","Insulysin."]},{"entry":[{},"3.4.24.57","O-sialoglycoprotein endopeptidase."]},{"entry":[{},"3.4.24.58","Russellysin."]},{"entry":[{},"3.4.24.59","Mitochondrial intermediate"]},{"entry":[{},{},"peptidase."]},{"entry":[{},"3.4.24.60","Dactylysin."]},{"entry":[{},"3.4.24.61","Nardilysin."]},{"entry":[{},"3.4.24.62","Magnolysin."]},{"entry":[{},"3.4.24.63","Meprin B."]},{"entry":[{},"3.4.24.64","Mitochondrial processing peptidase."]},{"entry":[{},"3.4.24.65","Macrophage elastase."]},{"entry":[{},"3.4.24.66","Choriolysin L."]},{"entry":[{},"3.4.24.67","Choriolysin H."]},{"entry":[{},"3.4.24.68","Tentoxilysin."]},{"entry":[{},"3.4.24.69","Bontoxilysin."]},{"entry":[{},"3.4.24.70","Oligopeptidase A."]},{"entry":[{},"3.4.24.71","Endothelin-converting enzyme 1."]},{"entry":[{},"3.4.24.72","Fibrolase."]},{"entry":[{},"3.4.24.73","Jararhagin."]},{"entry":[{},"3.4.24.74","Fragilysin."]},{"entry":[{},"3.4.24.75","Lysostaphin."]},{"entry":[{},"3.4.24.76","Flavastacin."]},{"entry":[{},"3.4.24.77","Snapalysin."]},{"entry":[{},"3.4.24.78","GPR endopeptidase."]},{"entry":[{},"3.4.24.79","Pappalysin-1."]},{"entry":[{},"3.4.24.80","Membrane-type matrix"]},{"entry":[{},{},"metalloproteinase-1."]},{"entry":[{},"3.4.24.81","ADAM10 endopeptidase."]},{"entry":[{},"3.4.24.82","ADAMTS-4 endopeptidase."]},{"entry":[{},"3.4.24.83","Anthrax lethal factor endopeptidase."]},{"entry":[{},"3.4.24.84","Ste24 endopeptidase."]},{"entry":[{},"3.4.24.85","S2P endopeptidase."]},{"entry":[{},"3.4.24.86","ADAM 17 endopeptidase."]},{"entry":[{},"3.4.25.1","Proteasome endopeptidase complex."]},{"entry":[{},"3.5.1.1","Asparaginase."]},{"entry":[{},"3.5.1.2","Glutaminase."]},{"entry":[{},"3.5.1.3","Omega-amidase."]},{"entry":[{},"3.5.1.4","Amidase."]},{"entry":[{},"3.5.1.5","Urease."]},{"entry":[{},"3.5.1.6","Beta-ureidopropionase."]},{"entry":[{},"3.5.1.7","Ureidosuccinase."]},{"entry":[{},"3.5.1.8","Formylaspartate deformylase."]},{"entry":[{},"3.5.1.9","Arylformamidase."]},{"entry":[{},"3.5.1.10","Formyltetrahydrofolate deformylase."]},{"entry":[{},"3.5.1.11","Penicillin amidase."]},{"entry":[{},"3.5.1.12","Biotinidase."]},{"entry":[{},"3.5.1.13","Aryl-acylamidase."]},{"entry":[{},"3.5.1.14","Aminoacylase."]},{"entry":[{},"3.5.1.15","Aspartoacylase."]},{"entry":[{},"3.5.1.16","Acetylornithine deacetylase."]},{"entry":[{},"3.5.1.17","Acyl-lysine deacylase."]},{"entry":[{},"3.5.1.18","Succinyl-diaminopimelate desuccinylase."]},{"entry":[{},"3.5.1.19","Nicotinamidase."]},{"entry":[{},"3.5.1.20","Citrullinase."]},{"entry":[{},"3.5.1.21","N-acetyl-beta-alanine deacetylase."]},{"entry":[{},"3.5.1.22","Pantothenase."]},{"entry":[{},"3.5.1.23","Ceramidase."]},{"entry":[{},"3.5.1.24","Choloylglycine hydrolase."]},{"entry":[{},"3.5.1.25","N-acetylglucosamine-6-phosphate"]},{"entry":[{},{},"deacetylase."]},{"entry":[{},"3.5.1.26","N(4)-(beta-N-acetylglucosaminyl)-L-"]},{"entry":[{},{},"asparaginase."]},{"entry":[{},"3.5.1.27","N-formylmethionylaminoacyl-tRNA"]},{"entry":[{},{},"deformylase."]},{"entry":[{},"3.5.1.28","N-acetylmuramoyl-L-alanine amidase."]},{"entry":[{},"3.5.1.29","2-(acetamidomethylene)succinate"]},{"entry":[{},{},"hydrolase."]},{"entry":[{},"3.5.1.30","5-aminopentanamidase."]},{"entry":[{},"3.5.1.31","Formylmethionine deformylase."]},{"entry":[{},"3.5.1.32","Hippurate hydrolase."]},{"entry":[{},"3.5.1.33","N-acetylglucosamine deacetylase."]},{"entry":[{},"3.5.1.35","D-glutaminase."]},{"entry":[{},"3.5.1.36","N-methyl-2-oxoglutaramate hydrolase."]},{"entry":[{},"3.5.1.38","Glutamin-(asparagin-)ase."]},{"entry":[{},"3.5.1.39","Alkylamidase."]},{"entry":[{},"3.5.1.40","Acylagmatine amidase."]},{"entry":[{},"3.5.1.41","Chitin deacetylase."]},{"entry":[{},"3.5.1.42","Nicotinamide-nucleotide amidase."]},{"entry":[{},"3.5.1.43","Peptidyl-glutaminase."]},{"entry":[{},"3.5.1.44","Protein-glutamine glutaminase."]},{"entry":[{},"3.5.1.46","6-aminohexanoate-dimer hydrolase."]},{"entry":[{},"3.5.1.47","N-acetyldiaminopimelate deacetylase."]},{"entry":[{},"3.5.1.48","Acetylspermidine deacetylase."]},{"entry":[{},"3.5.1.49","Formamidase."]},{"entry":[{},"3.5.1.50","Pentanamidase."]},{"entry":[{},"3.5.1.51","4-acetamidobutyryl-CoA deacetylase."]},{"entry":[{},"3.5.1.52","Peptide-N(4)-(N-acetyl-beta-"]},{"entry":[{},{},"glucosaminyl)asparagine amidase."]},{"entry":[{},"3.5.1.53","N-carbamoylputrescine amidase."]},{"entry":[{},"3.5.1.54","Allophanate hydrolase."]},{"entry":[{},"3.5.1.55","Long-chain-fatty-acyl-glutamate deacylase."]},{"entry":[{},"3.5.1.56","N,N-dimethylformamidase."]},{"entry":[{},"3.5.1.57","Tryptophanamidase."]},{"entry":[{},"3.5.1.58","N-benzyloxycarbonylglycine hydrolase."]},{"entry":[{},"3.5.1.59","N-carbamoylsarcosine amidase."]},{"entry":[{},"3.5.1.60","N-(long-chain-acyl)ethanolamine deacylase."]},{"entry":[{},"3.5.1.61","Mimosinase."]},{"entry":[{},"3.5.1.62","Acetylputrescine deacetylase."]},{"entry":[{},"3.5.1.63","4-acetamidobutyrate deacetylase."]},{"entry":[{},"3.5.1.64","N(alpha)-benzyloxycarbonylleucine"]},{"entry":[{},{},"hydrolase."]},{"entry":[{},"3.5.1.65","Theanine hydrolase."]},{"entry":[{},"3.5.1.66","2-(hydroxymethyl)-3-"]},{"entry":[{},{},"(acetamidomethylene)succinate hydrolase."]},{"entry":[{},"3.5.1.67","4-methyleneglutaminase."]},{"entry":[{},"3.5.1.68","N-formylglutamate deformylase."]},{"entry":[{},"3.5.1.69","Glycosphingolipid deacylase."]},{"entry":[{},"3.5.1.70","Aculeacin-A deacylase."]},{"entry":[{},"3.5.1.71","N-feruloylglycine deacylase."]},{"entry":[{},"3.5.1.72","D-benzoylarginine-4-nitroanilide"]},{"entry":[{},{},"amidase."]},{"entry":[{},"3.5.1.73","Carnitinamidase."]},{"entry":[{},"3.5.1.74","Chenodeoxycholoyltaurine hydrolase."]},{"entry":[{},"3.5.1.75","Urethanase."]},{"entry":[{},"3.5.1.76","Arylalkyl acylamidase."]},{"entry":[{},"3.5.1.77","N-carbamoyl-D-amino acid hydrolase."]},{"entry":[{},"3.5.1.78","Glutathionylspermidine amidase."]},{"entry":[{},"3.5.1.79","Phthalyl amidase."]},{"entry":[{},"3.5.1.81","N-acyl-D-amino-acid deacylase."]},{"entry":[{},"3.5.1.82","N-acyl-D-glutamate deacylase."]},{"entry":[{},"3.5.1.83","N-acyl-D-aspartate deacylase."]},{"entry":[{},"3.5.1.84","Biuret amidohydrolase."]},{"entry":[{},"3.5.1.85","(S)\u2014N-acetyl-1-phenylethylamine"]},{"entry":[{},{},"hydrolase."]},{"entry":[{},"3.5.1.86","Mandelamide amidase."]},{"entry":[{},"3.5.1.87","N-carbamoyl-L-amino-acid hydrolase."]},{"entry":[{},"3.5.1.88","Peptide deformylase."]},{"entry":[{},"3.5.1.89","N-"]},{"entry":[{},{},"acetylglucosaminylphosphatidylinositol"]},{"entry":[{},{},"deacetylase."]},{"entry":[{},"3.5.1.90","Adenosylcobinamide hydrolase."]},{"entry":[{},"3.5.2.1","Barbiturase."]},{"entry":[{},"3.5.2.2","Dihydropyrimidinase."]},{"entry":[{},"3.5.2.3","Dihydroorotase."]},{"entry":[{},"3.5.2.4","Carboxymethylhydantoinase."]},{"entry":[{},"3.5.2.5","Allantoinase."]},{"entry":[{},"3.5.2.6","Beta-lactamase."]},{"entry":[{},"3.5.2.7","Imidazolonepropionase."]},{"entry":[{},"3.5.2.9","5-oxoprolinase (ATP-hydrolyzing)."]},{"entry":[{},"3.5.2.10","Creatininase."]},{"entry":[{},"3.5.2.11","L-lysine-lactamase."]},{"entry":[{},"3.5.2.12","6-aminohexanoate-cyclic-dimer"]},{"entry":[{},{},"hydrolase."]},{"entry":[{},"3.5.2.13","2,5-dioxopiperazine hydrolase."]},{"entry":[{},"3.5.2.14","N-methylhydantoinase (ATP-"]},{"entry":[{},{},"hydrolyzing)."]},{"entry":[{},"3.5.2.15","Cyanuric acid amidohydrolase."]},{"entry":[{},"3.5.2.16","Maleimide hydrolase."]},{"entry":[{},"3.5.2.17","Hydroxyisourate hydrolase."]},{"entry":[{},"3.5.3.1","Arginase."]},{"entry":[{},"3.5.3.2","Guanidinoacetase."]},{"entry":[{},"3.5.3.3","Creatinase."]},{"entry":[{},"3.5.3.4","Allantoicase."]},{"entry":[{},"3.5.3.5","Formimidoylaspartate deiminase."]},{"entry":[{},"3.5.3.6","Arginine deiminase."]},{"entry":[{},"3.5.3.7","Guanidinobutyrase."]},{"entry":[{},"3.5.3.8","Formimidoylglutamase."]},{"entry":[{},"3.5.3.9","Allantoate deiminase."]},{"entry":[{},"3.5.3.10","D-arginase."]},{"entry":[{},"3.5.3.11","Agmatinase."]},{"entry":[{},"3.5.3.12","Agmatine deiminase."]},{"entry":[{},"3.5.3.13","Formimidoylglutamate deiminase."]},{"entry":[{},"3.5.3.14","Amidinoaspartase."]},{"entry":[{},"3.5.3.15","Protein-arginine deiminase."]},{"entry":[{},"3.5.3.16","Methylguanidinase."]},{"entry":[{},"3.5.3.17","Guanidinopropionase."]},{"entry":[{},"3.5.3.18","Dimethylargininase."]},{"entry":[{},"3.5.3.19","Ureidoglycolate hydrolase."]},{"entry":[{},"3.5.3.20","Diguanidinobutanase."]},{"entry":[{},"3.5.3.21","Methylenediurea deaminase."]},{"entry":[{},"3.5.3.22","Proclavaminate amidinohydrolase."]},{"entry":[{},"3.5.4.1","Cytosine deaminase."]},{"entry":[{},"3.5.4.2","Adenine deaminase."]},{"entry":[{},"3.5.4.3","Guanine deaminase."]},{"entry":[{},"3.5.4.4","Adenosine deaminase."]},{"entry":[{},"3.5.4.5","Cytidine deaminase."]},{"entry":[{},"3.5.4.6","AMP deaminase."]},{"entry":[{},"3.5.4.7","ADP deaminase."]},{"entry":[{},"3.5.4.8","Aminoimidazolase."]},{"entry":[{},"3.5.4.9","Methenyltetrahydrofolate cyclohydrolase."]},{"entry":[{},"3.5.4.10","IMP cyclohydrolase."]},{"entry":[{},"3.5.4.11","Pterin deaminase."]},{"entry":[{},"3.5.4.12","dCMP deaminase."]},{"entry":[{},"3.5.4.13","dCTP deaminase."]},{"entry":[{},"3.5.4.14","Deoxycytidine deaminase."]},{"entry":[{},"3.5.4.15","Guanosine deaminase."]},{"entry":[{},"3.5.4.16","GTP cyclohydrolase I."]},{"entry":[{},"3.5.4.17","Adenosine-phosphate deaminase."]},{"entry":[{},"3.5.4.18","ATP deaminase."]},{"entry":[{},"3.5.4.19","Phosphoribosyl-AMP cyclohydrolase."]},{"entry":[{},"3.5.4.20","Pyrithiamine deaminase."]},{"entry":[{},"3.5.4.21","Creatinine deaminase."]},{"entry":[{},"3.5.4.22","1-pyrroline-4-hydroxy-2-carboxylate"]},{"entry":[{},{},"deaminase."]},{"entry":[{},"3.5.4.23","Blasticidin-S deaminase."]},{"entry":[{},"3.5.4.24","Sepiapterin deaminase."]},{"entry":[{},"3.5.4.25","GTP cyclohydrolase II."]},{"entry":[{},"3.5.4.26","Diaminohydroxyphosphoribosylaminopyrimidine"]},{"entry":[{},{},"deaminase."]},{"entry":[{},"3.5.4.27","Methenyltetrahydromethanopterin"]},{"entry":[{},{},"cyclohydrolase."]},{"entry":[{},"3.5.4.28","S-adenosylhomocysteine deaminase."]},{"entry":[{},"3.5.4.29","GTP cyclohydrolase IIa."]},{"entry":[{},"3.5.4.30","dCTP deaminase (dUMP-forming)."]},{"entry":[{},"3.5.5.1","Nitrilase."]},{"entry":[{},"3.5.5.2","Ricinine nitrilase."]},{"entry":[{},"3.5.5.4","Cyanoalanine nitrilase."]},{"entry":[{},"3.5.5.5","Arylacetonitrilase."]},{"entry":[{},"3.5.5.6","Bromoxynil nitrilase."]},{"entry":[{},"3.5.5.7","Aliphatic nitrilase."]},{"entry":[{},"3.5.5.8","Thiocyanate hydrolase."]},{"entry":[{},"3.5.99.1","Riboflavinase."]},{"entry":[{},"3.5.99.2","Thiaminase."]},{"entry":[{},"3.5.99.3","Hydroxydechloroatrazine"]},{"entry":[{},{},"ethylaminohydrolase."]},{"entry":[{},"3.5.99.4","N-isopropylammelide"]},{"entry":[{},{},"isopropylaminohydrolase."]},{"entry":[{},"3.5.99.5","2-aminomuconate deaminase."]},{"entry":[{},"3.5.99.6","Glucosamine-6-phosphate deaminase."]},{"entry":[{},"3.5.99.7","1-aminocyclopropane-1-carboxylate"]},{"entry":[{},{},"deaminase."]},{"entry":[{},"3.6.1.1","Inorganic diphosphatase."]},{"entry":[{},"3.6.1.2","Trimetaphosphatase."]},{"entry":[{},"3.6.1.3","Adenosinetriphosphatase."]},{"entry":[{},"3.6.1.5","Apyrase."]},{"entry":[{},"3.6.1.6","Nucleoside-diphosphatase."]},{"entry":[{},"3.6.1.7","Acylphosphatase."]},{"entry":[{},"3.6.1.8","ATP diphosphatase."]},{"entry":[{},"3.6.1.9","Nucleotide diphosphatase."]},{"entry":[{},"3.6.1.10","Endopolyphosphatase."]},{"entry":[{},"3.6.1.11","Exopolyphosphatase."]},{"entry":[{},"3.6.1.12","dCTP diphosphatase."]},{"entry":[{},"3.6.1.13","ADP-ribose diphosphatase."]},{"entry":[{},"3.6.1.14","Adenosine-tetraphosphatase."]},{"entry":[{},"3.6.1.15","Nucleoside-triphosphatase."]},{"entry":[{},"3.6.1.16","CDP-glycerol diphosphatase."]},{"entry":[{},"3.6.1.17","Bis(5\u2032-nucleosyl)-tetraphosphatase"]},{"entry":[{},{},"(asymmetrical)."]},{"entry":[{},"3.6.1.18","FAD diphosphatase."]},{"entry":[{},"3.6.1.19","Nucleoside-triphosphate diphosphatase."]},{"entry":[{},"3.6.1.20","5\u2032-acylphosphoadenosine hydrolase."]},{"entry":[{},"3.6.1.21","ADP-sugar diphosphatase."]},{"entry":[{},"3.6.1.22","NAD+ diphosphatase."]},{"entry":[{},"3.6.1.23","dUTP diphosphatase."]},{"entry":[{},"3.6.1.24","Nucleoside phosphoacylhydrolase."]},{"entry":[{},"3.6.1.25","Triphosphatase."]},{"entry":[{},"3.6.1.26","CDP-diacylglycerol diphosphatase."]},{"entry":[{},"3.6.1.27","Undecaprenyl-diphosphatase."]},{"entry":[{},"3.6.1.28","Thiamine-triphosphatase."]},{"entry":[{},"3.6.1.29","Bis(5\u2032-adenosyl)-triphosphatase."]},{"entry":[{},"3.6.1.30","M(7)G(5\u2032)pppN diphosphatase."]},{"entry":[{},"3.6.1.31","Phosphoribosyl-ATP diphosphatase."]},{"entry":[{},"3.6.1.39","Thymidine-triphosphatase."]},{"entry":[{},"3.6.1.40","Guanosine-5\u2032-triphosphate,3\u2032-"]},{"entry":[{},{},"diphosphate diphosphatase."]},{"entry":[{},"3.6.1.41","Bis(5\u2032-nucleosyl)-tetraphosphatase"]},{"entry":[{},{},"(symmetrical)."]},{"entry":[{},"3.6.1.42","Guanosine-diphosphatase."]},{"entry":[{},"3.6.1.43","Dolichyldiphosphatase."]},{"entry":[{},"3.6.1.44","Oligosaccharide-diphosphodolichol"]},{"entry":[{},{},"diphosphatase."]},{"entry":[{},"3.6.1.45","UDP-sugar diphosphatase."]},{"entry":[{},"3.6.1.52","Diphosphoinositol-polyphosphate"]},{"entry":[{},{},"diphosphatase."]},{"entry":[{},"3.6.2.1","Adenylylsulfatase."]},{"entry":[{},"3.6.2.2","Phosphoadenylylsulfatase."]},{"entry":[{},"3.6.3.1","Phospholipid-translocating ATPase."]},{"entry":[{},"3.6.3.2","Magnesium-importing ATPase."]},{"entry":[{},"3.6.3.3","Cadmium-exporting ATPase."]},{"entry":[{},"3.6.3.4","Copper-exporting ATPase."]},{"entry":[{},"3.6.3.5","Zinc-exporting ATPase."]},{"entry":[{},"3.6.3.6","Proton-exporting ATPase."]},{"entry":[{},"3.6.3.7","Sodium-exporting ATPase."]},{"entry":[{},"3.6.3.8","Calcium-transporting ATPase."]},{"entry":[{},"3.6.3.9","Sodium\/potassium-exchanging"]},{"entry":[{},{},"ATPase."]},{"entry":[{},"3.6.3.10","Hydrogen\/potassium-exchanging"]},{"entry":[{},{},"ATPase."]},{"entry":[{},"3.6.3.11","Chloride-transporting ATPase."]},{"entry":[{},"3.6.3.12","Potassium-transporting ATPase."]},{"entry":[{},"3.6.3.14","H(+)-transporting two-sector ATPase."]},{"entry":[{},"3.6.3.15","Sodium-transporting two-sector"]},{"entry":[{},{},"ATPase."]},{"entry":[{},"3.6.3.16","Arsenite-transporting ATPase."]},{"entry":[{},"3.6.3.17","Monosaccharide-transporting ATPase."]},{"entry":[{},"3.6.3.18","Oligosaccharide-transporting ATPase."]},{"entry":[{},"3.6.3.19","Maltose-transporting ATPase."]},{"entry":[{},"3.6.3.20","Glycerol-3-phosphate-transporting"]},{"entry":[{},{},"ATPase."]},{"entry":[{},"3.6.3.21","Polar-amino-acid-transporting"]},{"entry":[{},{},"ATPase."]},{"entry":[{},"3.6.3.22","Nonpolar-amino-acid-transporting"]},{"entry":[{},{},"ATPase."]},{"entry":[{},"3.6.3.23","Oligopeptide-transporting ATPase."]},{"entry":[{},"3.6.3.24","Nickel-transporting ATPase."]},{"entry":[{},"3.6.3.25","Sulfate-transporting ATPase."]},{"entry":[{},"3.6.3.26","Nitrate-transporting ATPase."]},{"entry":[{},"3.6.3.27","Phosphate-transporting ATPase."]},{"entry":[{},"3.6.3.28","Phosphonate-transporting ATPase."]},{"entry":[{},"3.6.3.29","Molybdate-transporting ATPase."]},{"entry":[{},"3.6.3.30","Fe(3+)-transporting ATPase."]},{"entry":[{},"3.6.3.31","Polyamine-transporting ATPase."]},{"entry":[{},"3.6.3.32","Quaternary-amine-transporting"]},{"entry":[{},{},"ATPase."]},{"entry":[{},"3.6.3.33","Vitamin B12-transporting ATPase."]},{"entry":[{},"3.6.3.34","Iron-chelate-transporting ATPase."]},{"entry":[{},"3.6.3.35","Manganese-transporting ATPase."]},{"entry":[{},"3.6.3.36","Taurine-transporting ATPase."]},{"entry":[{},"3.6.3.37","Guanine-transporting ATPase."]},{"entry":[{},"3.6.3.38","Capsular-polysaccharide-transporting"]},{"entry":[{},{},"ATPase."]},{"entry":[{},"3.6.3.39","Lipopolysaccharide-transporting"]},{"entry":[{},{},"ATPase."]},{"entry":[{},"3.6.3.40","Teichoic-acid-transporting ATPase."]},{"entry":[{},"3.6.3.41","Heme-transporting ATPase."]},{"entry":[{},"3.6.3.42","Beta-glucan-transporting ATPase."]},{"entry":[{},"3.6.3.43","Peptide-transporting ATPase."]},{"entry":[{},"3.6.3.44","Xenobiotic-transporting ATPase."]},{"entry":[{},"3.6.3.45","Steroid-transporting ATPase."]},{"entry":[{},"3.6.3.46","Cadmium-transporting ATPase."]},{"entry":[{},"3.6.3.47","Fatty-acyl-CoA-transporting ATPase."]},{"entry":[{},"3.6.3.48","Alpha-factor-transporting ATPase."]},{"entry":[{},"3.6.3.49","Channel-conductance-controlling"]},{"entry":[{},{},"ATPase."]},{"entry":[{},"3.6.3.50","Protein-secreting ATPase."]},{"entry":[{},"3.6.3.51","Mitochondrial protein-transporting"]},{"entry":[{},{},"ATPase."]},{"entry":[{},"3.6.3.52","Chloroplast protein-transporting"]},{"entry":[{},{},"ATPase."]},{"entry":[{},"3.6.3.53","Ag(+)-exporting ATPase."]},{"entry":[{},"3.6.4.1","Myosin ATPase."]},{"entry":[{},"3.6.4.2","Dynein ATPase."]},{"entry":[{},"3.6.4.3","Microtubule-severing ATPase."]},{"entry":[{},"3.6.4.4","Plus-end-directed kinesin ATPase."]},{"entry":[{},"3.6.4.5","Minus-end-directed kinesin ATPase."]},{"entry":[{},"3.6.4.6","Vesicle-fusing ATPase."]},{"entry":[{},"3.6.4.7","Peroxisome-assembly ATPase."]},{"entry":[{},"3.6.4.8","Proteasome ATPase."]},{"entry":[{},"3.6.4.9","Chaperonin ATPase."]},{"entry":[{},"3.6.4.10","Non-chaperonin molecular chaperone"]},{"entry":[{},{},"ATPase."]},{"entry":[{},"3.6.4.11","Nucleoplasmin ATPase."]},{"entry":[{},"3.6.5.1","Heterotrimeric G-protein GTPase."]},{"entry":[{},"3.6.5.2","Small monomeric GTPase."]},{"entry":[{},"3.6.5.3","Protein-synthesizing GTPase."]},{"entry":[{},"3.6.5.4","Signal-recognition-particle GTPase."]},{"entry":[{},"3.6.5.5","Dynamin GTPase."]},{"entry":[{},"3.6.5.6","Tubulin GTPase."]},{"entry":[{},"3.7.1.1","Oxaloacetase."]},{"entry":[{},"3.7.1.2","Fumarylacetoacetase."]},{"entry":[{},"3.7.1.3","Kynureninase."]},{"entry":[{},"3.7.1.4","Phloretin hydrolase."]},{"entry":[{},"3.7.1.5","Acylpyruvate hydrolase."]},{"entry":[{},"3.7.1.6","Acetylpyruvate hydrolase."]},{"entry":[{},"3.7.1.7","Beta-diketone hydrolase."]},{"entry":[{},"3.7.1.8","2,6-dioxo-6-phenylhexa-3-enoate"]},{"entry":[{},{},"hydrolase."]},{"entry":[{},"3.7.1.9","2-hydroxymuconate-semialdehyde"]},{"entry":[{},{},"hydrolase."]},{"entry":[{},"3.7.1.10","Cyclohexane-1,3-dione hydrolase."]},{"entry":[{},"3.8.1.1","Alkylhalidase."]},{"entry":[{},"3.8.1.2","(S)-2-haloacid dehalogenase."]},{"entry":[{},"3.8.1.3","Haloacetate dehalogenase."]},{"entry":[{},"3.8.1.5","Haloalkane dehalogenase."]},{"entry":[{},"3.8.1.6","4-chlorobenzoate dehalogenase."]},{"entry":[{},"3.8.1.7","4-chlorobenzoyl-CoA dehalogenase."]},{"entry":[{},"3.8.1.8","Atrazine chlorohydrolase."]},{"entry":[{},"3.8.1.9","(R)-2-haloacid dehalogenase."]},{"entry":[{},"3.8.1.10","2-haloacid dehalogenase (configuration-"]},{"entry":[{},{},"inverting)."]},{"entry":[{},"3.8.1.11","2-haloacid dehalogenase (configuration-"]},{"entry":[{},{},"retaining)."]},{"entry":[{},"3.9.1.1","Phosphoamidase."]},{"entry":[{},"3.10.1.1","N-sulfoglucosamine sulfohydrolase."]},{"entry":[{},"3.10.1.2","Cyclamate sulfohydrolase."]},{"entry":[{},"3.11.1.1","Phosphonoacetaldehyde hydrolase."]},{"entry":[{},"3.11.1.2","Phosphonoacetate hydrolase."]},{"entry":[{},"3.12.1.1","Trithionate hydrolase."]},{"entry":[{},"3.13.1.1","UDP-sulfoquinovose synthase."]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"ENZYME: 4.\u2014.\u2014.\u2014"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"168pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"4.1.1.1","Pyruvate decarboxylase."]},{"entry":[{},"4.1.1.2","Oxalate decarboxylase."]},{"entry":[{},"4.1.1.3","Oxaloacetate decarboxylase."]},{"entry":[{},"4.1.1.4","Acetoacetate decarboxylase."]},{"entry":[{},"4.1.1.5","Acetolactate decarboxylase."]},{"entry":[{},"4.1.1.6","Aconitate decarboxylase."]},{"entry":[{},"4.1.1.7","Benzoylformate decarboxylase."]},{"entry":[{},"4.1.1.8","Oxalyl-CoA decarboxylase."]},{"entry":[{},"4.1.1.9","Malonyl-CoA decarboxylase."]},{"entry":[{},"4.1.1.11","Aspartate 1-decarboxylase."]},{"entry":[{},"4.1.1.12","Aspartate 4-decarboxylase."]},{"entry":[{},"4.1.1.14","Valine decarboxylase."]},{"entry":[{},"4.1.1.15","Glutamate decarboxylase."]},{"entry":[{},"4.1.1.16","Hydroxyglutamate decarboxylase."]},{"entry":[{},"4.1.1.17","Ornithine decarboxylase."]},{"entry":[{},"4.1.1.18","Lysine decarboxylase."]},{"entry":[{},"4.1.1.19","Arginine decarboxylase."]},{"entry":[{},"4.1.1.20","Diaminopimelate decarboxylase."]},{"entry":[{},"4.1.1.21","Phosphoribosylaminoimidazole"]},{"entry":[{},{},"carboxylase."]},{"entry":[{},"4.1.1.22","Histidine decarboxylase."]},{"entry":[{},"4.1.1.23","Orotidine-5\u2032-phosphate decarboxylase."]},{"entry":[{},"4.1.1.24","Aminobenzoate decarboxylase."]},{"entry":[{},"4.1.1.25","Tyrosine decarboxylase."]},{"entry":[{},"4.1.1.28","Aromatic-L-amino-acid decarboxylase."]},{"entry":[{},"4.1.1.29","Sulfinoalanine decarboxylase."]},{"entry":[{},"4.1.1.30","Pantothenoylcysteine decarboxylase."]},{"entry":[{},"4.1.1.31","Phosphoenolpyruvate carboxylase."]},{"entry":[{},"4.1.1.32","Phosphoenolpyruvate carboxykinase"]},{"entry":[{},{},"(GTP)."]},{"entry":[{},"4.1.1.33","Diphosphomevalonate decarboxylase."]},{"entry":[{},"4.1.1.34","Dehydro-L-gulonate decarboxylase."]},{"entry":[{},"4.1.1.35","UDP-glucuronate decarboxylase."]},{"entry":[{},"4.1.1.36","Phosphopantothenoylcysteine"]},{"entry":[{},{},"decarboxylase."]},{"entry":[{},"4.1.1.37","Uroporphyrinogen decarboxylase."]},{"entry":[{},"4.1.1.38","Phosphoenolpyruvate carboxykinase"]},{"entry":[{},{},"(diphosphate)."]},{"entry":[{},"4.1.1.39","Ribulose-bisphosphate carboxylase."]},{"entry":[{},"4.1.1.40","Hydroxypyruvate decarboxylase."]},{"entry":[{},"4.1.1.41","Methylmalonyl-CoA decarboxylase."]},{"entry":[{},"4.1.1.42","Carnitine decarboxylase."]},{"entry":[{},"4.1.1.43","Phenylpyruvate decarboxylase."]},{"entry":[{},"4.1.1.44","4-carboxymuconolactone"]},{"entry":[{},{},"decarboxylase."]},{"entry":[{},"4.1.1.45","Aminocarboxymuconate-semialdehyde"]},{"entry":[{},{},"decarboxylase."]},{"entry":[{},"4.1.1.46","O-pyrocatechuate decarboxylase."]},{"entry":[{},"4.1.1.47","Tartronate-semialdehyde synthase."]},{"entry":[{},"4.1.1.48","Indole-3-glycerol-phosphate synthase."]},{"entry":[{},"4.1.1.49","Phosphoenolpyruvate carboxykinase"]},{"entry":[{},{},"(ATP)."]},{"entry":[{},"4.1.1.50","Adenosylmethionine decarboxylase."]},{"entry":[{},"4.1.1.51","3-hydroxy-2-methylpyridine-4,5-"]},{"entry":[{},{},"dicarboxylate 4-decarboxylase."]},{"entry":[{},"4.1.1.52","6-methylsalicylate decarboxylase."]},{"entry":[{},"4.1.1.53","Phenylalanine decarboxylase."]},{"entry":[{},"4.1.1.54","Dihydroxyfumarate decarboxylase."]},{"entry":[{},"4.1.1.55","4,5-dihydroxyphthalate decarboxylase."]},{"entry":[{},"4.1.1.56","3-oxolaurate decarboxylase."]},{"entry":[{},"4.1.1.57","Methionine decarboxylase."]},{"entry":[{},"4.1.1.58","Orsellinate decarboxylase."]},{"entry":[{},"4.1.1.59","Gallate decarboxylase."]},{"entry":[{},"4.1.1.60","Stipitatonate decarboxylase."]},{"entry":[{},"4.1.1.61","4-hydroxybenzoate decarboxylase."]},{"entry":[{},"4.1.1.62","Gentisate decarboxylase."]},{"entry":[{},"4.1.1.63","Protocatechuate decarboxylase."]},{"entry":[{},"4.1.1.64","2,2-dialkylglycine decarboxylase"]},{"entry":[{},{},"(pyruvate)."]},{"entry":[{},"4.1.1.65","Phosphatidylserine decarboxylase."]},{"entry":[{},"4.1.1.66","Uracil-5-carboxylate decarboxylase."]},{"entry":[{},"4.1.1.67","UDP-galacturonate decarboxylase."]},{"entry":[{},"4.1.1.68","5-oxopent-3-ene-1,2,5-tricarboxylate"]},{"entry":[{},{},"decarboxylase."]},{"entry":[{},"4.1.1.69","3,4-dihydroxyphthalate decarboxylase."]},{"entry":[{},"4.1.1.70","Glutaconyl-CoA decarboxylase."]},{"entry":[{},"4.1.1.71","2-oxoglutarate decarboxylase."]},{"entry":[{},"4.1.1.72","Branched-chain-2-oxoacid"]},{"entry":[{},{},"decarboxylase."]},{"entry":[{},"4.1.1.73","Tartrate decarboxylase."]},{"entry":[{},"4.1.1.74","Indolepyruvate decarboxylase."]},{"entry":[{},"4.1.1.75","5-guanidino-2-oxopentanoate"]},{"entry":[{},{},"decarboxylase."]},{"entry":[{},"4.1.1.76","Arylmalonate decarboxylase."]},{"entry":[{},"4.1.1.77","4-oxalocrotonate decarboxylase."]},{"entry":[{},"4.1.1.78","Acetylenedicarboxylate decarboxylase."]},{"entry":[{},"4.1.1.79","Sulfopyruvate decarboxylase."]},{"entry":[{},"4.1.1.80","4-hydroxyphenylpyruvate decarboxylase."]},{"entry":[{},"4.1.1.81","Threonine-phosphate decarboxylase."]},{"entry":[{},"4.1.2.2","Ketotetrose-phosphate aldolase."]},{"entry":[{},"4.1.2.4","Deoxyribose-phosphate aldolase."]},{"entry":[{},"4.1.2.5","Threonine aldolase."]},{"entry":[{},"4.1.2.9","Phosphoketolase."]},{"entry":[{},"4.1.2.10","Mandelonitrile lyase."]},{"entry":[{},"4.1.2.11","Hydroxymandelonitrile lyase."]},{"entry":[{},"4.1.2.12","2-dehydropantoate aldolase."]},{"entry":[{},"4.1.2.13","Fructose-bisphosphate aldolase."]},{"entry":[{},"4.1.2.14","2-dehydro-3-deoxy-phosphogluconate"]},{"entry":[{},{},"aldolase."]},{"entry":[{},"4.1.2.17","L-fuculose-phosphate aldolase."]},{"entry":[{},"4.1.2.18","2-dehydro-3-deoxy-L-pentonate aldolase."]},{"entry":[{},"4.1.2.19","Rhamnulose-1-phosphate aldolase."]},{"entry":[{},"4.1.2.20","2-dehydro-3-deoxyglucarate aldolase."]},{"entry":[{},"4.1.2.21","2-dehydro-3-deoxy-6-"]},{"entry":[{},{},"phosphogalactonate aldolase."]},{"entry":[{},"4.1.2.22","Fructose-6-phosphate phosphoketolase."]},{"entry":[{},"4.1.2.23","3-deoxy-D-manno-octulosonate aldolase."]},{"entry":[{},"4.1.2.24","Dimethylaniline-N-oxide aldolase."]},{"entry":[{},"4.1.2.25","Dihydroneopterin aldolase."]},{"entry":[{},"4.1.2.26","Phenylserine aldolase."]},{"entry":[{},"4.1.2.27","Sphinganine-1-phosphate aldolase."]},{"entry":[{},"4.1.2.28","2-dehydro-3-deoxy-D-pentonate"]},{"entry":[{},{},"aldolase."]},{"entry":[{},"4.1.2.29","5-dehydro-2-deoxyphosphogluconate"]},{"entry":[{},{},"aldolase."]},{"entry":[{},"4.1.2.30","17-alpha-hydroxyprogesterone aldolase."]},{"entry":[{},"4.1.2.32","Trimethylamine-oxide aldolase."]},{"entry":[{},"4.1.2.33","Fucosterol-epoxide lyase."]},{"entry":[{},"4.1.2.34","4-(2-carboxyphenyl)-2-oxobut-3-enoate"]},{"entry":[{},{},"aldolase."]},{"entry":[{},"4.1.2.35","Propioin synthase."]},{"entry":[{},"4.1.2.36","Lactate aldolase."]},{"entry":[{},"4.1.2.37","Acetone-cyanohydrin lyase."]},{"entry":[{},"4.1.2.38","Benzoin aldolase."]},{"entry":[{},"4.1.2.39","Hydroxynitrilase."]},{"entry":[{},"4.1.2.40","Tagatose-bisphosphate aldolase."]},{"entry":[{},"4.1.2.41","Vanillin synthase."]},{"entry":[{},"4.1.3.1","Isocitrate lyase."]},{"entry":[{},"4.1.3.3","N-acetylneuraminate lyase."]},{"entry":[{},"4.1.3.4","Hydroxymethylglutaryl-CoA lyase."]},{"entry":[{},"4.1.3.6","Citrate (pro-3S)-lyase."]},{"entry":[{},"4.1.3.13","Oxalomalate lyase."]},{"entry":[{},"4.1.3.14","3-hydroxyaspartate aldolase."]},{"entry":[{},"4.1.3.16","4-hydroxy-2-oxoglutarate aldolase."]},{"entry":[{},"4.1.3.17","4-hydroxy-4-methyl-2-oxoglutarate"]},{"entry":[{},{},"aldolase."]},{"entry":[{},"4.1.3.22","Citramalate lyase."]},{"entry":[{},"4.1.3.24","Malyl-CoA lyase."]},{"entry":[{},"4.1.3.25","Citramalyl-CoA lyase."]},{"entry":[{},"4.1.3.26","3-hydroxy-3-isohexenylglutaryl-CoA"]},{"entry":[{},{},"lyase."]},{"entry":[{},"4.1.3.27","Anthranilate synthase."]},{"entry":[{},"4.1.3.30","Methylisocitrate lyase."]},{"entry":[{},"4.1.3.32","2,3-dimethylmalate lyase."]},{"entry":[{},"4.1.3.34","Citryl-CoA lyase."]},{"entry":[{},"4.1.3.35","(1-hydroxycyclohexan-1-yl)acetyl-CoA"]},{"entry":[{},{},"lyase."]},{"entry":[{},"4.1.3.36","Naphthoate synthase."]},{"entry":[{},"4.1.3.38","Aminodeoxychorismate lyase."]},{"entry":[{},"4.1.99.1","Tryptophanase."]},{"entry":[{},"4.1.99.2","Tyrosine phenol-lyase."]},{"entry":[{},"4.1.99.3","Deoxyribodipyrimidine photo-lyase."]},{"entry":[{},"4.1.99.5","Octadecanal decarbonylase."]},{"entry":[{},"4.1.99.11","Benzylsuccinate synthase."]},{"entry":[{},"4.2.1.1","Carbonate dehydratase."]},{"entry":[{},"4.2.1.2","Fumarate hydratase."]},{"entry":[{},"4.2.1.3","Aconitate hydratase."]},{"entry":[{},"4.2.1.4","Citrate dehydratase."]},{"entry":[{},"4.2.1.5","Arabinonate dehydratase."]},{"entry":[{},"4.2.1.6","Galactonate dehydratase."]},{"entry":[{},"4.2.1.7","Altronate dehydratase."]},{"entry":[{},"4.2.1.8","Mannonate dehydratase."]},{"entry":[{},"4.2.1.9","Dihydroxy-acid dehydratase."]},{"entry":[{},"4.2.1.10","3-dehydroquinate dehydratase."]},{"entry":[{},"4.2.1.11","Phosphopyruvate hydratase."]},{"entry":[{},"4.2.1.12","Phosphogluconate dehydratase."]},{"entry":[{},"4.2.1.17","Enoyl-CoA hydratase."]},{"entry":[{},"4.2.1.18","Methylglutaconyl-CoA hydratase."]},{"entry":[{},"4.2.1.19","Imidazoleglycerol-phosphate dehydratase."]},{"entry":[{},"4.2.1.20","Tryptophan synthase."]},{"entry":[{},"4.2.1.22","Cystathionine beta-synthase."]},{"entry":[{},"4.2.1.24","Porphobilinogen synthase."]},{"entry":[{},"4.2.1.25","L-arabinonate dehydratase."]},{"entry":[{},"4.2.1.27","Acetylenecarboxylate hydratase."]},{"entry":[{},"4.2.1.28","Propanediol dehydratase."]},{"entry":[{},"4.2.1.30","Glycerol dehydratase."]},{"entry":[{},"4.2.1.31","Maleate hydratase."]},{"entry":[{},"4.2.1.32","L(+)-tartrate dehydratase."]},{"entry":[{},"4.2.1.33","3-isopropylmalate dehydratase."]},{"entry":[{},"4.2.1.34","(S)-2-methylmalate dehydratase."]},{"entry":[{},"4.2.1.35","(R)-2-methylmalate dehydratase."]},{"entry":[{},"4.2.1.36","Homoaconitate hydratase."]},{"entry":[{},"4.2.1.39","Gluconate dehydratase."]},{"entry":[{},"4.2.1.40","Glucarate dehydratase."]},{"entry":[{},"4.2.1.41","5-dehydro-4-deoxyglucarate"]},{"entry":[{},{},"dehydratase."]},{"entry":[{},"4.2.1.42","Galactarate dehydratase."]},{"entry":[{},"4.2.1.43","2-dehydro-3-deoxy-L-arabinonate"]},{"entry":[{},{},"dehydratase."]},{"entry":[{},"4.2.1.44","Myo-inosose-2 dehydratase."]},{"entry":[{},"4.2.1.45","CDP-glucose 4,6-dehydratase."]},{"entry":[{},"4.2.1.46","dTDP-glucose 4,6-dehydratase."]},{"entry":[{},"4.2.1.47","GDP-mannose 4,6-dehydratase."]},{"entry":[{},"4.2.1.48","D-glutamate cyclase."]},{"entry":[{},"4.2.1.49","Urocanate hydratase."]},{"entry":[{},"4.2.1.50","Pyrazolylalanine synthase."]},{"entry":[{},"4.2.1.51","Prephenate dehydratase."]},{"entry":[{},"4.2.1.52","Dihydrodipicolinate synthase."]},{"entry":[{},"4.2.1.53","Oleate hydratase."]},{"entry":[{},"4.2.1.54","Lactoyl-CoA dehydratase."]},{"entry":[{},"4.2.1.55","3-hydroxybutyryl-CoA dehydratase."]},{"entry":[{},"4.2.1.56","Itaconyl-CoA hydratase."]},{"entry":[{},"4.2.1.57","Isohexenylglutaconyl-CoA"]},{"entry":[{},{},"hydratase."]},{"entry":[{},"4.2.1.58","Crotonoyl-[acyl-carrier-protein]"]},{"entry":[{},{},"hydratase."]},{"entry":[{},"4.2.1.59","3-hydroxyoctanoyl-[acyl-carrier-"]},{"entry":[{},{},"protein] dehydratase."]},{"entry":[{},"4.2.1.60","3-hydroxydecanoyl-[acyl-carrier-"]},{"entry":[{},{},"protein] dehydratase."]},{"entry":[{},"4.2.1.61","3-hydroxypalmitoyl-[acyl-carrier-"]},{"entry":[{},{},"protein] dehydratase."]},{"entry":[{},"4.2.1.62","5-alpha-hydroxysteroid dehydratase."]},{"entry":[{},"4.2.1.65","3-cyanoalanine hydratase."]},{"entry":[{},"4.2.1.66","Cyanide hydratase."]},{"entry":[{},"4.2.1.67","D-fuconate dehydratase."]},{"entry":[{},"4.2.1.68","L-fuconate dehydratase."]},{"entry":[{},"4.2.1.69","Cyanamide hydratase."]},{"entry":[{},"4.2.1.70","Pseudouridylate synthase."]},{"entry":[{},"4.2.1.73","Protoaphin-aglucone dehydratase"]},{"entry":[{},{},"(cyclizing)."]},{"entry":[{},"4.2.1.74","Long-chain-enoyl-CoA hydratase."]},{"entry":[{},"4.2.1.75","Uroporphyrinogen-III synthase."]},{"entry":[{},"4.2.1.76","UDP-glucose 4,6-dehydratase."]},{"entry":[{},"4.2.1.77","Trans-L-3-hydroxyproline"]},{"entry":[{},{},"dehydratase."]},{"entry":[{},"4.2.1.78","(S)-norcoclaurine synthase."]},{"entry":[{},"4.2.1.79","2-methylcitrate dehydratase."]},{"entry":[{},"4.2.1.80","2-oxopent-4-enoate hydratase."]},{"entry":[{},"4.2.1.81","D(\u2212)-tartrate dehydratase."]},{"entry":[{},"4.2.1.82","Xylonate dehydratase."]},{"entry":[{},"4.2.1.83","4-oxalmesaconate hydratase."]},{"entry":[{},"4.2.1.84","Nitrile hydratase."]},{"entry":[{},"4.2.1.85","Dimethylmaleate hydratase."]},{"entry":[{},"4.2.1.86","16-dehydroprogesterone hydratase."]},{"entry":[{},"4.2.1.87","Octopamine dehydratase."]},{"entry":[{},"4.2.1.88","Synephrine dehydratase."]},{"entry":[{},"4.2.1.89","Carnitine dehydratase."]},{"entry":[{},"4.2.1.90","L-rhamnonate dehydratase."]},{"entry":[{},"4.2.1.91","Carboxycyclohexadienyl dehydratase."]},{"entry":[{},"4.2.1.92","Hydroperoxide dehydratase."]},{"entry":[{},"4.2.1.93","ATP-dependent NAD(P)H-hydrate"]},{"entry":[{},{},"dehydratase."]},{"entry":[{},"4.2.1.94","Scytalone dehydratase."]},{"entry":[{},"4.2.1.95","Kievitone hydratase."]},{"entry":[{},"4.2.1.96","4a-hydroxytetrahydrobiopterin"]},{"entry":[{},{},"dehydratase."]},{"entry":[{},"4.2.1.97","Phaseollidin hydratase."]},{"entry":[{},"4.2.1.98","16-alpha-hydroxyprogesterone"]},{"entry":[{},{},"dehydratase."]},{"entry":[{},"4.2.1.99","2-methylisocitrate dehydratase."]},{"entry":[{},"4.2.1.100","Cyclohexa-1,5-dienecarbonyl-CoA"]},{"entry":[{},{},"hydratase."]},{"entry":[{},"4.2.1.101","Trans-feruloyl-CoA hydratase."]},{"entry":[{},"4.2.1.103","Cyclohexyl-isocyanide hydratase."]},{"entry":[{},"4.2.1.104","Cyanate hydratase."]},{"entry":[{},"4.2.2.1","Hyaluronate lyase."]},{"entry":[{},"4.2.2.2","Pectate lyase."]},{"entry":[{},"4.2.2.3","Poly(beta-D-mannuronate) lyase."]},{"entry":[{},"4.2.2.4","Chondroitin ABC lyase."]},{"entry":[{},"4.2.2.5","Chondroitin AC lyase."]},{"entry":[{},"4.2.2.6","Oligogalacturonide lyase."]},{"entry":[{},"4.2.2.7","Heparin lyase."]},{"entry":[{},"4.2.2.8","Heparin-sulfate lyase."]},{"entry":[{},"4.2.2.9","Pectate disaccharide-lyase."]},{"entry":[{},"4.2.2.10","Pectin lyase."]},{"entry":[{},"4.2.2.11","Poly(alpha-L-guluronate) lyase."]},{"entry":[{},"4.2.2.12","Xanthan lyase."]},{"entry":[{},"4.2.2.13","Exo-(1->4)-alpha-D-glucan lyase."]},{"entry":[{},"4.2.2.14","Glucuronan lyase."]},{"entry":[{},"4.2.2.15","Anhydrosialidase."]},{"entry":[{},"4.2.2.16","Levan fructotransferase (DFA-IV-"]},{"entry":[{},{},"forming)."]},{"entry":[{},"4.2.2.17","Inulin fructotransferase (DFA-I-forming)."]},{"entry":[{},"4.2.2.18","Inulin fructotransferase (DFA-III-forming)."]},{"entry":[{},"4.2.3.1","Threonine synthase."]},{"entry":[{},"4.2.3.2","Ethanolamine-phosphate phospho-lyase."]},{"entry":[{},"4.2.3.3","Methylglyoxal synthase."]},{"entry":[{},"4.2.3.4","3-dehydroquinate synthase."]},{"entry":[{},"4.2.3.5","Chorismate synthase."]},{"entry":[{},"4.2.3.6","Trichodiene synthase."]},{"entry":[{},"4.2.3.7","Pentalenene synthase."]},{"entry":[{},"4.2.3.8","Casbene synthase."]},{"entry":[{},"4.2.3.9","Aristolochene synthase."]},{"entry":[{},"4.2.3.10","(\u2212)-endo-fenchol synthase."]},{"entry":[{},"4.2.3.11","Sabinene-hydrate synthase."]},{"entry":[{},"4.2.3.12","6-pyruvoyltetrahydropterin"]},{"entry":[{},{},"synthase."]},{"entry":[{},"4.2.3.13","(+)-delta-cadinene synthase."]},{"entry":[{},"4.2.3.14","Pinene synthase."]},{"entry":[{},"4.2.3.15","Myrcene synthase."]},{"entry":[{},"4.2.3.16","(4S)-limonene synthase."]},{"entry":[{},"4.2.3.17","Taxadiene synthase."]},{"entry":[{},"4.2.3.18","Abietadiene synthase."]},{"entry":[{},"4.2.3.19","Ent-kaurene synthase."]},{"entry":[{},"4.2.3.20","(+)-limonene synthase."]},{"entry":[{},"4.2.3.21","Vetispiradiene synthase."]},{"entry":[{},"4.2.99.12","Carboxymethyloxysuccinate lyase."]},{"entry":[{},"4.2.99.18","DNA-(apurinic or apyrimidinic"]},{"entry":[{},{},"site) lyase."]},{"entry":[{},"4.2.99.19","2-hydroxypropyl-CoM lyase."]},{"entry":[{},"4.3.1.1","Aspartate ammonia-lyase."]},{"entry":[{},"4.3.1.2","Methylaspartate ammonia-lyase."]},{"entry":[{},"4.3.1.3","Histidine ammonia-lyase."]},{"entry":[{},"4.3.1.4","Formimidoyltetrahydrofolate"]},{"entry":[{},{},"cyclodeaminase."]},{"entry":[{},"4.3.1.5","Phenylalanine ammonia-lyase."]},{"entry":[{},"4.3.1.6","Beta-alanyl-CoA ammonia-lyase."]},{"entry":[{},"4.3.1.7","Ethanolamine ammonia-lyase."]},{"entry":[{},"4.3.1.9","Glucosaminate ammonia-lyase."]},{"entry":[{},"4.3.1.10","Serine-sulfate ammonia-lyase."]},{"entry":[{},"4.3.1.11","Dihydroxyphenylalanine ammonia-"]},{"entry":[{},{},"lyase."]},{"entry":[{},"4.3.1.12","Ornithine cyclodeaminase."]},{"entry":[{},"4.3.1.13","Carbamoyl-serine ammonia-lyase."]},{"entry":[{},"4.3.1.14","3-aminobutyryl-CoA ammonia-"]},{"entry":[{},{},"lyase."]},{"entry":[{},"4.3.1.15","Diaminopropionate ammonia-lyase."]},{"entry":[{},"4.3.1.16","Threo-3-hydroxyaspartate"]},{"entry":[{},{},"ammonia-lyase."]},{"entry":[{},"4.3.1.17","L-serine ammonia-lyase."]},{"entry":[{},"4.3.1.18","D-serine ammonia-lyase."]},{"entry":[{},"4.3.1.19","Threonine ammonia-lyase."]},{"entry":[{},"4.3.1.20","Erythro-3-hydroxyaspartate"]},{"entry":[{},{},"ammonia-lyase."]},{"entry":[{},"4.3.2.1","Argininosuccinate lyase."]},{"entry":[{},"4.3.2.2","Adenylosuccinate lyase."]},{"entry":[{},"4.3.2.3","Ureidoglycolate lyase."]},{"entry":[{},"4.3.2.4","Purine imidazole-ring cyclase."]},{"entry":[{},"4.3.2.5","Peptidylamidoglycolate lyase."]},{"entry":[{},"4.3.3.1","3-ketovalidoxylamine C-N-lyase."]},{"entry":[{},"4.3.3.2","Strictosidine synthase."]},{"entry":[{},"4.3.3.3","Deacetylisoipecoside synthase."]},{"entry":[{},"4.3.3.4","Deacetylipecoside synthase."]},{"entry":[{},"4.4.1.1","Cystathionine gamma-lyase."]},{"entry":[{},"4.4.1.2","Homocysteine desulfhydrase."]},{"entry":[{},"4.4.1.3","Dimethylpropiothetin dethiomethylase."]},{"entry":[{},"4.4.1.4","Alliin lyase."]},{"entry":[{},"4.4.1.5","Lactoylglutathione lyase."]},{"entry":[{},"4.4.1.6","S-alkylcysteine lyase."]},{"entry":[{},"4.4.1.8","Cystathionine beta-lyase."]},{"entry":[{},"4.4.1.9","L-3-cyanoalanine synthase."]},{"entry":[{},"4.4.1.10","Cysteine lyase."]},{"entry":[{},"4.4.1.11","Methionine gamma-lyase."]},{"entry":[{},"4.4.1.13","Cysteine-S-conjugate beta-lyase."]},{"entry":[{},"4.4.1.14","1-aminocyclopropane-1-carboxylate"]},{"entry":[{},{},"synthase."]},{"entry":[{},"4.4.1.15","D-cysteine desulfhydrase."]},{"entry":[{},"4.4.1.16","Selenocysteine lyase."]},{"entry":[{},"4.4.1.17","Holocytochrome-c synthase."]},{"entry":[{},"4.4.1.19","Phosphosulfolactate synthase."]},{"entry":[{},"4.4.1.20","Leukotriene-C(4) synthase."]},{"entry":[{},"4.5.1.1","DDT-dehydrochlorinase."]},{"entry":[{},"4.5.1.2","3-chloro-D-alanine dehydrochlorinase."]},{"entry":[{},"4.5.1.3","Dichloromethane dehalogenase."]},{"entry":[{},"4.5.1.4","L-2-amino-4-chloropent-4-enoate"]},{"entry":[{},{},"dehydrochlorinase."]},{"entry":[{},"4.5.1.5","S-carboxymethylcysteine synthase."]},{"entry":[{},"4.6.1.1","Adenylate cyclase."]},{"entry":[{},"4.6.1.2","Guanylate cyclase."]},{"entry":[{},"4.6.1.6","Cytidylate cyclase."]},{"entry":[{},"4.6.1.12","2-C-methyl-D-erythritol 2,4-"]},{"entry":[{},{},"cyclodiphosphate synthase."]},{"entry":[{},"4.6.1.13","Phosphatidylinositol diacylglycerol-lyase."]},{"entry":[{},"4.6.1.14","Glycosylphosphatidylinositol"]},{"entry":[{},{},"diacylglycerol-lyase."]},{"entry":[{},"4.6.1.15","FAD-AMP lyase (cyclizing)."]},{"entry":[{},"4.99.1.1","Ferrochelatase."]},{"entry":[{},"4.99.1.2","Alkylmercury lyase."]},{"entry":[{},"4.99.1.3","Sirohydrochlorin cobaltochelatase."]},{"entry":[{},"4.99.1.4","Sirohydrochlorin ferrochelatase."]},{"entry":[{},"4.99.1.5","Aliphatic aldoxime dehydratase."]},{"entry":[{},"4.99.1.6","Indoleacetaldoxime dehydratase."]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"ENZYME: 5.\u2014.\u2014.\u2014"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"168pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"5.1.1.1","Alanine racemase."]},{"entry":[{},"5.1.1.2","Methionine racemase."]},{"entry":[{},"5.1.1.3","Glutamate racemase."]},{"entry":[{},"5.1.1.4","Proline racemase."]},{"entry":[{},"5.1.1.5","Lysine racemase."]},{"entry":[{},"5.1.1.6","Threonine racemase."]},{"entry":[{},"5.1.1.7","Diaminopimelate epimerase."]},{"entry":[{},"5.1.1.8","4-hydroxyproline epimerase."]},{"entry":[{},"5.1.1.9","Arginine racemase."]},{"entry":[{},"5.1.1.10","Amino-acid racemase."]},{"entry":[{},"5.1.1.11","Phenylalanine racemase (ATP-"]},{"entry":[{},{},"hydrolyzing)."]},{"entry":[{},"5.1.1.12","Ornithine racemase."]},{"entry":[{},"5.1.1.13","Aspartate racemase."]},{"entry":[{},"5.1.1.14","Nocardicin-A epimerase."]},{"entry":[{},"5.1.1.15","2-aminohexano-6-lactam racemase."]},{"entry":[{},"5.1.1.16","Protein-serine epimerase."]},{"entry":[{},"5.1.1.17","Isopenicillin-N epimerase."]},{"entry":[{},"5.1.2.1","Lactate racemase."]},{"entry":[{},"5.1.2.2","Mandelate racemase."]},{"entry":[{},"5.1.2.3","3-hydroxybutyryl-CoA epimerase."]},{"entry":[{},"5.1.2.4","Acetoin racemase."]},{"entry":[{},"5.1.2.5","Tartrate epimerase."]},{"entry":[{},"5.1.2.6","Isocitrate epimerase."]},{"entry":[{},"5.1.3.1","Ribulose-phosphate 3-epimerase."]},{"entry":[{},"5.1.3.2","UDP-glucose 4-epimerase."]},{"entry":[{},"5.1.3.3","Aldose 1-epimerase."]},{"entry":[{},"5.1.3.4","L-ribulose-phosphate 4-epimerase."]},{"entry":[{},"5.1.3.5","UDP-arabinose 4-epimerase."]},{"entry":[{},"5.1.3.6","UDP-glucuronate 4-epimerase."]},{"entry":[{},"5.1.3.7","UDP-N-acetylglucosamine 4-"]},{"entry":[{},{},"epimerase."]},{"entry":[{},"5.1.3.8","N-acylglucosamine 2-epimerase."]},{"entry":[{},"5.1.3.9","N-acylglucosamine-6-phosphate 2-"]},{"entry":[{},{},"epimerase."]},{"entry":[{},"5.1.3.10","CDP-abequose epimerase."]},{"entry":[{},"5.1.3.11","Cellobiose epimerase."]},{"entry":[{},"5.1.3.12","UDP-glucuronate 5\u2032-epimerase."]},{"entry":[{},"5.1.3.13","dTDP-4-dehydrorhamnose 3,5-"]},{"entry":[{},{},"epimerase."]},{"entry":[{},"5.1.3.14","UDP-N-acetylglucosamine 2-"]},{"entry":[{},{},"epimerase."]},{"entry":[{},"5.1.3.15","Glucose-6-phosphate 1-epimerase."]},{"entry":[{},"5.1.3.16","UDP-glucosamine 4-epimerase."]},{"entry":[{},"5.1.3.17","Heparosan-N-sulfate-glucuronate 5-"]},{"entry":[{},{},"epimerase."]},{"entry":[{},"5.1.3.18","GDP-mannose 3,5-epimerase."]},{"entry":[{},"5.1.3.19","Chondroitin-glucuronate 5-"]},{"entry":[{},{},"epimerase."]},{"entry":[{},"5.1.3.20","ADP-glyceromanno-heptose 6-"]},{"entry":[{},{},"epimerase."]},{"entry":[{},"5.1.3.21","Maltose epimerase."]},{"entry":[{},"5.1.99.1","Methylmalonyl-CoA epimerase."]},{"entry":[{},"5.1.99.2","16-hydroxysteroid epimerase."]},{"entry":[{},"5.1.99.3","Allantoin racemase."]},{"entry":[{},"5.1.99.4","Alpha-methylacyl-CoA racemase."]},{"entry":[{},"5.2.1.1","Maleate isomerase."]},{"entry":[{},"5.2.1.2","Maleylacetoacetate isomerase."]},{"entry":[{},"5.2.1.3","Retinal isomerase."]},{"entry":[{},"5.2.1.4","Maleylpyruvate isomerase."]},{"entry":[{},"5.2.1.5","Linoleate isomerase."]},{"entry":[{},"5.2.1.6","Furylfuramide isomerase."]},{"entry":[{},"5.2.1.7","Retinol isomerase."]},{"entry":[{},"5.2.1.8","Peptidylprolyl isomerase."]},{"entry":[{},"5.2.1.9","Farnesol 2-isomerase."]},{"entry":[{},"5.2.1.10","2-chloro-4-carboxymethylenebut-2-en-1,4-"]},{"entry":[{},{},"olide isomerase."]},{"entry":[{},"5.2.1.11","4-hydroxyphenylacetaldehyde-oxime"]},{"entry":[{},{},"isomerase."]},{"entry":[{},"5.3.1.1","Triose-phosphate isomerase."]},{"entry":[{},"5.3.1.3","Arabinose isomerase."]},{"entry":[{},"5.3.1.4","L-arabinose isomerase."]},{"entry":[{},"5.3.1.5","Xylose isomerase."]},{"entry":[{},"5.3.1.6","Ribose-5-phosphate isomerase."]},{"entry":[{},"5.3.1.7","Mannose isomerase."]},{"entry":[{},"5.3.1.8","Mannose-6-phosphate isomerase."]},{"entry":[{},"5.3.1.9","Glucose-6-phosphate isomerase."]},{"entry":[{},"5.3.1.12","Glucuronate isomerase."]},{"entry":[{},"5.3.1.13","Arabinose-5-phosphate isomerase."]},{"entry":[{},"5.3.1.14","L-rhamnose isomerase."]},{"entry":[{},"5.3.1.15","D-lyxose ketol-isomerase."]},{"entry":[{},"5.3.1.16","1-(5-phosphoribosyl)-5-((5-"]},{"entry":[{},{},"phosphoribosylamino)methylideneamino)imidazole-4-"]},{"entry":[{},{},"carboxamide isomerase."]},{"entry":[{},"5.3.1.17","4-deoxy-L-threo-5-hexosulose-uronate"]},{"entry":[{},{},"ketol-isomerase."]},{"entry":[{},"5.3.1.20","Ribose isomerase."]},{"entry":[{},"5.3.1.21","Corticosteroid side-chain-isomerase."]},{"entry":[{},"5.3.1.22","Hydroxypyruvate isomerase."]},{"entry":[{},"5.3.1.23","S-methyl-5-thioribose-1-phosphate"]},{"entry":[{},{},"isomerase."]},{"entry":[{},"5.3.1.24","Phosphoribosylanthranilate isomerase."]},{"entry":[{},"5.3.1.25","L-fucose isomerase."]},{"entry":[{},"5.3.1.26","Galactose-6-phosphate isomerase."]},{"entry":[{},"5.3.2.1","Phenylpyruvate tautomerase."]},{"entry":[{},"5.3.2.2","Oxaloacetate tautomerase."]},{"entry":[{},"5.3.3.1","Steroid delta-isomerase."]},{"entry":[{},"5.3.3.2","Isopentenyl-diphosphate delta-isomerase."]},{"entry":[{},"5.3.3.3","Vinylacetyl-CoA delta-isomerase."]},{"entry":[{},"5.3.3.4","Muconolactone delta-isomerase."]},{"entry":[{},"5.3.3.5","Cholestenol delta-isomerase."]},{"entry":[{},"5.3.3.6","Methylitaconate delta-isomerase."]},{"entry":[{},"5.3.3.7","Aconitate delta-isomerase."]},{"entry":[{},"5.3.3.8","Dodecenoyl-CoA delta-isomerase."]},{"entry":[{},"5.3.3.9","Prostaglandin-A(1) delta-isomerase."]},{"entry":[{},"5.3.3.10","5-carboxymethyl-2-hydroxymuconate"]},{"entry":[{},{},"delta-isomerase."]},{"entry":[{},"5.3.3.11","Isopiperitenone delta-isomerase."]},{"entry":[{},"5.3.3.12","Dopachrome isomerase."]},{"entry":[{},"5.3.3.13","Polyenoic fatty acid isomerase."]},{"entry":[{},"5.3.4.1","Protein disulfide-isomerase."]},{"entry":[{},"5.3.99.2","Prostaglandin-D synthase."]},{"entry":[{},"5.3.99.3","Prostaglandin-E synthase."]},{"entry":[{},"5.3.99.4","Prostaglandin-I synthase."]},{"entry":[{},"5.3.99.5","Thromboxane-A synthase."]},{"entry":[{},"5.3.99.6","Allene-oxide cyclase."]},{"entry":[{},"5.3.99.7","Styrene-oxide isomerase."]},{"entry":[{},"5.4.1.1","Lysolecithin acylmutase."]},{"entry":[{},"5.4.1.2","Precorrin-8X methylmutase."]},{"entry":[{},"5.4.2.1","Phosphoglycerate mutase."]},{"entry":[{},"5.4.2.2","Phosphoglucomutase."]},{"entry":[{},"5.4.2.3","Phosphoacetylglucosamine mutase."]},{"entry":[{},"5.4.2.4","Bisphosphoglycerate mutase."]},{"entry":[{},"5.4.2.5","Phosphoglucomutase (glucose-"]},{"entry":[{},{},"cofactor)."]},{"entry":[{},"5.4.2.6","Beta-phosphoglucomutase."]},{"entry":[{},"5.4.2.7","Phosphopentomutase."]},{"entry":[{},"5.4.2.8","Phosphomannomutase."]},{"entry":[{},"5.4.2.9","Phosphoenolpyruvate mutase."]},{"entry":[{},"5.4.2.10","Phosphoglucosamine mutase."]},{"entry":[{},"5.4.3.2","Lysine 2,3-aminomutase."]},{"entry":[{},"5.4.3.3","Beta-lysine 5,6-aminomutase."]},{"entry":[{},"5.4.3.4","D-lysine 5,6-aminomutase."]},{"entry":[{},"5.4.3.5","D-ornithine 4,5-aminomutase."]},{"entry":[{},"5.4.3.6","Tyrosine 2,3-aminomutase."]},{"entry":[{},"5.4.3.7","Leucine 2,3-aminomutase."]},{"entry":[{},"5.4.3.8","Glutamate-1-semialdehyde 2,1-"]},{"entry":[{},{},"aminomutase."]},{"entry":[{},"5.4.4.1","(Hydroxyamino)benzene mutase."]},{"entry":[{},"5.4.4.2","Isochorismate synthase."]},{"entry":[{},"5.4.4.3","3-(hydroxyamino)phenol mutase."]},{"entry":[{},"5.4.99.1","Methylaspartate mutase."]},{"entry":[{},"5.4.99.2","Methylmalonyl-CoA mutase."]},{"entry":[{},"5.4.99.3","2-acetolactate mutase."]},{"entry":[{},"5.4.99.4","2-methyleneglutarate mutase."]},{"entry":[{},"5.4.99.5","Chorismate mutase."]},{"entry":[{},"5.4.99.7","Lanosterol synthase."]},{"entry":[{},"5.4.99.8","Cycloartenol synthase."]},{"entry":[{},"5.4.99.9","UDP-galactopyranose mutase."]},{"entry":[{},"5.4.99.11","Isomaltulose synthase."]},{"entry":[{},"5.4.99.12","tRNA-pseudouridine synthase I."]},{"entry":[{},"5.4.99.13","Isobutyryl-CoA mutase."]},{"entry":[{},"5.4.99.14","4-carboxymethyl-4-"]},{"entry":[{},{},"methylbutenolide mutase."]},{"entry":[{},"5.4.99.15","(1->4)-alpha-D-glucan 1-alpha-D-"]},{"entry":[{},{},"glucosylmutase."]},{"entry":[{},"5.4.99.16","Maltose alpha-D-"]},{"entry":[{},{},"glucosyltransferase."]},{"entry":[{},"5.4.99.17","Squalene--hopene cyclase."]},{"entry":[{},"5.5.1.1","Muconate cycloisomerase."]},{"entry":[{},"5.5.1.2","3-carboxy-cis,cis-muconate"]},{"entry":[{},{},"cycloisomerase."]},{"entry":[{},"5.5.1.3","Tetrahydroxypteridine cycloisomerase."]},{"entry":[{},"5.5.1.4","Inositol-3-phosphate synthase."]},{"entry":[{},"5.5.1.5","Carboxy-cis,cis-muconate cyclase."]},{"entry":[{},"5.5.1.6","Chalcone isomerase."]},{"entry":[{},"5.5.1.7","Chloromuconate cycloisomerase."]},{"entry":[{},"5.5.1.8","Geranyl-diphosphate cyclase."]},{"entry":[{},"5.5.1.9","Cycloeucalenol cycloisomerase."]},{"entry":[{},"5.5.1.10","Alpha-pinene-oxide decyclase."]},{"entry":[{},"5.5.1.11","Dichloromuconate cycloisomerase."]},{"entry":[{},"5.5.1.12","Copalyl diphosphate synthase."]},{"entry":[{},"5.5.1.13","Ent-copalyl diphosphate synthase."]},{"entry":[{},"5.99.1.1","Thiocyanate isomerase."]},{"entry":[{},"5.99.1.2","DNA topoisomerase."]},{"entry":[{},"5.99.1.3","DNA topoisomerase (ATP-hydrolyzing)."]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"ENZYME: 6.\u2014.\u2014.\u2014"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"168pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"6.1.1.1","Tyrosine--tRNA ligase."]},{"entry":[{},"6.1.1.2","Tryptophan--tRNA ligase."]},{"entry":[{},"6.1.1.3","Threonine--tRNA ligase."]},{"entry":[{},"6.1.1.4","Leucine--tRNA ligase."]},{"entry":[{},"6.1.1.5","Isoleucine--tRNA ligase."]},{"entry":[{},"6.1.1.6","Lysine--tRNA ligase."]},{"entry":[{},"6.1.1.7","Alanine--tRNA ligase."]},{"entry":[{},"6.1.1.9","Valine--tRNA ligase."]},{"entry":[{},"6.1.1.10","Methionine--tRNA ligase."]},{"entry":[{},"6.1.1.11","Serine--tRNA ligase."]},{"entry":[{},"6.1.1.12","Aspartate--tRNA ligase."]},{"entry":[{},"6.1.1.13","D-alanine--poly(phosphoribitol) ligase."]},{"entry":[{},"6.1.1.14","Glycine--tRNA ligase."]},{"entry":[{},"6.1.1.15","Proline--tRNA ligase."]},{"entry":[{},"6.1.1.16","Cysteine--tRNA ligase."]},{"entry":[{},"6.1.1.17","Glutamate--tRNA ligase."]},{"entry":[{},"6.1.1.18","Glutamine--tRNA ligase."]},{"entry":[{},"6.1.1.19","Arginine--tRNA ligase."]},{"entry":[{},"6.1.1.20","Phenylalanine--tRNA ligase."]},{"entry":[{},"6.1.1.21","Histidine--tRNA ligase."]},{"entry":[{},"6.1.1.22","Asparagine--tRNA ligase."]},{"entry":[{},"6.1.1.23","Aspartate--tRNA(Asn) ligase."]},{"entry":[{},"6.1.1.24","Glutamate--tRNA(Gln) ligase."]},{"entry":[{},"6.1.1.25","Lysine--tRNA(Pyl) ligase."]},{"entry":[{},"6.2.1.1","Acetate--CoA ligase."]},{"entry":[{},"6.2.1.2","Butyrate--CoA ligase."]},{"entry":[{},"6.2.1.3","Long-chain-fatty-acid--CoA ligase."]},{"entry":[{},"6.2.1.4","Succinate--CoA ligase (GDP-forming)."]},{"entry":[{},"6.2.1.5","Succinate--CoA ligase (ADP-"]},{"entry":[{},{},"forming)."]},{"entry":[{},"6.2.1.6","Glutarate--CoA ligase."]},{"entry":[{},"6.2.1.7","Cholate--CoA ligase."]},{"entry":[{},"6.2.1.8","Oxalate--CoA ligase."]},{"entry":[{},"6.2.1.9","Malate--CoA ligase."]},{"entry":[{},"6.2.1.10","Acid--CoA ligase (GDP-forming)."]},{"entry":[{},"6.2.1.11","Biotin--CoA ligase."]},{"entry":[{},"6.2.1.12","4-coumarate--CoA ligase."]},{"entry":[{},"6.2.1.13","Acetate--CoA ligase (ADP-forming)."]},{"entry":[{},"6.2.1.14","6-carboxyhexanoate--CoA ligase."]},{"entry":[{},"6.2.1.15","Arachidonate--CoA ligase."]},{"entry":[{},"6.2.1.16","Acetoacetate--CoA ligase."]},{"entry":[{},"6.2.1.17","Propionate--CoA ligase."]},{"entry":[{},"6.2.1.18","Citrate--CoA ligase."]},{"entry":[{},"6.2.1.19","Long-chain-fatty-acid--luciferin-"]},{"entry":[{},{},"component ligase."]},{"entry":[{},"6.2.1.20","Long-chain-fatty-acid--[acyl-carrier-"]},{"entry":[{},{},"protein] ligase."]},{"entry":[{},"6.2.1.22","[Citrate (pro-3S)-lyase] ligase."]},{"entry":[{},"6.2.1.23","Dicarboxylate--CoA ligase."]},{"entry":[{},"6.2.1.24","Phytanate--CoA ligase."]},{"entry":[{},"6.2.1.25","Benzoate--CoA ligase."]},{"entry":[{},"6.2.1.26","O-succinylbenzoate--CoA ligase."]},{"entry":[{},"6.2.1.27","4-hydroxybenzoate--CoA ligase."]},{"entry":[{},"6.2.1.28","3-alpha,7-alpha-dihydroxy-5-beta-"]},{"entry":[{},{},"cholestanate--CoA ligase."]},{"entry":[{},"6.2.1.29","3-alpha,7-alpha,12-alpha-trihydroxy-"]},{"entry":[{},{},"5-beta-cholestanate--CoA ligase."]},{"entry":[{},"6.2.1.30","Phenylacetate--CoA ligase."]},{"entry":[{},"6.2.1.31","2-furoate--CoA ligase."]},{"entry":[{},"6.2.1.32","Anthranilate--CoA ligase."]},{"entry":[{},"6.2.1.33","4-chlorobenzoate--CoA ligase."]},{"entry":[{},"6.2.1.34","Trans-feruloyl--CoA synthase."]},{"entry":[{},"6.3.1.1","Aspartate--ammonia ligase."]},{"entry":[{},"6.3.1.2","Glutamate--ammonia ligase."]},{"entry":[{},"6.3.1.4","Aspartate--ammonia ligase (ADP-"]},{"entry":[{},{},"forming)."]},{"entry":[{},"6.3.1.5","NAD(+) synthase."]},{"entry":[{},"6.3.1.6","Glutamate--ethylamine ligase."]},{"entry":[{},"6.3.1.7","4-methyleneglutamate--ammonia"]},{"entry":[{},{},"ligase."]},{"entry":[{},"6.3.1.8","Glutathionylspermidine synthase."]},{"entry":[{},"6.3.1.9","Trypanothione synthase."]},{"entry":[{},"6.3.1.10","Adenosylcobinamide-phosphate"]},{"entry":[{},{},"synthase."]},{"entry":[{},"6.3.2.1","Pantoate--beta-alanine ligase."]},{"entry":[{},"6.3.2.2","Glutamate--cysteine ligase."]},{"entry":[{},"6.3.2.3","Glutathione synthase."]},{"entry":[{},"6.3.2.4","D-alanine--D-alanine ligase."]},{"entry":[{},"6.3.2.5","Phosphopantothenate--cysteine ligase."]},{"entry":[{},"6.3.2.6","Phosphoribosylaminoimidazolesuccinocarboxamide"]},{"entry":[{},{},"synthase."]},{"entry":[{},"6.3.2.7","UDP-N-acetylmuramoyl-L-alanyl-D-"]},{"entry":[{},{},"glutamate--L-lysine ligase."]},{"entry":[{},"6.3.2.8","UDP-N-acetylmuramate--L-alanine"]},{"entry":[{},{},"ligase."]},{"entry":[{},"6.3.2.9","UDP-N-acetylmuramoylalanine--D-"]},{"entry":[{},{},"glutamate ligase."]},{"entry":[{},"6.3.2.10","UDP-N-acetylmuramoyl-tripeptide--"]},{"entry":[{},{},"D-alanyl-D-alanine ligase."]},{"entry":[{},"6.3.2.11","Carnosine synthase."]},{"entry":[{},"6.3.2.12","Dihydrofolate synthase."]},{"entry":[{},"6.3.2.13","UDP-N-acetylmuramoylalanyl-D-"]},{"entry":[{},{},"glutamate--2,6-diaminopimelate ligase."]},{"entry":[{},"6.3.2.14","2,3-dihydroxybenzoate--serine"]},{"entry":[{},{},"ligase."]},{"entry":[{},"6.3.2.16","D-alanine--alanyl-"]},{"entry":[{},{},"poly(glycerolphosphate) ligase."]},{"entry":[{},"6.3.2.17","Tetrahydrofolylpolyglutamate"]},{"entry":[{},{},"synthase."]},{"entry":[{},"6.3.2.18","Gamma-glutamylhistamine"]},{"entry":[{},{},"synthase."]},{"entry":[{},"6.3.2.19","Ubiquitin--protein ligase."]},{"entry":[{},"6.3.2.20","Indoleacetate-lysine synthetase."]},{"entry":[{},"6.3.2.21","Ubiquitin--calmodulin ligase."]},{"entry":[{},"6.3.2.22","Diphthine--ammonia ligase."]},{"entry":[{},"6.3.2.23","Homoglutathione synthase."]},{"entry":[{},"6.3.2.24","Tyrosine--arginine ligase."]},{"entry":[{},"6.3.2.25","Tubulin--tyrosine ligase."]},{"entry":[{},"6.3.2.26","N-(5-amino-5-carboxypentanoyl)-L-"]},{"entry":[{},{},"cysteinyl-D-valine synthase."]},{"entry":[{},"6.3.2.27","Aerobactin synthase."]},{"entry":[{},"6.3.3.1","Phosphoribosylformylglycinamidine"]},{"entry":[{},{},"cyclo-ligase."]},{"entry":[{},"6.3.3.2","5-formyltetrahydrofolate cyclo-"]},{"entry":[{},{},"ligase."]},{"entry":[{},"6.3.3.3","Dethiobiotin synthase."]},{"entry":[{},"6.3.3.4","(Carboxyethyl)arginine beta-lactam-"]},{"entry":[{},{},"synthase."]},{"entry":[{},"6.3.4.1","GMP synthase."]},{"entry":[{},"6.3.4.2","CTP synthase."]},{"entry":[{},"6.3.4.3","Formate--tetrahydrofolate ligase."]},{"entry":[{},"6.3.4.4","Adenylosuccinate synthase."]},{"entry":[{},"6.3.4.5","Argininosuccinate synthase."]},{"entry":[{},"6.3.4.6","Urea carboxylase."]},{"entry":[{},"6.3.4.7","Ribose-5-phosphate--ammonia"]},{"entry":[{},{},"ligase."]},{"entry":[{},"6.3.4.8","Imidazoleacetate--"]},{"entry":[{},{},"phosphoribosyldiphosphate ligase."]},{"entry":[{},"6.3.4.9","Biotin--[methylmalonyl-CoA-"]},{"entry":[{},{},"carboxytransferase] ligase."]},{"entry":[{},"6.3.4.10","Biotin--[propionyl-CoA-"]},{"entry":[{},{},"carboxylase (ATP-hydrolyzing)] ligase."]},{"entry":[{},"6.3.4.11","Biotin--[methylcrotonoyl-CoA-"]},{"entry":[{},{},"carboxylase] ligase."]},{"entry":[{},"6.3.4.12","Glutamate--methylamine ligase."]},{"entry":[{},"6.3.4.13","Phosphoribosylamine-glycine"]},{"entry":[{},{},"ligase."]},{"entry":[{},"6.3.4.14","Biotin carboxylase."]},{"entry":[{},"6.3.4.15","Biotin--[acetyl-CoA-carboxylase] ligase."]},{"entry":[{},"6.3.4.16","Carbamoyl-phosphate synthase"]},{"entry":[{},{},"(ammonia)."]},{"entry":[{},"6.3.4.17","Formate--dihydrofolate ligase."]},{"entry":[{},"6.3.5.1","NAD(+) synthase (glutamine-"]},{"entry":[{},{},"hydrolyzing)."]},{"entry":[{},"6.3.5.2","GMP synthase (glutamine-hydrolyzing)."]},{"entry":[{},"6.3.5.3","Phosphoribosylformylglycinamidine"]},{"entry":[{},{},"synthase."]},{"entry":[{},"6.3.5.4","Asparagine synthase (glutamine-"]},{"entry":[{},{},"hydrolyzing)."]},{"entry":[{},"6.3.5.5","Carbamoyl-phosphate synthase"]},{"entry":[{},{},"(glutamine-hydrolyzing)."]},{"entry":[{},"6.3.5.6","Asparaginyl-tRNA synthase (glutamine-"]},{"entry":[{},{},"hydrolyzing)."]},{"entry":[{},"6.3.5.7","Glutaminyl-tRNA synthase (glutamine-"]},{"entry":[{},{},"hydrolyzing)."]},{"entry":[{},"6.3.5.8","Aminodeoxychorismate synthase."]},{"entry":[{},"6.3.5.9","Hydrogenobyrinic acid a,c-diamide"]},{"entry":[{},{},"synthase (glutamine-hydrolyzing)."]},{"entry":[{},"6.3.5.10","Adenosylcobyric acid synthase"]},{"entry":[{},{},"(glutamine-hydrolyzing)."]},{"entry":[{},"6.4.1.1","Pyruvate carboxylase."]},{"entry":[{},"6.4.1.2","Acetyl-CoA carboxylase."]},{"entry":[{},"6.4.1.3","Propionyl-CoA carboxylase."]},{"entry":[{},"6.4.1.4","Methylcrotonoyl-CoA carboxylase."]},{"entry":[{},"6.4.1.5","Geranoyl-CoA carboxylase."]},{"entry":[{},"6.4.1.6","Acetone carboxylase."]},{"entry":[{},"6.5.1.1","DNA ligase (ATP)."]},{"entry":[{},"6.5.1.2","DNA ligase (NAD+)."]},{"entry":[{},"6.5.1.3","RNA ligase (ATP)."]},{"entry":[{},"6.5.1.4","RNA-3\u2032-phosphate cyclase."]},{"entry":[{},"6.6.1.1","Magnesium chelatase."]},{"entry":[{},"6.6.1.2","Cobaltochelatase."]},{"entry":[{},"6.3.4.17","Formate--dihydrofolate ligase."]},{"entry":[{},"6.3.5.1","NAD(+) synthase (glutamine-"]},{"entry":[{},{},"hydrolyzing)."]},{"entry":[{},"6.3.5.2","GMP synthase (glutamine-hydrolyzing)."]},{"entry":[{},"6.3.5.3","Phosphoribosylformylglycinamidine"]},{"entry":[{},{},"synthase."]},{"entry":[{},"6.3.5.4","Asparagine synthase (glutamine-"]},{"entry":[{},{},"hydrolyzing)."]},{"entry":[{},"6.3.5.5","Carbamoyl-phosphate synthase"]},{"entry":[{},{},"(glutamine-hydrolyzing)."]},{"entry":[{},"6.3.5.6","Asparaginyl-tRNA synthase (glutamine-"]},{"entry":[{},{},"hydrolyzing)."]},{"entry":[{},"6.3.5.7","Glutaminyl-tRNA synthase (glutamine-"]},{"entry":[{},{},"hydrolyzing)."]},{"entry":[{},"6.3.5.8","Aminodeoxychorismate synthase."]},{"entry":[{},"6.3.5.9","Hydrogenobyrinic acid a,c-diamide"]},{"entry":[{},{},"synthase (glutamine-hydrolyzing)."]},{"entry":[{},"6.3.5.10","Adenosylcobyric acid synthase (glutamine-"]},{"entry":[{},{},"hydrolyzing)."]},{"entry":[{},"6.4.1.1","Pyruvate carboxylase."]},{"entry":[{},"6.4.1.2","Acetyl-CoA carboxylase."]},{"entry":[{},"6.4.1.3","Propionyl-CoA carboxylase."]},{"entry":[{},"6.4.1.4","Methylcrotonoyl-CoA carboxylase."]},{"entry":[{},"6.4.1.5","Geranoyl-CoA carboxylase."]},{"entry":[{},"6.4.1.6","Acetone carboxylase."]},{"entry":[{},"6.5.1.1","DNA ligase (ATP)."]},{"entry":[{},"6.5.1.2","DNA ligase (NAD+)."]},{"entry":[{},"6.5.1.3","RNA ligase (ATP)."]},{"entry":[{},"6.5.1.4","RNA-3\u2032-phosphate cyclase."]},{"entry":[{},"6.6.1.1","Magnesium chelatase."]},{"entry":[{},"6.6.1.2","Cobaltochelatase."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"Table 3 summarizes exemplary functions of exemplary enzymes of the invention; these enzyme functions were determined using sequence identity comparison analysis using closest BLAST hits to the exemplary polypeptides and polynucleotides of the invention.","The invention also provides isolated and recombinant nucleic acids encoding polypeptides, e.g., SEQ ID NO:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9, etc., and all additional nucleic acids disclosed in the SEQ ID listing, which include all odd numbered SEQ ID NO:s from SEQ ID NO:1 through SEQ ID NO:26,897 (the exemplary polynucleotides of the invention). The invention also provides isolated and recombinant polypeptides, SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, etc., and all polypeptides disclosed in the SEQ ID listing, which include all even numbered SEQ ID NO:s from SEQ ID NO:2 through SEQ ID NO:26,898 (the exemplary polypeptides of the invention).","In another embodiment, the polypeptides of the invention can be expressed in any expression system, in vitro or in vivo, e.g., any microorganism or other cell system (e.g., eukaryotic, such as yeast or mammalian cells) using procedures known in the art. In other aspects, the polypeptides of the invention can be immobilized on a solid support prior to use in the methods of the invention. Methods for immobilizing enzymes on solid supports are commonly known in the art, for example J. Mol. Cat. B: Enzymatic 6 (1999) 29-39; Chivata et al. Biocatalysis: Immobilized cells and enzymes, J. Mol. Cat. 37 (1986) 1-24: Sharma et al., Immobilized Biomaterials Techniques and Applications, Angew. Chem. Int. Ed. Engl. 21 (1982) 837-54: Laskin (Ed.), Enzymes and Immobilized Cells in Biotechnology.","A \u201ccoding sequence of\u201d or a \u201csequence encodes\u201d a particular polypeptide or protein, is a nucleic acid sequence which is transcribed and translated into a polypeptide or protein when placed under the control of appropriate regulatory sequences.","A promoter sequence is \u201coperably linked to\u201d a coding sequence when RNA polymerase which initiates transcription at the promoter will transcribe the coding sequence into mRNA.","The phrase \u201csubstantially identical\u201d in the context of two nucleic acids or polypeptides, refers to two or more sequences that have, e.g., at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more nucleotide or amino acid residue (sequence) identity, when compared and aligned for maximum correspondence, as measured using one of the known sequence comparison algorithms or by visual inspection. In alternative aspects, the substantial identity exists over a region of at least about 100 or more residues and most commonly the sequences are substantially identical over at least about 150 to 200 or more residues. In some aspects, the sequences are substantially identical over the entire length of the coding regions.","Additionally a \u201csubstantially identical\u201d amino acid sequence is a sequence that differs from a reference sequence by one or more conservative or non-conservative amino acid substitutions, deletions, or insertions. In one aspect, the substitution occurs at a site that is not the active site of the molecule, or, alternatively the substitution occurs at a site that is the active site of the molecule, provided that the polypeptide essentially retains its functional (enzymatic) properties. A conservative amino acid substitution, for example, substitutes one amino acid for another of the same class (e.g., substitution of one hydrophobic amino acid, such as isoleucine, valine, leucine, or methionine, for another, or substitution of one polar amino acid for another, such as substitution of arginine for lysine, glutamic acid for aspartic acid or glutamine for asparagine). One or more amino acids can be deleted, for example, from a polypeptide, resulting in modification of the structure of the polypeptide, without significantly altering its biological activity. For example, amino- or carboxyl-terminal amino acids that are not required for a polypeptide, enzyme, protein, e.g. structural or binding protein, biological activity can be removed. Modified polypeptide sequences of the invention can be assayed for enzyme, structural or binding activity by any number of methods, including contacting the modified polypeptide sequence with a substrate and determining whether the modified polypeptide decreases the amount of specific substrate in the assay or increases the bioproducts of the reaction of a functional polypeptide, enzyme, protein, e.g. structural or binding protein, with the substrate. Assays for enzyme activity are well known in the art.","\u201cFragments\u201d as used herein are a portion of a naturally occurring protein which can exist in at least two different conformations. Fragments can have the same or substantially the same amino acid sequence as the naturally occurring protein. Fragments which have different three dimensional structures as the naturally occurring protein are also included. An example of this, is a \u201cpro-form\u201d molecule, such as a low activity proprotein that can be modified by cleavage to produce a mature enzyme with significantly higher activity.","The term \u201cvariant\u201d refers to polynucleotides or polypeptides of the invention modified at one or more base pairs, codons, introns, exons, or amino acid residues (respectively) yet still retain the biological activity of a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention. Variants can be produced by any number of means included methods such as, for example, error-prone PCR, shuffling, oligonucleotide-directed mutagenesis, assembly PCR, sexual PCR mutagenesis, in vivo mutagenesis, cassette mutagenesis, recursive ensemble mutagenesis, exponential ensemble mutagenesis, site-specific mutagenesis, gene reassembly, GSSM and any combination thereof.","The term \u201csaturation mutagenesis\u201d, Gene Site Saturation Mutagenesis, or \u201cGSSM\u201d includes a method that uses degenerate oligonucleotide primers to introduce point mutations into a polynucleotide, as described in detail, below.","The term \u201coptimized directed evolution system\u201d or \u201coptimized directed evolution\u201d includes a method for reassembling fragments of related nucleic acid sequences, e.g., related genes, and explained in detail, below.","The term \u201csynthetic ligation reassembly\u201d or \u201cSLR\u201d includes a method of ligating oligonucleotide fragments in a non-stochastic fashion, and explained in detail, below.","Nucleic Acids","The invention provides nucleic acids (e.g., the exemplary SEQ ID NO:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO:11, etc., including all nucleic acids disclosed in the SEQ ID listing, which include all odd numbered SEQ ID NO:s from SEQ ID NO:1 through SEQ ID NO:26,897), including expression cassettes such as expression vectors, encoding polypeptides (e.g., enzymes) of the invention. The invention also includes methods for discovering new polypeptide (e.g., enzyme) sequences using the nucleic acids of the invention. The invention also includes methods for inhibiting the expression of enzymes, genes, transcripts and polypeptides using the nucleic acids of the invention. Also provided are methods for modifying the nucleic acids of the invention by, e.g., synthetic ligation reassembly, optimized directed evolution system and\/or saturation mutagenesis.","The nucleic acids of the invention can be made, isolated and\/or manipulated by, e.g., cloning and expression of cDNA libraries, amplification of message or genomic DNA by PCR, and the like. For example, exemplary sequences of the invention were initially derived from environmental sources.","In one aspect, the invention provides nucleic acids, and the polypeptides encoded by them, with a common novelty in that they are derived from a common source, e.g., an environmental or a bacterial source.","In practicing the methods of the invention, homologous genes can be modified by manipulating a template nucleic acid, as described herein. The invention can be practiced in conjunction with any method or protocol or device known in the art, which are well described in the scientific and patent literature.","The phrases \u201cnucleic acid\u201d or \u201cnucleic acid sequence\u201d as used herein refer to an oligonucleotide, nucleotide, polynucleotide, or to a fragment of any of these, to DNA or RNA of genomic or synthetic origin which may be single-stranded or double-stranded and may represent a sense or antisense (complementary) strand, to peptide nucleic acid (PNA), or to any DNA-like or RNA-like material, natural or synthetic in origin. The phrases \u201cnucleic acid\u201d or \u201cnucleic acid sequence\u201d includes oligonucleotide, nucleotide, polynucleotide, or to a fragment of any of these, to DNA or RNA (e.g., mRNA, rRNA, tRNA, iRNA) of genomic or synthetic origin which may be single-stranded or double-stranded and may represent a sense or antisense strand, to peptide nucleic acid (PNA), or to any DNA-like or RNA-like material, natural or synthetic in origin, including, e.g., iRNA, ribonucleoproteins (e.g., e.g., double stranded iRNAs, e.g., iRNPs). The term encompasses nucleic acids, i.e., oligonucleotides, containing known analogues of natural nucleotides. The term also encompasses nucleic-acid-like structures with synthetic backbones, see e.g., Mata (1997) Toxicol. Appl. Pharmacol. 144:189-197; Strauss-Soukup (1997) Biochemistry 36:8692-8698; Samstag (1996) Antisense Nucleic Acid Drug Dev 6:153-156. \u201cOligonucleotide\u201d includes either a single stranded polydeoxynucleotide or two complementary polydeoxynucleotide strands which may be chemically synthesized. Such synthetic oligonucleotides have no 5\u2032 phosphate and thus will not ligate to another oligonucleotide without adding a phosphate with an ATP in the presence of a kinase. A synthetic oligonucleotide can ligate to a fragment that has not been dephosphorylated.","A \u201ccoding sequence of\u201d or a \u201cnucleotide sequence encoding\u201d a particular polypeptide or protein, is a nucleic acid sequence which is transcribed and translated into a polypeptide or protein when placed under the control of appropriate regulatory sequences. The term \u201cgene\u201d means the segment of DNA involved in producing a polypeptide chain; it includes regions preceding and following the coding region (leader and trailer) as well as, where applicable, intervening sequences (introns) between individual coding segments (exons). \u201cOperably linked\u201d as used herein refers to a functional relationship between two or more nucleic acid (e.g., DNA) segments. Typically, it refers to the functional relationship of transcriptional regulatory sequence to a transcribed sequence. For example, a promoter is operably linked to a coding sequence, such as a nucleic acid of the invention, if it stimulates or modulates the transcription of the coding sequence in an appropriate host cell or other expression system. Generally, promoter transcriptional regulatory sequences that are operably linked to a transcribed sequence are physically contiguous to the transcribed sequence, i.e., they are cis-acting. However, some transcriptional regulatory sequences, such as enhancers, need not be physically contiguous or located in close proximity to the coding sequences whose transcription they enhance.","As used herein, the term \u201cpromoter\u201d includes all sequences capable of driving transcription of a coding sequence in a cell, e.g., a plant cell. Thus, promoters used in the constructs of the invention include cis-acting transcriptional control elements and regulatory sequences that are involved in regulating or modulating the timing and\/or rate of transcription of a gene. For example, a promoter can be a cis-acting transcriptional control element, including an enhancer, a promoter, a transcription terminator, an origin of replication, a chromosomal integration sequence, 5\u2032 and 3\u2032 untranslated regions, or an intronic sequence, which are involved in transcriptional regulation. These cis-acting sequences typically interact with proteins or other biomolecules to carry out (turn on\/off, regulate, modulate, etc.) transcription. \u201cConstitutive\u201d promoters are those that drive expression continuously under most environmental conditions and states of development or cell differentiation. \u201cInducible\u201d or \u201cregulatable\u201d promoters direct expression of the nucleic acid of the invention under the influence of environmental conditions or developmental conditions. Examples of environmental conditions that may affect transcription by inducible promoters include anaerobic conditions, elevated temperature, drought, or the presence of light.","\u201cPlasmids\u201d can be commercially available, publicly available on an unrestricted basis, or can be constructed from available plasmids in accord with published procedures. Equivalent plasmids to those described herein are known in the art and will be apparent to the ordinarily skilled artisan.","In one aspect, the term \u201crecombinant\u201d means that the nucleic acid is adjacent to a \u201cbackbone\u201d nucleic acid to which it is not adjacent in its natural environment. Additionally, to be \u201cenriched\u201d the nucleic acids will represent 5% or more of the number of nucleic acid inserts in a population of nucleic acid backbone molecules. Backbone molecules according to the invention include nucleic acids such as expression vectors, self-replicating nucleic acids, viruses, integrating nucleic acids and other vectors or nucleic acids used to maintain or manipulate a nucleic acid insert of interest. Typically, the enriched nucleic acids represent 15% or more of the number of nucleic acid inserts in the population of recombinant backbone molecules. More typically, the enriched nucleic acids represent 50% or more of the number of nucleic acid inserts in the population of recombinant backbone molecules. In a one aspect, the enriched nucleic acids represent 90% or more of the number of nucleic acid inserts in the population of recombinant backbone molecules.","One aspect of the invention is an isolated nucleic acid comprising one of the sequences of the invention, or a fragment comprising at least 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, 150, 200, 300, 400, or 500 or more consecutive bases of a nucleic acid of the invention. The isolated, nucleic acids may comprise DNA, including cDNA, genomic DNA and synthetic DNA. The DNA may be double-stranded or single-stranded and if single stranded may be the coding strand or non-coding (anti-sense) strand. Alternatively, the isolated nucleic acids may comprise RNA.","The isolated nucleic acids of the invention may be used to prepare one of the polypeptides of the invention, or fragments comprising at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 or more consecutive amino acids of one of the polypeptides of the invention. Accordingly, another aspect of the invention is an isolated nucleic acid which encodes one of the polypeptides of the invention, or fragments comprising at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 or more consecutive amino acids of one of the polypeptides of the invention. The coding sequences of these nucleic acids may be identical to one of the coding sequences of one of the nucleic acids of the invention or may be different coding sequences which encode one of the of the invention having at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 or more consecutive amino acids of one of the polypeptides of the invention, as a result of the redundancy or degeneracy of the genetic code. The genetic code is well known to those of skill in the art and can be obtained, e.g., on page 214 of B. Lewin, Genes VI, Oxford University Press, 1997.","The isolated nucleic acid which encodes one of the polypeptides of the invention, but is not limited to: only the coding sequence of a nucleic acid of the invention and additional coding sequences, such as leader sequences or proprotein sequences and non-coding sequences, such as introns or non-coding sequences 5\u2032 and\/or 3\u2032 of the coding sequence. Thus, as used herein, the term \u201cpolynucleotide encoding a polypeptide\u201d encompasses a polynucleotide which includes only the coding sequence for the polypeptide as well as a polynucleotide which includes additional coding and\/or non-coding sequence.","Alternatively, the nucleic acid sequences of the invention, may be mutagenized using conventional techniques, such as site directed mutagenesis, or other techniques familiar to those skilled in the art, to introduce silent changes into the polynucleotides o of the invention. As used herein, \u201csilent changes\u201d include, for example, changes which do not alter the amino acid sequence encoded by the polynucleotide. Such changes may be desirable in order to increase the level of the polypeptide produced by host cells containing a vector encoding the polypeptide by introducing codons or codon pairs which occur frequently in the host organism.","The invention also relates to polynucleotides which have nucleotide changes which result in amino acid substitutions, additions, deletions, fusions and truncations in the polypeptides of the invention. Such nucleotide changes may be introduced using techniques such as site directed mutagenesis, random chemical mutagenesis, exonuclease III deletion and other recombinant DNA techniques. Alternatively, such nucleotide changes may be naturally occurring allelic variants which are isolated by identifying nucleic acids which specifically hybridize to probes comprising at least 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, 150, 200, 300, 400, or 500 consecutive bases of one of the sequences of the invention (or the sequences complementary thereto) under conditions of high, moderate, or low stringency as provided herein.","General Techniques","The nucleic acids used to practice this invention, whether RNA, iRNA, antisense nucleic acid, cDNA, genomic DNA, vectors, viruses or hybrids thereof, may be isolated from a variety of sources, genetically engineered, amplified, and\/or expressed\/generated recombinantly. Recombinant polypeptides generated from these nucleic acids can be individually isolated or cloned and tested for a desired activity. Any recombinant expression system can be used, including bacterial, mammalian, yeast, insect or plant cell expression systems.","Alternatively, these nucleic acids can be synthesized in vitro by well-known chemical synthesis techniques, as described in, e.g., Adams (1983) J. Am. Chem. Soc. 105:661; Belousov (1997) Nucleic Acids Res. 25:3440-3444; Frenkel (1995) Free Radic. Biol. Med. 19:373-380; Blommers (1994) Biochemistry 33:7886-7896; Narang (1979) Meth. Enzymol. 68:90; Brown (1979) Meth. Enzymol. 68:109; Beaucage (1981) Tetra. Lett. 22:1859; U.S. Pat. No. 4,458,066.","Techniques for the manipulation of nucleic acids, such as, e.g., subcloning, labeling probes (e.g., random-primer labeling using Klenow polymerase, nick translation, amplification), sequencing, hybridization and the like are well described in the scientific and patent literature, see, e.g., Sambrook, ed., MOLECULAR CLONING: A LABORATORY MANUAL (2ND ED.), Vols. 1-3, Cold Spring Harbor Laboratory, (1989); CURRENT PROTOCOLS 1N MOLECULAR BIOLOGY, Ausubel, ed. John Wiley & Sons, Inc., New York (1997); LABORATORY TECHNIQUES IN BIOCHEMISTRY AND MOLECULAR BIOLOGY: HYBRIDIZATION WITH NUCLEIC ACID PROBES, Part I. Theory and Nucleic Acid Preparation, Tijssen, ed. Elsevier, N.Y. (1993).","Another useful means of obtaining and manipulating nucleic acids used to practice the methods of the invention is to clone from genomic samples, and, if desired, screen and re-clone inserts isolated or amplified from, e.g., genomic clones or cDNA clones. Sources of nucleic acid used in the methods of the invention include genomic or cDNA libraries contained in, e.g., mammalian artificial chromosomes (MACs), see, e.g., U.S. Pat. Nos. 5,721,118; 6,025,155; human artificial chromosomes, see, e.g., Rosenfeld (1997) Nat. Genet. 15:333-335; yeast artificial chromosomes (YAC); bacterial artificial chromosomes (BAC); P1 artificial chromosomes, see, e.g., Woon (1998) Genomics 50:306-316; P1-derived vectors (PACs), see, e.g., Kern (1997) Biotechniques 23:120-124; cosmids, recombinant viruses, phages or plasmids.","In one aspect, a nucleic acid encoding a polypeptide of the invention is assembled in appropriate phase with a leader sequence capable of directing secretion of the translated polypeptide or fragment thereof.","The invention provides fusion proteins and nucleic acids encoding them. A polypeptide of the invention can be fused to a heterologous peptide or polypeptide, such as N-terminal identification peptides which impart desired characteristics, such as increased stability or simplified purification. Peptides and polypeptides of the invention can also be synthesized and expressed as fusion proteins with one or more additional domains linked thereto for, e.g., producing a more immunogenic peptide, to more readily isolate a recombinantly synthesized peptide, to identify and isolate antibodies and antibody-expressing B cells, and the like. Detection and purification facilitating domains include, e.g., metal chelating peptides such as polyhistidine tracts and histidine-tryptophan modules that allow purification on immobilized metals, protein A domains that allow purification on immobilized immunoglobulin, and the domain utilized in the FLAGS extension\/affinity purification system (Immunex Corp, Seattle Wash.). The inclusion of a cleavable linker sequences such as Factor Xa or enterokinase (Invitrogen, San Diego Calif.) between a purification domain and the motif-comprising peptide or polypeptide to facilitate purification. For example, an expression vector can include an epitope-encoding nucleic acid sequence linked to six histidine residues followed by a thioredoxin and an enterokinase cleavage site (see e.g., Williams (1995) Biochemistry 34:1787-1797; Dobeli (1998) Protein Expr. Purif. 12:404-414). The histidine residues facilitate detection and purification while the enterokinase cleavage site provides a means for purifying the epitope from the remainder of the fusion protein. Technology pertaining to vectors encoding fusion proteins and application of fusion proteins are well described in the scientific and patent literature, see e.g., Kroll (1993) DNA Cell. Biol., 12:441-53.","Transcriptional and Translational Control Sequences","The invention provides nucleic acid (e.g., DNA) sequences of the invention operatively linked to expression (e.g., transcriptional or translational) control sequence(s), e.g., promoters or enhancers, to direct or modulate RNA synthesis\/expression. The expression control sequence can be in an expression vector. Exemplary bacterial promoters include lacI, lacZ, T3, T7, gpt, lambda PR, PL and trp. Exemplary eukaryotic promoters include CMV immediate early, HSV thymidine kinase, early and late SV40, LTRs from retrovirus, and mouse metallothionein I.","Promoters suitable for expressing a polypeptide in bacteria include the lac or trp promoters, the lad promoter, the lacZ promoter, the T3 promoter, the T7 promoter, the gpt promoter, the lambda PR promoter, the lambda PL promoter, promoters from operons encoding glycolytic enzymes such as 3-phosphoglycerate kinase (PGK), and the acid phosphatase promoter. Eukaryotic promoters include the CMV immediate early promoter, the HSV thymidine kinase promoter, heat shock promoters, the early and late SV40 promoter, LTRs from retroviruses, and the mouse metallothionein-I promoter. Other promoters known to control expression of genes in prokaryotic or eukaryotic cells or their viruses may also be used. Promoters suitable for expressing the polypeptide or fragment thereof in bacteria include the lac or trp promoters, the lacI promoter, the lacZ promoter, the T3 promoter, the T7 promoter, the gpt promoter, the lambda Ppromoter, the lambda P, promoter, promoters from operons encoding glycolytic enzymes such as 3-phosphoglycerate kinase (PGK) and the acid phosphatase promoter. Fungal promoters include the \u03b1-factor promoter. Eukaryotic promoters include the CMV immediate early promoter, the HSV thymidine kinase promoter, heat shock promoters, the early and late SV40 promoter, LTRs from retroviruses and the mouse metallothionein-I promoter. Other promoters known to control expression of genes in prokaryotic or eukaryotic cells or their viruses may also be used.","Tissue-Specific Promoters","The invention provides expression cassettes that can be expressed in a tissue-specific manner, e.g., that can express a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention in a tissue-specific manner. The invention also provides plants or seeds that express a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention in a tissue-specific manner. The tissue-specificity can be seed specific, stem specific, leaf specific, root specific, fruit specific and the like.","The term \u201cexpression cassette\u201d as used herein refers to a nucleotide sequence which is capable of affecting expression of a structural gene (i.e., a protein coding sequence, such as a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention) in a host compatible with such sequences. Expression cassettes include at least a promoter operably linked with the polypeptide coding sequence; and, optionally, with other sequences, e.g., transcription termination signals. Additional factors necessary or helpful in effecting expression may also be used, e.g., enhancers, alpha-factors. Thus, expression cassettes also include plasmids, expression vectors, recombinant viruses, any form of recombinant \u201cnaked DNA\u201d vector, and the like. A \u201cvector\u201d comprises a nucleic acid which can infect, transfect, transiently or permanently transduce a cell. It will be recognized that a vector can be a naked nucleic acid, or a nucleic acid complexed with protein or lipid. The vector optionally comprises viral or bacterial nucleic acids and\/or proteins, and\/or membranes (e.g., a cell membrane, a viral lipid envelope, etc.). Vectors include, but are not limited to replicons (e.g., RNA replicons, bacteriophages) to which fragments of DNA may be attached and become replicated. Vectors thus include, but are not limited to RNA, autonomous self-replicating circular or linear DNA or RNA (e.g., plasmids, viruses, and the like, see, e.g., U.S. Pat. No. 5,217,879), and include both the expression and non-expression plasmids. Where a recombinant microorganism or cell culture is described as hosting an \u201cexpression vector\u201d this includes both extra-chromosomal circular and linear DNA and DNA that has been incorporated into the host chromosome(s). Where a vector is being maintained by a host cell, the vector may either be stably replicated by the cells during mitosis as an autonomous structure, or is incorporated within the host's genome.","\u201cTissue-specific\u201d promoters are transcriptional control elements that are only active in particular cells or tissues or organs, e.g., in plants or animals. Tissue-specific regulation may be achieved by certain intrinsic factors which ensure that genes encoding proteins specific to a given tissue are expressed. Such factors are known to exist in mammals and plants so as to allow for specific tissues to develop.","The term \u201cplant\u201d includes whole plants, plant parts (e.g., leaves, stems, flowers, roots, etc.), plant protoplasts, seeds and plant cells and progeny of same. The class of plants which can be used in the method of the invention is generally as broad as the class of higher plants amenable to transformation techniques, including angiosperms (monocotyledonous and dicotyledonous plants), as well as gymnosperms. It includes plants of a variety of ploidy levels, including polyploid, diploid, haploid and hemizygous states. As used herein, the term \u201ctransgenic plant\u201d includes plants or plant cells into which a heterologous nucleic acid sequence has been inserted, e.g., the nucleic acids and various recombinant constructs (e.g., expression cassettes) of the invention.","In one aspect, a constitutive promoter such as the CaMV 35S promoter can be used for expression in specific parts of the plant or seed or throughout the plant. For example, for overexpression, a plant promoter fragment can be employed which will direct expression of a nucleic acid in some or all tissues of a plant, e.g., a regenerated plant. Such promoters are referred to herein as \u201cconstitutive\u201d promoters and are active under most environmental conditions and states of development or cell differentiation. Examples of constitutive promoters include the cauliflower mosaic virus (CaMV) 35S transcription initiation region, the 1\u2032- or 2\u2032-promoter derived from T-DNA of , and other transcription initiation regions from various plant genes known to those of skill. Such genes include, e.g., ACT11 from (Huang (1996) 33:125-139); Cat3 from (GenBank No. U43147, Zhong (1996) 251:196-203); the gene encoding stearoyl-acyl carrier protein desaturase from (Genbank No. X74782, Solocombe (1994) 104:1167-1176); GPc1 from maize (GenBank No. X15596; Martinez (1989) 208:551-565); the Gpc2 from maize (GenBank No. U45855, Manjunath (1997) 33:97-112); plant promoters described in U.S. Pat. Nos. 4,962,028; 5,633,440.","The invention uses tissue-specific or constitutive promoters derived from viruses which can include, e.g., the tobamovirus subgenomic promoter (Kumagai (1995) Proc. Natl. Acad. Sci. USA 92:1679-1683; the rice tungro bacilliform virus (RTBV), which replicates only in phloem cells in infected rice plants, with its promoter which drives strong phloem-specific reporter gene expression; the cassava vein mosaic virus (CVMV) promoter, with highest activity in vascular elements, in leaf mesophyll cells, and in root tips (Verdaguer (1996) Plant Mol. Biol. 31:1129-1139).","Alternatively, the plant promoter may direct expression of a polypeptide, enzyme, protein, e.g. structural or binding protein-expressing nucleic acid in a specific tissue, organ or cell type (i.e. tissue-specific promoters) or may be otherwise under more precise environmental or developmental control or under the control of an inducible promoter. Examples of environmental conditions that may affect transcription include anaerobic conditions, elevated temperature, the presence of light, or sprayed with chemicals\/hormones. For example, the invention incorporates the drought-inducible promoter of maize (Busk (1997) supra); the cold, drought, and high salt inducible promoter from potato (Kirch (1997) Plant Mol. Biol. 33:897-909).","Tissue-specific promoters can promote transcription only within a certain time frame of developmental stage within that tissue. See, e.g., Blazquez (1998) Plant Cell 10:791-800, characterizing the LEAFY gene promoter. See also Cardon (1997) 12:367-77, describing the transcription factor SPL3, which recognizes a conserved sequence motif in the promoter region of the floral meristem identity gene AP1; and Mandel (1995) Plant Molecular Biology, Vol. 29, pp 995-1004, describing the meristem promoter eIF4. Tissue specific promoters which are active throughout the life cycle of a particular tissue can be used. In one aspect, the nucleic acids of the invention are operably linked to a promoter active primarily only in cotton fiber cells. In one aspect, the nucleic acids of the invention are operably linked to a promoter active primarily during the stages of cotton fiber cell elongation, e.g., as described by Rinehart (1996) supra. The nucleic acids can be operably linked to the Fbl2A gene promoter to be preferentially expressed in cotton fiber cells (Ibid). See also, John (1997) Proc. Natl. Acad. Sci. USA 89:5769-5773; John, et al., U.S. Pat. Nos. 5,608,148 and 5,602,321, describing cotton fiber-specific promoters and methods for the construction of transgenic cotton plants. Root-specific promoters may also be used to express the nucleic acids of the invention. Examples of root-specific promoters include the promoter from the alcohol dehydrogenase gene (DeLisle (1990) Int. Rev. Cytol. 123:39-60). Other promoters that can be used to express the nucleic acids of the invention include, e.g., ovule-specific, embryo-specific, endosperm-specific, integument-specific, seed coat-specific promoters, or some combination thereof; a leaf-specific promoter (see, e.g., Busk (1997) Plant J. 11:1285 1295, describing a leaf-specific promoter in maize); the ORF13 promoter from (which exhibits high activity in roots, see, e.g., Hansen (1997) supra); a maize pollen specific promoter (see, e.g., Guerrero (1990) Mol. Gen. Genet. 224:161 168); a tomato promoter active during fruit ripening, senescence and abscission of leaves and, to a lesser extent, of flowers can be used (see, e.g., Blume (1997) Plant J. 12:731 746); a pistil-specific promoter from the potato SK2 gene (see, e.g., Ficker (1997) Plant Mol. Biol. 35:425 431); the Blec4 gene from pea, which is active in epidermal tissue of vegetative and floral shoot apices of transgenic alfalfa making it a useful tool to target the expression of foreign genes to the epidermal layer of actively growing shoots or fibers; the ovule-specific BEL1 gene (see, e.g., Reiser (1995) Cell 83:735-742, GenBank No. U39944); and\/or, the promoter in Klee, U.S. Pat. No. 5,589,583, describing a plant promoter region is capable of conferring high levels of transcription in meristematic tissue and\/or rapidly dividing cells.","Alternatively, plant promoters which are inducible upon exposure to plant hormones, such as auxins, are used to express the nucleic acids of the invention. For example, the invention can use the auxin-response elements E1 promoter fragment (AuxREs) in the soybean (L.) (Liu (1997) Plant Physiol. 115:397-407); the auxin-responsive GST6 promoter (also responsive to salicylic acid and hydrogen peroxide) (Chen (1996) Plant J. 10: 955-966); the auxin-inducible parC promoter from tobacco (Sakai (1996) 37:906-913); a plant biotin response element (Streit (1997) Mol. Plant Microbe Interact. 10:933-937); and, the promoter responsive to the stress hormone abscisic acid (Sheen (1996) Science 274:1900-1902).","The nucleic acids of the invention can also be operably linked to plant promoters which are inducible upon exposure to chemicals reagents which can be applied to the plant, such as herbicides or antibiotics. For example, the maize In2-2 promoter, activated by benzenesulfonamide herbicide safeners, can be used (De Veylder (1997) Plant Cell Physiol. 38:568-577); application of different herbicide safeners induces distinct gene expression patterns, including expression in the root, hydathodes, and the shoot apical meristem. Coding sequence can be under the control of, e.g., a tetracycline-inducible promoter, e.g., as described with transgenic tobacco plants containing the L. (oat) arginine decarboxylase gene (Masgrau (1997) Plant J. 11:465-473); or, a salicylic acid-responsive element (Stange (1997) Plant J. 11:1315-1324). Using chemically- (e.g., hormone- or pesticide-) induced promoters, i.e., promoter responsive to a chemical which can be applied to the transgenic plant in the field, expression of a polypeptide of the invention can be induced at a particular stage of development of the plant. Thus, the invention also provides for transgenic plants containing an inducible gene encoding for polypeptides of the invention whose host range is limited to target plant species, such as corn, rice, barley, wheat, potato or other crops, inducible at any stage of development of the crop.","One of skill will recognize that a tissue-specific plant promoter may drive expression of operably linked sequences in tissues other than the target tissue. Thus, a tissue-specific promoter is one that drives expression preferentially in the target tissue or cell type, but may also lead to some expression in other tissues as well.","The nucleic acids of the invention can also be operably linked to plant promoters which are inducible upon exposure to chemicals reagents. These reagents include, e.g., herbicides, synthetic auxins, or antibiotics which can be applied, e.g., sprayed, onto transgenic plants. Inducible expression of the polypeptide, enzyme, protein, e.g. structural or binding protein-producing nucleic acids of the invention will allow the grower to select plants with the optimal polypeptide, enzyme, protein, e.g. structural or binding protein, expression and\/or activity. The development of plant parts can thus controlled. In this way the invention provides the means to facilitate the harvesting of plants and plant parts. For example, in various embodiments, the maize In2-2 promoter, activated by benzenesulfonamide herbicide safeners, is used (De Veylder (1997) Plant Cell Physiol. 38:568-577); application of different herbicide safeners induces distinct gene expression patterns, including expression in the root, hydathodes, and the shoot apical meristem. Coding sequences of the invention are also under the control of a tetracycline-inducible promoter, e.g., as described with transgenic tobacco plants containing the L. (oat) arginine decarboxylase gene (Masgrau (1997) Plant J. 11:465-473); or, a salicylic acid-responsive element (Stange (1997) Plant J. 11:1315-1324).","In some aspects, proper polypeptide expression may require polyadenylation region at the 3\u2032-end of the coding region. The polyadenylation region can be derived from the natural gene, from a variety of other plant (or animal or other) genes, or from genes in the T-DNA.","Expression Vectors and Cloning Vehicles","The invention provides expression vectors and cloning vehicles comprising nucleic acids of the invention, e.g., sequences encoding the polypeptide, enzyme, protein, e.g. structural or binding proteins of the invention. Expression vectors and cloning vehicles of the invention can comprise viral particles, baculovirus, phage, plasmids, phagemids, cosmids, fosmids, bacterial artificial chromosomes, viral DNA (e.g., vaccinia, adenovirus, foul pox virus, pseudorabies and derivatives of SV40), P1-based artificial chromosomes, yeast plasmids, yeast artificial chromosomes, and any other vectors specific for specific hosts of interest (such as and yeast). Vectors of the invention can include chromosomal, non-chromosomal and synthetic DNA sequences. Large numbers of suitable vectors are known to those of skill in the art, and are commercially available. Exemplary vectors are include: bacterial: pQE vectors (Qiagen), pBLUESCRIPT plasmids, pNH vectors, (lambda-ZAP vectors (Stratagene); ptrc99a, pKK223-3, pDR540, pRIT2T (Pharmacia); Eukaryotic: pXT1, pSG5 (Stratagene), pSVK3, pBPV, pMSG, pSVLSV40 (Pharmacia). However, any other plasmid or other vector may be used so long as they are replicable and viable in the host. Low copy number or high copy number vectors may be employed with the present invention.","The expression vector can comprise a promoter, a ribosome binding site for translation initiation and a transcription terminator. The vector may also include appropriate sequences for amplifying expression. Mammalian expression vectors can comprise an origin of replication, any necessary ribosome binding sites, a polyadenylation site, splice donor and acceptor sites, transcriptional termination sequences, and 5\u2032 flanking non-transcribed sequences. In some aspects, DNA sequences derived from the SV40 splice and polyadenylation sites may be used to provide the required non-transcribed genetic elements.","In one aspect, the expression vectors contain one or more selectable marker genes to permit selection of host cells containing the vector. Such selectable markers include genes encoding dihydrofolate reductase or genes conferring neomycin resistance for eukaryotic cell culture, genes conferring tetracycline or ampicillin resistance in , and the TRP 1 gene. Promoter regions can be selected from any desired gene using chloramphenicol transferase (CAT) vectors or other vectors with selectable markers.","Vectors for expressing the polypeptide or fragment thereof in eukaryotic cells can also contain enhancers to increase expression levels. Enhancers are cis-acting elements of DNA that can be from about 10 to about 300 bp in length. They can act on a promoter to increase its transcription. Exemplary enhancers include the SV40 enhancer on the late side of the replication origin by 100 to 270, the cytomegalovirus early promoter enhancer, the polyoma enhancer on the late side of the replication origin, and the adenovirus enhancers.","A nucleic acid sequence can be inserted into a vector by a variety of procedures. In general, the sequence is ligated to the desired position in the vector following digestion of the insert and the vector with appropriate restriction endonucleases. Alternatively, blunt ends in both the insert and the vector may be ligated. A variety of cloning techniques are known in the art, e.g., as described in Ausubel and Sambrook. Such procedures and others are deemed to be within the scope of those skilled in the art.","The vector can be in the form of a plasmid, a viral particle, or a phage. Other vectors include chromosomal, non-chromosomal and synthetic DNA sequences, derivatives of SV40; bacterial plasmids, phage DNA, baculovirus, yeast plasmids, vectors derived from combinations of plasmids and phage DNA, viral DNA such as vaccinia, adenovirus, fowl pox virus, and pseudorabies. A variety of cloning and expression vectors for use with prokaryotic and eukaryotic hosts are described by, e.g., Sambrook.","Particular bacterial vectors which can be used include the commercially available plasmids comprising genetic elements of the well known cloning vector pBR322 (ATCC 37017), pKK223-3 (Pharmacia Fine Chemicals, Uppsala, Sweden), GEM1 (Promega Biotec, Madison, Wis., USA) pQE70, pQE60, pQE-9 (Qiagen), pD10, psiX174 pBLUESCRIPT II KS, pNH8A, pNH16a, pNH18A, pNH46A (Stratagene), ptrc99a, pKK223-3, pKK233-3, DR540, pRIT5 (Pharmacia), pKK232-8 and pCM7. Particular eukaryotic vectors include pSV2CAT, pOG44, pXT1, pSG (Stratagene) pSVK3, pBPV, pMSG, and pSVL (Pharmacia). However, any other vector may be used as long as it is replicable and viable in the host cell.","The nucleic acids of the invention can be expressed in expression cassettes, vectors or viruses and transiently or stably expressed in plant cells and seeds. One exemplary transient expression system uses episomal expression systems, e.g., cauliflower mosaic virus (CaMV) viral RNA generated in the nucleus by transcription of an episomal mini-chromosome containing supercoiled DNA, see, e.g., Covey (1990) Proc. Natl. Acad. Sci. USA 87:1633-1637. Alternatively, coding sequences, i.e., all or sub-fragments of sequences of the invention can be inserted into a plant host cell genome becoming an integral part of the host chromosomal DNA. Sense or antisense transcripts can be expressed in this manner. A vector comprising the sequences (e.g., promoters or coding regions) from nucleic acids of the invention can comprise a marker gene that confers a selectable phenotype on a plant cell or a seed. For example, the marker may encode biocide resistance, particularly antibiotic resistance, such as resistance to kanamycin, G418, bleomycin, hygromycin, or herbicide resistance, such as resistance to chlorosulfuron or Basta.","Expression vectors capable of expressing nucleic acids and proteins in plants are well known in the art, and can include, e.g., vectors from spp., potato virus X (see, e.g., Angell (1997) EMBO J. 16:3675-3684), tobacco mosaic virus (see, e.g., Casper (1996) Gene 173:69-73), tomato bushy stunt virus (see, e.g., Hillman (1989) Virology 169:42-50), tobacco etch virus (see, e.g., Dolja (1997) Virology 234:243-252), bean golden mosaic virus (see, e.g., Morinaga (1993) Microbiol Immunol. 37:471-476), cauliflower mosaic virus (see, e.g., Cecchini (1997) Mol. Plant Microbe Interact. 10:1094-1101), maize Ac\/Ds transposable element (see, e.g., Rubin (1997) Mol. Cell. Biol. 17:6294-6302; Kunze (1996) Curr. Top. Microbiol. Immunol. 204:161-194), and the maize suppressor-mutator (Spm) transposable element (see, e.g., Schlappi (1996) Plant Mol. Biol. 32:717-725); and derivatives thereof.","In one aspect, the expression vector can have two replication systems to allow it to be maintained in two organisms, for example in mammalian or insect cells for expression and in a prokaryotic host for cloning and amplification. Furthermore, for integrating expression vectors, the expression vector can contain at least one sequence homologous to the host cell genome. It can contain two homologous sequences which flank the expression construct. The integrating vector can be directed to a specific locus in the host cell by selecting the appropriate homologous sequence for inclusion in the vector. Constructs for integrating vectors are well known in the art.","Expression vectors of the invention may also include a selectable marker gene to allow for the selection of bacterial strains that have been transformed, e.g., genes which render the bacteria resistant to drugs such as ampicillin, chloramphenicol, erythromycin, kanamycin, neomycin and tetracycline. Selectable markers can also include biosynthetic genes, such as those in the histidine, tryptophan and leucine biosynthetic pathways.","The DNA sequence in the expression vector is operatively linked to an appropriate expression control sequence(s) (promoter) to direct RNA synthesis. Particular named bacterial promoters include lacI, lacZ, T3, T7, gpt, lambda P, P, and trp. Eukaryotic promoters include CMV immediate early, HSV thymidine kinase, early and late SV40, LTRs from retrovirus and mouse metallothionein-I. Selection of the appropriate vector and promoter is well within the level of ordinary skill in the art. The expression vector also contains a ribosome binding site for translation initiation and a transcription terminator. The vector may also include appropriate sequences for amplifying expression. Promoter regions can be selected from any desired gene using chloramphenicol transferase (CAT) vectors or other vectors with selectable markers. In addition, the expression vectors in one aspect contain one or more selectable marker genes to provide a phenotypic trait for selection of transformed host cells such as dihydrofolate reductase or neomycin resistance for eukaryotic cell culture, or such as tetracycline or ampicillin resistance in ","Mammalian expression vectors may also comprise an origin of replication, any necessary ribosome binding sites, a polyadenylation site, splice donor and acceptor sites, transcriptional termination sequences and 5\u2032 flanking nontranscribed sequences. In some aspects, DNA sequences derived from the SV40 splice and polyadenylation sites may be used to provide the required nontranscribed genetic elements.","Vectors for expressing the polypeptide or fragment thereof in eukaryotic cells may also contain enhancers to increase expression levels. Enhancers are cis-acting elements of DNA, usually from about 10 to about 300 bp in length that act on a promoter to increase its transcription. Examples include the SV40 enhancer on the late side of the replication origin by 100 to 270, the cytomegalovirus early promoter enhancer, the polyoma enhancer on the late side of the replication origin and the adenovirus enhancers.","In addition, the expression vectors typically contain one or more selectable marker genes to permit selection of host cells containing the vector. Such selectable markers include genes encoding dihydrofolate reductase or genes conferring neomycin resistance for eukaryotic cell culture, genes conferring tetracycline or ampicillin resistance in and the TRP1 gene.","In some aspects, the nucleic acid encoding one of the polypeptides of the invention, or fragments comprising at least about 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 consecutive amino acids thereof is assembled in appropriate phase with a leader sequence capable of directing secretion of the translated polypeptide or fragment thereof. Optionally, the nucleic acid can encode a fusion polypeptide in which one of the polypeptides of the invention, or fragments comprising at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 consecutive amino acids thereof is fused to heterologous peptides or polypeptides, such as N-terminal identification peptides which impart desired characteristics, such as increased stability or simplified purification.","The appropriate DNA sequence may be inserted into the vector by a variety of procedures. In general, the DNA sequence is ligated to the desired position in the vector following digestion of the insert and the vector with appropriate restriction endonucleases. Alternatively, blunt ends in both the insert and the vector may be ligated. A variety of cloning techniques are disclosed in Ausubel et al. Current Protocols in Molecular Biology, John Wiley 503 Sons, Inc. 1997 and Sambrook et al., Molecular Cloning: A Laboratory Manual 2nd Ed., Cold Spring Harbor Laboratory Press (1989. Such procedures and others are deemed to be within the scope of those skilled in the art.","The vector may be, for example, in the form of a plasmid, a viral particle, or a phage. Other vectors include chromosomal, nonchromosomal and synthetic DNA sequences, derivatives of SV40; bacterial plasmids, phage DNA, baculovirus, yeast plasmids, vectors derived from combinations of plasmids and phage DNA, viral DNA such as vaccinia, adenovirus, fowl pox virus and pseudorabies. A variety of cloning and expression vectors for use with prokaryotic and eukaryotic hosts are described by Sambrook, et al., Molecular Cloning: A Laboratory Manual, 2nd Ed., Cold Spring Harbor, N.Y., (1989).","Host Cells and Transformed Cells","The invention also provides a transformed cell comprising a nucleic acid sequence of the invention, e.g., a sequence encoding a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention, or a vector of the invention. The host cell may be any of the host cells familiar to those skilled in the art, including prokaryotic cells, eukaryotic cells, such as bacterial cells, fungal cells, yeast cells, mammalian cells, insect cells, or plant cells. Exemplary bacterial cells include and various species within the genera and . Exemplary insect cells include S2 and Sf9. Exemplary animal cells include CHO, COS or Bowes melanoma or any mouse or human cell line. The selection of an appropriate host is within the abilities of those skilled in the art. Techniques for transforming a wide variety of higher plant species are well known and described in the technical and scientific literature. See, e.g., Weising (1988) Ann. Rev. Genet. 22:421-477; U.S. Pat. No. 5,750,870.","The vector can be introduced into the host cells using any of a variety of techniques, including transformation, transfection, transduction, viral infection, gene guns, or Ti-mediated gene transfer. Particular methods include calcium phosphate transfection, DEAE-Dextran mediated transfection, lipofection, or electroporation (Davis, L., Dibner, M., Battey, I., Basic Methods in Molecular Biology, (1986)).","In one aspect, the nucleic acids or vectors of the invention are introduced into the cells for screening, thus, the nucleic acids enter the cells in a manner suitable for subsequent expression of the nucleic acid. The method of introduction is largely dictated by the targeted cell type. Exemplary methods include CaPOprecipitation, liposome fusion, lipofection (e.g., LIPOFECTIN\u2122), electroporation, viral infection, etc. The candidate nucleic acids may stably integrate into the genome of the host cell (for example, with retroviral introduction) or may exist either transiently or stably in the cytoplasm (i.e. through the use of traditional plasmids, utilizing standard regulatory sequences, selection markers, etc.). As many pharmaceutically important screens require human or model mammalian cell targets, retroviral vectors capable of transfecting such targets can be used.","Where appropriate, the engineered host cells can be cultured in conventional nutrient media modified as appropriate for activating promoters, selecting transformants or amplifying the genes of the invention. Following transformation of a suitable host strain and growth of the host strain to an appropriate cell density, the selected promoter may be induced by appropriate means (e.g., temperature shift or chemical induction) and the cells may be cultured for an additional period to allow them to produce the desired polypeptide or fragment thereof.","Cells can be harvested by centrifugation, disrupted by physical or chemical means, and the resulting crude extract is retained for further purification. Microbial cells employed for expression of proteins can be disrupted by any convenient method, including freeze-thaw cycling, sonication, mechanical disruption, or use of cell lysing agents. Such methods are well known to those skilled in the art. The expressed polypeptide or fragment thereof can be recovered and purified from recombinant cell cultures by methods including ammonium sulfate or ethanol precipitation, acid extraction, anion or cation exchange chromatography, phosphocellulose chromatography, hydrophobic interaction chromatography, affinity chromatography, hydroxylapatite chromatography and lectin chromatography. Protein refolding steps can be used, as necessary, in completing configuration of the polypeptide. If desired, high performance liquid chromatography (HPLC) can be employed for final purification steps.","The constructs in host cells can be used in a conventional manner to produce the gene product encoded by the recombinant sequence. Depending upon the host employed in a recombinant production procedure, the polypeptides produced by host cells containing the vector may be glycosylated or may be non-glycosylated. Polypeptides of the invention may or may not also include an initial methionine amino acid residue.","Cell-free translation systems can also be employed to produce a polypeptide of the invention. Cell-free translation systems can use mRNAs transcribed from a DNA construct comprising a promoter operably linked to a nucleic acid encoding the polypeptide or fragment thereof. In some aspects, the DNA construct may be linearized prior to conducting an in vitro transcription reaction. The transcribed mRNA is then incubated with an appropriate cell-free translation extract, such as a rabbit reticulocyte extract, to produce the desired polypeptide or fragment thereof.","The expression vectors can contain one or more selectable marker genes to provide a phenotypic trait for selection of transformed host cells such as dihydrofolate reductase or neomycin resistance for eukaryotic cell culture, or such as tetracycline or ampicillin resistance in ","Host cells containing the polynucleotides of interest, e.g., nucleic acids of the invention, can be cultured in conventional nutrient media modified as appropriate for activating promoters, selecting transformants or amplifying genes. The culture conditions, such as temperature, pH and the like, are those previously used with the host cell selected for expression and will be apparent to the ordinarily skilled artisan. The clones which are identified as having the specified enzyme activity may then be sequenced to identify the polynucleotide sequence encoding an enzyme having the enhanced activity.","The invention provides a method for overexpressing a recombinant polypeptide, enzyme, protein, e.g. structural or binding protein, in a cell comprising expressing a vector comprising a nucleic acid of the invention, e.g., a nucleic acid comprising a nucleic acid sequence with at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more sequence identity to an exemplary sequence of the invention over a region of at least about 100 residues, wherein the sequence identities are determined by analysis with a sequence comparison algorithm or by visual inspection, or, a nucleic acid that hybridizes under stringent conditions to a nucleic acid sequence of the invention. The overexpression can be effected by any means, e.g., use of a high activity promoter, a dicistronic vector or by gene amplification of the vector.","The nucleic acids of the invention can be expressed, or overexpressed, in any in vitro or in vivo expression system. Any cell culture systems can be employed to express, or over-express, recombinant protein, including bacterial, insect, yeast, fungal or mammalian cultures. Over-expression can be effected by appropriate choice of promoters, enhancers, vectors (e.g., use of replicon vectors, dicistronic vectors (see, e.g., Gurtu (1996) Biochem. Biophys. Res. Commun. 229:295-8), media, culture systems and the like. In one aspect, gene amplification using selection markers, e.g., glutamine synthetase (see, e.g., Sanders (1987) Dev. Biol. Stand. 66:55-63), in cell systems are used to overexpress the polypeptides of the invention.","The host cell may be any of the host cells familiar to those skilled in the art, including prokaryotic cells, eukaryotic cells, mammalian cells, insect cells, or plant cells. As representative examples of appropriate hosts, there may be mentioned: bacterial cells, such as and various species within the genera and , fungal cells, such as yeast, insect cells such as S2 and Sf9, animal cells such as CHO, COS or Bowes melanoma and adenoviruses. The selection of an appropriate host is within the abilities of those skilled in the art.","The vector may be introduced into the host cells using any of a variety of techniques, including transformation, transfection, transduction, viral infection, gene guns, or Ti-mediated gene transfer. Particular methods include calcium phosphate transfection, DEAE-Dextran mediated transfection, lipofection, or electroporation (Davis, L., Dibner, M., Battey, I., Basic Methods in Molecular Biology, (1986)).","Where appropriate, the engineered host cells can be cultured in conventional nutrient media modified as appropriate for activating promoters, selecting transformants or amplifying the genes of the invention. Following transformation of a suitable host strain and growth of the host strain to an appropriate cell density, the selected promoter may be induced by appropriate means (e.g., temperature shift or chemical induction) and the cells may be cultured for an additional period to allow them to produce the desired polypeptide or fragment thereof.","Cells are typically harvested by centrifugation, disrupted by physical or chemical means and the resulting crude extract is retained for further purification. Microbial cells employed for expression of proteins can be disrupted by any convenient method, including freeze-thaw cycling, sonication, mechanical disruption, or use of cell lysing agents. Such methods are well known to those skilled in the art. The expressed polypeptide or fragment thereof can be recovered and purified from recombinant cell cultures by methods including ammonium sulfate or ethanol precipitation, acid extraction, anion or cation exchange chromatography, phosphocellulose chromatography, hydrophobic interaction chromatography, affinity chromatography, hydroxylapatite chromatography and lectin chromatography. Protein refolding steps can be used, as necessary, in completing configuration of the polypeptide. If desired, high performance liquid chromatography (HPLC) can be employed for final purification steps.","Various mammalian cell culture systems can also be employed to express recombinant protein. Examples of mammalian expression systems include the COS-7 lines of monkey kidney fibroblasts (described by Gluzman, Cell, 23:175, 1981) and other cell lines capable of expressing proteins from a compatible vector, such as the C127, 3T3, CHO, HeLa and BHK cell lines.","The constructs in host cells can be used in a conventional manner to produce the gene product encoded by the recombinant sequence. Depending upon the host employed in a recombinant production procedure, the polypeptides produced by host cells containing the vector may be glycosylated or may be non-glycosylated. Polypeptides of the invention may or may not also include an initial methionine amino acid residue.","Alternatively, the polypeptides of the invention, or fragments comprising at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 or more consecutive amino acids thereof can be synthetically produced by conventional peptide synthesizers. In other aspects, fragments or portions of the polypeptides may be employed for producing the corresponding full-length polypeptide by peptide synthesis; therefore, the fragments may be employed as intermediates for producing the full-length polypeptides.","Cell-free translation systems can also be employed to produce one of the polypeptides of the invention, or fragments comprising at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 or more consecutive amino acids thereof using mRNAs transcribed from a DNA construct comprising a promoter operably linked to a nucleic acid encoding the polypeptide or fragment thereof. In some aspects, the DNA construct may be linearized prior to conducting an in vitro transcription reaction. The transcribed mRNA is then incubated with an appropriate cell-free translation extract, such as a rabbit reticulocyte extract, to produce the desired polypeptide or fragment thereof.","Amplification of Nucleic Acids","In practicing the invention, nucleic acids encoding the polypeptides of the invention, or modified nucleic acids, can be reproduced by, e.g., amplification. The invention provides amplification primer sequence pairs for amplifying nucleic acids encoding polypeptides (e.g., enzymes) of the invention. In one aspect, the primer pairs are capable of amplifying nucleic acid sequences of the invention, e.g., including the exemplary SEQ ID NO:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO:11, etc., including all nucleic acids disclosed in the SEQ ID listing, which include all odd numbered SEQ ID NO:s from SEQ ID NO:1 through SEQ ID NO:26,897, or a subsequence thereof, etc. One of skill in the art can design amplification primer sequence pairs for any part of or the full length of these sequences.","In one aspect, the invention provides a nucleic acid amplified by a primer pair of the invention, e.g., a primer pair as set forth by about the first (the 5\u2032) 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, or 25 or more residues of a nucleic acid of the invention, and about the first (the 5\u2032) 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, or 25 or more residues of the complementary strand.","The invention provides an amplification primer sequence pair for amplifying a nucleic acid encoding a polypeptide having an enzyme, structural or binding activity, wherein the primer pair is capable of amplifying a nucleic acid comprising a sequence of the invention, or fragments or subsequences thereof. One or each member of the amplification primer sequence pair can comprise an oligonucleotide comprising at least about 10 to 50 or more consecutive bases of the sequence, or about 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, or 25 or more consecutive bases of the sequence. The invention provides amplification primer pairs, wherein the primer pair comprises a first member having a sequence as set forth by about the first (the 5\u2032) 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, or 25 or more residues of a nucleic acid of the invention, and a second member having a sequence as set forth by about the first (the 5\u2032) 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, or 25 or more residues of the complementary strand of the first member. The invention provides a polypeptide, enzyme, protein, e.g. structural or binding protein, generated by amplification, e.g., polymerase chain reaction (PCR), using an amplification primer pair of the invention. The invention provides methods of making a polypeptide, enzyme, protein, e.g. structural or binding protein, by amplification, e.g., polymerase chain reaction (PCR), using an amplification primer pair of the invention. In one aspect, the amplification primer pair amplifies a nucleic acid from a library, e.g., a gene library, such as an environmental library.","Amplification reactions can also be used to quantify the amount of nucleic acid in a sample (such as the amount of message in a cell sample), label the nucleic acid (e.g., to apply it to an array or a blot), detect the nucleic acid, or quantify the amount of a specific nucleic acid in a sample. In one aspect of the invention, message isolated from a cell or a cDNA library are amplified.","The skilled artisan can select and design suitable oligonucleotide amplification primers. Amplification methods are also well known in the art, and include, e.g., polymerase chain reaction, PCR (see, e.g., PCR PROTOCOLS, A GUIDE TO METHODS AND APPLICATIONS, ed. Innis, Academic Press, N.Y. (1990) and PCR STRATEGIES (1995), ed. Innis, Academic Press, Inc., N.Y., ligase chain reaction (LCR) (see, e.g., Wu (1989) Genomics 4:560; Landegren (1988) Science 241:1077; Barringer (1990) Gene 89:117); transcription amplification (see, e.g., Kwoh (1989) Proc. Natl. Acad. Sci. USA 86:1173); and, self-sustained sequence replication (see, e.g., Guatelli (1990) Proc. Natl. Acad. Sci. USA 87:1874); Q Beta replicase amplification (see, e.g., Smith (1997) J. Clin. Microbiol. 35:1477-1491), automated Q-beta replicase amplification assay (see, e.g., Burg (1996) Mol. Cell. Probes 10:257-271) and other RNA polymerase mediated techniques (e.g., NASBA, Cangene, Mississauga, Ontario); see also Berger (1987) Methods Enzymol. 152:307-316; Sambrook; Ausubel; U.S. Pat. Nos. 4,683,195 and 4,683,202; Sooknanan (1995) Biotechnology 13:563-564.","Determining the Degree of Sequence Identity","The invention provides nucleic acids comprising sequences having at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more, or complete (100%) sequence identity to an exemplary nucleic acid of the invention (e.g., SEQ ID NO:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO:11, etc., including all nucleic acids disclosed in the SEQ ID listing, which include all even numbered SEQ ID NO:s from SEQ ID NO:1 through SEQ ID NO:26,897, and nucleic acids encoding SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, etc., and all polypeptides disclosed in the SEQ ID listing, which include all even numbered SEQ ID NO:s from SEQ ID NO:2 through SEQ ID NO:26,898) over a region of at least about 50, 75, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000, 1050, 1100, 1150, 1200, 1250, 1300, 1350, 1400, 1450, 1500, 1550 or more, residues. The invention provides polypeptides comprising sequences having at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more, or complete (100%) sequence identity to an exemplary polypeptide of the invention. The extent of sequence identity (homology) may be determined using any computer program and associated parameters, including those described herein, such as BLAST 2.2.2. or FASTA version 3.0t78, with the default parameters.","As used herein, the terms \u201ccomputer,\u201d \u201ccomputer program\u201d and \u201cprocessor\u201d are used in their broadest general contexts and incorporate all such devices, as described in detail, below.","Nucleic acid sequences of the invention can comprise at least 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, 150, 200, 300, 400, or 500 or more consecutive nucleotides of an exemplary sequence of the invention and sequences substantially identical thereto. Homologous sequences and fragments of nucleic acid sequences of the invention can refer to a sequence having at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more sequence identity (homology) to these sequences. Homology (sequence identity) may be determined using any of the computer programs and parameters described herein, including FASTA version 3.0t78 with the default parameters. Homologous sequences also include RNA sequences in which uridines replace the thymines in the nucleic acid sequences of the invention. The homologous sequences may be obtained using any of the procedures described herein or may result from the correction of a sequencing error. It will be appreciated that the nucleic acid sequences of the invention can be represented in the traditional single character format (See the inside back cover of Stryer, Lubert. Biochemistry, 3rd Ed., W. H Freeman & Co., New York.) or in any other format which records the identity of the nucleotides in a sequence.","Various sequence comparison programs identified elsewhere in this patent specification are particularly contemplated for use in this aspect of the invention. Protein and\/or nucleic acid sequence homologies may be evaluated using any of the variety of sequence comparison algorithms and programs known in the art. Such algorithms and programs include, but are by no means limited to, TBLASTN, BLASTP, FASTA, TFASTA and CLUSTALW (see, e.g., Pearson and Lipman, Proc. Natl. Acad. Sci. USA 85(8):2444-2448, 1988; Altschul et al., J. Mol. Biol. 215(3):403-410, 1990; Thompson Nucleic Acids Res. 22(2):4673-4680, 1994; Higgins et al., Methods Enzymol. 266:383-402, 1996; Altschul et al., J. Mol. Biol. 215(3):403-410, 1990; Altschul et al., Nature Genetics 3:266-272, 1993).","Homology or identity is often measured using sequence analysis software (e.g., Sequence Analysis Software Package of the Genetics Computer Group, University of Wisconsin Biotechnology Center, 1710 University Avenue, Madison, Wis. 53705). Such software matches similar sequences by assigning degrees of homology to various deletions, substitutions and other modifications. The terms \u201chomology\u201d and \u201cidentity\u201d in the context of two or more nucleic acids or polypeptide sequences, refer to two or more sequences or subsequences that are the same or have a specified percentage of amino acid residues or nucleotides that are the same when compared and aligned for maximum correspondence over a comparison window or designated region as measured using any number of sequence comparison algorithms or by manual alignment and visual inspection.","For sequence comparison, typically one sequence acts as a reference sequence, to which test sequences are compared. When using a sequence comparison algorithm, test and reference sequences are entered into a computer, subsequence coordinates are designated, if necessary and sequence algorithm program parameters are designated. Default program parameters can be used, or alternative parameters can be designated. The sequence comparison algorithm then calculates the percent sequence identities for the test sequences relative to the reference sequence, based on the program parameters.","A \u201ccomparison window\u201d, as used herein, includes reference to a segment of any one of the number of contiguous positions selected from the group consisting of from 20 to 600, usually about 50 to about 200, more usually about 100 to about 150 in which a sequence may be compared to a reference sequence of the same number of contiguous positions after the two sequences are optimally aligned. Methods of alignment of sequence for comparison are well-known in the art. Optimal alignment of sequences for comparison can be conducted, e.g., by the local homology algorithm of Smith & Waterman, Adv. Appl. Math. 2:482, 1981, by the homology alignment algorithm of Needleman & Wunsch, J. Mol. Biol 48:443, 1970, by the search for similarity method of person & Lipman, Proc. Nat'l. Acad. Sci. USA 85:2444, 1988, by computerized implementations of these algorithms (GAP, BESTFIT, FASTA and TFASTA in the Wisconsin Genetics Software Package, Genetics Computer Group, 575 Science Dr., Madison, Wis.), or by manual alignment and visual inspection. Other algorithms for determining homology or identity include, for example, in addition to a BLAST program (Basic Local Alignment Search Tool at the National Center for Biological Information), ALIGN, AMAS (Analysis of Multiply Aligned Sequences), AMPS (Protein Multiple Sequence Alignment), ASSET (Aligned Segment Statistical Evaluation Tool), BANDS, BESTSCOR, BIOSCAN (Biological Sequence Comparative Analysis Node), BLIMPS (BLocks IMProved Searcher), FASTA, Intervals & Points, BMB, CLUSTAL V, CLUSTAL W, CONSENSUS, LCONSENSUS, WCONSENSUS, Smith-Waterman algorithm, DARWIN, Las Vegas algorithm, FNAT (Forced Nucleotide Alignment Tool), Framealign, Framesearch, DYNAMIC, FILTER, FSAP (Fristensky Sequence Analysis Package), GAP (Global Alignment Program), GENAL, GIBBS, GenQuest, ISSC (Sensitive Sequence Comparison), LALIGN (Local Sequence Alignment), LCP (Local Content Program), MACAW (Multiple Alignment Construction & Analysis Workbench), MAP (Multiple Alignment Program), MBLKP, MBLKN, PIMA (Pattern-Induced Multi-sequence Alignment), SAGA (Sequence Alignment by Genetic Algorithm) and WHAT-IF. Such alignment programs can also be used to screen genome databases to identify polynucleotide sequences having substantially identical sequences. A number of genome databases are available, for example, a substantial portion of the human genome is available as part of the Human Genome Sequencing Project (Gibbs, 1995). At least twenty-one other genomes have already been sequenced, including, for example, (Fraser et al., 1995), (Bult et al., 1996), (Fleischmann et al., 1995), (Blattner et al., 1997) and yeast () (Mewes et al., 1997) and (Adams et al., 2000). Significant progress has also been made in sequencing the genomes of model organism, such as mouse, and sp. Several databases containing genomic information annotated with some functional information are maintained by different organizations and may be accessible via internet.","One example of a useful algorithm is BLAST and BLAST 2.0 algorithms, which are described in Altschul et al., Nuc. Acids Res. 25:3389-3402, 1977 and Altschul et al., J. Mol. Biol. 215:403-410, 1990, respectively. Software for performing BLAST analyses is publicly available through the National Center for Biotechnology Information. This algorithm involves first identifying high scoring sequence pairs (HSPs) by identifying short words of length W in the query sequence, which either match or satisfy some positive-valued threshold score T when aligned with a word of the same length in a database sequence. T is referred to as the neighborhood word score threshold (Altschul et al., supra). These initial neighborhood word hits act as seeds for initiating searches to find longer HSPs containing them. The word hits are extended in both directions along each sequence for as far as the cumulative alignment score can be increased. Cumulative scores are calculated using, for nucleotide sequences, the parameters M (reward score for a pair of matching residues; always >0). For amino acid sequences, a scoring matrix is used to calculate the cumulative score. Extension of the word hits in each direction are halted when: the cumulative alignment score falls off by the quantity X from its maximum achieved value; the cumulative score goes to zero or below, due to the accumulation of one or more negative-scoring residue alignments; or the end of either sequence is reached. The BLAST algorithm parameters W, T and X determine the sensitivity and speed of the alignment. The BLASTN program (for nucleotide sequences) uses as defaults a wordlength (W) of 11, an expectation (E) of 10, M=5, N=\u22124 and a comparison of both strands. For amino acid sequences, the BLASTP program uses as defaults a wordlength of 3 and expectations (E) of 10 and the BLOSUM62 scoring matrix (see Henikoff & Henikoff, Proc. Natl. Acad. Sci. USA 89:10915, 1989) alignments (B) of 50, expectation (E) of 10, M=5, N=\u22124 and a comparison of both strands.","The BLAST algorithm also performs a statistical analysis of the similarity between two sequences (see, e.g., Karlin & Altschul, Proc. Natl. Acad. Sci. USA 90:5873, 1993). One measure of similarity provided by BLAST algorithm is the smallest sum probability (P(N)), which provides an indication of the probability by which a match between two nucleotide or amino acid sequences would occur by chance. For example, a nucleic acid is considered similar to a references sequence if the smallest sum probability in a comparison of the test nucleic acid to the reference nucleic acid is less than about 0.2, more in one aspect less than about 0.01 and most in one aspect less than about 0.001.","In one aspect, protein and nucleic acid sequence homologies are evaluated using the Basic Local Alignment Search Tool (\u201cBLAST\u201d) In particular, five specific BLAST programs are used to perform the following task:\n\n","The BLAST programs identify homologous sequences by identifying similar segments, which are referred to herein as \u201chigh-scoring segment pairs,\u201d between a query amino or nucleic acid sequence and a test sequence which is in one aspect obtained from a protein or nucleic acid sequence database. High-scoring segment pairs are in one aspect identified (i.e., aligned) by means of a scoring matrix, many of which are known in the art. In one aspect, the scoring matrix used is the BLOSUM62 matrix (Gonnet (1992) Science 256:1443-1445; Henikoff and Henikoff (1993) Proteins 17:49-61). Less in one aspect, the PAM or PAM250 matrices may also be used (see, e.g., Schwartz and Dayhoff, eds., 1978, Washington: National Biomedical Research Foundation). BLAST programs are accessible through the U.S. National Library of Medicine.","The parameters used with the above algorithms may be adapted depending on the sequence length and degree of homology studied. In some aspects, the parameters may be the default parameters used by the algorithms in the absence of instructions from the user.","Computer Systems and Computer Program Products","To determine and identify sequence identities, structural homologies, motifs and the like in silico, a nucleic acid or polypeptide sequence of the invention can be stored, recorded, and manipulated on any medium which can be read and accessed by a computer.","Accordingly, the invention provides computers, computer systems, computer readable mediums, computer programs products and the like recorded or stored thereon the nucleic acid and polypeptide sequences of the invention. As used herein, the words \u201crecorded\u201d and \u201cstored\u201d refer to a process for storing information on a computer medium. A skilled artisan can readily adopt any known methods for recording information on a computer readable medium to generate manufactures comprising one or more of the nucleic acid and\/or polypeptide sequences of the invention.","The polypeptides of the invention include the polypeptide sequences of the invention, e.g., the exemplary sequences of the invention, and sequences substantially identical thereto, and fragments of any of the preceding sequences. Substantially identical, or homologous, polypeptide sequences refer to a polypeptide sequence having at least 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more, or complete (100%) sequence identity (homology) to an exemplary sequence of the invention.","Homology (sequence identity) may be determined using any of the computer programs and parameters described herein. A nucleic acid or polypeptide sequence of the invention can be stored, recorded and manipulated on any medium which can be read and accessed by a computer. As used herein, the words \u201crecorded\u201d and \u201cstored\u201d refer to a process for storing information on a computer medium. A skilled artisan can readily adopt any of the presently known methods for recording information on a computer readable medium to generate manufactures comprising one or more of the nucleic acid sequences of the invention, one or more of the polypeptide sequences of the invention. Another aspect of the invention is a computer readable medium having recorded thereon at least 2, 5, 10, 15, or 20 or more nucleic acid or polypeptide sequences of the invention.","Another aspect of the invention is a computer readable medium having recorded thereon one or more of the nucleic acid sequences of the invention. Another aspect of the invention is a computer readable medium having recorded thereon one or more of the polypeptide sequences of the invention. Another aspect of the invention is a computer readable medium having recorded thereon at least 2, 5, 10, 15, or 20 or more of the nucleic acid or polypeptide sequences as set forth above.","Computer readable media include magnetically readable media, optically readable media, electronically readable media and magnetic\/optical media. For example, the computer readable media may be a hard disk, a floppy disk, a magnetic tape, CD-ROM, Digital Versatile Disk (DVD), Random Access Memory (RAM), or Read Only Memory (ROM) as well as other types of other media known to those skilled in the art.","Aspects of the invention include systems (e.g., internet based systems), particularly computer systems which store and manipulate the sequence information described herein. One example of a computer system  is illustrated in block diagram form in . As used herein, \u201ca computer system\u201d refers to the hardware components, software components and data storage components used to analyze a nucleotide sequence of a nucleic acid sequence of the invention, or a polypeptide sequence of the invention. The computer system  typically includes a processor for processing, accessing and manipulating the sequence data. The processor  can be any well-known type of central processing unit, such as, for example, the Pentium III from Intel Corporation, or similar processor from Sun, Motorola, Compaq, AMD or International Business Machines.","Typically the computer system  is a general purpose system that comprises the processor  and one or more internal data storage components  for storing data and one or more data retrieving devices for retrieving the data stored on the data storage components. A skilled artisan can readily appreciate that any one of the currently available computer systems are suitable.","In one particular aspect, the computer system  includes a processor  connected to a bus which is connected to a main memory  (in one aspect implemented as RAM) and one or more internal data storage devices , such as a hard drive and\/or other computer readable media having data recorded thereon. In some aspects, the computer system  further includes one or more data retrieving device  for reading the data stored on the internal data storage devices .","The data retrieving device  may represent, for example, a floppy disk drive, a compact disk drive, a magnetic tape drive, or a modem capable of connection to a remote data storage system (e.g., via internet) etc. In some aspects, the internal data storage device  is a removable computer readable medium such as a floppy disk, a compact disk, a magnetic tape, etc. containing control logic and\/or data recorded thereon. The computer system  may advantageously include or be programmed by appropriate software for reading the control logic and\/or the data from the data storage component once inserted in the data retrieving device.","The computer system  includes a display.  which is used to display output to a computer user. It should also be noted that the computer system  can be linked to other computer systems A-C in a network or wide area network to provide centralized access to the computer system .","Software for accessing and processing the nucleotide sequences of a nucleic acid sequence of the invention, or a polypeptide sequence of the invention, (such as search tools, compare tools and modeling tools etc.) may reside in main memory  during execution.","In some aspects, the computer system  may further comprise a sequence comparison algorithm for comparing a nucleic acid sequence of the invention, or a polypeptide sequence of the invention, stored on a computer readable medium to a reference nucleotide or polypeptide sequence(s) stored on a computer readable medium. A \u201csequence comparison algorithm\u201d refers to one or more programs which are implemented (locally or remotely) on the computer system  to compare a nucleotide sequence with other nucleotide sequences and\/or compounds stored within a data storage means. For example, the sequence comparison algorithm may compare the nucleotide sequences of a nucleic acid sequence of the invention, or a polypeptide sequence of the invention, stored on a computer readable medium to reference sequences stored on a computer readable medium to identify homologies or structural motifs.",{"@attributes":{"id":"p-0367","num":"0371"},"figref":"FIG. 2","b":["200","100"]},"The process  begins at a start state  and then moves to a state  wherein the new sequence to be compared is stored to a memory in a computer system . As discussed above, the memory could be any type of memory, including RAM or an internal storage device.","The process  then moves to a state  wherein a database of sequences is opened for analysis and comparison. The process  then moves to a state  wherein the first sequence stored in the database is read into a memory on the computer. A comparison is then performed at a state  to determine if the first sequence is the same as the second sequence. It is important to note that this step is not limited to performing an exact comparison between the new sequence and the first sequence in the database. Well-known methods are known to those of skill in the art for comparing two nucleotide or protein sequences, even if they are not identical. For example, gaps can be introduced into one sequence in order to raise the homology level between the two tested sequences. The parameters that control whether gaps or other features are introduced into a sequence during comparison are normally entered by the user of the computer system.","Once a comparison of the two sequences has been performed at the state , a determination is made at a decision state  whether the two sequences are the same. Of course, the term \u201csame\u201d is not limited to sequences that are absolutely identical. Sequences that are within the homology parameters entered by the user will be marked as \u201csame\u201d in the process .","If a determination is made that the two sequences are the same, the process  moves to a state  wherein the name of the sequence from the database is displayed to the user. This state notifies the user that the sequence with the displayed name fulfills the homology constraints that were entered. Once the name of the stored sequence is displayed to the user, the process  moves to a decision state  wherein a determination is made whether more sequences exist in the database. If no more sequences exist in the database, then the process  terminates at an end state . However, if more sequences do exist in the database, then the process  moves to a state  wherein a pointer is moved to the next sequence in the database so that it can be compared to the new sequence. In this manner, the new sequence is aligned and compared with every sequence in the database.","It should be noted that if a determination had been made at the decision state  that the sequences were not homologous, then the process  would move immediately to the decision state  in order to determine if any other sequences were available in the database for comparison.","Accordingly, one aspect of the invention is a computer system comprising a processor, a data storage device having stored thereon a nucleic acid sequence of the invention, or a polypeptide sequence of the invention, a data storage device having retrievably stored thereon reference nucleotide sequences or polypeptide sequences to be compared to a nucleic acid sequence of the invention, or a polypeptide sequence of the invention and a sequence comparer for conducting the comparison. The sequence comparer may indicate a homology level between the sequences compared or identify structural motifs in the above described nucleic acid code a nucleic acid sequence of the invention, or a polypeptide sequence of the invention, or it may identify structural motifs in sequences which are compared to these nucleic acid codes and polypeptide codes. In some aspects, the data storage device may have stored thereon the sequences of at least 2, 5, 10, 15, 20, 25, 30 or 40 or more of the nucleic acid sequences of the invention, or the polypeptide sequences of the invention.","Another aspect of the invention is a method for determining the level of homology between a nucleic acid sequence of the invention, or a polypeptide sequence of the invention and a reference nucleotide sequence. The method including reading the nucleic acid code or the polypeptide code and the reference nucleotide or polypeptide sequence through the use of a computer program which determines homology levels and determining homology between the nucleic acid code or polypeptide code and the reference nucleotide or polypeptide sequence with the computer program. The computer program may be any of a number of computer programs for determining homology levels, including those specifically enumerated herein, (e.g., BLAST2N with the default parameters or with any modified parameters). The method may be implemented using the computer systems described above. The method may also be performed by reading at least 2, 5, 10, 15, 20, 25, 30 or 40 or more of the above described nucleic acid sequences of the invention, or the polypeptide sequences of the invention through use of the computer program and determining homology between the nucleic acid codes or polypeptide codes and reference nucleotide sequences or polypeptide sequences.",{"@attributes":{"id":"p-0375","num":"0379"},"figref":"FIG. 3","b":["250","250","252","254","256","250","260","262"]},"A determination is then made at a decision state  whether the two characters are the same. If they are the same, then the process  moves to a state  wherein the next characters in the first and second sequences are read. A determination is then made whether the next characters are the same. If they are, then the process  continues this loop until two characters are not the same. If a determination is made that the next two characters are not the same, the process  moves to a decision state  to determine whether there are any more characters either sequence to read.","If there are not any more characters to read, then the process  moves to a state  wherein the level of homology between the first and second sequences is displayed to the user. The level of homology is determined by calculating the proportion of characters between the sequences that were the same out of the total number of sequences in the first sequence. Thus, if every character in a first 100 nucleotide sequence aligned with a every character in a second sequence, the homology level would be 100%.","Alternatively, the computer program may be a computer program which compares the nucleotide sequences of a nucleic acid sequence as set forth in the invention, to one or more reference nucleotide sequences in order to determine whether the nucleic acid code of the invention, differs from a reference nucleic acid sequence at one or more positions. Optionally such a program records the length and identity of inserted, deleted or substituted nucleotides with respect to the sequence of either the reference polynucleotide or a nucleic acid sequence of the invention. In one aspect, the computer program may be a program which determines whether a nucleic acid sequence of the invention, contains a single nucleotide polymorphism (SNP) with respect to a reference nucleotide sequence.","Accordingly, another aspect of the invention is a method for determining whether a nucleic acid sequence of the invention, differs at one or more nucleotides from a reference nucleotide sequence comprising the steps of reading the nucleic acid code and the reference nucleotide sequence through use of a computer program which identifies differences between nucleic acid sequences and identifying differences between the nucleic acid code and the reference nucleotide sequence with the computer program. In some aspects, the computer program is a program which identifies single nucleotide polymorphisms. The method may be implemented by the computer systems described above and the method illustrated in . The method may also be performed by reading at least 2, 5, 10, 15, 20, 25, 30, or 40 or more of the nucleic acid sequences of the invention and the reference nucleotide sequences through the use of the computer program and identifying differences between the nucleic acid codes and the reference nucleotide sequences with the computer program.","In other aspects the computer based system may further comprise an identifier for identifying features within a nucleic acid sequence of the invention or a polypeptide sequence of the invention.","An \u201cidentifier\u201d refers to one or more programs which identifies certain features within a nucleic acid sequence of the invention, or a polypeptide sequence of the invention. In one aspect, the identifier may comprise a program which identifies an open reading frame in a nucleic acid sequence of the invention.",{"@attributes":{"id":"p-0382","num":"0386"},"figref":"FIG. 4","b":["300","300","302","304","115","100","300","306"]},"Once the database of features is opened at the state , the process  moves to a state  wherein the first feature is read from the database. A comparison of the attribute of the first feature with the first sequence is then made at a state . A determination is then made at a decision state  whether the attribute of the feature was found in the first sequence. If the attribute was found, then the process  moves to a state  wherein the name of the found feature is displayed to the user.","The process  then moves to a decision state  wherein a determination is made whether move features exist in the database. If no more features do exist, then the process  terminates at an end state . However, if more features do exist in the database, then the process  reads the next sequence feature at a state  and loops back to the state  wherein the attribute of the next feature is compared against the first sequence. It should be noted, that if the feature attribute is not found in the first sequence at the decision state , the process  moves directly to the decision state  in order to determine if any more features exist in the database.","Accordingly, another aspect of the invention is a method of identifying a feature within a nucleic acid sequence of the invention, or a polypeptide sequence of the invention, comprising reading the nucleic acid code(s) or polypeptide code(s) through the use of a computer program which identifies features therein and identifying features within the nucleic acid code(s) with the computer program. In one aspect, computer program comprises a computer program which identifies open reading frames. The method may be performed by reading a single sequence or at least 2, 5, 10, 15, 20, 25, 30, or 40 of the nucleic acid sequences of the invention, or the polypeptide sequences of the invention, through the use of the computer program and identifying features within the nucleic acid codes or polypeptide codes with the computer program.","A nucleic acid sequence of the invention, or a polypeptide sequence of the invention, may be stored and manipulated in a variety of data processor programs in a variety of formats. For example, a nucleic acid sequence of the invention, or a polypeptide sequence of the invention, may be stored as text in a word processing file, such as Microsoft WORD\u2122 or WORDPERFECT\u2122 or as an ASCII file in a variety of database programs familiar to those of skill in the art, such as DB2\u2122, SYBASE\u2122, or ORACLE\u2122. In addition, many computer programs and databases may be used as sequence comparison algorithms, identifiers, or sources of reference nucleotide sequences or polypeptide sequences to be compared to a nucleic acid sequence of the invention, or a polypeptide sequence of the invention. The following list is intended not to limit the invention but to provide guidance to programs and databases which are useful with the nucleic acid sequences of the invention, or the polypeptide sequences of the invention.","The programs and databases which may be used include, but are not limited to: MacPattern (EMBL), DiscoveryBase (Molecular Applications Group), GeneMine (Molecular Applications Group), Look (Molecular Applications Group), MacLook (Molecular Applications Group), BLAST and BLAST2 (NCBI), BLASTN and BLASTX (Altschul et al, J. Mol. Biol. 215: 403, 1990), FASTA (Pearson and Lipman, Proc. Natl. Acad. Sci. USA, 85: 2444, 1988), FASTDB (Brutlag et al. Comp. App. Biosci. 6:237-245, 1990), Catalyst (Molecular Simulations Inc.), Catalyst\/SHAPE (Molecular Simulations Inc.), Cerius. DBAccess (Molecular Simulations Inc.), HypoGen (Molecular Simulations Inc.), Insight II, (Molecular Simulations Inc.), Discover (Molecular Simulations Inc.), CHARMm (Molecular Simulations Inc.), Felix (Molecular Simulations Inc.), DelPhi, (Molecular Simulations Inc.), QuanteMM, (Molecular Simulations Inc.), Homology (Molecular Simulations Inc.), Modeler (Molecular Simulations Inc.), ISIS (Molecular Simulations Inc.), Quanta\/Protein Design (Molecular Simulations Inc.), WebLab (Molecular Simulations Inc.), WebLab Diversity Explorer (Molecular Simulations Inc.), Gene Explorer (Molecular Simulations Inc.), SeqFold (Molecular Simulations Inc.), the MDL Available Chemicals Directory database, the MDL Drug Data Report data base, the Comprehensive Medicinal Chemistry database, Derwents's World Drug Index database, the BioByteMasterFile database, the Genbank database and the Genseqn database. Many other programs and data bases would be apparent to one of skill in the art given the present disclosure.","Motifs which may be detected using the above programs include sequences encoding leucine zippers, helix-turn-helix motifs, glycosylation sites, ubiquitination sites, alpha helices and beta sheets, signal sequences encoding signal peptides which direct the secretion of the encoded proteins, sequences implicated in transcription regulation such as homeoboxes, acidic stretches, enzymatic active sites, substrate binding sites and enzymatic cleavage sites.","Hybridization of Nucleic Acids","The invention provides isolated or recombinant nucleic acids that hybridize under stringent conditions to an exemplary sequence of the invention (e.g., SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9). The stringent conditions can be highly stringent conditions, medium stringent conditions and\/or low stringent conditions, including the high and reduced stringency conditions described herein. In one aspect, it is the stringency of the wash conditions that set forth the conditions which determine whether a nucleic acid is within the scope of the invention, as discussed below.","\u201cHybridization\u201d refers to the process by which a nucleic acid strand joins with a complementary strand through base pairing. Hybridization reactions can be sensitive and selective so that a particular sequence of interest can be identified even in samples in which it is present at low concentrations. Suitably stringent conditions can be defined by, for example, the concentrations of salt or formamide in the prehybridization and hybridization solutions, or by the hybridization temperature and are well known in the art. In particular, stringency can be increased by reducing the concentration of salt, increasing the concentration of formamide, or raising the hybridization temperature. In alternative aspects, nucleic acids of the invention are defined by their ability to hybridize under various stringency conditions (e.g., high, medium, and low), as set forth herein.","For example, hybridization under high stringency conditions could occur in about 50% formamide at about 37\u00b0 C. to 42\u00b0 C. Hybridization could occur under reduced stringency conditions in about 35% to 25% formamide at about 30\u00b0 C. to 35\u00b0 C. In one aspect, hybridization occurs under high stringency conditions, e.g., at 42\u00b0 C. in 50% formamide, 5\u00d7SSPE, 0.3% SDS and 200 n\/ml sheared and denatured salmon sperm DNA. Hybridization could occur under these reduced stringency conditions, but in 35% formamide at a reduced temperature of 35\u00b0 C. The temperature range corresponding to a particular level of stringency can be further narrowed by calculating the purine to pyrimidine ratio of the nucleic acid of interest and adjusting the temperature accordingly. Variations on the above ranges and conditions are well known in the art.","In alternative aspects, nucleic acids of the invention as defined by their ability to hybridize under stringent conditions can be between about five residues and the full length of nucleic acid of the invention; e.g., they can be at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 55, 60, 65, 70, 75, 80, 90, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000, or more, residues in length. Nucleic acids shorter than full length are also included. These nucleic acids can be useful as, e.g., hybridization probes, labeling probes, PCR oligonucleotide probes, iRNA (single or double stranded), antisense or sequences encoding antibody binding peptides (epitopes), motifs, active sites and the like.","In one aspect, nucleic acids of the invention are defined by their ability to hybridize under high stringency comprises conditions of about 50% formamide at about 37\u00b0 C. to 42\u00b0 C. In one aspect, nucleic acids of the invention are defined by their ability to hybridize under reduced stringency comprising conditions in about 35% to 25% formamide at about 30\u00b0 C. to 35\u00b0 C.","Alternatively, nucleic acids of the invention are defined by their ability to hybridize under high stringency comprising conditions at 42\u00b0 C. in 50% formamide, 5\u00d7SSPE, 0.3% SDS, and a repetitive sequence blocking nucleic acid, such as cot-1 or salmon sperm DNA (e.g., 200 n\/ml sheared and denatured salmon sperm DNA). In one aspect, nucleic acids of the invention are defined by their ability to hybridize under reduced stringency conditions comprising 35% formamide at a reduced temperature of 35\u00b0 C.","In nucleic acid hybridization reactions, the conditions used to achieve a particular level of stringency will vary, depending on the nature of the nucleic acids being hybridized. For example, the length, degree of complementarity, nucleotide sequence composition (e.g., GC v. AT content) and nucleic acid type (e.g., RNA v. DNA) of the hybridizing regions of the nucleic acids can be considered in selecting hybridization conditions. An additional consideration is whether one of the nucleic acids is immobilized, for example, on a filter.","Hybridization may be carried out under conditions of low stringency, moderate stringency or high stringency. As an example of nucleic acid hybridization, a polymer membrane containing immobilized denatured nucleic acids is first prehybridized for 30 minutes at 45\u00b0 C. in a solution consisting of 0.9 M NaCl, 50 mM NaHPO, pH 7.0, 5.0 mM NaEDTA, 0.5% SDS, 10\u00d7Denhardt's and 0.5 mg\/ml polyriboadenylic acid. Approximately 2\u00d710cpm (specific activity 4\u22129\u00d710cpm\/ug) of P end-labeled oligonucleotide probe are then added to the solution. After 12-16 hours of incubation, the membrane is washed for 30 minutes at room temperature in 1\u00d7SET (150 mM NaCl, 20 mM Tris hydrochloride, pH 7.8, 1 mM NaEDTA) containing 0.5% SDS, followed by a 30 minute wash in fresh 1\u00d7SET at T\u221210\u00b0 C. for the oligonucleotide probe. The membrane is then exposed to auto-radiographic film for detection of hybridization signals.","All of the foregoing hybridizations would be considered to be under conditions of high stringency.","Following hybridization, a filter can be washed to remove any non-specifically bound detectable probe. The stringency used to wash the filters can also be varied depending on the nature of the nucleic acids being hybridized, the length of the nucleic acids being hybridized, the degree of complementarity, the nucleotide sequence composition (e.g., GC v. AT content) and the nucleic acid type (e.g., RNA v. DNA). Examples of progressively higher stringency condition washes are as follows: 2\u00d7SSC, 0.1% SDS at room temperature for 15 minutes (low stringency); 0.1\u00d7SSC, 0.5% SDS at room temperature for 30 minutes to 1 hour (moderate stringency); 0.1\u00d7SSC, 0.5% SDS for 15 to 30 minutes at between the hybridization temperature and 68\u00b0 C. (high stringency); and 0.15M NaCl for 15 minutes at 72\u00b0 C. (very high stringency). A final low stringency wash can be conducted in 0.1\u00d7SSC at room temperature. The examples above are merely illustrative of one set of conditions that can be used to wash filters. One of skill in the art would know that there are numerous recipes for different stringency washes. Some other examples are given below.","In one aspect, hybridization conditions comprise a wash step comprising a wash for 30 minutes at room temperature in a solution comprising 1\u00d7150 mM NaCl, 20 mM Tris hydrochloride, pH 7.8, 1 mM NaEDTA, 0.5% SDS, followed by a 30 minute wash in fresh solution.","Nucleic acids which have hybridized to the probe are identified by autoradiography or other conventional techniques.","The above procedure may be modified to identify nucleic acids having decreasing levels of homology to the probe sequence. For example, to obtain nucleic acids of decreasing homology to the detectable probe, less stringent conditions may be used. For example, the hybridization temperature may be decreased in increments of 5\u00b0 C. from 68\u00b0 C. to 42\u00b0 C. in a hybridization buffer having a Na+ concentration of approximately 1M. Following hybridization, the filter may be washed with 2\u00d7SSC, 0.5% SDS at the temperature of hybridization. These conditions are considered to be \u201cmoderate\u201d conditions above 50\u00b0 C. and \u201clow\u201d conditions below 50\u00b0 C. A specific example of \u201cmoderate\u201d hybridization conditions is when the above hybridization is conducted at 55\u00b0 C. A specific example of \u201clow stringency\u201d hybridization conditions is when the above hybridization is conducted at 45\u00b0 C.","Alternatively, the hybridization may be carried out in buffers, such as 6\u00d7SSC, containing formamide at a temperature of 42\u00b0 C. In this case, the concentration of formamide in the hybridization buffer may be reduced in 5% increments from 50% to 0% to identify clones having decreasing levels of homology to the probe. Following hybridization, the filter may be washed with 6\u00d7SSC, 0.5% SDS at 50\u00b0 C. These conditions are considered to be \u201cmoderate\u201d conditions above 25% formamide and \u201clow\u201d conditions below 25% formamide. A specific example of \u201cmoderate\u201d hybridization conditions is when the above hybridization is conducted at 30% formamide. A specific example of \u201clow stringency\u201d hybridization conditions is when the above hybridization is conducted at 10% formamide.","However, the selection of a hybridization format is not critical\u2014it is the stringency of the wash conditions that set forth the conditions which determine whether a nucleic acid is within the scope of the invention. Wash conditions used to identify nucleic acids within the scope of the invention include, e.g.: a salt concentration of about 0.02 molar at pH 7 and a temperature of at least about 50\u00b0 C. or about 55\u00b0 C. to about 60\u00b0 C.; or, a salt concentration of about 0.15 M NaCl at 72\u00b0 C. for about 15 minutes; or, a salt concentration of about 0.2\u00d7SSC at a temperature of at least about 50\u00b0 C. or about 55\u00b0 C. to about 60\u00b0 C. for about 15 to about 20 minutes; or, the hybridization complex is washed twice with a solution with a salt concentration of about 2\u00d7SSC containing 0.1% SDS at room temperature for 15 minutes and then washed twice by 0.1\u00d7SSC containing 0.1% SDS at 68\u00b0 C. for 15 minutes; or, equivalent conditions. See Sambrook, Tijssen and Ausubel for a description of SSC buffer and equivalent conditions.","These methods may be used to isolate nucleic acids of the invention. For example, the preceding methods may be used to isolate nucleic acids having a sequence with at least about 97%, at least 95%, at least 90%, at least 85%, at least 80%, at least 75%, at least 70%, at least 65%, at least 60%, at least 55%, or at least 50% sequence identity (homology) to a nucleic acid sequence selected from the group consisting of one of the sequences of the invention, or fragments comprising at least about 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, 150, 200, 300, 400, or 500 consecutive bases thereof and the sequences complementary thereto. Sequence identity (homology) may be measured using the alignment algorithm. For example, the homologous polynucleotides may have a coding sequence which is a naturally occurring allelic variant of one of the coding sequences described herein. Such allelic variants may have a substitution, deletion or addition of one or more nucleotides when compared to the nucleic acids of the invention. Additionally, the above procedures may be used to isolate nucleic acids which encode polypeptides having at least about 99%, 95%, at least 90%, at least 85%, at least 80%, at least 75%, at least 70%, at least 65%, at least 60%, at least 55%, or at least 50% sequence identity (homology) to a polypeptide of the invention, or fragments comprising at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 consecutive amino acids thereof as determined using a sequence alignment algorithm (e.g., such as the FASTA version 3.0t78 algorithm with the default parameters).","Oligonucleotides Probes and Methods for Using them","The invention also provides nucleic acid probes that can be used, e.g., for identifying nucleic acids encoding a polypeptide with an enzyme, structural or binding activity or fragments thereof or for identifying polypeptide, enzyme, protein, e.g. structural or binding protein, genes. In one aspect, the probe comprises at least 10 consecutive bases of a nucleic acid of the invention. Alternatively, a probe of the invention can be at least about 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 30, 35, 40, 45, 50, 60, 70, 80, 90, 100, 110, 120, 130, 150 or about 10 to 50, about 20 to 60 about 30 to 70, consecutive bases of a sequence as set forth in a nucleic acid of the invention. The probes identify a nucleic acid by binding and\/or hybridization. The probes can be used in arrays of the invention, see discussion below, including, e.g., capillary arrays. The probes of the invention can also be used to isolate other nucleic acids or polypeptides.","The isolated nucleic acids of the invention, the sequences complementary thereto, or a fragment comprising at least 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, 150, 200, 300, 400, or 500 consecutive bases of one of the sequences of the invention, or the sequences complementary thereto may also be used as probes to determine whether a biological sample, such as a soil sample, contains an organism having a nucleic acid sequence of the invention or an organism from which the nucleic acid was obtained. In such procedures, a biological sample potentially harboring the organism from which the nucleic acid was isolated is obtained and nucleic acids are obtained from the sample. The nucleic acids are contacted with the probe under conditions which permit the probe to specifically hybridize to any complementary sequences from which are present therein.","Where necessary, conditions which permit the probe to specifically hybridize to complementary sequences may be determined by placing the probe in contact with complementary sequences from samples known to contain the complementary sequence as well as control sequences which do not contain the complementary sequence. Hybridization conditions, such as the salt concentration of the hybridization buffer, the formamide concentration of the hybridization buffer, or the hybridization temperature, may be varied to identify conditions which allow the probe to hybridize specifically to complementary nucleic acids.","If the sample contains the organism from which the nucleic acid was isolated, specific hybridization of the probe is then detected. Hybridization may be detected by labeling the probe with a detectable agent such as a radioactive isotope, a fluorescent dye or an enzyme capable of catalyzing the formation of a detectable product.","Many methods for using the labeled probes to detect the presence of complementary nucleic acids in a sample are familiar to those skilled in the art. These include Southern Blots, Northern Blots, colony hybridization procedures and dot blots. Protocols for each of these procedures are provided in Ausubel et al. Current Protocols in Molecular Biology, John Wiley 503 Sons, Inc. (1997) and Sambrook et al., Molecular Cloning: A Laboratory Manual 2nd Ed., Cold Spring Harbor Laboratory Press (1989.","Alternatively, more than one probe (at least one of which is capable of specifically hybridizing to any complementary sequences which are present in the nucleic acid sample), may be used in an amplification reaction to determine whether the sample contains an organism containing a nucleic acid sequence of the invention (e.g., an organism from which the nucleic acid was isolated). Typically, the probes comprise oligonucleotides. In one aspect, the amplification reaction may comprise a PCR reaction. PCR protocols are described in Ausubel and Sambrook, supra. Alternatively, the amplification may comprise a ligase chain reaction, 3SR, or strand displacement reaction. (See Barany, F., \u201cThe Ligase Chain Reaction in a PCR World\u201d, 1:5-16, 1991; E. Fahy et al., \u201cSelf-sustained Sequence Replication (3SR): An Isothermal Transcription-based Amplification System Alternative to PCR\u201d, 1:25-33, 1991; and Walker G. T. et al., \u201cStrand Displacement Amplification\u2014an Isothermal in vitro DNA Amplification Technique\u201d, 20:1691-1696, 1992). In such procedures, the nucleic acids in the sample are contacted with the probes, the amplification reaction is performed and any resulting amplification product is detected. The amplification product may be detected by performing gel electrophoresis on the reaction products and staining the gel with an intercalator such as ethidium bromide. Alternatively, one or more of the probes may be labeled with a radioactive isotope and the presence of a radioactive amplification product may be detected by autoradiography after gel electrophoresis.","Probes derived from sequences near the ends of the sequences of the invention, may also be used in chromosome walking procedures to identify clones containing genomic sequences located adjacent to the sequences of the invention. Such methods allow the isolation of genes which encode additional proteins from the host organism.","The isolated nucleic acids of the invention, the sequences complementary thereto, or a fragment comprising at least 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, 150, 200, 300, 400, or 500 consecutive bases of one of the sequences of the invention, or the sequences complementary thereto may be used as probes to identify and isolate related nucleic acids. In some aspects, the related nucleic acids may be cDNAs or genomic DNAs from organisms other than the one from which the nucleic acid was isolated. For example, the other organisms may be related organisms. In such procedures, a nucleic acid sample is contacted with the probe under conditions which permit the probe to specifically hybridize to related sequences. Hybridization of the probe to nucleic acids from the related organism is then detected using any of the methods described above.","By varying the stringency of the hybridization conditions used to identify nucleic acids, such as cDNAs or genomic DNAs, which hybridize to the detectable probe, nucleic acids having different levels of homology to the probe can be identified and isolated. Stringency may be varied by conducting the hybridization at varying temperatures below the melting temperatures of the probes. The melting temperature, T, is the temperature (under defined ionic strength and pH) at which 50% of the target sequence hybridizes to a perfectly complementary probe. Very stringent conditions are selected to be equal to or about 5\u00b0 C. lower than the Tfor a particular probe. The melting temperature of the probe may be calculated using the following formulas:","For probes between 14 and 70 nucleotides in length the melting temperature (T) is calculated using the formula: T=81.5+16.6(log [Na+])+0.41(fraction G+C)\u2212(600\/N) where N is the length of the probe.","If the hybridization is carried out in a solution containing formamide, the melting temperature may be calculated using the equation: T=81.5+16.6(log [Na+])+0.41(fraction G+C)\u2212(0.63% formamide)\u2212(600\/N) where N is the length of the probe.","Prehybridization may be carried out in 6\u00d7SSC, 5\u00d7Denhardt's reagent, 0.5% SDS, 100 \u03bcg denatured fragmented salmon sperm DNA or 6\u00d7SSC, 5\u00d7Denhardt's reagent, 0.5% SDS, 100 \u03bcg denatured fragmented salmon sperm DNA, 50% formamide. The formulas for SSC and Denhardt's solutions are listed in Sambrook et al., supra.","Hybridization is conducted by adding the detectable probe to the prehybridization solutions listed above. Where the probe comprises double stranded DNA, it is denatured before addition to the hybridization solution. The filter is contacted with the hybridization solution for a sufficient period of time to allow the probe to hybridize to cDNAs or genomic DNAs containing sequences complementary thereto or homologous thereto. For probes over 200 nucleotides in length, the hybridization may be carried out at 15-25\u00b0 C. below the T. For shorter probes, such as oligonucleotide probes, the hybridization may be conducted at 5-10\u00b0 C. below the T. In one aspect, for hybridizations in 6\u00d7SSC, the hybridization is conducted at approximately 68\u00b0 C. Usually, for hybridizations in 50% formamide containing solutions, the hybridization is conducted at approximately 42\u00b0 C.","Inhibiting Expression of Polypeptides, Enzymes, Proteins","The invention provides nucleic acids complementary to (e.g., antisense sequences to) the nucleic acids of the invention, e.g., nucleic acids comprising antisense, iRNA, ribozymes. Nucleic acids of the invention comprising antisense sequences can be capable of inhibiting the transport, splicing or transcription of polypeptide, enzyme, protein, e.g. structural or binding protein genes. The inhibition can be effected through the targeting of genomic DNA or messenger RNA. The transcription or function of targeted nucleic acid can be inhibited, for example, by hybridization and\/or cleavage. In one aspect, inhibitors of the invention include oligonucleotides which are able to either bind a polypeptide, enzyme, protein, e.g. structural or binding protein, gene or message, in either case preventing or inhibiting the production or function of a polypeptide, enzyme, protein, e.g. structural or binding protein. The association can be through sequence specific hybridization. Another useful class of inhibitors includes oligonucleotides which cause inactivation or cleavage of a polypeptide, enzyme, protein, e.g. structural or binding protein, message. The oligonucleotide can have enzyme activity which causes such cleavage, such as ribozymes. The oligonucleotide can be chemically modified or conjugated to an enzyme or composition capable of cleaving the complementary nucleic acid. A pool of many different such oligonucleotides can be screened for those with the desired activity. Thus, the invention provides various compositions for the inhibition of a polypeptide, enzyme, protein, e.g. structural or binding protein, expression on a nucleic acid and\/or protein level, e.g., antisense, iRNA and ribozymes comprising a polypeptide, enzyme, protein, e.g. structural or binding protein, sequences of the invention and the anti-polypeptide, anti-enzyme, anti-protein, e.g. anti-structural or anti-binding protein antibodies of the invention.","Inhibition of a polypeptide, enzyme, protein, e.g. structural or binding protein, expression can have a variety of industrial applications. For example, inhibition of a polypeptide, enzyme, protein, e.g. structural or binding protein, expression can slow or prevent spoilage. In one aspect, use of compositions of the invention that inhibit the expression and\/or activity of a polypeptide, enzyme, protein, e.g. structural or binding protein, e.g., antibodies, antisense oligonucleotides, ribozymes and RNAi, are used to slow or prevent spoilage. Thus, in one aspect, the invention provides methods and compositions comprising application onto a plant or plant product (e.g., a cereal, a grain, a fruit, seed, root, leaf, etc.) antibodies, antisense oligonucleotides, ribozymes and RNAi of the invention to slow or prevent spoilage. These compositions also can be expressed by the plant (e.g., a transgenic plant) or another organism (e.g., a bacterium or other microorganism transformed with a polypeptide, enzyme, protein, e.g. structural or binding protein, gene of the invention).","The compositions of the invention for the inhibition of a polypeptide, enzyme, protein, e.g. structural or binding protein, expression, e.g., antisense, iRNA (e.g., siRNA, miRNA), ribozymes, antibodies, can be used as pharmaceutical compositions, e.g., as anti-pathogen agents or in other therapies, e.g., as anti-microbials for, e.g., , or to neutralize a biological warfare agent, e.g., anthrax.","Antisense Oligonucleotides","The invention provides antisense oligonucleotides capable of binding a polypeptide, enzyme, protein, e.g. structural or binding protein, message which, in one aspect, can inhibit a polypeptide, enzyme, protein, e.g. structural or binding protein, activity by targeting mRNA. Strategies for designing antisense oligonucleotides are well described in the scientific and patent literature, and the skilled artisan can design such a polypeptide, enzyme, protein, e.g. structural or binding protein, oligonucleotides using the novel reagents of the invention. For example, gene walking\/RNA mapping protocols to screen for effective antisense oligonucleotides are well known in the art, see, e.g., Ho (2000) Methods Enzymol. 314:168-183, describing an RNA mapping assay, which is based on standard molecular techniques to provide an easy and reliable method for potent antisense sequence selection. See also Smith (2000) Eur. J. Pharm. Sci. 11:191-198.","Naturally occurring nucleic acids are used as antisense oligonucleotides. The antisense oligonucleotides can be of any length; for example, in alternative aspects, the antisense oligonucleotides are between about 5 to 100, about 10 to 80, about 15 to 60, about 18 to 40. The optimal length can be determined by routine screening. The antisense oligonucleotides can be present at any concentration. The optimal concentration can be determined by routine screening. A wide variety of synthetic, non-naturally occurring nucleotide and nucleic acid analogues are known which can address this potential problem. For example, peptide nucleic acids (PNAs) containing non-ionic backbones, such as N-(2-aminoethyl) glycine units can be used. Antisense oligonucleotides having phosphorothioate linkages can also be used, as described in WO 97\/03211; WO 96\/39154; Mata (1997) Toxicol Appl Pharmacol 144:189-197; Antisense Therapeutics, ed. Agrawal (Humana Press, Totowa, N.J., 1996). Antisense oligonucleotides having synthetic DNA backbone analogues provided by the invention can also include phosphoro-dithioate, methylphosphonate, phosphoramidate, alkyl phosphotriester, sulfamate, 3\u2032-thioacetal, methylene(methylimino), 3\u2032-N-carbamate, and morpholino carbamate nucleic acids, as described above.","Combinatorial chemistry methodology can be used to create vast numbers of oligonucleotides that can be rapidly screened for specific oligonucleotides that have appropriate binding affinities and specificities toward any target, such as the sense and antisense a polypeptide, enzyme, protein, e.g. structural or binding protein, sequences of the invention (see, e.g., Gold (1995) J. of Biol. Chem. 270:13581-13584).","Inhibitory Ribozymes","The invention provides ribozymes capable of binding a polypeptide, enzyme, protein, e.g. structural or binding protein, message. These ribozymes can inhibit a polypeptide, enzyme, protein, e.g. structural or binding protein, activity by, e.g., targeting mRNA. Strategies for designing ribozymes and selecting the polypeptide, enzyme, protein, e.g. structural or binding protein-specific antisense sequence for targeting are well described in the scientific and patent literature, and the skilled artisan can design such ribozymes using the novel reagents of the invention. Ribozymes act by binding to a target RNA through the target RNA binding portion of a ribozyme which is held in close proximity to an enzymatic portion of the RNA that cleaves the target RNA. Thus, the ribozyme recognizes and binds a target RNA through complementary base-pairing, and once bound to the correct site, acts enzymatically to cleave and inactivate the target RNA. Cleavage of a target RNA in such a manner will destroy its ability to direct synthesis of an encoded protein if the cleavage occurs in the coding sequence. After a ribozyme has bound and cleaved its RNA target, it can be released from that RNA to bind and cleave new targets repeatedly.","In some circumstances, the enzymatic nature of a ribozyme can be advantageous over other technologies, such as antisense technology (where a nucleic acid molecule simply binds to a nucleic acid target to block its transcription, translation or association with another molecule) as the effective concentration of ribozyme necessary to effect a therapeutic treatment can be lower than that of an antisense oligonucleotide. This potential advantage reflects the ability of the ribozyme to act enzymatically. Thus, a single ribozyme molecule is able to cleave many molecules of target RNA. In addition, a ribozyme is typically a highly specific inhibitor, with the specificity of inhibition depending not only on the base pairing mechanism of binding, but also on the mechanism by which the molecule inhibits the expression of the RNA to which it binds. That is, the inhibition is caused by cleavage of the RNA target and so specificity is defined as the ratio of the rate of cleavage of the targeted RNA over the rate of cleavage of non-targeted RNA. This cleavage mechanism is dependent upon factors additional to those involved in base pairing. Thus, the specificity of action of a ribozyme can be greater than that of antisense oligonucleotide binding the same RNA site.","The ribozyme of the invention, e.g., an enzymatic ribozyme RNA molecule, can be formed in a hammerhead motif, a hairpin motif, as a hepatitis delta virus motif, a group I intron motif and\/or an RNaseP-like RNA in association with an RNA guide sequence. Examples of hammerhead motifs are described by, e.g., Rossi (1992) Aids Research and Human Retroviruses 8:183; hairpin motifs by Hampel (1989) Biochemistry 28:4929, and Hampel (1990) Nuc. Acids Res. 18:299; the hepatitis delta virus motif by Perrotta (1992) Biochemistry 31:16; the RNaseP motif by Guerrier-Takada (1983) Cell 35:849; and the group I intron by Cech U.S. Pat. No. 4,987,071. The recitation of these specific motifs is not intended to be limiting. Those skilled in the art will recognize that a ribozyme of the invention, e.g., an enzymatic RNA molecule of this invention, can have a specific substrate binding site complementary to one or more of the target gene RNA regions. A ribozyme of the invention can have a nucleotide sequence within or surrounding that substrate binding site which imparts an RNA cleaving activity to the molecule.","RNA Interference (RNAi)","In one aspect, the invention provides an RNA inhibitory molecule, a so-called \u201cRNAi\u201d molecule, comprising a polypeptide, enzyme, protein, e.g. structural or binding protein, sequence of the invention. The RNAi molecule comprises a double-stranded RNA (dsRNA) molecule. The RNAi can inhibit expression of a polypeptide, enzyme, protein, e.g. structural or binding protein, gene. In one aspect, the RNAi is about 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25 or more duplex nucleotides in length. While the invention is not limited by any particular mechanism of action, the RNAi can enter a cell and cause the degradation of a single-stranded RNA (ssRNA) of similar or identical sequences, including endogenous mRNAs. When a cell is exposed to double-stranded RNA (dsRNA), mRNA from the homologous gene is selectively degraded by a process called RNA interference (RNAi). A possible basic mechanism behind RNAi, e.g., siRNA for inhibiting transcription and\/or miRNA to inhibit translation, is the breaking of a double-stranded RNA (dsRNA) matching a specific gene sequence into short pieces called short interfering RNA, which trigger the degradation of mRNA that matches its sequence. In one aspect, the RNAi's of the invention are used in gene-silencing therapeutics, see, e.g., Shuey (2002) Drug Discov. Today 7:1040-1046. In one aspect, the invention provides methods to selectively degrade RNA using the RNAi's of the invention. The process may be practiced in vitro, ex vivo or in vivo. In one aspect, the RNAi molecules of the invention can be used to generate a loss-of-function mutation in a cell, an organ or an animal. Methods for making and using RNAi molecules for selectively degrade RNA are well known in the art, see, e.g., U.S. Pat. Nos. 6,506,559; 6,511,824; 6,515,109; 6,489,127.","Modification of Nucleic Acids","The invention provides methods of generating variants of the nucleic acids of the invention, e.g., those encoding a polypeptide, enzyme, protein, e.g. structural or binding protein. These methods can be repeated or used in various combinations to generate a polypeptide, enzyme, protein, e.g. structural or binding protein, having an altered or different activity or an altered or different stability from that of a polypeptide, enzyme, protein, e.g. structural or binding protein, encoded by the template nucleic acid. These methods also can be repeated or used in various combinations, e.g., to generate variations in gene\/message expression, message translation or message stability. In another aspect, the genetic composition of a cell is altered by, e.g., modification of a homologous gene ex vivo, followed by its reinsertion into the cell.","A nucleic acid of the invention can be altered by any means. For example, random or stochastic methods, or, non-stochastic, or \u201cdirected evolution,\u201d methods, see, e.g., U.S. Pat. No. 6,361,974. Methods for random mutation of genes are well known in the art, see, e.g., U.S. Pat. No. 5,830,696. For example, mutagens can be used to randomly mutate a gene. Mutagens include, e.g., ultraviolet light or gamma irradiation, or a chemical mutagen, e.g., mitomycin, nitrous acid, photoactivated psoralens, alone or in combination, to induce DNA breaks amenable to repair by recombination. Other chemical mutagens include, for example, sodium bisulfite, nitrous acid, hydroxylamine, hydrazine or formic acid. Other mutagens are analogues of nucleotide precursors, e.g., nitrosoguanidine, 5-bromouracil, 2-aminopurine, or acridine. These agents can be added to a PCR reaction in place of the nucleotide precursor thereby mutating the sequence. Intercalating agents such as proflavine, acriflavine, quinacrine and the like can also be used.","Any technique in molecular biology can be used, e.g., random PCR mutagenesis, see, e.g., Rice (1992) Proc. Natl. Acad. Sci. USA 89:5467-5471; or, combinatorial multiple cassette mutagenesis, see, e.g., Crameri (1995) Biotechniques 18:194-196. Alternatively, nucleic acids, e.g., genes, can be reassembled after random, or \u201cstochastic,\u201d fragmentation, see, e.g., U.S. Pat. Nos. 6,291,242; 6,287,862; 6,287,861; 5,955,358; 5,830,721; 5,824,514; 5,811,238; 5,605,793. In alternative aspects, modifications, additions or deletions are introduced by error-prone PCR, shuffling, oligonucleotide-directed mutagenesis, assembly PCR, sexual PCR mutagenesis, in vivo mutagenesis, cassette mutagenesis, recursive ensemble mutagenesis, exponential ensemble mutagenesis, site-specific mutagenesis, gene reassembly, Gene Site Saturation Mutagenesis (GSSM), synthetic ligation reassembly (SLR), recombination, recursive sequence recombination, phosphothioate-modified DNA mutagenesis, uracil-containing template mutagenesis, gapped duplex mutagenesis, point mismatch repair mutagenesis, repair-deficient host strain mutagenesis, chemical mutagenesis, radiogenic mutagenesis, deletion mutagenesis, restriction-selection mutagenesis, restriction-purification mutagenesis, artificial gene synthesis, ensemble mutagenesis, chimeric nucleic acid multimer creation, and\/or a combination of these and other methods.","The following publications describe a variety of recursive recombination procedures and\/or methods which can be incorporated into the methods of the invention: Stemmer (1999) \u201cMolecular breeding of viruses for targeting and other clinical properties\u201d Tumor Targeting 4:1-4; Ness (1999) Nature Biotechnology 17:893-896; Chang (1999) \u201cEvolution of a cytokine using DNA family shuffling\u201d Nature Biotechnology 17:793-797; Minshull (1999) \u201cProtein evolution by molecular breeding\u201d Current Opinion in Chemical Biology 3:284-290; Christians (1999) \u201cDirected evolution of thymidine kinase for AZT phosphorylation using DNA family shuffling\u201d Nature Biotechnology 17:259-264; Crameri (1998) \u201cDNA shuffling of a family of genes from diverse species accelerates directed evolution\u201d Nature 391:288-291; Crameri (1997) \u201cMolecular evolution of an arsenate detoxification pathway by DNA shuffling,\u201d Nature Biotechnology 15:436-438; Zhang (1997) \u201cDirected evolution of an effective fucosidase from a galactosidase by DNA shuffling and screening\u201d Proc. Natl. Acad. Sci. USA 94:4504-4509; Patten et al. (1997) \u201cApplications of DNA Shuffling to Pharmaceuticals and Vaccines\u201d Current Opinion in Biotechnology 8:724-733; Crameri et al. (1996) \u201cConstruction and evolution of antibody-phage libraries by DNA shuffling\u201d Nature Medicine 2:100-103; Gates et al. (1996) \u201cAffinity selective isolation of ligands from peptide libraries through display on a lac repressor \u2018headpiece dimer\u2019\u201d Journal of Molecular Biology 255:373-386; Stemmer (1996) \u201cSexual PCR and Assembly PCR\u201d In: The Encyclopedia of Molecular Biology. VCH Publishers, New York. pp. 447-457; Crameri and Stemmer (1995) \u201cCombinatorial multiple cassette mutagenesis creates all the permutations of mutant and wildtype cassettes\u201d BioTechniques 18:194-195; Stemmer et al. (1995) \u201cSingle-step assembly of a gene and entire plasmid form large numbers of oligodeoxyribonucleotides\u201d Gene, 164:49-53; Stemmer (1995) \u201cThe Evolution of Molecular Computation\u201d Science 270: 1510; Stemmer (1995) \u201cSearching Sequence Space\u201d Bio\/Technology 13:549-553; Stemmer (1994) \u201cRapid evolution of a protein in vitro by DNA shuffling\u201d Nature 370:389-391; and Stemmer (1994) \u201cDNA shuffling by random fragmentation and reassembly: In vitro recombination for molecular evolution.\u201d Proc. Natl. Acad. Sci. USA 91:10747-10751.","Mutational methods of generating diversity include, for example, site-directed mutagenesis (Ling et al. (1997) \u201cApproaches to DNA mutagenesis: an overview\u201d Anal Biochem. 254(2): 157-178; Dale et al. (1996) \u201cOligonucleotide-directed random mutagenesis using the phosphorothioate method\u201d Methods Mol. Biol. 57:369-374; Smith (1985) \u201cIn vitro mutagenesis\u201d Ann. Rev. Genet. 19:423-462; Botstein & Shortle (1985) \u201cStrategies and applications of in vitro mutagenesis\u201d Science 229:1193-1201; Carter (1986) \u201cSite-directed mutagenesis\u201d Biochem. J. 237:1-7; and Kunkel (1987) \u201cThe efficiency of oligonucleotide directed mutagenesis\u201d in Nucleic Acids & Molecular Biology (Eckstein, F. and Lilley, D. M. J. eds., Springer Verlag, Berlin)); mutagenesis using uracil containing templates (Kunkel (1985) \u201cRapid and efficient site-specific mutagenesis without phenotypic selection\u201d Proc. Natl. Acad. Sci. USA 82:488-492; Kunkel et al. (1987) \u201cRapid and efficient site-specific mutagenesis without phenotypic selection\u201d Methods in Enzymol. 154, 367-382; and Bass et al. (1988) \u201cMutant Trp repressors with new DNA-binding specificities\u201d Science 242:240-245); oligonucleotide-directed mutagenesis (Methods in Enzymol. 100: 468-500 (1983); Methods in Enzymol. 154: 329-350 (1987); Zoller (1982) \u201cOligonucleotide-directed mutagenesis using M13-derived vectors: an efficient and general procedure for the production of point mutations in any DNA fragment\u201d Nucleic Acids Res. 10:6487-6500; Zoller & Smith (1983) \u201cOligonucleotide-directed mutagenesis of DNA fragments cloned into M13 vectors\u201d Methods in Enzymol. 100:468-500; and Zoller (1987) Oligonucleotide-directed mutagenesis: a simple method using two oligonucleotide primers and a single-stranded DNA template\u201d Methods in Enzymol. 154:329-350); phosphorothioate-modified DNA mutagenesis (Taylor (1985) \u201cThe use of phosphorothioate-modified DNA in restriction enzyme reactions to prepare nicked DNA\u201d Nucl. Acids Res. 13: 8749-8764; Taylor (1985) \u201cThe rapid generation of oligonucleotide-directed mutations at high frequency using phosphorothioate-modified DNA\u201d Nucl. Acids Res. 13: 8765-8787 (1985); Nakamaye (1986) \u201cInhibition of restriction endonuclease Nci I cleavage by phosphorothioate groups and its application to oligonucleotide-directed mutagenesis\u201d Nucl. Acids Res. 14: 9679-9698; Sayers (1988) \u201cY-T Exonucleases in phosphorothioate-based oligonucleotide-directed mutagenesis\u201d Nucl. Acids Res. 16:791-802; and Sayers et al. (1988) \u201cStrand specific cleavage of phosphorothioate-containing DNA by reaction with restriction endonucleases in the presence of ethidium bromide\u201d Nucl. Acids Res. 16: 803-814); mutagenesis using gapped duplex DNA (Kramer et al. (1984) \u201cThe gapped duplex DNA approach to oligonucleotide-directed mutation construction\u201d Nucl. Acids Res. 12: 9441-9456; Kramer & Fritz (1987) Methods in Enzymol. \u201cOligonucleotide-directed construction of mutations via gapped duplex DNA\u201d 154:350-367; Kramer (1988) \u201cImproved enzymatic in vitro reactions in the gapped duplex DNA approach to oligonucleotide-directed construction of mutations\u201d Nucl. Acids Res. 16: 7207; and Fritz (1988) \u201cOligonucleotide-directed construction of mutations: a gapped duplex DNA procedure without enzymatic reactions in vitro\u201d Nucl. Acids Res. 16: 6987-6999).","Additional protocols that can be used to practice the invention include point mismatch repair (Kramer (1984) \u201cPoint Mismatch Repair\u201d Cell 38:879-887), mutagenesis using repair-deficient host strains (Carter et al. (1985) \u201cImproved oligonucleotide site-directed mutagenesis using M13 vectors\u201d Nucl. Acids Res. 13: 4431-4443; and Carter (1987) \u201cImproved oligonucleotide-directed mutagenesis using M13 vectors\u201d Methods in Enzymol. 154: 382-403), deletion mutagenesis (Eghtedarzadeh (1986) \u201cUse of oligonucleotides to generate large deletions\u201d Nucl. Acids Res. 14: 5115), restriction-selection and restriction-selection and restriction-purification (Wells et al. (1986) \u201cImportance of hydrogen-bond formation in stabilizing the transition state of subtilisin\u201d Phil. Trans. R. Soc. Lond. A 317: 415-423), mutagenesis by total gene synthesis (Nambiar et al. (1984) \u201cTotal synthesis and cloning of a gene coding for the ribonuclease S protein\u201d Science 223: 1299-1301; Sakamar and Khorana (1988) \u201cTotal synthesis and expression of a gene for the a-subunit of bovine rod outer segment guanine nucleotide-binding protein (transducin)\u201d Nucl. Acids Res. 14: 6361-6372; Wells et al. (1985) \u201cCassette mutagenesis: an efficient method for generation of multiple mutations at defined sites\u201d Gene 34:315-323; and Grundstrom et al. (1985) \u201cOligonucleotide-directed mutagenesis by microscale \u2018shot-gun\u2019 gene synthesis\u201d Nucl. Acids Res. 13: 3305-3316), double-strand break repair (Mandecki (1986); Arnold (1993) \u201cProtein engineering for unusual environments\u201d Current Opinion in Biotechnology 4:450-455. \u201cOligonucleotide-directed double-strand break repair in plasmids of : a method for site-specific mutagenesis\u201d Proc. Natl. Acad. Sci. USA, 83:7177-7181). Additional details on many of the above methods can be found in Methods in Enzymology Volume 154, which also describes useful controls for trouble-shooting problems with various mutagenesis methods.","Protocols that can be used to practice the invention are described, e.g., in U.S. Pat. No. 5,605,793 to Stemmer (Feb. 25, 1997), \u201cMethods for In Vitro Recombination;\u201d U.S. Pat. No. 5,811,238 to Stemmer et al. (Sep. 22, 1998) \u201cMethods for Generating Polynucleotides having Desired Characteristics by Iterative Selection and Recombination;\u201d U.S. Pat. No. 5,830,721 to Stemmer et al. (Nov. 3, 1998), \u201cDNA Mutagenesis by Random Fragmentation and Reassembly;\u201d U.S. Pat. No. 5,834,252 to Stemmer, et al. (Nov. 10, 1998) \u201cEnd-Complementary Polymerase Reaction;\u201d U.S. Pat. No. 5,837,458 to Minshull, et al. (Nov. 17, 1998), \u201cMethods and Compositions for Cellular and Metabolic Engineering;\u201d WO 95\/22625, Stemmer and Crameri, \u201cMutagenesis by Random Fragmentation and Reassembly;\u201d WO 96\/33207 by Stemmer and Lipschutz \u201cEnd Complementary Polymerase Chain Reaction;\u201d WO 97\/20078 by Stemmer and Crameri \u201cMethods for Generating Polynucleotides having Desired Characteristics by Iterative Selection and Recombination;\u201d WO 97\/35966 by Minshull and Stemmer, \u201cMethods and Compositions for Cellular and Metabolic Engineering;\u201d WO 99\/41402 by Punnonen et al. \u201cTargeting of Genetic Vaccine Vectors;\u201d WO 99\/41383 by Punnonen et al. \u201cAntigen Library Immunization;\u201d WO 99\/41369 by Punnonen et al. \u201cGenetic Vaccine Vector Engineering;\u201d WO 99\/41368 by Punnonen et al. \u201cOptimization of Immunomodulatory Properties of Genetic Vaccines;\u201d EP 752008 by Stemmer and Crameri, \u201cDNA Mutagenesis by Random Fragmentation and Reassembly;\u201d EP 0932670 by Stemmer \u201cEvolving Cellular DNA Uptake by Recursive Sequence Recombination;\u201d WO 99\/23107 by Stemmer et al., \u201cModification of Virus Tropism and Host Range by Viral Genome Shuffling;\u201d WO 99\/21979 by Apt et al., \u201cHuman Papillomavirus Vectors;\u201d WO 98\/31837 by del Cardayre et al. \u201cEvolution of Whole Cells and Organisms by Recursive Sequence Recombination;\u201d WO 98\/27230 by Patten and Stemmer, \u201cMethods and Compositions for Polypeptide Engineering;\u201d WO 98\/27230 by Stemmer et al., \u201cMethods for Optimization of Gene Therapy by Recursive Sequence Shuffling and Selection,\u201d WO 00\/00632, \u201cMethods for Generating Highly Diverse Libraries,\u201d WO 00\/09679, \u201cMethods for Obtaining in Vitro Recombined Polynucleotide Sequence Banks and Resulting Sequences,\u201d WO 98\/42832 by Arnold et al., \u201cRecombination of Polynucleotide Sequences Using Random or Defined Primers,\u201d WO 99\/29902 by Arnold et al., \u201cMethod for Creating Polynucleotide and Polypeptide Sequences,\u201d WO 98\/41653 by Vind, \u201cAn in Vitro Method for Construction of a DNA Library,\u201d WO 98\/41622 by Borchert et al., \u201cMethod for Constructing a Library Using DNA Shuffling,\u201d and WO 98\/42727 by Pati and Zarling, \u201cSequence Alterations using Homologous Recombination.\u201d","Protocols that can be used to practice the invention (providing details regarding various diversity generating methods) are described, e.g., in U.S. patent application Ser. No. 09\/407,800, \u201cSHUFFLING OF CODON ALTERED GENES\u201d by Patten et al. filed Sep. 28, 1999; \u201cEVOLUTION OF WHOLE CELLS AND ORGANISMS BY RECURSIVE SEQUENCE RECOMBINATION\u201d by del Cardayre et al., U.S. Pat. No. 6,379,964; \u201cOLIGONUCLEOTIDE MEDIATED NUCLEIC ACID RECOMBINATION\u201d by Crameri et al., U.S. Pat. Nos. 6,319,714; 6,368,861; 6,376,246; 6,423,542; 6,426,224 and PCT\/US00\/01203; \u201cUSE OF CODON-VARIED OLIGONUCLEOTIDE SYNTHESIS FOR SYNTHETIC SHUFFLING\u201d by Welch et al., U.S. Pat. No. 6,436,675; \u201cMETHODS FOR MAKING CHARACTER STRINGS, POLYNUCLEOTIDES & POLYPEPTIDES HAVING DESIRED CHARACTERISTICS\u201d by Selifonov et al., filed Jan. 18, 2000, (PCT\/US00\/01202) and, e.g. \u201cMETHODS FOR MAKING CHARACTER STRINGS, POLYNUCLEOTIDES & POLYPEPTIDES HAVING DESIRED CHARACTERISTICS\u201d by Selifonov et al., filed Jul. 18, 2000 (U.S. Ser. No. 09\/618,579); \u201cMETHODS OF POPULATING DATA STRUCTURES FOR USE IN EVOLUTIONARY SIMULATIONS\u201d by Selifonov and Stemmer, filed Jan. 18, 2000 (PCT\/US00\/01138); and \u201cSINGLE-STRANDED NUCLEIC ACID TEMPLATE-MEDIATED RECOMBINATION AND NUCLEIC ACID FRAGMENT ISOLATION\u201d by Affholter, filed Sep. 6, 2000 (U.S. Ser. No. 09\/656,549); and U.S. Pat. Nos. 6,177,263; 6,153,410.","Non-stochastic, or \u201cdirected evolution,\u201d methods include, e.g., saturation mutagenesis, such as Gene Site Saturation Mutagenesis (GSSM), synthetic ligation reassembly (SLR), or a combination thereof are used to modify the nucleic acids of the invention to generate a polypeptide, enzyme, protein, e.g. structural or binding protein, with new or altered properties (e.g., activity under highly acidic or alkaline conditions, high or low temperatures, and the like). Polypeptides encoded by the modified nucleic acids can be screened for an activity before testing for glucan hydrolysis or other activity. Any testing modality or protocol can be used, e.g., using a capillary array platform. See, e.g., U.S. Pat. Nos. 6,361,974; 6,280,926; 5,939,250.","Gene Site Saturation Mutagenesis, or, GSSM","The invention also provides methods for making enzyme using Gene Site Saturation mutagenesis, or, GSSM, as described herein, and also in U.S. Pat. Nos. 6,171,820 and 6,579,258.","In one aspect, codon primers containing a degenerate N,N,G\/T sequence are used to introduce point mutations into a polynucleotide, e.g., a polypeptide, enzyme, protein, e.g. structural or binding protein, or an antibody of the invention, so as to generate a set of progeny polypeptides in which a full range of single amino acid substitutions is represented at each amino acid position, e.g., an amino acid residue in an enzyme active site or ligand binding site targeted to be modified. These oligonucleotides can comprise a contiguous first homologous sequence, a degenerate N,N,G\/T sequence, and, optionally, a second homologous sequence. The downstream progeny translational products from the use of such oligonucleotides include all possible amino acid changes at each amino acid site along the polypeptide, because the degeneracy of the N,N,G\/T sequence includes codons for all 20 amino acids. In one aspect, one such degenerate oligonucleotide (comprised of, e.g., one degenerate N,N,G\/T cassette) is used for subjecting each original codon in a parental polynucleotide template to a full range of codon substitutions. In another aspect, at least two degenerate cassettes are used\u2014either in the same oligonucleotide or not, for subjecting at least two original codons in a parental polynucleotide template to a full range of codon substitutions. For example, more than one N,N,G\/T sequence can be contained in one oligonucleotide to introduce amino acid mutations at more than one site. This plurality of N,N,G\/T sequences can be directly contiguous, or separated by one or more additional nucleotide sequence(s). In another aspect, oligonucleotides serviceable for introducing additions and deletions can be used either alone or in combination with the codons containing an N,N,G\/T sequence, to introduce any combination or permutation of amino acid additions, deletions, and\/or substitutions.","In one aspect, simultaneous mutagenesis of two or more contiguous amino acid positions is done using an oligonucleotide that contains contiguous N,N,G\/T triplets, i.e. a degenerate (N,N,G\/T)n sequence. In another aspect, degenerate cassettes having less degeneracy than the N,N,G\/T sequence are used. For example, it may be desirable in some instances to use (e.g. in an oligonucleotide) a degenerate triplet sequence comprised of only one N, where said N can be in the first second or third position of the triplet. Any other bases including any combinations and permutations thereof can be used in the remaining two positions of the triplet. Alternatively, it may be desirable in some instances to use (e.g. in an oligo) a degenerate N,N,N triplet sequence.","In one aspect, use of degenerate triplets (e.g., N,N,G\/T triplets) allows for systematic and easy generation of a full range of possible natural amino acids (for a total of 20 amino acids) into each and every amino acid position in a polypeptide (in alternative aspects, the methods also include generation of less than all possible substitutions per amino acid residue, or codon, position). For example, for a 100 amino acid polypeptide, 2000 distinct species (i.e. 20 possible amino acids per position\u00d7100 amino acid positions) can be generated. Through the use of an oligonucleotide or set of oligonucleotides containing a degenerate N,N,G\/T triplet, 32 individual sequences can code for all 20 possible natural amino acids. Thus, in a reaction vessel in which a parental polynucleotide sequence is subjected to saturation mutagenesis using at least one such oligonucleotide, there are generated 32 distinct progeny polynucleotides encoding 20 distinct polypeptides. In contrast, the use of a non-degenerate oligonucleotide in site-directed mutagenesis leads to only one progeny polypeptide product per reaction vessel. Nondegenerate oligonucleotides can optionally be used in combination with degenerate primers disclosed; for example, nondegenerate oligonucleotides can be used to generate specific point mutations in a working polynucleotide. This provides one means to generate specific silent point mutations, point mutations leading to corresponding amino acid changes, and point mutations that cause the generation of stop codons and the corresponding expression of polypeptide fragments.","In one aspect, each saturation mutagenesis reaction vessel contains polynucleotides encoding at least 20 progeny polypeptide (e.g., a polypeptide, enzyme, protein, e.g. structural or binding protein) molecules such that all 20 natural amino acids are represented at the one specific amino acid position corresponding to the codon position mutagenized in the parental polynucleotide (other aspects use less than all 20 natural combinations). The 32-fold degenerate progeny polypeptides generated from each saturation mutagenesis reaction vessel can be subjected to clonal amplification (e.g. cloned into a suitable host, e.g., E. coif host, using, e.g., an expression vector) and subjected to expression screening. When an individual progeny polypeptide is identified by screening to display a favorable change in property (when compared to the parental polypeptide, such as increased glucan hydrolysis activity under alkaline or acidic conditions), it can be sequenced to identify the correspondingly favorable amino acid substitution contained therein.","In one aspect, upon mutagenizing each and every amino acid position in a parental polypeptide using saturation mutagenesis as disclosed herein, favorable amino acid changes may be identified at more than one amino acid position. One or more new progeny molecules can be generated that contain a combination of all or part of these favorable amino acid substitutions. For example, if 2 specific favorable amino acid changes are identified in each of 3 amino acid positions in a polypeptide, the permutations include 3 possibilities at each position (no change from the original amino acid, and each of two favorable changes) and 3 positions. Thus, there are 3\u00d73\u00d73 or 27 total possibilities, including 7 that were previously examined-6 single point mutations (i.e. 2 at each of three positions) and no change at any position.","In yet another aspect, site-saturation mutagenesis can be used together with shuffling, chimerization, recombination and other mutagenizing processes, along with screening. This invention provides for the use of any mutagenizing process(es), including saturation mutagenesis, in an iterative manner. In one exemplification, the iterative use of any mutagenizing process(es) is used in combination with screening.","The invention also provides for the use of proprietary codon primers (containing a degenerate N,N,N sequence) to introduce point mutations into a polynucleotide, so as to generate a set of progeny polypeptides in which a full range of single amino acid substitutions is represented at each amino acid position; e.g., with Gene Site Saturation Mutagenesis (GSSM). The oligos used are comprised contiguously of a first homologous sequence, a degenerate N,N,N sequence and in one aspect but not necessarily a second homologous sequence. The downstream progeny translational products from the use of such oligos include all possible amino acid changes at each amino acid site along the polypeptide, because the degeneracy of the N,N,N sequence includes codons for all 20 amino acids.","In one aspect, one such degenerate oligo (comprised of one degenerate N,N,N cassette) is used for subjecting each original codon in a parental polynucleotide template to a full range of codon substitutions. In another aspect, at least two degenerate N,N,N cassettes are used\u2014either in the same oligo or not, for subjecting at least two original codons in a parental polynucleotide template to a full range of codon substitutions. Thus, more than one N,N,N sequence can be contained in one oligo to introduce amino acid mutations at more than one site. This plurality of N,N,N sequences can be directly contiguous, or separated by one or more additional nucleotide sequence(s). In another aspect, oligos serviceable for introducing additions and deletions can be used either alone or in combination with the codons containing an N,N,N sequence, to introduce any combination or permutation of amino acid additions, deletions and\/or substitutions.","In a particular exemplification, it is possible to simultaneously mutagenize two or more contiguous amino acid positions using an oligo that contains contiguous N,N,N triplets, i.e. a degenerate (N,N,N)sequence.","In another aspect, the present invention provides for the use of degenerate cassettes having less degeneracy than the N,N,N sequence. For example, it may be desirable in some instances to use (e.g. in an oligo) a degenerate triplet sequence comprised of only one N, where the N can be in the first second or third position of the triplet. Any other bases including any combinations and permutations thereof can be used in the remaining two positions of the triplet. Alternatively, it may be desirable in some instances to use (e.g., in an oligo) a degenerate N,N,N triplet sequence, N,N,G\/T, or an N,N, G\/C triplet sequence.","It is appreciated, however, that the use of a degenerate triplet (such as N,N,G\/T or an N,N, G\/C triplet sequence) as disclosed in the instant invention is advantageous for several reasons. In one aspect, this invention provides a means to systematically and fairly easily generate the substitution of the full range of possible amino acids (for a total of 20 amino acids) into each and every amino acid position in a polypeptide. Thus, for a 100 amino acid polypeptide, the invention provides a way to systematically and fairly easily generate 2000 distinct species (i.e., 20 possible amino acids per position times 100 amino acid positions). It is appreciated that there is provided, through the use of an oligo containing a degenerate N,N,G\/T or an N,N, G\/C triplet sequence, 32 individual sequences that code for 20 possible amino acids. Thus, in a reaction vessel in which a parental polynucleotide sequence is subjected to saturation mutagenesis using one such oligo, there are generated 32 distinct progeny polynucleotides encoding 20 distinct polypeptides. In contrast, the use of a non-degenerate oligo in site-directed mutagenesis leads to only one progeny polypeptide product per reaction vessel.","This invention also provides for the use of nondegenerate oligos, which can optionally be used in combination with degenerate primers disclosed. It is appreciated that in some situations, it is advantageous to use nondegenerate oligos to generate specific point mutations in a working polynucleotide. This provides a means to generate specific silent point mutations, point mutations leading to corresponding amino acid changes and point mutations that cause the generation of stop codons and the corresponding expression of polypeptide fragments.","Thus, in one aspect of this invention, each saturation mutagenesis reaction vessel contains polynucleotides encoding at least 20 progeny polypeptide molecules such that all 20 amino acids are represented at the one specific amino acid position corresponding to the codon position mutagenized in the parental polynucleotide. The 32-fold degenerate progeny polypeptides generated from each saturation mutagenesis reaction vessel can be subjected to clonal amplification (e.g., cloned into a suitable host using an expression vector) and subjected to expression screening. When an individual progeny polypeptide is identified by screening to display a favorable change in property (when compared to the parental polypeptide), it can be sequenced to identify the correspondingly favorable amino acid substitution contained therein.","It is appreciated that upon mutagenizing each and every amino acid position in a parental polypeptide using saturation mutagenesis as disclosed herein, favorable amino acid changes may be identified at more than one amino acid position. One or more new progeny molecules can be generated that contain a combination of all or part of these favorable amino acid substitutions. For example, if 2 specific favorable amino acid changes are identified in each of 3 amino acid positions in a polypeptide, the permutations include 3 possibilities at each position (no change from the original amino acid and each of two favorable changes) and 3 positions. Thus, there are 3\u00d73\u00d73 or 27 total possibilities, including 7 that were previously examined-6 single point mutations (i.e., 2 at each of three positions) and no change at any position.","Thus, in a non-limiting exemplification, this invention provides for the use of saturation mutagenesis in combination with additional mutagenization processes, such as process where two or more related polynucleotides are introduced into a suitable host cell such that a hybrid polynucleotide is generated by recombination and reductive reassortment.","In addition to performing mutagenesis along the entire sequence of a gene, the instant invention provides that mutagenesis can be use to replace each of any number of bases in a polynucleotide sequence, wherein the number of bases to be mutagenized is in one aspect every integer from 15 to 100,000. Thus, instead of mutagenizing every position along a molecule, one can subject every or a discrete number of bases (in one aspect a subset totaling from 15 to 100,000) to mutagenesis. In one aspect, a separate nucleotide is used for mutagenizing each position or group of positions along a polynucleotide sequence. A group of 3 positions to be mutagenized may be a codon. The mutations can be introduced using a mutagenic primer, containing a heterologous cassette, also referred to as a mutagenic cassette. Exemplary cassettes can have from 1 to 500 bases. Each nucleotide position in such heterologous cassettes be N, A, C, G, T, A\/C, A\/G, A\/T, C\/G, C\/T, G\/T, C\/G\/T, A\/G\/T, A\/C\/T, A\/C\/G, or E, where E is any base that is not A, C, G, or T (E can be referred to as a designer oligo).","In a general sense, saturation mutagenesis is comprised of mutagenizing a complete set of mutagenic cassettes (wherein each cassette is in one aspect about 1-500 bases in length) in defined polynucleotide sequence to be mutagenized (wherein the sequence to be mutagenized is in one aspect from about 15 to 100,000 bases in length). Thus, a group of mutations (ranging from 1 to 100 mutations) is introduced into each cassette to be mutagenized. A grouping of mutations to be introduced into one cassette can be different or the same from a second grouping of mutations to be introduced into a second cassette during the application of one round of saturation mutagenesis. Such groupings are exemplified by deletions, additions, groupings of particular codons and groupings of particular nucleotide cassettes.","Defined sequences to be mutagenized include a whole gene, pathway, cDNA, an entire open reading frame (ORF) and entire promoter, enhancer, repressor\/transactivator, origin of replication, intron, operator, or any polynucleotide functional group. Generally, a \u201cdefined sequences\u201d for this purpose may be any polynucleotide that a 15 base-polynucleotide sequence and polynucleotide sequences of lengths between 15 bases and 15,000 bases (this invention specifically names every integer in between). Considerations in choosing groupings of codons include types of amino acids encoded by a degenerate mutagenic cassette.","In one exemplification a grouping of mutations that can be introduced into a mutagenic cassette, this invention specifically provides for degenerate codon substitutions (using degenerate oligos) that code for 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19 and 20 amino acids at each position and a library of polypeptides encoded thereby.","Synthetic Ligation Reassembly (SLR)","The invention provides a non-stochastic gene modification system termed \u201csynthetic ligation reassembly,\u201d or simply \u201cSLR,\u201d a \u201cdirected evolution process,\u201d to generate polypeptides, e.g., a polypeptide, enzyme, protein, e.g. structural or binding protein, or antibodies of the invention, with new or altered properties. SLR is a method of ligating oligonucleotide fragments together non-stochastically. This method differs from stochastic oligonucleotide shuffling in that the nucleic acid building blocks are not shuffled, concatenated or chimerized randomly, but rather are assembled non-stochastically. See, e.g., U.S. Pat. Nos. 6,773,900; 6,740,506; 6,713,282; 6,635,449; 6,605,449; 6,537,776.","In one aspect, SLR comprises the following steps: (a) providing a template polynucleotide, wherein the template polynucleotide comprises sequence encoding a homologous gene; (b) providing a plurality of building block polynucleotides, wherein the building block polynucleotides are designed to cross-over reassemble with the template polynucleotide at a predetermined sequence, and a building block polynucleotide comprises a sequence that is a variant of the homologous gene and a sequence homologous to the template polynucleotide flanking the variant sequence; (c) combining a building block polynucleotide with a template polynucleotide such that the building block polynucleotide cross-over reassembles with the template polynucleotide to generate polynucleotides comprising homologous gene sequence variations.","SLR does not depend on the presence of high levels of homology between polynucleotides to be rearranged. Thus, this method can be used to non-stochastically generate libraries (or sets) of progeny molecules comprised of over 10different chimeras. SLR can be used to generate libraries comprised of over 10different progeny chimeras. Thus, aspects of the present invention include non-stochastic methods of producing a set of finalized chimeric nucleic acid molecule shaving an overall assembly order that is chosen by design. This method includes the steps of generating by design a plurality of specific nucleic acid building blocks having serviceable mutually compatible ligatable ends, and assembling these nucleic acid building blocks, such that a designed overall assembly order is achieved.","The mutually compatible ligatable ends of the nucleic acid building blocks to be assembled are considered to be \u201cserviceable\u201d for this type of ordered assembly if they enable the building blocks to be coupled in predetermined orders. Thus, the overall assembly order in which the nucleic acid building blocks can be coupled is specified by the design of the ligatable ends. If more than one assembly step is to be used, then the overall assembly order in which the nucleic acid building blocks can be coupled is also specified by the sequential order of the assembly step(s). In one aspect, the annealed building pieces are treated with an enzyme, such as a ligase (e.g. T4 DNA ligase), to achieve covalent bonding of the building pieces.","In one aspect, the design of the oligonucleotide building blocks is obtained by analyzing a set of progenitor nucleic acid sequence templates that serve as a basis for producing a progeny set of finalized chimeric polynucleotides. These parental oligonucleotide templates thus serve as a source of sequence information that aids in the design of the nucleic acid building blocks that are to be mutagenized, e.g., chimerized or shuffled. In one aspect of this method, the sequences of a plurality of parental nucleic acid templates are aligned in order to select one or more demarcation points. The demarcation points can be located at an area of homology, and are comprised of one or more nucleotides. These demarcation points are in one aspect shared by at least two of the progenitor templates. The demarcation points can thereby be used to delineate the boundaries of oligonucleotide building blocks to be generated in order to rearrange the parental polynucleotides. The demarcation points identified and selected in the progenitor molecules serve as potential chimerization points in the assembly of the final chimeric progeny molecules. A demarcation point can be an area of homology (comprised of at least one homologous nucleotide base) shared by at least two parental polynucleotide sequences. Alternatively, a demarcation point can be an area of homology that is shared by at least half of the parental polynucleotide sequences, or, it can be an area of homology that is shared by at least two thirds of the parental polynucleotide sequences. Even more in one aspect a serviceable demarcation points is an area of homology that is shared by at least three fourths of the parental polynucleotide sequences, or, it can be shared by at almost all of the parental polynucleotide sequences. In one aspect, a demarcation point is an area of homology that is shared by all of the parental polynucleotide sequences.","In one aspect, a ligation reassembly process is performed exhaustively in order to generate an exhaustive library of progeny chimeric polynucleotides. In other words, all possible ordered combinations of the nucleic acid building blocks are represented in the set of finalized chimeric nucleic acid molecules. At the same time, in another aspect, the assembly order (i.e. the order of assembly of each building block in the 5\u2032 to 3 sequence of each finalized chimeric nucleic acid) in each combination is by design (or non-stochastic) as described above. Because of the non-stochastic nature of this invention, the possibility of unwanted side products is greatly reduced.","In another aspect, the ligation reassembly method is performed systematically. For example, the method is performed in order to generate a systematically compartmentalized library of progeny molecules, with compartments that can be screened systematically, e.g. one by one. In other words this invention provides that, through the selective and judicious use of specific nucleic acid building blocks, coupled with the selective and judicious use of sequentially stepped assembly reactions, a design can be achieved where specific sets of progeny products are made in each of several reaction vessels. This allows a systematic examination and screening procedure to be performed. Thus, these methods allow a potentially very large number of progeny molecules to be examined systematically in smaller groups. Because of its ability to perform chimerizations in a manner that is highly flexible yet exhaustive and systematic as well, particularly when there is a low level of homology among the progenitor molecules, these methods provide for the generation of a library (or set) comprised of a large number of progeny molecules. Because of the non-stochastic nature of the instant ligation reassembly invention, the progeny molecules generated in one aspect comprise a library of finalized chimeric nucleic acid molecules having an overall assembly order that is chosen by design. The saturation mutagenesis and optimized directed evolution methods also can be used to generate different progeny molecular species. It is appreciated that the invention provides freedom of choice and control regarding the selection of demarcation points, the size and number of the nucleic acid building blocks, and the size and design of the couplings. It is appreciated, furthermore, that the requirement for intermolecular homology is highly relaxed for the operability of this invention. In fact, demarcation points can even be chosen in areas of little or no intermolecular homology. For example, because of codon wobble, i.e. the degeneracy of codons, nucleotide substitutions can be introduced into nucleic acid building blocks without altering the amino acid originally encoded in the corresponding progenitor template. Alternatively, a codon can be altered such that the coding for an originally amino acid is altered. This invention provides that such substitutions can be introduced into the nucleic acid building block in order to increase the incidence of intermolecular homologous demarcation points and thus to allow an increased number of couplings to be achieved among the building blocks, which in turn allows a greater number of progeny chimeric molecules to be generated.","In one aspect, the present invention provides a non-stochastic method termed synthetic gene reassembly, that is somewhat related to stochastic shuffling, save that the nucleic acid building blocks are not shuffled or concatenated or chimerized randomly, but rather are assembled non-stochastically.","The synthetic gene reassembly method does not depend on the presence of a high level of homology between polynucleotides to be shuffled. The invention can be used to non-stochastically generate libraries (or sets) of progeny molecules comprised of over 10different chimeras. Conceivably, synthetic gene reassembly can even be used to generate libraries comprised of over 10different progeny chimeras.","Thus, in one aspect, the invention provides a non-stochastic method of producing a set of finalized chimeric nucleic acid molecules having an overall assembly order that is chosen by design, which method is comprised of the steps of generating by design a plurality of specific nucleic acid building blocks having serviceable mutually compatible ligatable ends and assembling these nucleic acid building blocks, such that a designed overall assembly order is achieved.","The mutually compatible ligatable ends of the nucleic acid building blocks to be assembled are considered to be \u201cserviceable\u201d for this type of ordered assembly if they enable the building blocks to be coupled in predetermined orders. Thus, in one aspect, the overall assembly order in which the nucleic acid building blocks can be coupled is specified by the design of the ligatable ends and, if more than one assembly step is to be used, then the overall assembly order in which the nucleic acid building blocks can be coupled is also specified by the sequential order of the assembly step(s). In a one aspect of the invention, the annealed building pieces are treated with an enzyme, such as a ligase (e.g., T4 DNA ligase) to achieve covalent bonding of the building pieces.","In a another aspect, the design of nucleic acid building blocks is obtained upon analysis of the sequences of a set of progenitor nucleic acid templates that serve as a basis for producing a progeny set of finalized chimeric nucleic acid molecules. These progenitor nucleic acid templates thus serve as a source of sequence information that aids in the design of the nucleic acid building blocks that are to be mutagenized, i.e. chimerized or shuffled.","In one exemplification, the invention provides for the chimerization of a family of related genes and their encoded family of related products. In a particular exemplification, the encoded products are enzymes. The polypeptide, enzyme, protein, e.g. structural or binding proteins of the present invention can be mutagenized in accordance with the methods described herein.","Thus according to one aspect of the invention, the sequences of a plurality of progenitor nucleic acid templates (e.g., polynucleotides of the invention) are aligned in order to select one or more demarcation points, which demarcation points can be located at an area of homology. The demarcation points can be used to delineate the boundaries of nucleic acid building blocks to be generated. Thus, the demarcation points identified and selected in the progenitor molecules serve as potential chimerization points in the assembly of the progeny molecules.","Typically a serviceable demarcation point is an area of homology (comprised of at least one homologous nucleotide base) shared by at least two progenitor templates, but the demarcation point can be an area of homology that is shared by at least half of the progenitor templates, at least two thirds of the progenitor templates, at least three fourths of the progenitor templates and in one aspect at almost all of the progenitor templates. Even more in one aspect still a serviceable demarcation point is an area of homology that is shared by all of the progenitor templates.","In a one aspect, the gene reassembly process is performed exhaustively in order to generate an exhaustive library. In other words, all possible ordered combinations of the nucleic acid building blocks are represented in the set of finalized chimeric nucleic acid molecules. At the same time, the assembly order (i.e. the order of assembly of each building block in the 5\u2032 to 3 sequence of each finalized chimeric nucleic acid) in each combination is by design (or non-stochastic). Because of the non-stochastic nature of the method, the possibility of unwanted side products is greatly reduced.","In another aspect, the method provides that the gene reassembly process is performed systematically, for example to generate a systematically compartmentalized library, with compartments that can be screened systematically, e.g., one by one. In other words the invention provides that, through the selective and judicious use of specific nucleic acid building blocks, coupled with the selective and judicious use of sequentially stepped assembly reactions, an experimental design can be achieved where specific sets of progeny products are made in each of several reaction vessels. This allows a systematic examination and screening procedure to be performed. Thus, it allows a potentially very large number of progeny molecules to be examined systematically in smaller groups.","Because of its ability to perform chimerizations in a manner that is highly flexible yet exhaustive and systematic as well, particularly when there is a low level of homology among the progenitor molecules, the instant invention provides for the generation of a library (or set) comprised of a large number of progeny molecules. Because of the non-stochastic nature of the instant gene reassembly invention, the progeny molecules generated in one aspect comprise a library of finalized chimeric nucleic acid molecules having an overall assembly order that is chosen by design. In a particularly aspect, such a generated library is comprised of greater than 10to greater than 10different progeny molecular species.","In one aspect, a set of finalized chimeric nucleic acid molecules, produced as described is comprised of a polynucleotide encoding a polypeptide. According to one aspect, this polynucleotide is a gene, which may be a man-made gene. According to another aspect, this polynucleotide is a gene pathway, which may be a man-made gene pathway. The invention provides that one or more man-made genes generated by the invention may be incorporated into a man-made gene pathway, such as pathway operable in a eukaryotic organism (including a plant).","In another exemplification, the synthetic nature of the step in which the building blocks are generated allows the design and introduction of nucleotides (e.g., one or more nucleotides, which may be, for example, codons or introns or regulatory sequences) that can later be optionally removed in an in vitro process (e.g., by mutagenesis) or in an in vivo process (e.g., by utilizing the gene splicing ability of a host organism). It is appreciated that in many instances the introduction of these nucleotides may also be desirable for many other reasons in addition to the potential benefit of creating a serviceable demarcation point.","Thus, according to another aspect, the invention provides that a nucleic acid building block can be used to introduce an intron. Thus, the invention provides that functional introns may be introduced into a man-made gene of the invention. The invention also provides that functional introns may be introduced into a man-made gene pathway of the invention. Accordingly, the invention provides for the generation of a chimeric polynucleotide that is a man-made gene containing one (or more) artificially introduced intron(s).","Accordingly, the invention also provides for the generation of a chimeric polynucleotide that is a man-made gene pathway containing one (or more) artificially introduced intron(s). In one aspect, the artificially introduced intron(s) are functional in one or more host cells for gene splicing much in the way that naturally-occurring introns serve functionally in gene splicing. The invention provides a process of producing man-made intron-containing polynucleotides to be introduced into host organisms for recombination and\/or splicing.","A man-made gene produced using the invention can also serve as a substrate for recombination with another nucleic acid. Likewise, a man-made gene pathway produced using the invention can also serve as a substrate for recombination with another nucleic acid. In one aspect, the recombination is facilitated by, or occurs at, areas of homology between the man-made, intron-containing gene and a nucleic acid, which serves as a recombination partner. In one aspect, the recombination partner may also be a nucleic acid generated by the invention, including a man-made gene or a man-made gene pathway. Recombination may be facilitated by or may occur at areas of homology that exist at the one (or more) artificially introduced intron(s) in the man-made gene.","The synthetic gene reassembly method of the invention utilizes a plurality of nucleic acid building blocks, each of which in one aspect has two ligatable ends. The two ligatable ends on each nucleic acid building block may be two blunt ends (i.e. each having an overhang of zero nucleotides), or in one aspect one blunt end and one overhang, or more in one aspect still two overhangs.","A useful overhang for this purpose may be a 3\u2032 overhang or a 5\u2032 overhang. Thus, a nucleic acid building block may have a 3\u2032 overhang or alternatively a 5\u2032 overhang or alternatively two 3\u2032 overhangs or alternatively two 5\u2032 overhangs. The overall order in which the nucleic acid building blocks are assembled to form a finalized chimeric nucleic acid molecule is determined by purposeful experimental design and is not random.","In one aspect, a nucleic acid building block is generated by chemical synthesis of two single-stranded nucleic acids (also referred to as single-stranded oligos) and contacting them so as to allow them to anneal to form a double-stranded nucleic acid building block.","A double-stranded nucleic acid building block can be of variable size. The sizes of these building blocks can be small or large. Exemplary sizes for building block range from 1 base pair (not including any overhangs) to 100,000 base pairs (not including any overhangs). Other exemplary size ranges are also provided, which have lower limits of from 1 bp to 10,000 bp (including every integer value in between) and upper limits of from 2 bp to 100,000 bp (including every integer value in between).","Many methods exist by which a double-stranded nucleic acid building block can be generated that is serviceable for the invention; and these are known in the art and can be readily performed by the skilled artisan.","According to one aspect, a double-stranded nucleic acid building block is generated by first generating two single stranded nucleic acids and allowing them to anneal to form a double-stranded nucleic acid building block. The two strands of a double-stranded nucleic acid building block may be complementary at every nucleotide apart from any that form an overhang; thus containing no mismatches, apart from any overhang(s). According to another aspect, the two strands of a double-stranded nucleic acid building block are complementary at fewer than every nucleotide apart from any that form an overhang. Thus, according to this aspect, a double-stranded nucleic acid building block can be used to introduce codon degeneracy. In one aspect the codon degeneracy is introduced using the site-saturation mutagenesis described herein, using one or more N,N,G\/T cassettes or alternatively using one or more N,N,N cassettes.","The in vivo recombination method of the invention can be performed blindly on a pool of unknown hybrids or alleles of a specific polynucleotide or sequence. However, it is not necessary to know the actual DNA or RNA sequence of the specific polynucleotide.","The approach of using recombination within a mixed population of genes can be useful for the generation of any useful proteins, for example, interleukin I, antibodies, tPA and growth hormone. This approach may be used to generate proteins having altered specificity or activity. The approach may also be useful for the generation of hybrid nucleic acid sequences, for example, promoter regions, introns, exons, enhancer sequences, 31 untranslated regions or 51 untranslated regions of genes. Thus this approach may be used to generate genes having increased rates of expression. This approach may also be useful in the study of repetitive DNA sequences. Finally, this approach may be useful to mutate ribozymes or aptamers.","In one aspect the invention described herein is directed to the use of repeated cycles of reductive reassortment, recombination and selection which allow for the directed molecular evolution of highly complex linear sequences, such as DNA, RNA or proteins thorough recombination.","Optimized Directed Evolution System","The invention provides a non-stochastic gene modification system termed \u201coptimized directed evolution system\u201d to generate polypeptides, e.g., a polypeptide, enzyme, protein, e.g. structural or binding protein, or antibodies of the invention, with new or altered properties. Optimized directed evolution is directed to the use of repeated cycles of reductive reassortment, recombination and selection that allow for the directed molecular evolution of nucleic acids through recombination. Optimized directed evolution allows generation of a large population of evolved chimeric sequences, wherein the generated population is significantly enriched for sequences that have a predetermined number of crossover events.","A crossover event is a point in a chimeric sequence where a shift in sequence occurs from one parental variant to another parental variant. Such a point is normally at the juncture of where oligonucleotides from two parents are ligated together to form a single sequence. This method allows calculation of the correct concentrations of oligonucleotide sequences so that the final chimeric population of sequences is enriched for the chosen number of crossover events. This provides more control over choosing chimeric variants having a predetermined number of crossover events.","In addition, this method provides a convenient means for exploring a tremendous amount of the possible protein variant space in comparison to other systems. Previously, if one generated, for example, 10chimeric molecules during a reaction, it would be extremely difficult to test such a high number of chimeric variants for a particular activity. Moreover, a significant portion of the progeny population would have a very high number of crossover events which resulted in proteins that were less likely to have increased levels of a particular activity. By using these methods, the population of chimerics molecules can be enriched for those variants that have a particular number of crossover events. Thus, although one can still generate 10chimeric molecules during a reaction, each of the molecules chosen for further analysis most likely has, for example, only three crossover events. Because the resulting progeny population can be skewed to have a predetermined number of crossover events, the boundaries on the functional variety between the chimeric molecules is reduced. This provides a more manageable number of variables when calculating which oligonucleotide from the original parental polynucleotides might be responsible for affecting a particular trait.","One method for creating a chimeric progeny polynucleotide sequence is to create oligonucleotides corresponding to fragments or portions of each parental sequence. Each oligonucleotide in one aspect includes a unique region of overlap so that mixing the oligonucleotides together results in a new variant that has each oligonucleotide fragment assembled in the correct order. Alternatively protocols for practicing these methods of the invention can be found in U.S. Pat. Nos. 6,773,900; 6,740,506; 6,713,282; 6,635,449; 6,605,449; 6,537,776; 6,361,974.","The number of oligonucleotides generated for each parental variant bears a relationship to the total number of resulting crossovers in the chimeric molecule that is ultimately created. For example, three parental nucleotide sequence variants might be provided to undergo a ligation reaction in order to find a chimeric variant having, for example, greater activity at high temperature. As one example, a set of 50 oligonucleotide sequences can be generated corresponding to each portions of each parental variant. Accordingly, during the ligation reassembly process there could be up to 50 crossover events within each of the chimeric sequences. The probability that each of the generated chimeric polynucleotides will contain oligonucleotides from each parental variant in alternating order is very low. If each oligonucleotide fragment is present in the ligation reaction in the same molar quantity it is likely that in some positions oligonucleotides from the same parental polynucleotide will ligate next to one another and thus not result in a crossover event. If the concentration of each oligonucleotide from each parent is kept constant during any ligation step in this example, there is a \u2153 chance (assuming 3 parents) that an oligonucleotide from the same parental variant will ligate within the chimeric sequence and produce no crossover.","Accordingly, a probability density function (PDF) can be determined to predict the population of crossover events that are likely to occur during each step in a ligation reaction given a set number of parental variants, a number of oligonucleotides corresponding to each variant, and the concentrations of each variant during each step in the ligation reaction. The statistics and mathematics behind determining the PDF is described below. By utilizing these methods, one can calculate such a probability density function, and thus enrich the chimeric progeny population for a predetermined number of crossover events resulting from a particular ligation reaction. Moreover, a target number of crossover events can be predetermined, and the system then programmed to calculate the starting quantities of each parental oligonucleotide during each step in the ligation reaction to result in a probability density function that centers on the predetermined number of crossover events. These methods are directed to the use of repeated cycles of reductive reassortment, recombination and selection that allow for the directed molecular evolution of a nucleic acid encoding a polypeptide through recombination. This system allows generation of a large population of evolved chimeric sequences, wherein the generated population is significantly enriched for sequences that have a predetermined number of crossover events. A crossover event is a point in a chimeric sequence where a shift in sequence occurs from one parental variant to another parental variant. Such a point is normally at the juncture of where oligonucleotides from two parents are ligated together to form a single sequence. The method allows calculation of the correct concentrations of oligonucleotide sequences so that the final chimeric population of sequences is enriched for the chosen number of crossover events. This provides more control over choosing chimeric variants having a predetermined number of crossover events.","In addition, these methods provide a convenient means for exploring a tremendous amount of the possible protein variant space in comparison to other systems. By using the methods described herein, the population of chimerics molecules can be enriched for those variants that have a particular number of crossover events. Thus, although one can still generate 10chimeric molecules during a reaction, each of the molecules chosen for further analysis most likely has, for example, only three crossover events. Because the resulting progeny population can be skewed to have a predetermined number of crossover events, the boundaries on the functional variety between the chimeric molecules is reduced. This provides a more manageable number of variables when calculating which oligonucleotide from the original parental polynucleotides might be responsible for affecting a particular trait.","In one aspect, the method creates a chimeric progeny polynucleotide sequence by creating oligonucleotides corresponding to fragments or portions of each parental sequence. Each oligonucleotide in one aspect includes a unique region of overlap so that mixing the oligonucleotides together results in a new variant that has each oligonucleotide fragment assembled in the correct order. See also U.S. Ser. No. 09\/332,835.","Determining Crossover Events","Aspects of the invention include a system and software that receive a desired crossover probability density function (PDF), the number of parent genes to be reassembled, and the number of fragments in the reassembly as inputs. The output of this program is a \u201cfragment PDF\u201d that can be used to determine a recipe for producing reassembled genes, and the estimated crossover PDF of those genes. The processing described herein is in one aspect performed in MATLAB\u2122 (The Mathworks, Natick, Mass.) a programming language and development environment for technical computing.","Iterative Processes","In practicing the invention, these processes can be iteratively repeated. For example, a nucleic acid (or, the nucleic acid) responsible for an altered or new a polypeptide, enzyme, protein, e.g. structural or binding protein, phenotype is identified, re-isolated, again modified, re-tested for activity. This process can be iteratively repeated until a desired phenotype is engineered. For example, an entire biochemical anabolic or catabolic pathway can be engineered into a cell, including, e.g., a polypeptide, enzyme, protein, e.g. structural or binding protein, activity.","Similarly, if it is determined that a particular oligonucleotide has no affect at all on the desired trait (e.g., a new a polypeptide, enzyme, protein, e.g. structural or binding protein, phenotype), it can be removed as a variable by synthesizing larger parental oligonucleotides that include the sequence to be removed. Since incorporating the sequence within a larger sequence prevents any crossover events, there will no longer be any variation of this sequence in the progeny polynucleotides. This iterative practice of determining which oligonucleotides are most related to the desired trait, and which are unrelated, allows more efficient exploration all of the possible protein variants that might be provide a particular trait or activity.","In Vivo Shuffling","In vivo shuffling of molecules is use in methods of the invention that provide variants of polypeptides of the invention, e.g., antibodies, a polypeptide, enzyme, protein, e.g. structural or binding protein, and the like. In vivo shuffling can be performed utilizing the natural property of cells to recombine multimers. While recombination in vivo has provided the major natural route to molecular diversity, genetic recombination remains a relatively complex process that involves 1) the recognition of homologies; 2) strand cleavage, strand invasion, and metabolic steps leading to the production of recombinant chiasma; and finally 3) the resolution of chiasma into discrete recombined molecules. The formation of the chiasma requires the recognition of homologous sequences.","In another aspect, the invention includes a method for producing a hybrid polynucleotide from at least a first polynucleotide and a second polynucleotide. The invention can be used to produce a hybrid polynucleotide by introducing at least a first polynucleotide and a second polynucleotide (e.g., one, or both, being an exemplary polypeptide-, enzyme-, protein-, e.g. structural or binding protein-encoding sequence of the invention) which share at least one region of partial sequence homology into a suitable host cell. The regions of partial sequence homology promote processes which result in sequence reorganization producing a hybrid polynucleotide. The term \u201chybrid polynucleotide\u201d, as used herein, is any nucleotide sequence which results from the method of the present invention and contains sequence from at least two original polynucleotide sequences. Such hybrid polynucleotides can result from intermolecular recombination events which promote sequence integration between DNA molecules. In addition, such hybrid polynucleotides can result from intramolecular reductive reassortment processes which utilize repeated sequences to alter a nucleotide sequence within a DNA molecule.","In vivo reassortment is focused on \u201cinter-molecular\u201d processes collectively referred to as \u201crecombination\u201d which in bacteria, is generally viewed as a \u201cRecA-dependent\u201d phenomenon. The invention can rely on recombination processes of a host cell to recombine and re-assort sequences, or the cells' ability to mediate reductive processes to decrease the complexity of quasi-repeated sequences in the cell by deletion. This process of \u201creductive reassortment\u201d occurs by an \u201cintra-molecular\u201d, RecA-independent process.","Therefore, in another aspect of the invention, novel polynucleotides can be generated by the process of reductive reassortment. The method involves the generation of constructs containing consecutive sequences (original encoding sequences), their insertion into an appropriate vector and their subsequent introduction into an appropriate host cell. The reassortment of the individual molecular identities occurs by combinatorial processes between the consecutive sequences in the construct possessing regions of homology, or between quasi-repeated units. The reassortment process recombines and\/or reduces the complexity and extent of the repeated sequences and results in the production of novel molecular species. Various treatments may be applied to enhance the rate of reassortment. These could include treatment with ultra-violet light, or DNA damaging chemicals and\/or the use of host cell lines displaying enhanced levels of \u201cgenetic instability\u201d. Thus the reassortment process may involve homologous recombination or the natural property of quasi-repeated sequences to direct their own evolution.","Repeated or \u201cquasi-repeated\u201d sequences play a role in genetic instability. In the present invention, \u201cquasi-repeats\u201d are repeats that are not restricted to their original unit structure. Quasi-repeated units can be presented as an array of sequences in a construct; consecutive units of similar sequences. Once ligated, the junctions between the consecutive sequences become essentially invisible and the quasi-repetitive nature of the resulting construct is now continuous at the molecular level. The deletion process the cell performs to reduce the complexity of the resulting construct operates between the quasi-repeated sequences. The quasi-repeated units provide a practically limitless repertoire of templates upon which slippage events can occur. The constructs containing the quasi-repeats thus effectively provide sufficient molecular elasticity that deletion (and potentially insertion) events can occur virtually anywhere within the quasi-repetitive units.","When the quasi-repeated sequences are all ligated in the same orientation, for instance head to tail or vice versa, the cell cannot distinguish individual units. Consequently, the reductive process can occur throughout the sequences. In contrast, when for example, the units are presented head to head, rather than head to tail, the inversion delineates the endpoints of the adjacent unit so that deletion formation will favor the loss of discrete units. Thus, it is preferable with the present method that the sequences are in the same orientation. Random orientation of quasi-repeated sequences will result in the loss of reassortment efficiency, while consistent orientation of the sequences will offer the highest efficiency. However, while having fewer of the contiguous sequences in the same orientation decreases the efficiency, it may still provide sufficient elasticity for the effective recovery of novel molecules. Constructs can be made with the quasi-repeated sequences in the same orientation to allow higher efficiency.","Sequences can be assembled in a head to tail orientation using any of a variety of methods, including the following:\n\n","The recovery of the re-assorted sequences relies on the identification of cloning vectors with a reduced repetitive index (RI). The re-assorted encoding sequences can then be recovered by amplification. The products are re-cloned and expressed. The recovery of cloning vectors with reduced RI can be affected by:\n\n","Encoding sequences (for example, genes) from related organisms may demonstrate a high degree of homology and encode quite diverse protein products. These types of sequences are particularly useful in the present invention as quasi-repeats. However, while the examples illustrated below demonstrate the reassortment of nearly identical original encoding sequences (quasi-repeats), this process is not limited to such nearly identical repeats.","The following example demonstrates a method of the invention. Encoding nucleic acid sequences (quasi-repeats) derived from three (3) unique species are described. Each sequence encodes a protein with a distinct set of properties. Each of the sequences differs by a single or a few base pairs at a unique position in the sequence. The quasi-repeated sequences are separately or collectively amplified and ligated into random assemblies such that all possible permutations and combinations are available in the population of ligated molecules. The number of quasi-repeat units can be controlled by the assembly conditions. The average number of quasi-repeated units in a construct is defined as the repetitive index (RI).","Once formed, the constructs may, or may not be size fractionated on an agarose gel according to published protocols, inserted into a cloning vector and transfected into an appropriate host cell. The cells are then propagated and \u201creductive reassortment\u201d is effected. The rate of the reductive reassortment process may be stimulated by the introduction of DNA damage if desired. Whether the reduction in RI is mediated by deletion formation between repeated sequences by an \u201cintra-molecular\u201d mechanism, or mediated by recombination-like events through \u201cinter-molecular\u201d mechanisms is immaterial. The end result is a reassortment of the molecules into all possible combinations.","Optionally, the method comprises the additional step of screening the library members of the shuffled pool to identify individual shuffled library members having the ability to bind or otherwise interact, or catalyze a particular reaction (e.g., such as catalytic domain of an enzyme) with a predetermined macromolecule, such as for example a proteinaceous receptor, an oligosaccharide, virion, or other predetermined compound or structure.","The polypeptides that are identified from such libraries can be used for therapeutic, diagnostic, research and related purposes (e.g., catalysts, solutes for increasing osmolarity of an aqueous solution and the like) and\/or can be subjected to one or more additional cycles of shuffling and\/or selection.","In another aspect, it is envisioned that prior to or during recombination or reassortment, polynucleotides generated by the method of the invention can be subjected to agents or processes which promote the introduction of mutations into the original polynucleotides. The introduction of such mutations would increase the diversity of resulting hybrid polynucleotides and polypeptides encoded therefrom. The agents or processes which promote mutagenesis can include, but are not limited to: (+)-CC-1065, or a synthetic analog such as (+)-CC-1065-(N3-Adenine (See Sun and Hurley, (1992); an N-acetylated or deacetylated 4\u2032-fluoro-4-aminobiphenyl adduct capable of inhibiting DNA synthesis (See, for example, van de Poll et al. (1992)); or a N-acetylated or deacetylated 4-aminobiphenyl adduct capable of inhibiting DNA synthesis (See also, van de Poll et al. (1992), pp. 751-758); trivalent chromium, a trivalent chromium salt, a polycyclic aromatic hydrocarbon (PAH) DNA adduct capable of inhibiting DNA replication, such as 7-bromomethyl-benz[a]anthracene (\u201cBMA\u201d), tris(2,3-dibromopropyl)phosphate (\u201cTris-BP\u201d), 1,2-dibromo-3-chloropropane (\u201cDBCP\u201d), 2-bromoacrolein (2BA), benzo[a]pyrene-7,8-dihydrodiol-9-10-epoxide (\u201cBPDE\u201d), a platinum(II) halogen salt, N-hydroxy-2-amino-3-methylimidazo[4,5-f]-quinoline (\u201cN-hydroxy-IQ\u201d) and N-hydroxy-2-amino-1-methyl-6-phenylimidazo[4,5-f]-pyridine (\u201cN-hydroxy-PhIP\u201d). Exemplary means for slowing or halting PCR amplification consist of UV light (+)-CC-1065 and (+)-CC-1065-(N3-Adenine). Particularly encompassed means are DNA adducts or polynucleotides comprising the DNA adducts from the polynucleotides or polynucleotides pool, which can be released or removed by a process including heating the solution comprising the polynucleotides prior to further processing.","In another aspect the invention is directed to a method of producing recombinant proteins having biological activity by treating a sample comprising double-stranded template polynucleotides encoding a wild-type protein under conditions according to the invention which provide for the production of hybrid or re-assorted polynucleotides.","Producing Sequence Variants","The invention also provides additional methods for making sequence variants of the nucleic acid (e.g., polypeptide, enzyme, protein, e.g. structural or binding protein) sequences of the invention. The invention also provides additional methods for isolating a polypeptide, enzyme, protein, e.g. structural or binding protein, using the nucleic acids and polypeptides of the invention. In one aspect, the invention provides for variants of a polypeptide, enzyme, protein, e.g. structural or binding protein, coding sequence (e.g., a gene, cDNA or message) of the invention, which can be altered by any means, including, e.g., random or stochastic methods, or, non-stochastic, or \u201cdirected evolution,\u201d methods, as described above.","The isolated variants may be naturally occurring. Variant can also be created in vitro. Variants may be created using genetic engineering techniques such as site directed mutagenesis, random chemical mutagenesis, Exonuclease III deletion procedures, and standard cloning techniques. Alternatively, such variants, fragments, analogs, or derivatives may be created using chemical synthesis or modification procedures. Other methods of making variants are also familiar to those skilled in the art. These include procedures in which nucleic acid sequences obtained from natural isolates are modified to generate nucleic acids which encode polypeptides having characteristics which enhance their value in industrial or laboratory applications. In such procedures, a large number of variant sequences having one or more nucleotide differences with respect to the sequence obtained from the natural isolate are generated and characterized. These nucleotide differences can result in amino acid changes with respect to the polypeptides encoded by the nucleic acids from the natural isolates.","For example, variants may be created using error prone PCR. In error prone PCR, PCR is performed under conditions where the copying fidelity of the DNA polymerase is low, such that a high rate of point mutations is obtained along the entire length of the PCR product. Error prone PCR is described, e.g., in Leung (1989) Technique 1:11-15) and Caldwell (1992) PCR Methods Applic. 2:28-33. Briefly, in such procedures, nucleic acids to be mutagenized are mixed with PCR primers, reaction buffer, MgCl, MnCl, Taq polymerase and an appropriate concentration of dNTPs for achieving a high rate of point mutation along the entire length of the PCR product. For example, the reaction may be performed using 20 (moles of nucleic acid to be mutagenized, 30 pmole of each PCR primer, a reaction buffer comprising 50 mM KCl, 10 mM Tris HCl (pH 8.3) and 0.01% gelatin, 7 mM MgCl2, 0.5 mM MnCl, 5 units of Taq polymerase, 0.2 mM dGTP, 0.2 mM dATP, 1 mM dCTP, and 1 mM dTTP. PCR may be performed for 30 cycles of 94\u00b0 C. for 1 min, 45\u00b0 C. for 1 min, and 72\u00b0 C. for 1 min. However, it will be appreciated that these parameters may be varied as appropriate. The mutagenized nucleic acids are cloned into an appropriate vector and the activities of the polypeptides encoded by the mutagenized nucleic acids are evaluated.","Variants may also be created using oligonucleotide directed mutagenesis to generate site-specific mutations in any cloned DNA of interest. Oligonucleotide mutagenesis is described, e.g., in Reidhaar-Olson (1988) Science 241:53-57. Briefly, in such procedures a plurality of double stranded oligonucleotides bearing one or more mutations to be introduced into the cloned DNA are synthesized and inserted into the cloned DNA to be mutagenized. Clones containing the mutagenized DNA are recovered and the activities of the polypeptides they encode are assessed.","Another method for generating variants is assembly PCR. Assembly PCR involves the assembly of a PCR product from a mixture of small DNA fragments. A large number of different PCR reactions occur in parallel in the same vial, with the products of one reaction priming the products of another reaction. Assembly PCR is described in, e.g., U.S. Pat. No. 5,965,408.","Still another method of generating variants is sexual PCR mutagenesis. In sexual PCR mutagenesis, forced homologous recombination occurs between DNA molecules of different but highly related DNA sequence in vitro, as a result of random fragmentation of the DNA molecule based on sequence homology, followed by fixation of the crossover by primer extension in a PCR reaction. Sexual PCR mutagenesis is described, e.g., in Stemmer (1994) Proc. Natl. Acad. Sci. USA 91:10747-10751. Briefly, in such procedures a plurality of nucleic acids to be recombined are digested with DNase to generate fragments having an average size of 50-200 nucleotides. Fragments of the desired average size are purified and resuspended in a PCR mixture. PCR is conducted under conditions which facilitate recombination between the nucleic acid fragments. For example, PCR may be performed by resuspending the purified fragments at a concentration of 10-30 ng\/\u03bcl in a solution of 0.2 mM of each dNTP, 2.2 mM MgCl, 50 mM KCL, 10 mM Tris HCl, pH 9.0, and 0.1% Triton X-100. 2.5 units of Taq polymerase per 100:1 of reaction mixture is added and PCR is performed using the following regime: 94\u00b0 C. for 60 seconds, 94\u00b0 C. for 30 seconds, 50-55\u00b0 C. for 30 seconds, 72\u00b0 C. for 30 seconds (30-45 times) and 72\u00b0 C. for 5 minutes. However, it will be appreciated that these parameters may be varied as appropriate. In some aspects, oligonucleotides may be included in the PCR reactions. In other aspects, the Klenow fragment of DNA polymerase I may be used in a first set of PCR reactions and Taq polymerase may be used in a subsequent set of PCR reactions. Recombinant sequences are isolated and the activities of the polypeptides they encode are assessed.","Variants may also be created by in vivo mutagenesis. In some aspects, random mutations in a sequence of interest are generated by propagating the sequence of interest in a bacterial strain, such as an strain, which carries mutations in one or more of the DNA repair pathways. Such \u201cmutator\u201d strains have a higher random mutation rate than that of a wild-type parent. Propagating the DNA in one of these strains will eventually generate random mutations within the DNA. Mutator strains suitable for use for in vivo mutagenesis are described in PCT Publication No. WO 91\/16427, published Oct. 31, 1991, entitled \u201cMethods for Phenotype Creation from Multiple Gene Populations\u201d.","Variants may also be generated using cassette mutagenesis. In cassette mutagenesis a small region of a double stranded DNA molecule is replaced with a synthetic oligonucleotide \u201ccassette\u201d that differs from the native sequence. The oligonucleotide often contains completely and\/or partially randomized native sequence.","Recursive ensemble mutagenesis may also be used to generate variants. Recursive ensemble mutagenesis is an algorithm for protein engineering (protein mutagenesis) developed to produce diverse populations of phenotypically related mutants whose members differ in amino acid sequence. This method uses a feedback mechanism to control successive rounds of combinatorial cassette mutagenesis. Recursive ensemble mutagenesis is described, e.g., in Arkin (1992) Proc. Natl. Acad. Sci. USA 89:7811-7815.","In some aspects, variants are created using exponential ensemble mutagenesis. Exponential ensemble mutagenesis is a process for generating combinatorial libraries with a high percentage of unique and functional mutants, wherein small groups of residues are randomized in parallel to identify, at each altered position, amino acids which lead to functional proteins. Exponential ensemble mutagenesis is described, e.g., in Delegrave (1993) Biotechnology Res. 11:1548-1552. Random and site-directed mutagenesis are described, e.g., in Arnold (1993) Current Opinion in Biotechnology 4:450-455.","In some aspects, the variants are created using shuffling procedures wherein portions of a plurality of nucleic acids which encode distinct polypeptides are fused together to create chimeric nucleic acid sequences which encode chimeric polypeptides as described in U.S. Pat. No. 5,965,408, filed Jul. 9, 1996, entitled, \u201cMethod of DNA Reassembly by Interrupting Synthesis\u201d and U.S. Pat. No. 5,939,250, filed May 22, 1996, entitled, \u201cProduction of Enzymes Having Desired Activities by Mutagenesis.","The variants of the polypeptides of the invention may be variants in which one or more of the amino acid residues of the polypeptides of the sequences of the invention are substituted with a conserved or non-conserved amino acid residue (in one aspect a conserved amino acid residue) and such substituted amino acid residue may or may not be one encoded by the genetic code.","Conservative substitutions are those that substitute a given amino acid in a polypeptide by another amino acid of like characteristics. Typically seen as conservative substitutions are the following replacements: replacements of an aliphatic amino acid such as Alanine, Valine, Leucine and Isoleucine with another aliphatic amino acid; replacement of a Serine with a Threonine or vice versa; replacement of an acidic residue such as Aspartic acid and Glutamic acid with another acidic residue; replacement of a residue bearing an amide group, such as Asparagine and Glutamine, with another residue bearing an amide group; exchange of a basic residue such as Lysine and Arginine with another basic residue; and replacement of an aromatic residue such as Phenylalanine, Tyrosine with another aromatic residue.","Other variants are those in which one or more of the amino acid residues of a polypeptide of the invention includes a substituent group.","Still other variants are those in which the polypeptide is associated with another compound, such as a compound to increase the half-life of the polypeptide (for example, polyethylene glycol).","Additional variants are those in which additional amino acids are fused to the polypeptide, such as a leader sequence, a secretory sequence, a proprotein sequence or a sequence which facilitates purification, enrichment, or stabilization of the polypeptide.","In some aspects, the fragments, derivatives and analogs retain the same biological function or activity as the polypeptides of the invention. In other aspects, the fragment, derivative, or analog includes a proprotein, such that the fragment, derivative, or analog can be activated by cleavage of the proprotein portion to produce an active polypeptide.","Optimizing Codons to Achieve High Levels of Protein Expression in Host Cells","The invention provides methods for modifying polypeptide-, enzyme-, protein-, e.g. structural or binding protein-encoding nucleic acids to modify codon usage. In one aspect, the invention provides methods for modifying codons in a nucleic acid encoding a polypeptide, enzyme, protein, e.g. structural or binding protein, to increase or decrease its expression in a host cell. The invention also provides nucleic acids encoding a polypeptide, enzyme, protein, e.g. structural or binding protein, modified to increase its expression in a host cell, a polypeptide, enzyme, protein, e.g. structural or binding protein, so modified, and methods of making the modified a polypeptide, enzyme, protein, e.g. structural or binding protein. The method comprises identifying a \u201cnon-preferred\u201d or a \u201cless preferred\u201d codon in a polypeptide-, enzyme-, protein-, e.g. structural or binding protein-encoding nucleic acid and replacing one or more of these non-preferred or less preferred codons with a \u201cpreferred codon\u201d encoding the same amino acid as the replaced codon and at least one non-preferred or less preferred codon in the nucleic acid has been replaced by a preferred codon encoding the same amino acid. A preferred codon is a codon over-represented in coding sequences in genes in the host cell and a non-preferred or less preferred codon is a codon under-represented in coding sequences in genes in the host cell.","Host cells for expressing the nucleic acids, expression cassettes and vectors of the invention include bacteria, yeast, fungi, plant cells, insect cells and mammalian cells. Thus, the invention provides methods for optimizing codon usage in all of these cells, codon-altered nucleic acids and polypeptides made by the codon-altered nucleic acids. Exemplary host cells include gram negative bacteria, such as ; gram positive bacteria, such as sp., . Exemplary host cells also include eukaryotic organisms, e.g., various yeast, such as sp., including , and , and mammalian cells and cell lines and insect cells and cell lines. Thus, the invention also includes nucleic acids and polypeptides optimized for expression in these organisms and species.","For example, the codons of a nucleic acid encoding a polypeptide, enzyme, protein, e.g. structural or binding protein, isolated from a bacterial cell are modified such that the nucleic acid is optimally expressed in a bacterial cell different from the bacteria from which the polypeptide, enzyme, protein, e.g. structural or binding protein was derived, a yeast, a fungi, a plant cell, an insect cell or a mammalian cell. Methods for optimizing codons are well known in the art, see, e.g., U.S. Pat. No. 5,795,737; Baca (2000) Int. J. Parasitol. 30:113-118; Hale (1998) Protein Expr. Purif. 12:185-188; Narum (2001) Infect. Immun. 69:7250-7253. See also Narum (2001) Infect. Immun. 69:7250-7253, describing optimizing codons in mouse systems; Outchkourov (2002) Protein Expr. Purif. 24:18-24, describing optimizing codons in yeast; Feng (2000) Biochemistry 39:15399-15409, describing optimizing codons in ; Humphreys (2000) Protein Expr. Purif. 20:252-264, describing optimizing codon usage that affects secretion in ","Transgenic Non-Human Animals","The invention provides transgenic non-human animals comprising a nucleic acid, a polypeptide (e.g., a polypeptide, enzyme, protein, e.g. structural or binding protein), an expression cassette or vector or a transfected or transformed cell of the invention. The invention also provides methods of making and using these transgenic non-human animals.","The transgenic non-human animals can be, e.g., goats, rabbits, sheep, pigs (including all swine, hogs and related animals), cows, rats and mice, comprising the nucleic acids of the invention. These animals can be used, e.g., as in vivo models to study a polypeptide, enzyme, protein, e.g. structural or binding protein, activity, or, as models to screen for agents that change the polypeptide, enzyme, protein, e.g. structural or binding protein activity in vivo. The coding sequences for the polypeptides to be expressed in the transgenic non-human animals can be designed to be constitutive, or, under the control of tissue-specific, developmental-specific or inducible transcriptional regulatory factors. Transgenic non-human animals can be designed and generated using any method known in the art; see, e.g., U.S. Pat. Nos. 6,211,428; 6,187,992; 6,156,952; 6,118,044; 6,111,166; 6,107,541; 5,959,171; 5,922,854; 5,892,070; 5,880,327; 5,891,698; 5,639,940; 5,573,933; 5,387,742; 5,087,571, describing making and using transformed cells and eggs and transgenic mice, rats, rabbits, sheep, pigs and cows. See also, e.g., Pollock (1999) J. Immunol. Methods 231:147-157, describing the production of recombinant proteins in the milk of transgenic dairy animals; Baguisi (1999) Nat. Biotechnol. 17:456-461, demonstrating the production of transgenic goats. U.S. Pat. No. 6,211,428, describes making and using transgenic non-human mammals which express in their brains a nucleic acid construct comprising a DNA sequence. U.S. Pat. No. 5,387,742, describes injecting cloned recombinant or synthetic DNA sequences into fertilized mouse eggs, implanting the injected eggs in pseudo-pregnant females, and growing to term transgenic mice. U.S. Pat. No. 6,187,992, describes making and using a transgenic mouse.","\u201cKnockout animals\u201d can also be used to practice the methods of the invention. For example, in one aspect, the transgenic or modified animals of the invention comprise a \u201cknockout animal,\u201d e.g., a \u201cknockout mouse,\u201d engineered not to express an endogenous gene, which is replaced with a gene expressing a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention, or, a fusion protein comprising a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention.","Transgenic Plants and Seeds","The invention provides transgenic plants and seeds comprising a nucleic acid, a polypeptide (e.g., a polypeptide, enzyme, protein, e.g. structural or binding protein), an expression cassette or vector or a transfected or transformed cell of the invention. The invention also provides plant products, e.g., oils, seeds, leaves, extracts and the like, comprising a nucleic acid and\/or a polypeptide (e.g., a polypeptide, enzyme, protein, e.g. structural or binding protein) of the invention. The transgenic plant can be dicotyledonous (a dicot) or monocotyledonous (a monocot). The invention also provides methods of making and using these transgenic plants and seeds. The transgenic plant or plant cell expressing a polypeptide of the present invention may be constructed in accordance with any method known in the art. See, for example, U.S. Pat. No. 6,309,872.","Nucleic acids and expression constructs of the invention can be introduced into a plant cell by any means. For example, nucleic acids or expression constructs can be introduced into the genome of a desired plant host, or, the nucleic acids or expression constructs can be episomes. Introduction into the genome of a desired plant can be such that the host's a polypeptide, enzyme, protein, e.g. structural or binding protein, production is regulated by endogenous transcriptional or translational control elements. The invention also provides \u201cknockout plants\u201d where insertion of gene sequence by, e.g., homologous recombination, has disrupted the expression of the endogenous gene. Means to generate \u201cknockout\u201d plants are well-known in the art, see, e.g., Strepp (1998) Proc Natl. Acad. Sci. USA 95:4368-4373; Miao (1995) Plant J 7:359-365. See discussion on transgenic plants, below.","The nucleic acids of the invention can be used to confer desired traits on essentially any plant, e.g., on starch-producing plants, such as potato, wheat, rice, barley, and the like. Nucleic acids of the invention can be used to manipulate metabolic pathways of a plant in order to optimize or alter host's expression of polypeptide, enzyme, protein, e.g. structural or binding protein. The can change a polypeptide, enzyme, protein, e.g. structural or binding protein, activity in a plant. Alternatively, a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention can be used in production of a transgenic plant to produce a compound not naturally produced by that plant. This can lower production costs or create a novel product.","In one aspect, the first step in production of a transgenic plant involves making an expression construct for expression in a plant cell. These techniques are well known in the art. They can include selecting and cloning a promoter, a coding sequence for facilitating efficient binding of ribosomes to mRNA and selecting the appropriate gene terminator sequences. One exemplary constitutive promoter is CaMV35S, from the cauliflower mosaic virus, which generally results in a high degree of expression in plants. Other promoters are more specific and respond to cues in the plant's internal or external environment. An exemplary light-inducible promoter is the promoter from the cab gene, encoding the major chlorophyll a\/b binding protein.","In one aspect, the nucleic acid is modified to achieve greater expression in a plant cell. For example, a sequence of the invention is likely to have a higher percentage of A-T nucleotide pairs compared to that seen in a plant, some of which prefer G-C nucleotide pairs. Therefore, A-T nucleotides in the coding sequence can be substituted with G-C nucleotides without significantly changing the amino acid sequence to enhance production of the gene product in plant cells.","Selectable marker gene can be added to the gene construct in order to identify plant cells or tissues that have successfully integrated the transgene. This may be necessary because achieving incorporation and expression of genes in plant cells is a rare event, occurring in just a few percent of the targeted tissues or cells. Selectable marker genes encode proteins that provide resistance to agents that are normally toxic to plants, such as antibiotics or herbicides. Only plant cells that have integrated the selectable marker gene will survive when grown on a medium containing the appropriate antibiotic or herbicide. As for other inserted genes, marker genes also require promoter and termination sequences for proper function.","In one aspect, making transgenic plants or seeds comprises incorporating sequences of the invention and, optionally, marker genes into a target expression construct (e.g., a plasmid), along with positioning of the promoter and the terminator sequences. This can involve transferring the modified gene into the plant through a suitable method. For example, a construct may be introduced directly into the genomic DNA of the plant cell using techniques such as electroporation and microinjection of plant cell protoplasts, or the constructs can be introduced directly to plant tissue using ballistic methods, such as DNA particle bombardment. For example, see, e.g., Christou (1997) Plant Mol. Biol. 35:197-203; Pawlowski (1996) Mol. Biotechnol. 6:17-30; Klein (1987) Nature 327:70-73; Takumi (1997) Genes Genet. Syst. 72:63-69, discussing use of particle bombardment to introduce transgenes into wheat; and Adam (1997) supra, for use of particle bombardment to introduce YACs into plant cells. For example, Rinehart (1997) supra, used particle bombardment to generate transgenic cotton plants. Apparatus for accelerating particles is described U.S. Pat. No. 5,015,580; and, the commercially available BioRad (Biolistics) PDS-2000 particle acceleration instrument; see also, John, U.S. Pat. No. 5,608,148; and Ellis, U.S. Pat. No. 5,681,730, describing particle-mediated transformation of gymnosperms.","In one aspect, protoplasts can be immobilized and injected with a nucleic acids, e.g., an expression construct. Although plant regeneration from protoplasts is not easy with cereals, plant regeneration is possible in legumes using somatic embryogenesis from protoplast derived callus. Organized tissues can be transformed with naked DNA using gene gun technique, where DNA is coated on tungsten microprojectiles, shot 1\/100th the size of cells, which carry the DNA deep into cells and organelles. Transformed tissue is then induced to regenerate, usually by somatic embryogenesis. This technique has been successful in several cereal species including maize and rice.","Nucleic acids, e.g., expression constructs, can also be introduced in to plant cells using recombinant viruses. Plant cells can be transformed using viral vectors, such as, e.g., tobacco mosaic virus derived vectors (Rouwendal (1997) Plant Mol. Biol. 33:989-999), see Porta (1996) \u201cUse of viral replicons for the expression of genes in plants,\u201d Mol. Biotechnol. 5:209-221.","Alternatively, nucleic acids, e.g., an expression construct, can be combined with suitable T-DNA flanking regions and introduced into a conventional host vector. The virulence functions of the host will direct the insertion of the construct and adjacent marker into the plant cell DNA when the cell is infected by the bacteria. -mediated transformation techniques, including disarming and use of binary vectors, are well described in the scientific literature. See, e.g., Horsch (1984) 233:496-498; Fraley (1983) 80:4803 (1983); , Potrykus, ed. (Springer-Verlag, Berlin 1995). The DNA in an cell is contained in the bacterial chromosome as well as in another structure known as a Ti (tumor-inducing) plasmid. The Ti plasmid contains a stretch of DNA termed T-DNA (\u02dc20 kb long) that is transferred to the plant cell in the infection process and a series of vir (virulence) genes that direct the infection process. can only infect a plant through wounds: when a plant root or stem is wounded it gives off certain chemical signals, in response to which, the vir genes of become activated and direct a series of events necessary for the transfer of the T-DNA from the Ti plasmid to the plant's chromosome. The T-DNA enters the plant cell through the wound. One speculation is that the T-DNA waits until the plant DNA is being replicated or transcribed, then inserts itself into the exposed plant DNA. In order to use as a transgene vector, the tumor-inducing section of T-DNA have to be removed, while retaining the T-DNA border regions and the vir genes. The transgene is then inserted between the T-DNA border regions, where it is transferred to the plant cell and becomes integrated into the plant's chromosomes.","The invention provides for the transformation of monocotyledonous plants using the nucleic acids of the invention, including important cereals, see Hiei (1997) Plant Mol. Biol. 35:205-218. See also, e.g., Horsch, Science (1984) 233:496; Fraley (1983) Proc. Natl. Acad. Sci USA 80:4803; Thykjaer (1997) supra; Park (1996) Plant Mol. Biol. 32:1135-1148, discussing T-DNA integration into genomic DNA. See also D'Halluin, U.S. Pat. No. 5,712,135, describing a process for the stable integration of a DNA comprising a gene that is functional in a cell of a cereal, or other monocotyledonous plant.","In one aspect, the third step can involve selection and regeneration of whole plants capable of transmitting the incorporated target gene to the next generation. Such regeneration techniques rely on manipulation of certain phytohormones in a tissue culture growth medium, typically relying on a biocide and\/or herbicide marker that has been introduced together with the desired nucleotide sequences. Plant regeneration from cultured protoplasts is described in Evans et al., , pp. 124-176, MacMillilan Publishing Company, New York, 1983; and Binding, , pp. 21-73, CRC Press, Boca Raton, 1985. Regeneration can also be obtained from plant callus, explants, organs, or parts thereof. Such regeneration techniques are described generally in Klee (1987) Ann. Rev. of Plant Phys. 38:467-486. To obtain whole plants from transgenic tissues such as immature embryos, they can be grown under controlled environmental conditions in a series of media containing nutrients and hormones, a process known as tissue culture. Once whole plants are generated and produce seed, evaluation of the progeny begins.","After the expression cassette is stably incorporated in transgenic plants, it can be introduced into other plants by sexual crossing. Any of a number of standard breeding techniques can be used, depending upon the species to be crossed. Since transgenic expression of the nucleic acids of the invention leads to phenotypic changes, plants comprising the recombinant nucleic acids of the invention can be sexually crossed with a second plant to obtain a final product. Thus, the seed of the invention can be derived from a cross between two transgenic plants of the invention, or a cross between a plant of the invention and another plant. The desired effects (e.g., expression of the polypeptides of the invention to produce a plant in which flowering behavior is altered) can be enhanced when both parental plants express the polypeptides (e.g., a polypeptide, enzyme, protein, e.g. structural or binding protein) of the invention. The desired effects can be passed to future plant generations by standard propagation means.","The nucleic acids and polypeptides of the invention are expressed in or inserted in any plant or seed. Transgenic plants of the invention can be dicotyledonous or monocotyledonous. Examples of monocot transgenic plants of the invention are grasses, such as meadow grass (blue grass, ), forage grass such as , temperate grass, such as , and cereals, e.g., wheat, oats, rye, barley, rice, sorghum, and maize (corn). Examples of dicot transgenic plants of the invention are tobacco, legumes, such as lupins, potato, sugar beet, pea, bean and soybean, and cruciferous plants (family Brassicaceae), such as cauliflower, rape seed, and the closely related model organism . Thus, the transgenic plants and seeds of the invention include a broad range of plants, including, but not limited to, species from the genera , and ","In alternative embodiments, the nucleic acids of the invention are expressed in plants which contain fiber cells, including, e.g., cotton, silk cotton tree (Kapok, Ceiba pentandra), desert willow, creosote bush, winterfat, balsa, ramie, kenaf, hemp, roselle, jute, sisal abaca and flax. In alternative embodiments, the transgenic plants of the invention can be members of the genus , including members of any species, such as , and ","The invention also provides for transgenic plants to be used for producing large amounts of the polypeptides (e.g., a polypeptide, enzyme, protein, e.g. structural or binding protein, or antibody) of the invention. For example, see Palmgren (1997) Trends Genet. 13:348; Chong (1997) Transgenic Res. 6:289-296 (producing human milk protein beta-casein in transgenic potato plants using an auxin-inducible, bidirectional mannopine synthase (mas1\u2032,2\u2032) promoter with -mediated leaf disc transformation methods).","Using known procedures, one of skill can screen for plants of the invention by detecting the increase or decrease of transgene mRNA or protein in transgenic plants. Means for detecting and quantitation of mRNAs or proteins are well known in the art.","Polypeptides and Peptides","In one aspect, the invention provides isolated or recombinant polypeptides having a sequence identity (e.g., at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more, or complete (100%) sequence identity, or homology) to an exemplary sequence of the invention, e.g., proteins having a sequence as set forth in SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, etc., and all polypeptides disclosed in the SEQ ID listing, which include all even numbered SEQ ID NO:s from SEQ ID NO:2 through SEQ ID NO:26,898). The percent sequence identity can be over the full length of the polypeptide, or, the identity can be over a region of at least about 50, 60, 70, 80, 90, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700 or more residues.","\u201cAmino acid\u201d or \u201camino acid sequence\u201d as used herein refer to an oligopeptide, peptide, polypeptide, or protein sequence, or to a fragment, portion, or subunit of any of these and to naturally occurring or synthetic molecules. \u201cAmino acid\u201d or \u201camino acid sequence\u201d include an oligopeptide, peptide, polypeptide, or protein sequence, or to a fragment, portion, or subunit of any of these, and to naturally occurring or synthetic molecules. The term \u201cpolypeptide\u201d as used herein, refers to amino acids joined to each other by peptide bonds or modified peptide bonds, i.e., peptide isosteres and may contain modified amino acids other than the 20 gene-encoded amino acids. The polypeptides may be modified by either natural processes, such as post-translational processing, or by chemical modification techniques which are well known in the art. Modifications can occur anywhere in the polypeptide, including the peptide backbone, the amino acid side-chains and the amino or carboxyl termini. It will be appreciated that the same type of modification may be present in the same or varying degrees at several sites in a given polypeptide. Also a given polypeptide may have many types of modifications. Modifications include acetylation, acylation, ADP-ribosylation, amidation, covalent attachment of flavin, covalent attachment of a heme moiety, covalent attachment of a nucleotide or nucleotide derivative, covalent attachment of a lipid or lipid derivative, covalent attachment of a phosphatidylinositol, cross-linking cyclization, disulfide bond formation, demethylation, formation of covalent cross-links, formation of cysteine, formation of pyroglutamate, formylation, gamma-carboxylation, glycosylation, GPI anchor formation, hydroxylation, iodination, methylation, myristolyation, oxidation, pegylation, glucan hydrolase processing, phosphorylation, prenylation, racemization, selenoylation, sulfation and transfer-RNA mediated addition of amino acids to protein such as arginylation. (See Creighton, T. E., Proteins\u2014Structure and Molecular Properties 2nd Ed., W.H. Freeman and Company, New York (1993); , B. C. Johnson, Ed., Academic Press, New York, pp. 1-12 (1983)). The peptides and polypeptides of the invention also include all \u201cmimetic\u201d and \u201cpeptidomimetic\u201d forms, as described in further detail, below.","As used herein, the term \u201cisolated\u201d means that the material is removed from its original environment (e.g., the natural environment if it is naturally occurring). For example, a naturally-occurring polynucleotide or polypeptide present in a living animal is not isolated, but the same polynucleotide or polypeptide, separated from some or all of the coexisting materials in the natural system, is isolated. Such polynucleotides could be part of a vector and\/or such polynucleotides or polypeptides could be part of a composition and still be isolated in that such vector or composition is not part of its natural environment. As used herein, the term \u201cpurified\u201d does not require absolute purity; rather, it is intended as a relative definition. Individual nucleic acids obtained from a library have been conventionally purified to electrophoretic homogeneity. The sequences obtained from these clones could not be obtained directly either from the library or from total human DNA. The purified nucleic acids of the invention have been purified from the remainder of the genomic DNA in the organism by at least 10-10fold. However, the term \u201cpurified\u201d also includes nucleic acids which have been purified from the remainder of the genomic DNA or from other sequences in a library or other environment by at least one order of magnitude, typically two or three orders and more typically four or five orders of magnitude.","\u201cRecombinant\u201d polypeptides or proteins refer to polypeptides or proteins produced by recombinant DNA techniques; i.e., produced from cells transformed by an exogenous DNA construct encoding the desired polypeptide or protein. \u201cSynthetic\u201d polypeptides or protein are those prepared by chemical synthesis. Solid-phase chemical peptide synthesis methods can also be used to synthesize the polypeptide or fragments of the invention. Such method have been known in the art since the early 1960's (Merrifield, R. B., 85:2149-2154, 1963) (See also Stewart, J. M. and Young, J. D., Solid Phase Peptide Synthesis, 2nd Ed., Pierce Chemical Co., Rockford, Ill., pp. 11-12)) and have recently been employed in commercially available laboratory peptide design and synthesis kits (Cambridge Research Biochemicals). Such commercially available laboratory kits have generally utilized the teachings of H. M. Geysen et al, 81:3998 (1984) and provide for synthesizing peptides upon the tips of a multitude of \u201crods\u201d or \u201cpins\u201d all of which are connected to a single plate.","Polypeptides of the invention can also be shorter than the full length of exemplary polypeptides. In alternative aspects, the invention provides polypeptides (peptides, fragments) ranging in size between about 5 and the full length of a polypeptide, e.g., an enzyme, such as a polypeptide, enzyme, protein, e.g. structural or binding protein; exemplary sizes being of about 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 100, 125, 150, 175, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, or more residues, e.g., contiguous residues of an exemplary a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention. Peptides of the invention (e.g., a subsequence of an exemplary polypeptide of the invention) can be useful as, e.g., labeling probes, antigens, toleragens, motifs, a polypeptide, enzyme, protein, e.g. structural or binding protein, active sites (e.g., \u201ccatalytic domains\u201d), signal sequences and\/or prepro domains.","In alternative aspects, polypeptides of the invention having enzyme, structural or binding activity are members of a genus of polypeptides sharing specific structural elements, e.g., amino acid residues, that correlate with enzyme, structural or binding activity. These shared structural elements can be used for the routine generation of polypeptide, enzyme, protein, e.g. structural or binding protein, variants. These shared structural elements of a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention can be used as guidance for the routine generation of a polypeptide, enzyme, protein, e.g. structural or binding protein, variants within the scope of the genus of polypeptides of the invention.","Polypeptides and peptides of the invention can be isolated from natural sources, be synthetic, or be recombinantly generated polypeptides. Peptides and proteins can be recombinantly expressed in vitro or in vivo. The peptides and polypeptides of the invention can be made and isolated using any method known in the art. Polypeptide and peptides of the invention can also be synthesized, whole or in part, using chemical methods well known in the art. See e.g., Caruthers (1980) Nucleic Acids Res. Symp. Ser. 215-223; Horn (1980) Nucleic Acids Res. Symp. Ser. 225-232; Banga, A. K., Therapeutic Peptides and Proteins, Formulation, Processing and Delivery Systems (1995) Technomic Publishing Co., Lancaster, Pa. For example, peptide synthesis can be performed using various solid-phase techniques (see e.g., Roberge (1995) Science 269:202; Merrifield (1997) Methods Enzymol. 289:3-13) and automated synthesis may be achieved, e.g., using the ABI 431A Peptide Synthesizer (Perkin Elmer) in accordance with the instructions provided by the manufacturer.","The peptides and polypeptides of the invention can also be glycosylated. The glycosylation can be added post-translationally either chemically or by cellular biosynthetic mechanisms, wherein the later incorporates the use of known glycosylation motifs, which can be native to the sequence or can be added as a peptide or added in the nucleic acid coding sequence. The glycosylation can be O-linked or N-linked.","The peptides and polypeptides of the invention, as defined above, include all \u201cmimetic\u201d and \u201cpeptidomimetic\u201d forms. The terms \u201cmimetic\u201d and \u201cpeptidomimetic\u201d refer to a synthetic chemical compound which has substantially the same structural and\/or functional characteristics of the polypeptides of the invention. The mimetic can be either entirely composed of synthetic, non-natural analogues of amino acids, or, is a chimeric molecule of partly natural peptide amino acids and partly non-natural analogs of amino acids. The mimetic can also incorporate any amount of natural amino acid conservative substitutions as long as such substitutions also do not substantially alter the mimetic's structure and\/or activity. As with polypeptides of the invention which are conservative variants or members of a genus of polypeptides of the invention (e.g., having about 50% or more sequence identity to an exemplary sequence of the invention), routine experimentation will determine whether a mimetic is within the scope of the invention, i.e., that its structure and\/or function is not substantially altered. Thus, in one aspect, a mimetic composition is within the scope of the invention if it has a polypeptide, enzyme, protein, e.g. structural or binding protein's activity.","Polypeptide mimetic compositions of the invention can contain any combination of non-natural structural components. In alternative aspect, mimetic compositions of the invention include one or all of the following three structural groups: a) residue linkage groups other than the natural amide bond (\u201cpeptide bond\u201d) linkages; b) non-natural residues in place of naturally occurring amino acid residues; or c) residues which induce secondary structural mimicry, i.e., to induce or stabilize a secondary structure, e.g., a beta turn, gamma turn, beta sheet, alpha helix conformation, and the like. For example, a polypeptide of the invention can be characterized as a mimetic when all or some of its residues are joined by chemical means other than natural peptide bonds. Individual peptidomimetic residues can be joined by peptide bonds, other chemical bonds or coupling means, such as, e.g., glutaraldehyde, N-hydroxysuccinimide esters, bifunctional maleimides, N,N\u2032-dicyclohexylcarbodiimide (DCC) or N,N\u2032-diisopropylcarbodiimide (DIC). Linking groups that can be an alternative to the traditional amide bond (\u201cpeptide bond\u201d) linkages include, e.g., ketomethylene (e.g., \u2014C(\u2550O)\u2014CH\u2014 for \u2014C(\u2550O)\u2014NH\u2014), aminomethylene (CH\u2014NH), ethylene, olefin (CH\u2550CH), ether (CH-0), thioether (CH\u2014S), tetrazole (CN\u2014), thiazole, retroamide, thioamide, or ester (see, e.g., Spatola (1983) in Chemistry and Biochemistry of Amino Acids, Peptides and Proteins, Vol. 7, pp 267-357, \u201cPeptide Backbone Modifications,\u201d Marcell Dekker, NY).","A polypeptide of the invention can also be characterized as a mimetic by containing all or some non-natural residues in place of naturally occurring amino acid residues. Non-natural residues are well described in the scientific and patent literature; a few exemplary non-natural compositions useful as mimetics of natural amino acid residues and guidelines are described below. Mimetics of aromatic amino acids can be generated by replacing by, e.g., D- or L-naphylalanine; D- or L-phenylglycine; D- or L-2 thieneylalanine; D- or L-1, -2, 3-, or 4-pyreneylalanine; D- or L-3 thieneylalanine; D- or L-(2-pyridinyl)-alanine; D- or L-(3-pyridinyl)-alanine; D- or L-(2-pyrazinyl)-alanine; D- or L-(4-isopropyl)-phenylglycine; D-(trifluoromethyl)-phenylglycine; D-(trifluoromethyl)-phenylalanine; D-p-fluoro-phenylalanine; D- or L-p-biphenylphenylalanine; D- or L-p-methoxy-biphenylphenylalanine; D- or L-2-indole(alkyl)alanines; and, D- or L-alkylainines, where alkyl can be substituted or unsubstituted methyl, ethyl, propyl, hexyl, butyl, pentyl, isopropyl, iso-butyl, sec-isotyl, iso-pentyl, or a non-acidic amino acids. Aromatic rings of a non-natural amino acid include, e.g., thiazolyl, thiophenyl, pyrazolyl, benzimidazolyl, naphthyl, furanyl, pyrrolyl, and pyridyl aromatic rings.","Mimetics of acidic amino acids can be generated by substitution by, e.g., non-carboxylate amino acids while maintaining a negative charge; (phosphono)alanine; sulfated threonine. Carboxyl side groups (e.g., aspartyl or glutamyl) can also be selectively modified by reaction with carbodiimides (R\u2032\u2014N\u2014C\u2014N\u2014R\u2032) such as, e.g., 1-cyclohexyl-3(2-morpholinyl-(4-ethyl)carbodiimide or 1-ethyl-3(4-azonia-4,4-dimetholpentyl)carbodiimide. Aspartyl or glutamyl can also be converted to asparaginyl and glutaminyl residues by reaction with ammonium ions. Mimetics of basic amino acids can be generated by substitution with, e.g., (in addition to lysine and arginine) the amino acids ornithine, citrulline, or (guanidino)-acetic acid, or (guanidino)alkyl-acetic acid, where alkyl is defined above. Nitrile derivative (e.g., containing the CN-moiety in place of COOH) can be substituted for asparagine or glutamine. Asparaginyl and glutaminyl residues can be deaminated to the corresponding aspartyl or glutamyl residues. Arginine residue mimetics can be generated by reacting arginyl with, e.g., one or more conventional reagents, including, e.g., phenylglyoxal, 2,3-butanedione, 1,2-cyclo-hexanedione, or ninhydrin, in one aspect under alkaline conditions. Tyrosine residue mimetics can be generated by reacting tyrosyl with, e.g., aromatic diazonium compounds or tetranitromethane. N-acetylimidizol and tetranitromethane can be used to form O-acetyl tyrosyl species and 3-nitro derivatives, respectively. Cysteine residue mimetics can be generated by reacting cysteinyl residues with, e.g., alpha-haloacetates such as 2-chloroacetic acid or chloroacetamide and corresponding amines; to give carboxymethyl or carboxyamidomethyl derivatives. Cysteine residue mimetics can also be generated by reacting cysteinyl residues with, e.g., bromo-trifluoroacetone, alpha-bromo-beta-(5-imidozoyl)propionic acid; chloroacetyl phosphate, N-alkylmaleimides, 3-nitro-2-pyridyl disulfide; methyl 2-pyridyl disulfide; p-chloromercuribenzoate; 2-chloromercuri-4 nitrophenol; or, chloro-7-nitrobenzo-oxa-1,3-diazole. Lysine mimetics can be generated (and amino terminal residues can be altered) by reacting lysinyl with, e.g., succinic or other carboxylic acid anhydrides. Lysine and other alpha-amino-containing residue mimetics can also be generated by reaction with imidoesters, such as methyl picolinimidate, pyridoxal phosphate, pyridoxal, chloroborohydride, trinitro-benzenesulfonic acid, O-methylisourea, 2,4, pentanedione, and transamidase-catalyzed reactions with glyoxylate. Mimetics of methionine can be generated by reaction with, e.g., methionine sulfoxide. Mimetics of proline include, e.g., pipecolic acid, thiazolidine carboxylic acid, 3- or 4-hydroxy proline, dehydroproline, 3- or 4-methylproline, or 3,3,-dimethylproline. Histidine residue mimetics can be generated by reacting histidyl with, e.g., diethylprocarbonate or para-bromophenacyl bromide. Other mimetics include, e.g., those generated by hydroxylation of proline and lysine; phosphorylation of the hydroxyl groups of seryl or threonyl residues; methylation of the alpha-amino groups of lysine, arginine and histidine; acetylation of the N-terminal amine; methylation of main chain amide residues or substitution with N-methyl amino acids; or amidation of C-terminal carboxyl groups.","A residue, e.g., an amino acid, of a polypeptide of the invention can also be replaced by an amino acid (or peptidomimetic residue) of the opposite chirality. Thus, any amino acid naturally occurring in the L-configuration (which can also be referred to as the R or S, depending upon the structure of the chemical entity) can be replaced with the amino acid of the same chemical structural type or a peptidomimetic, but of the opposite chirality, referred to as the D-amino acid, but also can be referred to as the R- or S-form.","The invention also provides methods for modifying the polypeptides of the invention by either natural processes, such as post-translational processing (e.g., phosphorylation, acylation, etc), or by chemical modification techniques, and the resulting modified polypeptides. Modifications can occur anywhere in the polypeptide, including the peptide backbone, the amino acid side-chains and the amino or carboxyl termini. It will be appreciated that the same type of modification may be present in the same or varying degrees at several sites in a given polypeptide. Also a given polypeptide may have many types of modifications. Modifications include acetylation, acylation, ADP-ribosylation, amidation, covalent attachment of flavin, covalent attachment of a heme moiety, covalent attachment of a nucleotide or nucleotide derivative, covalent attachment of a lipid or lipid derivative, covalent attachment of a phosphatidylinositol, cross-linking cyclization, disulfide bond formation, demethylation, formation of covalent cross-links, formation of cysteine, formation of pyroglutamate, formylation, gamma-carboxylation, glycosylation, GPI anchor formation, hydroxylation, iodination, methylation, myristolyation, oxidation, pegylation, proteolytic processing, phosphorylation, prenylation, racemization, selenoylation, sulfation, and transfer-RNA mediated addition of amino acids to protein such as arginylation. See, e.g., Creighton, T. E., Proteins\u2014Structure and Molecular Properties 2nd Ed., W.H. Freeman and Company, New York (1993); Posttranslational Covalent Modification of Proteins, B. C. Johnson, Ed., Academic Press, New York, pp. 1-12 (1983).","Solid-phase chemical peptide synthesis methods can also be used to synthesize the polypeptide or fragments of the invention. Such method have been known in the art since the early 1960's (Merrifield, R. B., J. Am. Chem. Soc., 85:2149-2154, 1963) (See also Stewart, J. M. and Young, J. D., Solid Phase Peptide Synthesis, 2nd Ed., Pierce Chemical Co., Rockford, Ill., pp. 11-12)) and have recently been employed in commercially available laboratory peptide design and synthesis kits (Cambridge Research Biochemicals). Such commercially available laboratory kits have generally utilized the teachings of H. M. Geysen et al, Proc. Natl. Acad. Sci., USA, 81:3998 (1984) and provide for synthesizing peptides upon the tips of a multitude of \u201crods\u201d or \u201cpins\u201d all of which are connected to a single plate. When such a system is utilized, a plate of rods or pins is inverted and inserted into a second plate of corresponding wells or reservoirs, which contain solutions for attaching or anchoring an appropriate amino acid to the pin's or rod's tips. By repeating such a process step, i.e., inverting and inserting the rod's and pin's tips into appropriate solutions, amino acids are built into desired peptides. In addition, a number of available FMOC peptide synthesis systems are available. For example, assembly of a polypeptide or fragment can be carried out on a solid support using an Applied Biosystems, Inc. Model 431A\u2122 automated peptide synthesizer. Such equipment provides ready access to the peptides of the invention, either by direct synthesis or by synthesis of a series of fragments that can be coupled using other known techniques.","The polypeptides of the invention include a polypeptide, enzyme, protein, e.g. structural or binding protein, in an active or inactive form. For example, the polypeptides of the invention include proproteins before \u201cmaturation\u201d or processing of prepro sequences, e.g., by a proprotein-processing enzyme, such as a proprotein convertase to generate an \u201cactive\u201d mature protein. The polypeptides of the invention include a polypeptide, enzyme, protein, e.g. structural or binding protein, inactive for other reasons, e.g., before \u201cactivation\u201d by a post-translational processing event, e.g., an endo- or exo-peptidase or proteinase action, a phosphorylation event, an amidation, a glycosylation or a sulfation, a dimerization event, and the like. The polypeptides of the invention include all active forms, including active subsequences, e.g., catalytic domains or active sites, of the enzyme.","The invention includes immobilized polypeptides, enzymes, proteins, e.g. structural or binding proteins, anti-polypeptides, anti-enzymes, anti-proteins, e.g. anti-structural or anti-binding proteins, antibodies and fragments thereof. The invention provides methods for inhibiting a polypeptide, enzyme, protein, e.g. structural or binding protein, activity, e.g., using dominant negative mutants or anti-polypeptide, anti-enzyme, anti-protein, e.g. anti-structural or anti-binding protein antibodies of the invention. The invention includes heterocomplexes, e.g., fusion proteins, heterodimers, etc., comprising the polypeptide, enzyme, protein, e.g. structural or binding proteins of the invention.","Polypeptides of the invention can have an enzyme, structural or binding activity under various conditions, e.g., extremes in pH and\/or temperature, oxidizing agents, and the like. The invention provides methods leading to alternative a polypeptide, enzyme, protein, e.g. structural or binding protein, preparations with different catalytic efficiencies and stabilities, e.g., towards temperature, oxidizing agents and changing wash conditions. In one aspect, a polypeptide, enzyme, protein, e.g. structural or binding protein, variants can be produced using techniques of site-directed mutagenesis and\/or random mutagenesis. In one aspect, directed evolution can be used to produce a great variety of a polypeptide, enzyme, protein, e.g. structural or binding protein, variants with alternative specificities and stability.","The proteins of the invention are also useful as research reagents to identify a polypeptide, enzyme, protein, e.g. structural or binding protein, modulators, e.g., activators or inhibitors of a polypeptide, enzyme, protein, e.g. structural or binding protein, activity. Briefly, test samples (compounds, broths, extracts, and the like) are added to a polypeptide, enzyme, protein, e.g. structural or binding protein, assays to determine their ability to inhibit substrate cleavage. Inhibitors identified in this way can be used in industry and research to reduce or prevent undesired proteolysis. As with a polypeptide, enzyme, protein, e.g. structural or binding protein, inhibitors can be combined to increase the spectrum of activity.","The enzymes of the invention are also useful as research reagents to digest proteins or in protein sequencing. For example, the polypeptide, enzyme, protein, e.g. structural or binding proteins may be used to break polypeptides into smaller fragments for sequencing using, e.g. an automated sequencer.","The invention also provides methods of discovering new a polypeptide, enzyme, protein, e.g. structural or binding protein, using the nucleic acids, polypeptides and antibodies of the invention. In one aspect, phagemid libraries are screened for expression-based discovery of a polypeptide, enzyme, protein, e.g. structural or binding protein. In another aspect, lambda phage libraries are screened for expression-based discovery of a polypeptide, enzyme, protein, e.g. structural or binding protein. Screening of the phage or phagemid libraries can allow the detection of toxic clones; improved access to substrate; reduced need for engineering a host, by-passing the potential for any bias resulting from mass excision of the library; and, faster growth at low clone densities. Screening of phage or phagemid libraries can be in liquid phase or in solid phase. In one aspect, the invention provides screening in liquid phase. This gives a greater flexibility in assay conditions; additional substrate flexibility; higher sensitivity for weak clones; and ease of automation over solid phase screening.","The invention provides screening methods using the proteins and nucleic acids of the invention and robotic automation to enable the execution of many thousands of biocatalytic reactions and screening assays in a short period of time, e.g., per day, as well as ensuring a high level of accuracy and reproducibility (see discussion of arrays, below). As a result, a library of derivative compounds can be produced in a matter of weeks. For further teachings on modification of molecules, including small molecules, see PCT\/US94\/09174.","In one aspect, polypeptides or fragments of the invention may be obtained through biochemical enrichment or purification procedures. The sequence of potentially homologous polypeptides or fragments may be determined by a polypeptide, enzyme, protein, e.g. structural or binding protein, assays, gel electrophoresis and\/or microsequencing. The sequence of the prospective polypeptide or fragment of the invention can be compared to an exemplary polypeptide of the invention, or a fragment, e.g., comprising at least about 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 or more consecutive amino acids thereof using any of the programs described above.","Another aspect of the invention is an assay for identifying fragments or variants of the invention, which retain the enzymatic function of the polypeptides of the invention. For example the fragments or variants of said polypeptides, may be used to catalyze biochemical reactions (e.g., production of a nootkatone from a valencene), which indicate that the fragment or variant retains the enzymatic activity of a polypeptide of the invention.","An exemplary assay for determining if fragments of variants retain the enzymatic activity of the polypeptides of the invention includes the steps of: contacting the polypeptide fragment or variant with a substrate molecule under conditions which allow the polypeptide fragment or variant to function and detecting either a decrease in the level of substrate or an increase in the level of the specific reaction product of the reaction between the polypeptide and substrate.","The present invention exploits the unique catalytic properties of enzymes. Whereas the use of biocatalysts (i.e., purified or crude enzymes, non-living or living cells) in chemical transformations normally requires the identification of a particular biocatalyst that reacts with a specific starting compound, the present invention uses selected biocatalysts and reaction conditions that are specific for functional groups that are present in many starting compounds, such as small molecules. Each biocatalyst is specific for one functional group, or several related functional groups and can react with many starting compounds containing this functional group.","The biocatalytic reactions produce a population of derivatives from a single starting compound. These derivatives can be subjected to another round of biocatalytic reactions to produce a second population of derivative compounds. Thousands of variations of the original small molecule or compound can be produced with each iteration of biocatalytic derivatization.","Enzymes react at specific sites of a starting compound without affecting the rest of the molecule, a process which is very difficult to achieve using traditional chemical methods. This high degree of biocatalytic specificity provides the means to identify a single active compound within the library. The library is characterized by the series of biocatalytic reactions used to produce it, a so-called \u201cbiosynthetic history\u201d. Screening the library for biological activities and tracing the biosynthetic history identifies the specific reaction sequence producing the active compound. The reaction sequence is repeated and the structure of the synthesized compound determined. This mode of identification, unlike other synthesis and screening approaches, does not require immobilization technologies and compounds can be synthesized and tested free in solution using virtually any type of screening assay. It is important to note, that the high degree of specificity of enzyme reactions on functional groups allows for the \u201ctracking\u201d of specific enzymatic reactions that make up the biocatalytically produced library.","Many of the procedural steps are performed using robotic automation enabling the execution of many thousands of biocatalytic reactions and screening assays per day as well as ensuring a high level of accuracy and reproducibility. As a result, a library of derivative compounds can be produced in a matter of weeks, which would take years to produce using current chemical methods.","In a particular aspect, the invention provides a method for modifying small molecules, comprising contacting a polypeptide encoded by a polynucleotide described herein or enzymatically active fragments thereof with a small molecule to produce a modified small molecule. A library of modified small molecules is tested to determine if a modified small molecule is present within the library, which exhibits a desired activity. A specific biocatalytic reaction which produces the modified small molecule of desired activity is identified by systematically eliminating each of the biocatalytic reactions used to produce a portion of the library and then testing the small molecules produced in the portion of the library for the presence or absence of the modified small molecule with the desired activity. The specific biocatalytic reactions which produce the modified small molecule of desired activity is optionally repeated. The biocatalytic reactions are conducted with a group of biocatalysts that react with distinct structural moieties found within the structure of a small molecule, each biocatalyst is specific for one structural moiety or a group of related structural moieties; and each biocatalyst reacts with many different small molecules which contain the distinct structural moiety.","A Polypeptide, Enzyme, Protein, e.g. Structural or Binding Protein, Signal Sequences, Prepro and Catalytic Domains","The invention provides a polypeptide, enzyme, protein, e.g. structural or binding protein, signal sequences (e.g., signal peptides (SPs)), prepro domains and catalytic domains (CDs). The SPs, prepro domains and\/or CDs of the invention can be isolated or recombinant peptides or can be part of a fusion protein, e.g., as a heterologous domain in a chimeric protein. The invention provides nucleic acids encoding these catalytic domains (CDs), prepro domains and signal sequences (SPs, e.g., a peptide having a sequence comprising\/consisting of amino terminal residues of a polypeptide of the invention).","The invention provides isolated or recombinant signal sequences (e.g., signal peptides) consisting of or comprising a sequence as set forth in residues 1 to 14, 1 to 15, 1 to 16, 1 to 17, 1 to 18, 1 to 19, 1 to 20, 1 to 21, 1 to 22, 1 to 23, 1 to 24, 1 to 25, 1 to 26, 1 to 27, 1 to 28, 1 to 28, 1 to 30, 1 to 31, 1 to 32, 1 to 33, 1 to 34, 1 to 35, 1 to 36, 1 to 37, 1 to 38, 1 to 40, 1 to 41, 1 to 42, 1 to 43, 1 to 44, 1 to 45, 1 to 46, or 1 to 47, or more, of a polypeptide of the invention, e.g., SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO:11, SEQ ID NO:13, SEQ ID NO:15, SEQ ID NO:17, SEQ ID NO:19, SEQ ID NO:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO:31, SEQ ID NO:33, SEQ ID NO:35, SEQ ID NO:37, SEQ ID NO:39, SEQ ID NO:41, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID NO:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID NO:61, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, SEQ ID NO:69, SEQ ID NO:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID NO:81, SEQ ID NO:83, SEQ ID NO:85, SEQ ID NO:87, SEQ ID NO:89, SEQ ID NO:91, SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO:99, SEQ ID NO:101, SEQ ID NO:103, SEQ ID NO:105, and all polypeptides disclosed in the SEQ ID listing, which include all odd numbered SEQ ID NO:s from SEQ ID NO:3 through SEQ ID NO:26,898. In one aspect, the invention provides signal sequences comprising the first 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70 or more amino terminal residues of a polypeptide of the invention.","The invention also provides isolated or recombinant signal sequences comprising\/consisting of the signal sequences set forth in Table 4, and polypeptides comprising these signal sequences. The polypeptide can be enzyme or protein of the invention. For example, reading Table 4, the invention provides an isolated or recombinant signal sequence as set forth by residues 1 to 16 of SEQ ID NO:10010. This can be determined by reading the second column for the first row, \u201cProbability: 0.992 AA1: 16 AA2: 17\u201d, wherein the cleavage of signal sequence takes place between amino acid 16 (AA16) and amino acid 17 (AA17), with a probability of 0.992 that this is the correct cleavage site. Therefore, the signal sequence is predicted to be from the amino acid in position 1 of SEQ ID NO:10010 up to and including the amino acid in position 16 of SEQ ID NO:10010. This signal sequence, in one aspect, is encoded by a subsequence of SEQ ID NO:10009.",{"@attributes":{"id":"p-0607","num":"0618"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"161pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"119pt","align":"left"}}],"thead":{"row":[{"entry":"TABLE\u20034"},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["SEQ\u2003ID\u2003NO:","Signalp\u2003Cleavage\u2003Site","Predicted\u2003Signal\u2003Sequence"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["10009,\u200310010","Probability:\u20030.992\u2003AA1:\u200316\u2003AA2:\u200317","MKSYFLLLLFLLPLFA"]},{"entry":{}},{"entry":["10111,\u200310112","Probability:\u20030.964\u2003AA1:\u200317\u2003AA2:\u200318","MKYIFIILVFLTTTLFA"]},{"entry":{}},{"entry":["\u20031013,\u20031014","Probability:\u20030.584\u2003AA1:\u200320\u2003AA2:\u200321","MKRVLLAIIGIILAIIVVVG"]},{"entry":{}},{"entry":["10147,\u200310148","Probability:\u20030.999\u2003AA1:\u200319\u2003AA2:\u200320","MNKILIFIIISLFSLNISA"]},{"entry":{}},{"entry":["10157,\u200310158","Probability:\u20030.941\u2003AA1:\u200319\u2003AA2:\u200320","MLKRIFILSLIAILICSNG"]},{"entry":{}},{"entry":["10217,\u200310218","Probability:\u20030.999\u2003AA1:\u200318\u2003AA2:\u200319","MKKISILIIFILSTLTLS"]},{"entry":{}},{"entry":["10309,\u200310310","Probability:\u20030.994\u2003AA1:\u200320\u2003AA2:\u200321","MRANLKKSYLIGLLLLFSLA"]},{"entry":{}},{"entry":["10327,\u200310328","Probability:\u20030.647\u2003AA1:\u200316\u2003AA2:\u200317","MRYLFSLFIFTTLIFA"]},{"entry":{}},{"entry":["10355,\u200310356","Probability:\u20030.592\u2003AA1:\u200319\u2003AA2:\u200320","MTKKVIVLSLIILLFINSS"]},{"entry":{}},{"entry":["10441,\u200310442","Probability:\u20030.683\u2003AA1:\u200317\u2003AA2:\u200318","MKRTFLTITAAAFILVG"]},{"entry":{}},{"entry":["10447,\u200310448","Probability:\u20030.928\u2003AA1:\u200317\u2003AA2:\u200318","MKNKLIILFIFSLFLLA"]},{"entry":{}},{"entry":["10525,\u200310526","Probability:\u20030.728\u2003AA1:\u200316\u2003AA2:\u200317","MRVLFFIFISLTTLFA"]},{"entry":{}},{"entry":["10537,\u200310538","Probability:\u20030.998\u2003AA1:\u200317\u2003AA2:\u200318","MKKIILLSTLLFLALNA"]},{"entry":{}},{"entry":["10543,\u200310544","Probability:\u20030.991\u2003AA1:\u200317\u2003AA2:\u200318","MKRKWFIFILTALVTIA"]},{"entry":{}},{"entry":["10591,\u200310592","Probability:\u20030.922\u2003AA1:\u200317\u2003AA2:\u200318","MFKLLIGIFIFISVAYS"]},{"entry":{}},{"entry":["10659,\u200310660","Probability:\u20030.967\u2003AA1:\u200320\u2003AA2:\u200321","MKDVIIIGAGGAGLSAGLSA"]},{"entry":{}},{"entry":["10673,\u200310674","Probability:\u20030.711\u2003AA1:\u200319\u2003AA2:\u200320","MKIWSTIKLVFISLVALVA"]},{"entry":{}},{"entry":["10711,\u200310712","Probability:\u20030.876\u2003AA1:\u200316\u2003AA2:\u200317","MMKGISPGAALVFLMA"]},{"entry":{}},{"entry":["10731,\u200310732","Probability:\u20030.997\u2003AA1:\u200319\u2003AA2:\u200320","MLKLLMITILLSTSGVANS"]},{"entry":{}},{"entry":["\u20031079,\u20031080","Probability:\u20030.929\u2003AA1:\u200317\u2003AA2:\u200318","MRIIKLFALFFLTCACN"]},{"entry":{}},{"entry":["10915,\u200310916","Probability:\u20030.934\u2003AA1:\u200317\u2003AA2:\u200318","MKSRLLLSGFFIFVLMS"]},{"entry":{}},{"entry":["11047,\u200311048","Probability:\u20030.530\u2003AA1:\u200316\u2003AA2:\u200317","MPEAAFSMSLPSKVFA"]},{"entry":{}},{"entry":["\u20031109,\u20031110","Probability:\u20030.777\u2003AA1:\u200318\u2003AA2:\u200319","MKVLLYILILFSGFKSFG"]},{"entry":{}},{"entry":["\u20031111,\u20031112","Probability:\u20030.765\u2003AA1:\u200318\u2003AA2:\u200319","MKVLLYILILFSGFKSFG"]},{"entry":{}},{"entry":["\u20031119,\u20031120","Probability:\u20030.870\u2003AA1:\u200318\u2003AA2:\u200319","MKKLFLILCIFFSVESFS"]},{"entry":{}},{"entry":["11209,\u200311210","Probability:\u20030.910\u2003AA1:\u200319\u2003AA2:\u200320","MKQIILLFSILFIVGKSYS"]},{"entry":{}},{"entry":["11253,\u200311254","Probability:\u20030.987\u2003AA1:\u200319\u2003AA2:\u200320","MKNIFFFSILLFLSFTGKA"]},{"entry":{}},{"entry":["11339,\u200311340","Probability:\u20030.510\u2003AA1:\u200319\u2003AA2:\u200320","MLKSISLFILITIVTGCSV"]},{"entry":{}},{"entry":["\u20031137,\u20031138","Probability:\u20030.992\u2003AA1:\u200318\u2003AA2:\u200319","MKILTIVFLVGFFCFVQA"]},{"entry":{}},{"entry":["11401,\u200311402","Probability:\u20030.992\u2003AA1:\u200319\u2003AA2:\u200320","MTISKNKLLIASLLSVAFT"]},{"entry":{}},{"entry":["11495,\u200311496","Probability:\u20030.647\u2003AA1:\u200316\u2003AA2:\u200317","MRYLFSLFIFTTLIFA"]},{"entry":{}},{"entry":["11719,\u200311720","Probability:\u20030.998\u2003AA1:\u200317\u2003AA2:\u200318","MKIILLIFFLLLSFSFA"]},{"entry":{}},{"entry":["11745,\u200311746","Probability:\u20030.972\u2003AA1:\u200318\u2003AA2:\u200319","MKYKIIFIAAFMAFSTLV"]},{"entry":{}},{"entry":["\u20031177,\u20031178","Probability:\u20030.995\u2003AA1:\u200320\u2003AA2:\u200321","MDQKKSLSLLFLIPAVSVIA"]},{"entry":{}},{"entry":["11821,\u200311822","Probability:\u20030.663\u2003AA1:\u200318\u2003AA2:\u200319","MSNKSVISTLIISIFFTA"]},{"entry":{}},{"entry":["11827,\u200311828","Probability:\u20030.727\u2003AA1:\u200319\u2003AA2:\u200320","MYVMKILLLISILFYCLLA"]},{"entry":{}},{"entry":["11935,\u200311936","Probability:\u20031.000\u2003AA1:\u200320\u2003AA2:\u200321","MKKTILIIASLFVAAFIGQA"]},{"entry":{}},{"entry":["11965,\u200311966","Probability:\u20030.999\u2003AA1:\u200319\u2003AA2:\u200320","MKKILVLSVLLTVCLISFA"]},{"entry":{}},{"entry":["12071,\u200312072","Probability:\u20030.773\u2003AA1:\u200319\u2003AA2:\u200320","MNKELLSFFSIFIALFVGA"]},{"entry":{}},{"entry":["12157,\u200312158","Probability:\u20030.983\u2003AA1:\u200316\u2003AA2:\u200317","MRLLLLLSLLVYTVFA"]},{"entry":{}},{"entry":["12377,\u200312378","Probability:\u20030.562\u2003AA1:\u200315\u2003AA2:\u200316","MASTTMIVSLIVAVA"]},{"entry":{}},{"entry":["12709,\u200312710","Probability:\u20030.993\u2003AA1:\u200318\u2003AA2:\u200319","MNNLKQILAIVMLLSVTA"]},{"entry":{}},{"entry":["13005,\u200313006","Probability:\u20030.977\u2003AA1:\u200320\u2003AA2:\u200321","MFLRRLSILILLLFVFFTAK"]},{"entry":{}},{"entry":["13017,\u200313018","Probability:\u20030.995\u2003AA1:\u200317\u2003AA2:\u200318","MFKNIIMSLLLCTFLSA"]},{"entry":{}},{"entry":["13139,\u200313140","Probability:\u20030.849\u2003AA1:\u200317\u2003AA2:\u200318","MRVVVLVLFSLLHFLFA"]},{"entry":{}},{"entry":["13307,\u200313308","Probability:\u20030.995\u2003AA1:\u200319\u2003AA2:\u200320","MKKLILLLILGFSTNLIFS"]},{"entry":{}},{"entry":["13347,\u200313348","Probability:\u20030.788\u2003AA1:\u200318\u2003AA2:\u200319","MLLILLICAVYSVGCALA"]},{"entry":{}},{"entry":["\u20031343,\u20031344","Probability:\u20030.708\u2003AA1:\u200318\u2003AA2:\u200319","MKSLIIIFSLILFFTACK"]},{"entry":{}},{"entry":["13475,\u200313476","Probability:\u20030.998\u2003AA1:\u200317\u2003AA2:\u200318","MKIILLIFFLLLSFSFA"]},{"entry":{}},{"entry":["13531,\u200313532","Probability:\u20030.651\u2003AA1:\u200317\u2003AA2:\u200318","MSHLLFSTSVLILLVVS"]},{"entry":{}},{"entry":["13543,\u200313544","Probability:\u20030.995\u2003AA1:\u200319\u2003AA2:\u200320","MKFILTTLMMAYLILPGMA"]},{"entry":{}},{"entry":["13603,\u200313604","Probability:\u20030.734\u2003AA1:\u200318\u2003AA2:\u200319","MNFKNILYSLLISGCLYG"]},{"entry":{}},{"entry":["13607,\u200313608","Probability:\u20030.840\u2003AA1:\u200319\u2003AA2:\u200320","MKKIILSLGVATLLLTTNL"]},{"entry":{}},{"entry":["13699,\u200313700","Probability:\u20030.544\u2003AA1:\u200321\u2003AA2:\u200322","MMKLHTLISLIFAVLMFIFCM"]},{"entry":{}},{"entry":["13711,\u200313712","Probability:\u20030.815\u2003AA1:\u200320\u2003AA2:\u200321","MSNKSVISTLIISIFFTACT"]},{"entry":{}},{"entry":["13719,\u200313720","Probability:\u20031.000\u2003AA1:\u200320\u2003AA2:\u200321","MKLTKIITVFMMVFSLSLMA"]},{"entry":{}},{"entry":["13777,\u200313778","Probability:\u20030.682\u2003AA1:\u200319\u2003AA2:\u200320","MKSMRTIFISFLIILLLQG"]},{"entry":{}},{"entry":["13829,\u200313830","Probability:\u20030.940\u2003AA1:\u200319\u2003AA2:\u200320","MKNLGLILLVLFLGLISTS"]},{"entry":{}},{"entry":["13891,\u200313892","Probability:\u20030.993\u2003AA1:\u200316\u2003AA2:\u200317","MKYFLLLLLIITTLNA"]},{"entry":{}},{"entry":["13915,\u200313916","Probability:\u20031.000\u2003AA1:\u200320\u2003AA2:\u200321","MKKFFLALFLTSIVTISIAA"]},{"entry":{}},{"entry":["13933,\u200313934","Probability:\u20030.962\u2003AA1:\u200319\u2003AA2:\u200320","MFMNKKVYISLITALVVNA"]},{"entry":{}},{"entry":["14081,\u200314082","Probability:\u20030.918\u2003AA1:\u200318\u2003AA2:\u200319","MTYLFLAIAIGLITAASK"]},{"entry":{}},{"entry":["14133,\u200314134","Probability:\u20030.989\u2003AA1:\u200320\u2003AA2:\u200321","MNNLIKLILLITLSFSSLLS"]},{"entry":{}},{"entry":["14197,\u200314198","Probability:\u20030.995\u2003AA1:\u200318\u2003AA2:\u200319","MKKITLILFAIFTALSMS"]},{"entry":{}},{"entry":["14267,\u200314268","Probability:\u20030.815\u2003AA1:\u200320\u2003AA2:\u200321","MSNKSVISTLIISIFFTACT"]},{"entry":{}},{"entry":["14369,\u200314370","Probability:\u20030.669\u2003AA1:\u200317\u2003AA2:\u200318","MKKYIIIFCIFSGFLYG"]},{"entry":{}},{"entry":["14505,\u200314506","Probability:\u20030.951\u2003AA1:\u200320\u2003AA2:\u200321","MIRFGSSSSSILYFFRNTMA"]},{"entry":{}},{"entry":["14573,\u200314574","Probability:\u20030.992\u2003AA1:\u200319\u2003AA2:\u200320","MLRWFILLISVIVCLNVNA"]},{"entry":{}},{"entry":["\u20031461,\u20031462","Probability:\u20030.908\u2003AA1:\u200319\u2003AA2:\u200320","MKKFLIFCLFLFLNKPLIS"]},{"entry":{}},{"entry":["14655,\u200314656","Probability:\u20030.773\u2003AA1:\u200322\u2003AA2:\u200323","MAQAVAISIAFFSVLLSLLLFN"]},{"entry":{}},{"entry":["14705,\u200314706","Probability:\u20030.599\u2003AA1:\u200321\u2003AA2:\u200322","MGGLIAIIILSSRTVAPLGQA"]},{"entry":{}},{"entry":["14835,\u200314836","Probability:\u20030.999\u2003AA1:\u200317\u2003AA2:\u200318","MVKKLLFLALAFSISFA"]},{"entry":{}},{"entry":["14857,\u200314858","Probability:\u20031.000\u2003AA1:\u200321\u2003AA2:\u200322","MIRQKIVLTMLLFCFSLITVA"]},{"entry":{}},{"entry":["14863,\u200314864","Probability:\u20030.990\u2003AA1:\u200317\u2003AA2:\u200318","MRKYFLVLLLFCTSLLS"]},{"entry":{}},{"entry":["15045,\u200315046","Probability:\u20030.984\u2003AA1:\u200321\u2003AA2:\u200322","MKNIILSTLAFVLALFFSGCT"]},{"entry":{}},{"entry":["15049,\u200315050","Probability:\u20030.845\u2003AA1:\u200319\u2003AA2:\u200320","MNFFIMPFLLMFLFIGIFA"]},{"entry":{}},{"entry":["15055,\u200315056","Probability:\u20030.669\u2003AA1:\u200315\u2003AA2:\u200316","MKFNLNSFLMSVSLA"]},{"entry":{}},{"entry":["15111,\u200315112","Probability:\u20030.835\u2003AA1:\u200317\u2003AA2:\u200318","MIKRLFSIVLSLGLVFN"]},{"entry":{}},{"entry":["15135,\u200315136","Probability:\u20030.853\u2003AA1:\u200315\u2003AA2:\u200316","MKYLLALCIFLLLTG"]},{"entry":{}},{"entry":["15173,\u200315174","Probability:\u20030.513\u2003AA1:\u200319\u2003AA2:\u200320","MKKLNVAIYIVIVILSLFS"]},{"entry":{}},{"entry":["15179,\u200315180","Probability:\u20030.645\u2003AA1:\u200316\u2003AA2:\u200317","MRYLFSLFIFTTLIFA"]},{"entry":{}},{"entry":["15201,\u200315202","Probability:\u20030.883\u2003AA1:\u200320\u2003AA2:\u200321","MKLLGIGSILLQVLLCSVSA"]},{"entry":{}},{"entry":["15235,\u200315236","Probability:\u20030.792\u2003AA1:\u200319\u2003AA2:\u200320","MNFKQLFLSVLLILTIVLS"]},{"entry":{}},{"entry":["15251,\u200315252","Probability:\u20030.998\u2003AA1:\u200317\u2003AA2:\u200318","MKIILLIFFLLLSFSFA"]},{"entry":{}},{"entry":["\u2003\u2003153,\u2003154","Probability:\u20030.824\u2003AA1:\u200320\u2003AA2:\u200321","MIKTIXSLARCIIAFGILNA"]},{"entry":{}},{"entry":["15329,\u200315330","Probability:\u20030.557\u2003AA1:\u200320\u2003AA2:\u200321","MKNIYKIILLSLLIISIILG"]},{"entry":{}},{"entry":["\u20031541,\u20031542","Probability:\u20031.000\u2003AA1:\u200319\u2003AA2:\u200320","MKRNSLLLVLLALSLFTAA"]},{"entry":{}},{"entry":["15473,\u200315474","Probability:\u20030.934\u2003AA1:\u200319\u2003AA2:\u200320","MRGTICSILILSFIFLITA"]},{"entry":{}},{"entry":["15475,\u200315476","Probability:\u20030.934\u2003AA1:\u200320\u2003AA2:\u200321","MAAGDFFAIFGIFMSLSLLA"]},{"entry":{}},{"entry":["15495,\u200315496","Probability:\u20030.645\u2003AA1:\u200316\u2003AA2:\u200317","MRYLFSLFIFTTLIFA"]},{"entry":{}},{"entry":["15521,\u200315522","Probability:\u20030.972\u2003AA1:\u200318\u2003AA2:\u200319","MIKVSIYIVLLLTSYIHA"]},{"entry":{}},{"entry":["15585,\u200315586","Probability:\u20030.993\u2003AA1:\u200316\u2003AA2:\u200317","MKLLLLLFLVLLNVNA"]},{"entry":{}},{"entry":["15589,\u200315590","Probability:\u20030.967\u2003AA1:\u200317\u2003AA2:\u200318","MNKKILILMIILGLAVA"]},{"entry":{}},{"entry":["15623,\u200315624","Probability:\u20030.553\u2003AA1:\u200318\u2003AA2:\u200319","MSSRVFLTSFLIIVPLTA"]},{"entry":{}},{"entry":["15635,\u200315636","Probability:\u20031.000\u2003AA1:\u200319\u2003AA2:\u200320","MKNILSIALAVLMIGSLHS"]},{"entry":{}},{"entry":["15659,\u200315660","Probability:\u20031.000\u2003AA1:\u200320\u2003AA2:\u200321","MYKFITALISLFLLTTHSYA"]},{"entry":{}},{"entry":["15697,\u200315698","Probability:\u20030.561\u2003AA1:\u200318\u2003AA2:\u200319","MISIKTAIAIILVIVATN"]},{"entry":{}},{"entry":["15765,\u200315766","Probability:\u20030.936\u2003AA1:\u200318\u2003AA2:\u200319","MKFHKSLLLLLLLSFIVS"]},{"entry":{}},{"entry":["15783,\u200315784","Probability:\u20030.951\u2003AA1:\u200320\u2003AA2:\u200321","MKIAVLGAGISGLGSAYLLS"]},{"entry":{}},{"entry":["\u20031585,\u20031586","Probability:\u20030.668\u2003AA1:\u200319\u2003AA2:\u200320","MMFFTSISIXSXFPXIXLX"]},{"entry":{}},{"entry":["15855,\u200315856","Probability:\u20030.677\u2003AA1:\u200318\u2003AA2:\u200319","MKKLKLILGSVLSIVAFT"]},{"entry":{}},{"entry":["15873,\u200315874","Probability:\u20030.784\u2003AA1:\u200316\u2003AA2:\u200317","MIFFFIFVILFTFSVA"]},{"entry":{}},{"entry":["15907,\u200315908","Probability:\u20030.998\u2003AA1:\u200320\u2003AA2:\u200321","MSLKKYIFILTFLFISNLFA"]},{"entry":{}},{"entry":["15909,\u200315910","Probability:\u20030.935\u2003AA1:\u200320\u2003AA2:\u200321","MKQKLLKITLLTTLLTSAIA"]},{"entry":{}},{"entry":["16005,\u200316006","Probability:\u20030.932\u2003AA1:\u200320\u2003AA2:\u200321","MLKNLKNILFFLFFLIFCLN"]},{"entry":{}},{"entry":["16015,\u200316016","Probability:\u20030.541\u2003AA1:\u200316\u2003AA2:\u200317","MIIIAISALIATTIIA"]},{"entry":{}},{"entry":["16171,\u200316172","Probability:\u20030.985\u2003AA1:\u200320\u2003AA2:\u200321","MKLNLGKIFLLLIFPIITFA"]},{"entry":{}},{"entry":["16175,\u200316176","Probability:\u20030.957\u2003AA1:\u200317\u2003AA2:\u200318","MMKTFIVFCVMSISIFA"]},{"entry":{}},{"entry":["16183,\u200316184","Probability:\u20030.999\u2003AA1:\u200320\u2003AA2:\u200321","MKLISKILLILAIITSGVLS"]},{"entry":{}},{"entry":["16237,\u200316238","Probability:\u20030.792\u2003AA1:\u200319\u2003AA2:\u200320","MNFKQLFLSVLLILTIVLS"]},{"entry":{}},{"entry":["16289,\u200316290","Probability:\u20030.995\u2003AA1:\u200316\u2003AA2:\u200317","MRISILLAVVSSIIFA"]},{"entry":{}},{"entry":["\u2003\u2003163,\u2003164","Probability:\u20030.860\u2003AA1:\u200320\u2003AA2:\u200321","MQINRLIVLLLIMISHKNFA"]},{"entry":{}},{"entry":["\u20031633,\u20031634","Probability:\u20030.993\u2003AA1:\u200319\u2003AA2:\u200320","MKIYVILALLIFSSRSIYS"]},{"entry":{}},{"entry":["16339,\u200316340","Probability:\u20031.000\u2003AA1:\u200318\u2003AA2:\u200319","MKKLLLIYILLLSTITFA"]},{"entry":{}},{"entry":["16345,\u200316346","Probability:\u20030.776\u2003AA1:\u200319\u2003AA2:\u200320","MGNIKVILVFISLFLIAIT"]},{"entry":{}},{"entry":["16373,\u200316374","Probability:\u20030.995\u2003AA1:\u200316\u2003AA2:\u200317","MRISILLAVVSSIIFA"]},{"entry":{}},{"entry":["\u20031641,\u20031642","Probability:\u20030.879\u2003AA1:\u200318\u2003AA2:\u200319","MKKFILFLGFFYLISFFA"]},{"entry":{}},{"entry":["16455,\u200316456","Probability:\u20030.890\u2003AA1:\u200319\u2003AA2:\u200320","MKKFNIKLIIIFISSLFLA"]},{"entry":{}},{"entry":["16467,\u200316468","Probability:\u20030.681\u2003AA1:\u200320\u2003AA2:\u200321","MERRLFLKGATILASSAVIA"]},{"entry":{}},{"entry":["\u20031647,\u20031648","Probability:\u20030.812\u2003AA1:\u200320\u2003AA2:\u200321","MRLKLSLLILLLFSGINGIA"]},{"entry":{}},{"entry":["16487,\u200316488","Probability:\u20030.987\u2003AA1:\u200319\u2003AA2:\u200320","MRIFNYLIMSILLSVTLMA"]},{"entry":{}},{"entry":["\u20031669,\u20031670","Probability:\u20030.999\u2003AA1:\u200318\u2003AA2:\u200319","MRATFIVLSVLLTSSVMS"]},{"entry":{}},{"entry":["16711,\u200316712","Probability:\u20030.626\u2003AA1:\u200317\u2003AA2:\u200318","MFKTILFTFILITNIFS"]},{"entry":{}},{"entry":["16747,\u200316748","Probability:\u20030.628\u2003AA1:\u200319\u2003AA2:\u200320","MKNIFFLFIAVLILSNCKN"]},{"entry":{}},{"entry":["16825,\u200316826","Probability:\u20030.975\u2003AA1:\u200318\u2003AA2:\u200319","MFKKALLVFYIFLGITMA"]},{"entry":{}},{"entry":["16833,\u200316834","Probability:\u20030.857\u2003AA1:\u200320\u2003AA2:\u200321","MNNKTKIFLPILLAMAIVLG"]},{"entry":{}},{"entry":["16885,\u200316886","Probability:\u20030.993\u2003AA1:\u200316\u2003AA2:\u200317","MKLLLLLFLVLLNVNA"]},{"entry":{}},{"entry":["16967,\u200316968","Probability:\u20030.888\u2003AA1:\u200320\u2003AA2:\u200321","MKPTKLLFGLFILIFTFTTS"]},{"entry":{}},{"entry":["17035,\u200317036","Probability:\u20030.977\u2003AA1:\u200316\u2003AA2:\u200317","MMKKYIIALISTFLYA"]},{"entry":{}},{"entry":["17065,\u200317066","Probability:\u20030.982\u2003AA1:\u200317\u2003AA2:\u200318","MKHFLLCSVLLLGVLDA"]},{"entry":{}},{"entry":["\u2003\u2003171,\u2003172","Probability:\u20030.956\u2003AA1:\u200321\u2003AA2:\u200322","MKRIIYIILLFSVAVILSSCT"]},{"entry":{}},{"entry":["17157,\u200317158","Probability:\u20030.952\u2003AA1:\u200316\u2003AA2:\u200317","MKILLIVILFISSLFS"]},{"entry":{}},{"entry":["17331,\u200317332","Probability:\u20030.981\u2003AA1:\u200317\u2003AA2:\u200318","MLKKLLILTFITTISFA"]},{"entry":{}},{"entry":["17347,\u200317348","Probability:\u20030.999\u2003AA1:\u200316\u2003AA2:\u200317","MSKIIILILSFLIANA"]},{"entry":{}},{"entry":["17353,\u200317354","Probability:\u20030.993\u2003AA1:\u200320\u2003AA2:\u200321","MKLKYLLIIIIITLGQFVIA"]},{"entry":{}},{"entry":["17359,\u200317360","Probability:\u20030.932\u2003AA1:\u200319\u2003AA2:\u200320","MKIKHFILLFLFSLIALYS"]},{"entry":{}},{"entry":["17367,\u200317368","Probability:\u20030.912\u2003AA1:\u200320\u2003AA2:\u200321","MKKSKILFLLLTLLIIMGIG"]},{"entry":{}},{"entry":["\u20031749,\u20031750","Probability:\u20030.990\u2003AA1:\u200318\u2003AA2:\u200319","MNRIFLIVVLFISSTCFS"]},{"entry":{}},{"entry":["17537,\u200317538","Probability:\u20030.999\u2003AA1:\u200317\u2003AA2:\u200318","MKFFFILLILFMFNALS"]},{"entry":{}},{"entry":["17547,\u200317548","Probability:\u20030.959\u2003AA1:\u200319\u2003AA2:\u200320","MKNIITIYLFMLMSLFLLS"]},{"entry":{}},{"entry":["\u20031771,\u20031772","Probability:\u20030.931\u2003AA1:\u200320\u2003AA2:\u200321","MVMKSILGIVSFLIGLSLIA"]},{"entry":{}},{"entry":["17751,\u200317752","Probability:\u20030.561\u2003AA1:\u200318\u2003AA2:\u200319","MKYLLILILLVFTGCNNV"]},{"entry":{}},{"entry":["17783,\u200317784","Probability:\u20030.987\u2003AA1:\u200319\u2003AA2:\u200320","MTKIKVVGLLVLILSIALA"]},{"entry":{}},{"entry":["\u20031785,\u20031786","Probability:\u20030.716\u2003AA1:\u200317\u2003AA2:\u200318","MKLLSATFFMVVFSVIS"]},{"entry":{}},{"entry":["17915,\u200317916","Probability:\u20030.898\u2003AA1:\u200317\u2003AA2:\u200318","MVKIFLSIILFVNIVFA"]},{"entry":{}},{"entry":["18019,\u200318020","Probability:\u20030.993\u2003AA1:\u200318\u2003AA2:\u200319","MKKITFLLILFVTTFSFS"]},{"entry":{}},{"entry":["18039,\u200318040","Probability:\u20030.867\u2003AA1:\u200319\u2003AA2:\u200320","MQKVILTLVCIITSFFFQA"]},{"entry":{}},{"entry":["18057,\u200318058","Probability:\u20030.874\u2003AA1:\u200319\u2003AA2:\u200320","MRFLFVLFTFLIFSCSKNS"]},{"entry":{}},{"entry":["18131,\u200318132","Probability:\u20031.000\u2003AA1:\u200319\u2003AA2:\u200320","MKKTQIILLLILLSMASHA"]},{"entry":{}},{"entry":["18237,\u200318238","Probability:\u20030.975\u2003AA1:\u200318\u2003AA2:\u200319","MKKVLIFYCVLFSLQGFS"]},{"entry":{}},{"entry":["18249,\u200318250","Probability:\u20030.719\u2003AA1:\u200318\u2003AA2:\u200319","MKTKTLLTVLTILFSLQS"]},{"entry":{}},{"entry":["18329,\u200318330","Probability:\u20030.988\u2003AA1:\u200317\u2003AA2:\u200318","MSKLAVLFLFLFLACNN"]},{"entry":{}},{"entry":["18377,\u200318378","Probability:\u20030.983\u2003AA1:\u200318\u2003AA2:\u200319","MKKARIIILSFFIGMVAA"]},{"entry":{}},{"entry":["18403,\u200318404","Probability:\u20031.000\u2003AA1:\u200319\u2003AA2:\u200320","MKKTILVLICLFSISALFA"]},{"entry":{}},{"entry":["18435,\u200318436","Probability:\u20030.611\u2003AA1:\u200319\u2003AA2:\u200320","MKIGFILILSIAICTSCKV"]},{"entry":{}},{"entry":["18489,\u200318490","Probability:\u20030.914\u2003AA1:\u200317\u2003AA2:\u200318","MKKLTYLFLSITLLSFG"]},{"entry":{}},{"entry":["18495,\u200318496","Probability:\u20030.627\u2003AA1:\u200319\u2003AA2:\u200320","MKNSIAFLFLSLLIFTGCK"]},{"entry":{}},{"entry":["18507,\u200318508","Probability:\u20030.783\u2003AA1:\u200320\u2003AA2:\u200321","MKKIYLILASTIVLASCGNK"]},{"entry":{}},{"entry":["\u20031851,\u20031852","Probability:\u20030.998\u2003AA1:\u200318\u2003AA2:\u200319","MKKFLAIFLFFIAFHGNA"]},{"entry":{}},{"entry":["18529,\u200318530","Probability:\u20030.999\u2003AA1:\u200319\u2003AA2:\u200320","MKKNTILLFGIVLVFAAYG"]},{"entry":{}},{"entry":["18587,\u200318588","Probability:\u20030.956\u2003AA1:\u200319\u2003AA2:\u200320","MSFFKPSFVLFFCLLGLHG"]},{"entry":{}},{"entry":["18607,\u200318608","Probability:\u20030.636\u2003AA1:\u200318\u2003AA2:\u200319","MHGLHSIFSLLFLCTLSA"]},{"entry":{}},{"entry":["18713,\u200318714","Probability:\u20031.000\u2003AA1:\u200318\u2003AA2:\u200319","MKKTLLLFLFLSTLVVQA"]},{"entry":{}},{"entry":["18747,\u200318748","Probability:\u20030.985\u2003AA1:\u200318\u2003AA2:\u200319","MKKIIYIFVILLSVSVSG"]},{"entry":{}},{"entry":["18825,\u200318826","Probability:\u20030.999\u2003AA1:\u200318\u2003AA2:\u200319","MKFILLAFFTLISNYALS"]},{"entry":{}},{"entry":["18841,\u200318842","Probability:\u20031.000\u2003AA1:\u200319\u2003AA2:\u200320","MLKSAIFLVLLSLVGSAFG"]},{"entry":{}},{"entry":["18919,\u200318920","Probability:\u20030.995\u2003AA1:\u200320\u2003AA2:\u200321","MHIFLKSFILFIFLSFILQA"]},{"entry":{}},{"entry":["18921,\u200318922","Probability:\u20030.998\u2003AA1:\u200318\u2003AA2:\u200319","MKNLIVLIFVVLTQLSVA"]},{"entry":{}},{"entry":["18957,\u200318958","Probability:\u20030.719\u2003AA1:\u200318\u2003AA2:\u200319","MKTKTLLTVLTILFSLQS"]},{"entry":{}},{"entry":["18971,\u200318972","Probability:\u20030.775\u2003AA1:\u200317\u2003AA2:\u200318","MKKLIALFFLCILISCN"]},{"entry":{}},{"entry":["18977,\u200318978","Probability:\u20031.000\u2003AA1:\u200317\u2003AA2:\u200318","MKKLFLLILMLPFSLLA"]},{"entry":{}},{"entry":["19079,\u200319080","Probability:\u20030.895\u2003AA1:\u200319\u2003AA2:\u200320","MKFFSALLILTLVITSCKS"]},{"entry":{}},{"entry":["19143,\u200319144","Probability:\u20030.610\u2003AA1:\u200318\u2003AA2:\u200319","MKKNQLSFLLLVFLISNT"]},{"entry":{}},{"entry":["\u20031919,\u20031920","Probability:\u20030.996\u2003AA1:\u200319\u2003AA2:\u200320","MKTLIFFLILFFGSIPSYS"]},{"entry":{}},{"entry":["19211,\u200319212","Probability:\u20030.517\u2003AA1:\u200319\u2003AA2:\u200320","MFNKKQAITLFSGLLFCFT"]},{"entry":{}},{"entry":["19265,\u200319266","Probability:\u20031.000\u2003AA1:\u200319\u2003AA2:\u200320","MKKTILVLICLFSISALFA"]},{"entry":{}},{"entry":["19277,\u200319278","Probability:\u20030.945\u2003AA1:\u200319\u2003AA2:\u200320","MKFTSYLFFLFFIFMNCTA"]},{"entry":{}},{"entry":["19385,\u200319386","Probability:\u20030.835\u2003AA1:\u200318\u2003AA2:\u200319","MKKIIPYILSCMLLSLAA"]},{"entry":{}},{"entry":["19391,\u200319392","Probability:\u20030.999\u2003AA1:\u200319\u2003AA2:\u200320","MKSFLTILFSLFLTGSINS"]},{"entry":{}},{"entry":["19421,\u200319422","Probability:\u20030.998\u2003AA1:\u200320\u2003AA2:\u200321","MRKHQLLILIILMLTSVSNS"]},{"entry":{}},{"entry":["19467,\u200319468","Probability:\u20030.994\u2003AA1:\u200320\u2003AA2:\u200321","MSNYLLSVILLAFILASCST"]},{"entry":{}},{"entry":["19475,\u200319476","Probability:\u20030.998\u2003AA1:\u200319\u2003AA2:\u200320","MKIRFYLSALFACLLFASS"]},{"entry":{}},{"entry":["19527,\u200319528","Probability:\u20030.998\u2003AA1:\u200317\u2003AA2:\u200318","MKKILLLLPLISILSYA"]},{"entry":{}},{"entry":["19535,\u200319536","Probability:\u20030.596\u2003AA1:\u200320\u2003AA2:\u200321","MKSIKKLFIFIFLIILVVYL"]},{"entry":{}},{"entry":["\u20031955,\u20031956","Probability:\u20031.000\u2003AA1:\u200319\u2003AA2:\u200320","MRKIYGLLAFCLLMNTAKA"]},{"entry":{}},{"entry":["19563,\u200319564","Probability:\u20030.940\u2003AA1:\u200320\u2003AA2:\u200321","MKNSIAFLFLSLLIFTGCKS"]},{"entry":{}},{"entry":["19657,\u200319658","Probability:\u20030.994\u2003AA1:\u200320\u2003AA2:\u200321","MIAKKITLLILFVFNQYVIA"]},{"entry":{}},{"entry":["19733,\u200319734","Probability:\u20030.938\u2003AA1:\u200320\u2003AA2:\u200321","MNITIIGSGYVGLVSGTCFA"]},{"entry":{}},{"entry":["19785,\u200319786","Probability:\u20030.936\u2003AA1:\u200319\u2003AA2:\u200320","MRTLSFLIVTFSVLISGCA"]},{"entry":{}},{"entry":["19797,\u200319798","Probability:\u20031.000\u2003AA1:\u200318\u2003AA2:\u200319","MKTALFILFCTLGQMSLA"]},{"entry":{}},{"entry":["19915,\u200319916","Probability:\u20030.999\u2003AA1:\u200317\u2003AA2:\u200318","MKKLLILLFLFNSNLFA"]},{"entry":{}},{"entry":["19919,\u200319920","Probability:\u20030.676\u2003AA1:\u200320\u2003AA2:\u200321","MRISVFFYISLFIISSTKFS"]},{"entry":{}},{"entry":["19953,\u200319954","Probability:\u20031.000\u2003AA1:\u200319\u2003AA2:\u200320","MKKFLLLFVLPLLAISGFA"]},{"entry":{}},{"entry":["19993,\u200319994","Probability:\u20030.575\u2003AA1:\u200320\u2003AA2:\u200321","MRTTLKRIAFLFFFSFLIFS"]},{"entry":{}},{"entry":["20001,\u200320002","Probability:\u20030.674\u2003AA1:\u200318\u2003AA2:\u200319","MKKIIFLLVTLIIFNSCK"]},{"entry":{}},{"entry":["\u20032003,\u20032004","Probability:\u20031.000\u2003AA1:\u200318\u2003AA2:\u200319","MKVRYLLAFLIISSASLA"]},{"entry":{}},{"entry":["20069,\u200320070","Probability:\u20031.000\u2003AA1:\u200320\u2003AA2:\u200321","MKIKIRCLLIFLTLSPFVQA"]},{"entry":{}},{"entry":["20083,\u200320084","Probability:\u20030.637\u2003AA1:\u200320\u2003AA2:\u200321","MILGLLLFALIAIVFLFLLK"]},{"entry":{}},{"entry":["20111,\u200320112","Probability:\u20031.000\u2003AA1:\u200319\u2003AA2:\u200320","MRKILFTCFLLILSLFSYA"]},{"entry":{}},{"entry":["20143,\u200320144","Probability:\u20030.946\u2003AA1:\u200317\u2003AA2:\u200318","MKKFIFSVLTLLLVGLS"]},{"entry":{}},{"entry":["20161,\u200320162","Probability:\u20030.602\u2003AA1:\u200319\u2003AA2:\u200320","MKSIIYMAALIILSVKGTS"]},{"entry":{}},{"entry":["20187,\u200320188","Probability:\u20030.585\u2003AA1:\u200320\u2003AA2:\u200321","MKPMRNLLFLIVLIANTSLL"]},{"entry":{}},{"entry":["20225,\u200320226","Probability:\u20031.000\u2003AA1:\u200320\u2003AA2:\u200321","MLQKTILVLLFLLTTSNSFS"]},{"entry":{}},{"entry":["20229,\u200320230","Probability:\u20030.522\u2003AA1:\u200320\u2003AA2:\u200321","MKFIINPILILLAVIILSLN"]},{"entry":{}},{"entry":["20253,\u200320254","Probability:\u20030.906\u2003AA1:\u200320\u2003AA2:\u200321","MKTILRLTICGLILLNASLA"]},{"entry":{}},{"entry":["\u2003\u2003203,\u2003204","Probability:\u20030.824\u2003AA1:\u200319\u2003AA2:\u200320","MENKMKRLLTIFVFLIVFS"]},{"entry":{}},{"entry":["20301,\u200320302","Probability:\u20030.675\u2003AA1:\u200319\u2003AA2:\u200320","MIRIISFLLPLLFFLNCQK"]},{"entry":{}},{"entry":["20309,\u200320310","Probability:\u20030.989\u2003AA1:\u200319\u2003AA2:\u200320","MKMKFTFLLLIISTISIYG"]},{"entry":{}},{"entry":["20349,\u200320350","Probability:\u20030.967\u2003AA1:\u200319\u2003AA2:\u200320","MKKHLLFFLTTILSISGYS"]},{"entry":{}},{"entry":["20371,\u200320372","Probability:\u20030.605\u2003AA1:\u200318\u2003AA2:\u200319","MNKLISIVLVCITLMSTG"]},{"entry":{}},{"entry":["\u20032045,\u20032046","Probability:\u20030.906\u2003AA1:\u200320\u2003AA2:\u200321","MKKHLFITALLMLTVINYTG"]},{"entry":{}},{"entry":["20485,\u200320486","Probability:\u20030.951\u2003AA1:\u200317\u2003AA2:\u200318","MKNLFLFLTILSLSCMQ"]},{"entry":{}},{"entry":["20487,\u200320488","Probability:\u20030.951\u2003AA1:\u200317\u2003AA2:\u200318","MKNLFLFLTILSLSCMQ"]},{"entry":{}},{"entry":["20531,\u200320532","Probability:\u20030.895\u2003AA1:\u200319\u2003AA2:\u200320","MFSFLGVILIAFFGFIGFS"]},{"entry":{}},{"entry":["20569,\u200320570","Probability:\u20030.819\u2003AA1:\u200319\u2003AA2:\u200320","MKILLRISLLLITTISCTS"]},{"entry":{}},{"entry":["20571,\u200320572","Probability:\u20030.819\u2003AA1:\u200319\u2003AA2:\u200320","MKILLRISLLLITTISCTS"]},{"entry":{}},{"entry":["20665,\u200320666","Probability:\u20030.989\u2003AA1:\u200318\u2003AA2:\u200319","MRKKIFLVLIILMSLSLG"]},{"entry":{}},{"entry":["\u20032071,\u20032072","Probability:\u20030.994\u2003AA1:\u200319\u2003AA2:\u200320","MKKIVSILVFVLLANLSQA"]},{"entry":{}},{"entry":["20735,\u200320736","Probability:\u20031.000\u2003AA1:\u200318\u2003AA2:\u200319","MKLFVTLSVLFFCVFATA"]},{"entry":{}},{"entry":["20743,\u200320744","Probability:\u20030.998\u2003AA1:\u200319\u2003AA2:\u200320","MKNVIVLFSVASMAFSCFS"]},{"entry":{}},{"entry":["20805,\u200320806","Probability:\u20030.962\u2003AA1:\u200319\u2003AA2:\u200320","MMLKKSILLSIAALFISSA"]},{"entry":{}},{"entry":["20881,\u200320882","Probability:\u20030.574\u2003AA1:\u200318\u2003AA2:\u200319","MRVFILIISLFFTLFSCQ"]},{"entry":{}},{"entry":["\u20032093,\u20032094","Probability:\u20030.641\u2003AA1:\u200318\u2003AA2:\u200319","MNFIIFCFVLLLSGFLTG"]},{"entry":{}},{"entry":["21115,\u200321116","Probability:\u20030.990\u2003AA1:\u200319\u2003AA2:\u200320","MMKLYLFVFFFFIVAAGYA"]},{"entry":{}},{"entry":["21121,\u200321122","Probability:\u20030.680\u2003AA1:\u200317\u2003AA2:\u200318","MKPIYLIFSLLTFISLS"]},{"entry":{}},{"entry":["21133,\u200321134","Probability:\u20030.998\u2003AA1:\u200319\u2003AA2:\u200320","MKRTLVIMVFLMTISQIQA"]},{"entry":{}},{"entry":["\u20032115,\u20032116","Probability:\u20030.699\u2003AA1:\u200320\u2003AA2:\u200321","MKRHNIIYFAAILFACNGNT"]},{"entry":{}},{"entry":["\u20032117,\u20032118","Probability:\u20030.999\u2003AA1:\u200319\u2003AA2:\u200320","MNKLFLSIVLILTCCQLSA"]},{"entry":{}},{"entry":["21307,\u200321308","Probability:\u20030.814\u2003AA1:\u200316\u2003AA2:\u200317","MRKIILSILGVLFIIA"]},{"entry":{}},{"entry":["21361,\u200321362","Probability:\u20030.856\u2003AA1:\u200319\u2003AA2:\u200320","MKKSLLFFTIIFCILFLSQ"]},{"entry":{}},{"entry":["21363,\u200321364","Probability:\u20030.856\u2003AA1:\u200319\u2003AA2:\u200320","MKKSLLFFTIIFCILFLSQ"]},{"entry":{}},{"entry":["21395,\u200321396","Probability:\u20030.698\u2003AA1:\u200319\u2003AA2:\u200320","MMKRVIVGLSGGVDSSVAA"]},{"entry":{}},{"entry":["21427,\u200321428","Probability:\u20030.971\u2003AA1:\u200319\u2003AA2:\u200320","MRNSVIISLIFVMISNQLS"]},{"entry":{}},{"entry":["21563,\u200321564","Probability:\u20030.984\u2003AA1:\u200318\u2003AA2:\u200319","MIRLITILLLIISTNIYS"]},{"entry":{}},{"entry":["21595,\u200321596","Probability:\u20030.871\u2003AA1:\u200318\u2003AA2:\u200319","MNYSKLVAIIFFIAMLSA"]},{"entry":{}},{"entry":["21755,\u200321756","Probability:\u20030.998\u2003AA1:\u200318\u2003AA2:\u200319","MKNILLLTIALSFVNLFA"]},{"entry":{}},{"entry":["21773,\u200321774","Probability:\u20030.895\u2003AA1:\u200319\u2003AA2:\u200320","MKKKKKKNXSGVSLSFDSA"]},{"entry":{}},{"entry":["21777,\u200321778","Probability:\u20030.845\u2003AA1:\u200319\u2003AA2:\u200320","MKYFVFFFSIILCSCSSSE"]},{"entry":{}},{"entry":["21845,\u200321846","Probability:\u20030.940\u2003AA1:\u200319\u2003AA2:\u200320","MKMILHTLFILLISIPLYA"]},{"entry":{}},{"entry":["21851,\u200321852","Probability:\u20030.976\u2003AA1:\u200318\u2003AA2:\u200319","MKKFFKFLGILLVVLIVA"]},{"entry":{}},{"entry":["21863,\u200321864","Probability:\u20030.818\u2003AA1:\u200316\u2003AA2:\u200317","MKALFFIIIFVAIAVA"]},{"entry":{}},{"entry":["21939,\u200321940","Probability:\u20030.993\u2003AA1:\u200318\u2003AA2:\u200319","MKGIFFYLLLFVSALSFS"]},{"entry":{}},{"entry":["21987,\u200321988","Probability:\u20030.875\u2003AA1:\u200316\u2003AA2:\u200317","MKKIMLVLGILIVVFM"]},{"entry":{}},{"entry":["22293,\u200322294","Probability:\u20030.615\u2003AA1:\u200320\u2003AA2:\u200321","MRKSFKIILFSILGILLLLL"]},{"entry":{}},{"entry":["22337,\u200322338","Probability:\u20031.000\u2003AA1:\u200319\u2003AA2:\u200320","MKKIITLVALVFFSVSTFA"]},{"entry":{}},{"entry":["22363,\u200322364","Probability:\u20030.997\u2003AA1:\u200318\u2003AA2:\u200319","MKRLIMVIFLFLGVQSIA"]},{"entry":{}},{"entry":["22455,\u200322456","Probability:\u20031.000\u2003AA1:\u200318\u2003AA2:\u200319","MKTVLLLLSILVSSYSLA"]},{"entry":{}},{"entry":["22467,\u200322468","Probability:\u20030.546\u2003AA1:\u200317\u2003AA2:\u200318","MARMRISVLFFMFCVFA"]},{"entry":{}},{"entry":["22491,\u200322492","Probability:\u20031.000\u2003AA1:\u200318\u2003AA2:\u200319","MKKKLFLLLLLVTSATFA"]},{"entry":{}},{"entry":["22509,\u200322510","Probability:\u20031.000\u2003AA1:\u200318\u2003AA2:\u200319","MKKTLFFIAILLTFNSNA"]},{"entry":{}},{"entry":["22513,\u200322514","Probability:\u20030.542\u2003AA1:\u200313\u2003AA2:\u200314","MKKILTGVILTLA"]},{"entry":{}},{"entry":["22657,\u200322658","Probability:\u20031.000\u2003AA1:\u200319\u2003AA2:\u200320","MTKNLILLILVLIFNIGFA"]},{"entry":{}},{"entry":["22701,\u200322702","Probability:\u20030.970\u2003AA1:\u200317\u2003AA2:\u200318","MKKIIALLVLVSLFSCG"]},{"entry":{}},{"entry":["22703,\u200322704","Probability:\u20030.984\u2003AA1:\u200316\u2003AA2:\u200317","MIYKKIVFLVTLLAFA"]},{"entry":{}},{"entry":["22767,\u200322768","Probability:\u20030.988\u2003AA1:\u200314\u2003AA2:\u200315","MRLILLFTLCLAMA"]},{"entry":{}},{"entry":["22803,\u200322804","Probability:\u20030.575\u2003AA1:\u200319\u2003AA2:\u200320","MKKTTILIAIFLLIGLKIN"]},{"entry":{}},{"entry":["\u20032281,\u20032282","Probability:\u20030.999\u2003AA1:\u200318\u2003AA2:\u200319","MKLFISLSLLFISIFVIA"]},{"entry":{}},{"entry":["22817,\u200322818","Probability:\u20030.934\u2003AA1:\u200318\u2003AA2:\u200319","MKKSIIISAIVLGFSLVS"]},{"entry":{}},{"entry":["22903,\u200322904","Probability:\u20030.996\u2003AA1:\u200320\u2003AA2:\u200321","MKKLFTLLIMSLVLSSCTDA"]},{"entry":{}},{"entry":["22953,\u200322954","Probability:\u20030.593\u2003AA1:\u200318\u2003AA2:\u200319","MTKILMVCLGNICRSPLA"]},{"entry":{}},{"entry":["22993,\u200322994","Probability:\u20031.000\u2003AA1:\u200319\u2003AA2:\u200320","MKNLVLSLFLIAISISAFS"]},{"entry":{}},{"entry":["23055,\u200323056","Probability:\u20030.757\u2003AA1:\u200318\u2003AA2:\u200319","MKSIKFILVLFITVSIFS"]},{"entry":{}},{"entry":["23107,\u200323108","Probability:\u20030.782\u2003AA1:\u200318\u2003AA2:\u200319","MKKLFITFTTLLLLIACK"]},{"entry":{}},{"entry":["23167,\u200323168","Probability:\u20030.816\u2003AA1:\u200318\u2003AA2:\u200319","MKKFLLIVCIGLLMFSFT"]},{"entry":{}},{"entry":["23175,\u200323176","Probability:\u20030.631\u2003AA1:\u200318\u2003AA2:\u200319","MNRLLILLFSIFLISCYN"]},{"entry":{}},{"entry":["\u20032321,\u20032322","Probability:\u20031.000\u2003AA1:\u200320\u2003AA2:\u200321","MFRKTILGGLAIIAALTINA"]},{"entry":{}},{"entry":["23253,\u200323254","Probability:\u20030.930\u2003AA1:\u200319\u2003AA2:\u200320","MKKIIFVLLIVLLFASCSK"]},{"entry":{}},{"entry":["23277,\u200323278","Probability:\u20030.814\u2003AA1:\u200316\u2003AA2:\u200317","MRKIILSILGVLFIIA"]},{"entry":{}},{"entry":["23279,\u200323280","Probability:\u20030.771\u2003AA1:\u200318\u2003AA2:\u200319","MKQYLVITFLLSLTLGFS"]},{"entry":{}},{"entry":["23323,\u200323324","Probability:\u20030.536\u2003AA1:\u200319\u2003AA2:\u200320","MKYYISILITVLITCNNHS"]},{"entry":{}},{"entry":["23339,\u200323340","Probability:\u20030.821\u2003AA1:\u200318\u2003AA2:\u200319","MMKKISLLILIIIGSCQT"]},{"entry":{}},{"entry":["\u20032339,\u20032340","Probability:\u20031.000\u2003AA1:\u200318\u2003AA2:\u200319","MKKIFLSFCLLSSFIGFA"]},{"entry":{}},{"entry":["23415,\u200323416","Probability:\u20030.882\u2003AA1:\u200319\u2003AA2:\u200320","MIKSIIYLLFLIFSLTIIA"]},{"entry":{}},{"entry":["23431,\u200323432","Probability:\u20030.926\u2003AA1:\u200316\u2003AA2:\u200317","MKNTILLSVILLLLFS"]},{"entry":{}},{"entry":["23549,\u200323550","Probability:\u20030.516\u2003AA1:\u200317\u2003AA2:\u200318","MNKLAVILLVVIVAFLS"]},{"entry":{}},{"entry":["23559,\u200323560","Probability:\u20030.998\u2003AA1:\u200320\u2003AA2:\u200321","MKKLFCILSILILFGCGSTS"]},{"entry":{}},{"entry":["23587,\u200323588","Probability:\u20030.862\u2003AA1:\u200318\u2003AA2:\u200319","MKKICFLFFCFMTYIAVG"]},{"entry":{}},{"entry":["23631,\u200323632","Probability:\u20030.676\u2003AA1:\u200318\u2003AA2:\u200319","MKKLAVLIVLVCTILSCN"]},{"entry":{}},{"entry":["23635,\u200323636","Probability:\u20030.886\u2003AA1:\u200316\u2003AA2:\u200317","MKKIAVVFLLLIVVIS"]},{"entry":{}},{"entry":["23649,\u200323650","Probability:\u20030.993\u2003AA1:\u200316\u2003AA2:\u200317","MKKILTSLFFLLVLTA"]},{"entry":{}},{"entry":["23723,\u200323724","Probability:\u20030.996\u2003AA1:\u200318\u2003AA2:\u200319","MKLLKKILFALVILLAIA"]},{"entry":{}},{"entry":["23817,\u200323818","Probability:\u20030.635\u2003AA1:\u200319\u2003AA2:\u200320","MIMNKRSISVLLIAMMALL"]},{"entry":{}},{"entry":["23827,\u200323828","Probability:\u20030.871\u2003AA1:\u200317\u2003AA2:\u200318","MKNYILTTAALLAFLFT"]},{"entry":{}},{"entry":["23841,\u200323842","Probability:\u20030.647\u2003AA1:\u200319\u2003AA2:\u200320","MKKYFLIFMISALYSCGSA"]},{"entry":{}},{"entry":["23851,\u200323852","Probability:\u20030.861\u2003AA1:\u200318\u2003AA2:\u200319","MKKIITFLLLTIVLVSCG"]},{"entry":{}},{"entry":["23879,\u200323880","Probability:\u20030.710\u2003AA1:\u200319\u2003AA2:\u200320","MKEILSFILFTSVAINIIA"]},{"entry":{}},{"entry":["23913,\u200323914","Probability:\u20031.000\u2003AA1:\u200319\u2003AA2:\u200320","MKKSIIIVIVLIFGFSVNA"]},{"entry":{}},{"entry":["23937,\u200323938","Probability:\u20030.997\u2003AA1:\u200319\u2003AA2:\u200320","MKKTILLITCLLSIFSIYS"]},{"entry":{}},{"entry":["23941,\u200323942","Probability:\u20030.988\u2003AA1:\u200317\u2003AA2:\u200318","MKYIYLCLFLFSSFTFS"]},{"entry":{}},{"entry":["23967,\u200323968","Probability:\u20030.991\u2003AA1:\u200319\u2003AA2:\u200320","MKKFLITLVLIPFFGIAQT"]},{"entry":{}},{"entry":["24023,\u200324024","Probability:\u20030.572\u2003AA1:\u200320\u2003AA2:\u200321","MNKYFIIILIFICFDSGSQN"]},{"entry":{}},{"entry":["24063,\u200324064","Probability:\u20031.000\u2003AA1:\u200317\u2003AA2:\u200318","MKKILILFLLISSTVSA"]},{"entry":{}},{"entry":["\u20032411,\u20032412","Probability:\u20031.000\u2003AA1:\u200321\u2003AA2:\u200322","MKKIVVLLALITAMVPAGVFA"]},{"entry":{}},{"entry":["24141,\u200324142","Probability:\u20030.985\u2003AA1:\u200320\u2003AA2:\u200321","MNMMKGFLLFFLFSVHILSA"]},{"entry":{}},{"entry":["\u20032415,\u20032416","Probability:\u20030.907\u2003AA1:\u200320\u2003AA2:\u200321","MIMNQRIKKIVGIMLLSVVA"]},{"entry":{}},{"entry":["24159,\u200324160","Probability:\u20030.994\u2003AA1:\u200320\u2003AA2:\u200321","MNKIKYFLLTITLFSLSACT"]},{"entry":{}},{"entry":["24161,\u200324162","Probability:\u20030.578\u2003AA1:\u200320\u2003AA2:\u200321","MNKFLQRISFTLLLSCILLG"]},{"entry":{}},{"entry":["24267,\u200324268","Probability:\u20030.993\u2003AA1:\u200320\u2003AA2:\u200321","MKQLLALALLVLFCACKSTK"]},{"entry":{}},{"entry":["\u20032429,\u20032430","Probability:\u20030.925\u2003AA1:\u200318\u2003AA2:\u200319","MKYCLILFSIVNSLFTNA"]},{"entry":{}},{"entry":["24317,\u200324318","Probability:\u20030.592\u2003AA1:\u200321\u2003AA2:\u200322","MLFIAPLVFYILLLTGTNNFA"]},{"entry":{}},{"entry":["\u20032435,\u20032436","Probability:\u20030.594\u2003AA1:\u200320\u2003AA2:\u200321","MKKKINYIAFIILLICSVPA"]},{"entry":{}},{"entry":["24359,\u200324360","Probability:\u20030.943\u2003AA1:\u200318\u2003AA2:\u200319","MKKLIHLALIILISVSFS"]},{"entry":{}},{"entry":["24367,\u200324368","Probability:\u20030.968\u2003AA1:\u200320\u2003AA2:\u200321","MNMKILLSILSLFLLFNCVN"]},{"entry":{}},{"entry":["24397,\u200324398","Probability:\u20031.000\u2003AA1:\u200320\u2003AA2:\u200321","MKLKIVLLILFVSITTLVNA"]},{"entry":{}},{"entry":["24469,\u200324470","Probability:\u20030.998\u2003AA1:\u200318\u2003AA2:\u200319","MKKILLIAIFLVSWVSTA"]},{"entry":{}},{"entry":["24473,\u200324474","Probability:\u20030.996\u2003AA1:\u200318\u2003AA2:\u200319","MKLLKKILFALVILLAIA"]},{"entry":{}},{"entry":["24525,\u200324526","Probability:\u20030.992\u2003AA1:\u200319\u2003AA2:\u200320","MNRIKPFLILLFLTSLTYS"]},{"entry":{}},{"entry":["24535,\u200324536","Probability:\u20031.000\u2003AA1:\u200318\u2003AA2:\u200319","MKKLILICALLISVFSTA"]},{"entry":{}},{"entry":["24579,\u200324580","Probability:\u20030.998\u2003AA1:\u200319\u2003AA2:\u200320","MRKNMLIIIFVFLTTISFG"]},{"entry":{}},{"entry":["24607,\u200324608","Probability:\u20030.998\u2003AA1:\u200319\u2003AA2:\u200320","MKNVLFILIMTFGILSCNA"]},{"entry":{}},{"entry":["24633,\u200324634","Probability:\u20030.537\u2003AA1:\u200319\u2003AA2:\u200320","MKLKKYIALMLFCLFIGFV"]},{"entry":{}},{"entry":["24655,\u200324656","Probability:\u20031.000\u2003AA1:\u200321\u2003AA2:\u200322","MKKLSPALLILLFLIPNFLQA"]},{"entry":{}},{"entry":["24707,\u200324708","Probability:\u20030.704\u2003AA1:\u200318\u2003AA2:\u200319","MKLTTHIIILIIPFXCFA"]},{"entry":{}},{"entry":["24719,\u200324720","Probability:\u20030.989\u2003AA1:\u200319\u2003AA2:\u200320","MNKRHISILMLAMTALLSG"]},{"entry":{}},{"entry":["24821,\u200324822","Probability:\u20030.945\u2003AA1:\u200319\u2003AA2:\u200320","MQKILLIIVMFVFTIKVYS"]},{"entry":{}},{"entry":["24823,\u200324824","Probability:\u20030.975\u2003AA1:\u200318\u2003AA2:\u200319","MKNSITFFILLLTIIVSA"]},{"entry":{}},{"entry":["24853,\u200324854","Probability:\u20031.000\u2003AA1:\u200321\u2003AA2:\u200322","MRVFFINCLMVFLLSSCTSYA"]},{"entry":{}},{"entry":["24879,\u200324880","Probability:\u20030.996\u2003AA1:\u200318\u2003AA2:\u200319","MNRLIILILIFFSQISIA"]},{"entry":{}},{"entry":["25057,\u200325058","Probability:\u20030.548\u2003AA1:\u200320\u2003AA2:\u200321","MTFHLKPILVFALLFMSCNT"]},{"entry":{}},{"entry":["25085,\u200325086","Probability:\u20030.900\u2003AA1:\u200319\u2003AA2:\u200320","MQKKLYFSSLLFFLIVSFT"]},{"entry":{}},{"entry":["25115,\u200325116","Probability:\u20030.748\u2003AA1:\u200318\u2003AA2:\u200319","MKNSILIFTTLFSFICGT"]},{"entry":{}},{"entry":["25169,\u200325170","Probability:\u20030.703\u2003AA1:\u200319\u2003AA2:\u200320","MKTLKQLLFITAFALLSFT"]},{"entry":{}},{"entry":["25203,\u200325204","Probability:\u20030.505\u2003AA1:\u200318\u2003AA2:\u200319","MRLTFIFTLLIIGQFSYG"]},{"entry":{}},{"entry":["25225,\u200325226","Probability:\u20030.998\u2003AA1:\u200319\u2003AA2:\u200320","MSKTIFLFLILTTVLSAQN"]},{"entry":{}},{"entry":["25299,\u200325300","Probability:\u20030.513\u2003AA1:\u200317\u2003AA2:\u200318","MKKFLVGSCFSITMLMG"]},{"entry":{}},{"entry":["25307,\u200325308","Probability:\u20031.000\u2003AA1:\u200319\u2003AA2:\u200320","MKKHIITLLVLVFSFSAIA"]},{"entry":{}},{"entry":["25311,\u200325312","Probability:\u20030.999\u2003AA1:\u200320\u2003AA2:\u200321","MKKSIITIILLVFVATQSFA"]},{"entry":{}},{"entry":["25355,\u200325356","Probability:\u20030.751\u2003AA1:\u200316\u2003AA2:\u200317","MKKIFLLLIIVFVSCS"]},{"entry":{}},{"entry":["25361,\u200325362","Probability:\u20030.708\u2003AA1:\u200318\u2003AA2:\u200319","MNRLLILLFSIFLISCYN"]},{"entry":{}},{"entry":["25395,\u200325396","Probability:\u20030.982\u2003AA1:\u200318\u2003AA2:\u200319","MKSCFVLIILLCSSFCIS"]},{"entry":{}},{"entry":["25449,\u200325450","Probability:\u20030.969\u2003AA1:\u200319\u2003AA2:\u200320","MKKLAVLIVLVCTILSCNE"]},{"entry":{}},{"entry":["25563,\u200325564","Probability:\u20030.960\u2003AA1:\u200319\u2003AA2:\u200320","MKPIFSFFICMSICTTIFS"]},{"entry":{}},{"entry":["25571,\u200325572","Probability:\u20030.997\u2003AA1:\u200320\u2003AA2:\u200321","MHSRYFIILLLVLFTINSFS"]},{"entry":{}},{"entry":["25615,\u200325616","Probability:\u20030.955\u2003AA1:\u200320\u2003AA2:\u200321","MMSVVLLGLIASVITQYVAS"]},{"entry":{}},{"entry":["25621,\u200325622","Probability:\u20030.511\u2003AA1:\u200321\u2003AA2:\u200322","MIKNRPRKTYTILSLLVASYS"]},{"entry":{}},{"entry":["\u20032571,\u20032572","Probability:\u20030.871\u2003AA1:\u200320\u2003AA2:\u200321","MKKLKLLTLSSLAFLSTVPV"]},{"entry":{}},{"entry":["25761,\u200325762","Probability:\u20030.989\u2003AA1:\u200319\u2003AA2:\u200320","MNKRHISILMLAMTALLSG"]},{"entry":{}},{"entry":["25797,\u200325798","Probability:\u20030.999\u2003AA1:\u200318\u2003AA2:\u200319","MKNLILILALSIGFNAFS"]},{"entry":{}},{"entry":["25833,\u200325834","Probability:\u20030.998\u2003AA1:\u200318\u2003AA2:\u200319","MKKITTLLILFTSIISFS"]},{"entry":{}},{"entry":["25865,\u200325866","Probability:\u20030.745\u2003AA1:\u200315\u2003AA2:\u200316","MKKIIIALATTLVFG"]},{"entry":{}},{"entry":["25867,\u200325868","Probability:\u20030.949\u2003AA1:\u200318\u2003AA2:\u200319","MKKTIVILLIFISCSSTK"]},{"entry":{}},{"entry":["25899,\u200325900","Probability:\u20031.000\u2003AA1:\u200319\u2003AA2:\u200320","MKKIITLVALVFFSVSTFA"]},{"entry":{}},{"entry":["25985,\u200325986","Probability:\u20030.776\u2003AA1:\u200320\u2003AA2:\u200321","MNFKASLFFVVTIFSIGLMS"]},{"entry":{}},{"entry":["26069,\u200326070","Probability:\u20030.730\u2003AA1:\u200318\u2003AA2:\u200319","MKKNIYRIFLTILSIALA"]},{"entry":{}},{"entry":["26081,\u200326082","Probability:\u20031.000\u2003AA1:\u200318\u2003AA2:\u200319","MKNLFFTAIFLFCLPALA"]},{"entry":{}},{"entry":["26085,\u200326086","Probability:\u20030.980\u2003AA1:\u200317\u2003AA2:\u200318","MKWSFLILFVLSFPSSA"]},{"entry":{}},{"entry":["26127,\u200326128","Probability:\u20030.985\u2003AA1:\u200317\u2003AA2:\u200318","MKRNLAILLLILTTVLS"]},{"entry":{}},{"entry":["26133,\u200326134","Probability:\u20030.846\u2003AA1:\u200320\u2003AA2:\u200321","MPMKNASFLIVLLLFFSACK"]},{"entry":{}},{"entry":["26249,\u200326250","Probability:\u20030.999\u2003AA1:\u200317\u2003AA2:\u200318","MKQILILFSLLYFSAQA"]},{"entry":{}},{"entry":["26305,\u200326306","Probability:\u20030.996\u2003AA1:\u200318\u2003AA2:\u200319","MASGLLVFLALHPTQSNA"]},{"entry":{}},{"entry":["26341,\u200326342","Probability:\u20030.629\u2003AA1:\u200320\u2003AA2:\u200321","MKKSNSIIYLLSLTLLFFSS"]},{"entry":{}},{"entry":["26431,\u200326432","Probability:\u20030.663\u2003AA1:\u200318\u2003AA2:\u200319","MNIVILFFLLSFLALVLS"]},{"entry":{}},{"entry":["26445,\u200326446","Probability:\u20030.527\u2003AA1:\u200313\u2003AA2:\u200314","MKTKSLSFFMAHN"]},{"entry":{}},{"entry":["26489,\u200326490","Probability:\u20030.999\u2003AA1:\u200320\u2003AA2:\u200321","MKRTFSMSIILILALSFFNA"]},{"entry":{}},{"entry":["26595,\u200326596","Probability:\u20030.979\u2003AA1:\u200319\u2003AA2:\u200320","MKYYLFILLVGLSTTFGFS"]},{"entry":{}},{"entry":["26609,\u200326610","Probability:\u20030.997\u2003AA1:\u200320\u2003AA2:\u200321","MHSRYFIILLLVLFTINSFS"]},{"entry":{}},{"entry":["26625,\u200326626","Probability:\u20030.901\u2003AA1:\u200320\u2003AA2:\u200321","MKRSTLSLFLVITLSLFQIG"]},{"entry":{}},{"entry":["26627,\u200326628","Probability:\u20030.998\u2003AA1:\u200319\u2003AA2:\u200320","MNKYIPLLMLVLFANLSFS"]},{"entry":{}},{"entry":["26675,\u200326676","Probability:\u20030.939\u2003AA1:\u200317\u2003AA2:\u200318","MKKLLICMSALAIVACK"]},{"entry":{}},{"entry":["26681,\u200326682","Probability:\u20030.715\u2003AA1:\u200318\u2003AA2:\u200319","MKRTVILTLVFLPLILIG"]},{"entry":{}},{"entry":["26719,\u200326720","Probability:\u20030.996\u2003AA1:\u200318\u2003AA2:\u200319","MNKFILLIGLCVCSNIFS"]},{"entry":{}},{"entry":["26747,\u200326748","Probability:\u20030.689\u2003AA1:\u200318\u2003AA2:\u200319","MIKVFKLVVVALLCLACK"]},{"entry":{}},{"entry":["\u20032675,\u20032676","Probability:\u20031.000\u2003AA1:\u200318\u2003AA2:\u200319","MKKTMSLLLFFVSAIAFS"]},{"entry":{}},{"entry":["\u20032683,\u20032684","Probability:\u20030.939\u2003AA1:\u200319\u2003AA2:\u200320","MKKIILSFLFLLYCTFVQN"]},{"entry":{}},{"entry":["26855,\u200326856","Probability:\u20030.619\u2003AA1:\u200318\u2003AA2:\u200319","MNKLGIIIGLLIGLGLFG"]},{"entry":{}},{"entry":["26857,\u200326858","Probability:\u20030.501\u2003AA1:\u200320\u2003AA2:\u200321","MYVMKSIFFLFFLMCMNISN"]},{"entry":{}},{"entry":["\u2003\u2003275,\u2003276","Probability:\u20030.999\u2003AA1:\u200321\u2003AA2:\u200322","MLKNISIFSILFILLINASNA"]},{"entry":{}},{"entry":["\u20032761,\u20032762","Probability:\u20030.980\u2003AA1:\u200317\u2003AA2:\u200318","MKKIHIVFLILIVAALA"]},{"entry":{}},{"entry":["\u20032927,\u20032928","Probability:\u20030.964\u2003AA1:\u200318\u2003AA2:\u200319","MLKNISIFSILFILLINA"]},{"entry":{}},{"entry":["\u20032941,\u20032942","Probability:\u20030.997\u2003AA1:\u200319\u2003AA2:\u200320","MKSTLVTFFILLFSQFIVA"]},{"entry":{}},{"entry":["\u20032985,\u20032986","Probability:\u20030.957\u2003AA1:\u200319\u2003AA2:\u200320","MLRSKWGFLALCALLISWQ"]},{"entry":{}},{"entry":["\u2003\u2003303,\u2003304","Probability:\u20030.997\u2003AA1:\u200319\u2003AA2:\u200320","MSKIFLAVLTVFTGIGVQA"]},{"entry":{}},{"entry":["\u20033055,\u20033056","Probability:\u20030.545\u2003AA1:\u200319\u2003AA2:\u200320","MNSMNKVLIFIILLFSSVS"]},{"entry":{}},{"entry":["\u20033113,\u20033114","Probability:\u20030.978\u2003AA1:\u200319\u2003AA2:\u200320","MKRAILITFIIIFTNQLYA"]},{"entry":{}},{"entry":["\u20033117,\u20033118","Probability:\u20030.623\u2003AA1:\u200319\u2003AA2:\u200320","MKTKTIFILFFSIVSFCFK"]},{"entry":{}},{"entry":["\u20033119,\u20033120","Probability:\u20030.999\u2003AA1:\u200320\u2003AA2:\u200321","MKIILLYAAIVGSLFVSCNA"]},{"entry":{}},{"entry":["\u20033221,\u20033222","Probability:\u20030.995\u2003AA1:\u200320\u2003AA2:\u200321","MIKAKIFSGLLLFISTALFS"]},{"entry":{}},{"entry":["\u20033231,\u20033232","Probability:\u20030.992\u2003AA1:\u200319\u2003AA2:\u200320","MRIIIAIILIIVLPLISQG"]},{"entry":{}},{"entry":["\u20033233,\u20033234","Probability:\u20031.000\u2003AA1:\u200320\u2003AA2:\u200321","MKKDLILTLLIFLFTVSLTA"]},{"entry":{}},{"entry":["\u20033235,\u20033236","Probability:\u20031.000\u2003AA1:\u200320\u2003AA2:\u200321","MKKDLILTLLIFLXTVSLTA"]},{"entry":{}},{"entry":["\u20033245,\u20033246","Probability:\u20030.975\u2003AA1:\u200319\u2003AA2:\u200320","MKTNLKLFLLIMLSISIEA"]},{"entry":{}},{"entry":["\u20033277,\u20033278","Probability:\u20030.979\u2003AA1:\u200320\u2003AA2:\u200321","MRNSLTLLIFALLLTNCNNS"]},{"entry":{}},{"entry":["\u20033337,\u20033338","Probability:\u20030.994\u2003AA1:\u200319\u2003AA2:\u200320","MKKIVSILVFVLLANLSQA"]},{"entry":{}},{"entry":["\u20033397,\u20033398","Probability:\u20030.643\u2003AA1:\u200319\u2003AA2:\u200320","MEESMRLFVLLFLIFPVVA"]},{"entry":{}},{"entry":["\u20033405,\u20033406","Probability:\u20030.534\u2003AA1:\u200320\u2003AA2:\u200321","MIKKGLSLLRGILSPSITWK"]},{"entry":{}},{"entry":["\u20033471,\u20033472","Probability:\u20030.954\u2003AA1:\u200319\u2003AA2:\u200320","MKRIFFAFSILFISVCGFA"]},{"entry":{}},{"entry":["\u20033515,\u20033516","Probability:\u20030.686\u2003AA1:\u200319\u2003AA2:\u200320","MAFIFFALVIGTMVGISMT"]},{"entry":{}},{"entry":["\u20033525,\u20033526","Probability:\u20030.873\u2003AA1:\u200320\u2003AA2:\u200321","MKRKIFFYTLMSILLAGIFA"]},{"entry":{}},{"entry":["\u20033565,\u20033566","Probability:\u20030.646\u2003AA1:\u200319\u2003AA2:\u200320","MKRFSYLFLLILLINQCRN"]},{"entry":{}},{"entry":["\u2003\u2003365,\u2003366","Probability:\u20030.540\u2003AA1:\u200321\u2003AA2:\u200322","MHNSSPMKNLLILFASFILSS"]},{"entry":{}},{"entry":["\u20033705,\u20033706","Probability:\u20030.845\u2003AA1:\u200319\u2003AA2:\u200320","MKPTLLIMAAGMASRYGSM"]},{"entry":{}},{"entry":["\u20033711,\u20033712","Probability:\u20030.994\u2003AA1:\u200319\u2003AA2:\u200320","MKKIVSILVFVLLANLSQA"]},{"entry":{}},{"entry":["\u2003\u2003373,\u2003374","Probability:\u20030.665\u2003AA1:\u200320\u2003AA2:\u200321","MNKRIIILISATVVSISLQV"]},{"entry":{}},{"entry":["\u20033771,\u20033772","Probability:\u20030.860\u2003AA1:\u200320\u2003AA2:\u200321","MQINRLIVLLLIMISHKNFA"]},{"entry":{}},{"entry":["\u20033843,\u20033844","Probability:\u20031.000\u2003AA1:\u200319\u2003AA2:\u200320","MKSFIFFLFFVLAFSVANA"]},{"entry":{}},{"entry":["\u20033871,\u20033872","Probability:\u20030.973\u2003AA1:\u200318\u2003AA2:\u200319","MNKSVLVVGAGLGGMALA"]},{"entry":{}},{"entry":["\u20033913,\u20033914","Probability:\u20030.999\u2003AA1:\u200319\u2003AA2:\u200320","MNRFLIFIFVICFCGTAVS"]},{"entry":{}},{"entry":["\u2003\u2003397,\u2003398","Probability:\u20030.978\u2003AA1:\u200318\u2003AA2:\u200319","MIRFVIPVFFLLPFFSNA"]},{"entry":{}},{"entry":["\u20034037,\u20034038","Probability:\u20030.904\u2003AA1:\u200319\u2003AA2:\u200320","MKKEFLKIGIAILVLFAIA"]},{"entry":{}},{"entry":["\u20034087,\u20034088","Probability:\u20031.000\u2003AA1:\u200318\u2003AA2:\u200319","MRKIFAVAALFTSSQLAA"]},{"entry":{}},{"entry":["\u20034153,\u20034154","Probability:\u20030.515\u2003AA1:\u200318\u2003AA2:\u200319","MLKSLFLFTSFSSFFLLV"]},{"entry":{}},{"entry":["\u20034167,\u20034168","Probability:\u20030.784\u2003AA1:\u200317\u2003AA2:\u200318","MARLVLVFILLHQTLVA"]},{"entry":{}},{"entry":["\u2003\u2003419,\u2003420","Probability:\u20030.708\u2003AA1:\u200318\u2003AA2:\u200319","MKSLIIIFSLILFFTACK"]},{"entry":{}},{"entry":["\u20034223,\u20034224","Probability:\u20030.659\u2003AA1:\u200319\u2003AA2:\u200320","MRSKIFLLMLXTASFYSPS"]},{"entry":{}},{"entry":["\u20034241,\u20034242","Probability:\u20031.000\u2003AA1:\u200320\u2003AA2:\u200321","MKKKITILILFFFICLFALS"]},{"entry":{}},{"entry":["\u20034259,\u20034260","Probability:\u20030.788\u2003AA1:\u200317\u2003AA2:\u200318","MKTIIIISTILIITSCA"]},{"entry":{}},{"entry":["\u20034273,\u20034274","Probability:\u20030.987\u2003AA1:\u200319\u2003AA2:\u200320","MKAHIRIILLALFFGTAVQ"]},{"entry":{}},{"entry":["\u20034291,\u20034292","Probability:\u20031.000\u2003AA1:\u200318\u2003AA2:\u200319","MKKIFLSFCLLSSFIGFA"]},{"entry":{}},{"entry":["\u20034309,\u20034310","Probability:\u20030.924\u2003AA1:\u200318\u2003AA2:\u200319","MKYCLILFSIVNSLFTNA"]},{"entry":{}},{"entry":["\u20034313,\u20034314","Probability:\u20030.834\u2003AA1:\u200318\u2003AA2:\u200319","MKNKVLLSFLCFFLYTHV"]},{"entry":{}},{"entry":["\u20034319,\u20034320","Probability:\u20030.670\u2003AA1:\u200315\u2003AA2:\u200316","MKLAALILLIIFTQS"]},{"entry":{}},{"entry":["\u20034337,\u20034338","Probability:\u20030.537\u2003AA1:\u200320\u2003AA2:\u200321","MKKLTVILFFVIGVTYQVIG"]},{"entry":{}},{"entry":["\u20034361,\u20034362","Probability:\u20030.850\u2003AA1:\u200317\u2003AA2:\u200318","MKKILFSLIVCVFCLSS"]},{"entry":{}},{"entry":["\u20034405,\u20034406","Probability:\u20030.873\u2003AA1:\u200320\u2003AA2:\u200321","MKRKIFFYTLMSILLAGIFA"]},{"entry":{}},{"entry":["\u20034415,\u20034416","Probability:\u20030.923\u2003AA1:\u200318\u2003AA2:\u200319","MKQMLAAIFCFCFFISHS"]},{"entry":{}},{"entry":["\u20034483,\u20034484","Probability:\u20030.575\u2003AA1:\u200318\u2003AA2:\u200319","MRTIVILYLVMFSLSCQQ"]},{"entry":{}},{"entry":["\u20034623,\u20034624","Probability:\u20030.540\u2003AA1:\u200321\u2003AA2:\u200322","MHNSSPMKNLLILFASFILSS"]},{"entry":{}},{"entry":["\u2003\u2003463,\u2003464","Probability:\u20030.643\u2003AA1:\u200321\u2003AA2:\u200322","MIRTKGXXXXXXXXXXXXXXX"]},{"entry":{}},{"entry":["\u20034641,\u20034642","Probability:\u20030.976\u2003AA1:\u200320\u2003AA2:\u200321","MKRMIIITGMLAFLGTTGFG"]},{"entry":{}},{"entry":["\u20034771,\u20034772","Probability:\u20030.998\u2003AA1:\u200319\u2003AA2:\u200320","MSKIFLAVLTVFTGIGVQA"]},{"entry":{}},{"entry":["\u20034903,\u20034904","Probability:\u20030.998\u2003AA1:\u200317\u2003AA2:\u200318","MFRFIIVSVFAISVSFA"]},{"entry":{}},{"entry":["\u20034937,\u20034938","Probability:\u20030.977\u2003AA1:\u200319\u2003AA2:\u200320","MTIMKYKLITLFLLFQTLA"]},{"entry":{}},{"entry":["\u20034939,\u20034940","Probability:\u20030.600\u2003AA1:\u200319\u2003AA2:\u200320","MKTTVRILCVCILVCSSVS"]},{"entry":{}},{"entry":["\u2003\u2003495,\u2003496","Probability:\u20030.674\u2003AA1:\u200320\u2003AA2:\u200321","MGRHAVCXXXXXXXXXXXXX"]},{"entry":{}},{"entry":["\u20035031,\u20035032","Probability:\u20030.713\u2003AA1:\u200318\u2003AA2:\u200319","MKKSLTMTLFAGLFLINS"]},{"entry":{}},{"entry":["\u20035037,\u20035038","Probability:\u20030.987\u2003AA1:\u200319\u2003AA2:\u200320","MKAHIRIILLALFFGTAVQ"]},{"entry":{}},{"entry":["\u20035057,\u20035058","Probability:\u20030.990\u2003AA1:\u200319\u2003AA2:\u200320","MRNYILTILTFFFSITLFA"]},{"entry":{}},{"entry":["\u2003\u2003509,\u2003510","Probability:\u20030.558\u2003AA1:\u200318\u2003AA2:\u200319","MKVLLSTALPIALCCLLL"]},{"entry":{}},{"entry":["\u20035261,\u20035262","Probability:\u20030.558\u2003AA1:\u200318\u2003AA2:\u200319","MKVLLSTALPIALCCLLL"]},{"entry":{}},{"entry":["\u20035289,\u20035290","Probability:\u20030.979\u2003AA1:\u200315\u2003AA2:\u200316","MANKYLLCLLFLVSA"]},{"entry":{}},{"entry":["\u20035351,\u20035352","Probability:\u20030.928\u2003AA1:\u200318\u2003AA2:\u200319","MKFLLALLIITCSDICTA"]},{"entry":{}},{"entry":["\u20035371,\u20035372","Probability:\u20030.588\u2003AA1:\u200320\u2003AA2:\u200321","MKKLILLPFLALLLGFILPG"]},{"entry":{}},{"entry":["\u20035421,\u20035422","Probability:\u20031.000\u2003AA1:\u200319\u2003AA2:\u200320","MLKILLAVTVAFSFLPAFS"]},{"entry":{}},{"entry":["\u20035571,\u20035572","Probability:\u20030.836\u2003AA1:\u200318\u2003AA2:\u200319","MKLTVTAIFLCVAIQAIS"]},{"entry":{}},{"entry":["\u20035573,\u20035574","Probability:\u20030.985\u2003AA1:\u200320\u2003AA2:\u200321","MKKYFGILLITSLICLVQLQ"]},{"entry":{}},{"entry":["\u20035575,\u20035576","Probability:\u20030.999\u2003AA1:\u200319\u2003AA2:\u200320","MRTNCILLSVLLFTFSTSS"]},{"entry":{}},{"entry":["\u20035617,\u20035618","Probability:\u20030.968\u2003AA1:\u200319\u2003AA2:\u200320","MKNKIIAFLPLLICAGVIT"]},{"entry":{}},{"entry":["\u20035693,\u20035694","Probability:\u20030.992\u2003AA1:\u200318\u2003AA2:\u200319","MKILTIVFLVGFFCFVQA"]},{"entry":{}},{"entry":["\u20035801,\u20035802","Probability:\u20031.000\u2003AA1:\u200320\u2003AA2:\u200321","MKKILFIAATAVLFSTTTMA"]},{"entry":{}},{"entry":["\u20035811,\u20035812","Probability:\u20030.875\u2003AA1:\u200318\u2003AA2:\u200319","MKSFYFLIAMGISLNASA"]},{"entry":{}},{"entry":["\u20035943,\u20035944","Probability:\u20030.983\u2003AA1:\u200316\u2003AA2:\u200317","MTTNTILLLLLSLVIA"]},{"entry":{}},{"entry":["\u20035951,\u20035952","Probability:\u20030.722\u2003AA1:\u200320\u2003AA2:\u200321","MKLKKGIILIIAFLGFGLSN"]},{"entry":{}},{"entry":["\u20035953,\u20035954","Probability:\u20030.973\u2003AA1:\u200320\u2003AA2:\u200321","MMPKLKLLLLGCLLILLKNA"]},{"entry":{}},{"entry":["\u20036027,\u20036028","Probability:\u20030.682\u2003AA1:\u200318\u2003AA2:\u200319","MKSLIIIFSLILFFTACK"]},{"entry":{}},{"entry":["\u20036041,\u20036042","Probability:\u20030.873\u2003AA1:\u200318\u2003AA2:\u200319","MTTSKTLLFILFLVMTQL"]},{"entry":{}},{"entry":["\u20036185,\u20036186","Probability:\u20030.997\u2003AA1:\u200319\u2003AA2:\u200320","MKSTLVTFFILLFSQFIVA"]},{"entry":{}},{"entry":["\u20036245,\u20036246","Probability:\u20030.854\u2003AA1:\u200317\u2003AA2:\u200318","MKYFLFIFLLSCPVTLS"]},{"entry":{}},{"entry":["\u20036289,\u20036290","Probability:\u20030.999\u2003AA1:\u200319\u2003AA2:\u200320","MKKLFPVVLFLLSMNVLQA"]},{"entry":{}},{"entry":["\u20036323,\u20036324","Probability:\u20031.000\u2003AA1:\u200320\u2003AA2:\u200321","MKFFSLFYCFLLLGLNFALA"]},{"entry":{}},{"entry":["\u20036469,\u20036470","Probability:\u20030.997\u2003AA1:\u200319\u2003AA2:\u200320","MKSTLVTFFILLFSQFIVA"]},{"entry":{}},{"entry":["\u20036667,\u20036668","Probability:\u20030.912\u2003AA1:\u200318\u2003AA2:\u200319","MRKNFQIVLAFAMTIATS"]},{"entry":{}},{"entry":["\u20036719,\u20036720","Probability:\u20031.000\u2003AA1:\u200318\u2003AA2:\u200319","MRKVLLTMLFLSCFSGNA"]},{"entry":{}},{"entry":["\u20036741,\u20036742","Probability:\u20030.551\u2003AA1:\u200318\u2003AA2:\u200319","MKKVLLTGCLLITIISTG"]},{"entry":{}},{"entry":["\u20036755,\u20036756","Probability:\u20030.909\u2003AA1:\u200319\u2003AA2:\u200320","MKKFLIFCLFLFLNKPLIS"]},{"entry":{}},{"entry":["\u20036789,\u20036790","Probability:\u20030.707\u2003AA1:\u200318\u2003AA2:\u200319","MRAFLSITVCFFLFVNFA"]},{"entry":{}},{"entry":["\u20036885,\u20036886","Probability:\u20030.693\u2003AA1:\u200319\u2003AA2:\u200320","MKILKLFFLLFILPITTLQ"]},{"entry":{}},{"entry":["\u20036893,\u20036894","Probability:\u20030.911\u2003AA1:\u200316\u2003AA2:\u200317","MKYLILIIALCTLTFS"]},{"entry":{}},{"entry":["\u20036943,\u20036944","Probability:\u20031.000\u2003AA1:\u200318\u2003AA2:\u200319","MKKLILSLFILISLNVFA"]},{"entry":{}},{"entry":["\u20036951,\u20036952","Probability:\u20030.903\u2003AA1:\u200319\u2003AA2:\u200320","MLFMKKIILLLIVTAVVVS"]},{"entry":{}},{"entry":["\u20037029,\u20037030","Probability:\u20030.852\u2003AA1:\u200316\u2003AA2:\u200317","MKKIFLLIFTFIYLNA"]},{"entry":{}},{"entry":["\u2003\u2003711,\u2003712","Probability:\u20030.936\u2003AA1:\u200319\u2003AA2:\u200320","MKNQIFSGFLLLFIVNQVA"]},{"entry":{}},{"entry":["\u20037357,\u20037358","Probability:\u20030.610\u2003AA1:\u200316\u2003AA2:\u200317","MLRVILISIFVLNIYA"]},{"entry":{}},{"entry":["\u20037389,\u20037390","Probability:\u20030.987\u2003AA1:\u200319\u2003AA2:\u200320","MKYFTLIFLFTYLSLSSFG"]},{"entry":{}},{"entry":["\u20037491,\u20037492","Probability:\u20030.682\u2003AA1:\u200319\u2003AA2:\u200320","MKSMRTIFISFLIILLLQG"]},{"entry":{}},{"entry":["\u20037505,\u20037506","Probability:\u20030.995\u2003AA1:\u200319\u2003AA2:\u200320","MRRLILALMCVILLSSFVV"]},{"entry":{}},{"entry":["\u20037543,\u20037544","Probability:\u20030.997\u2003AA1:\u200319\u2003AA2:\u200320","MLIKNIVILSFLFVLFAQG"]},{"entry":{}},{"entry":["\u2003\u2003759,\u2003760","Probability:\u20030.711\u2003AA1:\u200319\u2003AA2:\u200320","MKFKLAIFALILISINLIS"]},{"entry":{}},{"entry":["\u20037765,\u20037766","Probability:\u20030.529\u2003AA1:\u200318\u2003AA2:\u200319","MKICIIGLGYVGLPLAHA"]},{"entry":{}},{"entry":["\u20037785,\u20037786","Probability:\u20030.983\u2003AA1:\u200319\u2003AA2:\u200320","MFKKVGILLFLIALSLVVL"]},{"entry":{}},{"entry":["\u20037835,\u20037836","Probability:\u20030.878\u2003AA1:\u200315\u2003AA2:\u200316","MRIIFLVLISFSLYA"]},{"entry":{}},{"entry":["\u20037907,\u20037908","Probability:\u20030.926\u2003AA1:\u200319\u2003AA2:\u200320","MKKSSLLLLITFVVVLFQG"]},{"entry":{}},{"entry":["\u2003\u2003821,\u2003822","Probability:\u20031.000\u2003AA1:\u200320\u2003AA2:\u200321","MKKWRFSLSIFLFVCVTCLA"]},{"entry":{}},{"entry":["\u20038247,\u20038248","Probability:\u20030.995\u2003AA1:\u200316\u2003AA2:\u200317","MRISILLAVVSSIIFA"]},{"entry":{}},{"entry":["\u2003\u2003827,\u2003828","Probability:\u20031.000\u2003AA1:\u200319\u2003AA2:\u200320","MKKFFSLIIFLIFSFASFA"]},{"entry":{}},{"entry":["\u20038279,\u20038280","Probability:\u20030.672\u2003AA1:\u200315\u2003AA2:\u200316","MKFNLNSFLMSVSLA"]},{"entry":{}},{"entry":["\u20038313,\u20038314","Probability:\u20030.992\u2003AA1:\u200319\u2003AA2:\u200320","MKHIKKALLILLFLFLSFS"]},{"entry":{}},{"entry":["\u20038349,\u20038350","Probability:\u20030.562\u2003AA1:\u200317\u2003AA2:\u200318","MFKTILFTIVFVTNIFS"]},{"entry":{}},{"entry":["\u20038437,\u20038438","Probability:\u20030.820\u2003AA1:\u200316\u2003AA2:\u200317","MKKRVMSSMKSGGVVA"]},{"entry":{}},{"entry":["\u20038461,\u20038462","Probability:\u20030.857\u2003AA1:\u200317\u2003AA2:\u200318","MKKYLALFAFILLVLSS"]},{"entry":{}},{"entry":["\u20038513,\u20038514","Probability:\u20030.993\u2003AA1:\u200316\u2003AA2:\u200317","MKLLLLLFLVLLNVNA"]},{"entry":{}},{"entry":["\u20038561,\u20038562","Probability:\u20030.618\u2003AA1:\u200317\u2003AA2:\u200318","MKLNEGAILVLSGPSGA"]},{"entry":{}},{"entry":["\u20038585,\u20038586","Probability:\u20030.997\u2003AA1:\u200317\u2003AA2:\u200318","MKKFTLSILISSSLAFG"]},{"entry":{}},{"entry":["\u20038727,\u20038728","Probability:\u20030.825\u2003AA1:\u200319\u2003AA2:\u200320","MKKITKILLIFALVAIFSG"]},{"entry":{}},{"entry":["\u20038793,\u20038794","Probability:\u20030.582\u2003AA1:\u200319\u2003AA2:\u200320","MKNKCLLIILLRVISTFLL"]},{"entry":{}},{"entry":["\u20038853,\u20038854","Probability:\u20030.649\u2003AA1:\u200317\u2003AA2:\u200318","MMKILILTITTTAILCA"]},{"entry":{}},{"entry":["\u20039021,\u20039022","Probability:\u20030.998\u2003AA1:\u200319\u2003AA2:\u200320","MNYKSFLALTAAVIISFSA"]},{"entry":{}},{"entry":["\u20039039,\u20039040","Probability:\u20030.977\u2003AA1:\u200319\u2003AA2:\u200320","MQKAFYILILLSVSLSSFG"]},{"entry":{}},{"entry":["\u20039213,\u20039214","Probability:\u20030.772\u2003AA1:\u200317\u2003AA2:\u200318","MKKKILIVGGGTAGTMT"]},{"entry":{}},{"entry":["\u20039351,\u20039352","Probability:\u20030.744\u2003AA1:\u200318\u2003AA2:\u200319","MIKRTTGILLLIFISIFA"]},{"entry":{}},{"entry":["\u20039373,\u20039374","Probability:\u20030.931\u2003AA1:\u200317\u2003AA2:\u200318","MKHILFITLFFLTSLFA"]},{"entry":{}},{"entry":["\u20039413,\u20039414","Probability:\u20030.540\u2003AA1:\u200315\u2003AA2:\u200316","MSNIALSLGSCFTIA"]},{"entry":{}},{"entry":["\u20039613,\u20039614","Probability:\u20030.998\u2003AA1:\u200317\u2003AA2:\u200318","MKIFSLIFILLFTSLSA"]},{"entry":{}},{"entry":["\u20039713,\u20039714","Probability:\u20030.911\u2003AA1:\u200317\u2003AA2:\u200318","MKKVITLSLITLNILFA"]},{"entry":{}},{"entry":["\u20039829,\u20039830","Probability:\u20030.768\u2003AA1:\u200319\u2003AA2:\u200320","MKNILKIIFIIFLFSSCQT"]},{"entry":{}},{"entry":["\u20039881,\u20039882","Probability:\u20030.963\u2003AA1:\u200319\u2003AA2:\u200320","MNKNLITAALLFVFGYTML"]},{"entry":{}},{"entry":["\u20039925,\u20039926","Probability:\u20030.751\u2003AA1:\u200318\u2003AA2:\u200319","MLQKSFISILLSLLLSLS"]},{"entry":{}},{"entry":["\u20039979,\u20039980","Probability:\u20030.799\u2003AA1:\u200319\u2003AA2:\u200320","MIMKDLILTLLLLSVYCLV"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}}}},"Methods for identifying \u201cprepro\u201d domain sequences and signal sequences are well known in the art, see, e.g., Van de Ven (1993) Crit. Rev. Oncog. 4(2):115-136. For example, to identify a prepro sequence, the protein is purified from the extracellular space and the N-terminal protein sequence is determined and compared to the unprocessed form.","The invention includes polypeptides with or without a signal sequence and\/or a prepro sequence. The invention includes polypeptides with heterologous signal sequences and\/or prepro sequences. The prepro sequence (including a sequence of the invention used as a heterologous prepro domain) can be located on the amino terminal or the carboxy terminal end of the protein. The invention also includes isolated or recombinant signal sequences, prepro sequences and catalytic domains (e.g., \u201cactive sites\u201d) comprising sequences of the invention. The polypeptide comprising a signal sequence of the invention can be a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention or another polypeptide, enzyme, protein, e.g. structural or binding protein, or another enzyme or other polypeptide.","The polypeptide, enzyme, protein, e.g. structural or binding protein signal sequences (SPs) and\/or prepro sequences of the invention can be isolated peptides, or, sequences joined to another a polypeptide, enzyme, protein, e.g. structural or binding protein, or a non-polypeptide, non-enzyme, non-protein, e.g. non-structural or non-binding protein, e.g., as a fusion (chimeric) protein. In one aspect, the invention provides polypeptides comprising a polypeptide, enzyme, protein, e.g. structural or binding protein, signal sequences of the invention. In one aspect, polypeptides comprising polypeptide, enzyme, protein, e.g. structural or binding protein signal sequences SPs and\/or prepro of the invention comprise sequences heterologous to a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention (e.g., a fusion protein comprising an SP and\/or prepro of the invention and sequences from another a polypeptide, enzyme, protein, e.g. structural or binding protein, or a non-polypeptide, non-enzyme, non-protein, e.g. non-structural or non-binding protein). In one aspect, the invention provides a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention with heterologous SPs and\/or prepro sequences, e.g., sequences with a yeast signal sequence. A polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention can comprise a heterologous SP and\/or prepro in a vector, e.g., a pPIC series vector (Invitrogen, Carlsbad, Calif.).","In one aspect, SPs and\/or prepro sequences of the invention are identified following identification of novel a polypeptide, enzyme, protein, e.g. structural or binding protein. The pathways by which proteins are sorted and transported to their proper cellular location are often referred to as protein targeting pathways. One of the most important elements in all of these targeting systems is a short amino acid sequence at the amino terminus of a newly synthesized polypeptide called the signal sequence. This signal sequence directs a protein to its appropriate location in the cell and is removed during transport or when the protein reaches its final destination. Most lysosomal, membrane, or secreted proteins have an amino-terminal signal sequence that marks them for translocation into the lumen of the endoplasmic reticulum. The signal sequences can vary in length from about 10 to 65, or more, amino acid residues. Various methods of recognition of signal sequences are known to those of skill in the art. For example, in one aspect, novel a polypeptide, enzyme, protein, e.g. structural or binding protein, signal peptides are identified by a method referred to as SignalP. SignalP uses a combined neural network which recognizes both signal peptides and their cleavage sites. (Nielsen (1997) \u201cIdentification of prokaryotic and eukaryotic signal peptides and prediction of their cleavage sites.\u201d Protein Engineering 10:1-6.","It should be understood that in some aspects a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention may not have SPs and\/or prepro sequences, or \u201cdomains.\u201d In one aspect, the invention provides the polypeptide, enzyme, protein, e.g. structural or binding proteins of the invention lacking all or part of an SP and\/or a prepro domain. In one aspect, the invention provides a nucleic acid sequence encoding a signal sequence (SP) and\/or prepro from one a polypeptide, enzyme, protein, e.g. structural or binding protein, operably linked to a nucleic acid sequence of a different a polypeptide, enzyme, protein, e.g. structural or binding protein, or, optionally, a signal sequence (SPs) and\/or prepro domain from a non-enzyme or non-protein, e.g. non-structural or non-binding protein, may be desired.","The invention also provides isolated or recombinant polypeptides comprising signal sequences (SPs), prepro domain and\/or catalytic domains (CDs) of the invention and heterologous sequences. The heterologous sequences are sequences not naturally associated (e.g., to a enzyme) with an SP, prepro domain and\/or CD. The sequence to which the SP, prepro domain and\/or CD are not naturally associated can be on the SP's, prepro domain and\/or CD's amino terminal end, carboxy terminal end, and\/or on both ends of the SP and\/or CD. In one aspect, the invention provides an isolated or recombinant polypeptide comprising (or consisting of) a polypeptide comprising a signal sequence (SP), prepro domain and\/or catalytic domain (CD) of the invention with the proviso that it is not associated with any sequence to which it is naturally associated (e.g., a polypeptide, enzyme, protein, e.g. structural or binding protein, sequence). Similarly in one aspect, the invention provides isolated or recombinant nucleic acids encoding these polypeptides. Thus, in one aspect, the isolated or recombinant nucleic acid of the invention comprises coding sequence for a signal sequence (SP), prepro domain and\/or catalytic domain (CD) of the invention and a heterologous sequence (i.e., a sequence not naturally associated with the a signal sequence (SP), prepro domain and\/or catalytic domain (CD) of the invention). The heterologous sequence can be on the 3\u2032 terminal end, 5\u2032 terminal end, and\/or on both ends of the SP, prepro domain and\/or CD coding sequence.","Hybrid (Chimeric) a Polypeptide, Enzyme, Protein, e.g. Structural or Binding Protein, and Peptide Libraries","In one aspect, the invention provides hybrid a polypeptide, enzyme, protein, e.g. structural or binding protein, and fusion proteins, including peptide libraries, comprising sequences of the invention. The peptide libraries of the invention can be used to isolate peptide modulators (e.g., activators or inhibitors) of targets, such as a polypeptide, enzyme, protein, e.g. structural or binding protein, substrates, receptors, enzymes. The peptide libraries of the invention can be used to identify formal binding partners of targets, such as ligands, e.g., cytokines, hormones and the like. In one aspect, the invention provides chimeric proteins comprising a signal sequence (SP), prepro domain and\/or catalytic domain (CD) of the invention or a combination thereof and a heterologous sequence (see above).","In one aspect, the fusion proteins of the invention (e.g., the peptide moiety) are conformationally stabilized (relative to linear peptides) to allow a higher binding affinity for targets. The invention provides fusions of a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention and other peptides, including known and random peptides. They can be fused in such a manner that the structure of the polypeptide, enzyme, protein, e.g. structural or binding proteins is not significantly perturbed and the peptide is metabolically or structurally conformationally stabilized. This allows the creation of a peptide library that is easily monitored both for its presence within cells and its quantity.","Amino acid sequence variants of the invention can be characterized by a predetermined nature of the variation, a feature that sets them apart from a naturally occurring form, e.g., an allelic or interspecies variation of a polypeptide, enzyme, protein, e.g. structural or binding protein, sequence. In one aspect, the variants of the invention exhibit the same qualitative biological activity as the naturally occurring analogue. Alternatively, the variants can be selected for having modified characteristics. In one aspect, while the site or region for introducing an amino acid sequence variation is predetermined, the mutation per se need not be predetermined. For example, in order to optimize the performance of a mutation at a given site, random mutagenesis may be conducted at the target codon or region and the expressed a polypeptide, enzyme, protein, e.g. structural or binding protein, variants screened for the optimal combination of desired activity. Techniques for making substitution mutations at predetermined sites in DNA having a known sequence are well known, as discussed herein for example, M13 primer mutagenesis and PCR mutagenesis. Screening of the mutants can be done using, e.g., assays of glucan hydrolysis. In alternative aspects, amino acid substitutions can be single residues; insertions can be on the order of from about 1 to 20 amino acids, although considerably larger insertions can be done. Deletions can range from about 1 to about 20, 30, 40, 50, 60, 70 residues or more. To obtain a final derivative with the optimal properties, substitutions, deletions, insertions or any combination thereof may be used. Generally, these changes are done on a few amino acids to minimize the alteration of the molecule. However, larger changes may be tolerated in certain circumstances.","The invention provides a polypeptide, enzyme, protein, e.g. structural or binding protein, where the structure of the polypeptide backbone, the secondary or the tertiary structure, e.g., an alpha-helical or beta-sheet structure, has been modified. In one aspect, the charge or hydrophobicity has been modified. In one aspect, the bulk of a side chain has been modified. Substantial changes in function or immunological identity are made by selecting substitutions that are less conservative. For example, substitutions can be made which more significantly affect: the structure of the polypeptide backbone in the area of the alteration, for example a alpha-helical or a beta-sheet structure; a charge or a hydrophobic site of the molecule, which can be at an active site; or a side chain. The invention provides substitutions in polypeptide of the invention where (a) a hydrophilic residues, e.g. seryl or threonyl, is substituted for (or by) a hydrophobic residue, e.g. leucyl, isoleucyl, phenylalanyl, valyl or alanyl; (b) a cysteine or proline is substituted for (or by) any other residue; (c) a residue having an electropositive side chain, e.g. lysyl, arginyl, or histidyl, is substituted for (or by) an electronegative residue, e.g. glutamyl or aspartyl; or (d) a residue having a bulky side chain, e.g. phenylalanine, is substituted for (or by) one not having a side chain, e.g. glycine. The variants can exhibit the same qualitative biological activity (i.e., an enzyme, structural or binding activity) although variants can be selected to modify the characteristics of the polypeptide, enzyme, protein, e.g. structural or binding proteins as needed.","In one aspect, a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention comprise epitopes or purification tags, signal sequences or other fusion sequences, etc. In one aspect, the polypeptide, enzyme, protein, e.g. structural or binding proteins of the invention can be fused to a random peptide to form a fusion polypeptide. By \u201cfused\u201d or \u201coperably linked\u201d herein is meant that the random peptide and the polypeptide, enzyme, protein, e.g. structural or binding protein are linked together, in such a manner as to minimize the disruption to the stability of the polypeptide, enzyme, protein, e.g. structural or binding protein structure, e.g., it retains a polypeptide, enzyme, protein, e.g. structural or binding protein, activity. The fusion polypeptide (or fusion polynucleotide encoding the fusion polypeptide) can comprise further components as well, including multiple peptides at multiple loops.","In one aspect, the peptides and nucleic acids encoding them are randomized, either fully randomized or they are biased in their randomization, e.g. in nucleotide\/residue frequency generally or per position. \u201cRandomized\u201d means that each nucleic acid and peptide consists of essentially random nucleotides and amino acids, respectively. In one aspect, the nucleic acids which give rise to the peptides can be chemically synthesized, and thus may incorporate any nucleotide at any position. Thus, when the nucleic acids are expressed to form peptides, any amino acid residue may be incorporated at any position. The synthetic process can be designed to generate randomized nucleic acids, to allow the formation of all or most of the possible combinations over the length of the nucleic acid, thus forming a library of randomized nucleic acids. The library can provide a sufficiently structurally diverse population of randomized expression products to affect a probabilistically sufficient range of cellular responses to provide one or more cells exhibiting a desired response. Thus, the invention provides an interaction library large enough so that at least one of its members will have a structure that gives it affinity for some molecule, protein, or other factor.","In one aspect, a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention is a multidomain enzyme that comprises a signal peptide, a carbohydrate binding module, a polypeptide, enzyme, protein, e.g. structural or binding protein, catalytic domain, a linker and\/or another catalytic domain.","The invention provides a means for generating chimeric polypeptides which may encode biologically active hybrid polypeptides (e.g., hybrid a polypeptide, enzyme, protein, e.g. structural or binding protein). In one aspect, the original polynucleotides encode biologically active polypeptides. The method of the invention produces new hybrid polypeptides by utilizing cellular processes which integrate the sequence of the original polynucleotides such that the resulting hybrid polynucleotide encodes a polypeptide demonstrating activities derived from the original biologically active polypeptides. For example, the original polynucleotides may encode a particular enzyme from different microorganisms. An enzyme encoded by a first polynucleotide from one organism or variant may, for example, function effectively under a particular environmental condition, e.g. high salinity. An enzyme encoded by a second polynucleotide from a different organism or variant may function effectively under a different environmental condition, such as extremely high temperatures. A hybrid polynucleotide containing sequences from the first and second original polynucleotides may encode an enzyme which exhibits characteristics of both enzymes encoded by the original polynucleotides. Thus, the enzyme encoded by the hybrid polynucleotide may function effectively under environmental conditions shared by each of the enzymes encoded by the first and second polynucleotides, e.g., high salinity and extreme temperatures.","A hybrid polypeptide resulting from the method of the invention may exhibit specialized enzyme activity not displayed in the original enzymes. For example, following recombination and\/or reductive reassortment of polynucleotides encoding a polypeptide, enzyme, protein, e.g. structural or binding protein, the resulting hybrid polypeptide encoded by a hybrid polynucleotide can be screened for specialized non-enzyme, non-structural or non-binding activities, obtained from each of the original enzymes. Thus, for example, the hybrid polypeptide may be screened to ascertain those chemical functionalities which distinguish the hybrid polypeptide from the original parent polypeptides, such as the temperature, pH or salt concentration at which the hybrid polypeptide functions.","In one aspect, the invention relates to a method for producing a biologically active hybrid polypeptide and screening such a polypeptide for enhanced activity by:\n\n","Isolating and Discovering a Polypeptide, Enzyme, Protein, e.g. Structural or Binding Protein","The invention provides methods for isolating and discovering a polypeptide, enzyme, protein, e.g. structural or binding protein, and the nucleic acids that encode them. Polynucleotides or enzymes may be isolated from individual organisms (\u201cisolates\u201d), collections of organisms that have been grown in defined media (\u201cenrichment cultures\u201d), or, uncultivated organisms (\u201cenvironmental samples\u201d). The organisms can be isolated by, e.g., in vivo biopanning (see discussion, below). The use of a culture-independent approach to derive polynucleotides encoding novel bioactivities from environmental samples is most preferable since it allows one to access untapped resources of biodiversity. Polynucleotides or enzymes also can be isolated from any one of numerous organisms, e.g. bacteria. In addition to whole cells, polynucleotides or enzymes also can be isolated from crude enzyme preparations derived from cultures of these organisms, e.g., bacteria.","\u201cEnvironmental libraries\u201d are generated from environmental samples and represent the collective genomes of naturally occurring organisms archived in cloning vectors that can be propagated in suitable prokaryotic hosts. Because the cloned DNA is initially extracted directly from environmental samples, the libraries are not limited to the small fraction of prokaryotes that can be grown in pure culture. Additionally, a normalization of the environmental DNA present in these samples could allow more equal representation of the DNA from all of the species present in the original sample. This can dramatically increase the efficiency of finding interesting genes from minor constituents of the sample which may be under-represented by several orders of magnitude compared to the dominant species.","For example, gene libraries generated from one or more uncultivated microorganisms are screened for an activity of interest. Potential pathways encoding bioactive molecules of interest are first captured in prokaryotic cells in the form of gene expression libraries. Polynucleotides encoding activities of interest are isolated from such libraries and introduced into a host cell. The host cell is grown under conditions which promote recombination and\/or reductive reassortment creating potentially active biomolecules with novel or enhanced activities.","In vivo biopanning may be performed utilizing a FACS-based and non-optical (e.g., magnetic) based machines. Complex gene libraries are constructed with vectors which contain elements which stabilize transcribed RNA. For example, the inclusion of sequences which result in secondary structures such as hairpins which are designed to flank the transcribed regions of the RNA would serve to enhance their stability, thus increasing their half life within the cell. The probe molecules used in the biopanning process consist of oligonucleotides labeled with reporter molecules that only fluoresce upon binding of the probe to a target molecule. These probes are introduced into the recombinant cells from the library using one of several transformation methods. The probe molecules bind to the transcribed target mRNA resulting in DNA\/RNA heteroduplex molecules. Binding of the probe to a target will yield a fluorescent signal which is detected and sorted by the FACS machine during the screening process.","Additionally, subcloning may be performed to further isolate sequences of interest. In subcloning, a portion of DNA is amplified, digested, generally by restriction enzymes, to cut out the desired sequence, the desired sequence is ligated into a recipient vector and is amplified. At each step in subcloning, the portion is examined for the activity of interest, in order to ensure that DNA that encodes the structural protein has not been excluded. The insert may be purified at any step of the subcloning, for example, by gel electrophoresis prior to ligation into a vector or where cells containing the recipient vector and cells not containing the recipient vector are placed on selective media containing, for example, an antibiotic, which will kill the cells not containing the recipient vector. Specific methods of subcloning cDNA inserts into vectors are well-known in the art (Sambrook et al., Molecular Cloning: A Laboratory Manual, 2nd Ed., Cold Spring Harbor Laboratory Press (1989)). In another aspect, the enzymes of the invention are subclones. Such subclones may differ from the parent clone by, for example, length, a mutation, a tag or a label.","In one aspect, the signal sequences of the invention are identified following identification of a novel polypeptide, enzyme, protein, e.g. structural or binding protein. The pathways by which proteins are sorted and transported to their proper cellular location are often referred to as protein targeting pathways. One of the most important elements in all of these targeting systems is a short amino acid sequence at the amino terminus of a newly synthesized polypeptide called the signal sequence. This signal sequence directs a protein to its appropriate location in the cell and is removed during transport or when the protein reaches its final destination. Most lysosomal, membrane, or secreted proteins have an amino-terminal signal sequence that marks them for translocation into the lumen of the endoplasmic reticulum. More than 100 signal sequences for proteins in this group have been determined. The sequences vary in length from 13 to 36 amino acid residues. Various methods of recognition of signal sequences are known to those of skill in the art. In one aspect, the peptides are identified by a method referred to as SignalP. SignalP uses a combined neural network which recognizes both signal peptides and their cleavage sites. See, e.g., Nielsen (1997) \u201cIdentification of prokaryotic and eukaryotic signal peptides and prediction of their cleavage sites.\u201d Protein Engineering, vol. 10, no. 1, p. 1-6. It should be understood that some of the polypeptide, enzyme, protein, e.g. structural or binding proteins of the invention may or may not contain signal sequences. It may be desirable to include a nucleic acid sequence encoding a signal sequence from a polypeptide, enzyme, protein, e.g. structural or binding protein, operably linked to a nucleic acid sequence of a different a polypeptide, enzyme, protein, e.g. structural or binding protein may be desired.","The microorganisms from which the polynucleotide may be discovered, isolated or prepared include prokaryotic microorganisms, such as and and lower eukaryotic microorganisms such as fungi, some algae and protozoa. Polynucleotides may be discovered, isolated or prepared from environmental samples in which case the nucleic acid may be recovered without culturing of an organism or recovered from one or more cultured organisms. In one aspect, such microorganisms may be extremophiles, such as hyperthermophiles, psychrophiles, psychrotrophs, halophiles, barophiles and acidophiles. Polynucleotides encoding enzymes isolated from extremophilic microorganisms can be used. Such enzymes may function at temperatures above 100\u00b0 C. in terrestrial hot springs and deep searmal vents, at temperatures below 0\u00b0 C. in arctic waters, in the saturated salt environment of the Dead Sea, at pH values around 0 in coal deposits and geothermal sulfur-rich springs, or at pH values greater than 11 in sewage sludge. For example, several esterases and lipases cloned and expressed from extremophilic organisms show high activity throughout a wide range of temperatures and pHs.","Polynucleotides selected and isolated as hereinabove described are introduced into a suitable host cell. A suitable host cell is any cell which is capable of promoting recombination and\/or reductive reassortment. The selected polynucleotides are in one aspect already in a vector which includes appropriate control sequences. The host cell can be a higher eukaryotic cell, such as a mammalian cell, or a lower eukaryotic cell, such as a yeast cell, or in one aspect, the host cell can be a prokaryotic cell, such as a bacterial cell. Introduction of the construct into the host cell can be effected by calcium phosphate transfection, DEAE-Dextran mediated transfection, or electroporation (Davis et al., 1986).","As representative examples of appropriate hosts, there may be mentioned: bacterial cells, such as ; fungal cells, such as yeast; insect cells such as S2 and Sf9; animal cells such as CHO, COS or Bowes melanoma; adenoviruses; and plant cells. The selection of an appropriate host is deemed to be within the scope of those skilled in the art from the teachings herein.","With particular references to various mammalian cell culture systems that can be employed to express recombinant protein, examples of mammalian expression systems include the COS-7 lines of monkey kidney fibroblasts, described in \u201cSV40-transformed simian cells support the replication of early SV40 mutants\u201d (Gluzman, 1981) and other cell lines capable of expressing a compatible vector, for example, the C127, 3T3, CHO, HeLa and BHK cell lines. Mammalian expression vectors will comprise an origin of replication, a suitable promoter and enhancer and also any necessary ribosome binding sites, polyadenylation site, splice donor and acceptor sites, transcriptional termination sequences and 5\u2032 flanking nontranscribed sequences. DNA sequences derived from the SV40 splice and polyadenylation sites may be used to provide the required nontranscribed genetic elements.","In another aspect, it is envisioned the method of the present invention can be used to generate novel polynucleotides encoding biochemical pathways from one or more operons or gene clusters or portions thereof. For example, bacteria and many eukaryotes have a coordinated mechanism for regulating genes whose products are involved in related processes. The genes are clustered, in structures referred to as \u201cgene clusters,\u201d on a single chromosome and are transcribed together under the control of a single regulatory sequence, including a single promoter which initiates transcription of the entire cluster. Thus, a gene cluster is a group of adjacent genes that are either identical or related, usually as to their function. An example of a biochemical pathway encoded by gene clusters are polyketides.","Gene cluster DNA can be isolated from different organisms and ligated into vectors, particularly vectors containing expression regulatory sequences which can control and regulate the production of a detectable protein or protein-related array activity from the ligated gene clusters. Use of vectors which have an exceptionally large capacity for exogenous DNA introduction are particularly appropriate for use with such gene clusters and are described by way of example herein to include the f-factor (or fertility factor) of . This f-factor of is a plasmid which affects high-frequency transfer of itself during conjugation and is ideal to achieve and stably propagate large DNA fragments, such as gene clusters from mixed microbial samples. One aspect is to use cloning vectors, referred to as \u201cfosmids\u201d or bacterial artificial chromosome (BAC) vectors. These are derived from f-factor which is able to stably integrate large segments of genomic DNA. When integrated with DNA from a mixed uncultured environmental sample, this makes it possible to achieve large genomic fragments in the form of a stable \u201cenvironmental DNA library.\u201d Another type of vector for use in the present invention is a cosmid vector. Cosmid vectors were originally designed to clone and propagate large segments of genomic DNA. Cloning into cosmid vectors is described in detail in Sambrook et al., Molecular Cloning: A Laboratory Manual, 2nd Ed., Cold Spring Harbor Laboratory Press (1989). Once ligated into an appropriate vector, two or more vectors containing different polyketide synthase gene clusters can be introduced into a suitable host cell. Regions of partial sequence homology shared by the gene clusters will promote processes which result in sequence reorganization resulting in a hybrid gene cluster. The novel hybrid gene cluster can then be screened for enhanced activities not found in the original gene clusters.","Methods for screening for various enzyme activities are known to those of skill in the art and are discussed throughout the present specification. Such methods may be employed when isolating the polypeptides and polynucleotides of the invention.","In one aspect, the invention provides methods for discovering and isolating polypeptides, enzymes, proteins, e.g. structural or binding proteins or compounds to modify the enzymatic activity, using a whole cell approach. Putative clones encoding polypeptides, enzymes, proteins, e.g. structural or binding proteins from genomic DNA library can be screened.","Screening Methodologies and \u201cOn-Line\u201d Monitoring Devices","In practicing the methods of the invention, a variety of apparatus and methodologies can be used to in conjunction with the polypeptides and nucleic acids of the invention, e.g., to screen polypeptides for a polypeptide, enzyme, protein, e.g. structural or binding protein, activity, to screen compounds as potential modulators, e.g., activators or inhibitors, of an enzyme, structural or binding activity, for antibodies that bind to a polypeptide of the invention, for nucleic acids that hybridize to a nucleic acid of the invention, to screen for cells expressing a polypeptide of the invention and the like. In addition to the array formats described in detail below for screening samples, alternative formats can also be used to practice the methods of the invention. Such formats include, for example, mass spectrometers, chromatographs, e.g., high-throughput HPLC and other forms of liquid chromatography, and smaller formats, such as 1536-well plates, 384-well plates and so on. High throughput screening apparatus can be adapted and used to practice the methods of the invention, see, e.g., U.S. Patent Application No. 20020001809.","Capillary Arrays","Nucleic acids or polypeptides of the invention can be immobilized to or applied to an array. Arrays can be used to screen for or monitor libraries of compositions (e.g., small molecules, antibodies, nucleic acids, etc.) for their ability to bind to or modulate the activity of a nucleic acid or a polypeptide of the invention. Capillary arrays, such as the GIGAMATRIX\u2122, Diversa Corporation, San Diego, Calif.; and arrays described in, e.g., U.S. Patent Application No. 20020080350 A1; WO 0231203 A; WO 0244336 A, provide an alternative apparatus for holding and screening samples. In one aspect, the capillary array includes a plurality of capillaries formed into an array of adjacent capillaries, wherein each capillary comprises at least one wall defining a lumen for retaining a sample. The lumen may be cylindrical, square, hexagonal or any other geometric shape so long as the walls form a lumen for retention of a liquid or sample. The capillaries of the capillary array can be held together in close proximity to form a planar structure. The capillaries can be bound together, by being fused (e.g., where the capillaries are made of glass), glued, bonded, or clamped side-by-side. Additionally, the capillary array can include interstitial material disposed between adjacent capillaries in the array, thereby forming a solid planar device containing a plurality of through-holes.","A capillary array can be formed of any number of individual capillaries, for example, a range from 100 to 4,000,000 capillaries. Further, a capillary array having about 100,000 or more individual capillaries can be formed into the standard size and shape of a Microtiter\u00ae plate for fitment into standard laboratory equipment. The lumens are filled manually or automatically using either capillary action or microinjection using a thin needle. Samples of interest may subsequently be removed from individual capillaries for further analysis or characterization. For example, a thin, needle-like probe is positioned in fluid communication with a selected capillary to either add or withdraw material from the lumen.","In a single-pot screening assay, the assay components are mixed yielding a solution of interest, prior to insertion into the capillary array. The lumen is filled by capillary action when at least a portion of the array is immersed into a solution of interest. Chemical or biological reactions and\/or activity in each capillary are monitored for detectable events. A detectable event is often referred to as a \u201chit\u201d, which can usually be distinguished from \u201cnon-hit\u201d producing capillaries by optical detection. Thus, capillary arrays allow for massively parallel detection of \u201chits\u201d.","In a multi-pot screening assay, a polypeptide or nucleic acid, e.g., a ligand, can be introduced into a first component, which is introduced into at least a portion of a capillary of a capillary array. An air bubble can then be introduced into the capillary behind the first component. A second component can then be introduced into the capillary, wherein the second component is separated from the first component by the air bubble. The first and second components can then be mixed by applying hydrostatic pressure to both sides of the capillary array to collapse the bubble. The capillary array is then monitored for a detectable event resulting from reaction or non-reaction of the two components.","In a binding screening assay, a sample of interest can be introduced as a first liquid labeled with a detectable particle into a capillary of a capillary array, wherein the lumen of the capillary is coated with a binding material for binding the detectable particle to the lumen. The first liquid may then be removed from the capillary tube, wherein the bound detectable particle is maintained within the capillary, and a second liquid may be introduced into the capillary tube. The capillary is then monitored for a detectable event resulting from reaction or non-reaction of the particle with the second liquid.","Arrays, or \u201cBiochips\u201d","Nucleic acids or polypeptides of the invention can be immobilized to or applied to an array. Arrays can be used to screen for or monitor libraries of compositions (e.g., small molecules, antibodies, nucleic acids, etc.) for their ability to bind to or modulate the activity of a nucleic acid or a polypeptide of the invention. For example, in one aspect of the invention, a monitored parameter is transcript expression of a polypeptide, enzyme, protein, e.g. structural or binding protein, gene. One or more, or, all the transcripts of a cell can be measured by hybridization of a sample comprising transcripts of the cell, or, nucleic acids representative of or complementary to transcripts of a cell, by hybridization to immobilized nucleic acids on an array, or \u201cbiochip.\u201d By using an \u201carray\u201d of nucleic acids on a microchip, some or all of the transcripts of a cell can be simultaneously quantified. Alternatively, arrays comprising genomic nucleic acid can also be used to determine the genotype of a newly engineered strain made by the methods of the invention. Polypeptide arrays\u201d can also be used to simultaneously quantify a plurality of proteins. The present invention can be practiced with any known \u201carray,\u201d also referred to as a \u201cmicroarray\u201d or \u201cnucleic acid array\u201d or \u201cpolypeptide array\u201d or \u201cantibody array\u201d or \u201cbiochip,\u201d or variation thereof. Arrays are generically a plurality of \u201cspots\u201d or \u201ctarget elements,\u201d each target element comprising a defined amount of one or more biological molecules, e.g., oligonucleotides, immobilized onto a defined area of a substrate surface for specific binding to a sample molecule, e.g., mRNA transcripts.","The terms \u201carray\u201d or \u201cmicroarray\u201d or \u201cbiochip\u201d or \u201cchip\u201d as used herein is a plurality of target elements, each target element comprising a defined amount of one or more polypeptides (including antibodies) or nucleic acids immobilized onto a defined area of a substrate surface, as discussed in further detail, below.","In practicing the methods of the invention, any known array and\/or method of making and using arrays can be incorporated in whole or in part, or variations thereof, as described, for example, in U.S. Pat. Nos. 6,277,628; 6,277,489; 6,261,776; 6,258,606; 6,054,270; 6,048,695; 6,045,996; 6,022,963; 6,013,440; 5,965,452; 5,959,098; 5,856,174; 5,830,645; 5,770,456; 5,632,957; 5,556,752; 5,143,854; 5,807,522; 5,800,992; 5,744,305; 5,700,637; 5,556,752; 5,434,049; see also, e.g., WO 99\/51773; WO 99\/09217; WO 97\/46313; WO 96\/17958; see also, e.g., Johnston (1998) Curr. Biol. 8:R171-R174; Schummer (1997) Biotechniques 23:1087-1092; Kern (1997) Biotechniques 23:120-124; Solinas-Toldo (1997) Genes, Chromosomes & Cancer 20:399-407; Bowtell (1999) Nature Genetics Supp. 21:25-32. See also published U.S. patent applications Nos. 20010018642; 20010019827; 20010016322; 20010014449; 20010014448; 20010012537; 20010008765.","Antibodies and Antibody-Based Screening Methods","The invention provides isolated or recombinant antibodies that specifically bind to a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention. These antibodies can be used to isolate, identify or quantify the polypeptide, enzyme, protein, e.g. structural or binding proteins of the invention or related polypeptides. These antibodies can be used to isolate other polypeptides within the scope the invention or other related a polypeptide, enzyme, protein, e.g. structural or binding protein. The antibodies can be designed to bind to an active site of a polypeptide, enzyme, protein, e.g. structural or binding protein. Thus, the invention provides methods of inhibiting a polypeptide, enzyme, protein, e.g. structural or binding protein, using the antibodies of the invention (see discussion above regarding applications for anti-polypeptide, anti-enzyme, anti-protein, e.g., anti-structural or anti-binding protein compositions of the invention).","The invention provides fragments of the enzymes of the invention, including immunogenic fragments of a polypeptide of the invention. The invention provides compositions comprising a polypeptide or peptide of the invention and adjuvants or carriers and the like.","The term \u201cantibody\u201d includes a peptide or polypeptide derived from, modeled after or substantially encoded by an immunoglobulin gene or immunoglobulin genes, or fragments thereof, capable of specifically binding an antigen or epitope, see, e.g. Fundamental Immunology, Third Edition, W. E. Paul, ed., Raven Press, N.Y. (1993); Wilson (1994) J. Immunol. Methods 175:267-273; Yarmush (1992) J. Biochem. Biophys. Methods 25:85-97. The term antibody includes antigen-binding portions, i.e., \u201cantigen binding sites,\u201d (e.g., fragments, subsequences, complementarity determining regions (CDRs)) that retain capacity to bind antigen, including (i) a Fab fragment, a monovalent fragment consisting of the VL, VH, CL and CH1 domains; (ii) a F(ab\u2032)2 fragment, a bivalent fragment comprising two Fab fragments linked by a disulfide bridge at the hinge region; (iii) a Fd fragment consisting of the VH and CH1 domains; (iv) a Fv fragment consisting of the VL and VH domains of a single arm of an antibody, (v) a dAb fragment (Ward et al., (1989) Nature 341:544-546), which consists of a VH domain; and (vi) an isolated complementarity determining region (CDR). Single chain antibodies are also included by reference in the term \u201cantibody.\u201d","The antibodies can be used in immunoprecipitation, staining, immunoaffinity columns, and the like. If desired, nucleic acid sequences encoding for specific antigens can be generated by immunization followed by isolation of polypeptide or nucleic acid, amplification or cloning and immobilization of polypeptide onto an array of the invention. Alternatively, the methods of the invention can be used to modify the structure of an antibody produced by a cell to be modified, e.g., an antibody's affinity can be increased or decreased. Furthermore, the ability to make or modify antibodies can be a phenotype engineered into a cell by the methods of the invention.","Methods of immunization, producing and isolating antibodies (polyclonal and monoclonal) are known to those of skill in the art and described in the scientific and patent literature, see, e.g., Coligan, CURRENT PROTOCOLS IN IMMUNOLOGY, Wiley\/Greene, NY (1991); Stites (eds.) BASIC AND CLINICAL IMMUNOLOGY (7th ed.) Lange Medical Publications, Los Altos, Calif. (\u201cStites\u201d); Goding, MONOCLONAL ANTIBODIES: PRINCIPLES AND PRACTICE (2d ed.) Academic Press, New York, N.Y. (1986); Kohler (1975) Nature 256:495; Harlow (1988) ANTIBODIES, A LABORATORY MANUAL, Cold Spring Harbor Publications, New York. Antibodies also can be generated in vitro, e.g., using recombinant antibody binding site expressing phage display libraries, in addition to the traditional in vivo methods using animals. See, e.g., Hoogenboom (1997) Trends Biotechnol. 15:62-70; Katz (1997) Annu Rev. Biophys. Biomol. Struct. 26:27-45.","The polypeptides of the invention or fragments comprising at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 consecutive amino acids thereof, may also be used to generate antibodies which bind specifically to the polypeptides or fragments. The resulting antibodies may be used in immunoaffinity chromatography procedures to isolate or purify the polypeptide or to determine whether the polypeptide is present in a biological sample. In such procedures, a protein preparation, such as an extract, or a biological sample is contacted with an antibody capable of specifically binding to one of the polypeptides of the invention, or fragments comprising at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 consecutive amino acids thereof.","In immunoaffinity procedures, the antibody is attached to a solid support, such as a bead or other column matrix. The protein preparation is placed in contact with the antibody under conditions in which the antibody specifically binds to one of the polypeptides of the invention, or fragment thereof. After a wash to remove non-specifically bound proteins, the specifically bound polypeptides are eluted.","The ability of proteins in a biological sample to bind to the antibody may be determined using any of a variety of procedures familiar to those skilled in the art. For example, binding may be determined by labeling the antibody with a detectable label such as a fluorescent agent, an enzymatic label, or a radioisotope. Alternatively, binding of the antibody to the sample may be detected using a secondary antibody having such a detectable label thereon. Particular assays include ELISA assays, sandwich assays, radioimmunoassays and Western Blots.","Polyclonal antibodies generated against the polypeptides of the invention, or fragments comprising at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 consecutive amino acids thereof can be obtained by direct injection of the polypeptides into an animal or by administering the polypeptides to an animal, for example, a nonhuman. The antibody so obtained can bind the polypeptide itself. In this manner, even a sequence encoding only a fragment of the polypeptide can be used to generate antibodies which may bind to the whole native polypeptide. Such antibodies can then be used to isolate the polypeptide from cells expressing that polypeptide.","For preparation of monoclonal antibodies, any technique which provides antibodies produced by continuous cell line cultures can be used. Examples include the hybridoma technique (Kohler and Milstein, Nature, 256:495-497, 1975), the trioma technique, the human B-cell hybridoma technique (Kozbor et al., Immunology Today 4:72, 1983) and the EBV-hybridoma technique (Cole, et al., 1985, in Monoclonal Antibodies and Cancer Therapy, Alan R. Liss, Inc., pp. 77-96).","Techniques described for the production of single chain antibodies (U.S. Pat. No. 4,946,778) can be adapted to produce single chain antibodies to the polypeptides of the invention, or fragments comprising at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 consecutive amino acids thereof. Alternatively, transgenic mice may be used to express humanized antibodies to these polypeptides or fragments thereof.","Antibodies generated against the polypeptides of the invention, or fragments comprising at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 consecutive amino acids thereof may be used in screening for similar polypeptides from other organisms and samples. In such techniques, polypeptides from the organism are contacted with the antibody and those polypeptides which specifically bind the antibody are detected. Any of the procedures described above may be used to detect antibody binding. One such screening assay is described in \u201cMethods for Measuring Cellulase Activities\u201d, , Vol 160, pp. 87-116.","Kits","The invention provides kits comprising the compositions, e.g., nucleic acids, expression cassettes, vectors, cells, transgenic seeds or plants or plant parts, polypeptides and\/or antibodies of the invention. The kits also can contain instructional material teaching the methodologies and uses of the enzymes of the invention (see, e.g., Table 3), including the industrial, experimental, food and feed processing and medical uses of the compositions and methods of the invention, as described herein.","Whole Cell Engineering and Measuring Metabolic Parameters","The methods of the invention provide whole cell evolution, or whole cell engineering, of a cell to develop a new cell strain having a new phenotype, e.g., a new or modified a polypeptide, enzyme, protein, e.g. structural or binding protein, activity, by modifying the genetic composition of the cell. The genetic composition can be modified by addition to the cell of a nucleic acid of the invention, e.g., a coding sequence for an enzyme of the invention. See, e.g., WO0229032; WO0196551.","To detect the new phenotype, at least one metabolic parameter of a modified cell is monitored in the cell in a \u201creal time\u201d or \u201con-line\u201d time frame. In one aspect, a plurality of cells, such as a cell culture, is monitored in \u201creal time\u201d or \u201con-line.\u201d In one aspect, a plurality of metabolic parameters is monitored in \u201creal time\u201d or \u201con-line.\u201d Metabolic parameters can be monitored using the polypeptide, enzyme, protein, e.g. structural or binding proteins of the invention.","Metabolic flux analysis (MFA) is based on a known biochemistry framework. A linearly independent metabolic matrix is constructed based on the law of mass conservation and on the pseudo-steady state hypothesis (PSSH) on the intracellular metabolites. In practicing the methods of the invention, metabolic networks are established, including the:\n\n","Once the metabolic network for a given strain is built, mathematic presentation by matrix notion can be introduced to estimate the intracellular metabolic fluxes if the on-line metabolome data is available. Metabolic phenotype relies on the changes of the whole metabolic network within a cell. Metabolic phenotype relies on the change of pathway utilization with respect to environmental conditions, genetic regulation, developmental state and the genotype, etc. In one aspect of the methods of the invention, after the on-line MFA calculation, the dynamic behavior of the cells, their phenotype and other properties are analyzed by investigating the pathway utilization. For example, if the glucose supply is increased and the oxygen decreased during the yeast fermentation, the utilization of respiratory pathways will be reduced and\/or stopped, and the utilization of the fermentative pathways will dominate. Control of physiological state of cell cultures will become possible after the pathway analysis. The methods of the invention can help determine how to manipulate the fermentation by determining how to change the substrate supply, temperature, use of inducers, etc. to control the physiological state of cells to move along desirable direction. In practicing the methods of the invention, the MFA results can also be compared with transcriptome and proteome data to design experiments and protocols for metabolic engineering or gene shuffling, etc.","In practicing the methods of the invention, any modified or new phenotype can be conferred and detected, including new or improved characteristics in the cell. Any aspect of metabolism or growth can be monitored.","Monitoring Expression of an mRNA Transcript","In one aspect of the invention, the engineered phenotype comprises increasing or decreasing the expression of an mRNA transcript (e.g., a polypeptide, enzyme, protein, e.g. structural or binding protein, message) or generating new (e.g., polypeptide, enzyme, protein, e.g. structural or binding protein) transcripts in a cell. This increased or decreased expression can be traced by testing for the presence of a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention or by a polypeptide, enzyme, protein, e.g. structural or binding protein, activity assays. Such assays are well known in the art. mRNA transcripts, or messages, also can be detected and quantified by any method known in the art, including, e.g., Northern blots, quantitative amplification reactions, hybridization to arrays, and the like. Quantitative amplification reactions include, e.g., quantitative PCR, including, e.g., quantitative reverse transcription polymerase chain reaction, or RT-PCR; quantitative real time RT-PCR, or \u201creal-time kinetic RT-PCR\u201d (see, e.g., Kreuzer (2001) Br. J. Haematol. 114:313-318; Xia (2001) Transplantation 72:907-914).","In one aspect of the invention, the engineered phenotype is generated by knocking out expression of a homologous gene. The gene's coding sequence or one or more transcriptional control elements can be knocked out, e.g., promoters or enhancers. Thus, the expression of a transcript can be completely ablated or only decreased.","In one aspect of the invention, the engineered phenotype comprises increasing the expression of a homologous gene. This can be effected by knocking out of a negative control element, including a transcriptional regulatory element acting in cis- or trans-, or, mutagenizing a positive control element. One or more, or, all the transcripts of a cell can be measured by hybridization of a sample comprising transcripts of the cell, or, nucleic acids representative of or complementary to transcripts of a cell, by hybridization to immobilized nucleic acids on an array.","Monitoring Expression of a Polypeptides, Peptides and Amino Acids","In one aspect of the invention, the engineered phenotype comprises increasing or decreasing the expression of a polypeptide (e.g., a polypeptide, enzyme, protein, e.g. structural or binding protein) or generating new polypeptides in a cell. This increased or decreased expression can be traced by determining the amount of a polypeptide, enzyme, protein, e.g. structural or binding protein, present or by a polypeptide, enzyme, protein, e.g. structural or binding protein, activity assays.","Polypeptides, peptides and amino acids also can be detected and quantified by any method known in the art, including, e.g., nuclear magnetic resonance (NMR), spectrophotometry, radiography (protein radiolabeling), electrophoresis, capillary electrophoresis, high performance liquid chromatography (HPLC), thin layer chromatography (TLC), hyperdiffusion chromatography, various immunological methods, e.g. immunoprecipitation, immunodiffusion, immuno-electrophoresis, radioimmunoassays (RIAs), enzyme-linked immunosorbent assays (ELISAs), immuno-fluorescent assays, gel electrophoresis (e.g., SDS-PAGE), staining with antibodies, fluorescent activated cell sorter (FACS), pyrolysis mass spectrometry, Fourier-Transform Infrared Spectrometry, Raman spectrometry, GC-MS, and LC-Electrospray and cap-LC-tandem-electrospray mass spectrometries, and the like. Novel bioactivities can also be screened using methods, or variations thereof, described in U.S. Pat. No. 6,057,103. Furthermore, as discussed below in detail, one or more, or, all the polypeptides of a cell can be measured using a protein array.","Pharmaceutical Compositions and Dietary Supplements","The invention provides pharmaceutical compositions, e.g., formulations, comprising a composition (including polypeptide, nucleic acid, or antibody) of the invention and a pharmaceutically acceptable excipient. The invention provides enteral and parenteral formulations comprising compositions of the invention. For example, the invention provides oral formulations (including or dietary supplements) comprising a composition of the invention. The invention provides formulations and methods for treating, ameliorating, diagnosing or preventing disease of condition of interest; e.g., in one aspect the invention provides methods comprising providing a pharmaceutical composition or dietary supplement comprising a composition of the invention; and administering an effective amount of the pharmaceutical composition or dietary supplement to a subject in need thereof.","The compositions and methods of the invention can also be practiced ex vivo or in vitro, or on a non-biological fluid or substance. In one aspect, the compositions and methods comprise providing a pharmaceutical composition or dietary supplement comprising a formulation of the invention; and administering an effective amount of the pharmaceutical composition or dietary supplement to a subject in need thereof.","The pharmaceutical compositions and dietary supplements used in the methods of the invention can be administered by any means known in the art, e.g., parenterally, topically, orally, or by local administration, such as by aerosol or transdermally. The compositions and dietary supplements of the invention can be formulated as a tablet, gel, geltab, pill, implant, liquid, spray, powder, food, feed pellet, as an injectable formulation or as an encapsulated formulation. The pharmaceutical compositions and dietary supplements can be formulated in any way and can be administered in a variety of unit dosage forms depending upon the condition or disease and the degree of illness, the general medical condition of each patient, the resulting preferred method of administration and the like. Details on techniques for formulation and administration are well described in the scientific and patent literature, see, e.g., the latest edition of Remington's Pharmaceutical Sciences, Maack Publishing Co, Easton Pa. (\u201cRemington's\u201d) (e.g., Remington, The Science and Practice of Pharmacy, 21st Edition, by University of the Sciences in Philadelphia, Editor).","Pharmaceutical formulations and dietary supplements can be prepared according to any method known to the art for the manufacture of pharmaceuticals and dietary supplements. Such drugs and dietary supplements can contain sweetening agents, flavoring agents, coloring agents and preserving agents. A formulation (which includes \u201cdietary supplements\u201d) can be admixtured with nontoxic pharmaceutically or orally acceptable excipients which are suitable for manufacture. Formulations may comprise one or more diluents, emulsifiers, preservatives, buffers, excipients, etc. and may be provided in such forms as liquids, powders, emulsions, lyophilized powders, sprays, creams, lotions, controlled release formulations, tablets, pills, gels, on patches, in implants, etc.","Pharmaceutical formulations and dietary supplements for oral administration can be formulated using pharmaceutically acceptable carriers well known in the art in appropriate and suitable dosages. Such carriers enable the pharmaceuticals and dietary supplements to be formulated in unit dosage forms as tablets, pills, powder, dragees, capsules, liquids, lozenges, gels, syrups, slurries, suspensions, etc., suitable for ingestion by the patient. Pharmaceutical preparations and dietary supplements for oral use can be formulated as a solid excipient, optionally grinding a resulting mixture, and processing the mixture of granules, after adding suitable additional compounds, if desired, to obtain tablets or dragee cores. Suitable solid excipients are carbohydrate or protein fillers include, e.g., sugars, including lactose, sucrose, mannitol, or sorbitol; starch from corn, wheat, rice, potato, or other plants; cellulose such as methyl cellulose, hydroxypropylmethyl-cellulose, or sodium carboxy-methylcellulose; and gums including arabic and tragacanth; and proteins, e.g., gelatin and collagen. Disintegrating or solubilizing agents may be added, such as the cross-linked polyvinyl pyrrolidone, agar, alginic acid, or a salt thereof, such as sodium alginate.","Dragee cores are provided with suitable coatings such as concentrated sugar solutions, which may also contain gum arabic, talc, polyvinylpyrrolidone, carbopol gel, polyethylene glycol, and\/or titanium dioxide, lacquer solutions, and suitable organic solvents or solvent mixtures. Dyestuffs or pigments may be added to the tablets or dragee coatings for product identification or to characterize the quantity of active compound (i.e., dosage). Pharmaceutical preparations and dietary supplements of the invention can also be used orally using, e.g., push-fit capsules made of gelatin, as well as soft, sealed capsules made of gelatin and a coating such as glycerol or sorbitol. Push-fit capsules can contain active agents mixed with a filler or binders such as lactose or starches, lubricants such as talc or magnesium stearate, and, optionally, stabilizers. In soft capsules, the active agents can be dissolved or suspended in suitable liquids, such as fatty oils, liquid paraffin, or liquid polyethylene glycol with or without stabilizers.","Aqueous suspensions can contain an active agent (e.g., an enzyme or peptidomimetic of the invention) in admixture with excipients suitable for the manufacture of aqueous suspensions. Such excipients include a suspending agent, such as sodium carboxymethylcellulose, methylcellulose, hydroxypropylmethylcellulose, sodium alginate, polyvinylpyrrolidone, gum tragacanth and gum acacia, and dispersing or wetting agents such as a naturally occurring phosphatide (e.g., lecithin), a condensation product of an alkylene oxide with a fatty acid (e.g., polyoxyethylene stearate), a condensation product of ethylene oxide with a long chain aliphatic alcohol (e.g., heptadecaethylene oxycetanol), a condensation product of ethylene oxide with a partial ester derived from a fatty acid and a hexitol (e.g., polyoxyethylene sorbitol mono-oleate), or a condensation product of ethylene oxide with a partial ester derived from fatty acid and a hexitol anhydride (e.g., polyoxyethylene sorbitan mono-oleate). The aqueous suspension can also contain one or more preservatives such as ethyl or n-propyl p-hydroxybenzoate, one or more coloring agents, one or more flavoring agents and one or more sweetening agents, such as sucrose, aspartame or saccharin. Formulations can be adjusted for osmolarity.","Oil-based pharmaceuticals are particularly useful for administration of hydrophobic formulations or active agents of the invention. Oil-based suspensions can be formulated by suspending an active agent (e.g., a composition of the invention) in a vegetable oil, such as arachis oil, olive oil, sesame oil or coconut oil, or in a mineral oil such as liquid paraffin; or a mixture of these. See e.g., U.S. Pat. No. 5,716,928 describing using essential oils or essential oil components for increasing bioavailability and reducing inter- and intra-individual variability of orally administered hydrophobic pharmaceutical compounds (see also U.S. Pat. No. 5,858,401). The oil suspensions can contain a thickening agent, such as beeswax, hard paraffin or cetyl alcohol. Sweetening agents can be added to provide a palatable oral preparation, such as glycerol, sorbitol or sucrose. These formulations and dietary supplements can be preserved by the addition of an antioxidant such as ascorbic acid. As an example of an injectable oil vehicle, see Minto (1997) J. Pharmacol. Exp. Ther. 281:93-102.","The pharmaceutical formulations and dietary supplements of the invention can also be in the form of oil-in-water emulsions. The oily phase can be a vegetable oil or a mineral oil, described above, or a mixture of these. Suitable emulsifying agents include naturally-occurring gums, such as gum acacia and gum tragacanth, naturally occurring phosphatides, such as soybean lecithin, esters or partial esters derived from fatty acids and hexitol anhydrides, such as sorbitan mono-oleate, and condensation products of these partial esters with ethylene oxide, such as polyoxyethylene sorbitan mono-oleate. The emulsion can also contain sweetening agents and flavoring agents, as in the formulation of syrups and elixirs. Such formulations can also contain a demulcent, a preservative, or a coloring agent.","In the methods of the invention, the pharmaceutical compounds and dietary supplements can also be administered by in intranasal, intraocular and intravaginal routes including suppositories, insufflation, powders and aerosol formulations (for examples of steroid inhalants, see Rohatagi (1995) J. Clin. Pharmacol. 35:1187-1193; Tjwa (1995) Ann. Allergy Asthma Immunol. 75:107-111). Suppositories formulations can be prepared by mixing the drug with a suitable non-irritating excipient which is solid at ordinary temperatures but liquid at body temperatures and will therefore melt in the body to release the drug. Such materials are cocoa butter and polyethylene glycols.","In the methods of the invention, the pharmaceutical compounds and dietary supplements can be delivered by transdermally, by a topical route, formulated as applicator sticks, solutions, suspensions, emulsions, gels, creams, ointments, pastes, jellies, paints, powders, and aerosols.","In the methods of the invention, the pharmaceutical compounds and dietary supplements can also be delivered as microspheres for slow release in the body. For example, microspheres can be administered via intradermal injection of drug which slowly release subcutaneously; see Rao (1995) J. Biomater Sci. Polym. Ed. 7:623-645; as biodegradable and injectable gel formulations, see, e.g., Gao (1995) Pharm. Res. 12:857-863 (1995); or, as microspheres for oral administration, see, e.g., Eyles (1997) J. Pharm. Pharmacol. 49:669-674.","In the methods of the invention, the pharmaceutical compounds can be parenterally administered, such as by intravenous (IV) administration or administration into a body cavity or lumen of an organ. These formulations can comprise a solution of active agent dissolved in a pharmaceutically acceptable carrier. Acceptable vehicles and solvents that can be employed are water and Ringer's solution, an isotonic sodium chloride. In addition, sterile fixed oils can be employed as a solvent or suspending medium. For this purpose any bland fixed oil can be employed including synthetic mono- or diglycerides. In addition, fatty acids such as oleic acid can likewise be used in the preparation of injectables. These solutions are sterile and generally free of undesirable matter. These formulations may be sterilized by conventional, well known sterilization techniques. The formulations may contain pharmaceutically acceptable auxiliary substances as required to approximate physiological conditions such as pH adjusting and buffering agents, toxicity adjusting agents, e.g., sodium acetate, sodium chloride, potassium chloride, calcium chloride, sodium lactate and the like. The concentration of active agent in these formulations can vary widely, and will be selected primarily based on fluid volumes, viscosities, body weight, and the like, in accordance with the particular mode of administration selected and the patient's needs. For IV administration, the formulation can be a sterile injectable preparation, such as a sterile injectable aqueous or oleaginous suspension. This suspension can be formulated using those suitable dispersing or wetting agents and suspending agents. The sterile injectable preparation can also be a suspension in a nontoxic parenterally-acceptable diluent or solvent, such as a solution of 1,3-butanediol. The administration can be by bolus or continuous infusion (e.g., substantially uninterrupted introduction into a blood vessel for a specified period of time).","The pharmaceutical compounds, formulations and dietary supplements of the invention can be lyophilized. The invention provides a stable lyophilized formulation comprising a composition of the invention, which can be made by lyophilizing a solution comprising a pharmaceutical of the invention and a bulking agent, e.g., mannitol, trehalose, raffinose, and sucrose or mixtures thereof. A process for preparing a stable lyophilized formulation can include the equivalent of lyophilizing a solution about 2.5 mg\/mL protein, about 15 mg\/mL sucrose, about 19 mg\/mL NaCl, and a sodium citrate buffer having a pH greater than 5.5 but less than 6.5. See, e.g., U.S. patent app. no. 20040028670.","The compositions (e.g., formulations, including dietary supplements) of the invention can be delivered by the use of liposomes. By using liposomes, particularly where the liposome surface carries ligands specific for target cells, or are otherwise preferentially directed to a specific organ, one can focus the delivery of the active agent into target cells in vivo. See, e.g., U.S. Pat. Nos. 6,063,400; 6,007,839; Al-Muhammed (1996) J. Microencapsul. 13:293-306; Chonn (1995) Curr. Opin. Biotechnol. 6:698-708; Ostro (1989) Am. J. Hosp. Pharm. 46:1576-1587.","The compositions (e.g., formulations, including dietary supplements) of the invention can be administered for prophylactic and\/or therapeutic treatments. In therapeutic applications, compositions are administered to a subject already suffering from a condition, infection or disease of interest in an amount sufficient to cure, alleviate or partially arrest the clinical manifestations of the condition, infection or disease and its complications (a \u201ctherapeutically effective amount\u201d). In the methods of the invention, a pharmaceutical composition is administered in an amount sufficient to treat (e.g., ameliorate) or prevent a disease-related condition, a diseases or a symptoms, or to decrease or increase the amount of substance in a body fluid such as blood, serum, CSF and the like. The amount of composition (e.g., pharmaceutical compositions, formulations, including dietary supplements) adequate to accomplish this is defined as a \u201ctherapeutically effective dose.\u201d The dosage schedule and amounts effective for this use, i.e., the \u201cdosing regimen,\u201d will depend upon a variety of factors, including the stage of the disease or condition, the severity of the disease or condition, the general state of the patient's health, the patient's physical status, age and the like. In calculating the dosage regimen for a patient, the mode of administration also is taken into consideration.","The dosage regimen also takes into consideration pharmacokinetics parameters well known in the art, i.e., the active agents' rate of absorption, bioavailability, metabolism, clearance, and the like (see, e.g., Hidalgo-Aragones (1996) J. Steroid Biochem. Mol. Biol. 58:611-617; Groning (1996) Pharmazie 51:337-341; Fotherby (1996) Contraception 54:59-69; Johnson (1995) J. Pharm. Sci. 84:1144-1146; Rohatagi (1995) Pharmazie 50:610-613; Brophy (1983) Eur. J. Clin. Pharmacol. 24:103-108; the latest Remington's, supra). The state of the art allows the clinician to determine the dosage regimen for each individual patient, active agent and disease or condition treated. Guidelines provided for similar compositions used as pharmaceuticals can be used as guidance to determine the dosage regiment, i.e., dose schedule and dosage levels, administered practicing the methods of the invention are correct and appropriate.","Single or multiple administrations of compositions (e.g., pharmaceutical compositions, formulations, including dietary supplements) of the invention can be given depending on the dosage and frequency as required and tolerated by the patient. The compositions should provide a sufficient quantity of active agent to effectively treat, ameliorate or prevent PKU or other PKU-related conditions, diseases or symptoms. For example, an exemplary pharmaceutical formulation for oral administration of a protein of the invention is in a daily amount of between about 0.1 to 0.5 to about 20, 50, 100 or 1000 or more ug per kilogram of body weight per day. In an alternative embodiment, dosages are from about 1 mg to about 4 mg per kg of body weight per patient per day are used. Lower dosages can be used, in contrast to administration orally, into the blood stream, into a body cavity or into a lumen of an organ. Substantially higher dosages can be used in topical or oral administration or administering by powders, spray or inhalation. Actual methods for preparing parenterally or non-parenterally administrable formulations will be known or apparent to those skilled in the art and are described in more detail in such publications as Remington's, supra.","The compositions (e.g., pharmaceutical compositions, formulations, including dietary supplements) of the invention can further comprise other drugs or pharmaceuticals, e.g., compositions for treating a disease of interest and related symptoms or conditions. The methods of the invention can further comprise co-administration with other drugs or pharmaceuticals, e.g., compositions for treating septic shock, infection, fever, pain and related symptoms or conditions. For example, the methods and\/or compositions and formulations of the invention can be co-formulated with and\/or co-administered with antibiotics (e.g., antibacterial or bacteriostatic peptides or proteins), particularly those effective against bacteria or toxins, e.g., germ warfare agents, gram negative bacteria, fluids, cytokines, immunoregulatory agents, anti-inflammatory agents, complement activating agents, such as peptides or proteins comprising collagen-like domains or fibrinogen-like domains (e.g., a ficolin), carbohydrate-binding domains, and the like and combinations thereof.","In one aspect, the polypeptide (e.g., including a pharmaceutical composition or dietary supplement) of the invention is chemically modified. For example, the polypeptide can be chemically modified to produce a protected form that possesses better specific activity, prolonged half-life, and\/or reduced immunogenicity in vivo. A polypeptide of the invention can be modified by any means known in the art, for example, by glycosylation, pegylation or a combination thereof.","In one aspect, the polypeptide (e.g., including a pharmaceutical composition or dietary supplement) of the invention is formulated by encapsulation in a liposome, or a micro- or nano-structure, such as a nanotubule or a nano- or microcapsule.","In one aspect, the polypeptide is formulated in a matrix stabilized enzyme crystal. The invention also provides matrix stabilized enzyme crystals comprising a polypeptide of the invention for use as pharmaceutical composition or dietary supplement, e.g., to treat or ameliorate a disease or condition of interest, e.g., as described in U.S. Patent App. No. 20020182201; for example, the formulation can be a cross-linked crystalline enzyme and a polymer with a reactive moiety effective to adhere to the crystal layer of the crystalline enzyme. The invention also provides polypeptides of the invention as polymers in the form of multimerized (e.g., multi-functional) cross-linking forms; which in one aspect comprise a matrix stabilized enzyme crystal, e.g., a form resistant to degradation by proteolytic enzymes; and in alternative aspects, the cross-linking reagents comprise a dialdehyde cross-linking reagent, such as a linear or branched dialdehyde, or a substituted or unsubstituted glutaraldehyde (1,5-pentanedial), malonaldehyde (1,3-propanedial), succinaldehyde (1,4-butanedial), adipaldehyde (1,6-hexanedial), pimelaldehyde (1,7-heptanedial), or, glutaraldehyde; in other alternative aspects, the cross-linking reagents comprise carbodiimides, isoxazolium derivatives, chloroformates, carbonyldiimidazole, bis-imidoesters, bis-succinimidyl derivatives, di-isocyanates, di-isothiocyanates, di-sylfonyl halides, bis-nitrophenyl esters, dialdehydes, diacylazides, bis-maleimides, bis-haloacetyl derivatives, di-alkyl halides and bis-oxiranes (e.g., as described in U.S. Pat. No. 5,753,487).","The compositions of the invention can also be manufactured into biocompatible matrices, e.g., sol-gels, for encapsulating a polypeptide of the invention for use as pharmaceutical composition or dietary supplement, e.g., to treat or ameliorate a disease or condition of interest. In one aspect, compositions of the invention are manufactured as silica-based (e.g., oxysilane) sol-gel matrices, e.g., as described in U.S. Pat. No. 6,395,299, Pat. App. No. 20040241205. The invention also provides nano- or microcapsules comprising a composition of the invention for use as pharmaceutical composition or dietary supplement, e.g., to treat or ameliorate a disease or condition of interest, e.g., as described in U.S. Patent App. No. 20030157181.","The pharmaceutical compositions of the invention can be manufactured using any conventional method, e.g., mixing, dissolving, granulating, drag\u00e9e-making, levigating, emulsifying, encapsulating, entrapping, melt-spinning, spray-drying, or lyophilizing processes. Alternative pharmaceutical formulations can be determined depending on the patient (e.g., adult or pediatric), condition, route of administration (e.g., oral) and the desired dosage.","Applications\u2014Industrial, Medical, Experimental, Food and Feed Processing","Polypeptides (including enzymes and antibodies) and nucleic acids of the invention can be used for a variety of industrial, experimental, food and feed processing, nutritional and pharmaceutical applications, e.g., for food and feed supplements, colorants, neutraceuticals, cosmetic and pharmaceutical needs.","Polypeptides of the invention (e.g., having enzyme, structural or binding activity) can be highly selective catalysts. The invention provides methods using enzymes of the invention in the food and feed industries, e.g., in methods for making food and feed products and food and feed additives. In one aspect, the invention provides processes using enzymes of the invention in the medical industry, e.g., to make pharmaceuticals.","The enzymes of the invention can catalyze reactions with exquisite stereo-, regio- and chemo-selectivities. The polypeptide, enzyme, protein, e.g. structural or binding proteins of the invention can be engineered to function in various solvents, operate at extreme pHs (for example, high pHs and low pHs) extreme temperatures (for example, high temperatures and low temperatures), extreme salinity levels (for example, high salinity and low salinity) and catalyze reactions with compounds that are structurally unrelated to their natural, physiological substrates.","Animal Feeds and Food or Feed Additives","The invention provides compositions (e.g., enzymes of the invention, as those described in Tables 1, 2, and 3) methods for treating animal feeds and foods and food or feed additives using a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention, and\/or the antibodies of the invention. The invention provides animal feeds, foods, and additives comprising a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention, antibodies of the invention. The animal can be any farm animal or any animal raised for its meat, e.g., a pig, goat, cattle, sheep, horse and the like.","The animal feed additive of the invention may be a granulated enzyme product that may readily be mixed with feed components. Alternatively, feed additives of the invention can form a component of a pre-mix. The granulated enzyme product of the invention may be coated or uncoated. The particle size of the enzyme granulates can be compatible with that of feed and pre-mix components. This provides a safe and convenient mean of incorporating enzymes into feeds. Alternatively, the animal feed additive of the invention may be a stabilized liquid composition. This may be an aqueous or oil-based slurry. See, e.g., U.S. Pat. No. 6,245,546.","A polypeptide, enzyme, protein, e.g. structural or binding protein, of the present invention, in the modification of animal feed or a food, can process the food or feed either in vitro (by modifying components of the feed or food) or in vivo. Polypeptides of the invention can be added to animal feed or food compositions.","In one aspect, an enzyme of the invention has any of the following enzyme activities, or is added in combination with another enzyme, e.g., beta-galactosidases, catalases, laccases, cellulases, endoglycosidases, endo-beta-1,4-laccases, amyloglucosidases, glucose isomerases, glycosyltransferases, lipases, phospholipases, lipooxygenases, beta-laccases, endo-beta-1,3(4)-laccases, cutinases, peroxidases, amylases, glucoamylases, pectinases, reductases, oxidases, decarboxylases, phenoloxidases, ligninases, pullulanases, arabinanases, hemicellulases, mannanases, xylolaccases, xylanases, pectin acetyl esterases, rhamnogalacturonan acetyl esterases, proteases, peptidases, proteinases, polygalacturonases, rhamnogalacturonases, galactanases, pectin lyases, transglutaminases, pectin methylesterases, cellobiohydrolases and\/or transglutaminases. These enzyme digestion products are more digestible by the animal. Thus, a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention can contribute to the available energy of the feed or food. Also, a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention can improve the digestibility and uptake of carbohydrate and non-carbohydrate feed or food constituents such as protein, fat and minerals.","In another aspect, a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention can be supplied by expressing the enzymes directly in transgenic feed crops (as, e.g., transgenic plants, seeds and the like), such as grains, cereals, corn, soy bean, rape seed, lupin and the like. As discussed above, the invention provides transgenic plants, plant parts and plant cells comprising a nucleic acid sequence encoding a polypeptide of the invention. In one aspect, the nucleic acid is expressed such that the polypeptide, enzyme, protein, e.g. structural or binding protein of the invention is produced in recoverable quantities. The polypeptide, enzyme, protein, e.g. structural or binding protein can be recovered from any plant or plant part. Alternatively, the plant or plant part containing the recombinant polypeptide can be used as such for improving the quality of a food or feed, e.g., improving nutritional value, palatability, etc.","The enzyme delivery matrix of the invention is in the form of discrete plural particles, pellets or granules. By \u201cgranules\u201d is meant particles that are compressed or compacted, such as by a pelletizing, extrusion, or similar compacting to remove water from the matrix. Such compression or compacting of the particles also promotes intraparticle cohesion of the particles. For example, the granules can be prepared by pelletizing the grain-based substrate in a pellet mill. The pellets prepared thereby are ground or crumbled to a granule size suitable for use as an adjuvant in animal feed. Since the matrix is itself approved for use in animal feed, it can be used as a diluent for delivery of enzymes in animal feed.","The polypeptide, enzyme, protein, e.g. structural or binding protein contained in the invention enzyme delivery matrix and methods is in one aspect thermostable polypeptide, enzyme, protein, e.g. structural or binding protein, as described herein, so as to resist inactivation of the polypeptide, enzyme, protein, e.g. structural or binding protein during manufacture where elevated temperatures and\/or steam may be employed to prepare the palletized enzyme delivery matrix. During digestion of feed containing the invention enzyme delivery matrix, aqueous digestive fluids will cause release of the active enzyme. Other types of thermostable enzymes and nutritional supplements that are thermostable can also be incorporated in the delivery matrix for release under any type of aqueous conditions.","A coating can be applied to the invention enzyme matrix particles for many different purposes, such as to add a flavor or nutrition supplement to animal feed, to delay release of animal feed supplements and enzymes in gastric conditions, and the like. Or, the coating may be applied to achieve a functional goal, for example, whenever it is desirable to slow release of the enzyme from the matrix particles or to control the conditions under which the enzyme will be released. The composition of the coating material can be such that it is selectively broken down by an agent to which it is susceptible (such as heat, acid or base, enzymes or other chemicals). Alternatively, two or more coatings susceptible to different such breakdown agents may be consecutively applied to the matrix particles.","The invention is also directed towards a process for preparing an enzyme-releasing matrix. In accordance with the invention, the process comprises providing discrete plural particles of a grain-based substrate in a particle size suitable for use as an enzyme-releasing matrix, wherein the particles comprise a polypeptide, enzyme, protein, e.g. structural or binding protein, encoded by an amino acid sequence of the invention. In one aspect, the process includes compacting or compressing the particles of enzyme-releasing matrix into granules, which most in one aspect is accomplished by pelletizing. The mold inhibitor and cohesiveness agent, when used, can be added at any suitable time, and in one aspect are mixed with the grain-based substrate in the desired proportions prior to pelletizing of the grain-based substrate. Moisture content in the pellet mill feed in one aspect is in the ranges set forth above with respect to the moisture content in the finished product, and in one aspect is about 14-15%. In one aspect, moisture is added to the feedstock in the form of an aqueous preparation of the enzyme to bring the feedstock to this moisture content. The temperature in the pellet mill in one aspect is brought to about 82\u00b0 C. with steam. The pellet mill may be operated under any conditions that impart sufficient work to the feedstock to provide pellets. The pelleting process itself is a cost-effective process for removing water from the enzyme-containing composition.","The compositions and methods of the invention can be practiced in conjunction with various nutritional and environmental factors including, e.g., (1) manipulation of gut microflora by supplementing feed with prebiotics and\/or antibiotics, (2) low fiber diet (low energy and low purine diet), (3) restricting feed for 48 hours and withholding feed for 12 hours before slaughter, (4) increasing consumption of water, and\/or (5) keeping animals clean.","The compositions and methods of the invention can be practiced in conjunction with administration of prebiotics, which are high molecular weight sugars, e.g., fructo-oligosaccharides (FOS); galacto-oligosaccharides (GOS), GRAS (Generally Recognized As Safe) material. These prebiotics can be metabolized by some probiotic lactic acid bacteria (LAB). They are non-digestible by the majority of intestinal microbes.","Treating Foods and Food Processing","The polypeptide, enzyme, protein, e.g. structural or binding proteins of the invention, e.g., as described in Tables 1, 2, and 3, have numerous applications in food processing industry. The invention provides treatment compositions, including, e.g., a plant cell, a bacterial cell, a yeast cell, an insect cell, or an animal cell, or any plant or plant part, or any food or feed, a waste product and the like.","The invention provides feeds or foods comprising a polypeptide, enzyme, protein, e.g. structural or binding protein, the invention, e.g., a feed, a liquid, e.g., a beverage (such as a fruit juice or a beer), a bread or a dough or a bread product, or a beverage precursor (e.g., a wort).","The food treatment processes of the invention can comprise use of any enzyme of the invention, which can have the following enzymatic activities, and also include the use of any combination of any enzyme, including lyases, laccases, catalases, laccases, cellulases, endoglycosidases, endo-beta-1,4-laccases, amyloglucosidases, glucose isomerases, glycosyltransferases, lipases, phospholipases, lipooxygenases, beta-laccases, endo-beta-1,3(4)-laccases, cutinases, peroxidases, amylases, glucoamylases, pectinases, reductases, oxidases, decarboxylases, phenoloxidases, ligninases, pullulanases, arabinanases, hemicellulases, mannanases, xylolaccases, xylanases, pectin acetyl esterases, rhamnogalacturonan acetyl esterases, proteases, peptidases, proteinases, polygalacturonases, rhamnogalacturonases, galactanases, pectin lyases, transglutaminases, pectin methylesterases, cellobiohydrolases and\/or transglutaminases.","Confectionaries, Cacao Butter and Foods","In one aspect, the compositions and methods of the invention can be used to make and process hard butters, such as cacao butter (cocao butter). The compositions and methods of the invention can be used to make cocoa butter alternatives by \u201cstructured\u201d synthetic techniques using the enzymes of the invention, e.g., as described in Tables 1, 2, and 3, including esterases, acylases, lipases, phospholipases or proteases of the invention. For example, in one aspect, the methods of the invention process or synthesize triacylglycerides, diacylglycerides and\/or monoacylglycerides for use as, e.g., cocoa butter alternatives. In one aspect, the methods of the invention generate a hard butter with a defined \u201cplastic region\u201d to maintain sufficient hardness below or at room temperature. In one aspect, the processed or synthesized lipid is designed to have a very narrow \u201cplastic region,\u201d e.g., in one aspect, where it rapidly melts at about body temperature. Natural cacao butter begins to soften at approximately 30\u00b0 C. to 32\u00b0 C., and completely melts at approximately 36\u00b0 C. Natural cacao butter can contain 70 wt % or more of three 1,3-disaturated-2-oleoyl glycerols, which are 1,3-dipalmitoyl-2-oleoyl glycerol (POP), 1-palmitoyl-2-oleoyl glycerol (POSt) and 1,3-distearoyl-2-oleoyl glycerol (StOSt). These three glycerols show a similar melting behavior to each other and are responsible for melting properties of the cacao butter, exhibiting a very narrow plastic region. The invention provides synthetic cacao butters or processed cacao butters (synthesized or processed using a hydrolase of the invention, all possible composition are referred to as cocoa-butter alternatives) with varying percentages of, 3-dipalmitoyl-2-oleoyl glycerol (POP), 1-palmitoyl-2-oleoyl glycerol (POSt) and 1,3-distearoyl-2-oleoyl glycerol (StOSt), depending on the desired properties of the synthetic cacao butter, and, synthetic cacao butters with more or less than 70 wt % of the three 1,3-disaturated-2-oleoyl glycerols. The synthetic cacao butters of the invention can partially or completely replace natural or unprocessed cacao butters and can maintain or improve essential hard butter properties.","The invention provides synthetic cacao butters or processed cacao butters (synthesized or processed using a hydrolase of the invention) with desired properties for use in confectionary, bakery and pharmaceutical products. In one aspect, the invention provides confectionary, bakery and pharmaceutical products comprising a hydrolase of the invention. In one aspect, the methods of the invention make or process a lipid (a fat) from a confection (e.g., a chocolate) or to be used in a confection. In one aspect, a lipid is made or processed such that the chocolate shows less finger-imprinting than chocolate made from natural cocoa butter, while still having sharp melting characteristics in the mouth. In one aspect, a lipid is made or processed such that a confection (e.g., chocolate) can be made at a comparatively high ambient temperature, or, be made using a cooling water at a comparatively high temperature. In one aspect, the lipid is made or processed such that a confection (e.g., chocolate) can be stored under relatively warmer conditions, e.g., tropical or semi-tropical conditions or in centrally heated buildings. In one aspect, the lipids are made or processed such that a confection (e.g., chocolate) will have a lipid (fat) content of consistent composition and quality. The enzymes of the invention can be used to provide a substitute composition for cacao butter which can significantly improve its thermal stability and replace it in a wide range of applications.","Margarine and Shortening Production","The invention provides synthetic or processed fats, e.g., margarine and shortening synthesized or processed using an enzyme of the invention, e.g., as described in Tables 1, 2, and 3, such as a hydrolase of the invention. In one aspect, the invention provides processed fats comprising a vegetable oil, such as soybean oil, corn oil, rapeseed oil, palm oil or lauric type oils synthesized or processed using a hydrolase of the invention. The synthetic or processed fats, e.g., margarine and shortening, are designed to have a desired \u201cplasticity.\u201d Many of the plastic fat products, such as margarine and shortening, are produced from hard stocks and liquid oils as raw materials. For example, liquid oils such as soybean oil, corn oil, palm oil and rapeseed oil, are blended with their hardened oils (hard stocks), and the blend is adjusted to have an appropriate consistency (plasticity). The plastic fat products such as margarine and shortening so produced tend to cause the formation of relatively coarse crystallines because fats and oils used as the raw materials are composed of fatty acids having almost the same carbon chain length. In other words, they have a highly-unified composition of fatty acids. For this reason, the plasticity of these products can be maintained at an appropriate degree only within a narrow temperature range, so that the liquid oils contained therein have a tendency to exude. In one aspect, the invention provides methods of making or processing fats designed such that they have a varied (and defined) composition of fatty acids. The resultant oil, e.g., margarine or shortening, can have a broader range of plasticity.","In one aspect, the methods and compositions of the invention are used to make or process vegetable oils, such as soybean oil, corn oil, rapeseed oil, palm oil or lauric type oils using the hydrolases of the invention, including inter-esterification and enzymatic transesterification, see e.g., U.S. Pat. No. 5,288,619. The methods and compositions of the invention can be used in place of random inter-esterification as described in, e.g., U.S. Pat. No. 3,949,105. In one aspect, the methods and compositions of the invention are used to in enzymatic transesterification for preparing an oil, e.g., a margarine oil, having both low trans-acid and low intermediate chain fatty acid content.","In one aspect, the symmetric structure of an oil, e.g., a palm or lauric type oils is modified, e.g., into a random structure. Thus, the methods of the invention can be used to modify the properties of plastic fat products. In one aspect, the modification of oils by the methods of the invention can be designed to prevent or slow gradually hardening of the oil with time, particularly when the products are being stored.","In one aspect, the methods and compositions of the invention in a trans-esterification reaction mixture comprising a stearic acid source material and an edible liquid vegetable oil, trans-esterifying the stearic acid source material and the vegetable oil using a 1-, 3-positionally specific lipase of the invention, and then hydrogenating the fatty acid mixture to provide a recycle stearic acid source material for a recyclic reaction with the vegetable oil. See e.g., U.S. Pat. No. 5,288,619.","In one aspect, an inter-esterification reaction is conducted with a lipase of the invention. In one aspect, the lipase of the invention has a selectivity for the 1- and 3-positions of triglyceride to slow or inhibit an increase in the amount of tri-saturated triglycerides in the oil. In this reaction of the invention, deficiencies of conventional random inter-esterification and the difficulty of inter-esterification with a non-specific lipase can be overcome because the inter-esterification is conducted by an enzyme of the invention having a specificity for the 1- and 3-positions of triglycerides. In one aspect, the exudation of liquid oils contained in the products is slowed or prevented with a temperature increase in the reaction to inhibit a rise in the melting point caused by an increase in the amount of tri-saturated triglycerides. This addresses the problem of hardening of products during long-term storage.","Brewing and Fermenting","The invention provides methods of brewing (e.g., fermenting) beer comprising hydrolases of the invention. In one exemplary process, starch-containing raw materials are disintegrated and processed to form a malt. A hydrolase of the invention is used at any point in the fermentation process. For example, hydrolases (e.g., proteases) of the invention can be used in the processing of barley malt. The major raw material of beer brewing is barley malt. This can be a three stage process. First, the barley grain can be steeped to increase water content, e.g., to around about 40%. Second, the grain can be germinated by incubation at 15 to 25\u00b0 C. for 3 to 6 days when enzyme synthesis is stimulated under the control of gibberellins. In one aspect, hydrolases of the invention are added at this (or any other) stage of the process. The action of hydrolases results in an increase in fermentable reducing sugars. This can be expressed as the diastatic power, DP, which can rise from around 80 to 190 in 5 days at 12\u00b0 C. Hydrolases (e.g., proteases) of the invention can be used in any beer or alcoholic beverage producing process, as described, e.g., in U.S. Pat. Nos. 5,762,991; 5,536,650; 5,405,624; 5,021,246; 4,788,066.","Waste Treatment","The polypeptide, enzyme, protein, e.g. structural or binding proteins of the invention, e.g., as described in Tables 1, 2, and 3, can be used in a variety of other industrial applications, e.g., in waste treatment (in addition to, e.g., biomass conversion to fuels). For example, in one aspect, the invention provides a solid waste digestion process using a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention. The methods can comprise reducing the mass and volume of substantially untreated solid waste. Solid waste can be treated with an enzymatic digestive process in the presence of an enzymatic solution (including a polypeptide, enzyme, protein, e.g. structural or binding protein, of the invention) at a controlled temperature. This results in a reaction without appreciable bacterial fermentation from added microorganisms. The solid waste is converted into a liquefied waste and any residual solid waste. The resulting liquefied waste can be separated from said any residual solidified waste. See e.g., U.S. Pat. No. 5,709,796.","In one aspect, the compositions and methods of the invention are used for odor removal or odor reduction in animal waste lagoons, e.g., on swine farms, and other animal waste management systems.","The waste treatment processes of the invention can comprise use of any enzyme of the invention, which can have the following enzymatic activities, and also include the use of any combination of any enzyme or protein, including e.g. structural or binding protein, catalases, lyases, laccases, cellulases, endoglycosidases, endo-beta-1,4-laccases, amyloglucosidases, glucose isomerases, glycosyltransferases, lipases, phospholipases, lipooxygenases, beta-laccases, endo-beta-1,3(4)-laccases, cutinases, peroxidases, amylases, glucoamylases, pectinases, reductases, oxidases, decarboxylases, phenoloxidases, ligninases, pullulanases, phytases, arabinanases, hemicellulases, mannanases, xylolaccases, xylanases, pectin acetyl esterases, rhamnogalacturonan acetyl esterases, proteases, peptidases, proteinases, polygalacturonases, rhamnogalacturonases, galactanases, pectin lyases, transglutaminases, pectin methylesterases, cellobiohydrolases and\/or transglutaminases.","Increasing the Flow of Production Fluids from a Subterranean Formation","The invention also includes a method using an enzyme of the invention, e.g., as described in Tables 1, 2, and 3, wherein the method increases the flow of production fluids from a subterranean formation by removing viscous (e.g., starch-containing) damaging fluids formed during production operations; these fluids can be found within the subterranean formation which surrounds a completed well bore. Thus, this method of the invention results in production fluids being able to flow from the well bore. This method of the invention also addresses the problem of damaging fluids reducing the flow of production fluids from a formation below expected flow rates. In one aspect, the invention provides for formulating an enzyme treatment (using an enzyme of the invention) by blending together an aqueous fluid and a polypeptide of the invention; pumping the enzyme treatment to a desired location within the well bore; allowing the enzyme treatment to degrade the viscous (e.g., starch-containing) damaging fluid, whereby the fluid can be removed from the subterranean formation to the well surface; and wherein the enzyme treatment is effective to attack linkages in the viscous (e.g., starch-containing) fluid.","The subterranean formation enzyme treatment processes of the invention can comprise use of any enzyme of the invention, which can have the following enzymatic activities, and also include the use of any combination of any enzyme, including tryptophanases or tyrosine decarboxylases, laccases, catalases, lyases, laccases, other cellulases, endoglycosidases, endo-beta-1,4-laccases, amyloglucosidases, other glucosidases, glucose isomerases, glycosyltransferases, lipases, phospholipases, lipooxygenases, beta-laccases, endo-beta-1,3(4)-laccases, cutinases, peroxidases, amylases, glucoamylases, pectinases, reductases, oxidases, decarboxylases, phenoloxidases, ligninases, pullulanases, arabinanases, hemicellulases, mannanases, xylolaccases, xylanases, pectin acetyl esterases, rhamnogalacturonan acetyl esterases, proteases, peptidases, proteinases, polygalacturonases, rhamnogalacturonases, galactanases, pectin lyases, transglutaminases, pectin methylesterases, other cellobiohydrolases and\/or transglutaminases.","Latex Processing","The methods and compositions (e.g., enzymes of the invention, e.g., as described in Tables 1, 2, and 3, including esterases, acylases, lipases, phospholipases or proteases of the invention) of the invention can be used to selectively hydrolyze saturated esters over unsaturated esters into acids or alcohols. In one aspect, the invention provides for the selective hydrolysis of ethyl propionate over ethyl acrylate. In one aspect, these methods are used to remove undesired esters from monomer feeds used in latex polymerization and from the latexes after polymerization. The methods and compositions (hydrolases) of the invention can be used to treat latexes for a variety of purposes, e.g., to treat latexes used in hair fixative compositions to remove unpleasant odors. Latexes treated by the methods and compositions of the invention include, e.g., polymers containing acrylic, vinyl and unsaturated acid monomers, including alkyl acrylate monomers such as methyl acrylate, ethyl acrylate, propyl acrylate and butyl acrylate, and acrylate acids such as acrylic acid, methacrylic acid, crotonic acid, itaconic acid and mixtures thereof. See, e.g., U.S. Pat. No. 5,856,150.","Biomass Conversion and Production of Clean Bio Fuels","The invention provides enzymes, e.g., as described in Tables 1, 2, and 3 (including mixtures, or \u201ccocktails\u201d of enzymes) and methods for the conversion of a biomass or any lignocellulosic material (e.g., any composition comprising cellulose, hemicellulose and lignin), to fuels (e.g., bioethanol, biodiesel), in addition to feeds, foods and chemicals. Thus, the compositions and methods of the invention provide effective and sustainable alternatives or adjuncts to use of petroleum-based products, e.g., as a mixture of bioethanol and gasoline. The invention provides organisms expressing enzymes of the invention for participation in chemical cycles involving natural biomass conversion. In one aspect, enzymes and methods for the conversion are used in enzyme ensembles for the efficient depolymerization of cellulosic and hemicellulosic polymers to metabolizeable carbon moieties. The invention provides methods for discovering and implementing the most effective of enzymes to enable these important new \u201cbiomass conversion\u201d and alternative energy industrial processes.","The methods of the invention also include taking the converted lignocellulosic material (processed by enzymes of the invention) and making it into a fuel (e.g. a bioethanol, a biodiesel) by fermentation and\/or by chemical synthesis. In one aspect, the produced sugars are fermented and\/or the non-fermentable products are gasified.","The enzymes of the invention (including, for example, organisms, such as microorganisms, e.g., fungi, yeast or bacteria, making and in some aspects secreting recombinant enzymes of the invention) can be used in or included\/integrated at any stage of any biomass conversion process, e.g., at any one step, several steps, or included in all of the steps, or all of the following methods of biomass conversion processes, or all of these biofuel alternatives:\n\n","In one aspect, the polypeptides of the invention have cellulolytic activity, e.g., cellulases activity, such as endoglucanase, cellobiohydrolase and\/or \u03b2-glucosidase (beta-glucosidase) activity, or other enzymatic activity for generating biodiesel or bioethanol from an organic material, e.g., a biomass, such as compositions derived from plants and animals, including any agricultural crop or other renewable feedstock, an agricultural residue or an animal waste, or the organic components of municipal and industrial wastes, or microorganisms such as algae or yeast. In one aspect, polypeptides of the invention are used in processes for converting lignocellulosic biomass to ethanol, or otherwise are used in processes for hydrolyzing or digesting biomaterials such that they can be used as a biofuel (including biodiesel or bioethanol), or for making it easier for the biomass to be processed into a fuel. In an alternative aspect, polypeptides of the invention are used in processes for a transesterification process reacting an alcohol (like methanol) with a triglyceride oil contained in a vegetable oil, animal fat or recycled greases, forming fatty acid alkyl esters (biodiesel) and glycerin. In one aspect, biodiesel is made from soybean oil or recycled cooking oils. Animal's fats, other vegetable oils, and other recycled oils can also be used to produce biodiesel, depending on their costs and availability. In another aspect, blends of all kinds of fats and oils are used to produce a biodiesel fuel of the invention.","Enzymes of the invention can also be used in glycerin refining. The glycerin by-product contains unreacted catalyst and soaps that are neutralized with an acid. Water and alcohol are removed to produce 50% to 80% crude glycerin. The remaining contaminants include unreacted fats and oils, which can be processes using the polypeptides of the invention. In a large biodiesel plants of the invention, the glycerin can be further purified, e.g., to 99% or higher purity, for the pharmaceutical and cosmetic industries.","Both bioethanol and biodiesel made using the polypeptides of the invention can be used with fuel oxygenates to improve combustion characteristics. Adding oxygen results in more complete combustion, which reduces carbon monoxide emissions. This is another environmental benefit of replacing petroleum fuels with biofuels (e.g., a fuel of the invention). A bioethanol made using the compositions and\/or methods of this invention can be blended with gasoline to form an E10 blend (about 5% to 10% ethanol and about 90% to 95% gasoline), but it can be used in higher concentrations such as E85 or in its pure form. A bioethanol made using the compositions and\/or methods of this invention can be blended with petroleum diesel to form a B20 blend (20% biodiesel and 80% petroleum diesel), although other blend levels can be used up to B100 (pure biodiesel).","The invention also provides processes for making ethanol (\u201cbioethanol\u201d) from compositions comprising lignocellulosic biomass. The lignocellulose biomass material can be obtained from agricultural crops, as a byproduct of food or feed production, or as lignocellulosic waste products, such as plant residues and waste paper. Examples of suitable plant sources or plant residues for treatment with polypeptides of the invention include kelp, algae, grains, seeds, stems, leaves, hulls, husks, corn cobs, corn stover, straw, grasses (e.g., Indian grass, such as ; or, switch grass, e.g., species, such as ), and the like, as well as wood, wood chips, wood pulp, and sawdust. Examples of paper waste suitable for treatment with polypeptides of the invention include discard photocopy paper, computer printer paper, notebook paper, notepad paper, typewriter paper, and the like, as well as newspapers, magazines, cardboard, and paper-based packaging materials.","In one aspect, the enzymes and methods of the invention can be used in conjunction with more \u201ctraditional\u201d means of making ethanol from biomass, e.g., as methods comprising hydrolyzing lignocellulosic materials by subjecting dried lignocellulosic material in a reactor to a catalyst comprised of a dilute solution of a strong acid and a metal salt; this can lower the activation energy, or the temperature, of cellulose hydrolysis to obtain higher sugar yields; see, e.g., U.S. Pat. Nos. 6,660,506; 6,423,145.","Another exemplary method that incorporated use of enzymes of the invention comprises hydrolyzing lignocellulosic material containing hemicellulose, cellulose and lignin by subjecting the material to a first stage hydrolysis step in an aqueous medium at a temperature and a pressure chosen to effect primarily depolymerization of hemicellulose without major depolymerization of cellulose to glucose. This step results in a slurry in which the liquid aqueous phase contains dissolved monosaccharides resulting from depolymerization of hemicellulose and a solid phase containing cellulose and lignin. A second stage hydrolysis step can comprise conditions such that at least a major portion of the cellulose is depolymerized, such step resulting in a liquid aqueous phase containing dissolved\/soluble depolymerization products of cellulose. See, e.g., U.S. Pat. No. 5,536,325. Enzymes of the invention can be added at any stage of this exemplary process.","Another exemplary method that incorporated use of enzymes of the invention comprises processing a lignocellulose-containing biomass material by one or more stages of dilute acid hydrolysis with about 0.4% to 2% strong acid; and treating an unreacted solid lignocellulosic component of the acid hydrolyzed biomass material by alkaline delignification to produce precursors for biodegradable thermoplastics and derivatives. See, e.g., U.S. Pat. No. 6,409,841. Enzymes of the invention can be added at any stage of this exemplary process.","Another exemplary method that incorporated use of enzymes of the invention comprises prehydrolyzing lignocellulosic material in a prehydrolysis reactor; adding an acidic liquid to the solid lignocellulosic material to make a mixture; heating the mixture to reaction temperature; maintaining reaction temperature for time sufficient to fractionate the lignocellulosic material into a solubilized portion containing at least about 20% of the lignin from the lignocellulosic material and a solid fraction containing cellulose; removing a solubilized portion from the solid fraction while at or near reaction temperature wherein the cellulose in the solid fraction is rendered more amenable to enzymatic digestion; and recovering a solubilized portion. See, e.g., U.S. Pat. No. 5,705,369. Enzymes of the invention can be added at any stage of this exemplary process.","The invention provides methods for making motor fuel compositions (e.g., for spark ignition motors) based on liquid hydrocarbons blended with a fuel grade alcohol made by using an enzyme or a method of the invention. In one aspect, the fuels made by use of an enzyme of the invention comprise, e.g., coal gas liquid- or natural gas liquid-ethanol blends. In one aspect, a co-solvent is biomass-derived 2-methyltetrahydrofuran (MTHF). See, e.g., U.S. Pat. No. 6,712,866.","In one aspect, methods of the invention for the enzymatic degradation of lignocellulose, e.g., for production of ethanol from lignocellulosic material, can also comprise use of ultrasonic treatment of the biomass material; see, e.g., U.S. Pat. No. 6,333,181.","In another aspect, methods of the invention for producing bioethanol from a cellulosic substrate comprise providing a reaction mixture in the form of a slurry comprising cellulosic substrate, an enzyme of this invention and a fermentation agent (e.g., within a reaction vessel, such as a semi-continuously solids-fed bioreactor), and the reaction mixture is reacted under conditions sufficient to initiate and maintain a fermentation reaction (as described, e.g., in U.S. Pat. App. No. 20060014260). In one aspect, experiment or theoretical calculations can determine an optimum feeding frequency. In one aspect, additional quantities of the cellulosic substrate and the enzyme are provided into the reaction vessel at an interval(s) according to the optimized feeding frequency.","One exemplary process for making a biofuels and biodiesels of the invention is described in U.S. Pat. App. Pub. Nos. 20050069998; 20020164730; and in one aspect comprises stages of grinding the lignocellulosic biomass (e.g., to a size of 15-30 mm), subjecting the product obtained to steam explosion pre-treatment (e.g., at a temperature of 190-230\u00b0 C.) for between 1 and 10 minutes in a reactor; collecting the pre-treated material in a cyclone or related product of manufacture; and separating the liquid and solid fractions by filtration in a filter press, introducing the solid fraction in a fermentation deposit and adding one or more enzymes of the invention, e.g., a cellulase and\/or beta-glucosidase enzyme (e.g., dissolved in citrate buffer pH 4.8).","Another exemplary process for making a biofuels and biodiesels of the invention comprising ethanol using enzymes of the invention comprises pretreating a starting material comprising a lignocellulosic feedstock comprising at least hemicellulose and cellulose. In one aspect, the starting material comprises potatoes, soybean (rapeseed), barley, rye, corn, oats, wheat, beets or sugar cane or a component or waste or food or feed production byproduct. The starting material (\u201cfeedstock\u201d) is reacted at conditions which disrupt the plant's fiber structure to effect at least a partial hydrolysis of the hemicellulose and cellulose. Disruptive conditions can comprise, e.g., subjecting the starting material to an average temperature of 180\u00b0 C. to 270\u00b0 C. at pH 0.5 to 2.5 for a period of about 5 seconds to 60 minutes; or, temperature of 220\u00b0 C. to 270\u00b0 C., at pH 0.5 to 2.5 for a period of 5 seconds to 120 seconds, or equivalent. This generates a feedstock with increased accessibility to being digested by an enzyme, e.g., a cellulase enzyme of the invention. U.S. Pat. No. 6,090,595.","Exemplary conditions for cellulase hydrolysis of lignocellulosic material include reactions at temperatures between about 30\u00b0 C. and 48\u00b0 C., and\/or a pH between about 4.0 and 6.0. Other exemplary conditions include a temperature between about 30\u00b0 C. and 60\u00b0 C. and a pH between about 4.0 and 8.0.","Detergent Compositions","The invention provides detergent compositions comprising one or more polypeptides of the invention (e.g., enzymes as described in Tables 1, 2, and 3, e.g., having cellulase, endoglucanase, cellobiohydrolase, mannanase and\/or beta-glucosidase activity) and methods of making and using these compositions. The invention incorporates all methods of making and using detergent compositions, see, e.g., U.S. Pat. Nos. 6,413,928; 6,399,561; 6,365,561; 6,380,147. The detergent compositions can be a one and two part aqueous composition, a non-aqueous liquid composition, a cast solid, a granular form, a particulate form, a compressed tablet, a gel and\/or a paste and a slurry form. The invention also provides methods capable of a rapid removal of gross food soils, films of food residue and other minor food compositions using these detergent compositions. Enzymes of the invention can facilitate the removal of starchy stains by means of catalytic hydrolysis of the starch polysaccharide. Enzymes of the invention can be used in dishwashing detergents in textile laundering detergents.","The actual active enzyme content depends upon the method of manufacture of a detergent composition and is not critical, assuming the detergent solution has the desired enzymatic activity. In one aspect, the amount of glucosidase present in the final solution ranges from about 0.001 mg to 0.5 mg per gram of the detergent composition. The particular enzyme chosen for use in the process and products of this invention depends upon the conditions of final utility, including the physical product form, use pH, use temperature, and soil types to be degraded or altered. The enzyme can be chosen to provide optimum activity and stability for any given set of utility conditions. In one aspect, the polypeptides of the present invention are active in the pH ranges of from about 4 to about 12 and in the temperature range of from about 20\u00b0 C. to about 95\u00b0 C. The detergents of the invention can comprise cationic, semi-polar nonionic or zwitterionic surfactants; or, mixtures thereof.","Enzymes of the present invention (e.g., enzymes having cellulase, endoglucanase, cellobiohydrolase, mannanase and\/or beta-glucosidase activity) can be formulated into powdered and liquid detergents having pH between 4.0 and 12.0 at levels of about 0.01 to about 5% (preferably 0.1% to 0.5%) by weight. These detergent compositions can also include other enzymes such as known proteases, cellulases, lipases or endoglycosidases, as well as builders and stabilizers. The addition of enzymes of the invention to conventional cleaning compositions does not create any special use limitation. In other words, any temperature and pH suitable for the detergent is also suitable for the present compositions as long as the pH is within the above range, and the temperature is below the described enzyme's denaturing temperature. In addition, the polypeptides of the invention can be used in a cleaning composition without detergents, again either alone or in combination with builders and stabilizers.","The present invention provides cleaning compositions including detergent compositions for cleaning hard surfaces, detergent compositions for cleaning fabrics, dishwashing compositions, oral cleaning compositions, denture cleaning compositions, and contact lens cleaning solutions.","In one aspect, the invention provides a method for washing an object comprising contacting the object with a polypeptide of the invention under conditions sufficient for washing. A polypeptide of the invention may be included as a detergent additive. The detergent composition of the invention may, for example, be formulated as a hand or machine laundry detergent composition comprising a polypeptide of the invention. A laundry additive suitable for pre-treatment of stained fabrics can comprise a polypeptide of the invention. A fabric softener composition can comprise a polypeptide of the invention. Alternatively, a polypeptide of the invention can be formulated as a detergent composition for use in general household hard surface cleaning operations. In alternative aspects, detergent additives and detergent compositions of the invention may comprise one or more other enzymes such as a protease, a lipase, a cutinase, another glucosidase, a carbohydrase, another cellulase, a pectinase, a mannanase, an arabinase, a galactanase, a xylanase, an oxidase, e.g., a lactase, and\/or a peroxidase. The properties of the enzyme(s) of the invention are chosen to be compatible with the selected detergent (i.e. pH-optimum, compatibility with other enzymatic and non-enzymatic ingredients, etc.) and the enzyme(s) is present in effective amounts. In one aspect, enzymes of the invention are used to remove malodorous materials from fabrics. Various detergent compositions and methods for making them that can be used in practicing the invention are described in, e.g., U.S. Pat. Nos. 6,333,301; 6,329,333; 6,326,341; 6,297,038; 6,309,871; 6,204,232; 6,197,070; 5,856,164.","The detergents and related processes of the invention can also include the use of any combination of other enzymes such as tryptophanases or tyrosine decarboxylases, laccases, catalases, laccases, other cellulases, endoglycosidases, endo-beta-1,4-laccases, amyloglucosidases, other glucosidases, glucose isomerases, glycosyltransferases, lipases, phospholipases, lipooxygenases, beta-laccases, endo-beta-1,3(4)-laccases, cutinases, peroxidases, amylases, glucoamylases, pectinases, reductases, oxidases, decarboxylases, phenoloxidases, ligninases, pullulanases, arabinanases, hemicellulases, mannanases, xylolaccases, xylanases, pectin acetyl esterases, rhamnogalacturonan acetyl esterases, proteases, peptidases, proteinases, polygalacturonases, rhamnogalacturonases, galactanases, pectin lyases, transglutaminases, pectin methylesterases, other cellobiohydrolases and\/or transglutaminases.","Treating Fabrics and Textiles","The invention provides compositions and methods of treating fabrics and textiles using one or more polypeptides of the invention, e.g., enzymes as described in Tables 1, 2, and 3, including enzymes having cellulase, endoglucanase, cellobiohydrolase, mannanase and\/or beta-glucosidase activity. The polypeptides of the invention can be used in any fabric-treating method, which are well known in the art, see, e.g., U.S. Pat. No. 6,077,316. For example, in one aspect, the feel and appearance of a fabric is improved by a method comprising contacting the fabric with an enzyme of the invention in a solution. In one aspect, the fabric is treated with the solution under pressure.","In one aspect, the enzymes of the invention are applied during or after the weaving of textiles, or during the desizing stage, or one or more additional fabric processing steps. During the weaving of textiles, the threads are exposed to considerable mechanical strain. Prior to weaving on mechanical looms, warp yarns are often coated with sizing starch or starch derivatives in order to increase their tensile strength and to prevent breaking. The enzymes of the invention can be applied to remove these sizing starch or starch derivatives. After the textiles have been woven, a fabric can proceed to a desizing stage. This can be followed by one or more additional fabric processing steps. Desizing is the act of removing size from textiles. After weaving, the size coating must be removed before further processing the fabric in order to ensure a homogeneous and wash-proof result. The invention provides a method of desizing comprising enzymatic hydrolysis of the size by the action of an enzyme of the invention.","The enzymes of the invention (e.g., enzymes having cellulase, endoglucanase, cellobiohydrolase, mannanase and\/or beta-glucosidase activity) can be used to desize fabrics, including cotton-containing fabrics, as detergent additives, e.g., in aqueous compositions. The invention provides methods for producing a stonewashed look on indigo-dyed denim fabric and garments. For the manufacture of clothes, the fabric can be cut and sewn into clothes or garments, which is afterwards finished. In particular, for the manufacture of denim jeans, different enzymatic finishing methods have been developed. The finishing of denim garment normally is initiated with an enzymatic desizing step, during which garments are subjected to the action of amylolytic enzymes in order to provide softness to the fabric and make the cotton more accessible to the subsequent enzymatic finishing steps. The invention provides methods of finishing denim garments (e.g., a \u201cbio-stoning process\u201d), enzymatic desizing and providing softness to fabrics using the Enzymes of the invention. The invention provides methods for quickly softening denim garments in a desizing and\/or finishing process.","The invention also provides disinfectants comprising enzymes of the invention (e.g., enzymes having cellulase, endoglucanase, cellobiohydrolase, mannanase and\/or beta-glucosidase activity).","The fabric or textile treatment processes of the invention can also include the use of any combination of other enzymes such as tryptophanases or tyrosine decarboxylases, laccases, catalases, laccases, other cellulases, endoglycosidases, endo-beta-1,4-laccases, amyloglucosidases, other glucosidases, glucose isomerases, glycosyltransferases, lipases, phospholipases, lipooxygenases, beta-laccases, endo-beta-1,3(4)-laccases, cutinases, peroxidases, amylases, glucoamylases, pectinases, reductases, oxidases, decarboxylases, phenoloxidases, ligninases, pullulanases, arabinanases, hemicellulases, mannanases, xylolaccases, xylanases, pectin acetyl esterases, rhamnogalacturonan acetyl esterases, proteases, peptidases, proteinases, polygalacturonases, rhamnogalacturonases, galactanases, pectin lyases, transglutaminases, pectin methylesterases, other cellobiohydrolases and\/or transglutaminases.","Paper or Pulp Treatment","The enzymes of the invention e.g., enzymes as described in Tables 1, 2, and 3, including enzymes having cellulase, endoglucanase, cellobiohydrolase, mannanase and\/or beta-glucosidase activity) can be in paper or pulp treatment or paper deinking. For example, in one aspect, the invention provides a paper treatment process using enzymes of the invention. In one aspect, the enzymes of the invention can be used to modify starch in the paper thereby converting it into a liquefied form. In another aspect, paper components of recycled photocopied paper during chemical and enzymatic deinking processes. In one aspect, Enzymes of the invention can be used in combination with other enzymes, including other cellulases (including other endoglucanases, cellobiohydrolases and\/or beta-glucosidases). The wood, paper, paper product or pulp can be treated by the following three processes: 1) disintegration in the presence of an enzyme of the invention, 2) disintegration with a deinking chemical and an enzyme of the invention, and\/or 3) disintegration after soaking with an enzyme of the invention. The recycled paper treated with an enzyme of the invention can have a higher brightness due to removal of toner particles as compared to the paper treated with just cellulase. While the invention is not limited by any particular mechanism, the effect of an enzyme of the invention may be due to its behavior as surface-active agents in pulp suspension.","The invention provides methods of treating paper and paper pulp using one or more polypeptides of the invention. The polypeptides of the invention can be used in any paper- or pulp-treating method, which are well known in the art, see, e.g., U.S. Pat. Nos. 6,241,849; 6,066,233; 5,582,681. For example, in one aspect, the invention provides a method for deinking and decolorizing a printed paper containing a dye, comprising pulping a printed paper to obtain a pulp slurry, and dislodging an ink from the pulp slurry in the presence of an enzyme of the invention (other enzymes can also be added). In another aspect, the invention provides a method for enhancing the freeness of pulp, e.g., pulp made from secondary fiber, by adding an enzymatic mixture comprising an enzyme of the invention (can also include other enzymes, e.g., pectinase enzymes) to the pulp and treating under conditions to cause a reaction to produce an enzymatically treated pulp. The freeness of the enzymatically treated pulp is increased from the initial freeness of the secondary fiber pulp without a loss in brightness.","The paper, wood or pulp treatment or recycling processes of the invention can also include the use of any combination of other enzymes such as tryptophanases or tyrosine decarboxylases, laccases, catalases, laccases, other cellulases, endoglycosidases, endo-beta-1,4-laccases, amyloglucosidases, other glucosidases, glucose isomerases, glycosyltransferases, lipases, phospholipases, lipooxygenases, beta-laccases, endo-beta-1,3(4)-laccases, cutinases, peroxidases, amylases, glucoamylases, pectinases, reductases, oxidases, decarboxylases, phenoloxidases, ligninases, pullulanases, arabinanases, hemicellulases, mannanases, xylolaccases, xylanases, pectin acetyl esterases, rhamnogalacturonan acetyl esterases, proteases, peptidases, proteinases, polygalacturonases, rhamnogalacturonases, galactanases, pectin lyases, transglutaminases, pectin methylesterases, other cellobiohydrolases and\/or transglutaminases.","Repulping: Treatment of Lignocellulosic Materials","The invention also provides a method for the treatment of lignocellulosic fibers, wherein the fibers are treated with a polypeptide of the invention e.g., enzymes as described in Tables 1, 2, and 3, including enzymes having cellulase, endoglucanase, cellobiohydrolase, mannanase and\/or beta-glucosidase activity), in an amount which is efficient for improving the fiber properties. The enzymes of the invention may also be used in the production or recycling of lignocellulosic materials such as pulp, paper and cardboard, from starch reinforced waste paper and cardboard, especially where repulping or recycling occurs at pH above 7 and where the enzymes of the invention can facilitate the disintegration of the waste material through degradation of the reinforcing starch. The enzymes of the invention can be useful in a process for producing a papermaking pulp from starch-coated printed paper. The process may be performed as described in, e.g., WO 95\/14807. An exemplary process comprises disintegrating the paper to produce a pulp, treating with a starch-degrading enzyme before, during or after the disintegrating, and separating ink particles from the pulp after disintegrating and enzyme treatment. See also U.S. Pat. No. 6,309,871 and other US patents cited herein. Thus, the invention includes a method for enzymatic deinking of recycled paper pulp, wherein the polypeptide is applied in an amount which is efficient for effective de-inking of the fiber surface.","Brewing and Fermenting","The invention provides compositions for and methods of brewing (e.g., fermenting) beer comprising an enzyme of the invention, e.g., enzymes as described in Tables 1, 2, and 3, including enzymes having cellulase, endoglucanase, cellobiohydrolase, mannanase and\/or beta-glucosidase activity. In one exemplary process, starch-containing raw materials are disintegrated and processed to form a malt. An enzyme of the invention is used at any point in the fermentation process. For example, enzymes of the invention can be used in the processing of barley malt. The major raw material of beer brewing is barley malt. This can be a three stage process. First, the barley grain can be steeped to increase water content, e.g., to around about 40%. Second, the grain can be germinated by incubation at 15-25\u00b0 C. for 3 to 6 days when enzyme synthesis is stimulated under the control of gibberellins. During this time enzyme levels rise significantly. In one aspect, enzymes of the invention are added at this (or any other) stage of the process. The action of the enzyme results in an increase in fermentable reducing sugars. This can be expressed as the diastatic power, DP, which can rise from around 80 to 190 in 5 days at 12\u00b0 C.","Enzymes of the invention can be used in any beer producing process, as described, e.g., in U.S. Pat. Nos. 5,762,991; 5,536,650; 5,405,624; 5,021,246; 4,788,066.","Pharmaceutical Compositions and Dietary Supplements","The invention also provides pharmaceutical compositions and dietary supplements (e.g., dietary aids) comprising an enzyme of the invention (e.g., an exemplary enzyme of the invention, including those described in Tables 1, 2, and 3). In one aspect, the pharmaceutical compositions and dietary supplements (e.g., dietary aids) are formulated for oral ingestion, e.g., to improve the digestibility of foods and feeds having a high cellulose or lignocellulosic component. See also discussion, above.","Periodontal treatment compounds can comprise an enzyme of the invention, e.g., as described in U.S. Pat. No. 6,776,979. Compositions and methods for the treatment or prophylaxis of acidic gut syndrome can comprise an enzyme of the invention, e.g., as described in U.S. Pat. No. 6,468,964.","In another aspect, wound dressings, implants and the like comprise antimicrobial (e.g., antibiotic-acting) enzymes, including an enzyme of the invention (including, e.g., exemplary sequences of the invention). Enzymes of the invention can also be used in alginate dressings, antimicrobial barrier dressings, burn dressings, compression bandages, diagnostic tools, gel dressings, hydro-selective dressings, hydrocellular (foam) dressings, hydrocolloid dressings, I.V dressings, incise drapes, low adherent dressings, odor absorbing dressings, paste bandages, post operative dressings, scar management, skin care, transparent film dressings and\/or wound closure. Enzymes of the invention can be used in wound cleansing, wound bed preparation, to treat pressure ulcers, leg ulcers, burns, diabetic foot ulcers, scars, IV fixation, surgical wounds and minor wounds. Enzymes of the invention can be used to in sterile enzymatic debriding compositions, e.g., ointments. In various aspects, the cellulase is formulated as a tablet, gel, pill, implant, liquid, spray, powder, food, feed pellet or as an encapsulated formulation.","The polypeptides of the invention, such as those having hydrolases activity (e.g., lipases, esterase, protease and\/or phospholipases) of the invention can be used in detoxification processes, e.g., for the detoxification of endotoxins, e.g., compositions comprising lipopolysaccharides (LPS), and, the invention provides detoxification processes using at least one enzyme of the invention. In one aspect, a lipase and\/or an esterase of the invention is used to detoxify a lipopolysaccharide (LPS). In one aspect, this detoxification is by deacylation of 2\u2032 and\/or 3\u2032 fatty acid chains from lipid A. In one aspect, a hydrolase (e.g., a lipase and\/or an esterase) of the invention is used to hydrolyze a 2\u2032-lauroyl and\/or a 3\u2032-myristoyl chain from a lipid, e.g., a lipid A (e.g., from a bacterial endotoxin). In one aspect, the process of the invention is used to destroy an endotoxin, e.g., a toxin from a gram negative bacteria, as from . In one aspect, a hydrolase (e.g., a lipase and\/or an esterase) of the invention is used to ameliorate the effects of toxin poisoning (e.g., from an on-going gram negative infection), or, to prophylactically to prevent the effects of endotoxin during an infection (e.g., an infection in an animal or a human). Accordingly, the invention provides a pharmaceutical composition comprising a hydrolase (e.g., a lipase and\/or an esterase) of the invention, and method using a hydrolase of the invention, for the amelioration or prevention of lipopolysaccharide (LPS) toxic effects, e.g., during sepsis.","Biodefense Applications","In other aspects, an enzyme of the invention (e.g., an exemplary enzyme of the invention, including those described in Tables 1, 2, and 3) can be used in biodefense (e.g., destruction of spores or bacteria comprising a lignocellulosic material). Use of enzymes of the invention in biodefense applications offer a significant benefit, in that they can be very rapidly developed against any currently unknown or biological warfare agents of the future. In addition, enzymes of the invention can be used for decontamination of affected environments. In aspect, the invention provides a biodefense or bio-detoxifying agent comprising a polypeptide of the invention (including, e.g., exemplary sequences of the invention), or a polypeptide encoded by a nucleic acid of the invention (including, e.g., exemplary sequences of the invention).","Nutraceuticals","In one aspect, the compositions, e.g., an exemplary enzyme of the invention, including those described in Tables 1, 2, and 3, and methods of the invention can be used to make nutraceuticals by processing or synthesizing lipids and oils using the enzymes of the invention, e.g., esterases, acylases, lipases, phospholipases or proteases of the invention. In one aspect, the processed or synthesized lipids or oils include poly-unsaturated fatty acids (PUFAs), diacylglycerides, e.g., 1,3-diacyl glycerides (DAGs), monoacylglycerides, e.g., 2-monoacylglycerides (MAGs) and triacylglycerides (TAGs). In one aspect, the nutraceuticals is made by processing diacylglycerides, e.g., 1,3-diacyl glycerides (DAGs), monoacylglycerides, e.g., 2-monoacylglycerides (MAGs) and\/or triacylglycerides (TAGs) from plant (e.g., oilseed) sources or from animal (e.g., fish oil) sources.","In one aspect, the compositions and methods of the invention can be used to fortify dietary compositions, especially cow's milk based products, e.g., cow's milk-based infant formulas, with bile salt-activated hydrolases. The compositions made by the methods and compositions of the invention can be used to feed newborn and premature infants, including administration of a bile salt-activated hydrolase of the invention to increase fat digestion and therefore growth rate. Similarly, the invention provides compositions and methods for treating subjects for inadequate pancreatic enzyme production by administration of bile salt-activated hydrolase in conjunction with ingestion of fats; see also discussion, below.","In one aspect, the invention provides a dietary composition comprising a hydrolase of the invention, e.g., bile salt-activated hydrolase of the invention. In one aspect, the invention provides a dietary composition comprising a nutritional base comprising a fat and an effective amount of bile salt-activated hydrolase of the invention. In one aspect, the invention provides a cow's milk-based infant formula comprising a hydrolase of the invention, e.g., bile salt-activated hydrolase of the invention. In one aspect, the hydrolase of the invention is active in the digestion of long chain fatty acids, e.g., Cto C, which make up a very high percentage of most milks, e.g., 99% of human breast milk. See, e.g., U.S. Pat. No. 5,000,975.","In one aspect, the invention provides a dietary composition comprising a vegetable oil fat and a hydrolase of the invention. The invention provides methods of processing milk based products and\/or vegetable oil-comprising compositions to make dietary compositions. In one aspect, the processed compositions comprise a lauric acid oil, an oleic acid oil, a palmitic acid oil and\/or a linoleic acid oil. In one aspect, a rice bran oil, sunflower oleic oil and\/or canola oil may be used as oleic acids oils. In one aspect, fats and oils, e.g., oilseeds, from plants, including, e.g., rice, canola, sunflower, olive, palm, soy or lauric type oils for use in the nutraceuticals and dietary compositions are processed or made using a hydrolase of the invention. See, e.g., U.S. Pat. No. 4,944,944.","In one aspect, the enzymes of the invention are provided in a form that is stable to storage in the formula and\/or the stomach, but active when the formulation reaches the portion of the gastrointestinal tract where the formula would normally be digested. Formulations (e.g., microcapsules) for release in the intestine are well known in the art, e.g., biodegradable polymers such as polylactide and polyglycolide, as described, e.g., in U.S. Pat. Nos. 4,767,628; 4,897,268; 4,925,673; 5,902,617.","A number of aspects of the invention have been described. Nevertheless, it will be understood that various modifications may be made without departing from the spirit and scope of the invention. Accordingly, other aspects are within the scope of the following claims."],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF DRAWINGS","p":["The following drawings are illustrative of aspects of the invention and are not meant to limit the scope of the invention as encompassed by the claims.",{"@attributes":{"id":"p-0082","num":"0081"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0083","num":"0082"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0084","num":"0083"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0085","num":"0084"},"figref":"FIG. 4","b":"300"}]},"DETDESC":[{},{}]}
