---
title: Modified oil encapsulating proteins and uses thereof
abstract: The invention provides modified oleosins, including at least one artificially introduced cysteine, and methods and compositions for producing the modified oleosins. Also provided are polynucleotides encoding the modified oleosins, constructs and host cells comprising the polynucleotides, methods for producing oil bodies comprising the modified oleosins, in vivo and in vitro, and methods for producing oil in host cells and plants. The invention also provides methods for increasing the rate of COassimilation in photosynthetic cells and plants, and involves reducing or preventing lipid recycling, and/or expressing modified oleosins with artificially introduced cysteine residues in the photosynthetic cells and plants. Also provided are methods for increasing oil production in plants, via expression of modified oleosins in the non-photosynthetic tissues/organs of plants. The method also optionally includes the step of extracting the oil from the non-photosynthetic tissues/organs of the plant, or processing the oil rich non-photosynthetic tissues/organs into animal or biofuel feedstocks.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08987551&OS=08987551&RS=08987551
owner: Agresearch Limited
number: 08987551
owner_city: East Street Hamilton
owner_country: NZ
publication_date: 20120430
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATION","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","EXAMPLES","Example 1","Creating Rabbit Anti-Sesame Seed Oleosin Antibodies","Example 2","Design and Expression of Modified Oleosins Containing One or More Artificially Introduced Cysteine Residue","Example 3","Use of Anti-Sesame Seed Oleosin Antibodies to Bind Sesame Seed Oleosin with Artificially Introduced Cysteines","Example 4","Creation of Artificial Oil Bodies with Expressed Modified Oleosins Containing at Least One Artificially Introduced Cysteine and Altering the Degree of Cross Linking","Example 5","Design and in Planta Expression of Modified Oleosin Containing One or More Artificially Introduced Cysteines","Example 6","Extraction and Purificiation Oil Bodies with Modified Oleosins Containing at Least One Artificially Introduced Cysteine from the Seeds of ","Example 7","Production of Oil Bodies in the Leaves of ","Example 8","Further Oleosins, Caloleosins and Steroleosins Engineered to Contain Additional Cysteine Residues in the N- and C-Terminal Hydrophilic Arms","Example 9","Increased Biomass Production Through Elevation of Chloroplast COConcentration, Elevation of COAssimilation Rate and Elevation of Intrinsic Water use Efficiency in the Leaves","Example 10","Production of TAG in the Roots of "],"p":["This application is a continuation-in-part of pending International Application PCT\/NZ2010\/000218, filed on Oct. 29, 2010, which claims priority to U.S. Provisional Application 61\/256,689, filed on Oct. 30, 2009. This application also claims priority to U.S. Provisional Application 61\/515,610, filed on Aug. 5, 2011, all of which are incorporated by reference in their entireties to the extent there is no inconsistency with the present disclosure.","The invention relates to compositions and methods for the production and modification of oil bodies in various host cell types. The invention relates to methods for the production of photosynthetic cells and plants with increased COassimilation rates and methods for the production of oil from plants.","In nature, flowering plants efficiently store energy in their seeds through the accumulation of oil, namely triacylglycerol (TAG) and store it in discreet oil bodies by embedding a phospholipid protein monolayer around the oil body. These seed crops have been used in a variety of agricultural applications as feed and more recently also as a feedstock source for biofuels. On a per weight basis, lipids have approximately double the energy content of either proteins or carbohydrates and as such, substantial focus has been placed on raising the oil content of various species, most notably plants. Beyond the energy aspect, the oil bodies themselves also have unique properties and form the basis for a number of biotechnical applications including but not limited to the purification of recombinant proteins, formation of multimeric protein complexes, emulsification and the delivery of bio-actives.","Unfortunately plant seeds represent a very small percentage of total plant biomass and with the demand for improved agricultural productivity and alternative energies it is recognised that current oil production from a number of devoted seed crops is insufficient. Research efforts have focused on not only increasing the productivity of oil production within plant seeds but also oil production in other cell types and species.","Traditional breeding and mutagenesis have offered incremental successes in this area; however genetic engineering has made the furthest strides in modifying organisms to produce elevated oil levels. While certain groups have worked along various parts of the oil synthesis pathway to up-regulate oil production within the seed, others groups have focused on increasing oil in cell types that represent a larger portion of the biomass.","While genetic engineering has made some progress in increasing oil content in certain targets, significant challenges still remain. Further productivity increases can still be realized in oil body production in the seed and the means to produce oil bodies similar to those of a plant seed in other cell types and species has yet to be achieved.","The increasing global population presents demand for higher yielding crops with enhanced production (photosynthetic carbon assimilation).","Ribulose biphosphate carboxlase (Rubisco) is the key enzyme responsible for photosynthetic carbon assimilation. In the presence of O, Rubisco also performs an oxygenase reaction which initiates the photorespiratory cycle which results in an indirect loss of fixed nitrogen and COfrom the cell which need to be recovered. Genetic modification to increase the specificity of Rubisco for COrelative to Oand to increase the catalytic rate of Rubisco in crop plants would be of great agronomic value. Parry et al, (2003) reviewed the progress to date, concluding that there are still many technical barriers to overcome and to date all engineering attempts have thus far failed to produce a better Rubisco (Peterhansel et al. 2008).","In nature, a number of higher plants (C4 plants) have evolved energy requiring mechanisms to increase the concentration of intracellular COin close proximity to Rubisco thereby increasing the proportion of carboxylase reactions. Maize for example has achieved this by a manipulation of the plant's architecture enabling a different initial process of fixing CO, known as C4 metabolism. The agronomic downside of this evolved modification is an increase in leaf fibre resulting in a comparatively poor digestibility of leaves from C4 plants. C4 photosynthesis is thought to be a product of convergent evolution having developed on separate occasions in very different taxa. However, this adaptation is only possible for multi-cellular organisms (and not for photosynthetic unicellular organisms such as algae). Algae have a variety of different mechanisms to concentrate CO2; however, there appears to be a continuum in the degree to which the CO2 concentration mechanism (CCM) is expressed in response to external dissolved inorganic carbon (DIC) concentration, with higher concentrations leading to a greater degree of suppression of CCM activity. Two reviews have covered the CCMs in algae as well as their modulation and mechanisms and are incorporated herein by reference (Giordano, Beardall et al. 2005; Moroney and Ynalvez 2007). The vascular plants that currently constituted the largest percentage of the human staple diet are C3 (rice and tubers) and not C4 plants. Similarly, many oil seed crops (canola, sunflower, safflower) and many meat and dairy animal feed crops (legumes, cereals, soy, forage grasses) are C3 plants.","Increasing the efficiency of COassimilation, should therefore concurrently increase abiotic stress tolerance and nitrogen use efficiency and would be of significant agronomical benefit for C3 plants and photosynthetic microorganisms.","Therefore, mechanisms for elevating COconcentration in the chloroplast, reducing photorespiration and subsequently increasing abiotic stress tolerance and productivity would be of significant agronomical benefit for C3 plants and photosynthetic microorganisms.","It is an object of the invention to provide methods for increasing the rate of COassimilation in photosynthetic cells and plants, and methods for producing photosynthetic cells and plants with an increased rate of COassimilation.","In nature, flowering plants efficiently store energy in their seeds through the accumulation of oil, namely triacylglycerol (TAG) and store it in discreet oil bodies by embedding a phospholipid protein monolayer around the oil body. These seed crops have been used in a variety of agricultural applications as feed and more recently also as a feedstock source for biofuels. On a per weight basis, lipids have approximately double the energy content of either proteins or carbohydrates and as such, substantial focus has been placed on raising the oil content of various species, most notably plants.","Unfortunately plant seeds represent a very small percentage of total plant biomass and with the demand for improved agricultural productivity and alternative energies it is recognised that current oil production from a number of devoted seed crops is insufficient. Research efforts have focused on not only increasing the productivity of oil production within plant seeds but also oil production in other cell types and species.","Traditional breeding and mutagenesis have offered incremental successes in this area; however genetic engineering has made the furthest strides in modifying organisms to produce elevated oil levels. While certain groups have worked along various parts of the oil synthesis pathway to up-regulate oil production within the seed, others groups have focused on increasing oil in cell types that represent a larger portion of the biomass.","It is therefore a further object of the invention to provide methods for increasing the level of oil production in plant tissues\/organs and\/or methods for increasing the production of oil from plants.","The present invention provides compositions and methods for producing oil bodies with varying degrees of stability. The invention involves producing modified oleosins with artificially introduced cysteine residues. The artificially introduced cysteine residues are preferably introduced in the N- and C-terminal hydrophilic arms of the modified oleosins.","Expression of the modified oleosins allows for the creation of stable oil bodies beyond the reproductive tissue of vascular plants into new cell types and even other species. When combined with a TAG synthesising enzyme, the invention leads to the accumulation and storage of TAG in eukaryotic cells as stable oil bodies. Compared with an unmodified cell or even one expressing just a TAG synthesis enzyme, the invention allows for the accumulation of TAG in excess levels achieved by other means.","For example the invention has shown that one can accumulate higher levels of stable oil bodies beyond the seed, in the vegetative portion of vascular plants.","Plants with increased levels of TAG in their vegetative tissues provide a valuable energy source for both animal feedstock and biofuel feedstock applications.","In addition recombinant modified oleosins purified from a host cell (such as ) can be used to generate artificial oil bodies. The modified oleosins in artificial oil bodies, or those purified form transformed cells, can optionally be made to cross-link via the cysteine residues in the modified oleosin. The degree of cross-linking may be controlled manipulating the redox environment. The degree of cross-linking can also be tailored by altering the number of cysteines in the modified oleosins.","Using combinations of these techniques the oil bodies formed with the modified oleosins can be tailored for their emulsification properties, to regulate thermal stability, chemical stability, and peptidase resistance.","The modified oleosins can also be fused to a protein of interest, to form a fusion protein. The fusion protein (modified oleosin plus protein of interest) can be recombinantly expressed in a cell or organism. In this way oil bodies containing the expressed fusion proteins can be used to purify and deliver the protein of interest, for a variety of applications.","In addition the oil bodies can protect, or at least delay, degradation and\/or biohydrogenation, of TAG, within the stomach and\/or rumen of an animal, allowing the intact individual lipids from the TAG to be absorbed by the animal in the intestine. Therefore, the invention is also useful in terms of dietary intake of an animal, particularly through expression of the modified oleosins in plants.","The invention also provides methods for increasing the rate of COassimilation in photosynthetic cells and plants. This aspect of the invention involves reducing lipid recycling and\/or expressing modified oleosins with artificially introduced cysteine residues in the photosynthetic cells and plants.","The invention also provides methods for increasing oil production in plants, via expression of modified oleosins with artificially introduced cysteine residues in the non-photosynthetic tissues\/organs of plants. The applicants have surprisingly shown that the non-photostnthetic tissues\/organs of plants expressing such modified oleosins accumulate oil to a higher level than do other tissues of the plant. The method also optionally includes the step of extracting the oil from the non-photostnthetic tissues\/organs of the plant, or processing the oil rich non-photosynthetic tissues\/organs into animal or biofuel feedstocks","Polynucleotides Encoding Modified Oleosins with Artificially Introduced Cysteines","In the first aspect the invention provides a polynucleotide encoding a modified oleosin including at least one artificially introduced cysteine. The term oleosin also includes steroleosin and caloleosin. The modified oleosin may therefore be selected from a modified oleosin, a modified caloleosin or a modified steroleosin. In one embodiment the modified oleosin is a modified oleosin. In another embodiment the modified oleosin is a modified caloleosin. In another embodiment the modified oleosin is a modified steroleosin. Examples of each type of oleosin (oleosin, caloleosin and steroleosin) are described herein","In one embodiment, the modified oleosin includes at least two cysteines, at least one of which is artificially introduced. In a further embodiment, the modified oleosin includes at least two to at least thirteen (i.e., 2, 3, 4, 5, 6, 7, 8, 9, 10, 1, 12, 13, 14 or more) artificially introduced cysteines. In one embodiment the cysteines are artificially introduced in the N-terminal hydrophilic region of the oleosin, or in the C-terminal hydrophilic region of the oleosin. In a further embodiment the modified oleosin includes at least one cysteine in the N-terminal hydrophilic region, and at least one cysteine in the C-terminal hydrophilic region. In a further embodiment the cysteines are distributed substantially evenly over the N-terminal and C-terminal hydrophilic regions of the oleosin.","In a further embodiment the polynucleotide encodes a fusion protein including the modified oleosin fused to a protein of interest.","Constructs","In a further aspect the invention provides a genetic construct comprising a polynucleotide of the invention. In a further aspect the invention provides an expression construct comprising a polynucleotide of the invention. In one embodiment the polynucleotide in the construct is operably linked to a promoter sequence. In one embodiment the promoter sequence is capable of driving expression of the polynucleotide in a vegetative tissue of a plant. In a further embodiment the promoter sequence is capable of driving expression of the polynucleotide in a seed of a plant. In a further embodiment the promoter sequence is capable of driving expression of the polynucleotide in the pollen of a plant. In a further embodiment the promoter sequence is capable of driving expression of the polynucleotide in an cell. In a further embodiment the promoter sequence is capable of driving expression of the polynucleotide in a yeast cell. In a further embodiment the promoter sequence is capable of driving expression of the polynucleotide in an algal cell.","In another aspect, the invention provides a construct containing a polynucleotide that encodes a modified neutral lipid protein. In one embodiment, the construct also contains a second polynucleotide that encodes a triacylglycerol (TAG) synthesizing enzyme. In various embodiments, the construct can be linked to a promoter sequence capable of driving its expression in various host cells. As such, the invention also provides use of the constructs to induce a host cell to express a modified oleosin and\/or a TAG synthesizing enzyme. In various embodiments, the construct expressing a modified oleosin and the construct expressing a TAG synthesizing enzyme may be driven by the same or by different promoters. In yet another embodiment the construct is located in an appropriate position and orientation of a suitable functional endogenous promoter such that the expression of the construct occurs. In various embodiments, the construct can be expressed in a bacterial, plant, fungal or algal cell. In one embodiment where the construct is expressed in a plant cell, the cell may be of vegetative, seed, pollen or fruit tissue.","Host Cells","In a further aspect the invention provides a host cell comprising a construct of the invention. In a further aspect the invention provides a host cell genetically modified to comprise a polynucleotide of the invention. In a further aspect the invention provides a host cell genetically modified to express a polynucleotide of the invention.","Host Cell Also Expressing a Tag Synthesising Enzyme","In a further embodiment the host cell is also genetically modified to express a triacylglycerol (TAG) synthesising enzyme. In a further embodiment the host cell is genetically modified to comprise a nucleic acid sequence encoding a triacylglycerol (TAG) synthesising enzyme. In a further embodiment the host cell comprises an expression construct including a nucleic acid sequence encoding a triacylglycerol (TAG) synthesising enzyme.","In a further embodiment the nucleic acid is operably linked to a promoter sequence. In a further embodiment the promoter sequence is capable of driving expression of the nucleic acid sequence in a vegetative tissue of a plant. In one embodiment the promoter sequence is capable of driving expression of the nucleic acid sequence in a seed of a plant. In one embodiment the promoter sequence is capable of driving expression of the nucleic acid sequence in the pollen of a plant.","In a further embodiment the promoter sequence is capable of driving expression of the polynucleotide in an cell. In a further embodiment the promoter sequence is capable of driving expression of the polynucleotide in a yeast cell. In a further embodiment the promoter sequence is capable of driving expression of the polynucleotide in an algal cell.","Host Cell Types","The host cell may be any type of cell. In on embodiment the host cell is a prokaryotic cell. In a further embodiment the host cell is a eukaryotic cell. In one embodiment the host cell is selected from a bacterial cell, a yeast cell, a fungal cell, an insect cell, algal cell, and a plant cell. In one embodiment the host cell is a bacterial cell. In a further embodiment the host cell is a yeast cell. In further embodiment the host cell is a fungal cell. In further embodiment the host cell is an insect cell. In further embodiment the host cell is an algal cell. In a further embodiment the host cell is a plant cell.","Plants","In a further aspect the invention provides a plant comprising a plant cell of the invention. In a further aspect the invention provides a plant comprising a construct of the invention. In a further aspect the invention provides a plant genetically modified to comprise a polynucleotide of the invention. In a further aspect the invention provides a plant genetically modified to express a polynucleotide of the invention. In a further embodiment the plant expresses a modified oleosin encoded by the polynucleotide of the invention.","In a further embodiment the modified oleosin is expressed in a vegetative tissue of the plant. In a further embodiment the modified oleosin is expressed in a seed of the plant. In a further embodiment the modified oleosin is expressed in the pollen of the plant.","Plant Also Expresses a TAG Enzyme","In a further embodiment the plant is also genetically modified to express a triacylglycerol (TAG) synthesising enzyme. In a further embodiment the triacylglycerol (TAG) synthesising enzyme is expressed in the same tissue as the modified oleosin.","In a further embodiment the plant is genetically modified to comprise a nucleic acid sequence encoding a triacylglycerol (TAG) synthesising enzyme. In a further embodiment the plant comprises an expression construct including a nucleic acid sequence encoding a triacylglycerol (TAG) synthesising enzyme.","In a further embodiment the nucleic acid is operably linked to a promoter sequence.","In a further embodiment the promoter sequence is capable of driving expression of the nucleic acid sequence in a vegetative tissue of a plant. In one embodiment the promoter sequence is capable of driving expression of the nucleic acid sequence in a seed of a plant. In one embodiment the promoter sequence is capable of driving expression of the nucleic acid sequence in the pollen of a plant.","Modified Oleosin Polypeptides with Artificially Introduced Cysteines","In a further aspect the invention provides a modified oleosin including at least one artificially introduced cysteine. In a further aspect the invention provides a modified oleosin encode by a polynucleotide of the invention. In one embodiment, the modified oleosin includes at least two cysteines, at least one of which is artificially introduced. In a further embodiment, the modified oleosin includes at least two to at least thirteen (i.e., 2, 3, 4, 5, 6, 7, 8, 9, 10, 1, 12, 13, 14 or more) artificially introduced cysteines.","In a further embodiment the modified oleosin includes at least one cysteine in the N-terminal hydrophilic region, and at least one cysteine in the C-terminal hydrophilic region. In a preferred embodiment the cysteins are artificially introduced in the N-terminal hydrophilic region of the oleosin, or in the C-terminal hydrophilic region of the oleosin. Preferably the cysteins are distributed substantially evenly between the N-terminal and C-terminal hydrophilic region of the oleosin.","Fusion Proteins with Modified Oleosins Including Artificially Introduced Cysteines","In a further aspect the invention provides a fusion protein comprising a modified oleosin of the invention and a protein of interest. The fusion protein thus comprises a modified oleosin portion, and a protein of interest portion.","Oil Bodies Comprising Modified Oleosins","In a further aspect the invention provides an oil body comprising a modified oleosin of the invention. In a further aspect the invention provides an oil body comprising at least two modified oleosins of the invention. In one embodiment at least two of the modified oleosins are cross-linked to each other via disulphide bridges between cysteine residues in the modified oleosins. In a further embodiment the modified oleosins are cross-linked via the artificially introduced cysteine residues in the modified oleosins.","In a further embodiment the oil body additionally comprises a fusion protein, wherein the fusion protein includes an oleosin fused to a protein of interest. In this embodiment, the oleosin in the fusion protein need not include an artificially introduced cysteine. Preferably the oleosin in the fusion protein does not include an artificially introduced cysteine.","The oil bodies of this embodiment are useful for purifying and delivering the protein of interest, as discussed in Roberts et al., (2008).","However in this embodiment it is possible to take advantage of the option to vary the stability\/integrity of the oil body provided by presence of the modified oleosins in the oil body, hence allowing for more stringent purification and delivery procedures.","Oil Bodies Comprising Fusion Proteins with Modified Oleosisn","In a further aspect the invention provides an oil body comprising a fusion protein of the invention, the fusion protein comprising a modified oleosin of the invention and a protein of interest. The fusion protein thus comprises a modified oleosin portion, and a protein of interest portion.","In one embodiment the oil body comprises at least two fusion proteins of the invention.","In one embodiment at least two of the fusion proteins are cross-linked to each other via disulphide bridges between cysteine residues in the modified oleosin portion of the fusion proteins. In one embodiment the fusion proteins are cross-linked via the artificially introduced cysteine residues in the modified oleosin portion of the fusion proteins.","In a further embodiment the oil body comprises at least one modified oleosin of the invention. In a further embodiment at least one fusion protein is cross-linked to at least one modified oleosin, via a cysteine in the modified oleosin portion of the fusion protein and a cysteine in the modified oleosin.","Again, the oil bodies of this embodiment are useful for purifying and delivering the protein of interest, as discussed in Roberts et al., (2008).","However in this embodiment it is possible to take advantage of the option to vary the stability\/integrity of the oil body provided by presence of the modified oleosins in the oil body, hence allowing for more stringent purification and delivery procedures.","Emulsion","In a further aspect the invention provides an emulsion comprising a modified oleosin of the invention. In one embodiment the emulsion comprises the modified oleosin and a suitable carrier. The carrier may be buffered, with the appropriate redox environment to retain the desired degree of cross-linking of the oleosins.","To resuspend the modified oleosin in the carrier may require sonication or high pressure homogenising, followed by exposure to the appropriate oxidising conditions.","Compositions","In a further aspect the invention provides a composition comprising a modified oleosin of the invention. In one embodiment the composition comprises the modified oleosin and a suitable carrier. The carrier may be buffered, with the appropriate redox environment to attain the desired degree of cross-linking of the modified oleosins.","To resuspend the modified oleosins in the carrier may require sonication or high pressure homogenising, followed by exposure to the appropriate oxidising conditions.","In a further aspect the invention provides a composition comprising an oil body of the invention. In one embodiment the composition comprises the oil body and a suitable carrier. The carrier may be buffered, with the appropriate redox environment to retain the desired degree of cross-linking of the modified oleosins. In a further embodiment the invention provides a composition formulated for dermal application comprising an oil body of the invention.","Plants, and Parts Thereof, Comprising Oil Bodies of the Invention","In a further aspect the invention provides a plant, or part thereof, comprising an oil body of the invention. In a further aspect the invention provides a vegetative tissue of a plant, comprising an oil body of the invention. In a further aspect the invention provides a seed of a plant, comprising an oil body of the invention.","Animal Feed Comprising Oil Bodies of the Invention","In a further aspect the invention provides an animal feed comprising an oil body of the invention. In a further aspect the invention provides an animal feed comprising a plant, or part thereof, of the invention.","Methods for Producing Oil Bodies","In a further aspect invention provides a method for producing an oil body, the method comprising the step of combining:\n\n","In one embodiment, the modified oleosins each include at least two cysteines, at least one of which is artificially introduced. In a further embodiment the modified oleosins each include at least one cysteine in the N-terminal hydrophilic region of the oleosin, and at least one cysteine in the C-terminal hydrophilic region of the oleosin.","In a further embodiment, the modified oleosin includes at least two to at least thirteen (i.e., 2, 3, 4, 5, 6, 7, 8, 9, 10, 1, 12, 13, 14 or more) artificially introduced cysteines.","In one embodiment the cysteines are artificially introduced in the N-terminal hydrophilic region of the oleosins, or in the C-terminal hydrophilic region of the oleosins. In a further embodiment the cysteines are distributed substantially evenly between the N-terminal and C-terminal hydrophilic region of the oleosins. In a further embodiment the modified oleosins are cross-linked via disulphide bridges between cysteine residues in the oleosins. In a further embodiment embodiment the modified oleosins are cross-linked between the artificially introduced cysteine residues in the oleosins.","In one embodiment the modified oleosins are part of fusion proteins wherein the fusion proteins comprise a modified oleosin, and a protein of interest.","In one embodiment the method comprises the additional step of regulating the degree of cross-linking of modified oleosins in the oil body by controlling the redox environment of the oil body produced.","All Components Combined In Vivo (In Vivo Oil Bodies)","In one embodiment the components of a), b) and c) are combined within a host cell. In this embodiment the modified oleosins are preferably expressed in the host cell.","The host cell is preferably genetically modified to express the modified oleosins.","The host cell is preferably comprises a construct of the invention. The host cell is preferably genetically modified to comprise a polynucleotide of the invention. The host cell is preferably genetically modified to express a polynucleotide of the invention.","Host Cell Also Expresses a TAG Synthesising Enzyme","In a further embodiment the host cell is also genetically modified to express a triacylglycerol (TAG) synthesising enzyme. In a further embodiment the host cell comprises an expression construct including a nucleic acid sequence encoding a triacylglycerol (TAG) synthesising enzyme.","In a further embodiment the nucleic acid sequence is operably linked to a promoter sequence. In one embodiment the promoter sequence is capable of driving expression of the nucleic acid sequence in a vegetative tissue of a plant. In one embodiment the promoter sequence is capable of driving expression of the nucleic acid sequence in a seed of a plant. In one embodiment the promoter sequence is capable of driving expression of the nucleic acid sequence in the pollen of a plant.","In a further embodiment the host cell is also genetically modified to comprise a nucleic acid sequence encoding a triacylglycerol (TAG) synthesising enzyme. In a further embodiment the host cell is also genetically modified to express a nucleic acid sequence encoding a triacylglycerol (TAG) synthesising enzyme.","It will be understood by those skilled in the art that the polynucleotide encoding the modified oleosin and the nucleic acid sequence encoding a triacylglycerol (TAG) synthesising enzyme can be placed on the same construct or on separate constructs to be transformed into the host cell. Expression of each can be driven by the same or different promoters, which may be included in the construct to be transformed. It will also be understood by those skilled in the art that alternatively the polynucleotide and nucleic acid can be transformed into the cell without a promoter, but expression of either the polynucleotide and nucleic acid could be driven by a promoter or promoters endogenous to the cell transformed.","In a further embodiment the host cell forms part of an organism. In a preferred embodiment the organism is a plant.","In a further embodiment the oil is produced in the vegetative tissues of the plant.","In one embodiment of the method the plant acumulates about 50% to about 400% more lipid than does a suitable control plant. In a further embodiment of the method the plant acumulates about 100% to about 300% more lipid than does a suitable control plant. In a further embodiment of the method the plant acumulates about 150% to about 250% more lipid than does a suitable control plant. Suitable control plants include non-transformed or wild-type versions of plant of the same variety and or species as the transformed plant used in the method of the invention.","In a further embodiment the plant is processed into an animal feed.","In a further embodiment the plant is processed into a biofuel feed stock.","Additional Method Step to Purify the In Vivo Produced Oil Bodies","In one embodiment the method includes the additional step of purifying the oil bodies from the cell or organisim.","Additional Method Step to Vary Degree of Cross-Linking of In Vivo Produced Purified Oil Bodies","In a further embodiment the method comprises the additional step of regulating the degree of cross-linking of modified oleosins in the in vivo produced purified oil bodies by controlling the redox environment of the purified oil bodies. In one embodiment the degree of cross-linking is increased by use of an oxidising environment. In a further embodiment the degree of cross-linking is decreased by use of a reducing environment.","Components Combined In Vitro (In Vitro\/Artificial Oil Bodies)","In certain embodiments the components of a), b) and c) may be combined in vitro.","In one embodiment, the modified oleosin of a) has been recombinantly expressed in, and purified from a host cell of the invention, before being combined with the components of b) and c).","Additional Method Step to Vary Degree of Cross-Linking of In Vitro\/Artificial Oil Bodies","In a further embodiment the method comprises the additional step of regulating the degree of cross-linking by controlling the redox environment in which the components of a), b) and c) are combined. In one embodiment the degree of cross-linking is increased by combining the components of a), b) and c) in on oxidising environment. In a further embodiment the degree of cross-linking is decreased by combining the components of a), b) and c) in a reducing environment. The degree of cross-linking may also be regulated after the oil body is formed, by controlling the redox environment in which the oil body is contained.","In a further aspect the invention provides a method of producing a plant that accumulates more oil than a suitable control plant the method comprising providing a plant transformed with a polynucleotide of the invention that expresses a modified oleosin encode by the polynucleotide.","In one embodiment the plant is also transformed with a polynucleotide encoding a TAG synthesising enzyme to express the TAG synthesising enzyme and thus synthesise TAG.","In one embodiment the plant the plant is produced by transforming a single plant, or plant cell, with both the polynucleotide of any one the invention and the polynucleotide encoding the TAG synthesising enzyme.","In a further embodiment the plant is produced by crossing a first plant transformed with a polynucleotide of any one of the invention, with second plant transformed the polynucleotide encoding the TAG synthesising enzyme, to produce the plant transformed with both a polynucleotide of the invention, and a polynucleotide encoding the TAG synthesising enzyme.","In a further embodiment the oil is TAG. In a further embodiment the oil is produced in the vegetative tissues of the plant.","In one embodiment of the method the plant acumulates about 50% to about 400% more lipid than does a suitable control plant. In a further embodiment of the method the plant acumulates about 100% to about 300% more lipid than does a suitable control plant. In a further embodiment of the method the plant acumulates about 150% to about 250% more lipid than does a suitable control plant","In a further embodiment the plant is processed into an animal feed.","In a further embodiment the plant is processed into a biofuel feed stock.","In a further aspect invention provides a method for producing an oil body in a host cell, the method comprising:\n\n","In a further aspect invention provides a method for producing an oil body in a host cell, the method comprising:\n\n","The host cell may be a host cell as herein described.","Oil Bodies","In a further aspect invention provides an oil body produced by a method of the invention.","Compositions","In a further aspect the invention provides a composition comprising an oil body of the invention. In one embodiment the composition comprises the oil body and a suitable carrier. The carrier may be buffered to provide the appropriate redox environment to retain the desired degree of cross-linking of the modified oleosin. In a further embodiment the invention provides a composition formulated for dermal application comprising an oil body of the invention.","Plants, and Parts Thereof, Comprising Oil Bodies of the Invention","In a further aspect the invention provides a plant, or part thereof, comprising an oil body of the invention. In a further aspect the invention provides a vegetative tissue of a plant, comprising an oil body of the invention. In a further aspect the invention provides a seed of a plant, comprising an oil body of the invention. In a further aspect the invention provides pollen of a plant, comprising an oil body of the invention. In a further aspect the invention provides a fruit, or fruiting body, of a plant, comprising an oil body of the invention.","Animal Feed Comprising Oil Bodies of the Invention","In a further aspect the invention provides an animal feed comprising an oil body of the invention. In a further aspect the invention provides an animal feed comprising a plant, or part thereof, of the invention.","In one embodiment the feed is suitable for a mammalian animal including humans. In a further embodiment the feed is suitable for non-human mammals. Preferred animals include farm animals such as but not limited to cows, sheep, horses, goats, pigs, chickens, and the like.","Plants","The modified oleosins may be modified naturally occurring oleosins. The plants from which the un-modified oleosin sequences are derived may be from any plant species that contains oleosins and polynucleotide sequences encoding oleosins.","The plant cells, in which the modified oleosins are expressed, may be from any plant species. The plants, in which the modified oleosins are expressed, may be from any plant species.","In one embodiment the plant cell or plant, is derived from a gymnosperm plant species.","In a further embodiment the plant cell or plant, is derived from an angiosperm plant species.","In a further embodiment the plant cell or plant, is derived from a from dicotyledonous plant species.","In a further embodiment the plant cell or plant, is derived from a monocotyledonous plant species.","Other preferred plants are forage plant species from a group comprising but not limited to the following genera: and ","Other preferred plants are leguminous plants. The leguminous plant or part thereof may encompass any plant in the plant family Leguminosae or Fabaceae. For example, the plants may be selected from forage legumes including, alfalfa, clover; leucaena; grain legumes including, beans, lentils, lupins, peas, peanuts, soy bean; bloom legumes including lupin, pharmaceutical or industrial legumes; and fallow or green manure legume species.","A particularly preferred genus is . Preferred species include ; and . A particularly preferred species is ","Another preferred genus is . Preferred species include and . A particularly preferred species is , commonly known as alfalfa.","Another preferred genus is . Preferred species include and (also known as ). A particularly preferred species is , commonly known as soy bean. A particularly preferred species is , commonly known as perennial soybean.","Another preferred genus is . A particularly preferred species is commonly known as cowpea.","Another preferred genus is . Preferred species include . A particularly preferred species is commonly known as velvetbean.","Another preferred genus is . A particularly preferred species is commonly known as perennial peanut.","Another preferred genus is . A preferred species is commonly known as pea.","Another preferred genus is . Preferred species include and . A preferred species is commonly known as Birdsfoot Trefoil. Another preferred species is commonly known as Narrow-leaf Birdsfoot Trefoil. Another preferred species is commonly known as Big trefoil. Another preferred species is commonly known as Slender trefoil.","Another preferred genus is . A preferred species is , commonly known as forage kale and cabbage.","Other preferred species are oil seed crops including but not limited to the following genera: and ","A preferred oil seed genera is . A preferred oil seed species is ","A preferred oil seed genera is . A preferred oil seed species is ","A preferred oil seed genera is . A preferred oil seed species is ","A preferred oil seed genera is . A preferred oil seed species is ","A preferred oil seed genera is . A preferred oil seed species is ","A preferred oil seed genera is . A preferred oil seed species is ","A preferred oil seed genera is . A preferred oil seed species is ","A preferred silage genera is . A preferred silage species is ","A preferred grain producing genera is . A preferred grain producing species is ","A preferred grazing genera is . A preferred grazing species is ","A preferred grazing genera is . A preferred grazing species is ","A preferred grazing genera is . A preferred grazing species is ","A preferred grazing genera is . A preferred grazing species is ","Preferred plants also include forage, or animal feedstock plants. Such plants include but are not limited to the following genera: ","A preferred biofuel genera is . A preferred biofuel species is ","A preferred biofuel genera is . A preferred biofuel species is ","A preferred biofuel genera is . A preferred biofuel speices is ","In a further aspect the invention provides a method for producing a photosynthetic cell with an increased rate of COassimilation, the method comprising at least one of the steps:\n\n","In one embodiment the method comprises the step of genetically modifying the photosynthetic cell to reduce or prevent lipid recycling.","In another embodiment, the method comprises the step of transforming the photosynthetic cell with a polynucleotide encoding a modified oleosin including at least one artificially introduced cysteine.","In one embodiment the cell is genetically modified to prevent lipid recycling, by transforming the photosynthetic cell with a polynucleotide encoding a modified oleosin including at least one artificially introduced cysteine.","In a preferred embodiment the modified oleosin is expressed in the photosynthetic cell. In one embodiment expression of the modified oleosin causes the increased rate of COassimilation. In one embodiment, expression of the modified oleosin reduces or prevents lipid recycling in the photosynthetic cell. In a preferred embodiment the reduced or prevented lipid recycling causes the increased COassimilation.","In a further embodiment the lipid recycling is initiated by the action of lipases releasing free fatty acids from a glycerol backbone. In a further embodiment the lipid recycling is driven by the reincorporation of fatty acids into glycerol backbones within the endoplasmic reticulum of the cell.","In one embodiment the rate of COassimilation is increased by at least 1%, more preferably at least 2%, more preferably at least 3%, more preferably at least 4%, more preferably at least 5%, more preferably at least 10%, more preferably at least 15%, more preferably at least 20%, more preferably at least 25%, more preferably at least 30%, more preferably at least 35%, more preferably at least 40%, more preferably at least 45%, more preferably at least 50%, relative to a control plant.","In one embodiment the rate of COassimilation increase is in the range of 1% to 50%, more preferably 2% to 40%, more preferably 3% to 30%, more preferably 4% to 25%, more preferably 5% to 20%, relative to a control plant.","In one embodiment the increase in COassimilation results from an elevated concentration of COin the chloroplast.","Modified Oleosin","The term oleosin also includes steroleosin and caloleosin. The modified oleosin may therefore be selected from a modified oleosin, a modified caloleosin or a modified steroleosin. In one embodiment the modified oleosin is a modified oleosin. In another embodiment the modified oleosin is a modified caloleosin. In another embodiment the modified oleosin is a modified steroleosin. Examples of each type of oleosin (oleosin, caloleosin and steroleosin) are described herein","In one embodiment, the modified oleosin includes at least two cysteines, at least one of which is artificially introduced. In a further embodiment, the modified oleosin includes at least two to at least thirteen (i.e., 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 or more) artificially introduced cysteines. In one embodiment the cysteines are artificially introduced in the N-terminal hydrophilic region of the oleosin, or in the C-terminal hydrophilic region of the oleosin. In a further embodiment the modified oleosin includes at least one cysteine in the N-terminal hydrophilic region, and at least one cysteine in the C-terminal hydrophilic region. In a further embodiment the cysteines are distributed substantially evenly over the N-terminal and C-terminal hydrophilic regions of the oleosin. In a further embodiment the cysteines are distributed evenly over the N-terminal and C-terminal hydrophilic regions of the oleosin.","Other Associated Phenotypes of the Photosynthetic Cell","In a further embodiment, in addition to the increased rate of COassimilation the method produces a photosynthetic cell with at least one of:\n\n","Preferably the photosynthetic cell produced has all of a) to i).","Promoters","In one embodiment the polynucleotide is operably linked to a promoter polynucleotide.","In one embodiment the promoter is capable of driving expression of the polynucleotide in a photosynthetic cell. In one embodiment the promoter drives expression of the polynucleotide preferentially in photosynthetic cells. In one embodiment the promoter is a photosynthetic cell preferred promoter. In a further embodiment the promoter is a photosynthetic cell specific promoter. In a further embodiment the promoter is a light regulated promoter.","Polynucleotide is Part of a Genetic Construct","In one embodiment the polynucleotide is transformed as part of a genetic construct. Preferably the genetic construct is an expression construct. Preferably the expression construct includes the polynucleotide operably linked to the promoter. In a further embodiment the polynucleotide is operably linked to a terminator sequence","Photosynthetic Cell is Also Transformed with a TAG Synthesising Enzyme","In a further embodiment the photosynthetic cell is also genetically modified to express a triacylglycerol (TAG) synthesising enzyme. In a further embodiment the photosynthetic cell is genetically modified to comprise a nucleic acid sequence encoding a triacylglycerol (TAG) synthesising enzyme. In a further embodiment the photosynthetic cell comprises an expression construct including a nucleic acid sequence encoding a triacylglycerol (TAG) synthesising enzyme.","In a further embodiment the nucleic acid is operably linked to a promoter polynucleotide.","In one embodiment the promoter is capable of driving expression of the polynucleotide in the photosynthetic cell. In one embodiment the promoter is a photosynthetic cell preferred promoter. In a further embodiment the promoter is a photosynthetic cell specific promoter. In a further embodiment the promoter is a light regulated promoter.","It will be understood by those skilled in the art that the polynucleotide encoding the modified oleosin and the nucleic acid sequence encoding a triacylglycerol (TAG) synthesising enzyme can be placed on the same construct or on separate constructs to be transformed into the host cell. Expression of each can be driven by the same or different promoters, which may be included in the construct to be transformed. It will also be understood by those skilled in the art that alternatively the polynucleotide and nucleic acid can be transformed into the cell without a promoter, but expression of either or both of the polynucleotide and nucleic acid could be driven by a promoter or promoters endogenous to the cell transformed.","Photosynthetic Cell Types","The photosynthetic cell may be of any type. In one embodiment the photosynthetic cell is a prokaryotic cell. In a further embodiment the photosynthetic cell is a eukaryotic cell. In one embodiment the photosynthetic cell is selected from a photosynthetic bacterial cell, a photosynthetic yeast cell, a photosynthetic fungal cell, a photosynthetic algal cell, and a plant cell. In one embodiment the photosynthetic cell is a bacterial cell. In a further embodiment the photosynthetic cell is a yeast cell. In further embodiment the photosynthetic cell is a fungal cell. In further embodiment the photosynthetic cell is an algal cell.","Photosynthetic Cell is an Algal Cell","In a preferred embodiment the photosynthetic cell is an algal cell. In one embodiment the photosynthetic algal cell is an algal cell selected from one of the following divisions: Chlorophyta (green algae), Rhodophyta (red algae), Phaeophyceae (brown algae), Bacillariophycaeae (diatoms), and Dinoflagellata (dinoflagellates).","In one embodiment the algal cell shows an increased growth rate, relative to a control algal cell, at an elevated concentration of oxygen (O).","In a further embodiment, the concentration of Ois elevated to at least 1.1 times air saturation, more preferably at least 1.5 times air saturation, more preferably at least 2 times air saturation, more preferably at least 4 times air saturation, more preferably at least 8 times air saturation, more preferably at least 16 times air saturation.","In a further embodiment, the increased growth rate of the algal cell is at least 10%, more preferably at least 20%, more preferably at least 30%, more preferably at least 40%, more preferably at least 50%, more preferably at least 60%, more preferably at least 70%, more preferably at least 80%, more preferably at least 90%, more preferably at least 100% more than the growth rate of a control algal cell at the same Oconcentration.","In a further embodiment, the increased growth rate of the algal cell is in the range 10% to about 130% more preferably 20% to 120%, more preferably 30% to 110%, more preferably 40% to 100%, more preferably 50% to 90%, more than the growth rate of a control algal cell at the same Oconcentration.","Photosynthetic Cell is Part of a Plant.","In a further preferred embodiment the photosynthetic cell is a plant cell. In a preferred embodiment the plant cell is part of a plant.","Thus the invention provides a method for producing a photosynthetic cell, or plant, with an increased rate of COassimilation.","Promoters for Plants","In one embodiment the promoter operably linked to the polynucleotide is capable of driving expression of the polynucleotide in a photosynthetic tissue of a plant. In one embodiment the promoter is a photosynthetic cell preferred promoter. In a further embodiment the promoter is a photosynthetic cell specific promoter. In a further embodiment the promoter is capable of driving expression of the polynucleotide in a vegetative photosynthetic tissue of a plant. In a further embodiment the promoter is capable of driving expression of the polynucleotide in a leaf of a plant.","Further Associated Phenotypes for Plants","In a further embodiment, in addition to the increased rate of COassimilation the plant also has at least one of:\n\n","Preferably the plant has all of a) to c).","In a further embodiment, in addition to the increased rate of COassimilation the plant also has at least one of:\n\n","Preferably the plant has all of a) to m).","In one embodiment biomass is increased by at least 5%, preferably by at least 10%, preferably by at least 20%, preferably by at least 30%, preferably by at least 40%, preferably by at least 50%, preferably by at least 60% relative to a control plant.","In one embodiment the increase in biomass is in the range 2% to 100%, preferably 4% to 90%, preferably 6% to 80%, preferably 8% to 70%, preferably 10% to 60% relative to a control plant.","Plant Types","In one embodiment the plant is a C3 plant.","In one embodiment the plant is selected from: rice, soybean, wheat, rye, oats, millet, barley, potato, canola, sunflower and safflower.","Preferred plants include those from the following genera: and ","Preferred species include and ","Preferred species include and (also known as ). A particularly preferred species is , commonly known as soy bean. A particularly preferred species is , commonly known as perennial soybean.","A preferred species is ","Preferred species include and ","A preferred species is cereal.","A preferred species is ","Preferred millet species include and ","Preferred species include and ","Preferred species include , and ","Preferred species include and ","Preferred species include and ","A preferred species is ","In a further aspect the invention provides a method for producing oil, the method comprising the steps:\n\n","In one embodiment the plant accumulates more total lipid in its non-photosynthetic tissues\/organs than does a control plant.","In a further embodiment the plant accumulates at least 10%, more preferably at least 20%, more preferably at least 50%, more preferably at least 100%, more preferably 150%, more preferably 200%, more preferably 250%, more preferably 300%, more preferably 350%, more preferably 400%, more preferably 450%, more preferably 500%, more total lipid in its non-photosynthetic tissues\/organs than does a control plant.","In one embodiment the plant produces total lipid in its non-photosynthetic tissues\/organs in the range 100% to 900%, more preferably 200% to 800%, more preferably 300% to 700%, more preferably 400% to 600%, more than a control plant.","Level Oil Production in Non-Photosynthetic Tissues\/Organs.","In one embodiment the plant accumulates more oil in its non-photosynthetic tissues\/organs than does a control plant.","In one embodiment the plant accumulates at least 1.2\u00d7, at least 1.5\u00d7, at least 2\u00d7, more preferably at least 3\u00d7, more preferably at least 4\u00d7, more preferably at least 5\u00d7, more preferably at least 6\u00d7, more preferably at least 7\u00d7, more preferably at least 8\u00d7, more preferably at least 9\u00d7, more preferably at least 10\u00d7, more preferably at least 11\u00d7, more preferably at least 12\u00d7, more preferably at least 13\u00d7, more preferably at least 14\u00d7, more preferably at least 15\u00d7, more oil in its non-photosynthetic tissues\/organs than does a control plant.","In one embodiment the plant produce oil in its non-photosynthetic tissues\/organs in the range 3\u00d7 to 15\u00d7, more preferably 4\u00d7 to 14\u00d7, more preferably 5\u00d7 to 13\u00d7, more preferably 6\u00d7 to 12\u00d7, more preferably 7\u00d7 to 11\u00d7, more preferably 8\u00d7 to 10\u00d7 more than a control plant.","Suitable control plants include non-transformed or wild-type versions of plant of the same variety and\/or species as the transformed plant used in the method of the invention. Suitable control plants also include plants of the same variety and or species as the transformed plant that are transformed with a control construct. Suitable control plants also include plants that have not been transformed with a polynucleotide encoding a modified oleosin including at least one artificially introduced cysteine. Suitable control plants also include plants that do not express a modified oleosin including at least one artificially introduced cysteine.","In a preferred embodiment oil is produced at a higher level in the non-photosynthetic tissues\/organs than in other tissues\/organs of the plant.","Preferably the increased level of oil production in the non-photosynthetic tissues\/organs is caused by expression of the modified oleosin in the non-photosynthetic tissues\/organs.","Non-Photosynthetic Tissues\/Organs","In one embodiment the non-photosynthetic tissue\/organ is selected from below ground tissue\/organs of the plant. In a further embodiment the below ground tissue\/organ is selected from root, tuber, bulb, corm and rhizome. In a further embodiment the non-photosynthetic tissue\/organ is selected from root, tuber, bulb, corm, rhizome, and endosperm. In a further embodiment the non-photosynthetic tissue\/organ is root.","Genetic Modification","In one embodiment the method includes the step of transforming the plant with the polynucleotide encoding a modified oleosin including at least one artificially introduced cysteine.","Promoters","In one embodiment the polynucleotide is operably linked to a promoter polynucleotide.","In one embodiment the promoter is capable of driving expression of the polynucleotide in the non-photosynthetic tissues of the plant. In one embodiment the promoter is a constitutive promoter. In one embodiment the promoter is a non-photosynthetic tissue preferred promoter. In a further embodiment the promoter is a root preferred promoter. In a further embodiment the promoter is a root specific promoter. In a further embodiment the promoter is a tuber preferred promoter. In a further embodiment the promoter is a tuber specific promoter. In a further embodiment the promoter is a bulb preferred promoter. In a further embodiment the promoter is a bulb specific promoter. In a further embodiment the promoter is a corm preferred promoter. In a further embodiment the promoter is a corm specific promoter. In a further embodiment the promoter is a rhizome preferred promoter. In a further embodiment the promoter is a rhizome specific promoter. In a further embodiment the promoter is an endosperm preferred promoter. In a further embodiment the promoter is an endosperm specific promoter.","Polynucleotide is Part of a Genetic Construct","In one embodiment the polynucleotide is transformed as part of a genetic construct. Preferably the genetic construct is an expression construct. Preferably the expression construct includes the polynucleotide operable linked to the promoter. In a further embodiment the polynucleotide is operably linked to a terminator sequence","Plant is Also Transformed with a TAG Synthesising Enzyme","In a further embodiment the plant is also genetically modified to express a triacylglycerol (TAG) synthesising enzyme. In a further embodiment the plant is genetically modified to comprise a nucleic acid sequence encoding a triacylglycerol (TAG) synthesising enzyme. In a further embodiment the plant comprises an expression construct including a nucleic acid sequence encoding a triacylglycerol (TAG) synthesising enzyme.","In a further embodiment the nucleic acid is operably linked to a promoter polynucleotide.","In one embodiment the promoter is capable of driving expression of the polynucleotide in the non-photosynthetic tissues of the plant. In one embodiment the promoter is a constitutive promoter. In one embodiment the promoter is a non-photosynthetic tissue preferred promoter. In a further embodiment the promoter is a root preferred promoter. In a further embodiment the promoter is a root specific promoter. In a further embodiment the promoter is a tuber preferred promoter. In a further embodiment the promoter is a tuber specific promoter. In a further embodiment the promoter is a corm preferred promoter. In a further embodiment the promoter is a corm specific promoter. In a further embodiment the promoter is a rhizome preferred promoter. In a further embodiment the promoter is a rhizome specific promoter. In a further embodiment the promoter is an endosperm preferred promoter. In a further embodiment the promoter is an endosperm specific promoter.","It will be understood by those skilled in the art that the polynucleotide encoding the modified oleosin and the nucleic acid sequence encoding a triacylglycerol (TAG) synthesising enzyme can be placed on the same construct or on separate constructs to be transformed into the host cell. Expression of each can be driven by the same or different promoters, which may be included in the construct to be transformed. It will also be understood by those skilled in the art that alternatively the polynucleotide and nucleic acid can be transformed into the cell without a promoter, but expression of either or both of the polynucleotide and nucleic acid could be driven by a promoter, or promoters, endogenous to the plant transformed.","Methods Including Further Processing Steps","In one embodiment the method comprises the additional step of processing the non-photosynthetic tissue\/organ of the plant into an animal feedstock.","In a further embodiment the method comprises the additional step of processing the non-photosynthetic tissue\/organ of the plant into a biofuel feedstock.","In a further embodiment the method comprises the additional step of extracting oil from the non-photosynthetic tissue\/organ of the plant.","In a further embodiment the method comprises the additional step of processing the non-photosynthetic tissue\/organ into an oil fraction.","In a further embodiment the oil from the non-photosynthetic tissue\/organ is processed into a fuel, oleochemical or nutritional or cosmetic oil, a polyunsaturated fatty acid (PUFA) or a combination thereof.","Non-Photosynthetic Tissue\/Organ of the Plant Produced by the Method of the Invention","In a further embodiment the invention provides a non-photosynthetic tissue\/organ of a plant produced by a method of the invention. In one embodiment the non-photosynthetic tissue\/organ is selected from below ground tissue\/organs of the plant. In a further embodiment the below ground tissue\/organ is selected from root, tuber, bulb, corm and rhizome. In a further embodiment the non-photosynthetic tissue\/organ is selected from root, tuber, bulb, corm, rhizome, and endosperm. In a further embodiment the non-photosynthetic tissue\/organ is root.","In one embodiment the non-photosynthetic tissue\/organ contains at least 100%, more preferably 150%, more preferably 200%, more preferably 250%, more preferably 300%, more preferably 350%, more preferably 400%, more preferably 450%, more preferably 500%, more total lipid than the corresponding non-photosynthetic tissue\/organ of a control plant.","In one embodiment the non-photosynthetic tissue\/organ contains 100% to 900%, more preferably 200% to 800%, more preferably 300% to 700%, more preferably 400% to 600%, more total lipid than the corresponding non-photosynthetic tissue\/organ of a control plant.","Level Oil Production in Non-Photosynthetic Tissues\/Organs.","In one embodiment the non-photosynthetic tissues\/organ contains at least 2\u00d7, more preferably 3\u00d7, more preferably 4\u00d7, more preferably 5\u00d7, more preferably 6\u00d7, more preferably 7\u00d7, more preferably 8\u00d7, more preferably 9\u00d7, more preferably 10\u00d7, more preferably 11\u00d7, more preferably 12\u00d7, more preferably 13\u00d7, more preferably 14\u00d7, more preferably 15\u00d7, more oil than the corresponding non-photosynthetic tissue\/organ of a control plant.","In one embodiment the non-photosynthetic tissue\/organ contains 3\u00d7 to 15\u00d7, more preferably 4\u00d7 to 14\u00d7, more preferably 5\u00d7 to 13\u00d7, more preferably 6\u00d7 to 12\u00d7, more preferably 7\u00d7 to 11\u00d7, more preferably 8\u00d7 to 10\u00d7 more oil than the corresponding non-photosynthetic tissue\/organ of a control plant.","Suitable control plants include non-transformed or wild-type versions of plant of the same variety and or species as the transformed plant used in the method of the invention. Suitable control plants also include plants of the same variety and or species as the transformed plant that are transformed with a control construct. Suitable control plants also include plants that have not been transformed with a polynucleotide encoding a modified oleosin including at least one artificially introduced cysteine. Suitable control plants also include plants that do not express a modified oleosin including at least one artificially introduced cysteine.","Preferably the increased level of oil production is caused by expression of the modified oleosin including at least one artificially introduced cysteine.","Animal Feed Comprising Non-Photosynthetic Tissue\/Organ of the Invention","In a further aspect the invention provides an animal feed comprising the non-photosynthetic tissue\/organ of the invention.","Biofuel Feedstock Comprising Non-Photosynthetic Tissue\/Organ of the Invention","In a further aspect the invention provides a biofuel feedstock comprising the non-photosynthetic tissue\/organ of the invention.","Source of Oleosins and Plants","The modified oleosins may be modified naturally occurring oleosins. The plants from which the un-modified oleosin sequences are derived may be from any plant species that contains oleosins and polynucleotide sequences encoding oleosins.","The plant cells, in which the modified oleosins are expressed, may be from any plant species. The plants, in which the modified oleosins are expressed, may be from any plant species.","In one embodiment the plant cell or plant, is derived from a gymnosperm plant species. In a further embodiment the plant cell or plant, is derived from an angiosperm plant species. In a further embodiment the plant cell or plant, is derived from a from dicotyledonous plant species. In a further embodiment the plant cell or plant, is derived from a monocotyledonous plant species.","Preferred plant species are those that produce tubers (modified stems) such as but not limited to species. Other preferred plant species are those that produce bulbs (below ground storage leaves) such as but not limited to , and species. Other preferred plant species are those that produce corms (swollen underground stems) such as but not limited to and species. Other preferred plant species are those that produce rhizomes (underground storage stem) such as but not limited to and species. Other preferred are those that produce substantial endosperm in their seeds, such as but not limited to maize and sorghum.","Preferred plants incude those from the following genera: , and ","A preferred species is var. (turnip)","Preferred species are those which produce tubers. A preferred species is (potato)","Preferred species include , and . A preferred species is (radish)","Preferred species include: (onion, shallot), (bunching onion), (chives), (Chinese chives), (leek, kurrat, great-headed garlic, pearl onion), (garlic) and (rakkyo). A preferred species is (onion)","Preferred species include: and . A preferred species is (banana, plantains)","A preferred species is (ginger)","A preferred species is (yam)","A preferred species is (taro).","Another preferred genera is Zea. A preferred Zea species is ","Another preferred genera is . A preferred species is ","Other preferred plants are forage plant species from a group comprising but not limited to the following genera: and ","Other preferred plants are leguminous plants. The leguminous plant or part thereof may encompass any plant in the plant family Leguminosae or Fabaceae. For example, the plants may be selected from forage legumes including, alfalfa, clover; leucaena; grain legumes including, beans, lentils, lupins, peas, peanuts, soy bean; bloom legumes including lupin, pharmaceutical or industrial legumes; and fallow or green manure legume species.","A particularly preferred genus is . Preferred species include ; and . A particularly preferred species is ","Another preferred genus is . Preferred species include and . A particularly preferred species is , commonly known as alfalfa.","Another preferred genus is . Preferred species include and (also known as ). A particularly preferred Glycine species is , commonly known as soy bean. A particularly preferred Glycine species is , commonly known as perennial soybean.","Another preferred genus is . A particularly preferred species is commonly known as cowpea.","Another preferred genus is . Preferred species include . A particularly preferred species is commonly known as velvetbean.","Another preferred genus is . A particularly preferred species is commonly known as perennial peanut.","Another preferred genus is . A preferred species is commonly known as pea.","Another preferred genus is . Preferred species include and . A preferred species is commonly known as Birdsfoot Trefoil. Another preferred species is commonly known as Narrow-leaf Birdsfoot Trefoil. Another preferred species is commonly known as Big trefoil. Another preferred species is commonly known as Slender trefoil.","Another preferred genus is . A preferred species is , commonly known as forage kale and cabbage.","Other preferred species are oil seed crops including but not limited to the following genera: and ","A preferred oil seed genera is . A preferred oil seed species is ","A preferred oil seed genera is . A preferred oil seed species is ","A preferred oil seed genera is . A preferred oil seed species is ","A preferred oil seed genera is . A preferred oil seed species is ","A preferred oil seed genera is . A preferred oil seed species is ","A preferred oil seed genera is . A preferred oil seed species is ","A preferred silage genera is Zea. A preferred silage species is ","A preferred grain producing genera is . A preferred grain producing species is ","A preferred grazing genera is . A preferred grazing species is ","A preferred grazing genera is . A preferred grazing species is ","A preferred grazing genera is . A preferred grazing species is ","A preferred grazing genera is . A preferred grazing species is ","Preferred plants also include forage, or animal feedstock plants. Such plants include but are not limited to the following genera: ","A preferred biofuel genera is . A preferred biofuel species is ","A preferred biofuel genera is . A preferred biofuel species is ","A preferred biofuel genera is . A preferred biofuel species is ","In this specification where reference has been made to patent specifications, other external documents, or other sources of information, this is generally for the purpose of providing a context for discussing the features of the invention. Unless specifically stated otherwise, reference to such external documents is not to be construed as an admission that such documents, or such sources of information, in any jurisdiction, are prior art, or form part of the common general knowledge in the art.","The term \u201ccomprising\u201d as used in this specification means \u201cconsisting at least in part of\u201d. When interpreting each statement in this specification that includes the term \u201ccomprising\u201d, features other than that or those prefaced by the term may also be present. Related terms such as \u201ccomprise\u201d and \u201ccomprises\u201d are to be interpreted in the same manner.","On a weight for weight basis lipids have approximately double the energy content of either proteins or carbohydrates. The bulk of the world's lipids are produced by plants and the densest form of lipid is as a triacylglycerol (TAG). Dicotyledonous plants can accumulate up to approximately 60% of their seed weight as TAG which is subsequently used as an energy source for germination. As such there have been a number of efforts targeted at using seeds rich in oils to sustainably produce sufficient lipids for both animal and biofuel feed stock.","Given that there is only a limited quantity of TAG able to be produced by seeds alternative approaches are being made to produce additional lipid (preferentially TAGs) in vegetative tissues. The majority of these approaches have pursued the up regulation or over expression of one or several enzymes in the Kennedy pathway in the leaves of plants in order to synthesise TAG. Typically however, the majority of additional lipids produced by this approach are re-mobilised within the plant by a combination of lipases and \u03b2-oxidation resulting in a limited increase in lipid content (usually 2-4% of the DM).","The TAG produced in developing seeds is typically contained within discreet structures called oil bodies (OBs) which are highly stable and remain as discrete tightly packed organelles without coalescing even when the cells desiccate or undergo freezing conditions (Siloto et al., 2006; Shimada et al., 2008). OBs consist of a TAG core surrounded by a phospholipid monolayer embedded with proteinaceous emulsifiers. The latter make up 0.5-3.5% of the OB; of this, 80-90% is oleosin with the remainder predominantly consisting of the calcium binding (caloleosin) and sterol binding (steroleosin) proteins (Lin and Tzen, 2004). The emulsification properties of oleosins derives from their three functional domains which consist of an amphipathic N-terminal arm, a highly conserved central hydrophobic core (\u02dc72 residues) and a C-terminal amphipathic arm. Similarly, both caloleosin and steroleosin possess hydrophilic N and C-terminal arms and their own conserved hydrophobic core.","It was previously speculated that the constitutive expression of oleosin or polyoleosin (tandem head-to-tale fusions of oleosins) with TAG synthesising enzymes in the leaves would result in the formation of stable oil bodies leading to the accumulation of TAG. We have subsequently found however, that oleosin and polyoleosins are ineffective and promoting the accumulation of TAG when co-expressed with DGAT1 in plant leaves (Roberts et al., unpublished data).","The current invention provides modified oleosins which contain one or more artificially introduced cysteine residues. The encapsulation of the neutral lipids by oleosins containing engineered cysteines provides an alternative mechanism to accumulate appreciable quantities of TAG in leaves without the requirement to wait until senescence and without producing extreme phenotypes. In addition the modified oleosin has a number of other applications involving modifying OB stability, emulsion properties as well as the generation and purification of recombinant proteins.","It has been shown (WO 2011\/053169) that expression of modified oleosins with artificially introduced cysteines can produce increased level of oil in the leaves of plants. However, the present applicants have now surprisingly shown that it is possible to accumulate significantly higher levels in the non-photosynthetic tissues of plants than in the other tissues of the plants.","Oil Bodies","OBs generally range from 0.5-2.5 \u03bcm in diameter and consist of a TAG core surrounded by a phospholipid monolayer embedded with proteinaceous emulsifiers\u2014predominantly oleosins (Tzen et al, 1993; Tzen, et al 1997). OBs consist of only 0.5-3.5% protein; of this 80-90% is oleosin with the remainder predominantly consisting of the calcium binding (caleosin) and sterol binding (steroleosin) proteins (Lin and Tzen, 2004). The ratio of oleosin to TAG within the plant cell influences the size and number of oil bodies within the cell (Sarmiento et al., 1997; Siloto et al., 2006).","While OBs are naturally produced predominantly in the seeds and pollen of many plants they are also found in some other organs (e.g., specific tubers).","Oleosins are comparatively small (15 24 kDa) proteins that are embedded in the surface of OBs.","Oil Body Stability","The suitability of oil bodies, and artificial oil bodies, for the applications discussed above, among others, is limited at least in part, by their stability. One approach to address oil body stability was to generate oil bodies comprising so-called polyoleosin. Polyoleosin is the head to tail fusion of two or more oleosin units (Roberts et al., 2008). Altering the number of oleosin units enables the properties (thermal stability and degradation rate) of the oil bodies to be tailored. Expression of polyoleosin in planta leads to incorporation of the polyoleosin units to the oil bodies as per single oleosin units (Scott et al., 2007). Multiple oleosin units in tandem head-to-tail arrangements were used to create polyoleosin. Separate constructs (containing from one to six oleosin repeats) were specifically designed for expression in planta and in . The majority of recombinant polyoleosin accumulated in the oil bodies of transgenic plants and in the inclusion bodies of . Purified prokaryotically produced polyoleosin was used to generate artificial oil bodies. Oil body and artificial oil body thermal stability and structural integrity in proteinase-K were raised by polyoleosin.","However, there are several limiting factors determining the degree of protection\/stability that polyoleosin can provide; these relate to the number of tandem repeats that can be joined before the process of translation and oil body targeting becomes limiting (Scott et al., 2007); while another limitation comes from the nature of the oleosin fusion which is achieved by generating a transcript with a head to tail fusion arrangement. This is essentially a linear protein of multimeric oleosin repeats that has a number of covalent-links and position of covalent-links per individual oleosin repeat (i.e., a maximum of one at each end). In addition this arrangement only affords protection against N-terminal degrading proteins but it does not provide any additional protection against other proteolytic enzymes that recognise specific internal peptide sequences. Furthermore, the linking between oleosin units in a polyoleosin molecule formed by tandem head to tail repeats is not readily altered in situ. While specific protease specific sites could be engineered into the joining regions in order to break apart fused polyoleosin molecules embedded into an oil body or artificial oil body they could not be re-fused easily.","Oleosins embedded in oil bodies have previously covalently cross-linked by the addition of cross-linking agents such as glutaraldehyde or genepin (Peng et al., 2004 & 2006), however, this random cross-linking requires the addition of cross-linking agents to oil body preparations, and is not easy to reverse.","Artificial Oil Bodies","Prokaryotically expressed recombinant oleosins can be used to generate artificial oil bodies (AOBs) who's properties are very similar to plant derived OBs (Peng et al. 2004; Roux et al. 2004; Chiang et al. 2005; Chiang et al. 2007).","Applications of Oil Bodies and Artificial Oil Bodies","The unique properties of oil bodies, and their constituent oleosins, form the basis of a number of biotechnical applications including: purifying recombinant proteins; formation of multimeric protein complexes; emulsification; delivery of bioactives; generation of multivalent bioactives and even as a potential flavour enhancer (for reviews see Capuano et al., 2007 and Roberts et al., 2008).","Emulsions","Emulsions are produced when one or more liquids that are immiscible in another liquid, usually due to different polarities and thus different hydrophobicities, are uniformly suspended within that liquid. Examples include oil droplets uniformly dispersed in water, or water droplets uniformly dispersed in oil. Generation of a relatively stable emulsion requires the use of an emulsifier, which lowers the interfacial tension between the liquids. The stability of an emulsion is generally measured in terms of the duration that the uniform dispersion persists under specified conditions. Emulsifiers are commonly used in the food and cosmetic industry; so need to have high emulsion stability and be safe for consumption and topical application.","Intact oil bodies containing oleosin naturally form a surfactant-free, oil-in water emulsion. It has been found that intact oil bodies or oil bodies in which the majority of TAG has been removed have a broad range of emulsification applications in food, topical personal care (skin creams) and pharmaceutical formulations (Harada et al., 2002; Deckers et al., 2003; Hou et al., 2003).","Biohydrogenation","It has been demonstrated that the lipid profile of ruminant animal feed in turn influences the lipid profile of meat and dairy products (Demeyer and Doreau, 1999). Different plants have different lipid profiles; by selectively feeding animals only plants with the desired lipid profile it is possible to positively influence the lipid profile of downstream meat and dairy products. In ruminants the final lipid make up of the meat and milk is not only influenced by the dietary lipids but is also heavily influenced by biohydrogenation (Jenkins and McGuire 2006; Firkins et al., 2006; Lock and Bauman, 2004). Biohydrogenation is the hydrogenation of non-reduced compounds (such as unsaturated fats) by the biota present in the rumen. Biohydrogenation can be prevented\/delayed by encapsulating the lipids in a protein or proteins that provide resistance to microbial degradation (Jenkins and Bridges 2007). The prevention of biohydrogenation by encapsulating triacylglycerides in polyoleosin or oleosins in planta was reported by Scott et al., (2007), Cookson et al., (2009) and Roberts et al., (2008).","Oleosins","Oleosins are comparatively small (15 to 24 kDa) proteins which allow the OBs to become tightly packed discrete organelles without coalescing as the cells desiccate or undergo freezing conditions (Leprince et al., 1998; Siloto et al., 2006; Slack et al., 1980; Shimada et al. 2008).","Oleosins have three functional domains consisting of an amphipathic N-terminal arm, a highly conserved central hydrophobic core (\u02dc72 residues) and a C-terminal amphipathic arm. The accepted topological model is one in which the N- and C-terminal amphipathic arms are located on the outside of the OBs and the central hydrophobic core is located inside the OB (Huang, 1992; Loer and Herman, 1993; Murphy 1993). The negatively charged residues of the N- and C-terminal amphipathic arms are exposed to the aqueous exterior whereas the positively charged residues are exposed to the OB interior and face the negatively charged lipids. Thus, the amphipathic arms with their outward facing negative charge are responsible for maintaining the OBs as individual entities via steric hinderance and electrostatic repulsion both in vivo and in isolated preparation (Tzen et al, 1992). The N-terminal amphipathic arm is highly variable and as such no specific secondary structure can describe all examples. In comparison the C-terminal arm contains a \u03b1-helical domain of 30-40 residues (Tzen et al, 2003). The central core is highly conserved and thought to be the longest hydrophobic region known to occur in nature; at the center is a conserved 12 residue proline knot motif which includes three spaced proline residues (for reviews see Frandsen et al, 2001; Tzen et al, 2003). The secondary, tertiary and quaternary structure of the central domain is still unclear. Modelling, Fourier Transformation-Infra Red (FT-IR) and Circular Dichromism (CD) evidence exists for a number of different arrangements (for review see Roberts et al., 2008).","The properties of the major oleosins is relatively conserved between plants and is characterised by the following:\n\n","Preferred oleosins for use in the invention are those which contain a central domain of approximately 70 non-polar amino acid residues (including a proline knot) uninterrupted by any charged residues, flanked by two hydrophilic arms.","The term \u201coleosin\u201d as used herein also includes steroleosin and caloleosin","Steroleosins","Steroleosins comprises an N-terminal anchoring segment comprising two amphipathic \u03b1-helices 912 residues in each helix) connected by a hydrophobic anchoring region of 14 residues. The soluble dehydrogenase domain contains a NADP+-binding subdomain and a sterol-binding subdomain. The apparent distinction between steroleosins-A and -B occurs in their diverse sterol-binding subdomains (Lin and Tzen, 2004). Steroleosins have a proline knob in their hydrophobic domain and contains a sterol-binding dehydrogenase in one of their hydrophilic arms.","Caloleosins","Caloleosins (Frandsen et al., 2001) have a slightly different proline knot than do the basic oleosins, and contain a calcium-binding motif and several potential phosphorylation sites in the hydrophilic arms. Similar to oleosin, caloleosin is proposed to have three structural domains, where the N- and C-terminal arms are hydrophilic while the central domain is hydrophobic and acts as the oil body anchor. The N-terminal hydrophilic domain consists of a helix-turn-helix calcium binding EF-hand motif of 28 residues including an invariable glycine residue as a structural turning point and five conserved oxygen-containing residues as calcium-binding ligands (Chen et al., 1999; Frandsen et al., 2001). The C-terminal hydrophilic domain contains several phosphorylation sites and near the C-terminus is an invariable cysteine that is not involved in any intra- or inter-disulfide linkages (Peng, 2004). The hydrophilic N- and C-termini of caloleosin are approximately 3 times larger than those of oleosin (Lin and Tzen, 2004). The hydrophobic domain is thought to consist of an amphipathic \u03b1-helix and an anchoring region (which includes a proline knot).","Examples of oleosin (oleosins, steroleosin and caloleosin) sequences suitable to be modified for use in the invention, by the addition of at least one artificially introduced cysteine, are shown in Table 1 below. The sequences (both polynucleotide and polypeptide are provided in the Sequence Listing)",{"@attributes":{"id":"p-0328","num":"0368"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"42pt","align":"center"}}],"thead":{"row":[{"entry":"TABLE 1"},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}},{"entry":[{},{},"cDNA",{},"Protein",{}]},{"entry":["Oleosin","Species","accession no.","SEQ ID NO:","accession no.","SEQ ID NO:"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Oleosin",{"i":"S. indicum"},"AF302907","34","AAG23840","35"]},{"entry":["Oleosin",{"i":"S. indicum"},"U97700","36","AAB58402","37"]},{"entry":["Oleosin",{"i":"A. thaliana"},"X62353","38","CAA44225","39"]},{"entry":["Oleosin",{"i":"A. thaliana"},"BT023738","40","AAZ23930","41"]},{"entry":["Oleosin",{"i":"H. annuus"},"X62352.1","42","CAA44224.1","43"]},{"entry":["Oleosin",{"i":"B. napus"},"X82020.1","44","CAA57545.1","45"]},{"entry":["Oleosin",{"i":"Z. mays"},"NM_001153560.1","46","NP_001147032.1","47"]},{"entry":["Oleosin",{"i":"O. sativa"},"AAL40177.1","48","AAL40177.1","49"]},{"entry":["Oleosin",{"i":"B. oleracea"},"AF117126.1","50","AAD24547.1","51"]},{"entry":["Oleosin",{"i":"C. arabica"},"AY928084.1","52","AAY14574.1","53"]},{"entry":["Steroleosin",{"i":"S. indicum"},"AAL13315","54","AAL13315","55"]},{"entry":["Steroleosin",{"i":"A. napus"},"EU678274","56","ACG69522","57"]},{"entry":["Steroleosin",{"i":"Z. mays"},"NM_001159142.1","58","NP_001152614.1","59"]},{"entry":["Steroleosin",{"i":"B. napus"},"EF143915.1","60","ABM30178.1","61"]},{"entry":["Caloleosin",{"i":"S. indicum"},"AF109921","62","AAF13743","63"]},{"entry":["Caloleosin",{"i":"G. max"},"AF004809","64","AAB71227","65"]},{"entry":["Caloleosin",{"i":"Z. mays"},"NM_001158434.1","66","NP_001151906","67"]},{"entry":["Caloleosin",{"i":"B. napus"},"AY966447.1","68","AAY40837","69"]},{"entry":["Caloleosin",{"i":"C. revoluta"},"FJ455154.1","70","ACJ70083","71"]},{"entry":["Caloleosin",{"i":"C. sativus"},"EU232173.1","72","ABY56103.1","73"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]}}}}},"Oleosin, steroleosin and caloleosins are well known to those skilled in the art. Further sequences from many different species can be readily identified by methods well-known to those skilled in the art. For example, further sequences can be easily identified by an NCBI Entrez Cross-Database Search (available at ncbi<dot>nlm<dot>nih<dot>gov\/sites\/gquery) using any one of the terms oleosin, steroleosin and caloleosin.","Plant Lipids Biosynthesis","All plant cells produce fatty acids from actetyl-CoA by a common pathway localized in plastids. Although a portion of the newly synthesized acyl chains is then used for lipid biosynthesis within the plastid (the prokaryotic pathway), a major portion is exported into the cytosol for glycerolipid assembly at the endoplasmic reticulum (ER) or other sites (the eukaryotic pathway). In addition, some of the extraplastidial glycerolipids return to the plastid, which results in considerable intermixing between the plastid and ER lipid pools (Ohlrogge and Jaworski 1997).","The simplest description of the plastidial pathway of fatty acid biosynthesis consists of two enzyme systems: acetyl-CoA carboxylase (ACCase) and fatty acid synthase (FAS). ACCase catalyzes the formation of malonyl-CoA from acetyl-CoA, and FAS transfers the malonyl moiety to acyl carrier protein (ACP) and catalyzes the extension of the growing acyl chain with malonyl-ACP.","The initial fatty acid synthesis reaction is catalyzed by 3-ketoacyl-ACP III (KAS III) which results in the condensation of acetyl-CoA and malonyl-ACP. Subsequent condensations are catalyzed by KAS I and KAS II. Before a subsequent cycle of fatty acid synthesis begins, the 3-ketoacyl-ACP intermediate is reduced to the saturated acyl-ACP in the remaining FAS reactions, catalyzed sequentially by the 3-ketoacyl-ACP reductase, 3 hydroxyacyl-ACP dehydrase, and the enoyl-ACP reductase.","The final products of FAS are usually 16:0 and 18:0-ACP, and the final fatty acid composition of a plant cell is in large part determined by activities of several enzymes that use these acyl-ACPs at the termination phase of fatty acid synthesis. Stearoyl-ACP desatruase modifies the final product of FAS by insertion of a cis double bond at the 9 position of the C18:0-ACP. Reactions of fatty acid synthesis are terminated by hydrolysis or transfer of the acyl chain from the ACP. Hydrolysis is catalyzed by acyl-ACP thioesterases, of which there are two main types: one thioesterase relatively specific for 18:1-ACP and a second more specific for saturated acyl-ACPs. Fatty acids that have been released from ACPs by thioesterases leave the plastid and enter into the eukaryotic lipid pathway, where they are primarily esterified to glycerolipids on the ER. Acyl transferases in the plastid, in contrast to thioesterases, terminate fatty acid synthesis by transesterifying acyl moieties from ACP to glycerol, and they are an essential part of the prokaryotic lipid pathway leading to plastid glycerolipid assembly.","Triacylglycerol Biosynthesis","The only committed step in TAG biosynthesis is the last one, i.e. the addition of a third fatty acid to an existing diacylglycerol, thus generating TAG. In plants this step is predominantly (but not exclusively) performed by one of five (predominantly ER localised) TAG synthesising enzymes including: acyl CoA: diacylglycerol acyltransferase (DGAT1); an unrelated acyl CoA: diacylglycerol acyl transferase (DGAT2); a soluble DGAT (DGAT3) which has less than 10% identity with DGAT1 or DGAT2 (Saha et al., 2006); phosphatidylcholine-sterol O-acyltransferase (PDAT); and a wax synthase (WSD1, Li et al., 2008). The DGAT1 and DGAT2 proteins are eoncoded by two distinct gene families, with DGAT1 containing approximately 500 amino acids and 10 predicted transmembrane domains and DGAT2 has only 320 amino acids and two transmembrane domains (Shockey et al., 2006).","The term \u201ctriacylglycerol synthesising enzyme\u201d or \u201cTAG synthesising enzyme\u201d as used herein means an enzyme capable of catalysing the addition of a third fatty acid to an existing diacylglycerol, thus generating TAG. Preferred TAG synthesising enzymes include but are not limited to: acyl CoA: diacylglycerol acyltransferase) (DGAT1); diacylglycerol acyl transferase2 (DGAT2); phosphatidylcholine-sterol O-acyltransferase (PDAT) and cytosolic soluble form of DGAT (soluble DGAT or DGAT3).","Given that endogenous DGAT1 and DGAT2 appear to play roles in mature and senescing leaves (Kaup et al. 2002; Shockey et al. 2006), it is likely that plants possess a number of feedback mechanisms to control their activity. Indeed, Zou et al. (2008) recently identified a consensus sequence (X-Leu-X-Lys-X-X-Ser-X-X-X-Val (SEQ ID NO:108) within Tropaeolum majus (garden nasturtium) DGAT1 (TmDGAT1) sequences as a targeting motif typical of members of the SNF1-related protein kinase-1 (SnRK1) with Ser being the residue for phosphorylation. The SnRK1 proteins are a class of Ser\/Thr protein kinases that have been increasingly implicated in the global regulation of carbon metabolism in plants, e.g. the inactivation of sucrose phosphate synthase by phosphorylation (Halford & Hardie 1998). Zou et al. (2008) went on to demonstrate that the obliteration of a potential SnRK1 phosphorylation site in DGAT1 by single point mutation (Ser197Ala of TmDGAT1) led to the accumulation of significantly higher levels of TAG in the seed. This mutation increased activity by 38-80%, which led to a 20-50% increase in oil content on a per seed basis in Arabidopsis.","Phospholipid:DGA acyltransferase (PDAT) forms TAG from a molecule of phospholipid and a molecule of diacyglycerol. PDAT is quite active when expressed in yeast but does not appreciably increase TAG yields when expressed in plant seeds. PDAT and a proposed DAG:DAG transacylase are neutral lipid synthesizing enzymes that produce TAG, but are not considered part of the Kennedy Pathway.","A combination of wax ester synthase and DGAT enzyme (WS\/DGAT) has been found in all neutral lipid producing prokaryotes studied so far. WS\/DAGAT has extraordinary broad activity on a variety of unusual fatty acids, alcohols and even thiols. This enzyme has a putative membrane-spanning region but shows no sequence homology to the DGAT1 and DGAT2 families from eukaryotes or the WE synthase from jojoba (Jojoba is the only eukaryote that has been found to accumulate wax ester).","It should be noted that Lecithin-Cholesterol AcylTransferase (LCAT) and Acyl-coenzyme:Cholesterol AcylTransferase (ACAT) are enzymes that produce sterol esters (a form of neutral lipid) not TAGs.","In applications requiring the increase of neutral lipids evidence suggests that the higher activity and broader specificity of DGAT1 relative to DGAT2 is preferential. Where a specific fatty acid is preferred, such as a long-chain PUFA, DGAT1 is still applicable, provided it accepts the fatty acid of choice. Plants generally incorporate long chain PUFAs in the sn-2 position. It is not known whether this is due to high activity of LPAT or low activity of DGAT1 on this substrate. For the improved specificity for PUFAs, a DGAT2 that prefers these fatty acids may be preferable, or the properties of DGAT1 could be altered using directed evolution or an equivalent procedure.","Examples of these TAG synthesising enzymes, suitable for use in the methods and compositions of the invention, from members of several plant species are provided in Table 2 below. The sequences (both polynucleotide and polypeptide are provided in the Sequence Listing)",{"@attributes":{"id":"p-0342","num":"0382"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"42pt","align":"center"}}],"thead":{"row":[{"entry":"TABLE 2"},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}},{"entry":["TAG",{},{},{},{},{}]},{"entry":["synthesising",{},"cDNA",{},"Protein",{}]},{"entry":["enzyme","Species","accession no.","SEQ ID NO:","accession no.","SEQ ID NO:"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["DGAT1",{"i":"A. thaliana"},"NM_127503","74","NP_179535","75"]},{"entry":["DGAT1",{"i":"T. majus"},"AY084052","76","AAM03340","77"]},{"entry":["DGAT1",{"i":"Z. mays"},"EU039830","78","ABV91586","79"]},{"entry":["DGAT2",{"i":"A. thaliana"},"NM_115011","80","NP_566952","81"]},{"entry":["DGAT2",{"i":"B. napus"},"FJ858270","82","AC090187","83"]},{"entry":["DGAT3",{"i":"A. hypogaea"},"AY875644","84","AAX62735","85"]},{"entry":["(soluble DGAT)",{},{},{},{},{}]},{"entry":["PDAT",{"i":"A. thaliana"},"NM_121367","86","NP_196868","87"]},{"entry":["PDAT",{"i":"R. communis"},"XM_002521304","88","XP_002521350","89"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]}}}}},"The inventions also contemplates use of modified TAG synthesizing enzymes, that are modified (for example in their sequence by substitutions, insertions or additions an the like) to alter their specificity and or activity.","TAG Accumulation in Leaves","A recent field survey of 302 angiosperm species in the north-central USA found that 24% have conspicuous cytosolic oil droplets in leaves, with usually one large oil droplet per mesophyll cell (Lersten et al., 2006 [from Slocombe et al 2009]). The role of cytosolic leaf TAG is thought to be involved in carbon storage and\/or membrane lipid re-modelling (for review see Slocombe et al., 2009). Indeed, in senescing leaves, plastidial fatty acids are partitioned into TAG prior for further mobilization, and DGAT1 is thought to be instrumental in this process (Kaup et al., 2002).","There have been several attempts to engineer plants to accumulate elevated levels of TAG in their leaves. The success of these has been somewhat limited by the relatively low level of TAG that accumulated and in some cases the majority of TAG accumulated in senescing leaves only, thus limiting the flexibility of harvesting and proportion of crop accumulating TAG at any one time (Bouvier-Nave et al, 2001; Xu et al., 2005; Winichayakul et al., 2008; Andrianov et al., 2010; Slocombe et al., 2009 and references therein).","To date the attempts to accumulate TAG in leaves have predominantly focussed on three particular gene candidates including over expression of DGAT (TAG biosynthesis), mutation of TGD1 or CTS (resulting in the prevention of lipid remobilisation), and over expression of LEC1, LEC2 and WRI1 (transcriptional factors involved in storage oil and protein accumulation in developing seeds). Over expression of TAG and other neutral lipid synthesizing enzymes relies on the presence of sufficient substrate, in the expanding and or mature leaf this is assumed to be provided by the plastid (chloroplast in the case of the leaf) which synthesises lipids for membranes. In photosynthetic leaves of it has been estimated that the turnover of membrane lipids is 4% of total fatty acids per day (Bao et al, 2000). In senescing leaves, the existing plastidal membranes provide the bulk of fatty acids for partitioning into TAG prior to further mobilization.","Over-expression of the DGAT1 gene in tobacco leaves results in enhanced TAG accumulation (Bouvier-Nave et al., 2001), this was later repeated and quantified by Andrianov et al., (2010). They calculated the TAG level increased 20 fold and lead to a doubling of lipid content from \u02dc3% to \u02dc6% of dry matter in mature leaves. A further increase to 6.8% was achieved by the over expression of LEC2 (a master regulator of seed maturation and seed oil storage) in mature leaves using the inducible Alc promoter (Andrianov et al., 2010). No estimation of the extractable TAG was given, nor was there any calculation on the accumulation of TAG in expanding leaves.","Mutations in a permease-like protein TRIGALACTOSYLDIACYLGLYCEROL (TGD1), in caused the accumulation of TAGs, oligogalactolipids and phosphatidate; this was accompanied by a high incidence of embryo abortion and comparatively poor overall plant growth (Xu et al., 2005).","Winichayakul et al., (2008) over expressed DGAT1 in the leaves of ryegrass () and found this lead to a 50% elevation of total extractable leaf lipid (from \u02dc4% to 6% of dry matter). Furthermore, the elevated lipid level was present in new leaves generated by repeated harvests spaced 2-3 weeks apart, indicating that the new emerging leaves were also capable of accumulating additional lipid. However, the elevated lipid level in these leaves typically began to decline to wild type levels when the leaves were more than 2 weeks old indicating that the lipids were being re-mobilised via catabolism (release from the glycerol backbone by lipase followed by \u03b2-oxidation).","Slocombe et al., (2009) demonstrated that mutations in the CTS peroxisomal ABC transporter (cts-2) led to accumulation of up to 1.4% TAG in leaves, particularly during the onset of senescence. They also ectopically expressed LEC2 during senescence in the cts-2 background; while this did not elevate the overall accumulation of TAG over the cts-2 mutant it did increase the accumulation of seed oil type species of TAG in senescing tissue. While cts-2 blocks fatty acid breakdown it also led to a severe phenotype. Slocombe et al., (2009) concluded that recycled membrane fatty acids may be able to be re-directed to TAG by expressing the seed-programme in senescing tissue or by a block in fatty acid breakdown.","Scott et al., (2007) claimed that the co-expression of a triacylglyceride synthesising enzyme and polyoleosin (two or more oleosin units fused in a tandem head-to-tail arrangement) would enable the storage of lipid in a plant cell. Similarly, Cookson et al., (2009) claimed that producing a single oleosin and a TAG synthesising enzyme within vegetative portions of a plant would lead to increased number of oil bodies and TAG in the vegetative tissue. Using either of these techniques leads to a maximum increase in lipid content (not necessarily in the form of TAG) of up to approximately 50%. Furthermore this level begins to decline as the leaves mature; typically in leaves greater than 2 weeks old (unpublished data).","Hence, the degree to which TAG can be accumulated in vegetative tissues appears to be limited to some extent by the fact that the endogenous fixed-carbon recovery machinery catabolises the TAG.","Leaf Senescence\u2014Recycling of Lipids Via TAG Intermediates","Leaf senescence is a highly controlled sequence of events leading ultimately to the death of cells, tissues and finally the whole organ. This entails regulated recruitment of nutrients together with their translocation from the senescing tissue to other tissues that are still growing and developing. The chloroplast is the first organelle of mesophyll cells to show symptoms of senescence and although breakdown of thylakoid membranes is initiated early in the leaf senescence cascade, the chloroplast envelope remains relatively intact until the very late stages of senescence. DGAT1 is up-regulated during senescence of leaves and this is temporally correlated with increased levels of TAG-containing fatty acids commonly found in chloroplast galactolipids. Recruitment of membrane carbon from senescing leaves, particularly senescing chloroplasts, to growing parts of the plant is a key feature of leaf senescence, and it involves de-esterification of thylakoid lipids and conversion of the resultant free fatty acids to phloem-mobile sucrose. De-esterification of thylakoid lipids appears to be mediated by one or more senescence induced galactolipases. The formation of TAG appears to be an intermediate step in the mobilisation of membrane lipid carbon to phloem mobile sucrose during senescence (Kaup et al., 2002).","Modified Oleosins Engineered to Include Artificially Introduced Cysteines","The modified oleosins of the invention, or for use in the methods of the invention, are modified to contain at least one artificially introduced cysteine residue. Preferably the engineered oleosins contain at least two cysteines.","The encapsulation of the neutral lipids by oleosins containing engineered cysteines provides an alternative mechanism to accumulate appreciable quantities of TAG in leaves without the requirement to wait until senescence and without producing extreme phenotypes.","Various methods well-known to those skilled in the art may be used in production of the modified oleosins with artificially introduced cysteines.","Such methods include site directed mutagenesis (U.S. Pat. No. 6,448,048) in which the polynucleotide encoding an oleosin is modified to introduce a cysteine into the encoded oleosin protein.","Alternatively the polynucleotide encoding the modified oleosins, may be synthesed in its entirety.","Further methodology for producing modified oleosins of the invention and for use in the methods of the invention, is provided in the Examples section.","The introduced cysteine may be an additional amino acid (i.e. an insertion) or may replace an existing amino acid (i.e. a replacement). Preferably the introduced cysteine replaces an existing amino acid. In a preferred embodiment the replaced amino acid is a charged residue. Preferably the charged residue is predicted to be in the hydrophilic domains and therefore likely to be located on the surface of the oil body.","The hydrophilic, and hydrophobic regions\/arms of the oleosin can be easily identified by those skilled in the art using standard methodology (for example: Kyte and Doolitle (1982).","The modified oleosins of the invention are preferably range in molecular weight from 5 to 50 kDa, more preferably, 10 to 40 kDa, more preferably 15 to 25 kDa.","The modified oleosins of the invention are preferably in the size range 100 to 300 amino acids, more preferably 110 to 260 amino acids, more preferably 120 to 250 amino acids, more preferably 130 to 240 amino acids, more preferably 140 to 230 amino acids.","Preferably the modified oleosins comprise an N-terminal hydrophilic region, two centre hydrophobic regions (joined by a proline knot or knob) and a C-terminal hydrophilic region.","Preferably the modified oleosins can be divided almost equally their length into four parts which correspond to the N-terminal hydrophilic region (or arm), the two centre hydrophobic regions (joined by a proline knot or knob) and a C-terminal hydrophilic region (or arm).","Preferably the topology of modified oleosin is attributed to its physical properties which include a folded hydrophobic core flanked by hydrophilic domains.","Preferably the modified oleosins can be formed into oil bodies when combined with triacylglycerol (TAG) and phospholipid.","Preferably topology confers an amphipathic nature to modified oleosin resulting in the hydrophobic domain being embedded in the phospholipid monolayer of the oil body while the flanking hydrophilic domains are exposed to the aqueous environment outside the oil body, such as in the cytoplasm.","In one embodiment the modified oleosin of the invention or used in the method of the invention, comprises a sequence with at least 70% identity the hydrophobic domain of any of the oleosin protein sequences referred to in Table 1 above.","In one embodiment the modified oleosin of the invention or used in the method of the invention, comprises a sequence with at least 70% identity to any of the protein sequences referred to in Table 1 above.","In further embodiment the modified oleosin is essentially the same as any of the oleosins referred to in Table 1 above, apart from the additional artificially introduced cysteine or cysteines.","In a further embodiment the modified oleosin of the invention or used in the method of the invention, comprises a sequence with at least 70% identity to the oleosin sequence of SEQ ID NO: 16.","In further embodiment the modified oleosin has the same amino acid sequence as that of SEQ ID NO: 16, apart from the additional artificially introduced cysteine or cysteines.","In further embodiment the modified oleosin is has the amino acid sequence of any one of SEQ ID NO: 16 to 20.","Fusion Proteins with Modified Oleosins","The invention also provides a fusion proteins including a modified oleosin of the invention fused to a protein of interest.","Preferably the protein of interest is at the N- or C-terminal end of the fusion protein.","Methods for recombinantly expressing fusion proteins are well known to those skilled in the art (Papapostolou and Howorka, 2009). Production of the fusion protein of the invention may typically involve fusing the coding sequence of the protein of interest to the coding sequence of the modified oleosin.","Such fusion proteins may be included in, or expressed in, the oil bodies of the invention and used to purify and deliver the protein of interest for a variety of applications, as discussed in Roberts et al, (2008).","However in the invention makes it possible to take advantage of the option to vary the stability\/integrity of the oil body provided by presence of the modified oleosins in the oil body, hence allowing for more stringent purification and delivery procedures.","Fusion Proteins with Un-Modified Oleosins","The invention also involves use of fusion protein including un-modified oleosin fused to a protein of interest. Production of the fusion protein of the invention may typically involve fusing the coding sequence of the protein of interest to the coding sequence of the un-modified oleosin.","Preferably the protein of interest is at the N- or C-terminal end of the fusion protein.","Such fusion proteins may be included or expressed in the oil bodies of the invention and used to purify and deliver the protein of interest for a variety of applications, as discussed in Roberts et al., (2008).","The present invention however, takes advantage of the option to vary the stability\/integrity of the oil body provided by presence of the modified oleosins in the oil body of the invention, hence allowing for more stringent purification and delivery procedures.","Overview of Photosynthesis","The overall process whereby algae and plants use light to synthesize organic compounds is called photosynthesis (). Photosynthesis encompasses a complex series of reactions that involve light absorption, production of stored energy and reducing power (the Light Reactions). It also includes a multistep enzymatic pathway that uses these to convert COand water into carbohydrates (the Calvin cycle, ). In plants the biophysical and biochemical reactions of photosynthesis occur within a single chloroplast (C3 photosynthesis) but can also be separated into chloroplasts of differing cell types (C4 photosynthesis).","Carbon fixation is a redox reaction, photosynthesis provides both the energy to drive this process as well as the electrons required to convert COto carbohydrate (). These two processes take place through a different sequence of chemical reactions and in different cellular compartments. In the first stage, light is used to generate the energy storage molecules ATP and NADPH. The thylakoid membranes contain the multiprotein photosynthetic complexes Photosystems I and II (PSI and PSII) which include the reaction centres responsible for converting light energy into chemical bond energy (via an electron transfer chain). The photosynthetic electron transfer chain moves electrons from water into the thylakoid lumen to soluble redox-active compounds in the stroma. A byproduct of this process (Hill Reaction) is oxygen.","The second part of the photosynthetic cycle is the fixation of COinto sugars (Calvin Cycle, ); this occurs in the stroma and uses the ATP and NADPH generated from the light reaction.","Rubisco","Ribulose biphosphate carboxlase (Rubisco) is the key enzyme responsible for photosynthetic carbon assimilation in catalysing the reaction of COwith ribulose 1,5biophosphate (RuBP) to form two molecules of D-phosphoglyceric acid (PGA) (Parry et al, 2003). Since Rubisco works very slowly, catalyzing only the reaction of a few molecules per second, large quantities of the enzyme are required; consequently Rubisco makes up 30-50% of the soluble protein in leaves (Bock and Khan, 2004). Genetic modification to increase the catalytic rate of Rubisco would have great importance. Parry et al, (2003) reviewed the progress to date, concluding that there are still many technical barriers to overcome and to date all engineering attempts have failed to produce a better Rubisco.","In the presence of O, Rubisco also performs an oxygenase reaction which initiates photorespiratory or C2 cycle () by the formation of phosphoglycolate and 3-phosphoglycerate (3-PGA). The recycling of phosphoglycolate results in an indirect loss of fixed nitrogen and COfrom the cell which need to be recovered. Genetic modification to increase the specificity of Rubisco for COrelative to Oand to increase the catalytic rate of Rubisco in crop plants would have great agronomic importance. Parry et al, (2003) reviewed the progress to date, concluding that there are still many technical barriers to overcome and to date all engineering attempts have thus far failed to produce a better Rubisco (Peterhansel et al. 2008). Furthermore, it has been demonstrated that photorespiration is required in C3 plants to protect plants from photoxidation under high light intensity (Kozaki and Takeba 1996).","C3 and C2 Cycles","In C3 plants under atmospheric conditions, approximately three out of four Rubisco enzymic reactions in C3 plants fix CO(carboxylase reaction, C3 cycle, ). The fourth reaction; however, catalyses an oxygenase reaction () which indirectly results in a net loss of fixed COand NH and the production of a number of intermediate metabolites via the C2 (photorespiration) cycle (). Ultimately, this incurs a substantial metabolic cost through the refixing of COand NH as well as the recycling of the intermediates. Furthermore, when C3 plants experience water stress and\/or elevated temperatures the portion of oxygenase to carboxylase reactions rises courtesy of the elevated Owithin the leaf. Nonetheless it has been demonstrated that photorespiration is required in C3 plants to protect plants from photoxidation under high light intensity (Kozaki and Takeba, 1996) and appears to provide much of the reducing power required for NO assimilation in the leaf (Rachmilevitch et al., 2004).","Organisms capable of oxygenic photosynthesis began their evolution in a vastly different atmosphere (Giordano et al. 2005). One of the most dramatic changes has been the rise in the O:COratio, where the competition between these two gasses for the active site of Rubisco has become progressively restrictive to the rate of carbon fixation. However, some have suggested that the gradual change appears to have provided a lack of evolutionary pressure for Rubisco with a high affinity for COor a Rubisco without oxygenase activity. Indeed, plant Rubiscos are considerd more evolutionarily recent than algal Rubiscos and as such they are much more selective for COover O. Genetic modifications to increase the specificity of Rubisco for COrelative to Ohave failed (Parry, Andralojc et al. 2003).","A significant role of the Coxidative photosynthetic carbon cycle or photorespiratory pathway is the recycling of 2-phosphoglycolate (2PG) produced by the oxygenase activity of Rubisco (Tolbert 1997). 2PG is toxic to the cell; hence it is rapidly dephosphorylated (via phosphoglycolate phosphatase, PGP) to glycolate (Tolbert et al, 1983). Furthermore, it has been demonstrated that photorespiration is required in C3 plants to protect plants from photoxidation under high light intensity (Kozaki and Takeba 1996).","The enzymes that oxidise glycolate to glycoxylate in the photorespiratory pathway are characterised into two structurally different groups. In higher plants, the peroxisome-localized, FMN-containing glycolate oxygenase, GOX (EC 1.1.3.15 ) catalyzes glycolate oxidation using molecular oxygen as the terminal electron acceptor and has a stereopsecificity for L-lactate as an alternative substrate. In contrast, glycolate dehydrogenase, GDH (EC 1.1.99.14) has been characterized only by its non-oxygen-requiring enzymatic reaction and its stereospecificity for D-lactate as an alternative substrate. In most algae, glycolate is oxidised in the mitochondria using a monomeric GDH which is dependent on organic co-factors. The capacity of the reaction seems to be limited by the organic co-factors and consequently many algae excrete glycolate into the medium under photorespiratory growth conditions (Bari et a,2009; Colman et al, 1974). GDH in is a mitochondrially located, low-CO-responsive gene (Nakamura et al, 2005). Other GDH homologs include the so-called glycolate oxidase (GOX) of and other bacteria. In , the GOX complex is composed of three functional subunits, GlcD, GIcE, and GIcF of which GlcD and GIcE share a highly conserved amino acid sequence that includes a putative flavin-binding region. In the GIcF protein, two highly conserved CxxCxxCxxxCP (SEQ ID NO:109) motifs have been recognized, which represent the typical 2x[4Fe-4S] iron-sulfur clusters, as found also in the GlpC subunit of anaerobic G3P dehydrogenase, and ubiquinone oxidoreductase homologs from prokaryotes and eukaryotes (Nakamura et al, 2005).","C4 Cycle","Not all plants use Rubisco to generate 3-PGA as the first stable photosynthetic intermediate. Maize, sugarcane, numerous tropical grasses and some dicotyledonous plants (e.g., ) initially use phosphoenolpyruvate to fix carbon, forming 4-carbon organic acids (Cplants). C4 plants avoid the C2 cycle through modifications to their architecture involving two different types of chloroplast containing cells, mesophyll cells and bundle sheath cells which isolates Rubisco in a relatively rich COenvironment thereby increasing the proportion of carboxylase reactions. This enables these plants to initially use phosphoenolpyruvate to fix carbon, forming 4-carbon organic acids (hence Cplants). Thus the C4 metabolism involves fixing inorganic carbon in one cell type (mesophyll), transporting it to a cell type partially shielded from atmospheric oxygen (bundle sheath), and releasing the inorganic carbon near Rubsico in this oxygen deprived environment.","The leaves of Cplants demonstrate an unusual anatomy involving two different types of chloroplast containing cells, mesophyll cells and bundle sheath cells. Where the mesophyll cells surround the bundle sheath cells which in turn surround the vascular tissue; the chloroplasts of the mesophyll cells contain all the trasmembrane complexes required for the light reactions of photosynthesis but little or no Rubisco while the bundle sheath cell chloroplasts lack stacked thylakoids and contain little PSII. Cplants concentrate COin the bundle sheath cells effectively suppressing Rubiscos oxygenase activity and eliminating photorespiration.","Oxaloacetate is generated from HCO and phosphoenolpyruvate (PEP) by phosphoenolpyruvate carboxylase (PEPC) in the cytosol of mesophyll cells. The HCO ion is used since its aqueous equilibrium is favoured over gaseous CO. Moreover, PEP carboxylase cannot fix oxygen, which has a 3D structure similar to that of CObut not HCO. Depending on the Cplant, oxaloacetate is oxidised to malate or condensed with glutamate to form aspartate and \u03b1 Keto glutarate. The malate and aspartate are transported into the bundle sheath cells and decarboxylated releasing COwhich is then available for Rubisco and incorporation into the Calvin cycle.","The agronomic downside of this evolved modification is an increase in leaf fibre resulting in a comparatively poor digestibility of leaves from C4 plants (e.g., maize, sugarcane, numerous tropical grasses and some dicotyledonous plants such as ). To date, the modification of a C3 plant to emulate the whole C4 process is beyond current biotechnology. Furthermore, attempts to engineer Rubisco to either obliterate oxygenase activity or to decrease the affinity for Ohave failed (for review see Peterhansel et al. 2008).","Interaction with of Nitrate Assimilation","Reducing photorespiration through manipulation of atmospheric COover long periods has led to the unexpected reduction of nitrate assimilation in C3 plants (Rachmilevitch et al., 2004). There are a number of possible explanations including the lowering of available reducing power, reduced ferredoxin and NADH, the former is required for nitrate reductase and glytamate synthetase while latter is required for the reduction of NO (where NADH is produced during the glycine decarboxylase photorespiratory step in the mitochondria). In addition, transport of NO from the cytosol into the chloroplast involves the net diffusion of HNOor co-transport of protons and NO across the chloroplast membrane. This requires the stroma to be more alkaline than the cytosol but the pH gradient is somewhat dissipated by elevated COlevels. Rachmilevitch et al (2004) concluded that nitrate reductase activity by itself was not limiting to nitrate assimilation under lowered photorespiration. They also concluded that it was the form of nitrogen available to the plant that determined the degree to which elevated COlevels would result in an increase in net primary production, i.e., where NH is the dominant nitrogen form. This would suggest that in the absence of changing agronomic fertilisation practices, the legumes stand to benefit most by the reduction of photorespiration since the rhizobial\/legume symbiosis results in the fixation of atmospheric nitrogen in the form of NH rather than NO.","Previous Efforts to Engineering Higher Chloroplast COLevels and Reduced Photorespiration in C3 Plants","A number of investigations have been performed in higher plants to address the limitations of photorespiration. Essentially only one of these appears to have potential applications in the adaptation to higher plants. A recent photorespiratory bypass which increased the efficiency of glycolate recycling was successfuly engineered into and resulted in a 30% increase in leaf biomass (Kebeish et al., 2007). Kebeish et al (2007) transformed to express three genes from (GDH), glyoxylate carboxyligase (GCL), tartronic semialdehyde reductase (TSR) in their chloroplasts (). Combined, these genes recycled glycolate to glycerate in the chloroplast, in other words without the involvement of the peroxisome or mitochondrion. GDH from is a heterotrimer, consisting of glcD, glcE and glcF resulting in plants with a 30% increase in leaf biomass by the end of the growth period (). This pathway included a chloroplast COrelease step which further reduced RubisCO's oxygenase activity in vivo. Moreover, energy and reducing equivalents were thought to be saved by the bypass as it no longer results in the release of ammonium and the energy from glycolate oxidation is saved in reducing equivalents and not consumed during the formation of HO(Maurino and Peterhansel 2010). Peterhansel (2011) concluded that to truly transform a C3 plant into a C4 plant will require the efficient transfer of multiple genes.","Plant Lipid Biosynthesis","All plant cells produce fatty acids from actetyl-CoA by a common pathway localized in plastids (). A portion of the newly synthesized acyl chains is then used for lipid biosynthesis within the plastid (the prokaryotic pathway); however, a major portion is exported into the cytosol for glycerolipid assembly at the endoplasmic reticulum (ER) or other sites (the eukaryotic pathway). In addition, some of the extraplastidial glycerolipids return to the plastid, which results in considerable intermixing between the plastid and ER lipid pools (Ohlrogge and Jaworski 1997).","The simplest description of the plastidial pathway of fatty acid biosynthesis consists of two enzyme systems: acetyl-CoA carboxylase (ACCase) and fatty acid synthase (FAS). ACCase catalyzes the formation of malonyl-CoA from acetyl-CoA, and FAS transfers the malonyl moiety to acyl carrier protein (ACP) and catalyzes the extension of the growing acyl chain with malonyl-ACP.","The initial fatty acid synthesis reaction is catalyzed by 3-ketoacyl-ACP III (KAS III) which results in the condensation of acetyl-CoA and malonyl-ACP. Subsequent condensations are catalyzed by KAS I and KAS II. Before a subsequent cycle of fatty acid synthesis begins, the 3-ketoacyl-ACP intermediate is reduced to the saturated acyl-ACP in the remaining FAS reactions, catalyzed sequentially by the 3-ketoacyl-ACP reductase, 3 hydroxyacyl-ACP dehydrase, and the enoyl-ACP reductase.","The final products of FAS are usually 16:0 and 18:0-ACP, and the final fatty acid composition of a plant cell is in large part determined by activities of several enzymes that use these acyl-ACPs at the termination phase of fatty acid synthesis. Stearoyl-ACP desaturase modifies the final product of FAS by insertion of a cis double bond at the 9 position of the C18:0-ACP. Reactions of fatty acid synthesis are terminated by hydrolysis or transfer of the acyl chain from the ACP. Hydrolysis is catalyzed by acyl-ACP thioesterases, of which there are two main types: one thioesterase relatively specific for 18:1-ACP and a second more specific for saturated acyl-ACPs. Fatty acids that have been released from ACPs by thioesterases leave the plastid and enter into the eukaryotic lipid pathway, where they are primarily esterified to glycerolipids on the ER. Acyl transferases in the plastid, in contrast to thioesterases, terminate fatty acid synthesis by transesterifying acyl moieties from ACP to glycerol, and they are an essential part of the prokaryotic lipid pathway leading to plastid glycerolipid assembly.","Predicted Link Between Elevating Lipid Biosynthesis, Higher Chloroplast COLevels and Reducing Chloroplast Photorespiration","In green seeds it was recently discovered that Rubisco with out the Calvin cycle bypasses the upper part of glycolysis in plastids and provides a higher carbon-use efficiency that allows re-fixation of COformed by the plastid pyruvate dehydrogenase complex (Schwender et al., 2004). Acetyl CoA produced in plastids from pyruvate is activated to malonyl CoA; the malonyl group is subsequently transferred to ACP giving rise to malonyl ACP, the primary substrate of the fatty acid synthase complex. The formation of malonyl CoA is the committed step in fatty acid synthesis and is catalyzed by the highly regulated plastidic acetyl CoA carboxylase complex (Nikolau et al., 2003).","It has been speculated that when leaves synthesize triacylglyceride (TAG) the re-fixation of COreleased by the activation of pyruvate to malonyl CoA will be re-fixed by photosynthesis (Durret et al 2008). Fatty acids synthesised in the plastid are transported to the ER and sequentially acylated onto a glycerol backbone via the Kennedy pathway. This culminates in the production of TAG via over expression of the enzyme DGAT. In this case the 3-phosphoglyceric acid is synthesised by Rubisco (without the Calvin cycle) rather than the transformation of sugars. The subsequent transformation of this to acetyl-CoA (via the pyruvate intermediate) results in the release of COin the chloroplast (). This increases the partial pressure of COrelative to Oin the chloroplast thus reducing the proportion of C2 to C3 cycles initiated by Rubisco. However, it has been found that the subsequent catabolism of this TAG negates this advantage (Winichayakul et al., 2008). The over expression of DGAT leads to the accumulation of TAG which is subsequently degraded by lipases resulting in the release of free fatty acids. Some of these free fatty acids are catabolised by \u03b2-oxidation in the peroxisome while others set up a futile cycle by re-entering the ER where they are re-incorporated into TAG (). This resulting futile cycle reduces the demand for the de-novo synthesis of new lipids; subsequently the level of COrecycling within the chloroplast is reduced to (or close to) wild type levels which leads to the resumption of the wild type ratio of Cto Ccycles being performed by Rubisco within the C3 photosynthetic cell.","Without being limited by theory, the applicants propose the following model for the observed increase in COassimilation. The co-expression of DGAT and a modified oleosin containing engineered cysteine residues leads to the accumulation of TAG which is encapsulated by the modified oleosin containing engineered cysteine residues (). This prevents the degradation of TAG by lipases and thus also prevents futile lipid recycling. Consequently, this ensures a continual demand for the de-novo lipid synthesis and the subsequent elevation of COpartial pressure in the photosynthetic cell which inturn results in a continued suppression of C2 cycles relative to C3 cycles and an elevation of the COassimilation rate.","Subsequently this should result in a number of benefits for all multicellular and unicellular organisms initially fixing carbon using the C3 photosynthetic pathway, including:\n\n","Vegetative tissue include, shoots, leaves, roots, stems. A preferred vegetative tissue is a leaf.","Non-Photosynthetic Tissues\/Organs","The term non-photosynthetic tissues\/organs means tissues or organs of the plant which do not undergo substantive photosynthesis during the normal life cycle of the plant.","It is understood by those skilled in the art that even non-photosynthetic tissues\/organs can be made to photosynthesise by exposure to light but when they do so the level of photosynthesis is not \u201csubstantive\u201d and is inconsequential relative to that performed by normal photosynthetic tissues.","In one embodiment the non-photosynthetic tissue\/organ is selected from below ground tissue\/organs of the plant. In a further embodiment the below ground tissue\/organ is selected from root, tuber, bulb, corm and rhizome. In a further embodiment the non-photosynthetic tissue\/organ is selected from root, tuber, bulb, corm, rhizome, and endosperm. In a further embodiment the non-photosynthetic tissue\/organ is root.","Tissue\/Organ Specific and Preferred Promoters","A tissue\/organ preferred promoter is a promoter that drives expression of an operably linked polynucleotide in a particular tissue\/organ at a higher level than in other tissues\/organs. A tissue specific promoter is a promoter that drives expression of an operably linked polynucleotide speicifically in a particular tissue\/organ. Even with tissue\/organ specific promoters, there is usually a small amount of expression in at least one other tissue. A tissue specific promoter is by definition also a tissue preferred promoter.","Vegetative Tissue Specific Promoters","An example of a vegetative specific promoter is found in U.S. Pat. No. 6,229,067; and 7,629,454; and 7,153,953; and 6,228,643.","Pollen Specific Promoters","An example of a pollen specific promoter is found in U.S. Pat. Nos. 7,141,424; and 5,545,546; and 5,412,085; and 5,086,169; and 7,667,097.","Seed Specific Promoters","An example of a seed specific promoter is found in U.S. Pat. Nos. 6,342,657; and 7,081,565; and 7,405,345; and 7,642,346; and 7,371,928.","Fruit Specific Promoters","An example of a fruit specific promoter is found in U.S. Pat. Nos. 5,536,653; and 6,127,179; and 5,608,150; and 4,943,674.","Non-Photosynthetic Tissue Preferred Promoters","Non-photosynthetic tissue preferred promoters include those preferentially expressed in non-photosynthetic tissues\/organs of the plant.","Non-photosynthetic tissue preferred promoters may also include light repressed promoters.","Light Repressed Promoters","An example of a light repressed promoter is found in U.S. Pat. Nos. 5,639,952 and in 5,656,496.","Root Specific Promoters","An example of a root specific promoter is found in U.S. Pat. No. 5,837,848; and US 2004\/0067506 and US 2001\/0047525.","Tuber Specific Promoters","An example of a tuber specific promoter is found in U.S. Pat. No. 6,184,443.","Bulb Specific Promoters","An example of a bulb specific promoter is found in Smeets et al., (1997) Plant Physiol. 113:765-771.","Rhizome Preferred Promoters","An example of a rhizome preferred promoter is found Seong Jang et al., (2006) Plant Physiol. 142:1148-1159.","Endosperm Specific Promoters","An example of an endosperm specific promoter is found in U.S. Pat. No. 7,745,697.","Corm Promoters","An example of a promoter capable of driving expression in a corm is found in Schenk et al., (2001) Plant Molecular Biology, 47:399-412.","Photosythetic Tissue Preferred Promoters","Photosythetic tissue preferred promoters include those that are preferrentially expressed in photosynthetic tissues of the plants. Photosynthetic tissues of the plant include leaves, stems, shoots and above ground parts of the plant. Photosythetic tissue preferred promoters include light regulated promoters.","Light Regulated Promoters","Numerous light regulated promoters are known to those skilled in the art and include for example chlorophyll a\/b (Cab) binding protein promoters and Rubisco Small Subunit (SSU) promoters. An example of a light regulated promoter is found in U.S. Pat. No. 5,750,385. Light regulated in this context means light inducible or light induced.","Relative Terms","The relative terms, such as increased and reduced as used herein with respect to plants, are relative to a control plant. Suitable control plants include non-transformed or wild-type versions of plant of the same variety and\/or species as the transformed plant used in the method of the invention. Suitable control plants also include plants of the same variety and\/or species as the transformed plant that are transformed with a control construct. Suitable control constructs include emptry vector constructs, known to those skilled in the art. Suitable control plants also include plants that have not been transformed with a polynucleotide encoding a modified oleosin including at least one artificially introduced cysteine. Suitable control plants also include plants that do not express a modified oleosin including at least one artificially introduced cysteine.","The term \u201ctotal lipid\u201d as used herein includes fats, oils, waxes, sterols, glycerol lipids, monoglycerides, diglycerides, phospholipids, monogalactolipids, digalactolipids, phosphatidylcholines, phosphatidylethanolamine, phosphatidylglycerol, phosphatidylinositol, sulfoguinovosyldiacylglycerol, and triglycerides.","The term \u201coil\u201d as used herein preferably refers to triacylglycerol (TAG)","The term \u201cbiomass\u201d refers to the size and\/or mass and\/or number of vegetative organs of the plant at a particular age or developmental stage. Thus a plant with increased biomass has increased size and\/or mass and\/or number of vegetative organs than a suitable control plant of the same age or at an equivalent developmental stage. Increased biomass may also involve an increase in rate of growth and\/or rate offormation of vegetative organs during some or all periods of the life cycle of a plant relative to a suitable control. Thus increased biomass may result in an advance in the time taken for such a plant to reach a certain developmental stage.","The terms \u201cseed yield\u201d, \u201cfruit yield\u201d and \u201corgan yield\u201d refer to the size and\/or mass and\/or number of seed, fruit or organs produced by a plant. Thus a plant with increased seed, fruit or organ yield has increased size and\/or mass and\/or number of seeds, fruit or organs respectively, relative to a control plant at the same age or an equivalent developmental stage.","The terms \u201cincreased drought tolerance\u201d and \u201cincreased water use efficiency\u201d or grammatical equivalents thereof, is intended to describe a plant which performs more favourably in any aspect of growth and development under, or after, sub-optimal hydration conditions than do control plants in the same conditions.","The term \u201cincreased high temperature tolerance\u201d or grammatical equivalents thereof, is intended to describe plant which performs more favourably in any aspect of growth and development under, or after, sub-optimal elevated temperature conditions than do control plants in the same conditions.","The term \u201cincreased high oxygen concentration tolerance\u201d or grammatical equivalents thereof is intended to describe plant which performs more favourably in any aspect of growth and development under, or after, sub-optimal elevated oxygen concentrations than do control plants in the same conditions.","The term \u201cincreased nitrogen use efficiency\u201d or grammatical equivalents thereof is intended to describe plant which performs more favourably in any aspect of growth and development under, or after, sub-optimal reduced nitrogen conditions than do control plants in the same conditions.","The term \u201cincreased rate of COassimilation\u201d or grammatical equivalents thereof is intended to describe plant which assimilates more COunder any given conditions than does a control plant in the same conditions.","The term \u201cincreased rate of photosynthesis\u201d or grammatical equivalents thereof is intended to describe plant which accumulates more photosynthate under any given conditions than does a control plant in the same conditions.","The term \u201cincreased growth rate\u201d or grammatical equivalents thereof is intended to describe plant which grows more quickly under any given conditions than does a control plant in the same conditions.","The term \u201cdelayed flowering\u201d or grammatical equivalents thereof is intended to describe plant which flowers later under any given conditions than does a control plant in the same conditions.","The term \u201cincreased chloroplast COconcentation\u201d or grammatical equivalents thereof is intended to describe a plant has a higher concentration of COin the chloroplast under any given conditions than does a control plant in the same conditions.","The term \u201cdecreased rate of photorespiration\u201d or grammatical equivalents thereof, is intended to describe a plant which shows less photorespiration under any given conditions than does a control plant in the same conditions.","The term \u201cdecreased loss of fixed carbon\u201d or grammatical equivalents thereof, is intended to describe plant which loses less fixed carbon under any given conditions than does a control plant in the same conditions.","Polynucleotides and Fragments","The term \u201cpolynucleotide(s),\u201d as used herein, means a single or double-stranded deoxyribonucleotide or ribonucleotide polymer of any length but preferably at least 15 nucleotides, and include as non-limiting examples, coding and non-coding sequences of a gene, sense and antisense sequences complements, exons, introns, genomic DNA, cDNA, pre-mRNA, mRNA, rRNA, siRNA, miRNA, tRNA, ribozymes, recombinant polypeptides, isolated and purified naturally occurring DNA or RNA sequences, synthetic RNA and DNA sequences, nucleic acid probes, primers and fragments.","A \u201cfragment\u201d of a polynucleotide sequence provided herein is a subsequence of contiguous nucleotides that is capable of specific hybridization to a target of interest, e.g., a sequence that is at least 15 nucleotides in length. The fragments of the invention comprise 15 nucleotides, preferably at least 16 nucleotides, more preferably at least 17 nucleotides, more preferably at least 18 nucleotides, more preferably at least 19 nucleotides, more preferably at least 20 nucleotides, more preferably at least 21 nucleotides, more preferably at least 22 nucleotides, more preferably at least 23 nucleotides, more preferably at least 24 nucleotides, more preferably at least 25 nucleotides, more preferably at least 26 nucleotides, more preferably at least 27 nucleotides, more preferably at least 28 nucleotides, more preferably at least 29 nucleotides, more preferably at least 30 nucleotides, more preferably at least 31 nucleotides, more preferably at least 32 nucleotides, more preferably at least 33 nucleotides, more preferably at least 34 nucleotides, more preferably at least 35 nucleotides, more preferably at least 36 nucleotides, more preferably at least 37 nucleotides, more preferably at least 38 nucleotides, more preferably at least 39 nucleotides, more preferably at least 40 nucleotides, more preferably at least 41 nucleotides, more preferably at least 42 nucleotides, more preferably at least 43 nucleotides, more preferably at least 44 nucleotides, more preferably at least 45 nucleotides, more preferably at least 46 nucleotides, more preferably at least 47 nucleotides, more preferably at least 48 nucleotides, more preferably at least 49 nucleotides, more preferably at least 50 nucleotides, more preferably at least 51 nucleotides, more preferably at least 52 nucleotides, more preferably at least 53 nucleotides, more preferably at least 54 nucleotides, more preferably at least 55 nucleotides, more preferably at least 56 nucleotides, more preferably at least 57 nucleotides, more preferably at least 58 nucleotides, more preferably at least 59 nucleotides, more preferably at least 60 nucleotides, more preferably at least 61 nucleotides, more preferably at least 62 nucleotides, more preferably at least 63 nucleotides, more preferably at least 64 nucleotides, more preferably at least 65 nucleotides, more preferably at least 66 nucleotides, more preferably at least 67 nucleotides, more preferably at least 68 nucleotides, more preferably at least 69 nucleotides, more preferably at least 70 nucleotides, more preferably at least 71 nucleotides, more preferably at least 72 nucleotides, more preferably at least 73 nucleotides, more preferably at least 74 nucleotides, more preferably at least 75 nucleotides, more preferably at least 76 nucleotides, more preferably at least 77 nucleotides, more preferably at least 78 nucleotides, more preferably at least 79 nucleotides, more preferably at least 80 nucleotides, more preferably at least 81 nucleotides, more preferably at least 82 nucleotides, more preferably at least 83 nucleotides, more preferably at least 84 nucleotides, more preferably at least 85 nucleotides, more preferably at least 86 nucleotides, more preferably at least 87 nucleotides, more preferably at least 88 nucleotides, more preferably at least 89 nucleotides, more preferably at least 90 nucleotides, more preferably at least 91 nucleotides, more preferably at least 92 nucleotides, more preferably at least 93 nucleotides, more preferably at least 94 nucleotides, more preferably at least 95 nucleotides, more preferably at least 96 nucleotides, more preferably at least 97 nucleotides, more preferably at least 98 nucleotides, more preferably at least 99 nucleotides, more preferably at least 100 nucleotides, more preferably at least 150 nucleotides, more preferably at least 200 nucleotides, more preferably at least 250 nucleotides, more preferably at least 300 nucleotides, more preferably at least 350 nucleotides, more preferably at least 400 nucleotides, more preferably at least 450 nucleotides and most preferably at least 500 nucleotides of contiguous nucleotides of a polynucleotide disclosed. A fragment of a polynucleotide sequence can be used in antisense, RNA interference (RNAi), gene silencing, triple helix or ribozyme technology, or as a primer, a probe, included in a microarray, or used in polynucleotide-based selection methods of the invention.","The term \u201cprimer\u201d refers to a short polynucleotide, usually having a free 3\u2032OH group, that is hybridized to a template and used for priming polymerization of a polynucleotide complementary to the target.","The term \u201cprobe\u201d refers to a short polynucleotide that is used to detect a polynucleotide sequence that is complementary to the probe, in a hybridization-based assay. The probe may consist of a \u201cfragment\u201d of a polynucleotide as defined herein.","Polypeptides and Fragments","The term \u201cpolypeptide\u201d, as used herein, encompasses amino acid chains of any length but preferably at least 5 amino acids, including full-length proteins, in which amino acid residues are linked by covalent peptide bonds. Polypeptides of the present invention, or used in the methods of the invention, may be purified natural products, or may be produced partially or wholly using recombinant or synthetic techniques. The term may refer to a polypeptide, an aggregate of a polypeptide such as a dimer or other multimer, a fusion polypeptide, a polypeptide fragment, a polypeptide variant, or derivative thereof.","A \u201cfragment\u201d of a polypeptide is a subsequence of the polypeptide that performs a function that is required for the biological activity and\/or provides three dimensional structure of the polypeptide. The term may refer to a polypeptide, an aggregate of a polypeptide such as a dimer or other multimer, a fusion polypeptide, a polypeptide fragment, a polypeptide variant, or derivative thereof capable of performing the above enzymatic activity.","The term \u201cisolated\u201d as applied to the polynucleotide or polypeptide sequences disclosed herein is used to refer to sequences that are removed from their natural cellular environment. An isolated molecule may be obtained by any method or combination of methods including biochemical, recombinant, and synthetic techniques.","The term \u201crecombinant\u201d refers to a polynucleotide sequence that is removed from sequences that surround it in its natural context and\/or is recombined with sequences that are not present in its natural context.","A \u201crecombinant\u201d polypeptide sequence is produced by translation from a \u201crecombinant\u201d polynucleotide sequence.","The term \u201cderived from\u201d with respect to polynucleotides or polypeptides of the invention being derived from a particular genera or species, means that the polynucleotide or polypeptide has the same sequence as a polynucleotide or polypeptide found naturally in that genera or species. The polynucleotide or polypeptide, derived from a particular genera or species, may therefore be produced synthetically or recombinantly.","Variants","As used herein, the term \u201cvariant\u201d refers to polynucleotide or polypeptide sequences different from the specifically identified sequences, wherein one or more nucleotides or amino acid residues is deleted, substituted, or added. Variants may be naturally occurring allelic variants, or non-naturally occurring variants. Variants may be from the same or from other species and may encompass homologues, paralogues and orthologues. In certain embodiments, variants of the inventive polypeptides and polypeptides possess biological activities that are the same or similar to those of the inventive polypeptides or polypeptides. The term \u201cvariant\u201d with reference to polypeptides and polypeptides encompasses all forms of polypeptides and polypeptides as defined herein.","Polynucleotide Variants","Variant polynucleotide sequences preferably exhibit at least 50%, more preferably at least 51%, more preferably at least 52%, more preferably at least 53%, more preferably at least 54%, more preferably at least 55%, more preferably at least 56%, more preferably at least 57%, more preferably at least 58%, more preferably at least 59%, more preferably at least 60%, more preferably at least 61%, more preferably at least 62%, more preferably at least 63%, more preferably at least 64%, more preferably at least 65%, more preferably at least 66%, more preferably at least 67%, more preferably at least 68%, more preferably at least 69%, more preferably at least 70%, more preferably at least 71%, more preferably at least 72%, more preferably at least 73%, more preferably at least 74%, more preferably at least 75%, more preferably at least 76%, more preferably at least 77%, more preferably at least 78%, more preferably at least 79%, more preferably at least 80%, more preferably at least 81%, more preferably at least 82%, more preferably at least 83%, more preferably at least 84%, more preferably at least 85%, more preferably at least 86%, more preferably at least 87%, more preferably at least 88%, more preferably at least 89%, more preferably at least 90%, more preferably at least 91%, more preferably at least 92%, more preferably at least 93%, more preferably at least 94%, more preferably at least 95%, more preferably at least 96%, more preferably at least 97%, more preferably at least 98%, and most preferably at least 99% identity to a sequence of the present invention. Identity is found over a comparison window of at least 20 nucleotide positions, preferably at least 50 nucleotide positions, more preferably at least 100 nucleotide positions, and most preferably over the entire length of a polynucleotide of the invention.","Polynucleotide sequence identity can be determined in the following manner. The subject polynucleotide sequence is compared to a candidate polynucleotide sequence using BLASTN (from the BLAST suite of programs, version 2.2.5 [Nov. 2002]) in bl2seq (Tatiana A. Tatusova, Thomas L. Madden (1999), \u201cBlast 2 sequences\u2014a new tool for comparing protein and nucleotide sequences\u201d, FEMS Microbiol Lett. 174:247-250), which is publicly available from NCBI (ftp:\/\/ftp.ncbi.nih.gov\/blast\/). The default parameters of bl2seq are utilized except that filtering of low complexity parts should be turned off.","The identity of polynucleotide sequences may be examined using the following unix command line parameters:\n\n","The parameter \u2212F F turns off filtering of low complexity sections. The parameter \u2212p selects the appropriate algorithm for the pair of sequences. The bl2seq program reports sequence identity as both the number and percentage of identical nucleotides in a line \u201cIdentities=\u201d.","Polynucleotide sequence identity may also be calculated over the entire length of the overlap between a candidate and subject polynucleotide sequences using global sequence alignment programs (e.g. Needleman, S. B. and Wunsch, C. D. (1970) J. Mol. Biol. 48, 443-453). A full implementation of the Needleman-Wunsch global alignment algorithm is found in the needle program in the EMBOSS package (Rice,P. Longden,l. and Bleasby,A. EMBOSS: The European Molecular Biology Open Software Suite, Trends in Genetics June 2000, vol 16, No 6. pp.276-277) which can be obtained from www<dot>hgmp<dot>mrc<dot>ac <dot>uk\/Software\/EMBOSS\/. The European Bioinformatics Institute server also provides the facility to perform EMBOSS-needle global alignments between two sequences on line at www<dot>ebi<dot>ac<dot>uk\/emboss\/align\/.","Alternatively the GAP program may be used which computes an optimal global alignment of two sequences without penalizing terminal gaps. GAP is described in the following paper: Huang, X. (1994) On Global Sequence Alignment. Computer Applications in the Biosciences 10, 227-235.","A preferred method for calculating polynucleotide % sequence identity is based on aligning sequences to be compared using Clustal X (Jeanmougin et al., 1998, Trends Biochem. Sci. 23, 403-5.)","Polynucleotide variants of the present invention also encompass those which exhibit a similarity to one or more of the specifically identified sequences that is likely to preserve the functional equivalence of those sequences and which could not reasonably be expected to have occurred by random chance. Such sequence similarity with respect to polypeptides may be determined using the publicly available bl2seq program from the BLAST suite of programs (version 2.2.5 [Nov. 2002]) from NCBI (ftp:\/\/ftp.ncbi.nih.gov\/blast\/).","The similarity of polynucleotide sequences may be examined using the following unix command line parameters:\n\n","The parameter \u2212F F turns off filtering of low complexity sections. The parameter \u2212p selects the appropriate algorithm for the pair of sequences. This program finds regions of similarity between the sequences and for each such region reports an \u201cE value\u201d which is the expected number of times one could expect to see such a match by chance in a database of a fixed reference size containing random sequences. The size of this database is set by default in the bl2seq program. For small E values, much less than one, the E value is approximately the probability of such a random match.","Variant polynucleotide sequences preferably exhibit an E value of less than 1\u00d710\u22126 more preferably less than 1\u00d710\u22129, more preferably less than 1\u00d710\u221212, more preferably less than 1\u00d710\u221215, more preferably less than 1\u00d710\u221218, more preferably less than 1\u00d710\u221221, more preferably less than 1\u00d710\u221230, more preferably less than 1\u00d710\u221240, more preferably less than 1\u00d710\u221250, more preferably less than 1\u00d710\u221260, more preferably less than 1\u00d710\u221270, more preferably less than 1\u00d710\u221280, more preferably less than 1\u00d710\u221290 and most preferably less than 1\u00d710\u2212100 when compared with any one of the specifically identified sequences.","Alternatively, variant polynucleotides of the present invention, or used in the methods of the invention, hybridize to the specified polynucleotide sequences, or complements thereof under stringent conditions.","The term \u201chybridize under stringent conditions\u201d, and grammatical equivalents thereof, refers to the ability of a polynucleotide molecule to hybridize to a target polynucleotide molecule (such as a target polynucleotide molecule immobilized on a DNA or RNA blot, such as a Southern blot or Northern blot) under defined conditions of temperature and salt concentration. The ability to hybridize under stringent hybridization conditions can be determined by initially hybridizing under less stringent conditions then increasing the stringency to the desired stringency.","With respect to polynucleotide molecules greater than about 100 bases in length, typical stringent hybridization conditions are no more than 25 to 30\u00b0 C. (for example, 10\u00b0 C.) below the melting temperature (Tm) of the native duplex (see generally, Sambrook et al., Eds, 1987, Molecular Cloning, A Laboratory Manual, 2nd Ed. Cold Spring Harbor Press; Ausubel et al., 1987, Current Protocols in Molecular Biology, Greene Publishing,). Tm for polynucleotide molecules greater than about 100 bases can be calculated by the formula Tm=81. 5+0.41% (G+C-log(Na+). (Sambrook et al., Eds, 1987, Molecular Cloning, A Laboratory Manual, 2nd Ed. Cold Spring Harbor Press; Bolton and McCarthy, 1962, PNAS 84:1390). Typical stringent conditions for polynucleotide of greater than 100 bases in length would be hybridization conditions such as prewashing in a solution of 6\u00d7SSC, 0.2% SDS; hybridizing at 65\u00b0 C., 6\u00d7SSC, 0.2% SDS overnight; followed by two washes of 30 minutes each in 1\u00d7SSC, 0.1% SDS at 65\u00b0 C. and two washes of 30 minutes each in 0.2\u00d7SSC, 0.1% SDS at 65\u00b0 C.","With respect to polynucleotide molecules having a length less than 100 bases, exemplary stringent hybridization conditions are 5 to 10\u00b0 C. below Tm. On average, the Tm of a polynucleotide molecule of length less than 100 bp is reduced by approximately (500\/oligonucleotide length)\u00b0 C.","With respect to the DNA mimics known as peptide nucleic acids (PNAs) (Nielsen et al., Science. 1991 Dec. 6; 254(5037):1497-500) Tm values are higher than those for DNA-DNA or DNA-RNA hybrids, and can be calculated using the formula described in Giesen et al., Nucleic Acids Res. 1998 Nov. 1; 26(21):5004-6. Exemplary stringent hybridization conditions for a DNA-PNA hybrid having a length less than 100 bases are 5 to 10\u00b0 C. below the Tm.","Variant polynucleotides of the present invention, or used in the methods of the invention, also encompasses polynucleotides that differ from the sequences of the invention but that, as a consequence of the degeneracy of the genetic code, encode a polypeptide having similar activity to a polypeptide encoded by a polynucleotide of the present invention. A sequence alteration that does not change the amino acid sequence of the polypeptide is a \u201csilent variation\u201d. Except for ATG (methionine) and TGG (tryptophan), other codons for the same amino acid may be changed by art recognized techniques, e.g., to optimize codon expression in a particular host organism.","Polynucleotide sequence alterations resulting in conservative substitutions of one or several amino acids in the encoded polypeptide sequence without significantly altering its biological activity are also included in the invention. A skilled artisan will be aware of methods for making phenotypically silent amino acid substitutions (see, e.g., Bowie et al., 1990, Science 247, 1306).","Variant polynucleotides due to silent variations and conservative substitutions in the encoded polypeptide sequence may be determined using the publicly available bl2seq program from the BLAST suite of programs (version 2.2.5 [Nov. 2002]) from NCBI (ftp:\/\/ftp.ncbi.nih.gov\/blast\/) via the tblastx algorithm as previously described.","Polypeptide Variants","The term \u201cvariant\u201d with reference to polypeptides encompasses naturally occurring, recombinantly and synthetically produced polypeptides. Variant polypeptide sequences preferably exhibit at least 50%, more preferably at least 51%, more preferably at least 52%, more preferably at least 53%, more preferably at least 54%, more preferably at least 55%, more preferably at least 56%, more preferably at least 57%, more preferably at least 58%, more preferably at least 59%, more preferably at least 60%, more preferably at least 61%, more preferably at least 62%, more preferably at least 63%, more preferably at least 64%, more preferably at least 65%, more preferably at least 66%, more preferably at least 67%, more preferably at least 68%, more preferably at least 69%, more preferably at least 70%, more preferably at least 71%, more preferably at least 72%, more preferably at least 73%, more preferably at least 74%, more preferably at least 75%, more preferably at least 76%, more preferably at least 77%, more preferably at least 78%, more preferably at least 79%, more preferably at least 80%, more preferably at least 81%, more preferably at least 82%, more preferably at least 83%, more preferably at least 84%, more preferably at least 85%, more preferably at least 86%, more preferably at least 87%, more preferably at least 88%, more preferably at least 89%, more preferably at least 90%, more preferably at least 91%, more preferably at least 92%, more preferably at least 93%, more preferably at least 94%, more preferably at least 95%, more preferably at least 96%, more preferably at least 97%, more preferably at least 98%, and most preferably at least 99% identity to a sequences of the present invention. Identity is found over a comparison window of at least 20 amino acid positions, preferably at least 50 amino acid positions, more preferably at least 100 amino acid positions, and most preferably over the entire length of a polypeptide of the invention.","Polypeptide sequence identity can be determined in the following manner. The subject polypeptide sequence is compared to a candidate polypeptide sequence using BLASTP (from the BLAST suite of programs, version 2.2.5 [Nov. 2002]) in bl2seq, which is publicly available from NCBI (ftp:\/\/ftp.ncbi.nih.gov\/blast\/). The default parameters of bl2seq are utilized except that filtering of low complexity regions should be turned off.","Polypeptide sequence identity may also be calculated over the entire length of the overlap between a candidate and subject polynucleotide sequences using global sequence alignment programs. EMBOSS-needle (available at www<dot>ebi<dot>ac<dot>uk\/emboss\/align\/)and GAP (Huang, X. (1994) On Global Sequence Alignment. Computer Applications in the Biosciences 10, 227-235.) as discussed above are also suitable global sequence alignment programs for calculating polypeptide sequence identity.","A preferred method for calculating polypeptide % sequence identity is based on aligning sequences to be compared using Clustal X (Jeanmougin et al., 1998, Trends Biochem. Sci. 23, 403-5.)","Polypeptide variants of the present invention, or used in the methods of the invention, also encompass those which exhibit a similarity to one or more of the specifically identified sequences that is likely to preserve the functional equivalence of those sequences and which could not reasonably be expected to have occurred by random chance. Such sequence similarity with respect to polypeptides may be determined using the publicly available bl2seq program from the BLAST suite of programs (version 2.2.5 [Nov. 2002]) from NCBI (ftp:\/\/ftp.ncbi.nih.gov\/blast\/). The similarity of polypeptide sequences may be examined using the following unix command line parameters:\n\n","Variant polypeptide sequences preferably exhibit an E value of less than 1\u00d710\u22126 more preferably less than 1\u00d710\u22129, more preferably less than 1\u00d710\u221212, more preferably less than 1\u00d710\u221215, more preferably less than 1\u00d710\u221218, more preferably less than 1\u00d710\u221221, more preferably less than 1\u00d710\u221230, more preferably less than 1\u00d710\u221240, more preferably less than 1\u00d710\u221250, more preferably less than 1\u00d710\u221260, more preferably less than 1\u00d710\u221270, more preferably less than 1\u00d710\u221280, more preferably less than 1\u00d710\u221290 and most preferably 1\u00d710\u2212100 when compared with any one of the specifically identified sequences.","The parameter \u2212F F turns off filtering of low complexity sections. The parameter \u2212p selects the appropriate algorithm for the pair of sequences. This program finds regions of similarity between the sequences and for each such region reports an \u201cE value\u201d which is the expected number of times one could expect to see such a match by chance in a database of a fixed reference size containing random sequences. For small E values, much less than one, this is approximately the probability of such a random match.","Conservative substitutions of one or several amino acids of a described polypeptide sequence without significantly altering its biological activity are also included in the invention. A skilled artisan will be aware of methods for making phenotypically silent amino acid substitutions (see, e.g., Bowie et al., 1990, Science 247, 1306).","Constructs, Vectors and Components Thereof.","The term \u201cgenetic construct\u201d refers to a polynucleotide molecule, usually double-stranded DNA, which may have inserted into it another polynucleotide molecule (the insert polynucleotide molecule) such as, but not limited to, a cDNA molecule. A genetic construct may contain the necessary elements that permit transcribing the insert polynucleotide molecule, and, optionally, translating the transcript into a polypeptide. The insert polynucleotide molecule may be derived from the host cell, or may be derived from a different cell or organism and\/or may be a recombinant polynucleotide. Once inside the host cell the genetic construct may become integrated in the host chromosomal DNA. The genetic construct may be linked to a vector.","The term \u201cvector\u201d refers to a polynucleotide molecule, usually double stranded DNA, which is used to transport the genetic construct into a host cell. The vector may be capable of replication in at least one additional host system, such as ","The term \u201cexpression construct\u201d refers to a genetic construct that includes the necessary elements that permit transcribing the insert polynucleotide molecule, and, optionally, translating the transcript into a polypeptide. An expression construct typically comprises in a 5\u2032 to 3\u2032 direction:\n\n","The term \u201ccoding region\u201d or \u201copen reading frame\u201d (ORF) refers to the sense strand of a genomic DNA sequence or a cDNA sequence that is capable of producing a transcription product and\/or a polypeptide under the control of appropriate regulatory sequences. The coding sequence may, in some cases, identified by the presence of a 5\u2032 translation start codon and a 3\u2032 translation stop codon. When inserted into a genetic construct, a \u201ccoding sequence\u201d is capable of being expressed when it is operably linked to promoter and terminator sequences.","\u201cOperably-linked\u201d means that the sequenced to be expressed is placed under the control of regulatory elements that include promoters, tissue-specific regulatory elements, temporal regulatory elements, enhancers, repressors and terminators.","The term \u201cnoncoding region\u201d refers to untranslated sequences that are upstream of the translational start site and downstream of the translational stop site. These sequences are also referred to respectively as the 5\u2032 UTR and the 3\u2032 UTR. These regions include elements required for transcription initiation and termination, mRNA stability, and for regulation of translation efficiency.","Terminators are sequences, which terminate transcription, and are found in the 3\u2032 untranslated ends of genes downstream of the translated sequence. Terminators are important determinants of mRNA stability and in some cases have been found to have spatial regulatory functions.","The term \u201cpromoter\u201d refers to nontranscribed cis-regulatory elements upstream of the coding region that regulate gene transcription. Promoters comprise cis-initiator elements which specify the transcription initiation site and conserved boxes such as the TATA box, and motifs that are bound by transcription factors. Introns within coding sequences can also regulate transcription and influence post-transcriptional processing (including splicing, capping and polyadenylation).","A promoter may be homologous with respect to the polynucleotide to be expressed. This means that the promoter and polynucleotide are found operably linked in nature.","Alternatively the promoter may be heterologous with respect to the polynucleotide to be expressed. This means that the promoter and the polynucleotide are not found operably linked in nature.","A \u201ctransgene\u201d is a polynucleotide that is taken from one organism and introduced into a different organism by transformation. The transgene may be derived from the same species or from a different species as the species of the organism into which the transgene is introduced.","An \u201cinverted repeat\u201d is a sequence that is repeated, where the second half of the repeat is in the complementary strand, e.g.,",{"@attributes":{"id":"p-0493","num":"0549"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"133pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["(5\u2032)GATCTA.......TAGATC(3\u2032)","(SEQ\u2003ID\u2003NO:\u2003110)"]},{"entry":{}},{"entry":["(3\u2032)CTAGAT.......ATCTAG(5\u2032)","(SEQ\u2003ID\u2003NO:\u2003111)"]}]}}}}},"Read-through transcription will produce a transcript that undergoes complementary base-pairing to form a hairpin structure provided that there is a 3-5 bp spacer between the repeated regions.","Host Cells","Host cells may be derived from, for example, bacterial, fungal, yeast, insect, mammalian, algal or plant organisms. Host cells may also be synthetic cells. Preferred host cells are eukaryotic cells. A particularly preferred host cell is a plant cell, particularly a plant cell in a vegetative tissue of a plant.","A \u201ctransgenic plant\u201d refers to a plant which contains new genetic material as a result of genetic manipulation or transformation. The new genetic material may be derived from a plant of the same species as the resulting transgenic plant or from a different species.","Methods for Isolating or Producing Polynucleotides","The polynucleotide molecules of the invention can be isolated by using a variety of techniques known to those of ordinary skill in the art. By way of example, such polypeptides can be isolated through use of the polymerase chain reaction (PCR) described in Mullis et al., Eds. 1994 The Polymerase Chain Reaction, Birkhauser, incorporated herein by reference. The polypeptides of the invention can be amplified using primers, as defined herein, derived from the polynucleotide sequences of the invention.","Further methods for isolating polynucleotides of the invention include use of all, or portions of, the polypeptides having the sequence set forth herein as hybridization probes. The technique of hybridizing labelled polynucleotide probes to polynucleotides immobilized on solid supports such as nitrocellulose filters or nylon membranes, can be used to screen the genomic or cDNA libraries. Exemplary hybridization and wash conditions are: hybridization for 20 hours at 65\u00b0 C. in 5. 0\u00d7SSC, 0.5% sodium dodecyl sulfate, 1\u00d7Denhardt's solution; washing (three washes of twenty minutes each at 55\u00b0 C.) in 1. 0\u00d7SSC, 1% (w\/v) sodium dodecyl sulfate, and optionally one wash (for twenty minutes) in 0.5\u00d7SSC, 1% (w\/v) sodium dodecyl sulfate, at 60\u00b0 C. An optional further wash (for twenty minutes) can be conducted under conditions of 0.1\u00d7SSC, 1% (w\/v) sodium dodecyl sulfate, at 60\u00b0 C.","The polynucleotide fragments of the invention may be produced by techniques well-known in the art such as restriction endonuclease digestion, oligonucleotide synthesis and PCR amplification.","A partial polynucleotide sequence may be used, in methods well-known in the art to identify the corresponding full length polynucleotide sequence. Such methods include PCR-based methods, 5\u2032RACE (Frohman M A, 1993, Methods Enzymol. 218: 340-56) and hybridization-based method, computer\/database-based methods. Further, by way of example, inverse PCR permits acquisition of unknown sequences, flanking the polynucleotide sequences disclosed herein, starting with primers based on a known region (Triglia et al., 1998, Nucleic Acids Res 16, 8186, incorporated herein by reference). The method uses several restriction enzymes to generate a suitable fragment in the known region of a gene. The fragment is then circularized by intramolecular ligation and used as a PCR template. Divergent primers are designed from the known region. In order to physically assemble full-length clones, standard molecular biology approaches can be utilized (Sambrook et al., Molecular Cloning: A Laboratory Manual, 2nd Ed. Cold Spring Harbor Press, 1987).","It may be beneficial, when producing a transgenic plant from a particular species, to transform such a plant with a sequence or sequences derived from that species. The benefit may be to alleviate public concerns regarding cross-species transformation in generating transgenic organisms. Additionally when down-regulation of a gene is the desired result, it may be necessary to utilise a sequence identical (or at least highly similar) to that in the plant, for which reduced expression is desired. For these reasons among others, it is desirable to be able to identify and isolate orthologues of a particular gene in several different plant species.","Variants (including orthologues) may be identified by the methods described.","Methods for Identifying Variants","Physical Methods","Variant polypeptides may be identified using PCR-based methods (Mullis et al., Eds. 1994 The Polymerase Chain Reaction, Birkhauser). Typically, the polynucleotide sequence of a primer, useful to amplify variants of polynucleotide molecules of the invention by PCR, may be based on a sequence encoding a conserved region of the corresponding amino acid sequence.","Alternatively library screening methods, well known to those skilled in the art, may be employed (Sambrook et al., Molecular Cloning: A Laboratory Manual, 2nd Ed. Cold Spring Harbor Press, 1987). When identifying variants of the probe sequence, hybridization and\/or wash stringency will typically be reduced relatively to when exact sequence matches are sought.","Polypeptide variants may also be identified by physical methods, for example by screening expression libraries using antibodies raised against polypeptides of the invention (Sambrook et al., Molecular Cloning: A Laboratory Manual, 2nd Ed. Cold Spring Harbor Press, 1987) or by identifying polypeptides from natural sources with the aid of such antibodies.","Computer Based Methods","The variant sequences of the invention, including both polynucleotide and polypeptide variants, may also be identified by computer-based methods well-known to those skilled in the art, using public domain sequence alignment algorithms and sequence similarity search tools to search sequence databases (public domain databases include Genbank, EMBL, Swiss-Prot, PIR and others). See, e.g., Nucleic Acids Res. 29: 1-10 and 11-16, 2001 for examples of online resources. Similarity searches retrieve and align target sequences for comparison with a sequence to be analyzed (i.e., a query sequence). Sequence comparison algorithms use scoring matrices to assign an overall score to each of the alignments.","An exemplary family of programs useful for identifying variants in sequence databases is the BLAST suite of programs (version 2.2.5 [Nov. 2002]) including BLASTN, BLASTP, BLASTX, tBLASTN and tBLASTX, which are publicly available from (ftp:\/\/ftp.ncbi.nih.gov\/blast\/) or from the National Center for Biotechnology Information (NCBI), National Library of Medicine, Building 38A, Room 8N, Bethesda, Md. 20894 USA. The NCBI server also provides the facility to use the programs to screen a number of publicly available sequence databases. BLASTN compares a nucleotide query sequence against a nucleotide sequence database. BLASTP compares an amino acid query sequence against a protein sequence database. BLASTX compares a nucleotide query sequence translated in all reading frames against a protein sequence database. tBLASTN compares a protein query sequence against a nucleotide sequence database dynamically translated in all reading frames. tBLASTX compares the six-frame translations of a nucleotide query sequence against the six-frame translations of a nucleotide sequence database. The BLAST programs may be used with default parameters or the parameters may be altered as required to refine the screen.","The use of the BLAST family of algorithms, including BLASTN, BLASTP, and BLASTX, is described in the publication of Altschul et al., Nucleic Acids Res. 25: 3389-3402, 1997.","The \u201chits\u201d to one or more database sequences by a queried sequence produced by BLASTN, BLASTP, BLASTX, tBLASTN, tBLASTX, or a similar algorithm, align and identify similar portions of sequences. The hits are arranged in order of the degree of similarity and the length of sequence overlap. Hits to a database sequence generally represent an overlap over only a fraction of the sequence length of the queried sequence.","The BLASTN, BLASTP, BLASTX, tBLASTN and tBLASTX algorithms also produce \u201cExpect\u201d values for alignments. The Expect value (E) indicates the number of hits one can \u201cexpect\u201d to see by chance when searching a database of the same size containing random contiguous sequences. The Expect value is used as a significance threshold for determining whether the hit to a database indicates true similarity. For example, an E value of 0.1 assigned to a polynucleotide hit is interpreted as meaning that in a database of the size of the database screened, one might expect to see 0.1 matches over the aligned portion of the sequence with a similar score simply by chance. For sequences having an E value of 0.01 or less over aligned and matched portions, the probability of finding a match by chance in that database is 1% or less using the BLASTN, BLASTP, BLASTX, tBLASTN or tBLASTX algorithm.","Multiple sequence alignments of a group of related sequences can be carried out with CLUSTALW (Thompson, J.D., Higgins, D.G. and Gibson, T.J. (1994) CLUSTALW: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, positions-specific gap penalties and weight matrix choice. Nucleic Acids Research, 22:4673-4680, www-igbmc<dot>u-strasbg<dot>fr\/BioInfo\/ClustalW\/Top<dot>html) or T-COFFEE (Cedric Notredame, Desmond G. Higgins, Jaap Heringa, T-Coffee: A novel method for fast and accurate multiple sequence alignment, J. Mol. Biol. (2000) 302: 205-217)) or PILEUP, which uses progressive, pairwise alignments. (Feng and Doolittle, 1987, J. Mol. Evol. 25, 351).","Pattern recognition software applications are available for finding motifs or signature sequences. For example, MEME (Multiple Em for Motif Elicitation) finds motifs and signature sequences in a set of sequences, and MAST (Motif Alignment and Search Tool) uses these motifs to identify similar or the same motifs in query sequences. The MAST results are provided as a series of alignments with appropriate statistical data and a visual overview of the motifs found. MEME and MAST were developed at the University of California, San Diego.","PROSITE (Bairoch and Bucher, 1994, Nucleic Acids Res. 22, 3583; Hofmann et al., 1999, Nucleic Acids Res. 27, 215) is a method of identifying the functions of uncharacterized proteins translated from genomic or cDNA sequences. The PROSITE database (www.expasy.org\/prosite) contains biologically significant patterns and profiles and is designed so that it can be used with appropriate computational tools to assign a new sequence to a known family of proteins or to determine which known domain(s) are present in the sequence (Falquet et al., 2002, Nucleic Acids Res. 30, 235). Prosearch is a tool that can search SWISS-PROT and EMBL databases with a given sequence pattern or signature.","Methods for Isolating Polypeptides","The polypeptides of the invention, or used in the methods of the invention, including variant polypeptides, may be prepared using peptide synthesis methods well known in the art such as direct peptide synthesis using solid phase techniques (e.g. Stewart et al., 1969, in Solid-Phase Peptide Synthesis, WH Freeman Co, San Francisco Calif., or automated synthesis, for example using an Applied Biosystems 431A Peptide Synthesizer (Foster City, Calif.). Mutated forms of the polypeptides may also be produced during such syntheses.","The polypeptides and variant polypeptides of the invention, or used in the methods of the invention, may also be purified from natural sources using a variety of techniques that are well known in the art (e.g. Deutscher, 1990, Ed, Methods in Enzymology, Vol. 182, Guide to Protein Purification,).","Alternatively the polypeptides and variant polypeptides of the invention, or used in the methods of the invention, may be expressed recombinantly in suitable host cells and separated from the cells as discussed below.","Methods for Producing Constructs and Vectors","The genetic constructs of the present invention comprise one or more polynucleotide sequences of the invention and\/or polynucleotides encoding polypeptides of the invention, and may be useful for transforming, for example, bacterial, fungal, insect, mammalian or plant organisms. The genetic constructs of the invention are intended to include expression constructs as herein defined.","Methods for producing and using genetic constructs and vectors are well known in the art and are described generally in Sambrook et al., Molecular Cloning: A Laboratory Manual, 2nd Ed. Cold Spring Harbor Press, 1987; Ausubel et al., Current Protocols in Molecular Biology, Greene Publishing, 1987).","Methods for Producing Host Cells Comprising Polynucleotides, Constructs or Vectors","The invention provides a host cell which comprises a genetic construct or vector of the invention.","Host cells comprising genetic constructs, such as expression constructs, of the invention are useful in methods well known in the art (e.g. Sambrook et al., Molecular Cloning: A Laboratory Manual, 2nd Ed. Cold Spring Harbor Press, 1987; Ausubel et al., Current Protocols in Molecular Biology, Greene Publishing, 1987) for recombinant production of polypeptides of the invention. Such methods may involve the culture of host cells in an appropriate medium in conditions suitable for or conducive to expression of a polypeptide of the invention. The expressed recombinant polypeptide, which may optionally be secreted into the culture, may then be separated from the medium, host cells or culture medium by methods well known in the art (e.g. Deutscher, Ed, 1990, Methods in Enzymology, Vol 182, Guide to Protein Purification).","Methods for Producing Plant Cells and Plants Comprising Constructs and Vectors","The invention further provides plant cells which comprise a genetic construct of the invention, and plant cells modified to alter expression of a polynucleotide or polypeptide of the invention, or used in the methods of the invention. Plants comprising such cells also form an aspect of the invention.","Methods for transforming plant cells, plants and portions thereof with polypeptides are described in Draper et al., 1988, Plant Genetic Transformation and Gene Expression. A Laboratory Manual. Blackwell Sci. Pub. Oxford, p. 365; Potrykus and Spangenburg, 1995, Gene Transfer to Plants. Springer-Verlag, Berlin.; and Gelvin et al., 1993, Plant Molecular Biol. Manual. Kluwer Acad. Pub. Dordrecht. A review of transgenic plants, including transformation techniques, is provided in Galun and Breiman, 1997, Transgenic Plants. Imperial College Press, London.","Methods for Genetic Manipulation of Plants","A number of plant transformation strategies are available (e.g. Birch, 1997, Ann Rev Plant Phys Plant Mol Biol, 48, 297, Hellens R P, et al (2000) Plant Mol Biol 42: 819-32, Hellens R et al Plant Meth 1: 13). For example, strategies may be designed to increase expression of a polynucleotide\/polypeptide in a plant cell, organ and\/or at a particular developmental stage where\/when it is normally expressed or to ectopically express a polynucleotide\/polypeptide in a cell, tissue, organ and\/or at a particular developmental stage which\/when it is not normally expressed. The expressed polynucleotide\/polypeptide may be derived from the plant species to be transformed or may be derived from a different plant species.","Transformation strategies may be designed to reduce expression of a polynucleotide\/polypeptide in a plant cell, tissue, organ or at a particular developmental stage which\/when it is normally expressed. Such strategies are known as gene silencing strategies.","Genetic constructs for expression of genes in transgenic plants typically include promoters for driving the expression of one or more cloned polynucleotide, terminators and selectable marker sequences to detect presence of the genetic construct in the transformed plant.","The promoters suitable for use in the constructs of this invention are functional in a cell, tissue or organ of a monocot or dicot plant and include cell-, tissue- and organ-specific promoters, cell cycle specific promoters, temporal promoters, inducible promoters, constitutive promoters that are active in most plant tissues, and recombinant promoters. Choice of promoter will depend upon the temporal and spatial expression of the cloned polynucleotide, so desired. The promoters may be those normally associated with a transgene of interest, or promoters which are derived from genes of other plants, viruses, and plant pathogenic bacteria and fungi. Those skilled in the art will, without undue experimentation, be able to select promoters that are suitable for use in modifying and modulating plant traits using genetic constructs comprising the polynucleotide sequences of the invention. Examples of constitutive plant promoters include the CaMV 35S promoter, the nopaline synthase promoter and the octopine synthase promoter, and the Ubi 1 promoter from maize. Plant promoters which are active in specific tissues, respond to internal developmental signals or external abiotic or biotic stresses are described in the scientific literature. Exemplary promoters are described, e.g., in WO 02\/00894, which is herein incorporated by reference.","Exemplary terminators that are commonly used in plant transformation genetic construct include, e.g., the cauliflower mosaic virus (CaMV) 35S terminator, the nopaline synthase or octopine synthase terminators, the zein gene terminator, the ADP-glucose pyrophosphorylase terminator and the PI-II terminator.","Selectable markers commonly used in plant transformation include the neomycin phophotransferase II gene (NPT II) which confers kanamycin resistance, the aadA gene, which confers spectinomycin and streptomycin resistance, the phosphinothricin acetyl transferase (bar gene) for Ignite (AgrEvo) and Basta (Hoechst) resistance, and the hygromycin phosphotransferase gene (hpt) for hygromycin resistance.","Use of genetic constructs comprising reporter genes (coding sequences which express an activity that is foreign to the host, usually an enzymatic activity and\/or a visible signal (e.g., luciferase, GUS, GFP) which may be used for promoter expression analysis in plants and plant tissues are also contemplated. The reporter gene literature is reviewed in Herrera-Estrella et al., 1993, Nature 303, 209, and Schrott, 1995, In: Gene Transfer to Plants (Potrykus, T., Spangenberg. Eds) Springer Verlag. Berline, pp. 325-336.","The following are representative publications disclosing genetic transformation protocols that can be used to genetically transform the following plant species: Rice (Alam et al., 1999, Plant Cell Rep. 18, 572); apple (Yao et al., 1995, Plant Cell Reports 14, 407-412); maize (U.S. Pat. Nos. 5,177,010 and 5,981,840); wheat (Ortiz et al., 1996, Plant Cell Rep. 15, 1996, 877); tomato (U.S. Pat. No. 5,159,135); potato (Kumar et al., 1996 Plant J. 9, :821); cassaya (Li et al., 1996 Nat. Biotechnology 14, 736); lettuce (Michelmore et al., 1987, Plant Cell Rep. 6, 439); tobacco (Horsch et al., 1985, Science 227, 1229); cotton (U.S. Pat. Nos. 5,846,797 and 5,004,863); grasses (U.S. Pat. Nos. 5,187,073 and 6,020,539); peppermint (Niu et al., 1998, Plant Cell Rep. 17, 165); citrus plants (Pena et al., 1995, Plant Sci. 104, 183); caraway (Krens et al., 1997, Plant Cell Rep, 17, 39); banana (U.S. Pat. No. 5,792,935); soybean (U.S. Pat. Nos. 5,416,011; 5,569,834; 5,824,877; 5,563,04455 and 5,968,830); pineapple (U.S. Pat. No. 5,952,543); poplar (U.S. Pat. No. 4,795,855); monocots in general (U.S. Pat. Nos. 5,591,616 and 6,037,522); (U.S. Pat. Nos. 5,188,958; 5,463,174 and 5,750,871); cereals (U.S. Pat. No. 6,074,877); pear (Matsuda et al., 2005, Plant Cell Rep. 24(1):45-51); (Ramesh et al., 2006 Plant Cell Rep. 25(8):821-8; Song and Sink 2005 Plant Cell Rep. 2006; 25(2):117-23; Gonzalez Padilla et al., 2003 Plant Cell Rep. 22(1):38-45); strawberry (Oosumi et al., 2006 Planta. 223(6):1219-30; Folta et al., 2006 Planta April 14; PMID: 16614818), rose (Li et al., 2003), (Graham et al., 1995 Methods Mol. Biol. 1995; 44:129-33), tomato (Dan et al., 2006, Plant Cell Reports V25:432-441), apple (Yao et al., 199514, 407-412), Canola (L.). (Cardoza and Stewart, 2006 Methods Mol. Biol. 343:257-66), safflower (Orlikowska et al, 1995, Plant Cell Tissue and Organ Culture 40:85-91), ryegrass (Altpeter et al, 2004 Developments in Plant Breeding 11(7):255-250), rice (Christou et al, 1991 Nature Biotech. 9:957-962), maize (Wang et al 2009 In: Handbook of Maize pp. 609-639) and (Wang et al., 2006, Plant Cell Rep. 25, 5: 425-31). Transformation of other species is also contemplated by the invention. Suitable methods and protocols are available in the scientific literature.","Plants","The term \u201cplant\u201d is intended to include a whole plant, any part of a plant, a seed, a fruit, propagules and progeny of a plant.","The term \u2018propagule\u2019 means any part of a plant that may be used in reproduction or propagation, either sexual or asexual, including seeds and cuttings.","The plants of the invention may be grown and either self-ed or crossed with a different plant strain and the resulting hybrids, with the desired phenotypic characteristics, may be identified. Two or more generations may be grown to ensure that the subject phenotypic characteristics are stably maintained and inherited. Plants resulting from such standard breeding approaches also form an aspect of the present invention.","Abbreviations",{"@attributes":{"id":"p-0534","num":"0000"},"ul":{"@attributes":{"id":"ul0026","list-style":"none"},"li":["Oleosin (or Ole)0-0 means an oleosin without engineered cysteines.","Oleosin (or Ole)1-1 means an oleosin with one engineered cysteine in each hydrophilic arm.","Oleosin (or Ole)1-3 means an oleosin with one engineered cysteine in the N-terminal hydrophilic arm and three engineered cysteines in the C-terminal hydrophilic arm.","Oleosin (or Ole)3-1 means an oleosin with three engineered cysteines in the N-terminal hydrophilic arm and one engineered cysteine in the C-terminal hydrophilic arm.","Oleosin (or Ole)3-3 means an oleosin with three engineered cysteines in the N-terminal hydrophilic arm and three engineered cysteines in the C-terminal hydrophilic arm.","Oleosin (or Ole)5-6 means an oleosin with five engineered cysteines in the N-terminal hydrophilic arm and six engineered cysteines in the C-terminal hydrophilic arm.","Oleosin (or Ole)6-7 means an oleosin with six engineered cysteines in the N-terminal hydrophilic arm and seven engineered cysteines in the C-terminal hydrophilic arm."]}},"This invention will now be illustrated with reference to the following non-limiting examples.","Generating Rabbit Anti-Sesame Seed Oleosin Antibodies","Full length sesame seed oleosin containing a C-terminal His tag (nucleotide sequence is shown in SEQ ID NO: 1) was expressed in and inclusion bodies were prepared by standard techniques. The inclusion bodies were solubilised in Binding Buffer (100 mM phosphate buffer pH 8.0, 500 mM NaCl, 8M urea and 10 mM imidazole) and loaded onto a column containing equilibrated ion metal affinity chromatography (IMAC) Ni agarose (Invitrogen). Non-bound proteins were removed from the column by washing with 6 volumes of Wash Buffer (100 mM phosphate buffer pH 8.0, 500 mM NaCl, 6M urea and 50 mM imidazole). Protein was eluted in 1 vol. aliquots of Elution Buffer (100 mM phosphate buffer pH 8.0, 500 mM NaCl, 6M urea and 250 mM imidazole). Eluted fractions were analysed by SDS-PAGE\/Coomassie stain and the protein concentration measured using the Bradford's Assay. 265 \u03bcg of the IMAC-purified recombinant oleosin protein was mixed with an equal amount of Freunds Complete Adjuvant to a final volume of 0.5 mL. Following collection of the pre-bleed, the first injection was administered into multiple sites across the back of the neck and shoulder area of a rabbit. Booster shots containing 77 \u03bcg of the purified oleosin were delivered at three and seven weeks after the primary, and a test bleed of \u02dc3 mL was removed for preliminary analysis at nine weeks. Serum was preserved by the addition of 0.25% v\/v phenol and 0.01% v\/v merthiolate, and stored in 200 \u03bcL aliquots at \u221220\u00b0 C.","The sensitivity of the rabbit anti-sesame seed oleosin antibodies was evaluated by immuno-dotting which indicated that 0.25 ng of sesame seed oleosin could be regularly detected with a 1\/2,000 dilution of the antibody ().","Construct Design for Expression in ","A number of modified oleosin constructs for expression in were designed. These contained either one or three cysteine residues on the N-terminal and C-terminal hydrophilic arms. The constructs were based on the nucleotide sequence and translated polypeptide sequence from a sesame seed oleosin, GenBank clone AF091840 which contains no cysteine residues (SEQ ID NO: 16).","All clones were subcloned into pET29b using engineered NdeI\/XhoI sites. In addition, a ProTrp coding sequence was added to the coding region of the 3\u2032 end of the C-terminal hydrophilic arm to mimic the amino acid residues encoded for by the NcoI site previously engineered by Peng et al (2006) Stability enhancement of native and artificial oil bodies by genipin crosslink. Taiwan Patent I 250466.","Oleosin-cysteine proteins mutated to include cysteine residues in both the N- and C-terminal hydrophilic regions described here are designated Ole-1-1, Ole-1-3, Ole-3-1, and Ole-3-3 (SEQ ID NO 2, 3, 4, and 5 respectively), where the first and the second numeral digits correspond to the number of disulfide bonds in the N- and C-terminus, respectively. The standard oleosin without the cysteine residues was used as a control and was designated as Ole-0-0 (SEQ ID NO 1).","The cysteines were substituted for charged residues predicted to be on the surface of the oil bodies and are listed below.\n\n","The constructs were designed so could be relatively simply sub cloned from the GENEART provided backbone (pCR4Blunt-TOPO) into pET29b (Novogen) via NcoI\/XhoI digestion and ligation. This placed the oleosin coding sequence downstream of the pET29 N-terminal S\u2022tag fusion and upstream of the C-terminal His tag ( and SEQ ID Nos 1-10). The oleosin and modified oleosin sequences used are summarised in the Summary of Sequences table.","Expression in and Purification of Modified Oleosins Containing at Least One Artificially Introduced Cysteine","Expression of the recombinant sesame seed oleosins (with and without engineered cysteines) in the expression system was evaluated by SDS-PAGE\/Coomassie brilliant blue staining and SDS-PAGE\/immunoblot analysis using antibodies raised against the sesame seed oleosin (described in Example 1).","Expression of recombinant modified oleosin was induced in a freshly inoculated 10 mL culture of (BL21 Rosetta-Gami) containing an oleosin (with or without engineered cysteine residues) coding sequence in the pET29 expression vector. The culture was grown at 37\u00b0 C., 220 rpm, until mid log phase (OD0.5-0.7); expression was induced by the addition of IPTG to 1 mM final concentration. The induced culture was incubated at 37\u00b0 C., 220 rpm, for a further 2-3 h. Given the properties of modified oleosin the applicants did not attempt to express it in a soluble form but rather chose to extract it from inclusion bodies. Aliquots (1 mL) of the culture were transferred to 1.5 mL microfuge tubes and the cells pelleted by centrifugation (2655\u00d7g for 5 min at 4\u00b0 C.).","Pelleted cells were resuspended in BugBuster\u00ae Reagent (Merck) at 5 mL\/g of wet cell pellet, with the addition of DNase to 40 \u03bcg\/mL and mixed gently on a rotator for 30 min followed by centrifugation at 8000 g for 10 min at 4\u00b0 C. The resultant cell pellet was retreated with BugBuster\u00ae and DNase as above. The remaining soluble protein and suspended cell debris was separated from the insoluble inclusion bodies by centrifugation at 8000 g for 10 min at 4\u00b0 C.","Recombinant oleosins were further purified from the inclusion bodies using a procedure adapted from D'Andr\u00e9a et al. (2007). Briefly: the inclusion body preparation was washed by re-suspension in 200 mM sodium carbonate buffer pH 11 (5 mL per gram of original cell pellet) and re-pelleted by centrifugation at 8000\u00d7g for 10 min at 4\u00b0 C. The washed inclusion body pellet was again re-suspended in 5 mL 200 mM sodium carbonate buffer per gram of pellet and added to 9 volumes of freshly prepared chloroform:methanol mix (5:4 v\/v) giving a final ratio of 5:4:1 (chloroform:methanol:buffer). The suspension was gently mixed for 5 min which formed a milky, single phase mixture; this was centrifuged at 10,000\u00d7g for 10 min at 4\u00b0 C., and the supernatant containing the modified oleosin was carefully separated from the pellet and transferred into a new tube. Aliquots of the supernatant were dried down under a stream of nitrogen and the protein re-solubilised in 8M urea and quantified by Qubit\u2122 (Invitrogen).","A dot-blot was used to compare the ability of the anti-sesame seed oleosin antibodies (Abs) described in Example 1 to bind to oleosin without cysteines versus the oleosins containing cysteines (described in Example 2). Dilution series from 12 to 0.25 ng of purified Ole-0-0, Ole-1-3 and Ole-3-1 were spotted onto a pre-equilibrated Hybond-P PVDF Transfer membrane. This was incubated with the anti-sesame seed oleosin antibodies at 1:2000 as the primary Ab. The blot was then incubated with the appropriate secondary Ab and developed by chemiluminescence (). The results indicate that on an immunoblot, the anti-sesame seed oleosin antibodies are up to an order of magnitude more sensitive to the oleosin without cysteine residues than the oleosins with cysteine residues. As a consequence of the different sensitivities it was necessary to load different quantities of recombinant protein onto the gels for analysis by immunoblotting. Despite the non uniform lane loading it is still possible to compare different oleosins between lanes in terms of their relative distribution between monomeric and oligomeric forms.","Preparation of Artificial Oil Bodies","Artificial oil bodies (AOBs) were then prepared by drying down aliquots of the supernatant described in Example 3, calculated to contain either 150 \u03bcg or 1 mg of recombinant oleosin.","The process of generating AOBs involved combining PL, TAG, and the recombinant oleosin\/modified oleosin. In the absence of strong chaotropic agents the disruptive force required to dissociate individual recombinant oleosins from the purified fraction involved several alternating cycles of sonicating and cooling. This was achieved by solubilising the 150 \u03bcg and 1 mg oleosin\/modified oleosin samples in 20 \u03bcL chloroform containing 150 \u03bcg PL (Sigma, Cat#P3644) and mixed with 60 \u03bcL of purified sesame seed oil (Tzen and Huang 1992) and 940 \u03bcL of AOB buffer (50 mM sodium phosphate buffer pH 8.0, 100 mM NaCl). The complete mixture was then sonicated three times for 30 sec (Sonics & Materials Vibra\u02dcCell VC600, 600 W, 20 kHz; \u215b\u2033 tapered micro-tip probe, power setting #3).","The applicants also found that the purification procedure could be successfully scaled up and when a 50 g cell pellet was used as the starting material it was necessary to substitute the stream of nitrogen with a rotary vacuum evaporator to remove the chloroform and the majority of the methanol. At this point the majority of oleosin\/modified oleosin precipitated out of the azeotropic solvent and was separated by centrifugation at 12,000 g for 10 min.","Inclusion bodies were suspended in 1 mL AOB Buffer II (50 mM sodium phosphate, pH 8.0, 100 mM NaCl, 20 mM \u03b2-mercaptoethanol, 10 mM DTT and 5% [v\/v] sesame oil) and then sonicated 4\u00d7. AOBs were concentrated by centrifugation at 12,000 rpm for 10 min, this resulted in the formation of a suspension of AOBs overlaying the aqueous fraction. The underlying aqueous fraction was removed by pipette, and the remaining AOBs were washed (to remove soluble proteins and reducing agents) by gentle agitation in 1 mL AOB Buffer III (50 mM sodium phosphate, pH 8.0, 100 mM NaCl). After washing, the AOBs were re-concentrated by centrifugation, and the underlying aqueous fraction removed, then re-suspended by vortexing in AOB Buffer IV (50 mM sodium phosphate buffer, pH 8.0, 100 mM NaCl, 1 mM GSSG) and the AOBs stored at 4\u00b0 C. for further analyses.","Recombinant Ole-0-0, and all variations of the oleosin-cysteines were successfully expressed and located in inclusion bodies (). Ole-0-0 was predominantly present as a monomer (in both inclusion bodies as well as AOBs); this migrated fractionally faster than the 20 kDa molecular weight marker (in reducing and non reducing SDS and SDS-UREA PAGE). Also present were two slower migrating immunoreactive bands of approximately 35 and 36 kDa which likely correspond to two forms of dimeric oleosin. While Ole-0-0 is not predicted to contain any cysteine residues the overall intensity and ratio of the two apparent dimers was influenced by the presence of reducing agents (\u03b2-ME @ 5% of the sample loading buffer and 10 mM DTT).","In the inclusion bodies, the predominant form of Ole-1-1 is monomeric. Only one dimeric form appeared to be present and this was not influenced by reducing agents or urea. Ole-1-1 from AOBs (generated in the presence of reducing agent and then in the presence of oxidising agent) showed a large increase in the ratio of dimer to monomer as well as the formation of trimeric, tetrameric and likely pentameric oligomers (the electrophoretic focus of these oligomers was considerably improved in the SDS-UREA gel). Removal of the GSSG and re-introduction of reducing agents to the AOBs resulted in the presence of only monomer and dimer in similar proportions seen in the inclusion bodies. AOBs generated with Ole-1-1 (in the absence of both reducing agents and GSSG) showed the presence of almost equal portions of monomer and dimer and a small amount of trimer, indicating that the conditions under which the AOBs are formed have some reducing potential. The subsequent addition of GSSG resulted in an increase in the oligomeric portions as well as the appearance of a tetrameric form.","While the monomer was the predominant form of Ole-3-3 in the inclusion bodies, a comparatively high percentage was also present in multiple oligomeric forms. The proportion of oligomers declined to a small extent with the addition of reducing agent and slightly more by the addition of both reducing and chaotropic agents. Oligomeric forms of Ole-3-3 that were larger than a trimer were poorly resolved when the recombinant protein was extracted from AOBs. The creation of large oligomeric forms was promoted by the addition of GSSG and in the absence of reducing and chaotropic agents a portion of these oligomeric forms failed to enter the stacking gel. Combined, these results indicate that on the AOBs, Ole-3-3 was highly cross-linked and the position of the cross-links was more variable compared to the Ole-3-3 recovered from the inclusion bodies. This suggests that, despite considerable pre-existing cross-linking (within the inclusion bodies), on the AOB Ole-3-3 has access to a high number of potential partners for cross-linking. Similarly for Ole-1-3 and Ole-3-1, the number of cross-linked species increased when there was more than one cysteine on one or both hydrophilic regions ().","It could be anticipated that in non-reducing SDS-PAGE, oligomers containing the same number of oleosins but with the disulfide bonds in different positions would migrate differently to each other. Indeed this can be seen in  where the data indicates that the position of the oleosin arms, relative to one another are at different positions over the oil body. For example the Ole-1-1 can only form one disulfide bond per arm and this has to form at the same position, where as the presence of three cysteines enables more than one disulfide bond to form but it also allows the disulfide bonds to form with different degrees of hydrophilic arm overlap as well as having multiple oleosins bound to the same arm ().","The addition of SDS and reducing agents (DTT and (3-ME) decreased the number of oligomeric complexes (). The addition of SDS and urea results in a similar pattern to SDS alone except that the previously resolved multiple dimeric forms migrated as one and the trimeric and tetrameric forms appear to be in higher abundance presumably because they are also migrating as single bands which increases intensity correspondingly (). In contrast, the presence of SDS, reducing agent and urea resulted in fewer oligomeric forms of Ole-1-1 and Ole-1-3 but not Ole-3-1 or Ole-3-3 (). In the case of Ole-3-1 and Ole-3-3 it appears that the urea does not completely denature the disulfides oleosins and may indeed prevent the complete reduction of the disulfide bonds. It could be that these bonds are formed during the generation of inclusion bodies (would need to see reduced and non reduced inclusion body preps). Furthermore, the presence of the dimeric oleosin formed in the absence of engineered cysteine residues () indicates that some oligomerisation is due to other types of attraction, e.g, strong hydrophobic bonding that is not fully disrupted by SDS but can be almost completely disrupted by the combination of both SDS and urea ().","The effect of increasing the number of potential cross-linking sites in an oleosin peptide on AOB integrity and emulsion stability can be assessed as follows.","Quantitative Determination of AOB Integrity","Assessment of AOB stability and integrity using either absorbance (OD), direct counting of AOBs using a hemocytometer, or visual evaluation of coalescence by microscopy proved to be highly variable and amongst other things was influenced by the: degree of pre-sampling agitation; quantity of sample removed; time left under the microscope. To avoid this the applicants devised a simple method to quantify the amount of TAG released from the AOBs into the surrounding media during a variety of treatments as a means of comparing integrity. Essentially equal quantities (based on FAMES-GC\/MS estimation of TAG and Bradfords determination of protein) of AOB preparations are made up to a total volume of 200 \u03bcL using AOB buffer (containing Proteinase K [PNK] when appropriate at a 1:1 ratio of PNK:total proteins in OB or AOB samples in a 250 \u03bcL GC glass insert tubes and covered with a plastic cap. Following the treatment (elevated temperature or exposure to PNK) 15 \u03bcL of fish oil (Vitamax\u00ae, Australia) is added to the sample and mixed by vortexing followed by centrifugation at 5,200 g for 1 min. The addition of fish oil followed by vortexing enables any TAG that had leaked from the AOBs to mix with the added fish oil and be floated by brief centrifugation. 4 \u03bcL of the oil phase is sampled and subjected to fatty acid methyl esterification (FAME) and then analysed by GC-MS (Shimadzu model numbers, fitted with a 50mQC2\/BPX70-0.25 GC capillary column (SGE) as described by Browse et al. (1986). In the absence of added fish oil the quantity of TAG that had leaked from the AOBs was too small to form a samplable visible layer even after centrifugation, in such a case the maximum volume would have been 6 \u03bcL. The very different lipid profiles of fish oil and sesame oil enabled us to easily distinguish the leaked TAG from the added TAG.","Using the internal C15:0 and C17:0 standards the applicants can calculate the absolute amounts of C18:2 (the major lipid in sesame seed oil) recovered after treatment.","Determination of AOB Integrity and Emulsion Stability at Elevated Temperature","Oil in water emulsions are less stable at elevated temperatures; hence, it is of interest to investigate if modified oleosins with varying numbers in introduced cysteines influence AOB integrity at elevated temperature. To achieve this the applicants determine the integrity (using the method described above) of OBs and AOBs (containing different oleosins) in a phosphate buffer (50 mM Na-phosphate buffer pH8, 100 mM NaCl) at 95\u00b0 C. AOBs are heated for 2 h. Integrity is determined as above.","The effect of higher ratios of crosslinked oleosin:TAG on the stability of AOBs in rumen fluid can be assessed as follows.","Determination of AOB Integrity in Rumen Fluid","One of the aims of disulfide was to provide some degree of protection from biohydrogenation by rumen microflora. Assessment of AOB stability with rumen fluid can be assessed as follows. AOBs are added to an equal volume (25 \u03bcL) of rumen fluid. Samples are incubated at 39\u00b0 C. for 0, 15, 30, 60, 120 and 240 min, at the end of the incubation an equal volume of loading buffer (Invitrogen) is added, mixed and heated at 70\u00b0 C. for 10 min. 15 \u03bcL of each sample\/loading buffer mix is compared by SDS-PAGE\/immunoblot. Integrity is determined as above.","Analysis of AOB Integrity in Proteinase K","To investigate the influence of modified oleosin in a controllable and repeatable highly degradative environment integrity is determined (using the method described above) of AOBs (containing different modified oleosins) after incubation in an phosphate buffer (50 mM Na-phosphate buffer pH8, 100 mM NaCl) containing 1:1 (g\/g protein) Proteinase K (Invitrogen) at 37\u00b0 C. for 4 h. While the maximum activity of Proteinase K is achieved below 65\u00b0 C. the lower temperature is used in order to reduce the influence of temperature on AOB instability. Integrity is determined as above.","Construct Design for Expression in Planta","The applicants synthesised individual coding sequences for the sesame seed oleosin (based on GenBank clone AF091840) with different numbers of cysteines in the N- and C-terminal arms. The coding sequence was flanked by a 5\u2032 NotI site and a 3\u2032 NdeI site. A separate acceptor cassette was synthesised containing an attL1 site, a NotI site and NdeI site followed by a nos termination sequence, a forward facing CaMV35s promoter, the DGAT1 (S205A) (SEQ ID NOs 11-20 and ) plus its own UBQ10 intron, an attL2 site. The sesame seed oleosins with different numbers of cysteines were individually transferred to the acceptor cassette via the NotI and NdeI sites. Each of these completed cassettes were then transferred to a plant binary vector pRSh1,  (Winichayakul et al., 2008) via the LR recombination reaction. This placed the oleosin downstream of a CaMV35S promoter (already contained within pRSh1) and placed a nos terminator (already contained within pRSh1) downstream of the DGAT1 (S205A) (). The nucleotide sequences encoding the sesame seed oleosins (with cysteines) and DGAT1 were optimised for expression in , this included optimisation of codon frequency, GC content, removal of cryptic splice sites, removal of mRNA instability sequences, removal of potential polyadenylation recognition sites, and addition of tetranucleotide stop codon (Brown et al, 1990; Beelman and Parker, 1995; Rose, 2004; Rose and Beliakoff, 2000; Norris et al., 1993).","It should be noted that the oleosin sequence used is for example only. Any oleosin or steroleosin or caoleosin sequences could be engineered to contain cross-linking regions. The coding sequences of the complete ORFs (after splicing) were checked against repeat of the original oleosin translated sequence and found to be identical over the oleosin coding regions.","Transformation of with Sesame Seed Oleosins Containing Cysteines","Transformation of var (with constructs described above), analyses of T2 seeds for modified oleosin, immunoblot analysis of oil bodies containing sesame seed oleosin with different numbers of cysteines was performed as described previously (Scott et al., 2007).","Both the floral-dip (Clough, 1998) and floral-drop methods (Martinez-Trujillo, 2004) were used in the transformation of by GV3101 containing the binary constructs. T1 seed was collected from the treated plants, germinated and selected by spraying at 14 d and 21 d post-germination with Basta\u00ae. Basta\u00ae resistant T1 plants (71, 62 and 23 transformants containing the single sesame seed oleosin, and modified oldeosin constructs respectively) were transplanted, allowed to self-fertilise, set seed and the T2 seed was collected. Equal quantities of seed extract from Basta\u00ae resistant plants were analysed by SDS-PAGE\/immunoblot with the anti-sesame seed oleosin antibodies; recombinant sesame seed oleosin and modified oldeosin of the appropriate size was observed in the majority of samples (). Southern blot analysis was performed on selected T2 lines to determine the number of insertion sites.","Crude Oil Body Preparations from Seeds","Crude OB preparations were prepared, from seed of plants produced as described in Example 5, by either grinding 200 mg seed with a mortar and pestle containing a spatula tip of sand and 750 \u03bcL Extraction Buffer (10 mM phosphate buffer, pH 7.5 containing 600 mM sucrose) or by homogenising 25 mg of seed in 300 \u03bcL Extraction Buffer using a Wiggenhauser D-130 Homogenizer. A further 750 \u03bcL Extraction Buffer was added and the slurry in the mortar and transferred to a 2 mL microfuge tube whereas the homogenizer tip was rinsed in 1 mL Extraction Buffer and this volume was added to the homogenised seed. Samples were then centrifuged at 20,000\u00d7g for 5 min; this left a pellet and aqueous supernatant which was overlaid by an immiscible oily layer containing both intact and disrupted oil bodies as well as free TAG. The upper oil layer was gently pushed to the side of the tube, and the aqueous layer and pelleted material discarded. The oil layer was then re-suspended from the side of the tube by vortexing in Extraction Buffer and placed in a fresh 2 mL microfuge tube. The final volume was made up to 0.5 mL with Extraction Buffer.","Purified Oil Body Preparations from Seeds and Cross Linking Cysteine Residues Between the Engineered Oleosins","25 mg of seed (of plants transformed as described in Example 5) was ground in 300 \u03bcl extraction buffer (10 mM Phosphate buffer, pH 7.5 containing 600 mM sucrose) using a Wiggenhauser D-130 Homogenizer. Seed was ground until crushed and the sample appeared \u201ccreamy\u201d and frothy as starch was released from the seeds. The homogenizer tip was rinsed in 1 ml buffer and this volume was added to the crushed seed. Samples were prepared up to this point in lots of 4, then centrifuged 14,000 rpm for 5 mins. A thin gel loading tip was used to gently push the oil layer to the side of the tube, and the aqueous layer removed to a fresh tube. The oil layer was resuspended from the side of the tube using extraction buffer and placed in a fresh 2 ml tube. The final volume was made up to 0.5 ml (as read on the side of the tube) with extraction buffer, samples were divided into two and oxidising agent (3 mM GSSG) was added to one tube and incubated at room temperature for 10 min. Oil body preparations were then added to an equal volume of 2\u00d7 gel loading buffer and boiled for 5 min before loading on to a gel.","Samples were run either on pre-cast NuPAGE Novex 4-12% Bis-Tris Midi Gels(Invitrogen) on a Criterion gel rig system (Bio-Rad), or NuPAGE\u00ae Novex 12% Bis-Tris gradient Gel 1.0 mm, 15 well, cat# NP0343BOX, with NuPAGE\u00ae MES SDS Running Buffer (for Bis-Tris Gels only) (20\u00d7), cat# NP0002-02, or on hand-cast Tris-HCl gels. Gels were stained by SafeStain (Invitrogen) to show total protein loaded or blotted onto Nitrocellulose membrane using the iBlot system (Invitrogen). In each case, the negative control was a sample extracted from wild type Columbia seed and the positive control was the same extraction method (although grinding was by mortar and pestle) performed on wild type sesame seed. 10 \u03bcl of each sample and the negative control were loaded onto the gel, and 5 \u03bcl was used for the positive control.","Following blotting, the membrane was blocked in a solution of 12.5% skim milk powder in TBST (50 mM Tris pH 7.4, 100 mM NaCl, 0.2% Tween) for at least 1.5 hours. The membrane was then washed in TBST 3\u00d75 mins before incubating with primary antibody (anti-sesame) at 1\/1000 in TBST for 1 hour at room temperature. Following 3 further TBST washes, incubation with secondary antibody (anti-rabbit) at 1\/5000 was carried out for 1 hour at room temperature. The membrane underwent 3 further washes then the signal was developed using standard chemiluminesence protocol.",{"@attributes":{"id":"p-0572","num":"0638"},"figref":["FIG. 11","FIG. 11"],"i":"Arabidopsis thaliana "},"The effect of increasing the number of potential cross-linking sites in an oleosin peptide on in planta OB integrity and emulsion stability can be assessed as follows.","Quantitative Determination of OB Integrity","Assessment of OB stability and integrity using either absorbance (OD), direct counting of AOBs using a hemocytometer, or visual evaluation of coalescence by microscopy proved to be highly variable and amongst other things was influenced by the: degree of pre-sampling agitation; quantity of sample removed; time left under the microscope. To avoid this the applicants devised a simple method to quantify the amount of TAG released from the OBs into the surrounding media during a variety of treatments as a means of comparing integrity. Essentially equal quantities (based on FAMES-GC\/MS estimation of TAG and Bradfords determination of protein) of OB preparations are made up to a total volume of 200 \u03bcL using AOB buffer (containing Proteinase K [PNK] when appropriate at a 1:1 ratio of PNK:total proteins in OB samples in a 250 \u03bcL GC glass insert tubes and covered with a plastic cap. Following the treatment (elevated temperature or exposure to PNK) 15 \u03bcL of fish oil (Vitamax\u00ae, Australia) is added to the sample and mixed by vortexing followed by centrifugation at 5,200 g for 1 min. The addition of fish oil followed by vortexing enables any TAG that had leaked from the OBs to mix with the added fish oil and be floated by brief centrifugation. 4 \u03bcL of the oil phase is sampled and subjected to fatty acid methyl esterification (FAME) and then analysed by GC-MS (Shimadzu model numbers, fitted with a 50mQC2\/BPX70-0.25 GC capillary column (SGE) as described by Browse et al. (1986). In the absence of added fish oil the quantity of TAG that had leaked from the OBs was too small to form a samplable visible layer even after centrifugation, in such a case the maximum volume would have been 6 \u03bcL. The very different lipid profiles of fish oil and sesame oil enabled us to easily distinguish the leaked TAG from the added TAG.","Using the internal C15:0 and C17:0 standards the applicants can calculate the absolute amounts of C18:2 (the major lipid in sesame seed oil) recovered after treatment.","Determination of OB Integrity and Emulsion Stability at Elevated Temperature","Oil in water emulsions are less stable at elevated temperatures; hence, it is of interest to investigate if modified oleosins with varying numbers in introduced cysteines influence OB and AOB integrity at elevated temperature. To achieve this the applicants determine the integrity (using the method described above) of OBs (containing different oleosins) in an phosphate buffer (50 mM Na-phosphate buffer pH8, 100 mM NaCl) at 95\u00b0 C. AOBs are heated for 2 h. Integrity is determined as above.","The effect of higher ratios of crosslinked oleosin:TAG increase the stability of OBs in rumen fluid can be assessed as follows:","Determination of OB Integrity in Rumen Fluid","One of the aims of disulfide was to provide some degree of protection from biohydrogenation by rumen microflora. Assessment of OB stability with rumen fluid can be assessed as follows. OBs are added to an equal volume (25 \u03bcL) of rumen fluid. Samples are incubated at 39\u00b0 C. for 0, 15, 30, 60, 120 and 240 min, at the end of the incubation an equal volume of loading buffer (Invitrogen) is added, mixed and heated at 70\u00b0 C. for 10 min. 15 \u03bcL of each sample\/loading buffer mix is compared by SDS-PAGE\/immunoblot. Integrity is determined as above.","Analysis of OB Integrity in Proteinase K","To investigate the influence of modified oleosin in a controllable and repeatable highly degradative environment integrity is determined (using the method described above) of AOBs (containing different modified oleosins) after incubation in an phosphate buffer (50 mM Na-phosphate buffer pH8, 100 mM NaCl) containing 1:1 (g\/g protein) Proteinase K (Invitrogen) at 37\u00b0 C. for 4 h. While the maximum activity of Proteinase K is achieved below 65\u00b0 C. the lower temperature is used in order to reduce the influence of temperature on OB instability. Integrity is determined as above.","In order to produce oil bodies in vegetative tissue, it is necessary to produce triacyclglycerol in such tissue (e.g. leaves).","Production of Triacylglycerol in the Vegetative Portions of the Plant","In most plants (including ) the majority of leaf lipids are attached to a glycerol backbone and exist as diacylglycerols. These are incorporated into lipid bi-layers where they function as membranes of multiple sub-cellular organelles or the as the membrane of the cell itself. The majority of lipid bilayer in the leaf is the chloroplast thylakoid membrane. A smaller amount of leaf lipid exists as epicuticular waxes and an even smaller percentage is present in the form of triacylglycerol (TAG).","Most plants synthesise and store TAG in developing embryos and pollen cells where it is subsequently utilised to provide catabolizable energy during germination and pollen tube growth. Dicotyledonous plants can accumulate up to approximately 60% of their seed weight as TAG. Ordinarily, this level is considerably lower in the monocotyledonous seeds where the main form of energy storage is carbohydrates (e.g., starch) The only committed step in TAG biosynthesis is the last one, i.e., the addition of a third fatty acid to an existing diacylglycerol, thus generating TAG. In plants this step is performed by one of three enzymes including: acyl CoA:diacylglycerol acyltransferase (DGAT1), an unrelated acyl CoA:diacylglycerol acyl transferase (DGAT2), and phospholipid:diacylglycerol acyltransferase (Zou et al., 1999; Bouvier-Nav\u00e9 et al., 2000; Dahlqvist et al., 2000; Lardizabal et al., 2001). Over expression of the transcribed region of any of these genes in the vegetative portions of plants leads to the formation of TAG droplets in the cytoplasm of leaf cells, as demonstrated by the over expression of: DGAT1 in tobacco by Bouvier-Nav\u00e9 et al., (2000); Tung tree DGAT2 in yeast and tobacco by Shockey et al., (2006); PDAT in by Stahl et al., (2004). Over expression of DGAT1 in some cases was demonstrated to increase the total lipid level but not necessarily by the accumulation of TAG, e.g. in hairy roots (Bryan et al., 2004) and in leaves (Cookson et al., 2009).","To demonstrate the accumulation of TAG in the leaves of these plants you can compare the total quantity of lipid extract from leaves of these plants with those of untransformed plants or plants transformed with the empty binary vector. Ensuring the plants are grown under the same environmental conditions and that the leaves sampled are physiologically equivalent. With the appropriate internal standards the quantification of the total lipid extract can be achieved using FAMES GC-MS analysis (as described by Winichayakul et al, 2008 Delivery of grasses with high levels of unsaturated, protected fatty acids. Proceedings of the New Zealand Grassland Association, 70:211-216.). Alternatively, the total lipids can be extracted using the Folsch method (Folsch et al., 1957J. Folsch, M. Lees and G. A. Slone-Stanley, A simple method for the determination of total lipid extraction and purification, Journal of Biological Chemistry 226 (1957), pp. 497-507.) and quantified using appropriate internal standards with a GC-MS fitted with a Restek (Restek Corp., Bellefonte, Pa.) RTX65TG column.","Leaves were sampled from plants over expressing the DGAT1 (S205A) and the sesame seed oleosin construct (either Oleo0-0, or Oleo1-1, or Oleo1-3, or Oleo3-1, or Oleo3-3, SEQ ID NOs 11-20, ) and analysed by SDS-PAGE\/immunoblot using the polyclonal anti-sesame seed oleosin antisera. It can be seen that recombinant oleosin was found to accumulate in the leaves of leaves ().","The simultaneous expression and accumulation of oleosin\/modified oleosin protein in the same cell (for example leaf cell) will result in the production of triglyceride oil bodies encapsulated by a phospholipid monolayer embedded with oleosin; this has been demonstrated with un-modified oleosin in yeast (Ting et al., 1997) and seeds (Abell et al., 2004).","Oil Body Preparations from the Leaves of Transgenic ","Oil bodies can be extracted from the leaves of transgenic expressing DGAT1 (S205A) and the sesame seed oleosin construct (either Oleo0-0, or Oleo1-1, or Oleo1-3, or Oleo3-1, or Oleo3-3, SEQ ID NOs 11-20, ).","The effect of increasing the number of potential cross-linking sites in an oleosin peptide on the OBs of such plants can be assessed by measuring OB integrity and emulsion stability can as described in Example 6.","Design and Construction of Oleosins Containing More than Three Cysteine Residues in Each Hydrophilic Arms","The ole-3,3 lines had substantial levels of elevated lipid levels in the form of TAGs when co-expressed with DGAT1 (S205A) while the lines containing ole-0,0 did not have elevated lipid levels above the DGAT1 over expressing control. The ole-1,1, ole-1,3 and ole-3,1 showed there was a correlation between the level of lipid accumulation in the leaf and the increase in the number of cysteines engineered into each arm (Table 3).",{"@attributes":{"id":"p-0589","num":"0655"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"371pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Fatty acid composition (as % Dry Weight) of leaves expressing"},{"entry":"either vector control, DGAT1 (S205A) alone, or DGAT1 (S205A) and different forms of oleosin"},{"entry":"(containing either no additional cysteines or up to 3 additional cysteines in each hydrophilic arm)."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"8"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"DGAT1",{},{},{},{},{}]},{"entry":["Fatty",{},"ALONE","DGAT1 +","DGAT1 +","DGAT1 +","DGAT1 +","DGAT1 +"]},{"entry":["acid","Vector","DGAT1SA","OLE 0-0","OLE 1-1","OLE 1-3","OLE 3-1","OLE 3-3"]},{"entry":["profile","control","#2","(#11)","(#9)","(#5)","(#18)","(#47)"]},{"entry":{"@attributes":{"namest":"1","nameend":"8","align":"center","rowsep":"1"}}},{"entry":["C16:0","0.55 \u00b1 0.035","0.55 \u00b1 0.001","0.54 \u00b1 0.014","0.57 \u00b1 0.001","\u20020.68 \u00b1 0.042*","0.62 \u00b1 0.084","0.95 \u00b1 0.049*"]},{"entry":["C16:1","0.085 \u00b1 0.007\u2002","0.105 \u00b1 0.007\u2002","0.11 \u00b1 0.001","0.13 \u00b1 0.014","\u20020.1 \u00b1 0.001","0.135 \u00b1 0.021\u2002","0.11 \u00b1 0.001\u2002"]},{"entry":["C16:3","0.34 \u00b1 0.021","0.41 \u00b1 0.028","0.42 \u00b1 0.007","0.48 \u00b1 0.028","0.51 \u00b1 0.035","\u20020.55 \u00b1 0.071*","0.62 \u00b1 0.049*"]},{"entry":["C18:1","0.095 \u00b1 0.007\u2002","0.075 \u00b1 0.007\u2002","\u20020.1 \u00b1 0.001","0.185 \u00b1 0.007*","0.345 \u00b1 0.007*","\u20030.2 \u00b1 0.014*","0.61 \u00b1 0.014*"]},{"entry":["C18:2","0.55 \u00b1 0.014","0.46 \u00b1 0.035","0.56 \u00b1 0.014","\u20020.77 \u00b1 0.049*","\u20020.97 \u00b1 0.007*","\u20020.79 \u00b1 0.113*","1.82 \u00b1 0.113*"]},{"entry":["C18:3","1.67 \u00b1 0.056","1.91 \u00b1 0.028","1.78 \u00b1 0.014","1.68 \u00b1 0.028","1.74 \u00b1 0.014","1.9 \u00b1 0.28","2.29 \u00b1 0.056*"]},{"entry":["C20:0","Not","Not","Not","Not","Not","Not","0.054 \u00b1 0.003\u2003"]},{"entry":[{},"detected","detected","detected","detected","detected","detected"]},{"entry":{"@attributes":{"namest":"1","nameend":"8","align":"center","rowsep":"1"}}}]}}]}}},"The correlation between the increase in total lipid (shown to be TAG) and the number of cysteines engineered into the hydrophilic domains indicated that the number of cysteines may be a way to influence the level of TAG desired. Consequently new constructs containing more than 3 cysteines per hydrophilic arm were designed. While it is not possible to put an infinite number of cysteines\/hydrophilic arm; the limitations include:\n\n","The original cysteine-oleosin was engineered to carry 3 relatively evenly spaced unpaired cysteines in each arm by replacing amino acids and predominantly those that could be predicted to be neutral or charged but not hydrophobic.","The oleosin presumably needs to have a certain level of negative charge and in the C-terminus this appears to be achieved by K (Lys), hence continuing the strategy of swapping charged or neutral residues with additional cysteines may result in poor stability in terms of preventing coalescence. Furthermore, in the N-terminal hydrophilic region there appears to be too few residues left between the engineered cysteines to enable further substitution of residues whilst maintaining the spacing and oscillation between positive and negatively charged amino acids. Hence, for both N- and C-termini added additional residues (cysteines) rather than substitute existing residues with cysteines. Alternatively, an oleosin with longer hydrophilic arms could have been used.","Two additional constructs (Ole-5, 6 and Ole-6,7) were also designed. These are not purposely unbalanced in terms of cysteine residues per arm but organised to attempt to give typically 4-5 residues between each cysteine. In fact to increase the cysteines to 6 in the N-terminal arm it was necessary to generate additional residues (as opposed to substitution of existing residues); this as achieved by replicate the first 6 residues from the Ole-3,3.","Rather than have completely new nucleotide sequences designed the triplet TGT to code for cysteine was added (where appropriate) to generate Ole5,6. For additional glutamine residues the codon triplet GGA was used. For the additional N-terminal 6 residues on Ole6-7 the N-terminus of Ole3,3 was replicated and fused in frame.","Sublconing strategy was designed to be identical to initial cysteine oleosins, i.e., subcloned into oleoacceptor by NotI\/NdeI. This is then recombined by LR reaction into pRSH1 (Winichayakul et al., 2008). Essentially places both DGAT1 (S205A) and oleosin under their own CaMV35s promoters and OCS terminators. Both DGA1 and oleosin clones contain a UBQ10 intron.","NetGene2 was used to predict the splicing pattern of Ole5,6 and Ole6,7. Both were predicted to have only one donor and acceptor site on the direct strand (both were predicted to have a very high probability of recognition) and no sites on the complementary strand.","The data indicates that the oleosins containing 1,3 or 3,1 cysteines do accumulate detectable levels of TAG but this is certainly less than the 3,3 cysteine oleosins (the 1,1 accumulated trace amounts while the 0, 0 did not). This suggests even more strongly that the 5,6 and 6,7 oleosins are likely to accumulate even more TAG than the 3,3 construct. The first data from the 5,6 and 6,7 constructs will be available soon.","Transformation of Oleosins Containing Engineered Cysteines and DGAT1 into Wild Type ","Five disulfide-oleosin\/DGAT1 (S205A) gene constructs and one control (construct containing DGAT1 (S205A) but not oleosin) were been transferred to the plant binary vector pRSh1 (Winichayakul et al., 2008) and transformed into wild type using \u2014mediated transformation.","A modification of the traditional floral dip method was followed since it has been reported that floral dipping tends to damage developing siliques due to the presence of detergent in the inoculums (Martinez-Trujillo et al., 2004). Therefore, a drop by drop inoculation to every flower was carried out using a micropipette. The inoculation was repeated after one week to introduce the inoculum to the newly developed flowers. Seeds were collected when the siliques have dried up, then cleaned and planted for screening of transformants.","Screening for transformants was performed by BASTA selection and homozygous transformants were selected using segregation ratio analysis for BASTA resistance.","Transformation of Oleosins Containing Engineered Cysteines and DGAT1 into Wild Type ","Transformation into (white clover) was performed according to the procedure of Voisey et al., (1994).","Seeds were weighed to provide approximately 400-500 cotyledons (ie. 200-250 seeds) for dissection (0.06 gm=100 seeds). In a centrifuge tube, seeds were rinsed with 70% ethanol for 1 minute. Surface sterilised in bleach (5% available chlorine) by shaking on a circular mixer for 15 minutes followed by four washes in sterile water. Seeds were imbibed overnight at 4 degC.","The same constructs used to transform (abover) were maintained in strain GV3101 and inoculated into 25 mL of MGL broth (Table 4) containing spectinomycin at a concentration of 100 mg\/L. Cultures were grown overnight (16 hours) on a rotary shaker (200 rpm) at 28\u00b0 C. Bacterial cultures were harvested by centrifugation (3000\u00d7g, 10 minutes). The supernatant was removed and the cells resuspended in a 5 mL solution of 10 mM MgSO4.","Cotyledons were dissected from seeds using a dissecting microscope. First, the seed coat and endosperm were removed. Cotyledons were separated from the radical with the scalpel by placing the blade between the cotyledons and slicing through the remaining stalk. Cotyledonary explants were harvested onto a sterile filter disk on CR7 media.","For transformation, a 3 ul aliquot of suspension was dispensed to each dissected cotyledon. Plates were sealed and cultured at 25 degC under a 16 hour photoperiod. Following a 72 hour period of co-cultivation, transformed cotyledons were transferred to plates containing CR7 medium supplemented with ammonium glufosinate (2.5 mg\/L) and timentin (300 mg\/L) and returned to the culture room.","Following the regeneration of shoots, explants were transferred to CR5 medium supplemented with ammonium glufosinate (2.5 mg\/L) and timentin (300 mg\/L). Regenerating shoots are subcultured three weekly to fresh CR5 media containing selection.","As root formation occurs, plantlets were transferred into tubs containing CR0 medium containing ammonium glufosinate selection. Large clumps of regenerants were divided to individual plantlets at this stage. Whole, rooted plants growing under selection were then potted into sterile peat plugs. Once established in peat plugs plants were then transfer to the greenhouse.",{"@attributes":{"id":"p-0608","num":"0677"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 4"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Media compositions used for transformation."},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"right"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"A.","CR#0",{},{}]},{"entry":[{},{},"MS salts",{},{}]},{"entry":[{},{},"B5 vitamins",{},{}]},{"entry":[{},{},"sucrose","30","g\/L"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"pH 5.8","(KOH)"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"agar","8.0","g\/L"]},{"entry":[{},"CR#5",{},{}]},{"entry":[{},"MS salts",{},{}]},{"entry":[{},"B5 vitamins",{},{}]},{"entry":[{},"sucrose","30","g\/L"]},{"entry":[{},"BA","0.1","mg\/L"]},{"entry":[{},"NAA","0.05","mg\/L"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"pH 5.8","(KOH)"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"right"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"agar","8.0","g\/L"]},{"entry":[{},"B.","CR#7",{},{}]},{"entry":[{},{},"MS salts",{},{}]},{"entry":[{},{},"B5 vitamins",{},{}]},{"entry":[{},{},"sucrose","30","g\/L"]},{"entry":[{},{},"BA","1.0","mg\/L"]},{"entry":[{},{},"NAA","0.05","mg\/L"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"pH 5.8","(KOH)"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"right"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"agar","8.0","g\/L"]},{"entry":[{},"C.","MGL",{},{}]},{"entry":[{},{},"Mannitol","5.0","g\/L"]},{"entry":[{},{},"L glutamic acid","1.0","g\/L"]},{"entry":[{},{},"KH2PO4","250","mg\/L"]},{"entry":[{},{},"MgSO4","100","mg\/L"]},{"entry":[{},{},"NaCl","100","mg\/L"]},{"entry":[{},{},"Biotin","100","mg\/L"]},{"entry":[{},{},"Bactotryptone","5.0","g\/L"]},{"entry":[{},{},"Yeast extract","2.5","g\/L"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"pH 7.0","(NaOH)"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"FAMES GC\/MS results showed the transgeneic (containing DGAT1 (S205A) and either Ole3,3 or Ole5,6 or Ole 6,7) had elevated total leaf lipid profiles compared to wild type (). There was a general correlation between the highest level of leaf lipid and the highest number of cysteines engineered into the oleosin.","FAMES GC\/MS results showed the transgeneic (containing DGAT1 (S205A) and either Ole3,3 or Ole5,6 or Ole 6,7) had elevated C18:1 and C18:2 leaf lipid profiles compared to wild type as also seen in (). The highest level of leaf C18:1 and C18:2 was found in plants transformed with the oleosin containing the highest number of engineered cysteines.","Determination of Oil Body Assembly in Leaves (and Seeds)","Further screening was conducted using immunoblot analysis (with an anti-sesame seed oleosin antibody, Scott et al., 2007) to determine the lines overexpressing the oleosin protein. Using this method, either oil bodies (OBs) were extracted from T2 seeds of putative transformants using sucrose density gradient or total protein was extracted from leaves in a denaturing\/reducing buffer and proteins were separated in SDS-PAGE, transferred to nitrocellulose membrane, and challenged with an antibody raised against the sesame oleosin (Scott et al., 2007).","Crude oil body (OB) was extracted from 25 mg of seeds in 500 \u03bcL OB buffer (10 mM Sodium phosphate, pH 7.5 containing 600 mM sucrose). After centrifugation at 13,000\u00d7g, the aqueous layer was carefully suck out and the fat pad layer was resuspended in 200 \u03bcL of OB buffer without disturbing the pellet at the bottom. 20 \u03bcL of each OB extract was added with 4\u00d7 loading dye and 10\u00d7 reducing agent, heated up to 70\u00b0 C. for 5 min and loaded onto 4-12% polyacrylamide gel for immunoblot analysis. The blot was incubated in \u03b1-sesame oleosin antibody (1\u00b0 Ab) at 1:750 dilution for one hour, and another one hour in secondary antibody (1:10,000).","Oleosin is naturally expressed in seeds and not in the leaves. However, since we have co-expressed DGAT1 with oleosin both under the control of CaMV35S promoters it could be anticipated that this would enable detectable levels of oleosin to accumulate in the leaves. Leaves from transformed lines with high expression of recombinant oleosin in the seeds (identified by immunoblot analysis) were analyzed by immunoblot using antibodies raised against the sesame oleosin.","Table 5 below summarises the number of putative transformants generated and the number of plants expressing recombinant oleosin in the seed and leaf.",{"@attributes":{"id":"p-0615","num":"0684"},"tables":{"@attributes":{"id":"TABLE-US-00006","num":"00006"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"56pt","align":"center"}}],"thead":{"row":[{"entry":"TABLE 5"},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":[{},{},"Number of","Number of lines","Number of lines"]},{"entry":[{},"Number of","lines seeds","with a positive","with a positive"]},{"entry":[{},"putative","were analysed","immunoreactive","immunoreactive"]},{"entry":[{},"transformants","by immunoblot","band at the","band at the"]},{"entry":[{},"(based on","(anti sesame","appropriate","appropriate"]},{"entry":["Gene","BASTA","seed oleosin","size in the","size in the"]},{"entry":["construct ID","resistance)","antibody)","seed extract","leaf extract"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"56pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["pRSh1-DGAT1","8","N\/A","N\/A",{}]},{"entry":["(S205A) control",{},{},{},{}]},{"entry":["pRSh1-DGAT1","14","8","7","3"]},{"entry":["(S205A)-Ole_0-0",{},{},{},{}]},{"entry":["pRSh1-DGAT1","22","2","1","1"]},{"entry":["(S205A)-Ole_1-1",{},{},{},{}]},{"entry":["pRSh1-DGAT1","20","0","0","1"]},{"entry":["(S205A)-Ole_1-3",{},{},{},{}]},{"entry":["pRSh1-DGAT1","23","8","4","2"]},{"entry":["(S205A)-Ole_3-1",{},{},{},{}]},{"entry":["pRSh1-DGAT1","54","22","16","5"]},{"entry":"(S205A)-Ole_3-3"},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},"It should be noted the level of recombinant oleosin that accumulated in the leaves was considerably lower than in the seeds. However, the proportion of individual lines accumulating detectable levels in both the leaves was much greater than when oleosin was expressed alone (Roberts Lab, unpublished data) indicating that the co-expression of both DGAT1 and oleosin in the leaf has lead to the accumulation of higher levels of oleosin.","Analysis of Leaves from Transgenic Plants Co-Expressing DGAT1 (S205A) and Disulfide Oleosins","The seeds from homozygous lines over expressing the oleosin protein in the seeds were germinated to allow growth of 2, 3, 4 or 5 weeks. Sufficient leaf material was harvested for FAMES GC-MS, as well as by GC-MS using a RTX 65-TG Restek column which enable the separation and identification of free fatty acids, diacylglycerides, wax esters, sterol esters and triacylglycerides without derivatization.","Preparation of Material for FAMES-GC\/MS Analysis","10 mg of freeze-dried leaf powder was placed in a 13\u00d7100 mm screw-cap tube, 10 \u03bcL of non methylated internal standard (C15:0 FA, 4 mg\/mL dissolved in heptane) was added, To this mixture, 1 mL of the methanolic HCl reagent (1 mL of 3 M solution diluted to 1 M using dry methanol which had been treated with 5% 2,2-dimeethoxypropane as a water scavenger. The tube was then flushed with N2 gas then sealed immediately with Teflon-lined cap, and heated at 80\u00b0 C. for 1 h. After the tubes had cooled to room temperature, 10 \u03bcL pre-methylated standard (4 mg\/mL of C17:0 dissolved in heptane) was added. To this mixture, 0.6 mL heptane and 1.0 mL of 0.9% (w\/v) NaCl was added, and mixed thoroughly by vortexing. Following centrifugation at 500 rpm for 1 min at room temperature, 100 \u03bcL of the top layer (containing heptanes) was collected and transferred to a flat-bottom glass insert fitted into a brown vial for GC\/MS analysis.","FAMES GC\/MS Analysis","The FAMES GC\/MS was analysed using the SGE capillary column BPX70 (50m\u00d70.22 mm\u00d70.25 \u03bcm). The condition of GC-MS was as follows: the temperature was programmed from 80\u00b0 C. to 150\u00b0 C. at 15\u00b0 C.\/min and then to 250\u00b0 C. at 8\u00b0 C.\/min and held isothermal for 10 min. Samples were injected in a split mode; total flow of 28.4 mL\/min; column flow of 0.82 mL\/min; and a purge flow of 3.0 mL\/min. The pressure was kept at 150 kPa; ion source temperature was 200\u00b0 C. and an interface temperature was kept at 260\u00b0 C. The target compounds were acquired by mass spectrometry in a scan mode starting at 50 m\/z and ending at 350 m\/z.","TAG and Polar Lipid Extraction","TAG was extracted using a modified method of Ruiz-L\u00f3pez et al., (2003). Briefly, for each TAG analysis, betweeen 34-80 mg of freeze-dried leaf powder was placed into tared 13-mm screw cap tube and weighed, 2.4 mL of 0.17 M NaCl in MeOH was added and mixed by vortexing. Following the addition of 4.8 mL heptane and 10 \u03bcL of internal standard (C14:0, 10 \u03bcg.\u03bcL-1), the suspension was mixed gently and incubated without shaking in 80\u00b0 C. water bath for 2 h. After cooling to room temperature, the upper phase (containing lipids) was transferred to fresh screw-cap tube and evaporated to dryness under stream of N gas. Finally, the dried powder were resuspended in 100 \u03bcL heptanes, mixed thoroughly then transferred to a flat-bottom glass insert fitted into a brown glass vial for TAG analysis.","TAG GC-MS Analysis","TAG analysis was performed on a Hewlett Packard (HP) GC and Shimadzu Scientific Instruments Inc. MS (QP2010). All analyses were performed with a RESTEK capillary column MXT-65TG (65% diphenyl-35% dimethyl polysiloxane, 30.0 m\u00d70.10 \u03bcm thickness\u00d70.25 mm diameter) in Electron Impact (EI) ionization mode. Helium was used as the carrier gas. All samples were injected in splitless mode, in 1.0 \u03bcl aliquots, and a column flow of 1.2 mL\u00b7min\u22121. The gas chromatograph was programmed from 200 to 370\u00b0 C. at 15\u00b0 C.\u00b7min\u22121 and kept isothermal at 370\u00b0 C. for 15 min. The sample injector port temperature was maintained at 350\u00b0 C., column oven temperature at 200\u00b0 C., with a pressure of 131.1 kPa and a purge flow of 3.0 mL\u00b7min\u22121. The mass spectrometric conditions were as follows: ion source temperature was held at 260\u00b0 C. during the GC-MS runs, the mass spectra were obtained at ionization voltage of 70 eV at an emission current of 60 \u03bcA and an interface temperature of 350\u00b0 C. Acquisition mode was by scanning at a speed of 5000, 0.25 sec per scan. Chromatograph peaks with mass to charge ratio of 45 m\/z to 1090 m\/z were collected starting at 9 min and ending at 25 min.","The applicants have used the same strategy as for sesame seed oleosin, accession number AAD42942, (i,e., substituting charged residues predicted to be on the surface of OBs with cysteines) to engineer cysteines into the N- and C-terminal hydrophilic arms of oleosins caoleosins and steroleosins. In some cases it has been possible to substitute only negatively charged amino acids (Glutamic acid and Aspartic acid) that are relatively evenly spaced. In the case of the sesame oleosin AAD42942 it was necessary to sometimes compromise on the charge substitution. It should be noted in the examples below that two caleosins (AAB71227 and AAF13743) contain two endogenous cysteines in their C-terminal arm. These are left unaltered in the engineering.","To determine the position of the amino acid substitution each protein was aligned with the sesame oleosin (AAD42942) in the original form as well as the forms containing 1 or 3 cysteines per hydrophilic arm (i.e., ole0,0; ole1,1; ole3,1; ole1,3; ole3,3). The potential glutamic acids and aspartic acids in N-terminus or C-terminus of each of the hydrophilic arms (determined by hydrophobicity plots) were then highlighted with grey boxes, as were the relevant lysine, arginine and glutamine residues (which were all successfully altered in the sesame oleosin (AAD42942). The mutation of these residues to cysteine were then considered along with their spacing with each other. The final substitutions are then shown with the original peptide sequence and the engineered sequence only. In this case only 3 cysteines were engineered into each arm, however, the number could have been greater or fewer. An alternative approach would have been to work with each protein in isolation and simply begin by identifying the hydrophilic regions by hydrophobicity plot then begin the process of substitution with the most appropriate charged amino acid.","Table 6 below shows additional oleosin and caoleosins that the applicants have modified to introduce cysteines in the hydrophilic portions.",{"@attributes":{"id":"p-0625","num":"0694"},"tables":{"@attributes":{"id":"TABLE-US-00007","num":"00007"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}}],"thead":{"row":[{"entry":"TABLE 6"},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["Protein Type","Plant Source","Accession Number","SEQ ID NO"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["oleosin",{"i":"Brassica oleraceae"},"CAA65272.1","90"]},{"entry":[{},"(pollen oleosin)",{},{}]},{"entry":["oleosin","Maize","NP_001147032.1","91"]},{"entry":["oleosin","Rice","AAL40177.1","92"]},{"entry":["caoleosin","Sesame","AAF13743","93"]},{"entry":["caoleosin","Soybean","AAB71227","94"]},{"entry":["caoleosin","Maize","NP_001151906","95"]},{"entry":["steroleosin","Sesame","AAL13315","96"]},{"entry":["steroleosin",{"i":"Brassica napus"},"ACG69522","97"]},{"entry":["steroleosin","Maize","NP_001152614.1","98"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}}}},"Table 7 below references the SEQ ID NO in the modified oleosins",{"@attributes":{"id":"p-0627","num":"0696"},"tables":{"@attributes":{"id":"TABLE-US-00008","num":"00008"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}}],"thead":{"row":[{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["Protein Type","Plant Source","Accession Number","SEQ ID NO"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["oleosin",{"i":"Brassica oleraceae"},"X96409","99"]},{"entry":[{},"(pollen oleosin)",{},{}]},{"entry":["oleosin","Maize","NP_001147032.1","100"]},{"entry":["oleosin","Rice","AAL40177.1","101"]},{"entry":["caoleosin","Sesame","AAF13743","102"]},{"entry":["caoleosin","Soybean","AAB71227","103"]},{"entry":["caoleosin","Maize","NP_001151906","104"]},{"entry":["steroleosin","Sesame","AAL13315","105"]},{"entry":["steroleosin",{"i":"Brassica napus"},"ACG69522","106"]},{"entry":["steroleosin","Maize","NP_001152614.1","107"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},"The modified sequence can be expressed as described in the examples above to produce oil bodies, emulsions, transgenic host cells, plants etc, and to test the properties of each.","The applicants have used the same strategy in Example 7 (by preventing the catabolism of TAG in the leaf which inturn ensures there is a continual recycling of COfrom pyruvate as it is used by the pyruvate dehydrogenase complex to generate Acetyl-CoA for lipid biosynthesis) to not only increase the COassimilation rate but also elevate intrinsic Water Use Efficiency. The net effect of this is to elevate the partial pressure of COcompared to Oin the chloroplast. COassimilation rates in both air (containing rougly 79% N, 21% Oand 400 ppm CO) and in a calibrated gas mixture with reduced O(containing 98% N, 2% Oand 400 ppm CO) were measured to demonstrate that the plants which had protected TAG (via the co-expression of DGAT and cysteine oleosin) had reduced levels of photorespiration","IRGA Settings","Rates of photosynthesis were measured for 6 wild type (WT) and 6 transgenic (T) plants (DGAT+Ole-3,3), at 200 \u03bcmol msPAR (growing condition), between 11:00 and 16:00 on the 24th of May 2011 using a portable photosynthesis system (Li6400, LiCor Inc., Lincoln, Nebr., USA) fitted with standard 2\u00d73 cm leaf chamber, leaf thermocouple and a blue-red LED light source. Intrinsic water-use efficiency was estimated from the ratio of photosynthesis\/conductance (Osmond et al. 1980). Block temperature was held at 20\u00b0 C., stomata ratio set at 1.6 and the vapour pressure deficit was between 0.6 and 0.9 kPa. For measurements under nonphotorespiratory conditions, a tank of 2% oxygen (certified) in nitrogen was connected to the Li-6400 inlet.","Statistical Analysis","A standard t-test statistic (R 2.12) was used in this study for comparison between treatments or between genotypes. The means of each genotype were obtained, together with the average standard error of the difference between two means (SED).","Net Photosynthesis and Intrinsic Water-Use Efficiency","Rates of photosynthesis were significantly greater (under photorespiratory conditions) in transgenic plants compared to wild type plants; similarly, rates of photosynthesis were significantly greater (under nonphotorespiratory conditions) in transgenic plants compared to wild type plants. (, left hand panel). The DGAT1-Ole3,3 plants had greater increases in photosynthesis when photorespiration was completely removed using a low Oenvironment compared to wild type (, right hand panel). Thus showing that DGAT1-Ole3,3 plants have elevated COassimilation rates compared to wild type plants.","Intrinsic water-use efficiency measurements were significantly greater under nonphotorespiratory conditions than under ambient oxygen concentration for both the WT and T genotypes (, left hand panel). At ambient Olevels the iWUE was consistently higher for plants transformed with DGAT1-Ole3,3 than wild type plants; this was further demonstrated by the fact that the DGAT1-Ole3,3 plants had smaller increases in iWUE when photorespiration was completely removed using a low Oenvironment (, right hand panel). Thus showing that DGAT1-Ole3,3 plants have higher iWUE compared to wild type plants.","Stomatal conductance was significantly higher in wild type plants than plants transformed with DGAT1-Ole3,3 under ambient oxygen conditions (, left hand panel). In the wild typ plants, stomatal conductance was reduced slighlyt under nonphotorespiratory conditions where as in plants transformed with DGAT1-Ole3,3 the stomatal conductance increased by over 25% compared to ambient conditions (, right hand panel). The stomatal conductance for both wild type and plants transformed with DGAT1-Ole3,3 were approximately the same when photorespiration was eliminated by the low Oconditions (, left hand panel).","The stomatal density of the wild type plants and plants transformed with DGAT1-Ole3,3 were similar ().","The consequence of transforming plants with DGAT1-Ole3,3 was an elevation in photosynthetic assimilation rates, increased water use efficiencies and decreased stomatal conductance leading to an increase in growth rate seen by higher biomass, shown in Table 8 below and .",{"@attributes":{"id":"p-0637","num":"0706"},"tables":{"@attributes":{"id":"TABLE-US-00009","num":"00009"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"center"}}],"thead":{"row":[{"entry":"TABLE 8"},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":[{},"Average leaf","Average leaf"]},{"entry":[{},"DW\/plant (mg)","DW\/plant (mg)"]},{"entry":[{},"20 Days after","30 Days after"]},{"entry":[{},"germination","germination"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"84pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Wild Type","30.9 \u00b1 3.5","61.1 \u00b1 3.3"]},{"entry":[{},"OLE1_3,3","47.6 \u00b1 4.4","90.0 \u00b1 6.1"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},"Plants over expressing DGAT1(S205A) and OLE13,3 had approximately 50% more biomass than the wild type; this included at the onset of flower stalk emergence (\u02dc20 days after germination) as well as at mid to late floral stalk development (\u02dc35 days after germination).","Roots from Ole3,3 and wild type plants were extracted using the same procedures described in Example 7. Quantitative FAMES analysis ( left panel) showed that the total lipid content of the roots from Ole3,3 was 8.2% of the DM while the total lipid content of the wild type roots was 1.7% of the DM. FAMES also showed that the lipid profile of the Ole3,3 roots was not too different from the wild type ( right panel). The most noticeable change was the proportion of C18:1 was 4.0% in the wild type roots and rose over four fold to 18.1% of the total fatty acids in the roots of DGAT1-Ole3,3 plants. Despite the similar total lipid content (\u02dc8%) of the leaves and roots from the DGAT1-Ole3,3 plants the TAG analysis demonstrated that a higher portion of the total lipids in the roots was TAG when compared with the leaf material (). This was likely due to a much higher portion of the total lipid in the leaf being membrane lipid (predominantly thylakoid membrane).","It is not the intention to limit the scope of the invention to the abovementioned examples only. As would be appreciated by a skilled person in the art, many variations are possible without departing from the scope of the invention.","References",{"@attributes":{"id":"p-0641","num":"0000"},"ul":{"@attributes":{"id":"ul0030","list-style":"none"},"li":["Abell et al., (2004). Plant J., 37: 461-70.","Altschul et al., (1997) Nucleic Acids Res. 25: 3389-3402,","Andrianov et al., (2010). Plant Biotechnol J. 8(3):277-87.","Ausubel et al., (1987) Current Protocols in Molecular Biology, Greene Publishing Bairoch and Bucher (1994), Nucleic Acids Res. 22, 3583","Bao et al, (2000) Plant J. 22(1):39-50.","Bari et al., (2009). J. Exp. Bot. 55:623-630.","Birch (1997) Ann Rev Plant Phys Plant Mol Biol, 48, 297","Bock & Khan (2004). Trends in Biotech. 22:311-318.","Bolton and McCarthy (1962) PNAS 84:1390","Bowie et al., (1990). Science 247, 1306","Bouvier-Nave et al., (2000) Eur. J. Biochem. 267, 85-96.","Bryan et al., (2007) Modification of fatty acid biosynthesis. United States Patent 20070118927.","Capuano et al., (2007). Biotechnol Adv. 25:203-206.","Chen et al., (1999). Plant Cell Physiol., 40:1079-1086 Chiang et al., (2005). J Agric Food Chem 53:4799-804.","Chiang et al., (2007). Protein Expr Purif. 52:14-8.","Chisti (2007). Biotech. Adv. 25:294-306.","Colman et al., (1974). Plant Phys, 53: 395-397.","Cookson et al., (2009). Improvements in and relating to oil production. PCT\/NZ2008\/000085 WO\/2008\/130248","Dahlqvist et al., (2000). Proc Natl Acad Sci USA. 97, 6487-6492.","Deckers et al., (2003). U.S. Pat. No. 6,582,710","Demeyer and Doreau, (1999). Proc Nutr Soc. 58(3):593-607.","Deutscher (1990) Ed, Methods in Enzymology, Vol. 182, Guide to Protein Purification","Draper et al., 1988, Plant Genetic Transformation and Gene Expression. A Laboratory Manual. Blackwell Sci. Pub. Oxford, p. 365","Falquet et al., 2002, Nucleic Acids Res. 30, 235","Feng and Doolittle, 1987, J. Mol. Evol. 25, 351","Firkins et al., (2006). J Dairy Sci. 89 Suppl 1:E31-51. Review. Greenspan.","Frandsen et al., (2001). Physiologia Plantarum, 112:301-307.","Frohman (1993). Methods Enzymol. 218: 340-56","Galun and Breiman (1997). Transgenic Plants. Imperial College Press, London","Gelvin et al., 1993, Plant Molecular Biol. Manual. Kluwer Acad. Pub. Dordrecht","Giesen et al., Nucleic Acids Res. 1998 Nov. 1; 26(21):5004-6","Giordano et al., (2005). Ann. Rev. Pl. Biol. 56:99-131.","Halford & Hardie (1998). Plant Mol. Biol. 37:735-48. Review","Harada et al., (2002). OLEOSIN\/PHOSPHOLIPID COMPLEX AND PROCESS FOR PRODUCING THE SAME. World Patent WO\/2002\/026788.","Hellens et al., (2000). Plant Mol Biol 42: 819-32","Hellens et al., (2005). Plant Meth 1: 13","Herrera-Estrella et al., (1993). Nature 303, 209","Hofmann et al., (1999). Nucleic Acids Res. 27, 215","Hou et al., (2003). J Dairy Sci; 86: 424-8.","Huang (1992). Ann. Rev. Plant Physiol. Plant Mol. Biol. 43:177-200.","Huang, X. (1994) Computer Applications in the Biosciences 10, 227-235","Jeanmougin et al., (1998) Trends Biochem. Sci. 23, 403-5.","Jenkins and Bridges (2007). Eur. J. Lipid Sci. Technol. 109:778-789.","Jenkins and McGuire (2006). J Dairy Sci. 89(4):1302-10. Review.","Kaup et al., (2002) Plant Physiol. 129(4):1616-26.","Kebeish et al., (2007). Nature Biotech, 25:593-599.","Kozaki & Takeba (1996). Nature, 384:557-560.","Kyte and Doolitle (1982) J. Mol. Biol. 157:105-132","Lanfranco L. (2003). Riv Biol. 96(1):31-54.","Lardizabal et al., (2001). J.B.C. 276, 38862-38869.","Leprince et al., (1998). Planta 204 109-119.","Lin and Tzen. (2004). Plant Physiology and Biochemistry. 42:601-608.","Lock and Bauman (2004). Lipids. 39(12):1197-206. Review.","Loer and Herman (1993). Plant Physiol. 101(3):993-998.","Mayer and Fowler (1985). J. Cell Biol. 100(3):965-73.","Mekhedov et al., (2000). Plant Physiol. 122(2):389-402).","Mullis et al., Eds. 1994 The Polymerase Chain Reaction, Birkhauser","Murphy (1993). Prog. Lipid Res. 32:247-280.","Nakamura et al., (2005). Can. J. Bot. 83:820-833.","Needleman and Wunsch, (1970) J. Mol. Biol. 48, 443-453","Nielsen et al., Science. (1991) 254(5037):1497-500","Notredame et al., (2000). J. Mol. Biol. 302: 205-217","Ohlrogge and Jaworski (1997). Annu Rev Plant Physiol Plant Mol. Biol. 48:109-136.","Papapostolou and Howorka (2009). Mol. Biosyst. 5(7):723-32. Review.","Parry et al., (2003). J. Exp. Bot., 54:1321-1333.","Peng (2004). Development and applications of artificial sesame oil body. Ph.D. Dissertation. National ChunHsing University Graduate Institute of Biotechnology. Taichung, Taiwan.","Peng et al., (2006). Stability enhancement of native and artificial oil bodies by genipin crosslink. Taiwan patent 1250466.","Peng et al., (2004). J Biotechnol 2004; 111: 51-7.","Potrykus and Spangenburg (1995). Gene Transfer to Plants. Springer-Verlag, Berlin","Roberts et al., (2008). The Open Biotechnology Journal 2:13-21.","Roux et al., (2004). J Agric Food Chem. 52(16):5245-9.","Scott et al., (2007). Polyoleosins. WO2007045019.","Saha et al., (2006). Plant Physiol. 141(4):1533-43.","Sambrook et al., Eds, 1987, Molecular Cloning, A Laboratory Manual, 2nd Ed. Cold Spring Harbor Press","Sarmiento et al., (1997). Plant J. 11(4):783-96.","Schrott (1995) In: Gene Transfer to Plants (Potrykus, T., Spangenberg. Eds) Springer Verlag. Berline, pp. 325-336","Shimada et al., (2008). Plant J. 55(5):798-809.","Shockey et al., (2006). Plant Cell., 18, 2294-2313.","Siloto et al., (2006). Plant Cell. 18(8):1961-74.","Slack et al., (1980). Biochem J. 190(3):551-561.","Slocombe et al., (2009). Plant Biotechnol J. 7(7):694-703.","Stahl et al., (2004). Plant Physiology, 135: 1324-1335.","Stewart et al., (1969), In: Solid-Phase Peptide Synthesis, WH Freeman Co, San Francisco Calif.","Thompson et al., (1994) Nucleic Acids Research, 22:4673-4680","Ting et al., (1997). J Biol. Chem., 272: 3699-3706.","Tadege et al., (2005). Trends Plant Sci. 10(5):229-35.","Triglia et al., 1998, Nucleic Acids Res 16, 8186","Tolbert (1997). Ann. Rev. Pl. Phys. Pl. Molec. Biol. 48:1-25.","Tolbert et al., (1983). Pl. Physiol. 72:1075-1083.","Tzen et al., (1992). J. Biol. Chem. 267: 15626-34","Tzen et al., (2003). Adv Plant Physiol., 6: 93-104.","Tzen et al., (1997). J. Biochem. 121(4):762-8.","Voisey et al., (1994). Plant Cell Reports 13: 309 314.","Winichayakul et al., (2008). Proc. NZGA, 70:211-216","Xu et al., (2005). Plant Cell. 17(11):3094-110.","Zou et al., (1999). Plant J. 19, 645-653.","Zou et al., (2008). Plant Biotech. J. 6(8):799-818."]}},{"@attributes":{"id":"p-0642","num":"0807"},"tables":{"@attributes":{"id":"TABLE-US-00010","num":"00010"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"273pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"SUMMARY OF SEQUENCE LISTING"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"140pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SEQ ID NO:","Type","SPECIES","COMMENTS"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"140pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1","polynucleotide","artificial","Oleosin disulfide 0,0 nucleotide sequence,"]},{"entry":[{},{},{},"as cloned into pET29b using NdeI and XhoI"]},{"entry":[{},{},{},"restriction sites (adds N-terminal S\u2022tag"]},{"entry":[{},{},{},"thrombin cleavage site and C-terminal His tag)."]},{"entry":["2","polynucleotide","artificial","Oleosin disulfide 1,1 nucleotide sequence,"]},{"entry":[{},{},{},"as cloned into pET29b using NdeI and XhoI"]},{"entry":[{},{},{},"restriction sites (adds N-terminal S\u2022tag"]},{"entry":[{},{},{},"thrombin cleavage site and C-terminal His tag)."]},{"entry":["3","polynucleotide","artificial","Oleosin disulfide 1,3 nucleotide sequence,"]},{"entry":[{},{},{},"as cloned into pET29b using NdeI and XhoI"]},{"entry":[{},{},{},"restriction sites (adds N-terminal S\u2022tag"]},{"entry":[{},{},{},"thrombin cleavage site and C-terminal His tag)."]},{"entry":["4","polynucleotide","artificial","Oleosin disulfide 3,1 nucleotide sequence,"]},{"entry":[{},{},{},"as cloned into pET29b using NdeI and XhoI"]},{"entry":[{},{},{},"restriction sites (adds N-terminal S\u2022tag"]},{"entry":[{},{},{},"thrombin cleavage site and C-terminal His tag)."]},{"entry":["5","polynucleotide","artificial","Oleosin disulfide 3,3 nucleotide sequence,"]},{"entry":[{},{},{},"as cloned into pET29b using NdeI and XhoI"]},{"entry":[{},{},{},"restriction sites (adds N-terminal S\u2022tag"]},{"entry":[{},{},{},"thrombin cleavage site and C-terminal His tag)."]},{"entry":["6","Polypeptide","Artificial","Oleosin disulfide 0,0 peptide sequence, as"]},{"entry":[{},{},{},"cloned into pET29b using NdeI and XhoI"]},{"entry":[{},{},{},"restriction sites (adds N-terminal S\u2022tag"]},{"entry":[{},{},{},"thrombin cleavage site and C-terminal His tag)."]},{"entry":["7","Polypeptide","Artificial","Oleosin disulfide 1,1 peptide sequence, as"]},{"entry":[{},{},{},"cloned into pET29b using NdeI and XhoI"]},{"entry":[{},{},{},"restriction sites (adds N-terminal S\u2022tag"]},{"entry":[{},{},{},"thrombin cleavage site and C-terminal His tag)."]},{"entry":["8","Polypeptide","Artificial","Oleosin disulfide 1,3 peptide sequence, as"]},{"entry":[{},{},{},"cloned into pET29b using NdeI and XhoI"]},{"entry":[{},{},{},"restriction sites (adds N-terminal S\u2022tag"]},{"entry":[{},{},{},"thrombin cleavage site and C-terminal His tag)."]},{"entry":["9","Polypeptide","Artificial","Oleosin disulfide 3,1 peptide sequence, as"]},{"entry":[{},{},{},"cloned into pET29b using NdeI and XhoI"]},{"entry":[{},{},{},"restriction sites (adds N-terminal S\u2022tag"]},{"entry":[{},{},{},"thrombin cleavage site and C-terminal His tag)."]},{"entry":["10","Polypeptide","Artificial","Oleosin disulfide 3,3 peptide sequence, as"]},{"entry":[{},{},{},"cloned into pET29b using NdeI and XhoI"]},{"entry":[{},{},{},"restriction sites (adds N-terminal S\u2022tag"]},{"entry":[{},{},{},"thrombin cleavage site and C-terminal His tag)."]},{"entry":["11","Polynucleotide","Artificial","(Nucleotide sequence of Oleosin disulfide"]},{"entry":[{},{},{},"0,0 including Kozac sequence and UBQ10"]},{"entry":[{},{},{},"intron, as transformed into "]},{"entry":[{},{},{},{"i":"thaliana "}]},{"entry":[{},{},{},"promoter.)"]},{"entry":["12","Polynucleotide","Artificial","Nucleotide sequence of Oleosin disulfide 1,1"]},{"entry":[{},{},{},"including Kozac sequence and UBQ10"]},{"entry":[{},{},{},"intron, as transformed into "]},{"entry":[{},{},{},{"i":"thaliana "}]},{"entry":[{},{},{},"promoter."]},{"entry":["13","Polynucleotide","Artificial","(Nucleotide sequence of Oleosin disulfide"]},{"entry":[{},{},{},"1,3 including Kozac sequence and UBQ10"]},{"entry":[{},{},{},"intron, as transformed into "]},{"entry":[{},{},{},{"i":"thaliana "}]},{"entry":[{},{},{},"promoter.)"]},{"entry":["14","Polynucleotide","Artificial","Nucleotide sequence of Oleosin disulfide 3,1"]},{"entry":[{},{},{},"including Kozac sequence and UBQ10"]},{"entry":[{},{},{},"intron, as transformed into "]},{"entry":[{},{},{},{"i":"thaliana "}]},{"entry":[{},{},{},"promoter."]},{"entry":["15","Polynucleotide","Artificial","Nucleotide sequence of Oleosin disulfide 3,3"]},{"entry":[{},{},{},"including Kozac sequence and UBQ10"]},{"entry":[{},{},{},"intron, as transformed into "]},{"entry":[{},{},{},{"i":"thaliana "}]},{"entry":[{},{},{},"promoter."]},{"entry":["16","Polypeptide","Artificial","Peptide sequence of Oleosin disulfide 0,0,"]},{"entry":[{},{},{},"as transformed into "]},{"entry":[{},{},{},"under the control of the CaMV35s promoter."]},{"entry":["17","Polypeptide","Artificial","Peptide sequence of Oleosin disulfide 1,1,"]},{"entry":[{},{},{},"as transformed into "]},{"entry":[{},{},{},"under the control of the CaMV35s promoter.)"]},{"entry":["18","Polypeptide","Artificial","Peptide sequence of Oleosin disulfide 1,3,"]},{"entry":[{},{},{},"as transformed into "]},{"entry":[{},{},{},"under the control of the CaMV35s promoter."]},{"entry":["19","Polypeptide","Artificial","Peptide sequence of Oleosin disulfide 3,1,"]},{"entry":[{},{},{},"as transformed into "]},{"entry":[{},{},{},"under the control of the CaMV35s promoter."]},{"entry":["20","Polypeptide","Artificial","Peptide sequence of Oleosin disulfide 3,3,"]},{"entry":[{},{},{},"as transformed into "]},{"entry":[{},{},{},"under the control of the CaMV35s promoter"]},{"entry":["21","Polynucleotide","Artificial","Nucleotide sequence of Oleosin disulfide 5,6"]},{"entry":[{},{},{},"including Kozac sequence and UBQ10"]},{"entry":[{},{},{},"intron, as transformed into "]},{"entry":[{},{},{},{"i":"thaliana "}]},{"entry":[{},{},{},"promoter."]},{"entry":["22","Polynucleotide","Artificial","Nucleotide sequence of Oleosin disulfide 6,7"]},{"entry":[{},{},{},"including Kozac sequence and UBQ10"]},{"entry":[{},{},{},"intron, as transformed into "]},{"entry":[{},{},{},{"i":"thaliana "}]},{"entry":[{},{},{},"promoter."]},{"entry":["23","Polypeptide","Artificial","Peptide sequence of Oleosin disulfide 5,6,"]},{"entry":[{},{},{},"as transformed into "]},{"entry":[{},{},{},"under the control of the CaMV35s promoter"]},{"entry":["24","Polypeptide","Artificial","Peptide sequence of Oleosin disulfide 6,7,"]},{"entry":[{},{},{},"as transformed into "]},{"entry":[{},{},{},"under the control of the CaMV35s promoter"]},{"entry":["25","Polynucleotide","Artificial","Oleoacceptor (contains OCS terminator,"]},{"entry":[{},{},{},"CAMV35S promoter, DGAT1 (S205A) from"]},{"entry":[{},{},{},{"i":"Arabidopsis "}]},{"entry":["26","Polynucleotide","Artificial","Oleosin_0,0 and DGAT1 (S205A) in pRSH1"]},{"entry":["27","Polynucleotide","Artificial","Oleosin_1,1 and DGAT1 (S205A) in pRSH1"]},{"entry":["28","Polynucleotide","Artificial","Oleosin_1,3 and DGAT1 (S205A) in pRSH1"]},{"entry":["29","Polynucleotide","Artificial","Oleosin_3,1 and DGAT1 (S205A) in pRSH1"]},{"entry":["30","Polynucleotide","Artificial","Oleosin_3,3 and DGAT1 (S205A) in pRSH1"]},{"entry":["31","Polynucleotide","Artificial","Oleosin_5,6 and DGAT1 (S205A) in pRSH1"]},{"entry":["32","Polynucleotide","Artificial","Oleosin_6,7 and DGAT1 (S205A) in pRSH1"]},{"entry":["33","Polypeptide","Artificial","DGAT1 (S205A)"]},{"entry":["34","Polynucleotide",{"i":"S. indicum"},"Oleosin - AF302807"]},{"entry":["35","Polypeptide",{"i":"S. indicum"},"Oleosin - AAG23840"]},{"entry":["36","Polynucleotide",{"i":"S. indicum"},"Oleosin - U97700"]},{"entry":["37","Polypeptide",{"i":"S. indicum"},"Oleosin - AAB58402"]},{"entry":["38","Polynucleotide",{"i":"A. thaliana"},"Oleosin - X62353"]},{"entry":["39","Polypeptide",{"i":"A. thaliana"},"Oleosin - CAA44225"]},{"entry":["40","Polynucleotide",{"i":"A. thaliana"},"Oleosin - BT023738"]},{"entry":["41","Polypeptide",{"i":"A. thaliana"},"Oleosin - AAZ23930"]},{"entry":["42","Polynucleotide",{"i":"H. annuus"},"Oleosin - X62352.1"]},{"entry":["43","Polypeptide",{"i":"H. annuus"},"Oleosin - CAA44224.1"]},{"entry":["44","Polynucleotide",{"i":"B. napus"},"Oleosin - X82020.1"]},{"entry":["45","Polypeptide",{"i":"B. napus"},"Oleosin - CAA57545.1"]},{"entry":["46","Polynucleotide",{"i":"Z. mays"},"Oleosin - NM_001153560.1"]},{"entry":["47","Polypeptide",{"i":"Z. mays"},"Oleosin - NP_001147032.1"]},{"entry":["48","Polynucleotide",{"i":"O. sativa"},"Oleosin - L76464"]},{"entry":["49","Polypeptide",{"i":"O. sativa"},"Oleosin - AAL40177.1"]},{"entry":["50","Polynucleotide",{"i":"B. oleracea"},"Oleosin - AF117126.1"]},{"entry":["51","Polypeptide",{"i":"B. oleracea"},"Oleosin - AAD24547.1"]},{"entry":["52","Polynucleotide",{"i":"C. arabica"},"Oleosin - AY928084.1"]},{"entry":["53","Polypeptide",{"i":"C. arabica"},"Oleosin - AAY14574.1"]},{"entry":["54","Polynucleotide",{"i":"S. indicum"},"Steroleosin - AF421889"]},{"entry":["55","Polypeptide",{"i":"S. indicum"},"Steroleosin - AAL13315"]},{"entry":["56","Polynucleotide",{"i":"B. napus"},"Steroleosin - EU678274"]},{"entry":["57","Polypeptide",{"i":"B. napus"},"Steroleosin - ACG69522"]},{"entry":["58","Polynucleotide",{"i":"Z. mays"},"Steroleosin - NM_001159142.1"]},{"entry":["59","Polypeptide",{"i":"Z. mays"},"Steroleosin - NP_001152614.1"]},{"entry":["60","Polynucleotide",{"i":"B. napus"},"Steroleosin - EF143915.1"]},{"entry":["61","Polypeptide",{"i":"B. napus"},"Steroleosin - ABM30178.1"]},{"entry":["62","Polynucleotide",{"i":"S. indicum"},"Caleosin - AF109921"]},{"entry":["63","Polypeptide",{"i":"S. indicum"},"Caleosin - AAF13743"]},{"entry":["64","Polynucleotide",{"i":"G. max"},"Caleosin - AF004809"]},{"entry":["65","Polypeptide",{"i":"G. max"},"Caleosin - AAB71227"]},{"entry":["66","Polynucleotide",{"i":"Z. mays"},"Caleosin - NM_001158434.1"]},{"entry":["67","Polypeptide",{"i":"Z. mays"},"Caleosin - NP_001151906"]},{"entry":["68","Polynucleotide",{"i":"B. napus"},"Caleosin - AY966447.1"]},{"entry":["69","Polypeptide",{"i":"B. napus"},"Caleosin - AAY40837.1"]},{"entry":["70","Polynucleotide",{"i":"C. revoluta"},"Caleosin - FJ455154.1"]},{"entry":["71","Polypeptide",{"i":"C. revoluta"},"Caleosin - ACJ70083.1"]},{"entry":["72","Polynucleotide",{"i":"C. sativus"},"Caleosin - EU232173.1"]},{"entry":["73","Polypeptide",{"i":"C. sativus"},"Caleosin - ABY56103.1"]},{"entry":["74","Polynucleotide",{"i":"A. thaliana"},"DGAT1 - NM_127503"]},{"entry":["75","Polypeptide",{"i":"A. thaliana"},"DGAT1 - NP_179535"]},{"entry":["76","Polynucleotide",{"i":"T. majus"},"DGAT1 - AY084052"]},{"entry":["77","Polypeptide",{"i":"T. majus"},"DGAT1 - AAM03340"]},{"entry":["78","Polynucleotide",{"i":"Z. mays"},"DGAT1 - EU039830.1"]},{"entry":["79","Polypeptide",{"i":"Z. mays"},"DGAT1 - ABV91586.1"]},{"entry":["80","Polynucleotide",{"i":"A. thaliana"},"DGAT2 - NM_115011"]},{"entry":["81","Polypeptide",{"i":"A. thaliana"},"DGAT2 - NP_566952.1"]},{"entry":["82","Polynucleotide",{"i":"B. napus"},"DGAT2 - FJ858270"]},{"entry":["83","Polypeptide",{"i":"B. napus"},"DGAT2 - AC090187.1"]},{"entry":["84","Polynucleotide",{"i":"A. hypogaea"},"DGAT3 (soluble DGAT) - AY875644"]},{"entry":["85","Polypeptide",{"i":"A. hypogaea"},"DGAT3 (soluble DGAT) - AAX62735.1"]},{"entry":["86","Polynucleotide",{"i":"A. thaliana"},"PDAT - NM_121367"]},{"entry":["87","Polypeptide",{"i":"A. thaliana"},"PDAT - NP_196868.1"]},{"entry":["88","Polynucleotide",{"i":"R. communis"},"PDAT - XM_002521304"]},{"entry":["89","Polypeptide",{"i":"R. communis"},"PDAT - XP_002521350"]},{"entry":["90","Polypeptide",{"i":"B. oleraceae"},"Oleosin - CAA65272.1"]},{"entry":["91","Polypeptide",{"i":"Z. mays"},"Oleosin - NP_001147032.1"]},{"entry":["92","Polypeptide",{"i":"O. sativa"},"Oleosin - AAL40177.1"]},{"entry":["93","Polypeptide",{"i":"S. indicum"},"Caleosin - AAF13743"]},{"entry":["94","Polypeptide",{"i":"G. Max"},"Caleosin - AAB71227"]},{"entry":["95","Polypeptide",{"i":"Z. mays"},"Caleosin - NP_001151906"]},{"entry":["96","Polypeptide",{"i":"S. indicum"},"Steroleosin - AAL13315"]},{"entry":["97","Polypeptide",{"i":"Brassica"},"steroleosin ACG69522"]},{"entry":[{},{},{"i":"napus"},{}]},{"entry":["98","Polypeptide",{"i":"Z. mays"},"steroleosin NP_001152614.1"]},{"entry":["99","Polypeptide",{"i":"Brassica"},"Modified pollen oleosin - CAA65272.1"]},{"entry":[{},{},{"i":"oleraceae"},{}]},{"entry":["100","Polypeptide",{"i":"Zea mays"},"Modified oleosin - NP_001147032.1"]},{"entry":["101","Polypeptide",{"i":"Oryza"},"Modified oleosin - AAL40177.1"]},{"entry":[{},{},{"i":"sativa"},{}]},{"entry":["102","Polypeptide",{"i":"S. indicum"},"Modified caoleosin - AAF13743"]},{"entry":["103","Polypeptide",{"i":"G. soja"},"Modified caoleosin - AAB71227"]},{"entry":["104","Polypeptide",{"i":"Z. mays"},"Modified caoleosin - NP_001151906"]},{"entry":["105","Polypeptide",{"i":"S. indicum"},"Modified steroleosin - AAL13315"]},{"entry":["106","Polypeptide",{"i":"Brassica"},"Modified steroleosin - ACG69522"]},{"entry":[{},{},{"i":"napus"},{}]},{"entry":["107","Polypeptide",{"i":"Z. mays"},"Modified steroleosin - NP_001152614.1"]},{"entry":["108","Polypeptide","Artificial","Consensus in  (garden"]},{"entry":[{},{},{},"nasturtium) DGAT1 sequences"]},{"entry":["109","Polypeptide","Artificial","GlcF protein consensus sequence"]},{"entry":["110","Polynucleotide","Artificial","Example sequence"]},{"entry":["111","Polynucleotide","Artificial","Example sequence"]},{"entry":["112","Polynucleotide","Artificial","Sequence of construct in FIG. 1"]},{"entry":["113","Polypeptide","Artificial","First portion of Oleosin 0-0 in FIG. 1"]},{"entry":["114","Polypeptide","Artificial","Second portion of Oleosin 0-0 in FIG. 1"]},{"entry":["115","Polypeptide","Artificial","First portion of DGAT1 (S205A) in "]},{"entry":[{},{},{},"FIGS. 1, 2, 3, 4 and 5"]},{"entry":["116","Polypeptide","Artificial","Second portion of DGAT1 (S205A) in "]},{"entry":[{},{},{},"FIGS. 1, 2, 3, 4 and 5"]},{"entry":["117","Polynucleotide","Artificial","Sequence of construct in FIG. 2"]},{"entry":["118","Polypeptide","Artificial","First portion of Oleosin 1-1 in FIG. 2"]},{"entry":["119","Polypeptide","Artificial","Second portion of Oleosin 1-1 in FIG. 2"]},{"entry":["120","Polynucleotide","Artificial","Sequence of construct in FIG. 3"]},{"entry":["121","Polypeptide","Artificial","First portion of Oleosin 1-3 in FIG. 3"]},{"entry":["122","Polypeptide","Artificial","Second portion of Oleosin 1-3 in FIG. 3"]},{"entry":["123","Polynucleotide","Artificial","Sequence of construct in FIG. 4"]},{"entry":["124","Polypeptide","Artificial","First portion of Oleosin 3-1 in FIG. 4"]},{"entry":["125","Polypeptide","Artificial","Second portion of Oleosin 3-1 in FIG. 4"]},{"entry":["126","Polynucleotide","Artificial","Sequence of construct in FIG. 5"]},{"entry":["127","Polypeptide","Artificial","First portion of Oleosin 3-3 in FIG. 5"]},{"entry":["128","Polypeptide","Artificial","Second portion of Oleosin 3-3 in FIG. 5"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE FIGURES","p":[{"@attributes":{"id":"p-0264","num":"0296"},"figref":"FIG. 1","sub":"\u2014","i":"A. thaliana "},{"@attributes":{"id":"p-0265","num":"0297"},"figref":"FIG. 2","sub":"\u2014","i":"Arabidopsis thaliana. "},{"@attributes":{"id":"p-0266","num":"0298"},"figref":"FIG. 3","sub":"\u2014","i":"A. thaliana "},{"@attributes":{"id":"p-0267","num":"0299"},"figref":"FIG. 4","sub":"\u2014","i":"A. thaliana "},{"@attributes":{"id":"p-0268","num":"0300"},"figref":"FIG. 5","sub":"\u2014","i":"A. thaliana "},{"@attributes":{"id":"p-0269","num":"0301"},"figref":"FIG. 6","sub":["\u2014","\u2014"],"i":"Arabidopsis thaliana "},{"@attributes":{"id":"p-0270","num":"0302"},"figref":"FIG. 7"},{"@attributes":{"id":"p-0271","num":"0303"},"figref":"FIG. 8","i":"E. coli "},{"@attributes":{"id":"p-0272","num":"0304"},"figref":"FIG. 9","i":"E. coli "},{"@attributes":{"id":"p-0273","num":"0305"},"figref":"FIG. 10","sub":["\u2014","\u2014","\u2014","\u2014"],"i":"Arabidopsis thaliana "},{"@attributes":{"id":"p-0274","num":"0306"},"figref":"FIG. 11","sub":["\u2014","\u2014","\u2014","\u2014"],"i":"Arabidopsis thaliana "},{"@attributes":{"id":"p-0275","num":"0307"},"figref":"FIG. 12","sub":["\u2014","\u2014","\u2014","\u2014"],"i":"Arabidopsis thaliana "},{"@attributes":{"id":"p-0276","num":"0308"},"figref":"FIG. 13","i":"Arabidopsis "},{"@attributes":{"id":"p-0277","num":"0309"},"figref":"FIG. 14","i":"Arabidopsis ","sub":"\u2014"},{"@attributes":{"id":"p-0278","num":"0310"},"figref":"FIG. 15","i":"Arabidopsis ","sub":"\u2014"},{"@attributes":{"id":"p-0279","num":"0311"},"figref":"FIG. 16","i":"Arabidopsis ","sub":"\u2014"},{"@attributes":{"id":"p-0280","num":"0312"},"figref":"FIG. 17","i":"Trifolium repens ","sub":"\u2014"},{"@attributes":{"id":"p-0281","num":"0313"},"figref":"FIG. 18","i":"Trifolium repens ","sub":"\u2014"},{"@attributes":{"id":"p-0282","num":"0314"},"figref":"FIG. 19"},{"@attributes":{"id":"p-0283","num":"0315"},"figref":"FIG. 20","sub":["3","2 "]},"The general equation for photosynthesis by algae and plants (where the electron donor is water) is: 2n CO+2n HO+photons\u21922(CHO)n+2n O",{"@attributes":{"id":"p-0285","num":"0317"},"figref":"FIG. 21"},{"@attributes":{"id":"p-0286","num":"0318"},"figref":"FIG. 22"},{"@attributes":{"id":"p-0287","num":"0319"},"figref":"FIG. 23","sub":"2 "},{"@attributes":{"id":"p-0288","num":"0320"},"figref":"FIG. 24","i":"Arabidopsis "},{"@attributes":{"id":"p-0289","num":"0321"},"figref":"FIG. 25"},{"@attributes":{"id":"p-0290","num":"0322"},"figref":"FIG. 26","sub":["2 ","2 ","2 ","2 "]},{"@attributes":{"id":"p-0291","num":"0323"},"figref":"FIG. 27","sub":"2 "},{"@attributes":{"id":"p-0292","num":"0324"},"figref":"FIG. 28","sub":"2 "},{"@attributes":{"id":"p-0293","num":"0325"},"figref":"FIG. 29","i":"Arabidopsis "},{"@attributes":{"id":"p-0294","num":"0326"},"figref":"FIG. 30","sub":["2 ","\u2014","2 ","\u2014","2 ","\u2014","2 "]},{"@attributes":{"id":"p-0295","num":"0327"},"figref":"FIG. 31","sub":["\u2014","2 ","\u2014","\u2014","2 "]},{"@attributes":{"id":"p-0296","num":"0328"},"figref":"FIG. 32","sub":["\u2014","2 ","\u2014","\u2014","2 "]},{"@attributes":{"id":"p-0297","num":"0329"},"figref":"FIG. 33","sub":["\u2014","\u2014"]},{"@attributes":{"id":"p-0298","num":"0330"},"figref":"FIG. 34","sub":"\u2014"},{"@attributes":{"id":"p-0299","num":"0331"},"figref":"FIG. 35","sub":["\u2014","\u2014"]},{"@attributes":{"id":"p-0300","num":"0332"},"figref":"FIG. 36","ul":{"@attributes":{"id":"ul0012","list-style":"none"},"li":{"@attributes":{"id":"ul0012-0001","num":"0000"},"ul":{"@attributes":{"id":"ul0013","list-style":"none"},"li":["Trace A shows TAG extracted from 100 mg of roots from plants transformed with DGAT1-Ole3,3.","Trace B shows TAG extracted from 100 mg of leaves from plants transformed with DGAT1-Ole3,3","Trace C shows TAG extracted from 100 mg of roots from wild type plants.","Trace D shows TAG extracted from 100 mg of leaves from wild type plants."]}}}}]},"DETDESC":[{},{}]}
