---
title: Differentially expressed microRNA molecules for the treatment and diagnosis of cancer
abstract: A significant challenge in cancer research field is to define molecular features that distinguish cancer stem cells from normal stem cells. In this study, microRNA (miRNA) expression profiles in human glioblastoma stem cells were compared to that of normal neural stem cells using combined microarray and deep sequencing analyzes. These studies led to the identification of several miRNAs that are differentially expressed in glioblastoma stem cells and normal neural stem cells. Characterizing the role of these miRNAs in glioblastoma stem cells is important for the development of miRNA-based therapies that specifically target tumor stem cells, but spare normal stem cells.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09315809&OS=09315809&RS=09315809
owner: CITY OF HOPE
number: 09315809
owner_city: Duarte
owner_country: US
publication_date: 20130829
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS REFERENCE TO RELATED APPLICATIONS","STATEMENT OF GOVERNMENT INTEREST","BACKGROUND","SUMMARY","DETAILED DESCRIPTION","EXAMPLES","Example 1","Identification of a Set of miRNAs that are Differentially Expressed in Glioblastoma Stem Cells and Normal Neural Stem Cells","Example 2","miR-874 Acts as a Tumor Suppressor to Target and Suppress Glioblastoma Stem Cells","REFERENCES"],"p":["This application claims the benefit of U.S. Provisional Patent Application No. 61\/694,698, filed Aug. 29, 2012, which is incorporated herein by reference in its entirety.","The present invention was made with government support under R01-NS059546 and RC1-NS068370, each awarded by the National Institutes of Health and the National Institute of Neurological Disorders and Stroke (NIH NINDS). The Government has certain rights in the invention.","MicroRNAs (miRNAs) are short 20-22 nucleotide RNA molecules that are expressed in a tissue-specific and developmentally-regulated manner and function as negative regulators of gene expression in a variety of eukaryotes. miRNAs are involved in numerous cellular processes including development, proliferation, and differentiation [Ambros 2004; Bartel 2004; Shi et al. 2010]. Increasing evidence has linked miRNAs to cancer [Esquela-Kerscher & Slack 2006] and are important regulators of many key pathways implicated in tumor pathogenesis [Asadi-Moghaddam et al. 2010], functioning as oncogenes or tumor suppressors in various tumors [Cheng et al. 2010].","Although miRNAs have been shown to be differentially expressed in various types of cancer cells as compared to normal cells, many cancers are thought to be maintained by a population of cancer stem cells that retain stem cell properties, are highly tumorigenic, and display increased resistance to radiation and chemotherapy. As such, cancer therapies should target tumor stem cells, but spare normal stem cells. Therefore, there is a need for identifying miRNA molecules that are differentially expressed in tumor stem cell subpopulations as compared to normal stem cells for the development of miRNA-based cancer therapeutics and diagnostics.","One embodiment, methods for treating a cancer are provided. In certain embodiments, such methods include administering a therapeutically effective amount of a pharmaceutical composition to a subject having the cancer, wherein the pharmaceutical composition comprises one or more therapeutic agents which target one or more miRNA molecules that are differentially expressed in cancer stem cells as compared to normal cells. In other embodiments, such methods include contacting a cancer cell or cancer stem cell with one or more miRNA molecules that have or impart at least one tumor suppressor activity.","In another embodiment, methods for diagnosing a cancer are provided. Such methods include detecting a test level of one or more miRNA molecules in a biological sample from a subject; comparing the test level to a reference level; and diagnosing the subject as having the cancer when the test level is significantly different than the reference level.","In another embodiment, an miRNA expression signature is provided that is specific to a cancer stem cell. The miRNA expression signature includes one or more miRNA molecules that are differentially expressed in the cancer stem cell.","According to the embodiments described above, the one or more miRNA molecules may be selected from miR-10a, miR-10b, miR-140-3p, miR-140-5p, miR-204, miR-424, miR-34a, miR-193a-3p, miR-455-5p, miR-455-3p, miR-9, miR-10a, miR-148a, miR-488, miR-196a1, miR-182, miR-96, miR-193b, miR-27a, miR-196b, miR-10b, miR-29b2, miR-23a, miR-107, miR-542-3p, miR-93, miR-365a-4, miR-450a, miR-100, miR-105, miR-363, miR-105, miR-106b, miR-15b, miR-21, miR-376c, miR-93, miR-99b, miR-155, miR-33a, miR-876-3p, miR-362-3p, miR-25, let-7i, miR-423-3p, miR-34b, miR-16-2, -miR-29a, miR-30d, miR-320, miR-181c, miR-128a, miR-21, let-7d, miR-450b-5p, miR-371-5p, miR-1245, miR-335, miR-492, miR-874, miR-30b, miR-193a-5p, miR-602, miR-346, miR-663, miR-25, miR-219-5p6, miR-184, miR-135a7, miR-584, miR-665, miR-638, miR-503, miR-628-3p, miR-381, miR-78, miR-92b, miR-149, miR-135b, miR-302d, miR-498, miR-766, miR-1389, miR-623, miR-519c-5p, miR-182, miR-494, miR-129-5p10, miR-513-5p, miR-200b, miR-634, miR-654-5p, miR-518b, miR-658, miR-373, miR-30c-2, miR-130a, miR-557, miR-551a, miR-637, miR-518c, miR-525-5p, miR-596, miR-552, miR-625, miR-183, miR-187, miR-544, miR-891a, miR-519e, miR-933, miR-939, miR-214, miR-671-5p, miR-137, miR-92b, miR-525-3p, miR-19a, and miR-409-5p.","The embodiments described above may be specific to and may be used to treat or diagnose any applicable cancer including, but not limited to, bone cancer, bladder cancer, brain cancer, breast cancer, cancer of the urinary tract, carcinoma, cervical cancer, colon cancer, esophageal cancer, gastric cancer, head and neck cancer, hepatocellular cancer, liver cancer, lung cancer, lymphoma and leukemia, melanoma, ovarian cancer, pancreatic cancer, pituitary cancer, prostate cancer, rectal cancer, renal cancer, sarcoma, testicular cancer, thyroid cancer, and uterine cancer. In one embodiment, the cancer is brain cancer, in particular, glioblastoma.","Methods for diagnosing and treating cancer using one or more miRNA molecules are provided herein. Such methods may be used to treat or diagnose any cancer or tumor cell type including bone cancer, bladder cancer, brain cancer, breast cancer, cancer of the urinary tract, carcinoma, cervical cancer, colon cancer, esophageal cancer, gastric cancer, head and neck cancer, hepatocellular cancer, liver cancer, lung cancer, lymphoma and leukemia, melanoma, ovarian cancer, pancreatic cancer, pituitary cancer, prostate cancer, rectal cancer, renal cancer, sarcoma, testicular cancer, thyroid cancer, and uterine cancer. In addition, the methods may be used to treat tumors that are malignant (e.g., primary or metastatic cancers) or benign (e.g., hyperplasia, cyst, pseudocyst, hematoma, and benign neoplasm).","In one embodiment, the methods described herein may be used to treat or diagnose a brain cancer including, but not limited to, gliomas (e.g., astrocytomas, oligodiendrogliomas, mixed gliomas, enendymomas, brain stem gliomas) meningiomas, pineal gland and pituitary gland tumors, primary central nervous system lymphomas, medulloblastomas, craniopharyngiomas and acoustic neuromas.","Astrocytomas, the most common type of glioma, have been subclassified into four subtypes (\u201cor grades\u201d) using the World Health Organization (WHO) classification system for tumor identification [Louis et al. 2007]. This grading scheme represents a malignancy scale and is an important factor influencing the choice of therapies. According to the WHO, a grade I astrocytoma is classified as a pilocytic astrocytoma, a grade II astrocytoma is classified as a diffuse astrocytoma, a grade III astrocytoma is classified as an anaplastic astrocytoma, and a grade IV astrocytoma is classified as a glioblastoma\u2014the most malignant grade [Louis et al. 2007]. Glioblastoma is the most common and aggressive primary brain tumor with median survival time of 14 months after diagnosis [Louis et al. 2007]. Currently, no effective treatment has been developed for glioblastoma patients. Therefore, although the miRNA molecules used according to the methods described herein may be used to treat or diagnose any type of cancer, the Examples are primarily directed to the identification of miRNA molecules related to glioblastoma.","Recent studies have suggested that glioblastomas are maintained by a small population of cancer stem cells that retain stem cell properties, are highly tumorigenic, and display increased resistance to radiation and chemotherapy [Singh et al. 2004; Bao et al. 2006; Godlewski et al. 2010]. These treatment-resistant tumor cell subpopulations are the cell populations that effective therapies must target [Godlewski et al. 2010].","miRNAs have been shown to be differentially expressed in glioblastoma tissues compared to normal brain tissues. For example, miRNA 21 is overexpressed in glioblastoma tissues, relative to surrounding normal brain tissues [Conti et al. 2009]. miR-26a is also amplified in glioblastoma tissues. By targeting the tumor suppressor Pten, overexpression of miR-26a facilitates tumorigenesis and predicts a poor survival [Huse et al. 2009; Kim et al. 2010]. On the other hand, miR-124, miR-137 and miR-451 exhibit reduced expression in malignant glioblastoma tissues relative to normal brain tissues [Silber et al. 2008; Gal et al. 2008]. The expression of these miRNAs is also reduced in glioblastoma stem cells relative to bulk tumor cells. Overexpression of these miRNAs in glioblastoma stem cells inhibits cell proliferation and induces neural differentiation, suggesting a tumor suppressor role for these miRNAs. These studies suggest that some miRNAs may be used as therapeutic agents for targeting glioblastoma stem cells. However, brain tumor stem cells share a core developmental program with normal neural stem cells [Cheng et al. 2010]. Optimal therapies should be designed to target tumor stem cells, but spare normal stem cells. Therefore, identifying miRNAs that are differentially expressed in glioblastoma stem cells and normal neural stem cells is important for the development of optimal miRNA-based therapies and diagnostics for glioblastoma patients.","Therefore, in some embodiments, methods for treating cancer (e.g., glioblastoma) include a step of administering a therapeutically effective amount of a pharmaceutical composition that includes one or more therapeutic agents which target and\/or affect the expression level of one or more miRNA molecules that are differentially expressed in cancer stem cells (e.g., glioblastoma stem cells) and\/or other cancer cells as compared to healthy cells or healthy stem cells. A therapeutic agent which targets a target molecule (e.g., a target miRNA molecule that is differentially expressed) means that said therapeutic agent, when administered to a subject or is otherwise exposed to the target molecule, results in an alteration in the expression or activity of the target molecule. Such alteration may include, but is not limited to, inhibition, suppression activation, agonization, or otherwise cause a change in the expression or activity level of the target molecule. The therapeutic agent may directly cause such a change, or may act on upstream or downstream targets that ultimately result in a change to the target molecule.","According to some embodiments, the one or more miRNA molecules that are targeted may include, but are not limited to, miR-10a, miR-10b, miR-140-3p, miR-140-5p, miR-204, miR-424, miR-34a, miR-193a-3p, miR-455-5p, miR-455-3p, miR-9, miR-10a, miR-148a, miR-488, miR-196a1, miR-182, miR-96, miR-193b, miR-27a, miR-196b, miR-10b, miR-29b2, miR-23a, miR-107, miR-542-3p, miR-93, miR-365a-4, miR-450a, miR-100, miR-105, miR-363, miR-105, miR-106b, miR-15b, miR-21, miR-376c, miR-93, miR-99b, miR-155, miR-33a, miR-876-3p, miR-362-3p, miR-25, let-7i, miR-423-3p, miR-34b, miR-16-2, -miR-29a, miR-30d, miR-320, miR-181c, miR-128a, miR-21, let-7d, miR-450b-5p, miR-371-5p, miR-1245, miR-335, miR-492, miR-874, miR-30b, miR-193a-5p, miR-602, miR-346, miR-663, miR-25, miR-219-5p6, miR-184, miR-135a7, miR-584, miR-665, miR-638, miR-503, miR-628-3p, miR-381, miR-78, miR-92b, miR-149, miR-135b, miR-302d, miR-498, miR-766, miR-1389, miR-623, miR-519c-5p, miR-182, miR-494, miR-129-5p10, miR-513-5p, miR-200b, miR-634, miR-654-5p, miR-518b, miR-658, miR-373, miR-30c-2, miR-130a, miR-557, miR-551a, miR-637, miR-518c, miR-525-5p, miR-596, miR-552, miR-625, miR-183, miR-187, miR-544, miR-891a, miR-519e, miR-933, miR-939, miR-214, miR-671-5p, miR-137, miR-92b, miR-525-3p, miR-19a, and miR-409-5p.","In one embodiment, the one or more therapeutic agents target one or more miRNA molecules that are significantly upregulated in cancer stem cells (e.g., glioblastoma stem cells) and\/or other cancer cells as compared to healthy cells or healthy stem cells. The one or more miRNA molecules that are significantly upregulated include, but are not limited to, miR-10a, miR-10b, miR-140-3p, miR-140-5p, miR-204, miR-424, miR-34a, miR-193a-3p, miR-455-5p, miR-455-3p, miR-9, miR-10a, miR-148a, miR-488, miR-196a1, miR-182, miR-96, miR-193b, miR-27a, miR-196b, miR-10b, miR-29b2, miR-23a, miR-107, miR-542-3p, miR-93, miR-365a-4, miR-450a, miR-100, miR-105, miR-363, miR-105, miR-106b, miR-15b, miR-21, miR-376c, miR-93, miR-99b, miR-155, miR-33a, miR-876-3p, miR-362-3p, miR-25, let-7i, miR-423-3p, miR-34b, miR-16-2, -miR-29a, miR-30d, miR-320, miR-181c, miR-128a, miR-21, let-7d, and miR-450b-5p. In one embodiment, the one or more miRNA molecules that are significantly upregulated include, but are not limited to miR-10a, miR-10b, miR-140-3p, miR-140-5p, miR-204, miR-424, miR-455-5p, miR-455-3p.","In another embodiment, the one or more therapeutic agents target one or more miRNA molecules that are significantly downregulated in cancer stem cells (e.g., glioblastoma stem cells) and\/or other cancer cells as compared to healthy cells or healthy stem cells. The one or more miRNA molecules that are significantly downregulated include, but are not limited to, miR-371-5p, miR-1245, miR-335, miR-492, miR-874, miR-30b, miR-193a-5p, miR-602, miR-346, miR-663, miR-25, miR-219-5p6, miR-184, miR-135a7, miR-584, miR-665, miR-638, miR-503, miR-628-3p, miR-381, miR-78, miR-92b, miR-149, miR-135b, miR-302d, miR-498, miR-766, miR-1389, miR-623, miR-519c-5p, miR-182, miR-494, miR-129-5p10, miR-513-5p, miR-200b, miR-634, miR-654-5p, miR-518b, miR-658, miR-373, miR-30c-2, miR-130a, miR-557, miR-551a, miR-637, miR-518c, miR-525-5p, miR-596, miR-552, miR-625, miR-183, miR-187, miR-544, miR-891a, miR-519e, miR-933, miR-939, miR-214, miR-671-5p, miR-137, miR-92b, miR-525-3p, miR-19a, and miR-409-5p. In one embodiment, the one or more miRNA molecules that are significantly downregulated include, but are not limited to hsa-miR-371-5p, hsa-miR-124-1, hsa-miR-124-2, hsa-miR-124-3, hsa-miR-335, hsa-miR-492, hsa-miR-874, hsa-miR-30b, and hsa-miR-602.","In certain embodiments, treating cancer (e.g., glioblastoma) in a subject having the cancer may be accomplished by contacting a cancer cell (e.g., glioblastoma cell) or cancer stem cell (e.g., glioblastoma stem cell) with one or more miRNA molecules that have or impart at least one tumor suppressor activity. In such embodiments, the one or more miRNA molecules act as a therapeutic agent directly. Tumor suppressor activities may include, but are not limited to, suppression or inhibition of cell growth and\/or cell division. In some examples, an miRNA that is found to be significantly down-regulated in cancer cells or cancer stem cells relative to normal stem cells or normal cells may act as a tumor suppressor. Thus, miRNA molecules that have or impart tumor suppressor activities may include, but are not limited to miR-371-5p, miR-1245, miR-335, miR-492, miR-874, miR-30b, miR-193a-5p, miR-602, miR-346, miR-663, miR-25, miR-219-5p6, miR-184, miR-135a7, miR-584, miR-665, miR-638, miR-503, miR-628-3p, miR-381, miR-78, miR-92b, miR-149, miR-135b, miR-302d, miR-498, miR-766, miR-1389, miR-623, miR-519c-5p, miR-182, miR-494, miR-129-5p10, miR-513-5p, miR-200b, miR-634, miR-654-5p, miR-518b, miR-658, miR-373, miR-30c-2, miR-130a, miR-557, miR-551a, miR-637, miR-518c, miR-525-5p, miR-596, miR-552, miR-625, miR-183, miR-187, miR-544, miR-891a, miR-519e, miR-933, miR-939, miR-214, miR-671-5p, miR-137, miR-92b, miR-525-3p, miR-19a, and miR-409-5p. In one embodiment, the one or more miRNA molecules that have or impart tumor suppressor activities may include, but are not limited to, hsa-miR-371-5p, hsa-miR-124-1, hsa-miR-124-2, hsa-miR-124-3, hsa-miR-335, hsa-miR-492, hsa-miR-874, hsa-miR-30b, and hsa-miR-602.","In certain embodiments, the one or more therapeutic agents target at least one miRNA molecules that is significantly upregulated and at least one miRNA molecule that is significantly downregulated in cancer stem cells (e.g., glioblastoma stem cells) and\/or other cancer cells as compared to healthy cells or healthy stem cells. The one or more miRNA molecules that are significantly upregulated include, but are not limited to miR-10a, miR-10b, miR-140-3p, miR-140-5p, miR-204, miR-424, miR-455-5p, miR-455-3p; and the one or more miRNA molecules that are significantly downregulated include, but are not limited to hsa-miR-371-5p, hsa-miR-124-1, hsa-miR-124-2, hsa-miR-124-3, hsa-miR-335, hsa-miR-492, hsa-miR-874, hsa-miR-30b, and hsa-miR-602.","Therapeutic agents that may be used in accordance with the methods described herein may include, but are not limited to, (i) miRNA inhibitors to inhibit or silence specific miRNA molecules that are upregulated in glioblastoma or other cancers; (ii) miRNA inhibitors to sequester endogenous miRNA, thereby blocking the endogenous miRNA function (i.e., the \u201csponge method,\u201d see Ebert et al. 2007, which is hereby incorporated by reference as if fully set forth herein); and (iii) agents that increase the level of specific miRNA molecules, for example, an miRNA that has been identified as a downregulated miRNA molecule to replace the same miRNA that has a lower expression (such as those described above) or an miRNA expression vector that causes a cell to overexpresses such a downregulated miRNA molecule; and (iv) miRNA molecules that can directly replace a downregulated miRNA molecule.","Because the sequences of many miRNA molecules have been previously described (see www.mirbase.org\/), inhibitors specific for one or more upregulated miRNA molecules, such as those described above, may be obtained commercially (e.g., from Thermo Scientific Dharmacon), or may be developed based on designing microRNA hairpin inhibitors, antisense inhibitors (e.g., 2\u2032-O-methyl miRNA antisense RNAs), LNA miRNA inhibitors, RNA interference molecules (e.g., shRNA, sRNA), aptamers, or other suitable complementary miRNA inhibitors.","Delivery of the therapeutic agents described above may be accomplished by any suitable method including, but not limited to, viral vector-delivery (e.g., lentiviral vector delivery, AAV-viral vector delivery, adenoviral vector delivery), dendrimer-mediated delivery, nanoparticle-mediated delivery or a combination thereof (e.g., dendrimer-based nanoparticle delivery).","In addition to the therapeutic agents, the pharmaceutical composition may also include a pharmaceutical carrier. A \u201cpharmaceutically acceptable carrier\u201d refers to a pharmaceutically acceptable material, composition, or vehicle that is involved in carrying or transporting a compound of interest from one tissue, organ, or portion of the body to another tissue, organ, or portion of the body. For example, the carrier may be a liquid or solid filler, diluent, excipient, solvent, or encapsulating material, or some combination thereof. Each component of the carrier must be \u201cpharmaceutically acceptable\u201d in that it must be compatible with the other ingredients of the formulation. It also must be suitable for contact with any tissue, organ, or portion of the body that it may encounter, meaning that it must not carry a risk of toxicity, irritation, allergic response, immunogenicity, or any other complication that outweighs its therapeutic benefits.","The pharmaceutical composition that may be used in accordance with the methods described herein may be administered, by any suitable route of administration, alone or as part of a pharmaceutical composition. A route of administration may refer to any administration pathway known in the art, including but not limited to aerosol, enteral, nasal, ophthalmic, oral, parenteral, rectal, transdermal (e.g., topical cream or ointment, patch), or vaginal. \u201cTransdermal\u201d administration may be accomplished using a topical cream or ointment or by means of a transdermal patch. \u201cParenteral\u201d refers to a route of administration that is generally associated with injection, including infraorbital, infusion, intraarterial, intracapsular, intracardiac, intradermal, intramuscular, intraperitoneal, intrapulmonary, intraspinal, intrasternal, intrathecal, intrauterine, intravenous, subarachnoid, subcapsular, subcutaneous, transmucosal, or transtracheal.","The term \u201ceffective amount\u201d as used herein refers to an amount of a compound that produces a desired effect. For example, a population of cells may be contacted with an effective amount of a compound to study its effect in vitro (e.g., cell culture) or to produce a desired therapeutic effect ex vivo or in vitro. An effective amount of a compound may be used to produce a therapeutic effect in a subject, such as preventing or treating a target condition, alleviating symptoms associated with the condition, or producing a desired physiological effect. In such a case, the effective amount of a compound is a \u201ctherapeutically effective amount,\u201d \u201ctherapeutically effective concentration\u201d or \u201ctherapeutically effective dose.\u201d The precise effective amount or therapeutically effective amount is an amount of the composition that will yield the most effective results in terms of efficacy of treatment in a given subject or population of cells. This amount will vary depending upon a variety of factors, including but not limited to the characteristics of the compound (including activity, pharmacokinetics, pharmacodynamics, and bioavailability), the physiological condition of the subject (including age, sex, disease type and stage, general physical condition, responsiveness to a given dosage, and type of medication) or cells, the nature of the pharmaceutically acceptable carrier or carriers in the formulation, and the route of administration. Further an effective or therapeutically effective amount may vary depending on whether the compound is administered alone or in combination with another compound, drug, therapy or other therapeutic method or modality. One skilled in the clinical and pharmacological arts will be able to determine an effective amount or therapeutically effective amount through routine experimentation, namely by monitoring a cell's or subject's response to administration of a compound and adjusting the dosage accordingly. For additional guidance, see Remington: The Science and Practice of Pharmacy, 21Edition, Univ. of Sciences in Philadelphia (USIP), Lippincott Williams & Wilkins, Philadelphia, Pa., 2005, which is hereby incorporated by reference as if fully set forth herein.","\u201cTreating\u201d or \u201ctreatment\u201d of a condition may refer to preventing the condition, slowing the onset or rate of development of the condition, reducing the risk of developing the condition, preventing or delaying the development of symptoms associated with the condition, reducing or ending symptoms associated with the condition, generating a complete or partial regression of the condition, or some combination thereof. Treatment may also mean a prophylactic or preventative treatment of a condition.","In some embodiments, an miRNA molecule or other biomarker that is either (i) upregulated or overexpressed; or (ii) downregulated or underexpressed can also be referred to as being \u201cdifferentially expressed\u201d as compared to a \u201cnormal\u201d expression level or value of the miRNA molecule or other biomarker that indicates or is a sign of a normal process or an absence of a disease or other condition in an individual. Thus, \u201cdifferential expression\u201d of an miRNA molecule or other biomarker can also be referred to as a variation from a \u201cnormal\u201d expression level of the biomarker. Differential expression includes quantitative, as well as qualitative, differences in the temporal or cellular expression pattern in a gene or its expression products among, for example, normal and diseased cells (e.g., normal stem cells vs. cancer stem cells; normal cells vs. cancer cells, or a combination thereof), or among cells which have undergone different disease events or disease stages.","Further, the phrase \u201cdifferentially expressed\u201d refers to a difference in the quantity or intensity of a marker (e.g., miRNA) present in a biological sample taken from subjects having a cancer as compared to a comparable sample taken from subjects who do not have the cancer. For example, an miRNA molecule is differentially expressed between the samples if the amount of the miRNA molecule in one sample is significantly different (i.e., p<0.05) from the amount of the miRNA molecule in the other sample. It should be noted that if the miRNA molecule or other marker is detectable in one sample and not detectable in the other, then the miRNA molecule can be considered to be differentially present.","The identification of the miRNAs described above and in the Examples below suggests that these miRNAs may be used as novel diagnostic markers. Thus, according to some embodiments, methods for diagnosing cancer (e.g., glioblastoma) include a step of detecting a test level of one or more miRNA molecules in a biological sample from a subject who is suspected of having the cancer.","According to some embodiments, the one or more miRNA molecules that are detected may include, but are not limited to, miR-10a, miR-10b, miR-140-3p, miR-140-5p, miR-204, miR-424, miR-34a, miR-193a-3p, miR-455-5p, miR-455-3p, miR-9, miR-10a, miR-148a, miR-488, miR-196a1, miR-182, miR-96, miR-193b, miR-27a, miR-196b, miR-10b, miR-29b2, miR-23a, miR-107, miR-542-3p, miR-93, miR-365a-4, miR-450a, miR-100, miR-105, miR-363, miR-105, miR-106b, miR-15b, miR-21, miR-376c, miR-93, miR-99b, miR-155, miR-33a, miR-876-3p, miR-362-3p, miR-25, let-7i, miR-423-3p, miR-34b, miR-16-2, -miR-29a, miR-30d, miR-320, miR-181c, miR-128a, miR-21, let-7d, miR-450b-5p, miR-371-5p, miR-1245, miR-335, miR-492, miR-874, miR-30b, miR-193a-5p, miR-602, miR-346, miR-663, miR-25, miR-219-5p6, miR-184, miR-135a7, miR-584, miR-665, miR-638, miR-503, miR-628-3p, miR-381, miR-78, miR-92b, miR-149, miR-135b, miR-302d, miR-498, miR-766, miR-1389, miR-623, miR-519c-5p, miR-182, miR-494, miR-129-5p10, miR-513-5p, miR-200b, miR-634, miR-654-5p, miR-518b, miR-658, miR-373, miR-30c-2, miR-130a, miR-557, miR-551a, miR-637, miR-518c, miR-525-5p, miR-596, miR-552, miR-625, miR-183, miR-187, miR-544, miR-891a, miR-519e, miR-933, miR-939, miR-214, miR-671-5p, miR-137, miR-92b, miR-525-3p, miR-19a, and miR-409-5p.","In one embodiment, the one or more miRNA molecules are significantly upregulated in cancer stem cells (e.g., glioblastoma stem cells) and\/or other cancer cells as compared to healthy cells or healthy stem cells. The one or more miRNA molecules that are significantly upregulated include, but are not limited to, miR-10a, miR-10b, miR-140-3p, miR-140-5p, miR-204, miR-424, miR-34a, miR-193a-3p, miR-455-5p, miR-455-3p, miR-9, miR-10a, miR-148a, miR-488, miR-196a1, miR-182, miR-96, miR-193b, miR-27a, miR-196b, miR-10b, miR-29b2, miR-23a, miR-107, miR-542-3p, miR-93, miR-365a-4, miR-450a, miR-100, miR-105, miR-363, miR-105, miR-106b, miR-15b, miR-21, miR-376c, miR-93, miR-99b, miR-155, miR-33a, miR-876-3p, miR-362-3p, miR-25, let-7i, miR-423-3p, miR-34b, miR-16-2, -miR-29a, miR-30d, miR-320, miR-181c, miR-128a, miR-21, let-7d, and miR-450b-5p. In one embodiment, the one or more miRNA molecules that are significantly upregulated include, but are not limited to miR-10a, miR-10b, miR-140-3p, miR-140-5p, miR-204, miR-424, miR-455-5p, miR-455-3p.","In another embodiment, the one or more miRNA molecules are significantly downregulated in cancer stem cells (e.g., glioblastoma stem cells) and\/or other cancer cells as compared to healthy cells or healthy stem cells. The one or more miRNA molecules that are significantly downregulated include, but are not limited to, miR-371-5p, miR-1245, miR-335, miR-492, miR-874, miR-30b, miR-193a-5p, miR-602, miR-346, miR-663, miR-25, miR-219-5p6, miR-184, miR-135a7, miR-584, miR-665, miR-638, miR-503, miR-628-3p, miR-381, miR-78, miR-92b, miR-149, miR-135b, miR-302d, miR-498, miR-766, miR-1389, miR-623, miR-519c-5p, miR-182, miR-494, miR-129-5p10, miR-513-5p, miR-200b, miR-634, miR-654-5p, miR-518b, miR-658, miR-373, miR-30c-2, miR-130a, miR-557, miR-551a, miR-637, miR-518c, miR-525-5p, miR-596, miR-552, miR-625, miR-183, miR-187, miR-544, miR-891a, miR-519e, miR-933, miR-939, miR-214, miR-671-5p, miR-137, miR-92b, miR-525-3p, miR-19a, and miR-409-5p. In one embodiment, the one or more miRNA molecules that are significantly downregulated include, but are not limited to hsa-miR-371-5p, hsa-miR-124-1, hsa-miR-124-2, hsa-miR-124-3, hsa-miR-335, hsa-miR-492, hsa-miR-874, hsa-miR-30b, and hsa-miR-602.","In certain embodiments, the one or more miRNA molecules may include at least one miRNA molecule that is significantly upregulated and at least one miRNA molecule that is significantly downregulated in cancer stem cells (e.g., glioblastoma stem cells) and\/or other cancer cells as compared to healthy cells or healthy stem cells. The one or more miRNA molecules that are significantly upregulated include, but are not limited to miR-10a, miR-10b, miR-140-3p, miR-140-5p, miR-204, miR-424, miR-455-5p, miR-455-3p; and the one or more miRNA molecules that are significantly downregulated include, but are not limited to hsa-miR-371-5p, hsa-miR-124-1, hsa-miR-124-2, hsa-miR-124-3, hsa-miR-335, hsa-miR-492, hsa-miR-874, hsa-miR-30b, and hsa-miR-602.","According to the embodiments described herein, a biological sample may refer to any material, biological fluid, tissue, or cell obtained or otherwise derived from an individual including, but not limited to, blood (including whole blood, leukocytes, peripheral blood mononuclear cells, buffy coat, plasma, and serum), sputum, tears, mucus, nasal washes, nasal aspirate, breath, urine, semen, saliva, meningeal fluid, amniotic fluid, glandular fluid, lymph fluid, milk, bronchial aspirate, synovial fluid, joint aspirate, cells, a cellular extract, and cerebrospinal fluid. This also includes experimentally separated fractions of all of the preceding. For example, a blood sample can be fractionated into serum or into fractions containing particular types of blood cells, such as red blood cells or white blood cells (leukocytes). If desired, a sample can be a combination of samples from an individual, such as a combination of a tissue and fluid sample. A biological sample may also include a biological tissue sample, such as a sample obtained by tissue biopsy or by surgical excision. A biological sample may also include materials derived from a tissue culture or a cell culture. Further, a biological sample may be derived by taking biological samples from a number of individuals and pooling them or pooling an aliquot of each individual's biological sample. The pooled sample can be treated as a sample from a single individual and if the presence of cancer is established in the pooled sample, then each individual biological sample can be re-tested to determine which individuals have cancer.","A test level of an miRNA molecule or other biomarker refers to an amount of a biomarker, such as an miRNA molecule, in a subject's undiagnosed biological sample. The test level may be compared to that of a control sample, or may be analyzed based on a reference or control level that has been previously established to determine a status of the sample. Such a status may be a diagnosis, prognosis or evaluation of a disease or condition. In one embodiment, the disease is cancer. A test sample or test amount can be either in absolute amount (e.g., nanogram\/mL or microgram\/mL) or a relative amount (e.g., relative intensity of signals).","The test level of the one or more miRNA molecules may be detected by any suitable method of measurement or quantification known in the art including, but not limited to, reverse transcriptase-polymerase chain reaction (RT-PCR) methods (including quantitative and qualitative RT-PCR methods), microarray, serial analysis of gene expression (SAGE), gene expression analysis by massively parallel signature sequencing (MPSS) or deep sequencing methods such as those described below, immunoassays such as ELISA, immunohistochemistry (IHC), mass spectrometry (MS) methods, transcriptomics and proteomics. Several of the detection methods described above include a transformative step, wherein the one or more miRNA molecules that are present in a biological sample are converted or transformed into structural, visual or other tangible manifestation of the amount of miRNA (e.g., a cDNA molecule, a reporter signal, a fluorescent, luminescent or radioactive signal, a labeled antibody, a graph, a tracing).","In some embodiments, the test level of the one or more miRNA molecules may be detected using a set of reagents which contain miRNA detection agents specific to a set of one or more miRNA molecules. The miRNA detection agents may be any suitable molecule that binds to and\/or is complementary to the one or more miRNA molecules including, but not limited to, complementary oligonucleotides, antisense oligonucleotides, and aptamers. The set of reagents maybe used in accordance with any method of measurement or quantification, such as those described above, and\/or may be provided in a kit for detecting the set of one or more miRNA molecules. According to some embodiments, the kit may include, in addition to the set of reagents which contain miRNA detection agents, at least one detection label to produce a detectable or visible signal (e.g., fluorescent labels, dyes, etc.), instructional materials, one or more reference standards, additional agents (e.g., buffers, stabilizers), vessels for storing or transporting the detection agents, or a combination thereof.","According to some embodiments, the test level is compared to a reference level (or a \u201creference standard\u201d) or a control level and the subject may then be diagnosed as having cancer when the test level is significantly different than the reference level or control level. A reference or control level of an miRNA molecule may be any amount or a range of amounts to be compared against the test level. For example, a reference or control level of an miRNA molecule may be the level detected in a population of patients with a specified condition or disease (e.g., malignancy, cancer or non-cancerous lung disease or condition) or the level detected in a control population of individuals without the condition or disease. A control amount can be either in absolute amount (e.g., nanogram\/mL or microgram\/mL) or a relative amount (e.g., relative intensity of signals).","In one embodiment, a test level of an miRNA molecule is considered to be significantly different than a reference or control level if said test level is at least 1.5-fold higher (i.e., upregulated) or lower (i.e., downregulated) as compared to a reference or control level of the miRNA. In some embodiments, the test level is considered to be significantly different than the reference or control level is the test level is at least 5-fold higher (i.e., upregulated) or lower (i.e., downregulated) as compared to the reference or control level. Alternatively, an increase or decrease in an mRNA molecule is typically significantly different if said increase or decrease has a p value of less than 0.5, or less than 0.05 (p<0.5 or p<0.05).","The one or more miRNA molecules described above may be part of an miRNA expression signature or profile that is specific to a cancer stem cell, such as those discussed above. Such an expression signature may be used in accordance with the methods herein to diagnose cancer, or may serve as a target for developing treatments for cancer. In one embodiment, the miRNA expression signature is specific to glioblastoma and includes, but is not limited to, one or more miRNA molecules selected from miR-10a, miR-10b, miR-140-3p, miR-140-5p, miR-204, miR-424, miR-34a, miR-193a-3p, miR-455-5p, miR-455-3p, miR-9, miR-10a, miR-148a, miR-488, miR-196a1, miR-182, miR-96, miR-193b, miR-27a, miR-196b, miR-10b, miR-29b2, miR-23a, miR-107, miR-542-3p, miR-93, miR-365a-4, miR-450a, miR-100, miR-105, miR-363, miR-105, miR-106b, miR-15b, miR-21, miR-376c, miR-93, miR-99b, miR-155, miR-33a, miR-876-3p, miR-362-3p, miR-25, let-7i, miR-423-3p, miR-34b, miR-16-2, -miR-29a, miR-30d, miR-320, miR-181c, miR-128a, miR-21, let-7d, miR-450b-5p, miR-371-5p, miR-1245, miR-335, miR-492, miR-874, miR-30b, miR-193a-5p, miR-602, miR-346, miR-663, miR-25, miR-219-5p6, miR-184, miR-135a7, miR-584, miR-665, miR-638, miR-503, miR-628-3p, miR-381, miR-78, miR-92b, miR-149, miR-135b, miR-302d, miR-498, miR-766, miR-1389, miR-623, miR-519c-5p, miR-182, miR-494, miR-129-5p10, miR-513-5p, miR-200b, miR-634, miR-654-5p, miR-518b, miR-658, miR-373, miR-30c-2, miR-130a, miR-557, miR-551a, miR-637, miR-518c, miR-525-5p, miR-596, miR-552, miR-625, miR-183, miR-187, miR-544, miR-891a, miR-519e, miR-933, miR-939, miR-214, miR-671-5p, miR-137, miR-92b, miR-525-3p, miR-19a, and miR-409-5p.","In another embodiment, the miRNA expression signature includes one or more miRNA molecules that are significantly upregulated in cancer stem cells (e.g., glioblastoma stem cells) and\/or other cancer cells as compared to healthy cells or healthy stem cells. The one or more miRNA molecules that are significantly upregulated include, but are not limited to, miR-10a, miR-10b, miR-140-3p, miR-140-5p, miR-204, miR-424, miR-34a, miR-193a-3p, miR-455-5p, miR-455-3p, miR-9, miR-10a, miR-148a, miR-488, miR-196a1, miR-182, miR-96, miR-193b, miR-27a, miR-196b, miR-10b, miR-29b2, miR-23a, miR-107, miR-542-3p, miR-93, miR-365a-4, miR-450a, miR-100, miR-105, miR-363, miR-105, miR-106b, miR-15b, miR-21, miR-376c, miR-93, miR-99b, miR-155, miR-33a, miR-876-3p, miR-362-3p, miR-25, let-7i, miR-423-3p, miR-34b, miR-16-2, -miR-29a, miR-30d, miR-320, miR-181c, miR-128a, miR-21, let-7d, and miR-450b-5p. In one embodiment, the one or more miRNA molecules that are significantly upregulated include, but are not limited to miR-10a, miR-10b, miR-140-3p, miR-140-5p, miR-204, miR-424, miR-455-5p, miR-455-3p.","In another embodiment, the miRNA expression signature includes one or more miRNA molecules that are significantly downregulated in cancer stem cells (e.g., glioblastoma stem cells) and\/or other cancer cells as compared to healthy cells or healthy stem cells. The one or more miRNA molecules that are significantly downregulated include, but are not limited to, miR-371-5p, miR-1245, miR-335, miR-492, miR-874, miR-30b, miR-193a-5p, miR-602, miR-346, miR-663, miR-25, miR-219-5p6, miR-184, miR-135a7, miR-584, miR-665, miR-638, miR-503, miR-628-3p, miR-381, miR-78, miR-92b, miR-149, miR-135b, miR-302d, miR-498, miR-766, miR-1389, miR-623, miR-519c-5p, miR-182, miR-494, miR-129-5p10, miR-513-5p, miR-200b, miR-634, miR-654-5p, miR-518b, miR-658, miR-373, miR-30c-2, miR-130a, miR-557, miR-551a, miR-637, miR-518c, miR-525-5p, miR-596, miR-552, miR-625, miR-183, miR-187, miR-544, miR-891a, miR-519e, miR-933, miR-939, miR-214, miR-671-5p, miR-137, miR-92b, miR-525-3p, miR-19a, and miR-409-5p. In one embodiment, the one or more miRNA molecules that are significantly downregulated include, but are not limited to hsa-miR-371-5p, hsa-miR-124-1, hsa-miR-124-2, hsa-miR-124-3, hsa-miR-335, hsa-miR-492, hsa-miR-874, hsa-miR-30b, and hsa-miR-602.","In certain embodiments, the miRNA expression signature includes at least one miRNA molecule that is significantly upregulated and at least one miRNA molecule that is significantly downregulated in cancer stem cells (e.g., glioblastoma stem cells) and\/or other cancer cells as compared to healthy cells or healthy stem cells. miRNA molecules that are significantly upregulated include, but are not limited to miR-10a, miR-10b, miR-140-3p, miR-140-5p, miR-204, miR-424, miR-455-5p, miR-455-3p; and miRNA molecules that are significantly downregulated include, but are not limited to hsa-miR-371-5p, hsa-miR-124-1, hsa-miR-124-2, hsa-miR-124-3, hsa-miR-335, hsa-miR-492, hsa-miR-874, hsa-miR-30b, and hsa-miR-602.","The following examples are intended to illustrate various embodiments of the invention. As such, the specific embodiments discussed are not to be construed as limitations on the scope of the invention. It will be apparent to one skilled in the art that various equivalents, changes, and modifications may be made without departing from the scope of invention, and it is understood that such equivalent embodiments are to be included herein. Further, all references cited in the disclosure are hereby incorporated by reference in their entirety, as if fully set forth herein.","The Examples described below show the results of a genome-wide miRNA expression profiling study in human glioblastoma stem cells and normal neural stem cells using combined miRNA microarray and deep sequencing analyses. This study led to the identification of eight miRNAs that are substantially up-regulated and two miRNAs that are significantly down-regulated in glioblastoma stem cells, relative to normal neural stem cells. Differential expression of four of these miRNAs, 2 up-regulated and 2 down-regulated, was further validated by real-time RT-PCR in both glioblastoma stem cells and glioblastoma patient tumor tissues. Moreover, it was demonstrated that these up-regulated or down-regulated miRNAs inhibit the expression of genes that are involved in tumor suppression or tumorigenesis, respectively.","Materials and Methods","Ethics Statement.","The derivation of PBT003 (GSC1) and PBT017 (GSC2) has been described by Brown et al [Brown et al. 2009]. PBT707 (GSC3) is a de novo cell line derived from anonymized leftover tissues with the approval of the City of Hope Institutional Review Board. The study involves the use of completely anonymized specimens. No informed consent is involved. NOD-scid IL2Rgamma(NSG) mice (6-8 weeks) were used for glioblastoma stem cell transplantation. Tumor cell transplantation was performed under an IACUC protocol approved by the City of Hope Institutional Animal Care and Use Committee.","Glioblastoma Stem Cell and Neural Stem Cell Culture.","Glioblastoma stem cells were derived from newly diagnosed WHO grade IV glioblastoma tissues. Specifically, freshly isolated glioblastoma tissues were minced with sterile scissors and dissociated into single cells using 400 units\/ml of collagenase III in DMEM\/F12 medium supplemented with 5 \u03bcg\/ml heparin, 1\u00d7B27 (GIBCO\/BRL), and 2 mM L-glutamine. Dissociated cells were then centrifuged at 1,200 rpm for 5 min and the supernatant was discarded. To eliminate red blood cells, the resultant cells were incubated in 10 ml red blood cell lysis buffer (Invitrogen) for 10 min. Cells were centrifuged again at 1,200 rpm for 5 min and supernatant was discarded. The resultant cells were resuspended in DMEM\/F12 medium supplemented with 20 ng\/ml EGF, 20 ng\/ml FGF, 5 \u03bcg\/ml heparin, 1\u00d7B27 (GIBCO\/BRL), and 2 mM L-glutamine and cultured in this medium thereafter. Tumor spheres appeared around one week in culture. Normal human neural stem cells were derived from primary human brain tissues and maintained in the same culture media. Specifically, human fetal brain tissues (Biosciences Resources) were dissociated in cold Hanks balance salt solution (HBSS) using polished glass pipette. The resultant cells were centrifuged and resuspended in DMEM\/F12 medium supplemented with 0.5\u00d7B27, 25 \u03bcg\/ml insulin, 20 \u03bcg\/ml apo-transferrin, 30 nM sodium selenite, 20 nM progesterone, 100 mM putrescine, 20 ng\/ml FGF and 10 ng\/ml LIF. The initial culture was split at 1:2 each day for 4 days, followed by media change every other day till day 21. Human neurospheres started to appear around day 14. The spheres were split around day 21 with Accutase (Sigma) and cultured in DMEM\/F12 medium supplemented with 20 ng\/ml EGF, 20 ng\/ml FGF, 5 \u03bcg\/ml heparin, 1\u00d7B27 (GIBCO\/BRL), and 2 mM L-glutamine thereafter. Both tumor spheres and normal neurospheres were characterized for their self-renewal and multipotency. Glioblastoma stem cell spheres were also characterized for their ability to derive brain tumors.","For differentiation, both glioblastoma stem cells and neural stem cells were induced into differentiation using 0.5% fetal bovine serum and 1 \u03bcM all-trans retinoic acid. For in vivo tumor formation assays, 2\u00d710dissociated glioblastoma stem cells were injected into cerebral cortex of NSG mice by stereotaxic injection. The coordinates for the injection were AP 0.6 mm, ML +1.6 mm and DV \u22122.6 mm. Brains were harvested 5 weeks after cell transplantation. Frozen brains were cut into 20 \u03bcm coronal sections, followed by Hematoxylin & Eosin (H&E) staining.","Glioblastoma Stem Cell Transfection Using a Dendrimer-Based Delivery System.","Spheres of glioblastoma stem cells were dissociated and seeded into 24-well plates at 2\u00d710cells per well in 300 \u03bcl of medium. The generation-5 (G5) dendrimers and Opti-MEM solution were mixed by vortex for 10 seconds, and incubated at room temperature (RT) for 10 min. The miRNA duplexes or the combination of miRNAs and their short hairpin RNA inhibitors were added into dendrimer\/Opti-MEM solution in a total volume of 100 \u03bcl, mixed gently for 10 sec, and incubated at RT for 25 min. The nitrogen-to-phosphorus (N\/P) ratio of the dendrimer\/RNA complex is 5. The 100 \u03bcl dendrimer\/RNA complex was added into 300 \u03bcl cell suspension in each well of 24-well plates, shake gently and put back to COincubator. Forty-eight hr after transfection, cells were collected and subjected to Western blot analysis.","Western Blot Analysis.","Whole cell extracts of glioblastoma stem cells were prepared using RIPA buffer (50 mM Tris-HCl pH 7.5, 150 mM NaCl, 1% NP40, 0.5% deoxycholate and 0.1% SDS) containing protease inhibitor cocktail (Roche). Western blotting was performed with anti-NRAS (sc-31, 1:100) and anti-PIM3 (sc-98959, 1:100) antibodies from Santa Cruz.","Glioblastoma Stem Cell Transplantation.","NSG mice (6-8 weeks) were used for glioblastoma stem cell transplantation. Tumor cell transplantation was performed under the IACUC protocol 05050 approved by the City of Hope Institutional Animal Care and Use Committee. 5\u00d710dissociated glioblastoma stem cells were injected into the front lobe of forebrains by stereotaxic injection. The coordinates for the injection were AP 0.6 mm, ML +1.6 mm and DV \u22122.6 mm.","Reporter Construct Preparation.","DNA fragments encoding the 3\u2032 UTR of putative miRNA targets were cloned into psiCHECK 2 (Promega), downstream of a luciferase reporter gene. The PCR primers that were used for 3\u2032 UTR cloning of each gene are as follows: CSMD1 forward: 5\u2032 GAT CCT CGA GCT GTT CTG TCG CAG AAT G 3\u2032 (SEQ ID NO: 13) and CSMD1 reverse: 5\u2032 GAT CGC GGC CGC GTC AGC ATT TTG CAC CTA G3\u2032 (SEQ ID NO: 14); PIM3 forward: 5\u2032 GAT CCT CGA GGC TTG TGA GGA GCT GCA C 3\u2032 (SEQ ID NO: 15) and PIM3 reverse: 5\u2032 GAT CGC GGC CGC GGA AAC TTG TCA GGT CAC C 3\u2032 (SEQ ID NO: 16); NRAS forward: 5\u2032 GAT CCT CGA GCT GGA GGA GAA GTA TTC CTG 3\u2032 (SEQ ID NO: 17) and NRAS reverse: 5\u2032 GAT CGC GGC CGC TGC AAA TGT AGA GCT TTC TGG 3\u2032 (SEQ ID NO: 18). Corresponding miRNA binding sites on the 3\u2032 UTRs were mutated by site-directed mutagenesis according to the manufacturer's instructions (Stratagene). The binding site of hsa-miR-124 on the 3\u2032 UTR of PIM3 and NRAS was mutated from GTGCCTT to GTG; the binding site of hsa-miR-10b on the 3\u2032 UTR of CSMD1 was mutated from ACAGGGT to ACAG.","Transfection and Reporter Assay.","Plasmid DNA or DNA-miRNA mixture was transfected into HEK293 cells using Transfectin (Bio-Rad) as described [Zhao et al. 2009; Zhao et al. 2010; Sun et al. 2011]. miR-10a, miR-10b or miR-124 RNA duplexes and\/or their correspondent RNA inhibitors (Dharmacon) were mixed in 50 \u03bcl serum free media with Transfectin, incubated at RT for 20 min. Negative controls for miRNA and their hairpin inhibitors were included. The final concentration of miRNAs or their inhibitors was 20 nM. The resultant mixture was added dropwise to HEK293 cells in a 24-well plate with 450 \u03bcl medium per well to a total volume of 500 \u03bcl per well. The transfected cells were harvested 48 h after transfection and subjected to subsequent reporter assays as described [Sun et al. 2007]. Reporter luciferase activity was measured 48 hrs after transfection using Dual Luciferase Assay kit (Promega). The luciferase activity was normalized by firefly luciferase internal control and expressed as relative luciferase activity. The miR-10a RNA duplex sense sequence is 5\u2032 TAC CCT GTA GAT CCG AAT TTG TG 3\u2032 (SEQ ID NO: 19). The miR-10b RNA duplex sense sequence is 5\u2032 TAC CCT GTA GAA CCG AAT TTG TG 3\u2032 (SEQ ID NO: 20). The miR-124 RNA duplex sense sequence is 5\u2032 TAA GGC ACG CGG TGA ATG CC 3\u2032 (SEQ ID NO: 21). And the control RNA duplex sense sequence is 5\u2032 UCA CAA CCU CCU AGA AAG AGU AGA 3\u2032 (SEQ ID NO: 22).","Real-Time RT-PCR Analysis.","For miRNA expression, total RNAs were reversely transcribed and quantified by real-time RT-PCR with TaqMan MicroRNA Assay kit (Applied Biosystems). The expression of specific miRNAs was normalized using human U18 snRNA. For mRNA expression, putative miRNA targets were quantified by iTaq SYBR Green Supermix with ROX (Bio-Rad). Primers used for RT-PCR include PIM3 forward: 5\u2032 AGC TCA AGC TCA TCG ACT TC 3\u2032 (SEQ ID NO: 23) and PIM3 reverse: 5\u2032 TAG CGG TGG TAG CGG ATC 3\u2032 (SEQ ID NO:24); NRAS forward: 5\u2032 CCA TGA GAG ACC AAT ACA TGA G 3\u2032 (SEQ ID NO: 25) and NRAS reverse: 5\u2032 GCT TAA TCT GCT CCC TGT AG 3\u2032 (SEQ ID NO: 26); HOXD10 forward: 5\u2032 TTC CCG AAG AGA GGA GCT G 3\u2032 (SEQ ID NO: 27) and HOXD10 reverse: 5\u2032 CTG CCA CTC TTT GCA GTG AG 3\u2032 (SEQ ID NO: 28); CSMD1 forward: 5\u2032 GCA GAA ATG CTT ACT GAG GAT G 3\u2032 (SEQ ID NO: 29) and CSMD1 reverse: 5\u2032 AGA ACC CTC AAA CTG CAA CTG 3\u2032 (SEQ ID NO: 30); GAPDH forward: 5\u2032 ATC ACC ATC TTC CAG GAG C 3\u2032 (SEQ ID NO: 31) and GAPDH reverse 5\u2032 CCT TCT CCA TGG TGG TGA AG 3\u2032 (SEQ ID NO: 32).","miRNA Microarray and Deep Sequencing Analysis.","Total RNAs were extracted from glioblastoma stem cells or human neural stem cells by TRIzol (Invitrogen) method according to manufacturer's protocol. Ten \u03bcg of RNA was used for miRNA microarray using Exiqon platform. One \u03bcg of RNA was used for deep sequencing using Illumina Genome Analyser II (GAII). All data are MIAME compliant.","Pathway Analysis.","Common putative targets of either the down-regulated or the up-regulated miRNAs were uploaded onto the Database for Annotation, Visualization and Integrated Discovery (DAVID) Functional Annotation Bioinformatics Microarray Analysis (david.abcc.ncifcrf.gov\/). Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway in DAVID was used to depict the biological meanings of the common miRNA targets.","Results","Differential miRNA Expression in Glioblastoma Stem Cells and Normal Neural Stem Cells.","To identify miRNAs that are differentially expressed in glioblastoma stem cells and normal neural stem cells, three primary glioblastoma stem cell lines and three normal human neural stem cell lines were established to determine if miRNA expression was significantly different in tumor stem cells as compared to normal stem cells. Human primary glioblastoma stem cells were derived from newly diagnosed glioblastoma multiforme IV patients and cultured in DMEM\/F12 media supplemented with epithelial growth factor (EGF), fibroblast growth factor (FGF), and B27 supplement. Human normal neural stem cells were derived from normal human brain tissues and cultured in the same media. Both glioblastoma stem cells and normal neural stem cells grew as neurospheres under the culture conditions (). Both types of cells are multipotent, having the ability to differentiate into Tuj1-positive neurons and\/or GFAP-positive astrocytes when induced into differentiation using fetal bovine serum and all-trans retinoic acid (). However, the glioblastoma stem cells were able to generate tumors when transplanted to the immunodeficient NSG mice (), whereas the human neural stem cells did not (data not shown).","Combined microarray and deep sequencing analyses were performed to determine the expression profile of miRNAs in glioblastoma stem cells and normal neural stem cells. Total RNAs were prepared from both glioblastoma stem cells and neural stem cells for miRNA microarray analysis. In microarray analysis, 10 miRNAs were identified as having more than a 5-fold up-regulation in expression in glioblastoma stem cells and 8 miRNAs were identified as having more than a 5-fold down-regulation in expression in glioblastoma stem cells, relative to neural stem cells (Table 1). The differentially expressed miRNAs that exhibit more than 1.5-fold difference in the expression between glioblastoma stem cells and neural stem cells are shown in Table 2 below.",{"@attributes":{"id":"p-0081","num":"0080"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Up-regulated and down-regulated miRNAs in human glioblastoma stem"},{"entry":"cells, compared to human neural stem cells."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Chromosomal",{},{}]},{"entry":["miRNA","location","Fold-Change","p-value"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["up-regulated",{},{},{}]},{"entry":["hsa-miR-10a","17q21.32","93.65","3.28E\u221210"]},{"entry":["hsa-miR-10b","2q31.1","90.38","2.06E\u221209"]},{"entry":["hsa-miR-140-3p","16q22.1","14.10","1.62E\u221210"]},{"entry":["hsa-miR-140-5p","16q22.1","12.19","5.56E\u221209"]},{"entry":["hsa-miR-204","9q21.12","9.05","4.08E\u221208"]},{"entry":["hsa-miR-424","Xq26.3","8.38","2.53E\u221208"]},{"entry":["hsa-miR-455-5p","9q32","5.87","1.48E\u221205"]},{"entry":["hsa-miR-455-3p","9q32","5.41","9.32E\u221205"]},{"entry":"down-regulated"},{"entry":["hsa-miR-371-5p","19q13.42","\u221215.27","8.52E\u221211"]},{"entry":["hsa-miR-124-1*","8p23.1"]},{"entry":["hsa-miR-124-2*","8q12.3","\u221213.37","8.33E\u221205"]},{"entry":["hsa-miR-124-3*","20q13.33"]},{"entry":["hsa-miR-335","7q32.2","\u221213.24","1.43E\u221209"]},{"entry":["hsa-miR-492","12q22","\u22127.77","5.38E\u221208"]},{"entry":["hsa-miR-874","5q31.2","\u22126.76","1.53E\u221208"]},{"entry":["hsa-miR-30b*","8q24.22","\u22126.54","2.73E\u221209"]},{"entry":["hsa-miR-602","9q34.3","\u22125.75","2.63E\u221208"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":"*hsa-miR-124 is transcribed from three chromosomal locations, but the mature sequences are the same."}]}}]}}},{"@attributes":{"id":"p-0082","num":"0081"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Up-regulated and down-regulated miRNAs (>1.5 fold) in human"},{"entry":"glioblastoma stem cells, compared to human neural stem cells."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Chromosomal",{},{}]},{"entry":["miRNA","location","Fold-Change","p-value"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Up-regulated",{},{},{}]},{"entry":["hsa-miR-10a","17q21.32","93.64622","3.28E\u221210"]},{"entry":["hsa-miR-10b","2q31.1","90.38293","2.06E\u221209"]},{"entry":["hsa-miR-140-3p","16q22.1","14.10262","1.62E\u221210"]},{"entry":["hsa-miR-140-5p","16q22.1","12.19325","5.56E\u221209"]},{"entry":["hsa-miR-204","9q21.12","9.05353","4.08E\u221208"]},{"entry":["hsa-miR-424","Xq26.3","8.38382","2.53E\u221208"]},{"entry":["hsa-miR-34a","1p36.22","7.73283","2.21E\u221207"]},{"entry":["hsa-miR-193a-3p","17q11.2","6.39914","7.95E\u221206"]},{"entry":["hsa-miR-455-5p","9q32","5.87119","1.48E\u221205"]},{"entry":["hsa-miR-455-3p","9q32","5.40680","9.32E\u221205"]},{"entry":["hsa-miR-9*","1q22","4.43204","8.28E\u221208"]},{"entry":["hsa-miR-10a*","17q21.32","3.89600","1.92E\u221205"]},{"entry":["hsa-miR-148a","7p15.2","3.16202","1.93E\u221205"]},{"entry":["hsa-miR-488","1q25.2","2.77759","1.69E\u221203"]},{"entry":["hsa-miR-196a","17q21.32","2.76521","2.69E\u221203"]},{"entry":["hsa-miR-182","7q32.2","2.75689","1.52E\u221203"]},{"entry":["hsa-miR-96","7q32.2","2.61400","1.61E\u221203"]},{"entry":["hsa-miR-193b","16p13.12","2.57981","4.90E\u221206"]},{"entry":["hsa-miR-27a","19p13.13","2.53543","1.73E\u221207"]},{"entry":["hsa-miR-196b","7p15.2","2.50868","3.54E\u221203"]},{"entry":["hsa-miR-10b*","2q31.1","2.40614","8.55E\u221203"]},{"entry":["hsa-miR-29b","7q32.3","2.39299","3.27E\u221207"]},{"entry":["hsa-miR-23a","19p13.13","2.34219","9.88E\u221208"]},{"entry":["hsa-miR-107","10q23.31","2.31374","4.42E\u221207"]},{"entry":["hsa-miR-542-3p","Xq26.3","2.28873","4.59E\u221203"]},{"entry":["hsa-miR-9","1q22","2.24593","1.27E\u221207"]},{"entry":["hsa-miR-365a","16p13.12","2.20050","2.54E\u221206"]},{"entry":["hsa-miR-450a","Xq26.3","2.11076","1.43E\u221202"]},{"entry":["hsa-miR-100","11q24.1","2.04624","5.71E\u221207"]},{"entry":["hsa-miR-105","Xq28","1.99529","3.86E\u221203"]},{"entry":["hsa-miR-363","Xq26.2","1.99346","1.23E\u221202"]},{"entry":["hsa-miR-105*","4q24","1.88156","1.57E\u221202"]},{"entry":["hsa-miR-106b","7q22.1","1.86896","5.63E\u221206"]},{"entry":["hsa-miR-15b","3q25.33","1.79677","3.00E\u221206"]},{"entry":["hsa-miR-21","17q23.1","1.76837","3.03E\u221205"]},{"entry":["hsa-miR-376c","14q32.31","1.76028","5.72E\u221204"]},{"entry":["hsa-miR-93","7q22.1","1.74398","6.15E\u221206"]},{"entry":["hsa-miR-99b","19q13.41","1.73839","1.77E\u221205"]},{"entry":["hsa-miR-155","21q21.3","1.72212","1.59E\u221202"]},{"entry":["hsa-miR-33a","22q13.2","1.72056","8.82E\u221205"]},{"entry":["hsa-miR-876-3p","9p21.1","1.68669","4.45E\u221202"]},{"entry":["hsa-miR-362-3p","Xp11.23","1.67653","4.12E\u221202"]},{"entry":["hsa-miR-25","7q22.1","1.66555","1.31E\u221204"]},{"entry":["hsa-let-7i","12q14.1","1.66413","1.12E\u221205"]},{"entry":["hsa-miR-423-3p","17q11.2","1.64838","2.50E\u221204"]},{"entry":["hsa-miR-34b","11q23.1","1.62786","9.72E\u221205"]},{"entry":["hsa-miR-16-2*","3q25.33","1.62586","1.81E\u221203"]},{"entry":["hsa-miR-29a","7q32.3","1.61583","9.27E\u221206"]},{"entry":["hsa-miR-30d","8q24.2","1.61189","9.38E\u221204"]},{"entry":["hsa-miR-320","8p21.3","1.60864","8.71E\u221205"]},{"entry":["hsa-miR-181c","19p13.13","1.56262","1.30E\u221202"]},{"entry":["hsa-miR-128a","2q21.3","1.55521","4.20E\u221202"]},{"entry":["hsa-miR-21*","17q23.1","1.54998","2.94E\u221202"]},{"entry":["hsa-let-7d","9q22.32","1.53430","1.10E\u221203"]},{"entry":["hsa-miR-450b-5p","Xq26.3","1.53397","3.49E\u221202"]},{"entry":"Down-regulated"},{"entry":["hsa-miR-371-5p","19q13.42","\u221215.26715","8.52E\u221211"]},{"entry":["hsa-miR-124","8p23.1","\u221213.37236","8.33E\u221205"]},{"entry":["hsa-miR-335","7q32.2","\u221213.23553","1.43E\u221209"]},{"entry":["hsa-miR-492","12q22","\u22127.76626","5.38E\u221208"]},{"entry":["hsa-miR-874","5q31.2","\u22126.76244","1.53E\u221208"]},{"entry":["hsa-miR-30b*","8q24.22","\u22126.54116","2.73E\u221209"]},{"entry":["hsa-miR-193a-5p","17q11.2","\u22125.76493","2.38E\u221208"]},{"entry":["hsa-miR-602","9q34.3","\u22125.74746","2.63E\u221208"]},{"entry":["hsa-miR-346","10q23.2","\u22125.69712","4.97E\u221202"]},{"entry":["hsa-miR-663","20p11.1","\u22125.37200","1.31E\u221202"]},{"entry":["hsa-miR-25*","7q22.1","\u22124.97894","2.47E\u221206"]},{"entry":["hsa-miR-219-5p","6p21.32","\u22124.91797","5.92E\u221207"]},{"entry":["hsa-miR-184","15q25.1","\u22124.87568","4.42E\u221208"]},{"entry":["hsa-miR-135a","3p21.1","\u22124.87064","1.94E\u221207"]},{"entry":["hsa-miR-584","5q32","\u22124.60287","1.77E\u221208"]},{"entry":["hsa-miR-665","14q32.2","\u22124.35369","7.57E\u221209"]},{"entry":["hsa-miR-638","19p13.2","\u22123.47384","3.88E\u221204"]},{"entry":["hsa-miR-503","Xq26.3","\u22123.43502","1.20E\u221208"]},{"entry":["hsa-miR-628-3p","15q21.3","\u22123.42905","1.71E\u221207"]},{"entry":["hsa-miR-381","14q32.31","\u22123.33814","2.55E\u221207"]},{"entry":["hsa-miR-7","9q21.32","\u22122.91031","4.96E\u221204"]},{"entry":["hsa-miR-92b","1q22","\u22122.90543","2.73E\u221207"]},{"entry":["hsa-miR-149*","2q37.3","\u22122.87318","1.44E\u221203"]},{"entry":["hsa-miR-135b","1q32.1","\u22122.87250","4.47E\u221207"]},{"entry":["hsa-miR-302d*","4q25","\u22122.77368","2.82E\u221203"]},{"entry":["hsa-miR-498","19q13.42","\u22122.75015","2.70E\u221203"]},{"entry":["hsa-miR-766","Xq24","\u22122.49979","1.89E\u221203"]},{"entry":["hsa-miR-138","3p21.32","\u22122.48259","9.79E\u221207"]},{"entry":["hsa-miR-623","13q32.3","\u22122.43864","3.92E\u221203"]},{"entry":["hsa-miR-519c-5p","19q13.42","\u22122.38133","1.61E\u221205"]},{"entry":["hsa-miR-182*","7q32.2","\u22122.26786","4.18E\u221202"]},{"entry":["hsa-miR-494","14q32.31","\u22122.15454","3.51E\u221206"]},{"entry":["hsa-miR-129-5p","7q32.1","\u22122.13428","6.47E\u221204"]},{"entry":["hsa-miR-513-5p","11q23.1","\u22122.12005","9.55E\u221203"]},{"entry":["hsa-miR-200b*","1p36.33","\u22122.04703","5.69E\u221205"]},{"entry":["hsa-miR-634","17q24.2","\u22122.02966","6.14E\u221206"]},{"entry":["hsa-miR-654-5p","14q32.31","\u22122.01663","7.48E\u221204"]},{"entry":["hsa-miR-518b","19q13.42","\u22121.98208","1.05E\u221203"]},{"entry":["hsa-miR-658","22q13.1","\u22121.94699","1.80E\u221206"]},{"entry":["hsa-miR-373*","19q13.42","\u22121.91189","4.25E\u221203"]},{"entry":["hsa-miR-30c-2*","6q13","\u22121.88988","2.12E\u221206"]},{"entry":["hsa-miR-130a","11q12.1","\u22121.87039","3.78E\u221205"]},{"entry":["hsa-miR-557","1q24.2","\u22121.83994","1.62E\u221203"]},{"entry":["hsa-miR-551a","1p36.32","\u22121.82115","2.53E\u221203"]},{"entry":["hsa-miR-637","19p13.3","\u22121.81083","2.07E\u221202"]},{"entry":["hsa-miR-518c*","19q13.42","\u22121.77801","4.15E\u221205"]},{"entry":["hsa-miR-525-5p","19q13.42","\u22121.75943","4.76E\u221202"]},{"entry":["hsa-miR-596","8p23.3","\u22121.74891","1.75E\u221203"]},{"entry":["hsa-miR-552","1p34.3","\u22121.72726","7.31E\u221204"]},{"entry":["hsa-miR-625*","14q23.3","\u22121.71695","7.00E\u221204"]},{"entry":["hsa-miR-183*","7q32.2","\u22121.70781","1.41E\u221203"]},{"entry":["hsa-miR-187*","18q12.2","\u22121.70468","1.12E\u221202"]},{"entry":["hsa-miR-544","14","\u22121.69268","4.74E\u221202"]},{"entry":["hsa-miR-891a","Xq27.3","\u22121.67598","1.29E\u221202"]},{"entry":["hsa-miR-519e*","19q13.42","\u22121.67235","1.56E\u221202"]},{"entry":["hsa-miR-933","2q31.1","\u22121.66767","5.19E\u221205"]},{"entry":["hsa-miR-939","8q24.3","\u22121.66214","5.40E\u221203"]},{"entry":["hsa-miR-214","1q24.3","\u22121.64500","6.92E\u221203"]},{"entry":["hsa-miR-671-5p","7q36.1","\u22121.64192","7.94E\u221205"]},{"entry":["hsa-miR-137","1p21.3","\u22121.63014","3.96E\u221202"]},{"entry":["hsa-miR-92b*","1q22","\u22121.54966","4.57E\u221202"]},{"entry":["hsa-miR-525-3p","19q13.42","\u22121.54729","1.45E\u221202"]},{"entry":["hsa-miR-19a","13q31.3","\u22121.51239","1.90E\u221204"]},{"entry":["hsa-miR-409-5p","14q32.31","\u22121.51024","5.12E\u221203"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"left","id":"FOO-00002"},"sup":"1-10"}}]}}]}}},"Using an Illumina Genome Analyzer II (GAII) sequencing system, whole-genome small RNA sequencing was performed in glioblastoma stem cells and neural stem cells. Significantly more miRNAs were detected to be differentially expressed in glioblastoma stem cells and neural stem cells in deep sequencing analysis. For example, deep sequencing analysis revealed 105 miRNAs that were up-regulated more than 5-fold in glioblastoma stem cells. However, microarray analysis revealed only 10 miRNAs showing more than 5-fold increase in glioblastoma stem cells. Interestingly, 8 out of the 10 miRNAs that were up-regulated more than 5-fold in microarray analysis also exhibited significantly increased expression in deep sequencing analysis (Table 3). Two of the miRNAs that had more than 5-fold decrease of expression in microarray analysis also showed more than 5-fold reduction of expression in deep-sequencing analysis (Table 3). Taken together, the combined results of the microarray and deep sequencing analyses identified a set of miRNAs that are differentially expressed in glioblastoma stem cells as compared to normal neural stem cells.",{"@attributes":{"id":"p-0084","num":"0083"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"The miRNA signature of glioblastoma stem cells identified using both"},{"entry":"microarray and deep sequencing analyses."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"77pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Microarray",{}]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Fold",{},"Deep sequencing"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"change","p value","Fold change","p value"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Up-regulated",{},{},{},{}]},{"entry":["hsa-miR-10a","93.65","\u2002\u2009<1E\u221207","35,949","0.00E+00"]},{"entry":["hsa-miR-10b","90.38","\u2002\u2009<1E\u221207","4,128","0.00E+00"]},{"entry":["hsa-miR-140-5p","12.19","\u2002\u2009<1E\u221207","7.2","0.00E+00"]},{"entry":["hsa-miR-204","9.05","\u2002\u2009<1E\u221207","5","0.00E+00"]},{"entry":["hsa-miR-424","8.38","\u2002\u2009<1E\u221207","66","0.00E+00"]},{"entry":["hsa-miR-34a","7.73","2.00E\u221207","2.5","4.00E\u2212240"]},{"entry":["hsa-miR-193a-3p","6.4","8.00E\u221206","93","9.00E\u221221"]},{"entry":["hsa-miR-455-5p","5.87","1.00E\u221205","5.3","5.00E\u2212214"]},{"entry":"Down-regulated"},{"entry":["hsa-miR-124","\u221213.37","8.00E\u221205","\u221210","5.00E\u221280"]},{"entry":["hsa-miR-874","\u22126.76","\u2002\u2009<1E\u221207","\u221233","1.50E\u221298"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},"Validation of the differentially expressed miRNAs using real-time RT-PCR. The distinct expression of these miRNAs in glioblastoma stem cells and neural stem cells was further validated using real-time RT-PCR analysis. RT-PCR results of the top three miRNAs that are up-regulated in glioblastoma stem cells in both microarray and deep sequencing analyses (Table 3) are shown in . All three miRNAs showed a significant up-regulation in the three primary glioblastoma stem cell lines (GSC1-3) tested, compared to three lines of normal neural stem cells. miR-10a revealed a dramatic increase of expression in all three glioblastoma stem cell lines tested, with more than 100-fold up-regulation of expression in two of the glioblastoma stem cell lines (). miR-10b exhibited even higher expression in glioblastoma stem cell lines GSC1 and GSC3, with up to 2,505-fold increase of expression in GSC1 (). miR-140-5p also displayed significant increase of expression in all three glioblastoma stem cell lines tested, although with much lower fold induction ().","As discussed above, two miRNAs are down-regulated more than 5-fold in glioblastoma stem cells as shown by both microarray and deep sequencing analyses. The expression of these two miRNAs was also validated using real-time RT-PCR assays. Both miR-124 and miR-874 exhibited reproducible decrease of expression in all three glioblastoma stem cell lines tested, compared to normal neural stem cells (, E). Specifically, miR-874, which has not been well characterized to date, exhibited a significant reduction of expression in all three glioblastoma stem cell lines, with more than 20-fold reduction in two of the glioblastoma stem cell lines GSC1 and GSC3 ().","In addition, normal and glioblastoma brain tissues were analyzed to determine whether the set of miRNAs identified above are differentially expressed in distinct expression patterns as shown in the cultured cell studies descried above. For this purpose, RNAs were isolated from 9 grade IV glioblastoma multiforme brain tissue samples and 4 non-tumor normal brain tissue samples. Real-time RT-PCR analyses were performed to detect the expression of two up-regulated miRNAs and two down-regulated miRNAs. Consistent with the results from tumor stem cells described above, miR-10a exhibited a substantial increase of expression in most glioblastoma tissues (). miR-10b also exhibited a substantial up-regulation of expression in all of the glioblastoma tissues tested, with an average increase of 142-fold (). For miRNAs that were down-regulated in glioblastoma stem cells, both miR-124 and miR-874 displayed a significant decrease of expression in most of the glioblastoma tissues tested, compared to their average expression in normal brain tissues (, D).","Target Identification of the Differentially-Expressed miRNAs.","By using a Targetscan algorithm [Lewis et al. 2003], CUB and SUSHI multiple domain protein 1 (CSMD1) were identified as candidate downstream targets for miR-10a and miR-10b, the most highly up-regulated miRNAs in glioblastoma stem cells in the profiling analyses. CSMD1 is a tumor suppressor gene that maps to chromosome 8p23, a region deleted in many tumor types [Kamal et al. 2010]. Sequence analysis revealed that the seed region of both miR-10a and miR-10b could form complementary base pairs with the 3\u2032 untranslated region (3\u2032 UTR) of human and mouse CSMD1 mRNAs (, B). To demonstrate a direct interaction between the 3\u2032 UTR of CSMD1 and miR-10 (miR10a and miR-10b), the 3\u2032 UTR region of human CSMD1 that contains the putative miR-10 recognition sites and flanking sequences was inserted downstream of a luciferase reporter gene into a siCheck vector. RNA duplexes of mature miR-10a or miR-10b were transfected into human embryonic kidney HEK293 cells along with the reporter gene. Significant repression of the reporter gene was observed in both miR-10a and miR-10b-transfected cells (, D). Mutation of the miR-10 targeting sites abolished the repression (, D). Furthermore, treatment of the inhibitors of miR-10a and miR-10b reversed the inhibitory effect of miR-10a and miR-10b on the luciferase reporter activity, respectively (, D). These results suggest that both miR-10a and miR-10b repress CSMD1 expression through the predicted targeting sites in CSMD1 3\u2032 UTR.","Since miR-10a and miR-10b are both up-regulated in glioblastoma stem cells relative to neural stem cells, the expression of CSMD1 was examined in both cell types. Dramatic reduction of CSMD1 mRNA expression was detected in glioblastoma stem cells by RT-PCR analysis, compared to neural stem cells (), consistent with the observation that CSMD1 expression is repressed by miR-10 (, D). The homeobox transcription factor HOXD10 has been identified as a tumor suppressor gene targeted by miR-10b in breast cancers [Ma et al. 2007]. Here, it was shown that the HOXD10 mRNA expression is also dramatically reduced (>20-fold) in glioblastoma stem cells examined, compared to normal neural stem cells (). Together, these results suggest that miR-10 targets the expression of tumor suppressor genes, CSMD1 and HOXD10, in glioblastoma stem cells.","Furthermore, using the Targetscan algorithm, the oncogenes NRAS and PIM3 were selected as putative target genes of miR-124, one of the down-regulated miRNAs in glioblastoma stem cells. NRAS is a small guanine-nucleotide binding protein and one of the three RAS (KRAS, NRAS, HRAS) isoforms [Kiessling et al. 2011]. The RAS signaling pathway plays an important role in many cancers by regulating cell proliferation, differentiation, and survival [Kan et al. 2010]. Using Targetscan algorithm, miR-124 was predicted to have a targeting site at the 3\u2032 UTR of the NRAS gene. This targeting site is conserved in human, mouse, and dog NRAS (). To validate the targeting of NRAS by miR-124, a luciferase reporter construct with human NRAS 3\u2032 UTR containing the predicted miR-124 targeting site was made, and the flanking sequences inserted into the 3\u2032UTR of a luciferase reporter gene in a siCHECK vector. Transfection of miR-124 RNA duplexes led to significant repression of the reporter gene (). Mutation of the putative miR-124 targeting site abolished the repression (). Furthermore, treatment with a miR-124 inhibitor reversed the inhibitory effect of miR-124 on the luciferase reporter activity (). These results suggest that miR-124 represses NRAS expression through the predicted targeting site in NRAS 3\u2032UTR.","Next, it was tested whether miR-124 targets NRAS expression in glioblastoma stem cells. Mature miR-124 RNA duplexes were introduced into GSC1 cells using a cationic triethanolamine-core polyamidoamine (PAMAM) dendrimer-mediated small RNA delivery system [Zhou et al. 2006; Zhou et al. 2011]. A control RNA duplex was included as a negative control. NRAS expression levels were examined by Western blot analysis. Reduction of NRAS protein level was detected in miR-124-transfected cells. Co-transfection of a miR-124 RNA inhibitor abolished the inhibitory effect of miR-124 on NRAS expression (). This result indicates that miR-124 down-regulates endogenous NRAS expression in glioblastoma stem cells.","The expression of NRAS in glioblastoma stem cells and neural stem cells was examined, where miR-124 exhibits differential expression. A significant increase of NRAS mRNA expression was detected in glioblastoma stem cells, compared to neural stem cells (). The inverse expression pattern of NRAS and miR-124 is consistent with the observation that NRAS expression is repressed by miR-124 (, C).","A putative targeting site of miR-124 was also identified in the 3\u2032 UTR of both human and mouse PIM3, a proto-oncogene with serine\/threonine kinase activity (). PIM3 has been shown to promote tumor cell growth through modulating cell cycle regulators [Wu et al. 2010; Brault et al. 2010]. To validate the targeting of PIM3 by miR-124, a luciferase reporter construct with human PIM3 3\u2032 UTR was made, which contained the predicted miR-124 targeting site and the flanking sequences inserted into the 3\u2032UTR of a luciferase reporter gene. Transfection of miR-124 led to significant repression of the reporter gene and mutation of the putative miR-124 targeting site abolished the repression (). Furthermore, treatment with a miR-124 inhibitor reversed the inhibitory effect of miR-124 on the luciferase reporter activity (). These results suggest that miR-124 represses PIM3 expression through the predicted targeting site in its 3\u2032 UTR.","To test whether miR-124 targets PIM3 expression in glioblastoma stem cells, mature miR-124 RNA duplexes were introduced into GSC1 cells using the dendrimer-mediated delivery system [Zhou et al. 2006; Zhou et al. 2011]. A control RNA duplex was included as a negative control. Reduction of PIM3 protein level was detected in miR-124-transfected cells by Western blot analysis. Co-transfection of a miR-124 RNA inhibitor abolished the inhibitory effect of miR-124 on PIM3 expression (). This result indicates that miR-124 down-regulates endogenous PIM3 expression in glioblastoma stem cells. Moreover, a significant increase of PIM3 mRNA expression was detected in the glioblastoma stem cells tested, compared to neural stem cells (), further supporting the idea that miR-124 represses PIM3 expression.","It is increasingly clear that miRNAs are important regulators of key signaling pathways implicated in tumorigenesis. Using Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analysis, the predicted targets of miRNAs that showed more than 5-fold up-regulation or down-regulation in glioblastoma stem cells in the microarray analysis were compared. Seven of the ten miRNAs that were up-regulated more than 5-fold in glioblastoma stem cells are predicted to have components of the p53 pathway as common targets (). The p53 pathway has been shown to be involved in cell cycle arrest, apoptosis, inhibition of cell migration, inhibition of angiogenesis, and affect genomic stability [Junttila & Evan 2009]. In contrast, five out of eight miRNAs that exhibited more than 5-fold down-regulation in glioblastoma stem cells were predicted to target components of the IGF pathway () that has been implicated in promoting cell growth, survival and migration [Clayton et al. 2011].","Discussion","The present study investigated genome-wide miRNA expression in tumor stem cell populations of glioblastoma, the most frequent and malignant primary brain tumor. In spite of recent improvement of surgical and radiotherapeutic techniques, the prognosis for glioblastoma patients is still very poor. The search for molecular targets is fundamental to develop effective treatments for glioblastoma.","Global profiling is an effective approach to identify abnormally expressed miRNAs in tumor genomes. Three different technical platforms were used to determine the differential expression of miRNAs in glioblastoma stem cells and neural stem cells. The microarray platform was combined with the newly emerged small RNA deep sequencing technology to profile miRNA expression in glioblastoma stem cells and normal neural stem cells and validated the profiling results using quantitative RT-PCR. Although the absolute fold change obtained from each platform is different due to the different sensitivity of the techniques, the trend of the change for the miRNAs studied is consistent. The miRNA expression profile could clearly distinguish glioblastoma stem cells from normal neural stem cells, allowing us to identify a miRNA signature of glioblastoma stem cells that were significantly up-regulated or down-regulated in glioblastoma stem cells, relative to neural stem cells.","In line with the findings that a set of miRNAs are differentially expressed in glioblastoma stem cells and normal neural stem cells, certain miRNAs also exhibit distinct expression profiles in glioblastoma tissues and normal brain tissues. For example, it was demonstrated that the expression of miR-874 is dramatically reduced in glioblastoma tissues, compared to normal brain tissues. miR-124, another miRNA that was down-regulated in glioblastoma stem cells, also exhibited reduced expression in glioblastoma tissues in this study, consistent with the results of previous glioblastoma tumor tissue profiling [Huse et al. 2009; Silber et al. 2008; Clafre et al. 2005; Godlewski et al. 2008; Xia et al. 2012; Skalsky & Cullen 2011; Fowler et al. 2011].","In this study, miR-10b was shown to be highly expressed in both glioblastoma stem cells and in glioblastoma tumor tissues. Up-regulation of miR-10b was also observed in other glioblastoma samples [H use et al. 2009; Clafre et al. 2005; Godlewski et al. 2008], suggesting an important role for miR-10b in glioblastoma tumorigenesis. Moreover, a recent study revealed that miR-10b expression is inversely correlated with glioblastoma patient survival [Gabriely et al. 2011]. Interestingly, miR-10b was also found to be up-regulated in breast cancer, leukemia, and pancreatic cancer and promote tumor invasion and metastasis in breast cancer [Ma et al. 2007; Cahn et al. 2004; Bloomston et al. 2007]. Together, these results suggest that some miRNAs, such as miR-10b, may function as a global oncogene to stimulate tumorigenesis in multiple tissues. Likewise, miR-124 is also frequently down-regulated in other cancers, such as medulloblastoma, hepatocellular carcinoma, and oral squamous carcinoma [Li et al. 2009; Furuta et al. 2010; Hunt et al. 2011], suggesting that it may function as a general tumor suppressor. Therefore the knowledge of miRNAs that we have obtained for glioblastoma stem cells may be applicable to other types of cancer stem cells.","Pathway analysis revealed that most of the significantly up-regulated miRNAs, with more than 5-fold increase in glioblastoma stem cells as shown in the microarray analysis described herein, have putative targets in a common pathway, the p53 pathway. The dysregulation of the p53 pathway has been shown to be an underlying mechanism for tumorigenesis [Junttila & Evan 2009], thus the up-regulated miRNAs may function as oncomiRs by targeting the p53 pathway if their role in regulating the p53 pathway is confirmed. On the other hand, most of the down-regulated miRNAs, with more than 5-fold decrease in glioblastoma stem cells, share their predicted targets in the IGF signaling. Repression of the IGF signaling has been shown to inhibit tumorigenesis [Clayton et al. 2011]. Thus, these down-regulated miRNAs may assume a role of tumor suppressors by targeting components of the IGF pathway if their role in regulating the IGF signaling is confirmed. The identification of miRNAs as oncogenes or tumor suppressors holds the promise of identifying novel diagnostic markers or molecular targets for antitumor therapies. The prediction that the differentially expressed miRNAs have the ability to target multiple components in one or more pathways makes them potential molecular targets for cancer therapy.","Glioblastoma stem cells represent a subpopulation of cancer cells with extraordinary capacities to promote tumor growth, invasion and therapeutic resistance, making them an ideal target cell population for anti-glioblastoma therapies. However, a major challenge is to define functional and molecular features that can distinguish cancer stem cells from normal stem cells in order to develop therapeutic strategies that specifically target the tumor population, but leave normal stem cells intact. Therefore comparing miRNA expression between glioblastoma stem cells and normal neural stem cells is highly relevant in that it may lead to the identification of glioblastoma stem cell-specific miRNAs, thus resulting in the development of novel glioblastoma therapies by targeting only tumor stem cells. A comparison of the miRNA expression between glioblastoma stem cells from adult glioblastoma patients and normal neural stem cells from human fetal brains was performed. Fetal brains were used instead of human adult brains due to the inaccessibility of normal human adult brain tissues containing neural stem cells. Although differences do exist between embryonic neural stem cells and adult neural stem cells, it has been proposed that embryonic neural stem cells resemble adult neural stem cells in many ways [Ming & Song 2011]. Therefore this comparison will provide useful information regarding glioblastoma stem cell-specific miRNA expression and provide a basis for strategic targeting glioblastoma stem cells through modulation of tumor stem cell-specific miRNA expression.","miRNAs have been shown to be involved in tumor initiation and progression, functioning as oncogenes or tumor suppressor [Asadi-Moghaddam et al. 2010; Cheng et al. 2010]. Therefore, modulation of miRNA expression provides great hope for potential cancer therapy [Verissimo et al. 2011]. Furthermore, since each miRNA may have more than one targets, miRNA-based gene therapy offers the therapeutic appeal of targeting multiple gene networks that are controlled by a single miRNA [Asadi-Moghaddam et al. 2010]. Strategies for miRNA-based cancer therapy include overexpression of tumor suppressor miRNAs and targeting oncogenic miRNAs using their antagonists. Based on the miRNA signature that was identified in glioblastoma stem cells, targeted glioblastoma therapies may be developed by inhibiting the up-regulated miR-10a or miR-10b function using miR-10 antagonists or overexpressing the down-regulated miR-124 or miR-874. Of note, 16 miRNAs that were up-regulated in glioblastoma stem cells (>1.5-fold, as shown in Table 2), including miR-10a and miR-10b, are also up-regulated in malignant astrocytomas (glioblastomas and anaplastic astrocytomas) in a genome-wide miRNA expression profiling between malignant astrocytomas and normal brain samples [Rao et al. 2010]. Eleven miRNAs that were down-regulated in glioblastoma stem cells (>1.5-fold, as shown in Table 2), including miR-124, are also down-regulated in malignant astrocytomas.","Moreover, the prognosis of glioblastoma patients remains poor. Biomarkers for this disease are needed for early detection of tumor progression [Roth et al. 2011]. The miRNA signature that was identified herein may be used as biomarkers to differentiate glioblastoma stem cells from normal neural stem cells. Recently, an miRNA signature was identified in the peripheral blood of glioblastoma patients [Roth et al. 2011]. Interestingly, several of the miRNAs that showed elevated expression in the blood samples of glioblastoma patients (vs healthy control) also exhibited increased expression in glioblastoma stem cells (vs normal neural stem cells) in the study described herein (Table 2), including miR-424, miR-148a, miR-362-3p, miR-30d, miR-128. These miRNAs may therefore represent easily accessible biomarkers that can be used for diagnostic purposes in glioblastoma patients.","In the studies described above, an miRNA profiling analysis using both microarray and deep sequencing analyses revealed that one of the miRNAs\u2014miR-874\u2014was significantly down-regulated in glioblastoma stem cells (GSCs) relative to normal stem cells (NSCs). These studies suggest that this miRNA may function as a tumor suppressor. As such, the role of miR-874 in GSC growth was characterized in a study using two GSC cell lines derived from WHO grade IV glioblastoma patients, PBT017 and PBT707. A lentiviral expression system was used to deliver miR-874 to the two GCS cells lines.","Transduction of the GSC lines with miR-874-expressing lentivirus led to robust expression of miR-874 in both types of cells, as shown in  (PBT017 cells) and  (PBT707 cells). The PBT017 and PBT707 cells overexpressing miR-874 were cultured for approximately 8-10 days to observe miR-874's effect on cell growth. A growth curve analysis revealed that overexpression of miR-874 led to dramatically reduced cell numbers in both PBT017 and PBT707 glioblastoma stem cell lines, as shown in . These results suggest that miR-874 may act as a potential tumor suppressor miRNA and may be used in accordance with the methods for treating cancers (e.g., glioblastoma) described herein.","The references, patents and published patent applications listed below, and all references cited in the specification above are hereby incorporated by reference in their entirety, as if fully set forth herein.\n\n"],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0011","num":"0010"},"figref":["FIGS. 1A-1C","FIG. 1A","FIG. 1B","FIG. 1C"]},{"@attributes":{"id":"p-0012","num":"0011"},"figref":["FIGS. 2A-2E","FIG. 2A","FIG. 2B","FIG. 2C","FIG. 2D","FIG. 2E"]},{"@attributes":{"id":"p-0013","num":"0012"},"figref":["FIGS. 3A-3D","FIG. 3A","FIG. 3B","FIG. 3C","FIG. 3D"]},{"@attributes":{"id":"p-0014","num":"0013"},"figref":["FIGS. 4A-4F","FIGS. 4A and 4B","FIG. 4C","FIG. 4D","FIG. 4E","FIG. 4F"],"i":"homo sapiens. "},{"@attributes":{"id":"p-0015","num":"0014"},"figref":["FIGS. 5A-5H","FIG. 5A","FIG. 5B","FIG. 5C","FIG. 5D","FIG. 5E","FIG. 5F","FIG. 5G","FIG. 5H"]},{"@attributes":{"id":"p-0016","num":"0015"},"figref":["FIGS. 6A-6B","FIG. 6A","FIG. 6B"]},{"@attributes":{"id":"p-0017","num":"0016"},"figref":["FIGS. 7A-7D","FIGS. 7A and 7C","FIGS. 7B and 7D"]}]},"DETDESC":[{},{}]}
