---
title: Non-replicative transduction particles and transduction particle-based reporter systems
abstract: Methods and systems are provided for packaging reporter nucleic acid molecules into non-replicative transduction particles for use as reporter molecules. The non-replicative transduction particles can be constructed from viruses and use viral transduction and replication systems. The reporter nucleic acid molecules include a reporter gene, such as a reporter molecule or selectable marker, for detecting target genes or cells. Methods and systems are provided for detection of cells and target nucleic acid molecules using the non-replicative transduction particles as reporter molecules.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09388453&OS=09388453&RS=09388453
owner: GENEWEAVE BIOSCIENCES, INC.
number: 09388453
owner_city: Los Gatos
owner_country: US
publication_date: 20141121
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS REFERENCE TO RELATED APPLICATIONS","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","I. Definitions","II. Lysogenic and Lytic Cycle of Viruses","III. Methods for Producing Non-Replicative Transduction Particles (NRTP)","IV. Reporters","V. NRTPs and Reporter Assays","EXAMPLES","Example 1","Silent Mutation\/Complementation Packaging System","Example 2","Deletion\/Complementation Packaging System","Example 3","SaPIbov2-Based Packaging System Lacking Integrase","Example 4","terS Deletion\/Complementation-Based SarS Reporter Transduction Particles","Example 5","terS Deletion\/Complementation-Based \u03b2-Lactamase Reporter Transduction Particles","Example 6","terS Deletion\/Complementation-Based Intracellular Molecule Reporter Transduction Particles","Example 7","Non-Replicative Transduction Particle-Based Reporter System","Example 8","NRTP-Based Assay for Antimicrobial Susceptibility Testing\u2014Correlation of Minimum Inhibitory Concentration to Luminescence Output","Example 9","Transcript Reporter Assay: Mechanism of Conformational Change by RBS-Blocking Cis-Repression of Luxab Translation Activated by the mecA Gene Transcript of MRSA","Example 10","Transcript Reporter Assay: Methods of Detecting Target Transcripts or Genes Using a mecA-luxAB Reporter System","REFERENCES CITED","INFORMAL SEQUENCE LISTING"],"p":["This application claims the benefit of International Application No. PCT\/US2014\/026536, filed on Mar. 13, 2014, U.S. Provisional Application No. 61\/779,177, filed on Mar. 13, 2013, U.S. Provisional Application No. 61\/897,040, filed on Oct. 29, 2013, and U.S. Provisional Application No. 61\/939,126, filed on Feb. 12, 2014, each of which is hereby incorporated in its entirety by reference.","1. Field of the Invention","The invention relates to methods and compositions for packaging and delivery of non-replicative transduction reporter molecules into cells for detecting target genes in cells.","2. Description of the Related Art","A transduction particle refers to a virus capable of delivering a non-viral nucleic acid into a cell. Viral-based reporter systems have been used to detect the presence of cells and rely on the lysogenic phase of the virus to allow expression of a reporter molecule from the cell. These viral-based reporter systems use replication-competent transduction particles that express reporter molecules and cause a target cell to emit a detectable signal.","However, the lytic cycle of the virus has been shown to be deleterious to viral-based reporter assays. Carriere, C. et al., . Journal of Clinical Microbiology, 1997. 35(12): p. 3232-3239. Carri\u00e8re et al. developed Calmette-Gu\u00e9rin (BCG) luciferase reporter phages that have their lytic cycles suppressed at 30\u00b0 C., but active at 37\u00b0 C. Using this system, Carri\u00e8re et al. have demonstrated the detection of BCG using phage reporters with a suppressed lytic cycle.","There are disadvantages, however, associated with suppressing but not eliminating the replication functions of the bacteriophage in bacteriophage-based reporter assays. First, controlling replication functions of the bacteriophage imposes limiting assay conditions. For example, the lytic cycle of the reporter phage phAE40 used by Carri\u00e8re et al. was repressed when the phage was used to infect cells at the non-permissive temperature of 30\u00b0 C. This temperature requirement imposed limiting conditions on the reporter assay in that the optimum temperature for the target bacteria was 37\u00b0 C. These limiting conditions hinder optimum assay performance.","Moreover, the replication functions of the virus are difficult to control. The replication of the virus should be suppressed during the use of the transduction particles as a reporter system. For example, the lytic activity of the reporter phage phAE40 reported by Carri\u00e8re et al. was reduced but was not eliminated, resulting in a drop in luciferase signal in the assay. Carri\u00e8re et al. highlighted possible causes for the resulting drop in reporter signal, such as intact phage-expressed genes and temperature limitations of the assay, all stemming from the fact that the lytic cycle of the phage reporter was not eliminated.","Reporter assays relying on the natural lysogenic cycle of phages can be expected to exhibit lytic activity sporadically. In addition, assays that rely on the lysogenic cycle of the phage can be prone to superinfection immunity from target cells already lysogenized with a similar phage, as well as naturally occurring host restriction systems that target incoming virus nucleic acid, thus limiting the host range of these reporter phages.","In other examples, transduction particle production systems are designed to package exogenous nucleic acid molecules, but the transduction particle often contains a combination of exogenous nucleic acid molecules and native progeny virus nucleic acid molecules. The native virus can exhibit lytic activity that is a hindrance to assay performance, and the lytic activity of the virus must be eliminated in order to purify transduction particles. However, this purification is generally not possible. In U.S. 2009\/0155768 A, entitled Reporter Plasmid Packaging System for Detection of Bacteria, Scholl et al. describes the development of such a transduction particle system. The product of the system is a combination of reporter transduction particles and native bacteriophage (FIG. 8 in the reference). Although the authors indicate that the transduction particle and native bacteriophage can be separated by ultracentrifugation, this separation is only possible in a system where the transduction particle and the native virus exhibit different densities that would allow separation by ultracentrifugation. While this characteristic is exhibited by the bacteriophage T7-based packaging system described in the reference, this is not a characteristic that is generally applicable for other virus systems. It is common for viral packaging machinery to exhibit headful packaging that would result in native virus and transduction particles to exhibit indistinguishable densities that cannot be separated by ultracentrifugation. Virus packaging systems also rely on a minimum amount of packaging as a requirement for proper virus structural assembly that results in native virus and transduction particles with indistinguishable densities.","Thus, there is a need for non-replicative transduction particles that do not suffer from the deleterious effects from lytic functions of the virus and the possibility of being limited by superinfection immunity and host restriction mechanisms that target virus nucleic acid molecules and viral functions, all of which can limit the performance of the reporter assay by increasing limits of detection and resulting in false negative results.","Even where transduction particles have been engineered, methods for using the transduction particles to detect and report the presence of target nucleic acid molecules in cells have limitations. Some methods require disruption of the cell and cumbersome techniques to isolate and detect transcripts in the lysate. Detection methods include using labeled probes such as antibodies, aptamers, or nucleic acid probes. Labeled probes directed to a target gene can result in non-specific binding to unintended targets or generate signals that have a high signal-to-noise ratio. Therefore, there is a need for specific, effective and accurate methods for detection and reporting of endogenous nucleic acid molecules in cells.","Accordingly, methods and systems are needed for generating non-replicative transduction particles that allow packaging and expression of reporter molecules in cells, while eliminating replication-competent progeny virus. Effective and accurate methods for detecting molecules in cells using the expressed reporter molecules are also needed.","Disclosed herein is a bacterial cell packaging system for packaging a reporter nucleic acid molecule into a non-replicative transduction particle, said bacterial cell comprising a lysogenized bacteriophage genome lacking a bacteriophage gene encoding a packaging initiation site sequence, wherein deletion of said bacteriophage gene prevents packaging of a bacteriophage nucleic acid molecule into said non-replicative transduction particle; and a reporter nucleic acid molecule comprising a second bacteriophage gene, wherein said second bacteriophage gene encodes a packaging initiation site sequence and facilitates the packaging a replica of said reporter nucleic acid molecule into said non-replicative transduction particle, wherein said second bacteriophage gene is capable of expressing a protein that is encoded by said gene, wherein said replica of said reporter nucleic acid molecule forms a replicon amenable to packaging into said non-replicative transduction particle.","In some embodiments, the reporter nucleic acid molecule is operatively linked to a promoter. In another embodiment, the promoter is selected for contributing to reactivity of a reporter molecule expressed from said reporter nucleic acid molecule in said bacterial cell. In one embodiment, the reporter nucleic acid molecule comprises an origin of replication. In yet another embodiment, the replicon comprises a concatamer amenable to packaging into said non-replicative transduction particle.","In an embodiment, the first and said second bacteriophage genes each comprises a pacA gene of the Enterobacteriaceae bacteriophage P1 and comprises said packaging initiation site sequence. In one embodiment, the second bacteriophage gene comprises the sequence of SEQ ID NO:9. In another embodiment, the replicon is the Enterobacteriaceae bacteriophage P1 lytic replicon. In certain embodiments, the replicon comprises a C1 repressor-controlled P53 promoter, a promoter P53 antisense, a repL gene, and an in-frame deletion of a kilA gene. In one embodiment, the replicon comprises of the sequence of SEQ ID NO:3.","In yet another embodiment, the first and said second bacteriophage genes each comprises a small terminase (terS) gene comprising said packaging initiation site sequence. In one embodiment, the terS gene is a bacteriophage \u03c611 or \u03c680\u03b1 terS gene.","In another embodiment, the replicon is derived from a pT181 plasmid origin of replication. In yet another embodiment, the replicon comprises the sequence of SEQ ID NO:5. In some embodiments, the packaging initiation site sequence of said second bacteriophage gene comprises a pac-site. In other embodiments, the pac-site of said second bacteriophage gene comprises the sequence of SEQ ID NO:7. In one aspect, the packaging initiation site sequence of said second bacteriophage gene comprises a cos-site. In another aspect, the packaging initiation site sequence of said second bacteriophage gene comprises a concatamer junction.","In another aspect, a plasmid comprises said reporter nucleic acid molecule. In one aspect, the second bacteriophage gene is operatively linked to a promoter. In another embodiment, the promoter is an inducible promoter or a constitutive promoter. In one embodiment, the bacteriophage comprises the Enterobacteriaceae bacteriophage P1. In yet another embodiment, the bacteriophage comprises a bacteriophage \u03c680\u03b1 or a bacteriophage \u03c611. In one aspect, the bacterial cell comprises an cell. In another aspect, the bacterial cell comprises an cell. In yet another embodiment, the bacterial cell comprises a Gram-negative cell. In other embodiments, the bacterial cell comprises a Gram-positive cell.","In another aspect, the reporter nucleic acid molecule comprises a reporter gene. In one aspect, the reporter gene encodes a detectable and\/or a selectable marker. In certain aspects, the reporter gene is selected from the group consisting of enzymes mediating luminescence reactions (luxA, luxB, luxAB, luc, ruc, nluc), enzymes mediating colorimetric reactions (lacZ, HRP), fluorescent proteins (GFP, eGFP, YFP, RFP, CFP, BFP, mCherry, near-infrared fluorescent proteins), affinity peptides (His-tag, 3X-FLAG), and selectable markers (ampC, tet(M), CAT, erm). In another aspect, the reporter nucleic acid molecule comprises an aptamer. In yet another aspect, the reporter nucleic acid molecule comprises a nucleic acid transcript sequence that is complementary to a second sequence in said reporter nucleic acid molecule.","In one embodiment, the nucleic acid transcript sequence is complementary to a cellular transcript. In another embodiment, the nucleic acid transcript sequence comprises a cis-repressing sequence. In yet another embodiment, the replica of said reporter nucleic acid molecule comprises a nucleic acid transcript sequence that is complementary to a second sequence in said replica of said reporter nucleic acid molecule, wherein the nucleic acid transcript sequence is complementary to a cellular transcript and wherein said nucleic acid transcript sequence comprises a cis-repressing sequence.","In some embodiments, the method for packaging a reporter nucleic acid molecule into a non-replicative transduction particle, comprising providing conditions to said bacterial cell described herein that induce a lytic phase of said bacteriophage to produce non-replicative transduction particles packaged with said reporter nucleic acid molecule; and isolating said non-replicative transduction particle comprising said reporter nucleic acid molecule. In one embodiment, the non-replicative transduction particle does not contain a replicated bacteriophage genome. In another embodiment, induction of said lytic phase triggers excision of said genomic island nucleic acid molecule from said genome of said bacterial cell.","In another embodiment, the composition comprising said non-replicative transduction particle comprising a replica of said reporter nucleic acid molecule produced from the method described herein.","The invention comprises a bacterial cell packaging system for packaging a reporter nucleic acid molecule into a non-replicative transduction particle, said bacterial cell comprising a lysogenized bacteriophage genome comprising a first bacteriophage packaging initiation site sequence, wherein said first bacteriophage packaging initiation site sequence comprises a mutation that prevents packaging of a bacteriophage nucleic acid molecule into said non-replicative transduction particle; and a reporter nucleic acid molecule comprising a second bacteriophage packaging initiation site sequence, wherein said second bacteriophage packaging initiation site sequence lacks said mutation and facilitates the packaging of a replica of said reporter nucleic acid molecule into said non-replicative transduction particle, wherein said replica of said reporter nucleic acid molecule forms a replicon for packaging into said non-replicative transduction particle.","In one embodiment, the reporter nucleic acid molecule is operatively linked to a promoter. In another embodiment, the promoter is selected for contributing to reactivity of a reporter molecule expressed from said reporter nucleic acid molecule in said bacterial cell. In yet another embodiment, the reporter nucleic acid molecule comprises an origin of replication. In one embodiment, the replicon comprises a concatamer amenable to packaging into said non-replicative transduction particle. In another aspect, the first and said second bacteriophage packaging initiation site sequences each comprise a packaging initiation site sequence from a small terminase gene. In one aspect, the first and said second bacteriophage packaging initiation site sequences each comprise a pac-site sequence from a pacA gene of the Enterobacteriaceae bacteriophage P1. In another aspect, the first bacteriophage packaging initiation site sequence comprises SEQ ID NO:2. In yet another aspect, the second bacteriophage packaging initiation site sequence comprises SEQ ID NO:1. In one embodiment, the replicon comprises an Enterobacteriaceae bacteriophage P1 lytic replicon. In another embodiment, the replicon comprises a C1 repressor-controlled P53 promoter, a promoter P53 antisense, a repL gene, and an in-frame deletion of a kilA gene. In another aspect, the replicon comprises the sequence of SEQ ID NO:3. In certain aspects, the first and said second bacteriophage packaging initiation site sequences each comprise a pac-site sequence from a small terminase (terS) gene of an bacteriophage \u03c611 or \u03c680\u03b1. In another aspect, the replicon is derived from a pT181 plasmid origin of replication. In yet another aspect, the replicon comprises the sequence of SEQ ID NO:5. In one aspect, the first bacteriophage packaging initiation site sequence comprises the sequence of SEQ ID NO:2. In some embodiments, the second bacteriophage packaging initiation site sequence comprises the sequence of SEQ ID NO:1. In other embodiments, the packaging initiation site sequence comprises a pac-site. In another embodiment, the packaging initiation site sequence comprises a cos-site. In yet another embodiment, the packaging initiation site sequence comprises a concatamer junction. In some embodiments, the mutation in said first bacteriophage packaging initiation site sequence comprises a silent mutation. In another embodiment, the mutation in said first bacteriophage packaging initiation site sequence prevents cleavage of said packaging initiation sequence. In another embodiment, a plasmid comprises said reporter nucleic acid molecule. In one embodiment, the bacteriophage comprises Enterobacteriaceae bacteriophage P1.","In another embodiment, the bacteriophage comprises the bacteriophage \u03c611 or \u03c680\u03b1. In one embodiment, the bacterial cell comprises an cell. In another embodiment, the bacterial cell comprises an cell. In some embodiments, the bacterial cell comprises a Gram-negative bacterial cell. In one aspect, the bacterial cell comprises a Gram-positive bacterial cell. In another aspect, the reporter nucleic acid molecule comprises a reporter gene. In yet another aspect, the reporter gene encodes a detectable marker and\/or a selectable marker.","In other aspects, the reporter gene is selected from the group consisting of: genes encoding enzymes mediating luminescence reactions (luxA, luxB, luxAB, luc, ruc, nluc), genes encoding enzymes mediating colorimetric reactions (lacZ, HRP), genes encoding fluorescent proteins (GFP, eGFP, YFP, RFP, CFP, BFP, mCherry, near-infrared fluorescent proteins), nucleic acid molecules encoding affinity peptides (His-tag, 3X-FLAG), and genes encoding selectable markers (ampC, tet(M), CAT, erm). In another aspect, the reporter nucleic acid molecule comprises an aptamer. In other aspects, the replicon is packaged into said non-replicative transduction particle by bacteriophage packaging machinery. In some embodiments, the reporter nucleic acid molecule comprises a nucleic acid transcript sequence that is complementary to a second sequence in said reporter nucleic acid molecule. In another embodiment, the nucleic acid transcript sequence is complementary to a cellular transcript.","In one aspect, the nucleic acid transcript sequence comprises a cis-repressing sequence. In another aspect, the replica of said reporter nucleic acid molecule comprises a nucleic acid transcript sequence that is complementary to a second sequence in said replica of said reporter nucleic acid molecule, wherein said nucleic acid transcript sequence is complementary to a cellular transcript, and wherein said nucleic acid transcript sequence comprises a cis-repressing sequence.","In certain aspects, the method for packaging a reporter nucleic acid molecule into a non-replicative transduction particle, comprising: providing conditions to said bacterial cell described herein that induce a lytic phase of said bacteriophage to produce non-replicative transduction particles packaged with said reporter nucleic acid molecule; and isolating said non-replicative transduction particle comprising said reporter nucleic acid molecule.","In other aspects, the non-replicative transduction particle does not contain a replicated bacteriophage genome. In one aspect, the induction of said lytic phase triggers excision of said genomic island nucleic acid molecule from said genome of said bacterial cell.","In another aspect, the invention comprises a composition comprising said non-replicative transduction particle comprising a replica of said reporter nucleic acid molecule produced from said method described herein.","In one aspect, the invention includes a bacterial cell packaging system for packaging a reporter nucleic acid molecule into a non-replicative transduction particle, said bacterial cell comprising: a lysogenized bacteriophage genome comprising a first bacteriophage gene comprising a deletion of a packaging initiation site sequence of said first bacteriophage gene that prevents packaging of a bacteriophage nucleic acid molecule into said non-replicative transduction particle; and a reporter nucleic acid molecule comprising a second bacteriophage gene comprising a second packaging initiation site sequence that facilitates the packaging a replica of said reporter nucleic acid molecule into said non-replicative transduction particle, wherein said second bacteriophage gene encodes a protein, wherein said replica of said reporter nucleic acid molecule forms a replicon for packaging into said non-replicative transduction particle.","In another aspect, the reporter nucleic acid molecule is operatively linked to a promoter. In one aspect, the promoter is selected for contributing to reactivity of a reporter molecule expressed from said reporter nucleic acid molecule in said bacterial cell. In certain aspects, the reporter nucleic acid comprises an origin of replication. In another aspect, the replicon comprises a concatamer amenable to packaging into said non-replicative transduction particle. In one aspect, the first and said second bacteriophage genes each comprises a pacA gene of the Enterobacteriaceae bacteriophage P1 and comprise said packaging initiation site sequence. In another aspect, the first bacteriophage gene comprises the sequence of SEQ ID NO:6. In certain aspects, the second bacteriophage gene comprises the sequence SEQ ID NO:7. In one aspect, the replicon comprises an Enterobacteriaceae bacteriophage P1 lytic replicon. In yet another aspect, the replicon comprises a C1 repressor-controlled P53 promoter, a promoter P53 antisense, a repL gene, and an in-frame deletion of a kilA gene. In another aspect, the replicon comprises the sequence of SEQ ID NO:3. In other aspects, the first and said second bacteriophage genes each comprises a small terminase (terS) gene comprising said packaging initiation site sequence. In one aspect, the terS gene is a bacteriophage \u03c611 or \u03c680\u03b1 terS gene. In another aspect, the first bacteriophage gene comprises the sequence of SEQ ID NO:8. In yet another aspect, the second bacteriophage gene comprises the sequence of SEQ ID NO:9. In one aspect, the replicon is derived from a pT181 plasmid origin of replication. In one embodiment, the replicon comprises the sequence of SEQ ID NO:5, In another embodiment, the packaging initiation site sequence of said second bacteriophage gene comprises a pac-site. In yet another embodiment, the packaging initiation site sequence of said second bacteriophage gene comprises a cos-site.","In certain embodiments, the packaging initiation site sequence of said second bacteriophage gene comprises a concatamer junction. In one embodiment, a plasmid comprises said reporter nucleic acid molecule. In another embodiment, the second bacteriophage gene is operatively linked to a promoter. In yet another embodiment, the promoter is an inducible promoter or a constitutive promoter. In certain embodiments, the bacteriophage comprises the Enterobacteriaceae bacteriophage P1. In one embodiment, the bacteriophage comprises the bacteriophage \u03c680\u03b1 or bacteriophage \u03c611. In other embodiments, the bacterial cell comprises an cell. In another embodiment, the bacterial cell comprises an cell. In one embodiment, the bacterial cell comprises a Gram-negative cell. In another embodiment, the bacterial cell comprises a Gram-positive cell.","In another aspect, the reporter nucleic acid molecule comprises a reporter gene. In one aspect, the reporter gene encodes a detectable and\/or a selectable marker. In another aspect, the reporter gene is selected from the group consisting of genes encoding enzymes mediating luminescence reactions (luxA, luxB, luxAB, luc, ruc, nluc), genes encoding enzymes mediating colorimetric reactions (lacZ, HRP), genes encoding fluorescent proteins (GFP, eGFP, YFP, RFP, CFP, BFP, mCherry, near-infrared fluorescent proteins), nucleic acid molecules encoding affinity peptides (His-tag, 3X-FLAG), and genes encoding selectable markers (ampC, tet(M), CAT, erm). In one embodiment, the reporter nucleic acid molecule comprises an aptamer. In another embodiment, the replicon is packaged into said non-replicative transduction particle by bacteriophage packaging machinery. In yet another embodiment, the reporter nucleic acid molecule comprises a nucleic acid transcript sequence that is complementary to a second sequence in said reporter nucleic acid molecule. In one embodiment, the nucleic acid transcript sequence is complementary to a cellular transcript. In another embodiment, the nucleic acid transcript sequence comprises a cis-repressing sequence. In certain embodiments, the replica of said reporter nucleic acid molecule comprises a nucleic acid transcript sequence that is complementary to a second sequence in said replica of said reporter nucleic acid molecule, wherein said nucleic acid transcript sequence is complementary to a cellular transcript and wherein said nucleic acid transcript sequence comprises a cis-repressing sequence.","The invention includes a method for packaging a reporter nucleic acid molecule into a non-replicative transduction particle, comprising: providing conditions to said bacterial cell of any of the claims Error! Reference source not found.-Error! Reference source not found. that induce a lytic phase of said bacteriophage to produce non-replicative transduction particles packaged with said reporter nucleic acid molecule; and isolating said non-replicative transduction particle comprising said reporter nucleic acid molecule. In one embodiment, the non-replicative transduction particle does not contain a replicated bacteriophage genome. In another embodiment, the induction of said lytic phase triggers excision of said genomic island nucleic acid molecule from said genome of said bacterial cell.","In some aspects, the invention includes a composition comprising said non-replicative transduction particle comprising a replica of said reporter nucleic acid molecule produced from said method described herein.","In another aspect, the invention includes a bacterial cell packaging system for packaging a reporter nucleic acid molecule into a non-replicative transduction particle, said bacterial cell comprising: a lysogenized bacteriophage genome lacking a packaging gene and comprising genes that encode proteins that form said non-replicative transduction particle; and a genomic island nucleic acid molecule comprising a reporter nucleic acid molecule and a packaging gene. In one aspect, the packaging gene comprises a small terminase (terS) gene. terS gene comprises a bacteriophage \u03c680\u03b1 terS gene or a bacteriophage \u03c611 terS gene.","In one aspect, the terS gene comprises the sequence of SEQ ID NO:9. In another aspect, the genomic island nucleic acid molecule comprises a SaPIbov2 genomic island nucleic acid molecule. In yet another aspect, the genomic island nucleic acid molecule is selected from the group consisting of a SaPI, a SaPI1, a SaPI2, a SaPIbov1 and a SaPibov2 genomic island nucleic acid molecule. In another embodiment, the reporter nucleic acid molecule is operatively linked to a promoter. In yet another embodiment, the reporter nucleic acid molecule comprises an origin of replication. In some embodiments, the bacteriophage comprises a bacteriophage \u03c680\u03b1 or bacteriophage \u03c611. In other embodiments, the bacterial cell comprises an cell. In one embodiment, the genomic island nucleic acid molecule comprises an integrase gene and wherein said integrase gene encodes an integrase protein for excising and integrating said genomic island nucleic acid molecule out of and into a bacterial genome of said bacterial cell. In another embodiment, the integrase gene comprises the sequence of SEQ ID NO:10. In yet another embodiment, the genomic island nucleic acid molecule is integrated into a bacterial genome of said bacterial cell.","In certain aspects, the genomic island nucleic acid molecule can be replicated and forms molecule replicon that is amenable to packaging by the bacteriophage packaging machinery in said bacterial cell. In another aspect, the nucleic acid molecule forms a concatamer. In yet another aspect, the replicated genomic island nucleic acid molecule is capable of being packaged into said non-replicative transduction particle. In certain aspects, the packaging gene comprises a pac site sequence. In another aspect, the packaging gene comprises a cos-site sequence. In yet another embodiment, the packaging gene comprises a concatamer junction.","In other embodiments, the reporter nucleic acid molecule comprises a reporter gene. In some embodiments, the reporter gene encodes a selectable marker and\/or a selectable marker. In another embodiment, the reporter gene is selected from the group consisting of enzymes mediating luminescence reactions (luxA, luxB, luxAB, luc, ruc, nluc), enzymes mediating colorimetric reactions (lacZ, HRP), fluorescent proteins (GFP, eGFP, YFP, RFP, CFP, BFP, mCherry, near-infrared fluorescent proteins), affinity peptides (His-tag, 3X-FLAG), and selectable markers (ampC, tet(M), CAT, erm). In certain embodiments, the reporter nucleic acid molecule comprises an aptamer. In other embodiments, the genomic island nucleic acid molecule lacks an integrase gene. In another embodiment, the invention includes a bacterial gene comprising an integrase gene operatively linked to a promoter and wherein said integrase gene encodes an integrase protein for excising and integrating said genomic island nucleic acid molecule out of and into a bacterial genome of said bacterial cell. In one embodiment, the reporter nucleic acid molecule comprises a nucleic acid transcript sequence that is complementary to a second sequence in said reporter nucleic acid molecule. In other embodiments, the nucleic acid transcript sequence is complementary to a cellular transcript. In yet other embodiments, the nucleic acid transcript sequence comprises a cis-repressing sequence. In another embodiment, the replica of said reporter nucleic acid molecule comprises a nucleic acid transcript sequence that is complementary to a second sequence in said replica of said reporter nucleic acid molecule. In other embodiments, the nucleic acid transcript sequence is complementary to a cellular transcript. In other embodiments, the nucleic acid transcript sequence comprises a cis-repressing sequence.","The invention includes a method for packaging a reporter nucleic acid molecule into a non-replicative transduction particle, comprising: providing conditions to said bacterial cell of any of the claims Error! Reference source not found.-Error! Reference source not found. that induce a lytic phase of said bacteriophage to produce non-replicative transduction particles packaged with said reporter nucleic acid molecule; and isolating said non-replicative transduction particle comprising said reporter nucleic acid molecule. In some embodiments, the non-replicative transduction particle does not contain a replicated bacteriophage genome. In one embodiment, the induction of said lytic phase triggers excision of said genomic island nucleic acid molecule from said genome of said bacterial cell.","In another embodiment, the invention includes a composition comprising said non-replicative transduction particle comprising a replica of said reporter nucleic acid molecule produced from said method described herein.","The invention also includes a method for detecting a presence or an absence of a bacterial cell in a sample, comprising: introducing into a sample a non-replicative transduction particle comprising a reporter gene encoding a reporter molecule and lacking a bacteriophage genome under conditions such that said non-replicative transduction particle can transduce said bacterial cell and wherein said reporter gene can be expressed in said bacterial cell; providing conditions for activation of said reporter molecule; and detecting for a presence or an absence of a reporter signal transmitted from said expressed reporter molecule, wherein a presence of said reporter signal correctly indicates said presence of said bacterial cell.","In one embodiment, the method achieves at least 80% specificity of detection with reference to a standard, at least 90% specificity of detection with reference to a standard, or at least 95% specificity of detection with reference to a standard. In another embodiment, the method achieves at least 80% sensitivity of detection with reference to a standard, at least 85% sensitivity of detection with reference to a standard, or at least 90% sensitivity of detection with reference to a standard, or at least 95% sensitivity of detection with reference to a standard. In yet another embodiment, the method achieves at least 95% specificity of detection and at least 90% sensitivity of detection with reference to a standard. In another embodiment, the standard is a Gold standard. In yet another embodiment, the bacterial cell comprises a Methicillin Resistant (MRSA) cell. In other embodiments, the bacterial cell comprises a Methicillin Sensitive (MSSA) cell.","In another embodiment, the reporter gene encodes a detectable or selectable marker. In one embodiment, the reporter gene is selected from the group consisting of genes encoding enzymes mediating luminescence reactions (luxA, luxB, luxAB, luc, ruc, nluc), genes encoding enzymes mediating colorimetric reactions (lacZ, HRP), genes encoding fluorescent proteins (GFP, eGFP, YFP, RFP, CFP, BFP, mCherry, near-infrared fluorescent proteins), nucleic acid molecules encoding affinity peptides (His-tag, 3X-FLAG), and genes encoding selectable markers (ampC, tet(M), CAT, erm). In one embodiment, the reporter gene is operatively linked to a constitutive promoter.","In another aspect, the reporter signal can be detected from a sample at a limit of detection (LoD) of less than 1,000 colony forming units (CFU). In other aspects, the reporter signal can be detected from a sample at a limit of detection (LoD) of less than 100 colony forming units (CFU). In one aspect, the reporter signal can be detected from a sample at a limit of detection (LoD) of less than 10 colony forming units (CFU). In other aspects, the reporter signal can be detected from a sample at a LoD less than five CFU. In another aspect, the reporter signal can be detected from a sample at a LoD of three or less CFU.","In one embodiment, the method includes providing an antibiotic to said sample at a pre-determined concentration and detecting a presence or absence of said reporter signal to determine whether said bacterial cell is resistant or sensitive to said antibiotic. In another embodiment, the method includes providing varying pre-determined concentrations antibiotic to said sample and detecting the amount of said reporter signal to determine the minimum inhibitory concentration of said bacterial cell to said antibiotic.","In one aspect, the invention includes a composition comprising a nucleic acid construct that encodes a nucleic acid reporter transcript that is capable of forming at least two conformations comprising a first conformation that prevents reporter expression comprising an intramolecular double stranded region comprising a first subsequence and a second subsequence, and a second conformation that lacks said intramolecular double-stranded region and allows reporter gene expression, wherein conversion between said first and second conformations is mediated by competitive binding of a cellular transcript to said first and\/or said second subsequence.","In another aspect, the invention includes a non-replicative transduction particle comprising said nucleic acid construct. In yet another aspect, the competitive binding of said cellular transcript to said first and\/or said second subsequence results in said second conformation of said nucleic acid reporter construct. In one aspect, the first subsequence or said second subsequence comprises a cis-repressing sequence. In another aspect, the cis-repressing sequence comprises a sequence that is complementary or substantially complementary to a portion of said cellular transcript. In other aspects, the first subsequence or said second subsequence comprises a reporter gene sequence. In yet another aspect, the reporter gene sequence comprises a ribosome binding site. In other aspects, the reporter gene sequence encodes a detectable molecule. In another aspect, the detectable marker comprises a fluorescent molecule or an enzyme capable of mediating a luminescence or colorimetric reaction. In one embodiment, the reporter gene sequence encodes a selectable marker. In another embodiment, the selectable marker comprises an antibiotic resistance gene.","In other embodiments, the first subsequence and said second subsequence are located cis to each other on said nucleic acid construct to form said intramolecular double stranded region. In certain embodiments, the first subsequence and said second subsequence are complementary or substantially complementary to each other to form said intramolecular double stranded region. In one embodiment, the first subsequence or said second subsequence of said first conformation comprises a transcriptional enhancer sequence, and wherein said transcriptional enhancer sequence is upstream from a coding region of said reporter gene sequence. In another embodiment, the first conformation of said nucleic acid reporter transcript is capable of binding to a cleaving enzyme. In other embodiments, the first conformation of said nucleic acid reporter transcript is a target for degradation by a cellular enzyme. In other aspects, the first conformation comprises a non-binding intramolecular region. In another aspect, the non-binding intramolecular region is located 3\u2032 of said first subsequence and 5\u2032 of said second subsequence. In other aspects, the non-binding intramolecular region comprises a sequence YUNR, wherein Y is a pyrimidine, U is a Uracil, N is any nucleotide, and R is a purine.","In one embodiment, the first subsequence or said second subsequence comprises a modified sequence of said cellular transcript. In another embodiment, the modified sequence comprises a nucleotide substitution. In yet another embodiment, the modified sequence comprises a sequence insertion, a deletion or an inversion of said cellular transcript.","The method includes a composition comprising a nucleic acid construct that encodes a nucleic acid reporter transcript comprising a gene reporter sequence and that is capable of forming at least two conformations of said nucleic acid reporter transcript, a first unstable conformation that prevents translation of said reporter gene sequence in said nucleic acid reporter transcript, and a second stable conformation resulting from binding of said first unstable conformation with a cellular transcript, said second stable secondary conformation allowing translation of said reporter gene sequence of said nucleic acid reporter transcript.","In one embodiment, the composition comprises a non-replicative transduction particle comprising said nucleic acid construct. In another embodiment, the cellular transcript binds at a 3\u2032UTR sequence of said nucleic acid reporter transcript. In one embodiment, the second stable secondary conformation is formed by cleavage of a portion of a sequence of said first unstable secondary conformation. In another embodiment, the reporter gene sequence encodes a detectable molecule. In some embodiments, the detectable marker comprises a fluorescent molecule or an enzyme capable of mediating a luminescence or colorimetric reaction. In other embodiments, the reporter gene sequence encodes a selectable marker. In another embodiment, the selectable marker comprises an antibiotic resistance gene.","The invention also includes a composition comprising a nucleic acid construct that encodes a nucleic acid reporter transcript comprising a reporter gene sequence and that is capable of forming at least two conformations of said nucleic acid reporter transcript, comprising a first conformation that prevents further transcription of said nucleic acid construct, and a second conformation formed upon binding of said first conformation with a cellular transcript, wherein said second conformation allows transcription of said nucleic acid construct. In some embodiments, the composition comprises a non-replicative transduction particle comprising said nucleic acid construct. In another embodiment, the nucleic acid reporter transcript comprises a cis-repressing sequence.","In one embodiment, the nucleic acid reporter transcript comprises a reporter gene sequence. In another embodiment, the first conformation forms from a binding of said cis-repressing sequence to said reporter gene sequence. In some embodiments, the first conformation is a substrate for a cleaving enzyme. In one embodiment, the first conformation of said nucleic acid reporter transcript comprises a sequence that forms a transcription termination structure. In other embodiments, the binding of said cellular transcript to said sequence that forms a transcription termination structure results in cleavage of a portion of said nucleic acid reporter transcript and formation of said second conformation.","The invention comprises a vector comprising a regulatory sequence operably linked to a nucleic acid sequence that encodes said nucleic acid reporter transcript described herein.","The invention includes a method for detecting a target transcript in a cell, comprising: introducing into said cell said nucleic acid reporter construct described herein; and detecting the presence or absence of an output signal from said cell, wherein said presence of said output signal indicates the presence of the target transcript in said cell. The method includes detecting a presence of a bacterial cell based on detecting said presence of said target transcript.","In one embodiment, the method for detecting a presence of a bacterial cell in a sample comprising introducing into said sample said nucleic acid reporter construct described herein; and detecting the presence or absence of an output signal from said sample, wherein said presence of said output signal indicates the presence of the bacterial cell in said sample.","The invention comprises a kit, comprising a compartment for holding a sample comprising a cell and said nucleic acid reporter construct described herein; and instructions for detecting the presence or absence of an output signal from said sample, wherein the presence of the output signal indicates the presence of a target transcript in said cell","The invention comprises a composition, comprising a non-replicative transduction particle comprising a nucleic acid reporter construct, the nucleic acid reporter construct comprising a first promoter operatively linked a reporter gene, wherein said first promoter is capable of being induced by an inducer protein endogenous in a bacterial cell.","The invention includes a method for detecting a presence of a bacterial cell in a sample comprising contacting said sample with a non-replicative transduction particle comprising nucleic acid reporter construct comprising a first promoter operatively linked to a reporter gene, wherein said first promoter is capable of being induced by an inducer protein endogenous to said bacterial cell; and detecting the presence or absence of an output signal from said reporter gene, wherein said presence of said output signal indicates the presence of said bacterial cell in said sample.","In one embodiment, the first promoter is the same as an inducible promoter operatively linked to a target nucleic acid molecule in said bacterial cell.","The invention comprises a composition, comprising a non-replicative transduction particle comprising a nucleic acid reporter construct, the nucleic acid reporter construct comprising a reporter gene that encodes a reporter molecule, the non-replicative transduction particle capable of entering a bacterial cell; and a caged substrate that exogenous to said bacterial cell that once un-caged is capable of reacting to said reporter molecule in said cell.","The invention comprises a method for detecting a presence of a bacterial cell in a sample comprising contacting said sample with a caged substrate and a non-replicative transduction particle comprising a nucleic acid reporter construct, the nucleic acid reporter construct comprising a reporter gene that encodes a reporter molecule, the caged substrate exogenous to said cell that once un-caged is capable of binding to said reporter molecule in said bacterial cell; and detecting the presence or absence of an output signal from said reporter molecule, wherein said presence of said output signal indicates the presence of said bacterial cell in said sample.","In one embodiment, a target enzyme in said cell binds said caged substrate to produce an un-caged substrate. In some embodiments, the un-caged substrate reacts with said reporter molecule to produce said output signal.","The invention also includes a composition, comprising a non-replicative transduction particle comprising a nucleic acid reporter construct, the nucleic acid reporter construct encoding a switchable molecule capable of binding to a target molecule in a bacterial cell to form a complex; and a substrate capable of penetrating said cell and binding said complex to produce a detectable signal from said cell.","The invention includes a method for detecting a presence of a bacterial cell in a sample comprising contacting said sample with a substrate and a non-replicative transduction particle comprising a nucleic acid reporter construct encoding a switchable molecule, the switchable molecule capable of binding a target molecule in said cell to form a complex, the substrate capable of binding said complex to form a substrate-bound complex; and detecting the presence or absence of an output signal from said substrate-bound complex, wherein said presence of said output signal indicates the presence of said bacterial cell in said sample. In one embodiment, the binding of said switchable molecule to said target molecule produces a conformational change in said switchable molecule. In another embodiment, the conformational change in said switchable molecule allows said substrate to bind to said complex.","Terms used in the claims and specification are defined as set forth below unless otherwise specified.","As used herein, \u201creporter nucleic acid molecule\u201d refers to a nucleotide sequence comprising a DNA or RNA molecule. The reporter nucleic acid molecule can be naturally occurring or an artificial or synthetic molecule. In some embodiments, the reporter nucleic acid molecule is exogenous to a host cell and can be introduced into a host cell as part of an exogenous nucleic acid molecule, such as a plasmid or vector. In certain embodiments, the reporter nucleic acid molecule can be complementary to a target gene in a cell. In other embodiments, the reporter nucleic acid molecule comprises a reporter gene encoding a reporter molecule (e.g., reporter enzyme, protein). In some embodiments, the reporter nucleic acid molecule is referred to as a \u201creporter construct\u201d or \u201cnucleic acid reporter construct.\u201d","A \u201creporter molecule\u201d or \u201creporter\u201d refers to a molecule (e.g., nucleic acid or protein) that confers onto an organism a detectable or selectable phenotype. The detectable phenotype can be colorimetric, fluorescent or luminescent, for example. Reporter molecules can be expressed from reporter genes encoding enzymes mediating luminescence reactions (luxA, luxB, luxAB, luc, ruc, nluc), genes encoding enzymes mediating colorimetric reactions (lacZ, HRP), genes encoding fluorescent proteins (GFP, eGFP, YFP, RFP, CFP, BFP, mCherry, near-infrared fluorescent proteins), nucleic acid molecules encoding affinity peptides (His-tag, 3X-FLAG), and genes encoding selectable markers (ampC, tet(M), CAT, erm). The reporter molecule can be used as a marker for successful uptake of a nucleic acid molecule or exogenous sequence (plasmid) into a cell. The reporter molecule can also be used to indicate the presence of a target gene, target nucleic acid molecule, target intracellular molecule, or a cell, as described herein. Alternatively, the reporter molecule can be a nucleic acid, such as an aptamer or ribozyme.","In some aspects of the invention, the reporter nucleic acid molecule is operatively linked to a promoter. In other aspects of the invention, the promoter can be chosen or designed to contribute to the reactivity and cross-reactivity of the reporter system based on the activity of the promoter in specific cells (e.g., specific species) and not in others. In certain aspects, the reporter nucleic acid molecule comprises an origin of replication. In other aspects, the choice of origin of replication can similarly contribute to reactivity and cross-reactivity of the reporter system, when replication of the reporter nucleic acid molecule within the target cell contributes to or is required for reporter signal production based on the activity of the origin of replication in specific cells (e.g., specific species) and not in others. In some embodiments, the reporter nucleic acid molecule forms a replicon capable of being packaged as concatameric DNA into a progeny virus during virus replication.","As used herein, a \u201ctarget transcript\u201d refers to a portion of a nucleotide sequence of a DNA sequence or an mRNA molecule that is naturally formed by a target cell including that formed during the transcription of a target gene and mRNA that is a product of RNA processing of a primary transcription product. The target transcript can also be referred to as a cellular transcript or naturally occurring transcript.","As used herein, the term \u201ctranscript\u201d refers to a length of nucleotide sequence (DNA or RNA) transcribed from a DNA or RNA template sequence or gene. The transcript can be a cDNA sequence transcribed from an RNA template or an mRNA sequence transcribed from a DNA template. The transcript can be protein coding or non-coding. The transcript can also be transcribed from an engineered nucleic acid construct.","A transcript derived from a reporter nucleic acid molecule can be referred to as a \u201creporter transcript.\u201d The reporter transcript can include a reporter sequence and a cis-repressing sequence. The reporter transcript can have sequences that form regions of complementarity, such that the transcript includes two regions that form a duplex (e.g., an intermolecular duplex region). One region can be referred to as a \u201ccis-repressing sequence\u201d and has complementarity to a portion or all of a target transcript and\/or a reporter sequence. A second region of the transcript is called a \u201creporter sequence\u201d and can have complementarity to the cis-repressing sequence. Complementarity can be full complementarity or substantial complementarity. The presence and\/or binding of the cis-repressing sequence with the reporter sequence can form a conformation in the reporter transcript, which can block further expression of the reporter molecule. The reporter transcript can form secondary structures, such as a hairpin structure, such that regions within the reporter transcript that are complementary to each other can hybridize to each other.","\u201cIntroducing into a cell,\u201d when referring to a nucleic acid molecule or exogenous sequence (e.g., plasmid, vector, construct), means facilitating uptake or absorption into the cell, as is understood by those skilled in the art. Absorption or uptake of nucleic acid constructs or transcripts can occur through unaided diffusive or active cellular processes, or by auxiliary agents or devices including via the use of bacteriophage, virus, and transduction particles. The meaning of this term is not limited to cells in vitro; a nucleic acid molecule may also be \u201cintroduced into a cell,\u201d wherein the cell is part of a living organism. In such instance, introduction into the cell will include the delivery to the organism. For example, for in vivo delivery, nucleic acid molecules, constructs or vectors of the invention can be injected into a tissue site or administered systemically. In vitro introduction into a cell includes methods known in the art, such as electroporation and lipofection. Further approaches are described herein or known in the art.","A \u201ctransduction particle\u201d refers to a virus capable of delivering a non-viral nucleic acid molecule into a cell. The virus can be a bacteriophage, adenovirus, etc.","A \u201cnon-replicative transduction particle\u201d refers to a virus capable of delivering a non-viral nucleic acid molecule into a cell, but does not package its own replicated viral genome into the transduction particle. The virus can be a bacteriophage, adenovirus, etc.","A \u201cplasmid\u201d is a small DNA molecule that is physically separate from, and can replicate independently of, chromosomal DNA within a cell. Most commonly found as small circular, double-stranded DNA molecules in bacteria, plasmids are sometimes present in archaea and eukaryotic organisms. Plasmids are considered replicons, capable of replicating autonomously within a suitable host.","A \u201cvector\u201d is a nucleic acid molecule used as a vehicle to artificially carry foreign genetic material into another cell, where it can be replicated and\/or expressed.","A \u201cvirus\u201d is a small infectious agent that replicates only inside the living cells of other organisms. Virus particles (known as virions) include two or three parts: i) the genetic material made from either DNA or RNA molecules that carry genetic information; ii) a protein coat that protects these genes; and in some cases, iii) an envelope of lipids that surrounds the protein coat.","\u201cMRSA\u201d refers to Methicillin-resistant ","\u201cMSSA\u201d refers to Methicillin-sensitive ","The term \u201cameliorating\u201d refers to any therapeutically beneficial result in the treatment of a disease state, e.g., a disease state, including prophylaxis, lessening in the severity or progression, remission, or cure thereof.","The term \u201cin situ\u201d refers to processes that occur in a living cell growing separate from a living organism, e.g., growing in tissue culture.","The term \u201cin vivo\u201d refers to processes that occur in a living organism.","The term \u201cmammal\u201d as used herein includes both humans and non-humans and include but is not limited to humans, non-human primates, canines, felines, murines, bovines, equines, and porcines.","\u201cG,\u201d \u201cC,\u201d \u201cA\u201d and \u201cU\u201d each generally stand for a nucleotide that contains guanine, cytosine, adenine, and uracil as a base, respectively. \u201cT\u201d and \u201cdT\u201d are used interchangeably herein and refer to a deoxyribonucleotide wherein the nucleobase is thymine, e.g., deoxyribothymine. However, it will be understood that the term \u201cribonucleotide\u201d or \u201cnucleotide\u201d or \u201cdeoxyribonucleotide\u201d can also refer to a modified nucleotide, as further detailed below, or a surrogate replacement moiety. The skilled person is well aware that guanine, cytosine, adenine, and uracil may be replaced by other moieties without substantially altering the base pairing properties of an oligonucleotide comprising a nucleotide bearing such replacement moiety. For example, without limitation, a nucleotide comprising inosine as its base may base pair with nucleotides containing adenine, cytosine, or uracil. Hence, nucleotides containing uracil, guanine, or adenine may be replaced in the nucleotide sequences of the invention by a nucleotide containing, for example, inosine. Sequences comprising such replacement moieties are embodiments of the invention.","As used herein, the term \u201ccomplementary,\u201d when used to describe a first nucleotide sequence in relation to a second nucleotide sequence, refers to the ability of an oligonucleotide or polynucleotide comprising the first nucleotide sequence to hybridize and form a duplex structure under certain conditions with an oligonucleotide or polynucleotide comprising the second nucleotide sequence, as will be understood by the skilled person. Complementary sequences are also described as binding to each other and characterized by binding affinities.","For example, a first nucleotide sequence can be described as complementary to a second nucleotide sequence when the two sequences hybridize (e.g., anneal) under stringent hybridization conditions. Hybridization conditions include temperature, ionic strength, pH, and organic solvent concentration for the annealing and\/or washing steps. The term stringent hybridization conditions refers to conditions under which a first nucleotide sequence will hybridize preferentially to its target sequence, e.g., a second nucleotide sequence, and to a lesser extent to, or not at all to, other sequences. Stringent hybridization conditions are sequence dependent, and are different under different environmental parameters. Generally, stringent hybridization conditions are selected to be about 5\u00b0 C. lower than the thermal melting point (T) for the nucleotide sequence at a defined ionic strength and pH. The Tis the temperature (under defined ionic strength and pH) at which 50% of the first nucleotide sequences hybridize to a perfectly matched target sequence. An extensive guide to the hybridization of nucleic acids is found in, e.g., Tijssen (1993) Laboratory Techniques in Biochemistry and Molecular Biology\u2014Hybridization with Nucleic Acid Probes part I, chap. 2, \u201cOverview of principles of hybridization and the strategy of nucleic acid probe assays,\u201d Elsevier, N.Y. (\u201cTijssen\u201d). Other conditions, such as physiologically relevant conditions as may be encountered inside an organism, can apply. The skilled person will be able to determine the set of conditions most appropriate for a test of complementarity of two sequences in accordance with the ultimate application of the hybridized nucleotides.","This includes base-pairing of the oligonucleotide or polynucleotide comprising the first nucleotide sequence to the oligonucleotide or polynucleotide comprising the second nucleotide sequence over the entire length of the first and second nucleotide sequence. Such sequences can be referred to as \u201cfully complementary\u201d with respect to each other herein. However, where a first sequence is referred to as \u201csubstantially complementary\u201d with respect to a second sequence herein, the two sequences can be fully complementary, or they may form one or more, but generally not more than 4, 3 or 2 mismatched base pairs upon hybridization, while retaining the ability to hybridize under the conditions most relevant to their ultimate application. However, where two oligonucleotides are designed to form, upon hybridization, one or more single stranded overhangs, such overhangs shall not be regarded as mismatches with regard to the determination of complementarity. For example, a dsRNA comprising one oligonucleotide 21 nucleotides in length and another oligonucleotide 23 nucleotides in length, wherein the longer oligonucleotide comprises a sequence of 21 nucleotides that is fully complementary to the shorter oligonucleotide, may yet be referred to as \u201cfully complementary\u201d for the purposes described herein.","\u201cComplementary\u201d sequences, as used herein, may also include, or be formed entirely from, non-Watson-Crick base pairs and\/or base pairs formed from non-natural and modified nucleotides, in as far as the above requirements with respect to their ability to hybridize are fulfilled. Such non-Watson-Crick base pairs includes, but not limited to, G:U Wobble or Hoogstein base pairing.","The terms \u201ccomplementary,\u201d \u201cfully complementary\u201d and \u201csubstantially complementary\u201d herein may be used with respect to the base matching between two strands of a dsRNA, or between the antisense strand of a dsRNA and a target sequence, between complementary strands of a single stranded RNA sequence or a single stranded DNA sequence, as will be understood from the context of their use.","As used herein, a \u201cduplex structure\u201d comprises two anti-parallel and substantially complementary nucleic acid sequences. Complementary sequences in a nucleic acid construct, between two transcripts, between two regions within a transcript, or between a transcript and a target sequence can form a \u201cduplex structure.\u201d In general, the majority of nucleotides of each strand are ribonucleotides, but as described in detail herein, each or both strands can also include at least one non-ribonucleotide, e.g., a deoxyribonucleotide and\/or a modified nucleotide. The two strands forming the duplex structure may be different portions of one larger RNA molecule, or they may be separate RNA molecules. Where the two strands are part of one larger molecule, and therefore are connected by an uninterrupted chain of nucleotides between the 3\u2032-end of one strand and the 5\u2032-end of the respective other strand forming the duplex structure, the connecting RNA chain is referred to as a \u201chairpin loop.\u201d Where the two strands are connected covalently by means other than an uninterrupted chain of nucleotides between the 3\u2032-end of one strand and the 5\u2032-end of the respective other strand forming the duplex structure, the connecting structure is referred to as a \u201clinker.\u201d The RNA strands may have the same or a different number of nucleotides. The maximum number of base pairs is the number of nucleotides in the shortest strand of the duplex minus any overhangs that are present in the duplex. Generally, the duplex structure is between 15 and 30 or between 25 and 30, or between 18 and 25, or between 19 and 24, or between 19 and 21, or 19, 20, or 21 base pairs in length. In one embodiment the duplex is 19 base pairs in length. In another embodiment the duplex is 21 base pairs in length. When two different siRNAs are used in combination, the duplex lengths can be identical or can differ.","As used herein, the term \u201cregion of complementarity\u201d refers to the region on the antisense strand that is substantially complementary to a sequence, for example a target sequence, as defined herein. Where the region of complementarity is not fully complementary to the target sequence, the mismatches are most tolerated in the terminal regions and, if present, are generally in a terminal region or regions, e.g., within 6, 5, 4, 3, or 2 nucleotides of the 5\u2032 and\/or 3\u2032 terminus.","The term percent \u201cidentity,\u201d in the context of two or more nucleic acid or polypeptide sequences, refer to two or more sequences or subsequences that have a specified percentage of nucleotides or amino acid residues that are the same, when compared and aligned for maximum correspondence, as measured using one of the sequence comparison algorithms described below (e.g., BLASTP and BLASTN or other algorithms available to persons of skill) or by visual inspection. Depending on the application, the percent \u201cidentity\u201d can exist over a region of the sequence being compared, e.g., over a functional domain, or, alternatively, exist over the full length of the two sequences to be compared.","For sequence comparison, typically one sequence acts as a reference sequence to which test sequences are compared. When using a sequence comparison algorithm, test and reference sequences are input into a computer, subsequence coordinates are designated, if necessary, and sequence algorithm program parameters are designated. The sequence comparison algorithm then calculates the percent sequence identity for the test sequence(s) relative to the reference sequence, based on the designated program parameters.","Optimal alignment of sequences for comparison can be conducted, e.g., by the local homology algorithm of Smith & Waterman, Adv. Appl. Math. 2:482 (1981), by the homology alignment algorithm of Needleman & Wunsch, J. Mol. Biol. 48:443 (1970), by the search for similarity method of Pearson & Lipman, Proc. Nat'l. Acad. Sci. USA 85:2444 (1988), by computerized implementations of these algorithms (GAP, BESTFIT, FASTA, and TFASTA in the Wisconsin Genetics Software Package, Genetics Computer Group, 575 Science Dr., Madison, Wis.), or by visual inspection (see generally Ausubel et al., infra).","One example of an algorithm that is suitable for determining percent sequence identity and sequence similarity is the BLAST algorithm, which is described in Altschul et al., J. Mol. Biol. 215:403-410 (1990). Software for performing BLAST analyses is publicly available through the National Center for Biotechnology Information.","The term \u201csufficient amount\u201d means an amount sufficient to produce a desired effect, e.g., an amount sufficient to produce a detectable signal from a cell.","The term \u201ctherapeutically effective amount\u201d is an amount that is effective to ameliorate a symptom of a disease. A therapeutically effective amount can be a \u201cprophylactically effective amount\u201d as prophylaxis can be considered therapy.","It must be noted that, as used in the specification and the appended claims, the singular forms \u201ca,\u201d \u201can\u201d and \u201cthe\u201d include plural referents unless the context clearly dictates otherwise.","Viruses undergo lysogenic and lytic cycles in a host cell. If the lysogenic cycle is adopted, the phage chromosome can be integrated into the bacterial chromosome, or it can establish itself as a stable plasmid in the host, where it can remain dormant for long periods of time. If the lysogen is induced, the phage genome is excised from the bacterial chromosome and initiates the lytic cycle, which culminates in lysis of the cell and the release of phage particles. The lytic cycle leads to the production of new phage particles which are released by lysis of the host.","Certain temperate phage can exhibit lytic activity, and the propensity for this may vary with varying host bacteria. To illustrate this phenomenon, the lytic activity of two temperate phages on ten MRSA clinical isolates was examined via plaque assay (Table 1). The phage \u03c611 exhibited lytic activity on 10 out of 10 clinical MRSA isolates and \u03c680\u03b1 exhibited lytic activity on six of the 10 clinical MRSA isolates. Thus, reporter assays relying on the natural lysogenic cycle of phages can be expected to exhibit lytic activity sporadically.",{"@attributes":{"id":"p-0142","num":"0141"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Lytic activity (denoted by the letter \u201cx\u201d) of the temperate"},{"entry":"phases \u03c611 and \u03c680\u03b1 on ten clinical MRSA isolates"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["MRSA isolate","\u03c611","\u03c680\u03b1"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"112pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1","x",{}]},{"entry":["2","x"]},{"entry":["3","x","x"]},{"entry":["4","x","x"]},{"entry":["5","x","x"]},{"entry":["6","x"]},{"entry":["7","x","x"]},{"entry":["8","x"]},{"entry":["9","x","x"]},{"entry":["10","x","x"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"In addition, virus-based reporter assays, such as phage-based reporters, can suffer from limited reactivity (i.e., analytical inclusivity) due to limits in the phage host range caused by host-based and prophage-derived phage resistance mechanisms. These resistance mechanisms target native phage nucleic acid that can result in the degradation or otherwise inhibition of the phage DNA and functions. Such resistance mechanisms include restriction systems that cleave phage DNA and CRISPR systems that inhibit phage-derived transcripts.","Both lytic activity and phage resistance can be inhibitory to assays based on reporter phages. Lytic activity can inhibit signal by destroying or otherwise inhibiting the cell in its ability to generate a detectable signal and thus affecting limits of detection by reducing the amount of detectable signal or preventing the generation of a detectable signal. Phage resistance mechanisms can limit the host range of the phage and limit the inclusivity of the phage-based reporter, similarly affecting limits of detection by reducing the amount of detectable signal or preventing the generation of a detectable signal. Both lytic activity and phage resistance caused by the incorporation of phage DNA in a reporter phage can lead to false-negative results in assays that incorporate these phage reporters.","A. Disruption\/Complementation-Based Methods for Producing Non-Replicative Transduction Particles","1) Silent Mutation\/Complementation Packaging System","The invention includes methods for producing NRTPs using a silent mutation\/complementation-based method.","This non-replicative transduction particle packaging system is based on introducing a silent mutation into a component of the genome of a virus that is recognized by the viral packaging machinery as the element from which genomic packaging is initiated during viral production. Examples of such an element include the pac-site sequence of pac-type bacteriophages and the cos-site sequence of cos-type bacteriophages.","Because these packaging initiation sites are often found within coding regions of genes that are essential to virus production, the silent mutation is introduced such that the pac-site is no longer recognized as a site of packaging initiation by the viral packaging machinery. At the same time, the mutation does not disrupt the gene in which the site is encoded. By disrupting the packaging site sequence, the mutated virus is able to undergo a lytic cycle, but is unable to package its genomic DNA into its packaging unit.","An exogenous reporter nucleic acid molecule, such as plasmid DNA, can be introduced into a host cell that has been lysogenized with a viral genome with a mutated packaging initiation site sequence. The exogenous reporter nucleic acid molecule can include a native packaging initiation site sequence. The exogenous reporter nucleic acid molecule can be introduced into the cell and replicated in the cell. When the mutated virus is undergoing a lytic cycle, the expressed viral packaging machinery packages the exogenous reporter nucleic acid molecule with the native packaging initiation site sequence into the viral packaging unit. The viral genome is not packaged into the packaging unit because its packaging initiation site sequence has been mutated. In certain embodiments, the mutation in the packaging initiation site sequence comprises a silent mutation that prevents cleavage of the packaging initiation sequence, but does not disrupt the expression of the gene product that encompasses the packaging initiation site sequence. This produces non-replicative transduction particles, e.g., viral structural components carrying the replicated exogenous nucleic acid molecule.","An example of such a system is based on the bacteriophage P1, a pac-type phage. In an embodiment, a plasmid including a native P1 pac site is transformed into a cell. The cell is lysogenized with a P1 prophage genome. The P1 prophage genome includes a silent mutation in the pac-site sequence encoded within the pacA gene of P1. When the lytic cycle of the prophage is induced, the system results in the production of P1-based transduction particles carrying the plasmid DNA. An example of a silent mutation that is suitable for this system is described in U.S. Pub. No. 2005\/0118719, filed on Nov. 7, 2002, which is incorporated by reference in its entirety. An example is also found in SEQ ID NO: 2, listed below (P1 pac-site with silent mutations, lower case letters signify mutated bases).",{"@attributes":{"id":"p-0152","num":"0151"},"figref":"FIG. 1","b":["100","101","102","103","104","105","106"],"i":"E. coli "},"An example vector for use in the silent mutation\/complementation-based P1 plasmid packaging system is shown in . Details about how to construct the strains and vectors of the silent mutation\/complementation-based P1 plasmid packaging system are described in detail in Example 1 below.","2) Deletion\/Complementation-Based Packaging System","The invention includes methods for producing NRTPs using a deletion\/complementation-based method.","This non-replicative transduction particle packaging system is based on deletion of a component of the genome of a virus that is recognized by the viral packaging machinery as the element from which genomic packaging is initiated during viral production. Examples of such an element include the pac-site sequence of pac-type bacteriophages and the cos-site sequence of cos-type bacteriophages. These packaging initiation sites are often found within coding regions of genes that are essential to virus production. In some embodiments, the packaging initiation site alone is deleted, which allows the mutated virus to undergo a lytic cycle but does not allow the virus to package its genomic DNA. For example, SEQ ID NO: 6 is an example of a P1 pacA gene with a deleted pac-site sequence (lower case letters indicate the deleted pac-site sequence). In other embodiments, the entire gene comprising the packaging initiation site is deleted. For example, SEQ ID NO: 8 shows the deletion of the terS gene (lower case characters show the deleted sequence).","In one example, a cell's genome is lysogenized with a viral genome where the packaging initiation site has been deleted. A complementing plasmid is introduced into the cell, and the plasmid DNA includes a gene with a packaging initiation site sequence that complements the deleted packaging initiation site sequence in the viral genome. When the mutated virus is undergoing a lytic cycle, the viral packaging proteins package a replicon of the plasmid DNA into the packaging unit because of its packaging initiation site, and non-replicative transduction particles are produced carrying the replicated plasmid DNA.","In some embodiments, it is preferable that the deletion\/complementation is designed such that there is no homology between the mutated virus DNA and the complementing exogenous DNA. This is because lack of homology between the mutated virus DNA and the complementing exogenous DNA avoids the possibility of homologous recombination between the two DNA molecules that can result in re-introduction of a packaging sequence into the virus genome. To accomplish a lack of homology, one strategy is to delete the entire gene that contains the packaging initiation site sequence from the virus genome and then complement this gene with an exogenous DNA molecule that contains no more than exactly the DNA sequence that was deleted from virus. In this strategy, the complementing DNA molecule is designed to express the gene that was deleted from the virus.","Another example of such a system is provided using the bacteriophage \u03c680\u03b1, a pac-type phage. The phage genome is lysogenized in a host bacterial cell, and the phage genome includes a small terminase gene where the pac-site of a pac-type prophage \u03c680\u03b1 has been deleted. A plasmid including a complementary small terminase gene with a native pac-site is transformed into the cell. When the lytic cycle of the lysogenized prophage is induced, the bacteriophage packaging system packages plasmid DNA into progeny bacteriophage structural components, rather than packaging the native bacteriophage DNA. The packaging system thus produces non-replicative transduction particles carrying plasmid DNA.",{"@attributes":{"id":"p-0160","num":"0159"},"figref":"FIG. 3","b":["300","301","302","303","303"]},"A protein complex comprising the small terminase and large terminase proteins is able to recognize and cleave a double-stranded DNA molecule at or near the pac-site, and this allows the plasmid DNA molecule to be packaged into a phage capsid. When the prophage in the cell is induced, the lytic cycle of the phage produces the phage's structural proteins  and the phage's large terminase protein . The complementing plasmid is replicated, and the small terminase protein  is expressed. The replicated plasmid DNA  containing the terS gene (and the reporter gene) are packaged into phage capsids, resulting in non-replicative transduction particles carrying only plasmid DNA .  shows an example of a resulting vector used in the pac-site deletion\/complementation plasmid packaging system. Further details about the components and construction of pac-site deletion\/complementation plasmid packaging system are in Example 2 below.","B. Pathogenicity Island-Based Packaging System","Pathogenicity islands (PTIs) are a subset of horizontally transferred genetic elements known as genomic islands. There exists a particular family of highly mobile PTIs in that are induced to excise and replicate by certain resident prophages. These PTIs are packaged into small headed phage-like particles and are transferred at frequencies commensurate with the plaque-forming titer of the phage. This process is referred to as the SaPI excision replication-packaging (ERP) cycle, and the high-frequency SaPI transfer is referred to as SaPI-specific transfer (SPST) to distinguish it from classical generalized transduction (CGT). The SaPIs have a highly conserved genetic organization that parallels that of bacteriophages and clearly distinguishes them from all other horizontally acquired genomic islands. The SaPI1-encoded and SaPIbov2-encoded integrases are required for both excision and integration of the corresponding elements, and it is assumed that the same is true for the other SaPIs. Phage 80\u03b1 can induce several different SaPIs, including SaPI1, SaPI2, and SaPIbov1, whereas \u03c611 can induce SaPIbov1 but neither of the other two SaPIs.",{"@attributes":{"id":"p-0164","num":"0163"},"figref":"FIG. 5","b":["500","501","503","504","512","502","505","506","507","508","509","510","511","512"]},"In natural systems, as depicted in , the resulting lysate produced from phage production includes both native phage particles, as well as GI-containing phage particles. The native phage particles are a result of packaging of the native phage genome due to recognition of the pac-site within phage genome concatamers.","1) Genomic Island (GI) Packaging System Design and Function","Methods of the invention for producing NRTPs include a GI based-packaging system.","Compared to a plasmid packaging system, the natural GI-packaging system benefits from the fact that the DNA that is packaged is derived from a genomic region within the bacterial genome and thus does not require the maintenance of a plasmid by the bacterial host.","In some embodiments, the invention includes a bacterial cell packaging system for packaging a reporter nucleic acid molecule into a non-replicative transduction particle, wherein the bacterial cell comprises a lysogenized bacteriophage genome lacking a packaging gene, and a genomic island, cryptic phage, or other nucleic acid molecule requiring a bacteriophage (e.g., a heper phage) for mobilization of the nucleic acid molecule and comprising a reporter nucleic acid molecule and a packaging gene. Genomic island-based systems can be based on Pathogenicity Islands (SaPIs), the criptic phage P4 and helper phage P2, and the criptic phage P7 and helper phage P1, for example.","GI-packaging systems can be exploited such that exogenous nucleic acid sequences are packaged by the bacteriophage. This can be accomplished by incorporating such exogenous nucleic acids sequences into the GI.","In order to eliminate the native phage from this process, the small terminase gene of the prophage can be deleted. The small terminase gene sequence contains the pac-site sequence of the native phage, and this deletion has the effect of preventing the packaging of native phage DNA. In other embodiments, only the pac site of the small terminase gene can be deleted. The GI that will be packaged includes its own pac-site and a small terminase gene that expresses a suitable small terminase protein, and only GI DNA will be amenable for packaging in this system.",{"@attributes":{"id":"p-0172","num":"0171"},"figref":"FIG. 6","b":["600","601","603","602","604","602","605","606","607","608","609","610","611","612"]},"When phage particles containing packaged GI DNA are administered to a recipient cell, the phage will bind to the recipient cell's surface and then introduce the packaged GI DNA concatamer into the cell. Once inside the cell, the GI can again express its integrase protein, and the GI can then integrate into its specific site in the recipient cell's genome. If exogenous DNA sequences are included in the GI prior to packaging, the packaging system thus allows for delivering exogenous DNA sequences to a recipient cell and integrating these exogenous DNA sequences into the recipient cell's genome.","2) GI-Based Packaging System Lacking Integrase","In another embodiment, the packaging system described above is designed such that packaged GI DNA cannot integrate into a recipient cell's genome. This can be accomplished by deleting the integrase gene in the GI and complementing the deletion by causing the expression of the integrase gene in trans from the GI. In this manner, the integrase protein is available for excision of the GI in the packaging host cell, and the GI DNA that has been packaged in a bacteriophage does not contain the integrase gene and cannot express the integrase protein, thus preventing integration of the delivered GI.",{"@attributes":{"id":"p-0176","num":"0175"},"figref":"FIG. 7","b":["700","701","703","702","704","705","706","702","707","708","707","709","710","711","712","713"]},"When phage particles containing packaged GI DNA lacking the int gene are administered to a recipient cell, the phage will bind to the recipient cell's surface and then introduce the packaged GI DNA concatamer into the cell. Once inside the cell, the GI cannot express its integrase protein due to the lack of the integrase gene and the GI cannot then integrate into its specific site in the recipient cell's genome. If exogenous DNA sequences are included in the GI prior to packaging, the packaging system thus allows for delivering exogenous DNA sequences to a recipient cell and the delivered DNA sequences do not integrate into the recipient cell's genome at the specific site for GI integration.","3) Design and Function of SaPIbov2-Based Packaging Lacking Integrase","In some embodiments, the method of producing NRTPs employ a GI SaPIbov2 and a bacteriophage \u03c611 in a GI-based packaging system. Alternative embodiments can employ other SaPI GI's and other suitable bacteriophages, including the SaPI's SaPI1, SaPI2, SaPIbov1, and SaPIbov2 along with the bacteriophage 80\u03b1, and the SaPI's SaPIbov1 and SaPIbov2 along with the bacteriophage \u03c611. Based on the description below, one of skill in the art would know how to develop a GI-based packaging system that does not lack the int gene, as described in Section II A.",{"@attributes":{"id":"p-0180","num":"0179"},"figref":"FIG. 8","b":["800","801","803","802","804","805","806","802","807","808","806","809","810","811","812","813"],"i":"S. aureus "},"In some embodiments, the NRTPs and constructs of the invention comprise a reporter nucleic acid molecule including a reporter gene. The reporter gene can encode a reporter molecule, and the reporter molecule can be a detectable or selectable marker. In certain embodiments, the reporter gene encodes a reporter molecule that produces a detectable signal when expressed in a cell.","In certain embodiments, the reporter molecule can be a fluorescent reporter molecule, such as, but not limited to, a green fluorescent protein (GFP), enhanced GFP, yellow fluorescent protein (YFP), cyan fluorescent protein (CFP), blue fluorescent protein (BFP), red fluorescent protein (RFP) or mCherry, as well as near-infrared fluorescent proteins.","In other embodiments, the reporter molecule can be an enzyme mediating luminescence reactions (luxA, luxB, luxAB, luc, ruc, nluc, etc.). Reporter molecules can include a bacterial luciferase, a eukaryotic luciferase, an enzyme suitable for colorimetric detection (lacZ, HRP), a protein suitable for immunodetection, such as affinity peptides (His-tag, 3X-FLAG), a nucleic acid that function as an aptamer or that exhibits enzymatic activity (ribozyme), or a selectable marker, such as an antibiotic resistance gene (ampC, tet(M), CAT, erm). Other reporter molecules known in the art can be used for producing signals to detect target nucleic acids or cells.","In other aspects, the reporter molecule comprises a nucleic acid molecule. In some aspects, the reporter molecule is an aptamer with specific binding activity or that exhibits enzymatic activity (e.g., aptazyme, DNAzyme, ribozyme).","Reporters and reporter assays are described further in Section V herein.","A. Inducer Reporter Assay","The invention comprises methods for the use of NRTPs as reporter molecules for use with endogenous or native inducers that target gene promoters within viable cells. The NRTPs of the invention can be engineered using the methods described in Section III and below in Examples 1-6.","In some embodiments, the method comprises employing a NRTP as a reporter, wherein the NRTP comprises a reporter gene that is operably linked to an inducible promoter that controls the expression of a target gene within a target cell. When the NRTP that includes the reporter gene is introduced into the target cell, expression of the reporter gene is possible via induction of the target gene promoter in the reporter nucleic acid molecule.",{"@attributes":{"id":"p-0189","num":"0188"},"figref":"FIG. 9","b":["900","902","903","904","905","906","904","902","907","907","906","908"]},"When the reporter nucleic acid molecule  is present within the target organism, the inducer  is also able to induce the target gene promoter  present within the reporter nucleic acid molecule , thus causing the expression of the reporter gene  resulting in the production of a reporter molecule  capable of generating a detectable signal.","Thus, the production of a detectable signal from the reporter molecule  is indicative of the presence of the cell, based on the presence of the inducer protein  within a target cell.","1) VanR Reporter System","In one embodiment, the reporter system includes NRTP comprising a reporter nucleic acid molecule (e.g., plasmid). The reporter nucleic acid molecule can be constructed for detecting VanR, the inducer of the promoter of the vancomycin resistance (vanA) gene in (or ). The reporter plasmid carries a reporter gene that is operatively linked to the vanA gene promoter.",{"@attributes":{"id":"p-0194","num":"0193"},"figref":["FIG. 10","FIG. 10"],"b":["1001","1002","1003","1004","1004","1005","1006","1002","1007","1006","1008","1003","1006","1003","1009"],"i":"E. faecium","sub":["H ","H ","H "]},"Examples of promoters that are suitable for the development of a VRE assay include: the vanA gene promoter and a vanB gene promoter. Arthur, M., et al., -1546 . J. Bacteriol., 1997. 179(1): p. 97-106.","2) TcdD Reporter System","In another embodiment of this system, a reporter nucleic acid molecule is introduced into a cell using a NRTP. The reporter nucleic acid molecule can be constructed for detecting TcdD, the inducer of the promoters of the toxins A and B genes (tcdA and tcdB, respectively) of . The reporter nucleic acid molecule includes a reporter gene that is operatively linked to the tcdA gene promoter.",{"@attributes":{"id":"p-0198","num":"0197"},"figref":["FIG. 11","FIG. 11"],"b":["1101","1102","1103","1104","1104","1105","1106"],"i":"C. difficile","sub":"tcdA"},"In the native cell, when the tcdD gene is expressed and produces the TcdD protein , TcdD is able to induce P in the PaLoc transposon , thus causing the expression of the tcdA gene  and thus producing the toxin A protein .","When the reporter nucleic acid molecule  is present within the target organism, TcdD is also able to induce P within the reporter vector, thus causing the expression of a reporter molecule . Thus, the production of a reporter molecule  is indicative of the presence of TcdD within a target cell.","Examples of promoters suitable for the development of a assay include: the tcdA gene promoter and the tcdB gene promoter. Karlsson, S., et al., 2003. 71(4): p. 1784-1793.","Target Cells and Inducers:","Target cells can include eukaryotic and prokaryotic cell targets and associated inducers.","Vector Delivery Systems:","The delivery of the vector containing the recombinant DNA can by performed by abiologic or biologic systems. Including but not limited to liposomes, virus-like particles, transduction particles derived from phage or viruses, and conjugation.","3) Bacteriophage-Based SarS Reporter System","In another embodiment of the invention, a reporter nucleic acid molecule is constructed for detecting SarS, the inducer of the promoter of the Protein A gene (spa) in . The reporter nucleic acid molecule can be introduced into the cell in a NRTP and includes the bacterial luciferase genes luxA and luxB operatively linked to the spa gene promoter (P). The reporter nucleic acid molecule is delivered to via a NRTP, for example. If SarS is present in the cell, it will induce the expression of the luxAB genes, thus producing luciferase enzyme that is capable of generating a luminescent signal.",{"@attributes":{"id":"p-0208","num":"0207"},"figref":["FIG. 12","FIG. 12"],"i":"S. aureus ","b":["1201","1202","1203","1204","1205","1206","1203"],"sub":"spa "},"In the native cell, when the sarS gene  is expressed, producing SarS protein , the protein is able to induce P in the genome transposon, thus causing the expression of the spa gene  and producing the Protein A .","When the reporter nucleic acid molecule  is present within the target organism, SarS  is also able to induce P within the reporter nucleic acid molecule , thus causing the expression of luxAB resulting in the production of the luciferase enzyme  that can generate a luminescent signal. Thus, the production of luciferase is indicative of the presence of SarS within a target cell.","B. Enzyme Reporter Assay","The invention comprises a system for the detection of intracellular enzymes within viable cells that employs caged substrate molecules that can be un-caged by a target intracellular enzyme, according to an embodiment of the invention.",{"@attributes":{"id":"p-0213","num":"0212"},"figref":"FIG. 13","b":["1301","1302","1301","1302","1303","1302","1304","1303","1305","1302","1302","1307","1306","1307","1305","1308","1308","1304","1302","1309"]},"Target Cells and Enzymes:","Target cells can include eukaryotic and prokaryotic cell targets and associated enzymes, including, for example, \u03b2-lactamases in ","Vector Delivery Systems:","The delivery of the vector containing the recombinant DNA can by performed by abiologic or biologic systems. Including but not limited to liposomes, virus-like particles, transduction particles derived from phage or viruses, and conjugation.","Reporter Molecules and Caged Substrates:","Various reporter molecules and caged substrates can be employed as those described in Daniel Sobek, J. R., Enzyme detection system with caged substrates, 2007, Zymera, Inc.","1) Bacteriophage-Based \u03b2-Lactamase Reporter","In one embodiment, a reporter molecule-expressing vector can be carried by a NRTP, such that the vector can be delivered into a bacterial cell. The reporter molecule to be expressed can be luciferase, and the caged substrate can be luciferin that is caged, such that a \u03b2-lactamase enzyme that is endogenous to the target cell is able to cleave the caging compound from the caged luciferin and release un-caged luciferin.",{"@attributes":{"id":"p-0222","num":"0221"},"figref":"FIG. 14","i":["Renilla ","S. aureus ","Renilla ","Renilla ","Renilla ","Renilla ","Renilla "],"b":["1401","1402","1402","1403","1402","1404","1405","1402","1402","1407","1402","1406","1408","1408","1404","1402","1409"]},"In this manner, when a target cell that contains the \u03b2-lactamase is exposed to the NRTP and caged luciferin, the cell will exhibit a luminescent signal that is indicative of the presence of the \u03b2-lactamase present in the cell.","C. Intracellular Molecule Reporter","The invention includes a system for the detection of intracellular molecules within viable cells that employs switchable molecules capable of generating a detectable signal upon their binding to a target molecule.",{"@attributes":{"id":"p-0226","num":"0225"},"figref":"FIG. 15","b":["1501","1502","1501","1502","1503","1502","1504","1503","1504","1505","1506","1504","1505","1504","1508","1507","1502","1502","1509","1508","1510"]},"Target Cells and Molecules:","Various eukaryotic and prokaryotic cell targets can be employed and switchable aptamer-based SM's can be designed to target various nucleic acid and amino acid-based intracellular molecular targets as described in Samie Jaffrey, J. P., 2010, Cornell University.","Vector Delivery Systems:","The delivery of the vector containing the recombinant DNA can by performed by abiologic or biologic systems. Including but not limited to liposomes, virus-like particles, transduction particles derived from phage or viruses, and conjugation.","1) Non-Replicative Transduction Particle\/Switchable Aptamer-Based Intracellular Molecule Reporter System","In one example of this method, a switchable molecule-expressing vector can be carried by a bacteriophage-based transduction particle such that the vector can be delivered into a bacterial cell. The switchable molecule to be expressed can be a switchable aptamer that is designed to undergo a conformational change upon its binding to an intracellular target molecule. The conformational change allows the aptamer to then bind a fluorophore that exhibits enhanced fluorescence when bound by the aptamer.",{"@attributes":{"id":"p-0233","num":"0232"},"figref":"FIG. 16","b":["1601","1602","1601","1603","1602","1604","1603","1604","1605","1606","1604","1605","1608","1607","1608","1609","1610"]},"D. Transcript Reporter Assay","The invention comprises a reporter assay comprising an antisense RNA-based method for detecting target transcripts within viable cells by causing the expression of a reporter molecule if a target transcript is present within a cell.","Certain intracellular methods in the art for inhibiting gene expression employ small interfering RNA, such as double-stranded RNA (dsRNA), to target transcribed genes in cells. The dsRNA comprise antisense and sense strands that are delivered into or expressed in cells, and the strands of the dsRNA act via a trans-acting inhibition mechanism, where one strand (typically the antisense strand) binds to a target gene sequence (RNA transcript) and prevents expression of the target gene sequence. Double-stranded RNA molecules have been shown to block (knock down) gene expression in a highly conserved regulatory mechanism known as RNA interference (RNAi). WO 99\/32619 (Fire et al.) discloses the use of a dsRNA of at least 25 nucleotides in length to inhibit the expression of genes in . dsRNA has also been shown to degrade target RNA in other organisms, including plants (see, e.g., WO 99\/53050, Waterhouse et al.; and WO 99\/61631, Heifetz et al.), (see, e.g., Yang, D., et al., Curr. Biol. (2000) 10:1191-1200), and mammals (see WO 00\/44895, Limmer; and DE 101 00 586.5, Kreutzer et al.). However, binding of a strand of the dsRNA to the target gene can be non-specific. If a similar mechanism were to be applied to a detection system, this non-specific binding can result in high false positive rates, which make it unsuitable for the development of clinically useful detection systems.","Previous trans-acting inhibition mechanisms have been shown to be unsuitable for development of clinically useful detection systems. For example, some methods result in high levels of non-specific signals and up to 90% false positive rate, when achieving a 90% sensitivity of the assay. See U.S. Pat. No. 8,329,889. Certain methods for post-transcriptional regulation of gene expression have been developed that use a cis-repressed marker transcript, such as a green fluorescent protein marker, where the ribosomal binding site of the marker is blocked by the cis-repressing sequence, along with a trans-activating RNA transcript. When the trans-activating RNA transcript binds to the cis-repressed marker transcript, the hairpin structure of the cis-repressed marker transcript is altered, and the upstream ribosome binding site of the marker gene is exposed, allowing transcription and expression of the marker gene. However, these methods have not previously been used for the detection of endogenous transcripts, nor successful beyond a basic switching mechanism for controlling expression of genes in cells.","1) Nucleic Acid Molecule Interactions and Mechanisms","The methods of the invention take advantage of the transcript-level regulation mechanisms, including antisense RNA (asRNA) mechanism in cells, to deliver nucleic acid molecules into cells. The antisense mechanism includes all forms of sequence-specific mRNA recognition leading to reduced, eliminated, increased, activated, or otherwise altered expression of a target transcript. See Good, L., . Cellular and Molecular Life Sciences, 2003. 60(5): p. 854-861, and Lioliou, E., -, New Biotechnology. 2010. 27(3): p. 222-235. Naturally occurring asRNAs are found in all three kingdoms of life, and they affect messenger RNA (mRNA) destruction, repression and activation, as well as RNA processing and transcription. See Sabine, B., -. Biochimica et Biophysica Acta (BBA)\u2014Gene Structure and Expression, 2001. 1575(1-3): p. 15-25. This mechanism has been exploited in inhibiting protein synthesis for therapeutic applications.","Antisense RNA is a single-stranded RNA that is complementary to a messenger RNA (mRNA) strand transcribed within a cell. asRNA may be introduced into a cell to inhibit translation of a complementary mRNA by base pairing to it and physically obstructing the translation machinery. Antisense RNA anneal to a complementary mRNA target sequence, and translation of the mRNA target sequence is disrupted as a result of steric hindrance of either ribosome access or ribosomal read through.","The antisense RNA mechanism is different from RNA interference (RNAi), a related process in which double-stranded RNA fragments (dsRNA, also called small interfering RNAs (siRNAs)) trigger catalytically mediated gene silencing, most typically by targeting the RNA-induced silencing complex (RISC) to bind to and degrade the mRNA. Annealing of a strand of the dsRNA molecule to mRNA or DNA can result in fast degradation of duplex RNA, hybrid RNA\/DNA duplex, or duplex RNA resembling precursor tRNA by ribonucleases in the cell, or by cleavage of the target RNA by the antisense compound itself.","The RNAi pathway is found in many eukaryotes and is initiated by the enzyme Dicer, which cleaves long double-stranded RNA (dsRNA) molecules into short double stranded fragments of \u02dc20 nucleotides that are called siRNAs. Each siRNA is unwound into two single-stranded RNAs (ssRNA), namely the passenger strand and the guide strand. The passenger strand is degraded, and the guide strand is incorporated into the RNA-induced silencing complex (RISC). In post-transcriptional gene silencing, the guide strand base pairs with a complementary sequence in a messenger RNA molecule, and cleavage is induced by a protein called Argonaute, the catalytic component of the RISC complex.","In regards to the nucleic acid interactions of the mechanisms of the invention, interactions between a reporter transcript and a target transcript can rely on base pairings between loops present in both transcripts (e.g., \u201ckissing complexes\u201d), or between a loop and a single-stranded (ss) region. In some cases, the kissing complex formation suffices for mediating the desired effect of the interaction, and in other cases, propagation of the primary contacts will lead to an interaction resulting in the desired effect.","2) Mechanisms for Cis-Repression and Trans-Activation of Translation of a Reporter Construct Via Transcript-Level Regulation","The following description illustrates transcript reporter systems based on various repression\/activation mechanisms that can be used, according to embodiments of this invention. In each of , a vector includes a reporter construct comprising a reporter sequence, and the regions on the reporter construct are shown in each of the figures, including regions that can be targeted for repression by a cis-repressing sequence. The description below provides non-limiting examples of various inhibition mechanisms, including transcription attenuation, translation attenuation, and destabilization of the transcript, and various activation mechanisms including conformational changes and cleavage.",{"@attributes":{"id":"p-0246","num":"0245"},"figref":"FIG. 17","b":["1700","1701","1702","1703","1702","5","1701","1705","1701","1704","1703","1706"]},"At some point during transcription, the transcription process is stopped by the formation of a transcription termination (TT) stem-loop structure  in the reporter transcript , due to an interaction within the transcribed cis-repressing sequence . The transcription termination  structure stops  the RNA polymerase  from transcribing the vector . In some embodiments, a transcription termination protein (e.g., NusA in ) binds to RNA polymerase and\/or to the transcription termination  structure to cease transcription of the reporter construct.","When a target transcript  is present in the cell, the target transcript  binds to the reporter transcript . In some embodiments, the binding between the target transcript and the reporter transcript is by base pairing of the nucleotides in each sequence. The interaction between the target transcript  and the reporter transcript  causes the transcription termination (TT) stem-loop structure  to be cleaved . Cleavage of the reporter transcript  can occur by a cellular enzyme, such as RNase III, for example. In this case, the secondary structure of a target transcript is analyzed for the presence of an RNAse III consensus sequence among the ssRNA regions of the secondary structure, for example 5\u2032-nnWAWGNNNUUN-3\u2032 or 5\u2032-NAGNNNNCWUWnn-3\u2032 where \u201cN\u201d and \u201cn\u201d are any nucleotide and \u201cW\u201d is A or U and \u201cN\u201d indicates a relatively strict requirement for Watson-Crick base pairing, while \u201cn\u201d indicates a minimal requirement for base pairing. When such a consensus sequence is found on a target transcript, the loop of the transcription termination structure  can be designed to be complementary to said RNAse III consensus sequence such that when the ssRNA in each RNA molecule hybridize, the RNAse III cleavage site is formed allowing for cleavage of the transcription termination structure . In the mecA transcript, loop T23, starting at nucleotide 1,404, has the sequence CAGAUAACAUUUU that is suitable for such an approach.","In some embodiments, a cleavage site is engineered in the reporter construct, such that the reporter transcript is cleaved after transcription. The cleavage, in the example provided, can occur immediately adjacent to the location of the loop in the transcription terminator structure. Transcription is re-initiated  by the RNA polymerase . Cleavage of the transcription termination (TT) stem-loop structure  allows the remainder of the reporter sequence  to be transcribed and subsequently translated. This results in the production of a detectable or selectable marker from the translated reporter molecule.","In prokaryotes, the transcription termination structure  involves a Rho-independent mechanism with a stem-loop structure that is 7-20 base pairs in length, rich in cytosine-guanine base pairs and is followed by a chain of uracil residues. NusA binds to the transcription termination stem-loop structure  causing RNA polymerase to stall during transcription of the poly-uracil sequence. Weak Adenine-Uracil bonds lower the energy of destabilization for the RNA-DNA duplex, allowing it to unwind and dissociate from the RNA polymerase. In eukaryotes, the transcription termination structure  is recognized by protein factors and involves cleavage of the new transcript followed by polyadenylation.",{"@attributes":{"id":"p-0251","num":"0250"},"figref":"FIG. 18","b":["1800","1801","1702","1703","1801","1802","1705","1801","1705","1801","1801","1705","1802","1703","109","1703","1709","1801","1703","1705","1801","1802","1801","1703"]},{"@attributes":{"id":"p-0252","num":"0251"},"figref":"FIG. 19","b":["1900","1901","1702","1703","1705","1901","1702","1901","1705","1901","1902","1703","1703"]},"When a target transcript  is present in the cell, the target transcript  binds to the cis-repressing sequence  in a manner that causes a conformational change in the reporter transcript . This conformational change prevents or removes the interaction between the cis-repressing sequence  and the coding region  of the reporter sequence , thereby allowing for translation of the reporter sequence .",{"@attributes":{"id":"p-0254","num":"0253"},"figref":"FIG. 20","b":["2000","2001","2001","2001","2001","1709","2001","2005","2005","1703","1709","2005","2003","2001","2004","2001","1709","1709","405","400","1702"]},"In some embodiments, the cis-repressing sequence itself comprises two sequences that can bind to each other (e.g., complementary to each other), and the conformation of the reporter transcript that results from the binding of the two sequences of the cis-repressing sequence prevents translation of the reporter sequence in the reporter transcript.","3) Naturally Occurring and Synthetic Systems for Repression\/Activation Mechanisms","Several naturally occurring and synthetically produced transcript-level mechanisms have been described that demonstrate the individual mechanisms (i.e., conformational change and cleavage) employed in each of the examples illustrated in .","Transcription termination has been observed in antisense RNA (asRNA)-mediated transcriptional attenuation. In one example, two loop-loop interactions between RNAIII\/repR mRNA are subsequently followed by the formation of a stable duplex. This complex stabilizes a Rho-independent terminator structure to arrest elongation by RNA polymerase (RNAP).","The RBS sequestration mechanism has been described via the development of a synthetic riboswitch system. In this system, a sequence complementary to a RBS is placed upstream of the RBS, allowing the presence of a linker sequence between the two regions. After transcription of the mRNA, the two complementary regions hybridize. creating a hairpin that prevents docking of the ribosome. To activate translation, a synthetic trans-activating RNA carrying the RBS sequence binds to the hybridized RNA, allowing the RBS to be exposed and available for translation.","The prevention of translation due to the cleaving of RNA has also been described in a natural system where the asRNA MicC targets a sequence inside the coding region of ompD mRNA. The interaction, which is promoted by Hfq, causes the cleaving of the mRNA by RNase E.","Yet another natural mechanism demonstrates a cleaving event to activate translation rather than inhibiting it. The GadY asRNA targets the intergenic region between two genes of the gadXW operon. Following the formation of a stable helix between GadY and the 3\u2032UTR of gadX, an RNase cleavage occurs in the transcript and stabilizes gadX transcript allowing for its translation.","4) Mechanism of Conformational Change by Cis-Repression of the Reporter Sequence and by Binding of a Target Transcript","The general mechanisms employed in the invention are intermolecular nucleic acid molecule interactions that may result in two subsequent mechanisms: (1) a conformational change in the secondary structure of the nucleic acid molecules, and (2) a cleaving event. Described herein are methods for designing reporter transcripts that can undergo a conformational change between a cis-repressed conformation and a de-repressed conformation, such that the conformational change is induced by binding of a target transcript to the reporter transcript.","As described above, a reporter transcript can comprise a reporter sequence and be designed such that translation of the reporter gene sequence is blocked by cis-repression of the ribosome binding site (RBS) of the reporter gene.","In some embodiments, the following tools can be used for designing the reporter transcripts of the invention.","1) RNA secondary structure is calculated using secondary structure program, such as Mfold available at a server maintained by The RNA Institute College of Arts and Sciences, University at Albany, State University of New York (Mfold web server for nucleic acid folding and hybridization prediction. Nucleic Acids Res. 31 (13), 3406-15, (2003)).","2) Intermolecular RNA interactions are calculated using a software program such as RNA-RNA InterACTion prediction using Integer Programming (RactIP) available at a server maintained by the Graduate School of Information Science, Nara Institute of Science and Technology (NAIST), Department of Biosciences and Informatics, Keio University Japan.","3) RNA secondary structure is visualized using Visualization Applet for RNA (VARNA), which is a Java lightweight Applet dedicated to drawing the secondary structure of RNA.","A secondary structure of the target transcript can be generated based on the lowest energy conformation calculated by MFold and visualized with VARNA.","ssRNA regions or target regions can be identified within the target transcript that can be ideal for binding to a reporter transcript. In some instances, the secondary structure of the target transcript includes a consensus sequence or loop sequence that can bind to a portion of the reporter sequence. For example, in the mecA transcript of methicillin-resistant , there is a terminal loop that includes a consensus YUNR sequence (\u201cUUGG\u201d) that can be used to bind to a cis-repressing sequence of a reporter transcript. Analysis of the secondary structure of the target transcript can reveal these one or more ssRNA regions that can be suitable for binding to a cis-repressing sequence. The cis-repressing sequence of the reporter transcript can then be designed to bind to these one or more ssRNA regions.","In some embodiments, the cis-repressing sequence can be designed to bind to the RBS of the reporter sequence in the reporter transcript and form a stem-loop structure within the reporter transcript, such that the cis-repressing sequence blocks binding of an RNA polymerase to the RBS of the reporter sequence. Upon binding of the cis-repressing sequence to the ssRNA region of the target transcript, the RBS of the reporter sequence can be exposed and translation of the reporter sequence can be initiated.","In some embodiments, the cis-repressing sequence of the reporter transcript can be designed to be positioned at the 5\u2032 terminus of the reporter sequence and designed to generate a stem-loop structure in the reporter sequence, such that the RBS sequence of the reporter sequence is blocked. The cis-repressing stem-loop structure can be designed to block the RBS sequence based on the lowest energy conformation of the reporter transcript, as calculated by MFold and visualized with VARNA. The predicted inter-molecular interactions between the target transcript and the cis-repressing sequence of the reporter transcript can be calculated by RactIP and visualized by VARNA. A diagram can be drawn to visualize the base pairing between the target transcript and the cis-repressing sequence of the reporter transcript, as shown in  below.","The interaction can include base pairing between 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, or 50 or more nucleotides in the target sequence and cis-repressing sequence. The complementary binding between the two sequences can be fully complementary, substantially complementary or partially complementary. The base pairing can be across contiguous nucleotide sequences or regions within the target and cis-repressing sequences, for example, as shown in .","5) Cleavage Mechanisms for Cis-Repressed Transcripts or Reporter Transcripts","The general mechanisms employed in the invention are intermolecular nucleic acid molecule interactions that may result in two subsequent mechanisms: (1) a conformational change in the secondary structure of the nucleic acid molecules, and (2) a cleaving event. Described herein are methods and systems for designing reporter transcripts that employ a cleaving event.","In some embodiments, a cleaving mechanism can be employed in the system and methods of the invention for cis-repression or for trans-activation. For example, as described above in , a system can be designed to take advantage of a cleaving mechanism by exposing a nucleic acid sequence of the reporter transcript to a cleaving enzyme (RNase) or sequestering a single-stranded sequence that is recognized by a sequence specific RNAase.","In one example, an ribonuclease E (RNAse E) site can be designed in the reporter transcript (\u201c*\u201d indicates the cleaving site): (G,A)N(C,A)N(G)(G,U,A)*(A,U)(C,U)N(C,A)(C,A). See Kaberdin et al., -. Nucleic Acids Research, 2003, Vol. 31, No. 16 (doi: 10.1093\/nar\/gkg690).","In a cis-repression system, a cis-repressing sequence can be incorporated in the design of a reporter transcript, such that when transcribed, the conformation of the reporter transcript exposes a single stranded region containing a sequence RNAse E recognition motif at the desired site to be cleaved. In some embodiments, the cleavage site can be involved in repression of the transcription of the reporter transcript, for example, if the cleavage site is within the coding region of the reporter gene.","For a trans-derepression system, the cis-repressed transcript can be engineered to bind to a target transcript, such that the interaction causes a conformational change in the reporter transcript that sequesters the single-stranded region containing the RNAse E site.","The system can be designed such that the cis-repressing mechanism is due to a specific secondary structure generated by a conformation of the cis-repressing sequence, such as the transcription termination structure described above. In this example, a cleaving event serves to de-repress the reporter sequence. This can be accomplished by designing the cis-repressing sequence to interact with (bind to) a naturally-occurring plasmid or other cellular transcript, such that the interaction results in the generation of a single-stranded region containing the RNAse E site that can be cleaved and thus removes the cis-repressing sequence from the reporter transcript.","In some embodiments, when a cleavage event is employed for expression of the reporter, the RNAse E site is designed to be outside of the coding region of a reporter sequence with enough sequence length in the 5\u2032 and 3\u2032 UTR in order to allow for a viable reporter transcript. In this case, the RNAse E site is designed to be at least 0, 1, 2, 3, 4, 5, 10, 20, 30, 40, 50, 60, 70, 80, 90, 100, 200, 300, 400, 500, or more base pairs upstream of the start codon in prokaryotic systems and at least 18, 20, 30, 40, 50, 60, 70, 80, 90, 100, 200, 300, 400, 500, or more base pairs upstream of the start codon in eukaryotic systems or at least 10, 20, 30, 40, 50, 60, 70, 80, 90, 100, 200, 300, 400, 500, or more base pairs downstream of the stop codon. In other embodiments, when a cleavage event is employed for repression of the reporter, the RNAse E site is designed to be within the coding region of the reporter sequence or otherwise placed in order to inhibit expression of the reporter.","6) Transcripts","As described above, a transcript is a length of nucleotide sequence (DNA or RNA) transcribed from a DNA or RNA template sequence or gene. The transcript can be a cDNA sequence transcribed from an RNA template or an mRNA sequence transcribed from a DNA template. The transcript can be transcribed from an engineered nucleic acid construct. The transcript can have regions of complementarity within itself, such that the transcript includes two regions that can form an intra-molecular duplex. One region can be referred to as a \u201ccis-repressing sequence\u201d that binds to and blocks translation of a reporter sequence. A second region of the transcript is called a \u201creporter sequence\u201d that encodes a reporter molecule, such as a detectable or selectable marker.","The transcripts of the invention can be a transcript sequence that can be 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, or 25 nucleotides in length. In other embodiments, the transcript can be at least 25, 30, 40, 50, 60, 70, 80, 90, 100, 500, 1000, 1500, 2000, 3000, 4000, 5000 or more nucleotides in length. The cis-repressing sequence and the reporter sequence can be the same length or of different lengths.","In some embodiments, the cis-repressing sequence is separated from the reporter sequence by 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 35, 40, 45, 50, 55, 60, or more spacer nucleotides.","7) Vectors","In another aspect, the transcripts (including antisense and sense sequences) of the invention are expressed from transcription units inserted into DNA or RNA vectors (see, e.g., Couture, A, et al., . (1996), 12:5-10; Skillern, A., et al., International PCT Publication No. WO 00\/22113, Conrad, International PCT Publication No. WO 00\/22114, and Conrad, U.S. Pat. No. 6,054,299). These sequences can be introduced as a linear construct, a circular plasmid, or a viral vector, including bacteriophage-based vectors, which can be incorporated and inherited as a transgene integrated into the host genome. The transcript can also be constructed to permit it to be inherited as an extrachromosomal plasmid (Gassmann, et al., (1995) 92:1292).","The transcript sequences can be transcribed by a promoter located on the expression plasmid. In one embodiment, the cis-repressing and reporter sequences are expressed as an inverted repeat joined by a linker polynucleotide sequence such that the transcript has a stem and loop structure.","Recombinant expression vectors can be used to express the transcripts of the invention. Recombinant expression vectors are generally DNA plasmids or viral vectors. Viral vectors expressing the transcripts can be constructed based on, but not limited to, adeno-associated virus (for a review, see Muzyczka, et al., (1992) 158:97-129)); adenovirus (see, for example, Berkner, et al., (1998) 6:616), Rosenfeld et al. (1991, Science 252:431-434), and Rosenfeld et al. (1992), 68:143-155)); or alphavirus as well as others known in the art. Retroviruses have been used to introduce a variety of genes into many different cell types, including epithelial cells, in vitro and\/or in vivo (see, e.g., Eglitis, et al., (1985) 230:1395-1398; Danos and Mulligan, Proc. Natl. Acad. Sci. USA (1998) 85:6460-6464; Wilson et al., 1988, Proc. Natl. Acad. Sci. USA 85:3014-3018; Armentano et al., 1990, Proc. Natl. Acad. Sci. USA 87:61416145; Huber et al., 1991, Proc. Natl. Acad. Sci. USA 88:8039-8043; Ferry et al., 1991, Proc. Natl. Acad. Sci. USA 88:8377-8381; Chowdhury et al., 1991, Science 254:1802-1805; van Beusechem. et al., 1992, Proc. Natl. Acad. Sci. USA 89:7640-19; Kay et al., 1992, Human Gene Therapy 3:641-647; Dai et al., 1992, Proc. Natl. Acad. Sci. USA 89:10892-10895; Hwu et al., 1993, J. Immunol. 150:4104-4115; U.S. Pat. No. 4,868,116; U.S. Pat. No. 4,980,286; PCT Application WO 89\/07136; PCT Application WO 89\/02468; PCT Application WO 89\/05345; and PCT Application WO 92\/07573). Recombinant retroviral vectors capable of transducing and expressing genes inserted into the genome of a cell can be produced by transfecting the recombinant retroviral genome into suitable packaging cell lines such as PA317 and Psi-CRIP (Comette et al., 1991, Human Gene Therapy 2:5-10; Cone et al., 1984, Proc. Natl. Acad. Sci. USA 81:6349). Recombinant adenoviral vectors can be used to infect a wide variety of cells and tissues in susceptible hosts (e.g., rat, hamster, dog, and chimpanzee) (Hsu et al., 1992, J. Infectious Disease, 166:769), and also have the advantage of not requiring mitotically active cells for infection.","Any viral vector capable of accepting the coding sequences for the transcript(s) to be expressed can be used, for example, vectors derived from adenovirus (AV); adeno-associated virus (AAV); retroviruses (e.g., lentiviruses (LV), Rhabdoviruses, murine leukemia virus); herpes virus, and the like. The tropism of viral vectors can be modified by pseudotyping the vectors with envelope proteins or other surface antigens from other viruses, or by substituting different viral capsid proteins, as appropriate.","For example, lentiviral vectors featured in the invention can be pseudotyped with surface proteins from vesicular stomatitis virus (VSV), rabies, Ebola, Mokola, and the like. AAV vectors featured in the invention can be made to target different cells by engineering the vectors to express different capsid protein serotypes. Techniques for constructing AAV vectors which express different capsid protein serotypes are within the skill in the art; see, e.g., Rabinowitz J E et al. (2002), J Virol 76:791-801, the entire disclosure of which is herein incorporated by reference.","Selection of recombinant viral vectors suitable for use in the invention, methods for inserting nucleic acid sequences for expressing the transcripts into the vector, and methods of delivering the viral vector to the cells of interest are within the skill in the art. See, for example, Dornburg R (1995), Gene Therap. 2: 301-310; Eglitis M A (1988), Biotechniques 6: 608-614; Miller A D (1990), Hum Gene Therap. 1: 5-14; Anderson W F (1998), Nature 392: 25-30; and Rubinson D A et al., Nat. Genet. 33: 401-406, the entire disclosures of which are herein incorporated by reference.","Viral vectors can be derived from AV and AAV. A suitable AV vector for expressing the transcripts featured in the invention, a method for constructing the recombinant AV vector, and a method for delivering the vector into target cells, are described in Xia H et al. (2002), 20: 1006-1010. Suitable AAV vectors for expressing the transcripts featured in the invention, methods for constructing the recombinant AV vector, and methods for delivering the vectors into target cells are described in Samulski R et al. (1987), J. Virol. 61: 3096-3101; Fisher K J et al. (1996), J. Virol, 70: 520-532; Samulski R et al. (1989), J. Virol. 63: 3822-3826; U.S. Pat. No. 5,252,479; U.S. Pat. No. 5,139,941; International Patent Application No. WO 94\/13788; and International Patent Application No. WO 93\/24641, the entire disclosures of which are herein incorporated by reference.","The promoter driving transcript expression in either a DNA plasmid or viral vector featured in the invention may be a eukaryotic RNA polymerase I (e.g., ribosomal RNA promoter), RNA polymerase II (e.g., CMV early promoter or actin promoter or U1 snRNA promoter) or generally RNA polymerase III promoter (e.g., U6 snRNA or 7SK RNA promoter) or a prokaryotic promoter, for example the T7 promoter, provided the expression plasmid also encodes T7 RNA polymerase required for transcription from a T7 promoter. The promoter can also direct transgene expression to the pancreas (see, e.g., the insulin regulatory sequence for pancreas (Bucchini et al., 1986, Proc. Natl. Acad. Sci. USA 83:2511-2515)).","In addition, expression of the transcript can be precisely regulated, for example, by using an inducible regulatory sequence and expression systems such as a regulatory sequence that is sensitive to certain physiological regulators, e.g., circulating glucose levels, or hormones (Docherty et al., 1994, FASEB J. 8:20-24). Such inducible expression systems, suitable for the control of transgene expression in cells or in mammals include regulation by ecdysone, by estrogen, progesterone, tetracycline, chemical inducers of dimerization, and isopropyl-beta-D-1-thiogalactopyranoside (IPTG). A person skilled in the art would be able to choose the appropriate regulatory\/promoter sequence based on the intended use of the dsRNA transgene.","Generally, recombinant vectors capable of expressing transcript molecules are delivered as described below, and persist in target cells. Alternatively, viral vectors can be used that provide for transient expression of transcript molecules. Such vectors can be repeatedly administered as necessary. Once expressed, the transcript binds to target RNA and modulates its function or expression. Delivery of transcript expressing vectors can be systemic, such as by intravenous or intramuscular administration, by administration to target cells ex-planted from the patient followed by reintroduction into the patient, or by any other means that allows for introduction into a desired target cell.","Transcript expression DNA plasmids are typically transfected into target cells as a complex with cationic lipid carriers (e.g., Oligofectamine) or non-cationic lipid-based carriers (e.g., Transit-TKO\u2122). Multiple lipid transfections for dsRNA-mediated knockdowns targeting different regions of a single PROC gene or multiple PROC genes over a period of a week or more are also contemplated by the invention. Successful introduction of vectors into host cells can be monitored using various known methods. For example, transient transfection can be signaled with a reporter, such as a fluorescent marker, such as Green Fluorescent Protein (GFP). Stable transfection of cells ex vivo can be ensured using markers that provide the transfected cell with resistance to specific environmental factors (e.g., antibiotics and drugs), such as hygromycin B resistance.","The delivery of the vector containing the recombinant DNA can by performed by abiologic or biologic systems. Including but not limited to liposomes, virus-like particles, transduction particles derived from phage or viruses, and conjugation.","8) Reporters for Transcript Assay","In some embodiments, the nucleic acid construct comprises a reporter sequence (e.g., a reporter gene sequence). The reporter gene encodes a reporter molecule that produces a signal when expressed in a cell. In some embodiments, the reporter molecule can be a detectable or selectable marker. In certain embodiments, the reporter molecule can be a fluorescent reporter molecule, such as a green fluorescent protein (GFP), yellow fluorescent protein (YFP), cyan fluorescent protein (CFP), blue fluorescent protein (BFP), or red fluorescent protein (RFP). In other embodiments, the reporter molecule can be a chemiluminescent protein.","Reporter molecules can be a bacterial luciferase, an eukaryotic luciferase, a fluorescent protein, an enzyme suitable for colorimetric detection, a protein suitable for immunodetection, a peptide suitable for immunodetection or a nucleic acid that function as an aptamer or that exhibits enzymatic activity.","Selectable markers can also be used as a reporter. The selectable marker can be an antibiotic resistance gene, for example.","9) Cells and Target Genes for Transcript Reporter Assay","Examples of cells that can be used for detection include Gram-positive and Gram-negative bacteria, such as , etc., fungi such as , and other eukaryotic cells, including cells from humans, other mammals, insects, invertebrates, or plants.","Target transcripts can include any endogenous transcript, whether coding or non-coding. Target transcripts can be derived from eukaryotic and prokaryotic cells, including, for example, mecA transcript in cells (indicative of MRSA), the tcdB transcript in (indicative of toxigenic C. diff), and HPV E6\/E7 transcripts in cervical epithelial cells (indicative of cervical cancer). Genes associated with infectious agents, such as viruses, can be targets as well, including HIV, HPV, etc. Other examples of target genes include non-coding RNA such as transfer RNA (tRNA) and ribosomal RNA (rRNA), as well as RNAs such as snoRNAs, microRNAs, siRNAs, snRNAs, exRNAs, and piRNAs and ncRNAs.","Below are examples of specific embodiments for carrying out the present invention. The examples are offered for illustrative purposes only, and are not intended to limit the scope of the present invention in any way. Efforts have been made to ensure accuracy with respect to numbers used (e.g., amounts, temperatures, etc.), but some experimental error and deviation should, of course, be allowed for.","The practice of the present invention will employ, unless otherwise indicated, conventional methods of protein chemistry, biochemistry, recombinant DNA techniques and pharmacology, within the skill of the art. Such techniques are explained fully in the literature. See, e.g., T. E. Creighton, (W.H. Freeman and Company, 1993); A. L. Lehninger, (Worth Publishers, Inc., current addition); Sambrook, et al., (2nd Edition, 1989); (S. Colowick and N. Kaplan eds., Academic Press, Inc.); 18th Edition (Easton, Pa.: Mack Publishing Company, 1990); Carey and Sundberg 3Ed. (Plenum Press) Vols A and B(1992).","The following is an example of the design and construction of a silent mutation\/complementation-based packaging system for producing non-replicative transduction particles.","The materials used for developing the packaging system are listed below:","Bacterial Strains:","N1706, an K-12 P1 c1-100 Tn9 lysogen","Vectors:","Y14439 (pBHR1 backbone)","The following GenBank accession numbers (N.B., the sequences referred to by accession number are those listed in the database as of the priority date of this application) or SEQ ID NOs. can be used for the vector backbone and cassette sequences:\n\n","Construction of N1706(Pac): pacA Mutated Strain:","An exemplary sequence of a pacA mutated sequence is shown in SEQ ID NO: 2, shown in the informal sequence listing below. The mutation can be accomplished by constructing the mutated sequence via gene synthesis and then replacing the native sequence in N1706 with the mutated sequence via an allelic exchange approach.","Construction of the GWP10001 Reporter Vector:","The GWP10001 vector contains the pBHR1 origin of replication exhibiting broad Gram-negative activity, two selectable markers for kanamycin and chloramphenicol, the native bacteriophage P1 pac-site sequence, the luxA and luxB genes are from operatively linked to the constitutive blasticillin promoter (Pblast), and the P1 lytic replicon containing the C1 repressor-controlled P53 promoter, the promoter P53 antisense, the repL genes, and an in-frame deletion of the kilA gene.",{"@attributes":{"id":"p-0319","num":"0322"},"figref":"FIG. 2"},"Silent\/Complementation Packaging System:","The packaging system includes the pacA mutant strain N1706(pac) complemented with the vector pGWP10001. As known to one of skill in the art, the manner of constructing this system can be accomplished by transformation N1706(pac) with vector pGWP10001. The vector pGWP10001 can be maintained in cultures of the transformed N1706(pac) by growing the transformant in the presence of 50 ug\/mL of kanamycin.","Production of Transduction Particles Carrying Plasmid DNA:","Non-replicative transduction particles carrying vector pGWP10001 can be produced from N1706(pac) transformants via thermal induction at 42\u00b0 C. Incubation at 42\u00b0 C. results in induction of the P1 lytic cycle in which the prophage excises from the N1706 genome, produces phage structural elements, and packages pGWP10001 concatameric DNA formed by the lytic replicon in progeny phage particles, as depicted in . The resulting cell lysate is then collected and contains non-replicative transduction particles, each consisting of bacteriophage P1 particles carrying a linear concatamer of pGWP10001 DNA.","The following is an example of the design and construction of a deletion\/complementation-based packaging system for producing non-replicative transduction particles.","The materials used for developing the packaging system are listed below:","Bacterial Strains:","RN4220 is a restriction defective strain that is a non-lysogenic derivative of NCTC 8325 and is an efficient recipient for DNA. It was first described in Kreiswirth, B. N. et al., . Nature, 1983. 305(5936): p. 709-712.","RN10616 is derived by lysogenizing RN4220 with bacteriophage \u03c680\u03b1. Ubeda, C. et al., . Molecular Microbiology, 2009. 72(1): p. 98-108.","ST24 is derived from deleting the small terminase gene terS from the lysogenized bacteriophage \u03c680\u03b1 in RN10616. Ubeda, C. et al., . Molecular Microbiology, 2009. 72(1): p. 98-108.","Vectors:","Examples of plasmids that can be used as source plasmids for cassettes, in some embodiments of the invention are described in Charpentier, E., et al., --. Appl. Environ. Microbiol., 2004. 70(10): p. 6076-6085.","The following GenBank accession numbers can be used for cassette sequences:\n\n","terS Deletion:","The construction of the terS knockout strain ST24 can be accomplished via an allelic-exchange-based strategy resulting in an in-frame deletion removing most of the coding sequence of the \u03c680\u03b1 small terminase gene. The details of this strategy are described in Ubeda, C. et al., . Molecular Microbiology, 2009. 72(1): p. 98-108.","An exemplary sequence of a terS knockout strain is shown in SEQ ID NO:13, (shown in the sequence listing below). SEQ ID NO:13 is a RN10616 genomic sequence loci showing the \u03c680\u03b1 terS deletion and complementation.","Vector Construction:","The GW80A0001 vector is an shuttle vector. The vector contains (pT181cop-623 repC) and (ColE1ori) origins of replication, the selectable markers for ampicillin (amp) and tetracycline (tet(M)) resistance for selection in and , respectively, the \u03c611 small terminase (terS) gene sequence that includes its own promoter, the luxA and luxB genes are from operatively linked to the constitutive Ppromoter, and a transcription termination sequence (TT).",{"@attributes":{"id":"p-0338","num":"0348"},"figref":"FIG. 4","sub":"clpB ","i":"S. aureus "},"Deletion\/Complementation Packaging System:","The packaging system can include the terS knockout strain ST24 complemented with the vector pGW80A0001 to generate strain GW24. As known to one of skill in the art, the manner of constructing this system can be accomplished by transformation ST24 with vector pGW80A0001. The vector pGW80A0001 can be maintained in cultures of the transformed ST24 by growing the transformant in the presence of 5 ug\/mL of tetracycline.","Production of Transduction Particles Carrying Plasmid DNA:","Non-replicative transduction particles carrying vector pGW80A0001 can be produced from GW24 via a Mitomycin C-induction method that was first demonstrated in and is now a standard technique for obtaining prophages from lysogenized bacteria. Otsuji, N. et al., -12 . Nature, 1959. 184(4692): p. 1079-1080. This prophage induction method results in induction of the \u03c680\u03b1 lytic cycle in which the prophage excises from the GW24 genome, produces phage structural elements, and packages pGW80A0001 concatameric DNA in progeny phage particles, as depicted in . The resulting cell lysate is then collected and contains non-replicative transduction particles, each consisting of bacteriophage \u03c680\u03b1 particles carrying a linear concatamer of pGW80A0001 DNA.","The following is an example of the design and construction of a SaPIbov2-based packaging system for producing non-replicative transduction particles.","The materials used for developing the packaging system are listed below:","The following materials can be used to develop a SaPIbov2-based packaging system lacking integrase.","Bacterial Strains:","RN451 is a strain lysogenized with bacteriophage \u03c611.","JP2131 is RN451 that has been lysogenized with SaPIbov2. See Maiques, E. et al., Role of Staphylococcal Phage and SaPI Integrase in Intra- and Interspecies SaPI Transfer. J. Bacteriol., 2007. 189(15): p. 5608-5616.","JP2488 is strain JP2131 in which the int gene has been deleted from SapIbov2 (SaPIbov2\u0394int) Maiques, E. et al., Role of Staphylococcal Phage and SaPI Integrase in Intra- and Interspecies SaPI Transfer. J. Bacteriol., 2007. 189(15): p. 5608-5616.","Bacteriophage:","Bacteriophage \u03c611 can be obtained from strain RN0451 via a Mitomycin C-induction method that was first described in and is now a standard technique for obtaining prophages from lysogenized bacteria. Otsuji, N. et al., -12 . Nature, 1959. 184(4692): p. 1079-1080.","Promoters:","Pcan be used as a promoter in this example. The clpB gene promoter is a constitutive promoter used for controlling the expression of the int gene. The clpB (P) gene promoter sequence was first described in 2004. Frees, D., et al., Clp ATPases are required for stress tolerance, intracellular replication and biofilm formation in . Molecular Microbiology, 2004. 54(5): p. 1445-1462. It was also first employed for controlling the gene expression in a plasmid in 2004. Arnaud, M., A. Chastanet, and M. Debarbouille, ---. Appl. Environ. Microbiol., 2004. 70(11): p. 6887-6891. The promoter can be obtained from RN4220 using primers described in 2004. Id.","Production of \u03c611\/SaPIbov2\u0394int Co-Lysogen (RN451(\u03c611 SaPIbov2\u0394int)):","The strain JP2488(\u03c611 SaPIbov2\u0394int) can be produced by lysogenizing JP2488 with \u03c611.","Deletion of \u03c611 terS (RN451(\u03c611\u0394terS SaPIbov2\u0394int)):","The strain RN451(\u03c611\u0394terS SaPIbov2\u0394int) can be produced by deleting the \u03c611 terS gene from RN451(\u03c611SaPIbov2\u0394int), as described in Tormo, M. A. et al., . J. Bacteriol., 2008. 190(7): p. 2434-2440.","Incorporation of P-Int into Genome (RN451(\u03c611\u0394terS SaPIbov2\u0394int P-int)):","RN451(\u03c611\u0394terS SaPIbov2\u0394int P-int) can be produced by first fusing Pand int via standard molecular biology techniques then inserting the P-int fusion into the genome of RN451(\u03c611\u0394terS SaPIbov2\u0394int) and then selecting clones that have P-int inserted outside of the \u03c611 and SaPIbov2 regions.","Production of \u03c611 Particles Carrying Only SaPIbov2\u0394int P-Int Concatamers:","\u03c611 particles carrying only SaPIbov2\u0394int P-int concatamers can be produced via mitomycin-C induction of RN451(\u03c611\u0394terS SaPIbov2\u0394int P-int), as described by Otsuji, N. et al., -12 . Nature, 1959. 184(4692): p. 1079-1080. The cell lysate contains non-replicative transduction particles, each consisting of bacteriophage \u03c611 structural proteins carrying a linear concatamer of GI-derived DNA.","One of skill in the art will understand how to construct the NRTPs of the invention using the above-referenced materials and well-known molecular biology and genetic techniques in the art.","The following is an example of an inducer reporter-based SarS reporter system that employs a terS deletion\/complementation-based non-replicative transduction particle.","Reporter Gene:","Bacterial luciferase (luxAB). The luxA and luxB genes are from . They lack a transcriptional promoter and each contains their own ribosomal binding site.","Spa gene promoter (P): The spa gene promoter will be used for controlling the expression of the luxAB genes.","Construction of P-luxAB Fusion:","The luxAB genes can be fused to the Ppromoter sequence such that the luxAB genes are operatively linked to the Ppromoter.","Construction of the luxAB-Expressing Reporter Vector","The luxAB-expressing reporter vector can be constructed via standard molecular biological techniques by incorporating the P-luxAB fusion product into the MCS of the shuttle vector depicted below.",{"@attributes":{"id":"p-0371","num":"0381"},"chemistry":{"@attributes":{"id":"CHEM-US-00001","num":"00001"},"img":{"@attributes":{"id":"EMI-C00001","he":"4.91mm","wi":"67.82mm","file":"US09388453-20160712-C00001.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},{"@attributes":{"id":"p-0372","num":"0382"},"i":["E. coli\/S. aureus ","S. aureus ","E. coli "],"sub":"clpB"},"GenBank accession numbers for cassette sequences:","J01764 (pT181 replicons)","M21136 (tetA(M))","Accession number not yet available (P)","AF424781 REGION: 16526 . . . 16966 (terS)","L09137 (amp ColE1 ori)","M62650 (TT)","Propagation of the vector for conducting in vitro manipulations and for verification of manipulations can be accomplished via the Top 10 and the final modified vector can then be introduced into RN0451\u0394terS. Transduction particles carrying shuttle vector can be produced from the RN0451\u0394terS transformants via a Mitomycin C-induction method that was first described in 1959 and is now a standard technique for obtaining prophages from lysogenized bacteria. Otsuji, N., et al., Induction of Phage Formation in the Lysogenic K-12 by Mitomycin C. Nature, 1959. 184(4692): p. 1079-1080. The cell lysate is then collected and contains non-replicative transduction particles each consisting of bacteriophage \u03c611 structural proteins carrying a linear concatamer of plasmid DNA capable of reporting on the presence of SarS in target cells.","The following is an example of an intracellular enzyme reporter-based \u03b2-lactamase reporter system that employs a terS deletion\/complementation-based non-replicative transduction particle.","Reporter Gene:",{"@attributes":{"id":"p-0383","num":"0393"},"i":"Renilla "},"Promoter:","The promoter can be P. The constitutive beta-lactamase promoter can be used for driving the expression of the mc gene.","Caged Substrate:","Caged coelenterazine-phosphate as described in Daniel Sobek, J. R., 2007, Zymera, Inc.","Construction of P-Ruc Fusion:","The ruc genes can be fused to the Ppromoter sequence such that the ruc genes are operatively linked to the Ppromoter.","Construction of the Ruc-Expressing Reporter Vector:","The ruc-expressing reporter vector can be constructed via standard molecular biological techniques by incorporating the P-mc fusion product into the MCS of the shuttle vector depicted in Section V, A, 3), i) above.","Propagation of the vector for conducting in vitro manipulations and for verification of manipulations can be accomplished via the Top 10 and the final modified vector can then be introduced into RN0451\u0394terS. Transduction particles carrying shuttle vector can be produced from the RN0451\u0394terS transformants via a Mitomycin C-induction method that was first described in 1959 and is now a standard technique for obtaining prophages from lysogenized bacteria. Otsuji, N., et al., Induction of Phage Formation in the Lysogenic K-12 by Mitomycin C. Nature, 1959. 184(4692): p. 1079-1080. The cell lysate is then collected and contains NRTPs each consisting of bacteriophage \u03c611 structural proteins carrying a linear concatamer of plasmid DNA capable of expressing luciferase within viable cells within the \u03c611 host range.","The following is an example of an intracellular molecule reporter-based reporter system that employs a terS deletion\/complementation-based non-replicative transduction particle.","Promoter:","The promoter can be P. The constitutive beta-lactamase promoter can be used for driving the expression of the mc gene.","Switchable Aptamer:","Switchable aptamers can be designed and constructed as described in Samie Jaffrey, J. P., 2010, Cornell University.","Fluorophore Substrate:","Corresponding fluorophore substrates in conjunction with the above switchable aptamers can be designed and constructed as described in Samie Jaffrey, J. P., 2010, Cornell University.","Construction of P-SA Fusion:","The SA gene can be fused to the Ppromoter sequence such that the SA gene is operatively linked to the Ppromoter.","Construction of the SA-Expressing Reporter Vector:","The SA-expressing reporter vector can be constructed via standard molecular biological techniques by incorporating the P-SA fusion product into the MCS of the shuttle vector depicted in Example 4 above. Propagation of the vector for conducting in vitro manipulations and for verification of manipulations can be accomplished via the Top 10 and the final modified vector can then be introduced into RN0451\u0394terS. Transduction particles carrying shuttle vector can be produced from the RN0451\u0394terS transformants via a Mitomycin C-induction method that was first described in 1959 and is now a standard technique for obtaining prophages from lysogenized bacteria. Otsuji, N. et al., -12 . Nature, 1959. 184(4692): p. 1079-1080. The cell lysate is then collected and contains non-replicative transduction particles each consisting of bacteriophage \u03c611 structural proteins carrying a linear concatamer of plasmid DNA capable of expressing the SA within viable cells within the \u03c611 host range.","The non-replicative transduction particles described above can be used in a reporter system for detecting the presence of viable bacteria via the expression of a reporter molecule (e.g. luxAB). When this transduction particle introduces a reporter vector (e.g. pGW80A0001) into a cell within the host range of the transduction particle, cells in which the promoter (e.g. P) is recognized by the cells transcription machinery are able to drive the expression of the reporter molecule within that cell.","To test the functionality of non-replicative transduction particles as reporters for detecting the presence of cells, various MSSA\/MRSA reporter assays were developed. In an embodiment, a non-replicative transduction particle was developed from a -specific bacteriophage, and the bacterial luciferase genes luxAB under the control of a constitutive promoter were incorporated. When the non-replicative transduction particle delivered the reporter nucleic acid into , the constitutive promoter expressed luxAB suitable for reporting on the presence of a viable ","In addition, the antibiotic cefoxitin was added prior to, simultaneously with, or after the addition of the transduction particles to a sample containing cells. If the cells were not phenotypically resistant to cefoxitin (i.e., were not MRSA), luminescence was decreased or eliminated, indicating that the cells were MSSA. If, however, the cells were phenotypically resistant to cefoxitin (i.e., were MRSA), increased or detectable luminescence was observed, indicating that the cells were MRSA.","Non-Replicative Transduction Particle-Based Viable Cell Reporter Assay Function","The function of the non-replicative transduction particle as a reporter was assayed. The transduction host range of the bacteriophage \u03c680\u03b1-based non-replicative transduction particle was examined in 101 clinical MRSA isolates. The transduction assay was conducted by exposing cultures of each bacterial isolate grown in modified TSB to GW24 cell lysate containing the non-replicative transduction particles and culturing the mixture on solid media containing tetracycline.","In this example, the non-replicative transduction particle carried a tetracycline selectable marker. Cells transduced with the non-replicative transduction particles were expected to be resistant to tetracycline. In addition, transduction was examined via luminescence assay by exposing each bacterial isolate in liquid culture to cell lysate containing the non-replicative transduction particles and evaluating the mixture for bacterial luciferase luminescence activity after an incubation period.","The transduction assay showed that the \u03c680\u03b1-based non-replicative transduction particle was able to transduce all of the 101 clinical isolates of MRSA and none of the non-Staphylococci.",{"@attributes":{"id":"p-0411","num":"0421"},"figref":"FIG. 21"},{"@attributes":{"id":"p-0412","num":"0422"},"figref":"FIG. 22","i":["S. aureus ","S. aureus","S. aureus "]},"Optimization of Non-Replicative Transduction Particle-Based Viable Cell Reporter MRSA Assay\u2014Transduction Particle Reagent Formulation","The production and formulation of the non-replicative transduction particle reagent was optimized to a final formulation. In summary, a 15 L scale fermentation was performed using TSB media including peroxide induction of GW24. The 15 L fermenter batch was inoculated from a 200 mL overnight seed culture (an inoculum ratio of 1.3% (v\/v)). The culture was induced at an O.D. of 0.8 with hydrogen peroxide and cooled to 25\u00b0 C. post induction without pH or DO control. Culture supernatant was harvested by tangential flow filtration (TFF) the following morning for the purpose of clarifying the phage transduction particles from the cell debris. The material was then further concentrated and diafiltered into SM Buffer without gelatin and stored at 2-8\u00b0 C. prior to final sterile filtration and storage.","A detailed summary of the process is outlined below:",{"@attributes":{"id":"p-0416","num":"0426"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Seed flask growth"},{"entry":"(1) Inoculate 200 mL of TSB containing 5 ug\/mL tetracycline with GW24"},{"entry":"(2) Incubate at 37\u00b0 C., 200 RPM for 10-18 hours."},{"entry":"Fermentation Innoculation (15 L TSB with 5 ug\/mL tetracycline)"},{"entry":"(1) Prepare the fermenter skid with the following fermentation conditions: 37\u00b0 C.,"},{"entry":"agitation at 250 RPM, airflow at 15 LPM, and backpressure at 3 psig."},{"entry":"(2) Inoculate the fermentor using the 200 mL overnight seed culture."},{"entry":"Induce Culture"},{"entry":"(1) Once the OD600 nm reaches 0.8 (0.6-0.9), induce the culture with 0.5 mM H2O2"},{"entry":"(2) Increase temperature setpoint of fermenter to 42\u00b0 C."},{"entry":"Post-Induction Conditions and Monitoring"},{"entry":"(1) Once 30 minute induction is complete, reset temperature target for fermenter"},{"entry":"to 25\u00b0 C."},{"entry":"(2) One hour post cooling, turn off air feed to fermenter and set agitation to zero"},{"entry":"(3) Monitor the fermentation culture at hour intervals, or more frequently as"},{"entry":"necessary, until the OD600 nm has decreased to or below 0.40."},{"entry":"Harvest\/Clarification"},{"entry":"(1) After the fermentation culture OD600 has reached a minimum less than or"},{"entry":"equal to 0.40, take a 20 mL aseptic sample and add 30 \u03bcL of Benzonase to the"},{"entry":"fermentor."},{"entry":"(2) Reset agitation to 250 rpm. Allow 60 minutes with agitation for Benzonase"},{"entry":"incubation."},{"entry":"(3) Clarify the EOF sample with a 15 minute centrifugation at 3000 g."},{"entry":"(4) Pass the clarified material through a 0.45 uM membrane filter"},{"entry":"Concentration and Buffer Exchange"},{"entry":"(1) Concentrate the clarified culture by TFF using 500 kDa flat sheet membrane 10-fold."},{"entry":"(2) Diafilter the concentrated culture at a constant volume against SM Buffer"},{"entry":"without gelatin using the 500 kDa TFF membrane used for concentration"},{"entry":"Final Filtration"},{"entry":"(1) Filter the concentrated buffer exchanged material through a 0.2 \u03bcm filter."},{"entry":"(2) Store the final filtered phage material at 2-8\u00b0 C."},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}}}},"Various other reagents and formulations can be used as known to those of skill in the art to derive the formulation.","Optimization of Non-Replicative Transduction Particle-Based Viable Cell Reporter MRSA Assay\u2014Growth Media Formulation","A growth media formulation was optimized for the NRTP-based viable cell reporter MRSA assay. In order to produce luminescence in the NRTP-based MRSA Assay, the media needs to be balanced for growth and have adequate concentration of cations and additives to favor NRTP transduction. The TSBmod media used in assays prior to this development study was known to have precipitation issues that would affect the stability of the media. Growth media formulation required stability in the final formulation with a goal of 1 year at room temperature.","Methods\/Procedures: Cells Preparation for MRSA Assay",{"@attributes":{"id":"p-0421","num":"0431"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"(1) Ten unique strains of MRSA for the Subset Assay and one unique"},{"entry":"strains of MSSA were tested in the MRSA Assay."},{"entry":"(2) Overnight cultures were started in a deep 96 well plate at a 1:50"},{"entry":"dilution in TSB from a frozen one-time use stock and incubated at 37\u00b0 C."},{"entry":"on an orbital shaker for >15 hours. MRSA\/MSSA (8 \u03bcl) in TSB (392 \u03bcl)"},{"entry":"(3) The next day, a day culture at a 1:50 dilution from the overnight"},{"entry":"culture was started in TSB in 96 well deep well plate (392 \u03bcl TSB + 8 ul"},{"entry":"cells) and incubated at 37\u00b0 C. on an orbital shaker for 4 hours."},{"entry":"(4) Cells were spun in a centrifuge for 5 minutes at 1800 g force and"},{"entry":"10\u00b0 C., spent media was aspirated without disturbing the pellet."},{"entry":"(5) Spun cells were washed in 50 mM Tris-HCl pH 7.2, centrifuged,"},{"entry":"buffer aspirated without disturbing pellet and re-suspended in"},{"entry":"400 \u03bcl RPMI. RPMI is used in order to reduce variability in the metabolic"},{"entry":"state of cells and to mimic low metabolism as found in clinical samples."},{"entry":"(6) Plate was covered with airpore seal and incubated on bench for 48 hrs."},{"entry":"(7) OD was read by transferring 200 \u03bcl of RPMI culture in a shallow well"},{"entry":"OD plate and blank well with RPMI media alone was used to subtract"},{"entry":"blank OD."},{"entry":"(8) Cells were normalized to OD 0.1 in 100 \u03bcl"},{"entry":"(9) Another dilution was made 1:10 in RPMI to yield an OD of 0.005"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}}}},"Assay base media was prepared to be tested as shown in Table 2 and a representative set of media modifications in preparation for MRSA assay are shown in Table 3.",{"@attributes":{"id":"p-0423","num":"0433"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Base Media for Growth Media Formulation Development"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Components","TSB","B2","BSS-2","Notes"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Enzymatic Digest of Soybean Meal","3","0","3","Adjust pH to"]},{"entry":["(g)",{},{},{},"7.2 with 10N"]},{"entry":["Enzymatic Digest of Casein (g)","17","10","10","NaOH."]},{"entry":["Yeast Extract (g)","N\/A","25","25","Autoclave or"]},{"entry":["Sodium Chloride (g)","5","25","25","filter sterilize"]},{"entry":["Dipotassium Phosphate (g)","2.5","1","0"]},{"entry":["alpha-D Glucose (g)","2.5","5","5"]},{"entry":["Volume (litre)","1","1","1"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0424","num":"0434"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"266pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Base Media Modifications for Growth Media Formulation"},{"entry":"Development"},{"entry":"Concentration of salt\/"},{"entry":"additives for modification"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"8"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Mod","CaCl2","MgCl2","BGP","Tris-HCl","EDTA","HEPES"]},{"entry":["Base media (30 ml)","number","(mM)","(mM)","(mM)","pH7.0 (mM)","(mM)","(mM)"]},{"entry":{"@attributes":{"namest":"1","nameend":"8","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"8"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"6","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"8","colwidth":"28pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["B2","M53","5.0","2.0","0.0","50.0","10.0","0.0"]},{"entry":["BSS-2","M50","10.0","2.0","60.0","50.0","10.0","0.0"]},{"entry":["BSS-2","M54","6.7","3.3","60.0","50.0","0.0","0.0"]},{"entry":["BSS-2","M55","5.0","5.0","60.0","50.0","0.0","0.0"]},{"entry":["BSS-2","M56","6.7","3.3","60.0","0.0","0.0","10.0"]},{"entry":["BSS-2","M57","5.0","5.0","60.0","0.0","0.0","10.0"]},{"entry":["TSB","M1","5.0","10.0","60.0","0.0","0.0","0.0"]},{"entry":[{},"(original)",{},{},{},{},{},{}]},{"entry":["TSB","M58","5.0","10.0","60.0","0.0","11.1","0.0"]},{"entry":{"@attributes":{"namest":"1","nameend":"8","align":"center","rowsep":"1"}}}]}}]}}},"To each media preparation, NRTP and Cefoxitin was added according to Table 4 below to make the NRTP media reagent:",{"@attributes":{"id":"p-0426","num":"0436"},"tables":{"@attributes":{"id":"TABLE-US-00006","num":"00006"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 4"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"MRSA Assay Growth Media\/Transduction particle reagent"},{"entry":"combination"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Final"]},{"entry":[{},"30 ml media","Concentration"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"Cefoxitin","5 ug\/ml"]},{"entry":[{},"GW24 Lysate","30X"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The MRSA Assay was run with the following steps:",{"@attributes":{"id":"p-0428","num":"0438"},"tables":{"@attributes":{"id":"TABLE-US-00007","num":"00007"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"203pt","align":"left"}}],"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["(1)","Assay Plate Setup: Add 198 \u03bcl of Phage Media Reagent and 2.0 \u03bcl of"]},{"entry":[{},"each dilution of bacteria 0.05 OD and 0.005 OD in RPMI (roughly"]},{"entry":[{},"equivalent to 20,000 and 2,000 CFU\/mL, respectively) or 2.0 uL of"]},{"entry":[{},"RPMI as a blank."]},{"entry":["(2)","Incubate Assay Plate: Incubate Assay plate on orbital shaker at ~100"]},{"entry":[{},"rpm for 4 hours at 37\u00b0 C."]},{"entry":["(3)","Prepare Luminometer (Molecular Devices SpectraMax L): Wash"]},{"entry":[{},"reagent line with 70% ethanol followed by DI water then prime with"]},{"entry":[{},"the substrate reagent. Set up software as Fast Kinetic with injection"]},{"entry":[{},"of 50 \u03bcL of substrate reagent at 250 \u03bcl\/sec after 10 baseline points"]},{"entry":[{},"and read at 40 points every 0.25 seconds."]},{"entry":["(4)","Run Assay: Test each bacterial dilution plate, after letting plate"]},{"entry":[{},"equilibrate to room temperature for 5 minutes."]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}}}},"Analysis",{"@attributes":{"id":"p-0430","num":"0440"},"tables":{"@attributes":{"id":"TABLE-US-00008","num":"00008"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"203pt","align":"left"}}],"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["(1)","Determine cutoff by averaging blank RLU across all replicates and"]},{"entry":[{},"time points and adding three standard deviations."]},{"entry":["(2)","Determine maximum RLU for each sample using SoftMaxPro."]},{"entry":["(3)","Determine if the maximum RLU was greater than the cutoff RLU,"]},{"entry":[{},"and if so, then the sample data was used for comparisons of"]},{"entry":[{},"media performance."]},{"entry":["(4)","Normalize all max RLU values to the Max RLU in TSB M1 (media"]},{"entry":[{},"in use until development started) for the strain being analyzed at"]},{"entry":[{},"the specific dilution."]},{"entry":["(5)","Average the normalized RLU values across all MRSA strains for a"]},{"entry":[{},"particular media and its modification"]},{"entry":["(6)","Average the averages for the two dilution plates, ultimately leading"]},{"entry":[{},"to a single numerical value representing the fold increase in"]},{"entry":[{},"performance based on RLU of a particular media across 10 different"]},{"entry":[{},"MRSA strains in 2 cell dilutions tested."]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}}}},"Results of NRTP-Based Viable Cell Reporter MRSA Assay","Determination of Cutoff RLU: The average and standard deviation of the RLU was calculated across all time points (25) for each blank replicate (4). The cutoff was calculated for each plate as the average blank RLU plus three standard deviations.","Determination of Relative Improvements: The maximum RLU was exported for each sample (blanks, MSSA and MRSA at all dilutions) from SoftMaxPro and compared to the cutoff RLU. If the sample had 2 data points greater than the cutoff for phage concentration, then the max RLU value was utilized for analysis.","The values were normalized by dividing a particular max RLU by the max RLU of its control condition (that strain in TSB M1-original media, at the dilution being analyzed). The ratios obtained were averaged across 10 MRSA for each media condition and each dilution, as shown in Table 5. The average across the two dilutions is also shown in the table.",{"@attributes":{"id":"p-0435","num":"0445"},"tables":{"@attributes":{"id":"TABLE-US-00009","num":"00009"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 5"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"MRSA Assay Results from Various Growth Media Formulations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"70pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},{},"Average for"]},{"entry":[{},"Media","Plate 1","Plate 2","both dilutions"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"70pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"B2","M53","1.89","1.88","1.89"]},{"entry":[{},"BSS-2","M50","1.37","1.47","1.42"]},{"entry":[{},"BSS-2","M54","1.50","1.76","1.63"]},{"entry":[{},"BSS-2","M55","1.82","2.90","2.36"]},{"entry":[{},"BSS-2","M56","2.38","6.00","4.19"]},{"entry":[{},"BSS-2","M57","2.00","3.92","2.96"]},{"entry":[{},"TSB","M1","1.00","1.00","1.00"]},{"entry":[{},"TSB","M58","1.18","0.96","1.07"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"5","align":"center","rowsep":"1"}}]}]}}]}}},"Conclusions","BSS2-M56 exhibited the best performance on average across the various media tested. HEPES buffer based media performed better than Tris-HCl buffered media. HEPES is known to be a biologically favorable buffering system as opposed to Tris-HCl. B2 based base\/broth had better performance than TSB based broth.","Various other reagents and formulations can be used as known to those of skill in the art to derive the formulation. Other suitable formulations were developed via similar experiments as described above. Examples of other suitable formulations are included below in Tables 6, 7, and 8.",{"@attributes":{"id":"p-0439","num":"0449"},"tables":{"@attributes":{"id":"TABLE-US-00010","num":"00010"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 6"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"BSC Media Formulation"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"BSC",{}]},{"entry":[{},"Components","Amount"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"Enzymatic","14.5 g"]},{"entry":[{},"Digest of Casein"]},{"entry":[{},"Yeast Extract","35.5 g"]},{"entry":[{},"Sodium","35.5 g"]},{"entry":[{},"Chloride"]},{"entry":[{},"alpha-D Glucose","\u2003\u20097 g"]},{"entry":[{},"Total Vol","\u20031 L"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0440","num":"0450"},"tables":{"@attributes":{"id":"TABLE-US-00011","num":"00011"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 7"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"BSC Media Modification"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"126pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"BSC-M64",{}]},{"entry":[{},"Chemical Name","Final (Assay) Conc"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"BGP (mM)","60.0"]},{"entry":[{},"HEPES (mM)","10.0"]},{"entry":[{},"LiCl(mM)","84.0"]},{"entry":[{},"BSC","To 1 L"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0441","num":"0451"},"tables":{"@attributes":{"id":"TABLE-US-00012","num":"00012"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 8"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Transduction Particle Media Modification"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Transduction Particle Formulation",{}]},{"entry":[{},"(PM4)"]},{"entry":[{},"Chemicals","Final (Assay) conc"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"CaCl2 (M)","0.00667"]},{"entry":[{},"MgCl2 (M)","0.00335"]},{"entry":[{},"HEPES (M)","0.01000"]},{"entry":[{},"GW24 lysate stock","0.01250"]},{"entry":[{},"Sodium Azide (%)","0.0006"]},{"entry":[{},"Water","To 1 mL"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"Optimization of Non-Replicative Transduction Particle-Based Viable Cell Reporter MRSA Assay\u2014Substrate Reagent Formulation","In order to produce luminescence in the MRSA Assay, the Substrate Reagent must include an aldehyde as a substrate for luciferase. An initially developed aliphatic aldehyde formulation (4.2 mM Tridecanal in TSB) was not stable and formed a heterogeneous emulsion rather than a solution. This example outlines the development of a Substrate Reagent formulation that addresses these issues with a goal of 6 months at room temperature or 2-8\u00b0 C. stability.","This example describes the steps that were taken to develop the Substrate Reagent to a final formulation.","Methods\/Procedures","All screening and stability experiments were tested using a \u201cModel System\u201d that consists of strain RN4220 harboring a LuxAB-expressing plasmid. The typical preparation and testing method was as follows.",{"@attributes":{"id":"p-0447","num":"0457"},"tables":{"@attributes":{"id":"TABLE-US-00013","num":"00013"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"231pt","align":"left"}}],"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["(1)","Overnight Culture: 2 mL TSB + 1 uL of 10 mg\/mL Tetracycline + 1 colony"]},{"entry":[{},"of Model System Bacteria from TSA plate, shaking at 225 rpm overnight"]},{"entry":[{},"at 37\u00b0 C."]},{"entry":["(2)","Day Culture: Diluted overnight culture 1:50 or 1:100 into TSB + 5 ug\/mL"]},{"entry":[{},"Tetracycline, shaking at 225 rpm for 1.5-2 hours at 37\u00b0 C."]},{"entry":["(3)","Normalize Day Culture: Measured 1 mL of day culture on Nanodrop"]},{"entry":[{},"with cuvette at 600 nm, blanking with TSB + 5 ug\/mL Tetracycline."]},{"entry":[{},"Diluted to 0.1 OD with TSB + 5 ug\/mL Tetracycline."]},{"entry":["(4)","Dilute Culture for Testing: Diluted 0.1 OD Culture with TSB + 5 ug\/mL"]},{"entry":[{},"Tetracycline to a 1:200, 1:2000 and 1:20000 dilution which was roughly"]},{"entry":[{},"equivalent to 100000, 10000 and 1000 CFU\/mL."]},{"entry":["(5)","Plate Bacteria: Added 200 uL of each dilution and a blank (TSB + 5"]},{"entry":[{},"ug\/mL Tetracycline with no bacteria) in three replicates to a Greiner Bio-"]},{"entry":[{},"one white assay plate for each substrate to be tested."]},{"entry":["(6)","Prepare Luminometer (SpectraMax L): Wash reagent line with 70%"]},{"entry":[{},"ethanol followed by DI water then prime with the substrate. Set up"]},{"entry":[{},"software as Fast Kinetic with injection of 50 uL substrate at 250 ul\/sec"]},{"entry":[{},"after 10 baseline points and read at 40 points every 0.25 seconds."]},{"entry":["(7)","Run Assay: Test each formulation of Substrate Reagents with washing"]},{"entry":[{},"and priming SpectraMax L between each substrate. Bring all Substrate"]},{"entry":[{},"Reagents to room temperature before testing."]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}}}},"All confirmation experiments were tested using the MRSA Assay in order to ensure similar results on the actual assay as the Model System used to screen new formulations.",{"@attributes":{"id":"p-0449","num":"0459"},"tables":{"@attributes":{"id":"TABLE-US-00014","num":"00014"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"224pt","align":"left"}}],"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["(1)","Prepare Culture: Ten MRSA low performing strains and one MSSA strain"]},{"entry":[{},"were grown to log-phase in TSB in a 2 mL deep well block. Cells were"]},{"entry":[{},"spun down, washed with 1x PBS then resuspended in RPMI media."]},{"entry":["(2)","Normalize Bacteria: Measure 200 uL of RPMI culture and RPMI blank in"]},{"entry":[{},"Greiner Bio-one clear plate on VersaMax at 600 nm. Subtract blank OD"]},{"entry":[{},"from each strain. Normalize each strain to 0.05 OD in RPMI media."]},{"entry":["(3)","Dilute Bacteria: Dilute 0.05 OD culture 1:10 in RPMI media to 0.005 OD."]},{"entry":["(4)","Prepare Phage Media Reagent: Add Phage, Cefoxitin and Sodium"]},{"entry":[{},"Pyruvate to BSS-M56 including:"]},{"entry":[{},"a. Cefoxitin (5 ug\/mL)"]},{"entry":[{},"b. GW24 lysate stock (0.03X)"]},{"entry":[{},"c. Sodium Pyruvate (0.025M)"]},{"entry":["(5)","Set up Assay Plate = Add 198 uL of Phage Media Reagent and 2 uL of"]},{"entry":[{},"each dilution of bacteria (0.05 OD and 0.005 OD in RPMI, roughly"]},{"entry":[{},"equivalent to 20000 or 2000 CFU\/mL) or 2 uL of RPMI as a blank in two"]},{"entry":[{},"replicates."]},{"entry":["(6)","Incubate Assay Plate = Incubate Assay plate on orbital shaker at ~100"]},{"entry":[{},"rpm (speed 3) for 4 hours at 37\u00b0 C."]},{"entry":["(7)","Prepare Luminometer (SpectraMax L) = Wash reagent line with 70%"]},{"entry":[{},"ethanol followed by DI water then prime with the substrate. Set up"]},{"entry":[{},"software as Fast Kinetic with injection of 50 uL substrate at 250 ul\/sec"]},{"entry":[{},"after 10 baseline points and read at 40 points every 0.25 seconds."]},{"entry":["(8)","Run Assay = Test each formulation of Substrate Reagents with washing"]},{"entry":[{},"and priming SpectraMax L between each substrate."]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}}}},"Experiments for the development of Substrate Reagent formulation were designed to improve the following:",{"@attributes":{"id":"p-0451","num":"0461"},"tables":{"@attributes":{"id":"TABLE-US-00015","num":"00015"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"224pt","align":"left"}}],"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["(1)","Improve solubility via adding surfactants (Tween 20, Triton X-100, NP-"]},{"entry":[{},"40, Brij-35, SNS, etc.), adding solvents (Ethanol, Methanol, DMSO, etc.),"]},{"entry":[{},"adding non-volatile oils (Castor Oil)"]},{"entry":["(2)","Improve stability via adding stabilizers (Triethanolamine, Cyclodextrin"]},{"entry":[{},"etc.), adding antioxidants (Vitamin E, Vitamin E Acetate, Vitamin E PEG"]},{"entry":[{},"1000, Oxyrase, etc.), adjust method of Tridecanal addition (with"]},{"entry":[{},"surfactant, with solvent, into final solution, with antioxidant, etc.), storing"]},{"entry":[{},"Tridecanal and Substrate Reagent under nitrogen to reduce oxidation of"]},{"entry":[{},"aldehyde, and reducing possibility of microbial contamination by adding"]},{"entry":[{},"preservatives such as ProClin and by sterile filtration of the Substrate"]},{"entry":[{},"Reagent."]},{"entry":["(3)","Improve Assay Performance via adjustment of the pH of the formulation"]},{"entry":[{},"and the pH buffer system"]},{"entry":["(4)","Improve overall performance via determining the aldehyde with highest"]},{"entry":[{},"RLU output (tested aldehydes from 6-14 carbons in multiple formulations"]},{"entry":[{},"to determine if an improvement in solubility, stability and assay"]},{"entry":[{},"performance was observed)."]},{"entry":["(5)","Improve overall performance via adding antifoam in order to reduce"]},{"entry":[{},"foaming during preparation of reagent and addition of reagent to sample"]},{"entry":[{},"during the assay."]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}}}},"Analysis and Results","The kinetic reaction was plotted for each sample and a line fit to the average at each read point of three replicates. Typically results showed at 1:2000 dilution of 0.1 OD model system bacteria, roughly equivalent to 10,000 CFU\/mL or 2,000 CFU\/assay. The normalized maximum RLU to that of the reference substrate reagent was analyzed for stability experiments. At each stability time point, maximum RLU for each sample was normalized to the reference substrate maximum RLU. Normalized Maximum RLU was plotted over time points and linear regression with 95% CI was plotted.","Conclusions","The key parameters adjusted from the reference formulation for producing a final Substrate Reagent formulation are summarized in Table 9.",{"@attributes":{"id":"p-0456","num":"0466"},"tables":{"@attributes":{"id":"TABLE-US-00016","num":"00016"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 9"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Summary of Reagent Formulation Development Results"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Modification to Substrate Reagent","Reason"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["4.2 mM Tridecanal + TSB","Original Substrate Reagent"]},{"entry":["Remove TSB","Reduce possibility"]},{"entry":[{},"of contamination"]},{"entry":["Add 1% Tween 20","Improve Solubility"]},{"entry":["Adjust to pH 3 with 79.45% 0.1M","Improve Assay"]},{"entry":["Citric Acid-19.55% 0.2M Sodium","Performance"]},{"entry":"Phosphate Dibasic Buffer"},{"entry":["Add Tridecanal directly to","Improve Stability"]},{"entry":"concentrated surfactant"},{"entry":["Add Filtering of Substrate Reagent","Improve Stability"]},{"entry":"through 0.2 um PES membrane"},{"entry":["Add 0.05% ProClin 300","Improve Stability"]},{"entry":["Add Triethanolamine","Improve Stability"]},{"entry":["Change 1% Tween 20 to 0.5% Triton","Improve Stability,"]},{"entry":["X-100","Improve Solubility"]},{"entry":["Change from 79.45% 0.1M Citric","Improve Assay Performance,"]},{"entry":["Acid-19.55% 0.2M Sodium Phosphate","reduce possibility of"]},{"entry":["Dibasic Buffer to 82% 0.1M Citric","precipitation with removal"]},{"entry":["Acid-18% 0.1M Sodium Citrate","of phosphate buffer"]},{"entry":"Buffer, remain at pH 3"},{"entry":["Add 100 ppm Antifoam Y30","Improve Assay Performance"]},{"entry":["Add 0.5% Vitamin E Acetate","Improve Stability,"]},{"entry":[{},"reduce precipitation"]},{"entry":["Change Primary Tridecanal","Improve Assay Performance"]},{"entry":"Manufacturer from Alfa Aesar"},{"entry":"to Sigma\/OmegaChem"},{"entry":["Change 0.5% Vitamin E Acetate","Improve Assay Performance,"]},{"entry":["to 1-2% Vitamin E PEG 1000","Improve Solubility,"]},{"entry":[{},"Improve Stability"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"Two Substrate Reagent Formulations were prepared for two different storage temperatures, one for storage at 2-8\u00b0 C. and one at 18-24\u00b0 C.","Final Substrate Reagent Formulations Stored at 2-8\u00b0 C.","Formulation: 0.5% Triton X-100+4.2 mM Tridecanal+0.5% Vitamin E Acetate+100 ppm Antifoam Y30+0.5% Triethanolamine+82% 0.1 M Citric Acid+18% 0.1 M Sodium Citrate @pH3+0.05% ProClin 300. The formulation did not precipitate after 1 month at 2-8\u00b0 C. and was able to detect MRSA strains the same as on Day 0.","Final Substrate Reagent Formulations Stored at 18-24\u00b0 C.","Formulation: 0.5% Triton X-100+6.3 mM Tridecanal+100 ppm Antifoam Y30+0.5% Triethanolamine+82% 0.1 M Citric Acid+18% 0.1 M Sodium Citrate @pH3+2% a-Tocopherol-PEG 1000 Succinate+0.05% ProClin 300. The formulation did not precipitate after 1 month at 18-24\u00b0 C. and was able to detect MRSA strains the same as on Day 0.","Various other reagents and formulations can be used as known to those of skill in the art to derive the formulation.","Analytical Performance of Non-Replicative Transduction Particle-Based Viable Cell Reporter MRSA Assay","The analytical performance of the optimized NRTP MRSA assay was examined, including an analysis of the assay's limit of detection and an analysis of the cross-reactivity and microbial interference of the assay when challenged with non-target organisms.","A) Limit of Detection Assay","The Limit of Detection of the NRTP assay was assessed via determining the lowest amount of MRSA cells representing various strains that could produce a relative light unit (RLU) signal above that of a threshold determined from blank samples. MRSA strains included the SCCmec Types I, II, and IV as well as a MRSA strain carrying the mecA gene variant mecC\u2014a strain of MRSA that conventional FDA-cleared MRSA PCR assays have failed to detect.","The following key materials were used in the clinical performance study:","Growth Media Reagent: BSS-M56","Substrate Reagent: Final Substrate Reagent Formulations to be stored at 18-24\u00b0 C. as described above.","Transduction Particle Reagent: BSS-M56 base with 10 ug\/mL (i.e. 2\u00d7 concentration) cefoxitin and transduction particle reagent as described above at 2\u00d7 concentration.","LoD Study Protocol:","Overnight Culture:","For each MRSA strain and a MSSA negative control strain, 2 mL of TSB were inoculated with a colony of the strain previously grown on TSA plates. Overnight MRSA cultures included 5 ug\/mL cefoxitin. All samples were incubated overnight at 37\u00b0 C. in a shaking incubator.","Day Culture:","20 uL of each of the overnight cultures were transferred into a new culture tube containing 2 mL of Growth Media Reagent. The inoculums were then incubated at 37 C with shaking for approximately 1 hr 45 min, until the OD (600 nm) reached 0.1.","Serial Dilutions:\n\n","Enumeration of Bacterial Load:","5 uL of each well of row E was spotted onto a TSA plate which was then tilted to allow the spot of liquid to spread onto the plate (in order to later facilitate colony counting). (Row E is a 10dilution of row A). Plates were then incubated overnight at 37\u00b0 C.","Assay Preparation:","a) Wells of a white 96 well assay plate were filled with 100 uL of 2\u00d7 Transduction Particle Reagent.","b) Rows F and G (i.e., 10and 10-fold dilutions of row A, respectively) were then used to fill wells of the 96 well assay plate containing Transduction Particle Reagent such that each sample was added to the plate in four-replicates.","c) The plate was then sealed with a breathable seal and incubated for 4 hours at 37\u00b0 C. with moderate shaking, 50 rpm.","At the end of 4 hours, the plate was remove from incubator and immediately measured for luminescence on a SpectraMax L that injected 50 \u03bcl of the Substrate Reagent and measured luminescence for a period of 1 minute.","Analysis:","The luminescence data from each sample was plotted as RLU vs. time. Blank samples were used to determine a Cutoff calculated from all time points of the blank samples using the following formula: (Mean Blank RLU+3*SD Blank RLU)","The average peak RLU post-substrate injection was then obtained for each sample in order to determine the sample of highest dilution for which an RLU value was generated that was above the blank samples Cutoff. The colony forming unit (CFU) counts at the highest dilution for which an RLU value was generated that was above the blank samples Cutoff was determined from the enumeration study, and this CFU count was reported as the LoD in the study.","Results:","The LoD for all MRSA samples tested was determined to be below 10 CFU. Table 11 summarizes the results of the lowest LoDs obtained in the study.",{"@attributes":{"id":"p-0489","num":"0502"},"tables":{"@attributes":{"id":"TABLE-US-00017","num":"00017"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 11"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Results of the lowest LoDs obtained in the LoD study."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"126pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"SCCmec","LoD"]},{"entry":[{},"Type","(CFU)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"I","3"]},{"entry":[{},"II","2"]},{"entry":[{},"IV","3"]},{"entry":[{},"mecC","1"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"All MRSA strains tested resulted in fewer than 10 CFU detected with the NRTP assay above a Cutoff calculated from blank samples. MSSA did not generate RLU values above the blank samples Cutoff.","RLU values are shown at the highest dilution for which an RLU value was generated that was above the blank samples Cutoff were plotted as the average RLU value and standard deviation for the four replicates tested for each sample. The horizontal axis is set at the blank samples Cutoff and the CFU counts for the sample that generated each RLU data point is superimposed with the data. All MRSA samples generated RLU values above the Cutoff while MSSA did not.","Cross-Reactivity and Microbial Interference Study","A cross-reactivity and microbial interference study was performed. The purpose of the study was to test a set of bacterial strains commonly encountered in clinical samples and known to potentially be in the host range of the bacteriophage \u03c680\u03b1 in the MRSA Assay to see if there was cross reactivity or interference of these strains with phage or substrate used in the test.","Previous experiments with clinical samples had resulted in false positive results with a presence of and as indicated from the presence of blue and white colonies when plating on BBL\u2122 CHROMagar\u2122 plates. In addition, and may be within the infective or penetrative host range of the phage \u03c680\u03b1 which may also contribute to cross-reactivity in the MRSA assay. The study tested and for cross reactivity\/interference with Viability MRSA assay. Each strain was tested at high cell numbers in the order of 10, 10or 10cells in the assay volume. Tests were done without the addition of GW24 lysate to address potential autoluminescence of strains.","Experiment 1 tested various strains (MSSA-S121, NRS#9, NRS #6, ATCC 12228, ATCC 15305, ATCC 29212, ATCC 60193, ATCC 12453) for luminescence at high cell numbers under normal assay conditions.","Experiment 2: A subset of strains that were luminescent from Experiment 1 were re-assayed in the presence of various antibiotics at various concentrations to quench background luminescence.","Experiment 3and S32 (MRSA) were tested with various substrate formulations developed as described above without GW24 lysate and without incubation.","Experiment 4: ATCC 33090innocua and ATCC 19111were tested for background signal and non-specific luminescence and retested with various substrate formulations developed as described above along with and ","Experiment 5was retested with a final substrate formulation developed as described above.","Substrate Reagent formulations tested in this study are summarized in Table 9.",{"@attributes":{"id":"p-0501","num":"0514"},"tables":{"@attributes":{"id":"TABLE-US-00018","num":"00018"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 10"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Substrate Reagent Formulations"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"133pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Experiment","Substrate","Description"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["1","Original","1% Tween20 + 4.2 mM Tridecanal,"]},{"entry":[{},"Substrate","pH 3.0"]},{"entry":["2","Original"]},{"entry":[{},"Substrate"]},{"entry":["3","Substrate 1","6.3 mM Tridecanal + 0.5% Vitamin"]},{"entry":[{},{},"E Acetate, pH 3.0"]},{"entry":[{},"Substrate 2","20 mM Nonanal + 0.5% Vitamin E"]},{"entry":[{},{},"Acetate, pH 3.0"]},{"entry":[{},"Substrate 3","8.4 mM Tridecanal + 0.5% Vitamin"]},{"entry":[{},{},"E Acetate, pH 3.0"]},{"entry":[{},"Substrate 4","6.3 mM Tridecanal + 1% a-Tocopherol-"]},{"entry":[{},{},"PEG 1000 Succinate, pH 3.0"]},{"entry":["4","Original","1% Tween20 + 4.2 mM Tridecanal,"]},{"entry":[{},"substrate","pH 3.0"]},{"entry":[{},"Substrate 5","0.5% Triton + 4.2 mM Tridecanal (Sigma) +"]},{"entry":[{},{},"0.5% Vitamin E Acetate, pH 3.0"]},{"entry":["5","Substrate 6","6.3 mM Tridecanal + 2% VitE PEG,"]},{"entry":[{},{},"pH 3.0"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Methods\/Procedures:","The following were steps performed for the MRSA Assay.","A) Strains Grown for Experiments 1-5","On the day before the assay, an overnight culture was started in a deep 96 well plate at a 1:50 dilution in TSB from a frozen one-time use stock and incubated at 37\u00b0 C. on an orbital shaker for >15 hours. Bacteria (8 uL) in TSB (392 uL).","The absorbance of culture was measured on Versamax. The TSB was set as blank in template on SoftmaxPro. Optical density (OD) was measured at 600 nm.","On the day of the assay, cells were re-suspended to an OD 0.5 to set up the assays. Prepared BSS-M56 for Experiments 1-5.","B) Transduction Particle Media Reagent was Prepared for all Experiments 1, 2, 4 and 5 (No Transduction Particle Reagent Used in Experiment 3): 15 Ug\/mL Cefoxitin+GW24 Lysate Stock from as Described Above at 30\u00d7.","C) Sample Preparation:","Various dilutions were made from overnight cultures of strains. All strains were diluted BSS M56.","D) MRSA Assay was Run for Experiments 1-5","Media was loaded with or without phage and cefoxitin at 5 \u03bcg\/ml to assay plate. 2.5 ul cells were added. The assay plate was incubated with a plate lid at 37\u00b0 C. on an orbital shaker with the speed set to approximately 100 rpm for 4 hours.","Next, the assay plates were measured on the SpectraMax L with the following standard assay parameters:","Fast Kinetic Luminescence","Read for 20 time points at 0.5 second intervals. Substrate was injected with M injector with 50 uL\/well at 250 ul\/sec including 5 baseline reads. No incubation temperature was set and was read at room temperature.","The SpectraMax L was primed with Substrate Reagent before running the assay.","The Results were Analyzed with the Following:","A) Determined cutoff by averaging blank RLU across all replicates and time points and adding three standard deviations.","B) Determined maximum RLU for each sample using SoftMaxPro.","C) Determined if the maximum RLU was greater than the cutoff RLU, and if so, then the sample data was used for analysis.","Results Summary","Experiment 1:","Various strains were tested for cross reactivity and interference using the Original Substrate formulation, out of those tested, NRS#9, NRS #6and tested false positive in MRSA assay.","Experiment 2:","Out of the three strains tested, NRS #9 and tested MRSA positive with all Cefoxitin conditions tested. All three strains (NRS #9, , NRS #6) tested positive when no transduction particle reagent was used in the assay, indicating that non-specific luminescence was not transduction particle reagent-dependent but rather strain and substrate reagent dependent. Carb (Carbencillin) at all concentrations tested was effective in removing the false positive signal.","Experiment 3:",{"@attributes":{"id":"p-0527","num":"0540"},"i":"E. faecalis "},"Experiment 4:","Strains ATCC 33090 innocua, ATCC 19111-, were tested for luminescence with transduction particle reagent and substrate reagent as sp. can be within the host range of the bacteriophage used in the MRSA assay. Luminescence was observed from with and without transduction particle reagent using Original Substrate formulation indicating that the luminescence was due to non-specific reaction potentially with the substrate. Substrate 5 was effective in eliminating luminescence from but not ","Experiment 5:","Retested with Substrate 6. In two independent runs on two different days with high load of cells at 0.5 OD, the assay yielded negative results.","Conclusions","The cross-reactivity study demonstrated background luminescence from several bacterial species at high loads. The light output did not require transduction particle reagent and certain substrate formulations utilizing phosphate ions contributed to non-specific signal. Because no light output from cross-reactive species was observed from the use of transduction particle reagent, in the case that \u03c680\u03b1 penetrates cross-reactive species, light output is prevented from the lack of activity of the PclpB promoter that is operatively linked to the bacterial luciferase genes and\/or the lack of activity of the pT181 origin of replication within these species.","Replacing the buffer from sodium phosphate dibasic in the formulation with sodium citrate and citric acid eliminated background luminescence from all cross-reactive species tested except for . Substrate 6 with the added ingredient of a tocopherol-PEG 1000 Succinate eliminated the remaining non-specific signal from ","Clinical Performance of Non-Replicative Transduction Particle-Based Viable Cell Reporter MRSA Assay\u2014Results with Reference to Direct Plating onto CHROMAgar MRSA II","A MRSA screening assay was developed employing \u03c680\u03b1-based luxAB expressing non-replicative transduction particles (NRTP). The assay consisted of adding NRTP to a clinical sample suspected of containing MRSA, incubating the sample for a period of 4 hours at 37\u00b0 C., and then assaying the incubated sample by injecting an aldehyde into the sample while measuring for luminescence with a photomultiplier tube. The results of the assay were compared to that of commercially available chromogenic media designed for the detection of MRSA as a reference in order to determine the sensitivity and specificity of the assay. The NRTP-based assay was expected to correlate well with the culture-based reference since both require the presence of viable MRSA cells and both rely on the expression of the MRSA phenotype. The results showed excellent correlation with the reference.","The purpose of the study was to determine the performance of the NRTP-based MRSA Assay with reference to CHROMAgar MRSA II from testing remnant nasal swab samples collected for the purpose of MRSA screening.","Scope:","De-identified nasal swab samples collected from patients for the purpose of MRSA surveillance by a clinical institution were tested for the presence of MRSA using the NRTP-based MRSA Assay, CHROMAgar MRSA II, CHROMAgar SA and Blood Agar TSA via direct plating and via enriched culture followed by plating. The results of the NRTP-based MRSA Assay were compared with the results of the CHROMAgar MRSA II assay in order to calculate the sensitivity and specificity of the NRTP-based MRSA Assay with reference to CHROMAgar MRSA II.","The following key materials were used in the clinical performance study:","Growth Media Reagent: BSS-M56","Substrate Reagent: Final Substrate Reagent Formulations to be stored at 18-24\u00b0 C. as described as described above","Transduction Particle Reagent: BSS-M56 base with 10 ug\/mL (i.e. 2\u00d7 concentration) cefoxitin and transduction particle reagent as described above at 2\u00d7 concentration.","Methods\/Procedures","Clinical Sample Description:","Sample transport tubes containing liquid Amies (220093\u2014BD BBL\u2122 CultureSwab\u2122 Liquid Amies) were provided to a clinical institution for collecting de-identified remnant nasal swabs collected by the clinical institution. Prior to placing the nasal swabs into the provided sample transport tube, the clinical institution used the swab for performing their own direct culture MRSA screening by streaking the swab onto a culture plate. More specifically, anterior nares specimens were collected at the clinical institution internal standard procedures and using the clinical institution's standard collection swab. The clinical institution then performed direct culture screening with the swab. The remnant swab was then added to the sample transport tube in which the swab tip was submerged in the Amies buffer in the sample transport tube. Samples were then kept at room temperature for 2-24 hours prior to further processing.","Sample Handling:","Upon receipt, samples were stored overnight at room temperature in a biosafety cabinet upright to ensure swab immersion in the sample transport tube Amies buffer. After overnight storage, samples were further processed as follows.","Clinical Sample Preparation","Using a 1 mL Pipette, 300 \u03bcl of Growth Media Reagent was added to 15 mL falcon tubes.","The swabs from remnant nasal swabs were removed from the original transport tube and immersed into the Growth Media Reagent in a corresponding falcon tube. The swab contents were then eluted into the Growth Media Reagent in the falcon tube by rolling it back and forth in the Growth Media Reagent 4-6 times. The swab was then placed back into the original transport tube and stored at 2-8\u00b0 C. until the end of the study while the eluted clinical samples in the falcon tube were transferred to 1.5 mL tubes and kept at room temperature until further processing.","Running the NRTP MRSA Assay:","The following samples were loaded directly into a white 96 well assay plate.","Clinical Samples: 100 \u03bcl of the eluted material of each clinical sample in singlet.","MRSA positive control: 2 \u03bcl of a thoroughly mixed 0.1 OD culture of a known MRSA isolate into 98 uL of Growth Media Reagent in triplicate.","MSSA negative control: 2 \u03bcl of a thoroughly mixed 0.1 OD culture of a known MSSA isolate into 98 uL of Growth Media Reagent in triplicate.","Blanks: 100 \u03bcl of Growth Media Reagent in triplicate.","To each sample, 100 \u03bcL of Transduction Particle Reagent was added. The assay plate was then placed in an incubator set at 37\u00b0 C., shaking on orbital shaker for 4 hours. At the end of 4 hours, the plate was removed from incubator and immediately measured for luminescence on a SpectraMax L that injected 50 \u03bcl of the Substrate Reagent and measured luminescence for a period of 1 minute.","Bacteria Plating for Clinical Sample CFU Enumeration:","Each eluted clinical sample was plated in order to determine bacterial colony counts on CHROMAgar MRSA II, CHROMAgar SA and Blood Agar (TSA II) via direct and enriched culture as follows. Organism CFU counts were determined by direct plating. MRSA CFU counts were determined by plating on CHROMAgar MRSA II. CFU counts were determined by plating on CHROMAgar SA plate. CFU counts of any organism whose growth is supported by Blood Agar TSA were determined by plating on Blood Agar TSA. In the case that direct plating did not produce colonies due to the load of organisms being below the limit of detection of the plates used, sample enrichment was also performed by incubating a portion of the eluted clinical sample in TSB overnight at 37\u00b0 C. with shaking and then again plating the enriched culture on CHROMAgar MRSA II. All plates were incubated for 20-24 hours at 37\u00b0 C. After incubation, the CFU counts of any colonies appearing on each plate were recorded.","Analysis:","The presence and CFU load of MRSA, , and total organisms per eluted clinical sample were calculated based on the CFU counts obtained on CHROMAgar MRSA II, CHROMAgar SA, and Blood Agar TSA, respectively.","NRTP Assay Analysis:","Data from each sample were plotted as RLU vs. time.","Cutoff Determination:","The Assay Cutoff was calculated from all time points of the blank samples using the following formula: (Mean Blank RLU+3*SD Blank RLU).","MRSA Positive Determination:","The RLU of each time point after substrate injection was determined to be above or below the Assay Cutoff. If two or more data points after injection were above the Assay Cutoff then the sample was designated as \u201cMRSA Positive.\u201d","Results:","The MRSA positive results of the NRTP Assay were compared to those of the direct and enriched culture plating onto CHROMAgar MRSA II. The following calculations were conducted in order to determine the NTRP Assay Sensitivity and Specificity with reference to CHROMAgar MRSA II.",{"@attributes":{"id":"p-0571","num":"0584"},"tables":{"@attributes":{"id":"TABLE-US-00019","num":"00019"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"True Positive (TP)"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"7pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"210pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Sample that produced a MRSA positive result on both the NRTP"]},{"entry":[{},"Assay and CHROMAgar MRSA II"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"True Negative (TN)"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"7pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"210pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Sample that produced a MRSA negative result on both the NRTP"]},{"entry":[{},"Assay and CHROMAgar MRSA II"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"False Positive (FP)"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"7pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"210pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Sample that produced a MRSA positive result on the NRTP Assay and"]},{"entry":[{},"a MRSA negative result on CHROMAgar MRSA II"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"False Negative (FN)"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"7pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"210pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Sample that produced a MRSA negative result on the NRTP Assay and"]},{"entry":[{},"a MRSA positive result on CHROMAgar MRSA II"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Sensitivity = TP\/(TP + FN)"},{"entry":"Specificity = TN\/(TN + FP)"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}]}}},"Results with Reference to Direct Plating onto CHROMAgar MRSA II","Table 11 shows following results were obtained comparing the NRTP Assay with reference to direct plating on CHROMAgar MRSA II.",{"@attributes":{"id":"p-0574","num":"0587"},"tables":{"@attributes":{"id":"TABLE-US-00020","num":"00020"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"315pt","align":"center"}},"thead":{"row":{"entry":"TABLE 11"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"NRTP Assay Results vs. Direct Plating on CHROMAgar MRSA II"},{"entry":"Results"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"9"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"CHROMAgar","CHROMAgar","NRTP","NRTP",{},{},{},{}]},{"entry":["Total","MRSA II","MRSA II","ASSAY","ASSAY","True","True","False","False"]},{"entry":["Samples ","Positive","Negative","Positive","Negative","Positive","Negative","Positive","Negative"]},{"entry":{"@attributes":{"namest":"1","nameend":"9","align":"center","rowsep":"1"}}},{"entry":["69","7","62","12","57","7","57","5","0"]},{"entry":{"@attributes":{"namest":"1","nameend":"9","align":"center","rowsep":"1"}}}]}}]}}},"Based on the above data, the sensitivity and specificity of the assay with reference to direct plating onto CHROMAgar MRSA II were calculated to be:","Sensitivity=100%","Specificity=92%","Clinical Performance of Non-Replicative Transduction Particle-Based Viable Cell Reporter MRSA Assay\u2014Results with Reference to Enriched Culture, Followed by Plating onto CHROMAgar MRSA II","Based on the results with reference to direct plating on CHROMAgar MRSA II, all clinical samples were re-tested with reference to enriched culture, followed by plating on CHROMAgar MRSA II. The rationale for the follow-on testing was based on the possibility that false positive results when compared to direct plating may indeed be true positives that were detected by the NRTP assay but may have been missed by direct plating. A portion of the remaining eluted swab samples were re-tested via the NRTP assay as described above. Another portion of the remaining eluted swab samples were also tested via enriched culture, followed by plating onto CHROMAgar MRSA II. Enriched culture testing consisted of adding 100 uL of the remaining eluted swab material to 2 mL of TSB and incubating at 37 C with shaking for a period of 18-24 hours. The resulting culture was then streaked onto CHROMAgar MRSA II in order to determine the presence of MRSA in the culture. Table 12 summarizes the data from both the direct plating and enrichment followed by plating assays\u2014only the samples that produced a MRSA positive result on either NRTP Assay or CHROMAgar MRSA II are shown.",{"@attributes":{"id":"p-0580","num":"0593"},"tables":{"@attributes":{"id":"TABLE-US-00021","num":"00021"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 12"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"NRTP Assay Results vs. Direct Plating and Enriched"},{"entry":"Culture followed by Plating on CHROMAgar MRSA II"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Direct",{},"Enrichment +"]},{"entry":["Sample","NRTP","CHROMagar","Enrichment +","CHROMagar"]},{"entry":["#","Assay","MRSA II","NRTP Asssay","MRSA II"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1","+","+","+","+"]},{"entry":["2","+","+","+","+"]},{"entry":["3","+","+","+","+"]},{"entry":["4","+","+","+","+"]},{"entry":["5","+","+","+","+"]},{"entry":["6","+","+","+","+"]},{"entry":["7","+","+","+","+"]},{"entry":["8","+","\u2212","+","+"]},{"entry":["9","+","\u2212","+","+"]},{"entry":["10","+","\u2212","+","+"]},{"entry":["11","+","\u2212","+","+"]},{"entry":["12","+","\u2212","+","\u2212"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},"Only the samples that produced a MRSA positive result on either NRTP Assay or CHROMAgar MRSA II are shown.","Table 13 shows following results were obtained comparing the NRTP Assay with reference to enriched culture of clinical samples, followed by plating on CHROMAgar MRSA II.",{"@attributes":{"id":"p-0583","num":"0596"},"tables":{"@attributes":{"id":"TABLE-US-00022","num":"00022"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"315pt","align":"center"}},"thead":{"row":{"entry":"TABLE 13"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"NRTP Assay Results vs. Enriched Culture Followed By Plating on"},{"entry":"CHROMAgar MRSA II Results"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"9"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"CHROMAgar ","CHROMAgar ","NRTP","NRTP",{},{},{},{}]},{"entry":["Total","MRSA","MRSA","ASSAY","ASSAY","True","True","False","False"]},{"entry":["Samples","II Positive","II Negative","Positive","Negative","Positive","Negative","Positive","Negative"]},{"entry":{"@attributes":{"namest":"1","nameend":"9","align":"center","rowsep":"1"}}},{"entry":["69","11","58","12","57","11","57","1","0"]},{"entry":{"@attributes":{"namest":"1","nameend":"9","align":"center","rowsep":"1"}}}]}}]}}},"Based on the above data, the sensitivity and specificity of the assay with reference to enriched culture followed by plating onto CHROMAgar MRSA II was calculated to be:","Sensitivity=100%","Specificity=98.3%","In another example, a cefoxitin susceptibility assay was developed to determine the minimum inhibitory concentration of cefoxitin required to inhibit the growth of cefoxitin resistant . Unlike a MRSA cefoxitin resistance assay as described above, which differentiates cefoxitin sensitive from cefoxitin resistant , the MRSA cefoxitin susceptibility assay in this example describes the development of an assay to determine the minimum amount of cefoxitin needed to inhibit the grown of in the presence of cefoxitin.","The following key materials were used in the clinical performance study:","Growth Media Reagent: BSS-M56","Substrate Reagent: Final Substrate Reagent Formulations to be stored at 18-24\u00b0 C. as described in Example 7.","Transduction Particle Reagent: BSS-M56 base with 10 ug\/mL (i.e. 2\u00d7 concentration) cefoxitin and transduction particle reagent as described in Example 7 at 2\u00d7 concentration MIC Study Protocol.","Overnight Culture: For each MRSA strain (NRS35 and S7) and a MSSA negative control strain (MSSA121), 2 mL of TSB were inoculated with a colony of the strain previously grown on TSA plates. Overnight MRSA cultures included 5 ug\/mL cefoxitin. All samples were incubated overnight at 37\u00b0 C. in a shaking incubator.","Day Culture: 20 uL of each of the overnight cultures were transferred into a new culture tube containing 2 mL of Growth Media Reagent. The inoculums were then incubated at 37 C with shaking for approximately 1 hr 45 min, until the OD (600 nm) reached 0.1.","MIC Determination Via Plating:","a) Each of the day cultures was streaked onto TSA plates containing cefoxitin at 4, 8, 16, 32, 64, and 128 ug\/mL.","b) Plates were incubated for 18 hours at 37 C to determine growth.","NRTP Assay Preparation:","a) Wells of a white 96 well assay plate were filled with 100 uL of 2\u00d7 Transduction Particle Reagent.","b) For each of the day cultures, five wells were then filled with 100 uL of day culture.","c) For each of the day cultures, cefoxitin was added to one well each such that the cefoxitin concentration in the well was at 4, 8, 16, 32, 64, and 128 ug\/mL.","d) The plate was then sealed with a breathable seal and incubated for 4 hours at 37\u00b0 C. with moderate shaking, 50 rpm.","At the end of 4 hours, the plate was remove from incubator and immediately measured for luminescence on a SpectraMax L that injected 50 \u03bcl of the Substrate Reagent and measured luminescence for a period of 1 minute.","Analysis:","The maximum luminescence value after Substrate Reagent addition from each sample was plotted. MSSA sample RLU values were used to determine a Cutoff calculated using the following formula: (Mean MSSA RLU+3*SD MSSA RLU).","Results:",{"@attributes":{"id":"p-0605","num":"0618"},"figref":["FIG. 23","FIG. 24","FIG. 24"],"i":"S. aureus "},"As can be seen in , MRSA NRS25 exhibited a MIC of 128 ug\/mL cefoxitin while MRSA S7 exhibited a MIC of 64 ug\/mL cefoxitin. Correspondingly, MRSA NRS25 exhibited appreciable luminescence above the MSSA RLU cutoff to a cefoxitin concentration up to 64 ug\/mL cefoxitin while MRSA S7 exhibited luminescence above the MSSA RLU cutoff to a cefoxitin concentration up to 32 ug\/mL.","Based on the above data, the NRTP assay demonstrates that RLU values obtained from the assay correlate with MIC results and thus the NRTP assay may be used to develop antibiotic susceptibility assays.","As described above, a reporter transcript can be designed such that translation of the reporter gene sequence is blocked by cis-repression of the ribosome-binding site (RBS) of the reporter gene.","The following tools were used for designing the reporter transcripts of the invention.","1) RNA secondary structure was calculated using secondary structure program, such as Mfold.","2) Intermolecular RNA interactions were calculated using a software program such as RNA-RNA InterACTion prediction using Integer Programming (RactIP).","3) RNA secondary structure was visualized using Visualization Applet for RNA (VARNA).",{"@attributes":{"id":"p-0613","num":"0626"},"figref":"FIG. 25"},"As shown in detail in , the terminal loop 23 (T23) of the mecA transcript contains a YUNR consensus sequence. A YUNR (pYrimidine-Uracil-Nucleotide-puRine) consensus sequence has been shown to be a critical target for intermolecular RNA complexes in natural systems. A cis-repressing sequence was designed to form a stem-loop structure with the RBS of the reporter sequence, such that the cis-repressing sequence blocks binding of an RNA polymerase to the RBS of the reporter sequence. The reporter sequence was exposed upon binding of the loop of the cis-repressing stem-loop structure with T23 of the mecA transcript.","As shown in , a cis-repressing sequence  was added to the 5\u2032 terminus of the luxAB genes and designed to form a stem-loop structure that blocks the RBS sequence (\u201cAAGGAA\u201d)  of the luxA gene. The cis-repressing stem-loop structure was predicted to block the luxA RBS (\u201cAAGGAA\u201d) sequence, based on the lowest energy conformation of the luxAB transcript including the cis-repressing sequence at the 5\u2032 terminus of the luxAB transcript, as calculated by MFold and visualized with VARNA.","The first 61 nucleotides of the cis-repressed luxAB genes are shown in , up to the start codon AUG of the luxA gene. The RBS sequence \u201cAAGGAA\u201d includes bases 47-52. This terminal loop of the reporter transcript was designed to interact with (bind to) the terminal loop 23 (T23) of the mecA transcript, which contains a YUNR sequence.","The terminal loop of the cis-repressing sequence was designed to interact with T23 of the mecA transcript, such that hybridization of the cis-repressed luxAB transcript and the mecA transcript via the interaction of the loop from the cis-repressing stem-loop structure and T23 of the mecA transcript results in exposure of the RBS of the luxA gene.  shows the predicted inter-molecular interactions between the mecA T23 sequence and the cis-repressing sequence on the luxAB transcript calculated by RactIP and visualized by VARNA. Lines indicate base pairing between the mecA transcript and the cis-repressed luxAB transcript. The interaction between the two sequences results in exposure of the luxA RBS sequence AAGGAA and thus de-repression of the luxAB reporter.","In another example, a method for detecting a target mecA gene is provided using a mecA-luxAB reporter system. Here, mecA is the target transcript, and luxAB is the reporter molecule.","I. Construction of the Reporter Construct","A vector comprising a reporter construct encoding luxAB can be constructed via standard molecular biological techniques by incorporating the reporter construct into a shuttle vector capable of propagating in both and . The vector can contain an origin of replication that is functional in and a selectable marker that is expressed in and suitable for allowing the growth of cells transformed with the vector and grown under selective conditions. The vector can also contain an origin of replication that is functional in and a selectable marker that is expressed in and suitable for allowing the growth of cells transformed with the vector and grown under selective conditions. Propagation of the vector for conducting in vitro manipulations and for verification of manipulations can be accomplished via a suitable laboratory cloning strain of and the final modified vector can then be introduced into strains.","The reporter construct can be first introduced into a cell for transcribing the construct and producing the reporter transcript.","2. Construction of a cis-Repressed Reporter Transcript","Methods are provided for constructing a cis-repressed reporter transcript that can bind to a mecA-target transcript. The reporter transcript can be constructed via standard molecular biological techniques. The luxA and luxB genes serve as reporter genes and can be derived from . The genes lack a transcriptional promoter, and each contains its own ribosomal binding site (RBS). When both the luxA and luxB genes are translated in a cell, the luxA and luxB proteins complex to form the active luciferase enzyme (LuxAB). See Farinha, M. A. and A. M. Kropinski, Construction of broad-host-range plasmid vectors for easy visible selection and analysis of promoters. J. Bacteriol., 1990. 172(6): p. 3496-3499.","The cis-repressing sequence can be situated upstream of the luxAB genes and downstream of a promoter and includes a sequence that is complementary to the luxA RBS. A linker sequence can separate the complementary regions of the cis-repressing sequence and the luxA sequence. After transcription of the vector, the complementary regions of the cis-repressing sequence and the luxA RBS sequence complex, creating a stem loop that prevents docking of a ribosome and hence translation.","The stem loop of the reporter transcript is designed to destabilize and form an open complex when it interacts with a naturally-occurring mecA transcript sequence (endogenous to the cell). To activate translation of the luxA gene sequence, the natural mecA transcript serves as a trans-activating RNA that binds to the cis-repressed reporter transcript and opens the inhibitory stem loop that sequesters the RBS of the luxA gene. Once the RBS is not sequestered by the cis-repressing sequence, translation of luxA can occur. Transcription of the reporter construct is accomplished via operatively linking the reporter sequence to a constitutive promoter, upstream of the cis-repressing sequence.","An example of a target mecA gene sequence is shown in . The sequence is a mecA gene loci DNA sequence (from subsp. SA40, complete genome GenBank: CP003604.1; SEQ ID NO:15) and can be used for generating a reporter construct comprising a reporter sequence and a cis-repressing sequence. The \u221210 position , the transcription start position , the RBS , the coding region (in grey ) and the transcription termination sequence  are shown.",{"@attributes":{"id":"p-0627","num":"0640"},"figref":"FIG. 30","b":["3001","3002"]},{"@attributes":{"id":"p-0628","num":"0641"},"figref":"FIG. 31","i":"Vibrio fischeri ","b":["3101","3102","3103","3104","3105","3106"]},{"@attributes":{"id":"p-0629","num":"0642"},"figref":"FIG. 32","b":["3201","3202","3203","3204"]},{"@attributes":{"id":"p-0630","num":"0643"},"figref":"FIG. 33","b":["3301","3302","3303","3304","3305"]},"3. Methods for Detecting the Presence or Absence of a mecA Target Transcript Using the Reporter Transcript","Examples are provided for detecting the presence or absence of a mecA target transcript in a cell using the reporter transcripts of the invention.  shows an example of a cell comprising a vector  that encodes a reporter transcript , where there is no endogenous mecA transcript in the cell  (e.g., the cell's genome does not contain the mecA gene). In this case, the cis-repressing sequence  binds to the RBS  of the luxAB genes. In some embodiments, the cis-repressing sequence  can bind to a portion of or all of the RBS of the luxA gene, the RBS of the luxB gene, or both. This binding event blocks and prevents the translation of the luxAB genes, and the reporter molecule (e.g., luciferase) is not produced in the cell. Thus, no signal is detected, indicating the absence of the mecA gene in the cell.","In another example, the cell includes an endogenous mecA transcript (e.g., the cell's genome contains the mecA gene).  shows a vector  introduced into a cell . The vector  encodes the reporter transcript , which includes a cis-repressing sequence  and a reporter sequence (luxA and luxB genes). When the mecA transcript  present in the cell binds to the cis-repressing sequence , the inhibitory hairpin loop opens up and the RBS  for the luxA gene is exposed. Translation of the reporter sequences (luxA and luxB) can occur, resulting in the formation of a luxAB enzyme . The luxAB enzyme  produces a detectable luminescent signal . In this manner, the transcript reporter vector  reports the presence of endogenous mecA transcripts  within a cell .","While the invention has been particularly shown and described with reference to a preferred embodiment and various alternate embodiments, it will be understood by persons skilled in the relevant art that various changes in form and details can be made therein without departing from the spirit and scope of the invention.","All references, issued patents and patent applications cited within the body of the instant specification are hereby incorporated by reference in their entirety, for all purposes.",{"@attributes":{"id":"p-0636","num":"0000"},"ul":{"@attributes":{"id":"ul0007","list-style":"none"},"li":["1. Michael G. Schmidt, D. A. S., Caroline Westwater, Joseph W. Dolan, Brian D. Hoel, Philip A. Werner, James S. Norris, Laura M. Kasman, Nucleic Acid Delivery and Expression, 2005.","2. Kreiswirth, B. N. et al., The toxic shock syndrome exotoxin structural gene is not detectably transmitted by a prophage. Nature, 1983. 305(5936): p. 709-712.","3. Ubeda, C. et al., Specificity of staphylococcal phage and SaPI DNA packaging as revealed by integrase and terminase mutations. Molecular Microbiology, 2009. 72(1): p. 98-108.","4. Otsuji, N. et al., Induction of Phage Formation in the Lysogenic K-12 by Mitomycin C. Nature, 1959. 184(4692): p. 1079-1080.","5. Brantl, S. (2007) Regulatory mechanisms employed by cis-encoded antisense RNAs. Curr. Opin. Microbiol. 10, 102-109.","6. Isaacs, F. J. et al. (2004) Engineered riboregulators enable post-transcriptional control of gene expression. Nat. Biotechnol. 22, 841-847.","7. Pfeiffer, V. et al. (2009) Coding sequence targeting by MicC RNA reveals bacterial mRNA silencing downstream of translational initiation. Nat. Struct. Mol. Biol. 16, 840-846.","8. Opdyke, J. A. et al. (2004) GadY, a small-RNA regulator of acid response genes in . J. Bacteriol. 186, 6698-6705.","9. Carriere, C., et al., . Journal of Clinical Microbiology, 1997. 35(12): p. 3232-3239.","10. Merten, O.-W. and M. Al-Rubeai, . Methods in Molecular Biology. Vol. 737. 2011.","11. Lofdahl, S., J. E. Sjostrom, and L. Philipson, --11. Journal of Virology, 1981. 37(2): p. 795-801.","12. Charpentier, E., et al., --. Appl. Environ. Microbiol., 2004. 70(10): p. 6076-6085.","13. Novick, R. P., I. Edelman, and S. Lofdahl, -Journal of Molecular Biology, 1986. 192(2): p. 209-220.","14. Westwater, C., et al., Development of a P1 phagemid system for the delivery of DNA into Gram-negative bacteria. Microbiology, 2002. 148(4): p. 943-950.","15. Norris, J. U., et al., Tissue-Specific and Pathogen-Specific Toxic Agents and Ribozymes. 1999.","16. Maiques, E., et al., Role of Staphylococcal Phage and SaPI Integrase in Intra- and Interspecies SaPI Transfer. J. Bacteriol., 2007. 189(15): p. 5608-5616.","17. Frees, D., et al., Clp ATPases are required for stress tolerance, intracellular replication and biofilm formation in . Molecular Microbiology, 2004. 54(5): p. 1445-1462.","18. Arnaud, M., A. Chastanet, and M. Debarbouille, New Vector for Efficient Allelic Replacement in Naturally Nontransformable, Low-GC-Content, Gram-Positive Bacteria. Appl. Environ. Microbiol., 2004. 70(11): p. 6887-6891.","19. Tormo, M. A., et al., Pathogenicity Island DNA Is Packaged in Particles Composed of Phage Proteins. J. Bacteriol., 2008. 190(7): p. 2434-2440.","20. Arthur, M., et al., The VanS sensor negatively controls VanR-mediated transcriptional activation of glycopeptide resistance genes of Tn1546 and related elements in the absence of induction. J. Bacteriol., 1997. 179(1): p. 97-106.","21. Karlsson, S., et al., Expression of Toxins A and B and Their Sigma Factor TcdD Is Controlled by Temperature. Infect. Immun., 2003. 71(4): p. 1784-1793.","22. Daniel Sobek, J. R., Enzyme detection system with caged substrates, 2007, Zymera, Inc.","23. Samie Jaffrey, J. P., Coupled recognition\/detection system for in vivo and in vitro use, 2010, Cornell University.","24. Good, L., Translation repression by antisense sequences. Cellular and Molecular Life Sciences, 2003. 60(5): p. 854-861.","25. Sabine, B., Antisense-RNA regulation and RNA interference. Biochimica et Biophysica Acta (BBA)\u2014Gene Structure and Expression, 2002. 1575(1-3): p. 15-25."]}},{"@attributes":{"id":"p-0637","num":"0674"},"tables":{"@attributes":{"id":"TABLE-US-00023","num":"00023"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"273pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"168pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Native\u2003P1\u2003pac-site",{}]},{"entry":["SEQ\u2003ID\u2003NO:\u20031",{}]},{"entry":["CCACTAAAAAGCATGATCATTGATCACTCTAATGATCAACATGCAGGTGATCACATTGCGGC",{}]},{"entry":"TGAAATAGCGGAAAAACAAAGAGTTAATGCCGTTGTCAGTGCCGCAGTCGAGAATGCGAAGC"},{"entry":"GCCAAAATAAGCGCATAAATGATCGTTCAGATGATCATGACGTGATCACCCGC"},{"entry":{}},{"entry":"P1\u2003pac-site\u2003with\u2003silent\u2003mutations,\u2003lower\u2003case\u2003letters\u2003signify"},{"entry":"mutated\u2003bases"},{"entry":["SEQ\u2003ID\u2003NO:\u20032",{}]},{"entry":["CCACTAAAAAGCATGATaATaGAcCACTCTAAcGAcCAACATGCAGGgGAgCACATTGCGGC",{}]},{"entry":"TGAAATAGCGGAAAAgCAgAGgGTgAATGCCGTTGTCAGTGCCGCAGTCGAGAATGCGAAGC"},{"entry":"GCCAAAATAAGCGCATAAAcGAcCGTTCAGAcGAcCATGACGTtATtACCCGC"},{"entry":{}},{"entry":{}},{"entry":"P1\u2003lytic\u2003replicon\u2003containing\u2003the\u2003C1\u2003repressor-controlled\u2003P53"},{"entry":"promoter,\u2003the\u2003promoter\u2003P53\u2003antisense,\u2003the\u2003repL\u2003genes,\u2003and\u2003an"},{"entry":"in-frame\u2003deletion\u2003of\u2003the\u2003kilA\u2003gene"},{"entry":["SEQ\u2003ID\u2003NO:\u20033",{}]},{"entry":["CACTATAGGGCGAATTGGCGGAAGGCCGTCAAGGCCGCATTTGGGCCCGGCGCGCCGGATCC",{}]},{"entry":"GCTAGCTCTAGACTGGCAGGTTTCTGAGCAGATCGTCCAACCCGATCTGGATCGGGTCAGAA"},{"entry":"AAATTTGCTCTAATAAATTTCGTTTTCTAAGTGCAAAGAATCACCATTTCGAGCTGGTGATT"},{"entry":"GAAGGTTGATGCAAATTTGGAGAAAAAATGCAACAAACATTCAATGCGGATATGAATATATC"},{"entry":"AAACCTTCATCAAAATGTCGATCCTTCAACCACTCTGCCCGTTATTTGTGGTGTTGAAATTA"},{"entry":"CGACCGACCGCGCTGGCCGTTACAACCTTAATGCTCTACACAGAGCGAGCGGACTCGGTGCC"},{"entry":"CATAAAGCGCCAGCTCAATGGCTAAGAACGCTGTCAGCTAAACAGCTCATCGAAGAGCTTGA"},{"entry":"AAAAGAAACTATGCAGAATTGCATAGTTTCGTTCACAAGCAATGGAAGCAGGATTTCTTTCA"},{"entry":"CGACTCGTATAACCGGCAAAGGTCAGCAGTGGCTGATGAAGCGATTGCTTGATGCTGGTGTG"},{"entry":"CTGGTACCTGTCGCGGCAACGCGCTAACAGACGTAGTAAGAACCACCAGCATTGTAATGCTG"},{"entry":"GCTAAAGTCACTTTCCTGAGCTGTATAACGATGAGCGATTTTACTTTTTCTGGCTATGAATT"},{"entry":"GGCCTGCTTTGTAACACACTCCGGTCTATCCCGTAGCGCCGGGCATATCCTGTCGCAATGTG"},{"entry":"CAAATCTCGCGGCAACAACCAGTGAATACTTCATTCACAAGCCTCACCGCCTGATCGCGGCA"},{"entry":"GAAACTGGTTATAGCCAATCAACCGTCGTTCGTGCATTCCGTGAAGCTGTAAACAAAGGAAT"},{"entry":"TCTGTCTGTAGAGATTGTTATCGGCGATCACCGTGAACGTCGCGCTAACCTGTACCGGTTTA"},{"entry":"CACCATCCTTTTTGGCCTTCGCACAACAAGCCAAAAATGCGCTGATAGAAAGCAAATTAAAG"},{"entry":"ATCTCTTCAGCGGCAACCAAGGTTAAAGCTGTTCTCGCTAAGACATTGGCTTTATTTAATTT"},{"entry":"TTTATCCACACCCCCATGTCAAAATGATACCCCCTCCCCCTGTCAGGATGACGTGGCAATAA"},{"entry":"AGAATAAGAAGTCACAAGTTAAAAAAACAAAAAGATCAGTTTCCGGCGGTGCCGGAACAACC"},{"entry":"AGCCTCAAAAAATTGACTTCATGGATCGCTAAGGCAAAAGCAAAGGCTGACAATCTGCGGTT"},{"entry":"ATCCAAAAAACGCACTCAAAAACATGAGTTCAAGCAGAAAGTAGAGGCGGCTGCGCGGAAAT"},{"entry":"ATGCTTACCTGAAGAACAAGCGTTCGCCTGATATTGGCGGGATATCAAACTTCGATAACCTA"},{"entry":"CCGCATTGCATGACGGTAAACGAAGCTCTTAATGCGGTTTTAGCCAAAAATAAAGATAACGA"},{"entry":"ACAATGGGGTATACCGGCAGGATTCAGAGGGTAATGAATTGCTCTAATTATAACCATGCATA"},{"entry":"CTTTCAACACCTCTAGTTTGCCATGAGGCAAACTCATAGGTGTCCTGGTAAGAGGACACTGT"},{"entry":"TGCCAAAACTGGACGCCCCATTATTGCAATTAATAAACAACTAACGGACAATTCTACCTAAC"},{"entry":"AATAAGTGGCTTAAAAAAACCCGCCCCGGCGGGTTTTTTTATCTAGAGCTAGCGGATCCGGC"},{"entry":"GCGCCGGGCCCTTCTGGGCCTCATGGGCCTTCCGCTCACTGCCCGCTTTCCAG"},{"entry":{}},{"entry":"Pblast\u2003promoter\u2003sequence"},{"entry":["SEQ\u2003ID\u2003NO:\u20034",{}]},{"entry":["CGTCAGGTGGCACTTTTCGGGAAATGTGCGCGGAACCCCTATTTGTTTATTTTCTAAATACA",{}]},{"entry":"TTCAAATATGTATCCGCTCATGAGACAATAACCCTGATAAATGCTTCAATAATATTGAAAAG"},{"entry":"GAAGAGT"},{"entry":{}},{"entry":{"i":"S.\u2003aureus"}},{"entry":"variant\u2003pT181cop-623\u2003repC"},{"entry":["SEQ\u2003ID\u2003NO:\u20035",{}]},{"entry":["TTTGCGGAAAGAGTTAGTAAGTTAACAGAAGACGAGCCAAACCTAAATGGTTTAGCAGGAAA",{}]},{"entry":"CTTAGATAAAAAAATGAATCCAGAATTATATTCAGAACAGGAACAGCAACAAGAGCAACAAA"},{"entry":"AGAATCAAAAACGAGATAGAGGTATGCACTTATAGAACATGCATTTATGCCGAGAAAACTTA"},{"entry":"TTGGTTGGAATGGGCTATGTGTTAGCTAACTTGTTAGCGAGTTGGTTGGACTTGAATTGGGA"},{"entry":"TTAATCCCAAGAAAGTACCGGCTCAACAACCCATAAAGCCCTGTAGGTTCCGNCCAATAAGG"},{"entry":"AAATTGGAATAAAGCAATAAAAGGAGTTGAAGAAATGAAATTCAGAGAAGCCTTTGAGAATT"},{"entry":"TTATAACAAGTAAGTATGTACTTGGTGTTTTAGTAGTCTTAACTGTTTACCAGATAATACAA"},{"entry":"ATGCTTAAATAAAAAAAGACTTGATCTGATTAGACCAAATCTTTTGATAGTGTTATATTAAT"},{"entry":"AACAAAATAAAAAGGAGTCGCTCACGCCCTACCAAAGTTTGTGAACGACATCATTCAAAGAA"},{"entry":"AAAAACACTGAGTTGTTTTTATAATCTTGTATATTTAGATATTAAACGATATTTAAATATAC"},{"entry":"ATCAAGATATATATTTGGGTGAGCGATTACTTAAACGAAATTGAGATTAAGGAGTCGATTTT"},{"entry":"TTATGTATAAAAACAATCATGCAAATCATTCAAATCATTTGGAAAATCACGATTTAGACAAT"},{"entry":"TTTTCTAAAACCGGCTACTCTAATAGCCGGTTGGACGCACATACTGTGTGCATATCTGATCC"},{"entry":"AAAATTAAGTTTTGATGCAATGACGATCGTTGGAAATCTCAACCGAGACAACGCTCAGGCCC"},{"entry":"TTTCTAAATTTATGAGTGTAGAGCCCCAAATAAGACTTTGGGATATTCTTCAAACAAAGTTT"},{"entry":"AAAGCTAAAGCACTTCAAGAAAAAGTTTATATTGAATATGACAAAGTGAAAGCAGATAGTTG"},{"entry":"GGATAGACGTAATATGCGTATTGAATTTAATCCAAACAAACTTACACGAGATGAAATGATTT"},{"entry":"GGTTAAAACAAAATATAATAAGCTACATGGAAGATGACGGTTTTACAAGATTAGATTTAGCC"},{"entry":"TTTGATTTTGAAGATGATTTGAGTGACTACTATGCAATGTCTGATAAAGCAGTTAAGAAAAC"},{"entry":"TATTTTTTATGGTCGTAATGGTAAGCCAGAAACAAAATATTTTGGCGTGAGAGATAGTAATA"},{"entry":"GATTTATTAGAATTTATAATAAAAAGCAAGAACGTAAAGATAATGCAGATGCTGAAGTTATG"},{"entry":"TCTGAACATTTATGGCGTGTAGAAATCGAACTTAAAAGAGATATGGTGGATTACTGGAATGA"},{"entry":"TTGCTTTAGTGATTTACATATCTTGCAACCAGATTGGAAAACTATCCAACGCACTGCGGATA"},{"entry":"GAGCAATAGTTTTTATGTTATTGAGTGATGAAGAAGAATGGGGAAAGCTTCACAGAAATTCT"},{"entry":"AGAACAAAATATAAGAATTTGATAAAAGAAATTTCGCCAGTCGATTTAACGGACTTAATGAA"},{"entry":"ATCGACTTTAAAAGCGAACGAAAAACAATTGCAAAAACAAATCGATTTTTGGCAACATGAAT"},{"entry":"TTAAATTTTGGAAATAGTGTACATATTAATATTACTGAACAAAAATGATATATTTAAACTAT"},{"entry":"TCTAATTTAGGAGGATTTTTTTATGAAGTGTCTATTTAAAAATTTGGGGAATTTATATGAGG"},{"entry":"TGAAAGAATAATTTACCCCTATAAACTTTAGCCACCTCAAGTAAAGAGGTAAAATTGTTTAG"},{"entry":"TTTATATAAAAAATTTAAAGGTTTGTTTTATAGCGTTTTATTTTGGCTTTGTATTCTTTCAT"},{"entry":"TTTTTAGTGTATTAAATGAAATGGTTTTAAATGTTTCTTTACCTGATATTGCAAATCATTTT"},{"entry":"AATACTACTCCTGGAATTACAAACTGGGTAAACACTGCATATATGTTAACTTTTTCGATAGG"},{"entry":"AACAGCAGTATATGGAAAATTATCTGATTATATAAATATAAAAAAATTGTTAATTATTGGTA"},{"entry":"TTAGTTTGAGCTGTCTTGGTTCATTGATTGCTTTTATTGGGCCCACCTAGGCAAATATGCTC"},{"entry":"TTACGTGCTATTATTTAAGTGACTATTTAAAAGGAGTTAATAAATATGCGGCAAGGTATTCT"},{"entry":"TAAATAAACTGTCAATTTGATAGCGGGAACAAATAATTAGATGTCCTTTTTTAGGAGGGCTT"},{"entry":"AGTTTTTTGTACCCAGTTTAAGAATACCTTTATCATGTGATTCTAAAGTATCCAGAGAATAT"},{"entry":"CTGTATGCTTTGTATACCTATGGTTATGCATAAAAATCCCAGTGATAAAAGTATTTATCACT"},{"entry":"GGGATTTTTATGCCCTTTTGGGTTTTTGAATGGAGGAAAATCACATGAAAATTATTAATATT"},{"entry":"GGAGTTTTAGCTCATGTTGATGCAGGAAAAACTACCTTAACAGAAAGCTTATTATATAACAG"},{"entry":"TGGAGCGATTACAGAATTAGGAAGCGTGGACAAAGGTACAACGAGGACGGATAATACGCTTT"},{"entry":"TAGAACGTCAGAGAGGAATTACAATTCAGACAGGAATAACCTCTTTTCAGTGGGAAAATACG"},{"entry":"AAGGTGAACATCATAGACACGCCAGGACATATGGATTTCTTAGCAGAAGTATATCGTTCATT"},{"entry":"ATCAGTTTTAGATGGGGCAATTCTACTGATTTCTGCAAAAGATGGCGTACAAGCACAAACTC"},{"entry":"GTATATTATTTCATGCACTTAGGAAAATGGGGATTCCCACAATCTTTTTTATCAATAAGATT"},{"entry":"GACCAAAATGGAATTGATTTATCAACGGTTTATCAGGATATTAAAGAGAAACTTTCTGCCGA"},{"entry":"AATTGTAATCAAACAGAAGGTAGAACTGTATCCTAATATGTGTGTGACGAACTTTACCGAAT"},{"entry":"CTGAACAATGGGATACGGTAATAGAGGGAAACGATAACCTTTTAGAGAAATATATGTCCGGT"},{"entry":"AAATCATTAGAAGCATTGGAACTCGAACAAGAGGAAAGCATAAGATTTCAGAATTGTTCTCT"},{"entry":"GTTCCCTCTTTATCATGGAAGTGCAAAAAGTAATATAGGGATTGATAACCTTATAGAAGTTA"},{"entry":"TTACTAATAAATTTTATTCATCAACACATCGAGGTCCGTCTGAACTTTGCGGAAATGTTTTC"},{"entry":"AAAATTGAATATACAAAAAAAAGACAACGTCTTGCATATATACGCCTTTATAGTGGAGTACT"},{"entry":"ACATTTACGAGATTCGGTTAGAGTATCAGAAAAAGAAAAAATAAAAGTTACAGAAATGTATA"},{"entry":"CTTCAATAAATGGTGAATTATGTAAGATTGATAGAGCTTATTCTGGAGAAATTGTTATTTTG"},{"entry":"CAAAATGAGTTTTTGAAGTTAAATAGTGTTCTTGGAGATACAAAACTATTGCCACAGAGAAA"},{"entry":"AAAGATTGAAAATCCGCACCCTCTACTACAAACAACTGTTGAACCGAGTAAACCTGAACAGA"},{"entry":"GAGAAATGTTGCTTGATGCCCTTTTGGAAATCTCAGATAGTGATCCGCTTCTACGATATTAC"},{"entry":"GTGGATTCTACGACACATGAAATTATACTTTCTTTCTTAGGGAAAGTACAAATGGAAGTGAT"},{"entry":"TAGTGCACTGTTGCAAGAAAAGTATCATGTGGAGATAGAACTAAAAGAGCCTACAGTCATTT"},{"entry":"ATATGGAGAGACCGTTAAAAAATGCAGAATATACCATTCACATCGAAGTGCCGCCAAATCCT"},{"entry":"TTCTGGGCTTCCATTGGTTTATCTGTATCACCGCTTCCGTTGGGAAGTGGAATGCAGTATGA"},{"entry":"GAGCTCGGTTTCTCTTGGATACTTAAATCAATCATTTCAAAATGCAGTTATGGAAGGGGTAC"},{"entry":"GCTATGGTTGCGAACAAGGATTATATGGTTGGAATGTGACGGATTGTAAAATCTGTTTTAAG"},{"entry":"TACGGTTTATACTATAGCCCTGTTAGTACTCCAGCAGATTTTCGGATGCTTACTCCTATTGT"},{"entry":"ACTGGAGCAAGCCTTTAGAAAAGCTGGAACAGAATTGTTAGAGCCATATCTTAGTTTTAAAG"},{"entry":"TTTATGCACCACAGGAATATCTTTCNCGGGCATATAACGATGCTCCCAAATATTGTGCAAAT"},{"entry":"ATCGTAAATACTCAACTGAAAAATAATGAGGTCATTATTATTGGAGAAATTCCTGCTCGATG"},{"entry":"TATTCAAGATTATCGCAATGATTTAACTTTTTTTACAAATGGGCTTAGTGTTTGTTTAGCAG"},{"entry":"AGCTAAAAGGATATCAGGTTACCACTGGCGAACCTGTTTGCCAGACCCGTCGTCTAAATAGT"},{"entry":"CGGATAGATAAAGTAAGATATATGTTCAATAAAATAACTTAGTGCGTTTTATGTTGTTATAT"},{"entry":"AAATATGGTTTCTTATTAAATAAGATGAAATATTCTTTAATATAGATTTGAATTAAAGTGGA"},{"entry":"AAGGAGGAGATTGTTATTATAAACTACAAGTGGATATTGTGTCCTATTTGTGGAAATAAAAC"},{"entry":"AAGACTACGAATACGAGTGGATACTATACTTAAAAATTTCCCTTTATACAGCCCCAAATGTA"},{"entry":"AGAACGAAACTTTAATTAATGTTCAAAAAATGAATATAATAACAATCAAAGAGCCAGACGCC"},{"entry":"AAGACGCAGAGCCGATAATTTGAGAAATGAAACTCTCATCTTATCGGCTCTTTTTGTTTATC"},{"entry":"TGAATTTTACTGACTAGCCTTCAATATTTCC"},{"entry":{}},{"entry":"P1\u2003pacA\u2003gene,\u2003lower\u2003case\u2003letters\u2003signify\u2003deleted\u2003pac-site"},{"entry":"sequence"},{"entry":["SEQ\u2003ID\u2003NO:\u20036",{}]},{"entry":["GTGACCTGGGACGATCACAAGAAGAATTTTGCTCGCCTGGCGCGAGATGGTGGTTACACCAT",{}]},{"entry":"CGCACAGTATGCCGCCGAGTTTAATCTTAACCCTAATACCGCACGTCGTTATCTCCGTGCCT"},{"entry":"TCAAAGAAGACACCAGGACTACGGACAGCCGCAAGCCAAATAAGCCAGTCAGGAAGccacta"},{"entry":"aaaagcatgatcattgatcactctaatgatcaacatgcaggtgatcacattgcggctgaaat"},{"entry":"agcggaaaaacaaagagttaatgccgttgtcagtgccgcagtcgagaatgcgaagcgccaaa"},{"entry":"ataagcgcataaatgatcgttcagatgatcatgacgtgatcacccgcGCCCACCGGACCTTA"},{"entry":"CGTGATCGCCTGGAACGCGACACCCTGGATGATGATGGTGAACGCTTTGAATTCGAAGTTGG"},{"entry":"CGATTACCTGATAGATAACGTTGAAGCGCGGAAGGCCGCGCGCGCTATGTTGCGTCGGTCCG"},{"entry":"GGGCCGATGTTCTGGAAACCACTCTTCTGGAAAAGTCTCTTTCTCATCTCCTTATGCTGGAG"},{"entry":"AACGCCAGGGATACGTGTATTCGCCTGGTGCAGGAAATGCGCGATCAGCAAAAAGACGATGA"},{"entry":"TGAAGGTACTCCGCCTGAATACCGTATCGCGAGCATGCTAAACAGCTGTTCCGCGCAGATAA"},{"entry":"GCAGCCTGATCAACACCATTTACAGCATCCGGAATAACTATCGAAAAGAAAGCCGGGAGGCG"},{"entry":"GAAAAGCACGCTTTATCTATGGGGCAAGCTGGCATTGTTAAGCTGGCATACGAACGAAAGCG"},{"entry":"TGAAAATAACTGGTCAGTGCTGGAAGCGGCTGAATTCATCGAGGCGCATGGAGGAAAAGTGC"},{"entry":"CGCCCCTGATGCTGGAGCAAATCAAAGCCGATCTGCGTGCTCCTAAGACCAATACCGATGAT"},{"entry":"GAGGAAAACCAAACAGCATCTGGCGCTCCATCACTTGAAGATCTGGATAAAATCGCGCGAGA"},{"entry":"ACGGGCCGCCAGCCGCCGCGCTGATGCCGCATTGTGGATTGAGCATCGTAGAGAAGAAATTG"},{"entry":"CCGATATCGTCGATACAGGTGGTTATGGTGATGTCGATGCGGAAGGCATATCAAACGAAGCA"},{"entry":"TGGCTTGAACAGGATCTGGACGAAGACGAGGAGGAAGACGAAGAAGTTACCCGCAAACTGTA"},{"entry":"CGGGGATGATGATTAA"},{"entry":{}},{"entry":"Native\u2003P1\u2003pacA\u2003gene"},{"entry":["SEQ\u2003ID\u2003NO:\u20037",{}]},{"entry":["GTGACCTGGGACGATCACAAGAAGAATTTTGCTCGCCTGGCGCGAGATGGTGGTTACACCAT",{}]},{"entry":"CGCACAGTATGCCGCCGAGTTTAATCTTAACCCTAATACCGCACGTCGTTATCTCCGTGCCT"},{"entry":"TCAAAGAAGACACCAGGACTACGGACAGCCGCAAGCCAAATAAGCCAGTCAGGAAGCCACTA"},{"entry":"AAAAGCATGATCATTGATCACTCTAATGATCAACATGCAGGTGATCACATTGCGGCTGAAAT"},{"entry":"AGCGGAAAAACAAAGAGTTAATGCCGTTGTCAGTGCCGCAGTCGAGAATGCGAAGCGCCAAA"},{"entry":"ATAAGCGCATAAATGATCGTTCAGATGATCATGACGTGATCACCCGCGCCCACCGGACCTTA"},{"entry":"CGTGATCGCCTGGAACGCGACACCCTGGATGATGATGGTGAACGCTTTGAATTCGAAGTTGG"},{"entry":"CGATTACCTGATAGATAACGTTGAAGCGCGGAAGGCCGCGCGCGCTATGTTGCGTCGGTCCG"},{"entry":"GGGCCGATGTTCTGGAAACCACTCTTCTGGAAAAGTCTCTTTCTCATCTCCTTATGCTGGAG"},{"entry":"AACGCCAGGGATACGTGTATTCGCCTGGTGCAGGAAATGCGCGATCAGCAAAAAGACGATGA"},{"entry":"TGAAGGTACTCCGCCTGAATACCGTATCGCGAGCATGCTAAACAGCTGTTCCGCGCAGATAA"},{"entry":"GCAGCCTGATCAACACCATTTACAGCATCCGGAATAACTATCGAAAAGAAAGCCGGGAGGCG"},{"entry":"GAAAAGCACGCTTTATCTATGGGGCAAGCTGGCATTGTTAAGCTGGCATACGAACGAAAGCG"},{"entry":"TGAAAATAACTGGTCAGTGCTGGAAGCGGCTGAATTCATCGAGGCGCATGGAGGAAAAGTGC"},{"entry":"CGCCCCTGATGCTGGAGCAAATCAAAGCCGATCTGCGTGCTCCTAAGACCAATACCGATGAT"},{"entry":"GAGGAAAACCAAACAGCATCTGGCGCTCCATCACTTGAAGATCTGGATAAAATCGCGCGAGA"},{"entry":"ACGGGCCGCCAGCCGCCGCGCTGATGCCGCATTGTGGATTGAGCATCGTAGAGAAGAAATTG"},{"entry":"CCGATATCGTCGATACAGGTGGTTATGGTGATGTCGATGCGGAAGGCATATCAAACGAAGCA"},{"entry":"TGGCTTGAACAGGATCTGGACGAAGACGAGGAGGAAGACGAAGAAGTTACCCGCAAACTGTA"},{"entry":"CGGGGATGATGATTAA"},{"entry":{}},{"entry":"terS\u2003gene,\u2003lower\u2003case\u2003characters\u2003signify\u2003deleted\u2003sequence"},{"entry":["SEQ\u2003ID\u2003NO:\u20038",{}]},{"entry":["ATGAACGAAAAACAAAAGAGATTCGCAGATGAATATATAATGAATGGATGTAATGGTAAAAA",{}]},{"entry":"AGCAGCAATTTCAGCAggttatagtaagaaaacagcagagtctttagcaagtcgattgttaa"},{"entry":"gaaatgttaatgtttcggaatatattaaagaacgattagaacagatacaagaagagcgttta"},{"entry":"atgagcattacagaagctttagcgttatctgcttctattgctagaggagaacctcaagaggc"},{"entry":"ttacagtaagaaatatgaccatttaaacgatgaagtggaaaaagaggttacttacacaatca"},{"entry":"caccaacttttgaagagcgtcagagatctattgaccacatactaaaagttcatggtgcgtat"},{"entry":"atcgacaaaaaagaaattactcagaagaatattgagattaatattAGATCTATTGACCACAT"},{"entry":"ACTAAAAGTICATGGTGCGTATATCGACAAAAAAGAAATTACTCAGAAGAATATTGAGATTA"},{"entry":"ATATTGGTGAGTACGATGACGAAAGTTAA"},{"entry":{}},{"entry":"Sequence\u2003containing\u2003native\u2003terS\u2003gene"},{"entry":["SEQ\u2003ID\u2003NO:\u20039",{}]},{"entry":["AATTGGCAGTAAAGTGGCAGTTTTTGATACCTAAAATGAGATATTATGATAGTGTAGGATAT",{}]},{"entry":"TGACTATCTTACTGCGTTTCCCTTATCGCAATTAGGAATAAAGGATCTATGTGGGTTGGCTG"},{"entry":"ATTATAGCCAATCCTTTTTTAATTTTAAAAAGCGTATAGCGCGAGAGTTGGTGGTAAATGAA"},{"entry":"ATGAACGAAAAACAAAAGAGATTCGCAGATGAATATATAATGAATGGATGTAATGGTAAAAA"},{"entry":"AGCAGCAATTTCAGCAGGTTATAGTAAGAAAACAGCAGAGTCTTTAGCAAGTCGATTGTTAA"},{"entry":"GAAATGTTAATGTTTCGGAATATATTAAAGAACGATTAGAACAGATACAAGAAGAGCGTTTA"},{"entry":"ATGAGCATTACAGAAGCTTTAGCGTTATCTGCTTCTATTGCTAGAGGAGAACCTCAAGAGGC"},{"entry":"TTACAGTAAGAAATATGACCATTTAAACGATGAAGTGGAAAAAGAGGTTACTTACACAATCA"},{"entry":"CACCAACTTTTGAAGAGCGTCAGAGATCTATTGACCACATACTAAAAGTTCATGGTGCGTAT"},{"entry":"ATCGACAAAAAAGAAATTACTCAGAAGAATATTGAGATTAATATTGGTGAGTACGATGACGA"},{"entry":"AAGTTAAATTAAACTTTAACAAACCATCTAATGTTTTCAACAG"},{"entry":{}},{"entry":"SaPibov2\u2003integrase\u2003gene"},{"entry":["SEQ\u2003ID\u2003NO:\u200310",{}]},{"entry":["TCATAAATATTTAACTATTTCTTTCTGTGTACTAGGGTACAAATGACCGTATCGGTTATATA",{}]},{"entry":"CTTCATTACTATCAGCATGGCCTAAACGCTGTGCTATTACCATGATACTTGCGCCATGATTG"},{"entry":"ACTAGCATAGACGCATGGCTATGTCTTAACTCATGAATTACAATTCTAGGGAATGTCTGACC"},{"entry":"GTCTGGTAGTTGTTCATCTAATACTTTTAATGCAGCGGTAAACCAACGATCTATAGTTGATT"},{"entry":"CACTATAAGCTTTGAAGAATGTACCGAATAATACATAATCATCTTTATATACATTGTTTTCT"},{"entry":"TTGTACCATTTTAAATATTCTTTGATATCATTCATCATGTGAACAGGTAAGTATATATCACG"},{"entry":"TATTGCTGCTTTTGTTTTAGGGGCTGTCACTTCACCGTGATAGTCTGTTTTGTTAATATGTA"},{"entry":"TGAAATCATCATCATAGTTAATATCACGCCATGTGAGGGCTCTAATTTCGCCCTTACGTGCA"},{"entry":"CCAGAGTAAAACAGTAGCTTAAAGAATAACTTTTGTTGTTGTGTAGCTAAAGCCTCATAGAA"},{"entry":"TTGATTGAATTGTTCTAATGTCCAATAGTTCAAACGCTTATTTGATTCTATTTCAAAGTTAC"},{"entry":"CTACTAGAGAGGCTACATTTTGCTTTAGATCATGAAACTTCATAGCATGGTTAAGTAACGAT"},{"entry":"ACTAAGAACACGTGCATTTTCTTTAGGTACTCTCCAGAGTGTCCCTCTTTTAACTTCGTATT"},{"entry":"CTGAAACTTCATAATATCTTGTGTAGTCATATTAAACACGTCCATAGACTTAAAATAGGGTA"},{"entry":"GCAAATGGTTGTTTGTATGTGTCTTTAATGCTTTCACACTAGATGACTTACGACGTGCAGAA"},{"entry":"TACCACTCTATATACTCATCTACGAGCTTATCAAAGGGCAGTTTGTTTATCTGTCCTACACC"},{"entry":"CTCTAACTCGTCCATAATTTCATTACATTTCTTCAATGCCTCTTTACGCTGTTTAAAGCCAC"},{"entry":"TCTTTTTTATTTCTTTACGTTGATTAAATTTATCATAGTATTTTATACGAAAATAGTATGTA"},{"entry":"CCACGTTTAGCGTCTTTATATATGTTGTGGGATAGGTTTAAGTTGTGTTCTATGGGAATCAC"},{"entry":{}},{"entry":"pGWP10001\u2003full\u2003sequence"},{"entry":["SEQ\u2003ID\u2003NO:\u200311",{}]},{"entry":["CTCGGGCCGTCTCTTGGGCTTGATCGGCCTTCTTGCGCATCTCACGCGCTCCTGCGGCGGCC",{}]},{"entry":"TGTAGGGCAGGCTCATACCCCTGCCGAACCGCTTTTGTCAGCCGGTCGGCCACGGCTTCCGG"},{"entry":"CGTCTCAACGCGCTTTGAGATTCCCAGCTTTTCGGCCAATCCCTGCGGTGCATAGGCGCGTG"},{"entry":"GCTCGACCGCTTGCGGGCTGATGGTGACGTGGCCCACTGGTGGCCGCTCCAGGGCCTCGTAG"},{"entry":"AACGCCTGAATGCGCGTGTGACGTGCCTTGCTGCCCTCGATGCCCCGTTGCAGCCCTAGATC"},{"entry":"GGCCACAGCGGCCGCAAACGTGGTCTGGTCGCGGGTCATCTGCGCTTTGTTGCCGATGAACT"},{"entry":"CCTTGGCCGACAGCCTGCCGTCCTGCGTCAGCGGCACCACGAACGCGGTCATGTGCGGGCTG"},{"entry":"GTTTCGTCACGGTGGATGCTGGCCGTCACGATGCGATCCGCCCCGTACTTGTCCGCCAGCCA"},{"entry":"CTTGTGCGCCTTCTCGAAGAACGCCGCCTGCTGTTCTTGGCTGGCCGACTTCCACCATTCCG"},{"entry":"GGCTGGCCGTCATGACGTACTCGACCGCCAACACAGCGTCCTTGCGCCGCTTCTCTGGCAGC"},{"entry":"AACTCGCGCAGTCGGCCCATCGCTTCATCGGTGCTGCTGGCCGCCCAGTGCTCGTTCTCTGG"},{"entry":"CGTCCTGCTGGCGTCAGCGTTGGGCGTCTCGCGCTCGCGGTAGGCGTGCTTGAGACTGGCCG"},{"entry":"CCACGTTGCCCATTTTCGCCAGCTTCTTGCATCGCATGATCGCGTATGCCGCCATGCCTGCC"},{"entry":"CCTCCCTTTTGGTGTCCAACCGGCTCGACGGGGGCAGCGCAAGGCGGTGCCTCCGGCGGGCC"},{"entry":"ACTCAATGCTTGAGTATACTCACTAGACTTTGCTTCGCAAAGTCGTGACCGCCTACGGCGGC"},{"entry":"TGCGGCGCCCTACGGGCTTGCTCTCCGGGCTTCGCCCTGCGCGGTCGCTGCGCTCCCTTGCC"},{"entry":"AGCCCGTGGATATGTGGACGATGGCCGCGAGCGGCCACCGGCTGGCTCGCTTCGCTCGGCCC"},{"entry":"GTGGACAACCCTGCTGGACAAGCTGATGGACAGGCTGCGCCTGCCCACGAGCTTGACCACAG"},{"entry":"GGATTGCCCACCGGCTACCACTATAGGGCGAATTGGCGGAAGGCCGTCAAGGCCGCATTTGG"},{"entry":"GCCCGGCGCGCCGGATCCGCTAGCTCTAGACCTCTAGACCAGCCAGGACAGAAATGCCTCGA"},{"entry":"CTTCGCTGCTGCCCAAGGTTGCCGGGTGACGCACACCGTGGAAACGGATGAAGGCACGAACC"},{"entry":"CAGTGGACATAAGCCTGTTCGGTTCGTAAGCTGTAATGCAAGTAGCGTGCCCGTTCCGGGCC"},{"entry":"TTTTGACATGTGACTTTCGTTACCCTCGCGTCAAAAAGAGTTTTTACGAAAGGAAGCATAAG"},{"entry":"TGACCTGGGACGATCACAAGAAGAATTTTGCTCGCCTGGCGCGAGATGGTGGTTACACCATC"},{"entry":"GCACAGTATGCCGCCGAGTTTAATCTTAACCCTAATACCGCACGTCGTTATCTCCGTGCCTT"},{"entry":"CAAAGAAGACACCAGGACTACGGACAGCCGCAAGCCAAATAAGCCAGTCAGGAAGCCACTAA"},{"entry":"AAAGCATGATCATTGATCACTCTAATGATCAACATGCAGGTGATCACATTGCGGCTGAAATA"},{"entry":"GCGGAAAAACAAAGAGTTAATGCCGTTGTCAGTGCCGCAGTCGAGAATGCGAAGCGCCAAAA"},{"entry":"TAAGCGCATAAATGATCGTTCAGATGATCATGACGTGATCACCCGCGCCCACCGGACCTTAC"},{"entry":"GTGATCGCCTGGAACGCGACACCCTGGATGATGATGGTGAACGCTTTGAATTCGAAGTTGGC"},{"entry":"GATTACCTGATAGATAACGTTGAAGCGCGGAAGGCCGCGCGCGCTATGTTGCGTCGGTCCGG"},{"entry":"GGCCGATGTTCTGGAAACCACTCTTCTGGAAAAGTCTCTTTCTCATCTCCTTATGCTGGAGA"},{"entry":"ACGCCAGGGATACGTGTATTCGCCTGGTGCAGGAAATGCGCGATCAGCAAAAAGACGATGAT"},{"entry":"GAAGGTACTCCGCCTGAATACCGTATCGCGAGCATGCTAAACAGCTGTTCCGCGCAGATAAG"},{"entry":"CAGCCTGATCAACACCATTTACAGCATCCGGAATAACTATCGAAAAGAAAGCCGGGAGGCGG"},{"entry":"AAAAGCACGCTTTATCTATGGGGCAAGCTGGCATTGTTAAGCTGGCATACGAACGAAAGCGT"},{"entry":"GAAAATAACTGGTCAGTGCTGGAAGCGGCTGAATTCATCGAGGCGCATGGAGGAAAAGTGCC"},{"entry":"GCCCCTGATGCTGGAGCAAATCAAAGCCGATCTGCGTGCTCCTAAGACCAATACCGATGATG"},{"entry":"AGGAAAACCAAACAGCATCTGGCGCTCCATCACTTGAAGATCTGGATAAAATCGCGCGAGAA"},{"entry":"CGGGCCGCCAGCCGCCGCGCTGATGCCGCATTGTGGATTGAGCATCGTAGAGAAGAAATTGC"},{"entry":"CGATATCGTCGATACAGGTGGTTATGGTGATGTCGATGCGGAAGGCATATCAAACGAAGCAT"},{"entry":"GGCTTGAACAGGATCTGGACGAAGACGAGGAGGAAGACGAAGAAGTTACCCGCAAACTGTAC"},{"entry":"GGGGATGATGATTAATTAAAAAAACCCGCCCCGGCGGGTTTTTTTATCTAGAGCTAGCGGAT"},{"entry":"CCGGCGCGCCGGGCCCTTCTGGGCCTCATGGGCCTTCCGCTCACTGCCCGCTTTCCAGCACT"},{"entry":"ATAGGGCGAATTGGCGGAAGGCCGTCAAGGCCGCATTTGGGCCCGGCGCGCCGGATCCGCTA"},{"entry":"GCTCTAGACTGGCAGGTTTCTGAGCAGATCGTCCAACCCGATCTGGATCGGGTCAGAAAAAT"},{"entry":"TTGCTCTAATAAATTTCGTTTTCTAAGTGCAAAGAATCACCATTTCGAGCTGGTGATTGAAG"},{"entry":"GTTGATGCAAATTTGGAGAAAAAATGCAACAAACATTCAATGCGGATATGAATATATCAAAC"},{"entry":"CTTCATCAAAATGTCGATCCTTCAACCACTCTGCCCGTTATTTGTGGTGTTGAAATTACGAC"},{"entry":"CGACCGCGCTGGCCGTTACAACCTTAATGCTCTACACAGAGCGAGCGGACTCGGTGCCCATA"},{"entry":"AAGCGCCAGCTCAATGGCTAAGAACGCTGTCAGCTAAACAGCTCATCGAAGAGCTTGAAAAA"},{"entry":"GAAACTATGCAGAATTGCATAGTTTCGTTCACAAGCAATGGAAGCAGGATTTCTTTCACGAC"},{"entry":"TCGTATAACCGGCAAAGGTCAGCAGTGGCTGATGAAGCGATTGCTTGATGCTGGTGTGCTGG"},{"entry":"TACCTGTCGCGGCAACGCGCTAACAGACGTAGTAAGAACCACCAGCATTGTAATGCTGGCTA"},{"entry":"AAGTCACTTTCCTGAGCTGTATAACGATGAGCGATTTTACTTTTTCTGGCTATGAATTGGCC"},{"entry":"TGCTTTGTAACACACTCCGGTCTATCCCGTAGCGCCGGGCATATCCTGTCGCAATGTGCAAA"},{"entry":"TCTCGCGGCAACAACCAGTGAATACTTCATTCACAAGCCTCACCGCCTGATCGCGGCAGAAA"},{"entry":"CTGGTTATAGCCAATCAACCGTCGTTCGTGCATTCCGTGAAGCTGTAAACAAAGGAATTCTG"},{"entry":"TCTGTAGAGATTGTTATCGGCGATCACCGTGAACGTCGCGCTAACCTGTACCGGTTTACACC"},{"entry":"ATCCTTTTTGGCCTTCGCACAACAAGCCAAAAATGCGCTGATAGAAAGCAAATTAAAGATCT"},{"entry":"CTTCAGCGGCAACCAAGGTTAAAGCTGTTCTCGCTAAGACATTGGCTTTATTTAATTTTTTA"},{"entry":"TCCACACCCCCATGTCAAAATGATACCCCCTCCCCCTGTCAGGATGACGTGGCAATAAAGAA"},{"entry":"TAAGAAGTCACAAGTTAAAAAAACAAAAAGATCAGTTTCCGGCGGTGCCGGAACAACCAGCC"},{"entry":"TCAAAAAATTGACTTCATGGATCGCTAAGGCAAAAGCAAAGGCTGACAATCTGCGGTTATCC"},{"entry":"AAAAAACGCACTCAAAAACATGAGTTCAAGCAGAAAGTAGAGGCGGCTGCGCGGAAATATGC"},{"entry":"TTACCTGAAGAACAAGCGTTCGCCTGATATTGGCGGGATATCAAACTTCGATAACCTACCGC"},{"entry":"ATTGCATGACGGTAAACGAAGCTCTTAATGCGGTTTTAGCCAAAAATAAAGATAACGAACAA"},{"entry":"TGGGGTATACCGGCAGGATTCAGAGGGTAATGAATTGCTCTAATTATAACCATGCATACTTT"},{"entry":"CAACACCTCTAGTTTGCCATGAGGCAAACTCATAGGTGTCCTGGTAAGAGGACACTGTTGCC"},{"entry":"AAAACTGGACGCCCCATTATTGCAATTAATAAACAACTAACGGACAATTCTACCTAACAATA"},{"entry":"AGTGGCTTAAAAAAACCCGCCCCGGCGGGTTTTTTTATCTAGAGCTAGCGGATCCGGCGCGC"},{"entry":"CGGGCCCTTCTGGGCCTCATGGGCCTTCCGCTCACTGCCCGCTTTCCAGCCAGCCTTCGACC"},{"entry":"ACATACCCACCGGCTCCAACTGCGCGGCCTGCGGCCTTGCCCCATCAATTTTTTTAATTTTC"},{"entry":"TCTGGGGAAAAGCCTCCGGCCTGCGGCCTGCGCGCTTCGCTTGCCGGTTGGACACCAAGTGG"},{"entry":"AAGGCGGGTCAAGGCTCGCGCAGCGACCGCGCAGCGGCTTGGCCTTGACGCGCCTGGAACGA"},{"entry":"CCCAAGCCTATGCGAGTGGGGGCAGTCGAAGGCGAAGCCCGCCCGCCTGCCCCCCGAGACCT"},{"entry":"GCAGGGGGGGGGGGGCGCTGAGGTCTGCCTCGTGAAGAAGGTGTTGCTGACTCATACCAGGC"},{"entry":"CTGAATCGCCCCATCATCCAGCCAGAAAGTGAGGGAGCCACGGTTGATGAGAGCTTTGTTGT"},{"entry":"AGGTGGACCAGTTGGTGATTTTGAACTTTTGCTTTGCCACGGAACGGTCTGCGTTGTCGGGA"},{"entry":"AGATGCGTGATCTGATCCTTCAACTCAGCAAAAGTTCGATTTATTCAACAAAGCCGCCGTCC"},{"entry":"CGTCAAGTCAGCGTAATGCTCTGCCAGTGTTACAACCAATTAACCAATTCTGATTAGAAAAA"},{"entry":"CTCATCGAGCATCAAATGAAACTGCAATTTATTCATATCAGGATTATCAATACCATATTTTT"},{"entry":"GAAAAAGCCGTTTCTGTAATGAAGGAGAAAACTCACCGAGGCAGTTCCATAGGATGGCAAGA"},{"entry":"TCCTGGTATCGGTCTGCGATTCCGACTCGTCCAACATCAATACAACCTATTAATTTCCCCTC"},{"entry":"GTCAAAAATAAGGTTATCAAGTGAGAAATCACCATGAGTGACGACTGAATCCGGTGAGAATG"},{"entry":"GCAAAAGCTTATGCATTTCTTTCCAGACTTGTTCAACAGGCCAGCCATTACGCTCGTCATCA"},{"entry":"AAATCACTCGCATCAACCAAACCGTTATTCATTCGTGATTGCGCCTGAGCGAGACGAAATAC"},{"entry":"GCGATCGCTGTTAAAAGGACAATTACAAACAGGAATCGAATGCAACCGGCGCAGGAACACTG"},{"entry":"CCAGCGCATCAACAATATTTTCACCTGAATCAGGATATTCTTCTAATACCTGGAATGCTGTT"},{"entry":"TTCCCGGGGATCGCAGTGGTGAGTAACCATGCATCATCAGGAGTACGGATAAAATGCTTGAT"},{"entry":"GGTCGGAAGAGGCATAAATTCCGTCAGCCAGTTTAGTCTGACCATCTCATCTGTAACATCAT"},{"entry":"TGGCAACGCTACCTTTGCCATGTTTCAGAAACAACTCTGGCGCATCGGGCTTCCCATACAAT"},{"entry":"CGATAGATTGTCGCACCTGATTGCCCGACATTATCGCGAGCCCATTTATACCCATATAAATC"},{"entry":"AGCATCCATGTTGGAATTTAATCGCGGCCTCGAGCAAGACGTTTCCCGTTGAATATGGCTCA"},{"entry":"TAACACCCCTTGTATTACTGTTTATGTAAGCAGACAGTTTTATTGTTCATGATGATATATTT"},{"entry":"TTATCTTGTGCAATGTAACATCAGAGATTTTGAGACACAACGTGGCTTTCCCCCCCCCCCCT"},{"entry":"GCAGGTCCCGAGCCTCACGGCGGCGAGTGCGGGGGTTCCAAGGGGGCAGCGCCACCTTGGGC"},{"entry":"AAGGCCGAAGGCCGCGCAGTCGATCAACAAGCCCCGGAGGGGCCACTTTTTGCCGGAGGGGG"},{"entry":"AGCCGCGCCGAAGGCGTGGGGGAACCCCGCAGGGGTGCCCTTCTTTGGGCACCAAAGAACTA"},{"entry":"GATATAGGGCGAAATGCGAAAGACTTAAAAATCAACAACTTAAAAAAGGGGGGTACGCAACA"},{"entry":"GCTCATTGCGGCACCCCCCGCAATAGCTCATTGCGTAGGTTAAAGAAAATCTGTAATTGACT"},{"entry":"GCCACTTTTACGCAACGCATAATTGTTGTCGCGCTGCCGAAAAGTTGCAGCTGATTGCGCAT"},{"entry":"GGTGCCGCAACCGTGCGGCACCCCTACCGCATGGAGATAAGCATGGCCACGCAGTCCAGAGA"},{"entry":"AATCGGCATTCAAGCCAAGAACAAGCCCGGTCACTGGGTGCAAACGGAACGCAAAGCGCATG"},{"entry":"AGGCGTGGGCCGGGCTTATTGCGAGGAAACCCACGGCGGCAATGCTGCTGCATCACCTCGTG"},{"entry":"GCGCAGATGGGCCACCAGAACGCCGTGGTGGTCAGCCAGAAGACACTTTCCAAGCTCATCGG"},{"entry":"ACGTTCTTTGCGGACGGTCCAATACGCAGTCAAGGACTTGGTGGCCGAGCGCTGGATCTCCG"},{"entry":"TCGTGAAGCTCAACGGCCCCGGCACCGTGTCGGCCTACGTGGTCAATGACCGCGTGGCGTGG"},{"entry":"GGCCAGCCCCGCGACCAGTTGCGCCTGTCGGTGTTCAGTGCCGCCGTGGTGGTTGATCACGA"},{"entry":"CGACCAGGACGAATCGCTGTTGGGGCATGGCGACCTGCGCCGCATCCCGACCCTGTATCCGG"},{"entry":"GCGAGCAGCAACTACCGACCGGCCCCGGCGAGGAGCCGCCCAGCCAGCCCGGCATTCCGGGC"},{"entry":"ATGGAACCAGACCTGCCAGCCTTGACCGAAACGGAGGAATGGGAACGGCGCGGGCAGCAGCG"},{"entry":"CCTGCCGATGCCCGATGAGCCGTGTTTTCTGGACGATGGCGAGCCGTTGGAGCCGCCGACAC"},{"entry":"GGGTCACGCTGCCGCGCCGGTAGCACTTGGGTTGCGCAGCAACCCGTAAGTGCGCTGTTCCA"},{"entry":"GACTATCGGCTGTAGCCGCCTCGCCGCCCTATACCTTGTCTGCCTCCCCGCGTTGCGTCGCG"},{"entry":"GTGCATGGAGCCGGGCCACCTCGACCTGAATGGAAGCCGGCGGCACCTCGCTAACGGATTCA"},{"entry":"CCGTTTTTATCAGGCTCTGGGAGGCAGAATAAATGATCATATCGTCAATTATTACCTCCACG"},{"entry":"GGGAGAGCCTGAGCAAACTGGCCTCAGGCATTTGAGAAGCACACGGTCACACTGCTTCCGGT"},{"entry":"AGTCAATAAACCGGTAAACCAGCAATAGACATAAGCGGCTATTTAACGACCCTGCCCTGAAC"},{"entry":"CGACGACCGGGTCGAATTTGCTTTCGAATTTCTGCCATTCATCCGCTTATTATACTTATTCA"},{"entry":"GGCGTAGCACCAGGCGTTTAAGGGCACCAATAACTGCCTTAAAAAAATTACGCCCCGCCCTG"},{"entry":"CCACTCATCGCACTCGTCAGGTGGCACTTTTCGGGGAAATGTGCGCGGAACCCCTATTTGTT"},{"entry":"TATTTTTCTAAATACATTCAAATATGTATCCGCTCATGAGACAATAACCCTGATAAATGCTT"},{"entry":"CAATAATATTGAAAAAGGAAGAGTATGAAGTTTGGAAATATTTGTTTTTCGTATCAACCACC"},{"entry":"AGGTGAAACTCATAAGCTAAGTAATGGATCGCTTTGTTCGGCTTGGTATCGCCTCAGAAGAG"},{"entry":"TAGGGTTTGATACATATTGGACCTTAGAACATCATTTTACAGAGTTTGGTCTTACGGGAAAT"},{"entry":"TTATTTGTTGCTGCGGCTAACCTGTTAGGAAGAACTAAAACATTAAATGTTGGCACTATGGG"},{"entry":"GGTTGTTATTCCGACAGCACACCCAGTTCGACAGTTAGAAGACGTITTATTATTAGATCAAA"},{"entry":"TGTCGAAAGGTCGTTTTAATTTTGGAACCGTTCGAGGGCTATACCATAAAGATTTTCGAGTA"},{"entry":"TTTGGTGTTGATATGGAAGAGTCTCGAGCAATTACTCAAAATTTCTACCAGATGATAATGGA"},{"entry":"AAGCTTACAGACAGGAACCATTAGCTCTGATAGTGATTACATTCAATTTCCTAAGGTTGATG"},{"entry":"TATATCCCAAAGTGTACTCAAAAAATGTACCAACCTGTATGACTGCTGAGTCCGCAAGTACG"},{"entry":"ACAGAATGGCTAGCAATACAAGGGCTACCAATGGTTCTTAGTTGGATTATTGGTACTAATGA"},{"entry":"AAAAAAAGCACAGATGGAACTCTATAATGAAATTGCGACAGAATATGGTCATGATATATCTA"},{"entry":"AAATAGATCATTGTATGACTTATATTTGTTCTGTTGATGATGATGCACAAAAGGCGCAAGAT"},{"entry":"GTTTGTCGGGAGTTTCTGAAAAATTGGTATGACTCATATGTAAATGCGACCAATATCTTTAA"},{"entry":"TGATAGCAATCAAACTCGTGGTTATGATTATCATAAAGGTCAATGGCGTGATTTTGTTTTAC"},{"entry":"AAGGACATACAAACACCAATCGACGTGTTGATTATAGCAATGGTATTAACCCTGTAGGCACT"},{"entry":"CCTGAGCAGTGTATTGAAATCATTCAACGTGATATTGATGCAACGGGTATTACAAACATTAC"},{"entry":"ATGCGGATTTGAAGCTAATGGAACTGAAGATGAAATAATTGCTTCCATGCGACGCTTTATGA"},{"entry":"CACAAGTCGCTCCTTTCTTAAAAGAACCTAAATAAATTACTTATTTGATACTAGAGATAATA"},{"entry":"AGGAACAAGTTATGAAATTTGGATTATTTTTTCTAAACTTTCAGAAAGATGGAATAACATCT"},{"entry":"GANGAAACGTTGGATAATATGGTAAAGACTGTCACGTTAATTGATTCAACTAAATATCATTT"},{"entry":"TAATACTGCCTTTGTTAATGAACATCACTTTTCAAAAAATGGTATTGTTGGAGCACCTATTA"},{"entry":"CCGCAGCTGGTTTTTTATTAGGGTTAACAAATAAATTACATATTGGTTCATTAAATCAAGTA"},{"entry":"ATTACCACCCATCACCCTGTACGTGTAGCAGAAGAAGCCAGTTTATTAGATCAAATGTCAGA"},{"entry":"GGGACGCTTCATTCTTGGTTTTAGTGACTGCGAAAGTGATTTCGAAATGGAATTTTTTAGAC"},{"entry":"GTCATATCTCATCAAGGCAACAACAATTTGAAGCATGCTATGAAATAATTAATGACGCATTA"},{"entry":"ACTACAGGTTATTGTCATCCCCAAAACGACTTTTATGATTTTCCAAAGGTTTCAATTAATCC"},{"entry":"ACACTGTTACAGTGAGAATGGACCTAAGCAATATGTATCCGCTACATCAAAAGAAGTCGTCA"},{"entry":"TGTGGGCAGCGAAAAAGGCACTGCCTTTAACATTTAAGTGGGAGGATAATTTAGAAACCAAA"},{"entry":"GAACGCTATGCAATTCTATATAATAAAACAGCACAACAATATGGTATTGATATTTCGGATGT"},{"entry":"TGATCATCAATTAACTGTAATTGCGAACTTAAATGCTGATAGAAGTACGGCTCAAGAAGAAG"},{"entry":"TGAGAGAATACTTAAAAGACTATATCACTGAAACTTACCCTCAAATGGACAGAGATGAAAAA"},{"entry":"ATTAACTGCATTATTGAAGAGAATGCAGTTGGGTCTCATGATGACTATTATGAATCGACAAA"},{"entry":"ATTAGCAGTGGAAAAAACAGGGTCTAAAAATATTTTATTATCCTTTGAATCAATGTCCGATA"},{"entry":"TTAAAGATGTAAAAGATATTATTGATATGTTGAACCAAAAAATCGAAATGAATTTACCATAA"},{"entry":"AGTAGTACTGTTGTAATTCATTAAGCATTCTGCCGACATGGAAGCCATCACAGACGGCATGA"},{"entry":"TGAACCTGAATCGCCAGCGGCATCAGCACCTTGTCGCCTTGCGTATAATATTTGCCCATGGT"},{"entry":"GAAAACGGGGGCGAAGAAGTTGTCCATATTGGCCACGTTTAAATCAAAACTGGTGAAACTCA"},{"entry":"CCCAGGGATTGGCTGAGACGAAAAACATATTCTCAATAAACCCTTTAGGGAAATAGGCCAGG"},{"entry":"TTTTCACCGTAACACGCCACATCTTGCGAATATATGTGTAGAAACTGCCGGAAATCGTCGTG"},{"entry":"GTATTCACTCCAGAGCGATGAAAACGTTTCAGTTTGCTCATGGAAAACGGTGTAACAAGGGT"},{"entry":"GAACACTATCCCATATCACCAGCTCACCGTCTTTCATTGCCATACGGAATTCCGGATGAGCA"},{"entry":"TTCATCAGGCGGGCAAGAATGTGAATAAAGGCCGGATAAAACTTGTGCTTATTTTTCTTTAC"},{"entry":"GGTCTTTAAAAAGGCCGTAATATCCAGCTGAACGGTCTGGTTATAGGTACATTGAGCAACTG"},{"entry":"ACTGAAATGCCTCAAAATGTTCTTTACGATGCCATTGGGATATATCAACGGTGGTATATCCA"},{"entry":"GTGATTTTTTTCTCCATTTTAGCTTCCTTAGCTCCTGAAAATCTCGATAACTCAAAAAATAC"},{"entry":"GCCCGGTAGTGATCTTATTTCATTATGGTGAAAGTTGGAACCTCTTACGTGCCGATCAACGT"},{"entry":"CTCATTTTCGCCAAAAGTTGGCCCAGGGCTTCCCGGTATCAACAGGGACACCAGGATTTATT"},{"entry":"TATTCTGCGAAGTGATCTTCCGTCACAGGTATTTATTCGAAGACGAAAGGGCCTCGTGATAC"},{"entry":"GCCTATTTTTATAGGTTAATGTCATGATAATAATGGTTTCTTAGACGTCAGGTGGCACTTTT"},{"entry":"CGGGGAAATGTGCGCGCCCGCGTTCCTGCTGGCGCTGGGCCTGTTTCTGGCGCTGGACTTCC"},{"entry":"CGCTGTTCCGTCAGCAGCTTTTCGCCCACGGCCTTGATGATCGCGGCGGCCTTGGCCTGCAT"},{"entry":"ATCCCGATTCAACGGCCCCAGGGCGTCCAGAACGGGCTTCAGGCGCTCCCGAAGGT"},{"entry":{}},{"entry":{"i":"S.\u2003aureus","sub":"clpB"}},{"entry":["SEQ\u2003ID\u2003NO:\u200312",{}]},{"entry":["GTCTAGTTAATGTGTAACGTAACATTAGCTAGATTTTTTTATTCAAAAAAATATTTACAAAT",{}]},{"entry":"ATTAGGAAATTTAAGTGTAAAAGAGTTGATAAATGATTATATTGGGACTATAATATAATTAA"},{"entry":"GGTC"},{"entry":{}},{"entry":"RN10616\u2003genomic\u2003sequence\u2003loci\u2003showing\u2003the\u2003(p80\u03b1 terS\u2003deletion"},{"entry":"and\u2003complementation.\u2003terS\u2003= Bracketed\u2003Text,\u2003Deletion\u2003="},{"entry":"Underlined,\u2003Complement\u2003= Bold"},{"entry":["SEQ\u2003ID\u2003NO:\u200313",{}]},{"entry":["ATTAGACAACAAACAAGTCATTGAAAATTCCGACTTATTATTCAAAAAGAAATTTGATAGCG",{}]},{"entry":"CAGATATACAAGCTAGGTTAAAAGTAGGCGATAAGGTAGAAGTTAAAACAATCGGTTATAGA"},{"entry":"ATACACTTTTTAAATTTATATCCGGTCTTATACGAAGTAAAGAAGGTAGATAAACAATGATT"},{"entry":"AAACAAATACTAAGACTATTATTCTTACTAGCAATGTATGAGTTAGGTAAGTATGTAACTGA"},{"entry":"GCAAGTATATATTATGATGACGGCTAATGATGATGTAGAGGTGCCGAGTGACTTCGCGAAGT"},{"entry":"TGAGCGATCAGTCAGATTTGATGAGGGCGGAGGTGACGGAGTAGATGATGTGGTTAGTCATA"},{"entry":"GCAATTATATTACTAGTCATCTTATTGTTTGGTGTGATGTTGCAAGCTGAACAGTTAAAAGG"},{"entry":"CGATGTGAAAGTTAAAGAGCGGGAGATAGAGATATTAAGAAGTAGATTGAGACATTTTGAAG"},{"entry":"ATTAAAAATATTTGTATGGAGGGTATTCATGACTAAAAAGAAATATGGATTAAAATTATCAA"},{"entry":"CAGTTCGAAAGTTAGAAGATGAGTTGTGTGATTATCCTAATTATCATAAGCAACTCGAAGAT"},{"entry":"TTAAGAAGTGAAATAATGACACCATGGATTCCAACAGATACAAATATAGGCGGGGAGTTTGT"},{"entry":"ACCGTCTAATACATCGAAAACAGAAATGGCAGTAACTAATTATCTTTGTAGTATACGAAGAG"},{"entry":"GTAAAATCCTTGAGTTTAAGAGCGCTATTGAACGTATAATCAACACATCAAGTAGGAAAGAA"},{"entry":"CGCGAATTCATTCAAGAGTATTATTTTAATAAAAAGGAATTAGTGAAAGTTTGTGATGACAT"},{"entry":"ACACATTTCTGATAGAACTGCTCATAGAATCAAAAGGAAAATCATATCTAGATTGGCGGAAG"},{"entry":"AGTTAGGGGAAGAGTGA"},{"entry":{"b":"ATGATAGTGTAGGATATTGACTATCTTACTGCGTTTCCCTTATCGCAATTAGGAATAAAGGA"}},{"entry":{"b":"TCTATGTGGGTTGGCTGATTATAGCCAATCCTTTTTTAATTTTAAAAAGCGTATAGCGCGAG"}},{"entry":{"b":["AGTTGGTGGTAAATGAA","ATGAACGAAAAACAAAAGAGATTCGCAGATGAATATATAATGA"]}},{"entry":{"b":"ATGGATGTAATGGTAAAAAAGCAGCAATTTCAGCA","u":{"@attributes":{"style":"single"},"b":"GGTTATAGTAAGAAAACAGCAGAGTCT"}}},{"entry":{"u":{"@attributes":{"style":"single"},"b":"TTAGCAAGTCGATTGTTAAGAAATGTTAATGTTTCGGAATATATTAAAGAACGATTAGAACA"}}},{"entry":{"u":{"@attributes":{"style":"single"},"b":"GATACAAGAAGAGCGTTTAATGAGCATTACAGAAGCTTTAGCGTTATCTGCTTCTATTGCTA"}}},{"entry":{"u":{"@attributes":{"style":"single"},"b":"GAGGAGAACCTCAAGAGGCTTACAGTAAGAAATATGACCATTTAAACGATGAAGTGGAAAAA"}}},{"entry":{"u":{"@attributes":{"style":"single"},"b":"GAGGTTACTTACACAATCACACCAACTTTTGAAGAGCGTCAGAGATCTATTGACCACATACT"}}},{"entry":{"u":{"@attributes":{"style":"single"},"b":"AAAAGTTCATGGTGCGTATATCGACAAAAAAGAAATTACTCAGAAGAATATTGAGATTAATA"}}},{"entry":{"u":{"@attributes":{"style":"single"},"b":"TT"},"b":["GGTGAGTACGATGACGAAAGTTAA","ATTAAACTTTAACAAACCATCTAATGTTTTCAAC"]}},{"entry":{"b":"AG"}},{"entry":"TTCGAGTGGTAAGTCTCACGGCGTTATACAAAAAGTTGTACTTAAAGCATTGCAAGACTGGA"},{"entry":"AATATCCTAGGCGTATACTATGGCTTAGAAAAGTCCAATCAACAATTAAAGATAGTTTATTC"},{"entry":"GAAGATGTCAAAGATTGTTTGATAAACTTCGGTATTTGGGACATGTGCCTTTGGAATAAGAC"},{"entry":"TGATAACAAAGTTGAATTGCCAAACGGCGCAGTTTTTTTGTTTAAAGGATTAGATAACCCAG"},{"entry":"AGAAAATAAAGTCGATAAAAGGCATATCAGACATAGTCATGGAAGAAGCGTCTGAATTCACA"},{"entry":"CTAAATGATTACACGCAATTAACGTTGCGTTTGAGGGAGCGTAAACACGTGAATAAGCAAAT"},{"entry":"ATTTTTGATGTTTAACCCAGTATCTAAACTGAATTGGGTTTATAAGTATTTCTTTGAACATG"},{"entry":"GTGAACCAATGGAAAATGTCATGATTAGACAATCTAGTTATCGAGATAATAAGTTTCTTGAT"},{"entry":"GAAATGACACGACAAAACTTAGAGTTGTTAGCAAATCGTAATCCAGCATATTACAAAATTTA"},{"entry":"TGCGTTAGGTGAATTTTCTACACTAGACAAATTGGTTTTCCCTAAGTATGAAAAACGTTTAA"},{"entry":"TAAATAAAGATGAGTTAAGACATTTACCTTCTTATTTTGGATTGGACTTTGGCTACGTTAAT"},{"entry":"GATCCTAGTGCTTTTATACATTCTAAAATAGATGTAAAGAAAAAGAAGTTATACATCATTGA"},{"entry":"AGAGTATGTTAAACAAGGTATGCTGAATGATGAAATAGCTAATGTCATAAAGCAACTTGGTT"},{"entry":"ATGCTAAAGAAGAAATTACAGCAGATAGTGCAGAACAAAAAAGTATAGCTGAATTAAGGAAT"},{"entry":"CTAGGGCTTAAAAGGATTTTACCAACCAAAAAAGGGAAGGGCTCGGTTGTACAAGGGTTACA"},{"entry":"ATTCTTAATGCAATTTGAAATCATTGTTGATGAACGTTGTTTCAAGACTATTGAAGAGTTTG"},{"entry":"ACAACTACACATGGCAAAAGGACAAAGATACAGGTGAATATACCAATGAACCAGTAGATACA"},{"entry":"TACAATCATTGTATCGATTCGTTGCGTTATTCAGTGGAACGATTC"},{"entry":{}},{"entry":{}},{"entry":"pGW80A0001\u2003Full\u2003Sequence"},{"entry":["SEQ\u2003ID\u2003NO:\u200314",{}]},{"entry":["GGCGCCATGGTTAAGGGCCCTTTGCGGAAAGAGTTAGTAAGTTAACAGAAGACGAACCAAAA",{}]},{"entry":"CTAAATGGTTTAGCAGGAAACTTAGATAAAAAAATGAATCCAGAATTATATTCAGAACAGGA"},{"entry":"ACAGCAACAAGAACAACAAAAGAATCAAAAACGAGATAGAGGTATGCACTTATAGAACATGC"},{"entry":"ATTTATGCCGAGAAAACTTATTGGTTGGAATGGGCTATGTGTTAGCTAACTTGTTAGCGAGT"},{"entry":"TGGTTGGACTTGAATTGGGATTAATCCCAAGAAAGTACCAACTCAACAACACATAAAGCCCT"},{"entry":"GTAGGTTCCGACCAATAAGGAAATTGGAATAAAGCAATAAAAGGAGTTGAAGAAATGAAATT"},{"entry":"CAGAGAAGCCTTTGAGAATTTTATAACAAGTAAGTATGTACTTGGTGTTTTAGTAGTCTTAA"},{"entry":"CTGTTTACCAGATAATACAAATGCTTAAATAAAAAAAGACTTGATCTGATTAGACCAAATCT"},{"entry":"TTTGATAGTGTTATATTAATAACAAAATAAAAAGGAGTCGCTCACGCCCTACCAAAGTTTGT"},{"entry":"GAACGACATCATTCAAAGAAAAAAACACTGAGTTGTTTTTATAATCTTGTATATTTAGATAT"},{"entry":"TAAACGATATTTAAATATACATCAAGATATATATTTGGGTGAGCGATTACTTAAACGAAATT"},{"entry":"GAGATTAAGGAGTCGATTTTTTATGTATAAAAACAATCATGCAAATCATTCAAATCATTTGG"},{"entry":"AAAATCACGATTTAGACAATTTTTCTAAAACCGGCTACTCTAATAGCCGGTTGGACGCACAT"},{"entry":"ACTGTGTGCATATCTGATCCAAAATTAAGTTTTGATGCAATGACGATCGTTGGAAATCTCAA"},{"entry":"CCGAGACAACGCTCAGGCCCTTTCTAAATTTATGAGTGTAGAGCCCCAAATAAGACTTTGGG"},{"entry":"ATATTCTTCAAACAAAGTTTAAAGCTAAAGCACTTCAAGAAAAAGTTTATATTGAATATGAC"},{"entry":"AAAGTGAAAGCAGATAGTTGGGATAGACGTAATATGCGTATTGAATTTAATCCAAACAAACT"},{"entry":"TACACGAGATGAAATGATTTGGTTAAAACAAAATATAATAAGCTACATGGAAGATGACGGTT"},{"entry":"TTACAAGATTAGATTTAGCCTTTGATTTTGAAGATGATTTGAGTGACTACTATGCAATGTCT"},{"entry":"GATAAAGCAGTTAAGAAAACTATTTTTTATGGTCGTAATGGTAAGCCAGAAACAAAATATTT"},{"entry":"TGGCGTGAGAGATAGTAATAGATTTATTAGAATTTATAATAAAAAGCAAGAACGTAAAGATA"},{"entry":"ATGCAGATGCTGAAGTTATGTCTGAACATTTATGGCGTGTAGAAATCGAACTTAAAAGAGAT"},{"entry":"ATGGTGGATTACTGGAATGATTGCTTTAGTGATTTACATATCTTGCAACCAGATTGGAAAAC"},{"entry":"TATCCAACGCACTGCGGATAGAGCAATAGTTTTTATGTTATTGAGTGATGAAGAAGAATGGG"},{"entry":"GAAAGCTTCACAGAAATTCTAGAACAAAATATAAGAATTTGATAAAAGAAATTTCGCCAGTC"},{"entry":"GATTTAACGGACTTAATGAAATCGACTTTAAAAGCGAACGAAAAACAATTGCAAAAACAAAT"},{"entry":"CGATTTTTGGCAACATGAATTTAAATTTTGGAAATAGTGTACATATTAATATTACTGAACAA"},{"entry":"AAATGATATATTTAAACTATTCTAATTTAGGAGGATTTTTTTATGAAGTGTCTATTTAAAAA"},{"entry":"TTTGGGGAATTTATATGAGGTGAAAGAATAATTTACCCCTATAAACTTTAGCCACCTCAAGT"},{"entry":"AAAGAGGTAAAATTGTTTAGTTTATATAAAAAATTTAAAGGTTTGTTTTATAGCGTTTTATT"},{"entry":"TTGGCTTTGTATTCTTTCATTTTTTAGTGTATTAAATGAAATGGTTTTAAATGTTTCTTTAC"},{"entry":"CTGATATTGCAAATCATTTTAATACTACTCCTGGAATTACAAACTGGGTAAACACTGCATAT"},{"entry":"ATGTTAACTTTTTCGATAGGAACAGCAGTATATGGAAAATTATCTGATTATATAAATATAAA"},{"entry":"AAAATTGTTAATTATTGGTATTAGTTTGAGCTGTCTTGGTTCATTGATTGCTTTTATTGGGC"},{"entry":"CCACCTAGGCAAATATGCTCTTACGTGCTATTATTTAAGTGACTATTTAAAAGGAGTTAATA"},{"entry":"AATATGCGGCAAGGTATTCTTAAATAAACTGTCAATTTGATAGCGGGAACAAATAATTAGAT"},{"entry":"GTCCTTTTTTAGGAGGGCTTAGTTTTTTGTACCCAGTTTAAGAATACCTTTATCATGTGATT"},{"entry":"CTAAAGTATCCAGAGAATATCTGTATGCTTTGTATACCTATGGTTATGCATAAAAATCCCAG"},{"entry":"TGATAAAAGTATTTATCACTGGGATTTTTATGCCCTTTTGGGTTTTTGAATGGAGGAAAATC"},{"entry":"ACATGAAAATTATTAATATTGGAGTTTTAGCTCATGTTGATGCAGGAAAAACTACCTTAACA"},{"entry":"GAAAGCTTATTATATAACAGTGGAGCGATTACAGAATTAGGAAGCGTGGACAAAGGTACAAC"},{"entry":"GAGGACGGATAATACGCTTTTAGAACGTCAGAGAGGAATTACAATTCAGACAGGAATAACCT"},{"entry":"CTTTTCAGTGGGAAAATACGAAGGTGAACATCATAGACACGCCAGGACATATGGATTTCTTA"},{"entry":"GCAGAAGTATATCGTTCATTATCAGTTTTAGATGGGGCAATTCTACTGATTTCTGCAAAAGA"},{"entry":"TGGCGTACAAGCACAAACTCGTATATTATTTCATGCACTTAGGAAAATGGGGATTCCCACAA"},{"entry":"TCTTTTTTATCAATAAGATTGACCAAAATGGAATTGATTTATCAACGGTTTATCAGGATATT"},{"entry":"AAAGAGAAACTTTCTGCCGAAATTGTAATCAAACAGAAGGTAGAACTGTATCCTAATATGTG"},{"entry":"TGTGACGAACTTTACCGAATCTGAACAATGGGATACGGTAATAGAGGGAAACGATAACCTTT"},{"entry":"TAGAGAAATATATGTCCGGTAAATCATTAGAAGCATTGGAACTCGAACAAGAGGAAAGCATA"},{"entry":"AGATTTCAGAATTGTTCTCTGTTCCCTCTTTATCATGGAAGTGCAAAAAGTAATATAGGGAT"},{"entry":"TGATAACCTTATAGAAGTTATTACTAATAAATTTTATTCATCAACACATCGAGGTCCGTCTG"},{"entry":"AACTTTGCGGAAATGTTTTCAAAATTGAATATACAAAAAAAAGACAACGTCTTGCATATATA"},{"entry":"CGCCTTTATAGTGGAGTACTACATTTACGAGATTCGGTTAGAGTATCAGAAAAAGAAAAAAT"},{"entry":"AAAAGTTACAGAAATGTATACTTCAATAAATGGTGAATTATGTAAGATTGATAGAGCTTATT"},{"entry":"CTGGAGAAATTGTTATTTTGCAAAATGAGTTTTTGAAGTTAAATAGTGTTCTTGGAGATACA"},{"entry":"AAACTATTGCCACAGAGAAAAAAGATTGAAAATCCGCACCCTCTACTACAAACAACTGTTGA"},{"entry":"ACCGAGTAAACCTGAACAGAGAGAAATGTTGCTTGATGCCCTTTTGGAAATCTCAGATAGTG"},{"entry":"ATCCGCTTCTACGATATTACGTGGATTCTACGACACATGAAATTATACTTTCTTTCTTAGGG"},{"entry":"AAAGTACAAATGGAAGTGATTAGTGCACTGTTGCAAGAAAAGTATCATGTGGAGATAGAACT"},{"entry":"AAAAGAGCCTACAGTCATTTATATGGAGAGACCGTTAAAAAATGCAGAATATACCATTCACA"},{"entry":"TCGAAGTGCCGCCAAATCCTTTCTGGGCTTCCATTGGTTTATCTGTATCGCCGCTTCCGTTG"},{"entry":"GGAAGTGGAATGCAGTATGAGAGCTCGGTTTCTCTTGGATACTTAAATCAATCATTTCAAAA"},{"entry":"TGCAGTTATGGAAGGGGTACGCTATGGTTGCGAACAAGGATTATATGGTTGGAATGTGACGG"},{"entry":"ATTGTAAAATCTGTTTTAAGTACGGTTTATACTATAGCCCTGTTAGTACTCCAGCAGATTTT"},{"entry":"CGGATGCTTACTCCTATTGTACTGGAGCAAGCCTTTAGAAAAGCTGGAACAGAATTGTTAGA"},{"entry":"GCCATATCTTAGTTTTAAAGTTTATGCACCACAGGAATATCTTTCACGGGCATATAACGATG"},{"entry":"CTCCCAAATATTGTGCAAATATCGTAAATACTCAACTGAAAAATAATGAGGTCATTATTATT"},{"entry":"GGAGAAATTCCTGCTCGATGTATTCAAGATTATCGCAATGATTTAACTTTTTTTACAAATGG"},{"entry":"GCTTAGTGTTTGTTTAGCAGAGCTAAAAGGATATCAGGTTACCACTGGCGAACCTGTTTGCC"},{"entry":"AGACCCGTCGTCTAAATAGTCGGATAGATAAAGTAAGATATATGTTCAATAAAATAACTTAG"},{"entry":"TGCGTTTTATGTTGTTATATAAATATGGTTTCTTATTAAATAAGATGAAATATTCTTTAATA"},{"entry":"TAGATTTGAATTAAAGTGGAAAGGAGGAGATTGTTATTATAAACTACAAGTGGATATTGTGT"},{"entry":"CCTAGTTGTGGAAATAAAACAAGACTACGAATACGAGTGGATACTATACTTAAAAATTTCCC"},{"entry":"TTTATACAGCCCCAAATGTAAGAACGAAACTTTAATTAATGTTCAAAAAATGAATATAATAA"},{"entry":"CAATCAAAGAGCCAGACGCCAAGACGCAGAGCCGATAATTTGAGAAATGAAACTCTCATCTT"},{"entry":"ATCGGCTCTTTTTGTTTATCTGAATTTTACTGACTAGCCTTCAATATTTCCGCGGCCAGCTT"},{"entry":"ACTATGCCATTATTAAGCTTGTAATATCGGAGGGTTTATTAATTGGCAGTAAAGTGGCAGTT"},{"entry":"TTTGATACCTTAAATGAGATATTATGATAGTGTAGGATATTGACTATCGTACTGCGTTTCCC"},{"entry":"TACCGCAAATTAGGAATAAAGGATCTATGTGGGTTGGCTGATTATAGCCAATCCTTTTTTAA"},{"entry":"TTTTAAAAAGCGTATAGCGCGAGAGTTGGTGGTAAATGAAATGAACGAAAAACAAAAGAGAT"},{"entry":"TCGCAGATGAATATATAATGAATGGATGTAATGGTAAAAAAGCAGCAATTACAGTAGGTTAT"},{"entry":"AGTAAGAAAACAGCAGAGTCTTTAGCAAGTCGATTGTTAAGAAATGTTAATGTTTCGGAATA"},{"entry":"TATTAAAGAACGATTAGAACAGGTACAAGAAGAGCGTTTAATGAGTATTACAGAAGCTTTAG"},{"entry":"CGTTATCTGCTTCTATTGCTAGAGGAGAACCTCAAGAGGCTTACAGTAAGAAATATGACCAT"},{"entry":"TTAAACGATGAAGTGGAAAAAGAGGTTACTTACACAATCACACCAACTTTTGAAGAGCGTCA"},{"entry":"GAGATCTATTGACCACATACTAAAAGTACATGGTGCGTATATCGATAAAAAAGAAATTACTC"},{"entry":"AGAAGAATATTGAGATTAATATTGGTGAGTACGATGACGAAAGTTAAATTGAACTTTAACAA"},{"entry":"ACCGTCTAATGTTTTCAATAGCCGCGGGGGCCCAACACACCAACTTTTGAAGAGCGTCAGAG"},{"entry":"ATCTATTGACCACATACTAAAAGTACATGGTGCGTATATCGATAAAAAAGAAATTACTCAGA"},{"entry":"AGAATATTGAGATTAATATTGGTGAGTACGATGACGAAAGTTAAATTAAACTTTAACAAACC"},{"entry":"GTCTAATGTTTTCAATAGCCGCGGGGGCCCAACGAGCGGCCGCATAGTTAAGCCAGCCCCGA"},{"entry":"CACCCGCCAACACCCGCTGACGCGCCCTGACGGGCTTGTCTGCTCCCGGCATCCGCTTACAG"},{"entry":"ACAAGCTGTGACCGTCTCCGGGAGCTGCATGTGTCAGAGGTTTTCACCGTCATCACCGAAAC"},{"entry":"GCGCGAGACGAAAGGGCCTCGTGATACGCCTATTTTTATAGGTTAATGTCATGATAATAATG"},{"entry":"GTTTCTTAGACGTCAGGTGGCACTTTTCGGGGAAATGTGCGCGGAACCCCTATTTGTTTATT"},{"entry":"TTTCTAAATACATTCAAATATGTATCCGCTCATGAGACAATAACCCTGATAAATGCTTCAAT"},{"entry":"AATATTGAAAAAGGAAGAGTATGAGTATTCAACATTTCCGTGTCGCCCTTATTCCCTTTTTT"},{"entry":"GCGGCATTTTGCCTTCCTGTTTTTGCTCACCCAGAAACGCTGGTGAAAGTAAAAGATGCTGA"},{"entry":"AGATCAGTTGGGTGCACGAGTGGGTTACATCGAACTGGATCTCAACAGCGGTAAGATCCTTG"},{"entry":"AGAGTTTTCGCCCCGAAGAACGTTTTCCAATGATGAGCACTTTTAAAGTTCTGCTATGTGGC"},{"entry":"GCGGTATTATCCCGTATTGACGCCGGGCAAGAGCAACTCGGTCGCCGCATACACTATTCTCA"},{"entry":"GAATGACTTGGTTGAGTACTCACCGGTCACAGAAAAGCATCTTACGGATGGCATGACAGTAA"},{"entry":"GAGAATTATGCAGTGCTGCCATAACCATGAGTGATAACACTGCGGCCAACTTACTTCTGACA"},{"entry":"ACGATCGGAGGACCGAAGGAGCTAACCGCTTTTTTGCACAACATGGGGGATCATGTAACTCG"},{"entry":"CCTTGATCGTTGGGAACCGGAGCTGAATGAAGCCATACCAAACGACGAGCGTGACACCACGA"},{"entry":"TGCCTGTAGCAATGGCAACAACGTTGCGCAAACTATTAACTGGCGAACTACTTACTCTAGCT"},{"entry":"TCCCGGCAACAATTAATAGACTGGATGGAGGCGGATAAAGTTGCAGGACCACTTCTGCGCTC"},{"entry":"GGCCCTTCCGGCTGGCTGGTTTATTGCTGATAAATCTGGAGCCGGTGAGCGTGGGTCTCGCG"},{"entry":"GTATCATTGCAGCACTGGGGCCAGATGGTAAGCCCTCCCGTATCGTAGTTATCTACACGACG"},{"entry":"GGGAGTCAGGCAACTATGGATGAACGAAATAGACAGATCGCTGAGATAGGTGCCTCACTGAT"},{"entry":"TAAGCATTGGTAACTGTCAGACCAAGTTTACTCATATATACTTTAGATTGATTTAAAACTTC"},{"entry":"ATTTTTAATTTAAAAGGATCTAGGTGAAGATCCTTTTTGATAATCTCATGACCAAAATCCCT"},{"entry":"TAACGTGAGTTTTCGTTCCACTGAGCGTCAGACCCCGTAGAAAAGATCAAAGGATCTTCTTG"},{"entry":"AGATCCTTTTTTTCTGCGCGTAATCTGCTGCTTGCAAACAAAAAAACCACCGCTACCAGCGG"},{"entry":"TGGTTTTTTTGCCGGATCAAGAGCTACCAACTCTTTTTCCGAAGGTAACTGGCTTCAGCAGA"},{"entry":"GCGCAGATACCAAATACTGTTCTTCTAGTGTAGCCGTAGTTAGGCCACCACTTCAAGAACTC"},{"entry":"TGTAGCACCGCCTACATACCTCGCTCTGCTAATCCTGTTACCAGTGGCTGCTGCCAGTGGCG"},{"entry":"ATAAGTCGTGTCTTACCGGGTTGGACTCAAGACGATAGTTACCGGATAAGGCGCAGCGGTCG"},{"entry":"GGCTGAACGGGGGGTTCGTGCACACAGCCCAGCTTGGAGCGAACGACCTACACCGAACCTGA"},{"entry":"GATACCTACAGCGTGAGCTATGAGAAAGCGCCACGCTTCCCGAAGGGAGAAAGGCGGACAGG"},{"entry":"TATCCGGTAAGCGGCAGGGTCGGAACAGGAGAGCGCACGAGGGAGCTTCCAGGGGGAAACGC"},{"entry":"CTGGTATCTTTATAGTCCTGTCGGGTTTCGCCACCTCTGACTTGAGCGTCGATTTTTGTGAT"},{"entry":"GCTCGTCAGGGGGGCGGAGCCTATGGAAAAACGCCAGCAACGCGGCCTTTTTACGGTTCCTG"},{"entry":"GCCTTTTGCTGGCCTTTTGCTCACATGTTCTTTCCTGCGTTATCCCCTGATTCTGTGGATAA"},{"entry":"CCGTATTACCGCCTTTGAGTGAGCTGGCGGGTCTAGTTAATGTGTAACGTAACATTAGCTAG"},{"entry":"ATITTTTTATTCAAAAAAATATTTACAAATATTAGGAAATTTAAGTGTAAAAGAGTTGATAA"},{"entry":"ATGATTATATTGGGACTATAATATAATTAAGGTCGATTGAATTCGTTAACTAATTAATCACC"},{"entry":"AAAAAGGAATAGAGTATGAAGTTTGGAAATATTTGTTTTTCGTATCAACCACCAGGTGAAAC"},{"entry":"TCATAAGCAAGTAATGGATCGCTTTGTTCGGCTTGGTATCGCCTCAGAAGAGGTAGGGTTTG"},{"entry":"ATACATATTGGACCTTAGAACATCATTTTACAGAGTTTGGTCTTACGGGAAATTTATTTGTT"},{"entry":"GCTGCGGCTAACCTGTTAGGAAGAACTAAAACATTAAATGTTGGCACTATGGGGGTTGTTAT"},{"entry":"TCCGACAGCACACCCAGTTCGACAGTTAGAAGACGTTTTATTATTAGATCAAATGTCGAAAG"},{"entry":"GTCGTTTTAATTTTGGAACCGTTCGAGGGCTATACCATAAAGATTTTCGAGTATTTGGTGTT"},{"entry":"GATATGGAAGAGTCTCGAGCAATTACTCAAAATTTCTACCAGATGATAATGGAAAGCTTACA"},{"entry":"GACAGGAACCATTAGCTCTGATAGTGATTACATTCAATTTCCTAAGGTTGATGTATATCCCA"},{"entry":"AAGTGTACTCAAAAAATGTACCAACCTGTATGACTGCTGAGTCCGCAAGTACGACAGAATGG"},{"entry":"CTAGCAATACAAGGGCTACCAATGGTTCTTAGTTGGATTATTGGTACTAATGAAAAAAAAGC"},{"entry":"ACAGATGGAACTCTATAATGAAATTGCGACAGAATATGGTCATGATATATCTAAAATAGATC"},{"entry":"ATTGTATGACTTATATTTGTTCTGTTGATGATGATGCACAAAAGGCGCAAGATGTTTGTCGG"},{"entry":"GAGTTTCTGAAAAATTGGTATGACTCATATGTAAATGCGACCAATATCTTTAATGATAGCAA"},{"entry":"TCAAACTCGTGGTTATGATTATCATAAAGGTCAATGGCGTGATTTTGTTTTACAAGGACATA"},{"entry":"CAAACACCAATCGACGTGTTGATTATAGCAATGGTATTAACCCCGTAGGCACTCCTGAGCAG"},{"entry":"TGTATTGAAATCATTCAACGTGATATTGATGCAACGGGTATTACAAACATTACATGCGGATT"},{"entry":"TGAAGCTAATGGAACTGAAGATGAAATAATTGCTTCCATGCGACGCTTTATGACACAAGTCG"},{"entry":"CTCCTTTCTTAAAAGAACCTAAATAAATTACTTATTTGATACTAGAGATAATAAGGAACAAG"},{"entry":"TTATGAAATTTGGATTATTTTTTCTAAACTTTCAGAAAGATGGAATAACATCTGAAGAAACG"},{"entry":"TTGGATAATATGGTAAAGACTGTCACGTTAATTGATTCAACTAAATATCATTTTAATACTGC"},{"entry":"CTTTGTTAATGAACATCACTTTTCAAAAAATGGTATTGTTGGAGCACCTATTACCGCAGCTG"},{"entry":"GTTTTTTATTAGGGTTAACAAATAAATTACATATTGGTTCATTAAATCAAGTAATTACCACC"},{"entry":"CATCACCCTGTACGTGTAGCAGAAGAAGCCAGTTTATTAGATCAAATGTCAGAGGGACGCTT"},{"entry":"CATTCTTGGTTTTAGTGACTGCGAAAGTGATTTCGAAATGGAATTTTTTAGACGTCATATCT"},{"entry":"CATCAAGGCAACAACAATTTGAAGCATGCTATGAAATAATTAATGACGCATTAACTACAGGT"},{"entry":"TATTGCCATCCCCAAAACGACTTTTATGATTTTCCAAAGGTTTCAATTAATCCACACTGTTA"},{"entry":"CAGTGAGAATGGACCTAAGCAATATGTATCCGCTACATCAAAAGAAGTCGTCATGTGGGCAG"},{"entry":"CGAAAAAGGCACTGCCTTTAACGTTTAAGTGGGAGGATAATTTAGAAACCAAAGAACGCTAT"},{"entry":"GCAATTCTATATAATAAAACAGCACAACAATATGGTATTGATATTTCGGATGTTGATCATCA"},{"entry":"ATTAACTGTAATTGCGAACTTAAATGCTGATAGAAGTACGGCTCAAGAAGAAGTGAGAGAAT"},{"entry":"ACTTAAAAGACTATATCACTGAAACTTACCCTCAAATGGACAGAGATGAAAAAATTAACTGC"},{"entry":"ATTATTGAAGAGAATGCAGTTGGGTCTCATGATGACTATTATGAATCGACAAAATTAGCAGT"},{"entry":"GGAAAAAACAGGGTCTAAAAATATTTTATTATCCTTTGAATCAATGTCCGATATTAAAGATG"},{"entry":"TAAAAGATATTATTGATATGTTGAACCAAAAAATCGAAATGAATTTACCATAATAAAATTAA"},{"entry":"AGGCAATTTCTATATTAGATTGCCTTTTTGGCGCGCCTATTCTAATGCATAATAAATACTGA"},{"entry":"TAACATCTTATATTTTGTATTATATTTTGTATTATCGTTGACATGTATAATTTTGATATCAA"},{"entry":"AAACTGATTTTCCCTCTATTATTTTCGAGATTTATTTTCTTAATTCTCTTTAACAAACTAGA"},{"entry":"AATATTGTATATACAAAAAATTATAAATAATAGATGAATAGTTTAATTATAGGTGTTCATCA"},{"entry":"ATCGAAAAAGCAACGTATCTTATTTAAAGTGCGTTGCTTTTTTCTCATTTATAAGGTTAAAT"},{"entry":"AATTCTCATATATCAAGCAAAGTGACA"},{"entry":{}},{"entry":"(mecA\u2003gene\u2003loci\u2003DNA\u2003Sequence\u2003(from\u2003\u2003subsp."},{"entry":{"i":"aureus"}},{"entry":["SEQ\u2003ID\u2003NO:\u200315",{}]},{"entry":["TATACTACAAATGTAGTCTTATATAAGGAGGATATTGATGAAAAAGATAAAAATTGTTCCAC",{}]},{"entry":"TTATTTTAATAGTTGTAGTTGTCGGGTTTGGTATATATTTTTATGCTTCAAAAGATAAAGAA"},{"entry":"ATTAATAATACTATTGATGCAATTGAAGATAAAAATTTCAAACAAGTTTATAAAGATAGCAG"},{"entry":"TTATATTTCTAAAAGCGATAATGGTGAAGTAGAAATGACTGAACGTCCGATAAAAATATATA"},{"entry":"ATAGTTTAGGCGTTAAAGATATAAACATTCAGGATCGTAAAATAAAAAAAGTATCTAAAAAT"},{"entry":"AAAAAACGAGTAGATGCTCAATATAAAATTAAAACAAACTACGGTAACATTGATCGCAACGT"},{"entry":"TCAATTTAATTTTGTTAAAGAAGATGGTATGTGGAAGTTAGATTGGGATCATAGCGTCATTA"},{"entry":"TTCCAGGAATGCAGAAAGACCAAAGCATACATATTGAAAATTTAAAATCAGAACGTGGTAAA"},{"entry":"ATTTTAGACCGAAACAATGTGGAATTGGCCAATACAGGAACAGCATATGAGATAGGCATCGT"},{"entry":"TCCAAAGAATGTATCTAAAAAAGATTATAAAGCAATCGCTAAAGAACTAAGTATTTCTGAAG"},{"entry":"ACTATATCAAACAACAAATGGATCAAAATTGGGTACAAGATGATACCTTCGTTCCACTTAAA"},{"entry":"ACCGTTAAAAAAATGGATGAATATTTAAGTGATTTCGCAAAAAAATTTCATCTTACAACTAA"},{"entry":"TGAAACAGAAAGTCGTAACTATCCTCTAGAAAAAGCGACTTCACATCTATTAGGTTATGTTG"},{"entry":"GTCCCATTAACTCTGAAGAATTAAAACAAAAAGAATATAAAGGCTATAAAGATGATGCAGTT"},{"entry":"ATTGGTAAAAAGGGACTCGAAAAACTTTACGATAAAAAGCTCCAACATGAAGATGGCTATCG"},{"entry":"TGTCACAATCGTTGACGATAATAGCAATACAATCGCACATACATTAATAGAGAAAAAGAAAA"},{"entry":"AAGATGGCAAAGATATTCAACTAACTATTGATGCTAAAGTTCAAAAGAGTATTTATAACAAC"},{"entry":"ATGAAAAATGATTATGGCTCAGGTACTGCTATCCACCCTCAAACAGGTGAATTATTAGCACT"},{"entry":"TGTAAGCACACCTTCATATGACGTCTATCCATTTATGTATGGCATGAGTAACGAAGAATATA"},{"entry":"ATAAATTAACCGAAGATAAAAAAGAACCTCTGCTCAACAAGTTCCAGATTACAACTTCACCA"},{"entry":"GGTTCAACTCAAAAAATATTAACAGCAATGATTGGGTTAAATAACAAAACATTAGACGATAA"},{"entry":"AACAAGTTATAAAATCGATGGTAAAGGTTGGCAAAAAGATAAATCTTGGGGTGGTTACAACG"},{"entry":"TTACAAGATATGAAGTGGTAAATGGTAATATCGACTTAAAACAAGCAATAGAATCATCAGAT"},{"entry":"AACATTTTCTTTGCTAGAGTAGCACTCGAATTAGGCAGTAAGAAATTTGAAAAAGGCATGAA"},{"entry":"AAAACTAGGTGTTGGTGAAGATATACCAAGTGATTATCCATTTTATAATGCTCAAATTTCAA"},{"entry":"ACAAAAATTTAGATAATGAAATATTATTAGCTGATTCAGGTTACGGACAAGGTGAAATACTG"},{"entry":"ATTAACCCAGTACAGATCCTTTCAATCTATAGCGCATTAGAAAATAATGGCAATATTAACGC"},{"entry":"ACCTCACTTATTAAAAGACACGAAAAACAAAGTTTGGAAGAAAAATATTATTTCCAAAGAAA"},{"entry":"ATATCAATCTATTAACTGATGGTATGCAACAAGTCGTAAATAAAACACATAAAGAAGATATT"},{"entry":"TATAGATCTTATGCAAACTTAATTGGCAAATCCGGTACTGCAGAACTCAAAATGAAACAAGG"},{"entry":"AGAAACTGGCAGACAAATTGGGTGGTTTATATCATATGATAAAGATAATCCAAACATGATGA"},{"entry":"TGGCTATTAATGTTAAAGATGTACAAGATAAAGGAATGGCTAGCTACAATGCCAAAATCTCA"},{"entry":"GGTAAAGTGTATGATGAGCTATATGAGAACGGTAATAAAAAATACGATATAGATGAATAACA"},{"entry":"AAACAGTGAAGCAATCCGTAACGATGGTTGCTTCACTGTTTT"},{"entry":{}},{"entry":"(mecA\u2003transcript\u2003sequence)"},{"entry":["SEQ\u2003ID\u2003NO:\u200316",{}]},{"entry":["UAGUCUUAUAUAAGGAGGAUAUUGAUGAAAAAGAUAAAAAUUGUUCCACUUAUUUUAAUAGU",{}]},{"entry":"UGUAGUUGUCGGGUUUGGUAUAUAUUUUUAUGCUUCAAAAGAUAAAGAAAUUAAUAAUACUA"},{"entry":"UUGAUGCAAUUGAAGAUAAAAAUUUCAAACAAGUUUAUAAAGAUAGCAGUUAUAUUUCUAAA"},{"entry":"AGCGAUAAUGGUGAAGUAGAAAUGACUGAACGUCCGAUAAAAAUAUAUAAUAGUUUAGGCGU"},{"entry":"UAAAGAUAUAAACAUUCAGGAUCGUAAAAUAAAAAAAGUAUCUAAAAAUAAAAAACGAGUAG"},{"entry":"AUGCUCAAUAUAAAAUUAAAACAAACUACGGUAACAUUGAUCGCAACGUUCAAUUUAAUUUU"},{"entry":"GUUAAAGAAGAUGGUAUGUGGAAGUUAGAUUGGGAUCAUAGCGUCAUUAUUCCAGGAAUGCA"},{"entry":"GAAAGACCAAAGCAUACAUAUUGAAAAUUUAAAAUCAGAACGUGGUAAAAUUUUAGACCGAA"},{"entry":"ACAAUGUGGAAUUGGCCAAUACAGGAACAGCAUAUGAGAUAGGCAUCGUUCCAAAGAAUGUA"},{"entry":"UCUAAAAAAGAUUAUAAAGCAAUCGCUAAAGAACUAAGUAUUUCUGAAGACUAUAUCAAACA"},{"entry":"ACAAAUGGAUCAAAAUUGGGUACAAGAUGAUACCUUCGUUCCACUUAAAACCGUUAAAAAAA"},{"entry":"UGGAUGAAUAUUUAAGUGAUUUCGCAAAAAAAUUUCAUCUUACAACUAAUGAAACAGAAAGU"},{"entry":"CGUAACUAUCCUCUAGAAAAAGCGACUUCACAUCUAUUAGGUUAUGUUGGUCCCAUUAACUC"},{"entry":"UGAAGAAUUAAAACAAAAAGAAUAUAAAGGCUAUAAAGAUGAUGCAGUUAUUGGUAAAAAGG"},{"entry":"GACUCGAAAAACUUUACGAUAAAAAGCUCCAACAUGAAGAUGGCUAUCGUGUCACAAUCGUU"},{"entry":"GACGAUAAUAGCAAUACAAUCGCACAUACAUUAAUAGAGAAAAAGAAAAAAGAUGGCAAAGA"},{"entry":"UAUUCAACUAACUAUUGAUGCUAAAGUUCAAAAGAGUAUUUAUAACAACAUGAAAAAUGAUU"},{"entry":"AUGGCUCAGGUACUGCUAUCCACCCUCAAACAGGUGAAUUAUUAGCACUUGUAAGCACACCU"},{"entry":"UCAUAUGACGUCUAUCCAUUUAUGUAUGGCAUGAGUAACGAAGAAUAUAAUAAAUUAACCGA"},{"entry":"AGAUAAAAAAGAACCUCUGCUCAACAAGUUCCAGAUUACAACUUCACCAGGUUCAACUCAAA"},{"entry":"AAAUAUUAACAGCAAUGAUUGGGUUAAAUAACAAAACAUUAGACGAUAAAACAAGUUAUAAA"},{"entry":"AUCGAUGGUAAAGGUUGGCAAAAAGAUAAAUCUUGGGGUGGUUACAACGUUACAAGAUAUGA"},{"entry":"AGUGGUAAAUGGUAAUAUCGACUUAAAACAAGCAAUAGAAUCAUCAGAUAACAUUUUCUUUG"},{"entry":"CUAGAGUAGCACUCGAAUUAGGCAGUAAGAAAUUUGAAAAAGGCAUGAAAAAACUAGGUGUU"},{"entry":"GGUGAAGAUAUACCAAGUGAUUAUCCAUUUUAUAAUGCUCAAAUUUCAAACAAAAAUUUAGA"},{"entry":"UAAUGAAAUAUUAUUAGCUGAUUCAGGUUACGGACAAGGUGAAAUACUGAUUAACCCAGUAC"},{"entry":"AGAUCCUUUCAAUCUAUAGCGCAUUAGAAAAUAAUGGCAAUAUUAACGCACCUCACUUAUUA"},{"entry":"AAAGACACGAAAAACAAAGUUUGGAAGAAAAAUAUUAUUUCCAAAGAAAAUAUCAAUCUAUU"},{"entry":"AACUGAUGGUAUGCAACAAGUCGUAAAUAAAACACAUAAAGAAGAUAUUUAUAGAUCUUAUG"},{"entry":"CAAACUUAAUUGGCAAAUCCGGUACUGCAGAACUCAAAAUGAAACAAGGAGAAACUGGCAGA"},{"entry":"CAAAUUGGGUGGUUUAUAUCAUAUGAUAAAGAUAAUCCAAACAUGAUGAUGGCUAUUAAUGU"},{"entry":"UAAAGAUGUACAAGAUAAAGGAAUGGCUAGCUACAAUGCCAAAAUCUCAGGUAAAGUGUAUG"},{"entry":"AUGAGCUAUAUGAGAACGGUAAUAAAAAAUACGAUAUAGAUGAAUAACAAAACAGUGAAGCA"},{"entry":"AUCCGUAACGAUGGUUGCUUCACUGUUUU"},{"entry":{}},{"entry":"(luxAB\u2003gene\u2003loci\u2003DNA\u2003sequence\u2003(from\u2003\u2003genes\u2003luxA"},{"entry":"and\u2003luxB\u2003for\u2003luciferase\u2003alpha\u2003and\u2003beta\u2003subunits-GenBank:"},{"entry":"X06758.1))"},{"entry":["SEQ\u2003ID\u2003NO:\u200317",{}]},{"entry":["GGCTTAAATAAACAGAATCACCAAAAAGGAATAGAGTATGAAGTTTGGAAATATTTGTTTTT",{}]},{"entry":"CGTATCAACCACCAGGTGAAACTCATAAGCTAAGTAATGGATCGCTTTGTTCGGCTTGGTAT"},{"entry":"CGCCTCAGAAGAGTAGGGTTTGATACATATTGGACCTTAGAACATCATTTTACAGAGTTTGG"},{"entry":"TCTTACGGGAAATTTATTTGTTGCTGCGGCTAACCTGTTAGGAAGAACTAAAACATTAAATG"},{"entry":"TTGGCACTATGGGGGTTGTTATTCCGACAGCACACCCAGTTCGACAGTTAGAAGACGTTTTA"},{"entry":"TTATTAGATCAAATGTCGAAAGGTCGTTTTAATTTTGGAACCGTTCGAGGGCTATACCATAA"},{"entry":"AGATTTTCGAGTATTTGGTGTTGATATGGAAGAGTCTCGAGCAATTACTCAAAATTTCTACC"},{"entry":"AGATGATAATGGAAAGCTTACAGACAGGAACCATTAGCTCTGATAGTGATTACATTCAATTT"},{"entry":"CCTAAGGTTGATGTATATCCCAAAGTGTACTCAAAAAATGTACCAACCTGTATGACTGCTGA"},{"entry":"GTCCGCAAGTACGACAGAATGGCTAGCAATACAAGGGCTACCAATGGTTCTTAGTTGGATTA"},{"entry":"TTGGTACTAATGAAAAAAAAGCACAGATGGAACTCTATAATGAAATTGCGACAGAATATGGT"},{"entry":"CATGATATATCTAAAATAGATCATTGTATGACTTATATTTGTTCTGTTGATGATGATGCACA"},{"entry":"AAAGGCGCAAGATGTTTGTCGGGAGTTTCTGAAAAATTGGTATGACTCATATGTAAATGCGA"},{"entry":"CCAATATCTITAATGATAGCAATCAAACTCGTGGTTATGATTATCATAAAGGTCAATGGCGT"},{"entry":"GATTTTGTTTTACAAGGACATACAAACACCAATCGACGTGTTGATTATAGCAATGGTATTAA"},{"entry":"CCCTGTAGGCACTCCTGAGCAGTGTATTGAAATCATTCAACGTGATATTGATGCAACGGGTA"},{"entry":"TTACAAACATTACATGCGGATTTGAAGCTAATGGAACTGAAGATGAAATAATTGCTTCCATG"},{"entry":"CGACGCTTTATGACACAAGTCGCTCCTTTCTTAAAAGAACCTAAATAAATTACTTATTTGAT"},{"entry":"ACTAGAGATAATAAGGAACAAGTTATGAAATTTGGATTATTTTTTCTAAACTTTCAGAAAGA"},{"entry":"TGGAATAACATCTGANGAAACGTTGGATAATATGGTAAAGACTGTCACGTTAATTGATTCAA"},{"entry":"CTAAATATCATTTTAATACTGCCTTTGTTAATGAACATCACTTTTCAAAAAATGGTATTGTT"},{"entry":"GGAGCACCTATTACCGCAGCTGGTTTTTTATTAGGGTTAACAAATAAATTACATATTGGTTC"},{"entry":"ATTAAATCAAGTAATTACCACCCATCACCCTGTACGTGTAGCAGAAGAAGCCAGTTTATTAG"},{"entry":"ATCAAATGTCAGAGGGACGCTTCATTCTTGGTTTTAGTGACTGCGAAAGTGATTTCGAAATG"},{"entry":"GAATTTTTTAGACGTCATATCTCATCAAGGCAACAACAATTTGAAGCATGCTATGAAATAAT"},{"entry":"TAATGACGCATTAACTACAGGTTATTGTCATCCCCAAAACGACTTTTATGATTTTCCAAAGG"},{"entry":"TTTCAATTAATCCACACTGTTACAGTGAGAATGGACCTAAGCAATATGTATCCGCTACATCA"},{"entry":"AAAGAAGTCGTCATGTGGGCAGCGAAAAAGGCACTGCCTTTAACATTTAAGTGGGAGGATAA"},{"entry":"TTTAGAAACCAAAGAACGCTATGCAATTCTATATAATAAAACAGCACAACAATATGGTATTG"},{"entry":"ATATTTCGGATGTTGATCATCAATTAACTGTAATTGCGAACTTAAATGCTGATAGAAGTACG"},{"entry":"GCTCAAGAAGAAGTGAGAGAATACTTAAAAGACTATATCACTGAAACTTACCCTCAAATGGA"},{"entry":"CAGAGATGAAAAAATTAACTGCATTATTGAAGAGAATGCAGTTGGGTCTCATGATGACTATT"},{"entry":"ATGAATCGACAAAATTAGCAGTGGAAAAAACAGGGTCTAAAAATATTTTATTATCCTTTGAA"},{"entry":"TCAATGTCCGATATTAAAGATGTAAAAGATATTATTGATATGTTGAACCAAAAAATCGAAAT"},{"entry":"GAATTTACCATAATAAAATTAAAGGCAATTTCTATATTAGATTGCCTTTTTAAATTTC"},{"entry":{}},{"entry":"(luxAB\u2003Transcript\u2003Sequence)"},{"entry":["SEQ\u2003ID\u2003NO:\u200318",{}]},{"entry":["AAUCACCAAAAAGGAAUAGAGUAUGAAGUUUGGAAAUAUUUGUUUUUCGUAUCAACCACCAG",{}]},{"entry":"GUGAAACUCAUAAGCUAAGUAAUGGAUCGCUUUGUUCGGCUUGGUAUCGCCUCAGAAGAGUA"},{"entry":"GGGUUUGAUACAUAUUGGACCUUAGAACAUCAUUUUACAGAGUUUGGUCUUACGGGAAAUUU"},{"entry":"AUUUGUUGCUGCGGCUAACCUGUUAGGAAGAACUAAAACAUUAAAUGUUGGCACUAUGGGGG"},{"entry":"UUGUUAUUCCGACAGCACACCCAGUUCGACAGUUAGAAGACGUUUUAUUAUUAGAUCAAAUG"},{"entry":"UCGAAAGGUCGUUUUAAUUUUGGAACCGUUCGAGGGCUAUACCAUAAAGAUUUUCGAGUAUU"},{"entry":"UGGUGUUGAUAUGGAAGAGUCUCGAGCAAUUACUCAAAAUUUCUACCAGAUGAUAAUGGAAA"},{"entry":"GCUUACAGACAGGAACCAUUAGCUCUGAUAGUGAUUACAUUCAAUUUCCUAAGGUUGAUGUA"},{"entry":"UAUCCCAAAGUGUACUCAAAAAAUGUACCAACCUGUAUGACUGCUGAGUCCGCAAGUACGAC"},{"entry":"AGAAUGGCUAGCAAUACAAGGGCUACCAAUGGUUCUUAGUUGGAUUAUUGGUACUAAUGAAA"},{"entry":"AAAAAGCACAGAUGGAACUCUAUAAUGAAAUUGCGACAGAAUAUGGUCAUGAUAUAUCUAAA"},{"entry":"AUAGAUCAUUGUAUGACUUAUAUUUGUUCUGUUGAUGAUGAUGCACAAAAGGCGCAAGAUGU"},{"entry":"UUGUCGGGAGUUUCUGAAAAAUUGGUAUGACUCAUAUGUAAAUGCGACCAAUAUCUUUAAUG"},{"entry":"AUAGCAAUCAAACUCGUGGUUAUGAUUAUCAUAAAGGUCAAUGGCGUGAUUUUGUUUUACAA"},{"entry":"GGACAUACAAACACCAAUCGACGUGUUGAUUAUAGCAAUGGUAUUAACCCUGUAGGCACUCC"},{"entry":"UGAGCAGUGUAUUGAAAUCAUUCAACGUGAUAUUGAUGCAACGGGUAUUACAAACAUUACAU"},{"entry":"GCGGAUUUGAAGCUAAUGGAACUGAAGAUGAAAUAAUUGCUUCCAUGCGACGCUUUAUGACA"},{"entry":"CAAGUCGCUCCUUUCUUAAAAGAACCUAAAUAAAUUACUUAUUUGAUACUAGAGAUAAUAAG"},{"entry":"GAACAAGUUAUGAAAUUUGGAUUAUUUUUUCUAAACUUUCAGAAAGAUGGAAUAACAUCUGA"},{"entry":"AGAAACGUUGGAUAAUAUGGUAAAGACUGUCACGUUAAUUGAUUCAACUAAAUAUCAUUUUA"},{"entry":"AUACUGCCUUUGUUAAUGAACAUCACUUUUCAAAAAAUGGUAUUGUUGGAGCACCUAUUACC"},{"entry":"GCAGCUGGUUUUUUAUUAGGGUUAACAAAUAAAUUACAUAUUGGUUCAUUAAAUCAAGUAAU"},{"entry":"UACCACCCAUCACCCUGUACGUGUAGCAGAAGAAGCCAGUUUAUUAGAUCAAAUGUCAGAGG"},{"entry":"GACGCUUCAUUCUUGGUUUUAGUGACUGCGAAAGUGAUUUCGAAAUGGAAUUUUUUAGACGU"},{"entry":"CAUAUCUCAUCAAGGCAACAACAAUUUGAAGCAUGCUAUGAAAUAAUUAAUGACGCAUUAAC"},{"entry":"UACAGGUUAUUGUCAUCCCCAAAACGACUUUUAUGAUUUUCCAAAGGUUUCAAUUAAUCCAC"},{"entry":"ACUGUUACAGUGAGAAUGGACCUAAGCAAUAUGUAUCCGCUACAUCAAAAGAAGUCGUCAUG"},{"entry":"UGGGCAGCGAAAAAGGCACUGCCUUUAACAUUUAAGUGGGAGGAUAAUUUAGAAACCAAAGA"},{"entry":"ACGCUAUGCAAUUCUAUAUAAUAAAACAGCACAACAAUAUGGUAUUGAUAUUUCGGAUGUUG"},{"entry":"AUCAUCAAUUAACUGUAAUUGCGAACUUAAAUGCUGAUAGAAGUACGGCUCAAGAAGAAGUG"},{"entry":"AGAGAAUACUUAAAAGACUAUAUCACUGAAACUUACCCUCAAAUGGACAGAGAUGAAAAAAU"},{"entry":"UAACUGCAUUAUUGAAGAGAAUGCAGUUGGGUCUCAUGAUGACUAUUAUGAAUCGACAAAAU"},{"entry":"UAGCAGUGGAAAAAACAGGGUCUAAAAAUAUUUUAUUAUCCUUUGAAUCAAUGUCCGAUAUU"},{"entry":"AAAGAUGUAAAAGAUAUUAUUGAUAUGUUGAACCAAAAAAUCGAAAUGAAUUUACCAUAAUA"},{"entry":"AAAUUAAAGGCAAUUUCUAUAUUAGAUUGCCUUUU"},{"entry":{}},{"entry":"(cis-repressed\u2003luxAB\u2003transcript\u2003sequence)"},{"entry":["SEQ\u2003ID\u2003NO:\u200319",{}]},{"entry":["AAAGGCAUGAAAAAACUUGGUAUCUUCACCAACACCUAGCUUUUUGAAGGAAUUGAGUAUGA",{}]},{"entry":"AGUUUGGAAAUAUUGGUUGUUCGUAUCAACCACCAGGUGAAACUCAUAAGCUAAAGGCAUGA"},{"entry":"AAAAACUAGGUGAUCUUCACCAACACCUAGUUUUUUCAAGGAAUUGAGUAUGAAGUUUGGAA"},{"entry":"AUAUUUGUUUUUCGUAUCAACCACCAGGUGAAACUCAUAAGCUAAGUAAUGGAUCGCUUUGU"},{"entry":"UCGGCUUGGUAUCGCCUCAGAAGAGUAGGGUUUGAUACAUAUUGGACCUUAGAACAUCAUUU"},{"entry":"UACAGAGUUUGGUCUUACGGGAAAUUUAUUUGUUGCUGCGGCUAACCUGUUAGGAAGAACUA"},{"entry":"AAACAUUAAAUGUUGGCACUAUGGGGGUUGUUAUUCCGACAGCACACCCAGUUCGACAGUUA"},{"entry":"GAAGACGUUUUAUUAUUAGAUCAAAUGUCGAAAGGUCGUUUUAAUUUUGGAACCGUUCGAGG"},{"entry":"GCUAUACCAUAAAGAUUUUCGAGUAUUUGGUGUUGAUAUGGAAGAGUCUCGAGCAAUUACUC"},{"entry":"AAAAUUUCUACCAGAUGAUAAUGGAAAGCUUACAGACAGGAACCAUUAGCUCUGAUAGUGAU"},{"entry":"UACAUUCAAUUUCCUAAGGUUGAUGUAUAUCCCAAAGUGUACUCAAAAAAUGUACCAACCUG"},{"entry":"UAUGACUGCUGAGUCCGCAAGUACGACAGAAUGGCUAGCAAUACAAGGGCUACCAAUGGUUC"},{"entry":"UUAGUUGGAUUAUUGGUACUAAUGAAAAAAAAGCACAGAUGGAACUCUAUAAUGAAAUUGCG"},{"entry":"ACAGAAUAUGGUCAUGAUAUAUCUAAAAUAGAUCAUUGUAUGACUUAUAUUUGUUCUGUUGA"},{"entry":"UGAUGAUGCACAAAAGGCGCAAGAUGUUUGUCGGGAGUUUCUGAAAAAUUGGUAUGACUCAU"},{"entry":"AUGUAAAUGCGACCAAUAUCUUUAAUGAUAGCAAUCAAACUCGUGGUUAUGAUUAUCAUAAA"},{"entry":"GGUCAAUGGCGUGAUUUUGUUUUACAAGGACAUACAAACACCAAUCGACGUGUUGAUUAUAG"},{"entry":"CAAUGGUAUUAACCCUGUAGGCACUCCUGAGCAGUGUAUUGAAAUCAUUCAACGUGAUAUUG"},{"entry":"AUGCAACGGGUAUUACAAACAUUACAUGCGGAUUUGAAGCUAAUGGAACUGAAGAUGAAAUA"},{"entry":"AUUGCUUCCAUGCGACGCUUUAUGACACAAGUCGCUCCUUUCUUAAAAGAACCUAAAUAAAU"},{"entry":"UACUUAUUUGAUACUAGAGAUAAUAAGGAACAAGUUAUGAAAUUUGGAUUAUUUUUUCUAAA"},{"entry":"CUUUCAGAAAGAUGGAAUAACAUCUGAAGAAACGUUGGAUAAUAUGGUAAAGACUGUCACGU"},{"entry":"UAAUUGAUUCAACUAAAUAUCAUUUUAAUACUGCCUUUGUUAAUGAACAUCACUUUUCAAAA"},{"entry":"AAUGGUAUUGUUGGAGCACCUAUUACCGCAGCUGGUUUUUUAUUAGGGUUAACAAAUAAAUU"},{"entry":"ACAUAUUGGUUCAUUAAAUCAAGUAAUUACCACCCAUCACCCUGUACGUGUAGCAGAAGAAG"},{"entry":"CCAGUUUAUUAGAUCAAAUGUCAGAGGGACGCUUCAUUCUUGGUUUUAGUGACUGCGAAAGU"},{"entry":"GAUUUCGAAAUGGAAUUUUUUAGACGUCAUAUCUCAUCAAGGCAACAACAAUUUGAAGCAUG"},{"entry":"CUAUGAAAUAAUUAAUGACGCAUUAACUACAGGUUAUUGUCAUCCCCAAAACGACUUUUAUG"},{"entry":"AUUUUCCAAAGGUUUCAAUUAAUCCACACUGUUACAGUGAGAAUGGACCUAAGCAAUAUGUA"},{"entry":"UCCGCUACAUCAAAAGAAGUCGUCAUGUGGGCAGCGAAAAAGGCACUGCCUUUAACAUUUAA"},{"entry":"GUGGGAGGAUAAUUUAGAAACCAAAGAACGCUAUGCAAUUCUAUAUAAUAAAACAGCACAAC"},{"entry":"AAUAUGGUAUUGAUAUUUCGGAUGUUGAUCAUCAAUUAACUGUAAUUGCGAACUUAAAUGCU"},{"entry":"GAUAGAAGUACGGCUCAAGAAGAAGUGAGAGAAUACUUAAAAGACUAUAUCACUGAAACUUA"},{"entry":"CCCUCAAAUGGACAGAGAUGAAAAAAUUAACUGCAUUAUUGAAGAGAAUGCAGUUGGGUCUC"},{"entry":"AUGAUGACUAUUAUGAAUCGACAAAAUUAGCAGUGGAAAAAACAGGGUCUAAAAAUAUUUUA"},{"entry":"UUAUCCUUUGAAUCAAUGUCCGAUAUUAAAGAUGUAAAAGAUAUUAUUGAUAUGUUGAACCA"},{"entry":"AAAAAUCGAAAUGAAUUUACCAUAAUAAAAUUAAAGGCAAUUUCUAUAUUAGAUUGCCUUUU"}]}}}}}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE SEVERAL VIEWS OF THE DRAWINGS","p":["These and other features, aspects, and advantages of the present invention will become better understood with regard to the following description, and accompanying drawings, where:",{"@attributes":{"id":"p-0071","num":"0070"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0072","num":"0071"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0073","num":"0072"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0074","num":"0073"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0075","num":"0074"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0076","num":"0075"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0077","num":"0076"},"figref":"FIG. 7"},{"@attributes":{"id":"p-0078","num":"0077"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0079","num":"0078"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0080","num":"0079"},"figref":"FIG. 10","i":["Enterococcus faecium ","E. faecalis"]},{"@attributes":{"id":"p-0081","num":"0080"},"figref":"FIG. 11","i":"C. difficile"},{"@attributes":{"id":"p-0082","num":"0081"},"figref":"FIG. 12","i":"S. aureus","sub":"spa"},{"@attributes":{"id":"p-0083","num":"0082"},"figref":"FIG. 13"},{"@attributes":{"id":"p-0084","num":"0083"},"figref":"FIG. 14"},{"@attributes":{"id":"p-0085","num":"0084"},"figref":"FIG. 15"},{"@attributes":{"id":"p-0086","num":"0085"},"figref":"FIG. 16"},{"@attributes":{"id":"p-0087","num":"0086"},"figref":"FIG. 17"},{"@attributes":{"id":"p-0088","num":"0087"},"figref":"FIG. 18"},{"@attributes":{"id":"p-0089","num":"0088"},"figref":"FIG. 19"},{"@attributes":{"id":"p-0090","num":"0089"},"figref":"FIG. 20"},{"@attributes":{"id":"p-0091","num":"0090"},"figref":"FIG. 21"},{"@attributes":{"id":"p-0092","num":"0091"},"figref":"FIG. 22","i":"S. aureus "},{"@attributes":{"id":"p-0093","num":"0092"},"figref":"FIG. 23","i":"S. aureus "},{"@attributes":{"id":"p-0094","num":"0093"},"figref":["FIG. 24","FIG. 24"]},{"@attributes":{"id":"p-0095","num":"0094"},"figref":"FIG. 25"},{"@attributes":{"id":"p-0096","num":"0095"},"figref":"FIG. 26"},{"@attributes":{"id":"p-0097","num":"0096"},"figref":"FIG. 27"},{"@attributes":{"id":"p-0098","num":"0097"},"figref":"FIG. 28"},{"@attributes":{"id":"p-0099","num":"0098"},"figref":"FIG. 29"},{"@attributes":{"id":"p-0100","num":"0099"},"figref":"FIG. 30"},{"@attributes":{"id":"p-0101","num":"0100"},"figref":"FIG. 31"},{"@attributes":{"id":"p-0102","num":"0101"},"figref":"FIG. 32"},{"@attributes":{"id":"p-0103","num":"0102"},"figref":"FIG. 33"},{"@attributes":{"id":"p-0104","num":"0103"},"figref":"FIG. 34"},{"@attributes":{"id":"p-0105","num":"0104"},"figref":"FIG. 35"}]},"DETDESC":[{},{}]}
