---
title: Methods and compositions of improved modified siRNA
abstract: Chemically modified small interfering RNAs (siRNAs) that include both phosphorodithioate modifications (PS2-RNA) and 2â€²-O-methyl modifications (MePS2) provide improved RNA silencing. Specific chemically modified siRNA that show enhanced silencing of RNAs involved in resistance to chemotherapeutic agents are provided.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09629867&OS=09629867&RS=09629867
owner: AM Biotechnologies, Inc.
number: 09629867
owner_city: Houston
owner_country: US
publication_date: 20150311
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS REFERENCE TO RELATED APPLICATIONS","STATEMENT REGARDING GOVERNMENT INTERESTS","REFERENCE TO SEQUENCE LISTING SUBMITTED ELECTRONICALLY","FIELD OF THE INVENTION","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","ABBREVIATIONS","EXAMPLE 1","Locations of PS2 Modifications","EXAMPLE 2","Synthesis and Activity of Chemically Modified siRNAs","EXAMPLE 3","Biophysical Properties of MePS2-1 siRNA","EXAMPLE 4","In Vivo Application of MePS2-1 siRNA","EXAMPLE 5","Targeting GRAMD1B for Ovarian Cancer Treatment","EXAMPLE 6","Therapeutic Effect of MePS2-1 siGRAMD1B In Vitro and In Vivo","EXAMPLE 7","Methods","EXAMPLE 8","Further Analysis of Mechanisms of Benefits Seen with MePS2 Modification"],"p":["This application claims priority based on U.S. Provisional Application Ser. No. 61\/951,518 filed Mar. 11, 2014, which is incorporated herein by reference.","This invention was made with government support under SBIR R44GM086937 awarded by the National Institutes of Health. The Government has certain rights in the invention.","The Sequence Listing associated with the application is provided in text format in lieu of a paper copy, and is hereby incorporated by reference into the specification. The name of the text file containing the Sequence Listing is SequenceListing_ST25.txt. The text file is 24 kilobytes, was created on May 21, 2015, and is being submitted electronically via EFS-Web.","This invention relates generally to compositions and methods for improving siRNA based therapies.","Without limiting the scope of the invention, its background is described in connection with existing siRNA based therapies. SiRNA-based therapies have shown promise in recent clinical trials as anti-cancer agents. See Tabernero, J. et al. \u201cFirst-in-humans trial of an RNA interference therapeutic targeting VEGF and KSP in cancer patients with liver involvement\u201d 3, 406-417 (2013). This technology offers tremendous potential in silencing \u201cnon-druggable\u201d targets that are not amenable to conventional therapeutics such as small molecules or monoclonal antibodies. However, the lack of stability and low potency at target sites following systemic administration remain the major obstacles to clinical translation of this therapy. See e.g. Pecot, C., et al. \u201cRNA interference in the clinic: challenges and future directions\u201d 11, 59-67 (2011).","To overcome these problems, efforts have been made to chemically modify the 2\u2032-position of the ribose moiety (e.g. 2\u2032-OMe or 2\u2032-F) or phosphate backbone of the siRNA molecule (e.g. phosphoromonothioate (PS) modification) such that enhanced serum stability, bioavailability, and prolonged duration of action could be achieved. See e.g. Behlke, M. A. \u201cProgress towards in vivo use of siRNAs\u201d 13, 644-670 (2006). Importantly, combining PS and 2\u2032-modification impart superior stability over single modifications leading to the approval of Kynamro, an antisense therapeutic, by the Food and Drug Administration. See Stein, D., et al. \u201cA specificity comparison of four antisense types: morpholino, 2\u2032-O-methyl RNA, DNA, and phosphorothioate DNA\u201d 7, 151-157 (1997).","Despite the advance, the combination of PS and 2\u2032-modification shows limited enhancement of siRNA potency. See e.g. Prakash, T. et al. \u201cPositional effect of chemical modifications on short interference RNA activity in mammalian cells\u201d 48, 4247-4253 (2005).","From the foregoing it is apparent the there is a need in the art for improved strategies for the chemical modification of siRNAs such that their therapeutic potential can be realized.","Provide herein are chemically modified siRNA molecules including a combination of 2\u2032-OMe and PS2 modifications on either or both sense and antisense strands that improve serum stability and gene silencing efficacy. In certain embodiments, both sense and antisense-strands of the siRNA contain dTdT on their 3\u2032 terminus. In other embodiments, a 3\u2032 end of the sense strand terminates in a sequence UGdTdT-3\u2032. In certain embodiments, the 2\u2032-OMe and PS2 modifications are in a backbone region adjacent to a 3\u2032-dTdT overhang of the siRNA sense strand. In certain embodiments, a length of either the sense or antisense strand of the chemically modified siRNA is 21 nucleotides including a 3\u2032 dTdT sequence.","Also provided herein are chemically modified siRNAs directed against an RNA that generates a protein conferring chemo resistance wherein the chemical modifications comprise a combination of 2\u2032-OMe and PS2 modifications on either or both sense and antisense strands that improve serum stability and gene silencing efficacy. Exemplified siRNAs include those directed to interference with the RISC Ago2 slicer associated with chemoresistance in ovarian cancer cell lines and tumors as well as chemically modified siRNAs directed to interference with GRAM Domain Containing 1B (GRAMD1B).","As disclosed herein, combination therapies for cancer chemotherapy were found particularly efficacious where a chemotherapeutic agent was delivered in the same course of treatment with a chemically modified siRNA that interferes with expression of a protein conferring resistance to the chemotherapeutic agent, wherein the chemical modifications to the siRNA comprise a combination of 2\u2032-OMe and PS2 modifications on either or both sense and antisense strands of the siRNA that improve serum stability and gene silencing efficacy of the siRNA. Method of improving serum stability and gene silencing efficacy of the siRNAs are provided that include introducing a combination of 2\u2032-OMe and PS2 modifications on either or both sense and antisense strands of the siRNA (sometimes referred to herein as MePS2 modifications) and testing for improved serum stability and gene silencing efficacy of the siRNA.","The present inventor appreciated that, while the combination of PS and 2\u2032-modifications to siRNA might be expected to solve stability problems that limited the therapeutic potential of siRNA, these modifications showed limited enhancement of siRNA potency. Furthermore, the present inventor appreciated that synthesis of PS or MePS by standard phosphoramidite methodology also generates a mixture of unresolvable diastereomeric oligomers, which could potentially result in variable biochemical, biophysical, and biological properties. The present inventor recently reported that PS2 modification, which substitutes sulfur atoms for both of the non-bridging phosphate and oxygen atoms, significantly improves serum stability and gene silencing over PS-modified siRNAs. Yang, X. et al. \u201cGene silencing activity of siRNA molecules containing phosphorodithioate substitutions\u201d 7, 1214-1220 (2012) 15. The achiral nature of PS2 modifications at the phosphorous center eliminates the generation of isomeric byproducts, thereby permitting batch to batch quality control, which is a feature that cannot be achieved with the MePS platform. See Wiesler, W. T. & Caruthers, M. H. \u201cSynthesis of phosphorodithioate DNA via sulfur-linked, base-labile protecting groups\u201d 61, 4272-4281 (1996).","The solution described herein provides novel methods and compositions that introduce critical new features to PS2-modified siRNA thus further improving siRNA efficacy both in vitro and in vivo. The current inventor hypothesized that the incorporation of a sugar modification into a phosphorodithioate (PS2) modified construct could lead to further enhancement of the PS2 siRNA stability and increase its clinical potential. However, also important to the generation of these constructs was the placement of the modifications as will be later described. Two sugar modifications, 2\u2032-O-Methyl (2\u2032-OMe) and 2\u2032-fluoride (2\u2032-F) were tested. As demonstrated herein, the novel exemplified MePS2 design significantly enhances serum stability and silencing activity of siRNAs. This enhanced potency stems from an unforeseen increase in the loading of siRNAs to the RNA-induced silencing complex (RISC), likely due to the unique interaction mediated by 2\u2032-OMe and PS2.","The therapeutic utility of MePS2-modified siRNAs is exemplified by targeting GRAM Domain Containing 1B (GRAMD1B). GRAMD1B is a novel gene identified through a whole genome taxane sensitivity screen and encodes a protein involved in chemoresistance. GRAMD1B silencing was achieved in tumors following MePS2-modified siRNA treatment, leading to a synergistic anti-tumor effect in combination with paclitaxel.","Given that limited success has been achieved thus far toward broadly enhancing siRNA potency with chemically modified siRNAs, the MePS2 modification provided herein represents a major new and unexpected direction for enhancing the clinical application of existing RNA interference (RNAi)-based therapies.","While the making and using of various embodiments of the present invention are discussed in detail below, it should be appreciated that the present invention provides many applicable inventive concepts which can be employed in a wide variety of specific contexts. The specific embodiment discussed herein are merely illustrative of specific ways to make and use the invention and do not delimit the scope of the invention.","The following abbreviations are used throughout this application:","GRAMD1B GRAM Domain Containing 1B","PS2 phosphorodithioate","RISC RNA-induced silencing complex","siRNA small interfering RNA","UM unmodified","To facilitate the understanding of this invention, a number of terms are defined below. Terms defined herein have meanings as commonly understood by a person of ordinary skill in the areas relevant to the present invention. Terms such as \u201ca\u201d, \u201can\u201d and \u201cthe\u201d are not intended to refer to only a singular entity, but include the general class of which a specific example may be used for illustration. The terminology herein is used to describe specific embodiments of the invention, but their usage does not delimit the invention, except as outlined in the claims.","Herein it is shown that novel dual modified siRNAs featuring 2\u2032-X modification(s) of the ribose moiety on a backbone having a PS2 modification on the same base exhibit surprisingly improved siRNA silencing by a new mechanism. In one example, the siRNA chemistry provides MePS2 modified siRNAs selected with a whole genome lethality screen aimed at identifying novel genes involved in taxane resistance. This example demonstrates for the first time the feasibility of using highly potent MePS2 modified siRNAs for effective therapeutic targeting. Using GRAMD1B as a model system, it is demonstrated that robust gene silencing in tumors can be achieved following systemic administration of MePS2 modified siRNAs. This ultimately led to significant re-sensitization of chemoresistant ovarian tumors to taxane therapy.","Most of the chemical modification strategies, to date, have focused on improving siRNA stability in serum. While this is critical, improving siRNA potency at target sites following systemic administration still remains the major obstacle to clinical translation of siRNAs as therapeutic modalities. Through a systematic approach, a unique combination of 2\u2032-OMe and PS2 modifications (MePS2) was developed that significantly improves stability and potency of siRNAs simultaneously (). As shown herein, the increase in potency could not be explained by the design's enhanced stability, consistent with previous findings that this mechanism has a limited role within rapidly-dividing cells. In investigating other potential causes it was determined that, compared to PS2 modification, the exemplified MePS2 modification remarkably further enhances the loading of siRNAs to RISC resulting in enhanced silencing. While the addition of 2-O-methyl groups to RNA typically elevates Tm, here it is shown that the addition of 2\u2032-OMe to PS2-modified siRNAs does not increase the Tm of the siRNA duplex, suggesting a unique property arising from this combination. Computational modeling also indicates an unusual favorable stereoelectronic effect on the siRNA backbone following MePS2 modification. These results indicate that certain modifications, including MePS2 modification, can improve the interaction of siRNA with RISC via several mechanisms, including potentially the thiol-protein interaction with the RISC complex, modulation of Tm, and altered stereoelectronics. This is in contrast to currently existing modifications, where RISC loading efficiency is dependent on alteration in Tm, thus leading to variable silencing effects depending on the Tm of the sequences to be modified. Hence, modifications, such as the exemplified MePS2 modification, have a unique potential to provide more consistent, sequence-independent, silencing effects compared to existing chemistries.","In the examples provided herein it is shown that a combination of 2\u2032-OMe and PS2 modifications not only significantly increases the serum stability of siRNAs but also triggers a more potent intrinsic RNAi response than the individual modifications. In addition to its achiral nature, which favors clinical translation, the ability of the MePS2 modification to overcome major obstacles of bringing siRNA therapeutics into the clinic is demonstrated. In addition the achiral PS2 allows direct sulfur-protein interaction from either of the phosphoryl sulfur atoms whereas in a diastereomeric mixture with a monothiophosphate only one half of the diasteromers will have the phosphoryl sulfur pointed to the corrected interaction with the protein, while the other diasteromer will have a phosphoryl oxygen pointed to the specific interaction with the RISC protein. In the latter case this diastereomer would exhibit much poorer binding, comparable to the UM form. If more than one PS2 group were to be replaced by monothiophosphates, then even more of the diastereomeric mixture (\u00befor two modifications, \u215eths for three, etc.) would have improper phosphoryl oxygen-protein interactions thus highlighting the value of the PS2 group.","The following examples are included for the sake of completeness of disclosure and to illustrate the methods of making the compositions and composites of the present invention as well as to present certain characteristics of the compositions. In no way are these examples intended to limit the scope or teaching of this disclosure.","Favorable positions of PS2 modification were identified using an siRNA that targets Green Fluorescence Protein (GFP) as shown in Table 1.  shows the silencing efficiency of the PS2-modified siRNAs of . In , P-values were obtained with Student's t-test; *=P<0.05, compared to unmodified (UM) siGFP; bars and error bars represent mean values and the corresponding S.E.M.s (n=3).",{"@attributes":{"id":"p-0059","num":"0058"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u20031"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"SiRNA\u2003sequences\u2003for\u2003a\u2003panel\u2003of\u2003PS2-modified"},{"entry":"siGFPs.\u2003Nucleotides\u2003underlined\u2003are\u2003modified\u2003with"},{"entry":"PS2\u2003chemistry."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["No.",{},"SiRNA\u2003sequences","SEQ.ID."]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["Control","S","5\u2032-AAUCAGAUUGAACCUUCAUTT-3\u2032","\u20031"]},{"entry":[{},"AS","3\u2032-TTUUAGUCUAACUUGGAAGUA-5\u2032","\u20032"]},{"entry":{}},{"entry":["UM","S","5\u2032-ACAUGAAGCAGCACGACUUTT-3\u2032","\u20033"]},{"entry":[{},"AS","3\u2032-TTUGUACUUCGUGCUGCUGAA-5\u2032","\u20034"]},{"entry":{}},{"entry":["\u20031","S","5\u2032-ACAUGAAGCAGCACGACUTT-3\u2032","\u20035"]},{"entry":[{},"AS","3\u2032-TTUGUACUUCGUCGUGCUGAA-5\u2032",{}]},{"entry":{}},{"entry":["\u20032","S","5\u2032-ACAUGAAGCAGCACGACTT-3\u2032","\u20036"]},{"entry":[{},"AS","3\u2032-TTUGUACUUCGUCGUGCUGAA-5\u2032",{}]},{"entry":{}},{"entry":["\u20033","S","5\u2032-ACAUGAAGCAGCACGACUUTT-3\u2032","\u20037"]},{"entry":[{},"AS","3\u2032-TTUGUACUUCGUCGUGCUGA-5\u2032",{}]},{"entry":{}},{"entry":["\u20034","S","5\u2032-ACAUGAAGCAGCACGACUUTT-3\u2032","\u20038"]},{"entry":[{},"AS","3\u2032-TTUGUACUUCGUCGUGCUA-5\u2032",{}]},{"entry":{}},{"entry":["\u20035","S","5\u2032-ACUGAAGCACACACUTT-3\u2032","\u20039"]},{"entry":[{},"AS","3\u2032-TTUGUACUUCGUCGUGCUGAA-5\u2032",{}]},{"entry":{}},{"entry":["\u20036","S","5\u2032-CAUGAGCAGCGACUTT-3\u2032","10"]},{"entry":[{},"AS","3\u2032-TTUGUACUUCGUCGUCUGAA-5\u2032",{}]},{"entry":{}},{"entry":["\u20037","S","5\u2032-ACAGAAGCACACGACUTT-3\u2032","11"]},{"entry":[{},"AS","3\u2032-TTUGUACUUCGUCGUGCUGAA-5\u2032",{}]},{"entry":{}},{"entry":["\u20038","S","5\u2032-ACAUGAAGCGCACGACUTT-3\u2032","12"]},{"entry":[{},"AS","3\u2032-TTUGUACUUCGUCGUGCUGAA-5\u2032",{}]},{"entry":{}},{"entry":["\u20039","S","5\u2032-ACUGAAGCGCACGACUUTT-3\u2032","13"]},{"entry":[{},"AS","3\u2032-TTUGUACUUCGUCGUGCUGAA-5\u2032",{}]},{"entry":{}},{"entry":["10","S","5\u2032-ACAUGAAGCAGCACGACUUTT-3\u2032","14"]},{"entry":[{},"AS","3\u2032-TTUUACUCGUCGGCUGA-5\u2032",{}]},{"entry":{}},{"entry":["11","S","5\u2032-ACAUGAAGCAGCACGACUUTT-3\u2032","15"]},{"entry":[{},"AS","3\u2032-TTUGACUUCGCGUGCUGA-5\u2032",{}]},{"entry":{}},{"entry":["12","S","5\u2032-ACAUGAAGCAGCACGACUUTT-3\u2032","16"]},{"entry":[{},"AS","3\u2032-TTGUACUUCGUCGUGCUGA-5\u2032",{}]},{"entry":{}},{"entry":["13","S","5\u2032-ACAUGAAGCAGCACGACUUTT-3\u2032","17"]},{"entry":[{},"AS","3\u2032-TTUGUACUUCGCGUGCUGAA-5\u2032",{}]},{"entry":{}},{"entry":["14","S","5\u2032-ACAUGAAGCAGCACGACUUTT-3\u2032","18"]},{"entry":[{},"AS","3\u2032-TTUGUACUUCUCGUGCUGAA-5\u2032",{}]},{"entry":{}},{"entry":["15","S","5\u2032-ACAUGAAGCAGCACGACUUTT-3\u2032","19"]},{"entry":[{},"AS","3\u2032-TTUGUACUUCCGUGCUGAA-5\u2032",{}]},{"entry":{}},{"entry":["16","S","5\u2032-ACAUGAAGCAGCACGACUUTT-3\u2032","20"]},{"entry":[{},"AS","3\u2032-TTUGUACUUCGUCUGCUAA-5\u2032",{}]},{"entry":{}},{"entry":["17","S","5\u2032-ACAUGAAGCAGCACGACUUTT-3\u2032","21"]},{"entry":[{},"AS","3\u2032-TTUGUACUUCGUCGUGCUAA-5\u2032",{}]},{"entry":{}},{"entry":["18","S","5\u2032-ACAUGAAGCAGCACGACUUTT-3\u2032","22"]},{"entry":[{},"AS","3\u2032-TTUGUACUUCGUCUGCUGAA-5\u2032",{}]},{"entry":{}},{"entry":["19","S","5\u2032-ACAUGAAGCAGCACGACUUTT-3\u2032","23"]},{"entry":[{},"AS","3\u2032-TTUUACUUCUCGUGCUGAA-5\u2032",{}]},{"entry":{}},{"entry":["20","S","5\u2032-ACAUGAAGCAGCACGACUUTT-3\u2032","24"]},{"entry":[{},"AS","3\u2032-TTUUACUCGUCGUGCUGAA-5\u2032",{}]},{"entry":{}},{"entry":["21","S","5\u2032-ACAUGAAGCAGCACGACUUTT-3\u2032","25"]},{"entry":[{},"AS","3\u2032-TTUUACUUCGUCGUGCUGAA-5\u2032",{}]},{"entry":{}},{"entry":["22","S","5\u2032-ACAUGAAGCAGCACGACUUTT-3\u2032","26"]},{"entry":[{},"AS","3\u2032-TTUGUCUUCGUCGUGCUGAA-5\u2032"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},"The silencing efficacy of selected PS2-modified sequences was further validated using another siRNA sequence. See Yang, X. et al. \u201cGene silencing activity of siRNA molecules containing phosphorodithioate substitutions\u201d 7, 1214-1220 (2012).","To investigate the gene silencing activity of MePS2- or FPS2-siRNAs and to gain insight into their regulation of the RNAi pathway, the novel siRNAs were first synthesized via the solid-phase synthesis technique. EPH receptor A2 (EphA2), a tyrosine kinase receptor that promotes tumor progression in cancer, was initially chosen to evaluate the novel MePS2 modified siRNAs using EphA2 as a model system. Informed by observations with the GFP system, MePS2\/FPS2-1 through -3 constructs for siEphA2 (, Table 2) were designed. MePS2\/FPS2-4 sequences were also created having two MePS2\/FPS2 groups present at the seed region of the antisense strand. Lastly, counterpart sequences were synthesized with either 2\u2032-OMe\/2\u2032-F alone, PS2 alone, or MePS\/FPS placed at the same locations.",{"@attributes":{"id":"p-0062","num":"0061"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"280pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u20032"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Chemical\u2003structures\u2003of\u2003modified\u2003siRNAs."},{"entry":"Chemical\u2003structures\u2003of\u2003modified\u2003sense\u2003(S)\u2003and\u2003antisense\u2003(AS)"},{"entry":"\u2003\u2003siRNAs\u2003used\u2003for\u2003the\u2003initial\u2003screen.\u2003Locations\u2003of\u2003modification\u2003"},{"entry":"are\u2003indicated\u2003for\u2003each\u2003sequence."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"119pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Sense\u2003(5\u2032 to\u20033\u2032)","Antisense\u2003(3\u2032 to\u20035\u2032)"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["UM","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},"SEQ.ID.NO.\u200327","SEQ.ID.NO.\u200328"]},{"entry":{}},{"entry":["PS2-1","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},"SEQ.ID.NO.\u200329",{}]},{"entry":{}},{"entry":["PS2-2","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},"SEQ.ID.NO.\u200330",{}]},{"entry":{}},{"entry":["PS2-3","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},{},"SEQ.ID.NO.\u200331"]},{"entry":{}},{"entry":["PS2-4","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},{},"SEQ.ID.NO.\u200332"]},{"entry":{}},{"entry":["MePS2-1","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},"SEQ.ID.NO.\u200333",{}]},{"entry":{}},{"entry":["MePS2-2","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},"SEQ.ID.NO.\u200334",{}]},{"entry":{}},{"entry":["MePS2-3","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},{},"SEQ.ID.NO.\u200335"]},{"entry":{}},{"entry":["MePS2-4","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},{},"SEQ.ID.NO.\u200336"]},{"entry":{}},{"entry":["MePS-1","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},"SEQ.ID.NO.\u200337",{}]},{"entry":{}},{"entry":["MePS-2","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},"SEQ.ID.NO.\u200338",{}]},{"entry":{}},{"entry":["MePS-3","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},{},"SEQ.ID.NO.\u200339"]},{"entry":{}},{"entry":["MePS-4","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},{},"SEQ.ID.NO.\u200340"]},{"entry":{}},{"entry":["Me-1","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},"SEQ.ID.NO.\u200341",{}]},{"entry":{}},{"entry":["Me-2","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},"SEQ.ID.NO.\u200342",{}]},{"entry":{}},{"entry":["Me-3","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},{},"SEQ.ID.NO.\u200343"]},{"entry":{}},{"entry":["Me-4","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},{},"SEQ.ID.NO.\u200344"]},{"entry":{}},{"entry":["FPS2-1","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},"SEQ.ID.NO.\u200345",{}]},{"entry":{}},{"entry":["FPS2-2","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},"SEQ.ID.NO.\u200346",{}]},{"entry":{}},{"entry":["FPS2-3","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},{},"SEQ.ID.NO.\u200347"]},{"entry":{}},{"entry":["FPS2-4","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},{},"SEQ.ID.NO.\u200348"]},{"entry":{}},{"entry":["FPS-1","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},"SEQ.ID.NO.\u200349",{}]},{"entry":{}},{"entry":["FPS-2","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},"SEQ.ID.NO.\u200350",{}]},{"entry":{}},{"entry":["FPS-3","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},{},"SEQ.ID.NO.\u200351"]},{"entry":{}},{"entry":["FPS-4","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},{},"SEQ.ID.NO.\u200352"]},{"entry":{}},{"entry":["F-1","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},"SEQ.ID.NO.\u200353",{}]},{"entry":{}},{"entry":["F-2","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},"SEQ.ID.NO.\u200354",{}]},{"entry":{}},{"entry":["F-3","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},{},"SEQ.ID.NO.\u200355"]},{"entry":{}},{"entry":["F-4","UGACAUGCCGAUCUACAUGTT","TTACUGUACGGCUAGAUGUAC"]},{"entry":[{},{},"SEQ.ID.NO.\u200356"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"SKOV3ip1 and HeyA8, both epithelial ovarian cancer cell lines that highly express EphA2, were transfected with synthesized siRNAs in serum-free conditions to test silencing efficacy. Two independent experiments were performed and EphA2 protein quantification was done at 48 h post-transfection. MePS2-1, MePS2-2, Me-4, FPS-1, FPS-4, and F-1 significantly decreased EphA2 levels in SKOV3ip1 compared to unmodified (UM) siRNA and outperformed their counterparts (). In particular, it was found that the MePS2-1 modification silenced EphA2 to a much greater degree compared to UM (6-fold enhancement). It also showed 4, 7.5, and 7-fold enhancement in EphA2 silencing compared to PS2-1, MePS-1, and Me-1 modified sequences, respectively. In contrast to MePS2-1 and MePS2-2, FPS2-1 and FPS2-2 modifications did not increase silencing efficiency of the siRNA. Given the significant improvement in gene silencing observed with MePS2-1, subsequent efforts were focused on Me-modified constructs. In HeyA8 cells, MePS2-1 modified siEphA2 also showed superior silencing activity compared to the UM sequence (80% vs. 50% knockdown, respectively, ). Interestingly, both PS2-1 and PS2-2 modified sequences resulted in significant silencing of EphA2 in this cell line.","Given the promise of the MePS2-1 modification in both of the cell lines examined, the impact of different modifications at the same position on gene silencing and serum stability (, D) was next investigated. To mimic the condition in the pre-clinical or clinical setting, sequences were introduced into SKOV3ip1 cells in media containing 10% Fetal Bovine Serum (FBS). EphA2 protein levels were examined 48 h post-transfection. MePS2-1 displayed a significant improvement over all other sequences tested, including PS2-1 ( and ). For the data presented in , EphA2 mRNA levels were assessed at 24 h post-transfection. Transfection was performed at 50 nM in SKOV3ip1 cells in the presence of 10% FBS. [P-value obtained with Student's t-test; *, P<0.05, compared to UM siEphA2; bars and error bars represent mean values and the corresponding S.E.M.s (n=3)].","The improved gene-silencing as the result of MePS2-1 modification was also observed when another siRNA sequence was used [parathymosin (PTMS) targeted siRNA, data not shown]. SiRNA stability assays show that the majority of UM siRNA was broken down within 20 min, with near-complete degradation at 3 h (). Overall improvements in serum stability were seen for all modified sequences in comparison to UM, but MePS2-1 displayed the highest sequence stability, with a large portion of the input molecules remaining intact at 3 h. An even greater enhancement in stability was observed with PTMS-targeted siRNA (data not shown).","It was next investigated whether the enhanced stability after modification accounted for these constructs' increased silencing efficacy. To test this, stem loop polymerase chain reaction (PCR) technique was employed to quantify intact siRNA within the cell following transfection (). Increased intracellular siRNA stability via MePS2 modification was confirmed, but not to a degree that would explain its dramatic increase in siRNA potency. It was concluded that MePS2-1 showed increased potency independent of improvements in stability. This was consistent with the finding that the duration of silencing was not significantly prolonged for MePS2-1 sequence compared to the UM counterpart ().","Of note, further modifications at different positions of the MePS2-1 siEphA2 sequence resulted in reduced silencing efficiency (, I).  shows chemical structures of modified siRNAs with the locations of modification indicated for each sequence.  depicts knockdown of EphA2 in SKOV3ip1 cells using the chemically modified siRNAs of  (50 nM, 10% FBS containing media). [P-values obtained with Student's t-test; **, P<0.01; ***, P<0.001; ****, P<0.0001; compared to UM siEphA2; bars and error bars represent mean values and the corresponding S.E.M.s (n=3)].","The mechanism by which the MePS2-1 sequence enhances the gene silencing effect was next examined. First, the thermodynamic stability [measured by RNA duplex melting temperature (Tm)] was examined since the destabilization of the 5\u2032-end of duplex by introducing modified units into 3\u2032-end of the sense strand is important in creating favorable thermodynamic asymmetry for siRNA activity. Measuring the Tm of MePS2-1 and its counterpart sequences showed MePS2-1 to have the largest decrease in Tm (1.24\u00b0 C., ATm, P<0.05) when compared to UM siRNA, suggesting duplex unwinding may have a role in its enhanced RNAi activity (Table 3). Importantly, in contrast to 2\u2032-OMe modification alone which increased Tm, its addition at the site of PS2 modification did not elevate Tm.",{"@attributes":{"id":"p-0069","num":"0068"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Thermostability of chemically modified siRNAs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"siRNA","Average Tm",{},{}]},{"entry":[{},"sequence","(\u00b0 C.)","Std. Dev. (\u00b0 C.)","\u0394Tm (\u00b0 C.)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]},{"entry":[{},"UM","76.51","0.64","N\/A"]},{"entry":[{},"Me-1","76.68","0.50","0.17"]},{"entry":[{},"PS-1","75.91","0.65","\u22120.60"]},{"entry":[{},"MePS-1","76.37","0.08","\u22120.14"]},{"entry":[{},"PS2-1","75.82","0.50","\u22120.69"]},{"entry":[{},"MePS2-1","75.27","0.07","\u22121.24"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}}]}}},"Investigations were made into whether the MePS2 modification altered helix conformation between the target and guide strand by changing local structure, which can affect silencing activity. However, circular dichroism (CD) spectra of all the modified sequences tested were similar to the spectrum of UM duplexes and are consistent with the typical A-type structure (a maximum of the positive Cotton effect at 268 nm and a crossover point at 240 nm) ().","Investigations were undertaken into whether the thiol groups on MePS2-1's phosphate backbone were interacting with metal ions or amino acid residues in RISC to increase the sequence's affinity for the complex. This hypothesis was tested by transfecting UM, PS2-1, and MePS2-1 into SKOV3ip1 cells, then immunoprecipitating argonaute-2 (Ago2). SiEphA2 with two 2\u2032-OMe residues at the 5\u2032-end of the sense strand (Me2-siEphA2) was used as a positive control. From a panel of 10 different microRNAs (miR) tested, hsa-miR-484 was found to be both unaffected by siEphA2 and present at a high level in SKOV3ip1 cells, thus it was chosen as a normalizing control. It was found that Ago2-associated MePS2-1 and PS2-1 sequences were enriched 7-fold and 3-fold compared to UM siRNA, respectively (). In contrast to UM siRNA where the level of associated Ago2 protein was below the limit of detection via Western blotting, a high level of siRNA-associated Ago2 was detected for both PS2-1 and MePS2-1 siRNAs in the biotin-siRNA pull-down study (). In both of these experiments, there was a significant increase in Ago2 binding for MePS2-1 compared to PS2-1 sequence (>2-fold). The ability of MePS2-1 modification to enhance loading of siRNAs into RISC was also demonstrated using another siRNA sequence (data not shown). Of note, the enhanced binding of MePS2-1 siRNAs to RISC had minimal effect on miR biogenesis.","The impact of MePS2-1 modification on strand loading was further investigated. SKOV3ip1 cells were transfected with siEphA2 sense (S) and antisense (AS) strand-specific luciferase reporters. MePS2-1(S) decreased the luciferase signal to a much greater degree (2.2-fold enhancement) compared to UM siRNA in cells transfected with the AS reporter construct. As expected, MePS2-1(AS) did not reduce the luciferase signal significantly when compared to control siRNA (siCon) treatment. Conversely, in cells transfected with the S reporter construct, MePS2-1(AS) further improved the silencing ability of the siRNAs when compared to the UM sequence, while no silencing was observed with MePS2-1(S).","The silencing efficacy of MePS2-1(S) was compared with Me2(S)-siRNA. In cells transfected with the AS reporter construct, Me2(S) siRNAs decreased the luciferase signal to a greater extent compared to UM, as expected. This extent of silencing was comparable to that achieved by MePS2-1(S). The addition of MePS2-1(S) modification to Me2(S)-siRNA (Me2MePS2-1) did not significantly alter its silencing efficiency. These results are consistent with the level of EphA2 mRNA knockdown observed with these siRNAs.","Computational modeling was next used to examine the potential effects of the MePS2 modification on the interaction of siRNAs with RISC, focusing on the PAZ domain that recognizes 3\u2032-overhanging nucleotides of siRNA duplexes. In the crystal structure, the siRNA 5\u2032-(CGUGACU)-d(CU)-3\u2032 forms a duplex with one G-G and two U-C mismatch pairs and 3\u2032-terminal d(CU) overhangs. Modeling entailed changing the penultimate 3\u2032-dC to T, converting the terminal RNA U-C pair in the native duplex to G:C, and changing the adjacent G:C pair to A:U. Methyl groups were also added to the sugar 2\u2032-oxygen atoms of U and G in the antiperiplanar orientation (torsion angle C3\u2032-C2\u2032-02\u2032-C). Inspection of the model of MePS2-1 bound to PAZ indicates that the combination of the 2\u2032-OMe and PS2 modifications will likely benefit the interaction between siRNA and PAZ (). Thus, the proximities of the 2\u2032-OMe moiety of G and C\u03b5 of M273 as well as the PS2 moiety of T and both C\u03b5 of Met273 and the sulfhydryl group of C270 are consistent with favorable hydrophobic interactions that are absent in the complex with the UM siRNA and are superior to those for the Me-1 and PS2-1 siRNAs. Taken together, the combination of PS2 and 2\u2032-OMe modification triggers an unusual stereoelectronic effect in the backbone of the siRNA, which leads to increased RISC loading. This, along with an increase in resistance to nuclease degradation as well as a decrease in Tm, provides a likely explanation for enhanced RNAi activity by MePS2-1 modification.","Having identified a promising chemical modification that could significantly enhance the stability and silencing activity of siRNAs, its potential off-target immunotoxicity was next characterized. C57BL\/6-derived dendritic cells were treated with UM control siRNA (siCon), UM siEphA2, MePS2-1 siCon, and MePS2-1 siEphA2. Interferon alpha (IFN-\u03b1) levels, a surrogate marker for non-specific induction of innate immunity, were similar for all four sequences examined (). No difference in Toll like receptor 7 (TLR7)-induced IFN-\u03b1 or tumor necrosis factor alpha (TNF-\u03b1) levels was observed between UM siEphA2 and MePS2-1 siEphA2 at all time-points examined. Of note, serum IFN-\u03b1 and TNF-\u03b1 levels were both undetectable at 2, 6, and 24 h following intraperitoneal (i.p.) administration of 1,2-dioleoyl-sn-glycero-3-phosphatidylcholine (DOPC) nanoliposomes-delivered UM-siEphA2 (UM-siEphA2-DOPC) or MePS2-1 siEphA2 (MePS2-1-siEphA2-DOPC) in mice.","The efficacy of the MePS2-1 siRNA in vivo was next assessed using an orthotopic mouse model of ovarian cancer (SKOV3ip1). The effect of EphA2 knockdown in tumors was first examined following a single injection. While some degree of knockdown was achieved with MePS2-1-siEphA2-DOPC at the dose of 1.25 \u03bcg, (), >2.5 \u03bcg per dose was determined to be necessary to achieve consistent and significant gene silencing in tumors. At the dose of 2.5 \u03bcg, measurement of EphA2 by Western blot of tumor lysate revealed superior gene silencing effect of MePS2-1-siEphA2-DOPC compared to UM-siEphA2-DOPC (80% vs. 30% reduction, respectively). Mice were next treated with 2.5 \u03bcg of UM-siCon-DOPC, UM-siEphA2-DOPC, MePS2-1-siCon-DOPC, and MePS2-1-siEphA2-DOPC twice weekly (n=10). After four weeks of therapy, the animals were sacrificed and necropsies were performed. UM-siEphA2-DOPC reduced the tumor burden compared to UM-siCon-DOPC group (50% reduction), however, the difference is not statistically significant. In contrast, MePS2-1-siEphA2-DOPC exhibited significant reduction in tumor weight (, 70% reduction) compared to the MePS2-1-siCon-DOPC treatment group. No group showed decreased body weight, one major indicator of toxicity (). Overall, these results demonstrate superior gene silencing and therapeutic effects with MePS2-1-siEphA2-DOPC compared to UM-siEphA2-DOPC.","The pharmacokinetic profile and biodistribution pattern of MePS2-1-siEphA2-DOPC was next evaluated following a single i.p. injection in tumor-bearing mice. The level of siRNAs accumulated in tumors was found to be significantly higher than that observed in other major organs, including lung, heart, liver, spleen, kidneys, and brain (). The localization of DOPC nanoliposomes to tumors was further confirmed using Cy5.5-labeled siRNAs. This ability of DOPC nanoliposomes to preferentially deliver siRNAs to tumors while avoiding significant uptake in other organs is similar to that observed with several other systems, including pRNA nanoparticles. The pharmacokinetic analysis of MePS2-1-siEphA2-DOPC showed an Area Under the Curve (AUC)0-72 of 477 pg*h mL-1, Clearance (CL) of 5.23 L h\u22121, and Volume of distribution (Vd) of 26.3 L. Distribution half-life of MePS2-1-siEphA2-DOPC was 3.7 min and the elimination half-life was 9.3 h.","Having demonstrated that MePS2-1 siRNA has superior gene silencing activity compared to its UM counterpart for an established target, its silencing, and in turn therapeutic capability for a novel ovarian cancer target gene were next assessed. Since acquired chemoresistance is a major contributor to patient mortality from ovarian cancer (OvCa), targets playing substantial roles in this process are important targets. SiRNAs against approximately 22,000 individual genes were evaluated for their lethality and ability to induce taxane sensitivity in taxane-resistant epithelial ovarian cancer cells (SKOV3-TR) (data not shown). High correlation between cell viability score was observed across triplicate plates (median coefficient of variation=8.6%), indicating high data precision in the screen.","A total of 178 genes were identified to have reduced cell viability upon siRNA treatment and showed further cell death when combined with paclitaxel (data not shown). Importantly, the dose of paclitaxel used in the screen was IC30 of that achieved in parental taxane-sensitive SKOV3ip1 cells to permit identification of genes that, when silenced, can dramatically sensitize cells to taxane treatment. Out of 178 hits, eight genes were identified to be up-regulated in ovarian cancer epithelial cells compared to normal ovarian surface epithelial cells. Within this group, three genes associated with the highest lethality upon knockdown when combined with paclitaxel were focused on: GRAMD1B, RBBP6, and SLC23A1. Taxane-resistant ovarian cancer cell lines (HeyA8-MDR, SKOV3-TR) showed elevated levels of GRAMD1B mRNA and protein relative to their sensitive counterparts (HeyA8, SKOV3ip1). The expression of GRAMD1B remained unchanged for platinum resistant A2780-CP20 cells when compared to the parental line (A2780), indicating the lack of involvement of this gene in platinum resistance. GRAMD1B was also confirmed to be highly elevated in HeyA8 and SKOV3ip1 compared to the non-transformed HIO-180 ovarian epithelial cells. RBBP6 and SLC23A1 mRNA expression levels, though elevated in some ovarian cancer lines, were not found to be increased in resistant cell lines examined.","The effect of silencing GRAMD1B in the taxane-resistant cell line with the highest target expression was next examined, HeyA8-MDR (, B, C). GRAMD1B silencing via introducing siGRAMD1B into cells using Lipofectamine 2000 exhibited both lethality as monotherapy (40% decrease in cell viability and 3-fold increase in apoptosis,  A, B) and also taxane sensitization at 10 nM paclitaxel (>75% vs. <5% cell death, siGRAMD1B plus paclitaxel vs. siCon plus paclitaxel) (). Treatment with docetaxel produced a similar pattern. Significant taxane sensitization was also observed when another GRAMD1B-targeting siRNA was transfected into cells using another transfecting reagent, Lipofectamine RNAiMax. These results indicate the potential of GRAMD1B silencing in overcoming taxane resistance in ovarian cancer.","Having established the importance of targeting GRAMD1B in ovarian cancer treatment, the ability of MePS2-1 modification to significantly enhance the silencing efficiency of UM siGRAMD1B was examined. It was found that MePS2-1 siGRAMD1B efficiently silenced GRAMD1B both at the mRNA (70%, data not shown) and protein (85%) level in HeyA8-MDR cell line in the presence of serum (40 nM, ). This level of GRAMD1B knockdown was significantly better than that achieved by UM siGRAMD1B (85% vs. 50%). Consistent with findings using siEphA2 and siPTMS, MePS2-1 modification also significantly enhances the serum and intracellular stabilities of siGRAMD1B (, E) and increases its binding to Ago2 protein when compared to UM siGRAMD1B (). Let-7a was used as a normalizing control for the Ago2 binding study as its expression is not altered by GRAMD1B silencing.","Moving into an orthotopic mouse model, HeyA8-MDR cells were injected into the peritoneal cavity and, after one week, mice were divided into 4 treatment groups: (a) MePS2-1-siCon-DOPC, (b) MePS2-1-siGRAMD1B-DOPC, (c) MePS2-1-siCon-DOPC plus paclitaxel, or (d) MePS2-1-siGRAMD1B-DOPC plus paclitaxel. Dosing was twice weekly for siRNA and weekly for paclitaxel (at a sub-therapeutic 100 \u03bcg dose), followed by tumor excision and quantification after five weeks of treatment. No significant reduction in tumor burden was observed for sub-therapeutic paclitaxel with MePS2-1-siCon-DOPC treatment group, as expected (). However, MePS2-1-siGRAMD1B-DOPC alone resulted in 50% reduction in tumor burden, consistent with in vitro findings (). Combining it with paclitaxel induced a 55% further reduction in total tumor weight compared to MePS2-1-siGRAMD1B-DOPC monotherapy, indicating the potential for taxane sensitization after GRAMD1B silencing. The reduction of GRAMD1B levels in tumors by MePS2-1-siGRAMD1B-DOPC was significant, with the protein level being down-regulated by 80% in tumors from mice treated with MePS2-1-siGRAMD1B-DOPC. This down-regulation of GRAMD1B in tumors was also significantly associated with the induction of cleaved caspase 3. This is consistent with the increased level of apoptotic cells observed following GRAMD1B silencing in vitro (). Replicating the experiment with a different GRAMD1B-targeted siRNA sequence in SKOV3-TR, another taxane-resistant ovarian cancer model, produced a similar therapeutic effect. In this model, mice treated with both MePS2-1-siGRAMD1B-DOPC and paclitaxel had an 80% further decrease in tumor burden compared with mice treated with MePS2-1-siGRAMD1B-DOPC alone, which again indicated the role of GRAMD1B in taxane resistance. Importantly, in both of the models tested, there was no alteration in mouse body weights in any of the treatment groups.","Cell Lines:","The ovarian cancer cell lines, SKOV3ip1 and HeyA8, were obtained from ATCC and were maintained in RPMI 1640 supplemented with 15% fetal bovine serum and 0.1% gentamicin sulfate (Gemini Bioproducts; Calabasas, Calif.). SKOV3-TR and HeyA8-MDR were kind gifts from Dr. Isaiah Fidler, Department of Cancer Biology, University of Texas M. D. Anderson Cancer Center and Dr. Michael Seiden, Department of Medicine, Massachusetts General Hospital, respectively.","SiRNA Synthesis:","SiRNAs were synthesized at AM Biotechnologies. Modified and unmodified 21-nt RNAs (sense and antisense strands) were synthesized on the 1 \u03bcmole scale on an Expedite 8909 DNA\/RNA Synthesizer using commercially available 5\u2032-DMT-2\u2032-O-TBDMS nucleoside (A, C, G, and U) phosphoramidite monomers and 5\u2032-DMT-2\u2032-O-Methyl nucleoside (A, C, G, and U) phosphoramidite monomers as well as in house produced 5\u2032-DMT-2\u2032-O-Methyl nucleoside (A, C, G, and U) thiophosphoramidite monomers. See Yang, X. et al. \u201cGene silencing activity of siRNA molecules containing phosphorodithioate substitutions\u201d 7, 1214-1220 (2012); Wiesler, W. T. & Caruthers, M. H. \u201cSynthesis of phosphorodithioate DNA via sulfur-linked, base-labile protecting groups\u201d 61, 4272-4281 (1996); Jackson, A. et al. \u201cPosition-specific chemical modification of siRNAs reduces \u2018offtarget\u2019 transcript silencing\u201d RNA 12, 1197-1205 (2006).","Preparation of the Thiophosphoramidites:","5\u2032-O-(4,4\u2032-Dimethoxytrityl)-2\u2032-O-(tert-butyldimethylsilyl) ribonucleosides were phosphitylated by means of tris-(pyrrolidino)phosphine (130 \u03bcL, 1.0 mmol) in the presence of 1H-tetrazole (0.35 mmol) in anhydrous dichloromethane (15 mL) containing a spatula of 3 \u212b molecular sieves for 15 min at RT. Trimethylsilylimidazole (15 \u03bcL, 0.1 mmol) was then added to the reaction mixture, and the reaction was stirred for 5 min. The resulting bis(pyrrolidino)phosphite intermediates were converted into the desired thiophosphoramidites (, a-d) by treatment in situ with monobenzoylethanedithiol (220 \u03bcL, 1.3 mmol) and additional 1H-tetrazole (2.7 mmol). After an aqueous workup to remove 1H tetrazole and tetrazolide salts, the fully protected RNA thiophosphoramidites (-) were isolated by precipitation from hexane in 75-85% yield at more than 90% purity as assessed by P NMR. (B=Ade): yield, 85%, P NMR, \u03b4=166.7, 174.2 ppm. (B=Cyt): yield, 75%, P NMR, \u03b4=168.4, 175.2. ppm. (B=Gua): yield, 87%, P NMR, \u03b4=163.8, 175.3 ppm. (B=Ura): 77%, P NMR, \u03b4=170.01, 177.1 ppm.","Synthesis of RNA Containing PS2 Substitutions.","All PS2-modified RNAs were synthesized on 1000 \u212b controlled pore glass T support (resulting in a thymidine deoxyribonucleoside at the 3\u2032 end). The standard 2\u2032-O-TBDMS RNA phosphoramidites were used for incorporation of A, C, G, and U residues. Each standard nucleotide was coupled using approximately 120 \u03bcL of a 0.1 M solution of the appropriate phosphoramidite in anhydrous acetonitrile. Each PS2 substitution was coupled using approximately 250 \u03bcL of a 0.15 M solution of the appropriate thiophosphoramidite in anhydrous acetonitrile, except that G thiophosphoramidite () was prepared in anhydrous acetonitrile containing 10% anhydrous dichloromethane. All coupling times were 10 min. A 0.05 M solution of EDITH reagent in anhydrous acetonitrile was used as a sulfurizing agent to oxidize the internucleosidic thiophosphite to the phosporodithiotriester. See Ma, M. Y., et al. \u201cEvaluation of 3-ethoxy-1,2,4-dithiazoline-5-one (EDITH) as a new sulfurizing reagent in combination with labile exocyclic amino protecting groups for solid phase oligonucleotide synthesis\u201d 25 (1997) 3590-3593.5.","A solution of 0.02 M Iin THF\/pyridine\/water was used to oxidize internucleosidic phosphites to phosphorotriesters. The average stepwise coupling efficiencies of all thiophosphoramidites were about 97% as estimated by the dimethoxytrityl cation assay. After completion of the synthesis, the solid support was suspended in ammonium hydroxide\/methylamine (AMA) solution (prepared by mixing 1 volume of ammonium hydroxide (28%) with 1 volume of 40% aqueous methylamine) and heated at 65\u00b0 C. for 15 min to release the product from the support and to complete the removal of all protecting groups except the TBDMS group at the 2\u2032-position. The solid support was filtered, and the filtrate was concentrated to dryness.","The obtained residue was resuspended in 115 \u03bcL of anhydrous DMF and then heated for 5 min at 65\u00b0 C. to dissolve the crude product. Triethyl amine (TEA\u201460 \u03bcL) was added to each solution, and the solutions were mixed gently. TEA.3HF (75 \u03bcL) was added to each solution, and the tubes were then sealed tightly and incubated at 65\u00b0 C. for 2.5 h. The reaction was quenched with 1.75 mL of DEPC-treated water. The samples were analyzed by UV-Vis spectroscopy, and the OD readings were recorded.","Purification and Characterization of RNAs and PS2-RNAs:","Purification was performed on an Amersham Biosciences P920 FPLC instrument fitted with a Mono Q 10\/100 GL column. The buffers were prepared with DEPC-treated water, and their compositions were as follows: Buffer A: 25 mM Tris-HCl, 1 mM EDTA, pH 8.0; Buffer B: 25 mM Tris-HCl, 1 mM EDTA, 1 M NaCl, pH 8.0. The FPLC gradient profile was as follows: 0-100% B over 50 min, 100% B for 5 min, 100-0% B over 5 min. Purified fractions were pooled and desalted via RP-HPLC on an Amersham Biosciences P900 fitted with a PRP-1 Column (Hamilton Co., Reno, Nev.). The buffers used were as follows: Buffer A: 0.1 M NH4OAc in DEPC-treated water, pH 7.0; Buffer B: HPLC grade acetonitrile. The gradient run was as follows: 0-35% B over 25 min, 35-0% B over 1 min. After desalting, the samples were frozen at \u221280\u00b0 C. for 1 h, lyophilized, and dissolved in DEPC-treated water. All the RNA, PS-RNA and PS2-RNA oligonucleotides were characterized by denaturing polyacrylamide gel electrophoresis. Representative PS2-RNAs were further characterized by ESI-MS and 31P-NMR.","Preparation of SiRNA Duplexes.","Assembly of siRNA duplexes was performed in phosphate buffered saline (PBS) by heating the equivalent mixture of RNA oligonucleotides coding the sense and antisense strands of siRNA at 95\u00b0 C. for 2 min followed by slow cooling to room temperature (over 2 h). The assembly of the resulting duplexes was confirmed by a 4% agarose gel electrophoresis. For in vivo experiments, ammonium counter cation was replaced by sodium cation.","In vitro siRNA transfection and luciferase assays: Cells were transfected with 10-100 nmol Lof specified siRNAs using Lipofectamine 2000 or Lipofectamine RNAiMax reagent (Invitrogen) at 3 \u03bcL reagent: 1 \u03bcg siRNA ratio. Cells were treated with siRNAs for 4 h in serum-free or 10% FBS-containing media before incubation in fresh complete media for the specified timeframe. GoClone pLightSwitch luciferase reporter constructs containing AS and S siRNA target sequences were obtained from SwitchGear genomics (Menlo Park, Calif.).","Antisense (AS) construct contains the following sequence:",{"@attributes":{"id":"p-0099","num":"0098"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"right"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"SEQ.\u2003ID.\u2003NO.\u200357"},{"entry":"5\u2032-CAAAGGGTGGGACCTGATGCAGAACATCATGAATGACATGCCGATCT"},{"entry":"ACATGTACTCCGTGTGCAACGTGATGTCTGGCGAC-3\u2032.."}]}}}}},"Sense (S) construct contains the following sequence:",{"@attributes":{"id":"p-0101","num":"0100"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"right"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"SEQ.\u2003ID.\u2003NO.\u200358"},{"entry":"5\u2032-ACCTGATGCAGAACATCATGAACATGTAGATCGGCATGTCATACTCC"},{"entry":"GTGTGCAACGTGATGTCT-3\u2032.."}]}}}}},"SKOV3ip1 cells were transfected with FuGENE HD reagent in a 96-well plate with specified siRNAs (100 nM) along with S or AS reporter construct, and Cypridina TK control construct (pTK-Cluc). After 24 h of transfection, luciferase activity was obtained using LightSwitch Dual Luciferase assay kits. Luciferase activity was normalized with the Cypridina TK control construct. All assays were performed in triplicate.","Quantitative Reverse Transcription-Polymerase Chain Reaction:","RNA was isolated from cells, Ago2 pull down products, blood samples, and tissues using Trizol (Invitrogen) according to the manufacturer's protocol. Using 1 \u03bcg of RNA, cDNA was synthesized by using a Verso cDNA kit (Thermo Scientific) as per the manufacturer's instructions. Analysis of mRNA levels was performed on a 7500 Fast Real-Time PCR System (Applied Biosystems) with SYBR Green-based qRT-PCR. PCR was performed with reverse-transcribed RNA and 100 ng \u03bcL-1 of forward and reverse primers in a total volume of 20 \u03bcL. Each cycle consisted of 15 s of denaturation at 95\u00b0 C. and 1 min of annealing and extension at 60\u00b0 C. (40 cycles). Stem loop PCRs for siRNA detection were performed using TaqMan miRNA assays followed by SYBR Greenbased qRT-PCR41. For miRNA quantification, total RNA was isolated using Trizol (Invitrogen) extraction. TaqMan miRNA assays (Applied Biosystems) were used for reverse transcription and qRT-PCR was performed according to the manufacturer's instructions. For microRNA detection, Ambion assay probe sets were used according to the manufacturer's protocol. The relative amount of mRNA\/siRNA in each sample was normalized to \u03b2-actin mRNA, RNU44, RNU6B, or respective microRNAs.","Melting Profiles of siRNAs:","Certain of the absorption measurements were performed in a 1-cm path length cell with a Cintra 4040 spectrophotometer equipped with a Peltier Thermocell (GBC, Dandenong, Australia). Complementary RNA strands were mixed in 10 mM Tris-HCl buffer (pH 7.4) with 100 mM NaCl, and 0.1 mM Ethylenediaminetetraacetic acid (EDTA) at a final concentration of 2 \u03bcM (0.8 OD of both mixed strands). Annealing was performed at a temperature gradient of 1.5\u00b0 C.\/min, from 85 to 15\u00b0 C. Melting profiles were measured using a temperature gradient of 1\u00b0 C.\/min, from 15 to 85\u00b0 C., with the detector set at 260 nm. Each Tm reported was an average of values from three to six independent experiments.","Circular Dichroism Measurements:","Circular Dichroism (CD) spectra were recorded on a CD6 dichrograph (Jabin-Yvon) using cells with 0.5 cm path length, 2 nm bandwidth, and 1-2 s integration time. Each spectrum was smoothed with a 25-point algorithm. The spectra from 200-340 nm were recorded at 25\u00b0 C. in the same buffer as in the melting experiments (n=3). The concentration of the two complementary RNA oligonucleotides was ca. 2 \u03bcM.","Western Blot:","Protein lysates were prepared from cultured cells or tumors using modified RIPA buffer (50 mM Tris-HCl (pH 7.4), 150 mM NaCl, 1% Triton, 0.5% deoxycholate) plus 25 \u03bcg mLleupeptin, 10 \u03bcg mLaprotinin, 2 mM EDTA, and 1 mM sodium orthovanadate. Membranes were probed with primary antibodies against EphA2 (1:1000, Millipore, 05-480), \u03b2-actin (1:5000, Sigma, A5316), vinculin (1:2500, Sigma, V9131), GAPDH (1:5000, Sigma, G8795), Ago2 (1:100, Wako Chemicals, 015-22031), and GRAMD1B (1:500, Sigma, HPA008557) in 5% skim milk in TBS-T. Quantification was performed using ImageJ.","Intracellular Ago2 Binding Study:","Cells were lysed at 7 h post-transfection using ice-cold lysis buffer (50 mM Tris-HCl at pH 7.5, 200 mM NaCl, 0.5% Triton, 2 mM EDTA, 1 mg mLheparin, and protease inhibitor cocktail). Ago2 antibody (Wako Chemicals, 015-22031, 20 \u03bcL) was absorbed onto magnetic protein G Dynabeads (Invitrogen, 40 \u03bcL) by incubation at 4\u00b0 C. for 2 h on a rotator. The beads were then washed twice with lysis buffer to remove unbound antibodies. Following washing, beads were incubated with protein lysate (360 \u03bcg) overnight at 4\u00b0 C. on a rotator. Following removal of unbound proteins, bound material was eluted from beads using 50 \u03bcL of 0.1 M glycine (pH 2.3) for 15 min at RT. Eluted fractions were neutralized immediately with an equal volume of 1 M Tris-HCl (pH 8) and then treated with 20 U of proteinase K for 10 min at 65\u00b0 C. The amount of Ago2-bound siRNA was then quantified using stem loop PCR technique. For the biotin-siRNA pull down experiment, strepavidin beads were blocked and incubated with cell lysates at 4\u00b0 C. for 2 h. The amount of siRNA-associated Ago2 was quantified via Western blotting.","Serum Stability Assay:","SiRNAs (1.33 \u03bcg) were incubated in 10 \u03bcL, of 10% FBS in PBS at 37\u00b0 C. for up to 24 h. All samples were separated in 20% polyacrylamide gels. The percentage of intact siRNA present was quantified using ImageJ.","Immunogenicity of siRNAs:","Bone marrow progenitor cells were isolated from C57BL\/6 mice and were differentiated into dendritic cells in vitro for 7 days using FLT3 ligand (100 ng mL; PeproTech). Cells were treated with 75 nM of siRNAs for 6 h in serum-free media using RNAiMax\u2122. A high interferon-inducing siRNA sequence (BP1Mod2) and CpG2216 (1 \u03bcg mL-1, Sigma) were used as positive controls. The level of IFN-\u03b1 and TNF-\u03b1 in the culturing media was assessed using murine IFN-\u03b1 and TNF-\u03b1 ELISA kits at 2, 6, and 24 h following transfection (R&D Systems). For TLR7 experiments, peripheral blood mononuclear cells (PBMCs) were obtained from wild-type and TLR7-KO C57BL\/6 mice and were treated with 75 nM of siRNAs for 6 h. IFN-\u03b1 and TNF-\u03b1 levels in the culturing media was measured 24 h post-transfection. For in vivo studies, serum was collected from mice at 2, 6, and 24 h post siRNA treatment and IFN-\u03b1 and TNF-\u03b1 levels were measured as described above.","Liposomal Nanoparticle Preparation:","SiRNAs for in vivo delivery were incorporated into DOPC liposomes essentially according to the method of Landen, C. J. et al. \u201cTherapeutic EphA2 gene targeting in vivo using neutral liposomal small interfering RNA delivery\u201d 65, 6910-6918 (2005). DOPC and siRNA were mixed in the presence of excess tertiary butanol at a ratio of 1:10 (w\/w) siRNA\/DOPC. Tween 20 was added to the mixture in a ratio of 1:19 Tween 20:siRNA\/DOPC. The mixture was vortexed, frozen in an acetone\/dry ice bath, and lyophilized. Before in vivo administration, this preparation was hydrated with PBS.","Orthotopic In Vivo Models of Ovarian Cancer:","Female athymic nude mice (8-12 weeks old) were obtained from the National Cancer Institute, Frederick Cancer Research and Development Center (Frederick, Md.). All mouse studies were approved and supervised by the M.D. Anderson Cancer Center Institutional Animal Care and Use Committee. For therapeutic experiments, ten mice were assigned per treatment group. This sample size gave 80% power to detect a 50% reduction in tumor weight with 95% confidence. To establish intraperitoneal tumors, cells were injected into the peritoneal cavity (i.p.) at a concentration of 5\u00d710cells mL(200 \u03bcL per injection). All mice were treated with siRNAs (2.5 or 5 \u03bcg for EphA2 and GRAMD1B experiments, respectively) twice weekly beginning at one week following tumor cell injection. Paclitaxel (100 \u03bcg) was injected i.p. once weekly. Mice (n=10 per group) were monitored for adverse effects and tumors were harvested after 5 weeks of therapy or when any of the mice began to appear moribund. Mouse weight, tumor weight, and the number\/location of tumor nodules were recorded. Tumor tissues were fixed in formalin for paraffin embedding or were snap frozen for lysate preparation. For gene silencing (1.25 and 2.5 \u03bcg), immune toxicity (5 \u03bcg), pharmacokinetic (2.5 \u03bcg), and biodistribution studies (2.5 \u03bcg), a single dose of siRNA was administered i.p. once tumors became palpable. Blood and\/or tissues samples were collected at specified time points for RNA\/protein extraction or ELISA assay. For the fluorescence-based biodistribution study, Cy5.5-labeled siRNAs were administered i.p. and 48 h later, fluorescence imaging of excised tumor and organs was performed using the Xenogen IVIS 200 system. Cy5.5 fluorophore excitation (678 nm) and emission (703 nm) filters were used. Using Living image 2.5 software, regions of interest were drawn for each organ and the total radiant efficiency ps\u03bcWcmwas measured.","Pharmacokinetic Analysis:","Blood concentrations of siRNA at the indicated times, C(t), were expressed as pg siRNA per mL of blood. Pharmacokinetic parameters were determined by fitting the C(t) with a two-compartment model using the MULTI program. The area under the blood concentration-time curve (AUC) was calculated by integration of C(t) up to 72 h following administration. Vd was calculated by dividing the injected dose by C(0). CLtotal was calculated by dividing injected dose by AUC.","Microarray and Pathway Enrichment Analysis:","HeyA8-MDR cells were treated with siGRAMD1B and total RNA was extracted 48 h following transfection using mirVana RNA isolation labeling kit (Ambion). Five hundred nanograms of total RNA were used for labeling and hybridization on a Human HT-12 v4 Beadchip (Illumina) according to the manufacturer's protocols. After the bead chips were scanned with an Illumina BeadArray Reader (Illumina), the microarray data were normalized using the quantile normalization method in the Linear Models for Microarray Data (LIMMA) package in the R language environment. The expression level of each gene was transformed into a log 2 base before further analysis. GEOID for the microarray data is GSE54459. The top ten pathways perturbed by siGRAMD1B treatment were identified using NetWalker software. Each interaction in the global network of biological relationships was scored based on combined assessment of the network connectivity and the input data. Biological pathways were ranked based on geometric P-values.","SiRNA Lethality Screen:","SKOV3-TR cells used for the screen were maintained in antibiotic-free RPMI1640 supplemented with 10% fetal bovine serum48 and were authenticated prior to their utilization in the screen. For the screen, SKOV3-TR (2,000 cells per well) were transfected using siRNAs in individual wells of black 384-well plates for 48 h. Transfection was performed using a pool of 4 individual siRNAs targeting individual genes (siGenome Smart pool, G-004605, Dharmacon) using Dharmafect 1 (Dharmacon). SiTOX was used as a control for transfection efficiency. Non-targeting siRNA number 4 (siOTP4, Dharmacon) was used as a transfection control in all experiments. In summary, 0.2 \u03bcL of 20 \u03bcM siRNA was mixed with 0.2 \u03bcL of Dharmafect 1 and 20 \u03bcL of Optimem medium (Invitrogen) for 20 min in a well of 384-well plate and overlaid with 80 \u03bcL of cell suspension (2,000 cells) in antibiotic-free RPMI1640 supplemented with 10% fetal bovine serum. This gave a final siRNA concentration of 40 nM per well. For each plate, 4 types of control were included: siOTP4 as a negative control (16 wells), siTOX as a control for transfection efficiency (8 wells), cells only (24 wells), and blank wells (16 wells). Following siRNA treatment, cells were treated with paclitaxel (3.5 nM). All experiments were performed at 37\u00b0 C. in the presence of 5% CO2. Finally, cell viability was determined using 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide (MTT, USB Corporation) as per manufacturer's instruction three days after treatment and signal was detected using a PHERAstar FS plate reader (BMG Labtech). Following data collection, the values in each plate were centered on 100 using median scaling and cell viabilities were expressed as percentages of siOTP4 treated samples. Student's t-test was used to examine chemosensitization following conversion of raw values to log 2 scale.","Cell Viability Assays:","HeyA8-MDR cells were transfected with siRNAs (100 nM) on day 1 and 3 before paclitaxel or docetaxel treatment. Cell viability was assessed three days following taxane treatment using MTT assay as per manufacturer's protocol.","Statistical Analysis:","Kaplan-Meier plots were constructed and a log-rank test was used to determine differences among survival curves according to tumoral GRAMD1B expression level. Contingency tables and Fisher's exact test or Chi-square test were used to evaluate the relationship between death and categorical variables. Multivariate analyses were performed with the use of a Cox proportional-hazards model to examine the effects of tumoral GRAMD1B expression on death from disease while adjusting for other covariates. Chi-square test was performed to assess the effect of tumoral GRAMD1B expression on drug response. For siRNA lethality screen, student t-test was used to examine chemosensitization following conversion of raw values to log 2 scale. For other assays, student's t-test was performed to examine the difference between control and treatment groups.","Apoptosis Assay:","Apoptosis was studied using the Annexin V apoptosis detection kit (BD Biosciences) according to the manufacturer's protocol. Apoptotic cells were analyzed using a FACSCalibur flow cytometer (BD Biosciences). CellQuest Pro software (BD Biosciences) was used to determine the number of apoptotic cells.","Modeling of siRNA:PAZ Domain Interactions:","Coordinates of the complex between a 9 mer siRNA and the PAZ domain from human Argonaute eIF2c1 were retrieved from the Protein Data Bank (http:\/\/www.rcsb.org; PDB ID code 1si2). The program UCSF Chimera was used to convert the sequence of the last four nucleotides in the siRNA from the 1si2 crystal structure to match the sequence and chemistry of MePS2-1 ( . . . ACUdCdU-3\u2032 and . . . AUMeS2GMeS2dTdT-3\u2032, respectively).","Changes to reflect the chemistries of the Me-1 and PS2-1 siRNAs were accomplished in a similar manner. Side chain torsion angles of C270 and M273 were adapted in USCF Chimera, and the program was used for measuring distances and producing figures.","The complex models generated were not subjected to energy minimization with either molecular mechanics or dynamic approaches. The important role of M273 in stabilizing the siRNA:PAZ interaction is reflected by the reported change in Kd upon mutation of residue 273 from methionine to alanine Ma, J., Ye, K. & Patel, D. \u201cStructural basis for overhang-specific small interfering RNA recognition by the PAZ domain\u201d 429, 318-322 (2004). The side chain of M273 snakes along the terminal base pair of the siRNA duplex, and the mutation to alanine abolishes this stacking interaction and doubles the Kd. The effect of hydrophobic interactions between siRNAs and M273\/C270 on Kd was also examined.","Crystal structures of PS2- and MePS2-modified RNAs were undertaken and revealed subtle changes in geometry and hydration compared with natural RNA. The model of an MePS2-RNA:PAZ domain complex disclosed herein points to a hydrophobic effect as the source of the higher affinity of MePS2-RNA for Ago2.","Chemical modification of the antisense (guide) and\/or sense (passenger) siRNA strand modulates RNA affinity, nuclease resistance, immune stimulation, uptake and biodistribution, and is widely expected to be required for efficacy in vivo. SiRNAs containing phosphorodithioate modifications (PS2-RNA) were recently shown to exhibit favorable properties for therapeutic applications.","Thus, the PS2 moiety, despite destabilizing the duplex slightly relative to the native phosphate (PO2), appears not to distort the A-form geometry, improves serum stability, and is tolerated in the central part of the antisense strand as well as in most of the positions tested in the sense strand.",{"@attributes":{"id":"p-0140","num":"0139"},"chemistry":{"@attributes":{"id":"CHEM-US-00001","num":"00001"},"img":{"@attributes":{"id":"EMI-C00001","he":"102.19mm","wi":"52.83mm","file":"US09629867-20170425-C00001.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"Unlike the phosphorothioate moiety (PS, shown above), PS2 is achiral, thus precluding potential drawbacks that may arise from chiral P-substituted oligonucleotides, owing to variable biophysical and biochemical properties of individual (e.g. PS-) diastereoisomers. In addition, PS2-modified siRNAs display increased protection against degradation by nucleases compared with PS-modified oligonucleotides. Combining the PS2 with the 2\u2032-O-methyl ribose modification (MePS2-RNA, above) afforded increased loading of modified siRNA duplexes into the RNA-induced silencing complex (RISC) as well as enhanced anti-tumor activity. Further efforts were undertaken to understand the basis of this benefit.","Synthesis of Native and Me-, PS-, PS2- and MePS2-Modified siRNAs:","The siRNA and modified RNAs were synthesized on an Expedite 8909 DNA\/RNA Synthesizer using commercial 5\u2032-DMT-2\u2032-O-TBDMS nucleoside (, C, G, and U) phosphoramidite monomers as well as 2\u2032-OMe-thiophosphoramidites. The average stepwise coupling efficiency of all phosphoramidites including thiophosphoramidites was about 97% as estimated by the DMT-cation assay. After deprotection, all of the modified RNAs were isolated by FPLC according to Yang, X.; et al. 2012, 7, 1214-1220 and Wu, S. Y.; Yang, X.; et al. 2014, 5, 3459. The PS2-RNAs were desalted using reverse-phase HPLC to yield the PS2-RNA final products. The representative structures of the MePS2-RNAs were confirmed by ESI-MS. Assembly of siRNA duplexes was performed in ammonium acetate buffer (pH 7.4) by heating the equivalent mixture of RNA oligonucleotides coding for the sense and antisense strands of siRNA at 90\u00b0 C. for 2 min, followed by slow cooling to room temperature (over 2 h). Assembly of the resulting duplexes was confirmed by electrophoresis using a 4% agarose gel.","Tmeasurements were undertaken as previously described herein. The results for melting temperature studies of the following sequence are shown in Table 4 below.",{"@attributes":{"id":"p-0145","num":"0144"},"tables":{"@attributes":{"id":"TABLE-US-00006","num":"00006"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"343pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u20034"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Melting\u2003temperatures\u2003T\u2003of\u2003native\u2003and\u2003MePS2-siRNAs"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"217pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SEQ.\u2003ID.","T"]},{"entry":["RNA","Sequence","NO.","[\u00b0 C.]"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["siRNA-1","\u2003\u2003\u20035\u2032 AUA\u2003CAG\u2003GCA\u2003GCA\u2003GUA\u2003ACU\u2003UU\u2003TT\u20033\u2032","59","73.7\u2003\u00b1 "]},{"entry":[{},"3\u2032 TT\u2003UAU\u2003GUC\u2003CGU\u2003CGU\u2003CAU\u2003UGA\u2003AA\u20035\u2032","60","0.8"]},{"entry":{}},{"entry":["MePS2-1","\u2003\u2003\u20035\u2032 AUA\u2003CAG\u2003GCA\u2003GCA\u2003GUA\u2003ACU\u2003TT\u20033\u2032","61","74.0\u2003\u00b1"]},{"entry":[{},"3\u2032 TT\u2003UAU\u2003GUC\u2003CGU\u2003CGU\u2003CAU\u2003UGA\u2003A\u2003\u2003\u2003\u2003A\u20035\u2032",{},"0.2"]},{"entry":{}},{"entry":["MePS2-3","\u2003\u2003\u20035\u2032 AUA\u2003CGGCACAGUACU\u2003UTT\u20033\u2032","62","73.7\u2003\u00b1"]},{"entry":[{},"3\u2032 TT\u2003UAU\u2003GU\u2003\u2003\u2003CCGUC\u2003\u2003\u2003GUCAU\u2003\u2003\u2003UGA\u2003\u2003AA\u20035\u2032",{},"0.5"]},{"entry":{}},{"entry":["MePS2-4","\u2003\u2003\u20035\u2032 AUA\u2003CAG\u2003GCA\u2003GCA\u2003GUA\u2003AC\u2003\u2003\u2003U\u2003\u2003\u2003\u2003UU\u2003TT\u20033\u2032","63","74.2\u2003\u00b1"]},{"entry":[{},"3\u2032 TT\u2003UAU\u2003GUC\u2003CGU\u2003CGU\u2003CAU\u2003UGAA\u20035\u2003\u2032",{},"0.8"]},{"entry":{}},{"entry":["MePS2-9S","\u2003\u2003\u20035\u2032 AUCAG\u2003GCA\u2003GCGUA\u2003ACU\u2003UU\u2003TT\u20033\u2032","64","74.4\u2003\u00b1"]},{"entry":["MePS2-9AS","3\u2032 TT\u2003UAU\u2003\u2003\u2003GUC\u2003CGU\u2003CGUCAU\u2003UGA\u2003AA\u20035\u2032","65","0.5"]},{"entry":{}},{"entry":["MePS2-3","\u2003\u2003\u20035\u2032 AUA\u2003CGGCA\u2003\u2003\u2003CAGUACU\u2003UTT\u20033\u2032","62","73.2\u2003\u00b1"]},{"entry":["MePS2-11","3\u2032 TT\u2003UAU\u2003GU\u2003\u2003\u2003CCGUGUCAU\u2003UGA\u2003AA\u20035\u2032","66","0.2"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},"Ago2 Affinity Assays:","The antisense strand siRNA was biotinylated at its 3\u2032-terminus, allowing immobilization of an annealed duplex on to a streptavidin (SA) coated sensor surface. This enabled kinetic analysis of siRNA duplex binding to Ago2 on a fort\u00e9BIO Octet Red96 instrument set at 30\u00b0 C. Samples were agitated at 1000 rpm. The annealing method, binding buffer condition, siRNA loading concentration, siRNA loading time, Ago2 titer range, appropriate blocking reagent, blocking time, Ago2 stability over assay duration, and octet baseline correction methods were thoroughly investigated and optimized in the current study. Ago2 in HSCMT was prepared as a dilution series (0, 12.5, 25, 50, 100, and 200 nM) along with HSCMT buffer blanks (unloaded sensor controls). Association was monitored for 300 sec and dissociation was followed for 300 sec. If necessary, the dissociation was stretched to at least 900 sec to verify tight binding. The data were fit to a 1:1 binding model using fort\u00e9BIO Octet data analysis software. Kinetic constants were determined by integration of the experimental data using the differential rate equation dR\/dt=k\u00b7C\u00b7(R\u2212R)\u2212k\u00b7R to obtain both the kand kvalues (R=observed response, R=maximum response upon saturation, C=analyte concentration, k=association rate constant, k=dissociation rate constant). The ratio between kand kcorresponds to the reported dissociation constants (k\/k=K). The goodness of the fit was judged by the reduced \u03c7and Rvalues.","CD Measurements:","CD spectra were recorded on a CD6 dichrograph (Jobin-Yvon, Longjumeau, France) using cells with 0.5 cm path length, 2 nm bandwidth, and 1-2 sec integration time. Each spectrum was smoothed with a 25-point algorithm (included in the manufacturer's software, version 2.2.1) after averaging of at least three scans. The spectra from 200 nm to 340 nm were recorded at 25\u00b0 C. in the same buffer used for the melting experiments. The concentration of the two complementary RNA oligonucleotides was ca. 2 \u03bcM.","Modeling of PAZ:siRNA Complexes:","Coordinates for the crystal structure of a self-complementary RNA with a 3\u2032-dCT overhang bound to the human Ago2 PAZ domain were obtained from the Protein Data Bank (http:\/\/www.rcsb.org\/pdb\/home\/home.do; PDB ID 1si2). Using the program UCSF Chimera, non-canonical Py:Py and Pu:Pu pairs were replaced by Watson-Crick pairs according to the EphA2 siRNA sequence (Table 5), and the 2\u2032-deoxy-C in the 3\u2032-overhang was replaced by T.",{"@attributes":{"id":"p-0152","num":"0151"},"tables":{"@attributes":{"id":"TABLE-US-00007","num":"00007"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"308pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u20035"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Human\u2003Ago2\u2003binding\u2003affinities\u2003of\u2003EphA2\u2003siRNAs\u2003with\u2003various\u2003sense"},{"entry":"strand\u2003modifications\u2003adjacent\u2003to\u2003the\u20033\u2032-overhang"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"210pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["sRNA",{},"SEQID.",{}]},{"entry":["ID","Sequence:\u2003-B-=Biotin","NO.","K"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["AF163A","\u2003\u2003\u2003\u2003\u2003\u2003\u20035\u2032-UGA\u2003CAU\u2003GCC\u2003GAU\u2003CUA\u2003CA\u2003-3\u2032","67","21.1"]},{"entry":[{},"3\u2032-B-\u2003ACU\u2003GUA\u2003CGG\u2003CUA\u2003GAU\u2003GUA\u2003\u2003\u2003\u2003C-5\u2032 ","68","pM"]},{"entry":{}},{"entry":["AF163B","\u2003\u2003\u2003\u2003\u2003\u2003\u20035\u2032-UGA\u2003CAU\u2003GCC\u2003GAU\u2003CUA\u2003CAU\u2003G-3\u2032","69","0.6"]},{"entry":[{},"3\u2032-B-\u2003ACU\u2003GUA\u2003CGG\u2003CUA\u2003GAU\u2003GUA\u2003C-5\u2032",{},"\u03bcM"]},{"entry":{}},{"entry":["AF163C","\u2003\u2003\u2003\u2003\u2003\u2003\u20035\u2032-UGA\u2003CAU\u2003GCC\u2003GAU\u2003CUA\u2003CA\u2003-3\u2032","70","41.1"]},{"entry":[{},"3\u2032-B-\u2003ACU\u2003GUA\u2003CGG\u2003CUA\u2003GAU\u2003GUA\u2003\u2003\u2003\u2003C-5\u2032",{},"nM"]},{"entry":{}},{"entry":["AF163D","\u2003\u2003\u2003\u2003\u2003\u2003\u20035\u2032-UGA\u2003CAU\u2003GCC\u2003GAU\u2003CUA\u2003CA-3\u2032","71","12.4"]},{"entry":[{},"3\u2032-B-\u2003ACU\u2003GUA\u2003CGG\u2003CUA\u2003GAU\u2003GUA\u2003\u2003\u2003C-5\u2032",{},"pM"]},{"entry":{}},{"entry":["AF163F","\u2003\u2003\u2003\u2003\u2003\u2003\u20035\u2032-UGA\u2003CAU\u2003GCC\u2003GAU\u2003CUA\u2003CA-3\u2032","72","14.5"]},{"entry":[{},"3\u2032-B-\u2003ACU\u2003GUA\u2003CGG\u2003CUA\u2003GAU\u2003GUA\u2003\u2003C-5\u2032",{},"nM"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"left","id":"FOO-00001"},"sup":"a"}}]}}]}}},"In addition all water molecules were removed and hydrogen atoms were added. Following rebuilding of the RNA, the native complex was subjected to minimization by steepest descent and conjugate gradient using the Amber ff12SB force field in Chimera. In the complex with the MePS2-modified sense strand, two phosphates adjacent to the 3\u2032-overhang were replaced by phosphorodithioate moieties (taken from the crystal structure of CPs2G-RNA). In addition, methyl groups were added to the 2\u2032-oxygen atoms, such that the conformation of the CH3-02\u2032-C2\u2032-C3\u2032 torsion angle is in the antiperiplanar range. The complex was subsequently subjected to refinement as described above for the native PAZ:siRNA complex.","Crystallization Experiments:","To gain insight into the conformational consequences of PS2 modification and potential changes as a result of combining it with the ribose 2\u2032-O-methyl substitution, crystal structures were determined of the RNA duplexes (CGCGAAUUAGCG (SEQ. ID. NO.73))(CG-RNA), (CGCGAAUUAGCG (SEQ. ID. NO.74))(AA-RNA), and (CGCGAAUUAGCG (SEQ. ID. NO. 75))(AG-RNA) at resolutions of between 1.13 and 1.19 \u212b. The structures were phased by molecular replacement, using a previously determined structure of the native RNA as search model.","The above referenced modified RNA dodecamers were dissolved in deionized water and the stock concentrations adjusted to ca. 1.2 mM. Crystallization trials were performed with the hanging drop vapor diffusion technique, using the 24 conditions of the Nucleic Acid Miniscreen (Hampton Research Inc., Also Viejo, Calif.).Droplets (1 \u03bcL) of modified RNA dodecamer were mixed with droplets of equal volume of the individual sparse matrix screen solutions and equilibrated against 0.6 mL of a 35% v\/v solution of 2-methyl-2,4-pentanediol (MPD) at 18\u00b0 C. The optimal crystallization conditions for the three oligonucleotides were as follows. CPS2G-RNA: condition 20; sodium cacodylate buffer, pH 7.0, 80 mM sodium chloride, 20 mM barium chloride, 12 mM spermine tetrahydrochloride, and 10% v\/v MPD. AA-RNA: condition 15; 40 mM sodium cacodylate pH 7.0, 12 mM spermine tetrahydrochloride, 80 mM potassium chloride, and 10% v\/v MPD. AG-RNA: condition 8; 40 mM sodium cacodylate, pH 6.0, 80 mM sodium chloride, 12 mM spermine tetrahydrochloride, and 10% v\/v MPD.","X-Ray Data Collection, Structure Determination and Refinement:","Crystals were mounted in nylon loops, flash-frozen in liquid nitrogen without further cryo-protection and stored in liquid nitrogen prior to data collection. Diffraction data were collected on the 21-ID-F or 21-ID-G beam lines of the Life Sciences Collaborative Access Team (LS-CAT) at the Advanced Photon Source (APS), located at Argonne National Laboratory (Argonne, Ill.), using MARCCD 225\/300 detectors. The wavelength was 0.98 \u212b and crystals were kept at 100K during data collection. Diffraction data were integrated, scaled and merged using HKL2000.6. Selected data collection and refinement statistics are listed in . The structure was determined by the molecular replacement method with the program MOLREP7 in the CCP4 suite of crystallographic software, using the native RNA dodecamer as the search model (PDB ID 2Q1R9; http:\/\/www.rcsb.org). Following initial positional and isotropic temperature factor refinements with the program REFMAC, the R-work dropped to the mid 20s and R-free to the upper 20s. After further refinement in REFMAC, refinements were continued with SHELX. The phosphorodithioate and 2\u2032-O-methyl moieties (the asymmetric unit contains a single strand) were built into Fourier (2Fo-Fc) sum and (Fo-Fc) difference electron density maps that were visualized with COOT12 and the refinement was continued after adaptation of the dictionary files. Further refinements were carried out and after each refinement the nucleotides and water molecules were checked for possible alternate conformations\/positions and corrections were made where necessary. A final refinement was carried out with all nucleic acid atoms and water molecules being treated with anisotropic B-factors and these refinement parameters are summarized in . All figures were generated with the program UCSF Chimera.","Interestingly, it was determined that the increased loading of modified siRNA duplexes into the RNA-induced silencing complex (RISC) obtained by combining the PS2 with the 2\u2032-O-methyl ribose modification (MePS2-RNA, above) can be attributed to MePS2 modification at two residues adjacent to the 3\u2032-TT overhang in the sense siRNA strand. This conclusion is supported by higher association (based on in vitro pull-down assays), and tighter intracellular binding between MePS2-siRNA and Ago2 protein, relative to the corresponding P02, PS2-, or methoxy-modified RNAs (RNA, PS2-RNA, or Me-RNA). MePS2-siRNA showed significantly enhanced silencing of EphA2 involved in taxane resistance compared to all other RNA chemistries in both the SKOV3ip1 and HeyA8 epithelial ovarian cancer cell lines that highly express target protein. Other proteins with a key role in chemo-resistance besides EphA2, such as GRAMD1B, were subsequently targeted by systemic administration of MePS2-siRNA, leading to a re-sensitization of chemo-resistant ovarian tumors to taxane therapy.","The data showed that the structures of the PS2-modified RNA duplexes are isomorphous with that of the native RNA. Thus, the duplex is located on a crystallographic dyad and a single dodecamer strand constitutes the asymmetric unit. Superimposition of the CPS2G and native RNA duplexes demonstrated their virtually identical conformations (data not shown). In particular PS2 modification does not trigger any changes in the ribose conformation and the backbone torsion angles. The most obvious deviation concerned the P\u2014S bond lengths in the phosphorodithioate moiety (ca. 1.94 \u212b) compared to the P\u2014O bonds in the native phosphate (ca. 1.60 \u212b)","Crystal structures of the CG-RNA duplex are depicted in  A-C.  shows a view across the major and minor grooves. Fourier Fo-Fc difference electron density (2.5\u03c3 level) around O1P and O2P of G4 and G16 (indicating the presence of sulfur) is depicted as a meshwork, terminal residues are labeled, and water molecules are small spheres.  shows a close-up view of residue G4 with difference density around O1P and O2P before incorporation of sulfur.  shows a view into the central major groove with sulfur atoms shown as balls.","It was further shown that, in the AA- and AG-RNA duplexes, 2\u2032-O-methyl groups are directed into the minor groove, a result of the familiar ap orientation around the C2\u2032-O2\u2032 bond. Therefore, neither the PS2 modification alone nor the combination of the PS2 and 2\u2032-O-methyl modifications appear to have a notable effect on the conformation of A-form RNA judging from the crystallographic results.","However, the 2\u2032-O-methyl modification can offset the slight thermodynamic destabilization caused by the PS2 moiety compared with the native RNA duplex. We determined the melting temperatures (T) of siRNAs containing two to six MePS2 modifications and found that they exhibit thermal stabilities that are comparable to that of the unmodified RNA (PO2) (Table 4 above).","To assess potential conformational consequences of the MePS2 modification in solution, we measured the circular dichroism (CD) spectra of the siRNA duplexes listed in Table 4. All spectra were closely similar to the spectrum of the native duplex and are consistent with the typical A-type structure (a maximum of the positive Cotton effect at 268 nm and a crossover point at 240 nm.","The lack of an obvious change in both the conformation and stability of MePS2-RNA relative to native RNA points to another cause of the favorable therapeutic properties of this modification. To better understand the increased Ago2 affinity of siRNA containing MePS2-modified sense strands, we built models of PAZ:siRNA complexes based on the crystal structure of a short RNA fragment featuring a 3\u2032 two-residue overhang bound to the human Ago2 PAZ domain. This domain is most likely the only region of Ago2 affected by MePS2 modification near the siRNA 3\u2032-end. Comparison of the complexes with siRNA and MePS2-siRNA reveals formation of a hydrophobic patch in the latter, that involves both the 2\u2032-O-methyl and PS2 moieties as well as methionine and cysteine side chains from the PAZ domain ( and B).  and B depict models of the interactions between the human Ago2 PAZ domain and the (A) MePS2-modified and (B) native siRNA sense strands. Only the 3\u2032-terminal five residues of the RNA are shown (n\u22125 to n) and the TT-overhang is visible on the upper right in both panels. The formation of a hydrophobic patch between the Met-273 and Cys-270 side chains, methoxy moiety of Gn-2 and PS2 of Tis highlighted with a dashed circle in . PS2 groups, sulfur atoms (large pale balls) and 2\u2032-O-methyl carbons are depicted in ball-and-stick mode and hydrogen atoms are white spheres of smaller radius. Note the significantly more polar environment of the phosphate (His, Tyr 2\u00d7, Lys) between overhanging dTs, visible on the right-hand side of the two panels.","The hypothesis that increased hydrophobicity conferred by the MePS2 modification plays a key role in the stronger binding to Ago2 and its enhanced efficacy is supported by the relative Ago2 affinities of siRNAs containing no modification, PS, MePS, PS2 or MePS2.","The MePS2 modification in the backbone region adjacent to the 3\u2032-dTdT overhang of the siRNA sense strand triggers a strongly enhanced affinity to the RISC Ago2 slicer and silencing of proteins that confer chemoresistance in ovarian cancer cell lines and tumors. This finding is surprising given the fact that the antisense siRNA strand mediates cleavage of the target RNA at the Ago2 active site and indicates that chemical modification of the sense strand can play a key role in siRNA loading and efficacy. The PS2 moiety disrupts H-bonding interactions in the water chain bridging adjacent O2P atoms in the major groove.","We demonstrate here that MePS2 modification does not affect RNA conformation and stability. Instead the increased Ago2 affinity is likely the result of favorable hydrophobic interactions with the PAZ domain mediated by both the 2\u2032-O-methyl and PS2 moieties. This conclusion is consistent with the inability of PS2 sulfur atoms to form H-bonds with water molecules lining the RNA backbone in the crystal structure.","Both steric and electronic factors can contribute to the increased nuclease resistance afforded by a modification. The bulkier sulfur atoms in the PS2 moiety along with the increased hydrophobicity relative to PO2 might exclude nonthiophilic metal ions (e.g. Mg) from an exonuclease active site or result in suboptimal positioning of the water nucleophile for attack at the phosphate group.","All publications, patents and patent applications cited herein are hereby incorporated by reference as if set forth in their entirety herein. While this invention has been described with reference to illustrative embodiments, this description is not intended to be construed in a limiting sense. Various modifications and combinations of illustrative embodiments, as well as other embodiments of the invention, will be apparent to persons skilled in the art upon reference to the description. It is therefore intended that the appended claims encompass such modifications and enhancements."],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":["For a more complete understanding of the present invention, including features and advantages, reference is now made to the detailed description of the invention along with the accompanying figures:",{"@attributes":{"id":"p-0014","num":"0013"},"figref":"FIG. 1A"},{"@attributes":{"id":"p-0015","num":"0014"},"figref":"FIG. 1B"},{"@attributes":{"id":"p-0016","num":"0015"},"figref":"FIG. 1C"},{"@attributes":{"id":"p-0017","num":"0016"},"figref":"FIG. 1D"},{"@attributes":{"id":"p-0018","num":"0017"},"figref":"FIG. 2A"},{"@attributes":{"id":"p-0019","num":"0018"},"figref":"FIG. 2B"},{"@attributes":{"id":"p-0020","num":"0019"},"figref":"FIG. 2C"},{"@attributes":{"id":"p-0021","num":"0020"},"figref":"FIG. 2D"},{"@attributes":{"id":"p-0022","num":"0021"},"figref":"FIG. 2E"},{"@attributes":{"id":"p-0023","num":"0022"},"figref":"FIG. 2F"},{"@attributes":{"id":"p-0024","num":"0023"},"figref":"FIG. 2G"},{"@attributes":{"id":"p-0025","num":"0024"},"figref":"FIG. 2H"},{"@attributes":{"id":"p-0026","num":"0025"},"figref":["FIG. 2I","FIG. 2H"]},{"@attributes":{"id":"p-0027","num":"0026"},"figref":"FIG. 3A"},{"@attributes":{"id":"p-0028","num":"0027"},"figref":"FIG. 3B"},{"@attributes":{"id":"p-0029","num":"0028"},"figref":"FIG. 3C"},{"@attributes":{"id":"p-0030","num":"0029"},"figref":"FIG. 3D"},{"@attributes":{"id":"p-0031","num":"0030"},"figref":"FIG. 3E"},{"@attributes":{"id":"p-0032","num":"0031"},"figref":"FIG. 3F"},{"@attributes":{"id":"p-0033","num":"0032"},"figref":"FIGS. 3G-I"},{"@attributes":{"id":"p-0034","num":"0033"},"figref":"FIGS. 4A-C"},{"@attributes":{"id":"p-0035","num":"0034"},"figref":"FIG. 4D"},{"@attributes":{"id":"p-0036","num":"0035"},"figref":"FIGS. 5A-C"},{"@attributes":{"id":"p-0037","num":"0036"},"figref":"FIGS. 5D"},{"@attributes":{"id":"p-0038","num":"0037"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0039","num":"0038"},"figref":"FIGS. 7A-C"},{"@attributes":{"id":"p-0040","num":"0039"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0041","num":"0040"},"figref":"FIGS. 9A"}]},"DETDESC":[{},{}]}
