---
title: Computer software for automated annotation of biological sequences
abstract: Methods, software products and systems are provided for automated high throughout gene characterization. In one exemplary embodiment, HMM models are used to characterize genes. Biological sequences are classified according to their hit with different HMM models using a curve analysis.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=07133780&OS=07133780&RS=07133780
owner: Affymetrix, Inc.
number: 07133780
owner_city: Santa Clara
owner_country: US
publication_date: 20011205
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS REFERENCE TO RELATED APPLICATIONS","COPYRIGHT STATEMENT","BACKGROUND OF INVENTION","SUMMARY OF INVENTION","DETAILED DESCRIPTION","EXAMPLE","PROGRAM LISTING DEPOSIT"],"p":["Referenced-applications","This application claims the priority of U.S. Provisional Application Nos. 60\/285,144, filed on Apr. 19, 2001 and 60\/285,403, filed on Apr. 20, 2001. The 60\/285,144 and 60\/285,403 applications are incorporated herein by reference for all purposes.","A portion of the disclosure of this patent document contains material which is subject to copyright protection. The copyright owner has no objection to the facsimile reproduction by anyone of the patent document or the patent disclosure, as it appears in the Patent and Trademark Office patent file or records, but otherwise reserves all copyright rights whatsoever. The following notice applies to the software and data as described below and in the drawing hereto: Copyright 2001, Affymetrix, Inc. All Rights Reserved.","The present invention relates to bioinformatics and biological computing. More specifically, embodiments of the invention provides methods, systems and computer software for annotating biological sequences.","In recent years, the growth of the genomic databases has been explosive. With the recent completion of the human genome and the upcoming mouse genome, this growth is likely to continue. This explosion of data has highlighted the need for accurate, high-throughput methods for gene characterization.","Methods, computer software and systems are provided for highly accurate and automated gene characterization. The methods, computer software and system of the invention can be used to assign genes to families based upon protein (domain) structural similarity.","In one aspect of the invention, methods, software and systems are provided for automated threshold (cutoff) analysis. The methods, software and systems are particularly suitable for automated gene annotation\/characterization. In some embodiments of the invention, curve analysis is used to process Hidden Markov Model (HMM) scores, independent of the HMM properties.","Some embodiments of the computer implemented methods of the invention include the steps of obtaining a plurality of models (such as a battery of HMMs), where each of the models represents a classification of biological sequences with structural or functional similarity; determining distances of the biological sequences to the models; and automatically classifying the sequences according to the distances to the models. The methods are particularly suitable for annotating a large number of sequences, preferably at least 50, 100, 500, 1000, or 5000 sequences. In preferred embodiments, the sequences are protein sequences. In such embodiments, the models are typically established according to structural relationships among known proteins. Data from many protein databases, such as the Structural Classification of Proteins (SCOP) World Wide Web Address: scop.berkeley.edu) and Protein Data Bank (PDB) World Wide Web Address: www.rcsb.org\/pdb\/) are useful for building the HMM libraries in at least some embodiments of the invention.","In one preferred embodiment, the computer implemented method for gene characterization include generating libraries of models (such as the hidden markov models) using structural relationships of known proteins; inputting a plurality of protein sequences;comparing the plurality of protein sequences with the models; automatically establishing criteria for assigning the sequences for each model; and assigning the sequences to the models based upon the criteria.","In some instances, nucleic acid sequences are characterized by first translating the coding regions of the nucleic acid sequences into peptide sequences. In some preferred embodiments, genes are predicted genomic sequences, such as the human genome sequences. The genes are then translated and characterized at the protein level. In preferred embodiments, at least 50, 100, 500, 1000, or 5000 predicted genes are annotated.","In another aspect of the invention, systems and computer software are provided for performing the methods of the invention. The systems include a processor; and a memory coupled with the processor, the memory storing a plurality of machine instructions that cause the processor to perform logical steps of the methods of the invention. The computer software products of the invention include a computer readable medium having computer-executable instructions for performing the methods of the invention.","Reference will now be made in detail to the preferred embodiments of the invention. While the invention will be described in conjunction with the preferred embodiments, it will be understood that they are not intended to limit the invention to these embodiments. On the contrary, the invention is intended to cover alternatives, modifications and equivalents, which may be included within the spirit and scope of the invention.","The present invention provides computational methods, computer software products and systems for rapid automatic characterization of genes. While the invention will be primarily described using protein analysis as exemplary embodiments of the invention, this invention is not limited to protein analysis.","All patents and publications are herein incorporated by reference to the same extent as if each individual publication was specifically and individually indicated to be incorporated by reference.","Systems and Software for High Throughput Automated Gene Characterization","In aspects of the invention, methods, computer software and systems for gene characterization are provided. Systems of the invention typically include one or more processing units and a coupled memory for executing the software of the invention or otherwise executing logic steps of the methods of the invention. One of skill in the art would appreciate that computer software according to the embodiments of the invention can be executed in a wide variety of computer systems.","For a description of basic computer systems and computer networks, see, e.g., Introduction to Computing Systems: From Bits and Gates to C and Beyond by Yale N. Patt, Sanjay J. Patel, 1st edition (Jan. 15, 2000) McGraw Hill Text; ISBN: 0072376902; and Introduction to Client\/Server Systems: A Practical Guide for Systems Professionals by Paul E. Renaud, 2nd edition (June 1996), John Wiley & Sons; ISBN: 0471133337, both are incorporated herein by reference in their entireties for all purposes.",{"@attributes":{"id":"p-0028","num":"0027"},"figref":["FIG. 1","FIG. 1","FIG. 2"],"b":["101","103","105","107","109","111","111","107","112","102","113","114"]},{"@attributes":{"id":"p-0029","num":"0028"},"figref":["FIG. 2","FIG. 1"],"b":["101","101","201","209","101","203","202","210","208","206","204","211","203"]},{"@attributes":{"id":"p-0030","num":"0029"},"figref":"FIG. 3","b":["302","301","305","304","305","308","307","308","305","310"]},"Computer software products of the invention typically include computer readable medium having computer-executable instructions for performing the logic steps of the methods of the invention. Suitable computer readable medium include floppy disk, CD-ROM\/DVD\/DVD-ROM, hard-disk drive, flash memory, ROMIRAM, magnetic tapes and etc. The computer executable instructions may be written in any suitable computer language or combination of several languages. Suitable computer languages include C##, C\/C++ (such as Visual C\/C++), Java, Basic (such as Microsoft Visual Basic), SQL, Fortran, SAS and Perl. The software may be written in the form of independent software, software components (such as Java beans, Enterprise Java Beans). Software of the invention may also be implemented to provide application programming interfaces (APIs). In preferred embodiments, the software of the invention may provide remote services using remote execution in a distribute fashion. For example, in some embodiments, some or all of the logic steps of the methods and software of the invention are implemented as Web Services. For a detailed description of Web Services, see, e.g., \u201cWeb Services Insider,\u201d by James Snell, available at: World Wide Web Address: www-106.ibm.com\/developerworks\/webservices\/library\/we-refl.html, last visited Sep. 20, 2001.","Protein Databases","The well-characterized proteins in the Protein Database (PDB) have been studied at sequence, structure, and function levels, providing a richer description than the sequence-level information one finds in GenBank. As a result, these database entries are closely studied by scientists interested in protein folding, drug design, protein-protein interaction elucidation, evolutionary context, and gene annotation. In some aspects of the invention, protein databases are employed for rapid automated gene characterization. Some exemplary databases suitable for use with the methods of the invention are described in this section.","Most, if not all proteins, have some structural similarities with other proteins, often as a result of a common evolutionary origin. The Structural Classification of Proteins (SCOP) (Murzin, A., Brenner, S. E, Hubbard, T. Chothia, C., 1995. SCOP: A Structural Classification of Proteins Database for the Investigation of Sequences and Structures. J. Mol. Biol., 247:536\u2013540, incorporated herein by reference in its entirety for all purposes) scheme categorizes all the domains in the PDB into a hierarchy based on their structural and evolutionary proximity. The SCOP database provides a comprehensive description of the structural and evolutionary relationships between all proteins whose structure is known, including all entries in the Protein Data Bank (PDB).","The SCOP classification of proteins is constructed manually by visual inspection and comparison of structures with the assistance of computational tools. Proteins are classified according to their structural and evolutionary relationship. Many levels exist in the hierarchy, but the principal levels are family, superfamily and fold.","In the SCOP scheme, proteins clustered into families are clearly evolutionarily related. Generally, this means that pairwise residue identities between the proteins are 30% and greater. However, in some cases similar functions and structures provide definitive evidence of common descent in the absense of high sequence identity; for example, many globins form a family though some members have sequence identities of only 15%.","Proteins that have low sequence identities, but whose structural and functional features suggest that a common evolutionary origin is probable are placed together in superfamilies. For example, actin, the ATPase domain of the heat shock protein, and hexakinase together form a superfamily.","Proteins are defined as having a common fold if they have the same major secondary structures in the same arrangement and with the same topological connections. Different proteins with the same fold often have peripheral elements of secondary structure and turn regions that differ in size and conformation. In some cases, these differing peripheral regions may comprise half the structure. Proteins placed together in the same fold category may not have a common evolutionary origin: the structural similarities could arise just from the physics and chemistry of proteins favoring certain packing arrangements and chain topologies.","Protein Information Resource (PIR) is another database with protein classifications. See, e.g., Barker et al., (2001). Protein Information Resource: a community resource for expert annotation of protein data. Nucleic Acids Research, 29, 29\u201332. In the PIR scheme, database entries are organized into families of closely related sequences. These generally represent proteins with the same function in various organisms.","Generalized Rapid Automated Protein Analysis","Many functionally and evolutionarily important protein similarities can be analyzed by comparing three-dimensional structures. When such structures are not available, patterns of conservation identified from the alignment of related sequences can be used to discover distant similarities (homology detection).","Iterative profile search methods are the most widely-used family of homology detection methods. This family of methods includes PSI-BLAST (Altschul, Madden et al. 1997, Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res 25(17), 3389\u2013402.), SAM-T99 (Karplus, Barrett et al. 1998, Hidden Markov models for detecting remote protein homologies. Bioinformatics 14(10), 846\u201356.), and Pfam (Bateman, Birney et al. 2000, The Pfam protein families database. Nucleic Acids Res 28(1), 263\u20136.). They are more sensitive than comparisons of single sequences, detecting as many as three times the number of remote homologs as pairwise comparison methods (Park, Karplus et al. 1998, Sequence comparisons using multiple sequences detect three times as many remote homologues as pairwise methods. J Mol Biol 284(4), 1201\u201310.). They require no structural information, making them viable when structural information is limited. Finally, they are relatively easy to automate, and efficient enough to make them feasible for large-scale efforts such as whole-genome annotation. There are a variety of such methods. PSI-BLAST iteratively builds a statistical profile of the query sequence and homologs on the fly as it searches for other homologs, matching the profile. GRAPA (using SAM-T99) and Pfam (using HMMR) both compare the query sequence to pre-computed HMMs based on hand-curated family alignments in Pfam, and automatically-generated superfamily alignments in SAM-T99. While these methods search according to domain-wide conservation patterns, meta-MEME (Grundy, Bailey et al. 1997, Meta-MEME: motif-based hidden Markov models of protein families. Comput Appl Biosci 13(4), 397\u2013406.) searches according to patterns of highly-represented motifs. Other methods, such as FORESST (Geetha, Di Francesco et al. 1999, Comparing protein sequence-based and predicted secondary structure-based methods for identification of remote homologs. Protein Eng 12(7), 527\u201334.) ssHMM (Hargbo and Elofsson 1999, Hidden Markov models that use predicted secondary structures for fold recognition. Proteins 36(1), 68\u201376.) search for homology according to comparisons of secondary structure rather than purely sequence-based comparisons. MetaFam (Silverstein, Shoop et al. 2001, The MetaFam Server: a comprehensive protein family resource. Nucleic Acids Res 29(1), 49\u201351.) and Interpro (Apweiler, Attwood et al. 2001, The InterPro database, an integrated documentation resource for protein families, domains and functional sites. Nucleic Acids Res 29(1), 37\u201340.) provide meta-searches combining the results of multiple methods.","In one aspect of the invention, methods, software and systems are provided for using a high-accuracy system of hidden Markov models (HMMs) for automated genome annotation. Proteins are annotated by competing batteries of HMMs to yield family designations. This HMM library is applied to exploring uncharacterized genes. The Generalized Rapid Automated Protein Analysis (GRAPA) system assesses annotation significance with a dynamic cutoff, which compensates for the variance in the quality of distance calculations from one HMM to another.","GRAPA's foundation is a battery of Hidden Markov Models (HMMs) automatically generated to detect a protein's nearest family from the Structural Classification of Proteins (SCOP) hierarchy (Murzin, Brenner et al. 1995, SCOP: a structural of proteins database for the investigation of sequences and structures. J Mol Biol 247 (4), 536\u201340), a categorization of the Protein Data Bank (PDB) (Berman, Westbrook et al. 2000, The Protein Data Bank. Nucleic Acids Res 28(1), 235\u201342.).","GRAPA has an estimated false positive rate of only 5.7%, with a false negative rate of only 22.8%. While exemplary embodiments of GRAPA is based on the SCOP hierarchy, the method requires no structural information, and is applicable to cases where structural information is limited such as transmembrane proteins. As demonstrated by detailed analysis of phosphatase protein categories, GRAPA finds roughly three times as many significant sequences from a non-redundant set of human genes as BLASTP. PSI-BLAST has a comparable selectivity, but yields 20% fewer high quality putative phosphatases when compared to GRAPA. Given these properties, GRAPA gives researchers access to a highly sensitive homology-derived functional database for classifying genes and determining gene function.","In contrast to other gene characterization applications, GRAPA uses clusters of broadly trained HMMs. In contrast to annotation schemes which use a single HMM to identify the family or the superfamily of a protein, GRAPA uses an HMM based on each SCOP family member which incorporates a broad range of homologous proteins, comparable to those used to identify a protein superfamily. This gives the HMM a broader range of coverage in sequence space than a PSI-BLAST profile, while still allowing GRAPA to assign a protein family to each candidate. As illustrated in , a protein sequence is assigned to a SCOP family by comparing the distance scores of each candidate for each of the HMMs in the superfamily group. While GRAPA's HMMs do not always present a broader search in sequence space, they should usually present more results than a profile generated by the approach used by programs like PSI-BLAST.","In preferred embodiments, the SAM-T99 or similar systems are used. In blind experiments in the CASP contests, the SAM-T99 HMM system (Karplus, Barrett et al. 1998) has been shown consistently to be one of the best homology recognition systems that require no structural information (Karplus, Sjolander et al. 1997; Karplus, Barrett et al. 1999; Karplus, Karchin et al. 2001). In a recent investigation of family, superfamily, and fold level homology recognition, HMMs were the most successful method for family-level recognition (Lindahl and Elofsson 2000). Starting with this seed sequence or alignment, a model is built in an iterative fashion, the model used to search for additional homologs, and reestimates the alignment with the new homologs. HMMs are among the best homology recognition tools available at this time.","As  shows, HMMs (dark areas) have a wider range in sequence space than PSI-BLAST (light area), from the query (seed) sequences (dark points). The HMM candidates are then sorted into their most likely family (sorting represented by the black lines). This enables GRAPA to cover a larger region of sequence space while still giving family information.","SAM-T99 was designed for protein structure prediction, and optimized for recognizing proteins of common structure (superfamily recognition). Gough et al. (Gough, Chothia et al. 2000, Optimal Hidden Markov Models for All Sequences of Known Structure. Currents in Computational Molecular Biology 2000) provide Superfamily, a web-based service, which uses HMMs built around SCOP seed sequences with SAM-T99 default settings for superfamily recognition. Superfamily is meant as a service for superfamily identification of sequences submitted to the web-site.","Generally, sequence can be compared to a model by calculating the probability that the sequence was generated by that model. For SAM-T99, the negative (natural) logarithm of this probability gives the NLL score. However, the NLL score has a strong dependence on sequence length and model length. The SAM-T99 Hmmscore program provides several less biased means of scoring by reporting NLL scores as the difference between a null model and trained model NLL score (a log-odds score, as used in HMMER). Null model scoring is discussed in more detail in Hughey Christian Barrett, Richard Hughey, and Kevin Karplus, 1997, Scoring Hidden Markov Models, CABIOS 13(2):191\u2013199, incorporated herein by reference. This paper is also available at the following URL (www.cse.ucsc.edu\/research\/compbio\/papers\/nullmod\/nullmod.html), last visited on Sep. 20, 2001. The E-value is the expected number of sequences with a particular score. For methods of calculating the E-value, see, e.g., Richard Hughey\u2014Kevin Karplus\u2014Anders Krogh, SAM Sequence Alignment and Modeling Software System, Baskin Center for Computer Engineering and Science, University of California Santa Cruz, Calif. 95064, Technical Report UCSC-CRL-99-11, Updated for SAM Version 3.2, Jul. 31, 2000, incorporated herein by reference. This document is also available at the following URL (www.cse.ucsc.edu\/research\/compbio\/papers\/sam_doc\/sam_doc.html, last visited on Sep. 20, 2001).","Many of the SCOP HMMs (4,369 models built for SCOP v.1.53) exhibit a lack of consistency in HMM scoring, expressed as an E-value probability. Since scoring is dependent upon sequence length and the significance of specific amino acid positions, there has not been any automated scoring of HMMbased annotation systems for high throughput annotation. Instead, SAM-99 provides a set of tools for various plot scores. SAM-T99 recommends that plots of the scores be used to visually look for a break between significant and insignificant matches. The visualization examination, while effective in handling small set of models, are not practical for handling a large number of models in a high throughput environment.","In one aspect of the invention, methods, software and systems are provided for automated threshold (cutoff) analysis. The methods, software and systems are particularly suitable for automated gene annotation\/characterization.  illustrates an embodiment of the process. Candidate proteins are scored against HMM models (). The scores are analyzed to determine significant hits (). The proteins are then assigned into subfamily according to the hits to the models ().","In preferred embodiments, automated threshold analysis is used to distinguish between hits and non-hits, allowing the results for each HMM to be analyzed in an unsupervised annotation.  shows a process for threshold analysis. A large number of protein sequences, for example, at least 5, 10, 50, 100, 500, 1000, 5000, protein sequences are inputted () and scored for a large number of HMMs (). The resulting score (E-value) is analyzed for hits (). The hits are outputted ().","In preferred embodiments, the threshold analysis is performed with a curve analysis. In a particularly preferred embodiment, five criteria are used to distinguish the hits to be kept: (1) when the E-values rise above a value of e-05 they are discarded regardless of the other criteria; (2) the first ten hits are kept provided they score better than e-05; (3) the hits more than 70% of the log of the maximum score are automatically kept; (4) the point where the E-value plot drops abruptly or flattens is used as a threshold; (5) no more than a maximum number of hits (e.g., 1000) are kept.","An exemplary PERL script for performing one embodiment of the curve analysis is provided in the computer program listing.",{"@attributes":{"id":"p-0054","num":"0053"},"figref":"FIGS. 7","i":"a\u2013d ","b":"1"},"The following example used a high-accuracy system of hidden Markov models (HMMs) for automated genome annotation. Proteins were annotated by competing batteries of HMMs to yield family designations. This HMM library was applied to exploring uncharacterized genes. The Generalized Rapid Automated Protein Analysis (GRAPA) system assesses annotation significance with a dynamic threshold, which compensates for the variance in the quality of distance calculations from one HMM to another. This example illustrates various aspects of the invention.","A. Materials and Methods.  is a flow chart showing the steps of the method used in this example.","Whole genome gene set. A set of protein sequences covering the Golden Path of the human genome (Oct. 7, 2000 freeze World Wide Web Address: genome.ucsc.edu\/) was generated by the Genie (Reese, M. G., Kuip, D., Tammana, H. and Haussler, D. (2000) Genie\u2014gene finding in Drosophila melanogaster. Genome Res 10(4), 529\u201338) programs suite (Kulp, D. & Wheeler, R., available at the following URL (genome.ucsc.edu, last visited on Sep. 20, 2001), with the repeat regions masked out (). The data set consists of three sets of amino acid sequences: (1) the set derived by alignment of mRNA to the genomic DNA corresponding to sequences in RefSeq and GenPept, (2) a set of alternatively spliced variants, which are generated using mRNA\/EST-to-genomic alignments in combination with purely statistical methods, each of which contain the largest subset of exons for a particular gene, and (3) genes predicted by purely ab initio methods. These sequences are non-redundant; none of the included genes overlap the same genomic region. In cases where there were many genes overlapping the same region, the one with the longest CDS (translation) was kept. This set, known as annot10, contains 59,378 putative protein sequences.","Generating Hidden Markov Models. The GRAPA system characterizes each SCOP protein domain found in ASTRAL (Brenner, S. E., Koehl, P. and Levitt, M. (2000)(, ). The ASTRAL compendium for protein structure and sequence analysis. Nucleic Acids Res 28(1), 254\u20136.) (v1.53), a web site which allows the user to set various parameters for selecting proteins from SCOP. For SCOP version 1.53, the total set of non-redundant proteins consisted of 4369 entries. Multiple species were included to capture entries with both mammalian and non-mammalian proteins.","For each entry in the SCOP protein set (, ), a hidden Markov model (HMM) was built using the Sequence Alignment and Modeling system (SAM 3.0) target99 protocol (Karplus, K., Barrett, C. and Hughey, R. (1998). Hidden Markov models for detecting remote protein homologies. Bioinformatics 14(10), 846\u201356). It might seem redundant to build a model for each SCOP protein since there are sometimes proteins from multiple species, all of which are the same protein; however, it has been shown (Aravind, L. and Koonin, E. V. (1999). Gleaning non-trivial structural, functional and evolutionary information about proteins by iterative database searches. J Mol Biol 287(5), 1023\u201340) that models built with certain seed sequences will capture a broader set of hits. Therefore, one should know a priori which sequences to use or should exhaustively use them all. This protocol profiles the family of a seed sequence by iteratively searching for homologs of the seed sequence, estimating a multiple alignment of the seed sequence and its homologs, estimating an HMM from the multiple alignment, and using this HMM to search for additional homologs. At each iteration, the HMM yields a generalized profile of the sequence family, describing the salient characteristics expected in additional homologs not yet seen. The early iterations produce very specific profiles, recognizing only close homologs of the seed sequence. Each subsequent iteration yields a broader profile. The generality of each profile is controlled by the thresholds for admitting new sequences to the family. The E-value thresholds used for SAM-T99 are 0.001 and 0.02 for two cycles, higher than those suggested for superfamily HMM compilation (0.00001, 0.0001, 0.001, 0.01) and with fewer cycles (four is often used).","Screening Genes Against SCOP HMMs (). After each gene had been scored against all the HMMs, it was assigned to one model according to the best score within the SCOP superfamily. Before accepting an annotation, the alignment to the model further evaluated by other metrics including sequence identity. In some cases, genes might fail to be annotated due to incorrect intron boundaries. However, those predicted genes that meet the thresholds for matching a particular family or superfamily, may be refined by mapping of known, related genes back onto the genomic DNA with programs like GeneWise (Birney, E. and Durbin, R. (2000). Using GeneWise in the Drosophila annotation experiment. Genome Res 10(4), 547\u20138). GRAPA then generated a report for a more careful examination of genes discovered de novo, integrating the protein annotation information into the gene-finding process.","Assigning Annotations Based on SCOP Family with Distsieve (). A correspondence between a SCOP family and a gene was based on the E-value scored by SAM-T99. The program distsieve (attached computer program listing) examined all the sequences scored against a particular SCOP model. Because the interpretation of distance scores, expressed as logarithmic E-values was dependent upon the HMM generated for each sequence, there is no single E-value threshold that can be applied to all models. Instead, for each HMM, the distsieve method examined the set of scores by curve analysis to determine a reasonable E-value threshold. The distance scores included controls representing the proteins used to build all the models; thus, the best hit was often the seed sequence for the HMM.","Five criteria were used to distinguish the hits to be kept: (1) when the E-values rised above a value of e-05 they were discarded regardless of the other criteria; (2) the first ten hits were kept provided they score better than e-05; (3) the hits more than 70% of the log of the maximum score were automatically kept; (4) the point where the E-value plot drops abruptly or flattens was used as a threshold; (5) no more than a maximum number of hits (e.g., 1000) are kept.","Next, the set of hits accepted by distsieve were grouped by superfamily (), where each gene is assigned to one SCOP family HMM according to the best distance score. While SAM-T99 was designed and optimized for superfamily recognition, GRAPA was designed for function prediction at the family level. Among the representative HMMs in the superfamily, the scores for each hit was compared and assigned to the family which generated the best score by the program, sortDist (). The program, align2model was used to create an alignment between a successful hit and the SCOP sequence according to the SAM-T99 generated model (). The alignments were then screened by identity scores and annotations created in XML database format ().","RESULTS AND DESCUSSION. Model Validation. All SCOP sequences were scored against each model to assess the accuracy of the model library. The hits to each model were assessed before and after the distsieve threshold. Each SCOP control sequence hit was classified as a family hit (true positive) or false positive according to its relation to the model's seed sequence; true positives are defined as members of the same SCOP family, while false positives were members of different SCOP families.","For each model, sensitivity, the percentage of correct SCOP family members found; selectivity, the percentage of SCOP family members found among the hits were evaluated. Thus, GRAPA was designed to be very selective, even at the expense of some potential sensitivity.",{"@attributes":{"id":"p-0066","num":"0065"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"center"}},"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Model selectivity. True positives are defined as control sequences from the same SCOP"},{"entry":"family as the seed sequence, and false positives are defined as SCOP sequences from"},{"entry":"different SCOP families than the seed sequence. PSI-BLAST selectivity figures are"},{"entry":"shown for comparison."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Before",{},"After",{},{},{}]},{"entry":["Control","distsieve",{},"distsieve",{}]},{"entry":["(SCOP) hits,","threshold",{},"threshold",{},"PSI-BLAST"]},{"entry":["all models","mean","total","mean","total","Mean","total"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"7","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Total control","114","497,509","18","79,578","18","\u200982876"]},{"entry":"(SCOP) hits"},{"entry":["Family","21","\u200291,024","17","75,036","17","77,950"]},{"entry":["members",{},"(18.3%)",{},"(94.3%)",{},"(94.1%)"]},{"entry":"(true positives)"},{"entry":["False Positives","93","406,485","1","\u20024,542","1","\u20024,926"]},{"entry":[{},{},"(81.7%)",{},"\u2002(5.7%)",{},"\u2002(5.9%)"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}}]}}]}}},"Table 1 describes the selectivity of the annotation process in comparison to the standard method PSI-BLAST (Altschul, S. F., Madden, T. L., Schaffer, A. A., Zhang, J., Zhang, Z., Miller, W. and Lipman, D. J. (1997). Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res 25(17), 3389\u2013402). Before applying distsieve, the hits to each model included an average of 114 SCOP sequences: 21 family hits and 93 false positives. After applying distsieve, the hits to each model included 18 control sequences: 17 family hits and 1 false positive. The percentage of false positives at the family level was 81.7% before any threshold is applied. After distsieve, this value dropped to 5.6%. Thus, distsieve succeeded in weeding out most of the false positives while preserving most of the family-level homologs. After applying distsieve, the performance of GRAPA was comparable to that of PSI-BLAST, with PSI-BLAST getting slightly more hits and a slightly lower selectivity.",{"@attributes":{"id":"p-0068","num":"0067"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Model sensitivity. On average, each SCOP HMM finds 93.6% of"},{"entry":"its actual family members before the distsieve threshold, and 77.2%"},{"entry":"after the distsieve threshold. While distsieve cuts out some number of"},{"entry":"family hits, false positives present a bigger problem in gene annota-"},{"entry":"tion than false negatives. PSI-BLAST sensitivity figures are shown"},{"entry":"for comparison."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"140pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Mean","Total"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"140pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SCOP family members","22","97,282 (100%)"]},{"entry":["SCOP family hits before distsieve threshold","21","91,024 (93.6%)"]},{"entry":["SCOP family hits after distsieve threshold","17","75,036 (77.2%)"]},{"entry":["PSI-BLAST hits","17","77,950 (80.1%)"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Table 2 shows the sensitivity of the annotation process in comparison to PSI-BLAST. For each model, the number of SCOP family hits was compared to SCOP family members. Each sequence had an average of 22 SCOP family members. 21 of these were among the top hits to the model, and 17 were selected by distsieve. Thus, while some family members were lost in the selection process, most (77.2%) were not.","The distsieve threshold was weighted against a simple BLAST E-value threshold. This number of false positives would be accepted if an E-value threshold of 1.0e-86 was applied to all models. Note that this E-value threshold would admit only 21,799 family members. By using the distsieve threshold for each model rather than an E-value threshold, three times as many family members (75,036) were admitted for a comparable number of false positives. The lowest E-value threshold that would admit 75,036 family members is 1.0e-22; at that threshold, 32,822 false positives would be admitted.","As shown in Tables 1 and 2, the selectivity of PSI-BLAST and GRAPA are comparable. Further, in detailed examination of the results, GRAPA and PSI-BLAST tend to score well in the same cases, and score poorly in the same cases. In general, both methods perform worst in cases such as globins and EF-hands, when some SCOP superfamily contains a number of closely-related families. In such cases, this behavior should be viewed as an effect of where the threshold was drawn in the SCOP hierarchy.","Protein Phosphatase Candidates in the Human Genome. To compare GRAPA to other high-throughput homology search methods, searches were performed on protein phosphatases, enzymes that remove phosphate groups from proteins at specific amino acid side chains. The amino acid sequences for the SCOP phosphatase domains (Table 3) were run through BLASTP and PSI-BLAST (with a threshold of 1e-05) and compared to the results from GRAPA with distsieve threshold. The resulting hits were compared using identity scores by pairwise CLUSTALW alignments between each hit and the five target sequences, where the best identity score kept for each hit.",{"@attributes":{"id":"p-0073","num":"0072"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Full genome protein set scored against SCOP hidden Markov"},{"entry":"models. Known Genie-derived phosphatases from GenBank are"},{"entry":"separated out, showing that they are all identified by three"},{"entry":"models: 2567, 3788, and 3795."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},{},{},"Number"]},{"entry":[{},{},{},{},"Number","non-"]},{"entry":["HMM","PDB",{},{},"GenBank","GenBank"]},{"entry":["ID","code","SCOP ID","protein name","Entries","Entries"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["692","1a17","1.111.8.1.1","Protein","1","78"]},{"entry":[{},{},{},"phosphatases 5"]},{"entry":[{},{},{},"(Human)"]},{"entry":["694","1qqe","1.111.8.1.3","Vesicular","\u2014","4"]},{"entry":[{},{},{},"transport"]},{"entry":[{},{},{},"protein sec17"]},{"entry":[{},{},{},"(yeast)"]},{"entry":["2567","1vhr","3.40.1.1.1","VH1-related dual-","6","52"]},{"entry":[{},{},{},"specificity"]},{"entry":[{},{},{},"phosphatases,"]},{"entry":[{},{},{},"VHR (Human)"]},{"entry":["3788","1a6q","4.144.1.1.1","protein Ser\/Thr","3","2"]},{"entry":[{},{},{},"phosphatases 2C"]},{"entry":[{},{},{},"(Human)"]},{"entry":["3793","1fjm","4.145.1.3.1","protein","5","7"]},{"entry":[{},{},{},"phosphatases-1"]},{"entry":[{},{},{},"(PP-1) (rabbit)"]},{"entry":["3794","1tco","4.145.1.3.2","protein","2","\u2014"]},{"entry":[{},{},{},"phosphatases-2B"]},{"entry":[{},{},{},"calcineurin A"]},{"entry":[{},{},{},"submit (bovine)"]},{"entry":["3795","1aui","4.145.1.3.3","protein","1","\u2014"]},{"entry":[{},{},{},"phosphatases-2B"]},{"entry":[{},{},{},"calcineurin A"]},{"entry":[{},{},{},"submit (Human)",{},{}]},{"entry":["Total",{},{},{},"18","143"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]}}]}}},"Of the 18 Genie-derived human protein phosphatases in GenBank, every one mapped to the SCOP phosphatase families with better than 20% sequence identity, as shown in Table 3. As in Table 4, GRAPA found the most phosphatase candidates, PSIBLAST, nearly as many, and BLASTP only one third as many.",{"@attributes":{"id":"p-0075","num":"0074"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 4"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Identity of GRAPA verses BLAST candidates for GenBank and"},{"entry":"novel hits divided by whole sequence percent identity against the"},{"entry":"SCOP seed sequences for relevant families (Table 3). Novel hits,"},{"entry":"in parentheses, are those found exclusively by these methods."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Percent Identity","GRAPA (HMM)","PSIBLST","BLASTP","GENBANK"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":["\u20020\u20135%","\u20030","\u20030","\u20020","\u20020"]},{"entry":["\u20025\u201310%","\u20030","\u20031 (1)","\u20020","\u20020"]},{"entry":["10\u201315%","\u200210 (5)","\u20036 (1)","\u20022","\u20020"]},{"entry":["15\u201320%","\u200215 (6)","\u200212 (3)","\u20021","\u20020"]},{"entry":["20\u201325%","\u200269 (27)","\u200247 (5)","13","\u20020"]},{"entry":["25\u201330%","\u200230 (7)","\u200224 (0)","14","\u20023"]},{"entry":["30\u201340%","\u200216","\u200216","16","\u20025"]},{"entry":["40\u201350%","\u20034","\u20034","\u20024","\u20024"]},{"entry":["50\u201350%","\u20030","\u20030","\u20020","\u20020"]},{"entry":["60\u201370%","\u20030","\u20030","\u20020","\u20020"]},{"entry":["70\u201380%","\u20031","\u20031","\u20021","\u20020"]},{"entry":["80\u201390%","\u20032","\u20032","\u20022","\u20022"]},{"entry":["90\u2013100%","\u20034","\u20034","\u20024","\u20024"]},{"entry":["Total","151","117","57","18"]},{"entry":["Known","\u200218","\u200218","18","18"]},{"entry":"(GenBank)"},{"entry":["Novel","\u200245","\u200210","\u20020","\u2014"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},"BLASTP is the most conservative of the three methods with only 57 total hits. These hits include the 18 known protein phosphatases in human annot10 gene set; all exhibit greater 30% sequence identity to a seed sequence.","From table 4, it is clear that the methods vary mostly in sensitivity. GRAPA finds 30% more candidates than PSI-BLAST. PSI-BLAST finds 11 candidates which the other methods do not find and GRAPA finds 45 hits unique to it, whereas BLASTP found essentially no unique hits. In the grey, 20\u201330% identity range, for homologues, PSI-BLAST and family HMMs performance are known to be comparable, but GRAPA found more candidates with identity scores in every identity bracket, finding seven unique candidates with 25\u201330% identity while PSI-BLAST found only one.",{"@attributes":{"id":"p-0078","num":"0077"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0079","num":"0078"},"figref":"FIG. 10"},"Genome analysis by GRAPA. An overview of human genes shows a concordance between GRAPA and PFAM. The most common SCOP domains in the human annot gene set are shown in Table 5. The corresponding Pfam domain and rank in the human genome is provided for each of the top twenty SCOP domains. In general, these numbers are equivalent to those previously published for the Fly genome by the application of the InterPro analysis (Rubin, Yandell et al. 2000). PFAM confirms the top ten GRAPA domains; kinases, zinc fingers, G proteins, ankyrin repeats, SH3-domain family and WD repeats.",{"@attributes":{"id":"p-0081","num":"0080"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 5"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Most frequently occurring domains of human genes according"},{"entry":"to GRAPA (SCOP v 1.53) and PFAM. Rank in Pfam column is"},{"entry":"from the equivalent Pfam family ranking (not shown)"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"SCOP",{},{},"Pfam (rank"]},{"entry":[{},"Family Id","H","SCOP family","equivalent)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["\u20021","4.130.1.1","349","Serine\/threonine kinases","pkinase (3)"]},{"entry":["\u20022","7.37.1.1","296","Classic zinc finger, C2H2","zf-C2H2 (1)"]},{"entry":["\u20023","4.130.1.2","246","Tyrosine kinase","pkinase (3)"]},{"entry":["\u20024","3.32.1.8","184","G proteins","ras (20)"]},{"entry":["\u20025","1.111.2.1","135","Ankyrin repeat (SH3-domain","ank (5)"]},{"entry":[{},{},{},"superfamily)"]},{"entry":["\u20026","2.64.3.1","115","Trp-Asp repeat (WD-repeat)","WD40 (8)"]},{"entry":["\u20027","3.9.2.1","\u200299","Internalin B LRR domain","LRR (9)"]},{"entry":["\u20028","3.9.2.3","\u200297","U2A\u2032-like Leucine Rich Re-","LRR (9)"]},{"entry":[{},{},{},"peat Fold, RNA recog."]},{"entry":["\u20029","3.32.1.13","\u200290","Extended AAA-ATPase","helicase_C (26)"]},{"entry":[{},{},{},"domain (DNA helicases bac-","and DEAD (27)"]},{"entry":[{},{},{},"terial yeast)"]},{"entry":["10","3.9.1.2","\u200286","Rna1p (in Leucine Rich Fold)","LRR (9)"]},{"entry":["11","2.44.1.2","\u200282","Eukaryotic proteases","\u2014"]},{"entry":["12","2.44.1.1","\u200278","Prokaryotic proteases","\u2014"]},{"entry":["13","1.4.1.1","\u200274","Homeodomain","homeobox (14)"]},{"entry":["14","4.37.1.1","\u200273","BTB\/POZ domain (zinc","BTB (24)"]},{"entry":[{},{},{},"finger)"]},{"entry":["15","1.23.1.1","\u200265","Nucleosome core histones","histone (29)"]},{"entry":["16","3.9.1.1","\u200264","Ribonuclease inhibitor (LRR","LRR (9)"]},{"entry":[{},{},{},"fold)"]},{"entry":["17","4.82.1.1","\u200263","SH2 domain","SH2 (25)"]},{"entry":["18","3.9.2.2","\u200263","Rab geranylgeranyl-trans-","C2 (22)"]},{"entry":[{},{},{},"ferase a-subunit, N-terminal"]},{"entry":[{},{},{},"(C2 domain-like Fold)"]},{"entry":["19","2.1.1.1","\u200261","V set domains (Ab variable","Ig (4)"]},{"entry":[{},{},{},"domain-like) Ig superfamily"]},{"entry":["20","3.32.1.9","\u200261","Motor proteins (nucleoside","myosin head"]},{"entry":[{},{},{},"triphosphate hydrolase fam)","motor domain"]},{"entry":[{},{},{},{},"(52)"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},"While SCOP defines (and GRAPA identifies) leucine rich repeats (LRRs) via their presence as part of whole proteins, Pfam has a specific domain (LRR) which identifies this motif independent of the protein type. This difference gives rise to the disparities of the figures in SCOP where LRRs appear as Internalin B LRR domain, U2A\u2032-like (Leucine Rich Repeat Fold and RNA recognition), and Rna 1 p (in Leucine Rich Fold).","Significant differences between GRAPA and PFAM are attributable to differences in the construction between the two suites. SCOP groups eukaryotic and prokaryotic proteases under the trypsin-like serine protease superfamily. The eukaryotic family contains 39 different proteases and the prokaryotic family contains nine different proteases. The trypsin Pfam domain appears to be more specific and would likely not rank in the top twenty. Immunoglobulin domains appear as a top hit under Pfam (rank=4), while SCOP finds many such hits via the V set domains (antibody variable domain-like) (rank=19).","Given the high rank of the Zinc finger C3HC4 type RING domain (rank=13) in the Pfam top twenty, and the RING models in SCOP (rank=90), more careful examination would be required to determine whether these Pfam hits appeared in the SCOP C2H2 (rank=2) or BTB\/POZ zinc finger (rank=14) domains. In the case for the Pfam EF-HAND domain (rank=15), SCOP has broken the EF-hand superfamily into seven families, which would yield fewer hits per family and therefore make them less likely to appear in the top twenty models.","The prevalence of signaling proteins such as kinases, proteases, and G proteins in the human genome reflects the importance and variety of signaling mechanisms within higher-order organisms such as humans.","The top twenty Pfam entries (data not shown) with no corresponding SCOP entry, correspond to transmembrane proteins (7tm1) and other entries for which there are no solved 3D structures.","The annotation and examination of genome-derived sequences for known classes of proteins is an important approach for exploring the new biological data in silico. More than 40% of the current genes in the genome are previously uncharacterized and the combination of protein categorization schemes with the unknown genes will be very useful for that work. The GRAPA method successfully assigns genes to SCOP families based on a highly automated and accurate scoring method against a battery of hidden Markov Models. Its performance is comparable to PSI-BLAST. The entire human genome (known and predicted) has been screened against these models. Results similar to PFAM were found, where the top categories of genes overlapped considerably.","In a general comparison, GRAPA's performance was found to be similar to that of PSI-BLAST. In a more detailed comparison, searching on phosphatases, GRAPA found 20% more candidates of comparable quality than PSI-BLAST, and nearly three times as many hits as BLASTP.","With the innovation of using curve analysis to analyze high throughput HMM scores, independent of the HMMs' properties, GRAPA could be configured, without extensive manual intervention, to reflect varied ontological or biochemical categorization schemes for which protein sequences are available. With a scheme such as GRAPA, HMMs may be generated from a plethora of diverse protein databases, allowing the identification of the potential function on the sets of new genes being identified in the genome.","The GRAPA methodology was also applied to the Enzyme Commission classification scheme. This scheme contains a hierarchical representation based on broad enzymatic classes, sets of substrates, and cofactors\/reagents. Enzymes are principally classified and named according to the reaction they catalyze. The highest level distinction, the enzyme class is broken into oxidoreductases, transferases, hydrolases, lyases, isomerases, and ligases. The next level of distinction in the EC hierarchy is the substrate upon which the enzyme acts, for example, CH\u2014OH, aldehyde, CH\u2014NH2, etc. The third level is the co-factor or reagent required for conversion of the substrate. These reagents might include NAD+, NADP+, oxygen, quinone, etc.","For each enzyme in the EC set, an HMM was built using SAM-T99. Model building and scoring were performed in a similar manner to SCOP. Since EC is a hierarchical classification scheme with a tree structure, a set of hits to HMMs may be grouped into superfamilies. Putative hits were assigned to a single EC HMM within each superfamily according to the best E-value, and this HMM was used to align the query sequence to the EC sequence.","In instances, the annotations were stored in a Relational Database Management system, i.e., postgres. This affords the following advantages: a)\u2014data format is secured from errors by the data types of the table schemas; b)\u2014the pipeline can be recovered if it is interrupted.","In some instances, the alignments of the sequences are checked for quality and removed before they are accepted by the famsort algorithm (which is now a series of database commands), e.g, the number of gaps in the alignment of the hit and the size of the largest gap are set at 8 and 40 respectively.","It is to be understood that the above description is intended to be illustrative and not restrictive. Many embodiments will be apparent to those skilled in the art upon reviewing the above description. The scope of the invention should, therefore, be determined not with reference to the above description, but should instead be determined with reference to the appended claims, along with the full scope of equivalents to which such claims are entitled.",{"@attributes":{"id":"p-0095","num":"0094"},"tables":{"@attributes":{"id":"TABLE-US-00006","num":"00006"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Copyright 2001, Affymetrix, Inc."},{"entry":"#!\/usr\/bin\/perl -w"},{"entry":"# distsieve - a perl script to collate and sort hmmscore output"},{"entry":"# use: distsieve [FAMILYNAME]"},{"entry":"use strict;"},{"entry":"# parameters for the filter"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"168pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":["my $maxE","= \u22128;","# highest allowed E value (log 10)"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"154pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["my $maxN","= 1000;","# maximum number of hits per dist file"]},{"entry":["my $DEFminN","= 10;","# number of hits before looking for a cut"]},{"entry":["my $Efrac","= 0.8;","# log fraction of Emax where we start loo"]},{"entry":["my $minE","= \u221250;","# default minE -calculated in pass1"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"# other variables"},{"entry":"my @files;"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"154pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["my $family","= \u201csieve\u201d;",{}]},{"entry":["my $minN","= $DEFminN;"]},{"entry":"my $logmax;"},{"entry":"my @delta;"},{"entry":"my @eval;"},{"entry":"my @elog;"},{"entry":"my @entries;"},{"entry":"my @drop;"},{"entry":"my $ndrop;"},{"entry":["my $emin","= 2.0e00;"]},{"entry":["my $emax","= \u22121.0e00;"]},{"entry":["my $cutoff","= $maxN;"]},{"entry":["my $fnum","= 0;"]},{"entry":["my $ecount","= 0;","# entries read in"]},{"entry":["my $totcnt","= 0;"]},{"entry":["my $tottot","= 0;"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"# decide mode"},{"entry":"if ($ARGV[0]) {"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"$family = $ARGV[0];"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"# modified for SCOP"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"@files = <GLOB*\/*\/*.dist>;"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"} else {"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"@files = <*dist>;"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"}"},{"entry":"# initialize - open files"},{"entry":"open DISTOUT,\u201c>$family.ds.dist\u201d || die \u201ccan't open DISTOUT\\n\u201d;"},{"entry":"open LOG, \u201c>$family.distsieve.log\u201d;"},{"entry":"#open FAMOUT, \u201c>$family.list\u201d || die \u201cCan't open $family.list\\n\u201d;"},{"entry":"print LOG \u201c\\#DistSieve Log - totals are hits below $maxE - \u201d;"},{"entry":"print LOG scalar localtime;"},{"entry":"print LOG \u201c\\nFile Name\\t\\t\\tN. hits \\tEmin \\t\\tEmax \\t\\tplotfile\\n\u201d;"},{"entry":"foreach my $file (@files) {"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"$fnum++;"]},{"entry":[{},"$minN = $DEFminN;"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":["#","print FAMOUT \u201c>$fnum $file\\n\u201d;"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"open INFILE, \u201c<$file\u201d || die \u201ccan't open $file\\n\u201d;"]},{"entry":[{},"my $comment = \u201c\u201d;"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"#############################################"},{"entry":"# read in each file gather statistics"},{"entry":"# cutoff only if > maxN hits are obtained"},{"entry":"# or if there are less than $DEFminN hits"},{"entry":"#############################################"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"my @entries;"]},{"entry":[{},"$ecount = 0;"]},{"entry":[{},"while (<INFILE>) {"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"231pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"if (\/%\/) {"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"$comment .= $_;"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"231pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"} else {"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"# get some max val stats and cutoff at $maxE"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"chomp;"]},{"entry":[{},"$entries[$ecount] = $_;"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"# get emin cutoff emax"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"my @toks = split \/\\s+\/, $entries[$ecount];"]},{"entry":[{},"$eval[$ecount] = $toks[4];"]},{"entry":[{},"@toks = split \/e\/, $eval[$ecount];"]},{"entry":[{},"$elog[$ecount] = $toks[1];"]},{"entry":[{},"$delta[$ecount] = $elog[$ecount] - $elog[$ecount-1];"]},{"entry":[{},"if ($ecount == 0) {"]},{"entry":[{},"$emin = $eval[0];"]},{"entry":[{},"$logmax = $elog[0];"]},{"entry":[{},"$minE = $logmax * $Efrac;"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"231pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"}"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"if ( ($ecount < $DEFminN) && ($elog[$ecount] > $maxE) && $minN==$DEFmi"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"$minN = $ecount;"]},{"entry":[{},"goto PASS2;"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"}"]},{"entry":[{},"if ($ecount > $maxN) {"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"goto PASS2;"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"}"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"224pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"$ecount++;"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"231pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"}"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"} # while"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"#############################################"},{"entry":"# perform analysis, get cutoff"},{"entry":"#############################################"},{"entry":"PASS2:"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"$cutoff = 0;"]},{"entry":[{},"$ndrop = 0;"]},{"entry":[{},"for (my $i=$minN; $i < $ecount; $i++) {"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"# enforce E value cutoffs"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"231pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"if ($elog[$i] < $minE) {"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"next;"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"231pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"}"]},{"entry":[{},"if ( $elog[$i] > $maxE ) {"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"if ($cutoff < 1) {"]},{"entry":[{},"$cutoff = $i;"]},{"entry":[{},"}"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"231pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"}"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"# do curve analysis here"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"231pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"if ( ( ($delta[$i] > $delta[$i -1]) && ($delta[$i] < 5) && ($delt"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"if ($cutoff < 1) {"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"$cutoff = $i;"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"}"]},{"entry":[{},"$drop[$ndrop] = $i;"]},{"entry":[{},"$ndrop++;"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"231pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"}"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"}"]},{"entry":[{},"if ($cutoff==0) { $cutoff = $minN }"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"PLOT:"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"$emax = $eval[$cutoff];"]},{"entry":[{},"print \u201c$file has $cutoff \/ $ecount entries emin:$emin emax:$emax logm"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"#############################################"},{"entry":"# output the saved up file, a plot of cutoff"},{"entry":"#############################################"},{"entry":"open PLOTTEMP, \u201c>temp.dat\u201d || die \u201ccan't open plot data file\\n\u201d;"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"for (my $i=0; $i < $ecount; $i++) {"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"print PLOTTEMP \u201c$elog[$i]","$minE \u2003$maxE\\n\u201d;"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"}"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"close PLOTTEMP;"},{"entry":"# output DIST"},{"entry":"$totcnt += $cutoff;"},{"entry":"$tottot += $ecount;"},{"entry":"my $ylab = $elog[$cutoff] -3;"},{"entry":"print DISTOUT \u201c==> $file <==\\n\u201d;"},{"entry":"print DISTOUT \u201c$comment\u201d;"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"189pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":["print LOG \u201c$file","$cutoff\/$ecount \u2003\u2003\\t$emin \u2003\\t$emax \u2003$minE $ndrop\\t\\t"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"open TEMP, \u201c>plot.temp\u201d;"},{"entry":"my $tmp = $file;"},{"entry":"$tmp =~ s\/\\_\/ \/g;"},{"entry":"print TEMP \u201cset title \\\u201cHMM SCREENS FROM $tmp\\\u201d\\n\u201d;"},{"entry":"print TEMP \u201cset out \\\u201c$file.ps\\\u201d\\n\u201d;"},{"entry":"print TEMP \u201cset ylabel \\\u201cE value\\\u201d\\n\u201d;"},{"entry":"print TEMP \u201cset xlabel \\\u201cHit #\\\u201d\\n\u201d;"},{"entry":"print TEMP \u201cset size 1.0, 0.5\\n\u201d;"},{"entry":"print TEMP \u201cset terminal postscript portrait enhanced \\\u201cHelvetica\\\u201d 14\\n\u201d"},{"entry":"print TEMP \u201cset label \\\u201c | cutoff\\\u201d at $cutoff, $ylab \\n\u201d;"},{"entry":"print TEMP \u201cplot \\\u201ctemp.dat\\\u201d using :1 title \\\u201cE scores\\\u201d with lines, \\\\\\"},{"entry":"print TEMP \u201c \\\u201ctemp.dat\\\u201d using :2 title \\\u201cE_M_i_n\\\u201d with lines, \\\\\\n\u201d;"},{"entry":"print TEMP \u201c \\\u201ctemp.dat\\\u201d using :3 title \\\u201cE_M_a_x\\\u201d with lines\\n\u201d;"},{"entry":"close TEMP;"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"for (my $i=0; $i < $cutoff; $i++) {"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"231pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"print DISTOUT \u201c$entries[$i]\\n\u201d;"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"}"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"# gnuplot our file"},{"entry":"close TEMP;"},{"entry":"\u2018gnuplot plot.temp\u2019;"},{"entry":"\u2018rm -f plot.temp\u2019;"},{"entry":"\u2018rm -f temp.dat\u2019;"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"231pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":["}","#foreach"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"print LOG \u201d--------------------------------------------------------------"},{"entry":"TOTAL \u2003\\t\\t\\t$totcnt\/$tottot hits in $fnum files \\n\u201d;"},{"entry":"close LOG;"},{"entry":"close DISTOUT;"},{"entry":"#close FAMOUT;"},{"entry":"exit;"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}]}}}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF DRAWINGS","p":["The accompanying drawings, which are incorporated in and form a part of this specification, illustrate embodiments of the invention and, together with the description, serve to explain the principles of the invention:",{"@attributes":{"id":"p-0013","num":"0012"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0014","num":"0013"},"figref":["FIG. 2","FIG. 1"]},{"@attributes":{"id":"p-0015","num":"0014"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0016","num":"0015"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0017","num":"0016"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0018","num":"0017"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0019","num":"0018"},"figref":"FIG. 7","i":"a\u2013d "},{"@attributes":{"id":"p-0020","num":"0019"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0021","num":"0020"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0022","num":"0021"},"figref":"FIG. 10"}]},"DETDESC":[{},{}]}
