---
title: Method and system for searching for relationships between base sequences in genes
abstract: The present invention provides a method and a system for searching for relationships between base sequences in order to efficiently analyze the relationship between a sample base sequence and known base sequences. In the system, a theoretical value calculating portion calculates theoretical restriction fragment length values based on restriction enzyme data. The comparing portion produces analysis result data by comparing theoretical restriction fragment patterns with measured restriction fragment patterns, calculating the degree of similarity of the sample base sequence to known base sequences.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=07006924&OS=07006924&RS=07006924
owner: 
number: 07006924
owner_city: Kikuchi
owner_country: JP
publication_date: 20010710
---

{"@attributes":{"id":"description"},"BRFSUM":[{},{}],"heading":["BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","TEST EXAMPLE 1","TEST EXAMPLE 2"],"p":["1. Field of the Invention","The present invention relates to a method and a system for searching for relationships between base sequences in genes. More particularly, the present invention relates to a method and a system for searching for the relationships between a sample base sequence and known base sequences based on restriction fragment length polymorphisms (RFLP).","2. Description of Related Art","An initial bacterial classification system has been constructed based on bacterial cell morphology, physiological properties, biochemical activities and the like. For the purpose of identifying an unknown bacterium based on this initial bacterial classification system, a plurality of physiological test results for a bacterium being identified are automatically recorded. The recorded physiological test results are compared with the physiological test results of known bacterial strains already recorded in a stored database. A search is made for the relationship between the unknown bacterium and known bacterial strains to infer the species name of the unknown bacterium. The identification method described above has been utilized in medical and clinical fields as well as in a wide variety of industrial fields such as cosmetics, food hygiene, quality control, and environmental hygiene.","However, precise definitions of species for eubacteria and archaeobacteria, which propagate by cell division, are difficult to construct, in contrast to eucaryotes, which reproduce sexually, and for which precise definitions have been established. Therefore, recently, a new classification system based on ribosomal RNA as a molecular clock has been constructed. To be more precise, in the new classification system, the bacteria are classified in accordance with differences in bacterial 16S rDNA sequences. As a result, there are large discrepancies between the results by the identification method described above and those according to 16S rDNA sequences.","To determine the base sequences, a method utilizing a sequencer device has been widely used. However, the sequencer device is expensive and the number of bases that one sequencer device can read is limited. Thus, the method utilizing a sequencer device is not efficient and incurs high costs for determining the base sequences.","Alternatively, a method has been used which involves digesting an unknown DNA molecule by restriction enzymes and inferring the base sequence thereof based on the lengths of resulting fragments, so as to determine the base sequence. However, this method requires a great amount of time and labor to infer the base sequence by restriction pattern matching of the unknown DNA molecule and known DNA molecules. Thus, this method is also inefficient for identifying base sequences.","The present invention has been made in view of the above problems, and an object thereof is to provide a method and a system for searching for the relationships in base sequences in order to efficiently analyze the relationships of an unknown base sequence to known base sequences based on restriction fragment length polymorphisms (RFLP) thereof.","To solve the above problem, a system for searching for relationships between base sequences in genes according to the present invention comprises a theoretical value calculating portion which calculates theoretical restriction fragment length values of known genes digested by restriction enzymes based on restriction enzyme data in which kinds of restriction enzymes, recognition sites recognized by the restriction enzymes, and restriction sites at which the restriction enzymes cut in the recognition sites are stored in a related manner, and base sequence data in which kinds of known genes and base sequences of the known genes are stored in a related manner, and which outputs theoretical restriction fragment patterns in which the kinds of known genes, the kinds of restriction enzymes, and theoretical restriction fragment length values are stored in a related manner; and a comparing portion which compares the theoretical restriction fragment patterns with measured restriction fragment patterns in which the kinds of restriction enzymes and measured restriction fragment length values obtained as a result of digesting a sample by the restriction enzymes followed by measurement are stored in a related manner, calculates the degree of similarity of the known gene to the sample, and outputs the similarity as analysis result data.","The system for searching for relationships between base sequences according to the present invention may further comprise a displaying portion that diagrammatically displays samples having a high degree of similarity therebetween or a known gene and a sample having a high degree of similarity therebetween in a related manner based on the analysis result data.","In the system for searching for relationships between base sequences in genes according to the present invention, the displaying portion may display samples having a high degree of similarity therebetween or a known gene and a sample having a high degree of similarity therebetween in a related manner by a dendrogram.","In the system for searching for relationships in base sequences according to the present invention, the comparing portion may calculate the degree of similarity based on the theoretical restriction fragment patterns and the measured restriction fragment patterns by using an unweighted-pair-group method with arithmetic mean.","The system for searching for relationships between base sequences according to the present invention may further comprise an amplified sequence recognizing portion which reads pre-amplification base sequence data, and which produces post-amplification base sequence data based on primer data including recognition site data of primers used for the amplification, and the theoretical value calculating portion may calculate the theoretical restriction fragment length values based on the post-amplification base sequence data. Here, the term \u201camplification\u201d means extraction of a certain portion of a base sequence located between a predetermined forward primer and a predetermined reverse primer.","A method for searching for relationships between base sequences in genes according to the present invention comprises: a theoretical value calculating step for calculating theoretical restriction fragment length values of known genes digested by restriction enzymes based on restriction enzyme data in which kinds of restriction enzymes, recognition sites which are recognized by the restriction enzymes, and restriction sites at which the restriction enzymes cut in the recognition sites, are stored in a related manner, and base sequence data in which kinds of known genes and base sequences of the known genes are stored in a related manner, and outputting theoretical restriction fragment patterns in which kinds of known genes, the kinds of restriction enzymes, and theoretical restriction fragment length values posited to be obtained as a result of digesting known genes by the restriction enzymes are stored in a related manner, and a comparing step for comparing the theoretical restriction fragment patterns with measured restriction fragment patterns including measured restriction fragment length values obtained as a result of digesting a sample by the restriction enzymes followed by measurement are stored in a related manner, calculating the degree of similarity of the known genes to the sample, and outputting it as analysis result data.","The method for searching for relationships between base sequences according to the present invention may further comprise a displaying step for diagrammatically displaying samples having a high degree of similarity therebetween or a known gene and a sample having a high degree of similarity therebetween in a related manner based on the analysis result data.","In the method for searching for relationships in base sequences according to the present invention, the displaying step may display samples having a high degree of similarity therebetween or a known gene and a sample having a high degree of similarity therebetween in a related manner by a dendrogram.","In the method for searching for relationships between base sequences according to the present invention, the comparing portion may calculate the degree of similarity based on the theoretical restriction fragment patterns and the measured restriction fragment patterns by using an unweighted-pair-group method with arithmetic mean.","The method for searching for relationships between base sequences according to the present invention may further comprise an amplified sequence recognizing step for reading pre-amplification base sequence data, and producing post-amplification base sequence data based on primer data including recognition site data of the primer used for the amplification, and the theoretical value calculating step may calculate the theoretical restriction fragment length values based on the post-amplification base sequence data.","A recording medium readable on a computer according to the present invention has recorded therein a computer program for implementing: a theoretical value calculating step for calculating theoretical restriction fragment length values of knowns gene digested by restriction enzymes based on restriction enzyme data in which kinds of restriction enzymes, recognition sites recognized by the restriction enzymes, and restriction sites at which the restriction enzymes cut in the recognition sites are stored in a related manner, and base sequence data in which kinds of known genes and base sequences of the known genes are stored in a related manner, and outputting theoretical restriction fragment patterns in which the kinds of known genes, the kinds of restriction enzymes, and theoretical restriction fragment length values are stored in a related manner; and a comparing step for comparing the theoretical restriction fragment patterns with measured restriction fragment patterns in which the kinds of restriction enzymes and measured restriction fragment length values obtained as a result of digesting a sample by the restriction enzymes followed by measurement are stored in a related manner, calculating the degree of similarity of the known genes to the sample, and outputting it as analysis result data.","The recording medium readable on a computer according to the present invention may further have recorded therein a computer program for implementing a displaying step for diagrammatically displaying samples having a high degree of similarity therebetween or a known gene and a sample having a high degree of similarity therebetween in a related manner based on the analysis result data.","In the recording medium readable on a computer according to the present invention, the displaying step may display samples having a high degree of similarity therebetween or a known gene and a sample having a high degree of similarity therebetween in a related manner by a dendrogram.","In the recording medium readable on a computer according to the present invention, the comparing portion may calculate the degree of similarity based on the theoretical restriction fragment patterns and the measured restriction fragment patterns by using an unweighted-pair-group method with arithmetic mean.","The recording medium readable on a computer according to the present invention may further have recorded therein a computer program for implementing an amplified sequence recognizing step for reading pre-amplification base sequences, and producing post-amplification base sequences based on primer data including the recognition site data of primers used for the amplification, and the theoretical value calculating step may calculate the theoretical restriction fragment length values based on the post-amplification base sequence data.","Hereinafter, embodiments according to the present invention will be explained in detail with reference to the attached drawings.","First Embodiment",{"@attributes":{"id":"p-0046","num":"0045"},"figref":["FIG. 1","FIG. 1"],"b":["21","21"]},"A post-amplification base sequence data shows information on a known gene in a region where it is expected to be amplified by a polymerase chain reaction using the known gene as a template and using a desired primer set (including a forward primer and a reverse primer). The post-amplification base sequence data includes the kind of known gene and the base sequence of the known gene in the region to be amplified in a related manner. The number of the above-described primer set and known gene may be one or more and the above-described post-amplification base sequence data may show information on one or more known genes in one or more kinds of regions. The post-amplification base sequence data can be obtained from the open base sequence database, and be manually or otherwise edited, for example.","In a theoretical value calculating portion , theoretical restriction fragment length values of known genes, which are expected to be obtained by digesting the known genes in the amplified regions by the restriction enzymes, are calculated based on the above-described restriction enzyme data  and post-amplification base sequence data , and theoretical restriction fragment patterns  are output.","In a measured value inputting portion , measured restriction fragment patterns  are input. The measured restriction fragment patterns  include measured restriction fragment length values obtained by digesting a sample by the same restriction enzymes as used for calculating the above-described theoretical restriction fragment length values and measuring the lengths of the digested base sequences as well as the kind of the restriction enzymes in a related manner.","In a comparing portion , the theoretical restriction fragment patterns  and the measured restriction fragment patterns  are compared with each other, the degree of similarity between the theoretical restriction fragment patterns  and the measured restriction fragment patterns  is calculated, and the degree of similarity of the known gene and the sample is output as analysis result data .","An allowance setting portion  sets up allowance limits of error for comparing the theoretical restriction fragment patterns  and the measured restriction fragment patterns . The allowance limits of error are stored as allowance limits data . The comparing portion  refers to the allowance limits data  and treats the theoretical restriction fragment patterns  and the measured restriction fragment patterns  as being identical with each other when the difference therebetween is within the allowance limits of error, and then calculates the degree of similarity between the theoretical restriction fragment patterns and the measured restriction fragment patterns .","A displaying portion  relates samples having a high degree of similarity therebetween to each other or relates a known gene and a sample having a high degree of similarity therebetween to each other based on the analysis result data  output by the comparing portion  and displays them diagrammatically.","Here, the details of the post-amplification base sequence data will be described.  shows the first half of the post-amplification base sequence data (base sequence data) while  shows the second half of the post-amplification base sequence data . As shown in , the post-amplification base sequence data are text format data. The first half of the post-amplification base sequence data includes at least the DNA number (ACCESSION), the name of organism of origin (ORGANISM), and the name of the known gene (KEYWORDS) while the second half of the post-amplification base sequence data includes known gene base sequence (ORIGIN) (SEQ ID NO. 1) in the region where it is expected to be amplified by polymerase chain reaction using the known gene as a template and using a desired primer set (forward primer and reverse primer). In the base sequence data, the letter \u201ca\u201d represents adenine, \u201cg\u201d represents guanine, \u201cc\u201d represents cytosine, and \u201ct\u201d represents thymine.","Next, the details of the restriction enzyme data  will be described.  shows the data structure of the restriction enzyme data . The restriction enzyme data  are registered in the restriction enzyme database, which are shown in a table format having columns for the restriction enzyme number, the restriction enzyme name, the recognition site recognized by each restriction enzyme, and the restriction site at which the restriction enzyme cut in the recognition site, as shown in . The restriction enzyme number is a number automatically given when a new restriction enzyme data is added to the database through a user interface (not shown in the figures). The recognition site recognized by each restriction enzyme includes the restriction site at which the restriction enzyme cut in the recognition site. The restriction site is indicated by the number of bases from the 5\u2032-terminal side of the recognition site. For example, SmaI, which is set to be restriction enzyme number  in , recognizes a recognition site \u201c . . . cccggg . . . \u201dof a target DNA molecule, and cuts it at the restriction site which is the third bond, between the third base (c) and the fourth base (g). As a result, the target DNA molecule is digested into \u201c . . . ccc\u201d and \u201cggg. . . \u201d. At present, data on 57 kinds of restriction enzymes are registered.",{"@attributes":{"id":"p-0055","num":"0054"},"figref":"FIG. 5","b":["11","22","21"],"i":"b "},"The user may select the desired kind of known gene and restriction enzyme name and push the \u201cRESTRICTION FRAGMENT LENGTH CALCULATION\u201d button on the lower portion of the user interface to give the instruction for the calculation of the theoretical restriction fragment length value to the theoretical value calculating portion .","When the \u201cRESTRICTION FRAGMENT LENGTH CALCULATION\u201d button is pushed, the theoretical value calculating portion  reads the base sequence of a known gene in the amplified region of the post-amplification base sequence data and the recognition site of the restriction enzyme from the restriction enzyme data , and compares them with each other. A search is conducted from the 5\u2032 end terminal of the base sequence of the known gene to determine whether or not the recognition site exists in the base sequence of the known gene in the region to be amplified. If recognition sites are present in the base sequence of the known gene, the number of bases present from the recognition site closest to the 5\u2032 end terminal of the base sequence up to the 5\u2032 end terminal is calculated as a theoretical restriction fragment length value. Then, the numbers of bases present between the adjacent restriction sites are respectively calculated and are input as theoretical restriction fragment length values. Then, the theoretical restriction fragment length values are output as theoretical restriction fragment pattern .",{"@attributes":{"id":"p-0058","num":"0057"},"figref":["FIG. 6","FIG. 6"],"b":["23","23","23","23","22","11","21","11","11","1"],"i":"b "},"Next, the measured restriction fragment pattern  will be described.","To measure the restriction fragment length of a sample, the base sequence contained in the sample is amplified by polymerase chain reaction (PCR). As a primer set used for the PCR, the primer set used for obtaining the post-amplification base sequence data is preferred. After the amplification, the amplified DNA molecule is digested by the restriction enzyme used for obtaining the theoretical restriction fragment pattern . Then, the lengths of digested fragments are measured, for example, by an agarose gel electrophoresis method, polyacrylamide gel electrophoresis method, a microchip electrophoresis method, or the like method, and are input into the measured value inputting portion  as measured restriction fragment length value.",{"@attributes":{"id":"p-0061","num":"0060"},"figref":["FIG. 7","FIG. 7"],"b":["12","21","25","12","12","12"]},{"@attributes":{"id":"p-0062","num":"0061"},"figref":["FIG. 8","FIG. 8"],"b":["25","25","25","2"]},"The comparing portion  compares the theoretical restriction fragment patterns  with the measured restriction fragment patterns , and analyzes the relationship therebetween. In the present embodiment, the comparing portion  produces analysis result data  by using an unweighted-pair-group method with arithmetic mean (UPGMA).",{"@attributes":{"id":"p-0064","num":"0063"},"figref":["FIG. 9","FIG. 9","FIG. 9","FIG. 9"],"b":["13","23","25","1","8"]},"The comparing portion  reads the theoretical restriction fragment patterns  or the measured restriction fragment patterns  and calculates the degree of similarity between the data given by two DNA numbers. For example, the definition of degree of similarity S(A, B) is given by the following formula:\n\n()=(total number of common bands that and have)\u00d72\/(number of bands that has+number of bands that has).\n","In , A has five bands with band numbers of , , ,  and , respectively, while B has six bands with band numbers of , , , , , and , respectively. Therefore, the four bands with band numbers , , , and  are common bands that A and B have, and the total number of common bands that A and B have is 8. Therefore,\n\n()=4\u00d72\/(6+5)\n\nand hence the degree of similarity is calculated to be 8\/11 (about 0.727).\n","As the measured restriction fragment length value contains an error, the comparing portion  uses allowance limits data  set by an allowance setting portion before a judgment can be made as to whether or not a certain band is a common band. More particularly, when the distance between the bands to be judged is within the allowance limits of error contained in the allowance limits data , they are treated as common bands. In , the allowance limits of error is expressed as a ratio of the longitudinal distance between the bands compared with each other to the longitudinal distance between the bands with band numbers  and . The allowance limits of error is properly set depending on the electrophoresis method used and preferably is on the order of 10% to 5%. For example, preferably it is about 10% in agarose gel electrophoresis, about 7% in low melting point agarose gel electrophoresis, and about 5% in polyacrylamide gel electrophoresis. If the allowance limits of error is set too low, a problem arises in that inherently common bands will appear to be different bands. On the other hand, if the allowance limits is too high, a problem arises in that inherently different bands will appear to be common bands.","Next, the comparing portion  performs cluster analysis of base sequences based on the calculated degrees of similarity. , B, and C are tables showing an example of the process of cluster analysis performed by the comparing portion  using a mean distance method based on the degrees of similarity. In , B, and C, four base sequences A, B, C, and D are taken as subjects of analysis.  shows the calculated degrees of similarity between any two of A, B, C, and D. In the table shown in , the combination that shows the highest degree of similarity is A and C (the degree of similarity: 0.8). Accordingly, A and C are combined to form a cluster of (A+C) and the next step is followed.",{"@attributes":{"id":"p-0069","num":"0068"},"figref":["FIG. 10B","FIG. 10A"]},"For example, as the similarity between B and the cluster (A+C), i.e., S(B, A+C), a mean value of S(B, A) and S(B, C) is calculated to be 0.6. Also, as the degree of similarity between D and the cluster (A+C), i.e., S(D, A+C), a mean value of S(D, A) and S(D, C) is calculated to be 0.2. Then, in the table shown in , the combination that shows the highest degree of similarity is (A+C) and B. Accordingly, (A+C) and B are combined to form a cluster of ((A+C)+B), and the next step is followed.",{"@attributes":{"id":"p-0071","num":"0070"},"figref":["FIG. 10C","FIG. 10B"]},{"@attributes":{"id":"p-0072","num":"0071"},"figref":["FIG. 11","FIG. 11"]},"The displaying portion  reads the analysis result data  and diagrammatically displays the samples registered by the user and a known gene having a high degree of similarity thereto in a related manner.  is a desktop image diagram showing an example of an analysis result displayed by the displaying portion  as a dendrogram.  shows the results of cluster analysis of a plurality of DNA samples (AP1, AP2, and AP3) and a plurality of known genes.","Second Embodiment","In the second embodiment, the post-amplification base sequence data are automatically produced based on the pre-amplification base sequence data . In other words, the region that is to be amplified by the polymerase chain reaction using a primer set (forward primer and reverse primer) is automatically recognized in the pre-amplification base sequence of a known gene and the post-amplification base sequence is extracted.",{"@attributes":{"id":"p-0075","num":"0074"},"figref":["FIG. 13","FIG. 13"],"b":["22","31","22","22"],"i":["a ","a","b. "]},{"@attributes":{"id":"p-0076","num":"0075"},"figref":["FIG. 14","FIG. 15","FIG. 15","FIG. 15","FIG. 3","FIG. 15","FIG. 3","FIG. 15"],"b":["22","22","22","22","22"],"i":["a ","a","a ","a ","b"]},"At first, the primer data, which includes information on the base sequences, the names, and the allowance limits for mismatch of predetermined forward primer and reverse primer, are input and registered in the primer database. The folder having the names of both primers, in which the post-amplification base sequence is saved, is automatically formed when a new primer data is added to the database (not shown in the figures).  is a desktop image diagram showing a selection of the primer data of the primer set as described above. (In the screen shown in , the user can select the respective primer data.) The forward primer used in  is named \u201c, has a base sequence of \u201cgctcagattgaactcggcg\u201d of SEQ ID NO. 3, and has an allowance limit for mismatch of 4%. Similarly, the reverse primer is named \u201c1066r\u201d, has a base sequence of \u201cacatttcacaacacgagctg\u201d, and has an allowance limit for mismatch of 4. The primer data thus selected are utilized by the amplified sequence recognizing portion  when the base sequence in the region to be amplified by using the primer set is extracted from the pre-amplification base sequence of the known gene.","The amplified sequence recognizing portion  reads the pre-amplification base sequence data , and compares the pre-amplification base sequence of the known gene with the base sequence of the forward primer. Within the allowance limits for mismatch, the region, where the pre-amplification base sequence and the base sequence of the forward primer match with each other, is searched from the 5\u2032 terminal side thereof in order. Then, the pre-amplification base sequence is compared with the base sequence of the reverse primer. Within the allowance limits for mismatch, the region, where the pre-amplification base sequence and the base sequence of the reverse primer match with each other, is searched from the 3\u2032 terminal side thereof in order. The base sequence in the region sandwiched by the forward primer and the reverse primer is output as the post-amplification base sequence of the known gene (ORIGIN) in the post-amplification base sequence data and is saved in the primer folder described above.","The above-described system for searching for relationships between base sequences in genes is realized by using a computer system. A computer program that enables a computer to implement respective operations of the above-described theoretical value calculating portion, measured value inputting portion, comparing portion, allowance limits setting portion, and displaying portion is stored in a recording medium readable on a computer. The computer reads out the computer program and implements it so that the above-described operations can be performed. Here, the recording medium readable on a computer includes a floppy disk, a magneto-optical disk, CD-ROM, DVD-ROM, a magnetic hard disk, a semiconductor memory and the like.","As the computer system for realizing the system for searching for relationships between base sequences in genes, a general-purpose computer such as a personal computer or a workstation can be used.","The format of data treated by the system for searching for relationships between base sequences in genes is not limited to those described in the first and second embodiments described above, and data in other formats or expressions may also be used. A portion of the data in a table format as described in the above embodiments is in a non-normalized form. Such data may be normalized. For example, in the theoretical restriction fragment patterns shown in , the DNA numbers, the names of the organisms, and the names of genes are shown in the same table. However, they may be shown in separate tables.","Next, test examples in which classification of bacteria using the system for searching for relationships between base sequences in genes according to the present invention are performed will be described.","The 16S rDNA sequence data is downloaded genus by genus from the DNA information supplied by the National Institute of Agrobiological Sciences, Ministry of Agriculture and Fisheries, Japan. The DNA information supplied by the National Institute of Agrobiological Sciences is constructed based on the DNA information obtained from the Gene Bank (National Center for Biotechnology Information), DDBJ (National Institute of Genetics), and EMBL (European Molecular Biology Laboratory). Also, 5\u2032-gctcagattgaacgctggcg-3\u2032 (SEQ ID NO:3) as the forward primer (), 5\u2032-acatttcacaacacgagctg-3\u2032 (SEQ ID NO:4) as the reverse primer (1066r), and fourteen kinds of restriction enzymes were input. The region sandwiched by the forward primer and the reverse primer was extracted as post-amplification base sequence data from the pre-amplification base sequence data by the amplified sequence recognizing portion , and 357 genera, 1233 species, or 1503 kinds of base sequence data were obtained as the post-amplification base sequence data . Then, the lengths of fragments digested by the restriction enzymes were calculated and output as theoretical restriction fragment patterns by the theoretical value calculating portion .","Then, alkylphenol decomposing bacteria isolated from various soils (from Kyushu Kyoritsu University, Meijo University) were used as samples AP1 to AP9 and alkali resistant bacteria (from Yamaguchi University) were used as samples MA1 to MA4. Respective chromosomal DNAs was extracted from the samples by a conventional method and PCR was performed by using the forward primer and reverse primer to amplify 16S rDNA. The amplified 16S rDNA was digested by the restriction enzymes used for calculating the theoretical restriction fragment length value. Then, the lengths of the resulting digested fragments were measured using densitometory and defined as measured restriction fragment length values. As sample MA11, the same strain as the sample MA1 was used, the same operations as for MA1 were performed, the measured restriction fragment length values were determined, and measured restriction fragment patterns were obtained.",{"@attributes":{"id":"p-0085","num":"0084"},"figref":["FIG. 17","FIGS. 18 to 27","FIGS. 18","FIGS. 18 to 27"],"b":["27","13","27","13","19","20","21","22","23","24","25","26","27","27","23"]},"From the above-described analysis results, it was revealed that samples MA1 and MA11 have the closest relationship to of the group belonging to the Proteobacteria c subdivision and also that they have close relationships to the genus , the genus , or the genus of the Enterobacteriaceae belonging to the Proteobacteria c subdivision.","Also, it was revealed that the sample MA2 even has a close relationship to the genus or the genus (both are low GC gram positive bacteria, , and Bacilluceae (family)).","The sample MA3 was revealed to have the closest relationship to and (low GC gram positive bacteria, , and Bacilluceae). Note that and have been indicated to be related to each other and there are many opinions that they are of the same species.","The sample MA4 was indicated to have the closest relationship to the genus or the genus (both are Actinobacteria, Actinobacteriadae, Actinomycetales, Micrococcineae, and Cellulomonadaceae) and a close relationship to the genus (Actinobacteria, Actinobacteridae, Actinomycetales, Actinomyccineae, and Actinomycetaceae), and the genus (Actinobacteria, Actinobacteridae, Actinomycetales, Streptomycineae, and Streptomycetaceae).","It was revealed that the samples AP1 and AP3 each have a close relationship to , and (each belonging to the Proteobacteria c subdivision, Pseudomonadaceae).","Further, the sample AP6 has a close relationship to bacteria of the genus (Proteobacteria c subdivision, Lysobacterrales, and Xanthomonas group) and the samples AP2 and AP5 were posited to belong to the genus , the genus , the genus (Proteoacteria, a subdivision, Rhizobiaceae group, and Phylobacteriaceae or Rhizobiaceae, respectively).",{"@attributes":{"id":"p-0092","num":"0091"},"figref":["FIG. 28","FIGS. 18 to 27"],"b":["27","13"]},"Thus, using the present test example, data on very many kinds of known genes including base sequence data newly included in the future could be obtained at low cost and in a short time based on publically available data. Upon performing classification and identification of the kind of unknown gene contained in the sample, comparison with many kinds of known genes can be performed at high speed so that the degree of similarity between known bacterial strains and unknown bacterial strains can be calculated efficiently and with high precision.","The species names of 134 strains of denitrifying bacterium isolated by a conventional method were inferred based on the above-described method. Thereafter, the partial 16S rDNA sequences of representative bacterial strains were actually determined by the ABI 373 sequencing system (Perkin-Elmer Japan). Then, a homology search was performed using public sequence databases using the BLAST algorithm, so as to search for the species name of the bacterium having the closest base sequence to that of the 16S rDNA.  shows the group number (I to XIII) obtained by classifying the 134 strains of isolated denitrifying bacterium, the number of isolates contained in the group, the species name inferred by the above-described method (RFLP), the name of a bacterium having a base sequence having a high degree of homology revealed by actually determining the base sequence of 16S rDNA, and degrees of difference. The degrees of difference as described above were obtained by determining the 16S rDNA sequence obtained from each isolate belonging to the same group, performing a homology search for the obtained base sequences, respectively, indicating the ratio of the number of matched bases to the total number of bases in the base sequence in the region having high homology in terms of percentage (%). This revealed that the species names posited by the above-described method well correspond with those obtained by the comparative analyses of base sequences.","As described above, according to the present invention, it is possible to calculate the degree of similarity between a known gene and a sample efficiently and with high precision. Furthermore, use of the present invention enables not only a search of the relationships between procaryotic organisms, but also a search of the relationships between eucaryotic organisms. Also, in order to relate those eucaryotic organisms having a high degree of similarity to each other based on the degree of similarity between many kinds of known genes and a sample, and diagrammatically displaying the relationship using a dendrogram, it is possible to provide analysis results in a form that is understandable to users.","Furthermore, according to the present invention, the amplified sequence recognizing portion automatically recognizes the amplified portion and produces post-amplification base sequence data and hence post-amplification base sequence data can be prepared without much time and labor so that the analysis of base sequences can be made more efficient.","Moreover, according to the present invention, realization of the system for searching for relationships in base sequences using a general purpose computer such as a personal computer or a workstation can provide a system that is extremely inexpensive compared with a special sequencer or the like."],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF DRAWINGS","p":[{"@attributes":{"id":"p-0025","num":"0024"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0026","num":"0025"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0027","num":"0026"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0028","num":"0027"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0029","num":"0028"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0030","num":"0029"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0031","num":"0030"},"figref":"FIG. 7"},{"@attributes":{"id":"p-0032","num":"0031"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0033","num":"0032"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0034","num":"0033"},"figref":"FIGS. 10A to 10C"},{"@attributes":{"id":"p-0035","num":"0034"},"figref":"FIG. 11"},{"@attributes":{"id":"p-0036","num":"0035"},"figref":"FIG. 12"},{"@attributes":{"id":"p-0037","num":"0036"},"figref":"FIG. 13"},{"@attributes":{"id":"p-0038","num":"0037"},"figref":"FIG. 14"},{"@attributes":{"id":"p-0039","num":"0038"},"figref":"FIG. 15"},{"@attributes":{"id":"p-0040","num":"0039"},"figref":"FIG. 16"},{"@attributes":{"id":"p-0041","num":"0040"},"figref":"FIG. 17"},{"@attributes":{"id":"p-0042","num":"0041"},"figref":"FIGS. 18 to 27"},{"@attributes":{"id":"p-0043","num":"0042"},"figref":"FIG. 28"},{"@attributes":{"id":"p-0044","num":"0043"},"figref":"FIG. 29"}]},"DETDESC":[{},{}]}
