---
title: Expression of human milk proteins in transgenic plants
abstract: The invention is directed to food and food additive compositions comprising one or more human milk proteins produced in the seeds of a transgenic plant and methods of making the same. The invention is further directed to improved infant formula comprising such food supplement composition.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=06991824&OS=06991824&RS=06991824
owner: Ventria Bioscience
number: 06991824
owner_city: Sacramento
owner_country: US
publication_date: 20020214
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"p":["This application claims priority benefit to U.S. provisional application Ser. No. 60\/269,199, filed Feb. 14, 2001, for \u201cExpression of Human Milk Proteins in Transgenic Plants\u201d, which is incorporated herein in its entirety. The present application is also a continuation-in-part of U.S. patent application Ser. No. 09\/847,232, filed May 2, 2001, for \u201cPlant Transcription Factors and Enhanced Gene Expression\u201d, which claims the benefit of U.S. provisional application Ser. No. 60\/266,929, filed Feb. 6, 2001, and U.S. provisional application Ser. No. 60\/201,182, filed May 2, 2000, all of which are incorporated herein by reference. The corresponding PCT application No. PCT\/US01\/14234, International Publication No. WO 01\/83792 A1, published Nov. 8, 2001, is also incorporated herein by reference.","The present invention relates to human milk proteins produced in the seeds of transgenic plants, seed extracts containing the proteins, transgenic plants and seeds, and methods for producing and using the same.","The following references are cited herein, and to the extent they may be pertinent to the practice of the invention, are incorporated herein by reference.","Alber and Kawasaki, 1:419-434, 1982.","Aniansson, G. et al. (1990). \u201cAnti-Adhesive Activity Of Human Casein Against And 8(5):315-323.","Arnold R. R. et al., 28:893-898, 1980.","Ausubel F. M. et al., , John Wiley & Sons, New York, N.Y., 1993.","Bhan, M. K. et al., 7:208-213, 1988.","Beatty, K., et al., \u201cKinetics of association of serine proteinases with native and oxidized alpha-1-proteinase inhibitor and alpha-1-antichymotrypsin,\u201d 255:3931-3934, 1980.","Boesman-Finkelstein M. and Finkelstein R. A. 144:1-5, 1982.","Bradford M. 72:248-254, 1976.","Brandt, et al., 50:333-345, 1985.","Briggs, D, et al. (1981). 1, (Chapman & Hall, New York, 2nd edition).","Briggs, D (1998). (Blackie Academic And Professional, New York).","Bullen J. J. et al., 1, 69-75, 1972.","Carrell, R. W. et al. \u201cStructure and variation of human alpha-1-antitrypsin\u201d, 298:329-334, 1983.","Casta\u00f1\u00f3n M. J. et al., 66:223-234, 1988.","Chandan R. C., 51:606-607, 1968.","Chong, D. K. and Langridge, W. H., 9:71-8, 2000.","Chowanadisai, W. and L\u00f6nnerdal, B., \u201cAlpha-1-antitrypsin and antichymotrypsin in human milk: origin, concentrations, and stability\u201d, 2001 (in press).","Chrispeels K., 4:21-53, 1991.","Clark et al., 264:17544-17550, 1989.","Davidson L A, et al. (1987). \u201cPersistence Of Human Milk Proteins In The Breast-Fed Infant\u201d, 76(5):733-740.","Davidson, L. A., and L\u00f6nnerdal, B. \u201cFecal alpha-1-antitrypsin in breast-fed infants is derived from human milk and is not indicative of enteric protein loss\u201d, 79:137-141, 1990.","della-Cioppa et al., 84:965-968, 1987.","Dellaporta S. L. et al., 1:19-21, 1983.","Depicker et al., 1:561-573, 1982.","Dewey K. G. et al., 126:696-702, 1995.","Dewey K. G. et al., 89:1035-1041, 1992.","Dewey K. G. et al., 57:140-145, 1993.","Ditta et al., 77:7347-7351, 1980.","Doncheck, J (1995). In: -(John Wiley & Sons, New York, 4th edition), volume 15, pp. 947-962.","Eley, E, (1990), August 1990, pp. 17-18","Faure A. and Joll\u00e8s P., et al., 271:1916-1918, 1970.","Fujihara T. and Hayashi K., 140:1469-1472, 1995.","Gasta\u00f1aduy, A. et al., 11:240-6, 1990.","Gelvin, S. B. et al., eds. 1990.","Gielen, et al., 3:835-846, 1984.","Goto, F et al., 17:282-286, 1999.","Grover M. et al., 86:315-316, 1997.","Hamosh, M, et al. (1999). \u201cProtective Function Of Human Milk: The Fat Globule,\u201d23(3):242-9.","Harmsen M. C. et al., 172:380-388, 1995.","Hickenbottom, J W (1977). December 1977, pp. 18-22.","Hickenbottom, J W (1983). V(3), March 1983, pp. 1-8.","Hickenbottom, J W (1996). 41(10): 778-790.","Hickenbottom J W (1997). Technical Bulletin #238, August 1997, pp. 1010-1013.","Huang N. et al., 23:737-747, 1993.","Hough, J S, et al. (1982).). 2, (Chapman & Hall, New York, 2nd edition).","Huang N. et al., 1:73-86, 1990a.","Horvath H. et al., 97:1914-1919, 2000.","Humphreys, D. P. and Glover, D. J. \u201cTherapeutic antibody production technologies: molecules, applications, expression and purification,\u201d March; 4(2):172-85, 2001.","Huang N. et al., 14:655-668, 1990b.","Huang, J., et al., \u201cExpression and purification of functional human alpha-1-antitrypsin from cultured plant cells\u201d, 17:126-133, 2001.","Jensen L. G. et al., 93:3487-3491, 1996.","Jigami Y. et al., 43:273-279, 1986.","Johnson, T. \u201cHuman alpha-1-proteinase inhibitor\u201d, 80:756-757, 1981.","Joll\u00e8s P., . Birkh\u00e4user Verlag, Basel; Boston, 1996.","Jouanin et al., 201:370-374, 1985.","Kortt A. A. et al., \u201cDimeric and trimeric antibodies: high avidity scFvs for cancer targeting,\u201d 18:95-108, 2001.","Kovar M. G. et al., 74:615-638, 1984.","Kunz C. et al., 26(2):307-33, 1999.","Lake, C, (1988). February 1988, pp. 55-57.","Langridge et al., 156:166-170, 1982.","Larrick, J. W. et al., \u201cProduction of antibodies in transgenic plants\u201d, 149:603-8, 1998.","L\u00e1sztity, R. CRC Press, Boca Raton, 1996.","Lee-Huang S. et al., 96:2678-2681, 1999.","Lindberg, T. \u201cProtease inhibitors in human milk\u201d, 13:969-972, 1979.","Lollike K. et al., 9:206-209, 1995a.","L\u00f6nnerdal B., 42:1299-1317, 1985.","L\u00f6nnerdal, B. \u201cRecombinant milk proteins\u2014an opportunity and a challenge\u201d, 63:622S-626S, 1996.","Maga E. et al., 61:52-56, 1998.","Maga E. et al., 3:36-42, 1994.","Maga E. et al., 78:2645-2652, 1995.","Maniatis, et al., MCA LM, 2nd Edition, 1989.","Matsumoto, A. et al., 27:1163-72, 1995.","Maynard, J. and Georgiou, G. Antibody engineering, Annu Rev Biomed Eng 2:339-76, 2000.","McBride and Summerfelt, 14:269-276, 1990.","Mcgilligan, K M, et al., (1987), \u201cAlpha-1-Antitrypsin Concentration In Human Milk,\u201d 22(3):268-270.","Mitra A. and Zhang Z., 106:977-981, 1994.","McGilligan, K. M. et al., \u201cAlpha-1-antitrypsin concentration in human milk,\u201d 22:268-270, 1987.","Mitra, A, and Zhang, Z., 106:977-81, 1994.","Moe, T, et al. (1995). -29:635-649.","Moore, K (1978). \u201cRice-Barley Malt Combo Improves Functionality And Nutrition Of Meats, Breads, And Baked Goods\u201d, 12(5):74.","Moreira-Ludewig, R. and Healy, C., 27:75-100, 1992.","Motil K. J., 12(5):469-76, 2000.","Murphy M S (1998). \u201cGrowth Factors And The Gastrointestinal Tract\u201d, 14(10):771-4.","Nakajima H. et al., 16:674-679, 1997.","Newburg, D S (1999). \u201cHuman Milk Glycoconjugates That Inhibit Pathogens\u201d, 6(2):117-127.","Newburg, D S, et al. (1998). \u201cRole Of Human-Milk Lactadherin In Protection Against Symptomatic Rotavirus Infection\u201d, 351(1910):1160-4.","NIH publication, American Academy of Allergy and Immunology Committee on adverse reaction to food and National Institute of Allergy and Infectious Diseases, NIH, Bethesda, 1984.","Peeters, K., et al., \u201cProduction of antibodies and antibody fragments in plants\u201d, 9:2756-61, 2001.","Peterson J A, et al. (1998). \u201cGlycoproteins Of The Human Milk Fat Globule In The Protection Of The Breast-Fed Infant Against Infections\u201d, 74(2):143-162.","Piper, D. W. and Fenton, B. H. \u201cpH stability and activity curves of pepsin with special reference to their clinical importance\u201d, 6:506-508, 1965.","Prosser, C G (1996). \u201cInsulin-Like Growth Factors In Milk And Mammary Gland,\u201d (3):297-306.","Raikhel N., 100:1627-1632, 1992.","Rey, M. W., 18\u20335288, 1990.","Romer et al., 196:1414-1421, 1993.","Rudloff, S. and L\u00f6nnerdal, B. \u201cSolubility and digestibility of milk proteins in infant formulas exposed to different heat treatments\u201d, 15:25-33, 1992.","Saarinen K. M. et al., 478:121-30, 2000.","Salmon V. et al., 9:203-210, 1997.","Salmon, V. et al., 13:127-135, 1998.","Samaranayake Y. H. et al., 105:875-883, 1997.","Sambrook J. et al., (Second Edition), Cold Spring arbor Press, Plainview, N.Y., 1989.","Satue-Gracia M. T. et al., 48(10):4984-90, 2000.","Sch\u00fctte H. and Kula M. R., 12:599-620, 1990.","Sfat, M, et al (1981). In: -(John Wiley & Sons, New York, 3rd edition), volume 14, pp. 810-823.","Shah et al., 233:478-481, 1986.","Shin, K, et al. (2000). \u201cPCR Cloning And Baculovirus Expression Of Human Lactoperoxidase And Myeloperoxidase\u201d, 27(13):831-836.","Shugar D., 8:680-68, 1952.","Symbicon A B (1995). \u201cHuman Milk Kappa-Casein And Inhibition Of Adhesion To Human Gastric Mucosa\u201d, 21(3):288-296.","Takai I. et al., 685:21-25, 1996.","Tsuchiya K. et al., 38:109-114, 1992.","Von Heijne et al., 9:104-126, 1991.","Wang C. S. and Kloer H. U., 139:224-227, 1984.","Wang C. et al., 78B:575-580, 1984.","Ward P. P. et al., 10:784-789, 1992.","Ye X. et al., 287:303-305, 2000.","Yoshimura K. et al., 150:794-801, 1988.","Milk proteins such as lactoferrin (LF), lysozyme (LZ), lactoperoxidase (LP), immunoglobulin-A (IgA), alpha-lactalbumin, beta-lactoglobulin, alpha-, beta- and kappa-caseins, serum albumin, lipase and others are known to have a number of nutritional and other beneficial effects, particularly for infants. Breast feeding of fresh human milk has traditionally been considered the best means to provide nutrition to an infant. Although all physiological roles of human milk proteins have not yet been elucidated, evidence has been obtained that lysozyme, lactoferrin and other milk proteins control the microflora in the gut of infants (L\u00f6nnerdal, 1985). Breast milk has been suggested to contain many immune factors that compensate for the undeveloped defense mechanisms of the gut of infants (Saarinen K M et al., 2000). Several human milk proteins have been demonstrated to have beneficial physiologic effects in infants, particularly in the defense against infection and in the optimization of nutrient uptake.","However, many situations arise where the infant cannot be fed mother's milk and synthetic infant milk formulas are used in the place of breast feeding (Motil K J, 2000). Considerable effort has been made to improve synthetic infant milk formulas in order to closely simulate mother's milk.","The protein and non-protein composition of the human milk and cow milk is described by Kunz et al., 1999. The relative concentrations of milk proteins vary between human and cows' milk. For example, lactoferrin and lysozyme are present in a relatively high amount in human milk but in only low or trace amounts in cow's milk.","In general, synthetic infant formula is prepared using cow's milk that does not closely resemble the protein composition found in human milk. Accordingly, cow's milk based infant formula is typically supplemented with various human milk protein components. Typically, commercial infant formulas based on cow's milk contain approximately of 0.1 mg\/mL lactoferrin whereas natural human breast milk contains an average concentration of 1.4 mg\/mL. Soy-based infant formulas contain no added lactoferrin.","Although addition of recombinant human milk proteins to infant milk or milk formula has been proposed, e.g., using transgenic cows or by addition of microbially produced human milk proteins to milk or milk formula, these approaches do not overcome the various problems of (i) allergies to cow's milk, (ii) the high cost of recombinant protein production and\/or (iii) safety issues related to food products.","It would therefore be desirable to provide a plant-derived infant formula having beneficial levels of one or more proteins normally present in human milk, while largely avoiding costly recombinant protein production techniques and associated safety issues. More generally, it would be desirable to provide a nutritional food extract that may be readily and inexpensively obtained in large quantities, can be delivered by itself, as a nutraceutical or added to processed foods, for supplying one or more human milk proteins beneficial to human health.","In one aspect, the invention includes a nutritionally enhanced food having one or more plant-derived food ingredients, and as an additive, a seed composition containing a flour, extract, or malt obtained from mature monocot seeds and one or more seed-produced human milk proteins in substantially unpurified form. The seed-produced protein(s) include lactoferrin, lysozyme, lactoferricin, epidermal growth factor, insulin-like growth factor-1, lactadherin, kappa-casein, haptocorrin, lactoperoxidase, and\/or alpha-1-antitrypsin, preferably at least lysozyme and\/or lactoferrin.","The seed composition preferably comprises between 0.1 to 20% of the total solid weight of the food. The seed-produced human milk protein(s) are preferably present in an amount that is at least 50% of the amount of the protein(s) in human milk, on a weight\/weight basis.","In one embodiment, the food is an infant formula, either in dry or liquid form. The milk proteins include at least lysozyme and lactoferrin, and the seed composition contains a seed extract or malt obtained from mature seeds of rice or barley. The lysozyme is preferably present in an amount between 0.03 to 0.3 g protein\/liter formula, and lactoferrin, in an amount between 0.3 and 3 g protein\/liter.","In another aspect, the invention includes an ingestible monocot-seed composition containing a flour, extract, or malt obtained from mature monocot seeds and one or more seed-produced human milk proteins in substantially unpurified form. As above, the seed-produced protein(s) preferably include lactoferrin and\/or lysozyme, but may alternatively or in addition, include lactoferricin, epidermal growth factor, insulin-like growth factor-1, lactadherin, kappa-casein, haptocorrin, lactoperoxidase, IgA, and alpha-1-antitrypsin. The one or more milk proteins are in the composition extract in an amount greater than 1 mg\/gram dry weight of extract.","The flour may be prepared by milling mature monocot seeds; the extract, by suspending milled flour in a buffered aqueous medium; and the malt, by (i) steeping barley seeds to a desired water content, (ii) germinating the steeped barley, (iii) drying the germinated seeds under conditions effective to stop germination, (iv) crushing the dried seeds, (v) optionally, adding crushed seeds from a non-barley monocot plant, (vi) forming a mixture of the crushed seeds in water, and (vii) malting the crushed seed mixture until a desired malt is achieved, where at least one of the barley or non-barley monocot seeds contain such milk protein(s).","Also disclosed is a monocot seed containing, in extractable form, one or more proteins normally present in human milk, where the human-milk protein(s) include lactoferrin, lysozyme, lactoferricin, EGF, IGF-I, lactadherin, kappa-casein, haptocorrin, lactoperoxidase, alpha-1-antitrypsin, and immunoglobulins, preferably at least lactoferrin and\/or lysozyme. The milk protein preferably includes at least 0.25 weight percent of the total protein in the harvested mature seeds.","In a related aspect, the invention includes a method of producing an ingestible seed composition. In practicing the method, there is first obtained a monocot plant that has been stably transformed with a first chimeric gene having (i) a transcriptional regulatory region from a monocot gene having a seed maturation-specific promoter, (ii) operably linked to said transcriptional regulatory region, a leader DNA sequence encoding a monocot seed-specific transit sequence capable of targeting a linked polypeptide to an endosperm-cell organelle, and (iii) a protein-coding sequence encoding a protein normally present in human milk. The transformed plant is cultivated under seed-maturation conditions, and the mature seeds harvested. From the harvested seeds is obtained a flour, extract, or malt composition containing the human milk protein in substantially unpurified form. The human milk protein(s) preferably constitute at least 0.1 percent of the total protein in the harvested mature seeds.","The flour may be prepared by milling mature monocot seeds; the extract, by suspending milled flour in a buffered aqueous medium; and the malt, by (i) steeping barley seeds to a desired water content, (ii) germinating the steeped barley, (iii) drying the germinated seeds under conditions effective to stop germination, (iv) crushing the dried seeds, (v) optionally, adding crushed seeds from a non-barley monocot plant, (vi) forming a mixture of the crushed seeds in water, and (vii) malting the crushed seed mixture until a desired malt is achieved, where at least one of the barley or non-barley monocot seeds contain such milk protein(s).","The monocot plant obtained may be further transformed with a second chimeric gene having (i) a transcriptional regulatory region from a monocot gene having a seed maturation-specific promoter, (ii) operably linked to said transcriptional regulatory region, a transit DNA sequence encoding a monocot seed-specific transit sequence capable of targeting a linked polypeptide to an endosperm-cell organelle, and (iii) a protein-coding sequence encoding a protein normally present in human breast milk other than that encoded by the first chimeric gene.","In a related aspect, the invention includes a transgenic monocot plant which is stably transformed with a first chimeric gene having (i) a transcriptional regulatory region from a monocot gene having a seed maturation-specific promoter, (ii) operably linked to said transcriptional regulatory region, a transit DNA sequence encoding a monocot seed-specific transit sequence capable of targeting a linked polypeptide to an endosperm-cell organelle, and (iii) a protein-coding sequence encoding a protein normally present in human milk.","Exemplary transcriptional regulatory regions in the chimeric gene are from the promoter of the group of genes: rice glutelins, rice globulins, oryzins, and prolamines, barley hordeins, wheat gliadins and glutenins, maize zeins and glutelins, oat glutelins, and sorghum kafirins, millet pennisetins, and rye secalins genes. The leader sequence is likewise from the group of genes: gene selected from the group of rice glutelins, rice globulins oryzins, and prolamines, barley hordeins, wheat gliadins and glutenins, maize zeins and glutelins, oat glutelins, and sorghum kafirins, millet pennisetins, and rye secalins genes.","In one preferred embodiment, the transcriptional regulatory region in the chimeric gene is a rice glutelin Gt1 promoter, and the leader DNA sequence is a rice glutelin Gt1 signal sequence capable of targeting a linked polypeptide to a protein storage body. An exemplary glutelin Gt1 promoter and glutelin Gt1 signal sequence are included within the sequence identified by SEQ ID NO:15. In another preferred embodiment, the transcriptional regulatory region in the chimeric gene is a rice globulin Glb promoter, and the leader DNA sequence is a rice glutelin Gt1 signal sequence capable of targeting a linked polypeptide to a protein storage body. An exemplary globulin Glb promoter and glutelin Gt1 signal sequence are included within the sequence identified by SEQ ID NO:16.","The transformed monocot seed may further encode at least one transcription factors O2, PBF, and Reb, as exemplified by SEQ ID NOS: 31, 32, and 33, respectively, and preferably O2 and\/or PBF.","The protein-coding sequence is the a coding sequence for a human milk protein selected from the group consisting of lactoferrin, lysozyme, lactoferricin, EGF, IGF-I, lactadherin, kappa-casein, haptocorrin, lactoperoxidase, alpha-1-antitrypsin, and immunoglobulins, preferably a sequence which has been codon-optimized for expression in monocots. Exemplary codon-optimized sequences for these proteins are represented by SEQ ID NOS: 1, 3, and 7-14.","The plant may be further stably transformed with a second chimeric gene having","(i) a transcriptional regulatory region from a monocot gene having a seed maturation-specific promoter, (ii) operably linked to said transcriptional regulatory region, a transit DNA sequence encoding a monocot seed-specific transit sequence capable of targeting a linked polypeptide to an endosperm-cell organelle, and (iii) a protein-coding sequence encoding a protein normally present in human breast milk other than that encoded by the first chimeric gene.","In still another aspect, the invention includes a method of forming a malt syrup containing one or more human milk proteins. The method includes the steps of (i) steeping barley seeds to a desired water content, (ii) germinating the stepped barley, (iii) drying the germinated seeds, under conditions effective to stop germination, (iv) crushing the dried seeds, (v) optionally, adding crushed seeds from a non-barley monocot plant, (vi) forming a mixture of crushed seeds in water, and (vii) malting the crushed seed mixture until a desired malt is achieved. At least one of the barley or non-barley monocot seeds are obtained from plants that have been stably transformed with a first chimeric gene having (i) a transcriptional regulatory region from a monocot gene having a seed maturation-specific promoter, (ii) operably linked to said transcriptional regulatory region, a transit DNA sequence encoding a monocot seed-specific transit sequence capable of targeting a linked polypeptide to an endosperm-cell organelle, and (iii) a protein-coding sequence encoding a protein normally present in human breast milk.","These and other objects and features of the invention will become more fully apparent when the following detailed description of the invention is read in conjunction with the accompanying drawings.","I. Definitions","Unless otherwise indicated, all terms used herein have the meanings given below, and are generally consistent with same meaning that the terms have to those skilled in the art of the present invention. Practitioners are particularly directed to Sambrook et al. (1989) Molecular Cloning: A Laboratory Manual (Second Edition), Cold Spring Harbor Press, Plainview, N.Y. and Ausubel F M et al. (1993) Current Protocols in Molecular Biology, John Wiley & Sons, New York, N.Y., for definitions and terms of the art. It is to be understood that this invention is not limited to the particular methodology, protocols, and reagents described, as these may vary.","All publications cited herein are expressly incorporated herein by reference for the purpose of describing and disclosing compositions and methodologies that might be used in connection with the invention.","The term \u201cpolypeptide\u201d refers to a biopolymer compound made up of a single chain of amino acid residues linked by peptide bonds. The term \u201cprotein\u201d as used herein may be synonymous with the term \u201cpolypeptide\u201d or may refer, in addition, to a complex of two or more polypeptides.","The term \u201canti-microbial protein\u201d refers to a protein that is anti-bacterial and can include acute phase proteins, cationic anti-microbial peptides and probiotic proteins. Such anti-microbial proteins are capable of inhibiting the growth of one or more of Gram-negative bacteria, Gram-positive bacteria, fungi (including yeast), parasites (including planaria and nematodes) and viruses. Typically, such anti-microbial peptides exhibit selective biological activity against such microbes over eukaryotic cells.","The term \u201canti-bacterial protein\u201d refers to a protein that is bacteriostatic or bactericidal in nature.","The term \u201cbacteriostatic protein\u201d refers to refers to a protein capable of inhibiting the growth of, but not capable of killing bacteria.","The term \u201cbactericidal protein\u201d refers to a protein capable of killing bacteria.","The term \u201cvector\u201d refers to a nucleic acid construct designed for transfer between different host cells. An \u201cexpression vector\u201d refers to a vector that has the ability to incorporate and express heterologous DNA fragments in a foreign cell. Many prokaryotic and eukaryotic expression vectors are commercially available. Selection of appropriate expression vectors is within the knowledge of those having skill in the art. Accordingly, an \u201cexpression cassette\u201d or \u201cexpression vector\u201d is a nucleic acid construct generated recombinantly or synthetically, with a series of specified nucleic acid elements that permit transcription of a particular nucleic acid in a target cell. The recombinant expression cassette can be incorporated into a plasmid, chromosome, mitochondrial DNA, plastid DNA, virus, or nucleic acid fragment. Typically, the recombinant expression cassette portion of an expression vector includes, among other sequences, a nucleic acid sequence to be transcribed and a promoter.","The term \u201cplasmid\u201d refers to a circular double-stranded (ds) DNA construct used as a cloning vector, and which forms an extrachromosomal self-replicating genetic element in many bacteria and some eukaryotes.","The term \u201cselectable marker-encoding nucleotide sequence\u201d refers to a nucleotide sequence capable of expression in plant cells and where expression of the selectable marker confers to plant cells containing the expressed gene the ability to grow in the presence of a selective agent. As used herein, the term \u201cBar gene\u201d refers to a nucleotide sequence encoding a phosphinothricin acetyltransferase enzyme that upon expression confers resistance to the herbicide glufosinate-ammonium (\u201cBasta\u201d).","A \u201ctranscription regulatory region\u201d or \u201cpromoter\u201d refers to nucleic acid sequences that influence and\/or promote initiation of transcription. Promoters are typically considered to include regulatory regions, such as enhancer or inducer elements. The promoter will generally be appropriate to the host cell in which the target gene is being expressed. The promoter, together with other transcriptional and translational regulatory nucleic acid sequences (also termed \u201ccontrol sequences\u201d), is necessary to express any given gene. In general, the transcriptional and translational regulatory sequences include, but are not limited to, promoter sequences, ribosomal binding sites, transcriptional start and stop sequences, translational start and stop sequences, and enhancer or activator sequences.","\u201cChimeric gene\u201d or \u201cheterologous nucleic acid construct\u201d, as defined herein refers to a construct which has been introduced into a host and may include parts of different genes of exogenous or autologous origin, including regulatory elements. A chimeric gene construct for plant\/seed transformation is typically composed of a transcriptional regulatory region (promoter) operably linked to a heterologous protein coding sequence, or, in a selectable marker heterologous nucleic acid construct, to a selectable marker gene encoding a protein conferring antibiotic resistance to transformed plant cells. A typical chimeric gene of the present invention, includes a transcriptional regulatory region inducible during seed development, a protein coding sequence, and a terminator sequence. A chimeric gene construct may also include a second DNA sequence encoding a signal peptide if secretion of the target protein is desired.","A nucleic acid is \u201coperably linked\u201d when it is placed into a functional relationship with another nucleic acid sequence. For example, DNA for a presequence or secretory leader is operably linked to DNA for a polypeptide if it is expressed as a preprotein that participates in the secretion of the polypeptide; a promoter or enhancer is operably linked to a coding sequence if it affects the transcription of the sequence; or a ribosome binding site is operably linked to a coding sequence if it is positioned so as to facilitate translation. Generally, \u201coperably linked\u201d means that the DNA sequences being linked are contiguous, and, in the case of a secretory leader, contiguous and in reading frame. However, \u201coperably linked\u201d elements, e.g., enhancers, do not have to be contiguous. Linking is accomplished by ligation at convenient restriction sites. If such sites do not exist, the synthetic oligonucleotide adaptors or linkers are used in accordance with conventional practice.","The term \u201cgene\u201d means the segment of DNA involved in producing a polypeptide chain, which may or may not include regions preceding and following the coding region, e.g. 5\u2032 untranslated (5\u2032 UTR) or \u201cleader\u201d sequences and 3\u2032 UTR or \u201ctrailer\u201d sequences, as well as intervening sequences (introns) between individual coding segments (exons).","The term \u201csequence identity\u201d means nucleic acid or amino acid sequence identity in two or more aligned sequences, aligned using a sequence alignment program. The term \u201c% homology\u201d is used interchangeably herein with the term \u201c% identity\u201d and refers to the level of nucleic acid or amino acid sequence identity between two or more aligned sequences, when aligned using a sequence alignment program. For example, 70% homology means the same thing as 70% sequence identity determined by a defined algorithm, and accordingly a homologue of a given sequence has greater than 80% sequence identity over a length of the given sequence. Exemplary levels of sequence identity include, but are not limited to, 80, 85, 90 or 95% or more sequence identity to a given sequence, e.g., the coding sequence for lactoferrin, as described herein.","Exemplary computer programs which can be used to determine identity between two sequences include, but are not limited to, the suite of BLAST programs, e.g., BLASTN, BLASTX, and TBLASTX, BLASTP and TBLASTN, publicly available on the Internet at \u201cwww.ncbi.nlm.gov\/BLAST\/\u201d. See, also, Altschul, S. F. et al., 1990 and Altschul, S. F. et al., 1997.","Sequence searches are typically carried out using the BLASTN program when evaluating a given nucleic acid sequence relative to nucleic acid sequences in the GenBank DNA Sequences and other public databases. The BLASTX program is preferred for searching nucleic acid sequences which have been translated in all reading frames against amino acid sequences in the GenBank Protein Sequences and other public databases. Both BLASTN and BLASTX are run using default parameters of an open gap penalty of 11.0, and an extended gap penalty of 1.0, and utilize the BLOSUM-62 matrix. [See, Altschul, et al., 1997.]","A preferred alignment of selected sequences in order to determine \u201c% identity\u201d between two or more sequences, is performed using for example, the CLUSTAL-W program in MacVector version 6.5, operated with default parameters, including an open gap penalty of 10.0, an extended gap penalty of 0.1, and a BLOSUM 30 similarity matrix.","A nucleic acid sequence is considered to be \u201cselectively hybridizable\u201d to a reference nucleic acid sequence if the two sequences specifically hybridize to one another under moderate to high stringency hybridization and wash conditions. Hybridization conditions are based on the melting temperature (Tm) of the nucleic acid binding complex or probe. For example, \u201cmaximum stringency\u201d typically occurs at about Tm-5\u00b0 C. (5\u00b0 below the Tm of the probe); \u201chigh stringency\u201d at about 5-10\u00b0 below the Tm; \u201cintermediate stringency\u201d at about 10-20\u00b0 below the Tm of the probe; and \u201clow stringency\u201d at about 20-25\u00b0 below the Tm. Functionally, maximum stringency conditions may be used to identify sequences having strict identity or near-strict identity with the hybridization probe; while high stringency conditions are used to identify sequences having about 80% or more sequence identity with the probe.","Moderate and high stringency hybridization conditions are well known in the art (see, for example, Sambrook et al, 1989, Chapters 9 and 11, and in Ausubel et al., 1993, expressly incorporated by reference herein). An example of high stringency conditions includes hybridization at about 42\u00b0 C. in 50% formamide, 5\u00d7SSC, 5\u00d7Denhardt's solution, 0.5% SDS and 100 \u03bcg\/ml denatured carrier DNA followed by washing two times in 2\u00d7SSC and 0.5% SDS at room temperature and two additional times in 0.1\u00d7SSC and 0.5% SDS at 42\u00b0 C.","As used herein, \u201crecombinant\u201d includes reference to a cell or vector, that has been modified by the introduction of a heterologous nucleic acid sequence or that the cell is derived from a cell so modified. Thus, for example, recombinant cells express genes that are not found in identical form within the native (non-recombinant) form of the cell or express native genes that are otherwise abnormally expressed, under expressed or not expressed at all as a result of deliberate human intervention.","A plant cell, tissue, organ, or plant into which a heterologous nucleic acid construct comprising the coding sequence for an anti-microbial protein or peptide has been introduced is considered transformed, transfected, or transgenic. A transgenic or transformed cell or plant also includes progeny of the cell or plant and progeny produced from a breeding program employing such a transgenic plant as a parent in a cross and exhibiting an altered phenotype resulting from the presence of the coding sequence for an anti-microbial protein. Hence, a plant of the invention will include any plant which has a cell containing introduced nucleic acid sequences, regardless of whether the sequence was introduced into the plant directly through transformation means or introduced by generational transfer from a progenitor cell which originally received the construct by direct transformation.","The term \u201ctransgenic plant\u201d refers to a plant that has incorporated exogenous nucleic acid sequences, i.e., nucleic acid sequences which are not present in the native (\u201cuntransformed\u201d) plant or plant cell. Thus a plant having within its cells a heterologous polynucleotide is referred to herein as a \u201ctransgenic plant\u201d. The heterologous polynucleotide can be either stably integrated into the genome, or can be extra-chromosomal. Preferably, the polynucleotide of the present invention is stably integrated into the genome such that the polynucleotide is passed on to successive generations. The term \u201ctransgenic\u201d as used herein does not encompass the alteration of the genome (chromosomal or extra-chromosomal) by conventional plant breeding methods or by naturally occurring events such as random cross-fertilization, non-recombinant viral infection, non-recombinant bacterial transformation, non-recombinant transposition, or spontaneous mutation. \u201cTransgenic\u201d is used herein to include any cell, cell line, callus, tissue, plant part or plant, the genotype of which has been altered by the presence of heterologous nucleic acids including those transgenics initially so altered as well as those created by sexual crosses or asexual reproduction of the initial transgenics.","Terms \u201ctransformed\u201d, \u201cstably transformed\u201d or \u201ctransgenic\u201d with reference to a plant cell means the plant cell has a non-native (heterologous) nucleic acid sequence integrated into its genome which is maintained through two or more generations.","The term \u201cexpression\u201d with respect to a protein or peptide refers to the process by which the protein or peptide is produced based on the nucleic acid sequence of a gene. The process includes both transcription and translation. The term \u201cexpression\u201d may also be used with respect to the generation of RNA from a DNA sequence.","The term \u201cintroduced\u201d in the context of inserting a nucleic acid sequence into a cell, means \u201ctransfection\u201d, or \u201ctransformation\u201d or \u201ctransduction\u201d and includes the incorporation of a nucleic acid sequence into a eukaryotic or prokaryotic cell where the nucleic acid sequence may be incorporated into the genome of the cell (for example, chromosome, plasmid, plastid, or mitochondrial DNA), converted into an autonomous replicon, or transiently expressed (for example, transfected mRNA).","By \u201chost cell\u201d is meant a cell which contains a vector and supports the replication, and\/or transcription or transcription and translation (expression) of the expression construct. Host cells for use in the present invention can be prokaryotic cells, such as , or eukaryotic cells such as yeast, plant, insect, amphibian, or mammalian cells. In general, host cells are monocotyledenous or dicotyledenous plant cells.","A \u201cplant cell\u201d refers to any cell derived from a plant, including undifferentiated tissue (e.g., callus) as well as plant seeds, pollen, progagules and embryos.","The term \u201cmature plant\u201d refers to a fully differentiated plant.","The terms \u201cnative\u201d and \u201cwild-type\u201d relative to a given plant trait or phenotype refers to the form in which that trait or phenotype is found in the same variety of plant in nature.","The term \u201cplant\u201d includes reference to whole plants, plant organs (for example, leaves, stems, roots, etc.), seeds, and plant cells and progeny of same. Plant cell, as used herein includes, without limitation, seeds, suspension cultures, embryos, meristematic regions, callus tissue, leaves roots shoots, gametophytes, sporophytes, pollen, and microspores. The class of plants that can be used in the methods of the present invention is generally as broad as the class of higher plants amenable to transformation techniques, including both monocotyledenous and dicotyledenous plants.","The term \u201cseed\u201d is meant to encompass all seed components, including, for example, the coleoptile and leaves, radicle and coleorhiza, scutulum, starchy endosperm, aleurone layer, pericarp and\/or testa, either during seed maturation and seed germination.","The term \u201cseed in a form for use as a food or food supplement\u201d includes, but is not limited to, seed fractions such as de-hulled whole seed, flour (seed that has been de-hulled by milling and ground into a powder) a seed protein extract (where the protein fraction of the flour has been separated from the carbohydrate fraction) and\/or a purified protein fraction derived from the transgenic grain.","The term \u201cpurifying\u201d is used interchangeably with the term \u201cisolating\u201d and generally refers to the separation of a particular component from other components of the environment in which it was found or produced. For example, purifying a recombinant protein from plant cells in which it was produced typically means subjecting transgenic protein containing plant material to biochemical purification and\/or column chromatography.","The term \u201cactive\u201d refers to any biological activity associated with a particular milk protein, such as the enzymatic activity associated with human lysozyme. It follows that the biological activity of a given milk protein refers to any biological activity typically attributed to that factor by those of skill in the art.","The term \u201chuman milk protein\u201d or \u201cproteins normally present in human milk\u201d refers to one or more proteins, or biologically active fragments thereof, found in normal human milk, including, without limitation, of lactoferrin, lysozyme, lactoferricin, EGF, IGF-I, lactadherin, kappa-casein, haptocorrin, lactoperoxidase, alpha-1-antitrypsin, and immunoglobulins, and biologicaliy active fragments thereof.","The term \u201cnutritionally enhanced food\u201d refers to a food, typically a processed food, to which a seed-produced human milk protein has been added, in an amount effective to confer some health benefit, such as improved gut health, resistance to pathogenic bacteria, or iron transport, to a human consuming the food.","\u201cPlant-derived food ingredients\u201d refers to plant-derived food stuff, typically monocot grain, but also including, separately, lectins, gums, sugars, plant-produced proteins and lipids, that may be blended or combined, alone or in combination with one or more plant-derived ingredients, to form an edible food.","\u201cMonocot seed components\u201d refers to carbohydrate, protein, and lipid components extractable from monocot seeds, typically mature monocot seeds.","\u201cMalted-seed components\u201d refers to seed-derived components, predominantly carbohydrate components, after conversion of complex carbohydrates to malt sugars by malting, i.e., treating with malting enzymes such as a barley amylase and glucanases, under conditions effective to conversion seed-derived carbohydrates to malt sugars.","\u201cSubstantially unpurified form\u201d, as applied to human milk proteins in a seed extract means that the protein or proteins present in the extract are present in an amount less than 50% by weight, typically between 0.1 and 10 percent by weight.","\u201cSeed maturation\u201d or \u201cgrain development\u201d refers to the period starting with fertilization in which metabolizable reserves, e.g., sugars, oligosaccharides, starch, phenolics, amino acids, and proteins, are deposited, with and without vacuole targeting, to various tissues in the seed (grain), e.g., endosperm, testa, aleurone layer, and scutellar epithelium, leading to grain enlargement, grain filling, and ending with grain desiccation.","\u201cInducible during seed maturation\u201d refers to promoters which are turned on substantially (greater than 25%) during seed maturation. \u201cHeterologous DNA\u201d or \u201cforeign DNA\u201d refers to DNA which has been introduced into plant cells from another source, or which is from a plant source, including the same plant source, but which is under the control of a promoter or terminator that does not normally regulate expression of the heterologous DNA.","\u201cHeterologous protein\u201d is a protein, including a polypeptide, encoded by a heterologous DNA.","A \u201csignal\/targeting\/transport sequence\u201d is an N- or C-terminal polypeptide sequence which is effective to localize the polypeptide or protein to which it is attached to a selected intracellular or extracellular region, including an intracellular vacuole or other protein storage body, chloroplast, mitochondria, or endoplasmic reticulum, or extracellular space or seed region, such as the endosperm, following secretion from the cell.","A \u201cproduct\u201d encoded by a DNA molecule includes, for example, RNA molecules and polypeptides.","A DNA sequence is \u201cderived from\u201d a gene if it corresponds in sequence to a segment or region of that gene. Segments of genes which may be derived from a gene include the promoter region, the 5\u2032 untranslated region, and the 3\u2032 untranslated region of the gene.","\u201cAlpha-amylase\u201d as used herein refers to an enzyme which principally breaks starch into dextrins.","\u201cBeta-amylase\u201d as used herein refers to an enzyme which converts start and dextrins into maltose.","\u201cCereal adjuncts\u201d as used herein refers to cereal grains, principally barley, wheat, rye, oats, maize, sorghum and rice, or processed whole or portions thereof, especially the starch fraction, which are added to the barley mash, which allows the barley enzymes to hydrolyze both the barley starch and the starch derived from the cereal adjunct. \u201cTransgenic cereal adjuncts\u201d as used herein refers to transgenic cereal grains, principally barley, wheat, rye, oats, maize, sorghum and rice, and which is expressing a recombinant molecule in a grain part, principally the endosperm (starch) layer.","\u201cConversion\u201d as used herein refers to the process of starch hydrolysis, usually catalyzed by acid or enzyme action, which produces dextrose, maltose, and higher polysaccharides from starch.","\u201cDiastatic enzyme (amylolytic)\u201d as used herein refers to an enzyme capable of causing the hydrolysis of starch.","\u201cDiastatic malt flour\u201d as used herein refers to enzyme active flour milled from germinated (malted) barley.","\u201cDiastatic malt syrup\u201d as used herein refers to enzyme active liquid malt syrup (barley and cereal adjuncts).","\u201cDry diastatic malt\u201d as used herein refers to a blend of diastatic malted barley flour, wheat flour and dextrose with standardized enzyme levels at 20 degrees and 60 degrees Lintner.","\u201cDry nondiastatic malt\u201d as used herein refers to spray dried form of liquid nondiastatic malt extract or syrup.","\u201cLintner\u201d as used herein refers to a laboratory measurement of enzyme activity strength. The higher the value, the higher activity.","\u201cDried malt\u201d as used herein refers to the dried grain resulting form controlled germination of cereal grins, usually barley, but other cereals can be malted as well.","\u201cMalt extract\u201d as used herein refers to a viscous concentrate of the water extract of dried malt.","\u201cMaltodextrin\u201d as used herein refers to a purified, concentrated aqueous solution of nutritive saccharides, obtained form edible starch, or the dried product derived from the solution. Maotodextrins have a dextrose equivalent of less than 20 and are considered \u2018non-sweet soluble solids\u2019. They are usually marketed dry, but may be obtained as a concentrated solution. Maltodextrins are usually offered as 10 to 14 D.E. products or as 15 to 19 D.E. versions. Another maltodextrin, with a D.E. of about 5, is sometimes manufactured, but currently not used widely. Composition of maltodextrins is roughly 65 to 80% higher saccharides, 4 to 9% pentasaccharides, 4 to 7% tetrasaccharides, and 5 to 9% trisaccharides. Traces of mono and disaccharides are present. They are usually used as bulking agents or viscosity builders, without sweetness.","\u201cMalt syrup\u201d as used herein refers to viscious concentrate of the water extract of dried \u2018malt\u2019 and other cereal grains.","\u201cMalt\u201d refers to a malt extract or malt syrup.","\u201cNondiastatic malt syrup\u201d as used herein refers to liquid malt syrup (barley and cereal adjuncts) without enzyme activity.","\u201cTransgenic malt extract\u201d as used herein refers to a vicious concentrate of the water extract of dried malt which includes a recombinant protein, polypeptide and\/or metabolite.","\u201cTransgenic malt syrup\u201d as used herein refers to vicious concentrate of the water extract of dried \u2018malt\u2019 and other cereal grains which includes a recombinant protein, polypeptide and\/or metabolite.","II. Milk Products-state of the Art\/Issues","Human milk provided by healthy and well-nourished mothers is believed by pediatricians and nutritionists to be the optimal way to feed infants during the first six months of life. Breast milk not only provides the infant with a well-balanced supply of nutrients, but also a multitude of unique components that facilitate nutrient digestion and absorption, protect against microorganisms and promote growth and development. Human milk is a source of peptides, amino acids, and nitrogen and also contains whey proteins involved in the development of the immune response (e.g., immunoglobulins), together with other non-immunologic defense proteins (e.g., lactoferrin).","However, infant formulas are often used as a nutritional source for infants less than one year of age for a variety of reasons, e.g., insufficient milk production by or pathogenic infection of, the mother. Infant formulas and not standard cow's milk are used because (1) cow's milk has more than twice the protein of breast milk or infant formula and this protein may be hard for babies to digest; (2) the level of iron, zinc and vitamin C (which babies need in their diet) is low in cow's milk; and (3) the level of sodium level is three to four times that of breast milk and generally too high for infants less than a year old. A number of types of infant formulas which vary in caloric content, nutrient composition, digestibility, taste, and cost are available as an alternative to breast milk. Examples include standard cow milk-based formulas, soy protein formulas and formulas for premature infants or infants with special dietary needs due to allergies, etc.","During the last several decades, improved infant formulas have become available that are safe and contain nutrient concentrations similar to, or higher than, breast milk. However, breast-fed infants still have a lower prevalence of infection than formula-fed infants and when they become ill, the duration of both diarrhea and upper respiratory infections is shorter than in formula-fed infants. (See, e.g., Kovar et al., 1984, and Dewey et al., 1995). In addition, it has been reported that breast-fed infants have a different growth pattern than formula-fed infants (Dewey et al., 1992; Dewey et al., 1993), and epidemiological studies show that they have a lower incidence of chronic diseases, such as diabetes and coronary heart disease.","It has been postulated that many of advantages to infants provided by mother's milk are effectuated through unique proteins present in breast milk, but not in baby formula (L\u00f6nnerdal, 1985). Human milk proteins are unique and even if the alternative protein sources used in infant formulas (e.g., skim milk, whey protein and soy isolates) mimic the amino acid concentration and ratio found in breast milk, the biological properties of human proteins cannot be readily copied.","Exemplary unique proteins present in human milk include lactoferrin and lysozyme. Lactoferrin is an iron-binding protein found in the granules of neutrophils which exerts an antimicrobial activity and lysozyme is a crystalline, basic protein present in saliva, tears, egg white, and many animal fluids, which functions as an antibacterial enzyme.","Improved food compositions containing human milk proteins are provided by the present invention. The human milk proteins are produced in the grain of transgenic plants and added to novel food compositions, infant formula being one example. Infant formula containing such recombinant human milk proteins are useful in supplementing or enhancing the diet of infants; particularly very-low-birth-weight infants.","Other foods that may be supplemented with human milk proteins include, but are not limited to foods where recombinant lactoferrin can be added and utilized as an iron supplement replacing the need for exogenously added iron in the final food formulation.","III. Compositions Containing Human Milk Proteins","The present invention provides food supplement compositions (also termed \u201cimproved food compositions\u201d comprising human milk proteins and methods of making such compositions. In practicing the invention, a human milk protein is produced in the seeds or grain of transgenic plants which express the nucleic acid coding sequence for the human milk protein and the transgenic grains added to a food such as an infant formula to result in an \u201cimproved food compositions\u201d. More specifically, the invention is based on the expression of human milk proteins, exemplified by human lactoferrin (hLF) and human lysozyme, under the control of a seed specific promoter in rice. The human protein produced by transgenic plants is compared to the native form of the same protein, information on the stability of the recombinant protein and the advantages of using rice grain containing such recombinant human milk protein in infant formula and\/or other foods, is further described below.","The invention relies on the use of heterologous nucleic acid constructs including the coding sequence for a commercially important milk protein or polypeptide of nutritional and\/or therapeutic value, exemplified herein by lactoferrin and lysozyme.","The exemplary milk proteins, lysozyme and lactoferrin are an integral part of the immune system of multicellular animals. They are found in epithelial secretions (tears, mucous, gastric juice) and blood plasma of mammals, birds, reptiles, amphibia, and a variety of invertebrates. They are also enriched in mammalian milk and avian eggs, where they serve as primary antimicrobial proteins. Furthermore, lysozyme is a major component of the secretory granules of neutrophils and macrophages and is released at the site of infection in the earliest stages of the immune response. Lactoferrin is found at high concentrations within specific granules of polymorphonuclear leukocytes.","It has previously been demonstrated that lysozyme and lactoferrin are efficacious in promoting resistance to infectious diseases in experimental animals and humans and that they play a role of primary defense proteins on epithelial surfaces in addition to being important determinants in the establishment of a healthy microflora within the digestive tract. These properties suggest that food supplements comprising lysozyme and\/or lactoferrin will be beneficial to the overall health of infants.","The improved food compositions of the invention include milk proteins such as lactoferrin, and lysozyme, produced in the grain of transgenic plants, which are useful for improved nutrition. In one preferred approach, the improved food compositions are administered to an infant. Typically the improved food compositions, e.g., infant formula contain one or more recombinant human milk proteins in an amount that corresponds to the amount and proportions of the same human milk proteins found in endogenous human milk.","A. Lysozymes","Human milk lysozyme, called muramidase or peptidoglycan N-acetylmuramoyl-hydrolase (EC 3.2.1.17) contains 130 amino acid residues and is a protein of 14.7 kDa in size. Human lysozyme is non-glycosylated and possesses unusual stability in vitro and in vivo due to its amino acid and secondary structure.","Lysozyme is one of the most abundant proteins present in human milk with a concentration of about 400 \u03bcg\/ml. The concentration of lysozyme is approximately 0.13 \u03bcg\/ml in cow's milk (almost 3000 times less than found in human milk), 0.25 \u03bcg\/ml in goat's milk, 0.1 \u03bcg\/ml in sheep's milk and almost absent in rodent's milk (Chandan R C, 1968). Lysozyme is also found in other mammalian secretions, such as tears and saliva.","The protective role of lysozyme has been observed to include lysis of microbial cell walls, adjuvant activity of the end products peptidoglycan lysis, direct immunomodulating effects on leukocytes, and neutralization of bacterial endotoxins. The bacteriostatic and bactericidal actions of lysozyme were originally discovered by Flemming in 1922 and have been studied in detail. Lysozyme is effective against both gram positive and gram negative bacteria, as well as some types of yeasts. The antimicrobial effects of lysozyme often act synergistically with other defense molecules, including immunoglobulin and lactoferrin. Furthermore, structural changes in the cell wall due to lysozyme render bacteria more susceptible to phagocytosis by macrophages and neutrophils.","The hydrolysis of microbial peptidoglycans results in the release of the cleavage product, muramyl dipeptide, which is a potent adjuvant and is the active component of Freund's complete adjuvant. Muramyl dipeptide enhances IgA production, macrophage activation, and rapid clearance of a variety of bacterial pathogens in vivo. Lysozyme itself is also immunomodulatory. It directly interacts with the cell membrane of phagocytes to increase their uptake of bacteria. Lysozyme also augments the proliferative response of mitogen stimulated lymphocytes to interleukin-2 and increases the rate of synthesis of IgG and IgM by more than 5- and 2-fold respectively. Furthermore, the immunomodulatory action of lysozyme is not dependent upon enzymatic activity and is retained following denaturation. When lysozyme is fed to mice, it increases the number of intraepithelial and mesenteric lymph node lymphocytes that display antigens.","Lysozymes act as enzymes that cleave peptidoglycans, and ubiquitous cell wall component of microorganisms, in particular bacteria. Specifically, lysozymes are 1,4-acetylmuramidases that hydrolyze the glycoside bond between N-acetylmuramic acid and N-acetylglucosamine. Gram-positive bacteria are highly susceptible to lysozyme due to the polypeptidoglycan on the outside of the cell wall. Gram-negative strains have a single polypeptidoglycan layer covered by lipopolysaccharides and are therefore less susceptible to lysis by lysozyme, however, the sensitivity can be increased by the addition of EDTA (Sch\u00fctte and Kula, 1990). Lysozyme also exhibits antiviral activity, as exemplified by the significant reduction in recurrent occurrences of genital and labial herpes after oral treatment of patients with lysozyme (Joll\u00e8s, 1996). More recently, lysozyme from chicken egg whites, human milk and human neutrophils has been shown to inhibit the growth of HIV-1 in an in vitro assay (Lee-Huang et al., 1999). In addition, an anti-fungal activity has been demonstrated for lysozymes using oral isolates of (the most common fungal causative agent of oropharyngeal infection in humans; (Samaranayake et al., 1997). Lysozyme thus functions as a broad spectrum antimicrobial agent.","The ability of lysozyme to bind bacterial endotoxins, especially LPS, confers an important anti-microbial property to the molecule. Lysozyme binds electrostatically to the lipid A component of bacterial endotoxins at a 1:3 molar ratio. The resulting conformational change in endotoxin keeps it from interacting with macrophage receptors and dampens the release of pro-inflammatory cytokines such as interleukin-1 (IL-1), interleukin-6 (IL-6), and tumor necrosis factor (TNF). Thus, lysozyme exhibits anti-inflammatory activity during pathogen challenges.","The current major commercial source for lysozyme is chicken egg whites. Sequence analysis shows that lysozyme from chicken egg whites exhibits only partial homology (60%) with that synthesized by humans. Chicken and human lysozyme do not cross-react with their respective antibodies (Faure et al., 1970), indicating significant structural differences between these two lysozymes. Human lysozyme has been purified from breast milk (Boesman-Finkelstein et al., 1982; Wang et al., 1984), neutrophills (Lollike et al., 1995), and urine of hemodialysis patients (Takai et al., 1996). Breast milk remains the main source for isolation of human lysozyme, but the supply is limited. Precautions are required for isolation of the enzyme from human sources to avoid contamination with viral and microbial pathogens.","Recombinant human lysozyme has been produced in the mammary gland of transgenic mice. The enzyme retained its antimicrobial activity, but the final concentration in the milk was low (Maga et al., 1998; Maga et al., 1994; Maga et al., 1995). Human lysozyme has been expressed in () (Tsuchiya et al., 1992) yeast (Casta\u00f1\u00f3n et al., 1988; Jigami et al., 1986; and Yoshimura et al., 1988) and in small amounts in tobacco leaves (Nakajima et al., 1997). However, the expression level of recombinant human lysozyme in these organisms could be very low, and the cost of these forms may be prohibitive for food applications. In addition, human lysozyme produced in microorganisms may require extensive purification before it can be used in foods, particularly for infants and children.","In contrast to many other proteins, lysozyme is highly resistant to digestion in the gastrointestinal tract. In vitro studies have demonstrated that both molecules are resistant to hydrolysis by pepsin in the pH range found in the stomach. Furthermore, partial denaturation of lysozyme increases its bactericidal activity against some types of bacteria, and low pH, such as found in the stomach, increases the bactericidal effects of lysozyme. A proteolytic fragment (amino acids 98-112 of chicken egg white lysozyme) completely lacking enzymatic activity has been found to be the active bactericidal component of lysozyme. Additionally, a fragment of lactoferrin, known as lactoferricin, is formed by limited proteolytic digestion and has been shown to have extremely effective antibacterial activity.","The rice produced human lysozyme of the present invention exhibits acid pH resistance, as well as resistance to pepsin and pancreatin to make it resistant to digestion in the gastrointestinal tract. The excellent thermostability provides the feasibility to pasteurize products that include the recombinant human lysozyme.","B. Lactoferrin","Lactoferrin is an iron-binding protein found in the granules of neutrophils where it apparently exerts an antimicrobial activity by withholding iron from ingested bacteria and fungi; it also occurs in many secretions and exudates (milk, tears, mucus, saliva, bile, etc.). In addition to its role in iron transport, lactoferrin has bacteriostatic and bactericidal activities, in addition to playing a role as an anti-oxidant (Satue-Gracia et al., 2000).","The mature lactoferrin (LF) polypeptide consists of 692 amino acids, consists of a single-chain polypeptide that is relatively resistant to proteolysis, is glycosylated at two sites (N138 and N478) and has a molecular weight of about 80 kD. Human lactoferrin (hLF) is found in human milk at high concentrations (at an average of 1-3 mg\/ml), and at lower concentration (0.1-0.3 mg\/ml), in exocrine fluids of glandular epithelium cells such as bile, tears, saliva etc.","The primary functions of lactoferrin have been described as iron regulation, immune modulation and protection from infectious microbes. Lactoferrin can bind two ferric ions and has been shown to have biological activities including bacteriostatic (Bullen et al., 1972), bactericidal (Arnold, et al., 1980) and growth factor activity in vitro. Further, lactoferrin can promote the growth of bacteria that are beneficial to the host organism by releasing iron in their presence. Additional studies have recently shown lactoferrin to have antiviral activity towards cytomegalovirus, herpes simples virus, rotovirus and HIV both in vitro and in vivo. (See, e.g., Fujihara et al., 1995; Grover et al., 1997; and Harmsen et al., 1995.)","Lactoferrin, like transferrin, has a strong capacity to bind free iron under physiological conditions due to its tertiary structure, which consists of two globular lobes linked by an extended alpha-helix. The ability of lactoferrin to scavenge iron from the physiological environment can effectively inhibit the growth of \u201cmore than 90% of all microorganisms\u201d by depriving them of a necessary component of their metabolism, which will inhibit their growth in vivo and in vitro.","Unrelated to iron binding, the bactericidal activity of lactoferrin stems from its ability to destabilize the outer membrane of gram-negative bacteria through the liberation of lipopolysaccaharides that constitute the cell walls of the bacteria. Additionally, lactoferrin has recently been shown to bind to prions, a group of molecules common in , causing permeability changes in the cell wall. Studies in germfree piglets fed lactoferrin before being challenged with show significant decrease in mortality compared to the control group.","Recombinant LF (rLF) has been produced as a fusion protein in (Ward et al., 1992) and in the baculovirus expression system (Salmon et al., 1997). The -produced protein will require a high degree of purification as well as safety and toxicity testing prior to using it as a food additive (L\u00f6nnerdal, 1996). Lactoferrin has also been expressed in tobacco (L. cv Bright Yellow) cell culture (Mitra and Zhang, 1994), tobacco plants (Salmon et al., 1998) and potato () plants (Chong and Langridge, 2000). In tobacco cell culture the protein was truncated, whereas in tobacco and potato plants the rLF was processed correctly, but its expression level was very low (0.1% of total soluble protein) (Chong and Langridge, 2000). However, the expression level of recombinant human lactoferrin in these organisms could be very low, and the cost of these forms may be prohibitive for food applications. In addition, human lactoferrin produced in microorganisms may require extensive purification before it can be used in foods, particularly for infants and children.","In contrast to most other proteins, lactoferrin has also been shown to be resistant to proteolytic degradation in vitro, with trypsin and chymotrypsin remarkably ineffective in digesting lactoferrin, particularly in its iron-saturated form. Some large fragments of lactoferrin were formed, but proteolysis was clearly limited.","C. Lactoperoxidase","Lactoperoxidase is an enzyme which catalyzes the conversion of hydrogen peroxide to water. This enzyme is found in human milk, and plays host defensive roles through antimicrobial activity. When hydrogen peroxide and thiocyanate are added to raw milk, the SCN's oxidized by the enzyme-hydrogen peroxide complex producing bactericidal compounds which destroy Gram-negative bacteria (Shin).","D. Kappa-Casein","This group of proteins are readily digested and account for almost half of the protein content in human milk They are important as nutritional protein for breast-fed infants. It has also been advocated that part of the antimicrobial activity of human milk resides in the caseins, most likely the glycosylated kappa-casein (Aniansson).","E. Alpha-1-antitrypsin (\u201cAAT\u201d)","AAT belongs to the class of serpin inhibitors, has a molecular mass of 52 kD, and contains about 15% carbohydrate (Carrell et al, 1983). Concentrations of AAT in human milk range from 0.1 to 0.4 mg\/mL (Davidson and L\u00f6nnerdal, 1979; McGilligan et al., 1987). While the binding affinity of AAT is highest for human neutrophil elastase, it also has affinity for pancreatic proteases such as chymotrypsin and trypsin (Beatty et al., 1980).","While milk proteins have been expressed in systems such as transgenic cows and (L\u00f6nnerdal, 1996), transgenic rice provides a more attractive vehicle for the production of recombinant human AAT for food applications. High levels of expression are possible by using the combination of regulatory elements such as promoter, signal peptide, and terminator as disclosed herein. In addition, rice is often one of the first foods introduced to infants because of its nutritional value and low allergenicity. Safety concerns about microbial expression systems (e.g. ) limit the feasibility of using proteins from such sources as food components in formula (L\u00f6nnerdal, 1996). In addition, the cost of producing and purifying proteins from these other systems is often prohibitive for food applications. Thus, expression of recombinant human milk proteins in rice may be a safe and economically viable possibility for supplementing infant formula with such proteins. (See also Chowanadisai; Huang; Johnson; Lindberg; and Rudloff).","F. Lactadherin","Lactadherin is a protective glycoprotein present in human milk that helps protect breast-fed infants against infection by microorganisms. Protection against certain virus infections by human milk is also associated with lactadherin. (Newburg, 1999, 1998; Peterson; Hamosh).","G. Epidermal Growth Factor and Insulin-like Growth Factor","Epidermal Growth Factor and Insulin-like Growth Factor-1 are two growth factors present in human milk. These molecules may stimulate growth and development of the infant gastrointestinal tract. (Murphy; Prosser).","H. Immunoglobulins","Immunoglobulins present in human act to confer resistance to a variety of pathogens to which the mother may have been exposed. (See, for example, Humphreys; Kortt; Larrick; Maynard; and Peeters).","IV. Expression Vectors for Generation of Transgenic Plants Expressing Human Milk Proteins","Expression vectors for use in the present invention are chimeric nucleic acid constructs (or expression vectors or cassettes), designed for operation in plants, with associated upstream and downstream sequences.","In general, expression vectors for use in practicing the invention include the following operably linked components that constitute a chimeric gene: (i) a transcriptional regulatory region from a monocot gene having a seed maturation-specific promoter, (ii) operably linked to said transcriptional regulatory region, a leader DNA sequence encoding a monocot seed-specific transit sequence capable of targeting a linked polypeptide to an endosperm-cell organelle, such as the leader sequence for targeting to a protein-storage body, and (iii) a protein-coding sequence encoding a protein normally present in human milk.","The chimeric gene, in turn, is typically placed in a suitable plant-transformation vector having (i) companion sequences upstream and\/or downstream of the chimeric gene which are of plasmid or viral origin and provide necessary characteristics to the vector to permit the vector to move DNA from bacteria to the desired plant host; (ii) a selectable marker sequence; and (iii) a transcriptional termination region generally at the opposite end of the vector from the transcription initiation regulatory region.","Exemplary methods for constructing chimeric genes and transformation vectors carrying the chimeric genes are given in the examples below.","A. Promoters","In one aspect of this embodiment, the expression construct includes a transcription regulatory region (promoter) which exhibits specifically upregulated activity during seed maturation. Examples of such promoters include the maturation-specific promoter region associated with one of the following maturation-specific monocot storage proteins: rice glutelins, oryzins, and prolamines, barley hordeins, wheat gliadins and glutenins, maize zeins and glutelins, oat glutelins, and sorghum kafirins, millet pennisetins, and rye secalins. Exemplary regulatory regions from these genes are exemplified by SEQ ID NOS: 15-23, as identified in the Description of the Sequences.","Of particular interest is the expression of the nucleic acid encoding a human milk protein from a transcription initiation region that is preferentially expressed in plant seed tissue. Examples of such seed preferential transcription initiation sequences include those sequences derived from sequences encoding plant storage protein genes or from genes involved in fatty acid biosynthesis in oilseeds. Exemplary preferred promoters include a glutelin (Gt-1) promoter, as exemplified by SEQ ID NO: 18, which effects gene expression in the outer layer of the endosperm and a globulin (Glb) promoter, as exemplified by SEQ ID NO: 16, which effects gene expression in the center of the endosperm. Promoter sequences for regulating transcription of gene coding sequences operably linked thereto include naturally-occurring promoters, or regions thereof capable of directing seed-specific transcription, and hybrid promoters, which combine elements of more than one promoter. Methods for construction such hybrid promoters are well known in the art.","In some cases, the promoter is derived from the same plant species as the plant cells into which the chimeric nucleic acid construct is to be introduced. Promoters for use in the invention are typically derived from cereals such as rice, barley, wheat, oat, rye, corn, millet, triticale or sorghum.","Alternatively, a seed-specific promoter from one type of monocot may be used regulate transcription of a nucleic acid coding sequence from a different monocot or a non-cereal monocot.","Numerous types of appropriate expression vectors, and suitable regulatory sequences are known in the art for a variety of plant host cells. The transcription regulatory or promoter region is chosen to be regulated in a manner allowing for induction under seed-maturation conditions. Examples of such promoters include those associated with the following monocot storage proteins: rice glutelins, oryzins, and prolamines, barley hordeins, wheat gliadins and glutelins, maize zeins and glutelins, oat glutelins, and sorghum kafirins, millet pennisetins, and rye secalins. Exemplary promoter sequences are identified herein as SEQ ID NOS: 15-23. Other promoters suitable for expression in maturing seeds include the barley endosperm-specific B1-hordein promoter (Brandt, A., et al., (1985), Glub-2 promoter, Bx7 promoter, Gt3 promoter, Glub-1 promoter and Rp-6 promoter, particularly if these promoters are used in conjunction with transcription factors. The primary structure of a B1 hordein gene from barley is provided in Carlsberg Res. Commun. 50, 333-345.","B. Signal\/Targeting\/Transport Sequences","In addition to encoding the protein of interest, the expression cassette or heterologous nucleic acid construct may encode a signal\/targeting\/transport peptide that allows processing and translocation of the protein, as appropriate. Exemplary signal\/targeting\/transport sequences, particularly for targeting proteins to intracellular bodies, such as vacuoles, are signal\/targeting sequences associated with the monocot maturation-specific genes: glutelins, prolamines, hordeins, gliadins, glutenins, zeins, albumin, globulin, ADP glucose pyrophosphorylase, starch synthase, branching enzyme, Em, and lea. Exemplary sequences encoding a leader sequence for protein storage body are identified herein as SEQ ID NOS: 24-30.","In one preferred embodiment, the method is directed toward the localization of recombinant milk protein expression in a given subcellular compartment, in particular a protein-storage body, but also including the mitochondrion, endoplasmic reticulum, vacuoles, chloroplast or other plastidic compartment. For example, when recombinant milk protein expression is targeted to plastids, such as chloroplasts, in order for expression to take place the construct also employ the use of sequences to direct the gene to the plastid. Such sequences are referred to herein as chloroplast transit peptides (CTP) or plastid transit peptides (PTP). In this manner, when the gene of interest is not directly inserted into the plastid, the expression construct additionally contains a gene encoding a transit peptide to direct the gene of interest to the plastid. The chloroplast transit peptides may be derived from the gene of interest, or may be derived from a heterologous sequence having a CTP. Such transit peptides are known in the art. See, for example, Von Heijne et al., 1991; Clark et al., 1989; della-Cioppa et al., 1987; Romer et al., 1993; and Shah et al., 1986. Additional transit peptides for the translocation of the protein to the endoplasmic reticulum (ER) (Chrispeels, K., 1991), nuclear localization signals (Raikhel, 1992), or vacuole may also find use in the constructs of the present invention.","Another exemplary class of signal\/targeting\/transport sequences are sequences effective to promote secretion of heterologous protein from aleurone cells during seed germination, including the signal sequences associated with \u03b1-amylase, protease, carboxypeptidase, endoprotease, ribonuclease, DNase\/RNase, (1-3)-\u03b2-glucanase, (1-3)(1-4)-\u03b2-glucanase, esterase, acid phosphatase, pentosamine, endoxylanase, \u03b2-xylopyranosidase, arabinofuranosidase, \u03b2-glucosidase, (1-6)-\u03b2-glucanase, perioxidase, and lysophospholipase.","Since many protein storage proteins are under the control of a maturation-specific promoter, and this promoter is operably linked to a leader sequence for targeting to a protein body, the promoter and leader sequence can be isolated from a single protein-storage gene, then operably linked to a milk-protein storage protein in the chimeric gene construction. One preferred and exemplary promoter-leader sequence is from the rice Gt1 gene, having an exemplary sequence identified by SEQ ID NO: 15. Alternatively, the promoter and leader sequence may be derived from different genes. One preferred and exemplary promoter\/leader sequence combination is the rice Glb promoter linked to the rice Gt1 leader sequence, as exemplified by SEQ ID NO: 16.","C. Protein Coding Sequences","The construct also includes the nucleic acid coding sequence for a heterologous protein, under the control of a promoter, preferably a seed-specific promoter. In accordance with the present invention, polynucleotide sequences which encode human milk proteins such as lysozyme or lactoferrin, include splice variants, fragments of such human milk proteins, fusion proteins, modified forms or functional equivalents thereof, collectively referred to herein as \u201chuman milk protein-encoding nucleic acid sequences\u201d.","Such \u201chuman milk protein-encoding nucleic acid sequences\u201d may be used in recombinant expression vectors (also termed heterologous nucleic acid constructs), that direct the expression of a human milk protein in appropriate host cells.","Due to the inherent degeneracy of the genetic code, a number of nucleic acid sequences which encode substantially the same or a functionally equivalent amino acid sequence may be generated and used to clone and express a given human milk protein, as exemplified herein by the codon optimized coding sequences used to practice the invention (further described below). Thus, for a given human milk protein-encoding nucleic acid sequence, it is appreciated that as a result of the degeneracy of the genetic code, a number of coding sequences can be produced that encode the same human milk protein amino acid sequence. For example, the triplet CGT encodes the amino acid arginine. Arginine is alternatively encoded by CGA, CGC, CGG, AGA, and AGG. Therefore such substitutions in the coding region fall within the range of sequence variants covered by the present invention. Any and all of these sequence variants can be utilized in the same way as described herein for a \u201creference\u201d human milk protein-encoding nucleic acid sequence.","A \u201cvariant\u201d human milk protein-encoding nucleic acid sequence may encode a \u201cvariant\u201d human milk protein amino acid sequence which is altered by one or more amino acids from the native milk protein sequence, both of which are included within the scope of the invention. Similarly, the term \u201cmodified form of\u201d, relative to a given human milk protein, means a derivative or variant form of a native human milk protein or the coding sequence therefor. That is, a \u201cmodified form of\u201d a human milk protein has a derivative sequence containing at least one nucleic acid or amino acid substitution, deletion or insertion. The nucleic acid or amino acid substitution, insertion or deletion may occur at any residue within the sequence, as long as the encoded amino acid sequence maintains the biological activity of the native human milk protein, e.g., the bactericidal effect of lysozyme.","A \u201cvariant\u201d human milk protein-encoding nucleic acid sequence may encode a \u201cvariant\u201d human milk protein sequence which contains amino acid insertions or deletions, or both. Furthermore, a variant human milk protein coding sequence may encode the same polypeptide as the reference polynucleotide or native sequence but, due to the degeneracy of the genetic code, has a nucleic acid coding sequence which is altered by one or more bases from the reference or native polynucleotide sequence.","The variant nucleic acid coding sequence may encode a variant amino acid sequence which contains a \u201cconservative\u201d substitution, wherein the substituted amino acid has structural or chemical properties similar to the amino acid which it replaces and physicochemical amino acid side chain properties and high substitution frequencies in homologous proteins found in nature (as determined, e.g., by a standard Dayhoff frequency exchange matrix or BLOSUM matrix). In addition, or alternatively, the variant nucleic acid coding sequence may encode a variant amino acid sequence which contains a \u201cnon-conservative\u201d substitution, wherein the substituted amino acid has dissimilar structural or chemical properties to the amino acid which it replaces.","Standard substitution classes include six classes of amino acids based on common side chain properties and highest frequency of substitution in homologous proteins in nature, as is generally known to those of skill in the art and may be employed to develop variant human milk protein-encoding nucleic acid sequences. A \u201cvariant\u201d human milk protein-encoding nucleic acid sequence may encode a \u201cvariant\u201d human milk protein sequence which contains a combination of any two or three of amino acid insertions, deletions, or substitution.","Human milk protein-encoding nucleotide sequences also include \u201callelic variants\u201d defined as an alternate form of a polynucleotide sequence which may have a substitution, deletion or addition of one or more nucleotides, which does not substantially alter the function of the encoded polypeptide.","The polynucleotides for use in practicing the invention include sequences which encode human milk proteins and splice variants thereof, sequences complementary to the protein coding sequence, and novel fragments of the polynucleotide. The polynucleotides may be in the form of RNA or in the form of DNA, and include messenger RNA, synthetic RNA and DNA, cDNA, and genomic DNA. The DNA may be double-stranded or single-stranded, and if single-stranded may be the coding strand or the non-coding (anti-sense, complementary) strand.","As will be understood by those of skill in the art, in some cases it may be advantageous to use a human milk protein-encoding nucleotide sequences possessing non-naturally occurring codons. Codons preferred by a particular eukaryotic host (Murray et al., 1989) can be selected, for example, to increase the rate of human milk protein expression or to produce recombinant RNA transcripts having desirable properties, such as a longer half-life, than transcripts produced from naturally occurring sequence. Codon-optimized sequences for use in practicing the invention are further described below.","A human milk protein-encoding nucleotide sequence may be engineered in order to alter the human milk protein coding sequence for a variety of reasons, including but not limited to, alterations which modify the cloning, processing and\/or expression of the human milk protein by a cell.","Heterologous nucleic acid constructs may include the coding sequence for a given human milk protein, a variant, fragment or splice variant thereof: (i) in isolation; (ii) in combination with additional coding sequences; such as fusion protein or signal peptide, in which the human milk protein coding sequence is the dominant coding sequence; (iii) in combination with non-coding sequences, such as introns and control elements, such as promoter and terminator elements or 5\u2032 and\/or 3\u2032 untranslated regions, effective for expression of the coding sequence in a suitable host; and\/or (iv) in a vector or host environment in which the human milk protein coding sequence is a heterologous gene.","Depending upon the intended use, an expression construct may contain the nucleic acid sequence which encodes the entire human milk protein, or a portion thereof. For example, where human milk protein sequences are used in constructs for use as a probe, it may be advantageous to prepare constructs containing only a particular portion of the human milk protein encoding sequence, for example a sequence which is discovered to encode a highly conserved human milk protein region.","In one general embodiment, a human lysozyme amino acid sequence encoded by a human lysozyme-encoding nucleic acid sequence in an expression vector used to practice the invention has at least 70%, preferably 80%, 85%, 90% or 95% or more sequence identity to the human lysozyme amino acid sequence presented as SEQ ID NO: 2.","In another general embodiment, a human lactoferrin amino acid sequence encoded by a human lactoferrin-encoding nucleic acid sequence in an expression vector used to practice the invention has at least 70%, preferably 80%, 85%, 90% or 95% or more sequence identity to the human lactoferrin amino acid sequence presented as SEQ ID NO: 4.","D. Codon Optimization","It has been shown that production of recombinant protein in transgenic barley grain was enhanced by codon optimization of the gene (Horvath et al., 2000; Jensen et al., 1996). The intent of codon optimization was to change an A or T at the third position of the codons of G or C. This arrangement conforms more closely with codon usage in typical rice genes (Huang et al., 1990a).","In order to obtain a high expression level for human lysozyme in rice cells, the coding sequence was codon optimized. The G+C content was thus increased from 46% to 68%. The codon optimized lysozyme coding sequence for use in practicing the invention is presented as SEQ ID NO: 1.","Similarly, in order to obtain high level expression level of human lactoferrin (hLF) in rice cells, the native hLF coding sequence was codon optimized. Out of 693 codons used in the lactoferrin gene, 413 codons were changed by one or two nucleotides. The amino acid sequence of LF was unchanged. The codon optimized lactoferrin coding sequence for use in practicing the invention is presented as SEQ ID NO: 3.","Codon optimized sequences for other human milk proteins are given as follows: for lactoferricin, (SEQ ID NO: 7; for EGF, SEQ ID NO: 8; for IGF-1, SEQ ID NO: 9; for lactadherin, SEQ ID NO: 10; for kappa-casein, SEQ ID NO: 11; for haptocorrin, SEQ ID NO: 12; for lactoperoxidase, SEQ ID NO: 13; and for alpha-1-antitrypsin, SEQ ID NO: 14.","E. Transcription Factor Coding Sequences","In one embodiment of the invention, the transgenic plant is also transformed with the coding sequence of one or more transcription factors capable of stimulating the expression of a maturation-specific promoter. Specifically, the embodiment involves the use of the maize Opaque 2 (O2) and prolamin box binding factor (PBF) together with the rice endosperm bZip (Reb) protein as transcriptional activators of monocot storage protein genes. Exemplary sequence for these three transcription factors are given identified below as SEQ ID NOS: 31-33. Transcription factor sequences and constructs applicable to the present invention are detailed in co-owned PCT application No. PCT\/US01\/14234, International Publication number WO 01\/83792 A1, published Nov. 8, 2001, which is incorporated herein by reference.","Transcription factors are capable of sequence-specific interaction with a gene sequence or gene regulatory sequence. The interaction may be direct sequence-specific binding in that the transcription factor directly contacts the gene or gene regulatory sequence or indirect sequence-specific binding mediated by interaction of the transcription factor with other proteins. In some cases, the binding and\/or effect of a transcription factor is influenced (in an additive, synergistic or inhibitory manner) by another transcription factor. The gene or gene regulatory region and transcription factor may be derived from the same type (e.g., species or genus) of plant or a different type of plant. The binding of a transcription factor to a gene sequence or gene regulatory sequence may be evaluated by a number of assays routinely employed by those of skill in the art, for example, sequence-specific binding may be evaluated directly using a label or through gel shift analysis.","As detailed in the cited PCT application, the transcription factor gene is introduced into the plant in a chimeric gene containing a suitable promoter, preferably a maturation-specific seed promoter operably linked to the transcription factor gene. Plants may be stably transformed with a chimeric gene containing the transcription factor by methods similar to those described with respect to the milk-protein gene(s). Plants stably transformed with both exogeneous transcription factor(s) and milk-protein genes may be prepared by co-transforming plant cells or tissue with both gene constructs, selecting plant cells or tissue that have been co-transformed, and regenerating the transformed cells or tissue into plants. Alternatively, different plants may be separately transformed with exogeneous transcription factor genes and milk-protein genes, then crossed to produce plant hybrids containing by added genes.","F. Additional Expression Vector Components","Expression vectors or heterologous nucleic acid constructs designed for operation in plants, comprise companion sequences upstream and downstream to the expression cassette. The companion sequences are of plasmid or viral origin and provide necessary characteristics to the vector to permit the vector to move DNA from bacteria to the plant host, such as, sequences containing an origin of replication and a selectable marker. Typical secondary hosts include bacteria and yeast.","In one embodiment, the secondary host is , the origin of replication is a colE1-type, and the selectable marker is a gene encoding ampicillin resistance. Such sequences are well known in the art and are commercially available as well (e.g., Clontech, Palo Alto, Calif.; Stratagene, La Jolla, Calif.).","The transcription termination region may be taken from a gene where it is normally associated with the transcriptional initiation region or may be taken from a different gene. Exemplary transcriptional termination regions include the NOS terminator from Ti plasmid and the rice \u03b1-amylase terminator.","Polyadenylation tails (Alber et al., 1982) may also be added to the expression cassette to optimize high levels of transcription and proper transcription termination, respectively. Polyadenylation sequences include, but are not limited to, the octopine synthetase signal, Gielen, et al., 1984 or the nopaline synthase of the same species, Depicker, et al., 1982.","Suitable selectable markers for selection in plant cells include, but are not limited to, antibiotic resistance genes, such as, kanamycin (nptII), G418, bleomycin, hygromycin, chloramphenicol, ampicillin, tetracycline, and the like. Additional selectable markers include a bar gene which codes for bialaphos resistance; a mutant EPSP synthase gene which encodes glyphosate resistance; a nitrilase gene which confers resistance to bromoxynil; a mutant acetolactate synthase gene (ALS) which confers imidazolinone or sulphonylurea resistance; and a methotrexate resistant DHFR gene.","The particular marker gene employed is one which allows for selection of transformed cells as compared to cells lacking the DNA which has been introduced. Preferably, the selectable marker gene is one which facilitates selection at the tissue culture stage, e.g., a kanamyacin, hygromycin or ampicillin resistance gene.","The vectors of the present invention may also be modified to include intermediate plant transformation plasmids that contain a region of homology to an vector, a T-DNA border region from , and chimeric genes or expression cassettes (described above). Further, the vectors of the invention may comprise a disarmed plant tumor inducing plasmid of ","In general, a selected nucleic acid sequence is inserted into an appropriate restriction endonuclease site or sites in the vector. Standard methods for cutting, ligating and transformation, known to those of skill in the art, are used in constructing vectors for use in the present invention. (See generally, Maniatis, et al., MCA LM, 2d Edition (1989); Ausubel, et al., (c) 1987, 1988, 1989, 1990, 1993, John Wiley & Sons, New York, N.Y; and Gelvin, S. B., et al., eds. (1990), all three of which are expressly incorporated by reference, herein.","V. Generation of Transgenic Plants","Plant cells or tissues are transformed with expression constructs (heterologous nucleic acid constructs, e.g., plasmid DNA into which the gene of interest has been inserted) using a variety of standard techniques. Effective introduction of vectors in order to facilitate enhanced plant gene expression is an important aspect of the invention. It is preferred that the vector sequences be stably integrated into the host genome.","The method used for transformation of host plant cells is not critical to the present invention. The transformation of the plant is preferably permanent, i.e. by integration of the introduced expression constructs into the host plant genome, so that the introduced constructs are passed onto successive plant generations. The skilled artisan will recognize that a wide variety of transformation techniques exist in the art, and new techniques are continually becoming available.","Any technique that is suitable for the target host plant may be employed within the scope of the present invention. For example, the constructs can be introduced in a variety of forms including, but not limited to, as a strand of DNA, in a plasmid, or in an artificial chromosome. The introduction of the constructs into the target plant cells can be accomplished by a variety of techniques, including, but not limited to calcium-phosphate-DNA co-precipitation, electroporation, microinjection, -mediated transformation, liposome-mediated transformation, protoplast fusion or microprojectile bombardment. The skilled artisan can refer to the literature for details and select suitable techniques for use in the methods of the present invention. Exemplary methods for plant transformation are given in Example 2.","When is used for plant cell transformation, a vector is introduced into the host for homologous recombination with T-DNA or the Ti- or Ri-plasmid present in the host. The Ti- or Ri-plasmid containing the T-DNA for recombination may be armed (capable of causing gall formation) or disarmed (incapable of causing gall formation), the latter being permissible, so long as the vir genes are present in the transformed host. The armed plasmid can give a mixture of normal plant cells and gall.","In some instances where is used as the vehicle for transforming host plant cells, the expression or transcription construct bordered by the T-DNA border region(s) is inserted into a broad host range vector capable of replication in and examples of which are described in the literature, for example pRK2 or derivatives thereof. See, for example, Ditta et al., 1980 and EPA 0 120 515, expressly incorporated by reference herein. Alternatively, one may insert the sequences to be expressed in plant cells into a vector containing separate replication sequences, one of which stabilizes the vector in , and the other in See, for example, McBride et al., 1990, wherein the pRiHRI (Jouanin, et al., 1985, origin of replication is utilized and provides for added stability of the plant expression vectors in host cells.","Included with the expression construct and the T-DNA is one or more selectable marker coding sequences which allow for selection of transformed and transformed plant cells. A number of markers have been developed for use with plant cells, such as resistance to chloramphenicol, kanamycin, the aminoglycoside G418, hygromycin, or the like. The particular marker employed is not essential to this invention, with a particular marker preferred depending on the particular host and the manner of construction.","For -mediated transformation of plant cells, explants are incubated with for a time sufficient to result in infection, the bacteria killed, and the plant cells cultured in an appropriate selection medium. Once callus forms, shoot formation can be encouraged by employing the appropriate plant hormones in accordance with known methods and the shoots transferred to rooting medium for regeneration of plants. The plants may then be grown to seed and the seed used to establish repetitive generations and for isolation of the recombinant protein produced by the plants.","There are a number of possible ways to obtain plant cells containing more than one expression construct. In one approach, plant cells are co-transformed with a first and second construct by inclusion of both expression constructs in a single transformation vector or by using separate vectors, one of which expresses desired genes. The second construct can be introduced into a plant that has already been transformed with the first expression construct, or alternatively, transformed plants, one having the first construct and one having the second construct, can be crossed to bring the constructs together in the same plant.","A. Plants","Host cells of the present invention include plant cells, both monocotyledenous and dicotyledenous. In one preferred embodiment, the plants used in the methods of the present invention are derived from monocots, particularly the members of the taxonomic family known as the Gramineae. This includes all members of the grass family of which the edible varieties are known as cereals. The cereals include a wide variety of species such as wheat (sps.), rice (sps.) barley (sps.) oats, (sps.) rye (sps.), corn (maize) (sps.) and millet (sps.). In practicing the present invention, preferred grains are rice, wheat, maize, barley, rye, triticale. Also preferred are dicots exemplified by soybean (spp.)","In order to produce transgenic plants that express human milk protein, monocot plant cells or tissues derived from them are transformed with an expression vector comprising the coding sequence for a human milk protein. Transgenic plant cells obtained as a result of such transformation express the coding sequence for a human milk protein, such as lysozyme or lactoferrin. The transgenic plant cells are cultured in medium containing the appropriate selection agent to identify and select for plant cells which express the heterologous nucleic acid sequence. After plant cells that express the heterologous nucleic acid sequence are selected, whole plants are regenerated from the selected transgenic plant cells. Techniques for regenerating whole plants from transformed plant cells are generally known in the art. Transgenic plant lines, e.g., rice, wheat, corn or barely, can be developed and genetic crosses carried out using conventional plant breeding techniques.","Production of recombinant proteins in monocot seeds, e.g., rice (L.) seeds has the advantages that (a) high level expression make it an economically practical strategy, and (b) rice is a normal part of the diet of infants and children, has good nutritional value and low allergenicity. Thus, the use of rice as the basis for a food supplement is unlikely to introduce any risk and thereby eliminates the need for a high degree of purification when included in infant formula.","In addition, rice is the staple food crop of more than half the world's population. Recent reports on the production of provitamin A (beta-Carotene) in rice seeds exemplifies the need for value added food crops especially in the developing world (Ye et al., 2000) where rice is used as major food crop.","VI. Detecting Expression of Recombinant Human Milk Proteins","Transformed plant cells are screened for the ability to be cultured in selective media having a threshold concentration of a selective agent. Plant cells that grow on or in the selective media are typically transferred to a fresh supply of the same media and cultured again. The explants are then cultured under regeneration conditions to produce regenerated plant shoots. After shoots form, the shoots are transferred to a selective rooting medium to provide a complete plantlet. The plantlet may then be grown to provide seed, cuttings, or the like for propagating the transformed plants. The method provides for efficient transformation of plant cells with expression of a gene of autologous or heterologous origin and regeneration of transgenic plants, which can produce a recombinant human milk protein.","The expression of the recombinant human milk protein may be confirmed using standard analytical techniques such as Western blot, ELISA, PCR, HPLC, NMR, or mass spectroscopy, together with assays for a biological activity specific to the particular protein being expressed.","Example 3 describes the characterization of human lysozyme produced in the seeds of transgenic rice plants. Analyses used to confirm that recombinant lysozyme produced in transgenic rice is essentially the same as the native form of the protein both in physical characteristics and biological activity included, SDS-PAGE, reverse IEF gel electrophoresis, Western blot analysis, enzyme linked immunosorbant assay (ELISA), enzymatic activity assay and bactericidal activity assay using indicator strains, and strain JM109.","Example 4 describes the characterization of human lactoferrin produced in the seeds of transgenic rice plants. Analyses used to confirm that recombinant lactoferrin produced in transgenic rice is essentially the same as the native form of the protein both in physical characteristics and biological activity included, Southern blot, Western blot, ELISA, N-Terminal Amino Acid Sequencing, analysis of glycosylation and determination of sugar content, a determination of the isoelectric point, pH dependent iron release of rLF, bacteriostatic activity assay of rLF using enteropathogenic as the indicator strain.","Example 5 details the characterization of alpha-1-antitrypsin produced by transgenic monocot plant cells. Example 6 details the characterization of other milk proteins also produced by monocot plant transformed with the chimeric genes of the invention.","VII. Preparation of Seed Composition and Processed Foods","The invention provides, in one embodiment, a seed composition containing a flour, extract, or malt obtained from mature monocot seeds and one or more seed-produced human milk proteins in substantially unpurified form. Where the milk protein is expressed at a level of between about 0.1 to 1 percent of the seed weight, the composition will contain the same or preferably a higher percentage of milk protein, e.g., 0.1 to 20% of the composition depending on the composition added. In particular, a grain composition will yield an amount of milk protein that is comparable to that in the mature seed; the extract composition, by contrast, in which most of the starch has been removed, will typically show a severalfold increase in percentage of milk protein, e.g., 10-40% of the total weight of the extract. The malt composition will contain an intermediate level, typically greater than grain, but less than extract.","In determining the amount of grain, extract, or malt composition to be added to a food, it is useful to determine the amount of any milk protein present (see Section VI above), and add an amount of composition which brings the final level of milk protein to a desired level in the food. For example, in infant formula, it may be desired to have a final concentration of lysozyme between 0.03 and 0.3 grams\/liter of formula, and an amount of lactoferrin between about 0.3 to 3 grams\/liter formula. Thus, if a seed composition is found to contain 10 g\/kg lysozyme, about 10 grams of the composition would be added to make up a liter of formula with a final lysozyme concentration of about 0.1 g\/liter. Below are described methods for preparing each of the three types of milk-protein-containing seed compositions.","A. Flour Composition","The flour composition is prepared by milling mature monocot plant seeds, using standard milling and, optionally, flour purification methods, e.g., in preparing refined flour. Briefly, mature seeds are dehusked, and the dehusked seeds then ground into a fine flour by conventional milling equipment.","The flour may be added to foods during food processing according to standard food processing methods. Preferably, the processing temperature does not lead to denaturation of the milk proteins, e.g., above 60\u00b0-70\u00b0 C. The flour may also be used directly, either in capsule, tabletized, or powder form, as a neutriceutical composition. One preferred flour composition contains lactoferrin and\/or lysozyme. The flour may alternatively, or in addition, include one or more of the other human milk proteins such as epidermal growth factor, insulin-like growth factor-1, lactadherin, kappa-casein, haptocorrin, lactoperoxidase, and alpha-1-antitrypsin.","Flour containing two or more milk proteins may be prepared by combining flour from seeds that separately produce the different proteins, for example, equal amounts of a flour containing lysozyme and a flour containing lactoferrin. Alternatively, a multi-protein composition can be prepared as seed flour from plants, such as monocot plants co-transformed with chimeric genes expressing different milk proteins, e.g., lactoferrin and lysozyme.","B. Extract Composition","An extract composition is prepared by milling seeds to form a flour, extracting the flour with am aqueous buffered solution, and optionally, further treating the extract to partially concentrate the extract and\/or remove unwanted components. Details of exemplary methods for producing the extract composition are given in Example 9. Briefly, mature monocot seeds, such as rice seeds, are milled to a flour, and the flour then suspended in saline or in a buffer, such as Phosphate Buffered Saline (\u201cPBS\u201d), ammonium bicarbonate buffer, ammonium acetate buffer or Tris buffer. A volatile buffer or salt, such as ammonium bicarbonate or ammonium acetate may obviate the feed for a salt-removing step, and thus simplify the extract processing method.","The flour suspension is incubated with shaking for a period typically between 30 minutes and 4 hours, at a temperature between 20-55\u00b0 C. The resulting homogenate is clarified either by filtration or centrifugation. The clarified filtrate or supernatant may be further processed, for example by ultrafiltration or dialysis or both to remove contaminants such as lipids, sugars and salt. Finally, the material may dried, e.g., by lyophilization, to form a dry cake or powder. The extract combines advantages of high milk-protein yields, essentially limiting losses associated with protein purification. At the same time, the milk proteins are in a form readily usable and available upon ingestion of the extract or food containing the extract. One particular advantage for use in infant formula or infant foods is the low amount of seed starch present in the extract. In particular, the extract may increase the concentration of recombinant protein from about 0.5% of total soluble protein (\u201cTSP\u201d) in conventional approaches to over about 25% of TSP in the extract approach. Some of the present extract approach even reached 40% of TSP depending on the expression level of the recombinant protein in the seeds. In addition, the extract approach removes starch granules, which require high gelling temperature, for example above about 75\u00b0 C. Consequently, the extract approach provides more flexibility in processing the rice grain and the recombinant proteins into food and nutritional drinks, particularly infant food and formula, because of the difficulty infants have in digesting undenatured seed starch. Undenatured starch granule cannot be digested by human gut without initial gelatinization, by for example high temperature.","The extract can be used as a nutraceutical for direct use, e.g., in capsule, tabletized or powder form, or as food additive in food processing. In one embodiment, the extract is added to an infant milk formula, in an amount typically between 0.1 to 10 percent by dry weight, preferably 1-5% by dry weight of the total formula weight. One preferred infant formula contains both lactoferrin and lysozyme, preferably in an amount between 50-200% of the amount of human lactoferrin or lysozyme, respectively, of that found in normal human milk. As noted above, lactoferrin is present in a concentration of about 1 gram\/liter human milk, and lysozyme, about 0.1\/liter human milk. The extract may alternatively, or in addition, include one or more of the other human milk proteins including epidermal growth factor, insulin-like growth factor-1, lactadherin, kappa-casein, haptocorrin, lactoperoxidase, alpha-1-antitrypsin and immunoglobulins. Similarly, for use as an additive to solid baby food, or to nutritional drinks, the extract is added in amounts preferably between about 0.1 to 10% of the food\/drink material by dry weight.","As above, extract containing two or more milk proteins may be prepared by combining extracts from seeds that separately produce the different proteins, or by processing seeds from plants co-transformed with chimeric genes expressing different milk proteins, e.g., lactoferrin and lysozyme.","C. Malt Composition","One technical challenge to commercialization of engineered monocot grains expressing human milk proteins is to formulate the transgenic grains into edible products without loss of bioavailable milk protein in the final product.","In accordance with another embodiment, the invention provides a malt extract or malt syrup (\u201cmalt\u201d) in which seed starches have been largely reduced to malt sugars, and the milk protein(s) are in an active, bioavailable form. A wide range of food products and\/or food additive may be produced by varying the types of malt used, the mashing program and the ways in which the wort is subsequently handled. If materials other than barley malt are used in the mash (such as starch from other grains), the resulting product is classified as a malt syrup. Malt extracts, which may have a syrupy consistency or may be powders, are made by mashing ground malt, usually barley malt, in conventional brewery equipment, collecting the wort and concentrating it or drying it. Modern production of food malt extracts and malt syrups has evolved into three basic grain stages: steeping, germination, and drying of the germinated seed, followed by three more steps involving liquefaction of the germinated grain, mashing of the germinated grain, lautering (filtering), and evaporation. Many variations of malt extracts or syrups are possible. Flavor, color, solids, enzymatic activity, and protein are the basic characteristics that can be adjusted during production to provide malts specific for given food applications. (See, generally, Eley; Hickenbottom, 1996,1997a, 1997b, 1983; Lake; Moore; Moe; Sfat; Doncheck; Briggs, 1981, 1998; and Hough).","C1. Steeping","After the barley of choice has been cleaned of foreign material, it is graded to size and transferred to steep tanks equipped with water inlet and outlet pipes. Compressed air is fed from the tank bottom for vigorous aeration and mixing for the barley\/water mixture. When the barley has reached a water content of 43-45%, steeping is stopped.","C2. Germination","The steeped barley is moved to germination floors or rooms depending on the particular malt house's capabilities and allowed to germinate under controlled temperature, air, and moisture conditions. Total germination varies from four to seven days, depending on the barley type, density end use of the malt, and the controls or germination method used. All aspects of germination must be kept in constant balance to ensure proper kernel modification and yield.","Many enzymatic systems are activated during germination. Two of the systems are the oxidative and reductive systems involved with the respiration phase. Other enzymes break down the endosperm cell structure, which in itself if a measure of germination rate when the pentose production is evaluated. The proteolytic enzymes release or active beta-amylase and also work on the proteins present to render them soluble. In fact, about 40% of the total protein is made soluble in water. Optimum germination activates a balanced enzyme system, which hydrolyzes the starch present.","C3. Kilning","Drying or kilning, when done at the proper time and optimum degree of starch modification, stops the germination. The heat also catalyzes additional reactions, notably flavor and color development. The heating step is carried out according to well known kilning conditions. When drying is complete, the sprouts and other extraneous materials are removed, and the kernels are then ready for further processing.","C4. Malt Extracts and Syrups","The malted barley (kernel) is coarsely ground in crushers and fed into mash tuns where it is mixed with water. During a series of time and temperature changes, some of the starch is converted into fermentable sugars by action of the natural alpha- and beta-amylases, better known as the diastatic system. If cereal adjuncts are to be added, which result in malt syrups with mellower and sweeter flavors than the extracts, they are added at this stage usually derived from the cereal grains, corn and rice, although barley, wheat, rye, millet and sorghum are sometimes used, derived from mature seeds that produce the desired recombinant milk proteins.","Once the mash batch has achieved the correct degree of hydrolysis, it is transferred to lauter tuns. The lauter tun has a slotted or false bottom a few inches above the real bottom to allow for filtration and is also equipped with some means of agitation. During this extraction stage, the amyolytic enzymes liquefy additional insoluble starches, converting them to maltose and dextrins. At the same time, the proteolytic enzymes attach certain proteins converting them into simpler, soluble forms. After the appropriate conditions have been met, the liquid phase, or wort, is drawn from the lauter tuns into evaporators.","Evaporation of the wort is conducted under vacuum where it is converted into a syrup of about 80% solids. Depending on the temperatures used, malt extracts or syrups of high, medium, or zero enzymatic activity can be produced. Color and flavor also can be controlled during this stage. The finishing steps of filtering, cooling, and packaging complete the malt extract\/syrup process.","C5. Transgenic Malt Extract","For a transgenic malt extract, the starting barley is a transgenic barley engineered to produce on or more human milk proteins in the endosperm either in grain maturation or in the malting process, or at both times. Malting and processing times and conditions are adjusted so that the bioactivity of the target recombinant molecules is preserved and the bioavailability of the target recombinant molecule is maximized. The resulting malt extract is either consumed directly as a concentrate that is either consumed directly as a food, or is incorporated as an ingredient in a food mixture. Studies conducted in support of the present invention demonstrate that recombinant proteins retain activity after malting for up to at least 288 hrs.","C6. Transgenic Malt Syrup","For a transgenic malt syrup, the starting barley can be a non-transgenic barley, or a transgenic barley, or a mixture of both. The barley is processed as described, except that during the mashing process, a cereal adjunct is added in a form that it is converted during the mashing process with the concurrent retention and generation of bioavailability and bioactivity of the target recombinant molecule fond within the transgenic cereal adjunct. The use of a transgenic cereal adjunct enables the production in the malt syrup of the target recombinant molecule expressed in the transgenic grain endosperm.","The malt extract or syrup may be used directly as a syrup, or added to processed foods or drinks, according to standard food processing procedures that employ grain extracts or syrups, e.g., for sweetening. One preferred food is an infant formula containing between 0.1 to 10% malt (extract or syrup). The malt is also useful as a sweetener\/nutritional additive in baby and adult foods, and nutritional drinks.","Preferred malt extracts or syrups contain lactoferrin and\/or lysozyme. The malt may alternatively, or in addition, include one or more of the human milk proteins such as epidermal growth factor, insulin-like growth factor-1, lactadherin, kappa-casein, haptocorrin, lactoperoxidase, alpha-1-antitrypsin and immunoglobulins. As above, malt containing two or more milk proteins may be prepared by combining or preparing malts from seeds that separately produce the different proteins, or by preparing a malt from the seeds of plants co-transformed with chimeric genes expressing different milk proteins, e.g., lactoferrin and lysozyme.","From the foregoing, it can be appreciated how various objects and features of the invention are met. The production of high levels of human milk proteins in grains, exemplified herein by rice provides the distinct advantage that food supplements may be prepared with little or no purification. In a preferred approach, the human milk protein containing transgenic grain is ground (e.g., into flour) and directly added to a food such as infant formula, without additional processing. Since the recombinant grain finds utility as a food or food supplement, the regulatory requirements for purity are not stringent.","Transgenic seeds are ideal bioreactors, combining low production costs and low or minimal downstream processing costs prior to use. Seed grain proteins can accumulate to 9-19% of grain weight (L\u00e1sztitym 1996); the endosperm proteins are synthesized during grain maturation and stored in protein bodies for use in the germination and seedling growth of the next plant generation; grains can be stored for years without loss of functionality, and therefore the downstream processing can be conducted independently of growing seasons.","The human milk protein-containing transgenic grains of the invention may be used directly as food, e.g., rice, corn, wheat, barley, soybeans, etc. Alternatively, food supplements are prepared from the human milk protein-containing transgenic grain. The results presented herein demonstrate that human milk proteins may be expressed at high levels in the seeds of transgenic plants, e.g., up to 0.25 to 1% of total seed dry weight. The production of high levels of human milk proteins in grains, exemplified herein by rice, provides the distinct advantage that food supplements may be prepared with little or no purification. In a preferred approach, the human milk protein containing transgenic grain is ground (e.g., into flour) and directly added to a food, or in the form of an extract or malt, such as for preparing a nutritionally enhanced infant formula, without additional processing. Since the recombinant grain finds utility as a food or food supplement, as a flour, extract or malt, the regulatory requirements for purity are not stringent. Accordingly, human milk protein-containing transgenic grains are ideal bioreactors, combining low production costs and low or minimal downstream processing costs prior to use.","The human milk protein-containing transgenic grains of the invention may be used directly as food, e.g., rice, corn, wheat, barley, soybeans, etc. Alternatively, food supplements are prepared from the human milk protein-containing transgenic grain. Where the transgenic seed is rice, the invention provides additional advantages in that: rice is consumed by a majority of the population in the world and is being generally regarded as safe for human consumption. Rice-based foods are considered hypoallergenic (NIH publication, 1984). In many countries, rice is the first solid food for infants and rice-based infant formulas are commercially available (Bhan et al., 1988; Gasta\u00f1aduy et al, 1990). These make rice attractive as a \u201cprotein factory\u201d to produce biomedicals and nutraceuticals for human consumption. The cloning and expression of human proteins, for example, human milk proteins lysozyme and lactoferrin in rice grains has opened a new avenue for the bioproduction of other milk proteins.","All publications, patents and patent applications are herein expressly incorporated by reference in their entirety to the same extent as if each individual publication, patent or patent application was specifically and individually indicated to be incorporated by reference in its entirety.","The following examples illustrate but are not intended in any way to limit the invention.","In general, expression vectors were constructed using standard molecular biological techniques as described in Ausubel et al., 1987. The vectors contain a heterologous protein coding sequence for lactoferrin or lysozyme under the control of a rice tissue-specific promoter, as further described below.","A. An Expression Vector for Human Lysozyme Expression in Transgenic Rice Cells","The synthesized lysozyme gene was cloned into an API base vector pAPI137 by conventional molecular cloning techniques (Sambrook et al., 1989). Plasmid pAPI137 contains the RAmy3D promoter (Huang et al., 1993), the codons for the RAmy3D signal peptide and the RAmy3D terminator. The RAmy3D promoter, isolated from the rice amylase gene family, is activated in rice calli by sugar starvation (Huang et al., 1993). The human lysozyme gene was placed between the sequences of the RAmy3D signal peptide and the RAmy3D terminator to give plasmid pAPI156 having a size of 4829 bp.","The promoter of the rice Glutelin 1 gene (Gt-1) and the nucleotide sequence of the signal peptide were cloned with two primers based on the published Gt1 gene sequence (Okita et al. J Biol Chem 264: 12573-12581, 1989). The forward primer with HindIII site was named MV-Gt-1-F1; 5\u2032-ATCGAACTTCATGAGTAAGTGTGAGCATTATGGGACCACG-3\u2032 (SEQ ID NO:5). The reverse primer was named Xba-Gt-1-R1; 5\u2032-CTAGTCTAGACTCGAGCCACGGCCATGGGGCCGGCTAGGGAGCCATCGCAAAGAGGAA-3\u2032 (SEQ ID NO: 6). Genomic DNA was isolated from leaves of rice variety M202 (Dellaporta et al., 1983). The PCR product amplified from the genomic DNA was cloned into pCR 2.1 (Invitrogen, Carlsbad, Calif.). The resulting plasmid was named pCRGt-1 or pAPI134.","To generate a Gt-1 expression plasmid, pAPI134 was digested with HindIII and XbaI. The fragment containing the Gt-1 promoter and Gt-1 signal peptide was cloned into a pUC19 based plasmid containing the nopaline synthase 3\u2032 (nos) terminator. The resulting plasmid was named pAPI141 and contains the rice Gt-1 promoter, the Gt-1 signal peptide, a multiple cloning site and the nos terminator.","The synthesized human lysozyme gene \u201clys-ger\u201d (by Operon Technologies, Inc., Alameda, Calif.) that was optimized based on the rice gene codon usage was digested with DraI and XhoI and cloned into pAPI141 digested with NaeI and XhoI according to standard cloning techniques (Sambrook et al., 1989). The resulting plasmid was called pAPI159 () having a size of 4131 bp.","B. An Expression Vector for Human Lactoferrin Expression in Transgenic Rice","The hLF gene (Rey, MW, 1990) was codon optimized and synthesized by Operon Technologies (CA, USA). The plasmid containing the codon-optimized gene was called Lac-ger. Lac-ger was digested with SmaI\/XhoI and the fragment containing the lactoferrin gene was cloned into pAPI141 which was partially digested with NaeI and completely digested with XhoI. The resulting plasmid was named pAPI164. For expression of hLF in rice seeds, the codon optimized gene was operably linked to the rice endosperm specific glutelin (Gt1) promoter and NOS terminator (FIG. ).","The procedure of microprojectile-mediated rice transformation (U.S. Pat. No. 6,284,956) was followed. Calli was raised from TP309 mature rice seeds, with calli two to four mm in diameter selected and placed on N6 media supplemented with 0.3 M mannitol and 0.3 M sorbitol for 20 hours before bombardment. Biolistic bombardment was carried out with the biolistic PDC-1000\/He system (Bio-Rad, USA). Plasmid carrying milk protein genes and pAPI76, a plasmid carrying hygromycin selectable marker gene were gold-coated and co-bombarded at a ratio of 6:1 with a helium pressure of 1100 psi. Two day old bombarded calli were then transferred to N6 selection media supplemented with 20 mg\/I hygromycin B and allowed to grow in the dark at 26\u00b0 C. for 45 days.","In order to develop transgenic rice plants, the selected calli were transferred to pre-regeneration and regeneration media. When regenerated plants became 1-3 cm in height, the plantlets were transferred to rooting media which consisted of half concentration of MS and 0.05 mg\/l NAA. After two weeks, plantlets with developed roots and shoots were transferred to soil and kept under the cover of plastic container for a week. The plants were allowed to grow about 12 cm tall and shifted to the green house where they were grown up to maturity.","A. Generation of Human Lysozyme Expressing Transgenic Rice Cells and Plants","The synthetic human lysozyme (hLys) gene under the control of the RAmy3D promoter and terminator in the pAPI156 plasmid (example 1A) was used to generate sixty independent transformants by particle bombardment-mediated transformation.","Particle bombardment mediated transformation of rice was carried out as described above. Briefly, rice calli derived from TP309 were bombarded with gold particles coated with plasmids pAPI156 and pAPI76 in a ratio of 6:1 using the helium biolistic particle delivery system, PDS 1000 (Bio-Rad, CA). Transformed calli were selected in the presence of hygromycin B (35 mg\/L) on N6 (Sigma, Mo).","Selected cell lines were maintained in culture media with 3% sucrose (Huang et al., 1993). Lysozyme expression was induced by sugar starvation. Briefly, AA medium (containing 3% sucrose) was removed by aspiration, followed by washing the cells three times with AA minus sucrose (AA-S). The cells were then incubated with AA-S at 40% (v\/v) density for three and a half days to obtain the optimal level of lysozyme expression.","Transformants expressing lysozyme were identified by immunoblot analysis, turbidimetric rate determination with or ELISA. Calli were ranked according to the expressed lysozyme level. Suspension cell cultures from the top lines were established following the procedure described previously (Huang et al., 1993). The amount of total protein (Bradford assay) and lysozyme (ELISA) was evaluated in selected calli (Table 1).",{"@attributes":{"id":"p-0414","num":"0413"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Expression Level Of Human Milk Lysozyme In Transformed Calli"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Cell",{},{},{},{}]},{"entry":["line","Calli (g)","Total protein (\u03bcg)","Lysozyme (\u03bcg)","Lysozyme\/protein"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"56pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["156-1\u2002","0.39","2626.5","65.7","2.5"]},{"entry":["156-5\u2002","0.38","5510","68.9","1.25"]},{"entry":["156-16","0.4","4815","120.4","2.5"]},{"entry":["156-19","0.44","2440","30.5","1.25"]},{"entry":["156-28","0.49","4910","24.6","0.5"]},{"entry":["156-43","0.56","8150","101.9","1.25"]},{"entry":["156-47","0.37","2472","6.2","0.25"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},"The synthetic human lysozyme (hLys) gene under the control of the Gt1 promoter and Nos terminator in the pAPI159 plasmid () was used to generate independent transformants by particle bombardment-mediated transformation. Transformed calli were selected as described above, then transferred to pre-regeneration and regeneration media. When regenerated plants became 1-3 cm in height, the plantlets were transferred to rooting media which consisted of half concentration of MS and 0.05 mg\/l NAA. After two weeks, plantlets with developed roots and shoots were transferred to soil and kept under the cover of plastic container for a week. The plants were allowed to grow about 12 cm tall and shifted to the green house where they were grown up to maturity (R0 plants).","Screening for R0 plants expressing human lysozyme. Individual rice endosperms or grains were ground with cold phosphate buffered-saline (PBS) with the addition of 0.35 M NaCl. Grinding was conducted with a pre-cooled mortar and pestle at 1 ml buffer\/grain. Clear grain homogenate was obtained by subjecting the resulting grain extract to centrifugation at 14,000 rpm for 10 min at 40\u00b0 C.","Embryos from individual Rseed (derived from Rplants) that showed a level of lysozyme expression that was greater than 10 \u03bcg\/seed were saved and used to generate R1 plants. Briefly, seeds were dissected into embryo and endosperm portions. The endosperm was ground and assayed for lysozyme expression (as further described below). Embryos were sterilized in 50% commercial bleach for 25 minutes and washed with sterile HO three times for 5 minutes each. Sterilized embryos were placed in a tissue culture tube that contained MS solid medium. Embryos germinated and plantlets having about three inches shoots and healthy root systems were obtained in two weeks. The plantlets were then transferred to pots to obtain mature plants (R).","A total of 197 embryos from 12 selected R0 plants were germinated and 157 Rseedlings planted in the greenhouse for generation of R2 grains. Individual R2 grains (n=1502) from 109 R1 fertile plants were screened for lysozyme expression by lysozyme activity assay in order to identify 42 homozygous plants.","Homozygous R1 plants were identified by analyzing positive expressions of recombinant human lysozyme (rHlys) from a minimum of 20 individual R2 grains. Homozygous lines derived from these plants were planted in a rice field in California. During growth, agronomic characteristics of both transgenic and non-transgenic plants, such as plant height, percentage of fertility, number of effective tillers, filled grains\/panicle, non-filled grains\/plant, time to maturity and 1000 grain weight were determined and compared. Plants with satisfactory agronomic traits were selected and rHlys expression levels were determined by lysozyme activity assay. Plants that met the criteria for satisfactory agronomic traits and had more than 35 \u03bcg of rHlys\/grain were advanced to next generations.","SDS-PAGE, electroblotting and Western blot analysis were carried out with 18% precast gel (Invitrogen, Carlsbad, Calif.) as described in Example 3. The primary rabbit polyclonal antibody against human lysozyme was purchased from Dako A\/S (Denmark) and used at 1:5000. Lysozyme was quantified by a turbidimetric activity assay with (Sigma) on 96-well microtiter plate as described in Example 3. Briefly, 250 \u03bcl of 0.015% cell suspension was incubated with 10 \u03bcl of samples containing lysozyme with a concentration less than 2.4 \u03bcg\/ml. The reaction was followed by the kinetic mode in Microplate Manager (Bio-Rad, CA) for 5 min at 450 nm. The concentration of lysozyme was then determined in reference to the standard curve.","The stable expression level of human lysozyme (rHlys) reached at least about 0.6% rHlys per brown rice weight amounting to 45% of the total soluble protein extract from rice grain.  illustrates the seed specific expression of human lysozyme in transgenic plants. rHlys is only found in mature and germinated grain, but not in any other tissues tested.  shows the expression level of human lysozyme in powdered R3 seeds taken from transgenic rice plants","B. Generation of Human Lysozyme-expressing Transgenic Wheat Cells and Plants","Plasmid API159 () and API230 () were used to transform wheat cells substantially in the same manner as in transforming rice cells. Eight transgenic wheat lines were produced with API159, generating an expression level of about 150 to 300 \u03bcg of lysozyme per grain. Two transgenic wheat lines were produced with API230, yielding an expression level of about 50 to 120 \u03bcg of lysozyme per grain.","C. Generation of Human Lysozyme-expressing Transgenic Barley Cells and Plants","The plasmid API159 was also used to transform barley cells substantially as described as transformation of rice cells. Five transgenic barley lines were produced, yielding about 3.9 to 12.3 \u03bcg of lysozyme per grain.","D. Generation of Human Lactoferrin Expressing Transgenic Rice Cells and Plants","The synthetic human lactoferrin gene under the control of the Gt1 promoter in the pAPI164 plasmid was used to generate over 100 independent transformants by particle bombardment-mediated transformation.","Particle bombardment mediated transformation of rice was carried out as described above. At least 20 R1 grains from each R0 plant were analyzed for rHLF expression. Individual R1 grains were cut into halves. The endospermic half was subjected to rHLF expression analysis by Western blot or ELISA and the corresponding positive embryonic half was germinated to generate R1 seedlings. The seedlings were transplanted to generate R2 grains. During the screening of R1 grains we observed that all the positive grains were opaque-pinkish in color in comparison to negative or control grains. The opaque-pinkish color in rice grains was then used to identify homozygous lines. A transgenic plant was considered to be homozygous and expressing rHLF if all grains from that plant were opaque-pinkish. Homozygous lines were then confirmed by ELISA analysis. Based on the expression analysis and agronomic characters, selected homozygous R2 lines were advanced to R3 and R6 generations.","A. Southern Blot Analysis","About three grams of young leaves were collected and grounded with liquid nitrogen into a fine powder. The genomic DNA was isolated according to the procedure as described in Dellaporta et al., 1983, and purified by phenol-chloroform extraction. Approximately 5 \u03bcg of DNA was then with HindIII and EcoRI, separated on a 1% agarose gel, blotted onto a Hybond membrane (Amersham Pharmacia Biotech, Piscataway, N.J.). The blot was probed with gel purified human Hlys gene and developed by ECL\u2122 direct nucleic acid labeling and detection system (Amersham Pharmacia). By comparing to known amounts of the intact 1470 bp human lysozyme (Hlys) gene, the intact copy number of the transgenes, including promoter and Hlys gene, was estimated to vary from about 1 to about 6. No positive correlation between copy number of the rHlys transgene and amount of rHlys synthesized was discernible.","B. SDS-PAGE and Reverse IEF Gel Electrophoresis","Induced calli or harvested cells from suspension cell cultures were ground with cold phosphate buffered-saline (PBS) with a protease inhibitor cocktail (2 \u03bcg\/ml aprotonin, 0.5 \u03bcg\/ml leupeptin, 1 mM EDTA and 2 mM Pefabloc). The protease inhibitor cocktail was excluded from the buffer used subsequently during the purification of the enzyme, since the inhibitors did not increase the lysozyme expression yield. Grinding was conducted with a pre-chilled mortar and pestle at approximately 2 ml buffer\/g calli or cells. A clear homogenate was obtained by subjecting the resulting extract to centrifugation at 16,000\u00d7g for 10 minutes at 4 C.","SDS-PAGE was carried out using an 18% precast gel (Novex, CA). The resulting gel was stained with 0.1% Coomassie brilliant blue R-250 at 45% methanol and 10% glacial acetic acid for three hours. Gel destaining was conducted with 45% methanol and 10% glacial acetic acid until the desired background was reached.","Reverse IEF gel electrophoresis was carried out using a precast Novex pH 3-10 IEF gel according to the manufacturer's instructions (Novex, CA). About 30 \u03bcg of lysozyme was loaded onto the gel and electrophoresed at 100 V for 50 minutes followed by application of 200 V for 20 minutes. The gel was then fixed in 136 mM sulphosalicylic acid and 11.5% TCA for 30 minutes and stained in 0.1% Coomassie brilliant blue R-250, 40% ethanol, 10% glacial acetic acid for 30 minutes. The destaining solution contained 25% ethanol and 8% acetic acid.","C. Western Blot Analysis","A SDS-PAGE gel was electroblotted to a 0.45 \u03bcm nitrocellulose membrane using a Mini Trans-Blot Electrophoretic Transfer Cell (Bio-Rad, CA) and subsequently subjected to immuno-blotting analysis. The blot was blocked with 5% non-fat dry milk in PBS, pH 7.4 for at least two hours followed by three washes with PBS, pH 7.4 for 10 minutes each. The primary rabbit polyclonal antibody against human lysozyme (Dako A\/S, Denmark) was diluted at 1:2000 in the blocking buffer and the blot was incubated in the solution for at least one hour. The blot was then washed with PBS three times for 10 minutes each. The secondary goat anti-rabbit IgG (H+L)-alkaline phosphatase conjugate (Bio-Rad, CA) was diluted in the blocking buffer at 1:4000. The membrane was then incubated in the secondary antibody solution for one hour and then washed three times. Color development was initiated by adding the substrate system BCIP-NBT (Sigma) and the process was stopped by rinsing the blot with HO once the desirable intensity of the bands had been achieved.","D. Enzyme Linked Immunosorbant Assay (ELISA)","An indirect sandwich ELISA was developed to quantify total lysozyme expressed in rice calli or cells and used as an alternative assay to determine the lysozyme expression yield. A direct sandwich ELISA for lysozyme quantification has been previously reported (Lollike et al., 1995, Taylor, 1992), however an alternate assay was developed as a key reagent used in the assay is no longer commercially available.","In carrying out the assay, rabbit anti-human lysozyme antibody (Dako D\/K, Denmark) was used to coat a 96 well plate at 1:5000 diluted in PBS overnight at room temperature. After washing with PBS, the plate was blocked with 5% normal donkey serum (Jackson ImmunoResearch Laboratories, Pa) in PBS for one hour. The plate was washed again with PBS. Lysozyme samples were diluted in 0.05% Tween in PBS and captured by adding to the plate and incubating for one hour. After washing the plate with PBS, sheep anti-human lysozyme at 1:1000 diluted with 0.05% Tween in PBS was added and incubated for one hour. The plate was washed again with PBS. Peroxidase-conjugated affinipure donkey anti-sheep IgG (H+L) diluted in 0.05% Tween in PBS at 1:10,000 was added and incubated for one hour. After a final wash of the plate with PBS, color was developed by incubating the plate with TMB substrate (Sigma, Mo) for 5-15 minutes and the absorbance read at 655 nm.","E. Enzymatic Activity Assay for Lysozyme","A reliable and quantitative method was developed to analyze the expression level of enzymatically active lysozyme. The turbidimetric assay was developed using a 96-well microtiter plate format and based on the standard lysozyme assay that is carried out spectrophotometrically in cuvettes. A microtiter plate based method previously described for the detection of lysozyme release from human neutrophils had a detection range of 1-100 ng\/ml (Moreira-Ludewig et al., 1992). The assay conditions were modified to maintain the linearity of detection up to 3.0 \u03bcg\/ml.","The enzymatic activity of lysozyme was routinely determined by spectrophotometric monitoring of the decrease in turbidity at 450 nm of a suspension of () cells (Shugar, 1952). Specifically, 250 \u03bcl of a 0.015% (w\/v) cell suspension was prepared in 66 mM potassium phosphate, pH 6.24 (buffer A). Cell suspensions were equilibrated at room temperature and the reaction was initiated by adding 10 \u03bcl samples containing lysozyme with concentrations from 0 to 2.4 \u03bcg\/ml. Lysozyme activity was determined in a kinetic mode for 5 minutes at 450 nm. The concentration of lysozyme was then calculated by reference to the standard curve constructed with human milk-derived lysozyme.","The enzymatic activity of human milk lysozyme and the rice cell derived lysozyme of the invention was compared. As shown in , the lysozyme effected reduction of the turbidity of cell suspensions at 450 nm was very similar for lysozyme from the two sources, while buffer alone did not have any effect on the reduction of turbidity.","Three selected suspension cell culture lines were induced to express lysozyme and the yield estimated in parallel by ELISA and the enzymatic activity assay described above (Table 2). T-test analysis showed that there was no significant difference between the lysozyme concentration measured by ELISA and enzymatic activity assay (p<0.05). These results demonstrate that active recombinant human milk lysozyme is synthesized and maintained in rice callus cells and can be isolated without losing its activity.",{"@attributes":{"id":"p-0445","num":"0444"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Comparison of Lysozyme Yields Estimates by Enzymatic Activity"},{"entry":"Assay and ELISA"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Lysozyme yield by enzymatic","Lysozyme yield by ELISA"]},{"entry":[{},"activity assay (lysozyme\/","(lysozyme\/total protein"]},{"entry":["Cell line","total protein \u03bcg\/mg)","\u03bcg\/mg)"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["156-5\u2002","25.8 +\/\u2212 6.3","30.3 +\/\u2212 3.9"]},{"entry":["156-16","32.1 +\/\u2212 5.7","32.9 +\/\u2212 3.2"]},{"entry":["156-31","47.0 +\/\u2212 6.2","42.3 +\/\u2212 7.0"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"F. Recombinant Human Lysozyme has Bactericidal Function","The sensitive lysis of cells in a turbidimetric assay () indicates that recombinant human lysozyme possesses enzymatic activity and functions as a bactericide. To confirm this with a gram-negative bacterium, a bactericidal assay was carried out using an strain (JM109) as a test organism (FIG. ).","In carrying out the assay, an aliquot of overnight JM109 culture was grown in LB medium until mid log phase. A standard innoculum of mid-log phase JM109 at 2\u00d710CFU (colony forming units)\/ml was used in the bactericidal assay. Buffer (20 mM Sodium phosphate, pH 7.0, 0.5 mM EDTA) alone, buffer containing human milk lysozyme or rice seed derived lysozyme at about 30 \u03bcg\/ml were sterilized by filtration. The mixture of cells and lysozyme solution was then incubated at 37\u00b0 C. for the specified length of time. One-fifth of the mixture volume was plated onto the LB agar plates and incubated overnight at 37 C in order to determine the number of colony forming units. At the concentration of 30 \u03bcg\/ml, recombinant human lysozyme exhibited a similar bactericidal effect as lysozyme from human milk. There was no reduction of colony forming units using an extract from the non-transgenic control.","G. Purification of Lysozyme from Rice Calli, Suspension Cultures and Transgenic Rice Grains","Five rice calli lines expressing high levels of lysozyme were propagated and induced by sucrose starvation. The calli or cells were ground by a Tissuemizer in extraction buffer (PBS, 0.35 M NaCl) at 2 ml buffer\/g of wet calli. The resulting tissue homogenate was centrifuged at 25,000\u00d7g for 30 minutes at 4 C. The supernatant was removed and subjected to filtration through a pre-filter and then through a 0.45 \u03bcm nitrocellulose filter.","Approximately 1 liter of filtered supernatant from 500 grams of induced wet calli were then dialyzed against 50 mM sodium phosphate, pH 8.5 at 4 C overnight. The supernatant was loaded onto a 200 ml SP Sepharose fast flow column (XK26\/40, Pharmacia) equilibrated with the loading buffer (50 mM sodium phosphate, pH 8.5) at a flow rate of four ml\/min. The column was then washed with the same buffer until a baseline of A280 was achieved. Lysozyme was eluted by 0.2 M NaCl in the loading buffer and fractions containing lysozyme activity were pooled, concentrated and reapplied to a Sephacryl-100 column equilibrated and run with PBS at a flow rate of one ml\/min. Proteins were eluted and separated by using PBS at a flow rate of one ml\/min.","Pure lysozyme fractions were identified by activity assay and total protein assay (Bradford) and the purity of lysozyme was confirmed by SDS-PAGE.","The five lines with the highest lysozyme expression level were selected and propagated continuously in petri dishes or shake flasks for lysozyme isolation and purification. A crude extract from rice callus contains both recombinant human lysozyme and large amounts of native rice proteins. Since the calculated pl of lysozyme is approximately 11, a strong cation exchange column, SP-Sepharose fast flow (Pharmacia), was chosen as the first column to separate the rice proteins from recombinant human lysozyme. Most of the rice proteins did not bind to the column when equilibrated with 50 mM sodium phosphate, pH 8.5. The recombinant human lysozyme, on the other hand, bound to the column and was eluted by 0.2 M NaCl. Rice proteins that co-eluted with recombinant human lysozyme, were separated from lysozyme by gel filtration through a Sephacryl S-100 column and highly purified recombinant human lysozyme was obtained.","To purify human lysozyme from rice grains, Rrice seeds from transgenic plants were dehusked and milled to flour using conventional methods. Lysozyme was extracted by mixing the rice flour with 0.35 N NaCl in PBS at 100 grams\/liter at room temperature for one hour. The resulting mixture was subjected to filtration through 3 \u03bcm of a pleated capsule, then through 1.2 \u03bcm of a serum capsule and finally through a Suporcap 50 capsule with a 0.8 \u03bcm glass filter on top of 0.45 \u03bcm filter (Pall, Mich.).","The clear rice extract (1 liter) was then dialyzed against 50 mm sodium phosphate, pH 8.5 at 4\u00b0 C. overnight and the dialyzed sample was loaded onto a cation exchange resin SP-Sepharose (Pharmacia Amersham), which was pre-conditioned with 50 mm sodium phosphate, pH 8.5 before loading. After loading, the column was washed with the same buffer until a base line A280 reading was achieved, then lysozyme was eluted with 0.2 N NaCl in 50 mm sodium phosphate, pH 8.5. Fractions containing lysozyme were pooled and reapplied to a Sephacryl S-100 column (Bio-Rad; equilibrated and run with PBS). Pure lysozyme was fractions were identified by enzymatic assay and total protein assay (Bradford). Finally the purity of lysozyme was confirmed by SDS-PAGE.","H. Attributes of Recombinant Human Lysozyme Produced in Rice","(i). N-terminal Amino Acid Sequencing","Recombinant human lysozyme (rLys) isolated from rice cells as described above, was separated by 18% SDS-PAGE followed by electroblotting to a PVDF membrane (Bio-Rad, CA). The lysozyme band was identified by staining the membrane with 0.1% Coomassie Brilliant Blue R-250 in 40% methanol and 1% glacial acetic acid for 1 minute. The stained PVDF membrane was immediately destained in 50% methanol until the band was clearly visible. After the blot was thoroughly washed with HO and air-dried, it was sequenced with a sequencer ABI 477 by Edman degradation chemistry at the Protein Structure Laboratory of the University of California at Davis. The results showed that the rLys produced in transgenic rice seed had an identical N-terminal sequences to the human lysozyme, as follows:\n\n","The blank parenthesis in recombinant lysozyme represents residue Cys which cannot be detected by the machine. This cycle was not defined, and could be due to the un-modified cysteine residue which cannot form a stable derivative in Edman degradation analysis.","Additionally, a number of structural and functional attributes of human lysozyme and recombinant lysozyme produced in rice were found to be the same, including molecular weight, pl, bactericidal effect with , thermal and pH stability and specific activity.","(ii). Thermal and pH Stability of Lysozyme","For biotechnological applications of the recombinant human lysozyme, its thermal and pH stability as well as its resistance to proteases is of decisive importance. A human lysozyme standard and lysozyme from rice were diluted to a final concentration of 50 \u03bcg\/ml in PBS and subjected to the following thermal treatment in a sequential mode: (1): 62\u00b0 C. for 15 minutes; (2): 72\u00b0 C. for 20 seconds; (3): 85\u00b0 C. for 3 minutes and finally; (4): 100\u00b0 C. for about 8 to about 20 seconds. Studies were conducted with 100 \u03bcl per tube and repeated three times. Aliquots were saved at the end of each treatment and the remaining lysozyme activity was measured by activity assay. The result showed that recombinant lysozyme exhibited the same degree of thermal stability in the temperature range from 62\u00b0 C. to 100\u00b0 C. as human lysozyme.","In another embodiment, approximately 50 \u03bcl of Hlys or rHlys was dissolved in PBS at 100 \u03bcg\/ml and subjected to heat treatment. Four different temperatures of 65\u00b0 C. (FIG. A), 72\u00b0 C. (FIG. B), 85\u00b0 C. () and 100\u00b0 C. () were tested. With each temperature, 0 min, 0.33 min, 1.5 min, 3 min, 5 min and 15 min were selected to analyze the impact of incubation time on the stability of lysozyme.","For studies on pH stability, lysozyme was dissolved in 0.9% NaCl at 100 \u03bcg\/ml at pH 10, 9, 7.4, 5, 4, and 2. The solutions were incubated at 24\u00b0 C. for one hour. Experiments were conducted with 200 \u03bcl per tube and repeated three times. Remaining lysozyme was detected by lysozyme activity assay.","For pH treatments at pH 2, 4 and 5, Hlys and rHlys was dissolved in PBS adjusted to the corresponding pHs with HCl at 100 \u03bcg\/ml. For pH 9 and 10, lysozyme was dissolved in TBS and 150 mM sodium carbonate\/bicarbonate at 100 \u03bcg\/ml, respectively. Approximately 100 \u03bcl of lysozyme solution was incubated at 37\u00b0 C. for 30 min. The lysozyme activity was assessed by activity assay (FIG. E).","Both Hlys and rHlys displayed similar thermal and pH stability.","(iii). Determination of In Vitro Protease Resistance of Lysozyme","Lysozyme was dissolved in 0.9% NaCl at 100 \u03bcg\/ml. The pH of the solution was reduced to 3, 4 and 5 with HCl. Pepsin (Sigma, MO) (pepsin: lysozyme=1:22 (w\/w)) was added and the solutions were incubated at 37\u00b0 C. for one hour. Then the pH of all treatments was raised to pH 7 with bicarbonate. Pancreatin (Sigma, MO) (pancreatin: lysozyme=1:110 (w\/w)) was added to the neutral solution and incubated at 37 C for two hours. The remaining lysozyme activity was measured by activity assay.","In in vitro digestion experiments with pepsin and pancreatin, the native and recombinant human lysozyme displayed very similar resistance to pepsin and pancraetin digestion. Under these conditions, human albumin was degraded as demonstrated by SDS-PAGE (data not shown).","(iv). Biochemical Characterization of Lysozyme","After recombinant human lysozyme was purified to near homogeneity, several biochemical characterizations were carried out to compare human milk lysozyme with recombinant human milk lysozyme derived from rice cells. The results summarized in Table 3 show that by SDS-PAGE, native human milk lysozyme and recombinant lysozyme migrated to the same position.","Nucleotides encoding the rice Ramy3D signal peptide were attached to the human lysozyme gene in the expression vector pAPI156. Determination of the N-terminal amino acid sequence of the purified recombinant human lysozyme revealed an N-terminal sequence identical with that of native human lysozyme, as detailed above. Rice cells thus cleave the correct peptide bond to remove the RAmy3D signal peptide, when it is attached in the human lysozyme precursor.","The overall charge of recombinant and native human lysozyme were compared by isoelectric-focusing (IEF) gel electrophoresis and pl values determined. Since lysozyme is a basic protein with a calculated pl of 10.20, the pl comparison studies were carried out by reverse IEF gel electrophoresis. Recombinant and native human lysozyme displayed identical pl, indicating the same overall charge (data not shown).","Recombinant human lysozyme derived from transgenic rice had a specific activity similar to the native lysozyme (200,000 units\/mg (Sigma, MO), whereas, lysozyme from chicken egg whites had the expected 3-4 fold lower specific activity (Sigma, MO) (FIG. ).",{"@attributes":{"id":"p-0475","num":"0476"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Comparison of Biochemical Characteristics of Human Milk"},{"entry":"Lysozyme and Recombinant Lysozyme"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},"Glyco-","Specific",{}]},{"entry":["Lysozyme","N-terminal","Size","syla-","activity"]},{"entry":["source","sequence","(kDa)","tion","(units\/mg)","pl"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}},{"entry":["Human milk","KVFER C ELART","14","No","201,526","10.2"]},{"entry":["rice","KVFER(\u2212)*ELART","14","No","198,000","10.2"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}},{"entry":"*This cycle was not defined, and could be due to the un-modified cysteine residue which cannot form a stable derivative in Edman degradation analysis. "}]}}]}}},"The results described above demonstrate the ability to use rice cells as a production system to express human lysozyme from milk. Over 160 individual transformants were screened by immunoblot, enzymatic activity assay and ELISA. Yields of recombinant human milk lysozyme reached 4% of soluble cell proteins in culture cells and over 40% of soluble proteins in rice grains. Although the mechanism is not part of the invention, the high expression level may be explained by the utilization of the strong RAmy3D promoter (Huang et al., 1993) in culture cell system and Gt1 promoter in grain expression system and the codon-optimized gene.","The plant derived human milk lysozyme obtained by the methods of the present invention was identical to endogenous human lysozyme in electrophoretic mobility, molecular weight, overall surface charges and specific bactericidal activity.","A. Southern Blot Analysis","About three grams of young leaf were collected and ground with liquid nitrogen into a very fine powder. The DNA was isolated according to the procedure as described in Dellaporta et al., 1983, and purified by phenol-chloroform extraction. Approximately 5 \u03bcg of ECoRI and HindIII digested DNA from each line was used to make blot for Southern analysis. The ECLTM direct nucleic acid labeling and detection system (Amersham, USA) was used for analysis.","The lactoferrin gene copy number was estimated to be from about 1 to about 10 as determined by Southern blot hybridization using EcoRI and HindIII digested genomic DNA. The API164-12-1 (R0) transgenic plant line was subjected to Southern analysis together with ten Western blot positive, field grown R1 lines. A typical Southern blot shows that there are at least three fragments above the original plasmid derived plant transformation unit (3156 bp). All the LF inserts appear to be inherited from the original R0 transgenic plant event to R5 generation.","B. Protein Isolation and Western Blot","Rice seeds were ground with 1 ml of 0.35 N NaCl in phosphate buffer saline (PBS), pH 7.4 using an ice-cold mortar and pestle and the resulting homogenate was centrifuged at 15000 rpm for 15 min at 4\u00b0 C. The supernatant was used as a protein extract and about 1\/25 or 1\/50 of the salt soluble content was loaded onto a 10% pre cast gel (Novex, USA) and electrophoresis was carried according to the manufacturer's instructions. For total protein detection, the polyacrylamide gel was stained with 0.1% Coomassie brilliant blue R-250 (dissolved in 45% methanol and 10% glacial acetic acid) for at least three hours and destained with 45% methanol and 10% glacial acetic acid until the desired background was achieved.","For Western blot analysis, SDS-PAGE gels were electroblotted onto a 0.45 \u03bcm nitrocellulose membrane with a Mini-Trans-Blot Electrophoretic Transfer Cell System (Bio-Rad, USA) and subsequently subjected to immuno-blotting analysis. The blot was blocked with 5% non-fat dry milk in PBS for at least two hours followed by three washes with PBS for 10 minutes each. The primary rabbit polyclonal antibody against hLF (Daka A\/S, Denmark) was diluted at 1:2500 in the blocking buffer and the blot was incubated in the solution for one hour. The blot was washed with PBS for three times with 10 minutes each. The secondary goat anti-rabbit IgG (H+L)-alkaline phosphatase conjugated (Bio-Rad, USA) was diluted in the blocking buffer at 1:5000 ratio. The membrane was incubated in the secondary antibody solution for one hour and followed by three washes with PBS. Color development was initiated by adding the substrate system BCIP-NBT (Sigma, USA) and the process was stopped by rinsing the blot with H2O once the desirable intensity of the bands was achieved.","One hundred eight (108) R0 plants were grown to maturity, seeds were harvested from 56 fertile plants and individual seeds analyzed by Western blot to detect the expression of rLF. Coomassie blue staining was carried out to compare the mobility of rLF with native human lactoferrin (hLF) (FIG. ), with 40 \u03bcg of total protein loaded onto each lane, along with 40 ng of native purified hLF per lane as the positive control.","Estimation of total rLF by ELISA indicated that from 93 \u03bcg to 130 \u03bcg rLF was expressed in transformed rice seeds. A typical Western blot analysis () illustrates that both rLF and native hLF migrate at approximately the same rate with the molecular weight about 80 kDa, consistent with that determined by other researchers (Wang et al., 1984).","C. Protein Purification","Rice seeds from R2 homozygous generation were dehusked and milled to flour conventionally. Recombinant lactoferrin was extracted by mixing the rice flour with 0.35 N NaCl in PBS at 100 g\/l at room temperature for two hours. The resulting mixture was centrifuged at 15,000 rpm for one hour at 4\u00b0 C. The collected supernatant was subjected to the following steps of filtration before loading onto a Sepharose column. First, the supernatant was run through a few layers of cheesecloth. Then the filtrate was passed sequentially through an 8\u03bcm paper, 1 \u03bcm paper and a 0.25\u03bcm nitrocellulose membrane. The clear protein solution was loaded onto a ConA Sepharose column (Pharmacia, XK 26) which had been equilibrated with 0.5 N NaCl in 20 mM Tris, pH 7.4 (binding buffer) at a flow rate at 4 ml\/min. After the loading was complete, the column was washed with binding buffer until the baseline at A280 nm was achieved. Lactoferrin was eluted with 0.1N mannoside in the binding buffer. Fractions containing lactoferrin were pooled and loaded onto a second column SP-Sepharose (Bio-Rad, USA) which has been equilibrated with 0.4 N NaCl in 50 mM sodium phosphate, pH 8.0 (binding buffer B) at the flow rate 4 ml\/min. Then the column was washed with the binding buffer B until the baseline at A280 nm was obtained. Lactoferrin was eluted by 1 N NaCl in 50 mM sodium phosphate, pH 8.0 and the fractions containing LF were pooled and dialyzed against PBS. Finally the purity of LF was assessed by SDS-PAGE and stored at \u221280\u00b0 C.","In another embodiment, recombinant human lactoferrin (rHLF) was extracted by mixing rice flour with 0.35 M NaCl in PBS at 75 g\/L at room temperature for 2.5 hours. The extract was passed through six layers of cheesecloth before centrifugation (10,000 g for 1 hour at 4\u00b0 C.). The supernatant was recovered and the NaCl concentration was adjusted to 0.4 M (pH 8.0). After a second centrifugation at 10,000 g for 10 minutes at 4\u00b0 C., the supernatant was collected and filtered through 0.45 \u03bcm nitrocellulose membrane. The filtrate was loaded onto a SP-Sepharose column (Bio-Rad, Hercules, Calif.) which had been equilibrated with 0.4 M NaCl in 50 mM sodium phosphate, pH 8.0 (binding buffer) at a flow rate of 4 ml\/min. The column was washed with the binding buffer until baseline A280 was obtained. Lactoferrin was eluted by a linear gradient and dialyzed against PBS. The purified rHLF was analyzed by SDS-PAGE and stored at \u221280\u00b0 C.","D. Enzyme Linked Immunosorbant Assay (ELISA)","ELISA was conducted using seed extracts, isolated as described above, with total protein assayed using the Bradford method (Bradford, M., 1976). The ELISA was based on a typical sandwich format generally known in the art. Briefly, 96 well plates were coated with rabbit anti-human lactoferrin antibody (Daka A\/S, Denmark), then rLF and control samples were added to individual wells of the plate and incubated for 1 hour at 35\u00b0 C. Rabbit anti-human lactoferrin horseradish peroxidase conjugate (Biodesign, USA) was then added to each well and incubated for 1 hour at 35\u00b0 C., followed by addition of the tetramethylbenzidine substrate (Sigma, USA) and incubation for 3 minutes at room temperature. The reaction was stopped by adding 1N H2SO4 to each well. The plates were read at dual wavelengths of 450 and 650 nm in a Microplate Reader (Bio-Rad, model 3550) and the data was processed by using Microplate Manager III (Bio-Rad). The results of an analysis of 10 homozygous selected lines showed that from 93 \u03bcg to 130 \u03bcg rLF was expressed per seed.","E. Selection of Plants for Advance Generations","At least 20-40 seeds from 11 independent lines were analyzed. Individual R1 seeds were cut into half and endospermic halves were subjected to analysis by Western blot with the positive corresponding embryonic halves germinated on 3% sucrose medium with 0.7% agar. The seedlings were transplanted to the field for R1 generation. Out of 11 individual lines, 3 lines were expressed. A total of 38 plants were grown in the field derived from the 3 expressed mother lines. Based on the agronomic character (Table 4) of those 38 plants, 28 plants were selected.","It was observed that all the Western positive R1 seeds were opaque to pinkish in color in comparison to control seeds, so this criterion was applied in screening the R2 seeds. Mature R2 seeds were harvested at maturity and dehusked. The pinkish R2 seeds were confirmed by Western dot blot and ELISA as expressing rLF (data not presented). Finally 10 homozygous R2 lines were selected and grown in the field in order to advance the generation.",{"@attributes":{"id":"p-0494","num":"0495"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 4"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Comparison Of Phenotypic Characteristics Of Native TP-309 And"},{"entry":"Transformed TP-309 Rice Seeds"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Effective",{},"1000 seed",{}]},{"entry":["Source","tiller","Blank grain","weight (g)","\u03bcg of rLF\/seed"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["TP-309","43","5.0","25",{}]},{"entry":["Homozygous","42","19.7","20.2","125"]},{"entry":"transgenic lines"},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},"During R2 and R3 generation the percentage of blank seeds was higher in homozygous transgenic lines than in the non-transgenic control. This affected the 1000 seed weight. However, in the R4 generation no significant differences in phenotypic character were observed in homozygous transgenic lines when compared to non-transformed TP309 (Table 4).","F. Attributes of Recombinant Human Lactoferrin Produced in Rice","Physical characterization of the rLF showed there was no significant difference between the rLF and a commercially available purified form of hLF based on N-terminal amino acid sequencing, and physical characteristics of rLF such as molecular weight as determined by MALDI-MS, HPLC profile of which showed a comparable peptide map, pH dependent iron release and bacteriostatic activity, using the analyses described below.","(i). N-terminal Amino Acid Sequencing","Purified rLF from rice seeds was resolved by 10% SDS-PAGE, followed by electroblotting to PVDF membrane (Bio-Rad, USA). The target band was identified by staining the membrane with 0.1% Coomassie brilliant blue R-250 in 40% methanol and 1% glacial acetic acid for 1 minute. The stained PVDF membrane was immediately destained in 50% methanol until the band is clearly visible. The blot was thoroughly washed with ddH2O and air dried. Finally this sample was sent to the Protein Structure Laboratory in University of California at Davis (CA, USA) for sequencing analysis.","(ii). Detection of Glycosylation and Determination of Sugar Content","Glycosylation of the recombinant human lactoferrin produced in rice was analyzed by an immunoblot kit for glycoprotein detection (Bio-Rad, USA) per instructions from the manufacturer. An increase of molecular weight of lactoferrin due to carbohydrate content was determined by Matrix Assisted Laser Desorption Ionization-Mass spectrometry (MALDI-MS) (PE Applied Biosystems, Voyager System).","Recombinant lactoferrin produced in rice is glycosylated as evident from the binding to Con A resin, the positive staining by glycoprotein detection kit as well as the larger detected mass as compared to the calculated mass (76.2 kDa) based on the peptide backbone. MALDI-MS showed that seed derived recombinant lactoferrin has molecular weight of 78.5 kD while human milk lactoferrin is 80.6 kDa (Table 5). The difference could be due to the lesser degree of glycosylation in the rice seed-derived lactoferrin. Analysis shows that the purified rHLF contains xylose but lacks sialic acid, which is consistent with plant post-translational modification patterns (Matsumoto et al., 1995).","(iii). Determination of Isoelectric Point of Lactoferrin","Reverse isoelectric focusing (IEF) gel electrophoresis was carried out with a precast Novex IEF gel, pH 3-10 according to the manufacturer' instruction. About 30 \u03bcg of purified rLF was loaded and the running condition was 100 V for 50 minutes and 200 V for 20 minutes. The gel was then fixed in 136 mM sulphosalicylic acid and 11.5% TCA for 30 minutes, stained in 0.1% Coomassie brilliant blue R-250, 40% ethanol, 10% glacial acetic acid for 30 minutes and destained in a solution containing 25% ethanol and 8% acetic acid.","(iv). Comparison of Physical Characteristics of rLF with Native hLF","The HPLC profile of native and rLF showed a comparable peptide map. This confirmed that LF from the two sources have an identical amino acid sequence (data not presented). Additional comparisons confirm that human lactoferrrin produced in transgenic rice closely resembles native human lactoferrin, as evidenced by (1) the N-terminal sequence of purified rLF from homozygous R2 seeds and hLF (Dakao A\/S, Denmark), which were shown to be identical (Table 5); (2) the isoelectric point (pl) of native and rice seed derived LF which is the same, indicating that they have similar surface charges (Table 5); (3) the pH dependent iron release of rLF which was shown to be closely related to that of native hLF (FIG.  and see section vii of example 4); and (4) the bacteriostatic activity of rHLf which was shown to be similar to that of native human lactoferrin (nHLf) on enteropathogenic (EPEC; ) and confirmed the presence of active recombinant LF in extracts derived from transformed rice seeds (see section ix of Example 4).",{"@attributes":{"id":"p-0507","num":"0508"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"266pt","align":"center"}},"thead":{"row":{"entry":"TABLE 5"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Physical characterization data for human (hLF) and rice seed derived"},{"entry":"recombinant lactoferrin (rLF)"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["LF","Size",{},{},"Glycosy-","Sugar"]},{"entry":["source","(kDa)","N-terminal sequence","pl","lated","content (%)"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}},{"entry":["hLF","80.6","GlyArgArgArgArgSerValGlnTrpCysAla","8.2","YES","5.5"]},{"entry":["rLF","78.5","GlyArgArgArgArgSerValGlnTrp( )Ala","8.2","YES","2.9"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]}}]}}},"(v). Iron Content and Nutrient Value Determination of Rice Seeds","The iron content of R2 homozygous seeds was determined. Two grams of dry mature seeds from each transformed and non-transformed line were weighed and wet-ashed with HNO3 and H2O2 solution at 110\u00b0 C. (Goto et al., 1999). The ash was dissolved in 1N HCl solution. The iron content was then measured by absorbance of Fe\u2014O-phenanthrolin at 510 nm, using a Sigma kit (Sigma, USA) per instructions of manufacturer.","The different values of nutrient facts of homozygous transgenic seeds and non transgenic seeds were measured by standard procedure at A & L Western Agricultural Laboratories (Modesto, Calif., USA).","A comparative analysis of transgenic Iactoferrrin-expressing rice seeds with non transformed native Teipei-309 showed that there is no significant difference between transformed and non transformed seeds in nutrient value with the exception that the concentration of iron is 50% greater (Table 6). The increased level of iron may be the reason for the opaqueness and pink coloration of the rLF expressing transgenic rice seeds.","In another embodiment, 0.2 grams of dried, dehusked grains expressing rHLF were wet-ashed with concentrated HNO3 for two days and dissolved in 5 ml of DDI H2O. The iron contents of the samples were measured by flame atomic absorption spectrophotometry (Thermo Jarrel Ash SH4000, Franklin, Mass.). NIST liver was analyzed concurrently to verify the accuracy of the standard curve.","The iron content of transgenic rice grains was more than twice that of non-transformed TP309 grains, while there were no significant differences in other tested nutrition factors between transformed and non-transformed grains (Table 7). This suggests that groups ingesting transgenic rice with rHLF will increase the iron intake.","The transgenic grains with increased iron content were opaque-pinkish in color. The opaque-pinkish color was observed inside as well as outside the rice endosperm. This opaque-pinkish color, segregated in Mendelian fashion, was linked with expression of rHLF and was inherited through the R4 generation.","There was no difference noticed during the seed germination of transgenic seeds, the phenotype of R2 R3 and R4 plants was vigorous and the seed yield was similar to that of non-transgenic Teipei-309 plants (data not shown).",{"@attributes":{"id":"p-0516","num":"0517"},"tables":{"@attributes":{"id":"TABLE-US-00006","num":"00006"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"center"}},"thead":{"row":{"entry":"TABLE 6"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Comparison of Nutrition Value (in mg) Per 100 Gram of Non Transformed and"},{"entry":"Transformed Rice Seeds"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"10"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"10","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Carbohy-",{},{},{},{},{},{},{},{}]},{"entry":["Source","drate","Protein","Fat","Ca","K","Na","Fe","Water","Calories"]},{"entry":{"@attributes":{"namest":"1","nameend":"10","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"10"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"14pt","align":"char","char":"."}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"8","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"9","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"10","colwidth":"28pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["TP-309","76.0","8.7","2.4","9","370","<10","0.8","11.3","369"]},{"entry":["Homozygous","75.7","8.7","2.2","8","330","<10","1.2","11.8","367"]},{"entry":"transformed"},{"entry":"lines"},{"entry":{"@attributes":{"namest":"1","nameend":"10","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0517","num":"0518"},"tables":{"@attributes":{"id":"TABLE-US-00007","num":"00007"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 7"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Comparison of Mineral Contents (in \u03bcg) Per Gram of rHLF-"},{"entry":"Transformed and Non-Transformed Rice Grains"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Source","Cu","Fe","Mn","Zn"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"5","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Non-transformed","2.9","8.7","33.1","20.8"]},{"entry":[{},"Transformed","4.7","19.2","17.7","28.7"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"5","align":"center","rowsep":"1"}}]}]}}]}}},"(vi). Tissue Specificity and Stability of rLF","An endosperm specific rice glutelin promoter was used to express recombinant lactoferrin in maturing or matured seeds. To confirm the tissue specificity of the expressed lactoferrin, protein was extracted from root, shoot, leaf beside mature seed and subjected to Western blot and the results indicated that there was no detectable expression of rLF except in the seed\/endosperm (FIG. ). Furthermore, the presence of rLF in 5 day old germinated seeds showed the stability of stored rLF within the plant cell during germination.","(vii). Iron Saturation and pH Dependent Iron Release","Lactoferrin was incubated with 2M excess ferric iron (FeCl3 :NTA=1:4) and sodium bicarbonate (Fe:HCO3-=1:1) for 2h at room temperature. Excess free iron was removed by using a PD-10 desalting column (Pharmacia, USA) and the iron saturation level was determined by the A280\/A456 ratio. Both native hLF and rLF were completely saturated by iron. Holo hLF was incubated in buffers with a pH between 2 and 7.4, at room temperature for 24 h. Free iron released from hLF was removed and the iron saturation level was determined by A280\/A456 ratio.","The results showed that iron release was similar for both hLF and rLF. Iron release began around pH 4 and was completed around pH 2 (FIG. ). The iron binding was reversible since iron-desaturated rLF was re-saturated by raising the pH to 7 (data not shown). The similarity in pH dependent iron release of rLF to that of the hLF standard demonstrated that rLF is able to adapt the appropriate tertiary structure for proper iron binding and release (Salmon, Legrand et al. 1997).","(viii). Binding and Uptake by Caco-2 Cells","50,000 Caco-2 cells\/well were seeded and grown in Minimum Essential Medium (GIBCO, Rockville, Md.) containing 10% fetal bovine serum in 24 or48 well tissue culture plates for 3 weeks. For binding studies, Caco-2 cells were incubated with varying concentrations (0-2 \u03bcM) of 125I-HLf in the presence or absence of 100-fold excess of unlabeled nHLf for 2 hours at 4 oC and cells were washed 5 times with ice-cold PBS. Cells were solubilized with 0.5 ml of 0.1% SDS and radioactivity was quantified in a gamma counter. For uptake studies, 0.4 \u03bcM of 125I-HLf was incubated with Caco-2 cells for 0 to 24 hours at 37\u00b0 C. and cells were washed, dissociated by the same way as in the binding study. 0.5 ml of 24% TCA solution was added to the dissociated cells and free iodine was removed by the centrifugation. Free and protein-bound I were quantified separately to evaluate how much of HLf was degraded in the cells. Receptor-binding of rHLf to the human intestinal Caco-2 cell line was saturable and specific, indicating that rHLf bound to the Lf receptor. The binding constant was similar for rHLf and nHLf, but the number of binding sites was slightly higher for rHLf, which may be due to the difference in glycosylation. Uptake of HLf by Caco-2 cells was identical for rHLf and nHLf.","(ix). In Vitro Digestion: Effect on Antimicrobial Activity and Binding\/Uptake to Caco-2 Cells","Lactoferrin is known to inhibit the growth of a variety of bacterial species based on its iron chelation and direct bactericidal properties. The anti-microbial effect of rLF extracted from rice seeds was tested following treatment using an in vitro digestion model with an enzymatic system containing pepsin (an enzyme active in stomach) and pancreatin (an enzyme active in deodenum).","LF proteins were dissolved in PBS at 1 mg\/ml, and either left untreated, pepsin treated (0.08mg\/ml at 37\u00b0 C. for 30 min), or pepsin\/pancreatin treated (0.016 mg\/ml at 37\u00b0 C. for 30 min). LF proteins were sterilized by passing through a membrane filter with a pore size of 0.2 \u03bcm [Rudloff, 1992]. The filter sterilized LF (0.5 \u03bcg\/ml) was incubated with 104 colony forming unit (CFU) enteropathogenic (EPEC)\/\u03bcl in 100 \u03bcl sterile synthetic broth (1.7% : AOAC) containing 0.1% dextrose and 0.4 ppm ferrous sulfate at 37\u00b0 C. for 12 h and colony forming units (CFU) were determined.","Starting with an enteropathogenic (EPEC) concentration of 10CFU (colony forming units), the untreated samples of rLF reached up to 10CFU after 12 h of incubation at 37\u00b0 C. in comparison to hLF, which produced up to 10CFU. An in vitro digestion model using an enzymatic system containing pepsin (enzyme active in stomach) and pancreatin (enzyme active in deodenum) with moderate shaking to imitate the transit of protein through infant gut [Rudloff, 1992] was used. rLf and nHLf were treated with active pepsin and pancreatic enzymes and exposed to 10CFU EPEC cells for 12 h at 37\u00b0 C. (FIG. ). Both the native human lactoferrin standard (nHLf) and the recombinant rice-derived lactoferrin (rLf) remained active in inhibiting growth of enteropathogenic , indicating that both nHLf and rHLf are resistant to protease digestion.","SDS-PAGE and ELISA revealed that nHLf and rHLf resist digestion by pepsin (at pH 3.8) and pancreatin, whereas human serum albumin is completely digested after in vitro digestion. Western blots revealed that immunoreactivity was also maintained after digestion. Although some smaller molecules were generated during digestion of HLf, most of the immunologically detectable HLf retained its intact size. More than 50% of rHLf and nHLf was immunologically detectable by ELISA, but I-HLf was around 40% and Fe-HLf was only 20% detectable, indicating that ELISA detects small peptide fragments of HLf, which are removed by the PD-10 column and that about 50-60% of Fe was released from detectable HLf after in vitro digestion. The iron-holding capacity was not significantly different.","The dissociation constant (Kd) and the number of binding sites for HLf to its receptor were determined from the binding study. Both Kd and the number of bindings sites were not significantly different between nHLf and rHLf after in vitro digestion (, B). Digestion did not appear to affect on the Kd but made the number of binding sites much lower. Total Lf uptake was not significantly different between nHLf and rHLf after in vitro digestion (FIG. C), though uptake was about one third when compared with undigested nHLf. Total iron uptake from nHLf was twice as high as that from rHLf. Percent degradation of HLf was similar regardless of digestion or not, and the native or recombinant form (FIG. D).","(x). Thermal Stability: Effect on Antimicrobial Activity and Binding\/Uptake to Caco-2 Cells","1.0 mg\/ml of holo-HLf in PBS was treated by the following conditions: (a) 62\u00b0 C. for 15 minutes, (b) 72\u00b0 C. for 20 seconds, (c) 85\u00b0 C. for 3 minutes, or (d) 100 oC for 8 seconds. Survival ratio of HLf determined by ELISA were more than 90% following treatment at 62\u00b0 C. for 15 minutes, at 72\u00b0 C. for 20 seconds, or at 85\u00b0 C. for 3 minutes, but it was considerably lower after 100\u00b0 C. for 8 seconds. This high temperature precipitated both types of HLf and only 10% of HLf was detectable by ELISA. More than 80% of iron was still bound to both rHLf and nHLf after all thermal treatments with the exception of 100\u00b0 C. for 8 sec. In 10% of survived HLf after 100\u00b0 C. for 8 sec, the iron saturation level of nHLf was above 80% whereas that of rHLf was only about 40%.","SDS-PAGE and Western blots revealed no difference in immunoreactivity between nHLf and rHLf at 62\u00b0 C. for 15 minutes, at 72\u00b0 C. for 20 seconds, and at 85\u00b0 C. for 3 minutes, but at 100\u00b0 C. for 8 seconds, rHLf almost completely lost its immunological activity, whereas nHLf still maintained detectable immunoreactivity.","There was no significant difference in anti-microbial activity between nHLf and rHLf after heat-treatment. Anti-microbial activity of HLf was not affected by treatment at either 62\u00b0 C. for 15 min, 72\u00b0 C. for 20 sec or 85\u00b0 C. for 3 min.","The Kd and the number of binding site for nHLf and rHLf were not significantly different at 62\u00b0 C. and 72\u00b0 C. though there is a trend that nHLf is somewhat lower Kd and binding sites than rHLf. As the temperature was increased (such as 85\u00b0 C. and 100\u00b0 C.), more rHLf bound to Caco-2 cells, most likely by non-specific binding due to more rHLf being denatured than nHLf. Uptake properties were similar for nHLf and rHLf even in the group treated at 100\u00b0 C. where uptake of both types of HLf was highest among all the thermal treatments. Free iodine levels in the cells were also evaluated since it reflects degradation of HLf. About 20% of HLf was degraded in the untreated sample. There was no significant difference between nHLf and rHLf. Interestingly, samples treated at 100\u00b0 C. were degraded twice as much as untreated samples of nHLf and rHLf, which may indicate that denaturation of HLf caused by heat treatment will make the protein more susceptible to proteases in the cells.","(xi). pH Stability: Effect on Antimicrobial Activity and Binding\/uptake to Caco-2 Cells","1.0 mg\/ml of holo-HLf in PBS was adjusted to pH 2, 4, 6, or 7.4 by the addition of 1 M HCl and incubated for 1 h at room temperature. The pH was then adjusted to 7.0 with 1 M NaHCO3. Free iron released from HLf, was removed by a desalting column.","After low pH treatment, 100% of both nHLf and rHLf survived. The iron-holding capacity was maintained in all samples and the iron saturation level was above 95%. SDS-PAGE and Western blots revealed that there was no difference between nHLf and rHLf for any of the treatments. A slightly smaller immunoreactive molecule (\u02dc70 kD) was detected after exposure of nHLf to pH 2 and 4 and of rHLf to pH 2.","Antimicrobial activities of nHLf and rHLf were stable after exposure to low pH in the range of pH 2.0 to 7.4. As the pH was lowered, the activity of rHLf appeared to be higher and constant, whereas nHLf did not show any pH dependency.","Kd and the number of binding sites for nHLf were not significantly different from those for rHLf but a trend was always lower for nHLf within the range of pH 2.0 to 7.4, which is similar to control and thermal treatment samples. The Kd and the number of binding sites for nHLf and rHLf were unaffected by pH treatment down to 2.0 for 1 hour. Uptake properties were similar for nHLf and rHLf in the pH range of 2.0 to 7.4. Degradation of HLf in Caco-2 cells was also evaluated and there was no significant difference between nHLf and rHLf.","A. Construction and Expression of Human AAT in Rice Cells","The construction and purification of functional recombinant human AAT were carried out as exemplified in previous examples. Briefly, codon-optimized AAT gene was cloned into an pAPI145 that contains the rice Gt1 promoter, Gt1 signal peptide, and Nos terminator, pAPI241 that contains Glb promoter, Glb signal peptide, and Nos terminator, and API280 that contains Bx7 promoter, Bx7 signal peptide, and Nos terminator, as exemplified in Example 1. The resulting plasmids were named pAPI250, API255 and pAPI282, respectively (FIG. ). Transgenic plants expressing AAT were generated as above, and plant-generated recombinant AAT was characterized. To express AAT in culture cells, codon-optimized AAT gene was cloned into an expression cassette that contains the rice RAmy3D promoter, signal peptide, and terminator. Recombinant AAT expression was induced and secreted to the culture medium under the sugar starvation condition. Purification of rAAT was achieved through a scheme that consisted of an affinity column (Con A), anion exchange column (DEAE), and a hydrophobic interaction column (Octyl).","B. SDS-polyacrylamide Gel Electrophoresis (SDS-PAGE)","AAT samples were ground with PBS with mortar and pestle. The resulting extract was spun and 20 microliters of supernatant loaded into a precast SDS-PAGE gel. The AAT protein was clearly visualized with Coomassie brilliant blue staining (FIG. ).","C. Western Blot Analysis","For immunoblotting analysis, gels were electroblotted to a 0.45 \u03bcm nitrocellulose membrane with a Mini Trans-blot Electrophoretic Transfer cell (Bio-Rad, USA) and subsequently subjected to immunoblotting analysis. Blots were blocked with 5% non-fat dry milk in PBS, pH 7.4 for at least two hours followed by three washes with PBS, pH 7.4 for 10 minutes each. The primary rabbit polyclonal antibody against human alpha-1-antitrypsin (Dako A\/S, Denmark) was diluted to 1:2500 in the blocking buffer and the blot was incubated for at least one hour. The blot was then washed as described previously. The secondary antibody, goat anti-rabbit IgG (H+L)-alkaline phosphatase conjugated (Bio-Rad), was diluted in the blocking buffer at a dilution of 1:4000. The membrane was then incubated in the secondary antibody solution for one hour and followed by the same wash process. Color development was initiated by adding the substrate BCIP\/NBT from Sigma.","The western result showed that AAT protein is clearly visualized and confirmed that AAT expressed and deposited in transgenic rice grain, has a molecular weight that is somewhat smaller than that of native AAT (FIG. ).","D. ELISA","Standards for this assay ranged from 1.25-20 ng\/mL of AAT (Athena) diluted in PBST. Nunc Immuno-plate Maxisorp 96-well plates (Nunc, Denmark) were coated for 16 h at 4\u00b0 C. by a 1:10,000 dilution of rabbit anti-human AAT in 0.05 M sodium bicarbonate, pH 9.6. The plates were washed 3 times with PBST (PBS, pH=7.4, 0.05% Tween-20) and subsequently incubated with sample for 1 h at room temperature while rocking. The plates were washed again 3 times with PBST, followed by incubation with a 1:50,000 dilution of goat anti-human AAT conjugated to HRP for 1 h at room temperature. The plates were washed 3 times with PBST, and bound antibody was detected by the HRP\/hydrogen peroxide catalyzed reaction of TMB. The reaction was stopped with 2 M sulfuric acid, and the plates were read on a microtiter plate reader at 450 nm, using 620 nm as a reference filter.","Recombinant AAT is 2.1 times more immunoreactive, when comparing equal concentrations as determined by the Lowry assay.","E. AAT Activity Assay","AAT activity was analyzed using a modified method published by Travis and Johnson (1981). In 96-well microtiter plates, 60 \u03bcL samples diluted in Tris buffer, (0.2 M Tris, pH 8.0) were added. In each well, 60 \u03bcL of elastase [0.01 mg\/mL porcine pancreatic elastase (PPE) in Tris buffer] was also added. The plate was rocked for 5 min at room temperature to allow any available AAT to bind to the elastase. Another 120 \u03bcL of substrate solution (10 M N-Succinyl-AAA-p-nitroanilide in DMSO diluted in Tris buffer to give 0.33 M N-Succinyl-AAA-p-nitroanilide) was added, and the plate was rocked for 1-2 min at room temperature. The plate was immediately read on a microtiter plate at 405 nm. The plate was read again after 5 min, and the change in absorbance was calculated. AAT activity was determined using linear regression from a standard curve. The results show that AAT protein produced in rice grain has similar bioactivity as that of native AAT.","F. Band Shift Assay","The unique property of the covalent-linked complex formed between AAT and PPE permits an analysis of the activity of AAT by SDS-PAGE. Briefly, 20 \u03bcl of tested samples containing AAT from the screening or purification processes was incubated with 100 ng PPE at 37\u00b0 C. for 15 minutes. Five \u03bcl of SDS-loading dye was added and the reaction mixture boiled for five minutes. The sample was then centrifuged and kept on ice until loaded onto a 10% precast SDS-PAGE gel. The resulting gel was stained with 0.1% Coomassie brilliant blue R-250 as described below. For immunodetection, a western blot analysis was carried as described above. Again the band shift assay indicated that AAT protein produced in rice grain has similar bioactivity as that of native AAT (FIGS. A and B).","G. In vitro digestion. The digestion was carried out using a modified method of Rudloff and Lonnerdal (1992) was used after some modifications. Native and recombinant AAT were diluted in PBS or formula to 0.5 mg\/mL. Hydrochloric acid (1 M) was added to all samples to adjust the pH 3, 4, and 5, then 2.5 \u03bcL of 2% pepsin in 0.01 M HCl (3,100 U\/mg solid) were added and all samples were placed in a shaking incubator for 30 or 60 min at 37\u00b0 C. The pH was restored by drop-wise addition of 1 M NaHCO3, and 2.5 \u03bcL of 0.4% pancreatin in 0.1 M NaHCO3 were added. Samples were incubated for 1 or 2 hours at 37\u00b0 C., and the reaction was halted by dilution 1:2 in sample buffer and boiling for 3 min. For samples subjected to pepsin digestion only, boiling was unnecessary since the pepsin was inactivated when the pH was raised above pH 6 with NaHCO3 (Piper and Fenton, 1965). The enzyme: substrate ratio was approximately 1:20 for samples in buffer only and about 1:600 for samples in formula.","A significant amount of recombinant and native AAT survived the in vitro digestion, and both forms were more resistant to degradation than human serum albumin. Digestion with pepsin at pH 4 shows that 65% of recombinant AAT is detectable by ELISA after digestion, which is similar to 67% of native AAT surviving. The trypsin assay shows that much of the inhibitory properties of both forms are still intact, and the activity assay reveals that 63% and 59% of the activity of native and recombinant AAT remains, respectively. When exposed to both pepsin and pancreatin in buffer, native AAT resisted degradation when the pH of the pepsin incubation was pH 4 or higher. Under this condition, the recombinant form was less resistant, although a large part remained after pepsin digestion at pH 5 and pancreatin digestion. At pH 4, more of the recombinant protein was degraded, either due to pepsin activity or pH instability. AAT activity could not be determined after digestion by pepsin and pancreatin because of the inactivation of pancreatin by boiling which also inactivates AAT activity. In formula, both forms appeared to be equally resistant to degradation. While both native and recombinant AAT were still present after pepsin digestion at pH 5 followed by pancreatin digestion, bands at about 33 kD (casein) are faint or missing. It is possible that other proteins in formula are preferentially cleaved, reducing the amount of AAT being digested.","H. Thermal Stability of recombinant human ATT. Both native and recombinant human AAT were diluted in phosphate buffered saline (PBS) or infant formula (Enfamil with Iron, Mead Johnson, Evansville, Ill.) to a concentration of 0.1 mg\/mL. Samples, 100 \u03bcL in capped, 10\u00d775 mm glass tubes, were treated as follows: 60\u00b0 C. for 15 min, 72\u00b0 C. for 20 sec, 85\u00b0 C. for 3 min, and 137\u00b0 C. (temperature of oil bath) for 20 sec. The samples were allowed to cool to room temperature after heat treatment. For formula samples with bile extract added, 2.5 \u03bcL of 12% porcine bile extract (Sigma) were added, then vortexed quickly, incubated at 37\u00b0 C. for 10 min, and vortexed again. All samples were diluted 1:10 in PBS and transferred to 1.5 mL tubes. Formula samples were centrifuged at 15,000 g for 20 min to remove the insoluble fraction, and the supernatant was withdrawn after skimming off the fat. All samples were subsequently transferred to 1.5 mL tubes and analyzed.","The thermal stability of native AAT exceeded that of the recombinant form in buffer, but the recombinant AAT retained significant stability under most conditions. When heated in buffer only, SDS-PAGE and Western blots show that the two forms of AAT have similar structural stability. While the ELISA data show that the recombinant protein is less stable at the higher temperatures, the recombinant protein is similar to the native form under the other conditions. However, the functional stability of the recombinant protein may be affected. The thermal stability assay shows that the recombinant protein lost functional ability at several of the heat conditions, whereas the native protein was functional at all heat conditions except for at 62\u00b0 C. for 15 minutes. While the elastase-inhibiting properties of native AAT were about 90% after all heat treatments, 62 and 51% of the recombinant protein's activity remained after 85\u00b0 C., 3 minutes, and 137\u00b0 C., 20 seconds, respectively.","The heat treatments of native and recombinant AAT in formula affected the detection of the proteins, but the addition of bile extract following heat treatment restored antibody recognition of the recombinant form. While the Western blot data show less detectable protein only at 85\u00b0 C., 3 min for the native AAT and at 72\u00b0 C., 20 sec and 137\u00b0 C., 20 sec for the recombinant AAT, the ELISA data shows less than 20% protein detected for both forms and for all heat conditions. When bile extracts were added to the heated formula samples, the ELISA data for the recombinant form showed that more than 50% was still detectable after heat treatment. The bile extract did affect detection of the native form by ELISA for most of the heat treatments. The Western blots corroborated the ELISA data and showed that the bile extract may dissociate the recombinant AAT from other formula proteins, but it is not effective for native AAT at the higher temperatures.","I. pH stability of human ATT. Native and recombinant AAT were diluted in PBS or formula to 0.1 mg\/mL. The sample volume was 1 mL, and the pH of each sample was adjusted drop-wise with 1 M HCl. The range of pHs tested was from pH 2 to 8 for the samples in PBS and pH 2 to 7 for samples in formula. After a 1 hour incubation at room temperature, the pH was restored to pH 7 with 1M NaHCO3. Formula samples were centrifuged as exemplified in above Thermal Stability section.","Both native and recombinant AAT appear resistant to low pH conditions in both PBS and formula. There were no differences between treatment groups and controls for pH 3 through 7, and controls or between the native and recombinant AAT according to SDS-PAGE, Western blots, and trypsin assay. However, the elastase assay and ELISA data show that recombinant AAT is more affected by acidic conditions than the native form. In PBS, native AAT was more than 95% intact, while about 60-80% of the recombinant AAT activity was intact. Infant formula may have a stabilizing effect on the recombinant protein, since it was found to be as stable as the native form according to ELISA and the Western blot.","Native and recombinant AAT can withstand acidic and digestive conditions as assessed by SDS-PAGE, Western blots, ELISA and activity assay. Native AAT regains much of its structural and functional stability after treatment at acidic conditions followed by neutralization, whereas recombinant AAT shows some loss of activity at a wide pH range, which may reflect a different glycosylation pattern. The conditions of the infant-modeled digestion, pH 5 during pepsin treatment, are not ideal for pepsin, which normally possesses full activity at pH 2. AAT has been detected in human infant feces, which supports the notion that it is capable of surviving digestion in vivo, particularly during the first three months of the infant's life. This evidence also supports the validity of the in vitro digestion system. It is likely that AAT possesses enough resistance to acidic and digestive conditions to allow a significant amount to survive and affect the digestion process.","Recombinant AAT remained functionally intact after being exposed to low pH, in vitro digestion, and several types of heat treatment. It is therefore possible that recombinant AAT may be added to infant formula, can tolerate some processing conditions, and remain intact in the gastrointestinal tract of infants. Thus, recombinant AAT may help protect other physiologically active proteins, such as lactoferrin and lysozyme, which also may be added in recombinant forms in the gut of formula-fed infants. In conclusion, addition of recombinant AAT together with other recombinant proteins may enhance their bioactivity and make the formula more similar to human milk.","J. Expression of AAT in transgenic wheat. The plasmid API282 containing the Bx7 promoter, Bx7 signal peptide and AAT gene, Nos terminator and ampicillin resistance gene was used to transform wheat cells, substantially as in the transformation of rice cells. Twenty one transgenic lines were produced. Expression of AAT was determined to be about 5 to 12 \u03bcg per grain of wheat seeds.","A. Generation of Recombinant Antibodies","Recombinant antibodies have been expressed in transgenic plants (for -examples, see Peeters et al., 2001; Giddings et al., 2000; Larrick et al., 1998). However, expression and production of recombinant antibodies in the seeds of transgenic plants have certain advantages. The production of high levels of antibodies in grains, for example rice grains, provides distinct advantage that food supplements may be prepared with little or no purification, and other advantages that are illustrated herein the patent application.","In one embodiment, an expression vector is constructed as illustrated in Example 1 that includes codon optimized nucleotide sequences encoding functional components of an antibody. For example, the components can be a heavy chain, a light chain, a linker region or a J chain and a secretory component. The expression vector may also include a promoter, a signal\/target\/transport sequence or sequences and a terminal sequence or sequences. Preferred promoter, signal\/target\/transport sequence and terminal sequence are exemplified herein. For example, for expression of each functional component of an antibody in rice seeds, a codon-optimized component gene is operably linked to the rice endosperm specific glutelin (Gt1) promoter, a Gt1 signal peptide and NOS terminator to form a component expression vector.","Each component expression vector is introduced to rice cells and plants to generate antibody component-expressing transgenic rice cells and plants, as exemplified in Example 2. In one embodiment, the expression vectors containing antibody heavy chain, light chain, linker region or J chain, and a secretory component can be introduced individually. The plants expressing each individual component can be crossed to generate plants that express a functional antibody.","In another embodiment, the expression vectors containing functional components of an antibody can be introduced to the plant at the same time, using the transformation methods exemplified in Example 2, such as by co-bombardment. A plant that expresses functional antibody is selected for further propagation.","In another embodiment, the expression vector containing codon optimized nucleotide sequence encoding a single chain antibody is introduced to rice cells and plants to generate antibody expressing transgenic rice cells and plants, as exemplified in Example 2. The nucleotide sequence encoding a single chain antibody can be constructed as conventional in the art, for example Kortt et al., 2001, Maynard and Georgiou, 2000; Humphreys DP and Glover, 2001.","The plant-generated recombinant antibody can be isolated and purified as exemplified in the patent application.","B. Generation of Human EGF","The Epidermal Growth Factor (EGF) gene was codon optimized as shown in , and synthesized by Operon Technologies (CA, USA) with a SEQ ID NO: 8. The gene was cloned into pAPI145 and pAPI241 respectively, as exemplified in Example 1. The resulting plasmids were named API270 () and API303 (FIG. ), respectively. For expression of EGF in rice seeds, the codon optimized gene was operably linked to the rice endosperm specific glutelin (Gt1) promoter, Gt1 signal peptide and NOS terminator in pAPT303, and to the rice endosperm specific globulin (Glb) promoter, Glb signal peptide and NOS terminator in API270. The transgenic plant expressing EGF was generated, and plant-generated recombinant EGF was detected, as shown in FIG.  and as exemplified herein.","C. Generation of Human IGF","The Insulin-like Growth Factor (IGF) gene was codon optimized as shown in , and synthesized by Operon Technologies (CA, USA) with SEQ ID NO: 9. The gene was cloned into pAPI145 and pAPI241 respectively, as exemplified in Example 1. The resulting plasmids were named API271 () and API304 (FIG. ), respectively. For expression of IGF in rice seeds, the codon optimized gene was operably linked to the rice endosperm specific glutelin (Gt1) promoter, Gt1 signal peptide and NOS terminator in pAPI304, and to the rice endosperm specific globulin (Glb) promoter, Glb signal peptide and NOS terminator in API271. The transgenic plant expressing IGF was generated, and plant-generated recombinant IGF was detected as shown in FIG.  and as exemplified herein.","D. Generation of Other Expression Plasmids","Other expression plasmids for use in transforming plants herein for the production of recombinant polypeptides in transgenic plants were made substantially as previously described. These plasmids are shown in , showing API321, containing a Glb promoter, a Gt1 signal peptide, codon-optimized haptocorrin gene, Nos terminator, and an amipicillin resistance gene; , showing API320, containing a Gt1 promoter, a Gt1 signal peptide, codon-optimized human haptocorrin gene, Nos terminator, and an amipicillin resistance gene; , showing API292, containing a Glb promoter, a Glb signal peptide, kappa-casein gene, Nos terminator, and an amipicillin resistance gene; , showing API297, containing a Gt1 promoter, a Gt1 signal peptide, a gene encoding mature kappy-casein polypeptide, Nos terminator, and an amipicillin resistance gene; , showing API420, containing a Gt1 promoter, a Gt1 signal peptide, lactadherin gene, Nos terminator, and a kanamycin resistance gene; , showing API418, containing a Gt1 promoter, a Gt1 signal peptide, lactoperoxidase gene minus the sequence encoding the propeptide, Nos terminator, and a kanamycin resistance gene; , showing API416, containing a rice Gt1 promoter, a Gt1 signal peptide, codon-optimized lactoperoxidase gene, Nos terminator, and a kanamycin resistance gene; and , showing API230, containing a Bx7 promoter, a Gt1 signal peptide, codon-optimized lysozyme gene, Nos terminator, and an amipicillin resistance gene; , showing API254, containing a Glb promoter, a Glb signal peptide, lactoferrin gene, Nos terminator, and an amipicillin resistance gene; , showing API264, containing a Glb promoter, a Glb signal peptide, human lysozyme gene, Nos terminator, and an amipicillin resistance gene; , showing API225, containing a GT3 promoter, a Gt1 signal peptide, codon-optimized lysozyme gene, Nos terminator, and an amipicillin resistance gene; and , showing API229, containing a RP-6 promoter, a Gt1 signal peptide, codon-optimized lysozyme gene, Nos terminator, and an amipicillin resistance gene.","A. Comparison between Gt1 and Glb Promoters and Signal Peptides","In earlier studies, inconsistencies were observed between promoter activity of Glb and Gt1 from transient assay data and the protein accumulation level in transgenic plants bearing the same promoters with signal peptides. These unpublished studies suggested that post-translational regulation was involved in recombinant protein expression and accumulation in the endosperm. It was unknown whether the storage protein signal peptide played a role in recombinant protein expression level or whether heterologous proteins could be sent to the protein bodies along the sorting pathways of native storage proteins. In order to improve the expression level of recombinant proteins in cereal crop seed, it is important to understand recombinant protein targeting and trafficking in the endosperm expression system. Hence, comparison was made between the rice storage protein promoters and signal peptides from the Glutelin-1 gene (\u201cGt1\u201d) and the globulin gene (\u201cGlb\u201d) showed that both promoters and both signal peptides were capable of effecting expression of lysozyme.","(i). Storage Proteins","Rice endosperm contains four main storage proteins: acid-soluble glutelin, alcohol-soluble prolamin, water-soluble albumin and salt-soluble globulin (Juliano BO. Polysaccharides, proteins, and lipids of rice. Am. Assoc. Cereal Chem., St. Paul, Minn. (1985)). They are targeted into two types of protein bodies in rice endosperm. Prolamin aggregates within the endoplasmic reticulum (\u201cER\u201d) lumen into regularly shaped vacuole called protein body type I. The formation of these protein bodies is dependent on the chaperone BiP80 in the ER. Glutelin is deposited into protein storage vacuoles (PSV) via the Golgi apparatus into irregularly shaped vacuole called type II protein body. The components in the protein body type II and its sorting pathway are not well known. The targeting locations and sorting pathway of globulin and albumin also remain unknown. It appears that once the signal sequence is removed in the ER, the sorting and trafficking depend on the targeting information within the polypeptides and chaperones in the ER. The sorting signals are divided into three categories: sequence-specific vacuole sorting signals (ssVSS), C-terminal vacuole sorting signal (ctVSS), and physical structure vacuole sorting signals (psVSS), as described in Frigerio L. et al., Plant Physiol. 126: 167-175 (2001); Matsuoka K. et al., J. Exp. Botany 50: 165-174 (1999) and Vitale A. & Raikhel N. V., Trends in Plant Science 4: 149-155 (1999).","(ii). Method","Two promoters storage protein genes, Gt1 and Glb, and the corresponding glutelin-1 and globulin signal peptide coding sequences were used to express the human lysozyme protein in developing endosperm. In the three plasmids, pAPI264, pAPI159 and pAPI228, the human lysozyme gene was fused with the nucleotide sequences of the Glb promoter and globulin signal peptide coding sequences, the Gt1 promoter and glutelin signal peptide coding sequences and the combination of the Glb promoter with the glutelin (GT1) signal peptide coding sequences, respectively (FIG. A). Lysozyme amounts of T1 seeds were determined for 23 independently transformant lines of pAPI264, 10 lines of pAPI159 and 7 lines of pAPI159. The transgenic lines of pAPI159, which synthesized lysozyme using the Gt1 promoter and the glutelin signal peptide, produced the enzyme in amounts ranging from 34.25 \u03bcg to 297.23 \u03bcg\u00b7mgtotal soluble protein (TSP) with an average of 133.76 \u03bcg\u00b7mgTSP. Plants transformed with pAPI264 carrying the Glb promoter and the globulin signal peptide yielded between 4.09 and 63.64 \u03bcg\u00b7mgTSP lysozyme with an average of 33.96 \u03bcg\u00b7mgTSP, while lines of pAPI228, which combined the Glb promoter and the glutelin signal peptide, yielded between 8.9 and 203.46 \u03bcg\u00b7mgTSP with an average of 87.70 \u03bcg\u00b7mgTSP.","The lysozyme expression amounts achieved with the Gt1 promoter+GT1 signal peptide was 3.94 fold higher than that with the Glb promoter+GLB signal peptide, while the expression amounts of lysozyme obtained with the Glb promoter+GT1 signal peptide was intermediate but increased 2.58 fold over that produced with the GLB signal peptide (FIG. B). Apparently the GT1 signal peptide is more efficient than the GLB signal peptide at lysozyme expression and deposition in rice endosperm. This demonstrates the importance of choosing an optimal signal peptide for the production of recombinant proteins in developing rice grains.","(iii). Chimeric Gene Components","Time course of human lysozyme expression during rice endosperm development. We monitored lysozyme accumulation during endosperm development of transgenic lines 159-1-53-16-1 and 264-92-6. Immature spikelets were harvested at 7, 14, 21, 28, 35, 42 and 49 days after pollination (\u201cDAP\u201d). The lysozyme amounts in the endosperms were measured by the activity assay. Lysozyme accumulation in the seeds of transgenic plant 159-1-53-16-1 began at 7 DAP and peaked at 21 DAP. Thereafter lysozyme content decreased until 35 DAP and then stabilized until seed maturity (FIG. ). Lysozyme accumulation in developing seeds of the transgenic plant 264-92-6 likewise began at 7 DAP, peaked at 28 DAP, after which lysozyme content steadily decreased through seed maturation (FIG. ). These results show that lysozyme accumulation in the two types of transgenic lines during endosperm development follows the same pattern as that of the native globulin and glutelin storage proteins.","(iv). Subcellular Localization of Human Lysozyme in Transgenic Seeds","In order to determine whether the recombinant lysozyme was targeted to protein bodies in the endosperm, we investigated its subcellular localization by immunofluorescence microscopy. Transgenic plant 264-92-6 synthesizing lysozyme with the Glb promoter and globulin signal peptide and transgenic plant 159-1-53-16-1, producing human lysozyme with the Gt-1 promoter and the glutelin signal peptide were analyzed. Dual localization with either native glutelin or globulin was used to determine the site of lysozyme deposition.","Synthetic peptides derived from the amino acid sequences of rice glutelin and globulin were used to raise antibodies in rabbits. Antibody specificity was confirmed with Western blots of endosperm proteins. No cross-reaction of glutelin and globulin antibodies with other endosperm proteins was detected with the host TP309 or the transgenic lines 264-92-6 and 159-1-53-16-1. The human lysozyme specific antibody detected the 13 kD of lysozyme protein exclusively in the fractionated and total protein extracts.","Immature seeds from two transgenic lines, 159-1-53-16-1(T4) and 264-92-6(T2) and untransformed control, TP309, were harvested at 14 DAP and fixed and a comparable analysis was conducted. In transgenic line 264-92-6, strong immunofluorescence signals of lysozyme and native proteins were detected with fully overlapping pattern, both when lysozyme and globulin or lysozyme and glutelin were compared (data not shown). Merging the two separately recorded images produces a yellow pseudo color signal. A scan for green and red wavelength emission across the 5 protein bodies along the white line in the not quite perfectly aligned images identifies the co-localization of human lysozyme with globulin. The orange tinge of the protein bodies is due to the stronger emission of red fluorescence than green. The perfect image merger provides a bright yellow color, and the recording of the green and red fluorescence emission along the white line identifies the same 5 protein bodies. These results demonstrated that lysozyme was colocalized in protein bodies with the native storage proteins. The results also demonstrated that the storage proteins globulin and glutelin are localized in the same cell compartment, substantiating the indication that globulin and glutelin are targeted into the same type II protein bodies in rice endosperm. We conclude that lysozyme contains all sorting information for protein body targeting, at least when co-expressed with rice storage proteins.","The localization patterns of lysozyme and native storage proteins in 159-1-53-16-1 are, however, quite different and more complex than those of transgenic line 264-92-6. In transgenic 159-1-53-16-1, lysozyme does not completely colocalize with the native storage proteins. Globulin localized preferentially in smaller, peripheral protein bodies in the younger cells of the cortical region from 14 DAP endosperm, while lysozyme localized preferentially in irregularly shaped protein bodies. However, lysozyme did colocalize more evenly with globulin in the older cells of the central region from the developing endosperm. Merging the two separately scanned images visualized green fluorescing, lysozyme-rich type II protein bodies and red fluorescing, smaller, globulin-rich protein bodies. Recording of the red and green fluorescence emission along the white scanning line reveals that there is almost twice as much lysozyme in the large type II protein bodies as there is in the small protein bodies, while the globulin signal in the small protein bodies is 2-3 times observed in type II protein bodies. Thus, there appears to be a preferential targeting of the two proteins. In the central region of the endosperm, a more equal co-localization of lysozyme and globulin was observed, especially in the larger type II protein bodies, when judged by the intensity of green and red fluorescence, which provides the yellow color upon merging the two images. This is evident from the merged image scan at the two emission wavelengths. However, there are also small protein bodies containing a dominant portion of globulin or lysozyme in these cells.","Distinct patterns were also found in 159-1-53-16-1 when anti-glutelin antibody was co-incubated with anti-lysozyme antibody in the younger cells of the cortical region from mid-developing endosperm. Like globulin, most of the glutelin localized in the smaller, peripheral protein bodies in younger cells, while lysozyme localized in irregularly shaped protein bodies. Lysozyme partially colocalized with glutelin in the older cells from the center region of mid-developing endosperm. Merging the two images and scanning for fluorescence-intensity at the two wavelengths reveals co-localization of the two proteins in the large and small protein bodies, some being highly enriched in lysozyme and others in glutelin. A comparable distribution is observed in the cells of the central part of the endosperm. The results suggested that lysozyme distorted the native storage protein targeting\/sorting when under the control of the Gt1 promoter\/GT signal peptide, producing high lysozyme expression, but not when under the control of Glb promoter\/GLB signal peptide with lower lysozyme expression.","To determine if native protein accumulation was affected in the endosperm of the transgenic plants, we analyzed the amounts of glutelin, globulin and lysozyme proteins from two transgenic lines and TP309 by Western blotting. The results showed that glutelin protein was reduced in 159-1-53-16-1, but was increased in 264-92-6 in comparison to TP309. Amounts of globulin protein were reduced in both 264-92-6 and 159-1-53-16-1. This change is particularly significant in the transgenic line 159-1-53-16-1 with its higher lysozyme expression level. The results showed that globulin was more affected than glutelin, no matter which signal peptide was used.","Based on the results, we conclude that lysozyme was targeted to the protein bodies and that the signal peptide played an important role in lysozyme expression. The plants with high expression levels of the recombinant protein showed distorted native protein expression and trafficking.","Thus, the combination of the Gt1 promoter and Gt1 signal peptide was more effective than the combination of the Glb promoter and Glb signal peptide, with the combination of Glb promoter and Gt1 signal peptide having intermediate level of activity. Results showed that the high level expression of recombinant protein distorted the trafficking and sorting of the native storage proteins and affected the native storage protein expression. Results also indicated that mature human lysozyme protein contains a determinant recognized in the plant cell for the protein storage vacuole (PSV) sorting following signal peptide cleavage, and that the lysozyme was sorted to Type II protein bodies.","B. Comparison of Seven Promoters and Gt1 Signal Peptide in Regulating the Expression of Lysozyme","Plasmids API159 (Gt1 promoter) (FIG. ), API228 (Glb promoter)(FIG. ), API230 (FIG. ), API229 (RP-6 promoter) (FIG. ), API225 (GT3 promoter), a plasmid carry the Glub-2 promoter, and another plasmid carrying the Club-1 promoter, were compared in their ability to effect the expression of lysozyme in transgenic rice T1 seeds. Results shown in  indicate that for expression of lyzoyme, Gt1 was the strongest promoter, followed by Glb, Glub-2, Bx7, Gt3, Glub-1 and Rp6, in order of promoter strength.","A. Enhanced Lysozyme Expression in Transgenic Rice Seed Co-Transformed with Reb","Codon-optimized human lysozyme gene was linked to Glb promoter and Glb signal peptide to generate plasmid Glb-Lys (API264) as shown in , which was used to transform rice with or without Native-Reb, as previously described and as described in WO 01\/83792. Normal plant phenotypes were obtained among transformants containing Glb-Lys alone or with Native-Reb. To determine the presence of Reb gene and Glb-Lys in the transgenic rice genome, one primer designed from vector sequence and another designed from the Reb gene 3\u2032 terminator were used to identify these lines. In this case, only the recombinant Reb gene could be amplified. PCR analysis confirmed the presence of transgenes in the rice genome. Ten of 11 plants from independent transformation events contained both Reb and the lysozyme transgenes. The REB protein of immature seeds from five randomly selected transgenic lines was detected by Western blotting. The expression level of the REB protein in transgenic lines ranged from 25% to 71% higher than that in untransformed TP309. This demonstrated that the transgenic Reb gene was active in transgenic plants.","Seeds of confirmed transgenic rice plants were harvested at maturity, and the lysozyme activity was analyzed. As shown in , lysozyme expression in the seeds from 30 independent transformation events containing both the Native-Reg and the Glb-Lys ranged from 30.57 to 279.61 \u03bcg\/mg TSP with an average of 125.75\u00b168.65 \u03bcg\/mg TSP. Seeds of 17 transgenic events containing the Glb-Lys gene alone expressed lysozyme in amounts ranging from 7 to 76 \u03bcg\/mg TSP with an average of 33.95\u00b120.55 \u03bcg\/mg TSP. No lysozyme activity was detected in untransformed rice seeds. The results showed that the expression level of lysozyme increased an average of 3.7-fold when seeds were transgenic for both the Reb gene and Glb-Lys. Statistical analysis (t test) showed that the amount of lysozyme in seeds from the plants transgenic for the Reb gene and Glb-Lys is significantly higher than that in the plants transgenic for Glb-Lys alone (p<0.001).","B. Enhanced Human Lysozyme Expression in Transgenic Rice Seed Co-Transformed with Maize Transcriptional Factor, Prolamin-box Binding Factor (PBF)","Three transcriptional factors were tested; rice endosperm bZIP protein (REB), Opaque2 (O2) and PBF. The transcriptional factors and human lysozyme gene under the control of rice glutelin 1 (Gt1) or globulin (Glb) promoter were co-bombarded into rice callus. Transgenic Rgrains carrying both genes were obtained. The effect of transcriptional factors on the expression of human lysozyme were monitored. Under the control of Glb promoter, REB increased Lys expression by about 3-fold. REB showed no effect on a stronger promoter, Gt1. Transcription factor increased Lys expression, but not significantly. PBF increased Lys expression on average 1.5-fold over Gt1-Lys alone. The highest Lys-expressing lines were selected and advanced to Rgeneration in the greenhouse. As shown in Table 8 below, Lys expression level from an Rline, 265\/159-41-5, was about 190 \u03bcg per grain and 9.5 mg\/gram of brown rice flour (equivalent to 0.8% grain weight). The level of expression was about 1.5-fold higher than that of the highest expression line without the transcription factor. In addition, data showed that PBF not only increased the expression of Lys, but also increased the expression of native storage proteins such as glutelin and globulin, and the protein related to protein trafficking. It implies that PBF can act on the promoters of multiple genes to increase the expression of those proteins in rice endosperm.",{"@attributes":{"id":"p-0603","num":"0604"},"tables":{"@attributes":{"id":"TABLE-US-00008","num":"00008"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"left"}}],"thead":{"row":[{"entry":"TABLE 8"},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":[{},"R* (\u03bcg\/","R* (\u03bcg\/","RLysozyme",{}]},{"entry":["Line Number","grain)","grain)","(mg\/g brown rice)","Homozygous"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["285\/159-41\u2002","150.23","190.00","9.5","homozygous"]},{"entry":["285\/159-43\u2002","114.30","155.00","9.0","homozygous"]},{"entry":["285\/159-81\u2002","103.58","175.59",{},"heterozygous"]},{"entry":["285\/159-286","152.07","180.00",{},"heterozygous"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":"* The expression data were averaged from 10 seeds in R; and from 10 lines in R. "}]}}}}},"A. General Procedure for Production of Rice Extract","Transgenic rice containing heterologous polypeptides can be converted to rice extracts by either a dry milling or wet milling process. In the dry milling process, transgenic paddy rice seeds containing the heterologous polypeptides, such as recombinant human lysozyme or lactoferrin were dehusked with a dehusker. The rice was grounded into a fine flour though a dry milling process, for example, in one experiment, at speed 3 of a model 91 Kitchen Mill from K-TEC. Phosphate buffered saline (\u201cPBS\u201d), containing 0.135 N NaCl, 2.7 mM KCl, 10 mM Na2HPO4, 1.7 mM KH2PO4, at pH 7.4, with or without additional NaCl, such as 0.35 N NaCl, was added to the rice flour. In some experiments, approximately 10 ml of extraction buffer was used for each 1 g of flour. In other experiments, the initial flour\/buffer ratio varied over a range such as 1 g\/40 ml to 1 g\/10 ml. The mixture was incubated at room temperature with gentle shaking for 1 hr. In other experiments, the incubation temperature was lower or higher, such as from about 22\u00b0 C. to about 60\u00b0 C., and the incubation time was longer or shorter, such as from about 10 minutes to about 24 hr. A Thermolyne VariMix platform mixer set at high speed was used to keep the particulates suspended.","In place of PBS, other buffers were used in some experiments, such as ammonium bicarbonate. In one embodiment, 10 liters of 0.5M ammonium bicarbonate was added to 1 kg of rice flour.","The resulting homogenate was clarified either by filtration or centrifugation. For the filtration method, the mixture was allowed to settle for about 30 minutes at room temperature, after which the homogenate was collected and filtered. Filters in three different configurations were purchased from Pall Gemansciences and used. They were: a 3 \u03bcm pleated capsule, a 1.2 \u03bcm serum capsule and a Suporcap capsule 50 (0.2 \u03bcm). For centrifugation, a Beckman J2-HC centrifuge was used and the mixture was centrifuged at 30,000 g at 4\u00b0 C. for about 1 hr. The supernatant was kept and the pellet was discarded.","In one embodiment, the filtrate and supernatant were further processed, for example by ultra-filtration or dialysis or both to remove components such as lipids, sugars and salt.","The filtrate from the above filtration procedure, which is also called the clarified extract, was then concentrated using a spiral wound tangential flow filter operated in a batch recirculation mode. In one embodiment, PES (polyethersulfone) 3000-4000 molecular weight cutoff membranes was used for this step. These final concentrated extracts were held overnight in a cold room.","The concentrated extracts were next dried to a powder by lyophilization. During loading of the lyophilizer trays, the extracts were not subjected to a final 0.2 or 0.45 micron depth filtration to minimize loss of target proteins. The lyophilized material was scraped from the lyophilizer trays and combined into a plastic bag. The dry material was compressed by drawing a vacuum on the bag and then the material was blended and the particle size reduced by hand-kneading it through the plastic.","The lyophilized materials were then suitable for use as an extract directly or in admixture with other food. In one experiment, the lyophilized materials were blended with various ingredients to produce control and test infant-formula. The ingredients were blended using a Hobart mixer (140 quart size) equipped with a paddle agitator. These final blends were packed in 1 kg double Mylar bags and the headspace was filled with nitrogen before sealing.","Table 9 shows the recovery of recombinant human lactoferrin from 105 kg transgenic rice flour during each extraction step. The amount of recombinant human LF present was determined quantitatively as described in Example 4.",{"@attributes":{"id":"p-0613","num":"0614"},"tables":{"@attributes":{"id":"TABLE-US-00009","num":"00009"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"98pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"14pt","align":"center"}}],"thead":{"row":{"entry":[{},"TABLE 9"]}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"0 Lactoferrin",{}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Stage of process","Lac mass","% of max"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"right"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Baseline extraction yield","4.0","mg\/g flour",{}]},{"entry":[{},"Expected maximum","420","g","100%"]},{"entry":[{},"Initial extract","338","g","\u200280"]},{"entry":[{},"Clarified extract","373","g","\u200289"]},{"entry":[{},"Concentrated extract","343","g","\u200282"]},{"entry":[{},"Dried extract","340","g","\u200281"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}}]}}},"Rice extract can also be produced using a wet milling procedure. Transgenic paddy rice seeds containing recombinant human lysozyme were re-hydrated for a period of 0 to 288 hrs at 30\u00b0 C. The rehydrated seeds were ground in PBS extraction buffer. The initial seed\/buffer ratio varied over a range such as 1 g\/40 ml to 1 g\/10 ml. Table 10 shows recovery of human lysozyme from rice seeds soaked from 0 to 288 hrs.",{"@attributes":{"id":"p-0615","num":"0616"},"tables":{"@attributes":{"id":"TABLE-US-00010","num":"00010"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"center"}}],"thead":{"row":[{"entry":"TABLE 10"},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Rehydration time (hrs)","Lysozyme (ug\/grain)","Recovery (%)"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["0","87","100"]},{"entry":["48","69","79"]},{"entry":["60","79","91"]},{"entry":["168","60","69"]},{"entry":["216","56","64"]},{"entry":["288","58","67"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Over 60% human lysozyme was recovered from the wet milling process. The result of the wet milling becomes initial extract which may be stored frozen until use. The processing of initial extract to obtain dried extract was the same as that described for dry milling in this section.","B. Concentration and Diafiltration of Recombinant Lysozyme and Control Rice Extracts","The conditions used in concentration and diafiltration varied depending on volume, speed, cost, etc. These conditions are all routine in the art based on the description herein. The frozen initial extract was thawed in the coldroom (about 2-8\u00b0 C.) for six hours. The thawed material were clarified though a 0.45 \u03bcm filter and concentrated using a 5000 Nominal Molecular Weight Cutoff membrane of Polyethersultone.","90 ml of the filtrate of control extract was concentrated to 10 ml and additional 10 ml of deionized water was added to the concentrated filtrate. The diluted filtrate was diafiltrated one more time using water. The precipitate started forming at 16 mS and increased as the ionic strength decreased. 1M ammonium bicarbonate was added to the retentate to add ionic strength. The haze decreased although did not disappear completely. The material was diafiltered multiple times, in one embodiment three times, with water and multiple times, in one embodiment three times, with 0.1 M ammonium bicarbonate. It was concentrated to 9 ml and the membrane was rinsed with 0.1 M ammonium bicarbonate. The concentrate was filtered through several 0.2 \u03bcm button filters. In one embodiment, 2.3 ml of the filtrate was lyophilized as is; 2.3 ml of the filtrate was diluted to 12 ml with deionized water and lyophilized, and 2.0 ml of the filtrate was diluted to 25 ml with deionized water and lyophilized. All remained clear.","A total of 89 ml of the filtrate of rHLys extract was concentrated to 10 ml, and additional 10 ml of 0.1 M ammonium bicarbonate was added. The resulting mixture was concentrated back to 10 ml and another 10 ml of 0.1 M ammonium bicarbonate was added. The retentate started to haze up. The material was diafiltered multiple times, in one embodiment three times, with 0.1 M ammonium bicarbonate. It was concentrated to 9 ml and the membrane was rinsed with 0.1 M ammonium bicarbonate. The concentrate was filtered through several 0.45 \u03bcm button filters. In one embodiment, 2.0 ml of the filtrate was lyophilized as is; 2.0 ml of the filtrate was diluted to 12 ml with deionized water where a haze formed, and lyophilized, and 2.0 ml of the filtrate was diluted to 12 ml with 0.1 M ammonium bicarbonate which remained clear, and lyophilized.","C. Comparison of Trial Extraction of Recombinant Lysozyme Rice with PBS and Ammonium Bicarbonate","The conditions used in concentration and diafiltration varied depending on volume, speed, cost, etc. These conditions are all routine in the art based on the description herein. rHlys rice flour was mixed with extraction buffer at about 100 g\/L for about 1 hour using a magnetic stir bar. In one 2 liter beaker, the extraction buffer was PBS, pH7.4 plus 0.35 M NaCl. In another 2 liter beaker, the extraction buffer was 0.5 M ammonium bicarbonate. A 15 cm buchner was pre-coated with about 6 g of Cel-pure C300 before adding another 20 g of Cel-pure C300. The mixture was filtered at about 3-4 Hg. It was then washed twice with about 100 ml of respective extraction buffer. The extracted filtrate was collected and concentrated with ultra-filtration cartridges: 5K Regenerate Cellulose, 5K PES, and 1K Regenerated Cellulose. The concentrates were lyophilized and analyzed for rHlys contents. The ammonium bicarbonate and PBS, pH7.4 plus 0.35 M NaCl both extracted approximately the same amount of rHlys. There was little loss of lysozyme units in the permeate with any of the ultrafiltration units that were used.","Other extraction buffer can also be used to extract recombinant proteins expressed in transgenic rice grains, for example Tris buffer, ammonium acetate, depending on applications. For example, for using recombinant human LF for iron supplement, iron may be added to the extraction buffer and the buffer is set at a pH so that the apo-LF can pick up iron during the extraction process. Under this condition, LF can become saturated with iron (holo-LF). In another example, a buffer lacking of iron and a pH resulting in iron release from LF is used to produce apo-LF.","D. Production of Rice Extracts Containing Recombinant Proteins","The conditions used in concentration and diafiltration varied depending on volume, speed, cost, etc. These conditions are all routine in the art based on the description herein. All equipment was soaked in hot 0.1M NaOH at a starting temperature of about 55\u00b0 C. Rice flour was added to an about 250-500 gal. stainless steel tank containing 0.5M ammonium bicarbonate at about 95-105 g\/L. It was mixed for about 60-80 minutes at about 9\u00b0 C.","12 plates of 36 inch filter press C300 were pre-coated with about 3-6 kg Cel-pure C300. About 19-26 g\/L of Cel-pure was added to the extract and mixed thoroughly. The mixture was pressed at a pressure of about 22 psi at a flow rate of about 82 liters\/minute. The filtrate was collected into a 250 gal. stainless steel tank and washed with 0.5M ammonium bicarbonate. The press was blown dry. The process was carried out at about 10\u00b0 C.","The 300 NMW cut-off membranes (Polysulfone) which had been cleaned and stored with 0.1M NaOH after control run was rinsed thoroughly with deionized water. The extract was concentrated and bumped to a 100 gal stainless steel tank. The membrane and the concentration tank were flushed with 0.1M ammonium bicarbonate to recover all of the products. The product were covered with plastic and left in the 100 gal tank overnight at room temperature. The concentrate was filtered through spiral wound 1 \u03bcm filter and into 5 gal poly container. The concentrate was lyophilized. About 81% of lactoferrin and about 58% of lysozyme was recovered from transgenic rice grains, respectively.","E. Blending of Rice Extract Containing Recombinant Proteins into Infant Formula","The three types of lyophilized dry extract that contains rice proteins (control) or rice proteins with lysozyme or lactoferrin were combined with standard infant formula. The blending was done such the final infant formula contained about 1 gram lactoferrin and 0.1 gram lysozyme per liter of infant formula. The ingredients were blended using a Hobart mixer (140 quart size) equipped with a paddle agitator. These final blends were packed in 1 kg double Mylar bags and the headspace was filled with nitrogen before sealing.","Samples of infant formula containing human lysozyme and lactoferrin were quantified using procedures described in Example 3 and 4.","Table 11 shows human lysozyme and lactoferrin in infant formula.",{"@attributes":{"id":"p-0632","num":"0633"},"tables":{"@attributes":{"id":"TABLE-US-00011","num":"00011"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"center"}}],"thead":{"row":[{"entry":"TABLE 11"},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Infant Formula","Lactoferrin (mg\/ml)","Lysozyme (mg\/ml)"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["With control rice extract","0.0","0.0"]},{"entry":["With transgenic rice extract","1.03","0.13"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Using extract as a delivery method of recombinant protein has clear advantages over the purified form or in the whole grain. The conventional approach, such as in the whole grain form, has limitations such as protein stability during high temperature and pressure processing. Furthermore, the purification approach is expensive. Therefore the extract approach 1) maintains a low cost compared to purification approach; 2) requires much smaller volume, for example about 1-10% of whole grain weight; 3) increases the concentration of recombinant protein from about 0.05-0.5% in whole grain form to about 10 to 20% in the extract form. Some extract form even reaches 40% depending on the expression level of recombinant protein. Therefore, the extract approach will allow broader application of the recombinant proteins compared to the whole grain approach. In addition, the extract approach removes starch granule, which requires high gelling temperature, for example about 75\u00b0 C. Consequently, the extract approach provides more flexibility in processing the rice grain and the recombinant proteins into food and diet, and the alike, without worrying about using high temperature to denature starch granule. The undenatured starch granule cannot be digested by human gut without gelatinization by for example high temperature.",{"@attributes":{"id":"p-0634","num":"0635"},"tables":{"@attributes":{"id":"TABLE-US-00012","num":"00012"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"266pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}}],"thead":{"row":[{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":[{},"SEQ"]},{"entry":[{},"ID"]},{"entry":["Description","NO"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Codon optimized lysozyme coding sequence:","\u20031"]},{"entry":"AAAGTCTTCGAGCGGTGCGAGCTGGCCCGCACGCTCAAGCGGCTCGGCAT"},{"entry":"GGACGGCTACCGGGGCATCAGCCTCGCCAACTGGATGTGCCTCGCCAAGT"},{"entry":"GGGAGTCGGGCTACAACACCCGCGCAACCAACTACAACGCCGGCGACCGC"},{"entry":"TCCACCGACTACGGCATCTTCCAGATCAACTCCCGCTACTGGTGCAACGAC"},{"entry":"GGCAAGACGCCCGGGGCCGTCAACGCCTGCCACCTCTCCTGCTCGGCCCT"},{"entry":"GCTGCAAGACAACATCGCCGACGCCGTCGCGTGCGCGAAGCGCGTCGTCC"},{"entry":"GCGACCCGCAGGGCATCCGGGCCTGGGTGGCCTGGCGCAACCGCTGCCA"},{"entry":"GAACCGGGACGTGCGCCAGTACGTCCAGGGCTGCGGCGTCTGA"},{"entry":["Amino acid sequence based on codon optimized lysozyme coding","\u20032"]},{"entry":"sequence:"},{"entry":"KVFERCELARTLKRLGMDGYRGISLANWMCLAKWESGYNTRATNYNAGDRST"},{"entry":"DYGIFQINSRYWCNDGKTPGAVNACHLSCSALLQDNIADAVACAKRVVRDPQGI"},{"entry":"RAWVAWRNRCQNRDVRQYVQGCGV"},{"entry":["Codon optimized lactoferrin coding","\u20033"]},{"entry":"sequence:GGGCGGCGGCGGCGCTCGGTGCAGTGGTGCGCCGTGTCCCAGC"},{"entry":"CCGAGGCGACCAAGTGCTTCCAGTGGCAGCGCAACATGCGGAAGGTGCGC"},{"entry":"GGCCCGCCGGTCAGCTGCATCAAGCGGGACTCCCCCATCCAATGCATCCAG"},{"entry":"GCCATCGCGGAGAACCGCGCCGACGCGGTCACCCTGGACGGCGGGTTCAT"},{"entry":"CTACGAGGCGGGGCTCGCCCCGTACAAGCTCCGCCCGGTGGCGGCGGAG"},{"entry":"GTGTACGGCACCGAGCGCCAGCCGCGCACGCACTACTACGCGGTGGCCGT"},{"entry":"CGTCAAGAAGGGCGGGTCCTTCCAGCTCAACGAGCTGCAGGGCCTGAAGT"},{"entry":"CGTGCCACACGGGCCTCCGGCGGACGGCGGGCTGGAACGTGCCCATCGG"},{"entry":"CACCCTGCGCCCCTTCCTGAACTGGACCGGCCCGCCGGAGCCGATCGAGG"},{"entry":"CCGCCGTGGCCCGCTTCTTCAGCGCCTCCTGCGTCCCCGGCGCCGACAAG"},{"entry":"GGCCAGTTCCCGAACCTCTGCCGGCTCTGCGCCGGGACGGGCGAGAACAA"},{"entry":"GTGCGCCTTCTCCTCGCAGGAGCCGTACTTCTCCTACTCGGGCGCGTTCAA"},{"entry":"GTGCCTCCGCGACGGGGCCGGCGACGTGGCGTTCATCCGCGAGTCCACCG"},{"entry":"TGTTCGAGGACCTCTCCGACGAGGCGGAGCGGGACGAGTACGAGCTGCTG"},{"entry":"TGCCCCGACAACACCCGCAAGCCGGTGGACAAGTTCAAGGACTGCCACCTG"},{"entry":"GCGCGGGTGCCCTCGCACGCGGTCGTCGCCCGCAGCGTCAACGGCAAGGA"},{"entry":"GGACGCGATCTGGAACCTCCTCCGCCAGGCCCAGGAGAAGTTCGGCAAGG"},{"entry":"ACAAGTCCCCCAAGTTCCAGCTCTTCGGGAGCCCCAGCGGCCAGAAGGACC"},{"entry":"TCCTCTTCAAGGACTCCGCGATCGGCTTCTCCCGCGTCCCCCCGCGCATCG"},{"entry":"ACTCCGGCCTGTACCTCGGCTCCGGGTACTTCACCGCGATCCAGAACCTCC"},{"entry":"GGAAGAGCGAGGAGGAGGTGGCGGCGCGGCGGGCCCGCGTCGTGTGGTG"},{"entry":"CGCCGTGGGCGAGCAGGAGCTGCGGAAGTGCAACCAGTGGAGCGGCCTGA"},{"entry":"GCGAGGGGTCGGTGACCTGCTCGTCCGCCAGCACCACCGAGGACTGCATC"},{"entry":"GCGCTCGTCCTCAAGGGGGAGGCCGACGCGATGAGCCTCGACGGGGGGTA"},{"entry":"CGTCTACACCGCCGGCAAGTGCGGCCTGGTCCCGGTCCTGGCGGAGAACT"},{"entry":"ACAAGTCGCAGCAGTCCAGCGACCCCGACCCGAACTGCGTGGACCGCCCC"},{"entry":"GTCGAGGGCTACCTCGCCGTGGCCGTCGTGCGCCGGTCCGACACCTCCCT"},{"entry":"GACGTGGAACAGCGTCAAGGGCAAGAAGAGCTGCCACACCGCCGTGGACC"},{"entry":"GCACCGCCGGCTGGAACATCCCGATGGGCCTCCTCTTCAACCAGACCGGCT"},{"entry":"CCTGCAAGTTCGACGAGTACTTCTCCCAGTCCTGCGCCCCCGGCTCGGACC"},{"entry":"CCCGCTCCAACCTGTGCGCCCTCTGCATCGGGGACGAGCAGGGCGAGAAC"},{"entry":"AAGTGCGTGCCCAACAGCAACGAGCGGTACTACGGCTACACGGGGGCCTT"},{"entry":"CCGCTGCCTGGCGGAGAACGCCGGGGACGTCGCGTTCGTGAAGGACGTGA"},{"entry":"CCGTGCTGCAAAACACGGACGGGAACAACAACGAGGCGTGGGCGAAGGAC"},{"entry":"CTCAAGCTCGCCGACTTCGCCCTGCTGTGCCTCGACGGCAAGCGCAAGCCC"},{"entry":"GTCACCGAGGCGCGGTCCTGCCACCTGGCGATGGCCCCCAACCACGCCGT"},{"entry":"CGTCTCCCGCATGGACAAGGTCGAGCGCCTCAAGCAGGTGCTCCTGCACCA"},{"entry":"GCAGGCCAAGTTCGGCCGGAACGGCAGCGACTGCCCGGACAAGTTCTGCC"},{"entry":"TGTTCCAGTCGGAGACCAAGAACCTCCTCTTCAACGACAACACCGAGTGCCT"},{"entry":"GGCGCGCCTCCACGGCAAGACCACCTACGAGAAGTACCTCGGCCCGCAGT"},{"entry":"ACGTCGCCGGCATCACCAACCTCAAGAAGTGCTCCACCTCCCCCCTCCTGG"},{"entry":"AGGCGTGCGAGTTCCTCCGCAAGTGA"},{"entry":["Amino acid sequence based on codon optimized lactoferrin","\u20034"]},{"entry":"coding sequence:"},{"entry":"GRRRRSVQWCAVSQPEATKCFQWQRNMRKVRGPPVSCIKRDSPIQCIQAIAEN"},{"entry":"RADAVTLDGGFIYEAGLAPYKLRPVAAEVYGTERQPRTHYYAVAVVKKGGSFQL"},{"entry":"NELQGLKSCHTGLRRTAGWNVPIGTLRPFLNWTGPPEPIEAAVARFFSASCVPG"},{"entry":"ADKGQFPNLCRLCAGTGENKCAFSSQEPYFSYSGAFKCLRDGAGDVAFIREST"},{"entry":"VFEDLSDEAERDEYELLCPDNTRKPVDKFKDCHLARVPSHAVVARSVNGKEDAI"},{"entry":"WNLLRQAQEKFGKDKSPKFQLFGSPSGQKDLLFKDSAIGFSRVPPRIDSGLYLG"},{"entry":"SGYFTAIQNLRKSEEEVAARRARVVWCAVGEQELRKCNQWSGLSEGSVTCSS"},{"entry":"ASTTEDCIALVLKGEADAMSLDGGYVYTAGKCGLVPVLAENYKSQQSSDPDPN"},{"entry":"CVDRPVEGYLAVAVVRRSDTSLTWNSVKGKKSCHTAVDRTAGWNIPMGLLFNQ"},{"entry":"TGSCKFDEYFSQSCAPGSDPRSNLCALCIGDEQGENKCVPNSNERYYGYTGAF"},{"entry":"RCLAENAGDVAFVKDVTVLQNTDGNNNEAWAKDLKLADFALLCLDGKRKPVTE"},{"entry":"ARSCHLAMAPNHAVVSRMDKVERLKQVLLHQQAKFGRNGSDCPDKFCLFQSE"},{"entry":"TKNLLFNDNTECLARLHGKTTYEKYLGPQYVAGITNLKKCSTSPLLEACEFLRK"},{"entry":["MV-Gt1-F1 primer:","\u20035"]},{"entry":"5\u2032\u2003ATC GAA GCT TCA TGA GTA ATG TGT GAG CAT TAT GGG ACC ACG 3\u2032"},{"entry":["Xba-Gt1-R1 primer: 5\u2032\u2003CTA GTC TAG ACT CGA GCC ATG GGG CCG GCT","6"]},{"entry":"AGG GAG CCA TCG CAC AAG AGG AA 3\u2032"},{"entry":["Codon optimized lactoferricin coding sequence","\u20037"]},{"entry":"ACCAAGTGCTTCCAGTGGCAGCGCAACATGCGGAAGGTGCGCGGCCCGCC"},{"entry":"GGTCAGCTGCATCAAGCGGGAC"},{"entry":["Codon optimized EGF coding sequence","\u20038"]},{"entry":"AACTCCGACTCGGAGTGCCCCCTCTCCCACGACGGTTACTGCCTCCACGAC"},{"entry":"GGGGTCTGCATGTACATCGAGGCCCTCGACAAGTACGCCTGCAACTGCGTC"},{"entry":"GTGGGCTACATCGGCGAGCGGTGCCAGTACCGCGACCTCAAGTGGTGGGA"},{"entry":"GCTGCGCTGA"},{"entry":["Codon optimized IGF-1 coding sequence","\u20039"]},{"entry":"GGCCCGGAGACCCTCTGCGGCGCCGAGCTCGTGGACGCCCTCCAGTTCGT"},{"entry":"GTGCGGCGACCGCGGCTTCTACTTCAACAAGCCGACCGGCTACGGCAGCA"},{"entry":"GCAGCCGCCGCGCCCCGCAGACCGGCATCGTGGACGAGTGCTGCTTCCGC"},{"entry":"AGCTGCGACCTCCGCCGCCTGGAGATGTACTGCGCCCCGCTCAAGCCCGC"},{"entry":"CAAGAGCGCCTGA"},{"entry":["Codon optimized lactadherin coding sequence","10"]},{"entry":"CTGGACATCTGCTCGAAGAACCCGTGCCACAACGGCGGGCTCTGCGAGGA"},{"entry":"GATCAGCCAGGAGGTGCGGGGCGACGTGTTCCCCTCGTACACCTGCACCT"},{"entry":"GCCTGAAGGGCTACGCCGGGAACCACTGCGAGACGAAGTGCGTGGAGCCC"},{"entry":"CTGGGGATGGAGAACGGCAACATCGCCAACTCCCAGATCGCCGCCTCCTCC"},{"entry":"GTGCGGGTGACCTTCCTCGGCCTCCAGCACTGGGTCCCGGAGCTGGCCCG"},{"entry":"GCTCAACCGGGCGGGCATGGTGAACGCGTGGACCCCCTCGTCCAACGACG"},{"entry":"ACAACCCGTGGATCCAAGTGAACCTGCTCCGCCGCATGTGGGTCACCGGCG"},{"entry":"TGGTCACCCAAGGCGCCAGCCGCCTGGCCAGCCACGAGTACCTCAAGGCC"},{"entry":"TTCAAGGTCGCCTACAGCCTCAACGGCCACGAGTTCGACTTCATCCACGAC"},{"entry":"GTCAACAAGAAGCACAAGGAGTTCGTGGGCAACTGGAACAAGAACGCGGTC"},{"entry":"CACGTGAACCTCTTCGAGACCCCCGTCGAGGCCCAGTACGTCCGCCTCTAC"},{"entry":"CCCACGAGCTGCCACACCGCCTGCACGCTCCGCTTCGAGCTGCTGGGGTG"},{"entry":"CGAGCTGAACGGGTGCGCGAACCCGCTGGGGCTCAAGAACAACAGCATCC"},{"entry":"CCGACAAGCAGATCACGGCCTCGTCGTCGTACAAGACCTGGGGCCTGCACC"},{"entry":"TCTTCTCGTGGAACCCGAGCTACGCCCGGCTGGACAAGCAGGGCAACTTCA"},{"entry":"ACGCCTGGGTCGCCGGGAGCTACGGGAACGACCAGTGGCTCCAGGTGGAC"},{"entry":"CTCGGCAGCTCCAAGGAGGTCACCGGCATCATCACGCAGGGGGCCCGCAA"},{"entry":"CTTCGGCTCCGTGCAGTTCGTGGCCTCCTACAAGGTGGCCTACTCGAACGA"},{"entry":"CAGCGCCAACTGGACCGAGTACCAGGACCCGCGCACCGGGTCCAGCAAGA"},{"entry":"TCTTCCCCGGCAACTGGGACAACCACAGCCACAAGAAGAACCTGTTCGAGA"},{"entry":"CCCCCATCCTCGCCCGGTACGTCCGCATCCTCCCCGTCGCTTGGCACAACC"},{"entry":"GGATCGCGCTCCGGCTGGAGCTCCTCGGCTGCTGA"},{"entry":["Codon optimized kappa-casein coding sequence","11"]},{"entry":"GAGGTCCAAAACCAGAAGCAGCCCGCCTGCCACGAGAACGACGAGCGCCC"},{"entry":"CTTCTACCAGAAGACCGCACCCTACGTCCCGATGTACTACGTCCCGAACAG"},{"entry":"CTACCCCTACTACGGTACGAACCTGTACCAGCGCCGCCCGGCCATCGCTAT"},{"entry":"CAACAACCCCTACGTCCCCCGGACCTACTACGCGAACCCGGCCGTGGTGC"},{"entry":"GGCCCCACGCGCAGATCCCGCAGCGGCAGTACCTGCCAAACAGCCACCCC"},{"entry":"CCCACCGTGGTGCGGCGGCCCAACCTCCACCCGAGCTTCATCGCTATCCCC"},{"entry":"CCCAAGAAGATCCAGGACAAGATCATCATCCCGACCATCAACACCATCGCCA"},{"entry":"CCGTGGAGCCGACGCCAGCCCCCGCGACCGAGCCCACGGTGGACAGCGT"},{"entry":"CGTGACCCCAGAGGCGTTCTCCGAATCGATCATCACCTCCACCCCCGAGAC"},{"entry":"CACCACGGTGGCCGTCACGCCGCCGACGGCATGA"},{"entry":["Codon optimized haptocorrin coding sequence","12"]},{"entry":"GAGATCTGCGAGGTCTCCGAGGAGAACTACATCCGCCTCAAGCCCCTCCTG"},{"entry":"AACACCATGATCCAGAGCAACTACAACCGGGGCACGTCGGCCGTGAACGTC"},{"entry":"GTGCTCTCCCTGAAGCTCGTGGGCATCCAGATCCAGACCCTCATGCAGAAG"},{"entry":"ATGATCCAGCAGATCAAGTACAACGTGAAGAGCCGCCTCTCGGACGTGTCC"},{"entry":"AGCGGCGAGCTGGCGCTCATCATCCTCGCGCTCGGCGTGTGCCGGAACGC"},{"entry":"GGAGGAGAACCTCATCTACGACTACCACCTCACGGACAAGCTGGAGAACAA"},{"entry":"GTTCCAGGCCGAGATCGAGAACATGGAGGCCCACAACGGCACCCCGCTGA"},{"entry":"CCAACTACTACCAGCTCAGCCTGGACGTCCTCGCGCTCTGCCTGTTCAACG"},{"entry":"GGAACTACTCCACCGCCGAGGTGGTCAACCACTTCACCCCCGAGAACAAGA"},{"entry":"ACTACTACTTCGGCTCGCAGTTCTCCGTGGACACCGGGGCCATGGCCGTCC"},{"entry":"TGGCCCTCACCTGCGTGAAGAAGTCCCTCATCAACGGCCAGATCAAGGCCG"},{"entry":"ACGAGGGCTCCCTGAAGAACATCTCGATCTACACCAAGAGCCTCGTGGAGA"},{"entry":"AGATCCTCAGCGAGAAGAAGGAGAACGGGCTGATCGGCAACACCTTCTCGA"},{"entry":"CCGGCGAGGCGATGCAGGCCCTGTTCGTGAGCAGCGACTACTACAACGAG"},{"entry":"AACGACTGGAACTGCCAGCAGACCCTCAACACGGTCCTGACCGAGATCAGC"},{"entry":"CAGGGCGCGTTCAGCAACCCCAACGCCGCCGCCCAGGTCCTGCCGGCCCT"},{"entry":"GATGGGCAAGACCTTCCTCGACATCAACAAGGACAGCTCCTGCGTGTCCGC"},{"entry":"GAGCGGCAACTTCAACATCTCCGCCGACGAGCCGATCACGGTGACGCCGC"},{"entry":"CCGACAGCCAGTCGTACATCTCCGTGAACTACAGCGTGCGGATCAACGAGA"},{"entry":"CCTACTTCACGAACGTGACGGTCCTCAACGGCTCGGTCTTCCTGAGCGTGA"},{"entry":"TGGAGAAGGCGCAGAAGATGAACGACACGATCTTCGGCTTCACGATGGAGG"},{"entry":"AGCGCAGCTGGGGCCCCTACATCACCTGCATCCAGGGCCTCTGCGCCAACA"},{"entry":"ACAACGACCGCACCTACTGGGAGCTGCTGAGCGGCGGCGAGCCGCTGAGC"},{"entry":"CAGGGGGCCGGCAGCTACGTGGTCCGCAACGGCGAGAACCTGGAGGTCCG"},{"entry":"GTGGAGCAAGTACTGA"},{"entry":["Codon optimized lactoperoxidase coding sequence","13"]},{"entry":"CAAACGACCCGGACGTCGGCGATCTCCGACACGGTCTCGCAGGCCAAGGT"},{"entry":"GCAAGTCAACAAGGCATTCCTGGATTCGCGCACGCGGCTGAAGACCGCGAT"},{"entry":"GTCGTCCGAGACCCCGACGAGCCGGCAGCTGAGCGAGTACCTCAAGCACG"},{"entry":"CGAAGGGGCGGACGCGCACCGCCATCCGCAATGGCCAAGTGTGGGAGGAA"},{"entry":"TCCCTGAAGCGGCTGCGGCAGAAGGCGTCGCTCACCAACGTGACCGACCC"},{"entry":"GTCCCTCGACCTGACCAGCCTCTCCCTGGAGGTCGGCTGCGGCGCCCCGG"},{"entry":"CGCCCGTCGTGCGCTGCGACCCCTGCTCGCCATACCGCACGATCACGGGC"},{"entry":"GACTGCAACAACCGGCGGAAGCCGGCACTGGGGGCTGCGAACCGCGCCCT"},{"entry":"CGCGCGCTGGCTCCCCGCCGAGTACGAGGACGGCCTCAGCCTCCCCTTCG"},{"entry":"GTTGGACCCCCGGCAAGACGCGCAACGGCTTCCCGCTCCCGCTCGCTCGC"},{"entry":"GAGGTCAGCAACAAGATCGTCGGTTACCTGAACGAGGAGGGGGTCCTCGAC"},{"entry":"CAAAACCGCTCCCTCCTCTTCATGCAGTGGGGGCAGATCGTGGACCACGAC"},{"entry":"CTGGACTTCGCCCCGGACACGGAGCTGGGCTCCAGCGAGTACAGCAAGAC"},{"entry":"CCAGTGCGACGAATACTGCATCCAGGGCGACAACTGCTTCCCGATCATGTT"},{"entry":"CCCCCCGAACGACCCGAAGGCGGGCACCCAGGGCAAGTGCATGCCGTTCT"},{"entry":"TCCGGGCAGGCTTCGTCTGCCCGACCCCCCCGTACAAGTCCCTCGCGCGC"},{"entry":"GAGCAGATCAACGCGCTCACGTCCTTCCTCGACGCCAGCTTCGTCTACAGC"},{"entry":"AGCGAGCCGTCCCTCGCCAGCCGCCTCCGCAACCTCAGCAGCCCCCTCGG"},{"entry":"CCTCATGGCGGTCAACCAGGAGGTGTCGGACCACGGCCTCCCATACCTGCC"},{"entry":"GTACGACAGCAAGAAGCCGTCCCCCTGCGAGTTCATCAACACCACCGCGCG"},{"entry":"CGTCCCGTGCTTCCTCGCCGGCGATTCGCGGGCGAGCGAGCACATCCTCC"},{"entry":"TCGCCACGAGCCACACCCTGTTCCTCCGCGAGCACAACCGCCTCGCCCGG"},{"entry":"GAGCTGAAGCGCCTCAACCCGCAGTGGGACGGCGAGAAGCTCTACCAGGA"},{"entry":"GGCCCGGAAGATCCTCGGCGCTTTCGTCCAGATCATCACCTTCCGGGACTA"},{"entry":"CCTCCCCATCCTGCTCGGTGACCACATGCAGAAGTGGATCCCCCCCTACCA"},{"entry":"AGGCTACTCCGAGAGCGTGGACCCGCGCATCTCCAACGTCTTCACGTTCGC"},{"entry":"GTTCCGCTTCGGGCACCTGGAGGTGCCGTCGTCGATGTTCCGCCTCGACGA"},{"entry":"GAACTACCAGCCCTGGGGCCCAGAGCCGGAGCTGCCGCTCCACACCCTGT"},{"entry":"TCTTCAACACCTGGCGGATGGTCAAGGACGGCGGCATCGACCCGCTCGTGC"},{"entry":"GCGGGCTCCTGGCTAAGAAGTCGAAGCTCATGAAGCAGAACAAGATGATGA"},{"entry":"CCGGCGAGCTGCGCAACAAGCTGTTCCAGCCCACCCACCGCATCCACGGG"},{"entry":"TTCGACCTGGCTGCAATCAACACCCAGCGGTGCCGCGACCACGGCCAGCC"},{"entry":"CGGCTACAACTCGTGGCGCGCGTTCTGCGACCTCTCCCAGCCACAGACGCT"},{"entry":"GGAGGAGCTCAACACCGTGCTCAAGAGCAAGATGCTCGCCAAGAAGCTGCT"},{"entry":"CGGGCTCTACGGCACGCCCGACAACATCGACATCTGGATCGGGGCCATCG"},{"entry":"CGGAGCCGCTCGTGGAGCGCGGGCGCGTCGGCCCGCTGCTCGCGTGCCT"},{"entry":"CCTGGGCAAGCAATTCCAACAGATCCGCGACGGGGACCGGTTCTGGTGGG"},{"entry":"AGAACCCCGGCGTGTTCACCAACGAGCAGAAGGATTCGCTCCAAAAGATGA"},{"entry":"GCTTCTCCCGCCTGGTGTGCGACAACACCCGCATCACCAAGGTCCCGCGCG"},{"entry":"ACCCATTCTGGGCCAACTCCTACCCGTACGACTTCGTGGACTGCTCCGCCA"},{"entry":"TCGACAAGCTCGACCTGTCCCCCTGGGCATCGGTGAAGAACTGA"},{"entry":["Codon optimized alpha-1-antitrypsin coding sequence","14"]},{"entry":"GAGGACCCGCAGGGCGACGCCGCCAGAAGACCCGACACCAGCCACCACGA"},{"entry":"CCAGGACCACCCGACGTTCAACAAGATCACCCCGAATTTGGCCGAATTCGC"},{"entry":"CTTCAGCCTGTACCGCCAGCTCGCGCACCAGTCCAACTCCACCAACATCTTC"},{"entry":"TTCAGCCCGGTGAGCATCGCCACCGCCTTCGCCATGCTGTCCCTGGGTACC"},{"entry":"AAGGCGGACACCCACGACGAGATCCTCGAAGGGCTGAACTTCAACCTGACG"},{"entry":"GAGATCCCGGAGGCGCAGATCCACGAGGGCTTCCAGGAGCTGCTCAGGAC"},{"entry":"GCTCAACCAGCCGGACTCCCAGCTCCAGCTCACCACCGGCAACGGGCTCTT"},{"entry":"CCTGTCCGAGGGCCTCAAGCTCGTCGATAAGTTCCTGGAGGACGTGAAGAA"},{"entry":"GCTCTACCACTCCGAGGCGTTCACCGTCAACTTCGGGGACACCGAGGAGGC"},{"entry":"CAAGAAGCAGATCAACGACTACGTCGAGAAGGGGACCCAGGGCAAGATCGT"},{"entry":"GGACCTGGTCAAGGAATTGGACAGGGACACCGTCTTCGCGCTCGTCAACTA"},{"entry":"CATCTTCTTCAAGGGCAAGTGGGAGCGCCCGTTCGAGGTGAAGGACACCGA"},{"entry":"GGAGGAGGACTTCCACGTCGACCAGGTCACCACCGTCAAGGTCCCGATGAT"},{"entry":"GAAGAGGCTCGGCATGTTCAACATCCAGCACTGCAAGAAGCTCTCCAGCTG"},{"entry":"GGTGCTCCTCATGAAGTACCTGGGGAACGCCACCGCCATCTTCTTCCTGCC"},{"entry":"GGACGAGGGCAAGCTCCAGCACCTGGAGAACGAGCTGACGCACGACATCA"},{"entry":"TCACGAAGTTCCTGGAGAACGAGGACAGGCGCTCCGCTAGCCTCCACCTCC"},{"entry":"CGAAGCTGAGCATCACCGGCACGTACGACCTGAAGAGCGTGCTGGGCCAG"},{"entry":"CTGGGCATCACGAAGGTCTTCAGCAACGGCGCGGACCTCTCCGGCGTGAC"},{"entry":"GGAGGAGGCCCCCCTGAAGCTCTCCAAGGCCGTGCACAAGGCGGTGCTCA"},{"entry":"CGATCGACGAGAAGGGGACGGAAGCTGCCGGGGCCATGTTCCTGGAGGCC"},{"entry":"ATCCCCGTGTCCATCCCGCCCGAGGTCAAGTTCAACAAGCCCTTCGTCTTCC"},{"entry":"TGATGATCGAGCAGAACACGAAGAGCCCCCTCTTCATGGGGAAGGTCGTCA"},{"entry":"ACCCCACGCAGAAGTGA"},{"entry":"Codon optimized immunoglobulin-A coding sequence"},{"entry":["Rice Gt1 promoter and Gt1 leader coding sequence","15"]},{"entry":"CATGAGTAATGTGTGAGCATTATGGGACCACGAAATAAAAAGAACATTTTGAT"},{"entry":"GAGTCGTGTATCCTCGATGAGCCTCAAAAGTTCTCTCACCCCGGATAAGAAA"},{"entry":"CCCTTAAGCAATGTGCAAAGTTTGCATTCTCCACTGACATAATGCAAAATAAG"},{"entry":"ATATCATCGATGACATAGCAACTCATGCATCATATCATGCCTCTCTCAACCTA"},{"entry":"TTCATTCCTACTCATCTACATAAGTATCTTCAGCTAAATGTTAGAACATAAACC"},{"entry":"CATAAGTCACGTTTGATGAGTATTAGGCGTGACACATGACAAATCACAGACT"},{"entry":"CAAGCAAGATAAAGCAAAATGATGTGTACATAAAACTCCAGAGCTATATGTCA"},{"entry":"TATTGCAAAAAGAGGAGAGCTTATAAGACAAGGCATGACTCACAAAAATTCA"},{"entry":"CTTGCCTTTCGTGTCAAAAAGAGGAGGGCTTTACATTATCCATGTCATATTGC"},{"entry":"AAAAGAAAGAGAGAAAGAACAACACAATGCTCGTCAATTATACATATCTGTA"},{"entry":"TGTCCATCATTATTCATCCACCTTTCGTGTACCACACTTCATATATCATAAGA"},{"entry":"GTCACTTCACGTCTGGACATTAACAAACTCTATCTTAACATTTAGATGCAAGA"},{"entry":"GCCTTTATCTCACTATAAATGCACGATGATTTCTCATTGTTTCTCACAAAAAG"},{"entry":"CGGCCGCTTCATTAGTCCTACAACAACATGGCATCCATAAATCGCCCCATAG"},{"entry":"TTTTCTTCACAGTTTGCTTGTTCCTCTTGTGCGATGGCTCCCTAGCC"},{"entry":"Codon optimized alpha-1-antitrypsin coding sequence"},{"entry":["Rice Glb promoter and Gt1 leader coding sequence","16"]},{"entry":"CTGCAGGGAGGAGAGGGGAGAGATGGTGAGAGAGGAGGAAGAAGAGGAG"},{"entry":"GGGTGACAATGATATGTGGGGCATGTGGGCACCCAATTTTTTAATTCATTCT"},{"entry":"TTTGTTGAAACTGACATGTGGGTCCCATGAGATTTATTATTTTTCGGATCGAA"},{"entry":"TCGCCACGTAAGCGCTACGTCAATGCTACGTCAGATGAAGACCGAGTCAAAT"},{"entry":"TAGCCACGTAAGCGCCACGTCAGCCAAAACCACCATCCAAACCGCCGAGGG"},{"entry":"ACCTCATCTGCACTGGTTTTGATAGTTGAGGGACCCGTTGTATCTGGTTTTTC"},{"entry":"GATTGAAGGACGAAAATCAAATTTGTTGACAAGTTAAGGGACCTTAAATGAA"},{"entry":"CTTATTCCATTTCAAAATATTCTGTGAGCCATATATACCGTGGGCTTCCAATC"},{"entry":"CTCCTCAAATTAAAGGGCCTTTTTAAAATAGATAATTGCCTTCTTTCAGTCAC"},{"entry":"CCATAAAAGTACAAAACTACTACCAACAAGCAACATGCGCAGTTACACACATT"},{"entry":"TTCTGCACATTTCCGCCACGTCACAAAGAGCTAAGAGTTATCCCTAGGACAA"},{"entry":"TCTCATTAGTGTAGATACATCCATTAATCTTTTATCAGAGGCAAACGTAAAGC"},{"entry":"CGCTCTTTATGACAAAAATAGGTGACACAAAAGTGTTATCTGCCACATACATA"},{"entry":"ACTTCAGAAATTACCCAACACCAAGAGAAAAATAAAAAAAAATCTTTTTGCAA"},{"entry":"GCTCCAAATCTTGGAAACCTTTTTCACTCTTTGCAGCATTGTACTCTTGCTCT"},{"entry":"TTTTCCAACCGATCCATGTCACCCTCAAGCTTCTACTTGATCTACACGAAGCT"},{"entry":"CACCGTGCACACAACCATGGCCACAAAAACCCTATAAAACCCCATCCGATCG"},{"entry":"CCATCATCTCATCATCAGTTCATTACCAACAAACAAAAGAGGAAAAAAAACAT"},{"entry":"ATACACTTCTAGTGATTGTCTGATTGATCATCAATCTAGAGGCGGCCGCATG"},{"entry":"GCTAGCAAGGTCGTCTTCTTCGCGGCGGCGCTCATGGCGGCCATGGTGGC"},{"entry":"CATCTCCGGC"},{"entry":"Rice Gt1 promoter and Gt1 leader coding sequence"},{"entry":["Other monocot maturation specific promoter sequences","17-23"]},{"entry":"Seq #17 Bx7 promoter seq"},{"entry":"CTGCAGGCCAGGGAAAGACAATGGACATGCAAAGAGGTAGGGGCAGGGAA"},{"entry":"GAAACACTTGGAGATCATAGAAGAACATAAGAGGTTAAACATAGGAGGGCAT"},{"entry":"AATGGACAATTAAATCTACATTAATTGAACTCATTTGGGAAGTAAACAAAATC"},{"entry":"CATATTCTGGTGTAAATCAAACTATTTGACGCGGATTTAACTAAGATCCTATGT"},{"entry":"TAATTTTAGACATGACTGGCCAAAGGTTTCAGTTAGTTCATTTGTCACGGAAA"},{"entry":"GGTGTTTTCATAAGTCCAAAACTCTACCAACTTTTTTGCACGTCATAGCATAG"},{"entry":"ATAGATGTTGTGAGTCATTGGATAGATATTGTGAGTCAGCATGGATTTGTGTT"},{"entry":"GCCTGGAAATCCAACTAAATGACAAGCAACAAAACCTGAAATGGGCTTTAGG"},{"entry":"AGAGATGGTTTATCAATTTACATGTTCCATGCAGGCTACCTTCCACTACTCGA"},{"entry":"CATGGTTAGAAGTTTTGAGTGCCGCATATTTGCGGAAGCAATGGCACTACTC"},{"entry":"GACATGGTTAGAAGTTTTGAGTGCCGCATATTTGCGGAAGCAATGGCTAACA"},{"entry":"GATACATATTCTGCCAAACCCCAAGAAGGATAATCACTCCTCTTAGATAAAAA"},{"entry":"GAACAGACCAATGTACAAACATCCACACTTCTGCAAACAATACACCAGAACT"},{"entry":"AGGATTAAGCCCATTACGTGGCTTTAGCAGACCGTCCAAAAATCTGTTTTGC"},{"entry":"AAGCACCAATTGCTCCTTACTTATCCAGCTTCTTTTGTGTTGGCAAACTGCCC"},{"entry":"TTTTCCAACCGATTTTGTTTCTTCTCACGCTTTCTTCATAGGCTAAACTAACCT"},{"entry":"CGGCGTGCACACAACCATGTCCTGAACCTTCACCTCGTCCCTATAAAAGCCC"},{"entry":"ATCCAACCTTACAATCTCATCATCACCCACAACACCGAGCACCCCAATCTAC"},{"entry":"AGATCAATTCACTGACAGTTCACTGATCTAGA"},{"entry":"Seq #18 Glub-2 promoter seq"},{"entry":"CTGCAGTAATGGATACCTAGTAGCAAGCTAGCTTAAACAAATCTAAATTCCAA"},{"entry":"TCTGTTCGTAAACGTTTTCTCGATCGCAATTTTGATCAAAACTATTGAAAACC"},{"entry":"TCAATTAAACCATTCAAAATTTTTAATATACCCAACAAGAGCGTCCAAACCAA"},{"entry":"ATATGTAAATATGGATGTCATGATAATTGACTTATGACAATGTGATTATTTCAT"},{"entry":"CAAGTCTTTAAATCATTAATTCTAGTTGAAGGTTTATGTTTTCTTATGCTAAAG"},{"entry":"GGTTATGTTTATATAAGAATATTAAAGAGCAAATTGCAATAGATCAACACAAC"},{"entry":"AAATTTGAATGTTTCCAGATGTGTAAAAATATCCAAATTAATTGTTTTAAAATA"},{"entry":"GTTTTAAGAAGGATCTGATATGCAAGTTTGATAGTTAGTAAACTGCAAAAGGG"},{"entry":"CTTATTACATGGAAAATTCCTTATTGAATATGTTTCATTGACTGGTTTATTTTA"},{"entry":"CATGACAACAAAGTTACTAGTATGTCAATAAAAAAATACAAGGTTACTTGTCA"},{"entry":"ATTGTATTGTGCCAAGTAAAGATGACAACAAACATACAAATTTATTTGTTCTTT"},{"entry":"TATAGAAACACCTAACTTATCAAGGATAGTTGGCCACGCAAAAATGACAACAT"},{"entry":"ACTTTACAATTGTATCATCATAAAGATCTTATCAAGTATAAGAACTTTATGGTG"},{"entry":"ACATAAAAAATAATCACAAGGGCAAGACACATACTAAAAGTATGGACAGAAAT"},{"entry":"TTCTTAACAAACTCCATTTGTTTTGTATCCAAAAGCATAAGAAATGAGTCATG"},{"entry":"GCTGAGTCATGATATGTAGTTCAATCTTGCAAAATTGCCTTTTTGTTAAGTATT"},{"entry":"GTTTTAACACTACAAGTCACATATTGTCTATACTTGCAACAAACACTATTACC"},{"entry":"GTGTATCCCAAGTGGCCTTTTCATTGCTATATAAACTAGCTTGATCGGTCTTT"},{"entry":"CAACTCACATCAATTAGCTTAAGTTTCCATTAGCAACTGCTAATAGCT"},{"entry":"Seq#19 Gt3 promoter seq"},{"entry":"CTGCAGTGTAAGTGTAGCTTCTTATAGCTTAGTGCTTTACTATCTTCACAAGC"},{"entry":"ACATGCTATAGTATTGTTCCAAGATGAAAGAATAATTCATCCTTGCTACCAAC"},{"entry":"TTGCATGATATTATATTTGTGAATATCCTATCTCTTGGCTTATAATGAAATGTG"},{"entry":"CTGCTGGGTTATTCTGACCATGGTATTTGAGAGCCTTTGTATAGCTGAAACC"},{"entry":"AACGTATATCGAGCATGGAACAGAGAACAAAATGCAAGGATTTTTTTATTCTG"},{"entry":"GTTCATGCCCTGGATGGGTTAATATCGTGATCATCAAAAAAGATATGCATAAA"},{"entry":"ATTAAAGTAATAAATTTGCTCATAAGAAACCAAAACCAAAAGCACATATGTCC"},{"entry":"TAAACAAACTGCATTTTGTTTGTCATGTAGCAATACAAGAGATAATATATGAC"},{"entry":"GTGGTTATGACTTATTCACTTTTTGTGACTCCAAAATGTAGTAGGTCTAACTG"},{"entry":"ATTGTTTAAAGTGATGTCTTACTGTAGAAGTTTCATCCCAAAAGCAATCACTA"},{"entry":"AAGCAACACACACGTATAGTCCACCTTCACGTAATTCTTTGTGGAAGATAACA"},{"entry":"AGAAGGCTCACTGAAAAATAAAAGCAAAGAAAAGGATATCAAACAGACCATT"},{"entry":"GTGCATCCCATTGATCCTTGTATGTCTATTTATCTATCCTCCTTTTGTGTACCT"},{"entry":"TACTTCTATCTAGTGAGTCACTTCATATGTGGACATTAACAAACTCTATCTTAA"},{"entry":"CATCTAGTCGATCACTACTTTACTTCACTATAAAAGGACCAACATATATCATC"},{"entry":"CATTTCTCACAAAAGCATTGAGTTCAGTCCCACAAAATCTAGA"},{"entry":"Seq #20 Glub-1 promoter seq"},{"entry":"CTGCAGAGATATGGATTTTCTAAGATTAATTGATTCTCTGTCTAAAGAAAAAA"},{"entry":"AGTATTATTGAATTAAATGGAAAAAGAAAAAGGAAAAAGGGGATGGCTTCTG"},{"entry":"CTTTTTGGGCTGAAGGCGGCGTGTGGCCAGCGTGCTGCGTGCGGACAGCG"},{"entry":"AGCGAACACACGACGGAGCAGCTACGACGAACGGGGGACCGAGTGGACCG"},{"entry":"GACGAGGATGTGGCCTAGGACGAGTGCACAAGGCTAGTGGACTCGGTCCC"},{"entry":"CGCGCGGTATCCCGAGTGGTCCACTGTCTGCAAACACGATTCACATAGAGC"},{"entry":"GGGCAGACGCGGGAGCCGTCCTAGGTGCACCGGAAGCAAATCCGTCGCCT"},{"entry":"GGGTGGATTTGAGTGACACGGCCCACGTGTAGCCTCACAGCTCTCCGTGGT"},{"entry":"CAGATGTGTAAAATTATCATAATATGTGTTTTTCAAATAGTTAAATAATATATAT"},{"entry":"AGGCAAGTTATATGGGTCAATAAGCAGTAAAAAGGCTTATGACATGGTAAAA"},{"entry":"TTACTTACACCAATATGCCTTACTGTCTGATATATTTTACATGACAACAAAGTT"},{"entry":"ACAAGTACGTCATTTAAAAATACAAGTTACTTATCAATTGTAGTGTATCAAGTA"},{"entry":"AATGACAACAAACCTACAAATTTGCTATTTTGAAGGAACACTTAAAAAAATCA"},{"entry":"ATAGGCAAGTTATATAGTCAATAAACTGCAAGAAGGCTTATGACATGGAAAAA"},{"entry":"TTACATACACCAATATGCTTTATTGTCCGGTATATTTTACAAGACAACAAAGTT"},{"entry":"ATAAGTATGTCATTTAAAAATACAAGTTACTTATCAATTGTCAAGTAAATGAAA"},{"entry":"ACAAACCTACAAATTTGTTATTTTGAAGGAACACCTAAATTATCAAATATAGCT"},{"entry":"TGCTACGCAAAATGACAACATGCTTACAAGTTATTATCATCTTAAAGTTAGAC"},{"entry":"TCATCTTCTCAAGCATAAGAGCTTTATGGTGCAAAAACAAATATAATGACAAG"},{"entry":"GCAAAGATACATACATATTAAGAGTATGGACAGACATTTCTTTAACAAACTCC"},{"entry":"ATTTGTATTACTCCAAAAGCACCAGAAGTTTGTCATGGCTGAGTCATGAAATG"},{"entry":"TATAGTTCAATCTTGCAAAGTTGCCTTTCCTTTTGTACTGTGTTTTAACACTAC"},{"entry":"AAGCCATATATTGTCTGTACGTGCAACAAACTATATCACCATGTATCCCAAGA"},{"entry":"TGCTTTTTTATTGCTATATAAACTAGCTTGGTCTGTCTTTGAACTCACATCAAT"},{"entry":"TAGCTTAAGTTTCCATAAGCAAGTACAAATAGCTCTAGA"},{"entry":"Seq #21 Rice prolamin promoter seq"},{"entry":"CTGCAGCATCGGCTTAGGTGTAGCAACACGACTTTATTATTATTATTATTATT"},{"entry":"ATTATTATTATTTTACAAAAATATAAAATAGATCAGTCCCTCACCACAAGTAGA"},{"entry":"GCAAGTTGGTGAGTTATTGTAAAGTTCTACAAAGCTAATTTAAAAGTTATTGC"},{"entry":"ATTAACTTATTTCATATTACAAACAAGAGTGTCAATGGAACAATGAAAACCAT"},{"entry":"ATGACATACTATAATTTTGTTTTTATTATTGAAATTATATAATTCAAAGAGAATA"},{"entry":"AATCCACATAGCCGTAAAGTTCTACATGTGGTGCATTACCAAAATATATATAG"},{"entry":"CTTACAAAACATGACAAGCTTAGTTTGAAAAATTGCAATCCTTATCACATTGA"},{"entry":"CACATAAAGTGAGTGATGAGTCATAATATTATTTTTCTTGCTACCCATCATGT"},{"entry":"ATATATGATAGCCACAAAGTTACTTTGATGATGATATCAAAGAACATTTTTAG"},{"entry":"GTGCACCTAACAGAATATCCAAATAATATGACTCACTTAGATCATAATAGAGC"},{"entry":"ATCAAGTAAAACTAACACTCTAAAGCAACCGATGGGAAAGCATCTATAAATAG"},{"entry":"ACAAGCACAATGAAAATCCTCATCATCCTTCACCACAATTCAAATATTATAGT"},{"entry":"TGAAGCATAGTAGTAGAATCCAACAACAATCTAGAG"},{"entry":"Seq #22 Rice cysteine peptidase promoter seq"},{"entry":"CCAGGCTTCATCCTAACCATTACAGGCAAGATGTTGTATGAAGAAGGGCGAA"},{"entry":"CATGCAGATTGTTAAACTGACACGTGATGGACAAGAATGACCGATTGGTGAC"},{"entry":"CGGTCTGACAATGGTCATGTCGTCAGCAGACAGCCATCTCCCACGTCGCGC"},{"entry":"CTGCTTCCGGTGAAAGTGGAGGTAGGTATGGGCCGTCCCGTCAGAAGGTGA"},{"entry":"TTCGGATGGCAGCGATACAAATCTCCGTCCATTAATGAAGAGAAGTCAAGTT"},{"entry":"GAAAGAAAGGGAGGGAGAGATGGTGCATGTGGGATCCCCTTGGGATATAAA"},{"entry":"AGGAGGACCTTGCCCACTTAGAAAGGAGAGGAGAAAGCAATCCCAGAAGAA"},{"entry":"TCGGGGGCTGACTGGCACTTTGTAGCTTCTTCATACGCGAATCCACCAAAAC"},{"entry":"ACAGGAGTAGGGTATTACGCTTCTCAGCGGCCCGAACCTGTATACATCGCC"},{"entry":"CGTGTCTTGTGTGTTTCCGCTCTTGCGAACCTTCCACAGATTGGGAGCTTAG"},{"entry":"AACCTCACCCAGGGCCCCCGGCCGAACTGGCAAAGGGGGGCCTGCGCGGT"},{"entry":"CTCCCGGTGAGGAGCCCCACGCTCCGTCAGTTCTAAATTACCCGATGAGAA"},{"entry":"AGGGAGGGGGGGGGGGGAAATCTGCCTTGTTTATTTACGATCCAACGGATT"},{"entry":"TGGTCGACACCGATGAGGTGTCTTACCAGTTACCACGAGCTAGATTATAGTA"},{"entry":"CTAATTACTTGAGGATTCGGTTCCTAATTTTTTACCCGATCGACTTCGCCATG"},{"entry":"GAAAATTTTTTATTCGGGGGAGAATATCCACCCTGTTTCGCTCCTAATTAAGA"},{"entry":"TAGGAATTGTTACGATTAGCAACCTAATTCAGATCAGAATTGTTAGTTAGCGG"},{"entry":"CGTTGGATCCCTCACCTCATCCCATCCCAATTCCCAAACCCAAACTCCTCTT"},{"entry":"CCAGTCGCCGACCCAAACACGCATCCGCCGCCTATAAATCCCACCCGCATC"},{"entry":"GAGCCTATCAAGCCCAAAAAACCACAAACCAAACGAAGAAGGAAAAAAAAAG"},{"entry":"GAGGAAAAGAAAAGAGGAGGAAAGCGAAGAGGTTGGAGAGAGACGCTCGT"},{"entry":"CTCCACGTCGCCGCC "},{"entry":"Seq #23 Barley D-Hordein promoter"},{"entry":"CTTCGAGTGCCCGCCGATTTGCCAGCAATGGCTAACAGACACATATTCTGCC"},{"entry":"AAAACCCCAGAACAATAATCACTTCTCGTAGATGAAGAGAACAGACCAAGAT"},{"entry":"ACAAACGTCCACGCTTCAGCAAACAGTACCCCAGAACTAGGATTAAGCCGAT"},{"entry":"TACGCGGCTTTAGCAGACCGTCCAAAAAAACTGTTTTGCAAAGCTCCAATTC"},{"entry":"CTCCTTGCTTATCCAATTTCTTTTGTGTTGGCAAACTGCACTTGTCCAACCGA"},{"entry":"TTTTGTTCTTCCCGTGTTTCTTCTTAGGCTAACTAACACAGCCGTGCACATAG"},{"entry":"CCATGGTCCGGAATCTTCACCTCGTCCCTATAAAAGCCCAGCCAATCTCCAC"},{"entry":"AATCTCATCATCACCGAGAACACCGAGAACCACAAAACTAGAGATCAATTCA"},{"entry":"TTGACAGTCCACCG"},{"entry":"Rice Glb promoter and Gt1 leader coding sequence"},{"entry":["Other storage body leader sequences","24-30"]},{"entry":"Bx7 #24 bx7 signal peptide seq"},{"entry":"ATGGCTAAGCGCCTGGTCCTCTTTGCGGCAGTAGTCGTCGCCCTCGTGGCT"},{"entry":"CTCACCGCC"},{"entry":"Seq #25 Glub-2 signal peptide seq"},{"entry":"ATGGCAACTACCATTTTCTCTCGTTTTTCTATATACTTTTGTGCTATGCTATTA"},{"entry":"TGCCAGGGTTCTATGGCC"},{"entry":"Seq #26 Gt3 signal peptide seq"},{"entry":"ATGTGGACATTAACAAACTCTATCTTAACATCTAGTCGATCACTACTTTACTTC"},{"entry":"ACTATAAAAGGACCAACATATATCATCCATT"},{"entry":"Seq #27 Glub-1 signal peptide seq"},{"entry":"ATGGCGAGTTCCGTTTTCTCTCGGTTTTCTATATACTTTTGTGTTCTTCTATTA"},{"entry":"TGCCATGGTTCTATGGCC"},{"entry":"Seq #28 prolamin signal peptide seq"},{"entry":"ATGAAGATCATTTTCGTATTTGCTCTCCTTGCTATTGTTGCATGCAACGCTTC"},{"entry":"TGCACGGTTTGATGCT"},{"entry":"Seq #29 Rice cysteine peptidase signal peptide seq"},{"entry":"ATGGCCGCCCGCGCCGCCGCCGCCGCGTTCCTGCTGCTGCTCATCGTCGT"},{"entry":"TGGTCACCGCGCC"},{"entry":"Seq #30 D-Holdein signal peptide"},{"entry":"ATGGCTAAGCGGCTGGTCCTCTTTGTGGCGGTAATCGTCGCCCTCGTGGCT"},{"entry":"CTCACCACCGCCOther monocot maturation specific promoter"},{"entry":"sequences"},{"entry":["O2 transcription factor sequence","31"]},{"entry":"ATGGAGCACGTCATCTCAATGGAGGAGATCCTCGGGCCCTTCTGGGAGCTG"},{"entry":"CTACCACCGCCAGCGCCAGAGCCAGAGCGAGAGCAGCCTCCGGTAACCGG"},{"entry":"CATCGTCGTCGGCAGTGTCATAGACGTTGCTGCTGCTGGTCATGGTGACGG"},{"entry":"GGACATGATGGATCAGCAGCACGCCACAGAGTGGACCTTTGAGAGGTTACT"},{"entry":"AGAAGAGGAGGCTCTGACGACAAGCACACCGCCGCCGGTGGTGGTGGTGC"},{"entry":"CGAACTCTTGTTGCTCAGGCGCCCTAAATGCTGACCGGCCGCCGGTGATGG"},{"entry":"AAGAGGCGGTAACTATGGCGCCTGCGGCGGTGAGTAGTGCCGTAGTAGGT"},{"entry":"GACCCCATGGAGTACAATGCCATACTGAGGAGGAAGCTGGAGGAGGACCTC"},{"entry":"GAGGCCTTCAAAATGTGGAGGGCGGCCTCCAGTGTTGTGACCTCAGATCAA"},{"entry":"CGTTCTCAAGGCTCAAACAATCACACTGGAGGTAGCAGCATCAGGAATAATC"},{"entry":"CAGTGCAGAACAAGCTGATGAACGGCGAAGATCCAATCAACAATAACCACG"},{"entry":"CTCAAACTGCAGGCCTTGGCGTGAGGCTTGCTACTAGCTCTTCCTCGAGAG"},{"entry":"ATCCTTCACCATCAGACGAAGACATGGACGGAGAAGTAGAGATTCTGGGGT"},{"entry":"TCAAGATGCCTACCGAGGAAAGAGTGAGGAAAAGAAAGGAATCCAATAGAG"},{"entry":"AATCAGCCAGACGCTCGAGATACAGGAAAGCCGCTCACCTGAAAGAACTGG"},{"entry":"AAGACCAGGTAGCACAGCTAAAAGCCGAGAATTCTTGCCTGCTGAGGCGCA"},{"entry":"TTGCCGCTCTGAACCAGAAGTACAACGACGCTAACGTCGACAACAGGGTGC"},{"entry":"TGAGAGCGGACATGGAGACCCTAAGAGCTAAGGTGAAGATGGGAGAGGACT"},{"entry":"CTCTGAAGCGGGTGATAGAGATGAGCTCATCAGTGCCGTCGTCCATGCCCA"},{"entry":"TCTCGGCGCCGACCCCCAGCTCCGACGCTCCAGTGCCGCCGCCGCCTATC"},{"entry":"CGAGACAGCATCGTCGGCTACTTCTCCGCCACAGCCGCAGACGACGATGCT"},{"entry":"TCGGTCGGCAACGGTTTCTTGCGACTGCAAGCTCATCAAGAGCCTGCATCC"},{"entry":"ATGGTCGTCGGTGGAACTCTGAGCGCCACAGAGATGAACCGAGTAGCAGCA"},{"entry":"GCCACGCATTGCGCGGGGGCCATGGAGCACATCCAGACGGCGATGGGATC"},{"entry":"CATGCCGCCGACCTCCGCCTCCGGATCTACACCGCCGCCGCAGGATTATGA"},{"entry":"GCTGCTGGGTCCAAATGGGGCCATACACATGGACATGTATTAG"},{"entry":["PBF transcription factor sequence","32"]},{"entry":"ATGGACATGATCTCCGGCAGCACTGCAGCAACATCAACACCCCACAACAAC"},{"entry":"CAACAGGCGGTGATGTTGTCATCCCCCATTATAAAGGAGGAAGCTAGGGAC"},{"entry":"CCAAAGCAGACACGAGCCATGCCCCAAATAGGTGGCAGTGGGGAGCGTAA"},{"entry":"GCCGAGGCCGCAACTACCTGAGGCGCTCAAGTGCCCACGCTGCGACTCCA"},{"entry":"ACAACACCAAGTTTTGCTACTACAACAATTATAGCATGTCACAACCACGCTAC"},{"entry":"TTTTGCAAGGCTTGCCGCCGCTATTGGACACATGGTGGTACCCTCCGCAAT"},{"entry":"GTCCCCATTGGTGGTGGGTGTCGCAAGAACAAACATGCCTCTAGATTTGTCT"},{"entry":"TGGGCTCTCACACCTCATCGTCCTCATCTGCTACCTATGCACCATTATCCCC"},{"entry":"TAGCACCAACGCTAGCTCTAGCAATATGAGCATCAACAAACATATGATGATG"},{"entry":"GTGCCTAACATGACGATGCCTACCCCAACGACAATGGGCTTATTCCCTAATG"},{"entry":"TGCTCCCAACACTTATGCCGACAGGTGGAGGCGGGGGCTTTGACTTCACTA"},{"entry":"TGGACAACCAACATAGATCATTGTCCTTCACACCAATGTCTCTACCTAGCCA"},{"entry":"GGGGCCAGTGCCTATGCTGGCTGCAGGAGGGAGTGAGGCAACACCGTCTT"},{"entry":"TCCTAGAGATGCTGAGAGGAGGGATTTTTCATGGTAGTAGTAGCTATAACAC"},{"entry":"AGaGTCTCACGATGAGTGGTGGCAACAATGGAATGGACAAGCCATTTTCGCTG"},{"entry":"CCATCATATGGTGCAATGTGCACAAATGGGTTGAGTGGCTCAACCACTAATG"},{"entry":"ATGCCAGACAACTGGTGGGGCCTCAGCAGGATAACAAGGCCATCATGAAGA"},{"entry":"GCAGTAATAACAACAATGGTGTATCATTGTTGAACCTCTACTGGAACAAGCA"},{"entry":"CAACAACAACAACAACAACAACAACAACAACAACAACAACAACAACAACAAG"},{"entry":"GGACAATAA"},{"entry":["Reb transcription factor sequence","33"]},{"entry":"ATGGAGCGGGTGTTCTCCGTGGAGGAGATCTCCGACCCATTCTGGGTCCCG"},{"entry":"CCTCCGCCGCCGCAGTCGGCGGCGGCGGCCCAGCAGCAGGGCGGCGGCG"},{"entry":"GCGTGGCTTCGGGAGGTGGTGGTGGTGTAGCGGGGGGCGGCGGCGGCGG"},{"entry":"GAACGCGATGAACCGGTGCCCGTCGGAGTGGTACTTCCAGAAGTTTCTGGA"},{"entry":"GGAGGCGGTGCTCGATAGCCCCGTCCCGAACCCTAGCCCGAGGGCCGAAG"},{"entry":"CGGGAGGGATCAGGGGCGCAGGAGGGGTGGTGCCGGTCGATGTTAAGCA"},{"entry":"GCCGCAGCTCTCGGCGGCGGCGACGACGAGCGCGGTGGTGGACCCCGTG"},{"entry":"GAGTACAACGCGATGCTGAAGCAGAAGCTGGAGAAGGACCTCGCCGCGGT"},{"entry":"CGCCATGTGGAGGGTACAGCCATTCTCCCCCCCTCTAGTACTCGAGAGCTT"},{"entry":"ACTGAGATCGGCAATGCTAGCTACTGTTTGCATCGAATGTTTATAGGTATTTA"},{"entry":"GATCGGGCATTTCTATAGACCAATGGCGTCCATGGTCTTGCAATGCGCTCTG"},{"entry":"TTGAGTGTCGGTGGTTGGTTCGACTCATAGTATGTAGGGTTGTGCGTATGTA"},{"entry":"CAAACGGAAGCTTCATAGACCTCGGTATTGAGATTGCGATATCGATGCAACC"},{"entry":"TGCGAATTGGCGATGTAATCAGTCATATTCTTACTAAACTGCGAGACAGTGG"},{"entry":"TTTGTTTGCAATTGCAATATTTTTGTATGGGGCTGCTTAAACTGTCATTGCCT"},{"entry":"TTTTAGATTGGCAATATGTGACTTTATGCAAGTATTTGATTGGGCGGATCCAG"},{"entry":"GAACAAAAAGTTGGGGGGATTCAACATACCGAGTACACTGGCATAAACACAT"},{"entry":"CATCTCAGTATTAAACTATGCTAAAATGCTATTAAGAGACCTTTAGCACCTCT"},{"entry":"TATCTTATCAACCATGGTGAAAAAATTGAAGGGGGGACTCAGGGGGGTATCC"},{"entry":"ATGGGTCCGATGGGTGCAGGGGGGACTGAGTCCCCCCTGCACCCACGTTG"},{"entry":"AATCCGCCCTGGCATGCGTATAAGCTGTCACAGCCATTTCTAGGTGCTTGTG"},{"entry":"CTTAGTTGGGTGATGTCAGCTTAATTTGTCTTTTCTATGTCGTCATCGATTTT"},{"entry":"CTAAGAAACGAAAAATAGCCTATTTATGTGCTCCAGAATTTGATGATCCCTGG"},{"entry":"CCCTTCATTTGCTGAAATTAGCCTATTTGTTGGTTGCCCTTCAGTTTTTTCCC"},{"entry":"AGCTTATGTTGTTGCAATGTGTGGCTATGCCTCGTTTTGTGCCCTATAATTTA"},{"entry":"TTATTTGCAATTCATTTTTGTACATGACTTAAAATGACACTAGAGCAACATGCA"},{"entry":"CTGATTGGTTATCCTATAATCATTTATGTAGTTCTGTTCATTTTATCATGCTAG"},{"entry":"CTCATGTCATTTTCATCTTCAGGCCTCTGGCACAGTTCCACCTGAGCGTCCT"},{"entry":"GGAGCTGGTTCATCCTTGCTGAATGCAGATGTTTCACACATAGGCGCTCCTA"},{"entry":"ATTCCATCGGAGGTACTTATCTTATCTGGTTACATTTTCAGATTGTTATGAAA"},{"entry":"CTACCCAAATATCCTGCACAATTGCATGGGATTAAATTTTAGTTTCTTTGAAAT"},{"entry":"AGAAGTAGAGTTGTATTGCTGTCACGTCATCAAATAGTTCTGAAGCTATGAAT"},{"entry":"AAATAAGTTCCGCATTTGTTAGTGATTCTTTGAACATTAGAATTGTTATGCTTA"},{"entry":"AGTAGATAGGGTTATGTTTGTTTGGAGTTCCCTTAAATCATTTCATTGCTGAC"},{"entry":"TGCCAGCTGGCAGGAGCATTTGTTGTTGCCTTGACCATGAATGAAGACCTTC"},{"entry":"CTGTTCTGAGTGCTCACAAGAAAACATATTTTGATTAATGCACCTTGAATCCT"},{"entry":"TAGGATCTTGCAAAGATGGGCACTTAGCTTTAGAATTGAGTAGTACTTAAATA"},{"entry":"GCTGTTGTTATCATGATTTGTCCTGTAGTGAAATGTCGACAAAACAGGAATG"},{"entry":"CTACTTTTGACTTCTGATATTTCATGCCTGGCTTTACTTATGCTCTGTTTGGAA"},{"entry":"CATGGGCACATATCAGGCAATGCTACTCCAGTTCAAAACATGCTAAGTGGCC"},{"entry":"CAAGTGGGGGATCGGGCTCACAGTTGGTACAGAATGTTGATGTCCTTGTAAA"},{"entry":"GCAGCCCACCAGCTCTTCATCAAGGGAGCAGTCAGATGATGATGACATGAA"},{"entry":"GGGAGAAGCTGAGACCACTGGAACTGCAAGACCTGCTGATCAAAGATTACA"},{"entry":"ACGAAGGTGATCATTCATTGCTTCCTTGTAATATAGATTCTGTACATAATTAA"},{"entry":"CCTACCTCGTCATGCATGCATGTGTCCTATTTTCACCTTAGCCCTTTCAGTTG"},{"entry":"GATTTCCACTTTCATCCGGTAGCCTTTCAGTTTCCTATTGCATCGCATATATG"},{"entry":"ATCTTTTACCTACCATATTAGTTCTCTGTGTGCCATACTCAGTGCTTAGTGTC"},{"entry":"TCGAGCAAGAGAGGAATTTGTATGGCTATTACACGTAGCACTTTGCTCTCTA"},{"entry":"CTTGTTTATTGACATAAGCAATTTGGGATGAATTAAATCTGAGTTCACATCAT"},{"entry":"ATTCCTTATGTCACAAGTTTCTGAAACCGATTGTATCTAGTATCTGGTTGATG"},{"entry":"CACCCCCATCTTGGATTTGCAAATCAAAGTTATACTCCCTAGAGAGCTTTACC"},{"entry":"TTTCATAAAGCAATTACCCCAATAAACCACGGATTTGATAGCTATTGACTATG"},{"entry":"ATTACCAGAATTCATTTGGCAGCTATTTTCTCAATTTAAGTTTGGTATTAGTCT"},{"entry":"CAGTTGGCTGTAAAATAATGTCACGGTAGGGTACATGTATGTGCAGCATACA"},{"entry":"AGGTATGGGTGAGTTATGATATGGACAGTGTGTACACCCCACATTTGCTCAC"},{"entry":"TAAAATCAAAATATTCAAACGTCACGTGATGATATGGTGGATTGCATTATACC"},{"entry":"TTGTATTGTTTATTATGTTACTTGTGCTAGACAATAATATAGGCTGTTCTTTTG"},{"entry":"GGTGATTTTGTATGAAGATGTTGAGCAAGCACTTCTCGATATAATGCTAGTTT"},{"entry":"TGTTGACCTGTTCCAGGAAGCAATCCAATCGGGAGTCAGCCAGGCGCTCAA"},{"entry":"GAAGCAGAAAGGCAGCTCACTTGAATGAGCTGGAGGCACAGGTGTGATAGT"},{"entry":"TCACATAGTTATTTTCGATAAGACATAAAATCCTAAATTACTGGCTACTGACTT"},{"entry":"CAGTTATGGATTTACTTGTTACAGGTATCGCAATTAAGAGTCGAGAACTCCTC"},{"entry":"GCTGTTAAGGCGTCTTGCTGATGTTAACCAGAAGTACAATGATGCTGCTGTT"},{"entry":"GACAATAGAGTGCTAAAAGCAGATGTTGAGACCTTGAGAGCAAAGGTATGCT"},{"entry":"ATATATGCCTTTTGCAATATGCATCCCATGGATTGCTACTTTGGCTTGTTTCA"},{"entry":"AACTTTCAACGTGACTTGTGTACCCTGTTATTAGAAGAATAATCCCGCCTACC"},{"entry":"ATTATACTCTATAAATCACCATTTGGCCAGTCCAAACATGATTATTAAATCAG"},{"entry":"GTCAATCTGAACATTGAAATGTATCAAAAATTCGCAGGTGAAGATGGCAGAG"},{"entry":"GACTCGGTGAAGCGGGTGACAGGCATGAACGCGTTGTTTCCCGCCGCTTCT"},{"entry":"GATATGTCATCCCTCAGCATGCCATTCAACAGCTCCCCATCTGAAGCAACGT"},{"entry":"CAGACGCTGCTGTTCCCATCCAAGATGACCCGAACAATTACTTCGCTACTAA"},{"entry":"CAACGACATCGGAGGTAACAACAACTACATGCCCGACATACCTTCTTCGGCT"},{"entry":"CAGGAGGACGAGGACTTCGTCAATGGCGCTCTGGCTGCCGGCAAGATTGG"},{"entry":"CCGGCCAGCCTCGCTGCAGCGGGTGGCGAGCCTGGAGCATCTCCAGAAGA"},{"entry":"GGATGTGCGGTGGGCCGGCTTCGTCTGGGTCGACGTCCTGA"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}}}}],"BRFSUM":[{},{}],"heading":["FIELD OF THE INVENTION","REFERENCES","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","EXAMPLE 1","Expression Vectors for Generation of Transgenic Plants","EXAMPLE 2","Generation of Transgenic Plant Cells Expressing Human Milk Proteins","EXAMPLE 3","Characterization of Recombinant Human Lysozyme (rLys) Produced by Transgenic Rice Cells and Plants","EXAMPLE 4","Characterization of Recombinant Human Lactoferrin (rLF) Produced by Transgenic Rice Plants","EXAMPLE 5","Generation and Characterization of Recombinant Human \u03b1-1-antitrypsin (AAT) Produced by Transgenic Rice Plants","EXAMPLE 6","Generation and Characterization of Recombinant Proteins Produced by Transgenic Rice Plants","EXAMPLE 7","Comparison of Promoter Activity in the Expression of Lysozyme in Transgenic Rice","EXAMPLE 8","Co-transformation of Heterologous Polypeptide and Reg Gene in Transgenic Rice Plants","EXAMPLE 9","Production of Rice Extract Containing Recombinant Proteins and its use in Food"],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE FIGURES","p":[{"@attributes":{"id":"p-0144","num":"0143"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0145","num":"0144"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0146","num":"0145"},"figref":"FIG. 3","i":"E. coli "},{"@attributes":{"id":"p-0147","num":"0146"},"figref":"FIG. 4","i":"M. luteus"},"FIGS. A-D: Thermal stability of human lysozyme (\u201cHlys\u201d) and recombinant human lysozyme from transgenic rice (\u201crHLys\u201d). Lysozyme was dissolved at 100 \u03bcg\/ml in PBS. The mixtures were subjected to different temperatures for different lengths of time. At the end of each heat treatment, the remaining lysozyme activity was assessed by activity assay.","FIG. E: pH stability of Hlys and rHlys. Lysozyme was dissolved in different buffers at 100 \u03bcg\/ml. The mixture was incubated at 37\u00b0 C. for 30 min. The lysozyme activity was determined by activity assay.",{"@attributes":{"id":"p-0150","num":"0149"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0151","num":"0150"},"figref":"FIG. 7"},{"@attributes":{"id":"p-0152","num":"0151"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0153","num":"0152"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0154","num":"0153"},"figref":"FIG. 10","i":"E. coli "},{"@attributes":{"id":"p-0155","num":"0154"},"figref":"FIG. 11"},{"@attributes":{"id":"p-0156","num":"0155"},"figref":["FIG. 12","FIG. 12","FIG. 12B","FIG. 12C","FIG. 12D"],"sup":"125"},{"@attributes":{"id":"p-0157","num":"0156"},"figref":"FIG. 13"},{"@attributes":{"id":"p-0158","num":"0157"},"figref":"FIG. 14"},{"@attributes":{"id":"p-0159","num":"0158"},"figref":"FIG. 15"},{"@attributes":{"id":"p-0160","num":"0159"},"figref":["FIG. 16","FIG. 16A","FIG. 16B"],"b":["16","16"]},{"@attributes":{"id":"p-0161","num":"0160"},"figref":["FIGS. 17A-C","FIG. 17A","FIG. 17B","FIG. 17C"]},{"@attributes":{"id":"p-0162","num":"0161"},"figref":["FIGS. 18A-B","FIG. 18A","FIG. 18B"]},{"@attributes":{"id":"p-0163","num":"0162"},"figref":"FIG. 19","sub":"0 "},{"@attributes":{"id":"p-0164","num":"0163"},"figref":"FIG. 20"},{"@attributes":{"id":"p-0165","num":"0164"},"figref":"FIG. 21"},{"@attributes":{"id":"p-0166","num":"0165"},"figref":"FIG. 22"},{"@attributes":{"id":"p-0167","num":"0166"},"figref":"FIG. 23"},{"@attributes":{"id":"p-0168","num":"0167"},"figref":"FIG. 24"},{"@attributes":{"id":"p-0169","num":"0168"},"figref":"FIG. 25"},{"@attributes":{"id":"p-0170","num":"0169"},"figref":"FIG. 26"},{"@attributes":{"id":"p-0171","num":"0170"},"figref":"FIG. 27"},{"@attributes":{"id":"p-0172","num":"0171"},"figref":"FIG. 28"},{"@attributes":{"id":"p-0173","num":"0172"},"figref":"FIG. 29"},{"@attributes":{"id":"p-0174","num":"0173"},"figref":"FIG. 30"},{"@attributes":{"id":"p-0175","num":"0174"},"figref":"FIG. 31"},{"@attributes":{"id":"p-0176","num":"0175"},"figref":"FIG. 32"},{"@attributes":{"id":"p-0177","num":"0176"},"figref":"FIG. 33"},{"@attributes":{"id":"p-0178","num":"0177"},"figref":"FIG. 34"},{"@attributes":{"id":"p-0179","num":"0178"},"figref":"FIG. 35"},{"@attributes":{"id":"p-0180","num":"0179"},"figref":["FIGS. 36A-B","FIG. 36A","FIG. 36B"]},{"@attributes":{"id":"p-0181","num":"0180"},"figref":"FIG. 37"},{"@attributes":{"id":"p-0182","num":"0181"},"figref":"FIG. 38"},{"@attributes":{"id":"p-0183","num":"0182"},"figref":["FIGS. 39A-B","FIG. 39A","FIG. 39B"]},{"@attributes":{"id":"p-0184","num":"0183"},"figref":"FIG. 40"},{"@attributes":{"id":"p-0185","num":"0184"},"figref":"FIG. 41"}]},"DETDESC":[{},{}]}
