---
title: Directed evolution of disulfide-bonded micro-proteins
abstract: The invention relates, in part, to a library of chimeric proteins, each chimeric protein including a mini-protein between about eight and about forty amino acids long, wherein the mini-protein has a single disulfide bond formed by a pair of invariant cysteines and has only two cysteines. The chimeric protein also includes at least a portion of an outer surface protein of a genetic package, wherein the chimeric protein is displayed on the outer surface of the genetic package. The invention also includes, in part, a mixture of nucleic acids that encode a library of the invention. The invention also includes, in part, a process for identifying proteins with a desired binding activity against a target, the process including screening a library of chimeric proteins of the invention; and identifying the chimeric protein. The invention, in part, also includes chimeric proteins expressed by a library of the invention.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=07413537&OS=07413537&RS=07413537
owner: Dyax Corp.
number: 07413537
owner_city: Cambridge
owner_country: US
publication_date: 20020731
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"p":["This is a continuation of Ser. No. 09\/893,878 filed Jun. 29, 2001 now U.S. Pat. No. 7,208,293, now pending, which is a continuation of Ser. No. 08\/993,776 filed Dec. 18, 1997, now abandoned; which is a continuation of Ser. No. 08\/415,922, filed Apr. 3, 1995, now U.S. Pat. No. 5,837,500; which is a continuation of Ser. No. 08\/009,319, filed Jan. 26, 1993, now U.S. Pat. No. 5,403,484; which is a division of Ser. No. 07\/664,989, filed Mar. 1, 1991, now U.S. Pat. No. 5,223,409. The prior application(s) set forth above are hereby incorporated by reference in their entirety.","The following related and commonly-owned applications are also incorporated by reference:","Robert Charles Ladner, Sonia Kosow Guterman, Rachael Baribault Kent, and Arthur Charles Ley are named as joint inventors on U.S. Ser. No. 07\/293,980, filed Jan. 8, 1989, now U.S. Pat. No. 5,096,815, and entitled GENERATION AND SELECTION OF NOVEL DNA-BINDING PROTEINS AND POLYPEPTIDES. This application has been assigned to Protein Engineering Corporation.","Robert Charles Ladner, Sonia Kosow Guterman, and Bruce Lindsay Roberts are named as a joint inventors on a U.S. Ser. No. 07\/470,651 filed 26 Jan. 1990, now abandoned, entitled \u201cPRODUCTION OF NOVEL SEQUENCE-SPECIFIC DNA-ALTERING ENZYMES\u201d, likewise assigned to Protein Engineering Corp.","Ladner, Guterman, Kent, Ley, and Markland, U.S. Ser. No. 07\/558,011, now U.S. Pat. No. 5,198,346, is also assigned to Protein Engineering Corporation.","This invention relates to development of novel binding proteins (including mini-proteins) by an iterative process of mutagenesis, expression, chromatographic selection, and amplification. In this process, a gene encoding a potential binding domain, said gene being obtained by random mutagenesis of a limited number of predetermined codons, is fused to a genetic element which causes the resulting chimeric expression product to be displayed on the outer surface of a virus (especially a filamentous phage) or a cell. Chromatographic selection is then used to identify viruses or cells whose genome includes such a fused gene which coded for the protein which bound to the chromatographic target.","A. Protein Structure","The amino acid sequence of a protein determines its three-dimensional (3D) structure, which in turn determines protein function (EPST63, ANFI73). Shortle (SHOR85), Sauer and colleagues (PAKU86, REID88a), and Caruthers and colleagues (EISE85) have shown that some residues on the polypeptide chain are more important than others in determining the 3D structure of a protein. The 3D structure is essentially unaffected by the identity of the amino acids at some loci; at other loci only one or a few types of amino acid is allowed. In most cases, loci where wide variety is allowed have the amino acid side group directed toward the solvent. Loci where limited variety is allowed frequently have the side group directed toward other parts of the protein. Thus substitutions of amino acids that are exposed to solvent are less likely to affect the 3D structure than are substitutions at internal loci. (See also SCHU79, p169-171 and CREI84, p239-245, 314-315).","The secondary structure (helices, sheets, turns, loops) of a protein is determined mostly by local sequence. Certain amino acids have a propensity to appear in certain \u201csecondary structures,\u201d they will be found from time to time in other structures, and studies of pentapeptide sequences found in different proteins have shown that their conformation varies considerably from one occurrence to the next (KABS84, ARGO87). As a result, a priori design of proteins to have a particular 3D structure is difficult.","Several researchers have designed and synthesized proteins de novo (MOSE83, MOSE87, ERIC86). These designed proteins are small and most have been synthesized in vitro as polypeptides rather than genetically. Hecht et al. (HECH90) have produced a designed protein genetically. Moser, et al. state that design of biologically active proteins is currently impossible.","B. Protein Binding Activity","Many proteins bind non-covalently but very tightly and specifically to some other characteristic molecules (SCHU79, CREI84). In each case the binding results from complementarity of the surfaces that come into contact: bumps fit into holes, unlike charges come together, dipoles align, and hydrophobic atoms contact other hydrophobic atoms. Although bulk water is excluded, individual water molecules are frequently found filling space in intermolecular interfaces; these waters usually form hydrogen bonds to one or more atoms of the protein or to other bound water. Thus proteins found in nature have not attained, nor do they require, perfect complementarity to bind tightly and specifically to their substrates. Only in rare cases is there essentially perfect complementarity; then the binding is extremely tight (as for example, avidin binding to biotin).","C. Protein Engineering","\u201cProtein engineering\u201d is the art of manipulating the sequence of a protein in order to alter its binding characteristics. The factors affecting protein binding are known, (CHOT75, CHOT76, SCHU79, p 98-107, and CREI84, Ch8), but designing new complementary surfaces has proved difficult. Although some rules have been developed for substituting side groups (SUTC87b), the side groups of proteins are floppy and it is difficult to predict what conformation a new side group will take. Further, the forces that bind proteins to other molecules are all relatively weak and it is difficult to predict the effects of these forces.","Recently, Quiocho and collaborators (QUIO87) elucidated the structures of several periplasmic binding proteins from Gram-negative bacteria. They found that the proteins, despite having low sequence homology and differences in structural detail, have certain important structural similarities. Based on their investigations of these binding proteins, Quiocho et al. suggest it is unlikely that, using current protein engineering methods, proteins can be constructed with binding properties superior to those of proteins that occur naturally.","Nonetheless, there have been some isolated successes. Wilkinson et al. (WILK84) reported that a mutant of the tyrosyl tRNA synthetase of with the mutation Thr\u2014>Pro exhibits a 100-fold increase in affinity for ATP. Tan and Kaiser (TANK77) and Tschesche et al. (TSCH87) showed that changing a single amino acid in mini-protein greatly reduces its binding to trypsin, but that some of the mutants retained the parental characteristic of binding to an inhibiting chymotrypsin, while others exhibited new binding to elastase. Caruthers and others (EISE85) have shown that changes of single amino acids on the surface of the lambda Cro repressor greatly reduce its affinity for the natural operator O3, but greatly increase the binding of the mutant protein to a mutant operator. Changing three residues in subtilisin from to be the same as the corresponding residues in subtilisin from produced a protease having nearly the same activity as the latter subtilisin, even though 82 amino acid sequence differences remained (WELL87a). Insertion of DNA encoding 18 amino acids (corresponding to Pro-Glu-Dynorphin-Gly) into the phoA gene so that the additional amino acids appeared within a loop of the alkaline phosphatase protein resulted in a chimeric protein having both phoA and dynorphin activity (FREI90). Thus, changing the surface of a binding protein may alter its specificity without abolishing binding activity.","D. Techniques Of Mutagenesis","Early techniques of mutating proteins involved manipulations at the amino acid sequence level. In the semisynthetic method (TSCH87), the protein was cleaved into two fragments, a residue removed from the new end of one fragment, the substitute residue added on in its place, and the modified fragment joined with the other, original fragment. Alternatively, the mutant protein could be synthesized in its entirety (TANK77).","Erickson et al. suggested that mixed amino acid reagents could be used to produce a family of sequence-related proteins which could then be screened by affinity chromatography (ERIC86). They envision successive rounds of mixed synthesis of variant proteins and purification by specific binding. They do not discuss how residues should be chosen for variation. Because proteins cannot be amplified, the researchers must sequence the recovered protein to learn which substitutions improve binding. The researchers must limit the level of diversity so that each variety of protein will be present in sufficient quantity for the isolated fraction to be sequenced.","With the development of recombinant DNA techniques, it became possible to obtain a mutant protein by mutating the gene encoding the native protein and then expressing the mutated gene. Several mutagenesis strategies are known. One, \u201cprotein surgery\u201d (DILL87), involves the introduction of one or more predetermined mutations within the gene of choice. A single polypeptide of completely predetermined sequence is expressed, and its binding characteristics are evaluated.","At the other extreme is random mutagenesis by means of relatively nonspecific mutagens such as radiation and various chemical agents. See Ho et al. (HOCJ85) and Lehtovaara, E.P. Appln. 285,123.","It is possible to randomly vary predetermined nucleotides using a mixture of bases in the appropriate cycles of a nucleic acid synthesis procedure. The proportion of bases in the mixture, for each position of a codon, will determine the frequency at which each amino acid will occur in the polypeptides expressed from the degenerate DNA population. Oliphant et al. (OLIP86) and Oliphant and Struhl (OLIP87) have demonstrated ligation and cloning of highly degenerate oligonucleotides, which were used in the mutation of promoters. They suggested that similar methods could be used in the variation of protein coding regions. They do not say how one should: a) choose protein residues to vary, or b) select or screen mutants with desirable properties. Reidhaar-Olson and Sauer (REID88a) have used synthetic degenerate oligo-nts to vary simultaneously two or three residues through all twenty amino acids. See also Vershon et al. (VERS86a; VERS86b). Reidhaar-Olson and Sauer do not discuss the limits on how many residues could be varied at once nor do they mention the problem of unequal abundance of DNA encoding different amino acids. They looked for proteins that either had wild-type dimerization or that did not dimerize. They did not seek proteins having novel binding properties and did not find any. This approach is likewise limited by the number of colonies that can be examined (ROBE86).","To the extent that this prior work assumes that it is desirable to adjust the level of mutation so that there is one mutation per protein, it should be noted that many desirable protein alterations require multiple amino acid substitutions and thus are not accessible through single base changes or even through all possible amino acid substitutions at any one residue.","D. Affinity Chromatography of Cells","Ferenci and coloborators have published a series of papers on the chromatographic isolation of mutants of the maltose-transport protein LamB of (FERE82a, FERE82b, FERE83, FERE84, CLUN84, HEIN87 and papers cited therein). The mutants were either spontaneous or induced with nonspecific chemical mutagens. Levels of mutagenesis were picked to provide single point mutations or single insertions of two residues. No multiple mutations were sought or found.","While variation was seen in the degree of affinity for the conventional LamB substrates maltose and starch, there was no selection for affinity to a target molecule not bound at all by native LamB, and no multiple mutations were sought or found. FERE84 speculated that the affinity chromatographic selection technique could be adapted to development of similar mutants of other \u201cimportant bacterial surface-located enzymes\u201d, and to selecting for mutations which result in the relocation of an intracellular bacterial protein to the cell surface. Ferenci's mutant surface proteins would not, however, have been chimeras of a bacterial surface protein and an exogenous or heterologous binding domain.","Ferenci also taught that there was no need to clone the structural gene, or to know the protein structure, active site, or sequence. The method of the present invention, however, specifically utilizes a cloned structural gene. It is not possible to construct and express a chimeric, outer surface-directed potential binding protein-encoding gene without cloning.","Ferenci did not limit the mutations to particular loci or particular substitutions. In the present invention, knowledge of the protein structure, active site and\/or sequence is used as appropriate to predict which residues are most likely to affect binding activity without unduly destabilizing the protein, and the mutagenesis is focused upon those sites. Ferenci does not suggest that surface residues should be preferentially varied. In consequence, Ferenci's selection system is much less efficient than that disclosed herein.","E. Bacterial and Viral Expression of Chimeric Surface Proteins","A number of researchers have directed unmutated foreign antigenic epitopes to the surface of bacteria or phage, fused to a native bacterial or phage surface protein, and demonstrated that the epitopes were recognized by antibodies. Thus, Charbit, et al. (CHAR86) genetically inserted the C3 epitope of the VP1 coat protein of poliovirus into the LamB outer membrane protein of and determined immunologically that the C3 epitope was exposed on the bacterial cell surface. Charbit, et al. (CHAR87) likewise produced chimeras of LamB and the A (or B) epitopes of the preS2 region of hepatitis B virus.","A chimeric LacZ\/OmpB protein has been expressed in and is, depending on the fusion, directed to either the outer membrane or the periplasm (SILH77). A chimeric LacZ\/OmpA surface protein has also been expressed and displayed on the surface of cells (Weinstock et al., WEIN83). Others have expressed and displayed on the surface of a cell chimeras of other bacterial surface proteins, such as type 1 fimbriae (Hedegaard and Klemm (HEDE89)) and type 1 fimbriae (Jennings et al., JENN89). In none of the recited cases was the inserted genetic material mutagenized.","Dulbecco (DULB86) suggests a procedure for incorporating a foreign antigenic epitope into a viral surface protein so that the expressed chimeric protein is displayed on the surface of the virus in a manner such that the foreign epitope is accessible to antibody. In 1985 Smith (SMIT85) reported inserting a nonfunctional segment of the EcoRI endonuclease gene into gene III of bacteriophage f1, \u201cin phase\u201d. The gene III protein is a minor coat protein necessary for infectivity. Smith demonstrated that the recombinant phage were adsorbed by immobilized antibody raised against the EcoRI endonuclease, and could be eluted with acid. De la Cruz et al. (DELA88) have expressed a fragment of the repeat region of the circumsporozoite protein from on the surface of M13 as an insert in the gene III protein. They showed that the recombinant phage were both antigenic and immunogenic in rabbits, and that such recombinant phage could be used for B epitope mapping. The researchers suggest that similar recombinant phage could be used for T epitope mapping and for vaccine development.","None of these researchers suggested mutagenesis of the inserted material, nor is the inserted material a complete binding domain conferring on the chimeric protein the ability to bind specifically to a receptor other than the antigen combining site of an antibody.","McCafferty et al. (MCCA90) expressed a fusion of an Fv fragment of an antibody to the N-terminal of the pIII protein. The Fv fragment was not mutated.","F. Epitope Libraries on Fusion Phage","Parmley and Smith (PARM88) suggested that an epitope library that exhibits all possible hexapeptides could be constructed and used to isolate epitopes that bind to antibodies. In discussing the epitope library, the authors did not suggest that it was desirable to balance the representation of different amino acids. Nor did they teach that the insert should encode a complete domain of the exogenous protein. Epitopes are considered to be unstructured peptides as opposed to structured proteins.","After the filing of the parent application whose benefit is claimed herein under 35 U.S.C. 120, certain groups reported the construction of \u201cepitope libraries.\u201d Scott and Smith (SCOT90) and Cwirla et al. (CWIR90) prepared \u201cepitope libraries\u201d in which potential hexapeptide epitopes for a target antibody were randomly mutated by fusing degenerate oligonucleotides, encoding the epitopes, with gene III of fd phage, and expressing the fused gene in phage-infected cells. The cells manufactured fusion phage which displayed the epitopes on their surface; the phage which bound to immobilized antibody were eluted with acid and studied. In both cases, the fused gene featured a segment encoding a spacer region to separate the variable region from the wild type pIII sequence so that the varied amino acids would not be constrained by the nearby pIII sequence. Devlin et al. (DEVL90) similarly screened; using M13 phage, for random 15 residue epitopes recognized by streptavidin. Again, a spacer was used to move the random peptides away from the rest of the chimeric phage protein. These references therefore taught away from constraining the conformational repertoire of the mutated residues.","Another problem with the Scott and Smith, Cwirla et al., and Devlin et al., libraries was that they provided a highly biased sampling of the possible amino acids at each position. Their primary concern in designing the degenerate oligonucleotide encoding their variable region was to ensure that all twenty amino acids were encodible at each position; a secondary consideration was minimizing the frequency of occurrence of stop signals. Consequently, Scott and Smith and Cwirla et al. employed NNK (N=equal mixture of G, A, T, C; K=equal mixture of G and T) while Devlin et al. used NNS (S=equal mixture of G and C). There was no attempt to minimize the frequency ratio of most favored-to-least favored amino acid, or to equalize the rate of occurrence of acidic and basic amino acids.","Devlin et al. characterized several affinity-selected streptavidin-binding peptides, but did not measure the affinity constants for these peptides. Cwirla et al. did determine the affinity constant for his peptides, but were disappointed to find that his best hexapeptides had affinities (350-300 nM), \u201corders of magnitude\u201d weaker than that of the native Met-enkephalin epitope (7 nM) recognized by the target antibody. Cwirla et al. speculated that phage bearing peptides with higher affinities remained bound under acidic elution, possibly because of multivalent interactions between phage (carrying about 4 copies of pIII) and the divalent target IgG. Scott and Smith were able to find peptides whose affinity for the target antibody (A2) was comparable to that of the reference myohemerythrin epitope (50 nM). However, Scott and Smith likewise expressed concern that some high-affinity peptides were lost, possibly through irreversible binding of fusion phage to target.","G. Non-Commonly Owned Patents and Applications Naming Robert Ladner as an Inventor","Ladner, U.S. Pat. No. 4,704,692, \u201cComputer Based System and Method for Determining and Displaying Possible Chemical Structures for Converting Double- or Multiple-Chain Polypeptides to Single-Chain Polypeptides\u201d describes a design method for converting proteins composed of two or more chains into proteins of fewer polypeptide chains, but with essentially the same 3D structure. There is no mention of variegated DNA and no genetic selection. Ladner and Bird, WO88\/01649 (Publ. Mar. 10, 1988) disclose the specific application of computerized design of linker peptides to the preparation of single chain antibodies.","Ladner, Glick, and Bird, WO88\/06630 (publ. 7 Sep. 1988 and having priority from U.S. application Ser. No. 07\/021,046, assigned to Genex Corp.) (LGB) speculate that diverse single chain antibody domains (SCAD) may be screened for binding to a particular antigen by varying the DNA encoding the combining determining regions of a single chain antibody, subcloning the SCAD gene into the gpV gene of phage lambda so that a SCAD\/gpV chimera is displayed on the outer surface of phage lambda, and selecting phage which bind to the antigen through affinity chromatography. The only antigen mentioned is bovine growth hormone. No other binding molecules, targets, carrier organisms, or outer surface proteins are discussed. Nor is there any mention of the method or degree of mutagenesis. Furthermore, there is no teaching as to the exact structure of the fusion nor of how to identify a successful fusion or how to proceed if the SCAD is not displayed.","Ladner and Bird, WO88\/06601 (publ. 7 Sep. 1988) suggest that single chain \u201cpseudodimeric\u201d repressors (DNA-binding proteins) may be prepared by mutating a putative linker peptide followed by in vivo selection that mutation and selection may be used to create a dictionary of recognition elements for use in the design of asymmetric repressors. The repressors are not displayed on the outer surface of an organism.","Methods of identifying residues in protein which can be replaced with a cysteine in order to promote the formation of a protein-stabilizing disulfide bond are given in Pantoliano and Ladner, U.S. Pat. No. 4,903,773 (PANT90), Pantoliano and Ladner (PANT87), Pabo and Suchenek (PABO86), MATS89, and SAUE86.","No admission is made that any cited reference is prior art or pertinent prior art, and the dates given are those appearing on the reference and may not be identical to the actual publication date. All references cited in this specification are hereby incorporated by reference.","The present invention is intended to overcome the deficiencies discussed above. It relates to the construction, expression, and selection of mutated genes that specify novel proteins with desirable binding properties, as well as these proteins themselves. The substances bound by these proteins, hereinafter referred to as \u201ctargets\u201d, may be, but need not be, proteins. Targets may include other biological or synthetic macromolecules as well as other organic and inorganic substances.","The fundamental principle of the invention is one of forced evolution. In nature, evolution results from the combination of genetic variation, selection for advantageous traits, and reproduction of the selected individuals, thereby enriching the population for the trait. The present invention achieves genetic variation through controlled random mutagenesis (\u201cvariegation\u201d) of DNA, yielding a mixture of DNA molecules encoding different but related potential binding proteins. It selects for mutated genes that specify novel proteins with desirable binding properties by 1) arranging that the product of each mutated gene be displayed on the outer surface of a replicable genetic package (GP) (a cell, spore or virus) that contains the gene, and 2) using affinity selection\u2014selection for binding to the target material\u2014to enrich the population of packages for those packages containing genes specifying proteins with improved binding to that target material. Finally, enrichment is achieved by allowing only the genetic packages which, by virtue of the displayed protein, bound to the target, to reproduce. The evolution is \u201cforced\u201d in that selection is for the target material provided.","The display strategy is first perfected by modifying a genetic package to display a stable, structured domain (the \u201cinitial potential binding domain\u201d, IPBD) for which an affinity molecule (which may be an antibody) is obtainable. The success of the modifications is readily measured by, e.g., determining whether the modified genetic package binds to the affinity molecule.","The IPBD is chosen with a view to its tolerance for extensive mutagenesis. Once it is known that the IPBD can be displayed on a surface of a package and subjected to affinity selection, the gene encoding the IPBD is subjected to a special pattern of multiple mutagenesis, here termed \u201cvariegation\u201d, which after appropriate cloning and amplification steps leads to the production of a population of genetic packages each of which displays a single potential binding domain (a mutant of the IPBD), but which collectively display a multitude of different though structurally related potential binding domains (PBDs). Each genetic package carries the version of the pbd gene that encodes the PBD displayed on the surface of that particular package. Affinity selection is then used to identify the genetic packages bearing the PBDs with the desired binding characteristics, and these genetic packages may then be amplified. After one or more cycles of enrichment by affinity selection and amplification, the DNA encoding the successful binding domains (SBDs) may then be recovered from selected packages.","If need be, the DNA from the SBD-bearing packages may then be further \u201cvariegated\u201d, using an SBD of the last round of variegation as the \u201cparental potential binding domain\u201d (PPBD) to the next generation of PBDs, and the process continued until the worker in the art is satisfied with the result. At that point, the SBD may be produced by any conventional means, including chemical synthesis.","When the number of different amino acid sequences obtainable by mutation of the domain is large when compared to the number of different domains which are displayable in detectable amounts, the efficiency of the forced evolution is greatly enhanced by careful choice of which residues are to be varied. First, residues of a known protein which are likely to affect its binding activity (e.g., surface residues) and not likely to unduly degrade its stability are identified. Then all or some of the codons encoding these residues are varied simultaneously to produce a variegated population of DNA. The variegated population of DNA is used to express a variety of potential binding domains, whose ability to bind the target of interest may then be evaluated.","The method of the present invention is thus further distinguished from other methods in the nature of the highly variegated population that is produced and from which novel binding proteins are selected. We force the displayed potential binding domain to sample the nearby \u201csequence space\u201d of related amino-acid sequences in an efficient, organized manner. Four goals guide the various variegation plans used herein, preferably: 1) a very large number (e.g. 10) of variants is available, 2) a very high percentage of the possible variants actually appears in detectable amounts, 3) the frequency of appearance of the desired variants is relatively uniform, and 4) variation occurs only at a limited number of amino-acid residues, most preferably at residues having side groups directed toward a common region on the surface of the potential binding domain.","This is to be distinguished from the simple use of indiscriminate mutagenic agents such as radiation and hydroxylamine to modify a gene, where there is no (or very oblique) control over the site of mutation. Many of the mutations will affect residues that are not a part of the binding domain. Moreover, since at a reasonable level of mutagenesis, any modified codon is likely to be characterized by a single base change, only a limited and biased range of possibilities will be explored. Equally remote is the use of site-specific mutagenesis techniques employing mutagenic oligonucleotides of nonrandomized sequence, since these techniques do not lend themselves to the production and testing of a large number of variants. While focused random mutagenesis techniques are known, the importance of controlling the distribution of variation has been largely overlooked.","In order to obtain the display of a multitude of different though related potential binding domains, applicants generate a heterogeneous population of replicable genetic packages each of which comprises a hybrid gene including a first DNA sequence which encodes a potential binding domain for the target of interest and a second DNA sequence which encodes a display means, such as an outer surface protein native to the genetic package but not natively associated with the potential binding domain (or the parental binding domain to which it is related) which causes the genetic package to display the corresponding chimeric protein (or a processed form thereof) on its outer surface.","It should be recognized that by expressing a hybrid protein which comprises an outer surface transport signal not natively associated with the binding domain, the utility of the present invention is greatly extended. The binding domain need not be that of a surface protein of the genetic package (or, in the case of a viral package, of its host cell), since the provided outer surface transport signal is responsible for achieving the desired display. Thus, it is possible to display on the surface of a phage, bacterial cell or bacterial spore a binding domain related to the binding domain of a normally cytoplasmic binding protein, or the binding domain of eukaryotic protein which is not found on the surface of prokaryotic cells or viruses.","Another important aspect of the invention is that each potential binding domain remains physically associated with the particular DNA molecule which encodes it. Thus, once successful binding domains are identified, one may readily recover the gene and either express additional quantities of the novel binding protein or further mutate the gene. The form that this association takes is a \u201creplicable genetic package\u201d, a virus, cell or spore which replicates and expresses the binding domain-encoding gene, and transports the binding domain to its outer surface.","It is also possible chemically or enzymatically to modify the PBDs before selection. The selection then identifies the best modified amino acid sequence. For example, we could treat the variegated population of genetic packages that display a variegated population of binding domains with a protein tyrosine kinase and then select for binding the target. Any tyrosines on the BD surface will be phosphorylated and this could affect the binding properties. Other chemical or enzymatic modifications are possible.","By virtue of the present invention, proteins are obtained which can bind specifically to targets other than the antigen-combining sites of antibodies. A protein is not to be considered a \u201cbinding protein\u201d merely because it can be bound by an antibody (see definition of \u201cbinding protein\u201d which follows). While almost any amino acid sequence of more than about 6-8 amino acids is likely, when linked to an immunogenic carrier, to elicit an immune response, any given random polypeptide is unlikely to satisfy the stringent definition of \u201cbinding protein\u201d with respect to minimum affinity and specificity for its substrate. It is only by testing numerous random polypeptides simultaneously (and, in the usual case, controlling the extent and character of the sequence variation, i.e., limiting it to residues of a potential binding domain having a stable structure, the residues being chosen as more likely to affect binding than stability) that this obstacle is overcome.","In one embodiment, the invention relates to:\n\n","The invention is not, however, limited to proteins with a single BD since the method may be applied to any or all of the BDs of the protein, sequentially or simultaneously. The invention is not, however, limited to biological synthesis of the binding domains; peptides having an amino-acid sequence determined by the isolated DNA can be chemically synthesized.","The invention further relates to a variegated population of genetic packages. Said population may be used by one user to select for binding to a first target, by a second user to select for binding to a second target, and so on, as the present invention does not require that the initial potential binding domain actually bind to the target of interest, and the variegation is at residues likely to affect binding. The invention also relates to the variegated DNA used in preparing such genetic packages.","The invention likewise encompasses the procedure by which the display strategy is verified. The genetic packages are engineered to display a single IPBD sequence. (Variability may be introduced into DNA subsequences adjacent to the ipbd subsequence and within the osp-ipbd gene so that the IPBD will appear on the GP surface.) A molecule, such as an antibody, having high affinity for correctly folded IPBD is used to: a) detect IPBD on the GP surface, b) screen colonies for display of IPBD on the GP surface, or c) select GPs that display IPBD from a population, some members of which might display IPBD on the GP surface. In one preferred embodiment, this verification process (part I) involves:\n\n","Once a GP(IPBD) is produced, it can be used many times as the starting point for developing different novel proteins that bind to a variety of different targets. The knowledge of how we engineer the appearance of one IPBD on the surface of a GP can be used to design and produce other GP(IPBD)s that display different IPBDs.","Knowing that a particular genetic package and osp-ipbd fusion are suitable for the practice of the invention, we may variegate the genetic packages and select for binding to a target of interest. Using IPBD as the PPBD to the first cycle of variegation, we prepare a wide variety of osp-pbd genes that encode a wide variety of PBDs. We use an affinity separation to enrich the population of GP(vgPBD)s for GPs that display PBDs with binding properties relative to the target that are superior to the binding properties of the PPBD. An SBD selected from one variegation cycle becomes the PPBD to the next variegation cycle. In a preferred embodiment, Part II of the process of the present invention involves:\n\n","Part II is repeated for each new target material. Part I need be repeated only if no GP(IPBD) suitable to a chosen target is available.","For each target, there are a large number of SBDs that may be found by the method of the present invention. The process relies on a combination of protein structural considerations, probabilities, and targeted mutations with accumulation of information. To increase the probability that some PBD in the population will bind to the target, we generate as large a population as we can conveniently subject to selection-through-binding in one experiment. Key questions in management of the method are \u201cHow many transformants can we produce?\u201d, and \u201cHow small a component can we find through selection-through-binding?\u201d. The optimum level of variegation is determined by the maximum number of transformants and the selection sensitivity, so that for any reasonable sensitivity we may use a progressive process to obtain a series of proteins with higher and higher affinity for the chosen target material.","The appended claims are hereby incorporated by reference into this specification as an enumeration of the preferred embodiments.",{"@attributes":{"id":"p-0077","num":"0000"},"ul":{"@attributes":{"id":"ul0007","list-style":"none"},"li":["I. Definitions and Abbreviations","II. The Initial Potential Binding Domain\n    \n    ","III. Variegation Strategy\u2014Mutagenesis to Obtain Potential Binding Domains with Desired Diversity\n    \n    ","IV. Display Strategy\u2014Displaying Foreign Binding Domains on the Surface of a \u201cGenetic Package\u201d\n    \n    ","V. Affinity Selection of Target-Binding Mutants\n    \n    ","VI. Exploitation of Successful Binding Domains and Corresponding DNAS\n    \n    "]}},"I. Definitions and Abbreviations","Let K(x,y) be a dissociation constant,",{"@attributes":{"id":"p-0079","num":"0156"},"maths":{"@attributes":{"id":"MATH-US-00001","num":"00001"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mrow":{"msub":{"mi":["K","d"]},"mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mi":["x","y"],"mo":","}}},"mo":"=","mfrac":{"mrow":[{"mrow":[{"mo":["[","]"],"mi":"x"},{"mo":["[","]"],"mi":"y"}],"mo":"\u2061"},{"mo":["[","]"],"mrow":{"mi":["x","y"],"mo":":"}}]}}}},"br":{},"sub":["D ","D "],"sup":["\u22126 ","\u22127 ","\u22124 ","\u22121 "]},"The exclusion of \u201cvariable domain of an antibody\u201d in (1) above is intended to make clear that for the purposes herein a protein is not to be considered a \u201cbinding protein\u201d merely because it is antigenic. However, an antigen may nonetheless qualify as a binding protein because it specifically binds to a substance other than an antibody, e.g., an enzyme for its substrate, or a hormone for its cellular receptor. Additionally, it should be pointed out that \u201cbinding protein\u201d may include a protein which binds specifically to the Fc of an antibody, e.g., staphylococcal protein A.","Normally, the binding protein will not be an antibody or a antigen-binding derivative thereof. An antibody is a crosslinked complex of four polypeptides (two heavy and two light chains). The light chains of IgG have a molecular weight of \u224823,000 daltons and the heavy chains of \u224853,000 daltons. A single binding unit is composed of the variable region of a heavy chain (V) and the variable region of a light chain (V), each about 110 amino-acid residues. The Vand Vregions are held in proximity by a disulfide bond between the adjoining Cand Cregions; altogether, these total 440 residues and correspond to an Fab fragment. Derivatives of antibodies include Fab fragments and the individual variable light and heavy domains. A special case of antibody derivative is a \u201csingle chain antibody.\u201d A \u201csingle-chain antibody\u201d is a single chain polypeptide comprising at least 200 amino acids, said amino acids forming two antigen-binding regions connected by a peptide linker that allows the two regions to fold together to bind the antigen in a manner akin to that of an Fab fragment. Either the two antigen-binding regions must be variable domains of known antibodies, or they must (1) each fold into a \u03b2 barrel of nine strands that are spatially related in the same way as are the nine strands of known antibody variable light or heavy domains, and (2) fit together in the same way as do the variable domains of said known antibody. Generally speaking, this will require that, with the exception of the amino acids corresponding to the hypervariable region, there is at least 88% homology with the amino acids of the variable domain of a known antibody.","While the present invention may be used to develop novel antibodies through variegation of codons corresponding to the hypervariable region of an antibody's variable domain, its primary utility resides in the development of binding proteins which are not antibodies or even variable domains of antibodies. Novel antibodies can be obtained by immunological techniques; novel enzymes, hormones, etc. cannot.","It will be appreciated that, as a result of evolution, the antigen-binding domains of antibodies have acquired a structure which tolerates great variability of sequence in the hypervariable regions. The remainder of the variable domain is made up of constant regions forming a distinctive structure, a nine strand \u03b2 barrel, which hold the hypervariable regions (inter-strand loops) in a fixed relationship with each other. Most other binding proteins lack this molecular design which facilitates diversification of binding characteristics. Consequently, the successful development of novel antibodies by modification of sequences encoding known hypervariable regions\u2014which, in nature, vary from antibody to antibody\u2014does not provide any guidance or assurance of success in the development of novel, non-immunoglobulin binding proteins.","It should further be noted that the affinity of antibodies for their target epitopes is typically on the order of 10to 10liters\/mole; many enzymes exhibit much greater affinities (10to 10liters\/mole) for their preferred substrates. Thus, if the goal is to develop a binding protein with a very high affinity for a target of interest, e.g., greater than 10the antibody design may in fact be unduly limiting. Furthermore, the complementarity-determining residues of an antibody comprises many residues, 30 to 50. In most cases, it is not known which of these residues participates directly in binding antigen. Thus, picking an antibody as PPBD does not allow us to focus variegation to a small number of residues.","Most larger proteins fold into distinguishable globules called domains (ROSS81). Protein domains have been defined various ways, but all definitions fall into one of three classes: a) those that define a domain in terms of 3D atomic coordinates, b) those that define a domain as an isolable, stable fragment of a larger protein, and c) those that define a domain based on protein sequence homology plus a method from class a) or b). Frequently, different methods of defining domains applied to a single protein yield identical or very similar domain boundaries. The diversity of definitions for domains stems from the many ways that protein domains are perceived to be important, including the concept of domains in predicting the boundaries of stable fragments, and the relationship of domains to protein folding, function, stability, and evolution. The present invention emphasizes the retention of the structured character of a domain even though its surface residues are mutated. Consequently, definitions of \u201cdomain\u201d which emphasize stability\u2014retention of the overall structure in the face of perturbing forces such as elevated temperatures or chaotropic agents\u2014are favored, though atomic coordinates and protein sequence homology are not completely ignored.","When a domain of a protein is primarily responsible for the protein's ability to specifically bind a chosen target, it is referred to herein as a \u201cbinding domain\u201d (BD). A preliminary operation is to engineer the appearance of a stable protein domain, denoted as an \u201cinitial potential binding domain\u201d (IPBD), on the surface of a genetic package.","The term \u201cvariegated DNA\u201d (vgDNA) refers to a mixture of DNA molecules of the same or similar length which, when aligned, vary at some codons so as to encode at each such codon a plurality of different amino acids, but which encode only a single amino acid at other codon positions. It is further understood that in variegated DNA, the codons which are variable, and the range and frequency of occurrence of the different amino acids which a given variable codon encodes, are determined in advance by the synthesizer of the DNA, even though the synthetic method does not allow one to know, a priori, the sequence of any individual DNA molecule in the mixture. The number of designated variable codons in the variegated DNA is preferably no more than 20 codons, and more preferably no more than 5-10 codons. The mix of amino acids encoded at each variable codon may differ from codon to codon.","A population of genetic packages into which variegated DNA has been introduced is likewise said to be \u201cvariegated\u201d.","For the purposes of this invention, the term \u201cpotential binding protein\u201d refers to a protein encoded by one species of DNA molecule in a population of variegated DNA wherein the region of variation appears in one or more subsequences encoding one or more segments of the polypeptide having the potential of serving as a binding domain for the target substance.","From time to time, it may be helpful to speak of the \u201cparent sequence\u201d of the variegated DNA. When the novel binding domain sought is an analogue of a known binding domain, the parent sequence is the sequence that encodes the known binding domain. The variegated DNA will be identical with this parent sequence at one or more loci, but will diverge from it at chosen loci. When a potential binding domain is designed from first principles, the parent sequence is a sequence which encodes the amino acid sequence that has been predicted to form the desired binding domain, and the variegated DNA is a population of \u201cdaughter DNAs\u201d that are related to that parent by a recognizable sequence similarity.","A \u201cchimeric protein\u201d is a protein composed of a first amino acid sequence substantially corresponding to the sequence of a protein or to a large fragment of a protein (20 or more residues) expressed by the species in which the chimeric protein is expressed and a second amino acid sequence that does not substantially correspond to an amino acid sequence of a protein expressed by the first species but that does substantially correspond to the sequence of a protein expressed by a second and different species of organism. The second sequence is said to be foreign to the first sequence.","One amino acid sequence of the chimeric proteins of the present invention is typically derived from an outer surface protein of a \u201cgenetic package\u201d as hereafter defined. The second amino acid sequence is one which, if expressed alone, would have the characteristics of a protein (or a domain thereof) but is incorporated into the chimeric protein as a recognizable domain thereof. It may appear at the amino or carboxy terminal of the first amino acid sequence (with or without an intervening spacer), or it may interrupt the first amino acid sequence. The first amino acid sequence may correspond exactly to a surface protein of the genetic package, or it may be modified, e.g., to facilitate the display of the binding domain.","In the present invention, the words \u201cselect\u201d and \u201cselection\u201d are used in the genetic sense; i.e. a biological process whereby a phenotypic characteristic is used to enrich a population for those organisms displaying the desired phenotype.","One affinity separation is called a \u201cseparation cycle\u201d; one pass of variegation followed by as many separation cycles as are needed to isolate an SBD, is called a \u201cvariegation cycle\u201d. The amino acid sequence of one SBD from one round becomes the PPBD to the next variegation cycle. We perform variegation cycles iteratively until the desired affinity and specificity of binding between an SBD and chosen target are achieved.","The following abbreviations will be used throughout the present specification:",{"@attributes":{"id":"p-0096","num":"0173"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"126pt","align":"left"}}],"thead":{"row":[{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"Abbreviation","Meaning"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"GP","Genetic Package, e.g. a"]},{"entry":[{},{},"bacteriophage"]},{"entry":[{},"wtGP","Wild-type GP"]},{"entry":[{},"X","Any protein"]},{"entry":[{},"x","The gene for protein X"]},{"entry":[{},"BD","Binding Domain"]},{"entry":[{},"BPTI","Bovine pancreatic trypsin"]},{"entry":[{},{},"inhibitor, identical to"]},{"entry":[{},{},"aprotinin (Merck Index,"]},{"entry":[{},{},"entry 784, p. 119(SEQ ID"]},{"entry":[{},{},"NO: 44))"]},{"entry":[{},"IPBD","Initial Potential Binding"]},{"entry":[{},{},"Domain, e.g. BPTI"]},{"entry":[{},"PBD","Potential Binding Domain,"]},{"entry":[{},{},"e.g. a derivative of BPTI"]},{"entry":[{},"SBD","Successful Binding Domain,"]},{"entry":[{},{},"e.g. a derivative of BPTI"]},{"entry":[{},{},"selected for binding to a"]},{"entry":[{},{},"target"]},{"entry":[{},"PPBD","Parental Potential Binding"]},{"entry":[{},{},"Domain, i.e. an IPBD or an"]},{"entry":[{},{},"SBD from a previous"]},{"entry":[{},{},"selection"]},{"entry":[{},"OSP","Outer Surface Protein,"]},{"entry":[{},{},"e.g. coat protein of a"]},{"entry":[{},{},"phage or LamB from "]},{"entry":[{},"OSP-PBD","Fusion of an OSP and a"]},{"entry":[{},{},"PBD, order of fusion not"]},{"entry":[{},{},"specified"]},{"entry":[{},"OSTS","Outer Surface Transport"]},{"entry":[{},{},"Signal"]},{"entry":[{},"GP(x)","A genetic package"]},{"entry":[{},{},"containing the x gene"]},{"entry":[{},"GP(X)","A genetic package that"]},{"entry":[{},{},"displays X on its outer"]},{"entry":[{},{},"surface"]},{"entry":[{},"GP(osp-pbd)","GP containing an osp-pbd"]},{"entry":[{},{},"gene"]},{"entry":[{},"GP(OSP-PBD)","A genetic package that"]},{"entry":[{},{},"displays PBD on its"]},{"entry":[{},{},"outside as a fusion to OSP"]},{"entry":[{},"GP(pbd)","GP containing a pbd gene,"]},{"entry":[{},{},"osp implicit"]},{"entry":[{},"GP(PBD)","A genetic package"]},{"entry":[{},{},"displaying PBD on its"]},{"entry":[{},{},"outside, OSP unspecified"]},{"entry":[{},"{Q}","An affinity matrix"]},{"entry":[{},{},"supporting \u201cQ\u201d, e.g. {T4"]},{"entry":[{},{},"lysozyme} is T4 lysozyme"]},{"entry":[{},{},"attached to an affinity"]},{"entry":[{},{},"matrix"]},{"entry":[{},"AfM(W)","A molecule having affinity"]},{"entry":[{},{},"for \u201cW\u201d, e.g. trypsin is"]},{"entry":[{},{},"an AfM(BPTI)"]},{"entry":[{},"AfM(W)*","AfM(W) carrying a label,"]},{"entry":[{},{},"e.g. I"]},{"entry":[{},"XINDUCE","A chemical that can induce"]},{"entry":[{},{},"expression of a gene, e.g."]},{"entry":[{},{},"IPTG for the lacUV5"]},{"entry":[{},{},"promoter"]},{"entry":[{},"OCV","Operative Cloning Vector"]},{"entry":[{},"K","A bimolecular dissociation"]},{"entry":[{},{},"constant, K= [A] [B]\/"]},{"entry":[{},{},"[A:B]"]},{"entry":[{},"K","K= [T] [SBD]\/[T:SBD] (T"]},{"entry":[{},{},"is a target)"]},{"entry":[{},"K","K= [N] [SBD]\/[N:SBD] (N"]},{"entry":[{},{},"is a non-target)"]},{"entry":[{},"DoAMoM","Density of AfM(W) on"]},{"entry":[{},{},"affinity matrix"]},{"entry":[{},"mfaa","Most-Favored amino acid"]},{"entry":[{},"lfaa","Least-Favored amino acid"]},{"entry":[{},"Abun(x)","Abundance of DNA molecules"]},{"entry":[{},{},"encoding amino acid x"]},{"entry":[{},"OMP","Outer membrane protein"]},{"entry":[{},"nt","nucleotide"]},{"entry":[{},"SP-I","Signal-sequence Peptidase I"]},{"entry":[{},"Y","Yield of ssDNA up to Q"]},{"entry":[{},{},"bases long"]},{"entry":[{},"M","Maximum length of ssDNA"]},{"entry":[{},{},"that can be synthesized in"]},{"entry":[{},{},"acceptable yield"]},{"entry":[{},"Y","Yield of plasmid DNA per"]},{"entry":[{},{},"volume of culture"]},{"entry":[{},"L","DNA ligation efficiency"]},{"entry":[{},"M","Maximum number of"]},{"entry":[{},{},"transformants produced"]},{"entry":[{},{},"from YDNA of Insert"]},{"entry":[{},"C","Efficiency of"]},{"entry":[{},{},"chromatographic"]},{"entry":[{},{},"enrichment, enrichment per"]},{"entry":[{},{},"pass"]},{"entry":[{},"C","Sensitivity of"]},{"entry":[{},{},"chromatographic"]},{"entry":[{},{},"separation, can find 1 in"]},{"entry":[{},{},"N,"]},{"entry":[{},"N","Maximum number of"]},{"entry":[{},{},"enrichment cycles per"]},{"entry":[{},{},"variegation cycle"]},{"entry":[{},"S","Error level in"]},{"entry":[{},{},"synthesizing vgDNA"]},{"entry":[{},"::","in-frame genetic fusion or"]},{"entry":[{},{},"protein produced from in-"]},{"entry":[{},{},"frame fused gene"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}}},"br":[{},{},{}]},"The initial potential binding domain may be: 1) a domain of a naturally occurring protein, 2) a non-naturally occurring domain which substantially corresponds in sequence to a naturally occurring domain, but which differs from it in sequence by one or more substitutions, insertions or deletions, 3) a domain substantially corresponding in sequence to a hybrid of subsequences of two or more naturally occurring proteins, or 4) an artificial domain designed entirely on theoretical grounds based on knowledge of amino acid geometries and statistical evidence of secondary structure preferences of amino acids. (However, the limitations of a priori protein design prompted the present invention.) Usually, the domain will be a known binding domain, or at least a homologue thereof, but it may be derived from a protein which, while not possessing a known binding activity, possesses a secondary or higher structure that lends itself to binding activity (clefts, grooves, etc.). The protein to which the IPBD is related need not have any specific affinity for the target material.","In determining whether sequences should be deemed to \u201csubstantially correspond\u201d, one should consider the following issues: the degree of sequence similarity when the sequences are aligned for best fit according to standard algorithms, the similarity in the connectivity patterns of any crosslinks (e.g., disulfide bonds), the degree to which the proteins have similar three-dimensional structures, as indicated by, e.g., X-ray diffraction analysis or NMR, and the degree to which the sequenced proteins have similar biological activity. In this context, it should be noted that among the serine protease inhibitors, there are families of proteins recognized to be homologous in which there are pairs of members with as little as 30% sequence homology.","A candidate IPBD should meet the following criteria:\n\n","Preferably, the IPBD is no larger than necessary because small SBDs (for example, less than 30 amino acids) can be chemically synthesized and because it is easier to arrange restriction sites in smaller amino-acid sequences. For PBDs smaller than about 40 residues, an added advantage is that the entire variegated pbd gene can be synthesized in one piece. In that case, we need arrange only suitable restriction sites in the osp gene. A smaller protein minimizes the metabolic strain on the GP or the host of the GP. The IPBD is preferably smaller than about 200 residues. The IPBD must also be large enough to have acceptable binding affinity and specificity. For an IPBD lacking covalent crosslinks, such as disulfide bonds, the IPBD is preferably at least 40 residues; it may be as small as six residues if it contains a crosslink. These small, crosslinked IPBDs, known as \u201cmini-proteins\u201d, are discussed in more detail later in this section.","Some candidate IPBDs, which meet the conditions set forth above, will be more suitable than others. Information about candidate IPBDs that will be used to judge the suitability of the IPBD includes: 1) a 3D structure (knowledge strongly preferred), 2) one or more sequences homologous to the IPBD (the more homologous sequences known, the better), 3) the pI of the IPBD (knowledge desirable when target is highly charged), 4) the stability and solubility as a function of temperature, pH and ionic strength (preferably known to be stable over a wide range and soluble in conditions of intended use), 5) ability to bind metal ions such as Ca or Mg (knowledge preferred; binding per se, no preference), 6) enzymatic activities, if any (knowledge preferred, activity per se has uses but may cause problems), 7) binding properties, if any (knowledge preferred, specific binding also preferred), 8) availability of a molecule having specific and strong affinity (K<10M) for the IPBD (preferred), 9) availability of a molecule having specific and medium affinity (10M<K<M) for the IPBD (preferred), 10) the sequence of a mutant of IPBD that does not bind to the affinity molecule(s) (preferred), and 11) absorption spectrum in visible, UV, NMR, etc. (characteristic absorption preferred).","If only one species of molecule having affinity for IPBD (AfM(IPBD)) is available, it will be used to: a) detect the IPBD on the GP surface, b) optimize expression level and density of the affinity molecule on the matrix, and c) determine the efficiency and sensitivity of the affinity separation. As noted above, however, one would prefer to have available two species of AfM(IPBD), one with high and one with moderate affinity for the IPBD. The species with high affinity would be used in initial detection and in determining efficiency and sensitivity, and the species with moderate affinity would be used in optimization.","If the IPBD is not itself a binding domain of a known binding protein, or if its native target has not been purified, an antibody raised against the IPBD may be used as the affinity molecule. Use of an antibody for this purpose should not be taken to mean that the antibody is the ultimate target.","There are many candidate IPBDs for which all of the above information is available or is reasonably practical to obtain, for example, bovine pancreatic trypsin inhibitor (BPTI, 58 residues), CMTI-III (29 residues), crambin (46 residues), third domain of ovomucoid (56 residues), heat-stable enterotoxin (ST-Ia of ) (18 residues), \u03b1-Conotoxin GI (13 residues), \u03bc-Conotoxin GIII (22 residues), Conus King Kong mini-protein (27 residues), T4 lysozyme (164 residues), and azurin (128 residues). Structural information can be obtained from X-ray or neutron diffraction studies, NMR, chemical cross linking or labeling, modeling from known structures of related proteins, or from theoretical calculations. 3D structural information obtained by X-ray diffraction, neutron diffraction or NMR is preferred because these methods allow localization of almost all of the atoms to within defined limits. Table 50 lists several preferred IPBDs. Works related to determination of 3D structure of small proteins via NMR inculde: CHAZ85, PEAS90, PEAS88, CLOR86, CLOR87a, HEIT89, LECO87, WAGN79, and PARD89.","In some cases, a protein having some affinity for the target may be a preferred IPBD even though some other criteria are not optimally met. For example, the V1 domain of CD4 is a good choice as IPBD for a protein that binds to gp120 of HIV. It is known that mutations in the region 42 to 55 of V1 greatly affect gp120 binding and that other mutations either have much less effect or completely disrupt the structure of V1. Similarly, tumor necrosis factor (TNF) would be a good initial choice if one wants a TNF-like molecule having higher affinity for the TNF receptor.","Membrane-bound proteins are not preferred IPBPs, though they may serve as a source of outer surface transport signals. One should distinguish between membrane-bound proteins, such as LamB or OmpF, that cross the membrane several times forming a structure that is embedded in the lipid bilayer and in which the exposed regions are the loops that join trans-membrane segments, from non-embedded proteins, such as the soluble domains of CD4, that are simply anchored to the membrane. This is an important distinction because it is quite difficult to create a soluble derivative of a membrane-bound protein. Soluble binding proteins are in general more useful since purification is simpler and they are more tractable and more versatile assay reagents.","Most of the PBDs derived from a PPBD according to the process of the present invention will have been derived by variegation at residues having side groups directed toward the solvent. Reidhaar-Olson and Sauer (REID88a) found that exposed residues can accept a wide range of amino acids, while buried residues are more limited in this regard. Surface mutations typically have only small effects on melting temperature of the PBD, but may reduce the stability of the PBD. Hence the chosen IPBD should have a high melting temperature (50\u00b0 C. acceptable, the higher the better; BPTI melts at 95\u00b0 C.) and be stable over a wide pH range (8.0 to 3.0 acceptable; 11.0 to 2.0 preferred), so that the SBDs derived from the chosen IPBD by mutation and selection-through-binding will retain sufficient stability. Preferably, the substitutions in the IPBD yielding the various PBDs do not reduce the melting point of the domain below \u224840\u00b0 C. Mutations may arise that increase the stability of SBDs relative to the IPBD, but the process of the present invention does not depend upon this occurring. Proteins containing covalent crosslinks, such as multiple disulfides, are usually sufficient stable. A protein having at least two disulfides and having at least 1 disulfide per every twenty residues may be presumed to be sufficiently stable.","Two general characteristics of the target molecule, size and charge, make certain classes of IPBDs more likely than other classes to yield derivatives that will bind specifically to the target. Because these are very general characteristics, one can divide all targets into six classes: a) large positive, b) large neutral, c) large negative, d) small positive, e) small neutral, and f) small negative. A small collection of IPBDs, one or a few corresponding to each class of target, will contain a preferred candidate IPBD for any chosen target.","Alternatively, the user may elect to engineer a GP(IPBD) for a particular target; criteria are given below that relate target size and charge to the choice of IPBD.","II.B. Influence of Target Size on Choice of IPBD","If the target is a protein or other macromolecule a preferred embodiment of the IPBD is a small protein such as the trypsin inhibitor III (29 residues), BPTI from (58 residues), crambin from rape seed (46 residues), or the third domain of ovomucoid from (Japanese quail) (56 residues), because targets from this class have clefts and grooves that can accommodate small proteins in highly specific ways. If the target is a macromolecule lacking a compact structure, such as starch, it should be treated as if it were a small molecule. Extended macromolecules with defined 3D structure, such as collagen, should be treated as large molecules.","If the target is a small molecule, such as a steroid, a preferred embodiment of the IPBD is a protein of about 80-200 residues, such as ribonuclease from (124 residues), ribonuclease from (104 residues), hen egg white lysozyme from (129 residues), azurin from (128 residues), or T4 lysozyme (164 residues), because such proteins have clefts and grooves into which the small target molecules can fit. The Brookhaven Protein Data Bank contains 3D structures for all of the proteins listed. Genes encoding proteins as large as T4 lysozyme can be manipulated by standard techniques for the purposes of this invention.","If the target is a mineral, insoluble in water, one considers the nature of the molecular surface of the mineral. Minerals that have smooth surfaces, such as crystalline silicon, are best addressed with medium to large proteins, such as ribonuclease, as IPBD in order to have sufficient contact area and specificity. Minerals with rough, grooved surfaces, such as zeolites, could be bound either by small proteins, such as BPTI, or larger proteins, such as T lysozyme.","II.C. Influence of Target Charge on Choice of IPBD","Electrostatic repulsion between molecules of like charge can prevent molecules with highly complementary surfaces from binding. Therefore, it is preferred that, under the conditions of intended use, the IPBD and the target molecule either have opposite charge or that one of them is neutral. In some cases it has been observed that protein molecules bind in such a way that like charged groups are juxtaposed by including oppositely charged counter ions in the molecular interface. Thus, inclusion of counter ions can reduce or eliminate electrostatic repulsion and the user may elect to include ions in the eluants used in the affinity separation step. Polyvalent ions are more effective at reducing repulsion than monovalent ions.","II.D. Other Considerations in the Choice of IPBD","If the chosen IPBD is an enzyme, it may be necessary to change one or more residues in the active site to inactivate enzyme function. For example, if the IPBD were T4 lysozyme and the GP were cells or M, we would need to inactivate the lysozyme because otherwise it would lyse the cells. If, on the other hand, the GP were \u03a6X174, then inactivation of lysozyme may not be needed because T4 lysozyme can be overproduced inside cells without detrimental effects and \u03a6X174 forms intracellularly. It is preferred to inactivate enzyme IPBDs that might be harmful to the GP or its host by substituting mutant amino acids at one or more residues of the active site. It is permitted to vary one or more of the residues that were changed to abolish the original enzymatic activity of the IPBD. Those GPs that receive osp-pbd genes encoding an active enzyme may die, but the majority of sequences will not be deleterious.","If the binding protein is intended for therapeutic use in humans or animals, the IPBD may be chosen from proteins native to the designated recipient to minimize the possibility of antigenic reactions.","II.E. Bovine Pancreatic Trypsin Inhibitor (BPTI) as an IPBD","BPTI is an especially preferred IPBD because it meets or exceeds all the criteria: it is a small, very stable protein with a well known 3D structure. Marks et al. (MARK86) have shown that a fusion of the phoA signal peptide gene fragment and DNA coding for the mature form of BPTI caused native BPTI to appear in the periplasm of demonstrating that there is nothing in the structure of BPTI to prevent its being secreted.","The structure of BPTI is maintained even when one or another of the disulfides is removed, either by chemical blocking or by genetic alteration of the amino-acid sequence. The stabilizing influence of the disulfides in BPTI is not equally distributed. Goldenberg (GOLD85) reports that blocking CYS14 and CYS38 lowers the Tm of BPTI to \u224875\u00b0 C. while chemical blocking of either of the other disulfides lowers Tm to below 40\u00b0 C. Chemically blocking a disulfide may lower Tm more than mutating the cysteines to other amino-acid types because the bulky blocking groups are more destabilizing than removal of the disulfide. Marks et al. (MARK87) replaced both CYS14 and CYS38 with either two alanines or two threonines. The CYS14\/CYS38 cystine bridge that Marks et al. removed is the one very close to the scissile bond in BPTI; surprisingly, both mutant molecules functioned as trypsin inhibitors. Schnabel et al. (SCHN86) report preparation of aprotinin (C14A, C38A) by use of Raney nickel. Eigenbrot et al. (EIGE90) report the X-ray structure of BPTI (C30A\/C51A) which is stable to at least 50\u00b0 C. The backbone of this mutant is as similar to BPTI as are the backbones of BPTI molecules that sit in different crystal lattices. This indicates that BPTI is redundantly stable and so is likely to fold into approximately the same structure despite numerous surface mutations. Using the knowledge of homologues, vide infra, we can infer which residues should not be varied if the basic BPTI structure is to be maintained.","The 3D structure of BPTI has been determined at high resolution by X-ray diffraction (HUBE77, MARQ83, WLOD84, WLOD87a, WLOD87b), neutron diffraction (WLOD84), and by NMR (WAGN87). In one of the X-ray structures deposited in the Brookhaven Protein Data Bank, entry 6PTI, there was no electron density for A58, indicating that A58 has no uniquely defined conformation. Thus we know that the carboxy group does not make any essential interaction in the folded structure. The amino terminus of BPTI is very near to the carboxy terminus. Goldenberg and Creighton reported on circularized BPTI and circularly permuted BPTI (GOLD83). Some proteins homologous to BPTI have more or fewer residues at either terminus.","BPTI has been called \u201cthe hydrogen atom of protein folding\u201d and has been the subject of numerous experimental and theoretical studies (STAT87, SCHW87, GOLD83, CHAZ83, CREI74, CREI77a, CREI77b, CREI80, SIEK87, SINH90, RUEH73, HUBE74, HUBE75, HUBE77 and others).","BPTI has the added advantage that at least 59 homologous proteins are known. Table 13 shows the sequences of 39 homologues. A tally of ionizable groups in 59 homologues is shown in Table 14 and the composite of amino acid types occurring at each residue is shown in Table 15.","BPTI is freely soluble and is not known to bind metal ions. BPTI has no known enzymatic activity. BPTI is not toxic.","All of the conserved residues are buried; of the six fully conserved residues only G37 has noticeable exposure. The solvent accessibility of each residue in BPTI is given in Table 16 which was calculated from the entry \u201c6PTI\u201d in the Brookhaven Protein Data Bank with a solvent radius of 1.4 \u212b, the atomic radii given in Table 7, and the method of Lee and Richards (LEEB71). Each of the 52 non-conserved residues can accommodate two or more kinds of amino acids. By independently substituting at each residue only those amino acids already observed at that residue, we could obtain approximately 1.6\u00b710different amino acid sequences, most of which will fold into structures very similar to BPTI.","BPTI will be especially useful as a IPBD for macromolecular targets. BPTI and BPTI homologues bind tightly and with high specificity to a number of enzyme macromolecules.","BPTI is strongly positively charged except at very high pH, thus BPTI is useful as IPBD for targets that are not also strongly positive under the conditions of intended use. There exist homologues of BPTI, however, having quite different charges (viz. SCI-III from at \u22127 and the trypsin inhibitor from bovine colostrum at \u22121). Once a genetic package is found that displays BPTI on its surface, the sequence of the BPTI domain can be replaced by one of the homologous sequences to produce acidic or neutral IPBDs.","BPTI is quite small; if this should cause a pharmacological problem, two or more BPTI-derived domains may be joined as in humans BPTI homologues, one of which has two domains (BALD85, ALBR83b) and another has three (WUNT88).","Another possible pharmacological problem is immun igenicity. BPTI has been used in humans with very few adverse effects. Siekmann et al. (SIEK89) have studied immunological characteristics of BPTI and some homologues. It is an advantage of the method of the present invention that a variety of SBDs can be obtained so that, if one derivative proves to be antigenic, a different SBD may be used. Furthermore, one can reduce the probability of immune response by starting with a human protein, such as LACI (a BPTI homologue) (WUNT88, GIRA89) or Inter-\u03b1-Trypsin Inhibitor (ALBR83a, ALBR83b, DIAR90, ENGH89, TRIB86, GEBH86, GEBH90, KAUM86, ODOM90, SALI90).","Further, a BPTI-derived gene fragment, coding for a novel binding domain, could be fused in-frame to a gene fragment coding for other proteins, such as serum albumin or the constant parts of IgG.","Tschesche et al. (TSCH87) reported on the binding of several BPTI derivatives to various proteases:","Dissociation constants for BPTI derivatives, Molar.",{"@attributes":{"id":"p-0130","num":"0210"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"center"}}],"thead":{"row":[{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":[{},"Trypsin","Chymotrypsin","Elastase","Elastase"]},{"entry":["Residue","(bovine","(bovine","(porcine","(human"]},{"entry":["#15","pancreas)","pancreas)","pancreas)","leukocytes)"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["lysine","6.0 \u00b7 10","9.0 \u00b7 10","\u2212","3.5 \u00b7 10"]},{"entry":["glycine","\u2212","\u2212","+","7.0 \u00b7 10"]},{"entry":["alanine","+","\u2212","2.8 \u00b7 10","2.5 \u00b7 10"]},{"entry":["valine","\u2212","\u2212","5.7 \u00b7 10",{"sup":["\u2002","\u221210"]}]},{"entry":["leucine","\u2212","\u2212","1.9 \u00b7 10","2.9 \u00b7 10"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}}},"br":{},"sub":["d","d","d ","d "],"sup":["\u22126 ","\u22126 ","\u221211 ","\u22126 "]},"Works concerning BPTI and its homologues include:",{"@attributes":{"id":"p-0132","num":"0212"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"KIDO88,",{}]},{"entry":[{},{}]},{"entry":[{},"PONT88,"]},{"entry":[{},{}]},{"entry":[{},"KIDO90,"]},{"entry":[{},{}]},{"entry":[{},"AUER87,"]},{"entry":[{},{}]},{"entry":[{},"AUER90,"]},{"entry":[{},{}]},{"entry":[{},"SCOT87b,"]},{"entry":[{},{}]},{"entry":[{},"AUER88,"]},{"entry":[{},{}]},{"entry":[{},"AUER89,"]},{"entry":[{},{}]},{"entry":[{},"BECK88b,"]},{"entry":[{},{}]},{"entry":[{},"WACH79,"]},{"entry":[{},{}]},{"entry":[{},"WACH80,"]},{"entry":[{},{}]},{"entry":[{},"BECK89a,"]},{"entry":[{},{}]},{"entry":[{},"DUFT85,"]},{"entry":[{},{}]},{"entry":[{},"FIOR88,"]},{"entry":[{},{}]},{"entry":[{},"GIRA89,"]},{"entry":[{},{}]},{"entry":[{},"GOLD84,"]},{"entry":[{},{}]},{"entry":[{},"GOLD88,"]},{"entry":[{},{}]},{"entry":[{},"HOCH84,"]},{"entry":[{},{}]},{"entry":[{},"RITO83,"]},{"entry":[{},{}]},{"entry":[{},"NORR89a,"]},{"entry":[{},{}]},{"entry":[{},"NORR89b,"]},{"entry":[{},{}]},{"entry":[{},"OLTE89,"]},{"entry":[{},{}]},{"entry":[{},"SWAI88,"]},{"entry":[{},"and"]},{"entry":[{},{}]},{"entry":[{},"WAGN79."]}]}}}},"br":{}},"A polypeptide is a polymer composed of a single chain of the same or different amino acids joined by peptide bonds. Linear peptides can take up a very large number of different conformations through internal rotations about the main chain single bonds of each \u03b1 carbon. These rotations are hindered to varying degrees by side groups, with glycine interfering the least, and valine, isoleucine and, especially, proline, the most. A polypeptide of 20 residues may have 10different conformations which it may assume by various internal rotations.","Proteins are polypeptides which, as a result of stabilizing interactions between amino acids that are not in adjacent positions in the chain, have folded into a well-defined conformation. This folding is usually essential to their biological activity.","For polypeptides of 40-60 residues or longer, noncovalent forces such as hydrogen bonds, salt bridges, and hydrophobic \u201cinteractions\u201d are sufficient to stabilize a particular folding or conformation. The polypeptide's constituent segments are held to more or less that conformation unless it is perturbed by a denaturant such as rising temperature or decreasing pH, whereupon the polypeptide unfolds or \u201cmelts\u201d. The smaller the peptide, the more likely it is that its conformation will be determined by the environment. If a small unconstrained peptide has biological activity, the peptide ligand will be in essence a random coil until it comes into proximity with its receptor. The receptor accepts the peptide only in one or a few conformations because alternative conformations are disfavored by unfavorable van der Waals and other non-covalent interactions.","Small polypeptides have potential advantages over larger polypeptides when used as therapeutic or diagnostic agents, including (but not limited to):\n\n","Moreover, polypeptides of under about 50 residues have the advantage of accessibility via chemical synthesis; polypeptides of under about 30 residues are more easily synthesized than are larger polypeptides. Thus, it would be desirable to be able to employ the combination of variegation and affinity selection to identify small polypeptides which bind a target of choice.","Polypeptides of this size, however, have disadvantages as binding molecules. According to Olivera et al. (OLIV90a): \u201cPeptides in this size range normally equilibrate among many conformations (in order to have a fixed conformation, proteins generally have to be much larger).\u201d Specific binding of a peptide to a target molecule requires the peptide to take up one conformation that is complementary to the binding site. For a decapeptide with three isoenergetic conformations (e.g., \u03b2 strand, \u03b1 helix, and reverse turn) at each residue, there are about 6\u00b710possible overall conformations. Assuming these conformations to be equi-probable for the unconstrained decapeptide, if only one of the possible conformations bound to the binding site, then the affinity of the peptide for the target is expected to be about 6\u00b710higher if it could be constrained to that single effective conformation. Thus, the unconstrained decapeptide, relative to a decapeptide constrained to the correct conformation, would be expected to exhibit lower affinity. It would also exhibit lower specificity, since one of the other conformations of the unconstrained decapeptide might be one which bound tightly to a material other than the intended target. By way of corollary, it could have less resistance to degradation by proteases, since it would be more likely to provide a binding site for the protease.","In one embodiment, the present invention overcomes these problems, while retaining the advantages of smaller polypeptides, by fostering the biosynthesis of novel mini-proteins having the desired binding characteristics. Mini-Proteins are small polypeptides (usually less than about 60 residues) which, while too small to have a stable conformation as a result of noncovalent forces alone, are covalently crosslinked (e.g., by disulfide bonds) into a stable conformation and hence have biological activities more typical of larger protein molecules than of unconstrained polypeptides of comparable size.","When mini-proteins are variegated, the residues which are covalently crosslinked in the parental molecule are left unchanged, thereby stabilizing the conformation. For example, in the variegation of a disulfide bonded mini-protein, certain cysteines are invariant so that under the conditions of expression and display, covalent crosslinks (e.g., disulfide bonds between one or more pairs of cysteines) form, and substantially constrain the conformation which may be adopted by the hypervariable linearly intermediate amino acids. In other words, a constraining scaffolding is engineered into polypeptides which are otherwise extensively randomized.","Once a mini-protein of desired binding characteristics is characterized, it may be produced, not only by recombi nant DNA techniques, but also by nonbiological synthetic methods.","In vitro, disulfide bridges can form spontaneously in polypeptides as a result of air oxidation. Matters are more complicated in vivo. Very few intracellular proteins have disulfide bridges, probably because a strong reducing environment is maintained by the glutathione system. Disulfide bridges are common in proteins that travel or operate in extracellular spaces, such as snake venoms and other toxins (e.g., conotoxins, charybdotoxin, bacterial enterotoxins), peptide hormones, digestive enzymes, complement proteins, immunoglobulins, lysozymes, protease inhibitors (BPTI and its homologues, CMTI-III (trypsin inhibitor III) and its homologues, hirudin, etc.) and milk proteins.","Disulfide bonds that close tight intrachain loops have been found in pepsin, thioredoxin, insulin A-chain, silk fibroin, and lipoamide dehydrogenase. The bridged cysteine residues are separated by one to four residues along the polypeptide chain. Model building, X-ray diffraction analysis, and NMR studies have shown that the \u03b1 carbon path of such loops is usually flat and rigid.","There are two types of disulfide bridges in immunoglobulins. One is the conserved intrachain bridge, spanning about 60 to 70 amino acid residues and found, repeatedly, in almost every immunoglobulin domain. Buried deep between the opposing \u03b2 sheets, these bridges are shielded from solvent and ordinarily can be reduced only in the presence of denaturing agents. The remaining disulfide bridges are mainly interchain bonds and are located on the surface of the molecule; they are accessible to solvent and relatively easily reduced (STEI85). The disulfide bridges of the mini-proteins of the present invention are intrachain linkages between cysteines having much smaller chain spacings.","For the purpose of the appended claims, a mini-protein has between about eight and about sixty residues. However, it will be understood that a chimeric surface protein presenting a mini-protein as a domain will normally have more than sixty residues. Polypeptides containing intrachain disulfide bonds may be characterized as cyclic in nature, since a closed circle of covalently bonded atoms is defined by the two cysteines, the intermediate amino acid residues, their peptidyl bonds, and the disulfide bond. The terms \u201ccycle\u201d, \u201cspan\u201d and \u201csegment\u201d will be used to define certain structural features of the polypeptides. An intrachain disulfide bridge connecting amino acids 3 and 8 of a 16 residue polypeptide will be said herein to have a cycle of 6 and a span of 4. If amino acids 4 and 12 are also disulfide bonded, then they form a second cycle of 9 with a span of 7. Together, the four cysteines divide the polypeptide into four inter cysteine segments (1-2, 5-7, 9-11, and 13-16). (Note that there is no segment between Cys3 and Cys4.)","The connectivity pattern of a crosslinked mini-protein is a simple description of the relative location of the termini of the crosslinks. For example, for a mini-protein with two disulfide bonds, the connectivity pattern \u201c1-3, 2-4\u201d means that the first crosslinked cysteine is disulfide bonded to the third crosslinked cysteine (in the primary sequence), and the second to the fourth.","The degree to which the crosslink constrains the conformational freedom of the mini-protein, and the degree to which it stabilizes the mini-protein, may be assessed by a number of means. These include absorption spectroscopy (which can reveal whether an amino acid is buried or exposed), circular dichroism studies (which provides a general picture of the helical content of the protein), nuclear magnetic resonance imaging (which reveals the number of nuclei in a particular chemical environment as well as the mobility of nuclei), and X-ray or neutron diffraction analysis of protein crystals. The stability of the mini-protein may be ascertained by monitoring the changes in absorption at various wavelengths as a function of temperature, pH, etc.; buried residues become exposed as the protein unfolds. Similarly, the unfolding of the mini-protein as a result of denaturing conditions results in changes in NMR line positions and widths. Circular dichroism (CD) spectra are extremely sensitive to conformation.","The variegated disulfide-bonded mini-proteins of the present invention fall into several classes.","Class I mini-proteins are those featuring a single pair of cysteines capable of interacting to form a disulfide bond, said bond having a span of no more than nine residues. This disulfide bridge preferably has a span of at least two residues; this is a function of the geometry of the disulfide bond. When the spacing is two or three residues, one residue is preferably glycine in order to reduce the strain on the bridged residues. The upper limit on spacing is less precise, however, in general, the greater the spacing, the less the constraint on conformation imposed on the linearly intermediate amino acid residues by the disulfide bond.","The main chain of such a peptide has very little freedom, but is not stressed. The free energy released when the disulfide forms exceeds the free energy lost by the main-chain when locked into a conformation that brings the cysteines together. Having lost the free energy of disulfide formation, the proximal ends of the side groups are held in more or less fixed relation to each other. When binding to a target, the domain does not need to expend free energy getting into the correct conformation. The domain can not jump into some other conformation and bind a non-target.","A disulfide bridge with a span of 4 or 5 is especially preferred. If the span is increased to 6, the constraining influence is reduced. In this case, we prefer that at least one of the enclosed residues be an amino acid that imposes restrictions on the main-chain geometry. Proline imposes the most restriction. Valine and isoleucine restrict the main chain to a lesser extent. The preferred position for this constraining non-cysteine residue is adjacent to one of the invariant cysteines, however, it may be one of the other bridged residues. If the span is seven, we prefer to include two amino acids that limit main-chain conformation. These amino acids could be at any of the seven positions, but are preferably the two bridged residues that are immediately adjacent to the cysteines. If the span is eight or nine, additional constraining amino acids may be provided.","The disulfide bond of a class I mini-proteins is exposed to solvent. Thus, one should avoid exposing the variegated population of GPs that display class I mini-proteins to reagents that rupture disulfides; Creighton names several such reagents (CREI88).","Class II mini-proteins are those featuring a single disulfide bond having a span of greater than nine amino acids. The bridged amino acids form secondary structures which help to stabilize their conformation. Preferably, these intermediate amino acids form hairpin supersecondary structures such as those schematized below:",{"@attributes":{"id":"p-0154","num":"0238"},"chemistry":{"@attributes":{"id":"CHEM-US-00001","num":"00001"},"img":{"@attributes":{"id":"EMI-C00001","he":"35.98mm","wi":"31.58mm","file":"US07413537-20080819-C00001.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{}},"The \u03b2 strand may be considered an elongated helix with 2.3 residues per turn, a translation of 3.3 \u212b per residue, and a helical radius of 1.0 \u212b. Alone, a \u03b2 strand forms no main-chain hydrogen bonds. Most commonly, \u03b2 strands are found in twisted (rather than planar) parallel, antiparallel, or mixed parallel\/antiparallel sheets.","A peptide chain can form a sharp reverse turn. A reverse turn may be accomplished with as few as four amino acids. Reverse turns are very abundant, comprising a quarter of all residues in globular proteins. In proteins, reverse turns commonly connect \u03b2 strands to form \u03b2 sheets, but may also form other connections. A peptide can also form other turns that are less sharp.","Based on studies of known proteins, one may calculate the propensity of a particular residue, or of a particular dipeptide or tripeptide, to be found in an \u03b1 helix, \u03b2 strand or reverse turn. The normalized frequencies of occurrence of the amino acid residues in these secondary structures is given in Table 6-4 of CREI84. For a more detailed treatment on the prediction of secondary structure from the amino acid sequence, see Chapter 6 of SCHU79.","In designing a suitable hairpin structure, one may copy an actual structure from a protein whose three-dimensional conformation is known, design the structure using frequency data, or combine the two approaches. Preferably, one or more actual structures are used as a model, and the frequency data is used to determine which mutations can be made without disrupting the structure.","Preferably, no more than three amino acids lie between the cysteine and the beginning or end of the \u03b1 helix or \u03b2 strand.","More complex structures (such as a double hairpin) are also possible.","Class III mini-proteins are those featuring a plurality of disulfide bonds. They optionally may also feature secondary structures such as those discussed above with regard to Class II mini-proteins. Since the number of possible disulfide bond topologies increases rapidly with the number of bonds (two bonds, three topologies; three bonds, 15 topologies; four bonds, 105 topologies) the number of disulfide bonds preferably does not exceed four. With two or more disulfide bonds, the disulfide bridge spans preferably do not exceed 50, and the largest intercysteine chain segment preferably does not exceed 20.","Naturally occurring class III mini-proteins, such as heat-stable enterotoxin ST-Ia frequently have pairs of cysteines that are adjacent in the amino-acid sequence. Adjacent cysteines are very unlikely to form an intramolecular disulfide and cysteines separated by a single amino acids form an intramolecular disulfide with difficulty and only for certain intervening amino acids. Thus, clustering cysteines within the amino-acid sequence reduces the number of realizable disulfide bonding schemes. We utilize such clustering in the class III mini-protein disclosed herein.","Metal Finger Mini-Proteins. The mini-proteins of the present invention are not limited to those crosslinked by disulfide bonds. Another important class of mini-proteins are analogues of finger proteins. Finger proteins are characterized by finger structures in which a metalion is coordinated by two Cys and two His residues, forming a tetrahedral arrangement around it. The metalion is most often zinc(II), but may be iron, copper, cobalt, etc. The \u201cfinger\u201d has the consensus sequence (Phe or Tyr)-(1 AA)-Cys -(2-4 AAs)-Cys-(3 AAs)-Phe-(5 AAs)-Leu-(2 AAs)-His-(3 AAs)-His -(5 AAs)(SEQ ID NOs: 1,2,3,4,5,6)(BERG88; GIBS88). While finger proteins typically contain many repeats of the finger motif, it is known that a single finger will fold in the presence of zinc ions (FRAN87; PARR88). There is some dispute as to whether two fingers are necessary for binding to DNA. The present invention encompasses mini-proteins with either one or two fingers. It is to be understood that the target need not be a nucleic acid.","G. Modified PBSs","There exist a number of enzymes and chemical reagents that can selectively modify certain side groups of proteins, including: a) protein-tyrosine kinase, Ellmans reagent, methyl transferases (that methylate GLU side groups), serine kinases, proline hydroxyases, vitamin-K dependent enzymes that convert GLU to GLA, maleic anhydride, and alkylating agents. Treatment of the variegated population of GP(PBD)s with one of these enzymes or reagents will modify the side groups affected by the chosen enzyme or reagent. Enzymes and reagents that do not kill the GP are much preferred. Such modification of side groups can directly affect the binding properties of the displayed PBDs. Using affinity separation methods, we enrich for the modified GPs that bind the predetermined target. Since the active binding domain is not entirely genetically specified, we must repeat the post-morphogenesis modification at each enrichment round. This approach is particularly appropriate with mini-protein IPBDs because we envision chemical synthesis of these SBDs.","III.A. Generally","Using standard genetic engineering techniques, a molecule of variegated DNA can be introduced into a vector so that it constitutes part of a gene (OLIP86, OLIP87, AUSU87, REID88a). When vector containing variegated DNA are used to transform bacteria, each cell makes a version of the original protein. Each colony of bacteria may produce a different version from any other colony. If the variegations of the DNA are concentrated at loci known to be on the surface of the protein or in a loop, a population of proteins will be generated, many members of which will fold into roughly the same 3D structure as the parent protein. The specific binding properties of each member, however, may be different from each other member.","We now consider the manner in which we generate a diverse population of potential binding domains in order to facilitate selection of a PBD-bearing GP which binds with the requisite affinity to the target of choice. The potential binding domains are first designed at the amino acid level. Once we have identified which residues are to be mutagenized, and which mutations to allow at those positions, we may then design the variegated DNA which is to encode the various PBDs so as to assure that there is a reasonable probability that if a PBD has an affinity for the target, it will be detected. Of course, the number of independent transformants obtained and the sensitivity of the affinity separation technology will impose limits on the extent of variegation possible within any single round of variegation.","There are many ways to generate diversity in a protein. (See RICH86, CARU85, and OLIP86.) At one extreme, we vary a few residues of the protein as much as possible (inter alia see CARU85, CARU87, RICH86, and WHAR86). We will call this approach \u201cFocused Mutagenesis\u201d. A typical \u201cFocused Mutagenesis\u201d strategy is to pick a set of five to seven residues and vary each through 13-20 possibilities. An alternative plan of mutagenesis (\u201cDiffuse Mutagenesis\u201d) is to vary many more residues through a more limited set of choices (See VERS86a and PAKU86). The variegation pattern adopted may fall between these extremes, e.g., two residues varied through all twenty amino acids, two more through only two possibilities, and a fifth into ten of the twenty amino acids.","There is no fixed limit on the number of codons which can be mutated simultaneously. However, it is desirable to adopt a mutagenesis strategy which results in a reasonable probability that a possible PBD sequence is in fact displayed by at least one genetic package. When the size of the set of amino acids potentially encoded by each variable codon is the same for all variable codons and within the set all amino acids are equiprobable, this probability may be calculated as follows: Let \u0393(k,q) be the probability that amino acid number k will occur at variegated codon q; these codons need not be contiguous. The probability that a particular vgDNA molecule will encode a PBD containing n variegated amino acids k, . . . , kis:\n\n()=\u0393(, 1)\u00b7 . . . \u00b7\u0393()\n\nConsider a library of Nindependent transformants prepared with said vgDNA; the probability that the sequence k, . . . , kis absent is:\n\n(missing )=exp{()}.\n\n(in lib)=1\u2212exp{()}.\n\nPreferably, the probability that a mutein encoded by the vgDNA and composed of the least favored amino acids at each variegated position will be displayed by at least one independent transformant in the library is at least 0.50, and more preferably at least 0.90. (Muteins composed of more favored amino acids would of course be more likely to occur in the same library.)\n","Preferably, the variegation is such as will cause a typical transformant population to display 10-10different amino acid sequences by means of preferably not more than 10-fold more (more preferably not more than 3-fold) different DNA sequences.","For a mini-protein that lacks a helices and \u03b2 strands, one will, in any given round of mutation, preferably variegate each of 4-6 non-cysteine codons so that they each encode at least eight of the 20 possible amino acids. The variegation at each codon could be customized to that position. Preferably, cysteine is not one of the potential substitutions, though it is not excluded.","When the mini-protein is a metal finger protein, in a typical variegation strategy, the two Cys and two His residues, and optionally also the aforementioned Phe\/Tyr, Phe and Leu residues, are held invariant and a plurality (usually 5-10) of the other residues are varied.","When the mini-protein is of the type featuring one or more \u03b1 helices and \u03b2 strands, the set of potential amino acid modifications at any given position is picked to favor those which are less likely to disrupt the secondary structure at that position. Since the number of possibilities at each variable amino acid is more limited, the total number of variable amino acids may be greater without altering the sampling efficiency of the selection process.","For the last-mentioned class of mini-proteins, as well as domains other than mini-proteins, preferably not more than 20 and more preferably 5-10 codons will be variegated. However, if diffuse mutagenesis is employed, the number of codons which are variegated can be higher.","The decision as to which residues to modify is eased by knowledge of which residues lie on the surface of the domain and which are buried in the interior.","We choose residues in the IPBD to vary through consideration of several factors, including: a) the 3D structure of the IPBD, b) sequences homologous to IPBD, and c) modeling of the IPBD and mutants of the IPBD. When the number of residues that could strongly influence binding is greater than the number that should be varied simultaneously, the user should pick a subset of those residues to vary at one time. The user picks trial levels of variegation and calculate the abundances of various sequences. The list of varied residues and the level of variegation at each varied residue are adjusted until the composite variegation is commensurate with the sensitivity of the affinity separation and the number of independent transformants that can be made.","Preferably, the abundance of PPBD-encoding DNA is 3 to 10 times higher than both 1\/Mand 1\/Cto provide a margin of redundancy. Mis the number of transformants that can be made from YDNA. With current technology Mntv is approximately 5\u00b710, but the exact value depends on the details of the procedures adapted by the user. Improvements in technology that allow more efficient: a) synthesis of DNA, b) ligation of DNA, or c) transformation of cells will raise the value of M. Cis the sensitivity of the affinity separation; improvements in affinity separation will raise C. If the smaller of Mand Csensi is increased, higher levels of variegation may be used. For example, if Cis 1 in 10and Mis 10, then improvements in Care less valuable than improvements in M.","While variegation normally will involve the substitution of one amino acid for another at a designated variable codon, it may involve the insertion or deletion of amino acids as well.","III.B. Identification of Residues to be Varied","We now consider the principles that guide our choice of residues of the IPBD to vary. A key concept is that only structured proteins exhibit specific binding, i.e. can bind to a particular chemical entity to the exclusion of most others. Thus the residues to be varied are chosen with an eye to preserving the underlying IPBD structure. Substitutions that prevent the PBD from folding will cause GPs carrying those genes to bind indiscriminately so that they can easily be removed from the population.","Sauer and colleagues (PAKU86, REID88a), and Caruthers and colleagues (EISE85) have shown that some residues on the polypeptide chain are more important than others in determining the 3D structure of a protein. The 3D structure is essentially unaffected by the identity of the amino acids at some loci; at other loci only one or a few types of amino acid is allowed. In most cases, loci where wide variety is allowed have the amino acid side group directed toward the solvent. Loci where limited variety is allowed frequently have the side group directed toward other parts of the protein. Thus substitutions of amino acids that are exposed to solvent are less likely to affect the 3D structure than are substitutions at internal loci. (See also SCHU79, p169-171 and CREI84, p239-245, 314-315).","The residues that join helices to helices, helices to sheets, and sheets to sheets are called turns and loops and have been classified by Richardson (RICH81), Thornton (THOR88), Sutcliffe et al. (SUTC87a) and others. Insertions and deletions are more readily tolerated in loops than elsewhere. Thornton et al. (THOR88) have summarized many observations indicating that related proteins usually differ most at the loops which join the more regular elements of secondary structure. (These observations are relevant not only to the variegation of potential binding domains but also to the insertion of binding domains into an outer surface protein of a genetic package, as discussed in a later section.)","Burial of hydrophobic surfaces so that bulk water is excluded is one of the strongest forces driving the binding of proteins to other molecules. Bulk water can be excluded from the region between two molecules only if the surfaces are complementary. We should test as many surface variations as possible to find one that is complementary to the target. The selection-through-binding isolates those proteins that are more nearly complementary to some surface on the target.","Proteins do not have distinct, countable faces. Therefore we define an \u201cinteraction set\u201d to be a set of residues such that all members of the set can simultaneously touch one molecule of the target material without any atom of the target coming closer than van der Waals distance to any main-chain atom of the IPBD. The concept of a residue \u201ctouching\u201d a molecule of the target is discussed below. From a picture of BPTI (such as , p. 225 of CREI84) we can see that residues 3, 7, 8, 10, 13, 39, 41, and 42 can all simultaneously contact a molecule the size and shape of myoglobin. We also see that residue 49 can not touch a single myoglobin molecule simultaneously with any of the first set even though all are on the surface of BPTI. (It is not the intent of the present invention, however, to suggest that use of models is required to determine which part of the target molecule will actually be the site of binding by PBD.)","Variations in the position, orientation and nature of the side chains of the residues of the interaction set will alter the shape of the potential binding surface defined by that set. Any individual combination of such variations may result in a surface shape which is a better or a worse fit for the target surface. The effective diversity of a variegated population is measured by the number of distinct shapes the potentially complementary surfaces of the PBD can adopt, rather than the number of protein sequences. Thus, it is preferable to maximize the former number, when our knowledge of the IPBD permits us to do so.","To maximize the number of surface shapes generated for when N residues are varied, all residues varied in a given round of variegation should be in the same interaction set because variation of several residues in one interaction set generates an exponential number of different shapes of the potential binding surface.","If cassette mutagenesis is to be used to introduce the variegated DNA into the ipbd gene, the protein residues to be varied are, preferably, close enough together in sequence that the variegated DNA (vgDNA) encoding all of them can be made in one piece. The present invention is not limited to a particular length of vgDNA that can be synthesized. With current technology, a stretch of 60 amino acids (180 DNA bases) can be spanned.","Further, when there is reason to mutate residues further than sixty residues apart, one can use other mutational means, such as single-stranded-oligonucleotide-directed mutagenesis (BOTS85) using two or more mutating primers.","Alternatively, to vary residues separated by more than sixty residues, two cassettes may be mutated as follows: 1) vg DNA having a low level of variegation (for example, 20 to 400 fold variegation) is introduced into one cassette in the OCV, 2) cells are transformed and cultured, 3) vg OCV DNA is obtained, 4) a second segment of vgDNA is inserted into a second cassette in the OCV, and 5) cells are transformed and cultured, GPs are harvested and subjected to selection-through-binding.","The composite level of variation preferably does not exceed the prevailing capabilities to a) produce very large numbers of independently transformed cells or b) detect small components in a highly varied population. The limits on the level of variegation are discussed later.","Data about the IPBD and the target that are useful in deciding which residues to vary in the variegation cycle include: 1) 3D structure, or at least a list of residues on the surface of the IPBD, 2) list of sequences homologous to IPBD, and 3) model of the target molecule or a stand-in for the target.","These data and an understanding of the behavior of different amino acids in proteins will be used to answer two questions:\n\n","Although an atomic model of the target material (obtained through X-ray crystallography, NMR, or other means) is preferred in such examination, it is not necessary. For example, if the target were a protein of unknown 3D structure, it would be sufficient to know the molecular weight of the protein and whether it were a soluble globular protein, a fibrous protein, or a membrane protein. Physical measurements, such as low-angle neutron diffraction, can determine the overall molecular shape, viz. the ratios of the principal moments of inertia. One can then choose a protein of known structure of the same class and similar size and shape to use as a molecular stand-in and yardstick. It is not essential to measure the moments of inertia of the target because, at low resolution, all proteins of a given size and class look much the same. The specific volumes are the same, all are more or less spherical and therefore all proteins of the same size and class have about the same radius of curvature. The radii of curvature of the two molecules determine how much of the two molecules can come into contact.","The most appropriate method of picking the residues of the protein chain at which the amino acids should be varied is by viewing, with interactive computer graphics, a model of the IPBD. A stick-figure representation of molecules is preferred. A suitable set of hardware is an Evans & Sutherland PS390 graphics terminal (Evans & Sutherland Corporation, Salt Lake City, Utah) and a MicroVAX II supermicro computer (Digital Equipment Corp., Maynard, Mass.). The computer should, preferably, have at least 150 megabytes of disk storage, so that the Brookhaven Protein Data Bank can be kept on line. A FORTRAN compiler, or some equally good higher-level language processor is preferred for program development. Suitable programs for viewing and manipulating protein models include: a) PS-FRODO, written by T. A. Jones (JONE85) and distributed by the Biochemistry Department of Rice University, Houston, Tex.; and b) PROTEUS, developed by Dayringer, Tramantano, and Fletterick (DAYR86). Important features of PS-FRODO and PROTEUS that are needed to view and manipulate protein models for the purposes of the present invention are the abilities to: 1) display molecular stick figures of proteins and other molecules, 2) zoom and clip images in real time, 3) prepare various abstract representations of the molecules, such as a line joining Cs and side group atoms, 4) compute and display solvent-accessible surfaces reasonably quickly, 5) point to and identify atoms, and 6) measure distance between atoms.","In addition, one could use theoretical calculations, such as dynamic simulations of proteins, to estimate whether a substitution at a particular residue of a particular amino-acid type might produce a protein of approximately the same 3D structure as the parent protein. Such calculations might also indicate whether a particular substitution will greatly affect the flexibility of the protein; calculations of this sort may be useful but are not required.","Residues whose mutagenesis is most likely to affect binding to a target molecule, without destabilizing the protein, are called the \u201cprincipal set\u201d. Using the knowledge of which residues are on the surface of the IPBD (as noted above), we pick residues that are close enough together on the surface of the IPBD to touch a molecule of the target simultaneously without having any IPBD main-chain atom come closer than van der Waals distance (viz. 4.0 to 5.0 \u212b) from any target atom. For the purposes of the present invention, a residue of the IPBD \u201ctouches\u201d the target if: a) a main-chain atom is within van der Waals distance, viz. 4.0 to 5.0 \u212b of any atom of the target molecule, or b) the C is within Dof any atom of the target molecule so that a side-group atom could make contact with that atom.","Because side groups differ in size (cf. Table 35), some judgment is required in picking D. In the preferred embodiment, we will use D=8.0 \u212b, but other values in the range 6.0 \u212b to 10.0 \u212b could be used. If IPBD has G at a residue, we construct a pseudo C with the correct bond distance and angles and judge the ability of the residue to touch the target from this pseudo C.","Alternatively, we choose a set of residues on the surface of the IPBD such that the curvature of the surface defined by the residues in the set is not so great that it would prevent contact between all residues in the set and a molecule of the target. This method is appropriate if the target is a macromolecule, such as a protein, because the PBDs derived from the IPBD will contact only a part of the macromolecular surface. The surfaces of macromolecules are irregular with varying curvatures. If we pick residues that define a surface that is not too convex, then there will be a region on a macromolecular target with a compatible curvature.","In addition to the geometrical criteria, we prefer that there be some indication that the underlying IPBD structure will tolerate substitutions at each residue in the principal set of residues. Indications could come from various sources, including: a) homologous sequences, b) static computer modeling, or c) dynamic computer simulations.","The residues in the principal set need not be contiguous in the protein sequence and usually are not. The exposed surfaces of the residues to be varied do not need to be connected. We desire only that the amino acids in the residues to be varied all be capable of touching a molecule of the target material simultaneously without having atoms overlap. If the target were, for example, horse heart myoglobin, and if the IPBD were BPTI, any set of residues in one interaction set of BPTI defined in Table 34 could be picked.","The secondary set comprises those residues not in the primary set that touch residues in the primary set. These residues might be excluded from the primary set because: a) the residue is internal, b) the residue is highly conserved, or c) the residue is on the surface, but the curvature of the IPBD surface prevents the residue from being in contact with the target at the same time as one or more residues in the primary set.","Internal residues are frequently conserved and the amino acid type can not be changed to a significantly different type without substantial risk that the protein structure will be disrupted. Nevertheless, some conservative changes of internal residues, such as I to L or F to Y, are tolerated. Such conservative changes subtly affect the placement and dynamics of adjacent protein residues and such \u201cfine tuning\u201d may be useful once an SBD is found.","Surface residues in the secondary set are most often located on the periphery of the principal set. Such peripheral residues can not make direct contact with the target simultaneously with all the other residues of the principal set. The charge on the amino acid in one of these residues could, however, have a strong effect on binding. Once an SBD is found, it is appropriate to vary the charge of some or all of these residues. For example, the variegated codon containing equimolar A and G at base 1, equimolar C and A at base 2, and A at base 3 yields amino acids T, A, K, and E with equal probability.","The assignment of residues to the primary and secondary sets may be based on: a) geometry of the IPBD and the geometrical relationship between the IPBD and the target (or a stand-in for the target) in a hypothetical complex, and b) sequences of proteins homologous to the IPBD. However, it should be noted that the distinction between the principal set and the secondary set is one more of convenience than of substance; we could just as easily have assigned each amino acid residue in the domain a preference score that weighed together the different considerations affecting whether they are suitable for variegation, and then ranked the residues in order, from most preferred to least.","For any given round of variegation, it may be necessary to limit the variegation to a subset of the residues in the primary and secondary sets, based on geometry and on the maximum allowed level of variegation that assures progressivity. The allowed level of variegation determines how many residues can be varied at once; geometry determines which ones.","The user may pick residues to vary in many ways. For example, pairs of residues are picked that are diametrically opposed across the face of the principal set. Two such pairs are used to delimit the surface, up\/down and right\/left. Alternatively, three residues that form an inscribed triangle, having as large an area as possible, on the surface are picked. One to three other residues are picked in a checkerboard fashion across the interaction surface. Choice of widely spaced residues to vary creates the possibility for high specificity because all the intervening residues must have acceptable complementarity before favorable interactions can occur at widely-separated residues.","The number of residues picked is coupled to the range through which each can be varied by the restrictions discussed below. In the first round, we do not assume any binding between IPBD and the target and so progressivity is not an issue. At the first round, the user may elect to produce a level of variegation such that each molecule of vgDNA is potentially different through, for example, unlimited variegation of 10 codons (20approx.=10). One run of the DNA synthesizer produces approximately 10molecules of length 100 nts. Inefficiencies in ligation and transformation will reduce the number of proteins actually tested to between 10and 5\u00b710. Multiple replications of the process with such very high levels of variegation will not yield repeatable results; the user decides whether this is important.","III.C. Determining the Substitution Set for Each Parental Residue","Having picked which residues to vary, we now decide the range of amino acids to allow at each variable residue. The total level of variegation is the product of the number of variants at each varied residue. Each varied residue can have a different scheme of variegation, producing 2 to 20 different possibilities. The set of amino acids which are potentially encoded by a given variegated codon are called its \u201csubstitution set\u201d.","The computer that controls a DNA synthesizer, such as the Milligen 7500, can be programmed to synthesize any base of an oligo-nt with any distribution of nts by taking some nt substrates (e.g. nt phosphoramidites) from each of two or more reservoirs. Alternatively, nt substrates can be mixed in any ratios and placed in one of the extra reservoir for so called \u201cdirty bottle\u201d synthesis. Each codon could be programmed differently. The \u201cmix\u201d of bases at each nucleotide position of the codon determines the relative frequency of occurrence of the different amino acids encoded by that codon.","Simply variegated codons are those in which those nucleotide positions which are degenerate are obtained from a mixture of two or more bases mixed in equimolar proportions. These mixtures are described in this specification by means of the standardized \u201cambiguous nucleotide\u201d code (Table 1 and 37 CFR \u00a71.822). In this code, for example, in the degenerate codon \u201cSNT\u201d, \u201cS\u201d denotes an equimolar mixture of bases G and C, \u201cN\u201d, an equimolar mixture of all four bases, and \u201cT\u201d, the single invariant base thymidine.","Complexly variegated codons are those in which at least one of the three positions is filled by a base from an other than equimolar mixture of two of more bases.","Either simply or complexly variegated codons may be used to achieve the desired substitution set.","If we have no information indicating that a particular amino acid or class of amino acid is appropriate, we strive to substitute all amino acids with equal probability because representation of one mini-protein above the detectable level is wasteful. Equal amounts of all four nts at each position in a codon (NNN) yields the amino acid distribution in which each amino acid is present in proportion to the number of codons that code for it. This distribution has the disadvantage of giving two basic residues for every acidic residue. In addition, six times as much R, S, and L as W or M occur. If five codons are synthesized with this distribution, each of the 243 sequences encoding some combination of L, R, and S are 7776-times more abundant than each of the 32 sequences encoding some combination of W and M. To have five Ws present at detectable levels, we must have each of the (L,R,S) sequences present in 7776-fold excess.","Preferably, we also consider the interactions between the sites of variegation and the surrounding DNA. If the method of mutagenesis to be used is replacement of a cassette, we consider whether the variegation will generate gratuitous restriction sites and whether they seriously interfere with the intended introduction of diversity. We reduce or eliminate gratuitous restriction sites by appropriate choice of variegation pattern and silent alteration of codons neighboring the sites of variegation.","It is generally accepted that the sequence of amino acids in a protein or polypeptide determine the three-dimensional structure of the molecule, including the possibility of no definite structure. Among polypeptides of definite length and sequence, some have a defined tertiary structure and most do not.","Particular amino acid residues can influence the tertiary structure of a defined polypeptide in several ways, including by:\n\n",{"@attributes":{"id":"p-0215","num":"0306"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"center"}}],"thead":{"row":[{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"Amino",{},{}]},{"entry":[{},"acid","distinction","ratio"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"MET","\u03b1\/turn","3.7"]},{"entry":[{},"PRO","turn\/\u03b1","3.7"]},{"entry":[{},"VAL","\u03b2\/turn","3.2"]},{"entry":[{},"GLY","turn\/\u03b1","2.9"]},{"entry":[{},"ILE","\u03b2\/turn","2.8"]},{"entry":[{},"PHE","\u03b2\/turn","2.3"]},{"entry":[{},"LEU","\u03b1\/turn","2.2"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}}},"br":{}},"PRO is generally taken to be a helix breaker. Nevertheless, proline often occurs at the beginning of helices or even in the middle of a helix, where it introduces a slight bend in the helix. Matthews and coworkers replaced a PRO that occurs near the middle of an \u03b1 helix in T4 lysozyme. To their surprise, the \u201cimproved\u201d protein is less stable than the wild-type. The rest of the structure had been adapted to fit the bent helix.","Lundeen (LUND86) has tabulated the frequencies of amino acids in helices, \u03b2 strands, turns, and coil in proteins of known 3D structure and has distinguished between CYSs having free thiol groups and half cystines. He reports that free CYS is found most often in helixes while half cystines are found more often in \u03b2 sheets. Half cystines are, however, regularly found in helices. Pease et al. (PEAS90) constructed a peptide having two cystines; one end of each is in a very stable \u03b1 helix. Apamin has a similar structure (WEMM83, PEAS88).","Flexibility:","GLY is the smallest amino acid, having two hydrogens attached to the C. Because GLY has no C, it confers the most flexibility on the main chain. Thus GLY occurs very frequently in reverse turns, particularly in conjunction with PRO, ASP, ASN, SER, and THR.","The amino acids ALA, SER, CYS, ASP, ASN, LEU, MET, PHE, TYR, TRP, ARG, HIS, GLU, GLN, and LYS have unbranched \u03b2 carbons. Of these, the side groups of SER, ASP, and ASN frequently make hydrogen bonds to the main chain and so can take on main-chain conformations that are energetically unfavorable for the others. VAL, ILE, and THR have branched \u03b2 carbons which makes the extended main-chain conformation more favorable. Thus VAL and ILE are most often seen in \u03b2 sheets. Because the side group of THR can easily form hydrogen bonds to the main chain, it has less tendency to exist in a \u03b2 sheet.","The main chain of proline is particularly constrained by the cyclic side group. The \u03c6 angle is always close to \u221260\u00b0. Most prolines are found near the surface of the protein.","Charge:","LYS and ARG carry a single positive charge at any pH below 10.4 or 12.0, respectively. Nevertheless, the methylene groups, four and three respectively, of these amino acids are capable of hydrophobic interactions. The guanidinium group of ARG is capable of donating five hydrogens simultaneously, while the amino group of LYS can donate only three. Furthermore, the geometries of these groups is quite different, so that these groups are often not interchangeable.","ASP and GLU carry a single negative charge at any pH above \u22484.5 and 4.6, respectively. Because ASP has but one methylene group, few hydrophobic interactions are possible. The geometry of ASP lends itself to forming hydrogen bonds to main-chain nitrogens which is consistent with ASP being found very often in reverse turns and at the beginning of helices. GLU is more often found in \u03b1 helices and particularly in the amino-terminal portion of these helices because the negative charge of the side group has a stabilizing interaction with the helix dipole (NICH88, SALI88).","HIS has an ionization pK in the physiological range, viz. 6.2. This pK can be altered by the proximity of charged groups or of hydrogen donators or acceptors. HIS is capable of forming bonds to metal ions such as zinc, copper, and iron.","Hydrogen Bonds:","Aside from the charged amino acids, SER, THR, ASN, GLN, TYR, and TRP can participate in hydrogen bonds.","Cross Links:","The most important form of cross link is the disulfide bond formed between two thiols, especially the thiols of CYS residues. In a suitably oxidizing environment, these bonds form spontaneously. These bonds can greatly stabilize a particular conformation of a protein or mini-protein. When a mixture of oxidized and reduced thiol reagents are present, exchange reactions take place that allow the most stable conformation to predominate. Concerning disul fides in proteins and peptides, see also KATZ90, MATS89, PERR84, PERR86, SAUE86, WELL86, JANA89, HORV89, KISH85, and SCHN86.","Other cross links that form without need of specific enzymes include:",{"@attributes":{"id":"p-0227","num":"0318"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"112pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1) (CYS):Fe","Rubredoxin (in CREI84, P.376)","(SEQ ID NO:122)",{}]},{"entry":{}},{"entry":["2) (CYS):Zn","Aspartate Transcarbamylase (in","(SEQ ID NO:122)"]},{"entry":[{},"CREI84, P.376) and Zn-fingers"]},{"entry":[{},"(HARD90)"]},{"entry":{}},{"entry":["3) (HIS)(MET) (CYS):Cu","Azurin (in CREI84, P.376) and Basic","(SEQ ID NO:123)"]},{"entry":[{},"\u201cBlue\u201d Cu Cucumber protein (GUSS88)"]},{"entry":{}},{"entry":["4) (HIS):Cu","CuZn superoxide dismutase","(SEQ ID NO:124)"]},{"entry":{}},{"entry":["5) (CYS):(FeS)","Ferredoxin (in CREI84, P.376)","(SEQ ID NO:122)"]},{"entry":{}},{"entry":["6) (CYS)(HIS):Zn","Zinc-fingers (GIBS88)","(SEQ ID NO:125)"]},{"entry":{}},{"entry":["7) (CYS)(HIS):Zn","Zinc-fingers (GAUS87, GIBS88)","(SEQ ID NO:126)"]}]}}}},"br":[{},{}],"sub":"2"},"The following simply variegated codons are useful because they encode a relatively balanced set of amino acids:\n\n","For the initial variegation, NNT is preferred, in most cases. However, when the codon is encoding an amino acid to be incorporated into an \u03b1 helix, NNG is preferred.","Below, we analyze several simple variegations as to the efficiency with which the libraries can be sampled.","Libraries of random hexapeptides encoded by (NNK)have been reported (SCOT90, CWIR90). Table 130 shows the expected behavior of such libraries. NNK produces single codons for PHE, TYR, CYS, TRP, HIS, GLN, ILE, MET, ASN, LYS, ASP, and GLU (\u03b1 set); two codons for each of VAL, ALA, PRO, THR, and GLY (\u03a6 set); and three codons for each of LEU, ARG, and SER (\u03a9 set). We have separated the 64,000,000 possible sequences into 28 classes, shown in Table 130A, based on the number of amino acids from each of these sets. The largest class is \u03a6\u03a9\u03b1\u03b1\u03b1\u03b1 with \u224814.6% of the possible sequences. Aside from any selection, all the sequences in one class have the same probability of being produced. Table 130B shows the probability that a given DNA sequence taken from the (NNK)library will encode a hexapeptide belonging to one of the defined classes; note that only \u22486.3% of DNA sequences belong to the \u03a6\u03a9\u03b1\u03b1\u03b1\u03b1 class.","Table 130C shows the expected numbers of sequences in each class for libraries containing various numbers of independent transformants (viz. 10, 3\u00b710, 10, 3\u00b710, 10, 3\u00b710, 10, and 3\u00b710). At 10independent transformants (ITs), we expect to see 56% of the \u03a9\u03a9\u03a9\u03a9\u03a9\u03a9 class, but only 0.1% of the \u03b1\u03b1\u03b1\u03b1\u03b1\u03b1 class. The vast majority of sequences seen come from classes for which less than 10% of the class is sampled. Suppose a peptide from, for example, class \u03a6\u03a6\u03a9\u03a9\u03b1\u03b1 is isolated by fractionating the library for binding to a target. Consider how much we know about peptides that are related to the isolated sequence. Because only 4% of the \u03a6\u03a6\u03a9\u03a9\u03b1\u03b1 class was sampled, we can not conclude that the amino acids from the \u03a9 set are in fact the best from the \u03a9 set. We might have LEU at position 2, but ARG or SER could be better. Even if we isolate a peptide of the \u03a9\u03a9\u03a9\u03a9\u03a9\u03a9 class, there is a noticeable chance that better members of the class were not present in the library.","With a library of 10ITs, we see that several classes have been completely sampled, but that the \u03b1\u03b1\u03b1\u03b1\u03b1\u03b1 class is only 1.1% sampled. At 7.6\u00b710ITs, we expect display of 50% of all amino-acid sequences, but the classes containing three or more amino acids of the a set are still poorly sampled. To achieve complete sampling of the (NNK)library requires about 3\u00b710ITs, 10-fold larger than the largest (NNK)library so far reported.","Table 131 shows expectations for a library encoded by (NNT)(NNG). The expectations of abundance are independent of the order of the codons or of interspersed unvaried codons. This library encodes 0.133 times as many amino-acid sequences, but there are only 0.0165 times as many DNA sequences. Thus 5.0\u00b710ITs (i.e. 60-fold fewer than required for (NNK)) gives almost complete sampling of the library. The results would be slightly better for (NNT)and slightly, but not much, worse for (NNG). The controlling factor is the ratio of DNA sequences to amino-acid sequences.","Table 132 shows the ratio of #DNA sequences\/#AA sequences for codons NNK, NNT, and NNG. For NNK and NNG, we have assumed that the PBD is displayed as part of an essential gene, such as gene III in Ff phage, as is indicated by the phrase \u201cassuming stops vanish\u201d. It is not in any way required that such an essential gene be used. If a non-essential gene is used, the analysis would be slightly different; sampling of NNK and NNG would be slightly less efficient. Note that (NNT)gives 3.6-fold more amino-acid sequences than (NNK)but requires 1.7-fold fewer DNA sequences. Note also that (NNT )gives twice as many amino-acid sequences as (NNK), but 3.3-fold fewer DNA sequences.","Thus, while it is possible to use a simple mixture (NNS, NNK or NNN) to obtain at a particular position all twenty amino acids, these simple mixtures lead to a highly biased set of encoded amino acids. This problem can be overcome by use of complexly variegated codons.","Complexly Variegated Codons","Let Abun(x) be the abundance of DNA sequences coding for amino acid x, defined by the distribution of nts at each base of the codon. For any distribution, there will be a most-favored amino acid (mfaa) with abundance Abun(mfaa) and a least-favored amino acid (lfaa) with abundance Abun(lfaa). We seek the nt distribution that allows all twenty amino acids and that yields the largest ratio Abun(lfaa)\/Abun(mfaa) subject, if desirable to further constraints.","We first will present the mixture calculated to be optimal when the nt distribution is subject to two constraints: equal abundances of acidic and basic amino acids and the least possible number of stop codons. Thus only nt distributions that yield Abun(E)+Abun(D)=Abun(R)+Abun(K) are considered, and the function maximized is:\n\n{(1(stop))(()\/())}.\n\nWe have simplified the search for an optimal nt distribution by limiting the third base to T or G (C or G is equivalent). All amino acids are possible and the number of accessible stop codons is reduced because TGA and TAA codons are eliminated. The amino acids F, Y, C, H, N, I, and D require T at the third base while W, M, Q, K, and E require G. Thus we use an equimolar mixture of T and G at the third base. However, it should be noted that the present invention embraces use of complexly variegated codons in which the third base is not limited to T or G (or to C or G).\n","A computer program, written as part of the present invention and named \u201cFind Optimum vgCodon\u201d (See Table 9), varies the composition at bases 1 and 2, in steps of 0.05, and reports the composition that gives the largest value of the quantity {(Abun(lfaa)\/Abun(mfaa)(1\u2212Abun(stop)))}. A vg codon is symbolically defined by the nucleotide distribution at each base:",{"@attributes":{"id":"p-0240","num":"0338"},"tables":{"@attributes":{"id":"TABLE-US-00006","num":"00006"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"center"}}],"thead":{"row":[{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]},{"entry":[{},"T","C","A","G"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"base #1 =","t1","c1","a1","g1"]},{"entry":[{},"base #2 =","t2","c2","a2","g2"]},{"entry":[{},"base #3 =","t3","c3","a3","g3"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"5","align":"center","rowsep":"1"}}]},{"entry":[{},"t1 + c1 + a1 + g1 = 1.0"]},{"entry":[{},"t2 + c2 + a2 + g2 = 1.0"]},{"entry":[{},"t3 = g3 = 0.5,"]},{"entry":[{},"c3 = a3 = 0."]}]}}]}},"br":[{},{},{},{},{},{},{},{},{},{},{}],"in-line-formulae":[{},{},{},{},{},{},{},{},{},{},{},{},{},{}],"i":["Abun","E","Abun","D","Abun","K","Abun","R","Abun","E","Abun","D","g","*a","Abun","K","Abun","R","a","*a","+c","*g","+a","*g","g","*a","=a","*a","+c","*g","+a","*g","g","g","*a","a","*a","c","*a","t","\u2212a","\u2212c","\u2212g","t","\u2212a","\u2212c","\u2212g"]},"Note that this chemistry encodes all twenty amino acids, with acidic and basic amino acids being equiprobable, and the most favored amino acid (serine) is encoded only 2.454 times as often as the least favored amino acid (tryptophan). The \u201cgfk\u201d vg codon improves sampling most for peptides containing several of the amino acids [F,Y,C,W,H,Q,I,M,N,K,D,E] for which NNK or NNS provide only one codon. Its sampling advantages are most pronounced when the library is relatively small.","A modification of \u201cFind Optimum vgCodon\u201d varies the composition at bases 1 and 2, in steps of 0.01, and reports the composition that gives the largest value of the quantity {(Abun(lfaa)\/Abun(mfaa))} without any restraint on the relative abundance of any amino acids. The results of this optimization is shown in Table 10B. The changes are small, indicating that insisting on equality of acids and bases and minimizing stop codons costs us little. Also note that, without restraining the optimization, the prevalence of acidic and basic amino acids comes out fairly close. On the other hand, relaxing the restriction leaves a distribution in which the least favored amino acid is only .412 times as prevalent as SER.","The advantages of an NNT codon are discussed elsewhere in the present application. Unoptimized NNT provides 15 amino acids encoded by only 16 DNA sequences. It is possible to improve on NNT as follows. First note that the SER codons occur in the T and A rows of the genetic-code table and in the C and G columns.\n\n[\n\nIf we reduce the prevalence of SER by reducing T, C, A, and Grelative to other bases, then we will also reduce the prevalence of PHE, TYR, CYS, PRO, THR, ALA, ARG, GLY, ILE, and ASN. The prevalence of LEU, HIS, VAL, and ASP will rise. If we assume that T, C, A, and Gare all lowered to the same extent and that C, G, T, and Aare increased by the same amount, we can compute a shift that makes the prevalence of SER equal the prevalences of LEU, HIS, VAL, and ASP. The decreases in each of PHE, TYR, CYS, PRO, THR, ALA, ARG, GLY, ILE, and ASN are not equal; CYS and THR are reduced more than the others.\n","Let the distribution be",{"@attributes":{"id":"p-0245","num":"0343"},"tables":{"@attributes":{"id":"TABLE-US-00007","num":"00007"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"center"}}],"thead":{"row":[{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]},{"entry":[{},"T","C","A","G"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"base #1 =",".25 \u2212 q",".25 + q",".25 \u2212 q",".25 + q"]},{"entry":[{},"base #2 =",".25 + q",".25 \u2212 q",".25 + q",".25 \u2212 q"]},{"entry":[{},"base #3 =",".1.00","0.0","0.0","0.0"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"5","align":"center","rowsep":"1"}}]}]}}]}},"br":[{},{},{},{},{}],"in-line-formulae":[{},{},{},{},{},{},{},{}],"i":["q","q","q","q","q","q","q","q","q","q+","q"],"sup":["2","2 ","2"]},"This distribution (shown in Table 10C) gives five amino acids (SER, LEU, HIS, VAL, ASP) in very nearly equal amounts. A further eight amino acids (PHE, TYR, ILE, ASN, PRO, ALA, ARG, GLY) are present at 78% the abundance of SER. THR and CYS remain at half the abundance of SER. When variegating DNA for disulfide-bonded mini-proteins, it is often desirable to reduce the prevalence of CYS. This distribution allows 13 amino acids to be seen at high level and gives no stops; the optimized fxS distribution allows only 11 amino acids at high prevalence.","The NNG codon can also be optimized. Table 10D shows an approximately optimized NNG codon. When equimolar T,C,A,G are used in NNG, one obtains double doses of LEU and ARG. To improve the distribution, we increase Gby 4 \u03b4, decrease Tand Aby \u03b4 each and Cby 2 \u03b4. We adopt this pattern because Caffects both LEU and ARG while Tand Aeach affect either LEU or ARG, but not both. Similarly, we decrease Tand Gby \u03c4 while we increase Cand Aby \u03c4. We adjusted \u03b4 and \u03c4 until [ALA]\u2248[ARG]. There are, under this variegation, four equally most favored amino acids: LEU, ARG, ALA, and GLU. Note that there is one acidic and one basic amino acid in this set. There are two equally least favored amino acids: TRP and MET. The ratio of lfaa\/mfaa is 0.5258. If this codon is repeated six times, peptides composed entirely of TRP and MET are 2% as common as peptides composed entirely of the most favored amino acids. We refer to this as \u201cthe prevalence of (TRP\/MET)in optimized NNGvgDNA\u201d.","When synthesizing vgDNA by the \u201cdirty bottle\u201d method, it is sometimes desirable to use only a limited number of mixes. One very useful mixture is called the \u201coptimized NNS mixture\u201d in which we average the first two positions of the fxS mixture: T=0.24, C=0.17, A=0.33, G=0.26, the second position is identical to the first, C=G=0.5. This distribution provides the amino acids ARG, SER, LEU, GLY, VAL, THR, ASN, and LYS at greater than 5% plus ALA, ASP, GLU, ILE, MET, and TYR at greater than 4%.","An additional complexly variegated codon is of interest. This codon is identical to the optimized NNT codon at the first two positions and has T:G::90:10 at the third position. This codon provides thirteen amino acids (ALA, ILE, ARG, SER, ASP, LEU, VAL, PHE, ASN, GLY, PRO, TYR, and HIS) at more than 5.5%. THR at 4.3% and CYS at 3.9% are more common than the LFAAs of NNK (3.125%). The remaining five amino acids are present at less than 1%. This codon has the feature that all amino acids are present; sequences having more than two of the low-abundance amino acids are rare. When we isolate an SBD using this codon, we can be reasonably sure that the first 13 amino acids were tested at each position. A similar codon, based on optimized NNG, could be used.","Table 10E shows some properties of an unoptimized NNS (or NNK) codon. Note that there are three equally most-favored amino acids: ARG, LEU, and SER. There are also twelve equally least favored amino acids: PHE, ILE, MET, TYR, HIS, GLN, ASN, LYS, ASP, GLU, CYS, and TRP. Five amino acids (PRO, THR, ALA, VAL, GLY) fall in between. Note that a six-fold repetition of NNS gives sequences composed of the amino acids [PHE, ILE, MET, TYR, HIS, GLN, ASN, LYS, ASP, GLU, CYS, and TRP] at only \u22480.1% of the sequences composed of [ARG, LEU, and SER]. Not only is this \u224820-fold lower than the prevalence of (TRP\/MET)in optimized NNGvgDNA, but this low prevalence applies to twelve amino acids.","Diffuse Mutagenesis","Diffuse Mutagenesis can be applied to any part of the protein at any time, but is most appropriate when some binding to the target has been established. Diffuse Mutagenesis can be accomplished by spiking each of the pure nts activated for DNA synthesis (e.g. nt-phosphoramidites) with a small amount of one or more of the other activated nts.","Contrary to general practice, the present invention sets the level of spiking so that only a small percentage (1% to 0.00001%, for example) of the final product will contain the initial DNA sequence. This will insure that many single, double, triple, and higher mutations occur, but that recovery of the basic sequence will be a possible outcome. Let Nbe the number of bases to be varied, and let Q be the fraction of all sequences that should have the parental sequence, then M, the fraction of the mixture that is the majority component, is\n\nexp{ log()\/}=1010.\n\nIf, for example, thirty base pairs on the DNA chain were to be varied and 1% of the product is to have the parental sequence, then each mixed nt substrate should contain 86% of the parental nt and 14% of other nts. Table 8 shows the fraction (fn) of DNA molecules having n non-parental bases when 30 bases are synthesized with reagents that contain fraction M of the majority component. When M=0.63096, f24 and higher are less than 10. The entry \u201cmost\u201d in Table 8 is the number of changes that has the highest probability. Note that substantial probability for multiple substitutions only occurs if the fraction of parental sequence (f0) is allowed to drop to around 10. The Nbase pairs of the DNA chain that are synthesized with mixed reagents need not be contiguous. They are picked so that between N\/3 and Ncodons are affected to various degrees. The residues picked for mutation are picked with reference to the 3D structure of the IPBD, if known. For example, one might pick all or most of the residues in the principal and secondary set. We may impose restrictions on the extent of variation at each of these residues based on homologous sequences or other data. The mixture of non-parental nts need not be random, rather mixtures can be biased to give particular amino acid types specific probabilities of appearance at each codon. For example, one residue may contain a hydrophobic amino acid in all known homologous sequences; in such a case, the first and third base of that codon would be varied, but the second would be set to T. Other examples of how this might be done are given in the horse heart myoglobin example. This diffuse structure-directed mutagenesis will reveal the subtle changes possible in protein backbone associated with conservative interior changes, such as V to I, as well as some not so subtle changes that require concomitant changes at two or more residues of the protein.\n\nIII.D. Special Considerations Relating to Variegation of Mini-Proteins with Essential Cysteines\n","Several of the preferred simple or complex variegated codons encode a set of amino acids which includes cysteine. This means that some of the encoded binding domains will feature one or more cysteines in addition to the invariant disulfide-bonded cysteines. For example, at each NNT-encoded position, there is a one in sixteen chance of obtaining cysteine. If six codons are so varied, the fraction of domains containing additional cysteines is 0.33. Odd numbers of cysteines can lead to complications, see Perry and Wetzel (PERR 84, PERR86). On the other hand, many disulfide-containing proteins contain cysteines that do not form disulfides, e.g. trypsin. The possibility of unpaired cysteines can be dealt with in several ways:","First, the variegated phage population can be passed over an immobilized reagent that strongly binds free thiols, such as SulfoLink (catalogue number 44895 H from Pierce Chemical Company, Rockford, Ill., 61105). Another product from Pierce is TNB-Thiol Agarose (Catalogue Code 20409 H). BioRad sells Affi-Gel 401 (catalogue 153-4599) for this purpose.","Second, one can use a variegation that excludes cysteines, such as:","NHT that gives [F,S,Y,L,P,H,I,T,N,V,A,D],","VNS that gives [L,P,H,Q,R,I,M,T,N,K,S,V,A,E,D,G],","NNG that gives [L,S,W,P,Q,R,M,T,K,R,V,A,E,G,stop],","SNT that gives [L,P,H,R,V,A,D,G],","RNG that gives [M,T,K,R,V,A,E,G],","RMG that gives [T,K,A,E],","VNT that gives [L,P,H,R,I,T,N,S,V,A,D,G], or","RRS that gives [N,S,K,R,D,E,G].","However, each of these schemes has one or more of the disadvantages, relative to NNT: a) fewer amino acids are allowed, b) amino acids are not evenly provided, c) acidic and basic amino acids are not equally likely), or d) stop codons occur. Nonetheless, NNG, NHT, and VNT are almost as useful as NNT. NNG encodes 13 different amino acids and one stop signal. Only two amino acids appear twice in the 16-fold mix.","Thirdly, one can enrich the population for binding to the preselected target, and evaluate selected sequences post hoc for extra cysteines. Those that contain more cysteines than the cysteines provided for conformational constraint may be perfectly usable. It is possible that a disulfide linkage other than the designed one will occur. This does not mean that the binding domain defined by the isolated DNA sequence is in any way unsuitable. The suitability of the isolated domains is best determined by chemical and biochemical evaluation of chemically synthesized peptides.","Lastly, one can block free thiols with reagents, such as Ellman's reagent, iodoacetate, or methyl iodide, that specifically bind free thiols and that do not react with disulfides, and then leave the modified phage in the population. It is to be understood that the blocking agent may alter the binding properties of the mini-protein; thus, one might use a variety of blocking reagent in expectation that different binding domains will be found. The variegated population of thiol-blocked genetic packages are fractionated for binding. If the DNA sequence of the isolated binding mini-protein contains an odd number of cysteines, then synthetic means are used to prepare mini-proteins having each possible linkage and in which the odd thiol is appropriately blocked. Nishiuchi (NISH82, NISH86, and works cited therein) disclose methods of synthesizing peptides that contain a plurality of cysteines so that each thiol is protected with a different type of blocking group. These groups can be selectively removed so that the disulfide pairing can be controlled. We envision using such a scheme with the alteration that one thiol either remains blocked, or is unblocked and then reblocked with a different reagent.","III.E. Planning the Second and Later Rounds of Variegation","The method of the present invention allows efficient accumulation of information concerning the amino-acid sequence of a binding domain having high affinity for a predetermined target. Although one may obtain a highly useful binding domain from a single round of variegation and affinity enrichment, we expect that multiple rounds will be needed to achieve the highest possible affinity and specificity.","If the first round of variegation results in some binding to the target, but the affinity for the target is still too low, further improvement may be achieved by variegation of the SBDs. Preferably, the process is progressive, i.e. each variegation cycle produces a better starting point for the next variegation cycle than the previous cycle produced. Setting the level of variegation such that the ppbd and many sequences related to the ppbd sequence are present in detectable amounts ensures that the process is progressive. If the level of variegation is so high that the ppbd sequence is present at such low levels that there is an appreciable chance that no transformant will display the PPBD, then the best SBD of the next round could be worse than the PPBD. At excessively high level of variegation, each round of mutagenesis is independent of previous rounds and there is no assurance of progressivity. This approach can lead to valuable binding proteins, but repetition of experiments with this level of variegation will not yield progressive results. Excessive variation is not preferred.","Progressivity is not an all-or-nothing property. So long as most of the information obtained from previous variegation cycles is retained and many different surfaces that are related to the PPBD surface are produced, the process is progressive. If the level of variegation is so high that the ppbd gene may not be detected, the assurance of progressivity diminishes. If the probability of recovering PPBD is negligible, then the probability of progressive behavior is also negligible.","A level of variegation that allows recovery of the PPBD has two properties:\n\n","An opposing force in our design considerations is that PBDs are useful in the population only up to the amount that can be detected; any excess above the detectable amount is wasted. Thus we produce as many surfaces related to PPBD as possible within the constraint that the PPBD be detectable.","If the level of variegation in the previous variegation cycle was correctly chosen, then the amino acids selected to be in the residues just varied are the ones best determined. The environment of other residues has changed, so that it is appropriate to vary them again. Because there are often more residues in the principal and secondary sets than can be varied simultaneously, we start by picking residues that either have never been varied (highest priority) or that have not been varied for one or more cycles. If we find that varying all the residues except those varied in the previous cycle does not allow a high enough level of diversity, then residues varied in the previous cycle might be varied again. For example, if M(the number of independent transformants that can be produced from Yof DNA) and C(the sensitivity of the affinity separation) were such that seven residues could be varied, and if the principal and secondary sets contained 13 residues, we would always vary seven residues, even though that implies varying some residue twice in a row. In such cases, we would pick the residues just varied that contain the amino acids of highest abundance in the variegated codons used.","It is the accumulation of information that allows the process to select those protein sequences that produce binding between the SBD and the target. Some interfaces between proteins and other molecules involve twenty or more residues. Complete variation of twenty residues would generate 10different proteins. By dividing the residues that lie close together in space into overlapping groups of five to seven residues, we can vary a large surface but never need to test more than 10to 10candidates at once, a savings of 10to 10fold. The power of selection with accumulation of information is well illustrated in Chapter 3 of DAWK86.","Use of NNT or NNG variegated codons leads to very efficient sampling of variegated libraries because the ratio of (different amino-acid sequences)\/(different DNA sequences) is much closer to unity than it is for NNK or even the optimized vg codon (fxS). Nevertheless, a few amino acids are omitted in each case. Both NNT and NNG allow members of all important classes of amino acids: hydrophobic, hydrophilic, acidic, basic, neutral hydrophilic, small, and large. After selecting a binding domain, a subsequent variegation and selection may be desirable to achieve a higher affinity or specificity. During this second variegation, amino acid possibilities overlooked by the preceding variegation may be investigated.","In the first round, we assume that the parental protein has no known affinity for the target material. For example, consider the parental mini-protein, similar to that discussed in Example 11, having the structure X\u2014C\u2014X\u2014X\u2014X\u2014X\u2014C\u2014X(SEQ ID NO:7) in which Cand Cform a disulfide bond. Introduction of extra cysteines may cause alternative structures to form which might be disadvantageous. Accidental cysteines at positions 4 or 5 are thought to be potentially more troublesome than at the other positions. We adopt the pattern of variegation: X:NNT, X:NNT, X:NNG, X:NNG, X:NNT, and X:NNT, so that cysteine can not occur at positions 4 and 5 (Table 131 shows the number of different amino acids expected in libraries prepared with DNA variegated in this way and comprising different numbers of independent transformants.)","In the second round of variegation, a preferred strategy is to vary each position through a new set of residues which includes the amino acid(s) which were found at that position in the successful binding domains, and which include as many as possible of the residues which were excluded in the first round of variegation.","A few examples may be helpful. Suppose we obtained PRO using NNT. This amino acid is available with either NNT or NNG. We can be reasonably sure that PRO is the best amino acid from the set [PRO, LEU, VAL, THR, ALA, ARG, GLY, PHE, TYR, CYS, HIS, ILE, ASN, ASP, SER]. Thus we need to try a set that includes [PRO, TRP, GLN, MET, LYS, GLU]. The set allowed by NNG is the preferred set.","What if we obtained HIS instead? Histidine is aromatic and fairly hydrophobic and can form hydrogen bonds to and from the imidazole ring. Tryptophan is hydrophobic and aromatic and can donate a hydrogen to a suitable acceptor and was excluded by the NNT codon. Methionine was also excluded and is hydrophobic. Thus, one preferred course is to use the variegated codon HDS that allows [HIS, GLN, ASN, LYS, TYR, CYS, TRP, ARG, SER, GLY, <stop>].","GLN can be encoded by the NNG codon. If GLN is selected, at the next round we might use the vg codon VAS that encodes three of the seven excluded possibilities, viz. HIS, ASN, and ASP. The codon VAS encodes 6 amino acid sequences in six DNA sequences. This leaves PHE, CYS, TYR, and ILE untested, but these are all very hydrophobic. Switching to NNT would be undesirable because that would exclude GLN. One could use NAS that includes TYR and <stop>. Suppose the successful amino acid encoded by an NNG codon was ARG. Here we switch to NNT because this allows ARG plus all the excluded possibilities.","THR is another possibility with the NNT codon. If THR is selected, we switch to NNG because that includes the previously excluded possibilities and includes THR. Suppose the successful amino acid encoded by the NNT codon was ASP. We use RRS at the next variegation because this includes both acidic amino acids plus LYS and ARG. One could also use VRS to allow GLN.","Thus, later rounds of variegation test both amino acid positions not previously mutated, and amino acid substitutions at a previously mutated position which were not within the previous substitution set.","If the first round of variegation is entirely unsuccessful, a different pattern of variegation should be used. For example, if more than one interaction set can be defined within a domain, the residues varied in the next round of variegation should be from a different set than that probed in the initial variegation. If repeated failures are encountered, one may switch to a different IPBD.","IV.A. General Requirements for Genetic Packages","It is emphasized that the GP on which selection-through-binding will be practiced must be capable, after the selection, either of growth in some suitable environment or of in vitro amplification and recovery of the encapsulated genetic message. During at least part of the growth, the increase in number is preferably approximately exponential with respect to time. The component of a population that exhibits the desired binding properties may be quite small, for example, one in 10or less. Once this component of the population is separated from the non-binding components, it must be possible to amplify it. Culturing viable cells is the most powerful amplification of genetic material known and is preferred. Genetic messages can also be amplified in vitro, e.g. by PCR, but this is not the most preferred method.","Preferred GPs are vegetative bacterial cells, bacterial spores and bacterial DNA viruses. Eukaryotic cells could be used as genetic packages but have longer dividing times and more stringent nutritional requirements than do bacteria and it is much more difficult to produce a large number of independent transformants. They are also more fragile than bacterial cells and therefore more difficult to chromatograph without damage. Eukaryotic viruses could be used instead of bacteriophage but must be propagated in eukaryotic cells and therefore suffer from some of the amplification problems mentioned above.","Nonetheless, a strain of any living cell or virus is potentially useful if the strain can be: 1) genetically altered with reasonable facility to encode a potential binding domain, 2) maintained and amplified in culture, 3) manipulated to display the potential binding protein domain where it can interact with the target material during affinity separation, and 4) affinity separated while retaining the genetic information encoding the displayed binding domain in recoverable form. Preferably, the GP remains viable after affinity separation.","When the genetic package is a bacterial cell, or a phage which is assembled periplasmically, the display means has two components. The first component is a secretion signal which directs the initial expression product to the inner membrane of the cell (a host cell when the package is a phage). This secretion signal is cleaved off by a signal peptidase to yield a processed, mature, potential binding protein. The second component is an outer surface transport signal which directs the package to assemble the processed protein into its outer surface. Preferably, this outer surface transport signal is derived from a surface protein native to the genetic package.","For example, in a preferred embodiment, the hybrid gene comprises a DNA encoding a potential binding domain operably linked to a signal sequence (e.g., the signal sequences of the bacterial phoA or bla genes or the signal sequence of M13 phage geneIII) and to DNA encoding a coat protein (e.g., the M13 gene III or gene VIII proteins) of a filamentous phage (e.g., M13). The expression product is transported to the inner membrane (lipid bilayer) of the host cell, whereupon the signal peptide is cleaved off to leave a processed hybrid protein. The C-terminus of the coat protein-like component of this hybrid protein is trapped in the lipid bilayer, so that the hybrid protein does not escape into the periplasmic space. (This is typical of the wild-type coat protein.) As the single-stranded DNA of the nascent phage particle passes into the periplasmic space, it collects both wild-type coat protein and the hybrid protein from the lipid bilayer. The hybrid protein is thus packaged into the surface sheath of the filamentous phage, leaving the potential binding domain exposed on its outer surface. (Thus, the filamentous phage, not the host bacterial cell, is the \u201creplicable genetic package\u201d in this embodiment.)","If a secretion signal is necessary for the display of the potential binding domain, in an especially preferred embodiment the bacterial cell in which the hybrid gene is expressed is of a \u201csecretion-permissive\u201d strain.","When the genetic package is a bacterial spore, or a phage whose coat is assembled intracellularly, a secretion signal directing the expression product to the inner membrane of the host bacterial cell is unnecessary. In these cases, the display means is merely the outer surface transport signal, typically a derivative of a spore or phage coat protein.","There are several methods of arranging that the ipbd gene is expressed in such a manner that the IPBD is displayed on the outer surface of the GP. If one or more fusions of fragments of x genes to fragments of a natural osp gene are known to cause X protein domains to appear on the GP surface, then we pick the DNA sequence in which an ipbd gene fragment replaces the x gene fragment in one of the successful osp-x fusions as a preferred gene to be tested for the display-of-IPBD phenotype. (The gene may be constructed in any manner.) If no fusion data are available, then we fuse an ipbd fragment to various fragments, such as fragments that end at known or predicted domain boundaries, of the osp gene and obtain GPs that display the osp-ipbd fusion on the GP outer surface by screening or selection for the display-of-IPBD phenotype. The OSP may be modified so as to increase the flexibility and\/or length of the linkage between the OSP and the IPBD and thereby reduce interference between the two.","The fusion of ipbd and osp fragments may also include fragments of random or pseudorandom DNA to produce a population, members of which may display IPBD on the GP surface. The members displaying IPBD are isolated by screening or selection for the display-of-binding phenotype.","The replicable genetic entity (phage or plasmid) that carries the osp-pbd genes (derived from the osp-ipbd gene) through the selection-through-binding process, is referred to hereinafter as the operative cloning vector (OCV). When the OCV is a phage, it may also serve as the genetic package. The choice of a GP is dependent in part on the availability of a suitable OCV and suitable OSP.","Preferably, the GP is readily stored, for example, by freezing. If the GP is a cell, it should have a short doubling time, such as 20-40 minutes. If the GP is a virus, it should be prolific, e.g., a burst size of at least 100\/infected cell. GPs which are finicky or expensive to culture are disfavored. The GP should be easy to harvest, preferably by centrifugation. The GP is preferably stable for a temperature range of \u221270 to 42\u00b0 C. (stable at 4\u00b0 C. for several days or weeks); resistant to shear forces found in HPLC; insensitive to UV; tolerant of desiccation; and resistant to a pH of 2.0 to 10.0, surface active agents such as SDS or Triton, chaotropes such as 4M urea or 2M guanidinium HCl, common ions such as K, Na, and SO, common organic solvents such as ether and acetone, and degradative enzymes. Finally, there must be a suitable OCV.","Although knowledge of specific OSPs may not be required for vegetative bacterial cells and endospores, the user of the present invention, preferably, will know: Is the sequence of any osp known? (preferably yes, at least one required for phage). How does the OSP arrive at the surface of GP? (knowledge of route necessary, different routes have different uses, no route preferred per se). Is the OSP post-translationally processed? (no processing most preferred, predictable processing preferred over unpredictable processing). What rules are known governing this processing, if there is any processing? (no processing most preferred, predictable processing acceptable). What function does the OSP serve in the outer surface? (preferably not essential). Is the 3D structure of an OSP known? (highly preferred). Are fusions between fragments of osp and a fragment of x known? Does expression of these fusions lead to X appearing on the surface of the GP? (fusion data is as preferred as knowledge of a 3D structure). Is a \u201c2D\u201d structure of an OSP available? (in this context, a \u201c2D\u201d structure indicates which residues are exposed on the cell surface) (2D structure less preferred than 3D structure). Where are the domain boundaries in the OSP? (not as preferred as a 2D structure, but acceptable). Could IPBD go through the same process as OSP and fold correctly? (IPBD might need prosthetic groups) (preferably IPBD will fold after same process). Is the sequence of an osp promoter known? (preferably yes). Is osp gene controlled by regulatable promoter available? (preferably yes). What activates this promoter? (preferably a diffusible chemical, such as IPTG). How many different OSPs do we know? (the more the better). How many copies of each OSP are present on each package? (more is better).","The user will want knowledge of the physical attributes of the GP: How large is the GP? (knowledge useful in deciding how to isolate GPs) (preferably easy to separate from soluble proteins such as IgGs). What is the charge on the GP? (neutral preferred). What is the sedimentation rate of the GP? (knowledge preferred, no particular value preferred).","The preferred GP, OCV and OSP are those for which the fewest serious obstacles can be seen, rather than the one that scores highest on any one criterion.","Viruses are preferred over bacterial cells and spores (cp. LUIT85 and references cited therein). The virus is preferably a DNA virus with a genome size of 2 kb to 10 kb base pairs, such as (but not limited to) the filamentous (Ff) phage M13, fd, and f1 (inter alia see RASC86, BOEK80, BOEK82, DAYL88, GRAY81b, KUHN88, LOPE85, WEBS85, MARV75, MARV80, MOSE82, CRIS84, SMIT88a, SMIT88b); the IncN specific phage Ike and If1 (NAKA81, PEET85, PEET87, THOM83, THOM88a); IncP-specific phage Pf1 (THOM83, THOM88a) and Pf3 (LUIT83, LUIT85, LUTI87, THOM88a); and the phage Xf (THOM83, THOM88a). Filamentous phage are especially preferred.","Preferred OSPs for several GPs are given in Table 2. References to osp-ipbd fusions in this section should be taken to apply, to osp-pbd and osp-sbd fusions as well.","The species chosen as a GP should have a well-characterized genetic system and strains defective in genetic recombination should be available. The chosen strain may need to be manipulated to prevent changes of its physiological state that would alter the number or type of proteins or other molecules on the cell surface during the affinity separation procedure.","IV.B. Phages for Use as GPs","Unlike bacterial cells and spores, choice of a phage depends strongly on knowledge of the 3D structure of an OSP and how it interacts with other proteins in the capsid. This does not mean that we need atomic resolution of the OSP, but that we need to know which segments of the OSP interact to make the viral coat and which segments are not constrained by structural or functional roles. The size of the phage genome and the packaging mechanism are also important because the phage genome itself is the cloning vector. The osp-ipbd gene is inserted into the phage genome; therefore: 1) the genome of the phage must allow introduction of the osp-ipbd gene either by tolerating additional genetic material or by having replaceable genetic material; 2) the virion must be capable of packaging the genome after accepting the insertion or substitution of genetic material, and 3) the display of the OSP-IPBD protein on the phage surface must not disrupt virion structure sufficiently to interfere with phage propagation.","The morphogenetic pathway of the phage determines the environment in which the IPBD will have opportunity to fold. Periplasmically assembled phage are preferred when IPBDs contain essential disulfides, as such IPBDs may not fold within a cell (these proteins may fold after the phage is released from the cell). Intracellularly assembled phage are preferred when the IPBD needs large or insoluble prosthetic groups (such as FeSclusters), since the IPBD may not fold if secreted because the prosthetic group is lacking.","When variegation is introduced in Part II, multiple infections could generate hybrid GPs that carry the gene for one PBD but have at least some copies of a different PBD on their surfaces; it is preferable to minimize this possibility by infecting cells with phage under conditions resulting in a low multiple-of-infection (MOI).","Bacteriophages are excellent candidates for GPs because there is little or no enzymatic activity associated with intact mature phage, and because the genes are inactive outside a bacterial host, rendering the mature phage particles metabolically inert.","The filamentous phages (e.g., M13) are of particular interest.","For a given bacteriophage, the preferred OSP is usually one that is present on the phage surface in the largest number of copies, as this allows the greatest flexibility in varying the ratio of OSP-IPBD to wild type OSP and also gives the highest likelihood of obtaining satisfactory affinity separation. Moreover, a protein present in only one or a few copies usually performs an essential function in morphogenesis or infection; mutating such a protein by addition or insertion is likely to result in reduction in viability of the GP. Nevertheless, an OSP such as M13 gIII protein may be an excellent choice as OSP to cause display of the PBD.","It is preferred that the wild-type osp gene be preserved. The ipbd gene fragment may be inserted either into a second copy of the recipient osp gene or into a novel engineered osp gene. It is preferred that the osp-ipbd gene be placed under control of a regulated promoter. Our process forces the evolution of the PBDs derived from IPBD so that some of them develop a novel function, viz. binding to a chosen target. Placing the gene that is subject to evolution on a duplicate gene is an imitation of the widely-accepted scenario for the evolution of protein families. It is now generally accepted that gene duplication is the first step in the evolution of a protein family from an ancestral protein. By having two copies of a gene, the affected physiological process can tolerate mutations in one of the genes. This process is well understood and documented for the globin family (cf. DICK83, p65ff, and CREI84, p117-125).","The user must choose a site in the candidate OSP gene for inserting a ipbd gene fragment. The coats of most bacteriophage are highly ordered. Filamentous phage can be described by a helical lattice; isometric phage, by an icosahedral lattice. Each monomer of each major coat protein sits on a lattice point and makes defined interactions with each of its neighbors. Proteins that fit into the lattice by making some, but not all, of the normal lattice contacts are likely to destabilize the virion by: a) aborting formation of the virion, b) making the virion unstable, or c) leaving gaps in the virion so that the nucleic acid is not protected. Thus in bacteriophage, unlike the cases of bacteria and spores, it is important to retain in engineered OSP-IPBD fusion proteins those residues of the parental OSP that interact with other proteins in the virion. For M13 gVIII, we retain the entire mature protein, while for M13 gIII, it might suffice to retain the last 100 residues (or even fewer). Such a truncated gIII protein would be expressed in parallel with the complete gIII protein, as gIII protein is required for phage infectivity.","Il'ichev et al. (ILIC89) have reported viable phage having alterations in gene VIII. In one case, a point mutation changed one amino acid near the amino terminus of the mature gVIII protein from GLU to ASP. In the other case, five amino acids were inserted at the site of the first mutation. They suggested that similar constructions could be used for vaccines. They did not report on any binding properties of the modified phage, nor did they suggest mutagenizing the inserted material. Furthermore, they did not insert a binding domain, nor did they suggest inserting such a domain.","Further considerations on the design of the ipbd::osp gene is discussed in section IV.F.","Filamentous Phage:","Compared to other bacteriophage, filamentous phage in general are attractive and M13 in particular is especially attractive because: 1) the 3D structure of the virion is known; 2) the processing of the coat protein is well understood; 3) the genome is expandable; 4) the genome is small; 5) the sequence of the genome is known; 6) the virion is physically resistant to shear, heat, cold, urea, guanidinium Cl, low pH, and high salt; 7) the phage is a sequencing vector so that sequencing is especially easy; 8) antibiotic-resistance genes have been cloned into the genome with predictable results (HINE80); 9) It is easily cultured and stored (FRIT85), with no unusual or expensive media requirements for the infected cells, 10) it has a high burst size, each infected cell yielding 100 to 1000 M13 progeny after infection; and 11) it is easily harvested and concentrated (SALI64, FRIT85).","The filamentous phage include M13, f1, fd, If1, Ike, Xf, Pf1, and Pf3.","The entire life cycle of the filamentous phage M13, a common cloning and sequencing vector, is well understood. M13 and f1 are so closely related that we consider the properties of each relevant to both (RASC86); any differentiation is for historical accuracy. The genetic structure (the complete sequence (SCHA78), the identity and function of the ten genes, and the order of transcription and location of the promoters) of M13 is well known as is the physical structure of the virion (BANN81, BOEK80, CHAN79, ITOK79, KAPL78, KUHN85b, KUHN87, MAKO80, MARV78, MESS78, OHKA81, RASC86, RUSS81, SCHA78, SMIT85, WEBS78, and ZIMM82); see RASC86 for a recent review of the structure and function of the coat proteins. Because the genome is small (6423 bp), cassette mutagenesis is practical on RF M13 (AUSU87), as is single-stranded oligo-nt directed mutagenesis (FRIT85). M13 is a plasmid and transformation system in itself, and an ideal sequencing vector. M13 can be grown on Rec strains of The M13 genome is expandable (MESS78, FRIT85) and M13 does not lyse cells. Because the M13 genome is extruded through the membrane and coated by a large number of identical protein molecules, it can be used as a cloning vector (WATS87 p278, and MESS77). Thus we can insert extra genes into M13 and they will be carried along in a stable manner.","Marvin and collaborators (MARV78, MAKO80, BANN81) have determined an approximate 3D virion structure of f1 by a combination of genetics, biochemistry, and X-ray diffraction from fibers of the virus.  is drawn after the model of Banner et al. (BANN81) and shows only the Cs of the protein. The apparent holes in the cylindrical sheath are actually filled by protein side groups so that the DNA within is protected. The amino terminus of each protein monomer is to the outside of the cylinder, while the carboxy terminus is at smaller radius, near the DNA. Although other filamentous phages (e.g. Pf1 or Ike) have different helical symmetry, all have coats composed of many short \u03b1-helical monomers with the amino terminus of each monomer on the virion surface.","The major coat protein is encoded by gene VIII. The 50 amino acid mature gene VIII coat protein is synthesized as a 73 amino acid precoat (ITOK79). The first 23 amino acids constitute a typical signal-sequence which causes the nascent polypeptide to be inserted into the inner cell membrane. Whether the precoat inserts into the membrane by itself or through the action of host secretion components, such as SecA and SecY, remains controversial, but has no effect on the operation of the present invention.","An signal peptidase (SP-I) recognizes amino acids 18, 21, and 23, and, to a lesser extent, residue 22, and cuts between residues 23 and 24 of the precoat (KUHN85a, KUHN85b, OLIV87). After removal of the signal sequence, the amino terminus of the mature coat is located on the periplasmic side of the inner membrane; the carboxy terminus is on the cytoplasmic side. About 3000 copies of the mature 50 amino acid coat protein associate side-by-side in the inner membrane.","The sequence of gene VIII is known, and the amino acid sequence can be encoded on a synthetic gene, using lacUV5 promoter and used in conjunction with the LacIrepressor. The lacUV5 promoter is induced by IPTG. Mature gene VIII protein makes up the sheath around the circular ssDNA. The 3D structure of f1 virion is known at medium resolution; the amino terminus of gene VIII protein is on surface of the virion. A few modifications of gene VIII have been made and are discussed below. The 2D structure of M13 coat protein is implicit in the 3D structure. Mature M13 gene VIII protein has only one domain.","When the GP is M13 the gene III and the gene VIII proteins are highly preferred as OSP (see Examples I through IV). The proteins from genes VI, VII, and IX may also be used.","As discussed in the Examples, we have constructed a tripartite gene comprising:\n\n","The gene VIII protein is a preferred OSP because it is present in many copies and because its location and orientation in the virion are known (BANN81). Preferably, the PBD is attached to the amino terminus of the mature M13 coat protein. Had direct fusion of PBD to M13 CP failed to cause PBD to be displayed on the surface of M13, we would have varied part of the mini-protein sequence and\/or insert short random or nonrandom spacer sequences between mini-protein and M13 CP. The 3D model of f1 indicates strongly that fusing IPBD to the amino terminus of M13 CP is more likely to yield a functional chimeric protein than any other fusion site.","Similar constructions could be made with other filamentous phage. Pf3 is a well known filamentous phage that infects cells that harbor an IncP-1 plasmid. The entire genome has been sequenced (LUIT85) and the genetic signals involved in replication and assembly are known (LUIT87). The major coat protein of PF3 is unusual in having no signal peptide to direct its secretion. The sequence has charged residues ASP, ARG, LYS, and PHE-COO which is consistent with the amino terminus being exposed. Thus, to cause an IPBD to appear on the surface of Pf3, we construct a tripartite gene comprising:\n\n","Optionally, DNA encoding a flexible linker of one to 10 amino acids is introduced between the ipbd gene fragment and the Pf3 coat-protein gene. Optionally, DNA encoding the recognition site for a specific protease, such as tissue plasminogen activator or blood clotting Factor Xa, is introduced between the ipbd gene fragment and the Pf3 coat-protein gene. Amino acids that form the recognition site for a specific protease may also serve the function of a flexible linker. This tripartite gene is introduced into Pf3 so that it does not interfere with expression of any Pf3 genes. To reduce the possibility of genetic recombination, part (3) is designed to have numerous silent mutations relative to the wild-type gene. Once the signal sequence is cleaved off, the IPBD is in the periplasm and the mature coat protein acts as an anchor and phage-assembly signal. It matters not that this fusion protein comes to rest anchored in the lipid bilayer by a route different from the route followed by the wild-type coat protein.","The amino-acid sequence of M13 pre-coat (SCHA78), called AA_seq1, is",{"@attributes":{"id":"p-0323","num":"0429"},"tables":{"@attributes":{"id":"TABLE-US-00008","num":"00008"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00002","num":"00002"},"img":{"@attributes":{"id":"EMI-C00002","he":"27.43mm","wi":"74.17mm","file":"US07413537-20080819-C00002.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}}},"br":{},"i":"E. coli "},"Another vehicle for displaying the IPBD is by expressing it as a domain of a chimeric gene containing part or all of gene III. This gene encodes one of the minor coat proteins of M13. Genes VI, VII, and IX also encode minor coat proteins. Each of these minor proteins is present in about 5 copies per virion and is related to morphogenesis or infection. In contrast, the major coat protein is present in more than 2500 copies per virion. The gene VI, VII, and IX proteins are present at the ends of the virion; these three proteins are not post-translationally processed (RASC86).","The single-stranded circular phage DNA associates with about five copies of the gene III protein and is then extruded through the patch of membrane-associated coat protein in such a way that the DNA is encased in a helical sheath of protein (WEBS78). The DNA does not base pair (that would impose severe restrictions on the virus genome); rather the bases intercalate with each other independent of sequence.","Smith (SMIT85) and de la Cruz et al. (DELA88) have shown that insertions into gene III cause novel protein domains to appear on the virion outer surface. The mini-protein's gene may be fused to gene III at the site used by Smith and by de la Cruz et al., at a codon corresponding to another domain boundary or to a surface loop of the protein, or to the amino terminus of the mature protein.","All published works use a vector containing a single modified gene III of fd. Thus, all five copies of gIII are identically modified. Gene III is quite large (1272 b.p. or about 20% of the phage genome) and it is uncertain whether a duplicate of the whole gene can be stably inserted into the phage. Furthermore, all five copies of gIII protein are at one end of the virion. When bivalent target molecules (such as antibodies) bind a pentavalent phage, the resulting complex may be irreversible. Irreversible binding of the GP to the target greatly interferes with affinity enrichment of the GPs that carry the genetic sequences encoding the novel polypeptide having the highest affinity for the target.","To reduce the likelihood of formation of irreversible complexes, we may use a second, synthetic gene that encodes carboxy-terminal parts of III. We might, for example, engineer a gene that consists of (from 5\u2032 to 3\u2032):\n\n","M13 gene VI, VII, and IX proteins are not processed after translation. The route by which these proteins are assembled into the phage have not been reported. These proteins are necessary for normal morphogenesis and infectivity of the phage. Whether these molecules (gene VI protein, gene VII protein, and gene IX protein) attach themselves to the phage: a) from the cytoplasm, b) from the periplasm, or c) from within the lipid bilayer, is not known. One could use any of these proteins to introduce an IPBD onto the phage surface by one of the constructions:\n\n","The bacteriophage \u03a6X174 is a very small icosahedral virus which has been thoroughly studied by genetics, biochemistry, and electron microscopy (See -(DENH78)). To date, no proteins from \u03a6X174 have been studied by X-ray diffraction. \u03a6X174 is not used as a cloning vector because \u03a6X174 can accept very little additional DNA; the virus is so tightly constrained that several of its genes overlap. Chambers et al. (CHAM82) showed that mutants in gene G are rescued by the wild-type G gene carried on a plasmid so that the host supplies this protein.","Three gene products of \u03a6X174 are present on the outside of the mature virion: F (capsid), G (major spike protein, 60 copies per virion), and H (minor spike protein, 12 copies per virion). The G protein comprises 175 amino acids, while H comprises 328 amino acids. The F protein interacts with the single-stranded DNA of the virus. The proteins F, G, and H are translated from a single mRNA in the viral infected cells. If the G protein is supplied from a plasmid in the host, then the viral g gene is no longer essential. We introduce one or more stop codons into g so that no G is produced from the viral gene. We fuse a pbd gene fragment to h, either at the 3\u2032 or 5\u2032 terminus. We eliminate an amount of the viral g gene equal to the size of pbd so that the size of the genome is unchanged.","Large DNA Phages","Phage such as \u03bb or T4 have much larger genomes than do M13 or \u03a6X174. Large genomes are less conveniently manipulated than small genomes. Phage \u03bb has such a large genome that cassette mutagenesis is not practicable. One can not use annealing of a mutagenic oligonucleotide either, because there is no ready supply of single-stranded \u03bb DNA. (\u03bb DNA is packaged as double-stranded DNA.) Phage such as \u03bb and T4 have more complicated 3D capsid structures than M13 or \u03a6X174, with more OSPs to choose from. Intracellular morphogenesis of phage \u03bb could cause protein domains that contain disulfide bonds in their folded forms not to fold.","Phage \u03bb virions and phage T4 virions form intracellularly, so that IPBDs requiring large or insoluble prosthetic groups might fold on the surfaces of these phage.","RNA Phages","RNA phage are not preferred because manipulation of RNA is much less convenient than is the manipulation of DNA. If the RNA phage MS2 were modified to make room for an osp-ipbd gene and if a message containing the A protein binding site and the gene for a chimera of coat protein and a PBD were produced in a cell that also contained A protein and wild-type coat protein (both produced from regulated genes on a plasmid), then the RNA coding for the chimeric protein would get packaged. A package comprising RNA encapsulated by proteins encoded by that RNA satisfies the major criterion that the genetic message inside the package specifies something on the outside. The particles by themselves are not viable unless the modified A protein is functional. After isolating the packages that carry an SBD, we would need to: 1) separate the RNA from the protein capsid; 2) reverse transcribe the RNA into DNA, using AMV or MMTV reverse transcriptase, and 3) use DNA polymerase for 25 or more cycles of Polymerase Chain Reaction\u2122 to amplify the osp-sbd DNA until there is enough to subclone the recovered genetic message into a plasmid for sequencing and further work.","Alternatively, helper phage could be used to rescue the isolated phage. In one of these ways we can recover a sequence that codes for an SBD having desirable binding properties.","IV.C. Bacterial Cells as Genetic Packages:","One may choose any well-characterized bacterial strain which (1) may be grown in culture (2) may be engineered to display PBDs on its surface, and (3) is compatible with affinity selection.","Among bacterial cells, the preferred genetic packages are and especially The potential binding mini-protein may be expressed as an insert in a chimeric bacterial outer surface protein (OSP). All bacteria exhibit proteins on their outer surfaces. Works on the localization of OSPs and the methods of determining their structure include: CALA90, HEIJ90, EHRM90, BENZ88a, BENZ88b, MANO88, BAKE87, RAND87, HANC87, HENR87, NAKA86b, MANO86, SILH85, TOMM85, NIKA84, LUGT83, and BECK83.","In LamB is a preferred OSP. As discussed below, there are a number of very good alternatives in and there are very good alternatives in other bacterial species. There are also methods for determining the topology of OSPs so that it is possible to systematically determine where to insert an ipbd into an osp gene to obtain display of an IPBD on the surface of any bacterial species.","In view of the extensive knowledge of a strain of defective in recombination, is the strongest candidate as a bacterial GP.","Oliver has reviewed mechanisms of protein secretion in bacteria (OLIV85a and OLIV87). Nikaido and Vaara (NIKA87), Benz (BENZ88b), and Baker et al. (BAKE87) have reviewed mechanisms by which proteins become localized to the outer membrane of gram-negative bacteria. While most bacterial proteins remain in the cytoplasm, others are transported to the periplasmic space (which lies between the plasma membrane and the cell wall of gram-negative bacteria), or are conveyed and anchored to the outer surface of the cell. Still others are exported (secreted) into the medium surrounding the cell. Those characteristics of a protein that are recognized by a cell and that cause it to be transported out of the cytoplasm and displayed on the cell surface will be termed \u201couter-surface transport signals\u201d.","Gram-negative bacteria have outer-membrane proteins (OMP), that form a subset of OSPs. Many OMPs span the membrane one or more times. The signals that cause OMPs to localize in the outer membrane are encoded in the amino acid sequence of the mature protein. Outer membrane proteins of bacteria are initially expressed in a precursor form including a so-called signal peptide. The precursor protein is transported to the inner membrane, and the signal peptide moiety is extruded into the periplasmic space. There, it is cleaved off by a \u201csignal peptidase\u201d, and the remaining \u201cmature\u201d protein can now enter the periplasm. Once there, other cellular mechanisms recognize structures in the mature protein which indicate that its proper place is on the outer membrane, and transport it to that location.","It is well known that the DNA coding for the leader or signal peptide from one protein may be attached to the DNA sequence coding for another protein, protein X, to form a chimeric gene whose expression causes protein X to appear free in the periplasm (BECK83, INOU86 Ch10, LEEC86, MARK86, and BOQU87). That is, the leader causes the chimeric protein to be secreted through the lipid bilayer; once in the periplasm, it is cleaved off by the signal peptidase SP-I.","The use of export-permissive bacterial strains (LISS85, STAD89) increases the probability that a signal-sequence-fusion will direct the desired protein to the cell surface. Liss et al. (LISS85) showed that the mutation prlA4 makes more permissive with respect to signal sequences. Similarly, Stader et al. (STAD89) found a strain that bears a prlG mutation and that permits export of a protein that is blocked from export in wild-type cells. Such export-permissive strains are preferred.","OSP-IPBD fusion proteins need not fill a structural role in the outer membranes of Gram-negative bacteria because parts of the outer membranes are not highly ordered. For large OSPs there is likely to be one or more sites at which osp can be truncated and fused to ipbd such that cells expressing the fusion will display IPBDs on the cell surface. Fusions of fragments of omp genes with fragments of an x gene have led to X appearing on the outer membrane (CHAR88b, BENS84, CLEM81). When such fusions have been made, we can design an osp-ipbd gene by substituting ipbd for x in the DNA sequence. Otherwise, a successful OMP-IPBD fusion is preferably sought by fusing fragments of the best omp to an ipbd, expressing the fused gene, and testing the resultant GPs for display-of-IPBD phenotype. We use the available data about the OMP to pick the point or points of fusion between omp and ipbd to maximize the likelihood that IPBD will be displayed. (Spacer DNA encoding flexible linkers, made, e.g., of GLY, SER, and ASN, may be placed between the osp- and ipbd-derived fragments to facilitate display.) Alternatively, we truncate osp at several sites or in a manner that produces osp fragments of variable length and fuse the osp fragments to ipbd; cells expressing the fusion are screened or selected which display IPBDs on the cell surface. Freudl et al. (FREU89) have shown that fragments of OSPs (such as OmpA) above a certain size are incorporated into the outer membrane. An additional alternative is to include short segments of random DNA in the fusion of omp fragments to ipbd and then screen or select the resulting variegated population for members exhibiting the display-of-IPBD phenotype.","In , the LamB protein is a well understood OSP and can be used (BENS84, CHAR90, RONC90, VAND90, CHAP90, MOLL90, CHAR88b, CHAR88c, CLEM81, DARG88, FERE82a, FERE82b, FERE83, FERE84, FERE86a, FERE86b, FERE89a, FERE89b, GEHR87, HALL82, NAKA86a, STAD86, HEIN88, BENS87b, BENS87c, BOUG84, BOUL86a, CHAR84). The LamB has been expressed in functional form in (DEVR84, BARB85, HARK87), (HARK86), and (DEVR84, WEHM89), so that one could display a population of PBDs in any of these species as a fusion to LamB. expresses a maltoporin similar to LamB (WEHM89) which could also be used. In the D1 protein (a homologue of LamB) can be used (TRIA88).","LamB of is a porin for maltose and malto dextrin transport, and serves as the receptor for adsorption of bacteriophages \u03bb and K10. LamB is transported to the outer membrane if a functional N-terminal sequence is present; further, the first 49 amino acids of the mature sequence are required for successful transport (BENS84). As with other OSPs, LamB of is synthesized with a typical signa l-sequence which is subsequently removed. Homology between parts of LamB protein and other outer membrane proteins OmpC, OmpF, and PhoE has been detected (NIKA84), including homology between LamB amino acids 39-49 and sequences of the other proteins. These subsequences may label the proteins for transport to the outer membrane.","The amino acid sequence of LamB is known (CLEM81), and a model has been developed of how it anchors itself to the outer membrane (Reviewed by, among others, BENZ88b). The location of its maltose and phage binding domains are also known (HEIN88). Using this information, one may identify several strategies by which a PBD insert may be incorporated into LamB to provide a chimeric OSP which displays the PBD on the bacterial outer membrane.","When the PBDs are to be displayed by a chimeric transmembrane protein like LamB, the PBD could be inserted into a loop normally found on the surface of the cell (cp. BECK83, MANO86). Alternatively, we may fuse a 5\u2032 segment of the osp gene to the ipbd gene fragment; the point of fusion is picked to correspond to a surface-exposed loop of the OSP and the carboxy terminal portions of the OSP are omitted. In LamB, it has been found that up to 60 amino acids may be inserted (CHAR88b) with display of the foreign epitope resulting; the structural features of OmpC, OmpA, OmpF, and PhoE are so similar that one expects similar behavior from these proteins.","It should be noted that while LamB may be characterized as a binding protein, it is used in the present invention to provide an OSTS; its binding domains are not variegated.","Other bacterial outer surface proteins, such as OmpA, OmpC, OmpF, PhoE, and pilin, may be used in place of LamB and its homologues. OmpA is of particular interest because it is very abundant and because homologues are known in a wide variety of gram-negative bacterial species. Baker et al. (BAKE87) review assembly of proteins into the outer membrane of and cite a topological model of OmpA (VOGE86) that predicts that residues 19-32, 62-73, 105-118, and 147-158 are exposed on the cell surface. Insertion of a ipbd encoding fragment at about codon 111 or at about codon 152 is likely to cause the IPBD to be displayed on the cell surface. Concerning OmpA, see also MACI88 and MAN088. Porin Protein F of has been cloned and has sequence homology to OmpA of (DUCH88). Although this homology is not sufficient to allow prediction of surface-exposed residues on Porin Protein F, the methods used to determine the topological model of OmpA may be applied to Porin Protein F. Works related to use of OmpA as an OSP include BECK80 and MACI88.","Misra and Benson (MISR88a, MISR88b) disclose a topological model of OmpC that predicts that, among others, residues GLYand LEUare exposed on the cell surface. Thus insertion of an ipbd gene fragment at about codon 164 or at about codon 250 of the ompC gene or at corresponding codons of the ompC gene is likely to cause IPBD to appear on the cell surface. The ompc genes of other bacterial species may be used. Other works related to OmpC include CATR87 and CLIC88.","OmpF of is a very abundant OSP, \u226710copies\/cell. Pages et al. (PAGE90) have published a model of OmpF indicating seven surface-exposed segments. Fusion of an ipbd gene fragment, either as an insert or to replace the 3\u2032 part of ompF, in one of the indicated regions is likely to produce a functional ompF::ipbd gene the expression of which leads to display of IPBD on the cell surface. In particular, fusion at about codon 111, 177, 217, or 245 should lead to a functional ompF::ipbd gene. Concerning OmpF, see also REID88b, PAGE88, BENS88, TOMM82, and SODE85.","Pilus proteins are of particular interest because piliated cells express many copies of these proteins and because several species (and ) express related pilins. Getzoff and coworkers (GETZ88, PARG87, SOME85) have constructed a model of the gonococcal pilus that predicts that the protein forms a four-helix bundle having structural similarities to tobacco mosaic virus protein and myohemerythrin. On this model, both the amino and carboxy termini of the protein are exposed. The amino terminus is methylated. Elleman (ELLE88) has reviewed pilins of and other species and serotype differences can be related to differences in the pilin protein and that most variation occurs in the C-terminal region. The amino-terminal portions of the pilin protein are highly conserved. Jennings et al. (JENN89) have grafted a fragment of foot-and-mouth disease virus (residues 144-159) into the type 4 fimbrial protein which is highly homologous to gonococcal pilin. They found that expression of the 3\u2032-terminal fusion in led to a viable strain that makes detectable amounts of the fusion protein. Jennings et al. did not vary the foreign epitope nor did they suggest any variation. They inserted a GLY-GLY linker between the last pilin residue and the first residue of the foreign epitope to provide a \u201cflexible linker\u201d. Thus a preferred place to attach an IPBD is the carboxy terminus. The exposed loops of the bundle could also be used, although the particular internal fusions tested by Jennings et al. (JENN89) appeared to be lethal in Concerning pilin, see also MCKE85 and ORND85.","Judd (JUDD86, JUDD85) has investigated Protein IA of and found that the amino terminus is exposed; thus, one could attach an IPBD at or near the amino terminus of the mature P.IA as a means to display the IPBD on the surface.","A model of the topology of PhoE of has been disclosed by van der Ley et al. (VAND86). This model predicts eight loops that are exposed; insertion of an IPBD into one of these loops is likely to lead to display of the IPBD on the surface of the cell. Residues 158, 201, 238, and 275 are preferred locations for insertion of and IPBD.","Other OSPs that could be used include BtuB, FepA, FhuA, IutA, FecA, and FhuE (GUDM89) which are receptors for nutrients usually found in low abundance. The genes of all these proteins have been sequenced, but topological models are not yet available. Gudmunsdottir et al. (GUDM89) have begun the construction of such a model for BtuB and FepA by showing that certain residues of BtuB face the peri plasm and by determining the functionality of various BtuB::FepA fusions. Carmel et al. (CARM90) have reported work of a similar nature for FhuA. All Neisseria species express outer surface proteins for iron transport that have been identified and, in many cases, cloned. See also MORS87 and MORS88.","Many gram-negative bacteria express one or more phospholipases. phospholipase A, product of the plda gene, has been cloned and sequenced by de Geus et al. (DEGE84). They found that the protein appears at the cell surface without any posttranslational processing. A ipbd gene fragment can be attached at either terminus or inserted at positions predicted to encode loops in the protein. That phospholipase A arrives on the outer surface without removal of a signal sequence does not prove that a PldA::IPBD fusion protein will also follow this route. Thus we might cause a PldA::IPBD or IPBD::PldA fusion to be secreted into the periplasm by addition of an appropriate signal sequence. Thus, in addition to simple binary fusion of an ipbd fragment to one terminus of plda, the constructions:\n\n","Bacterial spores have desirable properties as GP candidates. Spores are much more resistant than vegetative bacterial cells or phage to chemical and physical agents, and hence permit the use of a great variety of affinity selection conditions. Also, spores neither actively metabolize nor alter the proteins on their surface. Spores have the disadvantage that the molecular mechanisms that trigger sporulation are less well worked out than is the formation of M13 or the export of protein to the outer membrane of ","Bacteria of the genus form endospores that are extremely resistant to damage by heat, radiation, desiccation, and toxic chemicals (reviewed by Losick et al. (LOSI86)). This phenomenon is attributed to extensive intermolecular crosslinking of the coat proteins. Endospores from the genus are more stable than are exospores from forms spores in 4 to 6 hours, but species may require days or weeks to sporulate. In addition, genetic knowledge and manipulation is much more developed for than for other spore-forming bacteria. Thus spores are preferred over spores. Bacteria of the genus also form very durable endospores, but clostridia, being strict anaerobes, are not convenient to culture.","Viable spores that differ only slightly from wild-type are produced in even if any one of four coat proteins is missing (DONO87). Moreover, plasmid DNA is commonly included in spores, and plasmid encoded proteins have been observed on the surface of spores (DEBR86). For these reasons, we expect that it will be possible to express during sporulation a gene encoding a chimeric coat protein, without interfering materially with spore formation.","Donovan et al. have identified several polypeptide components of spore coat (DON087); the sequences of two complete coat proteins and amino-terminal fragments of two others have been determined. Some, but not all, of the coat proteins are synthesized as precursors and are then processed by specific proteases before deposition in the spore coat (DONO87). The 12 kd coat protein, CotD, contains 5 cysteines. CotD also contains an unusually high number of histidines (16) and prolines (7). The 11 kd coat protein, CotC, contains only one cysteine and one methionine. CotC has a very unusual amino-acid sequence with 19 lysines (K) appearing as 9 K-K dipeptides and one isolated K. There are also 20 tyro sines (Y) of which 10 appear as 5 Y\u2014Y dipeptides. Peptides rich in Y and K are known to become crosslinked in oxidizing environments (DEVO78, WAIT83, WAIT85, WAIT86). CotC contains 16 D and E amino acids that nearly equals the 19 Ks. There are no A, F, R, I, L, N, P, Q, S, or W amino acids in CotC. Neither CotC nor CotD is post-translationally cleaved, but the proteins CotA and CotB are.","Since, in , some of the spore coat proteins are post-translationally processed by specific proteases, it is valuable to know the sequences of precursors and mature coat proteins so that we can avoid incorporating the recognition sequence of the specific protease into our construction of an OSP-IPBD fusion. The sequence of a mature spore coat protein contains information that causes the protein to be deposited in the spore coat; thus gene fusions that include some or all of a mature coat protein sequence are preferred for screening or selection for the display-of-IPBD phenotype.","Fusions of ipbd fragments to cotC or cotD fragments are likely to cause IPBD to appear on the spore surface. The genes cotC and cotD are preferred osp genes because CotC and CotD are not post-translationally cleaved. Subsequences from cotA or cotB could also be used to cause an IPBD to appear on the surface of spores, but we must take the post-translational cleavage of these proteins into account. DNA encoding IPBD could be fused to a fragment of cotA or cotB at either end of the coding region or at sites interior to the coding region. Spores could then be screened or selected for the display-of-IPBD phenotype.","The promoter of a spore coat protein is most active: a) when spore coat protein is being synthesized and deposited onto the spore and b) in the specific place that spore coat proteins are being made. The sequences of several sporulation promoters are known; coding sequences operatively linked to such promoters are expressed only during sporulation. Ray et al. (RAYC87) have shown that the G4 promoter of is directly controlled by RNA polymerase bound to \u03c3. To date, no sporulation promoter has been shown to be inducible by an exogenous chemical inducer as the lac promoter of . Nevertheless, the quantity of protein produced from a sporulation promoter can be controlled by other factors, such as the DNA sequence around the Shine-Dalgarno sequence or codon usage. Chemically inducible sporulation promoters can be developed if necessary.","IV.E. Artificial OSPs","It is generally preferable to use as the genetic package a cell, spore or virus for which an outer surface protein which can be engineered to display a IPBD has already been identified. However, the present invention is not limited to such genetic packages.","It is believed that the conditions for an outer surface transport signal in a bacterial cell or spore are not particularly stringent, i.e., a random polypeptide of appropriate length (preferably 30-100 amino acids) has a reasonable chance of providing such a signal. Thus, by constructing a chimeric gene comprising a segment encoding the IPBD linked to a segment of random or pseudorandom DNA (the potential OSTS), and placing this gene under control of a suitable promoter, there is a possibility that the chimeric protein so encoded will function as an OSP-IPBD.","This possibility is greatly enhanced by constructing numerous such genes, each having a different potential OSTS, cloning them into a suitable host, and selecting for transformants bearing the IPBD (or other marker) on their outer surface. Use of secretion-permissive mutants, such as prlA4 (LISS85) or prlG (STAD89), can increase the probability of obtaining a working OSP-IPBD.","When seeking to display a IPBD on the surface of a bacterial cell, as an alternative to choosing a natural OSP and an insertion site in the OSP, we can construct a gene (the \u201cdisplay probe\u201d) comprising: a) a regulatable promoter (e.g. lacUV5), b) a Shine-Dalgarno sequence, c) a periplasmic transport signal sequence, d) a fusion of the ipbd gene with a segment of random DNA (as in Kaiser et al. (KAIS87)), e) a stop codon, and f) a transcriptional terminator.","When the genetic package is a spore, we can use the approach described above for attaching a IPBD to an cell, except that: a) a sporulation promoter is used, and b) no periplasmic signal sequence should be present.","For phage, because the OSP-IPBD fulfills a structural role in the phage coat, it is unlikely that any particular random DNA sequence coupled to the ipbd gene will produce a fusion protein that fits into the coat in a functional way. Nevertheless, random DNA inserted between large fragments of a coat protein gene and the pbd gene will produce a population that is likely to contain one or more members that display the IPBD on the outside of a viable phage.","As previously stated, the purpose of the random DNA is to encode an OSTS, like that embodied in known OSPs. The fusion of ipbd and the random DNA could be in either order, but ipbd upstream is slightly preferred. Isolates from the population generated in this way can be screened for display of the IPBD. Preferably, a version of selection-through-binding is used to select GPs that display IPBD on the GP surface. Alternatively, clonal isolates of GPs may be screened for the display-of-IPBD phenotype.","The preference for ipbd upstream of the random DNA arises from consideration of the manner in which the successful GP(IPBD) will be used. The present invention contemplates introducing numerous mutations into the pbd region of the osp-pbd gene, which, depending on the variegation scheme, might include gratuitous stop codons. If pbd precedes the random DNA, then gratuitous stop codons in pbd lead to no OSP-PBD protein appearing on the cell surface. If pbd follows the random DNA, then gratuitous stop codons in pbd might lead to incomplete OSP-PBD proteins appearing on the cell surface. Incomplete proteins often are non-specifically sticky so that GPs displaying incomplete PBDs are easily removed from the population.","The random DNA may be obtained in a variety of ways. Degenerate synthetic DNA is one possibility. Alternatively, pseudorandom DNA can be generated from any DNA having high sequence diversity, e.g., the genome of the organism, by partially digesting with an enzyme that cuts very often, e.g., Sau3AI. Alternatively, one could shear DNA having high sequence diversity, blunt the sheared DNA with the large fragment of DNA polymerase I (hereinafter referred to as Klenow fragment), and clone the sheared and blunted DNA into blunt sites of the vector (MANI82, p295, AUSU87).","If random DNA and phenotypic selection or screening are used to obtain a GP(IPBD), then we clone random DNA into one of the restriction sites that was designed into the display probe. A plasmid carrying the display probe is digested with the appropriate restriction enzyme and the fragmented, random DNA is annealed and ligated by standard methods. The ligated plasmids are used to transform cells that are grown and selected for expression of the antibiotic-resistance gene. Plasmid-bearing GPs are then selected for the display-of-IPBD phenotype by the affinity selection methods described hereafter, using AfM(IPBD) as if it were the target.","As an alternative to selecting GP(IPBD)s through binding to an affinity column, we can isolate colonies or plaques and screen for successful artificial OSPs through use of one of the methods listed below for verification of the display strategy.","IV.F Designing the osp-ipbd Gene Insert:","Genetic Construction and Expression Considerations","The (i)pbd-osp gene may be: a) completely synthetic, b) a composite of natural and synthetic DNA, or c) a composite of natural DNA fragments. The important point is that the pbd segment be easily variegated so as to encode a multitudinous and diverse family of PBDs as previously described. A synthetic ipbd segment is preferred because it allows greatest control over placement of restriction sites. Primers complementary to regions abutting the osp-ipbd gene on its 3\u2032 flank and to parts of the osp-ipbd gene that are not to be varied are needed for sequencing.","The sequences of regulatory parts of the gene are taken from the sequences of natural regulatory elements: a) promoters, b) Shine-Dalgarno sequences, and c) transcriptional terminators. Regulatory elements could also be designed from knowledge of consensus sequences of natural regulatory regions. The sequences of these regulatory elements are connected to the coding regions; restriction sites are also inserted in or adjacent to the regulatory regions to allow convenient manipulation.","The essential function of the affinity separation is to separate GPs that bear PBDs (derived from IPBD) having high affinity for the target from GPs bearing PBDs having low affinity for the target. If the elution volume of a GP depends on the number of PBDs on the GP surface, then a GP bearing many PBDs with low affinity, GP(PBDW), might co-elute with a GP bearing fewer PBDs with high affinity, GP(PBD). Regulation of the osp-pbd gene preferably is such that most packages display sufficient PBD to effect a good separation according to affinity. Use of a regulatable promoter to control the level of expression of the osp-pbd allows fine adjustment of the chromatographic behavior of the variegated population.","Induction of synthesis of engineered genes in vegetative bacterial cells has been exercised through the use of regulated promoters such as lacUV5, trpP, or tac (MANI82). The factors that regulate the quantity of protein synthesized include: a) promoter strength (cf. HOOP87), b) rate of initiation of translation (cf. GOLD87), c) codon usage, d) secondary structure of mRNA, including attenuators (cf. LAND87) and terminators (cf. YAGE87), e) interaction of proteins with mRNA (cf. MCPH86, MILL87b, WINT87), f) degradation rates of mRNA (cf. BRAW87, KING86), g) proteolysis (cf. GOTT87). These factors are sufficiently well understood that a wide variety of heterologous proteins can now be produced in and other host cells in at least moderate quantities (SKER88, BETT88). Preferably, the promoter for the osp-ipbd gene is subject to regulation by a small chemical inducer. For example, the lac promoter and the hybrid trp-lac (tac) promoter are regulatable with isopropyl thiogalactoside (IPTG). Hereinafter, we use \u201cXINDUCE\u201d as a generic term for a chemical that induces expression of a gene. The promoter for the constructed gene need not come from a natural osp gene; any regulatable bacterial promoter can be used.","Transcriptional regulation of gene expression is best understood and most effective, so we focus our attention on the promoter. If transcription of the osp-ipbd gene is controlled by the chemical XINDUCE, then the number of OSP-IPBDs per GP increases for increasing concentrations of XINDUCE until a fall-off in the number of viable packages is observed or until sufficient IPBD is observed on the surface of harvested GP(IPBD)s. The attributes that affect the maximum number of OSP-IPBDs per GP are primarily structural in nature. There may be steric hindrance or other unwanted interactions between IPBDs if OSP-IPBD is substituted for every wild-type OSP. Excessive levels of OSP-IPBD may also adversely affect the solubility or morphogenesis of the GP. For cellular and viral GPs, as few as five copies of a protein having affinity for another immobilized molecule have resulted in successful affinity separations (FERE82a, FERE82b, and SMIT85).","A non-leaky promoter is preferred. Non-leakiness is useful: a) to show that affinity of GP(osp-ipbd)s for AfM(IPBD) is due to the osp-ipbd gene, and b) to allow growth of GP(osp-ipbd) in the absence of XINDUCE if the expression of osp-ipbd is disadvantageous. The lacUV5 promoter in conjunction with the LacIrepressor is a preferred example.","An exemplary osp-ipbd gene has the DNA sequence shown in Table 25 and there annotated to explain the useful restriction sites and biologically important features, viz. the lacUV5 promoter, the lacO operator, the Shine-Dalgarno sequence, the amino acid sequence, the stop codons, and the trp attenuator transcriptional terminator.","The present invention is not limited to a single method of gene design. The osp-ipbd gene need not be synthesized in toto; parts of the gene may be obtained from nature. One may use any genetic engineering method to produce the correct gene fusion, so long as one can easily and accurately direct mutations to specific sites in the pbd DNA subsequence. In all of the methods of mutagenesis considered in the present invention, however, it is necessary that the coding sequence for the osp-ipbd gene be different from any other DNA in the OCV. The degree and nature of difference needed is determined by the method of mutagenesis to be used. If the method of mutagenesis is to be replacement of subsequences coding for the PBD with vgDNA, then the subsequences to be mutagenized are preferably bounded by restriction sites that are unique with respect to the rest of the OCV. Use of non-unique sites involves partial digestion which is less efficient than complete digestion of a unique site and is not preferred. If single-stranded-oligonucleotide-directed mutagenesis is to be used, then the DNA sequence of the subsequence coding for the IPBD must be unique with respect to the rest of the OCV.","The coding portions of genes to be synthesized are designed at the protein level and then encoded in DNA. The amino acid sequences are chosen to achieve various goals, including: a) display of a IPBD on the surface of a GP, b) change of charge on a IPBD, and c) generation of a population of PBDs from which to select an SBD. These issues are discuss in more detail below. The ambiguity in the genetic code is exploited to allow optimal placement of restriction sites and to create various distributions of amino acids at variegated codons.","While the invention does not require any particular number or placement of restriction sites, it is generally preferable to engineer restriction sites into the gene to facilitate subsequent manipulations. Preferably, the gene provides a series of fairly uniformly spaced unique restriction sites with no more than a preset maximum number of bases, for example 100, between sites. Preferably, the gene is designed so that its insertion into the OCV does not destroy the uniqueness of unique restriction sites of the OCV. Preferred recognition sites are those for restriction enzymes which a) generate cohesive ends, b) have unambiguous recognition, or c) have higher specific activity.","The ambiguity of the DNA between the restriction sites is resolved from the following considerations. If the given amino acid sequence occurs in the recipient organism, and if the DNA sequence of the gene in the organism is known, then, preferably, we maximize the differences between the engineered and natural genes to minimize the potential for recombination. In addition, the following codons are poorly translated in and, therefore, are avoided if possible: cta(L), cga (R), cgg (R), and agg (R). For other host species, different codon restrictions would be appropriate. Finally, long repeats of any one base are prone to mutation and thus are avoided. Balancing these considerations, we can design a DNA sequence.","Structural Considerations","The design of the amino-acid sequence for the ipbd-osp gene to encode involves a number of structural considerations. The design is somewhat different for each type of GP. In bacteria, OSPs are not essential, so there is no requirement that the OSP domain of a fusion have any of its parental functions beyond lodging in the outer membrane.","Relationship Between PBD and OSP","It is not required that the PBD and OSP domains have any particular spatial relationship; hence the process of this invention does not require use of the method of U.S. Pat. No. '692.","It is, in fact, desirable that the OSP not constrain the orientation of the PBD domain; this is not to be confused with lack of constraint within the PBD. Cwirla et al. (CWIR90), Scott and Smith (SCOT90), and Devlin et al. (DEVL90), have taught that variable residues in phage-displayed random peptides should be free of influence from the phage OSP. We teach that binding domains having a moderate to high degree of conformational constraint will exhibit higher specificity and that higher affinity is also possible. Thus, we prescribe picking codons for variegation that specify amino acids that will appear in a well-defined framework. The nature of the side groups is varied through a very wide range due to the combinatorial replacement of multiple amino acids. The main chain conformations of most PBDs of a given class is very similar. The movement of the PBD relative to the OSP should not, however, be restricted. Thus it is often appropriate to include a flexible linker between the PBD and the OSP. Such flexible linkers can be taken from naturally occurring proteins known to have flexible regions. For example, the gIII protein of M13 contains glycine-rich regions thought to allow the amino-terminal domains a high degree of freedom. Such flexible linkers may also be designed. Segments of polypeptides that are rich in the amino acids GLY, ASN, SER, and ASP are likely to give rise to flexibility. Multiple glycines are particularly preferred.","Constraints Imposed by OSP","When we choose to insert the PBD into a surface loop of an OSP such as LamB, OmpA, or M13 gIII protein, there are a few considerations that do not arise when PBD is joined to the end of an OSP. In these cases, the OSP exerts some constraining influence on the PBD; the ends of the PBD are held in more or less fixed positions. We could insert a highly varied DNA sequence into the osp gene at codons that encode a surface-exposed loop and select for cells that have a specific-binding phenotype. When the identified amino-acid sequence is synthesized (by any means), the constraint of the OSP is lost and the peptide is likely to have a much lower affinity for the target and a much lower specificity. Tan and Kaiser (TANN77) found that a synthetic model of BPTI containing all the amino acids of BPTI that contact trypsin has a Kfor trypsin \u224810higher than BPTI. Thus, it is strongly preferred that the varied amino acids be part of a PBD in which the structural constrains are supplied by the PBD.","It is known that the amino acids adjoining foreign epitopes inserted into LamB influence the immunological properties of these epitopes (VAND90). We expect that PBDs inserted into loops of LamB, OmpA, or similar OSPs will be influenced by the amino acids of the loop and by the OSP in general. To obtain appropriate display of the PBD, it may be necessary to add one or more linker amino acids between the OSP and the PBD. Such linkers may be taken from natural proteins or designed on the basis of our knowledge of the structural behavior of amino acids. Sequences rich in GLY, SER, ASN, ASP, ARG, and THR are appropriate. One to five amino acids at either junction are likely to impart the desired degree of flexibility between the OSP and the PBD.","Phage OSP","A preferred site for insertion of the ipbd gene into the phage osp gene is one in which: a) the IPBD folds into its original shape, b) the OSP domains fold into their original shapes, and c) there is no interference between the two domains.","If there is a model of the phage that indicates that either the amino or carboxy terminus of an OSP is exposed to solvent, then the exposed terminus of that mature OSP becomes the prime candidate for insertion of the ipbd gene. A low resolution 3D model suffices.","In the absence of a 3D structure, the amino and carboxy termini of the mature OSP are the best candidates for insertion of the ipbd gene. A functional fusion may require additional residues between the IPBD and OSP domains to avoid unwanted interactions between the domains. Random-sequence DNA or DNA coding for a specific sequence of a protein homologous to the IPBD or OSP, can be inserted between the osp fragment and the ipbd fragment if needed.","Fusion at a domain boundary within the OSP is also a good approach for obtaining a functional fusion. Smith exploited such a boundary when subcloning heterologous DNA into gene III of f1 (SMIT85).","The criteria for identifying OSP domains suitable for causing display of an IPBD are somewhat different from those used to identify and IPBD. When identifying an OSP, minimal size is not so important because the OSP domain will not appear in the final binding molecule nor will we need to synthesize the gene repeatedly in each variegation round. The major design concerns are that: a) the OSP::IPBD fusion causes display of IPBD, b) the initial genetic construction be reasonably convenient, and c) the osp::ipbd gene be genetically stable and easily manipulated. There are several methods of identifying domains. Methods that rely on atomic coordinates have been reviewed by Janin and Chothia (JANI85). These methods use matrices of distances between \u03b1 carbons (C), dividing planes (cf. ROSE85), or buried surface (RASH84). Chothia and col laborators have correlated the behavior of many natural proteins with domain structure (according to their definition). Rashin correctly predicted the stability of a domain comprising residues 206-316 of thermolysin (VITA84, RASH84).","Many researchers have used partial proteolysis and protein sequence analysis to isolate and identify stable domains. (See, for example, VITA84, POTE83, SCOT87a, and PAB079.) Pabo et al. used calorimetry as an indicator that the cI repressor from the coliphage \u03bb contains two domains; they then used partial proteolysis to determine the location of the domain boundary.","If the only structural information available is the amino acid sequence of the candidate OSP, we can use the sequence to predict turns and loops. There is a high probability that some of the loops and turns will be correctly predicted (cf. Chou and Fasman, (CHOU74)); these locations are also candidates for insertion of the ipbd gene fragment.","Bacterial OSPs","In bacterial OSPs, the major considerations are: a) that the PBD is displayed, and b) that the chimeric protein not be toxic.","From topological models of OSPs, we can determine whether the amino or carboxy termini of the OSP is exposed. If so, then these are excellent choices for fusion of the osp fragment to the ipbd fragment.","The lamB gene has been sequenced and is available on a variety of plasmids (CLEM81, CHAR88). Numerous fusions of fragments of lamB with a variety of other genes have been used to study export of proteins in . From various studies, Charbit et al. (CHAR88) have proposed a model that specifies which residues of LamB are: a) embedded in the membrane, b) facing the periplasm, and c) facing the cell surface; we adopt the numbering of this model for amino acids in the mature protein. According to this model, several loops on the outer surface are defined, including: 1) residues 88 through 111, 2) residues 145 through 165, and 3) 236 through 251.","Consider a mini-protein embedded in LamB. For example, insertion of DNA encoding GNXCXXXXCXSG(SEQ ID NO:8) between codons 153 and 154 of lamB is likely to lead to a wide variety of LamB derivatives being expressed on the surface of cells. G, N, S, and Gare supplied to allow the mini-protein sufficient orientational freedom that is can interact optimally with the target. Using affinity enrichment (involving, for example, FACS via a fluorescently labeled target, perhaps through several rounds of enrichment), we might obtain a strain (named, for example, BEST) that expresses a particular LamB derivative that shows high affinity for the predetermined target. An octapeptide having the sequence of the inserted residues 3 through 10 from BEST is likely to have an affinity and specificity similar to that observed in BEST because the octapeptide has an internal structure that keeps the amino acids in a conformation that is quite similar in the LamB derivative and in the isolated mini-protein.","Consideration of the Signal Peptide","Fusing one or more new domains to a protein may make the ability of the new protein to be exported from the cell different from the ability of the parental protein. The signal peptide of the wild-type coat protein may function for authentic polypeptide but be unable to direct export of a fusion. To utilize the Sec-dependent pathway, one may need a different signal peptide. Thus, to express and display a chimeric BPTI\/M13 gene VIII protein, we found it necessary to utilize a heterologous signal peptide (that of phoA).","Provision of a Means to Remove PBD from the GP","GPs that display peptides having high affinity for the target may be quite difficult to elute from the target, particularly a multivalent target. (Bacteria that are bound very tightly can simply multiply in situ.) For phage, one can introduce a cleavage site for a specific protease, such as blood-clotting Factor Xa, into the fusion OSP protein so that the binding domain can be cleaved from the genetic package. Such cleavage has the advantage that all resulting phage have identical OSPs and therefore are equally infective, even if polypeptide-displaying phage can be eluted from the affinity matrix without cleavage. This step allows recovery of valuable genes which might otherwise be lost. To our knowledge, no one has disclosed or suggested using a specific protease as a means to recover an information-containing genetic package or of converting a population of phage that vary in infectivity into phage having identical infectivity.","IV.G. Synthesis of Gene Inserts","The present invention is not limited as to how a designed DNA sequence is divided for easy synthesis. An established method is to synthesize both strands of the entire gene in overlapping segments of 20 to 50 nucleotides (nts) (THER88). An alternative method that is more suitable for synthesis of vgDNA is an adaptation of methods published by Oliphant et al. (OLIP86 and OLIP87) and Ausubel et al. (AUSU87). It differs from previous methods in that it: a) uses two synthetic strands, and b) does not cut the extended DNA in the middle. Our goals are: a) to produce longer pieces of dsDNA than can be synthesized as ssDNA on commercial DNA synthesizers, and b) to produce strands complementary to single-stranded vgDNA. By using two synthetic strands, we remove the requirement for a palindromic sequence at the 3\u2032 end.","DNA synthesizers can currently produce oligo-nts of lengths up to 200 nts in reasonable yield, M=200. The parameters N(the length of overlap needed to obtain efficient annealing) and N(the number of spacer bases needed so that a restriction enzyme can cut near the end of blunt-ended dsDNA) are determined by DNA and enzyme chemistry. N=10 and N=5 are reasonable values. Larger values of Nand Nare allowed but add to the length of ssDNA that is to be synthesized and reduce the net length of dsDNA that can be produced.","Let Abe the actual length of dsDNA to be synthesized, including any spacers. Amust be no greater than (2 M\u2212N). Let Qbe the number of nts that the overlap window can deviate from center,\n\n=(2 )\/2.\n\nQis never negative. It is preferred that the two fragments be approximately the same length so that the amounts synthesized will be approximately equal. This preference may be overridden by other considerations. The overall yield of dsDNA is usually dominated by the synthetic yield of the longer oligo-nt.\n","We use the following procedure to generate dsDNA of lengths up to (2 M\u2212N) nts through the use of Klenow fragment to extend synthetic ss DNA fragments that are not more than Mnts long. When a pair of long oligo-nts, complementary for Nnts at their 3\u2032 ends, are annealed there will be a free 3\u2032 hydroxyl and a long ssDNA chain continuing in the 5\u2032 direction on either side. We will refer to this situation as a 5\u2032 superoverhang. The procedure comprises:\n\n","Restriction enzymes do not cut well at sites closer than about five base pairs from the end of blunt ds DNA fragments (OLIP87 and p.132 New England BioLabs 1990-1991 Catalogue). Therefore Nnts (with Ntypically set to 5) of spacer are added to ends that we intend to cut with a restriction enzyme. If the plasmid is to be cut with a blunt-cutting enzyme, then we do not add any spacer to the corresponding end of the ds DNA fragment.","To choose the optimum site of overlap for the oligo-nt fragments, first consider the anti-sense strand of the DNA to be synthesized, including any spacers at the ends, written (in upper case) from 5\u2032 to 3\u2032 and left-to-right. N.B.: The Nnt long overlap window can never include bases that are to be variegated. N.B.: The Nnt long overlap should not be palindromic lest single DNA molecules prime themselves. Place a Nnt long window as close to the center of the anti-sense sequence as possible. Check to see whether one or more codons within the window can be changed to increase the GC content without: a) destroying a needed restriction site, b) changing amino acid sequence, or c) making the overlap region palindromic. If possible, change some AT base pairs to GC pairs. If the GC content of the window is less than 50%, slide the window right or left as much as Qnts to maximize the number of C's and G's inside the window, but without including any variegated bases. For each trial setting of the overlap window, maximize the GC content by silent codon changes, but do not destroy wanted restriction sites or make the overlap palindromic. If the best setting still has less than 50% GC, enlarge the window to N+2 nts and place it within five nts of the center to obtain the maximum GC content. If enlarging the window one or two nts will increase the GC content, do so, but do not include variegated bases.","Underscore the anti-sense strand from the 5\u2032 end up to the right edge of the window. Write the complementary sense sequence 3\u2032-to-5\u2032 and left-to-right and in lower case letters, under the anti-sense strand starting at the left edge of the window and continuing all the way to the right end of the anti-sense strand.","We will synthesize the underscored anti-sense strand and the part of the sense strand that we wrote. These two fragments, complementary over the length of the window of high GC content, are mixed in equimolar quantities and annealed. These fragments are extended with Klenow fragment and all four deoxynucleotide triphosphates to produce ds blunt-ended DNA. This DNA can be cut with appropriate restriction enzymes to produce the cohesive ends needed to ligate the fragment to other DNA.","The present invention is not limited to any particular method of DNA synthesis or construction. Conventional DNA synthesizers may be used, with appropriate reagent modifications for production of variegated DNA (similar to that now used for production of mixed probes). For example, the Milligen 7500 DNA synthesizer has seven vials from which phosphoramidites may be taken. Normally, the first four contain A, C, T, and G. The other three vials may contain unusual bases such as inosine or mixtures of bases, the so-called \u201cdirty bottle\u201d. The standard software allows programmed mixing of two, three, or four bases in equimolar quantities.","The synthesized DNA may be purified by any art recognized technique, e.g., by high-pressure liquid chromatography (HPLC) or PAGE.","The osp-pbd genes may be created by inserting vgDNA into an existing parental gene, such as the osp-ipbd shown to be displayable by a suitably transformed GP. The present invention is not limited to any particular method of introducing the vgDNA, however, two techniques are discussed below.","In the case of cassette mutagenesis, the restriction sites that were introduced when the gene for the inserted domain was synthesized are used to introduce the synthetic vgDNA into a plasmid or other OCV. Restriction digestions and ligations are performed by standard methods (AUSU87).","In the case of single-stranded-oligonucleotide-directed mutagenesis, synthetic vgDNA is used to create diversity in the vector (BOTS85).","The modes of creating diversity in the population of GPs discussed herein are not the only modes possible. Any method of mutagenesis that preserves at least a large fraction of the information obtained from one selection and then introduces other mutations in the same domain will work. The limiting factors are the number of independent transformants that can be produced and the amount of enrichment one can achieve through affinity separation. Therefore the preferred embodiment uses a method of mutagenesis that focuses mutations into those residues that are most likely to affect the binding properties of the PBD and are least likely to destroy the underlying structure of the IPBD.","Other modes of mutagenesis might allow other GPs to be considered. For example, the bacteriophage \u03bb is not a useful cloning vehicle for cassette mutagenesis because of the plethora of restriction sites. One can, however, use single-stranded-oligo-nt-directed mutagenesis on \u03bb without the need for unique restriction sites. No one has used single-stranded-oligo-nt-directed mutagenesis to introduce the high level of diversity called for in the present invention, but if it is possible, such a method would allow use of phage with large genomes.","IV.H. Operative Cloning Vector","The operative cloning vector (OCV) is a replicable nucleic acid used to introduce the chimeric ipbd-osp or ipbd-osp gene into the genetic package. When the genetic package is a virus, it may serve as its own OCV. For cells and spores, the OCV may be a plasmid, a virus, a phagemid, or a chromosome.","The OCV is preferably small (less than 10 KB), stable (even after insertion of at least 1 kb DNA), present in multiple copies within the host cell, and selectable with appropriate media. It is desirable that cassette mutagenesis be practical in the OCV; preferably, at least 25 restriction enzymes are available that do not cut the OCV. It is likewise desirable that single-stranded mutagenesis be practical. If a suitable OCV does not already exist, it may be engineered by manipulation of available vectors.","When the GP is a bacterial cell or spore, the OCV is preferably a plasmid because genes on plasmids are much more easily constructed and mutated than are genes in the bacterial chromosome. When bacteriophage are to be used, the osp-ipbd gene is inserted into the phage genome. The synthetic osp-ipbd genes can be constructed in small vectors and transferred to the GP genome when complete.","Phage such as M13 do not confer antibiotic resistance on the host so that one can not select for cells infected with M13. An antibiotic resistance gene can be engineered into the M13 genome (HINE80). More virulent phage, such as \u03a6174, make discernable plaques that can be picked, in which case a resistance gene is not essential; furthermore, there is no room in the \u03a6X174 virion to add any new genetic material. Inability to include an antibiotic resistance gene is a disadvantage because it limits the number of GPs that can be screened.","It is preferred that GP(IPBD) carry a selectable marker not carried by wtGP. It is also preferred that wtGP carry a selectable marker not carried by GP(IPBD).","A derivative of M13 is the most preferred OCV when the phage also serves as the GP. Wild-type M13 does not confer any resistances on infected cells; M13 is a pure parasite. A \u201cphagemid\u201d is a hybrid between a phage and a plasmid, and is used in this invention. Double-stranded plasmid DNA isolated from phagemid-bearing cells is denoted by the standard convention, e.g. pXY24. Phage prepared from these cells would be designated XY24. Phagemids such as Bluescript K\/S (sold by Stratagene) are not preferred for our purposes because Bluescript does not contain the full genome of M13 and must be rescued by coinfection with competent wild-type M13. Such coinfections could lead to genetic recombination yielding heterogeneous phage unsuitable for the purposes of the present invention. Phagemids may be entirely suitable for developing a gene that causes an IPBD to appear on the surface of phage-like genetic packages.","It is also well known that plasmids containing the ColE1 origin of replication can be greatly amplified if protein synthesis is halted in a log-phase culture. Protein synthesis can be halted by addition of chloram phenicol or other agents (MANI82).","The bacteriophage M13 bla 61 (ATCC 37039) is derived from wild-type M13 through the insertion of the \u03b2 lactamase gene (HINE80). This phage contains 8.13 kb of DNA. M13 bla cat 1 (ATCC 37040) is derived from M13 bla 61 through the additional insertion of the chloramphenicol resistance gene (HINE80); M13 bla cat 1 contains 9.88 kb of DNA. Although neither of these variants of M13 contains the ColE1 origin of replication, either could be used as a starting point to construct a cloning vector with this feature.","IV.I. Transformation of Cells:","When the GP is a cell, the population of GPs is created by transforming the cells with suitable OCVs. When the GP is a phage, the phage are genetically engineered and then transfected into host cells suitable for amplification. When the GP is a spore, cells capable of sporulation are transformed with the OCV while in a normal metabolic state, and then sporulation is induced so as to cause the OSP-PBDs to be displayed. The present invention is not limited to any one method of transforming cells with DNA. The procedure given in the examples is a modification of that of Maniatis (p250, MANI82). One preferably obtains at least 10and more preferably at least 10transformants\/\u03bcg of CCC DNA.","The transformed cells are grown first under non-selective conditions that allow expression of plasmid genes and then selected to kill untransformed cells. Transformed cells are then induced to express the osp-pbd gene at the appropriate level of induction. The GPs carrying the IPBD or PBDs are then harvested by methods appropriate to the GP at hand, generally, centrifugation to pelletize GPs and resuspension of the pellets in sterile medium (cells) or buffer (spores or phage). They are then ready for verification that the display strategy was successful (where the GPs all display a \u201ctest\u201d IPBD) or for affinity selection (where the GPs display a variety of different PBDs).","IV.J. Verification of Display Strategy:","The harvested packages are tested to determine whether the IPBD is present on the surface. In any tests of GPs for the presence of IPBD on the GP surface, any ions or cofactors known to be essential for the stability of IPBD or AfM(IPBD) are included at appropriate levels. The tests can be done: a) by affinity labeling, b) enzymatically, c) spectrophotometrically, d) by affinity separation, or e) by affinity precipitation. The AfM(IPBD) in this step is one picked to have strong affinity (preferably, K<10M) for the IPBD molecule and little or no affinity for the wtGP. For example, if BPTI were the IPBD, trypsin, anhydrotrypsin, or antibodies to BPTI could be used as the AfM(BPTI) to test for the presence of BPTI. Anhydrotrypsin, a trypsin derivative with serine 195 converted to dehydroalanine, has no proteolytic activity but retains its affinity for BPTI (AKOH72 and HUBE77).","Preferably, the presence of the IPBD on the surface of the GP is demonstrated through the use of a soluble, labeled derivative of a AfM(IPBD) with high affinity for IPBD. The label could be: a) a radioactive atom such as I, b) a chemical entity such as biotin, or 3) a fluorescent entity such as rhodamine or fluorescein. The labeled derivative of AfM(IPBD) is denoted as AfM(IPBD)*. The preferred procedure is:\n\n","If the IPBD has strong, characteristic absorption bands in the visible or UV that are distinct from absorption by the wtGP, then another alternative for measuring the IPBD displayed on the GP is a spectrophotometric measurement. For example, if IPBD were azurin, the visible absorption could be used to identify GPs that display azurin.","Another alternative is to label the GPs and measure the amount of label retained by immobilized AfM(IPBD). For example, the GPs could be grown with a radioactive precursor, such as P or H-thymidine, and the radioactivity retained by immobilized AfM(IPBD) measured.","Another alternative is to use affinity chromatography; the ability of a GP bearing the IPBD to bind a matrix that supports a AfM(IPBD) is measured by reference to the wtGP.","Another alternative for detecting the presence of IPBD on the GP surface is affinity precipitation.","If random DNA has been used, then affinity selection procedures are used to obtain a clonal isolate that has the display-of-IPBD phenotype. Alternatively, clonal isolates may be screened for the display-of-IPBD phenotype. The tests of this step are applied to one or more of these clonal isolates.","If no isolates that bind to the affinity molecule are obtained we take corrective action as disclosed below.","If one or more of the tests above indicates that the IPBD is displayed on the GP surface, we verify that the binding of molecules having known affinity for IPBD is due to the chimeric osp-ipbd gene through the use of standard genetic and biochemical techniques, such as:\n\n","Variation of: a) binding of GPs by soluble AfM(IPBD)*, b) absorption caused by IPBD, and c) biochemical reactions of IPBD are linear in the amount of IPBD displayed. Presence of IPBD on the GP surface is indicated by a strong correlation between [XINDUCE] and the reactions that are linear in the amount of IPBD. Leakiness of the promoter is not likely to present problems of high background with assays that are linear in the amount of IPBD. These experiments may be quicker and easier than the genetic tests. Interpreting the effect of [XINDUCE] on binding to a {AfM(IPBD)} column, however, may be problematic unless the regulated promoter is completely repressed in the absence of [XINDUCE]. The affinity retention of GP(IPBD)s is not linear in the number of IPBDs\/GP and there may be, for example, little phenotypic difference between GPs bearing 5 IPBDs and GPs bearing 50 IPBDs. The demonstration that binding is to AfM(IPBD) and the genetic tests are essential; the tests with XINDUCE are optional.","We sequence the relevant ipbd gene fragment from each of several clonal isolates to determine the construction. We also establish the maximum salt concentration and pH range for which the GP(IPBD) binds the chosen AfM(IPBD). This is preferably done by measuring, as a function of salt concentration and pH, the retention of AfM(IPBD)* on molecular sizing filters that pass AfM(IPBD)* but not GP. This information will be used in refining the affinity selection scheme.","IV.K. Analysis and Correction of Display Problems","If the IPBD is displayed on the outside of the GP, and if that display is clearly caused by the introduced osp-ipbd gene, we proceed with variegation, otherwise we analyze the result and adopt appropriate corrective measures. If we have unsuccessfully attempted to fuse an ipbd fragment to a natural osp fragment, our options are :1) pick a different fusion to the same osp by a) using opposite end of osp, b) keeping more or fewer residues from osp in the fusion; for example, in increments of 3 or 4 residues, c) trying a known or predicted domain boundary, d) trying a predicted loop or turn position, 2) pick a different osp, or 3) switch to random DNA method. If we have just tried the random DNA method unsuccessfully, our options are: 1) choose a different relationship between ipbd fragment and random DNA (ipbd first, random DNA second or vice versa), 2) try a different degree of partial digestion, a different enzyme for partial digestion, a different degree of shearing or a different source of natural DNA, or 3) switch to the natural OSP method. If all reasonable OSPs of the current GP have been tried and the random DNA method has been tried, both without success, we pick a new GP.","We may illustrate the ways in which problems may be attacked by using the example of BPTI as the IPBD, the M13 phage as the GP, and the major coat (gene VIII) protein as the OSP. The following amino-acid sequence, called AA_seq2, (SEQ ID NO:128), illustrates how the sequence for mature BPTI (shown underscored)(SEQ ID NO:44) may be inserted immediately after the signal sequence of M13 precoat protein (indicated by the arrow) and before the sequence for the M13 CP.",{"@attributes":{"id":"p-0443","num":"0578"},"tables":{"@attributes":{"id":"TABLE-US-00009","num":"00009"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00003","num":"00003"},"img":{"@attributes":{"id":"EMI-C00003","he":"35.22mm","wi":"74.59mm","file":"US07413537-20080819-C00003.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}}}},"We adopt the convention that sequence numbers of fusion proteins refer to the fusion, as coded, unless otherwise noted. Thus the alanine that begins M13 CP is referred to as \u201cnumber 82\u201d, \u201cnumber 1 of M13 CP\u201d, or \u201cnumber 59 of the mature BPTI-M13 CP fusion\u201d.","It is desirable to determine where, exactly, the BPTI binding domain is being transported: is it remaining in the cytoplasm? Is it free within the periplasm? Is it attached to the inner membrane? Proteins in the periplasm can be freed through spheroplast formation using lysozyme and EDTA in a concentrated sucrose solution (BIRD67, MALA64). If BPTI were free in the periplasm, it would be found in the supernatant. Trypsin labeled with I would be mixed with supernatant and passed over a non-denaturing molecular sizing column and the radioactive fractions collected. The radioactive fractions would then be analyzed by SDS-PAGE and examined for BPTI-sized bands by silver staining.","Spheroplast formation exposes proteins anchored in the inner membrane. Spheroplasts would be mixed with AHTrp* and then either filtered or centrifuged to separate them from unbound AHTrp*. After washing with hypertonic buffer, the spheroplasts would be analyzed for extent of AHTrp* binding.","If BPTI were found free in the periplasm, then we would expect that the chimeric protein was being cleaved both between BPTI and the M13 mature coat sequence and between BPTI and the signal sequence. In that case, we should alter the BPTI\/M13 CP junction by inserting vgDNA at codons for residues 78-82 of AA_seq2.","If BPTI were found attached to the inner membrane, then two hypotheses can be formed. The first is that the chimeric protein is being cut after the signal sequence, but is not being incorporated into LG7 virion; the treatment would also be to insert vgDNA between residues 78 and 82 of AA_seq2. The alternative hypothesis is that BPTI could fold and react with trypsin even if signal sequence is not cleaved. N-terminal amino acid sequencing of trypsin-binding material isolated from cell homogenate determines what processing is occurring. If signal sequence were being cleaved, we would use the procedure above to vary residues between C78 and A82; subsequent passes would add residues after residue 81. If signal sequence were not being cleaved, we would vary residues between 23 and 27 of AA_seq2. Subsequent passes through that process would add residues after 23.","If BPTI were found neither in the periplasm nor on the inner membrane, then we would expect that the fault was in the signal sequence or the signal-sequence-to-BPTI junction. The treatment in this case would be to vary residues between 23 and 27.","Analytical experiments to determine what has gone wrong take time and effort and, for the foreseen out comes, indicate variations in only two regions. There fore, we believe it prudent to try the synthetic experiments described below without doing the analysis. For example, these six experiments that introduce variegation into the bpti-gene VIII fusion could be tried:\n\n","To alter the BPTI-M13 CP junction, we introduce DNA variegated at codons for residues between 78 and 82 into the SphI and SfiI sites of pLG7. The residues after the last cysteine are highly variable in amino acid sequences homologous to BPTI, both in composition and length; in Table 25 these residues are denoted as G79, G80, and A81. The first part of the M13 CP is denoted as A82, E83, and G84. One of the oligo-nts olig#12, olig#12a, or olig#12b and the primer olig#13 are synthesized by standard methods. The oligo-nts are:",{"@attributes":{"id":"p-0452","num":"0593"},"chemistry":{"@attributes":{"id":"CHEM-US-00004","num":"00004"},"img":{"@attributes":{"id":"EMI-C00004","he":"81.87mm","wi":"76.12mm","file":"US07413537-20080819-C00004.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{}},"To vary the junction between M13 signal sequence and BPTI, we introduce DNA variegated at codons for residues between 23 and 27 into the KpI and XhoI sites of pLG7. The first three residues are highly variable in amino acid sequences homologous to BPTI. Homologous sequences also vary in length at the amino terminus. One of the oligo-nts olig#14, olig#14a, or olig#14b and the primer olig#15 are synthesized by standard methods. The oligo-nts are:",{"@attributes":{"id":"p-0454","num":"0595"},"chemistry":{"@attributes":{"id":"CHEM-US-00005","num":"00005"},"img":{"@attributes":{"id":"EMI-C00005","he":"68.33mm","wi":"75.86mm","file":"US07413537-20080819-C00005.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{}},"Other numbers of variegated codons could be used.","If none of these approaches produces a working chimeric protein, we may try a different signal sequence. If that doesn't work, we may try a different OSP.","V. Affinity Selection of Target-Binding Mutants","V.A. Affinity Separation Technology, Generally","Affinity separation is used initially in the present invention to verify that the display system is working, i.e., that a chimeric outer surface protein has been expressed and transported to the surface of the genetic package and is oriented so that the inserted binding domain is accessible to target material. When used for this purpose, the binding domain is a known binding domain for a particular target and that target is the affinity molecule used in the affinity separation process. For example, a display system may be validated by using inserting DNA encoding BPTI into a gene encoding an outer surface protein of the genetic package of interest, and testing for binding to anhydrotrypsin, which is normally bound by BPTI.","If the genetic packages bind to the target, then we have confirmation that the corresponding binding domain is indeed displayed by the genetic package. Packages which display the binding domain (and thereby bind the target) are separated from those which do not.","Once the display system is validated, it is possible to use a variegated population of genetic packages which display a variety of different potential binding domains, and use affinity separation technology to determine how well they bind to one or more targets. This target need not be one bound by a known binding domain which is parental to the displayed binding domains, i.e., one may select for binding to a new target.","For example, one may variegate a BPTI binding domain and test for binding, not to trypsin, but to another serine protease, such as human neutrophil elastase or cathepsin G, or even to a wholly unrelated target, such as horse heart myoglobin.","The term \u201caffinity separation means\u201d includes, but is not limited to: a) affinity column chromatography, b) batch elution from an affinity matrix material, c) batch elution from an affinity material attached to a plate, d) fluorescence activated cell sorting, and e) electrophoresis in the presence of target material. \u201cAffinity material\u201d is used to mean a material with affinity for the material to be purified, called the \u201canalyte\u201d. In most cases, the association of the affinity material and the analyte is reversible so that the analyte can be freed from the affinity material once the impurities are washed away.","The procedures described in sections V.H, V.I and V.J are not required for practicing the present invention, but may facilitate the development of novel binding proteins thereby.","V.B. Affinity Chromatography, Generally","Affinity column chromatography, batch elution from an affinity matrix material held in some container, and batch elution from a plate are very similar and hereinafter will be treated under \u201caffinity chromatography.\u201d","If affinity chromatography is to be used, then:\n\n","Affinity chromatography is the preferred separation means, but FACS, electrophoresis, or other means may also be used.","V.C. Fluorescent-Activated Cell Sorting, Generally","Fluorescent-activated cell sorting involves use of an affinity material that is fluorescent per se or is labeled with a fluorescent molecule. Current commercially available cell sorters require 800 to 1000 molecules of fluorescent dye, such as Texas red, bound to each cell. FACS can sort 10cells or viruses\/sec.","FACS (e.g. FACStar from Beckton-Dickinson, Mountain View, Calif.) is most appropriate for bacterial cells and spores because the sensitivity of the machines requires approximately 1000 molecules of fluorescent label bound to each GP to accomplish a separation. OSPs such as OmpA, OmpF, OmpC are present at \u226710\/cell, often as much as 10\/cell. Thus use of FACS with PBDs displayed on one of the OSPs of a bacterial cell is attractive. This is particularly true if the target is quite small so that attachment to a matrix has a much greater effect than would attachment to a dye. To optimize FACS separation of GPs, we use a derivative of Afm(IPBD) that is labeled with a fluorescent molecule, denoted Afm(IPBD)*. The variables to be optimized include: a) amount of IPBD\/GP, b) concentration of Afm(IPBD)*, c) ionic strength, d) concentration of GPs, and e) parameters pertaining to operation of the FACS machine. Because Afm(IPBD)* and GPs interact in solution, the binding will be linear in both [Afm(IPBD)*] and [displayed IPBD]. Preferably, these two parameters are varied together. The other parameters can be optimized independently.","If FACS is to be used as the affinity separation means, then:\n\n","Electrophoretic affinity separation involves electrophoresis of viruses or cells in the presence of target material, wherein the binding of said target material changes the net charge of the virus particles or cells. It has been used to separate bacteriophages on the basis of charge. (SERW87).","Electrophoresis is most appropriate to bacteriophage because of their small size (SERW87). Electrophoresis is a preferred separation means if the target is so small that chemically attaching it to a column or to a fluorescent label would essentially change the entire target. For example, chloroacetate ions contain only seven atoms and would be essentially altered by any linkage. GPs that bind chloroacetate would become more negatively charged than GPs that do not bind the ion and so these classes of GPs could be separated.","If affinity electrophoresis is to be used, then:\n\n","The present invention makes use of affinity separation of bacterial cells, or bacterial viruses (or other genetic packages) to enrich a population for those cells or viruses carrying genes that code for proteins with desirable binding properties.","V.E. Target Materials","The present invention may be used to select for binding domains which bind to one or more target materials, and\/or fail to bind to one or more target materials. Specificity, of course, is the ability of a binding molecule to bind strongly to a limited set of target materials, while binding more weakly or not at all to another set of target materials from which the first set must be distinguished.","The target materials may be organic macromolecules, such as polypeptides, lipids, polynucleic acids, and polysaccharides, but are not so limited. Almost any molecule that is stable in aqueous solvent may be used as a target. The following list of possible targets is given as illustration and not as limitation. The categories are not strictly mutually exclusive. The omission of any category is not to be construed to imply that said category is unsuitable as a target. Merck Index refers to the Eleventh Edition.\n\n",{"@attributes":{"id":"p-0475","num":"0699"},"tables":{"@attributes":{"id":"TABLE-US-00010","num":"00010"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["a. DNA",{},{}]},{"entry":["1) ds DNA:","5\u2032-ACTAGTCTC-3\u2032"]},{"entry":[{},"3\u2032-TGATCAGAG-5\u2032"]},{"entry":["2) ds DNA:","5\u2032-CCGTCGAATCCGC-3\u2032","(SEQ ID NO:90)"]},{"entry":[{},"3\u2032-GGCAGTTAGGCG-5\u2032","(SEQ ID NO:91)"]},{"entry":[{},"(Note mismatch)"]},{"entry":["3) ss DNA:","5\u2032-CGTAACCTCGTCATTA-3\u2032"]},{"entry":[{},"(No hair pin)","(SEQ ID NO:92)"]},{"entry":{}},{"entry":["4) ss DNA:",{"chemistry":{"@attributes":{"id":"CHEM-US-00006","num":"00006"},"img":{"@attributes":{"id":"EMI-C00006","he":"6.10mm","wi":"17.78mm","file":"US07413537-20080819-C00006.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}]},{"entry":[{},"(Note hair pin)","(SEQ ID NO:93)"]},{"entry":{}},{"entry":["5) dsDNA with","5\u2032-CACGGCTATTACGGT-3\u2032","(SEQ ID NO:94)"]},{"entry":["cohesive","3\u2032-\u2003\u2003\u2003\u2003CCGATAATGCCA-5\u2032","(SEQ ID NO:95)"]},{"entry":"ends:"}]}}}},"ul":{"@attributes":{"id":"ul0061","list-style":"none"},"li":[{"@attributes":{"id":"ul0061-0001","num":"0000"},"ul":{"@attributes":{"id":"ul0062","list-style":"none"},"li":"b. RNA\n        \n        "}},"D. Organic Molecules (Not Peptide, Protein, or Nucleic Acid)\n    \n    ","E. Inorganic Compounds\n    \n    ","F. Organometallic Compounds\n    \n    "]}},"Serine proteases are an especially interesting class of potential target materials. Serine proteases are ubiquitous in living organisms and play vital roles in processes such as: digestion, blood clotting, fibrinolysis, immune response, fertilization, and post-translational processing of peptide hormones. Although the role these enzymes play is vital, uncontrolled or inappropriate proteolytic activity can be very damaging. Several serine proteases are directly involved in serious disease states. Uncontrolled neutrophil elastase (NE) (also known as leukocyte elastase) is thought to-be the major cause of emphysema (BEIT86, HUBB86, HUBB89, HUTC87, SOMM90, WEWE87) whether caused by congenital lack of \u03b1-1-antitrypsin or by smoking. NE is also implicated as an essential ingredient in the pernicious cycle of:",{"@attributes":{"id":"p-0477","num":"0737"},"tables":{"@attributes":{"id":"TABLE-US-00011","num":"00011"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00007","num":"00007"},"img":{"@attributes":{"id":"EMI-C00007","he":"21.84mm","wi":"54.10mm","file":"US07413537-20080819-C00007.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}}},"br":{},"sup":"\u221210 "},"HNE is not the only serine protease for which an inhibitor would be valuable. Works concerning uses of protease inhibitors and diseases thought to result from inappropriate protease activity include: NADE87, REST88, SOMM90, and SOMM89. Tryptase and chymase may be involved in asthma, see FRAN89 and VAND89. There are reports that suggest that Proteinase 3 (also known as p29) is as important or even more important than HNE; see NILE89, ARNA90, KAOR88, CAMP90, and GUPT90. Cathepsin G is another protease that may cause disease when present in excess; see FERR90, PETE89, SALV87, and SOMM90. These works indicate that a problem exists and that blocking one or another protease might well alleviate a disease state. Some of the cited works report inhibitors having measurable affinity for a target protease, but none report truly excellent inhibitors that have Kin the range of 10M as may be obtained by the method of the present invention. The same IPBDs used for HNE can be used for any serine protease.","The present invention is not, however, limited to any of the above-identified target materials. The only limitation is that the target material be suitable for affinity separation.","A supply of several milligrams of pure target material is desired. With HNE (as discussed in Examples II and III), 400 \u03bcg of enzyme is used to prepare 200 \u03bcl of ReactiGel beads. This amount of beads is sufficient for as many as 40 fractionations. Impure target material could be used, but one might obtain a protein that binds to a contaminant instead of to the target.","The following information about the target material is highly desirable: 1) stability as a function of temperature, pH, and ionic strength, 2) stability with respect to chaotropes such as urea or guanidinium Cl, 3) pI, 4) molecular weight, 5) requirements for prosthetic groups or ions, such as haem or Ca, and 6) proteolytic activity, if any. It is also potentially useful to know: 1) the target's sequence, if the target is a macromolecule, 2) the 3D structure of the target, 3) enzymatic activity, if any, and 4) toxicity, if any.","The user of the present invention specifies certain parameters of the intended use of the binding protein: 1) the acceptable temperature range, 2) the acceptable pH range, 3) the acceptable concentrations of ions and neutral solutes, and 4) the maximum acceptable dissociation constant for the target and the SBD:\n\n=[Target][]\/[Target:].\n\nIn some cases, the user may require discrimination between T, the target, and N, some non-target.\n\nLet\n\n],\n\nand\n\n],\n\nthen\n\n=([])\/([])\n\nThe user then specifies a maximum acceptable value for the ratio K\/K.\n","The target material preferably is stable under the specified conditions of pH, temperature, and solution conditions.","If the target material is a protease, one considers the following points:\n\n","inactivation by mutation or chemical modification are preferred methods of inactivation and a protein protease inhibitor becomes a prime candidate for IPBD. For example, BPTI has been mutated, by the methods of the present invention, to bind to proteases other than trypsin.","Example III-VI disclose that uninhibited serine proteases may be used as targets quite successfully and that protein protease inhibitors derived from BPTI and selected for binding to these immobilized proteases are excellent inhibitors.","V.F. Immobilization or Labeling of Target Material","For chromatography, FACS, or electrophoresis there may be a need to covalently link the target material to a second chemical entity. For chromatography the second entity is a matrix, for FACS the second entity is a fluorescent dye, and for electrophoresis the second entity is a strongly charged molecule. In many cases, no coupling is required because the target material already has the desired property of: a) immobility, b) fluorescence, or c) charge. In other cases, chemical or physical coupling is required.","Various means may be used to immobilize or label the target materials. The means of immobilization or labeling is, in part, determined by the nature of the target. In particular, the physical and chemical nature of the target and its functional groups of the target material determine which types of immobilization reagents may be most easily used.","For the purpose of selecting an immobilization method, it may be more helpful to classify target materials as follows: (a) solid, whether crystalline or amorphous, and insoluble in an aqueous solvent (e.g., many minerals, and fibrous organics such as cellulose and silk); (b) solid, whether crystalline or amorphous, and soluble in an aqueous solvent; (c) liquid, but insoluble in aqueous phase (e.g., 2,3,3-trimethyldecane); or (d) liquid, and soluble in aqueous media.","It is not necessary that the actual target material be used in preparing the immobilized or labeled analogue that is to be used in affinity separation; rather, suitable reactive analogues of the target material may be more convenient. If 2,3,3-trimethyldecane were the target material, for example, then 2,3,3-trimethyl-10-aminodecane would be far easier to immobilize than the parental compound. Because the latter compound is modified at one end of the chain, it retains almost all of the shape and charge attributes that differentiate the former compound from other alkanes.","Target materials that do not have reactive functional groups may be immobilized by first creating a reactive functional group through the use of some powerful reagent, such as a halogen. For example, an alkane can be immobilized for affinity by first halogenating it and then reacting the halogenated derivative with an immobilized or immobilizable amine.","In some cases, the reactive groups of the actual target material may occupy a part on the target molecule that is to be left undisturbed. In that case, additional functional groups may be introduced by synthetic chemistry. For example, the most reactive groups in cholesterol are on the steroid ring system, viz, \u2014OH and >C\u2550C. We may wish to leave this ring system as it is so that it binds to the novel binding protein. In this case, we prepare an analogue having a reactive group attached to the aliphatic chain (such as 26-aminocholesterol) and immobilize this derivative in a manner appropriate to the reactive group so attached.","Two very general methods of immobilization are widely used. The first is to biotinylate the compound of interest and then bind the biotinylated derivative to immobilized avidin. The second method is to generate antibodies to the target material, immobilize the anti bodies by any of numerous methods, and then bind the target material to the immobilized antibodies. Use of antibodies is more appropriate for larger target materials; small targets (those comprising, for example, ten or fewer non-hydrogen atoms) may be so completely engulfed by an antibody that very little of the target is exposed in the target-antibody complex.","Non-covalent immobilization of hydrophobic molecules without resort to antibodies may also be used. A compound, such as 2,3,3-trimethyldecane is blended with a matrix precursor, such as sodium alginate, and the mixture is extruded into a hardening solution. The resulting beads will have 2,3,3-trimethyldecane dispersed throughout and exposed on the surface.","Other immobilization methods depend on the presence of particular chemical functionalities. A polypeptide will present \u2014NH(N-terminal; Lysines), \u2014COOH (C-terminal; Aspartic Acids; Glutamic Acids), \u2014OH (Serines; Threonines; Tyrosines), and \u2014SH (Cysteines). A polysaccharide has free \u2014OH groups, as does DNA, which has a sugar backbone.","The following table is a nonexhaustive review of reactive functional groups and potential immobilization reagents:",{"@attributes":{"id":"p-0497","num":"0762"},"tables":{"@attributes":{"id":"TABLE-US-00012","num":"00012"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"126pt","align":"left"}}],"thead":{"row":[{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"Group","Reagent"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"R-NH","Derivatives of 2,4,6-trinitro"]},{"entry":[{},{},"benzene sulfonates (TNBS),"]},{"entry":[{},{},"(CREI84, p. 11)"]},{"entry":[{},"R-NH","Carboxylic acid anhydrides,"]},{"entry":[{},{},"e.g. derivatives of succinic"]},{"entry":[{},{},"anhydride, maleic anhydride,"]},{"entry":[{},{},"citraconic anhydride (CREI84,"]},{"entry":[{},{},"p. 11)"]},{"entry":[{},"R-NH","Aldehydes that form reducible"]},{"entry":[{},{},"Schiff bases (CREI84, p. 12)"]},{"entry":[{},"guanido","cyclohexanedione derivatives"]},{"entry":[{},{},"(CREI84, p. 14)"]},{"entry":[{},"R-COH","Diazo cmpds (CREI84, p. 10)"]},{"entry":[{},"R-CO\u2014","Epoxides (CREI84, p. 10)"]},{"entry":[{},"R-OH","Carboxylic acid anhydrides"]},{"entry":[{},"Aryl\u2014OH","Carboxylic acid anhydrides"]},{"entry":[{},"Indole ring","Benzyl halide and sulfenyl"]},{"entry":[{},{},"halides (CREI84, p. 19)"]},{"entry":[{},"R-SH","N-alkylmaleimides (CREI84,"]},{"entry":[{},{},"p. 21)"]},{"entry":[{},"R-SH","ethyleneimine derivatives"]},{"entry":[{},{},"(CREI84, p. 21)"]},{"entry":[{},"R-SH","Aryl mercury compounds,"]},{"entry":[{},{},"(CREI84, P. 21)"]},{"entry":[{},"R-SH","Disulfide reagents, (CREI84,"]},{"entry":[{},{},"p. 23)"]},{"entry":[{},"Thiol ethers","Alkyl iodides, (CREI84, p. 20)"]},{"entry":[{},"Ketones","Make Schiff's base and reduce"]},{"entry":[{},{},"with NaBH. (CREI84, p. 12-13)"]},{"entry":[{},"Aldehydes","Oxidize to COOH, vide supra."]},{"entry":[{},"R-SOH","Convert to R-SOCl and react"]},{"entry":[{},{},"with immobilized alcohol or"]},{"entry":[{},{},"amine."]},{"entry":[{},"R-POH","Convert to R-POCl and react"]},{"entry":[{},{},"with immobilized alcohol or"]},{"entry":[{},{},"amine."]},{"entry":[{},"CC double bonds","Add HBr and then make amine or"]},{"entry":[{},{},"thiol."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}}},"br":{}},{"@attributes":{"id":"p-0498","num":"0763"},"tables":{"@attributes":{"id":"TABLE-US-00013","num":"00013"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"left"}}],"thead":{"row":[{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},{},"Reactive groups or"]},{"entry":[{},"Compound (Item#, page)*","[derivatives]"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"prostaglandin E2","\u2014OH, keto, \u2014COOH, C\u2550C"]},{"entry":[{},"(2893, 1251)"]},{"entry":[{},"aspartame (861, 132)","\u2014NH, \u2014COOH, \u2014COOCH"]},{"entry":[{},"haem (4558, 732)","vinyl, \u2014COOH, Fe"]},{"entry":[{},"bilirubin (1235, 189)","vinyl, \u2014COOH, keto, \u2014NH\u2014"]},{"entry":[{},"morphine (6186, 988)","\u2014OH, \u2014C\u2550C\u2014, reactive phenyl"]},{"entry":[{},{},"ring"]},{"entry":[{},"codeine (2459, 384)","\u2014OH, \u2014C\u2550C\u2014, reactive phenyl"]},{"entry":[{},{},"ring"]},{"entry":[{},"dichlorodiphenyltrichlorethane","aromatic chlorine,"]},{"entry":[{},"(2832, 446)","aliphatic chlorine"]},{"entry":[{},"benzo (a) pyrene","[Chlorinate->amine, or make"]},{"entry":[{},"(1113, 172)","sulfonate-> Aryl-SOCl]"]},{"entry":[{},"actinomycin D","aryl-NH, \u2014OH"]},{"entry":[{},"(2804, 441)"]},{"entry":[{},"cellulose","self immobilized"]},{"entry":[{},"hydroxylapatite","self immobilized"]},{"entry":[{},"cholesterol (2204, 341)","\u2014OH, >C\u2550C\u2014"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"*Note:"]},{"entry":[{},"Item# and page refer to The Merck Index, 11th Edition."]}]}}}}},"The extensive literature on affinity chromatography and related techniques will provide further examples.","Matrices suitable for use as support materials include polystyrene, glass, agarose and other chromatographic supports, and may be fabricated into beads, sheets, columns, wells, and other forms as desired. Suppliers of support material for affinity chromatography include: Applied Protein Technologies Cambridge, Mass.; Bio-Rad Laboratories, Rockville Center, N.Y.; Pierce Chemical Company, Rockford, Ill. Target materials are attached to the matrix in accord with the directions of the manufacturer of each matrix preparation with consideration of good presentation of the target.","Early in the selection process, relatively high concentrations of target materials may be applied to the matrix to facilitate binding; target concentrations may subsequently be reduced to select for higher affinity SBDs.","V.G. Elution of Lower Affinity PBD-Bearing Genetic Packages","The population of GPs is applied to an affinity matrix under conditions compatible with the intended use of the binding protein and the population is fractionated by passage of a gradient of some solute over the column. The process enriches for PBDs having affinity for the target and for which the affinity for the target is least affected by the eluants used. The enriched fractions are those containing viable GPs that elute from the column at greater concentration of the eluant.","The eluants preferably are capable of weakening noncovalent interactions between the displayed PBDs and the immobilized target material. Preferably, the eluants do not kill the genetic package; the genetic message corresponding to successful mini-proteins is most conveniently amplified by reproducing the genetic package rather than by in vitro procedures such as PCR. The list of potential eluants includes salts (including Na+, NH+, Rb+, SO\u2212\u2212, HPO\u2212, citrate, K+, Li+, Cs+, HSO\u2212, CO\u2212\u2212, Ca++, Sr++, Cl\u2212, PO\u2212\u2212\u2212, HCO\u2212, Mg++, Ba++, Br\u2212, HPO\u2212\u2212 and acetate), acid, heat, compounds known to bind the target, and soluble target material (or analogues thereof).","Because bacteria continue to metabolize during affinity separation, the choice of buffer components is more restricted for bacteria than for bacteriophage or spores. Neutral solutes, such as ethanol, acetone, ether, or urea, are frequently used in protein purification and are known to weaken non-covalent interactions between proteins and other molecules. Many of these species are, however, very harmful to bacteria and bacteriophage. Urea is known not to harm M13 up to 8 M. Bacterial spores, on the other hand, are impervious to most neutral solutes. Several affinity separation passes may be made within a single round of variegation. Different solutes may be used in different analyses, salt in one, pH in the next, etc.","Any ions or cofactors needed for stability of PBDs (derived from IPBD) or target are included in initial and elution buffers at appropriate levels. We first remove GP(PBD)s that do not bind the target by washing the matrix with the initial buffer. We determine that this phase of washing is complete by plating aliquots of the washes or by measuring the optical density (at 260 nm or 280 nm). The matrix is then eluted with a gradient of increasing: a) salt, b) [H+] (decreasing pH), c) neutral solutes, d) temperature (increasing or decreasing), or e) some combination of these factors. The solutes in each of the first three gradients have been found generally to weaken non-covalent interactions between proteins and bound molecules. Salt is a preferred solute for gradient formation in most cases. Decreasing pH is also a highly preferred eluant. In some cases, the preferred matrix is not stable to low pH so that salt and urea are the most preferred reagents. Other solutes that generally weaken non-covalent interaction between proteins and the target material of interest may also be used.","The uneluted genetic packages contain DNA encoding binding domains which have a sufficiently high affinity for the target material to resist the elution conditions. The DNA encoding such successful binding domains may be recovered in a variety of ways. Preferably, the bound genetic packages are simply eluted by means of a change in the elution conditions. Alternatively, one may culture the genetic package in situ, or extract the target-containing matrix with phenol (or other suitable solvent) and amplify the DNA by PCR or by recombinant DNA techniques. Additionally, if a site for a specific protease has been engineered into the display vector, the specific protease is used to cleave the binding domain from the GP.","V.H. Optimization of Affinity Chromatography Separation:","For linear gradients, elution volume and eluant concentration are directly related. Changes in eluant concentration cause GPs to elute from the column. Elution volume, however, is more easily measured and specified. It is to be understood that the eluant concentration is the agent causing GP release and that an eluant concentration can be calculated from an elution volume and the specified gradient.","Using a specified elution regime, we compare the elution volumes of GP(IPBD)s with the elution volumes of wtGP on affinity columns supporting AfM(IPBD). Com parisons are made at various: a) amounts of IPBD\/GP, b) densities of AfM(IPBD)\/(volume of matrix) (DoAMoM), c) initial ionic strengths, d) elution rates, e) amounts of GP\/(volume of support), f) pHs, and g) temperatures, because these are the parameters most likely to affect the sensitivity and efficiency of the separation. We then pick those conditions giving the best separation.","We do not optimize pH or temperature; rather we record optimal values for the other parameters for one or more values of pH and temperature. The pH used must be within the range of pH for which GP(IPBD) binds the AfM(IPBD) that is being used in this step. The conditions of intended use specified by the user may include a specification of pH or temperature. If pH is specified, then pH will not be varied in eluting the column. Decreasing pH may, however, be used to liberate bound GPs from the matrix. Similarly, if the intended use specifies a temperature, we will hold the affinity column at the specified temperature during elution, but we might vary the temperature during recovery. If the intended use specifies the pH or temperature, then we prefer that the affinity separation be optimized for all other parameters at the specified pH and temperature.","In the optimization devised in this step, we preferably use a molecule known to have moderate affinity for the IPBD (Kin the range 10M to 10M), for the following reason. When populations of GP(vgPBD)s are fractionated, there will be roughly three subpopulations: a) those with no binding, b) those that have some binding but can be washed off with high salt or low pH, and c) those that bind very tightly and are most easily rescued in situ. We optimize the parameters to separate (a) from (b) rather than (b) from (c). Let PBDbe a PBD having weak binding to the target and PBD, be a PBD having strong binding. Higher DoAMoM might, for example, favor retention of GP(PBD) but also make it very difficult to elute viable GP(PBD). We will optimize the affinity separation to retain GP(PBD) rather than to allow release of GP(PBD) because a tightly bound GP(PBD) can be rescued by in situ growth. If we find that DoAMoM strongly affects the elution volume, then in part III we may reduce the amount of target on the affinity column when an SBD has been found with moderately strong affinity (Kon the order of 10M) for the target.","In case the promoter of the osp-ipbd gene is not regulated by a chemical inducer, we optimize DoAMoM, the elution rate, and the amount of GP\/volume of matrix. If the optimized affinity separation is acceptable, we proceed. If not, we develop a means to alter the amount of IPBD per GP. Among GPs considered in the present invention, this case could arise only for spores because regulatable promoters are available for all other systems.","If the amount of IPBD\/spore is too high, we could engineer an operator site into the osp-ipbd gene. We choose the operator sequence such that a repressor sensitive to a small diffusible inducer recognizes the operator. Alternatively, we could alter the Shine-Dalgarno sequence to produce a lower homology with consensus Shine-Dalgarno sequences. If the amount of IPBD\/spore is too low, we can introduce variability into the promoter or Shine-Dalgarno sequences and screen colonies for higher amounts of IPBD\/spore.","In this step, we measure elution volumes of genetically pure GPs that elute from the affinity matrix as sharp bands that can be detected by UV absorption. Alternatively, samples from effluent fractions can be plated on suitable medium (cells or spores) or on sensitive cells (phage) and colonies or plaques counted.","Several values of IPBD\/GP, DoAMoM, elution rates, initial ionic strengths, and loadings should be examined. The following is only one of many ways in which the affinity separation could be optimized. We anticipate that optimal values of IPBD\/GP and DoAMoM will be correlated and therefore should be optimized together. The effects of initial ionic strength, elution rate, and amount of GP\/(matrix volume) are unlikely to be strongly correlated, and so they can be optimized independently.","For each set of parameters to be tested, the column is eluted in a specified manner. For example, we may use a regime called Elution Regime 1: a KCl gradient runs from 10 mM to maximum allowed for the GP(IPBD) viability in 100 fractions of 0.05 V, followed by 20 fractions of 0.05 Vat maximum allowed KCl; pH of the buffer is maintained at the specified value with a convenient buffer such as phosphate, Tris, or MOPS. Other elution regimes can be used; what is important is that the conditions of this optimization be similar to the conditions that are used in Part III for selection for binding to target and recovery of GPs from the chromatographic system.","When the osp-ipbd gene is regulated by [XINDUCE], IPBD\/GP can be controlled by varying [XINDUCE]. Appropriate values of [XINDUCE] depend on the identity of [XINDUCE] and the promoter; if, for example, XINDUCE is isopropylthiogalactoside (IPTG) and the promoter is lacUV5, then [IPTG]=0, 0.1 uM, 1.0 uM, 10.0 uM, 100.0 uM, and 1.0 mM would be appropriate levels to test. The range of variation of [XINDUCE] is extended until an optimum is found or an acceptable level of expression is obtained.","DoAMoM is varied from the maximum that the matrix material can bind to 1% or 0.1% of this level in appro priate steps. We anticipate that the efficiency of separation will be a smooth function of DoAMoM so that it is appropriate to cover a wide range of values for DoAMoM with a coarse grid and then explore the neighborhood of the approximate optimum with a finer grid.","Several values of initial ionic strength are tested, such as 1.0 mM, 5.0 mM, 10.0 mM and 20.0 mM. Low ionic strength favors binding between oppositely charged groups, but could also cause GP to precipitate.","The elution rate is varied, by successive factors of \u00bd, from the maximum attainable rate to 1\/16 of this value. If the lowest elution rate tested gives the best separation, we test lower elution rates until we find an optimum or adequate separation.","The goal of the optimization is to obtain a sharp transition between bound and unbound GPs, triggered by increasing salt or decreasing pH or a combination of both. This optimization need be performed only: a) for each temperature to be used, b) for each pH to be used, and c) when a new GP(IPBD) is created.","V.I. Measuring the Sensitivity of Affinity Separation:","Once the values of IPBD\/GP, DoAMoM, initial ionic strength, elution rate, and amount of GP\/(volume of affinity support) have been optimized, we determine the sensitivity of the affinity separation (C) by the following procedure that measures the minimum quantity of GP(IPBD) that can be detected in the presence of a large excess of wtGP. The user chooses a number of separation cycles, denoted N, that will be performed before an enrichment is abandoned; preferably, Nis in the range 6 to 10 and Nmust be greater than 4. Enrichment can be terminated by isolation of a desired GP(SBD) before Npasses.","The measurement of sensitivity is significantly expedited if GP(IPBD) and wtGP carry different selectable markers because such markers allow easy identification of colonies obtained by plating fractions obtained from the chromatography column. For example, if wtGP carries kanamycin resistance and GP(IPBD) carries ampicillin resistance, we can plate fractions from a column on non-selective media suitable for the GP. Transfer of colonies onto ampicillin- or kanamycin-containing media will determine the identity of each colony.","Mixtures of GP(IPBD) and wtGP are prepared in the ratios of 1:V, where Vranges by an appropriate factor (e.g. 1\/10) over an appropriate range, typically 10through 10. Large values of Vare tested first; once a positive result is obtained for one value of V, no smaller values of Vneed be tested. Each mixture is applied to a column supporting, at the optimal DoAMoM, an AfM(IPBD) having high affinity for IPBD and the column is eluted by the specified elution regime, such as Elution Regime 1. The last fraction that contains viable GPs and an inoculum of the column matrix material are cultured. If GP(IPBD) and wtGP have different selectable markers, then transfer onto selection plates identifies each colony. If GP(IPBD) and wtGP have no selectable markers or the same selectable markers, then a number (e.g. 32) of GP clonal isolates are tested for presence of IPBD. If IPBD is not detected on the surface of any of the isolated GPs, then GPs are pooled from: a) the last few (e.g. 3 to 5) fractions that contain viable GPs, and b) an inoculum taken from the column matrix. The pooled GPs are cultured and passed over the same column and enriched for GP(IPBD) in the manner described. This process is repeated until Npasses have been performed, or until the IPBD has been detected on the GPs. If GP(IPBD) is not detected after Npasses, Vis decreased and the process is repeated.","Once a value for Vis found that allows recovery of GP(IPBD)s, the factor by which Vis varied is reduced and additional values are tested until Vis known to within a factor of two.","Cequals the highest value of Vfor which the user can recover GP(IPBD) within Npasses. The number of chromatographic cycles (K) that were needed to isolate GP(IPBD) gives a rough estimate of C; Cis approximately the Kth root of Vlim:\n\n\u2248exp{log()\/}\n","For example, if Vwere 4.0\u00d710and three separation cycles were needed to isolate GP(IPBD), then C\u2248736.","V.J. Measuring the Efficiency of Separation:","To determine Cmore accurately, we determine the ratio of GP(IPBD)\/wtGP loaded onto an AfM(IPBD) column that yields approximately equal amounts of GP(IPBD) and wtGP after elution. We prepare mixtures of GP(IPBD) and wtGP in ratios GP(IPBD):wtGP::1:Q; we start Q at twenty times the approximate Cfound above. A 1:Q mixture of GP(IPBD) and wtGP is applied to a AfM(IPBD) column and eluted by the specified elution regime, such as Elution Regime 1. A sample of the last fraction that contains viable GPs is plated at a dilution that gives well separated colonies or plaques. The presence of IPBD or the osp-ipbd gene in each colony or plaque can be determined by a number of standard methods, including: a) use of different selectable markers, b) nitrocellulose filter lift of GPs and detection with AfM(IPBD)* (AUSU87), or c) nitrocellulose filter lift of GPs and detection with radiolabeled DNA that is complementary to the osp-ipbd gene (AUSU87). Let F be the fraction of GP(IPBD) colonies found in the last fraction containing viable GPs. When a Q is found such that 0.20<F<0.80, then\n\n\n","If F<0.2, then we reduce Q by an appropriate factor (e.g. 1\/10) and repeat the procedure. If F>0.8, then we increase Q by an appropriate factor (e.g. 2) and repeat the procedure.","V.K. Reducing Selection Due to Non-Specific Binding:","When affinity chromatography is used for separating bound and unbound GPs, we may reduce non-specific binding of GP(PBD)s to the matrix that bears the target in the following ways:\n\n","Variation in the support material (polystyrene, glass, agarose, cellulose, etc.) in analysis of clones carrying SBDs is used to eliminate enrichment for packages that bind to the support material rather than the target.","FACs may be used to separate GPs that bind fluorescent labeled target. We discriminate against artifactual binding to the fluorescent label by using two or more different dyes, chosen to be structurally different. GPs isolated using target labeled with a first dye are cultured. These GPs are then tested with target labeled with a second dye.","Electrophoretic affinity separation uses unaltered target so that only other ions in the buffer can give rise to artifactual binding. Artifactual binding to the gel material gives rise to retardation independent of field direction and so is easily eliminated.","A variegated population of GPS will have a variety of charges. The following 2D electrophoretic procedure accommodates this variation in the population. First the variegated population of GPs is electrophoresed in a gel that contains no target material. The electrophoresis continues until the GP s are distributed along the length of the lane. The gels described by Sewer for phage are very low in agarose and lack mechanical stability. The target-free lane in which the initial electrophoresis is conducted is separate from a square of gel that contains target material by a removable baffle. After the first pass, the baffle is removed and a second electrophoresis is conducted at right angles to the first. GPs that do not bind target migrate with unaltered mobility while GP s that do bind target will separate from the majority that do not bind target. A diagonal line of non-binding GPs will form. This line is excised and discarded. Other parts of the gel are dissolved and the GPs cultured.","V.L. Isolation of GP(PBD)s with Binding-to-Target Phenotypes","The harvested packages are now enriched for the binding-to-target phenotype by use of affinity separation involving the target material immobilized on an affinity matrix. Packages that fail to bind to the target material are washed away. If the packages are bacteriophage or endospores, it may be desirable to include a bacteriocidal agent, such as azide, in the buffer to prevent bacterial growth. The buffers used in chromatography include: a) any ions or other solutes needed to stabilize the target, and b) any ions or other solutes needed to stabilize the PBDs derived from the IPBD.","V.M. Recovery of Packages:","Recovery of packages that display binding to an affinity column may be achieved in several ways, including:\n\n","Although degradation of the bound GPs and recovery of genetic material is a possible mode of operation, inadvertent inactivation of the GPs is very deleterious. It is preferred that maximum limits for solutes that do not inactivate the GPs or denature the target or the column are determined. If the affinity matrices are expendable, one may use conditions that denature the column to elute GPs; before the target is denatured, a portion of the affinity matrix should be removed for possible use as an inoculum. As the GPs are held together by protein-protein interactions and other non-covalent molecular interactions, there will be cases in which the molecular package will bind so tightly to the target molecules on the affinity matrix that the GPs can not be washed off in viable form. This will only occur when very tight binding has been obtained. In these cases, methods (3) through (5) above can be used to obtain the bound packages or the genetic messages from the affinity matrix.","It is possible, by manipulation of the elution conditions, to isolate SBDs that bind to the target at one pH (pH) but not at another pH (pH). The population is applied at pHand the column is washed thoroughly at pH. The column is then eluted with buffer at pHand GPs that come off at the new pH are collected and cultured. Similar procedures may be used for other solution parameters, such as temperature. For example, GP(vgPBD)s could be applied to a column supporting insulin. After eluting with salt to remove GPs with little or no binding to insulin, we elute with salt and glucose to liberate GPs that display PBDs that bind insulin or glucose in a competitive manner.","V.N. Amplifying the Enriched Packages","Viable GPs having the selected binding trait are amplified by culture in a suitable medium, or, in the case of phage, infection into a host so cultivated. If the GPs have been inactivated by the chromatography, the OCV carrying the osp-pbd gene are recovered from the GP, and introduced into a new, viable host.","V.O. Determining whether Further Enrichment is Needed:","The probability of isolating a GP with improved binding increases by Cwith each separation cycle. Let N be the number of distinct amino-acid sequences produced by the variegation. We want to perform K separation cycles before attempting to isolate an SBD, where K is such that the probability of isolating a single SBD is 0.10 or higher.\n\n=the smallest integer>=log(0.10 )\/log()\n","For example, if N were 1.0\u00b710and C=6.31\u00b710, then log(1.0\u00b710)\/log(6.31\u00b710)=6.0000\/2.8000=2.14. Therefore we would attempt to isolate SBDs after the third separation cycle. After only two separation cycles, the probability of finding an SBD is\n\n(6.31\u00d7102)2\/(1.0\u00d7107)=0.04\n\nand attempting to isolate SBDs might be profitable.\n","Clonal isolates from the last fraction eluted which contained any viable GPs, as well as clonal isolates obtained by culturing an inoculum taken from the affinity matrix, are cultured in a growth step that is similar to that described previously. Other fractions may be cultured too. If K separation cycles have been completed, samples from a number, e.g. 32, of these clonal isolates are tested for elution properties on the {target} column. If none of the isolated, genetically pure GPs show improved binding to target, or if K cycles have not yet been completed, then we pool and culture, in a manner similar to the manner set forth previously, the GPs from the last few fractions eluted that contained viable GPs and from the GPs obtained by culturing an inoculum taken from the column matrix. We then repeat the enrichment procedure described above. This cyclic enrichment may continue Npasses or until an SBD is isolated.","If one or more of the isolated GPs has improved retention on the {target} column, we determine whether the retention of the candidate SBDs is due to affinity for the target material as follows. A second column is prepared using a different support matrix <image>","<\/image>material bound at the optimal density. The elution volumes, under the same elution conditions as used previously, of candidate GP(SBD)s are compared to each other and to GP(PPBD of this round). If one or more candidate GP(SBD)s has a larger elution volume than GP(PPBD of this round), then we pick the GP(SBD) having the highest elution volume and proceed to characterize the population. If none of the candidate GP(SBD)s has higher elution volume than GP(PPBD of this round), then we pool and culture, in a manner similar to the manner used previously, the GPs from the last few fractions that contained viable GPs and the GPs obtained by culturing an inoculum taken from the column matrix. We then repeat the enrichment procedure.","If all of the SBDs show binding that is superior to PPBD of this round, we pool and culture the GPs from the last fraction that contains viable GPs and from the inoculum taken from the column. This population is re-chromatographed at least one pass to fractionate further the GPs based on K.","If an RNA phage were used as GP, the RNA would either be cultured with the assistance of a helper phage or be reverse transcribed and the DNA amplified. The amplified DNA could then be sequenced or subcloned into suitable plasmids.","V.P. Characterizing the Putative SBDs:","We characterize members of the population showing desired binding properties by genetic and biochemical methods. We obtain clonal isolates and test these strains by genetic and affinity methods to determine genotype and phenotype with respect to binding to target. For several genetically pure isolates that show binding, we demonstrate that the binding is caused by the artificial chimeric gene by excising the osp-sbd gene and crossing it into the parental GP. We also ligate the deleted backbone of each GP from which the osp-sbd is removed and demonstrate that each backbone alone cannot confer binding to the target on the GP. We sequence the osp-sbd gene from several clonal isolates. Primers for sequencing are chosen from the DNA flanking the osp-ppbd gene or from parts of the osp-ppbd gene that are not variegated.","The present invention is not limited to a single method of determining protein sequences, and reference in the appended claims to determining the amino acid sequence of a domain is intended to include any practical method or combination of methods, whether direct or indirect. The preferred method, in most cases, is to determine the sequence of the DNA that encodes the protein and then to infer the amino acid sequence. In some cases, standard methods of protein-sequence determination may be needed to detect post-translational processing.","The present invention is not limited to a single method of determining the sequence of nucleotides (nts) in DNA subsequences. In the preferred embodiment, plasmids are isolated and denatured in the presence of a sequencing primer, about 20 nts long, that anneals to a region adjacent, on the 5\u2032 side, to the region of interest. This plasmid is then used as the template in the four sequencing reactions with one dideoxy substrate in each. Sequencing reactions, agarose gel electrophoresis, and polyacrylamide gel electrophoresis (PAGE) are performed by standard procedures (AUSU87).","For one or more clonal isolates, we may subclone the sbd gene fragment, without the osp fragment, into an expression vector such that each SBD can be produced as a free protein. Because numerous unique restriction sites were built into the inserted domain, it is easy to subclone the gene at any time. Each SBD protein is purified by normal means, including affinity chromatography. Physical measurements of the strength of binding are then made on each free SBD protein by one of the following methods: 1) alteration of the Stokes radius as a function of binding of the target material, measured by characteristics of elution from a molecular sizing column such as agarose, 2) retention of radiolabeled binding protein on a spun affinity column to which has been affixed the target material, or 3) retention of radiolabeled target material on a spun affinity column to which has been affixed the binding protein. The measurements of binding for each free SBD are compared to the corresponding measurements of binding for the PPBD.","In each assay, we measure the extent of binding as a function of concentration of each protein, and other relevant physical and chemical parameters such as salt concentration, temperature, pH, and prosthetic group concentrations (if any).","In addition, the SBD with highest affinity for the target from each round is compared to the best SBD of the previous round (IPBD for the first round) and to the IPBD (second and later rounds) with respect to affinity for the target material. Successive rounds of mutagenesis and selection-through-binding yield increasing affinity until desired levels are achieved.","If we find that the binding is not yet sufficient, we decide which residues to vary next. If the binding is sufficient, then we now have a expression vector bearing a gene encoding the desired novel binding protein.","V.Q. Joint Selections:","One may modify the affinity separation of the method described to select a molecule that binds to material A but not to material B. One needs to prepare two selection columns, one with material A and the other with material B. The population of genetic packages is prepared in the manner described, but before applying the population to A, one passes the population over the B column so as to remove those members of the population that have high affinity for B (\u201creverse affinity chromatography\u201d). In the preceding specification, the initial column supported some other molecule simply to remove GP(PBD)s that displayed PBDs having indiscriminate affinity for surfaces.","It may be necessary to amplify the population that does not bind to B before passing it over A. Amplification would most likely be needed if A and B were in some ways similar and the PPBD has been selected for having affinity for A. The optimum order of interactions might be determined empirically. For example, to obtain an SBD that binds A but not B, three columns could be connected in series: a) a column supporting some compound, neither A nor B, or only the matrix material, b) a column supporting B, and c) a column supporting A. A population of GP(vgPBD)s is applied to the series of columns and the columns are washed with the buffer of constant ionic strength that is used in the application. The columns are uncoupled, and the third column is eluted with a gradient to isolate GP(PBD)s that bind A but not B.","One can also generate molecules that bind to both A and B. In this case we can use a 3D model and mutate one face of the molecule in question to get binding to A. One can then mutate a different face to produce binding to B. When an SBD binds at least somewhat to both A and B, one can mutate the chain by Diffuse Mutagenesis to refine the binding and use a sequential joint selection for binding to both A and B.","The materials A and B could be proteins that differ at only one or a few residues. For example, A could be a natural protein for which the gene has been cloned and B could be a mutant of A that retains the overall 3D structure of A. SBDs selected to bind A but not B probably bind to A near the residues that are mutated in B. If the mutations were picked to be in the active site of A (assuming A has an active site), then an SBD that binds A but not B will bind to the active site of A and is likely to be an inhibitor of A.","To obtain a protein that will bind to both A and B, we can, alternatively, first obtain an SBD that binds A and a different SBD that binds B. We can then combine the genes encoding these domains so that a two-domain single-polypeptide protein is produced. The fusion protein will have affinity for both A and B because one of its domains binds A and the other binds B.","One can also generate binding proteins with affinity for both A and B, such that these materials will compete for the same site on the binding protein. We guarantee competition by overlapping the sites for A and B. Using the procedures of the present invention, we first create a molecule that binds to target material A. We then vary a set of residues defined as: a) those residues that were varied to obtain binding to A, plus b) those residues close in 3D space to the residues of set (a) but that are internal and so are unlikely to bind directly to either A or B. Residues in set (b) are likely to make small changes in the positioning of the residues in set (a) such that the affinities for A and B will be changed by small amounts. Members of these populations are selected for affinity to both A and B.","V.R. Selection for Non-Binding:","The method of the present invention can be used to select proteins that do not bind to selected targets. Consider a protein of pharmacological importance, such as streptokinase, that is antigenic to an undesirable extent. We can take the pharmacologically important protein as IPBD and antibodies against it as target. Residues on the surface of the pharmacologically important protein would be variegated and GP(PBD)s that do not bind to an antibody column would be collected and cultured. Surface residues may be identified in several ways, including: a) from a 3D structure, b) from hydrophobicity considerations, or c) chemical labeling. The 3D structure of the pharmacologically important protein remains the preferred guide to picking residues to vary, except now we pick residues that are widely spaced so that we leave as little as possible of the original surface unaltered.","Destroying binding frequently requires only that a single amino acid in the binding interface be changed. If polyclonal antibodies are used, we face the problem that all or most of the strong epitopes must be altered in a single molecule. Preferably, one would have a set of monoclonal antibodies, or a narrow range of antibody species. If we had a series of monoclonal antibody columns, we could obtain one or more mutations that abolish binding to each monoclonal antibody. We could then combine some or all of these mutations in one molecule to produce a pharmacologically important protein recognized by none of the monoclonal antibodies. Such mutants are tested to verify that the pharmacologically interesting proper ties have not be altered to an unacceptable degree by the mutations.","Typically, polyclonal antibodies display a range of binding constants for antigen. Even if we have only polyclonal antibodies that bind to the pharmacologically important protein, we may proceed as follows. We engineer the pharmacologically important protein to appear on the surface of a replicable GP. We introduce mutations into residues that are on the surface of the pharmacologically important protein or into residues thought to be on the surface of the pharmacologically important protein so that a population of GPs is obtained. Polyclonal antibodies are attached to a column and the population of GPs is applied to the column at low salt. The column is eluted with a salt gradient. The GPs that elute at the lowest concentration of salt are those which bear pharmacologically important proteins that have been mutated in a way that eliminates binding to the antibodies having maximum affinity for the pharmacologically important protein. The GPs eluting at the lowest salt are isolated and cultured. The isolated SBD becomes the PPBD to further rounds of variegation so that the antigenic determinants are successively eliminated.","V.S. Selection of PBDs for Retention of Structure:","Let us take an SBD with known affinity for a target as PPBD to a variegation of a region of the PBD that is far from the residues that were varied to create the SBD. We can use the target as an affinity molecule to select the PBDs that retain binding for the target, and that presumably retain the underlying structure of the IPBD. The variegations in this case could include insertions and deletions that are likely to disrupt the IPBD structure. We could also use the IPBD and AfM(IPBD) in the same way.","For example, if IPBD were BPTI and AfM(BPTI) were trypsin, we could introduce four or five additional residue after residue 26 and select GPs that display PBDs having specific affinity for AfM(BPTI). Residue 26 is chosen because it is in a turn and because it is about 25 A from K15, a key amino acid in binding to trypsin.","The underlying structure is most likely to be retained if insertions or deletions are made at loops or turns.","V.T. Engineering of Antagonists","It may be desirable to provide an antagonist to an enzyme or receptor. This may be achieved by making a molecule that prevents the natural substrate or agonist from reaching the active site. Molecules that bind directly to the active site may be either agonists or antagonists. Thus we adopt the following strategy. We consider enzymes and receptors together under the designation TER (Target Enzyme or Receptor).","For most TERs, there exist chemical inhibitors that block the active site. Usually, these chemicals are useful only as research tools due to highly toxicity. We make two affinity matrices: one with active TER and one with blocked TER. We make a variegated population of GP(PBD)s and select for SBPs that bind to both forms of the enzyme, thereby obtaining SDPs that do not bind to the active site. We expect that SBDs will be found that bind different places on the enzyme surface. Pairs of the sbd genes are fused with an intervening peptide segment. For example, if SBD-1 and SBD-2 are binding domains that show high affinity for the target enzyme and for which the binding is non-competitive, then the gene sbd-1::linker::sbd-2 encodes a two-domain protein that will show high affinity for the target. We make several fusions having a variety of SBDs and various linkers. Such compounds have a reasonable probability of being an antagonist to the target enzyme.","VI. Exploitation of Successful Binding Domains and Corresponding DNAs","VI.A. Generally","Using the method of the present invention, we can obtain a replicable genetic package that displays a novel protein domain having high affinity and specificity for a target material of interest. Such a package carries both amino-acid embodiments of the binding protein domain and a DNA embodiment of the gene encoding the novel binding domain. The presence of the DNA facilitates expression of a protein comprising the novel binding protein domain within a high-level expression system, which need not be the same system used during the developmental process.","VI.B. Production of Novel Binding Proteins","We can proceed to production of the novel binding protein in several ways, including: a) altering of the gene encoding the binding domain so that the binding domain is expressed as a soluble protein, not attached to a genetic package (either by deleting codons 5\u2032 of those encoding the binding domain or by inserting stop codons 3\u2032 of those encoding the binding domain), b) moving the DNA encoding the binding domain into a known expression system, and c) utilizing the genetic package as a purification system. (If the domain is small enough, it may be feasible to prepare it by conventional peptide synthesis methods.)","Option (c) may be illustrated as follows. Assume that a novel BPTI derivative has been obtained by selection of M13 derivatives in which a population of BPTI-derived domains are displayed as fusions to mature coat protein. Assume that a specific protease cleavage site (e.g. that of activated clotting factor X) is engineered into the amino-acid sequence between the carboxy terminus of the BPTI-derived domain and the mature coat domain. Furthermore, we alter the display system to maximize the number of fusion proteins displayed on each phage. The desired phage can be produced and purified, for example by centrifugation, so that no bacterial products remain. Treatment of the purified phage with a catalytic amount of factor X cleaves the binding domains from the phage particles. A second centrifugation step separates the cleaved protein from the phage, leaving a very pure protein preparation.","VI.C. Mini-Protein Production","As previously mentioned, an advantage inhering from the use of a mini-protein as an IPBD is that it is likely that the derived SBD will also behave like a mini-protein and will be obtainable by means of chemical synthesis. (The term \u201cchemical synthesis\u201d, as used herein, includes the use of enzymatic agents in a cell-free environment.)","It is also to be understood that mini-proteins obtained by the method of the present invention may be taken as lead compounds for a series of homologues that contain non-naturally occurring amino acids and groups other than amino acids. For example, one could synthesize a series of homologues in which each member of the series has one amino acid replaced by its D enantiomer. One could also make homologues containing constituents such as \u03b2 alanine, aminobutyric acid, 3-hydroxyproline, 2-Aminoadipic acid, N-ethylasperagine, norvaline, etc.; these would be tested for binding and other properties of interest, such as stability and toxicity.","Peptides may be chemically synthesized either in solution or on supports. Various combinations of stepwise synthesis and fragment condensation may be employed.","During synthesis, the amino acid side chains are protected to prevent branching. Several different protective groups are useful for the protection of the thiol groups of cysteines:\n\n","Other thiol protective groups may be found in standard reference works such as Greene, PROTECTIVE GROUPS IN ORGANIC SYNTHESIS (1981).","Once the polypeptide chain has been synthesized, disulfide bonds must be formed. Possible oxidizing agents include air (HOUG84; NISH86), ferricyanide (NISH82; HOUG84), iodine (NISH82), and performic acid (HOUG84). Temperature, pH, solvent, and chaotropic chemicals may affect the course of the oxidation.","A large number of mini-proteins with a plurality of disulfide bonds have been chemically synthesized in biologically active form: conotoxin G1 (13AA, 4 Cys) (NISH82); heat-stable enterotoxin ST (18AA, 6 Cys) (HOUG84); analogues of ST (BHAT86); \u03a9-conotoxin GVIA (27AA, 6Cys) (NISH86; RIVI87b); \u03a9-conotoxin MVIIA (27 AA, 6 Cys) (OLIV87b); \u03b1-conotoxin SI (13 AA, 4 Cys) (ZAFA88); \u03bc-conotoxin IIIa (22AA, 6 Cys) (BECK89c, CRUZ89, HATA90). Sometimes, the polypeptide naturally folds so that the correct disulfide bonds are formed. Other times, it must be helped along by use of a differently removable protective group for each pair of cysteines.","VI.D. Uses of Novel Binding Proteins","The successful binding domains of the present invention may, alone or as part of a larger protein, be used for any purpose for which binding proteins are suited, including isolation or detection of target materials. In furtherance of this purpose, the novel binding proteins may be coupled directly or indirectly, covalently or noncovalently, to a label, carrier or support.","When used as a pharmaceutical, the novel binding proteins may be contained with suitable carriers or adjuvanants.","All references cited anywhere in this specification are incorporated by reference to the extent which they may be pertinent.","Display of BPTI as a Fusion to M13 Gene VIII Protein:","Example I involves display of BPTI on M13 as a fusion to the mature gene VIII coat protein. Each of the DNA constructions was confirmed by restriction digestion analysis and DNA sequencing.","1. Construction of the VIII-Signal-Sequence::BPTI::Mature-VIII-Coat-Protein Display Vector.","A. Operative Cloning Vectors (OCV).","The operative cloning vectors are M13 and phagemids derived from M13 or f1. The initial construction was in the f1-based phagemid pGEM-3Zf(\u2212)\u2122 (Promega Corp., Madison, Wisc.).","A gene comprising, in order,: i) a modified lacUV5 promoter, ii) a Shine-Dalgarno sequence, iii) DNA encoding the M13 gene VIII signal sequence, iv) a sequence encoding mature BPTI, v) a sequence encoding the mature-M13-gene-VIII coat protein, vi) multiple stop codons, and vii) a transcription terminator, was constructed. This gene is illustrated in Tables 101-105; each table shows the same DNA sequence with different features annotated. There are a number of differences between this gene and the one proposed in the hypothetical example in the generic specification of the parent application. Because the actual construction was made in pGEM-3Zf(\u2212), the ends of the synthetic DNA were made compatible with SalI and BamHI. The lacO operator of lacUV5 was changed to the symmetrical lacO with the intention of achieving tighter repression in the absence of IPTG. Several silent codon changes were made so that the longest segment that is identical to wild-type gene VIII is minimized so that genetic recombination with the co-existing gene VIII is unlikely.","i) OCV Based Upon pGEM-3Zf.","pGEM-3Zf\u2122 (Promega Corp., Madison, Wisc.) is a plasmid-based vector containing the amp gene, bacterial origin of replication, bacteriophage f1 origin of replication, a lacZ operon containing a multiple cloning site sequence, and the T7 and SP6 polymerase binding sequences.","Two restriction enzyme recognition sites were introduced, by site-directed oligonucleotide mutagenesis, at the boundaries of the lacZ operon. This allowed for the removal of the lacZ operon and its replacement with the synthetic gene. A BamHI recognition site (GGATCC) was introduced at the 5\u2032 end of the lacZ operon by the mutation of bases Cand Tto G and A respectively (numbering of Promega). A SalI recognition site (GTCGAC) was introduced at the 3\u2032 end of the operon by the mutation of bases Cand Tto G and C respectively. A construct combining these variants of PGEM-3Zf was designated pGEM-MB3\/4.","ii) OCV Based Upon M13mp18.","M13mp18 (YANI85) is an M13 bacteriophage-based vector (available from, inter alia, New England Biolabs, Beverly, Mass.) consisting of the whole of the phage genome into which has been inserted a lacZ operon containing a multiple cloning site sequence (MESS77). Two restriction enzyme sites were introduced into M13mp18 using standard methods. A BamHI recognition site (GGATCC) was introduced at the 5\u2032 end of the lacZ operon by the mutation of bases Cand Gto A and T respectively (numbering of Messing). This mutation also destroyed a unique NarI site. A SalI recognition site (GTCGAC) was introduced at the 3\u2032 end of the operon by the mutation of bases Aand C6432 to C and A respectively. A construct combining these variants of M13mp18 was designated M13-MB1\/2.","B) Synthetic Gene.","A synthetic gene (VIII-signal-sequence::mature-bpti::mature-VIII-coat-protein) was constructed from 16 synthetic oligonucleotides (Table 105), custom synthesized by Genetic Designs Inc. of Houston, Tex., using methods detailed in KIMH89 and ASHM89. Table 101 shows the DNA sequence; Table 102 contains an annotated version of this sequence. Table 103 shows the overlaps of the synthetic oligonucleotides in relationship to the restriction sites and coding sequence. Table 104 shows the synthetic DNA in double-stranded form. Table 105 shows each of the 16 synthetic oligonucleotides from 5\u2032-to-3\u2032. The oligonucleotides were phosphorylated, with the exception of the 5\u2032 most molecules, using standard methods, annealed and ligated in stages such that a final synthetic duplex was generated. The overhanging ends of this duplex was filled in with T4 DNA polymerase and it was cloned into the HincII site of pGEM-3Zf(\u2212); the initial construct is called pGEM-MB1 (Table 101a). Double-stranded DNA of pGEM-MB1 was cut with PstI, filled in with T4 DNA polymerase and ligated to a SalI linker (New England BioLabs) so that the synthetic gene is bounded by BamHI and SalI sites (Table 101b and Table 102b). The synthetic gene was obtained on a BamHI-SalI cassette and cloned into pGEM-MB3\/4 and M13-MB1\/2 utilizing the BamHI and SalI sites previously introduced, to generate the constructs designated pGEM-MB16 and M13-MB15, respectively. The full length of the synthetic insert was sequenced and found to be unambiguously correct except for: 1) a missing G in the Shine-Dalgarno sequence; and 2) a few silent errors in the third bases of some codons (shown as upper case in Table 101). Table 102 shows the Ribosome-binding site AGGAGG but the actual sequence is AGAGG. Efforts to express protein from this construction, in vivo and in vitro, were unavailing.","C) Alterations to the Synthetic Gene.","i) Ribosome Binding Site (RBS).","Starting with the construct pGEM-MB16, a fragment of DNA bounded by the restriction enzyme sites SacI and NheI (containing the original RBS) was replaced with a synthetic oligonucleotide duplex (with compatible SacI and NheI overhangs) containing the sequence for a new RBS that is very similar to the RBS of phoA and that has been shown to be functional.","Original putative RBS (5\u2032-to-3\u2032)",{"@attributes":{"id":"p-0588","num":"0864"},"chemistry":{"@attributes":{"id":"CHEM-US-00008","num":"00008"},"img":{"@attributes":{"id":"EMI-C00008","he":"8.47mm","wi":"73.58mm","file":"US07413537-20080819-C00008.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{}},{"@attributes":{"id":"p-0589","num":"0865"},"chemistry":{"@attributes":{"id":"CHEM-US-00009","num":"00009"},"img":{"@attributes":{"id":"EMI-C00009","he":"8.47mm","wi":"73.66mm","file":"US07413537-20080819-C00009.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":[{},{}]},"In order to obtain higher expression levels of the fusion protein, the lacUV5 promoter was changed to a tac promoter. Starting with the construct pGEM-MB16, which contains the lacUV5 promoter, a fragment of DNA bounded by the restriction enzyme sites BamHI and HpaII was excised and replaced with a compatible synthetic oligonucleotide duplex containing the \u221235 sequence of the trp promoter, Cf RUSS82. This converted the lacUV5 promoter to a tac promoter in a construct designated pGEM-MB22, Table 112.","MB16",{"@attributes":{"id":"p-0591","num":"0867"},"chemistry":{"@attributes":{"id":"CHEM-US-00010","num":"00010"},"img":{"@attributes":{"id":"EMI-C00010","he":"21.93mm","wi":"75.86mm","file":"US07413537-20080819-C00010.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{}},{"@attributes":{"id":"p-0592","num":"0868"},"chemistry":{"@attributes":{"id":"CHEM-US-00011","num":"00011"},"img":{"@attributes":{"id":"EMI-C00011","he":"13.89mm","wi":"74.17mm","file":"US07413537-20080819-C00011.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"Promoter and RBS variants of the fusion protein gene were constructed by basic DNA manipulation techniques to generate the following:",{"@attributes":{"id":"p-0594","num":"0870"},"tables":{"@attributes":{"id":"TABLE-US-00014","num":"00014"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"left"}}],"thead":{"row":[{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"Promoter","RBS","Encoded Protein."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["pGEM-MB16","lac","old","VIIIs.p.-BPTI-matureVIII"]},{"entry":["pGEM-MB20","lac","new","\u2033"]},{"entry":["pGEM-MB22","tac","old","\u2033"]},{"entry":["pGEM-MB26","tac","new","\u2033"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},"The synthetic gene from variants pGEM-MB20 and pGEM-MB26 were recloned into the altered phage vector M13-MB1\/2 to generate the phage constructs designated M13-MB27 and M13-MB28 respectively.","iii. Signal Peptide Sequence.","In vitro expression of the synthetic gene regulated by tac and the \u201cnew\u201d RBS produced a novel protein of the expected size for the unprocessed protein (about 16 kd). In vivo expression also produced novel protein of full size; no processed protein could be seen on phage or in cell extracts by silver staining or by Western analysis with anti-BPTI antibody.","Thus we analyzed the signal sequence of the fusion. Table 106 shows a number of typical signal sequences. Charged residues are generally thought to be of great importance and are shown bold and underscored. Each signal sequence contains a long stretch of uncharged residues that are mostly hydrophobic; these are shown in lower case. At the right, in parentheses, is the length of the stretch of uncharged residues. We note that the fusions of gene VIII signal to BPTI and gene III signal to BPTI have rather short uncharged segments. These short uncharged segments may reduce or prevent processing of the fusion peptides. We know that the gene III signal sequence is capable of directing: a) insertion of the peptide comprising (mature-BPTI)::(mature-gene-III-protein) into the lipid bilayer, and b) translocation of BPTI and most of the mature gene III protein across the lipid bilayer (vide infra). That the gene III remains anchored in the lipid bilayer until the phage is assembled is directed by the uncharged anchor region near the carboxy terminus of the mature gene III protein (see Table 116) and not by the secretion signal sequence. The phoA signal sequence can direct secretion of mature BPTI into the periplasm of (MARK86). Furthermore, there is controversy over the mechanism by which mature authentic gene VIII protein comes to be in the lipid bilayer prior to phage assembly.","Thus we decided to replace the DNA coding on expression for the gene-VIII-putative-signal-sequence by each of: 1) DNA coding on expression for the phoA signal sequence, 2) DNA coding on expression for the bla signal sequence, or 3) DNA coding on expression for the M13 gene III signal. Each of these replacements produces a tripartite gene encoding a fusion protein that comprises, in order: (a) a signal peptide that directs secretion into the periplasm of parts (b) and (c), derived from a first gene; (b) an initial potential binding domain (BPTI in this case), derived from a second gene (in this case, the second gene is an animal gene); and (c) a structural packaging signal (the mature gene VIII coat protein), derived from a third gene.","The process by which the IPBD::packaging-signal fusion arrives on the phage surface is illustrated in . In , we see that authentic gene VIII protein appears (by whatever process) in the lipid bilayer so that both the amino and carboxy termini are in the cytoplasm. Signal peptidase-I cleaves the gene VIII protein liberating the signal peptide (that is absorbed by the cell) and mature gene VIII coat protein that spans the lipid bilayer. Many copies of mature gene VIII coat protein accumulate in the lipid bilayer awaiting phage assembly (). Some signal sequences are able to direct the translocation of quite large proteins across the lipid bilayer. If additional codons are inserted after the codons that encode the cleavage site of the signal peptidase-I of such a potent signal sequence, the encoded amino acids will be translocated across the lipid bilayer as shown in . After cleavage by signal peptidase-I, the amino acids encoded by the added codons will be in the periplasm but anchored to the lipid bilayer by the mature gene VIII coat protein, . The circular single-stranded phage DNA is extruded through a part of the lipid bilayer containing a high concentration of mature gene VIII coat protein; the carboxy terminus of each coat protein molecule packs near the DNA while the amino terminus packs on the outside. Because the fusion protein is identical to mature gene VIII coat protein within the trans-bilayer domain, the fusion protein will co-assemble with authentic mature gene VIII coat protein as shown in ","In each case, the mature VIII coat protein moiety is intended to co-assemble with authentic mature VIII coat protein to produce phage particle having BPTI domains displayed on the surface. The source and character of the secretion signal sequence is not important because the signal sequence is cut away and degraded. The structural packaging signal, however, is quite important because it must co-assemble with the authentic coat protein to make a working virus sheath.","a) Bacterial Alkaline Phosphatase (phoA) Signal Peptide.","Construct pGEM-MB26 contains a fragment of DNA bounded by restriction enzyme sites SacI and AccIII which contains the new RBS and sequences encoding the initiating methionine and the signal peptide of M13 gene VIII pro-protein. This fragment was replaced with a synthetic duplex (constructed from four annealed oligonucleotides) containing the RBS and DNA coding for the initiating methionine and signal peptide of PhoA (INOU82). The resulting construct was designated pGEM-MB42; the sequence of the fusion gene is shown in Table 113. M13MB48 is a derivative of GemMB42. A BamHI-SalI DNA fragment from GenMB42, containing the gene construct, was ligated into a similarly cleaved vector M13MB1\/2 giving rise to M13MB48. PhoA RBS and signal peptide sequence",{"@attributes":{"id":"p-0602","num":"0878"},"chemistry":{"@attributes":{"id":"CHEM-US-00012","num":"00012"},"img":{"@attributes":{"id":"EMI-C00012","he":"29.38mm","wi":"75.44mm","file":"US07413537-20080819-C00012.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{}},"To enable the introduction of the beta-lactamase (amp) promoter and DNA coding for the signal peptide into the gene encoding (mature-BPTI)::(mature-VIII-coat-protein) an initial manipulation of the amp gene (encoding beta-lactamase) was required. Starting with pGEM-3Zf an AccIII recognition site (TCCGGA) was introduced into the amp gene adjacent to the DNA sequence encoding the amino acids at the beta-lactamase signal peptide cleavage site. Using standard methods of in vitro site-directed oligonucleotide mutagenesis bases Cand Awere converted to T and G respectively to generate the construct designated pGEM-MB40. Further manipulation of PGEM-MB40 entailed the insertion of a synthetic oligonucleotide linker (CGGATCCG) containing the BamHI recognition sequence (GGATCC) into the AatII site (GACGTC starting at nucleotide number 2260) to generate the construct designated pGEM-MB45. The DNA bounded by the restriction enzyme sites of BamHI and AccIII contains the amp promoter, amp RBS, initiating methionine and beta-lactamase signal peptide. This fragment was used to replace the corresponding fragment from pGEM-MB26 to generate construct pGEM-MB46.","amp gene promoter and signal peptide sequences",{"@attributes":{"id":"p-0605","num":"0881"},"tables":{"@attributes":{"id":"TABLE-US-00015","num":"00015"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"231pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"140pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["5\u2032-GGATCCGGTGGCACTTTTCGGGGAAATGTGCGCGGAACCCCTATTTGTT-","(SEQ ID NO:145)",{}]},{"entry":{}},{"entry":"TATTTTTCTAAATACATTCAAATATGTATCCGCTCATGAGACAATAACC-"},{"entry":{}},{"entry":"CTGATAAATGCTTCAATAATATTGAAAAAGGAAGAGT-"},{"entry":{}},{"entry":"ATG.AGT.ATT.CAA.CAT.TTC.CGT.GTC.GCC.CTT.ATT.-"},{"entry":["met ser ile gln his phe arg val ala leu ile","(SEQ ID NO:146)"]},{"entry":{}},{"entry":"CCC.TTT.TTT.GCG.GCA.TTT.TGC.CTT.CCT.GTT.TTT.-"},{"entry":"pro phe phe ala ala phe cys leu pro val phe"},{"entry":{}},{"entry":"GCT.CAT.CCG.-3\u2032"},{"entry":"ala his pro . . ."}]}}}},"br":{}},"We may also construct, as depicted in , M13-MB51 which would carry a gene encoding a fusion of M13-gene-III-signal-peptide to the previously described BPTI::mature VIII coat protein. First the BstEII site that follows the stop codons of the synthetic gene VIII is changed to an AlwNI site as follows. DNA of pGEM-MB26 is cut with BstEII and the ends filled in by use of Klenow enzyme; a blunt AlwNI linker is ligated to this DNA. This construction is called pGEM-MB26Alw. The XhoI to AlwNI fragment (approximately 300 bp) of pGEM-MB26Alw is purified. RF DNA from phage MK-BPTI (vide infra) is cut with AlwNI and XhoI and the large fragment purified. These two fragments are ligated together; the resulting construction is named M13-MB51. Because M13-MB51 contains no gene III, the phage can not form plaques. M13-MB51 can, however, render cells Km. Infectious phage particles can be obtained by use of helper phage. As explained below, the gene III signal sequence is capable of directing (BPTI)::(mature-gene-III-protein) to the surface of phage. In M13-MB51, we have inserted DNA encoding gene VIII coat protein (50 amino acids) and three stop codons 5\u2032 to the DNA encoding the mature gene III protein.","Summary of signal peptide fusion protein variants.",{"@attributes":{"id":"p-0608","num":"0884"},"tables":{"@attributes":{"id":"TABLE-US-00016","num":"00016"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"left"}}],"thead":{"row":[{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]},{"entry":[{},{},{},"Signal","Fusion"]},{"entry":[{},"Promoter","RBS","sequence","protein"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["pGEM-MB26",{"u":"tac"},"new",{"u":"VIII"},"BPTI\/VIII-"]},{"entry":[{},{},{},{},"coat"]},{"entry":["pGEM-MB42",{"u":"tac"},"new",{"u":"phoA"},"BPTI\/VIII-"]},{"entry":[{},{},{},{},"coat"]},{"entry":["pGEM-MB46",{"u":"amp"},{"u":"amp"},{"u":"amp"},"BPTI\/VIII-"]},{"entry":[{},{},{},{},"coat"]},{"entry":["pGEM-MB51",{"u":"III"},{"u":"III"},{"u":"III"},"BPTI\/VIII-"]},{"entry":[{},{},{},{},"coat"]},{"entry":["M13 MB48",{"u":"tac"},"new",{"u":"phoA"},"BPTI\/VIII-"]},{"entry":[{},{},{},{},"coat"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}},"br":[{},{}]},"A coupled transcription\/translation prokaryotic system (Amersham Corp., Arlington Heights, Ill.) was utilized for the in vitro analysis of the protein products encoded by the BPTI\/VIII synthetic gene and the variants derived from this.","Table 107 lists the protein products encoded by the listed vectors which are visualized by the standard method of fluorography following in vitro synthesis in the presence of S-methionine and separation of the products using SDS polyacrylamide gel electrophoresis. In each sample a pre-beta-lactamase product (approximately 31 kd) can be seen. This is derived from the amp gene which is the common selection gene for each of the vectors. In addition, a (pre-BPTI\/VIII) product encoded by the synthetic gene and variants can be seen as indicated. The migration of these species (approximately 14.5 kd) is consistent with the expected size of the encoded proteins.","ii) In Vivo Analysis.","The vectors detailed in sections (B) and (C) were freshly transfected into the strain XL1-blue\u2122 (Stratagene, La Jolla, Calif.) and in strain SEF\u2032. strain SE6004 (LISS85) carries the prlA4 mutation and is more permissive in secretion than strains that carry the wild-type prlA allele. SE6004 is F and is deleted for lacI; thus the cells can not be infected by M13 and lacUV5 and tac promoters can not be regulated with IPTG. Strain SEF\u2032 is derived from strain SE6004 (LISS85) by crossing with XL1-Blue\u2122; the F\u2032 in XL1-Blue\u2122 carries Tcand lacI. SE6004 is streptomycin, Tcwhile XL1-Blue\u2122 is streptomycin, Tcso that both parental strains can be killed with the combination of Tc and streptomycin. SEF\u2032 retains the secretion-permissive phenotype of the parental strain, SE6004(prlA4).","The fresh transfectants were grown in NZYCM medium (SAMB89) for 1 hour after which IPTG was added over the range of concentrations 1.0 \u03bcM to 0.5 mM (to derepress the lacUV5 and tac promoters) and grown for an additional 1.5 hours.","Aliquots of the bacterial cells expressing the synthetic insert encoded proteins together with the appropriate controls (no vector, vector with no insert and zero IPTG) were lysed in SDS gel loading buffer and electrophoresed in 20% polyacrylamide gels containing SDS and urea. Duplicate gels were either silver stained (Daiichi, Tokyo, Japan) or electrotransferred to a nylon matrix (Immobilon from Millipore, Bedford, Mass.) for western analysis by standard means using rabbit anti-BPTI polyclonal antibodies.","Table 108 lists the interesting proteins visualized on a silver stained gel and by western analysis of an identical gel. We can see clearly in the western analysis that protein species containing BPTI epitopes are present in the test strains which are absent from the control strains and which are also IPTG inducible. In XL1-Blue\u2122, the migration of this species is predominantly that of the unprocessed form of the pro-protein although a small proportion of the encoded proteins appear to migrate at a size consistent with that of a fully processed form. In SEF\u2032, the processed form predominates, there being only a faint band corresponding to the unprocessed species.","Thus in strain SEF\u2032, we have produced a tripartite fusion protein that is specifically cleaved after the secretion signal sequence. We believe that the mature protein comprises BPTI followed by the gene VIII coat protein and that the coat protein moiety spans the membrane. We believe that it is highly likely that one or more copies, perhaps hundreds of copies, of this protein will co-assemble into M13 derived phage or M13-like phagemids. This construction will allow us to a) mutagenize the BPTI domain, b) display each of the variants on the coat of one or more phage (one type per phage), and c) recover those phage that display variants having novel binding properties with respect to target materials of our choice.","Rasched and Oberer (RASC86) report that phage produced in cells that express two alleles of gene VIII, that have differences within the first 11 residues of the mature coat protein, contain some of each protein. Thus, because we have achieved in vivo processing of the phoA(signal)::bpti::matureVIII fusion gene, it is highly likely that co-expression of this gene with wild-type VIII will lead to production of phage bearing BPTI domains on their surface. Mutagenesis of the bpti domain of these genes will provide a population of phage, each phage carrying a gene that codes for the variant of BPTI displayed on the phage surface.","VIII Display Phage: Production, Preparation and Analysis.","i. Phage Production.","The OCV can be grown in XL1-Blue\u2122 in the absence of the inducing agent, IPTG. Typically, a plaque plug is taken from a plate and grown in 2 ml of medium, containing freshly diluted bacterial cells, for 6 to 8 hours. Following centrifugation of this culture the supernatant is taken and the phage titer determined. This is kept as a phage stock for further infection, phage production and display of the gene product of interest.","A 100 fold dilution of a fresh overnight culture of SEF\u2032 bacterial cells in 500 ml of NZCYM medium is allowed to grow to a cell density of 0.4 (Ab 600 nm) in a shaker incubator at 37\u00b0 C. To this culture is added a sufficient amount of the phage stock to give a MOI of 10 together with IPTG to give a final concentration of 0.5 mM. The culture is allowed to grow for a further 2 hrs.","ii. Phage Preparation and Purification.","The phage producing bacterial culture is centrifuged to separate the phage in the supernatant from the bacterial pellet. To the supernatant is added one quarter by volume of phage precipitation solution (20% PEG, 3.75 M ammonium acetate) and PMSF to a final concentration of 1 mM. It is left on ice for 2 hours after which the precipitated phage is retrieved by centrifugation. The phage pellet is redissolved in TrisEDTA containing 0.1% Sarkosyl and left at 4\u00b0 C. for 1 hour after which any bacteria and bacterial debris is removed by centrifugation. The phage in the supernatant is reprecipitated with PEG overnight at 4\u00b0 C. The phage pellet is resuspended in LB medium and repreciptated another two times to remove the detergent. The phage is stored in LB medium at 4\u00b0 C., titered and used for analysis and binding studies.","A more stringent phage purification scheme involves centrifugation in a CsCl gradient. 3.86 g of CsCl is dissolved in NET buffer (0.1 M NaCl, 1 mM EDTA, 0.1M Tris pH 7.7) up to a volume of 10 ml. 10to 10phage in TE Sarkosyl buffer re mixed with 5 ml of CsCl NET buffer and transferred to a sealable ultracentrifuge tube. Centrifugation is performed overnight at 34K rpm in a Sorvall OTD-65B Ultracentrifuge. The tubes are opened and 400 \u03bcl aliqouts are carefully removed. 5 \u03bcl aliqouts are removed from the fractions and analysed by agarose gel electrophoresis after heating at 65\u00b0 C. for 15 minutes together with the gel loading buffer containing 0.1% SDS. Fractions containing phage are pooled, the phage reprecipitated and finally redissolved in LB medium to a concentration of 10to 10phage per ml.","iii. Phage Analysis.","The display phage, together with appropriate controls are analyzed using standard methods of polyacrylamide gel electrophoresis and either silver staining of the gel or electrotransfer to a nylon matrix followed by analysis with anti-BPTI antiserum (Western analysis). Quantitation of the display of heterologous proteins is achieved by running a serial dilution of the starting protein, for example BPTI, together with the display phage samples in the electrophoresis and Western analyses described above. An alternative method involves running a 2 fold serial dilution of a phage in which both the major coat protein and the fusion protein are visualized by silver staining. A comparison of the relative ratios of the two protein species allows one to estimate the number of fusion proteins per phage since the number of VIII gene encoded proteins per phage (approximately 3000) is known. Incorporation of fusion protein into bacteriophage.","In vivo expression of the processed BPTI:VIII fusion protein, encoded by vectors GemMB42 (above and Table 113) and M13MB48 (above), implied that the processed fusion product was likely to be correctly located within the bacterial cell membrane. This localization made it possible that it could be incorporated into the phage and that the BPTI moiety would be displayed at the bacteriophage surface.","SEF\u2032 cells were infected with either M13MB48 (consisting of the starting phage vector M13mp18, altered as described above, containing the synthetic gene consisting of a tac promoter, functional ribosome binding site, phoA signal peptide, mature BPTI and mature major coat protein) or M13mp18, as a control. Phage infections, preparation and purification was performed as described in Example VIII.","The resulting phage were electrophoresed (approximately 10phage per lane) in a 20% polyacrylamide gel containing urea followed by electrotransfer to a nylon matrix and western analysis using anti-BPTI rabbit serum. A single species of protein was observed in phage derived from infection with the M13MB48 stock phage which was not observed in the control infection. This protein had a migration of about 12 kd, consistent with that of the fully processed fusion protein.","Western analysis of SEF\u2032 bacterial lysate with or without phage infection demonstrated another species of protein of about 20 kd. This species was also present, to a lesser degree, in phage preparations which were simply PEG precipitated without further purification (for example, using nonionic detergent or by CsCl gradient centrifugation). A comparison of M13MB48 phage progoff eparations made in the presence or absence of detergent aldemonstrated that sarkosyl treatment and CsCl gradient purification did remove the bacterial contaminant while having no effect on the presence of the BPTI:VIII fusion protein. This indicates that the fusion protein has been incorporated and is a constituent of the phage body.","The time course of phage production and BPTI:VIII incorporation was followed post-infection and after IPTG induction. Phage production and fusion protein incorporation appeared to be maximal after two hours. This time course was utilized in further phage productions and analyses.","Polyacrylamide electrophoresis of the phage preparations, followed by silver staining, demonstrated that the preparations were essentially free of contaminating protein species and that an extra protein band was present in M13MB48 derived phage which was not present in the control phage. The size of the new protein was consistent with that seen by western analysis. A similar analysis of a serially diluted BPTI:VIII incorporated phage demonstrated that the ratio of fusion protein to major coat protein was typically in the range of 1:150. Since the phage is known to contain in the order of 3000 copies of the gene VIII product, this means that the phage population contains, on average, 10's of copies of the fusion protein per phage.","Altering the Initiating Methionine of the Natural Gene VIII.","The OCV M13MB48 contains the synthetic gene encoding the BPTI:VIII fusion protein in the intergenic region of the modified M13mp18 phage vector. The remainder of the vector consists of the M13 genome which contains the genes necessary for various bacteriophage functions, such as DNA replication and phage formation etc. In an attempt to increase the phage incorporation of the fusion protein, we decided to try to diminish the production of the natural gene VIII product, the major coat protein, by altering the codon for the initiating methionine of this gene to one encoding leucine. In such cases, methionine is actually incorporated, but the rate of initiation is reduced. The change was achieved by standard methods of site-specific oligonucleotide mutagenesis as follows.",{"@attributes":{"id":"p-0629","num":"0905"},"tables":{"@attributes":{"id":"TABLE-US-00017","num":"00017"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"217pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":["(SEQ ID NOs: 96, DNA and 97)",{}]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"196pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003M\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003S\u2003\u2003\u2003-rest of VIII",{}]},{"entry":[{},"ACT.TCC.TC.ATG.AAA.AAG.TCT."]}]}}]}},"br":[{},{}]},{"@attributes":{"id":"p-0630","num":"0906"},"tables":{"@attributes":{"id":"TABLE-US-00018","num":"00018"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"217pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":["(SEQ ID NOs:98, DNA and 99)",{}]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"196pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003(L)\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003S\u2003\u2003\u2003-rest of VIII",{}]},{"entry":[{},"ACT.TCC.AG.CTG.AAA.AAG.TCT."]}]}}]}},"br":[{},{}]},"Note that the 3\u2032 end of the XI gene overlaps with the 5\u2032 end of the VIII gene. Changes in DNA sequence were designed such that the desired change in the VIII gene product could be achieved without alterations to the predicted amino acid sequence of the gene XI product. A diagnostic PvuII recognition site was introduced at this site.","It was anticipated that initiation of the natural gene VIII product would be hindered, enabling a higher proportion of the fusion protein to be incorporated into the resulting phage.","Analyses of the phage derived from this modified vector indicated that there was a significant increase in the ratio of fusion protein to major coat protein. Quantitative estimates indicated that within a phage population as much as 100 copies of the BPTI:VIII fusion were incorporated per phage.","Incorporation of Interdomain Extension Fusion Proteins into Phage.","A phage pool containing a variegated pentapeptide extension at the BPTI:coat protein interface (see Example VII) was used to infect SEF\u2032 cells. IPTG induction, phage production and preparation were as described in Example VIII. Using the criteria detailed in the previous section, it was determined that extended fusion proteins were incorporated into phage. Gel electrophoresis of the generated phage, followed by either silver staining or western analysis with anti-BPTI rabbit serum, demonstrated fusion proteins that migrated similarly to but discernably slower that of the starting fusion protein.","With regard to the \u2018EGGGS linker\u2019 (SEQ ID NO:10) extensions of the domain interface, individual phage stocks predicted to contain one or more 5-amino-acid unit extensions were analyzed in a similar fashion. The migration of the extended fusion proteins were readily distinguishable from the parent fusion protein when viewed by western analysis or silver staining. Those clones analyzed in more detail included M13.3X4 (which contains a single inverted EGGGS (SEQ ID NO:10) linker with a predicted amino acid sequence of GSSSL (SEQ ID NO: 16)), M13.3X7 (which contains a correctly orientated linker with a predicted amino acid sequence of EGGGS (SEQ ID NO:10)), M13.3X11 (which contains 3 linkers with an inversion and a predicted amino acid sequence for the extension of EGGGSGSSSLGSSSL (SEQ ID NO:11)) and M13.3Xd which contains an extension consisting of at least 5 linkers or 25 amino acids.","The extended fusion proteins were all incorporated into phage at high levels (on average 10's of copies per phage were present and when analyzed by gel electrophoresis migrated rates consistent with the predicted size of the extension. Clones M13.3X4 and M13.3X7 migrated at a position very similar to but discernably different from the parent fusion protein, while M13.3X11 and M13.3Xd were markedly larger.","Display of BPTI:VIII Fusion Protein by Bacteriophage.","The BPTI:VIII fusion protein had been shown to be incorporated into the body of the phage. This phage was analyzed further to demonstrate that the BPTI moiety was accessible to specific antibodies and hence displayed at the phage surface.","The assay is detailed in Example II, but principally involves the addition of purified anti-BPTI IgG (from the serum of BPTI injected rabbits) to a known titer of phage. Following incubation, protein A-agarose beads are added to bind the IgG and left to incubate overnight. The IgG-protein A beads and any bound phage are removed by centrifugation followed by a retitering of the supernatant to determine any loss of phage. The phage bound to the beads can be acid eluted and titered also. Appropriate controls are included in the assay, such as a wild type phage stock (M13mp18) and IgG purified from normal rabbit pre-immune serum.","Table 140 shows that while the titer of the wild type phage is unaltered by the presence of anti-BPTI IgG, BPTI-IIIMK (the positive control for the assay), demonstrated a significant drop in titer with or without the extra addition of protein A beads. (Note that since the BPTI moiety is part of the III gene product which is involved in the binding of phage to bacterial pili, such a phenomenon is entirely expected.) Two batches of M13MB48 phage (containing the BPTI:VIII fusion protein) demonstrated a significant reduction in titer, as judged by plaque forming units, when anti-BPTI antibodies and protein A beads were added to the phage. The initial drop in titer with the antibody alone, differs somewhat between the two batches of phage. This may be a result of experimental or batch variation. Retrieval of the immunoprecipitated phage, while not quantitative, was significant when compared to the wild type phage control.","Further control experiments relating to this section are shown in Table 141 and Table 142. The data demonstrated that the loss in titer observed for the BPTI:VIII containing phage is a result of the display of BPTI epitopes by these phage and the specific interaction with anti-BPTI antibodies. No significant interaction with either protein A agarose beads or IgG purified from normal rabbit serum could be demonstrated. The larger drop in titer for M13MB48 batch five reflects the higher level incorporation of the fusion protein in this preparation.","Functionality of the BPTI Moiety in the BPTI-VIII Display Phage.","The previous two sections demonstrated that the BPTI:VIII fusion protein has been incorporated into the phage body and that the BPTI moiety is displayed at the phage surface. To demonstrate that the displayed molecule is functional, binding experiments were performed in a manner almost identical to that described in the previous section except that proteases were used in place of antibodies. The display phage, together with appropriate controls, are allowed to interact with immobilized proteases or immobilized inactivated proteases. Binding can be assessed by monitoring the loss in titer of the display phage or by determining the number of phage bound to the respective beads.","Table 143 shows the results of an experiment in which BPTI.VIII display phage, M13MB48, were allowed to bind to anhydrotrypsin-agarose beads. There was a significant drop in titer when compared to wild type phage, which do not display BPTI. A pool of phage (5AA Pool), each contain a variegated 5 amino acid extension at the BPTI:major coat protein interface, demonstrated a similar decline in titer. In a control experiment (table 143) very little non-specific binding of the above display phage was observed with agarose beads to which an unrelated protein (streptavidin) is attached.","Actual binding of the display phage is demonstrated by the data shown for two experiments in Table 144. The negative control is wild type M13mp18 and the positive control is BPTI-IIIMK, a phage in which the BPTI moiety, attached to the gene III protein, has been shown to be displayed and functional. M13MB48 and M13MB56 both bind to anhydrotrypsin beads in a manner comparable to that of the positive control, being 40 to 60 times better than the negative control (non-display phage). Hence functionality of the BPTI moiety, in the major coat fusion protein, was established.","To take this analysis one step further, a comparison of phage binding to active and inactivated trypsin is shown in Table 145. The control phage, M13mp18 and BPTI-III MK, demonstrated binding similar to that detailed in Example III. Note that the relative binding is enhanced with trypsin due to the apparent marked reduction in the non-specific binding of the wild type phage to the active protease. M13.3X7 and M13.3X11, which both contain \u2018EGGGS\u2019 linker (SEQ ID NO:10) extensions at the domain interface, bound to anhydrotrypsin and trypsin in a manner similar to BPTI-IIIMK phage. The binding, relative to non-display phage, was approximately 100 fold higher in the anhydrotrypsin binding assay and at least 1000 fold higher in the trypsin binding assay. The binding of another \u2018EGGGS\u2019 (SEQ ID NO:10) linker variant (M13.3Xd) was similar to that of M13.3X7.","To demonstrate the specificity of binding the assays were repeated with human neutrophil elastase (HNE) beads and compared to that seen with trypsin beads Table 146. BPTI has a very high affinity for trypsin and a low affinity for HNE, hence the BPTI display phage should reflect these affinities when used in binding assays with these beads. The negative and positive controls for trypsin binding were as already described above while an additional positive control for the HNE beads, BPTI(K15L,MGNG)-III MA (MGNG has SEQ ID NO:12) (see Example III) was included. The results, shown in Table 146, confirmed this prediction. M13MB48, M13.3X7 and M13.3X11 phage demonstrated good binding to trypsin, relative to wild type phage and the HNE control (BPTI(K15L,MGNG)-III MA) (The amino acid sequence MGNG has SEQ ID NO:12; BPTI (. . . , MGNG) denotes a homologue of BPTI having M, G, N, G, where . . . may indicate other alterations.), being comparable to BPTI-IIIMK phage. Conversely poor binding occurred when HNE beads were used, with the exception of the HNE positive control phage.","Taken together the accumulated data demonstrated that when BPTI is part of a fusion protein with the major coat protein of M13 phage, the molecule is both displayed at the surface of the phage and a significant proportion of it is functional in a specific protease binding manner.","Construction of BPTI\/Gene-III Display Vector","DNA manipulations were conducted according to standard procedures as described in Maniatis et al. (MANI82). First the unwanted lacZ gene of M13-MB1\/2 was removed. M13-MB1\/2 RF was cut with BamHI and SalI and the large fragment was isolated by agarose gel electrophoresis. The recovered 6819 bp fragment was filled in with Klenow fragment of DNA polymerase and ligated to a synthetic HindIII 8mer linker (CAAGCTTG). The ligation sample was used to transfect competent XL1-Blue\u2122 (Stratagene, La Jolla, Calif.) cells which were subsequently plated for plaque formation. RF DNA was prepared from chosen plaques and a clone, M13-MB1\/2-delta, containing regenerated BamHI and SalI sites as well as a new HindIII site, all 500 bp upstream of the BglII site (6935) was picked.","A unique NarI site was introduced into codons 17 and 18 of gene III (changing the amino acids from H\u2014S to G-A, Cf. Table 110). 10phage produced from bacterial cells harboring the M13-MB1\/2-delta RF DNA were used to infect a culture of CJ236 cells (relevant genotype: F\u2032, dut1, ung1, Cm) (OD595=0.35). Following overnight incubation at 37\u00b0 C., phage were recovered and uracil-containing ss DNA was extracted from phage in accord with the instructions for the MUTA-GENE\u00ae M13 in vitro Mutagenesis Kit (Catalogue Number 170-3571, Bio-Rad, Richmond, Calif.). Two hundred nanograms of the purified single stranded DNA was annealed to 3 picomoles of a phosphorylated 25mer mutagenic oligonucleotide, 5\u2032-gtttcagcggCgCCagaatagaaag-3\u2032, (SEQ ID NO:147) where upper case indicates the changes). Following filling in with T4 DNA polymerase and ligation with T4 DNA ligase, the reaction sample was used to transfect competent XL1-Blue\u2122 cells which were subsequently plated to permit the formation of plaques.","RF DNA, isolated from phage-infected cells which had been allowed to propagate in liquid culture for 8 hours, was denatured, spotted on a Nytran membrane, baked and hybridized to the 25mer mutagenic oligonucleotide which had previously been phosphorylated with P-ATP. Clones exhibiting strong hybridization signals at 70\u00b0 C. (6\u00b0 C. less than the theoretical Tm of the mutagenic oligonucleotide) were chosen for large scale RF preparation. The presence of a unique NarI site at nucleotide 1630 was confirmed by restriction enzyme analysis. The resultant RF DNA, M13-M1\/2-delta-NarI was cut with BamHI, dephosphorylated with calf intestinal phosphatase, and ligated to a 1.3 Kb BamHI fragment, encoding the kanamycin-resistance gene (kan), derived from plasmid pUC4K (Pharmacia, Piscataway, N.J.). The ligation sample was used to transfect competent XL1-Blue\u2122 cells which were subsequently plated onto LB plates containing kanamycin (Km). RF DNA prepared from Kmcolonies was prepared and subjected to restriction enzyme analysis to confirm the insertion of kan into M13-M1\/2-delta-NarI DNA thereby creating the phage MK. Phage MK grows as well as wild-type M13, indicating that the changes at the cleavage site of gene III protein are not detectably deleterious to the phage.","Insertion of Synthetic BPTI Gene","The construction of the BPTI-III expression vector is shown in . The synthetic bpti-VIII fusion contains a NarI site that comprises the last two codons of the BPTI-encoding region. A second NarI site was introduced upstream of the BPTI-encoding region as follows. RF DNA of phage M13-MB26 was cut with AccIII and ligated to the dsDNA adaptor:",{"@attributes":{"id":"p-0651","num":"0927"},"chemistry":{"@attributes":{"id":"CHEM-US-00013","num":"00013"},"img":{"@attributes":{"id":"EMI-C00013","he":"9.23mm","wi":"59.10mm","file":"US07413537-20080819-C00013.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":[{},{}],"sup":"32"},"MK-BPTI RF DNA was added to a coupled prokaryotic transcription-translation extract (Amersham). Newly synthesized radiolabelled proteins were produced and subsequently separated by electrophoresis on a 15% SDS-polyacrylamide gel subjected to fluorography. The MK-BPTI DNA directs the synthesis of an unprocessed gene III fusion protein which is 7 Kd larger than the gene III product encoded by MK. This is consistent with the insertion of 58 amino acids of BPTI into the gene III protein. Immunoprecipitation of radiolabelled proteins generated by the cell-free prokaryotic extract was conducted. Neither rabbit anti(M13-gene-VIII-protein) IgG nor normal rabbit IgG were able to immunoprecipitate the gene III protein encoded by either MK or MK-BPTI. However, rabbit anti-BPTI IgG is able to immunoprecipitate the gene III protein encoded by MK-BPTI but not by MK. This confirms that the increase in size of the III protein encoded by MK-BPTI is attributable to the insertion of the BPTI protein.","Western Analysis","Phage were recovered from bacterial cultures by PEG precipitation. To remove residual bacterial cells, recovered phage were resuspended in a high salt buffer and subjected to centrifugation, in accord with the instructions for the MUTA-GENE\u00ae M13 in vitro Mutagenesis Kit (Catalogue Number 170-3571, Bio-Rad, Richmond, Calif.). Aliquots of phage (containing up to 40 \u03bcg of protein) were subjected to electrophoresis on a 12.5% SDS-urea-polyacrylamide gel and proteins were transferred to a sheet of Immobilon by electro-transfer. Western blots were developed using rabbit anti-BPTI serum, which had previously been incubated with an extract, followed by goat ant-rabbit antibody conjugated to alkaline phosphatase. An immunoreactive protein of 67 Kd is detected in preparations of the MK-BPTI but not the MK phage. The size of the immunoreactive protein is consistent with the predicted size of a processed BPTI-III fusion protein (6.4 Kd plus 60 Kd). These data indicate that BPTI-specific epitopes are presented on the surface of the MK-BPTI phage but not the MK phage.","Neutralization of Phage Titer with Agarose-Immobilized Anhydro-Trypsin","Anhydro-trypsin is a derivative of trypsin in which the active site serine has been converted to dehydroalanine. Anhydro-trypsin retains the specific binding of trypsin but not the protease activity. Unlike polyclonalantibodies, anhydro-trypsin is not expected to bind unfolded BPTI or incomplete fragments.","Phage MK-BPTI and MK were diluted to a concentration 1.4\u00b710particles per ml. in TBS buffer (PARM88) containing 1.0 mg\/ml BSA. Thirty microliters of diluted phage were added to 2, 5, or 10 microliters of a 50% slurry of agarose-immobilized anhydro-trypsin (Pierce Chemical Co., Rockford, Ill.) in TBS\/BSA buffer. Following incubation at 25\u00b0 C., aliquots were removed, diluted in ice cold LB broth and titered for plaque-forming units on a lawn of XL1-Blue\u2122 cells. Table 114 illustrates that incubation of the MK-BPTI phage with immobilized anhydro-trypsin results in a very significant loss in titer over a four hour period while no such effect is observed with the MK (control) phage. The reduction in phage titer is also proportional to the amount of immobilized anhydro-trypsin added to the MK-BPTI phage. Incubation with five microliters of a 50% slurry of agarose-immobilized streptavidin (Sigma, St. Louis, Mo.) in TBS\/BSA buffer does not reduce the titer of either the MK-BPTI or MK phage. These data are consistent with the presentation of a correctly-folded, functional BPTI protein on the surface of the MK-BPTI phage but not on the MK phage. Unfolded or incomplete BPTI domains are not expected to bind anhydro-trypsin. Furthermore, unfolded BPTI domains are expected to be non-specifically sticky.","Neutralization of Phage Titer with Anti-BPTI Antibody","MK-BPTI and MK phage were diluted to a concentration of 4\u00b710plaque-forming units per ml in LB broth. Fifteen microliters of diluted phage were added to an equivalent volume of either rabbit anti-BPTI serum or normal rabbit serum (both diluted 10 fold in LB broth). Following incubation at 37\u00b0 C., aliquots were removed, diluted by 10in ice-cold LB broth and titered for plaque-forming units on a lawn of XL1-Blue\u2122 cells. Incubation of the MK-BPTI phage with anti-BPTI serum results in a steady loss in titer over a two hour period while no such effect is observed with the MK phage. As expected, normal rabbit serum does not reduce the titer of either the MK-BPTI or the MK phage. Prior incubation of the anti-BPTI serum with authentic BPTI protein but not with an equivalent amount of protein, blocks the ability of the serum to reduce the titer of the MK-BPTI phage. This data is consistent with the presentation of BPTI-specific epitopes on the surface of the MK-BPTI phage but not the MK phage. More specifically, the data indicates that these BPTI epitopes are associated with the gene III protein and that association of this fusion protein with an anti-BPTI antibody blocks its ability to mediate the infection of bacterial cells.","Neutralization Phage Titer with Trypsin","MK-BPTI and MK phage were diluted to a concentration of 4\u00b710plaque-forming units per ml in LB broth. Diluted phage were added to an equivalent volume of trypsin diluted to various concentrations in LB broth. Following incubation at 37\u00b0 C., aliquots were removed, diluted by 10in ice cold LB broth and titered for plaque-forming units on a lawn of XL1-Blue\u2122 cells. Incubation of the MK-BPTI phage with 0.15 \u03bcg of trypsin results in a 70% loss in titer after a two hour period while only a 15% loss in titer is observed for the MK phage. A reduction in the amount of trypsin added to phage results in a reduction in the loss of titer. However, at all trypsin concentrations investigated, the MK-BPTI phage are more sensitive to incubation with trypsin than the MK phage. An interpretation of this data is that association of the BPTI-III fusion protein displayed on the surface of the MK-BPTI phage with trypsin blocks its ability to mediate the infection of bacterial cells.","The reduction in titer of phage MK by trypsin is an example of a phenomenon that is likely to be general: proteases, if present in sufficient quantity, will degrade proteins on the phage and reduce infectivity. The present application lists several means that can be used to overcome this problem.","Affinity Selection System","Affinity Selection With Immobilized Anhydro-Trypsin","MK-BPTI and MK phage were diluted to a concentration of 1.4-10particles per ml in TBS buffer (PARM88) containing 1.0 mg\/ml BSA. We added 4.0\u00b710phage to 5 microliters of a 50% slurry of either agarose-immobilized anhydro-trypsin beads (Pierce Chemical Co.) or agarose-immobilized streptavidin beads (Sigma) in TBS\/BSA. Following a 3 hour incubation at room temperature, the beads were pelleted by centrifugation for 30 seconds at 5000 rpm in a microfuge and the supernatant fraction was collected. The beads were washed 5 times with TBS\/Tween buffer (PARM88) and after each wash the beads were pelleted by centrifugation and the supernatant was removed. Finally, beads were resuspended in elution buffer (0.1 N HCl containing 1.0 mg\/ml BSA adjusted to pH 2.2 with glycine) and following a 5 minute incubation at room temperature, the beads were pelleted by centrifugation. The supernatant was removed and neutralized by the addition of 1.0 M Tris-HCl buffer, pH 8.0.","Aliquots of phage samples were applied to a Nytran membrane using a Schleicher and Schuell (Keene, N.H.) filtration minifold and phage DNA was immobilized onto the Nytran by baking at 80\u00b0 C. for 2 hours. The baked filter was incubated at 42\u00b0 C. for 1 hour in pre-wash solution (MANI82) and pre-hybridization solution (5Prime-3Prime, West Chester, Pa.). The 1.0 Kb NarI (base 1630)\/XmnI (base 2646) DNA fragment from MK RF was radioactively labelled with P-dCTP using an oligolabelling kit (Pharmacia, Piscataway, N.J.). The radioactive probe was added to the Nytran filter in hybridization solution (5Prime-3Prime) and, following overnight incubation at 42\u00b0 C., the filter was washed and subjected to autoradiography.","The efficiency of this affinity selection system can be semi-quantitatively determined using the dot-blot procedure described elsewhere in the present application. Exposure of MK-BPTI-phage-treated anhydro-trypsin beads to elution buffer releases bound MK-BPTI phage. Streptavidin beads do not retain phage MK-BPTI. Anhydro-trypsin beads do not retain phage MK. In the experiment depicted in Table 115, we estimate that 20% of the total MK-BPTI phage were bound to 5 microliters of the immobilized anhydro-trypsin and were subsequently recovered by washing the beads with elution buffer (pH 2.2 HCl\/glycine). Under the same conditions, no detectable MK-BPTI phage were bound and subsequently recovered from the streptavidin beads. The amount of MK-BPTI phage recovered in the elution fraction is proportional to the amount of immobilized anhydro-trypsin added to the phage. No detectable MK phage were bound to either the immobilized anhydro-trypsin or streptavidin beads and no phage were recovered with elution buffer. These data indicate that the affinity selection system described above can be utilized to select for phage displaying a specific folded protein (in this case, BPTI). Unfolded or incomplete BPTI domains are not expected to bind anhydro-trypsin.","Affinity Selection With Anti-BPTI Antibodies","MK-BPTI and MK phage were diluted to a concentration of 1\u00b710particles per ml in Tris buffered saline solution (PARM88) containing 1.0 mg\/ml BSA. Two\u00b710phage were added to 2.5 \u03bcg of either biotinylated rabbit anti-BPTI IgG in TBS\/BSA or biotinylated rabbit anti-mouse antibody IgG (Sigma) in TBS\/BSA, and incubated overnight at 4\u00b0 C. A 50% slurry of streptavidin-agarose (Sigma), washed three times with TBS buffer prior to incubation with 30 mg\/ml BSA in TBS buffer for 60 minutes at room temperature, was washed three times with TBS\/Tween buffer (PARM88) and resuspended to a final concentration of 50% in this buffer. Samples containing phage and biotinylated IgG were diluted with TBS\/Tween prior to the addition of streptavidin-agarose in TBS\/Tween buffer. Following a 60 minute incubation at room temperature, streptavidin-agarose beads were pelleted by centrifugation for 30 seconds and the supernatant fraction was collected. The beads were washed 5 times with TBS\/Tween buffer and after each wash, the beads were pelleted by centrifugation and the supernatant was removed. Finally, the streptavidin-agarose beads were resuspended in elution buffer (0.1 N HCl containing 1.0 mg\/ml BSA adjusted to pH 2.2 with glycine), incubated 5 minute at room temperature, and pelleted by centrifugation. The supernatant was removed and neutralized by the addition of 1.0 M Tris-HCl buffer, pH 8.0.","Aliquots of phage samples were applied to a Nytran membrane using a Schleicker and Schuell minifold apparatus. Phage DNA was immobilized onto the Nytran by baking at 80\u00b0 C. for 2 hours. Filters were washed for 60 minutes in pre-wash solution (MANI82) at 42\u00b0 C. then incubated at 42\u00b0 C. for 60 minutes in Southern pre-hybridization solution (5Prime-3Prime). The 1.0 Kb NarI (1630 bp)\/XmnI (2646 bp) DNA fragment from MK RF was radioactively labelled with P-\u03b1dCTP using an oligolabelling kit (Pharmacia, Piscataway, N.J.). Nytran membranes were transferred from pre-hybridization solution to Southern hybridization solution (5Prime-3Prime) at 42\u00b0 C. The radioactive probe was added to the hybridization solution and following overnight incubation at 42\u00b0 C., the filter was washed 3 times with 2\u00d7SSC, 0.1% SDS at room temperature and once at 65\u00b0 C. in 2\u00d7SSC, 0.1% SDS. Nytran membranes were subjected to autoradiography. The efficiency of the affinity selection system can be semi-quantitatively determined using the above dot blot procedure. Comparison of dots A1 and B1 or C1 and D1 indicates that the majority of phage did not stick to the streptavidin-agarose beads. Washing with TBS\/Tween buffer removes the majority of phage which are non-specifically associated with streptavidin beads. Exposure of the streptavidin beads to elution buffer releases bound phage only in the case of MK-BPTI phage which have previously been incubated with biotinylated rabbit anti-BPTI IgG. This data indicates that the affinity selection system described above can be utilized to select for phage displaying a specific antigen (in this case BPTI). We estimate an enrichment factor of at least 40 fold based on the calculation",{"@attributes":{"id":"p-0664","num":"0940"},"maths":{"@attributes":{"id":"MATH-US-00002","num":"00002"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mrow":{"mi":["Enrichment","Factor"],"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}}},"mo":"=","mfrac":{"mrow":[{"mi":["Percent","MK","BPTI","phage","recovered"],"mo":["\u2062","\u2062","\u2062","\u2062","\u2062","\u2062","\u2062","\u2062"],"mstyle":[{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}},{"mtext":"-"},{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}},{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}}]},{"mi":["Percent","MK","phage","recovered"],"mo":["\u2062","\u2062","\u2062","\u2062","\u2062","\u2062"],"mstyle":[{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}},{"mspace":{"@attributes":{"width":"1.1em","height":"1.1ex"}}},{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}}]}]}}}}},"This Example demonstrates that chimeric phage proteins displaying a target-binding domain can be eluted from immobilized target by decreasing pH, and the pH at which the protein is eluted is dependent on the binding affinity of the domain for the target.","Standard Procedures:","Unless otherwise noted, all manipulations were carried out at room temperature. Unless otherwise noted, all cells are XL1-Blue(\u2122) (Stratagene, La Jolla, Calif.).","1) Demonstration of the Binding of BPTI-III MK Phage to Active Trypsin Beads","Previous experiments designed to verify that BPTI displayed by fusion phage is functional relied on the use of immobilized anhydro-trypsin, a catalytically inactive form of trypsin. Although anhydro-trypsin is essentially identical to trypsin structurally (HUBE75, YOKO77) and in binding properties (VINC74, AKOH72), we demonstrated that BPTI-III fusion phage also bind immobilized active trypsin. Demonstration of the binding of fusion phage to immobilized active protease and subsequent recovery of infectious phage facilitates subsequent experiments where the preparation of inactive forms of serine proteases by protein modification is laborious or not feasible.","Fifty \u03bcl of BPTI-III MK phage (identified as MK-BPTI is Example II) (3.7\u00b710pfu\/ml) in either 50 mM Tris, pH 7.5, 150 mM NaCl, 1.0 mg\/ml BSA (TBS\/BSA) buffer or 50 mM sodium citrate, pH 6.5, 150 mM NaCl, 1.0 mg\/ml BSA (CBS\/BSA) buffer were added to 10 \u03bcl of a 25% slurry of immobilized trypsin (Pierce Chemical Co., Rockford, Ill.) also in TBS\/BSA or CBS\/BSA. As a control, 50 \u03bcl MK phage (9.3\u00b710pfu\/ml) were added to 10 \u03bcl of a 25% slurry of immobilized trypsin in either TBS\/BSA or CBS\/BSA buffer. The infectivity of BPTI-III MK phage is 25-fold lower than that of MK phage; thus the conditions chosen above ensure that an approximately equivalent number of phage particles are added to the trypsin beads. After 3 hours of mixing on a Labquake shaker (Lab industries Inc., Berkeley, Calif.) 0.5 ml of either TBS\/BSA or CBS\/BSA was added where appropriate to the samples. Beads were washed for 5 min and recovered by centrifugation for 30 sec. The supernatant was removed and 0.5 ml of TBS\/0.1% Tween-20 was added. The beads were mixed for 5 minutes on the shaker and recovered by centrifugation as above. The supernatant was removed and the beads were washed an additional five times with TBS\/0.1% Tween-20 as described above. Finally, the beads were resuspended in 0.5 ml of elution buffer (0.1 M HCl containing 1.0 mg\/ml BSA adjusted to pH 2.2 with glycine), mixed for 5 minutes and recovered by centrifugation. The supernatant fraction was removed and neutralized by the addition of 130 \u03bcl of 1 M Tris, pH 8.0. Aliquots of the neutralized elution sample were diluted in LB broth and titered for plaque-forming units on a lawn of cells.","Table 201 illustrates that a significant percentage of the input BPTI-III MK phage bound to immobilized trypsin and was recovered by washing with elution buffer. The amount of fusion phage which bound to the beads was greater in TBS buffer (pH 7.5) than in CBS buffer (pH 6.5). This is consistent with the observation that the affinity of BPTI for trypsin is greater at pH 7.5 than at pH 6.5 (VINC72, VINC74). A much lower percentage of the MK control phage (which do not display BPTI) bound to immobilized trypsin and this binding was independent of the pH conditions. At pH 6.5, 1675 times more of the BPTI-III MK phage than of the MK phage bound to trypsin beads while at pH 7.5, a 2103-fold difference was observed. Hence fusion phage displaying BPTI adhere not only to anhydro-trypsin beads but also to active trypsin beads and can be recovered as infectious phage. These data, in conjunction with earlier findings, strongly suggest that BPTI displayed on the surface of fusion phage is appropriately folded and functional.","2) Generation of P1 Mutants of BPTI","To demonstrate the specificity of interaction of BPTI-III fusion phage with immobilized serine proteases, single amino acid substitutions were introduced at the P1 position (residue 15 of mature BPTI) of the BPTI-III fusion protein by site-directed mutagenesis. A 25mer mutagenic oligonucleotide (P1) was designed to substitute a LEU codon for the LYScodon. This alteration is desired because BPTI(K15L) is a moderately good inhibitor of human neutrophil elastase (HNE) (K=2.9\u00b710M) (BECK88b) and a poor inhibitor of trypsin. A fusion phage displaying BPTI(K15L) should bind to immobilized HNE but not to immobilized trypsin. BPTI-III MK fusion phage would be expected to display the opposite phenotype (bind to trypsin, fail to bind to HNE). These observations would illustrate the binding specificity of BPTI-III fusion phage for immobilized serine proteases.","Mutagenesis of the P1 region of the BPTI-VIII gene contained within the intergenic region of recombinant phage MB46 was carried out using the Muta-Gene M13 In Vitro Mutagenesis Kit (Bio-Rad, Richmond, Calif.). MB46 phage (7.5\u00b710pfu) were used to infect a 50 ml culture of CJ236 cells (O.D.600=0.5). Following overnight incubation at 37\u00b0 C., phage were recovered and uracil-containing single-stranded DNA was extracted from the phage. The single-stranded DNA was further purified by NACS chromatography as recommended by the manufacturer (B.R.L., Gaithersburg, Md.).","Two hundred nanograms of the purified single-stranded DNA were annealed to 3 picomoles of the phosphorylated 25mer mutagenic oligonucleotide (P1). Following filling in with T4 DNA polymerase and ligation with T4 DNA ligase, the sample was used to transfect competent cells which were subsequently plated on LB plates to permit the formation of plaques. Phage derived from picked plaques were applied to a Nytran membrane using a Schleicher and Schuell (Keene, N.H.) minifold I apparatus (Dot Blot Procedure). Phage DNA was immobilized onto the filter by baking at 80\u00b0 C. for 2 hours. The filter was bathed in 1\u00d7 Southern pre-hybridization buffer (5Prime-3Prime, West Chester, Pa.) for 2 hours. Subsequently, the filter was incubated in 1\u00d7 Southern hybridization solution (5Prime-3Prime) containing a 21mer probing oligonucleotide (LEUL) which had been radioactively labelled with gamma-P-ATP (N.E.N.\/DuPont, Boston, Mass.) by T4 polynucleotide kinase (New England BioLabs (NEB), Beverly, Mass.). Following overnight hybridization, the filter was washed 3 times with 6\u00d7SSC at room temperature and once at 60\u00b0 C. in 6\u00d7SSC prior to autoradiography. Clones exhibiting strong hybridization signals were chosen for large scale Rf preparation using the PZ523 spin column protocol (5Prime-3Prime). Restriction enzyme analysis confirmed that the structure of the Rf was correct and DNA sequencing confirmed the substitution of a LEU codon (TTG) for the LYScodon (AAA). This Rf DNA was designated MB46(K15L).","3) Generation of the BPTI-III MA Vector","The original gene III fusion phage MK can be detected on the basis of its ability to transduce cells to kanamycin resistance (Km). It was deemed advantageous to generate a second gene III fusion vector which can confer resistance to a different antibiotic, namely ampicillin (Ap). One could then mix a fusion phage conferring Apwhile displaying engineered protease inhibitor A (EPI-A) with a second fusion phage conferring Kmwhile displaying EPI-B. The mixture could be added to an immobilized serine protease and, following elution of bound fusion phage, one could evaluate the relative affinity of the two EPIs for the immobilized protease from the relative abundance of phage that transduce cells to Kmor Ap.","The apgene is contained in the vector pGem3Zf (Promega Corp., Madison, Wis.) which can be packaged as single stranded DNA contained in bacteriophage when helper phage are added to bacteria containing this vector. The recognition sites for restriction enzymes SmaI and SnaBI were engineered into the 3\u2032 non-coding region of the Ap(\u03b2-lactamase) gene using the technique of synthetic oligonucleotide directed site specific mutagenesis. The single stranded DNA was used as the template for in vitro mutagenesis leading to the following DNA sequence alterations (numbering as supplied by Promega): a) to create a SmaI (or XmaI) site, bases T\u2192C and A\u2192C, and b) to create a SnaBI site, G\u2192T, C\u2192T, and T\u2192A. The alterations were confirmed by radiolabelled probe analysis with the mutating oligonucleotide and restriction enzyme analysis; this plasmid is named pSGK3.","Plasmid SGK3 was cut with AatII and SmaI and treated with T4 DNA polymerase (NEB) to remove overhanging 3\u2032 ends (MANI82, SAMB89). Phosphorylated HindIII linkers (NEB) were ligated to the blunt ends of the DNA and following HindIII digestion, the 1.1 kb fragment was isolated by agarose gel electrophoresis followed by purification on an Ultrafree-MC filter unit as recommended by the manufacturer (Millipore, Bedford, Mass.). M13-MB1\/2-delta Rf DNA was cut with HindIII and the linearized Rf was purified and ligated to the 1.1 kb fragment derived from pSGK3. Ligation samples were used to transfect competent cells which were plated on LB plates containing Ap. Colonies were picked and grown in LB broth containing Ap overnight at 37\u00b0 C. Aliquots of the culture supernatants were assayed for the presence of infectious phage. Rf DNA was prepared from cultures which were both Apand contained infectious phage. Restriction enzyme analysis confirmed that the Rf contained a single copy of the Apgene inserted into the intergenic region of the M13 genome in the same transcriptional orientation as the phage genes. This Rf DNA was designated MA.","The 5.9 kb BglII\/BsmI fragment from MA Rf DNA and the 2.2 kb BglII\/BsmI fragment from BPTI-III MK Rf DNA were ligated together and a portion of the ligation mixture was used to transfect competent cells which were subsequently plated to permit plaque formation on a lawn of cells. Large and small size plaques were observed on the plates. Small size plaques were picked for further analysis since BPTI-III fusion phage give rise to small plaques due to impairment of gene III protein function. Small plaques were added to LB broth containing Ap and cultures were incubated overnight at 37\u00b0 C. An Apculture which contained phage which gave rise to small plaques when plated on a lawn of cells was used as a source of Rf DNA. Restriction enzyme analysis confirmed that the BPTI-III fusion gene had been inserted into the MA vector. This Rf was designated BPTI-III MA.","4) Construction of BPTI(K15L)-III MA","MB46(K15L) Rf DNA was digested with XhoI and EagI and the 125 bp DNA fragment was isolated by electrophoresis on a 2% agarose gel followed by extraction from an agarose slice by centrifugation through an Ultrafree-MC filter unit. The 8.0 kb XhoI\/EagI fragment derived from BPTI-III MA Rf was also prepared. The above two fragments were ligated and the ligation sample was used to transfect competent cells which were plated on LB plates containing Ap. Colonies were picked and used to inoculate LB broth containing Ap. Cultures were incubated overnight at 37\u00b0 C. and phage within the culture supernatants was probed using the Dot Blot Procedure. Filters were hybridized to a radioactively labelled oligonucleotide (LEU1). Positive clones were identified by autoradiography after washing filters under high stringency conditions. Rf DNA was prepared from Apcultures which contained phage carrying the K15L mutation. Restriction enzyme analysis and DNA sequencing confirmed that the K15L mutation had been introduced into the BPTI-III MA Rf. This Rf was designated BPTI(K15L)-III MA. Interestingly, BPTI(K15L)-III MA phage gave rise to extremely small plaques on a lawn of cells and the infectivity of the phage is 4 to 5 fold less than that of BPTI-III MK phage. This suggests that the substitution of LEU for LYSimpairs the ability of the BPTI:gene III fusion protein to mediate phage infection of bacterial cells.","5) Preparation of Immobilized Human Neutrophil Elastase","One ml of Reacti-Gel 6\u00d7CDI activated agarose (Pierce Chemical Co.) in acetone (200 \u03bcl packed beads) was introduced into an empty Select-D spin column (5Prime-3Prime). The acetone was drained out and the beads were washed twice rapidly with 1.0 ml of ice cold water and 1.0 ml of ice cold 100 mM boric acid, pH 8.5, 0.9% NaCl. Two hundred \u03bcl of 2.0 mg\/ml human neutrophil elastase (HNE) (CalBiochem, San Diego, Calif.) in borate buffer were added to the beads. The column was sealed and mixed end over end on a Labquake Shaker at 4\u00b0 C. for 36 hours. The HNE solution was drained off and the beads were washed with ice cold 2.0 M Tris, pH 8.0 over a 2 hour period at 4\u00b0 C. to block remaining reactive groups. A 50% slurry of the beads in TBS\/BSA was prepared. To this was added an equal volume of sterile 100% glycerol and the beads were stored as a 25% slurry at \u221220\u00b0 C. Prior to use, the beads were washed 3 times with TBS\/BSA and a 50% slurry in TBS\/BSA was prepared.","6) Characterization of the Affinity of BPTI-III MK and BPTI(K15L)-III MA Phage for Immobilized Trypsin and Human Neutrophil Elastase","Thirty \u03bcl of BPTI-III MK phage in TBS\/BSA (1.7\u00b710pfu\/ml) was added to 5 \u03bcl of a 50% slurry of either immobilized human neutrophil elastase or immobilized trypsin (Pierce Chemical Co.) also in TBS\/BSA. Similarly 30 \u03bcl of BPTI(K15L)-III MA phage in TBS\/BSA (3.2\u00b710pfu\/ml) was added to either immobilized HNE or trypsin. Samples were mixed on a Labquake shaker for 3 hours. The beads were washed with 0.5 ml of TBS\/BSA for 5 minutes and recovered by centrifugation. The supernatant was removed and the beads were washed 5 times with 0.5 ml of TBS\/0.1% Tween-20. Finally, the beads were resuspended in 0.5 ml of elution buffer (0.1 M HCl containing 1.0\u00b7mg\/ml BSA adjusted to pH 2.2 with glycine), mixed for 5 minutes and recovered by centrifugation. The supernatant fraction was removed, neutralized with 130 \u03bcl of 1 M Tris, pH 8.0, diluted in LB broth, and titered for plaque-forming units on a lawn of cells.","Table 202 illustrates that 82 times more of the BPTI-III MK input phage bound to the trypsin beads than to the HNE beads. By contrast, the BPTI(K15L)-III MA phage bound preferentially to HNE beads by a factor of 36. These results are consistent with the known affinities of wild type and the K15L variant of BPTI for trypsin and HNE. Hence BPTI-III fusion phage bind selectively to immobilized proteases and the nature of the BPTI variant displayed on the surface of the fusion phage dictates which particular protease is the optimum receptor for the fusion phage.","7) Effect of pH on the Dissociation of Bound BPTI-III MK and BPTI(K15L)-III MA Phage from Immobilized Neutrophil Elastase","The affinity of a given fusion phage for an immobilized serine protease can be characterized on the basis of the amount of bound fusion phage which elutes from the beads by washing with a pH 2.2 buffer. This represents rather extreme conditions for the dissociation of fusion phage from beads. Since the affinity of the BPTI variants described above for HNE is not high (K>1\u00b710M) it was anticipated that fusion phage displaying these variants might dissociate from HNE beads under less severe pH conditions. Furthermore fusion phage might dissociate from HNE beads under specific pH conditions characteristic of the particular BPTI variant displayed by the phage. Low pH buffers providing stringent wash conditions might be required to dissociate fusion phage displaying a BPTI variant with a high affinity for HNE whereas neutral pH conditions might be sufficient to dislodge a fusion phage displaying a BPTI variant with a weak affinity for HNE.","Thirty \u03bcl of BPTI(K15L)-III MA phage (1.7\u00b710pfu\/ml in TBS\/BSA) were added to 5 \u03bcl of a 50% slurry of immobilized HNE also in TBS\/BSA. Similarly, 30 \u03bcl of BPTI-III MA phage (8.6\u00b710pfu\/ml in TBS\/BSA) were added to 5 \u03bcl of immobilized HNE. The above conditions were chosen to ensure that an approximately equivalent number of phage particles were added to the beads. The samples were incubated for 3 hours on a Labquake shaker. The beads were washed with 0.5 ml of TBS\/BSA for 5 min on the shaker, recovered by centrifugation and the supernatant was removed. The beads were washed with 0.5 ml of TBS\/0.1% Tween-20 for 5 minutes and recovered by centrifugation. Four additional washes with TBS\/0.1% Tween-20 were performed as described above. The beads were washed as above with 0.5 ml of 100 mM sodium citrate, pH 7.0 containing 1.0 mg\/ml BSA. The beads were recovered by centrifugation and the supernatant was removed. Subsequently, the HNE beads were washed sequentially with a series of 100 mM sodium citrate, 1.0 mg\/ml BSA buffers of pH 6.0, 5.0, 4.0 and 3.0 and finally with the 2.2 elution buffer described above. The pH washes were neutralized by the addition of 1 M Tris, pH 8.0, diluted in LB broth and titered for plaque-forming units on a lawn of cells.","Table 203 illustrates that a low percentage of the input BPTI-III MK fusion phage adhered to the HNE beads and was recovered in the pH 7.0 and 6.0 washes predominantly. By contrast, a significantly higher percentage of the BPTI(K15L)-III MA phage bound to the HNE beads and was recovered predominantly in the pH 5.0 and 4.0 washes. Hence lower pH conditions (i.e. more stringent) are required to dissociate BPTI(K15L)-III MA than BPTI-MK phage from immobilized HNE. The affinity of BPTI(K15L) is over 1000 times greater than that of BPTI for HNE (based on reported Kvalues (BECK88b)). Hence this suggests that lower pH conditions are indeed required to dissociate fusion phage displaying a BPTI variant with a higher affinity for HNE.","8) Construction of BPTI(MGNG)-III MA Phage (MGNG has SEQ ID NO:12)","The light chain of bovine inter-\u03b1-trypsin inhibitor contains 2 domains highly homologous to BPTI. The amino terminal proximal domain (called BI-8e) has been generated by proteolysis and shown to be a potent inhibitor of HNE (K=4.4\u00b710M) (ALBR83). By contrast a BPTI variant with the single substitution of LEU for LYSexhibits a moderate affinity for HNE (K=2.9\u00b710M) (BECK88b). It has been proposed that the P1 residue is the primary determinant of the specificity and potency of BPTI-like molecules (BECK88b, LASK80 and works cited therein). Although both BI-8e and BPTI(K15L) feature LEU at their respective P1 positions, there is a 66 fold difference in the affinities of these molecules for HNE. Structural features, other than the P1 residue, must contribute to the affinity of BPTI-like molecules for HNE.","A comparison of the structures of BI-8e and BPTI(K15L) reveals the presence of three positively charged residues at positions 39, 41, and 42 of BPTI which are absent in BI-8e. These hydrophilic and highly charged residues of BPTI are displayed on a loop which underlies the loop containing the P1 residue and is connected to it via a disulfide bridge. Residues within the underlying loop (in particular residue 39) participate in the interaction of BPTI with the surface of trypsin near the catalytic pocket (BLOW72) and may contribute significantly to the tenacious binding of BPTI to trypsin. However, these hydrophilic residues might hamper the docking of BPTI variants with HNE. In support of this hypothesis, BI-8e displays a high affinity for HNE and contains no charged residues in the region spanning residues 39-42. Hence residues 39 through 42 of wild type BPTI were replaced with the corresponding residues of the human homologue of BI-8e. We anticipated that a BPTI derivative containing the MET-GLY-ASN-GLY (MGNG) sequence (SEQ ID NO:12) would exhibit a higher affinity for HNE than corresponding derivatives which retain the sequence of wild type BPTI at residues 39-42.","A double stranded oligonucleotide with AccI and EagI compatible ends was designed to introduce the desired alteration of residues 39 to 42 via cassette mutagenesis. Codon 45 was altered to create a new XmnI site, unique in the structure of the BPTI gene, which could be used to screen for mutants. This alteration at codon 45 does not alter the encoded amino-acid sequence. BPTI-III MA Rf DNA was digested with AccI. Two oligonucleotides (CYSB and CYST) corresponding to the bottom and top strands of the mutagenic DNA were annealed and ligated to the AccI digested BPTI-III MA Rf DNA. The sample was digested with BglII and the 2.1 kb BglII\/EagI fragment was purified. BPTI-III MA Rf was also digested with BglII and EagI and the 6.0 kb fragment was isolated and ligated to the 2.1 kb BglII\/EagI fragment described above. Ligation samples were used to transfect competent cells which were plated to permit the formation of plaques on a lawn of cells. Phage derived from plaques were probed with a radioactively labelled oligonucleotide (CYSB) using the Dot Blot Procedure. Positive clones were identified by autoradiography of the Nytran membrane after washing at high stringency conditions. Rf DNA was prepared from APR cultures containing fusion phage which hybridized to the CYSB probe. Restriction enzyme analysis and DNA sequencing confirmed that codons 39-42 of BPTI had been altered. The Rf DNA was designated BPTI(MGNG)-III MA (The amino acid sequence MGNG has SEQ ID NO:12; BPTI (. . . ,MGNG)-III MA (MGNG has SEQ ID NO:12) denotes a strain of M13 that displays BPTI (. . . , MGNG) (MGNG has SEQ ID NO:12) fused to the gIII protein and that carries the bla gene that confers Ap).","9) Construction of BPTI(K15L,MGNG)-III MA (MGNG has SEQ ID NO:12)","BPTI(MGNG)-III (MGNG has SEQ ID NO:12) MA Rf DNA was digested with AccI and the 5.6 kb fragment was purified. BPTI(K15L)-III MA was digested with AccI and the 2.5 kb DNA fragment was purified. The two fragments above were ligated together and ligation samples were used to transfect competent cells which were plated for plaque production. Large and small plaques were observed on the plate. Representative plaques of each type were picked and phage were probed with the LEU1 oligonucleotide via the Dot Blot Procedure. After the Nytran filter had been washed under high stringency conditions, positive clones were identified by autoradiography. Only the phage which hybridized to the LEU1 oligonucleotide gave rise to the small plaques confirming an earlier observation that substitution of LEU for LYSis substantially reduces phage infectivity. Appropriate cultures containing phage which hybridized to the LEU1 oligonucleotide were used to prepare Rf DNA. Restriction enzyme analysis and DNA sequencing confirmed that the K15L mutation had been introduced into BPTI(MGNG)-III MA (MGNG has SEQ ID NO:12). This Rf DNA was designated BPTI(K15L,MGNG)-III MA (MGNG has SEQ ID NO:12).","10) Effect of Mutation of Residues 39-42 of BPTI(K15L) on its Affinity for Immobilized HNE","Thirty \u03bcl of BPTI(K15L,MGNG)-III MA phage (9.2\u00b710pfu\/ml in TBS\/BSA) (MGNG has SEQ ID NO:12) were added to 5 \u03bcl of a 50% slurry of immobilized HNE also in TBS\/BSA. Similarly 30 \u03bcl of BPTI(K15L)-III MA phage (1.2\u00b710pfu\/ml in TBS\/BSA) were added to immobilized HNE. The samples were incubated for 3 hours on a Labquake shaker. The beads were washed for 5 min with 0.5 ml of TBS\/BSA and recovered by centrifugation. The beads were washed 5 times with 0.5 ml of TBS\/0.1% Tween-20 as described above. Finally, the beads were washed sequentially with a series of 100 mM sodium citrate buffers of pH 7.0, 6.0, 5.5, 5.0, 4.75, 4.5, 4.25, 4.0 and 3.5 as described above. pH washes were neutralized, diluted in LB broth and titered for plaque-forming units on a lawn of cells.","Table 204 illustrates that almost twice as much of the BPTI(K15L,MGNG)-III MA (MGNG has SEQ ID NO:12) as BPTI(K15L)-III MA phage bound to HNE beads. In both cases the pH 4.75 fraction contained the largest proportion of the recovered phage. This confirms that replacement of residues 39-42 of wild type BPTI with the corresponding residues of BI-8e enhances the binding of the BPTI(K15L) variant to HNE.","11) Fractionation of a Mixture of BPTI-III MK and BPTI(K15L,MGNG)-III MA Fusion Phage (MGNG has SEQ ID NO:12)","The observations described above indicate that BPTI(K15L,MGNG)-III MA (MGNG has SEQ ID NO:12) and BPTI-III MK phage exhibit different pH elution profiles from immobilized HNE. It seemed plausible that this property could be exploited to fractionate a mixture of different fusion phage.","Fifteen \u03bcl of BPTI-III MK phage (3.92\u00b710pfu\/ml in TBS\/BSA), equivalent to 8.91\u00b710Kmtransducing units, were added to 15 \u03bcl of BPTI(K15L,MGNG)-III MA (MGNG has SEQ ID NO:12) phage (9.85\u00b710pfu\/ml in TBS\/BSA), equivalent to 4.44\u00b710Aptransducing units. Five \u03bcl of a 50% slurry of immobilized HNE in TBS\/BSA was added to the phage and the sample was incubated for 3 hours on a Labquake mixer. The beads were washed for 5 minutes with 0.5 ml of TBS\/BSA prior to being washed 5 times with 0.5 ml of TBS\/2.0% Tween-20 as described above. Beads were washed for 5 minutes with 0.5 ml of 100 mM sodium citrate, pH 7.0 containing 1.0 mg\/ml BSA. The beads were recovered by centrifugation and the supernatant was removed. Subsequently, the HNE beads were washed sequentially with a series of 100 mM citrate buffers of pH 6.0, 5.0 and 4.0. The pH washes were neutralized by the addition of 130 \u03bcl of 1 M Tris, pH 8.0.","The relative proportion of BPTI-III MK and BPTI(K15L,MGNG)-III MA (MGNG has SEQ ID NO:12) phage in each pH fraction was evaluated by determining the number of phage able to transduce cells to Kmas opposed to Ap. Fusion phage diluted in 1\u00d7 Minimal A salts were added to 100 \u03bcl of cells (O.D.600=0.8 concentrated to 1\/20 original culture volume) also in Minimal salts in a final volume of 200 \u03bcl. The sample was incubated for 15 min at 37\u00b0 C. prior to the addition of 200 \u03bcl of 2\u00d7LB broth. After an additional 15 min incubation at 37\u00b0 C., duplicate aliquots of cells were plated on LB plates containing either Ap or Km to permit the formation of colonies. Bacterial colonies on each type of plate were counted and the data was used to calculate the number of Apand Kmtransducing units in each pH fraction. The number of Aptransducing units is indicative of the amount of BPTI(K15L,MGNG)-III MA (MGNG has SEQ ID NO:12) phage in each pH fraction while the total number of Kmtransducing units is indicative of the amount of BPTI-III MK phage.","Table 205 illustrates that a low percentage of the BPTI-III MK input phage (as judged by Kmtransducing units) adhered to the HNE beads and was recovered predominantly in the pH 7.0 fraction. By contrast, a significantly higher percentage of the BPTI(K15L,MGNG)-III MA phage (MGNG has SEQ ID NO:12) (as judged by Aptransducing units) adhered to the HNE beads and was recovered predominantly in the pH 4.0 fraction. A comparison of the total number of Apand Kmtransducing units in the pH 4.0 fraction shows that a 984-fold enrichment of BPTI(K15L,MGNG)-III MA (MGNG has SEQ ID NO:12) phage over BPTI-III MK phage was achieved. Hence, the above procedure can be utilized to fractionate mixtures of fusion phage on the basis of their relative affinities for immobilized HNE.","12) Construction of BPTI(K15V,R17L)-III MA","A BPTI variant containing the alterations K15V and R17L demonstrates the highest affinity for HNE of any BPTI variant described to date (K=6\u00b710M) (AUER89). As a means of testing the selection system described herein, a fusion phage displaying this variant of BPTI was generated and used as a \u201creference\u201d phage to characterize the affinity for immobilized HNE of fusion phage displaying a BPTI variant with a known affinity for free HNE. A 76 bp mutagenic oligonucleotide (VAL1) was designed to convert the LYScodon (AAA) to a VAL codon (GTT) and the ARGcodon (CGA) to a LEU codon (CTG). At the same time codons 11, 12 and 13 were altered to destroy the ApaI site resident in the wild type BPTI gene while creating a new RsrII site, which could be used to screen for correct clones.","The single stranded VAL1 oligonucleotide was converted to the double stranded form following the procedure described in Current Protocols in Molecular Biology (AUSU87). One \u03bcg of the VAL1 oligonucleotide was annealed to one \u03bcg of a 20 bp primer (MB8). The sample was heated to 80\u00b0 C., cooled to 62\u00b0 C. and incubated at this temperature for 30 minutes before being allowed to cool to 37\u00b0 C. Two \u03bcl of a 2.5 mM mixture of dNTPs and 10 units of Sequenase (U.S.B., Cleveland, Ohio) were added to the sample and second strand synthesis was allowed to proceed for 45 minutes at 37\u00b0 C. One hundred units of XhoI was added to the sample and digestion was allowed to proceed for 2 hours at 37\u00b0 C. in 100 \u03bcl of 1\u00d7 XhoI digestion buffer. The digested DNA was subjected to electrophoreses on a 4% GTG NuSieve agarose (FMC Bioproducts, Rockland, Me.) gel and the 65 bp fragment was excised and purified from melted agarose by phenol extraction and ethanol precipitation. A portion of the recovered 65 bp fragment was subjected to electrophoresis on a 4% GTG NuSieve agarose gel for quantitation. One hundred nanograms of the recovered fragment was dephosphorylated with 1.9 \u03bcl of HK\u2122 phosphatase (Epicentre Technologies, Madison, Wis.) at 37\u00b0 C. for 60 minutes. The reaction was stopped by heating at 65\u00b0 C. for 15 minutes. BPTI-MA Rf DNA was digested with XhoI and StuI and the 8.0 kb fragment was isolated. One \u03bcl of the dephosphorylation reaction (5 ng of double-stranded VAL1 oligonucleotide) was ligated to 50 ng of the 8.0 kb XhoI\/StuI fragment derived from BPTI-III MA Rf. Ligation samples were subjected to phenol extraction and DNA was recovered by ethanol precipitation. Portions of the recovered ligation DNA were added to 40 \u03bcl of electro-competent cells which were shocked using a Bio-Rad Gene Pulser device set at 1.7 kv, 25 \u03bcF and 800 \u03a9. One ml of SOC media was immediately added to the cells which were allowed to recover at 37\u00b0 C. for one hour. Aliquots of the electroporated cells were plated onto LB plates containing Ap to permit the formation of colonies.","Phage contained within cultures derived from picked Apcolonies were probed with two radiolabelled oligonucleotides (PRP1 and ESP1) via the Dot Blot Procedure. Rf DNA was prepared from cultures containing phage which exhibited a strong hybridization signal with the ESP1 oligonucleotide but not with the PRP1 oligonucleotide. Restriction enzyme analysis verified loss of the ApaI site and acquisition of a new RsrII site diagnostic for the changes in the P1 region. Fusion phage were also probed with a radiolabelled oligonucleotide (VLP1) via the Dot Blot Procedure. Autoradiography confirmed that fusion phage which previously failed to hybridize to the PRP1 probe, hybridized to the VLP1 probe. DNA sequencing confirmed that the LYSand ARGcodons had been converted to VAL and LEU codons respectively. The Rf DNA was designated BPTI(K15V,R17L)-III MA.","13) Affinity of BPTI(K15V,R17L)-III MA Phage for Immobilized HNE","Forty \u03bcl of BPTI(K15,R17L)-III MA phage (9.8\u00b710pfu\/ml) in TBS\/BSA were added to 10 \u03bcl of a 50% slurry of immobilized HNE also in TBS\/BSA. Similarly, 40 \u03bcl of BPTI(K15L,MGNG)-III MA (MGNG has SEQ ID NO:12) phage (5.13\u00b710pfu\/ml) in TBS\/BSA were added to immobilized HNE. The samples were mixed for 1.5 hours on a Labquake shaker. Beads were washed once for 5 min with 0.5 ml of TBS\/BSA and then 5 times with 0.5 ml of TBS\/1.0% Tween-20 as described previously. Subsequently the beads were washed sequentially with a series of 50 mM sodium citrate buffers containing 150 mM NaCl, 1.0 mg\/ml BSA of pH 7.0, 6.0, 5.0, 4.5, 4.0, 3.75, 3.5 and 3.0. In the case of the BPTI(K15L,MGNG)-III MA (MGNG has SEQ ID NO:12) phage, the pH 3.75 and 3.0 washes were omitted. Two washes were performed at each pH and the supernatants were pooled, neutralized with 1 M Tris pH 8.0, diluted in LB broth and titered for plaque-forming units on a lawn of cells.","Table 206 illustrates that the pH 4.5 and 4.0 fractions contained the largest proportion of the recovered BPTI(K15V,R17L)-III MA phage. By contrast, the BPTI(K15L,MGNG)-III MA (MGNG has SEQ ID NO:12) phage, like BPTI(K15L)-III MA phage, were recovered predominantly in the pH 5.0 and 4.5 fractions, as shown above. The affinity of BPTI(K15V,R17L) is 48 times greater than that of BPTI(K15L) for HNE (based on reported Kvalues, AUER89 for BPTI(K15V,R17L) and BECK88b for BPTI(K15L)). That the pH elution profile for BPTI(K15V,R17L)-III MA phage exhibits a peak at pH 4.0 while the profile for BPTI(K15L)-III MA phage displays a peak at pH 4.5 supports the contention that lower pH conditions are required to dissociate, from immobilized HNE, fusion phage displaying a BPTI variant with a higher affinity for free HNE.","Construction of a Variegated Population of Phage Displaying BPTI Derivates and Fractionation for Members that Display Binding Domains Having High Affinity for Human Neutrophil Elastase:","We here describe generation of a library of 1000 different potential engineered protease inhibitiors (PEPIs) and the fractionation with immobilized HNE to obtain an engineered protease inhibitor (Epi) having high affinity for HNE. Successful Epis that bind HNE are designated EpiNEs.","1) Design of a Mutagenic Oligonucleotide to Create a Library of Fusion Phage","A 76 bp variegated oligonucleotide (MYMUT) was designed to construct a library of fusion phage displaying 1000 different PEPIs derived from BPTI. The oligonucleotide contains 1728 different DNA sequences but due to the degeneracy of the genetic code, it encodes 1000 different protein sequences. The oligonucleotide was designed so as to destroy an ApaI site (shown in Table 113) encompassing codons 12 and 13. ApaI digestion could be used to select against the parental Rf DNA used to construct the library.","The MYMUT oligonucleotide permits the substitution of 5 hydrophobic residues (PHE, LEU, ILE, VAL, and MET via a DTS codon (D=approximately equimolar A, T, and G; S=approximately equimolar C and G)) for LYS. Replacement of LYSin BPTI with aliphatic hydrophobic residues via semi-synthesis has provided proteins having higher affinity for HNE than BPTI (TANK77, JERI74a,b, WENZ80, TSCH86, BECK88b). At position 16, either GLY or ALA are permitted (GST codon). This is in keeping with the predominance of these two residues at the corresponding positions in a variety of BPTI homologues (CREI87). The variegation scheme at position 17 is identical to that at 15. Limited data is available on the relative contribution of this residue to the interaction of BPTI homologues with HNE. A variety of hydrophobic residues at position 17 was included with the anticipation that they would enhance the docking of a BPTI variant with HNE. Finally at positions 18 and 19, 4 (PHE, SER, THR, and ILE via a WYC codon (W=approximately equimolar A and T; Y=approximately equimolar T and C)) and 5 (SER, PRO, THR, LYS, GLN, and stop via an HMA codon (H=approximately equimolar A, C, and T; M=approximately equimolar A and C)) different amino acids respectively are encoded. These different amino acid residues are found in the corresponding positions of BPTI homologues that are known to bind to HNE (CREI87). Although the amino acids included in the PEPI library were chosen because there was some indication that they might facilitate binding to HNE, it was not and is not possible to predict which combination of these amino acids will lead to high affinity for HNE. The mutagenic oligonucleotide MYMUT was synthesized by Genetic Design Inc. (Houston, Tex.).","2) Construction of Library of Fusion Phage Displaying Potential Engineered Protease Inhibitors","The single-stranded mutagenic MYMUT DNA was converted to the double stranded form with compatible XhoI and StuI ends and dephosphorylated with HK\u2122 phosphatase as described above for the VAL1 oligonucleotide. BPTI(MGNG)-III MA Rf DNA (MGNG has SEQ ID NO:12) was digested with XhoI and StuI for 3 hours at 37\u00b0 C. to ensure complete digestion. The 8.0 kb DNA fragment was purified by agarose gel electrophoresis and Ultrafree-MC unit filtration. One \u03bcl of the dephosphorylated MYMUT DNA (5 ng) was ligated to 50 ng of the 8.0 kb fragment derived from BPTI(MGNG)-III MA Rf DNA (MGNG has SEQ ID NO:12). Under these conditions, the 10:1 molar ratio of insert to vector was found to be optimal for the generation of transformants. Ligation samples were extracted with phenol, phenol\/chloroform\/IAA (25:24:1, v:v:v) and chloroform\/IAA (24:1, v:v) and DNA was ethanol precipitated prior to electroporation. One \u03bcl of the recovered ligation DNA was added to 40 \u03bcl of electro-competent cells. Cells were shocked using a Bio-Rad Gene Pulser device as described above. Immediately following electroshock, 1.0 ml of SOC media was added to the cells which were allowed to recover at 37\u00b0 C. for 60 minutes with shaking. The electroporated cells were plated onto LB plates containing Ap to permit the formation of colonies.","To assess the efficiency of the cassette mutagenesis procedure, 39 transformants were picked at random and phage present in culture supernatants were applied to a Nytran membrane and probed using the Dot Blot Procedure. Two Nytran membranes were prepared in this manner. The first filter was allowed to hybridize to the CYSB oligonucleotide which had previously been radiolabelled. The second membrane was allowed to hybridize to the PRP1 oligonucleotide which had also been radiolabelled. Filters were subjected to autoradiography following washing under high stringency conditions. Of the 39 phage samples applied to the membrane, all 39 hybridized to the CYSB probe. This indicated that there was fusion phage in the culture supernatants and that at least the DNA encoding residues 35-47 appeared to be present in the phage genomes. Only 11 of the 39 samples hybridized to the PRP1 oligonucleotide indicating that 28% of the transformants were probably the parental phage BPTI(MGNG)-III MA (MGNG has SEQ ID NO:12) used to generate the library. The remaining 28 clones failed to hybridize to the PRP1 probe indicating that substantial alterations were introduced into the P1 region by cassette mutagenesis using the MYMUT oligonucleotide. Of these 28 samples, all were found to contain infectious phage indicating that mutagenesis did not result in frame shift mutations which would lead to the generation of defective gene III products and non-infectious phage. (These 28 PEPI-displaying phage constitute a mini-library, the fractionation of which is discussed below.) Hence the overall efficiency of mutagenesis was estimated to be 72% in those cases where ligation DNA was not subjected to ApaI digestion prior to electroporation.","Bacterial colonies were harvested by overlaying chilled LB plates containing Ap with 5 ml of ice cold LB broth and scraping off cells using a sterile glass rod. A total of 4899 transformants were harvested in this manner of which 3299 were obtained by electroporation of ligation samples which were not digested with ApaI. Hence we estimate that 72% of these transformants (i.e. 2375) represent mutants of the parental BPTI(MGNG)-III MA (MGNG has SEQ ID NO:12) phage derived by cassette mutagenesis of the P1 position. An additional 1600 transformants were obtained by electroporation of ligation samples which had been digested with ApaI. If we assume that all of these clones contain new sequences at the P1 position then the total number of mutants in the pool of 4899 transformants is estimated to be 2375+1600=3975. The total number of potentially different DNA sequences in the MYMUT library is 1728. We calculate that the library should display about 90% of the potential engineered protease inhibitor sequences as follows:",{"@attributes":{"id":"p-0705","num":"0981"},"maths":[{"@attributes":{"id":"MATH-US-00003","num":"00003"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mrow":[{"msup":{"mo":"\u2009","mi":"N"},"mo":"\u2062","mi":"displayed"},{"mrow":[{"mrow":[{"msup":{"mo":"\u2009","mi":"N"},"mo":"\u2062","mi":"possible"},{"mo":["(",")"],"mrow":{"mn":"1","mo":"-","mrow":{"mi":"exp","mo":"\u2062","mrow":{"mo":["{","}"],"mrow":{"mrow":[{"mo":"-","mi":"Libsize"},{"mi":"N","mo":"\u2061","mrow":{"mo":["(",")"],"mi":"DNA"}}],"mo":"\/"}}}}}],"mo":"\u00b7"},{"mrow":{"mn":"1000","mo":"\u00b7","mrow":{"mo":["(",")"],"mrow":{"mn":"1","mo":"-","mrow":{"mi":"exp","mo":"\u2062","mrow":{"mo":["{","}"],"mrow":{"mrow":{"mo":"-","mn":"3975"},"mo":"\/","mn":"1728"}}}}}},"mo":"=","mn":"900"}],"mo":"="}],"mo":"="}}},{"@attributes":{"id":"MATH-US-00003-2","num":"00003.2"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mstyle":[{"mspace":{"@attributes":{"width":"3.1em","height":"3.1ex"}}},{"mspace":{"@attributes":{"width":"0.6em","height":"0.6ex"}}}],"mo":["\u2062","\u2062"],"mrow":{"mrow":[{"mi":["%","of","possible","sequences","displayed"],"mo":["\u2062","\u2062","\u2062","\u2062","\u2062","\u2062","\u2062","\u2062"],"mstyle":[{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}},{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}},{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}},{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}}]},{"mrow":[{"mn":"100","mo":"\u00b7","mrow":{"mo":["(",")"],"mrow":{"mn":["900","1000"],"mo":"\u00f7"}}},{"mn":"90","mo":"\u2062","mi":"%"}],"mo":"="}],"mo":"="}}}}],"br":{}},"We studied the fractionation of the mini library of 28 PEPIs to establish the appropriate parameters for fractionation of the entire MYMUT PEPI library. We anticipated that fractionation could be easier when the library of fusion phage was much less diverse than the entire MYMUT library. Fewer cycles of fractionation might be required to affinity purify a fusion phage exhibiting a high affinity for HNE. Secondly, since the sequences of all the fusion phage in the mini-library can be determined, one can determine the probability of selecting a given fusion phage from the initial population.","Two ml of the culture supernatants of the 28 PEPIs described above were pooled. Fusion phage were recovered, resuspended in 300 mM NaCl, 100 mM Tris, pH 8.0, 1 mM EDTA and stored on ice for 15 minutes. Insoluble material was removed by centrifugation for 3 minutes in a microfuge at 4\u00b0 C. The supernatant fraction was collected and PEPI phage were precipitated with PEG-8000. The final phage pellet was resuspended in TBS\/BSA. Aliquots of the recovered phage were titered for plaque-forming units on a lawn of cells. The final stock solution consisted of 200 \u03bcl of fusion phage at a concentration of 5.6\u00b710pfu\/ml.","a) First Enrichment Cycle","Forty \u03bcl of the above phage stock was added to 10 \u03bcl of a 50% slurry of HNE beads in TBS\/BSA. The sample was allowed to mix on a Labquake shaker for 1.5 hours. Five hundred \u03bcl of TBS\/BSA was added to the sample and after an additional 5 minutes of mixing, the HNE beads were collected by centrifugation. The supernatant fraction was removed and the beads were resuspended in 0.5 ml of TBS\/0.5% Tween-20. Beads were washed for 5 minutes on the shaker and recovered by centrifugation as above. The supernatant fraction was removed and the beads were subjected to 4 additional washes with TBS\/Tween-20 as described above to reduce non-specific binding of fusion phage to HNE beads. Beads were washed twice as above with 0.5 ml of 50 mM sodium citrate pH 7.0, 150 mM NaCl containing 1.0 mg\/ml BSA. The supernatants from the two washes were pooled. Subsequently, the HNE beads were washed sequentially with a series of 50 mM sodium citrate, 150 mM NaCl, 1.0 mg\/ml BSA buffers of pH 6.0, 5.0, 4.5, 4.0, 3.5, 3.0, 2.5 and 2.0. Two washes were performed at each pH and the supernatants were pooled and neutralized by the addition of 260 \u03bcl of 1 M Tris, pH 8.0. Aliquots of each pH fraction were diluted in LB broth and titered for plaque-forming units on a lawn of cells. The total amount of fusion phage (as judged by pfu) appearing in each pH wash fraction was determined.",{"@attributes":{"id":"p-0709","num":"0985"},"figref":"FIG. 7","sub":"d","sup":"\u22129 "},"To enrich for fusion phage displaying the highest affinity for HNE, phage contained in the lowest pH fraction (pH 2.0) from the first enrichment cycle were amplified and subjected to a second round of fractionation. Amplification involved the Transduction Procedure described above. Fusion phage (2000 pfu) were incubated with 100 \u03bcl of cells for 15 minutes at 37\u00b0 C. in 200 \u03bcl of 1\u00d7 Minimal A salts. Two hundred \u03bcl of 2\u00d7LB broth was added to the sample and cells were allowed to recover for 15 minutes at 37\u00b0 C. with shaking. One hundred \u03bcl portions of the above sample were plated onto LB plates containing Ap. Five such transduction reactions were performed yielding a total of 20 plates, each containing approximately 350 colonies (7000 transformants in total). Bacterial cells were harvested as described for the preparation of the MYMUT library and fusion phage were collected as described for the preparation of the mini-library. A total of 200 \u03bcl of fusion phage (4.3\u00b710pfu\/ml in TBS\/BSA) derived from the pH 2.0 fraction from the first passage of the mini-library was obtained in this manner.","b) Second Enrichment Cycle","Forty \u03bcl of the above phage stock was added to 10 \u03bcl of a 50% slurry of HNE beads in TBS\/BSA. The sample was allowed to mix for 1.5 hours and the HNE beads were washed with TBS\/BSA, TBS\/0.5% Tween and sodium citrate buffers as described above. Aliqouts of neutralized pH fractions were diluted and titered as described above.","The elution profile for the second passage of the mini-library over HNE beads is shown in . The largest percentage of the input phage which bound to the HNE beads was recovered in the pH 3.5 wash. A smaller peak centered on pH 4.5 may represent residual fusion phage from the first passage of the mini-library which eluted at pH 5.0. The percentage of total input phage which eluted at pH 3.5 in the second cycle exceeds the percentage of input phage which eluted at pH 5.0 in the first cycle. This is indicative of more avid binding of fusion phage to the HNE matrix. Taken together, the significant shift in the pH elution profile suggests that selection for fusion phage displaying BPTI variants with higher affinity for HNE occurred.","c) Third Cycle","Phage obtained in the pH 2.0 fraction from the second passage of the mini-library were amplified as above and subjected to a third round of fractionation. The pH elution profile is shown in . The largest percentage of input phage was recovered in the pH 3.5 wash as is the case with the second passage of the mini-library. However, the minor peak centered on pH 4.5 is diminished in the third passage relative to the second passage. Furthermore, the percentage of input phage which eluted at pH 3.5 is greater in the third passage than in the second passage. In comparison, the BPTI(K15V,R17L)-III fusion phage elute from HNE beads as a peak centered on pH 4.25. Taken together, the data suggests that a significant selection for fusion phage displaying PEPIs with high affinity for HNE occurred. Furthermore, since more extreme pH conditions are required to elute fusion phage in the third passage of the MYMUT library relative to those conditions needed to elute BPTI(K15V,R17L)-III MA phage, this suggests that those fusion phage which appear in the pH 3.5 fraction may display a PEPI with a higher affinity for HNE than the BPTI(K15V,R17L) variant (i.e. K<6\u00b710M).","d) Characterization of Selected Fusion Phage","The pH 2.0 fraction from the third passage of the mini-library was titered and plaques were obtained on a lawn of cells. Twenty plaques were picked at random and phage derived from plaques were probed with the CYSB oligonucleotide via the Dot Blot Procedure. Autoradiography of the filter revealed that all 20 samples gave a positive hybridization signal indicating that fusion phage were present and the DNA encoding residues 35 to 47 of BPTI(MGNG) (MGNG has SEQ ID NO:12) is contained within the recombinant M13 genomes. Rf DNA was prepared for the 20 clones and initial dideoxy sequencing revealed that 12 clones were identical. This sequence was designated EpiNE\u03b1 (SEQ ID NO:244 and SEQ ID NO:108) (Table 207). No DNA sequence changes were observed apart from the planned variegation. Hence the cassette mutagenesis procedure preserved the context of the planned variegation of the pepi gene. The Dot Blot Procedure was employed to probe all 20 selected clones from the pH 2.0 fraction from the third passage of the mini-library with an oligonucleotide homologous to the sequence of EpiNE\u03b1. Following high stringency washing, autoradiography revealed that all 20 selected clones were identical in the P1 region. Furthermore dot blot analysis revealed that of the 28 different phage samples pooled to create the mini-library, only one contained the EpiNE\u03b1 sequence. Hence in just three passes of the mini-library over HNE beads, 1 out of 28 input fusion phage was selected for and appears as a pure population in the lowest pH fraction from the third passage of the library. That the EpiNE\u03b1 phage elute at pH 3.5 while BPTI(K15V,R17L)-III MA phage elute at a higher pH strongly suggests that the EpiNE\u03b1 protein has a significantly higher affinity than BPTI(K15V,R17L) for HNE.","4) Fractionation of the MYMUT Library","a) Three Cycles of Enrichment","The same procedure used above to fractionation the mini-library was used to fractionate the entire MYMUT PEPI library consisting of fusion phage displaying 1000 different proteins. The phage inputs for the first, second and third rounds of fractionation were 4.0\u00b710, 5.8\u00b710, and 1.1\u00b710pfu respectively.  illustrates that the largest percentage of input phage which bound to the HNE matrix was recovered in the pH 5.0 wash in the first enrichment cycle. The pH elution profile is very similar to that seen for the first passage of the mini-library over HNE beads. A trailing edge is also observed on the low pH side of the pH 5.0 peak however this is not as prominent as that observed for the mini-library. The percentage of input phage which eluted in the pH 7.0 wash was greater than that eluted in the pH 6.0 wash. This is in contrast to the result obtained for the first passage of the mini library and may reflect the presence of \u224820% parental BPTI(MGNG)-III MA (MGNG has SEQ ID NO:12) phage in the MYMUT library pool. These phage adhere to the HNE beads weakly (if at all) and elute in the pH 7.0 fraction. That no parent phage were present in the mini-library is consistent with the absence of a peak at pH 7.0 in the first passage of the mini-library.","Phage present in the pH 2.0 fraction from the first passage of the MYMUT library were amplified as described previously and subjected to a second round of fractionation. The largest percentage of input phage which bound to the HNE beads was recovered in the pH 3.5 wash (). A minor peak centered on pH 4.5 was also evident. The fact that more extreme pH conditions were required to elute the majority of bound fusion phage suggested that selection of fusion phage displaying PEPIs with higher affinity for HNE had occurred. This was also indicated by the fact that the total percentage of input phage which appeared in the pH 3.5 wash in the second enrichment cycle was 10 times greater than the percentage of input which appeared in the pH 5.0 wash in the first cycle.","Fusion phage from the pH 2.0 fraction of the second pass of the MYMUT library were amplified and subjected to a third passage over HNE beads. The proportion of fusion phage appearing in the pH 3.5 fraction relative to that in the 4.5 fraction was greater in the third passage than in the second passage (). Also the amount of fusion phage appearing in the pH 3.5 fraction was higher in the third passage than in the second passage. The fact that wash conditions less than pH 4.25 were required to elute bound fusion phage derived from the MYMUT library suggests that the EpiNEs displayed by these phage possess a higher affinity for HNE than the BPTI(K15V,R17L) variant.","b) Characterization of Selected Clones","The pH 2.0 fraction from the third enrichment cycle of the MYMUT library was titered on a lawn of cells. Twenty plaques were picked at random. Rf DNA was prepared for each of the clones and fusion phage were collected by PEG precipitation. Clonally pure populations of fusion phage in TBS\/BSA were prepared and characterized with respect to their affinity for immobilized HNE. pH elution profiles were obtained to determine the stringency of the conditions required to elute bound fusion phage from the HNE matrix.  illustrates the pH profiles obtained for EpiNE clones 1, (SEQ ID NO:51), 3, (SEQ ID NO:46), and 7 (SEQ ID NO:48). The pH profiles for all 3 clones exhibit a peak centered on pH 3.5. Unlike the pH profile obtained for the third passage of the MYMUT library, no minor peak centered on pH 4.5 is evident. This is consistent with the clonal purity of the selected EpiNE phage utilized to generate the profiles. The elution peaks are not symmetrical and a prominent trailing edge on the low pH side. In all probability, the 10 minute elution period employed is inadequate to remove bound fusion phage at the low pH conditions. EpiNE clones 1 through 8 have the following characteristics: five clones (identified as EpiNE1 (SEQ ID NO:51), EpiNE3 (SEQ ID NO:46), EpiNE5 (SEQ ID NO:52), EpiNE6 (SEQ ID NO:47), and EpiNE7 (SEQ ID NO:48)) display very similar pH profiles centered on pH 3.5. The remaining 3 clones elute in the pH 3.5 to 4.0 range. There remains some diversity amongst the 20 randomly chosen clones obtained from the pH 2.0 fraction of the third passage of the MYMUT library and these clones might exhibit different affinities for HNE.","c) Sequences of the EpiNE Clones","The DNA sequences encoding the P1 regions of the different EpiNE clones were determined by dideoxy sequencing of Rf DNA. The sequences are shown in Table 208. Essentially, only the codons targeted for mutagenesis (i.e. 15 to 19) were altered as a consequence of cassette mutagenesis using the MYMUT oligonucleotide. Only 1 codon outside the target region was found to contain an unexpected alteration. In this case, codon 21 of EpiNE8 was altered from a tyrosine codon (TAT) to a SER codon (TCT) by a single nucleotide substitution. This error could have been introduced into the MYMUT oligonucleotide during its synthesis. Alternatively, an error could have been introduced when the single-stranded MYMUT oligonucleotide was converted to the double-stranded form by Sequenase. Regardless of the reason, the error rate is extremely low considering only 1 unexpected alteration was observed after sequencing 20 codons in 19 different clones. Furthermore, the value of such a mutation is not diminished by its accidental nature.","Some of the EpiNE clones are identical. The sequences of EpiNE1, EpiNE3, and EpiNE7 appear a total of 4, 6 and 5 times respectively. Assuming the 1745 potentially different DNA sequences encoded by the MYMUT oligonucleotide were present at equal frequency in the fusion phage library, the frequent appearance of the sequences for clones EpiNE1, EpiNE3, and EpiNE7 may have important implications. EpiNE1, EpiNE3, and EpiNE7 fusion phage may display BPTI variants with the highest affinity for HNE of all the 1000 potentially different BPTI variants in the MYMUT library.","An examination of the sequences of the EpiNE clones is illuminating. A strong preference for either VAL or ILE at the P1 position (residue 15) is indicated with VAL being favored over ILE by 14 to 6. In the MYMUT library, VAL at position 15 is approximately twice as prevalent as ILE. No examples of LEU, PHE, or MET at the P1 position were observed although the MYMUT oligonucleotide has the potential to encode these residues at P1. This is consistent with the observation that BPTI variants with single amino acid substitutions of LEU, PHE, or MET for LYSexhibit a significantly lower affinity for HNE than their counterparts containing either VAL or ILE (BECK88b).","PHE is strongly favored at position 17, appearing in 12 of 20 codons. MET is the second most prominent residue at this position but it only appears when VAL is present at position 15. At position 18 PHE was observed in all 20 clones sequenced even though the MYMUT oligonucleotide is capable of encoding other residues at this position. This result is quite surprising and could not be predicted from previous mutational analysis of BPTI, model building, or on any theoretical grounds. We infer that the presence of PHE at position 18 significantly enhances the ability each of the EpiNEs to bind to HNE. Finally at position 19, PRO appears in 10 of 20 codons while SER, the second most prominent residue, appears at 6 of 20 codons. Of the residues targeted for mutagenesis in the present study, residue 19 is the nearest to the edge of the interaction surface of a PEPI with HNE. Nevertheless, a preponderance of PRO is observed and may indicate that PRO at 19, like PHE at 18, enhances the binding of these proteins to HNE. Interestingly, EpiNE5 appears only once and differs from EpiNE1 only at position 19; similarly, EpiNE6 differs from EpiNE3 only at position 19. These alterations may have only a minor effect on the ability of these proteins to interact with HNE. This is supported by the fact that the pH elution profiles for EpiNE5 and EpiNE6 are very similar to those of EpiNE1 and EpiNE3 respectively.","Only EpiNE2 and EpiNE8 exhibit pH profiles which differ from those of the other selected clones. Both clones contain LYS at position 19 which may restrict the interaction of BPTI with HNE. However, we can not exclude the possibility that other alterations within EpiNE2 and EpiNE8 (R15L and Y21S respectively) influence their affinity for HNE.","EpiNE7 was expressed as a soluble protein and analyzed for HNE inhibition activity by the fluorometric assay of Castillo et al. (CAST79); the data were analyzed by the method of Green and Work (GREE53). Preliminary results indicate that K(HNE,EpiNE7)\u22668.\u00b710M, i.e. at least 7.5-fold lower than the lowest Kreported for a BPTI derivative with restect to HNE.","Taken together, these data show that the alterations which appear in the P1 region of the EPI mutants confer the ability to bind to HNE and hence be selected through the fractionation process. That the sequences of EpiNE1, EpiNE3, and EpiNE7 appear frequently in the population of selected clones suggests that these clones display BPTI variants with the highest affinity for HNE of any of the 1000 potentially different variants in the MYMUT library. Furthermore, that pH conditions less than 4.0 are required to elute these fusion phage from immobilized HNE suggests that they display BPTI variants having a higher affinity for HNE than BPTI (K15V,R17L). EpiNE7 exhibits a lower Ktoward HNE than does BPTI(K15V,R17L); EpiNE1 and EpiNE3 should are also expected to exhibit lower Ks for HNE than BPTI(K15V,R17L). It is possible that all of the listed EpiNEs have lower Ks than BPRI(K15V,R17L).","Position 18 has not previously been identified as a key position in determining specificity or affinity of aprotinin homologues or derivatives for particular serine proteases. None have reported or suggested that phenylalanine at position 18 will confer specificity and high affinity for HNE. One of the powerful advantages of the present invention is that many diverse amino-acid sequences may be tested simultaneously.","We fractionated the MYMUT library over immobilized human Cathepsin G to find an engineered protease inhibitor having high affinity for Cathepsin G, hereafter designated as an EpiC. The details of phage binding, elution of bound phage with buffers of decreasing pH (pH profile), titering of the phage contained in these fractions, composition of the MYMUT library, and the preparation of cathepsin G (Cat G) beads are essentially the same as detailed in Example IV.","A pH profile for the binding of two starting controls, BPTI-III MK and EpiNE1, are shown in . BPTI-III MK phage, which contains wild type BPTI fused to the III gene product, shows no apparent binding to Cat G beads in this assay. EpiNE1 phage was obtained by enrichment with HNE beads (Example IV and Table 208). EpiNE1-III MK demonstrated little binding to Cat G beads in the assay, although a small peak or shoulder is visible in the pH 5 eluted fraction.",{"@attributes":{"id":"p-0729","num":"1005"},"figref":"FIG. 11"},"To investigate the elution profile around the pH 5 point in more detail, the binding of phage taken from the pH 4 eluted fraction (bound to Cat G beads) rather than the previously used pH 2 fraction was examined.  demonstrates a marked enhancement of phage binding to the Cat G beads with an apparent elution peak of pH 5. The binding, as a fraction of the input phage population, increased with subsequent binding and elution cycles.","Individual phage clones were picked, grown and analyzed for binding to Cat G beads.  shows the binding and pH profiles for the individual Cat G binding clones (designated EpiC variants). All clones exhibited minor peaks, superimposed upon a gradual fall in bound phage, at pH elutions of 5 (clones 1 (SEQ ID NOs:54 and 117), 8 (SEQ ID NOs:56 and 119), 10 (SEQ ID NOs:57 and 120) and 11 (SEQ ID NOs:54 and 117)) or pH 4.5 (clone 7 (SEQ ID NOs:55 and 118)).","DNA sequencing of the EpiC clones, shown in Table 209 (SEQ ID NOs:54 through 58 and 117 through 121), demonstrated that the clones selected for binding to Cat G beads represented a distinct subset of the available sequences in the MYMUT library and a cluster of sequences different from that obtained when enriched with HNE beads. The P1 residue in the EpiC mutants is predominantly MET, with one example of PHE, while in BPTI it is LYS and in the EpiNE variants it is either VAL or LEU. In the EpiC mutants residue 16 is predominantly ALA with one example of GLY and residue 17 is PHE, ILE or LEU. Interestingly residues 16 and 17 appear to pair off by complementary size, at least in this small sample. The small GLY residue pairs with the bulky PHE while the relatively larger ALA residue pairs with the less bulky LEU and ILE. The majority of the available residues in the MYMUT library for positions 18 and 19 are represented in the EpiC variants.","Hence, a distinct subset of related sequences from the MYMUT library have been selected for and demonstrated to bind to Cat G. A comparison of the pH profiles elicited for the EpiC variants with Cat G and the EpiNE variants for HNE indicates that the EpiNE variants have a high affinity for HNE while the EpiC variants have a moderate affinity for Cat G. Nonetheless, the starting molecule, BPTI, has virtually no detectable affinity for Cat G and the selection of clones with a moderate affinity is a significant finding.","A. Mutageneisis of EpiNE7 Protein in the Loop Comprising Residues 34-41","In Example IV, we described engineered protease inhibitors EpiNE1 through EpiNE8 (SEQ ID NOs:46 through 53 and 109-116) that were obtained by affinity selection. Modeling of the structure of the BPTI-Trypsin complex (Brookhaven Protein Data Bank entry 1TPA) indicates that the EpiNE protein surface that interacts with HNE is formed not only by residues 15-19 but also by residues 34-40 that are brought close to this primary loop when the protein folds (HUBE74, HUBE75, OAST88). Acting upon this assumption, we changed amino acid residues in a second loop of the EpiNE7 protein to find EpiNE7 (SEQ ID NO:48) derivatives having higher affinity for HNE.","In the complex of BPTI and trypsin found in Brookhaven Protein Data Bank entry 1TPA (\u201c1TPA complex\u201d), VALcontacts TYRand GLN. (Residues in trypsin or HNE are underscored to distinguish them from the inhibitor.) In HNE, the corresponding residues are ILE, and PHE. ILE is smaller and more hydrophobic than TYR. PHE is larger and more hydrophobic than GLN. Neither of the HNE side groups have the possibility to form hydrogen bonds. When side groups larger than that of VAL are substituted at position 34, interactions with residues other than 151 and 192 may be possible. In particular, an acidic residue at 34 might interact with ARGof HNE that corresponds to SERof trypsin in 1TPA. Table 15 shows that, in 59 homologues of BPTI, 13 different amino acids have been seen at position 34. Thus we allow all twenty amino acids at 34.","Position 36 is not highly varied; only GLY, SER, and ARG have been observed with GLY by far the most prevalent. In the 1TPA complex, GLYcontacts HISand GLN. HISis conserved and GLNcorresponds to PHEof HNE. Adding a methyl group to GLYcould increase hydrophobic interactions with PHEof HNE. GLYis in a conformation that most amino acids can achieve: \u03c6=\u221279\u00b0 and \u03c8=9\u00b0 (Deisenhoffer cited in CREI84, p. 222.).","In the 1TPA complex, ARGcontacts SER, ASN, THR, LEU(SEQ ID NO:13), GLN, and TRP. In HNE, all of the corresponding residues are different! SERis deleted; ASNcorresponds to ASP(bearing a negative charge); THRcorresponds to PRO; LEUcorresponds to the residues VAL, ASN, and LEU; GLNis deleted; and TRPcorresponds to PHE. Position 39 shows a moderately high degree of variability with 7 different amino acids observed, viz. ARG, GLY, LYS, GLN, ASP, PRO, and MET. Having seen PRO (the most rigid amino acid), GLY (the most flexible amino acid), LYS and ASP (basic and acidic amino acids), we assume that all amino acids are structurally compatible with the aprotinin backbone. Because the context of residue 39 has changed so much, we allow all 20 amino acids.","Position 40 is not highly variable; only GLY and ALA have been observed (with similar frequency, 24:16). Position 41 is moderately varied, showing ASN, LYS, ASP, GLN, HIS, GLU, and TYR. The side groups of residues 40 and 41 are not thought to contact trypsin in the 1TPA complex. Nevertheless, these residues can exert electrostatic effects and can influence the dynamic properties of residues 39, 38, and others. The choice of residues 34, 36, 39, 40, and 41 to be varied simultaneously illustrates the rule that the varied residues should be able to touch one molecule of the target material at one time or be able to influence residues that touch the target. These residues are not contiguous in sequence, nor are they contiguous on the surface of EpiNE7. They can, nonetheless, all influence the contacts between the EpiNE and HNE.","Amino acid residues VAL, GLY, MET, GLY, and ASNwere variegated as follows: any of 20 genetically encodable amino acids at positions 34 and 39 (NNS codons in which N is approximately equimolar A,C,T,G and S is approximately equimolar C and G), GLY or ALA at position 36 and 40 (GST codon), and [ASP, GLU, HIS, LYS, ASN, GLN, TYR, or stop] at position 41 (NAS codon). Because the PEPIs are displayed fused to gIII protein, DNA containing stop codons will not give rise to infectuous phage in non-suppressor hosts.","For cassette mutagenesis, a 61 base long oligonucleotide DNA population was synthesized that contained 32,768 different DNA sequences coding on expression for a total of 11,200 amino acid sequences. This oligonucleotide extends from the third base of codon 51 in Table 113 (the middle of the StuI site) to base 2 of codon 70 (the EagI site (identified as XmaIII in Table 113)).","We used a mutagenesis method similar to that described by Cwirla et al. (CWIR90) and other standard DNA manipulations described in Maniatis et al. (MANI82) and Sambrook et al. (SAMB89). EpiNE7 RF DNA was restricted with EagI and StuI, agarose gel purified, and dephosphorylated using HK\u2122 phosphatase (Epicentre Technologies). We prepared insert by annealing two small, 16 base and 17 base, phosphorylated synthetic DNA primers to the phosphorylated 61 base long oligonucleotide population described above. The resulting insert DNA population had the following features: double stranded DNA ends capable of regenerating upon ligation the EagI (5\u2032 overhang) and StuI (blunt) restricted sites of the EpiNE7 RF DNA, and single stranded DNA in the central mutagenic region. Insert and EpiNE7 vector DNA were ligated. Ligation samples were used to transfect competent XL1-Blue\u2122 cells which were subsequently plated for formation of ampicillin resistant (Ap) colonies. The resulting phage-producing, Apcolonies were harvested and recombinant phage was isolated. By following these procedures, a phage library of 1.2\u00b710independent transformants was assembled. We estimated that 97.4% of the approximately 3.3\u00b710possible DNA sequences were represented:\n\n0.974=(1\u2212exp{\u22121.2\u00b710\/32768}).\n\nThe probability of observing the parental sequence is higher than 0.974 because VAL occurs twice in the NNS codon:\n",{"@attributes":{"id":"p-0742","num":"1018"},"maths":{"@attributes":{"id":"MATH-US-00004","num":"00004"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mrow":[{"mi":["Probability","of"],"mo":["\u2062","\u2062","\u2062","\u2062"],"mstyle":[{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}},{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}}],"mrow":{"mi":"seeing","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msub":[{"mi":"V","mn":"34"},{"mi":"G","mn":"36"},{"mi":"M","mn":"39"},{"mi":"G","mn":"40"},{"mi":"N","mn":"41"}],"mo":[",",",",",",","]}}}},{"mo":"(","mrow":{"mrow":[{"mn":"1","mo":"-","mrow":{"mi":"exp","mo":"\u2062","mrow":{"mo":["{","}"],"mrow":{"mo":"-","mrow":{"mo":["(",")"],"mrow":{"mrow":[{"mn":"1.2","mo":"\u00b7","msup":{"mn":["10","5"]}},{"mn":["2","32768"],"mo":"\/"}],"mo":"\u00d7"}}}}}},{"mrow":[{"mo":["(",")"],"mrow":{"mn":"1","mo":"-","mrow":{"mi":"exp","mo":"\u2062","mrow":{"mo":["{","}"],"mrow":{"mo":"-","mn":"7.32"}}}}},{"mrow":{"mo":["(",")"],"mrow":{"mn":"1","mo":"-","mrow":{"mn":"6.5","mo":"\u00b7","msup":{"mn":"10","mrow":{"mo":"-","mn":"4"}}}}},"mo":"=","mn":"0.99934"}],"mo":"="}],"mo":"="}}],"mo":"="}}},"br":[{},{},{}]},"We added 1.1\u00b710plaque forming units of the KLMUT library to 10 \u03bcl of a 50% slurry of agarose-immobilized human neutrophil elastase beads (HNE from Calbiochem cross-linked to Reacti-Gel\u2122 agarose beads from Pierce Chemical Co. following manufacturer's directions) in TBS\/BSA. Following 3 hours incubation at room temperature, the beads were washed and phage was eluted as done in the selection of EpiNE phage isolates (Example IV). The progression in lowering pH during the elution was: pH 7.0, 6.0, 5.0, 4.5, 4.0, 3.5, 3.0, 2.5, and 2.0. Beads carrying phage remaining after pH 2.0 elution were used to infect XL1-Blue\u2122 cells that were plated to allow plaque formation. The 348 resulting plaques were pooled to form a phage population for further affinity selection. A population of phage particles containing 6.0\u00b710plaque forming units was added to 10 \u03bcl of a 50% slurry of agarose-immobilized HNE beads in TBS\/BSA and the above selection procedure was repeated.","Following this second round of affinity selection, a portion of the beads was mixed with XL1-Blue\u2122 cells and plated to allow plaque formation. Of the resulting plaques, 480 were pooled to form a phage population for a third affinity selection. We repeated the selection procedure described above using a population of phage particles containing 3.0\u00b710plaque forming units. Portions of the pH 2.0 eluate and of the beads were plated with XL1-Blue\u2122 cells to allow formation of plaques. Individual plaques were picked for preparation of RF DNA. From DNA sequencing, we determined the amino acid sequence in the mutated secondary loop of 15 EpiNE7-homolog clones. The sequences are given in Table 210 as EpiNE7.1 through EpiNE7.20 (SEQ ID NOs:59-70). Three sequences were observed twice: EpiNE7.4 and EpiNE7.14 (SEQ ID NO:63); EpiNE7.8 and EpiNE7.9 (SEQ ID NO:60); and EpiNE7.10 and EpiNE7.20 (SEQ ID NO:65). EpiNE7.4 was eluted at pH 2 while EpiNE7.14 was obtained by culturing HNE beads that had been washed with pH 2 buffer. Similarly, EpiNE7.10 came from pH 2 elution but EpiNE7.20 came from beads. EpiNE7.8 and EpiNE7.9 both came from pH 2 elution. Interestingly, EpiNE7.8 is found in both the first and second fractionations (EpiNE7.31 (vide infra)).","2) Second Fractionation","The purpose of affinity fractionation is to reduce diversity on the basis of affinity for the target. The first enrichment step of the first fractionation reduced the population from 3\u00b710possible DNA sequences to no more than 348. This might be too severe and some of the loss of diversity might not be related to affinity. Thus we carried out a second fractionation of the entire KLMUT library seeking to reduce the diversity more gradually.","We added 2.0\u00b710plaque forming units of the KLMUT library to 10 \u03bcl of a 50% slurry of agarose-immobilized HNE beads in TBS\/BSA. Following 3 hours incubation at room temperature, phage were eluted as described above. We then transduced XL1-Blue\u2122 cells with portions of the pH 2.0 eluate and plated for APR colonies.","The resulting phage-producing colonies were harvested to obtain amplified phage for further affinity selection. A population of these phage particles containing 2.0\u00b710plaque forming units was added to 10 \u03bcl of a 50% slurry of agarose-immobilized HNE beads in TBS\/BSA and incubated for 90 minutes at room temperature. Phage were eluted as described above and portions of the pH 2.0 eluate were used to transduce XL1-Blue\u2122 cells. We plated the transductants for Apcolonies and obtained amplified phage from the harvested colonies.","In a third round of affinity selection, a population of phage particles containing 3.0\u00b710plaque forming units was added to 20 \u03bcl of 50% slurry of agarose-immobilized HNE beads and incubated for 2 hours at room temperature. We eluted the phage with the following pH washes: pH 7.0, 6.0, 5.0, 4.5, 4.0, 3.5, 3.25, 3.0, 2.75, 2.5, 2.25, and 2.0. After plating a portion of the pH 2.0 eluate fraction for plaque formation, we picked individual plaques for preparation of RF DNA. DNA sequencing yielded the amino acid sequence in the mutated secondary loop for 20 EpiNE7 homolog clones. These sequences, together with EpiNE7 (SEQ ID NO:48), are given in Table 210 as EpiNE7.21 through EpiNE7.40 (SEQ ID NOs:71 through 87). The plaques observed when EpiNEs are plated display a variety of sizes. EpiNE7.21 through EpiNE7.30 (SEQ ID NOs:71 through 80) were picked with attention to plaque size: 7.21, 7.22, and 7.23 from small plaques, 7.24 through 7.30 from plaques of increasing size, with 7.30 coming from a large plaque. TRP occurs at position 39 in EpiNE7.21, 7.22, 7.23, 7.25, and 7.30. Thus plaque size does not correlate with the appearance of TRP at 39. One sequence, EpiNE7.31, from this fractionation is identical to sequences EpiNE7.8 and EpiNE7.9 obtained in the first fractionation. EpiNE7.30, EpiNE7.34, and EpiNE7.35 are identical, indicating that the diversity of the library has been greatly reduced. It is believed that these sequences have an affinity for HNE that is at least comparable to that of EpiNE7 and probably higher. Because the parental EpiNE7 sequence did not recur, it is quite likely that some or all of the EpiNE7.nn derivatives have higher affinity for HNE than does EpiNE7.","3) Conclusions","One can draw some conclusions. First, because some sequences have been isolated repeatedly, the fractionation is nearly complete. The diversity has been reduced from \u226710to a few tens of sequences.","Second, the parental sequence has not recurred. At 39, MET did not occur! At position 34 VAL occurred only once in 35 sequences. At 41, ASN occurred only 4 of 35 times. At 40, GLY occurred 17 of 35 times. At position 36, GLY occurred 34 of 35 times, indicating that ALA is undesirable here. EpiNE7.24 (SEQ ID NO:74) and EpiNE7.36 (SEQ ID NO:83) are most like EpiNE7 (SEQ ID NO:48), having three of the varied residues identical to EpiNE7.","Third, the results of the first and second fractionation are similar. In the second fractionation, the prevalence of TRP at position 39 is more marked (5\/15 in fractionation #1, 14\/20 in #2). It is possible that the first fractionation lost some high-affinity EPIs through under-sampling. Nevertheless, the first fractionation was clearly quite successful.","Fourth, there are strong preferences at positions 39 and 36 and lesser but significant preferences at positions 34 and 41 with little preference at 40.","Heretofore, no homologues of aprotinin have been reported having ALA at 36. In the selected EpiNE7.nn sequences, the preference for GLY over ALA at position 36 is 34:1. This preference is probably not due to differences in protein stability. The process of the present invention, as applied in the present example, does not select against proteins on the basis of stability so long as the protein does fold and function at the temperature used in the procedure. ALA is probably tolerated at position 36 well enough to allow those proteins having ALAto fold and function; one example was found having ALA. It may be relevant that the sole sequence having ALAalso has GLY. The flexibility of GLY at 34 may allow the methyl of ALA at 36 to fit into HNE in a way that is not possible when other amino acids occupy position 34.","At position 39, all 20 amino acids were allowed, but only seven were seen. TRP is strongly preferred with 19 occurrences, HIS second with six occurences, and LEU third with 5 occurrences. No homologues of aprotinin have been reported having either TRP or HIS at position 39 as are now disclosed. Although LEU is represented in the NNS codon thrice, TRP and HIS have but one codon each and their prevalence is surprising. We constructed a model having HNE (Brookhaven Protein Data Bank entry 1HNE) and EpiNE7.9 (SEQ ID NO:60) spatially related as in the 1TPA complex. (The \u03b1 carbons of HNE of conserved internal residues were superimposed on the corresponding \u03b1 carbons of trypsin, rms deviation \u22480.5 \u212b.) Inspection of this model indicates that TRPcould interact with the loop of HNE that comprises VAL, ASN, and LEU. HIS is observed in six cases; HIS is hydrophobic, aromatic, and in some ways similar to TRP. LEUin EpiNE7.5 could also interact with these residues if the loop moves a short distance. GLU occurred twice while LYS, ARG, and GLN occurred once each. In BPTI, the C of residue 39 is \u224810 \u212b from the C of residue 15 so that TRPinteracts with different features of HNE than do the amino acids substituted at position 15. Residue 34 is well separated from each of the residues 15, 18, and 39; thus it contacts different features on the HNE surface from these residues. Although serine proteases are highly similar near the catalytic site, the similarity diminishes rapidly outside this conserved region. The specificity of serine proteases is in fact determined by more interactions than the P1 residue. To make an inhibitor that is highly specific to HNE, we must go beyond matching the requirement at P1. Thus, the substitutions at 18 (determined in Example IV), 39, 34, and other non-P1 positions are invaluable in customizing the EpiNE to HNE. When making an inhibitor customized to a different serine protease, it is likely that many, if not all, of these positions will be changed to obtain high affinity and specificity. It is a major advantage of the present method that many such derivatives may be tested rapidly.","At position 34, all 20 amino acids were allowed. Fourteen have been seen. LYS appeared seven times, GLU five times, THR four times, LEU three times, GLY, ASP, GLN, MET, ASN, and HIS twice each, and ARG, PRO, VAL, and TYR once each. There were no instances of ALA, CYS, PHE, ILE, SER, or TRP. No homologue of aprotinin with GLU, GLY, or MET at 34 has been reported heretofore. Here, as at position 39, the library contains an excess of LEU over LYS and GLU. Thus, we infer that the prevalence of LYS, GLU, THR, and LEU is related to tighter binding of EpiNEs having these amino acids at position 34. The prevalence of LYS is surprising, as there are no acidic groups on HNE in the neighborhood. The Nof LYScould interact with a main-chain carbonyl oxygen while the methylene groups interact with ILEand\/or PHE. LEUcould interact with ILEand\/or PHEwhile GLUcould interact with ARG.","There has been little if any enrichment at positions 40 and 41. Alanine is somewhat preferred at 40; ALA:GLY::18:17. Both ALA and GLY have been reported in aprotinin homologues.","Position 41 shows a preponderance of LYS (12 occurrences) and GLU (7), but all eight possibilities have been seen. The overall distribution is LYS, GLU, ASP, ASN, GLN, HIS, and TYR. Heretofore, no homologues of aprotinin having GLU, GLN, HIS, or TYR at position 41 have been reported.","One sequence, EpiNE7.25 (SEQ ID NO:75) contains an unexpected change at position 47, SER to LEU. Heretofore, all homologues of aprotinin reported have had either SER or THR at position 47. The side groups of SER and THR can form hydrogen bonds to main-chain atoms at the beginning of the short a helix.","The consensus sequence, LYS, GLY, TRP, ALA, LYSwas not observed. EpiNE7.23 (SEQ ID NO:73) is quite close, differing only at position 40 where the preference for ALA is very, very weak.","We tested EpiNE7.23 (the sequence closest to consensus) against EpiNE7 (SEQ ID NO:48) on HNE beads.  shows the fractionation of strains of phage that display these two EpiNEs. Phage that display EpiNE7 are eluted at higher pH than are phage that display EpiNE7.23. Furthermore, more of the EpiNE7.23 phage are retained than of the EpiNE7 phage. Note the peak at pH 2.25 in the EpiNE7.23 elution. This suggests that EpiNE7.23 has a higher affinity for HNE than does EpiNE7. In a similar way, we tested EpiNE7.4 (SEQ ID NO:63) and found that it is not retained on HNE so well as EpiNE7. This is consistent with the fractionation not being complete.","Further fractionation, characterization of clonally pure EpiNE7.nn strains, and biochemical characterization of soluble EpiNE7.nn derivatives will reveal which sequences in this collection have the highest affinity for HNE.","Fractionation of the library involves a number of factors. Differential binding allows phage that display PBDs having the desired binding properties to be enriched. Differences in infectivity, plaque size, and phage yield are related to differences in the sequence of the PBDs, but are not directly correlated to affinity for the target. These factors may reduce the effectiveness of the desired fractionation. An additional factor that may be present is differential abundance of PBD sequences in the initial library. One step we employ to reduce the effect of differential infectivity is to transduce cells with isolated phage rather than to infect them. In the first fractionation, we did not obtain sufficient material for transduction and so infected cells; this fractionation was successful. Because the parental sequence, EpiNE7, was selected for a sequence at residues 15 through 19 that confer high affinity for HNE, we believe that many, if not most, members of the KLMUT population have significant affinity for HNE. Thus the present fractionations must separate variants having very high affinity for HNE from those merely having high affinity for HNE. It is perhaps relevant that BPTI-III MK phage are only partially eluted from immobilized trypsin at pH 2.2.; K(trypsin,BPTI)=6.0\u00b710M. Elution of EpiNE7-III MA phage from immobilized HNE gives a peak at about pH 3.5 with some phage appearing at lower pH; K(HNE,EpiNE7)\u22661.\u00b710M. We recycled phage that either were eluted at pH 2.0 or that were retained after elution with pH 2.0 buffer. A large percentage of EpiNE7-III MA phage would have been washed away with the fractions at pHs less acid than 2.0. This, together with the marked preferences at positons 39, 36, and 34, strongly sugestes that we have successfully fractionated the KLMUT library on the basis of affinity for HNE and that the EpiNE7.nn proteins have higher affinity for HNE than does EpiNE7 or any other reported aprotinin derivative.","Fractionation in a few stringent steps emphasizes the affinity of the PBD and allows isolation of variants that confer a small-plaque phenotype on cells (through low infectivity or by slowing cell growth). More gradual fractionation allows observation of a wider variety of variants that show high affinity and favors sequences that start at low abundance. Gradual fractionation also favors selection of variants that do not confer a small-plaque phenotype; such variants may be easier to work with and are preferred for some purposes. In either case, it is preferred to fractionate until there is a manageable number of distinct isolates and to characterize these isolates as pure clones. Thus, it is desirable, in most cases, to fractionate a library in more than one way.","None have identified positions 39 and 34 as key in determining the affinity and specificity of aprotinin homologues and derivatives for particular serine proteases. None have suggested the tryptophan at 39 or charged amino acids (LYS or GLU) at 34 will enhance binding of an aprotinin homologue to HNE. Different substitutions at these positions is likely to confer different specificity on those derivatives. One of the major advantages of the present invention is that many substitutions at several locations may be tested with an amount of effort not much greater than is required to test a single derivative by previously used methods.","There exist a number of proteases produced by lymphocytes. Neutrophil elastase is not the only lymphocytic protease that degrades elastin. The protease p29 is related to HNE. Screening the MYMUT and KLMUT libraries against immobilized p29 is likely to allow isolation of an aprotinin derivative having high affinity for p29.","The aim of this work was to introduce peptide extensions between the C-terminus of the BPTI domain and the N-terminus of the M13 major coat protein within the fusion protein. The reasons for this were two fold; firstly to alter potential protease cleavage sites at the interdomain boundary (as evidenced by an apparent instability of the fusion protein) and secondly to increase interdomain flexibility.","1) Insertion of a Variegated Pentapeptide at the BPTI:VIII Interface.","The gene shown in Table 113 was modified by insertion of five RVT codons between codon 81 and 82. Two synthetic oligonucleotides were designed and custom synthesized. The first consisted of, from 5\u2032 to 3\u2032: a) from base 2 of codon 77 to the end of codon 81, b) five copies of RVT, and c) from codon 82 to the second base of codon 94. The second comprised 20 bases complementary to the 3\u2032 end of the first oligonucleotide. Each RVT codon allows one of the amino acids [T, N, S, A, D, and G] to be encoded. This variegation codon was picked because: a) each amino acid occurs once, and b) all these amino acids are thought to foster a flexible linker. When annealed, the primed variegated oligonucleotide was converted to double-stranded DNA using standard methods.","The duplex was digested with restriction enzymes SfiI and NarI and the resulting 45 base-pair fragment was ligated into a similarly cleaved OCV, M13MB48 (Example I.1.iii.a). The ligated material was transfected into competent cells (strain XL1-Blue\u2122) and plated onto a lawn of the same cells on normal bacterial growth plates to form plaques. The bacteriophage contained within the plaques were analyzed using standard methods of nitrocellulose lifts and probing using a P-labeled oligonucleotide complementary to the DNA sequence encoding the fusion protein interface. Approximately 80% of the plaques probed poorly with this oligonucleotide and hence contained new sequences at this position.","A pool of phages, containing the novel interface pentapeptide extensions, was collected by combining the phage extracted from the plated plaques.","2. Adding Multiple Unit Extensions to the Fusion Protein Interface.","The M13 gene III product contains \u2018stalk-like\u2019 regions as implied by electron micrographic visualization of the bacteriophage (LOPE85). The predicted amino acid sequence of this protein contains repeating motifs, which include: glu.gly.gly.gly.ser (EGGGS) (SEQ ID NO:10) seven times gly.gly.gly.ser (GGGS) (SEQ ID NO:14) three times glu.gly.gly.gly.thr (EGGGT) (SEQ ID NO:15) once.","The aim of this section was to insert, at the domain interface, multiple unit extensions which would mirror the repeating motifs observed in the III gene product.","Two synthetic oligonucleotides were designed and custom synthesized. GLY is encoded by four codons (GGN); when translated in the opposite direction, these codons give rise to THR, PRO, ALA, and SER. The third base of these codons was picked so that translation of the oligonucleotide in the opposite direction would encode SER. When annealed the synthetic oligonucleotides give the following unit duplex sequence (an EGGGS linker):",{"@attributes":{"id":"p-0773","num":"1049"},"tables":{"@attributes":{"id":"TABLE-US-00019","num":"00019"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"133pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["E\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003S","(SEQ ID NO:10)",{}]},{"entry":{}},{"entry":["5\u2032\u2003C.GAG.GGA.GGA.GGA.TC\u2003\u2003\u2003\u20033\u2032","(SEQ ID NO:100)"]},{"entry":{}},{"entry":["3\u2032\u2003\u2003\u2003\u2003TC.CCT.CCT.CCT.AGG.C 5\u2032","(SEQ ID NO:101)"]},{"entry":{}},{"entry":["(L) (S) (S) (S) (G)","(SEQ ID NO:261)"]}]}}}}},"The duplex has a common two base pair 5\u2032 overhang (GC) at either end of the linker which allows for both the ligation of multiple units and the ability to clone into the unique NarI recognition sequence present in OCV's M13MB48 and Gem MB42. This site is positioned within 1 codon of the DNA encoding the interface. The cloning of an EGGGS linker (SEQ ID NO:10) (or multiple linker) into the vector NarI site destroys this recognition sequence. Insertion of the EGGGS linker (SEQ ID NO:10) in reverse orientation leads to insertion of GSSSL (SEQ ID NO:16) into the fusion protein.","Addition of a single EGGGS linker (SEQ ID NO:10) at the NarI site of the gene shown in Table 113 leads to the following gene:",{"@attributes":{"id":"p-0776","num":"1052"},"tables":{"@attributes":{"id":"TABLE-US-00020","num":"00020"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"217pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":["(SEQ ID NO:17)",{}]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"79\u2003\u200380\u2003\u200380a\u200380b\u200380c\u200380d\u200380e\u200381\u2003\u200382\u2003\u200383\u2003\u200384",{}]},{"entry":[{},"\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003E\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003S\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003E\u2003\u2003\u2003G"]},{"entry":[{},"\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003-----------------"]},{"entry":[{},{}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"217pt","align":"right"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["(SEQ ID NO:102)",{}]},{"entry":"GGT.GGC.GAG.GGA.GGA.GGA.TCC.GCC.GCT.GAA.GGT"},{"entry":"\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003-------------------"}]}}]}}},"Note that there is no preselection for the orientation of the linker(s) inserted into the OCV and that multiple linkers of either orientation (with the predicted EGGGS (SEQ ID NO:10) or GSSSL (SEQ ID NO:16) amino acid sequence) or a mixture of orientations (inverted repeats of DNA) could occur.","A ladder of increasingly large multiple linkers was established by annealing and ligating the two starting oligonucleotides containing different proportions of 5\u2032 phosphorylated and non-phosphorylated ends. The logic behind this is that ligation proceeds from the 3\u2032 unphosphorylated end of an oligonucleotide to the 5\u2032 phosphorylated end of another. The use of a mixture of phosphorylated and non-phosphorylated oligonucleotides allows for an element of control over the extent of multiple linker formation. A ladder showing a range of insert sizes was readily detected by agarose gel electrophoresis spanning 15 bp (1 unit duplex-5 amino acids) to greater than 600 base pairs (40 ligated linkers-200 amino acids).","Large inverted repeats can lead to genetic instability. Thus we chose to remove them, prior to ligation into the OCV, by digesting the population of multiple linkers with the restriction enzymes AccIII or XhoI, since the linkers, when ligated \u2018head-to-head\u2019 or \u2018tail-to-tail\u2019, generate these recognition sequences. Such a digestion significantly reduces the range in sizes of the multiple linkers to between 1 and 8 linker units (i.e. between 5 and 40 amino acids in steps of 5), as assessed by agarose gel electrophoresis.","The linkers were ligated (as a pool of different insert sizes or as gel-purified discrete fragments) into NarI cleaved OCVs M13MB48 or GemMB42 using standard methods. Following ligation the restriction enzyme NarI was added to remove the self-ligating starting OCV (since linker insertion destroys the NarI recognition sequence). This mixture was used to transform competent XL-1 blue cells and appropriately plated for plaques (OCV M13MB48) or ampicillin resistant colonies (OCV GemMB42).","The transformants were screened using dot blot DNA analysis with one of two P labeled oligonucleotide probes. One probe consisted of a sequence complementary to the DNA encoding the P1 loop of BPTI while the second had a sequence complementary to the DNA encoding the domain interface region. Suitable linker candidates would probe positively with the first probe and negatively or poorly with the second. Plaque purified clones were used to generate phage stocks for binding analyses and BPTI display while the Rf DNA derived from phage infected bacterial cells was used for restriction enzyme analysis and sequencing. Representative insert sequences of selected clones analyzed are as follows:",{"@attributes":{"id":"p-0782","num":"1058"},"tables":{"@attributes":{"id":"TABLE-US-00021","num":"00021"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"168pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"161pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["M13.3X4","(GG)C.GGA.TCC.TCC.TCC.CT(C.GCC)","(SEQ ID NO: 103)",{}]},{"entry":{}},{"entry":[{},"gly ser ser ser leu","(AA 6-10 of SEQ"]},{"entry":[{},{},"ID NO:11 = SEQ ID"]},{"entry":[{},{},"NO:150)"]},{"entry":{}},{"entry":["M13.3X7","(G C.GAG.GGA.GGA.GGA.TC(C.GCC)","(SEQ ID NO: 104)"]},{"entry":{}},{"entry":[{},"glu gly gly gly ser","(SEQ ID NO:10)"]},{"entry":{}},{"entry":["M13.3X11","(GG) C.GAG.GGA.GGA.GGA.TCC.GGA.TCC.TCC.","(SEQ ID NO: 105)"]},{"entry":{}},{"entry":[{},"glu gly gly gly ser gly ser ser","(SEQ ID NO: 18)"]},{"entry":{}},{"entry":[{},"TCC.CTC.GGA.TCC.TCC.TCC.CT (C.GCCC)"]},{"entry":{}},{"entry":[{},"ser leu gly ser ser ser leu"]}]}}}},"br":{}},"The expression vectors were designed for the bacterial production of BPTI analogues resulting from the mutagenesis and screening for variants with specific binding properties. The expression vectors used are derivatives of the OCV's M13MB48 and GemMB42. The conversion was achieved by replacing the first codon of the mature VIII gene (codon 82 as shown in Table 113) with a translational stop codon by site specific mutagenesis.","The salient points of the expression vector composition are identical to that of the parent OCV's, namely a lacUV5 promoter (hence IPTG induction), ribosome binding site, initiating methionine, pho A signal peptide and transcriptional termination signal (see Table 113). The placement of the stop codon allows for the expression of only the first half the fusion protein. The Gem-based expression system, containing the genes encoding BPTI analogues, is stored as plasmid DNA, being freshly transfected into cells for expression of the analogue protein. The M13-based expression system is stored as both RF DNA and as phage stocks. The phage stocks are used to infect fresh bacterial cells for expression of the protein of interest.","Bacterial Expression of BPTI and Analogues.","i. Gem-based Expression Vector and Protocol.","The gem-based expression vector is a derivative of the OCV GemMB42 (Example I and Table 113). This vector, at least when it contains the BPTI or analogue genes, has demonstrated a degree of insert instability on prolonged growth in liquid culture. To reduce the risk of this the following protocol is used.","Expression vector DNA (containing the BPTI or analogue gene) is transfected into the strain, XL1-Blue\u2122, which is plated on bacterial plates containing ampicillin and allowed to incubate overnight at 37\u00b0 C. to give a dense population of colonies. The colonies are scraped from the plate with a glass spreader in 1 ml of NZCYM medium and combined with the scraped cells from other duplicate plates. This stock of cells is diluted approximately one hundred fold into NZCYM liquid medium containing ampicillin (100 \u03bcg per ml) and allowed to grow in a shaking incubator to a cell density of approximately half log (absorbance of 0.3 at 600 nm). IPTG is added to a final concentration of 0.5 mM and the induced culture allowed to grow for a further two hours when it is processed as described below.","ii. M13-Based Expression Vector and Protocol.","The M13-based expression vector is derived from OCV M13MB48 (Example I). The BPTI gene (or analogue) is contained within the intergenic region and its transcription is under the control of a lacUV5 promoter, hence IPTG inducible. The expression vector, containing the gene of interest, is maintained and utilized as a phage stock. This method enables a potentially lethal or deleterious gene to be supplied to a bacterial culture and gene induction to occur only when the bacterial culture has achieved sufficient mass. Poor growth and insert instability can be circumvented to a large extent, giving this system an advantage over the Gem-based vector described above.","An overnight bacterial culture of XL1-Blue\u2122 or SEF\u2032 is grown in LB medium containing tetracycline (50 \u03bcg per ml) to ensure the presence of pili as sites for bacteriophage binding and infection. This culture is diluted 100-fold into NZCYM medium containing tetracycline and bacterial growth allowed to proceed in an incubator shaker until a cell density of 1.0 (Ab 600 nm) has been achieved. Phage, containing the expression vector and gene of interest, are added to the bacterial culture at a multiplicity of infection (MOI) of 10 and allowed to infect the cells for 30 minutes. Gene expression is then induced by the addition of IPTG to a final concentration of 0.5 mM and the culture allowed to grow overnight. Media collection and cell fractionation is as described elsewhere.","Bacterial Cell Fractionation.","After heterologous gene expression the bacterial cell culture can be separated into the following fractions: conditioned medium, periplasmic fraction and post-periplasmic cell lysate. This is achieved using the following procedures.","The culture is centrifuged to pellet the bacteria, allowing the supernatant to be stored as conditioned medium. This fraction contains any exported proteins. The pellet is taken up in 20% sucrose, 30 mM Tris pH 8 and 1 mM EDTA (80 ml of buffer per gram of fresh weight pellet) and allowed to sit at room temperature for 10 minutes. The cells are repelleted and taken up in the same volume of ice cold 5 mM MgSOand left on ice for 10 minutes. Following centrifugation, to pellet the cells, the supernatant (periplasmic fraction) is stored. A second round of osmotic shock fractionation can be undertaken if desired.","The post-periplasmic pellet can be further lysed as follows. The pellet is resuspended in 1.5 ml of 20% sucrose, 40 mM Tris pH 8, 50mM EDTA and 2.5 mg of lysozyme (per gram fresh weight of starting pellet). After 15 minutes at room temperature 1.15 ml of 0.1% Triton X is added together with 300 \u03bcl of 5M NaCl and incubated for a further 15 minutes. 2.5 ml of 0.2 M triethanolamine (pH 7.8), 150 \u03bcl of 1M CaCl, 100 \u03bcl of 1M MgCland 5 \u03bcg of DNA'se are added and allowed to incubate, with end-over-end mixing, for 20 minutes to reduce viscosity. This is followed by centrifugation with the supernatant being retained as the post-periplasmic lysate.","The present invention is not, of course, limited to any particular expression system, whether bacterial or not.","1. ITI Domain I as an IPBD","Inter-\u03b1-trypsin inhibitor (ITI) is a large (Mca 240,000) circulating protease inhibitor found in the plasma of many mammalian species (for recent reviews see ODOM90, SALI90, GEBH90, GEBH86). The intact inhibitor is a glycoprotein and is currently believed to consist of three glycosylated subunits that interact through a strong glycosaminoglycan linkage (ODOM90, SALI90, ENGH89, SELL87). The anti-trypsin activity of ITI is located on the smallest subunit (ITI light chain, unglycosylated Mca 15,000) which is identical in amino acid sequence to an acid stable inhibitor found in urine (UTI) and serum (STI) (GEBH86, GEBH90). The mature light chain consists of a 21 residue N-terminal sequence, glycosylated at SER, followed by two tandem Kunitz-type domains the first of which is glycosylated at ASN(ODOM90). In the human protein, the second Kunitz-type domain has been shown to inhibit trypsin, chymotrypsin, and plasmin (ALBR83a, ALBR83b, SELL87, SWAI88). The first domain lacks these activities but has been reported to inhibit leukocyte elastase (10>K>10) (ALBR83a,b, ODOM90). cDNA encoding the ITI light chain also codes for \u03b1-1-microglobulin (TRAB86, KAUM86, DIAR90); the proteins are separated post-translationally by proteolysis.","The N-terminal Kunitz-type of the ITI light chain (ITI-D1, comprising residues 22 to 76 of the UTI sequence shown in  of GEBH86) possesses a number of characteristics that make it useful as an IPBD. The domain is highly homologous to both BPTI and the EpiNE series of proteins described elsewhere in the present application. Although an x-ray structure of the isolated domain is not available, crystallographic studies of the related Kunitz-type domain isolated from the Alzheimer's amyloid \u03b2-protein (AA\u03b2P) precursor show that this polypeptide assumes a crystal structure almost identical to that of BPTI (HYNE90). Thus, it is likely that the solution structure of the isolated ITI-D1 polypeptide will be highly similar to the structures of BPTI and AA\u03b2P. In this case, the advantages described previously for use of BPTI as an IPBD apply to ITI-D1. ITI-D1 provides additional advantages as an IDBP for the development of specific anti-elastase inhibitory activity. First, this domain has been reported to inhibit both leukocyte elastase (ALBR83a,b, ODOM90) and Cathepsin-G (SWAI88, ODOM90); activities which BPTI lacks. Second, ITI-D1 lacks affinity for the related serine proteases trypsin, chymotrypsin, and plasmin (ALBR83a,b, SWAI88), an advantage for the development of specificity in inhibition. Finally, ITI-D1 is a human-derived polypeptide so derivatives are anticipated to show minimal antigenicity in clinical applications.","2. Construction of the Display Vector.","For purposes of this discussion, numbering of the nucleic acid sequence for the ITI light chain gene is that of TRAB86 and of the amino acid sequence is that shown for UTI in  of GEBH86. DNA manipulations were conducted according to standard methods as described in SAMB89 and AUSU87.","The protein sequence of human ITI-D1 consists of 56 amino acid residues extending from LYSto ARGof the complete ITI light chain sequence. This sequence is encoded by the 168 bases between positions 750 and 917 in the cDNA sequence presented in TRAB86. The majority of the domain is contained between a BglI site spanning bases 663 to 773 and a PstI site spanning bases 903 to 908. The insertion of the ITI-D1 sequence into M13 gene III was conducted in two steps. First a linker containing the appropriate ITI sequences outside the central BglI to PstI region was ligated into the NarI site of phage MA RF DNA. In the second step, the remainder of the ITI-D1 sequence was incorporated into the linker-bearing phage RF DNA.","The linker DNA consisted of two synthetic oligonucleotides (top and bottom strands) which, when annealed, produced a 54 bp double-stranded fragment with the following structure (5\u2032 to 3\u2032):","NARI OVERHANG\/ITI-5\u2032\/BGLI\/STUFFER\/PSTI\/ITI-3\u2032\/NARI","Overhang","The NarI OVERHANG sequences provide compatible ends for ligation into a cut NarI site. The ITI-5\u2032 sequence consists of ds DNA corresponding to the thirteen positions from A750 to T662 immediately 5\u2032 adjacent to the BglI site in the ITI-D1 sequence. Two changes, both silent, are introduced in this sequence: T to C at position 658 (changes codon for ASPfrom GAT to GAC) and G to T at position 661 (changes codon for SERfrom TCG to TCT). The sequences BGLI and PSTI are identical to the BglI and PstI sites, respectively, in the ITI-D1 sequence. The ITI-3\u2032 sequence consists of dsDNA corresponding to the nine positions from A909 to T917 immediately 3\u2032 adjacent to the PstI site in the ITI-D1 sequence. The one base change included in this sequence, A to T at position 917, is silent and changes the codon for ARGfrom CGA to CGT. The STUFFER sequence consists of dsDNA encoding three residues (5\u2032 to 3\u2032): LEU (TTA), TRP(TGG), and SER(TCA). The reverse complement of the STUFFER sequence encodes two translation termination codons (TGA and TAA). Phage expressing gene III containing the linker in opposite orientation to that shown above will not produce a functional gene III product.","Phage MA RF DNA was digested with NarI and the linear ca. 8.2 kb fragment was gel purified and subsequently dephosphorylated using HK phosphatase (Epicentre). The linker oligonucleotides were annealed to form the linker fragment described above, which was then kinased using T4 Polynucleotide Kinase. The kinased linker was ligated to the NarI-digested MA RF DNA in a 10:1 (linker:RF) molar ratio. After 18 hrs at 16\u00b0 C., the ligation was stopped by incubation at 65\u00b0 C. for 10 min and the ligation products were ethanol precipitated in the presence of 10 \u03bcg of yeast tRNA. The dried precipitate was dissolved in 5 \u03bcl of water and used to transform D1210 cells by electroporation. After 60 min of growth in SOC at 37\u00b0 C., transformed cells were plated onto LB plates supplemented with ampicillin (Ap, 200 \u03bcg\/ml). RF DNA prepared from APisolates was subjected to restriction enzyme analysis. The DNA sequences of the linker insert and the immediately surrounding regions were confirmed by DNA sequencing. Phage strains containing the ITI Linker sequence inserted into the NarI site in gene III are called MA-IL.","Phage MA-IL RF DNA was partially digested with BglI and the ca. 8.2 kb linear fragment was gel purified. This fragment was digested with PstI and the large linear fragment was gel purified. The BglI to PstI fragment of ITI-D1 was isolated from pMG1A (a plasmid carrying the sequence shown in TRAB86). pMG1A was digested to completion with BglI and the ca. 1.6 kb fragment was isolated by agarose gel electrophoresis and subsequent Geneclean (Bio101, La Jolla, Calif.) purification. The purified BglI fragment was digested to completion with PstI and EcoRI and the resulting mixture of fragments was used in a ligation with the BglI and PstI cut MA-IL RF DNA described above. Ligation, transformation, and plating were as described above. After 18 hr. of growth on LB Ap plates at 37\u00b0 C., Apcolonies were harvested with LB broth supplemented with Ap (200 \u03bcg\/ml) and the resulting cell suspension was grown for two hours at 37\u00b0 C. Cells were pelleted by centrifugation (10 min at 5000\u00d7g, 4\u00b0 C.). The supernatant fluid was transferred to sterile centrifugation tubes and recentrifuged as above. The supernatant fluid from the second centrifugation step was retained as the phage stock POP1.","PCR was used to demonstrate the presence of phage containing the complete ITI-D1-III fusion gene. Upstream PCR primers, 1UP and 2UP, are located spanning nucleotides 1470 to 1494 and 1593 to 1618 of the phage M13 DNA sequence, respectively. A downstream PCR primer 3DN spans nucleotides 1779 to 1804. Two ITI-D1-specific primers, IAI-1 and IAI-2, are located spanning positions 789 to 810 and 894 to 914, respectively, in the ITI light chain sequence of TRAB86. IAI-1 and IAI-2 are used as downstream primers in PCR reactions with 1UP or 2UP. IAI-1 is entirely contained within the BglI to PstI region of the ITI-D1 sequence, while IAI-2 spans the PstI site in the ITI-D1 sequence. When aliquots of POP1 phage were used as substrates for PCR, template-specific products of characteristic size were produced in reactions containing 1UP or 2UP plus IAI-1 or IAI-2 primer pairs. No such products are obtained using MA-IL phage as template. No PCR products with sizes corresponding to complete ITI-D1-gene III templates were obtained using POP1 phage and the 1UP or 2UP plus 3DN primer pairs. This last result reflects the low abundance (<1%) of phage containing the complete ITI-D1 sequence in POP1.","Preparative PCR was used to generate substrate amounts of the 330 bp PCR product of a reaction using the 1UP and IAI-2 primer pair to amplify the POP1 template. The 330 bp PCR product was gel purified and then cut to completion with BglI and PstI. The 138 bp BglI to PstI fragment from ITI-D1 was isolated by agarose gel electrophoresis followed by Qiaex extraction (Qiagen, Studio City, Calif.). MA-IL phage RF DNA was digested to completion with PstI. The ca. 8.2 kb linear fragment was gel purified and subsequently digested to completion with BglI. The BglI digest was extracted once with phenol:chloroform (1:1), the aqueous phase was ethanol precipitated, and the pellet was dissolved in TE (pH8.0). An aliquot of this solution was used in a ligation reaction with the 138 bp BglI to PstI fragment as described above. The ethanol precipitated ligation products were used to transform XL1-Blue\u2122 cells by electroporation and after 1 hr growth in SOC at 37\u00b0 C., cells were plated on LB Ap plates. A phage population, POP2, was prepared from Apcolonies as described previously.","Phage stocks obtained from individual plaques produced on titration of POP2 were tested by PCR for the presence of the complete ITI-D1-III gene fusion. PCR results indicate the entire fusion gene was present in seven of nine isolates tested. RF DNA from the seven isolates testing positive was subjected to restriction enzyme analysis. The complete sequence of the ITI-D1 insertion into gene III was confirmed in four of the seven isolates by DNA sequence analysis. Phage isolates containing the ITI-D1-III fusion gene are called MA-ITI.","3. Expression and Display of ITI-DI.","Expression of the ITI domain I-Gene III fusion protein and its display on the surface of phage were demonstrated by Western analysis and phage titer neutralization experiments.","For Western analysis, aliquots of PEG-purified phage preparations containing up to 4\u00b710infective particles were subjected to electrophoresis on a 12.5% SDS-urea-polyacrylamide gel. Proteins were transferred to a sheet of Immobilon-P transfer membrane (Millipore, Bedford, Mass.) by electrotransfer. Western blots were developed using a rabbit anti-ITI serum (SALI87) which had previously been incubated with an extract, followed by goat anti-rabbit IgG conjugated to horse radish peroxidase (#401315, Calbiochem, La Jolla, Calif.). An immunoreactive protein with an apparent size of ca. 65-69 kD is detected in preparations of MA-ITI phage but not with preparations of the parental MA phage. The size of the immunoreactive protein is consistent with the expected size of the processed ITI-D1-III fusion protein (ca. 67 kD, as previously observed for the BPTI-III fusion protein).","Rabbit anti-BPTI serum has been shown to block the ability of MK-BPTI phage to infect cells (Example II). To test for a similar effect of rabbit anti-ITI serum on the infectivity of MA-ITI phage, 10 \u03bcl aliquots of MA or MA-ITI phage were incubated in 100 \u03bcl reactions containing 10 \u03bcl aliquots of PBS, normal rabbit serum (NRS), or anti-ITI serum. After a three hour incubation at 37\u00b0 C., phage suspensions were titered to determine residual plaque-forming activity. These data are summarized in Table 211. Incubation of MA-ITI phage with rabbit anti-ITI serum reduces titers 10- to 100-fold, depending on initial phage titer. A much smaller decrease in phage titer (10 to 40%) is observed when MA-ITI phage are incubated with NRS. In contrast, the titer of the parental MA phage is unaffected by either NRS or anti-ITI serum.","Taken together, the results of the Western analysis and the phage-titer neutralization experiments are consistent with the expression of an ITI-D1-III fusion protein in MA-ITI phage, but not in the parental MA phage, such that ITI-specific epitopes are present on the phage surface. The ITI-specific epitopes are located with respect to III such that antibody binding to these epitopes prevents phage from infecting cells.","4. Fractionation of MA-ITI Phage Bound to Agarose-Immobilized Protease Beads.","To test if phage displaying the ITI-DI-III fusion protein interact strongly with the proteases human neutrophil elastase (HNE) or cathepsin-G, aliquots of display phage were incubated with agarose-immobilized HNE or cathepsin-G beads (HNE beads or Cat-G beads, respectively). The beads were washed and bound phage eluted by pH fractionation as described in Examples II and III. The procession in lowering pH during the elution was: pH 7.0, 6.0, 5.5, 5.0, 4.5, 4.0, 3.5, 3.0, 2.5, and 2.0. Following elution and neutralization, the various input, wash, and pH elution fractions were titered.","The results of several fractionations are summarized in Table 212 (EpiNE-7 or MA-ITI phage bound to HNE beads) and Table 213 (EpiC-10 or MA-ITI phage bound to Cat-G beads). For the two types of beads (HNE or Cat-G), the pH elution profiles obtained using the control display phage (EpiNE-7 or EpiC-10, respectively) were similar to those seen previously (Examples II and III). About 0.3% of the EpiNE-7 display phage applied to the HNE beads were eluted during the fractionation procedure and the elution profile had a maximum for elution at about pH 4.0. A smaller fraction, 0.02%, of the EpiC-10 phage applied to the Cat-G beads were eluted and the elution profile displayed a maximum near pH 5.5.","The MA-ITI phage show no evidence of great affinity for either HNE or cathepsin-G immobilized on agarose beads. The pH elution profiles for MA-ITI phage bound to HNE or Cat-G beads show essentially monotonic decreases in phage recovered with decreasing pH. Further, the total fractions of the phage applied to the beads that were recovered during the fractionation procedures were quite low: 0.002% from HNE beads and 0.003% from Cat-G beads.","Published values of Kfor inhibition neutrophil elastase by the intact, large (M=240,000) ITI protein range between 60 and 150 nM and values between 20 and 6000 nM have been reported for the inhibition of Cathepsin G by ITI (SWAI88, ODOM90). Our own measurements of pH fraction of display phage bound to HNE beads show that phage displaying proteins with low affinity (>\u03bcM) for HNE are not bound by the beads while phage displaying proteins with greater affinity (nM) bind to the beads and are eluted at about pH 5. If the first Kunitz-type domain of the ITI light chain is entirely responsible for the inhibitory activity of ITI against HNE, and if this domain is correctly displayed on the MA-ITI phage, then it appears that the minimum affinity of an inhibitor for HNE that allows binding and fractionation of display phage on HNE beads is 50 to 100 nM.","5. Alteration of the P1 Region of ITI-DI.","If ITI-DI and EpiNE-7 assume the same configuration in solution as BPTI, then these two polypeptides have identical amino acid sequences in both the primary and secondary binding loops with the exception of four residues about the P1 position. For ITI-DI the sequence for positions 15 to 20 is (position 15 in ITI-DI corresponds to position 36 in the UTI sequence of GEBH86): MET15, GLY16, MET17, THR18, SER19, ARG20. In EpiNE-7 the equivalent sequence is: VAL15, ALA16, MET17, PHE18, PRO19, ARG20. These two proteins appear to differ greatly in their affinities for HNE. To improve the affinity of ITI-DI for HNE, the EpiNE-7 sequence shown above was incorporated into the ITI-DI sequence at positions 15 through 20.","The EpiNE-7 sequence was incorporated into the ITI-DI sequence in MA-ITI by cassette mutagenesis. The mutagenic cassette consisted of two synthetic 51 base oligonucleotides (top and bottom stands) which were annealed to make double stranded DNA containing an Eag I overhang at the 5\u2032 end and a Sty I overhang at the 3\u2032 end. The DNA sequence between the Eag I and Sty I overhangs is identical to the ITI-DI sequence between these sites except at four codons: the codon for position 15, AT (MET), was changed to GTC (VAL), the codon for position 16, GGA (GLY), was changed to GCT (ALA), the codon for position 18, ACC (THR) was changed to TTC (PHE), and the codon for position 19, AGC (SER), was changed to CCA (PRO). MA-ITI RF DNA was digested with Eag I and Sty I. The large, linear fragment was gel purified and used in a ligation with the mutagenic cassette described above. Ligation products were used to transform XL1-Blue\u2122 cells as described previously. Phage stocks obtained from overnight cultures of Aptransductants were screened by PCR for incorporation of the altered sequence and the changes in the codons for positions 15, 16, 18, and 19 were confirmed by DNA sequencing. Phage isolates containing the ITI-DI-III fusion gene with the EpiNE-7 changes around the P1 position are called MA-ITI-E7.","6. Fractionation of MA-ITI-E7 Phage.","To test if the changes at positions 15, 16, 18, and 19 of the ITI-DI-III fusion protein influence binding of display phage to HNE beads, abbreviated pH elution profiles were measured. Aliquots of EpiNE-7, MA-ITI, and MA-ITI-E7 display phage were incubated with HNE beads for three hours at room temperature. The beads were washed and phage were eluted as described (Example III), except that only three pH elutions were performed: pH 7.0, 3.5, and 2.0. The results of these elutions are shown in Table 214.","Binding and elution of the EpiNE-7 and MA-ITI display phage were found to be as previously described. The total fraction of input phages was high (0.4%) for EpiNE-7 phage and low (0.001%) for MA-ITI phage. Further, the EpiNE-7 phage showed maximum phage elution in the pH 3.5 fraction while the MA-ITI phage showed only a monotonic decrease in phage yields with decreasing pH, as seen above.","The two strains of MA-ITI-E7 phage show increased levels of binding to HNE beads relative to MA-ITI phage. The total fraction of the input phage eluted from the beads is 10-fold greater for both MA-ITI-E7 phage strains than for MA-ITI phage (although still 40-fold lower that EpiNE-7 phage). Further, the pH elution profiles of the MA-ITI-E7 phage strains show maximum elutions in the pH 3.5 fractions, similar to EpiNE-7 phage.","To further define the binding properties of MA-ITI-E7 phage, the extended pH fractionation procedure described previously was performed using phage bound to HNE beads. These data are summarized in Table 215. The pH elution profile of EpiNE-7 display phage is as previously described. In this more resolved, pH elution profile, MA-ITI-E7 phage show a broad elution maximum centered around pH 5. Once again, the total fraction of MA-ITI-E7 phage obtained on pH elution from HNE beads was about 40-fold less than that obtained using EpiNE-7 display phage.","The pH elution behavior of MA-ITI-E7 phage bound to HNE beads is qualitatively similar to that seen using BPTI[K15L]-III-MA phage. BPTI with the K15L mutation has an affinity for HNE of \u22483.\u00b710M. Assuming all else remains the same, the pH elution profile for MA-ITI-E7 suggests that the affinity of the free ITI-DI-E7 domain for HNE might be in the nM range. If this is the case, the substitution of the EpiNE-7 sequence in place of the ITI-DI sequence around the P1 region has produced a 20- to 50-fold increase in affinity for HNE (assuming K=60 to 150 nM for the unaltered ITI-DI).","If EpiNE-7 and ITI-DI-E7 have the same solution structure, these proteins present the identical amino acid sequences to HNE over the interaction surface. Despite this similarity, EpiNE-7 exhibits a roughly 1000-fold greater affinity for HNE than does ITI-DI-E7. Again assuming similar structure, this observation highlights the importance of non-contacting secondary residues in modulating interaction strengths.","Native ITI light chain is glycosylated at two positions, SER10 and ASN45 (GEBH86). Removal of the glycosaminoglycan chains has been shown to decrease the affinity of the inhibitor for HNE about 5-fold (SELL87). Another potentially important difference between EpiNE-7 and ITI-DI-E7 is that of net charge. The changes in BPTI that produce EpiNE-7 reduce the total charge on the molecule from +6 to +1. Sequence differences between EpiNE-7 and ITI-DI-E7 further reduce the charge on the latter to \u22121. Furthermore, the change in net charge between these two molecules arises from sequence differences occurring in the central portions of the molecules. Position 26 is LYS in EpiNE-7 and is THR in ITI-DI-E7, while at position 31 these residues are GLN and GLU, respectively. These changes in sequence not only alter the net charge on the molecules but also position negatively charged residue close to the interaction surface in ITI-DI-E7. It may be that the occurrence of a negative charge at position 31 (which is not found in any other of the HNE inhibitors described here) destabilized the inhibitor-protease interaction.","The following is a hypothetical example demonstating how to obtain a derivative of ITI having high affinity for HNE.","The results of Example IX demonstrate that the nature of the protein sequence around the P1 position in ITI-DI can significantly influence the strength of the interaction between ITI-DI and HNE. While incorporation of the EpiNE-7 sequence increases the affinity of ITI-DI for HNE, it is unlikely that this particular sequence is optimal for binding.","We generate a large population of potential binding proteins having differing sequences in the P1 region of ITI-DI using the oligonucleotide ITIMUT. ITIMUT is designed to incorporate variegation in ITI-DI at the six positions about and including the P1 residue: 13, 15, 16, 17, 18, and 19. ITIMUT is synthesized as one long (top strand) 73 base oligonucleotide and one shorter (24 base) bottom strand oligonucleotide. The top strand sequence extends from position 770 (G) to position 842 (G) in the sequence of TREB86. This sequence includes the codons for the positions of variegation as well as the recognition sequences for the flanking restriction enzymes Eag I (778 to 783) and Sty I (829 to 834). The bottom strand oligonucleotide comprises the complement of the sequence from positions 819 to 842.","To generate the mutagenic cassette, the top and bottom strand oligonucleotides are annealed and the resulting duplex is completed in an extension reaction using DNA polymerase. Following digestion of the 73 bp dsDNA with Eag I and Sty I, the purified 51 bp mutagenic cassette is ligated with the large linear fragment obtained from a similar digestion of MA-ITI RF DNA. Ligation products are used to transform competent cells by electroporation and phage stocks produced from Aptransductants are analyzed for the presence and nature of novel sequences as described previously.","The variegation in the ITIMUT cassette is confined to the codons for the six positions in ITI-DI (13, 15, 16, 17, 18, and 19), and employs three different nucleotide mixes: N, R, and S. For this mutagenesis, the composition of the N-mix is 36% A, 17% C, 23% G, and 24% T, and corresponds to the N-mix composition in the optimized NNS codon described elsewhere. The R-mix composition is 50% A, 50% G, and the S-mix composition is 50% C, 50% G.","The codon for ITI-DI position 13 (CCC, PRO) is changed to SNG in ITIMUT. This codon encodes the eight residues PRO, VAL, GLU, ALA, GLY, LEU, GLN, and ARG. The encoded group includes the parental residue (PRO) as well as the more commonly observed variants at the position, ARG and LEU (see Table 15), and also provides for the occurrence of acidic (GLU), large polar (GLN) and nonpolar (VAL), and small (ALA, GLY) residues.","The codons for positions 15 and 17 (ATG, MET) are changed to the optimized NNS codon. All 20 natural amino acid residues and a translation termination are allowed.","The codon for position 16 (CGA, GLY) is changed to RNS in ITIMUT. This codon encodes the twelve amino acids GLY, ALA, ASP, GLU, VAL, MET, ILE, THR, SER, ARG, ASN, and LYS. The encoded group includes the most commonly observed residues at this position, ALA and GLY, and provides for the occurrence of both positively (ARG, LYS) and negatively (GLU, ASP) charged amino acids. Large nonpolar residues are also included (ILE, MET, VAL).","Finally, at positions 18 and 19, the ITI-DI sequence is changed from ACC\u00b7AGC (THR\u00b7SER) to NNT\u00b7NNT. The NNT codon encodes the fifteen amino acid residues PHE, SER, TYR, CYS, LEU, PRO, HIS, ARG, ILE, THR, ASN, VAL, ALA, ASP, and GLY. This group includes the parental residues and the further advantages of the NNT codon have been discussed elsewhere.","The ITIMUT DNA sequence encodes a total of:\n\n8*20*12*20*15*15=8,640,000\n\ndifferent protein sequences in a total of:\n\n2=33,554,422\n\ndifferent DNA sequences. The total number of protein sequences encoded by ITIMUT is only 7.4-fold fewer than the total possible number of natural sequences obtained from variation at six positions (=20=6.4\u00b710). However, this degree of variation in protein sequence is obtained from a minimum of 1.07\u00d710(NNS=2) DNA sequences, a 32-fold greater number than that comprising ITIMUT. Thus, ITIMUT is an efficient vehicle for the generation of a large and diverse population of potential binding proteins.\n","The following example is hypothetical and illustrates alternative embodiments of the invention not given in other examples.","HHMb is chosen as a typical protein target; any other protein could be used. HHMb satisfies all of the criteria for a target: 1) it is large enough to be applied to an affinity matrix, 2) after attachment it is not reactive, and 3) after attachment there is sufficient unaltered surface to allow specific binding by PBDs.","The essential information for HHMb is known: 1) HHMb is stable at least up to 70\u00b0 C., between pH 4.4 and 9.3, 2) HHMb is stable up to 1.6 M Guanidinium Cl, 3) the pI of HHMb is 7.0, 4) for HHMb, M=16,000, 5) HHMb requires haem, 6) HHMb has no proteolytic activity.","In addition, the following information about HHMb and other myoglobins is available: 1) the sequence of HHMb is known, 2) the 3D structure of sperm whale myoglobin is known; HHMb has 19 amino acid differences and it is generally assumed that the 3D structures are almost identical, 3) HHMb has no enzymatic activity, 4) HHMb is not toxic.","We set the specifications of an SBD as: 1) T=25\u00b0 C.; 2) pH=8.0; 3) Acceptable solutes ((A) for binding: i) phosphate, as buffer, 0 to 20 mM, and ii) KCl, 10 mM; (B) for column elution: i) phosphate, as buffer, 0 to 30 mM, ii) KCl, up to 5 M, and iii) Guanidinium Cl, up to 0.8 M.); 4) Acceptable K<1.0\u00b710M.","As stated in Sec. III.B, the residues to be varied are picked, in part, through the use of interactive computer graphics to visualize the structures. In this example, all residue numbers refer to BPTI. We pick a set of residues that forms a surface such that all residues can contact one target molecule. Information that we refer to during the process of choosing residues to vary includes: 1) the 3D structure of BPTI, 2) solvent accessibility of each residue as computed by the method of Lee and Richards (LEEB71), 3) a compilation of sequences of other proteins homologous to BPTI, and 4) knowledge of the structural nature of different amino acid types.","Tables 16 and 34 indicate which residues of BPTI: a) have substantial surface exposure, and b) are known to tolerate other amino acids in other closely related proteins. We use interactive computer graphics to pick sets of eight to twenty residues that are exposed and variable and such that all members of one set can touch a molecule of the target material at one time. If BPTI has a small amino acid at a given residue, that amino acid may not be able to contact the target simultaneously with all the other residues in the interaction set, but a larger amino acid might well make contact. A charged amino acid might affect binding without making direct contact. In such cases, the residue should be included in the interaction set, with a notation that larger residues might be useful. In a similar way, large amino acids near the geometric center of the interaction set may prevent residues on either side of the large central residue from making simultaneous contact. If a small amino acid, however, were substituted for the large amino acid, then the surface would become flatter and residues on either side could make simultaneous contact. Such a residue should be included in the interaction set with a notation that small amino acids may be useful.","Table 35 was prepared from standard model parts and shows the maximum span between C and the tip of each type of side group. C is used because it is rigidly attached to the protein main-chain; rotation about the C-Cbond is the most important degree of freedom for determining the location of the side group.","Table 34 indicates five surfaces that meet the given criteria. The first surface comprises the set of residues that actually contacts trypsin in the complex of trypsin with BPTI as reported in the Brookhaven Protein Data Bank entry \u201c1TPA\u201d. This set is indicated by the number \u201c1\u201d. The exposed surface of the residues in this set (taken from Table 16) totals 1148 \u212b. Although this is not strictly the area of contact between BPTI and trypsin, it is approximately the same.","Other surfaces, numbered 2 to 5, were picked by first picking one exposed, variable residue and then picking neighboring residues until a surface was defined. The choice of sets of residues shown in Table 34 is in no way exhaustive or unique; other sets of variable, surface residues can be picked. Set #2 is shown in stereo view, , including the a carbons of BPTI, the disulfide linkages, and the side groups of the set. We take the orientation of BPTI in  as a standard orientation and hereinafter refer to K15 as being at the top of the molecule, while the carboxy and amino termini are at the bottom.","Solvent accessibilities are useful, easily tabulated indicators of a residue's exposure. Solvent accessibilities must be used with some caution; small amino acids are under-represented and large amino acids over-represented. The user must consider what the solvent accessibility of a different amino acid would be when substituted into the structure of BPTI.","To create specific binding between a derivative of BPTI and HHMb, we will vary the residues in set #2. This set includes the twelve principal residues 17(R), 19(I), 21(Y), 27(A), 28(G), 29(L), 31(Q), 32(T), 34(V), 48(A), 49(E), and 52(M) (Sec. III.B). None of the residues in set #2 is completely conserved in the sample of sequences reported in Table 34; thus we can vary them with a high probability of retaining the underlying structure. Independent substitution at each of these twelve residues of the amino acid types observed at that residue would produce approximately 4.4\u00b710amino acid sequences and the same number of surfaces.","BPTI is a very basic protein. This property has been used in isolating and purifying BPTI and its homologues so that the high frequency of arginine and lysine residues may reflect bias in isolation and is not necessarily required by the structure. Indeed, SCI-III from contains seven more acidic than basic groups (SASA84).","Residue 17 is highly variable and fully exposed and can contain R, K, A, Y, H, F, L, M, T, G, Y, P, or S. All types of amino acids are seen: large, small, charged, neutral, and hydrophobic. That no acidic groups are observed may be due to bias in the sample.","Residue 19 is also variable and fully exposed, containing P, R, I, S, K, Q, and L.","Residue 21 is not very variable, containing F or Y in 31 of 33 cases and I and W in the remaining cases. The side group of Y21 fills the space between T32 and the main chain of residues 47 and 48. The OH at the tip of the Y side group projects into the solvent. Clearly one can vary the surface by substituting Y or F so that the surface is either hydrophobic or hydrophilic in that region. It is also possible that the other aromatic amino acid (viz. H) or the other hydrophobics (L, M, or V) might be tolerated.","Residue 27 most often contains A, but S, K, L, and T are also observed. On structural grounds, this residue will probably tolerate any hydrophilic amino acid and perhaps any amino acid.","Residue 28 is G in BPTI. This residue is in a turn, but is not in a conformation peculiar to glycine. Six other types of amino acids have been observed at this residue: K, N, Q, R, H, and N. Small side groups at this residue might not contact HHMb simultaneously with residues 17 and 34. Large side groups could interact with HHMb at the same time as residues 17 and 34. Charged side groups at this residue could affect binding of HHMb on the surface defined by the other residues of the principal set. Any amino acid, except perhaps P, should be tolerated.","Residue 29 is highly variable, most often contain ing L. This fully exposed position will probably tolerate almost any amino acid except, perhaps, P.","Residues 31, 32, and 34 are highly variable, exposed, and in extended conformations; any amino acid should be tolerated.","Residues 48 and 49 are also highly variable and fully exposed, any amino acid should be tolerated.","Residue 52 is in an a helix. Any amino acid, except perhaps P, might be tolerated.","Now we consider possible variation of the secondary set (Sec. 13.1.2) of residues that are in the neighborhood of the principal set. Neighboring residues that might be varied at later stages include 9(P), 11(T), 15(K), 16(A), 18(I), 20(R), 22(F), 24(N), 26(K), 35(Y), 47(S), 50(D), and 53(R).","Residue 9 is highly variable, extended, and exposed. Residue 9 and residues 48 and 49 are separated by a bulge caused by the ascending chain from residue 31 to 34. For residue 9 and residues 48 and 49 to contribute simultaneously to binding, either the target must have a groove into which the chain from 31 to 34 can fit, or all three residues (9, 48, and 49) must have large amino acids that effectively reduce the radius of curvature of the BPTI derivative.","Residue 11 is highly variable, extended, and exposed. Residue 11, like residue 9, is slightly far from the surface defined by the principal residues and will contribute to binding in the same circumstances.","Residue 15 is highly varied. The side group of residue 15 points away form the face defined by set #2. Changes of charge at residue 15 could affect binding on the surface defined by residue set #2.","Residue 16 is varied but points away from the surface defined by the principal set. Changes in charge at this residue could affect binding on the face defined by set #2.","Residue 18 is I in BPTI. This residue is in an extended conformation and is exposed. Five other amino acids have been observed at this residue: M, F, L, V, and T. Only T is hydrophilic. The side group points directly away from the surface defined by residue set #2. Substitution of charged amino acids at this residue could affect binding at surface defined by residue set #2.","Residue 20 is R in BPTI. This residue is in an extended conformation and is exposed. Four other amino acids have been observed at this residue: A, S, L, and Q. The side group points directly away from the surface defined by residue set #2. Alteration of the charge at this residue could affect binding at surface defined by residue set #2.","Residue 22 is only slightly varied, being Y, F, or H in 30 of 33 cases. Nevertheless, A, N, and S have been observed at this residue. Amino acids such as L, M, I, or Q could be tried here. Alterations at residue 22 may affect the mobility of residue 21; changes in charge at residue 22 could affect binding at the surface defined by residue set #2.","Residue 24 shows some variation, but probably can not interact with one molecule of the target simul taneously with all the residues in the principal set. Variation in charge at this residue might have an effect on binding at the surface defined by the principal set.","Residue 26 is highly varied and exposed. Changes in charge may affect binding at the surface defined by residue set #2; substitutions may affect the mobility of residue 27 that is in the principal set.","Residue 35 is most often Y, W has been observed. The side group of 35 is buried, but substitution of F or W could affect the mobility of residue 34.","Residue 47 is always T or S in the sequence sample used. The Oprobably accepts a hydrogen bond from the NH of residue 50 in the alpha helix. Nevertheless, there is no overwhelming steric reason to preclude other amino acid types at this residue. In particular, other amino acids the side groups of which can accept hydrogen bonds, viz. N, D, Q, and E, may be acceptable here.","Residue 50 is often an acidic amino acid, but other amino acids are possible.","Residue 53 is often R, but other amino acids have been observed at this residue. Changes of charge may affect binding to the amino acids in interaction set #2.","Stereo  shows the residues in set #2, plus R39. From , one can see that R39 is on the opposite side of BPTI form the surface defined by the residues in set #2. Therefore, variation at residue 39 at the same time as variation of some residues in set #2 is much less likely to improve binding that occurs along surface #2 than is variation of the other residues in set #2.","In addition to the twelve principal residues and 13 secondary residues, there are two other residues, 30(C) and 33(F), involved in surface #2 that we will probably not vary, at least not until late in the procedure. These residues have their side groups buried inside BPTI and are conserved. Changing these residues does not change the surface nearly so much as does changing residues in the principal set. These buried, conserved residues do, however, contribute to the surface area of surface #2. The surface of residue set #2 is comparable to the area of the trypsin-binding surface. Principal residues 17, 19, 21, 27, 28, 29, 31, 32, 34, 48, 49, and 52 have a combined solvent-accessible area of 946.9 A. Secondary residues 9, 11, 15, 16, 18, 20, 22, 24, 26, 35, 47, 50, and 53 have combined surface of 1041.7 A. Residues 30 and 33 have exposed surface totaling 38.2 A. Thus the three groups' combined surface is 2026.8 A.","Residue 30 is C in BPTI and is conserved in all homologous sequences. It should be noted, however, that C14\/C38 is conserved in all natural sequences, yet Marks et al. (MARK87) showed that changing both C14 and C38 to A,A or T,T yields a functional trypsin inhibitor. Thus it is possible that BPTI-like molecules will fold if C30 is replaced.","Residue 33 is F in BPTI and in all homologous sequences. Visual inspection of the BPTI structure suggests that substitution of Y, M, H, or L might be tolerated.","Having identified twenty residues that define a possible binding surface, we must choose some to vary first. Assuming a hypothetical affinity separation sensitivity, C, of 1 in 4\u00b710, we decide to vary six residues (leaving some margin for error in the actual base composition of variegated bases). To obtain maximal recognition, we choose residues from the principal set that are as far apart as possible. Table 36 shows the distances between the \u03b2 carbons of residues in the principal and peripheral set. R17 and V34 are at one end of the principal surface. Residues A27, G28, L29, A48, E49, and M52 are at the other end, about twenty Angstroms away; of these, we will vary residues 17, 27, 29, 34, and 48. Residues 28, 49, and 52 will be varied at later rounds.","Of the remaining principal residues, 21 is left to later variations. Among residues 19, 31, and 32, we arbitrarily pick 19 to vary.","Unlimited variation of six residues produces 6.4\u00b710amino acid sequences. By hypothesis, Cis 1 in 4\u00b710. Table 37 shows the programmed variegation at the chosen residues. The parental sequence is present as 1 part in 5.5\u00b710, but the least favored sequences are present at only 1 part in 4.2\u00b710. Among single-amino-acid substitutions from the PPBD, the least favored is F17-I19-A27-L29-V34-A48 and has a calculated abundance of 1 part in 1.6\u00b710. Using the optimal qfk codon, we can recover the parental sequence and all one-amino-acid substitutions to the PPBD if actual nt compositions come within 5% of programmed compositions. The number of transformants is M=1.0\u00b710(also by hypothesis), thus we will produce most of the programmed sequences.","The residue numbers of the preceding section are referred to mature BPTI (R1-P2- . . . -A58). Table 25 has residue numbers referring to the pre-M13CP-BPTI protein; all mature BPTI sequence numbers have been increased by the length of the signal sequence, i.e. 23. Thus in terms of the pre-OSP-PBD residue numbers, we wish to vary residues 40, 42, 50, 52, 57, and 71. A DNA subsequence containing all these codons is found between the (ApaI\/DraII\/PssI) sites at base 191 and the Sph I site at base 309 of the osp-pbd gene. Among ApaI, DraI, and PssI, ApaI is preferred because it recognizes six bases without any ambiguity. DraII and PssI, on the other hand, recognize six bases with two-fold ambiguity at two of the bases. The vgDNA will contain more DraII and PssI recognition sites at the varied locations than it will contain ApaI recognition sites. The unwanted extraneous cutting of the vgDNA by ApaI and SphI will eliminate a few sequences from our population. This is a minor problem, but by using the more specific enzyme (ApaI), we minimize the unwanted effects. The sequence shown in Table 37 illustrates an additional way in which gratuitous restriction sites can be avoided in some cases. The osp-ipbd gene had the codon GGC for g51; because we are varying both residue 50 and 52, it is possible to obtain an ApaI site. If we change the glycine codon to GGT, the ApaI site can no longer arise. ApaI recognizes the DNA sequence (GGGCC\/C).","Each piece of dsDNA to be synthesized needs six to eight bases added at either end to allow cutting with restriction enzymes and is shown in Table 37. The first synthetic base (before cutting with ApaI and SphI) is 184 and the last is 322. There are 142 bases to be synthesized. The center of the piece to the synthesized lies between Q54 and V57. The overlap can not include varied bases, so we choose bases 245 to 256 as the overlap that is 12 bases long. Note that the codon for F56 has been changed to TTC to increase the GC content of the overlap. The amino acids that are being varied are marked as X with a plus over them. Codons 57 and 71 are synthesized on the sense (bottom) strand. The design calls for \u201cqfk\u201d in the antisense strand, so that the sense strand contains (from 5\u2032 to 3\u2032) a) equal part C and A (i.e. the complement of k), b) (0.40 T, 0.22 A, 0.22 C, and 0.16 G) (i.e. the complement of f), and c) (0.26 T, 0.26 A, 0.30 C, and 0.18 G).","Each residue that is encoded by \u201cqfk\u201d has 21 possible outcomes, each of the amino acids plus stop. Table 12 gives the distribution of amino acids encoded by \u201cqfk\u201d, assuming 5% errors. The abundance of the parental sequence is the product of the abundances of R\u00d7I\u00d7A\u00d7L\u00d7V\u00d7A. The abundance of the least-favored sequence is 1 in 4.2\u00b710.","Olig#27 and olig#28 are annealed and extended with Klenow fragment and all four (nt)TPs. Both the ds synthetic DNA and RF pLG7 DNA are cut with both ApaI and SphI. The cut DNA is purified and the appropriate pieces ligated (See Sec. 14.1) and used to transform competent PE383. (Sec. 14.2). In order to generate a sufficient number of transformants, Vc is set to 5000 ml.\n\n","IPTG is added to the growth medium at 2.0 mM (the optimal level) and virions are harvested in the usual way. It is important to collect virions in a way that samples all or nearly all the transformants. Because F cells are used in the transformation, multiple infections do not pose a problem.","HHMb has a pI of 7.0 and we carry out chromatography at pH 8.0 so that HHMb is slightly negative while BPTI and most of its mutants are positive. HHMb is fixed (Sec. V.F) to a 2.0 ml column on Affi-Gel 10\u2122 or Affi-Gel 15\u2122 at 4.0 mg\/ml support matrix, the same density that is optimal for a column supporting trp.","We note that charge repulsion between BPTI and HHMb should not be a serious problem and does not impose any constraints on ions or solutes allowed as eluants. Neither BPTI nor HHMb have special requirements that constrain choice of eluants. The eluant of choice is KCl in varying concentrations.","To remove variants of BPTI with strong, indiscriminate binding for any protein or for the support matrix, we pass the variegated population of virions over a column that supports bovine serum albumin (BSA) before loading the population onto the {HHMb} column. Affi-Gel 10\u2122 or Affi-Gel 15\u2122 is used to immobilize BSA at the highest level the matrix will support. A 10.0 ml column is loaded with 5.0 ml of","Affi-Gel-linked-BSA; this column, called {BSA}, has V=5.0 ml. The variegated population of virions containing 10pfu in 1 ml (0.2\u00d7V) of 10 mM KCl, 1 mM phosphate, pH 8.0 buffer is applied to {BSA}. We wash {BSA} with 4.5 ml (0.9\u00d7V) of 50 mM KCl, 1 mM phosphate, pH 8.0 buffer. The wash with 50 mM salt will elute virions that adhere slightly to BSA but not virions with strong binding. The pooled effluent of the {BSA} column is 5.5 ml of approximately 13 mM KCl.","The column {HHMb} is first blocked by treatment with 10virions of M13(am429) in 100 ul of 10 mM KCl buffered to pH 8.0 with phosphate; the column is washed with the same buffer until ODreturns to base line or 2\u00d7Vhave passed through the column, whichever comes first. The pooled effluent from {BSA} is added to {HHMb} in 5.5 ml of 13 mM KCl, 1 mM phosphate, pH 8.0 buffer. The column is eluted in the following way:\n\n","If multiple infection should pose a problem that cannot be solved by growth at low multiple-of-infection on F cells, the following procedure can be employed to obviate the problem. Virions obtained from the affinity separation are infected into Fand cultured to amplify the genetic messages (Sec. V). CCC DNA is obtained either by harvesting RF DNA or by in vitro extension of primers annealed to ss phage DNA. The CCC DNA is used to transform F cells at a high ratio of cells to DNA. Individual virions obtained in this way should bear only proteins encoded by the DNA within.","The phagemid population is grown and chromato graphed three times and then examined for SBDs (Sec. V). In each separation cycle, phage from the last three fractions that contain viable phage are pooled with phage obtained by removing some of the support matrix as an inoculum. At each cycle, about 10phage are loaded onto the column and about 10phage are cultured for the next separation cycle. After the third separation cycle, SBD colonies are picked from the last fraction that contained viable phage.","Each of the SBDs is cultured and tested for retention on a Pep-Tie column supporting HHMb. The phage showing the greatest retention on the Pep-Tie {HHMb} column. This SBD! becomes the parental amino-acid sequence to the second variegation cycle.","Assume for the sake of argument that, in SBD!, R40 changed to D, I42 changed to Q, A50 changed to E, L52 remained L, and A71 changed to W (see Table 38). If so, a rational plan for the second round of variegation would be that which is set forth in Table 39. The residues to be varied are chosen by: a) choosing some of the residues in the principal set that were not varied in the first round (viz. residues 42, 44, 51, 54, 55, 72, or 75 of the fusion), and b) choosing some residues in the secondary set. Residues 51, 54, 55, and 72 are varied through all twenty amino acids and, unavoidably, stop. Residue 44 is only varied between Y and F. Some residues in the secondary set are varied through a restricted range; primarily to allow different charges (+, 0, \u2212) to appear. Residue 38 is varied through K, R, E, or G. Residue 41 is varied through I, V, K, or E. Residue 43 is varied through R, S, G, N, K, D, E, T, or A.","Now assume that in the most successful SBD of the second round of variegation (SBD-2!), residue 38 (K15 of BPTI) changed to E, 41 becomes V, 43 goes to N, 44 goes to F, 51 goes to F, 54 goes to S, 55 goes to A, and 72 goes to Q (see Table 40). A third round of variation is illustrated in Table 41; eight amino acids are varied. Those in the principal set, residues 40, 55, and 57, are varied through all twenty amino acids. Residue 32 is varied through P, Q, T, K, A, or E. Residue 34 is varied through T, P, Q, K, A, or E. Residue 44 is varied through F, L, Y, C, W, or stop. Residue 50 is varied through E, K, or Q. Residue 52 is varied through L, F, I, M, or V. The result of this variation is shown in Table 42.","This example is hypothetical. It is anticipated that more variegation cycles will be needed to achieve dissociation constants of 10M. It is also possible that more than three separation cycles will be needed in some variegation cycles. Real DNA chemistry and DNA synthesizers may have larger errors than our hypothetical 5% . If S>0.05, then we may not be able to vary six residues at once. Variation of 5 residues at once is certainly possible.","Design and Mutagenesis of a Class 1 Mini-Protein","To obtain a library of binding domains that are conformationally constrained by a single disulfide, we insert DNA coding for the following family of mini-proteins into the gene coding for a suitable OSP.",{"@attributes":{"id":"p-0891","num":"1185"},"chemistry":{"@attributes":{"id":"CHEM-US-00014","num":"00014"},"img":{"@attributes":{"id":"EMI-C00014","he":"11.85mm","wi":"69.93mm","file":"US07413537-20080819-C00014.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"Where",{"@attributes":{"id":"p-0893","num":"1187"},"chemistry":{"@attributes":{"id":"CHEM-US-00015","num":"00015"},"img":{"@attributes":{"id":"EMI-C00015","he":"5.84mm","wi":"10.41mm","file":"US07413537-20080819-C00015.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{},"figref":"FIG. 3","sub":["n ","1 "]},"X, X, X, X, X, X, X, and Xcan be varied independently; i.e. a different scheme of variegation could be used at each position. Xand Xare the least constrained residues and may be varied less than other positions.","Xand Xcan be, for example, one of the amino acids [E, K, T, and A]; this set of amino acids is preferred because: a) the possibility of positively charged, negatively charged, and neutral amino acids is provided, b) these amino acids can be provided in 1:1:1:1 ratio via the codon RMG (R=equimolar A and G, M=equimolar A and C), and c) these amino acids allow proper processing by signal peptidases.","One option for variegation of X, X, X, X, X, and Xis to vary all of these in the same way. For example, each of X, X, X, X, X, and Xcan be chosen from the set [F, S, Y, C, L, P, H, R, I, T, N, V, A, D, and G] which is encoded by the mixed codon NNT. Tables 10 and 130 compares libraries in which six codons have been varied either by NNT or NNK codons. NNT encodes 15 different amino acids and only 16 DNA sequences. Thus, there are 1.139\u00b710amino-acid sequences, no stops, and only 1.678\u00b710DNA sequences. A library of 10independent transformants will contain 99% of all possible sequences. The NNK library contains 6.4\u00b710sequences, but complete sampling requires a much larger number of independent transformants.","A CYS::Helix::Turn::Strand::CYS Unit","The parental Class 2 mini-proteins may be a naturally-occurring Class 2 mini-protein. It may also be a domain of a larger protein whose structure satisfies or may be modified so as to satisfy the criteria of a class 2 mini-protein. The modification may be a simple one, such as the introduction of a cysteine (or a pair of cysteines) into the base of a hairpin structure so that the hairpin may be closed off with a disulfide bond, or a more elaborate one, so as the modification of intermediate residues so as to achieve the hairpin structure. The parental class 2 mini-protein may also be a composite of structures from two or more naturally-occuring proteins, e.g., an a helix of one protein and a \u03b2 strand of a second protein.","One mini-protein motif of potential use comprises a disulfide loop enclosing a helix, a turn, and a return strand. Such a structure could be designed or it could be obtained from a protein of known 3D structure. Scorpion neurotoxin, variant 3, (ALMA83a, ALMA83b) (hereafter ScorpTx) contains a structure diagrammed in  that comprises a helix (residues N22 through N33), a turn (residues 33 through 35), and a return strand (residues 36 through 41). ScorpTx contains disulfides that join residues 12-65, 16-41, 25-46, and 29-48. CYSand CYS4, are quite close and could be joined by a disulfide without deranging the main chain.  shows CYSjoined to CYS. In addition, CYShas been changed to GLN. It is expected that a disulfide will form between 25 and 41 and that the helix shown will form; we know that the amino-acid sequence shown is highly compatible with this structure. The presence of GLY, GLY, and GLYgive the turn and extended strand sufficient flexibility to accommodate any changes needed around CYSto form the disulfide.","From examination of this structure (as found in entry 1SN3 of the Brookhaven Protein Data Bank), we see that the following sets of residues would be preferred for variegation:","Set 1",{"@attributes":{"id":"p-0900","num":"1194"},"tables":{"@attributes":{"id":"TABLE-US-00022","num":"00022"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"0pt","align":"left"}}],"thead":{"row":[{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":[{},{},{},"Naa\/",{}]},{"entry":["Residue","Codon","Allowed amino acids","Ndna"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1) T","NNG","L,R,M,V,S,P,T,A,Q,K,E,W,G.","13\/15",{}]},{"entry":{}},{"entry":["2) E","VHG","L,M,V,P,T,A,G,K,E","9\/9"]},{"entry":{}},{"entry":["3) A","VHG","L,M,V,P,T,A,G,K,E","9\/9"]},{"entry":{}},{"entry":["4) K","VHG","L,M,V,P,T,A,G,K,E","9\/9"]},{"entry":{}},{"entry":["5) G24","NNG","L,R,M,V,S,P,T,A,Q,K,E,W,G.","13\/15"]},{"entry":{}},{"entry":["6) E23","VHG","L,M,V,P,T,A,G,K,E","9\/9"]},{"entry":{}},{"entry":["7) Q34","VAS","H,Q,N,K,E,D","6\/6"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":"Note:"},{"entry":"Exponents on amino acids indicate multiplicity of codons."}]}}}}},"Positions 27, 28, 31, 32, 24, and 23 comprise one face of the helix. At each of these locations we have picked a variegating codon that a) includes the parental amino acid, b) includes a set of residues having a predominance of helix favoring residues, c) provides for a wide variety of amino acids, and d) leads to as even a distribution as possible. Position 34 is part of a turn. The side group of residue 34 could interact with molecules that contact the side groups of residues 27, 28, 31, 32, 24, and 23. Thus we allow variegation here and provide amino acids that are compatible with turns. The variegation shown leads to 6.65\u00b710amino acid sequences encoded by 8.85\u00b710DNA sequences.","Set 2",{"@attributes":{"id":"p-0902","num":"1196"},"tables":{"@attributes":{"id":"TABLE-US-00023","num":"00023"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"133pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"0pt","align":"left"}}],"thead":{"row":[{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":["Resi-",{},{},"Naa\/",{}]},{"entry":["due","Codon","Allowed amino acids","Ndna"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1) D","VHS","L,I,M,V,P,T,A,H,Q,N,K,D,E","13\/18",{}]},{"entry":{}},{"entry":["2) T","NNG","L,R,M,V,S,P,T,A,Q,K,E,W,G","13\/15"]},{"entry":{}},{"entry":["3) K","VHG","K,E,Q,P,T,A,L,M,V","9\/9"]},{"entry":{}},{"entry":["4) A","VHG","K,E,Q,P,T,A,L,M,V","9\/9"]},{"entry":{}},{"entry":["5) K","VHG","L,M,V,P,T,A,G,K,E","9\/9"]},{"entry":{}},{"entry":["6) S","RRT","S,N,D,G","4\/4"]},{"entry":{}},{"entry":["7) Y","NHT","Y,S,F,H,P,L,N,T,I,D,A,V","9\/9"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}}}},"Positions 26, 27, 30, 31, and 32 are variegated so as to enhance helix-favoring amino acids in the population. Residues 37 and 38 are in the return strand so that we pick different variegation codons. This variegation allows 4.43\u00b710amino-acid sequences and 7.08\u00b710DNA sequences. Thus a library that embodies this scheme can be sampled very efficiently.","Design and Mutagenesis of Class 3 Mini-Protein","Two Disulfide Bond Parental Mini-Proteins","Mini-proteins with two disulfide bonds may be modelled after the \u03b1-conotoxins, e.g., GI, GIA, GII, MI, and SI. These have the following conserved structure (SEQ ID NOs:20-31):",{"@attributes":{"id":"p-0905","num":"1199"},"chemistry":{"@attributes":{"id":"CHEM-US-00016","num":"00016"},"img":{"@attributes":{"id":"EMI-C00016","he":"13.38mm","wi":"74.25mm","file":"US07413537-20080819-C00016.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"Hashimoto et al. (HASH85) reported synthesis of twenty-four analogues of \u03b1 conotoxins GI, GII, and MI. Using the numbering scheme for GI (CYS at positions 2, 3, 7, and 13), Hashimoto et al. reported alterations at 4, 8, 10, and 12 that allows the proteins to be toxic. Almquist et al. (ALMQ89) synthesized [des-GLU] \u03b1 Conotoxin GI and twenty analogues. They found that substituting GLY for PROgave rise to two isomers, perhaps related to different disulfide bonding. They found a number of substitutions at residues 8 through 11 that allowed the protein to be toxic. Zafaralla et al. (ZAFA88) found that substituting PRO at position 9 gives an active protein. Each of the groups cited used only in vivo toxicity as an assay for the activity. From such studies, one can infer that an active protein has the parental 3D structure, but one can not infer that an inactive protein lacks the parental 3D structure.","Pardi et al. (PARD89) determined the 3D structure of \u03b1 Conotoxin GI obtained from venom by NMR. Kobayashi et al. (KOBA89) have reported a 3D structure of synthetic \u03b1 Conotoxin GI from NMR data which agrees with that of PARD89. We refer to  of Pardi et al.","Residue GLUis known to accomodate GLU, ARG, and ILE in known analogues or homologues. A preferred variegation codon is NNG that allows the set of amino acids [L,R,M,V,S,P,T,A,Q,K,E,W,G<stop>]. From  of Pardi et al. we see that the side group of GLUprojects into the same region as the strand comprising residues 9 through 12. Residues 2 and 3 are cysteines and are not to be varied. The side group of residue 4 points away from residues 9 through 12; thus we defer varying this residue until a later round. PROmay be needed to cause the correct disulfides to form; when GLY was substituted here the peptide folded into two forms, neither of which is toxic. It is allowed to vary PRO5, but not perferred in the first round.","No substitutions at ALAhave been reported. A preferred variegation codon is RMG which gives rise to ALA, THR, LYS, and GLU (small hydrophobic, small hydro philic, positive, and negative). CYSis not varied. We prefer to leave GLYas is, although a homologous protein having ALAis toxic. Homologous proteins having various amino acids at position 9 are toxic; thus, we use an NNT variegation codon which allows F,S,Y,C,L,P,H,R,I,T,N,V,A,D,G. We use NNT at positions 10, 11, and 12 as well. At position 14, following the fourth CYS, we allow ALA, THR, LYS, or GLU (via an RMG codon). This variegation allows 1.053\u00b710amino-acid sequences, encoded by \u224870% , \u224883% , and \u224895% of the allowed sequences. Other variegations are also appropriate. Concerning \u03b1 conotoxins, see, inter alia, ALMQ89, CRUZ85, GRAY83, GRAY84, and PARD89.","The parental mini-protein may instead be one of the proteins designated \u201cHybrid-I\u201d and \u201cHybrid-II\u201d by Pease et al. (PEAS90); cf.  of PEAS90. One preferred set of residues to vary for either protein consists of:",{"@attributes":{"id":"p-0911","num":"1205"},"tables":{"@attributes":{"id":"TABLE-US-00024","num":"00024"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"0pt","align":"left"}}],"thead":{"row":[{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":["Parenta",{},"Allowed",{},{}]},{"entry":["Amino","Variegated","Amino","AA seqs\/"]},{"entry":["acid","Codon","acids","DNA seqs"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["A5","RVT","A,D,G,T,N,S","6\/6",{}]},{"entry":{}},{"entry":["P6","VYT","P,T,A,L,I,V","6\/6"]},{"entry":{}},{"entry":["E7","RRS","E,D,N,K,S,R,G","7\/8"]},{"entry":{}},{"entry":["T8","VHG","T,P,A,L,M,V,Q,K,E","9\/9"]},{"entry":{}},{"entry":["A9","VHG","A,T,P,L,M,V,Q,K,E","9\/9"]},{"entry":{}},{"entry":["A10","RMG","A,E,K,T","4\/4"]},{"entry":{}},{"entry":["K12","VHG","K,Q,E,T,P,A,L,M,V","9\/9"]},{"entry":{}},{"entry":["Q16","NNG","L,R,S,W,P,Q,M,T, K,V,A,E,G","13\/15"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}}},"br":{},"sup":["6 ","7 ","7 "]},"At position 5 we provide amino acids that are compatible with a turn. At position 6 we allow ILE and VAL because they have branched \u03b2 carbons and make the chain ridged. At position 7 we allow ASP, ASN, and SER that often appear at the amino termini of helices. At positions 8 and 9 we allow several helix-favoring amino acids (ALA, LEU, MET, GLN, GLU, and LYS) that have differing charges and hydrophobicities because these are part of the helix proper. Position 10 is further around the edge of the helix, so we allow a smaller set (ALA, THR, LYS, and GLU). This set not only includes 3 helix-favoring amino acids plus THR that is well tolerated but also allows positive, negative, and neutral hydrophilic. The side groups of 12 and 16 project into the same region as the residues already recited. At these positions we allow a wide variety of amino acids with a bias toward helix-favoring amino acids.","The parental mini-protein may instead be a polypeptide composed of residues 9-24 and 31-40 of aprotinin and possessing two disulfides (Cys9-Cys22 and Cys14-Cys38). Such a polypeptide would have the same disulfide bond topology as \u03b1-conotoxin, and its two bridges would have spans of 12 and 17, respectively.","Residues 23, 24 and 31 are variegated to encode the amino acid residue set [G,S,R,D,N,H,P,T,A] so that a sequence that favors a turn of the necessary geometry is found. We use trypsin or anhydrotrypsin as the affinity molucule to enrich for GPs that display a mini-protein that folds into a stable structure similar to BPTI in the P1 region.","Three Disulfide Bond Parental Mini-Proteins","The cone snails (Conus) produce venoms (conotoxins) which are 10-30 amino acids in length and exceptionally rich in disulfide bonds. They are therefore archetypal mini-proteins. Novel mini-proteins with three disulfide bonds may be modelled after the \u03bc-(GIIIA, GIIIB, GIIIC) or \u03a9-(GVIA, GVIB, GVIC, GVIIA, GVIIB, MVIIA, MVIIB, etc.) conotoxins. The \u03bc-conotoxins have the following conserved structure (SEQ ID NO:32):",{"@attributes":{"id":"p-0916","num":"1210"},"chemistry":{"@attributes":{"id":"CHEM-US-00017","num":"00017"},"img":{"@attributes":{"id":"EMI-C00017","he":"16.34mm","wi":"73.24mm","file":"US07413537-20080819-C00017.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"No 3D structure of a \u03bc-conotoxin has been published. Hidaka et al. (HIDA90) have established the connectivity of the disulfides. The following diagram depicts geographutoxin I (also known as \u03bc-conotoxin GIIIA), whose sequence is SEQ ID NO:33.",{"@attributes":{"id":"p-0918","num":"1212"},"chemistry":{"@attributes":{"id":"CHEM-US-00018","num":"00018"},"img":{"@attributes":{"id":"EMI-C00018","he":"42.59mm","wi":"40.98mm","file":"US07413537-20080819-C00018.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{}},"The \u03a9-conotoxins may be represented as follows (SEQ ID NO:34 through 39):",{"@attributes":{"id":"p-0920","num":"1214"},"chemistry":{"@attributes":{"id":"CHEM-US-00019","num":"00019"},"img":{"@attributes":{"id":"EMI-C00019","he":"16.51mm","wi":"71.37mm","file":"US07413537-20080819-C00019.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{}},"Another mini-protein which may be used as a parental binding domain is the Cucurbita maxima trypsin inhibitor I (CMTI-I); CMTI-III is also appropriate. They are members of the squash family of serine protease inhibitors, which also includes inhibitors from summer squash, zucchini, and cucumbers (WIEC85). McWherter et al. (MCWH89) describe synthetic sequence-variants of the squash-seed protease inhibitors that have affinity for human leukocyte elastase and cathepsin G. Of course, any member of this family might be used.","CMTI-I is one of the smallest proteins known, comprising only 29 amino acids held in a fixed comformation by three disulfide bonds. The structure has been studied by Bode and colleagues using both X-ray diffraction (BODE89) and NMR (HOLA89a,b). CMTI-I is of ellipsoidal shape; it lacks helices or \u03b2-sheets, but consists of turns and connecting short polypeptide stretches. The disulfide pairing is Cys3-Cys20, Cys10-Cys22 and Cys16-Cys28. In the CMTI-I:trypsin complex studied by Bode et al., 13 of the 29 inhibitor residues are in direct contact with trypsin; most of them are in the primary binding segment Val2(P4)-Glu9 (P4\u2032) which contains the reactive site bond Arg5(P1)-Ile6 and is in a conformation observed also for other serine proteinase inhibitors.","CMTI-I has a Kfor trypsin of \u22481.5\u00b710M. McWherter et al. suggested substitution of \u201cmoderately bulky hydrophobic groups\u201d at P1 to confer HLE specificity. They found that a wider set of residues (VAL, ILE, LEU, ALA, PHE, MET, and GLY) gave detectable binding to HLE. For cathepsin G, they expected bulky (especially aromatic) side groups to be strongly preferred. They found that PHE, LEU, MET, and ALA were functional by their criteria; they did not test TRP, TYR, or HIS. (Note that ALA has the second smallest side group available.)","A preferred initial variegation strategy would be to vary some or all of the residues ARG, VAL, PRO, ARG, ILE, LEU, MET, GLU, LYS, HIS, GLY, TYR, and GLY. If the target were HNE, for example, one could synthesize DNA embodying the following possibilities:",{"@attributes":{"id":"p-0925","num":"1219"},"tables":{"@attributes":{"id":"TABLE-US-00025","num":"00025"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"0pt","align":"left"}}],"thead":{"row":[{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":[{},"vg","Allowed","#AA seqs\/",{}]},{"entry":["Parental","Codon","amino acids","#DNA seqs"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["ARG1","VNT","R,S,L,P,H,I,T,N,V,A,D,G","12\/12",{}]},{"entry":{}},{"entry":["VAL2","NWT","V,I,L,F,Y,H,N,D","8\/8"]},{"entry":{}},{"entry":["PRO4","VYT","P,L,T,I,A,V","6\/6"]},{"entry":{}},{"entry":["ARG5","VNT","R,S,L,P,H,I,T,N,V,A,D,G","12\/12"]},{"entry":{}},{"entry":["ILE6","NNK","all 20","20\/31"]},{"entry":{}},{"entry":["LEU7","VWG","L,Q,M,K,V,E","6\/6"]},{"entry":{}},{"entry":["TYR27","NAS","Y,H,Q,N,K,D,E","7\/8"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}}},"br":{},"sup":["6 ","7 ","7 "]},"Other inhibitors of this family include:\n\n","Another mini-protein that may be used as an initial potential binding domain is the heat-stable enterotoxins derived from some enterotoxogenic , and other bacteria (GUAR89). These mini-proteins are known to be secreted from and are extremely stable. Works related to synthesis, cloning, expression and properties of these proteins include: BHAT86, SEKI85, SHIM87, TAKA85, TAKE90, THOM85a,b, YOSH85, DALL90, DWAR89, GARI87, GUZM89, GUZM90, HOUG84, KUB089, KUPE90, OKAM87, OKAM88, and OKAM90.","Another preferred IPBD is crambin or one of its homologues, the phoratoxins and ligatoxins (LECO87). These proteins are secreted in plants. The 3D structure of crambin has been determined. NMR data on homologues indicate that the 3D structure is conserved. Residues thought to be on the surface of crambin, phoratoxin, or ligatoxin are preferred residues to vary.","A Mini-Protein Having a Cross-Link Consisting of Cu(II), One Cysteine, Two Histidines, and One Methionine.","Sequences such as HIS-ASN-GLY-MET-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-HIS-ASN-GLY-CYS (SEQ ID NO:40) and CYS-ASN-GLY-MET-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-HIS-ASN-GLY-HIS (SEQ ID NO:41) are likely to combine with Cu(II) to form structures as shown in the diagram:",{"@attributes":{"id":"p-0930","num":"1235"},"chemistry":{"@attributes":{"id":"CHEM-US-00020","num":"00020"},"img":{"@attributes":{"id":"EMI-C00020","he":"85.43mm","wi":"69.93mm","file":"US07413537-20080819-C00020.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{}},{"@attributes":{"id":"p-0931","num":"1236"},"chemistry":{"@attributes":{"id":"CHEM-US-00021","num":"00021"},"img":{"@attributes":{"id":"EMI-C00021","he":"34.21mm","wi":"35.22mm","file":"US07413537-20080819-C00021.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{}},"The variable amino acids are held so that they have limited flexibility. This cross-linkage has some differences from the disulfide linkage. The separation between Cand Cis greater than the separation of the Cs of a cystine. In addition, the interaction of residues 1 through 4 and 11 through 14 with the metal ion are expected to limit the motion of residues 5 through 10 more than a disulfide between rsidues 4 and 11. A single disulfide bond exerts strong distance constrains on the \u03b1 carbons of the joined residues, but very little directional constraint on, for example, the vector from N to C in the main-chain.","For the desired sequence, the side groups of residues 5 through 10 can form specific interactions with the target. Other numbers of variable amino acids, for example, 4, 5, 7, or 3, are appropriate. Larger spans may be used when the enclosed sequence contains segments having a high potential to form \u03b1 helices or other secondary structure that limits the conformational freedom of the polypeptide main chain. Whereas a mini-protein having four CYSs could form three distinct pairings, a mini-protein having two HISs, one MET, and one CYS can form only two distinct complexes with Cu. These two structures are related by mirror symmetry through the Cu. Because the two HISs are distinguishable, the structures are different.","When such metal-containing mini-proteins are displayed on filamentous phage, the cells that produce the phage can be grown in the presence of the appropriate metal ion, or the phage can be exposed to the metal only after they are separated from the cells.","A Mini-Protein Having a Cross-Link Consisting of Zn(II) and Four Cysteines","A cross link similar to the one shown in Example XV is exemplified by the Zinc-finger proteins (GIBS88, GAUS87, PARR88, FRAN87, CHOW87, HARD90). One family of Zinc-fingers has two CYS and two HIS residues in conserved positions that bind Zn (PARR88, FRAN87, CHOW87, EVAN88, BERG88, CHAV88). Gibson et al. (GIBS88) review a number of sequences thought to form zinc-fingers and propose a three-dimensional model for these compounds. Most of these sequences have two CYS and two HIS residues in conserved positions, but some have three CYS and one HIS residue. Gauss et al. (GAUS87) also report a zinc-finger protein having three CYS and one HIS residues that bind zinc. Hard et al. (HARD90) report the 3D structure of a protein that comprises two zinc-fingers, each of which has four CYS residues. All of these zinc-binding proteins are stable in the reducing intracellular environment.","One preferred example of a CYS::zinc cross linked mini-protein comprises residues 440 to 461 of the sequence shown in  of HARD90. The resiudes 444 through 456 (SEQ ID NO:43) may be variegated. One such variegation is as follows:",{"@attributes":{"id":"p-0937","num":"1242"},"tables":{"@attributes":{"id":"TABLE-US-00026","num":"00026"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"0pt","align":"left"}}],"thead":{"row":[{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]},{"entry":[{},{},{},"#AA\/",{}]},{"entry":[{},"Parental","Allowed","#DNA"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"SER444","SER, ALA","2\/2",{}]},{"entry":[{},{}]},{"entry":[{},"ASP445","ASP, ASN, GLU, LYS","4\/4"]},{"entry":[{},{}]},{"entry":[{},"GLU446","GLU, LYS, GLN","3\/3"]},{"entry":[{},{}]},{"entry":[{},"ALA447","ALA, THR, GLY, SER","4\/4"]},{"entry":[{},{}]},{"entry":[{},"SER448","SER, ALA","2\/2"]},{"entry":[{},{}]},{"entry":[{},"GLY449","GLY, SER, ASN, ASP","4\/4"]},{"entry":[{},{}]},{"entry":[{},"CY5450","CYS, PHE, ARG, LEU","4\/4"]},{"entry":[{},{}]},{"entry":[{},"HIS451","HIS, GLN, ASN, LYS, ASP, GLU","6\/6"]},{"entry":[{},{}]},{"entry":[{},"TYR452","TYR, PHE, HIS, LEU","4\/4"]},{"entry":[{},{}]},{"entry":[{},"GLY453","GLY, SER, ASN, ASP","4\/4"]},{"entry":[{},{}]},{"entry":[{},"VAL454","VAL, ALA, ASP, GLY, SER, ASN,","8\/8"]},{"entry":[{},{},"THR, ILE"]},{"entry":[{},{}]},{"entry":[{},"LEU455","LEU, HIS, ASP, VAL","4\/4"]},{"entry":[{},{}]},{"entry":[{},"THR456","THR, ILE, ASN, SER","4\/4"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}}}},"br":{},"sup":["7 ","8 ","8 "]},{"@attributes":{"id":"p-0938","num":"1243"},"tables":{"@attributes":{"id":"TABLE-US-00027","num":"00027"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 1"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Single-letter codes."},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"182pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Single-letter code is used for proteins:"]},{"entry":[{},"a = ALA c = CYS d = ASP e = GLU f = PHE"]},{"entry":[{},"g = GLY h = HIS i = ILE k = LYS l = LEU"]},{"entry":[{},"m = MET n = ASN p = PRO q = GLN r = ARG"]},{"entry":[{},"s = SER t = THR v = VAL w = TRP y = TYR"]},{"entry":[{},". = STOP\u2003\u2003\u2003\u2003* = any amino acid"]},{"entry":[{},"b = n or d"]},{"entry":[{},"z = e or q"]},{"entry":[{},"x = any amino acid"]},{"entry":[{},"Single-letter IUB codes for DNA:"]},{"entry":[{},"T, C, A, G stand for themselves"]},{"entry":[{},"M for A or C"]},{"entry":[{},"R for puRines A or G"]},{"entry":[{},"W for A or T"]},{"entry":[{},"S for C or G"]},{"entry":[{},"Y for pYrimidines T or C"]},{"entry":[{},"K for G or T"]},{"entry":[{},"V for A, C, or G \u2003(not T)"]},{"entry":[{},"H for A, C, or T \u2003(not G)"]},{"entry":[{},"D for A, G, or T \u2003(not C)"]},{"entry":[{},"B for C, G, or T \u2003(not A)"]},{"entry":[{},"N for any base."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0939","num":"1244"},"tables":{"@attributes":{"id":"TABLE-US-00028","num":"00028"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Preferred Outer-Surface Proteins"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Preferred",{}]},{"entry":[{},"Genetic","Outer-Surface"]},{"entry":[{},"Package","Protein","Reason for preference"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"M13","coat protein","a) exposed amino terminus,"]},{"entry":[{},{},{},"(gpVIII)"]},{"entry":[{},{},{},"b) predictable post-"]},{"entry":[{},{},{},"translational"]},{"entry":[{},{},{},"processing,"]},{"entry":[{},{},{},"c) numerous copies in"]},{"entry":[{},{},{},"virion."]},{"entry":[{},{},{},"d) fusion data available"]},{"entry":[{},{},"gp III","a) fusion data available."]},{"entry":[{},{},{},"b) amino terminus exposed."]},{"entry":[{},{},{},"c) working example"]},{"entry":[{},{},{},"available."]},{"entry":[{},"PhiX174","G protein","a) known to be on virion"]},{"entry":[{},{},{},"exterior,"]},{"entry":[{},{},{},"b) small enough that"]},{"entry":[{},{},{},"the G-ipbd gene can"]},{"entry":[{},{},{},"replace H gene."]},{"entry":[{},{"i":"E. coli"},"LamB","a) fusion data available,"]},{"entry":[{},{},{},"b) non-essential."]},{"entry":[{},{},"OmpC","a) topological model"]},{"entry":[{},{},{},"b) non-essential; abundant"]},{"entry":[{},{},"OmpA","a) topological model"]},{"entry":[{},{},{},"b) non-essential; abundant"]},{"entry":[{},{},{},"c) homologues in other genera"]},{"entry":[{},{},"OmpF","a) topological model"]},{"entry":[{},{},{},"b) non-essential; abundant"]},{"entry":[{},{},"PhoE","a) topological model"]},{"entry":[{},{},{},"b) non-essential; abundant"]},{"entry":[{},{},{},"c) inducible"]},{"entry":[{},{"i":"B. subtilis"},"CotC","a) no post-translational"]},{"entry":[{},"spores",{},"processing,"]},{"entry":[{},{},{},"b) distinctive sdequence"]},{"entry":[{},{},{},"that causes protein to"]},{"entry":[{},{},{},"localize in spore coat,"]},{"entry":[{},{},{},"c) non-essential."]},{"entry":[{},{},"CotD","Same as for CotC."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0940","num":"1245"},"tables":{"@attributes":{"id":"TABLE-US-00029","num":"00029"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 3"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Ambiguous DNA for AA_seq2"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00022","num":"00022"},"img":{"@attributes":{"id":"EMI-C00022","he":"11.68mm","wi":"69.68mm","file":"US07413537-20080819-C00022.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00023","num":"00023"},"img":{"@attributes":{"id":"EMI-C00023","he":"11.68mm","wi":"69.77mm","file":"US07413537-20080819-C00023.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00024","num":"00024"},"img":{"@attributes":{"id":"EMI-C00024","he":"11.68mm","wi":"69.68mm","file":"US07413537-20080819-C00024.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00025","num":"00025"},"img":{"@attributes":{"id":"EMI-C00025","he":"11.68mm","wi":"69.68mm","file":"US07413537-20080819-C00025.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00026","num":"00026"},"img":{"@attributes":{"id":"EMI-C00026","he":"11.60mm","wi":"69.68mm","file":"US07413537-20080819-C00026.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00027","num":"00027"},"img":{"@attributes":{"id":"EMI-C00027","he":"9.23mm","wi":"69.60mm","file":"US07413537-20080819-C00027.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00028","num":"00028"},"img":{"@attributes":{"id":"EMI-C00028","he":"11.94mm","wi":"69.77mm","file":"US07413537-20080819-C00028.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00029","num":"00029"},"img":{"@attributes":{"id":"EMI-C00029","he":"11.94mm","wi":"69.85mm","file":"US07413537-20080819-C00029.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00030","num":"00030"},"img":{"@attributes":{"id":"EMI-C00030","he":"11.94mm","wi":"69.85mm","file":"US07413537-20080819-C00030.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00031","num":"00031"},"img":{"@attributes":{"id":"EMI-C00031","he":"9.23mm","wi":"69.68mm","file":"US07413537-20080819-C00031.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00032","num":"00032"},"img":{"@attributes":{"id":"EMI-C00032","he":"9.23mm","wi":"69.68mm","file":"US07413537-20080819-C00032.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00033","num":"00033"},"img":{"@attributes":{"id":"EMI-C00033","he":"10.24mm","wi":"69.77mm","file":"US07413537-20080819-C00033.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00034","num":"00034"},"img":{"@attributes":{"id":"EMI-C00034","he":"10.24mm","wi":"69.77mm","file":"US07413537-20080819-C00034.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00035","num":"00035"},"img":{"@attributes":{"id":"EMI-C00035","he":"7.87mm","wi":"69.77mm","file":"US07413537-20080819-C00035.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00036","num":"00036"},"img":{"@attributes":{"id":"EMI-C00036","he":"7.70mm","wi":"69.77mm","file":"US07413537-20080819-C00036.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00037","num":"00037"},"img":{"@attributes":{"id":"EMI-C00037","he":"10.24mm","wi":"69.77mm","file":"US07413537-20080819-C00037.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00038","num":"00038"},"img":{"@attributes":{"id":"EMI-C00038","he":"10.24mm","wi":"76.03mm","file":"US07413537-20080819-C00038.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}}}},{"@attributes":{"id":"p-0941","num":"1246"},"tables":{"@attributes":{"id":"TABLE-US-00030","num":"00030"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 4"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Table of Restriction Enzyme Suppliers"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"140pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Suppliers:","Sigma Chemical Co."]},{"entry":[{},{},"P.O.Box 14508"]},{"entry":[{},{},"St. Louis, Mo. 63178"]},{"entry":[{},{},"Bethesda Research Laboratories"]},{"entry":[{},{},"P.O.Box 6009"]},{"entry":[{},{},"Gaithersburg, Maryland, 20877"]},{"entry":[{},{},"Boehringer Mannheim Biochemicals"]},{"entry":[{},{},"7941 Castleway Drive"]},{"entry":[{},{},"Indianapolis, Indiana, 46250"]},{"entry":[{},{},"International Biochemicals, Inc."]},{"entry":[{},{},"P.O.Box 9558"]},{"entry":[{},{},"New Haven, Connecticutt, 06535"]},{"entry":[{},{},"New England BioLabs"]},{"entry":[{},{},"32 Tozer Road"]},{"entry":[{},{},"Beverly, Massachusetts, 01915"]},{"entry":[{},{},"Promega"]},{"entry":[{},{},"2800 S. Fish Hatchery Road"]},{"entry":[{},{},"Madison, Wisconsin, 53711"]},{"entry":[{},{},"Stratagene Cloning Systems"]},{"entry":[{},{},"11099 North Torrey Pines Road"]},{"entry":[{},{},"La Jolla, California, 92037"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0942","num":"1247"},"tables":{"@attributes":{"id":"TABLE-US-00031","num":"00031"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 5"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Potential sites in ipbd gene."},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Summary of cuts"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Enz = % Acc I has 3 elective sites:","96 169 281"]},{"entry":[{},"Enz = Afl II has 1 elective sites:","19"]},{"entry":[{},"Enz = Apa I has 2 elective sites:","102 103"]},{"entry":[{},"Enz = Asu II has 1 elective sites:","381"]},{"entry":[{},"Enz = Ava III has 1 elective sites:","314"]},{"entry":[{},"Enz = BspM II has 1 elective sites:","72"]},{"entry":[{},"Enz = BssH II has 2 elective sites:","67 115"]},{"entry":[{},"Enz = % BstX I has 1 elective sites:","323"]},{"entry":[{},"Enz = +Dra II has 3 elective sites:","102 103 226"]},{"entry":[{},"Enz = +EcoN I has 2 elective sites:","62 94"]},{"entry":[{},"Enz = +Esp I has 2 elective sites:","57 187"]},{"entry":[{},"Enz = Hind III has 6 elective sites:","9 23 60 287 361 386"]},{"entry":[{},"Enz = Kpn I has 1 elective sites:","48"]},{"entry":[{},"Enz = Mlu I has 1 elective sites:","314"]},{"entry":[{},"Enz = Nar I has 2 elective sites:","238 343"]},{"entry":[{},"Enz = Nco I has 1 elective sites:","323"]},{"entry":[{},"Enz = Nhe I has 3 elective sites:","25 289 388"]},{"entry":[{},"Enz = Nru I has 2 elective sites:","38 65"]},{"entry":[{},"Enz = +PflM I has 1 elective sites:","94"]},{"entry":[{},"Enz = PmaC I has 1 elective sites:","228"]},{"entry":[{},"Enz = +PpuM I has 2 elective sites:","102 226"]},{"entry":[{},"Enz = +Rsr II has 1 elective sites:","102"]},{"entry":[{},"Enz = +Sfi I has 2 elective sites:","24 261"]},{"entry":[{},"Enz = Spe I has 3 elective sites:","12 45 379"]},{"entry":[{},"Enz = Sph I has 1 elective sites:","221"]},{"entry":[{},"Enz = Stu I has 5 elective sites:","23 70 150 287 386"]},{"entry":[{},"Enz = % Sty I has 6 elective sites:","11 44 143 263 323 383"]},{"entry":[{},"Enz = Xba I has 1 elective sites:","84"]},{"entry":[{},"Enz = Xho I has 1 elective sites:","85"]},{"entry":[{},"Enz = Xma III has 3 elective sites:","70 209 242"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Enzymes not cutting ipbd"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Avr II","BamH I","Bcl I","BstE II"]},{"entry":[{},"EcoR I","EcoR V","Hpa I","Not I"]},{"entry":[{},"Sac I","Sal I","Sau I","Sma I"]},{"entry":[{},"Xma I"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0943","num":"1248"},"tables":{"@attributes":{"id":"TABLE-US-00032","num":"00032"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 6"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Exposure of amino acid types in T4 lzm & HEWL."},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"175pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["HEADER","HYDROLASE (O-GLYCOSYL) 18-AUG-86 2LZM"]},{"entry":["COMPND","LYSOZYME (E.C.3.2.1.17)"]},{"entry":["AUTHOR","L. H. WEAVER, B. W. MATTHEWS"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Coordinates from Brookhaven Protein Data Bank: 1LYM."]},{"entry":[{},"Only Molecule A was considered."]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"175pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["HEADER","HYDROLASE (O-GLYCOSYL) 29-JUL-82 1LYM"]},{"entry":["COMPND","LYSOZYME (E.C.3.2.1.17)"]},{"entry":["AUTHOR","J. HOGLE, S. T. RAO, M. SUNDARALINGAM"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Solvent radius =\u20031.40 \u2003Atomic radii in Table 7."]},{"entry":[{},"Surface area measured in \u212b."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"168pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Max"]},{"entry":[{},"exposed"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Type","N","<area>","sigma","max","min","(fraction)"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"7","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["ALA","27","211.0","1.47","214.3","207.1","\u200285.1(0.40)"]},{"entry":["CYS","10","239.8","3.56","245.5","234.4","\u200238.3(0.16)"]},{"entry":["ASP","17","271.1","5.36","281.4","262.5","127.1(0.47)"]},{"entry":["GLU","10","297.2","5.78","304.9","285.4","100.7(0.34)"]},{"entry":["PHE","8","316.6","5.92","325.4","307.5","\u200299.8(0.32)"]},{"entry":["GLY","23","185.5","1.31","188.3","183.3","\u200291.9(0.50)"]},{"entry":["HIS","2","297.7","3.23","301.0","294.5","\u200232.9(0.11)"]},{"entry":["ILE","16","278.1","3.61","285.6","269.6","\u200257.5(0.21)"]},{"entry":["LYS","19","309.2","5.38","321.9","300.1","147.1(0.48)"]},{"entry":["LEU","24","282.6","6.75","304.0","269.8","109.9(0.39)"]},{"entry":["MET","7","293.0","5.70","299.5","283.1","\u200288.2(0.30)"]},{"entry":["ASN","26","273.0","5.75","285.1","262.6","143.4(0.53)"]},{"entry":["PRO","5","239.9","2.75","242.1","234.6","128.7(0.54)"]},{"entry":["GLN","8","299.5","4.75","305.8","291.5","145.9(0.49)"]},{"entry":["ARG","24","344.7","8.66","355.8","326.7","240.7(0.70)"]},{"entry":["SER","16","228.6","3.59","236.6","223.3","\u200298.2(0.43)"]},{"entry":["THR","18","250.3","3.89","257.2","244.2","139.9(0.56)"]},{"entry":["VAL","15","254.3","4.05","261.8","245.7","111.1(0.44)"]},{"entry":["TRP","9","359.4","3.38","366.4","355.1","102.0(0.28)"]},{"entry":["TYR","9","335.8","4.97","342.0","325.0","\u200272.6(0.22)"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0944","num":"1249"},"tables":{"@attributes":{"id":"TABLE-US-00033","num":"00033"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 7"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Atomic radii"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"119pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"\u212b"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"119pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"C","1.70"]},{"entry":[{},"O","1.52"]},{"entry":[{},"N","1.55"]},{"entry":[{},"Other atoms","1.80"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0945","num":"1250"},"tables":{"@attributes":{"id":"TABLE-US-00034","num":"00034"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 8"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Fraction of DNA molecules having"},{"entry":"n non-parental bases when"},{"entry":"reagents that have fraction"},{"entry":"M of parental nucleotode."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["M",".9965",".97716",".92612",".8577",".79433",".63096"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"7","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["f0",".9000",".5000",".1000",".0100",".0010",".000001"]},{"entry":["f1",".09499",".35061",".2393",".04977",".00777",".0000175"]},{"entry":["f2",".00485",".1188",".2768",".1197",".0292",".000149"]},{"entry":["f3",".00016",".0259",".2061",".1854",".0705",".000812"]},{"entry":["f4",".000004",".00409",".1110",".2077",".1232",".003207"]},{"entry":["f8","0.","2 \u00b7 10",".00096",".0336",".1182",".080165"]},{"entry":["f16","0.","0.","0.","5 \u00b7 10",".00006",".027281"]},{"entry":["f23","0.","0.","0.","0.","0.",".0000089"]},{"entry":["most","0","0","2","5","7","12"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}},{"entry":"\u201cmost\u201d is the value of n having the highest probability."}]}}]}}},{"@attributes":{"id":"p-0946","num":"1251"},"tables":{"@attributes":{"id":"TABLE-US-00035","num":"00035"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 9"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"best vgCodon"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"\u2003Program \u201cFind Optimum vgCodon.\u201d"]},{"entry":[{},"\u2003INITIALIZE-MEMORY-OF-ABUNDANCES"]},{"entry":[{},"\u2003DO ( t1 = 0.21 to 0.31 in steps of 0.01 )"]},{"entry":[{},"\u2003. DO ( c1 = 0.13 to 0.23 in steps of 0.01 )"]},{"entry":[{},"\u2003. . DO ( a1 = 0.23 to 0.33 in steps of 0.01 )"]},{"entry":[{},"Comment calculate g1 from other concentrations"]},{"entry":[{},"\u2003. . . g1 = 1.0 \u2212 t1 \u2212 c1 \u2212 a1"]},{"entry":[{},"\u2003. . . IF( g1 .ge. 0.15 )"]},{"entry":[{},"\u2003. . . . DO ( a2 = 0.37 to 0.50 in steps of 0.01 )"]},{"entry":[{},"\u2003. . . . . DO ( c2 = 0.12 to 0.20 in steps of 0.01 )"]},{"entry":[{},"Comment Force D+E = R + K"]},{"entry":[{},"\u2003. . . . . . g2 = (g1*a2 \u2212.5*a1*a2)\/(c1+0.5*a1)"]},{"entry":[{},"Comment Calc t2 from other concentrations."]},{"entry":[{},"\u2003. . . . . . t2 = 1. \u2212 a2 \u2212 c2 \u2212 g2"]},{"entry":[{},"\u2003. . . . . . IF(g2.gt. 0.1.and. t2.gt.0.1)"]},{"entry":[{},"\u2003. . . . . . . CALCULATE-ABUNDANCES"]},{"entry":[{},"\u2003. . . . . . . COMPARE-ABUNDANCES-TO-PREVIOUS-ONES"]},{"entry":[{},"\u2003. . . . . . ..end_IF_block"]},{"entry":[{},"\u2003. . . . . ..end_DO_loop ! c2"]},{"entry":[{},"\u2003. . . . ..end_DO_loop ! a2"]},{"entry":[{},"\u2003. . . ..end_IF_block ! if g1 big enough"]},{"entry":[{},"\u2003. . ..end_DO_loop ! a1"]},{"entry":[{},"\u2003. ..end_DO_loop ! c1"]},{"entry":[{},"\u2003..end_DO_loop ! t1"]},{"entry":[{},"\u2003WRITE the best distribution and the abundances."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0947","num":"1252"},"tables":{"@attributes":{"id":"TABLE-US-00036","num":"00036"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 10"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Abundances obtained"},{"entry":"from various vgCodons"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"A. Optimized qfk Codon, Restrained by [D] + [E] = [K] + [R]"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"T","C","A","G"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"6","align":"center","rowsep":"1"}}]},{"entry":[{},"1",".26",".18",".26",".30","q"]},{"entry":[{},"2",".22",".16",".40",".22","f"]},{"entry":[{},"3",".5",".0",".0",".5","k"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"6","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Amino",{},"Amino",{}]},{"entry":["acid","Abundance","acid","Abundance"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["A","4.80%","C","2.86%"]},{"entry":["D","6.00%","E","6.00%"]},{"entry":["F","2.86%","G","6.60%"]},{"entry":["H","3.60%","I","2.86%"]},{"entry":["K","5.20%","L","6.82%"]},{"entry":["M","2.86%","N","5.20%"]},{"entry":["P","2.88%","Q","3.60%"]},{"entry":["R","6.82%","S","7.02% mfaa"]},{"entry":["T","4.16%","V","6.60%"]},{"entry":["W","2.86% lfaa","Y","5.20%"]},{"entry":["stop","5.20%"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"[D] + [E] \u2261 [K] + [R] = .12"},{"entry":"ratio = Abun(W)\/Abun(S) = 0.4074"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"j","(1\/ratio)","(ratio)","stop-free"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]},{"entry":[{},"1","2.454",".4074",".9480"]},{"entry":[{},"2","6.025",".1660",".8987"]},{"entry":[{},"3","14.788",".0676",".8520"]},{"entry":[{},"4","36.298",".0275",".8077"]},{"entry":[{},"5","89.095",".0112",".7657"]},{"entry":[{},"6","218.7","4.57 \u00b7 10\u22123",".7258"]},{"entry":[{},"7","536.8","1.86 \u00b7 10\u22123",".6881"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"B. Unrestrained, optimized"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"70pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"T","C","A","G"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":["1",".27",".19",".27",".27"]},{"entry":["2",".21",".15",".43",".21"]},{"entry":["3",".5",".0",".0",".5"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Amino",{},"Amino",{}]},{"entry":["acid","Abundance","acid","Abundance"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["A","4.05%","C","2.84%"]},{"entry":["D","5.81%","E","5.81%"]},{"entry":["F","2.84%","G","5.67%"]},{"entry":["H","4.08%","I","2.84%"]},{"entry":["K","5.81%","L","6.83%"]},{"entry":["M","2.84%","N","5.81%"]},{"entry":["P","2.85%","Q","4.08%"]},{"entry":["R","6.83%","S","6.89% mfaa"]},{"entry":["T","4.05%","V","5.67%"]},{"entry":["W","2.84% lfaa","Y","5.81%"]},{"entry":["stop","5.81%"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"[D] + [E] = 0.1162 [K] + [R] = 0.1264"},{"entry":"ratio = Abun(W)\/Abun(S) = 0.41176"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"j","(1\/ratio)","(ratio)","stop-free"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]},{"entry":[{},"1","2.4286",".41176",".9419"]},{"entry":[{},"2","5.8981",".16955",".8872"]},{"entry":[{},"3","14.3241",".06981",".8356"]},{"entry":[{},"4","34.7875",".02875",".7871"]},{"entry":[{},"5","84.4849",".011836",".74135"]},{"entry":[{},"6","205.180",".004874",".69828"]},{"entry":[{},"7","498.3","2.007 \u00b7 10",".6577"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"C. Optimized NNT"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"63pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"T","C","A","G"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":["1",".2071",".2929",".2071",".2929"]},{"entry":["2",".2929",".2071",".2929",".2071"]},{"entry":["3","1.",".0",".0",".0"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Amino",{},"Amino",{}]},{"entry":["acid","Abundance","acid","Abundance"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["A","6.06%","C","4.29% lfaa"]},{"entry":["D","8.58%","E","none"]},{"entry":["F","6.06%","G","6.06%"]},{"entry":["H","8.58%","I","6.06%"]},{"entry":["K","none","L","8.58%"]},{"entry":["M","none","N","6.06%"]},{"entry":["P","6.06%","Q","none"]},{"entry":["R","6.06%","S","8.58% mfaa"]},{"entry":["T","4.29% lfaa","V","8.58%"]},{"entry":["W","none","Y","6.06%"]},{"entry":["stop","none"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"j","(1\/ratio)","(ratio)","stop-free"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]},{"entry":[{},"1","2.0",".5","1."]},{"entry":[{},"2","4.0",".25","1."]},{"entry":[{},"3","8.0",".125","1."]},{"entry":[{},"4","16.0",".0625","1."]},{"entry":[{},"5","32.0",".03125","1."]},{"entry":[{},"6","64.0",".015625","1."]},{"entry":[{},"7","128.0",".0078125","1."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"D. Optimized NNG"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"63pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"T","C","A","G"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":["1",".23",".21",".23",".33"]},{"entry":["2",".215",".285",".285",".215"]},{"entry":["3",".0",".0",".0","1.0"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Amino",{},"Amino",{}]},{"entry":["acid","Abundance","acid","Abundance"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["A","9.40%","C","none"]},{"entry":["D","none","E","9.40%"]},{"entry":["F","none","G","7.10%"]},{"entry":["H","none","I","none"]},{"entry":["K","6.60%","L","9.50% mfaa"]},{"entry":["M","4.90%","N","none"]},{"entry":["P","6.00%","Q","6.00%"]},{"entry":["R","9.50%","S","6.60%"]},{"entry":["T","\u20026.6%","V","7.10%"]},{"entry":["W","4.90% lfaa","Y","none"]},{"entry":["stop","6.60%"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"j","(1\/ratio)","(ratio)","stop-free"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]},{"entry":[{},"1","1.9388",".51579","0.934"]},{"entry":[{},"2","3.7588",".26604","0.8723"]},{"entry":[{},"3","7.2876",".13722","0.8148"]},{"entry":[{},"4","14.1289",".07078","0.7610"]},{"entry":[{},"5","27.3929","3.65 \u00b7 10","0.7108"]},{"entry":[{},"6","53.109","1.88 \u00b7 10","0.6639"]},{"entry":[{},"7","102.96","9.72 \u00b7 10","0.6200"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"E. Unoptimized NNS (NNK gives identical distribution)"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"70pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"T","C","A","G"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":["1",".25",".25",".25",".25"]},{"entry":["2",".25",".25",".25",".25"]},{"entry":["3",".0",".0",".0","0.5"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Amino",{},"Amino",{}]},{"entry":["acid","Abundance","acid","Abundance"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["A","\u20026.25%","C","3.125%"]},{"entry":["D","3.125","E","3.125%"]},{"entry":["F","3.125","G","\u20026.25%"]},{"entry":["H","3.125","I","3.125%"]},{"entry":["K","3.125","L","9.375%"]},{"entry":["M","3.125","N","3.125%"]},{"entry":["P","\u20026.25%","Q","3.125%"]},{"entry":["R","9.375","S","9.375%"]},{"entry":["T","\u20026.25%","V","\u20026.25%"]},{"entry":["W","3.125%","Y","3.125%"]},{"entry":["stop","3.125%"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"j","(1\/ratio)","(ratio)","stop-free"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]},{"entry":[{},"1","3.0",".33333",".96875"]},{"entry":[{},"2","9.0",".11111",".93853"]},{"entry":[{},"3","27.0",".03704",".90915"]},{"entry":[{},"4","81.0",".01234567",".8807"]},{"entry":[{},"5","243.0",".0041152",".8532"]},{"entry":[{},"6","729.0","1.37 \u00b7 10",".82655"]},{"entry":[{},"7","2187.0","4.57 \u00b7 10",".8007"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0948","num":"1253"},"tables":{"@attributes":{"id":"TABLE-US-00037","num":"00037"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 11"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Calculate worst codon."},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"\u2003\u2002Program \u201cFind worst vgCodon within Serr of given"]},{"entry":[{},"\u2003\u2003\u2003\u2003\u2003\u2003\u2003distribution.\u201d"]},{"entry":[{},"\u2003INITIALIZE-MEMORY-OF-ABUNDANCES"]},{"entry":[{},"Comment Serr is % error level."]},{"entry":[{},"\u2003READ Serr"]},{"entry":[{},"Comment T1i,C1i,A1i,G1i, T2i,C2i,A2i,G2i, T3i,G3i"]},{"entry":[{},"Comment are the intended nt-distribution."]},{"entry":[{},"\u2003READ T1i, C1i, A1i, G1i"]},{"entry":[{},"\u2003READ T2i, C2i, A2i, G2i"]},{"entry":[{},"\u2003READ T3i, G3i"]},{"entry":[{},"\u2003Fdwn = 1.\u2212Serr"]},{"entry":[{},"\u2003Fup = 1.+Serr"]},{"entry":[{},"\u2003DO ( t1 = T1i*Fdwn to T1i*Fup in 7 steps)"]},{"entry":[{},"\u2003. DO ( c1 = C1i*Fdwn to C1i*Fup in 7 steps)"]},{"entry":[{},"\u2003. . DO ( a1 = A1i*Fdwn to A1i*Fup in 7 steps)"]},{"entry":[{},"\u2003. . . g1 = 1. \u2212 t1 \u2212 c1 \u2212 a1"]},{"entry":[{},"\u2003. . . IF( (g1\u2212G1i)\/G1i .lt. \u2212Serr)"]},{"entry":[{},"Comment g1 too far below G1i, push it back"]},{"entry":[{},"\u2003. . . . g1 = G1i*Fdwn"]},{"entry":[{},"\u2003. . . . factor = (1.\u2212g1)\/(t1 + c1 + a1)"]},{"entry":[{},"\u2003. . . . t1 = t1*factor"]},{"entry":[{},"\u2003. . . . c1 = c1*factor"]},{"entry":[{},"\u2003. . . . a1 = a1*factor"]},{"entry":[{},"\u2003. . . ..end_IF_block"]},{"entry":[{},"\u2003. . . IF( (g1\u2212G1i)\/G1i .gt. Serr)"]},{"entry":[{},"Comment g1 too far above G1i, push it back"]},{"entry":[{},"\u2003. . . . g1 = G1i*Fup"]},{"entry":[{},"\u2003. . . . factor = (1.\u2212g1)\/(t1 + c1 + a1)"]},{"entry":[{},"\u2003. . . . t1 = t1*factor"]},{"entry":[{},"\u2003. . . . c1 = c1*factor"]},{"entry":[{},"\u2003. . . . a1 = a1*factor"]},{"entry":[{},"\u2003. . . ..end_IF_block"]},{"entry":[{},"\u2003. . . DO ( a2 = A2i*Fdwn to A2i*Fup in 7 steps)"]},{"entry":[{},"\u2003. . . . DO ( c2 = C2i*Fdwn to C2i*Fup in 7 steps)"]},{"entry":[{},"\u2003. . . . . DO (g2=G2i*Fdwn to G2i*Fup in 7 steps)"]},{"entry":[{},"Comment Calc t2 from other concentrations."]},{"entry":[{},"\u2003. . . . . . t2 = 1. \u2212 a2 \u2212 c2 \u2212 g2"]},{"entry":[{},"\u2003. . . . . . IF( (t2\u2212T2i)\/T2i .lt. \u2212Serr)"]},{"entry":[{},"Comment t2 too far below T2i, push it back"]},{"entry":[{},"\u2003. . . . . . . t2 = T2i*Fdwn"]},{"entry":[{},"\u2003. . . . . . . factor = (1.\u2212t2)\/(a2 + c2 + g2)"]},{"entry":[{},"\u2003. . . . . . . a2 = a2*factor"]},{"entry":[{},"\u2003. . . . . . . c2 = c2*factor"]},{"entry":[{},"\u2003. . . . . . . g2 = g2*factor"]},{"entry":[{},"\u2003. . . . . . ..end_IF_block"]},{"entry":[{},"\u2003. . . . . . IF( (t2\u2212T2i)\/T2i .gt. Serr)"]},{"entry":[{},"Comment t2 too far above T2i, push it back"]},{"entry":[{},"\u2003. . . . . . . t2 = T2i*Fup"]},{"entry":[{},"\u2003. . . . . . . factor = (1.\u2212t2)\/(a2 + c2 + g2)"]},{"entry":[{},"\u2003. . . . . . . a2 = a2*factor"]},{"entry":[{},"\u2003. . . . . . . c2 = c2*factor"]},{"entry":[{},"\u2003. . . . . . . g2 = g2*factor"]},{"entry":[{},"\u2003. . . . . . ..end_IF_block"]},{"entry":[{},"\u2003. . . . . . IF(g2.gt. 0.0 .and. t2.gt.0.0)"]},{"entry":[{},"\u2003. . . . . . . t3 = 0.5*(1.\u2212Serr)"]},{"entry":[{},"\u2003. . . . . . . g3 = 1. \u2212 t3"]},{"entry":[{},"\u2003. . . . . . . CALCULATE-ABUNDANCES"]},{"entry":[{},"\u2003. . . . . . . COMPARE-ABUNDANCES-TO-PREVIOUS-ONES"]},{"entry":[{},"\u2003. . . . . . . t3 = 0.5"]},{"entry":[{},"\u2003. . . . . . . g3 = 1. \u2212 t3"]},{"entry":[{},"\u2003. . . . . . . CALCULATE-ABUNDANCES"]},{"entry":[{},"\u2003. . . . . . . COMPARE-ABUNDANCES-TO-PREVIOUS-ONES"]},{"entry":[{},"\u2003. . . . . . . t3 = 0.5*(1.+Serr)"]},{"entry":[{},"\u2003. . . . . . . g3 = 1. \u2212 t3"]},{"entry":[{},"\u2003. . . . . . . CALCULATE-ABUNDANCES"]},{"entry":[{},"\u2003. . . . . . . COMPARE-ABUNDANCES-TO-PREVIOUS-ONES"]},{"entry":[{},"\u2003. . . . . . ..end_IF_block"]},{"entry":[{},"\u2003. . . . . ..end_DO_loop ! g2"]},{"entry":[{},"\u2003. . . . ..end_DO_loop ! c2"]},{"entry":[{},"\u2003. . . ..end_DO_loop ! a2"]},{"entry":[{},"\u2003. . ..end_DO_loop ! a1"]},{"entry":[{},"\u2003. ..end_DO_loop ! c1"]},{"entry":[{},"\u2003..end_DO_loop ! t1"]},{"entry":[{},"\u2003WRITE the WORST distribution and the abundances."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0949","num":"1254"},"tables":{"@attributes":{"id":"TABLE-US-00038","num":"00038"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 12"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Abundances obtained"},{"entry":"using optimum vgCodon assuming"},{"entry":"5% errors"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"70pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Amino",{},"Amino",{}]},{"entry":["acid","Abundance","acid","Abundance"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["A","4.59%","C","2.76%"]},{"entry":["D","5.45%","E","6.02%"]},{"entry":["F","2.49% 1faa","G","6.63%"]},{"entry":["H","3.59%","I","2.71%"]},{"entry":["K","5.73%","L","6.71%"]},{"entry":["M","3.00%","N","5.19%"]},{"entry":["P","3.02%","Q","3.97%"]},{"entry":["R","7.68% mfaa","S","7.01%"]},{"entry":["T","4.37%","V","6.00%"]},{"entry":["W","3.05%","Y","4.77%"]},{"entry":["stop","5.27%"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"ratio = Abun(F) Abun(R) = 0.3248"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"j","(1\/ratio)","(ratio)","stop-free"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]},{"entry":[{},"1","3.079",".3248",".9473"]},{"entry":[{},"2","9.481",".1055",".8973"]},{"entry":[{},"3","29.193",".03425",".8500"]},{"entry":[{},"4","89.888",".01112",".8052"]},{"entry":[{},"5","276.78","3.61 \u00b7 10",".7627"]},{"entry":[{},"6","852.22","1.17 \u00b7 10",".7225"]},{"entry":[{},"7","2624.1","3.81 \u00b7 10",".6844"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0950","num":"1255"},"tables":{"@attributes":{"id":"TABLE-US-00039","num":"00039"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"294pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 13"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"BPTI Homologues"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"294pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["R#\u2003\u20031\u2003\u20032\u2003\u20033\u2003\u20034\u2003\u20035\u2003\u20036\u2003\u20037\u2003\u20038\u2003\u20039 10 11 12 13 14 15 16 17 18 19",{}]},{"entry":"\u22123\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003F\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003Z\u2003\u2003\u2014\u2003\u2003\u2014"},{"entry":"\u22122\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003Q\u2003\u2003T\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003Q\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003H\u2003\u2003G\u2003\u2003Z\u2003\u2003\u2014"},{"entry":"\u22121\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003T\u2003\u2003E\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003P\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003D\u2003\u2003D\u2003\u2003G\u2003\u2003\u2014"},{"entry":"\u20031\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003P\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003L\u2003\u2003A\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003K\u2003\u2003R\u2003\u2003A"},{"entry":"\u20032\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003R\u2003\u2003A\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003R\u2003\u2003P\u2003\u2003A"},{"entry":"\u20033\u2003\u2003D\u2003\u2003D\u2003\u2003D\u2003\u2003D\u2003\u2003D\u2003\u2003D\u2003\u2003D\u2003\u2003D\u2003\u2003D\u2003\u2003D\u2003\u2003D\u2003\u2003K\u2003\u2003K\u2003\u2003D\u2003\u2003R\u2003\u2003T\u2003\u2003D\u2003\u2003S\u2003\u2003K"},{"entry":"\u20034\u2003\u2003F\u2003\u2003F\u2003\u2003F\u2003\u2003L\u2003\u2003F\u2003\u2003F\u2003\u2003F\u2003\u2003F\u2003\u2003F\u2003\u2003F\u2003\u2003F\u2003\u2003L\u2003\u2003Y\u2003\u2003F\u2003\u2003F\u2003\u2003F\u2003\u2003I\u2003\u2003F\u2003\u2003Y"},{"entry":"\u20035"},{"entry":"\u20036\u2003\u2003L\u2003\u2003L\u2003\u2003L\u2003\u2003Q\u2003\u2003L\u2003\u2003L\u2003\u2003L\u2003\u2003L\u2003\u2003L\u2003\u2003L\u2003\u2003L\u2003\u2003I\u2003\u2003K\u2003\u2003E\u2003\u2003E\u2003\u2003N\u2003\u2003R\u2003\u2003N\u2003\u2003K"},{"entry":"\u20037\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003L\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003L\u2003\u2003L\u2003\u2003L\u2003\u2003L\u2003\u2003L\u2003\u2003L\u2003\u2003L\u2003\u2003L"},{"entry":"\u20038\u2003\u2003\u2003\u2003H\u2003\u2003"},{"entry":"\u20039\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003Q\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003R\u2003\u2003L\u2003\u2003A\u2003\u2003A\u2003\u2003P\u2003\u2003P\u2003\u2003A\u2003\u2003V"},{"entry":"10\u2003\u2003Y\u2003\u2003Y\u2003\u2003Y\u2003\u2003A\u2003\u2003Y\u2003\u2003Y\u2003\u2003Y\u2003\u2003Y\u2003\u2003Y\u2003\u2003Y\u2003\u2003Y\u2003\u2003N\u2003\u2003R\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003R"},{"entry":"11\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003R\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003P\u2003\u2003I\u2003\u2003T\u2003\u2003T\u2003\u2003S\u2003\u2003Q\u2003\u2003T\u2003\u2003Y"},{"entry":"12"},{"entry":"13\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003P\u2003\u2003R\u2003\u2003P\u2003\u2003L\u2003\u2003L\u2003\u2003R\u2003\u2003P\u2003\u2003P\u2003\u2003P"},{"entry":"14\u2003T\u2003\u2003A"},{"entry":"15\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003V\u2003\u2003G\u2003\u2003A\u2003\u2003L\u2003\u2003I\u2003\u2003K\u2003\u2003Y\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003R\u2003\u2003K\u2003\u2003K\u2003\u2003K"},{"entry":"16\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003Q\u2003\u2003R\u2003\u2003A\u2003\u2003A\u2003\u2003G\u2003\u2003G\u2003\u2003A\u2003\u2003K"},{"entry":"17\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003A\u2003\u2003A\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003K\u2003\u2003K\u2003\u2003Y\u2003\u2003R\u2003\u2003H\u2003\u2003R\u2003\u2003S\u2003\u2003K"},{"entry":"18\u2003\u2003I\u2003\u2003I\u2003\u2003I\u2003\u2003L\u2003\u2003M\u2003\u2003I\u2003\u2003I\u2003\u2003I\u2003\u2003I\u2003\u2003I\u2003\u2003I\u2003\u2003I\u2003\u2003I\u2003\u2003I\u2003\u2003I\u2003\u2003I\u2003\u2003L\u2003\u2003I\u2003\u2003F"},{"entry":"19\u2003\u2003I\u2003\u2003I\u2003\u2003I\u2003\u2003L\u2003\u2003I\u2003\u2003I\u2003\u2003I\u2003\u2003I\u2003\u2003I\u2003\u2003I\u2003\u2003I\u2003\u2003P\u2003\u2003P\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003P\u2003\u2003R\u2003\u2003P"},{"entry":"20\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003A\u2003\u2003S\u2003\u2003S\u2003\u2003S\u2003\u2003R\u2003\u2003R\u2003\u2003Q\u2003\u2003S"},{"entry":"21\u2003\u2003Y\u2003\u2003Y\u2003\u2003Y\u2003\u2003Y\u2003\u2003Y\u2003\u2003Y\u2003\u2003Y\u2003\u2003Y\u2003\u2003Y\u2003\u2003Y\u2003\u2003Y\u2003\u2003F\u2003\u2003F\u2003\u2003F\u2003\u2003F\u2003\u2003I\u2003\u2003Y\u2003\u2003Y\u2003\u2003F"},{"entry":"22\u2003\u2003F\u2003\u2003F\u2003\u2003F\u2003\u2003F\u2003\u2003F\u2003\u2003F\u2003\u2003F\u2003\u2003F\u2003\u2003F\u2003\u2003F\u2003\u2003F\u2003\u2003Y\u2003\u2003Y\u2003\u2003H\u2003\u2003H\u2003\u2003Y\u2003\u2003F\u2003\u2003Y\u2003\u2003Y"},{"entry":"23"},{"entry":"24"},{"entry":"25\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003S\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003Q\u2003\u2003W\u2003\u2003L\u2003\u2003R\u2003\u2003L\u2003\u2003P\u2003\u2003S\u2003\u2003W"},{"entry":"26\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003T\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003A\u2003\u2003A\u2003\u2003E\u2003\u2003A\u2003\u2003K\u2003\u2003K"},{"entry":"27\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003S\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003K\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003S\u2003\u2003S\u2003\u2003S\u2003\u2003A"},{"entry":"28\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003N\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003K\u2003\u2003K\u2003\u2003Q\u2003\u2003Q\u2003\u2003N\u2003\u2003R\u2003\u2003G\u2003\u2003K"},{"entry":"29\u2003\u2003L\u2003\u2003L\u2003\u2003L\u2003\u2003A\u2003\u2003F\u2003\u2003L\u2003\u2003L\u2003\u2003L\u2003\u2003L\u2003\u2003L\u2003\u2003L\u2003\u2003Q\u2003\u2003Q\u2003\u2003Q\u2003\u2003Q\u2003\u2003K\u2003\u2003M\u2003\u2003G\u2003\u2003Q"},{"entry":"30"},{"entry":"31\u2003\u2003Q\u2003\u2003Q\u2003\u2003Q\u2003\u2003E\u2003\u2003E\u2003\u2003Q\u2003\u2003Q\u2003\u2003Q\u2003\u2003Q\u2003\u2003Q\u2003\u2003Q\u2003\u2003E\u2003\u2003L\u2003\u2003L\u2003\u2003L\u2003\u2003K\u2003\u2003E\u2003\u2003Q\u2003\u2003L"},{"entry":"32\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003P\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003G\u2003\u2003P\u2003\u2003Q\u2003\u2003E\u2003\u2003V\u2003\u2003S\u2003\u2003Q\u2003\u2003P"},{"entry":"33"},{"entry":"34\u2003\u2003V\u2003\u2003V\u2003\u2003V\u2003\u2003T\u2003\u2003V\u2003\u2003V\u2003\u2003V\u2003\u2003V\u2003\u2003V\u2003\u2003V\u2003\u2003V\u2003\u2003T\u2003\u2003D\u2003\u2003I\u2003\u2003I\u2003\u2003F\u2003\u2003I\u2003\u2003I\u2003\u2003N"},{"entry":"35\u2003\u2003W"},{"entry":"36\u2003\u2003\u2003\u2003S\u2003\u2003S\u2003\u2003\u2003\u2003S"},{"entry":"37"},{"entry":"38\u2003\u2003\u2003\u2003T\u2003\u2003A"},{"entry":"39\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003Q\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003K\u2003\u2003R\u2003\u2003G"},{"entry":"40\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003G\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G"},{"entry":"41\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003N\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003N\u2003\u2003N\u2003\u2003N\u2003\u2003N\u2003\u2003N\u2003\u2003N\u2003\u2003N\u2003\u2003N"},{"entry":"42\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003N\u2003\u2003S\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003S\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003K\u2003\u2003Q\u2003\u2003A"},{"entry":"43"},{"entry":"44\u2003\u2003N\u2003\u2003N\u2003\u2003N\u2003\u2003N\u2003\u2003N\u2003\u2003N\u2003\u2003N\u2003\u2003N\u2003\u2003N\u2003\u2003N\u2003\u2003N\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003N\u2003\u2003N\u2003\u2003R\u2003\u2003R"},{"entry":"45"},{"entry":"46\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003E\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003K\u2003\u2003E\u2003\u2003K\u2003\u2003D\u2003\u2003K"},{"entry":"47\u2003\u2003S\u2003\u2003S\u2003\u2003S\u2003\u2003T\u2003\u2003S\u2003\u2003S\u2003\u2003S\u2003\u2003S\u2003\u2003S\u2003\u2003S\u2003\u2003S\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T"},{"entry":"48\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003T\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003I\u2003\u2003I\u2003\u2003I\u2003\u2003I\u2003\u2003R\u2003\u2003K\u2003\u2003T\u2003\u2003I"},{"entry":"49\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003D\u2003\u2003D\u2003\u2003D\u2003\u2003A\u2003\u2003Q\u2003\u2003E"},{"entry":"50\u2003\u2003D\u2003\u2003D\u2003\u2003D\u2003\u2003M\u2003\u2003D\u2003\u2003D\u2003\u2003D\u2003\u2003D\u2003\u2003D\u2003\u2003D\u2003\u2003D\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003E\u2003\u2003Q\u2003\u2003E"},{"entry":"51"},{"entry":"52\u2003\u2003M\u2003\u2003M\u2003\u2003M\u2003\u2003L\u2003\u2003M\u2003\u2003M\u2003\u2003M\u2003\u2003M\u2003\u2003M\u2003\u2003M\u2003\u2003E\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003H\u2003\u2003R\u2003\u2003V\u2003\u2003Q\u2003\u2003R"},{"entry":"53\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003R\u2003\u2003E\u2003\u2003R\u2003\u2003G\u2003\u2003R"},{"entry":"54\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003I\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003T\u2003\u2003A\u2003\u2003V\u2003\u2003T"},{"entry":"55"},{"entry":"56\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003E\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003I\u2003\u2003V\u2003\u2003V\u2003\u2003V\u2003\u2003G\u2003\u2003R\u2003\u2003V\u2003\u2003V"},{"entry":"57\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003P\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003R\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003G\u2003\u2003P\u2003\u2003\u2014\u2003\u2003G"},{"entry":"58\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003P\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003A\u2003\u2003K\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003K\u2003\u2003P\u2003\u2003\u2014\u2003\u2003\u2014"},{"entry":"59\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003Q\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003E\u2003\u2003\u2014\u2003\u2003\u2014"},{"entry":"60\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003Q\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003R\u2003\u2003\u2014\u2003\u2003\u2014"},{"entry":"61\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003T\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003P\u2003\u2003\u2014\u2003\u2003\u2014"},{"entry":"62\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003D\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014"},{"entry":"63\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003K\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014"},{"entry":"64\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003S\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014\u2003\u2003\u2014"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["R #\u2003\u200320\u2003\u200321\u2003\u200322\u2003\u200323\u2003\u200324\u2003\u200325\u2003\u200326\u2003\u200327\u2003\u200328\u2003\u200329\u2003\u200330\u2003\u200331\u2003\u200332\u2003\u200333\u2003\u200334\u2003\u200335",{}]},{"entry":"\u22125\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003D\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014"},{"entry":"\u22124\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003E\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014"},{"entry":"\u22123\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003T\u2003\u2003\u2003P\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014"},{"entry":"\u22122\u2003\u2003\u2003Z\u2003\u2003\u2003\u2014\u2003\u2003\u2003L\u2003\u2003\u2003Z\u2003\u2003\u2003R\u2003\u2003\u2003K\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003\u2014\u2003\u2003\u2003E\u2003\u2003\u2003T\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014"},{"entry":"\u22121\u2003\u2003\u2003P\u2003\u2003\u2003\u2014\u2003\u2003\u2003Q\u2003\u2003\u2003D\u2003\u2003\u2003D\u2003\u2003\u2003N\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003Q\u2003\u2003\u2003K\u2003\u2003\u2003\u2014\u2003\u2003\u2003R\u2003\u2003\u2003T\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014"},{"entry":"\u2003\u20031\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003H\u2003\u2003\u2003H\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003I\u2003\u2003\u2003K\u2003\u2003\u2003T\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003G\u2003\u2003\u2003D\u2003\u2003\u2003K\u2003\u2003\u2003T"},{"entry":"\u2003\u20032\u2003\u2003\u2003R\u2003\u2003\u2003P\u2003\u2003\u2003R\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003N\u2003\u2003\u2003E\u2003\u2003\u2003V\u2003\u2003\u2003H\u2003\u2003\u2003H\u2003\u2003\u2003P\u2003\u2003\u2003F\u2003\u2003\u2003L\u2003\u2003\u2003A\u2003\u2003\u2003V"},{"entry":"\u2003\u20033\u2003\u2003\u2003K\u2003\u2003\u2003Y\u2003\u2003\u2003T\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003T\u2003\u2003\u2003G\u2003\u2003\u2003D\u2003\u2003\u2003A\u2003\u2003\u2003R\u2003\u2003\u2003P\u2003\u2003\u2003D\u2003\u2003\u2003L\u2003\u2003\u2003P\u2003\u2003\u2003D\u2003\u2003\u2003E"},{"entry":"\u2003\u20034\u2003\u2003\u2003L\u2003\u2003\u2003A\u2003\u2003\u2003F\u2003\u2003\u2003F\u2003\u2003\u2003F\u2003\u2003\u2003F\u2003\u2003\u2003D\u2003\u2003\u2003S\u2003\u2003\u2003A\u2003\u2003\u2003D\u2003\u2003\u2003D\u2003\u2003\u2003F\u2003\u2003\u2003D\u2003\u2003\u2003I\u2003\u2003\u2003S\u2003\u2003\u2003A"},{"entry":"\u2003\u20035"},{"entry":"\u2003\u20036\u2003\u2003\u2003I\u2003\u2003\u2003E\u2003\u2003\u2003K\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003N\u2003\u2003\u2003E\u2003\u2003\u2003Q\u2003\u2003\u2003N\u2003\u2003\u2003D\u2003\u2003\u2003D\u2003\u2003\u2003L\u2003\u2003\u2003T\u2003\u2003\u2003E\u2003\u2003\u2003Q\u2003\u2003\u2003N"},{"entry":"\u2003\u20037\u2003\u2003\u2003L\u2003\u2003\u2003L\u2003\u2003\u2003L\u2003\u2003\u2003L\u2003\u2003\u2003L\u2003\u2003\u2003L\u2003\u2003\u2003L\u2003\u2003\u2003L\u2003\u2003\u2003L\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003E\u2003\u2003\u2003S\u2003\u2003\u2003Q\u2003\u2003\u2003L\u2003\u2003\u2003L"},{"entry":"\u2003\u20038\u2003\u2003\u2003H\u2003\u2003\u2003I\u2003\u2003\u2003\u2003\u2003\u2003L\u2003\u2003\u2003\u2003\u2003\u2003G\u2003\u2003\u2003\u2003\u2003\u2003A\u2003\u2003\u2003D\u2003\u2003\u2003"},{"entry":"\u2003\u20039\u2003\u2003\u2003R\u2003\u2003\u2003V\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003P\u2003\u2003\u2003K\u2003\u2003\u2003Y\u2003\u2003\u2003V\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003FG\u2003\u2003\u2003Y\u2003\u2003\u2003I"},{"entry":"\u200310\u2003\u2003\u2003N\u2003\u2003\u2003A\u2003\u2003\u2003E\u2003\u2003\u2003D\u2003\u2003\u2003D\u2003\u2003\u2003E\u2003\u2003\u2003V\u2003\u2003\u2003S\u2003\u2003\u2003I\u2003\u2003\u2003D\u2003\u2003\u2003D\u2003\u2003\u2003Y\u2003\u2003\u2003V\u2003\u2003\u2003D\u2003\u2003\u2003S\u2003\u2003\u2003V"},{"entry":"\u200311\u2003\u2003\u2003P\u2003\u2003\u2003A\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003T\u2003\u2003\u2003V\u2003\u2003\u2003A\u2003\u2003\u2003R\u2003\u2003\u2003K\u2003\u2003\u2003T\u2003\u2003\u2003T\u2003\u2003\u2003T\u2003\u2003\u2003A\u2003\u2003\u2003Q\u2003\u2003\u2003Q"},{"entry":"\u200312\u2003\u2003\u2003K"},{"entry":"\u200313\u2003\u2003\u2003R\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003N\u2003\u2003\u2003I\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003L\u2003\u2003\u2003P\u2003\u2003\u2003P"},{"entry":"\u200314"},{"entry":"\u200315\u2003\u2003\u2003Y\u2003\u2003\u2003M\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003L\u2003\u2003\u2003N\u2003\u2003\u2003R\u2003\u2003\u2003M\u2003\u2003\u2003R\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003K\u2003\u2003\u2003R\u2003\u2003\u2003F\u2003\u2003\u2003L\u2003\u2003\u2003R"},{"entry":"\u200316\u2003\u2003\u2003D\u2003\u2003\u2003F\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003G\u2003\u2003\u2003A\u2003\u2003\u2003G\u2003\u2003\u2003Q\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003A"},{"entry":"\u200317\u2003\u2003\u2003K\u2003\u2003\u2003F\u2003\u2003\u2003S\u2003\u2003\u2003H\u2003\u2003\u2003Y\u2003\u2003\u2003L\u2003\u2003\u2003R\u2003\u2003\u2003M\u2003\u2003\u2003F\u2003\u2003\u2003P\u2003\u2003\u2003T\u2003\u2003\u2003K\u2003\u2003\u2003G\u2003\u2003\u2003Y\u2003\u2003\u2003L\u2003\u2003\u2003F"},{"entry":"\u200318\u2003\u2003\u2003I\u2003\u2003\u2003I\u2003\u2003\u2003I\u2003\u2003\u2003I\u2003\u2003\u2003M\u2003\u2003\u2003I\u2003\u2003\u2003F\u2003\u2003\u2003T\u2003\u2003\u2003I\u2003\u2003\u2003V\u2003\u2003\u2003V\u2003\u2003\u2003M\u2003\u2003\u2003F\u2003\u2003\u2003M\u2003\u2003\u2003F\u2003\u2003\u2003I"},{"entry":"\u200319\u2003\u2003\u2003P\u2003\u2003\u2003S\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003S\u2003\u2003\u2003Q\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003I\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003Q"},{"entry":"\u200320\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003A\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003L\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003L\u2003\u2003\u2003R\u2003\u2003\u2003L"},{"entry":"\u200321\u2003\u2003\u2003F\u2003\u2003\u2003F\u2003\u2003\u2003F\u2003\u2003\u2003F\u2003\u2003\u2003F\u2003\u2003\u2003F\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003W\u2003\u2003\u2003F\u2003\u2003\u2003F\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003W"},{"entry":"\u200322\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003F\u2003\u2003\u2003A\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003F\u2003\u2003\u2003N\u2003\u2003\u2003S\u2003\u2003\u2003F\u2003\u2003\u2003A"},{"entry":"\u200323\u2003\u2003\u2003F\u2003\u2003\u2003F"},{"entry":"\u200324\u2003\u2003\u2003\u2003\u2003\u2003S\u2003\u2003\u2003\u2003\u2003\u2003D\u2003\u2003\u2003\u2003\u2003\u2003D\u2003\u2003\u2003D\u2003\u2003\u2003K\u2003\u2003\u2003\u2003\u2003\u2003D"},{"entry":"\u200325\u2003\u2003\u2003Q\u2003\u2003\u2003K\u2003\u2003\u2003W\u2003\u2003\u2003S\u2003\u2003\u2003P\u2003\u2003\u2003S\u2003\u2003\u2003S\u2003\u2003\u2003G\u2003\u2003\u2003A\u2003\u2003\u2003T\u2003\u2003\u2003P\u2003\u2003\u2003A\u2003\u2003\u2003T\u2003\u2003\u2003Q\u2003\u2003\u2003G\u2003\u2003\u2003A"},{"entry":"\u200326\u2003\u2003\u2003K\u2003\u2003\u2003G\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003H\u2003\u2003\u2003S\u2003\u2003\u2003T\u2003\u2003\u2003V\u2003\u2003\u2003R\u2003\u2003\u2003S\u2003\u2003\u2003K\u2003\u2003\u2003R\u2003\u2003\u2003E\u2003\u2003\u2003T\u2003\u2003\u2003V"},{"entry":"\u200327\u2003\u2003\u2003K\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003S\u2003\u2003\u2003S\u2003\u2003\u2003L\u2003\u2003\u2003S\u2003\u2003\u2003S\u2003\u2003\u2003K\u2003\u2003\u2003L\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003T\u2003\u2003\u2003T\u2003\u2003\u2003S\u2003\u2003\u2003K"},{"entry":"\u200328\u2003\u2003\u2003K\u2003\u2003\u2003N\u2003\u2003\u2003K\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003H\u2003\u2003\u2003K\u2003\u2003\u2003M\u2003\u2003\u2003G\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003G\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003M\u2003\u2003\u2003G"},{"entry":"\u200329\u2003\u2003\u2003Q\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003R\u2003\u2003\u2003A\u2003\u2003\u2003K\u2003\u2003\u2003T\u2003\u2003\u2003R\u2003\u2003\u2003F\u2003\u2003\u2003Q\u2003\u2003\u2003N\u2003\u2003\u2003A\u2003\u2003\u2003K"},{"entry":"\u200330"},{"entry":"\u200331\u2003\u2003\u2003E\u2003\u2003\u2003Y\u2003\u2003\u2003Q\u2003\u2003\u2003N\u2003\u2003\u2003E\u2003\u2003\u2003Q\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003V\u2003\u2003\u2003K\u2003\u2003\u2003V\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003V"},{"entry":"\u200332\u2003\u2003\u2003R\u2003\u2003\u2003P\u2003\u2003\u2003L\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003T\u2003\u2003\u2003L\u2003\u2003\u2003A\u2003\u2003\u2003Q\u2003\u2003\u2003T\u2003\u2003\u2003P\u2003\u2003\u2003E\u2003\u2003\u2003T\u2003\u2003\u2003R"},{"entry":"\u200333"},{"entry":"\u200334\u2003\u2003\u2003D\u2003\u2003\u2003T\u2003\u2003\u2003H\u2003\u2003\u2003I\u2003\u2003\u2003I\u2003\u2003\u2003N\u2003\u2003\u2003I\u2003\u2003\u2003Q\u2003\u2003\u2003P\u2003\u2003\u2003Q\u2003\u2003\u2003R\u2003\u2003\u2003V\u2003\u2003\u2003K\u2003\u2003\u2003I\u2003\u2003\u2003L\u2003\u2003\u2003S"},{"entry":"\u200335\u2003\u2003\u2003W"},{"entry":"\u200336\u2003\u2003\u2003S\u2003\u2003\u2003S\u2003\u2003\u2003\u2003\u2003\u2003R\u2003\u2003\u2003"},{"entry":"\u200337"},{"entry":"\u200338"},{"entry":"\u200339\u2003\u2003\u2003G\u2003\u2003\u2003R\u2003\u2003\u2003K\u2003\u2003\u2003P\u2003\u2003\u2003R\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003M\u2003\u2003\u2003Q\u2003\u2003\u2003D\u2003\u2003\u2003D\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003Q\u2003\u2003\u2003M\u2003\u2003\u2003K"},{"entry":"\u200340\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003C\u2003\u2003\u2003G\u2003\u2003\u2003C\u2003\u2003\u2003G\u2003\u2003\u2003A\u2003\u2003\u2003G\u2003\u2003\u2003C\u2003\u2003\u2003G\u2003\u2003\u2003G"},{"entry":"\u200341\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003D\u2003\u2003\u2003D\u2003\u2003\u2003K\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003N"},{"entry":"\u200342\u2003\u2003\u2003S\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003H\u2003\u2003\u2003H\u2003\u2003\u2003S\u2003\u2003\u2003G\u2003\u2003\u2003D\u2003\u2003\u2003L\u2003\u2003\u2003G"},{"entry":"\u200343\u2003\u2003G\u2003\u2003\u2003G"},{"entry":"\u200344\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003K\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003N\u2003\u2003\u2003K"},{"entry":"\u200345\u2003\u2003Y"},{"entry":"\u200346\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003S\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003H\u2003\u2003\u2003V\u2003\u2003\u2003Y\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003R\u2003\u2003\u2003K\u2003\u2003\u2003S\u2003\u2003\u2003L\u2003\u2003\u2003Y"},{"entry":"\u200347\u2003\u2003\u2003T\u2003\u2003\u2003T\u2003\u2003\u2003T\u2003\u2003\u2003T\u2003\u2003\u2003T\u2003\u2003\u2003T\u2003\u2003\u2003T\u2003\u2003\u2003T\u2003\u2003\u2003S\u2003\u2003\u2003T\u2003\u2003\u2003S\u2003\u2003\u2003S\u2003\u2003\u2003S\u2003\u2003\u2003T\u2003\u2003\u2003S\u2003\u2003\u2003S"},{"entry":"\u200348\u2003\u2003\u2003I\u2003\u2003\u2003I\u2003\u2003\u2003I\u2003\u2003\u2003W\u2003\u2003\u2003W\u2003\u2003\u2003I\u2003\u2003\u2003L\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003D\u2003\u2003\u2003A\u2003\u2003\u2003E\u2003\u2003\u2003L\u2003\u2003\u2003Q\u2003\u2003\u2003Q"},{"entry":"\u200349\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003D\u2003\u2003\u2003D\u2003\u2003\u2003D\u2003\u2003\u2003E\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003T\u2003\u2003\u2003H\u2003\u2003\u2003E\u2003\u2003\u2003Q\u2003\u2003\u2003A\u2003\u2003\u2003K\u2003\u2003\u2003K"},{"entry":"\u200350\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003K\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003L\u2003\u2003\u2003L\u2003\u2003\u2003D\u2003\u2003\u2003D\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003E"},{"entry":"\u200351"},{"entry":"\u200352\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003Q\u2003\u2003\u2003E\u2003\u2003\u2003L\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003M\u2003\u2003\u2003L\u2003\u2003\u2003E\u2003\u2003\u2003L\u2003\u2003\u2003K"},{"entry":"\u200353\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003H\u2003\u2003\u2003Q\u2003\u2003\u2003H\u2003\u2003\u2003R\u2003\u2003\u2003K\u2003\u2003\u2003Q\u2003\u2003\u2003E\u2003\u2003\u2003C\u2003\u2003\u2003C\u2003\u2003\u2003R\u2003\u2003\u2003D\u2003\u2003\u2003Q\u2003\u2003\u2003Q\u2003\u2003\u2003E"},{"entry":"\u200354\u2003\u2003\u2003T\u2003\u2003\u2003T\u2003\u2003\u2003A\u2003\u2003\u2003T\u2003\u2003\u2003T\u2003\u2003\u2003T\u2003\u2003\u2003V\u2003\u2003\u2003T\u2003\u2003\u2003Y\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003T\u2003\u2003\u2003A\u2003\u2003\u2003K\u2003\u2003\u2003T\u2003\u2003\u2003Y"},{"entry":"\u200355"},{"entry":"\u200356\u2003\u2003\u2003I\u2003\u2003\u2003V\u2003\u2003\u2003V\u2003\u2003\u2003G\u2003\u2003\u2003V\u2003\u2003\u2003A\u2003\u2003\u2003C\u2003\u2003\u2003R\u2003\u2003\u2003G\u2003\u2003\u2003L\u2003\u2003\u2003E\u2003\u2003\u2003G\u2003\u2003\u2003S\u2003\u2003\u2003I\u2003\u2003\u2003R\u2003\u2003\u2003G"},{"entry":"\u200357\u2003\u2003\u2003G\u2003\u2003\u2003V\u2003\u2003\u2003G\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003V\u2003\u2003\u2003\u2014\u2003\u2003\u2003V\u2003\u2003\u2003V\u2003\u2003\u2003L\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003N\u2003\u2003\u2003\u2014\u2003\u2003\u2003I"},{"entry":"\u200358\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003S\u2003\u2003\u2003S\u2003\u2003\u2003K\u2003\u2003\u2003R\u2003\u2003\u2003\u2014\u2003\u2003\u2003P\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003A\u2003\u2003\u2003F\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003P"},{"entry":"\u200359\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003A\u2003\u2003\u2003G\u2003\u2003\u2003Y\u2003\u2003\u2003S\u2003\u2003\u2003\u2014\u2003\u2003\u2003G\u2003\u2003\u2003P\u2003\u2003\u2003R\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003C"},{"entry":"\u200360\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003I\u2003\u2003\u2003G\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003D\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003E"},{"entry":"\u200361\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003E\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003A"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"294pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"R #\u2003\u200336\u2003\u200337\u2003\u200338\u2003\u200339\u2003\u200340"},{"entry":"\u2003\u2003\u22125\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014"},{"entry":"\u2003\u2003\u22124\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014"},{"entry":"\u2003\u2003\u22123\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014"},{"entry":"\u2003\u2003\u22122\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014"},{"entry":"\u2003\u2003\u22121\u2003\u2003\u2003\u2014\u2003\u2003\u2003Z\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014"},{"entry":"\u2003\u20031\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R"},{"entry":"\u2003\u20032\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003P"},{"entry":"\u2003\u20033\u2003\u2003\u2003D\u2003\u2003\u2003D\u2003\u2003\u2003D\u2003\u2003\u2003D\u2003\u2003\u2003D"},{"entry":"\u2003\u20034\u2003\u2003\u2003F\u2003\u2003\u2003F\u2003\u2003\u2003F\u2003\u2003\u2003F\u2003\u2003\u2003F"},{"entry":"\u2003\u20035"},{"entry":"\u2003\u20036\u2003\u2003\u2003L\u2003\u2003\u2003L\u2003\u2003\u2003L\u2003\u2003\u2003L\u2003\u2003\u2003L"},{"entry":"\u2003\u20037\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003E"},{"entry":"\u2003\u20038"},{"entry":"\u2003\u20039\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003P"},{"entry":"\u200310\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003Y"},{"entry":"\u200311\u2003\u2003\u2003T\u2003\u2003\u2003T\u2003\u2003\u2003T\u2003\u2003\u2003T\u2003\u2003\u2003T"},{"entry":"\u200312"},{"entry":"\u200313\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003P"},{"entry":"\u200314"},{"entry":"\u200315\u2003\u2003\u2003R\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003K"},{"entry":"\u200316\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A"},{"entry":"\u200317\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003K"},{"entry":"\u200318\u2003\u2003\u2003I\u2003\u2003\u2003M\u2003\u2003\u2003I\u2003\u2003\u2003M\u2003\u2003\u2003M"},{"entry":"\u200319\u2003\u2003\u2003I\u2003\u2003\u2003I\u2003\u2003\u2003I\u2003\u2003\u2003I\u2003\u2003\u2003I"},{"entry":"\u200320\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R"},{"entry":"\u200321\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003Y"},{"entry":"\u200322\u2003\u2003\u2003F\u2003\u2003\u2003F\u2003\u2003\u2003F\u2003\u2003\u2003F\u2003\u2003\u2003F"},{"entry":"\u200323"},{"entry":"\u200324"},{"entry":"\u200325\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A"},{"entry":"\u200326\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003K"},{"entry":"\u200327\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A"},{"entry":"\u200328\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G"},{"entry":"\u200329\u2003\u2003\u2003L\u2003\u2003\u2003L\u2003\u2003\u2003L\u2003\u2003\u2003L\u2003\u2003\u2003F"},{"entry":"\u200330"},{"entry":"\u200331\u2003\u2003\u2003Q\u2003\u2003\u2003Q\u2003\u2003\u2003Q\u2003\u2003\u2003Q\u2003\u2003\u2003E"},{"entry":"\u200332\u2003\u2003\u2003T\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003T"},{"entry":"\u200333"},{"entry":"\u200334\u2003\u2003\u2003V\u2003\u2003\u2003V\u2003\u2003\u2003V\u2003\u2003\u2003V\u2003\u2003\u2003V"},{"entry":"\u200335"},{"entry":"\u200336\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G"},{"entry":"\u200337"},{"entry":"\u200338"},{"entry":"\u200339\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003K"},{"entry":"\u200340\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A"},{"entry":"\u200341\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003K"},{"entry":"\u200342\u2003\u2003\u2003R\u2003\u2003\u2003S\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003S"},{"entry":"\u200343"},{"entry":"\u200344\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003N"},{"entry":"\u200345"},{"entry":"\u200346\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003R"},{"entry":"\u200347\u2003\u2003\u2003S\u2003\u2003\u2003S\u2003\u2003\u2003S\u2003\u2003\u2003S\u2003\u2003\u2003S"},{"entry":"\u200348\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003S\u2003\u2003\u2003A\u2003\u2003\u2003A"},{"entry":"\u200349\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003E"},{"entry":"\u200350\u2003\u2003\u2003D\u2003\u2003\u2003D\u2003\u2003\u2003D\u2003\u2003\u2003D\u2003\u2003\u2003D"},{"entry":"\u200351"},{"entry":"\u200352\u2003\u2003\u2003E\u2003\u2003\u2003M\u2003\u2003\u2003M\u2003\u2003\u2003M\u2003\u2003\u2003M"},{"entry":"\u200353\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R\u2003\u2003\u2003R"},{"entry":"\u200354\u2003\u2003\u2003T\u2003\u2003\u2003T\u2003\u2003\u2003T\u2003\u2003\u2003T\u2003\u2003\u2003T"},{"entry":"\u200355"},{"entry":"\u200356\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G"},{"entry":"\u200357\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G"},{"entry":"\u200358\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003A"},{"entry":"\u200359\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014"},{"entry":"\u200360\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014"},{"entry":"\u200361\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014\u2003\u2003\u2003\u2014"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"1 BPTI (SEQ ID NO:44)"},{"entry":"2 Engineered BPTI From MARK87 (SEQ ID NO:266)"},{"entry":"3 Engineered BPTI From MARK87 (SEQ ID NO:267)"},{"entry":"4 Bovine Colostrum (DUFT85) (SEQ ID NO:268)"},{"entry":"5 Bovine Serum (DUFT85) (SEQ ID NO:269)"},{"entry":"6 Semisynthetic BPTI, TSCH87 (SEQ ID NO:270)"},{"entry":"7 Semisynthetic BPTI, TSCH87 (SEQ ID NO:271)"},{"entry":"8 Semisynthetic BPTI, TSCH87 (SEQ ID NO:272)"},{"entry":"9 Semisynthetic BPTI, TSCH87 (SEQ ID NO:273)"},{"entry":"10 emisynthetic BPTI, TSCH87 (SEQ ID NO:274)"},{"entry":"11 Engineered BPTI, AUER87 (SEQ ID NO:275)"},{"entry":"12 (Black mamba) venom I (DUFT85) (SEQ ID NO:276)"},{"entry":"13 (Black Mamba) venom K (DUFT85) (SEQ ID NO:277)"},{"entry":"14 (Ringhals Cobra)HHV II (DUFT85) (SEQ ID NO:278)"},{"entry":"15 (Cape cobra) NNV II (DUFT85) (SEQ ID NO: 279)"},{"entry":"16 (Russel's viper) RVV II (TAKA74) (SEQ ID NO:280)"},{"entry":"17 Red sea turtle egg white (DUFT85) (SEQ ID NO:281)"},{"entry":"18 Snail mucus () (WAGN78) (SEQ ID NO:282)"},{"entry":"19 (Eastern green mamba) C13 S1 C3 toxin (DUFT85) (SEQ ID NO:283)"},{"entry":"20 (Eastern Green Mamba) C13 S2 C3 toxin (DUFT85) (SEQ ID NO:284)"},{"entry":"21 (Black mamba) B toxin (DUFT85) (SEQ ID NO:285)"},{"entry":"22 (Black Mamba) E toxin (DUFT85) (SEQ ID NO:286)"},{"entry":"23 TI toxin (DUFT85) (SEQ ID NO:287)"},{"entry":"24 CTI toxin (DUFT85) (SEQ ID NO:288)"},{"entry":"25 VIII B toxin(DUFT85) (SEQ ID NO:289)"},{"entry":"26 (sea anemone) 5 II(DUFT85) (SEQ ID NO:290)"},{"entry":"27 HI-14 \u201cinactive\u201d domain (DUFT85) (SEQ ID NO: 291)"},{"entry":"28 HI-B \u201cactive\u201d domain(DUFT85) (SEQ ID NO:292)"},{"entry":"29 beta bungarotoxin B1 (DUFT85) (SEQ ID NO:293)"},{"entry":"30 beta bungarotoxin B2 (DUFT85) (SEQ ID NO:294)"},{"entry":"31 Bovine spleen TI II (FIOR85) (SEQ ID NO:295)"},{"entry":"32 (Horseshoe crab) hemocyte inhibitor (NAKA87) (SEQ ID NO:296)"},{"entry":"33 (silkworm) SCI-III (SASA84) (SEQ ID NO: 297)"},{"entry":"34 (inactive) BI-14 (SEQ ID NO:298)"},{"entry":"35 (active) BI-8 (SEQ ID NO:299)"},{"entry":"36 Engineered BPTI (KR15, ME52): Auerswald '88, Biol Chem Hoppe-Seyler, 369 Supplement, pp27-35. (SEQ ID NO:300)"},{"entry":"37 Isoaprotinin G-1: Siekmann, Wenzel, Schroder, and Tschesche '88, Biol Chem Hoppe-Seyler, 369:157-163. (SEQ ID NO:301)"},{"entry":"38 Isoaprotinin 2: Siekmann, Wenzel, Schroder, and Tschesche '88, Bid Chem Hoppe-Seyler, 369:157-163. (SEQ ID NO:302)"},{"entry":"39 Isoaprotinin G-2: Siekmann, Wenzel, Schroder, and Tschesche '88, Biol Chem Hoppe-Seyler, 369:157-163. (SEQ ID NO:303)"},{"entry":"40 Isoaprotinin 1: Siekmann, Wenzel, Schroder, and Tschesche '88, Biol Chem Hoppe-Seyler, 369:157-163. (SEQ ID NO:304)"},{"entry":"Notes:"},{"entry":"a) both beta bungarotoxins have residue 15 deleted."},{"entry":"b) has an extra residue between CS and C14; we have assigned F and G to residue 9."},{"entry":"c) all natural proteins have C at 5, 14, 30, 38, 50, & 55."},{"entry":"d) all homologues have F33 and G37."},{"entry":"e) extra C's in bungarotoxins form interchain cystine bridges"}]}}]}}},{"@attributes":{"id":"p-0951","num":"1256"},"tables":{"@attributes":{"id":"TABLE-US-00040","num":"00040"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Identification codes for Tables 14 and 15"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"14pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1","BPTI"]},{"entry":["2","synthetic BPTI, Tan & Kaiser, biochem. 16(8)1531-41"]},{"entry":["3","Semisynthetic BPTI, TSCH87"]},{"entry":["4","Semisynthetic BPTI, TSCH87"]},{"entry":["5","Semisynthetic BPTI, TSCH87"]},{"entry":["6","Semisynthetic BPTI, TSCH87"]},{"entry":["7","Semisynthetic BPTI, TSCH87"]},{"entry":["8","Engineered BPTI, AUER87"]},{"entry":["9","BPTI Auerswald &al GB 2 208 511A"]},{"entry":["10","BPTI Auerswald &al GB 2 208 511A"]},{"entry":["11","Engineered BPTI From MARK87"]},{"entry":["12","Engineered BPTI From MARK87"]},{"entry":["13","BPTI(KR15, ME52): Auerswald \u203288, Biol Chem Hoppe-Seyler, 369"]},{"entry":[{},"Suppl, pp27-35."]},{"entry":["14","BPTI CA30\/CA51 Eigenbrot &al, Protein Engineering"]},{"entry":[{},"3(7)591-598 (\u203290)"]},{"entry":["15","Isoaprotinin 2 Siekmann et al \u203288, Biol Chem"]},{"entry":[{},"Hoppe-Seyler, 369: 157-163."]},{"entry":["16","Isoaprotinin G-2: Siekmann et al \u203288, Biol Chem"]},{"entry":[{},"Hoppe-Seyler, 369: 157-163."]},{"entry":["17","BPTI Engineered, Auerswald &al GB 2 208 511A"]},{"entry":["18","BPTI Engineered, Auerswald &al GB 2 208 511A"]},{"entry":["19","BPTI Engineered, Auerswald &al GB 2 208 511A"]},{"entry":["20","Isoaprotinin G-1 Siekmann &al \u203288, Biol Chem"]},{"entry":[{},"Hoppe-Seyler, 369: 157-163."]},{"entry":["21","BPTI Engineered, Auerswald &al GB 2 208 511A"]},{"entry":["22","BPTI Engineered, Auerswald &al GB 2 208 511A"]},{"entry":["23","Bovine Serum (in Dufton \u203285)"]},{"entry":["24","Bovine spleen TI II (FIOR85)"]},{"entry":["25","Snail mucus (Helix pomatia) (WAGN78)"]},{"entry":["26",{"i":"Hemachatus "}]},{"entry":[{},"\u203285)"]},{"entry":["27","Red sea turtle egg white (in Dufton \u203285)"]},{"entry":["28","Bovine Colostrum (in Dufton \u203285)"]},{"entry":["29","Naja nivea (Cape cobra) NNV II (in Dufton \u203285)"]},{"entry":["30","Bungarus fasciatus VIII B toxin (in Dufton \u203285)"]},{"entry":["31","Vipera ammodytes TI toxin (in Dufton \u203285)"]},{"entry":["32","Porcine ITI domain 1, (in CREI87)"]},{"entry":["33","Human Alzheimer's beta APP protease inhibitor, (SHIN90)"]},{"entry":["34","Equine ITI domain 1, in Creighton & Charles"]},{"entry":["35","Bos taurus (inactive) BI-8e (ITI domain 1)"]},{"entry":["36",{"i":"Anemonia sulcata "}]},{"entry":["37",{"i":"Dendroaspis polylepis "}]},{"entry":[{},"Dufton \u203285)"]},{"entry":["38",{"i":"Vipera russelli "}]},{"entry":["39","Tachypleus tridentatus (Horseshoe crab) hemocyte"]},{"entry":[{},"inhibitor (NAKA87)"]},{"entry":["40","LACI 2 (Factor Xa) (WUNT88)"]},{"entry":["41",{"i":"Vipera ammodytes "}]},{"entry":["42",{"i":"Dendroaspis polylepis polylepis "}]},{"entry":[{},"Dufton \u203285)"]},{"entry":["43",{"i":"Homo sapiens "}]},{"entry":["44","Green Mamba toxin K, (in CREI87)"]},{"entry":["45",{"i":"Dendroaspis angusticeps "}]},{"entry":[{},"toxin (in Dufton \u203285)"]},{"entry":["46","LACI 3"]},{"entry":["47","Equine ITI domain 2, (CREI87)"]},{"entry":["48","LACI 1 (VIIa)"]},{"entry":["49",{"i":"Dendroaspis polylepis "}]},{"entry":[{},"Dufton \u203285)"]},{"entry":["50","Porcine ITI domain 2, Creighton and Charles"]},{"entry":["51",{"i":"Homo sapiens "}]},{"entry":["52","Bos taurus (active) BI-8t"]},{"entry":["53","Trypstatin Kito &al (\u203288) J Biol Chem 263(34) 18104-07"]},{"entry":["54",{"i":"Dendroaspis angusticeps "}]},{"entry":[{},"toxin (in Dufton \u203285)"]},{"entry":["55","Green Mamba I venom Creighton & Charles \u203287 CSHSQB"]},{"entry":[{},"52: 511-519."]},{"entry":["56","beta bungarotoxin B2 (in Dufton \u203285)"]},{"entry":["57",{"i":"Dendroaspis polylepis polylepis "}]},{"entry":[{},"Dufton \u203285)"]},{"entry":["58","beta bungarotoxin B1 (in Dufton \u203285)"]},{"entry":["59",{"i":"Bombyx mori "}]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0952","num":"1257"},"tables":{"@attributes":{"id":"TABLE-US-00041","num":"00041"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 14"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Tally of Ionizable groups"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"11"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"10","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"11","colwidth":"21pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Identifier","D","E","K","R","Y","H","NH","CO2","+","ions"]},{"entry":{"@attributes":{"namest":"1","nameend":"11","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"11"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"14pt","align":"char","char":"."}},{"@attributes":{"colname":"6","colwidth":"14pt","align":"char","char":"."}},{"@attributes":{"colname":"7","colwidth":"14pt","align":"char","char":"."}},{"@attributes":{"colname":"8","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"9","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"10","colwidth":"14pt","align":"char","char":"."}},{"@attributes":{"colname":"11","colwidth":"21pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1","2","2","4","6","4","0","1","1","6","16"]},{"entry":["2","2","2","4","6","4","0","1","1","6","16"]},{"entry":["3","2","2","3","6","4","0","1","1","5","15"]},{"entry":["4","2","2","3","6","4","0","1","1","5","15"]},{"entry":["5","2","2","3","6","4","0","1","1","5","15"]},{"entry":["6","2","2","3","6","4","0","1","1","5","15"]},{"entry":["7","2","2","3","6","4","0","1","1","5","15"]},{"entry":["8","2","3","4","6","4","0","1","1","5","17"]},{"entry":["9","2","2","3","5","4","0","1","1","4","14"]},{"entry":["10","2","3","3","6","4","0","1","1","4","16"]},{"entry":["11","2","2","4","6","4","0","1","1","6","16"]},{"entry":["12","2","2","4","6","4","0","1","1","6","16"]},{"entry":["13","2","3","3","7","4","0","1","1","5","17"]},{"entry":["14","2","2","4","6","4","0","1","1","6","16"]},{"entry":["15","2","2","4","6","4","0","1","1","6","16"]},{"entry":["16","2","2","4","6","4","0","1","1","6","16"]},{"entry":["17","2","2","3","5","4","0","1","1","4","14"]},{"entry":["18","2","3","3","5","4","0","1","1","3","15"]},{"entry":["19","2","3","3","5","4","0","1","1","3","15"]},{"entry":["20","2","2","4","5","4","0","1","1","5","15"]},{"entry":["21","2","3","3","4","4","0","1","1","2","14"]},{"entry":["22","2","4","3","4","4","0","1","1","1","15"]},{"entry":["23","2","4","4","4","4","0","1","1","2","16"]},{"entry":["24","2","3","5","4","4","0","1","1","4","16"]},{"entry":["25","1","1","2","4","4","0","1","1","4","10"]},{"entry":["26","2","3","2","5","3","1","1","1","2","14"]},{"entry":["27","2","4","6","8","3","0","1","1","8","22"]},{"entry":["28","2","4","2","3","3","0","1","1","\u22121","13"]},{"entry":["29","1","4","2","7","2","2","1","1","4","16"]},{"entry":["30","1","2","5","3","4","2","1","1","5","13"]},{"entry":["31","4","1","5","3","4","2","1","1","3","15"]},{"entry":["32","1","4","3","2","4","1","1","1","0","12"]},{"entry":["33","2","6","1","5","3","0","1","1","\u22122","16"]},{"entry":["34","2","4","2","2","3","1","1","1","\u22122","12"]},{"entry":["35","2","2","3","2","4","0","1","1","1","11"]},{"entry":["36","1","5","4","5","4","1","1","1","3","17"]},{"entry":["37","0","2","6","3","3","3","1","1","7","13"]},{"entry":["38","2","5","3","7","3","2","1","1","3","19"]},{"entry":["39","3","3","5","5","4","0","1","1","4","18"]},{"entry":["40","3","7","4","3","4","0","1","1","\u22123","19"]},{"entry":["41","3","2","4","6","5","1","1","1","5","17"]},{"entry":["42","1","2","8","5","4","0","1","1","10","18"]},{"entry":["43","1","4","2","2","4","0","1","1","\u22121","11"]},{"entry":["44","1","2","9","4","5","0","1","1","10","18"]},{"entry":["45","0","2","8","4","5","0","1","1","10","16"]},{"entry":["46","1","3","5","5","3","0","1","1","6","16"]},{"entry":["47","3","4","4","3","3","0","1","1","0","16"]},{"entry":["48","3","6","5","4","1","1","1","1","0","20"]},{"entry":["49","0","3","3","5","5","0","1","1","5","13"]},{"entry":["50","2","6","4","2","3","0","1","1","\u22122","16"]},{"entry":["51","2","4","4","3","3","0","1","1","1","15"]},{"entry":["52","1","4","6","2","3","0","1","1","3","15"]},{"entry":["53","2","2","5","1","4","0","1","1","2","12"]},{"entry":["54","2","3","6","8","3","1","1","1","9","21"]},{"entry":["55","1","3","6","7","3","1","1","1","9","19"]},{"entry":["56","6","2","6","7","4","3","1","1","5","23"]},{"entry":["57","0","3","7","7","3","1","1","1","11","19"]},{"entry":["58","6","2","5","7","4","2","1","1","4","22"]},{"entry":["59","4","7","3","1","4","0","1","1","\u22127","17"]},{"entry":{"@attributes":{"namest":"1","nameend":"11","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0953","num":"1258"},"tables":{"@attributes":{"id":"TABLE-US-00042","num":"00042"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 15"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Frequency of Amino Acids at Each Position"},{"entry":"in BPTI and 58 Homologues"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Res.","Different",{},{}]},{"entry":["Id.","AAs","Contents","First"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["\u2002\u22125\u2009\u2009","2","\u221258","D","\u2014"]},{"entry":["\u2002\u22124\u2009\u2009","2","\u221258","E","\u2014"]},{"entry":["\u2002\u22123\u2009\u2009","5","\u221255","P T Z F","\u2014"]},{"entry":["\u2002\u22122\u2009\u2009","10","\u221243","R3 Z3 Q3 T2 E G H K L","\u2014"]},{"entry":["\u2002\u22121\u2009\u2009","11","\u221241","D4 P3 R2 T2 Q2 G K N Z E","\u2014"]},{"entry":["\u20021","13","R35","K6 T4 A3 H2 G2 L M N P I D \u2014","R"]},{"entry":["\u20022","10","P35","R6 A4 V4 H3 E3 N F I L","P"]},{"entry":["\u20023","11","D32","K8 S4 A3 T3 R2 E2 P2 G L Y","D"]},{"entry":["\u20024","9","F34","A6 D4 L4 S4 Y3 I2 W V","F"]},{"entry":["\u20025","1","C59",{},"C"]},{"entry":["\u20026","13","L25","N7 E6 K4 Q4 I3 D2 S2 Y2 R F T A","L"]},{"entry":["\u20027","7","L28","E25 K2 F Q S T","E"]},{"entry":["\u20028","10","P46","H3 D2 G2 E I K L A Q","P"]},{"entry":["\u20029","12","P30","A9 I4 V4 R3 Y3 L F Q H E K","P"]},{"entry":["\u20039a","2","\u221258","G","\u2014"]},{"entry":["10","9","Y24","E8 D8 V6 R3 S3 A3 N3 I","Y"]},{"entry":["11","11","T31","Q8 P7 R3 A3 Y2 K S D V I","T"]},{"entry":["12","2","G58","K","G"]},{"entry":["13","5","P45","R7 L4 I2 N","P"]},{"entry":["14","3","C57","A T","C"]},{"entry":["15","12","K22","R12 L7 V6 Y3 M2 \u22122 N I A F G","K"]},{"entry":["16","7","A41","G9 F2 D2 K2 Q2 R","A"]},{"entry":["17","14","R19","L8 K7 F5 M4 Y4 H2 A2 S2 G2 I N T P","R"]},{"entry":["18","8","I41","M7 F4 L2 V2 E T A","I"]},{"entry":["19","10","I24","P12 R8 K5 S4 Q2 L N E T","I"]},{"entry":["20","5","R39","A8 L6 S5 Q","R"]},{"entry":["21","5","Y35","F17 W5 I L","Y"]},{"entry":["22","6","F32","Y18 A5 H2 S N","F"]},{"entry":["23","2","Y52","F7","Y"]},{"entry":["24","4","N47","D8 K3 S","N"]},{"entry":["25","13","A29","S6 Q4 G4 W4 P3 T2 L2 R N K V I","A"]},{"entry":["26","11","K31","A9 T5 S3 V3 R2 E2 G H F Q","K"]},{"entry":["27","8","A32","S11 K5 T4 Q3 L2 I E","A"]},{"entry":["28","7","G32","K13 N5 M4 Q2 R2 H","G"]},{"entry":["29","10","L22","K13 Q11 A5 F2 R2 N G M T","L"]},{"entry":["30","2","C58","A","C"]},{"entry":["31","10","Q25","E17 L5 V5 K2 N A R I Y","Q"]},{"entry":["32","11","T25","P11 K4 Q4 L4 R3 E3 G2 S A V","t"]},{"entry":["33","1","F59",{},"F"]},{"entry":["34","13","V24","I10 T5 N3 Q3 D3 K3 F2 H2 R S P L","V"]},{"entry":["35","2","Y56","W3","Y"]},{"entry":["36","3","G50","S8 R","G"]},{"entry":["37","1","G59",{},"G"]},{"entry":["38","3","C57","A T","C"]},{"entry":["39","9","R25","G13 K6 Q4 E3 M3 L2 D2 P","R"]},{"entry":["40","2","G35","A24","A"]},{"entry":["41","3","N33","K24 D2","K"]},{"entry":["42","12","R22","A12 G8 S6 Q2 H2 N2 M D E K L","R"]},{"entry":["43","2","N57","G2","N"]},{"entry":["44","3","N40","R14 K5","N"]},{"entry":["45","2","F58","Y","F"]},{"entry":["46","11","K39","Y5 E4 S2 V2 D2 R H T A L","K"]},{"entry":["47","2","S36","T23","S"]},{"entry":["48","11","A23","I11 E6 Q6 L4 K2 T2 W2 S D R","A"]},{"entry":["49","8","E37","K8 D6 Q3 A2 P H T","E"]},{"entry":["50","7","E27","D25 K2 L2 M Q Y","D"]},{"entry":["51","2","C58","A","C"]},{"entry":["52","9","M17","R15 E8 L7 K6 Q2 T2 H V","M"]},{"entry":["53","11","R37","E6 Q5 K2 C2 H2 A N G D W"]},{"entry":["54","8","T41","Y5 A4 V3 I2 E2 M K"]},{"entry":["55","1","C59",{},"C"]},{"entry":["56","10","G33","V9 R5 I4 E3 L A S T K","G"]},{"entry":["57","12","G34","V6 \u22125 A3 R2 I2 P2 D K S L NG","G"]},{"entry":["58","10","A25","\u221215 P7 K3 S2 Y2 G2 F D RA","A"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0954","num":"1259"},"tables":{"@attributes":{"id":"TABLE-US-00043","num":"00043"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 16"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Exposure in BPTI"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Coordinates taken from"},{"entry":"Brookhaven Protein Data Bank entry 6PTI."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"175pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["HEADER","PROTEINASE INHIBITOR (TRYPSIN) 13-MAY-87 6PTI"]},{"entry":["COMPND","BOVINE PANCREATIC TRYPSIN INHIBITOR"]},{"entry":[{},"(SEQ ID NO: 44)"]},{"entry":["COMPND","2(\/BPTI$, CRYSTAL FORM\/III$)"]},{"entry":["AUTHOR","A. WLODAWER"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"196pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Solvent radius = 1.40"]},{"entry":[{},"Atomic radii given in Table 7"]},{"entry":[{},"Areas in \u212b."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Not",{},"Not",{}]},{"entry":[{},"Total","Covered",{},"covered"]},{"entry":["Residue","area","by M\/C","fraction","at all","fraction"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"6","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["ARG 1","342.45","205.09","0.5989","152.49","0.4453"]},{"entry":["PRO 2","239.12","92.65","0.3875","47.56","0.1989"]},{"entry":["ASP 3","272.39","158.77","0.5829","143.23","0.5258"]},{"entry":["PHE 4","311.33","137.82","0.4427","43.21","0.1388"]},{"entry":["CYS 5","241.06","48.36","0.2006","0.23","0.0010"]},{"entry":["LEU 6","280.98","151.45","0.5390","115.87","0.4124"]},{"entry":["GLU 7","291.39","128.91","0.4424","90.39","0.3102"]},{"entry":["PRO 9","236.12","128.71","0.5451","99.98","0.4234"]},{"entry":["PRO 9","236.09","109.82","0.4652","45.80","0.1940"]},{"entry":["TYR 10","330.97","153.63","0.4642","79.49","0.2402"]},{"entry":["THR 11","249.20","80.10","0.3214","64.99","0.2608"]},{"entry":["GLY 12","184.21","56.75","0.3081","23.05","0.1252"]},{"entry":["PRO 13","240.07","130.25","0.5426","75.27","0.3136"]},{"entry":["CYS 14","237.10","75.55","0.3186","53.52","0.2257"]},{"entry":["LYS 15","310.77","200.25","0.6444","192.00","0.6178"]},{"entry":["ALA 16","209.41","66.63","0.3182","45.59","0.2177"]},{"entry":["ARG 17","351.09","243.67","0.6940","201.48","0.5739"]},{"entry":["ILE 18","277.10","100.51","0.3627","58.95","0.2127"]},{"entry":["ILE 19","278.03","146.06","0.5254","96.05","0.3455"]},{"entry":["ARG 20","339.11","144.65","0.4266","43.81","0.1292"]},{"entry":["TYR 21","333.60","102.24","0.3065","69.67","0.2089"]},{"entry":["PHE 22","306.08","70.64","0.2308","23.01","0.0752"]},{"entry":["TYR 23","338.66","77.05","0.2275","17.34","0.0512"]},{"entry":["ASN 24","264.88","99.03","0.3739","38.69","0.1461"]},{"entry":["ALA 25","211.15","85.13","0.4032","48.20","0.2283"]},{"entry":["LYS 26","313.29","216.14","0.6899","202.84","0.6474"]},{"entry":["ALA 27","210.66","96.05","0.4560","54.78","0.2601"]},{"entry":["GLY 28","186.83","71.52","0.3828","32.09","0.1718"]},{"entry":["LEU 29","280.70","132.42","0.4718","93.61","0.3335"]},{"entry":["CYS 30","238.15","57.27","0.2405","19.33","0.0812"]},{"entry":["GLN 31","301.15","141.80","0.4709","82.64","0.2744"]},{"entry":["THR 32","251.26","138.17","0.5499","76.47","0.3043"]},{"entry":["PHE 33","304.27","59.79","0.1965","18.91","0.0622"]},{"entry":["VAL 34","251.56","109.78","0.4364","42.36","0.1684"]},{"entry":["TYR 35","332.64","80.52","0.2421","15.05","0.0452"]},{"entry":["GLY 36","187.06","11.90","0.0636","1.97","0.0105"]},{"entry":["GLY 37","185.28","84.26","0.4548","39.17","0.2114"]},{"entry":["CYS 38","234.56","73.64","0.3139","26.40","0.1125"]},{"entry":["ARG 39","417.13","304.62","0.7303","250.73","0.6011"]},{"entry":["ALA 40","209.53","94.01","0.4487","52.95","0.2527"]},{"entry":["LYS 41","314.60","166.23","0.5284","108.77","0.3457"]},{"entry":["ARG 42","349.06","232.83","0.6670","179.59","0.5145"]},{"entry":["ASN 43","266.47","38.53","0.1446","5.32","0.0200"]},{"entry":["ASN 44","269.65","91.08","0.3378","23.39","0.0867"]},{"entry":["PHE 45","313.22","69.73","0.2226","14.79","0.0472"]},{"entry":["LYS 46","309.83","217.18","0.7010","155.73","0.5026"]},{"entry":["SER 47","224.78","69.11","0.3075","24.80","0.1103"]},{"entry":["ALA 48","211.01","82.06","0.3889","31.07","0.1473"]},{"entry":["GLU 49","286.62","161.00","0.5617","100.01","0.3489"]},{"entry":["ASP 50","299.53","156.42","0.5222","95.96","0.3204"]},{"entry":["CYS 51","238.68","24.51","0.1027","0.00","0.0000"]},{"entry":["MET 52","293.05","89.48","0.3054","66.70","0.2276"]},{"entry":["ARG 53","356.20","224.61","0.6306","189.75","0.5327"]},{"entry":["THR 54","251.53","116.43","0.4629","51.64","0.2053"]},{"entry":["CYS 55","240.40","69.95","0.2910","0.00","0.0000"]},{"entry":["GLY 56","184.66","60.79","0.3292","32.78","0.1775"]},{"entry":["GLY 57","106.58","49.71","0.4664","38.28","0.3592"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"189pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["ALA 58","no position given in Protein Data Bank"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"(SEQ ID NO: 144)"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"175pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["\u201cTotal area\u201d","is the area measured by a rolling sphere of radius 1.4 \u212b,"]},{"entry":[{},"where only the atoms within the residue are considered."]},{"entry":[{},"This takes account of conformation."]},{"entry":["\u201cNot covered","is the area measured by a rolling sphere by M\/C\u201d of radius"]},{"entry":[{},"1.4 \u212b where all main-chain atoms are considered, fraction"]},{"entry":[{},"is the exposed area divided by the total area. Surface"]},{"entry":[{},"buried by main-chain atoms is more definitely covered"]},{"entry":[{},"than is surface covered by side group atoms."]},{"entry":["\u201cNot covered","is the area measured by a rolling sphere at all\u201d of radius"]},{"entry":[{},"1.4 \u212b where all atoms of the"]},{"entry":[{},"protein are considered."]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0955","num":"1260"},"tables":{"@attributes":{"id":"TABLE-US-00044","num":"00044"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 17"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Plasmids used in Detailed Example I"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"161pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Phage","Contents"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"LG1","M13mp18 with Ava II\/Aat II\/Acc I\/Rsr II\/Sau I"]},{"entry":[{},{},"adaptor"]},{"entry":[{},"pLG2","LG1 with ampand ColE1 of pBR322 cloned into"]},{"entry":[{},{},"Aat II\/Acc I sites"]},{"entry":[{},"pLG3","pLG2 with Acc I site removed"]},{"entry":[{},"pLG4","pLG3 with first part of osp-pbd gene cloned"]},{"entry":[{},{},"into Rsr II\/Sau I sites, Avr II\/Asu II sites"]},{"entry":[{},{},"created"]},{"entry":[{},"pLG5","pLG4 with second part of osp-pbd gene cloned"]},{"entry":[{},{},"into Avr II\/Asu II sites, BssH I site created"]},{"entry":[{},"pLG6","pLG5 with third part of osp-pbd gene cloned"]},{"entry":[{},{},"into Asu II\/BssH I sites, Bbe I site created"]},{"entry":[{},"pLG7","pLG6 with last part of osp-pbd gene cloned"]},{"entry":[{},{},"into Bbe I\/Asu II sites"]},{"entry":[{},"pLG8","pLG7 with disabled osp-pbd gene, same length"]},{"entry":[{},{},"DNA."]},{"entry":[{},"pLG9","pLG7 mutated to display BPTI(V15)"]},{"entry":[{},"pLG10","pLG8 + tetgene \u2212 ampgene"]},{"entry":[{},"pLG11","pLG9 + tetgene \u2212 ampgene"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0956","num":"1261"},"tables":{"@attributes":{"id":"TABLE-US-00045","num":"00045"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 18"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Enzyme sites eliminated when"},{"entry":"M13mp18 is cut by AvaII"},{"entry":"and Bsu36I"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"AhaII","NarI","GdiII","PvuI"]},{"entry":[{},"FspI","BglI","HgiEII","Bsu36I"]},{"entry":[{},"EcoRI","SacI","KpnI","XmaI"]},{"entry":[{},"SmaI","BamHI","XbaI","SalI"]},{"entry":[{},"HindIII","AccI","PstI","SphI"]},{"entry":[{},"HindII"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0957","num":"1262"},"tables":{"@attributes":{"id":"TABLE-US-00046","num":"00046"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 19"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Enzymes not cutting"},{"entry":"M13mp18"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"AatII","AflI","ApaI","AvrII"]},{"entry":[{},"BbvII","BclI","BspMI","BssHI"]},{"entry":[{},"BstBI","BstEII","BstXI","EagI"]},{"entry":[{},"Eco57I","EcoNI","EcoO109I","EcoRV"]},{"entry":[{},"EspI","HpaI","MluI","NcoI"]},{"entry":[{},"NheI","NotI","NruI","NsiI"]},{"entry":[{},"PflMI","PmaCI","PpaI","PpuMI"]},{"entry":[{},"RsrI","SacI","ScaI","SfiI"]},{"entry":[{},"SpeI","StuI","StyI","Tth111I"]},{"entry":[{},"XcaI","XhoI"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0958","num":"1263"},"tables":{"@attributes":{"id":"TABLE-US-00047","num":"00047"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 20"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Enzymes cutting"},{"entry":"AmpR gene and ori"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"AatII","BbvII","Eco57I","PpaI"]},{"entry":[{},"ScaI","Tth111I","AhaII","GdiII"]},{"entry":[{},"PvuI","FspI","BglI","HgiEII"]},{"entry":[{},"HindII","PstI","XbaI","AflIII"]},{"entry":[{},"NdeI"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0959","num":"1264"},"tables":{"@attributes":{"id":"TABLE-US-00048","num":"00048"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 21"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Enzymes tested on Ambig DNA"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Enzyme","Recognition","Symm","cuts","Supply"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["%AccI","GTMKAC","P","2 &","4 <B,M,I,N,P,T",{}]},{"entry":{}},{"entry":["AflII","CTTAAG","P","1 &","5 <N"]},{"entry":{}},{"entry":["APaI","GGGCCC","P","5 &","1 <M,I,N,P,T"]},{"entry":{}},{"entry":["AsuII","TTCGAA","P","2 &","4 <P,N(BstBI)"]},{"entry":{}},{"entry":["AvaIII","ATGCAT","P","5 &","1 <T; NsiI:M,N,P,T;"]},{"entry":[{},{},{},{},"EcoT22I:T"]},{"entry":{}},{"entry":["AvrII","CCTAGG","P","1 &","5 <N"]},{"entry":{}},{"entry":["BamHI","GGATCC","P","1 &","5 <S,B,M,I,N,P,T"]},{"entry":{}},{"entry":["BclI","TGATCA","P","1 &","5 <S,B,M,I,N,T"]},{"entry":{}},{"entry":["BsPMII","TCCGGA","P","1 &","5 <N"]},{"entry":{}},{"entry":["BssHII","GCGCGC","P","1 &","5 <N,T"]},{"entry":{}},{"entry":["+BstEII","GGTNACC","P","1 &","6 <S,B,M,N,T"]},{"entry":{}},{"entry":["%BstXI","CCANNNNN","P","8 &","4 <N,P,T"]},{"entry":{}},{"entry":["+DraII","RGGNCCY","P","2 &","5 <M,T; EcoO109I:N"]},{"entry":{}},{"entry":["+EcoNI","CCTNNNNN","P","5 &","6 <N(soon)"]},{"entry":{}},{"entry":["EcoRI","GAATTC","P","1 &","5 <S,B,M,I,N,P,T"]},{"entry":{}},{"entry":["EcoRV","GATATC","P","3 &","3 <S,B,M,I,N,P,T"]},{"entry":{}},{"entry":["+EsPI","GCTNAGC","P","2 &","5 <T"]},{"entry":{}},{"entry":["HindIII","AAGCTT","P","1 &","5 <S,B,M,I,N,P,T"]},{"entry":{}},{"entry":["HPaI","GTTAAC","P","3 &","3 <S,B,M,I,N,P,T"]},{"entry":{}},{"entry":["KPnI","GGTACC","P","5 &","1 <S,B,M,I,N,P,T"]},{"entry":[{},{},{},{},"AsP718:M"]},{"entry":{}},{"entry":["MluI","ACGCGT","P","1 &","5 <M,N,P,T"]},{"entry":{}},{"entry":["NarI","GGCGCC","P","2 &","4 <B,N,T"]},{"entry":{}},{"entry":["NcoI","CCATGG","P","1 &","5 <B,M,N,P,T"]},{"entry":{}},{"entry":["NheI","GCTAGC","P","1 &","S <M,N,P,T"]},{"entry":{}},{"entry":["NotI","GCGGCCGC","P","2 &","6 <M,N,P,T"]},{"entry":{}},{"entry":["NruI","TCGCGA","P","3 &","3 <B,M,N,T"]},{"entry":{}},{"entry":["+PflMI","CCANNNNN","P","7 &","4 <N"]},{"entry":{}},{"entry":["PmaCI","CACGTG","P","3 &","3 <none"]},{"entry":{}},{"entry":["+PPuMI","RGGWCCY","P","2 &","S <N"]},{"entry":{}},{"entry":["+RsrII","CGGWCCG","P","2 &","5 <N,T"]},{"entry":{}},{"entry":["SacI","GAGCTC","P","5 &","1 <B(SstI),M,I,N,P,T"]},{"entry":{}},{"entry":["SalI","GTCGAC","P","1 &","5 <B,M,I,N,P,T"]},{"entry":{}},{"entry":["+SauI","CCTNAGG","P","2 &","5 <M; CvnI:B; MstIT:"]},{"entry":[{},{},{},{},"T; Bsu36I:N; AocI:T"]},{"entry":{}},{"entry":["+SfiI","GGCCNNNN","P","8 &","5 <N,P,T (SEQ ID"]},{"entry":[{},"NGGCC",{},{},"NO: 151)"]},{"entry":{}},{"entry":["SmaI","CCCGGG","P","3 &","3 <B,M,I,N,P,T"]},{"entry":{}},{"entry":["SPeI","ACTAGT","P","1 &","5 <M,N,T"]},{"entry":{}},{"entry":["SphI","GCATGC","P","5 &","1 <B,M,I,N,P,T"]},{"entry":{}},{"entry":["StuI","AGGCCT","P","3 &","3 <M,N,I(AatI),P,T"]},{"entry":{}},{"entry":["%StyI","CCWWGG","P","1 &","5 <N,P,T"]},{"entry":{}},{"entry":["XcaI","GTATAC","P","3 &","3 N(soon)"]},{"entry":{}},{"entry":["XhoI","CTCGAG","P","1 &","5 <B,M,I,P,T; CcrI:"]},{"entry":[{},{},{},{},"T; PaeR7I:N"]},{"entry":{}},{"entry":["XmaI","CCCGGG","P","1 &","5 <I,N,P,T"]},{"entry":{}},{"entry":["XmaIII","CGGCCG","P","1 &","Eco52I:T"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}},{"entry":"Nrestrct = 43"}]}}]}}},{"@attributes":{"id":"p-0960","num":"1265"},"tables":{"@attributes":{"id":"TABLE-US-00049","num":"00049"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"266pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 22"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"ipbd gene (SEQ ID NO:152)"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"266pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["pbd mod10 29III88",{}]},{"entry":"\u2003\u2003\u2003\u2003\u2003lacUV5 RsrII\/AvrII\/gene\/TrpA attenuator\/MstII; !"},{"entry":{}},{"entry":"5\u2032-\u2003\u2003\u2003\u2003 TaT\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003!\u2003RsrII site"},{"entry":{}},{"entry":"CCAGGC  CTTTATGCTTCCGGCTCG  GTG ! lacUV5"},{"entry":{}},{"entry":"TGG \u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003! lacO operator"},{"entry":{}},{"entry":"CCT  CtcaCT\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003! Shine-Dalgarno seq."},{"entry":{}},{"entry":"atg aag aaa tct ctg gtt ctt aag gct agc ! 10, M13 leader"},{"entry":{}},{"entry":"gtt gct gtc gcg acc ctg gta ccg atg ctg ! 20"},{"entry":{}},{"entry":"tct ttt gct cgt ccg gat ttc tgt ctc gag ! 30"},{"entry":{}},{"entry":"ccg cca tat act ggg ccc tgc aaa gcg cgc ! 40"},{"entry":{}},{"entry":"atc atc cgt tat ttc tac aac gct aaa gca ! 50"},{"entry":{}},{"entry":"ggc ctg tgc cag acc ttt gta tac ggt ggt ! 60"},{"entry":{}},{"entry":"tgc cgt gct aag cgt aac aac ttt aaa tcg ! 70"},{"entry":{}},{"entry":"gcc gaa gat tgc atg cgt acc tgc ggt ggc ! 80"},{"entry":{}},{"entry":"gcc gct gaa ggt gat gat ccg gcc aaa gcg ! 90"},{"entry":{}},{"entry":"gcc ttt aac tct ctg caa gct tct gct acc ! 100"},{"entry":{}},{"entry":"gaa tat atc ggt tac gcg tgg gcc atg gtg ! 110"},{"entry":{}},{"entry":"gtg gtt atc gtt ggt gct acc atc ggt atc ! 120"},{"entry":{}},{"entry":"aaa ctg ttt aag aaa ttt act tcg aaa gcg ! 130"},{"entry":{}},{"entry":"tct taa tag tga  !\u2003\u2003\u2003BstEII"},{"entry":{}},{"entry":"agtcta agcccgc ctaatga gcgggct tttttttt ! terminator"},{"entry":{}},{"entry":"CCTgAGG\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003-3\u2032\u2003\u2003! MstII"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0961","num":"1266"},"tables":{"@attributes":{"id":"TABLE-US-00050","num":"00050"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"273pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 23 "},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"ipbd DNA sequence (SEQ ID NO:152)"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"273pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"DNA Sequence file = UV5_M13PTIM13.DNA;17"},{"entry":"DNA Sequence title = pbd mod10 29III88 : lac-UV5"},{"entry":"RsrII\/AvrII\/gene\/TrpA attenuator\/MstII; !"},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00039","num":"00039"},"img":{"@attributes":{"id":"EMI-C00039","he":"14.99mm","wi":"87.46mm","file":"US07413537-20080819-C00039.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00040","num":"00040"},"img":{"@attributes":{"id":"EMI-C00040","he":"17.44mm","wi":"87.46mm","file":"US07413537-20080819-C00040.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Total = 539 bases"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0962","num":"1267"},"tables":{"@attributes":{"id":"TABLE-US-00051","num":"00051"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 24"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Summary of Restriction Cuts"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"189pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Enz = % Acc I has 1 observed sites: 259"]},{"entry":[{},"Enz = Acc III has 1 observed sites: 162"]},{"entry":[{},"Enz = Acy I has 1 observed sites: 328"]},{"entry":[{},"Enz = Afl II has 1 observed sites: 109"]},{"entry":[{},"Enz = % Afl III has 1 observed sites: 404"]},{"entry":[{},"Enz = Aha III has 1 observed sites: 292"]},{"entry":[{},"Enz = Apa I has 1 observed sites: 193"]},{"entry":[{},"Enz = Asp718 has 1 observed sites: 138"]},{"entry":[{},"Enz = Asu II has 1 observed sites: 471"]},{"entry":[{},"Enz = % Ava I has 1 observed sites: 175"]},{"entry":[{},"Enz = Avr II has 1 observed sites: 76"]},{"entry":[{},"Enz = % Ban I has 3 observed sites: 138 328 540"]},{"entry":[{},"Enz = Bbe I has 1 observed sites: 328"]},{"entry":[{},"Enz = +Bgl I has 1 observed sites: 352"]},{"entry":[{},"Enz = +Bin I has 1 observed sites: 346"]},{"entry":[{},"Enz = % BspM I has 1 observed sites: 319"]},{"entry":[{},"Enz = BssH II has 1 observed sites: 205"]},{"entry":[{},"Enz = +BstE II has 1 observed sites: 493"]},{"entry":[{},"Enz = % BstX I has 1 observed sites: 413"]},{"entry":[{},"Enz = Cfr I has 2 observed sites: 299 350"]},{"entry":[{},"Enz = +Dra II has 1 observed sites: 193"]},{"entry":[{},"Enz = +Esp I has 1 observed sites: 277"]},{"entry":[{},"Enz = % Fok I has 1 observed sites: 213"]},{"entry":[{},"Enz = Gdi II has 2 observed sites: 299 350"]},{"entry":[{},"Enz = Hae I has 1 observed sites: 240"]},{"entry":[{},"Enz = Hae II has 1 observed sites: 328"]},{"entry":[{},"Enz = +Hga I has 1 observed sites: 478"]},{"entry":[{},"Enz = % HgiC I has 3 observed sites: 138 328 540"]},{"entry":[{},"Enz = % HgiJ II has 1 observed sites: 193"]},{"entry":[{},"Enz = Hind III has 1 observed sites: 377"]},{"entry":[{},"Enz = +Hph I has 1 observed sites: 340"]},{"entry":[{},"Enz = Kpn I has 1 observed sites: 138"]},{"entry":[{},"Enz = +Mbo II has 2 observed sites: 93 304"]},{"entry":[{},"Enz = Mlu I has 1 observed sites: 404"]},{"entry":[{},"Enz = Nar I has 1 observed sites: 328"]},{"entry":[{},"Enz = Nco I has 1 observed sites: 413"]},{"entry":[{},"Enz = Nhe I has 1 observed sites: 115"]},{"entry":[{},"Enz = Nru I has 1 observed sites: 128"]},{"entry":[{},"Enz = Nsp(7524) has 1 observed sites: 311"]},{"entry":[{},"Enz = NspB II has 1 observed sites: 332"]},{"entry":[{},"Enz = +PflM I has 1 observed sites: 184"]},{"entry":[{},"Enz = +Pss I has 1 observed sites: 193"]},{"entry":[{},"Enz = +Rsr II has 1 observed sites:"]},{"entry":[{},"Enz = +Sau I has 1 observed sites: 535"]},{"entry":[{},"Enz = % SfaN I has 2 observed sites: 144 209"]},{"entry":[{},"Enz = +Sfi I has 1 observed sites: 351"]},{"entry":[{},"Enz = Sph I has 1 observed sites: 311"]},{"entry":[{},"Enz = Stu I has1 observed sites: 240"]},{"entry":[{},"Enz = % Sty I has 2 observed sites: 76 413"]},{"entry":[{},"Enz = Xca I has 1 observed sites: 259"]},{"entry":[{},"Enz = Xho I has 1 observed sites: 175"]},{"entry":[{},"Enz = Xma III has 1 observed sites: 299"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Enzymes that do not cut"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Aat II","AlwN I","ApaL I","Ase I","Ava III"]},{"entry":[{},"Bal I","BamH I","Bbv I","Bbv II","Bcl I"]},{"entry":[{},"Bgl II","Bsm I","BspH I","Cla I","Dra III"]},{"entry":[{},"Eco47 III","EcoN I","EcoR I","EcoR V","HgiA I"]},{"entry":[{},"Hinc II","Hpa I","Mst I","Nae I","Nde I"]},{"entry":[{},"Not I","Ple I","PmaC I","PpuM I","Pst I"]},{"entry":[{},"Pvu I","Pvu II","Sac I","Sac II","Sal I"]},{"entry":[{},"Sca I","Sma I","SnaB I","Spe I","Ssp I"]},{"entry":[{},"Taq II","Tth111 I","Tth111 II","Xho II","Xma I"]},{"entry":[{},"Xmn I"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"5","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0963","num":"1268"},"tables":{"@attributes":{"id":"TABLE-US-00052","num":"00052"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 25"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Annotated Sequence of ipbd gene (SEQ ID NO:152)"},{"entry":"Protein Seqence SEQ ID NO:153"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"182pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00041","num":"00041"},"img":{"@attributes":{"id":"EMI-C00041","he":"4.74mm","wi":"61.47mm","file":"US07413537-20080819-C00041.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"28"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00042","num":"00042"},"img":{"@attributes":{"id":"EMI-C00042","he":"8.13mm","wi":"48.94mm","file":"US07413537-20080819-C00042.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"52"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00043","num":"00043"},"img":{"@attributes":{"id":"EMI-C00043","he":"5.16mm","wi":"43.18mm","file":"US07413537-20080819-C00043.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"73"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00044","num":"00044"},"img":{"@attributes":{"id":"EMI-C00044","he":"8.13mm","wi":"31.16mm","file":"US07413537-20080819-C00044.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"88"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00045","num":"00045"},"img":{"@attributes":{"id":"EMI-C00045","he":"10.58mm","wi":"59.69mm","file":"US07413537-20080819-C00045.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"118"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00046","num":"00046"},"img":{"@attributes":{"id":"EMI-C00046","he":"10.58mm","wi":"59.61mm","file":"US07413537-20080819-C00046.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"148"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00047","num":"00047"},"img":{"@attributes":{"id":"EMI-C00047","he":"12.62mm","wi":"59.61mm","file":"US07413537-20080819-C00047.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"178"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00048","num":"00048"},"img":{"@attributes":{"id":"EMI-C00048","he":"20.07mm","wi":"57.23mm","file":"US07413537-20080819-C00048.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"208"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00049","num":"00049"},"img":{"@attributes":{"id":"EMI-C00049","he":"8.64mm","wi":"52.15mm","file":"US07413537-20080819-C00049.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"235"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00050","num":"00050"},"img":{"@attributes":{"id":"EMI-C00050","he":"14.90mm","wi":"63.42mm","file":"US07413537-20080819-C00050.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"268"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00051","num":"00051"},"img":{"@attributes":{"id":"EMI-C00051","he":"11.94mm","wi":"52.15mm","file":"US07413537-20080819-C00051.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"295"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00052","num":"00052"},"img":{"@attributes":{"id":"EMI-C00052","he":"12.28mm","wi":"57.83mm","file":"US07413537-20080819-C00052.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"325"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00053","num":"00053"},"img":{"@attributes":{"id":"EMI-C00053","he":"14.22mm","wi":"41.15mm","file":"US07413537-20080819-C00053.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"346"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00054","num":"00054"},"img":{"@attributes":{"id":"EMI-C00054","he":"11.94mm","wi":"29.29mm","file":"US07413537-20080819-C00054.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"361"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00055","num":"00055"},"img":{"@attributes":{"id":"EMI-C00055","he":"12.02mm","wi":"52.15mm","file":"US07413537-20080819-C00055.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"388"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00056","num":"00056"},"img":{"@attributes":{"id":"EMI-C00056","he":"12.02mm","wi":"41.06mm","file":"US07413537-20080819-C00056.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"409"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00057","num":"00057"},"img":{"@attributes":{"id":"EMI-C00057","he":"15.66mm","wi":"29.29mm","file":"US07413537-20080819-C00057.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"424"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00058","num":"00058"},"img":{"@attributes":{"id":"EMI-C00058","he":"8.72mm","wi":"46.65mm","file":"US07413537-20080819-C00058.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"448"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00059","num":"00059"},"img":{"@attributes":{"id":"EMI-C00059","he":"11.60mm","wi":"58.08mm","file":"US07413537-20080819-C00059.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"478"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00060","num":"00060"},"img":{"@attributes":{"id":"EMI-C00060","he":"12.28mm","wi":"46.40mm","file":"US07413537-20080819-C00060.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"502"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00061","num":"00061"},"img":{"@attributes":{"id":"EMI-C00061","he":"7.87mm","wi":"57.74mm","file":"US07413537-20080819-C00061.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"532"]},{"entry":[{},{}]},{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00062","num":"00062"},"img":{"@attributes":{"id":"EMI-C00062","he":"7.79mm","wi":"20.91mm","file":"US07413537-20080819-C00062.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"539"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"Note the following enzyme equivalences,"]},{"entry":[{},"Xma III = Eag I"]},{"entry":[{},"Acc III = BspM II"]},{"entry":[{},"Dra II = EcoO109 I"]},{"entry":[{},"Asu II = BstB I"]},{"entry":[{},"Sau I = Bsu36 I"]}]}}]}}},{"@attributes":{"id":"p-0964","num":"1269"},"tables":{"@attributes":{"id":"TABLE-US-00053","num":"00053"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 26"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"DNA_seq1 (SEQ ID NO:154)"},{"entry":"Protein sequence SEQ ID NO:155"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00063","num":"00063"},"img":{"@attributes":{"id":"EMI-C00063","he":"5.50mm","wi":"74.76mm","file":"US07413537-20080819-C00063.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00064","num":"00064"},"img":{"@attributes":{"id":"EMI-C00064","he":"5.00mm","wi":"48.94mm","file":"US07413537-20080819-C00064.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00065","num":"00065"},"img":{"@attributes":{"id":"EMI-C00065","he":"5.42mm","wi":"43.10mm","file":"US07413537-20080819-C00065.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00066","num":"00066"},"img":{"@attributes":{"id":"EMI-C00066","he":"5.33mm","wi":"13.55mm","file":"US07413537-20080819-C00066.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00067","num":"00067"},"img":{"@attributes":{"id":"EMI-C00067","he":"10.67mm","wi":"36.66mm","file":"US07413537-20080819-C00067.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00068","num":"00068"},"img":{"@attributes":{"id":"EMI-C00068","he":"10.41mm","wi":"48.26mm","file":"US07413537-20080819-C00068.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00069","num":"00069"},"img":{"@attributes":{"id":"EMI-C00069","he":"5.50mm","wi":"60.79mm","file":"US07413537-20080819-C00069.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00070","num":"00070"},"img":{"@attributes":{"id":"EMI-C00070","he":"5.67mm","wi":"40.72mm","file":"US07413537-20080819-C00070.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}}}},{"@attributes":{"id":"p-0965","num":"1270"},"tables":{"@attributes":{"id":"TABLE-US-00054","num":"00054"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 27"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"DNA_synth1 (SEQ ID NO:154)"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00071","num":"00071"},"img":{"@attributes":{"id":"EMI-C00071","he":"2.54mm","wi":"76.62mm","file":"US07413537-20080819-C00071.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00072","num":"00072"},"img":{"@attributes":{"id":"EMI-C00072","he":"2.62mm","wi":"50.63mm","file":"US07413537-20080819-C00072.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00073","num":"00073"},"img":{"@attributes":{"id":"EMI-C00073","he":"5.50mm","wi":"43.10mm","file":"US07413537-20080819-C00073.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"175pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{"chemistry":{"@attributes":{"id":"CHEM-US-00074","num":"00074"},"img":{"@attributes":{"id":"EMI-C00074","he":"5.25mm","wi":"13.46mm","file":"US07413537-20080819-C00074.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"\/ 3\u2032 = olig#3 (SEQ ID NO:161)"]},{"entry":[{},{}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00075","num":"00075"},"img":{"@attributes":{"id":"EMI-C00075","he":"5.25mm","wi":"37.42mm","file":"US07413537-20080819-C00075.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00076","num":"00076"},"img":{"@attributes":{"id":"EMI-C00076","he":"5.08mm","wi":"48.94mm","file":"US07413537-20080819-C00076.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00077","num":"00077"},"img":{"@attributes":{"id":"EMI-C00077","he":"5.16mm","wi":"60.79mm","file":"US07413537-20080819-C00077.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00078","num":"00078"},"img":{"@attributes":{"id":"EMI-C00078","he":"5.16mm","wi":"61.47mm","file":"US07413537-20080819-C00078.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"119pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"119pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["\u201cTop\u201d strand","99",{}]},{"entry":["\u201cBottom\u201d strand","100"]},{"entry":["Overlap","23","(14 c\/g and 9 a\/t)"]},{"entry":["Net length","158"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0966","num":"1271"},"tables":{"@attributes":{"id":"TABLE-US-00055","num":"00055"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 28 "},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"DNA_seq2 (SEQ ID NO:157)"},{"entry":"Protein sequence SEQ ID NO:158"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00079","num":"00079"},"img":{"@attributes":{"id":"EMI-C00079","he":"6.60mm","wi":"22.69mm","file":"US07413537-20080819-C00079.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00080","num":"00080"},"img":{"@attributes":{"id":"EMI-C00080","he":"8.72mm","wi":"30.31mm","file":"US07413537-20080819-C00080.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00081","num":"00081"},"img":{"@attributes":{"id":"EMI-C00081","he":"10.84mm","wi":"60.45mm","file":"US07413537-20080819-C00081.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00082","num":"00082"},"img":{"@attributes":{"id":"EMI-C00082","he":"12.28mm","wi":"61.04mm","file":"US07413537-20080819-C00082.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00083","num":"00083"},"img":{"@attributes":{"id":"EMI-C00083","he":"14.22mm","wi":"59.94mm","file":"US07413537-20080819-C00083.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00084","num":"00084"},"img":{"@attributes":{"id":"EMI-C00084","he":"18.63mm","wi":"60.28mm","file":"US07413537-20080819-C00084.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00085","num":"00085"},"img":{"@attributes":{"id":"EMI-C00085","he":"9.14mm","wi":"19.05mm","file":"US07413537-20080819-C00085.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00086","num":"00086"},"img":{"@attributes":{"id":"EMI-C00086","he":"12.28mm","wi":"41.40mm","file":"US07413537-20080819-C00086.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}}}},{"@attributes":{"id":"p-0967","num":"1272"},"tables":{"@attributes":{"id":"TABLE-US-00056","num":"00056"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 29"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"DNA_synth2 (SEQ ID NO:157)"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00087","num":"00087"},"img":{"@attributes":{"id":"EMI-C00087","he":"3.89mm","wi":"22.61mm","file":"US07413537-20080819-C00087.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00088","num":"00088"},"img":{"@attributes":{"id":"EMI-C00088","he":"4.40mm","wi":"30.73mm","file":"US07413537-20080819-C00088.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00089","num":"00089"},"img":{"@attributes":{"id":"EMI-C00089","he":"4.40mm","wi":"60.45mm","file":"US07413537-20080819-C00089.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00090","num":"00090"},"img":{"@attributes":{"id":"EMI-C00090","he":"5.76mm","wi":"60.20mm","file":"US07413537-20080819-C00090.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00091","num":"00091"},"img":{"@attributes":{"id":"EMI-C00091","he":"7.45mm","wi":"72.81mm","file":"US07413537-20080819-C00091.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00092","num":"00092"},"img":{"@attributes":{"id":"EMI-C00092","he":"5.93mm","wi":"59.35mm","file":"US07413537-20080819-C00092.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00093","num":"00093"},"img":{"@attributes":{"id":"EMI-C00093","he":"5.93mm","wi":"17.53mm","file":"US07413537-20080819-C00093.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00094","num":"00094"},"img":{"@attributes":{"id":"EMI-C00094","he":"6.01mm","wi":"41.99mm","file":"US07413537-20080819-C00094.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["\u201cTop\u201d strand","99",{}]},{"entry":["\u201cBottom\u201d strand","99"]},{"entry":["Overlap","24","(14 c\/g and 10 a\/t)"]},{"entry":["Net length","155"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0968","num":"1273"},"tables":{"@attributes":{"id":"TABLE-US-00057","num":"00057"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 30"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"DNA_seq3 (SEQ ID NO:163)"},{"entry":"Protein sequence SEQ ID NO:164"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00095","num":"00095"},"img":{"@attributes":{"id":"EMI-C00095","he":"11.35mm","wi":"37.08mm","file":"US07413537-20080819-C00095.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00096","num":"00096"},"img":{"@attributes":{"id":"EMI-C00096","he":"9.23mm","wi":"55.12mm","file":"US07413537-20080819-C00096.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00097","num":"00097"},"img":{"@attributes":{"id":"EMI-C00097","he":"13.89mm","wi":"66.46mm","file":"US07413537-20080819-C00097.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00098","num":"00098"},"img":{"@attributes":{"id":"EMI-C00098","he":"11.51mm","wi":"54.19mm","file":"US07413537-20080819-C00098.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00099","num":"00099"},"img":{"@attributes":{"id":"EMI-C00099","he":"11.35mm","wi":"60.37mm","file":"US07413537-20080819-C00099.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00100","num":"00100"},"img":{"@attributes":{"id":"EMI-C00100","he":"14.05mm","wi":"24.47mm","file":"US07413537-20080819-C00100.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00101","num":"00101"},"img":{"@attributes":{"id":"EMI-C00101","he":"11.60mm","wi":"47.50mm","file":"US07413537-20080819-C00101.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}}}},{"@attributes":{"id":"p-0969","num":"1274"},"tables":{"@attributes":{"id":"TABLE-US-00058","num":"00058"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 31"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"DNA_synth3 (SEQ ID NO:163)"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00102","num":"00102"},"img":{"@attributes":{"id":"EMI-C00102","he":"4.32mm","wi":"35.98mm","file":"US07413537-20080819-C00102.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00103","num":"00103"},"img":{"@attributes":{"id":"EMI-C00103","he":"4.40mm","wi":"54.10mm","file":"US07413537-20080819-C00103.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00104","num":"00104"},"img":{"@attributes":{"id":"EMI-C00104","he":"6.69mm","wi":"66.21mm","file":"US07413537-20080819-C00104.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00105","num":"00105"},"img":{"@attributes":{"id":"EMI-C00105","he":"8.21mm","wi":"73.49mm","file":"US07413537-20080819-C00105.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00106","num":"00106"},"img":{"@attributes":{"id":"EMI-C00106","he":"6.27mm","wi":"59.69mm","file":"US07413537-20080819-C00106.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00107","num":"00107"},"img":{"@attributes":{"id":"EMI-C00107","he":"6.10mm","wi":"24.72mm","file":"US07413537-20080819-C00107.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00108","num":"00108"},"img":{"@attributes":{"id":"EMI-C00108","he":"5.33mm","wi":"46.57mm","file":"US07413537-20080819-C00108.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"98pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["\u201cTop\u201d strand","93",{}]},{"entry":["\u201cBottom\u201d strand","97"]},{"entry":["Overlap","25","(15 g\/c & 10 a\/t)"]},{"entry":["Net length","146"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0970","num":"1275"},"tables":{"@attributes":{"id":"TABLE-US-00059","num":"00059"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 32"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"DNA_seq4 (SEQ ID NO:159)"},{"entry":"Protein sequence = SEQ ID NO:165"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00109","num":"00109"},"img":{"@attributes":{"id":"EMI-C00109","he":"13.55mm","wi":"59.10mm","file":"US07413537-20080819-C00109.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00110","num":"00110"},"img":{"@attributes":{"id":"EMI-C00110","he":"11.35mm","wi":"30.31mm","file":"US07413537-20080819-C00110.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00111","num":"00111"},"img":{"@attributes":{"id":"EMI-C00111","he":"11.26mm","wi":"54.36mm","file":"US07413537-20080819-C00111.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00112","num":"00112"},"img":{"@attributes":{"id":"EMI-C00112","he":"11.18mm","wi":"43.52mm","file":"US07413537-20080819-C00112.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00113","num":"00113"},"img":{"@attributes":{"id":"EMI-C00113","he":"14.14mm","wi":"33.02mm","file":"US07413537-20080819-C00113.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00114","num":"00114"},"img":{"@attributes":{"id":"EMI-C00114","he":"8.21mm","wi":"52.49mm","file":"US07413537-20080819-C00114.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00115","num":"00115"},"img":{"@attributes":{"id":"EMI-C00115","he":"11.68mm","wi":"78.82mm","file":"US07413537-20080819-C00115.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}}}},{"@attributes":{"id":"p-0971","num":"1276"},"tables":{"@attributes":{"id":"TABLE-US-00060","num":"00060"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 33"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"DNA_synth4 (SEQ ID NO:89)"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00116","num":"00116"},"img":{"@attributes":{"id":"EMI-C00116","he":"3.98mm","wi":"62.74mm","file":"US07413537-20080819-C00116.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00117","num":"00117"},"img":{"@attributes":{"id":"EMI-C00117","he":"3.81mm","wi":"30.65mm","file":"US07413537-20080819-C00117.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00118","num":"00118"},"img":{"@attributes":{"id":"EMI-C00118","he":"4.23mm","wi":"54.10mm","file":"US07413537-20080819-C00118.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00119","num":"00119"},"img":{"@attributes":{"id":"EMI-C00119","he":"6.52mm","wi":"77.89mm","file":"US07413537-20080819-C00119.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00120","num":"00120"},"img":{"@attributes":{"id":"EMI-C00120","he":"8.97mm","wi":"55.54mm","file":"US07413537-20080819-C00120.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00121","num":"00121"},"img":{"@attributes":{"id":"EMI-C00121","he":"5.84mm","wi":"48.77mm","file":"US07413537-20080819-C00121.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00122","num":"00122"},"img":{"@attributes":{"id":"EMI-C00122","he":"5.67mm","wi":"76.20mm","file":"US07413537-20080819-C00122.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"119pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"119pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["\u201cTop\u201d strand","100",{}]},{"entry":["\u201cBottom\u201d strand","93"]},{"entry":["Overlap","25","(14 c\/g and 11 a\/t)"]},{"entry":["Net length","149"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0972","num":"1277"},"tables":{"@attributes":{"id":"TABLE-US-00061","num":"00061"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"center"}},"thead":{"row":{"entry":"TABLE 34"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Some interaction sets in BPTI"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"9"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"14pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Number",{},{},{},{},{},{},{}]},{"entry":[{},"Diff."]},{"entry":["Res. #","AAs","Contents","BPTI","1","2","3","4","5"]},{"entry":{"@attributes":{"namest":"1","nameend":"9","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"9"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"14pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["\u22125","2","D \u221232","\u2014",{},{},{},{},{}]},{"entry":["\u22124","2","E \u221232","\u2014"]},{"entry":["\u22123","5","T P F Z \u221229","\u2014"]},{"entry":["\u22122","10","Z3 R3 Q2 T2 H G L K E \u221218","\u2014"]},{"entry":["\u22121","10","D4 T2 P2 Q2 E G N K R \u221218","\u2014"]},{"entry":["1","10","R21 A2 K2 H2 P L I T G D","R",{},{},{},{},"5"]},{"entry":["2","9","P20 R4 A2 H2 N E V F L","P",{},{},{},"s","5"]},{"entry":["3","10","D15 K6 T3 R2 P2 S Y G A L","D",{},{},{},"4","s"]},{"entry":["4","7","F19 D4 L3 Y2 I2 A2 S","F",{},{},{},"s","5"]},{"entry":["5","1","C33","C",{},{},{},"x","x"]},{"entry":["6","10","L11 E5 N4 K3 Q2 I2 Y2 D2 T R","L",{},{},{},"4"]},{"entry":["7","5","L18 E11 K2 S Q","E",{},{},"s","4"]},{"entry":["8","7","P26 H2 A2 I L G F","P",{},{},"3","4"]},{"entry":["9","9","P17 A6 V3 R2 Q L K Y F","P",{},"s","3","4"]},{"entry":["10","10","Y11 E7 D4 A2 N2 R2 V2 S I D","Y","s",{},"s","4"]},{"entry":["11","10","T17 P5 A3 R2 I S Q Y V K","T","1","s","3","4"]},{"entry":["12","2","G32 K","G","x",{},"x","x"]},{"entry":["13","5","P22 R6 E L3 N I","P","1",{},"s","4","s"]},{"entry":["14","3","C31 T A","C","1",{},"s","s","5"]},{"entry":["15","12","K15 R4 Y2 M2 L2 \u22122 V G A I N F","K","1","s","3","4","s"]},{"entry":["16","7","A22 G5 Q2 R K D F","A","1","s","s","s","5"]},{"entry":["17","12","R12 K5 A2 Y3 H2 S2 F2 L M T G P","R","1","2","3",{},"s"]},{"entry":["18","6","I21 M4 F3 L2 V2 T","I","1","s","s",{},"5"]},{"entry":["19","7","I11 P10 R6 S2 K2 L Q","I","1","2","3",{},"s"]},{"entry":["20","5","R19 A7 S4 L2 Q","R","s","s","s",{},"5"]},{"entry":["21","4","Y18 F13 W I","Y",{},"2","s","s","s"]},{"entry":["22","6","F14 Y14 H2 A N S","F",{},"s","3","4"]},{"entry":["23","2","Y32 F","Y",{},{},"s","s"]},{"entry":["24","4","N26 K3 D3 S","N",{},"s","3"]},{"entry":["25","10","A12 S5 Q3 P3 W3 L2 T2 K G R","A",{},{},"s","s"]},{"entry":["26","9","K16 A6 T2 E2 S2 R2 G H V","K",{},"s","3","4"]},{"entry":["27","5","A18 S8 K3 L2 T2","A",{},"2","3","4"]},{"entry":["28","7","G13 K10 N5 Q2 R H M","G",{},"2","s","s"]},{"entry":["29","10","L9 Q7 K7 A2 F2 R2 M G T N","L",{},"2","3"]},{"entry":["30","1","C33","C",{},"x","x","x"]},{"entry":["31","7","Q12 E11 L4 K2 V2 Y N","Q",{},"2","3","4"]},{"entry":["32","11","T12 P5 K4 Q3 E2 L2 G V S R A","T",{},"2","3","s"]},{"entry":["33","1","F33","F","x","x","x","x"]},{"entry":["34","11","V11 I8 T3 D2 N2 Q2 F H P R K","V","1","2","3","s"]},{"entry":["35","2","Y31 W2","Y","s","s","s",{},"5"]},{"entry":["36","3","G27 S5 R","G","1"]},{"entry":["37","1","G33","G","x",{},{},{},"x"]},{"entry":["38","3","C31 T A","C","1",{},{},"s","5"]},{"entry":["39","7","R13 G9 K4 Q3 D2 P M","R","1",{},{},"4","s"]},{"entry":["40","2","G22 A11","A","s",{},{},"s","5"]},{"entry":["41","3","N20 K11 D2","K",{},{},{},"4","s"]},{"entry":["42","9","A11 R9 S4 G3 H2 D Q K N","R",{},{},{},"s","5"]},{"entry":["43","2","N31 G2","N",{},{},{},{},"s"]},{"entry":["44","3","N21 R11 K","N",{},{},{},{},"s"]},{"entry":["45","2","F32 Y","F",{},{},{},{},"s"]},{"entry":["46","8","K24 E2 S2 D H V Y R","K",{},{},{},{},"5"]},{"entry":["47","2","T19 S14","S",{},"s",{},{},"5"]},{"entry":["48","9","A11 I9 E4 T2 W2 L2 R K D","A",{},"2","s",{},"s"]},{"entry":["49","7","E19 D6 A2 Q2 K2 T H","E",{},"2",{},{},"s"]},{"entry":["50","6","E16 D12 L2 M Q K","D",{},"s",{},{},"5"]},{"entry":["51","1","C33","C",{},"x",{},{},"x"]},{"entry":["52","7","R13 M10 L3 E3 Q2 H V","M",{},"2",{},{},"s"]},{"entry":["53","8","R21 Q3 E2 H2 C2 G K D","R",{},"s",{},{},"5"]},{"entry":["54","7","T23 A3 V2 E2 I Y K","T",{},{},{},{},"5"]},{"entry":["55","1","C33","C",{},{},{},{},"x"]},{"entry":["56","8","G15 V8 I3 E2 R2 A L S","G"]},{"entry":["57","8","G19 V4 A3 P2 \u22122 R L N","G"]},{"entry":["58","8","A11 \u221210 P3 K3 S2 Y2 R F","A"]},{"entry":["59","9","\u221224 G2 Q E A Y S P R","\u2014"]},{"entry":["60","6","\u221228 Q R I G D","\u2014"]},{"entry":["61","3","\u221231 T P","\u2014"]},{"entry":["62","2","\u221232 D","\u2014"]},{"entry":["63","2","\u221232 K","\u2014"]},{"entry":["64","2","\u221232 S","\u2014"]},{"entry":{"@attributes":{"namest":"1","nameend":"9","align":"center","rowsep":"1"}}},{"entry":"s indicates secondary set"},{"entry":"x indicates in or close to surface but buried and\/or highly conserved."}]}}]}}},{"@attributes":{"id":"p-0973","num":"1278"},"tables":{"@attributes":{"id":"TABLE-US-00062","num":"00062"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 35"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Distances from Cto"},{"entry":"Tip of Side Group"},{"entry":"in \u212b"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"119pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Amino Acid type","Distance"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"A","0.0"]},{"entry":[{},"C","1.8"]},{"entry":[{},"(reduced)"]},{"entry":[{},"D","2.4"]},{"entry":[{},"E","3.5"]},{"entry":[{},"F","4.3"]},{"entry":[{},"G","\u2014"]},{"entry":[{},"H","4.0"]},{"entry":[{},"I","2.5"]},{"entry":[{},"K","5.1"]},{"entry":[{},"L","2.6"]},{"entry":[{},"M","3.8"]},{"entry":[{},"N","2.4"]},{"entry":[{},"P","2.4"]},{"entry":[{},"Q","3.5"]},{"entry":[{},"R","6.0"]},{"entry":[{},"S","1.5"]},{"entry":[{},"T","1.5"]},{"entry":[{},"V","1.5"]},{"entry":[{},"W","5.3"]},{"entry":[{},"Y","5.7"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"Notes:"]},{"entry":[{},"These distances were calculated for standard model parts with all side groups fully extended."]}]}}]}}},{"@attributes":{"id":"p-0974","num":"1279"},"tables":{"@attributes":{"id":"TABLE-US-00063","num":"00063"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 36"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Distances, BPTI residue set #2"},{"entry":"Distances in \u212b between C"},{"entry":"Hypothetical Cwas added to each Glycine."},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"11"},"colspec":[{"@attributes":{"colname":"1","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"10","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"11","colwidth":"21pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"R17","I19","Y21","A27","G28","L29","Q31","T32","V34","A48"]},{"entry":{"@attributes":{"namest":"1","nameend":"11","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"11"},"colspec":[{"@attributes":{"colname":"1","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"8","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"9","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"10","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"11","colwidth":"21pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["I19","7.7",{},{},{},{},{},{},{},{},{}]},{"entry":["Y21","15.1","8.4"]},{"entry":["A27","22.6","17.1","12.2"]},{"entry":["G28","26.6","20.4","13.8","5.3"]},{"entry":["L29","22.5","15.8","9.6","5.1","5.2"]},{"entry":["Q31","16.1","10.4","6.8","6.8","10.6","6.8"]},{"entry":["T32","11.7","5.2","6.1","12.0","15.5","10.9","5.4"]},{"entry":["V34","5.6","6.5","11.6","17.6","21.7","18.0","11.4","8.2"]},{"entry":["A48","18.5","11.0","5.4","12.6","13.3","8.4","8.8","8.3","15.7"]},{"entry":["E49","22.0","14.7","8.9","16.9","16.1","12.2","13.9","13.3","19.8","5.5"]},{"entry":["M52","23.6","16.3","8.6","12.2","10.3","7.6","11.3","13.2","20.0","6.2"]},{"entry":["P9","14.0","11.3","9.0","12.2","15.4","13.3","7.9","9.2","8.7","13.9"]},{"entry":["T11","9.5","11.2","13.5","18.8","22.5","19.8","13.5","12.1","5.7","18.5"]},{"entry":["K15","7.9","14.6","20.1","27.4","31.3","27.9","21.4","18.1","10.3","24.6"]},{"entry":["A16","5.5","10.1","15.9","25.2","28.5","24.6","18.6","14.5","8.6","19.8"]},{"entry":["I18","6.1","6.0","11.2","21.3","24.4","20.2","14.7","10.4","7.0","15.0"]},{"entry":["R20","10.6","5.9","5.4","16.0","18.5","14.6","9.8","6.9","7.8","10.2"]},{"entry":["F22","15.6","10.9","5.6","10.5","12.8","10.3","6.2","8.1","10.8","10.3"]},{"entry":["N24","19.9","14.7","9.4","4.1","7.3","6.1","4.8","10.0","14.7","11.4"]},{"entry":["K26","24.4","20.1","15.2","5.4","7.7","9.8","10.1","15.3","19.0","17.0"]},{"entry":["C30","18.9","12.1","4.6","8.8","9.5","5.3","5.9","8.2","14.9","4.9"]},{"entry":["F33","10.8","7.4","7.7","12.6","16.4","13.0","6.6","5.6","5.5","12.2"]},{"entry":["Y35","8.4","7.4","9.4","18.4","21.4","17.9","12.2","9.5","5.8","14.4"]},{"entry":["S47","17.6","10.6","6.6","17.3","17.9","13.4","12.6","10.4","15.9","5.3"]},{"entry":["D50","20.0","13.6","7.2","17.2","16.8","13.5","13.5","12.9","17.6","7.6"]},{"entry":["C51","18.9","12.2","4.0","12.1","12.2","8.8","8.8","9.7","15.3","5.4"]},{"entry":["R53","25.4","18.6","11.0","17.2","15.0","13.0","15.7","16.7","22.3","9.7"]},{"entry":["R39","15.4","16.9","17.1","24.9","27.2","24.9","20.1","18.7","13.8","22.3"]},{"entry":{"@attributes":{"namest":"1","nameend":"11","align":"center","rowsep":"1"}}},{"entry":[{},"E49","M52","P9","T11","K15","A16","I18","R20","F22","N24"]},{"entry":{"@attributes":{"namest":"1","nameend":"11","align":"center","rowsep":"1"}}},{"entry":["M52","6.1"]},{"entry":["P9","17.7","15.5"]},{"entry":["T11","22.1","21.5","7.2"]},{"entry":["K15","27.5","28.7","16.4","9.5"]},{"entry":["A16","22.2","24.2","14.9","9.8","6.2"]},{"entry":["I18","17.4","19.5","12.2","9.5","10.4","4.9"]},{"entry":["R20","13.0","13.8","8.0","9.4","14.9","10.6","6.2"]},{"entry":["F22","13.8","11.4","4.1","10.6","19.1","16.3","12.7","6.9"]},{"entry":["N24","15.6","11.2","8.4","15.3","24.1","21.9","18.2","12.7","6.6"]},{"entry":["K26","20.9","15.7","12.1","18.6","27.9","26.6","23.3","18.1","11.6","5.9"]},{"entry":["C30","8.7","5.6","10.6","16.6","24.1","20.2","15.7","9.8","6.8","6.9"]},{"entry":["F33","16.5","15.4","4.2","7.1","15.0","12.8","9.6","6.1","5.6","9.3"]},{"entry":["Y35","17.2","17.8","7.8","5.8","11.0","7.6","4.9","4.3","8.8","14.8"]},{"entry":["S47","4.7","9.1","15.3","18.5","23.1","17.6","12.8","9.1","12.0","15.3"]},{"entry":["D50","5.5","7.7","14.7","18.6","24.2","19.2","14.7","9.9","11.0","14.7"]},{"entry":["C51","7.1","5.4","11.0","16.4","23.5","19.2","14.6","8.7","6.9","9.6"]},{"entry":["R53","6.3","5.6","17.9","23.1","29.6","24.8","20.3","15.0","13.8","15.5"]},{"entry":["R39","23.9","24.0","13.0","9.5","12.0","11.8","12.5","12.8","14.7","20.8"]},{"entry":{"@attributes":{"namest":"1","nameend":"11","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"9"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"K26","C30","F33","Y35","S47","D50","C51","R53"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"8","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"9"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"7","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"8","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"9","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["C30","12.4",{},{},{},{},{},{},{}]},{"entry":["F33","13.9","10.1"]},{"entry":["Y35","19.5","13.5","6.4"]},{"entry":["S47","21.0","8.8","13.5","13.2"]},{"entry":["D50","20.1","8.6","14.3","13.7","5.0"]},{"entry":["C51","15.0","3.7","10.9","12.5","6.9","5.2"]},{"entry":["R53","19.9","9.9","18.2","18.8","9.4","5.8","7.4"]},{"entry":["R39","24.3","20.6","14.4","9.6","20.4","19.0","18.8","23.4"]},{"entry":{"@attributes":{"namest":"1","nameend":"9","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0975","num":"1280"},"tables":{"@attributes":{"id":"TABLE-US-00064","num":"00064"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 37"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"vgDNA to vary BPTI set #2.1 (SEQ ID NO:170)"},{"entry":"Protein sequence = SEQ ID NO:171"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"231pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00123","num":"00123"},"img":{"@attributes":{"id":"EMI-C00123","he":"12.95mm","wi":"52.66mm","file":"US07413537-20080819-C00123.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"208"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00124","num":"00124"},"img":{"@attributes":{"id":"EMI-C00124","he":"10.41mm","wi":"55.03mm","file":"US07413537-20080819-C00124.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"235"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00125","num":"00125"},"img":{"@attributes":{"id":"EMI-C00125","he":"17.36mm","wi":"75.69mm","file":"US07413537-20080819-C00125.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"268"]},{"entry":{}},{"entry":"Overlap = 12 (7 CG, 5 AT)"},{"entry":{}},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00126","num":"00126"},"img":{"@attributes":{"id":"EMI-C00126","he":"12.87mm","wi":"54.95mm","file":"US07413537-20080819-C00126.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"295"]},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00127","num":"00127"},"img":{"@attributes":{"id":"EMI-C00127","he":"15.32mm","wi":"60.88mm","file":"US07413537-20080819-C00127.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"k = equal parts of T and G; m = equal parts of C and A;"},{"entry":"q = (.26 T, .18 C, .26 A, and .30 G);"},{"entry":"f = (.22 T, .16 C, .40 A, and .22 G);"},{"entry":"* = complement of symbol above"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"189pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Residue","40\u2003\u2003\u200342\u2003\u2003\u200350\u2003\u2003\u200352\u2003\u2003\u200357\u2003\u2003\u200371"]},{"entry":["Possibilities","21 \u00d7 21 \u00d7 21 \u00d7 21 \u00d7 21 \u00d7 21 = 8.6 \u00d7 10"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"Abundance \u00d7 10:"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"189pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":["of PPBD",".768 .271 .459 .671 .600 .459"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Produce = 1.77 \u00d7 10"},{"entry":"Parent = 1\/(5.5 \u00d7 10) least favored = 1\/(4.2 \u00d7 10)"},{"entry":"Least favored one-amino-acid substitution from PPBD"},{"entry":"present at 1 in 1.6 \u00d7 10"}]}}]}}},{"@attributes":{"id":"p-0976","num":"1281"},"tables":{"@attributes":{"id":"TABLE-US-00065","num":"00065"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 38"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Result of varying set#2 of BPTI 2.1"},{"entry":"DNA sequence = SEQ ID NO:174"},{"entry":"Protein sequence = SEQ ID NO:175"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"196pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00128","num":"00128"},"img":{"@attributes":{"id":"EMI-C00128","he":"12.70mm","wi":"13.55mm","file":"US07413537-20080819-C00128.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"178"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00129","num":"00129"},"img":{"@attributes":{"id":"EMI-C00129","he":"17.61mm","wi":"60.88mm","file":"US07413537-20080819-C00129.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"208"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00130","num":"00130"},"img":{"@attributes":{"id":"EMI-C00130","he":"7.62mm","wi":"54.95mm","file":"US07413537-20080819-C00130.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"235"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00131","num":"00131"},"img":{"@attributes":{"id":"EMI-C00131","he":"7.54mm","wi":"66.80mm","file":"US07413537-20080819-C00131.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"268"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00132","num":"00132"},"img":{"@attributes":{"id":"EMI-C00132","he":"10.24mm","wi":"54.95mm","file":"US07413537-20080819-C00132.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"295"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00133","num":"00133"},"img":{"@attributes":{"id":"EMI-C00133","he":"10.24mm","wi":"60.88mm","file":"US07413537-20080819-C00133.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"325"]},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00134","num":"00134"},"img":{"@attributes":{"id":"EMI-C00134","he":"12.70mm","wi":"13.55mm","file":"US07413537-20080819-C00134.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0977","num":"1282"},"tables":{"@attributes":{"id":"TABLE-US-00066","num":"00066"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 39"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"vgDNA to vary set#2 BPTI 2.2 (SEQ ID NO:176)"},{"entry":"Protein sequence = SEQ ID NO:177"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"196pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00135","num":"00135"},"img":{"@attributes":{"id":"EMI-C00135","he":"12.78mm","wi":"59.52mm","file":"US07413537-20080819-C00135.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"208"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00136","num":"00136"},"img":{"@attributes":{"id":"EMI-C00136","he":"9.99mm","wi":"55.37mm","file":"US07413537-20080819-C00136.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"235"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00137","num":"00137"},"img":{"@attributes":{"id":"EMI-C00137","he":"12.87mm","wi":"87.29mm","file":"US07413537-20080819-C00137.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"268"]},{"entry":{}},{"entry":"Overlap = 15 (11 CG, 4 AT)"},{"entry":{}},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00138","num":"00138"},"img":{"@attributes":{"id":"EMI-C00138","he":"15.07mm","wi":"71.46mm","file":"US07413537-20080819-C00138.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"295"]},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00139","num":"00139"},"img":{"@attributes":{"id":"EMI-C00139","he":"15.24mm","wi":"58.50mm","file":"US07413537-20080819-C00139.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"k = equal parts of T and G;"},{"entry":"v = equal parts of C, A, and G;"},{"entry":"m = equal parts of C and A;"},{"entry":"r = equal parts of A and G;"},{"entry":"w = equal parts of A and T;"},{"entry":"q = (.26 T, .18 C, .26 A, and .30 G);"},{"entry":"f = (.22 T, .16 C, .40 A, and .22 G);"},{"entry":"* = complement of symbol above"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"168pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Residue","38\u2003\u2003\u200341\u2003\u2003\u200343\u2003\u2003\u200344\u2003\u2003\u200351\u2003\u2003\u200354\u2003\u2003\u200355\u2003\u2003\u200372"]},{"entry":["Possi-","\u20034 \u00d7 \u20034 \u00d7 \u20039 \u00d7 \u20032 \u00d7 21 \u00d7 21 \u00d7 21 \u00d7 21 ="]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"168pt","align":"right"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":["bilities","6.2 \u00d7 10"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"168pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Abundance \u00d7","2.5\u2003\u20032.5\u2003\u2003.833\u20035.\u2003\u2003\u2003.663\u2003.397\u2003.437\u2003.602"]},{"entry":"10"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Product = 2.3 \u00d7 10"},{"entry":"Parent = 1\/(4.4 \u00d7 10)"},{"entry":"least favored = 1\/(1.25 \u00d7 10)"},{"entry":"Least favored one-amino-acid substitution from"},{"entry":"PPBD present at 1 in 1.2 \u00d7 10"}]}}]}}},{"@attributes":{"id":"p-0978","num":"1283"},"tables":{"@attributes":{"id":"TABLE-US-00067","num":"00067"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 40"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Result of varying set#2 of BPTI 2.2"},{"entry":"DNA sequence = SEQ ID NO:180"},{"entry":"Protein sequence = SEQ ID NO:181"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"196pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00140","num":"00140"},"img":{"@attributes":{"id":"EMI-C00140","he":"10.24mm","wi":"13.46mm","file":"US07413537-20080819-C00140.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"178"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00141","num":"00141"},"img":{"@attributes":{"id":"EMI-C00141","he":"12.78mm","wi":"60.88mm","file":"US07413537-20080819-C00141.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"208"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00142","num":"00142"},"img":{"@attributes":{"id":"EMI-C00142","he":"7.54mm","wi":"54.86mm","file":"US07413537-20080819-C00142.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"235"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00143","num":"00143"},"img":{"@attributes":{"id":"EMI-C00143","he":"7.54mm","wi":"66.72mm","file":"US07413537-20080819-C00143.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"268"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00144","num":"00144"},"img":{"@attributes":{"id":"EMI-C00144","he":"10.33mm","wi":"54.86mm","file":"US07413537-20080819-C00144.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"295"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00145","num":"00145"},"img":{"@attributes":{"id":"EMI-C00145","he":"10.33mm","wi":"60.79mm","file":"US07413537-20080819-C00145.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"325"]},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00146","num":"00146"},"img":{"@attributes":{"id":"EMI-C00146","he":"12.70mm","wi":"13.46mm","file":"US07413537-20080819-C00146.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0979","num":"1284"},"tables":{"@attributes":{"id":"TABLE-US-00068","num":"00068"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 41"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"vg DNA set#2 of BPTI 2.3 (SEQ ID NO:182)"},{"entry":"Protein sequence = SEQ ID NO:183"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"196pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00147","num":"00147"},"img":{"@attributes":{"id":"EMI-C00147","he":"10.41mm","wi":"40.30mm","file":"US07413537-20080819-C00147.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"178"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00148","num":"00148"},"img":{"@attributes":{"id":"EMI-C00148","he":"10.33mm","wi":"60.88mm","file":"US07413537-20080819-C00148.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"208"]},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00149","num":"00149"},"img":{"@attributes":{"id":"EMI-C00149","he":"17.44mm","wi":"60.88mm","file":"US07413537-20080819-C00149.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"Overlap = 13 (7 CG, 6 AT)"},{"entry":{}},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00150","num":"00150"},"img":{"@attributes":{"id":"EMI-C00150","he":"12.53mm","wi":"66.80mm","file":"US07413537-20080819-C00150.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"268"]},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00151","num":"00151"},"img":{"@attributes":{"id":"EMI-C00151","he":"12.87mm","wi":"46.57mm","file":"US07413537-20080819-C00151.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"k = equal parts of T and G;"},{"entry":"m = equal parts of C and A;"},{"entry":"w = equal parts of A and T;"},{"entry":"n = equal parts of A, C, G, T;"},{"entry":"d = equal parts A, G, T;"},{"entry":"v = equal parts A, C, G;"},{"entry":"q = (.26 T, .18 C, .26 A, and .30 G);"},{"entry":"f = (.22 T, .16 C, .40 A, and .22 G);"},{"entry":"* = complement of symbol above"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"9"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"8","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"9","colwidth":"28pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Residues","32","34","40","44","50","52","55","57"]},{"entry":["Possibilities","\u20036 \u00d7","\u20036 \u00d7","21 \u00d7","\u20036 \u00d7","\u20033 \u00d7","\u20035 \u00d7","21 \u00d7","21 ="]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"right"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"3 \u00d7 10"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"9"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"8","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"9","colwidth":"28pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Abundance \u00d7","10\/6","10\/6",".545","10\/6","10\/3","30\/8",".459",".701"]},{"entry":"10 of PPBD"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"product = 1.01 \u00d7 10"},{"entry":"parent = 1\/(1 \u00d7 10)"},{"entry":"least favored = 1\/(4 \u00d7 10)"},{"entry":"Least favored one-amino-acid substitution from"},{"entry":"PPBD present at 1 in 3 \u00d7 10"}]}}]}}},{"@attributes":{"id":"p-0980","num":"1285"},"tables":{"@attributes":{"id":"TABLE-US-00069","num":"00069"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 42"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Result of varying set#2 of BPTI 2.3"},{"entry":"DNA sequence = SEQ ID NO:186"},{"entry":"Protein sequence = SEQ ID NO:187"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"196pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00152","num":"00152"},"img":{"@attributes":{"id":"EMI-C00152","he":"12.70mm","wi":"13.46mm","file":"US07413537-20080819-C00152.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"178"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00153","num":"00153"},"img":{"@attributes":{"id":"EMI-C00153","he":"10.41mm","wi":"60.79mm","file":"US07413537-20080819-C00153.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"208"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00154","num":"00154"},"img":{"@attributes":{"id":"EMI-C00154","he":"7.54mm","wi":"54.86mm","file":"US07413537-20080819-C00154.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"235"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00155","num":"00155"},"img":{"@attributes":{"id":"EMI-C00155","he":"7.54mm","wi":"66.72mm","file":"US07413537-20080819-C00155.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"268"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00156","num":"00156"},"img":{"@attributes":{"id":"EMI-C00156","he":"10.33mm","wi":"54.86mm","file":"US07413537-20080819-C00156.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"295"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00157","num":"00157"},"img":{"@attributes":{"id":"EMI-C00157","he":"10.33mm","wi":"60.79mm","file":"US07413537-20080819-C00157.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"325"]},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00158","num":"00158"},"img":{"@attributes":{"id":"EMI-C00158","he":"12.78mm","wi":"13.38mm","file":"US07413537-20080819-C00158.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0981","num":"1286"},"tables":{"@attributes":{"id":"TABLE-US-00070","num":"00070"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"56pt","align":"left"}}],"thead":{"row":[{"entry":"TABLE 50"},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}},{"entry":[{},"Number",{},{},{},{},{}]},{"entry":[{},"Amino",{},"Cross"]},{"entry":["IPBD","Acids","Structure","Links","Secreted","Source Organism","AfM"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"329pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"Preferred IPBDs"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Aprotinin","58","X-ray, NMR","3 SS","yes",{"i":"Bos taurus"},"trypsin"]},{"entry":[{},{},{},"5-55, 14-38"]},{"entry":[{},{},{},"30-51"]},{"entry":[{},{},{},"(1:6, 2:4, 3:5)"]},{"entry":["Crambin","46","X-ray, NMR","3 SS","yes","rape seed","?, Mab"]},{"entry":["CMTI-III","26","NMR","3 SS","yes","cucumber","trypsin"]},{"entry":["ST-I","13","NMR","3 SS","yes",{"i":"E. coli"},"MAbs &"]},{"entry":[{},{},{},{},{},{},"guanylate cyclase"]},{"entry":["Third domain,","56","X-ray, NMR","3 SS","yes",{"i":"Coturnix"},"trypsin"]},{"entry":["ovomucoid",{},{},{},{},{"i":"coturnix"}]},{"entry":[{},{},{},{},{},{"i":"japonica"}]},{"entry":["Ribonuclease A","124","X-ray, NMR",{},"yes",{"i":"Bos Taurus"},"RNA, DNA"]},{"entry":["Ribonuclease","104","X-ray, NMR?",{},"yes",{"i":"A. oruzae"},"RNA, DNA"]},{"entry":["Lysozyme","129","X-ray, NMR?","4 SS","yes",{"i":"Gallus gallus"},"NAG-NAM-NAG"]},{"entry":["Azurin","128","X-ray","Cu:CYS,",{},{"i":"P. aerugenosa"},"Mab"]},{"entry":[{},{},{},"HIS, MET"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"329pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"Characteristics of Known IPBDs"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["\u03b1-Conotoxins","13-15","NMR","2 SS","yes",{"i":"Conus "},"Receptor"]},{"entry":["\u03bc-Conotoxins","20-25","NMR","3 SS","yes",{"i":"Conus "},"Receptor"]},{"entry":["\u03a9-Conotoxins","25-30","\u2014","3 SS","yes",{"i":"Conus "},"Receptor"]},{"entry":["King-kong","25-30","\u2014","3 SS","yes",{"i":"Conus "},"Mabs"]},{"entry":"peptides"},{"entry":["Nuclease","141","X-ray","none","yes",{"i":"S aurius"},"RNA, DNA"]},{"entry":"(staphylococcal)"},{"entry":["Charybdotoxin 37",{},"NMR","3 SS","yes",{"i":"Leiurus"},"Ca-"]},{"entry":["(scorpion toxin)",{},{},"7-28, 13-33",{},{"i":"quinquestriatus"},"dependent"]},{"entry":[{},{},{},"17-35",{},{"i":"hebraeus"},"Kchannel"]},{"entry":[{},{},{},"(1:4, 2:5, 3:6)"]},{"entry":["Apamin","12","NMR","2SS","yes","Bees","Mabs,"]},{"entry":["(bee venom)",{},{},"(1:3, 2:4)",{},{},"Receptor (?)"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}},{"entry":"Other suitable IPBDs"},{"entry":"Ferredoxin"},{"entry":"Secretory trypsin inhibitor"},{"entry":"Soybean trypsin inhibitor"},{"entry":"SLPI (Secretory Leukocyte Protease Inhibitor) (THOM86) and SPAI (ARAK90)"},{"entry":"Cystatin and homologues (MACH89, STUB90)"},{"entry":"Eglin (MCPH85)"},{"entry":"Barley inhibitor (CLOR87a, CLOR87b, SVEN82)"}]}}]}}},{"@attributes":{"id":"p-0982","num":"1287"},"tables":{"@attributes":{"id":"TABLE-US-00071","num":"00071"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 101a"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"VIIIsignal::bpti::VIII-coat gene (SEQ ID NO:188)"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["pbd mod14: 9 V 89 : Sequence cloned into pGEM-MB1",{}]},{"entry":{}},{"entry":"pGEM-3Zf(\u2212) [HincII]::lacUV5 SacI\/gene\/"},{"entry":{}},{"entry":"TrpA attenuator\/(SalI)::pGEM-3Zf(\u2212) [HincII]\u2003!"},{"entry":{}},{"entry":"5\u2032-(GAATTC GAGCTCGGTACCCGG \u2003TCTAGAGTC)- !polylinker"},{"entry":{}},{"entry":"GGC \u2003CTTTATGCTTCCGGCTCG \u2003GTG !\u2003lacUV5"},{"entry":{}},{"entry":"TGG \u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003!\u2003lacO-symm operator"},{"entry":{}},{"entry":{"u":["gagctc","AG(G)AGG","Sac"],"sup":"a"}},{"entry":{}},{"entry":"atg aag aaa tct ctg gtt ctt aag gct agc !\u200310, M13 leader"},{"entry":{}},{"entry":"gtt gct gtc gcg acc ctg gta cct atg ttg !\u200320 <- codon #"},{"entry":{}},{"entry":"tcc ttc gct cgt ccg gat ttc tgt ctc gag !\u200330"},{"entry":{}},{"entry":"cca cca tac act ggg ccc tgc aaa gcg cgc !\u200340"},{"entry":{}},{"entry":"atc atc cgC tat ttc tac aat gct aaa gca !\u200350"},{"entry":{}},{"entry":"ggc ctg tgc cag acc ttt gta tac ggt ggt !\u200360"},{"entry":{}},{"entry":"tgc cgt gct aag cgt aac aac ttt aaa tcg !\u200370"},{"entry":{}},{"entry":"gcc gaa gat tgc atg cgt acc tgc ggt ggc !\u200380"},{"entry":{}},{"entry":"gcc gct gaa ggt gat gat ccg gcc aaG gcg !\u200390"},{"entry":{}},{"entry":"gcc ttc aat tct ctG caa gct tct gct acc !\u2003100"},{"entry":{}},{"entry":"gag tat att ggt tac gcg tgg gcc atg gtg !\u2003110"},{"entry":{}},{"entry":"gtg gtt atc gtt ggt gct acc atc ggg atc !\u2003120"},{"entry":{}},{"entry":"aaa ctg ttc aag aag ttt act tcg aag gcg !\u2003130"},{"entry":{}},{"entry":"tct taa tga tag \u2003!\u2003\u2003\u2003BstEII"},{"entry":{}},{"entry":"AGTCTA AGCCCGC CTAATGA GCGGGCT TTTTTTTT !\u2003terminator"},{"entry":{}},{"entry":"aTCGA-\u2003\u2003!\u2003(SalI ghost)"},{"entry":{}},{"entry":"(GACCTGCAGGCATGCAAGCTT . . . -3\u2032) !\u2003pGEM polylinker"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":"Notes:"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"left","id":"FOO-00076"},"sup":"a"}}]}}]}}},{"@attributes":{"id":"p-0983","num":"1288"},"tables":{"@attributes":{"id":"TABLE-US-00072","num":"00072"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 101b"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"VIII-signal::bpti::VIII-coat gene (SEQ ID NO:189)"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["BamHI-SalI cassette, after insertion of SalI linker",{}]},{"entry":"in PstI site of pGEM-MB1."},{"entry":{}},{"entry":"pGEM-3Zf(\u2212) [HincII]::lacUV5 SacI\/gene\/"},{"entry":{}},{"entry":"TrpA attenuator\/(SalI)::pGEM-3Zf(\u2212) [HincII]\u2003!"},{"entry":{}},{"entry":"5\u2032-GAATTC GAGCTC GGTACCCGG \u2003TCTAGA GTC- !\u2003BamHI"},{"entry":{}},{"entry":"GGC \u2003CTTTATGCTTCCGGCTCG tataat GTG !\u2003lacUV5"},{"entry":{}},{"entry":"TGG \u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003!\u2003lacO-symm operator"},{"entry":{}},{"entry":{"u":["gagctc","AGAGG"]}},{"entry":{}},{"entry":"atg aag aaa tct ctg gtt ctt aag gct agc !\u200310, M13 leader"},{"entry":{}},{"entry":"gtt gct gtc gcg acc ctg gta cct atg ttg !\u200320 <- codon #"},{"entry":{}},{"entry":"tcc ttc gct cgt ccg gat ttc tgt ctc gag !\u200330"},{"entry":{}},{"entry":"cca cca tac act ggg ccc tgc aaa gcg cgc !\u200340"},{"entry":{}},{"entry":"atc atc cgC tat ttc tac aat gct aaa gca !\u200350"},{"entry":{}},{"entry":"ggc ctg tgc cag acc ttt gta tac ggt ggt !\u200360"},{"entry":{}},{"entry":"tgc cgt gct aag cgt aac aac ttt aaa tcg !\u200370"},{"entry":{}},{"entry":"gcc gaa gat tgc atg cgt acc tgc ggt ggc !\u200380"},{"entry":{}},{"entry":"gcc gct gaa ggt gat gat ccg gcc aaG gcg !\u200390"},{"entry":{}},{"entry":"gcc ttc aat tct ctG caa gct tct gct acc !\u2003100"},{"entry":{}},{"entry":"gag tat att ggt tac gcg tgg gcc atg gtg !\u2003110"},{"entry":{}},{"entry":"gtg gtt atc gtt ggt gct acc atc ggg atc !\u2003120"},{"entry":{}},{"entry":"aaa ctg ttc aag aag ttt act tcg aag gcg !\u2003130"},{"entry":{}},{"entry":"tct taa tga tag \u2003!\u2003\u2003\u2003BstEII"},{"entry":{}},{"entry":"AGTCTA AGCCCGC CTAATGA GCGGGCT TTTTTTTT !\u2003terminator"},{"entry":{}},{"entry":"aTCGA GACctgca GGTCGACC ggcatgc-3\u2032"},{"entry":"\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0984","num":"1289"},"tables":{"@attributes":{"id":"TABLE-US-00073","num":"00073"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"266pt","align":"center"}},"thead":{"row":{"entry":"TABLE 102a"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Annotated Sequence of gene found in pGEM-MB1 (SEQ ID NO:190)"},{"entry":"Protein sequence = SEQ ID NO:191"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"217pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"nucleotide"]},{"entry":[{},"number"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00159","num":"00159"},"img":{"@attributes":{"id":"EMI-C00159","he":"5.08mm","wi":"39.88mm","file":"US07413537-20080819-C00159.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"3"]},{"entry":{}},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00160","num":"00160"},"img":{"@attributes":{"id":"EMI-C00160","he":"5.08mm","wi":"59.35mm","file":"US07413537-20080819-C00160.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"39"]},{"entry":{}},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00161","num":"00161"},"img":{"@attributes":{"id":"EMI-C00161","he":"5.00mm","wi":"31.24mm","file":"US07413537-20080819-C00161.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"59"]},{"entry":{}},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00162","num":"00162"},"img":{"@attributes":{"id":"EMI-C00162","he":"5.76mm","wi":"45.97mm","file":"US07413537-20080819-C00162.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"77"]},{"entry":{}},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00163","num":"00163"},"img":{"@attributes":{"id":"EMI-C00163","he":"9.99mm","wi":"62.31mm","file":"US07413537-20080819-C00163.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"107"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00164","num":"00164"},"img":{"@attributes":{"id":"EMI-C00164","he":"9.99mm","wi":"62.31mm","file":"US07413537-20080819-C00164.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"137"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00165","num":"00165"},"img":{"@attributes":{"id":"EMI-C00165","he":"12.36mm","wi":"62.31mm","file":"US07413537-20080819-C00165.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"167"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00166","num":"00166"},"img":{"@attributes":{"id":"EMI-C00166","he":"17.53mm","wi":"62.31mm","file":"US07413537-20080819-C00166.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"197"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00167","num":"00167"},"img":{"@attributes":{"id":"EMI-C00167","he":"7.54mm","wi":"62.31mm","file":"US07413537-20080819-C00167.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"226"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00168","num":"00168"},"img":{"@attributes":{"id":"EMI-C00168","he":"12.87mm","wi":"63.92mm","file":"US07413537-20080819-C00168.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"257"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00169","num":"00169"},"img":{"@attributes":{"id":"EMI-C00169","he":"10.24mm","wi":"56.39mm","file":"US07413537-20080819-C00169.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"284"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00170","num":"00170"},"img":{"@attributes":{"id":"EMI-C00170","he":"10.24mm","wi":"62.31mm","file":"US07413537-20080819-C00170.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"314"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00171","num":"00171"},"img":{"@attributes":{"id":"EMI-C00171","he":"17.78mm","wi":"74.25mm","file":"US07413537-20080819-C00171.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"350"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00172","num":"00172"},"img":{"@attributes":{"id":"EMI-C00172","he":"10.33mm","wi":"56.39mm","file":"US07413537-20080819-C00172.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"377"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00173","num":"00173"},"img":{"@attributes":{"id":"EMI-C00173","he":"7.54mm","wi":"44.53mm","file":"US07413537-20080819-C00173.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"398"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00174","num":"00174"},"img":{"@attributes":{"id":"EMI-C00174","he":"12.78mm","wi":"56.39mm","file":"US07413537-20080819-C00174.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"425"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00175","num":"00175"},"img":{"@attributes":{"id":"EMI-C00175","he":"7.54mm","wi":"26.75mm","file":"US07413537-20080819-C00175.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"437"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00176","num":"00176"},"img":{"@attributes":{"id":"EMI-C00176","he":"9.99mm","wi":"62.31mm","file":"US07413537-20080819-C00176.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"467"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00177","num":"00177"},"img":{"@attributes":{"id":"EMI-C00177","he":"9.99mm","wi":"43.01mm","file":"US07413537-20080819-C00177.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"486"]},{"entry":{}},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00178","num":"00178"},"img":{"@attributes":{"id":"EMI-C00178","he":"5.16mm","wi":"59.35mm","file":"US07413537-20080819-C00178.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"521"]},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00179","num":"00179"},"img":{"@attributes":{"id":"EMI-C00179","he":"5.08mm","wi":"44.28mm","file":"US07413537-20080819-C00179.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":"Notes:"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"left","id":"FOO-00078"},"sup":"a"}},{"entry":"Note the following enzyme equivalences,"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"left","id":"FOO-00080"},"u":["Xma","Eag"]}},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"left","id":"FOO-00081"},"u":["Acc","BspM"]}},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"left","id":"FOO-00082"},"u":["Dra","EcoO109"]}},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"left","id":"FOO-00083"},"u":["Asu","BstB"]}}]}}]}}},{"@attributes":{"id":"p-0985","num":"1290"},"tables":{"@attributes":{"id":"TABLE-US-00074","num":"00074"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"266pt","align":"center"}},"thead":{"row":{"entry":"TABLE 102b"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Annotated Sequence of gene"},{"entry":"after insertion of I linker (SEQ ID NO:192)"},{"entry":"Protein sequence = (SEQ ID NO:191)"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"217pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"nucleotide"]},{"entry":[{},"number"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00180","num":"00180"},"img":{"@attributes":{"id":"EMI-C00180","he":"5.00mm","wi":"39.29mm","file":"US07413537-20080819-C00180.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"3"]},{"entry":{}},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00181","num":"00181"},"img":{"@attributes":{"id":"EMI-C00181","he":"5.00mm","wi":"60.79mm","file":"US07413537-20080819-C00181.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"39"]},{"entry":{}},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00182","num":"00182"},"img":{"@attributes":{"id":"EMI-C00182","he":"5.08mm","wi":"31.16mm","file":"US07413537-20080819-C00182.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"59"]},{"entry":{}},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00183","num":"00183"},"img":{"@attributes":{"id":"EMI-C00183","he":"5.08mm","wi":"43.01mm","file":"US07413537-20080819-C00183.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"77"]},{"entry":{}},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00184","num":"00184"},"img":{"@attributes":{"id":"EMI-C00184","he":"9.99mm","wi":"62.31mm","file":"US07413537-20080819-C00184.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"107"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00185","num":"00185"},"img":{"@attributes":{"id":"EMI-C00185","he":"9.99mm","wi":"62.31mm","file":"US07413537-20080819-C00185.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"137"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00186","num":"00186"},"img":{"@attributes":{"id":"EMI-C00186","he":"12.53mm","wi":"62.31mm","file":"US07413537-20080819-C00186.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"167"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00187","num":"00187"},"img":{"@attributes":{"id":"EMI-C00187","he":"17.44mm","wi":"62.31mm","file":"US07413537-20080819-C00187.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"197"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00188","num":"00188"},"img":{"@attributes":{"id":"EMI-C00188","he":"7.54mm","wi":"62.31mm","file":"US07413537-20080819-C00188.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"226"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00189","num":"00189"},"img":{"@attributes":{"id":"EMI-C00189","he":"12.53mm","wi":"63.84mm","file":"US07413537-20080819-C00189.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"257"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00190","num":"00190"},"img":{"@attributes":{"id":"EMI-C00190","he":"9.99mm","wi":"56.39mm","file":"US07413537-20080819-C00190.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"284"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00191","num":"00191"},"img":{"@attributes":{"id":"EMI-C00191","he":"9.99mm","wi":"62.31mm","file":"US07413537-20080819-C00191.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"314"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00192","num":"00192"},"img":{"@attributes":{"id":"EMI-C00192","he":"17.78mm","wi":"74.17mm","file":"US07413537-20080819-C00192.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"350"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00193","num":"00193"},"img":{"@attributes":{"id":"EMI-C00193","he":"12.53mm","wi":"56.39mm","file":"US07413537-20080819-C00193.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"377"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00194","num":"00194"},"img":{"@attributes":{"id":"EMI-C00194","he":"10.08mm","wi":"45.97mm","file":"US07413537-20080819-C00194.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"398"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00195","num":"00195"},"img":{"@attributes":{"id":"EMI-C00195","he":"12.53mm","wi":"57.83mm","file":"US07413537-20080819-C00195.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"425"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00196","num":"00196"},"img":{"@attributes":{"id":"EMI-C00196","he":"7.54mm","wi":"28.19mm","file":"US07413537-20080819-C00196.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"437"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00197","num":"00197"},"img":{"@attributes":{"id":"EMI-C00197","he":"10.16mm","wi":"75.61mm","file":"US07413537-20080819-C00197.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"467"]},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00198","num":"00198"},"img":{"@attributes":{"id":"EMI-C00198","he":"10.24mm","wi":"41.57mm","file":"US07413537-20080819-C00198.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"486"]},{"entry":{}},{"entry":[{"chemistry":{"@attributes":{"id":"CHEM-US-00199","num":"00199"},"img":{"@attributes":{"id":"EMI-C00199","he":"5.16mm","wi":"60.28mm","file":"US07413537-20080819-C00199.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}},"521"]},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00200","num":"00200"},"img":{"@attributes":{"id":"EMI-C00200","he":"5.08mm","wi":"54.02mm","file":"US07413537-20080819-C00200.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":"Note the following enzyme equivalences,"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"left","id":"FOO-00085"},"u":["Xma","Eag"]}},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"left","id":"FOO-00086"},"u":["Acc","BspM"]}},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"left","id":"FOO-00087"},"u":["Dra","EcoO109"]}},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"left","id":"FOO-00088"},"u":["Asu","BstB"]}}]}}]}}},{"@attributes":{"id":"p-0986","num":"1291"},"tables":{"@attributes":{"id":"TABLE-US-00075","num":"00075"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 102c"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Calculated properties of Peptide"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"For the apoprotein",{},{}]},{"entry":[{},"Molecular weight of peptide =","16192"]},{"entry":[{},"Charge on peptide =","9"]},{"entry":[{},"[A+G+P]\u2003=","36"]},{"entry":[{},"[C+F+H+I+L+M+V+W+Y]\u2003=","48"]},{"entry":[{},"[D+E+K+R+N+Q+S+T+.]\u2003=","48"]},{"entry":[{},{}]},{"entry":[{},"For the mature protein"]},{"entry":[{},"Molecular weight of peptide =","13339"]},{"entry":[{},"Charge on peptide =","6"]},{"entry":[{},"[A+G+P]\u2003=","31"]},{"entry":[{},"[C+F+H+I+L+M+V+W+Y]\u2003=","37"]},{"entry":[{},"[D+E+K+R+N+Q+S+T+.]\u2003=","41"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0987","num":"1292"},"tables":{"@attributes":{"id":"TABLE-US-00076","num":"00076"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 102d"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Codon Usage"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"63pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"First",{},"Second Base",{}]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"63pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Base","t","c","a","g","Third base"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"6","align":"center","rowsep":"1"}}]},{"entry":[{},"t","3","4","2","1","t"]},{"entry":[{},{},"5","1","4","5","c"]},{"entry":[{},{},"0","0","0","0","a"]},{"entry":[{},{},"1","2","0","1","g"]},{"entry":[{},"c","1","1","0","4","t"]},{"entry":[{},{},"1","1","0","2","c"]},{"entry":[{},{},"0","2","1","0","a"]},{"entry":[{},{},"5","2","1","0","g"]},{"entry":[{},"a","1","2","2","0","t"]},{"entry":[{},{},"5","5","2","1","c"]},{"entry":[{},{},"0","0","5","0","a"]},{"entry":[{},{},"4","0","7","0","g"]},{"entry":[{},"g","4","9","4","6","t"]},{"entry":[{},{},"1","5","0","2","c"]},{"entry":[{},{},"2","1","2","0","a"]},{"entry":[{},{},"2","5","2","2","g"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"6","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0988","num":"1293"},"tables":{"@attributes":{"id":"TABLE-US-00077","num":"00077"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 102e"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Amino-acid frequency"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"9"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"AA","#","AA","#","AA","#","AA","#"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"8","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"Encoded polypeptide"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"9"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"A","20","C","6","D","4","E","4"]},{"entry":[{},"F","8","G","10","H","0","I","6"]},{"entry":[{},"K","12","L","8","M","4","N","4"]},{"entry":[{},"P","6","Q","2","R","6","S","8"]},{"entry":[{},"T","7","V","9","W","1","Y","6"]},{"entry":[{},".","1"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"Mature protein"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"9"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"A","16","C","6","D","4","E","4"]},{"entry":[{},"F","7","G","10","H","0","I","6"]},{"entry":[{},"K","9","L","4","M","2","N","4"]},{"entry":[{},"P","5","Q","2","R","6","S","5"]},{"entry":[{},"T","6","V","5","W","1","Y","6"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"8","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0989","num":"1294"},"tables":{"@attributes":{"id":"TABLE-US-00078","num":"00078"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 102f"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Enzymes used to manipulate BPTI-gp8 fusion"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SacI",{},"GC",{},{}]},{"entry":["AflII","CG"]},{"entry":["NheI","GC"]},{"entry":["NruI","TCGCGA"]},{"entry":["KpnI","GC"]},{"entry":["AccIII =","BspMII","TA"]},{"entry":["AvaI","CG"]},{"entry":["XhoI","CG"]},{"entry":["PflMI","CCAnnTGG","(SEQ ID"]},{"entry":[{},{},{},"NO: 88)"]},{"entry":["BssHII","GC"]},{"entry":["ApaI","GC"]},{"entry":["DraII =","Eco109I","rGCy","(Same as"]},{"entry":[{},{},{},"PssI)"]},{"entry":["StuI","AGGCCT"]},{"entry":["AccI","GTAC"]},{"entry":["XcaI","GTATAC"]},{"entry":["EspI","GCGC"]},{"entry":["XmaIII","CG","(Supplier ?)"]},{"entry":["SphI","GC"]},{"entry":["BbeI","GC","(Supplier ?)"]},{"entry":["NarI","GGCC"]},{"entry":["SfiI","(SEQ ID NO: 151)","GGCCnnGGCC"]},{"entry":["HindIII","A|AGCTT"]},{"entry":["BstXI","(SEQ ID NO: 193)","CCAnnTGG"]},{"entry":["NcoI","CG"]},{"entry":["AsuII =","BstBI","TTAA"]},{"entry":["BstEII","GC"]},{"entry":["SalI","GC"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0990","num":"1295"},"tables":{"@attributes":{"id":"TABLE-US-00079","num":"00079"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 103"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Annotated Sequence of  gene"},{"entry":"DNA sequence = SEQ ID NO:194"},{"entry":"Protein sequence = SEQ ID NO:191"},{"entry":"Underscored bases indicate sites of overlap"},{"entry":"between annealed synthetic duplexes."},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00201","num":"00201"},"img":{"@attributes":{"id":"EMI-C00201","he":"7.45mm","wi":"69.77mm","file":"US07413537-20080819-C00201.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00202","num":"00202"},"img":{"@attributes":{"id":"EMI-C00202","he":"4.91mm","wi":"31.24mm","file":"US07413537-20080819-C00202.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00203","num":"00203"},"img":{"@attributes":{"id":"EMI-C00203","he":"5.16mm","wi":"55.54mm","file":"US07413537-20080819-C00203.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00204","num":"00204"},"img":{"@attributes":{"id":"EMI-C00204","he":"10.16mm","wi":"63.84mm","file":"US07413537-20080819-C00204.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00205","num":"00205"},"img":{"@attributes":{"id":"EMI-C00205","he":"10.24mm","wi":"64.35mm","file":"US07413537-20080819-C00205.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00206","num":"00206"},"img":{"@attributes":{"id":"EMI-C00206","he":"13.21mm","wi":"63.84mm","file":"US07413537-20080819-C00206.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00207","num":"00207"},"img":{"@attributes":{"id":"EMI-C00207","he":"17.70mm","wi":"62.31mm","file":"US07413537-20080819-C00207.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00208","num":"00208"},"img":{"@attributes":{"id":"EMI-C00208","he":"7.79mm","wi":"65.28mm","file":"US07413537-20080819-C00208.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00209","num":"00209"},"img":{"@attributes":{"id":"EMI-C00209","he":"12.70mm","wi":"63.92mm","file":"US07413537-20080819-C00209.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00210","num":"00210"},"img":{"@attributes":{"id":"EMI-C00210","he":"10.41mm","wi":"59.44mm","file":"US07413537-20080819-C00210.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00211","num":"00211"},"img":{"@attributes":{"id":"EMI-C00211","he":"10.41mm","wi":"63.84mm","file":"US07413537-20080819-C00211.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00212","num":"00212"},"img":{"@attributes":{"id":"EMI-C00212","he":"18.37mm","wi":"75.86mm","file":"US07413537-20080819-C00212.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00213","num":"00213"},"img":{"@attributes":{"id":"EMI-C00213","he":"10.24mm","wi":"57.91mm","file":"US07413537-20080819-C00213.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00214","num":"00214"},"img":{"@attributes":{"id":"EMI-C00214","he":"7.62mm","wi":"44.53mm","file":"US07413537-20080819-C00214.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00215","num":"00215"},"img":{"@attributes":{"id":"EMI-C00215","he":"12.70mm","wi":"58.34mm","file":"US07413537-20080819-C00215.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00216","num":"00216"},"img":{"@attributes":{"id":"EMI-C00216","he":"8.21mm","wi":"28.28mm","file":"US07413537-20080819-C00216.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00217","num":"00217"},"img":{"@attributes":{"id":"EMI-C00217","he":"10.08mm","wi":"62.31mm","file":"US07413537-20080819-C00217.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00218","num":"00218"},"img":{"@attributes":{"id":"EMI-C00218","he":"9.91mm","wi":"43.43mm","file":"US07413537-20080819-C00218.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00219","num":"00219"},"img":{"@attributes":{"id":"EMI-C00219","he":"5.00mm","wi":"63.50mm","file":"US07413537-20080819-C00219.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00220","num":"00220"},"img":{"@attributes":{"id":"EMI-C00220","he":"5.08mm","wi":"17.02mm","file":"US07413537-20080819-C00220.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0991","num":"1296"},"tables":{"@attributes":{"id":"TABLE-US-00080","num":"00080"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"392pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 104"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Definition and alignment of oligonucleotides"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"392pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Lines ending with \u201c-\u201d\u2003are continued on a following line. Blocks of ten bases are delimited",{}]},{"entry":"by \u201c-\u201d\u2003within a line. When a break in one strand does not correspond to a ten-base mark in"},{"entry":"the other strand, \u201c--\u201d\u2003is inserted in the other strand."},{"entry":{}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"322pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["\u2003\u2003\u2003\u2193Olig #801 (68 bases)","(SEQ ID NO:195)",{}]},{"entry":"5\u2032-"},{"entry":"3\u2032-\u2003\u2003"},{"entry":"\u2003\u2003\u2003\u2003filled in\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2191"},{"entry":{}},{"entry":["\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2193\u2003Olig #802 (67 bases)","(SEQ ID NO:196)"]},{"entry":{"b":"GCTCGTATAA-TGTGTGGAAT-TGTGAGCGCT-CACAATTGAG-CTCAGG","u":"AGGC-TTACTATGAA-"}},{"entry":{"b":"cgagcatatt-acacacctta-acactcgcga-gtgttaactc-gagtcc--tccg-aatgatactt-"}},{"entry":{}},{"entry":["\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003Olig #803 (70 bases) \u2193","(SEQ ID NO:197)"]},{"entry":{"u":"G--AAATCTCTG-GTTCTTAAGG-CTAGCGTTGC-TGTCGCGACC-CTGGTACCTA-TGT","b":"TGTCCTT-"}},{"entry":{"b":"c","u":"tttagagac-caagaattcc-gatcgcaacg-acagcgctgg-gaccatggat-aca--acaggaa-"}},{"entry":["\u2191\u2003\u2003Olig #817 (68 bases)","(SEQ ID NO:199)"]},{"entry":{}},{"entry":{"b":"CGCTCG--TCCG-GATTTCTGTC-TCGAGCCACC-ATACACTGGG-CCCTGCAAAG-CGCGCATCAT-"}},{"entry":{"u":"gcgagc","b":"aggc-ctaaagacag-agctcggtgg-tatgtgaccc-gggacgtttc-gcgcgtagta-"}},{"entry":["\u2003\u2003\u2003\u2003\u2003\u2191\u2003\u2003Olig #816 (65 bases)","(SEQ ID NO:198)"]},{"entry":{}},{"entry":["\u2003\u2003\u2003\u2003\u2003\u2193\u2003Olig #804 (67 bases)","(SEQ ID NO:210)"]},{"entry":{"b":"CCG","u":"CTATTTC-TACAATGC--TA-AAGCAGGCCT-GTGCCAGACC-TTTGTATACG-GTGGTTGCCG-"}},{"entry":{"b":"ggc--gataaag-atgttacg","u":"at-ttcgtccgga-cacggtctgg-aaacatatgc-caccaacggc-"}},{"entry":["\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2191\u2003\u2003Olig #815 (72 bases)","(SEQ ID NO:214)"]},{"entry":{}},{"entry":["\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2193\u2003Olig #805 (76 bases)","(SEQ ID NO:211)"]},{"entry":{"u":"TGCTAAGCGT","b":"AACAACTTTA-AATCG--GCCGA-AGATTGCATG-CGTACCTGCG-GTGGCGCCGC-"}},{"entry":{"u":"acgattcgca--ttgttgaaat-ttagc","b":"cggct-tctaacgtac-gcatggacgc-caccgcggcg-"}},{"entry":["\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2191\u2003Olig #814 (67 bases)","(SEQ ID NO:215)"]},{"entry":{}},{"entry":["\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2193\u2003Olig #806 (67 bases)","(SEQ ID NO:212)"]},{"entry":{"b":"TGAAGGTGAT-GATCCGGCCA-AGGCGG","u":"CCTT-CAATTCTCTG-C--AAGCTTCTG-CTACCGAGTA-"}},{"entry":{"b":"acttccacta-ctaggccggt-tccgcc--ggaa-gttaagagac-g","u":"ttcgaagac-gatggctcat-"}},{"entry":["\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2191\u2003Olig #813 (76 bases)","(SEQ ID NO:216)"]},{"entry":{}},{"entry":["\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2193\u2003Olig #807 (69 bases)","(SEQ ID NO:213)"]},{"entry":{"u":"TATTGGTTAC-GCGTGGGCCA-TGGTGGTGGT-TAT","b":"CGTTGGT-GCTACC--ATCG-GGATCAAACT-"}},{"entry":{"u":"ataaccaatg-cgcacccggt-accaccacca-ata--gcaacca-cgatgg","b":"tagc-cctagtttga-"}},{"entry":["\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2191\u2003Olig #812 (65 bases)","(SEQ ID NO:217)"]},{"entry":{}},{"entry":["\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2193\u2003Olig #808 (38 bases)","(SEQ ID NO:200)"]},{"entry":{"b":"GTTCAAGAAG-TTTACTTCGA-AGGCGTCTTA-ATGATAGGGT-TA","u":"CCAGTCTA-AGCCC--GCCTA-"}},{"entry":{"b":"caagttcttc-aaatgaagct-tccgcagaat-tactatccca-at--ggtcagat-tcggg","u":"cggat-"}},{"entry":["\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003Olig #811 (69 bases) \u2191","(SEQ ID NO:202)"]},{"entry":{}},{"entry":"\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2193\u2003filled in"},{"entry":{"u":"ATGAGCGGGC-TTTTTTTTTA"}},{"entry":"tactcgcccg-aaaaaaaaat-agct-5\u2032\u2003(The full sequence is SEQ ID NO:264)"},{"entry":["\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003Olig #810 (29 bases)","(SEQ ID NO:201)"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"119pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"147pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Overlap",{},{},{},{}]},{"entry":[{},"Sequences","Junction","Tm"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"5","align":"center","rowsep":"1"}}]},{"entry":[{},"AGGCTTACTATGAAG","802:817","42.","(SEQ ID NO:203)",{}]},{"entry":[{},{}]},{"entry":[{},"TGTCCTTCGCTCG","803:816","42.","(SEQ ID NO:204)"]},{"entry":[{},{}]},{"entry":[{},"CTATTTCTACAATGC","804:815","40.","(SEQ ID NO:205)"]},{"entry":[{},{}]},{"entry":[{},"AACAACTTTAAATCG","805:814","38.","(SEQ ID NO:206)"]},{"entry":[{},{}]},{"entry":[{},"CCTTCAATTCTCTGC","806:813","44.","(SEQ ID NO:207)"]},{"entry":[{},{}]},{"entry":[{},"CGTTGGTGCTACC","807:812","42.","(SEQ ID NO:208)"]},{"entry":[{},{}]},{"entry":[{},"CCAGTCTAAGCCC","808:811","42.","(SEQ ID NO:209)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"5","align":"center","rowsep":"1"}}]},{"entry":[{},"All these ends, as well as the SalI end, was tested for self annealing, hair-pin loop formation, and cross hybridization. No unwanted hybridization is likely. Ideally, all fragments would be the same length, but placement of overlaps to avoid restriction sites (which are usually palindromic) and to avoid cross hybridization lead to fragments varying from 65 to 76 bases, plus two fragments of 29 to 38 bases."]}]}}]}}},{"@attributes":{"id":"p-0992","num":"1297"},"tables":{"@attributes":{"id":"TABLE-US-00081","num":"00081"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"441pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 105"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Individual sequences of Oligonucleotides 801-817."},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"364pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Olig #801 (68 bases)","(SEQ ID NO:195)",{}]},{"entry":"5\u2032-ggcTTTAcAc TTTATgcTTc cggcTcgTAT AATgTgTggA ATTgTgAgcg cTcAcAATTg AgcTcAgg-3\u2032"},{"entry":{}},{"entry":["Olig #802 (67 bases)","(SEQ ID NO:196)"]},{"entry":"5\u2032-AggcTTAcTA TgAAgAAATc TcTggTTcTT AAggcTAgcg TTgcTgTcgc gAcccTggTA ccTATgT-3\u2032"},{"entry":{}},{"entry":["Olig #803 (70 bases)","(SEQ ID NO:197)"]},{"entry":"5\u2032-TgTccTTcgc TcgTccggAT TTcTgTcTcg AgCCACCATA cAcTgggccc TgcAAAgcgc gcATcATccg-3\u2032"},{"entry":{}},{"entry":["Olig #804 (67 bases)","(SEQ ID NO:210)"]},{"entry":"5\u2032-cTATTTcTAc AATgcTAAAg cAggccTgTg ccAgAccTTT gTATAcggTg gTTgccgTgc TAAgcgT-3\u2032"},{"entry":{}},{"entry":["Olig #805 (76 bases)","(SEQ ID NO:211)"]},{"entry":"5\u2032-AAcAAcTTTA AATcggccgA AgATTgcATg cgTAccTgcg gTggcgccgc TgAAggTgAT gATccggccA"},{"entry":"Aggcgg-3\u2032"},{"entry":{}},{"entry":["Olig #806 (67 bases)","(SEQ ID NO:212)"]},{"entry":"5\u2032-ccTTcAATTc TcTgcAAgcT TcTgCTACCg AgTATATTgg TTAcgcgTgg gccATggTgg TggTTAT-3\u2032"},{"entry":{}},{"entry":["Olig #807 (69 bases)","(SEQ ID NO:213)"]},{"entry":"5\u2032-cgTTggTgcT AccATcgggA TcAAAcTgTT cAAgAAgTTT AcTTcgAAgg cgTcTTAATg ATAgggTTA-3\u2032"},{"entry":{}},{"entry":["Olig #808 (38 bases)","(SEQ ID NO:200)"]},{"entry":"5\u2032-ccAgTcTAAg cccgccTAAT gAgcgggcTT TTTTTTTA-3\u2032"},{"entry":{}},{"entry":["Olig #810 (29 bases)","(SEQ ID NO:201)"]},{"entry":"5\u2032-TcgATAAAAA AAAAgcccgc TcATTAggc-3\u2032"},{"entry":{}},{"entry":["Olig #811 (69 bases)","(SEQ ID NO:202)"]},{"entry":"5\u2032-gggcTTAgAc TggTAAcccT ATcATTAAgA cgccTTcgAA gTAAAcTTcT TgAAcAgTTT gATcccgAT-3\u2032"},{"entry":{}},{"entry":["Olig #812 (65 bases)","(SEQ ID NO:217)"]},{"entry":"5\u2032-ggTAgcAccA AcgATAAccA ccAccATggc ccAcgcgTAA ccAATATAcT cggTAgcAgA AgcTT-3\u2032"},{"entry":{}},{"entry":["Olig #813 (76 bases)","(SEQ ID NO:216)"]},{"entry":"5\u2032-gcAgAgAATT gAAggccgcc TTggccggAT cATcAccTTc AgcggcgccA ccgcAggTAc gcATgcAATc"},{"entry":"TTcggc-3\u2032"},{"entry":{}},{"entry":["Olig #814 (67 bases)","(SEQ ID NO:215)"]},{"entry":"5\u2032-cgATTTAAAg TTgTTAcgcT TAgcAcggcA AccAccgTAT AcAAAggTcT ggcAcAggcc TgcTTTA-3\u2032"},{"entry":{}},{"entry":["Olig #815 (72 bases)","(SEQ ID NO:214)"]},{"entry":"5\u2032-gcATTgTAgA AATAgcggAT gATgcgcgcT TTgcAgggcc cAgTgTATgg TggcTcgAgA cAgAAATccg gA-3\u2032"},{"entry":{}},{"entry":["Olig #816 (65 bases)","(SEQ ID NO:198)"]},{"entry":"5\u2032-cgAgcgAAgg AcAAcATAgg TAccAgggTc gcgAcAgcAA cgcTAgccTT AAgAAccAgA gATTT-3\u2032"},{"entry":{}},{"entry":["Olig #817 (68 bases)","(SEQ ID NO:199)"]},{"entry":"5\u2032-cTTcATAgTA AgccTccTgA gcTcAATTgT gAgcgcTcAc AATTccAcAc ATTATAcgAg ccggAAgc-3\u2032"},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0993","num":"1298"},"tables":{"@attributes":{"id":"TABLE-US-00082","num":"00082"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"406pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 106"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Signal Peptides"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"287pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["PhoA","M \u2003q s t i a l a l l p l l f t p v t \u2003A \/\u2003T . . . (17)","(SEQ ID NO:218)",{}]},{"entry":{}},{"entry":["MalE","M \u2003I \u2003T G A \u2003i l a l s a l t t m m f s a s a l a \/\u2003I . . . (18)","(SEQ ID NO:219)"]},{"entry":{}},{"entry":["OmpF","M M \u2003\u2003n i l a v i v p a l l v a g t a n a \/a \u2003. . . (19)","(SEQ ID NO:220)"]},{"entry":{}},{"entry":["Bla","M S I Q H F \u2003v a l i p f f a a f c l p v f a \/h p . . . (>18)","(SEQ ID NO:221)"]},{"entry":{}},{"entry":["LamB","M M I T L \u2003\u2003l p l a v a v a a g v m s a q a m a \/v \u2003. . . (19)","(SEQ ID NO:222)"]},{"entry":{}},{"entry":["Lpp","M \u2003A T \u2003l v l g a v i l g s t l l a g \/c s . . . (>17)","(SEQ ID NO:223)"]},{"entry":{}},{"entry":["gpIII","M \u2003\u2003l l f a i p l v v p f y s h s \/a \u2003T V \u2003. . . (16)","(SEQ ID NO:224)"]},{"entry":{}},{"entry":["gpIII-BPTI","M \u2003\u2003l l f a i p l v v p f y s g a \/\u2003P \u2003. . . (15)","(SEQ ID NO:225)"]},{"entry":{}},{"entry":["gpVIII","M \u2003\u2003S L V L \u2003a s v a v a t l v p m l s f a \/a \u2003G \u2003\u2003. . . (16)","(SEQ ID NO:226)"]},{"entry":{}},{"entry":["gpVIII-BPTI","M \u2003\u2003S L V L \u2003a s v a v a t l v p m l s f a \/\u2003P \u2003. . . (15)","(SEQ ID NO:227)"]},{"entry":{}},{"entry":["gpVIII\u2032","M \u2003\u2003s l V l l a s v a v a t l v p m l s f a \/a \u2003G \u2003\u2003. . . (21)","(SEQ ID NO:228)"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0994","num":"1299"},"tables":{"@attributes":{"id":"TABLE-US-00083","num":"00083"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 107"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"In vitro transcription\/translation"},{"entry":"analysis of vector-encoded"},{"entry":"signal::BPTI::mature VIII protein species"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"31 kd species","14.5 kd species"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"No DNA (control)","\u2212","\u2212"]},{"entry":[{},"pGEN-3Zf(\u2014)","+","\u2212"]},{"entry":[{},"pGEM-M316","+","\u2212"]},{"entry":[{},"pGEM-MB20","+","+"]},{"entry":[{},"pGEM-MB26","+","+"]},{"entry":[{},"pGEM-MB42","+","+"]},{"entry":[{},"pGEM-MB46","ND","ND"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"Notes:"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"left","id":"FOO-00091"},"sup":"a"}]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"left","id":"FOO-00092"},"sup":"b"}]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"left","id":"FOO-00093"},"sup":"c"}]},{"entry":[{},"ND for Not Determined."]}]}}]}}},{"@attributes":{"id":"p-0995","num":"1300"},"tables":{"@attributes":{"id":"TABLE-US-00084","num":"00084"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 108"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Western analysisof in vivo"},{"entry":"expressed"},{"entry":"signal::BPTI::mature VIII protein species"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"signal","14.5 kd species","12 kd species"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"A) expression in strain XL1-Blue"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"63pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"pGEM-3Zf(\u2014)","\u2014","\u2212","\u2212"]},{"entry":[{},"pGEM-MB16","VIII","\u2212","\u2212"]},{"entry":[{},"pGEM-MB20","VIII","++","\u2212"]},{"entry":[{},"pGEM-MB26","VIII","+++","+\/\u2212"]},{"entry":[{},"pGEM-MB42","phoA","++","+"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"B) expression in strain SEF\u2032"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"63pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"pGEM-MB42","phoA","+\/\u2212","+++"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Notes:"},{"entry":{"sup":"a"}},{"entry":"peroxidase-conjugated goat anti-rabbit IgG antibody."},{"entry":{"sup":"b"}},{"entry":"constructs."},{"entry":{"sup":"c"}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{"sup":"d"},"\u2212"]},{"entry":["weakly present","+\/\u2212"]},{"entry":["present","+"]},{"entry":["strong presence","++"]},{"entry":["very strong presence","+++"]}]}}]}}},{"@attributes":{"id":"p-0996","num":"1301"},"tables":{"@attributes":{"id":"TABLE-US-00085","num":"00085"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 109"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"M13 gene III (SEQ ID NO:299)"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1579\u2003\u2003\u2003\u2003\u2003\u20035\u2032-GT GAAAAAATTA TTATTCGCAA TTCCTTTAGT",{}]},{"entry":{}},{"entry":"1611 TGTTCCTTTC TATTCTCACT CCGCTGAAAC TGTTGAAAGT"},{"entry":{}},{"entry":"1651 TGTTTAGCAA AACCCCATAC AGAAAATTCA TTTACTAACG"},{"entry":{}},{"entry":"1691 TCTGGAAAGA CGACAAAACT TTAGATCGTT ACGCTAACTA"},{"entry":{}},{"entry":"1731 TGAGGGTTGT CTGTGGAATG CTACAGGCGT TGTAGTTTGT"},{"entry":{}},{"entry":"1771 ACTGGTGACG AAACTCAGTG TTACGGTACA TGGGTTCCTA"},{"entry":{}},{"entry":"1811 TTGGGCTTGC TATCCCTGAA AATGAGGGTG GTGGCTCTGA"},{"entry":{}},{"entry":"1851 GGGTGGCGGT TCTGAGGGTG GCGGTTCTGA GGGTGGCGGT"},{"entry":{}},{"entry":"1891 ACTAAACCTC CTGAGTACGG TGATACACCT ATTCCGGGCT"},{"entry":{}},{"entry":"1931 ATACTTATAT CAACCCTCTC GACGGCACTT ATCCGCCTGG"},{"entry":{}},{"entry":"1971 TACTGAGCAA AACCCCGCTA ATCCTAATCC TTCTCTTGAG"},{"entry":{}},{"entry":"2011 GAGTCTCAGC CTCTTAATAC TTTCATGTTT CAGAATAATA"},{"entry":{}},{"entry":"2051 GGTTCCGAAA TAGGCAGGGG GCATTAACTG TTTATACGGG"},{"entry":{}},{"entry":"2091 CACTGTTACT CAAGGCACTG ACCCCGTTAA AACTTATTAC"},{"entry":{}},{"entry":"2131 CAGTACACTC CTGTATCATC AAAAGCCATG TATGACGCTT"},{"entry":{}},{"entry":"2171 ACTGGAACGG TAAATTCAGA GACTGCGCTT TCCATTCTGG"},{"entry":{}},{"entry":"2211 CTTTAATGAG GATCCATTCG TTTGTGAATA TCAAGGCCAA"},{"entry":{}},{"entry":"2251 TCGTCTGACC TGCCTCAACC TCCTGTCAAT GCTGGCGGCG"},{"entry":{}},{"entry":"2291 GCTCTGGTGG TGGTTCTGGT GGCGGCTCTG AGGGTGGTGG"},{"entry":{}},{"entry":"2331 CTCTGAGGGT GGCGGTTCTG AGGGTGGCGG CTCTGAGGGA"},{"entry":{}},{"entry":"2371 GGCGGTTCCG GTGGTGGCTC TGGTTCCGGT GATTTTGATT"},{"entry":{}},{"entry":"2411 ATGAAAAGAT GGCAAACGCT AATAAGGGGG CTATGACCGA"},{"entry":{}},{"entry":"2451 AAATGCCGAT GAAAACGCGC TACAGTCTGA CGCTAAAGGC"},{"entry":{}},{"entry":"2491 AAACTTGATT CTGTCGCTAC TGATTACGGT GCTGCTATCG"},{"entry":{}},{"entry":"2531 ATGGTTTCAT TGGTGACGTT TCCGGCCTTG CTAATGGTAA"},{"entry":{}},{"entry":"2571 TGGTGCTACT GGTGATTTTG CTGGCTCTAA TTCCCAAATG"},{"entry":{}},{"entry":"2611 GCTCAAGTCG GTGACGGTGA TAATTCACCT TTAATGAATA"},{"entry":{}},{"entry":"2651 ATTTCCGTCA ATATTTACCT TCCCTCCCTC AATCGGTTGA"},{"entry":{}},{"entry":"2691 ATGTCGCCCT TTTGTCTTTA GCGCTGGTAA ACCATATGAA"},{"entry":{}},{"entry":"2731 TTTTCTATTG ATTGTGACAA AATAAACTTA TTCCGTGGTG"},{"entry":{}},{"entry":"2771 TCTTTGCGTT TCTTTTATAT GTTGCCACCT TTATGTATGT"},{"entry":{}},{"entry":"2811 ATTTTCTACG TTTGCTAACA TACTGCGTAA TAAGGAGTCT"},{"entry":{}},{"entry":"2851 TAATCATGCC AGTTCTTTTG GGTATTCCGT"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0997","num":"1302"},"tables":{"@attributes":{"id":"TABLE-US-00086","num":"00086"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 110"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Introduction of I into gene III"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["DNA sequence: SEQ ID NO:230",{}]},{"entry":"Protein sequence: SEQ ID NO:231"},{"entry":{}},{"entry":"A) Wild-type III, portion encoding the signal peptide"},{"entry":{}},{"entry":"\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003M\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003L\u2003\u2003\u2003L\u2003\u2003\u2003F\u2003\u2003\u2003A\u2003\u2003\u2003I\u2003\u2003\u2003P\u2003\u2003\u2003L"},{"entry":"\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u20031\u2003\u2003\u20032\u2003\u2003\u20033\u2003\u2003\u20034\u2003\u2003\u20035\u2003\u2003\u20036\u2003\u2003\u20037\u2003\u2003\u20038\u2003\u2003\u20039\u2003\u200310"},{"entry":"1579\u2003\u2003\u2003\u2003\u2003\u2003\u20035\u2032-GTG AAA AAA TTA TTA TTC GCA ATT CCT TTA"},{"entry":{}},{"entry":"\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\/ Cleavage site"},{"entry":"\u2003\u2003\u2003\u2003\u2003\u2003\u2003V\u2003\u2003\u2003V\u2003\u2003\u2003P\u2003\u2003\u2003F\u2003\u2003\u2003Y\u2003\u2003\u2003S\u2003\u2003\u2003H\u2003\u2003\u2003S\u2003\u2193\u2003A\u2003\u2003\u2003E\u2003\u2003\u2003T\u2003\u2003\u2003V"},{"entry":"\u2003\u2003\u2003\u2003\u2003\u200311\u2003\u200312\u2003\u200313\u2003\u200314\u2003\u200315\u2003\u200316\u2003\u200317\u2003\u200318\u2003\u200319\u2003\u200320\u2003\u200321\u2003\u200322"},{"entry":"1609\u2003\u2003GTT GTT CCT TTC TAT TCT CAC TCC GCT GAA ACT GTT-3\u2032"},{"entry":{}},{"entry":"DNA sequence: SEQ ID NO:232"},{"entry":"Protein sequence: SEQ ID NO:233"},{"entry":{}},{"entry":"B) III, portion encoding the signal peptide with I site"},{"entry":{}},{"entry":"\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003m\u2003\u2003\u2003k\u2003\u2003\u2003k\u2003\u2003\u2003l\u2003\u2003\u2003l\u2003\u2003\u2003f\u2003\u2003\u2003a\u2003\u2003\u2003I\u2003\u2003\u2003p\u2003\u2003\u2003l"},{"entry":"\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u20031\u2003\u2003\u20032\u2003\u2003\u20033\u2003\u2003\u20034\u2003\u2003\u20035\u2003\u2003\u20036\u2003\u2193\u20037\u2003\u2003\u20038\u2003\u2003\u20039\u2003\u200310"},{"entry":"1579\u2003\u2003\u2003\u2003\u2003\u2003\u20035\u2032-gtg aaa aaa tta tta ttc gca att cct tta"},{"entry":{}},{"entry":"\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\/ cleavage site"},{"entry":"\u2003\u2003\u2003\u2003\u2003\u2003\u2003v\u2003\u2003\u2003v\u2003\u2003\u2003p\u2003\u2003\u2003f\u2003\u2003\u2003y\u2003\u2003\u2003s\u2003\u2003\u2003G\u2003\u2003\u2003A\u2003\u2003\u2003a\u2003\u2003\u2003e\u2003\u2003\u2003t\u2003\u2003\u2003v"},{"entry":"\u2003\u2003\u2003\u2003\u2003\u200311\u2003\u200312\u2003\u200313\u2003\u200314\u2003\u200315\u2003\u200316\u2003\u200317\u2003\u200318\u2003\u200319\u2003\u200320\u2003\u200321\u2003\u200322"},{"entry":"1609\u2003\u2003gtt gtt cct ttc tat tct GGc Gcc gct gaa act gtt-3\u2032"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0998","num":"1303"},"tables":{"@attributes":{"id":"TABLE-US-00087","num":"00087"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 111"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":{"u":["IIIsp","bpti","matureIII"]}},{"entry":"DNA sequence: SEQ ID NO:234"},{"entry":"Protein sequence: SEQ ID NO:235"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00221","num":"00221"},"img":{"@attributes":{"id":"EMI-C00221","he":"17.86mm","wi":"62.99mm","file":"US07413537-20080819-C00221.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00222","num":"00222"},"img":{"@attributes":{"id":"EMI-C00222","he":"9.99mm","wi":"46.06mm","file":"US07413537-20080819-C00222.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00223","num":"00223"},"img":{"@attributes":{"id":"EMI-C00223","he":"12.36mm","wi":"62.31mm","file":"US07413537-20080819-C00223.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00224","num":"00224"},"img":{"@attributes":{"id":"EMI-C00224","he":"17.53mm","wi":"62.31mm","file":"US07413537-20080819-C00224.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00225","num":"00225"},"img":{"@attributes":{"id":"EMI-C00225","he":"7.54mm","wi":"62.31mm","file":"US07413537-20080819-C00225.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00226","num":"00226"},"img":{"@attributes":{"id":"EMI-C00226","he":"12.53mm","wi":"63.84mm","file":"US07413537-20080819-C00226.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00227","num":"00227"},"img":{"@attributes":{"id":"EMI-C00227","he":"10.33mm","wi":"56.39mm","file":"US07413537-20080819-C00227.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00228","num":"00228"},"img":{"@attributes":{"id":"EMI-C00228","he":"10.33mm","wi":"62.31mm","file":"US07413537-20080819-C00228.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00229","num":"00229"},"img":{"@attributes":{"id":"EMI-C00229","he":"18.12mm","wi":"38.61mm","file":"US07413537-20080819-C00229.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00230","num":"00230"},"img":{"@attributes":{"id":"EMI-C00230","he":"7.54mm","wi":"50.88mm","file":"US07413537-20080819-C00230.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":"1651 TGTTTAGCAA AACCCCATAC AGAAAATTCA TTTACTAACG"},{"entry":{}},{"entry":"1691 TCTGGAAAGA CGACAAAACT TTAGATCGTT ACGCTAACTA"},{"entry":{}},{"entry":"1731 TGAGGGTTGT CTGTGGAATG CTACAGGCGT TGTAGTTTGT"},{"entry":{}},{"entry":"1771 ACTGGTGACG AAACTCAGTG TTACGGTACA TGGGTTCCTA"},{"entry":{}},{"entry":"1811 TTGGGCTTGC TATCCCTGAA AATGAGGGTG GTGGCTCTGA"},{"entry":{}},{"entry":"1851 GGGTGGCGGT TCTGAGGGTG GCGGTTCTGA GGGTGGCGGT"},{"entry":{}},{"entry":"1891 ACTAAACCTC CTGAGTACGG TGATACACCT ATTCCGGGCT"},{"entry":{}},{"entry":"1931 ATACTTATAT CAACCCTCTC GACGGCACTT ATCCGCCTGG"},{"entry":{}},{"entry":"1971 TACTGAGCAA AACCCCGCTA ATCCTAATCC TTCTCTTGAG"},{"entry":{}},{"entry":"2011 GAGTCTCAGC CTCTTAATAC TTTCATGTTT CAGAATAATA"},{"entry":{}},{"entry":"2051 GGTTCCGAAA TAGGCAGGGG GCATTAACTG TTTATACGGG"},{"entry":{}},{"entry":"2091 CACTGTTACT CAAGGCACTG ACCCCGTTAA AACTTATTAC"},{"entry":{}},{"entry":"2131 CAGTACACTC CTGTATCATC AAAAGCCATG TATGACGCTT"},{"entry":{}},{"entry":"2171 ACTGGAACGG TAAATTCAGA GACTGCGCTT TCCATTCTGG"},{"entry":{}},{"entry":"2211 CTTTAATGAG GATCCATTCG TTTGTGAATA TCAAGGCCAA"},{"entry":{}},{"entry":"2251 TCGTCTGACC TGCCTCAACC TCCTGTCAAT GCTGGCGGCG"},{"entry":{}},{"entry":"2291 GCTCTGGTGG TGGTTCTGGT GGCGGCTCTG AGGGTGGTGG"},{"entry":{}},{"entry":"2331 CTCTGAGGGT GGCGGTTCTG AGGGTGGCGG CTCTGAGGGA"},{"entry":{}},{"entry":"2371 GGCGGTTCCG GTGGTGGCTC TGGTTCCGGT GATTTTGATT"},{"entry":{}},{"entry":"2411 ATGAAAAGAT GGCAAACGCT AATAAGGGGG CTATGACCGA"},{"entry":{}},{"entry":"2451 AAATGCCGAT GAAAACGCGC TACAGTCTGA CGCTAAAGGC"},{"entry":{}},{"entry":"2491 AAACTTGATT CTGTCGCTAC TGATTACGGT GCTGCTATCG"},{"entry":{}},{"entry":"2531 ATGGTTTCAT TGGTGACGTT TCCGGCCTTG CTAATGGTAA"},{"entry":{}},{"entry":"2571 TGGTGCTACT GGTGATTTTG CTGGCTCTAA TTCCCAAATG"},{"entry":{}},{"entry":"2611 GCTCAAGTCG GTGACGGTGA TAATTCACCT TTAATGAATA"},{"entry":{}},{"entry":"2651 ATTTCCGTCA ATATTTACCT TCCCTCCCTC AATCGGTTGA"},{"entry":{}},{"entry":"2691 ATGTCGCCCT TTTGTCTTTA GCGCTGGTAA ACCATATGAA"},{"entry":{}},{"entry":"2731 TTTTCTATTG ATTGTGACAA AATAAACTTA TTCCGTGGTG"},{"entry":{}},{"entry":"2771 TCTTTGCGTT TCTTTTATAT GTTGCCACCT TTATGTATGT"},{"entry":{}},{"entry":"2811 ATTTTCTACG TTTGCTAACA TACTGCGTAA TAAGGAGTCT"},{"entry":{}},{"entry":"2851 TCATGCC AGTTCTTTTG GGTATTCCGT"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0999","num":"1304"},"tables":{"@attributes":{"id":"TABLE-US-00088","num":"00088"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 112"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Annotated Sequence of"},{"entry":"Ptac::RBS(GGAGGAAATAAA) (SEQ ID NO:241)::"},{"entry":{"u":["VIII-signal","mature-bpti","mature-VIII-coat-protein"]}},{"entry":"gene (SEQ ID NO:236)"},{"entry":"Protein sequence: SEQ ID NO:153"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00231","num":"00231"},"img":{"@attributes":{"id":"EMI-C00231","he":"7.54mm","wi":"33.95mm","file":"US07413537-20080819-C00231.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00232","num":"00232"},"img":{"@attributes":{"id":"EMI-C00232","he":"5.00mm","wi":"62.31mm","file":"US07413537-20080819-C00232.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00233","num":"00233"},"img":{"@attributes":{"id":"EMI-C00233","he":"5.08mm","wi":"31.41mm","file":"US07413537-20080819-C00233.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00234","num":"00234"},"img":{"@attributes":{"id":"EMI-C00234","he":"6.01mm","wi":"52.24mm","file":"US07413537-20080819-C00234.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00235","num":"00235"},"img":{"@attributes":{"id":"EMI-C00235","he":"10.08mm","wi":"62.31mm","file":"US07413537-20080819-C00235.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00236","num":"00236"},"img":{"@attributes":{"id":"EMI-C00236","he":"10.41mm","wi":"62.31mm","file":"US07413537-20080819-C00236.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00237","num":"00237"},"img":{"@attributes":{"id":"EMI-C00237","he":"12.87mm","wi":"62.31mm","file":"US07413537-20080819-C00237.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00238","num":"00238"},"img":{"@attributes":{"id":"EMI-C00238","he":"17.61mm","wi":"62.31mm","file":"US07413537-20080819-C00238.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00239","num":"00239"},"img":{"@attributes":{"id":"EMI-C00239","he":"7.54mm","wi":"62.31mm","file":"US07413537-20080819-C00239.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00240","num":"00240"},"img":{"@attributes":{"id":"EMI-C00240","he":"12.78mm","wi":"63.75mm","file":"US07413537-20080819-C00240.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00241","num":"00241"},"img":{"@attributes":{"id":"EMI-C00241","he":"10.41mm","wi":"56.39mm","file":"US07413537-20080819-C00241.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00242","num":"00242"},"img":{"@attributes":{"id":"EMI-C00242","he":"10.41mm","wi":"62.31mm","file":"US07413537-20080819-C00242.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00243","num":"00243"},"img":{"@attributes":{"id":"EMI-C00243","he":"23.28mm","wi":"74.17mm","file":"US07413537-20080819-C00243.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00244","num":"00244"},"img":{"@attributes":{"id":"EMI-C00244","he":"10.41mm","wi":"56.39mm","file":"US07413537-20080819-C00244.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00245","num":"00245"},"img":{"@attributes":{"id":"EMI-C00245","he":"7.54mm","wi":"44.53mm","file":"US07413537-20080819-C00245.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00246","num":"00246"},"img":{"@attributes":{"id":"EMI-C00246","he":"12.78mm","wi":"56.39mm","file":"US07413537-20080819-C00246.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00247","num":"00247"},"img":{"@attributes":{"id":"EMI-C00247","he":"7.54mm","wi":"26.75mm","file":"US07413537-20080819-C00247.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00248","num":"00248"},"img":{"@attributes":{"id":"EMI-C00248","he":"10.24mm","wi":"62.31mm","file":"US07413537-20080819-C00248.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00249","num":"00249"},"img":{"@attributes":{"id":"EMI-C00249","he":"9.99mm","wi":"43.01mm","file":"US07413537-20080819-C00249.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00250","num":"00250"},"img":{"@attributes":{"id":"EMI-C00250","he":"5.16mm","wi":"60.03mm","file":"US07413537-20080819-C00250.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00251","num":"00251"},"img":{"@attributes":{"id":"EMI-C00251","he":"5.33mm","wi":"53.00mm","file":"US07413537-20080819-C00251.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-1000","num":"1305"},"tables":{"@attributes":{"id":"TABLE-US-00089","num":"00089"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 113"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Annotated Sequence of pGEM-MB42 comprising"},{"entry":"Ptac::RBS(GGAGGAAATAAA) (SEQ ID NO:241)::"},{"entry":{"u":["phoA-signal","mature-bpti","mature-VIII-coat-protein"]}},{"entry":"DNA sequence: SEQ ID NO:242"},{"entry":"Protein sequence: SEQ ID NO:240"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00252","num":"00252"},"img":{"@attributes":{"id":"EMI-C00252","he":"7.79mm","wi":"33.27mm","file":"US07413537-20080819-C00252.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00253","num":"00253"},"img":{"@attributes":{"id":"EMI-C00253","he":"5.00mm","wi":"62.31mm","file":"US07413537-20080819-C00253.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00254","num":"00254"},"img":{"@attributes":{"id":"EMI-C00254","he":"5.00mm","wi":"31.41mm","file":"US07413537-20080819-C00254.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00255","num":"00255"},"img":{"@attributes":{"id":"EMI-C00255","he":"10.33mm","wi":"72.64mm","file":"US07413537-20080819-C00255.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00256","num":"00256"},"img":{"@attributes":{"id":"EMI-C00256","he":"10.16mm","wi":"73.74mm","file":"US07413537-20080819-C00256.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"(There are no residues 20-23.)"]},{"entry":[{},{}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00257","num":"00257"},"img":{"@attributes":{"id":"EMI-C00257","he":"15.24mm","wi":"63.84mm","file":"US07413537-20080819-C00257.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00258","num":"00258"},"img":{"@attributes":{"id":"EMI-C00258","he":"17.61mm","wi":"64.01mm","file":"US07413537-20080819-C00258.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00259","num":"00259"},"img":{"@attributes":{"id":"EMI-C00259","he":"7.54mm","wi":"63.92mm","file":"US07413537-20080819-C00259.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00260","num":"00260"},"img":{"@attributes":{"id":"EMI-C00260","he":"12.62mm","wi":"63.84mm","file":"US07413537-20080819-C00260.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00261","num":"00261"},"img":{"@attributes":{"id":"EMI-C00261","he":"10.16mm","wi":"58.17mm","file":"US07413537-20080819-C00261.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00262","num":"00262"},"img":{"@attributes":{"id":"EMI-C00262","he":"12.62mm","wi":"64.01mm","file":"US07413537-20080819-C00262.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00263","num":"00263"},"img":{"@attributes":{"id":"EMI-C00263","he":"23.11mm","wi":"75.95mm","file":"US07413537-20080819-C00263.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00264","num":"00264"},"img":{"@attributes":{"id":"EMI-C00264","he":"10.41mm","wi":"58.17mm","file":"US07413537-20080819-C00264.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00265","num":"00265"},"img":{"@attributes":{"id":"EMI-C00265","he":"7.62mm","wi":"47.75mm","file":"US07413537-20080819-C00265.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00266","num":"00266"},"img":{"@attributes":{"id":"EMI-C00266","he":"12.62mm","wi":"59.01mm","file":"US07413537-20080819-C00266.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00267","num":"00267"},"img":{"@attributes":{"id":"EMI-C00267","he":"7.62mm","wi":"33.78mm","file":"US07413537-20080819-C00267.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00268","num":"00268"},"img":{"@attributes":{"id":"EMI-C00268","he":"10.16mm","wi":"62.31mm","file":"US07413537-20080819-C00268.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00269","num":"00269"},"img":{"@attributes":{"id":"EMI-C00269","he":"10.08mm","wi":"40.13mm","file":"US07413537-20080819-C00269.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00270","num":"00270"},"img":{"@attributes":{"id":"EMI-C00270","he":"5.16mm","wi":"60.03mm","file":"US07413537-20080819-C00270.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00271","num":"00271"},"img":{"@attributes":{"id":"EMI-C00271","he":"5.08mm","wi":"39.37mm","file":"US07413537-20080819-C00271.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-1001","num":"1306"},"tables":{"@attributes":{"id":"TABLE-US-00090","num":"00090"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 114"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Neutralization of Phage Titer Using"},{"entry":"Agarose-immobilized Anhydro-Trypsin"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"119pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Percent Residual Titer"]},{"entry":[{},"As a Function of Time (hours)"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Phage Type","Addition","1","2","4"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"5","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"MK-BPTI","\u20025 \u03bcl IS","99","104","105"]},{"entry":[{},{},"\u20022 \u03bcl IAT","82","71","51"]},{"entry":[{},{},"\u20025 \u03bcl IAT","57","40","27"]},{"entry":[{},{},"10 \u03bcl IAT","40","30","24"]},{"entry":[{},"MK","\u20025 \u03bcl IS","106","96","98"]},{"entry":[{},{},"\u20022 \u03bcl IAT","97","103","95"]},{"entry":[{},{},"\u20025 \u03bcl IAT","110","111","96"]},{"entry":[{},{},"10 \u03bcl IAT","99","93","106"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"5","align":"center","rowsep":"1"}}]},{"entry":[{},"Legend:"]},{"entry":[{},"IS = Immobilized streptavidin"]},{"entry":[{},"IAT = Immobilized anhydro-trypsin"]}]}}]}}},{"@attributes":{"id":"p-1002","num":"1307"},"tables":{"@attributes":{"id":"TABLE-US-00091","num":"00091"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 115"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Affinity Selection of MK-BPTI Phage"},{"entry":"on Immobilized Anhydro-Trypsin"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"119pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},"Percent of Total Phage"]},{"entry":[{},"Phage Type","Addition","Recovered in Elution Buffer"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"MK-BPTI","\u20025 \u03bcl IS",{"sup":["\u2002","a"]}]},{"entry":[{},{},"\u20022 \u03bcl IAT","\u2009\u2009\u2009\u20095"]},{"entry":[{},{},"\u20025 \u03bcl IAT","\u2009\u200920"]},{"entry":[{},{},"10 \u03bcl IAT","\u2009\u200950"]},{"entry":[{},"MK","\u20025 \u03bcl IS",{"sup":["\u2002","a"]}]},{"entry":[{},{},"\u20022 \u03bcl IAT","<<1"]},{"entry":[{},{},"\u20025 \u03bcl IAT","<<1"]},{"entry":[{},{},"10 \u03bcl IAT","<<1"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"Legend:"]},{"entry":[{},"IS = Immobilized streptavidin"]},{"entry":[{},"IAT = Immobilized anhydro-trypsin"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"left","id":"FOO-00101"},"sup":"a"}]}]}}]}}},{"@attributes":{"id":"p-1003","num":"1308"},"tables":{"@attributes":{"id":"TABLE-US-00092","num":"00092"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"266pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 116"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"translation of Signal-III::bpti::mature-III"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"266pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["DNA sequence: SEQ ID NO:262",{}]},{"entry":"Protein sequence: SEQ ID NO:243"},{"entry":{}},{"entry":"\u20031\u2003\u2003\u20032\u2003\u2003\u20033\u2003\u2003\u20034\u2003\u2003\u20035\u2003\u2003\u20036\u2003\u2003\u20037\u2003\u2003\u20038\u2003\u2003\u20039\u2003\u2003\u200310\u2003\u200311\u2003\u200312\u2003\u200313\u2003\u200314\u2003\u200315"},{"entry":"fM\u2003\u2003\u2003K\u2003\u2003\u2003K\u2003\u2003\u2003L\u2003\u2003\u2003L\u2003\u2003\u2003F\u2003\u2003\u2003A\u2003\u2003\u2003I\u2003\u2003\u2003P\u2003\u2003\u2003L\u2003\u2003\u2003V\u2003\u2003\u2003V\u2003\u2003\u2003P\u2003\u2003\u2003F\u2003\u2003\u2003Y"},{"entry":"GTG AAA AAA TTA TTA TTC GCA ATT CCT TTA GTT GTT CCT TTC TAT"},{"entry":"|<-------\u2003gene III signal peptide\u2003-------------------------"},{"entry":{}},{"entry":"\u200316\u2003\u200317\u2003\u200318\u2003\u200319\u2003\u200320\u2003\u200321\u2003\u200322\u2003\u200323\u2003\u200324\u2003\u200325\u2003\u200326\u2003\u200327\u2003\u200328\u2003\u200329\u2003\u200330"},{"entry":"\u2003S\u2003\u2003\u2003G\u2003\u2003\u2003A\u2003\u2003\u2003R\u2003\u2003\u2003P\u2003\u2003\u2003D\u2003\u2003\u2003F\u2003\u2003\u2003C\u2003\u2003\u2003L\u2003\u2003\u2003E\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003Y\u2003\u2003\u2003T\u2003\u2003\u2003G"},{"entry":"TCT GGC GCC "},{"entry":"----------->|<-----\u2003BPTI insertion\u2003------------------------"},{"entry":{}},{"entry":"\u200331\u2003\u200332\u2003\u200333\u2003\u200334\u2003\u200335\u2003\u200336\u2003\u200337\u2003\u200338\u2003\u200339\u2003\u200340\u2003\u200341\u2003\u200342\u2003\u200343\u2003\u200344\u2003\u200345"},{"entry":"\u2003P\u2003\u2003\u2003C\u2003\u2003\u2003K\u2003\u2003\u2003A\u2003\u2003\u2003R\u2003\u2003\u2003I\u2003\u2003\u2003I\u2003\u2003\u2003R\u2003\u2003\u2003Y\u2003\u2003\u2003F\u2003\u2003\u2003Y\u2003\u2003\u2003N\u2003\u2003\u2003A\u2003\u2003\u2003K\u2003\u2003\u2003A"},{"entry":{"u":"ccc\u2003tgc\u2003aaa\u2003gcg\u2003cgc\u2003atc\u2003atc\u2003cgc\u2003tat\u2003ttc\u2003tac\u2003aat\u2003gct\u2003aaa\u2003gca"}},{"entry":{}},{"entry":"\u200346\u2003\u200347\u2003\u200348\u2003\u200349\u2003\u200350\u2003\u200351\u2003\u200352\u2003\u200353\u2003\u200354\u2003\u200355\u2003\u200356\u2003\u200357\u2003\u200358\u2003\u200359\u2003\u200360"},{"entry":"\u2003G\u2003\u2003\u2003L\u2003\u2003\u2003C\u2003\u2003\u2003Q\u2003\u2003\u2003T\u2003\u2003\u2003F\u2003\u2003\u2003V\u2003\u2003\u2003Y\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003C\u2003\u2003\u2003R\u2003\u2003\u2003A\u2003\u2003\u2003K\u2003\u2003\u2003R"},{"entry":{"u":"ggc\u2003ctg\u2003tgc\u2003cag\u2003acc\u2003ttt\u2003gta\u2003tac\u2003ggt\u2003ggt\u2003tgc\u2003cgt\u2003gct\u2003aag\u2003cgt"}},{"entry":{}},{"entry":"\u200361\u2003\u200362\u2003\u200363\u2003\u200364\u2003\u200365\u2003\u200366\u2003\u200367\u2003\u200368\u2003\u200369\u2003\u200370\u2003\u200371\u2003\u200372\u2003\u200373\u2003\u200374\u2003\u200375"},{"entry":"\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003F\u2003\u2003\u2003K\u2003\u2003\u2003S\u2003\u2003\u2003A\u2003\u2003\u2003E\u2003\u2003\u2003D\u2003\u2003\u2003C\u2003\u2003\u2003M\u2003\u2003\u2003R\u2003\u2003\u2003T\u2003\u2003\u2003C\u2003\u2003\u2003G\u2003\u2003\u2003G"},{"entry":{"u":"aac\u2003aac\u2003ttt\u2003aaa\u2003tcg\u2003gcc\u2003gaa\u2003gat\u2003tgc\u2003atg\u2003cgt\u2003acc\u2003tgc\u2003ggt\u2003ggc"}},{"entry":{}},{"entry":"\u200376\u2003\u200377\u2003\u200378\u2003\u200379\u2003\u200380\u2003\u200381\u2003\u200382\u2003\u200383\u2003\u200384\u2003\u200385\u2003\u200386\u2003\u200387\u2003\u200388\u2003\u200389\u2003\u200390"},{"entry":"\u2003A\u2003\u2003\u2003G\u2003\u2003\u2003A\u2003\u2003\u2003A\u2003\u2003\u2003E\u2003\u2003\u2003T\u2003\u2003\u2003V\u2003\u2003\u2003E\u2003\u2003\u2003S\u2003\u2003\u2003C\u2003\u2003\u2003L\u2003\u2003\u2003A\u2003\u2003\u2003K\u2003\u2003\u2003P\u2003\u2003\u2003H"},{"entry":{"u":"gcc"}},{"entry":"\u2003\u2003\u2003\u2003|<-------\u2003mature gene III protein\u2003---------------------"},{"entry":{}},{"entry":"\u200391\u2003\u200392\u2003\u200393\u2003\u200394\u2003\u200395\u2003\u200396\u2003\u200397\u2003\u200398\u2003\u200399 100 101 102 103 104 105"},{"entry":"\u2003T\u2003\u2003\u2003E\u2003\u2003\u2003N\u2003\u2003\u2003S\u2003\u2003\u2003F\u2003\u2003\u2003T\u2003\u2003\u2003N\u2003\u2003\u2003V\u2003\u2003\u2003W\u2003\u2003\u2003K\u2003\u2003\u2003D\u2003\u2003\u2003D\u2003\u2003\u2003K\u2003\u2003\u2003T\u2003\u2003\u2003L"},{"entry":"ACA GAA AAT TCA TTT ACT AAC GTC TGG AAA GAC GAC AAA ACT TTA"},{"entry":{}},{"entry":"106 107 108 109 110 111 112 113 114 115 116 117 118 119 120"},{"entry":"\u2003D\u2003\u2003\u2003R\u2003\u2003\u2003Y\u2003\u2003\u2003A\u2003\u2003\u2003N\u2003\u2003\u2003Y\u2003\u2003\u2003E\u2003\u2003\u2003G\u2003\u2003\u2003C\u2003\u2003\u2003L\u2003\u2003\u2003W\u2003\u2003\u2003N\u2003\u2003\u2003A\u2003\u2003\u2003T\u2003\u2003\u2003G"},{"entry":"GAT CGT TAC GCT AAC TAT GAG GGT TGT CTG TGG AAT GCT ACA GGC"},{"entry":{}},{"entry":"121 122 123 124 125 126 127 128 129 130 131 132 133 134 135"},{"entry":"\u2003V\u2003\u2003\u2003V\u2003\u2003\u2003V\u2003\u2003\u2003C\u2003\u2003\u2003T\u2003\u2003\u2003G\u2003\u2003\u2003D\u2003\u2003\u2003E\u2003\u2003\u2003T\u2003\u2003\u2003Q\u2003\u2003\u2003C\u2003\u2003\u2003Y\u2003\u2003\u2003G\u2003\u2003\u2003T\u2003\u2003\u2003W"},{"entry":"GTT GTA GTT TGT ACT GGT GAC GAA ACT CAG TGT TAC GGT ACA TGG"},{"entry":{}},{"entry":"136 137 138 139 140 141 142 143 144 145 146 147 148 149 150"},{"entry":"\u2003V\u2003\u2003\u2003P\u2003\u2003\u2003I\u2003\u2003\u2003G\u2003\u2003\u2003L\u2003\u2003\u2003A\u2003\u2003\u2003I\u2003\u2003\u2003P\u2003\u2003\u2003E\u2003\u2003\u2003N\u2003\u2003\u2003E\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003S"},{"entry":"GTT CCT ATT GGG CTT GCT ATC CCT GAA AAT GAG GGT GGT GGC TCT"},{"entry":{}},{"entry":"151 152 153 154 155 156 157 158 159 160 161 162 163 164 165"},{"entry":"\u2003E\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003S\u2003\u2003\u2003E\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003S\u2003\u2003\u2003E\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003T"},{"entry":"GAG GGT GGC GGT TCT GAG GGT GGC GGT TCT GAG GGT GGC GGT ACT"},{"entry":{}},{"entry":"166 167 168 169 170 171 172 173 174 175 176 177 178 179 180"},{"entry":"\u2003K\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003E\u2003\u2003\u2003Y\u2003\u2003\u2003G\u2003\u2003\u2003D\u2003\u2003\u2003T\u2003\u2003\u2003P\u2003\u2003\u2003I\u2003\u2003\u2003P\u2003\u2003\u2003G\u2003\u2003\u2003Y\u2003\u2003\u2003T\u2003\u2003\u2003Y"},{"entry":"AAA CCT CCT GAG TAC GGT GAT ACA CCT ATT CCG GGC TAT ACT TAT"},{"entry":{}},{"entry":"181 182 183 184 185 186 187 188 189 190 191 192 193 194 195"},{"entry":"\u2003I\u2003\u2003\u2003N\u2003\u2003\u2003P\u2003\u2003\u2003L\u2003\u2003\u2003D\u2003\u2003\u2003G\u2003\u2003\u2003T\u2003\u2003\u2003Y\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003G\u2003\u2003\u2003T\u2003\u2003\u2003E\u2003\u2003\u2003Q\u2003\u2003\u2003N"},{"entry":"ATC AAC CCT CTC GAC GGC ACT TAT CCG CCT GGT ACT GAG CAA AAC"},{"entry":{}},{"entry":"196 197 198 199 200 201 202 203 204 205 206 207 208 209 210"},{"entry":"\u2003P\u2003\u2003\u2003A\u2003\u2003\u2003N\u2003\u2003\u2003P\u2003\u2003\u2003N\u2003\u2003\u2003P\u2003\u2003\u2003S\u2003\u2003\u2003L\u2003\u2003\u2003E\u2003\u2003\u2003E\u2003\u2003\u2003S\u2003\u2003\u2003Q\u2003\u2003\u2003P\u2003\u2003\u2003L\u2003\u2003\u2003N"},{"entry":"CCC GCT AAT CCT AAT CCT TCT CTT GAG GAG TCT CAG CCT CTT AAT"},{"entry":{}},{"entry":"211 212 213 214 215 216 217 218 219 220 221 222 223 224 225"},{"entry":"\u2003T\u2003\u2003\u2003F\u2003\u2003\u2003M\u2003\u2003\u2003F\u2003\u2003\u2003Q\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003R\u2003\u2003\u2003F\u2003\u2003\u2003R\u2003\u2003\u2003N\u2003\u2003\u2003R\u2003\u2003\u2003Q\u2003\u2003\u2003G\u2003\u2003\u2003A"},{"entry":"ACT TTC ATG TTT CAG AAT AAT AGG TTC CGA AAT AGG CAG GGG GCA"},{"entry":{}},{"entry":"226 227 228 229 230 231 232 233 234 235 236 237 238 239 240"},{"entry":"\u2003L\u2003\u2003\u2003T\u2003\u2003\u2003V\u2003\u2003\u2003Y\u2003\u2003\u2003T\u2003\u2003\u2003G\u2003\u2003\u2003T\u2003\u2003\u2003V\u2003\u2003\u2003T\u2003\u2003\u2003Q\u2003\u2003\u2003G\u2003\u2003\u2003T\u2003\u2003\u2003D\u2003\u2003\u2003P\u2003\u2003\u2003V"},{"entry":"TTA ACT GTT TAT ACG GGC ACT GTT ACT CAA GGC ACT GAC CCC GTT"},{"entry":{}},{"entry":"241 242 243 244 245 246 247 248 249 250 251 252 253 254 255"},{"entry":"\u2003K\u2003\u2003\u2003T\u2003\u2003\u2003Y\u2003\u2003\u2003Y\u2003\u2003\u2003Q\u2003\u2003\u2003Y\u2003\u2003\u2003T\u2003\u2003\u2003P\u2003\u2003\u2003V\u2003\u2003\u2003S\u2003\u2003\u2003S\u2003\u2003\u2003K\u2003\u2003\u2003A\u2003\u2003\u2003M\u2003\u2003\u2003Y"},{"entry":"AAA ACT TAT TAC CAG TAC ACT CCT GTA TCA TCA AAA GCC ATG TAT"},{"entry":{}},{"entry":"256 257 258 259 260 261 262 263 264 265 266 267 268 269 270"},{"entry":"\u2003D\u2003\u2003\u2003A\u2003\u2003\u2003Y\u2003\u2003\u2003W\u2003\u2003\u2003N\u2003\u2003\u2003G\u2003\u2003\u2003K\u2003\u2003\u2003F\u2003\u2003\u2003R\u2003\u2003\u2003D\u2003\u2003\u2003C\u2003\u2003\u2003A\u2003\u2003\u2003F\u2003\u2003\u2003H\u2003\u2003\u2003S"},{"entry":"GAC GCT TAC TGG AAC GGT AAA TTC AGA GAC TGC GCT TTC CAT TCT"},{"entry":{}},{"entry":"271 272 273 274 275 276 277 278 279 280 281 282 283 284 285"},{"entry":"\u2003G\u2003\u2003\u2003F\u2003\u2003\u2003N\u2003\u2003\u2003E\u2003\u2003\u2003D\u2003\u2003\u2003P\u2003\u2003\u2003F\u2003\u2003\u2003V\u2003\u2003\u2003C\u2003\u2003\u2003E\u2003\u2003\u2003Y\u2003\u2003\u2003Q\u2003\u2003\u2003G\u2003\u2003\u2003Q\u2003\u2003\u2003S"},{"entry":"GGC TTT AAT GAG GAT CCA TTC GTT TGT GAA TAT CPA GGC CAA TCG"},{"entry":{}},{"entry":"286 287 288 289 290 291 292 293 294 295 296 297 298 299 300"},{"entry":"\u2003S\u2003\u2003\u2003D\u2003\u2003\u2003L\u2003\u2003\u2003P\u2003\u2003\u2003Q\u2003\u2003\u2003P\u2003\u2003\u2003P\u2003\u2003\u2003V\u2003\u2003\u2003N\u2003\u2003\u2003A\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003S\u2003\u2003\u2003G"},{"entry":"TCT GAC CTG CCT CAA CCT CCT GTC AAT GCT GGC GGC GGC TCT GGT"},{"entry":{}},{"entry":"301 302 303 304 305 306 307 308 309 310 311 312 313 314 315"},{"entry":"\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003S\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003S\u2003\u2003\u2003E\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003S\u2003\u2003\u2003E\u2003\u2003\u2003G\u2003\u2003\u2003G"},{"entry":"GGT GGT TCT GGT GGC GGC TCT GAG GGT GGT GGC TCT GAG GGT GGC"},{"entry":{}},{"entry":"316 317 318 319 320 321 322 323 324 325 326 327 328 329 330"},{"entry":"\u2003G\u2003\u2003\u2003S\u2003\u2003\u2003E\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003S\u2003\u2003\u2003E\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003S\u2003\u2003\u2003G\u2003\u2003\u2003G\u2003\u2003\u2003G"},{"entry":"GGT TCT GAG GGT GGC GGC TCT GAG GGA GGC GGT TCC GGT GGT GGC"},{"entry":{}},{"entry":"331 332 333 334 335 336 337 338 339 340 341 342 343 344 345"},{"entry":"\u2003S\u2003\u2003\u2003G\u2003\u2003\u2003S\u2003\u2003\u2003G\u2003\u2003\u2003D\u2003\u2003\u2003F\u2003\u2003\u2003D\u2003\u2003\u2003Y\u2003\u2003\u2003E\u2003\u2003\u2003K\u2003\u2003\u2003M\u2003\u2003\u2003A\u2003\u2003\u2003N\u2003\u2003\u2003A\u2003\u2003\u2003N"},{"entry":"TCT GGT TCC GGT GAT TTT GAT TAT GAA AAG ATG GCA AAC GCT AAT"},{"entry":{}},{"entry":"346 347 348 349 350 351 352 353 354 355 356 357 358 359 360"},{"entry":"\u2003K\u2003\u2003\u2003G\u2003\u2003\u2003A\u2003\u2003\u2003M\u2003\u2003\u2003T\u2003\u2003\u2003E\u2003\u2003\u2003N\u2003\u2003\u2003A\u2003\u2003\u2003D\u2003\u2003\u2003E\u2003\u2003\u2003N\u2003\u2003\u2003A\u2003\u2003\u2003L\u2003\u2003\u2003Q\u2003\u2003\u2003S"},{"entry":"AAG GGG GCT ATG ACC GAA AAT GCC GAT GAA AAC GCG CTA CAG TCT"},{"entry":{}},{"entry":"361 362 363 364 365 366 367 368 369 370 371 372 373 374 375"},{"entry":"\u2003D\u2003\u2003\u2003A\u2003\u2003\u2003K\u2003\u2003\u2003G\u2003\u2003\u2003K\u2003\u2003\u2003L\u2003\u2003\u2003D\u2003\u2003\u2003S\u2003\u2003\u2003V\u2003\u2003\u2003A\u2003\u2003\u2003T\u2003\u2003\u2003D\u2003\u2003\u2003Y\u2003\u2003\u2003G\u2003\u2003\u2003A"},{"entry":"GAC GCT AAA GGC AAA CTT GAT TCT GTC GCT ACT GAT TAC GGT GCT"},{"entry":{}},{"entry":"376 377 378 379 380 381 382 383 384 385 386 387 388 389 390"},{"entry":"\u2003A\u2003\u2003\u2003I\u2003\u2003\u2003D\u2003\u2003\u2003G\u2003\u2003\u2003F\u2003\u2003\u2003I\u2003\u2003\u2003G\u2003\u2003\u2003D\u2003\u2003\u2003V\u2003\u2003\u2003S\u2003\u2003\u2003G\u2003\u2003\u2003L\u2003\u2003\u2003A\u2003\u2003\u2003N\u2003\u2003\u2003G"},{"entry":"GCT ATC GAT GGT TTC ATT GGT GAC GTT TCC GGC CTT GCT AAT GGT"},{"entry":{}},{"entry":"391 392 393 394 395 396 397 398 399 400 401 402 403 404 405"},{"entry":"\u2003N\u2003\u2003\u2003G\u2003\u2003\u2003A\u2003\u2003\u2003T\u2003\u2003\u2003G\u2003\u2003\u2003D\u2003\u2003\u2003F\u2003\u2003\u2003A\u2003\u2003\u2003G\u2003\u2003\u2003S\u2003\u2003\u2003N\u2003\u2003\u2003S\u2003\u2003\u2003Q\u2003\u2003\u2003M\u2003\u2003\u2003A"},{"entry":"AAT GGT GCT ACT GGT GAT TTT GCT GGC TCT AAT TCC CAA ATG GCT"},{"entry":{}},{"entry":"406 407 408 409 410 411 412 413 414 415 416 417 418 419 420"},{"entry":"\u2003Q\u2003\u2003\u2003V\u2003\u2003\u2003G\u2003\u2003\u2003D\u2003\u2003\u2003G\u2003\u2003\u2003D\u2003\u2003\u2003N\u2003\u2003\u2003S\u2003\u2003\u2003P\u2003\u2003\u2003L\u2003\u2003\u2003M\u2003\u2003\u2003N\u2003\u2003\u2003N\u2003\u2003\u2003F\u2003\u2003\u2003R"},{"entry":"CAA GTC GGT GAC GGT GAT AAT TCA CCT TTA ATG AAT AAT TTC CGT"},{"entry":{}},{"entry":"421 422 423 424 425 426 427 428 429 430 431 432 433 434 435"},{"entry":"\u2003Q\u2003\u2003\u2003Y\u2003\u2003\u2003L\u2003\u2003\u2003P\u2003\u2003\u2003S\u2003\u2003\u2003L\u2003\u2003\u2003P\u2003\u2003\u2003Q\u2003\u2003\u2003S\u2003\u2003\u2003V\u2003\u2003\u2003E\u2003\u2003\u2003C\u2003\u2003\u2003R\u2003\u2003\u2003P\u2003\u2003\u2003F"},{"entry":"CAA TAT TTA CCT TCC CTC CCT CAA TCG GTT GAA TGT CGC CCT TTT"},{"entry":{}},{"entry":"436 437 438 439 440 441 442 443 444 445 446 447 448 449 450"},{"entry":"\u2003V\u2003\u2003\u2003F\u2003\u2003\u2003S\u2003\u2003\u2003A\u2003\u2003\u2003G\u2003\u2003\u2003K\u2003\u2003\u2003P\u2003\u2003\u2003Y\u2003\u2003\u2003E\u2003\u2003\u2003F\u2003\u2003\u2003S\u2003\u2003\u2003I\u2003\u2003\u2003D\u2003\u2003\u2003C\u2003\u2003\u2003D"},{"entry":"GTC TTT AGC GCT GGT AAA CCA TAT GAA TTT TCT ATT GAT TGT GAC"},{"entry":{}},{"entry":"451 452 453 454 455 456 457 458 459 460 461 462 463 464 465"},{"entry":"\u2003K\u2003\u2003\u2003I\u2003\u2003\u2003N\u2003\u2003\u2003L\u2003\u2003\u2003F\u2003\u2003\u2003R\u2003\u2003\u2003G\u2003\u2003\u2003V\u2003\u2003\u2003F\u2003\u2003\u2003A\u2003\u2003\u2003F\u2003\u2003\u2003L\u2003\u2003\u2003L\u2003\u2003\u2003Y\u2003\u2003\u2003V"},{"entry":"AAA ATA AAC TTA TTC CGT "},{"entry":"\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003\u2003|<-----\u2003uncharged anchor region-----"},{"entry":{}},{"entry":"466 467 468 469 470 471 472 473 474 475 476 477 478 479 480"},{"entry":"\u2003A\u2003\u2003\u2003T\u2003\u2003\u2003F\u2003\u2003\u2003M\u2003\u2003\u2003Y\u2003\u2003\u2003V\u2003\u2003\u2003F\u2003\u2003\u2003S\u2003\u2003\u2003T\u2003\u2003\u2003F\u2003\u2003\u2003A\u2003\u2003\u2003N\u2003\u2003\u2003I\u2003\u2003\u2003L\u2003\u2003\u2003R"},{"entry":{"u":"GCC\u2003ACC\u2003TTT\u2003ATG\u2003TAT\u2003GTA\u2003TTT\u2003TCT\u2003ACG\u2003TTT\u2003GCT\u2003AAC\u2003ATA\u2003CTG"}},{"entry":"---------\u2003uncharged anchor region continues\u2003--------->|"},{"entry":{}},{"entry":"481 482 483 484 485"},{"entry":"\u2003N\u2003\u2003\u2003K\u2003\u2003\u2003E\u2003\u2003\u2003S\u2003\u2003\u2003."},{"entry":"AAT AAG GAG TCT TAA"},{"entry":{}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"154pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"7pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Molecular weight of peptide","=","58884",{}]},{"entry":["Charge on peptide","=","\u2003\u2003\u221220"]},{"entry":["[A +\u2003G +\u2003P]","=","\u2003\u2003143"]},{"entry":["[C +\u2003F +\u2003H +\u2003I +\u2003L +\u2003M +\u2003V +\u2003W +\u2003Y]","=","\u2003\u2003140"]},{"entry":["[D +\u2003E +\u2003K +\u2003R +\u2003N +\u2003Q +\u2003S +\u2003T +\u2003.]","=","\u2003\u2003202"]},{"entry":{}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"231pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Second Base",{}]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"8"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"7pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"t","c","a","g",{},{}]},{"entry":[{},"t","15","21","15","8","t"]},{"entry":[{},{},"12","5","10","6","c"]},{"entry":[{},{},"10","4","0","0","a"]},{"entry":[{},{},"0","3","0","4","g"]},{"entry":[{},{}]},{"entry":[{},"c","6","20","2","8","t"]},{"entry":[{},{},"3","4","0","3","c"]},{"entry":[{},{},"1","4","9","1","a"]},{"entry":[{},{},"4","3","7","0","g"]},{"entry":[{},{}]},{"entry":[{},"a","5","19","21","1","t"]},{"entry":[{},{},"5","4","11","1","c"]},{"entry":[{},{},"2","4","16","1","a"]},{"entry":[{},{},"8","2","4","2","g"]},{"entry":[{},{}]},{"entry":[{},"g","13","22","14","41","t"]},{"entry":[{},{},"6","7","12","29","c"]},{"entry":[{},{},"4","5","12","1","a"]},{"entry":[{},{},"1","3","16","4","g"]},{"entry":[{},{}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"10"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"AA","#","AA","#","AA","#","AA","#",{}]},{"entry":[{},"A","37","C","14","D","26","E","28"]},{"entry":[{},"F","27","G","75","H","2","I","12"]},{"entry":[{},"K","20","L","24","M","9","N","32"]},{"entry":[{},"P","31","Q","16","R","15","S","35"]},{"entry":[{},"T","29",".V\u2003","23","W","4","Y","25"]},{"entry":[{},".","1"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"9","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-1004","num":"1309"},"tables":{"@attributes":{"id":"TABLE-US-00093","num":"00093"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 130"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Sampling of a Library encoded by (NNK)"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"A. Numbers of hexapeptides in each class"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"210pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"total = 64,000,000 stop-free sequences."]},{"entry":[{},"\u03b1 can be one of [W, M, F, Y, C, I, K, D, E, N, H, Q]"]},{"entry":[{},"\u03a6 can be one of [P, T, A, V, G]"]},{"entry":[{},"\u03a9 can be one of [S, L, R]"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"112pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"112pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"\u03b1\u03b1\u03b1\u03b1\u03b1\u03b1 = 2985984.","\u03a6\u03b1\u03b1\u03b1\u03b1\u03b1 = 7464960."]},{"entry":[{},"\u03a9\u03b1\u03b1\u03b1\u03b1\u03b1 = 4478976.","\u03a6\u03a6\u03b1\u03b1\u03b1\u03b1 = 7776000."]},{"entry":[{},"\u03a6\u03a9\u03b1\u03b1\u03b1\u03b1 = 9331200.","\u03a9\u03a9\u03b1\u03b1\u03b1\u03b1 = 2799360."]},{"entry":[{},"\u03a6\u03a6\u03a6\u03b1\u03b1\u03b1 = 4320000.","\u03a6\u03a6\u03a9\u03b1\u03b1\u03b1 = 7776000."]},{"entry":[{},"\u03a6\u03a9\u03a9\u03b1\u03b1\u03b1 = 4665600.","\u03a9\u03a9\u03a9\u03b1\u03b1\u03b1 = 933120."]},{"entry":[{},"\u03a6\u03a6\u03a6\u03a6\u03b1\u03b1 = 1350000.","\u03a6\u03a6\u03a6\u03a9\u03b1\u03b1 = 3240000."]},{"entry":[{},"\u03a6\u03a6\u03a9\u03a9\u03b1\u03b1 = 2916000.","\u03a6\u03a9\u03a9\u03a9\u03b1\u03b1 = 1166400."]},{"entry":[{},"\u03a9\u03a9\u03a9\u03a9\u03b1\u03b1 = 174960.","\u03a6\u03a6\u03a6\u03a6\u03a6\u03b1 = 225000."]},{"entry":[{},"\u03a6\u03a6\u03a6\u03a6\u03a9\u03b1 = 675000.","\u03a6\u03a6\u03a6\u03a9\u03a9\u03b1 = 810000."]},{"entry":[{},"\u03a6\u03a6\u03a9\u03a9\u03a9\u03b1 = 486000.","\u03a6\u03a9\u03a9\u03a9\u03a9\u03b1 = 145800."]},{"entry":[{},"\u03a9\u03a9\u03a9\u03a9\u03a9\u03b1 = 17496.","\u03a6\u03a6\u03a6\u03a6\u03a6\u03a6 = 5625."]},{"entry":[{},"\u03a6\u03a6\u03a6\u03a6\u03a6\u03a9 = 56250.","\u03a6\u03a6\u03a6\u03a6\u03a9\u03a9 = 84375."]},{"entry":[{},"\u03a6\u03a6\u03a6\u03a9\u03a9\u03a9 = 67500.","\u03a6\u03a6\u03a9\u03a9\u03a9\u03a9 = 30375."]},{"entry":[{},"\u03a6\u03a9\u03a9\u03a9\u03a9\u03a9 = 7290.","\u03a9\u03a9\u03a9\u03a9\u03a9\u03a9 = 729."]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"224pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"\u03a6\u03a6\u03a9\u03a9\u03b1\u03b1, for example, stands for the set of peptides having two"]},{"entry":[{},"amino acids from the \u03b1 class, two from \u03a6, and two from \u03a9"]},{"entry":[{},"arranged in any order. There are, for example, 729 = 3"]},{"entry":[{},"sequences composed entirely of S, L, and R."]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"B. Probability that any given stop-free DNA sequence will"},{"entry":"encode a hexapeptide from a stated class."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colw
