---
title: Heterologous protein production using the twin arginine translocation pathway
abstract: Provided are means for evaluating and identifying putative substrates of the twin arginine translocation (Tat) secretory pathway in and other bacterial species. Also provided, therefore, are simple ways to express, secrete and purify correctly folded heterologous proteins on a large scale using host microorganisms, such as, and the Tat pathway therein. Many of the thus-produced proteins are of significant therapeutic value in the pharmaceutical and biochemical industries, particularly when they can be secreted from the host in fully-folded active form. Accordingly, there are further provided the heterologous proteins produced by the Tat secretion pathway using the foregoing methods, and the computer algorithm used to identify the Tat signal sequence and putative substrates.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=07447595&OS=07447595&RS=07447595
owner: The Trustees of the University of Pennsylvania
number: 07447595
owner_city: Philadelphia
owner_country: US
publication_date: 20030317
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["REFERENCE TO RELATED APPLICATION","GOVERNMENT INTEREST","FIELD OF THE INVENTION","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DESCRIPTION OF PREFERRED EMBODIMENTS OF THE INVENTION","EXAMPLES","Example 1","TATFIND 1.1","Example 2","TATFIND 1.2 and TATFIND 1.3","Example 3","Cloning of Tat Components and Putative Tat Substrates"],"p":["This application claims priority to U.S. Provisional Application Nos. 60\/364,877, filed Mar. 15, 2002, and 60\/441,576, filed Jan. 21, 2003, the contents of each of which are herein incorporated by reference.","This invention was supported in part by Grant Nos. T32GM-007229 from the National Institutes of Health, 0110093U from the American Heart Association, MCB 9816411 from the National Science Foundation and DE-FG02-01ER15169 from the Department of Energy. Accordingly, the Government may have certain rights in this invention.","This invention comprises a method of analyzing a protein secretion process in certain prokaryotic organisms, and methods for achieving the efficient, controlled production of folded heterologous proteins therefrom.","Genomics research has led to protein therapeutics that are claiming an increasingly large portion of the therapeutic drug market. Protein-based therapies constitute a growing portion of pharmaceutical and biotechnology pipelines. One of the biggest challenges in heterologous protein production on an industrial scale involves the purification of recombinant proteins. Because the protein concentration in the extra-cytoplasmic environment is drastically lower than in the cytoplasm, and often the secreted recombinant protein is the predominant protein in the supernatant, purification of secreted recombinant proteins from extra-cytoplasmic environment proves to be much easier compared to that from the cytoplam. The secreted recombinant protein can be separated from the cells by simple filtration or protein precipitation techniques. This approach of making recombinant proteins secreted has been extremely valuable for the production of numerous proteins, including insulin, growth hormones, various antibodies, and other immunoglobulins, whereby these recombinant proteins are secreted in an unfolded conformation via the universally conserved and essential general secretion (Sec) pathway (Matlack et al., 92:381-390 (1998); Mori et al., 9:494-500 (2001); Pohlschroder et al., 91:563-566 (1997); Fekkes et al., 63:161-173 (1999)).","However, the Sec pathway transports proteins only in an unfolded state, thus it is useless for exporting a significant number of proteins that fold prematurely in the cytoplasm or are unable to fold correctly in the extra-cytoplasmic environment.","Currently, proteins that cannot fold correctly in the extra-cytoplasmic environment must be expressed within the cytoplasm. However, that presents several problems. (i) Proteins folded prior to translocation cannot be secreted into the external environment when fused to a Sec signal sequence. Nevertheless, the Sec signal sequence still targets such folded proteins to the Sec pore, thereby most likely blocking the essential translocation pore, and thus severely affecting host growth. (ii) Purification of heterologously expressed proteins from complex cytoplasmic extracts is extremely challenging and very costly. (iii) Such heterologous proteins are often not stable in the cytoplasmic environment, and tend to aggregate, or be subjected to protein degradation by cytoplasmic proteases. (iv) In some cases cytoplasmically overexpressed proteins form inclusion bodies that are more easily purified, but then resolubilization of such inclusion bodies is often highly challenging and seldom recovers active proteins. Therefore, there is a need for the development of a new method that allows efficient translocation of pre-folded recombinant protein.","Consequently, the focus of the present invention is an alternate secretion mechanism, the twin arginine translocation (Tat) pathway, which secretes proteins in prefolded conformation (Hynds et al., 273:34868-34874 (1998); Rodrigue et al., 274:13223-13228 (1999); Thomas et al., 39:47-53 (2001)). Tat was originally identified in chloroplasts, has since also been found in bacteria and archaea (Santini et al., 17:101-112 (1998); Settles, et al., 278:1467-1470 (1997); Voelker et al., 14:3905-3914 (1995); Yen et al., 177:441-450 (2002).","Analyses of Tat mutants and substrates suggested that the major role of this pathway in prokaryotes is to translocate redox proteins that integrate their co-factors in the cytoplasm and, therefore, possess some degree of tertiary structure prior to secretion (Berks, 1996; Rodrigue et al., 1999; Weiner et al., 93:93-101 (1998)). However, the recent identification of non-redox Tat substrates (such as, virulence factors from ) indicates a broader role for the pathway in bacterial protein secretion (Ochsner et al., 99:8312-8317 (2001); Voulhoux et al., 20:6735-6741 (2001); Dilks et al., 185(4) (in press 2003)).","At least one copy of a TatA homolog and one copy of a TatC homolog are required for a functional Tat pathway (Bogsch et al., 273:18003-18006 (1998); Sargent et al., 17:3640-3650 (1998); Yen et al., 2002)). In certain prokaryotes (e.g., ) and many other organisms, multiple copies of TatA have been found and may be involved in the translocation of different substrates, and an additional protein, TatB, has been found to be necessary for Tat-dependent secretion in (Sargent et al., 274:36073-36082 (1999)). In , different TatC proteins are expressed under different conditions and seem to be responsible for the translocation of different substrates (Jongbloed et al., 275:41350-41357 (2000)).","The Tat pathway is distinct from the Sec pathway for, at least, the following reasons: (i) Tat substrates are secreted in a folded conformation (Hynds et al., 273:34868-34874 (1998); Rodrigue et al., 274:13223-13228 (1999); Thomas et al., 39:47-53 (2001)); (ii) Tat signal peptides contain a highly conserved twin arginine motif in their signal sequence (Cristobal et al., 18:2982-2990 (1999); Berks, 22:393-404 (1996) Berks et al., 35:260-274 (2000); Chaddock et al., 14:2715-2722 (1995); Niviere et al., 138 (Pt 10):2173-2183 (1992)); (iii) the energy necessary to drive translocation is provided solely by the proton motive force instead of ATP hydrolysis (Cline et al., 267:2688-2696 (1992); Santini et al., 1998)); and (iv) the Tat pathway is not a universally-conserved secretion mechanism (Wu et al., 2:179-189 (2000); Yen et al., 2002)).","Because of its unique ability of secreting pre-folded protein substrate, the Tat pathway represents an alternative mechanism for the production of recombinant secreted proteins, particularly for those pre-folded in the cytoplasm of the bacteria. However, in most commonly used bacterial species, such as , or , the Tat secretion pathway is not as efficient as the Sec pathway, and is not suitable for secreting high level of recombinant proteins under normal physiological growth conditions.","Thus, there is a need to discover or design a more efficient Tat secretion pathway for heterologous protein production and secretion in a biotechnologically amenable organism. To this end, a means is needed for identifying putative Tat substrates. This is necessary to permit investigation of the correlation between the number of components of the Tat system and the number of putative Tat substrates, and to understand how stably and efficiently the folded protein substrate can cross the cytoplasmic membrane.","Microorganisms that naturally utilize the Tat pathway frequently are likely to use the pathway more efficiently. are gram-positive spore-forming soil microorganisms that are well know in the biotechnology for their production of a large variety of secondary metabolites, many of which have antimicrobial, antifungal, and immunosuppressive activities. While particularly noted for antibiotic production, certain strains of have served as hosts for the heterologous production of human proteins with therapeutic value. This is partly due to the availability of well-established plasmid-based expression systems, developed fermentation technology and a low level of endogenous protease activity (Engels and Koller, in Transgenesis\u2014Application of Gene Transfer (Murray, ed.) Wiley & Sons, pp. 32-53 (1992)).",{"@attributes":{"id":"p-0015","num":"0014"},"i":["Streptomyces ","S. coelicolor ","Crit. Rev. Biotechnol. ","Streptomyces"]},"However, for every therapeutic protein that has been launched, hundreds more are in development. Many of these have been taken off the fast-track due to concerns about ways in which they can be manufactured efficiently, cheaply, and in large quantities. The secretion pathway currently utilized for mammalian cell culture and bacterial expression systems is limited to the production of proteins that fold correctly after translocation to the extra-cytoplasmic environment, but many proteins are incapable of folding correctly under those circumstances due to a lack of appropriate chaperones and other conditions, a fact which has held back the development of countless new treatments. Accordingly, a need has remained to exploit alternative methods to provide for the secretion of folded, active proteins in a highly specific and efficient manner that will be of critical importance to the pharmaceutical industry as it begins to investigate new therapeutic targets identified by the current advances in genomics.","The present invention studied the alternative twin arginine translocation (Tat) secretory pathway in and other bacterial species. It provides a means for evaluating and identifying putative substrates of the Tat secretory pathway. It also provides a simple way to produce and purify correctly folded heterologous proteins on a large scale using microorganisms, such as, as hosts and the Tat pathway therein. Thus, it still allows for protein purification from extra-cytoplasmic fractions.","Industrially tractable bacteria have been identified that are estimated to secrete more then 200 proteins via this alternative pathway. Moreover, it has been shown that these organisms efficiently secrete numerous heterologous proteins of interest in their correctly folded conformations by the use of selected signal leader sequences with a Tat motif to direct protein secretion through this alternative pathway. Thus, substrates that are of biotechnological interest are more likely to be efficiently secreted by microorganisms that naturally utilize the Tat pathway, such as strains, providing a mechanism for secretion of a larger number of fully-folded proteins, than through any of the other prokaryotic expression hosts, avoiding many of the problems encountered by the currently used Sec pathway.","It is an object of an embodiment of this invention, therefore, to provide methods for evaluating and identifying putative substrates utilizing a Tat secretory pathway, comprising comparing know and recorded amino acid sequences of a polypeptide in a microorganism against the Tat signal sequence motif, wherein the Tat signal sequence has the motif XRRXX within a first 35 N\u2032 terminal residues of the amino acid sequence of the polypeptide, wherein RR represents two adjacent arginine residues, and X designates positions restricted to other selected amino acid residues, as will be defined in greater detail below. It is a further object to provide those substrates that are identified by these methods.","It is a further object to produce recombinant proteins that are secreted via Tat secretion methods because in the extra-cellular environment there are fewer proteins, making it easier to isolate the protein of interest. By comparison, given the thousands of endogenous proteins inside of the host cell, separation of a protein of interest is very expensive and inefficient. However, the Sec pathway secretes unfolded proteins, and thus is unable to secrete a heterologously expressed protein that quickly folds in the cytoplasm, or one that has difficulty folding correctly in the external environment. Consequently, the ability of the Tat pathway to secrete folded proteins has made it the focus of the present invention.","Yet another object of the present invention is the production of desirable heterologous proteins from a bacterium utilizing the Tat pathway as a means of excretion. By fusing a selected signal sequence permitting optimal translocation efficiency to the N-terminus of the protein of interest, theoretically any protein of interest can be produced and secreted effectively. This is crucial in a market in which protein production is already important, and growing, as the pharmaceutical and biotech industries continue to develop new protein-based therapies, while struggling with facile, reliable and economical ways of producing proteins of interest.","Additional objects, advantages and novel features of the invention will be set forth in part in the description, examples and figures which follow, all of which are intended to be for illustrative purposes only, and not intended in any way to limit the invention, and in part will become apparent to those skilled in the art on examination of the following, or may be learned by practice of the invention.","Protein-based therapies constitute a growing portion of pharmaceutical and biotechnology industries, and the cellular process of protein secretion has been heavily exploited to produce heterologous proteins on a large scale. When secreted into the cellular environment, the gene product can often be separated from the cells by simple filtration or centrifugation and will often be the predominant protein in the supernatant. Recombinant proteins are currently engineered so that they are secreted via the general secretion (Sec) pathway. However, the Sec pathway transports proteins in an unfolded state, and is thus useless for exporting a significant number of proteins that fold prematurely or are unable to fold correctly in a hostile extra-cytoplasmic environment.","In accordance with the present invention, a twin arginine translocation (Tat) pathway has been developed and utilized to permit the controlled secretion of large, fully-folded heterologous proteins. A folded protein is a protein that has acquired its tertiary structure. Only a correctly folded protein can carry out the intended activity or function of the protein.","The Tat pathway is a protein secretion pathway that translocates proteins across chloroplast and certain bacterial hydrophobic membranes via the Tat pore, which has the ability to transport large, folded proteins. The Tat machinery includes the cytoplasmic components, as well as the pore components in the membrane that target Tat substrates to the membrane and translocate the substrates across the membrane, respectively. Heterologously expressed proteins that fail to fold correctly in the extra-cytoplasmic environment due to the lack of chaperones, are more likely to be properly folded in the cytoplasmic environment, wherein a large number of chaperones are available to assist in the folding of the protein. Accordingly, translocation via the Tat pathway has a unique advantage that it allows for the secretion of proteins that are already folded, and thus allows for the simpler protein purification from extra-cytoplasmic fractions. Even proteins that fold too quickly in the cytoplasm can still be secreted into the external environment via the Tat pathway, permitting them to be efficiently purified.","Initially, the utilization of the Tat and Sec pathways were examined in sp. NRC-1 by analyzing the entire genome, as set forth in Example 1, below. To better understand the role played by the Tat pathway in protein translocation, in vivo analyses of halophilic (\u03b1-amylase provided experimental evidence for the utilization of the Tat pathway by the Halobacteriaceae. From this information, the inventors have determined that (i) nearly all known extracellular proteins of the Halobacteriaceae have typical Tat signal sequences; and (ii) homologs of these proteins in organisms other than Halobacteriaceae contain Sec signal sequences.","To identify putative Tat and Sec substrates from the sp. NRC-1 genome, a PERL program, TATFIND (later referred to as TATFIND 1.1, as modifications were made to the original program) was developed by the inventors (see, Rose et al., 2002) to distinguish between the structurally similar signal sequences of these two substrate types. Thus, a scan of entire sp. NRC-1 for Tat and Sec substrates confirmed that almost all secreted proteins from halophilic archaeon are putative Tat substrates. Therefore, it appeared that extensive use of the Tat pathway by the Halobacteriaceae represents an evolutionary solution to the problem posed by the need for stabilization of secreted proteins in high-salt conditions. Folding of the secreted haloarchael proteins before translocation (i) permit them to be stabilized, and (ii) prevent aggregation, both intracellularly and extracellularly. Moreover, chaperones, which can play a crucial role in correct folding of proteins, are present in the cytoplasm but absent from the supernatant fluids. Thus, folding prior to secretion benefits from the help of chaperones.","To identify putative Tat substrates in the sp. NRC-1 genome, a PERL program, TATFIND (later referred to as TATFIND 1.1, as modifications were made to the original program) was developed by the inventors (see, Rose et al., 2002. Thus, a scan of entire sp. NRC-1 for Tat and Sec substrates confirmed that almost all secreted proteins from halophilic archaeon are putative Tat substrates. Therefore, it appeared that the Halobacteriaceae extensively use of the Tat pathway, which likely represents an evolutionary solution to the problem posed by the need for stabilization of secreted proteins in high-salt conditions. Folding of the secreted haloarchael proteins before translocation (i) permit them to be stabilized, and (ii) prevent aggregation, both intracellularly and extracellularly. Moreover, chaperones, which can play a crucial role in correct folding of proteins, are present in the cytoplasm but absent from the supernatant fluids. Thus, folding prior to secretion benefits from the help of chaperones.","In accordance with the findings of the present invention, (i) nearly all homologs of putative non-redox Tat substrates from sp. NRC-1 were also predicted to be Tat substrates in other Halobacteriaceae; and (ii) the majority of putative sp. NRC-1 Tat substrates were non-redox proteins. In addition, TATFIND analysis of the nearly completed genome sequence demonstrated an extensive usage of the Tat pathway for secreted proteins, similar to that observed for sp. NRC-1.","While there are a number of computer programs, such as SIGNALP (Nielsen et al., 10:1-6 (1997)) and PSORT, are designed to detect the signal sequences located on the N-terminus of Sec secreted proteins, they were developed before it was known that there was an alternative secretion mechanism. Consequently, such programs are unable to distinguish between Sec and Tat substrates, and thus are unable to predict the mechanism by which the protein is secreted (i.e., Sec or Tat).","TATFIND was designed by the inventors to detect essentially all of the properties of the signal sequence of the Tat secreted proteins TATFIND simplifies a tedious task, which until the present invention would have required the manual screening and comparison of all of the proteins in a genome. By comparison, TATFIND can search the entire genome for putative Tat substrates and yield output in less than 5 seconds. Moreover, it has proven to be highly accurate, despite the fact that putative structural determinants in the mature Tat substrate exist that have not yet been identified, and thus could not be included in TATFIND.","The algorithm as embodied in the present invention, when initially tested on the complete set of predicted proteins from the genome, successfully identified all experimentally confirmed Tat substrates with double arginines, and made only 4 erroneous predictions. Moreover, when tested on an organism () known to not use the Tat pathway, since the Tat genes are themselves missing from the archaeon, the program identified no substrates. Thus, the criteria used by this program were shown to provide sufficient information to predict Tat substrates with a high level of reliability.","TATFIND reads a single text file (containing one or more amino acid sequences for a protein) in FASTA format, and searches them using a series of rules to determine if any of them are likely to be secreted via the Tat pathway. Putative candidates are printed out along with a hydrophobicity score for the a 13 hydrophobic amino acid stretch of the h-region following the twin arginine residues. To operate properly the FASTA formatted files preferably have at least 60 characters per line. A hash is set up based upon the hydrophobicity values taken from Cid et al., 5:373-375 (1992), as follows: A=>0.02; R=>\u22120.42; N=>\u22120.77; D=>\u22121.04; C=>0.77; Q=>\u22121.10; E=>\u22121.14; G=>\u22120.80; H=>0.26; I=>1.81; L=>1.14; K=>\u22120.41; M=>1.00; F=>1.35; P=>\u22120.09; S=>\u22120.97; T=>\u22120.77; W=>1.71; Y=>1.11; V=>1.13. After grabbing the first 35 amino acids (aa) of each sequence, the program begins a series of tests \u201crules\u201d for Tat discovery:\n\n","Therefore, there is established a pattern sought within those first 35 amino acids of the protein sequence, which is:\n\n(X)RR(X)(X)(X)\u2003\u2003(Equation 1; SEQ ID NO:1)\n","wherein Xis H, A, P, K, R, N, T, G, S, D, Q, or E; wherein RRrepresent the twin arginines, wherein Xis A, P, K, R, N, T, G, S, D, Q, or E; wherein Xis I, W, F, L, V, Y, M, C, H, A, P, N, or T (positively charged residues were excluded from this position); and wherein Xis I, L, V, M, or F. The recognized single letter codes for amino acids are used consistently through this disclosure to represent the amino acids being identified as follows: alanine (A), arginine (R), asparagine (N), aspartic acid (D), cysteine (C), glutamic acid (E), glutamine (Q), glycine (G), histidine (H), isoleucine (I), leucine (L), lysine (K), methionine (M), phenylalanine (F), proline (P), serine (S), threonine (T), tryptophan (W), tyrosine (Y), valine (V).\n\n","If the pattern of Equation 1 was found, TATFIND assessed three additional criteria: (i) whether there was an uncharged stretch of at least 13 residues in the 22 residues following the R0; (ii) whether the uncharged stretch started behind a negatively charged residue (not allowed except behind positions +2 and +5) or behind a positively charged residue; and (iii) whether the hydrophobicity sum of the first 13 residues of the uncharged region was <8.0 according to Cid et al., 1992. If the above pattern was found and all the criteria were met, then a sequence was considered to be a putative Tat substrate.","Based upon the discovery that non-redox Tat substrates from sp. NRC-1 were also predicted Tat substrates in other Halobacteriaceae, the Tat process was evaluated 84 different microorganisms to determine the importance of this pathway in diverse bacteria and archaea (see Example 2). Using a genomic approach, and a slightly modified version of the TATFIND program (TATFIND 1.2), 84 diverse prokaryotic proteomes available from NCBI were evaluated, and from which a comprehensive list of putative Tat substrates was established.","TATFIND 1.2 still relies upon Equation 1 ((X)RR(X)(X)(X)), which remains SEQ ID NO:1, and the rules remained the same, as did the hydrophobicity values, but the residues permitted at each X value were expanded to include a possibility of M at position Xand of Q at position X. Therefore, TATFIND 1.2 applied the motif wherein Xis M, H, A, P, K, R, N, T, G, S, D, Q, or E; wherein RRrepresent the twin arginines, wherein Xis A, P, K, R, N, T, G, S, D, Q, or E; wherein Xis I, W, F, L, V, Y, M, C, H, A, P, N, or T (positively charged residues were excluded from this position); and wherein Xis Q, I, L, V, M, or F.","The results of the TATFIND studies, as well as phylogenetic analyses of Tat components, provided insight into the correlation between the number of components of the Tat system and the number of putative Tat substrates. Strikingly, these results indicated that contrary to earlier studies conducted by others on the bacterium , the Tat pathway is utilized to highly varying extents. Many prokaryotes secrete a large number of proteins via the Tat pathway. Furthermore, while many prokaryotes use this pathway predominantly for the secretion of redox proteins, analyses of the predicted substrates indicated that certain bacteria and archaea secrete mainly non-redox proteins via the Tat pathway. While no correlation was observed between the number of Tat machinery components encoded by an organism and the number of predicted Tat substrates, it was noted that the composition of this machinery was specific to phylogenetic taxa.","The signal sequences of Tat substrates resemble those of Sec substrates, in that both contain a positively charged hydrophilic n-region, followed by a hydrophobic stretch (h-region) and, in most cases, a cleavage site (c-region) (). However, Tat signal sequences are often significantly longer than Sec signal sequences, due to an extension of the n-region, which contains the conserved twin arginine motif (RR) (two neighboring arginines). The h-region of Tat signal sequences has a less conserved amino acid pattern, and it is generally less hydrophobic than that of Sec signal sequences and often consists of an uncharged region with neutral hydrophobicity. TatA\/tha4, TatB\/hcf106, and TatC\/cpTatC are the membrane components involved in Tat protein translocation in prokaryotes and chloroplasts, respectively.","In a preferred embodiment, the TATFIND-identified Tat signal sequence is used to effect the secretion of a large number of proteins into the surrounding environments via any of the prokaryotic expression hosts. However, as seen from the TATFIND analysis, most currently used Sec expression hosts have a low number of Tat substrates, and both the and the Tat systems are highly inefficient. Furthermore, it has been shown that only certain heterologously expressed proteins fused to an Tat signal sequence can be secreted, and that most of its native Tat substrates contain cofactors.","However, contrary to studies conducted in , many prokaryotes secrete a large number of proteins via the Tat pathway, most of which are non-redox proteins as reported by the inventors (see, Dilks et al., in press 2003). Thus, substrates that are of biotechnological interest are more likely to be efficiently secreted by strains, which appear to naturally use this pathway for a large number of proteins, in contrast to many other prokaryotic expression hosts. In fact, in accordance with the findings of the present invention, the industrially tractable organism , well known for its production of numerous antibacterial agents, is predicted to secrete more than 200 diverse proteins using this pathway (Dilks et al., 2003). The genus belongs to the family Actinomycetes, which are high G+C organisms, of which many develop a mycelial habit. The genera are defined by partial sequence analysis of 16S ribosomal RNA. They are soil microorganisms that are well know in the biotechnology for their production of a large variety of secondary metabolites, many of which have antimicrobial, antifungal, and immunosuppressive activities.","While the Sec pathway has been exploited for heterologous protein production in , little attention has been given to the Tat pathway as a general secretion pathway for heterologously expressed proteins. has one TatA and a TatC homolog and although a recent study (Schaerlaekens et al., 183(23):6727-6732 (2001)) reported the characterization of a TatC knockout, protein secretion analyses of the resulting mutant strain was only tested for its effect on translocation of two redox proteins, a bona fide Tat substrate and a redox protein. However, it is the ability to secrete non-redox proteins via the Tat pathway that is of particular significance in the present invention, which has not previously been suggested. In accordance with the present invention, uses the Tat pathway as a general secretion pathway. In fact, prior to the discovery of the Tat pathway, researchers observed arginines in signal sequences (\u201cArg cluster\u201d) that, if deleted, abolished protein translocation.","Furthermore, although unaware of the Tat pathway at the time of publication, two independent research groups reported a significant increase in protein secretion when a Sec signal sequence was replaced with a signal sequence containing what has now been identified by the present invention as a typical Tat motif: 1) Chang et al. (in Biology of Actinomycetes '88, Proc. of Seventh Internat'l Symposium on Biology of Actinomycetes (Okami, Beppu and Ogawara, eds.), Japanese Sci. Soc. Press, Tokyo, p. 103 (1988)) taught fusing human TNF to the signal sequences of two proteins, MelF and Spt-II, wherein the former resulting in an 18-fold increase in secretion efficiency; and 2) Rowland et al. (38(1):94-100 (1992)) replaced the species phosphodiesterase signal sequence with the galactosidase signal sequence, and observed an increased secretion efficiency of the thus heterologously expressed fusion protein in . In hindsight, it now appears that both the MelF and \u03b2-galactosidase signal sequences contain typical Tat motifs presented in the present invention, confirming that 1) secretion of these constructs via the Tat pathway was more efficient than when the proteins were directed to the Sec pathway, and 2) identifying the Tat pathway as the most efficient route of secretion for heterologously expressed proteins.","In a preferred embodiment, however, using a strain, such as or , using the identified Tat signal sequence, establishes an ideal expression host for heterologously expressed proteins for at least the following reasons: 1) are non-pathogenic; 2) as previously noted, are Gram positive, meaning that they have no outer membrane as an additional barrier, and thus proteins are secreted directly into the supernatant; 3) the necessary genetic and biochemical tools are known in the art for , and available for heterologous expression (e.g., expression vectors, inducible promoters, protein purification protocols, etc); 4) well-established, large-scale fermentation protocols are available, e.g., Gilbert et al., 15(1):13-39 (1995)); 5) efficient secretion of heterologous proteins fused to Sec and Tat signal sequences has been demonstrated, albeit before the Tat pathway had been identified (Noack et al., 68(1):53-62 (1988); Chang et al., 1988; Rowland et al., 1992; Bender et al., 34(2):203-207 (1990)); 6) contain a functional Tat pathway, as well as a large number of putative Tat substrates, indicating that these bacteria possess an efficient Tat pathway, and they contain a wide variety of Tat substrates, making them useful for the secretion of a diverse group of heterologous substrates; 7) moreover, regulatory agencies, such as the EPA and FDA, have an extensive history and are familiar with . Thus, the use of as expression host should not present additional difficulties for attaining regulatory approvals.","Heterologous proteins produced by recombinant techniques may be secreted and isolated from a mixture of cells or from medium containing the protein. \u201cRecombinant DNA\u201d is a molecule of containing DNA originating from two or more sources. The recombinant DNA encodes a recombinant protein, which may also be a \u201cheterologous protein.\u201d A \u201cheterologous protein\u201d is a protein that is expressed in a host other than its native host (i.e., the organism that naturally contains the DNA encoding the protein). Methods for generating fusion constructs for effecting the controlled secretion of a heterologous protein from a microorganism are well known in the art, once the sequence of the desired gene is known (see, for example, Ausubel et al. (1993) Current Protocols in Molecular Biology, Greene and Wiley, New York), and in this case once the heterologous gene is under the control of a Tat signal sequence.","For secretion of the subject protein, a DNA sequence encoding an appropriate Tat signal peptide is linked to the 5\u2032 end of the nucleotide sequence encoding the protein, such that the signal peptide directs the linked composition to the Tat pore, resulting in secretion of the protein from the cell via the Tat pathway. A cell culture typically includes host cells, media and other byproducts. Suitable mediums for cell culture are well known in the art. Protein can be isolated from cell culture medium, host cells, or both using techniques known in the art for purifying proteins. Suitable prokaryotic combinations will be readily apparent to those of skill in the art and can be found, for example, , Cold Spring Harbor Laboratory, New York (1989); Hopwood, 145(Pt 9):2183-2202 (1999)).","By \u201cexpression\u201d is meant the process, after the DNA for the heterologous protein is delivered into the selected prokaryotic cell, by which a nucleotide sequence undergoes successful transcription and translation such that detectable levels of the delivered nucleotide sequence are expressed in an amount and over a time period that a measurable amount of the heterologous protein is produced within the cell, properly folded and secreted via the Tat pathway into the culture medium.","The language \u201cnucleic acid molecule encoding a heterologous protein\u201d is intended to include any nucleic acid molecule that will be transcribed and translated into said protein in its biologically active and fully folded form upon introduction of such nucleic acid molecule into a viable host cell (e.g., the molecule can further contain appropriate control elements comprising the Tat signal sequence for regulating expression in the cell). The nucleic acid molecule encoding the heterologous protein can consist of the coding region of the corresponding gene, or alternatively it can further contain noncoding regions, such as 5\u2032 or 3\u2032 untranslated regions, introns, fragments thereof, or other sequences.","A nucleic acid sequence (or gene) encoding a heterologous protein is isolated using any one of several known molecular procedures. For example, primers comprising conserved regions of the sequences of interest may be used as probes to isolate, by polymerase chain reaction (PCR) or by direct hybridization, as yet unknown homologs in a DNA library comprising specific DNAs. Alternatively, antibodies directed against such a protein may be used to isolate clones encoding the protein from an expression library comprising specific DNAs. The isolation of primers, probes, molecular cloning and the generation of antibodies are procedures that are well known in the art and are described, for example, by Sambrook et al. or Ausubel et al., supra, or by Harlow et al. (in Antibodies, A Laboratory Manual, Cold Spring Harbor, N.Y., 1988).","Thus, the invention also includes the resulting heterologous protein as expressed by the isolated nucleic acid sequence and secreted from the host cell via the Tat pathway, wherein the nucleic acid encodes a protein which has preferably, at least about 35% identity to the corresponding full-length native protein. More preferably, the secreted heterologous protein, or biologically active fragment thereof, has at least about 45%, more preferably, at least about 55%, yet more preferably, at least about 65%, even more preferably, at least about 75%, yet more preferably, at least about 85% identity, and more preferably having, at least about 95%, and even more preferably, at least about 99% identity to the corresponding full-length native protein see above.","Thus, the nucleic acid molecule intended for use in accordance with the present invention can encode the full-length heterologous protein or a peptide fragment thereof that is of sufficient length to be biologically active, i.e., having the biological characteristics of the naturally occurring protein or modified forms thereof. The invention is also intended to include the use of fragments, mutants, or variants (e.g., modified forms) of the inserted heterologous gene, so long as it is capable of encoding and expressing a biologically active form of the encoded protein of interest.","The terms \u201cbiologically active\u201d or \u201cbiologically active form of the protein,\u201d as used herein, are meant to include forms of the heterologous protein that are capable of effecting the desired therapeutic activity of the native protein when used for its intended purposes. One skilled in the art can select such forms based upon the intended activity of the expressed and secreted protein via the Tat pathway. The activity of the resulting protein can be readily determined, for example, by comparing its functional ability with that of a comparable naturally occurring protein or one produced by standard means.","The nucleic acid used to encode the corresponding heterologous protein of interest can be a cDNA, or alternatively it can be a genomic DNA fragment. Mutants of the DNA sequence can be prepared and inserted into the host genome by a variety of known methods, such as, for example, by introducing nucleotide base pair modifications (e.g., substitutions, deletions, additions) to a nucleic acid molecule encoding the heterologous protein by standard methods, such as site-directed mutagenesis or polymerase chain reaction-mediated (PCR) mutagenesis.","Also included in the invention is a cell comprising an isolated nucleic acid encoding a heterologous protein secreted by the Tat pathway from the cell, and a cell comprising an isolated nucleic acid encoding a biologically active fragment thereof.","An \u201cisolated\u201d or \u201cpurified\u201d protein or biologically active portion thereof is substantially free of cellular material or other contaminating proteins from the cell or tissue source from which the protein is produced and isolated, or substantially free of chemical precursors or other chemicals when the protein is chemically synthesized. Thus, protein that is substantially free of cellular material includes preparations of protein having less than about 30%, 20%, 10%, or 5% (by dry weight) of contaminating proteins. When the protein or biologically active portion thereof is recombinantly produced, it is also preferably substantially free of culture medium, i.e., culture medium represents less than about 20%, 10%, or 5% of the volume of the protein preparation. When the protein is produced by chemical synthesis, it is preferably substantially free of chemical precursors or other chemicals, i.e., it is separated from chemical precursors or other chemicals that are involved in the synthesis of the protein. Accordingly such preparations of the protein have less than about 30%, 20%, 10%, 5% (by dry weight) of chemical precursors or compounds other than the polypeptide of interest.","An isolated biologically active polypeptide can have several different physical forms. The isolated heterologous polypeptide can exist as a full-length nascent or unprocessed polypeptide, or as partially processed polypeptides or combinations of processed polypeptides. The full-length nascent polypeptide can be postranslationally modified by specific proteolytic cleavage events that results in the formation of fragments of the full-length nascent polypeptide. A fragment, or physical association of fragments can have the biological activity associated with the full-length polypeptide, however, the degree of biological activity associated with individual fragments can vary. Accordingly, the present invention also provides for analogs of the heterologous proteins or peptides encoded by the isolated nucleic acid sequences introduced into the host cells for expression via the Tat pathway.","Analogs can differ from naturally occurring proteins or peptides by conservative amino acid sequence differences or by modifications which do not affect sequence, or by both. For example, conservative amino acid changes may be made, which although they alter the primary sequence of the protein or peptide, do not normally alter its function. Mutant forms of the protein of interest that have amino acid substitutions, deletions and\/or additions as compared to the naturally occurring amino acid sequence of a comparable native protein molecule, yet still retain the functional activity of the natural form of protein as described herein are also encompassed by the invention. To retain the functional properties, preferably only conservative amino acid substitutions are made at one or more amino acid residues.","A \u201cconservative amino acid substitution\u201d is one in which the amino acid residue is replaced with an amino acid residue having a similar side chain. Families of amino acid residues having similar side chains have been defined in the art, including basic side chains (e.g., lysine, arginine, histidine), acidic side chains (e.g., aspartic acid, glutamic acid), uncharged polar side chains (e.g., glycine, asparagine, glutamine, serine, threonine, tyrosine, cysteine), nonpolar side chains (e.g., alanine, valine, leucine, isoleucine, proline, phenylalanine, methionine, tryptophan), beta-branched side chains (e.g., threonine, valine, isoleucine) and aromatic side chains (e.g., tyrosine, phenylalanine, tryptophan, histidine). Modifications (which do not normally alter primary sequence) include in vivo, or in vitro chemical derivatization of the subject polypeptides, e.g., acetylation, or carboxylation. Also included are modifications of glycosylation, e.g., those made by modifying the glycosylation patterns of a polypeptide during its synthesis and processing or in further processing steps; e.g., by exposing the polypeptide to enzymes which affect glycosylation. Also embraced are sequences having phosphorylated amino acid residues, e.g., phosphotyrosine, phosphoserine, or phosphothreonine.","Also included are polypeptides that have been modified using ordinary molecular biological techniques so as to improve their resistance to proteolytic degradation or to optimize solubility properties or to render them more suitable as a therapeutic agent. Analogs of such polypeptides include those containing residues other than naturally-occurring L-amino acids, e.g., D-amino acids or non-naturally occurring synthetic amino acids. However, the peptides of the invention are not limited to products of any of the specific exemplary processes listed herein.","To express a heterologous protein as encoded by a nucleic acid molecule from a host cell, the nucleic acid must be operably linked to regulatory elements. \u201cOperably linked\u201d is intended to mean that the nucleotide sequence encoding the expressed protein is linked to at least one regulatory sequence in a manner which allows expression of the nucleotide sequence and secretion from the cell via the Tat pathway. Regulatory sequences are selected to direct expression of the desired protein in an appropriate host cell. Accordingly, the term \u201cregulatory sequence\u201d includes promoters, enhancers and other expression control elements. Such regulatory sequences are known to those skilled in the art and are further described, for example, in Goeddel, 185, Academic Press, San Diego, Calif. (1990).","These regulatory elements include those required for transcription and translation of the nucleic acid encoding the protein(s), and may include, in addition to the Tat signal sequence of the present invention, promoters, enhancers, polyadenylation signals, and sequences necessary for transport of the molecule to the appropriate cellular compartment to permit secretion from the cell via the Tat pore. When the nucleic acid is a cDNA in a recombinant expression vector, the regulatory functions responsible for transcription and\/or translation of the cDNA are often provided by viral sequences. Examples of commonly used viral promoters include those derived from polyoma, adenovirus 2, cytomegalovirus and simian virus 40, and retroviral LTRs. However, the promoter activity may be endogenously provided by the host cell.","Regulatory sequences linked to the cDNA can be selected to provide constitutive or inducible transcription. Inducible transcription can be accomplished by, for example, use of an inducible enhancer. Thus, in a specific embodiment of the invention the nucleic acid molecule encoding the heterologous fusion protein is under the control of an inducible control element, such that expression of the protein can be turned \u2018on\u2019 or \u2018off\u2019 (or intermediate levels in between) using an agent which affects the inducible control element (e.g., expression can be modulated by modulating the concentration of the inducing agent in the cell). This allows for switching \u2018on\u2019 or \u2018off\u2019 of the expression of the subject protein in the host cell, or the systems necessary for the enhanced proliferation of the host cells.","When the nucleic acid molecule encoding a heterologous protein is operably linked to regulatory elements, it is typically carried in a vector. A \u201cvector\u201d is a composition of matter which comprises nucleic acids, and which can be used to effect the secretion of the heterologous protein from the cell via the Tat mechanism, under the direction of the identified Tat signal sequence. The vector carries the gene encoding the fusion protein and a promoter that allows for the expression of the protein, whereas the Tat machinery allows for secretion. Numerous vectors are known in the art including, but not limited to, linear polynucleotides, polynucleotides associated with ionic or amphiphilic compounds, plasmids, viruses or other nucleic acid molecules comprising, for example, sequences that are necessary for selection and amplification of the nucleic acid molecule in bacteria. Thus, the term vector includes an autonomously replicating plasmid or a virus.","The term should also be construed to include non-plasmid and non-viral compounds, which facilitate transfer of nucleic acid into cells, such as, for example, polylysine compounds, liposomes, and the like. \u201cexpression vector\u201d refers to a vector comprising a recombinant polynucleotide comprising expression control sequences operatively linked to the nucleotide sequence encoding the heterologous protein to be expressed from the host cell. Thus, a nucleic acid molecule comprising a nucleotide sequence or gene encoding the heterologous protein operably linked to regulatory control elements, is also referred to herein as an \u201cexpression vector.\u201d An expression vector comprises sufficient cis-acting elements for expression; whereas other elements for expression can be supplied by the host cell or an in vitro expression system.","In one embodiment, a vector is employed which is capable of integrating the desired gene sequences into the host cell chromosome. Cells which have stably integrated the introduced DNA into their chromosomes can be selected by also introducing one or more markers which allow for selection of host cells which contain the expression vector. The marker may provide for prototrophy to an auxotrophic host, biocide resistance, e.g., antibiotics, or heavy metals, such as copper, or the like. The selectable marker gene sequence can either be directly linked to the DNA gene sequences to be expressed, or be introduced into the same cell by co-transfection. Additional elements may also be needed for optimal synthesis of protein mRNA. These elements may include splice signals, as well as transcription promoters, enhancers, and termination signals. cDNA expression vectors incorporating such elements include those described by Okayama et al., 3:280 (1983).","In a preferred embodiment, the introduced sequence will be incorporated into a plasmid or viral vector capable of autonomous replication in the recipient host. Any of a wide variety of vectors may be employed for this purpose. Factors of importance in selecting a particular plasmid or viral vector include: the ease with which recipient cells that contain the vector may be recognized and selected from those recipient cells which do not contain the vector; the number of copies of the vector which are desired in a particular host; and whether it is desirable to be able to \u201cshuttle\u201d the vector between host cells of different species. Preferably, vectors can be replicated in (for easy cloning) and then be transferred into where they also can replicate because they have two origins of replication (i.e., shuttle vectors). Alternatively, \u2018integration vectors,\u2019 are used which can replicate in (for easy cloning) and integrate into the chromosome by homologous recombination.","Expression vectors include all those known in the art, such as cosmids, plasmids (e.g., naked or contained in liposomes) and viruses that incorporate the recombinant polynucleotide. Expression vectors within the scope of the invention, therefore, comprise a nucleic acid encoding the subject heterologous protein as described herein and a promoter operably linked to the nucleic acid, further comprising the Tat signal sequence to control the secretion of the expression product in fully folded form from the cell via the Tat pathway. Such expression vectors can be used to transfect host cells to thereby produce the protein encoded by the nucleic acid as described herein. An expression vector of the invention, as described herein, typically further includes nucleotide sequences encoding the heterologous protein operably linked to at least one \u201cregulatory sequence.\u201d","An expression vector of the invention can be used to transfect cells, either prokaryotic or eukaryotic (e.g., mammalian, insect or yeast cells), but preferably prokaryotic, and more preferably , to thereby produce proteins encoded by nucleotide sequences transported by the vector. Expression in prokaryotes is most often carried out with vectors containing constitutive or inducible promoters. Certain expression vectors (so called fusion-vectors) are designed to add a number of amino acid residues to the expressed recombinant protein, usually to the amino terminus of the expressed protein. These are well known in the art and commercially available. One strategy to maximize expression of a heterologous protein in accordance with the present invention, is to express the protein from a host bacterium which has an impaired capacity to proteolytically cleave the protein of interest within or outside the cell, particularly when the protein is a recombinant protein.","As regulatory sequences have been described above, it should be understood that the design of the expression vector may depend on such factors as the choice of the host cell to be transfected, and\/or the type and\/or amount of protein desired to be expressed.","A \u201chost-vector system\u201d refers to host cells, which have been transfected with appropriate vectors using recombinant DNA techniques. The vectors and methods disclosed herein are suitable for use in host cells over a wide range of prokaryotic organisms when the DNA is operatively linked to a promoter that is highly active in the selected prokaryotic cells. The preferred host vector system is one utilizing the Tat expression pathway for secreting proteins into the extra-cytoplasmic environment, more preferably one in which the host has been entirely characterized and sequenced, even more preferably a , or or even an ","Most preferably the hosts are of the family of microorganisms, of which the or are very useful, or in the alternative, the hosts may be other microorganisms, wherein expression and secretion is controlled by the signal sequences and the secretion kinetics are those of the Tat substrates.","The nucleic acid molecule encoding a heterologous protein can be introduced into the host cell by various methods typically referred to as transfection. The terms \u201ctansfection\u201d or \u201ctransfected with\u201d refers to the introduction of exogenous nucleic acid into a host cell and are intended to encompass a variety of techniques useful for introduction of nucleic acids into such cells, including electroporation, calcium-phosphate precipitation, DEAE-dextran treatment, lipofection, microinjection, and viral infection. In particular, inserting DNA into requires making protoplasts or conjugation, for example, such methods are described in Kieser, et al., (Norwich; John Innes Foundation) p. 613 (2000). Suitable methods for transfection can be found in Sambrook et al. (2Ed., Cold Spring Harbor Laboratory Press (1989)) and other laboratory textbooks.","A number of procedures may be used to assess whether the microorganism comprises the desired heterologous nucleic acid, and whether the protein of interest has been secreted. For example, genomic DNA obtained from the cells of the transgenic microorganism may be analyzed by Southern blot hybridization or by PCR to determine the length and orientation of any inserted transgenic nucleic acid present therein. Northern blot hybridization analysis or PCR may be used to characterize mRNA transcribed in cells of the transgenic microorganism. The procedures for performing such analyses are well known in the art and are described, for example, in Sambrook et al., a western blot analysis may be used to identify, characterize or quantify any expressed proteins or polypeptides that are secreted into the culture medium, using e.g., antibody raised against the selected heterologous protein or active fragments thereof. In situations where a heterologous polypeptide is expressed in a catalytically active form, PC biosynthesis assays may be used to identify and characterize the enzyme molecules so expressed.","The Tat pathway, like the Sec pathway, has proven to be a general secretion apparatus, potentially allowing for the translocation of a wide variety of heterologous substrates. Thus, in a preferred embodiment, the fusion of a Tat signal sequence (i.e., one having the identified Tat motif) to a biologically valuable protein allows for the efficient production of the active protein targeted to the Tat machinery, thereby permitting translocation of the protein in a fully-folded manner. A signal sequence of the Tat pathway is a sequence of amino acids at the N-terminus of a Tat substrate that has a specific structure and sequence motif that is recognized by the Tat machinery. Tat substrates are proteins that contain a Tat signal sequence, and in turn, are targeted to and translocated across the hydrophobic membrane via the Tat machinery.","Moreover, several of the putative Tat substrates in prokaryotes themselves are likely to be useful for biotechnological uses when provided with an expression host with an effective Tat translocation system. Furthermore, the absence of Tat translocon components (TatA\/B and TatC) in mammals, along with the observation that the elimination of a functional TatC attenuates virulence of , suggest that the Tat translocon may represent a novel antibiotic target (Ochsner et al., 2002; Voulhoux et al., 2001).","The present invention is further described by example. Nevertheless, the following examples are not to be construed as limiting the scope of the appended claims.","The following experiments were conducted to evaluate the Tat secretory process in a necessarily diverse set of microorganisms to provide direct evidence of the importance of this pathway in bacteria and archaea, and to develop program means for identifying putative Tat substrates among the set of all putative coding sequences (CDS) from entire genomes.","To examine utilization of the Tat and Sec pathways in Halobacteriaceae, analyses were conducted in sp. NRC-1, currently the only known complete genomic sequence of a halophilic archaeon. The entire genome was analyzed using the following strategy: (i) a Tat substrate recognition program (TATFIND) was developed to detect putative Tat substrates in sp. NRC-1 (this program, described in greater detail below, is based on the position and sequence of the Tat pattern, as well as on the position, length and hydrophobicity of an uncharged region following the twin-arginine pattern); (ii) usage of the Sec pathway in sp. NRC-1 was examined by identifying putative secreted proteins with SIGNALP (Nielsen et al., 1997); (iii) all SIGNALP-positive candidates were analyzed further with the TMHMM program (Sonnhammer et al., 6:175-182 (1998)) that predicts membrane spanning segments, thereby eliminating proteins with multiple membrane-spanning segments; and (iv) all remaining candidates were classified based on their subcellular localization. Because in any genome analysis, only a fraction of coding sequences (CDS) can be positively identified, the proteins chosen for analysis were encoded by annotated CDS and it was assumed that such subsets were representative of the whole population of CDS. A detailed map of the analyses and complete results of individual steps of the process can be accessed on the Internet at sas.upenn.edu\/\u02dcpohlschr (herein incorporated by reference in the entirety).","Sequences Used in the Analyses","All predicted proteins (identified and putative) as annotated in GenBank records [NC000917.1 (-); AE004437, AE004438 (plasmid NRC100) and NC001869.1 (plasmid NRC200) (sp. NRC-1)] were analyzed to identify putative Tat and Sec substrates. Furthermore, U00096.1 (K12); NC000909, NC001732 (large extra-chromosomal element), NC001733 (small extra-chromosomal element) (); NC000854.1 (K1); and AE006641 ()] were analyzed to identify putative Tat substrates.","Tat Substrate Prediction","A PERL program (TATFIND) was written to identify putative Tat substrates among the set of all putative coding sequences (CDS) from the entire genome. The patterns recognized by TATFIND were taken from the list of putative secreted proteins from sp. NRC-1, and then refined with residues found in putative Tat substrates from other Halobacteriaceae. The position of the pattern in the N-terminus, as well as the length and position of the following uncharged region, was adjusted according to known extrema in bacterial Tat substrates (i.e., glucose: fructose oxidoreductase (GFOR): largest n-region with RR at position 30\/31; HybO: shortest uncharged stretch of only 13 residues length; NapG: uncharged stretch begins nine residues behind the RR motif) (Halbig et al., 1999; Robinson et al., 2001; Stanley et al., 2001).","The version TATFIND, used to analyze the complete set of predicted proteins of all species described in this work, searched for the following pattern between residues 2 and 35 of the predicted protein: (X)RR(X)(X)(X), where the amino acid at position Xhad a hydrophobicity score (0.26; Xhad a hydrophobicity score (0.02; Xhad a hydrophobicity score (\u22120.77 (positively charged residues were excluded from this position); and Xwas one of the following residues (I, L, V, M, or F). All hydrophobicity values were taken from Cid et al., 1992). If the above pattern was found, TATFIND assessed three additional criteria: (i) whether there was an uncharged stretch of at least 13 residues in the 22 residues following the RR; (ii) whether the uncharged stretch started behind a negatively charged residue (not allowed except behind positions +2 and +5); and (iii) whether the hydrophobicity sum of the first 13 residues of the uncharged region was <8.0. If the above pattern was found and all the criteria were met, then a sequence was considered to be a putative Tat substrate.","Identification and Analysis of Putative Tat Substrates","Applying TATFIND to the sp. NRC-1 genome, 64 putative secreted Tat substrates were identified. Of these, 34 had detectable homologs with assigned functions that could be classified into one of four groups: (i) eight putative extracellular enzymes; (ii) 13 binding proteins; (iii) seven redox proteins; and (iv) six other surface proteins (See, Rose et al., 2002, herein incorporated by reference). Homologs of putative sp. NRC-1 Tat substrates were identified in other organisms by BLASTP and screened with TATFIND and SIGNALP to determine whether they were potential Tat or Sec substrates respectively. Although all haloarchaeal homologs were putative Tat substrates, the non-haloarchaeal homologs of all non-redox Tat substrates were putative Sec substrates (Table 1).","Identification of Putative Tat Substrates in Non-Halobacteriaceae","The same analyses were carried out with three additional non-halophilic archaeal genomes. Only eight putative Tat substrates were identified in , a thermophilic non-halophilic euryarchaeon closely related to sp. NRC-1 (Table 1). Consistent with previous findings, of the seven proteins that had assigned functions, six were putative redox proteins. Similarly, in two thermophilic and , only four and 10 putative Tat substrates, respectively, were identified. Of these, all the assigned substrates were putative redox proteins (Table 1).",{"@attributes":{"id":"p-0092","num":"0096"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Classification of putative archaeal Tat substrates."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{"i":"Halobacterium"},{"i":"Archaeoglobus"},{"i":"Sulfolobus"},{"i":"Aeropyrum"}]},{"entry":[{},"sp. NRC-1",{"i":"fulgidus"},{"i":"solfataricus"},{"i":"pernix"}]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Redox","7","6","3","2"]},{"entry":"proteins"},{"entry":["Redox","7","6","3","2"]},{"entry":"proteins"},{"entry":["Non-redox","27","1","0","0"]},{"entry":"proteins"},{"entry":["Non-redox","27","1","0","0"]},{"entry":"proteins"},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},"The accuracy of TATFIND was assessed by screening the genomes of (i) , which has been estimated to contain more than 400 exported proteins including 26 that are known or hypothesized to be secreted via the Tat pathway; and (ii) , which lacks homologs of the Tat translocation genes. TATFIND predicted 34 substrates in , successfully identifying all 26 previously reported Tat substrates (Robinson et al., 2001; Stanley et al., 2001). Most remarkably, this algorithm did not predict any proteins in , which is consistent with the lack of the Tat genes in this methanogen. In addition, TATFIND identified no cytoplasmic proteins in and ","Taken together, these results supported the previous findings that the extensive use of the Tat pathway is an unusual feature of protein secretion in the Halobacteriaceae, and confirmed that TATFIND is a suitable tool for the prediction of Tat substrates in a broad range of prokaryotes.","Identification of Putative Archaeal Sec Substrates","To determine the significance of the relatively high number of putative extracellular Tat substrates identified in the Halobacteriaceae, it was necessary to examine the utilization of the Sec pathway for haloarchaeal secretion. This required multiple steps. SIGNALP (Nielsen et al., 1997) was used to identify putative signal sequence-containing proteins from the complete set of all predicted proteins. However, since SIGNALP has not been trained specifically to recognize archaeal signal sequences, broad parameters were used and options for Gram negative, Gram positive, and eukaryotic signal sequence predictions were included, as well as a lenient cut-off value (3 \u2018yes\u2019 responses by at least one predictive option). Of the SIGNALP positives, all TATFIND-positive proteins were excluded. From this subset, proteins with TMHMM-predicted transmembrane helices (Sonnhammer et al., 1998) outside the N-terminal 50 residues were removed, as these proteins are not secreted, but rather, are integral membrane proteins.","Putative Sec substrates from sp. NRC-1 were identified by submitting all CDS to SIGNALP. However, SIGNALP predictions cannot be considered to be Sec pathway specific per se, as this program is not able to clearly distinguish between substrates of the Sec and Tat translocation pathways, because the overall structure of Tat and Sec signal sequences is conserved. Furthermore, the Sec machinery, unlike the Tat pathway, does not recognize specific amino acid patterns. Therefore, TATFIND was used to identify and eliminate putative Tat substrates from the SIGNALP output.","All putative proteins that were positive by TATFIND or SIGNALP (after removing integral membrane proteins) were analyzed further by BLASTP to identify homologs (E-value (10) (Altschul et al., 215:403-410 (1990)). The homologs were then examined, and cytoplasmic proteins were removed. The non-cytoplasmic homologs were then analyzed by TATFIND, SIGNALP and TMHMM to determine their potential mode of secretion.","Of the SIGNALP positives, 171 did not contain membrane-spanning segments past the first 50 amino acids, as identified by TMHMM. Fifty of these could be putatively identified by BLASTP analysis, of which the majority (33) were false positives as they had significant similarity to cytoplasmic or membrane proteins. Furthermore, five proteins had significant similarity to flagellins, which are known to use a specialized translocation system. The remaining 12 proteins were putative Sec substrates, although their cell surface localization was evident in only three cases, including a homolog of a type IV protease and two proteins involved in uptake pathways. In contrast, when similar analyses were conducted with the genome, it was observed that the majority of proteins were secreted via the Sec pathway. These results indicated that the vast majority of putative secreted proteins of sp. NRC-1 are translocated via the Tat pathway.","Functional Analysis of the Haloarchaeal Tat Motif In Vivo","It is known that the exchange of the twin arginines (RR) with lysines (KK) in the twin-arginine motif of Tat substrates results in a block of Tat-dependent protein translocation in (Stanley et al., 2001). Thus, to determine the Tat specificity of an extracellular halophilic protein, the RR of the (\u03b1-amylase precursor from the alkalihalophile sp. strain Ah36 (Kobayashi et al., 176:5131-5134 (1994)) (AmyRR) was replaced with KK (AmyKK).","Construction of Wild-Type and Signal Sequence Mutant \u03b1-Amylase","Plasmids. The twin-arginine residues of the wild-type sp. strain Ah-36 (Kobayashi et al., 1994) \u03b1-amylase signal sequence were replaced with twin-lysine residues using site-directed polymerase chain reaction (PCR) mutagenesis. A 125 bp fragment encompassing the 5\u2032 end of the \u03b1-amylase gene (including 57 bp upstream of the open reading frame, ORF) was PCR amplified (forward primer 5\u2032-GTTAGCACTAAGCTTCGAAACCGAA TTAAAATCATTAT-3\u2032 (SEQ ID NO:2); reverse primer 5\u2032-CGAGCGCAGGACGGTCTTTTT GTCGATACCCGCCG-3\u2032 (SEQ ID NO:3)) from pANAM121 (which harbors the wild-type \u03b1-amylase; Kobayashi et al., 1994), thus replacing the twin arginines at amino acid positions 16 and 17 with twin lysines. A second 1543 bp fragment encompassing the \u03b1-amylase ORF starting from nucleotide 91 (amino acid 11) was PCR amplified (forward primer 5\u2032-TCGGCGGGTATCGACAAAAAGACCGTCC TGCGCTCG-3\u2032 (SEQ ID NO:4); reverse primer 5\u2032-GACTGT GGTACCTCAGTCGTCGTCGGACAG-3\u2032 (SEQ ID NO:5)) from pANAM121.","These two fragments were ligated together using a modified PCR, and the product was PCR amplified (forward primer 5\u2032-GTTAGCACTAAGCTTCGAAACCGAATTAAAATCATT AT-3\u2032 (SEQ ID NO:6); reverse primer 5\u2032-GACTGTGGTACCTCAGTCGTCGTCGGA CAG-3\u2032 (SEQ ID NO:7)) to produce the final signal sequence mutant \u03b1-amylase insert (AmyKK). The wild-type \u03b1-amylase insert (AmyRR) was also generated by PCR using these two primers. Both inserts were cut with HindIII and KpnI and cloned into pMLH3 (Holmes et al., 146: 117-121 (1994)), resulting in the expression vectors pAMY-KK and pAMY-RR. These constructs were then used to transform (strain WFD11) (Charlebois et al., 84: 8530-8534 (1987); Cline et al., 171:4987-4991 (1989)).","Expression and Detection of Wild-Type and Signal Sequence Mutant \u03b1-Amylase","In vivo starch hydrolysis assay. Secretion of the wild-type and mutant \u03b1-amylase was examined by a starch hydrolysis assay. Both harboring signal sequence-mutated pAMY-KK and expressing wild-type pAMY-RR were grown on rich medium (Charlebois et al., 1987) supplemented with 0.2% soluble starch. Once single, separated colonies were visible, the plates were exposed to iodine vapor to detect starch hydrolysis by extra-cytoplasmic \u03b1-amylase. Clear halos surrounding a colony indicated starch hydrolysis by extra-cytoplasmic \u03b1-amylase ().","Immunoblot analysis. To confirm that \u03b1-amylase was synthesized in these cells, Western blot analysis of SDS-denatured cell extracts (cyt) and culture supernatants (sup) from expressing either AmyRR or AmyKK was performed (). Cells were cultured in rich medium to an absorbance (600 nm) of 0.6, pelleted and lysed with SDS-PAGE sample buffer. The cell supernatant was precipitated with 10% trichloroacetic acid. Samples were electrophoresed on a SDS-polyacrylamide gel, electroblotted to nitrocellulose, probed with a polyclonal antibody against \u03b1-amylase and visualized with the ECL chemiluminescent system (Amersham).","It was observed that the export of active AmyKK was blocked. This was indicated by the absence of clear halos around cells grown on starch medium that was exposed to iodine vapor (). Immunoblot analysis of culture supernatants and cell extracts confirmed that the enzyme was synthesized in both cell types, but was secreted only in cells harboring wild-type \u03b1-amylase (). Approximately 90% of \u03b1-amylase activity was present in the culture supernatant of wild-type cells, whereas only cell-associated activity was observed for cells expressing AmyKK (data not shown), indicating that this halophilic \u03b1-amylase can fold inside the cytoplasm, and that the twin-arginine motif is indeed essential for its secretion.","A refined TATFIND program was used to look at all prokaryotic genomes available at NCBI. The recent identification of novel Tat substrates in (Ochsner et al., 2002) permitted the extension of the rules of TATFIND 1.1 to allow a methionine at position X, and glutamine at position X, thus creating a second program, TATFIND 1.2. Mutational analyses of certain Tat signal sequences suggested that in specific instances the substitution of lysine, asparagine, and glutamine for one of the two conserved arginines does not prevent Tat-dependent export However, only two naturally occurring Tat substrates are known to deviate from the conserved \u2018RR\u2019 motif in their signal sequence. Therefore, modifications of the program allowing for a variable RR motif due to these recent reports were not included in the present study, as these were likely to be exceptions that would lead to strong overprediction.","Tat Substrates\u2014TATFIND 1.2 and TATFIND 1.3","TATFIND 1.2 predicted that the Tat pathway is utilized to varying extents in the various organisms that were analyzed, based on the number and identity of their putative Tat substrates (Table 2). TATFIND 1.2 identified all previously-confirmed Tat substrates containing twin arginine motifs in and other prokaryotes (Jongbloed et al., 2000; Ochsner et al., 2002; Robinson et al., 2:350-356 (2001); Stanley et al., 183:139-144 (2001); Voulhoux et al., 2001) and predicted few or no substrates of this pathway in organisms which lack all Tat component homologs or contained only a TatA homolog (e.g., and AV19, respectively) (Table 2). Strikingly, in the analyses of the 29 prokaryotic genomes with zero or one identified Tat component(s), only 37 false positives were predicted (including cytoplasmic, membrane and secreted proteins), of which only 4 were putative secreted Sec substrates. Thus, the program is highly efficient in distinguishing Tat signal peptides from Sec signal peptides.",{"@attributes":{"id":"p-0112","num":"0116"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"294pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Predicted Tat Substrate and Component Numbers in a"},{"entry":"Diverse Group of Prokaryotes."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"196pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"# of","Tat"]},{"entry":[{},"TATFIND 1.2","Components"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"8"},"colspec":[{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"14pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Organism","Domain","Phylum","ORFs","Positives","A\/E","B","C"]},{"entry":{"@attributes":{"namest":"1","nameend":"8","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"8"},"colspec":[{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"char","char":"."}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"14pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{"i":"Streptomyces coelicolor"},"bac",{"i":"Actinobacteria"},"7899","145","1",{},"1"]},{"entry":"A3(2)"},{"entry":[{"i":"Mesorhizobium loti"},"bac",{"i":"Proteobacteria "},"7279","95","1","\u2014","1"]},{"entry":[{"i":"Sinorhizobium meliloti"},"bac",{"i":"Proteobacteria "},"6206","94","1","1","1"]},{"entry":[{"i":"Caulobacter crescentus"},"bac",{"i":"Proteobacteria "},"3737","88","1","1","1"]},{"entry":[{"i":"Ralstonia solanacearum"},"bac",{"i":"Proteobacteria "},"5116","71","1","1","1"]},{"entry":[{"i":"Halobacterium "},"archaea",{"i":"Euryarchaeota"},"2446","68","1","\u2014","2"]},{"entry":"NRC-1"},{"entry":[{"i":"Pseudomonas"},"bac",{"i":"Proteobacteria"},"5567","57","1","1","1"]},{"entry":[{"i":"aeruginosa"},{},"(gamma)"]},{"entry":[{"i":"Xanthomonas"},"bac",{"i":"Proteobacteria"},"4181","55","1","1","1"]},{"entry":[{"i":"campestris "},{},"(gamma)"]},{"entry":[{"i":"Agrobacterium"},"bac",{"i":"Proteobacteria "},"5299","51","1","1","1"]},{"entry":{"i":"tumefaciens"}},{"entry":[{"i":"Xanthomonas"},"bac",{"i":"Proteobacteria"},"4312","50","1","1","1"]},{"entry":[{"i":"axonopodis "},{},"(gamma)"]},{"entry":[{"i":"Escherichia coli "},"bac",{"i":"Proteobacteria"},"4279","34","2","1","1"]},{"entry":[{},{},"(gamma)"]},{"entry":[{"i":"Escherichia coli "},"bac",{"i":"Proteobacteria"},"5335","33","2","1","1"]},{"entry":["H7 EDL933",{},"(gamma)"]},{"entry":[{"i":"Salmonella"},"bac",{"i":"Proteobacteria"},"4559","33","2","1","1"]},{"entry":[{"i":"typhimurium "},{},"(gamma)"]},{"entry":[{"i":"Escherichia coli "},"bac",{"i":"Proteobacteria"},"5361","32","2","1","1"]},{"entry":["H7",{},"(gamma)"]},{"entry":[{"i":"Mycobacterium"},"bac",{"i":"Actinobacteria"},"3927","31","1","\u2014","1"]},{"entry":{"i":"tuberculosis "}},{"entry":[{"i":"Nostoc "},"bac",{"i":"Cyanobacteria"},"6129","31","2","\u2014","1"]},{"entry":[{"i":"Mycobacterium"},"bac",{"i":"Actinobacteria"},"4187","29","1","\u2014","1"]},{"entry":{"i":"tuberculosis "}},{"entry":[{"i":"Salmonella enterica"},"bac",{"i":"Proteobacteria"},"4768","28","2","1","1"]},{"entry":[{"i":"Typhi"},{},"(gamma)"]},{"entry":[{"i":"Deinococcus"},"bac",{"i":"Deinococcus-Thermus"},"2997","22","2","\u2014","1"]},{"entry":{"i":"radiodurans"}},{"entry":[{"i":"Synechocystis "},"bac",{"i":"Cyanobacteria"},"3167","21","2","\u2014","1"]},{"entry":"PPC6803"},{"entry":[{"i":"Yersinia pestis"},"bac",{"i":"Proteobacteria"},"4083","19","2","1","1"]},{"entry":[{},{},"(gamma)"]},{"entry":[{"i":"Brucella melitensis"},"bac",{"i":"Proteobacteria "},"3199","19","1","\u2014","1"]},{"entry":[{"i":"Xylella fastidiosa "},"bac",{"i":"Proteobacteria"},"2768","17","1","1","1"]},{"entry":[{},{},"(gamma)"]},{"entry":[{"i":"Aquifex aeolicus"},"bac",{"i":"Aquificae"},"1529","15","2","\u2014","1"]},{"entry":[{"i":"Corynebacterium"},"bac",{"i":"Actinobacteria"},"3041","15","2","\u2014","1"]},{"entry":{"i":"glutamicum"}},{"entry":[{"i":"Pyrobaculum"},"archaea",{"i":"Euryarchaeota"},"2605","14","1","\u2014","1"]},{"entry":{"i":"aerophilum"}},{"entry":[{"i":"Neisseria meningitidis"},"bac",{"i":"Proteobacteria "},"2065","12","1","1","1"]},{"entry":"Z2491"},{"entry":[{"i":"Pasteurella multocida"},"bac",{"i":"Proteobacteria"},"2015","12","1","1","1"]},{"entry":[{},{},"(gamma)"]},{"entry":[{"i":"Campylobacter jejuni"},"bac",{"i":"Proteobacteria"},"1654","11","1","1","1"]},{"entry":[{},{},"(epsilon)"]},{"entry":[{"i":"Neisseria meningitidis"},"bac",{"i":"Proteobacteria "},"2079","11","1","1","1"]},{"entry":"MC58"},{"entry":[{"i":"Haemophilus influenza"},"bac",{"i":"Proteobacteria"},"1714","9","1","1","1"]},{"entry":[{},{},"(gamma)"]},{"entry":[{"i":"Archaeoglobus fulgidus"},"archaea",{"i":"Euryarchaeota"},"2420","9","2","\u2014","2"]},{"entry":[{"i":"Mycobacterium leprae"},"bac",{"i":"Actinobacteria"},"2720","9","1","\u2014","1"]},{"entry":[{"i":"Vibrio cholerae"},"bac",{"i":"Proteobacteria"},"3835","7","2","1","1"]},{"entry":[{},{},"(gamma)"]},{"entry":[{"i":"Bacillus subtilis"},"bac",{"i":"Firmicutes"},"4112","7","3","\u2014","2"]},{"entry":[{"i":"Aeropyrum pernix"},"archaea",{"i":"Crenarchaeota"},"1840","7","2","\u2014","1"]},{"entry":[{"i":"Methanosarcina mazei"},"archaea",{"i":"Euryarchaeota"},"3371","6","2","\u2014","2"]},{"entry":"Goe1"},{"entry":[{"i":"Treponema pallidum"},"bac",{"i":"Spirochaetes"},"1036","6","\u2014","\u2014","\u2014"]},{"entry":[{"i":"Bacillus halodurans"},"bac",{"i":"Firmicutes"},"4066","5","2","\u2014","2"]},{"entry":[{"i":"Methanosarcina"},"archaea",{"i":"Euryarchaeota"},"4540","5","2","\u2014","2"]},{"entry":{"i":"acetivorans "}},{"entry":[{"i":"Sulfolobus solfataricus"},"archaea",{"i":"Crenarchaeota"},"2977","5","3","\u2014","2"]},{"entry":[{"i":"Chlorobium tepidum"},"bac",{"i":"Chlorobi"},"2252","5","2","\u2014","1"]},{"entry":"TLS"},{"entry":[{"i":"Pyrococcus horikoshii"},"archaea",{"i":"Euryarchaeota"},"1801","5","\u2014","\u2014","\u2014"]},{"entry":[{"i":"Sulfolobus tokodaii"},"archaea",{"i":"Crenarchaeota"},"2826","4","2","\u2014","1"]},{"entry":[{"i":"Helicobacter pylorii"},"bac",{"i":"Proteobacteria"},"1576","3","1","1","1"]},{"entry":["26695",{},"(epsilon)"]},{"entry":[{"i":"Helicobacter pylorii"},"bac",{"i":"Proteobacteria"},"1491","3","1","1","1"]},{"entry":["J99",{},"(epsilon)"]},{"entry":[{"i":"Clostridium perfringens"},"bac",{"i":"Firmicutes"},"2723","3","\u2014","\u2014","\u2014"]},{"entry":[{"i":"Pyrococcus furiosus"},"archaea",{"i":"Euryarchaeota"},"2065","3","\u2014","\u2014","\u2014"]},{"entry":"DSM3638"},{"entry":[{"i":"Thermotoga maritima"},"bac",{"i":"Thermotogae"},"1858","3","\u2014","\u2014","\u2014"]},{"entry":[{"i":"Listeria innocua"},"bac",{"i":"Firmicutes"},"3043","2","1","\u2014","1"]},{"entry":[{"i":"Staphylococcus aureus"},"bac",{"i":"Firmicutes"},"2714","2","1","\u2014","1"]},{"entry":"Mu50"},{"entry":[{"i":"Staphylococcus aureus"},"bac",{"i":"Firmicutes"},"2632","2","1","\u2014","1"]},{"entry":"MW2"},{"entry":[{"i":"Staphylococcus aureus"},"bac",{"i":"Firmicutes"},"2625","2","1","\u2014","1"]},{"entry":"N315"},{"entry":[{"i":"Thermoplasma"},"archaea",{"i":"Euryarchaeota"},"1482","2","1","\u2014","1"]},{"entry":{"i":"acidphilum"}},{"entry":[{"i":"Thermoplasma"},"archaea",{"i":"Euryarchaeota"},"1500","2","1","\u2014","1"]},{"entry":{"i":"volcanium"}},{"entry":[{"i":"Chlamydia trachomatis"},"bac",{"i":"Chlamydiae"},"895","2","\u2014","\u2014","\u2014"]},{"entry":[{"i":"Chlamydophila"},"bac",{"i":"Chlamydiae"},"1112","2","\u2014","\u2014","\u2014"]},{"entry":{"i":"pneumoniae "}},{"entry":[{"i":"Chlamydophila"},"bac",{"i":"Chlamydiae"},"1054","2","\u2014","\u2014","\u2014"]},{"entry":{"i":"pneumoniae "}},{"entry":[{"i":"Pyrococcus abyssi"},"archaea",{"i":"Euryarchaeota"},"1769","2","\u2014","\u2014","\u2014"]},{"entry":[{"i":"Listeria monocytogenes"},"bac",{"i":"Firmicutes"},"2846","1","1","\u2014","1"]},{"entry":"EGD-e"},{"entry":[{"i":"Rickettsia conoril"},"bac",{"i":"Proteobacteria "},"1374","1","1","\u2014","1"]},{"entry":[{"i":"Rickettsia prowazekii"},"bac",{"i":"Proteobacteria "},"835","1","1","\u2014","1"]},{"entry":[{"i":"Chlamydia muridarum"},"bac",{"i":"Chlamydiae"},"909","1","\u2014","\u2014","\u2014"]},{"entry":[{"i":"Chlamydophila"},"bac",{"i":"Chlamydiae"},"1069","1","\u2014","\u2014","\u2014"]},{"entry":{"i":"pneumoniae "}},{"entry":[{"i":"Clostridium"},"bac",{"i":"Firmicutes"},"3848","1","\u2014","\u2014","\u2014"]},{"entry":{"i":"acetobutylicum"}},{"entry":[{"i":"Lactococcus lactis"},"bac",{"i":"Firmicutes"},"2267","1","\u2014","\u2014","\u2014"]},{"entry":"subsp. "},{"entry":[{"i":"Methanothermo-bacter"},"archaea",{"i":"Euryarchaeota"},"1873","1","\u2014","\u2014","\u2014"]},{"entry":{"i":"thermauto-trophicus"}},{"entry":[{"i":"Streptococcus"},"bac",{"i":"Firmicutes"},"2043","1","\u2014","\u2014","\u2014"]},{"entry":{"i":"pneumoniae "}},{"entry":[{"i":"Streptococcus pyogenes"},"bac",{"i":"Firmicutes"},"1697","1","\u2014","\u2014","\u2014"]},{"entry":[{"i":"Streptococcus pyogenes"},"bac",{"i":"Firmicutes"},"1845","1","\u2014","\u2014","\u2014"]},{"entry":"MGAS8232"},{"entry":[{"i":"Thermoanaerobacter"},"archaea",{"i":"Euryarchaeota"},"2588","1","\u2014","\u2014","\u2014"]},{"entry":{"i":"tengcongensis"}},{"entry":[{"i":"Buchnera aphidicola"},"bac",{"i":"Proteobacteria"},"545","\u2014","\u2014","\u2014","\u2014"]},{"entry":[{},{},"(gamma)"]},{"entry":[{"i":"Buchnera "},"bac",{"i":"Proteobacteria"},"564","\u2014","\u2014","\u2014","\u2014"]},{"entry":[{},{},"(gamma)"]},{"entry":[{"i":"Fusobacterium"},"bac",{"i":"Fusobacteria"},"2067","\u2014","\u2014","\u2014","\u2014"]},{"entry":{"i":"nucleatum "}},{"entry":[{"i":"Methanococcus"},"archaea",{"i":"Euryarchaeota"},"1729","\u2014","\u2014","\u2014","\u2014"]},{"entry":{"i":"janaschii"}},{"entry":[{"i":"Methanopyrus kandleri"},"archaea",{"i":"Euryarchaeota"},"1687","\u2014","1","\u2014","\u2014"]},{"entry":"AV19"},{"entry":[{"i":"Mycoplasma genitalium"},"bac",{"i":"Firmicutes"},"484","\u2014","\u2014","\u2014","\u2014"]},{"entry":[{"i":"Mycoplasma"},"bac",{"i":"Firmicutes"},"689","\u2014","\u2014","\u2014","\u2014"]},{"entry":{"i":"pneumoniae"}},{"entry":[{"i":"Mycoplasma pulmonis"},"bac",{"i":"Firmicutes"},"782","\u2014","\u2014","\u2014","\u2014"]},{"entry":[{"i":"Streptococcus"},"bac",{"i":"Firmicutes"},"2094","\u2014","\u2014","\u2014","\u2014"]},{"entry":{"i":"pneumoniae "}},{"entry":[{"i":"Ureaplasma"},"bac",{"i":"Firmicutes"},"614","\u2014","\u2014","\u2014","\u2014"]},{"entry":{"i":"urealyticum"}},{"entry":[{"i":"Borrelia burgdorferi"},"bac",{"i":"Spirochaetes"},"1638","\u2014","\u2014","\u2014","\u2014"]},{"entry":{"@attributes":{"namest":"1","nameend":"8","align":"center","rowsep":"1"}}}]}}]}}},"Some organisms (e.g., and ) seem to have maintained the Tat pathway for the secretion of a small number of proteins. In fact, TATFIND 1.2 identified only one Tat substrate for and two for (Table 2). By comparison, other bacteria and archaea appear to make extensive use of this pathway, which originally was thought to be required for the translocation of only a minor subset of secreted proteins (Table 1). For example, 88, 94, and 145 putative Tat substrates were identified in , and , respectively (Table 2). While analyses of putative Sec substrates suggest that the haloarchaea remain the only organisms that use this pathway for the translocation of the majority of their secreted proteins, the present findings indicate that some prokaryotes use the Tat pathway for the secretion of as many as 20% of their extra-cytoplasmic proteins (see also, Bentley et al., 417:141-147 (2002)).","The diverse utilization of the Tat pathway observed in the 84 prokaryotes was also observed within phylogenetically-related groups. For example, a range of 9 () to 145 () putative Tat substrates were identified in the bacterial phylum Actinobacteria (Table 2). Similarly, the number of predicted Tat substrates were observed to vary widely within the phyla Proteobacteria, Cyanobacteria, Euryarchaeota, and Crenarchaeota (Table 2). Consequently, the degree to which the Tat pathway is used is apparently quite variable, even among related organisms.","To further characterize the utilization of the Tat pathway by the prokaryotes examined in this study, the function and localization of TATFIND 1.2 positive proteins of 4 organisms were identified by their annotation (or that of their homologs) in the SWISS-PROT database. Since the Tat pathway has been shown to play a role only in the transport of secreted proteins, known and putative multi-spanning membrane proteins (as predicted by TMHMM (Sonnhammer et al., 1998) and cytoplasmic proteins were excluded from further analyses. Proteins that were predicted to be secreted in were exclusively redox proteins, while the majority of putative Tat substrates of the gram-positive , the gram-negative and the archaeon sp. NRC-1 were found to be non-redox proteins (Table 3). A relatively large number of the TATFIND 1.2 positive non-redox proteins found in and sp. NRC-1 were identified as substrate-binding proteins, a phenomenon also predicted for the plant pathogen . Thus, while some organisms seem to preferentially use this pathway for the secretion of redox proteins (similar to , see Table 3), the Tat pathway is responsible for the secretion of a wider variety of substrates in other organisms.",{"@attributes":{"id":"p-0116","num":"0120"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Classification of TATFIND 1.2 positives"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Cytoplasmic\/",{}]},{"entry":[{},"Secreted","Integral","# annotated function"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"63pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Organism","Redox","Non-redox","Membrane","or localization"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"63pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{"i":"B. subtilis"},"1","2","0","4"]},{"entry":[{"i":"P. aerophilum"},"7","0","0","7"]},{"entry":[{"i":"E. coli "},"12","5","6","11"]},{"entry":[{"i":"Halobacterium"},"5","17","1","45"]},{"entry":"sp. NRC-1"},{"entry":[{"i":"C. crescentus"},"6","28","5","49"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},"For comparative purposes, only proteins whose function and putative localization could be determined (see text) were classified into redox, non-redox, and cytoplasmic\/integral membrane.","Certain protein homologs were identified as Tat substrates in many prokaryotes. Analyses of some redox proteins, such as the dimethyl sulfoxide reductase chain A (DmsA), found in many of the 84 organisms analyzed, revealed that the majority of the DmsA homologs in these prokaryotes contained Tat signal peptides (see supplemental webpage CD) where all proteins identified by TATFIND from all 84 genomes are listed on the webpage]. More surprisingly, however, virtually all homologs of certain non-redox proteins, such as alkaline phosphatase D and phospholipase C, were found to have typical Tat signal sequences (see, Dilks et al., 2003). As a result, there may exist an unknown selective pressure favoring the conserved targeting of these and other non-redox proteins to the Tat pathway.","An exception to the observed Tat motif conservation among phospholipase C homologs was found in . One of its two phospholipase C homologs was found to contain an aspartic acid residue at position X(a variation not accepted by the rules of TATFIND 1.2). While this phospholipase C homolog may in fact not be a Tat substrate, this single deviation may also reflect the existence of certain organism-specific Tat substrate characteristics. Possible organism specificity is also indicated by the 54 additional putative Tat substrates identified in when alanine is allowed at position Xas suggested by Ochsner et al., 2002. Alanine at this position, however, does not markedly affect predictions for virtually all other organisms (data not shown), although additional versions of TATFIND may be developed that are tailored to phylogenetically related groups of organisms.","Thus, identification of and other prokaryotic Tat substrates are most accurately achieved by using TATFIND 1.3, which differs from TATFIND 1.2 as it allows alanine at position X. The additional putative Tat substrates identified by TATFIND 1.3 in are listed below in Table 4, showing in a single representative species the additional sequences that can be identified simply by permitting alanine at position X. TATFIND 1.3 continues to rely upon Equation 1 ((X)RR(X)(X)(X)), which remains SEQ ID NO:1, and the rules remained the same, as do the hydrophobicity values, but the residues permitted at each X value were expanded to include a possibility of A at position X. Therefore, TATFIND 1.3 applies the motif wherein Xis M, H, A, P, K, R, N, T, G, S, D, Q, or E; wherein RRrepresent the twin arginines, wherein Xis A, P, K, R, N, T, G, S, D, Q, or E; wherein Xis I, W, F, L, V, Y, M, C, H, A, P, N, or T (positively charged residues were excluded from this position); and wherein Xis A, Q, I, L, V, M, or F. The possibility of harboring an alanine at this position of the Tat signal sequence has also been suggested by preliminary in vivo analyses of a Tat substrate.",{"@attributes":{"id":"p-0121","num":"0125"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 4"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Additional TATFIND 1.3 positives in 3(2) putative"},{"entry":"Tat substrates that were not found using TATFIND 1-2"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"133pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Gi number","Reference","Putative function or localization"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["21225458","NP_631237.1","Putative secreted amidase"]},{"entry":["21224914","NP_630693.1","Putative secreted protein"]},{"entry":["21224715","NP_630494.1","Putative gamma-glutamyltranspeptidase"]},{"entry":[{},{},"(putative secreted protein)"]},{"entry":["21224693","NP_630472.1","Putative lipoprotein"]},{"entry":["21224487","NP_630266.1","Putative secreted protein"]},{"entry":["21224144","NP_629923.1","Putative aminotransferase"]},{"entry":["21223819","NP_629598.1","Putative secreted protein"]},{"entry":["21223806","NP_629585.1","Putative ABC transporter"]},{"entry":["21223805","NP_629584.1","Putative neutral zinc metalloprotease"]},{"entry":["21223787","NP_629566.1","Putative integral membrane transport protein"]},{"entry":["21223681","NP_629460.1","Polyketide beta-ketoacyl synthase alpha"]},{"entry":["21223627","NP_629406.1","Secreted protein"]},{"entry":["21223309","NP_629088.1","Putative integral membrane protein"]},{"entry":["21222967","NP_628746.1","Putative membrane protein"]},{"entry":["21222748","NP_628527.1","Putative integral membrane protein"]},{"entry":["21222646","NP_628425.1","Putative secreted protein"]},{"entry":["21222540","NP_628319.1","Phosphate-binding protein precursor"]},{"entry":["21222270","NP_628049.1","Putative membrane protein"]},{"entry":["21222223","NP_628002.1","Putative membrane protein"]},{"entry":["21222221","NP_628000.1","Putative D-alanyl-D-alanine"]},{"entry":[{},{},"carboxypeptidase"]},{"entry":["21221844","NP_627623.1","Putative ABC transporter transmembrane"]},{"entry":[{},{},"component"]},{"entry":["21221843","NP_627622.1","Putative glutamate decarboxylase"]},{"entry":["21221454","NP_627233.1","Putative lipoprotein"]},{"entry":["21221288","NP_627067.1","Putative secreted endoglucanase"]},{"entry":["21221278","NP_627057.1","Probable amino acid ABC transporter"]},{"entry":[{},{},"protein, solute-binding component"]},{"entry":["21221218","NP_626997.1","Putative secreted ribonuclease"]},{"entry":["21221214","NP_626993.1","Putative secreted protein"]},{"entry":["21221169","NP_626948.1","Putative secreted protein"]},{"entry":["21221157","NP_626936.1","Putative secreted protein"]},{"entry":["21220742","NP_626521.1","FecCD-family membrane transport protein"]},{"entry":["21220739","NP_626518.1","Putative membrane protein"]},{"entry":["21220703","NP_626482.1","Putative maltose-binding protein"]},{"entry":["21220616","NP_626395.1","Putative integral membrane protein"]},{"entry":["21220523","NP_626302.1","Putative membrane protein"]},{"entry":["21220426","NP_626205.1","Putative secreted protein"]},{"entry":["21220393","NP_626172.1","Putative secreted protein"]},{"entry":["21220369","NP_626148.1","Putative secreted protein"]},{"entry":["2122313","NP_626092.1","Secreted subtilisin-like protease"]},{"entry":["21220235","NP_626014.1","Putative secreted serine protease"]},{"entry":["21220228","NP_626007.1","Putative secreted cellulose-binding protein"]},{"entry":["21220103","NP_625882.1","Putative membrane protein"]},{"entry":["2122007","NP_625850.1","Putative oxidoreductase"]},{"entry":["21219820","NP_625599.1","Putative integral membrane protein"]},{"entry":["21219739","NP_625518.1","Putative secreted tripeptidylaminopeptidase"]},{"entry":["21219565","NP_625344.1","Putative secreted oxidoreductase"]},{"entry":["21219513","NP_625292.1","Putative lipoprotein"]},{"entry":["21219470","NP_625249.1","Putative solute-binding protein"]},{"entry":["21219086","NP_624865.1","Putative histidine kinase protein"]},{"entry":["21219051","NP_624830.1","Putative lipoprotein"]},{"entry":["21219035","NP_624814.1","Putative iron-siderophore permease"]},{"entry":[{},{},"transmembrane protein"]},{"entry":["21219021","NP_624800.1","Putative secreted protein"]},{"entry":["21219020","NP_624799.1","Putative secreted chitin binding protein"]},{"entry":["21234268","NP_639879.1","Putative secreted protein"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Tat Components in Sequenced Prokaryotes","Expanding on previous analyses by others of the Tat pathway, the presence of TatA, TatB and TatC was evaluated in all organisms analyzed using PSI-BLAST and its iterations (Table 2 and Altschul et al., 25:3389-3402 (1997)). Detailed phylogenetic analyses revealed that all bacteria in the obligate intracellular pathogen phylum Chlamydiae completely lacked Tat components. Conversely, the presence of the Tat machinery was conserved in all crenarchaeota, actinobacteria, cyanobacteria, and proteobacteria (with the exception of the obligate symbiotic genus ) that were examined, even though the extent to which this machinery was utilized varied dramatically within each phylum (Table 2). Interestingly, homologs of TatB, found exclusively in the proteobacteria, were missing from only a few organisms in the \u03b1-subdivision (Table 2).","In the archaeal phylum euryarchaeota and the bacterial phylum firmicutes, approximately half of the organisms analyzed had components of the Tat machinery. However, a number of organisms from these phyla, unlike the proteobacteria or actinobacteria, were found to contain multiple copies of TatC (Table 2; Yen et al., 2002). It is intriguing that the majority of the firmicutes and euryarchaeota were predicted to have a relatively small number of putative Tat substrates, which suggests that in prokaryotes there is no direct correlation between the number of Tat substrates and TatC homologs. Furthermore, an additional examination of the 84 proteomes demonstrated that the number of TatA homologs or the presence of a TatB homolog in any of these organisms is also unrelated to the number of Tat substrates. This notion was underscored by the observation that contains three TatA and two TatC homologs, yet has only seven putative Tat substrates (Table 2).","To determine the role and efficiency of a , the preferred organism was because as an expression host include 1) it does not secrete a blue pigment like , and 2) it does not require non-methylated DNA. However, the genome for has not yet been completely sequenced. Therefore, , was selected for the following analysis because it is highly homologous to , its genome has been fully sequenced, permitting a TATFIND analysis of the genome, and complete characterization of the genomic information for the strain was advantageous for the analysis. It is intended that the findings in are also applicable to the closely related , as well as to other strains.","Accordingly in vivo analyses were initiated of the Tat pathway in . A His-tagged version of the Tat C components were cloned into a integration vector and expressed in its wild-type, native host. An integration vector is one that when transformed into will recombine onto the chromosome at a specific site. Consequently there is not an independent plasmid in the cytoplasm, which could otherwise become problematic. In addition, upstream and downstream parts of the TatC were cloned and an apramycin resistance gene was inserted between these regions into a conjugative plasmid, wherein DNA is transferred directly from one bacterium () to another\u2014. This construct is used for the construction of a TatC knockout (see Schaerlaekens, 2001). Methods for conjugation and recombination are described, for example, in the Kieser manual.","Identification of the Role of the Tat Pathway in ","Computational analyses of the genome revealed that a large number of secreted proteins contain a typical Tat signal sequence. Therefore, in vivo analyses are conducted to complement the TATFIND results and to define specific Tat signal sequences.",{"@attributes":{"id":"p-0129","num":"0133"},"i":"S. coelicolor "},{"@attributes":{"id":"p-0130","num":"0134"},"i":["S. lividans ","E. coli ","S. lividans ","Streptomyces ","S. coelicolor ","S. coelicolor","S. coelicolor "]},"The knockout is constructed in a fashion similar to that used for the construction of the TatC. Briefly, a plasmid containing approximately 800 bp of tatC flanking regions interrupted by an apramycin resistance gene that have been cloned into an vector containing a second resistance gene (spectinomycin) is transformed into . Apramycin resistant colonies that do not confer resistance to spectinomycin are screened by replica-plating to identify cells in which the tatC has been interrupted.","Identification of Putative Tat Substrates","Although it is believed that uses the Tat pathway extensively, it has been demonstrated that the , TatC is not essential for growth on rich medium. Given the close relationship between and , it is therefore concluded that the Tat pathway is also not essential and that both strains are viable when grown in rich medium. After growing the strains to mid or late log phase or stationary phase supernatant fractions of wild-type (wt) and tatC knockout strains are collected, and two dimensional (2D) gel electrophoresis is performed on the concentrated extracellular fraction. Proteins bands present in gels containing supernatant fractions from (wt) strains, but that are absent or present in significantly lower concentrations in the gels containing the supernatant fractions of the mutant knockout strains, are putative Tat substrates that failed to be secreted via the Tat pathway. Matrix-Assisted Laser Desorption\/Ionization Mass Spectrometry (MALDI-MS) is used to identify the masses of each tryptic fragment (the cleavage product of a proteins that has been treated with a protease) and will be compared to the calculated masses of tryptic fragments of all proteins.","This is a preferred method for identifying proteins from sequenced genomes, since it is rapid and, more importantly, requires significantly lower amounts of the pure protein than other methods (e.g., N-terminal sequencing). However, in cases in which post-translational modifications would prevent this method of detection, purified proteins obtained in co-purification experiments can also be submitted for internal sequencing, mass spectroscopy (MS) analysis or LC-MS-MS (e.g., Quadrupole Time-Of-Flight (Q-TOF) MS), which can produce partial amino acid sequence data from silver-stained proteins. The resulting partial amino acid sequences are compared with the genome database or the National Center for Bioinformatics (NCBI) databases to determine their sequence homology with previously-characterized proteins.","Once identified, the genes are cloned into shuttle vectors that can replicate in both and . The genes are expressed from either their own promoters or from a promoter already cloned into the shuttle vectors. This approach not only permits verification of TATFIND predicted Tat substrates, but it is intended to also identify additional Tat substrates. For example, as mentioned above, it is likely that permitting an additional \u201cA\u201d in position Xin Equation 1 of the TATFIND program, will add an additional 50 Tat substrates to the list, making them available for identification by this in vivo approach.","Signal sequences of proteins that were identified by TATFIND, but which could not be identified by the approach described above are then analyzed. The protein is expressed with a histidine (His)- or Flag-tag in the wild-type and mutant strains, and the secretion pattern observed. If both strains secrete the protein with similar efficiencies, it is likely that this motif, while a Tat motif in other organisms, is not recognized by the Tat pathway. A control Sec substrate is included to exclude the possibility that a Tat mutant is indirectly affecting Sec translocation.","These analyses permit the identification of a large number of bona fide Tat substrates in and provide more definitive information concerning a typical Tat signal sequence. Similarly, these steps can be applied to other species and strains of prokaryotes to determine the Tat substrates, if available.","Secretion Kinetics of Expressed Homologous and Heterologous Proteins Via the Tat Pathway","Determination of the translocation efficiency of Tat substrates. Protein translocation of Tat substrates across the cytoplasmic membrane via the Tat pathway is very inefficient. However, this Gram negative bacterium secretes only a minor subset of its secretome via this pathway, and most of the proteins are cofactor-containing proteins. In contrast, the present data suggest that secretes more than 200 proteins of various classes via this pathway. It is thus likely that has optimized its Tat translocation system for the efficient secretion of a diverse group of substrates.","To determine the translocation efficiency of Tat substrates using the Tat pathway, a number of substrates are selected, including (i) different classes of proteins (binding-protein, redox-protein, hydrolases), and (ii) proteins with various Tat signal sequences that contain distinct amino acid compositions, varying lengths of the signal sequence, and\/or varying lengths of the uncharged region. The proteins are fused with a C-terminal tag (e.g., 6\u00d7His, Flag, myc), and cloned into a intergration or expression vector (pIJ101 derivatives, such as pBM6) under the control of their own or heterologous promoters, e.g., like the inducible tip promoter. Pulse-chase experiments are conducted to determine the kinetics of protein secretion. These analyses are intended to indicate the efficiency of protein translocation across the cytoplasmic membrane, and potentially indicate preferred signal peptide sequences.","Highly efficient production of heterologous proteins fused to a Sec signal sequence in has previously been reported Chang et al. (in Biology of Actinomycetes '88, Proc. of Seventh Internat'l Symposium on Biology of Actinomycetes (Okami, Beppu and Ogawara, eds.), Japanese Sci. Soc. Press, Tokyo, p. 103 (1988)), Rowland et al. (38(1):94-100 (1992)). However, this will only work for proteins that fold after secretion, and that will fold correctly in the extracellular environment. Therefore, the identified signal sequences are fused to a protein of biotechnological interest (heterologous protein, e.g., TNF, interferon gamma) intended to be secreted in a folded conformation via the Tat translocation pathway. As mentioned above, increased production of TNF fused to a putative Tat signal sequence compared to a Sec signal sequence in has been reported, albeit without any knowledge that the effect was a result of the Tat pathway (Chang et al., 1988). While the mature part of the protein might be a crucial determinant for efficiency of translocation, signal sequences are chosen for proteins that have been shown to be secreted efficiently (see above).","If western blot analyses of cell and supernatant fractions indicate efficient secretion of the fusion construct, the kinetics of protein translocation is determined using pulse-chase analyses as described above. Activity assays specific for the expressed proteins are carried out to determine whether the proteins were correctly folded in the cytoplasmic fraction. Also the efficiency of signal sequence targeting is tested to determine substrate specificity. Finally, because has not been used extensively for the production of heterologous proteins, the expression of the fusion proteins are compared in other Streptomycetes, like , to determine a good expression host.","Tat Signal Sequence Mutagenesis","While the analyses of various signal sequences for their efficiency of protein translocation provides general characteristics of signal sequences and information regarding secretion efficiency, this approach is not effective for screening for identifying the most efficient signal sequences, although it definitely provides the foundation for further analyses of the Tat translocation system. Thus, screens are further developed to permit mutagenisis of the Tat signal sequences and to identify the most effective ones for targetingheterologous proteins to and translocating across the cytoplasmic membrane of . For this purpose, a screen is employed that is similar to that used to identify the more efficient of the Tat signal sequences in ","Briefly, in , mutagenized Tat signal sequences are fused to a green fluorescent protein (GFP), that in turn is fused to a cytoplasmic protease recognition site. Fusion proteins that are secreted into the periplasm are stable, whereas proteins retained in the cytoplasm are degraded. The fluorescence intensity of cells is determined by flow cytometry. Because are Gram positive, they do not have a periplasm. Therefore, secreted proteins are released into the external environment. The preferred screen does not require a protease site on the GFP, rather the fluorescence is monitored. Accordingly, a decrease in fluorescence indicates more efficient secretion of the fusion protein into the external environment, permitting optimization of the secretion of fully-folded heterologous proteins by the Tat pathway. To eliminate signal sequence mutations that result in decreased protein production, pulse-chase analyses with signal sequence mutant constructs are used to determine secretion kinetics.","Thus, the present invention provides a method for identifying signal sequences that efficiently target heterologous proteins to the Tat translocon, and to extensively characterize the Tat machinery in order to achieve high yields of active heterologous proteins allowing for their economic production.","The disclosures of each patent, patent application and publication cited or described in this document are hereby incorporated herein by reference, in their entirety.","While the foregoing specification has been described with regard to certain preferred embodiments, and many details have been set forth for the purpose of illustration, it will be apparent to those skilled in the art without departing from the spirit and scope of the invention, that the invention may be subject to various modifications and additional embodiments, and that certain of the details described herein can be varied considerably without departing from the basic principles of the invention. Such modifications and additional embodiments are also intended to fall within the scope of the appended claims."],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE FIGURES","p":["The foregoing summary, as well as the following detailed description of the invention, will be better understood when read in conjunction with the appended figures, which are not intended to be limiting.",{"@attributes":{"id":"p-0024","num":"0023"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0025","num":"0024"},"figref":["FIGS. 2A and 2B","FIG. 2A","FIG. 2B"],"i":["H. volcanii ","H. volcanii "]}]},"DETDESC":[{},{}]}
