---
title: Fluorescence polarization method for determining viral inhibitors
abstract: Methods for determining inhibitors of protein interactions are provided. The method involves evaluation of potential inhibitors that can inhibit or prevent protein interactions. The method provides for high-throughput identification of novel therapeutics that can treat a disease or disorder by inhibiting protein interactions.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=07132231&OS=07132231&RS=07132231
owner: President and Fellows of Harvard College
number: 07132231
owner_city: Cambridge
owner_country: US
publication_date: 20031113
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["RELATED APPLICATIONS","STATEMENT OF GOVERNMENT INTERESTS ","FIELD OF THE INVENTION","BACKGROUND OF THE INVENTION","SUMMARY","DETAILED DESCRIPTION OF CERTAIN PREFERRED EMBODIMENTS","EXAMPLE 1","Screening of a Library of Compounds Against Herpes Simplex Virus DNA Polynerase Fragments"],"p":["This application is a continuation of PCT application No. PCT\/US02\/15878, designating the United States and filed on May 20, 2002; which claims the benefit of U.S. Provisional Application No. 60\/291,901 filed on May 18, 2001, the entire disclosures of which are incorporated herein by reference for all purposes.","This invention was made with Government support under Grant Number AI26077 awarded by the National Institutes of Health. The Government has certain rights in the invention.","The present invention relates to methods and kits for screening potential inhibitors of protein interactions. More specifically, the present invention relates to screening of potential compounds to inhibit herpes simplex virus (HSV) DNA polymerase subunit interactions.","The mechanism of action for many drugs is to inhibit one or more enzymes of an organism or cell involved in a disease or disorder. The vast majority of small-molecule pharmaceuticals act by inhibiting enzymes or membrane-bound receptors. These protein targets have naturally selected pockets that enable them to bind to small molecules, e.g. typically they bind small molecules at the active site of the enzyme. The increased use of current therapeutics, however, has led to constant emergence of drug resistant organisms.","Within the past few years it has become apparent that coordinated protein-protein interactions play a major role in cellular, viral or disease processes. However, the search for drugs that block protein-protein interactions has yet to be successful for wide range applications.","It is an object of the present invention to provide a method for identifying potential inhibitors which can treat a disorder by inhibiting, disrupting, or altering protein:protein interactions. In certain preferred embodiments, the method comprises identifying compounds that can inhibit herpes simplex virus by altering or disrupting the interaction between the catalytic subunit and the processivity subunit of herpes simplex virus DNA polymerase.","Embodiments of the present invention are directed to methods for identifying compounds effective to dissociate, inhibit or otherwise interfere with protein binding interactions. Using the novel methods, one can identify compounds that can be used to inhibit the binding activity of certain proteins or protein subunits the binding of which are known to be causative of certain disease or viral conditions. Alternatively, promising lead candidate compounds can be identified by the present invention based upon an ability to interact with proteins or protein subunits known to bind to one another. Such compounds themselves can be therapeutically useful or they can serve as lead compounds to be further developed into pharmacologically active compounds or formulations.","According to the present invention, a test sample including a known binding pair of proteins or protein subunits including a fluorescent label is analyzed by fluorescence polarization. The test sample is then contacted with a candidate inhibitor compound and the fluorescence polarization is then determined. The ability of the compound to cause dissociation of or otherwise interfere with or prevent binding of the proteins or protein subunits is monitored by fluorescence polarization.","Embodiments of the present invention advantageously utilize a test sample including fragments of subunits of proteins known to bind to one another. The test sample includes at least a fragment of a first subunit and at least a fragment of a second subunit. The fragment of the first subunit and the fragment of the second subunit are capable of interacting to form a dimer and, according to one embodiment, when contacted within the sample, bind to one another. Either the fragment of the first subunit or the fragment of the second subunit includes a fluorescent label. The fluorescence polarization of the bound first and second subunits in the test sample is determined. At least one candidate test compound is then contacted with the test sample to form a test mixture. Fluorescence polarization of the test mixture is evaluated to determine if the test compound has disrupted interaction between, or otherwise interfered with binding between the fragment of the first subunit and the fragment of the second subunit. In accordance with one aspect, fluorescence polarization of the test mixture is compared to fluorescence polarization of the test sample. A decrease in the fluorescence polarization of the test mixture, when compared to the fluorescence polarization of the test sample, indicates the first fragment is no longer bound to the second fragment.","In accordance with an additional aspect, a method of testing compounds for inhibiting herpes simplex virus DNA polymerase subunit interactions is provided. The method includes providing a test sample which includes a peptide which is identical to or substantially homologous to an eighteen amino acid C-terminal fragment of catalytic unit of herpes simplex virus DNA polymerase. The C-terminal fragment of catalytic unit of herpes simplex virus DNA polymerase also includes a fluorescent label. The test sample further includes a functional fragment of processivity subunit of herpes simplex virus DNA polymerase. The test sample is combined with at least one test compound to form a test mixture. Fluorescence polarization of the test mixture is evaluated to determine if the test compound has disrupted or otherwise interfered with interaction of the C-terminal fragment and the functional fragment of processivity subunit. It will be recognized by those skilled in the art given the benefit of this disclosure that fluorescence polarization measurements can typically be correlated to the level of inhibition of the herpes simplex virus DNA polymerase.","In accordance with another aspect, a method for testing compounds to inhibit herpes simplex virus DNA polymerase subunits is provided. The method includes providing a test sample including a peptide including an amino acid sequence as shown in SEQ ID NO.:1 below. The peptide also includes a fluorescent label. The test sample further includes a protein fragment including an amino acid sequence as shown in SEQ ID NO.:2, and the peptide and the protein fragment are capable of interacting to form dimer. The test sample is combined with at least one test compound to form a test mixture. Fluorescence polarization of the test mixture is evaluated to determine if the at least one test compound has disrupted or otherwise interfered with interaction between the peptide and the protein fragment.","In accordance with another aspect, a kit for testing inhibitors of protein interactions is disclosed. The kit includes at least a first reagent which includes a fragment of a first subunit. The first reagent typically also includes a fragment of a second subunit. The fragment of the first subunit and the fragment of the second subunit are capable of interacting to form a dimer. The fragment of the first subunit also includes a fluorescent label.","In accordance with an additional aspect, a kit for testing compounds for inhibiting herpes simplex virus is disclosed. The kit includes at least a first reagent which includes a peptide that is identical to or substantially homologous to an eighteen amino acid C-terminal fragment of the catalytic unit of herpes simplex virus DNA polymerase. The C-terminal fragment also typically includes a fluorescent label. The first reagent typically also includes a fragment of processivity subunit of herpes simplex virus DNA polymerase. According to one embodiment, the reagent includes a peptide which includes an amino acid sequence as shown in SEQ ID NO.:1 and a protein fragment which includes an amino acid sequence as shown in SEQ ID NO.:2.","In accordance with an additional aspect, a composition for treating herpes simplex virus is disclosed. The composition includes a compound of formula I, or pharmaceutically suitable salt or solvate thereof.",{"@attributes":{"id":"p-0016","num":"0015"},"chemistry":{"@attributes":{"id":"CHEM-US-00001","num":"00001"},"img":{"@attributes":{"id":"EMI-C00001","he":"99.65mm","wi":"53.68mm","file":"US07132231-20061107-C00001.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{},"sub":["1 ","2 ","2","2"],"figref":"FIG. 2"},"In accordance with another aspect, a composition for treating herpes simplex virus is disclosed. The composition includes a compound of formula II, or pharmaceutically suitable salt or solvate thereof.",{"@attributes":{"id":"p-0018","num":"0017"},"chemistry":{"@attributes":{"id":"CHEM-US-00002","num":"00002"},"img":{"@attributes":{"id":"EMI-C00002","he":"72.81mm","wi":"61.21mm","file":"US07132231-20061107-C00002.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{},"sub":["1 ","2 ","2","2"],"figref":"FIG. 2"},"In accordance with an additional aspect, a method of treating herpes simplex virus in a mammal, e.g. a human, is provided. The method includes administering a pharmaceutically effective amount, e.g. an efficacious amount, of a compound of formula I or formula II, or pharmaceutically suitable salts or solvates thereof, to the mammal.","It will be recognized by those skilled in the art given the benefit of this disclosure, that certain preferred embodiments of the present invention provide for high-throughput testing of potential inhibitors of protein interactions. The robust methods and kits disclosed here can be employed to test large libraries of compounds in a rapid and cost-effective manner to develop therapeutics which target, e.g. inhibit, protein interactions.","It is a significant advantage that the methods and kits disclosed herein provide for screening of compounds that can dissociate, disrupt, inhibit or otherwise interfere with binding interactions between proteins or protein subunits or fragments thereof. Inhibition of such protein interactions can provide for treatment of numerous disorders by halting growth or metabolic functions of the disease causing organisms. Such testing can lead to drug candidates for treating the disease or disorder in which the protein is involved. Though the examples disclosed here are generally directed towards testing of inhibitors of viral DNA polymerases, one skilled in the art given the benefit of this disclosure will be readily able to adapt the methods disclose herein for these and other intended uses. For example, even though the methods disclosed herein are suitable for identifying compounds which can inhibit protein interactions to inhibit activity of the protein, the methods are equally applicable to disruption of protein interactions to activate a protein. For example, in equation I below interaction of subunits Aand Bresults in an active protein. When this interaction is disrupted by inhibitor 1 (I), the subunits dissociate and the protein is rendered inactive, e.g. activity is inhibited.",{"@attributes":{"id":"p-0031","num":"0030"},"chemistry":{"@attributes":{"id":"CHEM-US-00003","num":"00003"},"img":{"@attributes":{"id":"EMI-C00003","he":"7.20mm","wi":"64.43mm","file":"US07132231-20061107-C00003.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{},"sub":["2 ","2 ","2"]},{"@attributes":{"id":"p-0032","num":"0031"},"chemistry":{"@attributes":{"id":"CHEM-US-00004","num":"00004"},"img":{"@attributes":{"id":"EMI-C00004","he":"7.20mm","wi":"64.09mm","file":"US07132231-20061107-C00004.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{}},"The following terms are intended to have the following meanings unless otherwise clear from the context of usage.","As used here \u201cprotein interaction\u201d or \u201cprotein:protein interaction\u201d refers to the interaction of different subunits, or portions or fragments thereof, of a protein. That is, protein interaction refers to the interaction between at least a portion of two subunits which comprise a functional protein. In certain embodiments, protein interaction refers to an interaction between two or more distinct proteins which interact to form a functional and\/or non-function protein. For example, in embodiments where a first protein interacts with a second protein to inhibit the activity of the second protein (or activate the second protein as the case may be), the protein interaction typically occurs through weak forces, such as hydrogen bonding, salt bridges, Van der Waals forces, London dispersion forces and the like. Preferably, the protein interaction is reversible such that dissociation of the proteins, or protein subunits, can occur under suitable conditions. Preferably, such forces are weak, e.g. have K's in the \u03bcM range, such that a compound having suitable properties can disrupt the interaction between the two subunits. As used here \u201cdisrupt\u201d refers to breaking of, weakening of or interfering with, as the case may be, the interactions between the fragments and\/or subunits in the test sample.","As used here \u201ctest sample\u201d refers to a sample comprising two or more proteins, peptides or fragments thereof (collectively, \u201cfragments\u201d) which can interact under suitable conditions. That is, the fragments have the ability to interact with each other, e.g. through Van der Waals interactions, hydrogen bonds, hydrophobic interactions, and the like, under suitable conditions. In certain preferred embodiments, the fragments can interact to form a functional protein, or a functional portion of a protein. For example, in the case of many viral polymerases, a fragment of a catalytic subunit (or the entire catalytic subunit as the case may be) can interact with a fragment of processivity subunit (or the entire processivity subunit as the case may be) to form a functional protein, e.g. a functional polymerase capable of synthesizing viral DNA. It will be understood by those skilled in the art given the benefit of this disclosure that a sample, aliquot or portion of the test sample can be used to form test mixtures. That is, the entire test sample need not be combined with a test compound to form the test mixture, but, instead, only a suitable volume of test sample is necessary.","As used here \u201cat least one test compound\u201d refers to one or more compounds which have the potential of disrupting protein interactions. The test compound preferably is dissolved in a suitable solvent, e.g. buffer, such that the test compound can be added to the test sample to form a test mixture. In accordance with certain preferred embodiments, the test compounds may be selected based on numerous criteria, such as, for example, solubility, physical properties, structural similarity to compounds which are known or thought to be inhibitors of protein interactions, etc. In certain embodiments, the compounds are randomly selected from a library, e.g. a combinatorial library, and tested using the methods disclosed here. Suitable methods for synthesizing combinatorial libraries are known to those skilled in the art, and suitable combinatorial library members for testing using the methods disclosed here will be readily apparent to those skilled in the art given the benefit of this disclosure. Preferably, the number of members of the library is high, e.g. 10, 10, 10, 10, 10, 10or greater. In certain preferred embodiments, the test compounds are assayed individually to determine if the compound can inhibit protein interactions. In other embodiments, the test compounds are assayed in groups to determine if any member of the group can inhibit protein interactions. Such group testing allows for removal of large groups of compounds that do not inhibit protein interactions. For example, a group comprising 100 members of a library which does not inhibit protein interactions allows for elimination of all 100 members of the group in a single assay, whereas individual testing of the compounds would require 100 separate assays. One skilled in the art given the benefit of this disclosure will be able to design suitable assays for testing large numbers of compounds using the methods disclosed here.","In accordance with certain preferred embodiments, the test compounds are evaluated for their ability to inhibit protein interactions by measuring fluorescence polarization of the test mixture(s). Without wishing to be bound by any particular scientific theory, fluorescence polarization measurements allow for discrimination between fluorescently labeled bound and unbound proteins, peptides, subunits or fragments thereof, for example the first labeled fragment and the second fragment. Again without wishing to be bound by any particular scientific theory, the fluorescence polarization of the fluorescently labeled first fragment of first subunit is small because the fluorescently labeled first fragment rotates rapidly in solution and, therefore, has randomized photoselected distributions which result in the small observed fluorescence polarization. When the fluorescently labeled first fragment of the first subunit interacts with the fragment of the second subunit, which typically is a larger more slowly rotating molecule, the rotation of the fluorescently labeled first fragment slows and the fluorescence polarization increases dramatically. Accordingly, disruption of the subunit interaction by a test compound provides a decrease in the fluorescence polarization which is indicative of inhibition of the protein interactions. One skilled in the art given the benefit of this disclosure will be able to evaluate test compounds for their ability to disrupt protein:protein interaction using the methods disclosed herein.","In accordance with certain preferred embodiments, the fluorescence polarization measurements in the presence of at least one test compound can be compared with the fluorescence polarization measurements in the absence of the test compound. Depending on the assay method selected, such comparison can be made manually by the operator, e.g. by subtraction, or can be made automatically by a computer, for example. In certain preferred embodiments where high-throughput assays are used, using a 384 well plate, for example, one or more wells, or positions in an array, for example, may comprise only a sample of test sample to provide a reference or baseline for the measurements. These wells can act as a baseline for which all other wells, each of which may comprise a test mixture, can be compared. Accordingly, a difference in fluorescence polarization between the wells comprising test sample and wells comprising test mixture indicates that the test compound has disrupted the protein interactions in some manner. For example, when a test compound disrupts the interaction between fluorescently labeled first fragment of the first subunit and first fragment of second subunit, the fluorescence polarization typically decreases as the fluorescently labeled first fragment of the first subunit dissociates, e.g. stops interacting with, from the first fragment of second subunit. One skilled in the art given the benefit of this disclosure will be able to select suitable methods for comparing the fluorescence polarization of a test mixture with fluorescence polarization of a reference sample.","In accordance with certain preferred embodiments, at least one of the fragments of the test mixture typically includes a fluorescent label. The fluorescent label typically can be any fluorescent label which can be attached to a fragment, and preferably the fluorescent label is attached to a smaller fragment. Preferably, the fluorescent label does not interfere with the interaction between the fragments and\/or subunits. That is, preferably the fluorescent label does not alter the interaction between the components of the test sample, e.g. the first subunit and the second subunit. In certain preferred embodiments, the fluorescent label has a high quantum yield, e.g. greater than 0.5, more preferably greater than 0.75 and most preferably greater than 0.9, such that fluorescence measurements can be readily made. In certain preferred embodiments, the fluorescent label is selected from pentafluorofluorescein-derivative Oregon Green 514, Oregon Green 488, BODIPY fluorescent dye, tetramethylrhodamine dyes (each from Molecular Probes), and other labels suitable for performing fluorescence polarization measurements. Other suitable labels will be readily apparent to those skilled in the art given the benefit of this disclosure.","As used here \u201ctest mixture\u201d refers to the combination of at least one test compound and the test sample or portion or aliquot thereof. The test mixture may further comprise other suitable components, e.g. buffers, salts, organic solvents, proteins and the like, to facilitate protein interactions and\/or evaluation of the test mixture. In accordance with certain preferred embodiments, the test mixture used in the method disclosed here typically is formed by adding at least one test compound to a test sample. One skilled in the art given the benefit of this disclosure will recognize that a sample of the test sample could be added to the test compound to form a test mixture. That is, as long as the test sample and the test compound are combined, e.g. placed in the same solution, a test mixture suitable for performing the methods disclosed here will be formed. The test compound can be added to a sample of the test sample using numerous methods and apparatus. In embodiments comprising assays performed in test tubes or Eppendorf tubes, for example, the test compound can be added using conventional techniques such as a syringe, pipette, micropipette and the like, for example. In embodiments comprising high-throughput assays, the test compound(s) is preferably added using automated techniques, such as autoloaders, automatic pipettes, or arrays, for example. In certain preferred embodiments, a multi-well plate, e.g. a plate having 96 or 384 wells, is used to test a plurality of samples simultaneously. Use of a multi-well plate, along with a plate reader which is configured to read a plurality of the wells in a rapid manner, allows a significant number of test compounds to be assayed efficiently and rapidly for their ability to inhibit protein interactions by using the methods disclosed here. In certain embodiments, a plurality of test compounds are fixed in an array and a test sample is added to each position on the array. Such arrays provide for high-throughput testing of a plurality of test compounds. One skilled in the art given the benefit of this disclosure will be able to design suitable high-throughput assays for testing a large number of test compounds.","As used here \u201cSEQ ID NO.: 1\u201d refers to the following amino acid sequence: Ac-Ala-Thr-Ala-Glu-Glu-Glu-Thr-Arg-Arg-Met-Leu-His-Arg-Ala-Phe-Asp-Thr-Leu-Ala-NH, which is also referred to in the Example as peptide E. The amino acid sequence of peptide E (SEQ ID NO.:1) can be found in Digard P, Williams KP, Hensley P, Brooks IS, Dahl CE, Coen DM. \u201cSpecific inhibition of herpes simplex virus DNA polymerase by helical peptides corresponding to the subunit interface. Proc Natl Acad Sci USA 1995; 92:1456\u20131460,\u201d and also in Bridges K G, Hua Q, Brigham-Burke M R, Martin J D, Hensley P, Dahl C E, Digard P, Weiss M A, Coen D M. \u201cSecondary structure and structure-activity relationships of peptides corresponding to the subunit interface of herpes simplex virus DNA polymerase.\u201d J Biol Chem 2000; 274:472\u2013478. SEQ ID NO.:2, which corresponds to the amino acid sequence for UL42\u0394340 mutant of the processivity subunit of HSV DNA polymerase, which is described on page 4991 of Bridges K G, Chow C S, Coen D M. \u201cIdentification of crucial hydrogen-bonding residues for the interaction of herpes simplex virus DNA polymerase subunits via peptide display, mutational, and calorimetric approaches.\u201d J Virol 2001; 75:4990\u20134998, is shown in Table 4. SEQ ID NO.:3, which corresponds to the amino acid sequence of the catalytic subunit of HSV DNA polymerase, which can be found at http:\/\/www.stdgen.lanl.gov\/cgi-bin\/gene_id_search.cgi?dbname=hhv1&gene_id=HHVONE033, and SEQ ID NO.:4, which corresponds to the amino acid sequence for UL42, which can be found at http:\/\/www.stdgen.lanl.gov\/cgi-bin\/gene_id_search.cgi?dbname=hhv1&gene_id=HHVONE045, are also shown below in Table 4.","As used here \u201cpharmaceutically suitable salt\u201d refers to salts of compounds which are suitable for administration to a mammal, e.g. a human, for treating a disorder. For example, in embodiments where one or more nitrogen groups are present, chloride and hydrochloride salts may be pharmaceutically suitable salts. In embodiments where one or more carboxyl groups are present, alkali metal salts, e.g. sodium and potassium, may be pharmaceutically suitable salts. One skilled in the art given the benefit of this disclosure will be able to select suitable pharmaceutically suitable salts depending on the use and on the properties of the compound.","As used here \u201csolvate\u201d refers to an aggregate that comprises one or more molecules of the solute, such as a suitable amount of compound I, for example, with one or more molecules of a suitable solvent.","As used here \u201csuitable solvent\u201d refers to a solvent which sufficiently solubilizes one or more molecules of the solute to effect the desired result.","In accordance with certain preferred embodiments, a method is provided, in which the method includes providing a test sample which includes at least a fragment of a first subunit and at least a fragment of a second subunit. The fragment of the first subunit and the fragment of the second subunit are capable of interacting to form a dimer. The fragment of the first subunit includes a fluorescent label. The test mixture is formed by combining at least one test compound and a test sample. Fluorescence polarization of the test mixture is evaluated to determine if the test compound has disrupted interaction between the fragment of the first subunit and the fragment of the second subunit. In certain preferred embodiments, the fragments of first subunit and second subunit typically can be any size, e.g. any number of amino acids, as long as the fragments are capable of forming a dimer. In certain embodiments, the first fragment is a portion of a protein, e.g. a peptide. Preferably the first fragment of the first subunit is small enough such that a difference in fluorescence polarization can be observed when the first fragment of the first subunit and the first fragment of the second fragment interact. That is, the first fragment of the first subunit can generally be any size that is suitable for performing fluorescence polarization measurements which allow a distinction between first fragment of the first subunit which is free in solution and first fragment of the first subunit which is interacting with first fragment of the second subunit. Preferably, the first fragment of the first subunit comprises no more than about 40 amino acids, more preferably no more than about 30 amino acids and most preferably no more than about 20 amino acids, e.g. 18 amino acids or less. More preferably, the first fragment of the first subunit is a peptide which includes an amino acid sequence as shown in SEQ ID NO.: 1. According to one embodiment, the first fragment of the first subunit is substantially identical to that shown in SEQ ID NO.: 1. Preferably, the first fragment of the second subunit is substantially larger than the first fragment of the first subunit such the interaction of fluorescently labeled first fragment of the first subunit with the first fragment of the second subunit results in an increase in fluorescence polarization. More preferably, the first fragment of the second subunit comprises a protein which includes an amino acid sequence of either SEQ ID NO.: 2 or SEQ ID NO.:4. However, as discussed above, as long as a distinction can be made between free and interacting first fragment of the first subunit, the first fragment of the first subunit and the first fragment of the second subunit may comprise any number of amino acids. Depending on the nature and characteristics of the subunit interaction, the sizes of the fragments can vary. One skilled in the art given the benefit of this disclosure will be able to select and design suitable subunit fragments for use in the methods disclosed here.","In accordance with a certain preferred embodiments, a method is disclosed in which the method includes providing a test sample which includes a first fragment of a protein subunit bound to a second fragment of protein subunit. Preferably, the first fragment of protein subunit includes a fluorescent label. The method further includes measuring fluorescence polarization of the test sample. At least one test compound and the test sample are combined to form a test mixture, and fluorescence polarization of the test mixture is measured. The fluorescence polarization of the test mixture and the fluorescence polarization of the test sample are compared. A decrease in the fluorescence polarization of the test mixture, when compared to the fluorescence polarization of the test sample, indicates the first fragment is no longer bound to the second fragment. In certain preferred embodiments, the first and second fragments of protein subunit typically can be any size, e.g. any number of amino acids, as long as the fragments are capable of binding to form a dimer. In certain embodiments, the first and second fragments each are portions of a protein, e.g. a peptide. Preferably the first fragment of the protein subunit is small enough such that a difference in fluorescence polarization can be observed when the first fragment and second fragment bind. That is, the first fragment of the protein subunit can generally be any size that is suitable for performing fluorescence polarization measurements which allow a distinction between first fragment of protein subunit which is free in solution and first fragment of protein subunit which is bound to the second first fragment of the protein subunit. Preferably, the first fragment of the first subunit comprises no more than about 40 amino acids, more preferably no more than about 30 amino acids and most preferably no more than about 20 amino acids, e.g. 18 amino acids or less. More preferably, the first fragment is a peptide which includes an amino acid sequence as shown in SEQ ID NO.: 1. According to one embodiment, the first fragment is substantially identical to that shown in SEQ ID NO.: 1. Preferably, the second fragment is substantially larger than the first fragment such that interaction of fluorescently labeled first fragment with the second fragment results in an increase in fluorescence polarization. More preferably, the second fragment includes a protein which includes an amino acid sequence as shown in SEQ ID NO.:2 or SEQ ID NO.:4. However, as discussed above, as long as a distinction can be made between free and bound first fragment, the first and second fragments may comprise any number of amino acids. In certain embodiments, the fragments comprise only the minimal number of amino acids necessary for successful protein interaction. Thus, depending on the nature and characteristics of the fragment interactions, the sizes of the fragments can vary. One skilled in the art given the benefit of this disclosure will be able to select and design suitable subunit fragments for use in the methods disclosed here.","In accordance with certain preferred embodiments, a method of testing compounds for inhibiting herpes simplex virus DNA polymerase subunit interactions is disclosed. The method includes providing a test sample which includes a peptide which is substantially homologous to an eighteen amino acid C-terminal fragment of catalytic unit of herpes simplex virus DNA polymerase. The C-terminal fragment of catalytic unit of herpes simplex virus DNA polymerase typically includes a fluorescent label. The test sample also includes a functional fragment of processivity subunit of herpes simplex virus DNA polymerase. At least one test compound and the test sample are combined to form a test mixture, and fluorescence polarization of the test mixture is evaluated. In certain embodiments, the decrease in fluorescence polarization can be correlated to the level of DNA synthesis inhibition That is, the fluorescence polarization of the test mixture can be compared with the fluorescence polarization of the test sample to determine if there is any difference in fluorescence polarization. If such a difference is observed, the degree of difference can be correlated as a measure of the ability of the test compound to disrupt protein interactions. The herpes simplex virus (HSV) DNA polymerase is a heterodimer consisting of a catalytic (Pol) and a processivity (UL42) subunit. The interaction of these subunits is crucial for the synthesis of long-chain DNA products and for viral replication, representing an attractive target for antiviral drug discovery. This processivity factor allows Pol to synthesize extended stretches of DNA without dissociating from the template [Gottlieb et al. 1990; Gottlieb et al. 1994; Weisshart et al. 1999]. The residues necessary for the specific interaction with UL42 lie at the extreme C-terminus of Pol [Weisshart et al. 1994; Marsden et al. 1994; Digard et al. 1993; Tenney et al. 1993; Stow et al. 1993; Digard et al. 1990]. Recent work in our laboratory [Bridges et al. 2001] including the crystal structure of UL42 with the bound Pol-peptide [Zuccola et al. 2000] clarified the important residues of the other binding partner, UL42, and explained previous findings [Chow et al. 1995]. It has been observed that peptides corresponding to the extreme C-terminus of Pol inhibit UL42 stimulated long chain DNA polymerization by Pol [Digard et al. 1995]. Synthetic peptides ranging from 36 amino acids (peptide A) down to an 18 amino acid truncated peptide (peptide E) corresponding to the C-terminus of Pol exhibit ICvalues of 3 uM and 11 uM, respectively [Digard et al. 1995; Bridges et al. 2000] for specific inhibition of UL42 stimulated long-chain DNA synthesis. When a similar, 27-mer peptide is delivered into virally infected cells, it specifically interferes with HSV replication [Loregian et al. 1999].","In accordance with certain preferred embodiments, the peptide which is substantially homologous to an eighteen amino acid C-terminal fragment of catalytic unit of herpes simplex virus DNA polymerase is preferably at least about 90% homologous (in amino acid sequence), more preferably has a sequence homology of at least about 95% and most preferably has a sequence homology of at least about 99%. Determination of sequence homologies will be readily apparent to those skilled in the art given the benefit of this disclosure and exemplary methods include but are not limited to Swiss-PDB, sequence alignment algorithms, BLAST, etc. and the like. In certain preferred embodiments, the peptide has an amino acid sequence including SEQ ID NO.: 1. In certain preferred embodiments, the functional fragment of processivity subunit includes a protein which has an amino acid sequence including SEQ ID NO.: 2.","In accordance with certain preferred embodiments, a functional fragment of processivity subunit of herpes simplex virus DNA polymerase comprises a fragment of the processivity subunit which retains substantially all biochemical activities of the entire processivity subunit. For example, it has been shown that a truncated version of UL42 (UL42\u0394340 having SEQ ID NO.: 2) that was expressed in BL21(de3)pLysS as maltose binding protein fusion as described [Bridges et al. 2001] and purified over an amylose column (New England Biolabs), retained all known biochemical activities of UL42 [Hamatake et al. 1993; Digard et al. 1993; Tenney et al. 1993] but unlike full-length UL42. Thus in certain embodiments, the functional fragment of processivity subunit may comprise the entire processivity subunit, whereas in other embodiments, the functional fragment of processivity subunit comprises a truncated version of the processivity subunit which retains all known biochemical activities of the full processivity subunit. In certain preferred embodiments, the functional fragment of processivity subunit of herpes simplex virus DNA polymerase includes an amino acid sequence as shown in SEQ ID NO.: 2. In certain preferred embodiments, the peptide which is substantially homologous to an eighteen amino acid C-terminal fragment of catalytic unit of herpes simplex virus DNA polymerase includes an amino acid sequence which includes SEQ ID NO.: 1 and the functional fragment of processivity subunit of herpes simplex virus DNA polymerase includes an amino acid sequence which includes SEQ ID NO. 2.","In accordance with certain preferred embodiments, a kit for testing inhibitors of protein interactions is disclosed. The kit includes at least a first reagent including a fragment of a first subunit and a fragment of a second subunit. The first fragment of the first subunit includes a fluorescent label. The fragment of the first subunit and the fragment of the second subunit are capable of interacting to form a dimer. In certain embodiments, the first fragment of the first subunit includes an amino acid sequence as shown in SEQ ID NO.: 1. In certain embodiments, the first fragment of the second subunit includes an amino acid sequence as shown in SEQ ID NO.: 2 or SEQ ID NO. 4. In certain embodiments, the first fragment of the first subunit includes an amino acid sequence including SEQ ID NO.: 1 and the first fragment of the second subunit includes an amino acid sequence including SEQ ID NO.: 2 or SEQ ID NO. 4.","In accordance with certain preferred embodiments, a kit for testing compounds for inhibiting herpes simplex virus is provided. The kit includes at least a first reagent which includes a peptide that is substantially homologous to an eighteen amino acid C-terminal fragment of the catalytic unit of herpes simplex virus DNA polymerase. The reagent also includes a fragment of processivity subunit of herpes simplex virus DNA polymerase, in which the C-terminal fragment comprises a fluorescent label. In certain embodiments, the peptide is selected from peptide including SEQ ID NO.: 1. In certain embodiments, the fragment of processivity unit is selected from fragments including SEQ ID NO.: 2 or SEQ ID NO.:4. In certain embodiments, the peptide is selected from peptides including SEQ ID NO.: 1 and the fragment of processivity unit is selected from fragments including SEQ ID NO.: 2 or SEQ ID NO. 4.","In accordance with certain preferred embodiments, a kit is disclosed. The kit includes a reagent solution for testing compounds which inhibit herpes simplex virus. The reagent solution includes a peptide which includes SEQ ID NO.:1 and a protein fragment which includes SEQ ID NO.:2. The peptide includes a fluorescent label.","The kits disclosed here may further include one or more buffers, salts, solvents, proteins, indicators, dyes, or other suitable components necessary for determining protein interactions. One skilled in the art given the benefit of this disclosure will be able to select other suitable components for incorporating into the kits disclosed here. The kits may be designed such that one or more reagents which are temperature sensitive can be kept at a suitable temperature, e.g. \u221280\u00b0 C. That is, one or more components of the kits may be kept at a lower temperature than the other reagents and prior to use such reagent can be warmed to room temperature, or the temperature of the assay as the case may be. Accordingly, the kit may comprise a plurality of reagents, e.g. two, three or more, which are mixed together during or before the assay.","In accordance with certain preferred embodiments, all components of the kits disclosed here may be mixed such that a single solution comprises all necessary components for performing an assay to determine protein interaction inhibitors. That is, no mixture of reagents is required by the user. Typically the user would provide a test compound and mix the test compound with a sample of the kit reagent to determine if the test compound could disrupt protein interactions. In other embodiments, however, the components of the kit are stored in separate containers, e.g. vials. Thus, prior to performing assays to identify protein inhibitors, suitable amounts of each component can be added to the test mixture. One skilled in the art given the benefit of this disclosure will be able to design kits suitable for identifying test compounds which inhibit protein interactions.","In accordance with certain preferred embodiments, a composition for treating herpes simplex virus comprising a compound of formula I, or a pharmaceutically suitable salt or solvate thereof. Preferably a salt of compound I is used to treat herpes simplex virus, e.g. hydrochloride, chloride, bromide, sodium, and\/or potassium salts, etc. Suitable pharmaceutical salts typically depend on the solubility of the salt, bioavailability of the salt, synthetic feasibility of the salt, etc. One skilled in the art given the benefit of this disclosure will be able to select suitable salts of compound I for treating herpes simplex virus.","In accordance with certain preferred embodiments, a composition for treating herpes simplex virus is disclosed. The composition includes a compound of formula II, or pharmaceutically suitable salt or solvate thereof. Preferably a salt of compound of formula II is used to treat herpes simplex virus, e.g. hydrochloride, chloride, bromide, sodium, and\/or potassium salts, etc. Suitable pharmaceutical salts typically depend on the solubility of the salt, bioavailability of the salt, synthetic feasibility of the salt, etc. One skilled in the art given the benefit of this disclosure will be able to select suitable salts of compound I for treating herpes simplex virus.","In accordance with certain preferred embodiments, the compounds disclosed here may be placed into a suitable carrier. That is, compounds I\u2013II typically can be mixed with a carrier, e.g. powders, gelatin, etc., such that suitable forms, e.g. pills, capsules, tablets, etc. can be provided. For example, in preparing a formulation suitable for administration to a mammal, e.g. a human, bovine, etc., a suitable amount of compound I is typically combined with an excipient, diluted by an excipient or enclosed within a carrier that can be in the form of a capsule, sachet, or other container. When the excipient serves as a diluent, it can be a solid, semi-solid, or liquid material that acts as a vehicle, carrier or medium for the active ingredient. Thus, the compositions can be in the form of tablets, pills, powders, lozenges, sachets, cachets, elixirs, suspensions, emulsions, solutions, syrups, creams, aerosols (as a solid or in a liquid medium), ointments, soft and hard gelatin capsules, suppositories, sterile injectable solutions and sterile packaged powders. Examples of suitable carriers, excipients and diluents include but are not limited to lactose, dextrose, sucrose, sorbitol, mannitol, starches, gum acacia, calcium phosphate, alginates, carageenans, tragacanth, gelatin, calcium silicate, microcrystalline cellulose, polyvinylpyrrolidone, cellulose, water, syrup, and methyl cellulose. The formulations can additionally include: lubricating agents such as talc, magnesium stearate, and mineral oil; wetting agents; emulsifying and suspending agents; preserving agents such as methyl- and propylhydroxybenzoates; sweetening agents; flavoring agents; and dyes or other coloring agents. The compositions of the present invention can be formulated so they provide quick, sustained or delayed release of the active ingredient after administration to the mammal by procedures which will be readily apparent to those skilled in the art given the benefit of this disclosure. Depending on the intended use , e.g. veterinary uses, human treatment uses, etc., the nature and properties of the carrier may vary, and one skilled in the art given the benefit of this disclosure will be able to select suitable carriers for administration to mammals.","In accordance with certain preferred embodiments, a method of treating herpes simplex virus in a mammal, e.g. a human, comprising administering a pharmaceutically effective amount, e.g. an efficacious amount, of a compound of formula I or formula II, or pharmaceutically suitable salt or solvate thereof, to the mammal is provided. The pharmaceutically effective amount typically depends on the nature of the compound used, e.g. nature of the salt, and preferably comprises an amount suitable to inhibit the herpes simplex virus DNA polymerase. Preferably, the compound is administered to a patient inflicted with herpes simplex virus by oral administration, topical administration, intravenous administration, peritoneal administration, and the like. In embodiments, where the compound is administered orally, preferably the amount administered is between 0.001\u201320 mg\/kg of body weight, more preferably 0.01\u201315 mg\/kg of body weight and most preferably between 0.10\u201310 mg\/kg of body weight. One skilled in the art given the benefit of this disclosure will be able to select suitable amounts for administering to a mammal inflicted with herpes simplex virus. Preferably, the compound is mixed with a suitable carrier, as discussed above, before administration to a mammal. Suitable carriers will be readily apparent to those skilled in the art given the benefit of this disclosure.","It will be recognized by those skilled in the art given the benefit of this disclosure, that certain preferred embodiments of the present invention provide for high-throughput testing of potential inhibitors of protein interactions. The robust methods and kits disclosed here can be employed to test large libraries of compounds in a rapid and cost-effective manner. For example, the methods disclosed here can be used to design arrays comprising the components of the kit such that a plurality of test compounds can be tested in a rapid manner to identify potential test compounds which inhibit or disrupt protein interactions.","The following patent applications and\/or patents are incorporated herein by reference in their entireties for all purposes: Coen D M, Digard P E \u201cInhibitors of Herpes Simplex Virus Replication,\u201d U.S. Ser. No. 482,34, filed Feb. 21, 1990, issued as U.S. Pat. No. 5,223,391 Jun. 29, 1993, which mentions screening assays, and also case number unknown (Coen D M, Hogle J M, Elkin C, Zuccola H J, Bridges K G, Lokey, \u201cA Structure-Based Approach to Design Inhibitors of Protein-Processivity Factor Interactions,\u201d PCT filed May 12, 2000 for additional background information.","An example of the novel technology is disclosed below. The example is not intended to limit this novel technology in any manner.","Peptide probe and MBP-UL42. A peptide comprising the 18 C-terminal amino acids of HSV Pol (peptide E) was synthesized, N-terminally labeled with the pentafluorofluorescein-derivative Oregon Green 514 (Molecular Probes) and HPLC-purified. A truncated version of UL42 (UL42\u0394340) was expressed in BL21(de3)pLysS as maltose binding protein fusion as described [Bridges et al. 2001] and purified over an amylose column (New England Biolabs). UL42\u0394340 was shown to retain all known biochemical activities of UL42 [Hamatake et al. 1993; Digard et al. 1993; Tenney et al. 1993]. After elution with 10 mM maltose in 50 mM TrisHCl, 1 mM DTT, 0.5 mM EDTA, 4% Glycerol, 200 mM NaCl and Complete\u2122 protease inhibitors (Roche Molecular Biochemicals), the protein was concentrated when necessary and stored at \u221280\u00b0 C. until further use. After thawing, the purification buffer was exchanged to low fluorescent grade reagents (PanVera) and maltose was removed simultaneously using 10-DG Bio-Gel columns (BioRad).\n\nFluorescence polarization assays. 1 nM of Oregon Green-peptide E was added to 7 uM MBP-UL42\u0394340 and kept on ice until the mix was distributed into black 384-well plates (LJL Biosystems) in a total volume of 20 ul per well. The buffer conditions were 50 mM TrisHCl pH 7.5, 1 mM DTT, 0.5 mM EDTA, 150 mM NaCl, 4% Glycerol and 100 ug mlbovine gamma globulin. All chemicals were low fluorescent grade (PanVera).\n","Using this assay, the 16,320 compound Chembridge library that had been purchased by the Harvard Institute of Chemistry and Cell Biology (ICBB) was screened. Small molecules (5 mg mlin DMSO; ChemBridge) were screened using 384-pin arrays (Genetix), and 40 nl of each compound was transferred into every well. After incubation between 5 and 45 minutes at room temperature, the FP values were determined in the Analyst plate reader (LJL Biosystems). Chemicals for further testing were obtained from ChemBridge.","DNA polymerase long-chain synthesis assay. Compounds which elicited a decrease in the fluorescence polarization, when compared to control values, were tested for the ability to inhibit long-chain DNA synthesis via inhibition of polymerase subunit assembly [Digard et al. 1995; Bridges et al. 2001] using a poly-A template (Amersham Pharmacia) and an oligo-dT primer (Roche Molecular Biochemicals) on which Pol alone adds only a few bases. Reaction mixtures (50 mM Tris-HCl pH 7.5, 100 mM ammonium sulfate, 3 mM MgCl, 0.1 mM EDTA, 1 mM DTT, 4% Glycerol, 40 ug mlBSA, 100 ug mlprimer\/template, 50 uM \u03b1-P-TTP (111 TBq\/mmol; NEN Life Science products)), contained 200 fmol of HSV Pol ([Weisshart et al. 1994]), 400 fmol of HSV MBP-UL42 and varying amounts of compound in a final volume of 25 ul. Reactions were carried out at 37\u00b0 C. for 5 minutes and stopped by adding 5 ul of alkaline loading buffer (2mM EDTA, 50mM NaOH, 2.5% glycerol, 0.025% bromocresol green) and placing them on ice and were then loaded onto an 4% alkaline agarose gel. Gels were dried over night and exposed to phosphorescence screens (BioRad).\n\nCatalytic subunit activity filter binding assay. Compounds which elicited a decrease in fluorescence polarization were tested for the ability to inhibit DNA synthesis by Pol alone using the oligo-(dT)\/poly-(dA) primer-template. Reaction mixtures (50 mM TrisHCl pH 7.5, 100 mM ammonium sulfate, 3 mM MgCl, 0.1 mM EDTA, 1 mM DTT, 4% Glycerol, 40 \u03bcg mlBSA, 100 \u03bcg mlprimer\/template, 50 \u03bcM \u03b1-P-TTP (111 TBq\/mmol)) contained 200 fmol of HSV Pol and various amounts of each compound in a final volume of 25 \u03bcl. Reactions were carried out at 37\u00b0 C. for 25 minutes and stopped by placing 10 \u03bcl of the reaction mix onto DE81 filter discs. After washes in sodium hydrogen phosphate, water and ethanol, they were counted in a liquid scintillation analyzer (Packard 1600 TR).\n\nPlaque reduction assay. Confluent monolayers of Vero (African green monkey kidney) cells in 12-well dishes were infected with herpes virus strain 17 dilutions such that each well received between 25 and 50 PFU. Diluted virus was absorbed to cells in 2 ml Dulbecco's modified Eagle's medium containing 2.5% newborn calf serum, 100 U penicillin G mland 100 ug streptomycin sulfate mlfor 1 hour at 37\u00b0 C. in a humidified COincubator. After absorption, triplicate infected cultures were overlaid with 2% methyl cellulose in Dulbecco's modified Eagle's medium containing 5% newborn calf serum, 100 U penicillin G ml, 100 ug streptomycin sulfate mland various concentrations of the compounds which elicited a decrease in fluorescence polarization and incubated at 37\u00b0 C. in a humidified COincubator. Plaques were visualized by crystal violet staining at 2 to 3 days post infection and were counted on the following day [Coen et al. 1985].\n\nCytotoxicity assays. Vero cells (5\u00d710cells per well) were seeded into 96-well plates and treated with various concentrations of the compounds (which elicited a decrease in the fluorescence polarization) for 72 h in quadruplicates. Cell viability was determined with an XTT (sodium 3\u2032-[1-(phenylaminocarbonyl)-3,4-tetrazolium]-bis (4-methoxy-6-nitro) benzene sulfonic acid hydrate) assay (Roche Molecular Biochemicals) according to the manufacturer's suggestions using a Victor plate reader (Wallac). In parallel, equally compound-treated cells were fixed by adding 95% ethanol, stained with crystal violet solution and washed. After addition of 100 ul acidified ethanol (EtOH:HCl=99:1) to dissolve the crystal violet, the plates were read in the plate reader (Wallac).\n\nViral yield assay. Vero cells were seeded into 96-well (5\u00d710cells per well) tissue culture plates in 0.2 ml Dulbecco's modified Eagle's medium containing 2.5% newborn calf serum and 100 U penicillin G ml. Cultures were inoculated with HSV strain 17 at an MOI of 5 PFU per cell. The virus inoculum was replaced after two hours with medium containing dilution series of test compounds and left for 24 hours. After two cycles of freeze-thawing, each drug dilution row was transferred to a freshly seeded 96-well plate and serially 1:3 diluted over the whole plate. After two days, plaques were stained and counted under a microscope [Pritchard et al. 1990].\n\nA. Development and validation of a high throughput screen to identify small molecule inhibitors of Pol\/UL42 interaction. To demonstrate that the methods provided here are useful in identifying small molecules capable of preventing viral polymerase subunit assembly, a new homogeneous assay using fluorescence polarization (FP) was performed. The Pol portion of the polymerase was present as a fluorescently labeled 18-mer peptide and UL42 as MBP-fusion protein (as described above). Without wishing to be bound by any particular scientific theory, when the Oregon Green-labeled 18mer peptide, corresponding to the last 18 residues of the catalytic subunit is free, it tumbles fast, causing low polarization values. However, upon binding to MBP-UL42, the tumbling rate of the peptide adapts to the rate of the much bulkier binding partner and the polarization increases. The conditions for the assays were as follows: The labeled peptide was kept constant at 1 nM. MBP-UL42\u0394340 was used at a concentration range of 1 to 50 \u03bcM to determine the Kvalue of binding. Hofstee transformation was used to determine the Kvalue under the conditions used. A UL42 mutant (UL42I-160) defective for Pol-binding was used to show that the interaction of the peptide with wt UL42 was specific. The weak concentration dependent increase in polarization above 20 \u03bcM was mainly due to nonspecific binding (A, blue symbols). The values were used to correct the UL42 wt measurements. To further verify specificity, binding of peptide was competed with 36-mer peptide A derived from the C-terminus of Pol (B). UL42 was held constant at 7 \u03bcM.\n","To determine the Kvalue and test binding saturation, we titrated MBP-UL42 into a solution of labeled peptide at constant concentration (). The resulting Kvalue for the interaction of 5 uM is in agreement with constants determined earlier in our laboratory using other methods [Bridges et al. 2001; Bridges et al. 2000]. To further test binding specificity, we competed binding of the labeled peptide to UL42 with unlabeled peptide A, corresponding to the 36 last residues of Pol (). Both peptides show similar affinity to UL42 [Bridges et al. 2000]. The data indicated that peptide A was able to abrogate binding of the labeled tracer with an ICof approximately 6 \u03bcM using the Hill equation, four parameters, y=y=(axb\/cb+xb).","B. Screen of 16,320 compounds. 0.6% of all the compounds screened (Table 1), exhibited polarization values outside the \u00b13 SD-interval that served as selection criterion, indicating a change in the nature of the binding state of the labeled Pol-peptide. 0.4% of these however turned out to be false positives, due to intrinsic fluorescence of the small molecule, adsorption to the well and\/or precipitation. 0.2% of the initial hits were reproduced by the FP assay but only a third of these proved to be active in functional assays.",{"@attributes":{"id":"p-0067","num":"0066"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE I"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Summary of Screening"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"70pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Compounds Screened","16,320","100%"]},{"entry":[{},"Total Possible Hits","98","\u20090.6%"]},{"entry":[{},"False Positives","61","\u20090.4%"]},{"entry":[{},"Hits to Reproduce","37","\u20090.2%"]},{"entry":[{},"Most potential hits","9","0.06%\u2009"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}},"br":[{},{}],"figref":["FIG. 2","FIG. 3"]},"BP1, BP3, BP4, BP5 and BP9 did show a dose dependent change in the FP signal. As the structurally similar compounds behaved comparably in this assay, exhibiting ICvalues between 10 and 40 uM,. Therefore, it appears that some of these compounds act in a specific manner.","D. Pol\/UL42 assay. Using a long-chain DNA synthesis assay by DNA polymerase [Digard et al. 1995; Bridges et al. 2001], the compounds shown in  were tested for the ability to interfere with the subunit interaction in presence of the whole catalytic subunit and thus inhibiting the UL42-dependent long-chain DNA synthesis (). The conditions for the gel were as follows:","The selected compounds were tested for the ability to inhibit long-chain DNA synthesis via inhibition of polymerase subunit assembly using a poly-dA template and an oligo-dT primer on which Pol alone adds only a few bases (lane 1). Lane 2: Pol\/UL42 as positive control; lane 3: Pol\/UL42+40 mM peptide A as known inhibitor and negative control; lane 4: Pol\/UL42+20 mM peptide A; lane 5: Pol\/UL42+BP1; lane 6: Pol\/UL42+BP3; lane 7: Pol\/UL42\u00b1BP5. Concentration of compounds was 30 \u03bcM. Lane 8\u201312: Pol\/UL42BP9 at 100, 75, 50, 25 and 12 \u03bcM. Reaction mixtures (50 mM TrisHCl pH 7.5, 100 mM ammonium sulfate, 3 mM MgCl2, 0.1 mM EDTA, 1 mM DTT, 4% Glycerol, 40 \u03bcg mlBSA, 100 \u03bcg mlprimer\/template, 50 \u03bcM 32P-TTP), contained 200 fmol of HSV Pol, 400 fmol of HSV UL42 and indicated amounts of each compound in a final volume of 25 \u03bcl. Reactions were carried out at 37\u00b0 C. for 5 minutes and stopped by adding 5 \u03bcl of alkaline loading buffer and placing them on ice. They were then loaded onto an 4% alkaline agarose gel. Gels were dried over night and exposed to phosphorescence screens.","Referring to , BP1 (lane 5), BP3 (lane 6), BP5 (lane 7) and BP9 (lanes 8\u201312) greatly reduced the ability of UL42 to stimulate long-chain synthesis by Pol and allowed only formation of smaller products, in agreement with the dose dependent FP assay (see above). Again, the structurally related molecules BP3 and BP9 yielded similar effects. The other two compounds, BP1 and BP5, which show an effect in this experiment, exhibit both peptide-like features in their structure, thus enabling specific interactions with one of the subunits. As a control for the quantification via phosphor imager, the samples were also loaded onto DE-81 filters and analyzed in a scintillation counter. The results of the two methods were similar (data not shown).","E. Pol alone assay (Selectivity of identified small molecule inhibitors). In order to determine whether the inhibition of the long-chain DNA synthesis was in fact due to prevention of subunit interaction, DNA synthesis by Pol in absence of UL42 was tested using a filter binding assay. The amount of incorporated radioactive nucleotide was monitored at increasing concentration of the respective test compound (). In the presence of the structurally related BP3 and BP9, Pol activity was significantly reduced at higher concentrations. BP1 and BP5, both of which share peptide-like features, did not show a significant change in Pol activity in presence of higher compound concentration, inhibiting the Pol\/UL42 complex more potently than Pol alone, suggesting a different mode of action than BP3 and BP9. Aliquots of the reaction mix were also ran out on a sequencing gel (data not shown) showing a decreased amount of product formation in the case of BP3.","Table 2 summarizes the ICvalues of BP1, BP3, BP5 and BP9 as determined by FP analysis, long-chain DNA synthesis assay and filter binding assay for Pol activity in absence of UL42. For BP3 and BP9 all these values lay within the same range of magnitude, indicating long-chain DNA synthesis inhibition being caused by interference with Pol activity alone. For BP1 and BP5 the ICvalues for Pol activity alone are significantly higher than the other results, indicating a more specific mechanism of action different from the other two compounds.",{"@attributes":{"id":"p-0074","num":"0073"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"ICvalues of selected compounds"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"70pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"IC(uM) for","IC(uM)","IC(uM) for"]},{"entry":[{},"Pol\/UL42","for Pol","Pol\/UL42 long-chain"]},{"entry":["Compound","interaction by FP","alone activity","DNA synthesis"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"70pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["BP1","35",">100","25"]},{"entry":["BP3","12","25","30"]},{"entry":["BP5","15",">100","26"]},{"entry":["BP9","48","25","25"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}},"br":{}},"The compounds were tested for the ability to inhibit DNA synthesis by Pol alone using a poly-dA template and an oligo-dT primer Reaction mixtures (50 mM TrisHCl pH 7.5, 100 mM ammonium sulfate, 3 mM MgCl, 0.1 mM EDTA, 1 mM DTT, 4% Glycerol, 40 \u03bcg mlBSA, 100 \u03bcg mlprimer\/template, 50 \u03bcM P-TTP), contained 200 fmol of HSV Pol and indicated amounts of each compound in a final volume of 25 \u03bcl. Reactions were carried out at 37\u00b0 C. for 25 minutes and stopped by placing 10 \u03bcl of the reaction mix onto DE81 filter discs. After washes in sodium hydrogen phosphate, water and ethanol, they were counted in a liquid scintillation counter.","E. Antiviral assay. With two of the inhibitors showing specific effects in in vitro functional assays, experiments to test the influence of the compounds on herpes virus growth were performed. Monolayers of Vero cells in Dulbecco's modified Eagle's medium containing 2.5% newborn calf serum, 100 U penicillin G mland 100 \u03bcg streptomycin sulfate mlin 96-well dishes were infected with an MOI of 5. After absorption, selected compounds were added to duplicate wells and serially diluted to concentrations indicated in the figure and incubated at 37\u00b0 C. in a humidified COincubator for 24 h. Culture lysates were serially diluted in a separate set of uninfected cultures. After 2 d incubation, plaques were counted and virus titers calculated.","First, the 50% toxic dose of the compounds for the Vero cell line used in the viral yield reduction experiment (Table 3) was measured. BP5 exhibited a 50% toxic dose of 50 uM; determination of toxicity for BP1 was hampered by solubility problems in the culture medium used, but resulted in values similar to BP5 (see ). In the viral yield reduction assay [Pritchard et al. 1990] BP5 showed an EDvalue of 400 nM (see ). Although the 50% toxic dose of BP5 lies at 50 uM, its potential therapeutic range is still considerable. In parallel, plaque reduction assays (data not shown) were performed, supporting the findings of the viral yield reduction assay. BP5 was able to completely abolish plaque formation.",{"@attributes":{"id":"p-0078","num":"0077"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Toxicity of Selected Compounds"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Toxicity","IC(uM)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"BP1","20"]},{"entry":[{},"BP3","200"]},{"entry":[{},"BP5","50"]},{"entry":[{},"BP9","200"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0079","num":"0078"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 4"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"SEQUENCE LISTINGS"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["SEQ ID NO.: 1",{}]},{"entry":"Ac-ATAEETRRMLHRAFDTLA-NH"},{"entry":{}},{"entry":"SEQ ID NO.:2"},{"entry":"MTDSPGGVAPASPVEDASDASLGQPEEGAPCQVVLQGAELNGILQAFAPL"},{"entry":{}},{"entry":"RTSLLDSLLVMGDRGILIHNTIFGEQVFLPLEHSQFSRYRWRGPTAAFLS"},{"entry":{}},{"entry":"LVDQKRSLLSVFRANQYPDLRRVELAITGQAPFRTLVQRIWTTTSDGEAV"},{"entry":{}},{"entry":"ELASETLMKRELTSFVVLVPQGTPDVQLRLTRPQLTKVLNATGADSATPT"},{"entry":{}},{"entry":"TFELGVNGKFSVFTTSTCVTFAAREEGVSSSTSTQVQILSNALTKAGQAA"},{"entry":{}},{"entry":"ANAKTVYGENTHRTFSVVVDDCSMRAVLRRLQVGGGTLKFFLTTPVPSLC"},{"entry":{}},{"entry":"VTATGPNAVSAVFLLKLPQKICLDWLGHSQGSPSAGSSAS"},{"entry":{}},{"entry":"SEQ ID NO.:3"},{"entry":"MFSGGGGPLSPGGKSAARAASGFFAPAGPRGASRGPPPCLRQNFYKPYLA"},{"entry":{}},{"entry":"PVGTQQKPTGPTQRHTYYSECDEFRFIAPRVLDEDAPPEKRAGVHDGHLK"},{"entry":{}},{"entry":"RAPKVYCGGDERDVLRVGSGGFWPRRSRLWGGVDHAPAGFNPTVTVFHVY"},{"entry":{}},{"entry":"DILENVEHAYGMRAAQFHARFMDAITPTGTVITLLGLTPEGHRVAVHVYG"},{"entry":{}},{"entry":"TRQYFYMNKEEVDRHLQCRAPRDLCERMAAALRESPGASFRGISADHFEA"},{"entry":{}},{"entry":"EVVERTDVYYYETRPALFYRVYVRSGRVLSYLCDNFCPAIKKYEGGVDAT"},{"entry":{}},{"entry":"TRFILDNPGFVTFGWYRLKPGRNNTLAQPAAPMAFGTSSDVEFNCTADNL"},{"entry":{}},{"entry":"AIEGGMSDLPAYKLMCFDIECKAGGEDELAFPVAGHPEDLVIQISCLLYD"},{"entry":{}},{"entry":"LSTTALEHVLLFSLGSCDLPESHLNELAARGLPTPVVLEFDSEFEMLLAF"},{"entry":{}},{"entry":"MTLVKQYGPEFVTGYNIINFDWPFLLAKLTDIYKVPLDGYGRMNGRGVFR"},{"entry":{}},{"entry":"VWDIGQSHFQKRSKIKVNGMVNIDMYGIITDKIKLSSYKLNAVAEAVLKD"},{"entry":{}},{"entry":"KKKDLSYRDIPAYYAAGPAQRGVIGEYCIQDSLLVGQLFFKFLPHLELSA"},{"entry":{}},{"entry":"VARLAGINITRTIYDGQQIRVFTCLLRLADQKGFILPDTQGRFRGAGGEA"},{"entry":{}},{"entry":"PKRPAAAREDEERPEEEGEDEDEREEGGGEREPEGARETAGRHVGYQGAR"},{"entry":{}},{"entry":"VLDPTSGFHVNPVVVFDFASLYPSIIQAHNLCFSTLSLRADAVAHLEAGK"},{"entry":{}},{"entry":"DYLEIEVGGRRLFFVKAHVRESLLSILLRDWLAMRKQIRSRIPQSSPEEA"},{"entry":{}},{"entry":"VLLDKQQAAIKVVCNSVYGFTGVQHGLLPCLHVAATVTTIGREMLLATRE"},{"entry":{}},{"entry":"YVHARWAAFEQLLADFPEAADMRAPGPYSMRIIYGDTDSIFVLCRGLTAA"},{"entry":{}},{"entry":"GLTAVGDKMASHISRALFLPPIKLECEKTFTKLLLIAKKKYIGVIYGGKM"},{"entry":{}},{"entry":"LIKGVDLVRKNNCAFINRTSRALVDLLFYDDTVSGAAAALAERPAEEWLA"},{"entry":{}},{"entry":"RPLPEGLQAFGAVLVDAHRRITDPERDIQDFVLTAELSRHPRAYTNKRLA"},{"entry":{}},{"entry":"HLTVYYKLMARRAQVPSIKDRIPYVIVAQTREVEETVARLAALRELDAAA"},{"entry":{}},{"entry":"PGDEPAPPAALPSPAKRPRETPSPADPPGGASKPRKLLVSELAEDPAYAI"},{"entry":{}},{"entry":"AHGVALNTDYYFSHLLGAACVTFKALFGNNAKITESLLKRFIPEVWHPPD"},{"entry":{}},{"entry":"DVAARLRTAGFGAVGAGATAEETRRMLHRAFDTLA"},{"entry":{}},{"entry":"SEQ ID NO.:4"},{"entry":"MTDSPGGVAPASPVEDASDASLGQPEEGAPCQVVLQGAELNGILQAFAPL"},{"entry":{}},{"entry":"RTSLLDSLLVMGDRGILIHNTIFGEQVFLPLEHSQFSRYRWRGPTAAFLS"},{"entry":{}},{"entry":"LVDQKRSLLSVFRANQYPDLRRVELAITGQAPFRTLVQRIWTTTSDGEAV"},{"entry":{}},{"entry":"ELASETLMKRELTSFVVLVPQGTPDVQLRLTRPQLTKVLNATGADSATPT"},{"entry":{}},{"entry":"TFELGVNGKFSVFTTSTCVTFAAREEGVSSSTSTQVQILSNALTKAGQAA"},{"entry":{}},{"entry":"ANAKTVYGENTHRTFSVVVDDCSMRAVLRRLQVGGGTLKFFLTTPVPSLC"},{"entry":{}},{"entry":"VTATGPNAVSAVFLLKPQKICLDWLGHSQGSPSAGSSASRASGSEPTDSQ"},{"entry":{}},{"entry":"DSASDAVSHGDPEDLDGAARAGEAGALHACPMPSSTTRVTPTTKRGRSGG"},{"entry":{}},{"entry":"EDARADTALKKPKTGSPTAPPPADPVPLDTEDDSDAADGTAARPAAPDAR"},{"entry":{}},{"entry":"SGSRYACYFRDLPTGEASPGAFSAFRGGPQTPYGFGFP"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The following citations are incorporated herein by reference in their entireties for all purposes:","Gottlieb, J. Marcy, A. I., Coen, D. M. and Chaliberg, M. D. (1990). \u201cThe herpes simplex virus type 1 UL42 gene product; a subunit of DNA polymerase that functions to increase processivity. J. Virol. 64, 5976\u20135987;","Gottlieb J, Challberg M D. (1994) Interaction of herpes simplex virus type 1 DNA polymerase and the UL42 accessory protein with a model primer template. J Virol., Aug;68(8):4937\u201345;","Weisshart, K., Chow, C. S. and Coen, D. M. (1999) \u201cHerpes simplex virus processivity factor UL42 imparts increased DNA-binding specificity to the viral DNA polymerase and decreased dissociation from primer-template without reducing the elongation rate.\u201d J. Virol. 73, 55\u201366;","Marsden, H. S., Murphy M., McVery, G. L., MacEachren, K. A. Owsianka, A. M. and Stow, N. D. (1994) \u201cRole of the carboxy terminus of herpes simplex virus type 1 DNA polymerase in its interaction with UL42.\u201d J. Gen. Virol., 75, 3127\u20133135;","Digard, P., Chow, C. S. Pirrit, L. and Coen D. M. (1993) Functional analysis of herpes simplex virus UL42 protein. J. Virol. 67, 1159\u20131168;","Digard, P., Bebrin, W. R., Weisshart, K. and Coen, D. M. (1993) The extreme C terminus of herpes simplex virus DNA polymerase is crucial for functional interaction with processivity factor UL42 and for viral replication. J. Virol. 67, 398\u2013406;","Tenney, D. J., Huriburt, W. W., Bifano, M., Stevens, J. T., Michelleti, P. A., Hamatake, R. K. and Cordingley, M. G. (1993) Deletions of the carboxy terminus of herpes simplex virus type 1 UL42 define a conserved amino-terminal functional domain. J. Virol. 67, 1959\u20131966;","Tenney, D. J., Michelleti, P. A., Stevens, J. T., Hamatake, R. K., Mathews, J. T., Sanchez, A. R., Huriburt, W. W. Bifano, M. and Cordingley, M. G. (1993) Mutations in the C terminus of herpes simplex virus type 1 DNA polymerase can affect bonding and stimulation by its accessory protein UL42 without affecting basal polymerase activity. J. Virol. 67, 543\u2013547;","Stow N D, Hammarsten O, Arbuckle M I, Elias P. (1993) Inhibition of herpes simplex virus type 1 DNA replication by mutant forms of the origin-binding protein. Virology. Oct;196(2):413\u20138;","Digard, P., Coen, D. M. (1990) A novel functional domain of an alpha-like DNA polymerase. The binding site on the herpes simplex virus polymerase for the viral UL42 protein. J Biol Chem. October 15;265(29):17393\u20136;","Zuccola H J, Filman D J, Coen D M, Hogle J M.(2000) The crystal structure of an unusual processivity factor, herpes simplex virus UL42, bound to the C terminus of its cognate polymerase. Mol Cell. Feb;5(2):267\u201378;","Chow C S, Coen D M. (1995) Mutations that specifically impair the DNA binding activity of the herpes simplex virus protein UL42. J Virol., Nov;69(11):6965\u201371.","Digard P, Williams K P, Hensley P, Brooks I S, Dahl C E, Coen D M. (1995) Specific inhibition of herpes simplex virus DNA polymerase by helical peptides corresponding to the subunit interface. Proc Natl Acad Sci USA., February 28;92(5):1456\u201360;","Digard P, Bebrin W R, Coen D M. (1995) Mutational analysis of DNA polymerase substrate recognition and subunit interactions using herpes simplex virus as prototype. Methods Enzymol.;262:303\u201322;","Loregian A, Papini E, Satin B, Marsden H S, Hirst T R, Palu G. (1999) Intranuclear delivery of an antiviral peptide mediated by the B subunit of heat-labile enterotoxin. Proc Natl Acad Sci USA., April 27;96(9):5221\u20136;","Hamatake R K, Bifano M, Tenney D J, Hurlburt W W, Cordingley M G. (1993) The herpes simplex virus type 1 DNA polymerase accessory protein, UL42, contains a functional protease-resistant domain. J Gen Virol. Oct;74 ( Pt 10):2181\u20139;","Weisshart K, Kuo A A, Hwang C B, Kumura K, Coen D M. (1994) Structural and functional organization of herpes simplex virus DNA polymerase investigated by limited proteolysis. J Biol Chem., September 9;269(36):22788\u201396;","Coen D M, Fleming H E Jr, Leslie L K, Retondo M J. (1985) Sensitivity of arabinosyladenine-resistant mutants of herpes simplex virus to other antiviral drugs and mapping of drug hypersensitivity mutations to the DNA polymerase locus. J Virol., Feb53(2):477\u201388;","Pritchard C G, Stefano J E. (1990) Amplified detection of viral nucleic acid at subattomole levels using Q beta replicase. Ann Biol Clin (Paris).,48(7):492\u20137;","Bridges K G, Chow C S, Coen D M. (2001) Identification of crucial hydrogen-bonding residues for the interaction of herpes simplex virus DNA polymerase subunits via peptide display, mutational, and calorimetric approaches. J Virol., Jun;75(11):4990\u20138;","Bridges K G, Hua Q, Brigham-Burke M R, Martin J D, Hensley P, Dahl C E, Digard P, Weiss M A, Coen D M. (2000) Secondary structure and structure-activity relationships of peptides corresponding to the subunit interface of herpes simplex virus DNA polymerase. J Biol Chem., January 7;275(1):472\u20138; and","Degterev A, Lugovskoy A, Cardone M, Mulley B, Wagner G, Mitchison T, Yuan J. (2001) Identification of small-molecule inhibitors of interaction between the BH3 domain and Bcl-xL. Nat Cell Biol., Feb;3(2):173\u201382.","Although the invention has been described in terms of certain preferred embodiments, one skilled in the art given the benefit of this disclosure will recognize that other substitutions, modifications and alterations are possible. The following claims are intended to cover such substitutions, modification and alterations."],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE FIGURES","p":["The patent or application file contains at least one drawing executed in color. Copies of this patent or patent application publication with color drawing will be provided by the Office upon request and payment of the necessary fee.","Certain preferred embodiments of the present invention will be described below with reference to the accompanying figures in which:",{"@attributes":{"id":"p-0023","num":"0022"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0024","num":"0023"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0025","num":"0024"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0026","num":"0025"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0027","num":"0026"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0028","num":"0027"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0029","num":"0028"},"figref":"FIG. 7"}]},"DETDESC":[{},{}]}
