---
title: System for managing healthcare data including genomic and other patient specific information
abstract: A system, for processing patient medical information for storage in an electronic patient medical record repository, includes an interface, a repository, and a data processor. The interface receives data representing genomic information of a patient. The repository includes a patient record incorporating data representing genomic information specific to a particular patient. A data processor compares the genomic information specific to a particular patient with the received genomic information. The data processor identifies a genomic match in response to the comparison and predetermined matching criteria. The data processor initiates processing of patient record information specific to the particular patient in response to an identified match.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=07788040&OS=07788040&RS=07788040
owner: Siemens Medical Solutions USA, Inc.
number: 07788040
owner_city: Malvern
owner_country: US
publication_date: 20041217
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATIONS","FIELD OF THE INVENTION","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENTS"],"p":["The present application is a non-provisional application of provisional application having Ser. No. 60\/531,208 filed by Robert Haskell on Dec. 19, 2003.","The present invention generally relates to computer information systems. More particularly, the present invention relates to a system for managing healthcare data including genomic and other patient specific information.","Present healthcare delivery operations are fragmented and diverse. Clinical decisions are made without the benefit of evidence-based best practice or reference cases, health care is provided without regard to the genetic characteristics of individual patients, and historical clinical data is fragmented, diverse, and generally not structured or organized to facilitate information retrieval and knowledge discovery. Existing healthcare systems typically operate within a single site or enterprise offering limited administrative, clinical, and financial data in both operational and informational contexts and are generally passive in nature. Further, existing healthcare systems react to data entered, but generally do not provide proactive guidance to the health professional end users of the systems. Accordingly, there is a need for a system for managing healthcare data including genomic and other patient specific information that overcomes these and other disadvantages of the prior systems.","A system, for processing patient medical information for storage in an electronic patient medical record repository, includes an interface, a repository, and a data processor. The interface receives data representing genomic information of a patient. The repository includes a patient record incorporating data representing genomic information specific to a particular patient. A data processor compares the genomic information specific to a particular patient with the received genomic information. The data processor identifies a genomic match in response to the comparison and predetermined matching criteria. The data processor initiates processing of patient record information specific to the particular patient in response to an identified match.",{"@attributes":{"id":"p-0017","num":"0016"},"figref":"FIG. 1","b":["100","100","102","102","104","106"]},"Use of the stored healthcare information and knowledge provides a wide range of new opportunities to improve the quality and efficiency of healthcare delivery to patients and to facilitate clinical research. Such opportunities concern, for example and without limitation, medicine , patient , pharmacology , and industry . For example, the services and solutions provided from the stored healthcare information and knowledge help to:","1. Implement preventive therapy or lifestyle changes based on \u201cgenetic predisposition\u201d for disease.","2. Detect and control disease outbreak.","3. Establish efficient therapies with measurable outcomes.","4. Streamline and rationalize healthcare processes to reduce healthcare delivery costs.","5. Accelerate the pace of research.",{"@attributes":{"id":"p-0024","num":"0023"},"figref":["FIG. 2","FIG. 1"],"b":["200","100","200","202","204","206","208","210","212","214","216","218","220","222","224","226","228"]},"The system  may be employed by any type of enterprise, organization, or department, such as, for example, providers of healthcare products and\/or services responsible for servicing the health and\/or welfare of people in its care. For example, the system  represents a hospital information system. A healthcare provider may provide services directed to the mental, emotional, or physical well being of a patient. Examples of healthcare providers include a hospital, a nursing home, an assisted living care arrangement, a home health care arrangement, a hospice arrangement, a critical care arrangement, a health care clinic, a physical therapy clinic, a chiropractic clinic, a medical supplier, a pharmacy, and a dental office. When servicing a person in its care, a healthcare provider diagnoses a condition or disease, and recommends a course of treatment to cure the condition, if such treatment exists, or provides preventative healthcare services. Examples of the people being serviced by a healthcare provider include a patient, a resident, a client, and an individual.","Each of the elements in the system  may be fixed and\/or mobile (i.e., portable), and may be implemented in a variety of forms including, but not limited to, one or more of the following: a personal computer (PC), a desktop computer, a laptop computer, a workstation, a minicomputer, a mainframe, a supercomputer, a network-based device, a personal digital assistant (PDA), a smart card, a cellular telephone, a pager, and a wristwatch. The system  may be implemented in a centralized or decentralized configuration.","In the system , one or more elements may be implemented in hardware, software, or a combination of both, and may include one or more processors. A processor is a device and\/or set of machine-readable instructions for performing task. A processor includes any combination of hardware, firmware, and\/or software. A processor acts upon stored and\/or received information by computing, manipulating, analyzing, modifying, converting, or transmitting information for use by an executable application or procedure or an information device, and\/or by routing the information to an output device. An executable application comprises code or machine readable instruction for implementing predetermined functions including those of an operating system, healthcare information system, or other information processing system, for example, in response user command or input. For example, a processor may use or include the capabilities of a controller or microprocessor.","The elements in the system  are interconnected, as shown, using one or more networks  (otherwise called a communication path or link). The elements in the system  communicate over the network  using any type of protocol or data format including, but not limited to, the following: an Internet Protocol (IP), a Transmission Control Protocol Internet protocol (TCPIP), a Hyper Text Transmission Protocol (HTTP), an RS232 protocol, an Ethernet protocol, a Medical Interface Bus (MIB) compatible protocol, a Local Area Network (LAN) protocol, a Wide Area Network (WAN) protocol, a Campus Area Network (CAN) protocol, a Metropolitan Area Network (MAN) protocol, a Home Area Network (HAN) protocol, an Institute Of Electrical And Electronic Engineers (IEEE) bus compatible protocol, a Digital and Imaging Communications (DICOM) protocol, and a Health Level Seven (HL7) protocol.","The system  includes an integrated medical database to support the delivery of more efficient and higher quality health care. Information derived from the database is fed back into the health care delivery process for systems to provide more proactive and intelligent assistance to the health professional at the point of care, and is fed back into the analysis and mining process to facilitate the discovery of new knowledge by information analysts.","The system  provides a single-source, universal integrated medical database for stakeholder\/user access in any enterprise, university, local, regional, or national health market (assuming proper security clearance). Multiple data types from multiple data sources are linked and normalized within persons\/patients for easy access to complete information. Persons are additionally linked into other contexts, such as disease cohorts (i.e., diseases having a statistical factor in common). In contrast to the extensive installation effort needed for the typical operational health information system, little effort is required to start accepting data into the integrated medical database. Derived information is integrated back into the health care delivery process through interfaces for models and rules that are fed into the workflow, rules, and vocabulary engines within the local healthcare information systems.","In the context of data mining and data enhancement through data processing, it is common to differentiate between data, information, and knowledge. Although general definitions for these terms are available in various forms, definitions that uniquely and exactly differentiate the meaning of the terms are lacking. Therefore, wherever in the context of this patent application one of the terms data, information, or knowledge is used, these terms are not meant to restrict the scope of the claims herein or the data set addressed. For sake of clarity, in the formulation of the claims the term \u201cinformation\u201d is used, but it is to be understood that this term covers the complete range of data, information, and knowledge.","The integrated medical database provides \u201calways-on\u201d, pay-as-you-go or subscription-paid, discrete application and knowledge services for use and branding by any health information system (HIS) or health portal, whether an independent vendor solution, a proprietary health care provider solution, a government solution, a research system, a non-provider system, or an independent health care consumer, worldwide. The system helps optimize health care workflows, improving the quality and efficiency of the care delivered.","The system  advantageously performs the following functions, for example:","1. Collects, integrates, normalizes, stores, and manages many different data types from many different data sources.","2. Provides the tools, techniques, and applications to access the data.","3. Supports information and knowledge modeling to define relationships in the underlying data and the information derived from them.","4. Provides the means to feed the information back into point of care systems to recommend diagnoses and clinical actions, and to predict future behavior and outcomes, and to \u201cback office\u201d information systems to facilitate and validate the derivation of new information, models, and rules, thereby optimizing knowledge discovery.","The system  is used advantageously in the following functional areas, for example:","1. Clinical trial support (e.g., patient identification, outcomes analysis).","2. Clinical decisions support (e.g., feature extraction, exemplary cases, differential diagnosis, therapy simulation).","3. Consumer health service (e.g., lifetime record, personalized medicine).","4. Outcome analysis and process optimization (e.g., benchmarking, evidence-based best practice).","The system  provides the following advantages, for example, to enable proactive delivery of efficient and effective health and healthcare, as follows:","1. Data of multiple types is collected from multiple points of health care delivery and health care research (including, but not limited to, administrative, clinical, image, financial, and genomic\/proteomic data).","2. Data is integrated to create a complete, integrated, and patient-centered medical database. Available person identifiers (e.g., social security number) and genetic data is used to link patient data.","3. Data is separately transformed, enhanced, and stored in structures (data marts) to facilitate data mining, data analysis, and the monitoring of performance metrics.","4. Model and rules, which are derived from both the patterns and relationships discovered through mining and analysis and already established patterns and relationships, create predictive and actionable knowledge.","5. Application functions support clinical decision making by deriving and infusing structured information and evidence-base best practice into health care delivery (e.g., clinical trial support, clinical decision support, personalized medicine, outcome analysis).","6. Data and application functions are accessed through published, standard messaging protocols.","7. Data and application functions are accessed directly by health professionals at any point in health care delivery, and by information analysts in a back room.","8. Data and application functions are accessed directly or through any health information system, any time, anywhere.","9. Data and application functions are optionally packaged into and branded by any health information system.","10. Security and data integrity are tightly integrated and controlled.","Features of the system  include, for example, the following:","1. Feeding the derived information, models, and rules back to operational systems, where healthcare process can be optimized at the point of service through tightly coupled rules, workflow, and terminology services.","2. Feeding the derived information, models, and rules back to informational system knowledge stores, where rules and terminology services help facilitate and validate the derivation of new information, models, and rules, thereby optimizing knowledge discovery.","3. Using genetic data to control person identification and data management (e.g., merge person records if genetic profile is the same).","4. Using genetic data to control person access to his\/her own data as the central electronic health record.","5. Defining an installation process and tools for new data feeds, where install time in minimized.","6. Defining terms, codes, and identifiers that facilitate the integration of person, clinical, and genetic data.","7. Integrating genetic data into the existing administrative, clinical, and financial data sets, and integrating genetic data into health information system (HIS) applications.","8. Using the layered approach to integrated medical data and knowledge modeling as illustrated in .","9. Providing an open, pay-as-you-go, service-enabled function set for accessing integrated medical database capabilities, where a system can plug into and use the environment to enhance their own local capabilities.","A. Information Source\/Sink","The information source\/sink  (otherwise called \u201cdata suppliers and information consumers\u201d) includes information supplied by any source, and information received by any user or system, including for example, healthcare provider systems. Example of information sources include the following:","1. External knowledge and Benchmark information, which may come from public data sources.","2. Administrative, clinical, and financial data, which may come primarily from healthcare provider information systems.","3. Imaging data, which may come primarily from modality and PACS systems.","4. Non-healthcare provider data, which primarily may come from durable goods suppliers and payers.","5. Genomics and proteomics data, which may come from universities and testing labs.","6. Clinical trials data, which may come from clinical trial systems (clinical trials data is no different than other types of clinical and administrative data, but are different in terms of functions they support).","B. Data Interface Processor.","The data interface processor  receives data, transactions and files from the information source , and sends data, transactions and files to information sink . The data interface processor  provides functions, including for example, protocol and data conversions, routing, queuing, and error handling. The rules for transaction parsing and processing are maintained in an associated interface catalog (not shown in the system ). Standard interface protocols are supported (e.g., HL7, DICOM, X12N, MAGE, and CDISC, as shown in ), but non-standard protocols are also accommodated. Many different transaction formats exist in the health industry, and it is the responsibility of the data interface to collect and transform transactions to a format acceptable for the data update processor , which updates the health data repository . The data interface processor  also supports the initial back load of person and clinical data from existing enterprise repositories as part of the initial install of a new data supplier site.","C. Data Update Processor","The data update processor  receives transactions from the data interface processor  and updates the health data repository . The data update processor  understands the transaction formats and target data model, and provides the business logic that defines how data is to be inserted into the data model.","D. Health Data Repository",{"@attributes":{"id":"p-0077","num":"0076"},"figref":["FIG. 3","FIG. 2"],"b":["300","208","208","208","208","200"]},"Genomic and\/or proteomic data is associated with a person, and is stored in a model consistent with a standard, such as the MAGE-OM standard available on the Internet and from other sources, which aims to provide a standard for the representation of micro-array expression data that facilitates the exchange of micro-way information between different data systems.","The repository storage model  is a flexible and extensible meta-model, which is necessary to assimilate current and future data. For example, detailed clinical data is linked to a person and encounter, and are stored in a generic structure, as shown in .","In , the patient data  contains a complete description of a record associated with a patient (characteristics, demographics, etc.). The term  defines a unit of clinical data in the patient record, represented by, for example, identifier, type, name, effective date, and status. Elements  to  identifies various aspects of the terms , including data attributes relevant to each term. Synonym  identifies the multiple ways a single clinical concept may be described. External reference  points to equivalent terms in other terminologies, as well as supportive details stored in external data stores such as details of a drug. Description  provides the detail description of a term. Term type specific  contains data extending a term based on the type of term. Observation  is clinical data such as a lab result, radiology report, patient assessment, etc. Protocol  is a set of actions to be taken in the care of a disease or combination of diseases. Parameter  describes information such as duration and frequency of a service. Value  describes the actual numeric or text values, normal ranges, etc. Service  is an action taken such as a medication, physical therapy, vaccination, etc. Problem  describes the reasons for care such as sign, symptom, diagnosis, etc. Unit of measure  describes information such as milliliters, centimeters, inches, etc.","E. Data Transform Processor",{"@attributes":{"id":"p-0082","num":"0081"},"figref":["FIG. 4","FIG. 2"],"b":["210","400","200","400","400","400"]},"The data transformation system  includes document type definitions (DTD) , DICOM images , a medical image processing and understanding processor , lab data , legacy documents , a knowledge base storage device , a structured transformer processor , a structured content database , and a user interface . The structured transformer processor , representing the data transform processor , provides rule-based reasoning and transformation. The structured content database  provides indexing, searching, hyper-linking, navigation, and diagnosis support. The user interface  provides structured content query.","The data transform processor  (see ) transfers and reconfigures the data from the health data repository  into a mining, analysis, and reporting environment, which is called data warehousing  in the system . The data in the health data repository  is not directly usable (at least without sophisticated data processing tools) for mining purposes, and needs to be converted to another format and structure for more practical access. For example, content structuring occurs by taking the features extracted from images and text documents, other structured clinical and patient data, and models and rules defined through knowledge\/information modeling processor  described below, and applying rules-based reasoning and transformation to create structured content, typically in extensible markup language (XML) form.","The data transform processor  and\/or the data interface processor  also authorizes access by a user to a patient record of a particular patient in response to an identified match, and\/or authorizes access by a particular patient to his or her own patient record in response to an identified match.","Access to data at its source is an alternative to integrating data from all sources into a single physical repository. Typically, some combination of distributed and centralized access is used to implement the system .",{"@attributes":{"id":"p-0087","num":"0086"},"figref":["FIG. 5","FIG. 2"],"b":["500","210","200","500","500"]},"Genomic healthcare is healthcare that utilizes advances made by the science of genomics. Genomics is a branch of biotechnology concerned with applying the techniques of genetics and molecular biology to the genetic mapping and DNA sequencing of sets of genes or the complete genomes of selected organisms using high-speed methods, with organizing the results in databases, and with applications of the data (as in medicine or biology).","Genomics is the study of genes and their function. Recent advances in genomics are bringing about a revolution in our understanding of the molecular mechanisms of disease, including the complex interplay of genetic and environmental factors. Genomics is also stimulating the discovery of breakthrough healthcare products by revealing thousands of new biological targets for the development of drugs, and by giving scientists innovative ways to design new drugs, vaccines and DNA diagnostics. Genomics-based therapeutics includes traditional small chemical drugs, protein drugs, and potentially gene therapy.","Genomic information comprises, for example, at least one of the following: (a) DNA information, (b) RNA information, (c) complementary DNA or RNA information, (d) transfer RNA (tRNA) information, (e) messenger RNA (mRNA) information, and (f) Expressed Sequence Tags (EST).","Genome is the genetic material in the chromosomes of a particular organism; its size is generally given as its total number of base pairs. Genomic DNA is the basic chromosome set consisting of a species-specific number of linkage groups and the genes contained therein. A genomic library is a collection of clones made from a set of randomly generated overlapping DNA fragments representing the entire genome of an organism. Genetic testing is performed to gather information on an individual's genetic predisposition to particular health condition, or to confirm a diagnosis of genetic disease, for example.","Proteomics is a branch of biotechnology concerned with applying the techniques of molecular biology, biochemistry, and genetics to analyzing the structure, function, and interactions of the proteins produced by the genes of a particular cell, tissue, or organism, with organizing the information in databases, and with applications of the data (as in medicine or biology).","At step , the method  starts.","At step , the method  stores mapping information (otherwise called \u201ccommon elements\u201d) supporting conversion of genomic information in a first data format to a different second data format. Mapping information includes, for example, at least one of the following: (a) codes (or code sets), (b) terms, and (c) identifiers derived from multiple different sources and supporting interpretation of genomic information derived from different sources.","The codes, terms, and identifiers include HIPAA (Health Information Portability and Accountability Act) compatible code sets and other code sets used in a health care operation. Such code sets include, for example, ICD (International Classification of Diseases) codes, 9th Edition, Clinical Modification, (ICD-9-CM), Volumes 1, 2 and 3, as well as ICD-10 maintained and distributed by the U.S. Health and Human Services department. The code sets also include code sets compatible with HCPCS (Health Care Financing Administration Common Procedure Coding System), NDC (National Drug Codes), CPT-4 (Current Procedural Terminology), Fourth Edition CDPN (Code on Dental Procedures and Nomenclature). Further the code sets and terms include code sets compatible with SNOMED-RT \u201cSystematicized Nomenclature of Medicine, Reference Terminology\u201d by the College of American Pathologists, UMLS (Unified Medical Language System), by the National Library of Medicine, LOINC Logical Observation Identifiers, Names, and Codes Regenstrief Institute and the Logical Observation Identifiers Names and Codes (LOINC(r)) Committee, Clinical Terms also known as \u201cRead Codes\u201d, DIN Drug Identification Numbers, Reimbursement Classifications including DRGs Diagnosis Related Groups. The code sets also include code sets compatible with CDT Current Dental Terminology, NIC (Nursing intervention codes) and Commercial Vocabulary Services (such as HealthLanguage by HealthLanguage Inc., by Apelon Inc.) and other code sets used in healthcare.","The terminology, including vocabularies, code sets, and identifiers, is employed in characterizing or identifying a health provider organization, a location in an organization, a healthcare worker, a medical condition, a health service, a cost of a medical procedure or service, a payer organization, or a particular health plan. The health data repository  and\/or the data warehouse  contains medical terms, vocabularies and identifiers in addition to organizational characteristics, as well as location and other information supporting identification of location availability and suitability in a particular organization for delivering services by a particular physician to a patient with a particular medical condition. A medical code set as used herein is any set of codes used for encoding data elements, such as tables of terms, medical concepts, medical diagnosis codes, or medical procedure codes.","At step , the method  stores a patient record incorporating data representing genomic information specific to a particular patient in the second data format different from the first data format.","At step , the method  receives and stores data representing genomic information of a patient in the first data format.","At step , the method  applies the mapping information to convert the received genomic information from the first data format to the second data format.","At step , the method  stores the converted received genomic information in a patient record for the patient.","At step , the method  compares the stored genomic information specific to a particular patient with the stored converted received genomic information.","At step , the method  identifies a genomic match in response to the comparison and predetermined matching criteria.","At step , the method  initiates processing of patient record information specific to the particular patient in response to an identified match. The method  may initiate merging of at least a portion of the patient record information specific to the particular patient with another patient record in response to the identified match. The method  may identify a second patient record replicating patient record information specific to the particular patient in response to the identified match.","At step , the method  ends.","F. Data Warehouse\/Data Marts.",{"@attributes":{"id":"p-0106","num":"0105"},"figref":["FIG. 6","FIG. 2"],"b":["600","212","600","208"]},"Specialized cohorts or data marts (e.g., by disease, by market, etc.) are constructed from the main warehouse database, or directly from the health data repository . The warehousing environment provides both the tools necessary to normalize, transform, and manage data within the data warehousing environment, and the underlying structured data model into which data is stored.","The data warehouse  is used for storing, manipulating, and managing data for analysis purposes. Typically, topic-specific data marts are created using native data warehousing tools, or in some cases are created directly during the transformation process. Data enhancement is also done as part of the transformation process. New data is derived from existing data and physically added to the database (e.g., totaling numeric data, categorizing of detailed data into more general groups, assigning diagnosis related groups). For more complex, commonly needed derivations, the data warehouse  provides a more efficient resource than re-deriving new results for each new information request.","G. Data Service Processor","The data service processor  (see ) provides some important functions necessary to manage the data in the health data repository  and the data warehouse , described as follows.\n\n",{"@attributes":{"id":"p-0111","num":"0115"},"figref":["FIG. 7","FIG. 2"],"b":["700","200","702","704","706","708","700"],"ul":{"@attributes":{"id":"ul0003","list-style":"none"},"li":{"@attributes":{"id":"ul0003-0001","num":"0000"},"ul":{"@attributes":{"id":"ul0004","list-style":"none"},"li":"6. Data Quality. Other than the usual, physical, and logical edits on the incoming data, probabilistic inference is necessary for handling missing or inconsistent data. In addition, where possible, balancing and other forms of cross-data consistency checking are performed to assure overall completeness and data integrity."}}}},"H. Data Mining and Analysis Processors","The data mining and analysis processors  are used to create information about the large quantities of unstructured and structured data in the repositories.\n\n","I. Applications","The applications  represent packaged functions and\/or solutions that hide and organize the complexities of the underlying repositories, and provide ongoing function to knowledge professionals and health professionals. The underlying technology infrastructure  provides a set of common functions, common engines, and other common applications to facilitate the building of new applications. These applications have their own user interfaces, but are also service-enabled to expose application-programming interfaces (APIs) for external system use. Examples of application areas include:\n\n","J. Feedback Processor","The feedback processor  enables significant information derived by information analysts, which are organized and structured into standard formats, to be fed back to workflow, rules, and vocabulary engines in the operational systems, and to be fed back into the integrated medical database domain itself. In this fashion, both point of care and innovation processes are optimized. Standard interface protocols are used to transfer this information between systems.","K. Knowledge\/Information Modeling Processor.",{"@attributes":{"id":"p-0119","num":"0131"},"figref":["FIG. 8","FIG. 2"],"b":["800","222","808","806","810","812","806","810","804","802","804","223"]},"L. User Interface","Besides accessing functions and applications within the bounds of the integrated medical database, a single user, acting in a particular role at a particular workplace, potentially needs to access function from multiple systems within a single workflow. The user interface  provides the means for a user to initiate and manage a single session that includes, for example, diverse and separate products, applications, and functions, and to share patient context across them. Capabilities include cache management, linking patient data, and generating messages following SOAP and XML protocols.\n\n","The user interface  permits a user to interact with the system  by inputting user interface data into the system  via a data input device (not shown) and\/or receiving user interface data from the system , via a data output device (not shown). The user interface  generates one or more display images using a display processor (not shown). The display processor generates display data, representing one or more images for display, in response to receiving the input data or other data from the system . The display processor is a known element including electronic circuitry or software or a combination of both for generating display images or portions thereof.","M. Subscription\/Accounting Processor","The subscription\/accounting processor  provides subscription services and accounting services. Subscription services support the enrollment of stakeholders\/users (e.g., vendors, providers, knowledge users, and consumers) in the system , and the ongoing maintenance of their specific profile information, which is needed to control processing. Identification data (e.g., demographics, identifiers, access certificate), authorization and consent for data use, rules for transaction processing (e.g., patient identifier precedence, correction rules, special formats), and rules for data access (e.g., special formats) are defined and maintained. Accounting services support the recording, storage, and processing of activity as necessary to drive usage-based customer pricing and invoicing.","N. Technology Infrastructure","The technology infrastructure  contains the basic commodity technologies necessary to drive an IT system (e.g., operating system, database management, middleware, systems management, security, etc.).",{"@attributes":{"id":"p-0127","num":"0143"},"figref":["FIG. 9","FIG. 2"],"b":["900","200"]},"At step , the method  starts.","At step , the method  registers new data sources (e.g., interface protocols used, message types, patient identifier types, terminologies used).","At step , the method  registers end users to be accessing system function and operating aspects of infrastructure processing, and what specific functions they will be using.","At step , the method  registers computer systems to be accessing system function through service calls to system application programming interfaces (APIs)\/services, and what specific services they will be using.","At step , the method  develops data conversion plans to back load patient data (e.g., administrative, clinical, financial, genetic) from existing repositories.","At step , the method  establishes rules for handling a person's privacy, and identifies algorithms and process to be used for automatic patient identification.","At step , the method  establishes mapping from source terminologies to system reference terminologies.","At step , the method  ends.",{"@attributes":{"id":"p-0136","num":"0152"},"figref":["FIG. 10","FIG. 2"],"b":["1000","200"]},"At step , the method  starts.","At step , the method  initiates a data feed within the source system to route transactions to the knowledge source.","At step , the method  receives the transactions and map them to central healthcare repository formats in the interface engine that is part of the data interface.","At step , the method  identifies patient internal keys, but de-identifies and anonymizes patient data.","At step , the method  updates the health data repository, and associated audit repository in response to modification of data.","At step , the method  transfers and enhances the data as necessary to the data warehouse through the data transform (e.g., perform content extraction from images).","At step , the method  normalizes vocabulary to standard system terms.","At step , the method  creates disease-specific data marts for information processing purposes.","At step , the method  ends.",{"@attributes":{"id":"p-0146","num":"0162"},"figref":["FIG. 11","FIG. 2"],"b":["1100","200"]},"At step , the method  starts.","At step , the method  explores general content of a disease-specific data mart with simple reporting tools to understand general content of the data mart (e.g., patient listings).","At step , the method  uses an OLAP tool to help understand some of the basic performance characteristics of the patients and relationships between dependent and independent variables.","At step , the method  uses mining tools in the context of the constraints, after some of the basic characteristics and assumptions about the data are understood.","At step , the method  searches for new relationships to help optimize healthcare delivery and to predict patient behavior and outcomes.","At step , the method  sets up performance metrics to be monitored on a routine basis, including thresholds of appropriate variation.","At step , the method  combines derived internal information and establishes external knowledge into models and rules that help predict and direct future behavior.","At step , the method  applies the models and rules to the processes of healthcare delivery and clinical research, to help optimize their efficiency and quality.","At step , the method  ends.","Hence, while the present invention has been described with reference to various illustrative embodiments thereof, the present invention is not intended that the invention be limited to these specific embodiments. Those skilled in the art will recognize that variations, modifications, and combinations of the disclosed subject matter can be made without departing from the spirit and scope of the invention as set forth in the appended claims."],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0006","num":"0005"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0007","num":"0006"},"figref":["FIG. 2","FIG. 1"]},{"@attributes":{"id":"p-0008","num":"0007"},"figref":["FIG. 3","FIG. 2"]},{"@attributes":{"id":"p-0009","num":"0008"},"figref":["FIG. 4","FIG. 2"]},{"@attributes":{"id":"p-0010","num":"0009"},"figref":["FIG. 5","FIG. 2"]},{"@attributes":{"id":"p-0011","num":"0010"},"figref":["FIG. 6","FIG. 2"]},{"@attributes":{"id":"p-0012","num":"0011"},"figref":["FIG. 7","FIG. 2"]},{"@attributes":{"id":"p-0013","num":"0012"},"figref":["FIG. 8","FIG. 2"]},{"@attributes":{"id":"p-0014","num":"0013"},"figref":["FIG. 9","FIG. 2"]},{"@attributes":{"id":"p-0015","num":"0014"},"figref":["FIG. 10","FIG. 2"]},{"@attributes":{"id":"p-0016","num":"0015"},"figref":["FIG. 11","FIG. 2"]}]},"DETDESC":[{},{}]}
