---
title: System and method for improving clinical decisions by aggregating, validating and analysing genetic and phenotypic data
abstract: The information management system disclosed enables caregivers to make better decisions by using aggregated data. The system enables the integration, validation and analysis of genetic, phenotypic and clinical data from multiple subjects. A standardized data model stores a range of patient data in standardized data classes comprising patient profile, genetic, symptomatic, treatment and diagnostic information. Data is converted into standardized data classes using a data parser specifically tailored to the source system. Relationships exist between standardized data classes, based on expert rules and statistical models, and are used to validate new data and predict phenotypic outcomes. The prediction may comprise a clinical outcome in response to a proposed intervention. The statistical models and methods for training those models may be input according to a standardized template. Methods are described for selecting, creating and training the statistical models to operate on genetic, phenotypic, clinical and undetermined data sets.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08024128&OS=08024128&RS=08024128
owner: Gene Security Network, Inc.
number: 08024128
owner_city: Redwood City
owner_country: US
publication_date: 20041203
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCES TO RELATED APPLICATIONS","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENT"],"p":["This application claims priority under 35 U.S.C. \u00a7119(e) to U.S. Provisional Application No. 60\/607506, \u201cSystem and Method for Improving Clinical Decisions by Aggregating, Validating and Analyzing Genetic and Phenotypic Data\u201d by Matthew Rabinowitz, Wayne Chambliss, John Croswell, and Miro Sarbaev, filed Sep. 7, 2004, the disclosure of which is incorporated by reference herein in its entirety.","1. Field of the Invention","The invention relates generally to the field of analyzing, managing and acting upon clinical information, and specifically to a system which integrates genetic and phenotypic data from a group of subjects into a standardized format in order to validate the data and to make better decisions related to the genetic and phenotypic information of a particular subject.","2. Description of the Related Art","The current methods by which clinical decisions are made do not make the best possible use of existing information management technologies. Very little data from clinical trials and electronic clinical records\u2014collected as it is by a variety of methods, and stored in many different formats across a wide range of systems\u2014is immediately reusable by other groups in the biomedical community and can be accessed to aid caregivers and other decision makers. This situation will become considerably more complicated once personal genetic data occupies a more central role in understanding the causes and treatments of diseases and other predispositions of subjects. Within the next decade it will be possible to scan the entire genome of a patient either for clinical trials, or for the purpose of personalized drug assignment. Insofar as researchers continue to use different methods and systems for storing and analyzing this data, and the associated phenotypic data, all of the difficulties associated with sharing information within the biomedical community will persist and worsen.","A body of prior art exists to develop tools that manage the integration of existing data sets. For example, success has been achieved with tools that input textual data and generate standardized terminology in order to achieve information integration such as, for example, the Unified Medical Language System (UMLS): Integration Biomedical Terminology. Tools have been developed to inhale data into new ontologies from specific legacy systems, using object definitions and Extensible Markup Language (XML) to interface between the data model and the data source, and to validate the integrity of the data inhaled into the new data model. Bayesian classification schemes such as MAGIC (Multisource Association of Genes by Integration of Clusters) have been created to integrate information from multiple sources into a single normative framework, using expert knowledge about the reliability of each source. Several commercial enterprises are also working on techniques to leverage information across different platforms. For example, Expert Health Data Programming provides the Vitalnet software for linking and disseminating health data sets; CCS Informatics provides the eLoader software which automates loading data into ORACLE\u00ae Clinical; PPD Patient Profiles enables visualization of key patient data from clinical trials; and TABLETRANS\u00ae enables specification of data transformations graphically.","Depending on the tool, automated approaches to data integration can be far less resource intensive than the manual data integration, but will always be more constrained. It is exceedingly difficult to teach a computer how data of heterogeneous types should be sensibly merged. Prior art that most successfully approaches data integration makes use, in some form, of standardized master templates which define a data model, and provide a clear framework to researchers for inputs into, and augmentations of, that data model. This has been successfully applied, for example, in the GO (Gene Data model) project which provides a taxonomy of concepts and their attributes for annotating gene products. Similar projects include the Mouse Gene Database (MGD) and the Mouse Gene Expression Database (GXD). However, no system exists today to combine all phenotypic and genetic information associated with a patient into a single data model; to create a series of logical and statistical interrelationships between the data classes of that standard; to continually upgrade those relationships based on the data from multiple subjects and from different databases; and to use that information to make better decisions for an individual subject.","Prior art exists to manage information in support of caregivers and for streamlining clinical trials. Some of the enterprises involved in this space include Clinsource which specializes in software for electronic data capture, web randomization and online data management in clinical trials; Perceptive Informatics which specializes in electronic data capture systems, voice response systems, and web portal technologies for managing the back end information flow for a trial; and First Genetic Trust which has created a genetic bank that enables medical researchers to generate and manage genetic and medical information, and that enables patients to manage the privacy and confidentiality of their genetic information while participating in genetic research. None of these systems make use of expert and statistical relationships between data classes in a standardized data model in order to validate data or make predictions; or provide a mechanism by which electronically published rules and statistical models can be automatically input for validating data or making predictions; or guarantee strict compliance with data privacy standards by verifying the identity of the person accessing the data with biometric authentication; or associate all clinical data with a validator the performance of which is monitored so that the reliability of data from each independent source can be efficiently monitored; or allow for compensation of individuals for the use of their data; or allow for compensation of validators for the validation of that data.","Prior art exists in predictive genomics, which tries to understand the precise functions of proteins, RNA and DNA so that phenotypic predictions can be made based on genotype. Canonical techniques focus on the function of Single-Nucleotide Polymorphisms (SNP); but more advanced methods are being brought to bear on multi-factorial phenotypic features. These methods include regression analysis techniques for sparse data sets, as is typical of genetic data, which apply additional constraints on the regression parameters so that a meaningfuil set of parameters can be resolved even when the data is underdetermined. Other prior art applies principal component analysis to extract information from undetermined data sets. Recent prior art, termed logical regression, also describes methods to search for different logical interrelationships between categorical independent variables in order to model a variable that depends on interactions between multiple independent variables related to genetic data. However, all of these methods have substantial shortcomings in the realm of making predictions based on genetic and phenotypic data. None of the methods provide an effective means of extracting the most simple and intelligible rules from the data, by exploring a wide array of terms that are designed to model a wide range of possible interactions of variables related to genetic data. In addition, none of these prior techniques enable the extraction of the most simple intelligible rules from the data in the context of logistic regression, which models the outcome of a categorical variable using maximum a-posteriori likelihood techniques, without making the simplifying assumption of normally distributed data. These shortcomings are critical in the context of predicting outcomes based on the analysis of vast amounts of data classes relating to genetic and phenotypic information. They do not effectively empower individuals to ask questions about the likelihood of particular phenotypic features given genotype, or about the likelihood of particular phenotypic features in an offspring given the genotypic features of the parents.","The information management system disclosed enables the secure integration, management and analysis of phenotypic and genetic information from multiple subjects, at distributed facilities, for the purpose of validating data and making predictions for particular subjects. While the disclosure focuses on human subjects, and more specifically on patients in a clinical setting, it should be noted that the methods disclosed apply to the phenotypic and genetic data for a range of organisms. The invention addresses the shortcomings of prior art that are discussed above. In the invention, a standardized data model is constructed from a plurality of data classes that store patient information including one or more of the following sets of information: patient profile and administrative information, patient symptomatic information, patient diagnostic information, patient treatment information, and patient's genetic information. In one embodiment of this aspect of the invention, data from other systems is converted into data according to the standardized data model using a data parser. In the invention, relationships are stored between the plurality of data classes in the standardized data model. In one embodiment, these relationships are based on expert information. The relationships between the data classes based on expert information may one or more of the following types: integrity rules, best practice rules, or statistical models that may be associated with numerical parameters that are computed based on the aggregated data. In one embodiment, the statistical models are associated with a first data class, and describe the plurality of relevant data classes that should be used to statistically model said first data class. In one embodiment of the invention, the statistical and expert rules pertaining to the standardized data classes are automatically inhaled from electronic data that is published according to a standardized template.","In another aspect of the invention, methods are described for extracting the most simple and most generalized statistical rules from the data, by exploring a wide array of terms that model a wide range of possible interactions of the data classes. In one embodiment of this aspect of the invention, a method is described for determining the most simple set of regression parameters to match the data in the context of logistic regression, which models the outcome of a categorical variable using maximum a-posteriori likelihood techniques, without making the simplifying assumption of normally distributed data.","In another aspect of the invention, a method is described for addressing questions about the likelihood of particular phenotypic features in an offspring given the genotypic features of the parents.","In another aspect of the invention, for each user of the system is stored a biometric identifier that is used to authenticate the identity of said user, and to determine access privileges of said user to the data classes. In this aspect of the invention, each data class is associated with data-level access privileges and functional-level access privileges. These access privileges provide the ability for records to be accessed in a manner that does not violate the privacy rights of the individuals involved so that research can be performed on aggregated records, and so that individuals whose information satisfies certain criteria can be identified and contacted.","In another aspect of the invention, each data class in the system is associated with one or several validators, which are the entities who validated the accuracy of the information in that class. In this embodiment, data associated with the validator enables users to gauge the level of reliability of the validated information. In one embodiment of this aspect of the invention, the system provides a method by which individuals can be compensated for the use of their data, and validators can be compensated for the validation of that data.","Functional Overview of the System","Description of System Functionality","Assume in this discussion that the subjects are human patients in a clinical setting. A primary goal of the disclosed system is to enable clinicians to make better, faster, decisions using aggregated genetic and phenotypic data. In order to accomplish this, a secondary goal must be achieved, namely to enable research organizations to integrate and validate genetic and phenotypic data from multiple heterogeneous data sources. In a preferred embodiment, these two objectives are achieved with a system that has the following functionality: i) genetic, clinical and laboratory data are integrated into a standardized data model; ii) the integrity of information input into the data model is validated by a set of expert-rule based relationships and statistical relationships created between the data classes of the data model; iii) when data fails validation, the system facilitates remote upgrading of the data by data managers via a secure architecture; iv) clinicians are enabled by the system to subject the data of their patients to the models and methods of other researchers in order to have a meaningful prediction of clinical outcome; v) statistical relationships between data classes within the data model are refined based on the validated data that is input into the data model; vi) statistical models are selected to make the most accurate statistical predictions for data classes within the data model with the highest degree of confidence based on validated data within the data model; viii) statistical models are designed to make more accurate predictions correlating sparse genetic data with phenotypic data.","Integrating Clinical and Genetic data into a Standardized Data Model","The system disclosed makes clinical treatment, clinical trial and laboratory data useful to distributed organizations by inhaling the information into a format that has unambiguous definitions and by enabling contributions from different database systems. The data model consists of data classes that encompass the full range of patient profile, symptomatic, intervention, diagnostic and genetic information. In one embodiment, these data classes are composed from a set of emerging standards, including: The SNOMED Clinical Terms (CT) standard which enables a consistent way of indexing, storing and understanding clinical data; the Logical Observation Identifiers Names and Codes (LOINC) standard for all clinical and laboratory measurements; the Health Level 7 (HL7) standard which focuses on data processes and messaging; National Drug File Reference Terminology (NDF-RT) which provides standard terminology for drugs; and the FASTA format for representing genetic data. In one embodiment, the data classes in the standardized model are based on the Unified Medical Language System Metathesaurus developed by the National Library of Medicine, which integrates multiple emerging standards into one data ontology. Many of the illustrative examples in this disclosure relate to integrating data of HIV\/AIDS patients for the purpose of improving selection of anti-retroviral therapies (ART). This data serves as a good illustration of the functionality of our system because health care in developed and developing nations would be enhanced by a system that aggregates HIV\/AIDS genetic and phenotypic data as it becomes available to identify the best drug combinations for different mutations and strains of the HIV\/AIDS virus.","Integrity, Best-Practice and Statistical Relationships","The standardized data model is made useful by relationships that will be created between standardized data classes. The relationships will be of three types: i) data integrity relationships ii) best practice relationships, and iii) statistical relationships. The data integrity relationships will be expert rules that check for clear errors and inconsistencies in the data. For example, such rules would include checking that laboratory CD4+ lymphocyte counts and viral load counts are within reasonable ranges, or checking that a particular genetic sequence is not from a contaminated Polymerase Chain Reaction (PCR) by checking against other samples from the same laboratory to ensure they are not too closely correlated. Best practice relationships will be expert rules that describe correct methods for data collection and for clinical interventions. For example, such rules would include the requisite laboratory tests to establish that a patient requires Anti-Retroviral Therapy (ART), or rules describing which ART drugs should and should not be used together. Statistical relationships between data classes will be created by inhaling models created by experts, and continually refining the parameters of the models by aggregating patient data inhaled and validated in the standardized data model. In one embodiment, for each data class to be modeled, the system will inhale multiple different statistical models and choose from amongst them that which is best suited to the data\u2014this will be described later. In a preferred embodiment, as the data is inhaled, it is checked for errors, inconsistencies and omissions by a three step process. The three steps will correspond respectively to checks for violation of the data integrity relationships, the best practice relationships and the statistical relationships.","Applying Expert Knowledge in the Data Model Across Multiple Client Systems","In a preferred embodiment, the system is architected so that a limited subset of validation rules will check the integrity of the data at the time of parsing the data from the client database, before it enters the standardized data model. These client-system-dependent rules must be developed specifically for each client database in a specialized cartridge. For scalability, the cartridge size and complexity will be kept to a minimum. The majority of data validation will occur once data is inhaled into the data model. In this way, the relationships encoded into the standardized data model can be applied to validating a wide range of client databases, without having to be reconfigured for each client system. In addition, the previously validated data will be used to the refine statistical rules within the data model. In this way, new patient data from other client systems can be efficiently inhaled into the standardized data model.","Using an Automatic Notification System to Upgrade Invalid Data","When data fails the validation process described above, the system will, in a preferred embodiment, generate an automatic notification for the relevant data manager. In a preferred embodiment, an embedded web-link in this notification will link to a secure URL that can be used by the data manager to update and correct the information. The validation failure, whether based on integrity relationships, the best-practice relationships, or the statistical relationships, generates a human-readable explanatory text message. This will streamline the process of integrating data from multiple collaborating institutions and reduce data errors.","Enabling Updates to the Standardized Data Model","As upgrades to data standards are made, the standardized data model will need to be upgraded while preserving both the relationships between data classes and the validity of data formatted according to the affected standards. In a preferred embodiment, the standardized data model will uses a flex schema where possible (described below) for the standardized data model. While inefficient for queries, the flex schema will enable straightforward upgrades of the data model to accommodate evolving standards. In a preferred embodiment, a parsing tool will translate data from one version of the data model to another and will appropriately upgrade relationships between data classes based on a mapping between the old and the updated data classes. Where such mappings are not possible without further data or human input, automatic notifications will be generated for the data managers. If no mechanism can enable translation, affected patient data will be replaced by a pointer to the data in the old standardized data model. Upon completion of the data model upgrade, the re-validation of all updated data records will then be performed and should generate the same results. In addition, statistical analysis of the updated records will be re-computed to verify that the same set of outcomes is generated.","Enabling Clinicians to Subject the Data of their Patients to the Models and Methods of other Researchers","In one embodiment, the system will enable physicians to analyze their own patient's genetic and phenotypic data using the methods and algorithms of other researchers. This will enable physicians to gain knowledge from other collaborators, and will provide a streamlined validation of the results generated by clinical studies. A clinical study can typically be broken into three information modules: namely i) the clinical data, ii) the statistical model used, iii) and the statistical results. A template will be created to enable researchers to specify both their data and their statistical models, or the functions they use to act on that data. Researchers will use the template to specify two function types: a training function and a mapping function. The training function will input training data as dependent and independent variables, and will output the set of parameters required to map between them according to the model.","The mapping function will input the independent variables, and the parameters, and will generate a prediction of the dependent variable as well as a confidence measure with which the prediction is made. These functions may be defined using off-the-shelf software such as SAS or MATLAB.","In one embodiment, when a clinical study is published electronically according to the template, the clinical data for that trial and the model used to analyze that data will be automatically inhaled into the database. The statistical results will then be regenerated and checked for agreement with those published. Parameters that constitute the statistical model (such as regression parameters) will be recomputed, and refined as new relevant data is inhaled into the standardized database. An interface will be created for a physician to generate a clinical prediction for his own patient's data by applying the model of a chosen clinical study.","Technical Description of System","Creating the Standardized Data Model  based on Existing Standards",{"@attributes":{"id":"p-0037","num":"0036"},"figref":"FIG. 1","b":["110","108","112","113","116","115","117","122","118","119","120","121"]},"In a preferred embodiment, the patient profile standard , the diagnostic standard , the symptomatic standard  and the treatment standard  consist of data classes that are leveraged from the following emerging standards for clinical and laboratory data and protocols for data exchange:","Systematized Nomenclature of Medicine Clinical Terms (SNOMED-CT): Based on a terminology developed by the College of American Pathology and the United Kingdom National Health Service's Read Codes for general medicine, SNOMED-CT is the most comprehensive clinical ontology in the world. It contains over 357,000 health care concepts with unique meanings and logic-based definitions organized in hierarchies, and can be mapped to the existing International Classification of Disease 9Revision\u2014Clinical Modification. Through a licensing agreement with the National Library of Medicine, SNOMED-CT is available through the Unified Medical Language System Metathesaurus and is considered an essential component of President George Bush's ten year plan to create a national health information technology infrastructure (including electronic health records) for all Americans.","Logical Observation Identifiers Names and Codes (LOINC): Developed by the Regrienstreif Institute at the Indiana University School of Medicine, the LOINC laboratory terms set provides a standard set of universal names and codes for identifying individual laboratory and clinical results, allowing developers to avoid costly mapping efforts for multiple systems.","Health Level 7 (HL7) version 3.0: HL7 is the primary messaging standard for sharing healthcare information between clinical information systems supported by every major medical informatics vendor in the United States. In a preferred embodiment, the data in the standardized ontology will be exchangeable using HL7 compliant messaging.","RxNorm: RxNorm is a clinical drug nomenclature developed jointly by the National Library of Medicine, the Food and Drug Administration, Veterans Administration, and Health Level 7. The ontology includes information about generic and brand names, drug components, drug strength, and the National Drug Codes.","Fasta\/Blast: The Fasta and Blast algorithms are heuristic approximations of the Smith-Waterman algorithm for sequence alignment. Widely used for genetic sequence similarity analysis, the standardized database will utilize the readily sharable Fasta format to represent sequence information.","In one embodiment, the data classes of the standardized ontology are based on the data class definitions provided by the UMLS Metathesaurus managed by the National Library of Medicine.","Representation Patient Data  according to the Standardized Data Model ","A static\/flex approach will be used in the database to represent data entities of the standardized patient data . This approach balances the desire to have a data schema that is easy to read and tune for performance, versus a schema that supports flexibility in data representation. The static schema will define the non-volatile entities in the standardized patient data  and their associated attributes. Data that is called upon frequently in queries, unlikely to evolve with standards, and\/or essential to system function will be modeled in this static schema. illustrates part of a table in the standardized patient data  configured according to the static schema for the entity \u201cPatient\u201d. In this data schema, the columns in the data tables in the patient database  are pre-defined, and data is entered directly into these pre-defined columns. Although this scheme is easy to query, changes in this schema are problematic\u2014they involve adding columns and\/or tables, and rewriting code for data retrieval. Consequently, the majority of the patient data  will be stored according to the flex schema. In this scheme, the standardized data model  describes a layer of metadata, or data about data. The flex schema will be used to store all standardized patient data  that may be volatile. This will include data that is unique to a specialized medical scenario and is not in an accepted standard, and\/or data in a standard that may evolve. and illustrate a flex schema for storing laboratory measurements of HIV viral load. illustrates the table that is part of the standardized patient data . illustrates the table that is in the standardized data model . Although this schema is more complex, and slower to query, the addition of a measured parameter entity to the standardized patient data  does not require any new tables\/columns. In a preferred embodiment, wherever feasible from the perspective of system performance, the database will use a flex schema for each entity in the standardized patient data so that upgrades to the data standards can be easily accommodated.","The data model will address the full range of entities that relate to i) patient profile\u2014an individual's demographic and psychographic information; ii) clinical presentation; iii) treatment history; iv) diagnostic information; and v) genomic data. In a preferred embodiment, the latter will include both human genetic profile such as host genetic factors that may influence resistance as well as genetic data on any infectious agents, such as viral genomic data that relates to the patient's strain-of-infection. A unique identifier, compliant with the Health Insurance Portability and Accountability Act (HIPAA), will be used to link to other entities in the data model. In a preferred embodiment, all data representation and access will be HIPAA compliant.","Relationships between the Data Classes","A key element that makes the standardized data model unique and valuable is the set of relationships that exist between the data classes in the standardized data model . These relationships can be divided into computed statistical relationships  and expert relationships . Expert relationships are discussed first. In a preferred embodiment, three types of expert relationship are implemented\u2014integrity rules, best practice rules, and statistical models.","Integrity rules: The integrity rules are algorithms for checking the data based on heuristics described by domain experts. In a preferred embodiment, each integrity relationship is implemented as a software function that inputs certain elements of the patient data record, and outputs a message indicating success or failure in validation. Simple integrity functions include checking that all key data fields, such as a patient's last name or baseline CD4+ cell count, are present in the patient data record; confirming the market availability of a drug mentioned in the patient treatment record. More complex integrity functions include assessing the possibility of laboratory cross-contamination of a viral genotype PCR sample by aligning a new sequence and correlating it with other sequences generated from the same laboratory at the same time period; ensuring that samples aren't misnamed by comparing each sequence from an individual with previous samples from that individual to ensure they are not too dissimilar; and ensuring that a gene sequence itself is valid by checking for unexpected stop codons, frame shifts, and unusual residues.","Best practice rules: These will encode guidelines for collecting patient data, and for clinical patient management. In a preferred embodiment, these will be stored in the meta-data database  as best practice rules . Two examples of best practice guidelines are illustrated in . is the recommendation for administration of a second regimen of ART drugs for patients with and without Tuberculosis (TB), according to United Nations World Health Organization (WHO) guidelines. The best practice rules  are typically more complex than the excerpt shown. ART guidelines, for example, would include such considerations as: not using two anti-retroviral medications of the same nucleoside due to the potential for cross-resistance, nor using two medications that have the same side effect profile. The best practice rules  will be encoded using the data classes defined in the standardized data model . The best practice code, acting on the standardized data for a particular patient , will be able to determine the context (e.g. HIV\/AIDS patient on regimen 2, any decisions that need to be taken, for example whether or not a patient has TB, and any actions that should be, or should have been, taken (e.g. replacing Saqinavir with Lopinavir). shows another example set of guidelines in a flow-diagram form. In a preferred embodiment, guidelines such as these are encoded into the best practice rules  in the meta-data database , acting on the standardized data model , in order to determine context, decisions and actions for particular patient's standardized data .","Statistical models: Statistical models  will be stored in the meta-data database . The use of the statistical models  for prediction of clinical outcome for clinicians will be further elaborated below and is described in . The statistical models  will also be used for data validation. The statistical models will describe how to calculate the likelihood of data in a particular patient record  in the patient data database , given data about prior patients  () with similar characteristics\u2014termed the patient subgroup  (). In a preferred embodiment, the statistical model will contain a set of parameters which will be stored as part of the computed relationships data  of the meta-data database . Each of the statistical models  will specify the subgroup  to which it applies so that the DAME  can evaluate data from a particular patient  against the data for all other patients in that subgroup . Each data class in the patient record can be represented with a random variable, X, which describes feature f of patient n. This data could come from the patient profile information , symptomatic information , treatment information , or diagnostic and genetic information , and may be a continuous or categorical variable. In a preferred embodiment, some of the statistical models  are regression models of one or more variables. For example, the variable of interest, Y, could be the log of the CD4+ cell count of patient n, and the relevant subgroup  could be all patients who have received ART for more than 6 months without a resistant viral strain. The DAME could then compare datum Yassociated with the record  of a patient N+1, and determine whether or not it is reasonable. In the simplest regression model, the DAME  evaluates Ybased on the sample mean and variance of Yover the subgroup  of N patients. In a more complex model, the DAME  uses a regression model involving multiple variables to represent the dependent variable:",{"@attributes":{"id":"p-0051","num":"0050"},"maths":{"@attributes":{"id":"MATH-US-00001","num":"00001"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"msub":{"mi":["Y","n"]},"mo":"=","mrow":{"msub":[{"mi":"\u03b2","mn":"0"},{"mi":["\u025b","n"]}],"mo":["+","+"],"mrow":{"munderover":{"mo":"\u2211","mrow":[{"mi":"p","mo":"=","mn":"1"},{"mi":"P","mo":"-","mn":"1"}]},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"msub":{"mi":["\u03b2","p"]},"mo":"\u2062","mrow":{"msub":{"mi":["f","p"]},"mo":"\u2061","mrow":{"mo":["(",")"],"msub":{"mover":{"mi":"X","mo":"\u2192"},"mi":"n"}}}}}}}}},"br":{},"sub":["n ","p ","p ","n ","n","1","n","p","n","N+1","N+1","N+1 ","N+1 ","N+1 "],"sup":"th "},{"@attributes":{"id":"p-0052","num":"0051"},"maths":{"@attributes":{"id":"MATH-US-00002","num":"00002"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mrow":{"msub":{"mover":{"mi":"Y","mo":"^"},"mrow":{"mi":"N","mo":"+","mn":"1"}},"mo":"\u00b1","mrow":{"mrow":[{"mi":"t","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mrow":[{"mn":"1","mo":"-","mfrac":{"mi":"\u03b1","mn":"2"}},{"mi":["N","P"],"mo":"-"}],"mo":","}}},{"mi":"s","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msub":[{"mover":{"mi":"Y","mo":"^"},"mrow":{"mi":"N","mo":"+","mn":"1"}},{"mi":"Y","mrow":{"mi":"N","mo":"+","mn":"1"}}],"mo":"-"}}}],"mo":"\u2062"}},"mo":","}}},"br":{}},{"@attributes":{"id":"p-0053","num":"0052"},"maths":{"@attributes":{"id":"MATH-US-00003","num":"00003"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mi":"t","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mrow":[{"mn":"1","mo":"-","mfrac":{"mi":"\u03b1","mn":"2"}},{"mi":["N","P"],"mo":"-"}],"mo":","}}}}},"br":[{},{}],"sub":["T,N-P","N+1","N+1","N+1","N+1","N+1 "],"b":["208","108"]},"The preferred architecture for interfacing between external sources of data and the internal schemas of the underlying standardized database is shown in . In a preferred embodiment, the process of data acquisition begins with the data feeders , which send HL7-compliant XML-encoded data  through public Internet . Note that the data feeder  and user  could be the same entity in certain usage scenarios. XML files  are received by the feed stager , which can be a secure FTP site, or any other data transfer protocol\/messaging receiver server. Data is passed from the feed stager to the feed parser , which parses both XML and non-XML encoded data. The feed parser will input format validation cartridge  and translation cartridge  specifically adapted to the particular source of data. Importantly, the knowledge required for parsing and preliminary integrity checking by the feed parser will be maintained externally to the parser in the cartridges , , which are effectively mini-databases and include no program code. Thus, the replaceable cartridges may be updated and enhanced without changing the code. These cartridges ,  may be combined into a single cartridge.","The feed validator  will apply the relationships stored in the metadata database to the parsed data, assessing the data in terms of integrity rules , best practice rules , and statistical models  stored in the meta-data database . All relationships, including expert relationships  and computed relationships , will be stored along with human-readable text messages describing the relationships, or the underlying statistical model  in the case of computed relationships . Invalid data is sent back to the data source with a report  of data inconsistencies for correction, compiled using the human-readable messages. Valid data is sent to the data analysis and management engine (DAME) . The feed validator  may be partly implemented within the meta-data database  (expert relationships ) or partly implemented as modules, residing outside the database, that communicate with the feed stager and feed parser ","The DAME  performs standard operations on validated data  (e.g. updating statistical counts) prior to insertion into the patient database . The patient database is standards-aware and compliant, conforming to the templates in the standardized data model  of the metadata database . In a preferred embodiment, the DAME , the feed validator , and feed parser will be implemented as modules in an application server. These modules will then be utilized both by the user interface servers  and client-side applications to provide data input validation. In order to preserve data integrity in the path between feed stager and the patient database , data is moved in transactional sets such as a patient record. If some data are invalidated within a transactional set (say a missing patient name field), the whole set is rolled back and rejected without the generation of orphan records.","In a preferred embodiment, each information provider  generates and maintains local unique numeric identifiers for supplied records (e.g. patient ID) and uses this identifier when updating previously submitted patient data.","Validator Database","In a preferred embodiment, the system also includes a validator database  in which information is stored about all external entities who validate, or verify the correctness of, patient data . In the preferred embodiment, each validator is identified by a unique ORGANIZATION_ID as described above. In a preferred embodiment, every element of a patient record  in the patient data base  is associated with some validator  which is the outside entity that validated that data. The validator may be the clinician that submitted patient symptomatic  or treatment information , or a laboratory that submitted genetic information . Whenever a new patient record is validated by the feed validator  and added to the patient database , the new data is associated with the particular validator  that validated that submitted the data. For each validator , a set of profile information is stored (such as the institution, relevant contact information etc.) together with a record of the number of data records that have failed validation, and why, as well as the number of records that have passed validation. In this way, each validator can be gauged in terms of their reliability. The more reliable the validator, the more reliable and valuable the data. It is also possible that if information is sold, that information is priced based on the reliability of the validator, in addition to the value of the data itself. This concept is particularly relevant in the context of individuals being solicited for their genetic and phenotypic information by academic and research institutes, drug companies and others.","In one embodiment, illustrated in , the system has users  that are owners of valuable genetic and phenotypic information\u2014gene sellers \u2014as well as users  that are purchasers of genetic and phenotypic information\u2014gene buyers . In this scenario, the gene buyer  can view the information of the gene seller  as rendered by the UI server farm  with all personal identifiers removed and compliant with HIPAA requirements. In one embodiment, an entity termed the gene market manager  is responsible for putting gene buyers  in touch with gene sellers  while preserving the privacy rights of both parties. The market manager  has access to the contact information of both buyers  and sellers  while this information is filtered from each party by the UI server farm , which is protected behind a firewall  and various associated security measures discussed below. The market manager by also render other information for users  to indicate the market value of a certain information exchange, such as the price and terms of previous deals in which rights to relevant data was sold. In one embodiment, the market manager  makes a searchable database of previous transactions available for users  via the UI server . In one embodiment, when a transaction occurs, an electronic payment  is submitted by the buyer  to the system commercial interface  managed by the UI server farm , and a portion  is disbursed by the system interface  to the seller , a portion  to the market manager , a portion  to the validator  of the relevant information.","Distributed Users",{"@attributes":{"id":"p-0060","num":"0059"},"figref":"FIG. 6","b":["110","401","102","601","604","605","102","104","102","110","604","603","110","110","100","110","110"]},"Data Security and Robust User Authentication","The security architecture is illustrated in  and . In a preferred embodiment, the system will interact with users , namely caregivers , research organizations , or patients  by means of electronic messaging such as dynamic web-pages generated by the user interface servers  during a user's interaction with the system via the public internet . The system ensures the appropriate access to data for users by addressing security at multiple levels. In a preferred embodiment, user-level voluntary and involuntary password sharing is addressed by biometric authentication in addition to username\/password access control. The biometric authentication device will be local to the user , and connected via the public internet  to the user authentication engine  at which the user  is authenticated. There exist several technology options to fulfill this need, including a variety of fingerprint and iris recognition devices. For purposes of identity confirmation, patterns on the human iris are more complex than fingerprints or facial patterns. The false-acceptance and false rejection rate for iris recognition is typically lower than that for fingerprint recognition. One compelling current alternatives is the Panasonic DT 120 Authenticam which is a small, web-enabled iris-scanning camera which can be connected to a PC by means of a USB port. The camera uses the Private ID Iris Recognition System software developed by Iridian, together with the Iridian Know Who Server operating on an ORACLE database. Another cost-effective device candidate is the Targus Defcon fingerprint scanner which van interface to a client device via Universal Serial Bus (USB).","In one embodiment of a high security architecture, all traffic from outside users  will be encrypted with 128 bits, and will be received through the internet () by a UI server  residing behind a the firewall #1 . In the case of a UI server cluster, a load balancer  will be used to connect the user to the least loaded server in the farm. The firewall  will block all requests on all ports except those directly necessary to the system function. The function of the UI server is to encrypt\/decrypt packages and to serve UI screens. In a preferred embodiment, the UI servers  do not store any data. Each UI server will have two NICs (network interface cards) and will exist simultaneously on two subnets, one accessible from the outside and one not. The application server  (that hosts the DAME) may be blocked from the UI server by another firewall , and will also exist on two subnets. Subnet I () is behind the firewall, but accessible from the outside. Subnet II () is between UI servers and application server (), not accessible from the outside. In a preferred embodiment, the UI servers are separated from the application server by an additional Firewall II (). In a preferred embodiment, application server(s)  still do not contain any data, but they are aware of the rules for data retrieval and manipulation. Like UI servers, application server(s) also have 2 NICs and exist on 2 subnets simultaneously: Subnet II () is between UI servers  and application server(s) , one password layer away from the outside, and behind a firewall. Subnet III () is between application server(s) and database (), two password layers away from the outside, and behind one (optionally two) firewalls. Consequently, in the preferred embodiment, the database  is separated from the outside by two subnets and multiple firewalls.","In a preferred embodiment, the database  will be implemented using a robust industry-standard relational database platform from a vendor such as ORACLE and a UNIX-family operating system. In a preferred embodiment, access to each server is logged, and repetitive unsuccessful logins and unusual activities (possible attacks) are reported. Should local client data be stored, it will be encrypted (128 bits). Decryption keys are supplied by the server on logins and updated in regular intervals when client connects to the server (in case of offline work), or real time (in case of online work).","Operating Systems, Web Servers, Database and Programming Languages","In a preferred embodiment: The server operating system of choice is 64 bit LINUX. The database is based on ORACLE, insofar as it is both the industry leader for databases and an emerging leader in the life sciences industry and is fully interoperable with the LINUX operating system. System web servers run Apache, with PHP used to retrieve and construct dynamic content. This tool allows web page content to be published without the need for complex coding. An Extract-Transform-Load (ETL) tool is used to move data into the permanent database. The software languages are a combination of JAVA, PL\/SQL language and C\/C++. These languages are all widely adopted within the software industry and allow maximum flexibility to coders. Data Warehousing\/presentation tools will be supplied by Oracle. The front end application servers will be implemented in Java.","Example of the Data Taxonomy","In a preferred embodiment, patient data will span these four categories\n\n","In order to make this explanation concrete, the following example describes the information that may be contained in the standardized patient database, specifically pertaining to patients with AIDS and who have been or will be subjected to anti-retroviral therapy. Not all the information described here will typically be available. In addition, this describes both the type of information that would be in the standardized data model (profile information , symptomatic information , treatment information , diagnostic information ), as well as the type of information that would go into supporting tables (such as the drug dispense table and the lab test table). For illustrative purposes, the data classes related to patient diagnostic and treatment information have been labeled in brackets with a data class definition according to the UMLS Metathesaurus.\n\n",{"@attributes":{"id":"p-0067","num":"0208"},"figref":"FIG. 8","b":["100","122","106","202","203","117","104","103","103","407","100","108","117","100","123","124","204","108","123","124","117","203","106","124","100","108","208","210","123","204","106","123","100"],"i":["a","b"]},"Known Methods for Computing Statistical Relationships in the Standardized Data Model Database  and for Making Predictions ","This section describes statistical methods that can be applied in the DAME  to generate the computed statistical relationships  between the standardized data classes  that are contained in the standardized data model database , and that can be applied to generating feedback  about a particular subject. An exhaustive list is not provided, rather a limited subset of methods is provided to illustrate the concept of how the computed statistical relationships can be generated using the relevant set of subject data and used in the standardized data model database .","Determining the Validity of Data for a Patient  in a Subgroup :","It has been described how the DAME  can use a regression model of one or multiple variables to model some data class, or random variable for all patients in a subgroup , so that it is known when data - is sufficiently unlikely that it should be flagged as erroneous. It was assumed for simplicity that the distribution of the variable is approximately Gaussian; however a similar approach applies to variables with other probability distributions such as categorical variables. More details of one embodiment of the approach is now described, starting with modeling the probability distribution of a single random variable X. Assume that the true mean of the variable Xis \u03bc. The DAME estimates \u03bcwith the sample mean",{"@attributes":{"id":"p-0070","num":"0211"},"maths":{"@attributes":{"id":"MATH-US-00004","num":"00004"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"mover":{"msub":{"mi":["X","f"]},"mi":"_"},"mo":"=","mrow":{"mfrac":{"mn":"1","mi":"N"},"mo":"\u2062","mrow":{"munderover":{"mo":"\u2211","mrow":{"mi":"n","mo":"=","mn":"1"},"mi":"N"},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"msub":{"mi":"X","mrow":{"mi":["f","n"],"mo":","}}}}}},{"mrow":{"mo":["(",")"],"mn":"1"}}]}}}}},"The DAME computes the sample variance of Xover the set of N patients in the subgroup ",{"@attributes":{"id":"p-0072","num":"0213"},"maths":{"@attributes":{"id":"MATH-US-00005","num":"00005"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"msubsup":{"mi":["V","f"],"mn":"2"},"mo":"=","mrow":{"mfrac":{"mn":"1","mrow":{"mi":"N","mo":"-","mn":"1"}},"mo":"\u2062","mrow":{"munderover":{"mo":"\u2211","mrow":{"mi":"i","mo":"=","mn":"1"},"mi":"N"},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"msup":{"mrow":{"mo":["(",")"],"mrow":{"msub":{"mi":"X","mrow":{"mi":["f","i"],"mo":","}},"mo":"-","mrow":{"mfrac":{"mn":"1","mi":"N"},"mo":"\u2062","mrow":{"munderover":{"mo":"\u2211","mrow":{"mi":"n","mo":"=","mn":"1"},"mi":"N"},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"msub":{"mi":"X","mrow":{"mi":["f","n"],"mo":","}}}}}},"mn":"2"}}}}},{"mrow":{"mo":["(",")"],"mn":"2"}}]}}}}},"Given the DAME's estimate of mean and variance of the subgroup , assume that it next considers the data class Xassociated with a new patient data , and finds that it has some value xwhich is greater than the sample mean. DAME can estimate the probability of this value, given the assumptions that patient N+1 belongs in the same subgroup  as the other N patients . Namely, assuming that Xis a normally distributed random variable, and given the measurements of sample mean {right arrow over (X)}and sample variance Vbased on the previous N samples, the DAME can determine the probability PR{X>=x}. If the DAME  finds this probability to be very small (below some threshold) then it questions whether Xis valid and whether patient N+1 belongs in the same subgroup  as the previous N patients. Let the estimated variance of Xbe s(X\u2212 ) and break down the value x:\n\n() \u2003\u2003(3)\n\nwhere tis some multiplier of the sample variance that determines the extent of the distance from the sample mean to the value x. It can be shown that the probability Pr{X>=x} can be rewritten as:\n",{"@attributes":{"id":"p-0074","num":"0215"},"maths":{"@attributes":{"id":"MATH-US-00006","num":"00006"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"mrow":[{"mi":"Pr","mo":"\u2062","mrow":{"mo":["{","}"],"mrow":{"mfrac":{"mrow":{"msub":[{"mi":"X","mrow":{"mi":"f","mo":",","mrow":{"mi":"N","mo":"+","mn":"1"}}},{"mover":{"mi":["X","_"]},"mi":"f"}],"mo":"-"},"msqrt":{"mrow":{"msubsup":{"mi":["V","f"],"mn":"2"},"mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mn":"1","mo":"+","mfrac":{"mn":"1","mi":"N"}}}}}},"mo":">=","msub":{"mi":["t","f"]}}}},{"mi":"Pr","mo":"\u2062","mrow":{"mo":["{","}"],"mrow":{"mfrac":{"mfrac":{"mrow":[{"msub":[{"mi":"X","mrow":{"mi":"f","mo":",","mrow":{"mi":"N","mo":"+","mn":"1"}}},{"mover":{"mi":["X","_"]},"mi":"f"}],"mo":"-"},{"msub":{"mi":["\u03c3","f"]},"mo":"\u2062","msqrt":{"mrow":{"mn":"1","mo":"+","mfrac":{"mn":"1","mi":"N"}}}}]},"msqrt":{"mrow":{"mfrac":[{"mrow":{"msubsup":{"mi":["V","f"],"mn":"2"},"mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mi":"N","mo":"-","mn":"1"}}},"msubsup":{"mi":["\u03c3","f"],"mn":"2"}},{"mn":"1","mrow":{"mi":"N","mo":"-","mn":"1"}}],"mo":"\u2062"}}},"mo":">=","msub":{"mi":["t","f"]}}}}],"mo":"="}},{"mrow":{"mo":["(",")"],"mn":"4"}}]}}}}},"Now, (X\u2212 )\/(\u03c3\u221a{square root over (1+1\/N)}) is a normally distributed random variable with unit variance and zero mean, and it can be shown that V(N\u22121)\/\u03c3is a chi-squared random variable with N\u22121 degrees of freedom. Therefore, the left hand side of the inequality has a chi-squared distribution with N\u22121 degress of freedom, and can be set equal to random variable Twhich has a Student-T distribution with N\u22121 degrees of freedom f(t). So the probability Pr{X>=x} can be computed as",{"@attributes":{"id":"p-0076","num":"0217"},"maths":{"@attributes":{"id":"MATH-US-00007","num":"00007"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"mrow":[{"mi":"Pr","mo":"\u2062","mrow":{"mo":["{","}"],"mrow":{"msub":[{"mi":"T","mrow":{"mi":"N","mo":"-","mn":"1"}},{"mi":["t","f"]}],"mo":">="}}},{"mrow":[{"munderover":{"mo":"\u222b","msub":{"mi":["t","f"]},"mi":"\u221e"},"mo":"\u2062","mrow":{"mrow":[{"msub":{"mi":"f","mrow":{"mi":"T","mo":",","mrow":{"mi":"N","mo":"-","mn":"1"}}},"mo":"\u2061","mrow":{"mo":["(",")"],"mi":"t"}},{"mo":"\u2146","mi":"t"}],"mo":"\u2062"}},{"msub":{"mi":"F","mrow":{"mi":"T","mo":",","mrow":{"mi":"N","mo":"-","mn":"1"}}},"mo":"\u2061","mrow":{"mo":["(",")"],"msub":{"mi":["t","f"]}}}],"mo":"="}],"mo":"="}},{"mrow":{"mo":["(",")"],"mn":"5"}}]}}}}},"A value for tis picked such the cumulative Student-T distribution function with N\u22121 degrees of freedom, F(t), is equal to some probability bound, \u03b1\/2. This value of tis denoted as t(\u03b1\/2, N\u22121)\u2014this notation will be used in a later section. Then, assuming that the data is valid, there is a probability of 1\u2212\u03b1\/2 that T<t. So far, only the upper bound on Xhas been considered; the lower bound is dealt with just the same way. Considering the upper and lower bound, there is a probability of a that |T|>=t. If \u03b1 is (based on the expected reliability of the data) sufficiently small, but the DAME finds that |T|>=t, then the DAME  will flag the new patient data as potentially containing a bad data class Xor being in the wrong subgroup .","The Use of Expert Relationships  and Computed Statistical Relationships   illustrates the synergy between expert relationships  and computed relationships  in one embodiment of the invention. This is illustrated with reference to a particular example, where the data class of interest is future CD4+ cell count of a patient with Human Immunodeficiency Virus HIV. This data class is divided in the figure into two subclasses namely past CD4+ count  and future CD4+ count . In this example, the DAME  makes a prediction on the future value of CD4+ count based on a series of relevant data classes from the data model standards -, combined with expert relationships  and computed relationships  stored in the meta-data databases .","The DAME  uses the algorithmic relationships from experts ,  in order to determine which variables are relevant to modeling future CD  count , and in what way these relevant variables should be included in the model. In one embodiment, the algorithmic relationships , from various experts are electronically published according to a standard template and downloaded for conversion by the feed parser into a form that it can be stored in the expert relationships database  and applied to the data model standard . In order for an expert to specify a statistical model, the expert publishes the standardized data classes, or independent variables, used by the model; a training function that trains the parameters for the model; and a mapping function that inputs the relevant standardized data classes and the trained parameters in order to predict the datum of interest. For example, each hypothesis validated in a clinical trial may be electronically published in this way. In one embodiment, the expert also publishes as set of raw patient data that is either formatted according to the standardized data classes, or is accompanied by a translation cartridge, so that the data can be automatically inhaled into the standardized ontology, and can be acted upon by the published training and mapping functions in order to replicate the results published by the expert, and to refine these results as more data is inhaled. The algorithmic relationships , illustrated in  specify the set of data classes that different experts used to predict the relevant outcome , as well as a training function and mapping function describing how those data classes were mathematically manipulated in order to model the relevant outcome . The item \u201call relevant data classes\u201d  represents the union of all of the relevant data classes used as independent variables by different statistical models in the algorithmic relationships database  that relate to the relevant outcome . In this illustrative example, expert  includes nine variables - in the model for future CD4+ count , but does not include body mass data class  or mutations   to mutations A  data classes, which may represent human or viral mutations, as relevant independent variables in the model. Expert 2 does include body mass and mutation   in the model, but not include the data classes relating to the other mutations up to mutation A . In one embodiment, the model is a regression analysis model. In this case, the DAME  is guided by the algorithmic relationships ,  in terms of what independent variables to select for regression analysis, and what mathematical operations to perform on those variables in order to include them in the regression model so that it may compute regression parameters for the computed relationships database . The DAME  may then select a particular model, or select a particular subset of independent variables in a model, in order to make the best prediction.","Application of Regression Analysis to Generating Computed Relationships  and Feedback :","In order to illustrate the use of regression analysis according to one embodiment, assume that a clinician is addressing the problem of what mixture of Anti-Retroviral Drugs (ARV) should be given to a patient at a particular stage of progression of an HIV\/AIDS infection and immune system degradation. In order to illustrate the statistical approach, three separate questions are considered that can be addressed by the DAME . These example questions are merely illustrative and by no means exhaustive:\n\n","For simplicity, assume only the ARV drug dosages are considered as relevant independent variables for modeling the progression of the disease. Assume that the progress of the HIV\/AIDS virus is tracked by monitoring {right arrow over (X)}, the measure of the Total Lymphocite Count, over time. It has recently been shown that TLC count and Hemoglobin levels are an effective measure of the progress of the HIV\/AIDS infection, and can be used instead of the more expensive CD4+ cell counts. Of course, nothing in this example would change if use was made of the more conventional CD4+ count to track disease progress. The set of data classes relevant to our hypothetical example are:\n\n","One can monitor how data class {right arrow over (X)}varies with respect to the data classes {right arrow over (X)}, {right arrow over (X)}and {right arrow over (X)}by establishing a regression model:",{"@attributes":{"id":"p-0083","num":"0231"},"maths":{"@attributes":{"id":"MATH-US-00008","num":"00008"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mrow":[{"msub":{"mi":["f","TLC"]},"mo":"\u2061","mrow":{"mo":["(",")"],"msub":{"mover":{"mi":"X","mo":"\u2192"},"mrow":{"mi":["TLC","n","t"],"mo":[",",","]}}}},{"mrow":{"munderover":{"mo":"\u2211","mrow":[{"mi":"p","mo":"=","mn":"0"},{"mi":"P","mo":"-","mn":"1"}]},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"msub":{"mi":["\u03b2","p"]},"mo":"\u2062","mrow":{"msub":{"mi":["f","p"]},"mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msub":[{"mover":{"mi":"X","mo":"\u2192"},"mrow":{"mrow":{"mi":"ARV","mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mn":"1"},"mo":[",",","],"mi":["n","t"]}},{"mover":{"mi":"X","mo":"\u2192"},"mrow":{"mrow":{"mi":"ARV","mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mn":"2"},"mo":[",",","],"mi":["n","t"]}},{"mover":{"mi":"X","mo":"\u2192"},"mrow":{"mrow":{"mi":"ARV","mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mn":"3"},"mo":[",",","],"mi":["n","t"]}}],"mo":[",",","]}}}}},"mo":"+","msub":{"mi":["\u025b","n"]}}],"mo":"="}}},"br":[{},{}],"sub":["TLC,n,t","ARV1,n,t","ARV2,n,t ","ARV3,n,t","0 ","P-1 ","n","TLC ","p","TLC ","\u03b1","n ","n","TLC","n,t,TLC","TLC,n,t","TLC,n,t-t",{"sub2":"\u03b1"}],"in-line-formulae":[{},{}],"i":["Y","=f","{right arrow over (X)}","X","X"]},"Of course, many different functions fare possible. Often, in designing the function for generating the dependent variable, the goal is to make the resultant variable as linear in the independent variables as possible. Equation (6) assumes that the use of the logarithm would be effective. However, this is purely illustrative and is not necessarily an optimal approach. Let functions f. . . fbe linear in the data {right arrow over (X)}, {right arrow over (X)}and {right arrow over (X)}, and represent the average dosage of each of the ARTs for the period of t months leading up to t. Assume for the example that the number of parameters P=4 and simplify the above Equation as follows:",{"@attributes":{"id":"p-0085","num":"0233"},"maths":{"@attributes":{"id":"MATH-US-00009","num":"00009"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":[{"mtd":[{"mrow":{"mrow":[{"msub":{"mi":["f","TLC"]},"mo":"\u2061","mrow":{"mo":["(",")"],"msub":{"mover":{"mi":"X","mo":"\u2192"},"mrow":{"mi":["n","t","TLC"],"mo":[",",","]}}}},{"msub":[{"mi":"\u03b2","mn":"0"},{"mi":["\u025b","n"]}],"mo":["+","+"],"mrow":{"munderover":{"mo":"\u2211","mrow":[{"mi":"p","mo":"=","mn":"1"},{"mi":"P","mo":"-","mn":"1"}]},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"msub":{"mi":["\u03b2","p"]},"mo":"\u2062","mrow":{"msub":{"mi":["f","p"]},"mo":"\u2061","mrow":{"mo":["(",")"],"msub":{"mover":{"mi":"X","mo":"\u2192"},"mrow":{"mi":["ARVp","n","t"],"mo":[",",","]}}}}}}}],"mo":"="}},{"mrow":{"mo":["(",")"],"mn":"7"}}]},{"mtd":[{"mstyle":{"mtext":"where"}},{"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}}}]},{"mtd":[{"mrow":{"mrow":[{"msub":{"mi":["f","p"]},"mo":"\u2061","mrow":{"mo":["(",")"],"msub":{"mover":{"mi":"X","mo":"\u2192"},"mrow":{"mi":["ARVp","n","t"],"mo":[",",","]}}}},{"mfrac":{"mn":"1","msub":{"mi":["t","a"]}},"mo":"\u2062","mrow":{"munderover":{"mo":"\u2211","mrow":{"mi":"k","mo":"=","mrow":{"mi":"t","mo":"-","msub":{"mi":["t","a"]}}},"mi":"t"},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"msub":{"mi":"X","mrow":{"mi":["ARVp","n","k"],"mo":[",",","]}}}}],"mo":"="}},{"mrow":{"mo":["(",")"],"mn":"8"}}]}]}}}},"Define the new independent regression variables X=f({right arrow over (X)}) for p\u22600 and X=1 and so that the fundamental regression Equation is rewritten",{"@attributes":{"id":"p-0087","num":"0235"},"maths":{"@attributes":{"id":"MATH-US-00010","num":"00010"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"mrow":[{"msub":{"mi":["Y","n"]},"mo":"=","mrow":{"mrow":{"munderover":{"mo":"\u2211","mrow":[{"mi":"p","mo":"=","mn":"0"},{"mi":"P","mo":"-","mn":"1"}]},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"msub":[{"mi":["\u03b2","p"]},{"mi":"X","mrow":{"mi":["p","n"],"mo":","}}],"mo":"\u2062"}},"mo":"+","msub":{"mi":["\u025b","n"]}}},{"mi":"n","mo":"=","mrow":{"mn":"1","mo":["\u2062","\u2062","\u2062","\u2062"],"mstyle":[{"mspace":{"@attributes":{"width":"0.6em","height":"0.6ex"}}},{"mspace":{"@attributes":{"width":"0.6em","height":"0.6ex"}}}],"mi":["\u2026","N"]}}],"mo":[",","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}}}},{"mrow":{"mo":["(",")"],"mn":"9"}}]}}}}},"Represent these equations for N patients in the form of a matrix equation\n\n\u2003\u2003(10)\n","Where",{"@attributes":{"id":"p-0090","num":"0238"},"maths":{"@attributes":{"id":"MATH-US-00011","num":"00011"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"mrow":[{"mi":"Y","mo":"=","mrow":{"mo":["[","]"],"mtable":{"mtr":[{"mtd":{"msub":{"mi":"Y","mn":"0"}}},{"mtd":{"mi":"\u22ee"}},{"mtd":{"msub":{"mi":"Y","mrow":{"mi":"n","mo":"-","mn":"1"}}}}]}}},{"mi":"X","mo":"=","mrow":{"mo":["[","]"],"mtable":{"mtr":[{"mtd":[{"mn":"1"},{"msub":{"mi":"X","mrow":{"mn":["1","0"],"mo":","}}},{"mi":"\u22ef"},{"msub":{"mi":"X","mrow":{"mrow":{"mi":"P","mo":"-","mn":"1"},"mo":",","mn":"0"}}}]},{"mtd":[{"mi":"\u22ee"},{"mi":"\u22ee"},{"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}}},{"mi":"\u22ee"}]},{"mtd":[{"mn":"1"},{"msub":{"mi":"X","mrow":{"mn":"1","mo":",","mrow":{"mi":"N","mo":"-","mn":"1"}}}},{"mi":"\u22ef"},{"msub":{"mi":"X","mrow":{"mrow":[{"mi":"P","mo":"-","mn":"1"},{"mi":"N","mo":"-","mn":"1"}],"mo":","}}}]}]}}},{"mi":"b","mo":"=","mrow":{"mo":["[","]"],"mtable":{"mtr":[{"mtd":{"msub":{"mi":"b","mn":"0"}}},{"mtd":{"mi":"\u22ee"}},{"mtd":{"msub":{"mi":"b","mrow":{"mi":"N","mo":"-","mn":"1"}}}}]}}},{"mi":"\u025b","mo":"=","mrow":{"mo":["[","]"],"mtable":{"mtr":[{"mtd":{"msub":{"mi":"\u025b","mn":"0"}}},{"mtd":{"mi":"\u22ee"}},{"mtd":{"msub":{"mi":"\u025b","mrow":{"mi":"N","mo":"-","mn":"1"}}}}]}}}],"mo":[",",",",","]}},{"mrow":{"mo":["(",")"],"mn":"11"}}]}}}}},"Assuming that the disturbance is normally distributed, \u03b5\u02dcN(0,I\u03c3), the least-squares, and maximum-likelihood, solution to Equation (15) is given by:\n\n=()\u2003\u2003(12)\n","It should be noted that if (as in the more general case) the disturbances are correlated and have covariance C and non-zero mean \u03bc, so that \u03b5\u02dcN(\u03bc,C), then the least-squares solution is b=(XCX)XC(Y\u2212\u03b7). Based on the regression model, for some new patient N, and a proposed ART dosage X, one can predict what the expected change in the patients TLC count will be:\n\n\u0176=Xb \u2003\u2003(13)\n\nUsing the Confidence Bounds for Multivariate Regression:\n","One also seeks to determine the confidence bounds for the prediction so that one can, for example, judge when a particular patient is not responding as expected to a particular treatment regimen or when their data is not internally consistent. As discussed above, the 1\u2212\u03b1 confidence bound for Yis given by:",{"@attributes":{"id":"p-0094","num":"0242"},"maths":{"@attributes":{"id":"MATH-US-00012","num":"00012"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"msub":{"mover":{"mi":"Y","mo":"^"},"mi":"N"},"mo":"\u00b1","mrow":{"mrow":[{"mi":"t","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mrow":[{"mn":"1","mo":"-","mfrac":{"mi":"\u03b1","mn":"2"}},{"mi":["N","P"],"mo":"-"}],"mo":","}}},{"mi":"s","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msub":[{"mover":{"mi":"Y","mo":"^"},"mi":"N"},{"mi":["Y","N"]}],"mo":"-"}}}],"mo":"\u2062"}}},{"mrow":{"mo":["(",")"],"mn":"14"}}]}}}},"br":{}},{"@attributes":{"id":"p-0095","num":"0243"},"maths":{"@attributes":{"id":"MATH-US-00013","num":"00013"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mi":"t","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mrow":[{"mn":"1","mo":"-","mfrac":{"mi":"\u03b1","mn":"2"}},{"mi":["N","P"],"mo":"-"}],"mo":","}}}}},"br":[{},{}],"sub":["T,N-P","N","N","N","N","N","N","N","N","N","N"],"in-line-formulae":[{},{}],"sup":["2","2","2","2","T","T","\u22121","2 "],"i":["\u0176","\u2212Y","\u0176","Y","X","X","X","X"]},"Since the actual value of \u03c3is not know, use the unbiased estimator termed the Mean-Squared-Error (MSE) which measures the Sum of the Squared Error (SSE) of the estimate, per degree of freedom. The SSE, as defined, is computed from:\n\nSSE=()(\u2003\u2003(16)\n\nwhere \u0176=Xb. There are N measurements with uncorrelated disturbances, or N \u201cdegrees of freedom\u201d, which go into generating Y. Since one must estimate P parameters in order to create the estimate \u0176, one gives up P degrees of freedom in SSE, so the total degrees of freedom in SSE is N\u2212P. Hence one computes\n",{"@attributes":{"id":"p-0097","num":"0245"},"maths":{"@attributes":{"id":"MATH-US-00014","num":"00014"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"mi":"MSE","mo":"=","mfrac":{"mi":"SSE","mrow":{"mi":["N","P"],"mo":"-"}}}},{"mrow":{"mo":["(",")"],"mn":"17"}}]}}}}},"It can be shown that the expectation value E{MSE}=\u03c3. Consequently,\n\n()=(()+1) \u2003\u2003(18)\n","Hence, if patient N has a decrease in TLC count which places the patient's data out of the range of",{"@attributes":{"id":"p-0100","num":"0248"},"maths":{"@attributes":{"id":"MATH-US-00015","num":"00015"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mrow":{"msub":{"mover":{"mi":"Y","mo":"^"},"mi":"N"},"mo":"\u00b1","mrow":{"mrow":[{"mi":"t","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mrow":[{"mn":"1","mo":"-","mfrac":{"mi":"\u03b1","mn":"2"}},{"mi":["N","P"],"mo":"-"}],"mo":","}}},{"mi":"s","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msub":[{"mover":{"mi":"Y","mo":"^"},"mi":"N"},{"mi":["Y","N"]}],"mo":"-"}}}],"mo":"\u2062"}},"mo":","}}},"br":[{},{}],"b":"128"},"Regarding the second question, how does one determine the level of specificity with which to subdivide patients into groups to determine regression parameters (or other statistical properties) of a particular group. A simple way of addressing this issue, in a preferred embodiment, is to seek the lower bound that it places on the number of patients that are necessary as data points in order for the results of the analysis to be statistically significant. For example, consider the situation described above, except add the complication that the response of patients to different ARV regimens varies significantly based on the stage of degradation of the patient's immune system. In this case, rather than create a single linear model as described in Equation (15), create a piecewise linear model. The relevant group is determined by the patient's TLC count when ARV treatment is initiated at time t\u2212t. The model can then be characterized:",{"@attributes":{"id":"p-0102","num":"0250"},"maths":{"@attributes":{"id":"MATH-US-00016","num":"00016"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"mi":"Y","mo":"=","mrow":{"mo":["{","}"],"mtable":{"mtr":[{"mtd":[{"mrow":{"mrow":{"msub":[{"mi":"X","mn":"0"},{"mi":"b","mn":"0"}],"mo":"\u2062"},"mo":"+","msub":{"mi":"\u025b","mn":"0"}}},{"mstyle":{"mtext":"for"}},{"mrow":{"msub":[{"mi":"x","mn":"0"},{"mi":"X","mrow":{"mi":["n","TLC"],"mo":[",",","],"mrow":{"mi":["t","l"],"mo":"-"}}},{"mi":"x","mn":"1"}],"mo":["\u2264","<"]}}]},{"mtd":[{"mrow":{"mrow":{"msub":[{"mi":"X","mn":"1"},{"mi":"b","mn":"1"}],"mo":"\u2062"},"mo":"+","msub":{"mi":"\u025b","mn":"1"}}},{"mstyle":{"mtext":"for"}},{"mrow":{"msub":[{"mi":"x","mn":"1"},{"mi":"X","mrow":{"mi":["n","TLC"],"mo":[",",","],"mrow":{"mi":["t","l"],"mo":"-"}}},{"mi":"x","mn":"2"}],"mo":["\u2264","<"]}}]},{"mtd":[{"mi":"\u22ee"},{"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}}},{"mi":"\u22ee"}]},{"mtd":[{"mrow":{"mrow":{"msub":[{"mi":"X","mrow":{"mi":"D","mo":"-","mn":"1"}},{"mi":"b","mrow":{"mi":"D","mo":"-","mn":"1"}}],"mo":"\u2062"},"mo":"+","msub":{"mi":"\u025b","mrow":{"mi":"D","mo":"-","mn":"1"}}}},{"mstyle":{"mtext":"for"}},{"mrow":{"msub":[{"mi":"x","mrow":{"mi":"D","mo":"-","mn":"1"}},{"mi":"X","mrow":{"mi":["n","TLC"],"mo":[",",","],"mrow":{"mi":["t","l"],"mo":"-"}}},{"mi":["x","D"]}],"mo":["\u2264","<"]}}]}]}}}},{"mrow":{"mo":["(",")"],"mn":"19"}}]}}}}},"Rather than determine a single set of parameters, b, determine D sets of parameters, each for a different domain of TLC count, or a different domain of disease progression. The data can be divided into domains, D, based on the amount of patient data that is available in order to determine the parameters relevant to each domain in a statistically significant manner. Divide up the total set of N patients into each of the relevant domains such that",{"@attributes":{"id":"p-0104","num":"0252"},"maths":{"@attributes":{"id":"MATH-US-00017","num":"00017"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mi":"N","mo":"=","mrow":{"munderover":{"mo":"\u2211","mrow":[{"mi":"i","mo":"=","mn":"0"},{"mi":"D","mo":"-","mn":"1"}]},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"msub":{"mi":["N","d"]},"mo":"."}}}}},"br":[{},{},{},{}],"sub":["d ","d","d ","d","d","d","d","d","d,p","d","d,p","d,p ","d,p","d","d,p"],"sup":["2","2","T","\u22121 ","2","2"],"in-line-formulae":[{},{},{},{}],"i":["S","b","X","X"]},"Assume a choice of \u03b1=0.05 so that one finds the 90% confidence interval. One may now compare the parameters calculated for domain d with those parameters calculated for domains d\u22121 and d+1. In one embodiment, one wishes to establish that the differences between the parameters in adjacent domains are greater than the confidence interval for the estimates of parameters in the domains. For example, in a preferred embodiment, one seeks to ensure that\n\n|(1\u2212\u03b1\/2)() \u2003\u2003(22)\n","Note that the setting of \u03b1 is the choice of the user of system, and determines the level of confidence that is sought in the bounds that are defined for the regression parameters. If the above inequalities are not satisfied, then the solution may be to increasing the size of each group so that the resulting parameters for each group are clearly differentiated in a statistically significant way.","Recommending Preferred Treatment Methodologies Using Computed Relationships  Based on Statistical Models :","The third question is new addressed, namely how one uses these tools that perform statistical analysis between data classes to recommend preferred methodologies for treatment. Applying the technique described above, the response of a subject can be modeled in terms of the expected change in their TLC count over some time period t during which they are subjected to a particular regimen of ARV's. This response is based on the data of previous subjects whose data contains the relevant independent variables\u2014in this case, ART dosage\u2014and for whom the relevant outcome has been measured\u2014in this case TLC count. By breaking down subjects into smaller groups, one creates an accurate model for Y, based on X, where d is the domain or group into which the subject fits. In each domain d, based on the parameters bthat are estimated, One has an unbiased estimate of the change expected in TLC count with respect to a particular drug, p:",{"@attributes":{"id":"p-0108","num":"0256"},"maths":{"@attributes":{"id":"MATH-US-00018","num":"00018"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"mfrac":{"mrow":[{"mo":"\u2202","mover":{"mi":"Y","mo":"^"}},{"mo":"\u2202","msub":{"mi":"X","mrow":{"mi":["d","p"],"mo":","}}}]},"mo":"=","msub":{"mi":"b","mrow":{"mi":["d","p"],"mo":","}}}},{"mrow":{"mo":["(",")"],"mn":"23"}}]}}}}},"In the context of ARV medication, because the system is not linear, one could not say that because bis positive, one will increase the expected value of \u0176 indefinitely as one increases the dosage of ARV p. One approach, in a preferred embodiment, is that data class ARV p already has a expert-based rule associated with it, namely that this ARV p should not be given in doses exceeding a certain quantity either because it causes side effects or because the dosage reaches a point of diminishing returns. Consequently, if bis large and positive, the DAME  will prescribe a dosage of ARV p that is the limit defined by the expert-based rules.","Another approach to determining the best drug regimen would involve selecting those patients showing the largest change in Y, and modeling the parameters b for that set of highly successful cases. Initially, one doesn't know what combination of ARVs will perform best. Assume one has {tilde over (P)} different drugs to choose from, and that the number of drugs that a patient can be administered is P. The initial process would involve cycling through each of the {tilde over (P)} drugs for a range of patients, each of whom only receives a set of P drugs, until one can determine with a reasonable degree of certainty what drugs result in the best performance for each subgroup. Assume that one cycles through a total of N patients, and then select the set of Npatients who were most successful under their dose of drugs. Estimate, using the technique described above, the parameters bfor the set of Nsuccessful patients, and also estimate the set of parameters bfor the set of N\u2212Nunsuccessful (or less successful) patients. Then use a similar technique to that described above to ensure that the different between band bis statistically significant. In other words, Nand N\u2212Nshould be large enough such that for some confidence value a that depends on the particular case at hand:\n\n|(1\u2212\u03b1\/2)((1\u2212\u03b1\/2)(); =1 \u2003\u2003(24)\n","Then, select the top P largest values of bto indicate the drugs that should be used on subsequent patients. Note that the method of testing a range of inputs in order to model the response of a system is known in the art as persistent excitation, and there are many techniques for determining more optimally how to collect experimental data to a model a system in an optimal way so that the system can be identified as well as a particular response achieved. One seeks to identify the optimal parameters b and at the same time achieve the greatest positive change in Y for the greatest number of patients. Different methods can be applied than those outlined above without changing the fundamental idea. Of course, one could also test different dosages of drugs as well as testing the drugs in different combinations","Another approach to predicting the best possible drug regimen is to predict the outcome for the patient for each of a set of available or recommended drug regimens, and then simply select that regimen which has the best predicted outcome.","It will be clear to one skilled in the art that various other techniques exist for predicting an outcome, or the probability of an outcome, using the relationships in the standardized data model. These different techniques would be stored as different expert statistical models . For example, consider one is trying to predict the likelihood of a certain state being reached, such as CD4 count below 200\/uL. Rather than use the regression model described above, another approach is to consider the time to progression of the particular outcome. Different time-to-outcome estimates between different treatment methodologies can be compared using techniques that are known in the art such as log-rank tests. In order to evaluate different treatment methodologies from data within the patient data base , the time-to-outcome for each group treated in a different manner can be constructed using, for example, the Kaplan-Meier product limit method. It should also be noted that there will often be significant differences between the data in different patient groups which would affect outcome and is not related to the particular variable under consideration, such as for example treatment regimen. This will often occur when data from different clinical studies is aggregated, for example. Methods are known in the art to perform adjusted analyses to compare the time-to-outcome distribution for different groups of patients. For example, the method of Cox's proportional hazards model adjusts for the effect of the covariates that are predictive of progression-to-outcome. For example, in predicting the time to CD4 count below 200\/uL, these covariates, in addition to treatment type, will include such factors as age, gender, body mass index, diet, baseline CD4 and viral load counts at time of initial treatment, and treatment adherence. These adjusted comparisons entail score tests that are more powerful because of covariate adjustments. The proportional hazards assumption can be checked by considering the Schoenfeld residuals for significant covariates and interactions. When the proportional hazards assumption is violated, several approaches may be considered. One approach would be to use a stratified proportional hazards model in which the proportional hazards assumption holds within each stratum. Another approach introduces appropriate time-dependent covariates in place of those covariates with hazard functions non-proportional to the baseline hazard. An appropriate Cox model with time-dependent covariates can then be fitted.","Note that many other expert statistical models  can be used by the DAME  in selecting between different treatment strategies. For example, when the effectiveness of different ART treatments are being compared using outcomes that are continuous in nature, such as the level of CD4 count. A repeated measures Analysis of Variance (ANOVA) could be used to compare mean outcome profiles between different treatment arms. The analysis in this case could use canonical methods such as a Diggle's mixed-effects-model. In this case, treatment type would be a fixed effect (other covariates are possible), while patients will be the random effects. Appropriate transformations such as Box-Cox will be applied to these continuous outcomes to achieve approximately normality. These methods are known to those skilled in the art. Additional canonical methods can be brought to bear on outcomes that are categorical in nature. For example, one might be interested in whether it is more likely for a particular mutation to develop in the HIV virus after the patient is subjected for one year to one ART regiment instead of another. This type of categorical outcome can be analyzed using Fisher's exact test for 2\u00d72 contingency tables using the method of difference-in-proportions to measure dependence. When multiple different ART's are considered, one can use the more general Fisher-Freeman-Halton test with the likelihoods ratio test to determine dependence.","Other Approaches to Statistical Models  and Computed Relationships  for Predicting Outcome and Validating Subject's Data :","Another regression technique, which doesn't change the fundamental idea discussed above, is to take into account the interactions between the independent variables when modeling the response of a dependent variable, so that one no longer treats the system as linear. Instead, the model involves product terms between the dependent variables, and can be displayed in very general form as:",{"@attributes":{"id":"p-0116","num":"0264"},"maths":{"@attributes":{"id":"MATH-US-00019","num":"00019"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"mrow":[{"msub":{"mi":["Y","n"]},"mo":"=","mrow":{"msub":[{"mi":"\u03b2","mn":"0"},{"mi":["\u025b","n"]}],"mo":["+","+"],"mrow":{"munderover":{"mo":"\u2211","mrow":[{"mi":"i","mo":"=","mn":"1"},{"mi":"I","mo":"-","mn":"1"}]},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"msub":{"mi":["\u03b2","i"]},"mo":"\u2062","mrow":{"munderover":{"mo":"\u220f","mrow":[{"mi":"p","mo":"=","mn":"0"},{"mover":{"mi":"P","mo":"~"},"mo":"-","mn":"1"}]},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"msubsup":{"mi":"X","mrow":{"mi":["n","p"],"mo":","},"msub":{"mi":"power","mrow":{"mi":["p","i"],"mo":","}}}}}}}},{"mi":"n","mo":"=","mrow":{"mn":"1","mo":["\u2062","\u2062","\u2062","\u2062"],"mstyle":[{"mspace":{"@attributes":{"width":"0.6em","height":"0.6ex"}}},{"mspace":{"@attributes":{"width":"0.6em","height":"0.6ex"}}}],"mi":["\u2026","N"]}}],"mo":[",","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}}}},{"mrow":{"mo":["(",")"],"mn":"25"}}]}}}},"br":{},"sub":["i ","p,i ","i","p,i ","n"],"sup":["th ","th "]},"Notice that there is nothing in the description provided which restricts the independent variables to be derived from the patient treatment information data class, nor which limits the dependent variable to be derived from the patient symptomatic information data class. One could apply a similar set of techniques to data classes from the patient diagnostic information; including the vast body of genetic data where the set of independent variables refer to whether or not the patient has a particular allele of a particular gene. In this case, the dependent variable could refer to any phenotypic trait, such as height, eye color, propensity for breast cancer, propensity for psoriasis, propensity for heart illness etc. Using the same fundamental techniques described above, the DAME  can generate predictions about these phenotypic traits based on the genetic data in the patient diagnostic information data class, as well as based on an assessment of the validity or confidence bounds of these predictions.","In one embodiment, an individual's genetic information is used by the DAME  as a tool for Customer Relation Management (CRM), based on the phenotypic predictions that can be made, or expert rules that can be applied to, this genetic information. For example, an individual could ask to have information filtered to suite their individual needs, based on their genetic code. The DAME  would forward to this individual via the UI server  advertisements, or information about particular items of food, clothing or other items which would match the individual's nutritional needs, physical shape or mental propensities, based on their genetic dispositions. Companies wanting to target marketing effectively could provide a range of marketing material to the system, which could then be forwarded to suitable recipients based on the genetic information and contact information that reside within the data base . There are of course a host of other CRM applications, based on either customer pull or seller push to which the system can be applied, using the fundamental concepts disclosed herein.","Additional services, similar to the customer-pull services outlined above, involve individuals interacting with the UI server  to ask questions based on genetic information that the individuals provide. The DAME  could be used by primary care providers when they prescribe medication tailored for an individual, ranging from medication as general as pain killers, to medication for conditions such as psoriasis, cancer, HIV\/AIDS etc. The care provider would submit the subject's genetic and clinical information, in order for the DAME  to predict the expected outcome in response to different proposed drugs.","In another usage scenario, two potential parents might provide their genetic information in order for the DAME  to predict what would be their probability for producing a child with particular traits such as height, eye-color, probability for psoriasis, probability for breast cancer, particular mental propensities etc. The DAME  could make these predictions based on the same concepts disclosed previously. In performing a genetic analysis on a particular phenotypic feature, f, the DAME  would perform computation on the genetic data classes known to influence that feature f. For example, if f represents the probability that the individual develops psoriasis, the system would base the genetic analysis on a set of data classes referring to alleles of particular genes, where those alleles affects the likelihood of psoriasis. Let us assume that there are Nsuch data classes represented by X, i=1 . . . N, where the values of the data classes X=1 or 0, depending on whether or not the particular alleles of the relevant genes are present in individual n. Using the general formulation above, and assuming that Yrepresents that feature f for an individual N,",{"@attributes":{"id":"p-0121","num":"0269"},"maths":{"@attributes":{"id":"MATH-US-00020","num":"00020"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"msub":{"mi":["Y","N"]},"mo":"=","mrow":{"msub":{"mi":"\u03b2","mn":"0"},"mo":["+","+"],"mrow":{"munderover":{"mo":"\u2211","mrow":[{"mi":"p","mo":"=","mn":"1"},{"mi":"P","mo":"-","mn":"1"}]},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"msub":{"mi":["\u03b2","p"]},"mo":"\u2062","mrow":{"msub":{"mi":["f","p"]},"mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msub":[{"mi":"X","mrow":{"mi":"N","mo":",","mn":"0"}},{"mi":"X","mrow":{"mi":"N","mo":",","mn":"1"}}],"mo":[",",","],"mrow":{"mi":"\u2026","mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.6em","height":"0.6ex"}}},"msub":{"mi":"X","mrow":{"mi":"N","mo":",","mrow":{"msub":{"mi":["N","f"]},"mo":"-","mn":"1"}}}}}}}}},"mi":"\u025b"}}},{"mrow":{"mo":["(",")"],"mn":"26"}}]}}}}},"In many cases of binary data classes, the functions fwould emulate logical expressions of the dependent variable. For a simple linear model, one could use P=N+1 parameters as follows:",{"@attributes":{"id":"p-0123","num":"0271"},"maths":{"@attributes":{"id":"MATH-US-00021","num":"00021"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"msub":{"mi":["Y","N"]},"mo":"=","mrow":{"msub":{"mi":"\u03b2","mn":"0"},"mo":["+","+"],"mrow":{"munderover":{"mo":"\u2211","mrow":[{"mi":"p","mo":"=","mn":"1"},{"mi":"P","mo":"-","mn":"1"}]},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"msub":[{"mi":["\u03b2","p"]},{"mi":"X","mrow":{"mi":["N","p"],"mo":","}}],"mo":"\u2062"}},"mi":"\u025b"}}},{"mrow":{"mo":["(",")"],"mn":"27"}}]}}}}},"Subsequent discussion is based on the most general formulation described in (27). Consider the probability of having a particular set of values (x. . . x) for the data classes X. . . X, which is denoted p. . . (x. . . x). Based on the genetic information describing the alleles of genes on the homologous chromosomes of the parents, it will be straightforward to one skilled in the art to compute the probability p. . . X(x. . . x) for each possible combination of gene alleles in the child (individual N). For example, consider that data class Xis 1 (TRUE) when individual N has allele a of gene g on one or both chromosomes and data class Xis 1 (TRUE) when individual N has allele c of gene g on one or more chromosomes. Assume the one parent has alleles a and c of gene g on their homologous chromosomes, and the other parent has alleles d and e of gene g on their homologous chromosomes. Assuming Mendellian independent assortment of genes, one can say\n\n(0,0)=0(1,0)=0.5(0,1)=0.5(1,1)=0 \u2003\u2003(28)\n","One cannot always assume independent assortment since, due to the mechanism of crossover in gamete formation, genes in close proximity on the same chromosome will tend to stay together in the offspring. The proximity of gene loci are measured in centi-Morgans, such that two genetic loci that show a 1% chance of recombination are defined as being 1 cM apart on the genetic map. The rules for computing p. . . (x. . . x) for a progeny, based on the known proximity of the gene loci, the alleles present in the parents, and the mechanisms of meiosis, are well understood and these probabilities can be computed by one skilled in the art of genetics.","Assume that using a regression technique similar to that described above and data records from many individuals, a set of parameters b. . . has been computed, which let us estimate a particular Y. . . given a particular data vector X. . . according to the matrix formulation \u0176. . . =X. . . b. . . . In this case, X. . . is the matrix of values of the independent variables [x. . . x]. Assume that all of the different combination of the independent variables are contained in a set of the combinations: S={[x. . . x]}. Based on the genes of the parents, Yfor the progeny may be estimated by:",{"@attributes":{"id":"p-0127","num":"0275"},"maths":{"@attributes":{"id":"MATH-US-00022","num":"00022"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"msub":{"mover":{"mi":"Y","mo":"^"},"mi":"N"},"mo":"=","mrow":{"munder":{"mo":"\u2211","mrow":{"mrow":{"mo":["[","]"],"mrow":{"msub":[{"mi":"x","mrow":{"mrow":{"mn":"0","mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.6em","height":"0.6ex"}}},"mi":"\u2026"},"mo":"\u2062","mstyle":{"mspace":{"@attributes":{"width":"0.6em","height":"0.6ex"}}}}},{"mi":"x","mrow":{"mi":"P","mo":"-","mn":"1"}}],"mo":"\u2062"}},"mo":"\u2208","msub":{"mi":["S","p"]}}},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"mrow":{"msub":{"mi":"p","mrow":{"msub":[{"mi":"X","mrow":{"mn":"0","mo":"\u2062","mstyle":{"mspace":{"@attributes":{"width":"0.6em","height":"0.6ex"}}}}},{"mi":"X","mrow":{"mi":"P","mo":"-","mn":"1"}}],"mo":["\u2062","\u2062","\u2062"],"mi":"\u2026","mstyle":{"mspace":{"@attributes":{"width":"1.1em","height":"1.1ex"}}}}},"mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msub":[{"mi":"x","mrow":{"mrow":{"mn":"0","mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.6em","height":"0.6ex"}}},"mi":"\u2026"},"mo":"\u2062","mstyle":{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}}}},{"mi":"x","mrow":{"mi":"P","mo":"-","mn":"1"}}],"mo":"\u2062"}}},"mo":["\u2062","\u2062"],"msub":[{"mi":"X","mrow":{"mi":"N","mo":",","mrow":{"msub":[{"mi":"x","mn":"0"},{"mi":"x","mrow":{"mi":"P","mo":"-","mn":"1"}}],"mo":["\u2062","\u2062","\u2062"],"mi":"\u2026","mstyle":{"mspace":{"@attributes":{"width":"1.1em","height":"1.1ex"}}}}}},{"mi":"b","mrow":{"msub":[{"mi":"x","mrow":{"mn":"0","mo":"\u2062","mstyle":{"mspace":{"@attributes":{"width":"0.6em","height":"0.6ex"}}}}},{"mi":"x","mrow":{"mi":"P","mo":"-","mn":"1"}}],"mo":["\u2062","\u2062","\u2062"],"mi":"\u2026","mstyle":{"mspace":{"@attributes":{"width":"0.6em","height":"0.6ex"}}}}}]}}}},{"mrow":{"mo":["(",")"],"mn":"29"}}]}}}}},"In addition, for the purpose of understanding the confidence of this estimate, one could compute the estimate of the confidence bounds on \u0176using the same technique described above:",{"@attributes":{"id":"p-0129","num":"0277"},"maths":{"@attributes":{"id":"MATH-US-00023","num":"00023"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"msub":{"mover":{"mi":"Y","mo":"^"},"mi":"N"},"mo":"\u00b1","mrow":{"mrow":[{"mi":"t","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mrow":[{"mn":"1","mo":"-","mfrac":{"mi":"\u03b1","mn":"2"}},{"mi":["N","P"],"mo":"-"}],"mo":","}}},{"mi":"s","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msub":[{"mover":{"mi":"Y","mo":"^"},"mi":"N"},{"mi":["Y","N"]}],"mo":"-"}}}],"mo":"\u2062"}}},{"mrow":{"mo":["(",")"],"mn":"30"}}]}}}}},"For each separate estimate Y. . . for a given data set, compute a separate mean-squared error, MSE. . . , using the techniques described above. Hence, considering the Equation (35) estimate the variances (\u0176\u2212Y):",{"@attributes":{"id":"p-0131","num":"0279"},"maths":{"@attributes":{"id":"MATH-US-00024","num":"00024"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"mrow":[{"msup":{"mi":"s","mn":"2"},"mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msub":[{"mover":{"mi":"Y","mo":"^"},"mi":"N"},{"mi":["Y","N"]}],"mo":"-"}}},{"munder":{"mo":"\u2211","mrow":{"mrow":{"mo":["[","]"],"mrow":{"msub":[{"mi":"x","mrow":{"mn":"0","mo":"\u2062","mi":"\u2026"}},{"mi":"x","mrow":{"mi":"P","mo":"-","mn":"1"}}],"mo":"\u2062"}},"mo":"\u2208","msub":{"mi":["S","p"]}}},"mo":"\u2062","mrow":{"msup":{"mrow":{"msub":{"mi":"p","mrow":{"msub":[{"mi":"x","mn":"0"},{"mi":"x","mrow":{"mi":"P","mo":"-","mn":"1"}}],"mo":["\u2062","\u2062","\u2062"],"mi":"\u2026","mstyle":{"mspace":{"@attributes":{"width":"1.1em","height":"1.1ex"}}}}},"mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msub":[{"mi":"x","mrow":{"mrow":{"mn":"0","mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.6em","height":"0.6ex"}}},"mi":"\u2026"},"mo":"\u2062","mstyle":{"mspace":{"@attributes":{"width":"0.6em","height":"0.6ex"}}}}},{"mi":"x","mrow":{"mi":"P","mo":"-","mn":"1"}}],"mo":"\u2062"}}},"mn":"2"},"mo":["\u2062","\u2062"],"msub":{"mi":"MSE","mrow":{"msub":[{"mi":"x","mn":"0"},{"mi":"x","mrow":{"mi":"P","mo":"-","mn":"1"}}],"mo":["\u2062","\u2062","\u2062","\u2062"],"mstyle":[{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},{"mspace":{"@attributes":{"width":"1.1em","height":"1.1ex"}}}],"mi":"\u2026"}},"mrow":{"mo":"\u2003","mrow":{"mo":["(",")"],"mrow":{"mrow":{"msup":{"mrow":[{"msubsup":{"mi":["X","T"],"mrow":{"mi":"N","mo":",","mrow":{"msub":[{"mi":"x","mn":"0"},{"mi":"x","mrow":{"mi":"P","mo":"-","mn":"1"}}],"mo":["\u2062","\u2062","\u2062","\u2062"],"mstyle":[{"mspace":{"@attributes":{"width":"0.6em","height":"0.6ex"}}},{"mspace":{"@attributes":{"width":"1.1em","height":"1.1ex"}}}],"mi":"\u2026"}}},"mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msub":{"mi":"X","mrow":{"msub":[{"mi":"x","mn":"0"},{"mi":"x","mrow":{"mi":"P","mo":"-","mn":"1"}}],"mo":["\u2062","\u2062","\u2062","\u2062"],"mstyle":[{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},{"mspace":{"@attributes":{"width":"1.1em","height":"1.1ex"}}}],"mi":"\u2026"}},"mo":"\u2062","mmultiscripts":{"mi":["X","T"],"mrow":{"msub":[{"mi":"x","mn":"0"},{"mi":"x","mrow":{"mi":"P","mo":"-","mn":"1"}}],"mo":["\u2062","\u2062","\u2062","\u2062"],"mstyle":[{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},{"mspace":{"@attributes":{"width":"1.1em","height":"1.1ex"}}}],"mi":"\u2026"},"none":[{},{}],"mprescripts":{}}}}},{"mo":"-","mn":"1"}]},"mo":"\u2062","msub":{"mi":"X","mrow":{"mi":"N","mo":",","mrow":{"msub":[{"mi":"x","mn":"0"},{"mi":"x","mrow":{"mi":"P","mo":"-","mn":"1"}}],"mo":["\u2062","\u2062","\u2062"],"mi":"\u2026","mstyle":{"mspace":{"@attributes":{"width":"1.1em","height":"1.1ex"}}}}}}},"mo":"+","mn":"1"}}}}}],"mo":"="}},{"mrow":{"mo":["(",")"],"mn":"31"}}]}}}}},"In this way, based on the information collected in the database , one can estimate the phenotypic features of progeny, as well as our confidence in those estimates. There are many different techniques for refining these estimates, using the ideas described above and other statistical techniques, without changing the fundamental concept disclosed.","One possible variant on this theme is to analyze the genetic makeup of a male and female gamete to decide whether they should be combined to form a progeny. Since the gamete would typically be destroyed using modern techniques for sequencing its genome, in one embodiment the genetic makeup of the gamete is determined by a process of elimination. This method employs the fact that the genetic information of the parent cell is copied once and distributed among the four gametes produced in meiosis. In this technique, the genetic makeup of the parent cell is determined, as well as of three of the four gametes that result from a meiotic division. This would allow one to determine the genetic makeup of the fourth gamete resulting from the meiotic division, without directly measuring the genetic sequence of that gamete.","Making Inferences with Sparse Data","Consider the scenario, very common in dealing with genetic data, where there are a large number, P, of parameters in comparison to the number of subjects, N. Described here is a method for analyzing sparse data sets which can determine a set of parameters b=[bb. . . b]that can meaningfully map between the independent variables representing some parameters p of patient n, X, and some dependent variable or outcome for that patient, Y. Some types of genetic data that the independent variables can represent are discussed first.","Xcould be a categorical variable representing whether or not a particular allele ais present in the genes of patient n i.e. X=1 if allele ais present on either of patient n's homologous chromosomes, and X=0 if allele ais not present in either of patient n's homologous chromosomes. Note that one could also have different categorical variables representing whether the allele is present on only one homologous chromosome, on both homologous chromosomes, or on neither. For example, one could have a variable Xwhich is 1 if allele ais present on either homologous chromosome, and another variable Xwhich is 1 only if allele ais present on both homologous chromosomes. With these types of data, the number of regression parameters P could be many tens of thousands, roughly corresponding to the number of possibly relevant alleles. Xcould also be a non-categorical continuous variable. For example, Xcould represent the concentration of a particular type of Ribonucleic Acid (RNA) in the tissue sample of patient n, where that RNA corresponds to the expression of one (or several) alleles, a, of some gene. In this case as well, P would be on the order of many tens of thousands, corresponding to the number of possibly relevant genetic alleles that could be expressed as RNA in the tissue of a patient. After reading this disclosure, it would be clear to one skilled in the art the range of other categorical and non-categorical variable sets that would give rise to sparse data in the context of genetic analysis.","A versatile formulation of the independent variables, X, which is particularly relevant for genetic analysis, is as logical expressions involving other categorical variables. Assume that there are P logical expressions, which may greatly exceed the number of original categorical variables that are combined to form these expressions. For example, consider that there are a set of alleles, numbering A, that can be detected in a genetic sample. Let the variable {tilde over (X)}be True if allele ais present in the sample, and False if allele ais not present. Ignore, for notational simplicity, the question of dominant vs recessive alleles; it will be clear how the method would extend to deal with variables defined for dominant or recessive genetic alleles. Another scenario that the method could address is time-dependent relationships between variables, when one event needs to precede another. To address this, one could take longitudinal data on a particular variables sampled over time, and represent the results in a set of time bins. This would generate multiple independent variables for regression\u2014the number of variables corresponding to the number of time bins\u2014for each longitudinal variable. The number of time bins can be selected using canonical techniques so that the independent variables are not too closely correlated. A continuous variable may be converted into a categorical variable which represents whether or not some threshold was exceeded by the continuous variable.","Consider different combinations of logical expressions that can be created with the set of Boolean variables {{tilde over (X)}, a=0 . . . A\u22121}. The purpose of the method disclosed is to be able to explore a large range of possible interactions with many variables, even in the context of sparse data. Define the maximum number of independent variables to be used in each term of the logical expression as V \u2208[1 . . . A]. Note that a logical term is used here to refer to a set of variables, or their complements, that are related to one another by the AND () operator. Assume that multiple terms are combined into a logical expression by the OR () operator. In order to model our dependent variable, Y, create an expression using all of the possible logical combinations of the independent variables. For example, assume V=A=2. In other words, one has two different alleles, and one is looking at all logical expressions combining one or both of the corresponding independent variables {{tilde over (X)}, {tilde over (X)}}. Map the Boolean variables into corresponding numerical random variables, X, using an Indicator Function, I. For example, X=I({tilde over (X)})=1 if {tilde over (X)}is TRUE and 0 otherwise. Then the logical expression with these variables may look like:",{"@attributes":{"id":"p-0138","num":"0286"},"maths":{"@attributes":{"id":"MATH-US-00025","num":"00025"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"msub":{"mi":["Y","n"]},"mo":"=","mrow":{"mrow":[{"msub":{"mi":"b","mn":"0"},"mo":["+","+","+","+","+","+"],"mrow":[{"msub":{"mi":"b","mn":"1"},"mo":"\u2062","mrow":{"mi":"I","mo":"\u2061","mrow":{"mo":["(",")"],"msub":{"mover":{"mi":"X","mo":"~"},"mrow":{"mn":"0","mo":",","mi":"n"}}}}},{"msub":{"mi":"b","mn":"2"},"mo":"\u2062","mrow":{"mi":"I","mo":"\u2061","mrow":{"mo":["(",")"],"msub":{"mover":{"mi":"X","mo":"~"},"mrow":{"mn":"1","mo":",","mi":"n"}}}}},{"msub":{"mi":"b","mn":"3"},"mo":"\u2062","mrow":{"mi":"I","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msub":[{"mover":{"mi":"X","mo":"~"},"mrow":{"mn":"0","mo":",","mi":"n"}},{"mover":{"mi":"X","mo":"~"},"mrow":{"mn":"1","mo":",","mi":"n"}}],"mo":"\u22c0"}}}},{"msub":{"mi":"b","mn":"4"},"mo":"\u2062","mrow":{"mi":"I","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msub":{"mover":{"mi":"X","mo":"~"},"mrow":{"mn":"0","mo":",","mi":"n"}},"mo":"\u22c0","msubsup":{"mover":{"mi":"X","mo":"~"},"mrow":{"mn":"1","mo":",","mi":"n"},"mi":"c"}}}}},{"msub":{"mi":"b","mn":"5"},"mo":"\u2062","mrow":{"mi":"I","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msubsup":{"mover":{"mi":"X","mo":"~"},"mrow":{"mn":"0","mo":",","mi":"n"},"mi":"c"},"mo":"\u22c0","msub":{"mover":{"mi":"X","mo":"~"},"mrow":{"mn":"1","mo":",","mi":"n"}}}}}},{"msub":{"mi":"b","mn":"6"},"mo":"\u2062","mrow":{"mi":"I","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msubsup":[{"mover":{"mi":"X","mo":"~"},"mrow":{"mn":"0","mo":",","mi":"n"},"mi":"c"},{"mover":{"mi":"X","mo":"~"},"mrow":{"mn":"1","mo":",","mi":"n"},"mi":"c"}],"mo":"\u22c0"}}}}]},{"msub":{"mi":"b","mn":"0"},"mo":["+","+","+","+","+","+"],"mrow":[{"msub":[{"mi":"b","mn":"1"},{"mi":"X","mrow":{"mn":"1","mo":",","mi":"n"}}],"mo":"\u2062"},{"msub":[{"mi":"b","mn":"2"},{"mi":"X","mrow":{"mn":"2","mo":",","mi":"n"}}],"mo":"\u2062"},{"msub":[{"mi":"b","mn":"3"},{"mi":"X","mrow":{"mn":"3","mo":",","mi":"n"}}],"mo":"\u2062"},{"msub":[{"mi":"b","mn":"4"},{"mi":"X","mrow":{"mn":"4","mo":",","mi":"n"}}],"mo":"\u2062"},{"msub":[{"mi":"b","mn":"5"},{"mi":"X","mrow":{"mn":"5","mo":",","mi":"n"}}],"mo":"\u2062"},{"msub":[{"mi":"b","mn":"6"},{"mi":"X","mrow":{"mn":"6","mo":",","mi":"n"}}],"mo":"\u2062"}]}],"mo":["\u2062","\u2062","="],"mstyle":[{"mtext":{}},{"mspace":{"@attributes":{"width":"1.4em","height":"1.4ex"}}}]}}},{"mrow":{"mo":["(",")"],"mn":"32"}}]}}}},"br":{},"sup":["c ","c ","c","c","c","c","c"],"sub":["a,n","a,n ","n","0","1","0,n","0,n ","n","0","1","1","0,n","n","0","1","0,n","1,n","n","0","1","1","0,n","1,n","1","0,n","1,n","1","0,n","1,n"]},{"@attributes":{"id":"p-0139","num":"0287"},"maths":{"@attributes":{"id":"MATH-US-00026","num":"00026"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mo":"\u2062","mtable":{"mtr":[{"mtd":{"mrow":{"msub":{"mi":["Y","n"]},"mo":"=","mrow":{"msub":{"mi":"b","mn":"0"},"mo":["+","+","+","\u2062","+","+"],"mrow":[{"munderover":{"mo":"\u2211","mrow":[{"mi":"i","mo":"=","mn":"0"},{"mi":"A","mo":"-","mn":"1"}]},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"msub":{"mi":"b","mrow":{"mi":"i","mo":"+","mn":"1"}},"mo":"\u2062","mrow":{"mi":"I","mo":"\u2061","mrow":{"mo":["(",")"],"msub":{"mover":{"mi":"X","mo":"~"},"mrow":{"mi":["i","n"],"mo":","}}}}}},{"munderover":{"mo":"\u2211","mrow":[{"mi":"i","mo":"=","mn":"0"},{"mi":"A","mo":"-","mn":"1"}]},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"munder":{"mo":"\u2211","mrow":{"mrow":{"munder":{"mi":"j","mrow":{"mi":["j","i"],"mo":"\u2260"}},"mo":"=","mrow":{"mrow":{"mn":"0","mo":["\u2062","\u2062","\u2062"],"mi":["\u2026","A"],"mstyle":{"mspace":{"@attributes":{"width":"0.6em","height":"0.6ex"}}}},"mo":"-","mn":"1"}},"mo":","}},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"msub":{"mi":"b","mrow":{"mi":"f","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mi":["i","j"],"mo":","}}}},"mo":["\u2062","\u2062"],"mi":"I","mrow":{"mo":["(",")"],"mrow":{"msub":[{"mover":{"mi":"X","mo":"~"},"mrow":{"mi":["i","n"],"mo":","}},{"mover":{"mi":"X","mo":"~"},"mrow":{"mi":["j","n"],"mo":","}}],"mo":"\u22c0"}}}}},{"msub":{"mi":"b","mrow":{"mrow":{"mi":"f","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mi":["i","j"],"mo":","}}},"mo":"+","mn":"1"}},"mo":"\u2062","mrow":{"mi":"I","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msub":{"mover":{"mi":"X","mo":"~"},"mrow":{"mi":["i","n"],"mo":","}},"mo":"\u22c0","msubsup":{"mover":{"mi":"X","mo":"~"},"mrow":{"mi":["j","n"],"mo":","},"mi":"c"}}}}},{"msub":{"mi":"b","mrow":{"mrow":{"mi":"f","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mi":["i","j"],"mo":","}}},"mo":"+","mn":"2"}},"mo":"\u2062","mrow":{"mi":"I","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msubsup":{"mover":{"mi":"X","mo":"~"},"mrow":{"mi":["i","n"],"mo":","},"mi":"c"},"mo":"\u22c0","msub":{"mover":{"mi":"X","mo":"~"},"mrow":{"mi":["j","n"],"mo":","}}}}}},{"msub":{"mi":"b","mrow":{"mrow":{"mi":"f","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mi":["i","j"],"mo":","}}},"mo":"+","mn":"3"}},"mo":"\u2062","mrow":{"mi":"I","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msubsup":[{"mover":{"mi":"X","mo":"~"},"mrow":{"mi":["i","n"],"mo":","},"mi":"c"},{"mover":{"mi":"X","mo":"~"},"mrow":{"mi":["j","n"],"mo":","},"mi":"c"}],"mo":"\u22c0"}}}}],"mstyle":{"mtext":{}}}}}},{"mtd":{"mrow":{"mo":"=","mrow":{"msub":{"mi":"b","mn":"0"},"mo":["+","+"],"mrow":[{"munderover":{"mo":"\u2211","mrow":{"mi":"p","mo":"=","mn":"1"},"mi":"A"},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"msub":[{"mi":["b","p"]},{"mi":["X","p"]}],"mo":"\u2062"}},{"munderover":{"mo":"\u2211","mrow":[{"mi":"p","mo":"=","mrow":{"mi":"A","mo":"+","mn":"1"}},{"mi":"P","mo":"-","mn":"1"}]},"mo":"\u2062","mrow":{"msub":[{"mi":["b","p"]},{"mi":["X","p"]}],"mo":"\u2062"}}]}}}}]}}},{"mrow":{"mo":["(",")"],"mn":"33"}}]}}}}},"In the second line, one has used some indexing function f(i,j) to denote that the indexes of regression parameters will be some function of i and j without using overly complex notation. Furthermore, in order to simplify notation and without loss of generality, all of the terms in the second line are recast into the set of terms denoted by the last summation of the third line. In general, for any A and V, the number of resulting parameters necessary to combine the logical terms in the expression would be:",{"@attributes":{"id":"p-0141","num":"0289"},"maths":{"@attributes":{"id":"MATH-US-00027","num":"00027"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"mi":"P","mo":"=","mrow":{"mn":"1","mo":["+","+"],"mi":"A","mrow":{"munderover":{"mo":"\u2211","mrow":{"mi":"j","mo":"=","mn":"2"},"mi":"V"},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"mrow":{"mo":["(",")"],"mtable":{"mtr":[{"mtd":{"mi":"A"}},{"mtd":{"mi":"j"}}]}},"mo":"\u2062","msup":{"mn":"2","mi":"j"}}}}}},{"mrow":{"mo":["(",")"],"mn":"34"}}]}}}}},"The number of logical terms, and the associated parameters, grow rapidly with an increase in A and V. Techniques are understood in the art for reducing the number of terms in the regression by checking that the independent variables are not too closely correlated. For example, one may start creating independent variables from the logical expressions involving combinations of fewer categorical variables. Each time a new logical expression is added to create a new independent variable, one checks that the correlation coefficient between the new variable and any of the existing variables does not exceed some threshold. This helps to assure a well-conditioned and convergent solution. Many techniques exist in the art to eliminate redundant variables and higher order interaction variables, without changing the essential concept discussed here.","Despite techniques for reducing the variable set, P may still grow to become very large as A and V increase. It is now described how to manage this large number of terms, even though the number of parameters P may considerably exceed the number of outcomes, N. Let the set of outcomes be {Y, n=0 . . . N\u22121}. Note that Ymay be related to a categorical variable; for example Y=1 if the patient has a particular disease and Y=0 if not. Alternatively, Ymay be a non-categorical variable such as the probability of the patient developing a particular disease, or the probability of a patient developing resistance to a particular medication within some timeframe. First, the concept is illustrated in general. Different formulations of the dependent variable are discussed below, in particular those related to logistic regression where the outcomes are categorical. In order to use matrix notation, stack the dependent variables for N patients in vector y=[YY. . . . Y]. Represent in matrix notation a linear mapping of the independent variables to the dependent variables according to:\n\n\u2003\u2003(35)\n","Where",{"@attributes":{"id":"p-0145","num":"0293"},"maths":{"@attributes":{"id":"MATH-US-00028","num":"00028"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"mrow":[{"mi":"y","mo":"=","mrow":{"mo":["[","]"],"mtable":{"mtr":[{"mtd":{"msub":{"mi":"Y","mn":"0"}}},{"mtd":{"mi":"\u22ee"}},{"mtd":{"msub":{"mi":"Y","mrow":{"mi":"N","mo":"-","mn":"1"}}}}]}}},{"mi":"X","mo":"=","mrow":{"mo":["[","]"],"mtable":{"mtr":[{"mtd":[{"msub":{"mi":"X","mrow":{"mn":["0","0"],"mo":","}}},{"mi":"\u22ef"},{"msub":{"mi":"X","mrow":{"mrow":{"mi":"P","mo":"-","mn":"1"},"mo":",","mn":"0"}}}]},{"mtd":[{"mi":"\u22ee"},{"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}}},{"mi":"\u22ee"}]},{"mtd":[{"msub":{"mi":"X","mrow":{"mn":"0","mo":",","mrow":{"mi":"N","mo":"-","mn":"1"}}}},{"mi":"\u22ef"},{"msub":{"mi":"X","mrow":{"mrow":[{"mi":"P","mo":"-","mn":"1"},{"mi":"N","mo":"-","mn":"1"}],"mo":","}}}]}]}}},{"mi":"b","mo":"=","mrow":{"mo":["[","]"],"mtable":{"mtr":[{"mtd":{"msub":{"mi":"b","mn":"0"}}},{"mtd":{"mi":"\u22ee"}},{"mtd":{"msub":{"mi":"b","mrow":{"mi":"P","mo":"-","mn":"1"}}}}]}}},{"mi":"\u025b","mo":"=","mrow":{"mo":["[","\u2062","]"],"mtable":{"mtr":[{"mtd":{"msub":{"mi":"\u025b","mn":"0"}}},{"mtd":{"mi":"\u22ee"}},{"mtd":{"msub":{"mi":"\u025b","mrow":{"mi":"N","mo":"-","mn":"1"}}}}]},"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}}}}],"mo":[",","\u2062",",","\u2062","\u2062",",","\u2062"],"mstyle":[{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}},{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}},{"mtext":{}},{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}}]}},{"mrow":{"mo":["(",")"],"mn":"36"}}]}}}},"br":{},"sub":"0,n","sup":["T","\u22121","T ","T","T"]},"Consequently, this method enables us to create complex logical or arithmetic combinations of random variables when the coupling between these random variables is necessary to model the outcome.","In order to restrict the different values of {circumflex over (b)} in the sparse scenario, a shrinkage function s is created which is weighted with a complexity parameters \u03bb. b Is estimated by solving the optimization:\n\n=arg min() \u2003\u2003(37)\n","There are many different forms that s(b) can take, including:",{"@attributes":{"id":"p-0149","num":"0297"},"maths":{"@attributes":{"id":"MATH-US-00029","num":"00029"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mtable":{"mtr":[{"mtd":{"mrow":{"mrow":[{"mi":"s","mo":"\u2061","mrow":{"mo":["(",")"],"mi":"b"}},{"munder":{"mo":"\u2211","mi":"j"},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"msub":{"mrow":{"mo":["\uf603","\uf604"],"mi":"b"},"mi":"j"}}],"mo":"="}}},{"mtd":{"mrow":{"mrow":[{"mi":"s","mo":"\u2061","mrow":{"mo":["(",")"],"mi":"b"}},{"munder":{"mo":"\u2211","mi":"j"},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"msubsup":{"mi":["b","j"],"mn":"2"}}],"mo":"="}}},{"mtd":{"mrow":{"mrow":[{"mi":"s","mo":"\u2061","mrow":{"mo":["(",")"],"mi":"b"}},{"munder":{"mo":"\u2211","mi":"j"},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"mi":"ln","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msub":{"mrow":{"mo":["\uf603","\uf604"],"mi":"b"},"mi":"j"},"mo":"+","mi":"\u03b4"}}}}],"mo":"="}}}]}},{"mrow":{"mo":["(",")"],"mn":"38"}}]}}}}},"The last formulation is particularly well suited to the case where parameters correspond to a large set of logical expressions, as described above. The theoretical motivation behind this formulation is based on coding theory. In essence, one is trying to find a parameter set {circumflex over (b)} that can explain the data while remaining as simple or sparse as possible. Stated differently, one would like to apply \u201cOckham's Razor\u201d to estimating b. One mathematical approach to describing \u201csimplicity\u201d is to consider the length of a code necessary to capture the information in {circumflex over (b)}. It can be shown that real numbers can be coded to some precision \u03b4 using a code of length log (\u2514|r|\/\u03b4+1\u2518)+\u25ef(log log(|r|\/\u03b4)) bits. One may assume that the second term is negligible and focus on the first term. This can be used as a shrinkage function according to:",{"@attributes":{"id":"p-0151","num":"0299"},"maths":{"@attributes":{"id":"MATH-US-00030","num":"00030"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"mrow":[{"mi":"s","mo":"\u2061","mrow":{"mo":["(",")"],"mi":"b"}},{"mrow":[{"munder":{"mo":"\u2211","mi":"j"},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"mi":"log","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mrow":{"mrow":{"mo":["\uf603","\uf604"],"msub":{"mi":["b","j"]}},"mo":["\u2062","\u2062"],"mstyle":{"mtext":"\/"},"mi":"\u03b4"},"mo":"+","mn":"1"}}}},{"mrow":[{"munder":{"mo":"\u2211","mi":"j"},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"mi":"log","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mrow":{"mo":["\uf603","\uf604"],"msub":{"mi":["b","j"]}},"mo":"+","mi":"\u03b4"}}}},{"mi":["P","log","\u03b4"],"mo":["\u2062","\u2062","\u2062","\u2062"],"mstyle":[{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}}]}],"mo":"-"}],"mo":"="}],"mo":"="}},{"mrow":{"mo":["(",")"],"mn":"39"}}]}}}}},"Since the last term is independent of b, it can be dropped from the shrinkage function, leaving the last function described in (39) above. Hence, one can solve for sparse paramaters according to:",{"@attributes":{"id":"p-0153","num":"0301"},"maths":{"@attributes":{"id":"MATH-US-00031","num":"00031"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"msup":{"mrow":{"mrow":{"mrow":{"mrow":{"mrow":{"mover":{"mi":"b","mo":"^"},"mo":"=","msub":{"mstyle":{"mtext":"argmin"},"mi":"b"}},"mo":"\uf606"},"mo":"\u2062","mi":"y"},"mo":"-","mi":"Xb"},"mo":"\uf606"},"mn":"2"},"mo":"+","mrow":{"mi":"\u03bb","mo":"\u2062","mrow":{"munder":{"mo":"\u2211","mi":"j"},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"mi":"log","mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"mo":["(",")"],"mrow":{"mrow":{"mo":["\uf603","\uf604"],"msub":{"mi":["b","j"]}},"mo":"+","mi":"\u03b4"}}}}}}},{"mrow":{"mo":["(",")"],"mn":"40"}}]}}}}},"The parameter a can be viewed as the minimum penalty for each coefficient band is often set to 1 in practice although the exact choice will depend on the data, and can be made by one skilled in the art after reading this disclosure. The parameter \u03bb represents the conversion factor between the penalty function s(b) and the expression being minimized, in this case the squared residual error \u2225y\u2212Xb\u2225. In particular \u03bb represents how many units of residual error one is willing to sacrifice for a reduction of 1 unit of s(b). This optimization equation can be solved using techniques that are known in the art. The general technique involves solving the optimization for pairs of (\u03bb,\u03b4) over some reasonable range, and then using cross-validation with test data and an estimate of the prediction error to select the optimal (\u03bb,\u03b4) pair. For each (\u03bb,\u03b4) pair, the optimization can be solved by techniques that are understood in the art. One such technique involves updating the estimate {circumflex over (b)} at each step using a first order approximation of a log function, linearized around the current estimate at update epoch k, {circumflex over (b)}, and then dropping all terms from the minimization expression that are independent of b. The resultant iterative update simplifies to",{"@attributes":{"id":"p-0155","num":"0303"},"maths":{"@attributes":{"id":"MATH-US-00032","num":"00032"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"msup":{"mrow":{"mrow":{"mrow":{"mrow":{"mrow":{"msup":{"mover":{"mi":"b","mo":"^"},"mrow":{"mo":["(",")"],"mrow":{"mi":"k","mo":"+","mn":"1"}}},"mo":"=","msub":{"mi":["argmin","b"]}},"mo":"\uf606"},"mo":"\u2062","mi":"y"},"mo":"-","mi":"Xb"},"mo":"\uf606"},"mn":"2"},"mo":"+","mrow":{"mi":"\u03bb","mo":"\u2062","mrow":{"munder":{"mo":"\u2211","mi":"j"},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mfrac":{"mrow":[{"mo":["\uf603","\uf604"],"msub":{"mi":["b","j"]}},{"mrow":{"mo":["\uf603","\uf604"],"msubsup":{"mi":["b","j"],"mrow":{"mo":["(",")"],"mi":"k"}}},"mo":"+","mi":"\u03b4"}]}}}}},{"mrow":{"mo":["(",")"],"mn":"41"}}]}}}}},"Since this expression is convex in b, it can be easily solved by a variety of methods to generate the next estimate of b, {circumflex over (b)}. This process is continued until some convergence criteria is satisfied. A similar method can be applied to logistic regression such as in the case that Yis a categorical variable. This will be the case in predictive problems, such as estimating the probability of developing a particular disease or the probability of a viral or bacterial strain developing resistance to a particular medication. In general, logistic regression is useful when the dependent variable is categorical. If a linear probability model is used, y=Xb+\u03b5 (35, one can obtain useful results in terms of sign and significance levels of the dependent variables, but one does not obtain good estimates of the probabilities of particular outcomes. This is particularly important for the task of making probabilistic projections based on genetic data. In this regard, the linear probability model suffers from the following shortcomings when used in regression:\n\n","Logistic regression treats the random variable describing the event, Y, as a dummy variable and creates a different dependent variable for the regression, based on the probability that an event will occur. Denote this random variable, representing the probability that Y=1, as P. With the variable Pcreate the \u201clogit\u201d regression model (ignore the error term here for the sake of simplicity):\n\nlog(\/(1))=\u2003\u2003(42)\n","The expression log(P\/(1\u2212P)), or the \u201clog odds ratio\u201d, will constrain the prediction of Pbased on a set of parameters {circumflex over (b)} to lie between 0 and 1:",{"@attributes":{"id":"p-0159","num":"0310"},"maths":{"@attributes":{"id":"MATH-US-00033","num":"00033"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"msub":{"mover":{"mi":"P","mo":"^"},"mrow":{"msub":{"mi":["Y","n"]},"mo":"=","mn":"1"}},"mo":"=","mfrac":{"mn":"1","mrow":{"mn":"1","mo":"+","mrow":{"mi":"exp","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mrow":{"mo":"-","msub":{"mi":["X","n"]}},"mo":"\u2062","mover":{"mi":"b","mo":"^"}}}}}}}},{"mrow":{"mo":["(",")"],"mn":"43"}}]}}}}},"Rather than use the classic least-squares technique, solve for the parameters {circumflex over (b)} using a maximum likelihood method. In other words, looking over all of the events 1 . . . N, maximize the likelihood (the aposteriori probability) of all the independent observations. For each observation, the likelihood of that observation is found according to:",{"@attributes":{"id":"p-0161","num":"0312"},"maths":{"@attributes":{"id":"MATH-US-00034","num":"00034"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"mtable":{"mtr":[{"mtd":{"mrow":{"msub":{"mover":{"mi":"P","mo":"^"},"mrow":{"msub":{"mi":["Y","n"]},"mo":"=","mn":"1"}},"mo":"=","mfrac":{"mn":"1","mrow":{"mn":"1","mo":"+","mrow":{"mi":"exp","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mrow":{"mo":"-","msub":{"mi":["X","n"]}},"mo":"\u2062","mover":{"mi":"b","mo":"^"}}}}}}}}},{"mtd":{"mrow":{"msub":{"mover":{"mi":"P","mo":"^"},"mrow":{"msub":{"mi":["Y","n"]},"mo":"=","mn":"0"}},"mo":"=","mrow":{"mrow":{"mn":"1","mo":"-","msub":{"mover":{"mi":"P","mo":"^"},"mrow":{"msub":{"mi":["Y","n"]},"mo":"=","mn":"1"}}},"mo":"=","mfrac":{"mn":"1","mrow":{"mn":"1","mo":"+","mrow":{"mi":"exp","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msub":{"mi":["X","n"]},"mo":"\u2062","mover":{"mi":"b","mo":"^"}}}}}}}}}}]},"mo":"\u2062","mstyle":{"mtext":{}}}},{"mrow":{"mo":["(",")"],"mn":"44"}}]}}}}},"For all N observations with outcomes {Y=y,n=0 . . . N\u22121} the probability of the estimated collective outcome is computed as",{"@attributes":{"id":"p-0163","num":"0314"},"maths":{"@attributes":{"id":"MATH-US-00035","num":"00035"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"msub":{"mover":{"mi":"P","mo":"^"},"mi":"collective"},"mo":"=","mrow":{"munderover":{"mo":"\u220f","mrow":[{"mi":"n","mo":"=","mn":"0"},{"mi":"N","mo":"-","mn":"1"}]},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"msub":{"mover":{"mi":"P","mo":"^"},"mrow":{"msub":[{"mi":["Y","n"]},{"mi":["y","n"]}],"mo":"="}}}}},{"mrow":{"mo":["(",")"],"mn":"45"}}]}}}}},"Let us define the set of all indices, n, for which Y=1 as Sand the set of indices for which Y=0 as S. In order to maximize the collective probability, one can equally minimize the negative of the log of the collective probability. In other words, one can find the maximum likelihood estimate of the parameters by finding",{"@attributes":{"id":"p-0165","num":"0316"},"maths":{"@attributes":{"id":"MATH-US-00036","num":"00036"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"mrow":{"mover":{"mi":"b","mo":"^"},"mo":"=","mrow":{"mrow":[{"msub":{"mi":["argmin","b"]},"mo":"-","mrow":{"mi":"log","mo":"\u2061","mrow":{"mo":["(",")"],"msub":{"mover":{"mi":"P","mo":"^"},"mi":"collective"}}}},{"mrow":[{"msub":{"mi":["argmin","b"]},"mo":"\u2062","mrow":{"munder":{"mo":"\u2211","mrow":{"mi":"n","mo":"\u2208","msub":{"mi":"S","mn":"1"}}},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"mi":"log","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mn":"1","mo":"+","mrow":{"mi":"exp","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mrow":{"mo":"-","msub":{"mi":["X","n"]}},"mo":"\u2062","mi":"b"}}}}}}}},{"munder":{"mo":"\u2211","mrow":{"mi":"n","mo":"\u2208","msub":{"mi":"S","mn":"0"}}},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"mi":"log","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mn":"1","mo":"+","mrow":{"mi":"exp","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msub":{"mi":["X","n"]},"mo":"\u2062","mi":"b"}}}}}}}],"mo":"+"}],"mo":"="}},"mo":"\u2062","mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}}}},{"mrow":{"mo":["(",")"],"mn":"46"}}]}}}}},"Now, it can be shown that functions of the form log(1+exp(Xb)) are convex in the parameter set b and that the sum of convex functions is convex. Hence, one can apply the same approach as described above to the logistic optimization. In other words, one can find {circumflex over (b)} that both satisfies the Maximum Likelihood criterion and is sparse, by solving the convex optimization problem:",{"@attributes":{"id":"p-0167","num":"0318"},"maths":{"@attributes":{"id":"MATH-US-00037","num":"00037"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"mover":{"mi":"b","mo":"^"},"mo":"=","mrow":{"mrow":[{"msub":{"mi":["argmin","b"]},"mo":"\u2062","mrow":{"munder":{"mo":"\u2211","mrow":{"mi":"n","mo":"\u2208","msub":{"mi":"S","mn":"1"}}},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"mi":"log","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mn":"1","mo":"+","mrow":{"mi":"exp","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mrow":{"mo":"-","msub":{"mi":["X","n"]}},"mo":"\u2062","mi":"b"}}}}}}}},{"munder":{"mo":"\u2211","mrow":{"mi":"n","mo":"\u2208","msub":{"mi":"S","mn":"0"}}},"mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"mrow":{"mi":"log","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mn":"1","mo":"+","mrow":{"mi":"exp","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"msub":{"mi":["X","n"]},"mo":"\u2062","mi":"b"}}}}}}},{"mi":"\u03bb","mo":"\u2062","mrow":{"munder":{"mo":"\u2211","mi":"j"},"mo":"\u2062","mrow":{"mi":"log","mo":"\u2061","mrow":{"mo":["(",")"],"mrow":{"mrow":{"mo":["\uf603","\uf604"],"msub":{"mi":["b","j"]}},"mo":"+","mi":"\u03b4"}}}}}],"mo":["+","+"]}}},{"mrow":{"mo":["(",")"],"mn":"47"}}]}}}}},"Note that maximum likelihood method and least-squares regression are equivalent in the case of normally distributed residual error. In particular, form a log-odds ratios as explained above, and assume that\n\nlog(\/(1))=\u2003\u2003(48)\n\nwhere \u03b5 is distributed normally. Then the exact solution to b using least squares is the same as with the maximum likelihood method. The practical problem is that, even if one might assume normally distributed errors, one might not have enough data to form empirical odds ratios due to sparsity of data. In particular, for any particular combination of the variables X the is empirical odds ration is defined as\n",{"@attributes":{"id":"p-0169","num":"0320"},"maths":{"@attributes":{"id":"MATH-US-00038","num":"00038"},"math":{"@attributes":{"overflow":"scroll"},"mtable":{"mtr":{"mtd":[{"mrow":{"mrow":[{"mi":"LO","mo":"\u2061","mrow":{"mo":["(",")"],"mi":"X"}},{"mi":"log","mo":"\u2061","mrow":{"mo":["(",")"],"mfrac":{"mrow":[{"mrow":[{"mi":["number","of"],"mo":["\u2062","\u2062","\u2062","\u2062"],"mstyle":[{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}},{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}}],"msub":{"mi":["Y","n"]}},{"mn":"1","mo":"\u2062","mrow":{"mo":"\uf603","mi":"X"}}],"mo":"="},{"mrow":{"mrow":{"mrow":{"mi":["number","of"],"mo":["\u2062","\u2062","\u2062","\u2062"],"mstyle":[{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}},{"mspace":{"@attributes":{"width":"0.8em","height":"0.8ex"}}}],"msub":{"mi":["Y","n"]}},"mo":"=","mn":"0"},"mo":"\uf604"},"mo":"\u2062","mi":"X"}]}}}],"mo":"="}},{"mrow":{"mo":["(",")"],"mn":"49"}}]}}}}},"Due to the small number of observations LO(X) is likely to be small or very large and not suitable for the regression. The maximum likelihood solution, given the above, bypasses that problem by deriving the solution from joint likelihood of all the observations at once. The maximum likelihood formulation can be solved by existing tools that are known in the art to rapidly solve convex optimization problems with multiple of variables. This methodology will be relevant to addressing the type of genetic analysis questions that the DAME  will encounter. In particular, this method is important for problems of prediction, based on a limited number of patients, and a large quantity of genetic variables with unknown interrelations.","One consideration in assuring good solution to the above problem is choosing the appropriate conversion factor \u03bb Assume that the cost function of the least-squares formulation is denoted by R(b), and the cost function of the maximum likelihood formulation is denoted by M(b). In both cases, there is an addititive penalty function s(b). Since the units of the two cost functions R(b) and M(b) are different, the scale of the shrinkage function s(b) must also be appropriately changed. In particular the conversion factor \u03bb must be tuned to insure optimality of the solution as well as its coefficient sparsity. \u03bb May be tuned based on cross-validation with test data.","Other Approaches to Making Inference based on Genetic Data","Although the discussion has focused on regression analysis methods, many other techniques may also be used in the DAME  to either make predictions for, or to validate, subject data based on the data of a related group of subjects. One embodiment is described here in which a decision tree is used by the DAME  to predict the response of mutated HIV viruses to particular ART drugs. The data inhaled in the database  includes in-vitro phenotypic tests of the HIV-1 viruses from many hundred subjects who have had their viral Reverse Transcriptase (RT) encoding segments sequenced. The independent variables in the statistical model  consist of the list of mutations present in the viral RT genetic sequence. The dependent variable in the statistical model  consists of phenotypic tests of susceptibility to the Reverse Transcriptase Inhibitor drug AZT. The phenotypic tests are measured in terms of the log of the ratio of the concentration of the drug required to inhibit replication of the mutated HIV-1 by 50%, compared with the concentration of drug required to inhibit replication of the wild-type HIV-1 clone by 50%.","The statistical model makes use of a binary decision tree where each non-terminal node is split based on the values of one of the independent variables\u2014the presence or absence of a particular mutation. At each node, that independent variable is selected that will subdivide the data into two child nodes such that the dependent variable is as homogenous as possible in each of the child nodes. Each terminal node of the tree is assigned a value that most closely matches the values of the independent variables for all subjects that fall in that subgroup , or terminal node. In one embodiment, the match is in the least square error sense. After the decision tree has been generated, the tree can be pruned by eliminating those node splits that have least effect on the prediction error. In a preferred embodiment, the pruning level is determined based on cross validation with test data.",{"@attributes":{"id":"p-0174","num":"0325"},"figref":["FIG. 10","FIG. 11"],"b":"128"},"Enabling a Clinician to Subject multiple Different Statistical Models  and Computed Relationships  to a Particular Patient's Data ",{"@attributes":{"id":"p-0175","num":"0326"},"figref":"FIG. 12","b":["102","401","106","108","128","123","128","110","117","128","122","124","122"]},"In one embodiment, the clinician  will access the system on a device with a GUI (Graphical user Interface) served by a GUI server  over the Internet , and will login using a secure user name and password possibly augmented with biometric authentication. The system will prompt the clinician  to enter a patients name or number and the relevant patient record  will be retrieved. In one embodiment, the patient's information and the results of a query will be presented to the clinician  in  sections of the screen detailed below.","Section 1  at the top of the screen, displays:\n\n","In this embodiment, the additional screens accessible from the phenotype information button  display information such as CD4+ cell count, blood pressure, heart rate etc. and are all date stamped, associated with a validator , and in the format of the standardized data model . The genotype information button will display such information as known Single Nucleotide Polymorphisms (SNPs) in the genome, and other relevant genetic information associated with key loci in the HIV\/AIDS or human genome. The search criteria for the query ,  allows the clinician to perform a \u201cwhat if scenario\u201d search on existing inhaled statistical models  and computed relationships . The typical aim, in a preferred embodiment, is to determine what the outcome for the patient would be if subjected to different methods of treatment according to the collection of inhaled models . This enables the clinician to see what treatments have been used and analyzed by experts (for example in clinical trials) and determine if they would be suitable for the current patient given the patient's genotypic and phenotype information . The search criteria are in a drop down box . In the case of selecting models  related to ART for a patient with AIDS, the criteria may include for example: statistical models from the 10 most recent ART trials, statistical models based on the 5 largest ART trials, models from the 3 most recent NIH trials, all available predictive models applied to any patient with HIVAIDS and TB, all available predictive models applied to patients with only HIV\/AIDS etc. Once the search criteria have been entered the clinician clicks the run button . This sends the data set of genotype, phenotype information  to the DAME . The DAME  then processes this query using the methods discussed above, and the results are displayed in section 2 and 3.","Section 2  will display the summary results of the query. In one embodiment, it shows a summary of each of the relevant inhaled statistical models  (for example, from clinical trials) based on the search criteria supplied. The following information is displayed:\n\n","Section 3  will, in a preferred embodiment, display detailed information about one of the statistical models (or trials) highlighted in section 2. There is a drill down functionality from section 2 to section 3. The clinician selects the trial about which to display additional information. The following information is displayed about the statistical model, or trial;\n\n","The predictive outcome is based on the analysis performed by DAME  using the relevant statistical model , computed relationships , and particular patients's data in standardized format . The predicted results  would show the probable outcome if the patient had been in the particular trial or following the particular treatment regimen around which the statistical model was based . The clinician will then be able to use these predictive outcomes  to administer treatment to the patient or do further investigation. It enables the clinician to make an informative decision about treatment based on clinical trial data. In a preferred embodiment, there is also a predictive outcome on an aggregated level in section 2 . This is computed using the techniques discussed above to select the best statistical model, or select the most predictive set of independent variables, and train the model with all the available aggregated data. In one embodiment, all electronically published data models use the same basic format so that the associated data can be easily combined for the purpose of making predicted outcome on an aggregated level. For example, a generic statistical model format for representing the hypothesis of a clinical trial is as 2-by-2 contingency table. The MATLAB code below illustrates a generic template for a training and a mapping function based on a 2-by-2 contingency table, which would allow ease of inhalation and application of the statistical model in the disclosed system. Code is omitted that would be obvious to one skilled in the art. For this illustration, it is assumed that the user of the template is proficient in MATLAB and Structured Query Language (SQL.)\n\n","The invention can be implemented in digital electronic circuitry, or in computer hardware, firmware, software, or in combinations thereof. Apparatus of the invention can be implemented in a computer program product tangibly embodied in a machine-readable storage device for execution by a programmable processor; and method steps of the invention can be performed by a programmable processor executing a program of instructions to perform functions of the invention by operating on input data and generating output. The invention can be implemented advantageously in one or more computer programs that are executable on a programmable system including at least one programmable processor coupled to receive data and instructions from, and to transmit data and instructions to, a data storage system, at least one input device, and at least one output device. Each computer program can be implemented in a high-level procedural or object-oriented programming language, or in assembly or machine language if desired; and in any case, the language can be a compiled or interpreted language. Suitable processors include, by way of example, both general and special purpose microprocessors. Generally, a processor will receive instructions and data from a read-only memory and\/or a random access memory. Generally, a computer will include one or more mass storage devices for storing data files; such devices include magnetic disks, such as internal hard disks and removable disks; magneto-optical disks; and optical disks. Storage devices suitable for tangibly embodying computer program instructions and data include all forms of non-volatile memory, including by way of example semiconductor memory devices, such as EPROM, EEPROM, and flash memory devices; magnetic disks such as internal hard disks and removable disks; magneto-optical disks; and CD-ROM disks. Any of the foregoing can be supplemented by, or incorporated in, ASICs (application-specific integrated circuits)."],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0016","num":"0015"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0017","num":"0016"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0018","num":"0017"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0019","num":"0018"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0020","num":"0019"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0021","num":"0020"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0022","num":"0021"},"figref":"FIG. 7"},{"@attributes":{"id":"p-0023","num":"0022"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0024","num":"0023"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0025","num":"0024"},"figref":"FIG. 10"},{"@attributes":{"id":"p-0026","num":"0025"},"figref":["FIG. 11","FIG. 10"]},{"@attributes":{"id":"p-0027","num":"0026"},"figref":"FIG. 12"}]},"DETDESC":[{},{}]}
