---
title: Methods and systems for technology analysis and mapping
abstract: Systems and methods for cladistics-based content searching, analysis, and/or diagrammatic representation of results in graphical user interface format for viewing by at least one user on a computer-type device or network, in particular for technology and patent-related content stored in at least one database.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08694504&OS=08694504&RS=08694504
owner: Spore, Inc.
number: 08694504
owner_city: Durham
owner_country: US
publication_date: 20040304
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATIONS","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENTS","EXAMPLE"],"p":["This non-provisional utility patent application claims the benefit of one or more prior filed U.S. provisional applications, a reference to each such prior application is identified as follows: Ser. No. 60\/452,029 filed Mar. 5, 2003; Ser. No. 60\/510,734 filed Oct. 11, 2003; and Ser. No. 60\/527,788 filed Dec. 8, 2003.","(1) Field of the Invention","The present invention relates generally to automated work processes and, more particularly, to a automated searching of databases for content, including but not limited to technology, inventions, patents, and patent-related information and\/or documents.","(2) Description of the Prior Art","Database searching can be divided into two major types of searching\u2014non-patent legal database searching and patent legal database searching.","Non-patent legal database searching, for whom the computerized search engines where first designed, is directed toward the task of extracting few, precise records. These queries and searches engines are designed to bring out the fewest documents possible to fulfill the needs of the searcher. This type of prior art searching requires that the user submit keywords to a search engine to get a single set of results. The user can use Boolean operators, such as AND, OR, ANDNOT (also NOT), etc. to more precisely define the search. The user can also weight keywords such that the results are ranked. However, neither increasing the precision of the search nor ranking the results ensures the thoroughness of the search and, in fact, both methods are designed to reduce the number of returned records.","In contrast, prior art searching for patent legal needs is designed to be thorough to ensure that the invention is novel and non-obvious. The traditional method is to search through all the records in the classes to which an invention may belong. This method is thorough, but is laborious and time-consuming. Additionally, other sources of prior art exist that are not classified and patent databases per se are becoming to large to be searched by this method. Persons wishing to search these databases in a timely manner frequently construct one or more queries to thoroughly cover the invention field. However, these search methods are idiosyncratic and not generally executed according to a predetermined method. The thoroughness of the search must therefore be closely evaluated each time.","Thus, a need exists for database content searching, particular for technology, patent, and\/or prior art searching methods and systems that are precise and thorough, easily evaluated and supervised, and which minimizes the number of records that an examiner must review to perform a precise and thorough examination. In addition, a need exists for a method of displaying the large amounts of data in a single view while allowing examination of individual records, such that the user or viewer can assess the technology field density and inspect individual records from the same view.","The present invention is directed to systems and methods for cladistics-based content searching, analysis, and\/or diagrammatic representation of results in graphical user interface format for viewing by at least one user on a computer-type device or network, in particular for technology and patent-related content stored in at least one database.","Accordingly, one aspect of the present invention is to provide a method of analyzing inventions, including the steps of:\n\n","Another aspect of the present invention is to provide a method of graphically displaying relationships between inventions, including the steps of:\n\n","Still another aspect of the present invention is to provide a method of querying a patent database for prior art relating to an invention, the steps of:\n\n","Yet another aspect and embodiments of the present invention include systems for analyzing technology including at least one database having technology information electronically stored thereon; at least one computer running software for querying the database; the at least one computer connected to the database, either directly or via a network; wherein the software permits the querying to be done using a cladistics-based model, wherein at least one user is capable of searching the information in the database and reviewing search results presented in a graphical representation on a user interface of the at least one computer, in particular where the graphical representation is a tree diagram.","These and other aspects of the present invention will become apparent to those skilled in the art after a reading of the following description of the preferred embodiment when considered with the drawings.","In the following description, like reference characters designate like or corresponding parts throughout the several views. Also in the following description, it is to be understood that such terms as \u201cforward,\u201d \u201crearward,\u201d \u201cfront,\u201d \u201cback,\u201d \u201cright,\u201d \u201cleft,\u201d \u201cupwardly,\u201d \u201cdownwardly,\u201d and the like are words of convenience and are not to be construed as limiting terms.","Referring now to the drawings in general, the illustrations are for the purpose of describing a preferred embodiment of the invention and are not intended to limit the invention thereto.","The present invention uses techniques and methodologies developed to analyze evolutionary relationships among living and fossil organisms for the analysis of the relationship of intellectual assets, including but not limited to ideas, inventions, and\/or documents, e.g., patent applications, patent and technical publications, and issued patents and the like. The field of cladistics is a method of analyzing the evolutionary relationships between groups to construct their family tree. The principle behind cladistics is that organisms should be classified according to their evolutionary relationships, and that the way to discover these relationships is to analyze what are called primitive and derived characters. The present invention uses characters of the invention, also called components, elements, or functions, as the analog to organism characters; in the following description, discussion of cladistics and evolutionary biology are used as an analogy or parallel for similar methodologies applied to technology and intellectual assets. As such, any substitution of intellectual assets elements or components for biological characters and\/or characteristics is appropriate in describing the present invention. However, significantly the direct application of cladistics software and\/or methodologies must be adapted and modified in order to function effectively in an intellectual assets environment for research and analysis thereof. It is important to recognize that the invention, for example, must be broken into its functional components in order to be analyzed by a cladistics-based methodology; this selection is critical for an effective analysis of the invention and comparison with other inventions in a related field.","The real merit of cladistic methods is in their use of shared derived characters to unite groups and in the ability of the computer to handle large batches of data. Using cladistics methods, inventions can likewise be grouped and relationships between the objects studied, which in the case of the present invention may include at least one of the following and\/or combinations thereof: ideas, inventions, disclosure documents, patent applications, patent publications, issued patents, technical publications or documents, and the like.","Cladistics is a particular method of hypothesizing relationships among organisms. In the case of the present invention, cladistics-like methodologies are applied to intellectual asset research and analysis by substituting at least one of the following and\/or combinations thereof: ideas, inventions, disclosure documents, patent applications, patent publications, issued patents, technical publications or documents, and the like, in the place of each organism and\/or gene sequence. Like other methods, it has its own set of assumptions, procedures, and limitations. The basic idea behind cladistics is that members of a group share a common evolutionary history, and are \u201cclosely related\u201d, more so to members of the same group than to other organisms; similarly, the present invention employs cladistics-like methodologies to explore and analyze evolutionary patterns within intellectual asset groups and\/or subgroups. These groups are recognized by sharing unique features which were not present in distant ancestors. These shared derived characteristics are called synapomorphies. Note that it is not enough for organisms to share characteristics, in fact two organisms may share a great many characteristics and not be considered members of the same group. For example, consider a jellyfish, starfish, and a human, and consider which two are most closely related. The jellyfish and starfish both live in the water, have radial symmetry, and are invertebrates, so you might suppose that they belong together in a group. This would not reflect evolutionary relationships, however, since the starfish and human are actually more closely related. It is not just the presence of shared characteristics which is important, but the presence of shared derived characteristics. In the example above, all three characteristics are believed to have been present in the common ancestor of all animals, and so are trivial for determining relationships, since all three organisms in question belong to the group \u201canimals\u201d. While humans are different from the other two organisms, they differ only in characteristics which arose newly in an ancestor which is not shared with the other two. Therefore, choosing the right characters is one of the most important steps in a cladistic analysis. Similarly, selecting the appropriate key components and elements, and related subcomponents or subelements, sub-subcomponents or sub-subelements, etc. is one of the most important steps in cladistic-like intellectual asset research and analysis, according to the present invention.","In the analyses of intellectual assets, including but not limited to ideas, inventions, and\/or documents, including but not limited to patent applications, patent and technical publications, and issued patents, it is important to include the use of the invention in analyses. For example, the use of the invention, generally described in the preamble of the independent claim, can be included in the cladistic analysis to ensure that inventions with similar uses are grouped more closely than ones with different uses.","Alternatively, if the use character is not used in the analyses, the resulting analyses will more closely group inventions with similar non-use characters without regard to the invention use. This analysis may be useful for studying convergent evolution of inventions. Similarly, if, in the case of an invention, the element is not used in the analysis, the resulting diagrammatic representation of the group of inventions will produce a more closely related group of inventions with similar elements.","Following is an outline of the basic steps necessary for completing a cladistic analysis of inventions.","1. Determine the characters (features of the invention) and perform a search of an invention database to determine the inventions with one or more of the characters.","2. Group inventions by their shared characteristics.","3. Build an invention cladogram of the groups.","Because essential functional characters generally must be recited in the claims of a patent, searching only the claims field of inventions versus searching the specification field generally results in less erroneous hits. The reduction of these \u201cfalse positives\u201d greatly reduces the burden of reading through the hits. In addition to the basic analysis outlined above, more complicated analyses can be performed prior to mapping the results.","Analytical Methods","For example, the following cladistics or phylogenetic analysis methods, including tree mapping methods, can be used to analyze the dataset, including at least one intellectual asset, invention, and\/or document, including but not limited to patent applications, patent and technical publications, and patent applications: Parsimony methods such as Exhaustive, branch-and-bound and heuristic algorithms, Wagner, Fitch and Dollo characters, Bootstrap, strict and majority rule consensus trees, and Consistency and retention indices; Distance matrix methods; Computation of distances methods; Maximum likelihood methods; Quartets methods; Artificial-intelligence methods; Invariants, or Evolutionary Parsimony, methods; Bootstrapping and other measures of support; Compatibility analysis; Consensus trees and distances between trees; Tree-based alignment; Biogeographic analysis and host-parasite comparison; Comparative method analysis; Simulation of trees or data; Examination of shapes of trees; Clocks, dating and stratigraphy; Biostratigraphy with the methods of Unitary Associations and Ranking-Scaling, Description or prediction of data from trees; Tree plotting\/drawing; and the like.","The results can be mapped using tree mapping methods. In the case of tree mapping analyses, the resulting maps are already mapped.","Artificial intelligence algorithms can also be used. For example, SOTA, a software package that runs a Self Organizing Tree Algorithm, is based on Kohonen's unsupervised neural network of self-organizing maps and on Fritzke's growthing cell structures algorithm to construct phylogenetic trees from biological molecular sequence data. It is described in a paper: Dopazo, J. and J. M. Carazo. 1997. Phylogenetic reconstruction using an unsupervised growing neural network that adopts the topology of a phylogenetic tree. Journal of Molecular Evolution 44a: 226-233. incorporated herein by reference in its entirety. SOTA can use sequence data, distance matrix data, or dipeptide frequencies from proteins. SOTA is available as source code in C for Unix, as executables for SGI workstations, and also with a Windows program called Drawer that draws the resulting trees.","Additionally, because the time component and dependency, which are the time of filing and the cited prior art, are known for inventions, that is, analyses such as invention rate can be computed using mutation rate analysis and mapping methods and invention dependency can be analyzed and mapped using host-parasite analysis and mapping methods.","Tree Types","Once the analyses and groupings have been performed, the results can be displayed in a tree format. Various tree shapes can be used to display the results, including Radial, Slanted cladogram, Rectangular cladogram, Phylogram, and the like.","Phylogram mapping is only available if the tree has branch lengths. Radial trees draw the tree as an unrooted tree radiating from a central point, as shown in . Branches are scaled by their length (if the tree has branch lengths); otherwise each branch has the same length. Some programs store information about the internal nodes as labels for those nodes. Examples include PHYLIP CONSENSE and CLUSTALW which store cluster or split frequencies in the tree description, although they do it differently: The PHYLIP CONSENSE program stores the frequency of groups in the consensus tree as edge lengths. CLUSTALW *.PHB files store bootstrap values as labels of the internal nodes of the tree.","A tree can be ordered such that \u201cheavier\u201d nodes (i.e., those with more descendants) are either drawn to the left or right.","A set of inventions can be defined as the outgroup, such that the tree can be rooted to the group.","Tree Representation","The \u201cNewick 8:45\u201d format is one format used to describe a tree. A brief description follows.","To write a tree description visit all the nodes in the tree, starting at the root, and follow these rules: If the node is a leaf (=terminal taxon) Write the node's label, then return to the node's immediate ancestor. If the branch leading to the leaf has a length, write a colon then the length immediately after the leaf label, e.g \u201chuman:0.0167\u201d. If the node is an internal node: 1. If you're visiting the node for the first time, write a left parenthesis \u201c(\u201d, then visit the node's leftmost child. 2. If you've already visited the node before, but haven't yet visited all that node's descendants, write a comma \u201c,\u201d, then visit the next descendant of the node (going from left to right). 3. If you've already visited the node before, and you've visited all the node's descendants, write a right parenthesis \u201c)\u201d. If the node has a label (e.g. a bootstrap value) then write that label now, e.g. \u201c)100\u201d. If the branch leading to the node has a length, write a colon then the length immediately after the leaf label, e.g \u201c):0.08\u201d. and visit the node's immediate ancestor (if any). If the current node is the root then terminate the description with a semicolon \u201c;\u201d and stop. Here is a simple tree and the sequence of steps used to describe it. Applying the rules to this tree, the description evolves as follows: Step Tree description 1 (2 ((3 ((a 4 ((a, 5 ((a,b 6 ((a,b) 7 ((a,b), 8 ((a,b),(9 ((a,b),(c 10 ((a,b),(c, 11 ((a,b),(c,d 12 ((a,b),(c,d, 13 ((a,b),(c,d,e 14 ((a,b),(c,d,e) 15 ((a,b),(c,d,e)) 16 ((a,b),(c,d,e)); This tree is represented in .","Tree Manipulation","Branches can be moved for rerooting, polytomy formation, and rearranging the appearance of the tree. Some examples of these tree manipulations include:\n\n","The next four tools change the appearance of the tree in the window, but do not affect the cladistic topology:","Branch Rotation rotates the descendants of a binary node, so that the left descendant is now the right descendant, and visa-versa. You can only use this tool on binary internal nodes.","Branch Exchange, which interchanges pairs of branches, rearranges the descendants of a polytomy.","Ladderize left\/right orders the subtree rooted at the branch clicked on so that the \u201cheavier\u201d nodes (those nodes with the largest clusters) appear leftmost\/rightmost (uppermost\/lowermost).","Branch Lengths","If the tree being viewed has branch lengths, and you are viewing the tree as either unrooted or as a phylogram, then the branch lengths can be drawn to scale. The units for this scale depend on those used to construct the tree. For example, as shown in , most tree-building programs output distances as numbers of substitutions per site. For such trees, some programs typically displays a scale of \u201c0.1\u201d, meaning 0.1 nucleotide substitutions per site. The actual value will depend on the branch lengths in the tree. Other trees, such as those computed using parsimony, may have integer branch lengths (i.e., 1, 5, 10), and hence the scale bar will be in units appropriate to the tree. For example, a value of \u201c10\u201d means 10 steps.","Evolutionary Distances","Evolutionary distance between a pair of sequences is usually measured by the number of nucleotide (or amino acid) substitutions between them. Evolutionary distances are fundamental for the study of molecular evolution and are useful for phylogenetic reconstruction and estimation of divergence times.","In MEGA, most of the widely used methods for distance estimation for nucleotide and amino acid sequences are included. In the following, a brief discussion of these methods is presented in three sections: nucleotide substitutions, synonymous-nonsynonymous substitutions, and amino acid substitutions. Further details of these methods and general guidelines for the use of these methods are given in Nei and Kumar (2000). Note that in addition to the distance estimates, MEGA2 also computes the standard errors of the estimates using the analytical formulas and the bootstrap method.","Nucleotide Sequences are compared nucleotide-by-nucleotide. These distances can be computed for protein coding and non-coding nucleotide sequences.","Synonymous & nonsynonymous Sequences are compared codon-by-codon. These distances can only be computed for protein-coding sequences or domains.","Amino acid Amino acid sequences are compared residue-by-residue. These distances can be computed for protein sequences and protein-coding nucleotide sequences. In the latter case, protein-coding nucleotide sequences are automatically translated using the selected genetic code table.","Distance methods included in MEGA are as follows: Nucleotide No. of differences p-distance Jukes-Cantor distance (with Gamma model) Tajima-Nei distance Kimura 2-parameter distance (with Gamma model) Tamura 3-parameter distance Tamura-Nei distance (with Gamma model) Syn-Nonsynonymous Nei-Gojobori Method Modified Nei-Gojobori Method Li-Wu-Luo Method Pamilo-Bianchi-Li Method Kumar Method Amino acid No. of differences p-distance Poisson correction Gamma distance","Bootstrap","The Bootstrap method computes standard error of distance estimates. When you choose the bootstrap method for estimating the standard error, you need to specify the number of replicates and the seed for the psuedorandom number generator. In each bootstrap replicate, the desired quantity will be estimated and then the standard deviation of the original values will be computed (see Nei and Kumar [2000], page 25 for details). It is possible that in some bootstrap replicates the quantity you desired is not calculable due to statistical or technical reasons. In this case, MEGA will discard the results of those bootstrap replicates and the final estimates will be based on the results of all the valid replicates. This means that the actual number of bootstrap replicates used can be smaller than the number specified by the user. However, if the number of valid bootstrap replicates is less than 25, then MEGA will report that the standard error cannot be computed (a n\/c sign will appear in the result window).","DNA and RNA analysis programs, which are based on comparison using 4 character states\u2014ATCG or AUCG, can still be used to analyze inventions, although only 2 character states per function will be used.","Examples of other analytical software packages that use these and other methods that can be used to map invention relationships:","Unix (source code in C or executables): PHYLIP; PAUP*; Fitch programs; Phylo_win ODEN TreeTree GCG Wisconsin Package SeqPup Lintre RSVP Microsat OSA TREE-PUZZLE AMP fastDNAml MOLPHY PAML SplitsTree PLATO STATGEOM PHYLTEST PARBOOT TreeAlign ClustalW MALIGN GeneDoc COMPARE TheSiminator Seq-Gen TreeTool GDE sog TreePack Phylodendron Treevolve and PTreevolve PSeq-Gen njbafd gmaes GCUA DERANGE2 LVB BIONJ TAAR ANCML QDate Bootscanning Package Ctree SOTA PASSML TOPAL reticulate RecPars ARB BIOSYS-2 RAPD-PCR package TreeCons Diversi DISTANCE Darwin sendbs partimatrix BAMBE nneighbor unrooted ROSE weighbor PhyloQuart QR2 VeryfastDNAml LARD puzzleboot Willson quartets programs POY RIND TipDate RRTree Fels-Rand PAL Mavric dnarates CLINCH UO Arlequin vCEBL TrExMl HY-PHY Genie Vanilla PHYCON qclust fastDNAmlRev RevDNArates BRANCHLENGTH TCS CONSEL","PC's as Windows executables: PHYLIP PAUP* Tree Gardener TREECON GDA SeqPup MOLPHY WET GeneDoc COMPONENT TREEMAP COMPARE RAPDistance TreeView Phylodendron Molecular Analyst Fingerprinting POPGENE TFPGA Ctree GeneTree MVSP RSTCALC Genetix NJplot unrooted Arlequin DAMBE DnaSP PAML LVB DNASIS minspnet BioEdit ProSeq RRTree Fels-Rand PAL WINCLADA SECANT Nona DNASEP SEPAL Phylogenetic Independence vCEBL HY-PHY TreeExplorer Genie Vanilla MEGA TNT GelCompar II Bionumerics TCS","DOS executable (MSDOS, PCDOS) or in a Windows \u201cDOS box\u201d: PHYLIP PAUP* MEGA Fitch programs Hennig86 MEGA RA Nona TurboTree Freqpars Fitch programs TREECON Microsat DISPAN RESTSITE NTSYSpc METREE Hadtree, Prepare and Trees PHYLTEST RAPDistance DIPLOMO TREE-PUZZLE ABLE ClustalW MALIGN GeneDoc COMPARE CMAP Random Cladistics CoSta njbafd GEOMETRY PDAP PICA95 REDCON TAXEQ2 BIONJ ANCML REAP MVSP Lintre BIOSYS-2 RAPD-PCR package Diversi T-REX sendbs K2WuLi homoplasy test weighbor POY TreeDis QUARTET2 Network CLINCH Gambit MEAWILK qclust CONSEL","Macintosh or PowerMac executables: PHYLIP PAUP* CAFCA MacT TreeTree SeqPup Microsat TREE-PUZZLE fastDNAml MacClade Spectrum SplitsTree PLATO AutoDecay RASA ClustalW TREEMAP CAIC COMPARE PA Bi-De SEQEVOLVE TheSiminator Seq-Gen End-Epi StratCon CONSERVE TreeView NJplot DendroMaker MUST DNA Stacks Phylogenetic Investigator Tree Draw Deck Phylodendron TreeRot Treevolve and PTreevolve PSeq-Gen Molecular Analyst Fingerprinting BIONJ GCUA ACAP GeneTree QDate LVB T-REX unrooted GeneStrut COMPONENT Lite weighbor Modeltest PAML LARD MATRIX Willson quartets programs ALIGN CodonBootstrap DNASIS TipDate RRTree MacroCAIC Fels-Rand PAL RadCon TreeEdit Arlequin vCEBL HY-PHY TreeThief Genie Sequencer Vanilla TCS MrBayes","VMS executables or C sources with VMS compilation support: PHYLIP MARKOV TREE-PUZZLE fastDNAml TreeAlign ClustalW","An example of an analysis with mapping of the results follows:","A described invention was determined to have the following key characters: N-methyl-D-aspartate (nmda); neurotransmitter; and glycine. The N-methyl-D-aspartate AND nmda was used for N-methyl-D-aspartate and the wordstem neurotransmi# was used for the keyword neurotransmitter. Boolean searches of the USPTO database were performed with these words. The first search was performed requiring all characters to exist in the claims field. The resulting dataset had 6 positives, denoted as \u201cALL\u201d and listed in Table 1. Successive searches were performed by sequentially applying the ANDNOT Boolean modifier to each element in the first search. The results of each search are denoted as NOT glycine, NOT neurotransmi#, and NOT NMDA and shown in Tables 2, 3, and 4, respectively. The datasets were imported into a database and organized into a character matrix. The matrix was analyzed by heuristic analysis and the resulting tree mapped using tree mapping methods. The result is shown in .",{"@attributes":{"id":"p-0076","num":"0088"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Results of Search in 1976 to present db for: (((ACLM\/nmda"},{"entry":"OR ACLM\/N-methyl-D-aspartate) AND ACLM\/neurotransmi$) AND"},{"entry":"ACLM\/glycine): 6 patents. Hits 1 through 6 out of 6"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"126pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Pat. No.","Title"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"126pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"1","6,395,780","Cleavage system inhibitors as"]},{"entry":[{},{},{},"potential antipsychotics"]},{"entry":[{},"2","6,355,681","Glycine substitutes and"]},{"entry":[{},{},{},"precursors for treating a"]},{"entry":[{},{},{},"psychosis"]},{"entry":[{},"3","6,228,875","Methods for treating"]},{"entry":[{},{},{},"neuropsychiatric disorders"]},{"entry":[{},"4","5,854,286","Treatment of negative and cognitive"]},{"entry":[{},{},{},"symptoms of schizophrenia with"]},{"entry":[{},{},{},"glycine and its precursors"]},{"entry":[{},"5","5,837,730","Treatment of negative and cognitive"]},{"entry":[{},{},{},"symptoms of schizophrenia with"]},{"entry":[{},{},{},"glycine uptake antagonists"]},{"entry":[{},"6","5,668,117","Methods of treating neurological"]},{"entry":[{},{},{},"diseases and etiologically related"]},{"entry":[{},{},{},"symptomology using carbonyl"]},{"entry":[{},{},{},"trapping agents in combination with"]},{"entry":[{},{},{},"previously known medicaments"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0077","num":"0089"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Results of Search in 1976 to present db for: (((ACLM\/nmda"},{"entry":"OR ACLM\/N-methyl-D-aspartate) AND ACLM\/neurotransmi$) ANDNOT"},{"entry":"ACLM\/glycine): 10 patents. Hits 1 through 10 out of 10"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"126pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Pat. No.","Title"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"14pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"126pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"1","6,420,351","Methods for treating"]},{"entry":[{},{},{},"neuropsychiatric disorders"]},{"entry":[{},"2","6,391,922","Treatment of posttraumatic"]},{"entry":[{},{},{},"stress disorder, obsessive-"]},{"entry":[{},{},{},"compulsive disorder and"]},{"entry":[{},{},{},"related neuropsychiatric disorders"]},{"entry":[{},"3","6,391,871","Preventing neuronal degeneration"]},{"entry":[{},{},{},"in Alzheimer's disease"]},{"entry":[{},"4","6,294,583","Methods of treating tardive"]},{"entry":[{},{},{},"dyskinesia and other movement"]},{"entry":[{},{},{},"disorders"]},{"entry":[{},"5","6,057,373","Methods of treating tardive"]},{"entry":[{},{},{},"dyskinesia and other movement"]},{"entry":[{},{},{},"disorders using NMDA receptor"]},{"entry":[{},{},{},"antagonists"]},{"entry":[{},"6","5,804,550","Peptide antagonists at glutamate"]},{"entry":[{},{},{},"and NMDA receptors"]},{"entry":[{},"7","5,648,369","Aminoalkylpyridine compounds"]},{"entry":[{},{},{},"which are useful as anitconvulsant"]},{"entry":[{},{},{},"drugs, excitatory amino acid"]},{"entry":[{},{},{},"inhibitors and NMDA sigma receptor"]},{"entry":[{},{},{},"antagonists"]},{"entry":[{},"8","5,641,798","Bicyclic compounds and their use"]},{"entry":[{},{},{},"as excitatory amino acid receptor"]},{"entry":[{},{},{},"antagonists"]},{"entry":[{},"9","5,576,323","Excitatory amino acid receptor"]},{"entry":[{},{},{},"antagonists"]},{"entry":[{},"10","5,356,902","Decahydroisoquinoline compounds"]},{"entry":[{},{},{},"as excitatory amino acid receptor"]},{"entry":[{},{},{},"antagonists"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0078","num":"0090"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Results of Search in 1976 to present db for:"},{"entry":"(((ACLM\/nmda OR ACLM\/N-methyl-D-aspartate) AND"},{"entry":"ACLM\/glycine) ANDNOT ACLM\/neurotransmi$): 33"},{"entry":"patents. Hits 1 through 33 out of 33"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"140pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Pat. No.","Title"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"14pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"140pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"1","6,511,963","Allosteric modulators of the NMDA"]},{"entry":[{},{},{},"receptor and their use in the"]},{"entry":[{},{},{},"treatment of CNS disorders and"]},{"entry":[{},{},{},"enhancement of CNS function"]},{"entry":[{},"2","6,482,854","Glaucoma treatment"]},{"entry":[{},"3","6,413,985","Tetrahydroquinoline derivatives"]},{"entry":[{},{},{},"as glycine antagonists"]},{"entry":[{},"4","6,362,226","Modulation of in vivo glutamine"]},{"entry":[{},{},{},"and glycine levels in the treatment"]},{"entry":[{},{},{},"of autism"]},{"entry":[{},"5","6,362,199","Tetrahydroquinoline derivatives as"]},{"entry":[{},{},{},"glycine antagonists"]},{"entry":[{},"6","6,284,731","Allosteric modulators of the NMDA"]},{"entry":[{},{},{},"receptor and their use in the"]},{"entry":[{},{},{},"treatment of CNS disorders and"]},{"entry":[{},{},{},"enhancement of CNS function"]},{"entry":[{},"7","6,107,296","Neuroprotective use of triazolo-"]},{"entry":[{},{},{},"pyridazine derivatives"]},{"entry":[{},"8","6,107,271","Neuroactive peptides"]},{"entry":[{},"9","6,080,743","2,3-dioxo-1,2,3,4-tetrahydro-"]},{"entry":[{},{},{},"quinoxalinyl derivatives"]},{"entry":[{},"10","6,063,774","Tricyclic quinoxaline derivatives"]},{"entry":[{},{},{},"as neuroprotective agents"]},{"entry":[{},"11","6,025,369","N-methyl-D-aspartate (NMDA) receptor"]},{"entry":[{},{},{},"blockers for the prevention of"]},{"entry":[{},{},{},"atherosclerosis"]},{"entry":[{},"12","6,017,957","Partial agonists of the strychnine"]},{"entry":[{},{},{},"insensitive glycine modulatory site of"]},{"entry":[{},{},{},"the N-methyl-D-aspartate receptor"]},{"entry":[{},{},{},"complex as neuropsychopharma-"]},{"entry":[{},{},{},"cological agents"]},{"entry":[{},"13","5,942,540","Methods of providing symptomatic and"]},{"entry":[{},{},{},"prophylactic neuroprotection"]},{"entry":[{},"14","5,856,335","Substituted aminothienopyridines,"]},{"entry":[{},{},{},"pharmaceutical composition and use"]},{"entry":[{},"15","5,834,465","Treatment with combined NMDA and non-"]},{"entry":[{},{},{},"NMDA antagonists to reduce"]},{"entry":[{},{},{},"excitotoxic CNS damage"]},{"entry":[{},"16","5,760,059","Indole derivatives"]},{"entry":[{},"17","5,728,728","Methods of providing neuroprotection"]},{"entry":[{},"18","5,646,146","Heterocyclic compounds and their"]},{"entry":[{},{},{},"preparation and use"]},{"entry":[{},"19","5,633,379","3-heteroaliphatyl- and 3-"]},{"entry":[{},{},{},"hetero(aryl)aliphatyl-2(1H)-quinolone"]},{"entry":[{},{},{},"derivatives"]},{"entry":[{},"20","5,620,979","Glycine receptor antagonists and the"]},{"entry":[{},{},{},"use thereof"]},{"entry":[{},"21","5,614,509","Pharmaceutical agents for preventing"]},{"entry":[{},{},{},"the development of tolerance during"]},{"entry":[{},{},{},"the treatment with benzodiazepine-"]},{"entry":[{},{},{},"receptor-binding active ingredients"]},{"entry":[{},"22","5,576,436","Fluorescent ligands"]},{"entry":[{},"23","5,523,323","Use of partial agonists of the NMDA"]},{"entry":[{},{},{},"receptor to reduce opiate induced"]},{"entry":[{},{},{},"tolerance and dependence"]},{"entry":[{},"24","5,514,680","Glycine receptor antagonists and"]},{"entry":[{},{},{},"the use thereof"]},{"entry":[{},"25","5,506,231","Treatment of aids dementia, myelopathy"]},{"entry":[{},{},{},"and blindness"]},{"entry":[{},"26","5,474,990","Barbiturates as safening agents in"]},{"entry":[{},{},{},"conjunction with NMDA antagonists"]},{"entry":[{},"27","5,468,748","9H-indeno[1,2-b]pyrazine"]},{"entry":[{},{},{},"derivatives"]},{"entry":[{},"28","5,434,295","Neuroprotective pharmaceutical"]},{"entry":[{},{},{},"compositions of 4-phenylpinene"]},{"entry":[{},{},{},"derivatives and certain novel"]},{"entry":[{},{},{},"4-phenylpinene compounds"]},{"entry":[{},"29","5,256,690","Method for treating and controlling"]},{"entry":[{},{},{},"the symptoms of neurodegenerative"]},{"entry":[{},{},{},"disease and neuropsychopharmacological"]},{"entry":[{},{},{},"disorders"]},{"entry":[{},"30","5,240,946","((2-(amino-3,4-dioxo-1-cyclobuten-1-"]},{"entry":[{},{},{},"yl)amino)alkyl)-acid derivatives"]},{"entry":[{},"31","5,212,167","Modulation of receptor-mediated ion"]},{"entry":[{},{},{},"transport"]},{"entry":[{},"32","5,166,155","Quinoxaline-2,3-dione compounds and"]},{"entry":[{},{},{},"their preparation and use"]},{"entry":[{},"33","5,086,072","Treatment of mood disorders with"]},{"entry":[{},{},{},"functional antagonists of the"]},{"entry":[{},{},{},"glycine\/NMDA receptor complex"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0079","num":"0091"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 4"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Results of Search in 1976 to present db for:"},{"entry":"((ACLM\/glycine AND ACLM\/neurotransmi$) ANDNOT"},{"entry":"(ACLM\/nmda OR ACLM\/N-methyl-D-aspartate)): 10"},{"entry":"patents. Hits 1 through 10 out of 10"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"140pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Pat. No.","Title"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"14pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"140pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"1","6,287,254","Animal health diagnosis"]},{"entry":[{},"2","6,251,280","Imprint-coating synthesis of selective"]},{"entry":[{},{},{},"functionalized ordered mesoporous"]},{"entry":[{},{},{},"sorbents for separation and sensors"]},{"entry":[{},"3","6,106,858","Modulation of drug loading in"]},{"entry":[{},{},{},"multivescular liposomes"]},{"entry":[{},"4","5,938,903","Microelectrodes and their use in an"]},{"entry":[{},{},{},"electrochemical arrangement with"]},{"entry":[{},{},{},"telemetric application"]},{"entry":[{},"5","5,750,376","In vitro growth and proliferation of"]},{"entry":[{},{},{},"genetically modified multipotent"]},{"entry":[{},{},{},"neural stem cells and their progeny"]},{"entry":[{},"6","5,587,509","Caged caboxyl compounds and use thereof"]},{"entry":[{},"7","5,519,032","Substituted aminothienopyridines,"]},{"entry":[{},{},{},"pharmaceutical composition and use"]},{"entry":[{},"8","5,498,626","Acylaminoindole derivatives as 5-ht1"]},{"entry":[{},{},{},"agonists"]},{"entry":[{},"9","5,424,185","Human high-affinity neurotransmitter"]},{"entry":[{},{},{},"uptake system"]},{"entry":[{},"10","5,225,323","Human high-affinity neurotransmitter"]},{"entry":[{},{},{},"uptake system"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}},"br":{},"figref":"FIGS. 3 and 4"},"As best seen in , a flowchart of a method for invention or technology analysis according to the present invention includes the steps of invention deconstruction, database queries formulation, queries submission, results retrieval, and results display.","1. Invention Deconstruction","The invention or entity deconstruction step deconstructs an invention into key components. These key components are those that can be used to distinguish the invention from the prior art. Key components may include such things as physical features, feature location, benefit conferred by a feature, and the like. The field of the invention or the name of the invention may also be included as a key component.","2. Mapping Methods or Query Formulation","The mapping methods herein described segregate a technology or invention field into sub-fields by serial exclusion of key components from queries. The results are kept segregated and displayed as sub-fields. The mapping methods use the key components identified in the deconstruction step to form multiple combinatorial component queries. This mapping, herein termed combinatorial component exclusion mapping, is performed by formulating multiple database queries from the key components identified in the deconstruction step by using combinatorial component exclusion. These combinatorial queries include a first query that searches for all the components in the desired database field, termed the \u201cniche\u201d or \u201cALL\u201d query, and a series of \u201cneighborhood\u201d queries formed by serial exclusion of at least one component from the query strings. An example follows.","An invention was described as being a contact lens coated with the antimicrobial lactoferrin. The key components of this invention were determined to be a contact lens, lactoferrin, and a coating. Note that the first component is the invention, the second is a physical feature, and the third is the location of the physical feature. These key components are expanded to include synonyms and word stems and are then used to formulate queries according to the present invention. The queries resulting from these components are:",{"@attributes":{"id":"p-0084","num":"0096"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 5"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Unique combinatorial search strings for 3 components."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"126pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Query type","Search strings"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["Niche (ALL - A AND B AND C)","\u201ccontact lens\u201d"]},{"entry":[{},"AND lactoferrin AND (coat$"]},{"entry":[{},"or surface)"]},{"entry":["Neighborhood 2 (A AND B NOT C)","\u201ccontact lens\u201d"]},{"entry":[{},"AND lactoferrin NOT (coat$"]},{"entry":[{},"or surface)"]},{"entry":["Neighborhood 3 (A NOT B AND C)","\u201ccontact lens\u201d"]},{"entry":[{},"NOT lactoferrin AND (coat$"]},{"entry":[{},"or surface)"]},{"entry":["Neighborhood 4 (NOT A AND B AND C)","NOT \u201ccontact lens\u201d"]},{"entry":[{},"AND lactoferrin AND"]},{"entry":[{},"(coat$ or surface)"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"Advantages of Combinatorial Component Exclusion Mapping","Combinatorial component exclusion mapping offers several advantages over the prior art. The mapping is simultaneously precise and thorough. The mapping is precise because the niche group will generally contain the records most like the present invention. The mapping is thorough because adjacent technology fields that contain closely related prior art are also mapped and can be searched. The process is easily supervised because the mapping method is predetermined\u2014the supervisor simply has to determine whether the examiner has identified the proper key components for the mapping and has expanded them correctly and thoroughly.","The method also minimizes in several ways the number of records that an examiner must review to perform a precise and thorough examination. The examiner can look at the niche group to quickly determine if the prior art anticipates or obviates the invention. The examiner can next go to the neighborhood most likely to contain anticipating or obviating art. Knowing which components are missing in a neighborhood prepares the reviewer to look for records that will make the examined invention obvious when reviewing the neighborhoods.","Another advantage of a preferred embodiment of the present invention is the use of ANDNOT exclusion to eliminate overlap. Without the ANDNOT exclusion, the examiner would re-review the ALL group when reviewing each of the neighborhoods. The benefit of the ANDNOT exclusion increases when simultaneously excluding multiple components.","The multiple search strings generated according to the present invention are preferably automatically generated by a computer system.","3. The Search Strings are Submitted to at Least One Database Query Engine for Querying at Least One Database.","The submission to the at least one database query engine is preferably automated and performed by a computer system. The automated query generation and submission facilitates the method and reduces the possibility of error. An example of an automated interface is shown in .","4. Results Retrieval.","Preferably, the results are initially displayed as a summary or sounding page, showing the number of hits for each query in each database queried. An example of a sounding page is shown in . In this example, the user sees that the niche search shown in the first row produced 23 hit in the Publications database and 40 hits in the Patents database.","The user next retrieves preferably only the information needed. There are preferably at least two levels of retrieval. In the present example, by selecting a box in the sounding view from the first level, shown under columns \u201cPubs1\u201d and \u201cPats1\u201d, the user retrieves only the grant\/application number and title. The user can then link to the hits through the icon beside each box to get the hits screen, as shown in , as is commonly done on search engines. In addition to showing the number and title of each hit, the user can link through the Patent title to retrieve the patent from the USPTO or local database and can also launch the Patent Matrix software and import the claims from the USPTO by clicking on the software launching link beside each title.","By selecting a box under the columns \u201cPubs2\u201d or \u201cPats2\u201d, the user directs the system to retrieve relevant bibliographic and content information, such as the abstract, priority cross-reference data (CIP column), assignee information, inventor information, Issue date, filing date, publication date, contact information, and the like.","5. Results Display for Examination and Analysis.","The results can be displayed in several formats. A spreadsheet layout, can be used. Additionally or alternatively, a tree diagram layout can be used.","The spreadsheet layout, as shown in , preferably has separate sheets for each query result; each sheet preferably having at least one column for predetermined database fields and rows for the records retrieved. This format allows for the rapid review of records. This layout also allows for sorting the array by columns, such that reviewer can prioritize records according to one or more of the database fields.","The results displays also preferably incorporate information in mouse-over, or hover, pop-ups. These can be data which are too large to be reasonably displayed on the spreadsheet, but are immediately desirable to be viewed once the viewer would like to examine a record in more detail.","For example, the abstract can be inserted in a rollover box in the title and the priority cross-reference data can be inserted in a rollover box in the CIP status (YES\/NO), as shown in , respectively. The claims can also be extracted from the patent and shown in a rollover box. For example, the first independent claim can be inserted into the rollover box when the user hovers over a button that hyperlinks to or launches a patent matrix diagram of the patent, shown as \u201cPMD\u201d in . The presentation of the first independent claim allows the user to rapidly screen grants\/applications for relevance by minimizing click-through to other pages and thereby staying on the same page view.","6. Classification, Notes.","The user can then select and\/or flag documents based on their examination criteria. For example, as shown in , the user interface contains 3 columns named \u201c102\u201d, \u201c103\u201d, and \u201cCited\/NRO\u201d. These denominations are used for patent examination by the patent office to classify prior art references as providing anticipation (102) or obviousness (103) or \u201ccited but not relied upon\u201d to an application under examination.","The Tree Diagram Layout According to Another Embodiment of the Present Invention","The tree diagram layout displays all of the results from the mapping in a single diagram. This format allows for an overview of the field and conveys a sense of the density of the prior art. The tree diagrams can be rooted or unrooted. The prior described search was rerun limiting the components to the claims database field to reduce the number of records for display purposes. The results were mapped in an unrooted tree diagram, shown in . Shown are the four groups\u2014the niche and immediately adjacent neighborhoods. Each of the records is displayed in the group; the label for each record at the end of a spoke radiating from hub. In this view the patents and pregrant publications are group in a single hub. The resolution of each hub can be increased by expanding the hub, as shown in . In , the pointer is also hovering over content item or U.S. Pat. No. 6,565,861 and relevant patent information, as previously discussed herein, is displayed in a pop-up window. Notes and other information added during examination can also be displayed, as shown for this record.","7. Preprogrammed Searches","Niche and Neighborhood Search","The simplest search is a \u201cniche-and-neighborhood\u201d search, shown previously. In this search, each component is \u201cexcluded\u201d from the search string using the ANDNOT boolean operator. This use of the ANDNOT also prevents redundancy in the search results. Simply omitting a component, a component omission exclusion, would results in the inventions that have all the components (ALL group) showing up again in all the neighborhoods.","Teaching Specification Search","A teaching specification search is designed to assist in quickly screening patents for obviousness. Obviousness here refers to the criteria used by the Patent Offices.","In a Teaching specification search, the components are inputted into the search screen as before, and the components' are automatically delimited to the claims field. The software automatically generates the specific neighborhood search strings, which are designed to exclude the patents with the component in the claims, but include patents with the component in the specification. These strings are shown in the \u201cPermutation\u201d column in . An example string for a neighborhood is:\n\n","In this string\/neighborhood, grants\/applications with coat$ or surface in the claims are excluded, while grants\/applications with coat$ or surface not in the claims but in the specification are included.","Note that the permutation are similar to the Niche\/Neighborhood search, except that when a component is removed (ANDNOT) from the search in the claims field (aclm\/ . . . ); it is added to the search in the specification field (spec\/ . . . ). These groups of patents are ones that do not specifically claim the component, but do teach the component in some way in the specification. Therefore, it may be obvious to add this taught component to the prior art and create a new invention.","Note the reduced number of patents in the neighborhoods with respect to the search shown in . This reduction in number reduces the workload for a user examining an invention for obviousness or anticipation if a quick elimination can be made based on anticipation or obviousness.","Combinatorial Searches","Combinatorial searches are designed to break a technology field into multiple niches based on key components or technologies. They are an extension of the Niche\/Neighborhood searches. For example, if the key components to be examined were a) contact lens b) lactoferrin and c) coating, the possible unique combinatorial searches would be 7, as shown in Table 6.",{"@attributes":{"id":"p-0109","num":"0122"},"tables":{"@attributes":{"id":"TABLE-US-00006","num":"00006"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 6"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Unique combinatorial search strings for 3 components."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"119pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Search #","Search strings"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":[{},"1 (ALL)","a AND b AND c"]},{"entry":[{},"2","a AND b ANDNOT c"]},{"entry":[{},"3","a ANDNOT b AND c"]},{"entry":[{},"4","b AND c ANDNOT a"]},{"entry":[{},"5","a ANDNOT b ANDNOT c"]},{"entry":[{},"6","b ANDNOT c ANDNOT a"]},{"entry":[{},"7","c ANDNOT b ANDNOT a"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"For 3 components there are 7 combinatorial search strings. For 4 components there are 15 possible unique strings and for 5 components there are 31 possible unique strings. The results are mapped for visual display. A combinatorial search can be exhaustive, that is, every unique combination is submitted. This analysis is called an Exhaustive Combinatorial Search.","Sometimes the search strings with only a single component return a large number of meaningless hits. For example, searches 5, 6, and 7 in Table 6 may produce a large number of hits that are too burdensome and have too little meaning to be examined by the user. In these cases, a Proximal Combinatorial Search is used. This search generates strings only out to double component strings. That is, the strings with only a single component, such as strings 5, 6, and 7 above, are not generated. In the case of a 3-component system, the number of search strings is the same as for a Niche\/Neighborhood analysis. For a 4-component system, however, there will be 11 strings for a Proximal Combinatorial search, versus 5 for a Niche\/Neighborhood analysis and 15 for an Exhaustive Combinatorial Search. For a 5 components system, the Proximal Combinatorial Search will generate 26 search strings versus 6 for a Niche\/Neighborhood analysis. Table 7 illustrates the different number of groups generated by each search for a given number of components.",{"@attributes":{"id":"p-0112","num":"0125"},"tables":{"@attributes":{"id":"TABLE-US-00007","num":"00007"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"center"}}],"thead":{"row":[{"entry":"TABLE 7"},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":[{},"Niche\/Neighborhood",{},{}]},{"entry":["Compo-","OR Teaching","Proximal","Exhaustive"]},{"entry":["nents","specification search","Combinatorial","Combinatorial"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["3","4","4","7"]},{"entry":["4","5","11","15"]},{"entry":["5","6","26","31"]},{"entry":["6","7","57","63"]},{"entry":["7","8","120","127"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}},"br":{}},"Differential combinations of these searches can also be performed. For example, the user can perform a Niche-and-Neighborhood minus a Teaching specification search or vice-versus, or a Combinatorial search minus a Niche-and-Neighborhood or Teaching specification search. The ability to subtract the results of one search from another allows the user to progress through the search types with the same components without having to re-review records already reviewed in a previous search.","Trends Analysis","A system according to the present inventions also provides analyses of trends in the searched technology fields. These trends analyses, also called growth, flux or invention rate analyses, include analyses such as the Publication\/patent ratio; Publication and\/or patent trend, such as Publications\/unit time, Date difference between neighboring publications; Art unit or examination density or latency, such as Filing-to-issuance time period, Issuance\/Publication ratio; Technology field analyses, such as Emigration\/Immigration of assignees\/inventors, Technology field lifecycle analyses; and Date-based analyses, such as patent flux and parsing into time intervals.","Growth analyses, which may also be termed flux analyses or invention rate, are metrics used to determine the absolute and relative rate of new patent applications in a particular area, such as a niche, neighborhood, classification or similar area.","A preferred growth analysis is the publication\/patent ratio, shown in the \u201cRatio\u201d column in . In this analysis, the number of publication for a defined area is divided by the number of issued patents for the area to give the ratio. The number can be converted to a percentage, if desired. This ratio is an indication of the growth, since an area only grows by submission of new applications.","Another type of growth analysis is the applications and\/or patent trend analysis, which is the number of applications or patents\/unit time. This metric can be displayed as an absolute number, polynomial function to show change, or as a graph.","Another Patent trend analysis is the date difference between neighboring publications. This computation shows activity peaks and trends. For example, if this metric is decreasing with time, then patent applications are increasing in the analyzed area.","Examination latency analysis is another useful metric. This analysis is performed for an area by calculating the filing-to-issuance time period for patents in the area, the reasoning being that more mature technology areas will require longer prosecution prior to issuance and therefore will have a longer filing-to-issuance period. The issuance\/publication ratio can be used as a similar metric. This analysis is performed by dividing the patents issued in a time period by the patent applications in a time period. The time periods can be the same, or can be different. For example, if the earliest and latest filing dates for the issued patents in the patent time period is known, the applications time period can be the time period between the two filing dates.","Another indicator of the growth and\/or maturity status of an area is the emigration\/immigration of assignees and\/or inventors into and out of the area.","Date-based growth analyses may also be performed. For example, the distance between neighboring grants\/applications can be computed, for example based on the filing date. Alternatively or additionally, for patent systems with serially numbered patents, the difference between patent numbers may be used. The grants\/applications may also be parsed into time intervals, and graphed or converted to a polynomial to demonstrate growth.","Certain modifications and improvements will occur to those skilled in the art upon a reading of the foregoing description. By way of example, the diagrams may be produced in a variety of forms and formats without departing from the scope of the present invention. Also, the software used to generate the diagrams may vary widely; any examples provided herein are for the purpose of illustrating the present invention using commercially available software. Additionally, the methods and systems of the present invention may apply to searching and analysis of various types of content in databases. All modifications and improvements have been deleted herein for the sake of conciseness and readability but are properly within the scope of the following claims."],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0016","num":"0025"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0017","num":"0026"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0018","num":"0027"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0019","num":"0028"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0020","num":"0029"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0021","num":"0030"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0022","num":"0031"},"figref":"FIG. 7"},{"@attributes":{"id":"p-0023","num":"0032"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0024","num":"0033"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0025","num":"0034"},"figref":"FIG. 10"},{"@attributes":{"id":"p-0026","num":"0035"},"figref":"FIG. 11"},{"@attributes":{"id":"p-0027","num":"0036"},"figref":"FIG. 12"},{"@attributes":{"id":"p-0028","num":"0037"},"figref":"FIG. 13"},{"@attributes":{"id":"p-0029","num":"0038"},"figref":"FIG. 14"}]},"DETDESC":[{},{}]}
