---
title: Antisense antiviral compound and method for treating influenza viral infection
abstract: The invention provides antisense antiviral compounds and methods of their use and production in inhibition of growth of viruses of the Orthomyxoviridae family and in the treatment of a viral infection. The compounds are particularly useful in the treatment of influenza virus infection in a mammal. The antisense antiviral compounds are substantially uncharged morpholino oligonucleotides having 1) a nuclease resistant backbone, 2) 12-40 nucleotide bases, and 3) a targeting sequence of at least 12 bases in length that hybridizes to a target region selected from the following: a) the 5′ or 3′ terminal 25 bases of the negative sense viral RNA segment of Influenzavirus A, Influenzavirus B and Influenzavirus C; b) the terminal 25 bases of the 3′ terminus of the positive sense cRNA and; and c) the 50 bases surrounding the AUG start codon of an influenza viral mRNA.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08357664&OS=08357664&RS=08357664
owner: M.I.T.
number: 08357664
owner_city: Cambridge
owner_country: US
publication_date: 20051025
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"p":["This application claims priority to U.S. provisional patent application No. 60\/622,077 filed Oct. 26, 2004, which is incorporated herein in its entirety by reference.","This invention was made with government support under grant number R01 AI056267 awarded by the National Institutes of Health. The government has certain rights in this invention.","The invention relates to antisense oligonucleotide compounds for use in treating an influenza virus infection and antiviral treatment methods employing the compounds.",{"@attributes":{"id":"p-0005","num":"0000"},"ul":{"@attributes":{"id":"ul0001","list-style":"none"},"li":["Agrawal, S., S. H. Mayrand, et al. (1990). \u201cSite-specific excision from RNA by RNase H and mixed-phosphate-backbone oligodeoxynucleotides.\u201d 87(4): 1401-5.","Blommers, M. J., U. Pieles, et al. (1994). \u201cAn approach to the structure determination of nucleic acid analogues hybridized to RNA. NMR studies of a duplex between 2\u2032-OMe RNA and an oligonucleotide containing a single amide backbone modification.\u201d 22(20): 4187-94.","Bonham, M. A., S. Brown, et al. (1995). \u201cAn assessment of the antisense properties of RNase H-competent and steric-blocking oligomers.\u201d 23(7): 1197-203.","Boudvillain, M., M. Guerin, et al. (1997). \u201cTransplatin-modified oligo(2\u2032-O-methyl ribonucleotide)s: a new tool for selective modulation of gene expression.\u201d 36(10): 2925-31.","Cox, N. J. and K. Subbarao (1999). \u201cInfluenza.\u201d 354(9186): 1277-82.","Cox, N. J. and K. Subbarao (2000). \u201cGlobal epidemiology of influenza: past and present.\u201d 51: 407-21.","Cross, C. W., J. S. Rice, et al. (1997). \u201cSolution structure of an RNA\u00d7DNA hybrid duplex containing a 3\u2032-thioformacetal linker and an RNA A-tract.\u201d 36(14): 4096-107.","Dagle, J. M., J. L. Littig, et al. (2000). \u201cTargeted elimination of zygotic messages in embryos by modified oligonucleotides possessing terminal cationic linkages.\u201d 28(10): 2153-7.","Ding, D., S. M. Grayaznov, et al. (1996). \u201cAn oligodeoxyribonucleotide N3\u2032-->P5\u2032 phosphoramidate duplex forms an A-type helix in solution.\u201d 24(2): 354-60.","Egholm, M., O. Buchardt, et al. (1993). \u201cPNA hybridizes to complementary oligonucleotides obeying the Watson-Crick hydrogen-bonding rules.\u201d 365(6446): 566-8.","Felgner, P. L., T. R. Gadek, et al. (1987). \u201cLipofection: a highly efficient, lipid-mediated DNA-transfection procedure.\u201d 84(21): 7413-7.","Gait, M. J., A. S. Jones, et al. (1974). \u201cSynthetic-analogues of polynucleotides XII. Synthesis of thymidine derivatives containing an oxyacetamido- or an oxyformamido-linkage instead of a phosphodiester group.\u201d 1] 0(14): 1684-6.","Gee, J. E., I. Robbins, et al. (1998). \u201cAssessment of high-affinity hybridization, RNase H cleavage, and covalent linkage in translation arrest by antisense oligonucleotides.\u201d 8(2): 103-11.","Lesnikowski, Z. J., M. Jaworska, et al. (1990). \u201cOcta(thymidine methanephosphonates) of partially defined stereochemistry: synthesis and effect of chirality at phosphorus on binding to pentadecadeoxyriboadenylic acid.\u201d 18(8): 2109-15.","Mertes, M. P. and E. A. Coats (1969). \u201cSynthesis of carbonate analogs of dinucleosides. 3\u2032-Thymidinyl 5\u2032-thymidinyl carbonate, 3\u2032-thymidinyl 5\u2032-(5-fluoro-2\u2032-deoxyuridinyl) carbonate, and 3\u2032-(5-fluoro-2\u2032-deoxyuridinyl) 5\u2032-thymidinyl carbonate.\u201d 12(1): 154-7.","Moulton, H. M., M. H. Nelson, et al. (2004). \u201cCellular uptake of antisense morpholino oligomers conjugated to arginine-rich peptides.\u201d 15(2): 290-9.","Nelson, M. H., D. A. Stein, et al. (2005). \u201cArginine-rich peptide conjugation to morpholino oligomers: effects on antisense activity and specificity.\u201d 16(4): 959-66.","Strauss, J. H. and E. G. Strauss (2002). . San Diego, Academic Press.","Summerton, J. and D. Weller (1997). \u201cMorpholino antisense oligomers: design, preparation, and properties.\u201d 7(3): 187-95.","Toulme, J. J., R. L. Tinevez, et al. (1996). \u201cTargeting RNA structures by antisense oligonucleotides.\u201d 78(7): 663-73.","Williams, A. S., J. P. Camilleri, et al. (1996). \u201cA single intra-articular injection of liposomally conjugated methotrexate suppresses joint inflammation in rat antigen-induced arthritis.\u201d 35(8): 719-24.","Wu, G. Y. and C. H. Wu (1987). \u201cReceptor-mediated in vitro gene transformation by a soluble DNA carrier system.\u201d 262(10): 4429-32."]}},"Influenza viruses have been a major cause of human mortality and morbidity throughout recorded history. Influenza A virus infection causes millions of cases of severe illness and as many as 500,000 deaths each year worldwide. Epidemics vary widely in severity but occur at regular intervals and always cause significant mortality and morbidity, most frequently in the elderly population. Although vaccines against matched influenza strains can prevent illness in 60-80% of healthy adults, the rate of protection is much lower in high-risk groups. Furthermore, vaccination does not provide protection against unexpected strains, such as the H5 and H7 avian influenza outbreaks in Hong Kong in 1997 and Europe and Southeast Asia in 2003 and 2004. Current anti-influenza drugs are limited in their capacity to provide protection and therapeutic effect (Cox and Subbarao 1999; Cox and Subbarao 2000).","Influenza A is a segmented RNA virus of negative-polarity. Genome segments are replicated by a complex of 4 proteins: the 3 polymerase polypeptides (PA, PB1 and PB2) and NP (Nucleoprotein). The 5\u2032 and 3\u2032 terminal sequence regions of all 8 genome segments are highly conserved within a genotype (Strauss and Strauss 2002).","Influenza A viruses can be subtyped according to the antigenic and genetic nature of their surface glycoproteins; 15 hemagglutinin (HA) and 9 neuraminidase (NA) subtypes have been identified to date. Viruses bearing all known HA and NA subtypes have been isolated from avian hosts, but only viruses of the H1N1 (1918), H2N2 (1957\/58), and H3N2 (1968) subtypes have been associated with widespread epidemics in humans (Strauss and Strauss 2002).","Since 1997, when H5N1 influenza virus was transmitted to humans and killed 6 of 18 infected persons, there have been multiple transmissions of avian influenza viruses to mammals. Either the whole virus is transmitted directly or gene segments from the avian influenza virus are acquired by mammalian strains. Widespread infections of poultry with H5N1 viruses in Asia have caused increasing concern that this subtype may achieve human-to-human spread and establish interspecies transmission. The species which different types of influenza viruses are able to infect are determined by different forms of the virus glycoproteins (HA, NA). This provides a considerable species barrier between birds and humans which is not easily overcome. Pigs, however, provide a \u201cmixing pot\u201d-able to be infected by both types of virus and thereby allowing the passage of avian viruses to humans. When an individual pig cell is co-infected with both avian and human influenza viruses, recombinant forms can emerge that carry an avian HA genotype but readily infect humans. Avian HA can infect pigs, but not humans. In pigs, during genome segment packaging, it is possible to create a virus with several Avian segments and Human HA and\/or NA segments (Cox and Subbarao 2000).","Influenza viruses infect humans and animals (e.g., pigs, birds, horses) and may cause acute respiratory disease. There have been numerous attempts to produce vaccines effective against influenza virus. None, however, have been completely successful, particularly on a long-term basis. This may be due, at least in part, to the segmented characteristic of the influenza virus genome, which makes it possible, through re-assortment of the segments, for numerous forms to exist. For example, it has been suggested that there could be an interchange of RNA segments between animal and human influenza viruses, which would result in the introduction of new antigenic subtypes into both populations. Thus, a long-term vaccination approach has failed, due to the emergence of new subtypes (antigenic \u201cshift\u201d). In addition, the surface proteins of the virus, hemagglutinin and neuraminidase, constantly undergo minor antigenic changes (antigenic \u201cdrift\u201d). This high degree of variation explains why specific immunity developed against a particular influenza virus does not establish protection against new variants. Hence, alternative antiviral strategies are needed. Although influenza B and C viruses cause less clinical disease than the A types, new antiviral drugs should also be helpful in curbing infections caused by these agents.","Influenza viruses that occur naturally among birds are called avian influenza (bird flu). The birds carry the viruses in their intestines but do not generally get sick from the infection. However, migratory birds can carry the bird flu to infect domestic chickens, ducks and turkeys causing illness and even death. Avian flu does not easily infect humans but when human exposure is more frequent, such as contact with domestic birds, human infections occur. A dangerous bird flu (H5N1) was first identified in terns in South Africa in 1961 and was identified as a potentially deadly form of flu. Outbreaks of H5N1 occurred in eight Asian countries in late 2003 and 2004. At that time more than 100 million birds in these countries either died or were killed in order to control the outbreak. Beginning in June of 2004 new deadly outbreaks of H5N1 were reported in Asia which is currently ongoing. Human infections of H5N1 have been observed in Thailand, Vietnam and Cambodia with a death rate of about 50 percent. These infections have mostly occurred from human contact with infected poultry but a few cases of human-to-human spread of H5N1 have occurred.","Currently, there is no vaccine to protect humans against H5N1 but research efforts are underway. There are four currently approved influenza medications, amantadine, rimantadine, oseltamivir and zanamivir. Unfortunately, the H5N1 virus is resistant to both amantadine and rimantidine. The remaining oseltamivir and zanamivir may show some efficacy to H5N1 but need to be evaluated more extensively.","In view of the severity of the diseases caused by influenza viruses there is an immediate need for new therapies to treat influenza infection. Given the lack of effective prevention or therapies, it is therefore an object of the present invention to provide therapeutic compounds and methods for treating a host infected with an influenza virus.","The invention includes, in one aspect, an anti-viral compound effective in inhibiting replication within a host cell of an RNA virus having a single-stranded, negative sense genome and selected from the Orthomyxoviridae family including the Influenzavirus A, Influenzavirus B and Influenzavirus C genera. The compound targets viral RNA sequences within a region selected from the following: 1) the 5\u2032 or 3\u2032 terminal 25 bases of the negative sense viral RNA segments; 2) the terminal 25 bases of the 3\u2032 terminus of the positive sense cRNA and; 3) 50 bases surrounding the AUG start codons of influenza viral mRNAs.","The antiviral compound consists of an oligonucleotide analog characterized by: a) a nuclease-resistant backbone, b) 12-40 nucleotide bases, and c) a targeting sequence of at least 12 bases in length, that hybridizes to a target region selected from the following: i) the 5\u2032 or 3\u2032 terminal 25 bases of a negative sense viral RNA segment of Influenzavirus A, Influenzavirus B and Influenzavirus C, ii) the terminal 30 bases of the 3\u2032 terminus of a positive sense cRNA of Influenzavirus A, Influenzavirus B and Influenzavirus C, and iii) the 50 bases surrounding the AUG start codon of an influenza viral mRNA.","The oligonucleotide analog also has: a) the capability of being actively taken up by mammalian host cells, and b) the ability to form a heteroduplex structure with the viral target region, wherein said heteroduplex structure is: i) composed of the positive or negative sense strand of the virus and the oligonucleotide compound, and ii) characterized by a Tm of dissociation of at least 45\u00b0 C.","The invention includes, in another aspect, an antiviral compound that inhibits, in a mammalian host cell, replication of an infecting influenza virus having a single-stranded, segmented, negative-sense genome and selected from the Orthomyxoviridae family. The compound is administered to the infected host cells as an oligonucleotide analog characterized by the elements described above on pp. 5-6. The compound may be administered to a mammalian subject infected with the influenza virus, or at risk of infection with the influenza virus.","The compound may be composed of morpholino subunits linked by uncharged, phosphorus-containing intersubunit linkages, joining a morpholino nitrogen of one subunit to a 5\u2032 exocyclic carbon of an adjacent subunit. In one embodiment, the intersubunit linkages are phosphorodiamidate linkages, such as those having the structure:",{"@attributes":{"id":"p-0019","num":"0039"},"chemistry":{"@attributes":{"id":"CHEM-US-00001","num":"00001"},"img":{"@attributes":{"id":"EMI-C00001","he":"31.16mm","wi":"28.36mm","file":"US08357664-20130122-C00001.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{},"sub":["1","2"]},"The compound may be a covalent conjugate of an oligonucleotide analog moiety capable of forming such a heteroduplex structure with the positive or negative sense strand of the virus, and an arginine-rich polypeptide effective to enhance the uptake of the compound into host cells. Exemplary polypeptides have one of the sequences identified as SEQ ID NOs:25-30.","In a related aspect, the invention includes a heteroduplex complex formed between:\n\n","where said heteroduplex complex has a Tm of dissociation of at least 45\u00b0 C. and disruption of a stem-loop secondary structure.","An exemplary oligonucleotide analog is composed of morpholino subunits linked by uncharged, phosphorus-containing intersubunit linkages, joining a morpholino nitrogen of one subunit to a 5\u2032 exocyclic carbon of an adjacent subunit. The compound may have phosphorodiamidate linkages, such as in the structure",{"@attributes":{"id":"p-0024","num":"0051"},"chemistry":{"@attributes":{"id":"CHEM-US-00002","num":"00002"},"img":{"@attributes":{"id":"EMI-C00002","he":"31.16mm","wi":"28.36mm","file":"US08357664-20130122-C00002.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{},"sub":["1","2"]},"In still another aspect, the invention includes an oligonucleotide analog compound for use in inhibiting replication in mammalian host cells of an influenza virus having a single-stranded, segmented, negative-sense RNA genome and selected from the Orthomyxoviridae family. The compound is characterized by the elements described above on pp. 5-6.","An exemplary oligonucleotide analog is composed of morpholino subunits linked by uncharged, phosphorus-containing intersubunit linkages, joining a morpholino nitrogen of one subunit to a 5\u2032 exocyclic carbon of an adjacent subunit. The compound may have phosphorodiamidate linkages, such as in the structure",{"@attributes":{"id":"p-0027","num":"0054"},"chemistry":{"@attributes":{"id":"CHEM-US-00003","num":"00003"},"img":{"@attributes":{"id":"EMI-C00003","he":"31.16mm","wi":"28.36mm","file":"US08357664-20130122-C00003.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{},"sub":["1","2"]},"For treatment of Influenza A virus as given below, the targeting sequence hybridizes to a region associated with one of the group of sequences identified as SEQ ID NOs:1-9. Preferred targeting sequences are those complementary to either the minus strand target of SEQ ID NO:4 or the positive-strand target of SEQ ID NO:3. Exemplary antisense phosphorodiamidate morpholino oligomers (\u201cPMOs\u201d) that target these two regions are listed as SEQ ID NOs:12 and 13, respectively.","I. Definitions","The terms below, as used herein, have the following meanings, unless indicated otherwise:","\u201cAlkyl\u201d refers to a fully saturated monovalent radical containing carbon and hydrogen, which may be branched, linear, or cyclic (cycloalkyl). Examples of alkyl groups are methyl, ethyl, n-butyl, t-butyl, n-heptyl, isopropyl, cyclopropyl, cyclopentyl, ethylcyclopentyl, and cyclohexyl. Generally preferred are alkyl groups having one to six carbon atoms, referred to as \u201clower alkyl\u201d, and exemplified by methyl, ethyl, n-butyl, i-butyl, t-butyl, isoamyl, n-pentyl, and isopentyl. In one embodiment, lower alkyl refers to Cto Calkyl.","\u201cAlkenyl\u201d refers to an unsaturated monovalent radical containing carbon and hydrogen, which may be branched, linear, or cyclic. The alkenyl group may be monounsaturated or polyunsaturated. Generally preferred are alkenyl groups having one to six carbon atoms, referred to as \u201clower alkenyl\u201d.","\u201cAryl\u201d refers to a substituted or unsubstituted monovalent aromatic radical, generally having a single ring (e.g., benzene) or two condensed rings (e.g., naphthyl). This term includes heteroaryl groups, which are aromatic ring groups having one or more nitrogen, oxygen, or sulfur atoms in the ring, such as furyl, pyrrole, pyridyl, and indole. By \u201csubstituted\u201d is meant that one or more ring hydrogens in the aryl group is replaced with a halide such as fluorine, chlorine, or bromine; with a lower alkyl group containing one or two carbon atoms; nitro, amino, methylamino, dimethylamino, methoxy, halomethoxy, halomethyl, or haloethyl. Preferred substituents include halogen, methyl, ethyl, and methoxy. Generally preferred are aryl groups having a single ring.","\u201cAralkyl\u201d refers to an alkyl, preferably lower (C-C, more preferably C-C) alkyl, substituent which is further substituted with an aryl group; examples are benzyl (\u2014CHCH) and phenethyl (\u2014CHCHCH).","\u201cHeterocycle\u201d refers to a non-aromatic ring, preferably a 5- to 7-membered ring, whose ring atoms are selected from the group consisting of carbon, nitrogen, oxygen and sulfur. Preferably, the ring atoms include 3 to 6 carbon atoms. Such heterocycles include, for example, pyrrolidine, piperidine, piperazine, and morpholine.","The term \u201csubstituted\u201d, with respect to an alkyl, alkenyl, alkynyl, aryl, aralkyl, or alkaryl group, refers to replacement of a hydrogen atom with a heteroatom-containing substituent, such as, for example, halogen, hydroxy, alkoxy, thiol, alkylthio, amino, alkylamino, imino, oxo (keto), nitro, cyano, or various acids or esters such as carboxylic, sulfonic, or phosphonic.","The terms \u201coligonucleotide analog\u201d refers to an oligonucleotide having (i) a modified backbone structure, e.g., a backbone other than the standard phosphodiester linkage found in natural oligo- and polynucleotides, and (ii) optionally, modified sugar moieties, e.g., morpholino moieties rather than ribose or deoxyribose moieties. The analog supports bases capable of hydrogen bonding by Watson-Crick base pairing to standard polynucleotide bases, where the analog backbone presents the bases in a manner to permit such hydrogen bonding in a sequence-specific fashion between the oligonucleotide analog molecule and bases in a standard polynucleotide (e.g., single-stranded RNA or single-stranded DNA). Preferred analogs are those having a substantially uncharged, phosphorus containing backbone.","A substantially uncharged, phosphorus containing backbone in an oligonucleotide analog is one in which a majority of the subunit linkages, e.g., between 60-100%, are uncharged at physiological pH, and contain a single phosphorous atom. The analog contains between 8 and 40 subunits, typically about 8-25 subunits, and preferably about 12 to 25 subunits. The analog may have exact sequence complementarity to the target sequence or near complementarity, as defined below.","A \u201csubunit\u201d of an oligonucleotide analog refers to one nucleotide (or nucleotide analog) unit of the analog. The term may refer to the nucleotide unit with or without the attached intersubunit linkage, although, when referring to a \u201ccharged subunit\u201d, the charge typically resides within the intersubunit linkage (e.g. a phosphate or phosphorothioate linkage).","A \u201cmorpholino oligonucleotide analog\u201d is an oligonucleotide analog composed of morpholino subunit structures of the form shown in , where (i) the structures are linked together by phosphorus-containing linkages, one to three atoms long, joining the morpholino nitrogen of one subunit to the 5\u2032 exocyclic carbon of an adjacent subunit, and (ii) Pand Pare purine or pyrimidine base-pairing moieties effective to bind, by base-specific hydrogen bonding, to a base in a polynucleotide. The purine or pyrimidine base-pairing moiety is typically adenine, cytosine, guanine, uracil or thymine. The synthesis, structures, and binding characteristics of morpholino oligomers are detailed in U.S. Pat. Nos. 5,698,685, 5,217,866, 5,142,047, 5,034,506, 5,166,315, 5,521,063, and 5,506,337, all of which are incorporated herein by reference.","The subunit and linkage shown in  are used for six-atom repeating-unit backbones, as shown in  (where the six atoms include: a morpholino nitrogen, the connected phosphorus atom, the atom (usually oxygen) linking the phosphorus atom to the 5\u2032 exocyclic carbon, the 5\u2032 exocyclic carbon, and two carbon atoms of the next morpholino ring). In these structures, the atom Ylinking the 5\u2032 exocyclic morpholino carbon to the phosphorus group may be sulfur, nitrogen, carbon or, preferably, oxygen. The X moiety pendant from the phosphorus is any stable group which does not interfere with base-specific hydrogen bonding. Preferred X groups include fluoro, alkyl, alkoxy, thioalkoxy, and alkyl amino, including cyclic amines, all of which can be variously substituted, as long as base-specific bonding is not disrupted. Alkyl, alkoxy and thioalkoxy preferably include 1-6 carbon atoms. Alkyl amino preferably refers to lower alkyl (Cto C) substitution, and cyclic amines are preferably 5- to 7-membered nitrogen heterocycles optionally containing 1-2 additional heteroatoms selected from oxygen, nitrogen, and sulfur. Z is sulfur or oxygen, and is preferably oxygen.","A preferred morpholino oligomer is a phosphorodiamidate-linked morpholino oligomer, referred to herein as a PMO. Such oligomers are composed of morpholino subunit structures such as shown in , where X\u2550NH, NHR, or NR(where R is lower alkyl, preferably methyl), Y\u2550O, and Z=O, and Pand Pare purine or pyrimidine base-pairing moieties effective to bind, by base-specific hydrogen bonding, to a base in a polynucleotide. Also preferred are structures having an alternate phosphorodiamidate linkage, where, in , X=lower alkoxy, such as methoxy or ethoxy, Y\u2550NH or NR, where R is lower alkyl, and Z=O.","The term \u201csubstituted\u201d, particularly with respect to an alkyl, alkoxy, thioalkoxy, or alkylamino group, refers to replacement of a hydrogen atom on carbon with a heteroatom-containing substituent, such as, for example, halogen, hydroxy, alkoxy, thiol, alkylthio, amino, alkylamino, imino, oxo (keto), nitro, cyano, or various acids or esters such as carboxylic, sulfonic, or phosphonic. It may also refer to replacement of a hydrogen atom on a heteroatom (such as an amine hydrogen) with an alkyl, carbonyl or other carbon containing group.","As used herein, the term \u201ctarget\u201d, relative to the viral genomic RNA, refers to a viral genomic RNA, and specifically, to a region associated with stem-loop secondary structure within the 5\u2032-terminal end 40 bases of the positive-sense RNA strand of a single-stranded RNA (ssRNA) virus described herein.","The term \u201ctarget sequence\u201d refers to a portion of the target RNA against which the oligonucleotide analog is directed, that is, the sequence to which the oligonucleotide analog will hybridize by Watson-Crick base pairing of a complementary sequence. As will be seen, the target sequence may be a contiguous region of the viral positive-strand RNA, or may be composed of complementary fragments of both the 5\u2032 and 3\u2032 sequences involved in secondary structure.","The term \u201ctargeting sequence\u201d is the sequence in the oligonucleotide analog that is complementary (meaning, in addition, substantially complementary) to the target sequence in the RNA genome. The entire sequence, or only a portion, of the analog compound may be complementary to the target sequence. For example, in an analog having 20 bases, only 12-14 may be targeting sequences. Typically, the targeting sequence is formed of contiguous bases in the analog, but may alternatively be formed of non-contiguous sequences that when placed together, e.g., from opposite ends of the analog, constitute sequence that spans the target sequence. As will be seen, the target and targeting sequences are selected such that binding of the analog is to a region within; 1) the 5\u2032 or 3\u2032 terminal 25 bases of the negative sense viral RNA; 2) the terminal 25 bases of the 3\u2032 terminus of the positive sense mRNA and\/or; 3) 50 bases surrounding the AUG start codons of viral mRNA.","Target and targeting sequences are described as \u201ccomplementary\u201d to one another when hybridization occurs in an antiparallel configuration. A targeting sequence may have \u201cnear\u201d or \u201csubstantial\u201d complementarity to the target sequence and still function for the purpose of the present invention, that is, it may still be \u201ccomplementary.\u201d Preferably, the oligonucleotide analog compounds employed in the present invention have at most one mismatch with the target sequence out of 10 nucleotides, and preferably at most one mismatch out of 20. Alternatively, the antisense oligomers employed have at least 90% sequence homology, and preferably at least 95% sequence homology, with the exemplary targeting sequences as designated herein.","An oligonucleotide analog \u201cspecifically hybridizes\u201d to a target polynucleotide if the oligomer hybridizes to the target under physiological conditions, with a Tm substantially greater than 45\u00b0 C., preferably at least 50\u00b0 C., and typically 60\u00b0 C.-80\u00b0 C. or higher. Such hybridization preferably corresponds to stringent hybridization conditions. At a given ionic strength and pH, the Tm is the temperature at which 50% of a target sequence hybridizes to a complementary polynucleotide. Again, such hybridization may occur with \u201cnear\u201d or \u201csubstantial\u201d complementary of the antisense oligomer to the target sequence, as well as with exact complementarity.","A \u201cnuclease-resistant\u201d oligomeric molecule (oligomer) refers to one whose backbone is substantially resistant to nuclease cleavage, in non-hybridized or hybridized form; by common extracellular and intracellular nucleases in the body; that is, the oligomer shows little or no nuclease cleavage under normal nuclease conditions in the body to which the oligomer is exposed.","A \u201cheteroduplex\u201d refers to a duplex between an oligonculeotide analog and the complementary portion of a target RNA. A \u201cnuclease-resistant heteroduplex\u201d refers to a heteroduplex formed by the binding of an antisense oligomer to its complementary target, such that the heteroduplex is substantially resistant to in vivo degradation by intracellular and extracellular nucleases, such as RNAseH, which are capable of cutting double-stranded RNA\/RNA or RNA\/DNA complexes.","A \u201cbase-specific intracellular binding event involving a target RNA\u201d refers to the specific binding of an oligonucleotide analog to a target RNA sequence inside a cell. The base specificity of such binding is sequence specific. For example, a single-stranded polynucleotide can specifically bind to a single-stranded polynucleotide that is complementary in sequence.","An \u201ceffective amount\u201d of an antisense oligomer, targeted against an infecting influenza virus, is an amount effective to reduce the rate of replication of the infecting virus, and\/or viral load, and\/or symptoms associated with the viral infection.","As used herein, the term \u201cbody fluid\u201d encompasses a variety of sample types obtained from a subject including, urine, saliva, plasma, blood, spinal fluid, or other sample of biological origin, such as skin cells or dermal debris, and may refer to cells or cell fragments suspended therein, or the liquid medium and its solutes.","The term \u201crelative amount\u201d is used where a comparison is made between a test measurement and a control measurement. The relative amount of a reagent forming a complex in a reaction is the amount reacting with a test specimen, compared with the amount reacting with a control specimen. The control specimen may be run separately in the same assay, or it may be part of the same sample (for example, normal tissue surrounding a malignant area in a tissue section).","\u201cTreatment\u201d of an individual or a cell is any type of intervention provided as a means to alter the natural course of the individual or cell. Treatment includes, but is not limited to, administration of e.g., a pharmaceutical composition, and may be performed either prophylactically, or subsequent to the initiation of a pathologic event or contact with an etiologic agent. The related term \u201cimproved therapeutic outcome\u201d relative to a patient diagnosed as infected with a particular virus, refers to a slowing or diminution in the growth of virus, or viral load, or detectable symptoms associated with infection by that particular virus.","An agent is \u201cactively taken up by mammalian cells\u201d when the agent can enter the cell by a mechanism other than passive diffusion across the cell membrane. The agent may be transported, for example, by \u201cactive transport\u201d, referring to transport of agents across a mammalian cell membrane by e.g. an ATP-dependent transport mechanism, or by \u201cfacilitated transport\u201d, referring to transport of antisense agents across the cell membrane by a transport mechanism that requires binding of the agent to a transport protein, which then facilitates passage of the bound agent across the membrane. For both active and facilitated transport, the oligonucleotide analog preferably has a substantially uncharged backbone, as defined below. Alternatively, the antisense compound may be formulated in a complexed form, such as an agent having an anionic backbone complexed with cationic lipids or liposomes, which can be taken into cells by an endocytotic mechanism. The analog may also be conjugated, e.g., at its 5\u2032 or 3\u2032 end, to an arginine-rich peptide, such as a portion of the HIV TAT protein, polyarginine, or to combinations of arginine and other amino acids including the non-natural amino acids 6-aminohexanoic acid (Ahx) and beta-alanine (\u03b2Ala). Exemplary arginine-rich delivery peptides are listed as SEQ ID NOs:25-30. These exemplary arginine-rich delivery peptides facilitate transport into the target host cell as described (Moulton, Nelson et al. 2004; Nelson, Stein et al. 2005).","Rules for the selection of targeting sequences capable of inhibiting replication of the influenza viral genome are discussed below.","II. Targeted Viruses","The present invention is based on the discovery that effective inhibition of single-stranded, segmented, negative-sense RNA viruses can be achieved by exposing cells infected with influenza virus to antisense oligonucleotide analog compounds (i) that target 1) the 5\u2032 or 3\u2032 terminal 25 bases of the negative sense viral RNA; 2) the terminal 25 bases of the 3\u2032 terminus of the positive sense mRNA and\/or; 3) 50 bases surrounding the AUG start codons of viral mRNA; and (ii) having physical and pharmacokinetic features which allow effective interaction between the antisense compound and the virus within host cells. In one aspect, the oligomers can be used in treating a mammalian subject infected with influenza virus.","The invention targets RNA viruses having genomes that are: (i) single stranded, (ii) segmented and (iii) negative polarity. The targeted viruses also synthesize two different versions of a genomic complement of the negative sense virion RNA (vRNA) with positive polarity: 1) cRNA that is used as a template for replication of negative sense virion RNA, and 2) a complementary positive sense RNA (mRNA) that is used for translation of viral proteins.  is a schematic that shows these different RNA species and the target location of antisense PMO described in the present invention. In particular, targeted viral families include members of the Orthomyxoviridae family including the Influenzavirus A, Influenzavirus B and Influenzavirus C genera. Various physical, morphological, and biological characteristics of members of the Orthomyxoviridae family can be found, for example, in Textbook of Human Virology, R. Belshe, ed., 2Edition, Mosby, 1991, at the Universal Virus Database of the International Committee on Taxonomy of Viruses (www.ncbi.nlm.nih.gov\/ICTVdb\/index.htm) and in human virology textbooks (see, for example (Strauss and Strauss 2002). Some of the key biological characteristics of the Orthomxyoviridae family of viruses are described below.","Influenza Viruses","Influenza A, influenza B and influenza C viruses are the only members of the Influenzavirus A, Influenzavirus B and Influenzavirus C genera, respectively. These viruses are membrane-enclosed viruses whose genomes are segmented negative-sense (i.e. minus) strands of RNA ((\u2212)RNA). The ten influenza virus genes are present on eight segments of the single-stranded RNA of strains A and B, and on seven segments of strain C. The segments vary in size (from 890 to 2341 nucleotides in length) and each is a template for synthesis of different mRNAs. The influenza virus virion contains virus-specific RNA polymerases necessary for mRNA synthesis from these templates and, in the absence of such specific polymerases, the minus strand of influenza virus RNA is not infectious. Initiation of transcription of the mRNAs occurs when the influenza virus mRNA polymerase takes 12 to 15 nucleotides from the 5\u2032 end of a cellular mRNA or mRNA precursor and uses the borrowed oligonucleotide as a primer. This process has been termed \u201ccap-snatching\u201d because it places a 5\u2032 cap structure on the viral mRNA. Generally, the mRNAs made through this process encode only one protein. The M gene and NS gene viral RNA segments also code for spliced mRNAs, which results in production of two different proteins for each of these two segments.","Replication of influenza viral RNA occurs in the nucleus and involves the synthesis of three different species of RNA. A schematic of this process is shown in . After infection of a na\u00efve cell, the minus strand virion RNA (vRNA) is transported to the nucleus where RNA destined for translation (mRNA) is synthesized using 5\u2032-terminal 10-13 nucleotide primers cleaved by viral-encoded enzymes from capped cellular pre-mRNA molecules (i.e. cap-snatching). Synthesis of each mRNA continues to near the end of the genome segment where an oligo(U) stretch is encountered and a poly(A) tail is added. The dedicated viral mRNAs are transported to the cytoplasm for translation and after sufficient viral proteins are transported back into the nucleus, synthesis of vRNA destined for nascent virions is initiated. An exact antigenomic copy of vRNA is synthesized (termed cRNA) which is a perfect complement of the genomic vRNA and serves as a template for production of new vRNA. The different RNAs synthesized during influenza virus replication are shown schematically in .","GenBank references for exemplary viral nucleic acid target sequences representing influenza A genomic segments are listed in Table 1 below. The nucleotide sequence numbers in Table 1 are derived from the Genbank reference for the positive-strand RNA. It will be appreciated that these sequences are only illustrative of other sequences in the Orthomyxoviridae family, as may be available from available gene-sequence databases of literature or patent resources. The sequences below, identified as SEQ ID NOs:1-9, are also listed in the Sequence Listing at the end of the specification.","The target sequences in Table 1 represent; 1) the 5\u2032 or 3\u2032 terminal 25 bases of the negative sense viral RNA (SEQ ID NOs:4-9); 2) the terminal 25 bases of the 3\u2032 terminus of the positive sense mRNA (SEQ ID NOs:4-9) and; 3) 50 bases surrounding the AUG start codons of the indicated influenza virus genes (SEQ ID NOs:1-3). The sequences shown are the positive-strand (i.e., antigenomic or mRNA) sequence in the 5\u2032 to 3\u2032 orientation. It will be obvious that when the target is the minus-strand vRNA the targeted sequence is the complement of the sequence listed in Table 1.","Table 1 lists the targets for three different influenza A viral genes, PB2, PB1 and nucleoprotein (NP), encoded by genomic segments 1, 2 and 5, respectively. The PB1, PB2 and NP proteins are components of the viral RNA polymerase and PB2 also functions as the \u201ccap-snatching\u201d enzyme. The target sequences for the AUG start codons of the three genes are represented as SEQ ID NOs:1-3. The 3\u2032 terminal sequences of the three genomic segments are represented by SEQ ID NOs:4, 6 and 8 and can be targeted on both the positive strand and the negative strand of those segments. The 5\u2032 terminal sequences (SEQ ID NOs:5, 7 and 9) can be successfully targeted on the minus strand.",{"@attributes":{"id":"p-0074","num":"0101"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u20031"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Exemplary\u2003Influenza\u2003Viral\u2003Nucleic\u2003Acid"},{"entry":"Target\u2003Sequences"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"14pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"SEQ"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"14pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"GenBank","Nucleotide","Sequence","ID"]},{"entry":["Name","No.","Region","(5\u2032 to\u20033\u2032)","NO"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"14pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["NP-31","J02147","21-70","UCACUCACUGAGUGACAUCAA","1"]},{"entry":[{},{},{},"AAUCGCGUCCCAAGGCAC",{}]},{"entry":[{},{},{},"CAAACGGU",{}]},{"entry":{}},{"entry":["PB2-11","V00603","\u20031-50","AGCGAAAGCAGGUCAAUUAUA","2"]},{"entry":[{},{},{},"UUCAAUGAAAGAAUAAAA",{}]},{"entry":[{},{},{},"GAACUAAG",{}]},{"entry":{}},{"entry":["PB1-","J02151","\u20031-50","AGCGAAAGCAGGCAAACCAUU","3"]},{"entry":["AUG",{},{},"UGAGAUGUCAAUCCGACC",{}]},{"entry":[{},{},{},"UUACUUUU",{}]},{"entry":{}},{"entry":["NP-","J02147","1541-1565","AAAGAAAAAUACCCUUGUUUC","4"]},{"entry":["3\u2032term",{},{},"UACU",{}]},{"entry":{}},{"entry":["NP-","J02147","\u20031-25","AGCAAAAGCAGGGUAGAUAAU","5"]},{"entry":["5\u2032term",{},{},"CACU",{}]},{"entry":{}},{"entry":["PB1-","J02151","2317-2341","CAUGAAAAAAUGCCUUGUUCC","6"]},{"entry":["3\u2032term",{},{},"UACU",{}]},{"entry":{}},{"entry":["PB1-","J02151","\u20031-25","AGCGAAAGCAGGCAAACCAUU","7"]},{"entry":["5\u2032term",{},{},"UGAA",{}]},{"entry":{}},{"entry":["PB2-","V00603","2317-2341","GUUUAAAAACGACCUUGUUUC","8"]},{"entry":["3\u2032term",{},{},"UACU",{}]},{"entry":{}},{"entry":["PB2-","V00603","\u20031-25","AGCGAAAGCAGGUCAAUUAUA","9"]},{"entry":["5\u2032term",{},{},"UUCA"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0075","num":"0102"},"figref":"FIG. 4","i":"Options for the Control of Influenza IV"},"Targeting sequences are designed to hybridize to a region of the target sequence as listed in Table 1. Selected targeting sequences can be made shorter, e.g., 12 bases, or longer, e.g., 40 bases, and include a small number of mismatches, as long as the sequence is sufficiently complementary to disrupt the stem structure(s) upon hybridization with the target, and forms with the virus positive-strand, a heteroduplex having a Tm of 45\u00b0 C. or greater.","More generally, the degree of complementarity between the target and targeting sequence is sufficient to form a stable duplex. The region of complementarity of the antisense oligomers with the target RNA sequence may be as short as 8-11 bases, but is preferably 12-15 bases or more, e.g. 12-20 bases, or 12-25 bases. An antisense oligomer of about 14-15 bases is generally long enough to have a unique complementary sequence in the viral genome. In addition, a minimum length of complementary bases may be required to achieve the requisite binding Tm, as discussed below.","Oligomers as long as 40 bases may be suitable, where at least the minimum number of bases, e.g., 8-11, preferably 12-15 bases, are complementary to the target sequence. In general, however, facilitated or active uptake in cells is optimized at oligomer lengths less than about 30, preferably less than 25, and more preferably 20 or fewer bases. For PMO oligomers, described further below, an optimum balance of binding stability and uptake generally occurs at lengths of 14-22 bases.","The oligomer may be 100% complementary to the viral nucleic acid target sequence, or it may include mismatches, e.g., to accommodate variants, as long as a heteroduplex formed between the oligomer and viral nucleic acid target sequence is sufficiently stable to withstand the action of cellular nucleases and other modes of degradation which may occur in vivo. Oligomer backbones which are less susceptible to cleavage by nucleases are discussed below. Mismatches, if present, are less destabilizing toward the end regions of the hybrid duplex than in the middle. The number of mismatches allowed will depend on the length of the oligomer, the percentage of G:C base pairs in the duplex, and the position of the mismatch(es) in the duplex, according to well understood principles of duplex stability. Although such an antisense oligomer is not necessarily 100% complementary to the viral nucleic acid target sequence, it is effective to stably and specifically bind to the target sequence, such that a biological activity of the nucleic acid target, e.g., expression of viral protein(s), is modulated.","The stability of the duplex formed between the oligomer and the target sequence is a function of the binding Tm and the susceptibility of the duplex to cellular enzymatic cleavage. The Tm of an antisense compound with respect to complementary-sequence RNA may be measured by conventional methods, such as those described by Hames et al., Nucleic Acid Hybridization, IRL Press, 1985, pp. 107-108 or as described in Miyada C. G. and Wallace R. B., 1987, Oligonucleotide hybridization techniques, . Vol. 154 pp. 94-107. Each antisense oligomer should have a binding Tm, with respect to a complementary-sequence RNA, of greater than body temperature and preferably greater than 50\u00b0 C. Tm's in the range 60-80\u00b0 C. or greater are preferred. According to well known principles, the Tm of an oligomer compound, with respect to a complementary-based RNA hybrid, can be increased by increasing the ratio of C:G paired bases in the duplex, and\/or by increasing the length (in base pairs) of the heteroduplex. At the same time, for purposes of optimizing cellular uptake, it may be advantageous to limit the size of the oligomer. For this reason, compounds that show high Tm (50\u00b0 C. or greater) at a length of 20 bases or less are generally preferred over those requiring greater than 20 bases for high Tm values.","Table 2 below shows exemplary targeting sequences, in a 5\u2032-to-3\u2032 orientation, that are complementary to influenza A virus. The sequences listed provide a collection of targeting sequences from which targeting sequences may be selected, according to the general class rules discussed above. SEQ ID NOs:10-12, 15, 17, 20, 23 and 24 are antisense to the positive strand (mRNA or cRNA) of the virus whereas SEQ ID NOs:13, 14, 16, 18, 19, 21 and 22 are antisense to the minus strand (vRNA). Thus, for example, in selecting a target against the 3\u2032 terminus of the minus strand of the NP encoding segment (segment 5 of influenza A) SEQ ID NOs:13 or 16, or a portion of either sequence effective to block the function of the 3\u2032 terminus of the minus strand can be selected.",{"@attributes":{"id":"p-0082","num":"0109"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u20032"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Exemplary\u2003Antisense\u2003Oligomer\u2003Sequences"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"GenBank","Targeting","SEQ."]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Target","Acc.","Antisense\u2003Oligomer","ID"]},{"entry":["PMO","Nucleotides","No.","(5\u2032 to\u20033\u2032)","NO."]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["NP-AUG","39-58","J02147","CTTGGGACGCCATGATTTTG","10"]},{"entry":{}},{"entry":["PB2-AUG","24-43","V00603","CTTTTATTCTTTCCATATTG","11"]},{"entry":{}},{"entry":["PB1-AUG","13-33","J02151","GACATCCATTCAAATGGTTTG","12"]},{"entry":{}},{"entry":["(\u2212)NP-3\u2032trm","\u20031-22","J02147","AGCAAAAGCAGGGTAGATAATC","13"]},{"entry":{}},{"entry":["(\u2212)NP-5\u2032trm","1544-1565","J02147","GAAAAATACCCTTGTTTCTACT","14"]},{"entry":{}},{"entry":["(+)NP-3\u2032trm","1544-1565","J02147","AGTAGAAACAAGGGTATTTTTC","15"]},{"entry":{}},{"entry":["Flu(\u2212)3\u2032trm","\u20031-12","J02147","AGCAAAAGCAGG","16"]},{"entry":{}},{"entry":["Flu(+)3\u2032trm","1553-1565","J02147","AGTAGAAACAAGG","17"]},{"entry":{}},{"entry":["(\u2212)PB1-3\u2032trm","\u20031-20","J02151","AGCGAAAGCAGGCAAACCAT","18"]},{"entry":{}},{"entry":["(\u2212)PB1-5\u2032trm","2320-2341","J02151","GAAAAAATGCCTTGTTCCTACT","19"]},{"entry":{}},{"entry":["(+)PB1-3\u2032trm","2320-2341","J02151","AGTAGGAACAAGGCATTTTTTC","20"]},{"entry":{}},{"entry":["(\u2212)PB2-3\u2032trm","\u20031-20","V00603","AGCGAAAGCAGGTCAATTAT","21"]},{"entry":{}},{"entry":["(\u2212)PB2-5\u2032trm","2320-2341","V00603","TAAAAACGACCTTGTTTCTACT","22"]},{"entry":{}},{"entry":["(+)PB2-3\u2032trm","2320-2341","V00603","AGTAGAAACAAGGTCGTTTTTA","23"]},{"entry":{}},{"entry":["(+)NP-5\u2032trm","\u20031-20","J02147","AGTCTCGACTTGCTACCTCA","24"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}},"br":{}},"A. Properties","As detailed above, the antisense oligonucleotide analog compound (the term \u201cantisense\u201d indicates that the compound is targeted against either the virus' positive-sense strand RNA or negative-sense or minus-strand) has a base sequence targeting a region that includes one or more of the following; 1) the 5\u2032 or 3\u2032 terminal 30 bases of the negative sense viral RNA; 2) the terminal 30 bases of the 3\u2032 terminus of the positive sense mRNA and\/or; 3) 50 bases surrounding the AUG start codons of viral mRNA. In addition, the oligomer is able to effectively target infecting viruses, when administered to a host cell, e.g. in an infected mammalian subject. This requirement is met when the oligomer compound (a) has the ability to be actively taken up by mammalian cells, and (b) once taken up, form a duplex with the target RNA with a Tm greater than about 45\u00b0 C.","As will be described below, the ability to be taken up by cells requires that the oligomer backbone be substantially uncharged, and, preferably, that the oligomer structure is recognized as a substrate for active or facilitated transport across the cell membrane. The ability of the oligomer to form a stable duplex with the target RNA will also depend on the oligomer backbone, as well as factors noted above, the length and degree of complementarity of the antisense oligomer with respect to the target, the ratio of G: C to A:T base matches, and the positions of any mismatched bases. The ability of the antisense oligomer to resist cellular nucleases promotes survival and ultimate delivery of the agent to the cell cytoplasm.","Below are disclosed methods for testing any given, substantially uncharged backbone for its ability to meet these requirements.","B. Active or Facilitated Uptake by Cells","The antisense compound may be taken up by host cells by facilitated or active transport across the host cell membrane if administered in free (non-complexed) form, or by an endocytotic mechanism if administered in complexed form.","In the case where the agent is administered in free form, the antisense compound should be substantially uncharged, meaning that a majority of its intersubunit linkages are uncharged at physiological pH. Experiments carried out in support of the invention indicate that a small number of net charges, e.g., 1-2 for a 15- to 20-mer oligomer, can in fact enhance cellular uptake of certain oligomers with substantially uncharged backbones. The charges may be carried on the oligomer itself, e.g., in the backbone linkages, or may be terminal charged-group appendages. Preferably, the number of charged linkages is no more than one charged linkage per four uncharged linkages. More preferably, the number is no more than one charged linkage per ten, or no more than one per twenty, uncharged linkages. In one embodiment, the oligomer is fully uncharged.","An oligomer may also contain both negatively and positively charged backbone linkages, as long as opposing charges are present in approximately equal number. Preferably, the oligomer does not include runs of more than 3-5 consecutive subunits of either charge. For example, the oligomer may have a given number of anionic linkages, e.g. phosphorothioate or N3\u2192P5\u2032 phosphoramidate linkages, and a comparable number of cationic linkages, such as N,N-diethylenediamine phosphoramidates (Dagle, Littig et al. 2000). The net charge is preferably neutral or at most 1-2 net charges per oligomer.","In addition to being substantially or fully uncharged, the antisense agent is preferably a substrate for a membrane transporter system (i.e. a membrane protein or proteins) capable of facilitating transport or actively transporting the oligomer across the cell membrane. This feature may be determined by one of a number of tests for oligomer interaction or cell uptake, as follows.","A first test assesses binding at cell surface receptors, by examining the ability of an oligomer compound to displace or be displaced by a selected charged oligomer, e.g., a phosphorothioate oligomer, on a cell surface. The cells are incubated with a given quantity of test oligomer, which is typically fluorescently labeled, at a final oligomer concentration of between about 10-300 nM. Shortly thereafter, e.g., 10-30 minutes (before significant internalization of the test oligomer can occur), the displacing compound is added, in incrementally increasing concentrations. If the test compound is able to bind to a cell surface receptor, the displacing compound will be observed to displace the test compound. If the displacing compound is shown to produce 50% displacement at a concentration of 10\u00d7 the test compound concentration or less, the test compound is considered to bind at the same recognition site for the cell transport system as the displacing compound.","A second test measures cell transport, by examining the ability of the test compound to transport a labeled reporter, e.g., a fluorescence reporter, into cells. The cells are incubated in the presence of labeled test compound, added at a final concentration between about 10-300 nM. After incubation for 30-120 minutes, the cells are examined, e.g., by microscopy, for intracellular label. The presence of significant intracellular label is evidence that the test compound is transported by facilitated or active transport.","The antisense compound may also be administered in complexed form, where the complexing agent is typically a polymer, e.g., a cationic lipid, polypeptide, or non-biological cationic polymer, having an opposite charge to any net charge on the antisense compound. Methods of forming complexes, including bilayer complexes, between anionic oligonucleotides and cationic lipid or other polymer components, are well known. For example, the liposomal composition Lipofectin\u00ae (Felgner, Gadek et al. 1987), containing the cationic lipid DOTMA (N-[1-(2,3-dioleyloxy)propyl]-N,N,N-trimethylammonium chloride) and the neutral phospholipid DOPE (dioleyl phosphatidyl ethanolamine), is widely used. After administration, the complex is taken up by cells through an endocytotic mechanism, typically involving particle encapsulation in endosomal bodies.","The antisense compound may also be administered in conjugated form with an arginine-rich peptide linked covalently to the 5\u2032 or 3\u2032 end of the antisense oligomer. The peptide is typically 8-16 amino acids and consists of a mixture of arginine, and other amino acids including phenyalanine and cysteine. The use of arginine-rich peptide-PMO conjugates can be used to enhance cellular uptake of the antisense oligomer (See, e.g. (Moulton, Nelson et al. 2004; Nelson, Stein et al. 2005). Exemplary arginine-rich peptides for use in practicing the invention are listed as SEQ ID NOs:25-30. Non-natural amino acids can be used in combination with naturally occuring amino acids as shown in the Sequence listing table for SEQ ID NOs:26-30. In these examples 6-aminohexanoic acid (Ahx) and\/or beta-alanine (\u03b2-Ala) are used.","In some instances, liposomes may be employed to facilitate uptake of the antisense oligonucleotide into cells. (See, e.g., Williams, S. A., Leukemia 10(12):1980-1989, 1996; Lappalainen et al., Antiviral Res. 23:119, 1994; Uhlmann et al., antisense oligonucleotides: a new therapeutic principle, Chemical Reviews, Volume 90, No. 4, pages 544-584, 1990; Gregoriadis, G., Chapter 14, Liposomes, Drug Carriers in Biology and Medicine, pp. 287-341, Academic Press, 1979). Hydrogels may also be used as vehicles for antisense oligomer administration, for example, as described in WO 93\/01286. Alternatively, the oligonucleotides may be administered in microspheres or microparticles. (See, e.g. Wu and Wu 1987). Alternatively, the use of gas-filled microbubbles complexed with the antisense oligomers can enhance delivery to target tissues, as described in U.S. Pat. No. 6,245,747.","Alternatively, and according to another aspect of the invention, the requisite properties of oligomers with any given backbone can be confirmed by a simple in vivo test, in which a labeled compound is administered to an animal, and a body fluid sample, taken from the animal several hours after the oligomer is administered, assayed for the presence of heteroduplex with target RNA. This method is detailed in subsection D below.","C. Substantial Resistance to RNaseH","Two general mechanisms have been proposed to account for inhibition of expression by antisense oligonucleotides. (See e.g., Agrawal, Mayrand et al. 1990; Bonham, Brown et al. 1995; Boudvillain, Guerin et al. 1997). In the first, a heteroduplex formed between the oligonucleotide and the viral RNA acts as a substrate for RNaseH, leading to cleavage of the viral RNA. Oligonucleotides belonging, or proposed to belong, to this class include phosphorothioates, phosphotriesters, and phosphodiesters (unmodified \u201cnatural\u201d oligonucleotides). Such compounds expose the viral RNA in an oligomer:RNA duplex structure to hydrolysis by RNaseH, and therefore loss of function.","A second class of oligonucleotide analogs, termed \u201csteric blockers\u201d or, alternatively, \u201cRNaseH inactive\u201d or \u201cRNaseH resistant\u201d, have not been observed to act as a substrate for RNaseH, and are believed to act by sterically blocking target RNA nucleocytoplasmic transport, splicing or translation. This class includes methylphosphonates (Toulme, Tinevez et al. 1996), morpholino oligonucleotides, peptide nucleic acids (PNA's), certain 2\u2032-O-allyl or 2\u2032-O-alkyl modified oligonucleotides (Bonham, Brown et al. 1995), and N3\u2192P5\u2032 phosphoramidates (Ding, Grayaznov et al. 1996; Gee, Robbins et al. 1998).","A test oligomer can be assayed for its RNaseH resistance by forming an RNA: oligomer duplex with the test compound, then incubating the duplex with RNaseH under a standard assay conditions, as described in Stein et al. After exposure to RNaseH, the presence or absence of intact duplex can be monitored by gel electrophoresis or mass spectrometry.","D. In Vivo Uptake","In accordance with another aspect of the invention, there is provided a simple, rapid test for confirming that a given antisense oligomer type provides the required characteristics noted above, namely, high Tm, ability to be actively taken up by the host cells, and substantial resistance to RNaseH. This method is based on the discovery that a properly designed antisense compound will form a stable heteroduplex with the complementary portion of the viral RNA target when administered to a mammalian subject, and the heteroduplex subsequently appears in the urine (or other body fluid). Details of this method are also given in co-owned U.S. patent application Ser. No. 09\/736,920, entitled \u201cNon-Invasive Method for Detecting Target RNA\u201d (Non-Invasive Method), the disclosure of which is incorporated herein by reference.","Briefly, a test oligomer containing a backbone to be evaluated, having a base sequence targeted against a known RNA, is injected into a mammalian subject. The antisense oligomer may be directed against any intracellular RNA, including a host RNA or the RNA of an infecting virus. Several hours (typically 8-72) after administration, the urine is assayed for the presence of the antisense-RNA heteroduplex. If heteroduplex is detected, the backbone is suitable for use in the antisense oligomers of the present invention.","The test oligomer may be labeled, e.g. by a fluorescent or a radioactive tag, to facilitate subsequent analyses, if it is appropriate for the mammalian subject. The assay can be in any suitable solid-phase or fluid format. Generally, a solid-phase assay involves first binding the heteroduplex analyte to a solid-phase support, e.g., particles or a polymer or test-strip substrate, and detecting the presence\/amount of heteroduplex bound. In a fluid-phase assay, the analyte sample is typically pretreated to remove interfering sample components. If the oligomer is labeled, the presence of the heteroduplex is confirmed by detecting the label tags. For non-labeled compounds, the heteroduplex may be detected by immunoassay if in solid phase format or by mass spectroscopy or other known methods if in solution or suspension format.","When the antisense oligomer is complementary to a virus-specific region of the viral genome (such as those regions of influenza RNA, as described above) the method can be used to detect the presence of a given influenza virus, or reduction in the amount of virus during a treatment method.","E. Exemplary Oligomer Backbones","Examples of nonionic linkages that may be used in oligonucleotide analogs are shown in . In these figures, B represents a purine or pyrimidine base-pairing moiety effective to bind, by base-specific hydrogen bonding, to a base in a polynucleotide, preferably selected from adenine, cytosine, guanine and uracil. Suitable backbone structures include carbonate (3A, R\u2550O) and carbamate (2A, R\u2550NH) linkages (Mertes and Coats 1969; Gait, Jones et al. 1974); alkyl phosphonate and phosphotriester linkages (2B, R=alkyl or \u2014O-alkyl) (Lesnikowski, Jaworska et al. 1990); amide linkages (2C) (Blommers, Pieles et al. 1994); sulfone and sulfonamide linkages (2D, R, R\u2550CH); and a thioformacetyl linkage (2E) (Cross, Rice et al. 1997). The latter is reported to have enhanced duplex and triplex stability with respect to phosphorothioate antisense compounds (Cross, Rice et al. 1997). Also reported are the 3\u2032-methylene-N-methylhydroxyamino compounds of structure 2F.  is an example of a cationic linkage group.","Peptide nucleic acids (PNAs) are analogs of DNA in which the backbone is structurally homomorphous with a deoxyribose backbone, consisting of N-(2-aminoethyl) glycine units to which pyrimidine or purine bases are attached. PNAs containing natural pyrimidine and purine bases hybridize to complementary oligonucleotides obeying Watson-Crick base-pairing rules, and mimic DNA in terms of base pair recognition (Egholm, Buchardt et al. 1993). The backbone of PNAs is formed by peptide bonds rather than phosphodiester bonds, making them well-suited for antisense applications. The backbone is uncharged, resulting in PNA\/DNA or PNA\/RNA duplexes which exhibit greater than normal thermal stability. PNAs are not recognized by nucleases or proteases.","A preferred oligomer structure employs morpholino-based subunits bearing base-pairing moieties, joined by uncharged linkages, as described above. Especially preferred is a substantially uncharged phosphorodiamidate-linked morpholino oligomer, such as illustrated in , and . Morpholino oligonucleotides, including antisense oligomers, are detailed, for example, in co-owned U.S. Pat. Nos. 5,698,685, 5,217,866, 5,142,047, 5,034,506, 5,166,315, 5,185,444, 5,521,063, and 5,506,337, all of which are expressly incorporated by reference herein.","Important properties of the morpholino-based subunits include: the ability to be linked in a oligomeric form by stable, uncharged backbone linkages; the ability to support a nucleotide base (e.g. adenine, cytosine, guanine or uracil) such that the polymer formed can hybridize with a complementary-base target nucleic acid, including target RNA, with high Tm, even with oligomers as short as 10-14 bases; the ability of the oligomer to be actively transported into mammalian cells; and the ability of the oligomer:RNA heteroduplex to resist RNAse degradation.","Exemplary backbone structures for antisense oligonucleotides of the invention include the \u03b2-morpholino subunit types shown in , each linked by an uncharged, phosphorus-containing subunit linkage.  shows a phosphorus-containing linkage which forms the five atom repeating-unit backbone, where the morpholino rings are linked by a 1-atom phosphoamide linkage.  shows a linkage which produces a 6-atom repeating-unit backbone. In this structure, the atom Y linking the 5\u2032 morpholino carbon to the phosphorus group may be sulfur, nitrogen, carbon or, preferably, oxygen. The X moiety pendant from the phosphorus may be fluorine, an alkyl or substituted alkyl, an alkoxy or substituted alkoxy, a thioalkoxy or substituted thioalkoxy, or unsubstituted, monosubstituted, or disubstituted nitrogen, including cyclic structures, such as morpholines or piperidines. Alkyl, alkoxy and thioalkoxy preferably include 1-6 carbon atoms. The Z moieties are sulfur or oxygen, and are preferably oxygen.","The linkages shown in  are designed for 7-atom unit-length backbones. In Structure 1C, the X moiety is as in Structure 1B, and the Y moiety may be methylene, sulfur, or, preferably, oxygen. In Structure 1D, the X and Y moieties are as in Structure 1B. Particularly preferred morpholino oligonucleotides include those composed of morpholino subunit structures of the form shown in , where X\u2550NHor N(CH), Y\u2550O, and Z=O.","As noted above, the substantially uncharged oligomer may advantageously include a limited number of charged linkages, e.g. up to about 1 per every 5 uncharged linkages, more preferably up to about 1 per every 10 uncharged linkages. Therefore a small number of charged linkages, e.g. charged phosphoramidate or phosphorothioate, may also be incorporated into the oligomers.","The antisense compounds can be prepared by stepwise solid-phase synthesis, employing methods detailed in the references cited above. In some cases, it may be desirable to add additional chemical moieties to the antisense compound, e.g. to enhance pharmacokinetics or to facilitate capture or detection of the compound. Such a moiety may be covalently attached, typically to a terminus of the oligomer, according to standard synthetic methods. For example, addition of a polyethyleneglycol moiety or other hydrophilic polymer, e.g., one having 10-100 monomeric subunits, may be useful in enhancing solubility. One or more charged groups, e.g., anionic charged groups such as an organic acid, may enhance cell uptake. A reporter moiety, such as fluorescein or a radiolabeled group, may be attached for purposes of detection. Alternatively, the reporter label attached to the oligomer may be a ligand, such as an antigen or biotin, capable of binding a labeled antibody or streptavidin. In selecting a moiety for attachment or modification of an antisense oligomer, it is generally of course desirable to select chemical compounds of groups that are biocompatible and likely to be tolerated by a subject without undesirable side effects.","IV. Inhibition of Influenza Viral Replication","The antisense compounds detailed above are useful in inhibiting replication of single-stranded, negative-sense, segmented RNA viruses of the Orthomyxoviridae family. In one embodiment, such inhibition is effective in treating infection of a host animal by these viruses. Accordingly, the method comprises, in one embodiment, contacting a cell infected with the virus with a antisense agent effective to inhibit the replication of the specific virus. In this embodiment, the antisense agent is administered to a mammalian subject, e.g., human or domestic animal, infected with a given virus, in a suitable pharmaceutical carrier. It is contemplated that the antisense oligonucleotide arrests the growth of the RNA virus in the host. The RNA virus may be decreased in number or eliminated with little or no detrimental effect on the normal growth or development of the host.","In the present invention as described in the Examples, Phosphorodiamidate Morpholino Oligomers (PMOs), designed to hybridize to various gene segments of influenza A virus, were evaluated for their ability to inhibit influenza virus production in Vero cell culture. The PMOs were conjugated to a short arginine-rich peptide to facilitate entry into cells in culture. Vero cells were incubated with PMO compounds, inoculated with influenza virus, and viral titer determined by hemagglutinin assay and\/or plaque-assay. The compounds targeting the AUG translation start-sites of polymerase component PB1 and nuclear capsid protein (NP), the 5\u2032 and 3\u2032 ends of vRNA NP segment and the 3\u2032end of cRNA NP segment were very effective, reducing the titer of influenza virus by 1 to 3 orders of magnitude compared to controls, in a dose-dependent and sequence-specific manner over a period of 2 days. Combinations of some of the PMOs exhibited a synergistic antiviral effect as described in Example 3. These data indicate that several of the PMOs tested in this study are potential influenza A therapeutics.","The effective anti-influenza A PMO compounds were observed not to alter the titer of influenza B virus grown in Vero cells due to the lack of homology between the influenza A virus-specific PMOs and the corresponding influenza B virus targets. However, the PMO described herein (SEQ ID NOs:10-24) will target most, if not all, influenza A virus strains because of the high degree of homology between strains at the respective targets (SEQ ID NOs:1-9). An example of the sequence conservation at two preferred targets is shown in . In this case the sequence conservation between multiple isolates of H5N1 (e.g. bird flu) and H1N1 was determined for the targets of the PB1-AUG and (\u2212) NP-3\u2032trm PMOs (SEQ ID NOs:12 and 13, respectively).  shows a very high level of conservation at these target sites.","A. Identification of the Infective Agent","The specific virus causing the infection can be determined by methods known in the art, e.g. serological or cultural methods, or by methods employing the antisense oligomers of the present invention.","Serological identification employs a viral sample or culture isolated from a biological specimen, e.g., stool, urine, cerebrospinal fluid, blood, etc., of the subject. Immunoassay for the detection of virus is generally carried out by methods routinely employed by those of skill in the art, e.g., ELISA or Western blot. In addition, monoclonal antibodies specific to particular viral strains or species are often commercially available.","Culture methods may be used to isolate and identify particular types of virus, by employing techniques including, but not limited to, comparing characteristics such as rates of growth and morphology under various culture conditions.","Another method for identifying the viral infective agent in an infected subject employs one or more antisense oligomers targeting broad families and\/or genera of viruses. Sequences targeting any characteristic viral RNA can be used. The desired target sequences are preferably (i) common to broad virus families\/genera, and (ii) not found in humans. Characteristic nucleic acid sequences for a large number of infectious viruses are available in public databases, and may serve as the basis for the design of specific oligomers.","For each plurality of oligomers, the following steps are carried out: (a) the oligomer(s) are administered to the subject; (b) at a selected time after said administering, a body fluid sample is obtained from the subject; and (c) the sample is assayed for the presence of a nuclease-resistant heteroduplex comprising the antisense oligomer and a complementary portion of the viral genome. Steps (a)-(c) are carried for at least one such oligomer, or as many as is necessary to identify the virus or family of viruses. Oligomers can be administered and assayed sequentially or, more conveniently, concurrently. The virus is identified based on the presence (or absence) of a heteroduplex comprising the antisense oligomer and a complementary portion of the viral genome of the given known virus or family of viruses.","Preferably, a first group of oligomers, targeting broad families, is utilized first, followed by selected oligomers complementary to specific genera and\/or species and\/or strains within the broad family\/genus thereby identified. This second group of oligomers includes targeting sequences directed to specific genera and\/or species and\/or strains within a broad family\/genus. Several different second oligomer collections, i.e. one for each broad virus family\/genus tested in the first stage, are generally provided. Sequences are selected which are (i) specific for the individual genus\/species\/strains being tested and (ii) not found in humans.","B. Administration of the Antisense Oligomer","Effective delivery of the antisense oligomer to the target nucleic acid is an important aspect of treatment. In accordance with the invention, routes of antisense oligomer delivery include, but are not limited to, various systemic routes, including oral and parenteral routes, e.g., intravenous, subcutaneous, intraperitoneal, and intramuscular, as well as inhalation, transdermal and topical delivery. The appropriate route may be determined by one of skill in the art, as appropriate to the condition of the subject under treatment. For example, an appropriate route for delivery of an antisense oligomer in the treatment of a viral infection of the skin is topical delivery, while delivery of a antisense oligomer for the treatment of a viral respiratory infection is by inhalation. The oligomer may also be delivered directly to the site of viral infection, or to the bloodstream.","The antisense oligomer may be administered in any convenient vehicle which is physiologically acceptable. Such a composition may include any of a variety of standard pharmaceutically acceptable carriers employed by those of ordinary skill in the art. Examples include, but are not limited to, saline, phosphate buffered saline (PBS), water, aqueous ethanol, emulsions, such as oil\/water emulsions or triglyceride emulsions, tablets and capsules. The choice of suitable physiologically acceptable carrier will vary dependent upon the chosen mode of administration.","In some instances, liposomes may be employed to facilitate uptake of the antisense oligonucleotide into cells. (See, e.g., Williams, S. A., Leukemia 10(12):1980-1989, 1996; Lappalainen et al., Antiviral Res. 23:119, 1994; Uhlmann et al., antisense oligonucleotides: a new therapeutic principle, Chemical Reviews, Volume 90, No. 4, pages 544-584, 1990; Gregoriadis, G., Chapter 14, Liposomes, Drug Carriers in Biology and Medicine, pp. 287-341, Academic Press, 1979). Hydrogels may also be used as vehicles for antisense oligomer administration, for example, as described in WO 93\/01286. Alternatively, the oligonucleotides may be administered in microspheres or microparticles. (See, e.g., Wu, G. Y. and Wu, C. H., J. Biol. Chem. 262:4429-4432, 1987). Alternatively, the use of gas-filled microbubbles complexed with the antisense oligomers can enhance delivery to target tissues, as described in U.S. Pat. No. 6,245,747.","Sustained release compositions may also be used. These may include semipermeable polymeric matrices in the form of shaped articles such as films or microcapsules.","In one aspect of the method, the subject is a human subject, e.g., a patient diagnosed as having a localized or systemic viral infection. The condition of a patient may also dictate prophylactic administration of an antisense oligomer of the invention, e.g. in the case of a patient who (1) is immunocompromised; (2) is a burn victim; (3) has an indwelling catheter; or (4) is about to undergo or has recently undergone surgery. In one preferred embodiment, the oligomer is a phosphorodiamidate morpholino oligomer, contained in a pharmaceutically acceptable carrier, and is delivered orally. In another preferred embodiment, the oligomer is a phosphorodiamidate morpholino oligomer, contained in a pharmaceutically acceptable carrier, and is delivered intravenously (i.v.).","In another application of the method, the subject is a livestock animal, e.g., a chicken, turkey, pig, cow or goat, etc, and the treatment is either prophylactic or therapeutic. The invention also includes a livestock and poultry food composition containing a food grain supplemented with a subtherapeutic amount of an antiviral antisense compound of the type described above. Also contemplated is, in a method of feeding livestock and poultry with a food grain supplemented with subtherapeutic levels of an antiviral, an improvement in which the food grain is supplemented with a subtherapeutic amount of an antiviral oligonucleotide composition as described above.","The antisense compound is generally administered in an amount and manner effective to result in a peak blood concentration of at least 200-400 nM antisense oligomer. Typically, one or more doses of antisense oligomer are administered, generally at regular intervals, for a period of about one to two weeks. Preferred doses for oral administration are from about 1-100 mg oligomer per 70 kg. In some cases, doses of greater than 100 mg oligomer\/patient may be necessary. For i.v. administration, preferred doses are from about 0.5 mg to 100 mg oligomer per 70 kg. The antisense oligomer may be administered at regular intervals for a short time period, e.g., daily for two weeks or less. However, in some cases the oligomer is administered intermittently over a longer period of time. Administration may be followed by, or concurrent with, administration of an antibiotic or other therapeutic treatment. The treatment regimen may be adjusted (dose, frequency, route, etc.) as indicated, based on the results of immunoassays, other biochemical tests and physiological examination of the subject under treatment.","C. Monitoring of Treatment","An effective in vivo treatment regimen using the antisense oligonucleotides of the invention may vary according to the duration, dose, frequency and route of administration, as well as the condition of the subject under treatment (i.e., prophylactic administration versus administration in response to localized or systemic infection). Accordingly, such in vivo therapy will often require monitoring by tests appropriate to the particular type of viral infection under treatment, and corresponding adjustments in the dose or treatment regimen, in order to achieve an optimal therapeutic outcome. Treatment may be monitored, e.g., by general indicators of infection, such as complete blood count (CBC), nucleic acid detection methods, immunodiagnostic tests, viral culture, or detection of heteroduplex.","The efficacy of an in vivo administered antisense oligomer of the invention in inhibiting or eliminating the growth of one or more types of RNA virus may be determined from biological samples (tissue, blood, urine etc.) taken from a subject prior to, during and subsequent to administration of the antisense oligomer. Assays of such samples include (1) monitoring the presence or absence of heteroduplex formation with target and non-target sequences, using procedures known to those skilled in the art, e.g., an electrophoretic gel mobility assay; (2) monitoring the amount of viral protein production, as determined by standard techniques such as ELISA or Western blotting, or (3) measuring the effect on viral titer, e.g. by the method of Spearman-Karber. (See, for example, Pari, G. S. et al., Antimicrob. Agents and Chemotherapy 39(5): 1157-1161, 1995; Anderson, K. P. et al., Antimicrob. Agents and Chemotherapy 40(9):2004-2011, 1996, Cottral, G. E. (ed) in: Manual of Standard Methods for Veterinary Microbiology, pp. 60-93, 1978).","A preferred method of monitoring the efficacy of the antisense oligomer treatment is by detection of the antisense-RNA heteroduplex. At selected time(s) after antisense oligomer administration, a body fluid is collected for detecting the presence and\/or measuring the level of heteroduplex species in the sample. Typically, the body fluid sample is collected 3-24 hours after administration, preferably about 6-24 hours after administering. As indicated above, the body fluid sample may be urine, saliva, plasma, blood, spinal fluid, or other liquid sample of biological origin, and may include cells or cell fragments suspended therein, or the liquid medium and its solutes. The amount of sample collected is typically in the 0.1 to 10 ml range, preferably about 1 ml or less.","The sample may be treated to remove unwanted components and\/or to treat the heteroduplex species in the sample to remove unwanted ssRNA overhang regions, e.g., by treatment with RNase. It is, of course, particularly important to remove overhang where heteroduplex detection relies on size separation, e.g., electrophoresis of mass spectroscopy.","A variety of methods are available for removing unwanted components from the sample. For example, since the heteroduplex has a net negative charge, electrophoretic or ion exchange techniques can be used to separate the heteroduplex from neutral or positively charged material. The sample may also be contacted with a solid support having a surface-bound antibody or other agent specifically able to bind the heteroduplex. After washing the support to remove unbound material, the heteroduplex can be released in substantially purified form for further analysis, e.g., by electrophoresis, mass spectroscopy or immunoassay.","V. Examples","The following examples illustrate but are not intended in any way to limit the invention.","Standard recombinant DNA techniques were employed in all constructions, as described in Ausubel, F M et al., in CPMB, John Wiley and Sons, Inc., Media, Pa., 1992 and Sambrook, J. et al., in MC: A LM, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., Vol. 2, 1989).","All peptides were custom synthesized by Global Peptide Services (Ft. Collins, Colo.) or at AVI BioPharma (Corvallis, Oreg.) and purified to >90% purity (see Example 2 below). PMOs were synthesized at AVI BioPharma in accordance with known methods, as described, for example, in ((Summerton and Weller 1997) and U.S. Pat. No. 5,185,444. The structure of the PMO is as shown in .","PMO oligomers were conjugated at the 5\u2032 end with an arginine-rich peptide (RFRC-5\u2032-PMO, SEQ ID NO:25) to enhance cellular uptake as described (U.S. Patent Application 60\/466,703 and (Moulton, Nelson et al. 2004; Nelson, Stein et al. 2005).","Phosphorodiamidate Morpholino Oligomers (PMOs), designed to hybridize to various gene segments of influenza A virus, were evaluated for their ability to inhibit influenza virus production in Vero cell culture. The PMOs were conjugated to a short arginine-rich peptide (RFRC) to facilitate entry into cells in culture. Vero cells were incubated with PMO compounds, inoculated with influenza A virus (strain PR8, H1N1), and viral titer determined by hemagglutinin assay (HA) and\/or plaque-assay (CFU). The PMO compounds targeting the AUG translation start-sites of polymerase component PB1 and nuclear capsid protein (NP) (SEQ ID NOs:10 and 11), the 5\u2032 and 3\u2032 ends of the NP gene (SEQ ID NOs:13 and 14) encoded by the viral RNA (i.e. vRNA) and the 3\u2032 end of the NP gene (SEQ ID NO: 15) encoded by the complementary RNA (cRNA) were very effective in reducing the titer of influenza virus by 1 to 3 orders of magnitude compared to controls, in a dose-dependent and sequence-specific manner over a period of 2 days.",{"@attributes":{"id":"p-0145","num":"0172"},"figref":["FIG. 5","FIG. 5","FIG. 5"]},"PMOs that target the AUG start codons of three influenza virus genes, NP-AUG, PB2-AUG and PB1-AUG (SEQ ID NOs: 0-12, respectively) were assayed for their ability to inhibit influenza A virus replication in a dose response assay using the hemagglutinin assay and the plaque-forming assay. The results are shown in . For all three PMOs the concentration that effectively resulted in a 50% reduction in viral replication (EC50) was found to be less than 1 \u03bcM. A three-log reduction in viral replication using the plaque assay was observed for two of the PMOs, NP-AUG and PB1-AUG at 10 \u03bcM. Identical assays were performed using the termini-targeted PMO: NP(\u2212)3\u2032, NP(\u2212)5\u2032, NP(+)3\u2032 and NP(+)5\u2032 (SEQ ID NOs:13, 14, 15 and 24, respectively). All four PMOs demonstrated significant reduction in viral titer as shown in .","Quantitative RTPCR was used to determine the effect of one of the termini-targeted PMO, NP(\u2212)3\u2032 (SEQ ID NO:13) on the transcription of the NP vRNA segment into mRNA and cRNA species (i.e., see ). The mRNA transcription product is positive-sense RNA whereas the cRNA is a negative-sense RNA. The NP(\u2212)3\u2032 PMO was incubated with Vero cells for 6 hours followed by influenza A virus infection at an MOI of 0.05. Three hours post-infection, RNA was isolated and RNA species specific reverse transcription (RT) was performed followed by quantitative PCR on the reaction product.  shows that the NP(\u2212)3\u2032 PMO (SEQ ID NO:13) that targets the 3\u2032 end of the vRNA strongly suppressed the transcription of NP mRNA and cRNA.","Combinations of some of the PMOs exhibited a synergistic antiviral effect.  shows the synergistic effect of various combinations of PMO that target the NP vRNA termini and the NP-AUG region. PMO treatment and influenza A virus infection were as described in Example 1. The plaque assay was used to measure virus replication. Three termini-targeted PMO, NP(\u2212)3\u2032, NP(\u2212)5\u2032, NP(+)3\u2032 (SEQ ID NOs:13-15) and the NP-AUG PMO (SEQ ID NO:10) were mixed in various combinations as shown in . One combination, NP(+)3\u2032 with NP(\u2212)5\u2032 did not produce antiviral activity as this pair of PMO are predicted to hybridize to each other. All the other PMO combinations demonstrated significant inhibition of influenza A viral replication.",{"@attributes":{"id":"p-0149","num":"0176"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Sequence\u2003Listing"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"14pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"SEQ"]},{"entry":[{},"ID"]},{"entry":[{},"NO"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"14pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Target\u2003Sequences\u2003(5\u2032 to\u20033\u2032)",{}]},{"entry":["UCACUCACUGAGUGACAUCAAAAUCGCGUCCCAAGGCACCAAAC","1"]},{"entry":["GGU",{}]},{"entry":{}},{"entry":["AGCGAAAGCAGGUCAAUUAUAUUCAAUGAAAGAAUAAAAGAACU","2"]},{"entry":["AAG",{}]},{"entry":{}},{"entry":["AGCCAAAGCAGGCAAACCAUUUGAGAUGUCAAUCCGACCUUACU","3"]},{"entry":["UUU",{}]},{"entry":{}},{"entry":["AAAGAAAAAUACCCUUGUUUCUACU","4"]},{"entry":{}},{"entry":["AGCAAAAGCAGGGUAGAUAAUCACU","5"]},{"entry":{}},{"entry":["CAUGAAAAAAUGCCUUGUUCCUACU","6"]},{"entry":{}},{"entry":["AGCGAAAGCAGGCAAACCAUUUCAA","7"]},{"entry":{}},{"entry":["CUUUAAAAACGACCUUGUUUCUACU","8"]},{"entry":{}},{"entry":["AGCGAAAGCAGGUCAAUUAUAUUCA","9"]},{"entry":{}},{"entry":["Oligomer\u2003Targeting\u2003Sequences\u2003(5\u2032 to\u20033\u2032)",{}]},{"entry":["CTTGGGACGCGATTTTG","10"]},{"entry":{}},{"entry":["CTTTTATTCTTTCATTG","11"]},{"entry":{}},{"entry":["GACATCTCAAATGGTTTG","12"]},{"entry":{}},{"entry":["AGCAAAAGCAGGGTAGATAATC","13"]},{"entry":{}},{"entry":["GAAAAATACCCTTGTTTCTACT","14"]},{"entry":{}},{"entry":["AGTAGAAACAAGGGTATTTTTC","15"]},{"entry":{}},{"entry":["ACCAAAAGCAGG","16"]},{"entry":{}},{"entry":["AGTAGAAACAAGG","17"]},{"entry":{}},{"entry":["AGCGAAAGCAGGCAAACCAT","18"]},{"entry":{}},{"entry":["GAAAAAATGCCTTGTTCCTACT","19"]},{"entry":{}},{"entry":["AGTAGGAACAAGGCATTTTTTC","20"]},{"entry":{}},{"entry":["AGCGAAAGCAGGTCAATTAT","21"]},{"entry":{}},{"entry":["TAAAAACGACCTTGTTTCTACT","22"]},{"entry":{}},{"entry":["AGTAGAAACAAGGTCGTTTTTA","23"]},{"entry":{}},{"entry":["AGTCTCGACTTGCTACCTCA","24"]},{"entry":{}},{"entry":["Peptide\u2003Sequences",{}]},{"entry":["RFRC","25"]},{"entry":{}},{"entry":["(RAhxR)\u2003Ahx\u03b2Ala","26"]},{"entry":{}},{"entry":["(RAhx)\u2003\u03b2Ala","27"]},{"entry":{}},{"entry":["(RAhx)\u2003\u03b2Ala","28"]},{"entry":{}},{"entry":["(RAhxR)\u2003Ahx\u03b2Ala","29"]},{"entry":{}},{"entry":["(RAhxR)\u2003Ahx\u03b2Ala","30"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}}],"GOVINT":[{},{}],"heading":["STATEMENT OF GOVERNMENT INTEREST","FIELD OF THE INVENTION","REFERENCES","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","A. Materials and Methods","B. Example 1","Inhibition of Influenza A virus in Cell Culture with Phosphorodiamidate Morpholino Oligomers","C. Example 2","Effect of PMO Targeting the 3\u2032-Terminus of NP vRNA on NP mRNA and cRNA Transcription","D. Example 3","Synergistic Inhibition of Influenza A Virus Replication in Cell Culture Using Combinations of Anti-Influenza PMO"],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE FIGURES","p":[{"@attributes":{"id":"p-0029","num":"0056"},"figref":"FIGS. 1A-1D"},{"@attributes":{"id":"p-0030","num":"0057"},"figref":["FIGS. 2A-2G","FIG. 2H"]},{"@attributes":{"id":"p-0031","num":"0058"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0032","num":"0059"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0033","num":"0060"},"figref":["FIGS. 5A-5B","FIG. 5C"]},{"@attributes":{"id":"p-0034","num":"0061"},"figref":["FIG. 6","FIG. 6C"],"b":["6","6"]},{"@attributes":{"id":"p-0035","num":"0062"},"figref":["FIGS. 7A-7B","FIG. 6","FIG. 7C"]},{"@attributes":{"id":"p-0036","num":"0063"},"figref":["FIGS. 8A-8C","FIG. 8D"]},{"@attributes":{"id":"p-0037","num":"0064"},"figref":["FIG. 9A","FIG. 9B"]}]},"DETDESC":[{},{}]}
