---
title: Classification of patients having diffuse large b-cell lymphoma based upon gene expression
abstract: Methods and kits for classifying patients having diffuse large B-cell lymphoma (DLBCL) based upon expression of a plurality of genes are disclosed. Real-time quantitative RT-PCR can be used to measure expression values. Correlating expression values of the plurality of genes in a tumor sample from the patient to reference expression values obtained from DLBCL patients can stratify patients in the classification groups. The methods and kits can be used to predict overall patient survival.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=07622253&OS=07622253&RS=07622253
owner: 
number: 07622253
owner_city: 
owner_country: 
publication_date: 20061003
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATIONS","GOVERNMENT INTERESTS","FIELD","BACKGROUND","SUMMARY","DETAILED DESCRIPTION","EXAMPLE 1","EXAMPLE 2","EXAMPLE 3"],"p":["This application is a continuation of U.S. patent application Ser. No. 10\/792,374, filed on Mar. 3, 2004, which claims the benefit of U.S. Provisional Application No. 60\/510,822, filed on Oct. 14, 2003, and both are hereby incorporated in their entirety by reference.","This work was supported at least in part with funds from the federal government under U.S.P.H.S. Grants CA33399 and CA34233, awarded by the National Institutes of Health and under NIST Grant No. 70NANB8H4002, awarded by the National Institute of Standards and Technology. The U.S. Government may have certain rights in the inventions.","This application relates generally to gene expression in cancerous tissues and, more particularly, to gene expression in diffuse large B-cell lymphoma (DLBCL) tissues and to methods for classifying patients with DLBCL based upon gene expression in DLBCL tissues.","Although combination chemotherapy for the treatment of DLBCL patients has been available for several years, currently, over one-half of all patients do not achieve a durable remission (Vose, supra, 1998). Risk stratification of patients has been attempted to identify patients in which more aggressive treatment may be required. One risk stratification approach has involved use of the International Prognostic Index (IPI), which is based upon 5 clinical criteria (The International Non-Hodgkin's Lymphoma Prognostic Factors Project, 32:987-993, 1993). However, the IPI has not provided an accurate prediction of survival in a substantial number of patients.","Accordingly, the present inventors have succeeded in developing an approach for stratifying DLBCL patients at the molecular level based upon gene expression in DLBCL tissues. The approach involves correlating expression values of a plurality of genes in tumor samples from patients having DLBCL to classification characteristics of the disease, such as, for example, overall patient survival. A set of genes can be selected from the plurality of genes based upon the expression of the selected genes showing a correlation to the classification characteristics. The relationship developed from this correlation can then allow patient classification by measuring expression of the selected genes in a tumor sample from a patient and comparing with expression values obtained in the correlation study. The approach can be applied not only to DLBCL, but also to other cancers as well as non-cancerous diseases.","Thus, in various embodiments, the present invention can involve methods for classifying a patient or patients having DLBCL into groups based upon classification characteristics. The methods can comprise measuring expression of a plurality of genes, in a tumor sample from a patient and correlating tumor expression values to normalized reference expression values obtained for the plurality of genes from DLBCL patients stratified in the classification groups. In various aspects of this embodiment, the method can predict patient survival based upon the selected plurality of genes being predictive of survival by virtue of being identified in DLBCL patients stratified in groups of known overall survival. In various aspects of this embodiment as well as embodiments described below, classification characteristics other than or in addition to overall survival can be used such as, for example, likelihood of successful treatment for various treatments which can be used to select a specific therapy approach for a given patient. Gene expression can be measured by any method that quantifies gene expression such as real time RT-PCR. Quantification can be relative or absolute quantification or a combination of both as applied to the normalization process, which is discussed more fully below. Briefly, relative quantification references expression of a target gene to a control value for expression such as, for example, expression obtained from a control sample or pretreatment sample or expression of a reference gene. Absolute quantification is based upon an internal or external calibration curve (see for example, Pfaff et al., 30:e36, 2002; Livak et al., 25:402-408, 2001).","In various other embodiments, the present invention can involve a method for obtaining a formula for classifying patients having a disease, such as, for example, DLBCL. The method comprises correlating normalized expression values of a plurality of genes in tumor samples obtained from patients having the disease to at least one known classification characteristic of the disease. In various aspects of this embodiment, the method can predict patient survival and the classification characteristic of the disease can be overall survival. Gene expression can be measured by any method which quantitates gene expression such as real time RT-PCR. The plurality of genes can be at least two, at least three, at least four, at least five or all of the genes LMO2, BCL-6, FN1, CCND2, SCYA3 and BCL-2. Additional genes can also be included.","The present invention, in various embodiments, can also involve kits for classifying a patient having DLBCL into classification groups, such as, for example, groups predictive of the probability of survival of the patient. The kits contain assays for measuring expression of a plurality of genes in a tumor sample from a patient having DLBCL. The normalized expression of the plurality of genes in tumor samples from DLBCL patients stratifies the patients into classification groups. The assays in the kits can comprise real time RT-PCR assays. The kits can also contain software for using the expression data so as to simplify the assignment of patients to classification groups.","In various embodiments, the present invention can also involve a method for predicting survival in a patient having DLBCL. The method comprises measuring in a sample containing tumor cells from the patient, expression of a plurality of genes and determining whether normalized expression of the genes indicates increased or decreased probability of survival. The plurality of genes can be at least three, at least four, at least five or all of the genes LMO2, BCL-6, FN1, CCND2, SCYA3 and BCL-2. Additional genes can also be included. In one aspect, determining can involve determining whether normalized expression of the three or more genes matches expression criteria indicative of increased probability of survival, compared to expression in reference cells. The reference cells can be non-cancerous cells from the patient or cells other than DLBCL tumor cells obtained from sources other than the patient such as, for example, Raji cells. The expression criteria can be selected from the group consisting of increased expression of LMO2, increased expression of BCL-6, increased expression of FN1, decreased expression of CCND2, decreased expression of SCYA3 and decreased expression of BCL-2. In various aspects of this embodiment, the reference cells can be Raji cells. Gene expression can be measured by any of a number of methods such as, for example, cDNA or cRNA microarray test, tissue microarray test or real time RT-PCR.","In various of the embodiments above, normalized expression can comprise values calculated by one or both of calculating the ratio of expression values of the target gene and an endogenous reference gene and calculating the ratio of expression values of the target gene to expression of the same gene reference cells with or without normalization to the endogenous reference gene. The endogenous reference gene can be a housekeeping gene such as, for example, PGK1 or GAPDH. The reference cell line can be a Raji cell line. Reference stratification of patients based upon expression values can be generated using univariate Cox proportional hazards analysis with classification, such as, for example, overall survival as dependent variable. Moreover, the methods can use IPI scores in addition to the gene expression information obtained.","In various of the embodiments above, gene expression in a patient can be compared to gene expression in reference DLBCL patients of known survival using the formula:\n\n(2)+(-6)+(1)+(2)+(3)+(-2)\n","The terms LMO2, BCL-6, FN1, CCND2, SCYA3 and BCL-2 can be log base 2 of normalized expression values for genes LMO2, BCL-6, FN1, CCND2, SCYA3 and BCL-2, respectively. In various embodiments A can be about \u22120.03, B can be about \u22120.2, C can be about \u22120.2, D can be about 0.03, E can be about 0.2 and F can be about 0.6. Using these values, a Z value of less than about \u22120.06 can indicate high probability of survival, a Z value of from about \u22120.06 to about 0.09 can indicate medium probability of survival and a Z value of greater than about 0.09 can indicate low probability of survival. In various aspects of this embodiment, A can be about \u22120.0273, B is about \u22120.2103, C can be about \u22120.1878, D can be about 0.0346, E can be about 0.1888 and F is can be about 0.5527. Using these values, a Z value of less than about \u22120.063 indicates high probability of survival, a Z value of from about \u22120.063 to about 0.093 indicates medium probability of survival and a Z value of greater than about 0.093 indicates low probability of survival.","Application of the methods of the present invention to clinical practice allows identification of patients who are unlikely to be cured by conventional therapy and in whom investigational approaches would be justified in an effort to improve their outcome.","The present invention, in various embodiments, can involve methods for classifying patients having a disease into groups based upon gene expression values from a plurality of genes. The disease can be DLBCL or other cancers or a non-cancerous disease.","Classification groups or stratification groups for patients having DLBCL can involve any of a variety of features of the disease, in particular, various aspects that characterize the severity of the disease into groups based upon morbitity or mortality of the patients having the disease. One measure of mortality is \u201coverall survival\u201d sometimes referred to as \u201csurvival rate\u201d. The term \u201coverall survival\u201d refers to the percentage of subjects in a study who have survived for a defined period of time, usually measured from the time of diagnosis although it can also be measured from the time of initiation of treatment. Overall survival time of DLBCL patients as referenced herein, is calculated from the date of the diagnosis until death or last follow-up examination.","Inasmuch as DLBCL patients normally receive various treatments for the disease, overall survival time can mean survival time following chemotherapy. Chemotherapy can be Anthracycline-based chemotherapy and such anthracycline-based chemotherapy, as used herein, is intended to refer to the use of at least one anthracycline-class compound in chemotherapy treatment. As a non-limiting example, doxirubicin is an anthracycline-class compound used for treating non-Hodgkin's lymphoma and this compound can be used in a combination treatment of cyclophosphamide, doxorubicin, vincristine and prednisone (Vose, supra, 1998).","In various embodiments, the disease DLBCL can be identified in patients prior to applying the methods of the present invention. Methods of diagnosing DLBCL are well known in the art such as, for example, the use of histologic and immunologic criteria (see for example, Harris et al, 84:1361-1392, 1994; The Non-Hodgkin's Lymphoma classification Project, 89:3909-3918, 1997). After identification, the methods of the present invention can be used to classify patients having the disease.","In various embodiments, the methods of the present invention can also be used in determining whether DLBCL is present in a patient and in distinguishing of DLBCL from other diseases as well as in monitoring of the disease status or the recurrence of the disease, and in determining a preferred therapeutic regimen for the patient. Gene expression in DLBCL tumors can thus, be used in the diagnosis of DLBCL patients. Assessing the gene expression profile of DLBCL tumors can, in certain instances, provide a diagnostic basis for identifying disease aggressiveness and tumor progression (Lossos et al, 77:321-329, 2003). Thus, in various embodiments, classification of patients into survival probability groups can constitute the classification of patients into subsets of DLBCL diseases having different clinical prognoses.","Identification of patterns of gene expression can form the basis for understanding tumorigenesis at the molecular level as well as the underlying mechanisms that may contribute to disease aggressiveness and tumor progression (Lossos et al., 77:L321-329, 2003). Thus, evaluation of gene expression related to DLBCL can provide a more meaningful approach to understanding the disease than has been available in histologic or other clinical tests that have attempted to classify patients with DLBCL. Gene expression involves transcription of genomic DNA to form RNA's and ultimately proteins in the cell. Assessing gene expression can be done by determining cellular RNA or protein levels in a cell. Numerous methods for measuring gene expression at the RNA or protein level are known. Non-limiting examples of methods that measure RNA include Northern blotting, nuclease protection assays, DNA microarrays, serial analysis of gene expression, quantitative reverse transcription-polymerase chain reaction (RT-PCR), differential-display RT-PCR, massively parallel signature sequencing and the like. In particular, measurement of gene expression at the RNA level can be performed using real-time quantitative RT-PCR assay such as exonuclease-based assays, for example, TaqMan\u00ae assays. Non-limiting examples of methods of measuring protein expression levels include mass spectrometry, two-dimensional gel electrophoresis, antibody microarrays, tissue microarrays, ELISA, radioimmunoassay, immuno-PCR and the like.","In various embodiments, the methods of the present invention can be used to identify the pattern of gene expression in DLBCL and to determine the relationship to various aspects of DLBCL such as, for example, disease prognosis. A number of genes have been suggested to be related to DLBCL (see for example Alizedeh et al., 403:503-511, 2000; Shipp et al., supra, 2002; Rosenwald, et al., supra, 2002 and Table 1 below). These and other genes can be evaluated using various methods of the present invention to assess the relationship of gene expression to disease prognosis such as overall survival in a population of individuals having DLBCL and to determine the prognosis of an individual having the disease. In particular, BCL-6 has been shown to predict survival in DLBCL patients using real-time RT-PCR methods (Lossos et al., 98:945-951, 2001). Thus, in various embodiments, BCL-6 can be one of the genes used to classify DLBCL patients in overall survival groups.","In various embodiments gene expression values can be normalized to provide more accurate quantification and to correct for experimental variations. In various aspects of the invention, the calculation of gene expression values from the real-time RT-PCR tests can involve generating C(threshold cycle) values for target gene and an endogenous reference gene RNAs from control and experimental samples; determining nanogram amounts of each RNA using calibration standard curves; calculating the ratio of target and endogenous gene reference RNA; and calculating the ratio of nanograms target gene RNA in control and experiment samples. The endogenous reference RNA can be that of a housekeeping gene (see for example, Lossos et al, 17:789-795, 2003). In particular, phosphoglycerate kinase 1 (PGK1) or glyceraldehyde-3-phosphate dehydrogenase (GAPDH) can be used as the endogenous reference RNA. Calibration standard curves can be generated using cDNA from Raji cells or from Universal Human Reference RNA (Stratogene, La Jolla, Calif.). Raji cells can also be used for determining control target gene RNA and endogenous gene RNA. Normalization aspects of the calculations can comprise one or both of calculating the ratio of expression values of the target gene and an endogenous reference gene and calculating the ratio of expression values of the target gene to expression of the same gene in a reference cell line with or without normalization to the endogenous reference gene. Other normalization methods that correct for experimental variation can also be used (for review see Freeman et al, 26:112-125, 1999).","The normalized gene expression values can be transformed to log-base 2 values. Further evaluation can then be performed by comparing the transformed values with selected classification criteria using various statistical methods. In constructing a survival prediction model, the normalized gene expression can be compared to overall survival as estimated using the product-limit method of Kaplan-Meier with comparisons based upon the log-rank test. Cox proportional hazards analysis with overall survival as the dependent variable can then be performed. Genes with an absolute univariate Cox score between \u22121.5 and 1.5 can then be analyzed by multivariate regressions analysis using a Cox proportional hazards regression model with overall survival as the dependent variable.","The invention can be further understood by reference to the examples which follow.","This example illustrates the selection of genes potentially predictive of overall survival and the performance of quantitative RT-PCR on the selected genes.","Thirty-six genes were selected for inclusion in the study as shown in Table 1.",{"@attributes":{"id":"p-0031","num":"0030"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"280pt","align":"center"}},"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Sources of supporting evidence for panel of 36 prognostic genes assessed in this study"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"140pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"140pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Genes (total = 36 total)","Reference"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["ICAM1\/CD54","Terol et al., Ann Oncol 14: 467-74, 2003."]},{"entry":["PAX5","Krenacs et al., Blood 92: 1308-16, 1998"]},{"entry":["Ki-67","Miller et al., Blood 83: 1460-6, 1994"]},{"entry":["CD44","Drillenburg et al., Leukemia 13: 1448-55, 1999"]},{"entry":["P53","Ichikawa et al., N Engl J Med 337: 529-34,"]},{"entry":[{},"1997"]},{"entry":[{},"Koduru et al., Blood 90: 4078-91, 1997"]},{"entry":["BCL-2","Gascoyne, et al, Blood 90: 244-51, 1997"]},{"entry":[{},"Kramer et al., J Clin Onco 14: 2131-8, 1996"]},{"entry":[{},"Hermine et al., Blood 87: 265-72, 1996"]},{"entry":[{},"Hill et al., Blood 88: 1046-51, 1996"]},{"entry":["BIRC5\/SURVIVIN","Adida et al., Blood 96: 1921-5, 2000"]},{"entry":["BCL-6","Lossos et al., Blood 98: 945-951, 2001"]},{"entry":[{},"Barrans et al., Blood 99: 1136-43, 2002"]},{"entry":["PRDMI","Shaffer et al., Immunity 13: 199-212, 2000"]},{"entry":["HGAL","Lossos et al., Blood 101: 433-40, 2003"]},{"entry":["SCYA3","Shaffer et al., Immunity 13: 199-212, 2000"]},{"entry":["CCND1","Shaffer et al., Immunity 13: 199-212, 2000"]},{"entry":["CCND2","Shaffer et al., Immunity 13: 199-212, 2000"]},{"entry":["LMO2, LRMP, CD10, MYBL1\/A-MYB,","Alizadeh et al., Nature 403: 503-11, 2003"]},{"entry":"BCL7A, PIK3CG, CR2, CD38, SLAM,"},{"entry":"WASPIP, CFLAR, SLA, IRF4, PMS1, HGAL,"},{"entry":"BCL-6, BCL-2"},{"entry":["NR4A3, PDE4B","Shipp et al. Nat Med 8: 68-74, 2002"]},{"entry":["FN1, PLAU, HLA-DQA1, HLA-DRA,","Rosenwald et al., N Engl J Med 346: 1937-47,"]},{"entry":["EEF1A1L4, NPM3, MYC, BCL-6, HGAL","2002"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"left","id":"FOO-00001"},"sup":"\u2020"}},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"left","id":"FOO-00002"},"sup":"\u2020\u2020"}}]}}]}}},"The expression of each of these had previously been reported to predict DLBCL survival, either in single gene studies or in the analysis of large data sets derived from microarray studies. In addition, we applied Significance Analysis of Microarrays (Tusher et al., 98:5116-21, 2001)\u2014a supervised method for the identification of genes significantly associated with survival\u2014to the dataset of Alizadeh et al. (Alizadeh et al., supra, 2000), to detect and recover any significant genes missed in the exploratory analyses employed by the authors.","Tumor specimens from patients newly diagnosed with DLBCL were obtained during the course of diagnostic procedures at Stanford University medical center between the years of 1975 and 1995. Specimens were stored as previously reported. All the DLBCL tumors had the histological appearance of centroblastic large cell lymphomas demonstrating diffuse pattern of involvement without evidence of residual follicles. All patients were treated with an anthracycline containing chemotherapy regimen and had clinical follow up at Stanford University Hospital. A total of 66 primary DLBCL specimens fulfilled these inclusion criteria. Staging information was obtained for all the patients according to the Ann Arbor system. The IPI score was able to be determined for 59 of these patients.","For each of these 36 genes and a pair of internal controls for input mRNA (PGK1 and GAPDH), we measured gene expression using quantitative RT-PCR, based on primer and probe sets shown in Table 2. We assayed the expression of each gene in each of the 66 patient specimens relative to that in a reference RNA sample. Isolation of RNA, its quantification and the RT reactions were performed as previously reported (Lossos et al., 101:433-40, 2003; Lossos et al., 17:789-95, 2003).",{"@attributes":{"id":"p-0035","num":"0034"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"322pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"0pt","align":"left"}}],"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["Primer and probe sequences employed for panel of",{}]},{"entry":"36 genes surveyed"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"140pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},"SEQ.",{}]},{"entry":[{},"Accession No.",{},"ID."]},{"entry":["Gene","(Assay I.D. No.)*","Sequences","NO."]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"6","colwidth":"0pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["ICAM1\/","NM_000201","Probe","CTGTTCCCAGGACCTG","1",{}]},{"entry":["CD54","(Hs00277001_ml)","Forward Primer","ACGCTGAGCTCCTCTGCTACTC","2"]},{"entry":[{},{},"Reverse Primer","CACAGATGTCTGGGCATTGC","3"]},{"entry":{}},{"entry":["PMS1","NM_000534","Probe","CATAGTTCTCCAGTTTAAC","4"]},{"entry":[{},"(Hs00153333_ml)","Forward Primer","TGGATGCTGGTGCCACAA","5"]},{"entry":[{},{},"Reverse Primer","CGTTATCTCGCACCTCAATTTTATC","6"]},{"entry":{}},{"entry":["p53\/TP53","NM_000546","Probe","CTGGCTGCCAATCC","7"]},{"entry":[{},"(Hs00153340_ml)","Forward Primer","GCTTTCCACGACGGTGACA","8"]},{"entry":[{},{},"Reverse Primer","TGACTGCGGCTCCTCCAT","9"]},{"entry":{}},{"entry":["BCL-2","NM_000633","Probe","ACAAAGGCATCCCAGCC","10"]},{"entry":[{},"(Hs00153350_ml)","Forward Primer","ACCTGCACACCTGGATCCA","11"]},{"entry":[{},{},"Reverse Primer","ACAGCCAGGAGAAATCAAACAGA","12"]},{"entry":{}},{"entry":["BIRC5\/","NM_001168","Probe","CCTTTGCAATTTTG","13"]},{"entry":["SURVIVIN","(Hs00153353_ml)","Forward Primer","CTGGACAGAGAAAGAGCCAAGAA","14"]},{"entry":[{},{},"Reverse Primer","GGCACGGCGCACTTTCT","15"]},{"entry":{}},{"entry":["PRDM1","NM_001198","Probe","TGAATCTCACACAAACAC","16"]},{"entry":[{},"(Hs00153357_ml)","Forward Primer","CCCGGAGAGCTGACAATGA","17"]},{"entry":[{},{},"Reverse Primer","TCAGTGCTCGGTTGCTTTAGACT","18"]},{"entry":{}},{"entry":["BCL-6","NM_001706","Probe","CCTTACCATTGTGAGAAGT","19"]},{"entry":[{},"(Hs00277037_ml)","Forward Primer","GCGAATCCACACAGGAGAGAA","20"]},{"entry":[{},{},"Reverse Primer","TTGTGACGGAAATGCAGGTTAC","21"]},{"entry":{}},{"entry":["CCND1\/","NM_001758","Probe","TCCATTTGCAGCAGCT","22"]},{"entry":["PRAD1","(Hs00277039_)ml)","Forward Primer","CCGAGAAGCTGTGCATCTACAC","23"]},{"entry":[{},{},"Reverse Primer","AGGTTCCACTTGAGCTTGTTCAC","24"]},{"entry":{}},{"entry":["CCND2","NM_001759","Probe","ACAGACCTCCAGCATC","25"]},{"entry":[{},"(Hs00277041_ml)","Forward Primer","CCCTACATGCGCAGAATGGT","26"]},{"entry":[{},{},"Reverse Primer","GACCTCTTCTTCGCACTTCTGTTC","27"]},{"entry":{}},{"entry":["CD38","NM_001775","Probe","CTTCTGCAAACCTGC","28"]},{"entry":[{},"(Hs00277045_ml)","Forward Primer","GGAGAAAGGACTGCAGCAACA","29"]},{"entry":[{},{},"Reverse Primer","AGCATCACATGGACCACATCAC","30"]},{"entry":{}},{"entry":["CR2","NM_001877","Probe","AGGCACACACCAGTTT","31"]},{"entry":[{},"(Hs00153398_ml)","Forward Primer","CCAGCCAGCTGATCAGAAGAC","32"]},{"entry":[{},{},"Reverse Primer","TCCGCTGAATTCCAAGCAAT","33"]},{"entry":{}},{"entry":["Ki-67","NM_002417","Probe","CACTCTCATCAGGGTCAG","34"]},{"entry":[{},"(Hs00267195_ml)","Forward Primer","CGTCGTGTCTCAAGATCTAGCTTCT","35"]},{"entry":[{},{},"Reverse Primer","GGACACACGCCTTCTTTTCAA","36"]},{"entry":{}},{"entry":["IRF4","NM_002460","Probe","CCCAGCAGGTTCAC","37"]},{"entry":[{},"(Hs00277069_ml)","Forward Primer","CTACACCATGACAACGCCTTACC","38"]},{"entry":[{},{},"Reverse Primer","GGCTGATCCGGGACGTAGT","39"]},{"entry":{}},{"entry":["MYC","NM_002467","Probe","TCCTCCTCAGAGTCGC","40"]},{"entry":[{},"(Hs00153408_ml)","Forward Primer","CCCCTGGTGCTCCATGAG","41"]},{"entry":[{},{},"Reverse Primer","GCCTGCCTCTTTTCCACAGA","42"]},{"entry":{}},{"entry":["PDE4B","NM_002600","Probe","TCGCATTCAGGTCCTT","43"]},{"entry":[{},"(Hs00277080_ml)","Forward Primer","CAGGCGTTCTTCTCCTAGACAACTA","44"]},{"entry":[{},{},"Reverse Primer","GGTCTGCACAGTGTACCATGTTG","45"]},{"entry":{}},{"entry":["PIK3CG","NM_002649","Probe","ATGTCCTGAAATTTC","46"]},{"entry":[{},"(Hs00277090_ml)","Forward Primer","GGAAAGAAGACAAGCCCACACTT","47"]},{"entry":[{},{},"Reverse Primer","GTGATGACGAAGGGCTAGATAAGC","48"]},{"entry":{}},{"entry":["SCYA3\/","NM_002983","Probe","CTGCATCACTTGCTGC","49"]},{"entry":["CCL3","(Hs00234142_ml)","Forward Primer","ATGGCTCTCTGCAACCAGTTCT","50"]},{"entry":[{},{},"Reverse Primer","GCGGTCGGCGTGTCA","51"]},{"entry":{}},{"entry":["SLAM","NM_003037","Probe","ACCTGCTCATAAAGC","52"]},{"entry":[{},"(Hs00234150_ml)","Forward Primer","CGCTTTTGCCTGCAGTTGA","53"]},{"entry":[{},{},"Reverse Primer","CGTTCTCCTGGGTCTTGTTTAAAA","54"]},{"entry":{}},{"entry":["WASPIP","NM_003387","Probe","CTCCAGAATCATTATCC","55"]},{"entry":[{},"(Hs00277097_ml)","Forward Primer","TGAGATCCACGGCCAACAG","56"]},{"entry":[{},{},"Reverse Primer","CTCCCGGTGGCAACAATG","57"]},{"entry":{}},{"entry":["CFLAR","NM_003879","Probe","AGACAGAGCTTCTTCG","58"]},{"entry":[{},"(Hs00153439_ml)","Forward Primer","TGCCTGATAATCGATTGCATTG","59"]},{"entry":[{},{},"Reverse Primer","CCCAGGGAAGTGAAGGTGTCT","60"]},{"entry":{}},{"entry":["LMO2","NM_005574","Probe","AGACTATCTCAGGCTTTT","61"]},{"entry":[{},"(Hs00277106_ml)","Forward Primer","CAAACTGGGCCGGAAGCT","62"]},{"entry":[{},{},"Reverse Primer","ATGCGCAGAGACCGTCTTG","63"]},{"entry":{}},{"entry":["LRMP","NM_006152","Probe","AAAGCTTCTTCAGTTTCC","64"]},{"entry":[{},"(Hs00277107_ml)","Forward Primer","GCCAGGCACTTCAGCAGAA","65"]},{"entry":[{},{},"Reverse Primer","GGACAGGGAATCATAGTGAGGAAAT","66"]},{"entry":{}},{"entry":["SLA","NM_006748","Probe","CCTGCAGTCTGGACAC","67"]},{"entry":[{},"(Hs00277129_ml)","Forward Primer","TCACCTTGCGTCAGAAGACTGT","68"]},{"entry":[{},{},"Reverse Primer","CCCAAGCGGGTTCTCTGTT","69"]},{"entry":{}},{"entry":["NR4A3","NM_006981","Probe","CATGATCACAGAAAGAC","70"]},{"entry":[{},"(Hs00175077_ml)","Forward Primer","GCCTGCCTGTCAGCACTGA","71"]},{"entry":[{},{},"Reverse Primer","GCTCTTCGACTCTCTTTGGTTCTTT","72"]},{"entry":{}},{"entry":["CD10","NM_007289","Probe","CCCAATAATCCTGAAATT","73"]},{"entry":[{},"(Hs00153519_ml)","Forward Primer","CTATAGGCCAGAGTATGCGGTTAAC","74"]},{"entry":[{},{},"Reverse Primer","GCAGTGAAAGGCTTCTGAAAACTC","75"]},{"entry":{}},{"entry":["PAX5","NM_016734","Probe","CCAGTGGACACTATGC","76"]},{"entry":[{},"(Hs00277134_ml)","Forward Primer","CAGTCCCAGCTTCCAGTCACA","77"]},{"entry":[{},{},"Reverse Primer","ACGAGCCGGCCGAATC","78"]},{"entry":{}},{"entry":["CD44","AJ251595.1","Probe","CAGCCATTCTGGAATT","79"]},{"entry":[{},"(NM_000610)","Forward Primer","GCAAACACAACCTCTGGTCCTATAA","80"]},{"entry":[{},{},"Reverse Primer","CCAAGAGGGATGCCAAGATG","81"]},{"entry":{}},{"entry":["HGAL\/","AF521911","Probe","CCATCCAGGACAATGT","82"]},{"entry":["GCET2","(NM_152785)","Forward Primer","CCCAAAACGAAAATGAAAGAATGT","83"]},{"entry":[{},"(Hs00277164_ml)","Reverse Primer","GGGTATAGCACAGCTCCTCTGAGTA","84"]},{"entry":{}},{"entry":["MYBL1\/","X66087.1","Probe","CAATTTGACTGAAGATTC","85"]},{"entry":["A-MYB","(Hs00277146_ml)","Forward Primer","AACCAAACCCTAACACTTCCAAAG","86"]},{"entry":[{},{},"Reverse Primer","TTCTGTCTTCCCATAAACCACTGTT","87"]},{"entry":{}},{"entry":["BCL7A","NM_020993","Probe","CTTGTTTTTGTCATCAACC","88"]},{"entry":[{},"(Hs00277139_ml)","Forward Primer","AATCTACAAATGGGTCCCTGTGA","89"]},{"entry":[{},{},"Reverse Primer","ACACTTCTCGTCCTTGCCTTTT","90"]},{"entry":{}},{"entry":["PLAU","NM_002658","Probe","CCTGCCAGGGAGAC","91"]},{"entry":[{},"(Hs00170182_ml)","Forward Primer","TCTGAAGTCACCACCAAAATGCT","92"]},{"entry":[{},{},"Reverse Primer","CGGCCTTGGAGGGAACA","93"]},{"entry":{}},{"entry":["NPM3","NM_006993","Probe","CTCATCGTAACAATCTG","94"]},{"entry":[{},"(Hs00199625_ml)","Forward Primer","GGCCCTGTGCGGATCAC","95"]},{"entry":[{},{},"Reverse Primer","CTCTCCTCCTCAGAAACATCATTG","96"]},{"entry":{}},{"entry":["HLA-DQA1","NM_002122","Probe","CACTGGGAGCCTGAG","97"]},{"entry":[{},"(N\/A)","Forward Primer","CCTGGACCAGCCTCTTCTGA","98"]},{"entry":[{},{},"Reverse Primer","CAGTCTCTGTGAGCTCTGACATAGG","99"]},{"entry":{}},{"entry":["HLA-DRA","NM_019111","Probe","CATCACCTCCATGTGC","100"]},{"entry":[{},"(Hs00219578_ml)","Forward Primer","GCAAAAGCAATGCAGCAGAAC","101"]},{"entry":[{},{},"Reverse Primer","GGATTGTAATATTGCCAGCTTTGTAA","102"]},{"entry":{}},{"entry":["FN1","NM_002026","Probe","ATCCAAGCTCAAGTGGTCC","103"]},{"entry":[{},"(Hs00365058_ml)","Forward Primer","CTATGGCCGTGGCATTGG","104"]},{"entry":[{},{},"Reverse Primer","GTGGGAGTTGGGCTGACT","105"]},{"entry":{}},{"entry":["EEF1A1L14","NM_001403","Probe","CCAAGGCATGTTAGCAC","106"]},{"entry":[{},"(Hs00265885_ml)","Forward Primer","TTTCTGGTTGGAATGGTGACAA","107"]},{"entry":[{},{},"Reverse Primer","GGGTGACTTTCCATCCCTTGA","108"]},{"entry":{}},{"entry":["GAPDH","XM_171437","Probe","CCCTGGTGACCAGGC","109"]},{"entry":[{},"(4326317E)","Forward Primer","AGCCGAGCCACATCGCT","110"]},{"entry":[{},{},"Reverse Primer","TGGCAACAATATCCACTTTACCAGAGT","111"]},{"entry":{}},{"entry":["PGK1","NM_000291","Probe","AAGGTTAAAGCCGAGCCA","112"]},{"entry":[{},"(4326318E)","Forward Primer","GGGAAAAGATGCTTCTGGGAA","113"]},{"entry":[{},{},"Reverse Primer","TTGGAAAGTGAAGCTCGGAAA","114"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}},{"entry":"*Assay I.D. No. of Applied Biosystems, Foster City, CA."}]}}]}}},"Expression of mRNA for 36 tested genes and 2 endogenous control genes was measured in each DLBCL specimen-with real time PCR using the Applied Biosystems Assays-on-Demand.TM. Gene Expression Products on an ABI PRISM\u00ae 7900HT Sequence Detection System (Applied Biosystems, Foster City, Calif.) as previously reported (Lossos et al., 17:789-95, 2003). For each gene, 2-4 assays (TaqMan\u00ae probe and primer sets) were tested. The probes contain a 6-carboxy-fluorescein phosphoramidite (FAM\u00aedye) label at the 5\u2032 end and a minor groove binder (MGB) and non-fluorescent quencher (NFQ) at the 3\u2032 end, and designed to hybridize across exon junctions. The assays are supplied with primers and probe concentrations of 900 nM and 250 nM, respectively. Real-time assays used in this study had high (near 100%) amplification efficiencies.","No fluorescent signal was generated by these assays when genomic DNA was used as a substrate, validating the assays as measuring mRNA only. The assays were highly reproducible with inter-run variance of less than 0.16 for all the genes. Phosphoglycerate kinase 1 (PGK1) and glyceraldehyde-3-phosphate dehydrogenase (GAPDH) were used as the endogenous RNA\/cDNA quantity controls (P\/N 4326318E and P\/N 4326317E, respectively Applied Biosystems, Foster City, Calif.). We chose PGK1 and GAPDH based on an analysis of their relatively constant expression in DLBCL tumors. Since the normalization to PGK1 and GAPDH endogenous control genes lead to similar results and conclusions, we present only the data normalized to PGK1 expression. For calibration and generation of standard curves we used Raji cDNA and\/or cDNA prepared from Universal Human Reference RNA (Stratagene, La Jolla, Calif.). The latter was used for genes with low abundance in Raji cell line (CCND1, CCND2, SLA, NR4A3, CD44, PLAU, and FN1). To control for possible variability between different PCR runs performed on different days, expression of all the analyzed and endogenous control genes was assessed in Raji cell line before, midway and upon completion of the analysis of all the experimental DLBCL specimens. The variance between these 3 runs for all the genes assessed in the Raji cell line was less than 0.16.","Calculation of normalized gene expression values was performed as follows. Cvalues measured from tumor samples were converted to quantity of RNA expressed in ng\/\u03bcl, by referencing to the standard curve for the gene. For each gene the ratio of the quantity expressed to the quantity of expression of the reference gene, GAPDH was then calculated. For each gene, the same ratio was determined for calibrator RNA obtained from Raji cells or from the Universal Standard Reference. Finally, the ratio obtained from the tumor sample was divided by the ratio obtained for the calibrator cells.","Gene expression values for each of the 36 genes and 66 patients is shown in Table 3.",{"@attributes":{"id":"p-0040","num":"0039"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Normalized Gene Expression Values Determined in"},{"entry":"Sixty-Six Patients Referenced to GAPDH and Raji"},{"entry":"CellsUnless Otherwise Indicated"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"182pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Normalized Expression Values"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"63pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},{},"Standard"]},{"entry":[{},"Gene","Mean","Variance","Deviation"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"63pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"ICAM1\/CD54","2.26","8.16","2.86"]},{"entry":[{},"PMSI","3.26","6.31","2.51"]},{"entry":[{},"p53","2.58","3.66","1.91"]},{"entry":[{},"BCL-2","21.23","840.02","28.98"]},{"entry":[{},"BIRC5\/SURVIVIN","1.31","0.93","0.96"]},{"entry":[{},"PRDM1","32.44","817.94","28.60"]},{"entry":[{},"BCL-6","5.62","117.79","10.85"]},{"entry":[{},"CCND1*","0.78","4.04","2.01"]},{"entry":[{},"CCND2*","4.18","57.02","7.55"]},{"entry":[{},"CD38","11.01","85.15","9.23"]},{"entry":[{},"CR2","2.05","17.25","4.15"]},{"entry":[{},"Ki-67","1.77","1.01","1.00"]},{"entry":[{},"IRF4","49.66","4641.85","68.13"]},{"entry":[{},"MYC","2.07","7.69","2.77"]},{"entry":[{},"PDE4B","36.30","1238.24","35.19"]},{"entry":[{},"PIK3CG","9.20","56.19","7.50"]},{"entry":[{},"SCYA3","9.72","158.87","12.60"]},{"entry":[{},"SLAM","1.01","1.23","1.11"]},{"entry":[{},"WASPIP","6.95","42.93","6.55"]},{"entry":[{},"CFLAR","23.53","1800.59","42.43"]},{"entry":[{},"LMO2","7.34","62.07","7.88"]},{"entry":[{},"LRMP","3.90","8.12","2.85"]},{"entry":[{},"SLA*","108.59","23782.23","154.21"]},{"entry":[{},"NR4A3*","8.41","97.50","9.87"]},{"entry":[{},"CD10","1.51","3.38","1.84"]},{"entry":[{},"PAX5","6.46","176.48","13.28"]},{"entry":[{},"M17","1.87","3.11","1.76"]},{"entry":[{},"MYBL1\/A-MYB","3.72","17.33","4.16"]},{"entry":[{},"BCL7A","2.44","4.31","2.08"]},{"entry":[{},"CD44 (139)*","5.13","15.53","3.94"]},{"entry":[{},"PLAU*","6.51","99.49","9.97"]},{"entry":[{},"NPM3","1.49","2.28","1.51"]},{"entry":[{},"HLA-DQA1","2.91","7.19","2.68"]},{"entry":[{},"EEF1A1L4","1.42","0.70","0.84"]},{"entry":[{},"HLA-DRA","4.43","11.52","3.39"]},{"entry":[{},"FN1*","2.46","14.57","3.82"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"4","align":"center","rowsep":"1"}}]},{"entry":[{},"*Referenced to Stratagene Universal Reference RNA."]}]}}]}}},"This example illustrates the statistical evaluation for developing a survival predictive model.","The normalized gene expression values were log-transformed (base 2) similar to what is done with hybridization array data.","Overall survival time of DLBCL patients was calculated from the date of the diagnosis until death or last follow-up examination. Survival curves were estimated using the product-limit method of Kaplan-Meier and were compared using the log-rank test.","To determine a small list of genes whose expression segregated DLBCL tumors into subgroups with distinct overall survival, we performed a univariate Cox proportional hazards analysis with the overall survival as the dependent variable. Genes with an absolute univariate Cox score >1.5 or <\u22121.5 were analyzed by a multivariate regression analysis (with and without IPI components) using a Cox proportional hazards regression model with overall survival as the dependent variable. This same model was used to adjust the effects of gene expressions for IPI. p values<0.05 were considered to be significant. Backward stepwise analysis was also used, to find the minimal set of genes that were predictive. A p-value cutoff of 0.05 was used for deletion of model terms.","Results of the univariate analysis are shown in . The genes were ranked based upon their predictive power (univariate score) with negative score associated with longer overall survival while positive univariate score associated with shorter overall survival. Six genes with absolute univariate score >1.5 (LMO2, BCL-6, FN1, CCND2, SCYA3 and BCL-2) were selected for further analysis. On multivariate Cox regression analysis with DLBCL overall survival as a dependent variable, none of these genes independently predicted overall survival at a statistically significant level, however on backward stepwise analysis, expression of LMO2 correlated with DLBCL overall survival (p=0.011). Multivariate Cox regression analysis incorporating all the components of IPI together with the expression of these 6 genes disclosed that only LDH was an independent predictor of DLBCL overall survival (p=0.0038). However, on backward stepwise analysis, both LDH and LMO2 expression were independent predictors of DLBCL overall survival (p=0.0035 and p=0.025, respectively).","Since this analysis established an inter-correlation between the expressions of these 6 genes and survival, we constructed a model based on a weighted predictor derived from the relative contributions of each gene in the multivariate analysis. The weighted predictor (Z) was calculated for each tumor specimen and the tumors were ranked into 3 tertiles: low, medium and high using the \u22120.63 and 0.093 as cut points (<\u22120.063\u2014low risk, between \u22120.063 to <0.093, medium risk and >0.093\u2014high risk groups). The overall survival of these 3 groups was significantly different (p=0.004) with 5-year survival of 65%, 49% and 15% for the low, medium and high groups, respectively (mean overall survival [95% confidence interval] of 7.1{5.4\u2014not achieved}, 9.0{1.1\u2014not achieved} and 4.5{1.2-4.3} years, respectively, ). Consequently, patients with tumors expressing high levels of LMO2, BCL-6 and FN1 and low levels of CCND2, SCYA3 and BCL-2, survived longer.","For construction of the survival prediction model, we derived the weighted predictor (Z) from the multivariate analysis for each of the six genes:\n\n(\u22120.02732)+(\u22120.2103-6)+(\u22120.18781)+\n\n(0.03462)+(0.18883)+(0.5527-2).\n","Thus for example the negative weight on LMO2 means that higher expression correlates with lower risk (longer survival). The positive weight on CCND2 means that higher expression correlates with higher risk (shorter survival).","This example illustrates the validation of the survival prediction model.","To validate the usefulness of the model derived in Example 2, the model was applied to two independent previously published DLBCL gene expression data sets derived from DNA microarray methodology (Shipp et al., supra, 2003; Rosenwald et al., supra, 2003). Application of the 6 gene prediction model to data from Shipp et al. (Shipp et al., supra, 2003)() and to that of Rosenwald et al. (Rosenwald et al., supra, 2002) () confirmed its ability to predict survival since it could stratify DLBCL cases into 3 subgroups with statistically significant different overall survival (P=0.03 and P=0.0004, respectively). Although in the smaller DLBCL cohort reported by Shipp et al., the overall survival of the group in the medium tertile was similar at the 5 year point to that of their high risk tertile, this medium tertile did have an intermediate risk in the larger cohort of patients analyzed by Rosenwald et al. (Rosenwald et al., supra, 2002) ().","We next analyzed whether this prediction model could add to the prognostic value of the IPI. In our own series of 66 patients there were not enough patients in the lowest risk IPI group to achieve statistical significance. But in our patients within the high clinical risk IPI group, the six gene expression model could further subdivide the patients in respect to survival (P=0.006) (data not shown). We, therefore, tested the model on the larger DLBCL data set derived from microarray analysis reported by Rosenwald et al. (Rosenwald et al., supra, 2002) (). We used their same three subdivisions of the patients according to the IPI (low, medium and high risk). Within each of these subgroups we further divided the patients according to the 6 gene expression model. In some of these groups the patients numbers were limited. But in each IPI strata we could identify an especially poor surviving group ( blue lines). By combining the lowest surviving tertiles from the medium and high risk IPI strata, then we identify 30% of all patients that receive very little benefit from current therapy.","The present study defined and validated across the published studies a small set of genes whose expression can predict DLBCL survival and which can be measured by a clinically applicable method. To this end, we evaluated side-by side the prognostic significance of 36 representative genes chosen based on the previous reports suggesting their prognostic potential or from our own analysis of the existing microarray data (Table 1). We have designed a prediction model of overall survival consisting of 6 genes that subdivided DLBCL patients into three prognostic groups in our series of 66 patients and in independent groups of 58 and 240 DLBCL tumors analyzed by Shipp et al. (Shipp et al., supra, 2002) and Rosenwald et al. (Rosenwald et al., supra, 2002), respectively. The validation of our model did not require any adjustments of the published microarray data or any refinements of our gene list. Moreover, this model could further sub-classify DLBCL patients within IPI strata into longer- and shorter-term survivors. The genes comprising this model are present in each of the previously denoted lymphocyte signatures such as germinal (LMO2 and BCL-6), activated B cell (BCL-2, CCND2, SCYA3) and lymph node signatures (FN1) (Alizadeh et al., supra, 2000; Rosenwald et al., supra, 2002). However, the model is independent of these signatures and several genes associated with these signatures do not carry predictive power in our model.","LMO2, BCL-6 and FN1 were the genes whose expression correlated with prolonged survival. LMO2 was first discovered by its homology with the T cell oncogene LMO1 (Boehm et al., 88:4367-71, 1991). It plays an important role in erythropoiesis and angiogenesis presumably through transcriptional regulation (Warren et al., 1994; 78:45-57, 1994; Yamada et al., 97:320-4, 2000). The LMO2 locus on chromosome 11p13 is the most frequent site of chromosomal translocation in childhood T-cell acute lymphoblastic leukemia (Boehm et al., supra, 1991). LMO2 is expressed in myeloid and erythroid precursors of hematopoietic system and its expression decreases during differentiation. LMO2 expression is low in resting peripheral B cells, however it is markedly increased in GC lymphocytes (Alizadeh et al., supra, 2000). LMO2 is not expressed in normal T lymphocytes, however following chromosomal translocation, its ectopic expression in thymocytes contributes to the leukemogenesis (Royer-Pokora et al., 6:1887-93, 1991). Interestingly, in two recently observed cases of leukemia complicating retrovirus based gene therapy of X-linked severe combined immunodeficiency, the vector inserted itself near the LMO2 gene (Kaiser, 299:495, 2003). Neither the functional significance of increased LMO2 expression in GCB lymphocytes nor its potential role in GCB-derived tumors is known.","The BCL-6 gene, identified by virtue of its involvement in chromosomal translocations affecting band 3q27, encodes a POZ\/Zinc finger sequence-specific transcriptional repressor (Chang et al., 93:6947-52, 1996; Kerckaert et al., 1993; 5:66-70, 1993; Seyfert et al., 1996; 12:2331-42, 1996). The BCL-6 gene is normally expressed in B and CD4+T cells within the germinal center (GC), and it controls GC formation and T-cell-dependent antigen responses (Cattoretti et al., 86:45-53, 1995; Dent et al., 95:13823-8, 1998; Ye et al., 16:161-70, 1997). It is considered one of the hallmarks of the GC and is expressed in NHL whose origin is from GCB lymphocytes. BCL-6 expression was previously reported to predict DLBCL outcome (Lossos et al., 98:945-951, 2001).","FN1 is a component of extracellular matrix in the lymph-node signature. Its expression may reflect the response of the lymph node to the tumor cells. Indeed, some cases of DLBCL demonstrate a sclerotic reaction. This gene, together with BCL-6, was included in the survival prediction model constructed by Rosenwald et al. (Rosenwald et al., supra, 2002).","In contrast to these 3 genes, expression of BCL-2, CCND2, SCYA3 correlated with short survival. All of these 3 genes are included in the ABC-like signature (Alizadeh et al., supra, 2000). BCL-2 protein expression is down-regulated in normal GCB cells, but is frequently up-regulated in NHL by virtue of t(14;18) translocation (Alizadeh et al., supra, 2000; Kramer et al., 92:3152-62). Overexpression of the BCL-2 protein is known to prevent apoptosis. High BCL-2 protein expression has been repeatedly shown to be an independent poor prognostic indicator for DLBCL (Gascoyne et al., 90:244-51, 1997; Kramer et al., 14:2131-8; Hermine et al., 87:265-72, 1996; Hill et al., 88:1046-51,1996).","CCND2 encodes a protein that belongs to the highly conserved cyclin family, whose members are characterized by a dramatic periodicity in protein abundance through cell cycle. This cyclin forms a complex with CDK4 or CDK6 and regulates their activity thus controlling the cell cycle G1\/S transition. Consequently, its expression may be associated with higher proliferation rates of the tumors. SCYA3 is a CC chemokine that recruits inflammatory cells, including lymphocytes, monocytes, eosinophils and mast cells to sites of inflammation (Proost et al., 26:211-23, 1996). Its function in B cell lymphomas is unknown, but it is mainly expressed in the ABC-like group of DLBCL tumors and its expression in lymphocytes can be induced by B cell receptor stimulation (Alizadeh et al., supra, 2000). Interestingly, the promoter regions of both CCND2 and SCYA3 genes contain high-affinity BCL-6 binding sites and the expression of these two genes is repressed by BCL-6 (Shaffer et al., 13:199-212, 2000). This observation underscores the complex interrelation between the expression of individual genes singularly implicated in DLBCL prognosis (e.g. HGAL) (Lossos et al., 101:433-40, 2003), however not contributing to the model based on multivariate analysis.","All references cited in this specification are hereby incorporated by reference. Any discussion of references cited herein is intended merely to summarize the assertions made by their authors and no admission is made that any reference or portion thereof constitutes relevant prior art. Applicants reserve the right to challenge the accuracy and pertinency of the cited references.","The description of the invention is merely exemplary in nature and, thus, variations that do not depart from the gist of the invention are intended to be within the scope of the invention. Such variations are not to be regarded as a departure from the spirit and scope of the invention."],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0015","num":"0014"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0016","num":"0015"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0017","num":"0016"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0018","num":"0017"},"figref":"FIG. 4"}]},"DETDESC":[{},{}]}
