---
title: Method for spooling experimental data
abstract: Methods and system for efficient collection and storage of experimental data allow experimental data from high-throughput, feature-rich data collection systems, such as high-throughput cell data collection systems to be efficiently collected, stored, managed and displayed. The methods and system can be used, for example, for storing, managing, and displaying cell image data and cell feature data collected from microplates including multiple wells and a variety of bio-chips in which an experimental compound has been applied to a population of cells. The methods and system provide a flexible and scalable repository of experimental data including multiple databases at multiple locations including pass-through databases that can be easily managed and allows cell data to be analyzed, manipulated and archived. The methods and system may improve the identification, selection, validation and screening of new drug compounds that have been applied to populations of cells.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08024293&OS=08024293&RS=08024293
owner: Cellomics, Inc.
number: 08024293
owner_city: Pittsburgh
owner_country: US
publication_date: 20080808
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATIONS","FIELD OF THE INVENTION","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF PREFERRED EMBODIMENTS","Exemplary Data Storage System"],"p":["This application a divisional of U.S. application Ser. No. 10\/649,323, filed on Aug. 27, 2003, which is a continuation of U.S. application Ser. No. 09\/437,976, filed Nov. 10, 1999, abandoned, which claims the benefit of U.S. Provisional Application No. 60\/108,291, filed on Nov. 13, 1998; 60\/110,643, filed on Dec. 1, 1998; 60\/140,240, filed on Jun. 21, 1999; 60\/142,375, filed on Jul. 6, 1999; and 60\/142,646 filed on Jul. 6, 1999, which applications are incorporated herein by specific reference.","This invention relates to collecting and storing experimental data. More specifically, it relates to methods and system for efficient collection and storage of experimental data from automated feature-rich, high-throughput experimental data collection systems.","Historically, the discovery and development of new drugs has been an expensive, time consuming and inefficient process. With estimated costs of bringing a single drug to market requiring an investment of approximately 8 to 12 years and approximately $350 to $500 million, the pharmaceutical research and development market is in need of new technologies that can streamline the drug discovery process. Companies in the pharmaceutical research and development market are under fierce pressure to shorten research and development cycles for developing new drugs, while at the same time, novel drug discovery screening instrumentation technologies are being deployed, producing a huge amount of experimental data.","Innovations in automated screening systems for biological and other research are capable of generating enormous amounts of data. The massive volumes of feature-rich data being generated by these systems and the effective management and use of information from the data has created a number of very challenging problems. As is known in the art, \u201cfeature-rich\u201d data includes data wherein one or more individual features of an object of interest (e.g., a cell) can be collected. To fully exploit the potential of data from high-volume data generating screening instrumentation, there is a need for new informatic and bioinformatic tools.","Identification, selection, validation and screening of new drug compounds is often completed at a nucleotide level using sequences of Deoxyribonucleic Acid (\u201cDNA\u201d), Ribonucleic Acid (\u201cRNA\u201d) or other nucleotides. \u201cGenes\u201d are regions of DNA, and \u201cproteins\u201d are the products of genes. The existence and concentration of protein molecules typically help determine if a gene is \u201cexpressed\u201d or \u201crepressed\u201d in a given situation. Responses of genes to natural and artificial compounds are typically used to improve existing drugs, and develop new drugs. However, it is often more appropriate to determine the effect of a new compound on a cellular level instead of a nucleotide level.","Cells are the basic units of life and integrate information from DNA, RNA, proteins, metabolites, ions and other cellular components. New compounds that may look promising at a nucleotide level may be toxic at a cellular level. Florescence-based reagents can be applied to cells to determine ion concentrations, membrane potentials, enzyme activities, gene expression, as well as the presence of metabolites, proteins, lipids, carbohydrates, and other cellular components.","There are two types of cell screening methods that are typically used: (1) fixed cell screening; and (2) live cell screening. For fixed cell screening, initially living cells are treated with experimental compounds being tested. No environmental control of the cells is provided after application of a desired compound and the cells may die during screening. Live cell screening requires environmental control of the cells (e.g., temperature, humidity, gases, etc.) after application of a desired compound, and the cells are kept alive during screening. Fixed cell assays allow spatial measurements to be obtained, but only at one point in time. Live cell assays allow both spatial and temporal measurements to be obtained.","The spatial and temporal frequency of chemical and molecular information present within cells makes it possible to extract feature-rich cell information from populations of cells. For example, multiple molecular and biochemical interactions, cell kinetics, changes in sub-cellular distributions, changes in cellular morphology, changes in individual cell subtypes in mixed populations, changes and sub-cellular molecular activity, changes in cell communication, and other types of cell information can be obtained.","The types of biochemical and molecular cell-based assays now accessible through fluorescence-based reagents is expanding rapidly. The need for automatically extracting additional information from a growing list of cell-based assays has allowed automated platforms for feature-rich assay screening of cells to be developed. For example, the ArrayScan System by Cellomics, Inc. of Pittsburgh, Pa., is one such feature-rich cen\u2022screening system. Cell based systems such as FLIPR, by Molecular Devices, Inc. of Sunnyvale, Calif., FMAT, of PE Biosystems of Foster City, Calif., ViewLux by EG&G Wallac, now a subsidiary of Perkin-Elmer Life Sciences of Gaithersburg, Md., and others also generate large amounts of data and photographic images that would benefit from efficient data management solutions. Photographic images are typically collected using a digital camera. A single photographic image may take up as much as 512 Kilobytes (\u201cKB\u201d) or more of storage space as is explained below. Collecting and storing a large number of photographic images adds to the data problems encountered when using high throughput systems. For more information on fluorescence based systems, see \u201cBright ideas for high-throughput screening\u2014One-step fluorescence HTS assays are getting faster, cheaper, smaller and more sensitive,\u201d by Randy Wedin, Modern Drug Discovery, Vol. 2(3), pp. 61-71, May\/June 1999.","Such automated feature-rich cell screening systems and other systems known in the art typically include microplate scanning hardware, fluorescence excitation of cells, fluorescence captive emission optics, a photographic microscopic with a camera, data collection, data storage and data display capabilities. For more information on feature-rich cell screening see \u201cHigh Content Fluorescence-Based Screening,\u201d by Kenneth A. Guiliano, et al., Journal of Biomolecular Screening, Vol. 2, No. 4, pp. 249-259, Winter 1997, ISSN 1087-0571, \u201cPTH Receptor Internalization,\u201d Bruce R. Conway, et al., Journal of Biomolecular Screening, Vol. 4, No. 2, pp. 75-68, April 1999, ISSN 1087-0571, \u201cFluorescent-Protein Biosensors: New Tools For Drug Discovery,\u201d Kenneth A. Giuliano and D. Lansing Taylor, Trends in Biotechnology, (\u201cTIBTECH\u201d), Vol. 16, No. 3, pp. 99-146, March 1998, ISSN 0167-7799, all of which are incorporated by reference.","An automated feature-rich cell screening system typically automatically scans a microplate plate with multiple wells and acquires multi-color fluorescence data of cells at one or more instances of time at a pre-determined spatial resolution. Automated feature-rich cell screen systems typically support multiple channels of fluorescence to collect multi-color fluorescence data at different wavelengths and may also provide the ability to collect cell feature information on a cell-by-cell basis including such features as the size and shape of cells and sub-cellar measurements of organelles within a cell.","The collection of data from high throughput screening systems typically produces a very large quantity of data and presents a number of bioinformatics problems. As is known in the art, \u201cbioinformatic\u201d techniques are used to address problems related to the collection, processing, storage, retrieval and analysis of biological information including cellular information. Bioinformatics is defined as the systematic development and application of information technologies and data processing techniques for collecting, analyzing and displaying data obtained by experiments, modeling, database searching, and instrumentation to make observations about biological processes.","The need for efficient data management is not limited to feature-rich cell screening systems or to cell based arrays. Virtually any instrument that runs High Throughput Screening (\u201cRTS\u201d) assays also generate large amounts of data. For example, with the growing use of other data collection techniques such as DNA arrays, bio-chips, microscopy, micro-arrays, gel analysis, the amount of data collected, including photographic image data is also growing exponentially. As is known in the art, a \u201cbio-chip\u201d is a stratum with hundreds or thousands of absorbent micro-gels fixed to its surface. A single bio-chip may contain 10,000 or more micro-gels. When performing an assay test, each micro-gel on a bio-chip is like a micro-test tube or a well in a microplate. A bio-chip provides a medium for analyzing known and unknown biological (e.g., nucleotides, cells, etc.) samples in an automated, high-throughput screening system.","Although a wide variety of data collection techniques can be used, cell-based high throughput screening systems are used as an example to illustrate some of the associated data management problems encountered by virtually all high throughput screening systems. One problem with collecting feature-rich cell data is that a microplate plate used for feature-rich screening typically includes 96 to 1536 individual wells. As is known in the art, a \u201cmicroplate\u201d is a flat, shallow dish that stores multiple samples for analysis. A \u201cwell\u201d is a small area in a microplate used to contain an individual sample for analysis. Each well may be divided into multiple fields. A \u201cfield\u201d is a sub-region of a well that represents a field of vision (i.e., a zoom level) for a photographic microscope. Each well is typically divided into one to sixteen fields. Each field typically will have between one and six photographic images taken of it, each using a different light filter to capture a different wavelength of light for a different fluorescence response for desired cell components. In each field, a pre-determined number of cells are selected to analyze. The number of cells will vary (e.g., between ten and one hundred). For each cell, multiple cell features are collected. The cell features may include features such as size, shape, etc. of a cell. Thus, a very large amount of data is typically collected for just one well on a single microplate.","From a data volume perspective, the data to be saved for a well can be estimated by number of cell feature records collected and the number of images collected. The number of images collected can be typically estimated by: (number of wells\u00d7number of fields\u00d7images per field). The current size of an image file is approximately 512 Kilobytes (\u201cKB\u201d) of uncompressed data. As is known in the art, a byte is 8-bits of data. The number of cell feature records can typically be estimated by: (number of wells\u00d7number of fields\u00d7cells per field\u00d7features per cell). Data collected from multiple wells on a microplate is typically formatted and stored on a computer system. The collected data is stored in format that can be used for visual presentation software, and allow for data mining and archiving using bioinformatic techniques.","For example, in a typical scenario, scanning one low density microplate with 96 wells, using four fields per well, three images per field and an image size of 512 Kbytes per image, generates about 1,152 images and about 576 megabytes (\u201cMB\u201d) of image data (i.e., (96\u00d74\u00d73\u00d7512\u00d7(1 KB=1024 bytes)\/(1 MB=(1024 bytes\u00d71024 bytes))=576 MB). As is known in the art, a megabyte is 220 or 1,048,576 bytes and is commonly interpreted as \u201cone million bytes.\u201d","If one hundred cells per field are selected with ten features per cell calculated, such a scan also generates (96\u00d74\u00d7100\u00d710)=288,000 cell feature records, whose data size varies with the amount of cell features collected. This results in about 12,000 MB of data being generated per day and about 60,000 MB per week, scanning the 96 well microplates twenty hours a day, five days a week.","In a high data volume scenario based on a current generation of feature-rich cell screening systems, scanning one high-density microplate with 384 wells, using sixteen fields per well, four images per field, 100 cells per field, ten features per cell, and 512 KB per image, generates about 24,576 images or about 12,288 MB of image data and about 6,144,000 cell feature records. This results in about 14,400 MB of data being generated per day and about 100,800 MB per week, scanning the 384 well microplates twenty-four hours a day, seven days a week.","Since multiple microplates can be scanned in parallel, and multiple automated feature-rich cell screening systems can operate 24 hours a day, seven days a week, and 365 days a year, the experimental data collected may easily exceed physical storage limits for a typical computer network. For example, disk storage on a typical computer network may be in the range from about ten gigabytes (\u201cGB\u201d) to about one-hundred GB of data storage. As is known in the art, a gigabyte is 230 bytes, or 1024 MB and is commonly interpreted as \u201cone billion bytes.\u201d","The data storage requirements for using automated feature-rich cell screening on a conventional computer network used on a continuous basis could easily exceed a terabyte (\u201cTB\u201d) of storage space, which is extremely expensive based on current data storage technologies. As is known in the art, one terabyte equals 240 bytes, and is commonly interpreted as \u201cone trillion bytes.\u201d Thus, collecting and storing data from an automated feature-rich cell screening system may severely impact the operation and storage of a conventional computer network.","Another problem with feature-rich cell screening systems is even though a massive amount of cell data is collected, only a very small percentage of the total cell feature data and image data collected will ever be used for direct visual display. Nevertheless, to gather statistically relevant information about a new compound all of the cell data generated, is typically stored on a local hard disk and available for analysis. This may also severely impact a local hard disk storage.","Yet another problem is that microplate scan results information for one microplate can easily exceed about 1,000 database records per plate, and cell feature data and image data can easily exceed about 6,000,000 database records per plate. Most conventional databases used on personal computers can not easily store and manipulate such a large number of data records. In addition, waiting relatively long periods of time to open such a large database on a conventional computer personal computer to query and\/or display data may severely affect the performance of a network and may quickly lead to user frustration or user dissatisfaction.","Thus, it is desirable to provide a data storage system that can be used for feature-rich screening on a continuous basis. The data storage system should provide a flexible and scalable repository of cell data that can be easily managed and allows data to be analyzed, manipulated and archived.","In accordance with preferred embodiments of the present invention, some of the problems associated with collecting and storing feature-rich experimental data are overcome. Methods and system for efficient collection and storage of experimental data is provided. One aspect of the present invention includes a method for collecting experimental data. The method includes collecting image and feature data from desired sub-containers within a container. The image and feature data is stored in multiple image and feature databases. Summary data calculated for the desired sub-containers and the container are stored in sub-container and container databases.","Another aspect of the present invention includes a method for storing experimental data on a computer system. The method includes collecting image data and feature data from desired sub-containers in a container. The image and feature data is stored in multiple third databases comprising multiple database tables. Summary data calculated for desired sub-containers and the container is stored in a second database comprising multiple database tables. A first database is created that is a \u201cpass-through\u201d database. The first database includes a pass-through database table with links to the second database and links to the multiple third databases, but does not include any data collected from the container.","Another aspect of the present invention includes a method for spooling experimental data off devices that collect the data to a number of different remote storage locations. Links in a pass-through database table in a first database are updated to reflect the new locations of second database and multiple third databases.","Another aspect of the present invention includes a method for hierarchical management of experimental data. A pre-determined storage removal policy is applied to database files in a database. If any database files match the pre-determined storage removal policy, the database files are copied into a layer in a multi-layered hierarchical storage management system. The original database files are replaced with placeholder files that include a link to the original database files in the layer in the hierarchical storage management system.","Another aspect of the invention includes presenting the experimental data from a display application on a computer. The data presented by the display application is obtained from multiple databases obtained from multiple locations remote to the computer. The data displayed appears to be obtained from databases on local storage on the computer instead of from the remote locations.","Another aspect of the invention includes a data storage system that provides virtually unlimited amounts of \u201cvirtual\u201d disk space for data storage at multiple local and remote storage locations for storing experimental data that is collected.","These methods and system may allow experimental data from high-throughput data collection systems to be efficiently collected, stored, managed and displayed. For example, the methods and system can be used for, but is not limited to, storing managing and displaying cell image data and cell feature data collected from microplates including multiple wells or bio-chips including multiple micro-gels in which an experimental compound has been applied to a population of cells.","The methods and system may provide a flexible and scalable repository of experimental data that can be easily managed and allows the data to be analyzed, manipulated and archived. The methods and system may improve the identification, selection, validation and screening of new experimental compounds (e.g., drug compounds). The methods and system may also be used to provide new bioinformatic techniques used to make observations about experimental data.","The foregoing and other features and advantages of preferred embodiments of the present invention will be more readily apparent from the following detailed description. The detailed description proceeds with references to the accompanying drawings.",{"@attributes":{"id":"p-0048","num":"0047"},"figref":"FIG. 1A","b":["10","10","12","18","24","30","40","12","18","24","12","30","40","10"]},{"@attributes":{"id":"p-0049","num":"0048"},"figref":["FIG. 1B","FIG. 1B"],"b":["10","10","12","14","16","18","20","22","24","26","28","30","32","34","36","30","30","38","10","40","10"]},"In one exemplary preferred embodiment of the present invention, data storage system \u2032 includes the following specific components. However, the present invention is not limited to these specific components and other similar or equivalent components may also be used. Analysis instruments , , , comprise a feature-rich array scanning system capable of collecting and\/or analyzing experimental data such as cell experimental data from microplates, DNA arrays or other chip-based or bio-chip based arrays. Bio-chips include any of those provided by Motorola Corporation of Schaumburg, Ill., Packard Instrument, a subsidiary of Packard BioScience Co. of Meriden, Conn., Genometrix, Inc. of Woodlands, Tex., and others.","Analysis instruments , ,  include any of those provided by Cellomics, Inc. of Pittsburgh, Pa., Aurora Biosciences Corporation of San Diego, Calif., Molecular Devices, Inc. of Sunnyvale, Calif., PE Biosystems of Foster City, Calif., Perkin-Elmer Life Sciences of Gaithersburg, Md., and others. The one or more data store client computers, , , , are conventional personal computers that include a display application that provides a Graphical User Interface (\u201cGUI\u201d) to a local hard disk, the shared database , the data store server  and\/or the data store archive . The Gill display application is used to lead a scientist or lab technician through standard analyses, and supports custom and query viewing capabilities. The display application Gill also supports data exported into standard desktop tools such as spreadsheets, graphics packages, and word processors.","The data store client computers , ,  connect to the store server  through an Open Data Base Connectivity (\u201cODBC\u201d) connection over network . In one embodiment of the present invention, computer network  is a 100 Mega-bit (\u201cMbit\u201d) per second or faster Ethernet, Local Area Network (\u201cLAN\u201d). However, other types of LANs could also be used (e.g., optical or coaxial cable networks). In addition, the present invention is not limited to these specific components and other similar components may also be used.","As is known in the art, OBDC is an interface providing a common language for applications to gain access to databases on a computer network. The store server  controls the storage based functions plus an underlying Database Management System (\u201cDBMS\u201d).","The shared database  is a multi-user, multi-view relational database that stores summary data from the one or more analysis instruments , , . The shared database  uses standard relational database tools and structures. The data store archive  is a library of image and feature database files. The data store archive  uses Hierarchical Storage Management (\u201cHSM\u201d) techniques to automatically manage disk space of analysis instruments , ,  and the provide a multi-layer hierarchical storage management system. The HSM techniques are explained below.","An operating environment for components of the data storage system  and \u2032 for preferred embodiments of the present invention include a processing system with one or more high-speed Central Processing Unite(s) (\u201cCPU\u201d) and a memory. In accordance with the practices of persons skilled in the art of computer programming, the present invention is described below with reference to acts and symbolic representations of operations or instructions that are performed by the processing system, unless indicated otherwise. Such acts and operations or instructions are referred to as being \u201ccomputer-executed\u201d or \u201cCPU executed.\u201d","It will be appreciated that acts and symbolically represented operations or instructions include the manipulation of electrical signals by the CPU. An electrical system represents data bits which cause a resulting transformation or reduction of the electrical signals, and the maintenance of data bits at memory locations in a memory system to thereby reconfigure or otherwise alter the CPUs operation, as well as other processing of signals. The memory locations where data bits are maintained are physical locations that have particular electrical, magnetic, optical, or organic properties corresponding to the data bits.","The data bits may also be maintained on a computer readable medium including magnetic disks, optical disks, organic memory, and any other volatile (e.g., Random Access Memory (\u201cRAM\u201d)) or non-volatile (e.g., Read-Only Memory (\u201cROM\u201d)) mass storage system readable by the CPU. The computer readable medium includes cooperating or interconnected computer readable medium, which exist exclusively on the processing system or be distributed among multiple interconnected processing systems that may be local or remote to the processing system.","Array Scan Module Architecture",{"@attributes":{"id":"p-0058","num":"0057"},"figref":["FIG. 2","FIG. 1B"],"b":["42","42","12","14","16","44","46","46","44","44","48","48","18","20","22","48","50","50"]},"Collection of Experimental Data",{"@attributes":{"id":"p-0059","num":"0058"},"figref":["FIGS. 3A and 3B","FIG. 3A"],"b":["52","54","56","58","60","62","64","66","68","70","72","60","62","64","66"]},"In  at Step , the feature data is stored in a feature database. In one embodiment of the present invention, the image database and feature databases are combined into a single database comprising multiple tables including the image and feature data. In another embodiment of the present invention, the image database and feature databases are maintained as separate databases.","At Step , sub-container summary data is calculated. At Step , the sub-container summary data is stored in a sub-container database. In one embodiment of the present invention, the sub-container database and the container database are combined into a single database comprising multiple tables including the sub-container and container summary data. In another embodiment of the present invention, the sub-container and container databases are maintained as separate databases. The loop continues at Step  () until the desired sub-containers within a container have been analyzed. After the desired sub-containers have been processed in the container, the loop at Step  ends.","At Step  of , container summary data is calculated using sub-container summary data from the sub-container database. At Step , the container summary data is stored in the container database.","In a general use of the invention, at Step  features from any imaging-based analysis system can be used. Given a digitized image including one or more objects (e.g., cells), there are typically two phases to analyzing an image and extracting feature data as feature measurements. The first phase is typically called \u201cimage segmentation\u201d or \u201cobject isolation,\u201d in which a desired object is isolated from the rest of the image. The second phase is typically called \u201cfeature extraction,\u201d wherein measurements of the objects are calculated. A \u201cfeature\u201d is typically a function of one or more measurements, calculated so that it quantifies a significant characteristic of an object. Typical object measurements include size, shape, intensity, texture, location, and others.","For each measurement, several features are commonly used to reflect the measurement. The \u201csize\u201d of an object can be represented by its area, perimeter, boundary definition, length, width, etc. The \u201cshape\u201d of an object can be represented by its rectangularity (e.g., length and width aspect ratio), circularity (e.g., perimeter squared divided by area, bounding box, etc.), moment of inertia, differential chain code, Fourier descriptors, etc. The \u201cintensity\u201d of an object can be represented by a summed average, maximum or minimum grey levels of pixels in an object, etc. The \u201ctexture\u201d of an object quantifies a characteristic of grey-level variation within an object and can be represented by statistical features including standard deviation, variance, skewness, kurtosis and by spectral and structural features, etc. The \u201clocation\u201d of an object can be represented by an object's center of mass, horizontal and vertical extents, etc. with respect to a pre-determined grid system. For more information on digital image feature measurements, see: \u201cDigital Image Processing,\u201d by Kenneth R. Castleman, Prentice-Hall, 1996, ISBN-0132114674, \u201cDigital Image Processing: Principles and Applications,\u201d by G. A. Baxes, Wiley, 1994, ISBN-0471009490, \u201cDigital Image Processing,\u201d by William K. Pratt, Wiley and Sons, 1991, ISBN-0471857661, or \u201cThe Image Processing Handbook\u20142nd Edition,\u201d by John C. Russ, CRC Press, 1991, ISBN-0849325161, the contents of all of which are incorporated by reference.","In one exemplary preferred embodiment of the present invention, Method  is used to collect cell image data and cell feature data from wells in a \u201cmicroplate.\u201d In another preferred embodiment of the present invention, Method  is used to collect cell image and cell feature data from micro-gels in a bio-chip. As is known in the art, a \u201cmicroplate\u201d is a flat, shallow dish that stores multiple samples for analysis and typically includes 96 to 1536 individual wells. A \u201cwell\u201d is a small area in a microplate used to contain an individual sample for analysis. Each well may be divided into multiple fields. A \u201cfield\u201d is a sub-region of a well that represents a field of vision (i.e., a zoom level) for a photographic microscope. Each well is typically divided into one to sixteen fields. Each field typically will have between one and six photographic images taken of it, each using a different light filter to capture a different wavelength of light for a different fluorescence response for desired cell components. However, the present invention is not limited to such an embodiment, and other containers (e.g., varieties of biological chips, such as DNA chips, micro-arrays, and other containers with multiple sub-containers), sub-containers can also be used to collect image data and feature data from other than cells.","In an embodiment collecting cell data from wells in a microplate, at Step  a microplate with multiple wells is initialized using configuration information. At Step , the configuration information used for the microplate is stored in a microplate database. At Step , a loop is entered to repeat Steps , , , , ,  and  for desired wells in the microplate. At Step , a well in the microplate is selected. At Step , cell image data is collected from the well. In one preferred embodiment of the present invention, the cell image data includes digital photographic images collected with a digital camera attached to a robotic microscope. However, other types of cameras can also be used and other types of image data can also be collected. At Step , the cell image data is stored in an image database. In another exemplary preferred embodiment of the present invention, the image database is a collection of individual image files stored in a binary format (e.g., Tagged Image File Format (\u201cTIFF\u201d), Device-Independent Bit map (\u201cDIB\u201d) and others). The collection of individual image files mayor may not be included in a formal database framework. The individual image files may exist as a collection of individual image files in specified directories that can be accessed from another database (e.g., a pass-through database).","At Step , cell feature data is collected from the cell image data. In one preferred embodiment of the present invention, Step  includes collecting any of the cell feature data illustrated in Table 1. However, other feature data and other cell feature can also be collected and the present invention is not limited to the cell feature data illustrated in Table 1. Virtually any feature data can be collected from the image data.",{"@attributes":{"id":"p-0068","num":"0067"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"196pt","align":"left"}}],"thead":{"row":[{"entry":[{},"TABLE 1"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"CELL SIZE"]},{"entry":[{},"CELL SHAPE"]},{"entry":[{},"CELL INTENSITY"]},{"entry":[{},"CELL TEXTURE"]},{"entry":[{},"CELL LOCATION"]},{"entry":[{},"CELL AREA"]},{"entry":[{},"CELL PERIMETER"]},{"entry":[{},"CELL SHAPE FACTOR"]},{"entry":[{},"CELL EQUIVALENT DIAMETER"]},{"entry":[{},"CELL LENGTH"]},{"entry":[{},"CELL WIDTH"]},{"entry":[{},"CELL INTEGRATED FLUORESCENCE INTENSITY"]},{"entry":[{},"CELL MEAN FLUORESCENCE INTENSITY"]},{"entry":[{},"CELL VARIANCE"]},{"entry":[{},"CELL SKEWNESS"]},{"entry":[{},"CELL KURTOSIS"]},{"entry":[{},"CELL MINIMUM FLUORESCENCE INTENSITY"]},{"entry":[{},"CELL MAXIMUM FLUORESCENCE INTENSITY"]},{"entry":[{},"CELL GEOMETRIC CENTER."]},{"entry":[{},"CELL X-COORDINATE OF A GEOMETRIC CENTER"]},{"entry":[{},"CELL Y-COORDINATE OF A GEOMETRIC CENTER"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}}}},"In  at Step , the cell feature data is stored in a cell feature database. In one embodiment of the present invention, the image database and cell feature databases are combined into a single database comprising multiple tables including the cell image and cell feature data. In another embodiment of the present invention, the image database (or image files) and feature databases are maintained as separate databases.","Returning to  at Step , well summary data is calculated using the image data and the feature data collected from the well. In one preferred embodiment of the present invention, the well summary data calculated at Step  includes calculating any of the well summary data illustrated in Table 2. However, the present invention is not limited to the well summary data illustrated in Table 2, and the other sub-containers and other sub-container summary data can also be calculated. Virtually any sub-container summary data can be calculated for desired sub-containers. In Table 2, a \u201cSPOT\u201d indicates a block of fluorescent response intensity as a measure of biological activity.",{"@attributes":{"id":"p-0071","num":"0070"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"196pt","align":"left"}}],"thead":{"row":[{"entry":[{},"TABLE 2"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"WELL CELL SIZES"]},{"entry":[{},"WELL CELL SHAPES"]},{"entry":[{},"WELL CELL INTENSITIES"]},{"entry":[{},"WELL CELL TEXTURES"]},{"entry":[{},"WELL CELL LOCATIONS"]},{"entry":[{},"WELL NUCLEUS AREA"]},{"entry":[{},"WELL SPOT COUNT"]},{"entry":[{},"WELL AGGREGATE SPOT AREA"]},{"entry":[{},"WELL AVERAGE SPOT AREA"]},{"entry":[{},"WELL MINIMUM SPOT AREA"]},{"entry":[{},"WELL MAXIMUM SPOT AREA"]},{"entry":[{},"WELL AGGREGATE SPOT INTENSITY"]},{"entry":[{},"WELL AVERAGE SPOT INTENSITY"]},{"entry":[{},"WELL MINIMUM SPOT INTENSITY"]},{"entry":[{},"WELL MAXIMUM SPOT INTENSITY"]},{"entry":[{},"WELL NORMALIZED AVERAGE SPOT INTENSITY"]},{"entry":[{},"WELL NORMALIZED SPOT COUNT"]},{"entry":[{},"WELL NUMBER OF NUCLEI"]},{"entry":[{},"WELL NUCLEUS AGGREGATE INTENSITY"]},{"entry":[{},"WELL DYE AREA"]},{"entry":[{},"WELL DYE AGGREGATE INTENSITY"]},{"entry":[{},"WELL NUCLEUS INTENSITY"]},{"entry":[{},"WELL CYTOPLASM INTENSITY"]},{"entry":[{},"WELL DIFFERENCE BETWEEN NUCLEUS AND"]},{"entry":[{},"CYTOPLASM INTENSITY"]},{"entry":[{},"WELL NUCLEUS BOX-FILL RATIO"]},{"entry":[{},"WELL NUCLEUS PERIMETER SQUARED AREA"]},{"entry":[{},"WELL NUCLEUS HEIGHT\/WIDTH RATIO"]},{"entry":[{},"WELL CELL COUNT"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}}}},"Returning to  at Step , the well summary data is stored in a well database. In one embodiment of the present invention, the well database and the microplate database are combined into a single database comprising multiple tables including the well and microplate data. In another embodiment of the present invention, the well and microplate databases are maintained as separate databases. Returning to , the loop continues at Step  () until the desired sub-wells within a microplate have been analyzed.","After the desired wells have been processed in the microplate, the loop at Step  ends. At Step  of , summary data is calculated using well summary data from the microplate database. At Step , the microplate summary data is stored in the well database.","In one preferred embodiment of the present invention, the microplate summary data calculated at Step  includes calculating any of the microplate summary data illustrated in Table 3. However, the present invention is not limited to the microplate summary data illustrated in Table 3, and other container and other container summary data can also be calculated. Virtually any container summary data can be calculated for a container. In Table 3, \u201cMEAN\u201d indicates a statistical mean and \u201cSTDEV\u201d indicates a statistical standard deviation, known in the art, and a \u201cSPOT\u201d indicates a block of fluorescent response intensity as a measure of biological activity.",{"@attributes":{"id":"p-0075","num":"0074"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"thead":{"row":[{"entry":"TABLE 3"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"MEAN SIZE OF CELLS"},{"entry":"MEAN SHAPES OF CELLS"},{"entry":"MEAN INTENSITY OF CELLS"},{"entry":"MEAN TEXTURE OF CELLS"},{"entry":"LOCATION OF CELLS"},{"entry":"NUMBER OF CELLS"},{"entry":"NUMBER OF VALID FIELDS"},{"entry":"STDEV NUCLEUS AREA"},{"entry":"MEAN SPOT COUNT"},{"entry":"STDEV SPOT COUNT"},{"entry":"MEAN AGGREGATE SPOT AREA"},{"entry":"STDEV AGGREGATE SPOT AREA"},{"entry":"MEAN AVERAGE SPOT AREA"},{"entry":"STDEV AVERAGE SPOT AREA"},{"entry":"MEAN NUCLEUS AREA"},{"entry":"MEAN NUCLEUS AGGREGATE INTENSITY"},{"entry":"STDEV AGGREGATE NUCLEUS INTENSITY"},{"entry":"MEAN DYE AREA"},{"entry":"STDEV DYE AREA"},{"entry":"MEAN DYE AGGREGATE INTENSITY"},{"entry":"STDEV AGGREGATE DYE INTENSITY"},{"entry":"MEAN MINIMUM SPOT AREA"},{"entry":"STDEV MINIMUM SPOT AREA"},{"entry":"MEAN MAXIMUM SPOT AREA"},{"entry":"STDEV MAXIMUM SPOT AREA"},{"entry":"MEAN AGGREGATE SPOT INTENSITY"},{"entry":"STDEV AGGREGATE SPOT INTENSITY"},{"entry":"MEAN AVERAGE SPOT INTENSITY"},{"entry":"STDEV AVERAGE SPOT INTENSITY"},{"entry":"MEAN MINIMUM SPOT INTENSITY"},{"entry":"STDEV MINIMUM SPOT INTENSITY"},{"entry":"MEAN MAXIMUM SPOT INTENSITY"},{"entry":"STDEV MAXIMUM SPOT INTENSITY"},{"entry":"MEAN NORMALIZED AVERAGE SPOT INTENSITY"},{"entry":"STDEV NORMALIZED AVERAGE SPOT INTENSITY"},{"entry":"MEAN NORMALIZED SPOT COUNT"},{"entry":"STDEV NORMALIZED SPOT COUNT"},{"entry":"MEAN NUMBER OF NUCLEI"},{"entry":"STDEV NUMBER OF NUCLEI"},{"entry":"NUCLEI INTENSITIES"},{"entry":"CYTOPLASM INTENSITIES"},{"entry":"DIFFERENCE BETWEEN NUCLEI AND CYTOPLASM INTENSITIES"},{"entry":"NUCLEI BOX-FILL RATIOS"},{"entry":"NUCLEI PERIMETER SQUARED AREAS"},{"entry":"NUCLEI HEIGHTIWIDTH RATIOS"},{"entry":"WELL CELL COUNTS"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}}}},"In one exemplary preferred embodiment of the present invention, cell assays are created using selected entries from Tables 1-3. In a preferred embodiment of the present invention, a \u201ccell assay\u201d is a specific implementation of an image processing method used to analyze images and return results related to biological processes being examined. For more information on the image processing methods used in cell assays targeted to specific biological processes, see co-pending application Ser. Nos. 09\/031,217 and 09\/352,171, assigned to the same Assignee as the present application, and incorporated herein by reference.","In one exemplary preferred embodiment of the present invention, the microplate and well databases are stored in a single database comprising multiple tables called \u201cSYSTEM.MDB.\u201d The image and feature data for each well is stored in separate databases in the format \u201cID.MDB,\u201d where ID is a unique identifier for a particular scan. However, the present invention is not limited to this implementation, and other types, and more or fewer databases can also be used.","Storing Experimental Data",{"@attributes":{"id":"p-0078","num":"0077"},"figref":["FIG. 4","FIG. 3"],"b":["78","80","52","82","84","86"]},"In one exemplary preferred embodiment of the present invention, at Step , image data and feature data is collected from desired wells in a microplate using Method  of . However, the present invention is not limited to using Method  to collect experimental data and other methods can also be used. In addition, the present invention is not limited to collecting image data and feature data from wells in a microplate and other sub-containers and containers can also be used (e.g., bio-chips with multiple micro-gels).","At Step , an application database is created. In one exemplary preferred embodiment of the present invention, the application database includes links to other databases but does not include any data collected from the microplate. The application database is used by a display application to view data collected from a microplate. In another embodiment of the present invention, the application database may include actual data.",{"@attributes":{"id":"p-0081","num":"0080"},"figref":["FIG. 5","FIG. 4","FIG. 5","FIG. 5"],"b":["88","78","88","90","92","94","96","98","100"]},"In one exemplary preferred embodiment of the present invention, the application database  is called \u201cAPP.MDB.\u201d However, other names can also be used for the application database in the database system and the present invention is not limited to the name described.","In one exemplary preferred embodiment of the present invention, a display application used to display and analyze collected experimental does not access over a few thousand records at one time. This is because there is no need for evaluation of microplate detail data information (e.g., image or cell feature database data) across microplates. Summary microplate information is stored in microplate, well, microplate feature and well feature summary tables to be compared across microplates. Detailed information about individual cells is accessed within the context of evaluating one microplate test. This allows a display application to make use of pass-through tables in the application database .","In a preferred embodiment of the present invention, the application database  does not contain any actual data, but is used as a \u201cpass-through\u201d database to other databases that do contain actual data. As is known in the art, a pass-through database includes links to other databases, but a pass-through database typically does not contain any actual database data. In such and embodiment, the application database  uses links to the system database  and the multiple image and feature databases , , ,  to pass-through data requests to the application database  to these databases. In another exemplary preferred embodiment of the present invention, the application database  may include some of the actual data collected, or summaries of actual data collected. In one exemplary preferred embodiment of the present invention, the application database  is a Microsoft Access database, a Microsoft Structured Query Language (\u201cSQL\u201d) database or Microsoft SQL Server by Microsoft of Redmond, Wash. However, other databases such as Oracle databases by Oracle Corporation of Mountain View, Calif., could also be used for application database , and the present invention is not limited to Microsoft databases.","In another preferred embodiment of the present invention, a first pass-through database is not used at all. In such an embodiment, the first pass-through database is replaced by computer software that dynamically \u201cdirects\u201d queries to\/from the second and third databases without actually creating or using a first pass-through database.",{"@attributes":{"id":"p-0086","num":"0085"},"figref":["FIG. 6","FIG. 5","FIG. 6","FIG. 6"],"b":["102","90","102","104","90","92","106","108","110","112","94","96","98","100","92"]},"Returning to  at Step , a first entry is created in the application database  linking the application database  to a system database  (e.g., box , ). The system database  includes configuration data used to collect data from a microplate, summary data for the microplate calculated from the desired wells and summary data for selected wells in the microplate calculated from the image data and feature data. This information is organized in multiple tables in the system database .","In one exemplary preferred embodiment of the present invention, the system database  is called \u201cSYSTEM.MDB.\u201d However, other names could also be used and the present invention is not limited to this name. The system database  may also be linked to other databases including microplate configuration and microplate summary data and is used in a pass-through manner as was described above for the application database. In another exemplary preferred embodiment of the present invention, the system database  is not linked to other databases, but instead includes actual microplate configuration and microplate summary data in multiple internal tables.","However, in either case, in one preferred embodiment of the present invention, the name of the system database  is not changed from microplate-to-microplate. In another preferred embodiment of the present invention, the name of the system database  is changed from microplate-to-microplate. A display application will refer to the system database  using its assigned name (e.g., SYSTEM.MDB) for microplate configuration and microplate summary data. Data stored in the system database  may be stored in linked databases so that the actual microplate container configuration and microplate summary data can be relocated without changing the display application accessing the system database . In addition the actual database engine could be changed to another database type, such as a Microsoft SQL Server or Oracle databases by Oracle, or others without modifying the display application accessing the system database .",{"@attributes":{"id":"p-0090","num":"0089"},"figref":["FIG. 7","FIG. 5","FIG. 7"],"b":["114","92","114","116","116","118","122","124","126","118","122","124","122","120","120","126"]},"The protocol table  is linked to a protocol assay parameters table . In a preferred embodiment of the present invention, an \u201cassay\u201d is a specific implementation of an image processing method used to analyze images and return results related to biological processes being examined. The protocol assay parameters table  is linked to an assay parameters table . The assay parameters table  include parameters for an assay in use.","The protocol table  is also linked to a protocol channel table . Typically an assay will have two or more channels. A \u201cchannel\u201d is a specific configuration of optical filters and channel specific parameters and is used to acquire an image. In a typical assay, different fluorescent dyes are used to label different cell structures. The fluorescent dyes emit light at different wavelengths. Channels are used to acquire photographic images for different dye emission wavelengths. The protocol channel table  is linked to a protocol channel reject parameters table . The protocol channel reject parameters table  includes channel parameters used to reject images that do not meet the desired channel parameters.","The protocol table  is also linked to a protocol scan area table . The protocol scan area table  includes methods used to scan a well. The protocol scan area table  is linked to a system table . The system table  includes information configuration information and other information used to collect experimental data.","The well table  is linked to a well feature table . The well feature table  includes mapping of cell features to wells. The well feature table  is linked to a feature type table . The feature type table  includes a list of features (e.g., cell features) that will be collected. However, more or fewer tables can also be used, more or fewer links can be used to link the tables, and the present invention is not limited to the tables described for the system database .","Returning to  Step , multiple second entries (e.g., boxes , , ,  of ) are created in the application database  linking the application database  to multiple image and feature databases , , , . The multiple image and feature databases include image data and feature data collected from the desired wells in the microplate. The data is organized in multiple database tables in the image and feature databases.","In one exemplary preferred embodiment of the present invention, names of image and feature databases , , ,  that contain the actual image and feature data are changed dynamically from microplate-to-microplate. Since the image and feature data will include many individual databases, an individual image and feature database is created when a microplate record is created (e.g., in the plate table  () in the system database  ()) and has a name that is created by taking a plate field value and adding \u201c.MDB\u201d to the end. (For example, a record in a plate table  with a field identifier of \u201c1234569803220001\u201d will have it's data stored in a image and feature database with the name \u201c1234569803220001.MDB\u201d). However, other names can also be used for the image and feature databases and the present invention is not limited to the naming scheme using a field identifier from the plate table .",{"@attributes":{"id":"p-0097","num":"0096"},"figref":["FIG. 8","FIG. 5","FIG. 7","FIG. 7","FIG. 7"],"b":["144","94","96","98","100","116","142","120","138","92","94","96","98","100","116","142","94","96","98","100","116","142","92","92"]},"The image and feature databases , , , , tables  include a well field table  for storing information about fields in a well. The well field table  is linked to a well feature table  that includes information a list of features that will be collected from a well. The well field table  is also linked to a feature image table  that includes a list of images collected from a well and a cell table  that includes information to be collected about a cell. The cell table  is linked to a cell feature table  that includes a list of features that will be collected from a cell. However, more or fewer tables can also be used, more or fewer links can be used to link the tables, and the present invention is not limited to the tables described for the image and feature databases.","Spooling Experimental Data","As was discussed above, the analysis instruments modules , ,  generate a large amount of data including image data, feature data, and summary data for sub-containers and containers. The raw feature data values are stored as database files with multiple tables described above (e.g., ). To prevent analysis instruments , ,  and\/or the store clients , ,  from running out of file space, database files are managed using a hierarchical data management system.",{"@attributes":{"id":"p-0100","num":"0099"},"figref":"FIG. 9","b":["156","158","160","162"]},"In another preferred embodiment of the present invention, Method  further comprises copying the first database from the analysis instruments , ,  to a store client computers , , . Such an embodiment allows a display application on the store client computers , ,  to view the data collected from the container using the first database copied to local storage on the client computers , , .","In another preferred embodiment of the present invention, Method  further comprises locating the first database on the analysis instruments , ,  from store client computers , , . Such an embodiment allows a display application on the store client computers , ,  to view the data collected from the container at a remote location on the exemplary data storage system \u2032 from the store client computers , , .","The data collected is viewed from the display application on the store client computers , ,  by retrieving container and sub-container data from the second database on the shared database  and image and feature data from the multiple third databases on the shared database file server .","In one exemplary preferred embodiment of the present invention, at Step , a system database  () is copied from an analysis instrument , ,  to the shared database . The system database  includes configuration data used to collect data from a microplate, summary data for the microplate calculated from one or more wells in the microplate (e.g., Table 3) and summary data for wells in the microplate (e.g., Table 2) calculated from image data and feature data (e.g., Table 1) collected from desired wells as was described above. The data in the system database  is organized into one or more database tables (e.g., ).","At Step , one or more image and feature databases , , ,  are copied to the shared database file server . The one or more image and feature databases , , ,  include image data and a feature data collected from the desired wells in the microplate. The data in the one or more image and feature databases is organized into one or more database tables (e.g., ).","At Step , a location of the system database  and the one or more image and feature databases , , ,  is updated in an application database  () on the analysis instrument , ,  to reflect new storage locations for the system database  on the store database  and one or more image and feature databases , , ,  on the store archive .","In one preferred embodiment of the present invention, the application database  is a pass-through database that includes links (e.g., ) to the system database  and the one or more image and feature databases , , ,  but does not include any data collected from the microplate. In another embodiment of the present invention, the application database  includes data from the microplate. The application database  is used by a display application to view data collected from a microplate. However, the present invention is not limited to this embodiment and other containers, sub-containers, (e.g., bio-chips with multiple micro-gels) and databases can also be used.","Hierarchical Management of Experimental Data",{"@attributes":{"id":"p-0108","num":"0107"},"figref":"FIG. 10","b":["164","166","168","170","172","174","176"]},"In one exemplary preferred embodiment of the present invention, the pre-determined storage removal policy includes one or more rules illustrated by Table 4. However, more or fewer storage removal policy rules can also be used and the present invention is not limited to storage removal policy rules illustrated in Table 4.",{"@attributes":{"id":"p-0110","num":"0109"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"thead":{"row":[{"entry":[{},"TABLE 4"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"PERCENTAGE OF DISK SPACE AVAILABLE OR"]},{"entry":[{},"PERCENTAGE OF DISK SPACE USED."]},{"entry":[{},"NUMBER OF FILES."]},{"entry":[{},"DATE A FILE IS STORED."]},{"entry":[{},"SIZE OF A FILE."]},{"entry":[{},"NUMBER OF DAYS SINCE A FILE WAS LAST ACCESSED."]},{"entry":[{},"FILE TYPE."]},{"entry":[{},"FILE NAME."]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}}}},"Method  includes HSM steps that provide a method to allow on-line access to virtually unlimited amounts of \u201cvirtual\u201d disk space on data storage system \u2032. The virtual disk space is provided with a multi-layer hierarchical storage management system. The virtual disk space is provided without changing the layout of any database and is \u201cinvisible\u201d to a user.","In one exemplary preferred embodiment of the present invention, the HSM steps of Method  provide an archival method that implements a three-layer storage hierarchy including the disk archive , the optical jukebox  and the tape drive . However, more or fewer layers of storage can also be used and the present invention is not limited to HSM techniques with three-layer storage. Additional storage layers in the storage hierarchy are added as needed without changing the layout of any database or the functionality of the hierarchical storage manager. The hierarchical storage manager can copy database files to layers in an N-Layer storage hierarchy without modification.","In addition, virtually unlimited amounts of \u201cvirtual\u201d disk space can be provided with a three-layer hierarchical storage management system by periodically removing re-writeable optical disks, from the optical jukebox  and tapes from the tape drive  when these storage mediums are filled with data. The re-writeable optical disks and tapes are stored in a data library for later access. In another preferred embodiment of the present invention, the data library is directly accessible from computer network .","In a preferred embodiment of the present invention, Method  supports at least two modes of database file archiving. However, more or fewer modes of database archiving can also be used and the present invention is not limited to the two modes described.","In the first mode, the store server  retains database files on individual analysis instruments , , , where they were originally generated. The store server  uses Method  to automatically manage the free space on the analysis instrument , ,  disks to move files into a layer in the three-tiered storage management system. To the end user the files will appear to be in the same directories where they were originally stored. However, the files may actually be stored on the disk archive , the optical jukebox , or in a Digital Linear Tape (\u201cDLT\u201d)  library.","In the second mode, the store server  spools database files from the analysis instruments , , , to the shared database  and the shared database file server  (e.g., using Method ). The store server's  in turn manages database files on the shared database file server  using Method . In the second mode, the files may also be stored on the disk archive , the optical jukebox , or in a DLT  library.","Experimental Data Presentation","As was discussed above, an analysis instrument , ,  can generate a huge amount of experimental data. To be useful, the experimental data has to be visually presented to a scientist or technician for analysis.",{"@attributes":{"id":"p-0118","num":"0117"},"figref":"FIG. 11","b":["178","180","182","184","186","188","190"]},"In one exemplary preferred embodiment of the present invention, Method  is used for displaying experimental data collected from microplates with multiple wells. However, the present invention is not limited to this embodiment and can be used for other containers and sub-containers besides microplates with multiple wells (e.g., bio-chips with multiple micro-gels).","In such an exemplary embodiment at Step , a list including multiple microplates is displayed from a display application on a computer. The microplates include multiple wells. Cell image data and cell feature data were collected from the multiple microplates. The display application uses an application database  to locate other databases, including experimental data.","In one preferred exemplary embodiment of the present invention, the application database  is located on the exemplary data storage system \u2032 at a location remote from the computer including the display application. The application database  is used from the computer including the display application without copying the application database  from a remote location on the exemplary data storage system \u2032.","In another exemplary preferred embodiment of the present invention, the application database  is copied from a location on the exemplary data storage system \u2032 to local storage on the computer including the display application. In such an embodiment, the application database  is copied to, and exists on the computer including the display application.","At Step , a first selection input is received on the display application for a first microplate from the list. At Step , a system database  is obtained for the first microplate from a first remote storage location. The first remote storage location is remote to the computer running the display application. The system database  includes configuration data used to collect data from the first microplate summary data for the first microplate calculated from the wells in the first microplate and summary data for desired wells in the first microplate calculated from image data and feature data collected from desired wells.","At Step , a second selection input is received on the display application for one or more wells in the first microplate. At Step , multiple image and feature databases , , ,  are obtained from a second remote storage location. The multiple image and feature databases , , ,  include image data and feature data collected from the one or more wells in the first microplate. At Step , a graphical display is created from the display application including microplate and well summary data from the system database , image data and feature data from the multiple image and feature databases , , ,  collected from the one or more wells. Data displayed on the graphical display appears to be obtained from local storage on the computer instead of the first remote storage location and the second remote storage location.",{"@attributes":{"id":"p-0125","num":"0124"},"figref":["FIG. 12","FIG. 12"],"b":["192","192","194","196","194","192","198","200","202","204","192","192","3","206","196","178","192"]},"In one exemplary preferred embodiment of the present invention, a Store Application Programming Interface (\u201cAPI\u201d) is provided to access and use the methods and system described herein. As is known in the art, an API is set of interface routines used by an application program to access a set of functions that perform a desired task.","In one specific exemplary preferred embodiment of the present invention, the store API is stored in a Dynamic Link Library (\u201cDLL\u201d) used with the Windows 95\/98\/NT\/2000 operating system by Microsoft. The DLL is called \u201cmvPlateData.DLL.\u201d However, the present invention is not limited to storing an API in a Window's DLL or using the described name of the DLL and other methods and names can also be used to store and use the API. As is known in the art, a DLL is library that allows executable routines to be stored and to be loaded only when needed by an application. The Store API in a DLL is registered with the Window's \u201cREGSVR32.EXE\u201d application to make it available to other applications. The Store API provides an interface access to plate, well image and cell feature information and provides a facility to enter desired well feature information that will be collected.","These methods and system described herein may allow experimental data from high-throughput data collection\/analysis systems to be efficiently collected, stored, managed and displayed. The methods and system can be used for, but is not limited to storing managing and displaying cell image data and cell feature data collected from microplates including multiple wells or bio-chips including multiple micro-gels in which an experimental compound has been applied to a population of cells. If bio-chips are used, any references to microplates herein, can be replaced with bio-chips, and references to wells in a microplate can be replaced with micro-gels on a bio-chip and used with the methods and system described.","The methods and system may provide a flexible and scalable repository of cell data that can be easily managed and allows cell data to be analyzed, manipulated and archived. The methods and system may improve the identification, selection, validation and screening of new experimental compounds which have been applied to populations of cells. The methods and system may also be used to provide new bioinformatic techniques used to make observations about cell data.","It should be understood that the programs, processes, methods and systems described herein are not related or limited to any particular type of computer or network system (hardware or software), unless indicated otherwise. Various types of general purpose or specialized computer systems may be used with or perform operations in accordance with the teachings described herein.","In view of the wide variety of embodiments to which the principles of the present invention can be applied, it should be understood that the illustrated embodiments are exemplary only, and should not be taken as limiting the scope of the present invention.","For example, the steps of the flow diagrams may be taken in sequences other than those described, and more or fewer elements may be used in the block diagrams. While various elements of the preferred embodiments have been described as being implemented in software, in other embodiments in hardware or firmware implementations may alternatively be used, and vice-versa.","The claims should not be read as limited to the described order or elements unless stated to that effect. Therefore, all embodiments that come within the scope and spirit of the following claims and equivalents thereto are claimed as the invention."],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":["Preferred embodiments of the present invention are described with reference to the following drawings, wherein:",{"@attributes":{"id":"p-0035","num":"0034"},"figref":"FIG. 1A"},{"@attributes":{"id":"p-0036","num":"0035"},"figref":"FIG. 1B"},{"@attributes":{"id":"p-0037","num":"0036"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0038","num":"0037"},"figref":"FIGS. 3A and 3B"},{"@attributes":{"id":"p-0039","num":"0038"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0040","num":"0039"},"figref":["FIG. 5","FIG. 4"]},{"@attributes":{"id":"p-0041","num":"0040"},"figref":["FIG. 6","FIG. 5"]},{"@attributes":{"id":"p-0042","num":"0041"},"figref":["FIG. 7","FIG. 5"]},{"@attributes":{"id":"p-0043","num":"0042"},"figref":["FIG. 8","FIG. 5"]},{"@attributes":{"id":"p-0044","num":"0043"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0045","num":"0044"},"figref":"FIG. 10"},{"@attributes":{"id":"p-0046","num":"0045"},"figref":"FIG. 11"},{"@attributes":{"id":"p-0047","num":"0046"},"figref":"FIG. 12"}]},"DETDESC":[{},{}]}
