---
title: Transcription biomarkers of biological responses and methods
abstract: This invention provides transcription regulatory control sequences, the activity of which function as biomarkers for a variety of biological responses. This invention also provides expression constructs in which a biomarker transcription regulatory sequence is operably linked with a sequence for a reporter. Cells that comprise these expression constructs can be used in assays to identify conditions that modulate activity of the biological response.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08815779&OS=08815779&RS=08815779
owner: SwitchGear Genomics, Inc.
number: 08815779
owner_city: Menlo Park
owner_country: US
publication_date: 20090916
---

{"@attributes":{"id":"description"},"BRFSUM":[{},{}],"p":["A CD containing a formal Sequence Listing is filed with this application and the contents of the CD are expressly incorporated herein in their entirety by reference.","Reporter constructs including an expression control sequence operably linked with a reporter sequence are widely used in biology to test for the activity of the expression control sequence under selected conditions. Typically, such constructs are used to test for the expression of the gene normally under transcriptional control of the control sequence. Such constructs have many uses. They can be used, for example, to monitor gene expression during cellular responses to signaling cascades. They also are useful in drug development assays. In drug development, the induction or suppression of expression of a gene by a drug candidate may be a desired or undesired event. In such assays, a cell that contains an expression construct comprising a control sequence for the gene of interest is contacted with a drug candidate and the expression of the reporter is monitored. Compositions that elicit an undesirable result may be eliminated from further testing. Alternatively, compositions that elicit a desired result may be subject to further testing.","In this age of systems biology, investigators may wish to study the activity of many genes in a biological response or pathway, or to identify compositions that modulate the activity of a biological response. Biological responses are characterized by the induction or suppression of many genes. Biological responses of interest include, for example, (1) the hypoxia response, (2) the response to estrogens; (3) the response to androgens; (4) the response mediated by the p53 protein; (5) the response to inhibitors or activators of cholesterol biosynthesis; (6) the interferon-mediated response; (7) the CREB-mediated response; (8) the response to glucocorticoids; (9) the PPAR-mediated response; (10) the RAR-mediated response; (11) the inflammation response induced by TNFa; (12) the heat shock response; and (13) the serum response.","Trinklein et al. have described a library that contains transcription regulatory sequences for thousands of genes in the genome and that are associated with various pathways. See, e.g., U.S. patent publications 2007-0161031 and 2009-0018031.","In one aspect this invention provides an expression construct comprising a transcriptional regulatory sequence operatively linked with a heterologous sequence encoding a reporter, wherein the transcription regulatory sequence is one of a gene of Table 1. In one aspect this invention provides a collection comprising a plurality of expression constructs, each expression construct comprising a transcriptional regulatory sequence operatively linked with a heterologous sequence encoding a reporter, wherein at least 20% of the expression constructs in the collection each comprise a transcription regulatory sequence of a different gene of Table 1. In certain embodiments, the transcription regulatory sequences include at least a fragment of a sequence selected from SEQ ID NO: 1 to SEQ ID NO. 174. In other embodiments, the transcription regulatory sequences include at least a fragment of a sequence selected from SEQ ID NO: 175 to SEQ ID NO. 1619. In one embodiment at least 20% of the expression constructs comprise transcription regulatory sequences differently selected from the group consisting of (i) any of SEQ ID NO: 1 to SEQ ID NO: 174; (ii) a sequence of at least 50 nucleotides having at least 90% sequence identity to a sequence of (i); and (iii) a fragment of a sequence of (i). In another embodiment the transcription regulatory sequences of the different genes of Table 1 are biomarkers of the same biological response. In another embodiment the transcription regulatory sequences of the different genes of Table 1 comprise biomarkers of different biological responses. For example, the plurality of different biological response could be 2, at least 2, 3, at least 3, no more than 3, 4, at least 4, no more than 4, 5, at least 5, no more than 5, 6, at least 6, no more than 6, 7, at least 7, no more than 7, 8, at least 8, no more than 8, 9, at least 9, no more than 9, 10, at least 10, no more than 10, 11, at least 11, no more than 11, 12, at least 12 no more than 12, or 13 different biological responses selected from (1) the hypoxia response, (2) the response to estrogens; (3) the response to androgens; (4) the response mediated by the p53 protein; (5) the response to inhibitors or activators of cholesterol biosynthesis; (6) the interferon-mediated response; (7) the CREB-mediated response; (8) the response to glucocorticoids; (9) the PPAR-mediated response; (10) the RAR-mediated response; (11) the inflammation response induced by TNFa; (12) the heat shock response; and (13) the serum response In another embodiment the transcription regulatory sequences of the different genes of Table 1 comprise at least three biomarkers of each of a plurality different biological responses. In another embodiment the expression construct is comprised in a plasmid, a virus, or a transposon vector. In another embodiment the expression construct is integrated into a chromosome in a cell or maintained as a stable episomal vector. In another embodiment the reporter is a light-emitting reporter, a fluorescent reporter or a colorimetric reporter. In another embodiment the heterologous sequence encoding the reporter comprises a luciferase gene. In another embodiment the collection comprises at least 10 and no more than 1000 different expression constructs. In another embodiment the collection comprises at least 10 and no more than 100 different expression constructs. In another embodiment the collection comprises at least 10 and no more than 50 different expression constructs. In another embodiment the collection comprises at least 50 and no more than 100 different expression constructs. In another embodiment each expression construct is comprised in a different molecule. In another embodiment each recombinant nucleic acid molecule is comprised in different well of at least one microtiter plate. In another embodiment the transcription regulatory sequences of the collection consist of or consist essentially of transcription regulatory sequences of genes of Table 1, or those comprising all or part of the sequences in the sequence listing, including those in Part I and\/or Part II.","In one aspect this invention provides a cell comprising an expression construct comprising a transcriptional regulatory sequence operatively linked with a heterologous sequence encoding a reporter, wherein the transcription regulatory sequence is one of a gene of Table 1. In another aspect this invention provides a collection of a plurality of cells comprising expression constructs, each expression construct comprising a transcriptional regulatory sequence operatively linked with a heterologous sequence encoding a reporter, wherein at least 20% of the expression constructs each comprise a transcription regulatory sequence of a different gene of Table 1. In one embodiment each cell is comprised in different well of at least one microtiter plate. In another embodiment the cells are human cells. In another embodiment the cells are mammalian cells. In another embodiment at least one expression construct is stably integrated into a chromosome wherein the expression construct comprises a transcriptional regulatory sequence operatively linked with a heterologous sequence encoding a reporter, wherein the expression construct comprises a transcription regulatory sequence of a gene of Table 1. In another embodiment the cells comprise a plurality of different expression constructs, each comprising a transcription regulatory sequence of a different gene of Table 1. In another embodiment each different expression construct is comprised in a different cell of the cell line. In another embodiment the transcription regulatory sequences of the collection consist of or consist essentially of transcription regulatory sequences of genes of Table 1, or those comprising all or part of the sequences in the sequence listing, including those in Part I and\/or Part II.","In another aspect this invention provides a multicellular organism that contains at least one expression construct comprising a transcriptional regulatory sequence operatively linked with a heterologous sequence encoding a reporter, wherein the expression construct comprises a transcription regulatory sequence of a gene of Table 1. In one embodiment the multicellular organism is selected from a plant, a vertebrate and an invertebrate.","In another aspect this invention provides a device comprising a plurality of receptacles, wherein each receptacle comprises an expression construct comprising a transcriptional regulatory sequence operatively linked with a heterologous sequence encoding a reporter, wherein at least 20% of the expression constructs each comprise a transcription regulatory sequence of a different gene of Table 1. In one embodiment the device comprises at least one microtiter plate.","In another aspect this invention provides a device comprising a plurality of receptacles, wherein each receptacle contains at least one cell comprising an expression construct, each expression construct comprising a transcriptional regulatory sequence operatively linked with a heterologous sequence encoding a reporter, wherein at least 20% of the expression constructs each comprise a transcription regulatory sequence of a different gene of Table 1. In one embodiment the device comprises at least one microtiter plate.","In another aspect this invention provides a method comprising: a) exposing at least one cell to a test condition, wherein the at least one cell comprises at least one expression construct comprising a transcriptional regulatory sequence operatively linked with a heterologous sequence encoding a reporter, wherein the transcription regulatory sequence is a transcription regulatory sequence of a gene of Table 1; b) measuring expression of the reporter; and c) correlating the measurement with a biological response for which the transcription regulatory sequence is a biomarker. In one embodiment the method further comprises i) measuring the activity of the reporter in the absence of the test condition; ii) measuring the activity of the reporter in the presence of the test condition; iii) measuring the difference or ratio of reporter activity between the cells exposed to the test condition and cells that were not; and iv) correlating the difference or ratio between treated and untreated measurements with a biological response. In one embodiment the transcription regulatory sequence comprises a sequence selected from the group consisting of: (i) any of SEQ ID NO:1 to SEQ ID NO: 174; (ii) a sequence of at least 50 nucleotides having at least 95% sequence identity to a sequence of (i); and (iii) a fragment of a sequence of (i). In another embodiment the test condition is selected from the group consisting of contacting the cell with a test compound, exposing the cell to an environmental condition and inducing or repressing expression of one or more genes in the cell. In another embodiment the test condition comprises removing a compound from the culture media in which the cell grown. In another embodiment the test condition comprises contacting the cell with a test compound selected from: (1) a small organic molecule; (2) a nucleic acid derivative (e.g., a small interfering RNA, micro RNA mimic or micro RNA inhibitor); and (3) an expression construct that contains an open reading frame of a gene. In another embodiment the test condition comprises exposing the cell to an environmental condition selected from hyperthermia, hypothermia, hypoxia, osmotic stress, oxidative stress, radiation, or changes in atmospheric conditions. In another embodiment each expression construct is comprised in a different cell. In another embodiment the at least one cell comprises a plurality of the expression constructs, wherein each expression construct comprises a transcription regulatory sequence of a different gene of Table 1, wherein the transcription regulatory sequences of the different genes are biomarkers of the same biological response. In another embodiment the at least one cell comprises a plurality of the expression constructs, wherein each expression construct comprises a transcription regulatory sequence of a different gene of Table 1, wherein the transcription regulatory sequences of the different genes are biomarkers of different biological responses. In another embodiment the at least one cell comprises a plurality of the expression constructs, wherein each expression construct comprises a transcription regulatory sequence of a different gene of Table 1, wherein the transcription regulatory sequences of the different genes comprise at least three different biomarkers of each of a plurality different biological responses. In another embodiment the biological responses are responses selected from (1) the hypoxia response, (2) the response to estrogens; (3) the response to androgens; (4) the response mediated by the p53 protein; (5) the response to inhibitors or activators of cholesterol biosynthesis; (6) the interferon-mediated response; (7) the CREB-mediated response; (8) the response to glucocorticoids; (9) the PPAR-mediated response; (10) the RAR-mediated response; (11) the inflammation response induced by TNFa; (12) the heat shock response; and (13) the serum response. In another embodiment the method comprises contacting the test compound with a plurality of sets of different cell types, wherein each expression construct is comprised in each of the cell types. In another embodiment the method further comprises exposing a control cell to the test condition, wherein the control cell comprises an expression construct comprising a control transcriptional regulatory sequence operatively linked with a heterologous sequence encoding a reporter, wherein the control transcriptional regulatory sequence is a transcriptional regulatory sequence of a gene whose activity is not correlated with a biological response correlated with the activity of transcriptional control sequence of the gene of Table 1. In another embodiment the expression construct is comprised in a plasmid, a virus, or an artificial chromosome vector or a transposon vector. In another embodiment the expression construct is integrated into a chromosome in the cell or maintained as a stable episomal vector. In another embodiment the reporter is a light-emitting reporter, a fluorescent reporter or a colorimetric reporter. In another embodiment the reporter comprises luciferase. In another embodiment the method further comprises exposing the cell to a condition that induces the biological response for which the transcription regulatory sequence is a biomarker. In another embodiment correlating comprises correlating a change in biomarker activity with induction of the biological response. In another embodiment correlating comprises correlating a change in biomarker activity with inhibition of the biological response. In another embodiment the method further comprises inducing the biological response and wherein correlating comprises correlating inhibition of the biomarker with inhibition of the biological response. In another embodiment the method further comprises inhibiting the biological response and wherein correlating comprises correlating inhibition of the biomarker with induction of the biological response.","In another aspect this invention provides a method comprising: a) exposing at least one cell to a test condition, wherein the at least one cell comprises at least one expression construct comprising a transcriptional regulatory sequence operatively linked with a heterologous sequence encoding a reporter, wherein the transcription regulatory sequence is a transcription regulatory sequence of a gene of Table 1; b) measuring expression of the reporter; and if the measurement is at a target level; c) exposing at least one second cell to the test condition, wherein the at least one second cell comprises at least one expression construct comprising a transcriptional regulatory sequence operatively linked with a heterologous sequence encoding a reporter, wherein the transcription regulatory sequence is a transcription regulatory sequence of a different gene of the same biological response of Table 1; and; d) measuring expression of the reporter sequence in the at least one second cell. In one embodiment the method further comprises: e) correlating the measurement with the biological response.","In another aspect this invention provides a method of determining whether a test condition modulates a biological response. In one embodiment the method comprises exposing a cell comprising an expression construct of this invention to the test condition and determining whether test condition modulates the activity of the biomarker promoter, wherein modulation of the activity of the biomarker promoter indicates that that the test condition modulates the biological response for which the biomarker promoter is a biomarker. In another embodiment this method comprises exposing a cell comprising an expression construct of this invention to conditions that induce or repress the biological response for which the biomarker promoter is a biomarker; exposing the cell to the test condition and determining whether the test condition modulates the activity of the biomarker promoter, wherein modulation of the activity indicates that the test condition modulates the activity of the biological response. The activity of the biomarker promoter can be positively or negative correlated with the activity of the biological response. In the case of positive correlation, induction of the biomarker promoter is correlated with induction of the response and\/or repression of the biomarker promoter is correlated with repression of the biological response. In the case of negative correlation, repression of activity of the promoter is correlated with induction of the biological response and induction of activity of the promoter is correlated with repression of the biological response.","All publications and patent applications mentioned in this specification are herein incorporated by reference to the same extent as if each individual publication or patent application was specifically and individually indicated to be incorporated by reference.","1. Biomarkers","This invention provides nucleic acid molecules comprising transcription regulatory sequences for genes useful as biomarkers of a variety of biological responses or biological pathways. The genes and the biological responses\/pathways are identified in Table 1. Certain transcription regulatory sequences of these genes also are identified in Table 1 and provided in the sequence listing. This invention also provides expression constructs comprising biomarker transcription control sequences, devices including them and methods of use. The biomarker transcription expression control sequences are useful, among other things, for identifying test conditions that modulate activity of the biological response or pathway. Modulation includes changing the activity of a pathway, including increasing and decreasing activity.","Biomarkers exhibit different activity or expression under different conditions or states. A biomarker is differentially active or present between states if the mean or median activity or expression level of the biomarker in the different states is calculated to be statistically significant. Common tests for statistical significance include, among others, t-test, ANOVA, Kruskal-Wallis, Wilcoxon, Mann-Whitney and odds ratio. As such, the activity of a biomarker is correlated with a state and the biomarker is useful in classification algorithms to classify an entity as belonging to one or another category. In the present context, activity of a biomarker expression control sequence in a cell indicates whether a biological response or pathway is active in the cell. Cells that contain expression constructs comprising a biomarker expression control sequence operatively linked to a reporter sequence can be subject to a test condition. Activity of the biomarker expression control sequence provides an indication of whether the test condition elicits activity of the biological response or pathway for which the transcription regulatory sequence is a biomarker. For example, a cell comprising an expression construct of this invention can be exposed to a drug candidate to determine whether the drug candidate elicits or inhibits a desired or undesired response, or to an environmental stimulus or condition to determine whether the stimulus or condition elicits or inhibits the response.","2. Biological Responses and Pathways","Biological responses are often characterized by the change in expression of sets of genes. A set of genes whose expression is activated (or turned off) as part of the biological response is referred to as a \u201cpathway.\u201d One goal is to identify compounds that modulate or mimic a biological response, or activate or inhibit activity of genes in a pathway. A number of genes have been identified that have promoters whose activity is highly correlated with particular biological responses. Therefore, promoters of these genes function as biomarkers of the biological response. The genes are identified in Table 1, along with specific transcriptional regulatory sequences that were determined to function as biomarkers of the response. In general, a transcriptional regulatory sequence functions as a biomarker if the induction ratio","Biomarkers have been identified for the following biological responses: (1) the hypoxia response, (2) the response to estrogens; (3) the response to androgens; (4) the response mediated by the p53 protein; (5) the response to inhibitors or(the ratio between activity with induction of the response and the activity without induction of the response) is at least 2, at least 4, at least 10, at least 25 or at least 50.","Biomarkers have been identified for the following biological response: (1) the hypoxia response, (2) the response to estrogens; (3) the response to androgens; (4) the response mediated by the p53 protein; (5) the response to inhibitors or activators of cholesterol biosynthesis; (6) the interferon-mediated response; (7) the CREB-mediated response; (8) the response to glucocorticoids; (9) the PPAR-mediated response; (10) the RAR-mediated response; (11) the inflammation response induced by TNFa; (12) the heat shock response; and (13) the serum response.","This invention provides collections of these biomarker promoters, expression constructs in which they are operably linked with sequences encoding reporters, cells that contain these expression constructs, devices that contain the expression constructs and methods of using expression constructs, in particular to determine whether any test condition can modulate the activity of particular biological responses, based on its ability to regulate the activity of the biomarker expression control sequences.","A biological response biomarker is positively correlated with the biological response if its activity is induced or activated when a biological response is active, or if its activity is repressed when the biological response is repressed. A biological response biomarker is negatively correlated with the biological response if its activity is repressed when a biological response is active, or if its activity is induced or activated when the biological response is repressed.","Identifying test conditions that induce or repress the activity of biomarker promoters is useful to evaluate test conditions, such as test compositions, for the ability to induce or inhibit the biological response with which the biomarker activity is correlated. A test condition that induces activity of a biological response biomarker promoter positively correlated with the response is presumed to induce the biological response in full or in part, e.g., to cause a response that mimics the biological response, and its action is correlated positively with the biological response. A test condition that inhibits activity of a biological response biomarker promoter positively correlated with the response when cells are exposed to conditions that induce the biological response is presumed to inhibit the biological response, e.g., to inhibit events correlated with the biological response, and its action is correlated negatively with the biological response.","Alternatively, a test condition that inhibits activity of a biological response biomarker promoter negatively correlated with the response is presumed to induce the biological response in full or in part, e.g., to cause a response that mimics the biological response, and its action is correlated positively with the biological response. A test condition that induces activity of a biological response biomarker promoter negatively correlated with the response when cells are exposed to conditions that induce the biological response is presumed to inhibit the biological response, e.g., to inhibit events correlated with the biological response, and its action is correlated negatively with the biological response.","A method for identifying a test condition that elicits a biological response can comprise providing a cell comprising an expression construct comprising a biological response biomarker promoter operably linked to a reporter gene and exposing the cell to a test condition. The activity of a biological response biomarker promoter in response to the test condition can be measured. The measurement can be correlated with the biological response. Conditions resulting in activity of a desired correlation (e.g., induction of activity of a positively correlated biomarker, or inhibition of the activity a negatively correlated biomarker) can be identified as conditions that elicit the biological response.","A method for identifying a test condition that inhibits a biological response can comprise providing a cell comprising an expression construct comprising a biological response biomarker promoter operably linked to a reporter gene, exposing the cell to a condition that elicits a biological response (or to a condition that induces expression of a biological response biomarker promoter) and exposing the cell to a test condition. The activity of a biological response biomarker promoter in response to the test condition can be measured. The measurement can be correlated with the biological response. Conditions resulting in activity of a desired correlation (e.g., inhibition of the inducible activity of a positively correlated biomarker, or induction of activity of a negatively correlated biomarker) can be identified as conditions that inhibit the biological response.","In each situation, the test condition can be tested further. For example, the ability of the test condition to modulate activity of at least one or a plurality of different biomarkers for the biological response can be tested. The activity of these promoters can be correlated with the biological response. The test condition also can be tested for the ability to modulate a different biological response than the one originally tested.","2.1 Biological Response to Hypoxia","Several promoters have been identified, the activity of each of which is positively correlated with the biological response to hypoxia (\u201chypoxia response biomarker promoter\u201d). These hypoxia response biomarker promoters are identified in Table 1, section 1 and section 1a. Hypoxia response biomarker promoters generally have inducible activities ranging from 2 to 10-fold typically in this assay. The conditions used to induce the biological response to hypoxia included exposing cells to a 1% oxygen environment and 100 micromolar deferoxamine (DFO) for 24 hrs. The experimental conditions are set forth more completely in Example 1.","More specifically, transcription regulatory sequences have been found that are induced when exposed to low oxygen conditions. Low oxygen conditions induce the hypoxia response. One can use such a promoter in an assay to determine whether a test compound induces the hypoxia response. This is done, for example, by exposing a cell containing an expression construct comprising the transcription regulatory sequence operatively linked with a reporter sequence to a test compound. If the compound induces activity of the promoter, this means the compound induces the hypoxia response. Also, one can use the promoter in an assay to determine whether a test compound inhibits the hypoxia response. This is done, for example, by exposing a cell containing the expression construct to a condition that induces the hypoxia response (for example, exposing the cell to low oxygen conditions) and exposing the cell to the test compound. If the compound inhibits the inducible activity of the promoter, this means the compound inhibits the hypoxia response.","The biological response to hypoxia (or, \u201chypoxia response\u201d) involves a number of events at the phenotypic, cellular and molecular level. These events include increasing the expression of genes involved in cellular respiration and promotion of vascularization and oxygen uptake. The activity of hypoxia response biomarker promoters is, in turn, also correlated with these events.","In certain situations it may be useful to induce a hypoxia response. For example, hypoxia occurs under ischemic conditions, e.g., heart disease, transient ischemic attack, cerebrovascular accident, ruptured arteriovenous malformation and peripheral artery occlusive disease. Under such conditions, cells and tissue can be damaged. For example, a subject can experience cell death in the ischemic area. Pharmaceutical drugs that induce hypoxia may provide protection in situations of ischemia-induced hypoxia.","In certain situations it may be useful to inhibit a hypoxia response. For example, tumor growth results in hypoxic conditions. Under such conditions, it may be beneficial to block the hypoxia response in order to slow or stop tumor growth. Pharmaceutical drugs that inhibit or prevent the results of tumor-induced hypoxia are useful in treating such conditions.","Conditions can be tested for their ability to induce or inhibit the hypoxia response, as discussed above. Hypoxia conditions to which cells can be exposed for these tests include, for example, exposure of cells to an environment in which oxygen is present at no more than about 10%, no more than about 5%, no more than about 3%, no more than about 2%, no more than about 1% or below about 1%. Furthermore, small molecule inhibitors of the cells ability to make or use ATP also elicit the hypoxia response, e.g., or treating with deferoxamine (DFO) for 24 hrs. DFO is a known inhibitor of mitochondria and inducer of hypoxia. Also, cells can be exposed to any condition that induces expression of a hypoxia response biomarker promoter.","2.2 Biological Response to Estrogen (e.g., \u03b2-estradiol)","Several promoters have been identified, the activity of each of which is positively correlated with the biological response to estrogen (\u201cestrogen response biomarker promoter\u201d). These estrogen response biomarker promoters are identified in Table 1, section 2 and section 2a. The condition used to induce the biological response to estrogen was exposing cells to a 10 nanomolar environment \u03b2-estradiol for 24 hrs. The experimental conditions are set forth more completely in Example 2.","More specifically, transcription regulatory sequences have been found that are induced when exposed to estrogens. \u03b2-estradiol induces the estrogen receptor protein that regulates the estrogen response. One can use such a promoter in an assay to determine whether a test compound induces the estrogen response. This is done, for example, by exposing a cell containing an expression construct comprising the transcription regulatory sequence operatively linked with a reporter sequence to a test compound. If the compound induces activity of the promoter, this means the compound induces the estrogen response. Also, one can use the promoter in an assay to determine whether a test compound inhibits the estrogen response. This is done, for example, by exposing a cell containing the expression construct to a condition that induces the estrogen response and exposing the cell to the test compound. If the compound inhibits the inducible activity of the promoter, this means the compound inhibits the estrogen response.","The biological response to estrogen (or, \u201cestrogen response\u201d) involves a number of events at the cellular and molecular level. These events include cell proliferation and lipid metabolism. The activity of estrogen response biomarker promoters is, in turn, also correlated with these events.","In certain situations it is useful to elicit an estrogen response. For example, estrogens are administered as drugs in a variety of situations. In a form of hormone replacement therapy, estrogen is administered to peri-menopausal and post-menopausal women to counter the effects of diminished estrogen levels, such as osteoporosis. The estrogen response biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to elicit an estrogen response.","Also, in certain situations it is desirable to inhibit an estrogen response. For example, certain cancers, e.g., certain breast cancers, are hormone receptor positive. That is, cancer cells grow faster in the presence of estrogen. Drugs that lower the amount of estrogen, or that inhibit the activity of estrogen, such as tamoxifen, are used to treat such cancers. The estrogen response biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to inhibit an estrogen response.","Furthermore, it is known than many environmental toxins and other small molecules can act as estrogen mimics. Therefore, an estrogen response biomarker promoter can serve as a useful tool for establishing which toxins or other small molecules can serve as estrogen mimics by, for example, exposing a recombinant cell of this invention to the compound and determining whether it induces activity of an estrogen biomarker promoter.","Conditions can be tested for their ability to induce or inhibit the estrogen response, as discussed above. Estrogen conditions to which cells can be exposed for these tests include, for example, exposure of cells to an environment comprising an estrogen at a concentration of least 100 picomolar, at least 1 nM or at least 10 nM or at least 1 uM. Any estrogen can be used to induce this response. Also, cells can be exposed to any condition that induces expression of an estrogen response biomarker promoter.","2.3 Biological Response to Androgen (e.g., Methyltrienolone)","Several promoters have been identified, the activity of each of which is positively correlated with the biological response to androgen (\u201candrogen response biomarker promoter\u201d). These androgen response biomarker promoters are identified in Table 1, section 3 and section 3a. The condition used to induce the biological response to androgen was exposing cells to a 10 nanomolar environment of methyltrienolone (R1881) for 24 hrs. The experimental conditions are set forth more completely in Example 3.","More specifically, transcription regulatory sequences have been found that are induced when exposed to androgens. Methyltrienolone induces the androgen receptor protein that regulates the androgen response. One can use such a promoter in an assay to determine whether a test compound induces the androgen response. This is done, for example, by exposing a cell containing an expression construct comprising the transcription regulatory sequence operatively linked with a reporter sequence to a test compound. If the compound induces activity of the promoter, this means the compound induces the androgen response. Also, one can use the promoter in an assay to determine whether a test compound inhibits the androgen response. This is done, for example, by exposing a cell containing the expression construct to a condition that induces the androgen response and exposing the cell to the test compound. If the compound inhibits the inducible activity of the promoter, this means the compound inhibits the androgen response.","The biological response to androgen (or, \u201candrogen response\u201d) involves a number of events at the cellular and molecular level. These events include regulating gene expression patterns that result in the formation of male secondary sex characteristics in vertebrates. The activity of androgen response biomarker promoters is, in turn, also correlated with these events.","In certain situations it is useful to elicit an androgen response. For example, in a form of hormone replacement therapy, androgen is administered to older men to counter the effects of diminished androgen levels. The androgen response biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to elicit an androgen response.","Also, in certain situations it is desirable to inhibit an androgen response. For example, androgen ablation can be used as an effective therapy in prostate cancer. The androgen response biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to inhibit an androgen response.","Furthermore, it is known than many environmental toxins and other small molecules can act as androgen mimics. Therefore, an androgen response biomarker promoter can serve as a useful tool for establishing which toxins or other small molecules can serve as androgen mimics.","Androgen conditions to which cells can be exposed for these tests include, for example, exposure of cells to an environment comprising an androgen at a concentration of least 100 picomolar, at least 1 nM or at least 10 nM. Androgens include natural and artificial androgens, such as testosterones, methyltrienolone, and others. Also, cells can be exposed to any condition that induces expression of an androgen response biomarker promoter.","2.4 Biological Response to p53 Activation","Several promoters have been identified, the activity of each of which is positively correlated with the biological response to the activation of the p53 tumor suppressor protein (\u201cp53 response biomarker promoter\u201d). These p53 biomarker promoters are identified in Table 1, section 4 and section 4a. The conditions used to induce the biological response to p53 was exposing cells to a 10 micromolar environment of nutlin for 24 hrs or treatment with doxorubicin at 200 ng\/mL final concentration. The experimental conditions are set forth more completely in Example 4.","More specifically, transcription regulatory sequences have been found that are induced when exposed to activators of p53. Nutlin induces the p53 protein by inhibiting the MDM2 protein that is responsible for the degradation of the p53 protein. Doxorubicin is an antibiotic that interacts with DNA by intercalation. This in turn activates p53. One can use such a promoter in an assay to determine whether a test compound induces the p53 response. This is done, for example, by exposing a cell containing an expression construct comprising the transcription regulatory sequence operatively linked with a reporter sequence to a test compound. If the compound induces activity of the promoter, this means the compound induces the p53 response. Also, one can use the promoter in an assay to determine whether a test compound inhibits the p53 response. This is done, for example, by exposing a cell containing the expression construct to a condition that induces the p53 response and exposing the cell to the test compound. If the compound inhibits the inducible activity of the promoter, this means the compound inhibits the p53 response.","p53 is a master tumor suppressor gene that controls a number of critical events at the cellular and molecular level. The p53 protein is a transcription factor that directly binds to DNA and regulates the expression of genes involved in the cell cycle, the DNA repair pathway, and can initiate apoptosis if DNA damage is irreparable. The activity of p53-responsive biomarker promoters, in turn, also correlate with these events.","p53 is critical to proper functioning of cells. A large number of cancers develop from cells that have inactivated the p53 protein. The p53 response biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to activate the p53 response. Test conditions that stimulate the p53 response may be useful in treating cancer.","Conditions can be tested for their ability to induce or inhibit the p53 response, as discussed above. Conditions to which cells can be exposed for these tests include, for example, exposure of cells to an environment that induces DNA damage, apoptosis, or other disruptions to the cell cycle. Also, cells can be exposed to any condition that induces expression of an p53 response biomarker promoter.","2.5 Biological Response to Cholesterol and other Molecules that Modulate the Cholesterol Biosynthesis Pathway","Several promoters have been identified, the activity of which are positively or negatively correlated with the biological response to the activity of the cholesterol biosynthesis pathway (\u201ccholesterol biomarker promoter\u201d). These cholesterol response biomarker promoters are identified in Table 1, section 5 and section 5a. The conditions used to induce the cholesterol response were exposing cells to a 1 micromolar environment of lovastatin for 24 hrs (negatively correlated with activation of the pathway or positively correlated with inhibition of the pathway), U18666A at 1 micromolar for 24 hrs (negatively correlated with activation of the pathway or positively correlated with inhibition of the pathway), or synthecol at 5 ug\/mL for 24 hrs (positively correlated with activation of the pathway). The experimental conditions are set forth more completely in Example 5.","More specifically, transcription regulatory sequences have been found that are induced or repressed when the cholesterol biosynthesis is either inhibited or induced. Lovastatin is a known inhibitor of the HMG-coA reductase protein that is a critical enzyme in the synthesis of cholesterol. U18666A is a known inhibitor of intracellular cholesterol transport. Synthecol is a synthetic form of cholesterol. One can use such a promoter in an assay to determine whether a test compound affects the cholesterol response. This is done, for example, by exposing a cell containing an expression construct comprising the transcription regulatory sequence operatively linked with a reporter sequence to a test compound. If the compound induces activity of biomarker promoters positively correlated with inhibition of cholesterol biosynthesis (negatively correlated with cholesterol biosynthesis), this means the compound inhibits activity of the cholesterol biosynthesis pathway. If the compound induces activity of biomarker promoters positively correlated with cholesterol biosynthesis this means it induces activity of the cholesterol biosynthesis pathway. Similarly, one can induce or repress the cholesterol synthesis response and determine whether a test condition has the opposite effect, e.g., repressing activity of a biomarker positively correlated with the response or inducing activity of a biomarker negatively correlated with the response.","The cholesterol biosynthesis pathway includes a number of events at the cellular and molecular level. These events include the sensing of intracellular cholesterol in the endoplasmic reticulum by the SREBP protein. SREBP is a transcription factor that directly binds DNA and regulates the transcription of the HMG-CoA reductase and LDL receptor genes. The activity of cholesterol biomarker promoters is, in turn, also correlated with these events.","In certain situations it is useful to inhibit cholesterol biosynthesis as high levels of cholesterol are strongly associated with cardiovascular disease. For example, statins are some of the most widely prescribed drugs that inhibit the activity of HMG-CoA reductase, a key enzyme in the cholesterol biosynthesis pathway. The cholesterol biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to inhibit cholesterol biosynthesis.","Also, in certain situations it may be desirable to activate the cholesterol biosynthesis pathway. There are rare cases of individuals that suffer from hypocholesterolemia, or abnormally low levels of cholesterol. The cholesterol biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to activate cholesterol biosynthesis.","Conditions can be tested for their ability to induce or inhibit the cholesterol biosynthesis pathway, as discussed above. Conditions to which cells can be exposed for these tests include, for example, exposure of cells to any condition that inhibits the production of cholesterol or the uptake of cholesterol. Statins are examples of cholesterol synthesis inhibitors. Also, cells can be exposed to any condition that induces expression of a cholesterol response biomarker promoter.","2.6 Biological Response to Interferons","Several promoters have been identified, the activity of each of which is positively correlated with the biological response to interferons (\u201cinterferon biomarker promoter\u201d). These interferon response biomarker promoters are identified in Table 1, section 6 and section 6a. The conditions used to induce the interferon response were exposing cells to 500 units\/mL of interferon alpha for 8 hrs, or interferon gamma at 100 ng\/mL for 8 hrs. The experimental conditions are set forth more completely in Example 6. Certain of these biomarkers are correlated with the alpha and\/or gamma interferon responses.","More specifically, transcription regulatory sequences have been found that are induced when exposed to interferons, e.g., interferon alpha or interferon gamma. Interferon alpha and gamma induce an interferon-mediated response. One can use such a promoter in an assay to determine whether a test compound induces the interferon-mediated response. This is done, for example, by exposing a cell containing an expression construct comprising the transcription regulatory sequence operatively linked with a reporter sequence to a test compound. If the compound induces activity of the promoter, this means the compound induces the interferon-mediated response. Also, one can use the promoter in an assay to determine whether a test compound inhibits the interferon-mediated response. This is done, for example, by exposing a cell containing the expression construct to a condition that induces the interferon-mediated response (for example, exposing the cell to interferon alpha) and exposing the cell to the test compound. If the compound inhibits the inducible activity of the promoter, this means the compound inhibits the interferon-mediated response.","The biological response to interferons includes a number of events at the cellular and molecular level. Interferons are produced by a wide variety of cells and are key signaling molecules in the inflammation response and in response to the presence of double-stranded RNA viruses. Interferons assist the immune response by inhibiting viral replication within host cells, activating natural killer cells and macrophages, increasing antigen presentation to lymphocytes, and inducing the resistance of host cells to viral infection. The STAT family of transcription factors directly bind DNA and regulate the transcription of target genes in response to interferon signaling. The activity of interferon biomarker promoters is, in turn, also correlated with these events.","In certain situations it is useful to inhibit interferon activity. For example, inhibiting interferon activity may help in cases of acute or chronic inflammation. The interferon biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to inhibit interferon activity.","Also, in certain situations it may be desirable to activate interferon activity. For example, interferons have antiviral and antioncogenic properties and are used to treat viral infection and cancer. The interferon biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to activate interferon activity.","Conditions can be tested for their ability to induce or inhibit interferon activity, as discussed above. Conditions to which cells can be exposed for these tests include, for example, exposure of cells to interferon alpha or interferon gamma, or any condition that inhibits or activates the production of interferon proteins, such as a viral infection. Also, cells can be exposed to any condition that induces expression of an interferon biomarker promoter.","2.7. CREB-mediated Response and Cyclic AMP Signaling","Several promoters have been identified, the activity of each of which is positively correlated with the biological response to cAMP or CREB signaling (\u201cCREB biomarker promoter\u201d). These CREB response biomarker promoters are identified in Table 1, section 7 and section 7a. The conditions used to induce the cholesterol response were exposing cells to 20 micromolar of forskolin for 4 hrs, or phorbol 12-myristate 13-acetate (PMA) at 100 nanomolar for 4 hrs. The experimental conditions are set forth more completely in Example 7.","More specifically, transcription regulatory sequences have been found that are induced when exposed to conditions that activate the CREB signaling cascade. Forskolin is a known activator of protein kinase A that raises cellular cyclic AMP (cAMP) levels. The CREB protein is a transcription factor that is activated by protein kinase A and directly binds DNA to activate target genes. PMA is a known activator of protein kinase C that also activates the CREB signaling cascade. One can use such a promoter in an assay to determine whether a test compound induces a CREB-mediated response. This is done, for example, by exposing a cell containing an expression construct comprising the transcription regulatory sequence operatively linked with a reporter sequence to a test compound. If the compound induces activity of the promoter, this means the compound induces a CREB-mediated response. Also, one can use the promoter in an assay to determine whether a test compound inhibits a CREB-mediated response. This is done, for example, by exposing a cell containing the expression construct to a condition that induces a CREB-mediated response (for example, exposing the cell to forskolin) and exposing the cell to the test compound. If the compound inhibits the inducible activity of the promoter, this means the compound inhibits the CREB-mediated response.","The biological response to CREB-mediated cyclic AMP signaling includes a number of events at the cellular and molecular level. cAMP is a secondary messenger used for intracellular signaling that ultimately activates the CREB transcription factor. CREB-mediated cyclic AMP signaling is involved in many biological functions including regulating the effects of glucagons and adrenaline. The activity of CREB biomarker promoters is, in turn, also correlated with these events.","In certain situations it may be useful to inhibit certain aspects of CREB activity. For example, inhibiting CREB activity may help in cases of metabolic disorders or signaling through certain G-protein coupled receptor (GPCR) pathways. The CREB biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to inhibit CREB activity.","Also, in certain situations it may be desirable to activate CREB activity. For example, activating CREB activity may help in cases of metabolic disorders or signaling through certain GPCR pathways. The CREB biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to activate CREB activity.","Conditions can be tested for their ability to induce or inhibit CREB activity, as discussed above. Conditions to which cells can be exposed for these tests include, for example, exposure of cells to any conditions that change cAMP levels within the cell, the stimulation of any GPCR protein, or other ways of activating the CREB protein e.g., exposing cells to biogenic amines, chemokines, or peptide hormones. Also, cells can be exposed to any condition that induces expression of a CREB biomarker promoter.","2.8 Glucocorticoid Response","Several promoters have been identified, the activity of each of which is positively correlated","with the biological response to glucocorticoid receptor (GR) signaling (\u201cGR biomarker promoter\u201d). These GR response biomarker promoters are identified in Table 1, section 8 and section 8a. The conditions used to induce the GR response were exposing cells to 100 nanomolar dexamethasone, 1 micromolar prednisone, and 1 micromolar cortisone, each for 4 hrs. The experimental conditions are set forth more completely in Example 8.","More specifically, transcription regulatory sequences have been found that are induced when exposed to conditions that induce the GR-mediated response. Dexamethasone, prednisone, and cortisone are known ligands and activators of the GR protein. GR is a ligand-mediated transcription factor that binds directly to the DNA of target genes in the genome when it is bound to an activating ligand. Dexamethasone and prednisone are also potent anti-inflammatory therapeutic compounds. Activating GR with synthetic ligands is known to inhibit the inflammatory response, and is therefore, GR is an important drug target. One can use such a promoter in an assay to determine whether a test compound induces a GR-mediated response. This is done, for example, by exposing a cell containing an expression construct comprising the transcription regulatory sequence operatively linked with a reporter sequence to a test compound. If the compound induces activity of the promoter, this means the compound induces a GR-mediated response. Also, one can use the promoter in an assay to determine whether a test compound inhibits a GR-mediated response. This is done, for example, by exposing a cell containing the expression construct to a condition that induces a GR-mediated response (for example, exposing the cell to dexamethasone) and exposing the cell to the test compound. If the compound inhibits the inducible activity of the promoter, this means the compound inhibits the GR-mediated response.","The biological response to ligands for GR (or, \u201cGR response\u201d) involves a number of events at the cellular and molecular level. These events include GR-mediated expression of genes controlling development, metabolism, and the immune response. The activity of GR response biomarker promoters is, in turn, also correlated with these events.","In certain situations it is useful to activate a GR response. Activating GR activity is known to have potent anti-inflammatory effects. The GR response biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to elicit a GR response.","Also, in certain situations it is desirable to inhibit a GR response. For example, inhibiting GR may help in cases of metabolic disorders. The GR response biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to inhibit a GR response.","Furthermore, it is known than many environmental toxins and other small molecules can act as synthetic mimics of GR ligands. Therefore, an GR response biomarker promoter can serve as a useful tool for establishing which toxins or other small molecules can serve as GR ligand mimics by, for example, exposing a recombinant cell of this invention to the compound and determining whether it induces activity of an GR biomarker promoter.","Conditions can be tested for their ability to induce or inhibit the GR response, as discussed above. GR activating conditions to which cells can be exposed for these tests include, for example, any compound or condition that simulates GR binding to its natural ligands.","2.9. PPAR-mediated Response","Several promoters have been identified, the activity of each of which is positively correlated with the biological response to peroxisome proliferator-activated receptor (PPAR) signaling (\u201cPPAR biomarker promoter\u201d). These PPAR response biomarker promoters are identified in Table 1, section 9 and section 9a. The conditions used to induce the PPAR response were exposing cells to PPAR alpha and 75 micromolar of WY14643, PPAR gamma and 10 micromolar of ciglitazone, or PPAR delta and 100 nanomolar of GW501516 for 24 hrs. The experimental conditions are set forth more completely in Example 9.","More specifically, transcription regulatory sequences have been found that are induced when the PPAR-mediated response is activated. WY14643, ciglitazone and GW501516 are known ligands and activators of the PPAR alpha, gamma, and delta proteins, respectively. The PPAR family of transcription factors is a family of ligand-mediated transcription factors that binds directly to the DNA of target genes in the genome when it is bound to an activating ligand. One can use such a promoter in an assay to determine whether a test compound induces a PPAR-mediated response. This is done, for example, by exposing a cell containing an expression construct comprising the transcription regulatory sequence operatively linked with a reporter sequence to a test compound. If the compound induces activity of the promoter, this means the compound induces a PPAR-mediated response. Also, one can use the promoter in an assay to determine whether a test compound inhibits a PPAR-mediated response. This is done, for example, by exposing a cell containing the expression construct to a condition that induces a PPAR-mediated response (for example, exposing the cell to ciglitazone) and exposing the cell to the test compound. If the compound inhibits the inducible activity of the promoter, this means the compound inhibits the PPAR-mediated response.","The biological response to ligands binding to and activating PPAR (or \u201cPPAR response\u201d) involves a number of events at the cellular and molecular level. These events include PPAR-mediated expression of genes that play essential roles in the regulation of cellular differentiation, development, lipid metabolism, and tumorigenesis in higher organisms. The activity of PPAR response biomarker promoters is, in turn, also correlated with these events.","In certain situations it is useful to activate a PPAR response. Activating PPAR activity is known to have anti-diabetic effects. The PPAR response biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to elicit a PPAR response.","Also, in certain situations it is desirable to inhibit a PPAR response. For example, inhibiting PPAR may help in cases of metabolic disorders. The PPAR response biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to inhibit a PPAR response.","Furthermore, it is known than many environmental toxins and other small molecules can act as synthetic mimics of PPAR ligands. Therefore, a PPAR response biomarker promoter can serve as a useful tool for establishing which toxins or other small molecules can serve as PPAR ligand mimics by, for example, exposing a recombinant cell of this invention to the compound and determining whether it induces activity of an PPAR biomarker promoter.","Conditions can be tested for their ability to induce or inhibit the PPAR response, as discussed above. PPAR activating conditions to which cells can be exposed for these tests include, for example, any compound or condition that simulates PPAR binding to its ligand binding domain.","2.10 RAR-mediated Response","Several promoters have been identified, the activity of each of which is positively correlated with the biological response to retinoic acid receptor (RAR) signaling (\u201cRAR biomarker promoter\u201d). These RAR response biomarker promoters are identified in Table 1, section 10 and section 10a. The conditions used to induce the RAR response was exposing cells to the RAR beta protein and 100 nanomolar of adapalene for 24 hrs. The experimental conditions are set forth more completely in Example 10.","More specifically, transcription regulatory sequences have been found that are induced when upon activation of the RAR-mediated response. Adapalene is a known ligand and activator of the RAR beta protein. RAR is a ligand-mediated transcription factor that binds directly to the DNA of target genes in the genome when it is bound to an activating ligand. One can use such a promoter in an assay to determine whether a test compound induces a RAR-mediated response. This is done, for example, by exposing a cell containing an expression construct comprising the transcription regulatory sequence operatively linked with a reporter sequence to a test compound. If the compound induces activity of the promoter, this means the compound induces a RAR-mediated response. Also, one can use the promoter in an assay to determine whether a test compound inhibits a RAR-mediated response. This is done, for example, by exposing a cell containing the expression construct to a condition that induces a RAR-mediated response (for example, exposing the cell to adapalene) and exposing the cell to the test compound. If the compound inhibits the inducible activity of the promoter, this means the compound inhibits the RAR-mediated response.","The biological response to ligands binding to and activatin RAR (or, \u201cRAR response\u201d) involves a number of events at the cellular and molecular level. These events include RAR-mediated expression of genes that play essential roles in the regulation of cellular differentiation and development. The activity of RAR response biomarker promoters is, in turn, also correlated with these events.","In certain situations it is useful to activate a RAR response. Activating RAR activity through various retinoids is known to play roles in vision, cell proliferation, bone growth, and immune function. The RAR response biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to elicit a RAR response.","Also, in certain situations it is desirable to inhibit a RAR response. For example, inhibiting RAR may help in cases of carcinogenesis or tumor growth. The RAR response biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to inhibit a RAR response.","Furthermore, it is known than many environmental toxins and other small molecules can act as synthetic mimics of retinoids or other RAR ligands. Therefore, a RAR response biomarker promoter can serve as a useful tool for establishing which toxins or other small molecules can serve as RAR ligand mimics by, for example, exposing a recombinant cell of this invention to the compound and determining whether it induces activity of an RAR biomarker promoter.","Conditions can be tested for their ability to induce or inhibit the RAR response, as discussed above. RAR activating conditions to which cells can be exposed for these tests include, for example, any compound or condition that simulates RAR binding to its ligand binding domain.","2.11. Biological Response to TNFa and NFkB Activation","Several promoters have been identified, the activity of each of which is positively correlated with the biological response to tumor necrosis factor, alpha (TNFa) that is mediated by nuclear factor kappa-light-chain-enhancer of activated B cells (NFkB) (\u201cNFkB biomarker promoter\u201d). These NFkB biomarker promoters are identified in Table 1, section 11 and section 11a. The conditions used to induce the NFkB response was exposing cells to 20 ng\/mL of TNFa for 8 hrs. The experimental conditions are set forth more completely in Example 11.","More specifically, transcription regulatory sequences have been found that are induced when exposed to TNFa. TNFa is a cytokine involved in systemic inflammation and is a member of a group of cytokines that stimulate the acute phase reaction. NFkB is a heterodimeric transcription factor that translocates to the nucleus and mediates the transcription of a vast array of proteins involved in cell survival and proliferation, inflammatory response, and anti-apoptotic factors and is activated by TNFa. One can use such a promoter in an assay to determine whether a test compound induces a NFkB-mediated response. This is done, for example, by exposing a cell containing an expression construct comprising the transcription regulatory sequence operatively linked with a reporter sequence to a test compound. If the compound induces activity of the promoter, this means the compound induces a NFkB-mediated response. Also, one can use the promoter in an assay to determine whether a test compound inhibits a NFkB-mediated response. This is done, for example, by exposing a cell containing the expression construct to a condition that induces a NFkB-mediated response (for example, exposing the cell to TNFa) and exposing the cell to the test compound. If the compound inhibits the inducible activity of the promoter, this means the compound inhibits the NFkB-mediated response.","The biological response to TNFa the \u201cinflammation response induced by TNFa\u201d involves a number of events at the cellular and molecular level. These events include NFkB-mediated expression of genes controlling systemic inflammation. The activity of NFkB response biomarker promoters is, in turn, also correlated with these events.","In certain situations it is useful to activate an NFkB response. Activating NFkB activity is known to stimulate the immune system. The NFkB response biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to elicit an NFkB response.","Also, in certain situations it is desirable to inhibit a NFkB response. For example, inhibiting NFkB may inhibit the effects of chronic inflammation. The NFkB response biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to inhibit a NFkB response.","Conditions can be tested for their ability to induce or inhibit the NFkB response, as discussed above. NFkB activating conditions to which cells can be exposed for these tests include, for example, any compound or condition that simulates NFkB binding to genomic targets and affecting transcription of those target genes.","2.12. Heat Shock Response","Several promoters have been identified, the activity of each of which is positively correlated with the biological response to hyperthermia or heat shock signaling (\u201cheat shock biomarker promoter\u201d). These heat shock response biomarker promoters are identified in Table 1, section 12 and section 12a. The conditions used to induce the heat shock response was exposing cells to 43 degrees Celsius for 8 hrs. The experimental conditions are set forth more completely in Example 12.","More specifically, transcription regulatory sequences have been found that are induced when exposed to heat shock. When cells are exposed to elevated temperatures or other conditions that result in misfolded proteins, they respond by increasing the expression of chaperone proteins that help to stabilize proteins at higher temperatures. This response is mediated by the family of heat shock transcription factors. One can use such a promoter in an assay to determine whether a test compound induces the heat shock response. This is done, for example, by exposing a cell containing an expression construct comprising the transcription regulatory sequence operatively linked with a reporter sequence to a test compound. If the compound induces activity of the promoter, this means the compound induces a heat shock response. Also, one can use the promoter in an assay to determine whether a test compound inhibits a heat shock response. This is done, for example, by exposing a cell containing the expression construct to a condition that induces a heat shock response (for example, exposing the cell to 43 degrees C.) and exposing the cell to the test compound. If the compound inhibits the inducible activity of the promoter, this means the compound inhibits the heat shock response.","The biological response to heat shock (\u201cheat shock response\u201d) involves a number of events at the cellular and molecular level. These events include heat shock transcription factor-mediated expression of genes encoding molecular chaperones. The activity of heat shock response biomarker promoters is, in turn, also correlated with these events.","In certain situations it is useful to activate the heat shock response. Activating chaperone proteins may serve as a protection in conditions that may otherwise denature proteins. The heat shock response biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to elicit a heat shock response.","Also, in certain situations it is desirable to inhibit the heat shock response. For example, inhibiting heat shock may inhibit the effects of chronic inflammation. The heat shock response biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to inhibit a heat shock response.","Conditions can be tested for their ability to induce or inhibit the heat shock response, as discussed above. Heat shock activating conditions to which cells can be exposed for these tests include, for example, any compound or condition that denatures or otherwise affects native protein folding.","2.13. Serum Response","Several promoters have been identified, the activity of each of which is correlated with the biological response to serum (\u201cserum response biomarker promoter\u201d). These serum response biomarker promoters are identified in Table 1, section 13 and section 13a. The conditions used to induce the serum response was exposing cells to 20% fetal bovine serum for 8 hrs. The experimental conditions are set forth more completely in Example 13.","More specifically, transcription regulatory sequences have been found that are induced when exposed to serum. When cells are exposed to serum, they activate the serum response transcription factor that directly regulates a set of target genes. One can use such a promoter in an assay to determine whether a test compound induces the serum response. This is done, for example, by exposing a cell containing an expression construct comprising the transcription regulatory sequence operatively linked with a reporter sequence to a test compound. If the compound induces activity of the promoter, this means the compound induces a serum response. Also, one can use the promoter in an assay to determine whether a test compound inhibits a serum response. This is done, for example, by exposing a cell containing the expression construct to a condition that induces a serum response (for example, exposing the cell to fetal bovine serum) and exposing the cell to the test compound. If the compound inhibits the inducible activity of the promoter, this means the compound inhibits the serum response.","The biological response to serum involves a number of events at the cellular and molecular level. These events include serum response transcription factor-mediated expression of genes involved in cell-cycle regulation, development, and muscle growth. The activity of serum response biomarker promoters is, in turn, also correlated with these events.","In certain situations it is useful to activate the serum response. The serum response in some cases may stimulate muscle growth when needed. The serum response biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to elicit a serum response.","Also, in certain situations it is desirable to inhibit the serum response. For example, inhibiting serum may inhibit tumorigenesis or unregulated cell division. The serum response biomarker promoters of this invention are useful in discovering and evaluating pharmaceutical drugs and other conditions for the ability to inhibit a serum response.","Conditions can be tested for their ability to induce or inhibit the serum response, as discussed above. Serum response activating conditions to which cells can be exposed for these tests include, for example, any compound or condition that simulates the signaling molecules found in serum.","3. Expression Constructs","An expression construct is a recombinant nucleic acid molecule comprising at least one expression control sequence operatively linked with another nucleotide sequence. A recombinant nucleic acid molecule is a nucleic acid molecule comprising two sequences that are not naturally attached to one another, for example, sequences from different species. Two sequences are operably linked when they are placed in a functional relationship with each other so that activity of a first sequence results in an action on the other part. An expression control sequence is operatively linked with a nucleotide sequence when expression of the nucleotide sequence is regulated by the action of the expression control sequence. Expression control sequences include those that regulate transcription of a nucleotide sequence, stability of the transcript or translation of the transcript.","This invention provides expression constructs comprising a biomarker transcription regulatory sequence operatively linked with a reporter sequence. The expression constructs of this invention can be incorporated into a vector or integrated, stably or transiently, into a host cell chromosome.","3.1. Transcription Regulatory Sequences","Transcription regulatory sequences are nucleotide sequences that up- or down-regulate transcription of another nucleotide sequence to which they are operatively linked, e.g., a gene.","3.1.1. Promoters","Promoters are the best-characterized transcriptional regulatory sequences because of their predictable location immediately upstream of transcription start sites. Promoters include sequences that modulate the recognition, binding and transcription initiation activity of the RNA polymerase. These sequences can be cis acting or can be responsive to trans acting factors. Promoters, depending upon the nature of the regulation, can be constitutive or regulated. They are often described as having two separate segments: core and extended promoter regions.","The core promoter includes sequences that are sufficient for RNA polymerase recognition, binding and transcription initiation. The core promoter includes the transcriptional start site, an RNA polymerase binding site and other general transcription binding sites and is where the pre-initiation complex forms and the general transcription machinery assembles. It is generally within 50 nucleotides (nt) of the transcription start site (TSS).","The extended promoter region includes the so-called proximal promoter, which extends to about 250 nucleotides upstream of the transcriptional start site (i.e., \u2212250 nt). It includes primary regulatory elements such as specific transcription factor binding sites. It has been found that many genes have transcription regulatory elements located further up-stream. In particular, a fragment that includes most of the transcription regulatory elements of a gene can extend up to 700 nt or more up-stream of the transcription start site. (See, e.g., U.S. 2007-0161031.) In certain genes, transcription regulatory sequences have been found thousands of nucleotides upstream of the transcriptional start site.","3.1.2. Other Transcription Regulatory Sequences","Transcription regulatory sequences include nucleotide sequences that confer inducible expression of a gene (i.e., that require a substance or stimulus for increased transcription). When an inducer is present, or present at increased concentration, gene expression increases. Regulatory regions also include sequences that confer repression of gene expression (i.e., a substance or stimulus decreases transcription). When a repressor is present or at increased concentration, gene expression decreases. Regulatory regions typically bind one or more trans-acting or cis-acting proteins. Enhancers are known to influence gene expression when positioned 5\u2032 or 3\u2032 of the gene, or when positioned in or a part of an exon or an intron. Enhancers also can function at a significant distance from the gene, for example, at a distance from about 3 Kb, 5 Kb, 7 Kb, 10 Kb, 15 Kb or more.","Regulatory regions also include, in addition to transcription regulatory sequences, sequences in DNA or RNA molecules that regulate transcript stability, transcript localization, facilitate translation, splicing signals for introns, maintenance of the correct reading frame of the gene to permit in-frame translation of mRNA and, stop codons, leader sequences and fusion partner sequences, internal ribosome binding sites (IRES) elements for the creation of multigene, or polycistronic, messages, polyadenylation signals to provide proper polyadenylation of the transcript of a gene of interest and stop codons and can be optionally included in an expression vector.","3.1.3. Biomarker Transcription Regulatory Sequences","This invention provides isolated nucleic acid sequences that function as biomarker transcriptional regulatory sequences. As is well understood, DNA can be double stranded. The sequences of both strands, that is, one strand and its complement, are considered sequences useful in this invention.","Expression constructs comprising the sequences of the SEQ ID NOs of Table 1 operatively linked with a reporter sequence (e.g., a sequence encoding luciferase) were found to have activity correlated with the biological response indicated. Thus, transcription regulatory sequences that regulate transcription of genes identified in Table 1 are useful as biomarker transcription regulatory sequences for those responses or pathways.","The sequences of Table 1 were selected from transcription regulatory sequences derived from an analysis of the human genome. The analysis is based, in part, on work described in Cooper et al., Trinklein et al., US 2007-0161031 and US 2009-0018031. In short, full-length cDNAs from the Mammalian Gene Collection were aligned against the human genome. Sequences with less than 95% sequence identity, those with more than 200 bases at the 5\u2032 end of the cDNA that do not align with the genome, those that align to random sequence not assembled into reference chromosome sequences and those that represent alignment artifacts were filtered out. Alignment artifacts are identified as intronless (ungapped) alignments represented by a small number of independent cDNAs from existing cDNA libraries, as pseudogenes and as single exon genes. More specifically, a library of genetic sequences, such as GenBank, contains a number of molecules reported as cDNAs. When these sequences are aligned against the sequence of the genome, certain locations of the genome are mapped by many reported cDNAs, so that the alignment cannot be considered random: One can be highly confident that these locations represent biologically relevant cDNAs and that the up-stream sequences are active transcription regulatory sequences. Other locations in the genome are mapped by few reported cDNAs or none. If the cDNA sequences are unspliced (that is they contain no introns) and the number of cDNAs mapping to a location in the genome is no more than what one would expect under a random model, then these alignments are considered artifacts. Sequences upstream of the 5\u2032 end of these cDNAs (the transcription start site) were selected as transcription regulatory sequence.","Individual genes can have more than one transcriptional start site, and in turn multiple promoters containing different transcriptional regulatory sequences. Therefore in many cases we experimentally tested multiple promoters for each gene to identify the precise promoter that regulates the biological response of interest.","Each sequence identified in Table 1 by SEQ ID NO is useful as biomarker regulatory sequence of this invention. For example, the transcription regulatory sequence can be the entire sequence of the given SEQ ID NO. Alternatively, transcription regulatory sequence can be comprised within a longer sequence from the genome from which the sequence of the SEQ ID NO is taken. For example, this sequence can be upstream of the 5\u2032 end indicated in the SEQ ID NO. The sequences of Table 1 are human sequences. In other embodiments, the transcription regulatory sequence can comprise a fragment of a sequence of Table 1. Such fragments can comprise at least 25 nt, at least 50 nt, at least 100 nt, at least 250 nt, at least 500 nt, at least 600 nt, at least 700 nt or at least 900 nt of the sequence of the provided SEQ ID NO.","Any transcription regulatory sequence of a biomarker gene of Table 1 can be used in the expression constructs of this invention. This includes, for example, the transcription start site, the core promoter, the extended promoter, an enhancer region, an inducer region, a repressor region, insulators, suppressors or transcription factor binding site. For example, the transcription regulatory sequence can comprise a sequence sufficient to initiate transcription of a second sequence to which it is operatively linked.","In certain embodiments, the sequence includes a sequence extending at least 250 nucleotides up-stream of the transcription start site. However, closer fidelity of activity to the natural environment can be achieved by including transcription regulatory sequences that are further up-stream of the transcription start site, e.g., those that exist in the extended promoter region. For example, the sequence can extend at least or no more than any of \u2212300 nt, \u2212400 nt, \u2212500 nt, \u2212600 nt, \u2212700 nt, \u2212800 nt, \u2212900 nt, \u22121000 nt, \u22121200 nt, \u22121500 nt, \u22121800 nt, \u22122000 nt or \u22123000 nt from a transcription start site. Also, the sequence can comprise sequence downstream of the transcription start site. For example, up to +10 nt, +20 nt, up to +50 nt, up to +100 nt or more than +100 nt. All combinations of these ranges are contemplated. So, for example, the transcription regulatory sequence can include, consist essentially of, or consist of a sequence that spans a range from about +100 to about \u22123000, about +50 to about \u22122000, about +20 to about \u22121800, about +20 to about \u22121500, about +10 to about \u22121500, about +10 to about \u22121200, about +20 to about \u22121000, about +20 to about \u2212900, about +20 to about \u2212800, about +20 to about \u2212700, about +20 to about \u2212600, about +20 to about \u2212500, about +20 to about \u2212400, or about +20 to about \u2212300, relative to a transcription start site.","The transcription regulatory sequence can be any polymorphic form of a sequence of Table 1. A polymorphic form is one of two or more genetically determined alternative sequences or alleles in a population. The site of the alternative form is referred to as the locus. Polymorphic forms can occur at any frequency in a population. Alleles are those polymorphic forms that occur with a frequency of preferably greater than 1%, greater than 10% or greater than 20% of a population. A polymorphism may comprise one or more base changes, an insertion, a repeat, or a deletion. A polymorphic locus may be as small as one base pair. Polymorphic forms can include single nucleotide polymorphisms (SNPs), insertions, deletions and repeated sequences. For example, polymorphisms include variable number of tandem repeats (VNTR's), hypervariable regions, minisatellites, dinucleotide repeats, trinucleotide repeats, tetranucleotide repeats, simple sequence repeats, and insertion elements such as Alu. The allelic form occurring most frequently in a selected population is sometimes referred to as the wildtype form. It is estimated that there are as many as 3\u00d710SNPs in the human genome. Diploid organisms may be homozygous or heterozygous for allelic forms. A diallelic polymorphism has two forms. A triallelic polymorphism has three forms. A polymorphism between two nucleic acids can occur naturally, or be caused by exposure to or contact with chemicals, enzymes, or other agents, or exposure to agents that cause damage to nucleic acids, for example, ultraviolet radiation, mutagens or carcinogens.","Polymorphic forms typically will share at least 95% sequence identity, at least 96% sequence identity, at least 97% sequence identity, at least 98% sequence identity, at least 99% sequence identity, at least 99.5% sequence identity, at least 99.75% sequence identity, at least 99.9% sequence identity over a comparison window of at least 200 nucleotides with a sequence of any the SEQ ID NOs given here. Alternatively, the sequence can hybridize under stringent conditions with a sequence of any SEQ ID NO given here. Such a sequence can be at least 200 nt, at least 500 nt, at least 600 nt, at least 700 nt, at least 800 nt, etc.","3.1.4 Sequence Identity and Hybridization","Terms used to describe sequence relationships between two or more nucleotide sequences or amino acid sequences include \u201creference sequence,\u201d \u201cselected from,\u201d \u201ccomparison window,\u201d \u201cidentical,\u201d \u201cpercentage of sequence identity,\u201d \u201csubstantially identical,\u201d \u201ccomplementary,\u201d and \u201csubstantially complementary.\u201d","A \u201creference sequence\u201d is a defined sequence used as a basis for a sequence comparison and may be a subset of a larger sequence, e.g., a complete cDNA, protein, or gene sequence.","Because two polynucleotides or polypeptides each may comprise (1) a sequence (i.e., only a portion of the complete polynucleotide or polypeptide sequence) that is similar between the two polynucleotides, or (2) a sequence that is divergent between the two polynucleotides, sequence comparisons between two (or more) polynucleotides or polypeptides are typically performed by comparing sequences of the two polynucleotides over a \u201ccomparison window\u201d to identify and compare local regions of sequence similarity.","A \u201ccomparison window\u201d refers to a conceptual segment of typically at least 12 consecutive nucleotides or 4 consecutive amino acid residues that is compared to a reference sequence. The comparison window frequently has a length of at least 15 or at least 25 nucleotides or at least 5 or at least 8 amino acids. The comparison window may comprise additions or deletions (i.e., gaps) of about 20 percent or less as compared to the reference sequence (which does not comprise additions or deletions) for optimal alignment of the two sequences. Optimal alignment of sequences for aligning a comparison window may be conducted by computerized implementations of algorithms (GAP, BESTFIT, FASTA, and TFASTA in the Wisconsin Genetics Software Package Release 7.0, Genetics Computer Group, 575 Science Dr., Madison, Wis.) or by inspection, and the best alignment (i.e., resulting in the highest percentage of homology over the comparison window) generated by any of the various methods is selected.","A subject nucleotide sequence or amino acid sequence is \u201cidentical\u201d to a reference sequence if the two sequences are the same when aligned for maximum correspondence over the length of the nucleotide or amino acid sequence.","The \u201cpercentage of sequence identity\u201d between two sequences is calculated by comparing two optimally aligned sequences over a comparison window, determining the number of positions at which the identical nucleotide or amino acid occurs in both sequences to yield the number of matched positions, dividing the number of matched positions by the total number of positions in the window of comparison (i.e., the window size), and multiplying the result by 100 to yield the percentage of sequence identity.","Unless otherwise specified, the comparison window used to compare two sequences is the length of the shorter sequence.","Methods are described further in Natl. Acad. Sci. USA 85:2444; Higgins & Sharp (1988) Gene 73:237-244; Higgins & Sharp, CABIOS 5:151-153 (1989); Corpet et al. (1988) Nucleic Acids Research 16:10881-90; Huang et al. (1992) Computer Applications in the Biosciences 8:155-65; and Pearson et al. (1994) Methods in Molecular Biology 24:307-31. Alignment is also often performed by inspection and manual alignment.","A subject nucleotide sequence or amino acid sequence is \u201csubstantially identical\u201d to a reference sequence if the subject amino acid sequence or nucleotide sequence has at least 80% sequence identity over a comparison window. Thus, sequences that have at least 85% sequence identity, at least 90% sequence identity, at least 95% sequence identity, at least 98% sequence identity or at least 99% sequence identity with the reference sequence are also \u201csubstantially identical. Two sequences that are identical to each other are, of course, also \u201csubstantially identical\u201d.","\u201cHybridizing specifically to\u201d or \u201cspecific hybridization\u201d or \u201cselectively hybridize to\u201d, refers to the binding, duplexing, or hybridizing of a nucleic acid molecule preferentially to a particular nucleotide sequence under stringent conditions when that sequence is present in a complex mixture (e.g., total cellular) DNA or RNA.","The term \u201cstringent conditions\u201d refers to conditions under which a probe will hybridize preferentially to its target subsequence, and to a lesser extent to, or not at all to, other sequences. \u201cStringent hybridization\u201d and \u201cstringent hybridization wash conditions\u201d in the context of nucleic acid hybridization experiments such as Southern and northern hybridizations are sequence dependent, and are different under different environmental parameters. An extensive guide to the hybridization of nucleic acids is found in Tijssen (1993) Laboratory Techniques in Biochemistry and Molecular Biology\u2014Hybridization with Nucleic Acid Probes part I chapter 2 \u201cOverview of principles of hybridization and the strategy of nucleic acid probe assays\u201d, Elsevier, N.Y.","Generally, highly stringent hybridization and wash conditions are selected to be about 5\u00b0 C. lower than the thermal melting point (Tm) for the specific sequence at a defined ionic strength and pH. The Tm is the temperature (under defined ionic strength and pH) at which 50% of the target sequence hybridizes to a perfectly matched probe. Very stringent conditions are selected to be equal to the Tm for a particular probe.","An example of stringent hybridization conditions for hybridization of complementary nucleic acids which have more than 100 complementary residues on a filter in a Southern or northern blot is 50% formalin with 1 mg of heparin at 42\u00b0 C., with the hybridization being carried out overnight. An example of highly stringent wash conditions is 0.15 M NaCl at 72\u00b0 C. for about 15 minutes. An example of stringent wash conditions is a 0.2\u00d7SSC wash at 65\u00b0 C. for 15 minutes (see, Sambrook et al. for a description of SSC buffer). Often, a high stringency wash is preceded by a low stringency wash to remove background probe signal. An example medium stringency wash for a duplex of, e.g., more than 100 nucleotides, is 1\u00d7SSC at 45\u00b0 C. for 15 minutes. An example low stringency wash for a duplex of, e.g., more than 100 nucleotides, is 4-6\u00d7SSC at 40\u00b0 C. for 15 minutes. In general, a signal to noise ratio of 2\u00d7(or higher) than that observed for an unrelated probe in the particular hybridization assay indicates detection of a specific hybridization.","3.1.5 Collections","In certain embodiments, this invention provides collections that comprise the biomarker transcription regulatory sequences of this invention. The collections include isolated nucleic acids, that is, nucleic acids isolated away from their natural environment or existing in a composition in which the nucleic acid is the predominant species of organic or non-solvent molecule. The collections also include expression constructs of this invention and cells comprising the expression constructs of this invention.","In certain embodiments, the transcription regulatory sequences of the collection consist of or consist essentially of the biomarker transcription regulatory sequences of this invention. In other embodiments, at least 20%, at least 30%, at least 40%, at least 50%, at least 60%, at least 70%, at least 90%, at least 95%, at least 97% or at least 99% of the transcription regulatory sequences of the collection are transcription regulatory sequences of this invention. So, for example, at least 20% of the transcription regulatory sequences of the collection can be biomarkers selected from a selected from a section of Table 1, which are biomarkers for the same biological response. Alternatively, the at least 20% can be selected from biomarker transcription regulatory sequences of at least 2, at least 3, at least 4, at least 5, at least 6, at least 7, at least 8, at least 9, at least 10, at least 11, at least 12 or at least 13 different biological responses, e.g., different sections of Table 1, in which case, biomarkers for a plurality of different biological responses will be represented. The collections can comprise no more than 2, no more than 5, no more than 10, no more than 50, no more than 100, no more than 500, no more than 1000 or no more than 5000 transcription regulatory sequences.","3.2. Reporters","In certain embodiments, a transcription regulatory sequence is operatively linked with a heterologous reporter sequence. A reporter sequence is heterologous if it is not naturally under transcriptional regulatory control of the transcription regulatory sequence. For example the transcription regulatory sequence and the reporter can be from different species or the reporter can be an artificially constructed sequence.","A reporter sequence can be any sequence that allows for the detection of a molecule of interest, such as a protein expressed by a cell, or a biological particle. Typical reporter sequences include, include, for example, those encoding light emitting proteins (e.g., luciferase (see, e.g., U.S. Pat. No. 5,292,658), fluorescent proteins (e.g., red, blue and green fluorescent proteins (see, e.g., U.S. Pat. No. 6,232,107, which provides GFPs from species and other species and U.S. Pat. No. 5,625,048), lacZ (e.g., from ), alkaline phosphatase, secreted embryonic alkaline phosphatase (SEAP), chloramphenicol acetyl transferase (CAT), hormones and cytokines and other such well-known proteins. For expression in cells, a nucleic acid encoding the reporter sequence can be expressed as a fusion protein with a protein of interest or under to the control of a promoter of interest. The expression of these reporter genes can also be monitored by measuring levels of mRNA transcribed from these genes or from detection of the translated polypeptide.","In collections using proteins that emit a detectable signal it may be useful, but not essential, for all of the reporter proteins to emit the same signal. This simplifies detection during high-throughput methods.","Alternatively, the expression constructs in the collection can contain different reporter sequences that emit different detectable signals. For example, each reporter sequence can be a light emitting reporter that emits light of a different, distinguishable color. Alternatively, the reporter sequence in each of the constructs can be a unique, pre-determined nucleotide barcode. This allows assaying a large number of the nucleic acid segments in the same batch or receptacle of cells. In an embodiment, in each construct a unique promoter sequence is cloned upstream of a unique barcode reporter sequence yielding a unique promoter\/barcode reporter combination. The active promoter can drive the production of a transcript containing the unique barcode sequence. Thus, in a collection of expression constructs, each promoter's activity produces a unique transcript whose level can be measured. Since each reporter is unique, the library of expression constructs can be transfected into one large pool of cells (as opposed to separate wells) and all of the RNAs may be harvested as a pool. The levels of each of the barcoded transcripts can be detected using a microarray with the complementary barcode sequences. So the amount of fluorescence on each array spot corresponds to the strength of the promoter that drove the nucleotide barcode's transcription.","In certain embodiments of this invention, a cell or collection of cells can comprise a plurality of expression constructs of this invention which include biomarker expression control sequences of a plurality of different biological responses. In this case, it may be useful to use reporters for each expression control sequence that can be distinguished from each other, e.g., by emitting a different color or having a different bar code.","Optionally, an expression construct can contain both a first reporter sequence and a second reporter sequence. The first reporter sequence and a second reporter sequence are preferred to be different. For example, the first reporter sequence may encode the same reporter protein (e.g., luciferase or GFP), and the second reporter sequence may be a unique nucleotide barcode. In this way, transcription can yield a hybrid transcript of a reporter protein coding region and a unique barcode sequence. Such a construct could be used either in a receptacle-by-receptacle approach for reading out the signal emitted by the reporter protein (e.g., luminescence) and\/or in a pooled approach by reading out the barcodes.","By using the unique, molecular barcode for each member of the collection, a large collection can be assayed in a single receptacle (such as a vial or a well in a plate) rather than in thousands of individual receptacles. This approach is more efficient and economic as it can reduce costs at all levels: reagents, plasticware, and labor.","3.3 Vectors and Chromosomes","The recombinant nucleic acid molecule can be further comprised within a vector that can be used to either infect or transiently or stably transfect cells and that may be capable of replicating inside a cell. Expression constructs can be made part of vectors, such as plasmids, cosmids, viral genomes, bacterial artificial chromosomes and the like. They also can be integrated stably into a chromosome of a host cell.","Any suitable vector can be used. There are many known in the art. Examples of vectors that can be used include, for example, plasmids or modified viruses. The vector is typically compatible with a given host cell into which the vector is introduced to facilitate replication of the vector and expression of the encoded reporter. Examples of specific vectors that may be useful in the practice of the present invention include, but are not limited to, bacteriophages, for example, lambda derivatives, or plasmids, for example, pBR322 derivatives or pUC plasmid derivatives; phage DNAs, e.g., the numerous derivatives of phage 1, e.g., NM989, and other phage DNA, e.g., M13 and filamentous single stranded phage DNA; yeast vectors such as the 2 mu plasmid or derivatives thereof; vectors useful in eukaryotic cells, for example, vectors useful in insect cells, such as baculovirus vectors, vectors useful in mammalian cells such as retroviral vectors, adenoviral vectors, adenovirus viral vectors, adeno-associated viral vectors, SV40 viral vectors, herpes simplex viral vectors and vaccinia viral vectors; vectors derived from combinations of plasmids and phage DNAs, plasmids that have been modified to employ phage DNA or other expression control sequences; and the like.","4. Cells Containing Expression Constructs","In another aspect this invention provides recombinant cells comprising the expression constructs of this invention. Two different embodiments are contemplated in particular.","In a first embodiment each cell or group of cells comprises a different member of the expression library. Such a library of cells is particularly useful with the arrays of this invention. Typically, the library is indexed. For example, each different cell harboring a different expression vector can be maintained in a separate container that indicates the identity of the genomic segment within. The index also can indicate the particular gene or genes that is\/are under the transcriptional regulatory control of the sequences naturally in the genome.","In a second embodiment, a culture of cells is transfected with a library of expression constructs so that all of the members of the library exist in at least one cell and each cell has at least one member of the expression library. In this embodiment, a cell can comprise more than one different expression constructs (e.g., having transcription regulatory sequences of different genes). The second embodiment is particularly useful with libraries in which the reporter sequences are unique sequences that can be detected independently.","As used herein the term cells and grammatical equivalents herein in meant any cell, preferably any prokaryotic or eukaryotic cell.","Suitable prokaryotic cells include, but are not limited to, bacteria such as , various species, and the extremophile bacteria such as thermopiles, etc.","Suitable eukaryotic cells include, but are not limited to, fungi such as yeast and filamentous fingi, including species of , and ; plant cells including those of corn, sorghum, tobacco, canola, soybean, cotton, tomato, potato, alfalfa, sunflower, etc.; and animal cells, including fish, birds and mammals. Suitable fish cells include, but are not limited to, those from species of salmon, trout, tulapia, tuna, carp, flounder, halibut, swordfish, cod and zebrafish. Suitable bird cells include, but are not limited to, those of chickens, ducks, quail, pheasants and turkeys, and other jungle foul or game birds. Suitable mammalian cells include, but are not limited to, cells from horses, cows, buffalo, deer, sheep, rabbits, rodents such as mice, rats, hamsters and guinea pigs, goats, pigs, primates, marine mammals including dolphins and whales, as well as cell lines, such as human cell lines of any tissue or stem cell type, and stem cells, including pluripotent and non-pluripotent, and non-human zygotes.","Useful cell types include primary and transformed mammalian cell lines. Suitable cells also include those cell types implicated in a wide variety of disease conditions, even while in a non-diseased state. Accordingly, suitable cell types include, but are not limited to, tumor cells of all types (e.g. melanoma, myeloid leukemia, carcinomas of the lung, breast, ovaries, colon, kidney, prostate, pancreas and testes), cardiomyocytes, dendritic cells, endothelial cells, epithelial cells, lymphocytes (T-cell and B cell), mast cells, eosinophils, vascular intimal cells, macrophages, natural killer cells, erythrocytes, hepatocytes, leukocytes including mononuclear leukocytes, stem cells such as haemopoetic, neural, skin, lung, kidney, liver and myocyte stem cells (for use in screening for differentiation and de-differentiation factors), osteoclasts, chondrocytes and other connective tissue cells, keratinocytes, melanocytes, liver cells, kidney cells, and adipocytes. In some embodiments, the cells used with the methods described herein are primary disease state cells, such as primary tumor cells. Suitable cells also include known research cell lines, including, but not limited to, Jurkat T cells, NIH3T3 cells, CHO, COS, HT1080 human fibrosarcoma cells, HepG2 hepatocarcinoma cells, HeLa cells, etc. See the ATCC cell line catalog, hereby expressly incorporated by reference.","In some embodiments the cells used in the present invention are taken from an individual. In some embodiments the individual is a mammal, and in other embodiments the individual is human.","Exogenous DNA may be introduced to cells by lipofection, electroporation, or infection. Libraries in such cells may be maintained in growing cultures in appropriate growth media or as frozen cultures supplemented with Dimethyl Sulfoxide and stored in liquid nitrogen.","4.1. Transient Expression","In certain embodiments this invention provides transiently transformed cells. In this embodiment, vector DNA is delivered to living cells using a transfection reagent such as a lipofection compound. The cells express the content of this vector construct but the cell does not incorporate the vector sequence in its genome. A typical transient transfection experiment involves seeding cells in a plate or dish, then 24 hours later transfecting the vector DNA, then 12-24 hrs later applying a stimulus of interest, and then recording the reporter signal.","4.2. Stably Transformed Cells","In other embodiments, this invention provides stably transformed cells. In this embodiment, the vector DNA contains a gene for a mammalian selectable marker such as hygromycin. The vector is delivered to living cells using a transfection reagent such as a lipofection compound or in the form of a virus, and the cells are grown in the presence of the selecting antibiotic. Over the course of 5-20 or more cell passages, only cells that have integrated the vector DNA into their genome are selected for.","5. Devices and Kits","This invention provides kits and devices. The kits can include all reagents necessary to transiently transfect, infect, or transduce cells and perform assays. For example, the kits can include one or more containers comprising one or more expression constructs of this invention. They also can contain a container comprising a substrate for a reporter molecule made by the expression construct. The kit also can contain containers comprising appropriate solutions for carrying out the assay, such as appropriate buffers. The kit also can contain instructions on how to carry out the assay. The kit can also contain a container comprising cells to be transfected by the expression construct. In certain embodiments, the cells will be stably transformed to contain the expression construct. In certain embodiments, the kit contains microtiter plates, such as those having wells in multiples of 96, which are typically used in research.","6. Methods","6.1 Test Conditions","Certain methods of this invention involve exposing cells to test conditions. A test condition can be any condition different than a control condition to which a cell is exposed. Test conditions include, for example, contacting the cell with a test composition, exposing the cell to a test environmental condition, over-expressing or knocking down expression of a particular gene of interest in the cell, or combinations of these.","Test compositions include individual compounds and mixtures of compounds, e.g., a library of compounds. A compound can be any element or molecule, for example, small organic molecules and biopolymers. Drug candidates useful as test compositions in this invention include small organic molecules and biological molecules, e.g., biologics. Organic molecules used as pharmaceuticals generally are small organic molecules typically having a size up to about 5000 Da, up to about 2000 Da, or up to about 1000 Da. Certain hormones are small organic molecules. Organic biopolymers also are used as test compositions. These include, for example, polypeptides (e.g., peptides and proteins), polynucleotides (e.g., oligonucleotides or nucleic acids (e.g., inhibitory polynucleotides, such as ribozymes, antisense molecules or interfering RNA molecules), carbohydrates, lipids and molecules that combine these, for example glycoproteins, glycolipids and lipoproteins. Certain hormones are biopolymers. Antibodies find increasing use as biological pharmaceuticals. U.S. publication 2009-0035216 provides a list of antibody drugs. This list includes, for example herceptin, bevacizumab, avastin, erbitux and synagis (cell adhesion molecules).","6.2 Assays","This invention provides assays for determining whether and to what extent a test condition modulates a biological response. In certain embodiments, the methods involve exposing a recombinant cell of this invention to a test condition and measuring the amount of activity of a biological response biomarker promoter of this invention. In certain embodiments, the cells tested do not exhibit the biological response, and the methods involve exposing the cells to a test condition and determining whether the response is invoked. In other embodiments, the cells are exhibiting the biological response, e.g., by exposing the cells to conditions that elicit the response and then exposing the cells to the test condition. The expression of the reporter indicates the activity of the biomarker expression control sequence. The activity of the biomarker, in turn, indicates the activity of the biological response for which it is a biomarker.","Measuring a response includes quantitative and qualitative determinations. Qualitative measurement includes measuring a response or lack of response, regardless of intensity. A quantitative response generally involves measuring the intensity of a response.","The assays rely in part on the correlation between the activity of biomarker promoters and the activity of biological responses. This correlation can be positive (e.g., stimulation of biomarker activity correlates with activation of the response, or inhibition of the biomarker correlates with inhibition of the response) or the correlation can be negative (e.g., stimulating activity of the biomarker correlates with inhibition of the response, or inhibition of biomarker activity correlates with activation of the biological response). Thus, biomarker activity (its induction or inhibition of induction) functions as a proxy for the activity of the biological response. This, in turn, allows one to determine whether a test condition modulates (e.g., activates or inhibits) a biological response, based on the ability of the test condition to modulate activity of the biomarker promoter and the correlation of activity of the biomarker promoter with the activity of the biological response. A test condition that modulates or alters activity of a biomarker transcription regulatory sequence, by extension, modulates or alters activity of the biological response for which the transcription regulatory sequence is a biomarker.","This invention contemplates a number of different types of assays. In one assay, a cell comprising an expression construct of this invention is exposed to a test condition and the activity of the biomarker expression control sequence is determined based on expression of the reporter sequence. In another assay, a biomarker expression control sequence for a single biological response is exposed to a plurality of different test conditions, e.g., a plurality of different test compositions, and the effect of each test condition on the activity of the biomarker is determined. This can be, for example, a high throughput assay in which tens, hundreds or thousands of conditions are tested. In another embodiment, a single test condition is tested for the ability to modulate activity of one or more biomarkers for a plurality of different biological responses. For example, one can test biomarkers for at least 2, at least 5, at least 10 or at least 13 different biological responses.","A container (e.g., a multiwell plate array) containing cells harboring an expression construct of this invention is useful for high-throughput screening of promoter activity. A cell comprising an expression construct that comprises a biological response biomarker promoter operably linked with a reporter gene is exposed to a test condition under conditions chosen by the operator. Cells in which the promoters are \u201cturned on\u201d will express the reporter sequences under their transcriptional control. The investigator then checks each well of the device to measure the amount of reporter transcribed. Generally, this involves measuring the signal produced by a reporter protein encoded by the reporter sequence. For example, if the reporter protein is a light emitting reporter, then the amount of light produced in each cell is measured. In the case of a fluorescent protein, light can be directed to each well to induce fluorescence. The amount of signal measured is a function of the expression of the reporter sequence which, in turn, is a function of the activity of the transcriptional regulatory sequences. In certain methods, the assay is multiplexed. A number of compounds, e.g., a library of compounds can be tested. This can involve testing each compound against a recombinant cell of this invention.","The choice of a proper detection system for a particular application is well within the abilities of one of skill in the art. Exemplary detection means include, but are not limited to, detection by unaided eye, light microscopy using the eye or an optical sensor as the detector, confocal microscopy, laser scanning confocal microscopy, imaging using quantum dot color, fluorescence spectrum or other quantum dot property and wide-field imaging with a 2D CCD camera. In an exemplary embodiment, the device is a fluorescent plate reader. For example, the assay can be performed in a multiwell plate, e.g., a plate with multiples of 96 wells (e.g., 96-well plate, 384-well plate, 1536-well plate). In assays in which the reporter is luminescent or fluorescent molecule, the reporter may have to be induced to produce light. For example, in assays involving luciferase, a substrate for the enzyme is provided. In the case of fluorescent proteins, light of an exciting wavelength is provided. Commercially available microplate luminometers can be used to detect the signal. These are available from, for example, Tecan, Molecular Devices and Berthold.","Assays can involve creation of a standard curve against which measurements are compared to quantify the amount of expression.","It also can be useful to identify differences in transcription regulatory sequence activity in two cell types. For example gene expression differs when cells transform from normal to cancerous. Promoters that are overactive in cancer cells may be targets of pharmacological intervention. The arrays of this invention are useful to identify such transcription regulatory sequences. Accordingly, the investigator provides two sets of arrays comprising expression constructs in the wells. Once cell type is used for transformation in a first device and a second cell type, for transformation in a second device. The expression of reporter sequences between the two devices is compared to identify those expressed differently in the two cell types.","Using expression constructs in which the transcription regulatory sequences are operably linked to unique reporter sequences opens the possibility of performing tests without the use of multiwell plates. In such situations a single culture of cells contains the entire expression library distributed among the cells. The culture can be incubated under conditions chosen by the investigator. Then the expression products are isolated. Reporters that emit different colors can be used. As described, if the reporter is a barcode, because each expression vector has a unique nucleotide sequence tag or barcode associated with its partner nucleic acid segment, the amount of each of the reporter sequences can be measured by measuring the amount of transcript comprising each unique sequence. For example, the molecules can be detected on a DNA array that contains probes complementary to the unique sequences. The amount of hybridization to each probe indicates the amount of the reporter sequence expressed, which, in turn, reflects the activity of the transcription regulatory sequences.","1. Biological Responses","1.1 Biological Response to Hypoxia","Transient transfection assays were conducted in HT1080 human fibrosarcoma cells (ATCC, Manassas, Va.) in 96-well white plates. After 5,000 cells per well were seeded in culture medium for 24 hr, 50 ng of plasmid DNA per well was transfected with Fugene-6 transfection reagent (Roche Diagnostics, Indianapolis, Ind.) according to Fugene standard protocols. After 16 hrs, the transfection medium was removed and fresh culture medium was added into each well, and 3 replicates of each construct were moved into a chamber containing 1% oxygen, 5% CO, and 94% nitrogen. Cells were kept in the low oxygen condition for 24 hrs. In addition, 3 replicates of each construct were treated with 100 uM DFO for 24 hrs. Furthermore, 3 replicates of each construct remained in normal oxygen conditions and were not treated with DFO to serve as untreated control replicates. After induction, 100 uL of Steady-Glo (Promega) was added into each well, incubated at room temperature for 30 min, and then read in a standard plate luminometer (Molecular Devices, Sunnyvale, Calif.). Each treatment was assayed in triplicate, and the average of the 3 replicates without hypoxia induction and the average of the 3 replicates with hypoxia induction were recorded.","Transcription regulatory sequences, the activity of which is highly correlated with the hypoxia response in this assay, and that can be used as biomarkers for the hypoxia response, are identified in Table 1, section 1 and section 1a.","1.2. Biological Response to Estrogen (\u03b2-estradiol)","Transient transfection assays were conducted in HT1080 human fibrosarcoma cells (ATCC, Manassas, Va.) in 96-well white plates. First, 10,000 cells per well were seeded in charcoal stripped culture medium for 24 hr. Next, 50 ng of reporter plasmid DNA and 10 ng of an ER cDNA expression plasmid (Origene) per well were transfected with Fugene-6 transfection reagent (Roche Diagnostics, Indianapolis, Ind.) according to Fugene standard protocols. After 16 hrs, the transfection medium was removed and fresh culture medium with or without the addition of an inducing compound was added to each well. For the no treatment control, only fresh media was added to the wells. \u03b2-estradiol (Sigma) was added to the media of the treated samples at a final concentration of 10 nM. Cells were incubated for 24 hrs after induction. After 24 hrs, 100 uL of Steady-Glo (Promega) was added into each well, incubated at room temperature for 30 min, and then read in a standard plate luminometer (Molecular Devices, Sunnyvale, Calif.). Each treatment for each reporter construct was assayed in triplicate, and the average of the 3 replicates without treatment and the average of the 3 replicates with treatment were recorded.","Transcription regulatory sequences, the activity of which is highly correlated with the estrogen response in this assay, and that can be used as biomarkers for the estrogen response, are identified in Table 1, section 2 and section 2a.","1.3. Biological Response to Artificial Androgen (Methyltrienolone)","Transient transfection assays were conducted in HT1080 human fibrosarcoma cells (ATCC, Manassas, Va.) in 96-well white plates. First, 10,000 cells per well were seeded in charcoal stripped culture medium for 24 hr. Next, 50 ng of reporter plasmid DNA and 10 ng of an AR cDNA expression plasmid (Origene) per well were transfected with Fugene-6 transfection reagent (Roche Diagnostics, Indianapolis, Ind.) according to Fugene standard protocols. After 16 hrs, the transfection medium was removed and fresh culture medium with or without the addition of an inducing compound was added to each well. For the no treatment control, only fresh media was added to the wells. Methyltrienolone (R1881) was added to the media of the treated samples at a final concentration of 10 nM. Cells were incubated for 24 hrs after induction. After 24 hrs, 100 uL of Steady-Glo (Promega) was added into each well, incubated at room temperature for 30 min, and then read in a standard plate luminometer (Molecular Devices, Sunnyvale, Calif.). Each treatment for each reporter construct was assayed in triplicate, and the average of the 3 replicates without treatment and the average of the 3 replicates with treatment were recorded.","Transcription regulatory sequences, the activity of which is highly correlated with the androgen response in this assay, and that can be used as biomarkers for the androgen response, are identified in Table 1, section 3 and section 3a.","1.4 Biological Response Mediated by p53","Transient transfection assays were conducted in HT1080 human fibrosarcoma cells (ATCC, Manassas, Va.) in 96-well white plates. First, 5,000 cells per well were seeded in standard culture medium for 24 hr. Next, 50 ng of reporter plasmid DNA per well was transfected with Fugene-6 transfection reagent (Roche Diagnostics, Indianapolis, Ind.) according to Fugene standard protocols. After 16 hrs, the transfection medium was removed and fresh culture medium with or without the addition of an inducing compound was added to each well. For the no treatment control, only fresh media was added to the wells. The p53 protein was induced by two separate treatments: nutlin (Sigma) was added to the media at a final concentration of 10 uM, and doxorubicin (Calbiochem) was added to the media at a final concentration of 200 ng\/ml. Cells were incubated for 24 hrs after induction. After 24 hrs, 100 uL of Steady-Glo (Promega) was added into each well, incubated at room temperature for 30 min, and then read in a standard plate luminometer (Molecular Devices, Sunnyvale, Calif.). Each treatment for each reporter construct was assayed in triplicate, and the average of the 3 replicates without treatment and the average of the 3 replicates with treatments were recorded.","Transcription regulatory sequences, the activity of which is highly correlated with the p53 activation response in this assay, and that can be used as biomarkers for the p53 activation response, are identified in Table 1, section 4 and section 4a.","1.5 Biological Response to Inhibitors or Activators of Cholesterol Biosynthesis","Transient transfection assays were conducted in HepG2 hepatocarcinoma cells (ATCC, Manassas, Va.) in 96-well white plates. First, 10,000 cells per well were seeded in delipidated culture medium for 24 hr. Next, 100 ng of reporter plasmid DNA per well was transfected with Fugene-6 transfection reagent (Roche Diagnostics, Indianapolis, Ind.) according to Fugene standard protocols. After 16 hrs, the transfection medium was removed and fresh culture medium with or without the addition of an inducing compound was added to each well. For the no treatment control, only fresh media was added to the wells. Inductions were done by three separate treatments: lovastatin (Sigma) (HMG-CoA reductase inhibitor) was added to the media at a final concentration of 1 uM, synthechol (Sigma) (activator of cholesterol utilization pathway) was added to the media at a final concentration of 5 ug\/ml, and U1866A (Sigma) (inhibitor of intracellular cholesterol transport) was added to the media at a final concentration of 1 uM. Cells were incubated for 24 hrs after induction. After 24 hrs, 100 uL of Steady-Glo (Promega) was added into each well, incubated at room temperature for 30 min, and then read in a standard plate luminometer (Molecular Devices, Sunnyvale, Calif.). Each treatment for each reporter construct was assayed in triplicate, and the average of the 3 replicates without treatment and the average of the 3 replicates with treatments were recorded.","Transcription regulatory sequences, the activity of which is highly correlated with the cholesterol response in this assay, and that can be used as biomarkers for the cholesterol response, are identified in Table 1, section 5 and section 5a.","1.6 Biological Response to Interferons","Transient transfection assays were conducted in Hela cells (ATCC, Manassas, Va.) in 96-well white plates. First, 10,000 cells per well were seeded in a starvation medium (phenol-free Opti-MEM) for 24 hr. Next, 100 ng of reporter plasmid DNA per well was transfected with Fugene-6 transfection reagent (Roche Diagnostics, Indianapolis, Ind.) according to Fugene standard protocols. After 16 hrs, the transfection medium was removed and fresh culture medium with or without the addition of an inducing compound was added to each well. For the no treatment control, only fresh media was added to the wells. Inductions were done by two separate treatments: interferon alpha (Calbiochem) was added to the media at a final concentration of 500 U\/ml, and interferon gamma (Sigma) was added to the media at a final concentration of 100 ng\/ml. Cells were incubated for 8 hrs after induction. After 8 hrs, 100 uL of Steady-Glo (Promega) was added into each well, incubated at room temperature for 30 min, and then read in a standard plate luminometer (Molecular Devices, Sunnyvale, Calif.). Each treatment for each reporter construct was assayed in triplicate, and the average of the 3 replicates without treatment and the average of the 3 replicates with treatments were recorded.","Transcription regulatory sequences, the activity of which is highly correlated with the interferon response in this assay, and that can be used as biomarkers for the interferon response, are identified in Table 1, section 6 and section 6a.","1.7 Biological Response to CREB Activation","Transient transfection assays were conducted in Hela cells (ATCC, Manassas, Va.) in 384-well white plates. First, 4,000 cells per well were seeded in standard culture medium for 24 hr. Next, 50 ng of reporter plasmid DNA per well was transfected with Fugene-6 transfection reagent (Roche Diagnostics, Indianapolis, Ind.) according to Fugene standard protocols. After 16 hrs, the transfection medium was removed and fresh culture medium with or without the addition of an inducing compound was added to each well. For the no treatment control, only fresh media was added to the wells. Inductions were done by two separate treatments: forskolin (Sigma) (protein kinase A activator) was added to the media at a final concentration of 20 uM, and phorbol 12-myristate 13-acetate (Promega) (protein kinase C activator) was added to the media at a final concentration of 100 nM. Cells were incubated for 4 hrs after induction. After 4 hrs, 100 uL of Steady-Glo (Promega) was added into each well, incubated at room temperature for 30 min, and then read in a standard plate luminometer (Molecular Devices, Sunnyvale, Calif.). Each treatment for each reporter construct was assayed in triplicate, and the average of the 3 replicates without treatment and the average of the 3 replicates with treatments were recorded.","Transcription regulatory sequences, the activity of which is highly correlated with the CREB activation response in this assay, and that can be used as biomarkers for the CREB activation response, are identified in Table 1, section 7 and section 7a.","1.8 Biological Response to Glucocorticoid Receptor Ligands","Transient transfection assays were conducted in HT1080 cells (ATCC, Manassas, Va.) in 96-well white plates. First, 5,000 cells per well were seeded in standard culture medium for 24 hr. Next, 50 ng of reporter plasmid DNA was transfected with Fugene-6 transfection reagent (Roche Diagnostics, Indianapolis, Ind.) according to Fugene standard protocols. After 16 hrs, the transfection medium was removed and fresh culture medium with or without the addition of an inducing compound was added to each well. For the no treatment control, only fresh media was added to the wells. Inductions were done by three separate treatments: dexamethasone (Sigma) was added to the media of the treated samples at a final concentration of 100 nM, prednisone (Sigma) was added to the media of the treated samples at a final concentration of 1 uM, and cortisone (Sigma) was added to the media of the treated samples at a final concentration of 1 uM. Cells were incubated for 4 hrs after induction. After 4 hrs, 100 uL of Steady-Glo (Promega) was added into each well, incubated at room temperature for 30 min, and then read in a standard plate luminometer (Molecular Devices, Sunnyvale, Calif.). Each treatment for each reporter construct was assayed in triplicate, and the average of the 3 replicates without treatment and the average of the 3 replicates with treatment were recorded.","Transcription regulatory sequences, the activity of which is highly correlated with the glucocorticoid-mediated response in this assay, and that can be used as biomarkers for the glucocorticoid-mediated response, are identified in Table 1, section 8 and section 8a.","1.9 Biological Response to PPAR Ligands","Transient transfection assays were conducted in HepG2 cells (ATCC, Manassas, Va.) in 96-well white plates. First, 10,000 cells per well were seeded in charcoal stripped culture medium for 24 hr. Next, 100 ng of reporter plasmid DNA and 20 ng of a PPAR cDNA expression plasmid (Origene) per well were transfected with Fugene-6 transfection reagent (Roche Diagnostics, Indianapolis, Ind.) according to Fugene standard protocols. After 16 hrs, the transfection medium was removed and fresh culture medium with or without the addition of an inducing compound was added to each well. For the no treatment control, only fresh media was added to the wells. Inductions were done by three separate treatments: WY14643 (Cayman) was added to the media of the treated samples at a final concentration of 75 uM to induce PPAR alpha, ciglitazone (Biomol) was added to the media of the treated samples at a final concentration of 10 uM to induce PPAR gamma, and GW501516 (Alexis) was added to the media of the treated samples at a final concentration of 100 nM to induce PPAR delta. Cells were incubated for 24 hrs after induction. After 24 hrs, 100 uL of Steady-Glo (Promega) was added into each well, incubated at room temperature for 30 min, and then read in a standard plate luminometer (Molecular Devices, Sunnyvale, Calif.). Each treatment for each reporter construct was assayed in triplicate, and the average of the 3 replicates without treatment and the average of the 3 replicates with treatment were recorded.","Transcription regulatory sequences, the activity of which is highly correlated with the PPAR-mediated response in this assay, and that can be used as biomarkers for the PPAR-mediated response, are identified in Table 1, section 9 and section 9a.","1.10 Biological Response to RAR Ligands","Transient transfection assays were conducted in HepG2 cells (ATCC, Manassas, Va.) in 96-well white plates. First, 10,000 cells per well were seeded in charcoal stripped culture medium for 24 hr. Next, 100 ng of reporter plasmid DNA and 20 ng of a RAR cDNA expression plasmid (Origene) per well were transfected with Fugene-6 transfection reagent (Roche Diagnostics, Indianapolis, Ind.) according to Fugene standard protocols. After 16 hrs, the transfection medium was removed and fresh culture medium with or without the addition of an inducing compound was added to each well. For the no treatment control, only fresh media was added to the wells. Inductions were done by two separate treatments: AM-580 (Biomol) was added to the media of the treated samples at a final concentration of 100 nM for RAR alpha and adapalene (Biomol) was added to the media of the treated samples at a final concentration of 100 nM for RAR beta. Cells were incubated for 24 hrs after induction. After 24 hrs, 100 uL of Steady-Glo (Promega) was added into each well, incubated at room temperature for 30 min, and then read in a standard plate luminometer (Molecular Devices, Sunnyvale, Calif.). Each treatment for each reporter construct was assayed in triplicate, and the average of the 3 replicates without treatment and the average of the 3 replicates with treatment were recorded.","Transcription regulatory sequences, the activity of which is highly correlated with the RAR-mediated response in this assay, and that can be used as biomarkers for the RAR-mediated response, are identified in Table 1, section 10 and section 10a.","1.11 Biological Response to TNFa and NFkB Activation","Transient transfection assays were conducted in HT1080 cells (ATCC, Manassas, Va.) in 96-well white plates. First, 5,000 cells per well were seeded in standard culture medium for 24 hr. Next, 50 ng of reporter plasmid DNA was transfected with Fugene-6 transfection reagent (Roche Diagnostics, Indianapolis, Ind.) according to Fugene standard protocols. After 16 hrs, the transfection medium was removed and fresh culture medium with or without the addition of an inducing compound was added to each well. For the no treatment control, only fresh media was added to the wells. TNFa (Abcam) was added to the media of the treated samples at a final concentration of 20 ng\/ml. Cells were incubated for 8 hrs after induction. After 8 hrs, 100 uL of Steady-Glo (Promega) was added into each well, incubated at room temperature for 30 min, and then read in a standard plate luminometer (Molecular Devices, Sunnyvale, Calif.). Each treatment for each reporter construct was assayed in triplicate, and the average of the 3 replicates without treatment and the average of the 3 replicates with treatment were recorded.","Transcription regulatory sequences, the activity of which is highly correlated with the TNF\u03b1-mediated response in this assay, and that can be used as biomarkers for the TNF\u03b1-mediated response, are identified in Table 1, section 11 and section 11a.","1.12 Biological Response to Heat Shock","Transient transfection assays were conducted in HT1080 cells (ATCC, Manassas, Va.) in 96-well white plates. First, 5,000 cells per well were seeded in standard culture medium for 24 hr. Next, 50 ng of reporter plasmid DNA was transfected with Fugene-6 transfection reagent (Roche Diagnostics, Indianapolis, Ind.) according to Fugene standard protocols. After 16 hrs, three replicates of each transfected construct was moved to 43 degrees C., and three replicates were kept at 37 degrees C. Cells were incubated for 8 hrs at their respective temperatures. After 8 hrs, 100 uL of Steady-Glo (Promega) was added into each well, incubated at room temperature for 30 min, and then read in a standard plate luminometer (Molecular Devices, Sunnyvale, Calif.). Each treatment for each reporter construct was assayed in triplicate, and the average of the 3 replicates without treatment and the average of the 3 replicates with treatment were recorded.","Transcription regulatory sequences, the activity of which is highly correlated with the heat shock response in this assay, and that can be used as biomarkers for the heat shock response, are identified in Table 1, section 12 and section 12a.","1.13 Biological Response to Serum","Transient transfection assays were conducted in HT1080 cells (ATCC, Manassas, Va.) in 96-well white plates. First, 7,500 cells per well were seeded in a starvation medium (phenol-free Opti-MEM) for 24 hr. Next, 50 ng of reporter plasmid DNA per well was transfected with Fugene-6 transfection reagent (Roche Diagnostics, Indianapolis, Ind.) according to Fugene standard protocols. After 16 hrs, the transfection medium was removed and fresh culture medium with or without the addition of fetal bovine serum was added to each well. For the no treatment control, only fresh media was added to the wells. Inductions were done by adding fetal bovine serum to the media at a final concentration of 20%. Cells were incubated for 8 hrs after induction. After 8 hrs, 100 uL of Steady-Glo (Promega) was added into each well, incubated at room temperature for 30 min, and then read in a standard plate luminometer (Molecular Devices, Sunnyvale, Calif.). Each treatment for each reporter construct was assayed in triplicate, and the average of the 3 replicates without treatment and the average of the 3 replicates with treatments were recorded.","Transcription regulatory sequences, the activity of which is highly correlated with the serum response in this assay, and that can be used as biomarkers for the serum response, are identified in Table 1, section 13 and section 13a.","References","U.S. 2007-0161031, Jul. 12, 2007, TRINKLEIN, N. D. et al.","U.S. 2009-0018031, Jan. 15, 2009, TRINKLEIN, N. D. et al.","COOPER, S. J. et al., Genome Res., 2006, 16:1-10","GUILLEM, K. et al., Cell, 1997 89:9-12","TRINKLEIN, N. D. et al., Genome Res., 2003, 13:308-312","TRINKLEIN, N. D. et al., Genome Res., 2004, 14:62-66","While preferred embodiments of the present invention have been shown and described herein, it will be obvious to those skilled in the art that such embodiments are provided by way of example only. Numerous variations, changes, and substitutions will now occur to those skilled in the art without departing from the invention. It should be understood that various alternatives to the embodiments of the invention described herein may be employed in practicing the invention. It is intended that the following claims define the scope of the invention and that methods and structures within the scope of these claims and their equivalents be covered thereby."],"heading":["BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","INCORPORATION BY REFERENCE","DETAILED DESCRIPTION OF THE INVENTION","EXAMPLES"],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":["The novel features of the invention are set forth with particularity in the appended claims. A better understanding of the features and advantages of the present invention will be obtained by reference to the following detailed description that sets forth illustrative embodiments, in which the principles of the invention are utilized, and the accompanying drawings of which:",{"@attributes":{"id":"p-0016","num":"0015"},"figref":"FIG. 1"}]},"DETDESC":[{},{}]}
