---
title: Oxygen-resistant hydrogenases and methods for designing and making same
abstract: The invention provides oxygen-resistant iron-hydrogenases ([Fe]-hydrogenases) for use in the production of H. Methods used in the design and engineering of the oxygen-resistant [Fe]-hydrogenases are disclosed, as are the methods of transforming and culturing appropriate host cells with the oxygen-resistant [Fe]-hydrogenases. Finally, the invention provides methods for utilizing the transformed, oxygen insensitive, host cells in the bulk production of Hin a light catalyzed reaction having water as the reactant.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08663958&OS=08663958&RS=08663958
owner: Alliance for Sustainable Energy, LLC
number: 08663958
owner_city: Golden
owner_country: US
publication_date: 20090127
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATIONS","GOVERNMENT INTEREST","FIELD OF THE INVENTION","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","Definitions","EXAMPLES","Example 1","Computer Modeling of Hydrogenase H-Channel for Design of Oxygen-Resistant Hydrogenase Enzymes","Example 2",{"@attributes":{"id":"h-0020","level":"1"},"i":"C. reinhardtii ","sub":"2"},"Example 3","Green Alga, , Transformed with Oxygen-Resistant Hydrogenases are Effective in the Bulk Production of H"],"p":["This application is a division of U.S. application Ser. No. 10\/553,097, filed Oct. 13, 2005, now U.S. Pat. No. 7,501,270, issued Mar. 10, 2009, which is a national stage entry of International Application No. PCT\/US04\/11830, filed Apr. 16, 2004, which claims priority to U.S. Provisional Application No. 60\/464,081, filed Apr. 18, 2003. The contents of each application listed above are incorporated by reference in their entirety.","The United States Government has rights in this invention pursuant to contract DE-AC36-08GO28308 between the United States Department of Energy and the Alliance for Sustainable Energy, LLC, manager and operator of the National Renewable Energy Laboratory.","The present invention relates to hydrogen-production by microorganisms, for example hydrogen production by green algae. More specifically, the invention relates to methods for designing and engineering hydrogenase enzymes with improved oxygen resistance, and to the methods for transforming microorganisms to express these oxygen-resistant hydrogenase enzymes for use in the production of hydrogen in an oxygen containing environment.","Hydrogen (H) is becoming an attractive alternative energy source to fossil fuels due to its clean emissions and potential for cost effective production by microorganisms. As such, microorganisms that metabolize Hare being investigated for their potential use in H-production. A microorganism of particular interest for Hproduction is the green alga, , which is able to catalyze light-dependent, Hproduction utilizing water as a reductant. Ghiradi et al., (2000) Trends Biotech. 18(12):506-511; Melis et al., (2001) Plant Physiol. 127:740-748. The benefits of using an algal system for H-production include the use of renewable substrates (light and water) and its potential cost-effectiveness. Melis A, Int. J. Hyd. Energy 27:1217-1228. As such, there is a great deal of interest in optimizing H-production by green algae to maximize the potential benefit as an alternative energy source.",{"@attributes":{"id":"p-0006","num":"0005"},"i":"Chlamydomonas reinhardtii","sub":["2 ","2","2"],"sup":"\u2212"},"Although there are differences within the active sites between different families of hydrogenase, as well as between the subunit composition and localization between hydrogenase families, most, if not all studied hydrogenases have exhibited some degree of sensitivity to inhibition by CO and O. Adams M. W. W; Volbeda et al., (1990) Int. J. Hyd. Energy 27:1449-1461. Hydrogenase sensitivity to these inhibitors correlates to some degree to the type of prosthetic group that forms the active site, for example, [Fe]-hydrogenase is highly sensitive to O. As such, for example, the activity of [Fe]-hydrogenase in is very sensitive to Oduring H-photoproduction under photosynthetic conditions. Ghirardi et al., (1997) App. Biochem. Biotech. 63-65:141-151. Oxygen inhibition of [Fe]-hydrogenases is a major drawback in the use of green alga for Hproduction.","One approach to overcoming this Hproduction limitation is to stress the under photoheterotrophic, sulfur-deprived conditions that minimize O-photoproduction levels and result in sustained H-production. However, this approach does not result in optimal yields and requires the use of sulfur-deprived\/oxygen limited production techniques. Recently, CO and Oinhibition of hydrogenase activity in alga has been focused on the putative role of the H-channel. For example, it has been shown that the positioning of the Fe-atom in the enzyme's active site is directly at the active-site\/H-channel interphase, where it is easily accessed by either CO or Odiffusing through the channel. Lemon et al., (1999) Biochem. 38:12969-12973; Bennett et al., (2000) Biochem. 39:7455-7460. Further, a naturally occurring O-resistant [NiFe]-hydrogenase has been shown to have a narrower active site\/H-channel interphase than the naturally occurring hydrogenase counterpart. Volbeda et al. (2002), Supra.","Against this backdrop the present invention has been developed.","The present invention provides oxygen-resistant hydrogenases for use in the bulk production of Hin green algae cultures. In a preferred embodiment, homology modeling between known hydrogenases, e.g., CpI, and target hydrogenases, e.g., HydA1, was used to design and in silico engineer an oxygen-resistant [Fe]-hydrogenase having a reduced diameter H-channel. Constructed polynucleotides that encode oxygen-resistant [Fe]-hydrogenase enzymes are used to transform target host cells which were subsequently used in the photoproduction of H. In preferred embodiments, the target host cells are . The invention provides a solution to the problem of Hproduction by green algae when Ois present in the environment.","The present invention also provides host cells expressing oxygen-resistant [Fe]-hydrogenase. Host cells expressing the oxygen-resistant [Fe]-hydrogenase have significantly increased Hproduction, in the presence of O, as compared to similarly treated cells that do not express oxygen-resistant [Fe]-hydrogenase.","The present invention also provides polynucleotide molecules encoding HydA1V240W and other like oxygen-resistant hydrogenase polypeptides. The invention includes nucleic acid molecules that hybridize under high stringency conditions to the HydA1V240W polynucleotides (and other like oxygen-resistant hydrogenase polynucleotides) of the present invention. The invention also includes variants and derivatives of the oxygen-resistant [Fe]-hydrogeanse polypeptides, including fusion proteins that confer a desired function. The invention also provides vectors, plasmids, expression systems, host cells and the like, containing the oxygen-resistant [Fe]-hydrogenase of the invention.","These and various other features and advantages of the invention will be apparent from a reading of the following detailed description and a review of the appended claims.","The following definitions are provided to facilitate understanding of certain terms used herein and are not meant to limit the scope of the present disclosure.","\u201cAmino acid\u201d or \u201cresidues\u201d refers to any of the twenty naturally occurring amino acids as well as any modified amino acid sequences. Modifications may include natural processes such as posttranslational processing, or may include chemical modifications which are known in the art. Modifications include but are not limited to: phosphorylation, ubiquitination, acetylation, amidation, glycosylation, covalent attachment of flavin, ADP-ribosylation, cross-linking, iodination, methylation, and alike. Amino acid residue characterization can be found in numerous citations, for example Stryer, 1995, Biochemistry, throughout the text and 17-44.","\u201cExpression\u201d refers to transcription and translation occurring within a host cell. The level of expression of a DNA molecule in a host cell may be determined on the basis of either the amount or corresponding mRNA that is present within the cell or the amount of DNA molecule encoded protein produced by the host cell (Sambrook et al., 1989, Molecular Cloning: A Laboratory Manual, 18.1-18.88).","\u201cGenetically engineered\u201d refers to any recombinant DNA or RNA method used to create a host cell that expresses a target protein at elevated levels, at lowered levels, or in a mutated form. Typically, the host cell has been transfected, transformed, or transduced with a recombinant polynucleotide molecule, and thereby been altered so as to cause the cell to alter expression of the desired protein. Methods for genetically engineering host cells are well known in the art. (See Current Protocols in Molecular Biology, Ausubel et al., eds. (Wiley & Sons, New York, 1988, and quarterly updates)). Genetically engineering techniques include but are not limited to expression vectors, targeted homologous recombination and gene activation (see, for example U.S. Pat. No. 5,272,071 to Chappel) and trans activation by engineered transcription factors (See Segal et al., 1999, Proc Natl Acad Sci USA 96(6):2758-63).","\u201cHybridization\u201d refers to the pairing of complementary polynucleotides during an annealing process. The strength of hybridization between two polynucleotide molecules is impacted by the homology between the two molecules, stringency of the conditions involved, and melting temperatures of the formed hybrid and the G:C ratio within the polynucleotide. For purposes of the present invention stringency hybridization conditions refers to the temperature, ionic strength, solvents, etc, under which hybridization between polynucleotides occurs.","\u201cIdentity\u201d refers to a comparison between pairs of nucleic acid or amino acid molecules. Methods for determining sequence identity are known in the art. For example, computer programs have been developed to perform the comparison, such as the Gap program (Wisconsin Sequence Analysis Package, Version 8 for Unix, Genetics Computer Group, University Research Park, Madison Wis.), that uses the algorithm of Smith and Waterman (1981) Adv Appl Math 2:482-489.","\u201cIsolating\u201d refers to a process for separating a nucleic acid or polypeptides from at least one contaminant with which it is normally associated. In preferred embodiments, isolating refers to separating a nucleic acid or polypeptide from at least 50% of the contaminants with which it is normally associated, and more preferably from at least 75% of the contaminants with which it is normally associated.","The term \u201cnucleic acid\u201d refers to a linear sequence of nucleotides. The nucleotides are either a linear sequence of polyribonucleotides or polydeoxyribonucleotides, or a mixture of both. Examples of nucleic acid in the context of the present invention include\u2014single and double stranded DNA, single and double stranded RNA, and hybrid molecules that have mixtures of single and double stranded DNA and RNA. Further, the nucleic acids of the present invention may have one or more modified nucleotides.","The term \u201cPCR\u201d or \u201cpolymerase chain reaction\u201d refers to the process to amplify nucleic acids as described in U.S. Pat. Nos. 4,683,105 and 4,683,202, both owned by Roche Molecular.","\u201cHost cell\u201d refers to cells containing a target nucleic acid molecule, for example a heterologous nucleic acid molecule such as a plasmid or other low molecular weight nucleic acid, in which case the host cell is typically suitable for replicating the nucleic acid molecule of interest. Examples of suitable host cells useful in the present invention include, bacteria, algae, and yeast, Specific examples of such cells include, DH5\u03b1 cells, as well as various other bacterial cell sources, for example the strains: DH10b cells, XL1Blue cells, XL2Blue cells, Top10 cells, HB101 cells, and DH12S cells, yeast host cells from the genera including , and and green alga, for example ","\u201cHybridization\u201d refers to the pairing of complementary polynucleotides during an annealing period. The strength of hybridization between two polynucleotides molecules is impacted by the homology between the two molecules, stringent conditions involved, the melting temperature of the formed hybrid and the G:C ratio within the polynucleotides. High stringency conditions include, for example, 42\u00b0 C., 6\u00d7SSC, 0.1% SDS for 2 hours.","\u201cNucleic acid\u201d or \u201cNA\u201d refers to both a deoxyribonucleic acid and a ribonucleic acid. As used herein, \u201cnucleic acid sequence\u201d refers to the order or sequence of deoxyribonucleotides or ribonucleotides along a strand. They may be natural or artificial sequences, and in particular genomic DNA (gDNA), complementary DNA (cDNA), messenger RNA (mRNA), transfer RNA (tRNA), ribosomal RNA (rRNA), hybrid sequences or synthetic or semisynthetic sequences, oligonucleotides which are modified or otherwise. These nucleic acids may be of human, animal, plant, bacterial or viral origin and the like. They may be obtained by any technique known to persons skilled in the art, and in particular by the screening of libraries, by chemical synthesis or by mixed methods including the chemical or enzymatic modification of sequences obtained by the screening of libraries. They may be chemically modified, e.g. they may be pseudonucleic acids (PNA), oligonucleotides modified by various chemical bonds (for example phosphorothioate or methyl phosphonate), or alternatively oligonucleotides which are functionalized, e.g. which are coupled with one or more molecules having distinct characteristic properties. In the case of deoxyribonucleic acids, they may be single- or double-stranded, as well as short oligonucleotides or longer sequences. In particular, the nucleic acids advantageously consist of plasmids, vectors, episomes, expression cassettes and the like. These deoxyribonucleic acids may carry genes of therapeutic interest, sequences for regulating transcription or replication, anti-sense sequences which are modified or otherwise, regions for binding to other cellular components, and the like.","\u201cOxygen resistant\u201d refers to any measurable decrease in oxygen sensitivity in a hydrogenase as compared to a hydrogenase having a reference oxygen sensitivity, for example, as compared to a wild type hydrogenase from which an oxygen-resistant hydrogenase enzyme has been made.","\u201cOxygen sensitive\u201d refers to the wild type or reference oxygen sensitivity found in a native hydrogenase.","\u201cProtein,\u201d \u201cpeptide,\u201d and \u201cpolypeptide\u201d are used interchangeably to denote an amino acid polymer or a set of two or more interacting or bound amino acid polymers.","Green Algae and Iron Hydrogenase","Green algae, e.g., , cultured under anaerobic conditions synthesize an enzyme known as iron-hydrogenase ([Fe]-hydrogenase). As shown in , several representative [Fe]-hydrogenase enzymes are aligned showing general sequence identity for the family of proteins. Generally, overall sequence identity for [Fe]-hydrogenase family members is usually at least 45% and sequence identity for the H-channel between family members is at least 66%.","In general, [Fe]-hydrogenase enzymes characteristically possess a catalytic site consisting of a bimetallic center containing two Fe atoms (2Fe-center), bridged by cysteinyl sulfur to an electron relay [4Fe4S] center (4Fe-center). The iron atoms of the catalytic 2Fe-center are joined together by a combination of organic, sulfur, and carbon monoxide ligands. The chemistry of the [Fe]-hydrogenase catalytic core is reactive with respect to hydrogen, typically possessing very high hydrogen-production rates. However, this same catalytic core is also highly sensitive to inactivation by oxygen. As a protective measure against inactivation by oxygen or other like molecules, the catalytic core is typically buried deep within the protein, where access to the core is limited. As a result, interface of the hydrogenase catalytic site with surface surroundings is principally limited to a single channel, termed the H-channel, that directs diffusion of synthesized hydrogen from the enzyme interior to the external environment. The H-channel is also the primary access route of oxygen to the metallo-catalytic site within hydrogenase enzyme. Reverse diffusion of the oxygen from the surface of the enzyme into the H-channel and on to the active site, allows oxygen to bind to the 2Fe-center, inactivating the enzyme. Under normal physiologic conditions this represents a fairly normal inhibitory response for the hydrogenase enzyme, however, under the artificial conditions of expressing bulk amounts of H, this is a fairly major limitation.","The present invention provides for the modification of the H-channel to reduce oxygen diffusion from the external environment to the enzyme's catalytic core. The present invention provides modifications to the H-channel that act as oxygen filters, preventing or reducing the diffusion of oxygen to the catalytic site within the hydrogenase enzyme. These modifications are at the same time insufficient to limit the ability of Hto diffuse out of the enzyme through the H-channel. Several mechanisms for the reduction of oxygen diffusion to the active site within the hydrogenase enzyme are provided, including targeted replacement of residues that line the H-channel with bulkier residues, so as to shield the 2Fe-center and\/or reduce the diameter of the H-channel. In particular, the residues that line the H-channel are replaced with bulkier, hydrophobic residues, for example tryptophan or phenylalanine, so as to shield the 2Fe-center, as well as to reduce the size or volume of the catalytic site-H-channel interface. In addition, modifications to residues on the channel interior that approach and define the channel-solvent boundary (see portion 4 of ).","In a preferred embodiment of the invention, a process for designing and engineering oxygen-resistant iron-hydrogenases has been developed. The engineering scheme targets the structure and or environment of the H-channel within the target hydrogenase, which is altered to be more selective in allowing the outward diffusion of hydrogen while simultaneously filtering our surface oxygen. Note that size-limited diffusion has been successfully used to generate filters for commercial use in the separation of gases, including the separation of hydrogen from oxygen. Menoff T. M., (2000) Proc. Hydrogen Program Review.","The present invention provides host cells for the expression of nucleic acid molecules for encoding an oxygen-resistant iron-hydrogenase, for example, that expresses an oxygen-resistant HydA1 or cyanobacteria that also expresses an oxygen-resistant HydA1. Example oxygen-resistant hydrogenases designed and engineered by the method of the present invention include V240W, A78W, A244W, A248W, G86W, and L93W. These oxygen-resistant hydrogenase enzymes have the same primary structure as HydA1 with the exception that A at residue 78 is replaced with W. Note that other residues besides W, including synthetic and derivatized amino acids, are envisioned for substitution into the H-channel, as long as they limit Odiffusion through the channel and allow Hdiffusion out of the channel.","In addition, the invention provides for the bulk photoproduction of Husing the transformed host cells of the invention.","Identification of the Residues that Form the H-Channel of a [Fe]-Hydrogenase","The iron-hydrogenase family of enzymes is a group of enzymes expressed in algae for metabolism of hydrogen. Iron-hydrogenase family members have been shown to have three distinct motifs that contain highly conserved residues, including a series of identifiable cysteine residues. Vignais et al., (2001) FEMS Micro. Rev. 25:455-501. In particular, motif 1 has the amino acid sequence PMFTSCCPxW, motif 2 has an amino acid sequence MPCxxKxxExxR and motif 3 has an amino acid sequence of FxExMACxGxCV. These three motifs have been identified in all iron-hydrogenase family members to date. The cysteine residues have been shown to either ligate the catalytic [4Fe-4S] center, or bridge the [4Fe-4S] to the [2Fe-2S] center, and there presence within the primary structure of the enzyme is highly conserved. One of the most studied iron-hydrogenase enzymes is CpI, having its primary, secondary and tertiary structures determined. Peters et al., (1998) Science 282:1853-1858. In preferred embodiments, CpI or other like known iron-hydrogenase enzymes, can be used in the design and engineering of oxygen-resistant hydrogenases (see below and  for potential iron-hydrogenase enzymes).","To identify the H-channel within a target hydrogenase, i.e., a polypeptide containing motifs 1-3 above, the primary sequence of the target hydrogenase must be compared to the primary sequence of a known hydrogenase. Once the two sequences have been aligned a level of identity is determined (see ). Stothard P., (2000) BioTechniques 28(6) 1102 (hereby incorporated by reference in its entirety). For purposes of the present invention an overall identity of approximately 40%-45% or better should be found for the target hydrogenase. Further, an analysis of the target polypeptide's primary sequence is performed to predict the sequences that share homology with the H-channel forming regions of other known iron-hydrogenases (similar patterns of residues that have been shown previously to form hydrophobic cavities). Montet et al., (1997) Nat. Struc. Biol. 4:523-526. It should be noted that because the H-channel is a conserved domain within all hydrogenases, other non-iron hydrogenase sequences can be used to identify the target hydrogenase H-channel. There should be at least 40%-45% identity between the known and unknown sequence between the H-channel sequences of the know and unknown hydrogenases. Once the region within the target polypeptide for the H-channel has been located, the channel is modeled into a three-dimensional structure showing the orientation of residues in relation to the channel and active site. Guex et al., (1997) Electrophoresis 18:2714-2723. (see below) In some embodiments, the analysis is extended to identify the residues corresponding to the active site within the target hydrogenase. Note that the active site of the target or unknown hydrogenase should share at least 90% homology for motifs 1-3, and in preferred embodiments shown complete identity with motifs 1-3 (see above). The combination of primary and tertiary structures of the target hydrogenase are compared to evaluate the identification of candidate regions for the final verification of the hydrogen-channel.","Methods for Designing and Engineering Oxygen-Resistant Iron-Hydrogenases","As noted above, the present invention provides a model for generating a theoretical structure of a target H-channel within a target hydrogenase enzyme. In one embodiment, the theoretical structure is generated by homology modeling (see above) to the solved structure of other known [Fe]-hydrogenases, for example CpI. (see ). In some embodiments, the homology modeling is limited to the known hydrogenase active site and H-channel, and in other embodiments the homology modeling can be limited to the known hydrogenase H-channel sub-domains. A percent homology of the known hydrogenase (both identity and similarity) can be used to determined residue identity and similarity for the entire enzyme, the active site, the H-channel and the H-channel sub-domains (see overhead arrows in  and see discussion in previous section above). As such, the present invention provides a known hydrogenase based homology model that gives a reliable approximation of the target hydrogenase structure and H-channel environment. In a preferred embodiment, the known hydrogenase is CpI and the target hydrogenase is HydA1. Homology modeling can be performed using Swiss-model software as described in Guex et al. Guex et al (1997) Electrophoresis 18:2714-2723. Note, however, that other like programs can be used in this aspect, as is known in the art, e.g., Modeller program designed by Marti-Renom et al., (2000) Ann. Rev. Biophy, Biomol. Struct. 29:291-325; EsyPred3D designed by Lambert C. et al., (2002) Bioinformatics 18(9):1250-1256.","Typically, the homology modeling identifies the residues that project into the H-channel interior of the target hydrogenase. The channel environment is often composed of smaller hydrophobic residues, e.g., glycine, alanine, valine, but can contain phenylalanine and other like residues. For example, the H-channel of HydA1 contains mostly small hydrophobic residues with the exception of the larger phenylalanines at positions 252 and 355 (see , black dotted residues). A secondary structure is determined from the active site to the enzyme surface using the modeled structure above, and distances between side chain atoms of identified residues opposed to each other are determined. Guex et al., Supra An approximate average diameter of the channel over the distance from the catalytic site (Fe-atom to the H-cluster [2Fe-2S]-center) to the protein surface is determined (see , 1-4) (typically by using the distances between the side chain atoms of opposed residues within the channel). In silico mutagenesis is performed on the identified hydrogenase H-channel structure to identify possible residues that can be modified to reduce the H-channel diameter. Mutagenesis criteria preferably involve conservative mutation of specific residues, selection of the lowest energy rotomer and energy minimization of the resulting structure using GROMOS. van Gunsteren, W. F. et al., (1996) Biomolecular Simulation, The GROMOS96 Manual and User Guide. Vdf Hochschulverlag ETHZ. Once an energy minimized structure is obtained, the dimensions of the target in silico mutagenized hydrogenase channel is determined. In preferred embodiments, one or more locations along the H-channel is designed via conservative mutation to be smaller in diameter than a corresponding non-mutated H-channel, typically this reduction is to a channel size of between approximately 5.0 and 2.4 \u212b in diameter, and preferably between 3.5 and 2.5 \u212b, a diameter that either limits or eliminates the ability of oxygen to diffuse through the modified H-channel. Note that the H-channel is in constant flux, as such diameter measurements are averages and not meant to be held to a static standard. Note that in embodiments of the present invention, more than one residue can be in silico mutated to design an optimum oxygen-resistant hydrogenase.","In an alternative embodiment, design of oxygen-resistant hydrogenase enzymes is provided by determining what substitutions\/modifications of residues within the identified H-channel of a target [Fe]-hydrogenase can be performed to reduce the volume of the H-channel. Volume considerations include a reduction in the flow of gasses, i.e., O, through the channel in accordance with Stokes Einstein Equation and Fick's law.","Designed oxygen-resistant hydrogenases, having a reduced diameter H-channel, are genetically engineered and transformed into target host cells, for example, into , and tested for hydrogenase activity in the presence of Ovia a modified Clark electrode or other known assay(s). In preferred embodiments, the oxygen-resistant hydrogenase is generated via site-directed mutagenesis. For example, to generate HydA1 mutants, the HydA1 gene of pA1ExBle can be mutagenized in vitro using the Quick Change XL Site-Directed Mutagenesis Kit (Stratagene). Host cells that have incorporated the designed enzymes (having reduced oxygen sensitivity) can be used to photoproduce Hin an oxygen containing environment. Note that these host cells can also be treated with mRNA interference to repress the expression of native hydrogenases, while continuing to allow expression of the inventive engineered hydrogenase(s).","Steered Molecular Dynamics (SMD)","In one embodiment, the in silico designed oxygen-resistant hydrogenase enzymes can be further analyzed for changed or reduced oxygen diffusion within their H-channel by applying SMD via the NAMD program. Kale L. et al., (1999) Computational Physics 151:283; Isralewitz B., (2001) Curr. Opin. Struc. Biol. 11:224. SMD analysis, therefore, provides confirmation and additional baseline data as to the efficiency of the channel modifications and their effects on Odiffusion within the proposed oxygen-resistant hydrogenase.","Oxygen-Resistant Hydrogenase Polypeptides","Oxygen-resistant hydrogenase enzymes of the invention include all proteins that can be constructed from the in silico mutagenesis methods discussed above. For example, any polypeptide having a predicted reduction in hydrogen-channel diameter or volume, as determined by the methods of the invention, is envisioned to be within the scope of the present invention.","In addition, oxygen-resistant hydrogenase enzymes of the invention include isolated polypeptides having an amino acid sequence as shown in  (Cr HydA1), and having one or more substitutions at residues V240, A78, A244, A248, G86, and L93 (note that substitution by tryptophan and other like amino acids is envisioned, including synthetic or derivatized amino acids) (also included are substitutions shown in Tables 1 and 2). The invention includes variants and derivatives of these oxygen-resistant [Fe]-hydrogenase enzymes, including fragments, having substantial identity to these amino acid sequences, and that retain both hydrogenase activity and enhanced tolerance to oxygen (see Example 3 for assays to determine hydrogenase activity in the presence of oxygen). In a preferred embodiment, the oxygen-resistant hydrogenase enzyme is HydA1V240W. Derivatives of the oxygen-resistant hydrogenases include, for example, oxygen-resistant HydA1 enzymes modified by covalent or aggregative conjugation with other chemical moieties, such as lipids, acetyl groups, glycosyl groups, and the like.","Oxygen-resistant hydrogenase enzymes of the present invention can be fused to heterologous polypeptides to facilitate purification. Many available heterologous peptides allow selective binding of the fusion protein to a binding partner, for example, 6-His, thioredoxin, hemaglutinin, GST, and the like.","Polypeptide fragments of the modified oxygen-resistant hydrogenase H-channel polypeptide (that include the relevant residue modification) can be used to generate specific anti-oxygen-resistant hydrogenase antibodies (monoclonal or polyclonal). Generated antibodies can be used to selectively identify expression of oxygen-resistant hydrogenases or in other known molecular and\/or biochemical techniques, for example, in immunoprecipitation or Western blotting.","Variant oxygen-resistant hydrogenase enzymes include fusion proteins formed of a oxygen-resistant hydrogenase and a heterologous polypeptide. Preferred heterologous polypeptides include those that facilitate purification, stability or secretion.","Oxygen-Resistant Hydrogenase Polynucleotides, Vectors and Host Cells","The invention also provides polynucleotide molecules encoding the oxygen-resistant polypeptides of the invention. The polynucleotide molecules of the invention can be cDNA, chemically synthesized DNA, DNA amplified by PCR, RNA or combinations thereof.","The present invention also provides vectors containing the polynucleotide molecules of the invention, as well as host cells transformed with such vectors. Any of the polynucleotide molecules of the invention may be contained in a vector, which generally includes a selectable marker, and an origin of replication, for propogation in a host. The vectors also include suitable transcriptional or translational regulatory sequences, such as those derived from algae operably linked to the oxygen-resistant hydrogenase polynucleotide molecule. Examples of such regulatory sequences include transcriptional promoters, operators, enhances, and mRNA binding sites. Nucleotide sequences are operably linked when the regulatory sequence functionally relates to the DNA encoding the target protein. Thus, a promoter nucleotide sequence is operably linked to a oxygen-resistant hydrogenase DNA sequence if the promoter nucleotide sequence directs the transcription of the oxygen-resistant hydrogenase sequence.","Selection of suitable vectors for the cloning of oxygen-resistant hydrogenase polynucleotides of the invention will depend on the host cell in which the vector will be transformed\/expressed. For example, the plasmid pLam91-1 (see ) was used to create a 980 bp HydA1 PstI promoter fragment cloned into a unique PstI site of pLam91-1, creating the HydA1 promoter-HydA1 cDNA fusion construct, pA1Ax. The Blecassette of pSP108 confers Bleomycin resistance in transformed , and was inserted into the Tfi1 site of pA1Ax, creating pA1AxBle (see ). This was particularly useful in the construction of expression oxygen-resistant [Fe]-hydrogenase vectors for use in green algae.","Suitable host cells for expression of target polypeptides of the invention include green algae, for example cells and cyanobacteria, both of which utilize water in growth, which is also a substrate for the hydrogenase enzymes. Typically, green algae cells are transformed by a glass bead method as is known in the art. Cells exhibiting the target selectable marker, for example resistance to bleomycin, are picked and patched onto fresh TAP+Ble plates and re-patched an additional 2-3 times to ensure the isolation of stable integrates.","HProduction","Green algal cultures that express oxygen-resistant hydrogenase of the invention may be used to photoproduce Hin the presence of oxygen. In one embodiment of the invention, the transformed cells are grown in a photobioreactor photoautotrophically, photoheterotrophically in TAP, or other like growth media to a concentration of 5-50 \u03bcg\/ml chlorophyll, and Hharvested. Note that in some embodiments, the cells are grown under selective pressure that ensures that the cells maintain the oxygen-resistant hydrogenase, for example in bleomycin, where the construct used to transform the host cell confers the selective pressure.","In another embodiment, the oxygen-resistant hydrogenase of the invention may be transformed into target algae, under the control of the endogenous HydA1 promoter, for nighttime enzyme generation and daytime H-production. See Boichenko et al., (2003) Photoconversion of Solar Energy, Molecular to Global Photosynthesis: In Press.","It is envisioned that the proceeding discussion on the design, engineering, and construction of oxygen-resistant hydrogenases, as well as the subsequent transformation of host cells with the designed hydrogenases, can be expanded to any iron hydrogenase known or identified in the future having the characteristics for iron hydrogenase enzymes discussed herein.","Having generally described the invention, the same will be more readily understood by reference to the following examples, which are provided by way of illustration and are not intended as limiting.","To facilitate the design and engineering of mutant oxygen-resistant HydA1 enzymes, a theoretical structure of HydA1 was generated by homology modeling to the solved X-ray structure of [Fe]-hydrogenase, CpI (). The theoretical model was generated by homology modeling using Swiss-model software as described by Guex et al. Guex et al., (1997) Electrophoresis 18:2714-2723. The resulting HydA1 model was subjected to several rounds of energy minimization using GROMOS. An alignment of the HydA1 and CpI amino acid sequences show they share a high degree of homology (45% identity, 58% similarity) within the essential domains, i.e., active site and H-channel, that comprise the core region of [Fe]-hydrogenases (see ). Stothard P., (2000) BioTechniques 28(6) 1102. Note that the degree of conservation increases for H-channel sub-domains, where the two proteins share 62% identity and 92% similarity (, overhead arrows). This high level of sequence identity\/similarity shows that the CpI-based HydA1 homology model provides a reasonable approximation of the HydA1 structure and the H-channel environment.","The detailed study of the HydA1H-channel structure was performed, at least partly, to identify residues that project into the H-channel interior. In general, the channel environment was primarily composed of smaller hydrophobic residues, e.g., glycine, alanine, valine, with the exception of the larger phenylalanines at positions 252 and 355 (, black dotted residues). The secondary structure of the H-channel was organized into two \u03b1-helices and two \u03b2-sheets, which extend from the active site to the enzyme surface. The distance between side chain atoms of residues that oppose each other were measured to approximate the average channel diameter over the distance from the catalytic site (Fe-atom of the H-cluster [2Fe-2S]-center) to the protein surface (1 to 4, ). The channel measured 3.85 to 7.44 \u212b in diameter over a distance of 24 to 27 \u212b, making the channel diameter greater than the effective diameters of both H(2.8 \u212b) and O(3.5 \u212b). As a result, the predicted size of the HydA1 H-channel is sufficient to function in H2 diffusion from the active site to the surface, but it is also large enough to allow for the inward diffusion of the inhibitor O.","These results suggest that engineering Otolerance into HydA1 might be accomplished by altering the residues that line the interior of the channel so as to reduce the diameter of the channel and thereby limit Odiffusion to the active site. The potential to reduce the diameter of the channel via residue substitution was initially tested in silico by mutating the H-channel of the HydA1 model. The mutagenesis criteria involved conservative mutation, i.e., hydrophobic\u2192hydrophobic, of specific residues, selection of the lowest energy rotomer, and energy minimization of the resulting structure using GROMOS. Once an energy-minimized structure was obtained, the dimensions of its channel was determined. Several of the channel residues proved to be unameanable to mutation and were left unchanged, i.e., I82, L89, F252 and F355, i.e., the Guex program determined that changes at these locations would provide only minimal (non-significant) change to the H-channel diameter\/volume. However, promising mutants were generated from alteration of several residues that were spaced over the entire length of the channel (see  and Table 1). Substantial reductions of channel diameter were obtained by mutating residues A78 and V240 (proximal to active site); A244, A248 and G86 (mid-channel); and L93 (protein-solvent boundary, distal to active site) to bulkier amino acids (Table 1 and ). The individual mutations listed in Table 1 caused reductions in diameter that ranged from 0.5 to 1.90 \u212b (Table 2). The HydA1 mutant that combined the A248I and L93F mutations located at the channel-solvent boundary (, zone 4) showed an average decrease in size from 5.21 to 3.34 \u212b, less than the effective diameter of O(3.5 \u212b). When the mutations listed in Table 2 were combined into a single HydA1 mutant, the average overall channel diameter was reduced from an average 5.71 to an average of 4.31 \u212b (Table 2), noting however that there are several locations along the H-channel with reductions in the diameter at or near the average diameter of O.",{"@attributes":{"id":"p-0067","num":"0066"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Predicted Effects of Selected HydA1 H2-Channel"},{"entry":"Mutations on Channel Environment"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Mutation","Location","Effects"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["A781","Adjacent to Fe-atom,","Bulkier isoleucine side"]},{"entry":[{},"across from V240","chain projects closer to"]},{"entry":[{},{},"V240, and the [2Fe\u20142S]-"]},{"entry":[{},{},"center Featom."]},{"entry":["V240W","Adjacent to A78, above","Bulkier tryptophan side"]},{"entry":[{},"Fe-atom.","chain reduces distance to"]},{"entry":[{},{},"A78, and partially shields"]},{"entry":[{},{},"Fe-atom."]},{"entry":["A244L","Mid-channel, opposes I82","Bulkier leucine side chain"]},{"entry":[{},{},"projects further into"]},{"entry":[{},{},"channel."]},{"entry":["G86I","Mid-channel, opposes A248","Isoleucine side chain adds"]},{"entry":[{},{},"bulk, and projects into"]},{"entry":[{},{},"channel."]},{"entry":["A248I","Mid-channel, near surface,","Isoleucine extends further"]},{"entry":[{},"opposes G86 and L89","into channel, adds more"]},{"entry":[{},{},"bulk to hydrophobic"]},{"entry":[{},{},"surface."]},{"entry":["L93F","Channel-Surface boundary","Narrows the channel"]},{"entry":[{},{},"opening at protein surface-"]},{"entry":[{},{},"solvent boundary"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0068","num":"0067"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Distances Between Channel Determinants In HydA1"},{"entry":"and HydA1 Mutants Based on Modeling Studies"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Distances (\u212b)","Average Zone Size (\u212b)"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Determinant","HydA1","HydA1","HydA1","HydA1"]},{"entry":["Zone","Pairs","wild type","mutant","wild type","mutant"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}},{"entry":["1","A(I)78::Fe","6.25","4.66","6.90","5.04"]},{"entry":[{},"V(W)240::Fe","6.14","5.02"]},{"entry":[{},"V(W)240::A(I)78","7.43","5.43"]},{"entry":["2","A(L)244::I82","4.50","3.90","5.30","4.87"]},{"entry":[{},"F355::G(I)86","6.86","5.80"]},{"entry":["3","G(I)86::A(I)248","4.38","3.54","5.42","3.92"]},{"entry":[{},"G(I)86::T247","6.01","4.39"]},{"entry":[{},"A(I)248::L89","3.85","3.26"]},{"entry":[{},"A(I)248::F355","7.44","4.54"]},{"entry":["4","L90::A(I)248","6.11","3.67","5.21","3.34"]},{"entry":[{},"L(F)93::F252","4.31","3.01"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"left","id":"FOO-00001"},"sup":"a"}},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"left","id":"FOO-00002"},"sup":"b"}},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"left","id":"FOO-00003"},"sup":"c"}}]}}]}}},"The above results indicate that modeling of the HydA1 structure has revealed a hydrophobic channel extending from the active site to the enzyme surface. This channel would appear to be conserved in other [Fe]-hydrogenases. The channel's secondary structure is mainly \u03b1-helical, which suggests that the channel domain is fairly rigid. Perhaps the rigidity of the channel structure helps to prevent its collapse during folding. Volbeda et al., (2002) Int. J. Hyd. Energy 27:1449-1461. Rigidity would also be expected to contribute to conformational stability of the channel in the folded protein, and a static model should give reasonable approximations of shape and size. Our measurements of the HydA1 channel demonstrate that it is sufficient in diameter not only to allow for diffusion of the product Hbut also the larger-sized inhibitors Oand CO. Since enzyme inhibition occurs quickly (minutes), following exposure of O(Happe et al., (1994) Eur. J. Biochem. 222:769-774), the channel would not appear to be highly restrictive to inhibitor diffusion, which is in agreement with our analysis. This data illustrates the utility of the present invention for engineering O-resistant, [Fe]-hydrogenase by manipulation of residues within the conserved Hchannel. This modeling approach can be used in enzymes that have one channel or multiple channels to reduce inhibitor access to an enzyme active site.","To test the ability of the predicted HydA1H-channel mutants for limiting Oinhibition, an algal HydA1 expression system was created using the HydA1 endogenous promoter. From the modeling discussed in Example 1, the V240W mutation was selected for further examination. In vivo expression of the V240W mutant was performed and further testing of the mutant for Oresistance hydrogenase activity performed. Note that the V240W mutation is predicted to cause a constriction of the channel near the active site (see ). In addition, the tryptophan projects over the Fe-atom, partially shielding it from the channel domain.","The strain cc849 (cw10, mt-) was used as the wild type parent strain throughout the remainder of this Example. Growth of liquid cultures were performed photoheterotrophically in TAP medium (Harris E, (1989) The Source Book, Academic Press, New York) with a continuous stream of 5% COunder cool-white fluorescent light (150 \u03bcE\/m\/sPAR). Growth on solid medium was performed on TAP agar plates (TAP medium with 1.4% w\/v agar). Note that when selection of Bleomycin resistance was performed, solid TAP medium was supplemented with 10 \u03bcg\/ml Zeocin (Invitrogen).","A plasmid construct pLam91-1, containing the HydA1 cDNA and 3\u2032-terminator regions cloned into the EcoRI-XhoI sites of pBluescript SK, was used to generate an algal HydA1 expression construct. A 980 bp HydA1 PstI promoter fragment was cloned into the unique PstI site of Lam91-1, creating the HydA1 promoter-HydA1 cDNA fusion construct, pA1Ex. The Blecassette of pSP108 that confers Bleomycin resistance in transformed (Stevens et al., (1996) Mol Gen Genet. 251:23-30) was inserted into the TfiI site of pA1Ex, creating pA1ExBle.","Site-directed mutagenesis was performed on HydA1 to generate HydA1 mutants for expression in . The HydA1 gene pA1ExBle was mutagenized in vitro using the Quick Change XL Site-Directed Mutagenesis Kit of Stratagene. Oligonucleotides (Integrated DNA Technologies) used for mutagenesis were designed based on the kit requirements. Mutant pA1ExBle constructs were sequenced to confirm the presence of individual mutations. The HydA1 mutant, V240W, contains a valine to tryptophan substitution at amino acid position 240 of the mature protein.",{"@attributes":{"id":"p-0074","num":"0073"},"i":"C. reinhardtii ","sup":["r","r "]},"To ensure that the HydA1 cDNA genomic insert having the V240W mutation was present in the transformed , PCR and sequencing was performed on Bletransformants. Total genomic DNA was isolated from individual Bletransformants using the Plant Genomic Kit (Qiagen). A total of 0.5 to 1.0 \u03bcg of purified genomic DNA was digested with either SacI or EcoRI and used as template in a PCR reaction consisting of the HydA1 internal primers (5\u2032-CACGCTGTTTGGCATCGACCTGACCATCATG-3\u2032 [SEQ ID NO:8] and 5\u2032-GCCACGGCCACGCGGAATGTGATGCCGCCCC-3\u2032 [SEQ ID NO:9]), 1 unit KOD HotStart polymerase (Novagen), 10 mM MgSO4, 25 mM of each dNTP, 2% DMSO (v\/v), and water to a total volume of 50 \u03bcl. The presence of a HydA1 cDNA genomic insert results in an additional 780 bp HydA1 cDNA product together with the 1120 bp HydA1 genomic product. PCR reactions were run on 1\u00d7TAE agarose gels (1.25% agarose w\/v), stained with ethidium bromide, and photographed (not shown). The 780 bp band, corresponding to the HydA1 cDNA insert, was purified and sequenced to confirm the presence of V240W mutation. Two Bleclones, mt18 and mt28, were shown to possess the HydA1V240W construct (see ).","The O-sensitivity of [Fe]-hydrogenase activity in strains mt18 and mt28 carrying the HydA1V240W mutation was tested in either whole cells or whole cell extracts of anaerobically induced cultures. Hydrogenase activities were measured as Hgas photoproduction by whole cells as previously described. Ghirardi et al., (1997) App. Biochem. Biotech. 63-65:141-151; Flynn et al., (2002) Int. J. Hyd Energy 27:1421-1430. Briefly, cells were grown photoheterotrophically in TAP to a concentration of 15-20 \u03bcg\/ml chlorophyll, harvested and resuspended at 200 \u03bcg\/ml chlorophyll in phosphate induction buffer. Ghirardi et al. Clark electrode measurement of O2-resistant hydrogenase activity was performed by adjusting the Oconcentration in the electrode chamber to a set level between 0% and 4%. Once the Olevel had stabilized, a stream of Ar gas was passed over the chamber to maintain a constant O2 concentration. A 0.2 ml sample of induced cell suspension was injected into the chamber, and the cells kept in the dark for a two minute period. Light dependent H-photoproduction activity was then induced by illumination.","In addition, to measure hydrogenase activity directly, reduced methyl viologen (MV) was used as an artificial electron donor for Hproduction by solubilizing whole cells as previously described. Flynn et al. Tolerance to Owas measured by incubating 1 ml of induced cells in a dark, sealed glass bottle and injecting Oto achieve a final atmosphere of 1 to 4% (v\/v). Samples were incubated for two minutes then purged with Ar gas for five minutes. A 1 ml mixture of reduced MV and Triton X-100 in a phosphate buffer was added, samples were mixed for three to five minutes, and 0.1 ml of 100 mM reduced Na-dithionite injected to start the reaction. The reaction mixtures were incubated for 30 minutes at room temperature with stirring, and reactions were stopped by the addition of 0.1 ml 20% trichloroacetic acid (TCA). The hydrogen content of a 0.2 ml headspace sample was measured by gas chromatograph. Three separate headspace samples were assayed, and the values were averaged to attain final hydrogen-production rates.","As shown in Table 3, all three strains, cc849, mt18, and mt28, exhibited similar levels of hydrogenase activity (rate of Hphotoproduction) under completely anaerobic conditions. Note that as has been shown in previous studies (Ghirardi et al, supra; Flynn et al, supra), pretreatment of induced wild type cells with Ois sufficient to cause a significant decline in H2 production rate (, white bars). When induced wild-type cells were pre-treated with O2 at a concentration of 1.7 to 3.5%, the H2 photoproduction rate declined by 90 to 100% respectively. However, the exposure of mt18 or mt28 induced cells to similar O2 treatments showed H2 photoproduction activity had significant resistance to inactivation. After exposure to 1.7 to 2.2% Oconcentrations, the Hphotoproduction rates remained 3.8 to 7 fold higher in mt18, and 3.2 to 13 fold higher in mt28 compared to activities in wild-type cells under identical conditions (see ). At 3.5% Otreatment, the H2 photoproduction rates in both mt18 and mt 28 were low, but detectable, whereas residual activity in wild-type cells was undetectable ().",{"@attributes":{"id":"p-0079","num":"0078"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Hydrogenase Activity By the Clark Electrode Assay"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"HPhotoproduction Rate"]},{"entry":[{},"Strain","(\u03bcmol H2\/mg chl\/h)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"cc849","10.4"]},{"entry":[{},"mt18","14.1"]},{"entry":[{},"mt28","10.7"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"The light-induced production of hydrogen by whole cells is a metabolic process and depends on many electron transfer steps. Zhang et al., (2000) Trends Biotech. 18(12):506-511; Melis et al., (2001) Plant Physiol. 127:740-748; Melis et al., (2000) Plant Physiol. 122:127-135. A more direct measurement of hydrogenase activity can be accomplished in solubilized whole cells using reduced MV (Mv) as electron donor for H2 gas production by hydrogenase in the dark. Under completely anaerobic conditions, the Mv\u2192Hreaction rates were similar in value for either induced wild-type or mutant cells (see Table 4). As shown in , a two-minute exposure of induced wild-type to various Oconcentrations caused hydrogenase activity to decline. After exposure of Oconcentrations of 1% to 4% hydrogenase activities in wild-type cells decreased to between 10 and 1.5% respectively (), similar to the results shown in . In comparison, both mt16 and mt28 containing the HydA1V240W construct exhibited significant levels of Oresistant hydrogenase activity (see ). Exposure of mt18 to Oat 1% to 4% concentration caused hydrogenase activities to decline by 76% to 96%, whereas mt28 activities declined only 12% to 76% (). As a result, mt18 hydrogenase activities were 2- to 3-fold higher, and mt28 activities 8- to 15-fold higher than activities in wild-type cells after exposure to similar Otreatments.",{"@attributes":{"id":"p-0081","num":"0080"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 4"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Hydrogenase Activity By the Methyl Viologen Assay"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"154pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"H2 Photoproduction Rate"]},{"entry":[{},"Strain","(\u03bcmol H2\/mg chl\/h)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"cc849","31.3"]},{"entry":[{},"mt18","32.9"]},{"entry":[{},"mt28","35.5"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}}},"This Example illustrated the utility of modeling residue substitutions within the H-channel to constrict the channel from Opassage to the [2Fe-2S]-center. In particular, the Example illustrated that substitution of tryptophan for valine at position 240 of HydA1 caused an increase tolerance to Oin the mutant hydrogenase. The difference in the structure change made to HydA1V240W and the effects of that change are similar to the observed differences in structure and O-resistance of H-sensing [NiFe]-hydrogenases compared to catalytic [NiFe]-hydrogenases. Volbeda et al., (2002) Int. J. Hyd. Energy 27:1449-1461; Bernhard et al., (2001) 276:15592-15597. Active-site proximal channel residues of O-resistant, H-sensing [NiFe]-hydrogenases contain the bulky, hydrophobic amino acids isoleucine and phenylalanine. Identical positions in the O-sensitive, catalytic [NiFe]-hydrogenases encode the smaller-sized residues valine and leucine respectively. The difference in amino acid composition is suggested to result in the shielding of the [NiFe]-cluster and constriction of the channel. Volbeda supra and Bernhard supra.","The invention has been described with reference to specific examples. These examples are not meant to limit the invention in any way. It is understood for purposes of this disclosure, that various changes and modifications may be made to the invention that are well within the scope of the invention. Numerous other changes may be made which will readily suggest themselves to those skilled in the art and which are encompassed in the spirit of the invention disclosed herein and as defined in the appended claims.","This specification contains numerous citations to patents and publications. Each is hereby incorporated by reference in their entirety for all purposes."],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0014","num":"0013"},"figref":["FIG. 1A","FIG. 1B"],"b":"1","sub":"2"},{"@attributes":{"id":"p-0015","num":"0014"},"figref":"FIG. 2","sub":"2"},{"@attributes":{"id":"p-0016","num":"0015"},"figref":"FIGS. 3A and 3B","sub":["2","2"],"b":["3","3"]},{"@attributes":{"id":"p-0017","num":"0016"},"figref":"FIGS. 4A and 4B","sub":"2","b":["4","4"]},{"@attributes":{"id":"p-0018","num":"0017"},"figref":"FIG. 5","i":"C. reinhardtii "},{"@attributes":{"id":"p-0019","num":"0018"},"figref":"FIG. 6","sub":["2 ","2","2 ","2 "],"sup":["\u22121","\u22121"]},{"@attributes":{"id":"p-0020","num":"0019"},"figref":"FIG. 7","sub":["2","2 ","2 ","2"],"sup":["\u22121","\u22121"]},{"@attributes":{"id":"p-0021","num":"0020"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0022","num":"0021"},"figref":"FIG. 9"}]},"DETDESC":[{},{}]}
