---
title: Receptor tyrosine kinase assays
abstract: Methods for detecting phosphorylation of receptor tyrosine kinases (“RTKs”) upon activation are provided. The method employs cells comprising two fusion products: (1) an RTK fused to a small fragment of β-galactosidase and (2) a phosphotyrosine binding peptide fused to the large fragment of β-galactosidase, where the 2 fragments weakly complex to form an active enzyme, and optionally a construct for a cytosolic RTK phosphorylating kinase, when the RTK does not autophosphoryate. To detect phosphorylation a β-galactosidase substrate is added to the cells, whereby product formation indicates the occurrence of phosphorylation.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08067155&OS=08067155&RS=08067155
owner: DiscoveRx Corporation
number: 08067155
owner_city: Fremont
owner_country: US
publication_date: 20090806
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATIONS","STATEMENT OF GOVERNMENTAL SUPPORT","REFERENCE TO SEQUENCE LISTING, COMPUTER PROGRAM, OR COMPACT DISK","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DESCRIPTION OF THE SPECIFIC EMBODIMENTS","EXPERIMENTAL","EXAMPLE:","CONCLUSION"],"p":["This application claims priority from U.S. Provisional Patent Application No. 61\/089,799, filed Aug. 18, 2008, which is hereby incorporated by reference in its entirety.","None.","Applicants assert that the paper copy of the Sequence Listing is identical to the Sequence Listing in computer readable form found on the accompanying computer file. Applicants incorporate the contents of the sequence listing by reference in its entirety.","1. Field of the Invention","The field of this invention is the determination of phosphorylation of tyrosine on a receptor tyrosine kinase and screening of compounds that affect the phosphorylation.","2. Background","Presented below is background information on certain aspects of the present invention as they may relate to technical features referred to in the detailed description, but not necessarily described in detail. These materials may be consulted for specific language, which may be omitted from the present specification and are, as stated in the Conclusion, incorporated by reference. The discussion below should not be construed as an admission as to the relevance of the information to the appended claims or the prior art effect of the material described.","Pharmaceutical small molecule drug discovery is predicated on discovering compounds that bind to receptors or cytosolic proteins and act as agonists, antagonists, inverse agonists or modulators. One important class of proteins known as receptor tyrosine kinases (\u201cRTKs\u201d) are attractive targets, as these proteins act to induce a number of disease associated pathways. An important focus of pharmaceutical drug discovery is the identification of surrogate ligands for proteins, e.g., receptors, kinases, or other proteins in the pathway of phosphorylation. Of particular interest in this respect is a subclass of cell surface receptor proteins known as receptor tyrosine kinases. Another important class of proteins is the cytosolic kinases, which can phosphorylate one or a plurality of RTKs. By activating or inhibiting these kinases, one can inhibit the activation of the RTK target of the cytosolic kinase.","The RTK family functions in the regulation of cell growth, cell differentiation, adhesion, migration and apoptosis (Blume-Jensen and Hunter 2001 411:355-65) (Ullrich and Schlessinger 1990 61:203-12) (Schlessinger 2000 103:211-25) (Hubbard and Till 2000 69:373-98). A number of human diseases have been linked to alterations in RTKs (Akin and Metcalfe 2004 114:13-9) (Verheul and Pinedo 2003 (Barc) 39 Suppl C: 81-93) (Corfas et al., 2004 7:575-80). Many RTKs have been identified as oncogenes in transforming retrovirus or human cancers (Hunter 2000 100:113-27) (Shawver et al., 2002) (Muller-Tidow et al., 2004), and recent reports indicate that RTKs may play a critical role in almost all types of human cancer (Shawver et al., 2002 1:117-23) (Prenzel et al., 2000 2:184-90) (Mass 2004 58:932-40). Both naturally occurring and artificial ligands that modulate RTK activity and signaling thus would be of tremendous interest from a therapeutic standpoint with respect to cancer and other diseases. (Haluska and Adjei 2001 2:280-6) (Sawyer et al., 2003 Suppl:2-10, 12-5). The ability to quickly, efficiently, and effectively screen vast libraries of compounds for particular activities has become a goal of the pharmaceutical industry. Desirably, the methods provide more than just binding information, frequently employing whole cells, where biological processes occur in relation to the compounds being screened.","Many cytokine receptors do not possess intrinsic kinase activity. However, they also initiate intracellular cascades of tyrosine phosphorylation. To do this they interact with separate proteins that are in the cytosol termed non-receptor tyrosine kinases (NRTK's). These proteins, such as the JAK kinases, bind to the intracellular domain of cytokine receptors. Once the cytokine receptor binds ligand and oligomerizes, this brings the JAK proteins into close proximity initiating trans-phosphorylation (by the JAK proteins) of the JAK proteins and the associated receptor.","High throughput screening has become a commonly employed strategy to identify novel compounds with particular activities from a diverse chemical library of compounds. Often, high throughput screening assays are either based upon measuring compound binding to defined molecular targets or measuring functional outputs resulting from compound\/receptor interactions. However, both binding assays and functional assays have limitations. For example, for various technical reasons, binding assays are preformed in non-physiological conditions. Artificial, non-physiological conditions may impact and influence receptor pharmacology, leading to increased unreliability and difficulty in accurate interpretation of the data. Another drawback arises from the nature of the assay, which measures receptor binding only. Thus, binding competition assays do not provide information regarding the physiological function of ligands, such as whether the ligand functions as an agonist or antagonist. Since the only information obtained is binding, where the binding need not be at the target site, there can be many false positives.","Functional assays overcome many of the limitations associated with binding competition assays. Normally, cells are employed, which have the capability to respond to agonist binding as part of the assay protocol Therefore, the assays can provide a measure of the activity resulting from binding and allow for activity\/concentration determinations. With the assay being performed under physiological conditions within the cell, one obtains results that more closely approximate the results that may be anticipated in vivo.","Several functional assays have been described for receptor tyrosine kinases. Exemplary assays include the quantification of autophosphorylation of RTKs (Olive 2004 1:327-41), measurement of phosphorylation of RTKs and downstream signaling molecules (ibid), measurement of intracellular calcium release (Dupriez et al., 2002 8:319-30), or measurement of RTK dependent cell proliferation (Mosmann 1983 65:55-63) (Bellamy 1992 44; 690-708). Despite the substantial variety of assays that have been developed for evaluating ligands for RTKs, there is still a substantial need for additional assays that can provide advantages as to the nature of the protocol, the involvement of the technician in performing the assay, the number of steps that can lead to errors in the result, the choice of equipment, the effect of organic solvents, the dynamic range and the sensitivity of the assay.","Relevant Patent Literature","U.S. patents and applications include U.S. Pat. Nos. 5,667,980; 5,773,237; 5,976,893; a group of patents with the same disclosure U.S. Pat. Nos. 5,891,650, 5,914,237, 6,025,145, and 6,287,784; 6,413,730, 2004\/0038298, 2004\/0161787; 2006\/0199226; and 2008\/0103107.","Mammalian cells are provided comprising at least two genetic expression constructs: a first construct of an RTK fused to a first member of a pair of fragments of \u03b2-galactosidase; a second construct of a polypeptide, (\u201ca phosphotyrosine binding peptide\u201d) that binds to the phosphorylated RTK fused to the second fragment of \u03b2-galactosidase; and as appropriate, a third construct expressing a cytosolic kinase phosphorylating tyrosine receptor kinases. Upon stimulation of the RTK, the expression product of the second construct binds to the phosphorylated RTK bringing the two fragments into proximity to form an active \u03b2-galactosidase, where phosphorylation may result from the RTK or the cytosolic kinase. The two fragments have a low affinity for each other, so that there is relatively low formation of \u03b2-galactosidase in the absence of the binding of the two expression products. Addition of a substrate for the \u03b2-galactosidase that produces a detectable product provides a readout related to the degree of binding of the two expression products. Examples of these peptides and kinases are given below.","Methods are provided for determining the phosphorylation of RTKs. Double or treble stable transformed cells are employed comprising two, and optionally a third, expression constructs: (1) a fusion of an RTK with a member of an enzyme fragment pair at the cytosolic C-terminus of the RTK; (2) a polypeptide sequence that binds to the phosphorylated RTK (a phosphotyrosine binding domain) fused to the complementary member of the enzyme fragment pair; and where the RTK does not auto-phosphorylate, (3) a non-receptor tyrosine kinase, generally with a strong promoter for over expression. The enzyme pair is derived from \u03b2-galactosidase, where the fragments are relatively unable to independently complex to form an active enzyme, namely having a weak affinity for each other, but able to form an active \u03b2-galactosidase when the proteins bind together to which the fragments are fused.","In performing the method, the cells are grown in an appropriate medium. The cells are seeded in basal media with Bovine Serum Albumin (BSA), using commonly employed conditions. For determining whether a candidate compound is an agonist or inverse agonist, the putative agonist is added and the cells incubated for a sufficient time for a reaction to occur. For determining whether a candidate compound is an antagonist, the cells are first incubated with the putative antagonist for sufficient time for the antagonist to bind, followed by the addition of an agonist and incubation for sufficient time for a reaction to occur. For determination of whether a candidate compound is a modulator, cells are first incubated with a limiting concentration of either an agonist or antagonist, followed by incubation with the proposed modulator to assess a change in the response. A commercially available \u03b2-galactosidase substrate is then added that provides a detectable signal, the substrate optionally combined with a lysing agent, and the signal detected as a measure of the binding activity of the agonist or antagonist.","The media will be conventional for the particular cells used; F-12 for CHO cells, modified Eagle's media for U2OS cells, standard DMEM for HEK cells, etc. Conveniently, the assays are performed in microtiter well plates, where the volumes may vary from about 4 to 100 \u03bcl, more usually 10 to 25 \u03bcl. Generally, about 2 to 20\u00d710cells per 10 \u03bcl are employed in the assay, more usually 3 to 10\u00d710cells per 10 \u03bcl are employed in the assay.","Temperatures will generally range from about 10 to 40\u00b0 C. With the agonist assay, ambient temperatures are convenient, while with the antagonist assay, physiologic temperature (37\u00b0 C.) is convenient. The incubation times employed with the ligands are to provide for a robust result, generally ranging from about 5 min to 2 h, more usually from about 10 min to 1 h, where the ligand has sufficient time to bind to the RTK, phosphorylation to occur and binding of the PTB-comprising-protein to the phosphorylated RTK in sufficient number to occur. (By PTB is intended all phosphotyrosine binding domains including domains referred to as phosphotyrosine domains, SH2 domains, artificially engineered domains, single chain, e.g., Fab, antibodies, and the like.) The precise time employed for achieving at least substantial optimization can be determined empirically.","With a \u03b2-galactosidase substrate able to cross the cell membrane, the substrate is added as a dissolving solid or in a solution. Alternatively, the reagent solution may provide for permeabilizing or lysis of the cells and release of any complex formed in the cell to the assay medium. Any conventional lysis buffer may be employed that does not interfere with the \u03b2-galactosidase reaction with its substrate. Various ionic buffers, such as CHAPS, may be employed at 1-5%, generally not more than 3%, in a convenient buffer, such as PBS or HEPES, where numerous other substitutes are known in the field. The reagent solution will generally be about 0.5-2 times the volume of the assay medium. After addition of the \u03b2-galactosidase substrate, the solution will usually be incubated for from 5-200 min, usually 10-150 min and the signal read. The temperature will generally be at the temperature of the incubation medium or conveniently in the range of about 20-40\u00b0 C.","The \u03b2-galactosidase substrate will provide a fluorescent or luminescent product. A fluorescent or chemiluminescent reader, respectively, is then used to read the signal. Desirably a luminescent reagent and optionally a signal enhancer are employed. The luminescent reagent will be in large excess in relation to the maximum amount of \u03b2-galactosidase that is likely to be formed. Conveniently, a luminescent substrate is used, available as Galacton Star from ABI in conjunction with the Emerald II enhancer. Any equivalent luminescent substrate composition may be employed. The substrate will be present in about 1 to 10 weight percent, while the enhancer will be present in about 10 to 30 weight percent of the reagent solution. These amounts will vary depending upon the particular substrate composition employed. The reagent solution may be prepared as a 5-20\u00d7 concentrate or higher for sale or the solids may be provided as powders and dissolved in water at the appropriate proportions.","Standards will usually be used, whereby the signal is related to the concentration of a known stimulator performed under the same conditions as the candidate compound. A graph can be prepared that shows the change in signal with the change in concentration of the standard compound. The assay is sensitive to ECs of not greater than nanomolar of candidate compound, generally sensitive to less than about 1 \u03bcM, in most cases sensitive to less than about 500 nM, frequently sensitive to less than 100 nM and can in many cases detect ECs of less than 50 nM. The S\/B (signal\/background) ratios are generally are at least about 2 fold, more usually at least about 3 fold, and can be greater than about 50 fold.","Instead of screening compounds for agonist or antagonist activity, e.g., an active ligand, one can screen physiological or other samples for ligand activity, namely as a diagnostic tool. A sample is used in place of the candidate compound and the assay is performed in the same way. Physiological samples may include blood, plasma, saliva, CSF, tissue, lysed cells, etc. The sample may be subject to prior treatment, such as filtration, centrifugation, citration, heating, precipitation, etc. The amount of sample will depend upon the anticipated level of the agonist or antagonist. The subject method has the advantage over an immunoassay in measuring only components that actively bind the RTK rather than epitopic sites of the components.","For convenience kits can be provided. In the subject assays, the EA fusion protein may be provided as a construct for expression of EA to be introduced into the cell or cells may be provided that are appropriately modified to provide EA in the cell. Generally, the kits would include an insert with instructions for performing the assay. The instructions may be printed or electronic, e.g., a CD or floppy disk. The kits find use in marketing the product and encouraging the use of the assay for research and commercial settings.","Various, known cell lines may be employed for the assay. Cell lines that find use include U2OS, CHO, HeLa, HepG2, HEK, and the like.","The RTKs may be divided into self-phosphorylating receptors and receptors that require an independent kinase, where a large number of cytosolic kinases are known that have a relatively narrow repertoire of RTKs that each phosphorylates when the RTK is activated. Therefore, the subject assays allow for the investigation of activity of compounds that are ligands for the RTKs or activators or inhibitors of cytosolic kinases, where the RTKs that are phosphorylated by the cytosolic kinase are known","There are a large number of RTKs that initiate a number of different pathways and new RTKs are likely to be discovered. The RTKs have been divided into a number of classes as follows: RTK class I (EGF receptor family); II (insulin receptor family); III (PDGR receptor family); IV (FGF receptor family); V (VEGF receptor family); VI (HGF receptor family); VII (Trk receptor family); VIII (AXL receptor family); IX (AXL receptor family); X (LTK receptor family); XI (TIE receptor family); XII (ROR receptor family); XIII (DDR receptor family); XV (KLG receptor family); XVI (RYK receptor family); and XVII (MuSK receptor family).","Each of the RTKs binds to one or more polypeptides having phosphotyrosine binding (\u201cPTB\u201d) domains. A large class of proteins have what is referred to as the SH2 (Src homology 2) domain. These proteins include Abl, GRB2, RasGAP, STAT proteins, ZAP70, SHP2, PI3K, Phospholipase C \u03b3 form, CRK, SOCS, Shc, and Src. Other proteins include FRS2, FE65, Xll\/MINT, NUMB, EPS8, RGS12, DAB, ODIN, JIP-1, ARH and ICAP1. (Further information on these proteins may be found by searching for symbols that contain these abbreviations, as given in Pubmed and\/or at genenames.org\/cgi-bin\/hgnc_search.pl.) Complete sequence information and annotations of the gene symbols used here may also be obtained by those skilled in the art from OMIM or Swiss-Prot.","The PTB proteins need not be from the same species as the RTK, so long as they have a sufficient binding affinity to provide for a robust assay. The entire PTB protein need not be used, so long as the fragment that is employed comprises the PTB domain and has the desired level of affinity for the RTK phosphotyrosine site.","The RTKs that depend upon cytosolic receptors include T and B-cell receptors, integrins, interferon receptors, interleukin receptors, GP130 associated proteins, etc. Among the families of receptors that find application in the subject invention, the following are illustrative. Single chain: EPOR, GHR, CFSR, PRLR, MPL; IFN Family: IFNAR1, 2, IFNGR1, 2; \u03b3C Family: IL2RA, B, G, IL4R, IL2RG (Type 1 receptor), IL4R-IL13RA1 (Type II receptor), IL7R, IL2RG, IL9R, IL15RA, IL2RB, IL10RA, B, IL12RB1, 2, IL13RA1; IL3 Family: IL3RA, CSF2RA, B, IL5RA, GP130 Family: IL6R, IL6ST, IL11RA, LIFR, OSMR, IL6GT, CNTFR, IL6ST, and LIFR.","Where a wild-type cytosolic kinase (NRTK) is not endogenously available and is required for phosphorylation, in addition to the RTKs indicated immediately above and the polypeptides having a PTB domain, there will also be expression of an exogenously introduced wild-type NRTK with a strong promoter to provide over expression of the NRTK. The overexpression can be determined empirically, but will usually provide a level of the NRTK in substantial excess, 2-fold or more, of the level of the NRTK present.","Typically individual plasmids are employed each with its own antibiotic resistance gene, except where one of the components is multiunit, the units may be on the same vector, e.g., plasmid. The vectors are introduced into the cells sequentially or simultaneously and the transformed cells selected by means of their antibiotic resistance. In order to facilitate the process bicistronic vectors may be used that include internal ribosome entry sites, such that both receptor subunits can be expressed from the same vector.","The detection system is dependent upon the use of \u03b2-galactosidase enzyme fragment complementation. In this system a small fragment of \u03b2-galactosidase and a larger fragment of \u03b2-galactosidase are employed, where the two fragments have a low affinity for each other. The small fragment of \u03b2-galactosidase (\u201cED\u201d) may have the naturally occurring sequence or a mutated sequence. Of particular interest are small fragments of from about 36 to 60, more usually not more than 50, amino acids. Desirably, the ED has a low affinity for the large fragment of \u03b2-galactosidase (\u201cEA), so that there is little complexation between the large and small fragments in the absence of binding of the RTK and PTB peptides. For further description of the small fragments, see U.S. Pat. No. 7,135,325. For further description of mutated EDs, see U.S. patent application publication no. 2007\/0275397, both of which references are incorporated herein in their entirety as if set forth herein. The small EDs and mutated EDs will generally have less than about 0.5, but at least about 0.1, of the activity of the wild-type sequence in the assay of interest or an analogous assay, while having less than about 60% of the conventionally used commercial sequence of about 90 amino acids in the absence of being fused to other proteins. For increasing affinity between the ED and EA, the longer EDs will be used and free of mutations from the wild-type sequence. One can determine empirically for a specific assay the desirable level of affinity of the two fragments, having a higher affinity when the affinity for the PTB peptide for the RTK is low.","Two expression constructs, and optionally a third, are employed: a fusion of one \u03b2-galactosidase fragment with the RTK, usually the small fragment; a fusion of the other \u03b2-galactosidase fragment with the PTB peptide, usually the large fragment; and optionally, an expression construct for an appropriate cytosolic kinase, particularly with a strong promoter. Conveniently, each protein is expressed from a different plasmid, with each plasmid having its own antibiotic resistance gene. Where the receptor is composed of multiple subunits, each encoded by a separate gene, conveniently, one may express more than one protein per plasmid using multiple promoters or bicistronic vectors or IRES.","For expression constructs and descriptions of other conventional manipulative processes, see, e.g., Sambrook, Fritsch & Maniatis, \u201cMolecular Cloning: A Laboratory Manual,\u201d Second Edition (1989) Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. (herein \u201cSambrook et al., 1989\u201d); \u201cDNA Cloning: A Practical Approach,\u201d Volumes I and II (D. N. Glover ed. 1985); \u201cOligonucleotide Synthesis\u201d (M. J. Gait ed. 1984); \u201cNucleic Acid Hybridization\u201d [B. D. Hames & S. J. Higgins eds. (1985)]; \u201cTranscription And Translation\u201d [B. D. Hames & S. J. Higgins, eds. (1984)]; \u201cAnimal Cell Culture\u201d [R. I. Freshney, ed. (1986)]; \u201cImmobilized Cells And Enzymes\u201d [IRL Press, (1986)]; B. Perbal, \u201cA Practical Guide To Molecular Cloning\u201d (1984).","The gene encoding the fusion protein will be part of an expression construct. The gene is positioned to be under transcriptional and translational regulatory regions functional in the cellular host. The regulatory region may include an enhancer, which may provide such advantages as limiting the type of cell in which the fusion protein is expressed, requiring specific conditions for expression, naturally being expressed with the protein, and the like. In many instances, the regulatory regions may be the native regulatory regions of the gene encoding the protein, where the fusion protein may replace the native gene, may be in addition to the native protein, either integrated in the host cell genome or non-integrated, e.g., on an extrachromosomal element.","As indicated, the \u03b2-galactosidase fragment joined to the RTK will be fused at the C-terminus of the RTK, generally linked through a linker that conveniently has from 1 to 2 GGGS units. The large fragment fused to the PTB peptide may be fused directly to the peptide terminus, either N- or C-terminus, or have a linker, the same or different from the small fragment linker.","The following examples are offered by way of illustration and not by way of limitation.","All cells lines used were from DiscoveRx Corporation and express various RTKs tagged with PK (42aa, DSLAVVLQRRDWENPGVTQLNRLAARPPFASWRNSEEARTDR) (SEQ ID NO: 2) in cells stably expressing SH2-EA (Large \u03b2-galactosidase fragment, Full length \u03b2-galactosidase deleted in amino acids 31-41).",{"@attributes":{"id":"p-0050","num":"0000"},"ul":{"@attributes":{"id":"ul0001","list-style":"none"},"li":["Amino acids 1-583=SHC1","Amino Acids 584-597=Linker","Amino Acids 598- 1589=Large \u03b2-galactosidase fragment (EA)"]}},{"@attributes":{"id":"p-0051","num":"0052"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"right"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"(SEQ ID NO: 1)"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"b":"MDLLPPKPKYNPLRNESLSSLEEGASGSTPPEELPSPSASSLGPILPPLP"}},{"entry":{}},{"entry":{"b":"GDDSPTTLCSFFPRMSNLRLANPAGGRPGSKGEPGRAADDGEGIVGAAMP"}},{"entry":{}},{"entry":{"b":"DSGPLPLLQDMNKLSGGGGRRTRVEGGQLGGEEWTRHGSFVNKPTRGWLH"}},{"entry":{}},{"entry":{"b":"PNDKVMGPGVSYLVRYMGCVEVLQSMRALDFNTRTQVTREAISLVCEAVP"}},{"entry":{}},{"entry":{"b":"GAKGATRRRKPCSRPLSSILGRSNLKFAGMPITLTVSTSSLNLMAADCKQ"}},{"entry":{}},{"entry":{"b":"IIANHHMQSISFASGGDPDTAEYVAYVAKDPVNQRACHILECPEGLAQDV"}},{"entry":{}},{"entry":{"b":"ISTIGQAFELRFKQYLRNPPKLVTPHDRMAGFDGSAWDEEEEEPPDHQYY"}},{"entry":{}},{"entry":{"b":"NDFPGKEPPLGGVVDMRLREGAAPGAARPTAPNAQTPSHLGATLPVGQPV"}},{"entry":{}},{"entry":{"b":"GGDPEVRKQMPPPPPCPGRELFDDPSYVNVQNLDKARQAVGGAGPPNPAI"}},{"entry":{}},{"entry":{"b":"NGSAPRDLFDMKPFEDALRVPPPPQSVSMAEQLRGEPWFHGKLSRREAEA"}},{"entry":{}},{"entry":{"b":"LLQLNGDFLVRESTTTPGQYVLTGLQSGQPKHLLLVDPEGVVRTKDHRFE"}},{"entry":{}},{"entry":{"b":"SVSHLISYHMDNHLPIISAGSELCLQQPVERKL","i":"GGGGSGGGGSLES"}},{"entry":{}},{"entry":"TDSLAVVARTDRPSQQLRSLNGEWRFAWFPAPEAVPESWLECDLPEADTV"},{"entry":{}},{"entry":"VVPSNWQMHGYDAPIYTNVTYPITVNPPFVPTENPTGCYSLTFNVDESWL"},{"entry":{}},{"entry":"QEGQTRIIFDGVNSAFHLWCNGRWVGYGQDSRLPSEFDLSAFLRAGENRL"},{"entry":{}},{"entry":"AVMVLRWSDGSYLEDQDMWRMSGIFRDVSLLHKPTTQISDFHVATRFNDD"},{"entry":{}},{"entry":"FSRAVLEAEVQMCGELRDYLRVTVSLWQGETQVASGTAPFGGEIIDERGG"},{"entry":{}},{"entry":"YADRVTLRLNVENPKLWSAEIPNLYRAVVELHTADGTLIEAEACDVGFRE"},{"entry":{}},{"entry":"VRIENGLLLLNGKPLLIRGVNRHEHHPLHGQVMDEQTMVQDILLMKQNNF"},{"entry":{}},{"entry":"NAVRCSHYPNHPLWYTLCDRYGLYVVDEANIETHGMVPMNRLTDDPRWLP"},{"entry":{}},{"entry":"AMSERVTRMVQRDRNHPSVIIWSLGNESGHGANHDALYRWIKSVDPSRPV"},{"entry":{}},{"entry":"QYEGGGADTTATDIICPMYARVDEDQPFPAVPKWSIKKWLSLPGETRPLI"},{"entry":{}},{"entry":"LCEYAHAMGNSLGGFAKYWQAFRQYPRLQGGFVWDWVDQSLIKYDENGNP"},{"entry":{}},{"entry":"WSAYGGDFGDTPNDRQFCMNGLVFADRTPHPALTEAKHQQQFFQFRLSGQ"},{"entry":{}},{"entry":"TIEVTSEYLFRHSDNELLHWMVALDGKPLASGEVPLDVAPQGKQLIELPE"},{"entry":{}},{"entry":"LPQPESAGQLWLTVRVVQPNATAWSEAGHISAWQQWRLAENLSVTLPAAS"},{"entry":{}},{"entry":"HAIPHLTTSEMDFCIELGNKRWQFNRQSGFLSQMWIGDKKQLLTPLRDQF"},{"entry":{}},{"entry":"TRAPLDNDIGVSEATRIDPNAWVERWKAAGHYQAEAALLQCTADTLADAV"},{"entry":{}},{"entry":"LITTAHAWQHQGKTLFISRKTYRIDGSGQMAITVDVEVASDTPHPARIGL"},{"entry":{}},{"entry":"NCQLAQVAERVNWLGLGPQENYPDRLTAACFDRWDLPLSDMYTPYVFPSE"},{"entry":{}},{"entry":"NGLRCGTRELNYGPHQWRGDFQFNISRYSQQQLMETSHRHLLHAEEGTWL"},{"entry":{}},{"entry":"NIDGFHMGIGGDDSWSPSVSAEFQLSAGRYHYQLVWCQK"}]}}]}}},"RTK Fusions include: ErbB-1 (EGFR), ErbB-2, ErbB3, ErbB4, INSR, IGF1R, IRR, PDGFRA, PDGFRB, CSF-1R, C-Kit, FGFR1, FGFR2, FGFR3, FGFR4, Flt3, VEGFR1(Flt-1), VEGFR-2 (Flk-1\/KDR), VEGFR-3 (Flt-4), C-Met, RON, TrkA, TrkB, TrkC, AXL, MER, SKY (TYRO3)(Dtk), LTK (TYK1), ALK, Tie-1, Tie-2, (TEK), DDR1, DDR2, MuSK, RET, EPHA1, EPHA2, EPHA3, EPHA4, EPHA5, EPHA6, EPHA7, EPHA8, EPHA9, EPHB1, EPHB2, EPHB3, EPHB4, EPHB5, EPHB6, CCK4 (PTK7), ROS, AATYK1, AATYK2, AATYK3, ROR1, and ROR2.","Cell lines include: U2OS containing TrkA-PK fusion and SHC1-EA, U2OS containing INSR (insulin receptor)-PK fusion and PLCG2 (Phospholipase C, gamma 2 (phosphatidylinositol-specific))-EA, U2OS containing IGF1R-PK (insulin like growth factor-1 receptor) and SHC1-EA, U2OS containing TrkB-PK fusion and SHC1-EA, TrkC-PK fusion and SHC1-EA, U2OS containing PDGFRB-PK fusion containing PLCG1 (Phospholipase C, gamma 1)-EA, U2OS containing PDGFRB-PK fusion containing Grb2 (Growth factor receptor-bound protein 2)-EA, U2OS containing PDGFRB-PK fusion containing PLCG2-EA, U2OS containing PDGFRB-PK fusion containing PTPN11 (protein tyrosine phosphatase, non-receptor type 11)-EA, U2OS containing PDGFRB-PK fusion containing SYK (spleen tyrosine kinase)-EA, ErbB4 (v-erb-a erythroblastic leukemia viral oncogene homolog 4)-PK fusion and Grb2 (Growth factor receptor-bound protein 2)-EA.","Generally, the expression constructs for the fusion proteins includes at least (in order of 5\u2032 to 3\u2032) a promoter, followed by the receptor or SH2 domain, followed by a linker, followed by either the EA or PK. For all assays, 10,000 cells per well were seeded in 20 \u03bcL media and incubated overnight in 0.1% BSA and appropriate basal media (F-12 or DMEM). For agonist assays, 5 \u03bcL compound was added to cells and incubated at room temperature. For antagonist assays, 5 \u03bcL 5\u00d7 compound was added to cells and incubated at 37\u00b0 C.\/5% COfor 10 minutes, after which 5 \u03bcL 6\u00d7 agonist was added and incubated for 60 minutes at room temperature. SH2-EA complex formation with the RTK was detected with 50% (v\/v) of PathHunter\u00ae Detection Reagent (Dx 93-0001, PathHunter reagents are available from DiscoveRx, Corp., Fremont, Calif.) (Lysis buffer active ingredient 1% CHAPS, Emerald II\u2122 and Galacton Star\u2122 are from Applied Biosystems). Data was read on Packard Victor 2\u00ae or PerkinElmer ViewLux\u00ae readers or comparable instrumentation and analyzed using GraphPad Prism 4\u00ae analysis software.","U2OS cells expressing the TrkA-PK and SHC1-EA fusion proteins were plated at 10K cells\/well in each well of a 384-well plate in serum-free medium with 0.1% FBS. The next day, the cells were treated with 100 ng\/ml NGF in PBS+1% BSA or PBS+1% BSA for different time periods at room temperature and assayed using PathHunter (DiscoveRx) Detection reagent. The results are shown in .","5K cells\/well U2OS TrkA-PK SHC1-EA double-stable cells were plated in each well of a 384-well plate in serum-free medium with 0.1% FBS. The next day, the cells were treated with different concentrations of NGF for 1 hr at room temperature (see above). Then PathHunter chemiluminescent substrate was added and the signal was read 1 hr later. ECof 9.3 ng\/ml and an assay window of 7.8 were obtained. The results are reported in .","5K cells\/well U2OS TrkA-PK SHC1-EA double-stable cells were plated in each well of a 384-well plate in serum-free medium with 0.1% FBS. The next day, the cells were treated with different concentrations of antagonists, such as a commercially available TrkA inhibitor or the Trk inhibitor K252a [(8R*,9S*,11S*)-(\u2212)-9-hydroxy-9-methoxycarbonyl-8-methyl-2,3,9,10-tetrahydro-8,11-epoxy-1H,8H,11H-2,7b,11a-triazadibenzo(a,g)cycloocta(cde)-trinden-1-one] for 10 min at room temperature followed by 20 ng\/ml NGF stimulation for 1 hr at room temperature (see above). Then PathHunter chemiluminescent substrate was added and the signal was read 1 hr later. TrkA inhibitor gave an ICof 18 nM and an assay window of 6.7. K-252a gave an ICof 37 nM and an assay window of 7.6. The results are reported in .","5K cells\/well PDGFRB-PK SH2-containing protein-EA double-stable cells were plated in each well of a 384-well plate in serum-free medium with 0.1% FBS. Six SH2-containing protein-EAs were used: SHC1-EA, Grb2-EA, PLCG-1-EA, PLCG2-EA, PTPN11-EA, and SYK-EA. The next day, the cells were treated with or without 100 ng\/ml PDGF-AB in serum-free medium with 0.1% FBS for 1 hr at room temperature. Then PathHunter chemiluminescent substrate was added and the signal was read 2 hrs later. The results are reported in .","10K cells\/well IGF1R-PK SH2-containing protein-EA double stable cells were plated in a 384-well plate, stimulated with IGF1 (Peprotech, Cat #\/AF-100-11), a known ligand for IFG1R for 3 hours at room temperature according to the assay procedure provided. Following stimulation, detection reagents were added and signal was detected after 1 hour using the PathHunter\u00ae Detection Kit (93-0001). An assay window of 4.4 fold was observed and the ECfor the ligand IGF1 was 17 ng\/ml. The results are reported in .","10 k cells\/well INSR-PK SH2-containing protein-EA double stable cells were plated in a 384-well plate, stimulated with insulin, a known ligand for INSR, for 3 hours at room temperature according to the assay procedure provided. Following stimulation, detection reagents were added and signal was detected after 1 hour using the PathHunter\u00ae Detection Kit (93-0001). An assay window of 7.6 fold was observed and the ECfor the ligand IGF1 was 2.0 ng\/ml.","10K cells\/well TrkB-PK SH2-containing protein-EA double stable cell were plated in a 384-well plate, stimulated with BDNF (Peprotech, Cat #\/450-02), a known ligand for TrkB for 3 hours at room temperature according to the assay procedure provided. Following stimulation, detection reagents were added and signal was detected after 1 hour using the PathHunter\u00ae Detection Kit (93-0001). An assay window of 4.0 fold was observed and the ECfor the ligand BDNF was 4.21 ng\/ml. The results are reported in .","10K cells\/well TrkC-PK SH2-containing protein-EA double stable cells were plated in a 384-well plate, stimulated with NT3 (Peprotech, Cat #\/450-03), a known ligand for TrkC for 3 hours at room temperature according to the assay procedure provided. Following stimulation, detection reagents were added and signal was detected after 1 hour using the PathHunter\u00ae Detection Kit (93-0001). An assay window of 8.1 fold was observed and the ECfor the ligand NT3 was 7 ng\/ml. The results are reported in .","There are a significant number of TKRs that depend upon cytosolic tyrosine kinases for phosphorylation. The assays for activation of such TKR receptors are substantially the same as described above, except that the cells are triply stable having an expression construct over expressing the cytosolic tyrosine kinase. In the following example, U2OS cells containing the following constructs were used: G-CSFR (granulocyte-colony stimulating factor receptor)-PK with neo selection; SHC1-EA with hygro selection and Jak2 with puromycin selection. The results are reported in .","The genes for the RTK, SH2 and NRTK domains may be obtained from any convenient source; commercial supplier, RT PCR from RNA isolated in accordance with conventional procedures using known sequences as probes, and PCR from genomic DNA using primers from known sequences. The genes are PCR amplified to remove the stop codon at the 3\u2032 end and then digested with restriction enzymes where the restriction site is included with the primer sequences. These products are then purified in conventional ways and then ligated into a commercial vector into which the PK or EA has been inserted, in reading frame with the PK or EA. Separating the PK and the EA from the gene is a gly-ser linker that provides flexibility to the fusion proteins to enhance complementation. This linker is not required for activity. The transcriptional regulatory region is generally present in commercial vectors, such as the 5\u2032LTR of the virus used for the vector. Alternatively, the CMV promoter may be used. The resulting vector is then introduced into the host cell by liposome mediated transfection or retrival infection with Moloney murine leukemia virus vector and packaging cell lines. The resulting virus is then used for viral infection. The vectors also include selection genes, such as hygromycin, puromycin and neomycin resistance and cells into which the construct is integrated are selected in a conventional selection medium. The surviving cells are then screened in an agonist dose response assay using the PathHunter\u00ae Detection Kit reagents in white-walled microplates.","It is evident from the above results that the subject method provides for a robust accurate assay for measuring agonists and antagonists for RTKs. Cells are provided that can be used effectively in high throughput screening in a cellular environment, so as to closely define the effect of candidate compounds in a mammalian environment. The protocols are easy, use standard equipment and can be readily automated.","Although the invention has been described with reference to the above examples, it will be understood that modifications and variations are encompassed within the spirit and scope of the invention. Accordingly, the invention is limited only by the following claims. All references referred to in the specification are incorporated by reference as if fully set forth therein."],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0017","num":"0016"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0018","num":"0017"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0019","num":"0018"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0020","num":"0019"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0021","num":"0020"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0022","num":"0021"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0023","num":"0022"},"figref":"FIG. 7"},{"@attributes":{"id":"p-0024","num":"0023"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0025","num":"0024"},"figref":"FIG. 9","sub":"50 "}]},"DETDESC":[{},{}]}
