---
title: Regioselective alkane hydroxylation with a mutant CYP153A6 enzyme
abstract: Cytochrome P450 CYP153A6 from sp. strain HXN1500 was engineered using in-vivo directed evolution to hydroxylate small-chain alkanes regioselectively. Mutant CYP153A6-BMO1 selectively hydroxylates butane and pentane at the terminal carbon to form 1-butanol and 1-pentanol, respectively, at rates greater than wild-type CYP153A6 enzymes. This biocatalyst is highly active for small-chain alkane substrates and the regioselectivity is retained in whole-cell biotransformations.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08361769&OS=08361769&RS=08361769
owner: U.S. Department of Energy
number: 08361769
owner_city: Washington
owner_country: US
publication_date: 20091116
---

{"@attributes":{"id":"description"},"GOVINT":[{},{}],"heading":["GOVERNMENT INTERESTS","REFERENCE TO SEQUENCE LISTING","FIELD OF THE INVENTION","BACKGROUND","DETAILED DESCRIPTION OF THE INVENTION","I. DEFINITIONS","II. DIRECTED EVOLUTION GENERALLY","II. GROWTH of GPo12 (pGEC47\u0394B) ON SHORT-CHAIN 1-ALKANOLS","III. DIRECTED EVOLUTION OF CYP153A6 VARIANTS","IV. WHOLE CELL BIOCONVERSION","EXAMPLES","Example 1","Directed Evolution of CYP153A6-BMO1","Example 2","Bioconversions with CYP153A6-BMO1","Example 3"],"p":["The United States Government has rights in this invention pursuant to Contract No. DE-FG02-06ER15762, between the U.S. Department of Energy (DOE) and the California Institute of Technology.","The electronic readable copy and paper copy of the sequence listings for this invention are identical.","One or more embodiments of the present invention are concerned with variants of novel, mutant CYP153A6 enzymes that display altered and improved regioselectivity in their selective hydroxylation of small-chain alkanes. One or more embodiments also relate to novel variants of CYP153A6 enzymes that are capable of hydroxylating butanes and smaller alkanes at the terminal position at a rate greater than wild-type CYP153A6 enzymes. One or more embodiments also relate to a method of altering the ability of a CYP153A6 enzyme to selectively hydroxylate small-chain alkanes at the terminal position.","Microbial utilization and degradation of alkanes was discovered almost a century ago. Since then, several enzyme families capable of hydroxylating alkanes to alkanols, the first step in alkane degradation, have been identified and categorized based on their preferred substrates. The soluble and particulate methane monooxygenases (sMMO and pMMO) and the related propane monooxygenase and butane monooxygenases (BMO) are specialized on gaseous small-chain alkanes (Cto C), while medium-chain (Cto C) alkane hydroxylation appears to be the domain of the CYP153A6 and AlkB enzyme families.","Conversion of Cto Calkanes to the corresponding alkanols is of particular interest for producing liquid fuels or chemical precursors from natural gas. The MMO-like enzymes that catalyze this reaction in nature, however, exhibit limited stability or poor heterologous expression and have not been suitable for use in a recombinant host that can be engineered to optimize substrate or cofactor delivery. (van Beilen, J. B., et. al., 200774, 13-21). Alkane monooxygenases often cometabolize a wider range of alkanes than those which support growth.","CYP153 is a family of enzymes that has recently been shown to hydroxylate alkanes to the corresponding alkanols. (Maier, T. H., et. al., 2001286, 652-658; Funhoff, E. G., et. al., 2006188, 5220-5227; Funhoff, E. G., et. al., 200740, 806-812; and, van Beilen, J. B., et. al. 200672, 59-65). (16, 9, 10, 31). This family of heme-containing cytochrome P450 monooxygenases has been the subject of biochemical studies and known substrates includes alkanes containing five to eleven carbons. (M\u00fcller, R., et. al., 198948, 243-254; Maier, T. H., et. al., 2001286, 652-658; and, Funhoff, E. G., et. al., 2006188, 5220-5227). (9,16,19). The best characterized member of this family, CYP153A6, hydroxylates it's preferred substrate, octane, predominantly at the terminal position. (M\u00fcller, R., et. al., 198948, 243-254; Funhoff, E. G., et. al., 200740, 806-812; and, Kubota, M., et. al., 200569, 2421-2430). (9,10,14). Nucleotide and amino acid sequences for CYP153A6 from sp. HXN-1500 can be found in, and are hereby incorporated by reference from, the GenBank database under the accession Nos. AJ783967 (SEQ ID NO: 1) and Q65A64 (SEQ ID NO: 2), respectively. However, the CYP153A6 gene used as a parent gene in this study was found to lack the first three codons compared to the AJ783967 sequence, probably due to cloning procedures. However, the encoded protein was otherwise identical, fully functional and performed identical to the Q65A64 protein under all analyzed conditions. Therefore, it will also be referred to as an amino acid having the sequence set forth in SEQ ID NO: 3.","The soluble class II-type three-component CYP153A6 enzymes are the main actors in medium-chain length alkane hydroxylation by the cultivated bacteria to date. (van Beilen, J. B., et. al. 200672, 59-65). Recent studies have shown that high activities on small alkanes can be obtained by engineering bacterial P450 enzymes such as P450cam (CYP101, a camphor hydroxylase) and P450 BM3 (CYP102A, a fatty acid hydroxylase) (Fasan, R., et. al., 200746, 8414-8418; Xu, F. et. al., 200544, 4029-4032). However, the resulting enzymes hydroxylate propane and higher alkanes primarily at the more energetically favorable subterminal positions. Highly selective and desirable terminal hydroxylation is difficult to achieve by engineering a subterminal hydroxylase. (Peters, M., et. al., 2003125, 13442-13450).","Previous approaches, some of which are described above, relate to in vitro evolution of hydroxylase enzymes that does not provide the opportunity to screen for improved activity on a specific alkane substrate directly. Further, the prior methods lead to low or no terminal hydroxylation activity and often result in high uncoupling.","One or more embodiments of the invention include a novel mutant CYP153A6 enzyme, wherein said mutant enzyme hydroxylates small-chain alkanes at significantly higher levels than that of the wild-type CYP153A6 enzyme.","A salient aspect of the one or more embodiments of the present invention is the surprising ability of the invented CYP153A6 mutant enzyme to hydroxylate small-chain alkanes at significantly higher levels than that of the wild-type CYP153A6 enzyme, as depicted in Tables 1 and 2 and .","Another salient aspect of the one or more embodiments of the present invention is the ability of the mutant CYP153A6 enzyme to selectively hydroxylate small-chain alkanes at the terminal position. This finding is both surprising and unexpected since the wild-type CYP153A6 enzyme exhibits a preference for alkane substrates having greater than five carbon atoms. However, the mutant CYP153A6-BMO1 enzyme exhibited a 75% higher reactive activity towards 1-butanol production than wild-type CYP153A6. Additionally, it is also surprising and unexpected that the mutant CYP153A6-BMO1 exhibits increased selectivity for terminal hydroxylation of butane when compared to the wild-type CYP153A6, as shown in Table 2 and . More specifically, the mutant CYP153A6-BMO1 enzyme surprisingly increased the selectivity for terminal hydroxylation of butanol from 78% to 89% of total alkanol product.","In one embodiment, mutants of CYP153A6 from sp. HXN-1500 were engineered using directed evolution, as discussed more completely below. The initial wild-type CYP153A6 enzyme allowed it to hydroxylate medium-chain alkanes to produce certain amounts of particular regiospecific alkane products\u2014i.e., 1-alkanols. This wild-type enzyme was then engineered, in vivo, to support bacterial growth on short-chain alkanes while maintaining its regioselectiveness for terminal hydroxylation. The mutant CYP153A6-BMO1 enzyme surprisingly exhibited an increased production of 1-alkanol when compared to the wild-type CYP153A6 enzyme.","Additional embodiments of the invention include mutant CYP153A6 enzymes with altered regioselectivity that are capable of supporting growth of an expression host on short-chain alkanes. For example, one mutant enzyme, CYP153A6-BMO1, was found to hydroxylate butane at the terminal position to form 1-butanol at a significantly greater rate than wild-type CYP153A6.","Additional embodiments include an isolated nucleic acid molecule of SEQ ID NO: 1 that encodes a polypeptide comprising the amino acid sequence of SEQ ID NO: 2, having an amino acid substitution corresponding to A97V.","A salient aspect of one or more embodiments of the preferred embodiments is a point mutation corresponding to an amino acid substitution of A97V in SEQ ID NO: 2. Experimental evidence suggests that this point mutation dramatically increases the ability of the CYP153A6 enzyme to hydroxylate small-chain alkanes at the terminal position. (See ).","One preferred embodiment of the present invention employs a contiguous span of amino acids, wherein the span is at least 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, or 99% identical to the amino acids of SEQ ID NO: 2, wherein the fragment has an amino acid substitution corresponding to A97V.","Another preferred embodiment employs a polypeptide fragment consisting essentially of SEQ ID NO: 2, wherein the fragment has an amino acid substitution corresponding to A97V.","Another embodiment includes an isolated polypeptide or fragment thereof having at least 90% identity, and preferably 95% identity, with the amino acid sequence of SEQ ID NO: 2 and having an amino acid substitution corresponding to A97V.","Another embodiment includes an isolated polypeptide comprising the amino acid sequence of SEQ ID NO: 3, having an amino acid substitution corresponding to A94V.","A salient aspect of one or more embodiments of the preferred embodiments is a point mutation corresponding to an amino acid substitution of A94V in SEQ ID NO: 3. Experimental evidence suggests that this point mutation dramatically increases the ability of the CYP153A6 enzyme to hydroxylate small-chain alkanes at the terminal position. (See ).","One preferred embodiment of the present invention employs a contiguous span of amino acids, wherein the span is at least 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, or 99% identical to the amino acids of SEQ ID NO: 3, wherein the fragment has an amino acid substitution corresponding to A94V.","Another preferred embodiment employs a polypeptide fragment consisting essentially of SEQ ID NO: 3, wherein the fragment has an amino acid substitution corresponding to A94V.","Another embodiment includes an isolated polypeptide or fragment thereof having at least 90% identity, and preferably 95% identity, with the amino acid sequence of SEQ ID NO: 3, and having a point mutation corresponding to an amino acid substitution of A94V.","Yet another embodiment includes a method of in vivo directed evolution to improve the ability of CYP153A6 enzyme to hydroxylate small-chain alkanes at the terminal position.","In another embodiment, an isolated nucleic acid encoding a CYP153A6 enzyme that has a higher capability than the corresponding wild-type CYP153A6 enzyme to oxidize at least one substrate at the terminal position selected from an alkane comprising carbon-chain of no more than five carbons is provided.","Yet another embodiment includes the method of using the above-described mutants for the selective hydroxylation of short-chain alkanes to produce well-characterized products in known quantities. As the mutant enzymes produce known products in a known amount, all that is required to create a desired product is to select an appropriate mutant CYP153A6 enzyme that catalyzes a reaction to produce a desired regiospecific product and then apply the substrate to the enzyme under conditions which allow for catalysis. Methods of selecting and isolating the desired product from the products created are also known and disclosed herein.","The invention also contemplates certain modifications to the sequences described above with codons that encode amino acids that are chemically equivalent to the amino acids in the native protein. An amino acid substitution involving the substitution of amino acid with a chemically equivalent amino acid is known as a conserved amino acid substitution.","Chemical equivalency can be determined by one or more the following characteristics: charge, size, hydrophobicity\/hydrophilicity, cyclic\/non-cyclic, aromatic\/non-aromatic etc. For example, a codon encoding a neutral non-polar amino acid can be substituted with another codon that encodes a neutral non-polar amino acid, with a reasonable expectation of producing a biologically equivalent protein. Amino acids can generally be classified into four groups. Acidic residues are hydrophillic and have a negative charge to loss of H+ at physiological pH. Basic residues are also hydrophillic but have a positive charge to association with H+ at physiological pH. Neutral nonpolar residues are hydrophobic and are not charged at physiological pH. Neutral polar residues are hydrophillic and are not charged at physiological pH. Amino acid residues can be further classified as cyclic or noncyclic and aromatic or nonaromatic, self-explanatory classifications with respect to side chain substituent groups of the residues, and as small or large. The residue is considered small if it contains a total of 4 carbon atoms or less, inclusive of the carboxyl carbon. Small residues are always non-aromatic. Of naturally occurring amino acids, aspartic acid and glutamic acid are acidic; arginine and lysine are basic and noncylclic; histidine is basic and cyclic; glycine, serine and cysteine are neutral, polar and small; alanine is neutral, nonpolar and small; threonine, asparagine and glutamine are neutral, polar, large and nonaromatic; tyrosine is neutral, polar, large and aromatic; valine, isoleucine, leucine and methionine are neutral, nonpolar, large and nonaromatic; and phenylalanine and tryptophan are neutral, nonpolar, large and aromatic. Proline, although technically neutral, nonpolar, large, cyclic and nonaromatic is a special case due to its known effects on secondary conformation of peptide chains, and is not, therefore included in this defined group.","There are also common amino acids which are not encoded by the genetic code included by example and not limitation: sarcosine, beta-alanine, 2,3 diamino propionic and alpha-aminisobutryric acid which are neutral, nonpolar and small; t-butylalanine, t-butylglycine, \u03b2-methylisoleucine, norleucine and cyclohexylalanine which are neutral, nonpolar, large and nonaromatic; ornithine which is basic and noncyclic; cysteic acid which is acidic; citrulline aceyl lysine and methionine sulfoxide which are neutral, polar, large and nonaromatic; and phenylglycine, 2-napthylalanine, \u03b2-thienylalanine and 1,2,3,4, tetrahydroisoquinoline-3 carboxylic acid which are neutral, nonpolar, large and aromatic. Other modifications are known in the art some of which are discussed in U.S. Pat. No. 6,465,237 issued to Tomlinson on Oct. 15, 2002, which is incorporated herein by reference.","Unless defined otherwise herein, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Singleton et. al. (2001) , third edition, John Wiley and Sons (New York); and, Hale and Marham (1991) , Harper Perennial, N.Y. provide one of skill with a general dictionary of many of the terms used in this invention. In practicing one or more embodiments of the present invention, several conventional techniques in molecular biology, proteomics, microbiology and recombinant DNA are used. Such techniques are well known and are explained in, for example, Sambrook, 1999, Molecular Cloning: A Laboratory Manual, Second Edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.; DNA Cloning: A practical Approach, 1985 (D. N. Glover ed.); Current Protocols in Molecular Biology, John Wiley & Sons, Inc. (1994); Proteins and Proteomics: A Laboratory Manual (Simpson ed.), Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. (2002), www.proeteinsandproteomics.org, and all more recent editions of these publications. Although any methods and materials similar or equivalent to those described herein can be used in the practice or testing of the present invention, the preferred methods and materials are described. For purposes of the present invention, the following terms are defined below.","Amino acids may be referred to herein by either their commonly known three letter symbols or by the one-letter symbols recommended by the IUPAC-IUB Biochemical Nomenclature Commission. Nucleotides, likewise, may be referred to by their commonly accepted single-letter codes.","As used herein, \u201cabout\u201d or \u201capproximately\u201d shall mean within 20 percent, preferably within 10 percent, and more preferably within 5 percent of a given value or range.","The term \u201cidentical\u201d in the context of two nucleic acid or polypeptide sequences refers to the residues in the two sequences which are the same when aligned for maximum correspondence over a specified comparison window. When percentage of sequence identity is used in reference to proteins or peptides it is recognized that residue positions which are not identical often differ by conservative amino acid substitutions, where amino acid residues are substituted for other amino acid residues with similar chemical properties (e.g. charge or hydrophobicity) and therefore do not change the functional properties of the molecule. Where sequences differ in conservative substitutions, the percent sequence identity may be adjusted upwards to correct for the conservative nature of the substitution. Means for making this adjustment are well-known to those of skill in the art. Typically this involves scoring a conservative substitution as a partial rather than a full mismatch, thereby increasing the percentage sequence identity. Thus, for example, where an identical amino acid is given a score of 1 and a non-conservative substitution is given a score of zero, a conservative substitution is given a score between zero and 1. The scoring of conservative substitutions is calculated, e.g., according to the algorithm of Meyers and Miller, Computer Applic. Biol. Sci., 4: 11-17 (1988) e.g., as implemented in the program PC\/GENE (Intelligenetics, Mountain View, Calif., USA).","The term \u201csubstrate\u201d means any substance or compound that is converted or meant to be converted into another compound by the action of an enzyme catalyst. The term includes aromatic and aliphatic compounds, and includes not only a single compound, but also combinations of compounds, such as solutions, mixtures and other materials which contain at least one substrate.","An \u201coxidation reaction\u201d or \u201coxygenation reaction\u201d, as used herein, is a chemical or biochemical reaction involving the addition of oxygen to a substrate, to form an oxygenated or oxidized substrate or product. An oxidation reaction is typically accompanied by a reduction reaction (hence the term \u201credox\u201d reaction, for oxidation and reduction). A compound is \u201coxidized\u201d when it receives oxygen or loses electrons. A compound is \u201creduced\u201d when it loses oxygen or gains electrons.","The term \u201cenzyme\u201d means any substance composed wholly or largely of protein or polypeptides that catalyzes or promotes, more or less specifically, one or more chemical or biochemical reactions.","A \u201cpolypeptide\u201d (one or more peptides) is a chain of chemical building blocks called amino acids that are linked together by chemical bonds called peptide bonds. A protein or polypeptide, including an enzyme, may be \u201cnative\u201d or \u201cwild-type\u201d, meaning that it occurs in nature or has the amino acid sequence of a native protein, respectively. These terms are sometimes used interchangeably. A polypeptide may or may not be glycosylated.","A \u201crecombinant wild-type\u201d typically means the wild type sequence in a recombinant host without glycosylation. Comparisons in the examples and figures of this application are generally with reference to a wild type that is a recombinant wild type.","A polypeptide may also be a \u201cmutant,\u201d \u201cvariant\u201d or \u201cmodified\u201d, meaning that it has been made, altered, derived, or is in some way different or changed from a native protein or its wild-type composition, or from another mutant. Mutant proteins typically have amino acid substitutions at one or more positions. Mutant DNA molecules typically have nucleotide substitutions in one or more positions. Mutant forms of a protein or DNA molecule can have the same, or altered, functions in comparison to the wild-type. For ease of discussion, mutants may be referred to by their variation from the single amino acid code from which the mutation arose. For example, in one format the mutant is referred to as XPOSY, where \u201cX\u201d refers to the single letter code of the amino acid in the original sequence, \u201cPOS\u201d refers to the position of the mutation in the sequence, and Y refers to the single letter code for the new amino acid appearing at the mutation's position. For example, V1751 would mean that in the original protein, the amino acid at position 175 is a valine (\u201cV\u201d), but in the mutant, the valine is replaced with an isoleucine (\u201cI\u201d).","A \u201cparent\u201d polypeptide or enzyme is any polypeptide or enzyme from which any other polypeptide or enzyme is derived or made, using any methods, tools or techniques, and whether or not the parent is itself a native or mutant polypeptide or enzyme. A parent polynucleotide is one that encodes a parent polypeptide.","As used herein, a \u201ccore mutation\u201d is a mutation of a wild-type CYP153A6 enzyme that provides the protein with enhanced alkane hydroxylase activity. It should be realized that any mutation, or set of mutations, that enhance the ability of a CYP153A6 protein to hydroxylate alkanes are considered core mutations.","A \u201ccore mutant\u201d is a CYP153A6 protein that has been altered to contain one or more core mutations. In one embodiment, a core mutant is the cytochrome CYP153A6-BMO1 protein which was derived from mutations of CYP153A6, and includes a A97V core mutation. In one embodiment, those mutations that revert the amino acid sequence back to the wild type sequence for the selective hydroxylation mutations are not considered core mutations.","As used herein, the terms \u201cselective hydroxylation mutations\u201d or \u201cselective mutations\u201d are used interchangeably and refer to mutations that provide a protein with altered regio-selectivity towards specific substrates. A protein having such mutations is termed a \u201cselective hydroxylation mutant\u201d or a \u201cselective mutant\u201d. In one embodiment, the target substrate of such mutants is a small-chain alkane. The selective hydroxylation mutations may simply alter the selectivity of the protein towards a single substrate, or across many substrates. The selective mutation may alter both the selectivity and increase the functional ability of the enzyme, so that more regio-selective end product is produced.","Non-limiting general examples of selective hydroxylation mutants showing altered or enhanced regioselective hydroxylation include CYP153A6-BMO1 proteins having one or more of the following additional mutations: A97V in SEQ ID NO: 2 and A94 V in SEQ ID NO: 3.","In some embodiments, more than a single mutation may be required in order for the desired result to occur, in such situations, each of the required mutations will be considered as either core, selective, or both, as appropriate.","An enzyme is \u201cregioselective\u201d if the product that results from the enzymatic reaction is positioned in an altered or specified position. In one embodiment, the enzyme is an alkane hydroxylase and the hydroxylation reaction results in a hydroxyl group positioned at the terminal position. This means that while the original enzyme may have created a first amount of product A and a second amount of product B, the regioselective enzyme could produce a third amount of product A and a fourth amount of product B. Thus, while the initial wild-type parent could be considered regioselective for particular substrates, the regioselective mutants described herein display different regioselectivity from the wild-type parent enzyme. In one embodiment, a distribution of hydroxyl groups in the final product that differs from the product of the wild-type enzyme is sufficient to demonstrate that the enzyme is regioselective. In another embodiment, an increase of 1, 1-2, 2-5, 5-10, 10-20, 20-30, 30-40, 40-50, 50-60, 60-70, 70-80, 80-90, 90-100, 100-200, 200-500 percent or more in the concentration of one product over another product is sufficient to demonstrate that the enzyme is regioselective. In one embodiment, an enzyme is regioselective when its selectivity is greater than the wild-type enzyme as shown in Table 2 and .","A \u201cconsistent\u201d selective mutant is a selective mutant that displays a consistent bias of selectivity of product produced for more than one starting substrate. Thus, for example, the mutant CYP153A6-BMO1, discussed below, is a consistent regioselective mutant for propane, butane, pentane and octane, as the products from propane, butane, pentane and octane all result predominantly in the 1-alcohol. In one embodiment, a mutant CYP153A6 is a consistent regioselective enzyme if the largest amount of product produced from propane, butane, pentane and octane is the 1-alcohol. In one embodiment, the majority of each of the products is made at the same position.","An alkane is typically defined as a non-aromatic saturated hydrocarbon with the sequence of CH(2+2).","A \u201cshort-chain alkane\u201d is defined as any alkane having less than six carbon atoms, e.g., methane, ethane, propane, butane or pentane.","The proteins of the present invention further include \u201cconservative amino acid substitution variants\u201d (i.e., conservative) of the proteins herein described. As used herein, a conservative variant refers to at least one alteration in the amino acid sequence that does not adversely affect the biological functions of the protein. A substitution, insertion or deletion is said to adversely affect the protein when the altered sequence prevents or disrupts a biological function associated with the protein. For example, the overall charge, structure or hydrophobic-hydrophilic properties of the protein can be altered without adversely affecting a biological activity. Accordingly, the amino acid sequence can often be altered, for example to render the peptide more hydrophobic or hydrophilic, without adversely affecting the biological activities of the protein.","The \u201cactivity\u201d of an enzyme is a measure of its ability to catalyze a reaction, and may be expressed as the rate at which the product of the reaction is produced. For example, enzyme activity can be represented as the amount of product produced per unit of time, per unit (e.g. concentration or weight) of enzyme.","The \u201cstability\u201d of an enzyme means its ability to function, over time, in a particular environment or under particular conditions. One way to evaluate stability is to assess its ability to resist a loss of activity over time, under given conditions. Enzyme stability can also be evaluated in other ways, for example, by determining the relative degree to which the enzyme is in a folded or unfolded state. Thus, one enzyme is more stable than another, or has improved stability, when it is more resistant than the other enzyme to a loss of activity under the same conditions, is more resistant to unfolding, or is more durable by any suitable measure.","The term \u201chost\u201d or \u201chost cell\u201d means any cell of any organism that is selected, modified, transformed, grown, or used or manipulated in any way, for the production of a substance by the cell, for example the expression by the cell of a gene, a DNA or RNA sequence, a protein or an enzyme.","\u201cDNA\u201d (deoxyribonucleic acid) means any chain or sequence of the chemical building blocks adenine (A), guanine (G), cytosine (C) and thymine (T), called nucleotide bases, that are linked together on a deoxyribose sugar backbone. DNA can have one strand of nucleotide bases, or two complimentary strands which may form a double helix structure. \u201cRNA\u201d (ribonucleic acid) means any chain or sequence of the chemical building blocks adenine (A), guanine (G), cytosine (C) and uracil (U), called nucleotide bases, that are linked together on a ribose sugar backbone. RNA typically has one strand of nucleotide bases.","A \u201cpolynucleotide\u201d or \u201cnucleotide sequence\u201d is a series of nucleotide bases (also called \u201cnucleotides\u201d) in DNA and RNA, and means any chain of two or more nucleotides. A nucleotide sequence typically carries genetic information, including the information used by cellular machinery to make proteins and enzymes. These terms include double or single stranded genomic and cDNA, RNA, any synthetic and genetically manipulated polynucleotide, and both sense and anti-sense polynucleotide (although only sense stands are being represented herein). This includes single- and double-stranded molecules, i.e. DNA-DNA, DNA-RNA and RNA-RNA hybrids, as well as \u201cprotein nucleic acids\u201d (PNA) formed by conjugating bases to an amino acid backbone. This also includes nucleic acids containing modified bases, for example thio-uracil, thio-guanine and fluoro-uracil.","Proteins and enzymes are made in the host cell using instructions in DNA and RNA, according to the genetic code. Generally, a DNA sequence having instructions for a particular protein or enzyme is \u201ctranscribed\u201d into a corresponding sequence of RNA. The RNA sequence in turn is \u201ctranslated\u201d into the sequence of amino acids which form the protein or enzyme.","An \u201camino acid sequence\u201d is any chain of two or more amino acids. Each amino acid is represented in DNA or RNA by one or more triplets of nucleotides. Each triplet forms a codon, corresponding to an amino acid. For example, the amino acid lysine (Lys) can be coded by the nucleotide triplet or codon AAA or by the codon AAG. (The genetic code has some redundancy, also called degeneracy, meaning that most amino acids have more than one corresponding codon.) Because the nucleotides in DNA and RNA sequences are read in groups of three for protein production, it is important to begin reading the sequence at the correct amino acid, so that the correct triplets are read. The way that a nucleotide sequence is grouped into codons is called the \u201creading frame.\u201d","The term \u201cgene\u201d, also called a \u201cstructural gene\u201d means a DNA sequence that codes for or corresponds to a particular sequence of amino acids which comprise all or part of one or more proteins or enzymes, and may or may not include regulatory DNA sequences, such as promoter sequences, which determine for example the conditions under which the gene is expressed. Some genes, which are not structural genes, may be transcribed from DNA to RNA, but are not translated into an amino acid sequence. Other genes may function as regulators of structural genes or as regulators of DNA transcription.","A \u201ccoding sequence\u201d or a sequence \u201cencoding\u201d a polypeptide, protein or enzyme is a nucleotide sequence that, when expressed, results in the production of that polypeptide, protein or enzyme, i.e., the nucleotide sequence encodes an amino acid sequence for that polypeptide, protein or enzyme.","The terms \u201cexpress\u201d and \u201cexpression\u201d mean allowing or causing the information in a gene or DNA sequence to become manifest, for example producing a protein by activating the cellular functions involved in transcription and translation of a corresponding gene or DNA sequence. A DNA sequence is expressed in or by a cell to form an \u201cexpression product\u201d such as a protein. The expression product itself, e.g. the resulting protein, may also be said to be \u201cexpressed\u201d by the cell. A polynucleotide or polypeptide is expressed recombinantly, for example, when it is expressed or produced in a foreign host cell under the control of a foreign or native promoter, or in a native host cell under the control of a foreign promoter.","The term \u201ctransformation\u201d means the introduction of a \u201cforeign\u201d (i.e. extrinsic or extracellular) gene, DNA or RNA sequence to a host cell, so that the host cell will express the introduced gene or sequence to produce a desired substance, typically a protein or enzyme coded by the introduced gene or sequence. The introduced gene or sequence may also be called a \u201ccloned\u201d or \u201cforeign\u201d gene or sequence, may include regulatory or control sequences, such as start, stop, promoter, signal, secretion, or other sequences used by a cell's genetic machinery. The gene or sequence may include nonfunctional sequences or sequences with no known function. A host cell that receives and expresses introduced DNA or RNA has been \u201ctransformed\u201d and is a \u201ctransformant\u201d or a \u201cclone.\u201d The DNA or RNA introduced to a host cell can come from any source, including cells of the same genus or species as the host cell, or cells of a different genus or species.","The terms \u201cvector\u201d, \u201ccloning vector\u201d and \u201cexpression vector\u201d mean the vehicle by which a DNA or RNA sequence (e.g. a foreign gene) can be introduced into a host cell, so as to transform the host and promote expression (e.g. transcription and translation) of the introduced sequence.","Vectors typically comprise the DNA of a transmissible agent, into which foreign DNA is inserted. A common way to insert one segment of DNA into another segment of DNA involves the use of enzymes called restriction enzymes that cleave DNA at specific sites (specific groups of nucleotides) called restriction sites. Generally, foreign DNA is inserted at one or more restriction sites of the vector DNA, and then is carried by the vector into a host cell along with the transmissible vector DNA. A segment or sequence of DNA having inserted or added DNA, such as an expression vector, can also be called a \u201cDNA construct.\u201d","A common type of vector is a \u201cplasmid\u201d, which generally is a self-contained molecule of double-stranded DNA, that can readily accept additional (foreign) DNA and which can readily introduced into a suitable host cell. A plasmid vector often contains coding DNA and promoter DNA and has one or more restriction sites suitable for inserting foreign DNA. Promoter DNA and coding DNA may be from the same gene or from different genes, and may be from the same or different organisms. A large number of vectors, including plasmid and fungal vectors, have been described for replication and\/or expression in a variety of eukaryotic and prokaryotic hosts. Non-limiting examples include pKK plasmids (Clonetech), pUC plasmids, pET plasmids (Novagen, Inc., Madison, Wis.), pRSET or pREP plasmids (Invitrogen, San Diego, Calif.), or pMAL plasmids (New England Biolabs, Beverly, Mass.), and many appropriate host cells, using methods disclosed or cited herein or otherwise known to those skilled in the relevant art. Recombinant cloning vectors will often include one or more replication systems for cloning or expression, one or more markers for selection in the host, e.g. antibiotic resistance, and one or more expression cassettes. Routine experimentation in biotechnology can be used to determine which vectors are best suited for used with the invention. In general, the choice of vector depends on the size of the polynucleotide sequence and the host cell to be employed in the methods of this invention.","The term \u201cexpression system\u201d means a host cell and compatible vector under suitable conditions, e.g. for the expression of a protein coded for by foreign DNA carried by the vector and introduced to the host cell. Common expression systems include bacteria (e.g. and ) or yeast (e.g. ) host cells and plasmid vectors, and insect host cells and Baculovirus vectors.","The term \u201cin vivo mutagenesis\u201d refers to a process of generating random mutations in any cloned DNA of interest which involves the propagation of the DNA in a strain of that carries mutations in one or more of the DNA repair pathways. These \u201cmutator\u201d strains have a higher random mutation rate than that of a wild-type parent. Propagating the DNA in one of these strains will eventually generate random mutations within the DNA.","\u201cIsolation\u201d or \u201cpurification\u201d of a polypeptide or enzyme refers to the derivation of the polypeptide by removing it from its original environment (for example, from its natural environment if it is naturally occurring, or form the host cell if it is produced by recombinant DNA methods). Methods for polypeptide purification are well-known in the art, including, without limitation, preparative disc-gel electrophoresis, isoelectric focusing, HPLC, reversed-phase HPLC, gel filtration, ion exchange and partition chromatography, and countercurrent distribution. For some purposes, it is preferable to produce the polypeptide in a recombinant system in which the protein contains an additional sequence tag that facilitates purification, such as, but not limited to, a polyhistidine sequence. The polypeptide can then be purified from a crude lysate of the host cell by chromatography on an appropriate solid-phase matrix. Alternatively, antibodies produced against the protein or against peptides derived therefrom can be used as purification reagents. Other purification methods are possible. A purified polynucleotide or polypeptide may contain less than about 50%, preferably less than about 75%, and most preferably less than about 90%, of the cellular components with which it was originally associated. A \u201csubstantially pure\u201d enzyme indicates the highest degree of purity which can be achieved using conventional purification techniques known in the art.","A \u201chydroxylation profile\u201d of a product is a description of the number and position of hydroxyl groups in the product. Thus, for example, an alkane hydroxylase enzyme typically creates products having a defined hydroxylation profile, such that hydroxyl groups are placed at certain positions on particular percentages of the final reaction products. Altering or modifying the hydroxylation profile of a product means changing the positions, or proportions, of hydroxyl groups in the final reaction products. In another example, all of the products listed in Table 2 are used for the members of hydroxylation profile. For example, 1-alcohol and 2-alcohols may make up the hydroxylation profile. Thus, Table 2 denotes the hydroxylation profiles of each of the mutants for substrates propane, butane, pentane, and octane, in this embodiment. A \u201cvariant\u201d is distinguished from a mutant.","The general genetic engineering tools and techniques discussed here, including transformation and expression, the use of host cells, vectors, expression systems, etc., are well known in the art.","As used herein, a nucleic acid molecule is said to be \u201cisolated\u201d when the nucleic acid molecule is substantially separated from contaminant nucleic acid encoding other polypeptides from the source of nucleic acid.","Embodiments of the present invention further include fragments of any one of the encoding nucleic acids molecules. As used herein, a fragment of an encoding nucleic acid molecule refers to a small portion of the entire protein coding sequence. The size of the fragment will be determined by the intended use. For example, if the fragment is chosen so as to encode an active portion of the protein, the fragment will need to be large enough to encode the functional region(s) of the protein. For instance, fragments of the invention include fragments of DNA encoding mutant CYP153A6 proteins that maintain altered or enhanced regioselectivity for small-chain alkanes.","The determination of percent identity or homology between two sequences is accomplished by BLAST (Basic Local Alignment Search Tool) analysis using the algorithm of Karlin and Altschul, 199087, 2264-2268, modified as in Karlin and Altschul, 199390, 5873-5877. Such an algorithm is incorporated into the NBLAST and XBLAST programs of Altschul, et al., 1990215, 403-410. BLAST nucleotide searches are performed with the NBLAST program, score=100, wordlength=12 to obtain nucleotide sequences homologous to the nucleic acid molecules of the invention. BLAST protein searches are performed with the XBLAST program, score=50, wordlength=3 to obtain amino acid sequences homologous to the protein molecules of the invention. To obtain gapped alignments for comparison purposes, Gapped BLAST is utilized as described in Altschul, et al., 199725, 3389-3402. When utilizing BLAST and Gapped BLAST programs, the default parameters of the respective programs (e.g., XBLAST and NBLAST) are used. See http:\/\/www.ncbi.nlm.nih.gov.","The encoding nucleic acid molecules of the present invention may further be modified so as to contain a detectable label for diagnostic and probe purposes. A variety of such labels is known in the art and can readily be employed with the encoding molecules herein described. Suitable labels include, but are not limited to, fluorescent-labeled, biotin-labeled, radio-labeled nucleotides and the like. A skilled artisan can employ any of the art known labels to obtain a labeled encoding nucleic acid molecule.","One technique to improve the alkane-oxidation capability of wild-type CYP153A6 enzymes is directed evolution. General methods for generating libraries and isolating and identifying improved proteins according to one or more embodiments of the present invention using directed evolution are described briefly below. More extensive descriptions can be found in, for example, Arnold, F., 31(3): 125-131 (1998); U.S. Pat. No. 5,741,691; U.S. Pat. No. 5,811,238; U.S. Pat. No. 5,605,793; U.S. Pat. No. 5,830,721; and, International Applications WO 98\/42832, WO 95\/22625, WO 97\/20078, WO 95\/41653 and WO 98\/27230, which are incorporated by reference herein.","The basic steps in directed evolution generally include: (1) the generation of mutant libraries of polynucleotides from a parent or wild-type sequence; (2) (optionally) expression of the mutant polynucleotides to create a mutant polypeptide library; (3) screening\/selecting the polynucleotide or polypeptide library for a desired property of a polynucleotide or polypeptide; and (4) selecting mutants which possess a higher level of the desired property; and (5) repeating steps (1) to (5) using the selected mutant(s) as parent(s) until one or more mutants displaying a sufficient level of the desired activity have been obtained. The property can be, but is not limited to, alkane oxidation capability and regiospecificity.","The parent protein or enzyme to be evolved can be a wild-type protein or enzyme, or a variant or mutant. The parent polynucleotide can be retrieved from any suitable commercial or non-commercial source. The parent polynucleotide can correspond to a full-length gene or a partial gene, and may be of various lengths. Preferably the parent polynucleotide is from 50 to 50,000 base pairs. It is contemplated that entire vectors containing the nucleic acid encoding the parent protein of interest may be used in the methods of this invention.","Any method can be used for generating mutations in the parent polynucleotide sequence to provide a library of evolved polynucleotides, including error-prone polymerase chain reaction, cassette mutagenesis (in which the specific region optimized is replaced with a synthetically mutagenized oligonucleotide), oligonucleotide-directed mutagenesis, parallel PCR (which uses a large number of different PCR reactions that occur in parallel in the same vessel, such that the product of one reaction primes the product of another reaction), random mutagenesis (e.g., by random fragmentation and reassembly of the fragments by mutual priming); site-specific mutations (introduced into long sequences by random fragmentation of the template followed by reassembly of the fragments in the presence of mutagenic oligonucleotides); parallel PCR (e.g., recombination on a pool of DNA sequences); sexual PCR; and chemical mutagenesis (e.g., by sodium bisulfite, nitrous acid, hydroxylamine, hydrazine, formic acid, or by adding nitrosoguanidine, 5-bromouracil, 2-aminopurine, and acridine to the PCR reaction in place of the nucleotide precursor; or by adding intercalating agents such as proflavine, acriflavine, quinacrine); irradiation (X-rays or ultraviolet light, and\/or subjecting the polynucleotide to propagation in a host cell that is deficient in normal DNA damage repair function); or DNA shuffling (e.g., in vitro or in vivo homologous recombination of pools of nucleic acid fragments or polynucleotides). Any one of these techniques can also be employed under low-fidelity polymerization conditions to introduce a low level of point mutations randomly over a long sequence, or to mutagenize a mixture of fragments of unknown sequence.","Once the evolved polynucleotide molecules are generated they can be cloned into a suitable vector selected by the skilled artisan according to methods well known in the art. If a mixed population of the specific nucleic acid sequence is cloned into a vector it can be clonally amplified by inserting each vector into a host cell and allowing the host cell to amplify the vector and\/or express the mutant or variant protein or enzyme sequence. Any one of the well-known procedures for inserting expression vectors into a cell for expression of a given peptide or protein may be utilized. Suitable vectors include plasmids and viruses, particularly those known to be compatible with host cells that express oxidation enzymes or oxygenases. is one exemplary preferred host cell. Other exemplary cells include other bacterial cells such as and , archaebacteria, yeast cells such as , insect cells and filamentous fungi such as any species of cells. For some applications, plant, human, mammalian or other animal cells may be preferred. Suitable host cells may be transformed, transfected or infected as appropriate by any suitable method including electroporation, CaClmediated DNA uptake, fungal infection, microinjection, microprojectile transformation, viral infection, or other established methods.","The mixed population of polynucleotides or proteins may then be tested or screened to identify the recombinant polynucleotide or protein having a higher level of the desired activity or property. The mutation\/screening steps can then be repeated until the selected mutant(s) display a sufficient level of the desired activity or property. Briefly, after the sufficient level has been achieved, each selected protein or enzyme can be readily isolated and purified from the expression system, or media, if secreted. It can then be subjected to assays designed to further test functional activity of the particular protein or enzyme. Such experiments for various proteins are well known in the art, and are described below and in the Examples below.","The evolved enzymes can be used in biocatalytic processes for, e.g., alkane hydroxylation. The enzyme mutants can be used in biocatalytic processes for production of chemicals from hydrocarbons. Furthermore, the enzyme mutants can be used in live cells or in dead cells, or it can be partially purified from the cells.","In order to determine whether the strain GPo12 (pGEc47\u0394B) could be used to select for improved terminal alkane hydroxylation activity, its ability to grow on primary and secondary Cto Calkanols was measured. It was previously shown that strain GPo12 (pGEc47\u0394B) would grow on medium-chain length alkanols and on the corresponding alkane only when complemented by a terminal alkane hydroxylase. (Smits, T. H., et. al., 2002184, 1733-1742). For these growth tests, cells from a Luria-Bertani broth preculture were washed three times with M9 medium and used to inoculate the 5-ml M9 main cultures in 14-ml tubes to an optical density (OD) of 0.1, then grown at 30\u00b0 C. with continuous shaking.","No growth was observed on any of the secondary alcohols or methanol during an 18 day period, but growth on ethanol, 1-propanol and 1-butanol was comparable to the growth exhibited on the positive control grown on glucose, as depicted in . These results indicate that the terminal hydroxylation products of all short-chain n-alkanes except for methane are readily utilized as carbon sources, while subterminal oxidation products are not. Thus, it was determined that strain GPo12 (pGEc47\u0394B) would be suitable for growth-based screening and selection for terminal hydroxylation of alkanes having various lengths.","The recombinant host GPo12 (pGEc47\u0394B) was engineered specifically for complementation studies with terminal alkane hydroxylases and was previously used to characterize members of the AlkB and CYP153 families. (Smits, T. H., et. al., 2002184, 1733-1742; van Beilen, J. B., et. al., 200672, 59-65). This strain is a derivative of the natural isolate GPo1 lacking its endogenous OCT plasmid, but containing cosmid pGEc47\u0394B, which carries all genes comprising the alk machinery necessary for alkane utilization, with the exception of the deleted AlkB gene. This recombinant host was complemented by a plasmid-encoded library of alkane hydroxylases and grown on various alkanes, resulting in an enrichment of novel, specific alkane-oxidizing terminal hydroxylases. Specifically, plasmid pCom8_cyp153A6 generated in Smits, T. H., et. al., 2001, Plasmid, 46, 16-24, were used with permission. Sequencing of pCom8_cyp153A6 revealed that it was missing the first nine nucleotides of the coding sequence of the monooxygenase gene compared to the published sequence. (Funhoff, E. G., et. al., 2006188, 5220-5227). Thus, the CYP153A6 parent enzyme (amino acid SEQ ID NO: 2) and its variant are three N-terminal amino acids shorter than the published sequence, but otherwise structurally identical and functional.","It was not efficient to use error-prone PCR to randomly mutate the target genes, as cloning of PCR products into the pCom vector resulted in fewer than 2,000 generated transformants. Therefore, mutant libraries were constructed by complementing GPo12 (pGEc47\u0394B) strains with randomly mutated plasmids encoding CYP153A6. Mutator strains XL1-Red (Stratagene) and JS200 pEP polymerase I were used to generate plasmid libraries according to the manufacturer's manual and Camps, M. and Loeb, L. A., 2003(F. H. Arnold and G. Georgiou, ed.), pp. 11-18, respectively. XL1-Red has deficiencies in the DNA repair mechanism that lead to a 5,000-fold increase in the general mutation rate. JS200 pEP Pol I expresses an engineered mutator DNA polymerase I which mainly amplifies plasmid DNA, with lower reliability, thus introducing mutations in the plasmid DNA. The nucleotide mutation level in XL1-Red after 2 weeks of continuous culturing was approximately 0.1\/kb, while four rounds of mutation in JS200 pEP Pol I yielded up to 0.4\/kb.","Cultures of both mutator strains were combined and the mutated plasmids were transformed into GPo12 (pGEc47\u0394B) through triparental mating with the mediator strain CC118 (pRK600) and the appropriate DH5a donor. The growth selection was performed by culturing the resulting strain library in minimal medium with a specific alkane as the sole carbon source for up to 3 weeks. For more details and further discussion of the materials and methods used in the generation of the libraries, see Koch, Daniel J., et al., 200975, 337-344, which is incorporated in its entirety by reference herein.","The host, vector and operons of the best mutants were analyzed individually by comparing them in growth tests to their wild-type counterparts. For example, adapted vectors were isolated from the strains and the CYP153A6 gene fdrA6 and fdxA6 operons were replaced with the wild-type sequence by cloning before being mated into the wild-type host. Additionally, potentially improved hydroxylase genes were recloned into a wild-type vector and transferred into a wild-type host.","A. CYP153A6 Variants","For GPo12 (pGEc47\u0394B) complemented with CYP153A6 genes, it had been observed that the host had to be adapted through prolonged cultivation on alkanes to obtain significant growth on alkanes, without any mutations occurring on the CYP153 genes themselves. Therefore, 21 colonies from the first round of enrichment cultures were obtained to compare to the parent strain in plate growth tests. Solid medium growth tests were chosen due to the instability of liquid minimal medium cultures of GPo12 (pGEc47\u0394B). Two strains adapted from GPo12 (pGEc47\u0394B), Pcyp1 and Pcyp2, demonstrated faster growth than their respective parents on pentane. For the CYP153A6 system, adapted plasmid pCom8* enabled faster growth of GPo12 (pGEc47\u0394B) on pentane, even when it contained the wild-type operon.","In addition to creating these adapted hosts and plasmid, the first round of in-vivo directed evolution generated enzyme variant CYP153A6-BMO1, which included a GCA to GTA change, resulting in the substitution A94V in CYP153A6-BMO1. (This corresponds to an A97V substitution in one published sequence as the plasmid pCom8_CYP153A6 was missing its first nine nucleotides when compared to the published sequence.) The CYP153A6-BMO1 variant supported growth on butane, while the wild-type CYP153A6 variant does not, as depicted in Table 1. Thus, the A94V variant enhances activity on smaller alkanes.",{"@attributes":{"id":"p-0097","num":"0096"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"140pt","align":"center"}}],"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["Complementing","Days required for growth to full lawn"]},{"entry":["alkane","with selected carbon source"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["monooxygnease","Ethane","Propane","Butane","Pentane","Octane"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["CYP153A6 wild-type","NG","NG","NG","2","2"]},{"entry":["CYP153A6-BMO1","NG","NG","5","1.5","5"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}},{"entry":"*NG = no growth detected during 3-week observation"}]}}]}}},"While plasmid pCom8*_cyp153A6-BMO1 was subjected to a second round of mutagenesis and mated into Pcyp1, there was no additional improvement in the CYP153A6 system from CYP153A6-BMO1.","A schematic representation of the directed evolution process and resulting variant strains for CYP153A6 is depicted in .","In order to quantify the effects of the mutations on enzyme performance, whole-cell bioconversions with the wild-type and mutant CYP153A6 enzymes were performed using growth-arrested BL21(DE3) cells.","Whole-cell bioconversions with CYP153A6 enzymes were performed using growth-arrested BL21(DE3) cells containing the CYP153A6 variants expressed from pCom8*. The typical concentration of folded CYP153A6 in the cell suspensions used for bioconversion was 0.1 to 0.2 \u03bcM, with CYP153A6-BMO1 expressing about 20% less than its parent. Neither enzyme exhibited a significant decrease in apparent P450 concentration after the 60 minute bioconversion reactions. The bioconversion study demonstrated significant altered activity and selectivity for CYP153A6-BMO1 that closely followed its growth complementation performance. Conversion of butane yielded an average total concentration of 393 uM 1-butanol in the aqueous phase after 60 minutes with CYP153A6-BMO1 versus 277 uM wild-type CYP153A6, as depicted in Table 2.",{"@attributes":{"id":"p-0102","num":"0101"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"147pt","align":"center"}}],"thead":{"row":{"entry":[{},"TABLE 2"]}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},{},"Total hydroxylated product (\u03bcM)**"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"105pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Substrate*","CYP153A6","CYP153A6-BMO1"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"Propane","421 (59)","244 (68)"]},{"entry":[{},"Butane","356 (22)","439 (11)"]},{"entry":[{},"Pentane","260 (62)","374 (49)"]},{"entry":[{},"Octane","78 (3)","\u200269 (46)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"*Bioconversions with propane and butane were performed in a bioreactor for 60 minutes with CYP153A6 variants. Bioconversion mixtures with pentane and octane were shaken in a glass vial for 60 minutes for all enzymes."]},{"entry":[{},"**Values in parentheses are the percentage of 2-alkanol in the total hydroxylated product. Only 1- and 2- alkanols were formed in detectable amounts."]}]}}]}}},"The average turnover rate of CYP153A6-BMO1 in 1-butanol production increased approximately 75% from the wild-type, from 28 minto 49 min. Surprisingly, the selectivity for terminal hydroxylation also increased for CYP153A6-BMO1 when compared to wild-type CYP153A6, from 78% to 89% of the total alkanols product, as shown in .","The A97V mutation in CYP153A6-BMO1 also improved activity and selectivity for conversion of pentane to 1-pentanol, but had the opposite effect with propane and octane.","CYP153A6-BMO1 is the first CYP153 enzyme shown to hydroxylate butane and enable its host to grow on this short-chain length alkane. Short-chain, gaseous alkanes are typically processed by specialized enzymes that are unrelated to the cytochrome P450s. Additionally, this is the first time that biotransformations using CYP153 demonstrated activity on alkanes smaller than hexane. Consequently, CYP153A6-BMO1 was converted into a genuine butane monooxygenases by in-vivo directed evolution. In addition, the A97V mutation stabilized CYP153A6 and nearly doubled its half-life at 45\u00b0 C.","One or more embodiments of the present invention are illustrated in the following examples, which are provided by way of illustration and are not intended to be limiting.","Directed evolution of CYP153A6 was performed in vivo through the mutagenesis of the pCom8 plasmid in XL1-Red (Stratagene) according to the manufacturer's manual and in JS200(pEP Pol I) as described in Camps, M. and Loeb, L. A., 2003(F. H. Arnold and G. Georgiou, ed.), pp. 11-18. Mutated CYP153A6 gene fdrA6 and fdxA6 operons were cloned into the original pCom8 plasmid as Kpn1-digested fragments. Restriction enzymes were obtained from Roche Molecular or New England Biolabs.","Cultures of both mutator strains were combined and the mutated plasmids were transformed into GPo12(pGEc47\u0394B) through triparental mating with the helper strain CC118 (pRK600) and the appropriate DH5a donor. Growth selection was performed by culturing the resulting strain library in minimal medium with an alkane as the sole carbon source for up to 3 weeks. More specifically, libraries of GPo12(pGEc47\u0394B) were precultured on E2 minimal medium plates with antibiotics and 0.2% (wt\/vol) citrate as the carbon source. The libraries were then enriched for improved strains through continuous growth in liquid E2 minimal medium with small-chain-length alkanes as the sole carbon source. The liquid minimal medium cultures growing on alkanes were shaken in custom-made gas-tight flasks with 1% liquid alkane (pentane and octane) in a reservoir or in gas-tight serum bottles (Alltech).","Growth of the resulting adapted strains on minimal medium plates with specific alkanes as the sole carbon source was measured after 3 weeks. Solid minimal medium growth tests on gaseous alkanes (Sigma-Aldrich) were conducted in gas-tight plastic containers (GasPak 150 large anaerobic vented system; VWR), pressurized at 20 lb\/infor 20 seconds (ethane and propane) or 10 lb\/infor 6 seconds (butane).","This example demonstrates one method by which evolution of the wild-type CYP153A6 enzyme may occur. A mutant, CYP153A6-BMO1, was obtained after enrichment and screening, as described above.  displays the regiospecific qualities of the CYP153A6-BMO1 mutant, which produced 1-alkanols at a greater rate than wild-type CYP153A6\u201489% of total alkanol product compared to 78%, respectively. As can be seen by comparing CYP153A6-BMO1 to wild-type CYP153A6 in Table 2 and , there is an increase in production of butanol in the variant. For example, CYP153A6-BMO1 showed a 75% higher relative activity towards 1-butanol production than wild-type CYP153A6. In terms of absolute product concentration, CYP153A6-BMO1 produced 40% more 1-butanol than wild-type CYP153A6 (390 \u03bcM versus 278 \u03bcM, respectively).","Sequencing of mutant CYP153A6-BMO1 revealed that it had a GCA to GTA nucleotide mutation in resulting in the amino acid substitutions A97V of SEQ ID NO: 2. This corresponds to a A94V amino acid substitution in SEQ ID NO: 3.",{"@attributes":{"id":"p-0112","num":"0111"},"i":"E. coli ","sub":"600"},"Pentane bioconversion samples were then left open at 60\u00b0 C. for 1 hour to allow the substrate to evaporate and subsequently pelleted, filtered, and subjected to gas chromatography analysis.","Octane bioconversion products were extracted by adding 250 \u03bcl hexane, vortexing for 30 seconds, and centrifuging at 14,000 rpm for 10 minutes. Alkanols were then detected in the top organic layer.","For bioconversion of gaseous alkanes, 80 ml of cell suspension and 15 \u03bcl of antifoam (Sigma-Aldrich) were stirred in a 100-ml bioreactor (Ochs-labor) at 25\u00b0 C. Propane or butane were mixed in a 1:1 ratio with air, and the mixture was fed to the cells at an inlet gas flow rate of approximately 10 liter\/hour. The reaction was started by addition of 20 mM glucose and stopped after 60 minutes.","The results of the bioconversion experiments using CYP153A6 variants are detailed in Table 2 and .","CO-Binding Activity of CYP153A6-BMO1","Although pCom plasmids are not efficient expression platforms, CYP153A6 expression was observed in , as indicated by the CO difference spectral peak at 450 nm in . In contrast, no 450 nm signal was observed in an empty pCom8* plasmid. Cytochrome P450s are typically difficult to express as it has been reported that the CO-binding activity of CYP153A6 is lost shortly after cell disruption at room temperature, even when the enzyme is isolated from its native host. However, CYP153A6 expressed well in DH5a and showed a stable CO difference spectrum for hours at room temperature if protease inhibitor was added before cell disruption. The need of a protease inhibitor was eliminated when BL21 (DE3) was used to express CYP153A6.",{"@attributes":{"id":"p-0119","num":"0118"},"i":"E. coli ","figref":"FIG. 5"},"It is to be understood that the above-described arrangements are only illustrative of the application of the principles of the present invention and it is not intended to be exhaustive or limit the invention to the precise form disclosed. Numerous modifications and alternative arrangements may be devised by those skilled in the art in light of the above teachings without departing from the spirit and scope of the present invention. It is intended that the scope of the invention be defined by the claims appended hereto.","In addition, the previously described versions of the present invention have many advantages, including but not limited to those described above. However, the invention does not require that all advantages and aspects be incorporated into every embodiment of the present invention.","All references cited herein, including but not limited to patents, patent applications, papers, text books, other publications, and the references cited therein, to the extent that they are not already, are hereby incorporated by reference in their entirety for all purposes."],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":["These and other features, aspects, and advantages of the multiple embodiments of the present invention will become better understood with reference to the following description, appended claims, and accompanied drawings where:",{"@attributes":{"id":"p-0011","num":"0010"},"figref":"FIG. 1","i":"P. putida "},{"@attributes":{"id":"p-0012","num":"0011"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0013","num":"0012"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0014","num":"0013"},"figref":"FIG. 4","i":"E. coli "},{"@attributes":{"id":"p-0015","num":"0014"},"figref":"FIG. 5"}]},"DETDESC":[{},{}]}
