---
title: User interface method and system for management and control of automated image processing in high content screening or high throughput screening
abstract: A user interface method and system for controlling automated image processing operations of HCS and/or HTS systems includes a graphical interface to enable user designation of an image naming convention, image sources and destinations, image processing channels, processing parameter values, and processing spatial designations.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08107711&OS=08107711&RS=08107711
owner: Vala Sciences, Inc.
number: 08107711
owner_city: San Diego
owner_country: US
publication_date: 20090512
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["PRIORITY","RELATED APPLICATIONS","STATEMENT OF GOVERNMENT INTEREST","BACKGROUND","SUMMARY","DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENTS","INDUSTRIAL APPLICATION"],"p":["This application claims priority to U.S. Provisional Application for Patent 61\/127,412, filed May 12, 2008.","The following patent applications contain subject matter related to this application. Both applications are incorporated herein by this reference.","U.S. patent application Ser. No. 11\/285,691, filed Nov. 21, 2005 for \u201cSystem, Method, And Kit For Processing A Magnified Image Of Biological Material To Identify Components Of A Biological Object\u201d, published as US 2008\/0144895 on Jun. 19, 2008; and,","PCT application PCT\/US2006\/044936, filed Nov. 17, 2005 for \u201cSystem, Method, And Kit For Processing A Magnified Image Of Biological Material To Identify Components Of A Biological Object\u201d, published as WO 2007\/061971 on May 31, 2007.","The invention described herein were made in part with government support under Grant No. 1R43DK074333-01, Grant No. 1R41DK076510-01, and Grant No. 1R42HL086076, all awarded by the National Institutes of Health. The United States Government has certain rights in the invention.","The technical field concerns high content screening (HCS) and\/or high throughput screening (HTS) using an automated image processing system having the capability to detect and measure one or more components of one or more objects in magnified images of biological material.","In HCS and\/or HTS, an automated image processing system obtains magnified images from an automated microscope and subjects those images to processing methods that are specially designed to detect and measure small components of biological material. The processing methods employ algorithms customized to respond to markings, such as colors, and to detect particular image characteristics, such as shapes, so as to quickly and reliably identify components or features of interest. Based upon the identification, the system then makes spatial and quantitative measurements useful in analysis of experimental results. Automated image processing systems are increasingly used as assay tools to determine, measure, and analyze the results of tests directed to development or evaluation of drugs and biological agents.","The related applications describe an automated image processing system that distinguishes densely packed shapes in cellular and sub-cellular structures that have been activated in some way. The system identifies components such as membranes, nuclei, lipid droplets, molecules, and so on, using image processing algorithms that are customized to detect the shapes or other visible features of such components.","Presently, HCS and\/or HTS systems quickly acquire and process large numbers of microscopic images and produce significant quantities of information. Substantial attention and time are required from a user to efficiently manage and accurately control the system operations including automated image processing, image data management, and image data analysis. Consequently, there is a need to provide tools that enhance user efficiency and convenience, while reducing the time spent and errors encountered in controlling operations of HCS and\/or HTS systems.","A user interface method and system for controlling automated image processing, image data management, and image data analysis operations of HCS and\/or HTS systems include a graphical user interface (\u201cGUI\u201d) to enable user designation of an image naming convention, image sources and destinations, image processing channels, processing parameter values, and processing spatial designations.","As will be evident, it is desirable to apply the principles of this description broadly to control of image processing algorithms tailored to many and varied analysis tasks in processing systems that process image data to analyze, screen, identify, and\/or classify image features, objects, and\/or contents. It is particularly desirable to afford a user the ability to prepare image data for processing, to selectively control modes and parameters of processing the image data, to view results produced by processing the image data, and to selectively step through successive cycles of image data processing in order to adjust results.","In this description, a specific biological process\u2014transcription\u2014is used to illustrate a user interface method and system for automated image processing in HCS or HTS. The example is intended to illustrate how a user can manage and control execution of an image processing algorithm selected to process magnified images of biological material in order to analyze features of the material affected by a biological assay. This example is not intended to limit the application of the principles of the user method and system only to a particular type of image.","Gene expression begins with transcription, the process by which messenger RNAs are transcribed from the genome. In transcription, messenger RNA (mRNA) is synthesized in a cell under control of DNA. The process copies a DNA sequence into a cell using mRNA. The copied sequence is in fact a strand of RNA in the cell. The number of mRNA copies present in a cell transcribed from a single gene can vary from 0 to >1000, as transcription is heavily regulated during cell differentiation or responses of the cells to hormones, drugs, or disease states.","Through use of inexpensive reagents and simple protocols a transcription assay can be conducted in which mRNA is synthesized and then captured. The location and number of individual mRNA species captured can be visualized in cells and tissue sections by fluorescence-based detection and quantified by automated image processing.","For visualization in images, a probe is used which binds to target mRNA species with very high specificity. It is possible to generate probes to virtually any known sequence. Preferably, such probes are hybridized to the target mRNAs in cell or tissue samples that have been fixed and permeabilized. A fluorescent reagent may then added, which binds to the probe. When slides and well plates containing cultured cells are processed in this manner, and viewed with fluorescence microscopy, bright spots (mRNA loci) are apparent that correspond to individual copies of the target mRNA.","Visual representations of these operations are presented in . However, these are only meant to illustrate how an automated image processing system operates. The panels of these figures are colored for convenience and ease of understanding. In fact, the image acquisition and processing operations of an automated image processing system are conducted on grey scale images that are acquired from stained samples via filtered imaging.","To quantify gene transcription, the mRNA loci can be individually counted for each cell. While this can be done manually, by loading such images in a general purpose image analysis program, manual analysis is very laborious and time consuming due to fatigue and inconsistency between researchers. A convenient, user-friendly, and accurate alternative may be provided by an image processing algorithm, which may be in the form of a Windows\u00ae compatible, Java-based software system, specifically engineered for this application. With reference to , for example, such a system identifies individual cells, and quantifies the number of mRNA loci on a per cell basis in fields of view imaged for nuclei (shown in blue with DAPI staining), and for mRNA (shown in green using fluorescent reagents). Results produced by such a system may be input into a quantitative modeling system (such as a spreadsheet process) in order to organize, quantify, model, and present the results for interpretation and analysis.",{"@attributes":{"id":"p-0035","num":"0034"},"figref":["FIG. 2","FIG. 2","FIG. 2"]},"A user interface method for management and control of automated image processing in high content screening or high throughput screening is now set forth. Although useful for a single image processing algorithm, the explanation presumes the installation and operation of an automated image processing system with a set, group, family, or library of image processing algorithms from which a user may select an algorithm for performing a specific task such as visualization and detection of mRNA loci. Such a system may be based on, for example, the system set forth in related U.S. patent application Ser. No. 11\/285,691. The automated image processing system is installed on or in computer, web, network, and\/or equivalent processing resources that include or execute cooperatively with other processes, including data and file management and quantitative modeling processes. The method includes some or all of the following acts.","1. Initially, an assay sample to be visualized is prepared. The sample may be, for example, cells on a tissue slide, a coverslip or in optically clear multiwall dishes.","2. The automated image processing system is launched and the system acquires magnified images of the sample. For the example mRNA assay described above, such images may include images represented by those of the left panels of . At each image location (well or slide area) the system obtains a grey scale image of nuclei (using a blue filter if nuclei are stained with blue dye) and a grayscale image of mRNA (using a green filter if mRNA strands are colored with a green probe). As they are acquired, the images are placed in a file system storage structure, for example a folder, by the automated image processing system. Preferably, each image has a tag appended to it by the automated image processing system. The tag may be called a \u201cname\u201d. Preferably, the automated image processing system observes and implements at least one, and preferably two or more image naming conventions. Preferably, the automated image processing system receives a command entered by the user as to which naming convention to use when acquiring images. One such naming convention is illustrated in . In the example of , the naming convention includes an alphanumeric image name followed by a designation of a well or a slide area at which the image was obtained, a field designation, and a channel designation. The field designation indicates a field of the designated well or slide area where the image was obtained. The channel designation indicates a processing channel that corresponds to some component of an object in the image. There may be one, two, or more, channels defined for a set of images obtained from an assay. Components that correspond to respective channels may include, for example cell membrane, cell nucleus, lipid droplet, mRNA strand, etc. In some instances, where the color of the component is known, the channel may be designated by the dye selected to stain the component (and thus the filter with which the component is found). Thus, with respect to the illustrative mRNA assay example, a \u201cnuclear channel\u201d (or a \u201cblue\u201d channel) may correspond to cell nuclei, and an \u201cRNA channel\u201d (or a \u201cgreen\u201d channel) to RNA dots.","3. When a set of magnified images has been obtained, named, and placed in a folder by the automated image processing system, an image processing algorithm is launched to obtain image data indicative of assay results from the images. The launch initially causes the graphical user interface (GUI) screen shown in  to be displayed. The screen enables the user to manage and control the automated image processing performed by the algorithm.","4. Using the GUI screen of , the user chooses an image naming convention by way of the drop-down menu entitled \u201cImage Naming Convention\u201d.","5. Using the GUI screen of , the user chooses a source folder containing images to be processed by way of the drop-down menu entitled \u201cSource Folder\u201d. For convenience, the user may browse to a source folder with images containing images tagged according to the selected image naming convention by way of the browse button to the right of the \u201cSource Folder\u201d drop-down menu. This choice will cause the \u201cWells To Run Algorithm On\u201d field to populate, displaying the well or slide area names of files. The result is shown in .","6. Using the GUI screen of , the user chooses a destination folder. Preferably, the automated image processing will generate reference \u201cmask\u201d images and *.csv files (Excel compatible) and place these files in the folder designated here. The destination folder may be found or created using the \u201cDestination Folder\u201d drop-down menu and the browse button to the right of it. The resulting choice is shown in .","7. Using the GUI screen of , the user associates image characteristics with two or more system-named channels for the automated image processing to be conducted. With the illustrated example, the user may associate a first color cannel (blue as channel 0, for example) with a nuclear channel and a second color (green as channel 1, for example) with an RNA channel. The choices designate respective nuclear and mRNA locii process streams in the image processing algorithm. The resulting choices are shown in .","8. Using the GUI screen of , the user establishes a well definition for a number of fields in a \u201cWell Definition\u201d control box. That is, the user indicates the number of fields to be processed in each well (or slide area). Thus, if there is one field (one image) per well, the user defines a single-field matrix on each well by setting both row and column indications to \u201c1\u201d. If 4 images are collected per well (or area) the user may designate 1 row by 4 columns, 2 rows by 2 columns, or 4 columns by 1 row. The images are analyzed independently by the automated image processing system. The resulting choices shown in  imply that only one image is obtained at each well or slide area.","9. Using the GUI screen of , the user establishes threshold parameter values for the channels in a \u201cThreshold Factor\u201d control box. That is, the user indicates a level of sensitivity to be observed by the selected image processing algorithm for each channel. In the illustrated example, the thresholds for the nuclear and RNA channels are set to 100%, which may be a default setting. Generally, as the threshold decreases, the sensitivity increase and dimmer objects will be identified for inclusion in processing operations. The resulting choices are shown in .","10. Using the GUI screen of , the user establishes nuclear size parameter value for the nuclear channel in a \u201cNuclear Size\u201d control box. That is, the user indicates a level of sensitivity to be observed by the selected image processing algorithm for the size of objects in the nuclear channel. The size selected depends on the cell type and magnification used in acquiring the images. The objective is to reduce instances where the selected algorithm will incorrectly separate a large object into two smaller objects. The resulting choice is shown in .","11. Using the GUI screen as per , the user selects the wells (or slide areas) whose images will be processed by the selected algorithm. That is, the GUI screen lists in the \u201cWell Name\u201d column all of the wells from which images have been acquired, and presents in the \u201cRun Algorithm\u201d column a box for each named well that the user can click to cause the algorithm to process the image or images acquired from that well.","12. Using the GUI screen as per , the user commands the algorithm to execute according to the entries on the screen. In response, the automated image processing system access the source folder in a predetermined sequence, subjects the acquired images in the source folder to the selected algorithm, and generates results including images or masks such as those showing the green mRNA loci in . The masks or images generated are named and stored as image files in the results folder. Using loci information in the images or masks produced, the automated image processing system extracts quantitive data",{"@attributes":{"id":"p-0049","num":"0048"},"figref":["FIG. 6","FIGS. 1 and 2","FIG. 1","FIG. 2"]},"Using well-known Excel spread sheet processing, the mRNA assay described above, and an image processing algorithm, examples of experimental data processing, handling, and storage are now described.","File Examples: Presume that the image processing algorithm creates data files in the *.csv (comma separated value) format that can be loaded easily into the well-known Excel\u00ae spreadsheet system. A file that represents a summary for an experimental data set is created and is placed at a first level within the Destination folder. One example is the PMAvsIL8_DataTable.csv shown in the upper panel of . Additionally, two data files are created within a subdirectory for each selected well. The wellname_DataTable.csv file (e.g., C15_DataTable.csv in , lower panel) contains a cell by cell data readout for every cell analyzed for the well (or slide area). A Well_name DataTable_Stats.csv file contains summary statistics for a selected well. For example, C15_DataTable_Stats.csv in , lower panel, contains summary statistics for well C15, selected as described above.","Data Table Examples: The experimental data may be stored in tables, such as the tables referenced in the files described above, and may be provided therein to a quantitative modeling system for further processing. One example of a table containing experimental data for use by a spreadsheet process is seen in . In this example, a user would launch a spreadsheet process and use an OPEN command to open the C15_DataTable.csv file shown in . It may be necessary to select \u201cAll Files\u201d in the \u201cFiles of type\u201d field within the Open menu of the spreadsheet process to view and select csv files. In response, the spreadsheet process will automatically open a \u201cworkbook\u201d\u2014style interface and the spreadsheet cells will range from addresses A1 to AA178 for C15_DataTable.csv. Note that a description of the file is automatically generated and displayed in addresses A1 to B2 (e.g., Data Table: C15 Data Table. Description: Data Table for cells in well C15), and the Legend portion of the file extends from A5 to C3. A7 to A33 indicate the data type of each parameter (integer, double precision, or Boolean). B7 to B33 contain short descriptions, which are also the column headers for the data displayed in the Data Table portion of the spreadsheet (A36 to AA178 for C15_DataTable.csv). C7 to C33 contain brief descriptions of each data parameter. The \u201cid\u201d label (Excel address B7) is the header for column A in the Data Table; this is an integer number that is uniquely assigned to each cell in the image corresponding to well C15.","The experimental data provided to the quantitative modeling system may include quantitative data obtained from the images acquired and\/or produced by the automated image processing system. For example, refer to , which represents a cell with mRNA according to the assay example described above. In , Nm is the nuclear mask and corresponds to the number of pixels that make up the nuclei. Cm is the cytoplasmic mask, which extends from the cell boundaries to the nucleus. Rm is the RNA mask and corresponds to the number of pixels found within RNA dots for the cell. The automated image processing system obtains quantitative experimental data from the acquired and\/or result images, and places the data into tables such as the table shown in . The examples shown in this table include data obtained from nuclear and loci images discussed above. Nm, which is the size of the nucleus for in units of pixel area, is obtained from an acquired image showing cell nucleii. Area Rm (Area of the RNA mask) represents the total number of pixels identified as corresponding to RNA dots within the RNA image for each cell as per , and is an index of mRNA expression, and will be of considerable interest to the majority of users. Data parameters XLeft Nm, YTop Nm, Width Nm, and Height Nm refer to the x,y location of each nucleus within a nuclear image, and the width and height dimensions, which will assist a user in identifying the location of each cell within a field of view. \u201cIsBoundaryNm\u201d can be either True or False; cells near the boundary of the image (IsBoundaryNm=True) might extend beyond the field of view, and the analysis for RNA expression by may be incomplete. The IsBoundaryNm parameter can be used to sort the cells within the spreadsheet program, and exclude boundary cells from further analysis, if desired. XCentroid Nm and YCentroid Nm are the x and y coordinates within the image for the center of each nucleus.","Continuing with the description of the data table example of , RNA spot count, Mean RNA Spot Area, and RMS RNA Spot Diameter are useful data parameters relating to RNA expression. RNA spot count is the number of mRNA loci for each cell. Mean RNA Spot Area is the average size of the RNA spots for a particular cell (in units of pixel area). RMS RNA Spot Diameter is an estimate of the mean diameter of the RNA spots in the cell (RMS stands for a Root Mean Square, and refers to the method used to estimate spot diameter). Area X Nm is the area of the nucleus that is NOT also part of the RNA mask; similarly, Area X Cm is the area of the cytoplasmic mask that is NOT also part of the RNA mask. Area X Nm and Area X Cm define the size of the \u201cbackground\u201d areas within the nucleus and cytoplasm. Advanced users may find these data parameters useful, especially with comparisons to the Area Rm; for example, it might be of interest to calculate: Area Rm \/(Area X Nm+Area X Cm+Area Rm), which is the ratio of the area of the RNA spots to the entire area of the cell.","In the example of , Total integrated intensity of the RNA image for the RNA mask is the sum of intensities of the pixels that have been assigned to the RNA mask for each cell (TII Ri Rm\u2014line 22 and column P of the Data Table), is a useful parameter related to mRNA expression. Similarly, the average and median pixel intensities of the RNA image for the RNA mask for the cell are the API Ri Rm, and MPI Ri Rm, respectively. The Standard Deviation of Pixel Intensities for the RNA image RNA mask (SPI Ri Rm) is also reported. This parameter may be of special interest to researchers performing screens of chemical or RNAi libraries involving thousands of samples, as standard deviations of intensity can sometimes be less variable than the means or total integrated intensity measurements.","Finally, in the table of , a series of data parameters are reported that correspond to the background pixel intensities. These include the total integrated, average, and median pixel intensities for the RNA image for pixels within the nuclear mask that are NOT RNA spots (TII Ri X Nm, API Ri X Nm, MPI Ri X Nm, where \u201cX\u201d means NOT RNA spots). The same series of values are also reported for the regions of the cytoplasm that are NOT RNA spots (TII Ri X Cm, API Ri X Cm, MPI Ri X Cm). These data parameters can be used, in combination with the data parameters for the RNA spots to quantify how bright the spots are with regard to the background. For example, differences between API Ri Rm\u2212API Ri X Cm represents the difference in intensity between the RNA spots and the background within the cytoplasmic region. Such differences may be useful parameters to monitor in a screening assay, and, also are likely to be useful for optimization of the assay conditions and imaging parameters for particular samples types.","In , two additional data tables useful for managing additional experimental data related to the mRNA example described above are shown. The first part of the data table portion of the C15_DataTable.csv file is shown in the upper panel of ; the analogous portion of the G15_DataTable.csv file is shown in the lower panel. For the mRNA experiment, cells in the C15 well of the dish were not exposed to an activator of IL-8 expression. Thus, cells in C15 represent the negative control for the assay. Alternatively, cells in G15 were exposed to 1 ng\/ml PMA, a phorbol ester that strongly activates IL-8 expression. For the first 10 cells analyzed for C15, no RNA spots were detected. Thus, there are \u201c0\u201d values in Columns C, K, L, and M, which correspond to the data parameters area of the RNA mask (Area Rm), RNA spot count, and mean RNA spot area, and RNA spot diameter, respectively. Note, also that the first two cells of C15 were boundary cells (IsBoundaryNm=\u201cTrue\u201d), where as the rest of the cells were judged as being entirely contained within the image (IsBoundaryNm=\u201cFalse\u201d). Data is reported on a total of 142 cells for well C15 in the C15_DataTable.csv file. In contrast, all of the first 11 cells in the G15 data table featured RNA spots ( lower panel). Thus, there are positive data entries for every line in columns C, K, L, and M. For G15, cell number 8 (Excel line 44), for example, featured 2106 pixels in the RNA mask (column C), an RNA spot count of 148 (column K), a mean RNA spot area of 14.23 pixels (Column L), and an average RMS spot diameter of 4.2565 pixels. Note that data is reported on a total of 136 cells for well C15 in the C15_DataTable.csv file.","With reference to , portions of the C15_DataTable_Stats.csv (found in the C15 directory) and the PMAvsIL8_DataTable_Stats.csv files (found under the parent directory for the experimental results) are illustrated. The layout of the DataTable_Stats.csv files is related to, but somewhat different than the previously described DataTable.csv files. For example, values in column A are the StatsID numbers. There are 6 useful statistics which are the Count (Row 39 in the C15_DataTableStats.csv file) which is the number of cells that were used in the calculations, the Mean, which is the average value obtained for all cells (the well population) that were analyzed in the well, Sigma, which is the standard deviation for the data parameter and for the well population, Median, which is the value of the data parameter for which 50% of the data values for the well exceeded (and 50% were below), the Min, which is the lowest value obtained, and the Max, which is the maximum value that was obtained. Column B displays the well designation for housekeeping purposes, and Column C displays the \u201cCount\u201d, \u201cMean\u201d, \u201cSigma\u201d, \u201cMedian\u201d, and \u201cMax\u201d titles. Note that all of the data that is displayed refers to values that were derived on a \u201cper cell\u201d basis. For well C15, 142 cells were identified and the data that is summarized in the DataTable_Stats.csv files includes data derived from all of the cells (including the boundary cells), so the count is 142 for every statistic in the report. The Mean value for the RNA Spot Count for well C15 was 0.03521 R for well C15, and a maximum of 2 spots per cell were found for the cell population. Note that the PMAvsIL8_DataTable_Stats.csv file (, lower portion), features the identical display for well C15, along with data obtained from all wells in the experimental analysis. Thus, this file provides a convenient reference, displaying a summary of all the results for the experiment.","Results for the experiment in which the effect of PMA was tested on IL8 mRNA expression are shown in . Results are graphed and tabulated for 3 key data parameters that describe mRNA expression. Area Rm, the average area, per cell, of the RNA mask was <1 for well C15, but >1100 for well G15. Thus, addition of 1 ng\/ml PMA elicited a 3000-fold increase in this parameter. For the RNA spot count, essentially no spots were found for the control well (the average number of spots was approx. 0.04\/cell), whereas 14.3 spots\/cell were found for cells exposed to 0.1 ng\/ml PMA (well E15), and 84.1 spots\/cell were found for 1 ng\/ml PMA (well G15). Also, note that the TII Ri Rm data parameter, which is the total intensity of the spots\/cell, went up by 8000-fold (Table in ). Since the assay results in a single RNA spot per mRNA, the RNA Spot Count data parameter may be of interest. Users screening large chemical or siRNA libraries vs. mRNA expression, utilizing automated methodology, may find the Area Rm and TII Ri Rm data parameters of interest, due to the very high dynamic range these parameters may provide for the assay.","Setting examples: Refer now to  for an understanding of Nuclear Size adjustment using the GUI of . A default setting (Nuclear Size=10, Nuclear Threshold=100, RNA Threshold=100) are appropriate for digital microscopy workstations utilizing 20\u00d7 objectives, and for images captured with typical digital cameras. While these settings are likely to be very good for most circumstances, a user may run test analyses at various settings, to further optimize the performance of the automated image processing system. To produce optimal data analysis, the automated image processing system should identify the position of each nucleus in the nuclear image for every field of view. To help the system recognize the nuclei of different cell types and at different magnifications, and different overall staining intensities, user-adjustable controls are provided on the user GUI of  that are relevant to the nuclear images. These are the expected Nuclear Size, and the Nuclear Threshold settings. In the example of , a number between 1 and 99 can be entered into the Nuclear Size field. These numbers may not correspond to an exact physical dimension of the nucleus, but, instead may be relative. To adjust the nuclear size adjustment for improved results, a user may set the Nuclear Size to 5, with the Nuclear and RNA Thresholds set at 100%, select a well (or slide area) for analysis and run the mRNA image processing algorithm. Next, a new output folder may be created and named, and, with the Nuclear Size set to another value (for example, 16) the algorithm may be run on the same well (or slide area). Images generated by the algorithm of the same well with different Nuclear Size settings are shown side by side in . The Nuclear edge mask shows the boundary circles for the nuclei identified by algorithm processing. For the Nuclear Size 5 analysis, many of the original nuclei are subdivided into two or more circles in the Nuclear edge mask. Thus, Nuclear Size 5 may be too low a value for this cell type and magnification. In this regard, consider the Whole cell mask-edges generated for the size 5 setting, which displays the boundaries of the cells as estimated by the algorithm; many very small shapes are shown that may be too small to represent authentic cells and many cell boundary lines cross nuclei (some are sectioned into 2 or even 4 cells). Consider next the Nuclear edge mask and Whole cell mask-edge images for the analysis with Nuclear Size 16. The Nuclear edge mask image includes single circles at the position of nearly every authentic nuclei in the field of view (lower middle panel, ), indicating that the algorithm performed correctly. Furthermore, the cell boundaries are appropriately sized and rarely cross nuclei. Thus, for the particular circumstances of this example, a Nuclear Size of 16 will result in accurate cell counts, and an accurate count of the number of mRNA spots per cell.","Refer now to  for an understanding of the Nuclear Threshold adjustment using the GUI of . Entry of a lower number may cause the algorithm to recognize dimmer nuclei in the nuclear channel, whereas entry of larger numbers will reduce the sensitivity of the system. To illustrate this principle, the acquired images that resulted in the images in  resulted in the images of , with the Nuclear Size set to 16, RNA Threshold to 100, with Nuclear Threshold settings of 100 and 300. The results indicate that a setting of 300 resulted in many nucleii being missed, indicating greater algorithm accuracy with the lower setting of 100.","The GUI of  also provides an RNA Threshold adjustment. The ability of the mRNA algorithm to analyze the RNA image may be adjusted by use of the RNA Threshold feature. With reference to the middle panel in , the smaller the number entered for this parameter, the more spots will be counted by the program. However, the smaller the number that is entered, the greater the risk of also quantifying small image artifacts as authentic RNA spots. Adjustment of the RNA threshold setting may cause the mRNA algorithm to match what a user may see when looking through a microscope, and using any image enhancement tools at hand. Another approach that may be preferred when performing screening assays may be to select RNA threshold parameters that yield the greatest separation between certain experimental conditions. For example, reducing the RNA channel sensitivity (by using a higher RNA threshold number), might diminish the number of \u201cfalse positives\u201d in a large screen.","A method and system for controlling automated image processing, image data management, and image data analysis operations of HCS and\/or HTS systems according the Detailed Description include a graphical user interface (\u201cGUI\u201d) to enable user designation of an image naming convention, image sources and destinations, image processing channels, processing parameter values, and processing spatial designations may be implemented in a software program, for example, a program written in the C++ and\/or Java programming languages, and a counterpart system may be a general purpose computer system programmed to execute the method. Of course, the method and the programmed computer system may also be embodied in a special purpose processor provided as a set of one or more chips. Further, there may be a program product constituted of a program of computer or software instructions or steps stored on a tangible article of manufacture that causes a computer to execute the method. The tangible article of manufacture may be constituted of one or more real and\/or virtual data storage articles.",{"@attributes":{"id":"p-0064","num":"0063"},"figref":"FIG. 15","b":["100","110","112","116","110","116","116","118","110","120","122","112","110","126","126","126","127","128","127","128","128","128","128","130"]},"With further reference to , the processor  may be a programmed general purpose digital processor having a standard architecture, such as a computer work station. The processor  includes a processing unit (CPU)  that communicates with a number of peripheral devices by way of a bus subsystem . The peripheral devices include a memory subsystem (MEMORY) , a file storage subsystem (FILE) , user interlace devices (USER) , an input device (INPUT) , and an interface device (INTERFACE) . It is not necessary that the processor  be connected directly to the microscope ; it may receive magnified images produced by the microscope from a portable storage device, or by way of a local or wide area network. For example, magnified images obtained by a microscope may be transported to the processor over the internet.","The bus subsystem  includes media, devices, ports, protocols, and procedures that enable the processing unit  and the peripheral devices , , , , and  to communicate and transfer data. The bus subsystem  provides generally for the processing unit and peripherals to be collocated or dispersed","The memory subsystem  includes read-only memory (ROM) for storage of one or more programs of instructions that implement a number of functions and processes. One of the programs is an automated image process for processing a magnified image of biological material to identify one or more components of an image. The memory subsystem  also includes random access memory (RAM) for storing instructions and results during process execution. The RAM is used by the automated image process for storage of images generated as the process executes. The file storage subsystem  provides non-volatile storage for program, data, and image files and may include any one or more of a hard drive, floppy drive, CD-ROM, and equivalent devices","The user interface devices  include interface programs and input and output devices supporting a graphical user interface (GUI) for entry of data and commands, initiation and termination of processes and routines and for output of prompts, requests, screens, menus, data, images, and results.","The input device  enables the processor  to receive digital images directly from the camera , or from another source such as a portable storage device, or by way of a local or wide area network. The interlace device  enables the processor  to connect to and communicate with other local or remote processors, computers, servers, clients, nodes and networks. For example, the interface device  may provide access to an output device  by way of a local or global network .","Methods and apparatuses for practicing the teachings of this specification may be constituted in whole or in part of a program product with a computer-readable storage medium, network, and\/or node that enables a computer, a processor, a fixed or scalable set of resources, a network service, or any equivalent programmable real and\/or virtual entity to execute a GUI and\/or perform image processing as described and illustrated below. The program product may include a portable medium suitable for temporarily or permanently storing a program of software instructions that may be read, compiled and executed by a computer, a processor, or any equivalent article. For example, the program product may include a programmed CD such as is seen in , or a network-accessible site, node, center, or any equivalent article."],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0012","num":"0011"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0013","num":"0012"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0014","num":"0013"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0015","num":"0014"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0016","num":"0015"},"figref":["FIG. 5","FIG. 4"]},{"@attributes":{"id":"p-0017","num":"0016"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0018","num":"0017"},"figref":"FIG. 7"},{"@attributes":{"id":"p-0019","num":"0018"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0020","num":"0019"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0021","num":"0020"},"figref":"FIG. 10"},{"@attributes":{"id":"p-0022","num":"0021"},"figref":"FIG. 11"},{"@attributes":{"id":"p-0023","num":"0022"},"figref":"FIG. 12"},{"@attributes":{"id":"p-0024","num":"0023"},"figref":["FIG. 13","FIG. 4"]},{"@attributes":{"id":"p-0025","num":"0024"},"figref":["FIG. 14","FIG. 4"]},{"@attributes":{"id":"p-0026","num":"0025"},"figref":"FIG. 15"},{"@attributes":{"id":"p-0027","num":"0026"},"figref":"FIG. 16"}]},"DETDESC":[{},{}]}
