---
title: Malignancy-risk signature from histologically normal breast tissue
abstract: The invention provides for malignancy-risk gene signatures that predict the risk of developing breast cancer, the recurrence of breast cancer, and/or the metastasis of breast cancer. These signatures have numerous clinical applications including assessing risk of breast cancer development following routine breast biopsy, assessing the need for adjuvant radiotherapy after lumpectomy, and determining the need for completion mastectomy following lumpectomy for the breast cancer patient and other treatment plans that are personalized for the patient.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09195796&OS=09195796&RS=09195796
owner: University of South Florida
number: 09195796
owner_city: Tampa
owner_country: US
publication_date: 20090112
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATIONS","STATEMENT OF GOVERNMENT INTEREST","FIELD OF INVENTION","BACKGROUND OF THE INVENTION","BRIEF SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENT","DEFINITIONS","EXAMPLES","Example 1","Materials and Methods","Example 2","IDC Gene Signature","Example 3","Outlier Breast Tissues","Example 4","Histologic Findings in Outlier Breast Tissues","Example 5","Outlier Gene","Example 6","Pathway Analysis of Outlier Genes","Example 7","Re-Examination of Outlier Tissues and the Adjacent Tissues","Example 8","Turashvili's Study of Normal and IDC Comparison","Example 9","Moffitt DCIS Samples","Example 10","Ma's Breast Cancer Study","Example 10A","Example 10B","Example 11","Poola's Atypical Ductal Hyperplastic (ADH) Study","Example 11A","Example 11B","Example 12","Van't Veer Breast Metastasis Dataset","Example 12A","Example 12B","Example 13","Additional Materials and Methods","Example 14","ER\/PR\/Her2 Analysis","Example 15","Cross-Validation","Example 16","Weak Correlation of Malignancy-Risk Score with ER, PR, and Her2","Example 17","Higher Malignancy-Risk Score of IDC-Like Normal Tissues","Example 18","RT-PCR Validation of Malignancy-Risk Genes","Example 19","Chanrion et al's Relapse Study","Example 20","Wang et al's Breast Cancer Relapse Free Survival Study","Example 21","Huang et al's Breast Lymph Node Study","REFERENCES"],"p":["This application claims the priority benefit of the U.S. Provisional Application Ser. No. 61\/020,575, filed Jan. 11, 2008; which is hereby incorporated by reference in its entirety.","This invention was made with Government support under Grant Nos. CA076292, CA098522 and CA112215 awarded by the National Institutes of Health. The Government has certain rights in the invention.","This invention relates to cancer screening and therapy. Specifically, the invention entails screening pre-pathological breast tissue for potential oncogenesis.","Throughout this specification, reference numbering is sometimes used to refer to the full citation for the references, which can be found in the \u201cReference Bibliography\u201d after the Examples section. The disclosure of all patents, patent applications, and publications cited herein are hereby incorporated by reference in their entirety for all purposes.","In 2004, 186,772 women and 1,815 men were diagnosed with breast cancer, making it the second most common cancer in women. Conventional treatment includes surgery, chemotherapy, hormone therapy, and radiation. Breast conserving approaches have been increasingly used to treat invasive breast cancer. Historical data, however, have reported a 40% local recurrence rate after lumpectomy without radiotherapy. Furthermore, the rate of local recurrence after mastectomy has been reported at 10-30%. Local recurrence rate following lumpectomy without radiotherapy for ductal carcinoma in situ is as high as 63%, with invasive cancer occurring in over 36% of cases, and a median recurrence between 2 and 6 years, depending on the initial stage of the resected tumor. Radiotherapy significantly reduces rate of local recurrence to 10% or less, but does not completely eliminate the risk of cancer. Collectively, these data demonstrate the potential apparent normal breast possesses to harbor pre-malignant changes or very early malignancy at molecular level, and emphasize the insensitivity of the current strategies to detect disease at an early molecular stage, even in patients known to be at high-risk. The potential benefit of a molecular signature as an indication of risk for subsequent development of breast cancer would, therefore, be very useful in screening applications.","Currently, it is estimated that false negatives and new cancers previously screened as negative may amount to 2-4% of the new cancer cases following breast biopsy. Determination of molecular markers of malignancy in histologically normal breast may improve the potential for breast biopsy to identify at-risk patients, refine the current practice of intra-operative assessment of margins of the resected breast tissues based on histology alone, and may prove useful in guiding treatment choices after lumpectomy. For example, it is clear that not all patients require or benefit from post-lumpectomy radiotherapy, which often leads to considerable cosmetic defects in the residual breast.","Accordingly, there is a need for the ability for to predict which individuals, who have histologically normal breast tissue, will likely develop breast cancer, or breast cancer recurrence, or metastasis of breast cancer. The ability for physicians and\/or others of skill in the art allows for personalized treatment of patients and avoids unnecessary treatments that are not beneficial to the patient's health.","The invention provides for, inter alia, malignancy-risk gene signatures that predict the risk of developing breast cancer, the recurrence of breast cancer, and\/or the metastasis of breast cancer; methods of using such signatures, and kits containing arrays of malignancy-risk gene signatures. Thus, one objective of the invention was to establish high cancer-risk gene signatures in histologically normal breast tissues obtained from patients with invasive breast cancer. Other objectives are described in greater detail infra.","As detailed further in the Examples, outlier gene signatures were derived to assess cancer risk from the 143 histologically normal breast tissues derived from patients who underwent mastectomy for breast carcinoma. Up to four normal breast samples, adjacent to an invasive ductal carcinoma, were obtained from each patient. Validation results indicated that the outlier gene signature had multiple predictive properties, including potential to predict cancer risk, disease progression, and metastasis. Since the outlier genes were highly associated with cell proliferation, it is conceivable that these proliferation genes may have a role in the earliest stages of breast cancer development and subsequent progression.","These signatures have numerous clinical applications including, but not limited to, assessing risk of breast cancer development following routine breast biopsy, assessing the need for adjuvant radiotherapy after lumpectomy, and determining the need for completion mastectomy following lumpectomy for the breast cancer patient and other treatment plans that are personalized for the patient and thus, fulfill several needs in this field.","The invention described herein provides for malignancy-risk gene signature and methods for determining or predicting the likelihood that an individual who has histologically normal breast tissue will develop breast cancer, have recurrence of breast cancer, and\/or for predicting spread of breast cancer by using malignancy-risk gene signatures.","Accordingly, in one aspect, the invention provides for methods for predicting the likelihood for the development of breast cancer in an individual comprising: (a) analyzing a breast tissue sample from the individual wherein the sample comprises substantially histologically normal cells from the individual to obtain a first gene expression profile; (b) comparing the first gene expression profile to a malignancy-risk gene signature as depicted in Table 6; and (c) predicting the likelihood that the individual will develop cancer if the individual expresses at least 10 of the malignancy-risk genes. In one embodiment, the individual is suspected of having breast cancer or who has had breast cancer or who is at risk for breast cancer. In another embodiment, the individual has had or currently has atypical hyperplasia. In another embodiment, the individual has had or currently has fibroadenoma. In another embodiment, the breast cancer is selected from the group consisting of: invasive ductal carcinoma (IDC), ductal carcinoma in situ (DCIS), lobular carcinoma in situ (LCIS), and invasive lobular carcinoma (ILC). In another embodiment, the individual had had a surgical procedure to remove breast cancer, breast tumor, or breast lesion. In one embodiment, the surgical procedure is a lumpectomy or a mastectomy. In another embodiment, the individual has initiated chemotherapy treatment either before, or after the surgical procedure. In another embodiment, the breast tissue sample is obtained ipsilaterally with the cancer, tumor, or lesion.","In another aspect, the invention provides for methods for predicting the likelihood for the spread of breast cancer in an individual who has had or currently has breast cancer comprising: (a) obtaining a breast tissue sample from the individual comprising substantially histologically normal cells from the individual; (b) analyzing the sample to obtain a first gene expression profile; (c) comparing the first gene expression profile to a malignancy-risk gene signature listed in Table 6; and (d) predicting the likelihood that the individual will develop spread of breast cancer if the individual expresses at least 10 of the malignancy-risk genes. In one embodiment, the breast cancer is selected from the group consisting of: invasive ductal carcinoma (IDC), ductal carcinoma in situ (DCIS), lobular carcinoma in situ (LCIS), and invasive lobular carcinoma (ILC). In another embodiment, the spread of cancer is metastasis. In another embodiment, the individual had had a surgical procedure to remove breast cancer. In another embodiment, the surgical procedure is a lumpectomy or a mastectomy. In another embodiment, the individual has initiated chemotherapy or radiation treatment before or after the surgical procedure.","In any of the aspects or embodiments, the malignancy-risk signature is at least 10 genes selected from Table 3. In any of the aspects or embodiments, the malignancy-risk signature is selected from the group consisting of the following genes from Table 6: APOBEC3B, C6orf115, CASC5, CORO2A, FAM83D, HIST1H2BF, MOBK1B, WDSOF1, IQGAP3, AMOTL2, CCDC50, CHRDL1, GPRASP1, MAGI2, RBMS3, SYNPO2, Affymetrix probe set id 213158_at, Affymetrix probe set id 226250_at, Affymetrix probe set id 226252_at, Affymetrix probe set id 227082_at, Affymetrix probe set id 227121_at, Affymetrix probe set id 227646_at, Affymetrix probe set id 227719_at, Affymetrix probe set id 235556_at, Affymetrix probe set id 235570_at, and Affymetrix probe set id 243584_at. In any of the aspects or embodiments, the malignancy-risk genes are selected from the genes in Table 7. In any of the aspects or embodiments, the malignancy-risk gene signature is selected from the genes in Table 8. In any of the aspects or embodiments, the malignancy-risk gene signature does not include at least one of the genes in . In any of the aspects or embodiments, the malignancy-risk gene signature does not include at least one of the genes in . In any of the aspects or embodiments, the predicting of the likelihood that the individual will develop breast cancer or spread of breast cancer is expression of at least 117 of the malignancy-risk genes. In any of the aspects or embodiments, the malignancy-risk gene signature does not include at least one of the genes in . In any of the aspects or embodiments, the malignancy-risk gene signature is selected from the genes in . In any of the aspects or embodiments, the malignancy-risk gene signature is selected from the genes in . In any of the aspects or embodiments, the malignancy-risk gene signature does not include at least one of the genes in . In any of the aspects or embodiments, the malignancy-risk gene signature is selected from the genes in . In any of the aspects or embodiments, the malignancy-risk gene signature is selected from the genes in .","In another aspect, the invention provides for malignancy-risk gene signatures for use in predicting or diagnosing cancer wherein the gene signature comprises at least 10 genes of Table 6. In one embodiment, the gene signature comprises at least 10 genes of Table 3. In another embodiment, the gene signature does not include at least 1 gene selected from the genes listed in , B, A, and A. In another embodiment, wherein the gene signature comprises at least about 30 genes selected from the group consisting of the genes in Table 3, Table 7, Table 8, , , , and . In another embodiment, the gene signature comprises at least 10 genes selected from the group consisting of: APOBEC3B, C6orf115, CASC5, CORO2A, FAM83D, HIST1H2BF, MOBK1B, WDSOF1, IQGAP3, AMOTL2, CCDC50, CHRDL1, GPRASP1, MAGI2, RBMS3, SYNPO2, Affymetrix probe set id 213158_at, Affymetrix probe set id 226250_at, Affymetrix probe set id 226252_at, Affymetrix probe set id 227082_at, Affymetrix probe set id 227121_at, Affymetrix probe set id 227646_at, Affymetrix probe set id 227719_at, Affymetrix probe set id 235556_at, Affymetrix probe set id 235570_at, and Affymetrix probe set id 243584_at.","In another aspect, the invention provides for arrays comprising polynucleotides hybridizing to at least about 30 malignancy-risk signature genes immobilized on a solid surface, wherein said gene signature genes are listed in Table 6.","In another aspect, the invention provides for arrays comprising polynucleotides hybridizing to at least about 30 malignancy-risk signature genes immobilized on a solid surface, wherein said gene signature genes are listed in Table 3, Table 7, Table 8, , , , and .","In another aspect, the invention provides for kits comprising the array comprising malignancy-risk gene signatures of any one of the above and a set of instructions for determining an individual's likelihood of developing breast cancer.","In another aspect, the invention provides for kits comprising the array comprising malignancy-risk gene signatures of any one of the above and a set of instructions for determining an individual's likelihood of having breast cancer spread to other locations in the individual's body.","In another aspect, the invention provides for computer readable media comprising a malignancy-risk signature which comprises at least about 30 genes from Table 6.","In another aspect, the invention provides for computer readable media comprising a malignancy-risk signature which comprises at least about 30 genes from Table 3, Table 7, Table 8, , , , and .","The inventors have discovered malignancy-risk gene signature that can assess an individual's risk of developing breast cancer. The invention described herein provides methods for determining or predicting the likelihood that an individual who has histologically normal breast tissue will develop breast cancer, have recurrence of breast cancer, and\/or for predicting spread of breast cancer by using malignancy-risk gene signatures. The invention also describes malignancy-risk gene signatures that can be used for such determination and arrays\/microarrays that comprise these malignancy-risk gene signatures. The invention further provides for kits comprising such arrays\/microarrays and computer readable media with such malignancy-risk gene signatures for use in determining if an individual will experience the development of breast cancer, recurrence of breast cancer and\/or spread of breast cancer.","For purposes of interpreting this specification, the following definitions will apply and whenever appropriate, terms used in the singular will also include the plural and vice versa. In the event that any definition set forth below conflicts with any document incorporated herein by reference, the definition set forth below shall control.","\u201cMalignancy-risk,\u201d \u201chigh risk gene signature\u201d and \u201coutlier gene signature\u201d are used interchangeably herein and are used herein to describe gene signatures that can predict if an individual with histologically normal breast tissue is at risk to develop breast cancer, to have recurrence of breast cancer, and\/or to have metastasis (i.e., spread) of breast cancer.","As used herein, an individual \u201cat risk\u201d of developing breast cancer may or may not have detectable disease or symptoms of disease, and may or may not have displayed detectable disease or symptoms of disease prior to the treatment methods described herein. \u201cAt risk\u201d denotes that a subject has one or more risk factors, which are measurable parameters that correlate with development of breast cancer, as described herein and known in the art. A subject having one or more of these risk factors has a higher probability of developing breast cancer than a subject without one or more of these risk factor(s). For example, in some embodiments, a subject \u201cat risk\u201d of developing breast cancer has a genetic signature comprising one or more of the genes set forth in Table 6. In another embodiment, a subject \u201cat risk\u201d of developing breast cancer has a genetic signature comprising one or more of the genes set forth in Table 3, 7, 8, , B, B and A.","\u201cBreast cancer\u201d as used herein refers to malignant, often uncontrolled, growth of cells in the breast. Although breast cancer occurs primarily in females, breast cancer can occur in men. As such, it is to be understood that the invention applies to both females and males.","\u201cOutlier breast tissue\u201d refer to breast tissue that is histologically normal but has a molecular signature that is abnormal and could has the capacity to develop into cancer. Non-limiting methods for determining if a tissue is an outlier tissue is described herein.","An \u201cindividual,\u201d \u201csubject,\u201d or \u201cpatient\u201d is a vertebrate. In certain embodiments, the vertebrate is a mammal. Mammals include, but are not limited to, primates (including human and non-human primates) and rodents (e.g., mice and rats). In certain embodiments, a mammal is a human (both female and male).","A \u201cpatient\u201d refers to an \u201cindividual\u201d who is under the care of a treating physician. In one embodiment, the patient is a female. In another embodiment, the patient is a female who had not been diagnosed with breast cancer. In yet other embodiments, the patient is a female who has been diagnosed with breast cancer but has had surgery to remove the breast cancer tissue.","A \u201cpatient subpopulation,\u201d and grammatical variations thereof, as used herein, refers to a patient subset characterized as having one or more distinctive measurable and\/or identifiable characteristics that distinguishes the patient subset from others in the broader disease category to which it belongs. Such characteristics include disease subcategories (e.g., invasive carcinoma vs. in situ carcinoma), treatment history, etc. In one embodiment, a patient subpopulation is characterized by genetic signatures, including malignancy-risk gene signatures.","The term \u201csample\u201d, as used herein, refers to a composition that is obtained or derived from an individual that contains a cellular and\/or other molecular entity that is to be characterized and\/or identified, for example based on physical, biochemical, chemical and\/or physiological characteristics. In one embodiment, the sample is taken from a breast that is ipsilateral to the breast cancer.","By \u201ctissue or cell sample\u201d is meant a collection of similar cells obtained from a tissue of a subject or patient. The source of the tissue or cell sample may be solid tissue as from a fresh, frozen and\/or preserved tissue sample. In one embodiment, the tissue or cell sample may be taken from a breast lumpectomy or mastectomy. The tissue sample may also be primary or cultured cells or cell lines taken from and\/or derived from an individual. The tissue sample may contain compounds which are not naturally intermixed with the tissue in nature such as preservatives, anticoagulants, buffers, fixatives, nutrients, antibiotics, or the like.","As used herein, \u201ctissue sample\u201d or \u201csample\u201d comprising substantially normal histological cells have at least about 50% of the cells in the sample which have normal histological appearance (as determined by one of skill in the art, e.g., a pathologist). In some embodiments, at least about 55% of the cells in the sample which have normal histological appearance. In yet other embodiments, at least about 60%, at least about 65%, at least about 70%, at least about 75%, at least about 80%, at least about 85%, at least about 90%, at least about 95%, at least about 96%, at least about 97%, at least about 98%, at least about 99%, at least about 99.5% of the cells in the sample which have normal histological appearance.","\u201cPre-pathological\u201d tissue refers to tissue which has a normal histological appearance.","For the purposes herein a \u201csection\u201d of a tissue sample is meant a single part or piece of a tissue sample, e.g. a thin slice of tissue or cells cut from a tissue sample. It is understood that multiple sections of tissue samples may be taken and subjected to analysis according to the present invention. It is understood that the present invention comprises a method whereby the same section of tissue sample can be analyzed at both morphological and molecular levels, or can be analyzed with respect to both protein and nucleic acid expression. The examples provided herein where nucleic acid expression is used for predictive purposes are non-limiting examples. It is to be understood that protein expression could also be used for predictive purposes.","As used herein, \u201carray\u201d and \u201cmicroarray\u201d are interchangeable and refer to an arrangement of a collection of nucleotide sequences in a centralized location. Arrays can be on a solid substrate, such as a glass slide, or on a semi-solid substrate, such as nitrocellulose membrane. The nucleotide sequences can be DNA, RNA, or any permutations thereof. The nucleotide sequences can also be partial sequences from a gene, primers, whole gene sequences, non-coding sequences, coding sequences, published sequences, known sequences, or novel sequences.","The term \u201cpolynucleotide\u201d or \u201cnucleic acid,\u201d as used interchangeably herein, refers to polymers of nucleotides of any length, and include DNA and RNA. The nucleotides can be deoxyribonucleotides, ribonucleotides, modified nucleotides or bases, and\/or their analogs, or any substrate that can be incorporated into a polymer by DNA or RNA polymerase. A polynucleotide may comprise modified nucleotides, such as methylated nucleotides and their analogs. If present, modification to the nucleotide structure may be imparted before or after assembly of the polymer. The sequence of nucleotides may be interrupted by non-nucleotide components. A polynucleotide may be further modified after polymerization, such as by conjugation with a labeling component. Other types of modifications include, for example, \u201ccaps\u201d, substitution of one or more of the naturally occurring nucleotides with an analog, internucleotide modifications such as, for example, those with uncharged linkages (e.g., methyl phosphonates, phosphotriesters, phosphoamidates, carbamates, etc.) and with charged linkages (e.g., phosphorothioates, phosphorodithioates, etc.), those containing pendant moieties, such as, for example, proteins (e.g., nucleases, toxins, antibodies, signal peptides, poly-L-lysine, etc.), those with intercalators (e.g., acridine, psoralen, etc.), those containing chelators (e.g., metals, radioactive metals, boron, oxidative metals, etc.), those containing alkylators, those with modified linkages (e.g., alpha anomeric nucleic acids, etc.), as well as unmodified forms of the polynucleotide(s). Further, any of the hydroxyl groups ordinarily present in the sugars may be replaced, for example, by phosphonate groups, phosphate groups, protected by standard protecting groups, or activated to prepare additional linkages to additional nucleotides, or may be conjugated to solid supports. The 5\u2032 and 3\u2032 terminal OH can be phosphorylated or substituted with amines or organic capping groups moieties of from 1 to 20 carbon atoms. Other hydroxyls may also be derivatized to standard protecting groups. Polynucleotides can also contain analogous forms of ribose or deoxyribose sugars that are generally known in the art, including, for example, 2\u2032-O-methyl-2\u2032-O-allyl, 2\u2032-fluoro- or 2\u2032-azido-ribose, carbocyclic sugar analogs, a-anomeric sugars, epimeric sugars such as arabinose, xyloses or lyxoses, pyranose sugars, furanose sugars, sedoheptuloses, acyclic analogs and abasic nucleoside analogs such as methyl riboside. One or more phosphodiester linkages may be replaced by alternative linking groups. These alternative linking groups include, but are not limited to, embodiments wherein phosphate is replaced by P(O)S(\u201cthioate\u201d), P(S)S (\u201cdithioate\u201d), \u201c(O)NR 2 (\u201camidate\u201d), P(O)R, P(O)OR\u2032, CO or CH 2 (\u201cformacetal\u201d), in which each R or R\u2032 is independently H or substituted or unsubstituted alkyl (1-20 C) optionally containing an ether (\u2014O\u2014) linkage, aryl, alkenyl, cycloalkyl, cycloalkenyl or araldyl. Not all linkages in a polynucleotide need be identical. The preceding description applies to all polynucleotides referred to herein, including RNA and DNA.","\u201cOligonucleotide,\u201d as used herein, refers to short, single stranded polynucleotides that are at least about seven nucleotides in length and less than about 250 nucleotides in length. Oligonucleotides may be synthetic. The terms \u201coligonucleotide\u201d and \u201cpolynucleotide\u201d are not mutually exclusive. The description above for polynucleotides is equally and fully applicable to oligonucleotides.","The term \u201cprimer\u201d refers to a single stranded polynucleotide that is capable of hybridizing to a nucleic acid and allowing the polymerization of a complementary nucleic acid, generally by providing a free 3\u2032-OH group.","\u201cPredicting\u201d and \u201cprediction\u201d as used herein does not mean that the event will happen with 100% certainty. Instead it is intended to mean the event will more likely than not happen. Acts taken to \u201cpredict\u201d or \u201cmake a prediction\u201d can include the determination of the likelihood that an event will be more likely than not to happen. Assessment of multiple factors described herein can be used to make such determination or prediction.","By \u201ccorrelate\u201d or \u201ccorrelating\u201d is meant comparing, in any way, the performance and\/or results of a first analysis or protocol with the performance and\/or results of a second analysis or protocol. For example, one may use the results of a first analysis or protocol in carrying out a second protocols and\/or one may use the results of a first analysis or protocol to determine whether a second analysis or protocol should be performed. With respect to the embodiment of gene expression analysis or protocol, one may use the results of the gene expression analysis or protocol to determine whether a specific therapeutic regimen should be performed.","The term \u201cdiagnosis\u201d is used herein to refer to the identification or classification of a molecular or pathological state, disease or condition. For example, \u201cdiagnosis\u201d may refer to identification of a particular type of breast cancer, e.g., invasive ductal carcinoma (IDC). \u201cDiagnosis\u201d may also refer to the classification of a particular sub-type of breast cancer, e.g., by tissue involvement (e.g., lobular or ductal), by molecular features (e.g., a patient with histologically normal breast tissue).","The term \u201caiding diagnosis\u201d is used herein to refer to methods that assist in making a clinical determination regarding the presence, degree or other nature, of a particular type of symptom or condition of breast cancer. For example, a method of aiding diagnosis of breast cancer can comprise measuring the amount or detecting the presence or absence of one or more malignancy-risk genes in a biological sample from an individual. In another example, a method of aiding diagnosis of breast cancer can comprise measuring the amount or detecting the presence of one or more malignancy-risk genes in a biological sample from an individual.","The term \u201cprognosis\u201d is used herein to refer to the prediction of the likelihood of the development of breast cancer (including recurrence of breast cancer). The predictive methods of the invention can be used clinically to make treatment decisions by choosing the most appropriate treatment modalities for any particular patient. The predictive methods of the present invention are valuable tools in predicting if and\/or aiding in the diagnosis as to whether a patient is likely to develop breast cancer, have recurrence of breast cancer, and\/or metastasis of the cancer. Diagnosis of breast cancer may be made according to any protocol that one of skill of art would use, for example, those set by the College of American Pathology.","As used herein, \u201ctreatment\u201d refers to clinical intervention in an attempt to alter the natural course of the individual or cell being treated, and can be performed before or during the course of clinical pathology. Desirable effects of treatment include preventing the occurrence or recurrence of a disease or a condition or symptom thereof, alleviating a condition or symptom of the disease, diminishing any direct or indirect pathological consequences of the disease, decreasing the rate of disease progression, ameliorating or palliating the disease state, and achieving remission or improved prognosis. In some embodiments, methods and compositions of the invention are useful in attempts to delay development of a disease or disorder.","A \u201cmedicament\u201d is an active drug to treat a disease, disorder, and\/or condition. In one embodiment, the disease, disorder, and\/or condition is breast cancer or its symptoms or side effects associated with treatment of breast cancer.","Reference to \u201cabout\u201d a value or parameter herein includes (and describes) embodiments that are directed to that value or parameter per se. For example, description referring to \u201cabout X\u201d includes description of \u201cX.\u201d","It is understood that aspect and embodiments of the invention described herein include \u201cconsisting\u201d and\/or \u201cconsisting essentially of\u201d aspects and embodiments.","General Techniques","Unless defined otherwise, technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs.","The practice of the present invention will employ, unless otherwise indicated, conventional techniques of molecular biology (including recombinant techniques), microbiology, cell biology, biochemistry, and immunology, which are within the skill of the art. Such techniques are explained fully in the literature, such as, \u201cMolecular Cloning: A Laboratory Manual\u201d, second edition (Sambrook et al., 1989); \u201cOligonucleotide Synthesis\u201d (M. J. Gait, ed., 1984); \u201cAnimal Cell Culture\u201d (R. I. Freshney, ed., 1987); \u201cMethods in Enzymology\u201d (Academic Press, Inc.); \u201cCurrent Protocols in Molecular Biology\u201d (F. M. Ausubel et al., eds., 1987, periodic updates); \u201cPCR: The Polymerase Chain Reaction\u201d, (Mullis et al., eds., 1994); Singleton et al., Dictionary of Microbiology and Molecular Biology 2nd ed., J. Wiley & Sons (New York, N.Y. 1994), and March, Advanced Organic Chemistry Reactions, Mechanisms and Structure 4th ed., John Wiley & Sons (New York, N.Y. 1992), and \u201cThe Breast\u201d by Copeland Bland.","Primers, oligonucleotides and polynucleotides employed in the present invention can be generated using standard techniques known in the art.","Malignancy-Risk Signature","The invention provides for methods and malignancy-risk gene signatures for predicting the likelihood that an individual with histologically normal breast tissue will develop breast cancer. The invention also provides for methods and malignancy-risk gene signatures for predicting the likelihood that an individual with histologically normal breast tissue will develop a recurrence of breast cancer. The invention further provides for methods and malignancy-risk gene signatures for predicting the likelihood for the spread of breast cancer (e.g., metastasis) in an individual.","Accordingly, the invention provides for methods of using the malignancy-risk gene signatures disclosed here to assess cancer risk, cancer relapse, cancer progression and prognosis. The use of such signatures has significant benefits for the diagnosis and the aiding of diagnosis of an individual who is at risk for developing breast cancer, developing a recurrence of breast cancer and\/or metastasis of breast cancer. The knowledge that an individual will likely develop breast cancer enables physicians to take actions which are personalized to treat the patient. In some cases, this involves more careful monitoring and\/or testing of the individual. In other cases, it may involve a more aggressive plan of using chemotherapeutic agents or radiation therapy.","The methods and malignancy-risk gene signatures of this invention may be used for any individual in need of such assessment. In other embodiments, the individual is anyone who is at risk of having breast cancer (whether for the first time or recurring breast cancer). In one embodiment, the individual is a patient who has had breast cancer and had some type of treatment to remove or reduce the breast cancer. The treatment can be surgical procedure (e.g., resection, lumpectomy, mastectomy, lymph node dissection) alone or with some type of cancer therapy (e.g., chemotherapy, radiation therapy, hormone therapy, and\/or targeted therapy like Herceptin\u00ae). The individual may have had just cancer therapy alone. In one embodiment, the individual appears to have had all the breast cancer removed and the tissue adjacent to the cancer (i.e., ipsilateral to the cancer) is histologically normal.","In another embodiment, methods and malignancy-risk gene signatures of this invention may be used for an individual who has not been diagnosed with breast cancer but has a family history of breast cancer. The individual is a vertebrate. In certain embodiments, the vertebrate is a mammal. Mammals include, but are not limited to, primates (including human and non-human primates) and rodents (e.g., mice and rats). In certain embodiments, a mammal is a human. In one embodiment, the individual is female. In another embodiment, the individual is a male. In some cases, the individual is a \u201cpatient\u201d who is under the care of a treating physician.","In some cases, a patient subpopulation is first determined before determining the likelihood of those patients developing cancer. In one aspect, the patient subpopulation is comprises of females who have histologically normal tissue in one or both breasts. In some cases, the patient subpopulation is further subdivided by their medical history, such as what type of breast cancer they had (e.g., lobular vs. ductal, invasive carcinoma vs. in situ carcinoma), or the severity of the cancer. In other cases, the patients are further subdivided by their treatment history, etc. In one embodiment, a patient subpopulation is characterized by genetic signatures, including malignancy-risk gene signatures. The patient subpopulation can express at least about 10 malignancy-risk genes from the malignancy-risk gene signatures disclosed herein (e.g., from Table 6). In one embodiment, the patient subpopulation expresses at least about 15 malignancy-risk genes, at least about 20 malignancy-risk genes, at least about 25 malignancy-risk genes, at least about 30 malignancy-risk genes, at least about 35 malignancy-risk genes, at least about 40 malignancy-risk genes, at least about 45 malignancy-risk genes, at least about 50 malignancy-risk genes, at least about 55 malignancy-risk genes, at least about 60 malignancy-risk genes, at least about 70 malignancy-risk genes, at least about 75 malignancy-risk genes, at least about 80 malignancy-risk genes, at least about 85 malignancy-risk genes, at least about 90 malignancy-risk genes, at least about 95 malignancy-risk genes, at least about 100 malignancy-risk genes, at least about 105 malignancy-risk genes, at least about 110 malignancy-risk genes, at least about 115 malignancy-risk genes, at least about 120 malignancy-risk genes, at least about 125 malignancy-risk genes, at least about 130 malignancy-risk genes, at least about 135 malignancy-risk genes, or at least about 140 malignancy-risk genes.","Identification of Malignancy-Risk Signature","Identification of malignancy-risk gene signatures is detailed in the Examples section. See, for example,  for a flow chart. Such malignancy-risk gene signatures can be obtained by determining an IDC (invasive ductal carcinoma) signature by comparing the gene expression profiles from IDC patients to that of normal histological tissue adjacent to breast cancer in patients who have had cancer and had some type of procedure (e.g., surgical or cancer therapy) to remove the breast cancer. Statistical analysis as described in the Examples can then be performed to identify \u201cIDC-like genes\u201d (see, e.g., Example 13) from which malignancy-risk scores can be calculated (see, e.g., Example 13) and a malignancy-risk gene signature is then obtained. The 140 malignancy-risk genes are disclosed in Table 6.","Malignancy-risk genes can be cell proliferation genes, however, not all cell proliferation genes are malignancy-risk genes. Others are involved in cell adhesion, cell mitosis, DNA replication. The Figures and Tables further describe the malignancy-risk genes and their pathway roles.","Accordingly, in one embodiment, the invention encompasses methods of using the malignancy-risk genes of Table 6 to predict an individual's likelihood of developing breast cancer, having recurrent breast cancer or to have metastasis of breast cancer. In other embodiments, the invention encompasses the use of such this malignancy-risk gene signature to assess an individual's risk of developing or having recurring breast cancer and\/or metastasis. In some embodiments, at least about 10 genes from the 140 malignancy-risk genes in Table 6 are used for this determination. In other embodiments, at least about 15 malignancy-risk genes are used. In yet other embodiments, at least about 20 malignancy-risk genes, at least about 25 malignancy-risk genes, at least about 30 malignancy-risk genes, at least about 35 malignancy-risk genes, at least about 40 malignancy-risk genes, at least about 45 malignancy-risk genes, at least about 50 malignancy-risk genes, at least about 55 malignancy-risk genes, at least about 60 malignancy-risk genes, at least about 70 malignancy-risk genes, at least about 75 malignancy-risk genes, at least about 80 malignancy-risk genes, at least about 85 malignancy-risk genes, at least about 90 malignancy-risk genes, at least about 95 malignancy-risk genes, at least about 100 malignancy-risk genes, at least about 105 malignancy-risk genes, at least about 110 malignancy-risk genes, at least about 115 malignancy-risk genes, at least about 120 malignancy-risk genes, at least about 125 malignancy-risk genes, at least about 130 malignancy-risk genes, or at least about 135 malignancy-risk genes are used to make these determinations of risk of developing breast cancer, having recurrent breast cancer or to have metastasis of breast cancer.","The invention also provides for methods for using the malignancy-risk genes of Table 3 to predict an individual's likelihood of developing breast cancer, having recurrent breast cancer or to have metastasis of breast cancer. In other embodiments, the invention encompasses the use of such this malignancy-risk gene signature to assess an individual's risk of developing or having recurring breast cancer and\/or metastasis. In some embodiments, at least about 5 genes from the malignancy-risk genes in Table 3 are used for this determination. In other embodiments, at least about 6, 7, 8, 9, or 10 malignancy-risk genes are used. In yet other embodiments, at least about 11, 12, 13, 14, 15, 16, 17, 18, 19, or 20 malignancy-risk genes are used. In yet other embodiments, at least about 21, 22, 23, 24, 25, 26, 27, 28, 29, or 30 malignancy-risk genes are used.","In another embodiment, the invention encompasses the malignancy-risk gene signature of Table 3 wherein the gene signature does not include at least about 1 gene selected from the genes listed in , B, A, and A. In other embodiments, the invention encompasses the malignancy-risk gene signature of Table 3 wherein the gene signature does not include at least about 2, 3, 4, 5, 6, 7, 8, 9, or 10 genes selected from the genes listed in , B, A, and A. In other embodiments, the invention encompasses the malignancy-risk gene signature of Table 3 wherein the gene signature does not include at least about 15, 20, 30 or more genes selected from the genes listed in , B, A, and A.","In another aspect of the invention, the invention provides for a malignancy-risk gene signature which comprises at least about 10 genes selected from the group consisting of the genes in Table 3, Table 7, Table 8, , , and . In another embodiment of the invention, the invention provides for a malignancy-risk gene signature which comprises at least about 20, 25, 30, 35, 40, 50 or more genes selected from the group consisting of the genes in Table 3, Table 7, Table 8, , , , and .","In other embodiments, each of the following genes: topoisomerase-2, Bub-1 and MDM-2, can be used as a predictor of developing breast cancer, having recurrent breast cancer or to have metastasis of breast cancer.","In another embodiment, the malignancy-risk signature is at least one or more genes selected from the group consisting of the following genes from Table 6: APOBEC3B, C6orf115, CASC5, CORO2A, FAM83D, HIST1H2BF, MOBK1B, WDSOF1, IQGAP3, AMOTL2, CCDC50, CHRDL1, GPRASP1, MAGI2, RBMS3, SYNPO2, Affymetrix probe set id 213158_at, Affymetrix probe set id 226250_at, Affymetrix probe set id 226252_at, Affymetrix probe set id 227082_at, Affymetrix probe set id 227121_at, Affymetrix probe set id 227646_at, Affymetrix probe set id 227719_at, Affymetrix probe set id 235556_at, Affymetrix probe set id 235570_at, and Affymetrix probe set id 243584_at.","In another embodiment, the malignancy-risk signature is at least about 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 or 15 more genes selected from the group consisting of the following genes from Table 6: APOBEC3B, C6orf115, CASC5, CORO2A, FAM83D, HIST1H2BF, MOBK1B, WDSOF1, IQGAP3, AMOTL2, CCDC50, CHRDL1, GPRASP1, MAGI2, RBMS3, SYNPO2, Affymetrix probe set id 213158_at, Affymetrix probe set id 226250_at, Affymetrix probe set id 226252_at, Affymetrix probe set id 227082_at, Affymetrix probe set id 227121_at, Affymetrix probe set id 227646_at, Affymetrix probe set id 227719_at, Affymetrix probe set id 235556_at, Affymetrix probe set id 235570_at, and Affymetrix probe set id 243584_at.","The malignancy-risk gene signature can also be at least one or more of the malignancy-risk genes listed in Table 7. In another embodiment, the malignancy-risk signature is at least about 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 or 15 more genes selected from the genes of Table 7. In another embodiment, the malignancy-risk signature is at least about 20, 25, 30 or 35 or more genes selected from the genes of Table 7.","The malignancy-risk gene signature can also be at least one or more of the malignancy-risk genes listed in Table 8. In another embodiment, the malignancy-risk signature is at least about 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 or 15 more genes selected from the genes of Table 8. In another embodiment, the malignancy-risk signature is at least about 16, 17, 18, 19, 20, 21, 22, 23, 24 or more genes selected from the genes of Table 8.","In one aspect, the malignancy-risk gene signature does not include at least one of the genes in . In other embodiments, the malignancy-risk gene signature does not include at least about 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 or 15 or more of the genes in .","In one aspect, the malignancy-risk gene signature does not include at least about one of the genes in . In other embodiments, the malignancy-risk gene signature does not include at least about 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 or 15 or more of the genes in .","In one aspect, the malignancy-risk gene signature does not include at least about one of the genes in . In other embodiments, the malignancy-risk gene signature does not include at least about 2, 3, 4, 5, 6 or more of the genes in .","In another aspect, the malignancy-risk gene signature can also be at least about one or more of the genes in . In another embodiment, the malignancy-risk signature is at least about 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 or 15 more genes selected from the genes of . In another embodiment, the malignancy-risk signature is at least about 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 30, 35, 40, 45 or more genes selected from the genes of .","In another aspect, the malignancy-risk gene signature can also be at least about one or more of the genes in . In another embodiment, the malignancy-risk signature is at least about 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 or 15 more genes selected from the genes of . In another embodiment, the malignancy-risk signature is at least about 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 30, 35, 40, 45, 47 or more genes selected from the genes of .","In one aspect, the malignancy-risk gene signature does not include at least about one of the genes in . In other embodiments, the malignancy-risk gene signature does not include at least about 2, 3, or 4 of the genes in .","In another aspect, the malignancy-risk gene signature can also be at least about one or more of the genes in . In another embodiment, the malignancy-risk signature is at least about 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 or 15 more genes selected from the genes of . In another embodiment, the malignancy-risk signature is at least about 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 30, 35, 40, 45, 50, 55, 60, 62 or more genes selected from the genes of .","In another aspect, the malignancy-risk gene signature can also be at least about one or more of the genes in . In another embodiment, the malignancy-risk signature is at least about 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 or 15 more genes selected from the genes of . In another embodiment, the malignancy-risk signature is at least about 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 30, 32 or more genes selected from the genes of .","The invention also comprises methods of identifying additional malignancy-risk genes and those additional malignancy-risk genes for inclusion in the malignancy-risk gene signature by using the methology described herein.","Arrays and Gene Chips and Kits Comprising Thereof","Arrays and microarrays which contain the malignancy-risk gene signatures as described herein for assessing the likelihood of developing cancer, cancer relapse, cancer progression, prognosis, and\/or metastasis are also encompassed within the scope of this invention. Methods of making arrays are well-known in the art and as such, do not need to be described in detail here.","Such arrays can contain the profiles of at least about 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, 100, 105, 110, 115, 120, 125, 130, 135, or 140 genes as disclosed in the Figures and Tables. Accordingly, arrays for assessing the likelihood of developing cancer, cancer relapse, cancer progression, and\/or metastasis can be customized for prognosis, diagnosis, aiding in the diagnosis or treatment of breast cancer. The array can be packaged as part of kit comprising the customized array itself and a set of instructions for how to use the array to determine an individual's likelihood of developing cancer, cancer relapse, cancer progression, and metastasis.","Also provided are reagents and kits thereof for practicing one or more of the above described methods. The subject reagents and kits thereof may vary greatly. Reagents of interest include reagents specifically designed for use in production of the above described malignancy-risk gene signatures.","One type of such reagent is an array probe of nucleic acids, such as a DNA chip, in which the malignancy-risk gene signatures are represented. A variety of different array formats are known in the art, with a wide variety of different probe structures, substrate compositions and attachment technologies. Representative array structures of interest include those described in U.S. Pat. Nos. 5,143,854; 5,288,644; 5,324,633; 5,432,049; 5,470,710; 5,492,806; 5,503,980; 5,510,270; 5,525,464; 5,547,839; 5,580,732; 5,661,028; 5,800,992; the disclosures of which are herein incorporated by reference; as well as WO 95\/21265; WO 96\/31622; WO 97\/10365; WO 97\/27317; EP 373 203; and EP 785 280. It is to be understood that the invention also encompasses detection of protein expression of the malignancy-risk genes as described herein. Detection of protein expression is known to one of skill in the art and may be done using any number of commercially available products.","The DNA chip is convenient to compare the expression levels of a number of genes at the same time. DNA chip-based expression profiling can be carried out, for example, by the method as disclosed in \u201cMicroarray Biochip Technology\u201d (Mark Schena, Eaton Publishing, 2000). A DNA chip comprises immobilized high-density probes to detect a number of genes. Thus, the expression levels of many genes can be estimated at the same time by a single-round analysis. Namely, the expression profile of a specimen can be determined with a DNA chip. A DNA chip may comprise probes, which have been spotted thereon, to detect the expression level of the malignancy-risk gene signatures of the present invention.","A probe may be designed for each malignancy-risk gene selected, and spotted on a DNA chip. Such a probe may be, for example, an oligonucleotide comprising 5-50 nucleotide residues. A method for synthesizing such oligonucleotides on a DNA chip is known to those skilled in the art. Longer DNAs can be synthesized by PCR or chemically. A method for spotting long DNA, which is synthesized by PCR or the like, onto a glass slide is also known to those skilled in the art. A DNA chip that is obtained by the method as described above can be used to determine the likelihood that an individual will develop breast disease, develop breast cancer, have recurrence of breast cancer, and\/or have metastasis of breast cancer according to the present invention.","DNA microarray and methods of analyzing data from microarrays are well-described in the art, including in , Ed. by Bowtel and Sambrook (Cold Spring Harbor Laboratory Press, 2002); by Kohana (MIT Press, 2002); , by Knudsen (Wiley, John & Sons, Incorporated, 2002); , Vol. 205 by Schema (Oxford University Press, 1999); and , ed. by Lin et al. (Kluwer Academic Publishers, 2002).","One aspect of the invention provides a gene chip having a plurality of different oligonucleotides attached to a first surface of the solid support and having specificity for a plurality of genes, wherein at least about 50% of the genes are common to those of the malignancy-risk genes in Table 6. In one embodiment, at least about 70%, 80%, 90% or 95% of the genes in the gene chip are common to those of the malignancy-risk genes in Table 6.","In another aspect of the invention, the invention provides a gene chip having a plurality of different oligonucleotides attached to a first surface of the solid support and having specificity for a plurality of genes, wherein at least about 50% of the genes are common to those of the malignancy-risk genes in Table 3. In one embodiment, at least about 70%, 80%, 90% or 95% of the genes in the gene chip are common to those of the malignancy-risk genes in Table 3.","In another aspect of the invention, the invention provides a gene chip having a plurality of different oligonucleotides attached to a first surface of the solid support and having specificity for a plurality of genes, wherein at least about 50% of the genes are common to those of the malignancy-risk genes in Table 7. In one embodiment, at least about 70%, 80%, 90% or 95% of the genes in the gene chip are common to those of the malignancy-risk genes in Table 7.","In another aspect of the invention, the invention provides a gene chip having a plurality of different oligonucleotides attached to a first surface of the solid support and having specificity for a plurality of genes, wherein at least about 50% of the genes are common to those of the malignancy-risk genes in Table 8. In one embodiment, at least about 70%, 80%, 90% or 95% of the genes in the gene chip are common to those of the malignancy-risk genes in Table 8.","In another aspect of the invention, the invention provides a gene chip having a plurality of different oligonucleotides attached to a first surface of the solid support and having specificity for a plurality of genes, wherein at least about 50% of the genes are common to those of the malignancy-risk genes in , B, B, or A. In one embodiment, at least about 70%, 80%, 90% or 95% of the genes in the gene chip are common to those of the malignancy-risk genes in , B, B, or A.","In another aspect of the invention, the invention provides a gene chip having a plurality of different oligonucleotides attached to a first surface of the solid support and having specificity for a plurality of genes, wherein at least about 50% of the genes are common to those of the malignancy-risk genes in Table 6 but does not include at least about one of the genes in . In one embodiment, at least about 70%, 80%, 90% or 95% of the genes in the gene chip are common to those of the malignancy-risk genes in Table 6 but does not include at least about one of the genes in . In other embodiments, the gene chip includes at least about 70%, 80%, 90% or 95% of the genes in the gene chip are common to those of the malignancy-risk genes in Table 6 but does not include at least about 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 or 15 or more of the genes in .","In another aspect, the invention provides a gene chip having a plurality of different oligonucleotides attached to a first surface of the solid support and having specificity for a plurality of genes, wherein at least about 50% of the genes are common to those of the malignancy-risk genes in Table 6 but does not include at least about one of the genes in . In other embodiments, the malignancy-risk gene signature does not include at least about 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 or 15 or more of the genes in .","In another aspect, the invention provides a gene chip having a plurality of different oligonucleotides attached to a first surface of the solid support and having specificity for a plurality of genes, wherein at least about 50% of the genes are common to those of the malignancy-risk genes in Table 6 but does not include at least about one of the genes in . In other embodiments, the malignancy-risk gene signature does not include at least about 2, 3, 4, 5, 6 or more of the genes in .","In one aspect, the gene chip comprises at least about 10 genes from the 140 malignancy-risk genes in Table 6. In other embodiments, the gene chip comprises at least about 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 70, 75, 80, 85, 90, 95, 100, 105, 110, 115, 120, 125, 130, 135 or more malignancy-risk genes are used to make these determinations of risk of developing breast cancer, having recurrent breast cancer or to have metastasis of breast cancer.","In another aspect, the gene chip comprises at least about 10 genes selected from the list consisting of: APOBEC3B, C6orf115, CASC5, CORO2A, FAM83D, HIST1H2BF, MOBK1B, WDSOF1, IQGAP3, AMOTL2, CCDC50, CHRDL1, GPRASP1, MAGI2, RBMS3, SYNPO2, Affymetrix probe set id 213158_at, Affymetrix probe set id 226250_at, Affymetrix probe set id 226252_at, Affymetrix probe set id 227082_at, Affymetrix probe set id 227121_at, Affymetrix probe set id 227646_at, Affymetrix probe set id 227719_at, Affymetrix probe set id 235556_at, Affymetrix probe set id 235570_at, and Affymetrix probe set id 243584_at. In other embodiments, the gene chip comprises at least about 15, 20, 25, or more selected from the list consisting of: APOBEC3B, C6orf115, CASC5, CORO2A, FAM83D, HIST1H2BF, MOBK1B, WDSOF1, IQGAP3, AMOTL2, CCDC50, CHRDL1, GPRASP1, MAGI2, RBMS3, SYNPO2, Affymetrix probe set id 213158_at, Affymetrix probe set id 226250_at, Affymetrix probe set id 226252_at, Affymetrix probe set id 227082_at, Affymetrix probe set id 227121_at, Affymetrix probe set id 227646_at, Affymetrix probe set id 227719_at, Affymetrix probe set id 235556_at, Affymetrix probe set id 235570_at, and Affymetrix probe set id 243584_at. In another embodiment, the gene chip comprises all of the following genes: the list consisting of: APOBEC3B, C6orf115, CASC5, CORO2A, FAM83D, HIST1H2BF, MOBK1B, WDSOF1, IQGAP3, AMOTL2, CCDC50, CHRDL1, GPRASP1, MAGI2, RBMS3, SYNPO2, Affymetrix probe set id 213158_at, Affymetrix probe set id 226250_at, Affymetrix probe set id 226252_at, Affymetrix probe set id 227082_at, Affymetrix probe set id 227121_at, Affymetrix probe set id 227646_at, Affymetrix probe set id 227719_at, Affymetrix probe set id 235556_at, Affymetrix probe set id 235570_at, and Affymetrix probe set id 243584_at.","One aspect of the invention provides a kit comprising: (a) any of the gene chips described herein; and (b) one of the computer-readable mediums described herein.","In some embodiments, the arrays include probes for at least about 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 40, 50, 60, 70, 80, 90, 100, 110, 120, 130, or 140 of the genes listed in Table 3, Table 6, Table 7, Table 8, , , , and\/or . Where the subject arrays include probes for additional genes not listed in the tables, in certain embodiments the number % of additional genes that are represented does not exceed about 50%, 40%, 30%, 20%, 15%, 10%, 8%, 6%, 5%, 4%, 3%, 2% or 1%.","The kits of the subject invention may include the above described arrays. The kits may further include one or more additional reagents employed in the various methods, such as primers for generating target nucleic acids, dNTPs and\/or rNTPs, which may be either premixed or separate, one or more uniquely labeled dNTPs and\/or rNTPs, such as biotinylated or Cy3 or Cy5 tagged dNTPs, gold or silver particles with different scattering spectra, or other post synthesis labeling reagent, such as chemically active derivatives of fluorescent dyes, enzymes, such as reverse transcriptases, DNA polymerases, RNA polymerases, and the like, various buffer mediums, e.g. hybridization and washing buffers, prefabricated probe arrays, labeled probe purification reagents and components, like spin columns, etc., signal generation and detection reagents, e.g. streptavidin-alkaline phosphatase conjugate, chemifluorescent or chemiluminescent substrate, and the like.","In addition to the above components, the kits will further include instructions for practicing the methods and arrays described herein. These instructions may be present in the kits in a variety of forms, one or more of which may be present in the kit. One form in which these instructions may be present is as printed information on a suitable medium or substrate, e.g., a piece or pieces of paper on which the information is printed, in the packaging of the kit, in a package insert, etc. Yet another means would be a computer readable medium, e.g., diskette, CD, etc., on which the information has been recorded. Yet another means that may be present is a website address which may be used via the internet to access the information at a removed site. Any convenient means may be present in the kits.","Computer Readable Media Comprising Gene Expression Profiles","The invention also contemplates computer readable media that comprises malignancy-risk gene signatures. Such media can contain all of part of the malignancy-risk gene signatures of the genes listed in Table 3, Table 6, Table 7, Table 8, , , , and\/or . The media can be a list of the genes or contain the raw data for running a user's own statistical calculation, such as the methods disclosed herein.","Program Products\/Systems","Another aspect of the invention provides a program product (i.e., software product) for use in a computer device that executes program instructions recorded in a computer-readable medium to perform one or more steps of the methods described herein, such for assessing the likelihood that an individual will develop breast cancer, have recurrence of breast cancer, and\/or metastasis.","On aspect of the invention provides a computer readable medium having computer readable program codes embodied therein, the computer readable medium program codes performing one or more of the following functions: defining the value of one or more risk values from the expression levels genes; calculating the risk of developing breast cancer, calculating the risk of developing breast cancer recurrence, and calculating the risk of developing metastasis.","Another related aspect of the invention provides kits comprising the program product or the computer readable medium, optionally with a computer system. On aspect of the invention provides a system, the system comprising: a computer; a computer readable medium, operatively coupled to the computer, the computer readable medium program codes performing one or more of the following functions: defining the value of one or more risk value from the expression levels genes; calculating the risk of developing breast cancer, calculating the risk of developing breast cancer recurrence, and calculating the risk of developing metastasis.","In one embodiment, the program product comprises: a recordable medium; and a plurality of computer-readable instructions executable by the computer device to analyze data from the array hybridization steps, to transmit array hybridization from one location to another, or to evaluate genome-wide location data between two or more genomes. Computer readable media include, but are not limited to, CD-ROM disks (CD-R, CD-RW), DVD-RAM disks, DVD-RW disks, floppy disks and magnetic tape.","A related aspect of the invention provides kits comprising the program products described herein. The kits may also optionally contain paper and\/or computer-readable format instructions and\/or information, such as, but not limited to, information on DNA microarrays, on tutorials, on experimental procedures, on reagents, on related products, on available experimental data, on using kits, on chemotherapeutic agents including there toxicity, and on other information. The kits optionally also contain in paper and\/or computer-readable format information on minimum hardware requirements and instructions for running and\/or installing the software. The kits optionally also include, in a paper and\/or computer readable format, information on the manufacturers, warranty information, availability of additional software, technical services information, and purchasing information. The kits optionally include a video or other viewable medium or a link to a viewable format on the internet or a network that depicts the use of the use of the software, and\/or use of the kits.","The analysis of data, as well as the transmission of data steps, can be implemented by the use of one or more computer systems. Computer systems are readily available. The processing that provides the displaying and analysis of image data for example, can be performed on multiple computers or can be performed by a single, integrated computer or any variation thereof. For example, each computer operates under control of a central processor unit (CPU), such as a \u201cPentium\u201d microprocessor and associated integrated circuit chips, available from Intel Corporation of Santa Clara, Calif., USA. A computer user can input commands and data from a keyboard and display mouse and can view inputs and computer output at a display. The display is typically a video monitor or flat panel display device. The computer also includes a direct access storage device (DASD), such as a fixed hard disk drive. The memory typically includes volatile semiconductor random access memory (RAM).","Each computer typically includes a program product reader that accepts a program product storage device from which the program product reader can read data (and to which it can optionally write data). The program product reader can include, for example, a disk drive, and the program product storage device can include a removable storage medium such as, for example, a magnetic floppy disk, an optical CD-ROM disc, a CD-R disc, a CD-RW disc and a DVD data disc. If desired, computers can be connected so they can communicate with each other, and with other connected computers, over a network. Each computer can communicate with the other connected computers over the network through a network interface that permits communication over a connection between the network and the computer.","The computer operates under control of programming steps that are temporarily stored in the memory in accordance with conventional computer construction. When the programming steps are executed by the CPU, the pertinent system components perform their respective functions. Thus, the programming steps implement the functionality of the system as described above. The programming steps can be received from the DASD, through the program product reader or through the network connection. The storage drive can receive a program product, read programming steps recorded thereon, and transfer the programming steps into the memory for execution by the CPU. As noted above, the program product storage device can include any one of multiple removable media having recorded computer-readable instructions, including magnetic floppy disks and CD-ROM storage discs. Other suitable program product storage devices can include magnetic tape and semiconductor memory chips. In this way, the processing steps necessary for operation can be embodied on a program product.","Alternatively, the program steps can be received into the operating memory over the network. In the network method, the computer receives data including program steps into the memory through the network interface after network communication has been established over the network connection by well known methods understood by those skilled in the art. The computer that implements the client side processing, and the computer that implements the server side processing or any other computer device of the system, can include any conventional computer suitable for implementing the functionality described herein.","It will be apparent to those of ordinary skill in the art that methods involved in the present invention may be embodied in a computer program product that includes a computer usable and\/or readable medium. For example, such a computer usable medium may consist of a read only memory device, such as a CD ROM disk or conventional ROM devices, or a random access memory, such as a hard drive device or a computer diskette, having a computer readable program code stored thereon.","The following examples are provided to illustrate aspects of the invention but are not intended to limit the invention in any manner.","Tissue Collection","Tissues were collected in accordance with the protocols approved by the Institutional Review Board of the University of South Florida, and stored in the tissue bank of Moffitt Cancer Center. Breast tissues from patients that underwent mastectomy at various stages of breast carcinoma were collected and frozen in liquid nitrogen. The tissues were embedded in Tissue-Tek\u00ae O.C.T., 5-\u03bcm sections cut and mounted on Mercedes Platinum StarFrost\u2122 Adhesive slides. The slides were stained using a standard H&E protocol, and tissue boundaries marked. Using the marked slide as a \u201cmap\u201d, tissues were microdissected. Adipose tissues were trimmed away; the tumor and \u201cnormal\u201d tissues were separated and stored in liquid nitrogen.","Histology","Histological examination of all tissue sections and microdissection of samples were conducted by pathologist to ensure consistency in the clinical diagnoses. From a large invasive breast cancer database, a set of 42 histologically invasive ductal carcinomas (IDC) were identified with various histologic grades (the modified Bloom and Richardson grading). In addition to 42 IDCs, 143 \u2018histologically normal breast\u2019 tissues were selected which were free of any other breast lesions.","RNA Extraction","Total RNA was extracted from breast tissues using the Trizol method. Briefly, tissues were ground in liquid nitrogen, resuspended in 5 ml of lysis buffer and incubated for 3 min. at room temperature and centrifuged at 11,500 g for 15 minutes at 4\u00b0. The aqueous phase was removed and put into another tube with 2.5 ml of isopropanol, mixed well and set at \u221220\u00b0 C. for 20 minutes. DNA was pelleted by centrifuging at 11,500 g for 10 minutes at 4\u00b0 C. The pellet was washed with 75% ethanol and resuspended in 100 \u03bcl of deionized water. The amount of RNA was quantitated by measuring A.","Statistical Methods","Statistical analysis was done by performing a comparison of normal breast and IDC tissues to develop IDC gene signature. Outlier tissues (potential high risk normal breast tissue) were then identified from the normal breast tissue using the IDC gene signature and used to develop outlier gene signature from the outlier tissues. A pathway analysis was performed for IDC and outlier gene signatures to demonstrate the uniqueness of the outlier gene signature. Finally, the outlier gene signature was validated regarding the features of cancer risk prediction, disease progression, and prognosis, in a series of external gene expression datasets for breast cancer.","Briefly, in the first step, Statistical Analysis of Microarray (SAM') was used to develop IDC gene signature which discriminates between the normal and IDC tissues. In the second step, this IDC gene signature was used as reference and applied the outlier tissue approach (OTA) to identify histologically normal breast tissues that had acquired the molecular fingerprint of IDC, and these tissues are referred to as outlier normal tissues to reflect their potential risk for tumor development. The OTA ranks all the normal tissues for each gene. If a normal tissue has its percentile rank over 80% for a majority of up-regulated genes (and\/or below 20% for most down-regulated genes), it was considered an outlier normal tissue. An outlier normal tissue tends to show higher level of expression of these genes (up- or down-regulated) and greater likelihood of cancer development. In the third step, once outlier normal tissues were identified, a percentile rank approach was used to find a common set of genes, \u201coutlier genes\u201d. Specifically, genes with expression percentile rank of greater than 80% (or less than 20%) in most outlier normal tissues were selected as outlier genes. Pathway analysis was done using MetaCore\u2122 by GeneGo in the step 4. The validation step (step 5) was implemented by first identifying overlapped outlier genes, and then performing principal component analysis (PCA) for prediction.","An IDC gene signature (1,554 probe sets: 1038 unique genes) was first developed from a set of 42 IDC and 143 normal breast tissues. This analysis was done using Statistical Analysis of Microarrayand based on a cutoff of false discovery rate (FDR) <0.01 and a fold change >2. Pathway analysis revealed two predominant cellular processes: cell cycle and cell adhesion, as seen in . There were 10 cell adhesion pathways and 7 cell cycle pathways with a significant p-value <0.01. A majority of the genes were down-regulated in the cell adhesion, but up-regulated in the cell cycle.","11 outlier breast tissues were identified using the outlier tissue approach (see, for example, Methods section infra) to re-evaluate the 143 normal breast tissues whose gene expression profiles more closely approximated that of the IDC samples rather than the rest of the 132 normal breast tissues. Eight of these 11 outlier tissues had a median percentile rank greater than 80% among all the 143 normal tissues (i.e., the top 20%) at the 2, 3, and 4 fold-change cutoffs, shown in . The other 3 outlier tissues had a median percentile rank less than 20% (i.e., the bottom 20%) for the under-expressed probe sets. Distinction of the outlier tissues from the normal breast tissues was further demonstrated in .",{"@attributes":{"id":"p-0154","num":"0153"},"figref":["FIG. 4","FIG. 5","FIG. 6"]},"An outlier gene signature was developed by forming a \u201ccommon set\u201d of genes whose expression varied (up or down) at high levels in the 11 outlier tissues. The outlier genes consisted of 109 up-regulated probe sets (96 unique genes) and 31 down-regulated probe sets (21 unique genes). Of the entire list, presented in  and Table 6, a subset of outlier genes was selected, seen in  and Table 7. Expression of the outlier genes discriminated the normal, outlier, and IDC tissues from each other. See .","Pathway analysis showed that the outlier gene set was remarkably over-represented by cell cycle genes. There were 11 cell cycle related pathways represented in the outlier signature (p value <0.01), depicted in . This result was distinct from the IDC gene signature which had the cell adhesion as the primary components and cell cycle as a secondary component. Since the outlier gene signature was derived from the IDC gene signature, the difference in functional classes of genes would not have been expected in the absence of a selection bias. The majority of the outlier genes were classified to be primarily associated with DNA replication and mitosis, two hallmark events associated with proliferation. See . This observation may indicate the importance of these features in early stages of tumorigenesis. Importantly, this class of mitotic genes is well known to exhibit periodic expression at the transcriptional level in cultured cells, and previous studies have reported that genes related to S-phase and mitosis are also found highly expressed in tumors in cases where there is a relatively high fraction of cycling cells.","Principal component analysis (PCA) was applied for the outlier gene signature using all IDC and normal breast tissues (excluding the outlier and the adjacent breast tissues) to calculate the first PCA score for IDC, normal, outlier, the adjacent normal tissues.  showed that, the median of the first PCA score was highest in the outlier tissues followed by the adjacent normal tissues (p value=0.0015 based on t-test for the outlier versus the adjacent normal tissues). The normal breast tissue had the lowest score (p value=0.011 for the comparison of the adjacent normal tissues to the normal breast tissues).","Examples 8-12 below describe the external validation of the inventors' malignancy-risk signature. The value of the outlier gene signature was assessed on five external, independent data sets. These external datasets permitted the evaluation of a number of properties of the outlier signature including differentiation of normal versus IDC tissues, disease progression, cancer risk, and metastasis.","This study examined 10 patients (5 IDCs and 5 ILCs) and collected one tumor tissue (IDC or ILC) with two normal tissues (ductal or lobular cells) from each subject. The PCA model was applied to obtain the first PCA score for the 5 IDCs and the associated 10 normal breast tissues. Results showed the first PCA score was higher in IDC than in normal tissue within the same patient (p value=0.029 based on the random effect model to control for subject variation). See  and Table 8. This result indicated the outlier gene signature was able to differentiate IDC from normal tissue.","A set of 23 DCIS samples were collected to evaluate the disease progression feature of the outlier gene signature. Based on the PCA model derived from the previous 132 normal breast and 42 IDC tissues (excluding the 11 outlier tissues), the first PCA score were calculated for the DCIS tissues. Results showed a clear progression pattern from normal breast, outlier, DCIS, to IDC, shown in ). Ranking the disease status from 0 to 3 for normal breast to IDC, the calculated Pearson or Spearman correlation was 0.87 and 0.8, respectively, with a significant p value <0.001.","Further analysis using logistic regression model (with the normal group as the control group) yielded a significant association (OR=1.73, 2, and 1.76 for IDC-like normal, DCIS, and IDC, respectively, with p value <0.0001). Furthermore, the malignancy-risk score (equivalent of PCA score) of DCIS was lower than IDC, but higher than normal tissue (p=0.0005 based on one sample t-test using the difference of the maximum risk score in the normal tissues and the minimum risk score in the DCIS samples) within each patient ().","Eight ADH, 30 DCIS, and 23 IDC samples were collected and cDNA microarray used to generate expression data. There were 21 genes overlapping with the outlier gene signature. Univariate analysis of these 21 genes showed a majority of them with a statistically significant fold change (>2), shown in ) to ). Moreover, 16 genes showed an increasing pattern from ADH to IDC based on the first PCA score. ) and (), adjusted p value <0.05 among the 3 pair-wise comparisons.","Background: To further evaluate the potential of the malignancy-risk signature to predict the risk of cancer progression, the Ma et al. data were analyzed. The study collected 8 atypical ductal hyperplastic (ADH), 30 DCIS, and 23 IDC samples and used cDNA microarray to generate expression data. There were 21 genes in common with the malignancy-risk gene signature. We compared the malignancy-risk score among the three groups: ADH, DCIS, and IDC.","Data Analysis: We used the 21 genes to calculate the malignancy-risk genes (see Statistical Methods, see Example 13). Correlation analysis was used to assess cancer progression and logistics regression model was used to examine the association of the malignancy-risk score with cancer status.","Results: Correlation analysis showed an increasing pattern of the risk score with cancer progression from ADH to IDC (: the first panel). Pearson or Spearman correlation coefficient was 0.5, with a significant p value <0.0001 by ranking the cancer status from 1 to 3 for ADH to IDC. Pair-wise comparison showed that the risk score was statistically significant difference between IDC\/DCIS and ADH (adjusted p value=0.0001, and 0.0147 for IDC and DCIS, respectively). Univariate correlation analysis also showed 16 genes with a p value <0.05 (). Further analysis using logistics regression model (with the ADH group as the control group) demonstrated a strong association (OR=2.28 and 3.31 for DCIS and IDC with p value=0.016 and 0.008, respectively).","Four ADH tissues were collected from patients without a history of breast cancer, and another 4 ADH tissues from patients where breast cancer developed, labeled as ADHC. There were 102 probe sets that overlapped with the outlier gene signature. PCA was applied to these 102 probe sets for the 4 ADHs and 4 ADHCs. The results showed that the ADHC group had a higher score than the ADH group for the first PCA score, seen in . The majority of ADHC tissues (3 out of 4) yielded a score above 5, in contrast to most ADH tissues, which had negative scores. The ADH group had a narrow range of scores between \u22121 and 2 in the second PCA, while ADHC yielded a wide range of scores. As a result, the first two PCA scores in the scatter plot were able to distinguish between ADH and ADHC.","Background: This study was selected in order to assess the potential of the malignancy-risk score to predict the risk of future cancer development in the breast associated with ADH. This study collected 4 ADH tissues from patients without a history of breast cancer (we labeled these tissues as ADHN), and another 4 ADH tissues from patients where breast cancer developed (we labeled these tissues as ADHC). There were 102 probe sets in common with the malignancy-risk gene signature. We compared the malignancy-risk score between the two groups (ADHN and ADHC).","Data analysis: We used the 102 probe sets to calculate malignancy-risk score. Logistic regression model was used to assess cancer risk. This was done in two ways. One was to use the median risk score to dichotomize patients into two risk groups (high risk with score >median and low risk with score <median) to calculate odds ratio (OR). The other way was to use the continuous risk score as the independent variable to calculate OR and to estimate the cancer-risk probability. We also performed two-sample t-test for the malignancy-risk score and for each malignancy-risk gene (univariate analysis).","Results: Analysis results from logistic regression model showed that the ADHC group had a higher risk score than the ADHN group () although the odds ratio was not statistically significant (OR=1.4 with p=0.123 for the continuous risk score and OR=9.0 with p=0.178 for the median-cutoff risk score) due to a very limited sample size (n=4 per group). Notably, three out the four ADHC patients had a risk score above 5 with a cancer-risk Probability >0.8 (based on the continuous risk score), in contrast to most ADHN patients with negative scores and a low cancer-risk probability. Two-sample t-test for the malignancy-risk score yielded a p value=0.08. For univariate analysis, there were 16 genes with p value <0.05 ().","78 breast cancer patient samples were collected and the time to metastasis was determined using a 70 gene signature and cDNA array. An independent set of 295 cancer patients were collected as validation for the 70 gene signature. The two datasets were used to examine if the outlier genes can predict metastasis. There were 117 features that overlapped with the outlier gene signature. First PCA scores were calculated based on these 117 features. The patients were grouped into low and high-risk groups by dichotomizing the first PCA score based on kmeanand the recursive partitioning and regression tree method. Logrank tests showed a significant separation between the two risk groups (\u03c7=15.2 with p<0.0001 for the 78 patient data and \u03c7=21.3 with p<0.0001 for the 295 patient data). Seven common genes were then analyzed (overlapping with the 70 genes and the 117 features), seen in . Analysis based on the same statistical method yielded a similar result (\u03c7=14.7 with p=0.0001 for the 78 patient data and \u03c7=20.5 with p<0.0001 for the 295 patient data), shown in ) through ().","Background: This study collected one training set (a total of 78 breast cancer patient samples) and one test set (n=295 patients, including 32 patients from the training set) with the time to metastasis as the clinical outcome to develop a 70 gene signature. In our study, we used the training set (n=78) and the test set which excluded the 32 patients from the training set (n=263) to examine if the malignancy-risk genes could predict metastasis. There were 117 features that were in common with the malignancy-risk gene signature. Among them, there were 7 genes in common () between the 70 gene signature and the malignancy-risk gene signature.","Data analysis: We compared performance of survival analysis for the 3 gene signatures (malignancy-risk signature, 70 gene signature, and 7 genes in common) based on the malignancy-risk score.","Malignancy-risk score: We first evaluated the overall performance of the 117 features for the malignancy-risk gene signature. The risk score was calculated by converting expression of the genes into the first principal component score (based on principal component analysis method; see Statistical Methods, see Example 13). We used median of the risk score as cutoff to dichotomize the 78 patients (training set) into two risk groups. The median cutoff of the risk score from the training set was also used to dichotomize the patients into two risk groups for the test set (n=263). Log rank test showed a significant separation between the two risk groups (\u03c72=12.2 with p=0.0005 for the training data; and \u03c72=22.4 with p<0.0001 for the test data). The risk score was calculated in the same way for the 70 gene signature and 7 common genes, respectively.","Results: The three gene signatures performed well to separate survival curves of the two risk groups () for both datasets (training and test sets). The 70 gene signature performed the best because the signature was derived from the dataset (Figure A). However, the performance for the malignancy-risk signature was comparable to the 70 gene signature, especially in the test set. Even for the 7 genes in common, it also had a comparable performance (). Univariate Cox proportional hazards model also showed 48 genes with p value <0.05 in both training and test sets ().","Discussion","Identification of high-risk normal tissue has great potential application in clinical practice, in both evaluating the risk associated with routine breast biopsies as well as the risk of local recurrence following lumpectomy. Detecting high-risk normal tissue, however, remains a challenging task. The invention described herein provides, in one aspect, identifying high-risk normal tissue using gene expression profiles. One underlying rationale, to which the inventors are not bound, for this invention is a histologically normal tissue with \u201ctumor-like\u201d gene expression pattern might harbor substantial risk for future cancer development. Genes associated with these high-risk tissues were referred to as \u201coutlier\u201d genes. Using this rationale, 11 outlier tissues out of 143 normal breast tissues were identified, and the outlier gene signature was developed using the outlier tissue approach. A careful re-examination of all outlier tissues showed the tissues were histologically normal with no observable indications of cancer development. See . However, the expression profile of these outlier tissues suggested similarity to tumor tissue, indicating that these tissues might harbor increased risk for cancer development.","The outlier gene signature was tested in four validations. The first verified that the outlier genes identified in histologically normal breast tissues adjacent to invasive cancers were also highly associated with invasive ductal carcinomas (IDC). The Turashvili et al.dataset was used for evaluation and the outlier gene signature was found able to differentiate the IDC and normal tissues not linked to cancer, confirming the outlier genes as a subset of IDC tumor associated genes.","The second validation tested the risk of disease progression, where cancer risk was considered on a continuous spectrum with normal tissue in the lower end and IDC tissue at the higher end. As ADH and DCIS have been shown as precursors of IDC, it was ascertained whether the outlier gene signature exhibited a progressive trend from normal to IDC with ADH and DCIS as intermediate stages in the cancer risk spectrum. The existence of a strong trend with these features would provide a compelling evidence for the application of this signature on early prevention of cancer development. The outlier genes were tested on two datasets: Moffitt Cancer Center derived DCIS samples and Ma et al data. Results from both datasets showed first PCA scores for the ADH or DCIS tissues were higher than outlier tissues, but lower than IDC, showing a disease progression pattern from outlier, to ADH, to DCIS, to IDC. See ) and (). Moreover, 16 genes in the Ma dataset were identified that overlapped with the outlier signature, with an increasing expression pattern from ADH to IDC, seen in  and ) to (). The majority of the overlapping genes are known to be involved in the cell cycle. Since these genes were highly associated with cell proliferation and exhibited expression changes that were proportional to disease stage, these genes might be risk genes (precursor genes) useful in predicting cancer development and recurrence.","The third validation was to evaluate the capability of the outlier gene signature in predicting cancer development risk using the Poola's dataset. Analysis showed 3 out of 4 ADHC patients had an unusually high PCA score. This result supports the concept that outlier gene signatures can predict cancer risk. Furthermore, the ADH gene signature and the outlier gene signature shared similar cellular proliferation functions. Specifically, the Poola's study reported 11 major categories of cellular functions, and most genes were up-regulated in ADHC patients. Two of 11 up-regulated functions were cell cycle check points and nucleic acid biosynthesis. The majority of over-expressed genes in both functions were found in the outlier gene signature. See .","The last validation assessment tested prognostic features. While the outlier gene signature may be principally useful to assess cancer risk, this property was assessed in a broader scope. Since patients with high cancer risk are likely to develop metastasis, the outlier genes may play a key role for disease development. Validation results of Vant der Veer et al. breast metastasis dataset further supported this rationale. The overlapping 7 genes between the outlier signature and the Vant der Veer et al. dataset, see , identified patients at risk for metastasis and were mainly involved in proliferation.","While identification of high risk normal tissue is crucial, it is also important to understand the basic mechanism of how the molecular function changes in high risk normal tissue. The outlier gene signature showed significant expression in these 11 outlier tissues which were molecularly-suspect but histologically normal tissues. Adjacent normal tissues to the outlier tissues also showed a relatively higher expression of this gene signature than the rest of normal tissues, seen in . Furthermore, the pathway analysis showed the outlier genes were predominantly comprised of cell cycle activities. Further, the outlier genes were highly associated with two proliferation-related pathways: DNA replication and mitosis (14 and 25 up-regulated outlier genes, respectively) as shown in ) through (). While the primary function of the majority of these genes spans a variety of metabolic processes, it is clear that nearly all of the components are associated with cellular proliferation, a process that should be limited in normal tissues.","Moreover, the outlier genes were highly associated with chromosomal instability (CIN), chromosomal damage during cell division and a potential driving force for tumor initiation. There were 15 up-regulated outlier genes in the CIN25 gene signature with an outstanding CIN score, and another 13 up-regulated outlier genes in the CIN70 gene signature. The high correlation with CIN evidences that the outlier gene signature can explain early stages of cancer development and provides a simple mechanistic perspective on distinguishing outlier samples in a population of normal tissues.","Finally, it is clear that up to 40% of patients undergoing lumpectomy are at risk for local recurrence of breast cancer and that this risk might be predictable based on an outlier gene signature. Moreover, it is also clear that radiotherapy mitigates this risk, suggesting that the genes identified in high risk patients could be the target genes for effective radiotherapy. If an over-expression of these genes might result in a new cancer, suppression of these genes via radiotherapy might prevent the development of cancer.","In summary, using over a hundred of histologically normal breast tissues, an outlier gene signature of potential risk has been identified. This signature has a number of potential clinical applications such as judging risk of breast cancer development following routine breast biopsy, judging the need for adjuvant radiotherapy, and determining the need for completion of mastectomy following lumpectomy for the breast cancer patient.","Tissues and their Associated Clinicopathological Data","Tissues were collected in accordance with the protocols approved by the Institutional Review Board of the University of South Florida, and stored in the tissue bank of Moffitt Cancer Center. The tissues were embedded in Tissue-Tek\u00ae O.C.T., 5-\u03bcm sections cut and mounted on Mercedes Platinum StarFrost\u2122 Adhesive slides. The slides were stained using a standard H&E protocol, and tissue boundaries marked. Using the marked slide as a \u201cmap\u201d, tissues were microdissected. Adipose tissues were trimmed away. Both histologically-normal breast tissues and IDCs were derived from 90 patients that underwent mastectomy for various stages of breast carcinoma and were collected and frozen in liquid nitrogen. Clinico-pathological data from the patients used in the study, including the tumor ER, PR and Her2\/Neu status and tumor grade, are shown in Table 1. When possible, each mastectomy specimen was prosected to yield an IDC and up to five sequentially-derived, adjacent normal tissue samples in the ipsilateral breast or from the four quadrants of the contralateral breast. As a result, we collected 42 IDCs and 143 normal breast tissues from the 90 patients for microarray analysis. Due to RNA quality issue in some IDC and normal tissues, we did not have a complete set of IDC and normal tissues for some patients. There were 11 patients (a total of 34 tissues) with at least one normal and one IDC tissue, 19 patients (a total of 28 tissues) with IDC tissue only, and 60 patients (a total of 123 tissues) with normal tissue only. Table 2 lists number of normal and IDC tissues and their geographical locations relative to the incident tumor (see also ).","Histology","Based on the histopathologic review by one breast pathologist (AN), all of the 143 histologically normal breast tissues were confirmed to be free of atypical ductal hyperplasia (ADH) and in-situ or invasive breast carcinoma. The 42 IDC tissues were also confirmed by the histopathologic review by the same pathologist, based on the modified Bloom and Richardson grading scheme.","RNA Extraction","Total RNA was extracted from the breast tissues using the Trizol method. Briefly, tissues were pulverized in liquid nitrogen, resuspended in 5 ml of lysis buffer, incubated for 3 min. at room temperature, and centrifuged at 11,500 g for 15 minutes at 4\u00b0. The aqueous phase was removed and put into another tube with 2.5 ml of isopropanol, mixed well and set at \u221220\u00b0 C. for 20 minutes. The amount of RNA was quantitated by measuring A260. Microarray analysis was performed using the Affymetrix U133Plus 2.0 GeneChips (54,675 probe sets). Expression values were calculated using the robust multi-array average (RMA) algorithm(Data is in the GEO repository: www.ncbi.nlm.nih.gov\/geo\/query\/acc.cgi?acc=GSE10780.","RT-PCR Validation","Validation of 30 selected malignancy-risk signature genes (of 117 available) (see Table 3) was done using the TaqMan Low Density Arrays (Applied Biosystems, Foster City, Calif., USA). Due to limitation of sample availability, 5 \u201cIDC-like\u201d normal tissues, 8 IDCs, and 8 normal tissues were used for validation. Single stranded cDNA was synthesized from 1 ug of total RNA using random primers in a 20 uL reaction volume using Applied Biosystem's High Capacity cDNA Reverse Transcription kit. The 20 uL reactions were incubated in a thermal cycler for 10 min at 25\u00b0 C., 120 min at 37\u00b0 C., 5 sec at 85\u00b0 C. and then held at 4\u00b0 C. Real-time PCR was carried out using sequence specific primers\/probes on the Applied Biosystems 7900 HT Real-Time PCR system. cDNA was diluted 2.5-fold; 5.0 uL of diluted cDNA was mixed with 45 uL of nuclease-free water and was added to 50 uL of TaqMan Universal PCR Master Mix (Applied Biosystems). The 100 uL total reaction mixture was loaded in the corresponding ports of a TaqMan Low Density Array (TLDA) card. Each TLDA card consisted of 3 replicates (4 samples per card). Expression value (\u0394Ct) was calculated by first averaging replicates for each gene and then normalized (subtraction) by an endogenous control gene (18S). Since a lower value of \u0394Ct indicates a higher expression, a \u2212\u0394Ct was used to correlate with microarray gene expression.","Signature Generation\/Statistical Methods","Statistical analysis included a series of steps to develop and validate the malignancy-risk gene signature (see ):","1. Identification of IDC gene signature: In this first step, a set of 1038 genes (1,554 probe sets) was identified that distinguished the IDCs (n=42) from the histologically-normal tissues (n=143). The IDC gene set was identified by treating IDC and normal tissues as two independent groups (although some were derived from the same patients) and using Statistical Analysis of Microarrayat 1% false discovery rate (FDR) with a fold change >2 (see ). The study aimed to collect multiple normal and IDC tissues from the same subjects, but due the heterogeneous nature of the sample set, some patients had only normal tissues sampled while others samples were limited to IDC tissues only. This nature of unbalanced data made it difficult to adjust for subject variation. Instead, we aggregated data into normal and IDC two groups for comparison. To ensure homogeneity for data aggregation, we checked whether overall gene expression from the normal tissues in patients with normal tissues available only was similar to the normal tissues in patients with both normal and IDC tissues available. We used Kmeans approach to classify all the normal tissues into two groups based on gene expression data. Fisher exact test did not show the two types of normal tissues were statistically different (p=0.53). We found similar results for the IDC tissues (p=0.99). These results suggested homogeneity for the two types of normal tissues (also for the IDC tissues).","2. Identification of \u201cIDC-like\u201d normal tissues: In this step, we used the IDC gene signature to identify 11 histologically normal breast tissues that had acquired the molecular fingerprint of IDC. The method first ranked all the normal tissues for each IDC tumor gene. (e.g., A normal tissue A is ranked as the top 1% (percentile rank=100%) for tumor gene X1, top 10% (percentile rank=90%) for tumor gene X2, top 20% for tumor gene X3, and so on). As a result, for the up-regulated IDC tumor genes (e.g., k1 genes), we will have a set (k1) of the tissue percentile ranks for each tissue. If a normal tissue displayed at least half (>k\u00bd) of the percentile ranks over 80% (i.e., the median percentile rank >0.8), we considered it as \u201cIDC-like\u201d normal tissue. Similarly, a normal tissue was also considered as an IDC-like tissue if a normal tissue had the median of the percentile ranks below 20% for down-regulated IDC tumor genes. A graphical presentation of the method is included in the .","A simulation was conducted and showed its effectiveness to identify IDC-like tissues (Table 4). Simulation Scheme: We generate two groups, normal and tumor tissues, with a sample size of 150 (normal) and 50 (tumor). We assume each gene chip contains 50,000 genes and these genes are independent. We consider a series of proportion of significant genes from 1% to 10% for evaluation. Here we label IDC-like normal tissue as \u2018outlier tissue\u2019.","For non-significant genes, their expressions follow a standard normal distribution (i.e., N(0,1)) for both normal and tumor tissues. For significant genes, half of them have a higher expression in tumor tissue (up-regulation) with a normal distribution of N(1,1); and half of them a lower expression in tumor tissue (down-regulation) with a normal distribution of N(\u22121,1). On the other hand, the gene expression in normal tissue follows a standard normal distribution except the outlier normal tissues. Here we assume 10% normal tissues to be outlier with a tumor-like gene expression profile. For these outlier tissues, we assume they have the same expression distribution as the tumor ones (i.e., N(1,1) or N(\u22121,1)).","At each simulation, we use the Statistical Analysis of Microarray (SAM) and the outlier tissue approach (OTA) to analyze the simulated data. The threshold settings for SAM and OTA are the same as the ones used in our breast expression data. That is, we use a cutoff of false discovery rate (FDR) <0.01 and a fold change >2 in SAM to select tumor genes. For OTA, we use the median percentile rank >0.8 in the up-regulated tumor genes or the median percentile rank <0.2 in the down-regulated tumor genes to determine outlier tissue. As a result, we will collect a set of outlier tissues being identified correctly and a set of outlier tissues being misclassified to come up a 2\u00d72 table (see Table 4A).","Evaluation: We perform 100 simulations. Each simulation generates a 2\u00d72 table which allows us to calculate sensitivity and predictive value positive (PV+) for evaluation. We average each measure (i.e., sensitivity and PV+) over 100 simulations to examine performance.","Rationale for the use of sensitivity and PV+: Since outlier tissue is likely rare, the proportion of the outlier tissue is expected to be small. Because of this feature, accuracy rate (the proportion to be classified correctly) tends to be high even sensitivity or PV+ is low. In addition, we would like to have a procedure that yields most outlier tissues selected (i.e., high sensitivity) and has few or none of false outlier tissues selected (i.e., high PV+). For this reason, we consider the use of both sensitivity and PV+ to examine our approach. The curve of sensitivity versus PV+, in fact, is equivalent to the conventional Receiver Operating Characteristic (ROC) curve (i.e., sensitivity versus 1-specificity or true positive versus false positive). However, the conventional ROC curve focuses one aspect of evaluation of a screening test without considering the other curve (e.g., a high sensitivity does not guarantee a high or low PV+). In contrast, we present a unique evaluation of sensitivity versus PV+ to effectively assess the procedure.","Results: Simulation results show a high value of sensitivity and PV+ by the proposed approach in various proportions of significant genes (1%-10%) (see Table 4B). This observation demonstrates effectiveness of the approach in identifying outlier tissue.","3. Derive malignancy-risk gene score: Once the IDC-like normal tissues were identified, we then formed a common set of genes, \u201cmalignancy-risk signature genes\u201d, whose expression percentile rank was greater than 80% (or less than 20%) in most IDC-like normal tissues. Using the principal components analysis (PCA) method, we derived a \u201crisk score\u201d (malignancy-risk score) to represent an overall gene expression level for the malignancy-risk gene signature. First, we performed principal components analysis to reduce data dimension into a small set of uncorrelated principal components. This set of principal components was generated based on its ability to account for variation. We used the first principal component, as it accounts for the largest variability in the data, as a malignancy-risk score to represent the overall expression level for the signature. That is, malignancy-risk score=\u03a3wx, an weighted average expression among the malignancy-risk genes, where xi represents gene i expression level, wi is the corresponding weight with \u03a3w=1, and the wi values maximize the variance of \u03a3wx.","It is likely, when benign appearing breast tissues harbor malignancy-risk genes, the gene data will generate a high signal-to-noise ratio such that the first principal component with the largest variance will correspond to cancer risk-related information.","4. Cross-validation: Leave-one-out cross validation (LOOCV) was performed to evaluate robustness of the IDC and malignancy-risk gene signatures. This was done by excluding one sample at a time and repeating steps 1-3 to see how many were correctly identified (IDC genes, IDC-like normal tissues, and malignancy-risk genes).","5. Pathway analysis: Pathway analysis was done using MetaCore\u2122 by GeneGo for steps 1 and 3 to identify biological functions associated with IDC genes and the malignancy-risk genes. We compared pathways of the two gene sets to reveal difference of biological processes between the IDC genes and the malignancy-risk genes.","6. RT-PCR validation: Pearson correlation was used to evaluate association of the malignancy-risk score between microarray and RT-PCR platforms. The malignancy-risk score was calculated using the 30 selected malignancy-risk signature genes (see Statistical Methods) for microarray and RT-PCR, respectively. Correlation analysis was also performed for each individual malignancy-risk gene. Analysis of variance was used to test the differences among the three groups (normal, IDC-like normal, and IDC) with the Tukey methodto adjust for p value for pair-wise comparison.","7. Clinical association: We assessed the prognostic potential of the malignancy-risk score on seven external independent data sets. Because each data set had a different set of available genes, we used whatever genes were in common with the malignancy-risk score to evaluate each data set (essentially a subset of the original malignancy-risk score). For binary clinical outcome (e.g., cancer development versus no development) or survival outcome (e.g., time to metastasis), the derived malignancy-risk score was dichotomized using the median cutoff (i.e., high risk with score >median and low risk with score <median). Logistic regression model was used to estimate odds ratio (OR) for binary outcome and log-rank test was used to test the difference of Kaplan-Meier survival curves for two risk groups for survival data. For ordinal clinical variable (e.g., from ADH, ductal carcinoma in situ (DCIS), to IDC), the continuous malignancy-risk score was used to correlate with disease severity using Pearson correlation to evaluate the trend of the malignancy-risk gene signature with cancer progression.","The 11 IDC-like normal breast tissues identified in Example 3 where analyzed for their ER, PR, and Her2\/neu status. Fisher exact test showed no significant association of patients harboring IDC-like normal tissues with ER\/PR\/Her2 grade (Table 5)","The malignancy-risk score (equivalent to PCA score) was analyzed by Leave-one-out crossvalidation (LOOCV). Leave-one-out cross validation (LOOCV) was implemented by excluding one sample at a time and repeating statistical steps 1-3 (i.e., IDC genes, outlier tissues (which can be used to represent IDC-like normal tissue), and malignancy-risk genes; see Statistical Methods). In other words, each time we hold one sample as a test sample and used the rest samples to identify an IDC gene set (step 1). The selected IDC genes were then used to identify outlier tissues (step 2). We used these selected outlier tissues to identify malignancy-risk genes (step 3). These malignancy-risk genes were used to predict the malignancy-risk score for the sample being hold. As a result, each time we collected four metrics: a set of IDC genes, a set of outlier tissues, a set of malignancy-risk genes, and the predicted malignancy-risk score (for the sample being hold). We examined the first three metrics from LOOCV to see how consistent with the ones from the whole dataset. Analysis of LOOCV yielded a high degree of consistency: most IDC genes (>98%), outlier tissues (>90%), and malignancy-risk genes (>90%) were identified at each leave-one-out iteration () Moreover, at each iteration, we calculated a predicted malignancy-risk score for the sample being excluded. Correlation analysis showed a high correlation of the predicted malignancy-risk score and the disease status (i.e., rank normal, outlier, and IDC from 0 to 2; Pearson correlation=0.89 and Spearman correlation=0.74 with p<0.0001; ).","Since ER, PR, and Her2 are key markers in cancer development, we examined their correlation with the malignancy-risk score. Results showed only a weak correlation for ER and PR (r=0.2\u02dc0.3) and a moderate correlation with Her2 (r=0.37\u02dc0.47 by spearman correlation and r=0.43\u02dc0.63 by Pearson correlation), suggesting relative independence of the risk score from these biomarkers ()","We identified 11 IDC-like normal tissues from 10 patients. There were another 12 normal tissues collected from the same 10 patients. These 12 normal tissues were molecularly and histologically normal and labeled as matched normal tissues to reflect they were derived from the same subject. The other normal tissues (n=120) from subjects without IDC-like normal tissues (i.e., not from the 10 subjects) were also molecularly and histologically normal and labeled as unmatched normal tissues for distinction. We found the malignancy-risk score was higher in the IDC-like normal tissues and the matched normal tissues than in the unmatched normal tissues. Difference of the risk score was statistically significant for (a) IDC-like normal tissues versus the matched normal tissues (adjusted p value <0.0001 using the Tukey method) and (b) matched versus unmatched normal tissues (adjusted p value=0.0054). An increasing trend of the malignancy-risk score was also seen from the unmatched normal tissues, the matched normal tissue, to the IDC-like normal tissues at the pooled data level (Pearson correlation=0.63 with p<0.0001; ). Moreover, among the 10 patients with IDC-like normal tissues, analysis results showed a higher malignancy-risk score in the IDC-like normal tissues than in the matched normal tissues at subject level (p=0.01 using the random effect model; ). Since the malignancy-risk score was derived without knowing subject information, a trend of the risk score decreasing from the IDC-like normal tissues, to the matched normal tissue, to the unmatched normal tissues would not be expected.","Expression of the 30 selected malignancy-risk signature genes identified by microarray profiling was successfully validated by RT-PCR. The 30 genes were selected based on expression and biological relevance. There were 27 genes showing a strong Pearson correlation >0.7 (correlation >0.9: 12 genes, 0.8-0.9: 13, and 0.7-0.8: 2; the p values were <0.0001) (). The composite malignancy-risk score (based on microarray data from 30 genes) also demonstrated a very high correlation (0.95) with RT-PCR results. The risk score for the IDC-like normal tissues fell in the middle between the IDC and normal samples ().","Examples 19-21 describe the clinical association of malignancy-risk signature with cancer risk, cancer relapse\/progression, and prognosis. We assessed the malignancy-risk score on three additional external independent datasets (see also Examples 8-12). Statistical procedures were described in Statistical Methods Section. These external datasets as well as the datasets presented in Examples 8-12 permitted the evaluation of a number of properties of the malignancy-risk signature including cancer risk, cancer relapse\/progression, and cancer prognosis. Table 9 summarizes the results for all these datasets.","Relapse of tamoxifen-treated primary breast cancers: (GEO: GSE9893)","Background: There were 155 patients (52 patients with relapse (R) and 103 patients with relapse-free (RF) who received adjuvant tamoxifen. The primary tumors from these patients were analyzed for expression profiles at the whole-genome level by 70-mer oligonucleotide microarrays (22,656 genes). There were 61 genes in common with the malignancy-risk gene signature, which was used to calculate the malignancy-risk score.","Results: Analysis by a median cutoff of the risk score (i.e., high risk with score >median and low risk with score <median) showed a significant association of the malignancy-risk score with the relapse of primary breast cancers (logistic regression: OR=7.82 with p<0.0001) Similarly, two sample t-test showed a statistically significant difference of the risk score between relapse versus relapse-free (p<0.0001; see ). Moreover, univariate analysis based on two-sample test showed most genes with p<0.05 (50 out of 61 genes; 82%; in contrast to 60% genes with p<0.05 when using all the 22,656 genes; see ).","Background: The data includes 286 lymph-node negative breast patients with metastasis-free survival as clinical survival outcome. A 76 gene signature was derived from this dataset to predict distant metastasis. The microarray platform was Affymetrix Human U133a GeneChips. We normalized data using quantile-quantile method. There were 102 probe sets (from the \u02dc20K probe sets) in common with the malignancy-risk gene signature. There were only 4 genes in common () between the 76 gene signature and the malignancy-risk gene signature.","Data analysis: We compared performance of survival analysis for the 3 gene signatures (malignancy-risk signature, 76 gene signature, and 4 genes in common) based on the malignancy-risk score.","Malignancy-risk score: We first evaluated the overall performance of the 102 probe sets for the malignancy-risk gene signature. The risk score was calculated by converting expression of the genes into the first principal component score (based on principal component analysis method; see Statistical Methods). We used median of the risk score as cutoff to dichotomize the 286 patients into two risk groups. Log-rank test showed a significant separation of KM survival curves between the two risk groups (\u03c72=12.6; p=0.0004). The risk score was calculated in the same way for the 76 gene signature and 4 common genes, respectively. Univariate Cox proportional hazards model was also used to test individual gene effect (of the 102 genes) on the metastasis-free survival.","Results: The three gene signatures performed well to separate survival curves of the two risk groups (). The 76 gene signature performed the best because the signature was derived from this dataset (). However, the performance for the malignancy-risk signature was almost comparable to the 76 gene signature. Even for the 4 genes in common, it also had a comparable performance ().  listed 64 genes (of the 102 genes) with p value <0.05 based on the univariate analysis.","Background: The breast cancer microarray data reported by Huang et al. (Huang et al., 2003) contained 18 patients with positive lymph node (LN) and 19 patients with negative LN. The gene expressions were obtained from the Affymetrix human U95a chip. Data were pre-processed using RMA (Irizarry et al. 2003). There were 112 probe sets (82 unique genes) in common with the outlier gene signature.","Data analysis: We evaluated the association of the malignancy-risk gene signature with lymph node development using the malignancy-risk score and univariate analysis.","Results: 1. Malignancy-risk score: The malignancy-risk score was generated using expression data from the 112 probe sets. We used the median cutoff of the risk score to dichotomize patients into two risk groups: high risk with score >median and low risk with score <median. Logistic regression model showed a significant association of the median cutoff malignancy-risk score with the LN status (logistic regression: OR=7.29 with p=0.007). Similarly, two sample t-test showed a statistically significant difference of risk score between positive LN versus negative LN (p=0.004) ().","2. Univariate analysis: There were 34 probe sets (34\/122=30%) with p value <0.05 (two-sample t-test) (). In contrast, there were only 7% genes (912 out of 12625 probe sets) with p<0.05 when using all probe sets. Fisher exact test showed a highly statistical significance (p<0.0001), indicating that it is unlikely by chance to have such large proportion of significant genes (30%).","Summary: Results suggested that the outlier gene signature was associated with lymph node development (LN positive tends to have a higher malignancy-risk score).","Discussion","Identification of normal tissue at risk for malignant conversion has great potential application in clinical practice, in both evaluating the risk associated with routine breast biopsies as well as the risk of local recurrence following lumpectomy. Detecting these high-risk normal appearing tissues, however, remains a challenging task. In one aspect of this invention, we developed an innovative approach to identify histologically-normal, but molecularly-abnormal tissue \u201cat risk\u201d for malignant degeneration. One rationale, to which the inventors are not bound, is that a histologically-normal tissue with \u201ctumor-like\u201d gene expression pattern might harbor substantial risk for future cancer development. Genes associated with these high-risk tissues were referred to as \u201cmalignancy-risk genes\u201d. Based on this rationale, we identified 11 \u201cIDC-like\u201d normal tissues (out of 143 normal breast tissues) and developed the malignancy-risk gene signature and risk score.","A careful re-examination of all the IDC-like normal tissues showed that they were histologically-normal, with no evidence of in situ or invasive carcinoma of the breast, and no atypia (). However, these IDC-like normal tissues showed gene expression profiles resembling invasive carcinomas, indicating that these tissues had already acquired the molecular fingerprint of cancer and, therefore, may be at increased risk for subsequent cancer development. Moreover, from these IDC-like normal tissues, we developed a \u201cmalignancy-risk\u201d gene signature that may serve as a marker of subsequent risk of breast cancer development. The malignancy-risk gene signature was internally validated by RT-PCR and leave-one-out cross validation. Analysis of external datasets also demonstrated its clinical relevance to cancer-risk, cancer relapse\/progression, and prognosis. This is an intriguing finding with substantive clinical implications. While several studies may have suggested that cell cycle\/proliferation are one of the hallmarks of existing cancer, this is the first study to suggest the proliferative program of gene expression may be the earliest detectable event in normal breast tissues at risk for developing breast cancer. A recently reported study of 14 normal breast tissues from breast cancer cases identified genes differentially expressed in these tissues versus normal breast reduction mammoplasties, but did not decipher a predominantly proliferative gene function. The large preponderance of proliferative genes in the malignancy-risk gene set was not expected. By comparison, IDC associated genes were biased towards both proliferative and adhesive gene sets. These findings suggest a temporal relationship between proliferative and adhesive gene expression programs, with the former being precursors to histological alterations and responsible for malignancy-risk. There was also no statistical association of the IDC-like normal tissues with ER\/PR, Her2\/neu, and grade suggesting the malignancy-risk signature may be not be dependent on these factors. The lack of association of the IDC-like normal tissues with the triple negative (ER\/PR\/Her2Neu) phenotype also suggests no link to BRCA1 and BRCA2.","Evaluation on external independent datasets demonstrated the clinical relevance of the malignancy-risk gene signature not only to cancer risk, but also to cancer relapse\/progression, and prognosis. As such, the signature has promise for impacting clinical decisions. These include altering strategies for follow-up of histologically-normal, but molecularly abnormal breast biopsies, determining which patients might benefit from radiotherapy following lumpectomy, or determining which patients might benefit from mastectomy due to multifocal disease risk.",{"@attributes":{"id":"p-0224","num":"0000"},"ul":{"@attributes":{"id":"ul0001","list-style":"none"},"li":["1. Kaplan, J., et al. Breast conservation in patients with multiple ipsilateral synchronous cancers. 197, 726-729 (2003).","2. Fisher, B., et al. Ten-year results of a reandomized clinical trial comparing radical mastectomy and total mastectomy with or without radiation. 312, 674-681 (1985).","3. Price, P., et al. Duct carcinoma insitu: predictors of local recurrence and progreassion in patients treated by surgery alone. 61, 869-872 (1990).","4. Page, D. L., Dupont, W. D., Rogers, L. W., Jensen, R. A. & Schuyler, P. A. Continued local recurrence of carcinoma 15-25 years after a diagnosis of low-grade ductal carcinoma in-situ of the breast treated only by biopsy. 76, 1197-1200 (1995).","5. Fredriksson, I., et al. Risk factors for local recurrence after breast-conserving surgery. 90, 1093-1102 (2003).","6. Shah, V. I., et al. False-negative core needle biopsies of the breast\u2014An analysis of clinical, radiologic, and pathologic findings in 27 consecutive cases of missed breast cancer. 97, 1824-1831 (2003).","7. Tusher, V. G., Tibshirani, R. & Chu, G. Significance analysis of microarrays applied to the ionizing radiation response. 98, 5116-5121 (2001).","8. Whitfield, M. L., et al. Identification of genes periodically expressed in the human cell cycle and their expression in tumors. 13, 1977-2000 (2002).","9. Turashvili, G., et al. Novel markers for differentiation of lobular and ductal invasive breast carcinomas by laser microdissection and microarray analysis. 7 (2007).","10. Ma, X. J., et al. Gene expression profiles of human breast cancer progression. 100, 5974-5979 (2003).","11. Poola, I., et al. Identification of MMP-1 as a putative breast cancer predictive marker by global gene expression analysis. 11, 481-483 (2005).","12. van de Vijver, M. J., et al. A gene-expression signature as a predictor of survival in breast cancer. 347, 1999-2009 (2002).","13. Hartigan J A, W. M. A K-Means Clustering Algorithm. 28, 100-108 (1979).","14. Breiman, L., Friedman, J., Olshen, R. & Stone, C. , (Wadsworth & Brooks, Monterey, Calif., 1984).","15. Carter, S. L., Eklund, A. C., Kohane, I. S., Harris, L. N. & Szallasi, Z. A signature of chromosomal instability inferred from gene expression profiles predicts clinical outcome in multiple human cancers. 38, 1043-1048 (2006).","16. Tuttle, T. M., Habermann, E., Grund, E., Morris, T. & Virnig, B. Increasing use of contralateral prophylactic mastectomy among breast cancer patients: a trend toward more aggressive surgical treatment. 14, 7-7 (2007).","17. Robbins, P., et al. Histological grading of breast carcinomas\u2014a study of interobserver agreement. 26, 873-879 (1995).","18. Tripathi A, King C, de la Morenas A, Perry V K, Burke B, Antoine G A, et al. Gene expression abnormalities in histologically normal breast epithelium of breast cancer patients. Int J Cancer 2008; 122(7):1557-66.","19. Irizarry R A, Bolstad B M, Collin F, Cope L M, Hobbs B, Speed T P. Summaries of Affymetrix GeneChip probe level data. In: Nucleic Acids Res; 2003. p. e15.","20. Tusher V G, Tibshirani R, Chu G. Significance analysis of microarrays applied to the ionizing radiation response. Proceedings of the National Academy of Sciences of the United States of America 2001; 98(9):5116-5121.","21. Miller R G. Simultaneous Statistical Inference: Springer; 1981.","22. Chanrion M, Negre V, Fontaine H, Salvetat N, Bibeau F, Mac Grogan G, et al. A gene expression signature that can predict the recurrence of tamoxifen-treated primary breast cancer. Clin Cancer Res 2008; 14(6):1744-52.","23. Wang Y, Klijn J G, Zhang Y, Sieuwerts A M, Look M P, Yang F, et al. Gene-expression profiles to predict distant metastasis of lymph-node-negative primary breast cancer. Lancet 2005; 365(9460):671-9.","24. Huang E, Cheng S H, Dressman H, Pittman J, Tsou M H, Horng C F, et al. Gene expression predictors of breast cancer outcomes. Lancet 2003; 361(9369):1590-6.","25. Sorlie T, Perou C M, Tibshirani R, Aas T, Geibreast cancerr S, Johnsen H, et al. Gene expression patterns of breast carcinomas distinguish tumor subclasses with clinical implications. Proc Natl Acad Sci USA 2001; 98(19):10869-74.","26. Rosenwald A, Wright G, Wiestner A, Chan W C, Connors J M, Campo E, et al. The proliferation gene expression signature is a quantitative integrator of oncogenic events that predicts survival in mantle cell lymphoma. Cancer Cell 2003; 3(2):185-97.","27. Whitfield M L, George L K, Grant G D, Perou C M. Common markers of proliferation. Nat Rev Cancer 2006; 6(2):99-106.","28. Chung C H, Bernard P S, Perou C M. Molecular portraits and the family tree of cancer. Nat Genet 2002; 32 Suppl:533-40."]}},{"@attributes":{"id":"p-0225","num":"0251"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 1"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Pathological data of the patients used in the"},{"entry":"study, including ER, PR, Her2, and grade."},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"ER\/PR\/Her2 status"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"ER","PR","Her2\/neu"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":[{},"Negative","25","38","43"]},{"entry":[{},"Positive","55","42","12"]},{"entry":[{},"other*","10","10","35"]},{"entry":[{},"Total cases","90","90","90"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Grade","frequency"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":[{},"Well differentiated","6"]},{"entry":[{},"Moderately differentiated","27"]},{"entry":[{},"Poorly differentiated","30"]},{"entry":[{},"Undifferentiated\/anaplastic","10"]},{"entry":[{},"No grade","17"]},{"entry":[{},"Total cases","90"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":"*Results not available"}]}}]}},"br":{}},{"@attributes":{"id":"p-0226","num":"0252"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"294pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2A"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Distribution of tumor and normal tissues per case and their"},{"entry":"geographical locations relative to the incident tumor."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"91pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Contralateral",{}]},{"entry":[{},"Ipsilateral breast","breast","Number of tissues"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Zone","Zone",{},"IDC-like",{}]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"15"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"10","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"11","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"12","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"13","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"14","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"15","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["ID","Tumor site","1","2","3","4","5","1","2","3","4","5","Normal","normal","Tumor"]},{"entry":{"@attributes":{"namest":"1","nameend":"15","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"15"},"colspec":[{"@attributes":{"colname":"1","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"10","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"11","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"12","colwidth":"14pt","align":"center"}},{"@attributes":{"colname":"13","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"14","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"15","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["7369",{},{},{},"N","N","N",{},{},{},{},{},"3","0","0"]},{"entry":["7397",{},"N",{},"N",{},"N",{},{},{},{},{},"3","0","0"]},{"entry":["7619",{},{},"N","N",{},{},{},{},{},{},{},"2","0","0"]},{"entry":["7954",{},{},{},"N",{},{},{},{},{},{},{},"1","0","0"]},{"entry":["8380","N",{},"O",{},{},"N",{},{},{},{},{},"2","1","0"]},{"entry":["8456",{},"N","N","N","N",{},{},{},{},{},{},"4","0","0"]},{"entry":["8463",{},"N","O","N","N",{},{},{},{},{},{},"3","1","0"]},{"entry":["8513",{},{},"T*",{},{},{},{},{},{},{},{},"0","0","1"]},{"entry":["8514",{},{},{},{},"N","N",{},{},{},{},{},"2","0","0"]},{"entry":["8542","T",{},{},{},{},{},{},{},{},{},{},"0","0","1"]},{"entry":["8607","O",{},{},{},{},{},{},{},{},{},{},"0","1","0"]},{"entry":["8608",{},{},"N",{},"N",{},{},{},{},{},{},"2","0","0"]},{"entry":["8615",{},"N",{},"N","N",{},{},"N",{},"N",{},"5","0","0"]},{"entry":["8626",{},"N",{},{},{},{},{},{},{},{},{},"1","0","0"]},{"entry":["8627",{},{},"O",{},{},{},{},{},{},{},{},"0","1","0"]},{"entry":["8628",{},"N",{},{},{},{},{},{},{},{},{},"1","0","0"]},{"entry":["8636","T, T, T",{},{},{},{},{},{},{},{},{},{},"0","0","3"]},{"entry":["8642","T, T, T, T",{},"T*",{},{},{},{},{},{},{},{},"0","0","5"]},{"entry":["8653",{},"N",{},{},"N",{},{},{},{},{},{},"2","0","0"]},{"entry":["8669",{},{},{},{},"N",{},{},{},{},{},{},"1","0","0"]},{"entry":["8671",{},{},{},{},"N","N",{},{},{},{},{},"2","0","0"]},{"entry":["8672","T",{},{},{},{},{},{},{},{},{},{},"0","0","1"]},{"entry":["8689",{},{},{},"N",{},{},{},{},{},{},{},"1","0","0"]},{"entry":["8696",{},"N","N","N",{},{},{},{},{},{},{},"3","0","0"]},{"entry":["8703","T, T",{},{},"N","N",{},{},{},{},{},{},"2","0","2"]},{"entry":["8706",{},{},{},{},"N",{},{},{},{},"N",{},"2","0","0"]},{"entry":["8717",{},{},"N",{},{},{},{},{},{},{},{},"1","0","0"]},{"entry":["8721",{},{},{},{},"N",{},{},{},{},{},{},"1","0","0"]},{"entry":["8735",{},"N",{},{},{},{},{},"N",{},{},{},"2","0","0"]},{"entry":["8803",{},"N",{},"N","N",{},{},{},"N","N",{},"5","0","0"]},{"entry":["8862","T",{},{},{},{},{},{},{},{},{},{},"0","0","1"]},{"entry":["8880",{},{},{},{},"N",{},{},{},{},{},{},"1","0","0"]},{"entry":["8881",{},{},{},{},"N",{},{},{},{},{},{},"1","0","0"]},{"entry":["8936","T",{},{},{},{},{},{},{},{},{},{},"0","0","1"]},{"entry":["9097","T",{},{},{},{},{},{},{},{},{},{},"0","0","1"]},{"entry":["9100",{},"N",{},"N",{},{},{},{},{},{},{},"2","0","0"]},{"entry":["9112",{},{},{},{},"N","N",{},{},{},{},{},"2","0","0"]},{"entry":["9393",{},{},{},"N",{},{},{},{},{},{},{},"1","0","0"]},{"entry":["9740","T",{},"N","T*","N",{},{},{},{},{},{},"2","0","2"]},{"entry":["9744",{},{},{},{},"N",{},{},{},{},{},{},"1","0","0"]},{"entry":["10143","T",{},"T*",{},{},{},{},{},{},{},{},"0","0","2"]},{"entry":["10176",{},{},"N","N","N",{},{},{},{},{},{},"3","0","0"]},{"entry":["10180","T",{},"O","N",{},{},{},{},{},{},{},"1","1","1"]},{"entry":["10443",{},{},"N",{},{},{},{},{},{},{},{},"1","0","0"]},{"entry":["10473","T",{},{},{},{},"N",{},{},{},{},{},"1","0","1"]},{"entry":["10475",{},"N","N","N",{},{},{},{},{},{},{},"3","0","0"]},{"entry":["10481",{},"N",{},{},{},{},{},{},{},{},{},"1","0","0"]},{"entry":["10496","T",{},"N",{},{},{},{},{},{},{},{},"1","0","1"]},{"entry":["10535",{},"N",{},{},{},{},{},{},{},{},{},"1","0","0"]},{"entry":["10559",{},{},{},"N",{},{},{},{},{},{},{},"1","0","0"]},{"entry":["10563",{},{},{},"N",{},{},{},{},{},{},{},"1","0","0"]},{"entry":["10584",{},{},{},"N",{},{},{},{},{},{},{},"1","0","0"]},{"entry":["10608",{},{},"N","N","N","N",{},{},{},{},{},"4","0","0"]},{"entry":["10650",{},{},{},"N",{},{},{},{},{},{},{},"1","0","0"]},{"entry":["10686","T",{},{},{},{},{},{},{},{},{},{},"0","0","1"]},{"entry":["10739","T",{},{},{},"O",{},{},{},{},{},{},"0","1","1"]},{"entry":["10781","T",{},"T*",{},{},{},{},{},{},{},{},"0","0","2"]},{"entry":["10786","T",{},{},{},{},{},{},{},{},{},{},"0","0","1"]},{"entry":["10874",{},"N",{},{},{},{},{},{},{},{},{},"1","0","0"]},{"entry":["10910",{},{},"N","O","O",{},{},{},{},{},{},"1","2","0"]},{"entry":["10918","T",{},{},{},{},{},{},{},{},{},{},"0","0","1"]},{"entry":["10928","T",{},{},{},{},{},{},{},{},{},{},"0","0","1"]},{"entry":["10957","T",{},{},{},{},{},{},{},{},{},{},"0","0","1"]},{"entry":["10962","N*",{},{},{},{},{},{},{},{},{},{},"1","0","0"]},{"entry":["10964",{},"N","N",{},"N",{},{},{},{},{},{},"3","0","0"]},{"entry":["11003",{},{},"N","N","N",{},{},{},{},{},{},"3","0","0"]},{"entry":["11063","T",{},"N",{},"N","N",{},{},{},{},{},"3","0","1"]},{"entry":["11103",{},"N","O",{},"N",{},{},{},{},{},{},"2","1","0"]},{"entry":["11123",{},{},"N","O","N",{},{},{},{},{},{},"2","1","0"]},{"entry":["11147","T","N",{},{},{},{},{},{},{},{},{},"1","0","1"]},{"entry":["11196",{},{},"N","N","N","N",{},{},{},{},{},"4","0","0"]},{"entry":["11209",{},"N","N",{},{},{},{},{},{},{},{},"2","0","0"]},{"entry":["11286","T",{},{},{},{},{},{},{},{},{},{},"0","0","1"]},{"entry":["11320",{},{},"N","N",{},{},{},{},{},{},{},"2","0","0"]},{"entry":["11354",{},"N",{},"N","N","N",{},{},{},{},{},"4","0","0"]},{"entry":["11365","T",{},{},{},{},{},{},{},{},{},{},"0","0","1"]},{"entry":["11451",{},{},"N",{},"O",{},{},{},{},{},{},"1","1","0"]},{"entry":["11455","T",{},"N",{},{},{},{},{},{},{},{},"1","0","1"]},{"entry":["11464",{},"N",{},"N",{},{},{},{},{},{},"N","3","0","0"]},{"entry":["11472",{},"N",{},"N","N",{},{},{},{},{},{},"3","0","0"]},{"entry":["11478","T, T",{},{},{},"N",{},{},{},"N","N",{},"3","0","2"]},{"entry":["11545",{},{},{},{},"N",{},{},{},{},{},{},"1","0","0"]},{"entry":["11580","T",{},{},"N","N","N",{},{},{},{},{},"3","0","1"]},{"entry":["11640",{},"N",{},{},{},{},{},{},{},{},{},"1","0","0"]},{"entry":["11688","T",{},{},{},"T*",{},{},{},{},{},{},"0","0","2"]},{"entry":["11689",{},{},{},{},"N",{},{},{},{},{},{},"1","0","0"]},{"entry":["11693",{},{},"N",{},{},{},{},{},{},{},{},"1","0","0"]},{"entry":["11732",{},{},"N",{},"N","N",{},{},{},{},{},"3","0","0"]},{"entry":["11770","T",{},{},{},{},{},{},{},{},{},{},"0","0","1"]},{"entry":["11792",{},"N",{},{},{},{},{},{},{},{},{},"1","0","0"]},{"entry":{"@attributes":{"namest":"1","nameend":"15","align":"center","rowsep":"1"}}},{"entry":"N* = histological normal, but sampled as tumor"},{"entry":"T* = histological tumor, but sampled as normal"},{"entry":"O: IDC-like normal"}]}}]}}},{"@attributes":{"id":"p-0227","num":"0253"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 2B"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Frequency of cases with number of IDC and normal tissues"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00001","num":"00001"},"img":{"@attributes":{"id":"EMI-C00001","he":"26.33mm","wi":"47.84mm","file":"US09195796-20151124-C00001.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":{"chemistry":{"@attributes":{"id":"CHEM-US-00002","num":"00002"},"img":{"@attributes":{"id":"EMI-C00002","he":"10.16mm","wi":"48.60mm","file":"US09195796-20151124-C00002.TIF","alt":"embedded image","img-content":"table","img-format":"tif"}}}}}]}}}}},{"@attributes":{"id":"p-0228","num":"0254"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"A list of 30 malignancy-risk genes for RT-PCR validation"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Affymetrix"]},{"entry":[{},"Gene name","probe set id"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"ANLN","222608_s_at"]},{"entry":[{},"BUB1","209642_at"]},{"entry":[{},"BUB3","201457_x_at"]},{"entry":[{},"CCNB1","214710_s_at"]},{"entry":[{},"CDC2","203213_at"]},{"entry":[{},"CDKN3","209714_s_at"]},{"entry":[{},"CENPA","204962_s_at"]},{"entry":[{},"CENPF","207828_s_at"]},{"entry":[{},"CKS2","204170_s_at"]},{"entry":[{},"DPP3","218567_x_at"]},{"entry":[{},"DTL","218585_s_at"]},{"entry":[{},"FOXM1","202580_x_at"]},{"entry":[{},"HN1","217755_at"]},{"entry":[{},"KPNA2","211762_s_at"]},{"entry":[{},"MELK","204825_at"]},{"entry":[{},"MLF1IP","218883_s_at"]},{"entry":[{},"NDC80","204162_at"]},{"entry":[{},"NME","201577_at"]},{"entry":[{},"NUSAP1","218039_at"]},{"entry":[{},"PAFAH1B3","203228_at"]},{"entry":[{},"PBK","219148_at"]},{"entry":[{},"PCNA","201202_at"]},{"entry":[{},"PRC1","218009_s_at"]},{"entry":[{},"RACGAP1","222077_s_at"]},{"entry":[{},"RRM2","201890_at"]},{"entry":[{},"SMC4","201663_s_at"]},{"entry":[{},"SQLE","209218_at"]},{"entry":[{},"TK1","202338_at"]},{"entry":[{},"TOP2A","201291_s_at"]},{"entry":[{},"TYMS","1554696_s_at"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}}]}},"br":{}},{"@attributes":{"id":"p-0229","num":"0255"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 4A"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Predictive Value Positive (PV+) versus Sensitivity."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"119pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Fact"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Outlier normal tissue","Normal tissue"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Test","Classify as outlier tissue","a","c"]},{"entry":[{},"Classify as normal tissue","b","d"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":"PV+ (= a\/(a + c)): Probability of a selected normal tissue as an outlier tissue."},{"entry":"Sensitivity (= a\/(a + b)): Probability of an outlier normal tissue being selected."}]}}]}}},{"@attributes":{"id":"p-0230","num":"0256"},"tables":{"@attributes":{"id":"TABLE-US-00006","num":"00006"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 4B"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Sensitivity and PV+ of identification"},{"entry":"of outlier tissue over 100 simulations."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"105pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"70pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Proportion of",{},{}]},{"entry":["significant genes","Sensitivity","PV+"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["1%","80.3%","100%"]},{"entry":["2%","81.3%","100%"]},{"entry":["3%","82.4%","100%"]},{"entry":["4%","83.9%","100%"]},{"entry":["5%","84.4%","100%"]},{"entry":["6%","84.0%","100%"]},{"entry":["7%","85.9%","100%"]},{"entry":["8%","86.5%","100%"]},{"entry":["9%","86.9%","100%"]},{"entry":["10%\u2002","91.3%","100%"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0231","num":"0257"},"tables":{"@attributes":{"id":"TABLE-US-00007","num":"00007"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"343pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 5"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Frequency of ER\/PR\/Her2\/Grade for the patients with IDC-like normal tissues and the"},{"entry":"patients with the remaining normal tissues (labeled as non-IDC-like normal)"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"98pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"98pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"ER","PR","Her2\/neu"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Patients with",{},"Patients",{},"Patients with"]},{"entry":[{},"Patients with non-","IDC-like","Patients with non-","with IDC-","Patients with non-","IDC-like"]},{"entry":["Type","IDS-like normal","normal","IDC-like normal","like normal","IDC-like normal","normal"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}},{"entry":["Negative","16","5","25","4","26","3"]},{"entry":["Positive","36","4","27","5","7","2"]},{"entry":["Total available","52","9","52","9","33","5"]},{"entry":["cases*",{},{},{},{},{},{}]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"98pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"98pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["p value**","0.25","0.73","0.57"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"343pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"*Cases with negative or positive results"},{"entry":"**p value was calculated using the Fisher exact test"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"105pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"119pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Grade Description","Patients with non-IDC-like normal","Patients with IDC-like normal"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":[{},"Well differentiated","6","0"]},{"entry":[{},"Moderately differentiated","16","4"]},{"entry":[{},"Poorly differentiated","20","1"]},{"entry":[{},"Undifferentiated\/anaplastic","6","2"]},{"entry":[{},"Total available cases","48","7"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"224pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"p value*","0.21"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"343pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"*p value was calculated using the Fisher exact test."},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"126pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Patients with non-IDS-like","Patients with"]},{"entry":[{},{},"normal","IDC-like normal"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":[{},"Age < 50","26","6"]},{"entry":[{},"Age > 50","31","3"]},{"entry":[{},"Total available cases","57","9"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"210pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"p value*","0.30"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"343pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"*p value was calculated using the Fisher exact test."},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0232","num":"0258"},"tables":{"@attributes":{"id":"TABLE-US-00008","num":"00008"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","orient":"land"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"602pt","align":"center"}},"thead":{"row":{"entry":"TABLE 6"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Malignancy-risk gene list."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"15"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"10","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"11","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"12","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"13","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"14","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"15","colwidth":"77pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},{},{},{},{},{},{},{},{},"van't",{},{},{}]},{"entry":[{},{},{},{},{},{},{},{},{},"Chanrion",{},"Veer","Wang","Huang",{}]},{"entry":["Affy probe set id","Gene Symbol","Fold change","FDR","Regulation","DNA replication","Mitosis","Poola et al","Turashvili et al","et al","Ma et al","et al","et al","et al","Gene Title"]},{"entry":{"@attributes":{"namest":"1","nameend":"15","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"15"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"10","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"11","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"12","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"13","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"14","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"15","colwidth":"77pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["222608_s_at","ANLN","4.01","<0.01","Up-Regulated",{},"Y",{},{},"Y","Y","Y",{},{},"anillin, actin binding"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"protein (scraps"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"homolog,"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},{"i":"Drosophila"}]},{"entry":["206632_s_at","APOBEC3B","3.11","<0.01","Up-Regulated",{},{},{},{},{},{},{},{},{},"apolipoprotein B"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"mRNA editing"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"enzyme, catalytic"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"polypeptide-like 3B"]},{"entry":["208750_s_at","ARF1","2.05","<0.01","Up-Regulated",{},{},{},{},"Y",{},{},{},{},"ADP-ribosylation"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"factor 1"]},{"entry":["201096_s_at","ARF4","2.16","<0.01","Up-Regulated",{},{},{},{},"Y",{},{},{},{},"ADP-ribosylation"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"factor 4"]},{"entry":["216266_s_at","ARFGEF1","2.43","<0.01","Up-Regulated",{},{},{},"Y",{},{},{},{},{},"ADP-ribosylation"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"factor guanine"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"nucleotide-"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"exchange factor"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"1(brefeldin A-"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"inhibited)"]},{"entry":["219918_s_at","ASPM","4.16","<0.01","Up-Regulated",{},{},{},{},{},{},{},"Y",{},"asp (abnormal"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"spindle)-like,"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"microcephaly"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"associated"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"()"]},{"entry":["208079_s_at","AURKA","2.98","<0.01","Up-Regulated",{},{},{},{},{},{},"Y","Y",{},"serine\/threonine"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"kinase 6"]},{"entry":["202095_s_at","BIRC5","2.95","<0.01","Up-Regulated",{},"Y",{},{},"Y","Y","Y","Y",{},"baculoviral IAP"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"repeat-containing 5"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(survivin)"]},{"entry":["209642_at","BUB1","2.71","<0.01","Up-Regulated",{},"Y",{},{},"Y","Y","Y","Y","Y","BUB1 budding"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"uninhibited by"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"benzimidazoles 1"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"homolog (yeast)"]},{"entry":["203755_at","BUB1B","3.05","<0.01","Up-Regulated",{},"Y",{},{},"Y",{},"Y","Y","Y","BUB1 budding"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"uninhibited by"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"benzimidazoles 1"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"homolog beta"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(yeast)"]},{"entry":["201457_x_at","BUB3","2.03","<0.01","Up-Regulated",{},{},{},{},"Y",{},"Y",{},{},"BUB3 budding"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"uninhibited by"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"benzimidazoles 3"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"homolog (yeast)"]},{"entry":["223361_at","C6orf115","2.4","<0.01","Up-Regulated",{},{},{},{},{},{},{},{},{},"chromosome 6"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"open reading frame"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"115"]},{"entry":["228323_at","CASC5","2.26","<0.01","Up-Regulated",{},{},{},{},{},{},{},{},{},"cancer"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"susceptibility"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"candidate 5"]},{"entry":["214710_s_at","CCNB1","4.03","<0.01","Up-Regulated",{},"Y",{},{},"Y",{},{},"Y","Y","cyclin B1"]},{"entry":["202705_at","CCNB2","2.35","<0.01","Up-Regulated",{},"Y",{},"Y","Y",{},"Y","Y","Y","cyclin B2"]},{"entry":["205034_at","CCNE2","3.99","<0.01","Up-Regulated","Y",{},{},{},"Y","Y","Y","Y",{},"cyclin E2"]},{"entry":["203213_at","CDC2","5.5","<0.01","Up-Regulated",{},"Y",{},{},"Y",{},"Y","Y","Y","Cell division cycle"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"2, G1 to S and G2"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"to M"]},{"entry":["203214_x_at","CDC2","2.89","<0.01","Up-Regulated",{},"Y",{},{},"Y",{},"Y","Y","Y","cell division cycle 2,"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"G1 to S and G2 to M"]},{"entry":["210559_s_at","CDC2","4.14","<0.01","Up-Regulated",{},"Y","Y",{},"Y",{},"Y","Y","Y","cell division cycle 2,"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"G1 to S and G2 to M"]},{"entry":["202870_s_at","CDC20","3.34","<0.01","Up-Regulated",{},"Y",{},{},{},{},{},"Y",{},"CDC20 cell division"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"cycle 20 homolog"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"()"]},{"entry":["223307_at","CDCA3","2.24","<0.01","Up-Regulated",{},"Y",{},"Y",{},{},{},{},{},"cell division cycle"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"associated 3"]},{"entry":["224753_at","CDCA5","2.21","<0.01","Up-Regulated","Y",{},{},{},{},{},{},{},{},"cell division cycle"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"associated 5"]},{"entry":["1555758_a_at","CDKN3","2.85","<0.01","Up-Regulated",{},{},{},{},"Y","Y","Y","Y","Y","cyclin-dependent"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"kinase inhibitor 3"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(CDK2-associated"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"dual specificity"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"phosphatase)"]},{"entry":["209714_s_at","CDKN3","2.97","<0.01","Up-Regulated",{},{},"Y",{},"Y","Y","Y","Y","Y","cyclin-dependent"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"kinase inhibitor 3"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(CDK2-associated"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"dual specificity"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"phosphatase)"]},{"entry":["204962_s_at","CENPA","2.71","<0.01","Up-Regulated",{},{},{},{},"Y","Y","Y","Y","Y","centromere protein"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"A, 17 kDa"]},{"entry":["207828_s_at","CENPF","2.6","<0.01","Up-Regulated",{},{},{},{},"Y",{},"Y","Y","Y","centromere protein"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"F, 350\/400ka"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(mitosin)"]},{"entry":["222848_at","CENPK","2.18","<0.01","Up-Regulated",{},{},{},"Y",{},{},{},{},{},"leucine zipper"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"protein FKSG14"]},{"entry":["218542_at","CEP55","3.46","<0.01","Up-Regulated",{},{},"Y","Y",{},{},"Y","Y",{},"chromosome 10"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"open reading frame 3"]},{"entry":["218252_at","CKAP2","2.72","<0.01","Up-Regulated",{},"Y","Y",{},"Y",{},{},"Y",{},"cytoskeleton"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"associated protein 2"]},{"entry":["204170_s_at","CKS2","6.32","<0.01","Up-Regulated",{},{},{},{},"Y","Y",{},"Y",{},"CDC28 protein"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"kinase regulatory"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"subunit 2"]},{"entry":["205538_at","CORO2A","2.24","<0.01","Up-Regulated",{},{},{},{},{},{},{},{},{},"coronin, actin"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"binding protein, 2A"]},{"entry":["202613_at","CTPS","2.04","<0.01","Up-Regulated",{},{},"Y",{},"Y",{},"Y",{},{},"CTP synthase"]},{"entry":["222958_s_at","DEPDC1","2.4","<0.01","Up-Regulated",{},{},{},{},{},{},"Y",{},{},"DEP domain"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"containing 1"]},{"entry":["218726_at","DKFZp762E1312","2","<0.01","Up-Regulated",{},"Y",{},"Y",{},{},"Y",{},{},"hypothetical protein"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"DKFZp762E1312"]},{"entry":["203764_at","DLG7","2.84","<0.01","Up-Regulated",{},"Y","Y",{},{},{},"Y","Y",{},"discs, large"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"homolog 7"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"()"]},{"entry":["221677_s_at","DONSON","2.42","<0.01","Up-Regulated","Y",{},{},{},"Y",{},{},"Y",{},"downstream"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"neighbor of SON"]},{"entry":["218567_x_at","DPP3","2.33","<0.01","Up-Regulated",{},{},{},{},"Y",{},{},"Y",{},"dipeptidylpeptidase 3"]},{"entry":["232510_s_at","DPP3","2.2","<0.01","Up-Regulated",{},{},{},{},"Y",{},{},"Y",{},"dipeptidylpeptidase 3"]},{"entry":["218585_s_at","DTL","4.78","<0.01","Up-Regulated",{},{},{},"Y",{},{},"Y","Y",{},"denticleless"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"homolog"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"()"]},{"entry":["219787_s_at","ECT2","3.89","<0.01","Up-Regulated",{},"Y",{},{},{},{},{},"Y",{},"epithelial cell"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"transforming"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"sequence 2"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"oncogene"]},{"entry":["203358_s_at","EZH2","2.69","<0.01","Up-Regulated","Y",{},{},{},{},{},"Y","Y","Y","enhancer of zeste"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"homolog 2"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"()"]},{"entry":["225687_at","FAM83D","3.33","<0.01","Up-Regulated",{},{},{},{},{},{},{},{},{},"chromosome 20"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"open reading frame"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"129"]},{"entry":["213007_at","FANCI","2.25","<0.01","Up-Regulated",{},{},{},"Y",{},{},{},{},{},"hypothetical protein"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"FLJ10719"]},{"entry":["202580_x_at","FOXM1","2.37","<0.01","Up-Regulated",{},"Y",{},{},"Y",{},{},"Y",{},"forkhead box M1"]},{"entry":["206102_at","GINS1","3.27","<0.01","Up-Regulated",{},{},{},{},{},{},{},"Y",{},"DNA replication"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"complex GINS"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"protein PSF1"]},{"entry":["205436_s_at","H2AFX","2.14","<0.01","Up-Regulated",{},{},{},{},"Y",{},{},{},{},"H2A histone family,"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"member X"]},{"entry":["200853_at","H2AFZ","2.23","<0.01","Up-Regulated",{},{},{},{},"Y",{},"Y","Y",{},"H2A histone family,"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"member Z"]},{"entry":["213911_s_at","H2AFZ","2.21","<0.01","Up-Regulated",{},{},"Y",{},"Y",{},"Y","Y",{},"H2A histone family,"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"member Z"]},{"entry":["208490_x_at","HIST1H2BF","2.5","<0.01","Up-Regulated",{},{},{},{},{},{},{},{},{},"histone 1, H2bf"]},{"entry":["203744_at","HMGB3","2.71","<0.01","Up-Regulated",{},{},{},{},{},{},"Y",{},{},"high-mobility group"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"box 3"]},{"entry":["207165_at","HMMR","3.05","<0.01","Up-Regulated",{},{},{},{},"Y",{},{},"Y",{},"hyaluronan-"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"mediated motility"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"receptor (RHAMM)"]},{"entry":["217755_at","HN1","3.4","<0.01","Up-Regulated",{},{},{},{},"Y","Y","Y",{},{},"hematological and"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"neurological"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"expressed 1"]},{"entry":["229538_s_at","IQGAP3","2.51","<0.01","Up-Regulated",{},{},{},"Y",{},{},{},{},{},"IQ motif containing"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"GTPase activating"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"protein 3"]},{"entry":["202503_s_at","KIAA0101","5.89","<0.01","Up-Regulated",{},{},{},{},"Y",{},"Y","Y","Y","KIAA0101"]},{"entry":["204444_at","KIF11","3.22","<0.01","Up-Regulated",{},"Y",{},{},{},{},{},"Y","Y","kinesin family"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"member 11"]},{"entry":["218755_at","KIF20A","2.93","<0.01","Up-Regulated",{},{},{},{},{},{},"Y","Y",{},"kinesin family"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"member 20A"]},{"entry":["204709_s_at","KIF23","2.14","<0.01","Up-Regulated",{},"Y",{},{},{},{},"Y","Y","Y","kinesin family"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"member 23"]},{"entry":["218355_at","KIF4A","2.67","<0.01","Up-Regulated",{},{},{},{},"Y",{},"Y","Y",{},"kinesin family"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"member 4A"]},{"entry":["211762_s_at","KPNA2","3.03","<0.01","Up-Regulated",{},{},"Y",{},{},{},"Y",{},"Y","karyopherin alpha 2"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(RAG cohort 1,"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"importin alpha 1) \/\/\/"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"karyopherin alpha 2"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(RAG cohort 1,"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"importin alpha 1)"]},{"entry":["219061_s_at","LAGE3","2.29","<0.01","Up-Regulated",{},{},{},{},{},{},"Y","Y",{},"DNA segment on"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"chromosome X"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(unique) 9879"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"expressed"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"sequence"]},{"entry":["202779_s_at","LOC731049","2.36","<0.01","Up-Regulated",{},{},{},{},{},{},"Y",{},{},"ubiquitin-"]},{"entry":[{},"\/\/\/ UBE2S",{},{},{},{},{},{},{},{},{},{},{},{},"conjugating"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"enzyme E2S"]},{"entry":["1554768_a_at","MAD2L1","2.29","<0.01","Up-Regulated",{},"Y",{},{},{},{},"Y",{},"Y","MAD2 mitotic arrest"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"deficient-like 1"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(yeast)"]},{"entry":["210058_at","MAPK13","2.01","<0.01","Up-Regulated",{},"Y",{},{},"Y",{},{},{},{},"mitogen-activated"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"protein kinase 13"]},{"entry":["202107_s_at","MCM2","2.08","<0.01","Up-Regulated","Y",{},{},"Y","Y",{},{},"Y",{},"MCM2"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"minichromosome"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"maintenance"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"deficient 2, mitotin"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"()"]},{"entry":["212141_at","MCM4","2.14","<0.01","Up-Regulated","Y",{},{},{},"Y",{},{},"Y",{},"MCM4"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"minichromosome"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"maintenance"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"deficient 4 ()"]},{"entry":["222036_s_at","MCM4","2.69","<0.01","Up-Regulated","Y",{},{},{},"Y",{},{},"Y",{},"MCM4"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"minichromosome"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"maintenance"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"deficient 4 ()"]},{"entry":["204825_at","MELK","3.76","<0.01","Up-Regulated",{},"Y","Y",{},{},{},"Y","Y",{},"maternal embryonic"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"leucine zipper"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"kinase"]},{"entry":["218883_s_at","MLF1IP","4.23","<0.01","Up-Regulated","Y",{},"Y",{},{},{},{},"Y",{},"MLF1 interacting"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"protein"]},{"entry":["201298_s_at","MOBK1B","2.1","<0.01","Up-Regulated",{},{},{},{},{},{},{},{},{},"MOB1, Mps One"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"Binder kinase"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"activator-like 1B"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(yeast)"]},{"entry":["217919_s_at","MRPL42","2.03","<0.01","Up-Regulated",{},{},{},{},"Y",{},{},{},{},"mitochondrial"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"ribosomal protein"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"L42"]},{"entry":["218663_at","NCAPG","2.11","<0.01","Up-Regulated",{},{},{},{},{},{},{},"Y",{},"chromosome"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"condensation"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"protein G"]},{"entry":["209520_s_at","NCBP1","2.3","<0.01","Up-Regulated",{},{},{},{},"Y",{},{},{},{},"nuclear cap binding"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"protein subunit 1,"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"80 kDa"]},{"entry":["204162_at","NDC80","2.41","<0.01","Up-Regulated",{},{},{},{},{},{},"Y",{},{},"kinetochore"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"associated 2"]},{"entry":["204641_at","NEK2","5.55","<0.01","Up-Regulated",{},{},{},"Y",{},{},"Y","Y","Y","NIMA (never in"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"mitosis gene a)-"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"related kinase 2"]},{"entry":["201577_at","NME1","2.15","<0.01","Up-Regulated",{},{},{},{},"Y","Y",{},"Y","Y","non-metastatic cells"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"1, protein (NM23A)"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"expressed in"]},{"entry":["212316_at","NUP210","2.23","<0.01","Up-Regulated",{},{},{},{},{},{},{},{},{},"nucleoporin"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"210 kDa"]},{"entry":["218039_at","NUSAP1","6.41","<0.01","Up-Regulated",{},"Y",{},"Y",{},{},"Y","Y",{},"nucleolar and"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"spindle associated"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"protein 1"]},{"entry":["219978_s_at","NUSAP1","5","<0.01","Up-Regulated",{},"Y","Y",{},{},{},"Y","Y",{},"nucleolar and"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"spindle associated"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"protein 1"]},{"entry":["203228_at","PAFAH1B3","2.68","<0.01","Up-Regulated",{},{},{},{},"Y","Y",{},"Y",{},"platelet-activating"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"factor"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"acetylhydrolase,"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"isoform Ib, gamma"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"subunit 29 kDa"]},{"entry":["219148_at","PBK","3.06","<0.01","Up-Regulated",{},"Y",{},{},{},{},{},"Y",{},"PDZ binding kinase"]},{"entry":["201202_at","PCNA","2.45","<0.01","Up-Regulated","Y",{},{},{},{},{},"Y","Y",{},"proliferating cell"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"nuclear antigen"]},{"entry":["201490_s_at","PPIF","2.4","<0.01","Up-Regulated",{},{},{},{},"Y",{},{},{},{},"peptidylprolyl"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"isomerase F"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(cyclophilin F)"]},{"entry":["218009_s_at","PRC1","3.72","<0.01","Up-Regulated",{},{},{},{},{},"Y","Y","Y",{},"protein regulator of"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"cytokinesis 1"]},{"entry":["222077_s_at","RACGAP1","3.36","<0.01","Up-Regulated",{},{},{},"Y","Y","Y","Y","Y",{},"Rac GTPase"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"activating protein 1"]},{"entry":["204146_at","RAD51AP1","2.79","<0.01","Up-Regulated","Y",{},{},{},{},{},{},"Y",{},"RAD51 associated"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"protein 1"]},{"entry":["201890_at","RRM2","8.07","<0.01","Up-Regulated","Y",{},{},{},{},"Y","Y","Y",{},"ribonucleotide"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"reductase M2"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"polypeptide"]},{"entry":["209773_s_at","RRM2","6.73","<0.01","Up-Regulated","Y",{},"Y",{},{},"Y","Y","Y",{},"ribonucleotide"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"reductase M2"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"polypeptide"]},{"entry":["204240_s_at","SMC2","2.01","<0.01","Up-Regulated","Y",{},"Y",{},{},{},{},{},{},"SMC2 structural"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"maintenance of"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"chromosomes 2-"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"like 1 (yeast)"]},{"entry":["201663_s_at","SMC4","2.44","<0.01","Up-Regulated","Y",{},{},{},{},{},{},"Y",{},"SMC4 structural"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"maintenance of"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"chromosomes 4-"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"like 1 (yeast)"]},{"entry":["209875_s_at","SPP1","5.7","<0.01","Up-Regulated",{},{},{},{},"Y",{},{},"Y",{},"secreted"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"phosphoprotein 1"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(osteopontin, bone"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"sialoprotein I, early"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"T-lymphocyte"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"activation 1)"]},{"entry":["209218_at","SQLE","3.25","<0.01","Up-Regulated",{},{},{},"Y","Y","Y",{},"Y","Y","squalene"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"epoxidase"]},{"entry":["203046_s_at","TIMELESS","2.12","<0.01","Up-Regulated",{},{},{},{},"Y",{},{},"Y",{},"timeless homolog"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"()"]},{"entry":["1554408_a_at","TK1","2.72","<0.01","Up-Regulated","Y",{},{},{},"Y","Y","Y","Y",{},"thymidine kinase 1,"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"soluble"]},{"entry":["202338_at","TK1","2.86","<0.01","Up-Regulated","Y",{},{},{},"Y","Y","Y","Y",{},"thymidine kinase 1,"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"soluble"]},{"entry":["222642_s_at","TMEM33","2.01","<0.01","Up-Regulated",{},{},{},{},{},{},{},{},{},"transmembrane"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"protein 33"]},{"entry":["201291_s_at","TOP2A","7.56","<0.01","Up-Regulated",{},"Y",{},{},"Y","Y",{},"Y","Y","topoisomerase"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(DNA) II alpha"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"170 kDa"]},{"entry":["201292_at","TOP2A","6.03","<0.01","Up-Regulated",{},"Y",{},{},"Y","Y",{},"Y","Y","topoisomerase"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(DNA) II alpha"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"170 kDa"]},{"entry":["210052_s_at","TPX2","3.73","<0.01","Up-Regulated",{},"Y","Y",{},{},{},{},{},{},"TPX2, microtubule-"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"associated,"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"homolog ("]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},{"i":"laevis"}]},{"entry":["204822_at","TTK","3.27","<0.01","Up-Regulated",{},"Y",{},{},"Y",{},"Y","Y",{},"TTK protein kinase"]},{"entry":["1554696_s_at","TYMS","2.05","<0.01","Up-Regulated","Y",{},{},{},"Y",{},"Y",{},{},"thymidylate"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"synthetase"]},{"entry":["202954_at","UBE2C","3.26","<0.01","Up-Regulated",{},{},{},{},{},{},"Y",{},"Y","ubiquitin-"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"conjugating"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"enzyme E2C"]},{"entry":["223229_at","UBE2T","4.99","<0.01","Up-Regulated",{},{},{},"Y",{},{},"Y",{},{},"ubiquitin-"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"conjugating"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"enzyme E2T"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(putative)"]},{"entry":["225655_at","UHRF1","6.34","<0.01","Up-Regulated",{},{},{},{},{},{},"Y",{},{},"ubiquitin-like,"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"containing PHD"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"and RING finger"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"domains, 1"]},{"entry":["222804_x_at","WDR32","2.04","<0.01","Up-Regulated",{},{},{},"Y",{},{},{},{},{},"WD repeat domain"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"32"]},{"entry":["225676_s_at","WDSOF1","2.26","<0.01","Up-Regulated",{},{},{},{},{},{},{},{},{},"WD repeats and"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"SOF1 domain"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"containing"]},{"entry":["218349_s_at","ZWILCH","2.11","<0.01","Up-Regulated",{},{},"Y",{},{},{},{},{},{},"Zwilch, kinetochore"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"associated,"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"homolog"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"()"]},{"entry":["204026_s_at","ZWINT","4.46","<0.01","Up-Regulated",{},{},{},{},"Y",{},"Y","Y","Y","ZW10 interactor"]},{"entry":["228273_at","FLJ11029","3.77","<0.01","Up-Regulated",{},{},{},"Y",{},{},{},{},{},"Hypothetical protein"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"FLJ11029"]},{"entry":["229490_s_at","IQGAP3","2.52","<0.01","Up-Regulated",{},{},{},{},{},{},{},{},{},"IQ motif containing"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"GTPase activating"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"protein 3"]},{"entry":["203002_at","AMOTL2","2.39","<0.01","Down-",{},{},{},{},{},{},{},{},{},"angiomotin like 2"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},{}]},{"entry":["212914_at","CBX7","2.61","<0.01","Down-",{},{},"Y",{},{},{},{},"Y","Y","chromobox"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"homolog 7"]},{"entry":["228693_at","CCDC50","3.07","<0.01","Down-",{},{},{},{},{},{},{},{},{},"chromosome 3"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"open reading frame 6"]},{"entry":["213348_at","CDKN1C","3.65","<0.01","Down-",{},{},{},"Y","Y",{},{},{},"Y","Cyclin-dependent"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"kinase inhibitor 1C"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(p57, Kip2)"]},{"entry":["224352_s_at","CFL2","2.42","<0.01","Down-",{},{},{},"Y","Y",{},{},{},{},"cofilin 2 (muscle) \/\/\/"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"cofilin 2 (muscle)"]},{"entry":["209763_at","CHRDL1","8.05","<0.01","Down-",{},{},{},{},{},{},{},{},{},"chordin-like 1"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},{}]},{"entry":["204455_at","DST","11.94","<0.01","Down-",{},{},{},"Y",{},{},{},{},{},"dystonin"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},{}]},{"entry":["206101_at","ECM2","2.97","<0.01","Down-",{},{},{},{},"Y",{},{},{},{},"extracellular matrix"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"protein 2, female"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"organ and"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"adipocyte specific"]},{"entry":["204363_at","F3","2.5","<0.01","Down-",{},{},{},{},{},{},{},"Y",{},"coagulation factor"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"III (thromboplastin,"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"tissue factor)"]},{"entry":["209220_at","GPC3","3.87","<0.01","Down-",{},{},{},"Y","Y",{},{},{},{},"glypican 3"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},{}]},{"entry":["204793_at","GPRASP1","3.04","<0.01","Down-",{},{},{},{},{},{},{},{},{},"G protein-coupled"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"receptor associated"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"sorting protein 1"]},{"entry":["209894_at","LEPR","5.51","<0.01","Down-",{},{},{},{},"Y",{},"Y",{},{},"leptin receptor"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},{}]},{"entry":["225956_at","LOC153222","2.22","<0.01","Down-",{},"Y",{},{},{},{},{},{},{},"adult retina protein"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},{}]},{"entry":["209737_at","MAGI2","2.06","<0.01","Down-",{},{},{},{},{},{},{},{},{},"membrane"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"associated"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"guanylate kinase,"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"WW and PDZ"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"domain containing 2"]},{"entry":["217546_at","MT1M","3.14","<0.01","Down-",{},{},{},"Y",{},{},{},{},{},"metallothionein 1M"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},{}]},{"entry":["209493_at","PDZD2","2.38","<0.01","Down-",{},{},{},"Y",{},{},{},{},{},"PDZ domain"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"containing 3"]},{"entry":["238447_at","RBMS3","5.26","<0.01","Down-",{},{},{},{},{},{},"Y",{},{},"RNA binding motif,"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"single stranded"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"interacting protein"]},{"entry":["222717_at","SDPR","6.01","<0.01","Down-",{},{},{},{},"Y",{},{},{},{},"serum deprivation"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"response"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(phosphatidylserine"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"binding protein)"]},{"entry":["227662_at","SYNPO2","4.91","<0.01","Down-",{},{},{},{},{},{},{},{},{},"synaptopodin 2"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},{}]},{"entry":["225093_at","UTRN","2.24","<0.01","Down-",{},{},{},"Y","Y",{},{},{},{},"utrophin"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"(homologous to"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"dystrophin)"]},{"entry":["235308_at","ZBTB20","2.89","<0.01","Down-",{},{},{},"Y",{},{},{},{},{},"zinc finger and BTB"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"domain containing"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"20"]},{"entry":["213158_at",{},"3.84","<0.01","Down-",{},{},{},{},{},{},{},{},{},"MRNA; cDNA"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"DKFZp586B211"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(from clone"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"DKFZp586B211)"]},{"entry":["226250_at",{},"2.81","<0.01","Down-",{},{},{},{},{},{},{},{},{},"CDNA FLJ34585"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"fis, clone"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"KIDNE2008758"]},{"entry":["226252_at",{},"2.89","<0.01","Down-",{},{},{},{},{},{},{},{},{},"CDNA FLJ34585"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"fis, clone"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"KIDNE2008758"]},{"entry":["227082_at",{},"3.51","<0.01","Down-",{},{},{},{},{},{},{},{},{},"MRNA; cDNA"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"DKFZp586K1922"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(from clone"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"DKFZp586K1922)"]},{"entry":["227121_at",{},"2.55","<0.01","Down-",{},{},{},{},{},{},{},{},{},"MRNA; cDNA"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"DKFZp586K1922"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"(from clone"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"DKFZp586K1922)"]},{"entry":["227646_at","EBF","5.35","<0.01","Down-",{},{},{},{},{},{},{},{},{},"Early B-cell factor"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},{}]},{"entry":["227719_at",{},"2.21","<0.01","Down-",{},{},{},{},{},{},{},{},{},"CDNA FLJ37828"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"fis, clone"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"BRSSN2006575"]},{"entry":["235556_at",{},"2.18","<0.01","Down-",{},{},{},{},{},{},{},{},{},"Transcribed locus,"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"weakly similar to"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"NP_703324.1"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"glutamic acid-rich"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"protein (garp)"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"["]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},{"i":"falciparum "}]},{"entry":["235570_at","RBMS3","4.02","<0.01","Down-",{},{},{},{},{},{},{},{},{},"RNA binding motif,"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"single stranded"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"interacting protein"]},{"entry":["243584_at",{},"3.93","<0.01","Down-",{},{},{},{},{},{},{},{},{},"Transcribed locus,"]},{"entry":[{},{},{},{},"Regulated",{},{},{},{},{},{},{},{},{},"weakly similar to"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"NP_060190.1"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"signal-transducing"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"adaptor protein-2;"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"brk kinase"]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},"substrate ["]},{"entry":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},{"i":"sapiens"}]},{"entry":{"@attributes":{"namest":"1","nameend":"15","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0233","num":"0259"},"tables":{"@attributes":{"id":"TABLE-US-00009","num":"00009"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"315pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 7"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"A subset of malignancy-risk genes associated with DNA replication, mitosis,"},{"entry":"cancer risk, disease relapse\/progression, and metastasis*."},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"9"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"9","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Affy probe set","Gene","Fold",{},{},"DNA",{},"Poola","Turashvili et"]},{"entry":["id","Symbol","change","FDR","Regulation","replication","Mitosis","et al","al"]},{"entry":{"@attributes":{"namest":"1","nameend":"9","align":"center","rowsep":"1"}}},{"entry":["222608_s_at","ANLN","4.01","<0.01","Up-Regulated",{},"Y",{},{}]},{"entry":["202095_s_at","BIRC5","2.95","<0.01","Up-Regulated",{},"Y",{},{}]},{"entry":["209642_at","BUB1","2.71","<0.01","Up-Regulated",{},"Y",{},{}]},{"entry":["203755_at","BUB1B","3.05","<0.01","Up-Regulated",{},"Y",{},{}]},{"entry":["214710_s_at","CCNB1","4.03","<0.01","Up-Regulated",{},"Y",{},{}]},{"entry":["202705_at","CCNB2","2.35","<0.01","Up-Regulated",{},"Y",{},"Y"]},{"entry":["205034_at","CCNE2","3.99","<0.01","Up-Regulated","Y",{},{},{}]},{"entry":["203213_at","CDC2","5.5","<0.01","Up-Regulated",{},"Y",{},{}]},{"entry":["203214_x_at","CDC2","2.89","<0.01","Up-Regulated",{},"Y",{},{}]},{"entry":["210559_s_at","CDC2","4.14","<0.01","Up-Regulated",{},"Y","Y",{}]},{"entry":["1555758_a_at","CDKN3","2.85","<0.01","Up-Regulated",{},{},{},{}]},{"entry":["209714_s_at","CDKN3","2.97","<0.01","Up-Regulated",{},{},"Y",{}]},{"entry":["204962_s_at","CENPA","2.71","<0.01","Up-Regulated",{},{},{},{}]},{"entry":["207828_s_at","CENPF","2.6","<0.01","Up-Regulated",{},{},{},{}]},{"entry":["218542_at","CEP55","3.46","<0.01","Up-Regulated",{},{},"Y","Y"]},{"entry":["218252_at","CKAP2","2.72","<0.01","Up-Regulated",{},"Y","Y",{}]},{"entry":["203764_at","DLG7","2.84","<0.01","Up-Regulated",{},"Y","Y",{}]},{"entry":["203358_s_at","EZH2","2.69","<0.01","Up-Regulated","Y",{},{},{}]},{"entry":["213911_s_at","H2AFZ","2.21","<0.01","Up-Regulated",{},{},"Y",{}]},{"entry":["202503_s_at","KIAA0101","5.89","<0.01","Up-Regulated",{},{},{},{}]},{"entry":["204709_s_at","KIF23","2.14","<0.01","Up-Regulated",{},"Y",{},{}]},{"entry":["202107_s_at","MCM2","2.08","<0.01","Up-Regulated","Y",{},{},"Y"]},{"entry":["204825_at","MELK","3.76","<0.01","Up-Regulated",{},"Y","Y",{}]},{"entry":["204641_at","NEK2","5.55","<0.01","Up-Regulated",{},{},{},"Y"]},{"entry":["201577_at","NME1","2.15","<0.01","Up-Regulated",{},{},{},{}]},{"entry":["218039_at","NUSAP1","6.41","<0.01","Up-Regulated",{},"Y",{},"Y"]},{"entry":["219978_s_at","NUSAP1","5","<0.01","Up-Regulated",{},"Y","Y",{}]},{"entry":["222077_s_at","RACGAP1","3.36","<0.01","Up-Regulated",{},{},{},"Y"]},{"entry":["201890_at","RRM2","8.07","<0.01","Up-Regulated","Y",{},{},{}]},{"entry":["209773_s_at","RRM2","6.73","<0.01","Up-Regulated","Y",{},"Y",{}]},{"entry":["209218_at","SQLE","3.25","<0.01","Up-Regulated",{},{},{},"Y"]},{"entry":["1554408_a_at","TK1","2.72","<0.01","Up-Regulated","Y",{},{},{}]},{"entry":["202338_at","TK1","2.86","<0.01","Up-Regulated","Y",{},{},{}]},{"entry":["201291_s_at","TOP2A","7.56","<0.01","Up-Regulated",{},"Y",{},{}]},{"entry":["201292_at","TOP2A","6.03","<0.01","Up-Regulated",{},"Y",{},{}]},{"entry":["204822_at","TTK","3.27","<0.01","Up-Regulated",{},"Y",{},{}]},{"entry":["204026_s_at","ZWINT","4.46","<0.01","Up-Regulated"]},{"entry":{"@attributes":{"namest":"1","nameend":"9","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"8"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"98pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},{},"van't",{},{},{}]},{"entry":[{},"Affy probe set","Chanrion","Ma","Veer","Wang","Huang",{}]},{"entry":[{},"id","et al","et al","et al","et al","et al","Gene Title"]},{"entry":{"@attributes":{"namest":"1","nameend":"8","align":"center","rowsep":"1"}}},{"entry":[{},"222608_s_at","Y","Y","Y",{},{},"anillin, actin binding protein"]},{"entry":[{},{},{},{},{},{},{},"(scraps homolog, )"]},{"entry":[{},"202095_s_at","Y","Y","Y","Y",{},"baculoviral IAP repeat-"]},{"entry":[{},{},{},{},{},{},{},"containing 5 (survivin)"]},{"entry":[{},"209642_at","Y","Y","Y","Y","Y","BUB1 budding uninhibited by"]},{"entry":[{},{},{},{},{},{},{},"benzimidazoles 1 homolog"]},{"entry":[{},{},{},{},{},{},{},"(yeast)"]},{"entry":[{},"203755_at","Y",{},"Y","Y","Y","BUB1 budding uninhibited by"]},{"entry":[{},{},{},{},{},{},{},"benzimidazoles 1 homolog"]},{"entry":[{},{},{},{},{},{},{},"beta (yeast)"]},{"entry":[{},"214710_s_at","Y",{},{},"Y","Y","cyclin B1"]},{"entry":[{},"202705_at","Y",{},"Y","Y","Y","cyclin B2"]},{"entry":[{},"205034_at","Y","Y","Y","Y",{},"cyclin E2"]},{"entry":[{},"203213_at","Y",{},"Y","Y","Y","Cell division cycle 2, G1 to S"]},{"entry":[{},{},{},{},{},{},{},"and G2 to M"]},{"entry":[{},"203214_x_at","Y",{},"Y","Y","Y","cell division cycle 2, G1 to S"]},{"entry":[{},{},{},{},{},{},{},"and G2 to M"]},{"entry":[{},"210559_s_at","Y",{},"Y","Y","Y","cell division cycle 2, G1 to S"]},{"entry":[{},{},{},{},{},{},{},"and G2 to M"]},{"entry":[{},"1555758_a_at","Y","Y","Y","Y","Y","cyclin-dependent kinase"]},{"entry":[{},{},{},{},{},{},{},"inhibitor 3 (CDK2-associated"]},{"entry":[{},{},{},{},{},{},{},"dual specificity phosphatase)"]},{"entry":[{},"209714_s_at","Y","Y","Y","Y","Y","cyclin-dependent kinase"]},{"entry":[{},{},{},{},{},{},{},"inhibitor 3 (CDK2-associated"]},{"entry":[{},{},{},{},{},{},{},"dual specificity phosphatase)"]},{"entry":[{},"204962_s_at","Y","Y","Y","Y","Y","centromere protein A, 17 kDa"]},{"entry":[{},"207828_s_at","Y",{},"Y","Y","Y","centromere protein F,"]},{"entry":[{},{},{},{},{},{},{},"350\/400ka (mitosin)"]},{"entry":[{},"218542_at",{},{},"Y","Y",{},"chromosome 10 open"]},{"entry":[{},{},{},{},{},{},{},"reading frame 3"]},{"entry":[{},"218252_at","Y",{},{},"Y",{},"cytoskeleton associated"]},{"entry":[{},{},{},{},{},{},{},"protein 2"]},{"entry":[{},"203764_at",{},{},"Y","Y",{},"discs, large homolog 7"]},{"entry":[{},{},{},{},{},{},{},"()"]},{"entry":[{},"203358_s_at",{},{},"Y","Y","Y","enhancer of zeste homolog 2"]},{"entry":[{},{},{},{},{},{},{},"()"]},{"entry":[{},"213911_s_at","Y",{},"Y","Y",{},"H2A histone family, member Z"]},{"entry":[{},"202503_s_at","Y",{},"Y","Y","Y","KIAA0101"]},{"entry":[{},"204709_s_at",{},{},"Y","Y","Y","kinesin family member 23"]},{"entry":[{},"202107_s_at","Y",{},{},"Y",{},"MCM2 minichromosome"]},{"entry":[{},{},{},{},{},{},{},"maintenance deficient 2,"]},{"entry":[{},{},{},{},{},{},{},"mitotin ()"]},{"entry":[{},"204825_at",{},{},"Y","Y",{},"maternal embryonic leucine"]},{"entry":[{},{},{},{},{},{},{},"zipper kinase"]},{"entry":[{},"204641_at",{},{},"Y","Y","Y","NIMA (never in mitosis gene"]},{"entry":[{},{},{},{},{},{},{},"a)-related kinase 2"]},{"entry":[{},"201577_at","Y","Y",{},"Y","Y","non-metastatic cells 1,"]},{"entry":[{},{},{},{},{},{},{},"protein (NM23A) expressed"]},{"entry":[{},{},{},{},{},{},{},"in"]},{"entry":[{},"218039_at",{},{},"Y","Y",{},"nucleolar and spindle"]},{"entry":[{},{},{},{},{},{},{},"associated protein 1"]},{"entry":[{},"219978_s_at",{},{},"Y","Y",{},"nucleolar and spindle"]},{"entry":[{},{},{},{},{},{},{},"associated protein 1"]},{"entry":[{},"222077_s_at","Y","Y","Y","Y",{},"Rac GTPase activating"]},{"entry":[{},{},{},{},{},{},{},"protein 1"]},{"entry":[{},"201890_at",{},"Y","Y","Y",{},"ribonucleotide reductase M2"]},{"entry":[{},{},{},{},{},{},{},"polypeptide"]},{"entry":[{},"209773_s_at",{},"Y","Y","Y",{},"ribonucleotide reductase M2"]},{"entry":[{},{},{},{},{},{},{},"polypeptide"]},{"entry":[{},"209218_at","Y","Y",{},"Y","Y","squalene epoxidase"]},{"entry":[{},"1554408_a_at","Y","Y","Y","Y",{},"thymidine kinase 1, soluble"]},{"entry":[{},"202338_at","Y","Y","Y","Y",{},"thymidine kinase 1, soluble"]},{"entry":[{},"201291_s_at","Y","Y",{},"Y","Y","topoisomerase (DNA) II alpha"]},{"entry":[{},{},{},{},{},{},{},"170 kDa"]},{"entry":[{},"201292_at","Y","Y",{},"Y","Y","topoisomerase (DNA) II alpha"]},{"entry":[{},{},{},{},{},{},{},"170 kDa"]},{"entry":[{},"204822_at","Y",{},"Y","Y",{},"TTK protein kinase"]},{"entry":[{},"204026_s_at","Y",{},"Y","Y","Y","ZW10 interactor"]},{"entry":{"@attributes":{"namest":"1","nameend":"8","align":"center","rowsep":"1"}}},{"entry":"*\u201cY\u201d symbol was used to indicate the association of each malignancy-risk gene with DNA replication, mitosis, cancer risk (Poola et al or Turashvili et al), cancer relapse\/progression (Chanrion et al or Ma et al), or metastasis (van't Veer et al, Wang et al, or Huang et al)."}]}}]}}},{"@attributes":{"id":"p-0234","num":"0260"},"tables":{"@attributes":{"id":"TABLE-US-00010","num":"00010"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 8"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"External evaluation for classification of normal"},{"entry":"and IDC tissues in Turashvili et al's study."},{"entry":"Malignancy-risk genes with p value < 0.05"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Affy.probe.set.id","Gene Symbol","Regulation","P value"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["216266_s_at","ARFGEF1","Up-Regulated","0.0318"]},{"entry":["202705_at","CCNB2","Up-Regulated","0.0398"]},{"entry":["223307_at","CDCA3","Up-Regulated","0.0210"]},{"entry":["213348_at","CDKN1C","Down-","0.0306"]},{"entry":[{},{},"Regulated"]},{"entry":["222848_at","CENPK","Up-Regulated","0.0223"]},{"entry":["218542_at","CEP55","Up-Regulated","0.0397"]},{"entry":["224352_s_at","CFL2","Down-","0.0301"]},{"entry":[{},{},"Regulated"]},{"entry":["218726_at","DKFZp762E1312","Up-Regulated","0.0391"]},{"entry":["204455_at","DST","Down-","0.0455"]},{"entry":[{},{},"Regulated"]},{"entry":["218585_s_at","DTL","Up-Regulated","0.0126"]},{"entry":["213007_at","FANCI","Up-Regulated","0.0015"]},{"entry":["209220_at","GPC3","Down-","0.0272"]},{"entry":[{},{},"Regulated"]},{"entry":["229538_s_at","IQGAP3","Up-Regulated","0.0320"]},{"entry":["202107_s_at","MCM2","Up-Regulated","0.0185"]},{"entry":["217546_at","MT1M","Down-","0.0439"]},{"entry":[{},{},"Regulated"]},{"entry":["204641_at","NEK2","Up-Regulated","0.0380"]},{"entry":["218039_at","NUSAP1","Up-Regulated","0.0243"]},{"entry":["209493_at","PDZD2","Down-","0.0356"]},{"entry":[{},{},"Regulated"]},{"entry":["222077_s_at","RACGAP1","Up-Regulated","0.0353"]},{"entry":["209218_at","SQLE","Up-Regulated","0.0458"]},{"entry":["223229_at","UBE2T","Up-Regulated","0.0478"]},{"entry":["225093_at","UTRN","Down-","0.0269"]},{"entry":[{},{},"Regulated"]},{"entry":["222804_x_at","WDR32","Up-Regulated","0.0371"]},{"entry":["235308_at","ZBTB20","Down-","0.0081"]},{"entry":[{},{},"Regulated"]},{"entry":["228273_at",{},"Up-Regulated","0.0243"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0235","num":"0261"},"tables":{"@attributes":{"id":"TABLE-US-00011","num":"00011"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"441pt","align":"center"}},"thead":{"row":{"entry":"TABLE 9"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Summary table of analysis results on seven external datasets for the"},{"entry":"clinical association of the malignancy-risk gene signature."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"70pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Sample",{},"Statistics",{},{}]},{"entry":["Dataset","size (n)","Endpoint","method","Test statistics","p value"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"70pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Cancer risk",{},{},{},{},{}]},{"entry":["Turashvili et al.'s IDC study","10","IDC versus normal","random effect",{},"p = 0.029"]},{"entry":[{},{},{},"model",{},{}]},{"entry":["Poola et al's ADH study","8","cancer (ADHC) versus no","logistic","OR = 1.4 (continuous risk score);","p = 0.131 (cont.) and"]},{"entry":[{},{},"cancer (ADH)","regression","OR = 9 (binary score)","p = 0.178 (binary)"]},{"entry":["Cancer relapse\/progression",{},{},{},{},{}]},{"entry":["Chanrion et al's Tamoxifen-","155","relapse of primary breast","logistic","OR = 7.82","<0.0001"]},{"entry":["Treated Primary Breast",{},"cancer","regression",{},{}]},{"entry":["Cancer",{},{},{},{},{}]},{"entry":["Ma et al's breast cancer study","61","disease status (ADH,","correlation","r = 0.50 (Pearson or Spearman)","<0.0001"]},{"entry":[{},{},"DCIS, IDC)","analysis",{},{}]},{"entry":[{},{},{},"logistic","OR (DCIS) = 2.28 (compared to","p = 0.016"]},{"entry":[{},{},{},"regression","ADH)",{}]},{"entry":[{},{},{},"logistic","OR (IDC) = 3.31 (compared to","p = 0.008"]},{"entry":[{},{},{},"regression","ADH)",{}]},{"entry":["Prognosis",{},{},{},{},{}]},{"entry":["van't Veer et al's breast","training = 78","time to metastasis","log-rank test","X= 12.2 (training set); X= 22.4","p = 0.0005 (training);"]},{"entry":["metastasis dataset","test = 263",{},{},"(test set)","<0.0001 (test)"]},{"entry":["Wang et al's breast cancer","286","metastasis-free survival","log-rank test","X= 12.6","p = 0.0004"]},{"entry":["relapse free survival study",{},{},{},{},{}]},{"entry":["Huang et al's breast lymph","37","lynph node (pos vs. neg)","logistic","OR = 7.29","p = 0.007"]},{"entry":["node study",{},{},"regression"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]}}]}}}],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0023","num":"0022"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0024","num":"0023"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0025","num":"0024"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0026","num":"0025"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0027","num":"0026"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0028","num":"0027"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0029","num":"0028"},"figref":"FIG. 7"},{"@attributes":{"id":"p-0030","num":"0029"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0031","num":"0030"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0032","num":"0031"},"figref":"FIG. 10"},{"@attributes":{"id":"p-0033","num":"0032"},"figref":"FIG. 11"},{"@attributes":{"id":"p-0034","num":"0033"},"figref":"FIG. 12"},{"@attributes":{"id":"p-0035","num":"0034"},"figref":"FIG. 13"},{"@attributes":{"id":"p-0036","num":"0035"},"figref":"FIG. 14"},{"@attributes":{"id":"p-0037","num":"0036"},"figref":"FIG. 15"},{"@attributes":{"id":"p-0038","num":"0037"},"figref":"FIG. 16"},{"@attributes":{"id":"p-0039","num":"0038"},"figref":"FIG. 17"},{"@attributes":{"id":"p-0040","num":"0039"},"figref":"FIG. 18"},{"@attributes":{"id":"p-0041","num":"0040"},"figref":"FIG. 19"},{"@attributes":{"id":"p-0042","num":"0041"},"figref":"FIG. 20"},{"@attributes":{"id":"p-0043","num":"0042"},"figref":"FIG. 21"},{"@attributes":{"id":"p-0044","num":"0043"},"figref":"FIG. 22"},{"@attributes":{"id":"p-0045","num":"0044"},"figref":"FIG. 23"},{"@attributes":{"id":"p-0046","num":"0045"},"figref":"FIG. 24"},{"@attributes":{"id":"p-0047","num":"0046"},"figref":["FIG. 25A","FIG. 25B"]},{"@attributes":{"id":"p-0048","num":"0047"},"figref":["FIG. 26A","FIG. 26B"]},{"@attributes":{"id":"p-0049","num":"0048"},"figref":"FIG. 27"},{"@attributes":{"id":"p-0050","num":"0049"},"figref":"FIG. 28"},{"@attributes":{"id":"p-0051","num":"0050"},"figref":"FIG. 29"},{"@attributes":{"id":"p-0052","num":"0051"},"figref":"FIG. 30"},{"@attributes":{"id":"p-0053","num":"0052"},"figref":["FIG. 31","FIG. 31A","FIG. 31B"]},{"@attributes":{"id":"p-0054","num":"0053"},"figref":["FIG. 32","FIG. 32A","FIG. 32B"]},{"@attributes":{"id":"p-0055","num":"0054"},"figref":["FIG. 33","FIG. 33A","FIG. 33B","FIGS. 33C"]},{"@attributes":{"id":"p-0056","num":"0055"},"figref":["FIG. 34","FIG. 34A","FIG. 34B"]},{"@attributes":{"id":"p-0057","num":"0056"},"figref":["FIG. 35","FIG. 35A","FIG. 35B","FIG. 35C","FIG. 35D"]},{"@attributes":{"id":"p-0058","num":"0057"},"figref":["FIG. 36","FIG. 36","FIG. 36B"]}]},"DETDESC":[{},{}]}
