---
title: Methods for controlling flower development in plants
abstract: The present invention provides a method of controlling the sexuality of a plant comprising treating the plant with a composition comprising a compound selected from the group consisting of jasmonic acid, a jasmonic acid derivative, and a salt thereof.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09258998&OS=09258998&RS=09258998
owner: Yale University
number: 09258998
owner_city: New Haven
owner_country: US
publication_date: 20100108
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATION","STATEMENT REGARDING FEDERALLY SPONSORED RESEARCH OR DEVELOPMENT","BACKGROUND OF THE INVENTION","BRIEF SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","EXAMPLES","Example 1","Example 2","Example 3","Example 4","Example 5","Example 6"],"p":["The present application is a 35 U.S.C. \u00a7371 national phase application from, and claiming priority to, International Application No. PCT\/US2010\/020505, filed Jan. 8, 2010, and published under PCT Article 21(2) in English, which is entitled to priority under 35 U.S.C. \u00a7119(e) to U.S. Provisional Application No. 61\/143,394, filed Jan. 8, 2009, all of which applications are hereby incorporated by reference herein in their entireties.","This invention was made with government support under Grant No. R01 GM38148, awarded by the National Institutes of Health. The Government has certain rights in the invention.","Most flowering plants produce perfect flowers containing both the male organs (stamens) and female organs (pistils). In maize, which has physically separated male and female inflorescences, floral meristems become unisexual through sex determination (Dellaporta & Calderon-Urrea, 1994, Science 266:1501; Irish, 1996, Bioessays 18:363). The basic unit of the maize inflorescence, called a spikelet, contains one upper and one lower flower (known as florets in grasses). Each floret initiates a series of floral organs including three stamen primordia and a central pistil primordium (Bonnet, 1940, J. Agric. Res. 60:25; Kiesselbach, \u201cThe Structure and Reproduction of Corn,\u201d Univ. of Nebraska Press, Lincoln, Nebr., 1949). These initially bisexual florets become exclusively staminate in the tassel (by abortion of pistil primordia) and exclusively pistillate in the ear (by arrest of developing stamens) (Bonnet, 1940, J. Agric. Res. 60:25; Cheng et al., 1983, Am. Bot. 70:450). Each ear spikelet produces a solitary functional pistil in the upper floret due to abortion of the pistil in the lower floret (Bonnet, 1940, J. Agric. Res. 60:25; Kiesselbach, \u201cThe Structure and Reproduction of Corn,\u201d Univ. of Nebraska Press, Lincoln, Nebr., 1949; Cheng et al., 1983, Am. Bot. 70:450).","Mutations altering the sexual fate of florets in maize indicate that sex determination is under genetic control. The non-homeotic tasselseed (ts) mutations ts1 and ts2 result in the conversion of the tassel inflorescence from staminate to pistillate (Emerson, 1920, J. Hered. 11:65; Nickerson & Dale, 1955, Ann. Mo. Bot. Gard. 42:195). Both ts1 and ts2 are required to eliminate pistil primordia through cell death (Calderon-Urrea & Dellaporta, 1999, Development 126:435; Kim et al., 2007, Genetics 177:2547). The ts2 gene encodes a short-chain dehydrogenase\/reductase (DeLong et al., 1993, Cell 74:757) with broad activity, which has complicated the discovery of its natural substrate (Wu et al., 2007, FEBS J. 274:1172). It is unknown how is genes mediate pistil cell death, although it has been suggested that the dehydrogenase\/reductase activity of ts2 may produce a pro-apoptotic signal or metabolize a substrate required for cell viability (Calderon-Urrea & Dellaporta, 1999, Development 126:435; Wu et al., 2007, FEBS J. 274:1172). Even less is known about the ts1 gene. TS2 transcripts are low or undetectable in ts1 mutant tassels, which suggests that ts1 may act upstream of ts2 by regulating ts2 RNA levels and possibly other sex determination genes (Calderon-Urrea & Dellaporta, 1999, Development 126:435).","Chemicals have been used to modulate and modify sexual differentiation in plants. The plant hormone ethylene has been observed to promote feminization in cucumber (Yamasaki et al., 2005, Vitam. Bonn. 72:79). Recent genetic and biochemical evidence has confirmed the role of ethylene in sex determination of melon, a related species (Boualem et al., 2008, Science 321:836). Conversely, gibberellin has masculinizing effects in cucumber but promotes feminization in maize (Bensen et al., 1995, Plant Cell 7:75), and auxin also has opposing effects in cucumber and (Yamasaki et al., 2005, Vitam. Horm. 72:79). However, little is known at this point about the role that chemicals play in sexual development and maturation of plants.","There is thus a great interest in identifying chemical compounds that modulate sexual differentiation in plants. Such compounds would be useful in suppressing or enhancing specific sexual phenotypes in plants, allowing the control of vast groups of plants without the need for time-consuming mechanical manipulation of the plants. The present invention fulfills these needs.","The invention includes an agriculturally compatible composition comprising an effective amount of a compound selected from the group consisting of jasmonic acid, a jasmonic acid derivative, and a salt thereof. In one aspect, the derivative is a jasmonic acid ester. In another aspect, the derivative is jasmonic acid methyl ester or methyl jasmonate.","The invention also includes a method of modulating sexuality in a plant. The method comprises the step of administering to the plant an agriculturally compatible composition comprising an effective amount of a compound selected from the group consisting of jasmonic acid, a jasmonic acid derivative, and a salt thereof.","The invention also includes a method of suppressing complete feminization or restoring male sexuality in a plant with a ts1 or ts2 mutation. The method comprises the step of administering to the plant an agriculturally compatible composition comprising an effective amount of a compound selected from the group consisting of jasmonic acid, a jasmonic acid derivative, and a salt thereof.","The invention also includes a method of creating homozygous stock in a plant with a ts1 or ts2 mutation. The method includes the step of administering to the plant an agriculturally compatible composition comprising an effective amount of a compound selected from the group consisting of jasmonic acid, a jasmonic acid derivative, and a salt thereof, wherein the progeny of the plant is homozygous for the mutation and is male-sterile.","In one aspect, the derivative is jasmonic acid methyl ester.","In one aspect, the plant is grass-related. In another aspect, the plant is maize or rice. In yet another aspect, the plant is maize.","In one aspect, the method of the invention further comprises the step of administering to the plant at least one additional compound useful for controlling plant sexuality. In another aspect, the at least one additional compound is selected from the group consisting of ethylene, gibberellin and auxin.","The present invention relates to the discovery that certain compounds are useful for controlling the sexuality of plants. In one aspect, these compounds act by restoring plant masculinization. In another aspect, these compounds suppress floral sexuality mutations and restore plant masculinization.","In one aspect, the invention provides an agriculturally compatible composition comprising a compound selected from the group consisting of jasmonic acid, a jasmonic acid derivative, and a salt thereof.","In another aspect, the invention provides a method of modulating sexuality in a plant. The method comprises the step of administering to the plant an agriculturally compatible composition comprising an effective amount of a compound selected from the group consisting of jasmonic acid, a jasmonic acid derivative, and a salt thereof.","In yet another aspect, the invention provides a method of suppressing complete feminization or restoring male sexuality in a plant with a ts1 or ts2 mutation. The method comprises the step of administering to the plant an agriculturally compatible composition comprising an effective amount of a compound selected from the group consisting of jasmonic acid, a jasmonic acid derivative, and a salt thereof.","In yet another aspect, the invention provides a method of creating homozygous stock in a plant with a ts1 or ts2 mutation. The method includes the step of administering to the plant an agriculturally compatible composition comprising an effective amount of a compound selected from the group consisting of jasmonic acid, a jasmonic acid derivative, and a salt thereof, wherein the progeny of the plant is homozygous for the mutation and is male-sterile.","In one embodiment, the derivative is a jasmonic acid ester. In another embodiment, the derivative is methyl jasmonate or jasmonic acid methyl ester.","In one embodiment, the plant is grass-related. In another embodiment, the plant is maize or rice. In yet another embodiment, the plant is maize.","In one embodiment, the method of the invention further comprises the step of administering to the plant at least one additional compound useful for controlling plant sexuality. In another aspect, the at least one additional compound is selected from the group consisting of ethylene, gibberelin and auxin.","Definitions","As used herein, each of the following terms has the meaning associated with it in this section.","Unless defined otherwise, all technical and scientific terms used herein generally have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Generally, the nomenclature used herein and the laboratory procedures in cell culture, molecular genetics, organic chemistry, and peptide chemistry are those well known and commonly employed in the art.","As used herein, the articles \u201ca\u201d and \u201can\u201d refer to one or to more than one (i.e. to at least one) of the grammatical object of the article. By way of example, \u201can element\u201d means one element or more than one element.","As used herein, the term \u201cabout\u201d will be understood by persons of ordinary skill in the art and will vary to some extent on the context in which it is used.","As used herein, the term \u201cjasmonic acid derivative\u201d refers to a derivative of jasmonic acid, such as but not limited to, a jasmonic acid ester. The class of jasmonic acid esters includes, but is not limited to, a jasmonic acid alkyl ester, jasmonic acid aryl ester, jasmonic acid heteroaryl ester, jasmonic acid aryalkyl ester, and jasmonic acid heteroaryl ester. The term \u201cjasmonic acid derivative\u201d also refers to chemical compounds that give rise to jasmonic acid or other jasmonic acid derivative by chemical or microorganism-based decomposition, regardless whether the decomposition takes place under controlled conditions or not.","As used herein, the term \u201cpolypeptide\u201d refers to a polymer composed of amino acid residues, related naturally occurring structural variants, and synthetic non-naturally occurring analogs thereof linked via peptide bonds. Synthetic polypeptides may be synthesized, for example, using an automated polypeptide synthesizer. As used herein, the term \u201cprotein\u201d typically refers to large polypeptides. As used herein, the term \u201cpeptide\u201d typically refers to short polypeptides. Conventional notation is used herein to represent polypeptide sequences: the left-hand end of a polypeptide sequence is the amino-terminus, and the right-hand end of a polypeptide sequence is the carboxyl-terminus.","As used herein, the polypeptides include natural peptides, recombinant peptides, synthetic peptides or a combination thereof. A peptide that is not cyclic will have an N-terminus and a C-terminus. The N-terminus will have an amino group, which may be five (i.e., as a NHgroup) or appropriately protected (for example, with a BOC or a Fmoc group). The C-terminus will have a carboxylic group, which may be free (i.e., as a COON group) or appropriately protected (for example, as a benzyl or a methyl ester). A cyclic peptide does not necessarily have free N- or C-termini, since they are covalently bonded through an amide bond to form the cyclic structure. The term \u201cpeptide bond\u201d means a covalent amide linkage formed by loss of a molecule of water between the carboxyl group of one amino acid and the amino group of a second amino acid.","As used herein, amino acids are represented by the full name thereof, by the three letter code corresponding thereto, or by the one-letter code corresponding thereto, as indicated below:",{"@attributes":{"id":"p-0041","num":"0040"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"thead":{"row":[{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},"Full Name","Three-Letter Code","One-Letter Code"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Aspartic Acid","Asp","D"]},{"entry":[{},"Glutamic Acid","Glu","E"]},{"entry":[{},"Lysine","Lys","K"]},{"entry":[{},"Arginine","Arg","R"]},{"entry":[{},"Histidine","His","H"]},{"entry":[{},"Tyrosine","Tyr","Y"]},{"entry":[{},"Cysteine","Cys","C"]},{"entry":[{},"Asparagine","Asn","N"]},{"entry":[{},"Glutamine","Gln","Q"]},{"entry":[{},"Serine","Ser","S"]},{"entry":[{},"Threonine","Thr","T"]},{"entry":[{},"Glycine","Gly","G"]},{"entry":[{},"Alanine","Ala","A"]},{"entry":[{},"Valine","Val","V"]},{"entry":[{},"Leucine","Leu","L"]},{"entry":[{},"Isoleucine","Ile","I"]},{"entry":[{},"Methionine","Met","M"]},{"entry":[{},"Proline","Pro","P"]},{"entry":[{},"Phenylalanine","Phe","F"]},{"entry":[{},"Tryptophan","Trp","W"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}}}},"A \u201cpolynucleotide\u201d means a single strand or parallel and anti-parallel strands of a nucleic acid. Thus, a polynucleotide may be either a single-stranded or a double-stranded nucleic acid.","The term \u201cnucleic acid\u201d typically refers to large polynucleotides.","The term \u201coligonucleotide\u201d typically refers to short polynucleotides, which are generally not greater than about 50 nucleotides. It will be understood that when a nucleotide sequence is represented by a DNA sequence (i.e., A, T, G, C), this also includes an RNA sequence (i.e., A, U, G, C) in which \u201cU\u201d replaces \u201cT.\u201d","Conventional notation is used herein to describe polynucleotide sequences: the left-hand end of a single-stranded polynucleotide sequence is the 5\u2032-end; the left-hand direction of a double-stranded polynucleotide sequence is referred to as the 5\u2032-direction.","The direction of 5\u2032 to 3\u2032 addition of nucleotides to nascent RNA transcripts is referred to as the transcription direction. The DNA strand having the same sequence as an mRNA is referred to as the \u201ccoding strand;\u201d sequences on the DNA strand which are located 5\u2032 to a reference point on the DNA are referred to as \u201cupstream sequences;\u201d sequences on the DNA strand which are 3\u2032 to a reference point on the DNA are referred to as \u201cdownstream sequences.\u201d","A \u201cportion\u201d of a polynucleotide means at least about twenty sequential nucleotide residues of the polynucleotide. It is understood that a portion of a polynucleotide may include every nucleotide residue of the polynucleotide.","\u201cPrimer\u201d refers to a polynucleotide that is capable of specifically hybridizing to a designated polynucleotide template and providing a point of initiation for synthesis of a complementary polynucleotide. Such synthesis occurs when the polynucleotide primer is placed under conditions in which synthesis is induced, i.e., in the presence of nucleotides, a complementary polynucleotide template, and an agent for polymerization such as DNA polymerase. A primer is typically single-stranded, but may be double-stranded. Primers are typically deoxyribonucleic acids, but a wide variety of synthetic and naturally occurring primers are useful for many applications. A primer is complementary to the template to which it is designed to hybridize to serve as a site for the initiation of synthesis, but need not reflect the exact sequence of the template. In such a case, specific hybridization of the primer to the template depends on the stringency of the hybridization conditions. Primers can be labeled with, e.g., chromogenic, radioactive, or fluorescent moieties and used as detectable moieties.","\u201cProbe\u201d refers to a polynucleotide that is capable of specifically hybridizing to a designated sequence of another polynucleotide. A probe specifically hybridizes to a target complementary polynucleotide, but need not reflect the exact complementary sequence of the template. In such a case, specific hybridization of the probe to the target depends on the stringency of the hybridization conditions. Probes can be labeled with, e.g., chromogenic, radioactive, or fluorescent moieties and used as detectable moieties.","An \u201cisolated nucleic acid\u201d refers to a nucleic acid segment or fragment which has been separated from sequences which flank it in a naturally occurring state, e.g., a DNA fragment which has been removed from the sequences which are normally adjacent to the fragment, e.g., the sequences adjacent to the fragment in a genome in which it naturally occurs. The term also applies to nucleic acids which have been substantially purified from other components which naturally accompany the nucleic acid, e.g., RNA or DNA or proteins, which naturally accompany it in the cell. The term therefore includes, for example, a recombinant DNA which is incorporated into a vector, into an autonomously replicating plasmid or virus, or into the genomic DNA of a prokaryote or eukaryote, or which exists as a separate molecule (e.g. as a cDNA or a genomic or cDNA fragment produced by PCR or restriction enzyme digestion) independent of other sequences. It also includes a recombinant DNA which is part of a hybrid gene encoding additional polypeptide sequence.","\u201cEncoding\u201d refers to the inherent property of specific sequences of nucleotides in a polynucleotide, such as a gene, a cDNA, or an mRNA, to serve as templates for synthesis of other polymers and macromolecules in biological processes having either a defined sequence of nucleotides (i.e., rRNA, tRNA and mRNA) or a defined sequence of amino acids and the biological properties resulting therefrom. Thus, a gene encodes a protein if transcription and translation of mRNA corresponding to that gene produces the protein in a cell or other biological system. Both the coding strand, the nucleotide sequence of which is identical to the mRNA sequence and is usually provided in sequence listings, and the non-coding strand, used as the template for transcription of a gene or cDNA, can be referred to as encoding the protein or other product of that gene or cDNA.","Unless otherwise specified, a \u201cnucleotide sequence encoding an amino acid sequence\u201d includes all nucleotide sequences that are degenerate versions of each other and that encode the same amino acid sequence. Nucleotide sequences that encode proteins and RNA may include introns.","As used herein, the term \u201cplant\u201d refers to a living organism belonging to the kingdom Plantae. Examples of plants are grasses, such as switchgrass, rice, oats, wheat, barley, sorghum, millet, rye, triticale, buckwheat, fonio, quinoa, teff, wild rice, amaranth, kaniwa, spelt, einkorn, emmer, durum, and maize (corn). Preferably, the plant is rice or maize. Most preferably, the plant is maize.","As used herein, the term \u201ceffective amount\u201d refers to a non-toxic but sufficient amount of an agent to provide the desired biological result. That result can be modulation of sexual differentiation in plants, suppression or enhancement of specific sexual phenotypes, or any other desired alteration of a plant phenotype. An appropriate effective amount in any individual case may be determined by one of ordinary skill in the art using routine experimentation.","As used herein, the term \u201cagriculturally acceptable\u201d refers to a material, such as a carrier or diluent, which does not abrogate the biological activity or properties of the compound, and is relatively non-toxic, i.e., the material may be administered to a plant without causing undesirable biological effects or interacting in a deleterious manner with any of the components of the composition in which it is contained.","As used herein, the term \u201cagriculturally acceptable composition\u201d refers to a mixture of at least one compound useful within the invention with agriculturally acceptable chemical components, such as carriers, stabilizers, diluents, dispersing agents, suspending agents, thickening agents, and\/or excipients. The agriculturally acceptable composition facilitates administration of the compound to a plant. Multiple techniques of administering an agriculturally acceptable composition exist in the art including, but not limited to: watering, spraying, fumigation, aerolization, injecting, and dusting.","As used herein, the language \u201cacceptable salt\u201d refers to a salt of the administered compounds prepared from agriculturally acceptable non-toxic acids including inorganic acids, organic acids, solvates, hydrates, or clathrates thereof. The compounds useful within the invention may form salts with acids or bases, and such salts are included in the present invention. The term \u201csalts\u201d embraces addition salts of free acids or free bases of the compounds useful within the invention. Preferred salts are formed from cationic and anionic counterions that have been approved or validated for agricultural applications. Unacceptable salts may nonetheless possess properties such as high crystallinity, which have utility in the practice of the present invention, such as for example utility in process of synthesis, purification or formulation of compounds useful within this invention.","As used herein, the \u201cinstructional material\u201d includes a publication, a recording, a diagram, or any other medium of expression that may be used to communicate the usefulness of the compounds described herein, In some instances, the instructional material may be part of a kit useful for effecting the alleviating or treating the various diseases or disorders recited herein. Optionally, or alternately, the instructional material may describe one or more methods of controlling plant sexuality. The instructional material of the kit may, for example, be affixed to a container that contains the compounds useful within the invention or be shipped together with a container that contains the compounds. Alternatively, the instructional material may be shipped separately from the container with the intention that the recipient uses the instructional material and the compound cooperatively. For example, the instructional material is for use of a kit; instructions for use of the compound; or instructions for use of a formulation of the compound.","Compounds of the Invention","The compounds useful within the invention may be synthesized using techniques well-known in the art of organic synthesis or may be isolated from natural sources.","In one aspect, the compound is jasmonic acid ((1R,2R)-3-oxo-2-(2Z)-2-pentenylcyclopentaneacetic acid). In another aspect, the compound is a jasmonic acid ester. In yet another aspect, the compound is a jasmonic acid alkyl ester, jasmonic acid aryl ester, jasmonic acid heteroaryl ester, jasmonic acid aryalkyl ester, and jasmonic acid heteroaryl ester. Non-limiting examples of jasmonic acid esters are methyl jasmonate (or jasmonic acid methyl ester), ethyl jasmonate, n-propyl jasmonate, isopropyl jasmonate, n-butyl jasmonate, sec-butyl jasmonate, t-butyl jasmonate, methoxyethyl jasmonate, pentyl jasmonate, phenyl jasmonate, 4-chloro jasmonate, 4-fluoro jasmonate, naphtyl jasmonate, benzyl jasmonate, pyridinyl jasmonate, phenylethyl jasmonate and so on. In yet another aspect, the compound is methyl jasmonate (methyl (1R,2R)-3-oxo-2-(2Z)-2-pentenylcyclopentaneacetate).",{"@attributes":{"id":"p-0061","num":"0060"},"chemistry":{"@attributes":{"id":"CHEM-US-00001","num":"00001"},"img":{"@attributes":{"id":"EMI-C00001","he":"90.93mm","wi":"56.73mm","file":"US09258998-20160216-C00001.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"Acceptable base addition salts of compounds useful within the present invention include, for example, metallic salts and non-metallic salts. Metallic cationic counterions include alkali metal, alkaline earth metal and transition metal ions such as, for example, aluminum, bismuth, calcium, lithium, magnesium, neodymium, potassium, rubidium, sodium, strontium and zinc. Non-metallic cationic counterions include organic basic amines such as, for example, ammonium, benethamine [N-benzylphenethylamine], benzathine [N,N\u2032-dibenzylethylenediamine], betaine [(carboxymethyl)trimethylammonium hydroxide], carnitine, clemizole [1-p-chloro-benzyl-2-pyrrolidin-1\u2032-ylmethylbenzimidazole], chlorcyclizine [144-chloro-benzhydryl)-4-methylpiperazine], choline, dibenzylamine, diethanolamine, diethylamine, diethylammonium, diolamine, meglumine [N-ethylglucamine], erbumine [t-butylamine], ethylenediamine, heptaminol [6-amino-2-methylheptan-2-ol], hydrabamine [N,N\u2032-di(dehydroabietyl)ethylenediamine], hydroxyethylpyrrolidine, imidazole, meglumine [N-methylglucamine], olamine, piperazine, 4-phenyl-cyclohexylamine, procaine, pyridoxine, triethanolamine, and tromethamine [tris(hydroxymethyl)aminomethane]. All of these salts may be prepared from the corresponding compound by reacting, for example, the appropriate acid or base with the compound.","Examples of such inorganic acids are hydrochloric, hydrobromic, hydroiodic, nitric, sulfuric, and phosphoric. Appropriate organic acids may be selected, for example, from aliphatic, aromatic, carboxylic and sulfonic classes of organic adds, examples of which are formic, acetic, propionic, succinic, camphorsulfonic, citric, fumaric, gluconic, isethionic, lactic, malic, mucic, tartaric, para-toluenesulfonic, glycolic, glucuronic, maleic, furoic, glutamic, benzoic, anthranilic, salicylic, phenylacetic, mandelic, embonic (pamoic), methanesulfonic, ethanesulfonic, pantothenic, benzenesulfonic (besylate), stearic, sulfanilic, alginic, galacturonic, and the like.","The compounds useful within the present invention may also be useful in combination with at least one additional compound useful for controlling plant sexuality, These additional compounds may comprise compounds described in the present invention or compounds, e.g., commercially available compounds, known to modify, modulate or alter plant sexuality.","In non-limiting examples, the compounds of the invention may be used in combination with at least one of the following compounds: ethylene, gibberellin and auxin.","A synergistic effect may be calculated, for example, using suitable methods such as, for example, the Sigmoid-Eequation (Holford & Scheiner, 19981, Clin. Pharmacokinet. 6: 429-453), the equation of Loewe additivity (Loewe & Muischnek, 1926, Arch. Exp. Pathol Pharmacol. 114: 313-326) and the median-effect equation (Chou & Talalay, 1984, Adv. Enzyme Regul., 22: 27-55). Each equation referred to above may be applied to experimental data to generate a corresponding graph to aid in assessing the effects of the compound combination. The corresponding graphs associated with the equations referred to above are the concentration-effect curve, isobologram curve and combination index curve, respectively.","Administration\/Dosage\/Formulations","Routes of administration of any of the compositions of the invention include, but are not limited to, watering, spraying, fumigation, aerolization, injecting, or dusting. Administration may involve, in non-limiting examples, direct surface application to an intact or cut portion of the plant, microinjection into a tissue or cell thereof, or micro-bombardment, preferably under low pressure.","The regimen of administration may affect what constitutes an effective amount. The formulations of the invention may be administered to the plant at any stage of its development. Preferably, the formulations of the invention may be administered to the plant at the time of inflorescence development, during which the sex determination process is taking place. In the case of most lines of maize, this time corresponds to the period when the developing inflorescence reaches approximately 1.0 cm in height. At this stage of floral development, most maize plants possess 6-8 fully expanded leaves. This precise inflorescence height and expanded leaf number vary according to variety of maize and environmental growth conditions. Further, several divided dosages, as well as staggered dosages, may be administered daily or sequentially, or the dose may be continuously administered. Further, the dosages of the formulations may be proportionally increased or decreased as indicated by the exigencies of the situation.","Administration of the compositions of the present invention to a plant, preferably a grass-related plant, more preferably maize, may be carried out using known procedures, at dosages and for periods of time effective to modulate the plant sexuality. An effective amount of the compound necessary to achieve the desired sexuality modulation may vary according to factors such as the nature of the plant, its age, status and location; and the ability of the compound to modulate plant sexuality. Dosage regimens may be adjusted to provide the optimum response. For example, several divided doses may be administered daily or the dose may be proportionally reduced as indicated by the exigencies of the situation. A non-limiting example of an effective dose range for a compound useful within the invention is from about 0.001 to 1,000 mg\/kg of plant weight\/per day. In a non-limiting example, an individual plant is treated with 1 mL, of a 1 mM solution of a jasmonic acid derivative, and the treatment is performed three times at 48-hour intervals. One of ordinary skill in the art would be able to study the relevant factors and make the determination regarding the effective amount of the compound without undue experimentation.","Actual dosage levels of the active ingredients in the compositions of this invention may be varied so as to obtain an amount of the active ingredient that is effective to achieve the desired response for a particular plant, without being toxic to the plant.","A plant specialist, e.g., botanist or agricultural technician, having ordinary skill in the art may readily determine and prescribe the effective amount of the composition required. For example, the plant specialist could start doses of the compounds useful within the invention at levels lower than that required in order to achieve the desired effect and gradually increase the dosage until the desired effect is achieved.","In one embodiment, the compositions of the invention are formulated using one or more agriculturally acceptable excipients or carriers. In one embodiment, the compositions of the invention comprise an effective amount of a compound of the invention and an agriculturally acceptable carrier.","In one embodiment, the compositions of the invention are administered to the plant in dosages that range from one to five times at two-day intervals. In another embodiment, the compositions of the invention are administered to the plant in range of dosages that include, but are not limited to, once every day, every two, days, every three days to once a week, and once every two weeks. In another embodiment, the composition of the invention is applied once to the plant as a slow-release preparation. It will be readily apparent to one skilled in the art that the frequency of administration of the various combination compositions of the invention will vary from plant to plant, depending on many factors including, but not limited to, age, disease or disorder to be treated, gender, overall health, and other factors. Thus, the invention should not be construed to be limited to any particular dosage regime and the precise dosage and composition to be administered to any plant will be determined by the plant specialist based on the evaluation of the plant in question.","In one embodiment, the present invention is directed to a packaged agriculturally acceptable composition comprising a container holding an effective amount of a compound of the invention, alone or in combination with a second agricultural agent; and instructions for using the compound to modulate plant sexuality.","The term \u201ccontainer\u201d includes any receptacle for holding the agriculturally acceptable composition. For example, in one embodiment, the container is the packaging that contains the agricultural composition. In other embodiments, the container is not the packaging that contains the agricultural composition, i.e., the container is a receptacle, such as a box or vial that contains the packaged agricultural composition or unpackaged agricultural composition and the instructions for use of the agricultural composition. Moreover, packaging techniques are well known in the art. It should be understood that the instructions for use of the agricultural composition may be contained on the packaging containing the agricultural composition, and as such the instructions form an increased functional relationship to the packaged product. However, it should be understood that the instructions may contain information pertaining to the compound's ability to perform its intended function, e.g., modulating plant sexuality.","Those skilled in the art will recognize, or be able to ascertain using no more than routine experimentation, numerous equivalents to the specific procedures, embodiments, claims, and examples described herein. Such equivalents were considered to be within the scope of this invention and covered by the claims appended hereto. For example, it should be understood, that modifications in reaction conditions, including but not limited to reaction times, reaction size\/volume, and experimental reagents, such as solvents, catalysts, pressures, atmospheric conditions, e.g., nitrogen atmosphere, and reducing\/oxidizing agents, with art-recognized alternatives and using no more than routine experimentation, are within the scope of the present application.","It is to be understood that wherever values and ranges are provided herein, all values and ranges encompassed by these values and ranges, are meant to be encompassed within the scope of the present invention. Moreover, all values that fall within these ranges, as well as the upper or lower limits of a range of values, are also contemplated by the present application.","The following examples further illustrate aspects of the present invention. However, they are in no way a limitation of the teachings or disclosure of the present invention as set forth herein.","The invention is now described with reference to the following Examples. These Examples are provided for the purpose of illustration only, and the invention is not limited to these Examples, but rather encompasses all variations that are evident as a result of the teachings provided herein.","Materials and Methods","Genetic Stocks and Mutants","Wild-type maize inbred lines W22, B73 and Mo17 were used. The ts1-ref allele was previously described (Emerson, 1920, J. Hered. 11:65; Emerson et al., 1935, Cornell Univ. Agric. Exp. Stn. Memoir 180). ts1-69-Alex-Mo17 was identified as a spontaneous mutant in Mo17; and ts1-0174 was discovered in an unknown genetic background. These three alleles were obtained from the Maize Genetics Cooperation Stock Center, Maize COOP (University of Illinois, Urbana\/Champaign, Ill.). The ts1-MuSH and ts1-SH04 alleles were obtained from Plant Gene Expression Center, United States Department of Agriculture\u2014Agricultural Research Service and the University of California, Albany, Calif.). The ts1-Mu92 and ts1-Mu93 alleles were recovered in progeny of self-pollinated maize plants known to contain active Mutator (Mu) elements (Monsanto, Creve Coeur, Mo.).","The remaining alleles, ts1-Mu01 and ts1-Mu02, were isolated by gene tagging over the course of several generations of testcrosses of ts1-ref\/ts1-ref plants to Mutator lines. In brief, ts1-ref mutant plants were crossed to a W22 line carrying actively transposing Mu elements. Since ts1-ref is a recessive mutation, only wild-type plants are expected from such a cross unless a new ts1 mutant allele is recovered in any gametes of the Mu-active W22 line. About 40,000 individuals of this cross were grown during the summer seasons of 2001 and 2002. Plants showing a tasselseed phenotype, potentially containing new ts1 mutant alleles, were outcrossed to the W22 inbred line for at least two generations in order to segregate the new alleles and to reduce the number of active Mu elements in the genome. Additionally, these F1 plants were crossed to ts1-ref mutants to confirm allelism. The ts1-linked hlm1 marker was used to distinguish the new ts1 mutant alleles from the original ts1-ref allele in the first outcrossed generation. Homozygotes for the new allele were usually obtained in the third generation by self-pollinating heterozygous F2 plants.","ts1 Mapping Populations","Since the ts1 mutant plants are completely feminized, the ts1-ref stock was maintained by sib mating ts1-ref\/ts1-ref with ts1-ref\/Ts1. These sib makings were used to generate the original mapping population (n=93), which was selected for plants phenotypically scored as ts1\/ts1 mutants. Any ts1-ref\/ts1-ref plant carrying one or more of the linked molecular markers in a heterozygote state were deemed to be recombinant between the heterozygous marker and the ts1 locus. Once ts1 tightly-linked flanking markers were identified, they were subsequently used to screen three additional testcross populations containing both heterozygote wild-type plants and plants homozygous for the mutant ts1-ref allele. Two of these populations (n=119) also come from the sib mating ts1-ref\/ts1-ref\u00d7ts1-ref\/Ts1, while another mapping population (n=208) carried a wild-type Ts1 allele from the inbred W22 (ts1-ref\/ts1-ref\u00d7ts1-ref\/Ts1-W22).","Molecular Marker Development","Candidate marker sequences were selected through genetic, physical and rice-maize synteny mapping. These sequences usually corresponded to (a) maize BAC ends or ESTs deposited in GenBank (NIH; Benson et al., 2008, Nucleic Acids Research 36(Database issue):D25-30) ; or (b) rice genes annotated in the regions syntenic to maize intervals at Gramene (Cold Spring Harbor Laboratory, www dot gramene dot org). Candidate sequences were analyzed by BLAST through the TIGR Maize Database. Maize sequences corresponding to repetitive DNA were not further considered for marker design. Non-repetitive DNA sequences were selected for marker development and in some cases these sequences were extended by alignment with AZM sequences (TIGR).","Candidate marker sequences were PCR-amplified from genomic DNA of W22, ts1-ref\/Ts1 and ts1-ref\/ts1-ref plants. In a few instances, the PCR products had size polymorphisms easily resolved by agarose gel electrophoresis. In most cases, however, the PCR products had the same apparent size. In these cases, the PCR products were sequenced and examined for the presence of SNPs. If present, SNPs that represented differences in restriction enzyme recognition sites were used as CAPS (Cleaved Amplified Polymorphic Sequences) markers (Konieczny & Ausubel, 1993, Plant J. 4:403) to distinguish between the different alleles in the mapping population.","New markers were initially evaluated in 20 non-recombinant individuals of known phenotype to confirm co-segregation of the marker with the appropriate ts1 allele. This was necessary to prevent the use of markers corresponding to a duplicate region of the ts1 interval in chromosome 10 or to repeat sequences that have \u201cescaped\u201d filtering in the repeat database. Table 1 displays the features of the molecular markers defining the ts1 genetic region.",{"@attributes":{"id":"p-0086","num":"0085"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"294pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u20031"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Molecular\u2003markers\u2003used\u2003for\u2003mapping\u2003the\u2003ts1\u2003locus"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"140pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},{},"Product"]},{"entry":[{},{},"Primer",{},"size"]},{"entry":["Marker","Type\u2003*","pair\u2003ID","Primer\u2003sequence\u2003(listed\u20035\u2032 to\u20033\u2032)","(bp)\u2003\u2020"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"140pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["CC760183","CAPS","1533","CACAGGAGATTCTGTACTGTGACCA","663"]},{"entry":[{},"(Tsp509I)",{},"A\u2003(SEQ\u2003ID\u2003NO:\u20031)",{}]},{"entry":[{},{},"1541","TGCAATGACAAGGGTATTCATGTG",{}]},{"entry":[{},{},{},"(SEQ\u2003ID\u2003NO:\u20032)",{}]},{"entry":{}},{"entry":["hlm1","CAPS","835","CTCTCATAACACACACAAGCCTCT","1200"]},{"entry":[{},"(ScrFI)",{},"(SEQ\u2003ID\u2003NO:\u20033)",{}]},{"entry":[{},{},{},"AGCTACCTTTCTGGAGGGTGAAGAA",{}]},{"entry":[{},{},"836","(SEQ\u2003ID\u2003NO:\u20034)",{}]},{"entry":{}},{"entry":["umc2251\u2003\u2021","SSR","1516","CCTGAATCGCTCATTCGCTC","178"]},{"entry":[{},{},{},"(SEQ\u2003ID\u2003NO:\u20035)",{}]},{"entry":[{},{},"1517","GTCGAGGGTTTGGAGGAGAGAC",{}]},{"entry":[{},{},{},"(SEQ\u2003ID\u2003NO:\u20036)"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":"* CAPS, Cleaved Amplified Polymorphic Sequence; restriction enzymes used for CAPS marker analysis are indicated in parenthesis. SSR, Simple Sequence Repeat (Microsatellite)"},{"entry":"\u2020 Size corresponds to the allele found in inbred line B73"},{"entry":"\u2021 Previously reported SSR"}]}}]}},"br":{}},"PCR primers were designed with Primer3 (Rozen & Skaletsky, 2000, Methods Mol. Biol. 132:365) and synthesized by the W. M. Keck Foundation Biotechnology Resource Laboratory (Yale University, New Haven, Conn.). PCR-based markers were routinely amplified with Tag DNA polymerase (Qiagen, Valencia, Calif., USA) in 20 \u03bcL reactions containing 1\u00d7PCR buffer and 1\u00d7 Q-Solution supplied by the manufacturer, 200 \u03bcM of each dNTP, 500 nM of each primer and 2.5 ng\/\u03bcl genomic template DNA. Q-Solution contains the chemical betaine, which improves the amplification of DNA by reducing the formation of secondary structure in GC-rich regions. The addition of Q-Solution to the PCR reaction was mandatory for amplification of most maize sequences, which have high GC contents. Q-solution was excluded only for target sequences with <50% GC content. PCR cycling conditions were as follows: 95\u00b0 C. (3 minutes); 35 cycles of 95\u00b0 C. (30 seconds), 59-63\u00b0 C. (30 seconds), 72\u00b0 C. (1 minute\/1 kb); 72\u00b0 C. (10 minutes). Annealing temperature was variable, adjusted to 3 to 5\u00b0 C. below the Primer3-calculated Tin of the primers used in each assay.","Southern Blot Analysis","Southern blot hybridization was performed with a published protocol (Dellaporta & Moreno, in \u201cThe Maize Handbook\u201d, M. Freeling, V. Walbot, Eds., Springer Verlag, New York, 1993, pp. 569-572), except that the hybridization solution consisted of 0.25 M sodium phosphate, pH 7.2 and 7% SDS, as suggested by the manufacturer of the Zeta-Probe GT blotting membranes (Biorad, Hercules, Calif., USA). The hybridization probe consisted of a 624 by fragment from the ts1-W22 gene (bases 2590-3213, spanning the end of exon 5 through the beginning of exon 7) and bore sufficient similarity to the duplicate ts1b gene (87-90%) so that both ts1 and ts1b were detected.","Phylogenetic Analysis","Sixty-four plant lipoxygenase amino acid sequences (Table 2), including that of TS1 and TS1b, were aligned with ClustalW2 (Chema et al., 2003, Nucleic Acids Res. 31:3497). Two different algorithms, maximum parsimony and Bayesian inference, were employed to estimate phylogenetic relationships of TS1 and related proteins. Maximum parsimony was implemented in PAUP*4.0b10 (Swofford, \u201cPAUP*, Phylogenetic Analysis Using Parsimony (* and other methods)\u201d, Sinauer Associates, Sunderland, Mass., 1998) where the heuristic search option was applied with default values. Bootstrap values for the maximum parsimony tree were obtained by resampling 1,000 replicates under the full heuristic search method. Bayesian inference was performed with MrBaycs 3.1.2 (Huelsenbeck & Ronquist, 2001, Bioinformatics 17:754) with a mixed amino acid substitution model, four independent chains run for 5,000,000 generations, and sampling every 1000th tree. Convergence was estimated when the standard deviation of split frequencies reached a plateau approaching zero, and the consensus tree was determined with a burn-in of 25% (1250 trees).",{"@attributes":{"id":"p-0090","num":"0089"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Plant lipoxygenase sequences used in TS1 phylogenetic analysis"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"70pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Organism","Sequence ID","GenBank Accession"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},{"i":"Arabidopsis thaliana"},"AtLOX1","Q06327"]},{"entry":[{},{},"AtLOX2","P38418"]},{"entry":[{},{},"AtLOX3","Q9SMW1"]},{"entry":[{},{},"AILOX4","Q9FNX8"]},{"entry":[{},{},"AtLOX5","Q9FNX7"]},{"entry":[{},{},"AtLOX6","Q9CAG3"]},{"entry":[{},{"i":"Cucumis sativus"},"CsLOX2","AAA79186"]},{"entry":[{},{},"CsLOX3","CAA63483"]},{"entry":[{},{},"CsLOX4","CAB83038"]},{"entry":[{},{"i":"Glycine max"},"GmLOX1","P08170"]},{"entry":[{},{},"GmLOX2","P09439"]},{"entry":[{},{},"GmLOX3","P09186"]},{"entry":[{},{},"GmLOX4","P38417"]},{"entry":[{},{},"GmLOX6","AAA96817"]},{"entry":[{},{},"GmLOX7","P24095"]},{"entry":[{},{"i":"Hordeum vulgare"},"HvLOX1","P93184"]},{"entry":[{},{},"HvLOX2","Q8GSM3"]},{"entry":[{},{},"HvLOX3","Q8GSM2"]},{"entry":[{},{},"HvLOX4","CAI84707"]},{"entry":[{},{},"HvLOXA","P29114"]},{"entry":[{},{},"HvLOXB","AAB60715"]},{"entry":[{},{},"HvLOXC","AAB70865"]},{"entry":[{},{"i":"Lens culinaris"},"LcLOX1","P38414"]},{"entry":[{},{"i":"Lycopersicon esculentum"},"LeLOX1","P38415"]},{"entry":[{},{},"LeLOX2","P38416"]},{"entry":[{},{},"LeLOX3","AAG21691"]},{"entry":[{},{},"LeLOX4","Q96573"]},{"entry":[{},{},"LeLOX5","Q96574"]},{"entry":[{},{"i":"Nicotiana attenuata"},"NaLOX1","AAP83136"]},{"entry":[{},{},"NaLOX2","AAP83137"]},{"entry":[{},{},"NaLOX3","AAP83138"]},{"entry":[{},{"i":"Nicotiana tabacum"},"NtLOX1","CAA58859"]},{"entry":[{},{"i":"Oryza sativa"},"OsLOX1","Q76I22"]},{"entry":[{},{},"OsLOX10","Q0DJB6"]},{"entry":[{},{},"OsLOX2","P29250"]},{"entry":[{},{},"OsLOX2.3","Q6H7Q6"]},{"entry":[{},{},"OsLOX3","Q7G794"]},{"entry":[{},{},"OsLOX3b","Q53RB0"]},{"entry":[{},{},"OsLOX5","Q7XV13"]},{"entry":[{},{},"OsLOX6","Q8H016"]},{"entry":[{},{},"OsLOX7","P38419"]},{"entry":[{},{},"OsLOX8","Q84YK8"]},{"entry":[{},{},"OsLOX9","Q0IS17"]},{"entry":[{},{},"OsLOXRCI1","Q9FSE5"]},{"entry":[{},{"i":"Pisum sativum"},"PsLOX1","AAB71759"]},{"entry":[{},{},"PsLOX2","P14856"]},{"entry":[{},{},"PsLOX3","P09918"]},{"entry":[{},{},"PsLOX7","CAC04380"]},{"entry":[{},{},"PsLOX8","CAA75609"]},{"entry":[{},{},"PsLOX9","CAG44504"]},{"entry":[{},{},"PsLOXG","CAA53730"]},{"entry":[{},{"i":"Phaseolus vulgaris"},"PvLOX1","P27480"]},{"entry":[{},{},"PvLOX1b","AAB18970"]},{"entry":[{},{},"PvLOX2b","AAG42354"]},{"entry":[{},{},"PvLOX2c","AAF15296"]},{"entry":[{},{"i":"Solanum tuberosum"},"StLOX1","CAA64765"]},{"entry":[{},{},"StLOX2","AAD09202"]},{"entry":[{},{},"StLOX3","AAB67865"]},{"entry":[{},{},"StLOX4","CAA65268"]},{"entry":[{},{},"StLOX5","CAA65269"]},{"entry":[{},{"i":"Zea mays"},"ZmLOX1","AAL73499"]},{"entry":[{},{},"ZmLOX2","AAF76207"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}},"br":{}},"Maize plants from inbred line W22 were grown in the greenhouse. All tissue samples used for qRT-PCR assays were quickly dissected and immediately frozen in liquid nitrogen. Approximately 100 mg of frozen tissue were ground in a mortar and pestle and quickly re-suspended in 1 nil of Trizol\u00ae Reagent (Invitrogen, Carlsbad, Calif., USA). Developing inflorescences between 0.8 and 3 cm in length were directly placed in 1.5 ml microcentrifuge tubes, resuspended in 1 nil of Trizol\u00ae reagent and ground with a plastic pestle attached to a table top drill press. Total RNA was isolated according to manufacturer's recommendations and re-suspended in water previously treated with diethylpyrocarbonate (DEPC; Sigma, St. Louis, Mo., USA) containing 20 units of Protector RNase Inhibitor (Roche, Indianapolis, Ind., USA).","Prior to cDNA synthesis, 1 \u03bcg of total RNA was treated with 1 unit of DNase I, Amplification Grade (Invitrogen) in a 10 \u03bcl reaction containing 1\u00d7 DNase I buffer supplied by the manufacturer. The reaction proceeded for 15 minutes at room temperature and the enzyme was inactivated by adding 1 \u03bcl of 25 mM EDTA and heating at 65\u00b0 C. for 10 min, The DNase-treated RNA (1 \u03bcg) was directly used in cDNA synthesis with the SuperScript\u00ae 111 First-Strand Synthesis SuperMix for qRT-PCR (Invitrogen) following the manufacturer's instructions. The 2\u00d7RT Reaction Mix included both oligo(dT)20 and random hexamers to prime the reverse transcription reaction, The cDNA was diluted to 100 \u03bcl with 1\u00d7TE and stored at \u221280\u00b0 C. PCR reactions were performed in optical 96-well plates with a 7500 Fast Real-Time PCR System sequence detection system (Applied Biosystems, Foster City, Calif., USA). Reactions contained 1\u00d7 Power SYBR Green Master Mix reagent (Applied Biosystems), 300 nM of each gene-specific primer (Table 3) and 1 \u03bcl of diluted cDNA in a final volume of 25 \u03bcl. The standard thermal profile recommended by the manufacturer of the PCR master mix was followed. Three technical (PCR) replicates were set up for each one of the three biological replicates of each tissue sample. qRT-PCR data were normalized with actin1 as a reference gene.",{"@attributes":{"id":"p-0093","num":"0092"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u20033"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Primers\u2003used\u2003for\u2003quantitative\u2003qRT-PCR\u2003analysis"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"147pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Target","Primer",{}]},{"entry":["gene","pair\u2003TD","Primer\u2003sequence\u2003(listed\u20035\u2032 to\u20033\u2032)"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["actin1","\u2003213","CATGAGGCCACGTACAACTCCATC"]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u20037)"]},{"entry":[{},"\u2003214","TCATACTCTCCCTTGGAGATCCAC"]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u20038)"]},{"entry":{}},{"entry":["ts1","2221\u2003*","GCTGCCGTACGAGCTCATGG"]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u20039)"]},{"entry":[{},"1894","TCCTTTCAGATCATCTCTGTCATGC"]},{"entry":[{},{},"(SEQ\u20031D\u2003NO:\u200310)"]},{"entry":{}},{"entry":["ts1b","2221\u2003*","GCTGCCGTACGAGCTCATGG"]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200311)"]},{"entry":[{},"2648\u2003\u2020","TTGGAGATCGGGGAGAAGACTAAA"]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200312)"]},{"entry":{}},{"entry":["ts2","2264","GTGGAGAAGATGGAGGAGGTGGT"]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200313)"]},{"entry":[{},"2306","ATTGATTCACAAGCCGATGAGGTT"]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200314)"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":"* The specificity for ts1 and ts1b is achieved with the reverse primers P1894 and P2648, respectively"},{"entry":"\u2020 P2648 is specific for the ts1-W22 allele"}]}}]}},"br":{}},"Primers P1930 (5\u2032-CCTCTCAGTACCGACAGACAGC-3\u2032; SEQ ID NO:15) and P1931 (5\u2032-CCATTCAGTTCCTCACAGTCTTGC-3\u2032; SEQ ID NO:16) were used to amplify a 217 by fragment of the ts1 gene corresponding to part of the 3\u2032 UTR. The PCR product was cloned into the pCRII-TOPO\u00ae vector (Invitrogen) generating pYU 1672, the plasmid used for synthesis of the ts1 in situ probe. The ts2 in situ hybridization probes have been previously described (DeLong et al., 1993, Cell 74:757) and are contained in the plasmids pYU59 and pYU60.","In situ hybridizations were performed as described (D. Jackson, in \u201cPlant Molecular Pathology: A Practical Approach,\u201d S. J. Gurr, M. J. McPherson, D. J. Bowles, Eds., Oxford University Press, Oxford, 1992, vol. I, pp. 163-174) with modifications as described (Bortiri et al., 2006, Plant Cell 18:574). Three additional modifications were adopted: (a) no RNase treatment was performed after hybridizations; (b) the anti-DIG antibody was diluted 1\/1000 and incubated for 2 hr at room temperature; (c) the buffer used for color detection included 10% polyvinyl alcohol (PVA) to increase reaction sensitivity.","Construction of a ts1:mCherry Fusion Gene","The mCherry gene was PCR amplified with PfuUltra\u2122 High-Fidelity DNA Polymerase (Stratagene, La Jolla, Calif., USA) from the pREST-B mCherry vector (S12) with primers P2672 (5\u2032-eggggtaccccATGGTGAGCAAGGGCGAGGAGGAT-3; SEQ ID NO:17) and P2673 (5\u2032-ctagtctagatggatceCTTGTACAGCTCGTCCATGCCGCC-3; SEQ ID NO:18), which added KpnI, NcoI and BamHI, XbaI sites respectively (lower case letters in primer sequences). The PCR product did not contain the endogenous stop codon, Instead, P2673 provided a new stop codon in the XbaI site downstream of the BamHI site. The PCR product was digested with KpnI and XbaI and the 733 by KpnI-XbaI mCherry fragment was gel purified. Plasmid pYU1721 was derived from the plant expression vector pRTL2 (Restrepo et al., 1990, Plant Cell 2:987) and contained the full-length ts1 CDS without the stop codon (ts1\u0394STOP) fused in frame with the gene encoding the monomer Red Fluorescent Protein (mRFP1) (construction details are available upon request). pYU1721 was digested with KpnI and XbaI to release the mRFP1 gene. The remaining \u02dc6.5 kb plasmid containing pRTL2 plus the ts1\u0394STOP CDS was gel purified and ligated to the 733 by KpnI-XbaI mCherry gene. The resulting plasmid, pYU1743, was shown by sequencing to contain an in-frame N-terminus fusion of the ts1\u0394STOP CDS to the mCherry gene.","Biolistic Experiments in Onion Epidermal Cells","One microgram of plasmid DNA was precipitated on gold particles (1.0 \u03bcm, Biorad) essentially as described (Kleinet et al., 1987, Nature 327:70), Onion (L.) bulbs were cut in small pieces. The epidermal cell layers were carefully peeled and transferred to the surface of Petri dishes containing Murashige and Skoog basal medium (MS from Invitrogen, or Sigma-Aldrich, St. Louis, Mo. USA) solidified with 3.5% Phytagel (Sigma-Aldrich). Epidermal cell layers were bombarded with a Biolistic PDS1000\/He Particle Delivery System (Biorad) with 1350 psi rupture discs. After bombardment the plates were incubated at 27\u00b0 C. in darkness for 8-20 h. The epidermal cell layers were then stained for 10 minutes in 1 \u03bcg\/ml DAPI dissolved in 1\u00d7PBS then washed for 10 minutes in 1\u00d7PBS and mounted in glass slides with 70% glycerol in 1\u00d7PBS.","Epifluorescence Microscopy, Photography and Image Analysis","Transformed cells were examined by epifluorescence microscopy with an Axioplan 2 microscope (Carl Zeiss Microimaging, Thornwood, N.Y., USA) with the appropriate excitation\/emission filters. Differential Interference Contrast (DIC or Nomarski microscopy) was used to visualize cells under transmitted light. Digital images were captured with a Zeiss Axiocam with several Image Acquisition Modules of the Zeiss Axiovision software. The Multichannel Fluorescence module allowed the sequential acquisition of DAPI, GFP, mCherry and DIC images for each sample. The onion epidermal cell layer is very thick and not all cell features appear in the same focal plane. Therefore, a series of images over a defined z-focus range were acquired with the Z-Stack module of Axiovision. This module automatically calculated the distance between individual images of the z-stack to achieve the maximum axial resolution of the objective used. The Z-stack was reconstructed with the 3D Deconvolution module where the Regularized Inverted Filter (RIF) method was used. Final image display was completed by applying a maximum intensity projection (MIP) over the entire image volume within the context of the orthogonal slice view (Cut View). The Colocalization module was used to quantitatively assess the colocalization of the TS1:mCherry red fluorescent signal along with the RbcSnt:GFP green fluorescent signal.","Positional Cloning and Mapping of ts1","The molecular marker hlm1 was isolated from the flanking sequence of a Mu4 element that was identified as tightly linked to the ts1-Mu92 mutation. The hlm1 marker was mapped approximately 1 cM from ts1 in a small testcross population (n=93) segregating for the ts1-ref allele. The maize ZMMBBb BAC library (CUGI) was probed with hlm1 and six BAC clones from contig 78 of the current maize physical map (Arizona Genomics Institute) were identified. A marker within this contig, umc2251, was tested for linkage and found to map distal to both ts1 and hlm1-8 and 9 cM respectively (). Therefore, the genetic interval containing ts1 was initially defined as a 9 cM region by the proximal hlm1 and the distal umc2251 markers. To further refine this interval, BAC end sequences in the physical region proximal to umc2251 were analyzed for potential low copy sequences. A molecular marker designed from the end sequence of BAC b0148G01 (CC760183) was tested and found to map 1 cM distal to ts1 (). A larger mapping population (total n=420) was analyzed with CC760183 and hlm1 and a total of 3 distal and 4 proximal recombinants to ts1 were detected. This analysis placed ts1 within a \u02dc1.6 cM genetic interval defined by the proximal hlm1 and the distal CC760183 markers, which spanned a physical region of \u02dc500 kb ().","A genomic sequence (AZM4115428) corresponding to hind was identified in the TIGR AZM 4.0 assembly of the maize methyl-filtered and high-Cot genomic libraries which consists of sequences highly enriched for coding regions (Palmer et al., 2003, Science 302:2115; Whitelaw et al., 2003, Science 302:2118). Orthologs of CC760183 and AZM4115428 were annotated in the rice genome delineating a 62 kb syntenic region that contained 9 predicted genes.","Predicted Function of TS1 Protein","The ts1 gene was thus located in a region with extensive synteny with rice (Salse et al., 2004, Plant J. 38:396). Out of the nine genes contained in the ts1 syntenic interval within the sequenced genome of rice, no maize orthologs were found for four of these genes, and another three were mapped to locations unlinked to the ts1 locus in maize. Maize homologs of the remaining two rice genes, one encoding a putative glutamate decarboxylase and the other encoding a putative lipoxygenase, were confirmed to be contained within the ts1 physical interval ().","Sequencing showed that the gene encoding glutamate decarboxylase was monomorphic, whereas the gene encoding lipoxygenase in the ts1-ref line showed an 864-base pair (bp) insertion in the predicted first exon with complete linkage with the ts1 phenotype in mapping populations. To confirm that the lipoxygenase corresponded to the ts1 gene, eight ts1 mutant alleles were analyzed. Each contained an independent mutation in the gene encoding lipoxygenase (, and Table 3). Complementary DNA sequence analysis showed that the ts1 gene contains seven exons with a coding sequence of 2757 by (). A closely related gene was also identified in the database of the TIGR AZM 4.0 assembly and by Southern blot analysis (). This gene, named ts1b, has an identical exon-intron structure to that of ts1 and shares 93% nucleotide similarity. The ts1b gene is located on maize chromosome 10S, a segmental duplication of chromosome 2S (Gaut, 2001, Genome Res. 11:55). The TS1 protein displays 38 to 60% similarity to plant lipoxygenases and contains two conserved domains characteristic of this family: a beta-barrel (cd01751) and a catalytic helical bundle (pfam00305) (Shibata & Axelrod, 1995, J. Lipid Mediat. Cell Signal. 12:213) (, and ).","Lipoxygenases are non-heme iron-containing fatty acid dioxygenases that catalyze the peroxidation of polyunsaturated fatty acids such as linoleic acid, \u03b1-linolenic acid, and arachidonic acid. They are classified according to the positional specificity of linoleic acid oxygenation, which occurs at carbon 9 of the hydrocarbon backbone for the 9-LOX types and at carbon 13 for the 13-LOX types; a further subdivision (classes 1 and 2) has been recognized for 13-lipoxygenases without or with a putative chloroplast transit peptide (cTP), respectively (Feussner &Wasternack, 2002, Annu. Rev. Plant Biol. 53:275). According to ChloroP, a neural network-based method for predicting cTPs, the N-terminal 48 amino acids of the TS1 protein contain a cTP (Emanuelsson et al., 1999, Protein Sci. 8:978) (, and ). Additionally, TS1 contains a conserved phenylalanine (Phe636) previously identified as a determinant of 13-LOX regiospecificity (Hornung et al., Proc. Natl. Acad. Sci. U.S.A. 96:4192; Liavonchanka & Feussner, 2006, J. Plant Physiol. 163:348). Therefore, the primary structure of TS1 suggests that it is a member of the class 2 plastid-localized 13-lipoxygenases. This prediction was supported by Bayesian and maximum parsimony phylogenetic analyses of plant lipoxygenases, which placed TS1 and TS1b in a Glade including characterized and predicted class 2 13-lipoxygenase (, and ).",{"@attributes":{"id":"p-0104","num":"0103"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"322pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u20033"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Characterization\u2003of\u2003nine\u2003ts1\u2003mutant\u2003alleles"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"70pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"ts1\u2003mutation",{},{},{}]},{"entry":[{},"(Sequences\u2003listed\u20035\u2032 to\u20033\u2032 on",{},"Genetic",{}]},{"entry":["Allele","coding\u2003strand)","Location","background","Source"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":["ts1-ref","864\u2003bp\u2003unknown\u2003insertion;","233-241","Unknown","Emerson,\u20031920,"]},{"entry":[{},"9\u2003bp\u2003target\u2003site\u2003duplication","(exon\u20031)",{},"J.\u2003Hered.\u200311:\u200365"]},{"entry":[{},"(ACCCCCAGA)",{},{},{}]},{"entry":{}},{"entry":["ts1-Mu93","Mu8\u2003insertion;","932-941","Unknown","P.\u2003Chomet\u2003&\u2003B."]},{"entry":[{},"10\u2003bp\u2003target\u2003site\u2003duplication","(exon\u20032)",{},"Lowe"]},{"entry":[{},"(CGTTGACGGG)",{},{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200319)",{},{},{}]},{"entry":{}},{"entry":["ts1-0174","1828\u2003bp\u2003unknown\u2003insertion;","1020-1025","Unknown","G.\u2003F.\u2003Sprague"]},{"entry":[{},"6\u2003bp\u2003target\u2003site\u2003duplication","(exon\u20032)",{},"&\u2003COOP"]},{"entry":[{},"(GGCTTT)",{},{},{}]},{"entry":{}},{"entry":["ts1-Mu92","C\u2192G\u2003transversion;","1308","Unknown","P.\u2003Chomet"]},{"entry":[{},"results\u2003in\u2003nonsense\u2003substitution","(exon\u20032)",{},"and\u2003B.\u2003Lowe"]},{"entry":[{},"Q288\u2003> STOP",{},{},{}]},{"entry":{}},{"entry":["ts1-MUSH","MuDR\u2003insertion;\u20039\u2003bp\u2003target\u2003site","2032-2040","Unknown","S.\u2003Hake"]},{"entry":[{},"duplication","(exon\u20034)",{},{}]},{"entry":[{},"(GGCTGCGGG)",{},{},{}]},{"entry":{}},{"entry":["ts1-Alex","Single\u2003base\u2003pair\u2003deletion;","2419","Mo17","D.\u2003E.\u2003Alexander\/"]},{"entry":[{},"results\u2003in\u2003frameshift","(exon\u20035)",{},"Maize\u2003COOP"]},{"entry":{}},{"entry":["ts1-SH04","Six\u2003by\u2003insertion","3343-3344","W22","S.\u2003Hake"]},{"entry":[{},"(GAGAAG);","(exon\u20037)",{},{}]},{"entry":[{},"results\u2003in\u2003extra\u2003glycine\u2003and",{},{},{}]},{"entry":[{},"glutamic\u2003acid",{},{},{}]},{"entry":{}},{"entry":["ts1Mu02","~3,600\u2003by\u2003unknown\u2003insertion","3655-3659","W22","this\u2003disclosure"]},{"entry":[{},"with\u2003311\u2003bp\u2003terminal\u2003repeats;","(exon\u20037)",{},{}]},{"entry":[{},"5\u2003bp\u2003target\u2003site\u2003duplication",{},{},{}]},{"entry":[{},"(TCCAC)",{},{},{}]},{"entry":{}},{"entry":["ts1-Mu01","Large\u2003deletion\u2003(unknown\u2003size)","\u2014","W22","this\u2003disclosure"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},"Tissue-Specific Expression of ts1 and ts1b Genes","The tissue-specific expression of both ts1 and ts1b was established by quantitative reverse transcription polymerase chain reaction (RT-PCR) analysis of root, stem, leaf, tassel, and ear transcripts. The ts1 RNA was detected in all maize tissues examined, whereas ts1b RNA was detected at very low levels (less than that of ts1 by a factor of 90 to 500) (). The low expression of ts1b may explain why it does not also appear to be a component of sex determination in ts1 mutant plants. The broad expression of ts1 was unexpected because its mutant phenotype suggests a sex-specific function. Although no alterations in other tissues have been reported, it is possible that additional phenotypes for the ts1 mutation may be uncovered by more careful analyses. The ts2 gene was expressed almost as broadly as ts1, except in stem tissue, where expression was less than that of ts1 by a factor of \u02dc35.","In situ hybridization showed that TS1 transcripts form stripes following the borders of the central inflorescence axis and projecting toward the spikelet attachment points (), In spikelet adaxial views, TS1 expression domains surround the spikelets, delineating their base (). None of these expression patterns were observed in a homozygous ts1-Mu01 deletion mutant line containing a functional ts1 b gene (). These observations indicate that TS1 transcripts subtend maize spikelets at their junction with the central inflorescence axis (rachis). This expression domain suggests a function for TS1, as metabolites synthesized through the lipoxygenase encoded by ts1 could act as diffusible signals affecting floral development in a non-cell-autonomous fashion.","The ChloroP-based prediction that TS1 localizes in plastids was confirmed with a fluorescent-tagged TS1 protein (TS1:mCherry) and a plastid-localized RbcSnt:GFP protein (Lee et al., 2002, Mol. Cells. 14:388). To quantitatively assess the colocalization of RbcSnt:GFP and TS1:mCherry fluorescent signals, correlation analysis of the intensity values of color (green and red) pixels in the dual-channel image was performed. Specifically, the Colocalization module in the Axiovision software (Carl Zeiss Microimaging, Thornwood, N.Y., USA) plotted the pixel grey values of fluorescent intensity of the x-axis and y-axis channels against each other (). Then, correlation coefficients were calculated to measure the strength of the linear relationship between the two variables (Bolte & Cordelieres, 2006, J. Microsc. 224:213; Manders et al., 1992, J. Cell Sci, 103:857). The images in  showed a Manders' overlap coefficient of 0.986 strongly indicating colocalization between the two signals. Additionally, weighted co-localization coefficients M1 and M2, which are independent of differences in fluorescence intensity between the two channels, were calculated. For TS1:mCherry, M2=0.906, indicating that a high proportion of red signal coincided with a signal in the green channel over its total intensity (Bolte & Cordelieres, 2006, J. Microsc. 224:213; Manders et al., 1992, J. Cell Sci. 103:857). A similar conclusion was drawn for the weighted co-localization coefficient of RbcSnt:GFP (M1=0.903), indicating that TS1:mCherry protein is targeted to the same subcellular compartment as RbcSnt:GFP. TS1 is thus targeted to plant plastids.","Analysis of Lipoxygenase Activity in Crude Protein Extracts from Tassel Inflorescences","Frozen tassels were ground in liquid nitrogen and\/or homogenized in a 1.5-ml microcentrifuge tube with 1 to 2 volumes of ice-cold 0.1 M Tris buffer, pH 7.5, containing 0.1 M NaCl, 5 mM EDTA, 0.1% \u03b2-mercaptoethanol, 0.6% Triton X-100, 1 mM PMSF and EDTA-free Protease Inhibitors (Roche, Indianapolis, Ind., USA), The mixture was clarified by centrifugation at 14,000\u00d7g for 30 minutes at 4\u00b0 C. The supernatant was transferred to a new tube and total protein concentration estimated with the Micro BCA Protein Assay Kit (Pierce, Rockford, Ill., USA),","Aliquots of crude protein extracts were added to 3 ml of potassium phosphate buffer, pH 6.0 containing 150 \u03bcM linoleic acid and stirred at 23\u00b0 C. for 15 min. Peroxidation products were reduced to hydroxides, HOD) by adding 12 ml of a solution of 5 mg\/ml SnClin ethanol and incubating for another 5 minutes at 23\u00b0 C. Products isolated by extraction with diethyl ether were methyl-esterified and analyzed by GC-MS and HPLC. For all analyses, reference oxylipins of high chemical purity (Larodan Fine Chemicals, Malm\u00f6, Sweden) were used. Material to be analyzed by GC-MS was derivatized by treatment with trimethylchlorosilane\/hexamethyldisilazane\/pyridine (2:1:2, v\/v\/v) at room temperature for 15 min. Excess reagent and solvent were removed in vacuo and the residue was resuspended in hexane. GC-MS was carried out with a mass-selective detector (Hewlett-Packard model 5970B, Avondale, Pa., USA) connected to a gas chromatograph (Hewlett-Packard model 5890) equipped with a capillary column of 5% phenylmethylsiloxane (12 m, 0.33 \u03bcm film thickness). Helium was used as the carrier gas, and the column temperature was raised from 120\u00b0 C. to 300\u00b0 C. at 10\u00b0 C.\/min.","SP-HPLC of methyl-esterified incubation products was carried out with a column of Nucleosil 50-5 (250\u00d74,6 mm) purchased from Macherey-Nagel, D\u00fcren, Germany, and a solvent system of 0.6% 2-propanol\/hexane at a flow rate of 2 ml\/min. The absorbance (234 nm) and radioactivity of HPLC effluents were determined on-line with a Spectromonitor III ultraviolet detector (Laboratory Data Control, Riviera Beach, Fla., USA) and a liquid scintillation counter (IMUS Systems, Tampa, Fla., USA), respectively. Under the conditions used, the elution order of hydroxyoctadecadienoates were: methyl 13-hydroxy-9(Z), 11(E)-octadecadienoate (first), methyl 13-hydroxy-9(E),11(E)- octadecadienoate, methyl 9-hydroxy-10(E),12(Z)-octadecadienoate and methyl 9-hydroxy-10(E),12(E)-octadecadienoate (last).","Biochemical analysis of protein extracts from developing tassels for activity on the lipoxygenase substrate linoleic acid suggests that TS1 is capable of lipid peroxidation. Protein extracts from wild-type tassels catalyzed hydroperoxidation of linoleic acid, whereas no such activity was detected in mutant ts1-ref\/ts-1ref developing tassels (). Mass spectrometry (MS) and high-performance liquid chromatography (HPLC) analyses showed that the products of this lipoxygenase activity are a mixture of 9- and 13-hydroperoxides in a 50:50 ratio (); primary structure analysis had suggested that TS1 was a lipoxygenase with 13-regiospecifieity. Therefore, it is possible that TS1 possesses dual 9- and 13-regiospecificity\u2014which has not previously been described for a plastid-localized lipoxygenase\u2014or that TS1 function promotes the action of a separate 9-lipoxygenase.","Quantification of Jasmonic Acid and Other Metabolites in Tassel Inflorescences","Class 2 13-lipoxygenases participate in the biosynthesis of the plant hormone jasmonic acid (JA) (Wasternack, 2007, Ann. Bot. 100:681) (). The involvement of TS1 in JA biosynthesis was evaluated by measuring endogenous JA levels in developing wild-type and ts1-ref\/ts1-ref mutant tassels.","As shown in , the first dedicated step in jasmonate biosynthesis is the peroxidation of \u03b1-linolenic acid (18:3) by 13-lipoxygenase to form (13S)-hydroperoxyoetadecatrienoic acid (13-HPOT). This is the putative function of TS1. 13-HPOT is transformed into the specific stereoisomer cis-(+)-12-oxophytodienoic acid (OPDA) through the sequential action of allene oxide synthase [yielding (13S)-12,13-epoxy-oetadecatrienoic acid (12,13-EOT)] and allene oxide cyclase. These steps in JA biosynthesis occur in plant plastids, where the corresponding enzymes are localized. Subsequent reactions occur in the peroxisomes. First, the cyclopentenone ring of OPDA is reduced to 12-oxophytoenoic acid (OPC-8) by OPDA reductase. Next, three \u03b2-oxidation cycles are proposed to shorten the carboxylic side chain of OPC-8 to produce the 12-carbon JA. A \u03b2-oxidation cycle is a set of four enzymatic reactions: oxidation, hydration, oxidation, and thiolysis. Not all enzymes acting on \u03b2-oxidation during JA biosynthesis have been identified. Because the oxidation in the third step is normally performed by a dehydrogenase activity, it is possible that TS2 may participate in this step of JA biosynthesis.","Maize plants of W22, ts1-ref\/ts1-ref and ts1-ref\/+ were field-grown during the summer of 2007. Developing tassel inflorescences between 0.8 and 3 cm in length were quickly dissected, placed in 1.5 ml microcentrifuge tubes and rapidly frozen in liquid nitrogen. Tissue samples were stored at \u221280\u00b0 C. prior to solvent extraction. Jasmonic acid quantification was performed with vapor phase extraction for sample preparation and chemical ionization gas chromatography\/mass spectrometry (CI-GC\/MS) as described (Sehmelz et al., 2004, Plant J. 39:790). Descriptive and comparative statistics were obtained with Analyse-it\u00ae Standard edition (Analyse-it Software, Ltd, Leeds, England), an add-in for Microsoft Excel.","The average concentration of JA in wild-type and ts1-ref\/+ heterozygotes was 44.2 T 13.9 ng per gram of fresh weight (ng\/g FW) and 40.3 T 20.2 ng\/g FW, respectively (). Homozygous ts1-ref\/ts1-ref tassels showed an average JA concentration of 4.3 T 2.1 ng\/g FW (), significantly below that of the wild type in a Kruskal-Wallis test and pair vise comparisons with a Bonferroni correction (P<0.0001). The ts1 mutation thus appears to reduce JA levels by a factor of \u02dc10, indicating a role for the hormone in the pistil cell death process. JA levels of wild-type and mutant ts1 tassels are similar to those of wounded and nonwounded maize seedlings, respectively (Engelberth et al., 2007, Mol. Plant. Microbe Interact. 20:707), which supports the notion that JA is actively synthesized during normal tassel development.","Chemical Treatment of Maize Plants","Maize plants were grown under greenhouse conditions. Two seeds were planted in 2-gallon reusable pots containing Super-Fine Germinating Mix (Fafard, Agawam, Mass., USA) or Redi-earth Professional Growing Mix (Sun Gro, Bellevue, Wash., USA). Fertilization was performed with controlled release Osmocote Plus 15-9-12 (Scotts, Marysville, Ohio, USA) following the manufacturer recommendations. Additional watering was performed with 4% ammonium iron citrate (Sigma-Aldrich) every 2 weeks to prevent chlorosis. Greenhouse average temperatures were 28\u00b0 C. (day) and 21\u00b0 C. (night). Supplemental lightning was provided to achieve a 16:8 hour photoperiod yearlong.","Tassels of about 1 cm were considered to be at an ideal stage to initiate chemical treatments. Since a thick leaf whorl covers the maize tassel at this time of development, only a destructive dissection of the plant permits to assess with certainty the stage of tassel development (Bonnet, 1940, J. Agricult. Res. 60:25; Bonnet, 1948, Ann. Mo. Bot. Gard. 35:269). Additionally, the sex determination phase was reached within a wide time window, between 28 and 46 days post-planting, depending on the growing conditions. Leaf number, node number or internode distance were not always reliable criteria to predict the tassel developmental stage.","Seedlings from a family segregating 1:1 for wild-type (ts1-ref\/Ts1) and ts1 mutant (ts1-ref\/ts1-ref) plants were genotyped for the Ts1 and ts1-ref alleles with a PCR-based assay. Forty-six days after planting, one ts1 and one heterozygote plant showing 6-8 fully expanded leaves were dissected and their tassels were shown to be approximately 1.0 cm in height. This was used as a rough estimate indicating that similar plants, as judged by height and leaf number, were at the right stage to receive chemical treatments. Jasmonic acid (JA, Sigma-Aldrich) was dissolved at a concentration of 200 mM in absolute ethanol and stored at \u221220\u00b0 C. Prior to plant treatment, the JA stock solution was diluted to a concentration of 1 mM in deionized water. Control plants were treated with a 0.005% ethanol solution (\u201cblank\u201d treatment or negative control). One ml of the corresponding solution was applied into the apical leaf cavity of each plant. Treatments were performed three times at 48-hour intervals.","The developmental timing of the pistil abortion process occurs in tassel inflorescences when they are 1.0 to 3.0 cm in length (Irish & Nelson, 1993, Am. J. Bot. 80:292), which was also the stage at which ts1 expression occurred in the subtending glumes (). A 0.005% ethanol solution, with or without 1 mM JA, was applied to tassels of \u02dc1 cm in wildtype (ts1-ref\/+) or ts1 mutant (ts1-ref\/ts1-ref) sibling plants. In ts1 mutant plants, JA application reversed feminization, as evidenced by the presence of staminate spikelets mostly in the mid- to apical regions (). Wildtype rescue in ts1 mutants was observed in the appearance of subtending floral bracts (glumes) about 3 to 4 weeks after treatment. JA-treated glumes in ts1 mutants were elongated, were covered with numerous trichomes, and had a ring of anthocyanin deposited at the base (), all three characteristics of wild-type staminate spikelets. Later in floral development, stamens emerged from JA-treated ts1 mutant spikelets ().","Staminate florets from rescued IA-treated ts1\/ts1 plants produced viable pollen, which was used for both self-pollination and test crosses to untreated ts1\/ts1 mutant sibs. All test cross progeny (n>100) were homozygous for the ts1-ref allele and displayed a complete ts1 mutant phenotype. The JA-rescued phenotype of the tassel inflorescence was incomplete in that some spikelets were bisexual (), containing both pistils and stamens, and others (mainly those located at the base of the inflorescence) were pistillate. These effects, however, may have been due to the timing of JA treatments, because the stage of floral maturation differs in a positionally dependent fashion throughout the inflorescence. Rescued staminate spikelets were never observed in blank-treated ts1-ref\/ts1-ref plants (; , and Table 4), nor did JA treatment affect heterozygous ts1-ref\/+ sibs (Table 4). The similar phenotype of ts1 and ts2 mutations indicates that both genes may act in the same metabolic pathway. Therefore, JA was also applied to mutant ts2-ref\/ts2-ref and ts2-ref\/+ plants, which responded in the same manner as the JA-treated ts1 mutants (; ; and Table 4). These results indicate that JA can restore the wild-type phenotype in both ts1 and ts2 mutant plants. Moreover, TS2 may have an unexpected role in IA biosynthesis, perhaps as one of the yet-unidentified enzymes catalyzing a series of \u03b2-oxidations in this metabolic pathway ().","Genes regulating meristem determinacy early in maize inflorescence development are expressed at the boundary of the meristem and the inflorescence axis rather than within the meristem itself. These genes, such as ramosa1, ramosa3, and barren stalk1 (Gallovatti et al, 2004, Nature 432:630; Satoh-Nagasawa et al., 2006, Nature 441:227; Vollbrecht et al., 2005, Nature 441:227), probably act non-cell-autonomously by producing a diffusible signal at the base of the meristem (Vollbrecht et al., 2005, Nature 441:227). Analogously, ts1 expression at the boundary of developing spikelet initials and inflorescence axis produce the hormone JA, which may diffuse within the spikelet to regulate sexual development. This situation parallels JA-mediated anther dehiscence in Arabidopsis, where JA biosynthetic genes are highly expressed in the anther filament where it signals development both in the filament and within the anther (Ishiguro et al., 2001, Plant Cell 13:2191; Sanders et al., 2000, Plant Cell 12:1041). The expression of ts2 is known to be reduced in ts\/mutants (Calderon-Urrea & Dellaporta, 1993, Development 126:435). The finding that ts2 may be involved in the same biosynthetic pathway as ts1 is not necessarily at odds with previous observation, as most genes encoding enzymes of the JA biosynthetic pathway are transcriptionally up-regulated by JA in a characteristic positive feedback loop (Wasternack, 2007, Ann. Bot. 100:681).","JA signals plant responses to biotic and abiotic stresses (Wasternack, 2007, Ann. Bot. 100:68) and regulates plant developmental processes such as root growth (Staswicknet al., 1992, PNAS U.S.A. 89:6837) and mechanotransduction (Weiler et al., 1993, Phytochemistry 32:591). In Arabidopsis, JA is required for male fertility because pollen maturation and anther dehiscence are blocked in mutations that impair JA biosynthesis (Ishiguro et al., 2001, Plant Cell 13:2191; Sanders et al., 2000, Plant Cell 12:1041). JA may promote anther dehiscence by signaling degeneration of the stomium, a group of specialized cells that run along the length of the anther and are necessary for dehiscence (Sanders et al., 1999, Sex, Plant Reprod. 11:297). The present results imply a role for JA in maize sex determination, wherein JA is necessary for signaling the tasselseed-mediated pistil abortion and the acquisition of the male characteristics of staminate spikelets. The diverse mechanisms of hormonal control in plant sex determination support the notion that the systems have evolved independently multiple times (Ainswortyh et al., 1998, Curr. Top. Dev. Biol. 38:167).","The disclosures of each and every patent, patent application, and publication cited herein are hereby incorporated herein by reference in their entirety.","While the invention has been disclosed with reference to specific embodiments, it is apparent that other embodiments and variations of this invention may be devised by others skilled in the art without departing from the true spirit and scope of the invention. The appended claims are intended to be construed to include all such embodiments and equivalent variations."],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":["For the purpose of illustrating the invention, there are depicted in the drawings certain embodiments of the invention. However, the invention is not limited to the precise arrangements and instrumentalities of the embodiments depicted in the drawings.",{"@attributes":{"id":"p-0016","num":"0015"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0017","num":"0016"},"figref":["FIG. 2","FIGS. 2A-2C","FIG. 2A","FIG. 2B","FIG. 2C","FIG. 5"]},{"@attributes":{"id":"p-0018","num":"0017"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0019","num":"0018"},"figref":["FIG. 4","FIGS. 4A-4C"],"b":"0"},{"@attributes":{"id":"p-0020","num":"0019"},"figref":["FIG. 5","FIGS. 5A-5B"]},{"@attributes":{"id":"p-0021","num":"0020"},"figref":["FIG. 6","FIGS. 6A-6I","FIG. 6A","FIGS. 6B to 6E","FIGS. 6B and 6C","FIG. 6D","FIG. 6E","FIGS. 6F to 6I","FIG. 6F","FIG. 6G","FIG. 6H","FIG. 6I"]},{"@attributes":{"id":"p-0022","num":"0021"},"figref":["FIG. 7","FIGS. 7A-7E","FIG. 7A","FIG. 7B","FIG. 6C","FIG. 6D","FIG. 6E"]},{"@attributes":{"id":"p-0023","num":"0022"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0024","num":"0023"},"figref":["FIG. 9","FIGS. 9A-9D","FIG. 9A","FIG. 9B","FIG. 9C","FIG. 9D"]}]},"DETDESC":[{},{}]}
