---
title: Method for enzymatic production of decarboxylated polyketides and fatty acids
abstract: Disclosed herein are methods of preparing alkenes from beta-hydroxy or beta-sulfate carboxylic acid or carboxylic acid derivatives using thioesterase and optionally a sulfotransferase.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08765431&OS=08765431&RS=08765431
owner: The Regents of the University of Michigan
number: 08765431
owner_city: Ann Arbor
owner_country: US
publication_date: 20100723
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS REFERENCE TO RELATED APPLICATIONS","STATEMENT OF U.S. GOVERNMENTAL INTEREST","BACKGROUND","SUMMARY","DETAILED DESCRIPTION","EXAMPLES","4-Methoxytridec-1-ene (3)","3-Methoxydodecanal (5)","(4R)-4-((2RS)-2-Methoxyundecyl)oxetan-2-one (6)","(3R,5RS)-3-Hydroxy-5-methoxytetradecanoic acid ((3R)-1)","(4S)-4-((2RS)-2-Methoxyundecyl)oxetan-2-one (7)","(3S,5RS)-3-Hydroxy-5-methoxytetradecanoic acid ((3S)-1)","(3R,5RS)-3-Sulfonatooxy-5-methoxytetradecanoic acid ((3R)-2)","REFERENCES"],"p":["The benefit of U.S. Provisional Application No. 61\/227,987, filed Jul. 23, 2009 is claimed, and the disclosure of which is incorporated by reference herein in its entirety.","This invention was made with government support under DK042303 and CA108874 awarded by the National Institutes of Health. The Government has certain rights in the invention.","Curacin A is a mixed polyketide\/non-ribosomal peptide with antimitotic properties produced by the marine cyanobacterium (4). The hybrid polyketide synthase (PKS)\/non-ribosomal peptide synthase (NRPS) biosynthetic pathways that produces curacin A (5) contains numerous unique chemical steps, many of which have been previously investigated (1,3,6,22). The synthesis of a terminal alkene, instead of the carboxyl typical for this class of linear natural products, and the unique domain arrangement in the terminal PKS module are mysteries yet to be fully elucidated.","In CurM, the terminal module, a sulfotransferase (ST) and thioesterase (TE) domain follow the acyl carrier protein (ACP) (.) STs catalyze the transfer of a sulfate group from the donor 3\u2032-adenosine 5\u2032-phosphosulfate (PAPS) to a hydroxyl or amine of their acceptor small molecule or protein (10). STs are involved in vital processes such as detoxification, hormone regulation, and signaling and have been studied in a wide variety of organisms. However, the ST in the curacin pathway is the first observation of a ST within a PKS module and contains low (15%) sequence identity to other studied STs.","The TE, although identifiable as a thioesterase, does not resemble any of the previously established fatty acid synthase (FAS), PKS or NRPS TE families (24). Many PKS offloading TEs have been studied to date including the TEs of the pikromycin synthase (Pik) (25,26) and erythromycin synthase (DEBS) (27) pathways. PKS offloading TEs typically perform either hydrolysis to produce a carboxylic acid or catalyze the attack of an intramolecular hydroxyl to form a macrolactone. These TEs are dimers, with two N-terminal alpha helices forming a lid-to-lid dimer interface, and adopt the \u03b1\/\u03b2 hydrolase fold characteristic of some serine hydrolases. Access to the classic nucleophile-His-acid catalytic triad active site is restricted by a narrow tunnel formed by a closed lid. Many PKS and NRPS pathways also include a second non-modular thioesterase called a TE II (in addition to an offloading TE, also known as TE I), which performs an editing function within the pathway. The curacin TE shows low similarity to sequences in all parts of the phylogenic tree (24), also pointing the need to more closely study curacin TE to understand its activity.","Disclosed herein are methods of preparing alkenes by decarboxylation of beta-sulfate carboxylic acids or carboxylic acid derivatives. More specifically, disclosed herein are methods of preparing alkenes by decarboxylative elimination to form the alkene.","Thus, in one aspect, provided herein is a method of contacting a beta-sulfate carboxylic acid or carboxylic acid derivative with a TE such that the TE mediates decarboxylative elimination of the beta-sulfate carboxylic acid or carboxylic acid derivative to form the alkene. In some cases, the TE comprises an amino acid sequence of SEQ ID NO: 3 or an enzymatically active fragment thereof which maintains the TE activity of SEQ ID NO: 3.","The method disclosed herein can further comprise contacting a beta-hydroxy carboxylic acid or carboxylic acid derivative with a sulfonating reagent and a sulfotransferase (ST) such that the ST mediates the formation of the beta-sulfate carboxylic acid or carboxylic acid derivative. In some cases, the ST comprises an amino acid sequence of SEQ ID NO: 4, or an enzymatically active fragment thereof which maintains the ST activity of SEQ ID NO: 4. The ST and TE can be in the same polypeptide or in different polypeptides.","The carboxylic acid derivative can comprise a carboxylic acid conjugated to an acyl carrier protein (ACP). In some cases, the ACP comprises an amino acid sequence SEQ ID NO: 5 or active fragment thereof. In various embodiments, the ACP and at least one of the TE and ST (e.g., ACP and TE and\/or ACP and ST) are in the same polypeptide. In some cases, the ACP, TE and ST are all in the same polypeptide. In some embodiments, the ACP, TE and ST are all in the same polypeptide and that polypeptide comprises an amino acid sequence of SEQ ID NO: 1.","In some embodiments, the alkene is a terminal alkene. In various cases, the alkene has a structure of formula (II) and the beta-sulfate or beta-hydroxy carboxylic acid or carboxylic acid derivative has a structure of formula (I):",{"@attributes":{"id":"p-0012","num":"0011"},"chemistry":{"@attributes":{"id":"CHEM-US-00001","num":"00001"},"img":{"@attributes":{"id":"EMI-C00001","he":"49.19mm","wi":"52.92mm","file":"US08765431-20140701-C00001.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{},"ul":{"@attributes":{"id":"ul0001","list-style":"none"},"li":["Rand Rare each independently selected from the group consisting of hydrogen, alkyl, heteroalkyl, cycloalkyl, aryl, heteroaryl, heterocycloalkyl, alkenyl, or alkynyl and can be optionally substituted with one or more of halo, alkyl, heteroalkyl, alkenyl, cycloalkyl, aryl, heteroaryl, heterocycloalkyl, nitro, cyano, amino, alkoxy, carboxy, carboxyalkyl, amido, thiol, hydroxy, and thioether;","X is OH, SH, OR, or SR;","Ris hydroxy or sulfate; and","Ris optionally substituted alkyl, optionally substituted alkenyl, or a peptide. One example of a peptide of formula (I) is an ACP."]}},"In another aspect, disclosed herein is an isolated crystalline form of a sulfotransferase (ST) polypeptide comprising an amino acid sequence of SEQ ID NO: 19, a space group P222, unit cell parameters of a=45.8 \u212b, b=67.3 \u212b, c=118.0 \u212b, \u03b2=\u03b2=\u03b3=90\u00b0, and one ST molecule in an asymmetric unit.","In yet another aspect, disclosed herein is an isolated crystalline form of a thioesterase (TE) polypeptide comprising an amino acid sequence of SEQ ID NO: 21, a space group P2, unit cell parameters of a=74.5 \u212b, b=86.9 \u212b, c=87.6 \u212b, \u03b1=\u03b3=90\u00b0, \u03b2=90.8\u00b0, and four TE molecules in an asymmetric unit.","In still another aspect, disclosed herein is an isolated thioesterase (TE) polypeptide comprising an amino acid sequence that is greater than 75% identical to SEQ ID NO: 3 and exhibits TE activity. In some cases, the sequence is greater than 90%, greater than 95%, or greater than 98% identical to SEQ ID NO: 3. In a specific embodiment, the sequence is SEQ ID NO: 3.","Further disclosed herein is a polynucleotide encoding an isolated TE polypeptide disclosed herein. Also disclosed herein is a vector comprising such a polynucleotide, and a host cell comprising the polynucleotide or vector.","In another aspect, provided herein is a method of preparing a disclosed isolated TE polypeptide comprising culturing a host cell as disclosed herein and recovering the polypeptide.","In a further aspect, the invention provides an antibody specifically reactive with a polypeptide described herein.","Disclosed herein are methods of synthesizing an alkene, e.g., a terminal alkene, using natural or engineered enzymes. In particular, disclosed herein are methods of preparing an alkene from a beta-sulfate (OSO) carboxylic acid or carboxylic acid derivative by contacting the beta-sulfate carboxylic acid or carboxylic acid derivative with a thioesterase (TE) or fragment of a TE having TE enzymatic activity to form the alkene by decarboxylative elimination. The method can further comprise formation of the beta-sulfate carboxylic acid or carboxylic acid derivative by contacting a beta-hydroxy carboxylic acid or carboxylic acid derivative with a sulfotransferase (ST) or fragment of a ST having ST enzymatic activity to form the beta-sulfate carboxylic acid or carboxylic acid derivative thereof. Any beta-hydroxy (or sulfate) carboxylic acid A representative beta-hydroxy (sulfate) carboxylic acid or derivative thereof is illustrated in Formula (I), and can form an alkene of Formula (II):",{"@attributes":{"id":"p-0033","num":"0036"},"chemistry":{"@attributes":{"id":"CHEM-US-00002","num":"00002"},"img":{"@attributes":{"id":"EMI-C00002","he":"49.19mm","wi":"52.92mm","file":"US08765431-20140701-C00002.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{},"ul":{"@attributes":{"id":"ul0002","list-style":"none"},"li":["Rand Rare each independently selected from the group consisting of hydrogen, alkyl, heteroalkyl, cycloalkyl, aryl, heteroaryl, heterocycloalkyl, alkenyl, or alkynyl and can be optionally substituted with one or more of halo, alkyl, heteroalkyl, alkenyl, cycloalkyl, aryl, heteroaryl, heterocycloalkyl, nitro, cyano, amino, alkoxy, carboxy, carboxyalkyl, amido, thiol, hydroxy, and thioether;","X is OH, SH, OR, or SR;","Ris hydroxy or sulfate (e.g., OSO); and","Ris optionally substituted alkyl, optionally substituted alkenyl, or a peptide, such as, e.g., an ACP."]}},"\u201cDecarboxylative elimination\u201d used herein refers to elimination of a carboxylic acid or derivative thereof and optionally a beta-hydroxy (or sulfate) moiety to form an alkene. This process can be illustrated in the following reaction scheme:",{"@attributes":{"id":"p-0035","num":"0042"},"chemistry":{"@attributes":{"id":"CHEM-US-00003","num":"00003"},"img":{"@attributes":{"id":"EMI-C00003","he":"10.67mm","wi":"44.20mm","file":"US08765431-20140701-C00003.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"A \u201ccarboxylic acid derivative\u201d as used herein refers to a moiety such as an ester, a thioester, an amide, or the like.","The term \u201calkyl\u201d used herein refers to a saturated or unsaturated straight or branched chain hydrocarbon group of one to ten carbon atoms, including, but not limited to, methyl, ethyl, n-propyl, isopropyl, n-butyl, isobutyl, tert-butyl, n-hexyl, and the like. Alkyls of one to six carbon atoms are also contemplated. The term \u201calkyl\u201d includes \u201cbridged alkyl,\u201d i.e., a bicyclic or polycyclic hydrocarbon group, for example, norbornyl, adamantyl, bicyclo[2.2.2]octyl, bicyclo[2.2.1]heptyl, bicyclo[3.2.1]octyl, or decahydronaphthyl. Alkyl groups optionally can be substituted, for example, with hydroxy (OH), halo, aryl, heteroaryl, cycloalkyl, heterocycloalkyl, and amino. It is specifically contemplated that in the analogs described herein the alkyl group consists of 1-40 carbon atoms, 1-25 carbon atoms, 1-15 carbon atoms, 1-12 carbon atoms, 1-10 carbon atoms, 1-8 carbon atoms, and 1-6 carbon atoms. \u201cHeteroalkyl\u201d is defined similarly as alkyl, except the heteroalkyl contains at least one heteroatom independently selected from the group consisting of oxygen, nitrogen, and sulfur.","As used herein, the term \u201ccycloalkyl\u201d refers to a cyclic hydrocarbon group, e.g., cyclopropyl, cyclobutyl, cyclohexyl, and cyclopentyl. \u201cHeterocycloalkyl\u201d is defined similarly as cycloalkyl, except the ring contains one to three heteroatoms independently selected from the group consisting of oxygen, nitrogen, and sulfur. Nonlimiting examples of heterocycloalkyl groups include piperdine, tetrahydrofuran, tetrahydropyran, dihydrofuran, morpholine, thiophene, and the like. Cycloalkyl and heterocycloalkyl groups can be saturated or partially unsaturated ring systems optionally substituted with, for example, one to three groups, independently selected from the group consisting of alkyl, alkyleneOH, C(O)NH, NH, oxo (\u2550O), aryl, haloalkyl, halo, and OH. Heterocycloalkyl groups optionally can be further N-substituted with alkyl, hydroxyalkyl, alkylenearyl, or alkyleneheteroaryl.","The term \u201calkenyl\u201d used herein refers to a straight or branched chain hydrocarbon group of two to thirty, or more, carbon atoms containing at least one carbon double bond including, but not limited to, 1-propenyl, 2-propenyl, 2-methyl-1-propenyl, 1-butenyl, 2-butenyl, and the like. \u201cAlkynyl\u201d refers to a straight or branched chain hydrocarbon group of two to thirty, or more, carbon atoms containing at least one carbon triple bond.","The term \u201chalo\u201d used herein refers to fluoro, chloro, bromo, or iodo.","The term \u201calkylene\u201d used herein refers to an alkyl group having a substituent. For example, the term \u201calkylene aryl\u201d refers to an alkyl group substituted with an aryl group. The alkylene group is optionally substituted with one or more substituent previously listed as an optional alkyl substituent. For example, an alkylene group can be \u2014CHCH\u2014.","As used herein, the term \u201calkenylene\u201d is defined identical as \u201calkylene,\u201d except the group contains at least one carbon-carbon double bond.","As used herein, the term \u201caryl\u201d refers to a monocyclic or polycyclic aromatic group, preferably a monocyclic or bicyclic aromatic group, e.g., phenyl or naphthyl. Unless otherwise indicated, an aryl group can be unsubstituted or substituted with one or more, and in particular one to four groups independently selected from, for example, halo, alkyl, alkenyl, OCF, NO, CN, NC, OH, alkoxy, amino, COH, COalkyl, aryl, and heteroaryl. Exemplary aryl groups include, but are not limited to, phenyl, naphthyl, tetrahydronaphthyl, chlorophenyl, methylphenyl, methoxyphenyl, trifluoromethylphenyl, nitrophenyl, 2,4-methoxychlorophenyl, and the like.","As used herein, the term \u201cheteroaryl\u201d refers to a monocyclic or bicyclic ring system containing one or two aromatic rings and containing at least one nitrogen, oxygen, or sulfur atom in an aromatic ring. Unless otherwise indicated, a heteroaryl group can be unsubstituted or substituted with one or more, and in particular one to four, substituents selected from, for example, halo, alkyl, alkenyl, OCF, NO, CN, NC, OH, alkoxy, amino, COH, COalkyl, aryl, and heteroaryl. Examples of heteroaryl groups include, but are not limited to, thienyl, furyl, pyridyl, oxazolyl, quinolyl, thiophenyl, isoquinolyl, indolyl, triazinyl, triazolyl, isothiazolyl, isoxazolyl, imidazolyl, benzothiazolyl, pyrazinyl, pyrimidinyl, thiazolyl, and thiadiazolyl.","The term \u201calkoxy\u201d used herein refers to straight or branched chain alkyl group covalently bonded to the parent molecule through an \u2014O\u2014 linkage. Examples of alkoxy groups include, but are not limited to, methoxy, ethoxy, propoxy, isopropoxy, butoxy, n-butoxy, sec-butoxy, t-butoxy and the like.","The term \u201camino\u201d as used herein refers to NR, where R is independently hydrogen, optionally substituted alkyl, optionally substituted heteroalkyl, optionally substituted cycloalkyl, optionally substituted heterocycloalkyl, optionally substituted aryl or optionally substituted heteroaryl. In some cases, R is independently hydrogen or alkyl. Non-limiting examples of amino groups include NHand N(CH).","The term \u201camido\u201d as used herein refers to \u2014C(O)NH, \u2014C(O)NR, \u2014NRC(O)R or \u2014NHC(O)H, where each R is independently hydrogen, optionally substituted alkyl, optionally substituted heteroalkyl, optionally substituted cycloalkyl, optionally substituted heterocycloalkyl, optionally substituted aryl or optionally substituted heteroaryl. In some cases, the amido group is \u2014NHC(O)alkyl or \u2014NHC(O)H. A non-limiting example of an amido group is \u2014NHC(O)CH.","The term \u201ccarboxy\u201d or \u201ccarboxyl\u201d used herein refers to \u2014COOH or its deprotonated form \u2014COO. Carboxyalkyl refers to optionally substituted alkyl or alkenyl groups having a carboxy moiety. Examples include, but are not limited to, \u2014CHCOOH, \u2014CHCH(COOH)CH, and CHCHCHCOOH.","The TE, ST, and\/or ACP can be in separate polypeptides or in the same polypeptide. For example, CurM (SEQ ID NO: 1\u2014amino acid sequence; SEQ ID NO: 2\u2014nucleic acid sequence) comprises a TE, ST, and ACP domain at residues 1929 to 2211 (TE) (SEQ ID NO: 3); 1622 to 1905 (ST) (SEQ ID NO: 4); and 1504 to 1592 (ACP) (SEQ ID NO: 5).","It is contemplated that longer or indeed shorter peptides of TE, ST, and\/or ACP also may prove useful. Thus, also contemplated are peptides that comprise at least 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90 or more amino acids from the TE, ST, and\/or ACP peptide added to its N-terminus. For example, the amino acid at position 1928 of SEQ ID NO: 1, 1621 of SEQ ID NO:1, and\/or 1503 of SEQ ID NO:1 could be added to a peptide described herein, if the addition of 1 amino acid to the N-terminus of a peptide sequence described herein is desired. Similarly, the amino acids from any one of positions 1502, 1501, 1500, 1499, 1498, 1497, 1496, 1495, 1494, 1489, or 1484 to 1503 of SEQ ID NO: 1 could be added to a peptide described herein if the addition of 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, or 20 amino acids, respectively, to the N-terminus of the ACP peptide is desired. The amino acids from any one of positions 1620, 1619, 1618, 1617, 1616, 1615, 1614, 1613, 1612, 1607, or 1602 to 1621 of SEQ ID NO: 1 could be added to a peptide described herein if the addition of 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, or 20 amino acids, respectively, to the N-terminus of the ST peptide is desired. The amino acids from any one of positions 1927, 1926, 1925, 1924, 1923, 1922, 1921, 1920, 1919, 1914, or 1909 to 1928 of SEQ ID NO: 1 could be added to a peptide described herein if the addition of 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, or 20 amino acids, respectively, to the N-terminus of the TE peptide is desired.","In some embodiments, a peptide described herein comprises at least 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90 or more amino acids added to its C-terminus. These amino acids can be from the native CurM sequence or can be unnatural additional amino acids added that still at least substantially maintain the enzymatic activity of the ACP, ST, and\/or TE. For example, the amino acid at position 1593 of SEQ ID NO: 1 and\/or 1906 of SEQ ID NO:1 could be added to a peptide described herein to the ACP and\/or ST peptide, respectively. Similarly, the amino acids from 1593 to any one of positions 1594, 1595, 1596, 1597, 1598, 1599, 1600, 1601, 1602, 1607, or 1612 of SEQ ID NO: 1 could be added to a peptide described herein if the addition of 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, or 20 amino acids, respectively, to the C-terminus of the ACP peptide is desired. The amino acids from 1906 to any one of positions 1607, 1908, 1909, 1910, 1911, 1912, 1913, 1914, 1915, 1920, or 1925 of SEQ ID NO: 1 could be added to a peptide described herein if the addition of 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, or 20 amino acids, respectively, to the C-terminus of the ST peptide is desired. Since the CurM protein (SEQ ID NO: 1) ends with the TE peptide, non-native amino acids can be added to the TE peptide C-terminus, e.g., 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 16, or 20 amino acids.","In some embodiments, the addition of amino acids to both the N- and C-termini of a peptide described herein is contemplated.","The term \u201cpolypeptide\u201d as used herein, refers to amino acids joined to each other by peptide bonds or modified peptide bonds, i.e., peptide isosteres, and may contain modified amino acids other than the 20 gene-encoded amino acids. The polypeptides may be modified by either natural processes, such as post-translational processing, or by chemical modification techniques which are well known in the art. Modifications can occur anywhere in the polypeptide, including the peptide backbone, the amino acid side-chains and the amino or carboxyl termini. It will be appreciated that the same type of modification may be present in the same or varying degrees at several sites in a given polypeptide. Also a given polypeptide may have many types of modifications. Modifications can include, but are not limited to, acetylation, acylation, ADP-ribosylation, amidation, covalent attachment of flavin, covalent attachment of a heme moiety, covalent attachment of a nucleotide or nucleotide derivative, covalent attachment of a lipid or lipid derivative, covalent attachment of a phosphytidylinositol, cross-linking cyclization, disulfide bond formation, demethylation, formation of covalent cross-links, formation of cysteine, formation of pyroglutamate, formylation, gamma-carboxylation, glycosylation, GPI anchor formation, hydroxylation, iodination, methylation, myristolyation, oxidation, pergylation, proteolytic processing, phosphorylation, prenylation, racemization, selenoylation, sulfation, and\/or transfer-RNA mediated addition of amino acids to protein such as arginylation. (See Proteins\u2014Structure and Molecular Properties 2nd Ed., T. E. Creighton, W. H. Freeman and Company, New York (1993); Posttranslational Covalent Modification of Proteins, B. C. Johnson, Ed., Academic Press, New York, pp. 1-12 (1983)).","As used herein, the term \u201cisolated\u201d means that the material is removed from its original environment (e.g., the natural environment if it is naturally occurring). For example, a naturally occurring polynucleotide or polypeptide present in a living animal is not isolated, but the same polynucleotide or polypeptide, separated from some or all of the coexisting materials in the natural system, is isolated. Such polynucleotides could be part of a vector and\/or such polynucleotides or polypeptides could be part of a composition, and still be isolated in that such vector or composition is not part of its natural environment.","As used herein, the term \u201cpurified\u201d does not require absolute purity; rather, it is intended as a relative definition. Individual nucleic acids obtained from a library have been conventionally purified to electrophoretic homogeneity. The sequences obtained from these clones could not be obtained directly either from the library or from total human DNA. The purified nucleic acids of the invention have been purified from the remainder of the genomic DNA in the organism by at least 10-10fold. However, the term \u201cpurified\u201d also includes nucleic acids that have been purified from the remainder of the genomic DNA or from other sequences in a library or other environment by at least one order of magnitude, typically two or three orders, and more typically four or five orders of magnitude.","As used herein, the term \u201crecombinant\u201d means that the nucleic acid is adjacent to \u201cbackbone\u201d nucleic acid to which it is not adjacent in its natural environment. Additionally, to be \u201cenriched\u201d the nucleic acids will represent 5% or more of the number of nucleic acid inserts in a population of nucleic acid backbone molecules. Backbone molecules according to the invention include nucleic acids such as expression vectors, self-replicating nucleic acids, viruses, integrating nucleic acids, and other vectors or nucleic acids used to maintain or manipulate a nucleic acid insert of interest. Typically, the enriched nucleic acids represent 15% or more of the number of nucleic acid inserts in the population of recombinant backbone molecules. More typically, the enriched nucleic acids represent 50% or more of the number of nucleic acid inserts in the population of recombinant backbone molecules. In a one embodiment, the enriched nucleic acids represent 90% or more of the number of nucleic acid inserts in the population of recombinant backbone molecules.","\u201cRecombinant\u201d polypeptides or proteins refer to polypeptides or proteins produced by recombinant DNA techniques; i.e., produced from cells transformed by an exogenous DNA construct encoding the desired polypeptide or protein or produced from a DNA construct by enzymes (transcribed by an RNA polymerase and translated by ribosomes, tRNAs and accessory proteins outside a cell). \u201cSynthetic\u201d polypeptides or protein are those prepared by chemical synthesis. Solid-phase chemical peptide synthesis methods can also be used to synthesize the polypeptide or fragments of the invention. Such method have been known in the art since the early 1960's (Merrifield, R. B., J. Am. Chem. Soc., 85:2149-2154, 1963) (See also Stewart, J. M. and Young, J. D., Solid Phase Peptide Synthesis, 2 ed., Pierce Chemical Co., Rockford, Ill., pp. 11-12)) and have recently been employed in commercially available laboratory peptide design and synthesis kits (Cambridge Research Biochemicals). Such commercially available laboratory kits have generally utilized the teachings of H. M. Geysen et al, Proc. Natl. Acad. Sci., USA, 81:3998 (1984) and provide for synthesizing peptides upon the tips of a multitude of \u201crods\u201d or \u201cpins\u201d all of which are connected to a single plate. When such a system is utilized, a plate of rods or pins is inverted and inserted into a second plate of corresponding wells or reservoirs, which contain solutions for attaching or anchoring an appropriate amino acid to the pin's or rod's tips. By repeating such a process step, i.e., inverting and inserting the rod's and pin's tips into appropriate solutions, amino acids are built into desired peptides. In addition, a number of available FMOC peptide synthesis systems are available. For example, assembly of a polypeptide or fragment can be carried out on a solid support using an Applied Biosystems, Inc. Model 431A automated peptide synthesizer. Such equipment provides ready access to the peptides of the invention, either by direct synthesis or by synthesis of a series of fragments that can be coupled using other known techniques.","A promoter sequence is \u201coperably linked to\u201d a coding sequence when the RNA polymerase that initiates transcription at the promoter will transcribe the coding sequence into mRNA.","\u201cPlasmids\u201d are designated by a lower case p preceded and\/or followed by capital letters and\/or numbers. The starting plasmids herein are either commercially available, publicly available on an unrestricted basis, or can be constructed from available plasmids in accord with published procedures. In addition, equivalent plasmids to those described herein are known in the art and will be apparent to the ordinarily skilled artisan.","\u201cDigestion\u201d of DNA refers to catalytic cleavage of the DNA with a restriction enzyme that acts only at certain sequences in the DNA. The various restriction enzymes used herein are commercially available and their reaction conditions, cofactors and other requirements were used as would be known to the ordinarily skilled artisan. For analytical purposes, typically 1 g of plasmid or DNA fragment is used with about 2 units of enzyme in about 201 of buffer solution. For the purpose of isolating DNA fragments for plasmid construction, typically 5 to 50 g of DNA are digested with 20 to 250 units of enzyme in a larger volume. Appropriate buffers and substrate amounts for particular restriction enzymes are specified by the manufacturer. Incubation times of about 1 hour at 37 C are ordinarily used, but may vary in accordance with the supplier's instructions. After digestion the gel electrophoresis may be performed to isolate the desired fragment.","\u201cOligonucleotide\u201d refers to either a single stranded polydeoxynucleotide or two complementary polydeoxynucleotide strands which may be chemically synthesized. Such synthetic oligonucleotides have no 5\u2032 phosphate and thus will not ligate to another oligonucleotide without adding a phosphate with an ATP in the presence of a kinase. A synthetic oligonucleotide will ligate to a fragment that has not been dephosphorylated.","The phrase \u201csubstantially identical\u201d in the context of two nucleic acid sequences or polypeptides, refers to two or more sequences that have greater than 60%, greater than 70%, greater than 80%, greater than 90%, greater than 91%, greater than 92%, greater than 93%, greater than 94%, greater than 95%, greater than 96%, greater than 97%, greater than 98%, or greater than 99% nucleotide or amino acid residue identity, when compared and aligned for maximum correspondence, as measured using one of the known sequence comparison algorithms or by visual inspection. The substantial identity can exists over a region of at least about 100 residues, and in some cases, the sequences are substantially identical over at least about 150-200 residues. In some embodiments, the sequences are substantially identical over the entire length of the nucleotide or polypeptide.","Additionally a \u201csubstantially identical\u201d amino acid sequence is a sequence that differs from a reference sequence by one or more conservative or non-conservative amino acid substitutions, deletions, or insertions, particularly when such a substitution occurs at a site that is not the active site of the molecule, and provided that the polypeptide essentially retains its functional properties. A conservative amino acid substitution, for example, substitutes one amino acid for another of the same class (e.g., substitution of one hydrophobic amino acid, such as isoleucine, valine, leucine, or methionine, for another, or substitution of one polar amino acid for another, such as substitution of arginine for lysine, glutamic acid for aspartic acid or glutamine for asparagine). One or more amino acids can be deleted, for example, from a haloalkane dehalogenase polypeptide, resulting in modification of the structure of the polypeptide, without significantly altering its biological activity. Modified polypeptide sequences of the invention can be assayed for haloalkane dehalogenase biological activity by any number of methods, including contacting the modified polypeptide sequence with an haloalkane dehalogenase substrate and determining whether the modified polypeptide decreases the amount of specific substrate in the assay or increases the bioproducts of the enzymatic reaction of a functional haloalkane dehalogenase polypeptide with the substrate.","\u201cFragments\u201d as used herein are a portion of a naturally occurring or recombinant protein that can exist in at least two different confirmations. Fragments can have the same or substantially the same amino acid sequence as the naturally occurring protein. \u201cSubstantially the same\u201d means that an amino acid sequence is largely, but not entirely, the same, but retains at least one functional activity of the sequence to which it is related. In general two amino acid sequences are \u201csubstantially the same\u201d or \u201csubstantially homologous\u201d if they are greater than about 50%, but more typically greater than about 70%, greater than about 85%, or greater than about 90% identical. Fragments that have different three-dimensional structures as the naturally occurring protein are also included. An example of this, is a \u201cpro-form\u201d molecule, such as a low activity proprotein that can be modified by cleavage to produce a mature enzyme with significantly higher activity.","\u201cEnzymatically active fragment\u201d refers to a fragment of, e.g., TE, ST, and ACP, which retains some or all of the enzymatic activity of the full TE, ST, and\/or ACP sequence. The activity of the fragment can be for example, at least 30%, at least 40%, at least 50%, at least 60%, at least 70%, at least 80%, at least 90%, or at least 95% the activity of the original sequence.","CurM","CurM was sequenced and additional parts of the 3\u2032 flanking region of the cur cluster in another cosmid (pLM14), from the genomic DNA library (14), and compared the data with those from a genome sequencing project. It revealed that the 3\u2032 end of the deposited cur gene cluster (5) starting from the middle of curM TE region was indeed chimeric. The revised gene cluster lacks \u201ccurN\u201d, and a complete TE domain is encoded by the 3\u2032 end of curM (5). The adjacent downstream genes show high homology to tRNA 2-selenouridine synthase (ATPase) and adenylate\/guanylate cyclase (Cy), and are not likely involved in curacin biosynthesis (). Thus, curM now appears to mark the 3\u2032-end boundary of the cur, gene cluster, and the ST-TE di-domain presumably functions as the chain termination module.","To biochemically assess the ST-TE mediated chain termination process, CurM ACP, ST and TE were cloned and overexpressed as soluble single domain constructs. ST was eluted as a monomer and TE as a dimer from an analytical size-exclusion column. ACP was overexpressed in the apo form in order to generate ACP-linked substrates. A simplified model substrate, 3-hydroxy-5-methoxytetradecanoyl-CoA (1-CoA, ) was designed to mimic the full-length chain intermediate tethered to CurM ACP (). Two \u03b2-hydroxyl enantiomers, (3S)-1-CoA and (3R)-1-CoA, were synthesized to test the stereoselectivity of the enzymes. The ACP-linked substrates were generated by loading the acyl-CoA substrates to the apo ACP by using the highly flexible Svp phosphopantetheinyltransferase (15).","With the soluble enzymes and model substrates in hand, the key issues in curacin A chain termination were investigated, including, 1) whether the sulfonated carboxylic acid (2, ) will be generated; 2) the timing of the putative ST sulfonation and TE hydrolysis (); 3) whether 2 can lead to the formation of 3 via a decarboxylative elimination mechanism ().","Polyketide Chain Release by TE hydrolysis.","First, it was investigated whether the CurM TE exhibits canonical hydrolysis activity to cleave the thioester bond. (3S)-1-ACP and (3R)-1-ACP were prepared as the TE substrates to test its stereoselectivity for the \u03b2-hydroxyl group. The reactions were analyzed by reverse-phase HPLC, and the separated ACP fractions were examined by Fourier transform ion cyclotron resonance mass spectrometry (FTICR-MS) and infrared multiphoton dissociation (IRMPD) (16) techniques. In addition, the chain-release products were detected by LC-MS and confirmed by co-injection with authentic standards. Both of the acyl groups were found to be hydrolyzed from (3S)-1-ACP and (3R)-1-ACP with low efficiency ( ii, B iii, A ii and A). However, TE-catalyzed hydrolysis of (3S)-1-ACP was about 5-fold faster than with (3R)-1-ACP (). In addition, the steady-state kinetic analysis using the CoA-linked substrates confirmed that (3S)-1 is the preferred substrate for TE (FIG. SB). However, based on bioinformatic analysis of PKS ketoreductase (KR) sequences (17), the CurM KR was predicted to reduce the \u03b2-keto to a (R)-\u03b2-hydroxyl group in the full-length chain intermediate.","On-Assembly-Line Sulfonation by CurM ST.","Based on the known mechanism of ST enzyme function, CurM ST was predicted to bind to PAPS, and transfer a sulfonate moiety to the \u03b2-hydroxyl group of the intermediate tethered to or released from CurM ACP (). First, the ST substrate was identified by testing the sulfonation of 1 and 1-ACP. The sulfonation of 1 was analyzed by LC-MS, and that of 1-ACP was examined by HPLC and FTICR-MS. No substrate conversion was observed for 1 in the presence of ST and PAPS. In contrast, complete conversion of (3R)-1-ACP was determined by HPLC ( iii), and the product with an 80-dalton mass addition was confirmed to be (3R)-2-ACP by FTICR-MS ( ii). Notably, for (3R)-1-ACP substrate, the ST catalyzed sulfonation was about 1000-fold faster than TE hydrolysis. Likewise, the stereoselectivity of ST for the \u03b2-hydroxyl group was determined by comparing the sulfonation efficiency of (3S)-1-ACP and (3R)-1-ACP. Under the same reaction condition, ST sulfonation with (3R)-1-ACP was about 2.5-fold faster than with (3S)-1-ACP (). Thus, CurM ST stereoselectivity is consistent with the predicted formation of a (R)-\u03b2-hydroxyl group by CurM KR. The catalytic efficiency and substrate preference of ST and TE revealed in these experiments provide convincing evidence that ST sulfonation precedes the TE hydrolysis step.","Terminal Olefin Formation via Decarboxylative Elimination in the Cur Pathway.","With the order of the ST and TE reactions established for the acyl-ACP intermediate, the next investigation was to couple the two reactions in one pot. When (3R)-1-ACP was treated with both ST and TE, complete release of the acyl chain from CurM ACP was observed by HPLC ( iv), and confirmed by FTICR-MS ( iii) and IRMPD. Under the same reaction conditions, it was discovered that the CurM TE catalyzed hydrolysis reaction with sulfonated product (3R)-2-ACP was about 800-fold faster than with (3R)-1-ACP. Moreover, TE catalyzed hydrolysis of (3R)-2-ACP was about 120-fold faster than (3S)-2-ACP (). These results further corroborated the timing and stereoselectivity of the ST sulfonation and TE hydrolysis reactions.","Next, experiments were performed to assess whether the proposed sulfonated intermediate (3R)-2 () is released following hydrolysis of (3R)-2-ACP, or whether formation of 3 () occurs directly by a coordinated decarboxylative elimination process. First, the ST-TE coupled reactions with (3R)-1-ACP were analyzed by LC-MS and a single product was confirmed to be (3R)-2 by co-injection with the corresponding authentic standard ( iii). Due to the expected volatility of 3, the contents of the coupled reaction were extracted with hexane and analyzed by GC-MS. A second product was thus detected and confirmed to be 3 by co-injection with the corresponding authentic standard ( iv). It was found that 2 was relatively stable in the reaction mixture, and was not spontaneously converted to 3. To determine which enzyme catalyzes the decarboxylative elimination of 2, authentic (e.g. synthetic) (3R)-2 was separately treated with CurM ST or TE, and a small amount of 3 was observed only for the TE reaction (about 2% conversion in 1 hour). Under the same conditions, the TE reaction with (3R)-2-ACP leading to 3 was more efficient than with (3R)-2 (about 20% conversion in 5 min), suggesting that the decarboxylative elimination is likely coupled with hydrolysis in the TE active site.","Disclosed herein are the biochemical reactions for the natural product biosynthesis of curacin A. A functional ST is inserted into the CurM PKS chain termination module leading to a unique series of on-assembly-line reactions. Specifically, these catalytic events transform the \u03b2-hydroxyl of the penultimate chain elongation intermediate into a \u03b2-sulfate, an excellent leaving group that is positioned chemically to facilitate decarboxylative elimination in the presence of the terminal carboxylate following TE-mediated hydrolysis of the acyl-thioester ().","The significant levels of (3R)-2 (greater than 50%) as a product of the TE reaction from model substrate (3R)-1-ACP () suggests that some interactions between the native substrate and TE active site are involved in efficient coupling of the hydrolysis and decarboxylative elimination reactions. This could reflect a kinetic propensity for the sulfonated acyl-acid product of the model substrate to be released from the CurM TE active site prematurely, thus enabling isolation of (3R)-2. Indeed, given the substrate tolerance of the ST-TE module, this discovery represents a useful new metabolic engineering tool for deliberate incorporation of terminal olefins into high value natural products and other long chain hydrocarbons, including liquid fuels.","Finally, a highly similar strategy of terminal olefin formation occurs in the mevalonate pathway for isoprenoid biosynthesis (18). Specifically, mevalonate-5-diphosphate decarboxylase (MDD), along with mevalonate kinase and mevalonate-5-phosphate kinase catalyze a decarboxylative elimination reaction by first converting a \u03b2-hydroxyl group into a phosphate leaving group (). In contrast, introduction of a terminal olefin as the final step in tautomycetin biosynthesis, a polyketide metabolite isolated from sp. CK4412, is more likely generated by the action of a discrete decarboxylase via yet another unique mechanism (19).","Structural Insights into Terminal Alkene Formation by the Thioesterase in the Curacin A Biosynthetic Pathway","The crystal structure of CurM TE was determined to 1.7 \u212b. CurM TE has the expected \u03b1\/\u03b2 hydrolase fold but differs from other offloading TEs in lid structure and dimer interface position, which results in an open-cleft active site. Comparison with uncharacterized sequences of putative tandem ST-TE domains with presumably similar activity reveals dense conservation within the cleft. A model of the predicted acyl enzyme intermediate shows a conserved Arg205, which may confer specificity to TE for the \u03b2-sulfate, a prediction that is supported by site-directed mutagenesis studies.","Using a simplified analogue, 3-hydroxy-5-methoxytetradecanoyl-ACP, of the penultimate pathway intermediate, it has been recently demonstrated that offloading and terminal bond formation starts with the ST sulfating the \u03b2-hydroxyl group of the intermediate using the sulfate donor PAPS (28) (). The TE acts upon the sulfated intermediate and produces two products: a desulfated and decarboxylated terminal double bond and a hydrolyzed carboxylic acid retaining the \u03b2-sulfate group (). The TE acted very slowly on the non-sulfated substrate. This unprecedented requirement of a \u03b2-sulfate for even hydrolytic catalysis as well as the unique decarboxylation and desulfation activity points to a different catalytic strategy and active site than other FAS, PKS, or NRPS TEs.","In order to understand this novel decarboxylation and desulfation activity, CurM TE was crystallized and its three-dimensional structure was determined. Significant differences from the PKS offloading TEs were observed, especially in the lid region. Using information from the structure, and sequence alignments, a prediction of \u03b2-sulfate recognition was developed. Point mutations were made and the activity in these mutants was tested to lend support to the initial prediction, giving a mechanism for the decarboxylation and desulfation activity and specificity towards the sulfated \u03b2-hydroxyl.","Structure Determination","Since the TE is a single domain at the terminus of a larger polypeptide (CurM), the N-terminal boundary of the TE domain was ambiguous. Constructs were made with three different N-termini with the addition of the fusion protein Mocr necessary to obtain a useable yield of soluble protein. The construct starting at amino acid 1929 of CurM yielded crystals with three amino acids (SNA) added to the N-terminus (SEQ ID NO: 21). The structure was solved by SAD phasing using selenomethionyl CurM TE to 2.2 \u212b. A native dataset of the same crystal form was collected to 1.7 \u212b and was used for refinement (Table 1). The CurM TE crystal structure is deposited in the public database for three-dimensional structures of biological macromolecules (Protein Data Bank, PDB, http:\/\/www.rcsb.org) and is available with the accession code 2H7X.",{"@attributes":{"id":"p-0083","num":"0090"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"70pt","align":"center"}}],"thead":{"row":[{"entry":"TABLE 1"},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":[{},"CurM TE (SeMet)","CurM TE (native)"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Diffraction Data",{},{}]},{"entry":["Space group","P2","P2"]},{"entry":["a, b, c (\u212b)","74.1, 86.9, 87.1","74.5, 86.9, 87.6"]},{"entry":["\u03b1, \u03b2, \u03b3 (\u00b0)","90, 90.4, 90","90, 90.8, 90"]},{"entry":["Wavelength (\u212b)","0.97948","1.0332"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"right"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"right"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Resolution(\u212b)","50-2.14","(2.22-2.14)","50-1.68","(1.74-1.68)"]},{"entry":["<I\/\u03c3>","16.3","(5.5)","18.1","(2.1)"]},{"entry":["R","0.113","(0.362)","0.059","(0.384)"]},{"entry":["Completeness","99.5","(99.9)","90.2","(49.8)"]},{"entry":["Average redundancy","6.4","(6.3)","3.4","(2.0)"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"70pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Unique reflections","60,509","127,036"]},{"entry":["Total reflections","784700","808,263"]},{"entry":["Refinement",{},{}]},{"entry":["Data range (\u212b)","47.46-2.14","34.82-1.68"]},{"entry":["No. reflections","57,064","108,716"]},{"entry":["R\/R","0.188\/0.237","0.178\/0.221"]},{"entry":["RMS deviations",{},{}]},{"entry":["Bonds (\u212b)","0.013","0.013"]},{"entry":["Angles (\u00b0)","1.351","1.331"]},{"entry":["B-factors (\u212b)",{},{}]},{"entry":["Protein","17.9","26.8"]},{"entry":["Water","22.5","39.6"]},{"entry":["Ramachandran",{},{}]},{"entry":["allow","99.90%","99.60%"]},{"entry":["outliers","0.10%","0.40%"]},{"entry":["Protein Atoms","8177","8349"]},{"entry":["Water Molecules","455","1119"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"The CurM TE structure adopts the \u03b1\/\u03b2 hydrolase fold with residues 1-128 and 217-283 of SEQ ID NO: 21 comprising the core domain and residues 136-204 comprising the lid (, C). Each of the 4 monomers in the asymmetric unit contains a disordered region in one of the two lid-to-core linking segments 129-140 and 205-216 (monomer A is disordered 131-135, B 205-216, C 130-140 and 207-215, D 134-135 and 212-215). While no single monomer provides a complete view of these loops, superposition of the monomers provides a representation of continuous connection.","Despite its sulfotransferase (ST) catalytic activity, the CurM ST sequence could not be mapped onto any known ST structure due to low sequence identity. The CurM ST was excised from CurM as an individual domain comprising residues 1598-1917 of SEQ ID NO:1 and a three amino acid addition to the N-terminus\u2014SNA (at positions-3, -2, and -1) (SEQ ID NO:19, renumbered as -3-320 for purposes of the description in this paragraph). Single amino acid substitutions of Gln259Ala and Lys260Ala (SEQ ID NO: 20) were engineered to reduce surface entropy and enable crystallization of the monomeric protein, and the 1.6-\u212b crystal structure of the recombinant ST was determined (Table 2). The core of CurM ST, representing only about 60% of the structure, has a fold similar to those of other STs, but the CurM ST has additional loops and helices in unique positions surrounding the core (, B). The structure revealed the detailed binding of adenosine-3\u2032,5\u2032-diphosphate (PAP), the product of the sulfate donor, 3\u2032-phosphoadenosine 5\u2032-phosphosulfate (PAPS). With knowledge of the PAPS\/PAP binding site, the catalytic base was identified as Glu60. Other active site residues, which may be involved in substrate recognition or may assist catalysis, include His62, Lys133 and Thr43 (). The CurM ST crystal structure is deposited in the public database for three-dimensional structures of biological macromolecules (Protein Data Bank, PDB, http:\/\/www.rcsb.org) and is available with the accession code 4 GBM.",{"@attributes":{"id":"p-0086","num":"0093"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 2"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"Diffraction Data"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Space group","P222"]},{"entry":[{},"a, b, c (\u212b)","45.8 \u212b, 67.3 \u212b, 118.0 \u212b"]},{"entry":[{},"\u03b1, \u03b2, \u03b3 (\u00b0)","90, 90, 90"]},{"entry":[{},"Wavelength (\u212b)","1.0332"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"right"}},{"@attributes":{"colname":"4","colwidth":"70pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Data range(\u212b)","50-1.62","(1.68-1.62)"]},{"entry":[{},"Avg. I\/\u03c3","17.5","(4.9)"]},{"entry":[{},"R","0.074","(0.375)"]},{"entry":[{},"Completeness (%)","98.5","(89.6)"]},{"entry":[{},"Average redundancy","6.6","(5.9)"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"133pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Unique reflections","46,830"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"Refinement"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"133pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Data range (\u212b)","36.34-1.62"]},{"entry":[{},"No. reflections","42,456"]},{"entry":[{},"R\/R","0.185\/0.205"]},{"entry":[{},"RMS deviations",{}]},{"entry":[{},"Bonds (\u212b)","0.012"]},{"entry":[{},"Angles (\u00b0)","1.416"]},{"entry":[{},"B-factors (\u212b)",{}]},{"entry":[{},"Protein","15.5"]},{"entry":[{},"Water","24.7"]},{"entry":[{},"Ramachandran",{}]},{"entry":[{},"Allowed","100.0%"]},{"entry":[{},"Outliers","0.0%"]},{"entry":[{},"Protein atoms (#)","2281"]},{"entry":[{},"Water sites (#)","187"]},{"entry":[{},"Ligands & ions (#)","15"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}},"br":{}},"CurM TE has similar secondary structure to other PKS offloading TEs in the core, but significant differences in secondary structure arrangement exist in the lid region (). CurM TE lacks the two N-terminal dimerization helices present in the lid of PKS TEIs, which are responsible for their lid-to-lid dimerization. In the CurM TE crystal structure, this lid-to-lid dimerization is not present. However, size exclusion chromatography data indicate that CurM TE is dimeric in solution and a dimer interface different that the PKS TEI interface is observed in the crystal structure (; A; B). The CurM TE dimer interface has an average surface area of 1220 \u212b. It is a side-to-side dimer with a protruding helix in the lid making extensive contacts with the core of its dimeric partner. This results in a dimer that serves to hold the lid in an open conformation. When active site accessibility is compared to Pik TE, drastic differences are seen. A narrow tunnel in Pik TE contrasts an exposed cleft in CurM TE (; B). In both instances the specific dimer interface serves to create the environment around the active site.","Despite these differences, the position of the catalytic triad in the active site is well conserved compared to the PKS TEIs, but with the replacement of the Asp from the PKS TEIs with Glu in CurM TE (). There are no other residues near the active site of CurM TE that appear to be assisting with catalysis. Using knowledge of the serine protease catalytic cycle, a catalytic scheme was predicted, which would produce the observed products (Scheme 1). In the scheme, CurM TE reacts in the same way as an offloading TE, up to the acyl-enzyme intermediate. At this point, instead of break down of the acyl-enzyme simply by hydrolysis, hydrolysis would be followed by an enzyme-assisted concerted decarboxylation and desulfation producing the terminal alkene (Scheme)). However, this scheme alone is not able to explain why CurM TE will perform the decarboxylation and desulfation instead of just hydrolysis or explain the selectivity of CurM TE for the sulfated \u03b2-hydroxyl. Presumably, other residues in CurM TE provide specificity and regulation. The structure was examined more closely to assess if any such residues existed.",{"@attributes":{"id":"p-0089","num":"0096"},"chemistry":{"@attributes":{"id":"CHEM-US-00004","num":"00004"},"img":{"@attributes":{"id":"EMI-C00004","he":"225.64mm","wi":"71.88mm","file":"US08765431-20140701-C00004.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}},"The predicted sulfated tetrahedral intermediate was modeled into the crystal structure using knowledge of nucleophile-His-acid active site catalysis and geometry and the affinity label from Pik TE (25) as a guide (). This shows Arg205 in a position to recognize the \u03b2-sulfate group of the modeled intermediate, suggesting it could be responsible for pushing catalysis toward decarboxylation and desulfation after hydrolysis (Scheme 1). Additionally, Arg205 can provide specificity towards sulfated substrates by helping to position the substrate for catalysis in the large open cleft, whereas the non-sulfated substrate will have no such extra guidance.","Comparison with Conserved ACP-ST-TE Sequences","When blasting the ACP-ST-TE amino acid sequence from CurM into the NCBI protein database, five other sequences were identified with 51-33% identity with CurM ACP-ST-TE (). These sequences are unstudied protein products from bacterial genome sequences. The high degree of conservation in all three domains led to an assumption that those gene products catalyze a similar decarboxylation and desulfation reaction. Subsequently, investigation of conserved residues within these sequences may illuminate residues important for this function.","Aligning these five sequences with CurM results in 51%-32% identity (). When mapped onto the CurM TE structure, a dense area of conservation appears in the active site cleft (). The protruding helix (\u03b14) participating in the dimer interface notably does not have invariant residues but has conservation of hydrophobic character, indicating all of the gene products should have the same dimer interface as CurM TE. The conservation in the cleft above the active site (in the opposite direction of the modeled intermediate) may serve as a docking and recognition site for ACP and Ppant arm. The conservation around the modeled intermediate could be playing a more direct role in catalysis especially R205, the residue in position to recognize the modeled sulfate. This residue is conserved in all but one sequence, , which has two other arginines in proximity, which could serve this role. To elucidate the role of R205 and other conserved residues along the active site cleft, a series of point mutations were made.","Activity of Point Mutants","The activity of these mutants was tested using the same assay described previously (28). A one-pot reaction was used where CurM ACP was loaded with a synthesized substrate mimic () using Svp, after which CurM ST sulfates the \u03b2-hydroxyl. Lastly, CurM TE was added and the reaction is quenched after 1 minute. The liberation of holo ACP was detected by a shift on the HPLC. The ratio of holo ACP to ACP with the sulfated substrate was measured. Mutant activity was compared to the activity of wild type CurM TE and free hydrolysis without enzyme in order to obtain percent activity for the mutants (Table 3). All mutants showed reduced or no activity compared to wild type. A control mutation in the catalytic triad, H266R, showed significantly reduced (1% of WT) activity, as expected. Two conserved Asn mutants, one next to the catalytic His (N267A) and one interacting with R205 (N211A) had catalytic activity, but at a reduced rate. Notably, all R205 mutants had very reduced (1-6% of wild type) activity. When tested with the non-sulfated substrate, neither the wild type nor any of the mutants showed any detectable reaction except for the non-physiological S-\u03b2-hydroxy-substrate with the wild type CurM TE.",{"@attributes":{"id":"p-0094","num":"0101"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"140pt","align":"center"}}],"thead":{"row":[{"entry":"TABLE 3"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":[{},"% Activity"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"140pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"WT","100"]},{"entry":[{},"R205Q","1.8 \u00b1 0.3"]},{"entry":[{},"R205E","2.3 \u00b1 2.1"]},{"entry":[{},"R205A","6.1 \u00b1 3.1"]},{"entry":[{},"N267A","38.4 \u00b1 3.1\u2002"]},{"entry":[{},"N211A","54.7 \u00b1 1.8\u2002"]},{"entry":[{},"H266R","1.4 \u00b1 0.2"]},{"entry":[{},"NO TE","\u20030"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"These studies result in an overall scheme of double bond formation where the \u03b2-sulfate group interacts with Arg205 to bind and react with CurM TE. It is important to note that the wild type and mutants do not react with the non-sulfated substrate, indicating CurM TE is selective for a \u03b2-sulfate group. This selection could come from the \u03b2-sulfate acting as a \u201chandle\u201d which R205 can use to position the substrate for catalysis. The open active site may not have sufficient specificity to allow for non-sulfated substrates to be positioned for catalysis, resulting in the observed inactivity. R205 could also plays a role in driving the concerted release of COand SOinstead of the production of a carboxylic acid as would be seen in a canonical PKS TE I. The production of the sulfated carboxylic acid as well as the terminal alkene product were detected (28), indicating that in the system tested, at least part of the catalysis proceeds via the carboxylic acid. The analogous carboxylate and sulfated curacin product was never detected as being produced from , on the experimental detection of the carboxylated and sulfated product may be an off-pathway result of the experimental conditions.","CurM TE represents a new branch of the thioesterase family, optimized to work in concert with a ST to create a terminal double bond. There are already five other cases in the protein database where this type of TE appears. It may be that other organisms that have been found to produce hydrocarbons with terminal alkenes, such as (44) may use this same ST-TE offloading strategy. This ability to create a terminal double bond could have applications in introducing diversity into natural products through combinatorial biosynthesis of FAS and PKS pathways, or producing hydrocarbons with terminal alkenes for possible use as a biofuel.","CurM TE has been found to have an intact catalytic triad active site, which is much more open compared to other studied PKS TEs. The unique lid arrangement and dimer interface facilitates the open-cleft surrounding the active site. High sequence conservation within the active site cleft point to its importance in CurM TE specificity and activity. Specificity is invoked through R205 guiding the \u03b2-sulfated substrate into position for catalysis in a cleft that is excessively for correct positioning of non-sulfated substrates.","Recombinant Production of Proteins","DNA encoding a polypeptide disclosed herein may be isolated and sequenced from a host cell secreting the protein using conventional procedures. Sequence determination will generally require isolation of at least a portion of the gene or cDNA of interest. Usually this requires cloning the DNA or, preferably, mRNA (i.e., cDNA) encoding the polypeptide. Cloning is carried out using standard techniques (see, e.g., Sambrook et al. (1989) , Vols 1-3, Cold Spring Harbor Press, which is incorporated herein by reference). For example, a cDNA library may be constructed by reverse transcription of polyA+ mRNA, preferably membrane-associated mRNA, and the library screened using probes specific for human immunoglobulin polypeptide gene sequences. Nucleotide probe reactions and other nucleotide hybridization reactions are carried out at conditions enabling the identification of polynucleotides that hybridize to each other under specified conditions.","One exemplary set of conditions is as follows: stringent hybridization at 42\u00b0 C. in 50% formamide, 5\u00d7SSC, 20 mM Na.PO4, pH 6.8; and washing in 1\u00d7SSC at 55\u00b0 C. for 30 minutes. Formulae for calculating equivalent hybridization conditions and\/or selecting other conditions to achieve a desired level of stringency are well known. It is understood in the art that conditions of equivalent stringency can be achieved through variation of temperature and buffer, or salt concentration as described Ausubel, et al. (Eds.), , John Wiley & Sons (1994), pp. 6.0.3 to 6.4.10. Modifications in hybridization conditions can be empirically determined or precisely calculated based on the length and the percentage of guanosine\/cytosine (GC) base pairing of the probe. The hybridization conditions can be calculated as described in Sambrook, et al., (Eds.), , Cold Spring Harbor Laboratory Press: Cold Spring Harbor, N.Y. (1989), pp. 9.47 to 9.51","In a preferred embodiment, however, the polymerase chain reaction (PCR) is used to amplify cDNAs (or portions of full-length cDNAs) encoding a polypeptide of interest. The amplified sequences can be readily cloned into any suitable vector, e.g., expression vectors, minigene vectors, or phage display vectors. It will be appreciated that the particular method of cloning used is not critical, an long as it is possible to determine the sequence of some portion of the polypeptide of interest. As used herein, an \u201cisolated\u201d nucleic acid molecule or \u201cisolated\u201d nucleic acid sequence is a nucleic acid molecule that is either (1) identified and separated from at least one contaminant nucleic acid molecule with which it is ordinarily associated in the natural source of the nucleic acid or (2) cloned, amplified, tagged, or otherwise distinguished from background nucleic acids such that the sequence of the nucleic acid of interest can be determined, is considered isolated. An isolated nucleic acid molecule is other than in the form or setting in which it is found in nature. Isolated nucleic acid molecules therefore are distinguished from the nucleic acid molecule as it exists in natural cells. However, an isolated nucleic acid molecule includes a nucleic acid molecule contained in cells that ordinarily express the antibody where, for example, the nucleic acid molecule is in a chromosomal location different from that of natural cells.","The sequence of the amplified or cloned nucleic acid is then determined. Typically the sequence encoding the entire polypeptide is determined.","Sequencing can be carried out on clones isolated from any source, such as a single isolate, a cDNA library, or, when PCR is used, after subcloning the amplified sequence or by direct PCR sequencing of the amplified segment. Sequencing is carried out using standard techniques (see, e.g., Sambrook et al. (1989) , Vols 1-3, Cold Spring Harbor Press, and Sanger, F. et al. (1977) Proc. Natl. Acad. Sci. USA 74: 5463-5467, which is incorporated herein by reference).","Once isolated, the DNA may be placed into expression vectors, which are then transfected into host cells such as cells, simian COS cells, human embryonic kidney 293 cells (e.g., 293E cells), Chinese hamster ovary (CHO) cells, or myeloma cells, to obtain the synthesis of the polypeptide of interest in the recombinant host cells. Recombinant production of polypeptides is well known in the art.","Expression control sequences refers to DNA sequences necessary for the expression of an operably linked coding sequence in a particular host organism. The control sequences that are suitable for prokaryotes, for example, include a promoter, optionally an operator sequence, and a ribosome-binding site. Eukaryotic cells are known to utilize promoters, polyadenylation signals, and enhancers.","Nucleic acid is operably linked when it is placed into a functional relationship with another nucleic acid sequence. For example, DNA for a presequence or secretory leader is operably linked to DNA for a polypeptide if it is expressed as a preprotein that participates in the secretion of the polypeptide; a promoter or enhancer is operably linked to a coding sequence if it affects the transcription of the sequence; or a ribosome binding site is operably linked to a coding sequence if it is positioned so as to facilitate translation. Generally, operably linked means that the DNA sequences being linked are contiguous, and, in the case of a secretory leader, contiguous and in reading phase. However, enhancers do not have to be contiguous. Linking is accomplished by ligation at convenient restriction sites. If such sites do not exist, the synthetic oligonucleotide adaptors or linkers are used in accordance with conventional practice.","Cell, cell line, and cell culture are often used interchangeably and all such designations herein include progeny. Transformants and transformed cells, as well as transfectants and transfected cells, include the primary subject cell and cultures derived therefrom without regard for the number of transfers. It is also understood that all progeny may not be precisely identical in DNA content, due to deliberate or inadvertent mutations. Mutant progeny that have the same function or biological activity as screened for in the originally transformed cell are included. Where distinct designations are intended, it will be clear from the context.","In an alternative embodiment, the amino acid sequence of a polypeptide of interest may be determined by direct protein sequencing. Suitable encoding nucleotide sequences can be designed according to a universal codon table.","Amino acid sequence variants of the desired polypeptide may be prepared by introducing appropriate nucleotide changes into the encoding DNA, or by peptide synthesis. Such variants include, for example, deletions from, and\/or insertions into and\/or substitutions of, residues within the amino acid sequences of the polypeptides. Any combination of deletion, insertion, and substitution is made to arrive at the final construct, provided that the final construct possesses the desired characteristics. The amino acid changes also may alter post-translational processes of the polypeptide, such as changing the number or position of glycosylation sites.","Nucleic acid molecules encoding amino acid sequence variants of the polypeptide are prepared by a variety of methods known in the art. These methods include, but are not limited to, isolation from a natural source (in the case of naturally occurring amino acid sequence variants) or preparation by oligonucleotide-mediated (or site-directed) mutagenesis, PCR mutagenesis, and cassette mutagenesis of an earlier prepared variant or a non-variant version of the polypeptide.","The invention also provides isolated nucleic acid encoding polypeptides of the invention, optionally operably linked to control sequences recognized by a host cell, vectors and host cells comprising the nucleic acids, and recombinant techniques for the production of the polypeptides, which may comprise culturing the host cell so that the nucleic acid is expressed and, optionally, recovering the polypeptide from the host cell culture or culture medium.","For recombinant production of the polypeptide, the nucleic acid encoding the polypeptide is isolated and inserted into a replicable vector for further cloning (amplification of the DNA) or for expression. DNA encoding the polypeptide is readily isolated and sequenced using conventional procedures (e.g., by using oligonucleotide probes that are capable of binding specifically to genes encoding the polypeptide). Many vectors are available. The vector components generally include, but are not limited to, one or more of the following: a signal sequence, an origin of replication, one or more selective marker genes, an enhancer element, a promoter, and a transcription termination sequence.","(1) Signal sequence component: The polypeptides of this invention may be produced recombinantly not only directly, but also as a fusion polypeptide with a heterologous polypeptide, which is preferably a signal sequence or other polypeptide having a specific cleavage site at the N-terminus of the mature protein or polypeptide. The signal sequence selected preferably is one that is recognized and processed (i.e., cleaved by a signal peptidase) by the host cell. If prokaryotic host cells do not recognize and process the native polypeptide signal sequence, the signal sequence may be substituted by a signal sequence selected, for example, from the group of the pectate lyase (e.g., pelB) alkaline phosphatase, penicillinase, 1 pp, or heat-stable enterotoxin II leaders. For yeast secretion the native signal sequence may be substituted by, e.g., the yeast invertase leader, a factor leader (including and \u03b1-factor leaders), or acid phosphatase leader, the glucoamylase leader, or the signal described in WO90\/13646. In mammalian cell expression, mammalian signal sequences as well as viral secretory leaders, for example, the herpes simplex gD signal, are available.","The DNA for such precursor region is ligated in reading frame to DNA encoding the polypeptide.","(2) Origin of replication component: Each of expression and cloning vectors contain a nucleic acid sequence that enables the vector to replicate in one or more selected host cells. Generally, in cloning vectors this sequence is one that enables the vector to replicate independently of the host chromosomal DNA, and includes origins of replication or autonomously replicating sequences. Such sequences are well known for a variety of bacteria, yeast, and viruses. The origin of replication from the plasmid pBR322 is suitable for most Gram-negative bacteria, the 2 \u03bcm plasmid origin is suitable for yeast, and various viral origins are useful for cloning vectors in mammalian cells. Generally, the origin of replication component is not needed for mammalian expression vectors (the SV40 origin may typically be used only because it contains the early promoter).","(3) Selective marker component: Expression and cloning vectors may contain a selective gene, also termed a selectable marker. Typical selection genes encode proteins that (a) confer resistance to antibiotics or other toxins, e.g., ampicillin, neomycin, methotrexate, tetracycline, G418, geneticin, histidinol, or mycophenolic acid (b) complement auxotrophic deficiencies, or (c) supply critical nutrients not available from complex media, e.g., the gene encoding D-alanine racemase from ","One example of a selection scheme utilizes a drug to arrest growth of a host cell. Those cells that are successfully transformed with a heterologous gene produce a protein conferring drug resistance and thus survive the selection regimen. Examples of such dominant selection use the drugs methotrexate, neomycin, histidinol, puromycin, mycophenolic acid and hygromycin.","Another example of suitable selectable markers for mammalian cells are those that enable the identification of cells competent to take up the polypeptide-encoding nucleic acid, such as DHFR, thymidine kinase, metallothionein-I and -II, preferably primate metallothionein genes, adenosine deaminase, ornithine decarboxylase, etc.","For example, cells transformed with the DHFR selection gene are first identified by culturing all of the transformants in a culture medium that contains methotrexate (Mtx), a competitive antagonist of DHFR. An appropriate host cell when wild-type DHFR is employed is the Chinese hamster ovary (CHO) cell line deficient in DHFR activity.","Alternatively, host cells (particularly wild-type hosts that contain endogenous DHFR) transformed or co-transformed with DNA sequences encoding the polypeptide of the invention, wild-type DHFR protein, and another selectable marker such as aminoglycoside 3\u2032-phosphotransferase (APH) can be selected by cell growth in medium containing a selection agent for the selectable marker such as an aminoglycoside antibiotic, e.g., kanamycin, neomycin, or G418. See U.S. Pat. No. 4,965,199.","A suitable selection gene for use in yeast is the trp1 gene present in the yeast plasmid YRp7 (Stinchcomb et al., Nature, 282: 39 (1979)). The trp1 gene provides a selection marker for a mutant strain of yeast lacking the ability to grow in tryptophan, for example, ATCC No. 44076 or PEP4-1. Jones, (Genetics 85:12 (1977)). The presence of the trp1 lesion in the yeast host cell genome then provides an effective environment for detecting transformation by growth in the absence of tryptophan. Similarly, Leu2-deficient yeast strains (ATCC 20,622 or 38,626) are complemented by known plasmids bearing the Leu2 gene. Ura3-deficient yeast strains are complemented by plasmids bearing the ura3 gene.","In addition, vectors derived from the 1.6 \u03bcm circular plasmid pKD1 can be used for transformation of yeasts. Alternatively, an expression system for large-scale production of recombinant calf chymosin was reported for Van den Berg, (Bio\/Technology, 8:135 (1990)). Stable multi-copy expression vectors for secretion of mature recombinant human serum albumin by industrial strains of have also been disclosed (Fleer et al, Bio\/Technology, 9:968-975 (1991)).","(4) Promoter component: Expression and cloning vectors usually contain a promoter that is recognized by the host organism and is operably linked to the polypeptide-encoding nucleic acid. Promoters suitable for use with prokaryotic hosts include the arabinose (e.g., araB) promoter phoA promoter, \u03b2-lactamase and lactose promoter systems, alkaline phosphatase, a tryptophan (trp) promoter system, and hybrid promoters such as the tac promoter. However, other known bacterial promoters are suitable. Promoters for use in bacterial systems also will contain a Shine-Dalgarno (S.D.) sequence operably linked to the DNA encoding the polypeptide of the invention.","Promoter sequences are known for eukaryotes. Virtually all eukaryotic genes have an AT-rich region located approximately 25 to 30 bases upstream from the site where transcription is initiated. Another sequence found 70 to 80 bases upstream from the start of transcription of many genes is a CNCAAT region where N may be any nucleotide. At the 3\u2032 end of most eukaryotic genes is an AATAAA sequence that may be the signal for addition of the poly A tail to the 3\u2032 end of the coding sequence. All of these sequences are suitably inserted into eukaryotic expression vectors.","Examples of suitable promoting sequences for use with yeast hosts include the promoters for 3-phosphoglycerate kinase or other glycolytic enzymes, such as enolase, glyceraldehyde-3-phosphate dehydrogenase, hexokinase, pyruvate decarboxylase, phosphofructokinase, glucose-6-phosphate isomerase, 3-phosphoglycerate mutase, pyruvate kinase, triosephosphate isomerase, phosphoglucose isomerase, and glucokinase.","Other yeast promoters, which are inducible promoters having the additional advantage of transcription controlled by growth conditions, are the promoter regions for alcohol dehydrogenase 2, isocytochrome C, acid phosphatase, degradative enzymes associated with nitrogen metabolism, metallothionein, glyceraldehyde-3-phosphate dehydrogenase, and enzymes responsible for maltose and galactose utilization. Suitable vectors and promoters for use in yeast expression are further described in EP 73,657. Yeast enhancers also are advantageously used with yeast promoters.","Polypeptide transcription from vectors in mammalian host cells is controlled, for example, by promoters obtained from the genomes of viruses such as Abelson leukemia virus, polyoma virus, fowlpox virus, adenovirus (such as Adenovirus 2), bovine papilloma virus, avian sarcoma virus, most preferably cytomegalovirus, a retrovirus, hepatitis-B virus, Simian Virus 40 (SV40), from heterologous mammalian promoters, e.g., the actin promoter or an immunoglobulin promoter, from heat-shock promoters, provided such promoters are compatible with the host cell systems.","The early and late promoters of the SV40 virus are conveniently obtained as an SV40 restriction fragment that also contains the SV40 viral origin of replication. The immediate early promoter of the human cytomegalovirus is conveniently obtained as a HindIII E restriction fragment. A system for expressing DNA in mammalian hosts using the bovine papilloma virus as a vector is disclosed in U.S. Pat. No. 4,419,446. A modification of this system is described in U.S. Pat. No. 4,601,978. See also Reyes et al., Nature 297: 598-601 (1982) on expression of human \u03b2-interferon cDNA in mouse cells under the control of a thymidine kinase promoter from herpes simplex virus. Alternatively, the Rous sarcoma virus long terminal repeat can be used as the promoter.","(5) Enhancer element component: Transcription of a DNA encoding the polypeptide of this invention by higher eukaryotes is often increased by inserting an enhancer sequence into the vector. Many enhancer sequences are known from mammalian genes (globin, elastase, albumin, alpha-fetoprotein, and insulin). Typically, however, one will use an enhancer from a eukaryotic cell virus. Examples include the SV40 enhancer on the late side of the replication origin (bp 100-270), the cytomegalovirus early promoter enhancer, the polyoma enhancer on the late side of the replication origin, and adenovirus enhancers. See also Yaniv, Nature 297:17-18 (1982) on enhancing elements for activation of eukaryotic promoters. The enhancer may be spliced into the vector at a position 5\u2032 or 3\u2032 to the antibody-encoding sequence, but is preferably located at a site 5\u2032 from the promoter.","(6) Transcription termination component: Expression vectors used in eukaryotic host cells (yeast, fungi, insect, plant, animal, human, or nucleated cells from other multicellular organisms) will also contain sequences necessary for the termination of transcription and for stabilizing the mRNA. Such sequences are commonly available from the 5\u2032 and, occasionally 3\u2032, untranslated regions of eukaryotic or viral DNAs or cDNAs. These regions contain nucleotide segments transcribed as polyadenylated fragments in the untranslated portion of the mRNA encoding antibody. One useful transcription termination component is the bovine growth hormone polyadenylation region. See WO94\/11026 and the expression vector disclosed therein. Another is the mouse immunoglobulin light chain transcription terminator.","(7) Selection and transformation of host cells: Suitable host cells for cloning or expressing the DNA in the vectors herein are the prokaryote, yeast, or higher eukaryote cells described above. Suitable prokaryotes for this purpose include eubacteria, such as Gram-negative or Gram-positive organisms, for example, Enterobacteriaceae such as , e.g., , e.g., , e.g., , and , as well as such as and (e.g., 41 P disclosed in DD 266,710 published Apr. 12, 1989), such as , and . One preferred cloning host is 294 (ATCC 31,446), although other strains such as B, X1776 (ATCC 31,537), and W3110 (ATCC 27,325) are suitable. These examples are illustrative rather than limiting.","In addition to prokaryotes, eukaryotic microbes such as filamentous fungi or yeast are suitable cloning or expression hosts for antibody-encoding vectors. , or common baker's yeast, is the most commonly used among lower eukaryotic host microorganisms. However, a number of other genera, species, and strains are commonly available and useful herein, such as hosts such as, e.g., (ATCC 12,424), (ATCC 16,045), (ATCC 24,178), (ATCC 56,500), (ATCC 36,906), , and (EP 402,226); (EP 183,070); (EP 244,234); such as ; and filamentous fungi such as, e.g., , and hosts such as and ","Suitable host cells for the expression of polypeptides are derived from multicellular organisms. Examples of invertebrate cells include plant and insect cells. Numerous baculoviral strains and variants and corresponding permissive insect host cells from hosts such as (caterpillar), (mosquito), (mosquito), (fruitfly), and have been identified. A variety of viral strains for transfection are publicly available, e.g., the L-1 variant of NPV and the Bm-5 strain of NPV, and such viruses may be used as the virus herein according to the present invention, particularly for transfection of cells.","Plant cell cultures of cotton, corn, potato, soybean, petunia, tomato, tobacco, lemna, and other plant cells can also be utilized as hosts.","Examples of useful mammalian host cell lines are Chinese hamster ovary cells, including CHOK1 cells (ATCC CCL61), DXB-11, DG-44, and Chinese hamster ovary cells\/-DHFR (CHO, Urlaub et al., Proc. Natl. Acad. Sci. USA 77: 4216 (1980)); monkey kidney CV1 line transformed by SV40 (COS-7, ATCC CRL 1651); human embryonic kidney line (293 or 293 cells subcloned for growth in suspension culture, (Graham et al., J. Gen Virol. 36: 59, 1977); baby hamster kidney cells (BHK, ATCC CCL 10); mouse sertoli cells (TM4, Mather, (Biol. Reprod. 23: 243-251, 1980); monkey kidney cells (CV1 ATCC CCL 70); African green monkey kidney cells (VERO-76, ATCC CRL-1587); human cervical carcinoma cells (HELA, ATCC CCL 2); canine kidney cells (MDCK, ATCC CCL 34); buffalo rat liver cells (BRL 3A, ATCC CRL 1442); human lung cells (W138, ATCC CCL 75); human liver cells (Hep G2, HB 8065); mouse mammary tumor (MMT 060562, ATCC CCL51); TRI cells (Mather et al., Annals N.Y Acad. Sci. 383: 44-68 (1982)); MRC 5 cells; FS4 cells; and a human hepatoma line (Hep G2).","Host cells are transformed or transfected with the above-described expression or cloning vectors for polypeptide production and cultured in conventional nutrient media modified as appropriate for inducing promoters, selecting transformants, or amplifying the genes encoding the desired sequences. In addition, novel vectors and transfected cell lines with multiple copies of transcription units separated by a selective marker are particularly useful and preferred for the expression of polypeptides.","(8) Culturing the host cells: The host cells used to produce the polypeptides of this invention may be cultured in media suitable for promoting growth in the cell expression system utilized. Yeast and baceterial cells may be expressed using media well-known in the art, such as defined media, undefined media or dropout media (e.g., lacking certain amino acids or sugars for selection of cells) as appropriate for the expression system used. Commercially available media such as Ham's F10 (Sigma), Minimal Essential Medium ((MEM), (Sigma), RPMI-1640 (Sigma), and Dulbecco's Modified Eagle's Medium ((DMEM), Sigma) are suitable for culturing mammalian host cells. In addition, any of the media described in Ham et al., (Meth. Enz. 58: 44, 1979), Barnes et al., Anal. Biochem. 102: 255 (1980), U.S. Pat. Nos. 4,767,704; 4,657,866; 4,927,762; 4,560,655; or 5,122,469; WO90103430; WO 87\/00195; or U.S. Pat. Re. No. 30,985 may be used as culture media for the host cells. Any of these media may be supplemented as necessary with hormones and\/or other growth factors (such as insulin, transferrin, or epidermal growth factor), salts (such as sodium chloride, calcium, magnesium, and phosphate), buffers (such as HEPES), nucleotides (such as adenosine and thymidine), antibiotics (such as GENTAMYCIN\u2122 drug), trace elements (defined as inorganic compounds usually present at final concentrations in the micromolar range), and glucose or an equivalent energy source. Any other necessary supplements may also be included at appropriate concentrations that would be known to those skilled in the art. The culture conditions, such as temperature, pH, and the like, are those previously used with the host cell selected for expression, and will be apparent to the ordinarily skilled artisan.","(9) Purification of polypeptides: When using recombinant techniques, the polypeptide can be produced intracellularly, in the periplasmic space, or directly secreted into the medium, including from microbial cultures. If the polypeptide is produced intracellularly, as a first step, the particulate debris, either host cells or lysed fragments, is removed, for example, by centrifugation or ultrafiltration. Better et al. (Science 240:1041-43, 1988; ICSU Short Reports 10:105 (1990); and Proc. Natl. Acad. Sci. USA 90:457-461 (1993) describe a procedure for isolating antibodies which are secreted to the periplasmic space of . [See also, (Carter et al., Bio\/Technology 10:163-167 (1992)].","The polypeptide composition prepared from microbial or mammalian cells can be purified using, for example, hydroxylapatite chromatography cation or anion exchange chromatography, and affinity chromatography, with affinity chromatography being a preferred purification technique. Mechanically stable matrices such as controlled pore glass or poly(styrenedivinyl)benzene allow for faster flow rates and shorter processing times than can be achieved with agarose. Other techniques for protein purification such as fractionation on an ion-exchange column, ethanol precipitation, Reverse Phase HPLC, chromatography on silica, chromatography on heparin SEPHAROSE\u00ae chromatography on an anion or cation exchange resin (such as a polyaspartic acid column), chromatofocusing, SDS-PAGE, ammonium sulfate precipitation and immunoaffinity are also available depending on the polypeptide to be recovered.","Antibodies","In one embodiment, the invention contemplates an antibody that is specifically reactive with the polypeptides described herein. As used herein \u201cantibody\u201d refers to an antibody or fragment thereof, or a polypeptide comprising an antigen binding domain of an antibody. Exemplary antibodies or antibody fragments include polyclonal antibodies, monoclonal antibodies, chimeric antibodies, humanized antibodies, human antibodies, multispecific antibodies, Fab, Fab\u2032, F(ab\u2032)2, Fv, domain antibody (dAb), complementarity determining region (CDR) fragments, CDR-grafted antibodies, single-chain antibodies (scFv), single chain antibody fragments, chimeric antibodies, diabodies, triabodies, tetrabodies, minibody, linear antibody; chelating recombinant antibody, a tribody or bibody, an intrabody, a nanobody, a small modular immunopharmaceutical (SMIP), a antigen-binding-domain immunoglobulin fusion protein, a camelized antibody, a VHH containing antibody, or a variant or a derivative thereof, and polypeptides that contain at least a portion of an immunoglobulin that is sufficient to confer specific antigen binding to the polypeptide, such as one, two, three, four, five or six CDR sequences.","As used herein, an antibody that \u201cspecifically binds\u201d is \u201cantigen specific,\u201d is \u201cspecific for\u201d antigen target or is \u201cimmunoreactive\u201d with an antigen refers to an antibody that binds an antigen with greater affinity than other antigens of similar sequence. In one aspect, the antibodies contemplated, or fragments, variants, or derivatives thereof, will bind with a greater affinity to target antigen as compared to its binding affinity to similar antigens derived form other sources, e.g., other species, but antibodies that recognize and bind orthologs of the target are within the scope of the invention.","Immunoglobulins can be assigned to different classes, IgA, IgD, IgE, IgG and IgM, which may be further divided into subclasses or isotypes, e.g. IgG1, IgG2, IgG3, IgG4, IgA1 and IgA2. The subunit structures and three-dimensional configurations of different classes of immunoglobulins are well known. An antibody contemplated herein, if it comprises a constant domain, may be of any of these subclasses or isotypes.","Various procedures known in the art may be used for the production of polyclonal or monoclonal antibodies to the polypeptide of the invention. For the production of antibodies, various host animals (including but not limited to rabbits, mice, rats, hamsters, and the like) are immunized by injection with a polypeptide described herein. Various adjuvants may be used to increase the immunological response, depending on the host species, including but not limited to Freund's (complete and incomplete) adjuvant, mineral gels such as aluminum hydroxide, surface active substances such as lysolecithin, pluronic polyols, polyanions, oil emulsions, keyhole limpet hemocyanins, dinitrophenol, and potentially useful human adjuvants such as BCG (Bacille Calmette-Guerin) and ","Monoclonal antibodies may be made by techniques well-known in the art, including, but not limited to the hybridoma technique originally described by K\u00f6hler et al., 256: 495-497 (1975), and the more recent human B-cell hybridoma technique [Kosbor et al., 4:72 (1983)] and the EBV-hybridoma technique [Cole et al., , Alan R Liss, Inc., pp. 77-96 (1985),] or by recombinant DNA methods (see, e.g., U.S. Pat. No. 4,816,567) (all specifically incorporated herein by reference). The monoclonal antibodies may also be isolated from phage antibody libraries using the techniques described in, for example, Clackson et al., (352:624-628, 1991) and Marks et al., (222:581-597, 1991).","In addition to the production of monoclonal antibodies, techniques developed for the production of \u201cchimeric antibodies,\u201d e.g., the splicing of an antibody gene from one species to antibody genes of another species to obtain a molecule with appropriate antigen specificity and biological activity, can be used [Morrison et al., 81:6851-6855 (1984); Neuberger et al., 312:604-608 (1984); Takeda et al., 314:452-454 (1985)]. Alternatively, techniques described for the production of single-chain antibodies (U.S. Pat. No. 4,946,778) can be adapted to produce antibodies.","Antibody fragments that contain the idiotype of the molecule may be generated by known techniques. For example, such fragments include, but are not limited to, the F(ab\u2032)fragment which may be produced by pepsin digestion; the Fab\u2032 fragments which may be generated by reducing the disulfide bridges of the F(ab\u2032)fragment, and the two Fab fragments which may be generated by treating the antibody molecule with papain and a reducing agent. Additionally, using techniques known in the art to isolate CDRs, compositions comprising CDRs are generated. CDRs are characterized by six polypeptide loops, three loops for each of the heavy or light chain variable regions. The amino acid position in a CDR is defined by Kabat et al., \u201cSequences of Proteins of Immunological Interest,\u201d U.S. Department of Health and Human Services (1983), which is incorporated herein by reference.","Screening assays to determine binding specificity of an antibody for use in the methods of the invention are well known and routinely practiced in the art. For a comprehensive discussion of such assays, see Harlow et al. (Eds), ; Cold Spring Harbor Laboratory; Cold Spring Harbor, N.Y. (1988), Chapter 6. Antibodies for use in the invention can be produced using any method known in the art.","The following examples are provided to illustrate the invention, but are not intended to limit the scope thereof.","Chemical Synthesis",{"@attributes":{"id":"p-0148","num":"0155"},"chemistry":[{"@attributes":{"id":"CHEM-US-00005","num":"00005"},"img":{"@attributes":{"id":"EMI-C00005","he":"180.26mm","wi":"75.86mm","file":"US08765431-20140701-C00005.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},{"@attributes":{"id":"CHEM-US-00006","num":"00006"},"img":{"@attributes":{"id":"EMI-C00006","he":"143.85mm","wi":"75.86mm","file":"US08765431-20140701-C00006.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}}]},{"@attributes":{"id":"p-0149","num":"0156"},"chemistry":{"@attributes":{"id":"CHEM-US-00007","num":"00007"},"img":{"@attributes":{"id":"EMI-C00007","he":"17.95mm","wi":"29.46mm","file":"US08765431-20140701-C00007.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{},"i":["J. Org. Chem. ","Tetrahedron Lett. "],"sub":["2","2 ","3 ","3 ","2","4","2 ","3","3"],"sup":["1","13"],"img":[{"@attributes":{"id":"CUSTOM-CHARACTER-00001","he":"3.13mm","wi":"1.78mm","file":"US08765431-20140701-P00001.TIF","alt":"custom character","img-content":"character","img-format":"tif"}},{"@attributes":{"id":"CUSTOM-CHARACTER-00002","he":"3.13mm","wi":"1.78mm","file":"US08765431-20140701-P00002.TIF","alt":"custom character","img-content":"character","img-format":"tif"}}]},{"@attributes":{"id":"p-0150","num":"0157"},"chemistry":{"@attributes":{"id":"CHEM-US-00008","num":"00008"},"img":{"@attributes":{"id":"EMI-C00008","he":"17.95mm","wi":"30.23mm","file":"US08765431-20140701-C00008.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{},"sub":["4 ","4 ","2","2","4","2 ","3"],"sup":"1","img":{"@attributes":{"id":"CUSTOM-CHARACTER-00003","he":"3.13mm","wi":"1.78mm","file":"US08765431-20140701-P00003.TIF","alt":"custom character","img-content":"character","img-format":"tif"}}},{"@attributes":{"id":"p-0151","num":"0158"},"chemistry":{"@attributes":{"id":"CHEM-US-00009","num":"00009"},"img":{"@attributes":{"id":"EMI-C00009","he":"17.95mm","wi":"38.61mm","file":"US08765431-20140701-C00009.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{},"sub":["2","2 ","2 ","3 ","2","2 ","3","2","4","2 ","3"],"img":{"@attributes":{"id":"CUSTOM-CHARACTER-00004","he":"3.13mm","wi":"1.78mm","file":"US08765431-20140701-P00004.TIF","alt":"custom character","img-content":"character","img-format":"tif"}}},{"@attributes":{"id":"p-0152","num":"0159"},"chemistry":{"@attributes":{"id":"CHEM-US-00010","num":"00010"},"img":{"@attributes":{"id":"EMI-C00010","he":"17.95mm","wi":"39.71mm","file":"US08765431-20140701-C00010.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{},"i":"Nature ","sub":["2","4","2 ","3","3"],"sup":["\u22121","1","13"],"img":[{"@attributes":{"id":"CUSTOM-CHARACTER-00005","he":"3.13mm","wi":"1.78mm","file":"US08765431-20140701-P00005.TIF","alt":"custom character","img-content":"character","img-format":"tif"}},{"@attributes":{"id":"CUSTOM-CHARACTER-00006","he":"3.13mm","wi":"1.78mm","file":"US08765431-20140701-P00006.TIF","alt":"custom character","img-content":"character","img-format":"tif"}}]},{"@attributes":{"id":"p-0153","num":"0160"},"chemistry":{"@attributes":{"id":"CHEM-US-00011","num":"00011"},"img":{"@attributes":{"id":"EMI-C00011","he":"17.95mm","wi":"38.61mm","file":"US08765431-20140701-C00011.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{},"i":"J. Am. Chem. Soc. ","sub":["2","2 ","2 ","3 ","2","2 ","3","2","4","2 "]},{"@attributes":{"id":"p-0154","num":"0161"},"chemistry":{"@attributes":{"id":"CHEM-US-00012","num":"00012"},"img":{"@attributes":{"id":"EMI-C00012","he":"17.95mm","wi":"39.71mm","file":"US08765431-20140701-C00012.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{},"sub":["2","4","3","3"],"sup":["\u22121","1","13","+"],"img":[{"@attributes":{"id":"CUSTOM-CHARACTER-00007","he":"3.13mm","wi":"1.78mm","file":"US08765431-20140701-P00007.TIF","alt":"custom character","img-content":"character","img-format":"tif"}},{"@attributes":{"id":"CUSTOM-CHARACTER-00008","he":"3.13mm","wi":"1.78mm","file":"US08765431-20140701-P00008.TIF","alt":"custom character","img-content":"character","img-format":"tif"}}]},{"@attributes":{"id":"p-0155","num":"0162"},"chemistry":{"@attributes":{"id":"CHEM-US-00013","num":"00013"},"img":{"@attributes":{"id":"EMI-C00013","he":"17.95mm","wi":"39.71mm","file":"US08765431-20140701-C00013.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{},"i":"Tetrahedron Letters ","sub":["2","3 ","2 "],"sup":"\u2212"},"(3R)-1-CoA\/(3S)-1-CoA.","The CoA thioesterification of (3R)-1 and (3S)-1, as we as product purification, was performed in the similar way as previously described Gu, et al., 128(28):9014-9015 (2006) and Geders, et al., 282:35954-35963(2007)). For both (3R)-1-CoA and (3S)-1-CoA, MS (ESI) calculated for [M-H] 1022.32, found 1022.28.","Bioinformatic Analysis for Prediction of CurM KR Stereospecificity","Based on alignments of KR domains, the CurL and CurM KR domains contain the LDD motif, indicating that they catalyze formation B-type hydroxyl groups. Chirality of the carbon atom bearing the methoxyl group in curacin A confirms the prediction for CurL KR stereospecificity.","Bacterial Strains, Media and Culture Conditions",{"@attributes":{"id":"p-0159","num":"0166"},"i":["Escherichia coli ","E. coli ","E. coli "]},"DNA Sequencing of curM and 3\u2032 Flanking Region","To identify cosmids containing the 3\u2032 end of cur gene cluster, the previous cosmid genomic library of strain L19 (Chang et al, 296, 235-247 (2002)) was screened by PCR using an oligonucleotide primer pair to amplify the curM ST gene: (F) 5\u2032-GGA TGC GGA TGC AAA AAC TTG-3\u2032 (SEQ ID NO: 6) and (R) 5\u2032-CGG ATG CAA AAA CTT GTC GGG-3\u2032 (SEQ ID NO: 7). Two cosmids, pLM14 and pLM19, in addition to pLM17, were identified to contain curM ST. These three cosmids were also examined by comparing their restriction enzyme digestion patterns. The pLM14 was chosen to be sequenced by primer walking method for the 3\u2032 end of curM and the flanking region. The new sequence was compared with the genome sequencing data (unpublished), and is SEQ ID NO: 1 (amino acid sequence) and SEQ ID NO: 2 (DNA sequence).","Plasmid Construction","CurM ACP, ST and TE genes were amplified from the pLM14. The CurM ACP and TE genes were inserted into pET24b plasmid at the NdeI and NotI restriction sites. The CurM ST gene was inserted into pET28b plasmid with NdeI and BamHI restriction sites. All the constructs were verified by DNA sequencing. The primers for the plasmid construction are: ACP (F): 5\u2032- ACA GAC GAA CGC ATT TTA G-3\u2032(SEQ ID NO: 8), ACP (R): 5\u2032-T AAG CTT GTT GGA GAT GG-3\u2032(SEQ ID NO: 9), ST (F): 5\u2032- ATC TTT GCA ACC AAA AGT TCA-3\u2032(SEQ ID NO: 10), ST (R): 5\u2032- TTA TTG AGG CTG TTG ATT TGT CG-3\u2032(SEQ ID NO: 16), TE (F): 5\u2032- CAA GTC TCT ACA ACT CCC T-3\u2032(SEQ ID NO: 17), and TE (R): 5\u2032-G GAT GTT AAG ATA AGT GAT GC-3\u2032(SEQ ID NO: 18).","Protein Overexpression",{"@attributes":{"id":"p-0162","num":"0169"},"i":"E. coli "},"Protein Purification","Protein purifications were performed at 4\u00b0 C. cells were harvested by centrifugation (5,000 g, 15 min, 4\u00b0 C.), resuspended in ice cold lysis buffer A (50 mM PBS buffer, pH 8.0, 300 mM NaCl, 10 mM imidazole, 20% glycerol) and disrupted by sonication on ice. The cell debris was removed by centrifugation at 15,000 g for 50 min. The supernatant was gently removed and loaded onto the 5 ml HisTrap column (GE Healthcare pre-equilibrated with lysis buffer A. The resin was washed successively with \u02dc10 column volumes of the washing buffer B (50 mM PBS buffer, pH 8.0, 300 mM NaCl, 20 mM imidazole, 10% glycerol) to remove nonspecifically bound contaminants. Bound proteins were eluted with imidazole by a linear gradient of the elution buffer C (50 mM PBS buffer, pH 8.0, 300 mM NaCl, 250 mM imidazole, 20% glycerol). The eluate fractions were examined by SDS-PAGE for purity, pooled and concentrated using Amicon Ultra-15 (10 kDa or 5 kDa) centrifugal devices (Millipore). The concentrated eluate was loaded onto HiLoad 26\/60 Superdex 200 column (GE Healthcare) equilibrated with the storage buffer D (50 mM PBS buffer, pH 7.5, 200 mM NaCl, 20% glycerol). The fractions were pooled, concentrated, flash-frozen in 50-100 \u03bcl aliquots in liquid N, and stored at \u221280\u00b0 C. for future use. The purity of the proteins was analyzed by SDS-PAGE and the protein concentrations were determined using the Bradford assay (Bio-Rad).","Preparation of the ACP-Linked Substrates","The (3R)-1-ACP and (3S)-1-ACP substrates were prepared by loading the (3R)-1-CoA and (3S)-1-CoA onto the (apo) CurM ACP by using Svp. Briefly, 500 \u03bcM acyl-CoA and 50 \u03bcM (apo) ACP were incubated with 10 \u03bcM Svp, and 10 mM MgClin 50 mM Tris-HCl buffer, pH 8.1, at room temperature for about 2 h. Reaction mixtures were desalted by PD10 column equilibrated with buffer D. The desalted acyl-ACPs were concentrated by using Amicon Ultra-4 (5 kDa, Millipore), flash-frozen in 10-50 \u03bcl aliquots in liquid N, and stored at \u221280\u00b0 C. The ACP samples were analyzed by reverse-phase HPLC using a Jupiter C4 column (250\u00d72.0 mm, 5 \u03bcm, 300 \u212b, Phenomenex), and a linear elution gradient from 5% to 100% of CHCN (0.1% CFCOH)\/HO (0.1% CFCOH).","Kinetic Studies of TE Hydrolysis Using CoA-Linked Substrates","HPLC-based analyses of the TE hydrolysis were performed using XBridge C18 column (4.6\u00d7250 mm, 5 \u03bcm, Waters) on the Gold HPLC system equipped with an autosampler and controlled by 32 Karat software (Beckman Coulter). Samples were eluted with a linear gradient from 10% to 90% of MeOH\/HO (10 mM CHCONH). For steady state kinetic studies. TE hydrolysis was examined in 40 \u03bcl 50 mM Tris-HCl buffer (pH 7.0) with 100 \u03bcM, 200 \u03bcM, 500 \u03bcM, 1000 \u03bcM and 2000 \u03bcM (3R)-1-CoA or (3S)-1-CoA. 1.25 \u03bcM TE was incubated with the CoA substrates at room temperature for 10 min before quenched by 40 \u03bcl 1M CHCOH. The reaction mixtures were then added with isovaleryl-CoA as an internal standard before filtered by Microcon YM-10 (Millipore), neutralized by 20 \u03bcl 1M NaOH, and stored at \u221280\u00b0 C. before HPLC analysis. Control reactions without enzymes were run at the same time. The TE hydrolysis reaction was measured by consumption of CoA substrates. The HPLC peak areas of CoA substrates were normalized based on the internal standards.","ST and TE Assays Using ACP-Linked Substrates","The ST and TE assays were performed using R)-1-ACP and (3S)-1-ACP substrates. Typically, for the ST reactions, about 300 \u03bcM ACP-linked substrate was added with 2 \u03bcM ST and 2 mM PAPS in 50 mM Tris-HCl buffer (pH 7.0). For the TE reactions, about 300 \u03bcM ACP-linked substrate was added with 2 \u03bcM TE in 50 mM Tris-HCl buffer (pH 7.0). All the reactions were incubated at room temperature, quenched by addition of 10% formic acid, and analyzed by reverse-phase HPLC using Jupiter C4 column. The ACP fractions were collected, lyophilized and analyzed by FTICR-MS and IRMPD. To detect products cleaved from ACP by LC-MS, the reaction mixtures were filtered Microcon YM-10 to remove the enzymes. The samples were loaded onto XBridge C18 column (2.1\u00d7150 mm, 3.5 \u03bcm, Waters), and LC-MS analysis was performed on a Surveyor HPLC system equipped with a ESI-LTQ mass spectrometer (Thermo Scientific). For the coupled ST-TE reactions, the products were extracted with 2\u00d72 ml hexane and dried under nitrogen prior to GC\/MS analysis.","GC\/EI-MS Analysis","The samples and authentic standard were analyzed by a 6890N gas chromatograph equipped with a 5973 mass selective quadrupole detector (Agilent). The butylamides were separated on a HP-5MS (Agilent J&W) capillary column (30 m\u00d7250 \u03bcm\u00d70.25 \u03bcm), which was operated with helium-carrier gas and splitless injection. Both the injector and detector temperatures were set as 250\u00b0 C. After an initial setting at 50\u00b0 C., the oven temperature was raised to 300\u00b0 C. at 6\u00b0 C.\/min and held for 20 minutes. Total ion chromatograms were recorded using a mass range of 60-420 amu, and the selective ion chromatograms were recorded by monitoring the two to three most abundant masses plus the parent masses of target compounds.","Analysis of ACP Samples by Electrospray Ionization (ESI)-FTICR-MS","The observed and calculated masses for the ACP samples are listed in Table 4. Preparation of ACP samples for FTICR-MS analysis was performed as previously described in Gu, et al., 318:970-974 (2007) All samples were analyzed with an actively shielded 7 Tesla quadrupole-FTICR mass spectrometer (APEX-Q, Bruker Daltonics). Target analytes in electrospray solution (1:1 CHCN:HO with 0.1% HCOOH) were directly infused into an electrospray ionization (ESI) source (Apollo II, Bruker Daltonics) operating in positive ion mode at a flow rate of 70 \u03bcL\/h and a voltage of \u22123.8 kV. A counterflow of hot (240\u00b0 C.) nitrogen gas was applied to assist desolvation of ESI droplets. Multiply protonated ions generated by ESI were externally accumulated in a hexapole and transferred via high voltage ion optics to the ICR cell for analysis. For IRMPD, precursor ions were mass-selectively accumulated in the hexapole with a 5-10 m\/z quadrupole isolation window, transferred to the ICR cell, and irradiated for 100-200 ms by 10.6 \u03bcm photons at 10 W laser power, (25 W COlaser, Synrad). All data were acquired with XMASS software (version 6.1, Bruker Daltonics) in broadband mode from m\/z=200 to 2000 with 512 k data points and summed over 10-30 scans. Mass spectra were analyzed with the MIDAS analysis software. For accurate mass determination, ubiquitin (Sigma) peaks on charge state of 10-11 (ubiquitin was spiked into the ESI solution prior to analysis) was used as internal calibrants to determine the mass of apo-ACP. Once the exact mass of apo-ACP had been determined, its 11 and 13 charge states were selected as external standards for further calibration (ubiquitin was not spiked into all reactions). All frequency-to-m\/z calibrations were performed with a two-term calibration equation.",{"@attributes":{"id":"p-0170","num":"0177"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"center"}}],"thead":{"row":{"entry":"TABLE 4"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":[{},"ESI-FTICR-MS","IRMPD (PEP)"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Obs.","Calc.","Obs.","Calc."]},{"entry":["ACPsamples","avg mass","avg mass","[M + H]","[M + H]"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["ACP-SH","13786.92","13786.88","261.134","261.127"]},{"entry":["1-ACP","14043.01","14043.08","517.349","517.331"]},{"entry":["2-ACP","14122.99","14123.04","517.349*","517.331*"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":"PEP, Phosphopantetheine ejection product."},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"left","id":"FOO-00002"},"sup":"\u00b6"}},{"entry":"*A same PEP product was observed for 2-ACP and 1-ACP due to laser-induced dissociation of the sulfate group on 2-ACP."}]}}]}},"br":{}},"Inserts for CurM ACP (residues 1514-1592), ST (residues 1598-1917) and TE (residues 1929-2211) were generated by polymerase chain reaction amplification from the cosmid pLM14 (28). ACP and ST were inserted into pMCSG7 (29) and TE into pMoCR (30), containing the fusion Mocr to enhance solubility. All constructs were verified by DNA sequencing. The plasmids were transformed into BL21(DE3) cells and grown at 37\u00b0 C. in 500 mL TB with 4% glycerol in 2 L baffle flasks until an ODof 1.0. Trace metals (50 \u03bcM FeCl, 20 \u03bcM CaCl, 10 \u03bcM MnCl, 10 \u03bcM ZnSO, 2 \u03bcM CoCl, 2 \u03bcM CuCl, 2 \u03bcM NiCl, 2 \u03bcM NaMoO, 2 \u03bcM NaSeO, 2 \u03bcM HBO) were added when growing the ACP. The temperature was lowered to 18\u00b0 C. and IPTG was added to a final concentration of 0.2 mM. The culture grew for an additional 18 hours, the cells were harvested by centrifugation, and frozen at \u221220\u00b0 C. Selenomethionyl (SeMet) protein was produce in BL21(DE3) in SelenoMet\u2122 Medium (AthenaES) containing 100 \u03bcg\/mL seleno-DL-methionine. Site directed mutagenesis was performed using the QuickChange protocol (Stratagene) and confirmed by DNA sequencing.","Protein Purification","Performing all steps at 4\u00b0 C. unless noted, the cell pellet from 500 mL of cell culture was re-suspended in 40 mL Buffer A (20 mM Tris pH 7.9, 500 mM NaCl, 20 mM imidazole, and 10% glycerol). DNase (2 mg), lysozyme (5 mg), and MgCl(4 mM final concentration) were added and incubated for 30 min. The cells were lysed by sonication and the lysate cleared by centrifugation. The supernatant was filtered through 0.45 \u03bcm filters and loaded onto a 5 mL HisTrap Ni NTA resin column (GE Healthcare). The column was washed with 8 column volumes Buffer A. The proteins eluted around 150 mM imidazole by a linear gradient up to 650 mM imidazole (Buffer B). The 6\u00d7 His-Mocr fusion on the TE was removed by incubating the pooled fractions with 1 mM DTT and 2% (w\/w) tobacco etch virus (TEV) protease for two hours at room temperature. The imidazole was removed by dialysis overnight at 4\u00b0 C. in Buffer C (20 mM Tris pH 7.9, 500 mM NaCl, 10% glycerol) with 1 mM DTT. The reaction mixture was loaded again on the HisTrap column and the flow-through fractions were collected and pooled. All proteins were further purified by size exclusion chromatography with a HiLoad 16\/60 Superdex 200 (GE Healthcare) pre-equilibrated with Buffer C. Fractions were pooled and concentrated to 5 mg\/mL, flash frozen in liquid N, and stored at \u221280\u00b0 C. The SeMet derivative of the TE was purified as described above with 2 mM DTT added to all buffers. 500 mL of culture yielded 5 mg of purified TE, 2 mg of SeMet TE, 10 mg of ACP, and 20 mg of ST.","Crystallization","Crystals of CurM TE were grown at 4\u00b0 C. within 24-48 hours in hanging drops using the vapor diffusion method. Protein solution containing 2 mg\/mL protein, 20 mM Tris pH 7.9, 200 mM NaCl and 2.5% glycerol was mixed in equal volumes with well solution containing 27-32% PEG3350, 100 mM Tris pH 8.3-8.5. Micro-seeding from native crystals was required for crystal growth of the SeMet protein in similar conditions. Crystals were transferred into cryo protection solution containing well solution with 15% glycerol, harvested in loops and flash frozen in liquid N.","Data Collection and Structure Determination","Data were collected at GM\/CA-CAT beamline 23ID-D at the Advanced Photon Source (APS) at Argonne National Lab (Argonne, Ill.). Among 25 SeMet TE crystals, only one diffracted beyond 4 \u212b, but had multiple lattices in the diffraction and two distinct crystals. A region visually identified as a single crystal was probed in three 10-\u03bcm steps using a 20-\u03bcm mini-beam (31). The center position was chosen for the best diffraction with the least interference from the second lattice, and data were collected in inverse-beam geometry (\u03c6=0\u00b0-90\u00b0 and 180\u00b0-270\u00b0 as wedges of 45\u00b0 with 1\u00b0 images). The diffraction images showed significant radiation damage, so a different region of the sample was probed in a perpendicular orientation. The crystal was rotate 90\u00b0 from the initial raster and now looking into the loop, visual identification of a region with a single crystal was impossible. Two separate regions in this orientation were rastered in 3 by 3 boxes in steps of 10-\u03bcm with the 10-\u03bcm collimator. From these rasters, a single lattice position was identified where data were collected again in inverse beam geometry (\u03c6=90\u00b0-150\u00b0 and 270\u00b0-330\u00b0 as 30\u00b0 wedges with 0.5\u00b0 images). The two partial datasets were indexed separately resulting in similar unit cell constants and scaled together, all using the HKL2000 suite (32), to yield a complete SAD dataset. The SeMet TE structure was solved using SOLVE\/RESOLVE (33, 34) in the PHENIX software suite. 28 Se sites were found (average figure of merit (FOM)=0.401). After density modification and fourfold noncrystallographic symmetry averaging in RESOLVE the figure of merit was 0.81. AUTOBUILD (35) was used to build an 86% complete initial model, which was completed manually in COOT (36). REFMAC5, from the CCP4 suite, was used for refinement with TLS (37-39).","Sequence Alignment, Structure Alignment and Substrate Modeling","Similar ACP-ST-TE sequences were identified by a BLAST search into the NCBI protein database. ClustalW was used to perform the multiple sequence alignment (40). Pymol was used to align structures and to prepare structure illustrations (41). CurM TE was aligned with affinity labeled PikTE (PDB code 2H7X, RMS=3.309) by superposition of the core domains (residues 55-176 and 232-292 in PikTE to residues 1-126 and 217-282 in CurTE). The PRODRG2 server (42) was used to generate initial atomic coordinates and a topology file for the predicted tetrahedral intermediate. The intermediate was modeled using the affinity label in the active site of PikTE (PDB code 2H7X) (25, 26) as a guide.","Preparation of Substrate-Loaded ACP","The substrate-loaded ACP was prepared by loading 3-hydroxy-5-methoxytetradecanoyl-CoA (28) onto the apo ACP using Svp(43). 50 \u03bcM ACP and 100 \u03bcM 3-hydroxy-5-methoxytetradecanoyl-CoA were incubated with 10 \u03bcM Svp and 10 mM MgClin 100 mM Tris pH 7.9 at 30\u00b0 C. for 2 hours. The reaction was exchanged into Buffer C, concentrated to 550 \u03bcM ACP using Amicon Ultra 10 kDa concentrators (Millipore), flash frozen in 20 \u03bcL aliquots, and stored at \u221280\u00b0 C. Loading efficiency was analyzed by HPLC using the protocol described in the activity assay.","Activity Assay","TE activity was assayed using a modification of the protocol developed by Gu et. al. (28). To generate the sulfated substrate for the TE assay, 225 \u03bcM loaded ACP was incubated with 5 \u03bcM ST, 1.75 mM PAPS (Sigma), in 100 mM Tris pH 7.9 at room temperature for 10 min. Four \u03bcM TE was then added to the mixture and the reaction was quenched with 10% formic acid after 1 min. The samples were analyzed by reverse phase HPLC using a Jupiter C4 column (250\u00d72.0 mm, 5 \u03bcm, 300 \u212b, Phenomenex) and a linear elution gradient from 30% to 90% CHCN (0.1% CFCOH)\/HO (0.1% CFCO2H) over 45 min.","Additional features and variations of the invention will be apparent to those skilled in the art from the entirety of this application, including the detailed description, and all such features are intended as aspects of the invention. Likewise, features of the invention described herein can be re-combined into additional embodiments that also are intended as aspects of the invention, irrespective of whether the combination of features is specifically mentioned above as an aspect or embodiment of the invention. Also, only such limitations which are described herein as critical to the invention should be viewed as such; variations of the invention lacking limitations which have not been described herein as critical are intended as aspects of the invention.",{"@attributes":{"id":"p-0179","num":"0000"},"ul":{"@attributes":{"id":"ul0003","list-style":"none"},"li":["1. Gu, et al., 318, 970-974 (2007).","2. Fischbach et al., 105, 4601-4608 (2008).","3. Gu, et al. 459, 731-735 (2009).","4. Verdier-Pinard P. et al. 53, 62-76 (1998).","5. Chang et al., 67, 1356-1367 (2004).","6. Gu et al., 128, 9014-9015 (2006).","7. Calderone et al., & 14, 835-846 (2007).","8. Calderone et al., 103, 8977-8982 (2006).","9. Walsh 41, 4-10 (2008).","10. Chapman et al Angew -43, 3526-3548 (2004).","11. Walsh C T 303, 1805-1810 (2004).","12. Fischbach et al., 106, 3468-3496 (2006).","13. Khosla et al., 76, 11.11-11.27 (2007).","14. Chang et al., 296, 235-247 (2002).","15. Sanchez et al., & 8, 725-738 (2001).","16. Dorrestein et al., 45, 12756-12766 (2006).","17. Caffrey & 12, 1060-1062 (2005).","18. Bonanno, et al. 98, 12896-12901 (2001).","19. Choi et al., -153, 1095-1102 (2007).","20. Senko et al, 10, 1839-1844 (1996).","21. Ledford et al, 56, 2744-2748 (1984).","22. Geders, et al., 2007. 282(49): p. 35954-63.","23. Akey, et al., 2010. 18(1): p. 94-105.","24. Roongsawang, et al., 2007. 8(5): p. 501-12.","25. Giraldes, et al., 2006. 2(10): p. 531-6.","26. Akey, et al., 2006. 2(10): p. 537-42.","27. Tsai, et al., 2001. 98(26): p. 14808-13.","28. Gu, et al., 2009. 131(44): p. 16033-5.","29. Donnelly, et al., 2006. 47(2): p. 446-54.","30. DelProposto, et al., 2009. 63(1): p. 40-9.","31. Fischetti, et al., 2009. 16(Pt 2): p. 217-25.","32. Otwinowski, -1997. 276(Macromolecular Crystallography, part A,): p. 307-326.","33. Terwilliger, et al., 1999. 55(Pt 4): p. 849-61.","34. Terwilliger, 2000. 56(Pt 8): p. 965-72.","35. Terwilliger, 2003. 59(Pt 1): p. 38-44.","36. Emsley, et al., 2004. 60(Pt 12 Pt 1): P. 2126-32.","37. The CCP4 suite: programs for protein crystallography. 1994. 50(Pt 5): p. 760-3.","38. Murshudov, et al., 1997. 53(Pt 3): p. 240-55.","39. Painter, et al., 2006. 62(Pt 4): p. 439-50.","40. Thompson, et al., 2002. Chapter 2: p. Unit 23.","41. DeLano, . In. 2002, DeLano Scientific: Palo Alto, Calif., USA.","42. Schuttelkopf, et al., 2004. 60(Pt 8): p. 1355-63.","43. Sanchez, et al., 2001. 8(7): p. 725-38.","44. Chan Young, et al., 1986. 10(12): p. 701-707."]}}],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE FIGURES","p":[{"@attributes":{"id":"p-0019","num":"0022"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0020","num":"0023"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0021","num":"0024"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0022","num":"0025"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0023","num":"0026"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0024","num":"0027"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0025","num":"0028"},"figref":"FIG. 7"},{"@attributes":{"id":"p-0026","num":"0029"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0027","num":"0030"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0028","num":"0031"},"figref":"FIG. 10","i":["Pseudomonas entomophila ","Haliangium ochraceum ","Synechococcus ","Cyanothece ","Cymothece "]},{"@attributes":{"id":"p-0029","num":"0032"},"figref":"FIG. 11"},{"@attributes":{"id":"p-0030","num":"0033"},"figref":"FIG. 12","i":["Pseudomonas entomophila ","Haliangium ochraceum ","Synechococcus ","Cyanothece ","Cyanothece "],"sub":["\u2014","\u2014","\u2014","\u2014","\u2014"]},{"@attributes":{"id":"p-0031","num":"0034"},"figref":"FIG. 13"}]},"DETDESC":[{},{}]}
