---
title: Methods and compositions for targeted protein degradation
abstract: Coronatine has been found to enhance binding of the JAZ1 degron to the F-box protein COI1, and analysis of the JAZ1 degron sequence has resulted in the identification of specific peptide sequences that bind COI1 with high affinity in the presence of coronatine. Crystal structure analysis has determined that coronatine and JA-Ile enhance the interaction between COI1 and JAZ1 via binding to a specific binding pocket on COI1. Attachment of one or more JAZ1 peptide tags as disclosed herein to a target protein in a non-plant cell expressing COI1 or a homolog thereof results in degradation of the target protein following addition of a molecule that binds the coronatine/JA-Ile binding pocket on COI1. Therefore, provided herein are compositions, methods, and kits for targeted protein degradation.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09297017&OS=09297017&RS=09297017
owner: University of Washington
number: 09297017
owner_city: Seattle
owner_country: US
publication_date: 20110608
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["RELATED APPLICATIONS","STATEMENT OF GOVERNMENT INTEREST","BACKGROUND","SUMMARY","DETAILED DESCRIPTION","EXAMPLES","Example 1","Effect of Coronatine on COI1 Binding to JAZ1 and JAZ6","Example 2","Characterization of JAZ1 Peptides","Example 3","Structural Relationship of COI1\/JAZ1 and Coronatine","Example 4","Identification of COI1 Cofactor","Example 5","Targeted Degradation of a Target Protein","REFERENCES"],"p":["The present application claims priority to U.S. Provisional Patent Application No. 61\/352,758, filed Jun. 8, 2010, the disclosure of which is incorporated by reference herein in its entirety.","This invention was made with government support under grant number 2RO1CA107134 awarded by National Institutes of Health. The government has certain rights in the invention.","Reverse genetic approaches are a powerful laboratory tool for determining the function of a target protein. The target protein is \u201cknocked down,\u201d and cellular changes are observed in order to infer the normal function of the knocked down target. Methods for knocking down a target protein by manipulating DNA transcription and RNA translation are well established. Among the most commonly used are gene knockout in whole animals and degradation of target mRNA using siRNA and shRNA techniques. However, delivery of RNA molecules can be cumbersome, and these methods often cannot achieve 100% efficiency.","Although several methods exist for knocking down a protein target at the transcription and translation levels, there are very few options for knocking down a protein target once the protein has been made. Such methods are desirable because most small molecule therapeutics operate by manipulating proteins directly. Therefore, targeted degradation techniques that operate at the protein level provide the best tool for examining the potential effects of small molecule therapeutics.","The ideal protein knock down system would function at the protein level, and would be capable of tightly controlling protein levels in a temporal manner. Temporal control of protein levels allows down-regulation of proteins that are essential for the full development of a system or pathway, and can be used to study dynamic biological functions which are otherwise difficult to manipulate.","In certain embodiments, methods are provided for targeted protein degradation. In certain of these embodiments, the protein targeted for degradation is tagged with one or more JAZ peptide tags as provided herein. The target protein is expressed in a non-plant host cell that also expresses the protein COI1 or a homolog thereof, and a molecule that binds the COI1\/JA-Ile binding pocket of COI1 is introduced into the cell to induce target protein degradation. In certain embodiments, the peptide tag comprises, consists of, or consists essentially of an amino acid sequence as set forth in SEQ ID NOs: 5, 6, 7, or 13. In certain embodiments, the molecule that binds the COI1\/JA-Ile binding pocket of COI1 is coronatine, JA, or a JA amino acid conjugate such as JA-Ile. In certain embodiments, an inositol pentakisphosphate cofactor may also be introduced into the cell. In certain embodiments, the non-plant host cell may be a eukaryotic cell such as a yeast or mammalian cell. In certain embodiments, the protein COI1 or a homolog thereof may be COI1 from or , or it may be a COI1 homolog from another plant or moss such as rice, tomato, grape, poplar, castor oil, corn, rubber tree, pea, wild tobacco, soybean, sorghum, wheat, or . The target protein may be either an endogenous host cell protein or an exogenous protein. In those embodiments wherein the target protein is an endogenous host cell protein, the peptide tag may be attached to the target protein by introducing a DNA sequence encoding the peptide tag adjacent to the DNA sequence encoding the target protein in the host cell, such that the target protein is expressed with the peptide tag attached. In those embodiments wherein the target protein is an exogenous protein, the target protein may be introduced into the host cell via a DNA sequence encoding the target protein and the peptide tag. In certain embodiments, target protein degradation may be halted by deactivating the molecule that binds the COI1\/JA-Ile binding pocket of COI1 or removing it from the cell. In other embodiments, target protein degradation may be halted by natural degradation of the molecule that binds the COI1\/JA-Ile binding pocket of COI1.","In certain embodiments, methods are provided for targeted protein degradation. In certain of these embodiments, the protein targeted for degradation is tagged with one or more peptide tags comprising an amino acid sequence as set forth in SEQ ID NOs:5, 6, 7, or 13. The target protein is expressed in a non-plant host cell that also expresses the protein COI1 or a homolog thereof, and coronatine or JA-Ile is introduced into the cell to induce target protein degradation. In certain embodiments, an inositol pentakisphosphate cofactor may also be introduced into the cell. In certain embodiments, the non-plant host cell may be a eukaryotic cell such as a yeast or mammalian cell. In certain embodiments, the protein COI1 or a homolog thereof may be COI1 from or , or it may be a COI1 homolog from another plant or moss such as rice, tomato, grape, poplar, castor oil, corn, rubber tree, pea, wild tobacco, soybean, sorghum, wheat, or . The target protein may be either an endogenous host cell protein or an exogenous protein. In those embodiments wherein the target protein is an endogenous host cell protein, the peptide tag may be attached to the target protein by introducing a DNA sequence encoding the peptide tag adjacent to the DNA sequence encoding the target protein in the host cell, such that the target protein is expressed with the peptide tag attached. In those embodiments wherein the target protein is an exogenous protein, the target protein may be introduced into the host cell via a DNA sequence encoding the target protein and the peptide tag. In certain embodiments, target protein degradation may be halted by deactivating coronatine or JA-Ile or removing them from the cell. In other embodiments, target protein degradation may be halted by natural degradation of coronatine or JA-Ile.","In certain embodiments, methods are provided for targeted protein degradation in a non-plant host cell by fusing a target protein to a peptide tag as provided herein, introducing a DNA sequence encoding COI1 or a homolog thereof into the host cell, culturing the host cell under conditions that result in the expression of the target protein and COI1 or a homolog thereof, and introducing a molecule that binds the COI1\/JA-Ile binding pocket of COI1 into the host cell. In certain embodiments, the peptide tag comprises, consists of, or consists essentially of an amino acid sequence as set forth in SEQ ID NOs:5, 6, 7, or 13. In certain embodiments, the molecule that binds the COI1\/JA-Ile binding pocket of COI1 is coronatine, JA, or a JA amino acid conjugate such as JA-Ile. In certain embodiments, an inositol pentakisphosphate cofactor may also be introduced into the cell. In certain embodiments, the non-plant host cell may be a eukaryotic cell such as a yeast or mammalian cell. In certain embodiments, the protein COI1 or a homolog thereof may be COI1 from or , or it may be a COI1 homolog from another plant or moss such as rice, tomato, grape, poplar, castor oil, corn, rubber tree, pea, wild tobacco, soybean, sorghum, wheat, or . The target protein may be either an endogenous host cell protein or an exogenous protein. In those embodiments wherein the target protein is an endogenous host cell protein, the peptide tag may be attached to the target protein by introducing a DNA sequence encoding the peptide tag adjacent to the DNA sequence encoding the target protein in the host cell, such that the target protein is expressed with the peptide tag attached. In those embodiments wherein the target protein is an exogenous protein, the target protein may be introduced into the host cell via a DNA sequence encoding the target protein and the peptide tag. In certain embodiments, target protein degradation may be halted by deactivating the molecule that binds the COI1\/JA-Ile binding pocket of COI1 or removing it from the cell. In other embodiments, target protein degradation may be halted by natural degradation of the molecule that binds the COI1\/JA-Ile binding pocket of COI1.","In certain embodiments, methods are provided for targeted protein degradation in a non-plant host cell by fusing a target protein to a peptide tag comprising, consisting of, or consisting essentially of an amino acid sequence as set forth in SEQ ID NOs:5, 6, 7, or 13, introducing a DNA sequence encoding COI1 or a homolog thereof into the host cell, culturing the host cell under conditions that result in the expression of the target protein and COI1 or a homolog thereof, and introducing coronatine or JA-Ile into the host cell. In certain embodiments, an inositol pentakisphosphate cofactor may also be introduced into the cell. In certain embodiments, the non-plant host cell may be a eukaryotic cell such as a yeast or mammalian cell. In certain embodiments, the protein COI1 or a homolog thereof may be COI1 from or , or it may be a COI1 homolog from another plant or moss such as rice, tomato, grape, poplar, castor oil, corn, rubber tree, pea, wild tobacco, soybean, sorghum, wheat, or . The target protein may be either an endogenous host cell protein or an exogenous protein. In those embodiments wherein the target protein is an endogenous host cell protein, the peptide tag may be attached to the target protein by introducing a DNA sequence encoding the peptide tag adjacent to the DNA sequence encoding the target protein in the host cell, such that the target protein is expressed with the peptide tag attached. In those embodiments wherein the target protein is an exogenous protein, the target protein may be introduced into the host cell via a DNA sequence encoding the target protein and the peptide tag. In certain embodiments, target protein degradation may be halted by deactivating coronatine or removing it from the cell. In other embodiments, target protein degradation may be halted by natural degradation of coronatine.","In certain embodiments, methods are provided for targeted protein degradation in a host animal by introducing a DNA sequence encoding the target protein linked to a peptide tag as provided herein and another DNA sequence encoding COI1 or a homolog thereof, expressing the tagged target protein and COI1, and then administering a molecule that binds the COI1\/JA-Ile binding pocket of COI1 to the animal. In certain embodiments, an inositol pentakisphosphate cofactor may also be introduced into the cell. In certain embodiments, the peptide tag comprises, consists of, or consists essentially of an amino acid sequence as set forth in SEQ ID NOs:5, 6, 7, or 13. In certain embodiments, the molecule that binds the COI1\/JA-Ile binding pocket of COI1 is coronatine, JA, or a JA amino acid conjugate such as JA-Ile. In certain embodiments, the animal is a mammal, and in certain of these embodiments the animal is a mouse. In certain embodiments, the protein COI1 or a homolog thereof may be COI1 from or , or it may be a COI1 homolog from another plant or moss such as rice, tomato, grape, poplar, castor oil, corn, rubber tree, pea, wild tobacco, soybean, sorghum, wheat, or ","In certain embodiments, methods are provided for targeted protein degradation in a host animal by introducing a DNA sequence encoding the target protein linked to a peptide tag comprising, consisting of, or consisting essentially of an amino acid sequence as set forth in SEQ ID NOs:5, 6, 7, or 13 and another DNA sequence encoding COI1 or a homolog thereof, expressing the tagged target protein and COI1, and then administering coronatine or JA-Ile to the animal. In certain embodiments, an inositol pentakisphosphate cofactor may also be introduced into the cell. In certain embodiments, the animal is a mammal, and in certain of these embodiments the animal is a mouse. In certain embodiments, the protein COI1 or a homolog thereof may be COI1 from or , or it may be a COI1 homolog from another plant or moss such as rice, tomato, grape, poplar, castor oil, corn, rubber tree, pea, wild tobacco, soybean, sorghum, wheat, or ","In certain embodiments, non-plant host cells are provided that comprise a DNA sequence encoding a target protein linked to a peptide tag comprising, consisting of, or consisting essentially of an amino acid sequence as set forth in SEQ ID NOs:5, 6, 7, or 13 and another DNA sequence encoding COI1 or a homolog thereof. In certain embodiments, the cells further comprise an inositol pentakisphosphate cofactor. In certain embodiments, the non-plant host cell is a eukaryotic cell, and in certain of these embodiments the non-plant host cell is a yeast or mammalian cell. In certain embodiments, the protein COI1 or a homolog thereof may be COI1 from or , or it may be a COI1 homolog from another plant or moss such as rice, tomato, grape, poplar, castor oil, corn, rubber tree, pea, wild tobacco, soybean, sorghum, wheat, or ","In certain embodiments, methods are provided for targeting an endogenous target protein in a non-plant host cell for coronatine- or JA-Ile-induced degradation by introducing a DNA sequence encoding a peptide tag comprising, consisting of, or consisting essentially of an amino acid sequence as set forth in SEQ ID NOs:5, 6, 7, or 13 such that the DNA sequence is inserted adjacent to the gene encoding the endogenous target protein, and such that the target protein is expressed fused to the peptide tag.","In certain embodiments, peptides are provided for tagging a target protein for degradation, and in certain of these embodiments, the peptide tags comprise, consist of, or consist essentially of an amino acid sequence as set forth in SEQ ID NOs:5, 6, 7, or 13. Also provided in certain embodiments are isolated nucleic acid sequences encoding these peptide tags, as well as the use of the peptide tags in tagging target proteins for degradation.","In certain embodiments, kits are provided for targeted protein degradation. In certain embodiments, these kits may include one or more of the following: an isolated nucleic acid encoding a peptide tag as provided herein, an isolated nucleic acid encoding COI1 or a homolog thereof, and a molecule that binds the COI1\/JA-Ile binding pocket of COI1. In certain embodiments, the kit may further comprise a target protein or an isolated nucleic acid encoding a target protein, and\/or an inositol pentakisphosphate cofactor. In certain embodiments, the peptide tag comprises, consists of, or consists essentially of an amino acid sequence as set forth in SEQ ID NOs:5, 6, 7, or 13. In certain embodiments, the molecule that binds the COI1\/JA-Ile binding pocket of COI1 is coronatine, JA, or a JA amino acid conjugate such as JA-Ile. In certain embodiments, the kit further comprises instructions for use and\/or other printed materials.","The following description of the invention is merely intended to illustrate various embodiments of the invention. As such, the specific modifications discussed are not to be construed as limitations on the scope of the invention. It will be apparent to one skilled in the art that various equivalents, changes, and modifications may be made without departing from the scope of the invention, and it is understood that such equivalent embodiments are to be included herein.","The following abbreviations are used herein: COR, coronatine; cpm, counts per minute; JA, jasmonic acid; JA-Ile, jasmonyl-L-isoleucine; JAZ, jasmonate ZIM-domain; LRR, leucine-rich repeat; ppm, parts per million; SCF, Skp1-Cullin-F box protein; sdm, site-directed mutants.","There are currently no satisfactory methods for knocking down a protein target in a temporally controlled manner using targeted protein degradation. Several previous attempts at developing such a method have utilized the ubiquitin proteasome system. However, all of these methods have serious drawbacks.","One method that employs the ubiquitin proteasome system utilizes peptide-small molecule hybrids (\u201cprotacs\u201d). These chimeric molecules encourage binding of a target protein to the SCFcomplex, resulting in proteolysis of the target (Sakamoto 2000). However, the protacs used in these methods are not membrane-permeable, and therefore require modifications to increase cell permeability or the use of microinjection. Another disadvantage of this system is off-target effects that arise from \u201cswamping out\u201d SCFand blocking its ability to interact with endogenous targets. Thus, modification of endogenous SCF complexes has shown very limited usefulness as a reverse genetics tool. Another previously developed method for targeted protein degradation utilized chimeric fusions consisting of the target protein and large binding domains of proteins targeted for degradation at SCF complexes. One such system paired the target protein with the chi retinoblastoma (Rb)-binding domain derived from the human papillomavirus (HPV) oncoprotein E7 (Zhou 2000). This method showed some success in yeast and mammalian systems, but it is not inducible and requires the use of an obstructive large tag. A similar chimeric approach required engineering of the F-box protein \u03b2-TrCP with a small, phosphorylated peptide that encourages binding of a particular target molecule to the F-box protein (Zhang 2003). However, this system required a great deal of engineering in order for the chimeric F-box protein to recognize only the target of interest and not endogenous targets. Therefore, it has not been widely used.","Recently, researchers in Japan have developed a method that utilizes the and tomato auxin receptor TIR1 for rapid, hormone-induced protein degradation (Nishimura Nature Methods 2009). TIR1 integrates into the yeast and mammalian SCF scaffold and degrades proteins with the AUX\/IAA tag, the natural substrate of TIR1. However, this tag is over 20 kDa, a large tag by biochemistry standards. A smaller tag has not yet been identified. As such, the TIR1 system has limited usefulness.","The phytohormone jasmonic acid (JA) and its metabolites regulate a wide spectrum of plant physiology, participating in normal development and growth processes as well as defense responses to environmental and pathogenic stressors. JA is activated upon specific conjugation to the amino acid L-isoleucine, which produces the highly bioactive hormonal signal (3R,7S)-jasmonyl-L-isoleucine (JA-Ile). Coronatine (COR) is a virulence factor that structurally mimics JA-Ile.","The discovery of coronatine-insensitive mutants enabled the identification of COI1 as a key player in the JA pathway. COI1 is an F-box protein that functions as the substrate-recruiting module of the Skp1-Cullin-F box protein (SCF) ubiquitin E3 ligase complex. Like other E3 ligases, SCFis involved in the ubiquitination of proteins, which targets the proteins for subsequent degradation by the 26S proteasome.",{"@attributes":{"id":"p-0049","num":"0048"},"i":"Arabidopsis ","sup":"CO1 "},"The experimental results provided herein disclose the identification and characterization of a complete jasmonate receptor comprising COI1, a JAZ peptide, and inositol pentakisphosphate. Coronatine was found to bind the complexes of COI1\/JAZ1 and COI1\/JAZ6 complexes with high affinity while displaying minimal binding affinity for COI1, JAZ1, or JAZ6 alone. Crystal structure studies were used to identify a coronatine- and JA-Ile-binding pocket on COI1. Binding of these molecules to the COI1 binding pocket increases binding affinity between COI1 and JAZ1. Coronatine has been found to play a similar role in the binding of JAZ2, JAZ3, JAZ4, JAZ5, JAZ6, JAZ8, JAZ9, JAZ10, JAZ11, and JAZ12 to COI1. A single isoform of inositol pentakisphosphate (Ins(1,2,4,5,6)P) was found to co-purify with COI1, and functional assays showed that this molecule is a critical cofactor in the interaction between COI1 and JAZ proteins.","The COI1 binding region of JAZ proteins had previously been mapped to a carboxy-terminal Jas motif. To precisely map the minimal region of the Jas motif that it is required for high affinity binding of COI1 to JAZ1 in the presence of coronatine, the JAZ1 degron and various derivatives thereof were analyzed. This led to the identification of specific JAZ peptide tags with enhanced binding affinity for COI1 in the presence of coronatine.","Provided herein in certain embodiments are compositions comprising one or more JAZ peptide tags capable of binding COI1 or a homolog thereof, as well as nucleic acids encoding these JAZ peptide tags, methods of using these peptide tags to mark a target protein for degradation, and the use of these peptide tags in various methods and kits for temporally controlled protein degradation.","In certain embodiments, the JAZ peptide tags provided herein consist of, consist essentially of, or comprise the JAZ1 degron as set forth in SEQ ID NO: 1. In other embodiments, the peptide tags consist of, consist essentially of, or comprise an amino acid sequence that corresponds to the JAZ1 degron of SEQ ID NO:1 but with one or more additions, deletions, or substitutions. In certain of these embodiments, the peptide tags consist of, consist essentially of, or comprise the amino acid sequence of SEQ ID NO: 1 but with one or more deletions from the C-terminal end. In certain other of these embodiments, the peptide tags consist of, consist essentially of, or comprise the amino acid sequence of SEQ ID NO:1 but with one or more additions to the N-terminal end. In other embodiments, the peptide tags provided herein comprise a fragment of an JAZ protein other than JAZ1, such as for example JAZ2, JAZ3, JAZ4, JAZ5, JAZ6, JAZ8, JAZ9, JAZ10, JAZ11, or JAZ12.","In certain preferred embodiments, the JAZ1 peptide tags disclosed herein consist of, consist essentially of, or comprise the amino acid sequence of SEQ ID NOs:5, 6, or 7. In other preferred embodiments, the peptide tags consist of, consist essentially of, or comprise an amino acid sequence that corresponds to the amino acid sequences of SEQ ID NOs:5, 6, or 7, but wherein one or more amino acid substitutions, additions, or deletions have been introduced into the sequence. For example, in certain embodiments the peptide tags may consist of, consist essentially of, or comprise the amino acid sequence XXXXXRRXSLHRFLEKRKDRVXXXXXX(SEQ ID NO: 13), wherein X, X, X, X, and Xare each independently absent or any amino acid, Xis either Ala or Lys, and X, X, X, X, X, and Xare each independently absent or any amino acid. In certain of these embodiments, Xis either absent, Glu, or Val; Xis either absent, Leu, or Glu; Xis either absent, Pro, or Arg; Xis either absent or Ile; and Xis either absent or Ala. In certain embodiments, Xis either absent or Thr; Xis either absent or Ser; Xis either absent or Lys; Xis either absent or Ala; Xis either absent or Pro; and Xis either absent or Tyr.","The small, unobtrusive peptide tags provided herein are superior to the large chimeric tags used in previously developed methods for targeted protein destruction because they are less likely to interfere with protein function, complex formation, and subcellular localization. However, in certain embodiments, the peptide tags disclosed herein may comprise a longer sequence, such as for example the full-length JAZ1 polypeptide sequence as set forth in SEQ ID NO: 14 or the full-length JAZ10 polypeptide sequence set forth in SEQ ID NO:29. In other of these embodiments, the peptide tag may comprise the full-length JAZ2, JAZ3, JAZ4, JAZ5, JAZ6, JAZ8, JAZ9, JAZ11, or JAZ12 polypeptide sequence.","COI1 binding to a protein comprising a JAZ peptide tag as results in degradation of the protein. As disclosed herein, binding of coronatine or JA-Ile to a specific binding pocket in COI1 enhances the interaction of COI1 and the JAZ peptide. Thus, targeted protein degradation can be accomplished by tagging a target protein with a JAZ peptide tag as provided herein, then contacting the protein with COI1 in the presence of a molecule that binds to the coronatine\/JA-Ile binding pocket of COI1.","As such, provided herein in certain embodiments are methods for targeted protein degradation that utilize one or more of 1) COI1 or a homolog thereof, 2) a molecule that binds the COI1\/JA-Ile binding pocket in COI1, 3) one or more JAZ peptide tags, and, optionally, 4) an inositol pentakisphosphate cofactor. Also provided herein are compositions and kits for carrying out these methods.","The crystal structure analysis provided herein shows that coronatine and JA-Ile interact with a specific set of residues in the COI1 binding pocket that includes R85, A86, F89, L91, R348, E350, A384, Y386, R409, V411, Y444, LA69, R496, and W519 of SEQ ID NO:15. Therefore, a \u201cmolecule that binds the COI1\/JA-Ile binding pocket of COI1\u201d as used herein refers to a molecule that interacts with one or more of these residues, and more preferably with all fourteen of these residues. As used herein, a molecule \u201cinteracts\u201d with a particular COI1 binding pocket residue if, when the molecule is bound to COI1, any portion of the molecule resides within a molecular distance that is within the hydrogen bond or Van der Waals interaction radius (approximately 2.5 to 4 \u212b) of the residue. In certain embodiments of the compositions, methods, and kits provided herein, a molecule that binds the COI1\/JA-Ile binding pocket of COI1 is coronatine. In other embodiments, the molecule is JA or a JA-amino acid conjugate such as JA-Ile, JA-L-leucine, JA-L-valine, or JA-L-alanine.","In those embodiments of the methods, compositions, and kits provided herein that utilize or comprise an inositol pentakisphosphate cofactor, the inositol pentakisphosphate cofactor may be inositol-1,2,4,5,6-pentakisphosphate. In other embodiments, the cofactor may be another molecule of the myo-inositol family, such as inositol-1,4,5,6-tetrakisphosphate.","In certain embodiments of the methods provided herein, one or more JAZ peptide tags are attached to or incorporated into a target protein. In the presence of molecule that binds the COI1\/JA-Ile binding pocket of COI1, the JAZ peptide tag (and hence the target protein) binds to COI1 or a homolog thereof with high affinity, resulting in target protein degradation.","Unlike previously developed methods for targeted protein destruction, the methods provided herein utilize small molecules that are membrane-permeable and require minimal engineering. The methods provided herein are rapidly inducible and easily reversible, and they do not require the use of large, obstructive tags. Thus, these methods represent a cheap, simple means for targeted protein destruction in vivo that is significantly superior to previously developed methods.","In certain embodiments of the methods disclosed herein, the target protein is tagged with one or more JAZ peptide tags as disclosed herein. JAZ peptide tags may be attached to a target protein using any methods known in the art. For example, in certain embodiments the peptide tag may be attached to the target protein prior to target protein expression. In these embodiments, a DNA sequence encoding the peptide tag is introduced into the host cell adjacent to the DNA sequence encoding the target protein, such that the peptide tag is expressed as part of the target protein. Methods for introducing a DNA sequence encoding a peptide tag into a cell and expressing a protein attached to the peptide tag are well known in the art. In other embodiments, the peptide tag may be attached to the target protein after the target protein has been expressed.","In certain embodiments of the methods disclosed herein, the target protein is tagged with a single peptide tag. In other embodiments, the target protein is tagged with two or more peptide tags. In those embodiments wherein the target protein is tagged with two or more peptide tags, the peptide tags may have the same or different sequences.","Previously developed targeted protein destruction systems have generally utilized chimeric mammalian F-box proteins. However, the methods disclosed herein utilize the F-box protein COI1 or plant or moss homologs thereof. In certain embodiments, the methods disclosed herein utilize or COI1 comprising the amino acid sequence set forth in SEQ ID NOs: 15 and 16, respectively. Specific plant or moss homologs that may be utilized include those from rice (, SEQ ID NO:17), tomato (, SEQ ID NO:18), grape (, SEQ ID NO:19), poplar (, SEQ ID NO:20 and SEQ ID NO:21), castor oil (, SEQ ID NO:22), corn (, SEQ ID NO:23), rubber tree (, SEQ ID NO:24), pea (, SEQ ID NO:25), wild tobacco (, SEQ ID NO:26), soybean (, SEQ ID NO:27), sorghum (, SEQ ID NO:28), wheat, or . COI1 and its homologs are preferable to mammalian proteins because their high specificity for plant substrates minimizes competition and unwanted degradation of endogenous targets.","In certain embodiments, an COI1 polypeptide or a homolog thereof may be introduced directly into a cell. In other embodiments, one or more genes encoding COI1 or a homolog thereof may be introduced into a cell such that the cell expresses COI1 or a homolog thereof. In these embodiments, the one or more genes encoding COI1 or a homolog thereof may be introduced into a cell via any method known in the art, including transformation or transient or stable transfection using viral and non-viral vectors. In certain embodiments, the one or more genes may be introduced using a viral vector such as an adenoviral, retroviral, lentiviral, or baculoviral vector. Provided herein in certain embodiments are viral and non-viral vectors comprising a DNA sequence encoding COI1 or a homolog thereof, as well as methods of using these vectors to achieve expression of COI1 or a homolog thereof in a non-plant host cell. In certain embodiments, a DNA sequence encoding COI1 or a homolog thereof may be incorporated into the host cell genome. In other embodiments, COI1 or a homolog thereof may be expressed from a vector that is not incorporated into the host cell genome. The DNA sequence encoding COI1 or a homolog thereof can be placed under the control of an endogenous promoter that is naturally present in a host cell, or it may be placed under the control of an exogenous promoter that has been introduced into the cell in conjunction with the COI1 or homolog sequence. In certain embodiments, COI1 or a homolog thereof may be constitutively expressed in the host cell. In other embodiments, COI1 or a homolog thereof may be expressed in a regulated manner. In certain of these embodiments, the DNA sequence encoding COI1 or a homolog thereof may be placed under the control of an inducible promoter, such as a chemically-regulated promoter or physically-regulated promoter. In these embodiments, the inducible promoter may provide an additional layer of control over activation of targeted protein degradation.","In certain embodiments of the methods provided herein, COI1 or a homolog thereof functions in conjunction with endogenous proteins to form a functional SCFE3 ligase in the non-plant cell into which COI1 has been introduced. For example, exogenous COI1 that has been introduced into a cell may function in combination with endogenous SKP1 to form a functional E3 ligase. Since SKP1 is highly conserved among species, the resultant complex is expected to be functional in most non-plant cell types. Nonetheless, in certain embodiments, one or more DNA sequences encoding SKP1 or other E3 ligase or ubiquitin pathway components may be introduced into a cell along with the one or more genes encoding COI1 or homologs thereof. These other DNA sequences may be incorporated as part of the same vector as COI1 or a homolog thereof, or they may be introduced via one or more separate vectors.","In certain embodiments, one or more modifications may be incorporated into COI1 or a homolog thereof to enhance the interaction between COI1 or a homolog thereof and endogenous E3 ligase or other ubiquitin pathway proteins. These modifications may include one or more additions, substitutions, or deletions to the encoded COI1 or homolog sequence. Modifications may also include the addition of one or more peptide tags or the introduction of one or more covalent or non-covalent modifications.","In the methods disclosed herein, a molecule that binds the COI1\/JA-Ile binding pocket of COI1 modulates the interaction between COI1 or a homolog thereof and a target protein tagged with a JAZ peptide tag. In certain embodiments, the peptide tag will only bind COI1 or a homolog thereof in the presence of the molecule that binds the COI1\/JA-Ile binding pocket of COI1. In other embodiments, the peptide tag may bind COI1 or a homolog thereof with very low affinity in the absence of the molecule, but do so with a significantly higher affinity in the presence of the molecule. The addition to or removal of the molecule that binds the COI1\/JA-Ile binding pocket of COI1 from a cell provides a precise mechanism whereby targeted protein degradation can be turned on and off. For example, coronatine can be introduced into a cell to induce specific degradation, then withdrawn to halt degradation. Thus, the methods provided herein allow for precise temporal control of target protein degradation.","A molecule that binds the COI1\/JA-Ile binding pocket of COI1 may be introduced into a cell and\/or animal via any administration pathway known in the art. For example, the molecule can be administered to a whole animal model via oral or parenteral administration routes. Introduction of the molecule into a host cell may be carried out via a single administration or via multiple administrations over a set time period. In certain embodiments, the molecule may be steadily administered to a host cell or animal over a set period of time. Withdrawal of the molecule from the host cell may occur via natural degradation of the molecule or by active removal or deactivation, such as by introduction of a neutralizing molecule that degrades or inactivates the molecule.","The methods and compositions disclosed herein may be utilized for targeted protein degradation in any non-plant host cell, including for example eukaryotic cells such as yeast or mammalian cells. Accordingly, provided herein in certain embodiments are non-plant host cells comprising DNA sequences encoding a target protein, a JAZ peptide tag, and COI1 or a homolog thereof. Also provided herein are cell culture systems comprising such non-plant host cells.","The methods and compositions disclosed herein may also be utilized for targeted protein degradation in whole animals and animal models. Therefore, in certain embodiments these animals and animal models are also provided herein. In certain embodiments, the animal model is a mammalian animal model, and in certain of these embodiments the mammal is a rat or mouse, such as for example a knockout mouse model.","A target protein to be tagged for degradation using the compositions and methods disclosed herein may be an endogenous host cell protein. Alternatively, the target protein may be an exogenous protein that has been stably or transiently introduced into the host cell. In those embodiments of the methods disclosed herein where the methods are carried out in an animal model and wherein the target protein is an exogenous protein, DNA sequences encoding the target protein, peptide tag, and COI1 or a homolog thereof may be introduced into the animal using standard protein knockout methods known in the art. For example, the DNA sequences may be introduced into an embryonic stem cell under the control of one or more exogenous or endogenous promoters. This stem cell may be introduced into animal blastocysts, followed by selection for progeny that are homozygous for the introduction. Alternatively, the DNA sequences may be introduced into the animal at a later stage via one or more non-plant cells comprising each of these DNA sequences or by direct transfection of one or more animal cells. In those embodiments wherein the methods are carried out in an animal model and the target protein is an endogenous protein, DNA sequences encoding the peptide tag and COI1 or a homolog thereof may be introduced into the animal by transfection of one or more animal cells. In these embodiments, the DNA sequence encoding the peptide tag is introduced in such a manner that is it is expressed as a fusion tag to the target protein.","In certain embodiments of the methods provided herein, targeted protein degradation is accomplished by the steps of 1) tagging a target protein with one or more peptide tags comprising, consisting of, or consisting essentially of an amino acid sequence selected from the group consisting of SEQ ID NOs:5, 6, 7, and 13, b) expressing the target protein in a non-plant host cell, c) expressing protein COI1 or a homolog thereof in the host cell, and 4) introducing coronatine or a jasmonic acid-amino acid conjugate into the host cell, resulting in degradation of the target protein.","In certain embodiments of the methods provided herein, targeted protein degradation in a non-plant host cell is accomplished by the steps of 1) attaching a peptide tag comprising, consisting of, or consisting essentially of an amino acid sequence selected from the group consisting of SEQ ID NOs:5, 6, 7, and 13 to a target protein, 2) introducing a DNA sequence encoding COI1 or a homolog thereof into the host cell, 3) culturing the host cell under conditions that result in the expression of COI1 or a homolog thereof, and 4) introducing coronatine or a jasmonic acid-amino acid conjugate into the host cell, resulting in degradation of the target protein.","In certain embodiments, kits are provided for carrying out targeted protein degradation in a non-plant host cell. In certain embodiments, these kits comprise one or more of the following components: an isolated nucleic acid encoding a JAZ peptide tag as disclosed herein, an isolated nucleic acid encoding COI1 or a homolog thereof, a molecule that binds to the coronatine\/JA-Ile binding pocket of COI1, and\/or an inositol pentakisphosphate cofactor. In certain embodiments, the kit may further comprise a target protein or an isolated nucleic acid encoding a target protein. In certain embodiments, these kits further comprise instructions for usage.","The following examples are provided to better illustrate the claimed invention and are not to be interpreted as limiting the scope of the invention. To the extent that specific materials are mentioned, it is merely for purposes of illustration and is not intended to limit the invention. One skilled in the art may develop equivalent means or reactants without the exercise of inventive capacity and without departing from the scope of the invention. It will be understood that many variations can be made in the procedures herein described while still remaining within the bounds of the present invention. It is the intention of the inventors that such variations are included within the scope of the invention.","Various radioligand binding experiments were performed to quantify the interaction between tritium (H)-labeled coronatine and COI1\/JAZ1 or COI1\/JAZ6.","(H)-labeled coronatine was synthesized by Amersham. Full-length COI1 and ASK1 were co-expressed as a glutathione S-transferase (GST) fusion protein and an untagged protein, respectively, in Hi5 suspension insect cells. The COI1\/ASK1 complex was isolated from the soluble cell lysate by glutathione affinity chromatography. After on-column tag cleavage by tobacco etch virus protease, the complex was further purified by anion exchange and gel filtration chromatography and concentrated by ultrafiltration to 12-18 mg ml. Full-length JAZ substrate proteins were expressed as 6\u00d7His-fusion proteins in and purified on Ni-NTA resin with subsequent dialysis into 20 mM Tris-HCl, pH 8.0, 200 mM NaCl, and 10% glycerol.","Radioligand binding was assayed on purified proteins, with 2 mg COI1\/ASK1 complex and JAZ proteins at a 1:3 molar ratio. Reactions were prepared in 100 ml final volume and in a binding buffer containing 20 mM Tris-HCl 200 mM NaCl, and 10% glycerol. Saturation binding experiments were conducted with serial dilutions of 3H-coronatine in binding buffer. Nonspecific binding was determined in the presence of 300 mM coronatine. Competition binding experiments were conducted with serial dilutions of JA-Ile in the presence of 100 nM H coronatine with nonspecific binding determined in the presence of 300 mM coronatine. Total binding was determined in the presence of vehicle only. Two-point binding experiments were performed in the presence of 100 nM or 300 nM H-coronatine with nonspecific binding determined in the presence of 300 mM coronatine. Following incubation with mixing at 4\u00b0 C., all samples were collected with a cell harvester (Brandel, Gaithersburg, Md.) on polyethyleneimine (Sigma)-treated filters. Samples were incubated in liquid scintillation fluid for >1 hour before counting with a Packard Tri-Carb 2200 CA liquid scintillation analyzer (Packard Instrument Co.). Saturation binding experiments were analyzed by nonlinear regression, competition binding experiments by nonlinear regression with Kcalculation as described previously (Cheng 1973), and concentration-response data by sigmoidal dose-response curve fitting, all using GraphPad Prism version 4.00 for MacOSX.",{"@attributes":{"id":"p-0080","num":"0079"},"sup":"3","figref":["FIG. 1A","FIG. 1B","FIG. 2A","FIG. 2B"],"sub":["D ","D ","i","i "]},"These results show that the COI1\/JAZ complex, rather than COI1 alone, functions as the genuine high-affinity jasmonate receptor in a co-receptor form.","The COI1-binding region of the JAZ proteins has previously been mapped to the carboxy-terminal Jas motif, which is characterized by the SLXXFXXKRXXRXXXXXPY consensus sequence (SEQ ID NO:30) preceded by two consecutive basic residues. As shown in Example 1, mutation of the conserved phenylalanine residue to alanine is sufficient to abolish the high affinity interaction between coronatine and COI1\/JAZ6 ().","Previous studies have shown that the highly conserved PY sequence at the C-terminal end of the Jas motif plays a role in JAZ localization and stability in vivo, but that the sequence was not necessary for ligand-dependent COI1-JAZ interaction. This was confirmed by an experiment showing that truncation of the PY motif in JAZ1 had little effect on in vitro ligand binding activity (). To further map the minimal region of the Jas motif required for high affinity ligand binding with COI1, the recombinant JAZ1 protein was replaced with a set of synthetic JAZ1 peptides in a ligand binding assay.","The JAZ1 degron of SEQ ID NO:1 (R205-Y226), which spans the central conserved Jas motif plus the two additional amino-terminal basic residues, did not bind to COI1 with high affinity in the presence of coronatine (). These results indicate that residues N-terminal to Arg205 must participate in the COI1\/JAZ interaction. Various derivatives of the JAZ1 degron sequence, as well as derivatives of the JAZ6 and JAZ7 degron sequences, were analyzed for their ability to bind COI1 with high affinity in the presence of coronatine. These derivatives are set forth in Table 1.",{"@attributes":{"id":"p-0085","num":"0084"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"left"}}],"thead":{"row":[{"entry":"TABLE\u20031"},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":[{},"SEQ",{}]},{"entry":[{},"ID",{}]},{"entry":["Peptide","NO","Sequence"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["JAZ1\u2003Jas\u2003motif",{},"PIARRASLHRFLEKRKDRVTSK"]},{"entry":[{},{},"APY"]},{"entry":{}},{"entry":["JAZ1\u2003degron","\u20031","RRASLHRFLEKRKDRVTSKAPY"]},{"entry":{}},{"entry":["JAZ1\u2003+ 1\u2003extension","\u20032","ARRASLHRFLEKRKDRV"]},{"entry":{}},{"entry":["JAZ1\u2003+ 2\u2003extension","\u20033","IARRASLHRFLEKRKDRV"]},{"entry":{}},{"entry":["JAZ1\u2003+ 3\u2003extension","\u20034","PIARRASLHRFLEKRKDRV"]},{"entry":{}},{"entry":["JAZ1\u2003+ 4\u2003extension","\u20035","LPIARRASLHRFLEKRKDRV"]},{"entry":{}},{"entry":["JAZ1\u2003+ 5\u2003extension","\u20036","ELPIARRASLHRFLEKRKDRV"]},{"entry":{}},{"entry":["JAZ1\u2003+ 5\u2003extension\u2003+","\u20037","ELPIARRASLHRFLEKRKDRVT"]},{"entry":["PY\u2003motif",{},"SKAPY"]},{"entry":{}},{"entry":["JAZ1\u2003+ polyA","\u20038","AAAAARRASLHRFLEKRKDRV"]},{"entry":["extension",{},{}]},{"entry":{}},{"entry":["JAZ1\u2003+ 5\u2003extension","\u20039","VERIARRASLHRFLEKRKDRV"]},{"entry":["from\u2003JAZ6",{},{}]},{"entry":{}},{"entry":["JAZ6\u2003+ 5\u2003extension","10","VERIARRASLHRFFAKRKDRV"]},{"entry":{}},{"entry":["JAZ6\u2003+ 10","11","QQHQVVERIARRASLHRFFAKR"]},{"entry":["extension",{},"KDRV"]},{"entry":{}},{"entry":["JAZ7\u2003+ 5\u2003extension","12","YQKASMKRSLHSFLQKRSLRI"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}}}},"The JAZ1 +1, +2, +3, +4, and +5 extensions (SEQ ID NOs:2-6, respectively) added on 1, 2, 3, 4, and 5 residues, respectively, to the N-terminus of JAZ1 while removing six residues from the C-terminus. The added residues were derived from the residues that are normally present N-terminal to the degron in JAZ1. An additional JAZ1 peptide (SEQ ID NO:7) contained the same additional N-terminal amino acids as the +5 extension but with the six C-terminal residues added back on.","The JAZ1 +4 and +5 extensions (SEQ ID NOs:5 and 6, respectively) were found to bind COI1 in the presence of coronatine with a much higher affinity than the JAZ1 degron and the +1, +2, and +3 extensions, with the +5 extension exhibiting the highest degree of binding (). The JAZ1 +5 extension peptide was found to permit coronatine binding with a K(\u02dc108 nM) comparable to that of the full-length JAZ1 protein (). The JAZ1 +5 extension with the six C-terminal residues added back on also exhibited significant binding (, \u201cJAZ1-extension +5+PY\u201d).","To test the specificity of the JAZ1 peptide tags of SEQ ID NOs:5, 6, and 7, an additional set of JAZ peptide tags was developed. The first two were essentially identical to the JAZ1 +5 extension in that they contained residues 1-16 of the JAZ1 degron plus a five amino acid N-terminal extension. However, the sequence of the extension was different. The first of these (SEQ ID NO:8) utilized a polyalanine extension, while the second (SEQ ID NO:9) utilized a five amino acid extension derived from JAZ6. The remaining JAZ peptide tags were based on the degrons of JAZ6 and JAZ7. These included two JAZ6 peptides that contained the JAZ6 degron plus an additional five and ten N-terminal residues, respectively (SEQ ID NOs: 10 and 11, respectively) and one JAZ7 peptide that contained the JAZ7 degron plus an additional five N-terminal residues (SEQ ID NO: 12). None of the additional JAZ peptide tags exhibited significant binding to COI1 in the presence of coronatine (). These results suggest that the system disclosed herein maintains a great deal of selectivity for side chain chemistry, and that simply inserting five \u201cfiller\u201d amino acid residues at the N-terminus of the peptide tag is insufficient to promote COI1 binding.","To evaluate the structure mechanism by which COI1\/JAZ1 co-receptor senses jasmonate, crystal structures were obtained for COI1\/ASK1\/JAZ1 peptides complexed with either coronatine or (3R,7S)-JA-Ile.","Crystals were grown at 4\u00b0 C. by the hanging-drop vapor diffusion method with 1.5 \u03bcL protein complex samples containing COI1\/ASK1, JAZ1 peptide, and hormone compound at 1:1:1 molar ratio mixed with an equal volume of reservoir solution containing 100 mM BTP, 1.7-1.9 M ammonium phosphate, and 100 mM NaCl, pH 7.0. Diffraction quality crystals were obtained by the micro-seeding method at 4\u00b0 C. The crystals all contained eight copies of the complex in the asymmetric unit. The data sets were collected at the Advanced Light Source in Lawrence Berkeley National Laboratory as well as the GM\/CA-CAT 23 ID-B beamline at the Advanced Photon Source in Argonne National Laboratory using crystals flash-frozen in the crystallization buffers supplemented with 15-20% ethylene glycol at \u2212170\u00b0 C. Reflection data were indexed, integrated, and scaled with the HKL2000 package. All crystal structures were solved by molecular replacement using the program Phaser and the TIR1\/ASK1 structure as search model. The structural models were manually built in the program O and refined using CNS and PHENIX. All final models had 96-98% of residues in the favored region and 0% in disallowed region of the Ramachandran plot.",{"@attributes":{"id":"p-0091","num":"0090"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"center"}}],"thead":{"row":[{"entry":[{},"TABLE 2"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},{},"COI1\/ASK1\/JAZ1",{}]},{"entry":[{},"COI1\/ASK1\/JAZ1","+5 extension\/","COI1\/ASK1\/JAZ1"]},{"entry":[{},"degron\/coronatine","coronatine","+5 extension\/JA-Ile"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"63pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Data collection",{},{},{}]},{"entry":["Space group","P21","P21","P21"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"63pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Cell","a, b, c","121.8, 221.5, 148.5","123.2, 220.8, 149.5","122.3, 220.8, 148.7"]},{"entry":["dimensions","(\u212b)"]},{"entry":[{},"\u03b1, \u03b2, \u03b3","90.0, 104.5, 90.0","90.0, 104.5, 90.0","90.0, 104.5, 90.0"]},{"entry":[{},"(\u00b0)"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"right"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"right"}},{"@attributes":{"colname":"5","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"right"}},{"@attributes":{"colname":"7","colwidth":"42pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Resolution (\u212b)","2.80","(2.80-2.90)","3.35","(3.35-3.41)","3.18","(3.18-3.31)"]},{"entry":["R","0.103","(0.816)","0.119","(0.700)","0.088","(0.462)"]},{"entry":["\/\/\u03c3\/","16.7","(2.0)","14.0","(2.1)","17.2","(2.8)"]},{"entry":["Completeness (%)","100","(100)","92.9","(94.6)","97.0","(93.3)"]},{"entry":["Redundancy","3.9","(3.8)","3.6","(3.3)","3.1","(2.7)"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"63pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Refinement",{},{},{}]},{"entry":["Resolution (\u212b)","50-2.80","50-3.35","50-3.18"]},{"entry":["No. reflections","174,966","95,997","116,337"]},{"entry":["R\/R","0.235\/0.270","0.225\/0.270","0.223\/0.264"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"63pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["R.m.s","Bond","0.008","0.010","0.010"]},{"entry":["deviations","lengths"]},{"entry":[{},"(\u212b)"]},{"entry":[{},"Bond","1.676","1.271","1.556"]},{"entry":[{},"angles"]},{"entry":[{},"(\u00b0)"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},"The crystal structure of COI1 revealed a TIR1-like overall architecture, with the N-terminal tri-helical F-box motif bound to ASK1 and a C-terminal horseshoe-shaped solenoid domain formed by 18 tandem leucine-rich repeats (). Similar to TIR1, the top surface of the COI1 leucine-rich repeat (LRR) domain has three long intra-repeat loops (loops 2, 12, and 14) that are involved in hormone and polypeptide substrate binding. Unlike TIR1, however, a fourth long loop (loop C) in the C-terminal capping sequence of the COI1 LRR domain folds over loop 2, partially covering it from above ().","Despite their similar overall fold, crystal structure analysis revealed that COI1 has evolved a hormone binding site that is distinct from that of TIR1. Configured between loop 2 and the inner wall of the LRR, the ligand binding pocket of COI1 is exclusively encircled by amino acid side chains (). Many of the pocket-forming residues on COI1 are large in size and carry a polar head group (). These properties allow them to be mold a binding pocket into a specific shape while forming close interactions with each chemical moiety of the ligand. These close interactions are critical to proper hormone sensing of the complex. In the binding pocket, both JA-Ile and coronatine sit in an \u2018upright\u2019 position with the keto group of their common cyclopentanone ring pointing up and forming a triangular hydrogen bond network with R496 and Y444 of COI1 at the pocket entrance ().","Without the JAZ degron peptide bound, the keto group of the ligand is accessible to solvent (). The rest of the cyclopentanone ring of both JA-Ile and coronatine is sandwiched between the aromatic groups of F89 and Y444 of COI1, stabilized by hydrophobic packing. The cyclohexene ring of coronatine provides a rigid surface area for close packing with F89, whereas the more flexible and extended pentenyl side chain of JA-Ile is more loosely accommodated by a hydrophobic pocket formed by A86, F89 and L91 from loop 2 as well as L469 and W519 from the LRRs (). Differences at this interface probably explain the approximately tenfold higher affinity of coronatine over (3R,7S)-JA-Ile detected in binding assays. Deeper in the ligand-binding pocket, the common amide and carboxyl groups of JA-Ile and coronatine bind to the bottom of the binding site by forming a salt bridge and hydrogen bond network with three basic residues of COI1:R85, R348 and R409 (). Together, these arginine residues constitute the charged floor of the ligand pocket. Y386 reinforces the interactions from above by forming a hydrogen bond with the amine group of the ligand. In doing so, Y386 approaches the cyclopentanone ring of the ligand, narrowing the pocket entrance and creating a hydrophobic cave below. The rest of the basin is carved out by V411, A384 and the aliphatic side chain of Arg 409 (). The ethyl-cyclopropane group of coronatine and the isoleucine side chain of JA-Ile can both comfortably fit in this space due to their similar size and hydrophobicity. The nature of the cave explains the preference of COI1 for jasmonate conjugates containing a moderately sized hydrophobic amino acid. Although most of the ligand is buried inside the binding site, the keto group at the top and the carboxyl group at the bottom remain exposed, available for additional interactions with the JAZ portion of the co-receptor ().","The JAZ1 degron peptide adopts a bipartite structure with a loop region followed by an \u03b1-helix to assemble with the COI1\/JA complex. The hallmark of the JAZ1 degron is the N-terminal five amino acids identified in the radioligand binding assay. In a largely extended conformation, this short sequence lies on top of the hormone-binding pocket and simultaneously interacts with both COI1 and the ligand, effectively trapping the ligand in the pocket (). At the N-terminal end, L201 of the JAZ1 peptide is embedded in a hydrophobic cavity presented by surface loops on top of COI1 ().","At the C-terminal end, A204 of JAZ1 uses its short side chain to pack against the keto group of the ligand and F89 of COI1 (). The same alanine residue of JAZ1 also donates a hydrogen bond through its backbone amide group to the keto moiety of the ligand emerging from the pocket (). The middle region of the five-amino-acid sequence is secured to the COI1 jasmonate complex through a hydrogen bond formed between the backbone carbonyl of P202 in JAZ1 and the ligand-interacting COI1 residue R496, which is critical for the hormone-dependent COI1\/JAZ interaction (data not shown). In agreement with its important role in forming the JA-Ile co-receptor, this short N-terminal region of the JAZ degron completely covers the opening of the ligand-binding pocket, conferring high-affinity binding to the hormone. The close interaction between the hormone and the co-receptor complex provides a plausible structural explanation for the favorable binding of the (3R,7S)-JA-Ile isomer, as the stereochemistry at the 7 position of (3R,7R)-JA-Ile may place the aliphatic chain unfavorably close to nearby JAZ1 and COI1 residues ().","Within the JAZ1 degron, two conserved basic residues, R205 and R206, were previously shown to have an important role in hormone-induced COI1 binding. In the structure, R205 contributes to COI1 binding by directly interacting with loop 12, whereas R206 points in the opposite direction and inserts deeply into the central tunnel of the COI1 solenoid. Approaching the bottom of the ligand-binding pocket, the guanidinium group of the R206 side chain joins the three basic COI1 residues that form the pocket floor and interacts directly with the carboxyl group of the ligand (). Thus, the N-terminal seven amino acids (ELPIARR) of the JAZ1 degron peptide act as a clamp that wraps the ligand-binding pocket from top to bottom, closing it completely ().","The highly conserved C-terminal half of the JAZ1 degron forms an amphipathic \u03b1-helix that strengthens the JAZ1\/COI1 interaction by binding to the top surface of the COI1 LRR domain, adjacent to the ligand-binding site (). With its N-terminal end directly packing against loop 2 of COI1, the Jas motif helix blocks the central tunnel of the COI1 LRR solenoid like a plug. The N-terminal half of the Jas motif helix is characterized by three hydrophobic residues (L209, F212 and L213) which are aligned on the same side of the helix and form a hydrophobic interface with COI1 ().","By soaking the COI1-ASK1 crystals with coronatine and a sufficiently high concentration of JAZ1 degron peptide lacking the N-terminal ELPIA sequence, the complex formed by COI1, coronatine, and the isolated Jas motif helix was trapped in the crystal (Table 2). This indicates that the \u03b1-helix may provide a low-affinity anchor for docking the JAZ protein on COI1.","The crystal structure of TIR1 revealed an unexpected inositol hexakisphosphate (InsP) molecule bound in the centre of the protein underneath the auxin-binding pocket. Sequence homology between COI1 and TIR1 suggests that COI1 might contain a similar small molecule. Before crystallization, the recombinant COI1\/ASK1 complex was analyzed by structural mass spectrometry.","Nano-electrospray ionization mass spectrometry (MS) and tandem MS (MS\/MS) experiments were performed on a SynaptHDMS instrument. Before MS analysis, 50 \u03bcL of a 16 mg mlsolution of COI1\/ASK1 in 20 mM Tris-HCl, pH 8, 0.2 M NaCl, and 5 mMDTT was buffer-exchanged twice into 0.5M ammonium acetate solution using Bio-Rad Biospin columns. To improve desolvation during ionization, samples were diluted 1:4 in 0.5 M ammonium acetate, and isopropanol was added to a final concentration of 5%. Typically an aliquot of 2 mL solution was loaded for sampling via nano-ESI capillaries, which were prepared in-house from borosilicate glass tubes as described previously (Nettleton 1998). The conditions within the mass spectrometer were adjusted to preserve non-covalent interactions. The following experimental parameters were used: capillary voltage up to 1.26 kV, sampling cone voltage 150 V, and extraction cone voltage 6 V, MCP 1590. For tandem MS experiments peaks centered at m\/z 4,564 and 4,588 were selected in the quadrupole and collision energy up to 65 V was used. Argon was used as a collision gas at maximum pressure. All spectra were calibrated externally using a solution of cesium iodide (100 mg ml). Spectra are shown with minimal smoothing and without background subtraction.","Nano-electrospray mass spectra of the intact COI1\/ASK1 complex revealed two populations differing by a mass of \u02dc568 Da, indicating that a small molecule was indeed co-purified with the proteins (). As shown in , only one population of the complex corresponding to COI\/ASK1 was apparent at both at low and high collision energy. At high collision energy, the COI\/ASK1 complex dissociates into its different subunits and one population of COI1 appears in the spectrum. This population of COI1 has a calculated mass of 67,944\u00b11 Da, which is in agreement with the theoretical mass of COI1 (67,947 Da). At low collision energy, the COI1\/ASK1\/ligand complex is apparent. However, elevating the collision energy releases some of the bound ligand and results in the appearance of a stripped COI1\/ASK1 complex. The theoretical mass of the apo COI1\/ASK1 complex is 86,458 Da, which is in close agreement with the observed mass of 86,543\u00b128 Da. The mass of the COI1\/ASK1\/ligand complex was found to be 87,112\u00b115 Da, suggesting that the mass of the ligand is around 568\u00b128 Da. The fact that both masses carry a charge of +19 indicates a neutral loss of the ligand, meaning that it cannot be detected in the spectrum. At high collision energy, some of the complex dissociates into its different subunits and two populations of COI1 appear in the spectrum. The smaller form, with a calculated mass of 67,952\u00b15 Da, fits the theoretical mass of COI1 (67,946.5 Da), whereas the other population, with a calculated mass of 68,518\u00b14 Da, corresponding to COI1-ligand, suggest that the mass of the ligand is around 568\u00b15 Da.","The mass-spectrometry-derived molecular mass of the unknown compound is different from the mass of nsP(651 Da) but matches that of an inositol pentakisphosphate (InsP) molecule. Unfortunately, mass spectrometry analyses of either the native COI1\/ASK1 complex or the denatured proteins were unable to achieve direct mass analysis of the small molecule.","To investigate the identity of the unknown compound, it was first estimated that the molecule contains four or five phosphate groups by P nuclear magnetic resonance (NMR) of trypsin-digested COI1\/ASK1 complex (data not shown). To identify unequivocally the unknown molecule, steps were taken to purify it away from the COI1\/ASK1 complex in a quantity sufficient for H NMR analysis. The high phosphate content of the molecule allowed us to trace it through a multi-step purification procedure (). Phenol was melted at 68\u00b0 C. and equilibrated with equal parts 0.5 M Tris-HCl, pH 8.0 until a pH of 7.8 was reached. The equilibrated phenol was then topped with 0.1 volume 100 mM Tris-HCl, pH 8.0 and stored at 4\u00b0 C. For extraction, 30-40 mg of 1 mg mlCOI1\/ASK1 protein was mixed in small batches with equal parts equilibrated phenol at room temperature. The samples were inverted and incubated for 30 minutes until phase separation occurred. With 30 second vortexing, the samples were incubated at room temperature for 30 minutes and spun at 15,000 rpm for 5 minutes. The aqueous phase was removed as a primary extraction. Equal parts of a solution containing 25 mM Tris-HCl, pH 8.0 was added to the phenol and collected as above as a secondary extraction. The primary and secondary extractions were combined and diluted 10\u00d7 in 25 mM Tris-HCl, pH 8.0, then further purified by gravity flow on Q sepharose high-performance anion exchange resin (GE Healthcare). Following column wash with 10\u00d7 column volumes of 0.1 N formic acid, stepwise elution was performed with 23 column volumes of 0.1 N formic acid (Thermo Scientific) with increasing concentrations of ammonium formate (Sigma) from 0 to 2 M. Fractions were analyzed for phosphate content by the wet-ashing method with perchloric acid in Pyrex culture tubes (13\u00d7100 mm). Typically, samples of 50-100 \u03bcL were ashed with 100-200 mL 70% perchloric acid (purified by redistillation, Sigma). Ashing was performed by heating the sample over a Bunsen-type burner with continuous shaking to prevent bumping. When the sample stopped emitting white smoke, the reaction was considered complete and then heated to dryness. 500 \u03bcL of distilled water was added to the room temperature tubes and vortexed. 100 \u03bcL samples containing up to 10 nmol inorganic phosphate were assayed for phosphate by a modification of a published procedure (Sadrzadeh 1993). A total of 125 \u03bcL of acid molybdate color reagent was added and the samples were incubated and covered at room temperature for 12-14 hours (overnight) for full color development (total volume 225 \u03bcL). Plates were read at 650 nm and unknowns were determined from the linear regression of the standard curve (0-10 nmol NaHPOper well). All assays were done in triplicate. Final fractions containing phosphate were combined and lyophilized repeatedly to remove residual ammonium formate.","After isolation of 150 nmol of the purified small molecule, a series of one-dimensional and two-dimensional NMR data were acquired, including a highly informative homonuclear total correlation (TOCSY) spectrum. NMR spectra were acquired on a Varian INOVA600 spectrometer equipped with a cold probe using 200 \u03bcM samples of purified compound X or synthetic inositol-1,2,4,5,6-pentakisphosphate (Cayman Chemical) dissolved in DO. TOCSY spectra were acquired with mixing times of 35 or 50 ms, processed with NMRPipe and visualized with NMRView.","The observed chemical shifts and TOCSY cross-peak patterns are clearly characteristic of inositol phosphates (). A comparison with previously reported NMR spectra of various inositol phosphates established that the unknown compound is either D- or L-inositol-1,2,4,5,6-pentakisphosphate (Ins(1,2,4,5,6)P; ). This conclusion was further supported by the TOCSY spectrum of synthetic Ins(1,2,4,5,6)P() and the subsequently acquired negative ion electrospray ionization mass spectrometry spectrum of the compound ().","As shown in , the negative-ion ESI-MS spectrum of the unknown contained the major ion at m\/z 192.3 ((579.8951\u22123\u00d71.0078)\/3), corresponding to the [M-3H]ion of inositol pentakisphosphate (InsP), and the ion at m\/z 288.9 ((579.8951\u22122\u00d71.0078)\/2), corresponding to the [M-2H]ion of InsP. The [M-H] ion expected at m\/z 579.9 was absent. The ions seen at m\/z 199.7 and 207.1 correspond to the sodiated ions of InsPseen as the [M+Na-4H], and [M+2Na-5H]ions, respectively; and the ions at m\/z 299.9 and 311.9 correspond to the [M+Na-3H]and [M+2Na-4H] ions, respectively. The spectrum also contains ions at m\/z 499 ([M-H\u2014HPO]), 419 ([M-H-2HPO]), and 441 ([M+Na-2H-2HPO]), arising from various losses of the phosphate residues of the molecule. The presence of the ion at m\/z 499 (579.9-HPO) is consistent with the observation of the ions at m\/z 249 ([M-2H\u2014HPO]), 259.9 ([M+Na-3H\u2014HPO]), and 165.7 ([M-3H\u2014HPO]), representing the various deprotonated InsPseen as doubly and triply charged anions. The ion at m\/z 419 represents a deprotonated InsParising from loss of two HPOresidues; while the ion at m\/z 441 represents a monosodiated InsPanion. The presence of the ions at m\/z 419 and 441 is also consistent with the observation of the doubly charged ions at m\/z 209 and 219, corresponding to the [M-2H-2HPO]and [M+Na-3H-2HPO]ions, respectively. The assignments of the ions observed are listed in Table 3. These ions were also observed for Ins(1,2,3,4,5)Pand Ins(1,2,4,5,6)Pstandards when subjected to ESI under the same condition, indicating that the unknown compound is an InsP. This InsPstructure is further confirmed by the MSn (n=2,3,4,5) mass spectra of the [M-3H]ion at m\/z 192.3 and of the [M-2H]ion at m\/z 288.9 deriving from the unknown compound and from the Ins(1,2,3,4,5)Pand Ins(1,2,4,5,6)Pstandards.",{"@attributes":{"id":"p-0108","num":"0107"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Ions observed for IP5 by negative-ion ESI-MS"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"126pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["m\/z","Structure"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"126pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["499","[M \u2212 H \u2212 HPO]"]},{"entry":["441","[M + Na \u2212 2H \u2212 2HPO]"]},{"entry":["419","[M \u2212 H \u2212 2HPO]"]},{"entry":["311","[M + 2Na \u2212 4H]"]},{"entry":["300","[M + Na \u2212 3H]"]},{"entry":["289","[M \u2212 2H]"]},{"entry":["271","[M + 2Na \u2212 2H \u2212 2HPO]"]},{"entry":["268","[M + K \u2212 3H \u2212 2HPO]"]},{"entry":["259.9","[M + Na \u2212 3H \u2212 HPO]"]},{"entry":["249","[M \u2212 2H \u2212 HPO]"]},{"entry":["219","[M + Na \u2212 3H \u2212 2HPO]"]},{"entry":["212","[M + Na + K \u2212 5H]"]},{"entry":["209","[M \u2212 2H \u2212 2HPO]"]},{"entry":["207","[M + 2Na \u2212 5H]"]},{"entry":["203","[M + Na \u2212 4H]"]},{"entry":["199.7","[M + Na \u2212 4H]"]},{"entry":["192.3","[M \u2212 3H]"]},{"entry":["165.7","[M \u2212 3H \u2212 HPO]"]},{"entry":["97","HPO"]},{"entry":["79","PO"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"Consistent with the binding of a small molecule cofactor, the crystal structure of COI1 (Example 3) showed strong unexplained electron densities clustered in the middle of the COI1 LRR domain. Like InsPin TIR1, these extra densities in COI1 are located directly adjacent to the bottom of the ligand binding pocket of the jasmonate co-receptor, interacting with multiple positively charged COI1 residues (). Unexpectedly, these islands of electron density cannot be explained by an Ins(1,2,4,5,6)Pmolecule. Instead, their intensity, overall symmetry, and poor connectivity indicate that they belong to multiple free phosphate molecules. Because a high concentration of ammonium phosphate was used as the major precipitant for crystallizing the JA co-receptor, it was postulated that the InsPmolecule that co-purified with COI1 was later displaced by phosphate molecules in the crystallization drops. In support of this scenario, the concave surface of the COI1 solenoid fold surrounding the phosphates is highly basic and decorated with residues conserved in plant COI1 orthologs, indicating a functionally important surface area ().","The highly selective co-purification of two different inositol phosphates, InsPand InsP, with two homologous plant hormone receptors, COI1 and TIR1, implies that the proper function of the two F-box proteins might require the binding of specific inositol phosphates. To assess the functional role of Ins(1,2,4,5,6)Pin the COI1\/JAZ1 co-receptor, a protocol was developed for stripping the co-purified InsPfrom COI1 without denaturing the protein. Briefly, proteins were mixed with 10% glycerol and incubated in 2 M ammonium phosphate, 100 mM Bis-Tris propane, pH 7.0, 200 mM NaCl, and 10% glycerol at 4\u00b0 C. for >24 hours with a minimum of 3\u00d7 buffer changes at 100\u00d7 sample volume. Samples were then transferred to 20 mM Tris-HCl, pH 8.0, 200 mM NaCl, and 10% glycerol at 4\u00b0 C. for >24 hours with a minimum of three buffer changes at 100\u00d7 sample volume. Inositol phosphate rescue experiments were conducted according to the radioligand binding assays described above in the presence of 300 nM H-coronatine with nonspecific binding determined in the presence of 300 \u03bcM coronatine.","The resulting COI1\/ASK1 complex was tested in a ligand-binding based reconstitution assay. As shown in , untreated COI1 formed a high-affinity jasmonate co-receptor with JAZ1. Addition of exogenous Ins(1,2,4,5,6)Pdid not significantly change its activity. In contrast, the dialyzed COI1 sample completely lacked ligand binding by itself and showed only trace activity in the presence of JAZ1. Supplementation with either synthetic Ins(1,2,4,5,6)P() or the purified and NMR analyzed InsPsample (data not shown) rescued the interaction in a dose-dependent manner and with a half-maximum effective concentration (EC) of 27 nM (). From this reconstitution result, it was concluded that Ins(1,2,4,5,6)Pbinding is crucial for the jasmonate coreceptor to perceive the hormone with high sensitivity.","A close examination of the phosphate molecules in the available COI1 structure indicates a mechanism by which the inositol phosphate molecule may modulate the activity of the jasmonate co-receptor. Among four COI1-bound phosphates, one stands out by binding at a critical position in the jasmonate co-receptor. This phosphate molecule interacts simultaneously with four basic residues at the bottom of the ligand-binding pocket, namely Arg 206 in the JAZ1 degron and the three COI1 arginine residues that form the floor of the pocket. As a result, a tetragonal bipyramidal interaction network is formed among four molecules at the core of the jasmonate co-receptor assembly. The four arginines from COI1 and JAZ1 sit at the four corners of the central plane, interacting with the hormone above and the phosphate below ().","As the free phosphate molecule probably mimics the action of a phosphate group on InsP, this four-molecule junction, together with additional phosphate-COI1 interactions seen in the crystal, conceivably represents the structural basis for InsPpotentiation of the jasmonate coreceptor. Consistent with this interpretation, coronatine-induced formation of a COI1\/JAZ1 complex was readily abolished by mutation of select COI1 residues adjacent to the phosphates, but not in contact with the hormone ().","The reconstitution assay was used to further investigate the specificity of jasmonate co-receptor regulation by inositol phosphates (). Notably, inositol-1,4,5,6-tetrakisphosphate supports the activity of the COI1\/JAZ1 co-receptor, whereas the second messenger signaling molecule inositol-1,4,5-trisphosphate does not. Addition of a phosphate to InsP, which gives rise to InsP, is also not favorable for activity. Although saturation binding of H-coronatine is stimulated by both Ins(1,2,4,5,6)Pand InsPwith similar 1Kvalues (30 nM and 37 nM, respectively), the two inositol phosphates yield markedly different Bvalues for coronatine binding, indicating that InsPis significantly less efficacious in activating the co-receptor despite having equal affinity as Ins(1,2,4,5,6)P(). Functional selectivity of COI1 for the inositol phosphate cofactor is consistent with the conservation of the putative inositol-phosphate-binding site, which is distinct in amino acid sequence from the InsP-binding site in TIR120 ().","Green fluorescent protein (GFP) will be tagged with the JAZ1 +5 extension peptide tag of SEQ ID NO:6 in budding yeast cells (e.g., ) and\/or mammalian cells. Where budding yeast cells are used, the tagged protein construct will be cloned into a standard yeast shuttling plasmid under the control of a strong, stable promoter, and the plasmid will be stably inserted into the yeast genome via chromosomal recombination sequences using methods well known in the art. Where mammalian cells are used, the gene encoding the tagged protein construct will be introduced via transient transfection or stable cell line generation.","The cells will be further engineered to express COI1 or a homolog thereof under the control of an inducible promoter. For example, exogenous COI1 expression may be placed under the control of a galactose promoter, such that expression may be controlled by sugar ratio.","After stable GFP signal has been monitored qualitatively using standard techniques such as microscopy and\/or quantitatively using techniques such as standard plate readers and\/or flow cytometry methods, expression of COI1 or a homolog thereof will be induced. COI1 expression should not significantly affect GFP signal levels.","Cells will be treated with titrating levels of coronatine. Cells will be harvested and fixed at various timepoints, and GFP signal will be quantified to determine the rate of GFP degradation. Degradation will increase as coronatine levels increase.","Additional experiments may be performed using one or more of the other JAZ1 peptide tags disclosed herein to determine the efficacy of slight changes to the peptide sequence. Similarly, additional experiments may be performed using molecules other than coronatine that bind to the COI1\/JA-Ile binding pocket of COI1.","As stated above, the foregoing is merely intended to illustrate various embodiments of the present invention. The specific modifications discussed above are not to be construed as limitations on the scope of the invention. It will be apparent to one skilled in the art that various equivalents, changes, and modifications may be made without departing from the scope of the invention, and it is understood that such equivalent embodiments are to be included herein. All references cited herein are incorporated by reference as if fully set forth herein.",{"@attributes":{"id":"p-0121","num":"0000"},"ul":{"@attributes":{"id":"ul0001","list-style":"none"},"li":["1. Adams Acta Crystallogr D 58:1948-1954 (2002)","2. Browse Annu Rev Plant Biol 60:183-205 (2009)","3. Br\u00fcnger Acta Crystallogr D 54:905-921 (1998)","4. Cheng Biochem Pharmacol 22:3099-3108 (1973)","5. Chini Nature 448:666-671 (2007)","6. Chung Plant Cell 21:131-145 (2009)","7. Chung Plant J 63:613-622 (2010)","8. Delaglio J Biomol NMR 6:277-293 (1995)","9. Dharmasiri Nature 435:441-445 (2005)","10. Feys Plant Cell 6:751-759 (1994)","11. Fonseca Nature Chem Biol 5:344-350 (2009)","12. Grunewald EMBO Rep 10:923-928 (2009)","13. Johnson Methods Mol Biol 278:313-352 (2004)","14. Jones Acta Crystallogr A 47:110-119 (1991)","15. Katsir Proc Natl Acad Sci USA 105:7100-7105 (2008)","16. Kepinski Nature 435:446-451 (2005)","17. Koo Plant J 59:974-986 (2009)","18. Lorenzo Plant Cell 16:1938-1950 (2004)","19. Melcher Nature 462:602-608 (2009)","20. Melotto Plant J 55:979-988 (2008)","21. Miyazono Nature 462:609-614 (2009)","22. Murase Nature 456:459-463 (2008)","23. Nettleton J Mol Biol 281:553-564 (1998)","24. Nishimura Nature Methods 6:917-922 (2009)","25. Nishimura Science 326:1373-1379 (2009)","26. Ogawa Tetrahedr Lett 49:7124-7127 (2008)","27. Sadrzadeh J Pharmacol Toxicol Methods 30:103-110 (1993)","28. Sakamoto Proc Natl Acad Sci USA 98:8554-8559 (2000)","29. Santiago Nature 462:665-668 (2009)","30. Sheard Nature 468:400-405 (2010)","31. Shimada Nature 456:520-523 (2008)","32. Staswick Plant Cell 16:2117-2127 (2004)","33. Stephens Biochem J 275:485-499 (1991)","34. Suza Planta 227:1221-1232 (2008)","35. Tan Nature 446:640-645 (2007)","36. Thines Nature 448:661-665 (2007)","37. Xie Science 280:1091-1094 (1998)","38. Yan Plant Cell 19:2470-2483 (2007)","39. Yan Plant Cell 21:2220-2236 (2009)","40. Yin Nature Struct Mol Biol 16:1230-1236 (2009)","41. Zhang Proc Natl Acad Sci USA 100:14127-14132 (2003)","42. Zhou Mol Cell 6:751-756 (2000)"]}}],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF DRAWINGS","p":[{"@attributes":{"id":"p-0017","num":"0016"},"figref":"FIG. 1","sup":["3","3"]},{"@attributes":{"id":"p-0018","num":"0017"},"figref":"FIG. 2","sup":["3","3"],"sub":"i "},{"@attributes":{"id":"p-0019","num":"0018"},"figref":"FIG. 3","sup":"3"},{"@attributes":{"id":"p-0020","num":"0019"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0021","num":"0020"},"figref":"FIG. 5","sub":"D "},{"@attributes":{"id":"p-0022","num":"0021"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0023","num":"0022"},"figref":"FIG. 7","i":"Arabidopsis "},{"@attributes":{"id":"p-0024","num":"0023"},"figref":"FIG. 8","sub":["o","c ","o","c "]},{"@attributes":{"id":"p-0025","num":"0024"},"figref":"FIG. 9","i":["Arabidopsis thaliana ","Arabidopsis thaliana","Solanum lycopersicum","Zea mays","; Oryza sativa","Vitis vinifera","; Hevea brasiliensis","Ricinus communis","Pisum sativum","Arabidopsis thaliana "]},{"@attributes":{"id":"p-0026","num":"0025"},"figref":"FIG. 10"},{"@attributes":{"id":"p-0027","num":"0026"},"figref":"FIG. 11"},{"@attributes":{"id":"p-0028","num":"0027"},"figref":"FIG. 12"},{"@attributes":{"id":"p-0029","num":"0028"},"figref":"FIG. 13"},{"@attributes":{"id":"p-0030","num":"0029"},"figref":"FIG. 14","sup":["3","3"],"sub":["5","5 ","6 ","d "]},{"@attributes":{"id":"p-0031","num":"0030"},"figref":"FIG. 15"},{"@attributes":{"id":"p-0032","num":"0031"},"figref":["FIG. 16","FIG. 24","FIG. 24"]},{"@attributes":{"id":"p-0033","num":"0032"},"figref":"FIG. 17"},{"@attributes":{"id":"p-0034","num":"0033"},"figref":"FIG. 18","sub":"5"},{"@attributes":{"id":"p-0035","num":"0034"},"figref":"FIG. 19","sub":"5 "},{"@attributes":{"id":"p-0036","num":"0035"},"figref":"FIG. 20","sub":"5 "},{"@attributes":{"id":"p-0037","num":"0036"},"figref":"FIG. 21","sub":["o","c ","5 ","5"]},{"@attributes":{"id":"p-0038","num":"0037"},"figref":"FIG. 22","i":"A. thaliana, H. brasiliensis, R. communis, P. trichocarpa, V. cinifera, P. sativum, S. lycopersicum, Z. mays, O. sativa"},{"@attributes":{"id":"p-0039","num":"0038"},"figref":"FIG. 23","sup":["3","3","3"],"sub":["5 ","5","5 ","5 ","5 ","50 "]},{"@attributes":{"id":"p-0040","num":"0039"},"figref":"FIG. 24"},{"@attributes":{"id":"p-0041","num":"0040"},"figref":"FIG. 25","sub":["o","c "]}]},"DETDESC":[{},{}]}
